
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01438-9'>‘Orwellian': planetary scientists outraged over deletion of research records</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 15:24:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar Among the conference abstracts deleted from a planetary science archive site recently is work involving chemistry measurements made by NASA's Mars Curiosity rover (pictured, on Mars). You have full access to this article via your institution. Over the past several weeks, hundreds of meeting abstracts have quietly vanished from the websites of research conferences organized by the Lunar and Planetary Institute (LPI) in Houston, Texas — a NASA-funded research institution that maintains key archives and information about US planetary science. The abstracts have a common theme: diversity, equity and inclusion (DEI). These topics were banned from receiving government funding by a 20 January executive order from US President Donald Trump, who called them “illegal and immoral discrimination programs”. “This is just egregious censorship of science,” says Ingrid Daubar, a planetary scientist at Brown University in Providence, Rhode Island, who co-authored several of the deleted abstracts. Several scientists told Nature that they think the LPI and its parent organization, the Universities Space Research Association (USRA) in Washington DC, have over-interpreted the Trump executive order as retroactively applying to independently performed work in public archives. In a statement provided to Nature, the USRA said it “believes that compliance with Administration directives requires DEI content to be out of public view”, unless it is “advised by an authoritative federal body that we can restore that content”. Many federal agencies, including NASA, have removed DEI-related content from their websites in response to the executive order. But the LPI is not a government institution; it was set up in 1968, during the Apollo era, primarily to foster relationships between university scientists and NASA. The missing meeting abstracts include research on workforce issues, such as how to build the most effective team of scientists for a mission. David Trang, a planetary scientist at the Space Science Institute who is based in Honolulu, Hawaii, had his work on improving mental health among planetary scientists deleted. Removing these abstracts “takes away our culture”, he says. Five key climate and space projects on Trump's chopping block How to know whether a conference is right for you ‘Anxiety is palpable': detention of researchers at US border spurs travel worries We moved a conference halfway around the globe to avoid visa discrimination How political attacks could crush the mRNA vaccine revolution Trump gutted two landmark environmental reports — can researchers save them? P hacking — Five ways it could happen to you Renewal of NIH grants linked to more innovative results, study finds Exclusive: documents reveal how NIH will axe climate studies Five key climate and space projects on Trump's chopping block An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64712954/free-wheeling-atoms/'>For the First Time, Scientists Caught Atoms Freely Interacting in Space—and It Was Stunning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. It turns out they do exactly what we thought. Enigmatic because of their quantum nature—meaning that they behave as both particles and waves—atoms had never been visualized freely interacting with each other. Now, a team of researchers from MIT have finally managed to capture their interactions for the first time. Led by physicist Martin Zwierlein, the team developed a technique that allowed them to freeze atoms in place and put them in the spotlight with lasers. They were able to successfully image bosons of a sodium isotope and fermions of a lithium isotope before these atoms scattered again. Both isotopes were in the form of quantum gases. Bosons were previously predicted to bunch together in a wave form, exhibiting the wave part of quantum behavior, while fermions were predicted to repel those like them and form pairs with different fermions, which illustrates particle behavior. The probability of finding bosons near each other is high, while there is a low chance of seeing fermions as close together, and the gases needed to be at ultracold temperatures to see anything at all (because atoms get the zoomies when agitated by heat). Their new technique, known as non-resolved microscopy, involves using a laser beam to trap the atoms in one place. There was some difficulty in lighting these atoms up, since too much heat would send them flying all over. As a result, this is the first time Zwierlein managed to freeze the motion of strongly interacting atoms in situ, and he was able to capture images of both bosons and fermions. Atoms in this type of wave show wave-like behavior as they keep changing in time and space, but the properties of those atoms are difficult to observe. The image of bosons shows atoms bunching together with wavelike trails of light behind them, indicating that they are moving in waves. To image a fermion cloud, the team needed two types of fermions—they wanted to capture pairs, and fermions of only one type would end up repelling each other. Zwierlein plans to continue using non-resolved microscopy to understand more about the physical world and potentially image stranger, more exotic quantum phenomena. But when you see pictures like these, it's showing in a photograph, an object that was discovered in the mathematical world,” Richard Fletcher, a coauthor of the study and physicist from MIT, said in a recent press release. “So it's a very nice reminder that physics is about physical things. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. A New DNA Test Could Solve the Lindbergh Baby Case Cats and Dogs Look Like Alike Now Thanks to Humans</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/linguists-find-proof-of-sweeping-language-pattern-once-deemed-a-hoax/'>Linguists Find Proof of Sweeping Language Pattern Once Deemed a ‘Hoax'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Linguists Find Proof of Sweeping Language Pattern Once Deemed a ‘Hoax' Inuit languages really do have many words for snow, linguists found—and other languages have conceptual specialties, too, potentially revealing what a culture values In 1884 the anthropologist Franz Boas returned from Baffin Island with a discovery that would kick off decades of linguistic wrangling: by his count, the local Inuit language had four words for snow, suggesting a link between language and physical environment. In a 1991 essay, British linguist Geoff Pullum called these claims a “hoax,” citing the work of linguist Laura Martin, who tracked the misinformation's evolution. He likened it to the xenomorph from Alien, a creature that “seemed to spring up everywhere once it got loose on the spaceship, and was very difficult to kill.” His acerbic critique rendered the subject taboo for a generation, says Victor Mair, an expert on Chinese language at the University of Pennsylvania. But now, he says, “it's coming back in a legitimate way.” In a sweeping new computational analysis of world languages, researchers not only confirmed the emphasis on snow in the Inuit language Inuktitut but also uncovered many similar patterns: what snow is to the Inuit, lava is to Samoans and oatmeal to Scots. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The researchers analyzed bilingual dictionaries between English and more than 600 languages, looking for what they call “lexical elaboration,” in which a language has many words related to a core concept. But this study brings a twist: rather than the number of words, it measured their proportion, the slice of dictionary real estate taken up by a concept. This produced elaboration scores for hundreds of concepts, from “abandonment” to “zoo,” based on how many times the English words for those concepts appeared in the definitions of foreign words. Often the elaboration is clearly a product of environment—small wonder that Arabic, Farsi and Indigenous Australian languages abound with words to describe the desert, and Sanskrit, Tamil and Thai with words for elephants. What historical and cultural circumstances lead a language down such obscure paths? At its boldest, linguistic relativity asserts that language determines how we perceive things, causing speakers of different languages to experience the world in radically different ways (think of the movie Arrival, in which a character becomes clairvoyant after learning an alien language). But in Mair's opinion, this study supports a softer claim: our brains all share the same basic machinery for perceiving the world, which language can subtly affect but not restrict. Similarly, Lynne Murphy, a linguist at the University of Sussex in England, who was not involved in this study, notes that “any language should be able to talk about anything.” We may not have the Marshallese word jatbo, but four words of English do the trick—“smell of damp clothing.” It's not that having many precise words for smell reveals mind-blowing cognitive abilities for processing smell; it's simply that single words are more efficient than phrases, so they tend to represent common subjects of discussion, highlighting areas of cultural significance. If we routinely needed to talk about the smell of damp clothing, we'd whittle that unwieldy phrase down to something like jatbo. Still, “lexical elaboration alone cannot tell us about the culture of its speakers,” at least not with certainty, says study lead author Temuulen Khishigsuren, a Ph.D. candidate at the University of Melbourne. And because this analysis was based on dictionaries, it comes with the biases and limitations of the lexicographers that wrote them. As Murphy puts it, they “offer only snapshots of a language at a particular time, from a particular angle.” Some of the dictionaries used are decades or centuries old, and they may reflect the archaic concerns of colonizers—to translate the Bible or establish a trade route—as much as those of modern-day speakers. Because dictionaries don't represent how people use language in the real world, the next step would be to measure how often people actually talk or write about the concepts being studied, such as snow and smells and elephants. This is difficult for languages without large bodies of written text but could be possible for many languages, especially those used heavily on social media. It bears remembering that these lexical elaborations come from comparison between languages—French only has “many” words for futility because other languages have fewer. If we find the patterns of elaboration in other languages unusual, it's safe to assume their speakers will return the favor. “English is as ‘different' as any other language,” Murphy says, which raises the question: “If we had started from, say, Spanish or Chinese or Malayalam, which concepts would have stood out for English?” Cody Cottier is a freelance journalist based in Fort Collins, Colo.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01662-5'>Import mechanism of peroxisomal proteins with an N-terminal signal sequence</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 10:25:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Most proteins imported into peroxisomes use a carboxy-terminal PTS1 signal, which is recognized by soluble receptors that transport the cargo through a nuclear pore-like conduit in the peroxisomal membrane formed by the tyrosine and glycine-rich YG domain of PEX13. Some peroxisomal proteins instead use an amino-terminal PTS2 signal that is recognized by an adaptor called PEX7, but how they are imported is poorly understood. Here we show that PTS2 cargo is moved through the YG phase by PEX7 bound to a receptor. After cargo release inside peroxisomes, PEX7 returns to the cytosol by moving back on its own through the YG phase. The chaperone PEX39 then extracts PEX7 from the phase on the cytosolic side and helps to reload PEX7 with a new receptor and cargo to start another import cycle. Our results provide a comprehensive model of PTS2 protein import. Peroxisomes are conserved organelles essential for human health. They consist of a single membrane enclosing a luminal matrix1 and play crucial metabolic roles that include fatty-acid β-oxidation and ether-lipid biosynthesis2. Peroxisomal matrix proteins are made in the cytosol and then imported, remarkably, in a folded state and even as oligomers or preformed complexes3, in contrast to the endoplasmic reticulum and mitochondria that can only import unfolded polypeptides4,5. Most peroxisomal matrix proteins contain a type 1 peroxisomal targeting signal (PTS1) that is located at their carboxy (C) terminus and comprises the amino-acid sequence SKL (or variants thereof)6. Some matrix proteins, however, use the more complex PTS2 signal located at their amino (N) terminus7. PTS2 proteins include the enzyme thiolase required for peroxisomal β-oxidation in nearly all eukaryotes8 as well as enzymes that detoxify branched-chain fatty acids and produce ether-lipid precursors essential for myelination2. Defective PTS2 protein import in humans causes Refsum disease9 and the life-threatening disorder rhizomelic chondrodysplasia punctata10. The import mechanism of PTS1 proteins is reasonably well understood11. Their PTS1 signal is recognized in the cytosol by the tetratricopeptide-repeat (TPR) domain of PEX5 (ref. The conduit resembles the selective phase formed by the phenylalanine and glycine-rich nucleoporin FG repeats in nuclear pores19 and can be reconstituted in vitro as a hydrogel into which PEX5 can drag folded cargo17. Passage through the phase requires conserved motifs in PEX5 comprising the amino-acid sequence WXXXF/Y (where ‘X' denotes any residue)17. PEX5 returns to the cytosol through a separate channel formed by a membrane-embedded ubiquitin ligase complex20. The receptor's flexible N terminus inserts into this retrotranslocon and gets mono-ubiquitinated on a conserved cysteine; ubiquitinated PEX5 is then pulled out of the organelle by a cytosolic AAA ATPase21. In the cytosol, PEX5 is deubiquitinated22,23 and presumably also refolds before starting another import cycle. PTS2 protein import is by contrast poorly understood. The PTS2 signal is recognized by an adaptor called PEX7 that anchors the cargo to a compatible receptor24. In humans and other animals, the receptor is a specific splice variant of PEX5 called the long isoform (PEX5L); a shorter isoform (PEX5S) lacks the PEX7-binding region25,26. In yeast, PEX7 associates instead with specialized receptors such as PEX18 and PEX21 that resemble PEX5L but lack a TPR domain27. PEX7 consists of WD40 repeats that form a seven-bladed β-propeller24 and has neither a flexible N terminus for insertion into the retrotranslocon nor a conserved cysteine for mono-ubiquitination. PEX7 therefore cannot be recycled like PEX5 and other receptors. To understand how PTS2 proteins are brought into peroxisomes, we reconstituted the process using a cytoplasmic extract from the eggs of the frog Xenopus laevis. As shown previously15,36,37, this cell-free system recapitulates the import of PTS1 cargo such as green fluorescent protein (GFP) fused to an SKL sequence: over time the cargo accumulates inside peroxisomes that then become visible as bright puncta under a microscope (Fig. To adapt this system for PTS2 cargo, we fused GFP to the import signal from different Xenopus PTS2 proteins (Extended Data Fig. 1a; purity of all recombinant proteins in Extended Data Fig. Each PTS2 cargo formed puncta in the extract (Extended Data Fig. 1a), demonstrating a functional PTS2 import pathway. The puncta were dimmer than those formed by PTS1 cargo, suggesting that PTS2 import is less efficient. As expected, PTS2 import was abolished following depletion of PEX5 from the extract (Extended Data Fig. 1a) and could be restored with purified recombinant PEX5L, the long isoform that can bind to PEX7 (schematic of the PEX5 variants used in Fig. 1b) but not with the short isoform PEX5S that cannot bind (Extended Data Fig. Either isoform could restore PTS1 import (Extended Data Fig. The PTS2 puncta result from multiple import cycles because they were not visible with a PEX5 mutant lacking the conserved cysteine (C11A), which cannot be mono-ubiquitinated and recycled (Extended Data Fig. Notably, both PTS2 and PTS1 import was abolished by a PEX5 mutant lacking the WXXXF/Y motifs (∆WF) that enable translocation through the YG phase (Extended Data Fig. a, Xenopus egg extract was incubated with fluorescent cargo, mounted on a slide and imaged using a spinning-disk confocal microscope. Repeated rounds of cargo import into peroxisomes produce bright puncta. b, Schematic of the PEX5 variants used. Numbers refer to residues in Xenopus PEX5L. Black bar, position of the conserved cysteine (C); white bars, locations of WXXXF/Y motifs; yellow box, PEX7-binding region. c, Extract was depleted of endogenous PEX5 using beads conjugated to the PEX5-binding domain from PEX14 and then supplemented with or without purified recombinant yeast PEX7 and the indicated purified recombinant PEX5 variants (described in b). Import was assessed using the fluorescent PTS1 and PTS2 cargos described in a. d, As in c except using the PEX5 variants shown. We next assessed the importance of PEX7. Because frog PEX7 was unstable when produced recombinantly, we instead used yeast PEX7 and cargo consisting of GFP fused to the PTS2 signal from the yeast peroxisomal thiolase POT1 (Fig. Yeast PEX7 together with PEX5L (but not PEX5S) could restore import of the PTS2 cargo in PEX5-depleted extract (Fig. 1c), albeit inefficiently, probably reflecting the relatively weak interaction between PEX5L and yeast PEX7 (Extended Data Fig. The resulting chimaeric receptor, PEX5-21, bound to yeast PEX7 more strongly (Extended Data Fig. 2a) and boosted import of the PTS2 cargo (Fig. Both the receptor and yeast PEX7 were necessary for import (Fig. The import rate of the PTS2 cargo was notably slower than that of PTS1 cargo (Extended Data Fig. A receptor mutant lacking the TPR domain could still bind to PEX7 and PTS2 cargo as expected (Extended Data Fig. 2c) as well as import the cargo into peroxisomes in egg extract (Fig. Importantly, PTS2 import by either PEX5L or PEX5-21 again depended on the WXXXF/Y motifs of the receptor (Fig. 1c,d), providing support for the hypothesis that PTS2 proteins are moved into peroxisomes through the YG conduit. To test whether PTS2 proteins are shuttled through the YG phase, we reconstituted the phase as a hydrogel using a recombinant fragment of PEX13 comprising the YG domain and upstream unstructured region (Extended Data Fig. As shown previously17, entry of fluorescently labelled proteins into the phase can be visualized under a microscope as a bright zone expanding inwards from the gel edge (Fig. Consistent with the egg extract experiments, PTS2 cargo accumulated in the YG hydrogel in the presence of PEX7 and the long receptor isoform PEX5L, whereas no entry was observed with either component alone or with the short isoform (Fig. By contrast, either isoform could drag PTS1 cargo into the gel independently of PEX7 (Fig. PTS2 entry was enhanced by the chimaeric receptor PEX5-21, consistent with its increased affinity for PEX7, whereas the efficiency of PTS1 entry remained unchanged (Fig. PTS2 entry also did not depend on the receptor's TPR domain, whereas PTS1 entry did (Fig. Thus, PTS2 proteins are probably translocated into peroxisomes through the YG phase, and PEX5 and PEX7 are both critical for this process. Interestingly, PTS2 cargo was dragged through the YG hydrogel more slowly than PTS1 cargo (Fig. 2c), in agreement with the slower rate of PTS2 import measured in egg extract. In addition, the entry of PTS2 cargo into the gel was not greatly affected by ablation of the receptor's WXXXF/Y motifs (Fig. 2d), which are otherwise essential for moving PTS1 cargo through the YG phase17. a, YG hydrogels were incubated with the fluorescent cargo shown (described in Fig. 1a) with or without purified recombinant yeast PEX7 and the indicated recombinant PEX5 variants (described in Fig. Cargo accumulation inside the gel was visualized on a point-scanning confocal microscope as a bright zone expanding inwards from the gel edge (left). Representative images after a 30-min incubation (right). The fold enrichment of cargo relative to buffer, across the imaged field, for each of three independent experiments is plotted above the images. The width of each image represents 300 µm. b, As in a but in the presence of the indicated PEX5 variants with or without PEX7. c, As in a using PEX5-21 and PEX7. Representative images after the indicated times (left). Linear fits to the data generated by least-squares regression yield the corresponding permeation rates (mean ± s.e.m.). d, As in a using PTS2 cargo in the presence of PEX7 and the indicated PEX5 variants. Source numerical data are provided. We identified within this region a conserved motif characterized by the amino-acid sequence KPWE (Fig. Beads conjugated to a synthetic peptide containing this motif could recruit yeast PEX7 fused at the C terminus to GFP (PEX7–GFP), whereas a peptide with the motif scrambled could not (Fig. PEX7–GFP was also recruited to beads conjugated to a polypeptide encompassing the entire unstructured region and YG domain but only when the KPWE motif was intact (Extended Data Fig. The horizontal line represents unstructured regions. The conserved KPWE motif is coloured pink in the amino-acid sequence alignment (bottom; UniProt ID: P80667, Q92968, Q9SRR0 and Q54CL3). The numbered residues correspond to yeast PEX13. b, Thiol-reactive beads were conjugated to a cysteine-containing synthetic peptide comprising the KPWE motif of yeast PEX13 or a peptide in which the relevant residues (underlined) were scrambled. The N-terminal cysteine used for conjugation is shaded. Beads were incubated with yeast PEX7 fused to GFP (PEX7–GFP) or GFP alone; recruitment to the beads was visualized after 1 h on a spinning-disk confocal microscope. c, AlphaFold model of the complex between yeast PEX7 and the indicated N-terminal segment of yeast PEX13 containing the KPWE motif. d, As in c with PEX7 represented as a surface and coloured according to the indicated amino-acid conservation scale. e, As in c but overlaid on the crystal structure (PDB ID: 3W15) of yeast PEX7, the PTS2 signal and the yeast PTS2 receptor PEX21. f, Hydrogels formed from a recombinant fragment of PEX13 comprising the wild-type N-terminal unstructured region and YG domain (WT) or an analogous fragment lacking the KPWE motif (∆KPWE). g, Hydrogels as in f were incubated with the indicated fluorescent proteins and components. The accumulation of each fluorescent protein inside the gel was visualized on a point-scanning confocal microscope after 1 h. The width of each image corresponds to 300 µm. 4 for the associated pLDDT confidence scores of all AlphaFold models). The interface is highly conserved (Fig. The two interfaces are indeed independent, as beads conjugated to the KPWE peptide could recruit PTS2–GFP only when PEX7 and the receptor were both present (Extended Data Fig. No binding was seen in the absence of PEX7 or with beads harbouring the scrambled peptide. These results imply that PEX7 might enter the YG phase either by binding directly to the KPWE motif or by piggybacking on an import receptor in the presence of cargo. Direct entry of PEX7 into the YG phase could be demonstrated with the hydrogel system. Using hydrogels prepared from a recombinant fragment of PEX13 consisting of the wild-type unstructured region and YG domain (Fig. 3f), we observed that PEX7–GFP readily entered the phase, whereas GFP alone did not (Fig. Ablation of the KPWE motif did not affect gelation (Fig. 3f) but it precluded the entry of PEX7–GFP (Fig. By contrast, PEX5 fused at the C terminus to GFP (PEX5–GFP) accumulated equally well in either wild-type or mutant hydrogels (Fig. Thus, PEX7 interacts with the YG phase through the KPWE motif instead of through the YG repeats to which import receptors bind. We next examined how PEX7 interacts with the YG phase when bound to a receptor and cargo. We prepared a complex between PEX7–GFP and the chimaeric receptor PEX5-21 in the presence of a synthetic peptide comprising the PTS2 signal. 3g), revealing that the receptor is sufficient to drag PEX7 and cargo into the YG phase. This result was recapitulated using a complex consisting of PEX7 bound to the receptor and PTS2–GFP cargo, which readily entered the gel regardless of the KPWE motif (Fig. Together, the hydrogel experiments demonstrate that PEX7 and import receptors can partition into the YG phase independently of each other. When both components assemble with cargo into a complex, each of these interactions may be utilized to move the cargo through the phase into peroxisomes. Interestingly, PEX5–GFP permeated further into the wild-type hydrogel than either PEX7–GFP alone or PEX7–GFP in the cargo complex (Fig. Thus, PEX7 seems to traverse the YG phase more slowly, consistent with PTS2 cargo being moved through the phase slower than PTS1 cargo (Fig. Efficient translocation into peroxisomes might therefore primarily be driven by the receptor. To test this hypothesis, we ablated the KPWE motif in PEX13 in yeast and investigated whether PEX7 would accumulate in peroxisomes. To enable detection of PEX7 by immunoblotting, we introduced a single FLAG tag at the C terminus of the protein (Extended Data Fig. The tagged form was expressed from the endogenous locus and was fully functional, as assessed in cells expressing fluorescently labelled PTS1 and PTS2 matrix proteins (Extended Data Fig. The corresponding strains were cultured in oleic acid to induce peroxisomes and the cells subsequently fractionated into a peroxisome-containing membrane pellet (Fig. Given that PEX7 completely enters the peroxisomal matrix during import28,29,30,31,32,33,34,35, this observation suggests that PEX7 cannot return to the cytosol when its interaction with the YG phase is disrupted. Consistent with this assumption, PEX7–FLAG accumulation was abolished after deletion of the two yeast PTS2 receptors PEX18 and PEX21 (Fig. By contrast, deletion of the two PTS1 receptors PEX5 and PEX9 had only a small effect (Fig. All receptor-pair knockouts correctly precluded import of their respective cargo (Extended Data Fig. a, PEX7 fused at the C terminus to a FLAG epitope tag (PEX7–FLAG) was expressed in yeast strains containing different PEX13 mutants. The cells were cultured in oleic acid to induce peroxisomes, lysed and then separated into a peroxisome-containing membrane fraction. PEX7–FLAG was immunoprecipitated (IP) from the lysate and the detergent-solubilized membranes, resolved by SDS–PAGE and detected by immunoblotting for the FLAG epitope. b, PEX7–FLAG was analysed as in a for cells lacking PEX13 (−), or cells containing either wild-type PEX13 (WT) or a PEX13 mutant lacking residues 2–13 (∆2–13) that encompass the KPWE motif, as shown (top). Immunoblots of total lysates and corresponding membrane fractions (bottom). c, As in b but including strains additionally lacking the PTS2 receptors PEX18 and PEX21 (∆PTS2) or the PTS1 receptors PEX5 and PEX9 (∆PTS1). d, PEX7 expression was decreased by inserting a tetracycline-responsive riboswitch upstream of the coding sequence (top). Total lysates from cells expressing PEX7–FLAG with or without the riboswitch were analysed by immunoblotting as in a (bottom). e, Peroxisomes were induced in cells expressing PEX7 (with or without the riboswitch as indicated) as well as either wild-type PEX13 or PEX13 lacking the KPWE motif (∆2–13). Protein import was assessed by fluorescence microscopy using endogenously expressed FOX2 fused to RFP (PTS1) and POT1 fused to GFP (PTS2). Bright puncta indicate successful import, whereas the absence of import manifests as a diffuse signal throughout the cytosol. Experiments in Xenopus egg extract confirmed that PEX7–GFP enters peroxisomes only when it is assembled into a complex with PEX5 and cargo (Extended Data Fig. Interestingly, PEX7–GFP puncta were considerably brighter than those formed by PEX5–GFP (Extended Data Fig. 7e), consistent with the hypothesis that the two proteins are not recycled together. This conclusion agrees with a previous study35 showing that PEX7 is recycled more slowly than PEX5. Together, our results suggest that PEX7 enters peroxisomes along with cargo by piggybacking on an import receptor; PEX7 then returns to the cytosol by binding to the KPWE motif of PEX13 and thereby partitioning back into the YG phase from inside the matrix. The proposed model implies that KPWE-motif deletion should inhibit PTS2 import because PEX7 should no longer be recycled. Surprisingly, however, cells lacking the KPWE motif showed no obvious PTS2 import defect (Extended Data Fig. We reasoned that in our assay, PEX7 might be sufficiently abundant to mask a recycling phenotype. We therefore inserted a tetracycline-responsive riboswitch39 upstream of the PEX7 open reading frame to reduce translation of the corresponding PEX7 messenger RNA. This strategy decreased the abundance of PEX7 by about an order of magnitude (Fig. Under these conditions, PTS2 import was still normal in cells that retained the KPWE motif, whereas cells that lacked the KPWE motif now had a pronounced import defect specific to PTS2 proteins (Fig. These results confirm the proposed role of the KPWE motif in PEX7 recycling and further demonstrate that PEX7 recycling is essential for protein import when PEX7 becomes limiting in cells. Given the favourable interaction between PEX7 and the YG phase, we next investigated how PEX7 would be extracted from the phase and returned to the cytosol to start another import cycle. A possible mechanism was suggested by large-scale protein-interaction studies in yeast40,41 and mammalian cells42,43,44, which had identified an uncharacterized binding partner of PEX7 in the cytosol that has recently been named PEX39 (ref. The protein is conserved throughout eukaryotes and contains an amino-acid sequence (RPWE in yeast) that resembles the KPWE motif of PEX13 (Fig. PEX39 also has a conserved amphipathic helix (APH) near its N terminus (Fig. AlphaFold predicts that PEX39 can bind to PEX7 using both of these features: the RPWE motif binds to the same site as the analogous motif of PEX13 and the APH binds to the opposite side, where the PTS2 signal and import receptors are recruited (Fig. Both interaction surfaces on PEX7 are conserved in addition to a groove that cradles the linker between the RPWE motif and the APH (Extended Data Fig. a, Location of the APH and KPWE-like motif (RPWE) in yeast PEX39 with corresponding amino-acid sequences from the indicated species (UniProt ID: P47087, Q5I0X4 and A0A5S9YGZ4). Numbered residues correspond to the yeast homologue. Conserved hydrophobic residues in the APH are marked with back dots in the helical-wheel diagram. b, AlphaFold model of the complex between yeast PEX7 and the region of yeast PEX39 that includes the APH and RPWE motif (left). c, Glutathione-conjugated beads were bound to glutathione-S-transferase (GST)-tagged wild-type yeast PEX39 (WT), or mutants whose RPWE motif (∆RPWE) or conserved hydrophobic residues in the APH (∆APH) were converted to alanine. Recruitment of yeast PEX7–GFP to the beads after 1 h was visualized on a spinning-disk confocal microscope. d, Glutathione-conjugated beads were incubated with 1 µM yeast PEX7 and varying concentrations of the indicated GST-tagged forms of yeast PEX39. The beads and any bound protein were sedimented after 1 h. Each supernatant fraction was resolved by SDS–PAGE and stained with Coomassie blue (representative images of the gels in Extended Data Fig. PEX7 remaining in the supernatant was quantified and used to calculate the bound fraction, which was plotted as a function of the PEX39 concentration. Lines represent fits, generated by least-squares regression, to the data from three independent experiments; the corresponding observed binding constants (Kobs) are indicated (right; mean ± s.e.m.). Source numerical data are provided. The predicted interactions were validated by showing that beads coated with PEX7 could recruit PEX39 fused at the C terminus to GFP (PEX39–GFP; Extended Data Fig. Conversely, beads coated with PEX39 could recruit PEX7–GFP and the recruitment required the RPWE motif (Fig. These results were quantitatively confirmed by incubating PEX7 with varying concentrations of PEX39 immobilized on beads (Extended Data Fig. Thus, PEX39 can directly bind to PEX7, with the RPWE motif driving the interaction. Because PEX39 and PEX13 can each bind to PEX7 using similar motifs, we investigated whether PEX39 could extract PEX7 from the YG phase. We tested this hypothesis by preloading YG hydrogels with PEX7–GFP, then spiking the buffer with increasing concentrations of PEX39 and following the PEX7–GFP fluorescence in the gel over time (Fig. Addition of PEX39 rapidly depleted PEX7–GFP from the hydrogel in a dose-dependent manner, whereas PEX7–GFP continued to accumulate in the gel in the absence of PEX39 (Fig. The extraction of PEX7–GFP was abolished by mutating the RPWE motif in PEX39 (Fig. 6c), whereas mutating the APH had a much smaller effect, consistent with our binding experiments. Ablation of both motifs had no additional effect. PEX39 was notably unable to extract PEX5–GFP from the hydrogel (Fig. These results thus support the hypothesis that PEX39 can extract PEX7 from the YG phase during recycling, when PEX7 would no longer be bound to cargo and an import receptor. a, YG hydrogels were prepared in a glass-bottomed dish and pre-equilibrated with PEX7–GFP. PEX39 was then added to the buffer and a point-scanning confocal microscope was used to follow the PEX7–GFP fluorescence over time. b, PEX7–GFP signal in the gel over time at the indicated molar ratios of wild-type PEX39 (relative to PEX7). The mean (bold line) and range (shading) of three experiments are shown. c, As in b using a fourfold excess of wild-type (WT) PEX39 or mutants harbouring alanines in place of the APH (∆APH), the RPWE motif (∆RPWE) or both (∆APH + ∆RPWE). d, As in b using PEX7–GFP or PEX5–GFP and a fourfold excess of wild-type PEX39. e, As in b using a fourfold excess of PTS2 peptide and the PEX7-binding chimaeric receptor PEX5-21 (left) or PTS2 peptide and the non-PEX7-binding receptor isoform PEX5S (right). Wild-type PEX39 was titrated as indicated. f, YG hydrogel droplets were incubated with PEX7–GFP in the presence of PEX39. The effect of PEX39 on the influx of PEX7–GFP into the gel was then assessed using a point-scanning confocal microscope. g, PEX7–GFP influx in the presence of the indicated molar ratios of wild-type PEX39 (relative to PEX7) with or without a fourfold excess of PTS2 peptide and either PEX5S or PEX5-21. The mean (bars) and individual values (circles) of three experiments are shown. h, As in g using PEX7–GFP or PEX5–GFP and the indicated molar ratios of wild-type PEX39. i, As in g using wild-type PEX39 or the indicated mutants. Source numerical data are provided. The proposed role of PEX39 in PEX7 recycling implies that PEX39 should not extract PEX7 from the YG phase during translocation into peroxisomes when PEX7 would be loaded with cargo and an import receptor. PEX39 was indeed unable to efficiently extract PEX7–GFP from YG hydrogels in the presence of a PTS2 peptide and the chimaeric receptor PEX5-21, even when PEX39 and the receptor were both present in excess molar ratios relative to PEX7–GFP (Fig. By contrast, PEX7–GFP was readily extracted by PEX39 in the presence of the short receptor isoform PEX5S that cannot bind to PEX7 (Fig. Thus, PEX39 does not interfere with the translocation of the fully assembled PEX7–receptor–cargo complex into peroxisomes. As expected, PEX39 blocked the influx of PEX7–GFP alone as well as PEX7–GFP in the presence of a PTS2 peptide and PEX5S (Fig. By contrast, PEX39 was much less inhibitory when PEX7–GFP was assembled into a complex with the PTS2 peptide and the chimaeric receptor PEX5-21 (Fig. Some inhibition was observed at high PEX39 concentrations, suggesting that PEX39 cannot be too abundant in vivo; PEX39 is indeed a few-fold less abundant than PEX7 in yeast (Extended Data Fig. Consistent with the hydrogel release experiments, PEX39 did not prevent the influx of PEX5–GFP (Fig. 6h), and the observed inhibition of PEX7–GFP influx depended on the RPWE motif and much less on the APH (Fig. PEX39–GFP did not by itself enter the YG hydrogel (Extended Data Fig. 9d) and was not imported into peroxisomes in Xenopus egg extract (Extended Data Fig. Together, our results lead to a model in which import receptors move cargo-loaded PEX7 through the YG conduit into peroxisomes. To start another round of translocation, PEX7 has to be reloaded with cargo and a new receptor molecule. 7a), suggesting that PEX39 might promote cargo binding. PTS2–GFP cargo was indeed recruited to PEX7-coated beads in the presence of PEX39 (Fig. 7b) and conversely, PTS2–GFP was recruited to PEX39-coated beads in the presence of PEX7 (Fig. The cargo did not bind to either component alone. APH mutation abolished cargo binding to the PEX39–PEX7 complex (Fig. Notably, cargo binding was also reduced following mutation of the RPWE motif, as expected from the role of the motif in driving the interaction with PEX7. Thus, PEX39 not only extracts PEX7 from the YG phase but also enables PEX7 to rebind to cargo (Fig. a, AlphaFold models of the complex between yeast PEX7 and the region of yeast PEX39 that includes the APH and RPWE motif, with and without a PTS2 signal and the yeast PTS2 receptor PEX21. Note that PEX7 alone binds to both the APH and RPWE motif (top). The RPWE motif remains bound in all cases. b, Glutathione-conjugated beads were incubated overnight with 5 µM GST-tagged yeast PEX7 (GST–PEX7) and 1 µM GFP fused at the N terminus to the PTS2 signal from yeast POT1 (PTS2–GFP). Where indicated, reactions included 5 µM wild-type (WT) yeast PEX39 or a mutant whose conserved hydrophobic residues in the APH were converted to alanine (∆APH). Recruitment of PTS2–GFP to the beads was visualized on a spinning-disk confocal microscope. c, As in b using GST-tagged wild-type yeast PEX39 (GST–PEX39) or the indicated GST-tagged mutants and yeast PEX7 at 5 µM. d, As in c with the additional presence of the chimaeric receptor PEX5-21 (5 µM). Contrast was adjusted as in b. e, Proposed steps for reloading of PEX7 with receptor and cargo. Dissociation of PEX39 would have to occur in two steps. First, an import receptor would displace the APH of PEX39, as predicted by AlphaFold (Fig. The prediction is supported by experiments in which a complex between PEX7, the chimaeric receptor PEX5-21 and PTS2–GFP cargo could be recruited to beads coated with wild-type PEX39 (Fig. Recruitment was abolished by mutation of the PEX39 RPWE motif and completely unaffected by mutation of the APH (Fig. Thus, after receptor binding, PEX39 only interacts with PEX7 through its RPWE motif (Fig. Finally, complete dissociation of PEX39 would occur following re-entry of the PEX7–receptor–cargo complex into the YG conduit, where the KPWE motif of PEX13 would displace the RPWE motif of PEX39. The PEX7–receptor–cargo complex was indeed no longer efficiently recruited to PEX39-coated beads in the presence of the KPWE peptide, whereas the scrambled peptide had no effect (Fig. Furthermore, the KPWE peptide could strip off PEX7 from PEX39 lacking the APH, whereas the scrambled peptide could not (Fig. Together, our results reveal that PEX39 not only extracts PEX7 from the YG conduit but also allows PEX7 to rebind to cargo. The subsequent recruitment of an import receptor partially displaces PEX39 and allows the cargo complex to partition back into the YG conduit. a, Glutathione-conjugated beads were incubated overnight with 5 µM GST-tagged wild-type yeast PEX39 or the indicated mutants in the presence of 5 µM PEX5-21 and yeast PEX7 as well as 1 µM PTS2–GFP. Where indicated, reactions included 100 µM of a peptide comprising the KPWE motif of PEX13 or a scrambled version. Recruitment of PTS2–GFP to the beads was visualized on a spinning-disk confocal microscope. b, Glutathione-conjugated beads were first incubated with yeast PEX7 and the ∆APH mutant of GST-tagged PEX39 (each at 1 µM), and then with varying concentrations of the KPWE or scrambled peptide. The beads were sedimented and the supernatant fraction was resolved by SDS–PAGE, followed by Coomassie blue staining. Representative gels are shown (top), with relative sizes marked in kDa on the left. Fraction of PEX7 remaining on the beads in three independent experiments plotted against the peptide concentration and fitted to a competitive-binding isotherm (bottom). observed binding constant (Kobs) for each peptide is shown; ND, binding not detectable. c, Model of PTS2 protein import into peroxisomes. Unprocessed gels and source numerical data are provided. Using a combination of in vitro reconstituted systems and experiments in vivo, we have determined the mechanism by which PTS2 matrix proteins are imported into peroxisomes (Fig. Translocation is initiated by the assembly of PTS2 cargo into a complex with PEX7 and an import receptor in the cytosol (step 1). The receptor can either be the long isoform of PEX5 in higher organisms or one of its paralogues such as PEX18 or PEX21 in yeast. The cargo complex then partitions into a conduit that consists of a selective phase formed from the YG domain and the preceding unstructured region of PEX13 (step 2). PEX7 is extracted out of the phase on the cytosolic side by PEX39 (step 7), which can then facilitate the reloading of PEX7 with cargo and a new receptor molecule to start another import cycle (step 8). During translocation into peroxisomes, both PEX7 and import receptors can interact with the YG phase independently of each other and probably cooperate to move cargo through the import conduit. However, receptors move through YG hydrogels faster than PEX7, suggesting that translocation is predominantly driven by the receptors. This observation probably explains why PTS2 import in Xenopus egg extract is slower than import of PTS1 cargo, which has also been reported in plants46. Because diffusion through the YG phase is non-directional, translocation must be biased towards the peroxisomal matrix. Directionality is exclusively imposed by the receptors, which can bind with high affinity to the luminal domain of PEX14 (a membrane protein closely associated with PEX13) using their WXXXF/Y motifs47. This model also agrees with the observation that the recombinant luminal domain of PEX14 blocks PTS2 import in rat-liver extract35. Translocation could additionally be driven inwards if import were coupled to receptor export48. Although this mechanism might not be needed for import in every case35,50, it might expedite the translocation of bulkier or more YG-phobic cargo whose passage through the YG phase might be unfavourable. Regardless of the mechanism, the biased movement of the receptors would result in import of both PTS1 and PTS2 cargo, and would also explain how PEX7 enters the matrix. Our work reveals that import receptors and PEX7 return to the cytosol by distinct mechanisms. Import receptors are actively pulled out through a retrotranslocon (the ubiquitin ligase complex) by a process powered by nucleotide hydrolysis21. By contrast, PEX7 recycling is driven solely by protein interactions with the YG phase and PEX39. The existence of separate recycling pathways is supported by experiments in rat-liver extract, which have shown that PEX7 returns to the cytosol slower than import receptors35; this difference is likely to reflect the slow permeation of PEX7 through the YG phase revealed by our hydrogel system. The role of PEX39 as a chaperone for PEX7 is supported by a recent preprint45. Although this study did not investigate PEX39 in the context of PEX7 recycling, it likewise reports that PEX39 functions in PTS2 import, stabilizes the interaction between PEX7 and cargo, and binds to PEX7 more strongly than PEX13. These observations are in accord with our model in which PEX39 extracts PEX7 from the YG phase and then enables PEX7 to rebind to cargo and re-enter the phase during the next import cycle. The authors additionally report that PTS2 import is attenuated by both downregulation and overexpression of PEX39, which is again consistent with our model: some PEX39 is required to recycle PEX7 but excess PEX39 would prevent PEX7 from binding to cargo and receptors, and would also inhibit the entry of the cargo complex into the YG phase. PEX39 does not need to be very abundant because PEX7 recycling is required only when PEX7 becomes limiting; a basal recycling rate coupled with new protein synthesis is probably sufficient to maintain a stable cytosolic pool of PEX7 under physiological conditions. Recycling might be critical under disease conditions where PEX7 expression is attenuated by hypomorphic alleles or destabilizing mutations. Boosting this recycling pathway might be an attractive therapeutic strategy to restore PTS2 import in such contexts. In fact, PEX7 is uncannily similar to the mRNA-export factor GLE2 (also known as RAE1 in mammals). Both proteins consist of WD40 repeats24,52 and bind to short linear motifs in phase-forming polypeptides: PEX7 to the KPWE sequence in the unstructured region of PEX13 and GLE2 to the GLE2-binding sequence (GLEBS) in the unstructured FG nucleoporin NUP116 (also known as NUP98 in mammals)53. Akin to nuclear transport receptors and their intrinsic affinity for the nucleoporin FG phase19, PEX7 also seems to have weak affinity for the peroxisomal YG phase because some entry of PEX7 into YG hydrogels was observed in the absence of the KPWE motif. The similarity between PEX7 and nuclear transport receptors therefore reinforces the analogy between peroxisomal matrix-protein import and nuclear-protein transport. The analogy implies that the two protein-transport systems probably evolved from the same ancestral machinery, which might have utilized a selective phase to move folded proteins between compartments. The most primitive systems might have allowed bidirectional transport of receptor–cargo complexes through a conduit containing the phase without directional bias. In the case of peroxisomal-protein import, this system might have consisted of PEX7, a PTS2 signal and a PEX13-like pore. Some directionality might have been provided by the removal of the PTS2 signal by a matrix protease, which still occurs in some organisms7. PEX5 and the PTS1 signal would have evolved later to provide directional bias and enhance the efficiency of translocation. Supporting this theory, only a few PTS2 proteins occur in most contemporary organisms yet include some of the most conserved and arguably ancient peroxisomal enzymes such as thiolase; furthermore, in certain species, PTS2 import has been completely supplanted by the PTS1 system7. All reagents were obtained from Millipore Sigma unless specified otherwise. Buffers were prepared in ultrapure water. Plasmids were derived from vector pET-28b(+) (Novagen) and are listed in Supplementary Table 1 along with details about their construction and relevant GenBank accession numbers. All constructs were codon-optimized for E. coli and verified by whole-plasmid sequencing. For constructs fused to GST, the GST coding sequence was first amplified by PCR from vector pGEX-6P-3 (Cytiva), incorporating at the 5′ end the nucleotide sequence CAT (which creates an NdeI restriction site in combination with the downstream start codon) and at the 3′ end the nucleotide sequence 5′-GGATCCGACTTGGAAGTACTGTTTCAGGGTCCCTAACTCGAG-3′ that encodes a short linker consisting of the amino acids GSD, a 3C protease-cleavage site (amino acids LEVLFQGP), a TAA stop codon and a XhoI restriction site. The complete backbone of vector pET-28b(+) was next amplified by PCR, starting with nucleotides 5′-GCTAACAAA-3′ downstream of the multiple-cloning site and proceeding through nucleotides 5′-GGAGATATA-3′ upstream of the multiple-cloning site. Both PCR amplicons were ligated by Gibson assembly (New England Biolabs). The coding sequence of individual constructs was then inserted immediately downstream of the 3C protease-cleavage site. Truncations and deletions were performed by site-directed mutagenesis (Agilent). Plasmids were assembled in the integrating vector pPICZα-LINK20 and are listed in Supplementary Table 2 along with details about their construction and relevant GenBank accession numbers. The vector was first modified to include the coding sequence of GST, followed by a tobacco etch virus (TEV) protease-cleavage site exactly as described previously24. The coding sequence of individual constructs was then inserted immediately downstream of the TEV protease-cleavage site by Gibson assembly. Plasmids are listed in Supplementary Table 3 along with details about their construction and relevant GenBank accession numbers. To assemble plasmid MSV-262, the following components were amplified by PCR and ligated by Gibson assembly: a segment of plasmid pWCD1401 located between and including the two NotI restriction sites (containing a high-copy ColE1 origin of replication and bacterial kanamycin-resistance cassette)54; the yeast PEX7 coding sequence along with 80 bp of the 5′ untranslated region; the yeast ADH1 terminator (that is, the last 13 bp of the ADH1 reading frame along with 190 bp of the 3′ untranslated region) flanked at the 5′ end by the nucleotide sequence 5′-GGCGCGC-3′ (to introduce an AscI restriction site) and at the 3′ end by the nucleotide sequence 5′-GATCCGCTAGATCTCGAGC-3′ (to introduce BglII and XhoI restriction sites); a hygMX6 hygromycin-resistance cassette (Addgene, catalogue number 19342), followed by an MfeI restriction site; and 80 bp of the 3′ untranslated region immediately downstream of the PEX7 reading frame. Using site-directed mutagenesis, the nucleotide pair GC was next inserted immediately upstream of the PEX7 start codon to introduce an SphI restriction site, nucleotides 462 and 468 within the PEX7 coding sequence were mutated to T and C, respectively, to destroy two BamHI restriction sites and the last four nucleotides of the PEX7 coding sequence (TTGA) were converted to CTAGC to introduce an NheI restriction site. To assemble plasmid MSV-269, the TAG stop codon in plasmid MSV-262 was replaced by the nucleotide sequence 5′-TCTGGTGATTACAAAGATGATGATGACAAGTAA-3′, which encodes a short linker comprising the amino acids SG, followed by a FLAG tag (amino acids DYKDDDDK) and a new stop codon. To assemble plasmid MSV-322, the nucleotides GC immediately upstream of the PEX7 start codon were replaced by a tetracycline-responsive riboswitch (5′-GGCCTAAAACATACCAGATCGCCACCCGCGCTTTAATCTGGAGAGGTGAAGAATACGACCACCTAGGCC-3′)39 preceded by a short linker (5′-ATCGTACG-3′) and followed by a minimal Kozak sequence (AAA). Fragments containing the ColE1 origin of replication and kanamycin-resistance cassette, as well as the ADH1 terminator region, were amplified by PCR as described earlier. These were ligated by Gibson assembly to the yeast PEX13 coding sequence and 69 bp of the 5′ untranslated region, a natMX6 nourseothricin-resistance cassette (Addgene, catalogue number 19343) and 68 bp of the 3′ untranslated region downstream of the PEX13 reading frame. The nucleotide sequence 5′-AGCTCCGAGGATCTGTACTTTCAGAGCTATCCATATGATGTTCCAGATTACGCT-3′, which encodes a linker composed of the amino acids SS, followed by a TEV protease-cleavage site (amino acids EDLYFQS) and an haemagglutinin tag (amino acids YPYDVPDYA), was then inserted immediately upstream of the PEX13 stop codon. Modifications of the PEX13 coding sequence were performed by site-directed mutagenesis. Plasmid MSV-324 was assembled like plasmid MSV-282, except using the coding sequence of yeast PEX39, 80 bp of the 5′ and 3′ untranslated regions, and the yeast LYS2 auxotrophic marker (Addgene, catalogue number 64188)55. The nucleotide sequence 5′-GGTTCTGGTGATTACAAAGATGATGATGATAAG-3′, which encodes a linker composed of the amino acids GSG, followed by a FLAG tag, was then inserted immediately upstream of the PEX39 stop codon. Yeast strains were routinely cultivated on YPD medium (1% wt/vol yeast extract, 2% wt/vol peptone and 2% wt/vol dextrose) at 30 °C, unless stated otherwise; for plates, 2% wt/vol agar (Becton Dickinson, catalogue number 214010) was added to the medium. P. pastoris strains were derived from the SMD1168 parental strain56 and are listed in Supplementary Table 4. Expression cassettes for PEX7 and PEX7–GFP were inserted at the AOX1 locus by homologous recombination using the integrating plasmids described earlier. Plasmids were linearized using the restriction enzyme PmeI (New England Biolabs) and delivered into the yeast cells by electroporation. Transformants were selected on YPD agar supplemented with 1 M sorbitol and 500 µg ml−1 zeocin (Thermo Fisher, R25001). Correct insertion was validated by PCR. S. cerevisiae strains were derived from the BY4742 reference strain57 and are listed in Supplementary Table 5. Deletions and insertions were performed by homologous recombination in a BY4742 derivative (referred herein as strain MSB-462) that expresses the peroxisomal PTS1 enzyme FOX2 fused at the N terminus to red fluorescent protein (RFP–FOX2) as well as the peroxisomal PTS2 enzyme POT1 fused at the C terminus to green fluorescent protein (POT1–GFP)58. For gene deletions, the appropriate antibiotic-resistance cassette or auxotrophic marker was amplified by PCR from the plasmids listed in Supplementary Table 3 using primers that introduce 60-bp overhangs corresponding to the 5′ and 3′ untranslated regions immediately upstream and downstream of the target open reading frame, respectively. The resulting amplicon was delivered into the yeast cells by lithium acetate-based transformation59. Transformants were selected on YPD agar supplemented with 250 µg ml−1 hygromycin (Thermo Fisher, catalogue number 10687010), 500 µg ml−1 geneticin (Thermo Fisher, catalogue number 10131035) or 100 µg ml−1 nourseothricin (Jena Bioscience, AB-101), as appropriate, or on synthetic defined agar medium lacking the appropriate amino acid or nucleotide (Sunrise Science). For inserting mutant genes or epitope tags, the relevant reading frame was first knocked out and then replaced by the modified coding sequence using the integrating plasmids described earlier. The plasmids were linearized using the restriction enzyme NotI (New England Biolabs) before lithium acetate-based transformation. All deletions and insertions were validated by PCR. PEX7 and PEX7–GFP were produced in P. pastoris as fusions to an N-terminal GST tag and a TEV protease-cleavage site, and purified by glutathione-affinity and size-exclusion chromatographies as described previously24 with the following modifications. Cells were cultured in 2-l baffled flasks for 36 h at 30 °C on an orbital shaker in BM medium (100 mM KH2PO4·KOH pH 6.0 at RT, 0.34% wt/vol yeast nitrogen base, 1% wt/vol ammonium sulfate and 4 × 10−5% wt/vol d-biotin) supplemented with 1% vol/vol glycerol. The cells were next sterilely resuspended in BM medium supplemented with 0.75% vol/vol methanol in clean 2-l baffled flasks and cultured for an additional 24 h at 28 °C. The final cell pellets were weighed and suspended (at a ratio of 1 ml g−1 wet cell weight) in lysis buffer (30 mM Tris–HCl pH 7.5 at RT, 500 mM NaCl, 5% wt/vol glycerol, 1 mM EDTA and 2 mM dithiothreitol (DTT)) supplemented with protease-inhibitor tablets (Roche, catalogue number 5056489001) according to the manufacturer's instructions. The cell suspension was divided among 50-ml beadbeater canisters (BioSpec, catalogue number 110803-50SS), supplemented with 2 mM phenylmethylsulfonyl fluoride, and the cells were lysed by beadbeating at 4 °C with 0.5-mm glass beads (BioSpec, catalogue number 11079105). The lysate was supplemented with 0.05% wt/vol Tween 20 and clarified by centrifugation at 40,000g and 4 °C for 40 min. The resulting supernatant was incubated overnight at 4 °C with glutathione agarose (Prometheus). To remove the GST tag, the eluate was mixed with homemade His-tagged TEV protease60 and dialysed overnight at 4 °C against elution buffer lacking glutathione. The dialysed solution was incubated with glutathione agarose and Ni-NTA agarose (Thermo Fisher) to adsorb the GST tag and TEV protease. The resulting flow-through was gel-filtered into 30 mM HEPES·NaOH (pH 8), 300 mM NaCl, 5% wt/vol glycerol and 1 mM DTT (at 4 °C) either on a Cytiva HiLoad 16/600 Superdex 75 prep-grade (for PEX7) or a Cytiva HiLoad 16/600 Superdex 200 PG (for PEX7–GFP) column. Peak fractions were pooled and concentrated before being snap-frozen and stored as single-use aliquots at −80 °C. The unstructured N-terminal region (including the YG domain) of A. thaliana PEX13 was produced in E. coli BL21 Rosetta 2(DE3) cells (Novagen) as a fusion to an N-terminal 14×His tag and a TEV protease-cleavage site, and purified under denaturing conditions in 8 M urea as previously described17. Proteins were produced in E. coli BL21 Rosetta 2(DE3) cells as fusions containing an N-terminal GST tag and a 3C protease-cleavage site. The proteins were purified by glutathione-affinity and size-exclusion chromatographies as described previously15 with the exception that after proteolytic removal of the GST tag using 3C protease, the proteins were gel-filtered into 25 mM HEPES·NaOH pH 7.8 at RT, 150 mM NaCl, 5% wt/vol glycerol and 1 mM DTT before being snap-frozen as single-use aliquots. Lyophilized peptides (Genscript) were dissolved in 1 M HEPES·NaOH pH 7.5 at RT, 10 mM tris(2-carboxyethyl)phosphine (TCEP) and 1 mM EDTA to a final concentration of 10 mM, snap-frozen as single-use aliquots and stored at −80 °C. The KPWE peptide corresponds to the amino-acid sequence CSSTAVPRPKPWETSASLEE consisting of residues 2–20 of S. cerevisiae PEX13 and an additional N-terminal cysteine for coupling to thiol-reactive agarose beads. In the scrambled peptide, amino acids 8 and 10–13 were rearranged into the sequence CSSTAVPEPRWKPTSASLEE. The PTS2 peptide corresponds to the amino-acid sequence SQRLQSIKDHLVESRS consisting of residues 2–15 of the S. cerevisiae peroxisomal thiolase POT1, followed by the amino acids R and S to improve solubility. Peroxisome biogenesis was induced in S. cerevisiae using an established procedure61. Briefly, cells were subcultured twice in YPD medium to mid-log phase (optical density at 600 nm (OD600) ≈ 2), harvested through centrifugation at 2,000g and 30 °C for 10 min, sterilely resuspended to an OD600 of 0.5 in YPGO medium (1% wt/vol yeast extract, 2% wt/vol peptone, 2% vol/vol glycerol, 0.1% wt/vol oleic acid and 0.25% vol/vol Tween 40) and cultured for an additional 16 h at 30 °C. For experiments involving PEX7 riboswitch constructs, cultures in YPGO were supplemented with 250 µM tetracycline (Millipore Sigma, T7660) as described previously62. S. cerevisiae cells expressing RFP–FOX2 and POT1–GFP (MSB-462) were induced to form peroxisomes as described earlier. A 1-ml aliquot of each culture was sedimented by centrifugation for 30 s at 13,000g and RT, and the cells were resuspended in 7 µl of RT complete synthetic defined medium (Sunrise Science, catalogue number 1001) supplemented with 10 mM potassium phosphate pH 6 at RT, 4 mM magnesium sulfate, 25 mM ammonium sulfate and 2% wt/vol d-glucose. An aliquot of the resulting suspension was spotted on a clean glass slide (Thermo Fisher, 22-310397), covered by a 22 × 22 mm no. 1.5 square glass coverslip (VWR, 48366-227) and imaged immediately. The cells were imaged on a spinning-disk confocal platform consisting of a Nikon Eclipse Ti inverted microscope, a Yokagawa CSU-X1 spinning-disk confocal scanner, a Hamamatsu ORCA-Fusion BT sCMOS camera and a Prior Proscan II motorized stage. Samples were illuminated sequentially by solid-state 65 mW 561-nm and 80 mW 488-nm lasers through a Di01-T405/488/568/647 dichroic (Semrock). Fluorescence was collected using a red-emission ET605/52 m (605 ± 26 nm) or a green-emission ET525/50 m (525 ± 25 nm) single-band bandpass filter (Chroma), respectively, through either a Nikon ×100 1.45 numerical aperture (NA) Plan Apo Lambda or Nikon ×40 0.95 NA Plan Apo Lambda air objective. Images were acquired using the Nikon NIS-Elements software with a 16-bit digitizer without binning. Multiple non-overlapping positions arrayed around the centre of the coverslip were imaged for each sample, with focus being maintained at a preset distance from the coverslip using the Nikon PFS. Imaging parameters were configured to maximize the signal-to-noise (SNR) ratio while avoiding saturation. All samples intended to be compared were imaged using identical acquisition settings. Interphase-locked egg extract lacking filamentous actin was prepared exactly as described in a previously published protocol37 using adult female X. laevis wild-type pigmented African clawed frogs (aged 20–30 yr; Xenopus1). The frogs were handled strictly within the confines of the Amphibian/Aquatics facility of the Cell Biology Department at Harvard Medical School according to protocols fully approved by the Harvard Institutional Animal Care and Use Committee (protocol number IS00000098-9). Import reactions were performed exactly as described in a previously published protocol37. Briefly, extract was mixed with 500 nM of the indicated cargo and 100 nM of the indicated receptor variants with or without 100 nM PEX7, and incubated at 24 °C. Where stated, endogenous PEX5 was first depleted from the extract using agarose beads covalently conjugated to the high-affinity PEX5-binding domain from PEX14. For imaging, an aliquot of each reaction was sandwiched between two 22 × 22 mm no. 1.5 square glass coverslips (VWR, 48366-227) that had been passivated with 20 kDa polyethylene glycol (JenKem, M-SLN-20K). The sandwich was mounted on a 25 × 75 mm aluminium slide and sealed with a mixture of Vaseline, lanoline and paraffin. The slides were imaged on a spinning-disk confocal platform consisting of an inverted Nikon Eclipse Ti2 microscope and a Yokagawa CSU-W1 spinning-disk confocal scanner. The samples were illuminated by an 80-mW solid-state 488-nm laser through a Di01-T405/488/568/647 dichroic (Semrock), and fluorescence was collected through a Nikon ×100 1.45 NA Plan Apo Lambda oil-immersion objective and a green-emission ET525/50 m (525 ± 25 nm) single-band bandpass filter (Chroma). Images were acquired using the Nikon NIS-Elements software with a Hamamatsu ORCA-Fusion BT CMOS camera with 2 × 2 binning and a 16-bit digitizer. For each sample, multiple non-overlapping positions arrayed around the centre of the coverslip were imaged using a Prior Proscan III motorized stage operating at 20% of the maximum speed (to avoid shaking the extract). Focus was maintained at a preset distance from the coverslip using the Nikon Perfect Focus System. Imaging parameters were configured to maximize the SNR ratio while avoiding saturation. All samples intended to be compared were imaged under identical acquisition settings. Samples in Laemmli buffer (50 mM Tris–HCl pH 6.8 at RT, 10% vol/vol glycerol, 2% wt/vol SDS, 0.01% bromophenol blue and 700 mM β-mercaptoethanol) were heated for 5 min at 95 °C and then electrophoretically resolved under denaturing conditions on 4–20% TGX precast polyacrylamide gels (Bio-Rad, catalogue number 5671095). For immunoblotting, samples were transferred from the gels to Immun-Blot PVDF membranes (Bio-Rad, catalogue number 13709A06) for 12 h at 25 V and RT in 25 mM Tris, 192 mM glycine and 10% vol/vol methanol. The membranes were blotted in 20 mM Tris–HCl pH 7.5 at RT, 150 mM NaCl, 0.01% vol/vol Tween 20 and 3% wt/vol fat-free milk solids (Apex, catalogue number 20241) using antibodies to the FLAG epitope (Sigma, F7425) and compatible horseradish peroxidase-conjugated secondary antibodies (Cytiva, NA934). The blots were developed by chemiluminescence (Perkin Elmer, NEL103001EA) and scanned on an Amersham ImageQuant 800 imaging system. For staining with Coomassie blue, gels were fixed in 50% vol/vol methanol and 10% vol/vol acetic acid for 30 min at RT, incubated for 30 min in 50% vol/vol methanol, 10% vol/vol acetic acid and 0.001% wt/vol Coomassie brilliant blue R-250 (Bio-Rad), and destained overnight in 10% vol/vol acetic acid before washing into water. The stained gels were digitized on an Epson Perfection V700 scanner using the Epson Scan 2 software. Cells cultured in YPGO medium (200 ml) were collected by centrifugation at 2,000g and 4 °C for 5 min, resuspended in an equivalent volume of water and re-pelleted. The cell suspension was divided among three 2-ml screw-cap tubes on ice, supplemented with 1 mM phenylmethylsulfonyl fluoride and lysed by beadbeating with 0.5 mm glass beads on a mini-beadbeater (BioSpec). The cell debris was removed by centrifugation at 2,000g and 4 °C for 10 min, and the resulting lysate (2 ml) was further fractionated into a soluble supernatant and a membrane pellet by centrifugation at 20,000g and 4 °C for 30 min. A 1-ml aliquot of the supernatant fraction was supplemented with 1% wt/vol highly pure Triton X-100 (Thermo Fisher, catalogue number 85111) and incubated with anti-FLAG agarose beads (Millipore Sigma, A2220) for 1 h at 4 °C. The pellet fraction was gently washed with lysis buffer, resuspended in 2 ml lysis buffer supplemented with 1% Triton X-100, solubilized for 15 min on ice and clarified by centrifugation at 20,000g and 4 °C for 10 min. A 1-ml aliquot of the resulting solubilisate was incubated with anti-FLAG beads as described earlier. The beads were washed several times with lysis buffer supplemented with 0.01% Triton X-100 and finally boiled in an equivalent volume of 2×Laemmli buffer. A solution of the purified wild-type YG-domain polypeptide (or the variant lacking the KPWE motif) was concentrated to 2 mM in 50 mM HEPES·NaOH pH 7.8 at RT, 2 M urea and 2 mM TCEP. Aliquots of 2 µl were then spotted on the bottom of multiple wells of a 96-well glass-bottomed plate (Cellvis, P96-1.5H-N), which was then sealed with a gas-impermeable AlumaSeal II film (Hampton Research, HR8-069) to prevent desiccation and incubated at RT for 24 h. Before each assay, the resulting gels were equilibrated overnight at RT in a 100-fold excess of Assay Buffer (25 mM HEPES·NaOH pH 7.8 at RT, 150 mM NaCl and 1 mM TCEP). A 0.5 µM solution of fluorescently labelled protein in Assay Buffer was then added to the samples, with or without additional components at the concentrations indicated in the text, and incubated at RT. Gels were imaged at the times indicated in the text using a Nikon A1R point-scanning confocal system consisting of an inverted Nikon Eclipse Ti microscope and a Nikon ×20 0.75 NA Plan Apo air objective. A 50-nm emission band was collected on a gallium arsenide phosphide detector, using an ET525/50m (525 ± 25 nm) single-band bandpass filter (Chroma) and a pinhole dilated to 0.9 Airy units. Images were acquired using the Nikon NIS-Elements software with a pixel size of 1 µm and a 12-bit digitizer. Imaging parameters were configured to maximize the SNR while avoiding saturation. All samples intended to be compared were imaged using identical acquisition settings. YG hydrogels were prepared in a glass-bottomed plate as described earlier and equilibrated for 3 h at RT in Assay Buffer containing 0.5 µM PEX7–GFP or PEX5–GFP with or without PTS2 peptide and receptor at the concentrations specified in the text. Following the initial incubation, the hydrogels were imaged on a Nikon A1R point-scanning confocal system, as described earlier, with a Nikon ×10 0.45 NA Plan Apo air objective, a pinhole dilated to 1.4 Airy units and a pixel size of 2 µm. A solution of purified PEX39 was then spiked into each reaction and the hydrogels imaged every 2 min for an additional 1 h at RT. Cysteine-containing peptides were covalently conjugated to thiol-reactive SulfoLink coupling resin (Thermo Fisher, catalogue number 20401) according to the manufacturer's instructions. The prepared beads were then mixed with Assay Buffer containing the fluorescently labelled protein as well as additional components indicated in the text, and the resulting bead suspensions were transferred to a 384-well glass-bottomed plate (Cellvis, P384-1.5H-N). GST-fusion proteins were instead mixed with glutathione agarose 4B (Prometheus, 20-543) in Assay Buffer containing the fluorescently labelled protein and additional components indicated in the text; the resulting bead suspensions were then transferred to a glass-bottomed plate as above. The plates were incubated at RT for the durations specified in the text and directly imaged on the spinning-disk confocal platform described earlier for peroxisomal import reactions in Xenopus egg extract. Note that imaging the beads directly in the binding reaction preserves the equilibrium and thereby allows even weak interactions to be detected. Fluorescence was collected through a Nikon ×10 0.45 NA Plan Apo air objective, and images were acquired using the Nikon NIS-Elements software and a 16-bit digitizer without binning. Imaging parameters were configured to maximize the SNR while avoiding saturation. Samples intended to be compared were imaged using identical acquisition settings. For the assay shown in Fig. 5d, 15 µl glutathione agarose 4B (Prometheus, 20-543) was mixed with 1 µM purified recombinant yeast PEX7 and varying concentrations of the GST-tagged variants of yeast PEX39, as indicated in the text, in a final volume of 50 µl. Reactions were prepared in Assay Buffer in low protein-binding tubes (Eppendorf, catalogue number 022431081) to minimize sample adhesion to the tube walls. The reactions were incubated at 25 °C for 1 h with agitation. The beads were then quickly sedimented for 30 s at 16,000g and RT, and 20 µl of each supernatant fraction was boiled in reducing Laemmli buffer. The samples were resolved by SDS–PAGE and stained with Coomassie blue as described earlier. For the assay shown in Fig. 8b, the beads were first incubated with yeast PEX7 and the GST-tagged ∆APH variant of yeast PEX39 (each at 1 µM) for 30 min at 25 °C with agitation. Varying concentrations of the peptides, as indicated in the text, were then added and the reactions were incubated for a further 1 h under the same conditions. The beads were finally sedimented and the resulting supernatant fractions resolved by SDS–PAGE as above. All experiments were repeated independently at least three times. No data were excluded from the analyses. Mathematical fitting was performed in GraphPad Prism using least-squares regression. Proteins containing KPWE motifs (or conservative variants) were identified in different species using the MOTIF Search tool at GenomeNet (https://www.genome.jp). Amino-acid sequences were aligned using Clustal Omega63. Secondary structure predictions were performed with AlphaFold 3 (ref. Amino-acid conservation was calculated using ConSurf65. Helical-wheel diagrams were prepared using HeliQuest66. All analyses were performed on the original unmodified images using routine functions in ImageJ67. Fluorescence images were first background-subtracted before further analysis. For the YG hydrogel permeation assay, images were first oriented such that each gel edge faced the same direction. Fluorescence profiles across the buffer–gel interface were then generated by centering a 60 × 300 µm rectangular area on the gel edge and then measuring the mean fluorescence at each position within this selection. This distance was sufficient to fully encompass the permeation front of each fluorescent species. To calculate permeation rates through the YG hydrogel, a 60 × 300 µm rectangular area was first centred on the gel edge. The width of the area above the threshold was then measured and plotted as a function of time. To calculate import rates in Xenopus egg extract, images were first thresholded to identify peroxisomes that were within the focal plane. A circular area with a diameter of 3 pixels was then centred on each local maximum and the mean fluorescence within this area was measured; the average fluorescence in the cytosol was then subtracted from the measured values. The resulting difference (that is, the peroxisome-associated fluorescence) was subsequently converted into a GFP concentration using a calibration curve of GFP fluorescence in the egg extract measured at different concentrations. The peroxisome-associated GFP concentrations were finally plotted as a function of time; linear fits to the data yielded the import rate. To calculate the fraction bound in the quantitative binding assay, digitized images of Coomassie-stained gels were first converted into 16-bit grayscale and their pixel intensities inverted. The mean pixel intensity was then measured within a rectangular selection encompassing each PEX7 protein band and the mean intensity of the background was subtracted from the measured values. The background-corrected intensities (corresponding to unbound PEX7 remaining in solution) were in turn subtracted from the intensity of the PEX7 band in a sample without PEX39 (that is, total PEX7). The data were fitted either to a standard binding isotherm or a competitive-binding isotherm, as appropriate, to estimate the observed binding constants. All images intended to be compared were processed identically. Fluorescence micrographs were normalized to the background before being linearly contrast-stretched in ImageJ to the same bit range. Digitized images of immunoblots and Coomassie-stained gels were linearly contrast-stretched in ImageJ to reveal relevant bands but avoid clipping of the background, whereas photographs of YG hydrogels were processed using Adobe Photoshop. Structural models of proteins were rendered using the PyMOL Molecular Graphics System (Schrödinger). Figures were assembled for publication in Adobe Illustrator. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Original, unmodified fluorescence images are available from Mendeley Data68. Reagents generated by this study are available from the corresponding authors on request. Source data are provided with this paper. De Duve, C. & Baudhuin, P. Peroxisomes (microbodies and related particles). A., Baes, M., Ribeiro, D., Ferdinandusse, S. & Waterham, H. R. The physiological functions of human peroxisomes. Léon, S., Goodman, J. M. & Subramani, S. Uniqueness of the mechanism of protein import into the peroxisome matrix: transport of folded, co-factor-bound and oligomeric proteins by shuttling receptors. Rapoport, T. A., Li, L. & Park, E. Structural and mechanistic insights into protein translocation. Wiedemann, N. & Pfanner, N. Mitochondrial machineries for protein import and assembly. Brocard, C. & Hartig, A. Peroxisome targeting signal 1: is it really a simple tripeptide? Kunze, M. The type-2 peroxisomal targeting signal. van Roermund, C. W. T., Waterham, H. R., Ijlst, L. & Wanders, R. J. A. Fatty acid metabolism in Saccharomyces cerevisiae. & Skjeldal, O. H. Refsum disease, peroxisomes and phytanic acid oxidation: a review. From peroxisomal disorders to common neurodegenerative diseases—the role of ether phospholipids in the nervous system. Towards solving the mystery of peroxisomal matrix protein import. & Berg, J. M. Peroxisomal targeting signal-1 recognition by the TPR domains of human PEX5. Characterization of proteome dynamics during growth in oleate reveals a new peroxisome-targeting receptor. Effelsberg, D., Cruz-Zaragoza, L. D., Schliebs, W. & Erdmann, R. Pex9p is a new yeast peroxisomal import receptor for PTS1-containing proteins. Skowyra, M. L. & Rapoport, T. A. PEX5 translocation into and out of peroxisomes drives matrix protein import. Dammai, V. & Subramani, S. The human peroxisomal targeting signal receptor, Pex5p, is translocated into the peroxisomal matrix and recycled to the cytosol. Gao, Y., Skowyra, M. L., Feng, P. & Rapoport, T. A. Protein import into peroxisomes occurs through a nuclear pore-like phase. Ravindran, R. et al. Peroxisome biogenesis initiated by protein phase separation. & Görlich, D. Transport selectivity of nuclear pores, phase separation, and membraneless organelles. Feng, P. et al. A peroxisomal ubiquitin ligase complex forms a retrotranslocation channel. Structure and function of the peroxisomal ubiquitin ligase complex. Debelyy, M. O. et al. Ubp15p, a ubiquitin hydrolase associated with the peroxisomal export machinery. Identification of ubiquitin-specific protease 9X (USP9X) as a deubiquitinase acting on ubiquitin–peroxin 5 (PEX5) thioester conjugate. Pan, D., Nakatsu, T. & Kato, H. Crystal structure of peroxisomal targeting signal-2 bound to its receptor complex Pex7p–Pex21p. The mammalian peroxin Pex5pL, the longer isoform of the mobile peroxisome targeting signal (PTS) type 1 transporter, translocates the Pex7p.PTS2 protein complex into peroxisomes via its initial docking site, Pex14p. Dodt, G., Warren, D., Becker, E., Rehling, P. & Gould, S. J. Domain mapping of human PEX5 reveals functional and structural similarities to Saccharomyces cerevisiae Pex18p and Pex21p. Schliebs, W. & Kunau, W. H. PTS2 co-receptors: diverse proteins with common features. Zhang, J. W. & Lazarow, P. B. PEB1 (PAS7) in Saccharomyces cerevisiae encodes a hydrophilic, intra-peroxisomal protein that is a member of the WD repeat family and is essential for the import of thiolase into peroxisomes. Elgersma, Y. et al. A mobile PTS2 receptor for peroxisomal protein import in Pichia pastoris. Mukai, S., Ghaedi, K. & Fujiki, Y. Intracellular localization, function, and dysfunction of the peroxisome-targeting signal type 2 receptor, Pex7p, in mammalian cells. Ghys, K., Fransen, M., Mannaerts, G. P. & Van Veldhoven, P. P. Functional studies on human Pex7p: subcellular localization and interaction with proteins containing a peroxisome-targeting signal type 2 and other peroxins. Nair, D. M., Purdue, P. E. & Lazarow, P. B. Pex7p translocates in and out of peroxisomes in Saccharomyces cerevisiae. Pilar, A. V. C., Madrid, K. P. & Jardim, A. Interaction of Leishmania PTS2 receptor peroxin 7 with the glycosomal protein import machinery. Rodrigues, T. A. et al. A PEX7-centered perspective on the peroxisomal targeting signal type 2-mediated protein import pathway. & Rapoport, T. A. Peroxisome protein import recapitulated in Xenopus egg extracts. Skowyra, M. L. & Rapoport, T. A. Cell-free reconstitution of peroxisomal matrix protein import using Xenopus egg extract. Stein, K., Schell-Steven, A., Erdmann, R. & Rottensteiner, H. Interactions of Pex7p and Pex18p/Pex21p with the peroxisomal docking machinery: implications for the first steps in PTS2 protein import. Kötter, P., Weigand, J. E., Meyer, B., Entian, K.-D. & Suess, B. A fast and efficient translational control system for conditional expression of yeast genes. Hazbun, T. R. et al. Assigning function to yeast proteins by integration of technologies. Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. Architecture of the human interactome defines protein communities and disease networks. Huttlin, E. L. et al. Dual proteome-scale networks reveal cell-specific remodeling of the human interactome. Chen, W. W. et al. PEX39 facilitates the peroxisomal import of PTS2 proteins. Import of the peroxisomal targeting signal type 2 protein 3-ketoacyl-coenzyme A thiolase into glyoxysomes. The di-aromatic pentapeptide repeats of the human peroxisome import receptor PEX5 are separate high affinity binding sites for the peroxisomal membrane protein PEX14. Schliebs, W., Girzalsky, W. & Erdmann, R. Peroxisomal protein import and ERAD: variations on a common theme. Hensel, A. et al. Cysteine-dependent ubiquitination of Pex18p is linked to cargo translocation across the peroxisomal membrane. Alencastre, I. S. et al. Mapping the cargo protein membrane translocation step into the PEX5 cycling pathway. Rodrigues, T. A., Grou, C. P. & Azevedo, J. E. Revisiting the intraperoxisomal pathway of mammalian PEX7. Ren, Y., Seo, H.-S., Blobel, G. & Hoelz, A. Bailer, S. M. et al. Nup116p and nup100p are interchangeable through a conserved motif which constitutes a docking site for the mRNA transport factor gle2p. DeLoache, W. C., Russ, Z. N. & Dueber, J. E. Towards repurposing the yeast peroxisome for compartmentalizing heterologous metabolic pathways. Mülleder, M., Campbell, K., Matsarskaia, O., Eckerstorfer, F. & Ralser, M. Saccharomyces cerevisiae single-copy plasmids for auxotrophy compensation, multiple marker selection, and for designing metabolically cooperating communities. Gleeson, M. A., White, C. E., Meininger, D. P. & Komives, E. A. Generation of protease-deficient strains and their use in heterologous protein expression. Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications. Saleem, R. A. et al. Genome-wide analysis of signaling networks regulating fatty acid-induced gene expression and organelle biogenesis. Gietz, R. D. & Schiestl, R. H. High-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method. Raran-Kurussi, S., Cherry, S., Zhang, D. & Waugh, D. S. Removal of affinity tags with TEV protease. Novel peroxisome clustering mutants and peroxisome biogenesis mutants of Saccharomyces cerevisiae. Suess, B., Entian, K.-D., Kötter, P. & Weigand, J. E. Aptamer-regulated expression of essential genes in yeast. Sievers, F. et al. Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Glaser, F. et al. ConSurf: identification of functional regions in proteins by surface-mapping of phylogenetic information. & Drin, G. HELIQUEST: a web server to screen sequences with specific alpha-helical properties. Import mechanism of peroxisomal proteins with an N-terminal signal sequence. Waters and the Core for Imaging Technology and Education (CITE) at Harvard Medical School for microscopy assistance, and C. Saenz and the Microfluidics Core Facility at Harvard Medical School for plasma-etching coverslips. is a Howard Hughes Medical Institute (HHMI) investigator. This work was supported by NIGMS grant number R01 GMO52586 to T.A.R. and an HHMI-sponsored Helen Hay Whitney Foundation fellowship (award number F-1255) to M.L.S. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Present address: Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, TX, USA Howard Hughes Medical Institute and Department of Cell Biology, Harvard Medical School, Boston, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Michael L. Skowyra or Tom A. Rapoport. The authors declare no competing interests. Nature Cell Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Xenopus egg extract was depleted or not (mock) of endogenous PEX5 using beads conjugated to the PEX5-binding domain from PEX14, then supplemented with buffer or the indicated purified recombinant PEX5 variants (described in Fig. Import was assessed by imaging the indicated fluorescent cargo on a spinning-disk confocal microscope. Repeated rounds of cargo import into peroxisomes produce bright puncta. b, The indicated recombinant proteins were produced and purified as described in the Methods, then resolved by SDS–PAGE and stained by Coomassie Blue. Molecular weights (in kD) are marked on the left; images of the unprocessed gels are provided. a, Glutathione-conjugated beads were bound to 1 µM GST alone or to the purified recombinant GST-tagged PEX5 variants shown (described in Fig. 1b), and incubated in the presence of 1 µM purified recombinant yeast PEX7 and 1 µM GFP fused at the N terminus to the import signal (residues 1–27) from yeast POT1 (PTS2). Recruitment of the fluorescent cargo to the beads was visualized after 1 h on a spinning-disk confocal microscope as a bright ring around the bead perimeter. b, Xenopus egg extract was depleted of endogenous PEX5 using beads conjugated to the PEX5-binding domain from PEX14, then supplemented with the chimeric receptor PEX5-21 (described in Fig. Import was assessed using fluorescent PTS1 and PTS2 cargo: the PTS2 cargo is described in panel a, whereas the PTS1 cargo consisted of GFP fused at the C terminus to the SKL peroxisome targeting signal. Reactions were imaged at the indicated times on a spinning-disk confocal microscope. The fluorescence intensity of the resulting puncta (corresponding to imported cargo) was then measured, converted into GFP concentrations as described in the Methods, and plotted as a function of time. Error bars span the range of at least 30 imaged fields; lines represent linear fits to the data generated by least-squares regression. c, As in panel a, using GST alone or the indicated GST-tagged PEX5 variants (described in Fig. 1b), in the presence or absence of purified recombinant yeast PEX7 and the fluorescent cargo described in panels a and b. Source numerical data are provided. a, Left, scheme depicting the domain organization of PEX13 from Arabidopsis thaliana. Note that A. thaliana PEX13 lacks the transmembrane segment and SH3 domain found in metazoan PEX13. Relevant residue numbers are specified on top, and the region that was produced recombinantly to prepare YG hydrogels is underlined beneath. The recombinant fragment was fused at the N terminus to a tetradecahistidine (14×His) tag and a TEV protease-cleavage site, and at the C terminus to a cysteine (Cys) residue. Lanes 1 and 5 show each His-tagged precursor purified by nickel-affinity chromatography in the presence of urea (as described in the Methods), resolved by SDS–PAGE and stained by Coomassie blue. Lanes 2 and 6 show each fragment after cleavage with TEV protease to release the His tag. Lanes 3 and 7 show the soluble fraction (sup) of each TEV-cleavage reaction, demonstrating that each fragment remains soluble in urea in the absence of the tag. The cleaved tag was removed from the reactions by a subsequent nickel-affinity step, and the resulting flow-through containing each final purified fragment is shown in lanes 4 and 8. Molecular weights (in kD) are marked on the side; an image of the unprocessed gel is available in source data. b, Thiol-reactive beads were conjugated to the purified wild-type (WT) YG-domain polypeptide or the version lacking the KPWE motif (∆KPWE) described in panel a. The beads were then incubated with PEX7 fused at the C terminus to GFP (PEX7–GFP) or with GFP alone, and binding of each fluorescent protein to the immobilized fragments was visualized on a spinning-disk confocal microscope. a, Top, AlphaFold model from Fig. Below, the model coloured according to the predicted local distance difference test (pLDDT) score computed by AlphaFold for each residue, with a score of 100 reflecting the highest confidence. b, As in panel a, showing the AlphaFold model from Fig. c, As in panel a, showing the AlphaFold model from Fig. d, As in panel a, showing the AlphaFold model from Fig. a, Interaction between full-length yeast PEX7 and residues 1–27 (including the KPWE motif) of yeast PEX13, predicted by AlphaFold. PEX7 is shown as a space-filling model in the same orientation as in Fig. 3c, coloured by surface electrostatic potential according to the scale on the right. The PEX13 fragment is depicted as a ribbon, with the side chain of lysine 10 (K10) of the KPWE motif shown. The region outlined by a dashed black line is magnified on the right. b, As above, with PEX7 coloured by surface hydrophobicity according to the scale on the right, and showing the side chain of proline 11 (P11) of the KPWE motif. c, As above, with PEX7 coloured yellow and showing the side chain of glutamate 13 (E13) of the KPWE motif. E13 is predicted to contact arginine 117 (R117) within the loop of one of the WD40 blades of the PEX7 β-propeller (overlaid on top as a cartoon representation). d, As in panel c, with the model rotated as indicated and showing the side chain of tryptophan 12 (W12) of the KPWE motif. W12 is predicted to contact the aliphatic stem of lysine 230 (K230) within the loop of one of the WD40 blades of the PEX7 β-propeller (overlaid on top as a cartoon representation). a, Thiol-reactive beads were conjugated to a cysteine-containing synthetic peptide comprising the region of yeast PEX13 that contains the KPWE motif, or a peptide in which the motif was scrambled (both described in Fig. The beads were then incubated with or without 1 µM yeast PEX7, in the presence of 1 µM cargo consisting of GFP fused at the N terminus to the import signal (residues 1–27) from yeast POT1 (PTS2). Where indicated, reactions included 1 µM of either the chimeric receptor PEX5-21 (described in Fig. 1b) that can bind to PEX7, or the short receptor isoform PEX5S that cannot bind. Recruitment of the cargo to the beads after 1 h was visualized on a spinning-disk confocal microscope. b, Glutathione-conjugated beads were bound to the purified recombinant GST-tagged chimeric receptor PEX5-21 (described in Fig. The beads were then incubated with 1 µM PEX7–GFP in the absence or presence of a synthetic peptide comprising the PTS2 signal (residues 2–15) from yeast POT1; the fold excess of the peptide relative to PEX7–GFP is indicated above. Recruitment of PEX7–GFP to the beads was visualized on a spinning-disk confocal microscope. Note that the beads are less porous than those in panel a, so binding appears as a bright ring around the bead perimeter. a, PEX7 was fused at the C terminus to a FLAG tag (PEX7–FLAG) and expressed at endogenous levels in yeast. Shown is a FLAG immunoblot of the total lysate from cells expressing wild-type (WT) untagged PEX7 or two clones expressing PEX7–FLAG. Asterisk indicates non-specific bands; molecular weights (in kD) are marked on the left. b, Yeast cells lacking PEX7 (pex7∆), or expressing PEX7–FLAG, were grown in oleic acid to induce peroxisomes. Peroxisomal protein import was assessed by imaging the endogenous matrix proteins FOX2 fused to RFP (PTS1) and POT1 fused to GFP (PTS2). Note that bright puncta indicate successful import; in the absence of import, the cargo is diffusely distributed throughout the cytosol. c, PEX7–FLAG was expressed in cells lacking PEX13 (pex13∆), or cells containing wild-type (WT) PEX13 or a mutant lacking the KPWE motif (∆KPWE). Cells were grown in oleic acid to induce peroxisomes, then lysed and separated into a peroxisome-containing membrane fraction. PEX7–FLAG was immunoprecipitated from the total lysate and the corresponding detergent-solubilized membranes, resolved by SDS–PAGE, and detected by immunoblotting for the FLAG epitope. Asterisk marks non-specific bands; molecular weights (in kD) are marked on the left. d, Yeast cells devoid of PEX13 (pex13∆), or expressing PEX13 lacking residues 2–13 that encompass the KPWE motif, were grown in oleic acid to induce peroxisomes. Peroxisomal protein import was assessed as in panel b. e, Xenopus egg extract was incubated for 1 h with 0.5 µM GFP alone or GFP fused to the C terminus of the indicated proteins, then imaged on a spinning-disk confocal microscope. Where indicated, reactions additionally included the chimeric receptor PEX5-21 (described in Fig. 1b) at 0.5 µM and a synthetic peptide comprising the PTS2 signal from yeast POT1. Note that puncta reveal entry of the fluorescent species into peroxisomes. The N terminus of PEX39 is marked for reference. A conserved groove along the side of PEX7 is framed by dashed black lines. The four rightmost panels show the same complex but rotated as indicated. b, As in panel a, showing the crystal structure (PDB: 3W15) of the complex between yeast PEX7, a PTS2 peptide, and the yeast PTS2 receptor PEX21. The complex is overlaid on the AlphaFold model of yeast PEX7 shown in panel a (with PEX7 from the crystal structure not visible). Note that a portion of the receptor snakes along the same conserved groove (framed by dashed black lines) on the side of PEX7 to which PEX39 is predicted to bind. Furthermore, the receptor and PTS2 peptide are cradled by the same conserved surface on top of PEX7 that also binds to the APH of PEX39. a, Glutathione-conjugated beads were bound to 1 µM purified recombinant GST-tagged yeast PEX7 or GST alone, and incubated with 1 µM purified recombinant yeast PEX39 fused at the C terminus to GFP (PEX39–GFP) or with GFP alone. Recruitment of each fluorescent protein to the beads was visualized after 1 h on a spinning-disk confocal microscope. b, Representative Coomassie-stained SDS–PAGE gels corresponding to the binding experiment in Fig. Briefly, yeast PEX7 was incubated with beads coated with GST-tagged wild-type yeast PEX39 or the indicated mutants; after sedimenting the beads, each supernatant fraction was resolved by SDS–PAGE and stained by Coomassie Blue. Molecular weights (in kD) are marked on the left. c, PEX7 and PEX39 were fused at their C terminus to a FLAG tag (PEX7–FLAG and PEX39–FLAG) and expressed from their respective endogenous promoters in yeast. Shown is a FLAG immunoblot of the total lysate from two different clones (lanes 3 and 4) following growth in oleic acid, compared to cells expressing PEX7–FLAG and untagged PEX39 (lane 2), and cells expressing untagged versions of both proteins (lane 1). Asterisk indicates non-specific bands; molecular weights (in kD) are marked on the left. d, YG hydrogels were incubated with GFP alone or with GFP fused to the C terminus of the indicated proteins. The accumulation of each fluorescent protein inside the gel after 30 min was visualized on a point-scanning confocal microscope. Representative images are shown; the fold enrichment of each protein relative to buffer, across the imaged field, is plotted above (mean ± range of 3 experiments). The width of each image corresponds to 300 µm. Source numerical data and unprocessed blots are provided. Plasmids and yeast strains used in this study. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Import mechanism of peroxisomal proteins with an N-terminal signal sequence. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/measuring-gluons-and-quarks-at-the-brookhaven-relativistic-heavy-ion/'>How the Nation's Only Particle Collider Is Accelerating Our Understanding of Physics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Brookhaven National Laboratory is pushing the boundaries of particle physics. By Rachel Feltman, Naeem Amarsy, Fonda Mwangi, Kelso Harper & Jeffery DelViscio Today we're taking you on another one of our Friday Fascination field trips with an auditory journey to Brookhaven National Laboratory. This Long Island facility boasts seven Nobel Prize–winning discoveries and more than 70 years of groundbreaking research into energy and the environment. Earlier this year the Science Quickly team visited Brookhaven to get a look at its Relativistic Heavy Ion Collider, or RHIC, which has been helping scientists study subatomic particles since 2000. To guide us through these weighty subatomic topics, I chatted with Brookhaven's own Alex Jentsch. You'll notice that this episode's audio quality is lower than usual, but that's because we were hanging out next to giant science machines. If you want to see those incredible instruments for yourself—and get access to an extended version of my conversation with Alex—check out our YouTube channel for a video edition of today's episode. For now here's part of our chat at Brookhaven. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. We take you inside the Relativistic Heavy Ion Collider, housed at Brookhaven National Laboratory. Alex Jentsch: Nice to meet you, too, Rachel. Jentsch: And this monstrosity behind us is basically just a very large digital camera that we use to take pictures of collisions of particles. And so basically you can imagine this digital camera taking a picture of a firecracker after it blew up and then trying to put it all back together. Feltman: And what kind of stuff can we learn with that? So the original reason that we built this facility and built this collider was to study the early universe, so we were actually trying to recreate stuff that would've existed right after the big bang. So we obviously can't go visit the big bang very easily, unless you're a Time Lord, and so ultimately the only way we can do this is by trying to recreate those conditions in the lab. So that was kind of the first main pillar of the science program, was looking at a situation where protons and neutrons were not protons and neutrons; they were actually melted into their constituents for a very short period of time. Alternatively, we can also look at just basic structure of the proton. We can understand things like its mass, its charge and its spin, which is kind of the esoteric aspect of a proton, but that's another thing we can do with this camera. It would be great if you could just give us a quick overview of sort of the basic terms we need ... Jentsch: So atoms are already fairly complicated. So we learn in chemistry that atoms have these electrons zipping around the outside in this cloud. So the nucleus is made up of these positively charged particles called protons and these neutral particles called neutrons. We got new tools that we could use to probe deeper inside and found, “Oh, there's stuff inside the proton and neutron. So we found these objects—we had no name for them yet; we didn't really expect them to be there. And one of the first things we had to figure out was what they're doing, what their properties are, how they're held together. They seem to be always bound up inside the protons and neutrons. So we start out calling these things quarks because we couldn't really figure out a name and we found something in a James Joyce novel that sounded like it matched the, the weird thing we found. And we know that, from the electromagnetic force, that photons are kind of the force-carrying particle between charged items. Jentsch: So electrons, protons, they see their electric force by being able to exchange a photon; there must be something similar for quarks. And so since they're kind of glued together, we thought, “Okay, maybe this particle's called a gluon. Maybe it's the thing that's actually communicating this much stronger force than what we see for electromagnetism.” And so from all of that we learned that the, the strong force is actually much stronger than electromagnetism, it's communicated via this thing we call the gluon. Feltman: Quick follow-up: And what's the strong force [laughs]? Jentsch: So the strong nuclear force is actually what binds protons and neutrons together and eventually binds them inside of atoms together. 'Cause if you think about it, protons have positive charge, neutrons have no charge—how are the positive charge items holding themselves together into a nucleus? You know, what does an experiment entail? Jentsch: So the accelerator part is immensely complicated; it's actually a whole complex of accelerators. So you can picture it kind of like the entrance ramp onto a highway. They eventually get on the ramp, and then they get on the highway and speed up. Jentsch: That's kind of how the RHIC facility operates. Jentsch: That part is a whole set of expertise and a whole lot of challenges, and getting the particles into RHIC is only half the battle. Jentsch: So those ring magnets are being ramped up in their field at the same time that we're speeding the particles up really, really close to the speed of light. And then eventually our friends at the collider department use other magnets to force the two beams to collide at the center of our detector, and then we start working. So we're in a control room on the other side of this building. We turn on all of our detectors and start taking data, and that process goes on for about eight hours at a time. And so while we're in the control room, we babysit [laughs] all of the detectors, make sure the data-taking is going smoothly, and things always happen that we don't expect. And so we try not to run in the middle of summer, for example, because it gets hot and the power grid on Long Island [laughs] is not so happy with how much power we're drawing, and so eventually we get little dips, and that can shut things down for quite a while. Feltman: So what are some examples of findings that have taken place because of the work here at Brookhaven? Jentsch: So one of the initial things that was really interesting is that when we smashed these particles together, we originally started smashing gold ions together, and there was a reason for that. To recreate the early universe we want a lot of density and a lot of temperature. Jentsch: And so by smashing heavy ions together the hope would be that in a very, very, very tiny nuclear scale you recreate the early universe—really, really, really small. Jentsch: And you get what, originally, we thought was a gas of the quarks and gluons inside. That wasn't really what we ended up finding. We actually found that they don't really behave like a gas at all. They kind of behave like a liquid … Jentsch: Which is really not something we expected. And so as we started to do more work and study different things about the properties of this quark-gluon plasma we created inside, a lot of it was not what we really expected it to be. So for example, we expected, because it might be a gas, that it would be fairly weakly interacting, which is not what you'd expect from the strong force, right? But it turns out it's actually quite strongly interacting—that's why it moves kind of like a fluid: all the particles actually kind of talk to each other. Jentsch: And that has consequences for some of the things we try to measure. As particles go through this plasma they lose energy. And so when we measure them in our detector they are not really behaving the way we originally thought they might. Jentsch: And so we actually made some of the first constraint measurements here at STAR and at PHENIX down the road that showed the amount of spin that could be coming from the gluons, and that was, like, a first-ever measurement. Can you tell us a little bit about the transition that's happening? Jentsch: Yeahthis year we'll have our final data-taking for RHIC. And at that point we spend a couple of years taking all this stuff out because we have to make room for the new facility. And that process is really difficult 'cause you can see this is not a small piece of equipment. So it's exciting—we're really ready for the new physics—but it's also bittersweet because a lot of us have been working on this for a long time. I've been working on STAR for, like, 13 years, and so I have fond memories of sitting in the control room with friends from all over the world eating, you know, really salty snacks to stay awake and drinking coffee. And that's gonna be, you know, going away at the end of this year, and we'll have quite a long time before we start taking data with the new facility. Jentsch: So the kinds of collisions we do right now are either gold smashing on gold or proton smashing on proton or even proton smashing on gold. Jentsch: They're really complicated items, and as you speed 'em up to really high energy they get more complicated. And smashing two really complicated objects together means that trying to study it is not easy. Jentsch: By using electrons as one of the beams instead of another proton or an ion— so you have an electron smashing into an ion—the electron basically serves as a source of light. It serves as a source of photons. Jentsch: You can't study everything that we study here—you can't study a quark-gluon plasma, for instance, 'cause you don't have the energy density of all those quarks together—but you can study what the nucleus really looks like at a very, very, very small scale. You can study what the nucleus might look like before it would've collided in RHIC. And so in some sense you can actually use the data we'll take 10 years from now to reanalyze data from now to try to understand more than we do now, which means we have to preserve [laughs] all the data that we've been taking for the last 25 years, and so there's a whole discussion now about the computing facility resources we're gonna need to store all of the data we've been taking … Jentsch: Make sure it stays safe [laughs]. Feltman: Can you give me a sense of how much data has been collected in that time [laughs]? Jentsch: Right now we're collecting an average of something like 15 petabytes per year … Feltman: I don't even know what a petabyte is [laughs]. The machine was at, you know, a much lower rate of collisions because we had to make sure the detector was safe, and computing power was not as good 25 years ago as it is now. So basically, as computing has caught up with what we wanna be able to do, we've increased the amount of data we take with each passing year to kind of make use of [laughs] all the computers we have available now. Jentsch: And so we've really stretched them to their max. I really wanna know the basic, fundamental aspects of the proton that we don't know. But the quarks don't make up most of that mass. Jentsch: We know how much things weigh—we measure this all the time. Why don't we know how a proton gets the mass it has? But then also even the proton spin. Spin is a less understood quantity, it's more esoteric, but we thought originally that this was coming just from the quarks. There's stuff that's more interesting to other people in the community, such as what happens when you take a nucleus of gold, for example, and you accelerate it near the speed of light? The question is: Does it just keep producing gluons infinitely? If it does the former, that's new physics; that's going outside of what we know from quantum mechanics. If it kind of saturates at some point, then that would be something we more or less would expect, but we don't know, and that's kind of studying something that's in the same ballpark as the quark-gluon plasma. It's basically a state of matter that is dominated by the strong force. Feltman: Yeah, and also very obvious question I should have started with: How big is it [laughs]? And most of what you see that's heavy is iron. So there's a big magnet inside of this, and that magnet has to have a place for those magnetic fields to go ... Jentsch: So we have iron to protect and remove all that magnetic field. Feltman: When you're, like, talking to a member of the public and physics makes their head spin, what is your headline—like, “This is why what we're doing is really cool and important”—for them? Jentsch: Well, we're made of protons and neutrons. Everything around us is made of protons,neutrons, electrons; they're fundamental. Jentsch: But isn't it interesting to wanna understand how these things achieve their basic properties? We know they have an electric charge. Additionally, it's really cool to be able to study things from the early universe in a laboratory environment. And the fact that you have to build something this large to study something so small is also rather insane. Jentsch: So in and of itself this is just kind of a wild idea: that we have a machine this large to study something that we cannot see with the human eye. Jentsch: And so my, my hope would be that that is enough to drive someone toward a little bit of curiosity: You know, what are we made of? That's really what kind of drives me toward it. Feltman: Yeah, I think most people would agree that understanding the basic particles that make up literally everything is pretty important, so thanks so much for taking the time to tell us about it and show us around. Jentsch: Yeah, thanks for having me, and I hope to see you again in 10 years when we start taking data with the EIC. Feltman: That's all for today's episode, at least in the audio universe. If you want to hear—and see—more of what we learned at Brookhaven, check out the extended video version of this episode over on our YouTube channel. You'll find a link to that in our show notes. You'll also see a link to our listener survey, and I'd be so grateful if you could take a second to fill it out. We're looking for information about our listeners and their preferences so we can continue making Science Quickly the best it can possibly be. If you submit your survey this month, you'll be entered to win some awesome Scientific American swag. We'll be back on Monday with our usual science news roundup. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper, Naeem Amarsy and Jeff DelViscio. This episode was edited by Alex Sugiura. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Naeem Amarsy is a documentary filmmaker and multimedia editor based in New York City. Fonda Mwangi is a multimedia editor at Scientific American. She previously worked as an audio producer at Axios, The Recount and WTOP News. Kelso Harper is an award-winning senior multimedia editor at Scientific American. As a producer, editor and host, they work on short documentaries, social videos and Scientific American's podcast Science Quickly. They have a bachelor's degree in chemistry from Johns Hopkins University and a master's degree in science writing from the Massachusetts Institute of Technology. Previously, they worked with Wired, Science, Popular Mechanics, and MIT News. Jeff DelViscio is currently chief multimedia editor/executive producer at Scientific American. He is former director of multimedia at STAT, where he oversaw all visual, audio and interactive journalism. Before that, he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02167-5'>The multilayered transcriptional architecture of glioblastoma ecosystems</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 09:58:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In isocitrate dehydrogenase wildtype glioblastoma (GBM), cellular heterogeneity across and within tumors may drive therapeutic resistance. Here we analyzed 121 primary and recurrent GBM samples from 59 patients using single-nucleus RNA sequencing and bulk tumor DNA sequencing to characterize GBM transcriptional heterogeneity. First, GBMs can be classified by their broad cellular composition, encompassing malignant and nonmalignant cell types. Second, in each cell type we describe the diversity of cellular states and their pathway activation, particularly an expanded set of malignant cell states, including glial progenitor cell-like, neuronal-like and cilia-like. Third, the remaining variation between GBMs highlights three baseline gene expression programs. These three layers of heterogeneity are interrelated and partially associated with specific genetic aberrations, thereby defining three stereotypic GBM ecosystems. How this architecture evolves during disease progression is addressed in the companion manuscript by Spitzer et al. Tumor heterogeneity is a hallmark of isocitrate dehydrogenase (IDH) wildtype (WT) glioblastoma (GBM)1. Nonmalignant cells in the tumor microenvironment (TME) establish an ecosystem with complex interactions that also plays critical roles in shaping GBM biology and treatment response5,7,8,9. Previous studies used single-cell RNA sequencing (scRNA-seq) to uncover gene expression heterogeneity within human gliomas3,4,10,11,12,13,14,15,16,17. These studies revealed that GBM malignant cells exhibit a variety of cellular states, including those resembling neurodevelopmental cell types. However, previous studies had several limitations: (1) they were based on limited number of samples and cells and could still miss rare cellular states; (2) they relied on enzymatic tumor dissociation, which may deplete certain cell types and states, as evident by the scarcity of detected astrocytes, neurons and endothelial cells; (2) they addressed either heterogeneity across tumors (that is, tumor subtypes) or within tumors (that is, cell states) but did not develop an integrated model that explains tumor heterogeneity by a combination of tumor-level and cell-level features; (4) they lacked in-depth genetic characterization of the tumors; (5) they had limited clinical metadata; and (6) they were published by different groups, and leveraged diverse pipelines, analytics and classification schemes4,10,12,15,16. To address these challenges, we created the GBM Cellular Analysis of Resistance and Evolution (CARE) consortium and profiled 121 IDH WT GBM samples using single-nucleus RNA sequencing (snRNA-seq) and bulk DNA sequencing, thereby establishing a multi-omic dataset of unprecedented size while eliminating the confounding effects of enzymatic tumor dissociation. In this article, we describe the identification of new GBM cellular states, for both the malignant and TME compartments. The main conclusions of this article are independent of the timing of sampling (primary or recurrent) and can be reached by only analyzing primary GBM specimens, yet we include the recurrent samples for more robust statistical analysis and to provide a broad view of the range of GBM tumors. In the accompanying study by Spitzer et al.18, we focus on characterizing the longitudinal evolution of GBM after standard-of-care therapy. Our groups jointly generated an snRNA-seq dataset of 121 tumor samples from 59 patients spanning initial disease and recurrence (Fig. 1a; see Supplementary Table 1 for detailed clinical information). All tumors were diagnosed as ‘Glioblastoma, IDH WT' based on the World Health Organization (WHO) 2021 classification, which was further confirmed using bulk DNA sequencing. We profiled all 121 samples with the droplet-based 10x Genomics platform, and also profiled a large subset of tumors (54 of 121) using plate-based full-length Smart-seq2 (ref. Most samples (116 of 121) were also profiled using bulk whole-exome sequencing (WES) or whole-genome sequencing (WGS), enabling the integration of genetic information with single-nucleus transcriptomics (Fig. a, Scheme describing the workflow of our study. b, The proportion (%) of patients (n = 46) in the dataset in which a genetic aberration was detected is shown. Arm-level refers to whole-chromosome arm amplification or deletion. CNA refers to gene-level amplification or deletion. c, Uniform manifold approximation and projection (UMAP) for dimension reduction based on gene expression of the 429,305 cells in our cohort colored according to the inferred cell type. d, Pie chart demonstrating the proportion (%) of each cell type in this dataset. In the 10x snRNA-seq dataset, 429,305 of 752,458 cells passed our stringent quality control (Methods) with 2,318 genes detected per cell on average (Extended Data Fig. Cells were classified as malignant (246,408 cells) or nonmalignant (182,897 cells) by inferring chromosomal copy number aberrations (CNAs) from gene expression data, as described previously13,21, but with additional steps to improve classification accuracy (Extended Data Fig. Clustering of all cells highlighted the diversity of malignant cells, including many small patient-specific clusters, while nonmalignant cells were intermixed across patients without apparent batch effects (Fig. Nonmalignant cells were assigned to cell types using clustering and well-established signatures (Extended Data Fig. Notably, we detected neurons and astrocytes (Fig. 1c,d) that were typically not captured in scRNA-seq studies with enzymatic tissue dissociation. In the Smart-seq2 dataset, 10,062 of 11,136 cells passed our quality control and were similarly classified to malignant and nonmalignant cell types (Extended Data Fig. Overall, the fractions of cell types were similar between 10x Genomics and Smart-seq2 datasets (Extended Data Fig. While this robust classification scheme has been broadly adopted and validated by the community5,27,28,29, we sought to extend the atlas of GBM malignant cellular states by leveraging our much larger snRNA-seq dataset. First, nonnegative matrix factorization (NMF) was used to derive gene expression programs that vary within each tumor30. After removing programs suspected of low technical quality (for example, enrichment of ribosomal proteins or pseudogenes; Methods), we found ten groups of similar programs and defined a consensus meta-program (MP) for each of them, representing the malignant cellular states in our dataset (Fig. a, Jaccard similarity indices of the robust NMF programs (n = 707) based on their top 50 genes. Programs are ordered according to clustering and grouped into MPs (Methods). MP10 (stress1) and MP15 (stress2) were coalesced into a single stress state. The ‘cycling' MP is separated to emphasize it as a feature that cells can have on top of their neurodevelopmental identity (and not as an independent state; Extended Data Fig. ‘Unresolved' represents cells that could not be confidently classified to a particular state. c, Relative expression (centered log2) of the three new MPs discovered in this work. d, Association between the malignant MPs (rows) and gene sets reflecting normal brain development and adult brain cell types (columns). e, Association between pathway-based (rows) and gene-based (columns) MPs. f, Demonstration of the hybrid state concept. The columns represent cells, which are aggregated according to their singular (AC, GPC) or hybrid (AC-GPC) state. The heatmap of genes' centered log2 expression shows that hybrids express two MPs to the same extent, whereas singulars highly express a single MP. g, GBM cellular hierarchy model stemming from the high-frequency hybrid pairs. Each quadrant reflects a cellular state and each dot represents a malignant cell; exact cell positions reflect their relative scores for the MPs, and their colors reflect the proportion of cells around them classified to the GPC state. One of these MPs represented the cell cycle and another five represented the previously defined cellular states: AC-like; OPC-like; NPC-like; MES-like; and hypoxia (previously denoted as MES2)4 (Fig. Another MP (stress) included genes associated with heat shock and unfolded protein response, which were previously included in the MES-like MP and associated with disordered DNA methylation12. Although these seven MPs produce highly correlated cell scores as those from ref. 3b), it is important to note that the exact sets of genes that define the MPs here and in ref. 3c), suggesting platform-dependent effects such that distinct genes are highlighted by scRNA-seq versus snRNA-seq (Extended Data Fig. We also compared these MPs to prior GBM cell-state classifications, further corroborating their identities (Extended Data Fig. The first new MP contained NPC-like genes but was mostly associated with markers of more differentiated neurons (NRG1, NRG3, NRXN3, CNTNAP2) and synaptic activity (SYN3, SYT1; Fig. 2c) and was similar to the NEU (neuronal) pathway-based program reported previously15. Consistent with a differentiated NEU state, cycling cells were depleted in this program relative to NPC-like cells (Extended Data Fig. Comparison to the signatures of neural cell types in development and in adult human brains showed the highest similarity to L2–L3 excitatory neurons (P = 7.2−5, hypergeometric test; Fig. This state is depleted in single-cell relative to single-nucleus data4, suggesting that it may be poorly captured by enzymatic tumor dissociation, possibly because of complex cellular morphology. Comparison to normal brain signatures showed the strongest association with a recently described signature of glial progenitor cells (GPCs)36; hence, we denoted it as GPC-like. Like the NEU-like state, GPC-like cells were depleted in single-cell relative to single-nucleus data4. The third new MP mostly contained cilia genes, including 12 of 50 DNAH and CFAP family members; therefore, it was named cilia-like. This MP was specific to a small fraction of cells (~1.6%), underscoring the utility of our large cohort for identifying rare cellular states. Notably, all of those cells also expressed the AC-like MP, indicating that a subset of AC-like cells activate a cilia program. As expected, this MP was most enriched with external cilia signatures. Therefore, all MPs are enriched with corresponding cell-type-signatures from developing or adult human brains24,26,36 (Fig. 2d), supporting our annotations and suggesting that GBM states mirror developmental trajectories32. Collectively, our MPs are reflective of cell identity (for example, GPC-like, AC-like, NEU-like) or cell activity (for example, response to hypoxia or stress). We also scored malignant cells for signatures of functional attributes5,37,38, which suggested that AC-like and MES-like cells had greater connectivity, while OPC-like, NPC-like and NEU-like cells had greater motility and invasion scores, while cilia-like and GPC-like cells were intermediate (Extended Data Fig. The new MPs defined from 10x data were validated in Smart-seq2; they were still detected when analyzing only primary samples and were detected independently in specimens profiled by different labs and obtained from different medical centers (Extended Data Fig. Overall, our analysis expands our original malignant GBM MPs to ten, some of which represent more granular subsets of the states in ref. As an orthogonal description of cellular states, we defined pathway-based MPs (PMPs) (Fig. We previously defined four PMPs, namely NEU, glycolytic/pluri-metabolic (GPM), mitochondrial (MTC) and proliferative/progenitor (PPR)15,39. To preserve a similar computational approach to the gene-expression-based MPs, we adapted the NMF-based algorithm to consider pathways instead of genes, including 3,015 pathway gene sets from the Molecular Signatures Database Hallmark, Gene Ontology (GO) and Reactome. This resulted in seven PMPs that recapitulated and extended the previously defined PMPs (Extended Data Fig. We identified two distinct PMPs related to the previous GPM, one linked to glycolysis and hypoxia, and another closely resembling the MES-like state (denoted as GPM-MES). MTC-related and PPR-related PMPs also emerged from this analysis. We additionally identified two new PMPs, one similar to the AC-like and cilia-like MPs (AC-CIL), and another that is highly correlated with the GPC-like state, denoted as GPC/radial glia-like (RGL) (GPC-RGL). Most malignant cells in our dataset were classified into a single state, yet roughly 20% of the cells could be classified into multiple states, with a limited difference between the two highest state scores (for an example, see Fig. These hybrid cells did not have a high number of detected genes as would be expected from technical doublets (two nuclei occupying the same droplet; Extended Data Fig. Moreover, hybrid cells were not distributed as expected across all state pairs but rather were highly enriched for certain state pairs (Extended Data Fig. Thus, we reasoned that these may represent cells in transition between two states, and constructed a GBM model where each node represents a cellular state and each edge reflects a high-frequency hybrid state (Fig. Specifically, the GPC-like state was positioned at the center of three trajectories (NEU, AC-cilia, hypoxia-MES), perhaps suggesting that this population represents an early progenitor state with differentiation potential along multiple trajectories. Consistent with this possibility, cells having high GPC-like scores tended to exhibit intermediate expression levels of programs for the other cellular states (Fig. When assessing differences between samples, a dominant signal reflects variations in the abundance of shared cellular states. To highlight additional patterns of intertumor differences, we developed a cell-state-controlled approach in which we compared tumors only based on cells in corresponding states. First, each tumor sample was decomposed into subpopulations of cells representing seven common malignant states. Then, for each state, the pseudo-bulk profiles were compared across tumors, thereby eliminating state-specific signals and highlighting tumor-specific differences in that state. Principal component analysis (PCA) was applied to the pseudo-bulk profiles of each cell state, defining high and low signatures for the top three principal components (Fig. a, Schematic of state-controlled analysis to define the BPs. Intertumor variability reflects the average variability measured between scores of different samples (defined as the average s.d. of program scores in each state across all states). Intratumor variability reflects the average variability measured between scores of same samples (defined as the average s.d. of program scores in each sample across all samples). Intertumor variability was significantly larger in BPs relative to MPs (mean score = 1.05 versus 0.65, P = 0.0005, one-sided t-test) whereas intratumor variability was significantly higher in MPs relative to BPs (mean score = 0.76 versus 0.62, P = 0.028, one-sided t-test). P values were not corrected for multiple testing. c, Intra-sample variability in BP scores across state-specific pseudo-bulk profiles. Each dot reflects a state-specific pseudo-bulk profile, is colored according to state and aggregated horizontally according to sample. The black horizontal lines connect pseudo-bulk profiles from the same sample as a visualization aid and have no other meaning. The black dots reflect the mean BP score of each sample and are connected using a line for visualization purposes. d, Top, two-dimensional representation of the BPs. Bottom, limited intratumor variability in BP scores. Cells are colored according to the maximal BP score. Cellular states that reflects each quadrant are indicated on the right. For example, signatures of variation across AC-like pseudo-bulk profiles were highly similar to signatures of variation across NPC-like pseudo-bulk profiles. We denoted their consensus signatures as state-controlled baseline profiles (BPs), that is, consistent malignant gene expression differences between tumors that are not accounted by differences in tumor cell-state distribution. We annotated BPs based on functional enrichment as NEU, glial and extracellular matrix (ECM) (Extended Data Fig. While BP annotations are reminiscent of cell-state MPs (that is, BP-NEU and NEU-like MP; BP-ECM and MES-like MP), we note that BPs and cell-state MPs reflected distinct sets of genes, such that MPs preferentially varied within tumors while BPs preferentially varied between tumors (Fig. 6c) and remain consistent within a tumor (P = 2.3 × 10−9, Wilcoxon rank-sum test; Extended Data Fig. Accordingly, when we assigned tumors to BPs by averaging across the respective pseudo-bulks (Fig. 6e), we found that this classification tended to be consistent across different cellular states in the tumor (Fig. In a tumor with high ECM BP, cells existed in multiple states but most tended to express high ECM, while in a tumor with high NEU BP most cells tended to express high NEU BP, even those in a MES-like state. To examine whether BPs are consistent across regions of the same tumor, we leveraged a GBM spatial transcriptomics dataset that includes multi-sampled tumors37. Sample pairs from the same tumor had significantly higher similarity in BP scores than sample pairs from different tumors (Extended Data Fig. 7c), supporting the consistency of BPs within tumors. Finally, we leveraged a subset of samples with both bulk RNA-seq and snRNA-seq data to compare the newly defined BPs to The Cancer Genome Atlas (TCGA) expression subtypes17,40. Taken together, the three BPs reflect core intertumor transcriptional variation (that is, tumor-level phenotypes) that are partially independent of the variation in cellular states within tumors. We next investigated how TME heterogeneity may influence malignant BPs and cell-state abundance. To accomplish this, we applied our NMF framework to derive TME MPs for each nonmalignant cell type (n = 62 total MPs; Fig. We annotated TME MPs based on pathway enrichment and by scoring cells for published signatures (Extended Data Fig. TME MPs were linked to diverse functions, for example, reflecting cortical layers (among excitatory neurons), immune phenotypes (microglia versus bone marrow-derived macrophages) and differentiation status (immature versus mature oligodendrocytes). A subset of TME MPs had high overlap with malignant cell MPs (for example, hypoxia and MES-like), suggesting a similar effect of microenvironmental niches on multiple cell types30 (Extended Data Fig. a, Donut chart representing the number of cells per cell type analyzed to identify robust NMF MPs, labeled with the number of cells and MPs found in each cell type. b, Selected TME NMF state composition of tumors assigned to each BPs. Each node represents malignant cell-state (red) or TME cell type nodes (colored according to cell type as presented). d, Correlation heatmap for the Pearson correlation between the relative malignant cell-state proportions (rows) and relative TME cell-state abundance (columns), which show representative associations for the hypoxic environment. Each green circled node represents a malignant cell state versus nonmalignant cell type pair, while each squared node represents a ligand–receptor interaction; there is an edge if that cross-talk is present. Open ‘(' and closed ‘)' parentheses reflect multiple ligand or receptor interaction from the same family. Specifically, closed parentheses ‘)' indicated multiple ligands interacting with the same receptor, and viceversa for closed ‘)' parentheses, multiple receptors interacting with the same ligand. TME MP abundances differed significantly across GBMs mapping to the three BPs, suggesting an association between TME and BPs (Fig. Abundance of NEU-like malignant cells was associated with neurons, suggesting that nonmalignant neurons may interact with malignant cells and shape their states. Abundance of GPC-like cells was positively correlated with TAM-microglia; abundance of OPC-like malignant cells correlated with nonmalignant OPCs. Collectively, these results indicate that malignant state abundance is associated with diverse TME cell types. Clustering of all states revealed two major TME-malignant compartments that probably reflect tumor periphery and core regions41 (Extended Data Fig. Thus, beyond shaping glioma cell identity3,4, hypoxia may cultivate an immunosuppressive ecosystem and induce astrocyte reactivity. This resulted in two separate subnetworks, supporting the division into tumor core regions (Fig. Taken together, GBM microenvironmental niches such as hypoxia, glial and NEU TME may serve as determinants of malignant and nonmalignant states. Malignant gene expression profiles may also be driven by genetic alterations. Previously, we found that the CNAs of CDK4, PDGFRA and EGFR and mutations in NF1 were associated with inferred abundance of cellular states in bulk GBM data from TCGA4. To examine these associations in our snRNA-seq dataset, we analyzed malignant state abundance in samples with matched DNA profiling and sufficient malignant cells (n = 108 samples). In contrast, we did not observe an association between EGFR amplifications and AC-like abundance. a, Association between genetic aberrations and malignant cell-state abundance. P value adjustments for multiple hypothesis testing were not performed. b, Cell type composition of each of the tumors (n = 121) in the dataset, aggregated based on their composition group. Tumors are aggregated according to the dominant malignant state (at least 30% of cells). d, Interaction graph between the different layers of GBM (composition groups, malignant state groups, BPs, gene-level CNAs and single-nucleotide mutations). Each node represents a layer; nodes are connected with an edge if a statistically significant association exists between the nodes. We next tested whether EGFR amplifications, or other genetic alterations, might be associated with the newly identified states (Fig. The strongest associations were between canonical genetic alterations of GBM (that is, chromosome 7 gain, chromosome 10p loss, EGFR amplification and TERT promoter mutations) and increased GPC-like state abundance (Fig. These canonical genetic alterations typically co-occur in GBM, so we compared samples with at least one of these alterations to those without. Tumors with canonical alterations had, on average, 12% GPC-like malignant cells, compared with less than 1% of those without (Wilcoxon rank-sum test, P = 0.003; Extended Data Fig. 10b), suggesting that the GPC-like state is rare in GBMs lacking canonical alterations. Other associations included TP53 mutations linked to NEU-like abundance and chromosome 7 amplifications linked to hypoxia abundance, which is consistent with a recent study28. Similar analyses of BPs found that glial BP is more closely aligned with canonical GBM genetic alterations compared to ECM or NEU BPs (Extended Data Fig. Together, these results indicate an interplay between genetic alterations and specific cellular states, although our dataset may still be underpowered to detect additional associations given the low frequencies of GBM genetic alterations. We defined three distinct layers of GBM transcriptional patterns: cell type composition; malignant cellular states; and BPs. To examine the associations across these layers, we first classified all samples according to each layer (Supplementary Table 5). Regarding cell type composition, tumors were divided into three main groups according to malignant fraction (high, intermediate and low); the latter group was further subdivided according to the dominant TME cell type (Fig. Finally, BPs were used to define three groups of tumors as described above (Fig. Next, we investigated associations across layers (Fig. Each of the three BPs was significantly associated with classifications according to cell type composition and malignant state, as well as specific genetic events. 6b and Supplementary Table 6), which is significantly more than expected by chance (P < 0.01 for each ecosystem according to a binomial test). The remaining samples could be described according to individual layers but did not correspond to a stereotypic association between these layers (Extended Data Fig. 10d), highlighting the complexity and diversity of GBM ecosystems. a, Layer map of the 54 samples that could be classified to one of the multilayer groups. b, Model for stereotype architecture in GBM, with exemplar tumors for each ecosystem and their compositional, cell-state and BPs. Multiple intercorrelated transcriptomic layers make up the GBM ecosystems. After the first report of scRNA-seq for 430 cells derived from five GBM samples3, technical progress has dramatically expanded the scale of single-cell profiling. In this study, we leveraged a large-scale single-nucleus dataset (~430,000 nuclei, 121 samples) to comprehensively redefine the GBM cellular architecture. We discovered three new cellular states that were not captured in previous glioma studies, including a differentiated NEU state (NEU-like), a new progenitor state (GPC-like) and a cilia-related state. The capacity to identify these new states may be due to the high number of samples and cells profiled in this study. Yet this explanation is insufficient given that two of the new states are relatively common in our data (6.7% NEU-like and 14.2% GPC-like cells). Thus, we speculate that these states were depleted by single-cell isolation protocols that require enzymatic digestion and may poorly capture specific cell states (for example, those with complex morphology). Single-nucleus protocols isolate nuclei using tissue homogenization in cold buffer, thus capturing a wider variety of cell types23,47. Accordingly, the newly described malignant states may resemble normal cells with complex cellular processes (for example, neurons, ciliated cells). GBM cellular states resemble the programs observed in normal glial or NEU cell types. This finding is further supported by analysis of potential cellular transitions. We explored this question through quantification of hybrid states, based on the assumption that cells mapping to two states often reflect a transition. These putative transitions also mirror normal brain development, supporting the view that many aspects of GBM biology are inherited from the cell of origin. Previous work noted that much of the diversity between GBMs is tightly linked to diversity within GBMs, such that tumor subtypes can be partially explained by the abundance of recurring cellular states. However, not all intertumor variation may be explained in this way; to our knowledge, no previous studies attempted to define the residual signal of intertumor heterogeneity. In this study, we addressed this issue by analyzing intertumor heterogeneity only among cells of the same cellular state, thereby controlling for the diversity of states within tumors. We found that the main patterns that vary between tumors in one state also vary in other states, thus defining a state-independent pattern of intertumor variability that may be thought of as reflecting a tumor baseline profile. Notably, the three BPs we identified—NEU, ECM and glial—are functionally reminiscent of some of the malignant states that vary within tumors (NEU-like, MES-like and GPC-like), although they include distinct sets of genes. Moreover, these features are correlated, such that tumors with one BP also tend to be enriched with the related malignant state. We speculate that certain genetic and environmental features give rise to two related effects: a BP that influences all cells in the tumor and a higher fraction of cells with a certain expression program (that is, state). Thus, tumors with an NEU BP (represented by one set of NEU-related genes) also tend to have an abundance of cells that activate another set of genes that corresponds to the NEU-like MP. Similarly, tumors with an ECM BP also have an abundance of cells that activate genes of the MES-like MP. At bulk, the two phenomena of BP and frequency of certain MPs coalesce into the signal of GBM subtypes, while single-cell analysis allows us to dissect these distinct effects. What might be the signals that concomitantly activate BP and frequencies of MPs? Finally, glial malignant features are associated with high purity. Thus, we speculate that tumor genetics along with tumor location, which influences the tumor TME, define the propensity of each GBM toward a particular BP and enriched MP, culminating in three stereotypic ecosystems, each associated with multiple correlated components—tumor genetics, nonmalignant TME and two malignant features (BP and MP frequency). Nevertheless, the coupling between these components is only partial, such that around half of the tumors cannot be assigned to a stereotypical ecosystem but rather reflect unique combinations of features, highlighting the diversity and complexity of GBM. Frozen GBM specimens, diagnosed as ‘glioblastoma, IDH WT' according to the WHO 2021 classification, were collected at seven institutions, as outlined in the sections below. Collection and genomic profiling were approved by the institutional review board (IRB) of each institution; all participants provided written informed consent accordingly. Clinical information of the cohorts is summarized in Supplementary Table 1. Additional clinical characteristics related to the longitudinal analyses are shown in the companion study by Spitzer et al.18. For this cohort (n = 4), tumors were collected from MD Anderson Cancer Center. Collection was approved by the Institutional Review Board of MD Anderson Cancer Center under the protocol number 2012-0441. For this cohort (n = 12), tumors were collected from the Preston Robert Tisch Brain Tumor Center Biorepository, Duke University Hospital. Collection was approved by the IRB of Duke University under protocol no. For this cohort (n = 30), tumors were collected from the Department of Neurosurgery, Tokyo University Hospital. Collection was approval by the IRB of Tokyo University Hospital under protocol no. For this cohort (n = 28), tumors were collected from the Pitié-Salpêtrière Hospital and were approved by the Onconeurotek Brain Tumor bank of the hospital Pitié-Salpêtrière under certification no. For this cohort (n = 14), tumors were collected from the Division of Neurosurgery, St. Michael's Hospital, Unity Health Toronto. Collection was approved by the Research Ethics Board of St. Michael's Hospital, Unity Health Toronto, under protocol no. REB 13-141; all patients provided signed informed consent accordingly. For this cohort (n = 20), tumors were collected from the Department of Neurosurgery, Seoul National University Hospital, Seoul. For this cohort (n = 13), tumors were collected from Centre Hospitalier de Luxembourg (Neurosurgical Department). Collection was approved by the National Committee for Ethics in Research Luxembourg, under protocol no. Cohorts were added to IRB protocol Dana-Farber/Harvard Cancer Center 10-417. Clinical information of the cohorts is summarized in Supplementary Table 1. Additional clinical characteristics related to longitudinal analyses are shown in the companion paper by Spitzer et al.18. In protocol 1 (laboratory 1 and laboratory 2 for the cohorts from Duke University, MD Anderson Cancer Center, Tokyo University Hospital, St. Michael's Hospital and Pitié-Salpêtrière Hospital), nuclei from frozen tumor tissue were isolated. Single-nucleus suspensions were filtered using a 40-μm strainer, centrifuged at 500g for 5 min and resuspended in ST buffer supplemented with 0.01% BSA (cat. Nuclear suspensions were inspected by microscope, counted using a hemocytometer and used for 10x Genomics workflow or fluorescence-activated cell sorting (FACS) for the Smart-seq2 workflow. In protocol 2 (laboratory 3 for the cohorts from NORLUX and Seoul National (SNU) cohorts), tissue samples were thawed and mechanically dissociated in Nuclei EZ Lysis Buffer (NUC101, Sigma-Aldrich) using dounce homogenization. Solutions were incubated on ice for 5 min and mixed 1–2 times during incubation. Single-nucleus suspensions were filtered through a 70-μm strainer and centrifuged at 500g for 5 min at 4 °C, resuspended in Nuclei EZ Lysis Buffer and incubated on ice for 5 min. Solutions were centrifuged at 500g for 5 min at 4 °C and resuspended in 1% BSA, 0.2 U μl−1 RNase inhibitor and PBS buffer (three times). For the final resuspension, 4′,6-diamidino-2-phenylindole was added to the buffer; the solution was filtered through a 40-μm strainer, cells were counted on a Countess II Automated Cell Counter (Thermo Fisher Scientific) and nuclei were taken into the 10x Genomics workflow. The Chromium Single Cell 3′ Reagent Kit Vv3 (cat. PN1000128, 10x Genomics) was used according to the manufacturer's protocol. Briefly, nuclei were loaded on the Chromium Chip (cat. PN12000120, 10x Genomics) with a target cell recovery of 6,000–8,000 nuclei and processed in the Chromium Controller. Single nuclei were partitioned into Gel Beads-in-emulsion, followed by RNA reverse transcription with barcoding. For the NORLUX and SNU cohorts, nuclei were loaded on a Chromium Chip with a target cell recovery of 6,000 nuclei for single-cell multi-omic assay for transposase-accessible chromatin and gene expression according to the manufacturer's protocol. Single-nucleus sorting was performed on a FACSAria Fusion Sorter (Becton Dickinson) using a 640-nm laser (Ruby, 670/14 filter) (Extended Data Fig. After doublet discrimination, intact nuclei were selected with ruby+ and were sorted into 96-well plates containing TCL buffer (cat. Plates were frozen on dry ice immediately after sorting and stored at −80 °C before whole-transcriptome amplification, library preparation and sequencing. Whole-transcriptome amplification, library construction and sequencing were performed as previously published with slight modification for the nucleus as follows4,19. Briefly, on 96-well plates, RNA derived from single nuclei was first purified with Agencourt RNAClean XP beads (cat. EP0753, Thermo Fisher Scientific) and locked TSO oligonucleotide (5′-AAGCAGTGGTATCAACGCAGAGTACATrGrG+G-3′, QIAGEN). This was followed by PCR amplification (22 cycles for snRNA-seq) using the KAPA HiFi HotStart ReadyMix (cat. K2602, KAPA Biosystems) with subsequent Agencourt AMPure XP Bead (cat. Libraries were tagmented using the Nextera XT Library Prep Kit (cat. Libraries from 768 cells with unique barcodes were pooled and sequenced on a NextSeq 500 or NovaSeq 6000 sequencer (Illumina), with paired-end 38 bp reads. WES for the samples from Duke University, the MD Anderson Cancer Center, Tokyo University Hospital and St. Michael's Hospital was performed as follows. DNA was extracted from each frozen tumor sample and blood samples corresponding to the patients using the DNeasy Blood & Tissue Kit (cat. Genomic DNA (100–250 ng) was acoustically sheared using an ultrasonicator (Covaris), targeting 150-bp fragments. Library preparation was performed using the KAPA HyperPrep Kit (cat. KK8504, KAPA Biosystems) followed by enzymatic cleanup using AMPure XP beads. Exome capture was performed using a custom exome bait (manufactured by Twist Bioscience to Broad Institute's specifications). Captured libraries were sequenced with 150 bp paired-end sequencing on a NovaSeq 6000. For the samples from Pitié-Salpêtrière Hospital, after DNA was fragmented by a LE220 ultrasonicator (Covaris) and size-selected, library preparation and capture were performed using Twist Human Core Exome kit (Twist Bioscience) according to manufacturer's protocol. Newly generated whole-genome DNA sequencing data were collected for a cohort of frozen samples from the NORLUX Neuro-oncology Laboratory. Briefly, DNA was extracted from each tumor sample using the AllPrep DNA/RNA Mini Kit (cat. 80204, QIAGEN) for samples with sufficient tumor tissue and matched normal blood when it was available. Whole-genome libraries were prepared and sequenced with 150 bp paired-end sequencing on a NovaSeq 6000 platform. WGS data for the SNU cohort was prepared in an identical manner and was previously reported in ref. Data for the SNU cohort are available on Synapse (https://www.synapse.org/glass). No statistical method was used to predetermine sample size. All available samples were provided by the aforementioned centers and no data were excluded from the analyses. The statistical analysis described in this work was done using R v.4.0.1 and above. DNA sequencing alignment, fingerprinting, somatic variant detection (Mutect2) and copy number segmentation was performed in accordance with the Genome Analysis Toolkit best practices using the Genome Analysis Toolkit v.4.0.10.1, as described previously9,48. DNA fingerprint analysis using ‘CrosscheckFingerprints' (Picard) confirmed that all samples belonging to a patient came from the same individual, indicating that there were no sample mismatches in this study. Somatic mutations were detected using Mutect2 (v.4.1.0.0) for tumor samples with a matched normal blood sample. A panel of normals was constructed for each sequencing batch to account for differences in DNA library preparation (for example, whole-genome versus whole-exome) and used to filter out common artifactual and germline variants. Patient tumor samples without matched normal blood were analyzed for copy number alterations but not for SNV detection. For droplet-based sequencing data, the Cellranger v.3.1.0 pipeline provided by 10x Genomics was used for alignment (GRCh38 release 93) and to generate count matrices. We divided the TPM and CPM values by ten as the size of single-cell libraries is estimated to be in the order of 100,000 transcripts; therefore, we wanted to avoid inflating the expression levels by counting each transcript approximately ten times. After these normalization steps, initial filtering of low-quality cells was done based on the low number of detected genes (fewer than 200 genes) or the high expression of mitochondrially encoded genes (more than 3%). Most low-quality cells were removed downstream in case they could not be robustly classified as malignant or nonmalignant (see the later sections on the CNA analysis). Lastly, we computed the average expression of each gene i as \(\log 2[(\frac{1}{n}{\sum }_{j=1}^{n}{{\mathrm{TPM|CPM}}}_{i,\,j})+1]\) and filtered out lowly expressed genes by limiting the analyzed genes to the top 3,000 most highly expressed genes (using the Seurat v.4.0.4 (ref. For the cells and genes that passed these quality control filters, we defined relative expression levels by centering the expression levels for each gene across all cells in the dataset as follows: \({{Er}}_{i,\,j}={E}_{i,\,j}-\,\frac{1}{N}{\sum }_{k=1}^{N}{E}_{i,k}\) where N is the number of cells in the dataset. To facilitate clustering, cell type annotation and scoring of droplet-based data, we used the R package Seurat v.4.0.4 (ref. Initial classification of tumor-associated macrophages, T cells, oligodendrocytes and endothelial cells was based on the expression of known marker genes22,23,24,25,26, whereas the rest of the cells were annotated as presumed malignant. After classification of cells as malignant or nonmalignant (see the following two sections), cells classified as nonmalignant were scored (using the AddModuleScore function) for known cell-type-signatures and assigned a cell type using the method explained in the section ‘Assignment of cells to states'. Finally, heterotypic doublets were filtered our using the R package scDblFinder50. 3, which is available at https://github.com/jlaffy/infercna) with default parameters. Briefly, the algorithm sorts the analyzed genes in each sample according to their chromosomal location and applies a moving average with a sliding window of 100 genes in each chromosome to the relative expression values. In cells classified using single-nucleus droplet-based sequencing, we generally encountered a lower CNA signal and continuous signal distributions related to (1) lower data quality relative to that observed in single-cell data, and (2) inclusion of nonmalignant cell types, such as astrocytes and neurons, in the presumed malignant group as these cell types could not be classified a priori as nonmalignant because of transcriptional similarity with malignant cells. Therefore, we used a multi-step classification scheme for the droplet-based, single-nucleus sequencing data. For each cell j, we computed the CNA signal on each chromosome separately as \({{\mathrm{CNA}}}_{j}^{C}=\frac{1}{n}{\sum }_{i=1}^{n}{{\mathrm{CNA}}}_{i,\,j}^{C}\) where n is the number of genes on chromosome C included in the analysis. This enabled assigning each cell with a P value for each chromosome using a two-sided z-test (function pnorm from the R package stats) reflecting whether the cell had a CNA signal that was significantly different than expected from nonmalignant cells. These P values were further corrected for multiple testing (for each chromosome) using the Benjamini–Hochberg method; cells with Benjamini–Hochberg-adjusted P < 0.05 for a particular chromosome were considered as having a copy number event on that chromosome (either amplification or deletion based on the sign of the CNA signal). Cells with an unadjusted P < 0.05 for a particular chromosome that failed to achieve statistical significance after adjustment were considered to have a ‘suspicious event' on that chromosome. Then, inferred copy number events were cross-referenced with copy number calls from the WES and WGS data; only events detected in both methods were considered for classification and were termed ‘real' events. For this subset of cases, we defined a panel of highly recurring CNA events (intersection of highly recurring events inferred from the snRNA-seq and WES and WGS data), which were used as a short-list of potential events. Cells classified a priori as nonmalignant were reclassified as ‘unresolved' in case a ‘real' or ‘suspicious' CNA event was detected and ‘nonmalignant' otherwise. Presumed malignant cells with at least one ‘real' CNA event were classified as ‘malignant'; those with a ‘suspicious' event were classified as ‘unresolved' and those with no events at all (‘real' or ‘suspicious') were classified as ‘nonmalignant'. Briefly, for each sample with available WES data, we called SNVs for each cell using the Vartrix tool (https://github.com/10XGenomics/vartrix) from the FASTQ files. Misclassified cells were defined as cells harboring a malignant mutation that were classified as nonmalignant in step 1 of the algorithm. After a cost-effectiveness analysis (misclassification rate versus percentage of excluded cells), we set 0.35 as the upper threshold for nonmalignancy. To refine the classification, we defined confidence levels. Confidence level 1 includes cells classified as malignant that harbored a malignant SNV, cells in which at least 50% of the expected CNA events were detected including chromosome 7 amplification or chromosome 10 deletion, and cells classified as nonmalignant with CNAcor < 0.25. Confidence level 2 includes cells classified as malignant in which at least 50% of the expected CNA events were detected that do not include chromosome 7 amplification or chromosome 10 deletion, cells classified as malignant in which less than 50% of the expected CNA events were detected, including chromosome 7 amplification or chromosome 10 deletion, and cells classified as nonmalignant with CNAcor < 0.5. NMF was performed on the relative expression values of each sample independently after setting negative values to zero. Since k varies between samples and is largely unknown, we ran the NMF algorithm on each sample using different values (3–10), thereby generating 52 programs for each sample. Each of these NMF programs was summarized using the top 50 genes based on the NMF coefficients. The method first filters out NMF programs that are not robust (do not recur in the sample they originated from or across several samples) or are redundant in the sample (that is, they highly overlap other NMF programs in that sample). Then, robust NMF programs are clustered according to Jaccard similarity using a customized clustering algorithm that iteratively considers the degree of overlap between robust NMF programs and combines highly overlapping ones to form a cluster. The R 50 most-recurring genes in each cluster define an MP. Overall, the algorithm yielded 16 MPs that were annotated based on functional enrichment analysis (using GO terms, Molecular Signatures Database Hallmark gene sets and gene sets derived from normal brain development datasets). MP16 included genes from several cell types resulting in an MP that fitted almost all cells and thereby did not reflect any heterogeneity; thus, it was excluded from the analysis. This same procedure was independently repeated for each nonmalignant cell type. To facilitate cell classification, we generated 20 shuffled expression matrices by sampling each time 5,000 cells and shuffling the expression values for each gene. We then scored each shuffled matrix for the NMF MPs, thereby yielding 100,000 normally distributed scores for each expression program. These served as null distributions for cell-state classification. We adjusted all P values for multiple testing using the Benjamini–Hochberg method. Each cell was classified into a specific state if the adjusted P value computed for that state was smaller than 0.05. Cells that achieved Padj < 0.05 for multiple states were assigned to the state for which they scored maximally. Cells that did not achieve Padj < 0.05 for any of the states were assigned an ‘unresolved' state. Cells were assigned a ‘cycling' state on top of their cellular state if they achieved Padj < 0.05 for the cell cycle MP and ‘non-cycling' otherwise. We define hybrids as cells scoring significantly (that is, with Padj < 0.05) for at least two cellular states with a difference between the top two significant scores smaller than expected by chance. The 95 quantiles of this score difference distribution (equal to a difference of 0.24) was set as the threshold for determining if a state was singular or hybrid. To estimate the expected frequency of technical hybrids, we first computed for each pair of states (A, B) the expected frequency of hybrids, defined as \({\mathrm{Exp}}\left({\mathrm{A,B}}\right)={\mathrm{Freq}}\left({\mathrm{A}}\right)\times {\mathrm{Freq}}\left({\mathrm{B}}\right)\) and the observed/expected ratio defined as \({\log }_{2}(\frac{{\mathrm{Obs}}({\mathrm{A,{B}}})}{{\mathrm{Exp}}({\mathrm{A,B}})})\). As this is an overestimation of the expected frequency attributed to technical effects, we estimated this technical factor (TF) by averaging across observed/expected ratios of hybrid pairs detected less than expected by chance (that is, observed/expected < 0). Finally, we filtered out insignificant hybrid pairs with large observed/expected ratio (attributed to small numbers) using a one-sided Fisher's test. We did not include the ‘cycling' state in this analysis because we did not view ‘cycling' as an independent state but rather as an additional feature that cells can have on top of their neurodevelopmental identity (that is, AC-like, OPC-like, and so on). Each tumor sample was decomposed to up to seven pseudo-bulk profiles (one for each of the common states—AC-like, MES-like, hypoxia, GPC-like, OPC-like, NPC-like, NEU-like—given at least 25 cells classified to that particular state) by averaging across the normalized unique molecular identifier counts and log2-transforming. Genes were included if their mean log2 expression value across all pseudo-bulks was at least 1 and if their median variance (computed separately within state) was at least 2.5. Then, pseudo-bulks were analyzed within state to remove the state-specific signal using PCA. We then derived six gene programs from each state (top and bottom 50 genes of the first three principal components), computed the Jaccard similarity index between each pair of programs and hierarchically clustered the similarity matrix using the ConsensusClusterPlus package (v.1.56.0). Then, we derived five consensus signatures by including genes that recurred in at least 25% of programs in each of the clusters (with a hard minimum of at least three times). Using manual inspection and GO enrichment, we annotated the consensus signatures. To measure the differences in tumor composition, we generated for each tumor sample a composition vector reflecting the proportion of each cell type in the tumor sample. Similarly, we defined the proportion vectors for the malignant cell states and assigned tumors to six groups according to the dominant cell state (at least 25% of cells assigned to the particular state)—AC, MES/hypoxia, GPC, OPC/NPC, NEU and mixed (in case of no dominant state). We then generated an undirected graph by defining each feature as a node and connecting the nodes with edges when a statistically significant association (P < 0.05, unadjusted for multiple testing) between the two features was observed. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Processed snRNA-seq data generated for this study are available at the Gene Expression Omnibus under accession no. All de-identified somatic mutation and copy number alteration calls, and genomic analysis tables, are available via Synapse (https://www.synapse.org/care_glioblastoma). Raw sequencing data are available with limitations in accordance with the consent forms from the Data Use Oversight System (DUOS) at https://duos.boardinstitute.org under the following IDs: DUOS-000475; DUOS-000476; DUOS-000477; DUOS-000478; DUOS-000479; and DUOS-000480. The analysis scripts used in this study are available via GitHub at https://github.com/dravishays/GBM-CARE-WT and via Zenodo at https://doi.org/10.5281/zenodo.14966015(ref. Scripts for processing the DNA sequencing data can be accessed at https://github.com/Kcjohnson/care-glass. Sottoriva, A. et al. Intratumor heterogeneity in human glioblastoma reflects cancer evolutionary dynamics. Patel, A. P. et al. Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma. Neftel, C. et al. An integrative model of cellular states, plasticity, and genetics for glioblastoma. Venkataramani, V. et al. Glioblastoma hijacks neuronal mechanisms for brain invasion. Meacham, C. E. & Morrison, S. J. Tumour heterogeneity and cancer cell plasticity. Hara, T. et al. Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma. Venkatesh, H. S. et al. Electrical and synaptic integration of glioma into neural circuits. Varn, F. S. et al. Glioma progression is shaped by genetic evolution and microenvironment interactions. Couturier, C. P. et al. Single-cell RNA-seq reveals that glioblastoma recapitulates a normal neurodevelopmental hierarchy. Filbin, M. G. et al. Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq. Johnson, K. C. et al. Single-cell multimodal glioma analyses identify epigenetic regulators of cellular plasticity and environmental stress response. Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Venteicher, A. S. et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Garofano, L. et al. Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities. Wang, Q. et al. Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment. Spitzer, A. et al. Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics. Picelli, S. et al. Full-length RNA-seq from single cells using Smart-seq2. Ceccarelli, M. et al. Molecular profiling reveals biologically discrete subsets and pathways of progression in diffuse glioma. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Campbell, J. N. et al. A molecular census of arcuate hypothalamus and median eminence cell types. Habib, N. et al. Div-Seq: single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Nowakowski, T. J. et al. Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex. Polioudakis, D. et al. A single-cell transcriptomic atlas of human neocortical development during mid-gestation. Velmeshev, D. et al. Single-cell genomics identifies cell type-specific molecular changes in autism. Single cell spatial analysis reveals the topology of immunomodulatory purinergic signaling in glioblastoma. Ravi, V. M. et al. Spatially resolved multi-omics deciphers bidirectional tumor–host interdependence in glioblastoma. Schmitt, M. J. et al. Phenotypic mapping of pathologic cross-talk between glioblastoma and innate immune cells by synthetic genetic tracing. Gavish, A. et al. Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours. Outer radial glia-like cancer stem cells contribute to heterogeneity of glioblastoma. Mathur, R. Glioblastoma evolution and heterogeneity from a 3D whole-tumor perspective. Gimple, R. C. et al. Glioma stem cell-specific superenhancer promotes polyunsaturated fatty-acid synthesis to support EGFR signaling. Mukasa, A. et al. Mutant EGFR is required for maintenance of glioma growth in vivo, and its ablation leads to escape from receptor dependence. Schönrock, A. et al. MEOX2 homeobox gene promotes growth of malignant gliomas. Greenwald, A. C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Hai, L. et al. A clinically applicable connectivity signature for glioblastoma includes the tumor network driver CHI3L1. Migliozzi, S. et al. Integrative multi-omics networks identify PKCδ and DNA-PK as master kinases of glioblastoma subtypes and guide targeted cancer therapy. Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1. Puchalski, R. B. et al. An anatomic transcriptional atlas of human glioblastoma. Inference and analysis of cell–cell communication using CellChat. The tumor microenvironment underlies acquired resistance to CSF-1R inhibition in gliomas. Gjørlund, M. D. et al. Neuroligin-1 induces neurite outgrowth through interaction with neurexin-1β and activation of fibroblast growth factor receptor-1. Müller, T. et al. Neuregulin 3 promotes excitatory synapse formation on hippocampal interneurons. Targeting neuronal activity-regulated neuroligin-3 dependency in high-grade glioma. Using single nuclei for RNA-seq to capture the transcriptome of postmortem neurons. Barthel, F. P. et al. Longitudinal molecular trajectories of diffuse glioma in adults. Germain, P.-L., Lun, A., Meixide, C. G., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Gaujoux, R. & Seoighe, C. A flexible R package for nonnegative matrix factorization. Spitzer, A., Johnson, K., Nomura, M. & Garofano, L. The multi-layered transcriptional architecture of glioblastoma ecosystems. We thank R. A. DePinho, D. Arons and K. Knight for support with the GBM CARE initiative. This work was supported by a GBM CARE grant (supplemented to no. is also supported by National Institutes of Health (NIH)-National Cancer Institute (NCI) grant nos. is supported by a European Research Council consolidator grant no. 101044318, the Zuckerman STEM Leadership Program and a Mexican Friends New Generation grant, and he is the incumbent of the Dr. Celia Zwillenberg-Fridman and Dr. Lutz Zwillenberg Career Development Chair. A. Lasorella is supported by NIH grant nos. A.S. is partially supported by the Israeli Council for Higher Education via the Weizmann Data Science Research Center. are supported by the Luxembourg National Research Fund and TRANSCAN3 grants (C20/BM/14646004/GLASSLUX, INTER/TRANSCAN22/17612718/PLASTIG). has received honoraria from Elsevier, BMJ Best Practice, Oakstone Publishing and Servier, as well as research support from Merck, Conquer Cancer (The ASCO Foundation), the Robert Wood Johnson Foundation and the NCI. We thank the PROACTIVE team at MD Anderson for their assistance in procuring patient material. These shared services are supported in part by The Jackson Laboratory Cancer Center (no. We thank SciStories for the figure schematics. These authors contributed equally: Masashi Nomura, Avishay Spitzer, Kevin C. Johnson, Luciano Garofano. These authors jointly supervised this work: Anna Lasorella, Roel G.W. Verhaak, Antonio Iavarone, Mario L. Suvà, Itay Tirosh. Department of Pathology and Krantz Family Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Masashi Nomura, Lillian Bussema, Simon Gritsch, L. Nicolas Gonzalez Castro & Mario L. Suvà Broad Institute of Harvard and MIT, Cambridge, MA, USA Masashi Nomura, Lillian Bussema, Simon Gritsch, L. Nicolas Gonzalez Castro & Mario L. Suvà Avishay Spitzer, Noam Galili Darnell, Alissa C. Greenwald & Itay Tirosh Luciano Garofano, Young Taek Oh, Fulvio D'Angelo, Simona Migliozzi, Anna Lasorella & Antonio Iavarone The Jackson Laboratory for Genomic Medicine, Farmington, CT, USA Institute for Systems Genomics, University of Connecticut, Storrs, CT, USA Department of Neurology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Department of Neurology, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA Center for Neuro-Oncology, Dana-Farber Cancer Institute, Harvard Medical School, Boston, MA, USA Anuja Lipsa, Frank Hertel, Anna Golebiewska & Simone P. Niclou Department of Neurosurgery, Cancer Research Institute, Hypoxia Ischemia Disease Institute, Seoul National University, Seoul, Republic of Korea Franck Bielle, Alice Laurenge, Alberto Picca & Marc Sanson AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neuropathology, Paris, France AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neuro-oncology, Paris, France AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neurosurgery, Paris, France AP-HP, Brain Tumor Bank Onconeurotek (ONT), Paris, France Department of Neurosurgery, The Preston Robert Tisch Brain Tumor Center, Duke University, Durham, NC, USA Department of Pathology, Brigham and Women's Hospital, Boston, MA, USA Department of Pathology, Dana-Farber Cancer Institute, Boston, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar wrote the manuscript, with feedback from all authors. Correspondence to Anna Lasorella, Roel G. W. Verhaak, Antonio Iavarone, Mario L. Suvà or Itay Tirosh. is equity holder, scientific cofounder and advisory board member of Immunitas Therapeutics. is a cofounder and equity holder of Boundless Bio. The other authors declare no competing interests. Nature Genetics thanks Richard Gilbertson, Wolfgang Wick and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Total counts (left), number of detected genes (middle), and mitochondrial expression proportion (right) per cell across the whole dataset. b, Flow diagram of the CNA calling algorithm used to classify cells as malignant or non-malignant. CNA signal reflects the extent of CNAs on each chromosome per cell. Rows represent samples, columns represent chromosomes, values reflect the average CNA signal across all cells classified as malignant in the tumor. Top panel shows the % of misclassified cells (that is cells harboring a malignant SNV that were classified as non-malignant in the 1st step) that remain for each CNA correlation threshold. Bottom panel shows the % discarded cells for each CNA correlation threshold. Dashed line marks the upper threshold for non-malignancy (0.35). Each dot represents a cluster and is colored by the dominant cell type in the cluster (n = 195, 11, 6, 6, 43, 5, 45, 6 for Malignant, Excitatory neuron, Endothelial, Astrocyte, Oligodendrocyte, Inhibitory neuron, TAM and Lymphocyte cell types respectively). c, Heatmaps representing the average gene expression level (log2 relative expression) of cell type marker genes within each TME cell type for the 10x dataset. First, ruby positive proportion (P1, top) was selected by 570 nm laser. Then, doublet discrimination was performed by strict forward scatter height (FSC-H) versus area (FSC-A) criteria (P2, bottom left). e, Heatmaps representing the average gene expression level (log2 relative expression) of cell type marker genes within each TME cell type for the Smart-seq2 dataset. Data generated at the two different labs was analyzed separately. f, Pie chart demonstrating the proportion (%) of each cell type in the Smart-seq2 dataset (two labs data combined, left), and 10x dataset considering only the same samples profiled also by Smart-seq2 (right). a, Jaccard similarity indices of the robust NMF programs (n = 835) based on their top 50 genes. Programs are ordered by clustering and grouped into meta-programs (Methods). MP10 (Stress1) and MP15 (Stress2) were coalesced into a single Stress state. MP1 (Ribosomal protein, RP), MP11 (Doublet), MP12 (Low quality, LQ) reflect low quality and are not considered a state. b, Correlation between cell state scores of meta-programs derived from this dataset (columns) and those derived by Neftel et al. (rows). Each cell was scored within tumor-of-origin for both sets of meta-program signatures. Pearson's correlation coefficient was computed across all cells for each pair of meta-programs. c, Jaccard similarity between meta-programs derived from this dataset (columns) and those derived by Neftel et al. (rows). d, Distribution of scores in this dataset (upper panel) and Neftel et al. (lower panel) dataset of cells that scored maximally for both corresponding states (for example to both AC-like signatures derived from this dataset and from Neftel et al.) across all corresponding states. Dashed vertical lines mark the mean score of each distribution. e, Association between the malignant MPs from our datasets (rows) and malignant signatures from other studies (columns). a, Percent cycling cells in each subpopulation across samples with at least 50 cells in any of the states (n = 84, 16, 51, 67, 60, 65, 61, 28, 70 for AC-like, Cilia-like, Hypoxia, MES-like, Stress, NEU-like, GPC-like, NPC-like and OPC-like states respectively). b, Pearson's correlation coefficient heatmap between GBM state signature scores defined in this dataset and published functional gene signature scores. The published signature name is followed by each study's lead author in parentheses. c, Pearson's correlation of the proportions (%) of cells classified in each MP per tumor in both 10x (rows) and Smart-seq2 (columns). d, Similarity of MPs derived using all samples (x-axis) and MPs derived using primary (untreated) samples. e, Proportion (%) of contributing samples to each MP (relative to the number of samples profiled by each lab). f, Proportion (%) of contributing samples to each MP (relative to the number of samples provided from each sample source). a, Jaccard similarity indices of the robust NMF programs based on their top 50 pathways. Programs are ordered by clustering and grouped into Pathway-based metaprograms (PMPs) according to their functional activities. b, Clustering of the PMPs based on their pairwise Pearson correlations reveal functional activity groups. c, Scores of each cell for the different PMPs. Cells are aggregated by the PMP to which they were classified. “Unresolved” represents cells that could not be classified to a PMP. d, Average number of detected genes per sample (n = 121) for singular and hybrid cells. Two-sided t-test was used to test for statistical significance. e, Relative frequency (in log2 scale) of observed hybrids vs. expected technical hybrids (Methods) for each of the possible state pairs (order within a pair is meaningless). Dashed line marks log2(Fold Change) = 2 to aid in visualization. Statistical significance was assessed for each hybrid state pair using a one-sided Fisher's exact test. P values were corrected for multiple testing using the Benjamini-Hochberg method. a, Jaccard similarity matrix of the PCA-derived gene programs colored by cluster (see Methods). b, GO enrichment results for C1 (BP-ECM), C3 (BP-Neuronal) and C5 (BP-Glial) program clusters. Node color represents the extent of statistical significance and size represents the number of overlapping genes. Edges connect terms with a Jaccard similarity above 0.2 c, Same as Fig. 3c but here shown are the three BP scores. d, Observed vs. expected within-sample standard deviation (SD) of BP scores. Observed SD was defined as the SD across state-specific pseudo-bulk profiles in each tumor, shown are n = 98 samples with at least 3 pseudo-bulk profiles. Expected SD was estimated by randomly sampling n = 100 times 3 pseudo-bulk profiles and computing the SD across these PBs. Statistical significance was assessed with a two-sided Wilcoxon rank sum test. Each box plot represents a specific group (that is observed or expected) that spans from the first to third quartiles, median values are indicated by a horizontal line, whiskers show 1.5× interquartile range, and each dot reflects a specific standard deviation value. b, Pearson's correlation between scores of de-novo spatial BPs (rows) and those derive from the snRNAseq data (columns) with p-values (unadjusted for multiple testing) for each comparison in the tile. Each point represents a Euclidian distance between BP scores of two samples. Statistical significance was estimated using a two-sided t-test. d, TCGA subtype and the association with BP assignment. The samples profiled by both bulk RNA-seq and snRNA-seq were used (n = 28). a-b, Gene ontology term (rows) enrichment (log10 of adjusted p-values) for each metaprogram's 50 gene list (columns) against background for a, TAMs and b, astrocytes. c-d, Pairwise Pearson correlation heatmap for all c, CARE TAMs and d, astrocytes scored for both this dataset's MPs as well as publicly available cell type-specific gene signatures. e, Jaccard similarity index representing the intersection of genes for both malignant and non-malignant (astrocytes and TAMs) metaprograms. a, Correlation heatmap for the Pearson correlation between the relative TME cell state abundances (n = 121 samples). Only correlations with unadjusted P < 0.05 are shown and increasing size of circle reflects increasing absolute value of correlation coefficient. a, Association between genetic aberrations and cell state abundances. Color intensity reflects the statistical significance from two-sided Wilcoxon rank sum tests comparing state abundances in tumors with and without each genetic alteration (log10 of adjusted p-values). b, Association between abundance of GPC-like cells and the representative genomic alterations in GBM (either of chr.7 gain, 10p loss, EGFR amplification, and TERT promoter mutations, n = 87, P = 0.003, two-sided Wilcoxon rank sum test). c, The difference of the BP scores between samples with and without the representative genomic alterations in GBM (either of chr.7 gain, 10p loss, EGFR amplification, and TERT promoter mutations). Two-sided Wilcoxon rank sum test (n = 95 samples with sufficient mutation, copy number, and malignant cell data) were not corrected for multiple hypothesis testing. Boxplots span from the first to third quartiles, median values are indicated by a horizontal line, whiskers show 1.5× interquartile range, and outlier points are not shown. 6a but for samples that could not be classified to an ecosystem. Sample, sequencing and clinical metadata; Supplementary Table 2. MP gene lists for nonmalignant cells; Supplementary Table 5. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. The multilayered transcriptional architecture of glioblastoma ecosystems. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02168-4'>Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 09:58:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM's diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them. Despite significant advances in basic and translational research, isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM)1 remains incurable. The standard treatment for newly diagnosed GBM consists of maximal surgical resection followed by radiotherapy and concurrent treatment with the alkylating agent temozolomide (TMZ). This treatment often elicits a partial response, particularly when the promoter of the O6-methylguanine-DNA methyltransferase (MGMT) gene is methylated, as this gene encodes a DNA-repair protein that can remove alkyl groups added to O6-methylguanine by TMZ. However, the disease invariably recurs with a median overall survival of 14.6 months2,3. The Glioma Longitudinal Analysis (GLASS) consortium reported high similarity in genetic driver alterations between primary and recurrent GBM, indicating that standard treatment does not exert substantial clonal selection7. Conversely, transcriptome and proteome studies have demonstrated more dynamic evolutionary patterns, such as a proneural-to-mesenchymal (MES) shift, enhanced neuronal signaling and upregulation of extracellular matrix (ECM)-related genes at recurrence5,6,8,9,10,11,12,13,14. However, published longitudinal GBM studies have typically relied on bulk tissue profiling (averaging the signal across the malignant and tumor microenvironment (TME) compartments) and lack the granularity to independently interrogate the evolution of malignant and TME cell types and cell states. Single-cell longitudinal studies have had limited cohort size and limited clinical metadata, precluding a direct analysis of single-cell profiles with clinical correlates14. These limitations, along with the challenge of reconciling results from different groups that leverage diverse analytical and classification schemes, underscore the need for a larger single-cell longitudinal GBM dataset with detailed clinical and genomic annotations and with robust analysis. We established the GBM Cellular Analysis of Resistance and Evolution (CARE) consortium to characterize the longitudinal evolutionary trajectories in GBM at single-cell resolution. We performed single-nuclei RNA sequencing (snRNA-seq) of paired primary and recurrent GBM samples from 59 patients with detailed clinical annotations and genetic profiles. 15, we conducted an analysis of the data independent of sampling time (primary versus recurrent) that characterized GBM transcriptional ecosystems across multiple layers. We show that a lower malignant cell fraction and a reciprocal increase in proportions of glial and neuronal (that is, glio-neural) TME cell types is the most consistent longitudinal change at recurrence, observed in 66% of cases. Contrary to prior studies, we observed no consistent malignant GBM state enrichment at recurrence across the cohort. However, specific trajectories were associated with defined subsets of cases, such as GBMs with MGMT methylation or radiation-related small deletion phenotype. Notably, changes in malignant state abundance were mirrored by corresponding changes in baseline profiles and TME composition, reflecting interdependent alterations of the GBM ecosystem. Our study highlights the diverse longitudinal GBM trajectories that are shaped by treatment response and TME interactions. We collected matched snap-frozen GBM specimens from primary and recurrent surgeries (Fig. Tumor samples were obtained from seven hospitals, resulting in a total of 121 tumors from 59 patients profiled by snRNA-seq and whole-exome sequencing (WES) or whole-genome sequencing (WGS; Fig. For the majority of patients, we profiled samples from two time points (labeled T1 and T2); for three patients, we profiled samples from three time points (T1, T2 and T3). Most matched sample pairs were derived from primary and first recurrent tumors, except for three pairs that consisted of two recurrent samples and four pairs that consisted of a primary and a second recurrence. a, Schematic representation describing the workflow of our study. b, Clinical and technical information on all 59 patients in this dataset, including surgical interval, age at diagnosis, gender, location at recurrence (rec), MGMT promoter status, therapy and omic data type. c, UMAP for dimension reduction based on gene expression values of the cells in our cohort colored according to the cell type (top) and time point (bottom). d, Proportion (%) of patients in the dataset in which a genetic aberration was detected, colored by time point. De novo hypermutation (HM) indicates a high mutation burden at both time points without evidence for treatment-associated mutational signatures. UMAP, uniform manifold approximation and projection; chr, chromosome; assoc., associated. Tumor location was known for 51 patients, of whom 45 cases locally recurred at the surgical margin and six cases recurred at a site distant from the resection cavity. The improved clinical outcomes in our cohort (median interval between two surgeries = 12.4 months, median overall survival = 24 months), compared with historical cohorts, likely reflect better performance status of patients eligible for multiple surgical resections, consistent with a younger age at diagnosis (median = 53 years, range = 19–73 years). In our 10x Genomics snRNA-seq dataset, 429,305 cells passed quality filters (Methods). Nonmalignant cells clustered together across time points (Fig. 1c, bottom), and we did not observe a significant difference in the number of cells profiled or the number of genes detected per cell between time points (Extended Data Fig. Mutational and CNA profiles were derived from bulk DNA-sequencing data (Fig. The frequency of GBM driver alterations was consistent with prior studies7 and remained largely stable across time points (Fig. However, 26% of cases (14 of 54 patients) acquired treatment-related DNA hypermutation20 or an increase in small deletion burden21 at recurrence (Fig. Global (nonmatched) comparison of cell-type abundance showed overall similarity between time points, except for significantly lower abundance of malignant cells and higher abundance of glio-neural TME (oligodendrocytes, neurons and astrocytes (ACs)) in recurrent tumors (Fig. We further separated the broad nonmalignant cell types (for example, myeloid) into more granular states (for example, microglia and bone marrow-derived macrophages) using non-negative matrix factorization (NMF; Methods). Analysis of these higher-resolution cell states demonstrated that recurrences were enriched with reactive ACs and with oligodendrocytes expressing an interferon program (Extended Data Fig. To further evaluate differences in cell-type composition, we categorized tumors into six compositional groups based on the relative abundance of malignant and TME cell types (Methods). The latter group was further subdivided by the dominant TME cell type (for example, LMF—tumor-associated macrophages (TAM), oligodendrocyte (OC) and glio-neural (GN); Extended Data Fig. The HMF group was enriched in primary tumors, while the LMF with glio-neural (LMF—GN) or oligodendrocytic TME (LMF—OC) were enriched at recurrence (P = 0.005 for both groups, two-sided Fisher's test; Fig. a, Proportion (%) of each cell type across the two time points (primary and recurrence, n = 118) in which a statistically significant difference was observed. P values were computed with a two-sided Wilcoxon rank-sum test. b, Group-differentiating features and breakdown across primary/recurrence status for composition (left), malignant state frequency (middle) and baseline profiles (right). P values were computed using Fisher's exact test. d, Heatmap showing the Pearson correlation between transcriptomic features. To validate our findings, we re-analyzed an independent longitudinal GBM snRNA-seq cohort14 (Extended Data Fig. Consistent with our cohort, we observed lower malignant cell abundance, higher oligodendrocyte abundance and similar enrichment of LMF compositional groups at recurrence (P = 0.04, two-sided Fisher's exact test; Extended Data Fig. Survival analysis stratified by compositional groups showed significant differences in survival time after the second surgery (Extended Data Fig. Patients with glio-neural TME at recurrence had significantly longer survival time after second surgery than patients with other recurrences (median = 19 versus 8 months, P = 0.012, log-rank test; Fig. These differences in clinical course were independent of other clinical characteristics, including MGMT methylation status (Extended Data Fig. Next, we investigated changes in malignant cell state abundance across time points. As described in more detail in the companion paper15, we applied NMF to malignant cells and identified ten GBM metaprograms (MPs) representing intratumoral cellular heterogeneity (Methods)22,23,24. This included the cell cycle and the previously identified AC-like, MES-like, stress, hypoxia, neural progenitor cell-like (NPC-like) and oligodendrocyte progenitor cell-like (OPC-like) states. In addition, we identified the following three previously undescribed states: a differentiated neuronal state (NEU-like), a glial progenitor-like state (GPC-like) and a cilia-related state (Cilia-like), characterized by marker genes, pathway enrichments and similarity with normal neural cell types. Beyond cell states, we defined three baseline gene expression profiles (BP), reflecting intertumor heterogeneity that is not driven by differences in malignant cell state abundance. The three BPs were annotated as glial-BP, ECM-BP and neuronal-BP. Comparison of malignant cell state abundance between primary and recurrent samples did not reveal significant differences both in our cohort and in the cohort of ref. Moreover, we did not detect significant differences across time points when considering a division of tumors into six groups based on dominant malignant states or by BPs (Fig. However, survival analysis stratified by dominant malignant cell states showed significantly worse prognosis in the MES-like/hypoxia group at recurrence (median survival time from second surgery = 5 versus 10.3 months, P = 0.00086, log-rank test; Fig. The multiple transcriptomic layers of cell-type composition, malignant cell states and malignant BPs are significantly associated with each other and define three stereotypical GBM ecosystems (that is, glial, neuronal and ECM)15. We next investigated whether interactions between components of the GBM ecosystems are maintained between primary and recurrent tumors. We separately computed for each time point the correlations between (1) proportions of 11 TME cell types, (2) proportions of ten malignant cell states and (3) three BP scores. In both time points, we found three comparable clusters of correlated features (Fig. These conserved associations (P < 0.01 by permutation test) remain largely stable. Nevertheless, we did observe some differences, such as preferential association of the neuronal-high ecosystem with OPC-like malignant cells (in primary samples) and with NEU-like malignant cells (in recurrent samples; Fig. Interestingly, ligand–receptor analysis with CellChat25 showed some differential enrichment of predicted interactions across time points. For example, neuroligin (NLGN) and neuregulin (NRG) gene families—expressed by oligodendrocytes and neurons—are predicted to interact with the neurexin (NRXN) gene family or ERBB4 on NEU-like and NPC-like malignant cell states, preferentially at recurrence (Extended Data Fig. Taken together, global analysis of our cohort (not matched per sample pairs) shows differences in cell-type composition that are consistent with decreased malignant cell fraction and further integration of recurrent GBM cells with the brain parenchyma9,29,30, but does not reveal substantial global differences either in malignant cellular states or in BPs. Despite global conservation of cell types and malignant cellular states, when examining individual matched sample pairs, we found extensive divergence (Fig. While no group was significantly enriched, there was a trend toward increased neuronal-BP and reduced glial-BP at recurrence (Fig. For example, patient P1 had an increase of AC-like and decrease of GPC-like malignant cells at recurrence, while patient P2 had an opposite trajectory (Extended Data Fig. Furthermore, patients profiled at three time points showed opposite trajectories between two treatment intervals (T1–T2 and T2–T3), despite recurrences occurring at the same site (Fig. Finally, the frequency of conservation was not significantly different between local and distant recurrences (P = 0.66, two-sided Fisher's test). a, Sankey plots depicting the transitions between time points across the composition (left), malignant states (middle) and baseline profile groups (right). b, Longitudinal transcriptomic changes in the two pairs profiled at three time points highlight unpredictable gene expression trajectories. Detailed clinical information including treatment regimen and tumor MRI (top), heatmap representing proportions and scores from three transcriptomic layers (middle) and mutation fraction between time points are shown (bottom). c, Observed (colored bar) versus expected (black dot) conservation across time points. Error bars represent confidence intervals derived from the binomial test. This divergence of individual sample pairs prompted us to compare the observed patterns of conservation to those expected by a null model in which matched primary–recurrent pairs are as similar to one another as pairs of randomly selected samples. This analysis revealed that, across all three layers, matched pairs were significantly more conserved than expected by chance (P < 0.004 for each layer, binomial test; Fig. Thus, matched samples usually diverge in their transcriptomic classification and may diverge in a wide range of trajectories between and within patients but remain significantly more likely to retain any specific classification compared to a random pair of samples. Previous studies have shown that patients with methylated MGMT promoter exhibit better response to TMZ and have more favorable prognosis2,31, but whether this is mirrored by differences in either cellular architecture or in cellular trajectories following treatment is unknown. As expected, MGMT-MET patients exhibited longer surgical intervals than MGMT-UM patients, both in our cohort and in the GLASS cohort (Extended Data Fig. We next assessed whether MGMT promoter methylation status correlated with MGMT expression in single cells, as this association has been challenging to establish in bulk analyses32. Malignant cell expression of MGMT was significantly lower in primary MGMT-MET samples than in primary MGMT-UM samples (P = 5.3 × 10−6, two-sided Wilcoxon rank-sum test; Fig. 4a)32,33, but this effect was not observed for TME cell types (Fig. In a subset of MGMT-MET cases, MGMT expression increased at recurrence, possibly reflecting a competitive disadvantage of cells with low MGMT expression during treatment. a, Average MGMT expression in malignant cells per sample across time points. Samples were included only if the MGMT promoter status was verified using a methylation assay and were annotated as methylated/unmethylated according to the status of the primary sample (n = 12 methylated and n = 19 unmethylated). Statistical significance was estimated using a two-sided Wilcoxon rank-sum test; unadjusted P values are shown. b, MGMT expression level in each cell type between methylated and unmethylated samples. Statistical significance was estimated using a two-sided Wilcoxon rank-sum test; unadjusted P values are shown. c, Survival curve for the patients included in this analysis stratified according to MGMT expression level. X axis reflects the time from first to second resection. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. d, Survival curves for MGMT-MET patients stratified according to MGMT expression level. X axis reflects the time from first to second resection. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. X axis reflects the time from first to second resection. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. Statistical significance was estimated using a two-sided Wilcoxon rank-sum test; unadjusted P values are shown. g, MGMT-specific signatures plotted for patients included in the analysis. Value represents the T2–T1 difference (log2(FC) of gene expression) for each patient. Patients are ordered according to the difference between signature scores. Genes are ordered according to their correlation (across patients) with the difference between signature scores. When applying this model, we found that classification by MGMT expression levels was associated with surgical resection interval (Fig. 4c) and, in fact, led to improved association compared to classification by MGMT promoter methylation alone—within the MGMT-MET group, we observed significantly longer surgical interval for the MGMT-low expression group relative to the MGMT-high/MGMT-intermediate expression groups (Fig. 4d); in contrast, within the MGMT-high expression group, there was no significant difference in surgical interval when further stratifying by MGMT promoter methylation status (Fig. This improved performance of MGMT expression level could be due to exclusion of nonmalignant cells that may affect MGMT promoter methylation assessment; alternatively, it could indicate that MGMT expression level in the therapeutically targeted malignant cells more directly reflects MGMT protein activity and hence is more predictive of treatment response. Comparison of cellular architecture between MGMT-low and MGMT-high expression samples showed no significant differences in cell-type composition and malignant state distribution (Extended Data Fig. Specifically, MGMT-low tumors recurred with a lower abundance of MES-like states compared to their primary samples, while MGMT-high tumors displayed the opposite pattern. Baseline profiles changed in a similar manner—trending toward decreased ECM in MGMT-low, highlighting the coordinated evolution of cellular states and BPs (Extended Data Fig. To further identify longitudinal gene expression patterns specific to MGMT-high and MGMT-low samples, we compared their malignant pseudobulk expression (aggregated gene expression from all malignant cells per sample) across time points. This revealed four groups of genes—those consistently upregulated or downregulated in recurrence, regardless of MGMT expression (CONS-UP and CONS-DN), and those upregulated specifically in MGMT-low (low-UP) or MGMT-high samples (high-UP; Extended Data Fig. CONS-UP was enriched with genes involved in postsynaptic activity (for example, GRIN1/GRIN2A/GRIN2B, GABRA1/GABRA4), consistent with the trend toward increased neuronal-BP in recurrence (Fig. 3a), while CONS-DN was enriched with genes related to oxidative phosphorylation (for example, NDUFB4/NDUFB11, UQCRB/UQCRQ) and stemness (for example, CD24, DCX, SOX4/SOX11 and SOX11; Extended Data Fig. In contrast to those MGMT-independent gene sets, low-UP genes were enriched with genes linked to gliogenesis (for example, ACSL1, OLIG1/OLIG2 and SOX8), whereas high-UP genes were enriched with mesenchyme development and ECM remodeling (for example, COL1A1/COL1A2, COL1A2, DCN, FN1 and LOXL1; Fig. To validate our findings, we repeated this analysis in the ref. 14 dataset and found significant similarities, especially in the high-UP signature (Jaccard index 8.5% versus 1.2% expected, P < 0.0001, bootstrapping). In summary, our single-cell analysis revealed that MGMT expression in malignant cells predicts time to recurrence and influences longitudinal cellular trajectories—MES-like state and ECM-related genes tend to be enriched in recurrence in MGMT-high (likely nonresponsive) patients, while in MGMT-low (likely responsive) patients, recurrence is associated with loss of MES-like states and enrichment of genes related to gliogenesis. Our results suggest that prior attempts that failed at correlating MGMT expression with treatment response32 were likely confounded by bulk analysis and expression of MGMT by nonmalignant cells. Treatment may affect GBM genetic evolution, as supported by reports of somatic hypermutation following TMZ chemotherapy and/or small deletion phenotype following radiotherapy7,20,21,34. To analyze the longitudinal mutation patterns in the CARE cohort, we examined mutation burden and mutational signatures following therapy. We observed that mutation burden (mutations per megabase) generally increased at recurrence (P = 0.01, two-sided paired Wilcoxon; Extended Data Fig. Two tumor pairs displayed high mutation burden at both time points without evidence of treatment-associated mutation signature (COSMIC single base substitution (SBS) 11; Extended Data Fig. To investigate new mutational signature associations at recurrence, we examined the frequency of the COSMIC v3 SBS signatures35, both in our CARE cohort (n = 46 patients) and in the GLASS cohort (n = 138 patients). Notably, patients with increased mutations attributed to SBS21 did not tend to acquire mutations in mismatch repair genes. While mutational signatures temporally differed, we did not observe individual mutations enriched at recurrence (adjusted Fisher's exact test P > 0.05). Among small insertion/deletion variants, we observed increased burden of deletions (Fig. 5d), including ten recurrences that acquired the small deletion phenotype without treatment-associated hypermutation (Methods; Fig. a, Longitudinal difference in the relative proportion of mutations assigned to COSMIC v3 mutational signatures at each time point for select SBS signatures (SBS1, clock-like; SBS11, alkylating agent; and SBS21, defective DNA mismatch repair). Dashed lines connect values from the same patient at T1 and T2. P values represent two-sided paired Wilcoxon rank-sum test. b, Longitudinal difference in small deletion burden between T1 and T2 with dashed lines connecting patient values and red indicating patients who were assigned to the increased small deletion burden group. P values reflect two-sided paired Wilcoxon rank-sum test. Patients are represented by a single bar (x axis). Presence of increased small deletion burden or SBS21 signature is presented in black annotation bars below. The panels show the longitudinal change in the hypoxia signature for those patients whose tumor had a small deletion increase (left) and for those that did not have small deletion burden increase (right). P values from two-sided paired Wilcoxon rank-sum test. e, Scatterplot that represents the scaled genetic distance (bulk DNA—proportions of private mutations and differences in copy-number altered regions) versus scaled transcriptional distance (snRNA), with the linear regression line shown in blue. The Pearson's correlation coefficient and associated P value are shown (n = 38). ssGSEA, single-sample gene set enrichment analysis; diff., differences. To understand whether post-treatment genetic alterations modify cellular state evolution, we investigated the specific trajectories of tumors with hypermutation, SBS21, and small deletion phenotypes. For hypermutant tumors, we did not observe a consistent cellular phenotype at recurrence, although this analysis was limited by sample size (n = 4). In contrast, patients who acquired small deletions were enriched with hypoxia and MES-like cell states (Fig. We observed a similar effect for tumors with increased SBS21 signal, although this effect was weaker when excluding samples with co-occurring small deletion phenotype (Fig. The association between recurrence-specific small deletions and hypoxia state was recapitulated in analysis of bulk samples from GLASS9 (Fig. 6g), possibly due to similarities between MES-like and nonmalignant profiles that make it difficult to disentangle these signals in bulk datasets. Thus, malignant cells in hypoxic regions may be better able to repair radiotherapy-associated DNA damage, leading to an outgrowth of hypoxic state at recurrence. Finally, we examined the correlation between overall genetic distance, as defined by longitudinal single-nucleotide variant (SNV)/CNA differences, and overall malignant transcriptional distance (Methods). 5e) that was retained when analyzing only SNV or CNAs (Extended Data Fig. Intertumoral and intratumoral heterogeneity in GBM has been associated with treatment resistance and disease recurrence, but how specific subsets of GBM cells respond to therapy remains poorly understood. Here we leveraged a longitudinally matched multi-omic GBM snRNA-seq and bulk DNA dataset of 121 tumors from 59 patients consisting of ~430,000 nuclei to comprehensively characterize the compositional, transcriptional and genetic changes that GBMs undergo following standard-of-care. We found extensive variability in longitudinal trajectories of different patients (Fig. While many trajectories were unpredictable, we identified patient characteristics—MGMT methylation and increased small deletion burden—that were associated with particular trajectories and expanded our understanding of recurrences. (1) The most common observation was in the cell-type composition layer, where many samples exhibited a reduced malignant cell fraction at recurrence. (2) At the malignant state level, many tumors recurred with an unpredictable trajectory that likely reflects patient-specific evolution (top). A smaller subset of tumors recurred with more predictable trajectories, such as reduced MES-like fraction for MGMT-MET versus MGMT-UM primary tumors (middle) or increased hypoxia fraction for samples with increased small deletion burden following treatment (bottom). Columns separated into malignant and nonmalignant fractions indicate the relative proportions per tumor. Only selected nonmalignant and malignant cell states are shown for clarity. Previous studies have highlighted specific transcriptional trajectories during GBM evolution. The recent GLASS consortium study highlighted three trajectories toward neuronal, MES and proliferative profiles9, and other studies have highlighted proneural-to-MES phenotypic shifts11,14,38. While we similarly identified these recurrence-associated patterns, our analysis suggests that their overall frequency is limited and that such trajectories are enriched only in specific subsets of patients, while global comparison of cell states between primary and recurrent samples shows overall similarity. This discordance with prior studies may be attributable to at least the following three effects: (1) differences in technology (for example, bulk versus single-cell profiling), (2) cohort/sampling (for example, single-center sampling, tissue digestion versus not) and (3) analysis methods. There may also be biases due to additional treatments that were not addressed (bevacizumab, corticosteroids, etc. Our dataset contains tumor samples that were collected from seven medical centers across six countries, limiting the possibility of results being skewed toward a particular recurrence trajectory due to institution-specific procedures. Moreover, the samples in our dataset were processed and profiled by three different labs, limiting the possibility of skewing due to a lab-specific batch effect. The snRNA-seq protocol used by our labs to process and profile tumor samples was digestion-independent and optimized for brain tumors39, capturing a higher number of transcripts than prior single-nucleus GBM studies (Extended Data Fig. This likely results in a more faithful representation of the tumor tissue, as evident by the higher diversity of TME cell types in our dataset and the discovery of new malignant cell states. Neuronal pathways were consistently found to be upregulated in recurrent GBM8,9,12. However, it is important to note that this reflects only subsets of patients in each study and that we observe a wide diversity of trajectories from primary to matched recurrent samples. Leveraging the resolution and scale of our dataset, we showed that this phenomenon correlates with decreased malignant cell fraction and concomitant increased proportions of neurons in the TME; only in a small subset of cases, this is accompanied by the enrichment of NEU-like malignant state and the neuronal baseline expression profile of the tumors. These results suggest that, following treatment, integration of the residual GBM cells with the brain parenchyma is likely enhanced29,30. Beyond defined subsets, our study makes the challenging observation that, for the majority of GBMs, longitudinal cellular trajectories are diverse and unpredictable. We speculate that this diversity reflects at least the following three phenomena: first, given the genetic and epigenetic heterogeneity in primary tumors, it might be expected that tumors with distinct underlying biology will also have distinct trajectories of recurrence. Second, even within a single GBM, there is extensive spatial heterogeneity23,40; as primary and recurrent samples are obtained from different locations, their differences might reflect such spatial heterogeneity and, therefore, depend on sampling40. Third, patients vary considerably both in their treatment histories and in their response to treatments, potentially leading to distinct recurrence trajectories. Many GBM patients are treated by additional therapies in diverse combinations beyond standard-of-care, which is a limitation in this study. Moreover, even standard-of-care induces distinct patient responses, and we identified the following two cases in which treatment responses are linked to recurrence trajectory: (1) MGMT-MET (MGMT-low expression) samples (more likely to respond to alkylating agents) are associated with depletion of MES-like states in recurrence and (2) the small deletion phenotype, previously linked to radiotherapy21, is associated with increased hypoxia states in recurrence. In summary, we present a comprehensive analysis of GBM recurrence at single-cell resolution. Our analysis shows that, globally, recurrent GBMs are generally similar to primary GBMs in their cellular states, composition and other features of the GBM ecosystem. Accordingly, a wide range of recurrence trajectories are observed, highlighting extensive diversity in GBM evolution. Nevertheless, subsets of tumors with evidence for particular treatment responses are skewed toward specific transcriptional trajectories. These findings offer more detailed insights over prior observations and provide an opportunity to stratify recurrent patients more precisely, potentially leading to specific treatment strategies. Frozen GBM specimens that were diagnosed as ‘glioblastoma, IDH-wildtype' according to the World Health Organization 2021 classification were collected at seven institutions, listed below. Collection and genomic profiling were approved by the institutional review board of each institution, and all patients provided informed consent accordingly. Duke University cohort (n = 12)—the tumors were collected from the Preston Robert Tisch Brain Tumor Center Biorepository, Duke University Hospital. St. Michael's Hospital cohort (n = 14)—the tumors were collected from the Division of Neurosurgery, St. Michael's Hospital, Unity Health Toronto. It was approved by the Research Ethics Board of St. Michael's Hospital, Unity Health Toronto, under the protocol REB 13-141, and all patients provided signed informed consent accordingly. It was approved by the Institutional Review Board of SNU Hospital (approval H-2004-049-1116). Cohorts were added to the Institutional Review Board protocol Dana–Farber/Harvard Cancer Center 10-417. Clinical information of the cohorts is summarized in Supplementary Table 1. No statistical method was used to determine the sample size. Data were excluded from select analyses when the number of malignant cells within a sample was low in at least one time point. Throughout the figures in this manuscript (Figs. The statistical analysis described in this work was done using R version 4.0.1 and above. Protocol 1 (laboratory 1 and laboratory 2 for the cohorts from Duke University, MD Anderson Cancer Center, Tokyo University Hospital, St. Michael's Hospital and Pitié-Salpêtrière Hospital)—nuclei from frozen tumor tissue were isolated as follows: tissue was thawed and mechanically dissociated in salt-Tris buffer (10 mM Tris–HCl (pH 7.5), 146 mM NaCl, 1 mM CaCl2 and 21 mM MgCl2) with 0.49% CHAPS (3-((3-cholamidopropyl) dimethylammonio)-1-propanesulfonate) (Millipore, 28300). Single-nuclei suspensions were filtered using a 40 μm strainer, centrifuged at 500g for 5 min and resuspended in salt-Tris buffer supplemented with 0.01% BSA (NEB, B9000S). Nuclear suspensions were inspected by microscope, counted using a hemocytometer and used for 10x Genomics workflow. Protocol 2 (laboratory 3 for the cohorts from NORLUX and SNU cohorts)—tissue samples were thawed and mechanically dissociated in nuclei EZ lysis buffer (Millipore Sigma, NUC101) via Dounce homogenization. The solutions were incubated on ice for 5 min and mixed one to two times during incubation. Single-nuclei suspensions were filtered through a 70 μm strainer and centrifuged at 500g for 5 min at 4 °C, resuspended in nuclei EZ lysis buffer and incubated on ice for 5 min. In brief, nuclei were loaded on the Chromium Chip (10x Genomics, PN12000120) with a target cell recovery of 6,000–8,000 nuclei and processed in the Chromium Controller. Single nuclei were partitioned into gel beads-in-emulsion (GEMs), followed by RNA reverse transcription with barcoding. For NORLUX and SNU cohorts, nuclei were loaded on a Chromium chip with a target cell recovery of 6,000 nuclei for single-cell multiome ATAC and gene expression according to the manufacturer's protocol. WES for the samples from Duke University, MDACC, Tokyo University Hospital and St. Michael's Hospital was performed as follows: DNA was extracted from each frozen tumor sample and blood sample corresponding to the patients using the DNeasy Blood & Tissue kit (Qiagen, 69504). The genomic DNA (100–250 ng) was acoustically sheared by an ultrasonicator (Covaris), targeting 150 bp fragments. Library preparation was performed by KAPA HyperPrep Kit (KAPA Biosystems, KK8504) followed by the enzymatic clean-up using AMPure XP beads (Beckman Coulter, A63882). Captured libraries were sequenced with 150-base-pair paired-end sequencing on a NovaSeq 6000 (Illumina). Newly generated whole-genome DNA-sequencing data were collected for a cohort of frozen samples from the NORLUX Neuro-oncology Laboratory. Briefly, DNA was extracted from each tumor sample using the AllPrep DNA/RNA Minikit (Qiagen, 80204) for samples with sufficient tumor tissue and matched normal blood when it was available. Note that the selected tissue for bulk DNA isolation was adjacent to the tissue used for single-nuclei data generation. Briefly, DNA was sheared to 400 bp using a LE220 ultrasonicator (Covaris) and size-selected using AMPure XP beads (Beckman Coulter, A63882). Whole-genome libraries were prepared and sequenced with 150-base-pair paired-end sequencing on a NovaSeq 6000 (Illumina). Data for the SNU cohort are available on Synapse (https://www.synapse.org/glass). DNA-sequencing alignment, fingerprinting, somatic variant detection (Mutect2) and copy-number segmentation were performed in accordance with the Genome Analysis Toolkit (GATK) best practices using GATK 4.0.10.1, as previously described7,9. Briefly, whole-exome and whole-genome reads were aligned to the b37 genome (human_g1k_v37_decoy) using BWA MEM 0.7.17. DNA fingerprint analysis using ‘CrosscheckFingerprints' (Picard) confirmed that all samples belonging to a patient came from the same individual, indicating that there were no sample mismatches in this study. Somatic mutations were detected using Mutect2 (v4.1.0.0) for tumor samples with a matched normal blood sample. A panel of normals was constructed for each sequencing batch to account for differences in DNA library prep (for example, whole genome versus whole exome) and used to filter out common artifactual and germline variants. Patient tumor samples without matched normal blood were analyzed for copy-number alterations and tumor-only SNV detection was performed with a panel of normal reference according to GATK best practices. Tumor-only somatic variants were used to identify hypermutant tumors. To identify samples with increased small deletion mutation burden, the following thresholds were applied: the recurrence-specific small deletion mutation burden needed to be greater than 0.2 Mut per Mb sequenced and greater than 0.1 Mut per Mb increase when comparing all small deletion variants in the recurrence versus the primary tumor. Mutational signature estimation was performed using COSMIC v3 signatures for the GLASS and CARE datasets according to previously published methods41. Briefly, all WGS samples were used to generate a mutational matrix and extract de novo signatures using SigProfilerExtractor (v.1.1.4)41 using maximum signatures set to 10 and NMF replicates set to 100. For samples with matched blood, hypermutation was determined according to the cutoff of 10 Mut per Mb and further separated into de novo versus treatment-associated based on COSMIC mutational signatures. Treatment-associated hypermutation in samples with matched normal blood was classified by a longitudinal increase in mutation burden (that is, from <10 Mut per Mb to >10 Mut per Mb) and SBS11 signal (alkylating agent-associated signature). For samples without matched blood, a single recurrent sample with a large increase in mutation burden at recurrence compared to the initial tumor (18.3-fold increase) and SBS11 signature was indicative of treatment-associated hypermutation. Genetic distance between time-separated samples from the same patient was defined as the bulk DNA proportion of private mutations (SNV) and differences in CNA segments between the two samples. For droplet-based snRNA-seq data, the Cellranger v3.1.0 pipeline provided by 10x Genomics was used for alignment (GRCh38 release 93) and to generate count matrices. Following these normalization steps, initial filtering of low-quality cells was done based on low number of detected genes (less than 200 genes) or high expression of mitochondrially encoded genes (more than 3%). The majority of low-quality cells were removed downstream in case they could not be robustly classified as malignant or nonmalignant (see in later section on CNA analysis ‘Classification of cells into malignant or nonmalignant cell types'). For the cells and genes that passed these quality control filters, we defined relative expression levels by centering the expression levels for each gene across all cells in the dataset as follows: \({{\mathrm{Er}}}_{i,j}={E}_{i,j}-\,\frac{1}{n}{\sum }_{k=1}^{n}{E}_{i,k}\), where n is the number of cells in the dataset. To facilitate clustering, cell-type annotation and scoring of droplet-based data, we used the R package Seurat (v4.0.4)43. Initial classification of tumor-associated macrophages, T cells, oligodendrocytes and endothelial cells was based on the expression of known marker genes16,17,18,19,44, whereas the rest of the cells were annotated as presumed malignant. Following classification of cells as malignant or nonmalignant (see sections ‘Inferring CNAs from gene expression data' and ‘Classification of cells into malignant or nonmalignant cell types'), the cells classified as nonmalignant were scored (using the AddModuleScore function) for known cell-type signatures and assigned with a cell type using the method explained in the section ‘Assignment of cells to states'. Finally, heterotypic doublets were filtered out using the R package scDblFinder45. 50, which is available at https://github.com/jlaffy/infercna) with default parameters. Briefly, the algorithm sorts the analyzed genes in each sample according to their chromosomal location and applies a moving average with a sliding window of 100 genes within each chromosome to the relative expression values. In cells classified using single-nucleus droplet-based sequencing, we generally encountered lower CNA signal and continuous signal distributions related to (1) lower data quality relative to that observed in single-cell data and (2) inclusion of nonmalignant cell types such as ACs and neurons in the presumed malignant group, as these cell types could not be classified a priori as nonmalignant due to transcriptional similarity with the malignant cells. We, therefore, used a multistep classification scheme for the droplet-based single-nucleus sequencing data. For each cell j, we computed the CNA signal on each chromosome separately as \({{\mathrm{CNA}}}_{j}^{C}=\frac{1}{n}{\sum }_{i=1}^{n}{{\mathrm{CNA}}}_{i,j}^{C}\), where n is the number of genes on chromosome C included in the analysis. This enabled assigning each cell with a P value for each chromosome using a two-sided z test (function pnorm from the R package stats v.4.1.1), reflecting whether the cell has a CNA signal that is substantially different than expected from nonmalignant cells. These P values were further corrected for multiple testing (for each chromosome) using the Benjamini–Hochberg (BH) method, and cells with BH-adjusted P value < 0.05 for a particular chromosome were considered as having a copy-number event on that chromosome (either amplification or deletion based on the sign of the CNA signal). Cells with an unadjusted P value < 0.05 for a particular chromosome that failed to achieve statistical significance following adjustment were considered to have a ‘suspicious event' on that chromosome. Following this, inferred copy-number events were cross-referenced with copy-number calls from WES/WGS data, and only events detected in both methods were considered for classification and were termed ‘real' events. Cells classified a priori as nonmalignant were reclassified as ‘unresolved' in case a ‘real' or ‘suspicious' CNA event was detected and ‘nonmalignant' otherwise. Presumed malignant cells with at least one ‘real' CNA event were classified as ‘malignant', those with a ‘suspicious' event were classified as ‘unresolved' and those with no events at all (‘real' or ‘suspicious') were classified as ‘nonmalignant'. We limit the computation to the chromosomes on which ‘real' events were detected and a control panel of ~1,000 genes from other chromosomes. Briefly, for each sample with available WES data, we called SNVs for each cell using the Vartrix tool (https://github.com/10XGenomics/vartrix) from the FASTQ files. Misclassified cells were defined as cells harboring a malignant mutation that were classified as nonmalignant in step 1 of the algorithm. Following a cost-effectiveness analysis (misclassification rate versus percentage of excluded cells), we set 0.35 as the upper threshold for nonmalignancy. To refine the classification, we defined confidence levels. Confidence level 1 includes cells classified as malignant that harbor a malignant SNV, cells in which at least 50% of the expected CNA events were detected, including chromosome 7 amplification or chromosome 10 deletion, and cells classified as nonmalignant with CNAcor < 0.25. Confidence level 2 includes cells classified as malignant in which at least 50% of the expected CNA events were detected that do not include chromosome 7 amplification or chromosome 10 deletion, cells classified as malignant in which less than 50% of the expected CNA events were detected, including chromosome 7 amplification or chromosome 10 deletion, and cells classified as nonmalignant with CNAcor < 0.5. NMF was performed on the relative expression values of each sample independently after setting negative values to zero. Because ‘k' varies between samples and is largely unknown, we ran the NMF algorithm on each sample using different values (3–10), thereby generating 52 programs for each sample. The method first filters out NMF programs that are not robust (do not recur within the sample it originated from or across several samples) or are redundant within the sample (that is, it highly overlaps with other NMF programs within that sample). Following this, the robust NMF programs are clustered according to the Jaccard similarity using a customized clustering algorithm that iteratively considers the degree of overlap between robust NMF programs and combines the highly overlapping ones to form a cluster. The top 50 most recurring genes in each cluster define a metaprogram. Overall, the algorithm yielded 16 metaprograms that were annotated based on functional enrichment analysis (using Gene Ontology (GO) terms, mSigDB Hallmark gene sets and gene sets derived from normal brain development datasets). MP 16 included genes from various cell types, resulting in a metaprogram that fits almost all cells and thereby did not reflect any heterogeneity and was therefore excluded from analysis. This same procedure was independently repeated for each nonmalignant cell type. To facilitate cell classification, we generated 20 shuffled expression matrices by sampling 5,000 cells each time and shuffling the expression values for each gene. We then scored each shuffled matrix for the NMF metaprograms, thereby yielding 100,000 normally distributed scores for each expression program. These served as null distributions for cell state classification. For each original cell, we computed a P value for each of the expression programs with a z test (R's pnorm function) using the statistics of the null distributions that we previously generated. We adjusted all P values for multiple testing using the BH method. Each cell was classified into a specific state if the adjusted P value computed for that state was smaller than 0.05. Cells that achieved an adjusted P < 0.05 for multiple states were assigned to the state for which they scored maximally. Cells that did not achieve an adjusted P < 0.05 for any of the states were assigned an ‘unresolved' state. Cells were assigned a ‘cycling' state on top of their cellular state if they achieved an adjusted P < 0.05 for the cell cycle metaprogram and ‘noncycling' otherwise. We define hybrids as cells scoring significantly (that is, with adjusted P < 0.05) for at least two cellular states with a difference between the top two significant scores smaller than expected by chance. The 95 quantiles of this score difference distribution (equal to a difference of 0.24) were set as the threshold for determining if a state is singular or hybrid. To estimate the expected frequency of technical hybrids, we first computed for each pair of states, \(\left(A,{B}\right)\), the expected frequency of hybrids, defined as \({\mathrm{Exp}}\left(A,B\right)={\mathrm{Freq}}\left(A\right)\times {\mathrm{Freq}}\left(B\right)\), and the O/E ratio defined as \({\log }_{2}(\frac{{\mathrm{Obs}}(A,{B})}{{\mathrm{Exp}}(A,B)})\). As this is an overestimation of the expected frequency attributed to technical effects, we estimated this technical factor by averaging across O/E ratios of hybrid pairs detected less than expected by chance (that is, O/E < 0). Finally, we filtered out insignificant hybrid pairs with large O/tE ratio (attributed to small numbers) using a one-sided Fisher test. We did not include the ‘cycling' state in this analysis because we don't view ‘cycling' as an independent state but rather as an additional feature that cells can have on top of their neurodevelopmental identity (that is, AC-like, OPC-like, etc. Each tumor sample was decomposed into seven pseudobulk profiles (one for each of the common states—AC-like, MES-like, hypoxia, GPC-like, OPC-like, NPC-like and NEU-like—given at least 25 cells classified to the particular state) by averaging across the normalized unique molecular identifier counts and log2-transforming. Genes were included if their mean log2-expression value across all pseudobulks was at least 1 and if their median variance (computed separately within state) was at least 2.5. We then derived six gene programs from each state (top and bottom 50 genes of the first three principal components), computed the Jaccard similarity index between each pair of programs and clustered the similarity matrix using hierarchical clustering. This revealed five distinct clusters from which we derived five consensus signatures by including genes that recurred in at least 25% of programs in each of the clusters (with a hard minimum of at least three times). By manual inspection and GO enrichment, we annotated the consensus signatures. Two signatures were excluded due to short length or high suspicion of reflecting a technical artifact. There was no difference in the estimated contamination level across the different baseline profiles as well as no significant overlap between the topmost contaminated genes per sample and the metaprogram or baseline profile signatures (data not shown due to lack of significance). To measure the differences in tumor composition, we generated for each tumor sample a composition vector reflecting the proportion of each cell type in the tumor sample. This enabled assigning tumors to the following three main composition groups: HMF (percentage of malignant cells >75%), IMF (percentage of malignant cells, 50–75%) and LMF (malignant cells <50%), where the LMF group could be further subdivided according to the dominant TME cell types (percentage of TAM/oligodendrocyte/glio-neural >40%, or mixed in case of no dominant TME cell type). Similarly, we defined proportion vectors for the malignant cell states and assigned tumors to six groups according to the dominant cell state (at least 25% of cells assigned to the particular state)—AC, MES/hypoxia, GPC, OPC/NPC, neuron and mixed (in case of no dominant state). We then generated an undirected graph by defining each feature as a node and connecting the nodes with edges in case a statistically significant association (unadjusted for multiple testing) between the two features was observed. To measure the transcriptional distance between matched pairs, we generated state-controlled pseudobulk profiles from each pair. State control was achieved by downsampling both samples to 25 cells from each state, given that both samples contain at least 25 cells from the particular state, resulting in samples that contain the same number of cells and are balanced across the different gene expression profiles of the cellular states. Following this, the pseudobulk profiles were computed by averaging for each gene across the selected cells, resulting in two pseudobulk profiles. Following, we computed the Euclidean distance between these two vectors. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Processed snRNA-seq data generated for this study are available at the Gene Expression Omnibus under accession GSE274546 (10x Genomics). Raw sequencing data are available with limitations in accordance with consent forms from the Data Use Oversight System (DUOS) at https://duos.broadinstitute.org/ under the following IDs: DUOS-000475, DUOS-000476, DUOS-000477, DUOS-000478, DUOS-000479 and DUOS-000480. Scripts for processing DNA-sequencing data can be accessed at https://github.com/Kcjohnson/care-glass. Hegi, M. E. et al. Gene silencing and benefit from temozolomide in glioblastoma. Stupp, R. et al. Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma. Mu, Q. et al. Identifying predictors of glioma evolution from longitudinal sequencing. & Park, P. J. Spatiotemporal evolution of the primary glioblastoma genome article spatiotemporal evolution of the primary glioblastoma genome. Kim, H. et al. Whole-genome and multisector exome sequencing of primary and post-treatment glioblastoma reveals patterns of tumor evolution. Barthel, F. P. et al. Longitudinal molecular trajectories of diffuse glioma in adults. Varn, F. S. et al. Glioma progression is shaped by genetic evolution and microenvironment interactions. Hoogstrate, Y. et al. Transcriptome analysis reveals tumor microenvironment changes in glioblastoma. Wang, J. et al. Clonal evolution of glioblastoma under therapy. Garofano, L. et al. Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities. Wang, L. et al. A single-cell atlas of glioblastoma evolution under therapy reveals cell-intrinsic and cell-extrinsic therapeutic targets. The multilayered transcriptional architecture of glioblastoma ecosystems. Campbell, J. N. et al. A molecular census of arcuate hypothalamus and median eminence cell types. Habib, N. et al. Div-Seq: single-nucleus RNA-seq reveals dynamics of rare adult newborn neurons. Polioudakis, D. et al. A single-cell transcriptomic atlas of human neocortical development during mid-gestation. Nowakowski, T. J. et al. Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex. Johnson, B. E. et al. Mutational analysis reveals the origin and therapy-driven evolution of recurrent glioma. Kocakavuk, E. et al. Radiotherapy is associated with a deletion signature that contributes to poor outcomes in patients with cancer. Kinker, G. S. et al. Pan-cancer single-cell RNA-seq identifies recurring programs of cellular heterogeneity. Greenwald, A. C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Gavish, A. et al. Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours. Inference and analysis of cell–cell communication using CellChat. Gjørlund, M. D. et al. Neuroligin-1 induces neurite outgrowth through interaction with neurexin-1β and activation of fibroblast growth factor receptor-1. Targeting neuronal activity-regulated neuroligin-3 dependency in high-grade glioma. Müller, T. et al. Neuregulin 3 promotes excitatory synapse formation on hippocampal interneurons. Venkatesh, S. et al. Electrical and synaptic integration of glioma into neural circuits. Venkataramani, V. et al. Glioblastoma hijacks neuronal mechanisms for brain invasion. Rivera, A. L. et al. MGMT promoter methylation is predictive of response to radiotherapy and prognostic in the absence of adjuvant alkylating chemotherapy for glioblastoma. Butler, M. et al. MGMT status as a clinical biomarker in glioblastoma. Costello, J. F., Futsched, B. W., Tanoil, K., Graunkes, D. M. & Pieper, R. Graded methylation in the promoter and body of the 06-methylguanine DNA methyltransferase (MGMT) gene correlates with MGMT expression in human glioma cells. Touat, M. et al. Mechanisms and therapeutic implications of hypermutation in gliomas. & Huang, M. N. The repertoire of mutational signatures in human cancer. Johnson, K. C. et al. Single-cell multimodal glioma analyses identify epigenetic regulators of cellular plasticity and environmental stress response. Telarovic, I., Wenger, R. H. & Pruschy, M. Interfering with tumor hypoxia for radiotherapy optimization. Wang, Q. et al. Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment. Mathur, R. et al. Glioblastoma evolution and heterogeneity from a 3D whole-tumor perspective. Islam, S. M. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. Shinde, J. et al. Palimpsest: an R package for studying mutational and structural variant signatures along clonal evolution in cancer. Velmeshev, D. et al. Single-cell genomics identifies cell type-specific molecular changes in autism. Germain, P., Lun, A., Meixide, C. G., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Neftel, C. et al. An integrative model of cellular states, plasticity, and genetics for glioblastoma. Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Venteicher, A. S. et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Patel, A. P. et al. Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma. Gaujoux, R. & Seoighe, C. A flexible R package for nonnegative matrix factorization. Spitzer, A., Johnson, K., Nomura, M. & Garofano, L. The multi-layered transcriptional architecture of glioblastoma ecosystems. We thank R.A. DePinho, D. Arons and K. Knight for their support with the GBM CARE initiative. A. Lasorella is supported by NIH grants (R01CA268592, R01CA280560 and R01CA239721). has received honoraria from Elsevier, BMJ Best Practice, Oakstone Publishing and Servier, as well as research support from Merck & Co., Conquer Cancer (the American Society of Clinical Oncology Foundation), the Robert Wood Johnson Foundation and the National Cancer Institute. We thank the PROACTIVE team at MD Anderson for their assistance in procuring patient material. We gratefully acknowledge the contribution of the Jackson Laboratory Single Cell Biology service, S. Bessonett, in particular, the Genome Technologies service and Cyberinfrastructure high-performance computing resources at the Jackson Laboratory for expert assistance related to processing NORLUX/SNU samples and data analysis. These shared services are supported in part by the Jackson Laboratory Cancer Center (P30CA034196). We thank SciStories for the figure schematics. These authors contributed equally: Avishay Spitzer, Kevin C. Johnson, Masashi Nomura, Luciano Garofano. These authors jointly supervised this work: Anna Lasorella, Antonio Iavarone, Roel G. W. Verhaak, Itay Tirosh, Mario L. Suvà. Avishay Spitzer, Noam Galili Darnell, Alissa C. Greenwald & Itay Tirosh Department of Pathology and Krantz Family Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Masashi Nomura, Lillian Bussema, Simon Gritsch, L. Nicolas Gonzalez Castro & Mario L. Suvà Broad Institute of Harvard and MIT, Cambridge, MA, USA Masashi Nomura, Lillian Bussema, Simon Gritsch, L. Nicolas Gonzalez Castro & Mario L. Suvà Luciano Garofano, Young Taek Oh, Fulvio D'Angelo, Simona Migliozzi, Anna Lasorella & Antonio Iavarone The Jackson Laboratory for Genomic Medicine, Farmington, CT, USA Institute for Systems Genomics, University of Connecticut, Storrs, CT, USA Department of Neurology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Department of Neurology, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA Center for Neuro-Oncology, Dana-Farber Cancer Institute, Harvard Medical School, Boston, MA, USA Anuja Lipsa, Frank Hertel, Anna Golebiewska & Simone P. Niclou Franck Bielle, Alice Laurenge, Alberto Picca & Marc Sanson AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neuropathology, Paris, France AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neuro-oncology, Paris, France AP-HP, Groupe Hospitalier Pitié-Salpêtrière, Neurosurgery, Paris, France Brain Tumor Bank Onconeurotek (ONT), AP-HP, Paris, France Department of Neurosurgery, The Preston Robert Tisch Brain Tumor Center, Duke University, Durham, NC, USA Department of Pathology, Brigham and Women's Hospital, Boston, MA, USA Department of Pathology, Dana–Farber Cancer Institute, Boston, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar provided patient samples and curated the clinical annotation. wrote the paper, with feedback from all authors. jointly supervised this work and interpreted the results. Correspondence to Anna Lasorella, Antonio Iavarone, Roel G. W. Verhaak, Itay Tirosh or Mario L. Suvà. is an equity holder, scientific cofounder and advisory board member of Immunitas Therapeutics. is a cofounder and equity holder in Boundless Bio. The other authors declare no competing interests. Nature Genetics thanks Richard Gilbertson, Wolfgang Wick and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Number of cells per sample (post QC) across the 3 time points. There are no statistically significant differences between the groups (tested with two-sided Kruskal–Wallis test, p = 0.51). b, Average number of detected genes per sample (post QC) across the 3 time points. There are no statistically significant differences between the groups (tested with two-sided Kruskal–Wallis test, p = 0.27). c, Overview of SNV and CNA in the GBM cohort. The top panel indicates the total proportion of mutations that are either detected across both T1 and T2 (that is, shared), only in T1 or only in T2, which are sorted by decreasing shared mutation fraction. a, Relative granular cell-type abundance across the two time points (primary and recurrence). Unadjusted two-sided paired Wilcoxon rank-sum test is shown (n = 56 patients). b, Relative granular non-malignant cell state abundance across the two time points (primary and recurrence) that showed statistical significance between time points following adjustment for multiple comparisons. Unadjusted two-sided paired Wilcoxon rank-sum test is shown (n = 56 patients). c, Average proportion of cellular composition (left) of malignant cell state composition (middle) and average baseline profile score (right) for each feature group. d, Available clinical data information for this study and ref. e, Comparison of the number of detected genes per cell in cells passing quality filters between this study and ref. Box plots span from the first to third quartiles, median values are indicated by a horizontal line, whiskers show 1.5× interquartile range and outlier points are not shown. 14 assigned to each compositional group across the two time points (primary and recurrence). Bottom panel is for malignant state groups. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. h, Forest plot reflects Cox proportional hazard model for time from 2nd resection to outcome, including relevant biological and clinical covariates in addition to composition group (top) and malignant state group (bottom). a,b, Malignant cell state abundance across time points. Unadjusted two-sided paired Wilcoxon rank-sum tests are shown. a, This study (n = 50 patients with at least 50 malignant cells in both time points). e, Interaction graph of associations between features conserved across time points. Edges represent statistically significant (p < 0.05) Pearson correlation in both time points. Each row represents a ligand–receptor interaction, while each column represents a non-malignant cell type—malignant cell state pair, such that the ligand is expressed by the corresponding non-malignant cell type and the receptor by the malignant cell state. a, Sankey plots depicting the transitions between time points across malignant states for ref. Heatmap representing three transcriptomic layers between time points are shown. X axis reflects the time from 1st to 2nd resection. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. X axis reflects the time from 1st to 2nd resection. Statistical significance of survival difference between the groups in each panel was computed using the log-rank test. d, Proportion (%) of each cell type across the two time points (primary and recurrence) stratified by MGMT expression level. Individual sample values are shown as jittered points. e, Proportion (%) of each malignant cell state across the two time points (primary and recurrence) stratified by MGMT expression level. h, Differentially expressed genes between MGMT-low and high groups across the recurrence trajectories (Methods). Each dot represents a gene, and dots are colored according to the group of DE genes to which they belong. i, CONS-UP and CONS-DN signatures plotted for patients included in the analysis. Value represents the T2–T1 difference (log2(FC) of gene expression) for each patient. Patients are ordered according to the difference between signature scores. Genes are ordered according to their correlation (across patients) with the difference between signature scores. a, Mutation burden at both time points for patients with matched normal blood DNA (n = 46 samples per time point, two-sided paired Wilcoxon test). b, COSMIC v3 mutational signature contribution for two patients with high mutation burden at both time points with minimal contribution from SBS11 (alkylating agent signature). c, Selected single base substitution (SBS) mutational signature proportions for GLASS IDH-wildtype cases for both time point 1 and time point 2. Dotted lines connect samples from the same patients. SBS1 (aging/clock-like etiology), SBS11 (alkylating agent) and SBS21 (defective mismatch repair). P values represent two-sided paired Wilcoxon rank-sum test. d, Longitudinal difference in small insertion burden between T1 and T2 with dotted lines connecting patient values. P-value reflects two-sided paired Wilcoxon rank-sum test. The Pearson correlation coefficient and P value are presented. f, Longitudinal change in the hypoxia and MES-like cell state relative abundances for patients that acquired SBS21 mutations but that did not acquire small deletion burden increase. P value from paired Wilcoxon rank-sum test. h,i, Scatterplots that represent the scaled genetic distance for SNVs (proportions of private mutations) h, and CNAs (differences in copy-number altered regions, i, versus scaled transcriptional distance (snRNA). Shown is the Pearson's correlation coefficient, associated P value, with the linear regression line shown in blue for both h and i. Cohort description, including sample source, sequencing platforms, patients' genetic and clinical metadata. Lab, laboratory; M, male; F, female; NA, not accessed; CCNU, lomustine; ACNU, nimustine hydrochloride. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Spitzer, A., Johnson, K.C., Nomura, M. et al. Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250508113115.htm'>Nature visits can improve well-being disparities among urban dwellers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-09 02:54:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The Kobe University analysis is based on survey results from two major Japanese metropolitan areas. Kobe University human environmental scientist UCHIYAMA Yuta specializes in studying how differences in urban and rural lifestyles affect quality of life. He and his team surveyed 3,500 residents in two major metropolitan areas of Japan (Tokyo-Yokohama and Osaka-Kobe) in order to determine their relationship with nature both objectively (based on proximity and access to natural spaces) and subjectively (based on perceived relatedness to nature). The responses were then categorized based on socioeconomic status by area and the degree of urban development of their residential area to serve as a third dimension in clarifying the relationship between well-being and nature. The results of this study, published in Landscape and Urban Planning, show that the more related to nature people feel, the better their overall well-being is, and that this relationship is particularly pronounced in urbanized areas with poorer socioeconomic conditions. Related studies have suggested that nature visits are important for maintaining and improving health for those under substantial stress, and that this is especially true for residents of areas with poorer socioeconomic conditions. Conversely, residents of areas with better socioeconomic conditions have access to additional resources for enhancing their health and overall well-being, making the effects of nature visits relatively less pronounced. Uchiyama explains what this means for policies aimed at reducing inequality in well-being, saying: "We anticipate that conserving and enhancing existing natural spaces and organizing community events will have a positive effect on well-being in areas of relatively poor socioeconomic status. Interestingly, of the factors we surveyed, childhood experience with nature in particular was a significant predictor of adult well-being." Moving forward, further analysis of the relationships between independent variables as well as key factors that vary over time (e.g., moving between regions of differing socioeconomic level) should prove useful in obtaining a clearer picture of the relationship between nature relatedness, socioeconomic level and well-being. Uchiyama says, "As these findings are consistent with similar studies conducted in other regions, interventions for both subjective and objective factors should be applicable to other regions with similar socioeconomic contexts, such as other cities in Monsoon Asia. This research was funded by the Japan Society for the Promotion of Science (grants JP23K25067, JP23K25106, JP23K28298, and JP23K28295), the Environment Research and Technology Development Fund (grant 1FS-2201), and Asia-Pacific Network for Global Change Research (CRRP2023-10MY-Uchiyama). Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            