
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251209043040.htm'>Simple light trick reveals hidden brain pathways in microscopic detail</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 15:52:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Every tissue in the human body contains exceptionally small fibers that help coordinate how organs move, function and communicate. Together, these intricate fiber systems help shape the structure of each organ and keep them operating properly. Although these microscopic structures play essential roles, they have long been challenging to study. Researchers have struggled to determine how fibers are oriented inside tissues, which has made it difficult to fully understand how they change in health and disease. A research team led by Marios Georgiadis, PhD, instructor of neuroimaging, has now introduced an approach that makes these hard-to-see fiber patterns visible with exceptional clarity and at a relatively low cost. Their technique, described in Nature Communications, is known as computational scattered light imaging (ComSLI). It can reveal the orientation and organization of tissue fibers at micrometer resolution on virtually any histology slide, regardless of how it was stained or preserved -- even if the slide is many decades old. Michael Zeineh, MD, PhD, professor of radiology, served as co-senior author with Miriam Menzel, PhD, a former visiting scholar in Zeineh's laboratory. "The information about tissue structures has always been there, hidden in plain sight," Georgiadis said. "ComSLI simply gives us a way to see that information and map it out." The method requires only a rotating LED light and a microscope camera, making the setup accessible compared with other forms of advanced microscopy. After the images are collected, software analyzes delicate patterns in the scattered light to generate color-coded maps of fiber orientation and density, known as microstructure-informed fiber orientation distributions. Scientists can also revisit slides originally created for unrelated projects, even those stored for decades, allowing new structural insights without altering the samples. "This is a tool that any lab can use," Zeineh said. "You don't need specialized preparation or expensive equipment. What excites me most is that this approach opens the door for anyone, from small research labs to pathology labs, to uncover new insights from slides they already have." A major goal in neuroscience has been to chart the brain's microscopic pathways with high precision. They also examined how these fibers change in neurological conditions such as multiple sclerosis, leukoencephalopathy and Alzheimer's disease. With these detailed maps, researchers can see how memory circuits break down as disease progresses. Even in this century-old sample, ComSLI identified intricate fiber patterns, allowing scientists to study historical specimens and explore how structural features evolve across generations of disease. Although first designed for brain research, ComSLI also works well in other tissues. In tongue muscle, the method highlighted layered fiber orientations linked to movement and flexibility. In bone, it captured collagen fibers that align with mechanical stress. It also means that millions of stored slides around the world may contain untapped microstructural information. "Although we just presented the method, there are already multiple requests for scanning samples and replicating the ComSLI setup -- so many labs and clinics would like to have micron-resolution fiber orientation and micro-connectivity on their histology sections," Georgiadis said. "Another exciting plan is to go back to well-characterized brain archives or brain sections of famous people, and recover this micro-connectivity information, revealing 'secrets' that have been considered long lost. New Paper-Thin Brain Implant Could Transform How Humans Connect With AI Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/energy/a69615419/new-material-supercapacitors/'>A New Material Could Finally Unlock the Incredible Power of Supercapacitors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>By creating a new graphene material, engineers were able to facilitate the movement of ions and increase the power and energy capacity of their supercapacitors. We may earn commission if you buy from a link. With more power density (or power per unit of space) than batteries and more energy density than dielectric capacitors (which store electric charge), these hybrids of the future should be able to merge superfast charging with a power boost. But they currently have one major drawback: The materials they are made of are not able to take advantage of their entire storage capacity. The issue is that only a fraction of that surface area is actually available, and previous efforts to optimize surface area have still not been enough. Enter mechanical engineer Mainak Majumder and his research team at Monash University in Australia. The reason that current technology falls short is because of chemical interactions that make it impossible for ions to adhere to spaces between sheets of graphene in 3D structures. While the design flexibility of graphene makes it possible to create denser electrodes, which in turn make it easier for ions to move around, they are still not using much of the potential storage capacity they have. “We design[ed] a graphene structure tailored to deliver volumetrically-efficient capacitive energy storage,” Majumder said in a study recently published in Nature Communications. “By integrating nano- and meso-scale features of graphene in micron-sized particles, we demonstrate that a multiscale structure, benefiting from each scale, enhances ion accessibility, transport kinetics, energy storage capacity and long-term stability.” Australia has vast reserves of graphite, from which graphene is created. The team heated graphite oxide powder rapidly and cooled the resulting substance—an emerging material known as multiscale reduced graphene oxide (M-rGO)—slowly so that it could be more workable and prevent internal stress that could cause it to break. Unlike graphite, graphene has the advantage of chemical interactions between its sheets, whose carbon atoms are arranged in honeycomb structures. These interactions create what Majumder calls “reactive energetic hotspots” that facilitate movement for ions. Majmuder's supercapacitors not only outperformed previous graphene structures with especially high energy and power densities, but remained stable, and charged in almost no time. They were so successful that commercialization is already on the horizon. “Thanks to an extended voltage window […] these pouches deliver a [high] volumetric energy density,” he said. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69622721/stonehenge-pits/'>Scientists Discovered the Truth About 16 Mysterious Pits Near Stonehenge</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>and determined they may have been part of a ceremonial boundry. The 16 uniformly shaped massive pits found at the Neolithic Durrington Walls henge just north of Stonehenge were not a random natural occurrence. “These features were not simply dug and abandoned—they were part of a structured, monumental landscape that speaks to the complexity and sophistication of Neolithic society.” The team's findings were detailed in a study published by Internet Archaeology, showing just how this structure, the largest of its type ever found in Britain, likely came to be. The 16 pits form a circle over 1.25 miles wide at Durrington Walls, itself one of the largest prehistoric enclosures in Britain, enclosing an area greater than one square mile. “The synchronous timing of this could only have been achieved by a dedicated and coordinated action.” “The skill and effort that must have been required not only to dig the pits, but also to place them so precisely within the landscape is a marvel,” Richard Bates, professor at St Andrews, said in a statement. “When you consider that the pits are spread over such a large distance, the fact they are located in a near perfect circular pattern is quite remarkable.” Archaeologists believe the pits may have marked a sacred boundary linked to ceremonial activity at Durrington Walls, like what happened at Stonehenge. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69596698/283-settlements-civilization/'>Archaeologists Unearthed 483 Ancient Settlements That Could Be Pieces of a Lost Civilization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Taken together, the discovery of 483 Late Bronze Age settlements in what was then Asia Minor/Anatolia tells an entirely new story about the region, including the powerful role it may have played with a culture completely independent of others. According to a new study published in Nature Scientific Data, a team of archaeologists, geologists, and geodata specialists wrote that they had identified 483 major settlements from the Middle and Late Bronze Age (2000 to 1300 B.C.E.) that give evidence of an entire culture lost to time. The modern-day region in western Turkey across from Greece largely went unstudied for centuries, according to the scholars, leaving a void in the history books about an area that may have been highly influential in its time. “The capacity to study these relationships on a regional scale is entirely new for Western Anatolia.” The team said that while the ancient settlements may have numbered twice what they have found so far, it's clear that fertile farmland was the deciding factor in choosing a settlement site. Communities also typically avoided floodplains, opting for villages on slightly elevated ground near reliable water sources. Hilltops were key, as they commanded control of strategic passes and allowed villagers to fortify their sites. The team found that settlements often appear at roughly 17-kilometer (10.5 miles) intervals, approximately a day's travel. “Once we do so, long-standing puzzles begin to fall into place: who the Sea Peoples were, why the Hittite kingdom collapsed, and whether there was, in fact, something akin to a Trojan War.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Divers Couldn't Believe a Lost Shipwreck's Age A Backwards Bible Map From 1525 Changed the World Scientists Found a Warning from the Roman Empire Archaeologists Found Gold Pieces at a Funeral Pyre</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03965-x'>What will be the first AI-designed drug? These disease-fighting antibodies are top contenders</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 12:44:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. An illustration of antibodies (pink) binding to a flu virus (brown).Credit: Juan Gaertner/Science Photo Library Biologists achieved a landmark in protein design last year: using artificial intelligence (AI) to draw up entirely new antibody molecules1. Yet the proof-of-principle designs lacked the potency and other key features of commercial antibody drugs that rack up tens of billions in annual sales. In recent weeks, numerous teams have reported successfully using proprietary commercial AI tools and open-source models to make antibodies that have properties of antibody drugs. ‘A landmark moment': scientists use AI to design antibodies from scratch ‘A landmark moment': scientists use AI to design antibodies from scratch “These latest efforts are remarkably powerful advances that enable a democratization of antibody engineering,” says Chang Liu, a synthetic biologist at University of California, Irvine. The latest wave of success with de novo antibody design “will have a big impact on how quickly and how many de novo therapeutics we will see in clinical trials”, adds Timothy Jenkins, a protein engineer at the Technical University of Denmark in Kongens Lyngby. “There isn't much precision,” says Surge Biswas, chief executive of antibody-design company Nabla Bio in Cambridge, Massachusetts. “The promise of AI-guided design is that you can be atomically precise,” Biswas adds. AlphaFold is five years old — these charts show how it revolutionized science AlphaFold is five years old — these charts show how it revolutionized science Antibodies — immune proteins that recognize foreign molecules, such as those made by pathogens, with exquisite specificity — have been a challenge for AI to design. AI models such as AlphaFold have struggled to predict the shape of flexible loop regions of antibodies, which they use to recognize their targets. But new tools developed in the past year — including an updated version of AlphaFold — have proven better at modelling these flexible regions, says Gabriele Corso, a machine-learning scientist at the Massachusetts Institute of Technology in Cambridge. For instance, a team at Stanford University in California and the Arc Institute in Palo Alto, California, has also released a model that can design nanobodies with high efficiencies3. Last month, scientists at Nabla and Chai Discovery in San Francisco, California, said they had made ‘drug-like' antibodies using AI tools. Lab experiments showed that some of the designer molecules recognized diverse disease targets — including molecules called G-protein coupled receptors (GPCRs), which have challenged conventional antibody design efforts — at potencies similar to those of commercial antibody drugs. They also boasted useful properties that can make or break drug candidates, such as the capacity to be produced at high levels and to recognize only their intended targets5. AlphaFold is running out of data — so drug firms are building their own version AlphaFold touted as next big thing for drug discovery — but is it? The huge protein database that spawned AlphaFold and biology's AI revolution AI is saving time and money in research — but at what cost? AI chatbots can sway voters with remarkable ease — is it time to worry? Pig-organ transplants are often rejected — researchers find a way to stop it The Dean of Systems Hub holds both an executive and faculty appointment and serves as a key member of the University's senior administrative team. AlphaFold is running out of data — so drug firms are building their own version AlphaFold touted as next big thing for drug discovery — but is it? The huge protein database that spawned AlphaFold and biology's AI revolution An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03977-7'>What we eat is making us obese and sick — but science shows solutions are within reach</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 12:04:04
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Diet champions celebrate a decade of weight loss, achieved by moving away from ultra-processed foods. Participants who ate ultra-processed foods (UPFs) consumed about 500 kilocalories more per day and gained weight, whereas those on a minimally processed diet lost weight. The results offered strong evidence that many industrially produced foods can override satiety signals and drive overeating. And it made Hall one of the most influential figures in nutritional science. Understandably, expectations were high for his book, Food Intelligence, co-written with Canadian journalist Julia Belluz. It digs deeper into the implications of the 2019 study — that the harms of processed foods can't be reduced to their fat, sugar or salt content alone, and that the prevalent government strategy of politely asking companies to tweak their recipes by cutting salt or sugar content, say, is unlikely to address rising obesity rates. In this wide-ranging primer on food science, Hall and Belluz revisit many food-related misconceptions. They debunk the idea that cutting 500 kilocalories a day reliably leads to the loss of half a kilogram a week and that “broken metabolism” explains obesity — as Hall highlighted in his role as a scientific adviser to the reality television programme The Biggest Loser, in which contestants attempt extreme weight loss. They are equally dismissive of the booming supplements industry, arguing that many products lack clinical evidence or have been shown to offer no real benefit. Their scepticism extends to continuous glucose monitoring in people without diabetes, which they warn might provoke needless anxiety without providing clear advantages. The critique culminates in a forceful take down of the personalized-nutrition industry — diet plans based on genetics, which they argue have no solid scientific basis. The FDA also allows companies to introduce additive chemicals into food if these are merely ‘generally recognized as safe'. This lax regulatory framework, shaped heavily by food-industry lobbying, stands in stark contrast to the stricter regimes in much of Europe and Latin America. Nutrition researcher Kevin Hall dispels widely held misconceptions about food.Credit: Mark Schiefelbein/AP Photo/Alamy In terms of solutions, the book argues that “personal responsibility” plays little part in nutrition — a point that, if pushed too rigidly, can feel disempowering. It's a powerful message, and one that deserves wider attention, even if the book's practical diet suggestions are underwhelming: eat less meat, eat more fruit and vegetables, and store junk food out of sight. Tim Spector is the co-founder of a science and nutrition company called ZOE. He has also published books on related topics, such as Food for Life: The New Science of Eating Well. Power and profit drive what we eat: here's why the food system needs a revolution Measles makes a comeback: four charts show where and how Vaccine advice based on science: US centre fills gaps in public-health information Gestational diabetes linked to autism in study: what scientists say Your risk of dying from chronic disease has dropped — if you live in these countries Hope for diabetes: CRISPR-edited cells pump out insulin in a person — and evade immune detection This scientist is breeding billions of mosquitoes to fight disease in Brazil Long-COVID research just got a big funding boost: will it find new treatments? The Dean of Systems Hub holds both an executive and faculty appointment and serves as a key member of the University's senior administrative team. The Hong Kong University of Science and Technology (Guangzhou) The Hong Kong University of Science and Technology (Guangzhou) Power and profit drive what we eat: here's why the food system needs a revolution An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/breakthrough-in-digital-screens-takes-color-resolution-to-incredibly-small/'>Breakthrough in Digital Screens Takes Color Resolution to Incredibly Small Scale</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 11:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Breakthrough in Digital Screens Takes Color Resolution to Incredibly Small Scale This shift has required higher image resolution (usually through increased pixel counts) to provide enough detail. Conventional light-emitting pixels work poorly below a certain size: brightness drops, and colors bleed. The same isn't true for reflective displays such as those used in many e-readers, whose pixels reflect ambient light rather than emitting their own—but creating those pixels typically requires larger components. A new reflective display could shatter those restrictions with resolutions beyond the limit of human perception. In a recent study in Nature, scientists describe a reflective retina e-paper that can display color video on screens smaller than two square millimeters across. The material is electrochromic, so its light absorption and reflection can be controlled with electrical signals. With this setup, “metapixels” consisting of the three subpixels can generate any color if you deliver appropriate signals. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. Each pixel is only 560 nanometers wide, creating a resolution above 25,000 pixels per inch—more than 50 times that of current smartphones. “We can make displays a similar size as your pupil, with a similar number of pixels as photoreceptors in your eyes,” says study co-author Kunli Xiong of Uppsala University in Sweden. Source: “Video‐Rate Tunable Color Electronic Paper with Human Resolution,” by Ade Satria Saloka Santosa et al., in Nature, Vol. E-paper screens also have relatively low energy requirements; the pixels retain their color for some time, so power is generally needed only when colors change. “For very small devices, it is not easy to integrate large batteries, so that energy saving becomes even more important.” “People have made these kinds of materials before, but usually they produce poor colors,” says Jeremy Baumberg, a nanotechnologist at the University of Cambridge, who studies how nanoscale materials interact with light. In comparison, the design of Xiong and his colleagues' subpixels “generates colors that look more compelling than I've seen before,” Baumberg says. Xiong and his colleagues anticipate that engineering companies will begin to develop such systems. Meanwhile Xiong's team plans to optimize other aspects of the technology such as its speed and lifetime. “Every time you switch [colors], the material's structure changes, and eventually it crumbles,” Baumberg says—similar to how batteries decay. He estimates that it'll be five to 10 years before we see commercially available devices. Simon Makin is a freelance science journalist based in the U.K. His work has appeared in New Scientist, the Economist, Scientific American and Nature, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01415-6'>T cell fate to enhance anti-tumour immunity and immunotherapy responses</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 11:39:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Reducing calorie intake through dietary restriction (DR) slows tumour growth in mammals, yet the underlying mechanisms are poorly defined. Here, we show that DR enhances anti-tumour immunity by optimizing CD8+ T cell function within the tumour microenvironment (TME). Using syngeneic xenograft tumour models, we found that DR induces a profound reprogramming of CD8+ T cell fate in the TME, favouring the expansion of effector T cell subsets with enhanced metabolic capacity and cytotoxic potential, while limiting the accumulation of terminally exhausted T cells. This metabolic reprogramming is driven by enhanced ketone body oxidation, particularly β-hydroxybutyrate (βOHB), which is elevated in both the circulation and tumour tissues of DR-fed mice. βOHB fuels T cell oxidative metabolism under DR, increasing mitochondrial membrane potential and tricarboxylic acid cycle-dependent pathways critical for T cell effector function, including acetyl-CoA production. By contrast, T cells deficient for ketone body oxidation exhibit reduced mitochondrial function, increased exhaustion and fail to control tumour growth under DR conditions. Importantly, DR synergizes with anti-PD1 immunotherapy, further augmenting anti-tumour T cell responses and limiting tumour progression. Our findings reveal that T cell metabolic reprogramming is central to the anti-tumour effects of DR, highlighting nutritional control of CD8+ T cell fate as a key driver of anti-tumour immunity. The balance between cancer cell proliferation and anti-tumour functions of the immune system influences tumour growth1. Immune checkpoint inhibitors (ICIs) have revolutionized the treatment landscape for various malignancies, in part by promoting the expansion of CD8+ T cell subsets that limit tumour progression4. However, given their central role in controlling tumour growth, CD8+ T cells are often the target of immune evasion and suppression mechanisms. Chronic exposure to tumour antigens and inflammatory conditions in the TME can promote CD8+ T cell dysfunction (also known as exhaustion), a terminally differentiated state characterized by reduced proliferative capacity, impaired cytokine production and increased expression of inhibitory receptors (that is, PD1, LAG3, TIM3) that limit T cell effector function5,6. The accumulation of terminally exhausted T cells (Tex cells) in tumours is driven by the transcription factor thymocyte selection-associated high mobility group box (TOX) and is reinforced through epigenetic programming7,8,9,10, ultimately limiting the ability of CD8+ T cells to control tumour growth. Understanding mechanisms that drive CD8+ T cell fate decisions in the TME is critical for developing new approaches to counter T cell dysfunction and overcome ICI resistance during tumour progression. Metabolism is a key determinant of CD8+ T cell function and survival that underlies effective anti-tumour immunity11. Activated CD8+ T cells rewire their metabolism to produce both energy (that is, ATP production) and the biosynthetic molecules required for cell proliferation and effector function, and they use both glycolysis and oxidative phosphorylation (OXPHOS) to fuel these processes12,13,14. T cell dysfunction is also influenced by environmental conditions in the TME that impact T cell metabolism15,16. Chronic antigen stimulation, hypoxic stress and PD1 signalling all promote metabolic and mitochondrial derangements in CD8+ T cells that precede their dysfunction17,18,19,20. Conversely, CD8+ T cell effector function is enhanced by exposure to nutrients that fuel mitochondrial oxidative metabolism and acetyl-Coenzyme A (acetyl-CoA) levels, including ketone bodies and acetate21,22,23. Nutrient levels in the TME are highly influenced by diet24. Therefore, dietary strategies that modulate metabolic conditions in the TME have the potential to influence anti-tumour immune responses—either in isolation or in combination with ICI treatments—by metabolic reprogramming of CD8+ T cell function. DR, a feeding regimen that reduces caloric intake without malnutrition, extends lifespan in mammals and delays the onset of age-related diseases, including cancer25,26,27,28,29. Dietary interventions that limit tumour growth—such as DR, fasting-mimicking diets or low glycemic diets—are presumed to act by altering growth factor (that is, insulin, IGF-1) signalling and/or nutrient availability (that is, glucose) for cancer cells30,31,32,33. Yet the contribution of the immune system to the anti-tumour effects of DR, either in isolation or in combination with ICI treatment, is poorly understood. Although malnutrition has deleterious effects on the immune system34,35, DR has been shown to enhance certain aspects of immune function36. Despite lower calorie intake, DR has been shown to enhance natural killer, myeloid and memory T cell responses in vivo37,38,39,40. Similarly, T cells isolated from calorie-restricted patients show enhanced proliferation and effector responses in vitro41. Here, we show that DR-driven changes in ketone body metabolism direct CD8+ T cell fate and function in the TME to suppress tumour growth. To assess the impact of lowering food intake on tumour growth in immunocompetent mice, we used an established model of DR in which daily food availability is reduced by 50% with no change to nutritional content (Supplementary Table 1)38. C57BL/6J mice were fed either ad libitum (AL) or subjected to 50% DR for 7 days before establishment of subcutaneous syngeneic tumour allografts (Fig. Consistent with previous reports, this 50% DR produced a marked reduction in fat mass with minimal loss of lean mass after 2 weeks (Fig. Strikingly, syngeneic melanoma (B16) and breast cancer (E0771) tumours grew more slowly in mice adapted to DR, resulting in a 30–80% extension of tumour-free survival (Fig. These results from DR are comparable to the delay in tumour growth observed in animals fed a fasting-mimicking diet42. a, Schematic representation of dietary interventions and tumour experimental design. b,c, EchoMRI analysis of mice on different dietary regimens: AL feeding or DR (50% reduction) for 2 weeks, showing the per cent change in total mouse body weight (b) and per cent change in fat mass and lean mass (c). Data are plotted as means; error bars, s.e.m. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. d,e, Growth of B16 melanoma (d) or EO771 breast cancer cells (e) in C57BL/6 mice fed an AL or DR diet. Right panels of each graph are spaghetti plots showing growth curves of individual tumour volumes over time. The darker lines indicate the average tumour growth for each diet group over time post tumour injection (PTI). Statistical significance was assessed by log-rank test. f, Histology of B16 tumours from AL-fed or DR-fed mice 14 days PTI. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. g, Bar graphs showing the percentage of PD1lo CD8⁺ T cells (Teff) isolated from B16-OVA melanoma tumours, AL versus DR. Data represent the means; error bars, s.e.m. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. h, Kaplan–Meier plot comparing tumour onset in B16 tumour-bearing WT and Rag2−/− mice on an AL or DR diet. Statistical significance was assessed by a log-rank test (n = 5 mice per group). i, Kaplan–Meier plot comparing time-to-humane endpoint in CD8+ T cell-depleted mice fed an AL or DR diet. Mice were treated with control (IgG) or CD8⁺ T cell depleting (anti-CD8) antibodies before tumour cell implantation. T cells have a critical role in controlling tumour cell growth. We quantified the impact of DR on CD8⁺ T cell infiltration into syngeneic EO771 breast cancer tumours by immunohistochemistry. DR-fed mice showed a marked increase in CD8+ tumour-infiltrating lymphocytes (TILs) compared to AL controls (Fig. Flow cytometry analysis of TILs from B16 melanoma tumours demonstrated a similar increase in the frequency of effector CD8⁺ (PD1lo) T cells in mice conditioned to DR versus AL-fed mice (Fig. Strikingly, the delay in tumour growth conferred by DR was lost in Rag2−/− mice, which lack mature T cells and B cells (Fig. Similarly, we observed no difference in B16 tumour growth in athymic nude mice that lack mature T cells, regardless of diet (Extended Data Fig. Together, these data indicate that the anti-tumour effects of DR require CD8⁺ T cells. To determine how DR impacts immune cells in the TME, we performed cellular indexing of transcriptomes and epitopes coupled to next-generation sequencing (CITE-seq) on live CD45+ cells sorted from B16 tumours from AL-fed or DR-fed mice. Although recent work suggests that DR can boost natural killer cell-mediated anti-tumour immunity39, tumour-infiltrating natural killer cell populations were similar between dietary conditions (Fig. Notably, the proportion of tumour-infiltrating CD45+ T cells was increased under DR, marked by an increase in the proportion of Gzmb-expressing CD8+ Teff cells (Fig. a, Weighted nearest-neighbour uniform manifold approximation and projection (wnnUMAP) of 45,455 CD45⁺ tumour-infiltrating cells (AL and DR combined) from B16 melanoma tumours (left; n = 4 mice per diet). b, Expression of GZMB in PD1lo CD8⁺ T cells isolated from B16-OVA melanoma tumours from AL-fed or DR-fed mice. Left, representative histograms of GZMB expression; right, bar graphs quantifying the percentage of CD8+ cells expressing GZMB. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. c, Representative histogram of LY108 expression in PD1lo CD8⁺ TILs from B16 tumours. Bar plots show relative geometric mean fluorescence intensity (gMFI) compared to AL controls. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. d, Mice were started on AL or DR diets 6 days before OT-I adoptive transfer. On day 0, mice were injected with LmOVA in the tail vein. At 7 dpi, spleens were dissected for downstream analyses, including effector function, cell expansion and metabolic function. A volcano plot depicting the proteomics data from OT-I CD8⁺ THY1.1⁺ T cells isolated as described is shown, with log2(fold change (FC)) on the x axis and −log10(P values) on the y axis. Statistical significance was determined using a −log10(P value) of 1.25 and −log2(FC) of 1.0 as cutoff values for data visualization. e, GSEA focusing on metabolic and functional pathways from MSigDB (Hallmark, C2 and C5) reveals top-enriched gene sets in the DR versus AL comparison (from proteomics data, same as in d). Bars indicate normalized enrichment score (NES; n = 3 mice per group). f, The relative abundance (z-score) of TCA cycle-derived metabolites in antigen-specific CD8+ T cells isolated from LmOVA-infected AL-fed or DR-fed mice at 7 dpi (n = 3 mice per group). g, Extracellular acidification rate (ECAR) plot of antigen-specific CD8⁺THY1.1⁺ T cells isolated from LmOVA-infected mice (7 dpi) under AL or DR conditions. h,i, Bioenergetic profile of AL-conditioned and DR-conditioned CD8⁺ T cells. h, Oxygen consumption rate (OCR) plot for antigen-specific CD8⁺ T cells isolated from LmOVA-infected AL-fed or DR-fed mice (7 dpi). Oligomycin (O), FCCP (F), rotenone and antimycin A (R/A) and monensin (M) were added to cells where indicated. i, Basal ATP production rates from OXPHOS and glycolysis (GLYC) for AL-conditioned or DR-conditioned CD8⁺ T cells from h. Data represent the means; error bars, s.d. Statistical significance was calculated with a two-way ANOVA multiple comparisons test. j, Mitochondrial membrane potential of AL-conditioned versus DR-conditioned CD8⁺ T cells. TMRM staining of CD8+ OT-I T cells isolated from the spleen of LmOVA-infected AL-fed or DR-fed mice (7 dpi). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. To determine the impact of DR on CD8+ Teff cell function(s) in the TME, we isolated TILs from excised B16 melanoma tumours from mice fed different diets. The majority of CD8+ TILs under DR conditions expressed low levels of the inhibitory receptor PD1 (Fig. Although interferon-γ (IFNγ) levels in PD1lo CD8+ TILs were similar between AL and DR conditions (Extended Data Fig. 2e), LY108/SLAMF6 expression was significantly higher in Teff cells under DR, suggesting a more stem-like phenotype (Fig. Collectively, these findings indicate that DR both expands the CD8+ Teff cell population within tumours and enhances features of cytotoxicity. We next evaluated mechanisms underlying the effects of DR on CD8+ T cell effector function. Animals on the DR regimen displayed normal levels of physical activity (Extended Data Fig. 3a) but consumed 50% less food and water on average (Extended Data Fig. 3b), resulting in weight loss that plateaued after 3–4 days (Extended Data Fig. EchoMRI analysis of mice subjected to various levels of DR (that is, 30%, 40% or 50% DR) for 2 weeks revealed greater body-weight loss that plateaued by day 4 (Extended Data Fig. Although 50% DR elicited the largest mean weight reduction, it was not significantly greater than that seen with 40% DR (Extended Data Fig. Importantly, weight loss under DR was driven almost entirely by fat-mass depletion—with >50% loss in mice on 50% DR—while lean mass loss was similar between the 40% and 50% DR groups (Extended Data Fig. Consistent with these effects on host metabolism, tumour onset was delayed under all DR regimens, with the greatest and most consistent protection seen with 50% DR (Extended Data Fig. Baseline immune profiling revealed an overall decrease in immune cell numbers in the spleens of DR-fed mice, although the proportion of CD4+ and CD8+ T cell populations between diets was not dramatically altered (Extended Data Fig. 4a,b and Supplementary Table 3), consistent with previous reports38,40. To control for systemic effects of DR on CD8+ T cell homoeostasis, we used a T cell adoptive transfer model coupled to Listeria monocytogenes (Lm) infection, which elicits robust CD8+ Teff cell responses in vivo21,43,44. Thy1.1+ OT-I T cell receptor transgenic CD8+ T cells were transferred into AL or DR-fed mice, followed by infection with attenuated Lm-expressing ovalbumin (LmOVA), and splenocytes extracted 7 days post infection (dpi) for ex vivo functional analysis (Extended Data Fig. Similar to CD8+ TIL from DR-fed mice, OT-I T cells responding to LmOVA infection under DR-fed conditions displayed increased effector function when analysed ex vivo, characterized by an increase in the percentage of IFNγ-producing T cells (Extended Data Fig. 5b), higher IFNγ production on a per-cell basis (Extended Data Fig. 5c) and an increased percentage of GZMB+ CD8+ Teff cells (Extended Data Fig. Elevated GZMB protein levels displayed by DR-conditioned OT-I T cells (Extended Data Fig. 5e) was consistent with the elevated GZMB production displayed by CD8+ TIL under DR (Fig. Ex vivo OT-I CD8+ T cells displayed increased effector function in DR-fed mice despite an overall reduction in expansion following LmOVA infection (Extended Data Fig. To identify mechanisms by which DR enhances CD8+ Teff cell function, we analysed changes in global protein levels in OT-I T cells isolated from AL-fed or DR-fed mice following LmOVA infection (as in Extended Data Fig. At 7 dpi, OT-I Teff cells isolated from DR-conditioned mice displayed enrichment of a distinct set of 277 proteins associated with anabolic growth and metabolism (that is, G2M checkpoint, E2F and MYC targets, interleukin-2 (IL-2) signalling) (Fig. 2d (with proteins first mapped to gene symbols) and the results in Supplementary Table 4, we performed gene set enrichment analysis (GSEA) against the Hallmark, C2 and C5 gene collections. This analysis revealed significant enrichment of the tricarboxylic acid (TCA) cycle pathway (Fig. Consistent with this observation, OT-I Teff cells isolated from DR-conditioned mice responding to LmOVA infection displayed increased levels of TCA cycle metabolites compared to control-fed animals (Fig. Seahorse analysis of OT-I Teff cells isolated from LmOVA-infected mice revealed an increase in both the extracellular acidification rate (a measure of glycolysis) and oxygen consumption rate (a measure of OXPHOS) in DR-conditioned Teff cells compared to AL-fed controls (Fig. This corresponded to significant increases in both oxidative and glycolytic ATP production in Teff cells from DR-conditioned mice compared to AL-fed controls (Fig. 2i) and almost a doubling in maximal ATP production capacity (Extended Data Fig. Moreover, OT-I Teff cells responding to LmOVA infection displayed increased mitochondrial membrane potential (as determined by TMRM staining) under DR conditions compared to AL-fed controls (Fig. Collectively, these data indicate that restricting food intake (by DR) enhances CD8+ Teff cell metabolic fitness through effects on the TCA cycle and OXPHOS. Our findings indicate that DR increases the proportion of functional CD8+ Teff cells within the TME. Therefore, we used our CITE-seq datasets to characterize how DR impacts CD8+ T cell subsets within TIL populations. Cell clusters were classified based on a combination of gene features, gene density, antibody-derived tags (ADTs) and MSigDB pathway analysis (Extended Data Fig. Cellular indexing of both protein epitopes and transcriptional profiling revealed two major CD8+ TIL populations in B16 tumours based on gene expression for Tox, a key transcriptional regulator of T cell exhaustion7,8 (Fig. The accumulation of Tex cells in tumours is driven by TOX and is reinforced through epigenetic programming7,8,9,10, ultimately limiting the ability of CD8+ T cells to control tumour growth. In B16 tumours, the Toxlo population consisted of five main subclusters, including two Mki67+ proliferating Teff cell populations (Prolif1 and Prolif2), two progenitor Tex cell populations (TexProg1, TexProg2), an intermediate exhausted (TexInt) population and a small naive-like cell population (Fig. Toxhi CD8+ T cells displayed elevated transcript levels of inhibitory receptor genes (Lag3+Havcr2+Pcdc1+) but clustered into two distinct groups: a conventional terminally exhausted (TexTerm) population and a proliferating (Mki67+) effector-like exhausted population (TexEff) (Fig. Fast gene set enrichment analysis (fgsea) identified the TexEff cluster as having the most effector-like properties, including enrichment for immune response and cell cycle genes (Fig. TexEff cells also displayed increased expression of genes encoding effector molecules (that is, Ifng, Tnfa and Gzmb) (Fig. 3b), despite elevated expression of transcripts encoding Tox and inhibitory receptors (that is, Pdcd1, Lag3, Havcr2) (Extended Data Fig. a, WnnUMAP of 7,005 activated (CD44+) CD8⁺ TILs from B16 melanoma tumours (combined AL and DR) (n = 4). Legend indicates unique T cell clusters called based on RNA expression and ADTs. T cell clusters with high and low Tox expression are highlighted by circles. b, Overlay of wnnUMAP for activated CD8+ TIL from a and MSigDB gene expression signatures for TexEff, cell cycle and Gene Ontology (GO) Immune Response pathways for TILs isolated from B16 tumours (combined AL and DR). Joint density plot indicates the highest combined expression/cell of Ifng, Tnfa and Gzmb among activated CD8⁺ T cells from B16 tumours. d, Stacked bar graphs showing the percentage of CD8⁺ T cell populations divided by Tox expression (Toxlo versus Toxhi). e, RNA velocity plots inferring cellular differentiation trajectory for CD8⁺ T cells infiltrating B16 tumours from AL-fed or DR-fed mice. Trajectories were derived from the dynamical prediction model scVelo, with representative directionality for T cells under each diet shown in the inset. f, PD1 and TOX expression in B16 tumour-infiltrating CD8⁺ T cells isolated from AL or DR mice 14 days PTI. Left, representative flow cytometry plots for PD1 versus TOX expression; right, bar graph showing the percentage of PD1⁺TOX⁺ CD8⁺ T cells isolated from tumours. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. g, Expression of stemness and exhaustion markers in TOX⁺ CD8⁺ T cells isolated from B16 tumours from AL-fed or DR-fed mice. Top, representative histograms of LY108, PD1 and TOX expression. Geometric mean fluorescence intensity averaged across all biological replicates is indicated. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. h, Expression of LY108 versus TIM3 among PD1⁺ CD8⁺ TILs from B16 tumours. Left, representative flow cytometry plots showing LY108 and TIM3 expression; right, bar graph showing percentages of progenitor (TexProg, LY108+TIM3−) and terminal (TexTerm, LY108−TIM3+) Tex cell subsets under AL and DR conditions. Statistical significance was calculated with a two-way ANOVA multiple comparisons test. i, PD1 and TOX expression in EO771 (breast cancer) tumour-infiltrating CD8⁺ T cells isolated from AL or DR mice 14 days PTI. Left, representative flow cytometry plots for PD1 versus TOX expression; right, bar graph showing the percentage of TOX⁺ CD8⁺ T cells isolated from tumours. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. j, CD44 and IFNγ expression in EO771 (breast cancer) tumour-infiltrating CD8⁺ T cells isolated from AL or DR mice 14 days PTI. Left, representative flow cytometry plots for CD44 versus IFNγ expression; right, bar graph showing the percentage of IFNγ-producing cells from PD1+TOX⁺ CD8⁺ T cells isolated from EO771 tumours. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. Single-cell hashtagging strategies allowed us to track CD8+ TIL populations in B16 tumours from either AL-fed or DR-fed mice. Analysis of CD8+ TIL populations revealed two major trends induced by DR feeding. First, for the Toxlo CD8+ TIL subset, we observed an expansion in proliferating Teff cell populations (Prolif1, Prolif2) and a decrease in TexInt cells in animals conditioned to DR (Fig. Second, the type of Tox-expressing Tex cells found in tumours was highly dependent on diet. Tumours from AL-fed mice contained more TexTerm CD8+ T cells, while tumours from DR mice contained a greater proportion of TexEff (Fig. To determine whether the diet-induced changes in Tex cell populations reflect differences in CD8+ T cell differentiation, we used RNA velocity analysis, a computational method that infers cellular differentiation trajectories based on the relative abundances of spliced and unspliced transcripts in single cells45. Under AL conditions, RNA velocity inference showed a directional flow away from the proliferating effector population towards TexInt and ultimately TexTerm populations (Fig. By contrast, under DR, the direction of CD8+ T cell trajectory culminated in the effector-like TexEff population (Fig. This change in differentiation trajectory corresponded to a major shift in the ratio of effector to terminally exhausted T cell populations in B16 tumours, moving from an equal ratio of effector (proliferating and TexEff) to TexTerm cells under AL-fed conditions to a 4:1 ratio under DR (Extended Data Fig. Collectively, these results suggest that DR alters CD8+ TIL fate, shifting CD8+ T cells away from terminal exhaustion and towards a more effector-like state that favours tumour control. We next used flow cytometry to characterize the impact of DR on CD8+ T cell function within the TME. Consistent with our CITE-seq analysis, the number of CD8+ TexTerm cells—as determined by reduced TOX and inhibitory receptor (PD1+TOX+) expression—decreased significantly in tumours from DR-conditioned animals compared to AL-fed mice 14 days post tumour implantation (Fig. TOX-expressing CD8+ TILs from DR-fed mice displayed increased expression of stemness markers (LY108/SLAMF6) while showing reduced features of terminal exhaustion (that is, lower PD1 and TOX expression) compared to AL-fed controls (Fig. Using conventional markers for terminally exhausted (LY108−TIM3+) versus progenitor exhausted (LY108+TIM3−) cells4, the majority of PD1+ CD8+ TILs in B16 tumours from DR-fed mice were progenitor exhausted or an intermediate phenotype (LY108+TIM3+) (Fig. To validate our observations in an independent tumour model, we analysed TILs from EO771 breast cancer tumours and found a similar reduction in PD1+TOX+ CD8+ T cells within tumours from DR-conditioned mice compared to AL-fed controls (Fig. Indeed, TOX-expressing CD8+ TILs from E0771 tumours grown in DR-fed mice exhibited increased IFNγ production (Fig. 3j) and elevated TCF1 levels, a marker of cell stemness (Extended Data Fig. These findings indicate that DR enhances both the frequency and functional capacity of effector CD8⁺ T cells, promoting a less exhausted, more stem-like phenotype across tumour models. CD8⁺ T cells rely on glycolysis and OXPHOS to fuel their proliferation and effector function12,13,14, but are capable of metabolizing a diverse set of metabolic substrates to fuel these processes21,46,47. Given the impact of DR on CD8+ T cell fate in the TME, we assessed whether DR alters the metabolic programming of CD8+ T cells needed for tumour control. We first conducted metabolism-focused GSEA on our CITE-seq datasets to define the metabolic features of CD8+ TIL subsets. This analysis revealed two distinct metabolically active CD8+ TIL populations in B16 tumours. CD8+ T cells with effector properties (Prolif1, TexEff) displayed the highest glycolytic signature (Fig. 4a), consistent with their high proliferative rate. Notably, TexEff cells—which preferentially expand in tumours under DR (Fig. 3a,b)—displayed the highest OXPHOS signature of all CD8+ TIL subsets (Fig. Moreover, like DR-conditioned Teff cells responding to Listeria infection (Fig. 2j), CD8+ TILs isolated from B16 tumours displayed increased TMRM staining (mitochondrial membrane potential) under DR conditions compared to CD8+ TILs from AL-fed controls (Fig. Therefore, the most metabolically active CD8+ TIL subsets—by gene expression signatures and mitochondrial membrane potential—are the effector cell populations enhanced by DR treatment. a, Overlay of wnnUMAP for 7,005 activated (CD44+) CD8⁺ TIL from B16 melanoma tumours (both AL and DR) and MSigDB gene expression signatures for organic oxidation and glycolysis pathways. b, Mitochondrial membrane potential of AL-conditioned and DR-conditioned CD8⁺ T cells. TMRM staining (gMFI) of CD8+ OT-I TIL isolated from B16 tumours from AL-fed or DR-fed mice (14 days PTI). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. c, Volcano plot showing the log2(fold change) in metabolite and lipid abundance in B16 tumours isolated from AL versus DR mice (n = 3–4 mice per group). Select metabolites enriched in AL and DR tumours are annotated. Statistical significance was determined using a −log10(P value) of 1.30 as a cutoff value for data visualization. d, Ketone body levels in the serum and tumours of AL-fed versus DR-fed mice after 21 days on diet (14 days of tumour growth). Left, βOHB abundance in B16 tumours from AL-fed versus DR-fed mice as quantified by mass spectrometry; right, ketone body abundance in serum as quantified by enzyme assay. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. e,f, Bioenergetic profile of AL-conditioned and DR-conditioned CD8⁺ T cells exposed to βOHB. e, OCR plot for antigen-specific CD8⁺ T cells isolated from LmOVA-infected AL-fed or DR-fed mice (7 dpi). T cells were cultured with or without 1.5 mM βOHB 60 min before the start of the assay. f, Maximum ATP production rates from OXPHOS for CD8⁺ T cells from e. Data represent the means; error bars, s.d. g, Schematic depicting 13C labelling patterns in acetyl-CoA (Ac-CoA) and TCA cycle intermediates from U-[13C6]-glucose and [2,4-13C2]-βOHB. h–j, 13C labelling of TCA cycle intermediates in AL-conditioned versus DR-conditioned CD8⁺ T cells. CD8⁺ T cells isolated from LmOVA-infected AL-fed or DR-fed mice (7 dpi) were cultured for 2 h ex vivo in VIM medium containing 5 mM U-[13C6]-glucose and 1.5 mM [2,4-13C2]-βOHB. Shown is the per cent incorporation of U-[13C6]-glucose-derived carbon (M + 2) (h) and [2,4-13C2]-βOHB-derived carbon (M + 1) (i) into citrate and malate. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. k, Bar graph of Ac-CoA abundance in CD8⁺ T cells isolated from LmOVA-infected AL-fed or DR-fed mice (7 dpi). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. Therefore, we reasoned that DR impacts CD8+ T cell metabolism, in part, by altering the availability of specific nutrients in vivo. Indirect calorimetry revealed that mice conditioned to DR displayed a significant decrease in their respiratory exchange ratio during the fasted—but not the fed—state (Extended Data Fig. 7a), indicating increased lipid oxidation in DR-fed mice. We next used mass spectrometry to profile diet-induced alterations in lipid and metabolite abundance in B16 tumours. We observed a general decrease in fatty acid and lipid abundance in DR tumours compared to AL-fed controls (Fig. This change in tumour lipid abundance is consistent with lipid mobilization from adipose tissue and a decrease in respiratory exchange rate, favouring oxidation triggered by DR30,38 (Extended Data Fig. By contrast, only a small number of metabolites were enriched in tumours from DR-treated mice, including increased abundance of the ketone body βOHB (Fig. Elevated βOHB levels were observed in both tumours and serum from DR-fed mice compared to AL-fed mice (Fig.4d). The increase in circulating and intratumoral βOHB levels under DR was notable, given recent evidence linking ketone body metabolism to enhanced T cell effector function22,49. Using Seahorse bioenergetic analysis and 13C-labelled metabolic tracers, we observed a striking difference in βOHB use by CD8+ Teff cells based on diet. First, in ex vivo assays, augmenting DR-conditioned OT-I Teff cells with βOHB further increased their maximal oxygen consumption rate (Fig. This change corresponded to an approximately fourfold increase in maximal oxidative ATP production capacity compared to CD8+ Teff cells from AL-fed animals (Fig. Next, we cultured OT-I Teff cells from LmOVA-infected mice ex vivo in physiologic medium (VIM)21 containing fully labelled [13C6]-glucose and partially labelled [2,4-13C2]-βOHB. In this strategy, breakdown of [13C6]-glucose generates M + 2 labelled TCA cycle intermediates, whereas [2,4-13C2]-βOHB generates M + 1 labelled intermediates (Fig. 4g), thereby facilitating a direct comparison of the contribution of each fuel type to TCA cycle metabolism22. We observed a small but significant increase in [13C6]-glucose-derived citrate (M + 2) in DR-conditioned CD8+ Teff cells (Fig. 4h and Supplementary Table 8); however, citrate labelling from [2,4-13C2]-βOHB (M + 1) doubled in CD8+ Teff cells from DR compared to AL-fed mice (Fig. DR-conditioned OT-I Teff cells also displayed a lower ratio of [2,4-13C2]-βOHB-labelled malate-to citrate (Fig. Consistent with these observations, steady-state acetyl-CoA levels were twofold higher in CD8+ Teff cells from DR-fed mice relative to T cells from AL-fed controls (Fig. Thus, DR increases both systemic βOHB availability and its use by CD8+ T cells. Chronic exposure of CD8+ T cells to tumour antigens in the TME is a precursor to terminal exhaustion. We therefore modelled this process using an in vitro chronic stimulation system and assessed metabolic changes on T cell self-renewal and terminal differentiation (Extended Data Fig. Chronic stimulation of in vitro-activated CD8+ T cells with anti-CD3 and anti-CD28 antibodies promotes several features of T cell exhaustion, including increased expression of TOX and inhibitory receptors (that is, PD1, TIM3) (Fig. 8c), compared to acutely stimulated CD8+ T cells maintained in IL-2. Notably, chronically stimulated CD8+ T cells displayed increased abundance of TCA cycle intermediates compared to acutely stimulated T cells (Fig. a, Inhibitory receptor expression on in vitro-stimulated T cells. Representative histograms of exhaustion marker expression (PD1, TOX) in CD8⁺ T cells under acute and chronic stimulation conditions. b, Heatmap showing the relative abundance (z-score) of TCA cycle-derived metabolites from CD8⁺ T cells exposed to acute versus chronic stimulation with anti-CD3 and anti-CD28 antibodies in vitro (n = 3 mice per group). c, Bar graph depicting total abundance of 13C-labelled βOHB in acute versus chronic stimulated CD8⁺ T cells. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. d, Bar graph depicting [U-13C6]-glucose labelling into glucose-6-phosphate (G6P), M + 6, in acute and chronic stimulated CD8+ T cells. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. e,f, 13C labelling of TCA cycle intermediates in CD8⁺ T cells exposed to chronic antigen stimulation. CD8⁺ T cells exposed to acute versus chronic stimulation were cultured in VIM medium containing 0.85 mM [13C4]-βOHB for 2 h. e, Total abundance of [13C4]-βOHB-derived citrate and malate. f, Ratio of [13C4]-βOHB-labelled malate (M + 2) to citrate (M + 2) for T cells from e. Data represent the means; error bars, s.e.m. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. g, Bar graph showing [13C2]-acetate into UDP-GlcNAc (M + 2) in acute and chronically stimulated CD8+ T cells, as well as [13C4]-βOHB into UDP-GlcNAc (M + 2) acute and chronically stimulated CD8+ T cells (total abundance). Right, bar graph showing [13C2]-acetate into Ac-Met (M + 2) in acute and chronically stimulated CD8+ T cells as well as [13C4]-βOHB into Ac-Met (M + 2) acute and chronically stimulated CD8+ T cells (total abundance). Statistical significance was calculated via two-way ANOVA with multiple comparisons. Shown are unique clusters for effector (Teff), memory (Tmem), terminally exhausted (TexTerm) and proliferating exhausted (TexEff) CD8⁺ T cell populations. i–k, Violin plots of gene expression across human CD8⁺ TIL subsets. i, Expression of MKI67 human gene between CD8+ T cell subsets. Statistical significance was assessed by one-way ANOVA with Dunnett's multiple comparisons test and a 5% significance level. k, Violin plots of BDH1 and OXCT1 gene expression between different human CD8+ T cell subsets. 2h), leading us to speculate that chronic antigen stimulation alters T cell nutrient use. Using different 13C-labelled metabolic tracers at their normal physiologic concentrations21, we found that βOHB and glutamine were the greatest contributors to citrate production under chronic stimulation in vitro (Extended Data Fig. 8d), with 13C-βOHB contributing predominantly to the first turn of the TCA cycle (M + 2 citrate; Extended Data Fig. In vitro chronic stimulation promoted increased 13C-βOHB uptake by CD8+ T cells compared to controls (Fig. 5c and Supplementary Table 9), while 13C-labelled glucose uptake and entry into glycolysis (measured by formation of 13C-labelled glucose-6-phosphate) was unchanged between conditions (Fig. Chronically stimulated T cells displayed higher βOHB oxidation in the TCA cycle as determined by increased levels of 13C-βOHB-derived citrate and malate compared to controls (Fig. Although both acetate and βOHB are major fuels for citrate production in Teff cells—contributing over 60% of the carbon for citrate (M + 2) production in Teff cells (Extended Data Fig. 8e)—only citrate production from βOHB was maintained under chronic stimulation conditions (Extended Data Fig. Ketone bodies and acetate are two major fuel sources for acetyl-CoA production in CD8+ T cells22,51, with Tex cells losing the ability to process acetyl-CoA from acetate52. In vitro chronically stimulated CD8+ T cells displayed a reduced malate-to-citrate ratio (Fig. 5f and Supplementary Table 9), suggesting a net export of βOHB-derived citrate to the cytosol for acetyl-CoA production like DR-conditioned Teff cells (Fig. Consistent with this concept, we observed increased βOHB-derived acetyl-CoA production in chronically stimulated versus acutely activated CD8+ T cells based on increased acetyl (M + 2) labelling of metabolites, including the nucleotide sugar UDP-GlcNAc and acetyl-methionine (Fig. Therefore, chronic stimulation promotes metabolic rewiring of oxidative metabolism in CD8+ T cells, with βOHB serving as a central fuel for TCA cycle metabolism and acetyl-CoA production. One of the defining features of CD8+ TILs from DR tumours is the expansion of proliferating TOX+ Tex cells with effector-like properties (that is, TexEff cells; Fig. To define the metabolic properties of these cells, we analysed transcriptional profiles of CD8+ TIL subsets from human tumours. Similar to CD8+ TILs from mouse tumours (Fig. 3), we identified CD8+ Teff cells and two Tex TIL clusters from human tumours (Fig. TexEff cells from human tumours most closely associated with the TexEff TIL cluster in mice (Fig. Proliferating TexEff cells (human TIL) displayed the highest ketone body metabolism (ketolysis) gene signature of all human TIL subsets (Fig. 5j), driven by increased expression of βOHB dehydrogenase (BDH1), which encodes the first rate-limiting enzyme required for βOHB breakdown, and 3-oxoacid CoA-transferase 1 (OXCT1), which converts βOHB to acetoacetate (AcAc) (Fig. Notably, both human Teff and TexEff cells displayed significant increases BDH1 transcript expression (Fig. 5k), which was associated with βOHB oxidation by both acutely activated and chronically stimulated CD8+ T cells (Extended Data Fig. Therefore, ketolysis is a metabolic signature of human TILs with effector-like properties. Next, we assessed the contribution of βOHB metabolism by T cells to the anti-tumour effects of DR. To test this effect, we generated mice with T cell-specific deletion of the enzymes required to process ketone bodies—βOHB dehydrogenase 1 (BDH1) and succinyl-CoA:3-ketoacid CoA-transferase (SCOT; encoded by Oxct1)—by crossing Cd4-Cre transgenic mice to mice with floxed alleles targeting the Bdh1 and Oxct1 genes (Bdh1fl/fl; Oxct1fl/fl Cd4-Cre; Fig. BDH1 mediates the conversion of βOHB to AcAc, while SCOT converts AcAc to acetoacetyl-CoA, ultimately leading to the production of acetyl-CoA that enters the TCA cycle (Fig. Depletion of BDH1 and SCOT proteins in double-knockout (DKO) T cells (from Bdh1fl/fl; Oxct1fl/fl Cd4-Cre+ mice) was confirmed by immunoblot (Fig. Deletion of BDH1 and SCOT ablated the βOHB-induced increase in oxidative ATP production in CD8⁺ T cells (Extended Data Fig. Mitochondrial mass and mitochondrial membrane potential were similar between activated wild-type (WT) and DKO CD8⁺ T cells (Extended Data Fig. 9b,c), as were cell viability and proliferation kinetics over 72 h of activation (Extended Data Fig. EchoMRI analysis of control and DKO mice subjected to 50% DR for 2 weeks revealed similar changes in body weight, fat mass and lean mass, regardless of genotype (Extended Data Fig. Finally, ex vivo [13C6]-glucose and [13C4]-βOHB tracing in antigen-specific CD8+ T cells (OT-I) isolated from DR-fed LmOVA-infected hosts confirmed that DKO T cells retain the ability to use [U-13C6]-glucose for TCA cycle metabolism but fail to channel [U-13C4]-βOHB into TCA intermediates (Fig. These results establish that deleting Bdh1 and Oxct1 genes in CD8⁺ T cells specifically impairs their ability to oxidize βOHB, while maintaining whole-body metabolic responses to DR feeding. a, Schematic detailing the generation of Bdh1fl/fl; Oxct1fl/fl Cd4-Cre mice. Bdh1-floxed mice were crossed with Oxct1-floxed mice to generate double-floxed mice expressing Cre recombinase under the Cd4 promoter, resulting in T cells unable to metabolize ketone bodies. AcAc is then converted to acetoacetyl-CoA by SCOT1 (Oxct1), ultimately leading to acetyl-CoA production for entry into the TCA cycle and ATP generation. Protein levels of α-tubulin are shown as a loading control. This was repeated three times independently with similar results. d, 13C labelling of TCA cycle intermediates in control (WT) and ketolysis-deficient (DKO) CD8⁺ T cells under DR conditions. Bar graph showing total labelling (pool size) U-[13C4]-βOHB labelling into citrate (left) and malate (right) in WT versus DKO CD8⁺ T cells isolated from DR-fed LmOVA-infected mice (7 dpi). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. e, Bioenergetic profile of control (WT) and ketolysis-deficient (DKO) CD8⁺ T cells under DR conditions. Left, OCR plot of WT and DKO OT-I T cells isolated from DR-fed LmOVA-infected mice (7 dpi); right, bar graph showing maximal ATP production rates from OXPHOS for WT versus DKO CD8⁺ T cells. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. f, Mitochondrial membrane potential of control (WT) versus ketolysis-deficient (DKO) CD8⁺ T cells under DR conditions. Bar plot showing TMRM staining of CD8⁺ T cells isolated from LmOVA-infected AL-fed or DR-fed mice (7 dpi). g, Kaplan–Meier plot comparing tumour onset (tumour volume, ≥250 mm3) in B16 tumour-bearing WT versus DKO mice fed an AL or DR diet as in Fig. Statistical significance was assessed by log-rank test with Bonferroni correction (n = 16–19 mice per group). h, Representative flow cytometry plots showing TOX and PD1 expression in CD8+ T cells from B16-OVA melanoma tumours under AL or DR conditions at 14 dpi. i,j, Enhanced exhaustion of DKO CD8⁺ T cells under DR-fed conditions. i, Bar plot showing the percentage of PD1⁺TOX⁺ CD8⁺ T cells isolated from B16 tumours from WT versus DKO mice under AL-fed or DR-fed conditions. j, Representative histograms of TOX expression in CD8⁺ TIL isolated from B16 tumours from WT and DKO mice fed under AL or DR conditions. Inset, gMFI values for TOX expression averaged across all biological replicates. (n = 8 AL-WT, 8 AL-DKO, 10 DR-WT and 9 DR-DKO mice per group). Statistical significance was calculated via two-way ANOVA multiple comparisons. We next asked whether ketone-body oxidation fuels the increase in CD8⁺ T cell bioenergetics induced by DR. Seahorse analysis of control (WT, from Cre− Bdh1fl/flOxct1fl/fl mice) or DKO (Cre+) OT-I CD8+ T cells isolated from LmOVA-infected mice demonstrated that loss of BDH1 and SCOT reversed the boost in OXPHOS and ATP production induced by DR feeding (Fig. TMRM staining of WT and DKO CD8+ Teff cells revealed that the increase in mitochondrial membrane potential induced by DR was also dependent on T cell-intrinsic ketolysis (Fig. Together, these data indicate that T cell-intrinsic ketone body oxidation drives the heightened bioenergetic capacity of CD8+ T cells induced by DR feeding. Finally, we assessed the contribution of T cell-intrinsic ketolysis to the anti-tumour effects of DR by challenging control (that is, WT) or DKO mice with syngeneic B16 melanoma tumours. We observed no difference in B16 melanoma tumour growth between DKO and control mice under AL conditions; however, tumour growth was significantly accelerated in DKO mice specifically under DR conditions (Fig. Analysis of CD8+ TILs from these mice revealed that the diet-induced change in CD8+ T cell fate within tumours was altered when T cells could not process ketone bodies. Tumours grown in DKO mice displayed a greater accumulation of PD1hiTOXhi Tex cells under DR when compared to WT animals on the same diet (Fig. 6h,i), with ketolysis-deficient (that is, DKO) CD8+ TILs maintaining high TOX levels even under DR feeding conditions that normally reduce TOX expression in WT CD8+ TILs (Fig. Together, these data link T cell-intrinsic ketone body oxidation to the anti-tumour effects of DR, with CD8+ TILs unable to process ketone bodies displaying increased features of exhaustion. Overcoming resistance to ICI therapy is a major challenge for cancer treatment. Given that DR enhances T cell-mediated control of tumour growth, we tested the efficacy of combining DR with anti-PD1 immunotherapy in mice bearing B16 melanoma tumours, which display resistance to PD1 blockade53. AL-fed or DR-fed C57BL/6J mice were administered anti-PD1 or IgG control antibodies beginning once tumours were palpable (Extended Data Fig. Anti-PD1 treatment promoted a small but significant increase in tumour-free survival under AL conditions; however, DR feeding greatly enhanced the efficacy of PD1 blockade, extending the tumour-free survival of animals over both anti-PD1 treatment under AL feeding and DR feeding alone (Fig. Strikingly, ~15% of animals on DR treated with anti-PD1 immunotherapy remained tumour-free after 80 days (Fig. a, Kaplan–Meier plot comparing B16 melanoma tumour onset in AL-fed or DR-fed mice that received anti-PD1 or IgG control antibodies by intraperitoneal injection (200 μg per dose). Antibody treatment was administered every 3 days for a total of five injections, beginning 7 days PTI (n = 11–20 mice per group). Statistical significance was assessed by a log-rank test with Bonferroni correction. b, PD1 and TOX expression in B16 tumour-infiltrating CD8⁺ T cells from DR-fed mice treated with anti-PD1 immunotherapy. DR-fed mice harbouring B16 tumours were administered anti-IgG or anti-PD1 antibodies (200 μg intraperitoneally) every 3 days for five doses, beginning on day 7 PTI. CD8⁺ T cells were isolated from B16 melanoma tumours at 21 days PTI. Representative flow cytometry plots for TOX versus PD1 expression. c, Bar plot representing the per cent of Teff cells (PD1−TOX−CD8+ TILs). Bottom, bar plot representing the per cent of TexTerm cells (PD1+TOX+CD8+ TILs). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. d, Representative histograms of GZMB expression of Teff from DR-fed mice harbouring B16 tumours that were administered anti-IgG or anti-PD1 antibodies. Right, bar graph showing the percentage of PD1−TOX− CD8⁺ T cells (Teff) expressing GZMB following IgG or anti-PD1 treatment. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. e, Representative flow cytometry plots of TNF versus IFNγ expression of PD1+TOX+ CD8+ TILs (TexTerm) from DR-fed mice harbouring B16 tumours that were administered anti-IgG or anti-PD1 antibodies. Right, bar graphs quantifying the percentage of PD1+TOX+ CD8⁺ T cells expressing IFNγ and TNF (IFNG⁺TNF⁺). Statistical significance was calculated with a two-tailed, unpaired Student's t-test. f, Representative histograms of GZMB expression of TexTerm T cells from DR-fed mice harbouring B16 tumours that were administered anti-IgG or anti-PD1 antibodies from a. Inset, gMFI values for GZMB. Right, bar graph showing the percentage of GZMB production from TexTerm CD8+ TILs that were previously treated with anti-IgG or anti-PD1 from a. Statistical significance was calculated with a two-tailed, unpaired Student's t-test. g, Kaplan–Meier analysis of tumour-free survival in dietary-restricted WT and ketolysis-deficient DKO mice (Cd4-Cre driven) treated with anti-PD1 or isotype control IgG. DR-conditioned WT (blue) and DKO (red) mice bearing subcutaneous B16 melanoma allografts received either control IgG (solid lines) or anti-PD1 antibody (dashed lines) once tumours reached 7 days PTI. Survival is plotted as days PTI. Statistical significance was assessed by a log-rank test. Proliferative Tex cells expressing LY108/SLAMF6 are highly responsive to ICI therapy4,54,55. Given that DR synergizes with anti-PD1 treatment to slow tumour growth (Fig. 7a), we investigated the impact of DR on anti-tumour CD8+ T cell populations following ICI treatment. B16 tumour-bearing C57BL/6J mice subjected to DR were administered either anti-PD1 or control IgG antibodies 7 days post tumour implantation, and CD8+ TIL phenotypes were analysed after 15 days of treatment (as in Extended Data Fig. Combining DR with anti-PD1 treatment shifted the effector versus exhausted CD8+ TIL populations within the tumours (Fig. Specifically, PD1 blockade promoted an increase in CD8+ Teff (PD1loTOXlo) cells and an approximately threefold decrease in the frequency of CD8+ Tex (TOX+PD1+) cells within tumours compared to DR treatment alone (Fig. PD1lo TILs from anti-PD1-treated mice displayed elevated GZMB production on a per-cell basis (Fig. Conversely, TOX-expressing CD8+ TILs from DR-fed mice were more functional following anti-PD1 treatment, expressing lower levels of TOX (Extended Data Fig. 10e) and displaying increased cytokine polyfunctionality (that is, IFNγ+TNF+ cells; Fig. Importantly, the synergy between anti-PD1 treatment and DR feeding was abolished when T cells were unable to process ketone bodies; under DR feeding, DKO mice treated with anti-PD1 antibodies showed no extension in tumour-free survival compared with IgG controls (Fig. These findings reveal that ketone body oxidation is essential for diet-induced enhancement of CD8⁺ T cell-mediated anti-tumour responses triggered by immune checkpoint blockade. The tumour-suppressive effects of reduced calorie intake have long been presumed to act directly on cancer cells. Here, we demonstrate that DR works more broadly to limit tumour growth by stimulating CD8+ T cell-mediated anti-tumour immunity. We show that DR induces a profound reprogramming of CD8+ T cells in the TME, promoting the expansion of tumour-controlling effector (Teff-like and TexEff) cells while limiting terminal T cell exhaustion. DR enhances CD8+ T cell-mediated anti-tumour immunity by increasing circulating ketone body levels, which in turn enhance TCA cycle metabolism and mitochondrial bioenergetics of CD8+ T cells. T cells that cannot metabolize ketone bodies display metabolic deficits, undergo premature exhaustion and fail to control tumour growth under DR conditions, identifying T cell-intrinsic ketolysis as a central mechanism for the anti-tumour effects of DR. These findings, along with our previous research identifying ketolysis as a regulator of CD8+ T cell cytolytic function22, suggest that DR regulates a nutrient-sensitive checkpoint within the TME—mediated by ketone bodies—that promotes the expansion of tumour-controlling effector (Teff-like and TexEff) T cells over terminal exhaustion, thereby improving tumour control. Overall, our study highlights how altering systemic nutrient availability through diet can influence CD8+ T cell fate within tumours to limit cancer progression. βOHB is preferentially oxidized over glucose for ATP production in CD8+ T cells and boosts T cell effector responses22,49. Our data indicate that ketolysis is a metabolic signature of Tex cells and that βOHB uptake and oxidation is prioritized by chronically stimulated CD8+ T cells to fuel bioenergetic and biosynthetic processes. Increased BDH1 expression in CD8+ T cells, which we observed in TILs from human tumours, facilitates greater βOHB oxidation in TILs as they infiltrate solid tumours. Consistent with this observation, CD8+ T cells that are adapted to DR display increased mitochondrial membrane potential and oxidative ATP production. This metabolic advantage promotes the expansion of TexEff cells, which retain proliferative capacity and effector function, over TexTerm cells, which have high inhibitory receptor expression and diminished functionality. These metabolic advantages are lost in ketolysis-deficient (that is, DKO) T cells, contributing to their premature exhaustion. Thus, DR aligns systemic nutrient supply with the metabolic needs of T cells at the tissue level. Acetyl-CoA is a critical metabolite for CD8+ T cell effector function, and failure to produce sufficient levels of cytosolic acetyl-CoA is a precursor for terminal exhaustion52. Additionally, diet directly impacts acetyl-CoA homoeostasis in CD8+ T cells, as acetyl-CoA levels in T cells double under DR-fed conditions. Our data indicate that chronic T cell receptor stimulation enhances acetyl-CoA production through βOHB oxidation and suggest that DR may enhance acetyl-CoA production through this route by increasing circulating ketone body levels. Acetyl-CoA has functions in addition to biosynthetic growth (that is, de novo lipogenesis) and energy production: it is the limiting substrate for histone acetylation reactions that regulate T cell differentiation and effector function51,57. Our results argue that nutritional regulation of ketone body metabolism is a critical determinant of CD8+ T cell fate in the TME, shifting differentiation between terminally exhausted (TexTerm) and effector-like (TexEff) states. Consistent with this idea, pantothenate/CoA increases CD8+ T cell differentiation towards effector lineages to enhance tumour control23. We speculate that dietary interventions or therapeutics that boost acetyl-CoA production in T cells, such as DR, limit tumour growth by driving CD8+ T cell differentiation towards effector lineages and away from dysfunctional states. The mammalian immune system evolved under cyclical periods of feast and famine. Immune challenges such as infection and cancer further impact systemic nutrient availability by disrupting feeding behaviours and remodelling host metabolism58,59,60. Mobilizing stored energy from adipose tissue into ketone body production provides the host with a versatile fuel to maintain acetyl-CoA production under conditions of metabolic stress61,62. In this vein, we speculate that T cells evolved the use of ketolysis to buffer against metabolic perturbations that negatively impact T cell bioenergetics and function. Our findings reveal the potential to exploit this system through nutritional interventions that enhance anti-tumour immunity. PD1 blockade amplifies the anti-tumour effects of DR by promoting the expansion of effector T cells (Teff and TexEff cells) to limit tumour growth, highlighting the potential to enhance the efficacy of immunotherapies through nutritional intervention. Using DR in clinical settings may face challenges regarding feasibility owing to patient health and compliance. However, combining ICIs with pharmacological agents that reduce appetite and food intake, such as GLP-1 agonists like semaglutide (that is, Ozempic), may mimic some metabolic effects of DR without necessitating strict dietary regimens63. Our findings also have potential implications for adoptive T cell therapies, such as chimeric antigen receptor T cell therapy, in which the metabolic state of T cells before infusion is critical for their persistence, functionality and anti-tumour efficacy in patients64. Exposing chimeric antigen receptor T cells to a DR-like environment during expansion or enhancing their capacity to oxidize ketone bodies may improve their metabolic fitness and resistance to terminal exhaustion in vivo, thereby increasing therapeutic efficacy. Understanding how dietary interventions such as DR impact T cell metabolism and differentiation fate in the TME may help inform evidence-based nutritional guidelines that complement or enhance current cancer immunotherapy strategies. Our findings establish a role for DR in reprogramming CD8⁺ T cell fate and function in transplantable syngeneic tumour models (that is, B16 melanoma, EO771 breast cancer). Although these models are useful for mechanistic evaluation of anti-tumour T cell responses, they do not fully capture tumour evolution, stromal remodelling or the cellular and antigenic heterogeneity characteristic of human tumours. The effects of DR on anti-tumour T cell function in spontaneously arising or genetically engineered mouse tumour models, therefore, remain to be determined. Although we demonstrate that CD8⁺ T cells are required for the anti-tumour effects of DR, the Cd4-Cre model used in our study deletes Bdh1 and Oxct1 in both CD4⁺ and CD8⁺ T cell lineages; therefore, we cannot exclude a contribution of ketolysis in CD4⁺ T cells to the observed effects. In addition, metabolic analysis of CD8⁺ TILs was limited by the number of cells that could be recovered from tumours. To overcome this limitation, we used our established LmOVA infection model to examine the metabolism of DR-conditioned Teff cells ex vivo; however, this system may not fully capture the nutritional and environmental cues of the TME under DR. We also show that DR elevates circulating and intratumoral βOHB, but the relative impact of DR compared to other methods of increasing ketones (for example, exogenous ketone esters or ketogenic diets) on CD8⁺ T cell anti-tumour function remains unclear. Finally, although effector-like Tex cells isolated from human tumours display a prominent ketolysis signature, the available datasets lack donor-matched dietary and metabolic metadata (for example, fasting state, BMI, βOHB levels). Consequently, these data cannot be used to establish a causal relationship between diet and TIL phenotype in patients. All research described in this study was conducted in accordance with relevant institutional and national ethical guidelines and regulations. All animal experiments were approved by the Institutional Animal Care and Use Committee of the Van Andel Institute (VAI) and performed in compliance with institutional and federal regulations governing the humane treatment of research animals. All mice were bred and housed under specific-pathogen-free conditions at VAI, following approved protocols. Genotyping was conducted using DNA extracted from tail or ear biopsies, with primer sets listed in the Key Resources Table. The study included both male and female mice aged 8–14 weeks. B16-F10 murine melanoma cells expressing OVA (B16-OVA67) and EO771 breast cancer cells (CRL-3461) were cultured in DMEM from Wisent, supplemented with 10% heat-inactivated FBS, 1% penicillin–streptomycin (Gibco) and 6 mM L-glutamine. All cell cultures were maintained in a humidified incubator at 37 °C with 5% CO2. Male and female C57BL/6J, Bdh1fl/fl; Oxct1fl/fl (WT), or Bdh1fl/fl; Oxct1fl/fl Cd4-Cre (DKO) mice were maintained on a 5010 diet. Mice fed AL were allowed free access to food at all times and never ran out of food. The mice on the DR regimen received 50% of their average daily (normal) intake of food. Mice on DR were given pre-weighed pellets between 09:00 h and 11:00 h every day of the experiment. Following 1 week on their respective diet, mice were subcutaneously injected with 2.5 × 105 (EO771) or 5 × 105 (B16-OVA) cells into the right abdominal flank. Tumour volume was measured every 2–3 days with a calliper once tumours became palpable. Mice were euthanized when the tumour volume reached 1,500 mm3. For experiments involving anti-PD1 treatment, mice received 200 µg of IgG (BP0091; RRID: AB_1107773) or anti-PD1 (BP0033-2; RRID:AB_1107747) antibodies intraperitoneally every 3 days, for a total of five injections (1 mg of antibody total). Treatment was initiated 7 days after tumour cell injection. The maximal tumour size permitted by the ethical regulations was 1,500 mm3, and this limit was not exceeded in any experiment. Animals were monitored regularly for signs of discomfort or distress, and any animal reaching predefined humane endpoints was euthanized in accordance with approved protocols. TIL were isolated from palpable tumours 12–14 days post tumour cell injection. The single-cell suspension was incubated with red blood cell lysis buffer for 1 min at 20–22 °C (room temperature), after which three volumes of T cell media (TCM) were added to halt the lysis reaction. Cells were collected by centrifugation at 500 RCF for 5 min at 4 °C and then resuspended in 0.5–1 ml of TCM and processed for flow cytometry. CD8+ T cells were purified from mouse spleens through negative selection using magnetic bead-based isolation kits (StemCell Technologies). T cells were cultured in TCM: Iscove's Modified Dulbecco's Medium (IMDM; Wisent) containing 10% Nu-Serum IV culture supplement (Corning), 50 U ml−1 penicillin, 50 µg ml−1 streptomycin (Gibco), 50 µM 2-mercaptoethanol (Gibco) and 2 mM L-glutamine (Gibco; final concentration 6 mM). T cells (1 × 106 cells per ml) were activated in vitro by stimulation with plate-bound anti-CD3ε (clone 145-2C11; 2 µg ml−1) and anti-CD28 (clone 37.51; 1 µg ml−1) antibodies for 48 72 h. After activation, cells were cultured in IMDM with 50 U ml−1 recombinant murine IL-2 (Peprotech) and re-seeded at 4 × 105 cells per ml in fresh medium supplemented with IL-2 every 2 days. For induction of in vitro chronically stimulated cells, CD8+ T cells were activated in vitro for 48 h as described above, followed by culture on fresh plate-bound anti-CD3ε and anti-CD28 antibodies for an additional 48 h as previously described20. Acutely activated cells were maintained in IL-2 (50 U ml−1) without restimulation. Mice were intravenously injected with a sublethal dose of recombinant L. monocytogenes expressing OVA (LmOVA, 2 × 106 CFU), following a previously established protocol22. For adoptive transfer studies involving analysis at 6–7 dpi, 5 × 103 OT-I CD8⁺ T cells (Thy1.1⁺ or CD45.2⁺) were transferred intravenously into C57BL/6J recipients (Thy1.2⁺CD45.2⁺ or CD45.1⁺) on day −1, and LmOVA infection was induced 24 h later (day 0). Splenocytes were collected at 7 dpi to evaluate OVA-specific CD8⁺ T cells through Thy1.1 or CD45.2 staining. Cytokine production was measured using intracellular cytokine staining following re-stimulation with OVA peptide (OVA257–264) for 4 h (GolgiStop added at 1 µg ml−1 after 2 h of restimulation) at 37 °C. For metabolic analysis of LmOVA-specific Thy1.1+ OT-I T cells ex vivo, Thy1.2+ C57BL/6 mice received 5 × 104 Thy1.1+ OT-I T cells at day −1, followed by LmOVA infection on day 0. LmOVA-specific CD8+ OT-I T cells were isolated from the spleen of infected mice by positive selection using the EasySep mouse CD90.1 positive selection kit (StemCell Technologies) as previously described43,68. For ex vivo 13C isotope tracing, 5 × 104 Thy1.1⁺ OT-I CD8⁺ T cells were injected into Thy1.2⁺CD45.2⁺ C57BL/6J mice, followed by LmOVA infection the next day. At 7 dpi, activated Thy1.1⁺ OT-I T cells were isolated from the spleens using magnetic bead purification43,68 and cultured in vitro for 2 h in VIM medium21 containing 1.5 mM βOHB. To monitor blood ketone levels, blood was collected from the submental vein of restrained mice, and ketone test strips (Keto-Mojo) were used to quantify blood ketone levels. For T cell tracing and metabolite profiling studies, metabolites were extracted by mixing with ice-cold acetonitrile:methanol:water (4:4:2, v/v)69, sonicating for 5 min and incubating on wet ice for 1 h. Extracts were centrifuged, and the soluble fraction was collected and dried under vacuum. Extracts were resuspended in 50 μl of water for liquid chromatography–mass spectrometry (LC–MS) analysis. For tissue and plasma metabolite and lipid profiling, samples were extracted with chloroform:methanol:water (2:2:1.8 v/v)69,70. For tissue, 690 μl of 1:1 chloroform:methanol was added to 40 mg of tissue and homogenized with a beadmill homogenizer. Then, 495 μl and 100 μl of the upper aqueous and bottom organic layers, respectively, were collected into separate tubes and dried in a rotary vacuum evaporator. The aqueous layer was resuspended in 100 μl of water for metabolomics analysis. The organic layer was resuspended in 200 μl of 50:50 (isopropanol:acetonitrile, v/v) for lipidomics analysis. Metabolomic profiling and stable isotope tracing data were collected using a Vanquish liquid chromatography system coupled to an Orbitrap Exploris 240 (Thermo Fisher Scientific) using a heated electrospray ionization (H-ESI) source in negative mode as previously described22,69. A total of 2 μl of each standard and/or sample was injected and run through a 24-min reversed-phase chromatography Zorbax RRHD extend-C18 column (1.8 μm, 2.1 mm × 150 mm; Agilent, 759700-902) combined with a Zorbax extend-C18 guard column (1.8 μm, 2.1 mm × 5 mm; Agilent, 821725-907). Mobile phase A consisted of LC–MS-grade water (W6, Fisher) with 3% LC–MS-grade methanol (A456, Fisher), mobile phase B was LC–MS-grade methanol and both mobile phases contained 10 mM tributylamine (Sigma-Aldrich, 90780), 15 mM LC–MS-grade acetic acid (Fisher, A11350) and 0.01% medronic acid (v/v; Agilent, 5191-4506). For the wash gradient, mobile phase A was kept the same, and mobile phase B was 99% LC–MS-grade acetonitrile (Fisher, A955). A 16-min wash gradient was run in reverse flow direction between every injection to back-flush the column and to re-equilibrate solvent conditions as follows: 0–3 min held at 100% B and 0.25 ml min−1; 3–3.5 min held at 100% B and ramp to 0.8 ml min−1; 3.5–7.35 min held at 100% B and 0.8 ml min−1; 7.35–7.5 min held at 100% B and ramp to 0.6 ml min−1; 7.5–8.25 min from 100% B to 0% B and ramp to 0.4 ml min−1; 8.25–15.5 min held at 0% B and ramp to 0.25 ml min−1; and 15.5–16 min held at 0% B and 0.25 ml min−1. Full scan data were collected using an Orbitrap with a scan range of 70–850 m/z at a resolution of 240,000 and RF lens at 35%. Fragmentation was induced in the Orbitrap using assisted higher-energy collisional dissociation (HCD) collision energies at 15, 30 and 45%. Orbitrap resolution was 15,000, the isolation window was 2 m/z and data-dependent scans were capped at five scans. For acetyl-CoA measurements, samples were analysed with a Vanquish liquid chromatography system coupled to an Orbitrap ID-X (Thermo Fisher Scientific) using an H-ESI source in positive mode. A total of 2 μl of each standard and/or sample was injected and run through a 5-min reversed-phase chromatography Cortecs T3 Column (1.6 μm, 2.1 mm × 150 mm; Waters, 186008500) combined with a Cortecs T3 VanGuard Pre-column (1.6 μm, 2.1 mm × 5 mm; Waters, 186008508). Mobile phase A consisted of 100% LC–MS-grade water (Fisher, W6), 0.01% ammonium hydroxide (Fisher, A470) and 5 mM ammonium acetate (Sigma-Aldrich, 73594), and mobile phase B consisted of 99% LC–MS-grade acetonitrile (Fisher, A955). A 5-min wash gradient was run between every injection to flush the column and to re-equilibrate solvent conditions as follows: 0–2 min held at 100% B; 2–3 min from 100% B to 0% B; and 3–5 min held at 0% B. MS parameters were: source voltage, 3,500 V; sheath gas, 70; aux gas, 25; sweep gas, 1; ion transfer tube temperature, 300 °C; and vaporizer temperature, 250 °C. A targeted single ion scan was performed in the Orbitrap to target acetyl-CoA and all of its carbon-13 isotopologues with a centre mass of 810.133 m/z and an isolation window of 48 m/z. Resolution was set at 60,000, RF lens at 60% and scan time was set for 2–4 min of the chromatographic gradient described above. Data-dependent MS2 fragmentation was induced in the Orbitrap using assisted HCD collision energies at 20, 40, 60, 80 and 100% as well as with collision-induced dissociation (CID) at a collision energy of 35%. For βOHB quantification in serum and tumours, metabolites were extracted with a 40% acetonitrile, 40% methanol and 20% water solution. An external standard curve for βOHB was generated from 10 µg ml−1 to 0.01 µg ml−1 by half-log serial dilutions. Standards were processed identically to tissue and serum samples throughout the workflow. Tissues were extracted at a concentration of 40 mg tissue per ml, and serum at 40 µl serum per ml of extraction solvent. Extracts were dried and reconstituted in water (200 µl for tissue, 1 ml for serum) containing 100 ng ml−1 of internal standard ([U-13C4]-βOHB). Samples and standards were analysed on an Orbitrap Exploris 240 (Thermo) in ESI-negative mode using tributylamine ion-paired chromatography as described above. Data were processed using Skyline, and internal standard-normalized peak areas were used to calculate βOHB concentrations, which are reported in units of mg g−1 for tissues and mM for serum. Lipidomics samples were analysed with a Thermo Vanquish dual liquid chromatography system using two alternating methods, referred to as chromatography 1 and chromatography 2, coupled to an Orbitrap ID-X (Thermo Fisher Scientific) using an H-ESI source in positive and negative mode, respectively. A total of 2 μl of each standard and/or sample was injected, column temperatures were kept at 50 °C and the flow rate was held at 0.4 ml min−1. For both chromatography 1 and 2, mobile phase A consisted of 60% LC–MS-grade acetonitrile (Fisher Scientific, A955), 40% LC–MS grade water (Fisher Scientific, W6), 0.1% LC–MS-grade formic acid (Fisher Scientific, A117) and 10 mM ammonium formate (Fisher Scientific, 70221), and mobile phase B consisted of 90% LC–MS-grade isopropanol (Fisher Scientific, A461), 8% LC–MS-grade acetonitrile, 2% LC–MS-grade water, 0.1% LC–MS-grade formic acid and 10 mM ammonium formate. A 30 min wash gradient was run between every injection (in parallel with chromatography 2) to flush the column and to re-equilibrate solvent conditions as follows: 0–2 min held at 100% B and 0.3 ml min−1; 2–2.1 min from 100% B to 25% B and held at 0.3 ml min−1; 2.1–4 min held at 25% B and ramp to 0.4 ml min−1; 4–6 min held at 25% B and ramp to 0.6 ml min−1; 6–17 min held at 25% B and 0.6 ml min−1; 17–17.1 min held at 25% B and ramp to 0.4 ml min−1; and 17.1–30 min held at 25% B and 0.4 ml min−1. For both methods, MS parameters were as follows: source voltage, +3,250 V or −3,000 V depending on method polarity; sheath gas, 40; aux gas, 10; sweep gas, 1; ion transfer tube temperature, 300 °C; and vaporizer temperature, 275 °C. Full scan data were collected using the Orbitrap with a scan range of 200-1,700 m/zat a resolution of 500,000 and RF lens at 45%. Data-dependent MS2 fragmentation was induced in the Orbitrap using assisted HCD collision energies at 15, 30, 45, 75 and 110% as well as with CID at a collision energy of 35%. A m/z 184 mass trigger, indicative of phosphatidylcholines, was used for CID fragmentation. Data-dependent MS3 fragmentation was induced in the ion trap with the scan rate set at Rapid, using CID at a collision energy of 35%. MS3 scans were triggered by specific acyl chain losses for detailed analysis of mono-, di- and triacylglycerides. Total cycle time was 2 s. Lipid identifications were assigned using LipidSearch (v.5.0; Thermo Fisher Scientific). For data analysis, peak picking and integration was conducted in Skyline (v.22-23) using in-house curated compound lists of accurate mass MS1 and retention time of chemical standards69. For lipidomics studies, the Lipidsearch identifications were used to populate the target compound list for Skyline. For tracing studies, this list was expanded to include all possible carbon isotopologues, and natural abundance correction was completed using IsoCorrectR71. Chronic stimulation of CD8+ T cells was conducted as previously described20. In brief, female P14 Thy1.1 C57BL/6 mice aged 8–12 weeks were used as a source for primary T cells. CD8+ (P14) cells were isolated from spleens and peripheral lymph nodes using the EasySep Mouse CD8+ T cell isolation kit following the manufacturer's instructions. Following isolation and flow cytometry-based purity assessment, 5 × 106 live P14 cells resuspended in TCM containing 10 ng ml−1 IL-2 were placed in overtop chambers of a six-well dish that were coated 24 h prior (at 4 °C) with 3 μg ml−1 anti-mouse CD3 (eBioscience, Clone 145-2C11) and 1 μg ml−1 anti-mouse CD28 (eBioscience, Clone 37.51) in 1× PBS (Wisent, 311-010-CL). After 48 h incubation (37 °C incubator, 5% CO2), cells were gently displaced from the coated wells, counted and 2 × 106 cells were resuspended in fresh media containing 10 ng ml−1 IL-2 in a final volume of 5 ml in VIM medium21. For continuous stimulation conditions, cells were placed in three wells coated with anti-CD3 and anti-CD28 antibodies in the same fashion as mentioned above, while for acute stimulation conditions, cells were placed directly in three uncoated wells of a six-well dish in VIM containing IL-2. For flow cytometric assessment of CD8+ T cells in vitro, 3 × 105 live cells were centrifuged at 500g for 5 min, followed by resuspension in 200 µl TCM with 50 U ml−1 IL-2. Acute and continuously stimulated cells were seeded in 96-well round-bottom plates and re-stimulated with PMA (50 ng ml−1) and ionomycin (500 ng ml−1) for 4 h (37 °C incubator, 5% CO2), with protein transport inhibitor GolgiStop (1:1,500 dilution; BD Biosciences, 5102092KZ) added for the last 2 h, followed by processing for downstream FACS analysis. Following incubation, cells were pipetted into 15 ml conical tubes and centrifuged at 600g for 3 min. The medium was aspirated, then cells were washed twice with 5 ml ice-cold 0.9% w/v NaCl (saline) and spun at 600g for 2 min. Samples were immediately frozen on dry ice for 5 min, then transferred to a −80 °C freezer until processing for metabolomics. Ex vivo stable isotope labelling experiments with CD8+ T cells isolated from mice using LC or gas chromatography coupled to MS were performed as outlined previously21,22. Antigen-specific T cells were isolated from AL or DR LmOVA-infected mice (BDH1/OXCT1 DKO or WT mice) using magnetic bead isolation as previously described22,43. CD8+ T cells were cultured for 2 h in VIM medium containing 1.5 mM 13C4-βOHB, followed by metabolite extraction as described above. For 13C co-tracing experiments, CD8+ T cells were cultured for 2 h in VIM medium containing 50 U ml−1 IL-2 and 5 mM 13C6-glucose and 1.5 mM 2,4-13C2-βOHB (Cambridge Isotope Laboratories). Identical unlabelled controls were performed using VIM medium containing 12C-glucose and 12C-βOHB. After the 2 h incubation period, cells were centrifuged at 600g for 4 min, media were collected and flash-frozen over dry ice and cell pellets were washed twice with 0.9% NaCl before flash-freezing and storage at −80 °C for downstream metabolomics analysis. Immunoblotting was performed by lysing cells in RIPA buffer containing protease and phosphatase inhibitors (Roche) on ice for a minimum of 30 min as previously described72. Equal amounts of protein from whole cell lysates were diluted in Laemmli sample buffer, boiled for 5 min, resolved on a 10% SDS–PAGE gel and transferred to nitrocellulose membranes. Membranes were blocked for 1 h at room temperature in 5% non-fat milk prepared in TBS-T, followed by overnight incubation at 4 °C with primary antibodies against BDH1 or SCOT (1:1,000 dilution in 5% non-fat milk). After washing three times in 1× TBS-T (5 min per wash), membranes were incubated for 1 h at room temperature with horseradish peroxidase-conjugated secondary antibodies (diluted in 5% non-fat milk). Membranes were washed again three times with 1× TBS-T before being developed using ECL solution (Cytiva). T cell oxygen consumption rate and extracellular acidification rate were measured using a Seahorse XF96 Extracellular Flux Analyzer, as previously described22. Antigen-specific Thy1.1+ OT-1 CD8+ T cells isolated from LmOVA-infected mice were seeded at 2 × 105 cells per well in Seahorse XF medium containing 5 mM glucose and 0.5 mM glutamine, with or without 1.5 mM βOHB (depending on experimental conditions). Bioenergetics data interpretation followed previously established protocols73. Single-cell suspensions were prepared from mouse spleens by mechanical dissociation. Red blood cells were lysed using red blood cell lysis buffer containing 0.15 M NH₄Cl, 10 mM KHCO₃ and 0.1 mM EDTA, followed by neutralization with three volumes of TCM. For cell staining, single-cell suspensions were incubated with a cocktail of fluorescently labelled antibodies and dyes as listed in the Key Resources Table. Cell viability was assessed using Fixable Viability Dye eFluor 506 (Thermo Fisher Scientific) according to the manufacturer's protocols. To assess cytokine production following LmOVA infection, splenocytes collected from infected mice at 7 dpi were stimulated with 1 µg ml−1 OVA257–264 peptide (Anaspec) and 50 U ml−1 IL-2 for 4 h at 37 °C. TIL were re-stimulated with PMA and ionomycin. GolgiStop protein transport inhibitor (BD Biosciences) was added at a 1:1,500 dilution during the last 2 h of incubation to block cytokine secretion. After stimulation, cells were stained with surface marker antibodies in staining buffer (PBS containing 2% FBS and 0.02% sodium azide) for 1 h at 4 °C. Cells were fixed and permeabilized using the Foxp3/Transcription Factor Staining Buffer Set (Thermo Fisher Scientific) at 4 °C for 1 h. Intracellular staining was performed by incubating cells with fluorescently labelled antibodies targeting intracellular markers for either 1 h or overnight at 4 °C. Flow cytometry data were acquired using a CytoFLEX (Beckman Coulter), Aurora Cytek or BD Accuri C6 Plus cytometer. Cell sorting was performed on an Astrios (Beckman Coulter) or BD FACSAria Fusion cell sorter. Data analysis was conducted using FlowJo software. C57BL/6J mice aged 10–12 weeks were allocated to either AL feeding or DR for 1 week before tumour cell injection. Following the feeding regimen, each mouse was subcutaneously injected with 5 × 105 B16-OVA melanoma cells. At 13 dpi, tumours were dissected from the mice for further analysis. To prepare single-cell suspensions, excised tumours were mechanically dissociated and filtered sequentially through 100 µm and then 50 µm cell strainers. Throughout all procedures, cells were kept on ice to preserve viability. The resulting cell suspensions were counted, and cell viability was assessed by trypan blue exclusion. To prevent nonspecific antibody binding, cells were blocked with TruStain FcX Plus (BioLegend, 156603) in Cell Staining Buffer (CSB; BD Biosciences, 420201) for 10 min at 4 °C. Each tumour sample was then labelled with a specific antibody (Hashtags 1–8, product information in Key Resource Table) according to the manufacturer's protocol, including appropriate incubation and washing steps. Cells were sorted by flow cytometry, and live CD45⁺ cells were collected into tubes containing IMDM supplemented with 10% FBS. Based on cell counts, CD45⁺ cells were adjusted to a concentration of 1 × 106 cells per 50 µl and blocked again with TruStain FcX Plus for 10 min at 4 °C. During this blocking step, TotalSeq C antibodies were prepared according to the manufacturer's instructions (BioLegend). The TotalSeq C antibodies were then added to the CD45⁺ cells and incubated for 30 min at 4 °C to label cell surface proteins for subsequent analysis. Following antibody staining, cells were washed three times with CSB to remove unbound antibodies. Cells were then resuspended in 1× PBS containing 0.04% pure BSA at an approximate concentration of 2,200 cells per µl. The prepared cell suspension was used for downstream library preparation and sequencing. Libraries were generated and sequenced by the VAI Genomics Core. Cells were processed with 10× Chromium Next GEM Single Cell 5ʹ GEM kit (v.1.1) according to the manufacturer's instructions to target an output of 13,000 cells per sample using a 10× Genomics Chromium Controller. In brief, single-cell suspensions in PBS + 0.04% BSA were assessed for quantity and viability on the CytoFLEX S (Beckman Coulter), then 20,000 cells per sample were loaded onto the Chromium Controller. Released RNA was barcoded and then converted to cDNA. Using beads, cDNA was separated from the hashtag oligonucleotides (HTO), and each was used to generate adaptor-ligated libraries. Quality and quantity of the finished gene expression and HTO libraries were assessed using a combination of Agilent DNA High Sensitivity Chip (Agilent Technologies) and QuantiFluor dsDNA System (Promega). Then, 2 × 100 bp, paired-end sequencing was performed on an Illumina NovaSeq 6000 sequencer using an S2, 200-cycle sequencing kit (v.1.5) to a minimum depth of 20 K reads per cell. Base calling was done by Illumina RTA3, and output was demultiplexed and converted to FastQ format with Cell Ranger (10× Genomics, v.3.1.0). Following library preparation, FastQ files were processed using Cell Ranger (v.7.0.1) to map and quantify expression profiles for both the transcriptome and epitopes. Single-cell 5′ paired-end sequencing chemistry was used, and the sequencing data were aligned to the mouse reference genome m10-2020-A. Initial data processing involved separating the gene expression (GEX) data, ADTs and HTOs, which was critical for downstream analysis. A Seurat object was created using the CreateSeuratObject function. Quality control and normalization steps were then performed: HTO data were normalized using the NormalizeData function with the method set to ‘CLR' (centred log ratio transformation), and the HTODemux function was used to classify cells based on HTO signals into ‘singlets', ‘doublets' and ‘negative' categories. Further quality control filtering was applied by retaining cells with ADT counts of at least 150, RNA counts of at least 200, an HTO classification of ‘singlet' and mitochondrial gene expression percentage less than 20%. Principal component analysis (PCA) was conducted on both RNA and ADT data using the RunPCA function, including all ADTs for dimensionality reduction. To generate a joint uniform manifold approximation and projection (UMAP) embedding that integrates both GEX and ADT data, the FindMultiModalNeighbors function was used, followed by the RunUMAP function, resulting in a weighted nearest-neighbour UMAP. Cluster identification was achieved by integrating gene expression profiles, ADT data and gene expression density visualizations using the plot_density() function from the Nebulosa package (v.1.14.0). Statistical analyses for violin plots were performed using the Kruskal–Wallis test, followed by pairwise Wilcoxon comparison tests with Bonferroni P value adjustment. Sequencing data from the single-cell RNA sequencing (scRNA-seq) experiments have been deposited in the Gene Expression Omnibus (GEO) under accession number GSE267070. To infer cellular trajectories from the scRNA-seq data, we performed RNA velocity analysis using the Python implementation of Velocyto (v.0.17.17), following previously described methods45. In brief, spliced and unspliced counts were generated using the velocyto run10x subcommand, outputting a loom file for importing into Python or R. Next, we used scvelo to infer RNA velocity vectors and projected them onto the UMAP embeddings generated above. This approach enabled us to determine the directional changes of cells between dietary conditions. Code used to make RNA velocity plots can be found at https://github.com/rgjcanada/DR_CITESEQ_2025. For analysis of the human scRNA-seq dataset74, count data were normalized and transformed through the SCT normalization method in Seurat, with 5,000 variable features retained for downstream dimension reduction techniques. Integration of data was performed on the patient level with canonical correlation analysis as the dimension reduction technique. Cells were clustered using the Louvain algorithm with multi-level refinement. PCA was performed, with the first 50 PCs used in UMAP generation. The data were subset to CD8+ T cells, which were identified using the labels provided in a prior work75 and confirmed by singleR using the ImmGen database76 and cell type annotation with the ProjectTILs T cell atlas77. These CD8+ T cells were subsequently normalized through the SCT method, with 3,000 variable features retained. Owing to the low number of cells on a per-patient level, Harmony was used to integrate the data at the patient level rather than Seurat78. PCA and UMAP dimension reduction were performed as above for clustering into distinct groups, Teff_ITGB1, Teff_TYROBP, Teff_TRIM34, TCM, Trm_TM, Trm_TMEM2, Trm_BAG3, Trm_ZNF20, IEL, Tex_MKI67 and Tex_CLNK, based on previously defined markers79. To simplify the analysis, we combined Teff_ITGB1, Teff_TYROBP and Teff_TRIM34 into the ‘T effector (Teff)' cluster, while grouping TCM, Trm_TM, Trm_TMEM2, Trm_BAG3, Trm_ZNF20 and IEL into the ‘T memory (Tmem)' cluster. Additionally, Tex_MKI67 and Tex_CLNK were combined into the ‘T exhausted (Tex)' cluster. Samples were processed using the EasyPep Mini MS Sample Prep Kit (Thermo Fisher Scientific, A40006) according to the manufacturer's instructions. In brief, cells were resuspended in 50 µl of lysis solution and proteins were quantified using the Pierce BCA Protein Assay Kit (Thermo Fisher Scientific, 23227) following the vendor-supplied protocol. Plates were read at an absorbance of 562 nm using the Synergy LX Multi-Mode Reader, and Gen5 software was used for data analysis (BioTek/Agilent). Polynomial regression was used in the Gen5 software to calculate protein concentrations from a protein standard curve after an average blank absorbance subtraction. Then, 25 µg of protein per sample was reduced and alkylated at 95 °C for 10 min, and samples were digested overnight with Trypsin/Lys-C at 30 °C at a ratio of 10:1 (protein:enzyme (w/w)). Resulting peptides were cleaned with the supplied peptide clean-up columns in the EasyPep mini kit and then dried down in a Genevac SpeedVac before resuspension for instrument analysis. Samples were resuspended in 12.5 µl 0.1% formic acid (Fisher Scientific, LS118-1) and diluted 1:1 with 6 µl of 0.1% trifluoroacetic acid (Fisher Scientific, LS119-500) in autosampler vials. Data-independent acquisition analyses were performed on an Orbitrap Eclipse coupled to a Vanquish Neo system (Thermo Fisher Scientific) with a FAIMS Pro source (Thermo Fisher Scientific) located between the nanoESI source and the mass spectrometer. A total of 2 μg of digested peptides were separated on a nano capillary column (20 cm × 75 μm inner diameter, 365 μm outer diameter, 1.7 μm C18; CoAnn Technologies, HEB07502001718IWF) at 300 nl min−1. Mobile phase A consisted of LC–MS-grade H2O (Fisher Scientific, LS118-500), mobile phase B consisted of 20% LC–MS-grade H2O and 80% LC–MS-grade acetonitrile (Fisher Scientific, LS122500), both containing 0.1% formic acid. The column temperature was kept constant at 50 °C using a customized column heater (Phoenix S&T). Full MS spectra were collected at 120,000 resolution (full width half-maximum; FWHM), and MS2 spectra at 30,000 resolution (FWHM). Both the standard automatic gain control target and the automatic maximum injection time were selected. A precursor range of 380–985 m/z was set for MS2 scans, and an isolation window of 50 m/z was chosen with a 1 m/z overlap for each scan cycle; 32% HCD collision energy was used for MS2 fragmentation. Data were searched against the Mus musculus proteome. In brief, trypsin/P was set as the digestion enzyme, and two missed cleavages were allowed. Cysteine carbamidomethylation was set as a fixed modification, and methionine oxidation and protein amino-terminal acetylation as variable modifications. Identification was performed using a 1% q value cutoff on precursor and protein levels. Both peptide precursors and protein false discovery rate (FDR) were controlled at 1%. Ion chromatograms of fragment ions were used for quantification. Proteomic values were imported into R from an Excel file and processed with dplyr and readxl. Replicate columns for each condition (AL-WT, AL-DKO, DR-WT, DR-DKO) were coerced to numeric and averaged to obtain per-protein means. Differential abundance was calculated as log2(fold change) (for example, DR-WT / AL-WT), and Welch's t-tests were performed between the corresponding replicate groups to obtain P values. Proteins lacking sufficient replicate measurements were excluded. For GSEA, proteins were mapped to gene symbols (duplicated symbols collapsed by the entry with the largest absolute log2(fold change). A pre-ranked vector was generated using the sign of the log2(fold change) multiplied by −log10(P value) (equivalent results were obtained using log2(fold change) alone). Mouse gene sets were retrieved from MSigDB via msigdbr (Hallmark ‘H', curated ‘C2' and Gene Ontology ‘C5' collections); sets with <15 or >500 genes were discarded. Enrichment analysis was performed with fgsea (10,000 permutations, default weighting), returning normalized enrichment scores, nominal P values and FDR-adjusted q values (Benjamini–Hochberg). Only pathways with an FDR q < 0.05 were considered significant and are displayed in the figures. Bar plots of normalized enrichment scores and volcano plots were generated in ggplot2 and EnhancedVolcano, respectively. Mice were individually housed in Promethion metabolic cages (Sable Systems International) for comprehensive metabolic phenotyping. Before data collection, animals underwent a 24 h acclimation period within the cages to minimize stress and ensure steady-state measurements. Following acclimation, continuous monitoring was performed over a desired period under controlled environmental conditions (temperature, humidity and a 12–12h light–dark cycle) as specified by the manufacturer's guidelines. Calibration of the sensors and verification of system performance were conducted in strict accordance with Sable Systems International's operating protocols. Data acquisition and subsequent analysis were carried out using the proprietary Promethion analysis software, ensuring that all measurements met the company's quality control standards. Body composition was measured using an EchoMRI-100 on conscious mice. Before each session, the instrument was calibrated with manufacturer-provided oil standards. Individual mice were weighed, placed in a ventilated acrylic restraining tube with an internal insert to minimize movement and scanned (standard mode; water stage on–off as indicated in figure legends). Each scan yielded fat mass, lean mass, free water and total water; values were exported directly from the EchoMRI software for downstream analysis. Restraining tubes were disinfected between animals with 70% ethanol or facility-approved disinfectant. Full-resolution SVS images of HDAB-stained tissue sections were imported into QuPath (v.0.5.1)80 for analysis. For all tissues to be analysed, a region of interest contouring the tissue was created, using the SAM plugin (v.0.6.0) for QuPath81,82 with the following settings: vit_h (huge) model, foreground with rectangle draw prompt and single mask output on live mode. To detect nuclei, an average of the deconvolved haematoxylin, DAB and residual channels was generated to create a fluorescence-like single-channel image using a custom script (hosted on https://github.com/vaioic). The detections and tissue outline were exported as an object GeoJSON and imported to the original HDAB-stained image. Detection measurements were generated with QuPath's built-in Add Intensity Features with the following settings: pixel size, 0.5 µm; region, region of interest; tile diameter, 0; channels, haematoxylin, DAB; basic features, mean, standard deviation, minimum, maximum and median. Nuclei were then classified as positive or negative for DAB staining using the mean DAB signal at a threshold of 0.117. A pixel thresholder was used to measure the area covered by DAB staining. A training image generated from an equal number and size of random rectangles from each image was used for parameter setting. QuPath's Create Thresholder was used with the following settings: resolution, very high (1.01 µm per pixel); channel, DAB; prefilter, Gaussian; smoothing sigma, 1; threshold, 0.25. for biological replicates, as indicated in figure legends. Comparisons between two groups were conducted using unpaired, two-tailed Student's t-tests. For comparisons involving more than two groups, one-way or two-way ANOVA with post hoc multiple-comparison corrections (Tukey's or Sidak's test, as appropriate) were applied. Kaplan–Meier survival curves were compared using the log-rank (Mantel–Cox) test, with Bonferroni correction applied for multiple comparisons when indicated. Data distribution was assumed to be normal. No statistical method was used to predetermine sample size, but our sample sizes are similar to those reported in previous publications. All data were included in the analyses. In rare cases for which data were excluded, exclusion occurred because of animal health concerns (for example, sickness or tissue necrosis) or based on statistical outlier identification using standard outlier exclusion tests. All experiments were independently repeated as described in the figure legends, and similar results were obtained, confirming reproducibility. During the preparation of this work, the author(s) used OpenAI's ChatGPT to review grammar, spelling and refine text style. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All unique and stable reagents generated in this study are available from the Lead Contact upon completion of a Materials Transfer Agreement. CITE-seq data of mouse TILs have been deposited in the National Center for Biotechnology Information Gene Expression Omnibus (NCBI GEO) under accession number GSE267070. Human scRNA-seq data of TILs used in this research are available at NCBI GEO under accession number GSE146771. All other data supporting the findings of this study are available within the article and its online supplementary material. Bioenergetics data were analysed using protocols developed by Mookerjee and Brand, available for download at https://russelljoneslab.vai.org. Source data are provided with this paper. R code used for data processing is available at our GitHub repository (https://github.com/rgjcanada/DR_CITESEQ_2025.git). Processed data files (RDS files) curated from the data processing can be found at Zenodo (https://doi.org/10.5281/zenodo.13920173)86. Hanahan, D. & Weinberg, R. A. Hallmarks of cancer: the next generation. Philip, M. & Schietinger, A. CD8+ T cell differentiation and dysfunction in cancer. Miller, B. C. et al. Subsets of exhausted CD8+ T cells differentially mediate tumor control and respond to checkpoint blockade. Daniel, B. et al. Divergent clonal differentiation trajectories of T cell exhaustion. McLane, L. M., Abdel-Hakeem, M. S. & Wherry, E. J. CD8 T cell exhaustion during chronic viral infection and cancer. Khan, O. et al. TOX transcriptionally and epigenetically programs CD8+ T cell exhaustion. Yao, C. et al. Single-cell RNA-seq reveals TOX as a key regulator of CD8+ T cell persistence in chronic infection. & Satpathy, A. T. Epigenetic regulation of T cell exhaustion. The epigenetic landscape of T cell exhaustion. Buck, M. D., Sowell, R. T., Kaech, S. M. & Pearce, E. L. Metabolic instruction of immunity. A., Landis, M. D. & Rathmell, J. C. Nutrient inputs and social metabolic control of T cell fate. Chapman, N. M. & Chi, H. Metabolic adaptation of lymphocytes in immunity and disease. Roy, D. G., Kaymak, I., Williams, K. S., Ma, E. H. & Jones, R. G. Immunometabolism in the tumor microenvironment. Lim, A. R., Rathmell, W. K. & Rathmell, J. C. The tumor microenvironment as a metabolic barrier to effector T cells and immunotherapy. Bengsch, B. et al. Bioenergetic insufficiencies due to metabolic alterations regulated by the inhibitory receptor PD-1 are an early driver of CD8+ T cell exhaustion. Scharping, N. E. et al. Mitochondrial stress induced by continuous stimulation under hypoxia rapidly drives T cell exhaustion. Disturbed mitochondrial dynamics in CD8+ TILs reinforce T cell exhaustion. Impaired mitochondrial oxidative phosphorylation limits the self-renewal of T cells exposed to persistent antigen. Carbon source availability drives nutrient utilization in CD8+ T cells. Luda, K. M. et al. Ketolysis drives CD8+ T cell effector function through effects on histone acetylation. St Paul, M. et al. Coenzyme A fuels T cell anti-tumor immunity. Sullivan, M. R. et al. Quantification of microenvironmental metabolites in murine cancers reveals determinants of tumor nutrient availability. Mishra, A., Giuliani, G. & Longo, V. D. Nutrition and dietary restrictions in cancer prevention. A., Berrigan, D., Perkins, S. N. & Barrett, J. C. Calorie restriction, aging, and cancer prevention: mechanisms of action and applicability to humans. Fontana, L., Partridge, L. & Longo, V. D. Extending healthy life span—from yeast to humans. Lien, E. C. et al. Low glycaemic diets alter lipid metabolism to influence tumour growth. & Sabatini, D. M. Tumours with PI3K activation are resistant to dietary restriction. Scrimshaw, N. S. & SanGiovanni, J. P. Synergism of nutrition, infection, and immunity: an overview. Control of immunity via nutritional interventions. Palma, C. et al. Caloric restriction promotes immunometabolic reprogramming leading to protection from tuberculosis. The bone marrow protects and optimizes immunological memory during dietary restriction. Delconte, R. B. et al. Fasting reshapes tissue-specific niches to improve NK cell-mediated anti-tumor immunity. Jordan, S. et al. Dietary intake regulates the circulating inflammatory monocyte pool. Ahmed, T. et al. Calorie restriction enhances T-cell-mediated immune response in adult overweight men and women. Di Biase, S. et al. Fasting-mimicking diet reduces HO-1 to promote T cell-mediated tumor cytotoxicity. Ma, E. H. et al. Metabolic profiling using stable isotope tracing reveals distinct patterns of glucose utilization by physiologically activated CD8+ T cells. Martin, M. D., Condotta, S. A., Harty, J. T. & Badovinac, V. P. Population dynamics of naive and memory CD8 T cell responses after antigen stimulations in vivo. La Manno, G. et al. RNA velocity of single cells. Ma, E. H. et al. 13C metabolite tracing reveals glutamine and acetate as critical in vivo fuels for CD8 T cells. & Lenardo, M. Human plasma-like medium improves T lymphocyte activation. Ringel, A. E. et al. Obesity shapes metabolism in the tumor microenvironment to suppress anti-tumor immunity. Hirschberger, S. et al. Ketone bodies improve human CD8+ cytotoxic T-cell immune response during COVID-19 infection. Arnold, P. K. et al. A non-canonical tricarboxylic acid cycle underlies cellular identity. Kaymak, I. et al. ACLY and ACSS2 link nutrient-dependent chromatin accessibility to CD8 T cell effector responses. Ma, S. et al. Nutrient-driven histone code determines exhausted CD8+ T cell fates. Response to programmed cell death-1 blockade in a murine melanoma syngeneic model requires costimulation, CD4, and CD8 T cells. Cheng, D. et al. Proliferative exhausted CD8+ T cells exacerbate long-lasting anti-tumor effects in human papillomavirus-positive head and neck squamous cell carcinoma.eLife 12, e82705 (2023). Pauken, K. E. et al. Epigenetic stability of exhausted T cells limits durability of reinvigoration by PD-1 blockade. Increased serine synthesis provides an advantage for tumors arising in tissues where serine levels are limiting. Distinct modes of mitochondrial metabolism uncouple T cell differentiation and function. Opposing effects of fasting metabolism on tissue tolerance in bacterial and viral inflammation. Type I interferon signaling disrupts the hepatic urea cycle and alters systemic metabolism to suppress T cell function. Demiroz, D. et al. Listeria monocytogenes infection rewires host metabolism with regulatory input from type I interferons. Multi-dimensional roles of ketone bodies in fuel metabolism, signaling, and therapeutics. Distinct signaling of coreceptors regulates specific metabolism pathways and impacts memory development in CAR T cells. The failing heart utilizes 3-hydroxybutyrate as a metabolic stress defense. Successful adaptation to ketosis by mice with tissue-specific deficiency of ketone body oxidation. Cordeiro, B. et al. MicroRNA-9 fine-tunes dendritic cell function by suppressing negative regulators in a cell-type-specific manner. Sheldon, R. D., Ma, E. H., DeCamp, L. M., Williams, K. S. & Jones, R. G. Interrogating in vivo T-cell metabolism in mice using stable isotope labeling metabolomics and rapid cell sorting. House, R. R. J. et al. A diverse proteome is present and enzymatically active in metabolite extracts. A rapid method of total lipid extraction and purification. Heinrich, P. et al. Correcting for natural isotope abundance and tracer impurity in MS-, MS/MS- and high-resolution-multiple-tracer-data from stable isotope labeling experiments with IsoCorrectoR. Differential effects of AMPK agonists on cell growth and metabolism. A., Nicholls, D. G. & Brand, M. D. Quantifying intracellular rates of glycolytic and oxidative ATP production and consumption using extracellular flux measurements. Zhang, L. et al. Single-cell analyses inform mechanisms of myeloid-targeted therapies in colon cancer. Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. The Immunological Genome Project: networks of gene expression in immune cells. Interpretation of T cell states from single-cell transcriptomics data using reference atlases. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Zhang, L. et al. Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Sugawara, K. Training deep learning models for cell image segmentation with sparse annotations. Stringer, C. & Pachitariu, M. Cellpose3: one-click image restoration for improved cellular segmentation. Pachitariu, M. & Stringer, C. Cellpose 2.0: how to train your own model. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Oswald, B., & Jones, R. Processed CITE Seq (RDS Files). We acknowledge J. Lum, T. Triche, B. Williams and members of the Jones and Krawczyk laboratories for scientific discussions contributing to this paper. We thank N. Lubben for technical assistance and J. Wedberg, M. Brewer and M. Minard for administrative assistance. The authors thank C. Capan and M. Soper-Hopper for sample preparation, data acquisition and analysis of proteomics data. Proteomics experiments were conducted in the VAI Mass Spectrometry Core (RRID:SCR_024903). Department of Metabolism and Nutritional Programming, Van Andel Institute, Grand Rapids, MI, USA Brandon M. Oswald, Lisa M. DeCamp, Joseph Longo, Michael S. Dahabieh, Nicholas Bunda, McLane J. Watson, Samuel E. J. Preston, Dahlya Kamarudin, Kelsey S. Williams, Evan C. Lien, Connie M. Krawczyk & Russell G. Jones Department of Epigenetics, Van Andel Institute, Grand Rapids, MI, USA Benjamin K. Johnson, Hui Shen & H. Josh Jang Salk Institute, La Jolla, CA, USA Mass Spectrometry Core, Van Andel Institute, Grand Rapids, MI, USA Ryan D. Sheldon, Michael P. Vincent, Abigail E. Ellis, Molly T. Soper-Hopper & Christine Isaguirre Metabolism and Nutrition (MeNu) Program, Van Andel Institute, Grand Rapids, MI, USA Michael P. Vincent, Kelsey S. Williams, Connie M. Krawczyk & Russell G. Jones Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar is on the scientific advisory boards and has equity in EvolveImmune Therapeutics, Affini-T Therapeutics, Arvinas and Pfizer. is a scientific advisor to Servier Pharmaceuticals and is a member of the Scientific Advisory Board of Immunomet Therapeutics. The other authors declare no competing interests. Nature Metabolism thanks Nabil Djouder and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Alfredo Gimenez-Cassina and Jean Nakhle, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A) Weight Change (% change from baseline) of mice after 2 weeks of feeding on AL or DR diets. Data represents the mean ± SEM (2 weeks, n = 12 mice/group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. B) Bar plot showing serum immunoglobulin levels in mice administered anti-IgG or anti-PD1 antibodies under AL or DR conditions. Data represent the mean ± SEM (n = 8 mice per group). Statistical significance was calculated via two-way ANOVA multiple comparisons test. C) Kaplan-Meier plot comparing tumor onset (tumor volume ≥250 mm3) in B16-OVA melanoma-bearing athymic nude mice fed ad libitum (AL) or dietary restriction (DR) diets. Statistical significance was assessed by log-rank test (n = 5 mice per group). Exact P-values are as follows: Extended Data Fig. A) Weighted nearest neighbor Uniform Manifold Approximation and Projection (wnnUMAP) of 45,455 CD45⁺ tumor-infiltrating cells from B16 melanoma tumors of mice on AL or DR diets (wnnUMAP combined; n = 4 mice per diet). Each plot represents the expression of key antibody-derived tags (ADTs) for markers including NK1.1, Ly6C, LAG3 (CD223), CD4, CD8, PD1, F4/80, CD11c, TIM3 (CD366), CD11b, CD69, and CD127. B) Density plots generated using the Nebulosa package, showing the expression density of ADTs (CD4, CD8, CD11c, CD11b, CD127) among activated CD8⁺ T cells from B16 tumors (combined AL and DR conditions). C) Density plots of gene expression for Cd8a, Tbx21, Tox, Prf1, and Il7r among activated CD8⁺ T cells from B16 tumors (joint plot of AL and DR conditions). D) Representative histogram of TBET expression in PD1LowTOXLow CD8⁺ TILs from B16 tumors. Bar plots show relative geometric mean fluorescence intensity (gMFI) compared to AL controls. Data represent the mean ± SEM (n = 8 mice per group). E) Representative histogram of IFN-γ expression in PD1LowTOXLow CD8⁺ TILs from B16 tumors. Bar plots show relative gMFI compared to AL controls. Data represent the mean ± SEM (n = 8 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. A) Total distance traveled (meters) by mice on ad libitum (AL) or dietary restriction (DR) feeding regimens. Activity was measured using metabolic Promethion cages over 168 hours (n = 6 mice per group). B) Daily food intake (grams) of mice on ad libitum (AL) or dietary restriction (DR) feeding regimens. Right, Total water intake (grams) over one week for mice on AL or DR feeding regimens. Data represent the mean ± SEM (n = 6 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. C) Change in body weight (grams) over time (hours) for mice on AL or DR conditions. Data represent the mean ± SEM (n = 6 mice per group). D-G) Mice were fed ad libitum (AL; black) or restricted to 30%, 40%, or 50% of AL intake (light- to dark-blue) for two weeks. Percent change from baseline in (D) total body weight, E) Mouse weight change across diets (AL, 30%, 40%, and 50% DR) over 32 days. Data represent mean ± SEM, (F) fat mass, and (G) lean mass was measured by EchoMRI. Each circle represents an individual mouse; bars indicate mean ± SEM. H) Kaplan–Meier curves of tumor onset in mice fed ad libitum (AL; black) or restricted to 30%, 40% or 50% of AL intake (light to dark blue). Each step down represents an individual mouse developing a tumor equal to or greater than 250mm3. Statistical significance was calculated with log rank test with Bonferroni correction (n = 26 (AL), 5 (30% DR), 8 (40%), and 24 (50% DR). Exact P-values are as follows: Extended Data Fig. Top panel shows the frequency of major adaptive subsets in spleen from mice fed ad libitum (AL, gray) or on dietary restriction (DR, blue) for two weeks. Frequencies are plotted as percentage of live cells (% CD3⁺ T cells; % CD19⁺ B cells) or as percentage of the parent T-cell pool (% CD4⁺ and % CD8⁺ of CD3⁺) and innate lymphoid subsets (% NK and % NKT of live). Bottom panel shows the corresponding absolute cell counts recovered per spleen. In all plots, each point is one mouse (n = 5 per group); bars indicate mean ± SEM. Statistical significance was calculated via two-tailed, unpaired Student's t-test. Bottom panel shows absolute numbers of each innate subset per spleen. In all plots, each point is one mouse (n = 5 per group); bars indicate mean ± SEM. Statistical significance was calculated via two-tailed, unpaired Student's t-test. Exact P-values are as follows: Extended Data Fig. A) Schematic depicting the experimental setup of adoptive OT-I T cell transfer coupled with attenuated LmOVA infection. Mice were started on AL or DR diets six days before OT-I adoptive transfer. On day 0, mice were injected with LmOVA via tail vein. Seven days post-infection (dpi), spleens were harvested for downstream analyses, including effector function, cell expansion, and metabolic function. B) Representative flow cytometry plots showing IFN-γ production and CD44 expression in splenic OT-I CD8⁺ T cells (CD8⁺THY1.1⁺) under AL or DR conditions at 7 dpi. Right, bar plot showing the percentage of IFN-γ⁺ cells. Data represent the mean ± SEM (n = 5 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. C) Bar plot showing the geometric mean fluorescence intensity (gMFI) of IFN-γ in CD8⁺THY1.1⁺ T cells under AL or DR conditions at 7 dpi. Data represent the mean ± SEM (n = 5 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. D) Representative flow cytometry plots showing GZMB production and CD44 expression in splenic OT-I CD8⁺ T cells (CD8⁺THY1.1⁺) under AL or DR conditions at 7 dpi. Right, bar plot showing the percentage of GZMB+ cells. Data represent the mean ± SEM (n = 5 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. Right, bar plot showing the geometric mean fluorescence intensity (gMFI) of IFN-γ in CD8⁺THY1.1⁺ T cells under AL or DR conditions at 7 dpi. Data represent the mean ± SEM (n = 5 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. F) Representative flow cytometry plots showing expansion of splenic OT-I CD8⁺ T cells (THY1.1 versus CD44) 7 dpi. Right, Bar plots showing the percentage and absolute number of THY1.1⁺ antigen-specific CD8⁺ T cells expanded under AL or DR conditions at 7 dpi. Data represent the mean ± SEM (n = 4 AL and 5 DR micegroup). Statistical significance was calculated via two-tailed, unpaired Student's t-test. G) Shown are the top significantly enriched pathways (false discovery rate [FDR] < 0.05) from the Hallmark gene set collection (MSigDB “H”) in the DR versus AL comparison from the proteomic data collected in (I). Proteins were mapped to their corresponding genes, and a ranked list (for example, by fold change) was analyzed using fgsea. Bars indicate the normalized enrichment score (NES), with positive NES reflecting enrichment in the DR condition. H) Bar plots showing the pool sizes of TCA cycle intermediates citrate and malate in antigen-specific CD8⁺THY1.1⁺ T cells from LmOVA-infected mice (7 dpi) under AL or DR conditions. Data represent the mean ± SEM (n = 12 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. I) Maximal ATP production rates (J_ATP) from OXPHOS and GLY for CD8⁺THY1.1⁺ T cells under AL or DR conditions. Data represent the mean ± SD (n = 23 AL and 44 DR technical replicates). Statistical significance was calculated via one-way ANOVA multiple comparisons test. Exact P-values are as follows: Extended Data Fig. A) Heatmap showing Log2-normalized expression of key genes highlighting population differences among cell clusters in TILs isolated from B16 tumors (combined AL and DR conditions). B) RNA density plots of activated CD8⁺ T cell clusters represented on a wnnUMAP, displaying the density of genes associated with exhaustion, proliferation, effector function, and activation gene sets (joint plot of AL and DR conditions). C) Bar plot depicting cell counts from CITE-sequencing experiments for Prolif1, Prolif2, TexEff, and TexTerm CD8⁺ TIL clusters under AL and DR conditions. The Effector:Exhausted ratio is displayed based on cell counts. D) Bar plot showing the percent expression of TCF1 of PD1+TOX+CD8+ TILs (EO771) between AL and DR conditions. Statistical significance was calculated via two-tailed, unpaired Student's t-test. Exact P-values are as follows: Extended Data Fig. A) Mean respiratory exchange ratio (RER; VCO2/VO2) of mice on AL or DR feeding regimens measured using Promethion metabolic cages. “Fed” represents the average peak RER post-feeding, while “Fasted” represents the lowest RER during fasting periods. Data represent the mean ± SEM (n = 6 mice per group). Statistical significance was calculated via two-way ANOVA multiple comparisons test. B) Heatmap showing the most significant lipid changes (Log2 fold change) in tumors isolated from mice on AL or DR diets 14 days post tumor implantation (n = 3–4 mice per group). C) Theoretical basal ATP production rates (J_ATP) from OXPHOS and GLY in CD8⁺ T cells cultured with or without β-hydroxybutyrate (βOHB) under AL or DR conditions. Data represent the mean ± SEM (n = 16 AL, 16 AL + βOHB, 16 DR, and 8 DR + βOHB, technical replicates). Statistical significance was calculated via two-way ANOVA multiple comparisons test. D) Theoretical maximal ATP production rates (J_ATP) from OXPHOS and GLY under the same conditions as in (C). Data represent the mean ± SEM (n = 16 AL, 16 AL + βOHB, 16 DR, and 8 DR + βOHB, technical replicates). Statistical significance was calculated via two-way ANOVA multiple comparisons test. Exact P-values are as follows: Extended Data Fig. A) Schematic illustrating the acute versus chronic in vitro stimulation assay, adapted from (5). CD8⁺ T cells isolated from C57BL/6 mice were activated with anti-CD3 and anti-CD28 antibodies (1 μg/mL and 3 μg/mL, respectively) for two days. “Acute” cells were removed from the activation plate and cultured with IL-2, while “chronic” cells were replated on fresh anti-CD3 and anti-CD28-coated plates for an additional 2–4 days. Cells were then assessed for markers of T cell exhaustion, activation, and effector function. B) Inhibitory receptor expression on in vitro-stimulated T cells. Bar graphs of (PD1, TIM3, TOX) percent expression level for each exhaustion marker. Data represent the mean ± SEM (n = 3 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. C) Representative histograms of effector cytokine production (IFN-γ and TNF-α) in CD8⁺ T cells under acute and chronic stimulation conditions. Right, bar plot showing the percentage of TNF⁺IFN-γ⁺ double-positive CD8⁺ T cells under acute and chronic stimulation conditions. Data represent the mean ± SEM (n = 3 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. D) Bar graph illustrating the fraction of 13C-labeled citric acid (calculated as 100% minus the unlabeled fraction, M + 0) from multiple 13C-labeled substrates—acetate, β-hydroxybutyrate (βOHB), glucose, glutamine, and lactate—in in vitro-stimulated CD8+ T cells under acute or chronic conditions. Data represent the mean ± SEM (n = 3 mice per group). E) Bar graph illustrating the isotopomer distribution (M + 1 through M + 6) of 13C-labeled citric acid derived from five different 13C tracers—acetate, β-hydroxybutyrate (βOHB), glucose, glutamine, and lactate—in in-vitro-stimulated CD8+ T cells under acute or chronic conditions. Data represent the mean ± SEM (n = 3 mice per group). F-G) 13C labeling of TCA cycle intermediates in CD8⁺ T cells exposed to acute and chronic antigen stimulation. CD8⁺ T cells exposed to acute versus chronic stimulation as in (A) were cultured in VIM medium containing 0.4 mM [13C2]-Acetate or 0.85 mM [13C4]-βOHB for 2 h. F) Bar graphs of the Mass Isotopomer distribution (MID) for [13C2]-Acetate -derived citrate (M + 2) and malate (M + 2) are shown. G) Bar graphs of the Mass Isotopomer distribution (MID) of [13C2]-Acetate -derived citrate (M + 2) and malate (M + 2) are shown. Data represent the mean ± SEM (n = 3 mice per group). Statistical significance was calculated via two-tailed, unpaired Student's t-test. Exact P-values are as follows: Extended Data Fig. WT (black) and DKO (red) cells were activated for 72 h on plate-bound anti-CD3/CD28 in the presence of 1.5 mM βOHB, then assayed on a Seahorse XF analyzer. Right, n = 10 technical replicates/group), bars are mean ± SEM. Statistical significance was calculated via two-way ANOVA multiple comparisons test. B) Mitochondrial mass in activated CD44⁺ CD8⁺ cells, measured by MitoSpy (geometric mean fluorescence intensity, gMFI). Right, mitochondrial membrane potential in the same population, measured by TMRM (gMFI). Statistical significance was calculated via two-way ANOVA multiple comparisons test. C) Ratio of TMRM to MitoSpy gMFI, indicating membrane potential per mitochondrial unit. For panels (B–C), each bar is mean ± SEM (n = 3 mice per group); no significant differences were observed. Statistical significance was calculated via two-way ANOVA multiple comparisons test. D) Viability of CD8⁺ T cells at 24, 48, and 72 h post-activation, assessed by live/dead staining. Statistical significance was calculated via two-way ANOVA. E) Proliferation kinetics: live cell counts at each timepoint normalized to input at 0 h. Lines represent mean ± SEM (n = 3); neither genotype nor βOHB altered viability or proliferation. Statistical significance was calculated via two-way ANOVA. F) Percent change in total body weight from baseline to two weeks in WT and DKO mice under ad libitum (AL) or dietary restriction (DR; 50%). Each point is one mouse (n = 4 per group); bars are mean ± SEM. Middle, percent change in lean mass over the same period and groups. Right, per cent change in fat mass over two weeks in AL vs. DR WT and DKO mice. Statistical significance was calculated via two-way ANOVA multiple comparisons test. Exact P-values are as follows: Extended Data Fig. A) DR antigen-specific OT-I CD8⁺ T cells from WT (blue) and BDH1/OXCT1 DKO (red) mice were incubated with either U-13C6-glucose (top row) or U-13C4-β-hydroxybutyrate (bottom row), and the incorporation of label into aspartate, citrate, malate, glutamate (M + 2 isotopologues) and fructose-1,6-bisphosphate (M + 3, glucose only) was quantified as isotopologue abundance × pool size. Bars show mean ± SEM (top, n = 3 AL and 4 DR mice per group. Statistical significance was calculated via two-tailed, unpaired Student's t-test. B) Basal ATP production rates (J_ATP) from oxidative phosphorylation (OXPHOS) and glycolysis (GLY) in antigen-specific CD8⁺THY1.1⁺ T cells from wild-type (WT) and double knockout (DKO; Bdh1fl/flOxct1fl/fl Cd4-Cre) mice under DR conditions. T cells were isolated from LmOVA-infected mice at 7 dpi. Statistical significance was calculated via two-way ANOVA multiple comparisons test. C) Maximal ATP production rates (J_ATP) from OXPHOS and GLY under the same conditions as in (B). Statistical significance was calculated via two-way ANOVA multiple comparisons test. D) Schematic representation of anti-IgG and anti-PD1 antibody treatment in mice bearing B16 melanoma tumors under ad libitum (AL) or dietary restriction (DR) feeding regimens. Antibody treatments (200 μg/dose) were administered every three days for a total of five injections, starting on day 7 post-tumor implantation. E) TOX expression in PD1⁺TOX⁺ CD8⁺ TILs: left, representative flow cytometry histograms for IgG- (blue) versus α-PD-1-treated (magenta) mice; right, percentage of TOXHi cells within the PD1⁺TOX⁺ gate. Each symbol is one mouse (n = 8 per group); bars show mean ± SEM. Statistical significance was calculated via two-tailed, unpaired Student's t-test. Exact P-values are as follows: Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Oswald, B.M., DeCamp, L.M., Longo, J. et al. Dietary restriction reprograms CD8+ T cell fate to enhance anti-tumour immunity and immunotherapy responses. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01820-9'>Endoplasmic reticulum disruption stimulates nuclear membrane mechanotransduction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 10:49:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cytosolic phospholipase A2 (cPLA2) controls some of the most powerful inflammatory lipids in vertebrates by releasing their metabolic precursor, arachidonic acid, from the inner nuclear membrane (INM). Ca2+ and INM tension (TINM) are thought to govern the interactions and activity of cPLA2 at the INM. However, as compensatory membrane flow from the contiguous endoplasmic reticulum (ER) may prevent TINM, the conditions permitting nuclear membrane mechanotransduction by cPLA2 or other mediators remain unclear. To test whether the ER buffers TINM, we created the genetically encoded, Ca²⁺-insensitive TINM biosensor amphipathic lipid-packing domain inside the nucleus (ALPIN). Confocal time-lapse imaging of ALPIN– or cPLA2–INM interactions, along with ER morphology, nuclear shape/volume and cell lysis revealed a link between TINM and disrupted ER–nuclear membrane contiguity in osmotically or ferroptotically stressed mammalian cells and at zebrafish wound margins in vivo. By combining ALPIN imaging with Ca2+-induced ER disruption, we reveal the causality of this correlation, which suggests that compensatory membrane flow from the ER buffers TINM without preventing it. Besides consolidating the biomechanical basis of cPLA2 activation by nuclear deformation, our results identify cell stress- and cell death-induced ER disruption as an additional nuclear membrane mechanotransduction trigger. Cells constantly reshape and hence must tell innocuous shape changes from deformations that endanger their integrity. Plasma membrane mechanotransduction is tuned for sensitivity: stretch-activated channels, integrin adhesions and other mechanisms sense minute strains at the cell body surface and its periphery. By contrast, nuclear membrane mechanotransduction (NMMT) engages when the cell body is subjected to severe confinement, stark osmotic swelling or other stresses that pressurize and stretch the nuclear envelope (NE)1. In other words, the nuclear membrane is tailored to detect bulk deformations that may threaten cell and tissue viability and call for an extension of its adaptive range, emergency repair or inflammatory signalling. One of the most powerful and versatile bioactive lipid signalling mechanisms involved in all these aspects, the eicosanoid cascade2, is controlled by NMMT. Specifically, inner nuclear membrane tension (TINM)-dependent cytosolic phospholipase A2 (cPLA2) activation leads to arachidonic acid release from the NE. Arachidonic acid-derived prostaglandins, leukotrienes, oxoeicosanoids and other bioactive lipids control diverse physiological processes ranging from inflammation to musculoskeletal adaptation3,4. The cPLA2–NMMT pathway has been reported to control the osmotic surveillance of epithelial barrier integrity by leukocytes; mitotic entry; confined migration of embryonic, immune and cancer cells; dendritic cell chemotaxis; and cell differentiation5,6,7,8,9,10. Via conserved hydrophobic residues in its Ca2+-dependent membrane-binding C2 domain and its catalytic site, cPLA2 adsorbs to TINM-induced lipid-packing defects (LPDs)11. Notably, a similar (that is, LPD) mechanism mediates curvature sensing by amphipathic lipid-packing sensor (ALPS) domains, for example, of ARFGAP1, on nanometre-sized vesicles12,13. Although the concept of LPD-dependent mechanosensing is well established14,15,16 and has been thoroughly explored in the context of membrane curvature sensing13,17,18, its physiological relevance in the context of NMMT is just emerging. NMMT assumes that strong nuclear deformation causes TINM, but this premise has been questioned. The endoplasmic reticulum (ER) is contiguous with the nuclear membrane, forming a large membrane reservoir. Before TINM can expose a hydrophobic binding surface, nuclear membrane invaginations and evaginations must be unfolded. If a contiguous ER were to behave like a large nuclear membrane reservoir, TINM may not develop until this reservoir is exhausted. If the ER were to instantly quench TINM by compensatory membrane flow, NMMT would be impossible. But whether or how the ER affects TINM and NMMT has never been experimentally tested. To this end, we developed the Ca2+-independent, intranuclear lipid-packing probe ‘amphipathic lipid-packing domain inside the nucleus' (ALPIN), which allowed us to monitor TINM dynamics during nuclear swelling in the presence or absence of a contiguous ER. The ER is normally contiguous with the nuclear membrane, but different types of stress can disrupt this connection through Ca2+-dependent ER vesiculation19,20,21,22,23,24,25. Confocal time-lapse analysis revealed that transient nuclear volume increases and envelope smoothening coincided with ER vesiculation and cPla2 adsorption to the INM (Fig. These acute osmotic responses (‘shock' phase) reversed within ~10–15 min probably through regulatory volume decrease (RVD)26. It is known that the cell nucleus behaves like an osmometer that follows cytoplasmic volume changes upon osmotic shock and RVD, respectively27,28,29,30. a, Left: representative confocal midplane slices of U2OS cells expressing the luminal ER marker eGFP–KDEL and zebrafish cPla2–mKate2 at baseline and 5 min after ΔΠ = 270 mOsm osmotic shock. Top middle: a plot of normalized ER surface area versus time after hypo-osmotic shock treatment (∆Π = 270 mOsm). The data represent six independent experiments. Bottom middle: a plot of normalized ER circularity, and cPla2–mKate2–INM adsorption versus time after hypo-osmotic shock (∆Π = 270 mOsm). The data represent three independent experiments. Top right: a scatter plot showing the correlation between cPla2–mKate2–INM adsorption and ER vesiculation (as measured by a decrease of ER surface area) in cells after being exposed to hypo-osmotic (∆Π = 303 mOsm, n = 33) or isosmotic medium (∆Π = 0 mOsm, n = 34) for 5 min. Bottom right: a scatter plot showing the correlation between cPla2–mKate2–INM adsorption and ER fragmentation (as measured by an increase of ER network circularity) in cells after being exposed to hypo-osmotic (∆Π = 303 mOsm, n = 33) or isosmotic solution (∆Π = 0 mOsm, n = 34) for 5 min. The data represent one independent experiment performed on cells from two distinct biological sources (separate frozen vials). b, Left: representative, confocal midplane slices of U2OS cells expressing the ER membrane marker eGFP–SEC61B acquired at the indicated times after ∆Π = 270 mOsm or 303 mOsm hypo-osmotic shock. Right: time-lapse quantification of ER circularity upon osmotic shock. Data represent two independent experiments for 222 mOsm, three independent experiments for 270 mOsm and two independent experiments for 303 mOsm. c, Left: representative lattice SIM2apotome super-resolution maximum intensity projections (MIPs) of U2OS cells expressing the ER membrane marker eGFP–SEC61B (∆Π = 0 mOsm, n = 6). Right: a representative super-resolution MIP of U2OS cells 10 min after hypo-osmotic shock (∆Π = 270 mOsm, n = 4). d, A cartoon scheme of the FLIP experiment. e, Fluorescence dynamics in response to bleaching the indicated sites. Left: FLIP of eGFP–KDEL at baseline (∆Π = 0 mOsm, n = 10) or during osmotic shock (∆Π = 270 mOsm, n = 8). Right: FLIP of membrane, eGFP–SEC61B at baseline (∆Π = 0 mOsm, n = 6) and osmotic shock (∆Π = 270 mOsm, n = 10). The data represent three independent experiments. If not otherwise indicated, n denotes the number of analysed cells or nuclei. Note that ER morphology measurements (normalized circularity/area) are always analysed on an entire FOV (that is, no structure segmentation, n represents FOV, 1–8 cells per FOV). For time-lapse movies, values were normalized to baseline at t = 0. For single images captured at the indicated time point, values were normalized to measurements obtained under isosmotic conditions. A, area; S, ratiometric signal; Cir, circularity. I, intensity; Ctr, non-targeted/bleached neighbouring cell control in whole field of view. The RVD-related nuclear volume decrease was accompanied by lamina shrivelling and cPla2–INM desorption, in line with NE relaxation (Extended Data Fig. Thereafter, the nuclear volume remained steady for ~1–2 h despite the ongoing osmotic stress, while ER vesiculation proceeded. As ER vesiculation plateaued, nuclear swelling, NE smoothening and cPla2–INM adsorption rebounded, indicating TINM. This second nuclear swelling was perhaps caused by RVD failure related to cell damage and/or metabolic exhaustion. The progressive nuclear shrinkage that followed markedly differed from RVD-related nuclear shrinkage: it took hours not minutes and did not involve nuclear shrivelling or cPla2desorption, arguing against NE relaxation. Conceivably, nuclear volume loss, unlike during RVD, was accompanied by a nuclear surface decrease of unknown mechanistic origin. Whether the ER–outer nuclear membrane (ONM) vesiculation directly contributed to the putative surface loss remains an interesting question for future research. After ~3 h of osmotic differential (ΔΠ) = 270 mOsm, cell lysis became visible by nuclear Sytox blue positivity (or earlier after ΔΠ = 303 mOsm shock). In such cases, cytoplasmic cPla2 fluorescence became an inverse cell lysis marker, whose loss mirrored the rise of the Sytox blue signal (Extended Data Fig. At ΔΠ = 220 mOsm, ER vesiculation rarely occurred, at ΔΠ = 303 mOsm it did not reverse (Fig. Apparently, ER vesiculation is linked to critical (that is, outside the adaptive range), rather than mild, nuclear volume perturbations. 1e and Supplementary Videos 2and 3) confirmed that ER vesiculation split the ER from the NE. Furthermore, fluorescence loss in photobleaching (FLIP) experiments revealed that diffusional eGFP–KDEL or eGFP–SEC61B marker exchange between the vesiculated ER and the NE (Fig. 1d,e) was completely abrogated, in line with previous findings25. By severely disrupting ER–NM contiguity, far more than any previously described ER shape factor mutation, this phenomenon offered itself to probe the role of the ER in TINM/NMMT regulation. The LPD-binding capacity of zebrafish cPla2 renders it sensitive to TINM, but only in the presence of Ca2+(refs. The codependence of Ca2+ for both sensing TINM and disrupting the ER renders the relationship between both parameters hard to disentangle by using the INM adsorption or enzymatic activity of cPla2 as TINM proxy. Its TM sensitivity has been previously confirmed by micropipette aspiration (cycling between 0.1 and 0.3 mN m−1) and osmotic swelling experiments using giant unilamellar vesicles (GUVs)11,13. Each ALPS motif forms an amphipathic helix that inserts its hydrophobic face into LPDs created by membrane tension (Fig. As predicted by its lack of known Ca2+ binding sites, the tension dependent GUV interactions of ARFGAP1–ALPS were Ca2+ insensitive (Extended Data Fig. Just like the C2 domain of cPla2, ALPS motifs target LPDs but because nuclear membranes are mostly flat, tension—not curvature—is the dominant generator of LPDs in this compartment. On stretched synthetic bilayers, ARFGAP1–ALPS competes with cPla2 for the same LPD sites, confirming a shared TINM-sensing mechanism. However, its apparent affinity for GUVs (Kd′ ≈ 150 nM) is roughly eightfold lower than that of the cPla2 C2 domain (Kd′ ≈ 20 nM at 20 µM Ca²⁺, ΔΠ = 240 mOsm)11. Nevertheless, computational image processing of confocal midsections allowed us to robustly measure ALPS recruitment to the nuclear rim despite its lower sensitivity (Methods). Yellow, hydrophobic; purple, serine and threonine; grey, glycine and alanine; blue, basic residues. Bottom: a hypothetical cartoon scheme of ALPS (orange) adsorption to stretch-induced LPDs (red). ALPS adopts a helical, amphipathic conformation upon insertion into LPDs but remains largely unstructured in solution. b, Left: cropped representative, confocal midplane slices of ALPIN1 (ALPS1–mKate2 3XNLS) and ALPIN2 (ALPS1–2–mKate2 3XNLS)–INM interactions before and after hypo-osmotic shock (∆Π = 270 mOsm). Right top: a scheme of the ALPS-based TINM biosensors used in this study. Right bottom: time series analysis showing normalized sensor adsorption (left, ALPIN1, n = 26 cells from 5 independent experiments; right, ALPIN2, n = 15 cells from 2 independent experiments) to the INM after hypo-osmotic shock (∆Π = 270 mOsm). Note that ALPIN1 was co-expressed with eGFP–SEC61B, and portions of the ALPIN1 data presented here and the 270 mOsm condition in Fig. 1b were reanalysed from the same experiment. c, Left: representative confocal midplane slices showing ALPIN1–INM interactions and ER morphology (eGFP–SEC61B) at baseline (0 min) and 5 min after hypo-osmotic shock (∆Π = 270 mOsm). Right: a scatter plot showing the correlation between ALPIN1 NM adsorption and ER disruption in cells treated for 5 min with ∆Π = 0 mOsm (open circles, n = 52), 270 mOsm (blue crosses, n = 38) or 303 mOsm (red squares, n = 39). The data represent one independent experiment performed on cells from two distinct biological sources (separate frozen vials). d, A cartoon scheme of the cell permeabilization experiment. e, Representative SIM midplane slices of U2OS cells permeabilized with the indicated lysis buffer supplements acquired at 60 nm xy resolution 60 min after the start of the experiment. Note that the representative slices were taken from the data analysed in Extended Data Fig. 2g representing one independent experiment performed on cells from two distinct source vials. f, Time series quantification of ALPIN1 adsorption to the INM after dilution of [PVP360] from 2.5% to 1.9% (n = 23 from 2 independent experiments), 1.25% (n = 24 from 4 independent experiments), 0.6% (n = 26 from 4 independent experiments) or 0% (n = 25 from 4 independent experiments) under Ca2+-free conditions (Methods). ***P = 0.0005, **P = 0.003 and *P = 0.02, determined by a one-way Welch's ANOVA (unequal variance) with post hoc Games–Howell test on ALPIN1 adsorption at t = 15 min. g, Left: representative confocal midplane slices of ALPIN1 NM adsorption at the indicated colloid stress and Ca2+ conditions at baseline (0 min) and 2 min after the shift. Middle: time series quantification of ALPIN1 adsorption to the INM at the indicated conditions. Insets show the adsorption rates determined by OLS fitting of the linear portion of the plots (between t = 4 min and 5 min). ****P = 2.5 × 10−5 (middle) and P = 1.5 × 10−5 (right), determined by a two-sided Student's t-test (equal variance; middle) and a two-sided Welch's t-test (unequal variance; right) on ALPIN1 adsorption at t = 20 min. If not otherwise indicated, n represents the number of analysed cells or nuclei. For time lapse movies, the values in subsequent time points were normalized to values at t = 0. For single time point images, values from hypo-osmotic conditions were normalized to isosmotic controls. We fused either one (ALPIN1) or both (ALPIN2) ALPS motifs of ARFGAP1 to a triple nuclear-localization signal (3×NLS) to create nuclear-targeted sensors (Fig. ALPS motifs attain their α-helical amphipathic conformation upon binding LPDs of stretched or curved membranes31. If no LPDs are available, ALPS domains are largely unstructured, which may favour their nucleolar sequestration32,33. Hypotonic shock caused partial nucleolar dissolution as previously reported34,35 along with ALPIN1 adsorption to the INM (Fig. Compared with ALPIN1, ALPIN2 showed a stronger baseline INM binding along with constitutive cytoplasmic vesicle association (Fig. These perinuclear ALPIN signals distracted our single-channel TINM detection method (nuclear edge segmentation along with rim:centre normalization based on the ALPIN channel; Methods), which allowed us to use the green and blue channel for simultaneous imaging of ER morphology and cell lysis (Sytox blue), respectively. When fluorescence channels are not limiting, ALPIN2 imaging with normalization by NE marker (eGFP–Lmnb1) fluorescence might yield more sensitive TINM detection, given its higher membrane affinity (Fig. Like cPla2, ALPIN1 showed reversible membrane adsorption during the acute phase and persistent INM-binding during and beyond the prelytic phase of hypotonic shock. Again, ER vesiculation and INM adsorption were correlated (Fig. Apart from its nucleolar localization and lower membrane binding, ALPIN1 mimicked the cPla2 response to osmotic shock, but without the Ca2+ requirement. By contrast, the cytosolic ALP-sensor amphipathic lipid-packing sensor domain inside the cytosol (ALPIC2; ARFGAP1–ALPS without NLS fusion) showed little ONM binding upon osmotic shock (Extended Data Fig. The discrepancy between the shock-induced INM or ONM adsorption of ALPIN or ALPIC (compare Fig. 2f), respectively, could be due to a ONM–INM membrane tension gradient. Perhaps compensatory lipid flow between the INM and ER–ONM is throttled by the nuclear pores36. It has previously been shown that the nuclear shape-sensing function of cPla2 depends on its nuclear localization6. Consistent with these results, our data highlight the INM as a ‘privileged' mechanosensing surface. To more closely examine how ER–NM continuity shapes TINM, we tracked ALPIN1 in a permeabilized cell assay where Ca²⁺ levels and colloid osmotic stress (ΔΠcolloid) can be independently altered (Fig. Using low-dose digitonin, we removed cholesterol from the plasma membrane, perforating it while leaving the cholesterol-poor nuclear membrane intact. The permeabilization buffer contains 360-kDa polyvinylpyrrolidone (PVP) to balance extra- and intranuclear colloid osmotic pressure upon lysis; stepwise dilution of PVP then produces controlled nuclear swelling at adjustable Ca²⁺ concentrations. In permeabilized cells, micromolar Ca2+ disrupts the ER, unlike nuclear swelling (Fig. Even with a contiguous ER, we observed ALPIN1–INM adsorption responding to colloid osmotic shock (Fig. 2f), suggesting that compensatory membrane flow from the ER does not prevent the INM from becoming tense and developing LPDs during nuclear swelling. To address this, we measured ALPIN1 adsorption upon increasing ΔΠcolloid in the presence (0 μM Ca2+) or absence (50 μM Ca2+) of an intact ER (Fig. The ALPIN1–INM adsorption, despite being itself Ca2+ independent (Extended Data Fig. 2a), was consistently higher when the ER membrane reservoir was severed from the nucleus by micromolar Ca2+. These data argue that the ER incompletely buffers TINM, even when ER–NM contacts are intact. Despite its overall structural contiguity, only parts of the ER seem to serve as nuclear membrane reservoir. Accordingly, we never witnessed an overall collapse of the ER into the nucleus. Even when digitonin-permeabilized cells were kept at colloid osmotic equilibrium (2.5% PVP) so that their nuclei did not show overt swelling, Ca2+-dependent ER vesiculation promoted ALPIN1–INM adsorption (Extended Data Fig. This suggests that TINM rises not only when the nuclear water influx outpaces the ER-supplied membrane area, but also when ER–ONM vesiculation withdraws membrane from the contiguous ONM–perinuclear ER, reducing the accessible area required to accommodate a given nuclear volume. Accordingly, ER vesiculation may directly elevate TINM and engage NMMT, even in the absence of overt nuclear swelling. Since ER vesiculation (equal to surface loss) has been observed under pathophysiological stress conditions that boost cytoplasmic Ca2+, and since such conditions are tightly linked to cell damage and death22,24, we wondered whether TINM and NMMT may be more broadly linked to critical cell stress and death, beyond the previously characterized mechanical perturbations (ΔΠ, compression). Indeed, we had previously observed cPla2–INM adsorption in the prelytic phase of ferroptotic cell death caused by the glutathione peroxidase 4 inhibitor ML162 (ref. 37); however, we did not consider possible contributions of the ER at this time. Starting ~40 min after ML162 exposure, we observed continuous nuclear shrinkage and ER vesiculation, along with ALPIN1/cPla2 adsorption to the INM (Fig. Nuclear swelling after ML162 treatment, if at all noticeable (Extended Data Fig. 3a), was milder and more inconsistent than upon osmotic shock (Fig. 3a,c,d and Extended Data Figs. Akin to the prelytic phase of osmotic shock (Extended Data Fig. 1c), nuclear shrinkage after ML162 treatment was accompanied by NE smoothening (Extended Data Fig. Towards the end of the experiment and just before plasma membrane failure, we frequently noticed pronounced NE blistering, consistent with previous reports38. These blisters contained eGFP–KDEL and appeared to expand at the expense of nuclear volume (Fig. Ferroptosis, like other necrotic or hypo-osmotic insults, produces prelytic cell swelling37,39, which dilutes cytosolic colloids and—provided NE area can be supplied—should drive nuclear volume expansion. However, depletion of the NE area reservoir (for example, via ER–ONM vesiculation or NE blistering) may reduce the envelope's accessible area and elevate TINM at near-constant (or even reduced) nuclear volume, thereby engaging NMMT, as observed. a, Left: a representative time lapse montage of ALPIN1–INM adsorption, ER morphology (eGFP–SEC61B) and cell lysis (Sytox blue) in a cell undergoing ferroptotic cell death upon exposure to the GPX4 inhibitor ML162. Top right: nuclear rim profiles (20 pixels wide) of ALPIN1 midplane fluorescence at the indicated location (inset, yellow line). Temporal order is indicated by line width and colour code. Bottom right: the temporal profile of the segmented nuclear area (open circles), cytoplasmic ALPIN1 (ROI on the extranuclear area, red circles) and nuclear Sytox blue signal (ROI on the intranuclear area, blue circles) as determined from confocal MIPs. Note that the representative time lapse montage was taken from the data analysed in Fig. b, Representative SIM midplane slices at 60 nm xy resolution of U2OS cells, expressing the luminal ER marker eGFP–KDEL, treated with the GPX4 inhibitor ML162 acquired 120 min after drug treatment. The data represent two independent experiments. c,d, Time lapse analysis of ER circularity (blue), nuclear volume (black) and ALPIN1–INM adsorption (red) (c) or cPla2–INM adsorption (d) at the indicated times after ML162 treatment. ALPIN1 data represent two independent experiments. Note that because ALPIN1 leaked from most cells after lysis, only movie segments acquired before leakage were analysed as leakage would otherwise result in broken tracks. Zebrafish are freshwater animals, and when their epidermal barrier is damaged, cells and nuclei around the wound site swell by hypotonic shock5,41. The wound-induced osmotic shock elicits wave-like cPla2 activation (travelling at ~50 μm s−1 through the tissue) with irreversible cPla2–INM adsorption observable exclusively at the wound margin. By contrast, reversible cPla2–INM adsorption is typically seen farther away, in cells not directly compromised by the injury procedure itself, such as perivascular macrophages5,42,43. The hypotonic shock causes cPla2-dependent arachidonic acid release5,44,45. Enzymatic or non-enzymatic lipid peroxidation converts arachidonic acid into chemotactic oxoeicosanoids that attract distant immune cells to the wound margin and regulate mucosal resilience46,47,48,49 To test whether the correlation between ER vesiculation and NMMT holds in live tissue, we turned to the well-established zebrafish tail fin wounding model. Using confocal microscopy, we simultaneously imaged cPla2 translocation (cPla2–mKate2) and ER morphology (eGFP–KDEL) in vivo. In larvae, transient cPla2 adsorption to the INM occurs extremely fast and lasts only ~10 s. To better capture transient INM translocation at the reduced time resolution of two-channel acquisition (that is, 15 s per frame), we depolymerized F-actin with latrunculin, which prolongs the osmotic swelling response to injury5, perhaps in part by modulating regulatory volume regulation50. Epithelial cells directly adjacent to the blast zone showed instant, irreversible ER fragmentation with permanent cPla2 translocation, as in the prelytic phase of osmotic shock and ferroptosis (Figs. In line with previous work5,42,43, cells further away displayed transient cPla2 recruitment without ER disruption. Expectedly, the reversible cPla2–INM adsorption was harder to temporally discern without latrunculin treatment, but the correlation of ER vesiculation and cPla2translocation directly at the wound margin was nevertheless evident (Extended Data Fig. This lasting cPla2 activation may be responsible for residual neutrophil recruitment, independent of hypotonic shock42. Transient cPla2 translocation to the INM, by contrast, requires osmotic shock. It mobilizes the majority of first-wave leukocytes and—via macrophages—promotes rapid serum exudation from nearby blood vessels, as reported previously42,43. a, Top: confocal maximum intensity projection of cPla2–mKate2 and luminal ER marker eGFP–KDEL in latrunculin A-pretreated zebrafish larvae after wounding under hypo-osmotic conditions (Π ~ 10 mOsm) at t = 00:01:15. Middle: magnification of eGFP–KDEL or cPla2 in individual cells within (position 1) and just outside (position 2) the wound region. Constitutive cPla2 translocation with ER fragmentation at the wound margin was observed in all wounded fish (n = 5). Reversible cPla2 translocation without ER fragmentation (in the second or third cell layer) in four of five fish. * indicates centre of the UV-laser blast. b, The average radius of the laser wound and ER vesiculation zone from n = 7 larvae. These radii values were approximated from polygon area measurements from the dataset shown in a. c, An overview cartoon scheme outlining the approximate timing of stress responses to osmotic (ΔΠ), oxidative (ML162) or physical (laser wound) insults in vitro (black) or in vivo (red). Moderate osmotic shock (ΔΠ = 270 mOsm) causes rapid nuclear swelling, partial ER fragmentation and TINM, which is recovered by RVD. Under prolonged osmotic stress, cells enter the prelytic stage, with strong ER vesiculation and TINM, before they eventually lose plasma membrane integrity. Strong osmotic shock (ΔΠ = 303 mOsm), oxidative stress, and physical injury cause irreparable cell damage (dashed line), which directly initiates the prelytic stage. Left-column engagement of NMMT during each stage. As a death-transcendent ‘danger-signalling' mechanism, NMMT via cPla2 is initiated before lysis and persists beyond. d, A hypothetical scheme of NMMT buffering by the ER. Middle: osmotic shock (ΔΠ) must unfold nuclear membrane invaginations before tension (T) can develop. Top: when the ER is contiguous with the nuclear membrane during homeostasis or mild stress, a compensatory lipid flow (dashed red line) partially relieves T. This amounts to less LPDs (shaded red regions/gauge) on the INM and less, or more reversible, hydrophobic cPla2–INM interactions, as observed in cultured cells exposed to moderate osmotic stress, in distal wound cells (a, position 2), or in perivascular macrophages (see ref. Bottom: when the ER is vesiculated by high and persistent Ca2+ after critical stress, there is no compensatory lipid flow to attenuate T and LPDs. This amounts to high (and more stable) cPla2–INM interactions. The respective cell is turned into a constitutive, inflammatory lipid signalling hub. This is observed, for example, in wound margin cells (a, position 1), prelytic osmotically stressed, ferroptotic cells, and post-lytic (that is, permeabilized) cell corpses. Depending on the amplitude or duration, different kind of cell stresses (osmotic, oxidative or laser) cause (1) reversible ER fragmentation with reversible TINM/NMMT or (2) persistent ER fragmentation with persistent TINM/NMMT, with the latter being indicative of prelytic cell damage (Fig. Importantly, cPla2 signals from stressed, severely damaged or dead cells alike, as long as Ca2+ and TINM are maintained5. The removal of nuclear membrane slack by ER vesiculation thus enhances TINM and contingent inflammatory NMMT, irrespective of a cell's vital state (Figs. By harnessing the Ca2+-independent TINM sensor ALPIN, this study reveals that the ER serves as a stress/Ca²⁺-gated nuclear membrane buffer, quenching NMMT under homeostatic conditions. This NMMT break is released upon critical stress. Persistent ER vesiculation in damaged cells locks the nucleus in a high-tension state that stabilizes cPla2 on the INM—this occurs while the cell is technically still alive and it persists beyond cell death: placing swollen HeLa cell nuclei next to an isotonically balanced zebrafish wound mediates cPla2-dependent neutrophil recruitment just like hypotonic shock at wound sites5. Thus, the cPla2–NMMT pathway is a cell death-transcendent mechanism, not exclusively tied to a particular type of stress (for example, ΔΠ), nor to cell viability or cell death—it is a simple, physical process that just relies on Ca2+ and concomitant nuclear pressurization. It can trigger restitutive mechanisms (repair, inflammation and so on) hours before the irreversible loss of plasma membrane integrity releases immunogenic molecules into the extracellular space that promote canonical damage-associated molecular pattern signalling. TINM is never fully abrogated by the ER, suggesting that compensatory lipid flow from this organelle into the INM is throttled by obstacles that remain to be described. Anchored transmembrane proteins inhibit lipid flow, as previously proposed for Lem2 in yeast51. It is also conceivable that the nuclear pores act as topological bottlenecks for nuclear membrane/ER area redistribution36. The incompleteness of TINM buffering by an intact ER permits reversible cPla2–NMMT in cells distal to the wound margin, such as perivascular macrophages43. To probe ER–NM interorganelle mechanics, we vesiculated the ER with Ca2+, mainly because the perturbation of known ER shape genes52,53 does not yield a comparable disruption. The molecular mechanisms underlying ER vesiculation deserve further investigation. Identifying the putative Ca2+-regulated vesiculation factor(s), could inform therapeutic strategies that attenuate sterile inflammatory signalling by preserving the ER–NM contiguity in damaged cells. U2OS (HTB-96, ATCC) cells were grown in Dulbecco's modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum, 2 mM l-glutamine and 1% penicillin–streptomycin. Lipofectamine 3000 (Thermo Fisher) was used to transfect U2OS cell lines according to the manufacturer's instructions. To establish stable transgenic cell lines, cells were co-transfected with SB100X Sleeping Beauty transposase and a pSB/MCS/Puro transposon vector encoding the genes of interest under the control of the EF1α or CMV promoter in 2:1 ratio. At 48 h after transfection, cells were grown in complete growth medium with 1 µg ml−1 puromycin for 1–2 weeks. mKate2 and eGFP double-positive cells were sorted by flow cytometry. For imaging experiments, cells were incubated with L-15 LEIBOVITZ (Thermo Scientific, 21083027), which is buffered by phosphate salts, free‑base amino acids, and 0.9 g l−1 galactose (in place of glucose), with 0.55 g l−1 sodium pyruvate and 2 mM l‑glutamine. The 1,2-dipalmitoyl-sn-glycero-3-phosphocholine (DPPC) and 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine-N-(lissamine rhodamine B sulfonyl) (18:1 Liss Rhod PE) were purchased from Avanti Research. For bacterial expression, human ARFGAP1 ALPS1–2-eGFP (human ARFGAP1, amino acids AA192–304) were cloned into a modified pGEX 6P-1 backbone via EcoRI and NotI, leading to N-terminal GST fusion with Human Rhinovirus-3C (HRV-3C) protease cleavage sites. For mammalian expression, rat ARFGAP1 ALPS1-mKate2 with 3XNLS (rat ARFGAP1, amino acids AA196-234) and human ARFGAP1 ALPS1–2-mKate2 with or without 3XNLS (human ARFGAP1, amino acids AA192–304) were fused to eGFP-Lmnb1 or eGFP-SEC61B through P2A peptides on the C-terminus and subsequently cloned into the pSB/CMV/MCS/Puro transposon plasmid. Within this plasmid, an expression cassette containing genes of interests is driven by a constitutive CMV promoter, along with another expression cassette that encodes a puromycin selection marker under the control of a synthetic SV40 promoter. Both expression cassettes are flanked by inverted terminal repeats that contain transposase recognition sites and allow transposition into host genome when co-expressed with the hyperactive Sleeping Beauty transposase (SB100X). For ALPS fusing with zebrafish eGFP-Lmnb1, ALPS sequences were inserted into the backbone harbouring eGFP-Lmnb1 via NheI and BshTI. For ALPS fusing with human eGFP–SEC61B, the plasmids harbouring both ALPS motifs and eGFP-LMNB1 were digested with BglII and XbaI, and SEC61B was ligated in to replace Lmnb1. mCherry-SEC61B and eGFP-Lmnb1 were also fused and subsequently cloned into the same transposon plasmid to investigate lamina shrivelling during hypo-osmotic stress and the prelytic phase of ferroptotic cell death. Briefly, mCherry-SEC61B was inserted into the vector backbone harbouring eGFP-Lmnb1 via NheI and EcoRI. Zebrafish cPla2-mKate2 was fused to eGFP-KDEL through a P2A peptide and subsequently cloned into a similar pSB/EF1 α/MCS/Puro transposon plasmid, in which the genes of interests were under the control of a constitutive and mammalian-endogenous EF1α promoter. Helical wheel representations of full-length ALPS1 and ALPS2 motifs are provided in Extended Data Fig. Detailed maps of pSB/CMV/MCS/Puro transposon plasmid are provided in Extended Data Fig. The full lists of plasmids and PCR primers used in this study are provided in Supplementary Tables 1 and 2. To generate GUV membranes composed of 99.5% DPPC and 0.5% 18:1 Liss Rhod PE (mol/mol), we used a standard electroformation protocol11,54. Briefly, 0.11 μM of lipid mixture dissolved in chloroform was spread on 30–60 Ω indium tin oxide coverslips (Structure Probe Inc. supplies) and dried under a N2 stream. Indium tin oxide coverslips were assembled into an electroformation chamber with 500 mM sucrose solution inside and incubated at 60 °C. The chambers were connected to a function generator (RIGOL DG1022). A sine wave with a frequency of 10.0 Hz and peak-to-peak voltage at 1.41 V was applied for 4–6 h followed by a square wave with a frequency of 4.5 Hz and peak-to-peak voltage at 2.12 V for 30 min. After electroformation, GUVs were extracted from the chamber using a gel-loading tip. The chamber was washed two times with 500 mM glucose solutions to collect residual GUVs. GUVs were used for experiments within 5 days after preparation. The pGEX 6P-1 expression vector harbouring tagged ARFGAP1-ALPS1–2-eGFP was transformed into Rosetta 2 pLySs cells (Novagen). This vector is used to express GST fusion proteins with a PreScission protease site. Expression of ARFGAP1-ALPS1–2-eGFP was induced by 0.2 mM IPTG at 37 °C under vigorous shaking (250 rpm) for 3 h in LB medium supplemented with ampicillin (100 µg m−1) at OD600 = 0.8. After expression, bacteria were first centrifuged at 2,600 rcf for 30 min, pelleted and frozen at −80 °C overnight. Afterwards, bacteria were resuspended in 25 mM HEPES and 0.1% Triton X-100 with Complete protease inhibitors (Roche) at pH 7.4. Sonication was conducted on an ice/water mixture for 7–10 min with the following settings: 50% amplitude, 1 s on/2 s off pulses (Thermo Fisher Scientific, Sonic Dismembrator Model 500). After sonication and centrifugation at 24,000 rcf for 30 min at 4 °C (Sorvall RC 6 PLUS, Rotor SS-34), the supernatants were collected and incubated with glutathione sepharose 4B gel beads (Cytiva) overnight at 4 °C. The beads were then washed three times in a buffer containing 25 mM HEPES and 200 mM KCl (pH 7.4). Beads with GST-tagged ARFGAP1-ALPS1–2-eGFP were incubated with HRV-3C protease (PreScission protease, Cytiva) in a buffer containing 25 mM HEPES and 200 mM KCl (pH 7.4) overnight at 4 °C to elute ARFGAP1-ALPS1–2-eGFP. Eluates were concentrated with Amicon Ultra-2 10 K centrifugal filter devices (Millipore Sigma). The total protein concentration was determined using the Pierce BCA Protein Assay Kit (Thermo Fisher Scientific). Protein purity was determined by SDS–PAGE and Coomassie blue staining. Aliquots were snap frozen in liquid nitrogen and stored at −80 °C. The binding experiments were performed according to a standard protocol11. In summary, concentrated GUVs were transferred to a hypo-osmotic binding buffer containing 0 or 20 µM Ca²⁺ (Supplementary Table 3) and ALPS1–2–eGFP domains at varying concentrations (ranging from 0 to 500 nM) in BSA-coated glass-bottom dishes (20 mg ml−1 BSA solution was added 1–2 h before the experiment to reduce GUV membrane ruptures during prolonged incubation). Specifically, GUVs were loaded into hypotonic solution by gently dispensing them with the pipette tip directed onto the glass surface to ensure the vesicles settled at the bottom. GUVs were left in the binding solution for 2 h until protein recruitment approached an equilibrium. Then GUV membrane and protein fluorescence images were simultaneously acquired using a 40× objective and the protein adsorption fluorescence was quantified, normalized and adsorption isotherms were fitted as described in ‘Image acquisition and computational analysis for cell culture and GUV experiments' section. All steps, including incubation, were carried out at room temperature to minimize the effects of thermal drifts during image acquisition. Ninety-six-well glass-bottom dishes (Cellvis) were coated with 0.1% gelatin in PBS without Ca2+ and Mg2+ for at least 15 min at 37 °C before washing 3× with PBS. U2OS cells stably expressing fluorescent markers were then seeded in 96-well dishes (~3 × 104 cells per well) 24 h before the start of the imaging experiments in the specified cell culture solutions (see above). Before the experiment, the cell culture medium was exchanged to Leibovitz's L-15 medium (Thermo Fisher). Under isosmotic conditions, U2OS cells were incubated in L-15 stock culture medium to obtain a final osmotic pressure of 341 mOsm. To induce different levels of hypo-osmotic shock, the cell imaging medium was diluted by the addition of double-distilled water supplemented with 1.26 mM CaCl2 (Supplementary Table 4). The wells were buffered by a combination of phosphates, free base amino acids (2 mM l-glutamine) and galactose, and did not rely on sodium bicarbonate and CO2 pH buffering. For standard time-lapse imaging, cells were first recorded under isosmotic conditions for 4–5 min before hypo-osmotic shock was applied by dilution. In extended time-lapse experiments, cells were imaged for three frames (6 min total, t = 2 min) under isosmotic conditions before applying the hypo-osmotic treatment and imaged for 6 h. The humidity and temperature of the imaging chamber was supported by a TOKAI-HIT substage heater, calibrated and set to 37 °C at least 30 min before the start of the experiment. During imaging, cells that were in the central region of the well were chosen for image acquisition. To evaluate plasma membrane integrity during osmotic swelling, 50 nM Sytox Blue (Molecular Probes) was included in the culture medium and the dilutant. For single time point experiments, U2OS cells expressing ALPIN1/cPla2–mKate2 and ER markers were exposed to hypo-osmotic solutions for 5 min before being imaged for nuclear membrane binding and ER morphology. To visualize the nucleolus, the adherent U2OS cells expressing either of ALPIN1 (ALPS1–mKate2 3XNLS) or cPla2–mKate2 were stained live with NUCLEOLAR-ID (Enzo, ENZ-51009-500) by incubating the cells with green detection reagent dissolved in L-15 media for 30 min at 37 °C. Subsequently, the cells were transferred to a confocal microscope and left untreated or shocked with hypo-osmotic dilution for 15–20 min (ΔΠ = 270 mOsm) and imaged. The permeabilization protocol was adapted from an established method5. To perform permeabilization in the presence of Ca2+, a modified version of BM was used in which all components remained unchanged except for the EGTA/Na⁺ concentration, which was reduced from 10 mM to 1 mM. Unless otherwise specified, cells were permeabilized in either BM or modified BM supplemented with 12.5 µg ml−1 Digitonin (Cayman Chemical) and 2.5% PVP360 (Millipore Sigma) for over 60 min before imaging, ensuring that the NE was fully relaxed at the start of acquisition. To induce and control TINM (tension-induced nuclear membrane deformation), the concentration of PVP360 was systematically reduced. To assess the behaviour of the ALPS1–mKate2-3xNLS (ALPIN1) sensor and its binding dynamics to the NE during increasing colloid osmotic pressure (Fig. 2f), permeabilized U2OS cells expressing ALPIN1 and eGFP–Lmnb1 were first incubated and imaged in BM supplemented with 2.5% PVP360 for 5 min. Cells were then washed twice with PBS and subsequently incubated and imaged in BM containing decreasing concentrations of PVP360 (ranging from 0% to 2.5%) for an additional 10 min. To systematically test the effects of Ca2+ concentration and colloid osmotic stress on ER structures (Extended Data Fig. 2g), U2OS cells expressing ER markers (eGFP–KDEL) were imaged after incubating for 1 h under four experimental conditions: (1) BM with 2.5% PVP360 (Ca2+ free, no colloid osmotic stress), (2) BM alone (Ca2+ free, while inducing colloid osmotic stress), (3) modified BM with 2.5% PVP360 and 50 µM CaCl2 (presence of Ca2+, but no colloid osmotic stress) and (4) modified BM with 50 µM CaCl2 alone (presence of Ca2+ and colloid osmotic stress). To evaluate TINM responses during increasing colloid osmotic pressures in cells with either intact or disrupted ER networks (Fig. 2g), permeabilized cells expressing ALPIN1 and eGFP–Lmnb1 were imaged for 5 min in BM containing 2.5% PVP360 (intact ER) or in modified BM with 2.5% PVP360 and 50 µM free Ca2+ (fragmented ER network). Subsequently, the PVP360 concentration was diluted to either 1.25% or 0%, while maintaining constant Ca2+ levels. Imaging was continued for an additional 15 min. For diluting the PVP360 concentration from 2.5% to 1.25% or 0%, permeabilized cells were washed two to three times before incubation. Without washes, residual high molecular weight and viscous PVP360 would differentially delay colloid osmotic shock, confounding the interpretation. Imaging was paused during the wash steps and resumed after permeabilized cells were incubated in the diluting medium. Note that cells remained only loosely adherent after permeabilization; therefore, all wash steps were performed gently. Medium was added and removed by dispensing against the well wall rather than directly onto the cells to minimize detachment. To evaluate whether ER vesiculation alone can induce a TINM response under osmotically balanced conditions (Extended Data Fig. 2h), single time point images of U2OS cells expressing ALPIN1 and eGFP–Lmnb1 were taken after permeabilizing in BM containing 2.5% PVP360 (intact ER) or in modified BM with 2.5% PVP360 and 50 µM free Ca2+ (fragmented ER network) for 30 min. For testing the contribution of the ER network to NMMT, membrane (eGFP–SEC61B) or lumen (eGFP–KDEL) localization was used as a proxy for ER dynamics in the U2OS cells. The cells were seeded in DMEM media at a density of 5 × 105 cell ml−1 in 96-well plates that were precoated with gelatin. The media were exchanged to L-15 and cells were imaged with the FLIP approach. FLIP imaging was performed on a Leica TCS SP8 Confocal Laser Scanning system equipped with a Leica white light laser (1.5 mW: 470–670 nm, 78 Mhz, output power of 70.00%), argon lasers (65 mW: 458, 476, 488, 496 and 514 nm continuous) that were used for excitation and photobleaching and a Plan Apochromatic objective (Leica HC PL APO CS2, 63×/NA 1.40 oil). The emission was detected on internal spectral HyD SMD2 sensors for mKate2 (618–660 nm, gain 100/offset −0.22) and eGFP 2× liquid-cooled PMT (498–540 nm eGFP gain 835.8/offset −0.22), respectively. The light path was mounted with a ×2 lens changer (CS2 UV Optics 1) and emission notch filter set and polarization FW mirror (NF 488/561/633). The fluorescent images were acquired at 1,024 × 1,024 pixel resolution in xyt scan mode at 600 Hz scan speed (pixel dwell time of 0.4 μs) at the midplane of the cells of interest in the field of view (FOV) or 184.52 × 184.52 μm at pixel size (voxel 0.18 μm) in x and y direction. The imaging set up was optimized for minimal fluctuations in baseline fluorescence for the duration of imaging, therefore power for lasers were adjusted to 1.5% for 488 nm excitation (eGFP–ER) and 2.5% for 561 nm (mKate2), where the xy scan was bidirectional and pinhole was adjusted to 191 μm (PinholeAiry of 2 AU) with line averaging of 3–4, and data were collected at 12-bit resolution. The photobleaching was achieved by ablating a 2.5-μm radius circle in the FOV with an argon bleach laser line (488 nm, output power of 19.84%) and a simple beam expander for FRAP booster mode. Each FRAP was set up in the Leica Microsystems software (Leica LAS X 3.5.7.23225) using the LAS X FRAP AB module where images were acquired once as a single frame for pre-bleach followed by a series of continuous photobleaching iterations in the same region for three cycles (loops), followed by full FOV acquisition. This sequence was set to repeat until the complete depletion of target cells with the fluorophore of interest (10–25 repeats). The pre-bleach took 5–7 s, while acquisition of FRAP sequence, including three loops per subsequent frame, took approximately 20 s for all imaging experiments. The settings that were determined for the ER network at baseline were applied to the hypo-osmotically shocked cells, with CaCl2 dilutant as described above as the depletion control for critically stressed cells in luminally (KDEL–eGFP) or membrane (eGFP–SEC61B) expressed eGFP variants. For ER vesiculation, the cells were hypotonically shocked (ΔΠ = 270 mOsm) and imaged 5–10 min after treatment. The obtained .lif files were then loaded into Fiji (ImageJ) and stitched, and an invariant circle region of interest (ROI) (width, 5 μm; length, 5 μm) was overlaid on the eGFP (ER) channel and analysed for ROI 1, ROI 2 and ROI 3, where ROI 1 corresponds to photobleached area, the median axis in plane with regard to the target position and the opposite location to the photobleached area with respect to the nucleus. The nucleus was segmented using binary (otsu) thresholding and the segmentation mask was overlaid to the eGFP (ER) channel for quantification of nuclear ER intensity over time. The baseline fluorescence intensity fluctuation in non-targeted cells is recorded as Ctr ROI, referring to non-targeted neighbouring cell control region within the field of view. The data were extracted and analysed in a custom Python script (v3.9). For delineating the states of ER network or its vesiculation, we distinguished these two morphologies with super-resolution fluorescence microscopy. Structured-illumination microscopy (SIM) systems, which achieve a nominal resolution of 60 nm in xy were used in this study. One such example is the Zeiss Elyra 7 microscope equipped with a Plan-Apochromat 63×/1.4 oil DIC M27 objective and 2× pco.edge sCMOS (version 4.2 CL HS, Innovatek LCS-BU) cameras. The 60-nm super-resolution cell images were acquired at baseline (ΔΠ = 0 mOsm) and 10 min after hypo-osmotic shock (ΔΠ = 270 mOsm) and imaged using the Zeiss Elyra 7 microscope with the following build. The light path was equipped with a reflector turret mounted with FW1:LBF (405/488/561/642 laser-blocking filter) beam splitters and filters for red (TV1:LP570) or green (TV2:BP 495–550) emission. The data were recorded in apotome SIM2 in non-delay (five phases for each of the z sections) with 3D leap mode and a FOV of 80.14 μm × 80.14 μm (or 1,280 ×1,280 pixels), 0.063 μm pixel size, 0.329 μm z step, stack 5–12 μm and 50–80 ms exposure time for both colours (camera sensor temperature, air 7 °C) and collected at 16-bit resolution. The acquisition of individual z stacks takes about 12–20 s. The mKate2 and eGFP fluorophores are excited with the 561 nm (1.6% power) and 488 nm (4.1% power) laser lines, respectively. We also generated single time point images in Lattice SIM2 (13 phases) with 3D leap mode with a FOV of 80.14 μm × 80.14 μm (or 1,280 × 1,280 pixels), 0.063 μm pixel size, 0.110–0.150 μm z step and 80 ms exposure time for both colours (camera temperature set to 7 °C). Overall, for each focal plane, 5 or 13 phase images were acquired. Exposure time and laser power were balanced for each fluorescence channel individually to minimize bleaching and exposure time. SIM and Lattice SIM reconstruction was performed with the SIM processing tool of ZEN 3.0 SR FP2 (64-bit, 16.0.20.306 black) software for applying SIM2 image reconstruction on the ER channel as follows: the advanced filter was set with best fit and Gauss (sectioning = 100), we did not use any detrending for either channel or scale to raw image, the set regularization weight was 0.015, with proc.Sampling and Out.Sampling set to (×2 or ×4) and the algorithm was run for 20 iterations. The images (.czi format) were then converted to .ims files and 3D rendered in Imaris with the filament and surface GUI function, where surface grain size (0.093 µm) and the diameter of the largest sphere (0.93 µm) were segmented automatically. The 3D segmentation mask was then triangulated and unpacked as a mesh of vertices and faces with a custom Python pipeline. The interactive data and the method of extraction from SIM2 are available at https://github.com/zazadovv/ER_Nucleus (representative interactive ER rendering data files are available in Demo_file_3D folder and extracted.html and can be viewed from any internet web browser). The mesh array was decimated to maintain only 70% of vertices (30% discarded) and transformed into an interactive 3D STL in a plotly.js (v3.0.1) model for the qualitative evaluation of nuclear ER connections or the absence of a fragmented state. To study ER fragmentation during ferroptosis, cells were imaged after incubating in L-15 medium supplemented with DMSO or 4 μM glutathione peroxidase 4 inhibitor ML162 (Selleckchem) for 2 h. For the temporal correlation of nuclear membrane binding, ER fragmentation and cell death, the time-lapse movies of ML162 were acquired on U2OS cells expressing eGFP–KDEL, cPla2–mKate2 or eGFP–SEC61B, ALPIN1 and Sytox blue. For ML162 the cells were seeded in a gelatin precoated (0.1% gelatin in PBS without Ca2+ and Mg2+) 96-well imaging plate (Cellvis, P96-1.5H-N) at a density of 1 × 104 cells ml−1 and allowed to adhere overnight. Before imaging, the DMEM culture medium was exchanged to L-15 supplemented with 50 nM Sytox blue and transferred to a 37 °C pre-heated microscope chamber. During imaging, after the first time point, the wells were either incubated with 4 μM ML162 or DMSO and 50 nM Sytox blue (Thermo Scientific, S11348) for 2 h, where each z stack (z = 17 steps) was collected every 10 min (t = 10 min, 13 frames) on a spinning-disc confocal microscope. Adult transgenic Casper Zebrafish (Danio rerio) strains were maintained by adhering to the guidelines of the care and use of laboratory animals of the National Institutes of Health and Office of Laboratory Animal Welfare recommendations. The animal procedures were in compliance with an animal protocol approved by the Institutional Animal Care and Use Committee of the Memorial Sloan Kettering Cancer Center (protocol no. The adult Tg(hsp70i:cPla2-mKate2-P2A-eGFP-KDEL) fish were reared in 2.8 l polycarbonate tanks at animal density 10 fish per litre. The anaesthesia for the offspring was all conducted with 0.2 mg ml−1 3-amino benzoic acid ethyl ester (Sigma, MS-222, E10521), (pH 7.0), buffered in 0.5 mg ml−1 anhydrous sodium phosphate dibasic (Fisher, BP332-500). The animals were staged by days post-fertilisation (d.p.f.). and all in vivo experiments were conducted at these larval stages. The embryos were collected from natural spawning and raised in standard hypo-osmotic E3 containing 0.1% (w/v) methylene blue (Sigma-Aldrich, M9140) for first 24 h followed by E3 medium (5 mM NaCl (Sigma-Aldrich, S7653), 0.17 mM KCl (Sigma-Aldrich, P9333), 0.33 mM CaCl2 (Sigma-Aldrich, C5670) and 0.33 mM MgSO4 (Sigma-Aldrich, M7506)) in 100 mm Petri dishes (Fisher Scientific, FB0875713) in complete darkness. Fertilized Casper zebrafish embryos were collected and injected at the one-cell stage into the cytoplasm with a Nanoject II microinjector (Drummond Scientific). Plasmids were assembled with the Gateway multisite cloning kit using LR Clonase (Invitrogen, C12537-023) and multicloning sites between the gateway att sites flanking the cassette of p5E (hsp70i), pME (cPla2-mKate2) and p3E (P2A-eGFP-KDEL) plasmids that were recombined into Tol2kit Destination pDESTcrybb1 for generating Tg(hsp70i:cPla2-mKate2-P2A-eGFP-KDEL). The middle entry pME vector was assembled from the cPla2 (Ensembl: ENSDARG00000024546) open reading frame amplified by PCR and ligated into pDONOR221 containing the pME gateway compatible att sites and subsequently fused in frame to mKate2 (Evrogen) as a C-terminus fluorescent tag with a 15 amino acid GS-enriched linker. The p3E entry vector bears a self-cleaving peptide in the NTD of eGFP that is followed by the ER localization signal KDEL and SV40 pA. For transgenesis, Tol2kit transposase mRNA was transcribed from the NotI linearized pCS2FA-transposase plasmid with the mMESSAGE mMACHINE SP6 reverse transcription kit (Thermo Scientific, AM1340). The final transgenesis construct was injected into one-cell stage zebrafish embryos (~2.7 nl per embryo) at a concentration of 25 ng μl−1 along with 25 ng μl−1 Tol2kit transposase mRNA. Among the injected animals, fluorescence-positive siblings in the eye were selected and raised in husbandry and backcrossed at sexual maturity to Casper fish and their progeny were used for larvae experiments. A detailed map of the Tg(hsp70i:cPla2-mKate2-P2A-eGFP-KDEL) plasmid and restriction digests results are provided in Extended Data Fig. For laser-wounding experiments, at 2 d.p.f., Tg(hsp70i:cPla2-mKate2-P2A-eGFP-KDEL) were heat shocked at 37 °C for 2 h to induce transgene expression. The fluorescent larvae were sorted for experiments at 12–18 h after heat treatment using a coaxial dissection stereomicroscope (MVX10, Olympus) equipped with a mercury lamp (LM200B1-A, Prior Scientific) and set dichroic mirrors (MVX-RFA; 540/35 U-MGFPHQ/XL, 625/55 U-MRFPHQ/XL). The selected larvae were anaesthetized in hypo-osmotic E3 medium56 containing 0.2 mg ml−1 3-amino benzoic acid ethyl ester (Sigma, MS-222, E10521) (pH 7.0), buffered in 0.5 mg ml−1 anhydrous sodium phosphate dibasic (Fisher, BP332-500). Using ~200 μl 1% low-melting (LM) agarose dissolved in E3, the anaesthetized 3 d.p.f. embryos were embedded on their right side in a 60 mm plastic Petri dish (Corning, 351007). The solidified LM agarose was covered with ~2–3 ml of standard E3 medium, or pre-incubated for 30 min in E3 supplemented with ethanol as vehicle (1% EtOH in E3) or latrunculin A (2.5 μM, Sigma-Aldrich, L5163) to create a submerging environment for the 25× objective lens (NA 1.1 ∞/0–0.17 WD 2 μm Water Dipping, Nikon) on an upright microscope. The samples were excited with 488 and 561 nm diode lasers (Andor Revolution XD). ER and cPla2–mKate2 fluorescence were acquired using a Andor iXon3 897 EMCCD camera mounted on a Nikon Eclipse FN1 microscope (25× objective lens; NA 1.1, ∞/0–0.17 WD 2 μm, Nikon) equipped with a Yokogawa CSU-X1 spinning-disc unit (pinhole of 50 μm) and NIS imaging software (NIS Elements, 3.22.14). Emission was acquired through band-pass filters for green (525/40, Semrock, FF02-525/40-25) or red (617/73, Semrock, FF02-617/73-25) fluorescence emission spectra of confocal stacks, collected at 1.5 µm z step size and repeated in no-delay intervals per position for up to 5 min. The wounds were induced at t ~ 1 min with several successive laser pulses targeted at the periphery of the epithelium using a microscope-mounted 435 nm micropoint laser (Andor). The experimental scheme for heat shock-induced expression is shown in Extended Data Fig. Tg(hsp70i:cPla2-mKate2-P2A-eGFP-KDEL) were heat shocked at 37 °C for 2 h approximately 24 h before the start of the imaging experiment. The heat-shocked larvae were anaesthetized in hypo-osmotic E3 medium containing tricaine analgesic. The larvae were prepared using the same protocol as for UV-laser wounding experiments except for latrunculin A pre-treatment. The anaesthetized live samples were embedded in 35-mm glass-bottom dishes (MatTek Corporation, P35G-1.5-14-C) before being transferred to a pre-heated Leica Stellaris 8 Dive two-photon microscope. The microscope is equipped with two tunable infrared lasers: Mai Tai HP (690–1,040 nm, continuous) and Insight X3 (680–1,300 nm, continuous), both capable of delivering up to 2.4 W average maximum power. Tissues were wounded using the FRAP ab1 XYZT module in galvanometric scanning mode with bidirectional scan direction, a scan speed of 1,000 Hz, and a pixel dwell time of 0.3508 μs. A HC FLUOTAR L VISIR 25×/0.95 WATER objective was used with 2.25× zoom and acquired at 15-s intervals. Images were collected at Nyquist resolution with a pixel size of 0.25 μm, yielding a FOV of 206.7 × 206.7 μm (840 × 840 pixels), and a z-stack depth of 10 μm with a step size of 2 μm per slice. For dual-colour excitation, the Mai Tai HP laser was tuned to 924 nm (MP1, 5.5% power) for eGFP excitation, and the Insight X3 laser was tuned to 1,118 nm (MP2, 2.6% power) for mKate2. Wounding was performed using the FRAP module with a sequence of four pre-bleach iterations, followed by six bleach iterations and 20 post-bleach iterations. For laser ablation, bleaching intensity was set to 100% on MP2 (1,118 nm) and 20% on MP1 (924 nm), with a total dwell time of 2 s at the mid plane of the tissue, targeting a circular ROI (radius 25 μm) for localized wounding. Emitted fluorescence was filtered using the Galvo Resonant Pan module (multifunction port RSP 1005) with SP815 and SP800 short-pass filters, in combination with a Notch FW 2 (OPO SP800) filter and a variable beam expander (expansion factor of 1.03, z-colour correction offset of 0). Detection was performed using external spectral non-descanned hybrid detectors (HyD NDD1 and NDD2) operating in photon-counting mode, with NDD1 configured for eGFP detection (480–527 nm) and NDD2 for mKate2 (620–732 nm). All imaging data were saved and processed using Leica LAS X software (Leica LAS X, 4.8.1.29271). The 4D image stacks of Tg(hsp70i: cPla2-mKate2-P2A-eGFP-KDEL) were corrected by 3D deconvolution, where sample PSF was derived and computed automatically with depth calibration and without image intensity subtraction or pre-processing in Nikon NIS elements (5.21.03, Build 1489). The deconvolution process was carried by the default method using the landweber algorithm for spinning-disc modality with a 50 µm pinhole size with an immersion refractive index 1.33 (water) for both 488 and 561 nm fluorescent light emission. The obtained files were loaded in Fiji (ImageJ, 1.53p) and the last frame was used for extraction of wound core and ER vesiculation area. Specifically, the polygon tool was used to trace parameter and set-scaled images for the dark area after 435 nm–UV harmonic laser wounding in tissue, and was designated as wound core in all biologically independent replicates. The obtained values were recorded per pixel of segmented polygon in Fiji (ImageJ, 1.53p) and converted to micrometres squared. The radius and correlation quantification were subsequently extracted, and box plots and Pearson tests with 95% linear regression were plotted in Python3 vector graphics. The dynamics of cPla2 and ALPIN binding and lamin B1 (Lmnb1) and ER structures were measured using a Nikon Eclipse Ti inverted spinning-disc confocal microscope with a Plan Apo 100×/1.45 oil objective, a Plan Apo 60×/1.4 oil objective or a Plan Apo 40×/0.95 air objective (cells were imaged with a 60× or 100× objective, while GUVs were imaged with a 40× objective) and equipped with either (1) a Yokogawa CSU-X1 spinning-disc unit, an ANDOR iXon ULTRA 897BV EMCCD camera, 405 nm, 488 nm and 561 nm solid-state laser lines (Andor Revolution XD, LCS-501A) and NIS Elements Software (4.13.04) or (2) a Yokogawa CSU-W1 spinning-disc unit, a Teledyne Photometrics Prime-BSI sCMOS camera and 405, 488 and 561 nm multidiode AOTF-controlled laser modules (Nikon Instruments, model LUN-F; F6A 250 V) with NIS-Elements software (version 5.21.03). mKate2 and 18:1 Liss Rhod PE emission was excited at 561 nm and collected through a 620 band-pass emission filter (590–650 nm, Chroma Technology., 49005, ET-DsRed; 88000v2 Quad Set). eGFP fluorescence was excited at 488 nm and collected through a 525 band-pass emission filter (515–555 nm, Chroma Technology, 49002, ET-GFP;88000v2 Quad Set). The Sytox blue fluorescence was excited at 405 nm and collected through a 425 band-pass emission filter (435–478 nm, Chroma Technology, 49000, ET-DAPI;88000v2 Quad Set). Identical microscope settings were used throughout imaging. Experiments involving GUVs and permeabilized cells took place at room temperature. Live imaging of cultured cells was performed at 37 °C. Nikon's Perfect Focus System was enabled throughout the imaging session. For short-term time lapse and permeabilized cell imaging experiments, z-stacks (1–1.7 μm steps over 30 μm) were acquired each minute for a maximum of 20 min. For extended time-lapse experiments, z-stacks (2 μm over 40 μm) were acquired every 2 min for total of 6 h (frames 181, step t interval 2 min) in Nikon NIS elements (5.21.03). For the ML162 treatment, the z-stacks (1 μm step over 40 μm) were acquired every 10 min. For single time point acquisition, single z-stacks (1–1.4 μm steps over 30 μm) were acquired immediately after the end of the incubation. For high-resolution imaging of the ER network, we used a structured-illumination microscope (27.5 µm G5). Images were acquired using Elyra 7 Zeiss Microscope with a 63× objective (Plan-Apochromat 63× NA 1.4 ꚙ/0.17 WD 0.19 mm oil, 420782-9900-000 Zeiss) oil immersion lens. Each emission for red and green were collected on two separate sCMOS camera (Excelitas technology, pco.edge 4.2 M) and aligned during pre-processing in Zeiss software (Zen Black 3.0 SR FP2, v16.0.20.306). A single z-stack (0.8 μm steps over 30 μm) was acquired at the end of the experiment. All images were taken in the identical microscope settings. All 3D segmentation and analysis were performed with the Anaconda distribution of Python (Python ≥3.11). Specifically, customized Python scripts were developed using the Numpy (v1.23.5)57, Scipy (v1.15.2)58, Scikit image (v0.20.0)59, Allen Cell Structure Segmenter (v0.5.0)60, trackpy (v0.6.4)61,62 and Napari (v0.5.5) libraries63. To quantify binding of ALPS1–2–eGFP to GUV membranes, a custom, semi-automatic Python script was developed. First, a line was manually drawn by the user from the centre of GUV to the edge to crop a selected GUV from the whole image stack. Next, intensity normalization and a 3D Gaussian blur (3-pixel radius) were applied to the cropped GUV stack to remove background noise. A marker-controlled watershed algorithm was used on each slice of the cropped stack to segment the GUV. The slice with largest segmented GUV area was defined as the middle section. To quantitatively measure the affinity of ALPS1–2–eGFP to GUV, the apparent dissociation constant \({K}_{{\rm{d}}}^{{\prime} }\), maximum binding intensity Bmax and the apparent Hill coefficient H were determined by nonlinear least-square analysis of bound-protein fluorescence (Bbound) versus the total domain concentration ([Domain]) using the Langmuir adsorption isotherm with Hill expansion (MATLAB Curve Fitting Toolbox v25.1 R2025a) A similar, fully autonomous Python pipeline was developed to segment the nucleus in 3D and quantify both the volume of nucleus and bindings of cPla2–mKate2 and ALPIN sensor. If cells expressed eGFP–Lmnb1, this marker was used for nuclear segmentation. Otherwise, cPla2–mKate2 or the ALPIN sensor was used for segmentation. First, intensity normalization, 3D Gaussian blur (1-pixel radius) and rolling-ball algorithms were applied to the images to remove uneven background noise. Second, a masked-object thresholding (this algorithm was employed to separate foreground objects from the background of the 3D image stack through an automatically determined threshold and to generate a marker image that distinguishes individual nucleus) followed by marker-controlled watershed algorithms were used to segment the NE within the 3D image stack. Since the movement of cell nuclei was limited (less than 50 µm) during the duration of the experiment, a standard particle-tracking algorithm was utilized to track individual nucleus over time. Nuclear volume was calculated by the summation of all voxels contained within the segmented binary mask. Ratiometric analysis was used to quantify cPla2–mKate2 and ALPIN nuclear membrane-binding dynamics. For each nucleus, the slice with largest segmented nuclear area was selected as the representative, middle section for each nucleus. Based on the watershed segmentation result of the middle section, a 3-pixel-wide contour (‘membrane contour') was drawn onto the NE and another 3-pixel-wide contour (‘background contour') was drawn to mark the centre of the nucleus (nucleoplasm). One caveat for this type of imaging analysis is that an increase of binding ratio could be due to a decrease of intensity in nucleoplasm instead of an increase in protein binding to the NE. A detailed scheme of nuclear membrane binding and volume analysis is shown in Extended Data Fig. To quantify individual NE shrivelling/invaginations, a filament-based filter algorithm was performed on the previously selected middle section from the eGFP–Lmnb1 image stack to create a binary mask that preserved the detailed structures of nuclear membranes. Note, to preserve nuclear membrane details and minimize processing-induced artefacts, only Gaussian blurring was applied to the eGFP–Lmnb1 stack before applying the filament-based filter algorithm. This was followed by a skeletonization algorithm to produce a precise 1-pixel outline of NE. Shrivelling often created branch points in the skeleton images. Therefore, the degree of nuclear invagination was approximated as the number of branch points in the skeleton outlines, which can be detected if a particular skeleton pixel contained more than three neighbours. Finally, a condensation algorithm was utilized to merge multiple nearby branch points that appeared owing to noise affecting accuracy of skeleton outlines. A scheme of the NE invagination analysis is shown in Extended Data Fig. To quantify ER structure, a Python-based automatic image analysis pipeline was developed. ER images or movies were first pre-processed with intensity normalization and 3D Gaussian blur (1-pixel radius). With luminal labelling (for example, eGFP–KDEL), ER was segmented from the 3D stack by a marker-controlled watershed algorithm the same way as NE segmentation. If ER membranes were labelled (for example, SEC61B), the image was processed by a filament filter algorithm that segments dense filamentous and network-like structures in the cell. The algorithm's thickness parameter was set to 1.6, corresponding to ER networks or vesicles approximately 3–4 pixels thick. For 3D ER metrics, calculations were technically and optically challenging. The slice with largest segmented area along the z axis was selected as the representative mid-section for downstream quantifications to compute surface area and circularity. The surface area of each segmented ER membrane in the mid-section was extracted from the corresponding segmentation mask and the circularity was calculated as \(\mathrm{Circularity}=4\uppi \,\times \frac{\mathrm{Surface}\,\mathrm{area}}{{\mathrm{Perimeter}}^{2}}\) (0 for a highly non-circular object and 1 for a perfect circle). For each time point, ER surface area and circularity were measured for all individually segmented ER membranes within a single FOV. To enable the quantitative comparison across samples and time, these measurements were summarized by recording the largest individual ER surface area and the mean circularity of all ER membranes in that FOV. This approach produced one surface area value and one circularity value per FOV at each time point, which were then used for normalization, plotting and statistical analysis. As ER membranes from neighbouring cells were often in close proximity and plasma membrane labelling was absent, segmentation at the single-cell level was not feasible; therefore, ER metrics were quantified per FOV. A detailed scheme of quantifying ER circularity and area are provided in Extended Data Fig. To ensure the comparability of results obtained across different experimental batches and days, multiple normalization methods were applied. For GUV experiments, raw rim binding of each domain was normalized by the fluorescence signal of a 1 μM protein solution from the same purification batch to reduce variabilities in fluorescence from different purification batches. For time-lapse experiments, numerical values for the fluorescent marker nuclear rim-to-centre ratio or rim-to-nuclear Lmnb1 (for example, cPla2, ALPIN and ALPIC), ER surface area, circularity, nuclear volume and nuclear folds at each subsequent time point were normalized to the corresponding value at t = 0 min. For FLIP experiments, the intensity of fluorescent ER markers at each subsequent time point was similarly normalized to the t = 0 min value. Notably, although ER circularity values inherently range from 0 to 1, normalization to the mean circularity at t = 0 min shifted the range to 1–2, such that a value of 1 represents the baseline circularity and values greater than 1 indicate an increase relative to baseline. For single time point osmotic shock experiments (Figs. 1a), cPla2 and ALPIN1 nuclear rim-to-centre ratio, ER area and ER circularity under hypo-osmotic condition values were normalized to the corresponding measurements obtained under isosmotic condition. For single time point permeabilization experiments (Extended Data Fig. 2g,h), values for ER circularity, ER area and ALPIN1 nuclear rim-to-centre ratios were normalized to measurements obtained under 0 μM Ca2+ and osmotic-balanced (2.5% PVP) conditions. As different normalization schemes were applied to time-lapse and single time point datasets, the resulting value ranges differed and they were not directly comparable. The results presented in this study were performed as multiple, biologically independent experiments, as indicated in figure legends, and no inconsistent results were observed. For experiments involving cell cultures, biological replicates were obtained using cells derived from distinct frozen vials and/or cultured at different times. For experiments involving GUVs, biological replicates were obtained using GUVs synthesized from independently prepared lipid mixtures. In animal experiments, each larva was considered to be an independent biological replicate, and at least six animals were used per imaging modality (spinning-disc confocal and two-photon microscopy). For the fitted coefficients of ARFGAP1 ALPS1–2–eGFP GUV equilibrium binding, the minimum and maximum values denote the 95% confidence interval (CI), while the error bars for individual GUV binding measurements indicate s.d. For single time point comparisons, error bars represent the s.d. The statistical analysis was performed in Python (v3.11) using Scipy (v1.15.2) and Pingouin (v0.5.5) packages58. For comparing statistical significance between two groups, a parametric method (an unpaired two-tailed Student's t-test if two groups have equal variance, otherwise an unpaired two-tailed Welch's t-test) was used if the data passed the normality test. A non-parametric method (two-tailed Mann–Whitney U test) was used for non-normally distributed data. For the comparison of more than two groups, a one-way Welch's ANOVA (data did not pass Levene's test for equal variance) followed by a post hoc Games–Howell test were used if the data followed normal distribution. Otherwise, statistical significance was calculated with the Kruskal–Wallis test followed by a post hoc Dunn's test for multiple comparison. ALPIN binding rates in permeabilized nuclei were estimated via ordinary least squares (OLS) on the linear part of the binding curves from Python's Statsmodel package64. Specifically, OLS regression models were fitted in Python 3.11 (statsmodels v0.14.4). Since there are only two time points (t = 4 min and t = 5 min), assumptions for linearity and multicollinearity were not applicable. The Breusch–Pagan test was used to assess heteroscedasticity; the Shapiro–Wilk test and Q–Qplots of standardized residuals were used to evaluate normality of residuals. If either test returned a P < 0.05, the model was re-estimated with heteroscedasticity-consistent (HC3) robust standard errors; otherwise, classical OLS standard errors were retained. The Durbin–Watson statistic was used to evaluate autocorrelations between variables. Estimated coefficients, s.e.m., 95% CIs, coefficient of determination (R² and adjusted R²) and P value are reported in Source data. Graphical plots were generated using Python 3.11 Matplotlib (v3.10.3)65 and seaborn (v0.13.2) libraries66. The helical-wheel representation of ALPS1 motif structure was created using the Python 3.11 Matplotlib library. The equilibrium GUV fluorescence-binding graph was generated using Curve Fitting Box (v25.1) from MATLAB R2025a. All plots and statistical analyses were generated using the standard packages listed above (version listed), with no additional custom plotting code. Statistical analyses were performed across biological replicates. Replication included both variation in biological source material and/or independent imaging sessions conducted on separate experimental days. Analysis was performed objectively without blinding. Time-lapse U2OS cell experiments were performed with 2–5 biological replicates, with key experiments repeated ≥3 independent replicates to ensure reproducibility. Single time point U2OS cell assays were conducted in one independent experiment using cells from two independent frozen vials. Equilibrium GUV experiments were conducted from two independently prepared lipid mixtures. For zebrafish imaging, each experiment represents a biologically independent replicate using animals derived from a distinct clutch of eggs. Experiments were repeated across multiple days, with one larva imaged per experiment for each UV- or infrared-induced tissue-wounding condition and repeated on more than six different live larvae to ensure reproducibility. Data exclusion criteria were predefined. Imaging datasets were excluded if they were affected by artificial fluorescence fluctuations (for example, immersion oil bubbles) or incompletely acquired z-stacks, and broken tracks were either cropped or completely discarded. All raw imaging files of cell experiments were loaded into custom written automatic programs for analysis, unless otherwise stated. Nuclei were removed from the analysis when segmented objects moved more than 130 pixels (~20 µm) between successive frames or broken tracks appeared due to leakage of nucleoplasmic fluorescent markers (cPla2–mKate2 and ALPIN) when these markers were used for nuclear segmentation. For the ER analysis program, segmented ER vesicles size less than 30–50 pixel2 (~1 µm2) were precluded from the analysis. Since the ER was analysed per FOV, no tracks were involved, and all time points were included in the analysis unless ER segmentation was precluded by the leakage of eGFP–KDEL. In long-term cell imaging experiments, datasets were excluded only when imaging artefacts prevented reliable analysis and data extraction. Specifically, exclusions were made if (1) fluorescent markers leaked from their designated organelle localization into the surrounding cytoplasmic space during acquisition, (2) cells became clumped or (3) automated tracking produced broken or fragmented tracks. For GUV experiments, the raw imaging files were loaded into a semi-automatic custom written analysis program where the individual GUV was randomly selected by the user for analysis. For zebrafish intravital imaging, datasets were excluded if tailfin tissue drift exceeded 100 pixels (~50 µm) in the xy plane or 20 pixels (~10 µm) in the z direction following UV-laser damage. Typically, experiments were repeated at least on two separate days (using cells thawed from different vials). Only the experiments shown in Figs. 1a and 2a,g,h were not replicated on different days; nevertheless they were conducted using cells or GUVs prepared from at least two independent stock sources, that is, storage vials or lipid mixtures, respectively. The details of statistical analyses used, exact P values, number of biological independent experiments and sample sizes for all graphs are listed in the figures or figure legends. Specimens (U2OS cells, GUVs and zebrafish larvae) were randomly chosen for imaging without bias or pre-selection. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All other data supporting the findings of this study are available from corresponding author upon reasonable request. Specifically, the time-lapse and super-resolution imaging data can be made available only upon request due to its proprietary file formats, very large file sizes (>10 TB) and repository storage limitation. The plasmids, cell lines and zebrafish strains created in this study are also available upon reasonable request. Source data are provided with this paper. The code for ER FLIP analysis and viewing 3D super-resolution segmentation masks is available as an open-source download via GitHub at https://github.com/zazadovv/ER_Nucleus.git. The code for analysing GUV protein binding and adsorption fitting is publicly available via GitHub at https://github.com/joeshen123/GUV-Protein-Binding-Analysis-Program.git. The code for analysing protein binding to the nuclear envelop, nuclear volume and invaginations is available via GitHub at https://github.com/joeshen123/Nuclear-Membrane-Binding-Analysis.git. The code for analysing ER areas and curvatures is available via GitHub at https://github.com/joeshen123/ER-Structure-Analysis.git. Niethammer, P. Components and mechanisms of nuclear mechanotransduction. & Gualde, N. Arachidonic-acid-derived eicosanoids: roles in biology and immunopathology. & Norris, P. C. Eicosanoid storm in infection and inflammation. Enyedi, B., Jelcic, M. & Niethammer, P. The cell nucleus serves as a mechanotransducer of tissue damage-induced inflammation. The nucleus measures shape changes for cellular proprioception to control dynamic cell behavior. The nucleus acts as a ruler tailoring cell responses to spatial constraints. Cell shape sensing licenses dendritic cells for homeostatic migration to lymph nodes. Dantas, M., Oliveira, A., Aguiar, P., Maiato, H. & Ferreira, J. G. Nuclear tension controls mitotic entry by regulating cyclin B1 nuclear translocation. Shen, Z. et al. A synergy between mechanosensitive calcium- and membrane-binding mediates tension-sensing by C2-like domains. Mesmin, B. et al. Two lipid-packing sensor motifs contribute to the sensitivity of ArfGAP1 to membrane curvature. Feedback between membrane tension, lipid shape and curvature in the formation of packing defects. & de Haas, G. H. Regulation of phospholipase A2 activity by the lipid–water interface: a monolayer approach. & Kinnunen, P. K. J. Biophysical properties of lipids and dynamic membranes. Vanni, S. et al. Amphipathic lipid packing sensor motifs: probing bilayer defects with hydrophobic residues. Campelo, F. & Kozlov, M. M. Sensing membrane stresses by protein insertions. Organization of the sea urchin egg endoplasmic reticulum and its reorganization at fertilization. & Terasaki, M. Structural changes of the endoplasmic reticulum of sea urchin eggs during fertilization. & Terasaki, M. Structural changes in the endoplasmic reticulum of starfish oocytes during meiotic maturation and fertilization. Subramanian, K. & Meyer, T. Calcium-induced restructuring of nuclear envelope and endoplasmic reticulum calcium stores. Kucharz, K., Krogh, M., Ng, A. N. & Toresson, H. NMDA receptor stimulation induces reversible fission of the neuronal endoplasmic reticulum. Kucharz, K., Wieloch, T. & Toresson, H. Rapid fragmentation of the endoplasmic reticulum in cortical neurons of the mouse brain in situ following cardiac arrest. & Lippincott-Schwartz, J. ER membranes exhibit phase behavior at sites of organelle contact. Hoffmann, E. K., Lambert, I. H. & Pedersen, S. F. Physiology of cell volume regulation in vertebrates. Lemière, J., Real-Calderon, P., Holt, L. J., Fai, T. G. & Chang, F. Control of nuclear size by osmotic forces in Schizosaccharomyces pombe. Mitchison, T. J. Colloid osmotic parameterization and measurement of subcellular crowding. Venkova, L. et al. A mechano-osmotic feedback couples cell volume to the rate of cell deformation. & Antonny, B. ArfGAP1 responds to membrane curvature through the folding of a lipid packing sensor motif. The nucleolus functions as a phase-separated protein quality control compartment. Mechanism of membrane curvature sensing by amphipathic helix containing proteins. Zatsepina, O. V. et al. Reassembly of functional nucleoli following in situ unraveling by low-ionic-strength treatment of cultured mammalian cells. Experimental induction of prenucleolar bodies (PNBs) in interphase cells: interphase PNBs show similar characteristics as those typically observed at telophase of mitosis in untreated cells. A. et al. Lipid flow through fusion pores connecting membranes of different tensions. Riegman, M. et al. Ferroptosis occurs through an osmotic mechanism and propagates independently of cell rupture. Liang, X. et al. Mechanosensitive fluorescence lifetime probes for investigating the dynamic mechanism of ferroptosis. Hirata, Y. et al. Lipid peroxidation increases membrane tension, Piezo1 gating, and cation permeability to execute ferroptosis. Shen, Z., Lengyel, M. & Niethammer, P. The yellow brick road to nuclear membrane mechanotransduction. A. Osmolarity-independent electrical cues guide rapid response to injury in zebrafish epidermis. Enyedi, B., Kala, S., Nikolich-Zugich, T. & Niethammer, P. Tissue damage detection by osmotic surveillance. Gelashvili, Z., Shen, Z., Ma, Y., Jelcic, M. & Niethammer, P. Perivascular macrophages convert physical wound signals into rapid vascular responses. Basavappa, S., Pedersen, S. F., Jørgensen, N. K., Ellory, J. C. & Hoffmann, E. K. Swelling-induced arachidonic acid release via the 85-kDa cPLA2 in human neuroblastoma cells. Thoroed, S. M., Lauritzen, L., Lambert, I. H., Hansen, H. S. & Hoffmann, E. K. Cell swelling activates phospholipase A2 in Ehrlich ascites tumor cells. Katikaneni, A. et al. Lipid peroxidation regulates long-range wound detection through 5-lipoxygenase in zebrafish. & Niethammer, P. Oxoeicosanoid signaling mediates early antimicrobial defense in zebrafish. Ma, Y. et al. DHRS7 integrates NADP+/NADPH redox sensing with inflammatory lipid signalling via the oxoeicosanoid pathway. The G-protein coupled receptor OXER1 is a tissue redox sensor essential for intestinal epithelial barrier integrity. & Jendrach, M. Functional interaction of the cation channel transient receptor potential vanilloid 4 (TRPV4) and actin in volume regulation. Cooperation of the ER-shaping proteins atlastin, lunapark, and reticulons to generate a tubular membrane network. Dynamic constriction and fission of endoplasmic reticulum membranes by reticulon. Schmid, E. M., Richmond, D. L. & Fletcher, D. A. Reconstitution of proteins on electroformed giant unilamellar vesicles. Schoenmakers, T. J., Visser, G. J., Flik, G. & Theuvenet, A. P. CHELATOR: an improved method for computing metal ion concentrations in physiological solutions. Nüsslein-Volhard, C. & Dahm, R. (eds) Zebrafish: A Practical Approach (Oxford Univ. van der Walt, S., Colbert, S. C. & Varoquaux, G. The NumPy array: a structure for efficient numerical computation. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. van der Walt, S. et al. scikit-image: image processing in Python. The Allen Cell Structure Segmenter: a new open source toolkit for segmenting 3D intracellular structures in fluorescence microscopy images. Crocker, J. C. & Grier, D. G. Methods of digital video microscopy for colloidal studies. Seabold, S. & Perktold, J. Statsmodels: econometric and statistical modeling with Python. 9th Python in Science Conference (SciPy, 2010); https://doi.org/10.25080/Majora-92bf1922-011 This research was supported by the NIH/NIGMS grant R35GM140883 to P.N., and in part by the NIH/NCI Cancer Center Support grant P30CA008748. was supported by a Bruce Charles Forbes Graduate Fellowship. was supported by Harold E. Varmus Graduate Fellowship. We thank M. Lengyel, Y. Ma and S.G. Rajan for editing and insightful comments on the draft. Present address: Bloomberg-Kimmel Institute for Cancer Immunotherapy, Department of Oncology, Johns Hopkins University School of Medicine, Baltimore, MD, USA These authors contributed equally: Zhouyang Shen, Zaza Gelashvili. Cell Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA Zhouyang Shen, Zaza Gelashvili & Philipp Niethammer Gerstner Sloan Kettering Graduate School of Biomedical Sciences, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors declare no competing interests. Nature Cell Biology thanks Dennis Discher, Ning Wang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Left panel, ER network area of U2OS cells at ΔΠ = 0 mOsm or 303 mOsm, respectively. Same FOVs were analyzed for normalized area and circularity. Right panel, cPla2-mKate2-INM adsorption in cells in the above FOVs at ΔΠ = 0 mOsm or ΔΠ = 303 mOsm, respectively. n, number of nuclei analyzed. ****, p = 2.8 ×10−7 for both the left and the middle graph and p = 7.8 ×10−16 for the right graph as determined by a two-sided Mann Whitney U-test (data are not normally distributed) for both the left and middle graph and a two-sided Welch's t-test for the right graph. Note: Both Extended Data Fig. 1a and the right panel of Fig. 1a are different representations of the same data set. The data represent one independent experiment performed on cells from two distinct biological sources (separate frozen vials). (b) Top panel, representative time-lapse montage of cPla2-INM adsorption, ER morphology (eGFP-KDEL), and cell lysis (Sytox blue) upon exposure to the hypotonic solution (ΔΠ = 270 mOsm). Middle panel, nuclear rim profiles (20 pixels wide) of cPla2 midplane fluorescence at indicated location (inset, yellow line). Temporal order is indicated by line width and color code. Bottom panel, temporal profile of segmented nuclear area (open circles), cytoplasmic cPla2 (ROI on extranuclear area, red circles), and nuclear Sytox blue signal (ROI on intranuclear area, blue circles) as determined from confocal MIPs. Scale bars, 10 µm (c) Top panel, time-lapse analysis of nuclear volume (black) and normalized cPla2-INM adsorption (red). Light blue shade, osmotic shock and RVD phase. Dark blue shade, pre-lytic phase. n, number of analyzed nuclei. Bottom panel, corresponding time-lapse analysis of ER circularity (black). The data represent three independent experiments. (d) Left panel, time-lapse analysis of nuclear envelope invaginations. Right panel, representative confocal MIPs of eGFP-Lmnb1 and Sytox blue channel. The data represent two independent experiments. (e) The top panel, super resolution lattice-SIM2-Apotome representative MIP image of U2OS cells for the ER lumen network at baseline (ΔΠ = 0 mOsm, n = 6) and 10 min after hypoosmotic shock (ΔΠ = 270 mOsm, n = 6). Bottom panel, the super resolution (XYZ, 60, 60, 150 nm) segmentation masks of eGFP-SEC61B at baseline (left, n = 4) and 10 min after hypoosmotic shock (right, n = 4). For time-lapse movies, values in subsequent time points were normalized to baseline at T = 0. For single time point images, values from hypoosmotic conditions were normalized to isosmotic controls. (a) Equilibrium GUV binding isotherms of ARFGAP1 ALPS 1–2-eGFP at hypotonic solution (ΔΠ = 240 mOsm) containing 0 µM (black lines) or 20 µM (red lines) Ca2+. Error bar, SD of normalized single GUV binding measurements per domain concentration. Binding isotherms were derived by titrating ALPS 1-2-eGFP (0–500 nM) on GUVs under hypotonic conditions (ΔΠ = 240 mOsm) in the presence of either 0 or 20 µM Ca²⁺. Fitted lines, Langmuir equations with Hill expansion fit (methods, GUV equilibrium binding experiments). Legend, apparent dissociation constants (Kd') and associated 95% confidence interval determined from the above Langmuir/Hill equations fitting. The data represent one independent experiment using GUVs from two distinct lipid mixtures. (b) Left panel, representative confocal MIPs depicting nucleolar colocalization of ALPIN1 and nucleolus marker (Nucleolar-ID) acquired before and 15 min after ΔΠ = 270 mOsm shock. Right panel, profile plot quantification (see inset). (c) Top panel, representative timelapse montage of ALPIN1–INM adsorption, ER morphology (eGFP-SEC61B), and cell lysis (Sytox blue) upon exposure to the hypotonic solution (ΔΠ = 270 mOsm). Top right panel, nuclear rim profiles (20 pixel wide) of ALPIN1 midplane fluorescence at indicated location (inset, yellow line). Temporal order is indicated by line width and color code. Bottom right panel, temporal profile of segmented nuclear area (open circles), cytoplasmic ALPIN1 (ROI on extranuclear area, red circles), and nuclear Sytox blue signal (ROI on intranuclear area, blue circles) as determined from confocal MIPs. (d) Timelapse comparison of ALPIN1- (black) and ALPIN2- (green) INM adsorption upon ΔΠ = 270 mOsm shock as normalized by Lmnb1 fluorescence across the nuclear midsection. Shaded error, SEM, n, number of nuclei analyzed. 2b and this figure present different type of analysis performed on the same ALPIN2 movie. (e) Timelapse of ER circularity (blue), nuclear volume (black), and ALPIN1 adsorption (red). The data represent two independent experiments. (f) Inset, representative confocal midplane-slices of ALPIC2 (ALPS1-2-mKate2)-ONM adsorption acquired before and after (t = 5 min) ΔΠ = 270 mOsm shock. (g) Quantification of ER area (left panel) and circularity (right panel) after digitonin-permeabilization as a function of [Ca2+] and [PVP360]. n, number of FOVs analyzed (containing ~1-10 cells each). The data represent one independent experiment performed with cells from two distinct biological sources. (h) Normalized ALPIN1-INM adsorption 30 min after permeabilization with 0 or 50 µM Ca²⁺. ****, two-sided Welch's t-test (unequal variance): p = 3.9 ×10−28. The data represent one independent experiment performed with cells from two distinct biological sources. For time courses, data normalized to T = 0; GUV data normalized to 1 µM ALPS1-2-eGFP control. For single timepoint permeabilization experiments, values were normalized to measurements obtained under 0 µM Ca2+ and osmotic-balanced (2.5% PVP) conditions. (a) Left panel, representative timelapse montage of cPla2-INM adsorption, ER morphology (eGFP-KDEL) and cell lysis (Sytox blue) in a cell undergoing ferroptotic cell death upon exposure to the GPX4 inhibitor ML162. Top right panel, nuclear rim profiles (20 pixel wide) of cPla2 midplane fluorescence at indicated location (inset, yellow line). Temporal order is indicated by line width and color code. Bottom right panel, temporal profile of segmented nuclear area (open circles), cytoplasmic cPla2 (ROI on extranuclear area, red circles), and nuclear Sytox blue signal (ROI on intranuclear area, blue circles) as determined from confocal MIPs. (b) Left panel, representative timelapse montage of eGFP-Lmnb1 and cell lysis (Sytox blue) in ferroptotic cell death with GPX4 inhibitor ML162. Right panel, time-lapse analysis of nuclear envelope invaginations. The data represent two independent experiments. (c) Two-photon MIPs of cPla2-mKate2 and luminal ER marker eGFP-KDEL in live zebrafish larvae at baseline (before wounding) and after wounding under hypoosmotic conditions – larvae are wounded (Π ~ 10 mOsm) at t = 00:00:25. Middle panels, magnification of eGFP-KDEL or cPla2 in individual cells within (position 1) and just outside (position 2) the wound region. Constitutive cPla2 translocation with ER fragmentation was observed in every (n = 10) wounding experiments. Yellow, hydrophobic; purple, serine and threonine; grey, glycine and alanine; blue, basic residues; red, acidic residues. (b) Circular plasmid maps of PSB/CMV/MCS constructs used to express ALPIN 1 or ALPIN 2 in U2OS cells together with nuclear (Lmnb1, left) or ER (SEC61B, right) markers (both protein sequences are linked via P2A sequence). To generate the constructs containing both ALPIN and nuclear marker, ARFGAP1 ALPS1 and ALPS 1-2 were inserted into the vector containing both eGFP-Lmnb1 and NLS tagged mKate2 sequences via NheI and BshT1 restriction sites. To generate the constructs containing both ALPIN and ER markers, the Lmnb1 sequence from the previously generated ALPIN-P2A-eGFP-Lmnb1 (left) was replaced by SEC61B via BgIII and XbaI restriction cut sites. The expression of the Puromycin resistant genes was under control of a SV40 promoter. Two inverted internal repeats sequences (IR-DR) flank the expression cassettes and allow the integration into host genomes when co-transfected with Sleeping Beauty transposase. (a) Left panel, circular DNA map of zebrafish transgenesis plasmid, pDest cryybb1:mKate2-P5E-HSP70i_pME-Ctd-mK2-cPla2_P3E-P2A-eGFP-KDEL overlaid with predicted restriction enzyme, BamHI digest cut sites as indicated by black arrow heads and black lines. (c) Scheme for laser wounding, Zebrafish Casper embryos are reared and selected for fluorescent selection marker, such as red eye. The embryos were heat-shocked in water bath at 37 °C for 2 h and subsequently returned to 28 °C. The following day, larvae are selected for pharmacological pre-treatment with Latrunculin A for 20-30 min and then wounded during confocal imaging with UV laser. (a) Overview of GUV membrane and ALPS1-2-eGFP adsorption analysis. Image stacks of GUV membrane (561 nm) and ALPS 1-2-eGFP (488 nm) were acquired, followed by manual selection of a radial line from the GUV centre to the membrane edge for automatic cropping (Step 2). Afterwards, a watershed segmentation algorithm was applied to the 3D cropped stack (Step 3), and the slice with the largest segmented area along z-axis (Midplane slice, Step 3 middle image with red edge) was selected to generate a membrane contour (white outline in Step 4) for quantifying GUV radius and ALPS 1-2-eGFP binding intensity. Quantified outputs were plotted and exported for downstream analysis (Step 5). (b) Overview of nuclear membrane volume and ALPIN/cPla2 binding analysis. Here, a eGFP-Lmnb1 and ALPIN2 labelled U2OS cell undergoing 270 mOsm hypoosmotic shock was chosen as an example. Raw 3D image stack was processed by gaussian smoothing and intensity thresholding (Step 2), followed by masked-object thresholding and segmentation of individual nuclei using a marker-controlled watershed algorithm (Step 3). The mid slice with largest segmented nuclear area was chosen to generate membrane contour for downstream analysis. Two strategies were used to quantify protein binding to the nuclear membrane (Step 4): (1). a rim:centre ratio comparing binding intensity at the membrane contour with that at the inner outline marking the nucleoplasm (Step 4 left panel); and (2) a rim: Lmnb1 ratio comparing binding intensity at the contour (561 nm channel, Step 4 right panel top image) with Lmnb1 intensity across segmented nuclear area at 488 nm channel (Step 4 right panel bottom image). Nuclear volume was calculated by summing all voxels within the segmented stack (Step 5). Binding ratio and nuclear volume were plotted and exported for downstream analysis (Step 6). (a) Overview of the Lamin branchpoint analysis to quantitatively measure nuclear envelope invaginations across hypoosmotic shock and pre-lytic phase of ferroptotic cell death. 270 mOsm hypoosmotic stress on eGFP-Lmnb1 and ALPIN2 labelled U2OS cells was used here as a demonstration example. Specifically, gaussian smooth followed by a custom-built filament-based filter algorithm (Step 2) was applied to the previously selected nuclear middle slice (Extended Data Fig. The resulting segmentation mask was skeletonized to a 1-pixel thick outline (Step 3). Branch points (pixels with >3 neighbors) were calculated to estimate nuclear envelope invagination (Step 4, branchpoints are indicated by white arrows and red circles). (b) Overview of the ER morphology analysis to quantify the change of ER area and circularity across osmotic and ferroptotic stress. 270 mOsm hypoosmotic stress on eGFP-KDEL and cPla2-mKate2 labelled U2OS cells was presented here as a demonstration example. ER image stack (488 nm channel) first underwent intensity normalization and 3D-gaussian blur (Step 2), followed by thresholded segmentation (Step 3 and 4) via a marker-controlled watershed algorithm (If the ER is tagged with SEC61B, a custom-built filament thresholding algorithm was used in place of marker-controlled watershed algorithm). Since ER network was morphologically complex and overlapped across neighboring cells, it was technologically infeasible to quantify ER morphology for individual cell. Confocal midplane slice of U2OS cells expressing cPla2-mKate2 and eGFP-KDEL under hypo-osmotic treatment (∆Π = 270 mOsm). cPla2-mKate2 NM translocation correlates with vesiculation of ER network upon application of hypo-osmotic shock. Super-resolution lattice SIM²-Apotome imaging of U2OS cells expressing membrane-localized eGFP-SEC61B under baseline conditions (ΔΠ = 0 mOsm). The video shows a 3D reconstruction of xy planes (pixel resolution: 60 × 60 nm) and a Z-stack acquired at 0.110 μm steps, comprising over 50 slices. Super-resolution lattice SIM²-Apotome imaging of U2OS cells expressing membrane-localized eGFP-SEC61B at 10 min post-hypo-osmotic shock (ΔΠ = 270 mOsm). The video shows a 3D reconstruction of xy planes (pixel resolution: 60 × 60 nm) and a Z-stack acquired at 0.150 μm steps, comprising over 50 slices. Confocal midplane slice of U2OS cells expressing ALPIN1 and eGFP-SEC61B under hypo-osmotic treatment (∆Π = 270 mOsm). ALPIN NM translocation correlates with vesiculation of ER network upon application of hypo-osmotic shock. Confocal midplane slice of U2OS cells expressing ALPIN2 and eGFP-SEC61B after hypo-osmotic treatment (∆Π = 270 mOsm). Confocal maximum intensity projection of cPla2-mKate2 (red) and luminal ER marker eGFP-KDEL (green) in latrunculin A-pretreated zebrafish larvae after wounding under hypo-osmotic conditions (Π ~10 mOsm) at t = 00:01:15. Middle panels: insets show cells within (rectangle 1) and just outside (rectangle 2) the wound region. Table 4: Cell culture hypotonic dilution buffer and light microscopy reporting. Tables 5–9: Brief technical description of microscopy used in manuscript. 1. cPla2–INM adsorption correlates with ER–NM disruption during osmotic stress. ALPIN reveals TINM buffering by the ER. Ferroptosis and tissue injury disrupt TINM buffering by the ER. ALPIN reveals TINM buffering by the ER. ER, cPla2 and Lmnb1 dynamics in hypo-osmotic shock. Characterization of ALPIN1/2 in nuclear membrane tension sensing. Collapse of ER network integrity in ferroptotic cell death and tissue wounding. Unprocessed DNA gel showing restriction enzyme digest fragment patterns supporting Extended Data Fig. Scheme of analysis for ER and lamin morphology. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. & Niethammer, P. Endoplasmic reticulum disruption stimulates nuclear membrane mechanotransduction. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02411-y'>A biobank-scale test of marginal epistasis reveals genome-wide signals of polygenic interaction effects</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-09 10:06:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The contribution of genetic interactions (epistasis) to human complex trait variation remains poorly understood due, in part, to the statistical and computational challenges involved in testing for interaction effects. Here we introduce FAME (FAst Marginal Epistasis test), a method that can test for marginal epistasis of a single-nucleotide polymorphism (SNP) on a quantitative trait (whether the effect of an SNP on the trait is modulated by genetic background). FAME is computationally efficient, enabling tests of marginal epistasis on biobank-scale data. Applying FAME to genome-wide association study (GWAS)-significant trait-SNP associations across 53 quantitative traits and ≈300 000 unrelated White British individuals in the UK Biobank (UKBB), we identified 16 significant marginal epistasis signals across 12 traits (\(P < \frac{5\times {10}^{-8}}{53}\)). Leveraging the scalability of FAME, we further localized marginal epistasis signals across chromosomes and estimated the proportion of variance explained by marginal epistasis effects. Our study provides evidence for interactions between individual genetic variants and polygenic background influencing complex traits. Understanding the role of epistasis is important for elucidating the genetic architecture of complex traits1. Epistasis could potentially explain the missing heritability underlying complex traits2 (although some studies suggest a limited contribution from genetic interactions3,4), why genetic effects differ across ancestral populations5 and the lack of transferability of genetic predictors both within6 and across ancestries7,8. Epistasis has also been hypothesized to play a role in variable expressivity of complex traits9. Nevertheless, our understanding of the role of epistasis in human traits remains limited10. One class of methods to detect epistasis explicitly searches for pairs of single-nucleotide polymorphisms (SNPs) that have a non-linear effect on a trait. Although allowing for an unbiased search for epistasis, exhaustively testing all pairs is computationally difficult and requires a stringent significance threshold to control the false positive rate. Efforts to overcome these difficulties have involved the use of statistical11,12,13 and algorithmic techniques14,15, hardware infrastructure16,17,18 or a biologically informed reduction in the space of SNP pairs tested19,20,21,22,23. An alternate approach to detect epistasis aims to test for the aggregate epistatic effect across SNPs, often using a variance component framework24,25,26. In this framework, models of marginal epistasis25,27 aim to test if the effect of an SNP on a trait is modulated by an individual's polygenic background. Such tests can improve power on account of the reduced multiple testing burden and the aggregation of weak epistatic signals. Even with the potential improvements in power, detecting robust signals of marginal epistasis likely requires application to large sample datasets1,4 that are now becoming available28,29. However, estimating marginal epistasis from biobank-scale data sets is computationally challenging. To address this challenge, we propose FAst Marginal Epistasis test (FAME), an algorithm that can jointly estimate additive and marginal epistasis variance components that is efficient. The efficiency of FAME enables its application to estimate marginal epistasis at a target SNP paired with genome-wide SNPs across a large number of individuals. We performed extensive simulations to show that FAME provides calibrated tests of marginal epistasis, is robust to model misspecifications and has adequate power to detect true marginal epistasis signals. We applied FAME to test for marginal epistasis at trait-associated genome-wide association study (GWAS) SNPs for 53 quantitative traits in the UK Biobank (UKBB) (N ≈ 300 000 unrelated White British individuals and M ≈ 500 000 SNPs on the UKBB genotyping array) and attempted to replicate the discovered signals in the All of Us (AoU) dataset. To better characterize our findings, we attempted to partition marginal epistasis signals across chromosomes, estimate the proportion of variance explained by the marginal epistasis effects and interpret these signals using functional genomic data. We aim to test whether the effect of a target SNP on a phenotype is modulated by the genetic background of the individual by assessing whether the pairwise interactions of the target SNP with each of the remaining SNPs contribute, in aggregate, to variance in the phenotype. In contrast to testing for interactions at a chosen pair of SNPs, testing for marginal epistasis could potentially be more powerful when interaction effects are polygenic; that is, we have a large number of interactions each with a weak effect while also benefiting from the reduced multiple testing burden. To ensure that additive genetic effects are not incorrectly attributed to interactions, we jointly model the additive effects from all genome-wide SNPs (including the target SNP) in addition to the marginal epistasis effects. We would like to test whether the marginal epistasis variance component is significantly different from zero and to be able to estimate its value. In this example, we have genotypes at four SNPs denoted by x1,x2,x3,x4. We would like to test for marginal epistasis between SNP 3 (the target SNP) and the remaining SNPs. The additive effect sizes β are drawn from a distribution with variance parameter proportional to \({\sigma }_{g}^{2}\), whereas the marginal epistasis effect sizes α are drawn from a distribution with variance parameter proportional to \({\sigma }_{{gxg},t}^{2}\) where t = 3 and M = 4. Fitting such a model to biobank-scale data, containing hundreds of thousands of individuals and millions of SNPs, is computationally challenging. Approaches that aim to efficiently compute genome-wide epistatic genomic relationship matrices4 do not scale well with sample size while not directly estimating marginal epistasis. The CORE GREML approach30 explicitly estimates covariance between random effects but does not directly model epistasis. MAPIT25, a previous test for marginal epistasis, does not scale to large samples. FAME expands on our recent work31,32 to test and estimate marginal epistasis on biobank-scale data. Specifically, FAME utilizes a randomized Method-of-Moments (MoM) estimator that reduces the size of the input genotype and interaction matrices by multiplying each of these matrices with a prespecified number (B) of random vectors (each entry of the random vector being drawn independently from a standard normal distribution). This approach is used to approximate key computations in the MoM estimator (Methods). The accuracy of this estimator depends on B. We show that modest values of B ≈ 100 lead to a method that yields accurate estimates while being highly scalable. We assessed the false positive rate of FAME on simulated phenotypes with additive genetic effects but no genetic interactions. We simulated phenotypes based on genotypes from unrelated White British individuals in UKBB (N = 291,273 individuals, M = 459,792 SNPs). We set the proportion of trait variation explained by additive genetic effects (additive heritability) \({\sigma }_{g}^{2}\) = 0.25 and varied the proportion of variants that have non-zero additive effects (causal variants) p ∈ {0.01,0.10}. The key parameter in FAME is the number of random vectors B, which determines its scalability and stability (‘Efficient computation of variance components' in Methods). We use B = 100 in all analyses while exploring the impact of this choice in ‘Robustness of marginal epistasis signals'. We assessed the calibration of FAME when applied to two sets of target SNPs: target SNPs chosen randomly from the UKBB array and SNPs identified based on a GWAS to mirror our analyses of traits in UKBB (‘Application to UK Biobank phenotypes'). Although FAME is calibrated when the target SNPs were selected at random (Supplementary Fig. 1), its P values tend to be inflated when target SNPs were selected from a GWAS (Supplementary Fig. To address this issue, we regressed out additive effects at SNPs that lie within the linkage disequilibrium (LD) block (identified in the 1000 Genomes European population33) around the target SNP and excluded these SNPs when constructing the set of genetic interactions (the Et matrix in Methods). This approach led to calibrated results (Fig. We applied FAME to simulated phenotypes with additive but no marginal epistasis effects. Phenotypes were simulated using genotypes measured on N = 291,273 unrelated White British individuals in the UK Biobank, with varying ratios of causal SNPs (causal ratio) and additive genetic variance component \({\sigma }_{g}^{2}=0.25\). We first ran GWAS to identify significant SNPs, which were then used as target SNPs in a marginal epistasis test. We find no significant marginal epistasis signals (P < 5 × 10−8) across any of the settings and that the P values are calibrated with genomic inflation factor λgc = 0.958 (0.801,1.06) and 1.053 (0.754,1.427) for causal ratio = 0.01 and 0.1, respectively. b, Power analysis of FAME. We simulated phenotypes by fixing the additive genetic variance component to 0.3 (approximately the median value estimated on real traits) across N = 291,273 unrelated White British individuals in the UKBB-small dataset. We varied the magnitude of the marginal epistasis variance component (\({\sigma }_{{gxg},t}^{2}\)). We generated 1,000 replicates for each setting. We plot power for detecting marginal epistasis at a P value threshold of 0.05 as well as the genome-wide threshold of 5 × 10−8, with error bars representing the 95% bootstrap CIs from 10 K replicates. c, Accuracy of estimates of the marginal epistasis variance component (\({\sigma }_{{gxg},t}^{2}\)) in simulations. We used the same simulation as in b. We plot the error in the parameter estimates for each parameter setting. d, Runtime analysis of FAME. We computed the runtime of FAME applied to common SNPs on the UKBB whole-genome array data and varying sample size. We report the average across three runs at each setting. To explore the impact of model misspecification on FAME, we considered scenarios in which (1) the relationship between covariates and phenotype is nonlinear, (2) the genetic architecture includes gene-environment interactions with a hidden environmental variable, and (3) the environmental noise is heavy tailed and (4) heteroskedastic (Supplementary Note 1). Prior work has shown that tests of epistasis can suffer from inflated false positive rates due to imperfect tagging of causal variants34,35,36. This led us to consider scenarios in which genetic variants have large additive effects and, additionally, not all such variants are observed. We simulated this scenario by drawing causal variants from imputed SNPs while we applied FAME to analyze SNPs on the UKBB array (section ‘Datasets' in Methods and Supplementary Note 1). Finally, we considered a scenario where the dependence of additive effect sizes on minor allele frequency (MAF) and LD has a different architecture37 than the one assumed by FAME (Supplementary Note 1). We found that FAME remains calibrated across each of these settings (Supplementary Figs. Because it is of interest to compare the strength of marginal epistasis to marginal GWAS, we also evaluated the calibration of tests of the ratio \(\frac{{\sigma }_{{gxg},t}^{2}}{{\sigma }_{{gwas},t}^{2}}\) both when the additive model is correctly specified (Supplementary Note 2, Supplementary Fig. 7 and Supplementary Table 3) and under each type of model misspecification considered above (Supplementary Fig. 7 and Supplementary Table 3) to find that FAME produces calibrated P values. To assess power, we simulated phenotypes with non-zero marginal epistasis variance components with N = 291,273 (section ‘Simulations' of Methods). We observed that FAME has power ≥90% at P < 5 × 10−8 even when the variance explained by marginal epistasis is fairly low (\({\sigma }_{{gxg},t}^{2}\) = 0.005) (Fig. Further, FAME produced marginal epistasis estimates that were relatively unbiased (Fig. FAME can test marginal epistasis on 500 K individuals on a genome-wide dataset containing ≈500 000 SNPs in about 6 h with 32 GB RAM (Fig. We attempted to benchmark a previously proposed method, MAPIT25 (section ‘Runtime comparisons' of Methods) to find that its runtime grows rapidly even for modest numbers of individuals and SNPs: requiring more than 3 days to run on N = 20 K individuals with M = 10 000 SNPs (Supplementary Fig. We applied FAME to 53 quantitative traits across N = 291,273 unrelated White British individuals (chosen to have adequate power and minimize population stratification) genotyped at common SNPs on the UKBB array. Our target SNPs were a set of 15,601 LD-pruned SNPs that were found to be associated with each trait in a GWAS (section ‘UKBB GWAS' in Methods). We tested for marginal epistasis at the GWAS significant SNPs in which we also accounted for the additive effect of genome-wide SNPs and included age, sex and the top 20 genetic principal components (PCs) as fixed effect covariates. Our tests yielded 21 significant trait-SNP pairs across 13 traits (\(P < \frac{5\times {10}^{-8}}{53}\) to account for the multiple traits tested; Fig. To additionally ensure that the additive genetic effects surrounding the target SNP do not impact estimates of marginal epistasis, we applied FAME to each of these 21 trait-SNP pairs after regressing out the additive effects of all of the SNPs in the LD block of the target SNP to observe 16 trait-SNP pairs across 12 traits that retain significant P values for marginal epistasis (\(P < \frac{5\times {10}^{-8}}{53}\); Table 1, additional information, including the marginal effect size of the target SNP, its nearest gene and the corresponding variant type in Table 2). a, Manhattan plot of marginal epistasis loci across 53 complex traits in UKBB. Colored shapes denote trait-SNP pairs that are significant at \(P < \frac{5\times {10}^{-8}}{53}\); shapes with colored triangles denote loci that are statistically significant in our initial analysis and remained significant after we regressed out all SNPs within the LD block as fixed effects. b, Localization of marginal epistasis signals. For each of 16 trait-SNP pairs, we tested whether the marginal epistasis signals remained significant when testing against all SNPs on the same chromosome as the target SNP (after removing SNPs in the same LD block as the target SNP), which we term local, and against all SNPs on chromosomes different from the chromosome containing the target SNP, which we term distal. We then compared the overlap between the local and distal significant signals (\(P < \frac{5\times {10}^{-8}}{53}\)). c, The fraction of phenotypic variance explained by marginal epistatic effects (\({h}_{{gxg},t}^{2}\)) vs that explained by additive effects tagged by the genome-wide significant SNP t (\({h}_{{\text{gwas}},t}^{2}\)) for each of the 16 marginal epistasis significant trait-SNP pairs. Each dot represents the point estimates. Vertical (horizontal) bars denote the standard error of \({h}_{{gxg},t}^{2}\) (\({h}_{{\text{gwas}},t}^{2}\)). We describe the estimation of \({h}_{{\text{gwas}},t}^{2}\) and its standard error in Supplementary Note 6. To explore the impact of the randomization underlying FAME, we selected two traits: body mass index (BMI) and serum urate levels (urate) as exemplary traits where FAME did not and did detect an epistatic signal. We experimented with the number of random vectors (B) and found that B = 100 yields consistent results (Pearson correlation of the negative log P values across seeds ρ = 0.99 and = 0.98 for urate and BMI, respectively; Supplementary Fig. We also found that the estimates of significant marginal epistasis signals were concordant across the number of PCs included (varying the number of PCs included from 20 to 40; Supplementary Fig. 10) and across the analysis of imputed SNPs compared to array SNPs (Supplementary Fig. 10; see Supplementary Note 3 for details). Additionally, we applied FAME to a permuted version of the epistatic matrix relative to the target variant to find that none of the SNP-trait pairs are significant (Supplementary Note 3 and Supplementary Table 4). Although phenotype scale can affect our results, traits are inverse rank normalized in all our analyses so that the results of FAME are invariant to monotone transformations (such as the scaling transformation considered here) and the P values remain calibrated, as we confirm in simulations (Supplementary Note 3 and Supplementary Fig. We performed a series of analyses to validate our discoveries and attempted to interpret these signals. We assessed the robustness of the marginal epistasis signals detected by FAME using an internal and an external replication study. First, we split the UKBB unrelated White British individuals into a discovery and replication cohort of equal size. A GWAS followed by application of FAME to GWAS significant SNPs in the discovery cohort yielded three significant marginal epistasis trait-SNP pairs, all of which replicated (\(P < \frac{0.05}{3}\); Fig. To validate our marginal epistasis signals in the AoU dataset29, we identified five traits (of the 12 with at least one significant marginal epistasis SNP in UKBB) for which we had sufficiently large sample size of European ancestry (N > 50 000) individuals in AoU. Applying a similar analysis as in UKBB (section ‘Datasets' of Methods), we found that all five of our marginal epistasis signals replicated in AoU (\(P < \frac{0.05}{5}\); Fig. We report point estimates (and 95% CIs) of \({\sigma }_{{gxg},t}^{2}\) for genome-wide significant marginal epistasis loci (\(P < \frac{5\times {10}^{-8}}{53}\)) discovered in a discovery cohort and their estimates in a replication cohort. a, Our discovery cohort is about one-half of the unrelated White British individuals in UKBB, whereas the replication cohort consists of unrelated White British individuals in UKBB not used in the discovery cohort. b, We use the discovery cohort comprising all unrelated White British individuals in UKBB and the replication cohort comprising individuals of European ancestry in the AoU dataset. The detailed sample size of each trait in AoU is reported in Supplementary Table 5. Having demonstrated evidence for genome-wide marginal epistasis, we sought to understand where these interactions localize. We extended FAME to test for marginal epistasis of a target SNP with only a subset of SNPs while accounting for the additive effects of genome-wide SNPs. For each target SNP, we tested, in turn, for marginal epistasis paired with other SNPs that fall on the same chromosome (local marginal epistasis; gxglocal) and with SNPs on chromosomes distinct from the chromosome containing the target SNP (distal marginal epistasis; gxgdist). We confirmed that tests of \({\sigma }_{{gxg},{\text{dist}}}^{2}\) and \({\sigma }_{{gxg},{\text{local}}}^{2}\) are well-calibrated in simulations (section ‘Regional simulation and estimation' of Methods; Supplementary Fig. Applying this test to each of the 16 previously identified marginal epistasis loci, we found 5 and 12 loci with significant local and distal marginal epistasis effects, respectively (\(P < \frac{5\times {10}^{-8}}{53}\); Fig. For each of the 16 SNPs, we then ran genome-wide pairwise interaction analysis (GxGWAS) where we tested for the interaction effect of the target SNP on the corresponding trait when paired with remaining SNPs on the UKBB array (after regressing out the same set of covariates as in the marginal epistasis analysis). We found that significant pairwise effects were almost always located on the same chromosome and close to the target SNP (Supplementary Table 7). Of the four trait-SNP pairs demonstrating significant pairwise interaction on the same chromosome, we quantified the distributions of physical distance (Supplementary Fig. 13 (right), mean: 0.0019; median: 0.0005; min: 0; max: 0.0176) between the target and interactive SNPs. Overall, these SNP pairs are not highly correlated (r2 < 0.10) suggesting that the marginal epistatic effect is unlikely to be driven by LD. The only significant interaction involving SNPs on distinct chromosomes is found in the case of rs72654473 on lipoA. rs72654473, downstream of the APOE gene on chromosome 19, shows a significant interaction effect with rs6935921, near the LPA gene on chromosome 6, where rs6935921-T allele attenuates the effect of rs72654473 on lipoA (Fig. ivrt, rank-based inverse normal transformation. We estimated the proportion of trait variance explained by marginal epistasis at a target SNP t (marginal epistasis heritability, \({h}_{{gxg},t}^{2}\)) from the variance components estimated by FAME (Supplementary Note 4). Across the 16 trait-SNP pairs with significant marginal epistasis signal, estimates of \({h}_{{gxg},t}^{2}\) tend to be modest: 10−3 − 10−2. We compared these estimates to the heritability of the SNP based on its GWAS effect size (\({h}_{{\text{gwas}},t}^{2}\) estimated as the square of the GWAS effect size for a standardized genotype). We found that \({h}_{{gxg},t}^{2}\) estimates are larger than the corresponding \({h}_{{\text{gwas}},t}^{2}\) estimates: about 12x larger on average with a range of 0.59 to 43.89 (Fig. Estimates of \({h}_{{gxg},t}^{2}\) are not strongly correlated with the \({h}_{{\text{gwas}},t}^{2}\) estimates (Pearson's correlation coefficient ρ = 0.022). We performed a series of analyses to better understand the biological relevance of trait-SNP pairs showing significant marginal epistasis. Overlapping the 14 unique SNPs significant for marginal epistasis with sets of eQTLs and pQTLs (section ‘Datasets' of Methods), we found eleven to regulate gene expression or protein levels with ten identified as cis-eQTLs, whereas rs146534110, which harbors a marginal epistasis signal for lipoprotein A, was previously identified as a cis-pQTL (Supplementary Table 8). For each significant trait-SNP pair, we selected the 1,000 SNP pairs with the most significant interaction P values. To assess whether these interacting SNPs have functional relevance, we combined sets of SNPs across all traits and used RegulomeDB38 to test if the interacting SNPs were enriched for functional annotations at the SNP sites (measured as the fraction of sites with rank score ≥ 1 f) relative to a background of GWAS variants pooled across all outcomes (repeated 10,000 times). For each background set, we randomly selected the same number of GWAS SNPs as the former set and computed their RegulomeDB functional importance scores. We observed that the analysis revealed significant functional enrichment for the interactive SNPs (P = 0.033). These results indicate that the SNPs that interact with the marginal epistasis SNPs are significantly enriched for high-ranking regulatory elements. To test whether the interactive SNPs are enriched for binding of specific transcription factors (TFs), we applied HOMER39 and analyzed a ± 100 bp window around the top 1,000 interactive SNPs for TF motif enrichments. We observed significant enrichment for multiple known (FDR < 0.05) (Supplementary Data 1) and de novo (P < 1 × 10−12) TF binding sites (Supplementary Data 2). Overall, these analyses revealed enrichments especially for TFs with helix-loop-helix domains as well as for GATA and nuclear hormone receptor TFs with zinc finger domains, which are known to be involved in the regulation of metabolic traits40,41,42. We then attempted to investigate individual marginal epistatic loci for alanine aminotransferase and testosterone (see Supplementary Note 5 for interpretation of loci underlying lipoprotein A and lipids). We investigated rs3827385, which shows significant marginal epistasis for alanine aminotransferase (ALT) and lies in the intron of the SAMM50 (Table 2), a gene whose protein product plays an important role in mitochondrial structure and function43 and in fatty acid oxidation44. This variant is located approximately 60 kb from and in moderate linkage disequilibrium (r2 = 0.36, Lewontin's D′ = 0.67) with a well-studied missense variant (rs738409, p.I148M) in the functionally relevant PNPLA3 gene. Previous studies have established the association of rs738409 with chronic liver disease45,46. Our analysis revealed significant marginal epistasis effects of rs738409 on ALT, with a proportion of variance explained comparable to that of rs3827385 (P = 4.90 × 10−24, \({\sigma }_{{gxg},t}^{2}\) = 14.8 × 10−3). Notably, prior work47 has identified an interaction between rs738409 and a splice variant rs72613567 in HSD17B13 on chromosome 4, affecting liver function as measured by ALT and aspartate aminotransferase (AST) levels. Specifically, it has been found that the HSD17B13 rs72613567:TA allele, associated with reduced aminotransferase levels, attenuates the effect of rs738409 on aminotransferase levels and is linked to reduced PNPLA3 mRNA expression. A pairwise epistasis test between rs72613567 and rs738409 for ALT in UKBB data confirmed this interaction is nominally significant (\({\hat{\beta }}_{\text{pairwise}}\) = −2.52×10−2, P = 1.94 × 10−7; Supplementary Table 9 and Fig. Given the established role of rs738409 in liver function, we next sought to investigate additional potential interactors. We selected rs58542926, a missense variant in the gene TM6SF2, that has been shown to be associated with ALT and liver disease48. We find that the interaction between rs58542926 and rs738409 on ALT is nominally significant with the rs58542926:T allele increasing the effect of rs734809 on ALT levels (\({\hat{\beta }}_{\text{pairwise}}\) = 2.91 × 10−2, P = 3.48 × 10−4; Supplementary Table 9 and Fig. Given that testosterone is a highly sex-specific trait, we investigated sex-specificity of marginal epistasis effects of the two testosterone-associated marginal epistasis variants, rs11555142 and rs28990703. We observed differences in marginal epistasis estimates across sexes at both SNPs, with rs11555142 being significant only in females and rs28990703 being significant only in males (Supplementary Fig. 49), we observed that the SNP rs58964157, which is in complete linkage disequilibrium (LD; R2 = 1.0 in Europeans) with rs11555142, alters the binding of estrogen receptor 2 (ESR2) and estrogen receptor α (ESR1), the two critical transcription factors (TFs) for estrogen regulation50,51, consistent with rs11555142 having a significant female-specific marginal epistasis effect. We also identified rs28990703 as a binding site for LEF1, a TF known to regulate androgen signaling and implicated in prostate cancer52. No tight LD proxies were found for rs28990703, with the closest LD proxy of R2 = 0.4 in HaploReg 4.2. Intriguingly, this site also binds ZNF652, a TF linked to prostate cancer and androgen metabolism53, which are both sex specific. These findings suggest that rs28990703 itself may directly contribute to the observed sex differences by interacting with other variants and influencing androgen-related pathways. We have presented a new method, FAME, that can detect marginal epistasis in biobank-scale data. A significant challenge to robust identification of marginal epistasis (and epistasis more generally) arises due to model violations. In extensive simulations that include non-linear covariates, gene-environment interactions, heavy-tailed environmental noise, and imperfect tagging of causal variants, we show that FAME yields calibrated results. Applying FAME to 53 quantitative phenotypes in the UK Biobank, we found 16 trait SNP pairs with genome-wide significant signals of marginal epistasis. Although the number of signals detected is modest (in part due to the stringent P value threshold employed and the strategy employed to select target SNPs), we observe that the proportion of variance explained by marginal epistasis is comparable to, and sometimes substantially larger than, the proportion of variance explained by GWAS. This observation suggests that polygenic background can substantially modulate the effect of a genetic variant on a trait and has implications for efforts to interpret variant effects, improve phenotype prediction and understand how genetic effects vary across populations5. We further partitioned the marginal epistasis signal within and across chromosomes to detect both within and cross-chromosomal signals. Although our current application has focused on testing for interactions of a single target SNP paired with SNPs across the genome, the model underlying FAME is flexible. For example, FAME can be extended to test for interactions of a target SNP or covariate (such as polygenic scores) with SNPs defined based on functional annotations such as genes or pathways. The scalability of FAME arises from its use of the randomized MoM estimator. Likelihood-based methods that maximize the likelihood or restricted likelihood (REML) (using the EM, NR, Fisher or AI algorithms for optimization) could be alternatively employed. When effect sizes and the environmental noise are all normally distributed (leading to a tractable likelihood), REML is more statistically efficient than MoM. Most important to our application, the computational efficiency of randomized MoM enables its application to biobank-scale datasets where, as far as we know, likelihood-based approaches are infeasible. The question of which estimator to use depends on considerations of model assumptions and dataset size. Efficient likelihood-based estimators in epistasis models remain an interesting topic for future research. Our study has several limitations. First, our approach of regressing out additive effects in the LD block surrounding the target SNP and only testing for interactions outside is important for calibration but likely misses local epistatic signals or signals in regions of long-range linkage disequilibrium54. Additionally, the model underlying FAME assumes that interaction effects are independent of main effects. It would be of interest to extend our method to settings where epistasis is coordinated26,55. Second, the scale on which phenotypes are measured can affect our results. In all our simulations and analyses, traits are inverse rank normalized so that they are invariant to monotone transformations. Although analyzing phenotypes on other scales might lead to more interpretable estimates, the robustness of such analyses needs to be assessed (Supplementary Note 3). Third, we have only applied FAME to quantitative traits. Analyses of binary (disease) traits would require exploring the impact of trait prevalence and ascertainment. Fourth, although we have focused on testing for marginal epistasis at common SNPs, it would be of great interest to apply FAME to rare variants. Fifth, our estimates of marginal epistasis effects are likely to be biased upwards due to winner's curse56 although our replication experiment suggests that the bias is modest. Sixth, we have limited our analysis to GWAS significant SNPs in this study. The ability to estimate marginal epistasis across the genome would allow us to obtain a more comprehensive understanding of the nature of epistasis underlying complex traits. Finally, integration of FAME with data from biological pathways and functional genomics will be essential to move towards an understanding of biological epistasis10. Extending the scope, efficiency and generality of FAME present important directions for future work. Ethics committee/IRB of UKBB gave ethical approval for collection of UKBB data (https://www.ukbiobank.ac.uk/learn-more-about-uk-biobank/about-us/ethics). Approval to use UKBB individual level in this work was obtained under application 33127 at http://www.ukbiobank.ac.uk. Ethics committee/IRB of AoU gave ethical approval for collection of AoU data (https://allofus.nih.gov/about/who-we-are/ institutional-review-board-irb-of-all-of-us-research-program). Approval to use AoU controlled tier data in this work was obtained through application at https://www.researchallofus.org. Given an N×M standardized genotype matrix X, an N-vector of centered phenotypes y and a target SNP t∈{1, …, M} we aim to jointly test the additive effect of the M SNPs and marginal epistasis of the target SNP using the following model that was originally introduced in25: Here \({\mathscr{N}}{({\mathbf{\upmu}},{\varSigma})}\) is a multivariate normal distribution with mean μ and covariance matrix Σ, \({E}_{t}\) denotes an N × (M − 1) genetic interaction matrix formed by pairing the target SNP with all other SNPs in the genome, defined formally as \({E}_{t}={X}_{-t}\odot {X}_{:t}\) where \({X}_{:t}\) is the t-th column (corresponding to the target SNP) of X and \({X}_{-t}\) is the matrix consisting of all SNPs except the target SNP formed by excluding the t-th column from X. In this model, \({\sigma }_{e}^{2},{\sigma }_{g}^{2}\) and \({\sigma }_{{gxg},t}^{2}\) are the residual, additive genetic and marginal epistasis variance components, respectively. β denotes the vector of additive SNP effects and \({{\mathbf{\upalpha }}}_{t}\) denotes the vector of interaction effects between target SNP t and each of the other SNPs in the genome. This model assumes that the interaction effects are independent of the main effects so that epistasis is uncoordinated26. To estimate the variance components of our LMM, we use a Method-of-Moments (MoM) estimator. As E[y] = 0, we derived the MoM estimates by equating the population covariance to the empirical covariance where the population covariance is given by:\({\varSigma }=\mathrm{Cov}\left({\mathbf{y}}\right)=E\left[{\mathbf{y}}{{\mathbf{y}}}^{T}\right]-E\left[{\mathbf{y}}\right]E\left[{{\mathbf{y}}}^{T}\right]={\sigma }_{g}^{2}\frac{1}{M}{X}{{X}}^{T}+{\sigma }_{{gxg},t}^{2}\frac{1}{M-1}{{E}}_{t}{{E}}_{t}^{T}+{\sigma }_{e}^{2}{I}\). Using \({\mathbf{y}}{{\mathbf{y}}}^{T}\) as our estimate of the empirical covariance, we need to solve the following least squares problem to estimate the variance parameters: We show that the MoM estimator satisfies the following normal equations (Lemma 1 in Supplementary Notes): Thus, the MoM estimate of the variance components \({\widetilde{{\boldsymbol{\sigma }}}}^{2}\) =(\({\widetilde{\sigma }}_{g}^{2},{\widetilde{\sigma }}_{{gxg},t}^{2},{\widetilde{\sigma }}_{e}^{2})\) is given by: To compute standard errors which, in turn, allow us to test the hypothesis of no marginal epistasis (\({\sigma }_{{gxg},t}^{2}=0\)), we compute the covariance matrix of \({\widetilde{{\boldsymbol{\sigma }}}}^{2}\) as (Lemma 2 in Supplementary Notes): Computing the MoM estimates and their standard errors requires O(N2 M) runtime and O(N2) memory that render it impractical for biobank-scale data. To efficiently compute \({\widetilde{{\boldsymbol{\sigma }}}}^{2}\), we note that each of the coefficients of the matrix T involves computing the trace of a matrix which we approximate by a stochastic trace estimator57. Specifically, we estimate \({T}_{{kl}}\) as: where each \({{\boldsymbol{v}}}_{b}\) is an independent random vector with mean zero and covariance \({I}_{N}\), B is the total number of random vectors used for the approximation, and Zk = X or \({E}_{t}\). Each term \({Z}_{k}{Z}_{k}^{T}{{\mathbf{v}}}_{b}\) can be computed in \(O(\frac{{NM}}{\max ({\log }_{3}N,{\log }_{3}M)}\)) by using the Mailman algorithm58 so that \({\hat{T}}_{{kl}}\) can be computed in \(O(\frac{{NMB}}{\max ({\log }_{3}N,{\log }_{3}M)}\)) time. We note that \({\hat{T}}_{{kl}}\) is an unbiased estimator of \({T}_{{kl}}\) provided \({{\mathbf{v}}}_{b}\) has zero mean and identity covariance matrix. In practice, we draw each entry of \({{\mathbf{v}}}_{b}\) independently from a standard normal distribution. The variance of \({\hat{T}}_{{kl}}\) decreases with the increasing number of random vectors (\(B\)). Empirically, we observed that utilizing 100 random vectors yields a sufficiently accurate estimator (Supplementary Fig. To estimate \({\rm{Cov}}\left[{\mathbf{q}}\right]\) efficiently, we use a plug-in estimate of \({\boldsymbol{\Sigma }}\) to obtain: Further, FAME uses a streaming implementation that does not require all the genotypes to be stored in memory leading to scalable memory requirements. FAME can also account for fixed-effects covariates such as age, sex and genetic PCs (Supplementary Note 7). We allow for negative variance component estimates that we exclude from the analysis, unless specified otherwise. The corresponding P value is computed using a two-sided test. We designed simulations to assess the power of FAME and the accuracy of its marginal epistasis variance components estimates using the following generative model: Here \({\beta }_{j}\) and \({\alpha }_{t,\,j}\) denotes the j-th element in the respective vectors of effect sizes. We set \({\sigma }_{g}^{2}\) to 0.3, which is approximately the median value of the additive heritability across all the traits that we analyzed in this study. We varied the value of \({\sigma }_{{gxg},t}^{2}\) from 0.001 to 0.1. As target SNP, we selected three representative SNPs with a range of MAF (∈{1%,14%, 49%}, respectively). For computational convenience, we used genotypes in the UKBB-small dataset (section ‘Datasets' of Methods). We simulated 1,000 replicates for each setting. To assess the accuracy of the marginal epistasis variance component estimates obtained by FAME, we used the same simulations as above. We used FAME to jointly fit the additive effect for both regions while only fitting the marginal epistasis effect on the second region. To validate the calibration of FAME when applied to test the marginal epistasis effect on a specified region, we used the simulation with a total heritability of 0.25 and ratio of causal SNPs of 1%. We applied FAME to estimate the calibration of \({\sigma }_{{gxg},{\text{local}}}^{2}\) and \({\sigma }_{{gxg},{\text{distal}}}^{2}\), respectively. All experiments used a machine equipped with AMD EPYC 7501 32-Core Processor, and a runtime budget of 3 days was provided to all tested methods. We obtained a set of N = 291,273 unrelated White British individuals measured at M = 459,792 common SNPs genotyped on the UK Biobank Axiom array to use in simulations by extracting individuals that are more distantly related than third-degree relatives and excluding individuals with putative sex chromosome aneuploidy. Unless otherwise specified, all simulations were conducted using this dataset. We used a subset of the UKBB array genotypes obtained by restricting to SNPs on chromosomes 12 and 20 across 291,273 unrelated White British individuals. We excluded SNPs with MAF less than 1% resulting in a final set of 32,708 SNPs. We term this dataset ‘UKBB-small'. For simulations that involve missing SNPs, we constructed imputed genotypes by restricting to SNPs on chromosomes 12 and 20 named ‘UKBB-small-imputed'. For analysis of real traits, we restricted our analysis to SNPs that were present in the UK Biobank Axiom array used to genotype the UK Biobank. SNPs with greater than 1% missingness and minor allele frequency smaller than 1% were removed. Moreover, SNPs that fail the Hardy–Weinberg test at significance threshold 10−7 were removed. We restricted our study to self-reported British White ancestry individuals who are more than third-degree relatives, defined as pairs of individuals with kinship coefficient <1/2(9/2) (ref. Furthermore, we removed individuals who are outliers for genotype heterozygosity and/or missingness and excluded SNPs that fall within the MHC region. Finally, we obtained a set of N = 291,273 individuals and M = 454,207 SNPs for real data analyses. We used this dataset in our analyses unless specified otherwise. We also analyzed imputed genotypes across N = 291,273 unrelated White British individuals. We removed SNPs with greater than 1% missingness, minor allele frequency smaller than 1%, SNPs that fail the Hardy–Weinberg test at significance threshold 10−7 as well as SNPs that lie within the MHC region (Chr6:25–35 Mb) to obtain 4,824,392 SNPs. We selected 53 quantitative traits in UKBB (Supplementary Table 10). These traits were chosen because they have been analyzed in prior work, by us and others, to estimate aspects of genetic architecture59,60 and span a diverse set of phenotypic categories: Anthropometry, Blood Biochemistry, Bone, Cardiovascular, Diabetes, Eye, Liver, and Renal. We included sex, age and the top 20 genetic PCs as covariates in our analysis for all phenotypes. Extra covariates were added for diastolic/systolic blood pressure (adjusted for cholesterol-lowering medication, blood pressure medication, insulin, hormone replacement therapy and oral contraceptives). We used the PCs computed in the UKBB from a superset of 488,295 individuals. Following prior studies, all traits were inverse rank normalized60. We ran GWAS on each of the 53 traits, including covariates as described above. For each trait, we selected SNPs with P < 5 × 10−8 followed by LD pruning (using a window size of 500 SNPs, we computed r2 between each pair and removed one of them if r2 > 0.1, shifting the window by 1 SNP, and repeating the process). We curated the AoU genotype dataset starting from the AoU srWGS dataset and identifying the subset of SNPs that matched the UKBB SNP set resulting in 457,218/459,792 (99.4%) SNPs. We restricted our analyses to individuals whose European ancestry component of genetically inferred ancestry (GIA) > 95%. We then applied a genotype QC procedure with the same parameters used for the UKBB yielding 392,756 SNPs. For each trait, when multiple measurements were available, we selected the earliest measurement and adjusted the age covariate accordingly for that individual. Individuals whose adjusted age was <18 years were excluded from the analysis. For the 12 traits that showed evidence for genome-wide marginal epistasis signals, we found five with >50 000 individuals with European ancestry. We analyzed associations that were discovered in the following three datasets: Võsa et al. identified cis-eQTLs (SNP-gene distance<1 Mb, FDR < 0.05) for 16,987 genes and trans-eQTLs (SNP-gene distance>5 Mb, FDR < 0.05) for 6,298 genes in 31,684 blood samples61. For trans-eQTL analyses, this study focused on 10,317 trait-associated SNPs from GWAS. Yao et al. measured the levels of 71 plasma proteins that are known to be associated with cardiovascular disease (CVD) in 6,861 Framingham Heart Study (FHS) participants. We analyzed the 16,602 protein QTL (pQTL) variants associated with 57 proteins identified in this study62. This set included 11,806 cis-pQTL variants (P < 1.25 ×10−7) for 40 proteins and 4,796 trans-pQTL variants (P < 7.04 ×10−10) for 44 proteins. Ferkingstad et al. measured the levels of 4,907 plasma proteins in 35,559 Icelandic individuals. We analyzed the 18,084 sentinel pQTL associations (defined as the most significant association detected within a 1 Mb region surrounding the gene encoding the protein) associated with 4,631 proteins identified in this study (P < 1.8×10−9 corresponding to an FDR of 1.3%; ref. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The UK Biobank dataset used in this study is not publicly available but can be obtained by application (https://www.ukbiobank.ac.uk/). The All of Us dataset used in this study can be accessed via the public Data Browser upon approval (https://databrowser.researchallofus.org/). The eQTL data analyzed in this study are available from the eQTLGen consortium61. The pQTL data were obtained from the supplementary materials of Yao et al.62 (Supplementary Table 3) and Ferkingstad et al.63 (Supplementary Table 2). FAME can be found at https://github.com/sriramlab/FAME and has been archived on Zenodo at https://doi.org/10.5281/zenodo.15814715 ref. MAPIT can be found at https://github.com/lorinanthony/MAPIT. Wei, W., Hemani, G. & Haley, C. S. Detecting epistasis in human complex traits. Missing heritability and strategies for finding the underlying causes of complex disease. Hill, W. G., Goddard, M. E. & Visscher, P. M. Data and theory point to mainly additive genetic variance for complex traits. Hivert, V. et al. Estimation of non-additive genetic variance in human complex traits from a large sample of unrelated individuals. Patel, R. A. et al. Genetic interactions drive heterogeneity in causal variant effect sizes for gene expression and complex traits. Variable prediction accuracy of polygenic scores within an ancestry group. Martin, A. R. et al. Human demographic history impacts genetic risk prediction across diverse populations. Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Thornton-Wells, T. A., Moore, J. H. & Haines, J. L. Dissecting trait heterogeneity: a comparison of three clustering methods applied to genotypic data. Singhal, P. et al. Gene interactions in human disease studies—evidence is mounting. & Liu, J. S. Bayesian inference of epistatic interactions in case-control studies. Tang, W. et al. Epistatic module detection for case-control studies: a Bayesian model with a Gibbs sampling strategy. Gyenesei, A. et al. High-throughput analysis of epistasis in genome-wide association studies with BiForce. Evans, D. M. et al. Two-stage two-locus models in genome-wide association. Prabhu, S. & Pe'er, I. Ultrafast genome-wide scan for SNP–SNP interactions in common complex disease. Kam-Thong, T. et al. Epistasis detection on quantitative phenotypes by exhaustive enumeration using GPUs. Wang, Z. et al. eCEO: an efficient Cloud Epistasis cOmputing model in genome-wide association study. Wienbrandt, L. et al. FPGA-based acceleration of detecting statistical epistasis in GWAS. Strange, A. et al. A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1. Efficient two-step testing of gene-gene interactions in genome-wide association studies. Ma, L. et al. Knowledge-driven analysis identifies a gene–gene interaction affecting high-density lipoprotein cholesterol levels in multi-ethnic populations. Chen, G. K. & Thomas, D. C. Using biological knowledge to discover higher order interactions in genetic association studies. Park, D. S. et al. An ancestry-based approach for detecting interactions. Crawford, L. et al. Detecting epistasis with the marginal epistasis test in genetic mapping studies of quantitative traits. Sheppard, B. et al. A model and test for coordinated polygenic epistasis in complex traits. Crawford, L. & Zhou, X. Genome-wide marginal epistatic association mapping in case-control studies. The UK Biobank resource with deep phenotyping and genomic data. All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. Zhou, X., Im, H. K. & Lee, S. H. CORE GREML for estimating covariance between random effects in linear mixed models for complex trait analyses. & Sankararaman, S. A scalable estimator of SNP heritability for biobank-scale data. Efficient variance components analysis across millions of genomes. Berisa, T. & Pickrell, J. K. Approximately independent linkage disequilibrium blocks in human populations. Detection and replication of epistasis influencing transcription in humans. Wood, A. R. et al. Another explanation for apparent epistasis. Improved heritability estimation from genome-wide SNPs. Boyle, A. P. et al. Annotation of functional variation in personal genomes using RegulomeDB. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Wang, L. H. & Baker, N. E. E proteins and ID proteins: helix-loop-helix partners in development and disease. Tong, Q. et al. Function of GATA transcription factors in preadipocyte-adipocyte transition. Emerging roles of zinc finger proteins in regulating adipogenesis. Ott, C. et al. Sam50 functions in mitochondrial intermembrane space bridging and biogenesis of respiratory complexes. The role of SAMM50 in non-alcoholic fatty liver disease: from genetics to mechanisms. Romeo, S. et al. Genetic variation in PNPLA3 confers susceptibility to nonalcoholic fatty liver disease. Eric, T. et al. PNPLA3 gene in liver diseases. Noura, S. A. et al. A protein-truncating HSD17B13 variant and protection from chronic liver disease. Liu, Y. L. et al. TM6SF2 rs58542926 influences hepatic fibrosis progression in patients with nonalcoholic fatty liver disease. Lucas, D. W. & Manolis, K. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Greene, G. L. et al. Sequence and expression of human estrogen receptor complementary DNA. Li, Y. et al. LEF1 in androgen-independent prostate cancer: regulation of androgen receptor expression, prostate cancer growth, and invasion. Callen, D. F. et al. Co-expression of the androgen receptor and the transcription factor ZNF652 is related to prostate cancer outcome. Evidence of epistasis in regions of long-range linkage disequilibrium across five complex diseases in the UK Biobank and eMERGE datasets. & Dahl, A. Factorizing polygenic epistasis improves prediction and uncovers biological pathways in complex traits. Xiao, R. & Boehnke, M. Quantifying and correcting for the winner's curse in genetic association studies. Hutchinson, M. F. A stochastic estimator of the trace of the influence matrix for Laplacian smoothing splines. Liberty, E. & Zucker, S. W. The mailman algorithm: a note on matrix–vector multiplication. Pazokitoroudi, A. et al. Quantifying the contribution of dominance deviation effects to complex trait variation in biobank-scale data. Sinnott-Armstrong, N. et al. Genetics of 35 blood and urine biomarkers in the UK Biobank. Large-scale cis-and trans-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression. Yao, C. et al. Genome-wide mapping of plasma protein QTLs identifies putatively causal genes and pathways for cardiovascular disease. Large-scale integration of the plasma proteome with genetics and disease. This research was conducted using the UK Biobank Resource under application 33127. We gratefully acknowledge the participants of UK Biobank and All of Us for their contributions, without whom this research would not have been possible. We also thank the National Institutes of Health's All of Us Research Program for making available the participant data examined in this study. This work was supported, in part, by NIH grants GM125055, GM153406 (B.F., A.P., A.A., P.A., Z.L. was supported by NIH grants R01MH130581, U01MH126798, R01MH122688 and R01GM142112. was supported by NIH grants R01HL170604 and R01DK132775. These authors contributed equally: Boyang Fu, Ali Pazokitoroudi. Department of Biomedical Informatics at HMS, Harvard University, Boston, MA, USA Department of Computer Science, University of California, Los Angeles, Los Angeles, CA, USA Graduate Group in Genomics and Computational Biology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Bioinformatics Interdepartmental Program, University of California, Los Angeles, Los Angeles, CA, USA Asha Kar, Albert Xue & Päivi Pajukanta Department of Human Genetics, David Geffen School of Medicine at UCLA, Los Angeles, CA, USA Asha Kar, Päivi Pajukanta, Noah Zaitlen & Sriram Sankararaman Department of Computational Biology, CMU, Pittsburgh, PA, USA Institute for Precision Health, David Geffen School of Medicine at UCLA, Los Angeles, CA, USA Department of Neurology, University of California, Los Angeles, Los Angeles, CA, USA Department of Computational Medicine, David Geffen School of Medicine at UCLA, Los Angeles, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to the AoU analysis. contributed to the biological interpretation. assisted with the experimental analysis. provided experimental guidance and manuscript proofreading. The authors declare no competing interests. Nature Genetics thanks Shefali Verma and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Fu, B., Pazokitoroudi, A., Shi, Z. et al. A biobank-scale test of marginal epistasis reveals genome-wide signals of polygenic interaction effects. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            