
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-30</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02085-w'>Ageing is linked to inflammation — but only in the industrialized world</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 15:47:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. People from Indigenous communities do not show the same link between chronic inflammation and age-related illness seen in industrialized societies, a study that looked at nearly 3,000 adults in four countries has found. The latest findings, published in Nature Aging on 30 June1, show that chronic inflammation — which has been long considered a hallmark of ageing — could be a feature of industrialized living. They found that inflammation levels increased with age and were linked with illnesses such as chronic kidney disease in the Italian and Singaporean groups. But in the two Indigenous groups, inflammation did not increase with age or lead to health conditions. This suggests that “our assumption that inflammation is an inexorable, inevitable part of ageing is not true”, says Thomas McDade, a biological anthropologist at Northwestern University in Evanston, Illinois. “We shouldn't assume that the links between inflammation and ageing are universal.” “We're at a point where we're rethinking the whole nature of inflammation,” says study co-author Alan Cohen, who researches ageing at Columbia University in New York City. “The things that we think of as universal based on a lot of studies in Western industrialized populations are probably just particular to our environment.” Much of what researchers know about the biological processes that underlie ageing is based on research in wealthy countries, which suggests that inflammation increases with age and can contribute to health conditions such as Alzheimer's disease, diabetes and heart problems. More than 50% of the people in each group were female. Ageing studies in five animals suggest how to reverse decline Ageing studies in five animals suggest how to reverse decline The researchers measured how eight types of proteins called cytokines — molecules released by immune cells that are involved in inflammation — changed with age in each group, and whether their levels were associated with age-related health problems. Ageing studies in five animals suggest how to reverse decline RFK Jr's vaccine advisers vote down flu-shot ingredient — but back some jabs The Integrated Science Lab (IceLab) offers three postdoctoral scholarships that will be affiliated with one of six possible multidisciplinary projects Ageing studies in five animals suggest how to reverse decline An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250630073442.htm'>Scientists just found a sugar switch that protects your brain from Alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 14:05:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new study from scientists at the Buck Institute for Research on Aging has revealed a surprising player in the battle against Alzheimer's disease and other forms of dementia: brain sugar metabolism. Published in Nature Metabolism, the research uncovers how breaking down glycogen -- a stored form of glucose -- in neurons may protect the brain from toxic protein buildup and degeneration. Glycogen is typically thought of as a reserve energy source stored in the liver and muscles. The research team, led by postdoc Sudipta Bar, PhD, discovered that in both fly and human models of tauopathy (a group of neurodegenerative diseases including Alzheimer's), neurons accumulate excessive glycogen. Bar says tau, the infamous protein that clumps into tangles in Alzheimer's patients, appears to physically bind to glycogen, trapping it and preventing its breakdown. When glycogen can't be broken down, the neurons lose an essential mechanism for managing oxidative stress, a key feature in aging and neurodegeneration. Rather than using glycogen as a fuel for energy production, these enzyme-supported neurons rerouted the sugar molecules into the pentose phosphate pathway (PPP) -- a critical route for generating NADPH (nicotinamide adenine dinucleotide phosphate) and Glutathione, molecules that protect against oxidative stress. "By increasing GlyP activity, the brain cells could better detoxify harmful reactive oxygen species, thereby reducing damage and even extending the lifespan of tauopathy model flies," said Bar. Even more promising, the team demonstrated that dietary restriction (DR) -- a well-known intervention to extend lifespan -- naturally enhanced GlyP activity and improved tau-related outcomes in flies. They further mimicked these effects pharmacologically using a molecule called 8-Br-cAMP, showing that the benefits of DR might be reproduced through drug-based activation of this sugar-clearing system. "This work could explain why GLP-1 drugs, now widely used for weight loss, show promise against dementia, potentially by mimicking dietary restriction," said Kapahi. Researchers also confirmed similar glycogen accumulation and protective effects of GlyP in human neurons derived from patients with frontotemporal dementia (FTD), strengthening the potential for translational therapies. Kapahi says this study not only highlights glycogen metabolism as an unexpected hero in the brain but also opens up a new direction in the search for treatments against Alzheimer's and related diseases. "By discovering how neurons manage sugar, we may have unearthed a novel therapeutic strategy: one that targets the cell's inner chemistry to fight age-related decline," he says. "As we continue to age as a society, findings like these offer hope that better understanding -- and perhaps rebalancing -- our brain's hidden sugar code could unlock powerful tools for combating dementia." Hilsabeck, Sydney Alderfer, Jordan B Burton, Samah Shah, Anja Holtz, Enrique M. Carrera, Jennifer N. Beck, Jackson H Chen, Grant Kauwe, Tara E. Tracy, Birgit Schilling, and Lisa M. Ellerby. Other collaborators include Eric B. Dammer, Fatemeh Seifar and Nicholas T. Seyfried, Emory Center for Neurodegenerative Disease, Emory University School of Medicine, Atlanta, GA as well as Ananth Shantaraman, Department of Biochemistry, Emory University School of Medicine, Atlanta, GA Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65207682/lost-egyptian-city/'>Archaeologists Found a Lost Egyptian City Buried in the Nile Delta</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“This discovery opens new doors to our understanding of daily life, spirituality, and urban planning in the Delta.” Excavations at Tell el-Fara'in reveal cultural norms during the early- to middle-fourth century B.C., which feature intricate urbanization with “substantial tower houses,” structures that were basically multi-story buildings supported by exceptionally thick foundation walls, according to a statement from the University of Manchester. “Their presence here shows that Imet was a thriving and densely built city with a complex urban infrastructure.” The houses, with their multiple floors, were crafted to accommodate many people, Mohamed Ismail Khaled, secretary general of Egypt's Supreme Council of Antiquities, said in a translated statement from the Egyptian Ministry of Tourism and Antiquities. It was then they discovered the tower houses. Nielsen said Imet was prominent during the New Kingdom, sometime between 1550 and 1069 B.C. and its location along trade routes allowed it to maintain importance, especially as it attracted Wadjet pilgrims and merchants. Researchers believe that this new building, placed over  the ceremonial road shows the worship of Wadjet had waned by that time, offering insight into a shifting religious landscape in ancient Egypt. The past religious culture of Imet has started to come into greater clarity in the present day, thanks to finds like a green faience ushabti (a ceramic funerary figurine) from the 26th Dynasty, a stela (an upright stone slab with a relief design) of the god Harpocrates (the Greek name for Horus the Child) standing on two crocodiles while holding a snake, and a bronze sistrum musical instrument adorned with the twin heads of Hathor, goddess of music and joy. “This discovery opens new doors to our understanding of daily life, spirituality, and urban planning in the Delta,” Nielsen said. “Imet is emerging as a key site for rethinking the archaeology of Late Period Egypt.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Archaeologists Found a Bevy of Medieval Artifacts Did This Newly Found Compass Belong to Copernicus? This Skull Belongs to a Lost Human Species</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a65193552/archaea-cell-virus/'>Scientists Discovered a New Creature That Exists Between Life and Not-Life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Because they rely on hosts for a majority of functions, viruses aren't considered alive. At first glance, creating a definition for “life” seems somewhat straightforward. Sentient animals all the way down to single-celled organisms capable of reproduction are welcome on the tree of life, but there are other organisms that challenge this understanding, like viruses. But once a virus infects a host, it's immensely active, and can be responsible for world-altering events (see: Spanish flu, ebola, COVID-19, and so on). Recently, scientists found a new member of this head-scratching cadre. In a new paper published on the bioRxiv server, researchers in Canada and Japan outlined how they identified a new cellular entity that appeared to straddle the typical definitions of a virality and cellular life. Currently named ‘Sukunaarchaeum mirabile' (after a deity in Japanese mythology known for its small stature), this entity contains the necessary genes to create its own ribosomes and messenger RNA—something your typical virus lacks. “Its genome is profoundly stripped-down, lacking virtually all recognizable metabolic pathways, and primarily encoding the machinery for its replicative core: DNA replication, transcription, and translation,” the authors wrote. “This suggests an unprecedented level of metabolic dependence on a host, a condition that challenges the functional distinctions between minimal cellular life and viruses.” Within the genomic data, Harada and his team found a loop of DNA that didn't match with any known species. They eventually determined that the organism belonged to the domain Archaea—a group associated with prokaryotic cells, but from which eukaryotic cells (i.e. you and me) ultimately descended a couple billion years ago. Viruses, as Live Science points out, can contain many hundreds of thousands more base pairs, and can even reach up into the millions. As for fellow archaea, the smallest known complete genome within this group stretches to 490,000 base pairs, meaning that Sukunaarchaeum contains less than half the number of base pairs posessed by even the smallest archaeal genome. A Colossal Squid Might Have Shown Its Secret Face The Rodents of Chicago Are Evolving in Real Time Dinos Had Cancer, Too, According to a New Study</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65203321/norfolk-highway-discovery/'>Archaeologists Found the Remains of a Medieval Village During Highway Renovations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Throughout its long existence, Norfolk has become a mosaic of British history. Many rulers, including Celtic Queens, Roman Empires, Vikings, and Saxons, have shaped the county into a rich blend of heritage, and our knowledge surrounding the area is constantly growing with new discoveries. Archeologists just made another one—and it was totally by accident. While doing construction on the A47, a major highway in England, the team discovered remains from a medieval village. “Our works are primarily about improving safety on the roads,” National Highways' A47 project leader Chris Griffin said in the email, “but they also present a great opportunity, through the archaeology, to find out more about the wonderfully rich cultural heritage in Norfolk.” Excavations took place between the villages of North Tuddenham and Easton. There, archaeologists from Headland Archeology used heavy equipment and hand tools to uncover artifacts such as both battle and hunting spearheads, pottery, metal work, stone carvings, decorative buckles, and animal remains. Norfolk was once a prosperous civilization during medieval times. But it wasn't all smooth sailing—some of the finds, including a bone flute, revealed a deeply traumatic part of Norfolk's history. “Unfortunately, [Norfolk] coincided with a climate change event called the ‘medieval warm period' where it started to get warmer and wetter and those crops started to fail which exposed the population to famine and disease,” Kate Bane, Project Manager at Headland Archaeology, explained. So, these villages, which would have been recognisable all the way through the landscape like little hamlets, disappeared as people fled to urban settlements, leaving them behind.” Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. Did This Newly Found Compass Belong to Copernicus? This Skull Belongs to a Lost Human Species</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250630073438.htm'>Ancient DNA reveals leprosy hit the Americas long before colonization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 12:08:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Today it exists in over 100 countries, and while the infection is treatable, access to treatment varies widely with socioeconomic conditions. From a genetic perspective, Hansen's Disease is caused by either the globally dominant Mycobacterium leprae or the newly identified and rare Mycobacterium lepromatosis. Similar emergence estimates have been proposed for other notorious diseases such as plague, tuberculosis, and typhoid fever. Ancient genomes for M. lepromatosis have remained elusive, and these may hold important clues on the history of Hansen's Disease. "Ancient DNA has become a great tool that allows us to dig deeper into diseases that have had a long history in the Americas," says Kirsten Bos, group leader for Molecular Paleopathology at the Max Planck Institute for Evolutionary Anthropology. She and her team have been studying pathological bone from American context for over a decade. Bos's team worked closely with researchers from Argentina and Chile to both identify bones suitable for analysis and to carry out the meticulous work of isolating the DNA of ancient pathogens. Doctoral candidate Darío Ramirez of the University of Córdoba, Argentina, worked extensively with such material, and was the first to identify a genetic signature related to leprosy in some 4000-year-old skeletons from Chile. This provided the first clue that M. lepromatosis and M. leprae, though nominally both pathogens that cause Hansen's Disease, might have very different histories. Reconstruction of the genome was key in looking into this. While putting the molecular puzzle of an ancient genome back together is never an easy task, these pathogens in particular had "amazing preservation, which is uncommon in ancient DNA, especially from specimens of that age," comments Lesley Sitter, a postdoctoral researcher in Bos's team who carried out the analysis. The pathogen is related to all known modern forms of M. lepromatosis, but as there are so few genomes available for comparison, there is still much to be learned about it. This work has shown that a pathogen considered rare in a modern context caused disease for thousands of years in the Americas. Rodrigo Nores, professor of Anthropology at the University of Córdoba, Argentina is convinced that more cases, both ancient and modern, will be identified in the coming years: "this disease was present in Chile as early as 4000 years ago, and now that we know it was there, we can specifically look for it in other contexts." Once more genomes surface, we'll be able to piece out further details of its history and better understand its global distribution today. With such little data, mystery still surrounds its origin. "So far the evidence points in the direction of an American origin, but we'll need more genomes from other time periods and contexts to be sure." Materials provided by Max Planck Institute for Evolutionary Anthropology. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a65210102/tesla-robo-taxi-supervisor/'>Elon Musk Promised Driverless Taxis—Instead, You Get a Guy in the Passenger Seat</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Waymo followed a similar approach, but unlike Tesla, they didn't claim for years that they wouldn't need a safety monitor. Seems like Elon Musk didn't deliver on his promise of delivering unsupervised rides this summer. After doing some digging, we discovered the invitation for those looking to participate in the early stages of Tesla's Robotaxi program. Pretty much everyone is well aware of Musk's over-promise, under-deliver approach by now, and the Robotaxi rollout is another tally in that column. Musk promised as far back as 2019 that Tesla would have one million autonomous taxis on the road by the following year—a prediction that has clearly failed to come true. Volkswagen's ADMT service, Hyundai's Motional fleet, and Amazon's Zoox Fleet  have all launched test fleets, and Waymo even rolled out a fleet in the same city Tesla is trying to enter with a similar chaperone-in-the-passenger-seat precaution. But it is worth noting that Musk has spent years hyping up the fact that his service wouldn't need them. Late last year, Electrek reported that Tesla was assembling a teleoperations team to monitor and remotely control each Robotaxi. Waymo employed a similar tactic when releasing its multiple fleets nationwide, allowing the vehicles to send alerts to the teleoperations team. That meant real humans would get a ping when a vehicle was stuck, and they could take steps to get the vehicle out of whatever bottleneck it found itself in. Tesla has defiantly stuck with their plan to exclusively use cameras to control its vehicles, while Waymo uses cameras that are backed up by radar sensors, LiDAR sensors, and ultrasonic sensors to understand the surroundings their cars drive through. Only time will tell if Tesla can remedy the apparent laundry list of problems it's having getting Full Self-Driving-equipped vehicles to operate without any training wheels. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. AI Is Giving This Car's Engine a Second Life You Can Soon Simply Swap Out Your Car's Battery Hyundai Built a Robot That Will Charge Your EV</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250629033347.htm'>JWST unlocks 10-billion-year mystery of how galaxies shape themselves</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 10:44:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Many galaxies, including our own Milky Way, are characterized by a flat, extended, rotating stellar disk. These distinct components hold fossil records that help astronomers understand how galaxies form stars, build up elements like oxygen and carbon, essential for life, and evolve into their present shapes. It has been impossible with previous telescopes to distinguish the thin edge of a distant galaxy when viewed from the side. Takafumi Tsukui (formerly of the Australian National University and now based at Tohoku University), who led the research team, says that observing distant galaxies is like using a time machine, allowing us to see how galaxies have built their disks over cosmic history. "Thanks to the JWST's sharp vision, we were able to identify thin and thick disks in galaxies beyond our local universe, some going as far back as 10 billion years ago." The study revealed a consistent trend: in the earlier universe, more galaxies appear to have had a single thick disk, while in later epochs, more galaxies showed a two-layered structure with an additional thin disk component. In more massive galaxies, this thin disk appears to have formed earlier. This figure aligns with formation timelines for the Milky Way itself, where stellar ages can be measured. Tsukui emphasizes that the images provided by JWST help answer one of the biggest questions in astronomy: was our galaxy's formation typical or unique? "The JWST images provided a window into galaxies that resemble the Milky Way's early state, bringing us valuable insights from galaxies far away." The team hopes that their study will help bridge studies of nearby galaxies with far away ones and refine our understanding of disk formation. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/home/tools/a65205322/prime-day-generator-deals-2025/'>Don't Get Caught in Dark! Take Advantage of These Prime Day Generator Deals Now</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. It's hard to beat Amazon Prime Day for good deals on tech, and that includes the ever-growing market for portable generators, power stations, solar panels, and even whole-home standby systems. The sale runs from Tuesday, July 8th through July 11th this year so we're still more than a week away, but we've already spotted early discounts as high as 50 percent off. Whether you're looking for a bit of backup power to top off a few phones and laptops, or need to keep your essential appliances on during an emergency, this is a good time to shop. Picking  the right generator will come down to when and where you need it. Others may be in the market for a whole-home option to power their homes during outages caused by storms or excessive heat. Whatever the need, you'll find a good variety of deals this Prime Day, sometimes for several hundred dollars off the list price. It's important to buy your generator before you need it, so this is a good time to invest if getting a generator has been on your agenda. They tend to fly off the shelves during major emergencies and prices will surge. The good news is that this year's Prime Day deals are starting early. There are early access deals from Bluetti, EcoFlow, Anker, and Jackery available right now. Just sign up for a free 30-day trial and cancel it later. We also found that some generator brands have opened up their sale prices to anyone, so you may be able to skip the membership altogether. (Though you won't get the faster free shipping). When shopping for power stations or generators, we often advise that you look for an option with more ports and a higher capacity than what you need. For example, if you see a deal on a power station that offers AC outlets for plugging in small appliances, and you don't think you need that, go for it anyway. Unlike fuel-based generators, these run on lithium-ion or lithium-iron-phosphate (LiFePO4) batteries that can be pre-charged from an outlet. That means you can use them indoors and free yourself from carbon emissions. This is already one of our favorite power stations: We named it "best overall" in our guide to the best solar generators. But like any power station, it's not limited to solar. You can pre-charge it from an outlet and store up to 1,070 watt-hours, enough to charge plenty of devices or power small appliances. When we tested the Elite 200 V2, we were impressed by the fact that it could store so much energy (2,073.6 watt-hours) in such a compact and comparatively light power station. In our review, we found that it was able to power a full-size, 25-cubic-foot refrigerator for a remarkable 51 hours and 24 minutes, making it handy for emergencies. If you only need your power station to keep your digital devices alive in the great outdoors, this tiny portable Anker model is perfect. It charges you up and functions as a lamp, to boot. Remember, folks: You need portable power station and a set of portable solar panels to set up a solar generator. The panels themselves can't store electricity; they can only convert it from sunlight. If charging with solar is your goal: These solar generator bundles give you the complete package, and there are plenty on sale for Prime Day. If you just need a small portable power station for off-grid getaways, this bundle features outputs for 110V AC, USB, and 12-volt. We've been fans of the EcoFlow Delta series for a long time. Some outdoor trips and camping adventures demand more charging power than others. With an output that supports up to 50 amps, you can't go wrong plugging this 4,096-watt-hour station into your RV or a home backup connection. It ships with a 400-watt panel array, but can support up to 2,600 watts. If you plan to run sensitive devices like TVs and computers off your generator, make sure you look for an inverter generator, which delivers more stable power. In our hands-on test of this generator, we found it to be a compact generator that was able to easily run a 15-amp table saw and a 10-amp portable air conditioner at the same time. And its durable construction is tough enough for the rigors of a jobsite. It can run for up to 14 hours on just 2.3 gallons of gas and has a carbon monoxide auto shutoff system. Homes plagued by constant power outages from severe weather may need long-term power solutions for demanding household appliances and utilities, such as well pumps, refrigerators, and water heaters. You can connect them directly to your home's electrical system so they can take over when the lights go out. It has a massive capacity—4,069-watt-hours—and can output 120-volt and 240-volt AC power. That's give you the means to keep serious power-hungry appliances like your central air conditioning system or well water pump up and running through a blackout. In our review, we praised the Solix F2000's design, which includes telescoping handle and wheels so you can move it around to different parts of your house when you need it. And it's powerful enough to keep a space heater or refrigerator running for hours. This is a next-level system that one of our test editors used to install a home backup system of his own. It can work in tandem with an EcoFlow Smart Home Panel 2 (also on sale for Prime Day) and connects to additional batteries so you can raise its power capacity to even greater heights. This home-backup-capable solar system can provide enough power (5,040 watt-hours) to back up several critical circuits in your home, and comes with large, 500-watt foldable solar panel array that will recharge it in 12 to 14 hours. You can put out up to 9,500 running watts of power (12,500 peak watts) from this generator, and it offers output up to 50 amps, connected directly to devices or using a transfer switch installed in your home. A permanent home standby generator, like this one from Generac, is the best way to set up a complete home backup, so everything just works when the power goes out. This model can output up to 10 kilowatts (10,000 watts) of power, and runs on either propane or natural gas. You'll definitely need to hire a professional to install it, but picking it up now will save you a few hundred bucks. David is a Reviews Editor for Hearst's Ehtusiast Group, contributing to Popular Mechanics, Runner's World, Bicycling, Best Products, and Biography. He also spent years covering the ever-changing world of online retail. To relax, he loves experimenting with new recipes, tweaking his smart home setup, and spending time outdoors on his bike or camping. Post-Prime Day Apple Deals Are Still Up to 40% Off Save Up to 56% on Leaf Blowers During Prime Day Generators Are Already Up to 54% Off for Prime Day</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01314-w'>Neuronal glycogen breakdown mitigates tauopathy via pentose-phosphate-pathway-mediated oxidative stress reduction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 09:50:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Tauopathies encompass a range of neurodegenerative disorders, such as Alzheimer's disease (AD) and frontotemporal lobar degeneration with tau inclusions (FTLD-tau), for which there are currently no successful treatments. Here, we show impaired glycogen metabolism in the brain of a tauopathy Drosophila melanogaster model and people with AD, indicating a link between tauopathies and glycogen metabolism. We demonstrate that the breakdown of neuronal glycogen ameliorates the tauopathy phenotypes in flies and induced pluripotent stem cell (iPSC)-derived neurons from people with FTLD-tau. Glycogen breakdown redirects glucose flux to the pentose phosphate pathway and alleviates oxidative stress. Our findings uncover a critical role for the neuroprotective effects of dietary restriction (DR) by increasing glycogen breakdown. Mechanistically, we show a potential interaction between tau protein and glycogen, suggesting a vicious cycle in which tau binding promotes glycogen accumulation in neurons, which in turn exacerbates tau accumulation which further disrupts cellular homeostasis. Our studies identify impaired glycogen metabolism as a key hallmark for tauopathies and offer a promising therapeutic target in tauopathy and other neurodegenerative diseases. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Raw data for complete MS data sets for proteomic analysis have been uploaded to the MassIVE repository of the Center for Computational Mass Spectrometry at UCSD and can be downloaded using the following link: https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=93387e0de9b7465ab0466d3bd4bb901f The data are also available at ProteomeXchange with the ID PXD044485. Raw data from RNA sequencing and list of targeted metabolites and their values are available in Figshare using the following link https://figshare.com/s/ad5c367a220c6cca66f8. Source data are provided with this paper. The code for Cox proportional hazard ratio calculation is available in Figshare using the following link: https://figshare.com/s/ad5c367a220c6cca66f8 (ref. Goedert, M., Eisenberg, D. S. & Crowther, R. A. Propagation of tau aggregates and neurodegeneration. Google Scholar Google Scholar Brunden, K. R., Trojanowski, J. Q. & Lee, V. M. Y. Advances in tau-focused drug discovery for Alzheimer's disease and related tauopathies. Drug Discov. Google Scholar Costantini, L. C., Barr, L. J., Vogel, J. L. & Henderson, S. T. Hypometabolism as a therapeutic target in Alzheimer's disease. BMC Neurosci. Google Scholar Johnson, E. C. B. et al. Large-scale proteomic analysis of alzheimer's disease brain and cerebrospinal fluid reveals early changes in energy metabolism associated with microglia and astrocyte activation. Google Scholar Sattler, R. et al. Frontotemporal dementia and glucose metabolism. Blin, J., Horwitz, B., Baron, J. C. & Agid, Y. Does frontal cortex hypometabolism in progressive supranuclear palsy result from subcortical dysfunction? Google Scholar Minhas, P. S. et al. Restoring hippocampal glucose metabolism rescues cognition across Alzheimer's disease pathologies. Google Scholar Gibbs, M. E., Anderson, D. G. & Hertz, L. Inhibition of glycogenolysis in astrocytes interrupts memory consolidation in young chickens. Google Scholar Mann, D. M. A., Sumpter, P. Q., Davies, C. A. & Yates, P. O. Glycogen accumulations in the cerebral cortex in Alzheimer's disease. Google Scholar Duran, J., Gruart, A., García-Rocha, M., Delgado-García, J. M. & Guinovart, J. J. Glycogen accumulation underlies neurodegeneration and autophagy impairment in Lafora disease. Google Scholar Li, C. et al. Decreased glycogenolysis by miR-338-3p promotes regional glycogen accumulation within the spinal cord of amyotrophic lateral sclerosis mice. Google Scholar Cai, Y. et al. Glycogenolysis is crucial for astrocytic glycogen accumulation and brain damage after reperfusion in ischemic stroke. Google Scholar Adeva-Andany, M. M., González-Lucán, M., Donapetry-García, C., Fernández-Fernández, C. & Ameneiros-Rodríguez, E. Glycogen metabolism in humans. BBA Clin. Brown, A. M. & Ransom, B. R. Astrocyte glycogen and brain energy metabolism. Oe, Y., Baba, O., Ashida, H., Nakamura, K. C. & Hirase, H. Glycogen distribution in the microwave-fixed mouse brain reveals heterogeneous astrocytic patterns. Fontana, L., Partridge, L. & Longo, V. D. Extending healthy life span—from yeast to humans. Mair, W. & Dillin, A. Aging and survival: the genetics of life span extension by dietary restriction. & Austad, S. N. The evolution of the antiaging action of dietary restriction: a hypothesis. Solon-Biet, S. M. et al. The ratio of macronutrients, not caloric intake, dictates cardiometabolic health, aging, and longevity in ad libitum-fed mice. Mattson, M. P., Duan, W., Lee, J. & Guo, Z. Suppression of brain aging and neurodegenerative disorders by dietary restriction and environmental enrichment: molecular mechanisms. Graff, J. et al. A dietary regimen of caloric restriction or pharmacological activation of SIRT1 to delay the onset of neurodegeneration. Kerr, F. et al. Dietary restriction delays aging, but not neuronal dysfunction, in Drosophila models of Alzheimer's disease. Van Cauwenberghe, C., Vandendriessche, C., Libert, C. & Vandenbroucke, R. E. Caloric restriction: beneficial effects on brain aging and Alzheimer's disease. Green, C. L., Lamming, D. W. & Fontana, L. Molecular mechanisms of dietary restriction promoting health and longevity. Wittmann, C. W. et al. Tauopathy in Drosophila: neurodegeneration without neurofibrillary tangles. Luis, N. M. et al. Intestinal IRE1 is required for increased triglyceride metabolism and longer lifespan under dietary restriction. Akagi, K. et al. Dietary restriction improves intestinal cellular fitness to enhance gut barrier function and lifespan in D. melanogaster. Chatterjee, S., Sang, T. K., Lawless, G. M. & Jackson, G. R. Dissociation of tau toxicity and phosphorylation: role of GSK-3β, MARK and Cdk5 in a Drosophila model. Gendron, C. M., Chakraborty, T. S., Duran, C., Dono, T. & Pletcher, S. D. Ring neurons in the Drosophila central complex act as a rheostat for sensory modulation of aging. Gillet, L. C. et al. Targeted data extraction of the MS/MS spectra generated by data-independent acquisition: a new concept for consistent and accurate proteome analysis. Mol Cell Proteomics 11, O111.016717 (2012). Collins, B. C. et al. Multi-laboratory assessment of reproducibility, qualitative and quantitative performance of SWATH-mass spectrometry. Jackson, G. R. et al. Human wild-type tau interacts with wingless pathway components and produces neurofibrillary pathology in Drosophila. Oliveira, A. C., Santos, M., Pinho, M. & Lopes, C. S. String/Cdc25 phosphatase is a suppressor of Tau-associated neurodegeneration. Zelhof, A. C. et al. The brachyceran de novo gene PIP82, a phosphorylation target of aPKC, is essential for proper formation and maintenance of the rhabdomeric photoreceptor apical domain in Drosophila. Koushika, S. P., Lisbin, M. J. & White, K. ELAV, a Drosophila neuron-specific protein, mediates the generation of an alternatively spliced neural protein isoform. Zirin, J., Nieuwenhuis, J. & Perrimon, N. Role of autophagy in glycogen breakdown and its relevance to chloroquine myopathy. Zhang, Y. et al. Decoding the molecular mechanism of selective autophagy of glycogen mediated by autophagy receptor STBD1. Kanungo, S., Wells, K., Tribett, T. & El-Gharbawy, A. Glycogen metabolism and glycogen storage disorders. Fernie, A. R., Carrari, F. & Sweetlove, L. J. Respiratory metabolism: glycolysis, the TCA cycle and mitochondrial electron transport. Plant Biol. Ge, T. et al. The role of the pentose phosphate pathway in diabetes and cancer. Dinuzzo, M., Mangia, S., Maraviglia, B. & Giove, F. Regulatory mechanisms for glycogenolysis and K+ uptake in brain astrocytes. Jiang, G. & Zhang, B. B. Glucagon and regulation of glucose metabolism. Petersen, M. C., Vatner, D. F. & Shulman, G. I. Regulation of hepatic glucose metabolism in health and disease. Hao, Y. et al. Activation of PKA via asymmetric allosteric coupling of structurally conserved cyclic nucleotide binding domains. Yamada, T. et al. The role of glycogen in development and adult fitness in Drosophila. Karch, C. M. et al. Resource a comprehensive resource for induced pluripotent stem cells from patients with primary tauopathies. Stem Cell Rep. 13, 939–955 (2019). Wang, C. et al. Scalable production of iPSC-derived human neurons to identify tau-lowering compounds by high-content screening. Stem Cell Rep. 9, 1221–1233 (2017). Zhu, Y. et al. Single-cell analysis for glycogen localization and metabolism in cultured astrocytes. Marr, L. et al. Mechanism of glycogen synthase inactivation and interaction with glycogenin. Nakamura, M. et al. Pathological progression induced by the frontotemporal dementia-associated R406W tau mutation in patient-derived iPSCs. Stem Cell Rep. 13, 684–699 (2019). Tracy, T. E. et al. Tau interactome maps synaptic and mitochondrial processes associated with neurodegeneration. Groen, A. C., Coughlin, M. & Mitchison, T. J. Microtubule assembly in meiotic extract requires glycogen. Puri, R., Jain, N. & Ganesh, S. Increased glucose concentration results in reduced proteasomal activity and the formation of glycogen positive aggresomal structures. Wu, J. et al. A conserved domain for glycogen binding in protein phosphatase-1 targeting subunits. Bhat, N. R. & Thirumangalakudi, L. Increased tau phosphorylation and impaired brain insulin/IGF signaling in mice fed a high fat/high cholesterol diet. Faraco, G. et al. Dietary salt promotes cognitive impairment through tau phosphorylation. Liang, Z. et al. Long-term high-fat diet consumption induces cognitive decline accompanied by tau hyper-phosphorylation and microglial activation in aging. Wilson, K. A. et al. GWAS for lifespan and decline in climbing ability in flies upon dietary restriction reveal decima as a mediator of insulin-like peptide production. Katewa, S. D. et al. Intramyocellular fatty-acid metabolism plays a critical role in mediating responses to dietary restriction in Drosophila melanogaster. A. et al. Dietary restriction and the transcription factor clock delay eye aging to extend lifespan in Drosophila melanogaster. Pandey, M. et al. Mir-125-chinmo pathway regulates dietary restriction-dependent enhancement of lifespan in Drosophila. Wilson, K. A. et al. Evaluating the beneficial effects of dietary restrictions: a framework for precision nutrigeroscience. Perlegos, A. E. et al. TDP-43 impairs sleep in Drosophila through Ataxin-2–dependent metabolic disturbance. Wander, C. M. et al. Corpora amylacea are associated with tau burden and cognitive status in Alzheimer's disease. Dallmeier, J. D. et al. Corpora amylacea negatively correlate with hippocampal tau pathology in Alzheimer's disease. Puri, R., Suzuki, T., Yamakawa, K. & Ganesh, S. Hyperphosphorylation and aggregation of tau in laforin-deficient mice, an animal model for Lafora disease. Machado-Salas, J. et al. Ontogeny of lafora bodies and neurocytoskeleton changes in laforin-deficient mice. Favaro, E. et al. Glucose utilization via glycogen phosphorylase sustains proliferation and prevents premature senescence in cancer cells. Zhang, H. et al. TCR activation directly stimulates PYGB-dependent glycogenolysis to fuel the early recall response in CD8+ memory T cells. Dias-Santagata, D., Fulga, T. A., Duttaroy, A. & Feany, M. B. Oxidative stress mediates tau-induced neurodegeneration in Drosophila. Ton, A. M. M. et al. Oxidative stress and dementia in Alzheimer's patients: effects of synbiotic supplementation. Sun, R. C. et al. Brain glycogen serves as a critical glucosamine cache required for protein glycosylation. Weimer, S. et al. d-Glucosamine supplementation extends life span of nematodes and of ageing mice. & Walsh, D. A. Phosphorylase kinase: the complexity of its regulation is reflected in the complexity of its structure. Zhuo, Y. et al. Inhibition of phosphodiesterase-4 reverses the cognitive dysfunction and oxidative stress induced by Aβ25-35 in rats. Giblin, A. et al. Neuronal polyunsaturated fatty acids are protective in ALS/FTD. Ju, Y., Li, S., Kong, X. Exploring fatty acid metabolism in Alzheimer's disease: the key role of CPT1A. Cook, K. R., Parks, A. L., Jacobus, L. M., Kaufman, T. C. & Matthews, K. A. New research resources at the Bloomington Drosophila Stock Center. Zid, B. M. et al. 4E-BP extends lifespan upon dietary restriction by enhancing mitochondrial activity in Drosophila. Katewa, S. D. et al. Peripheral circadian clocks mediate dietary restriction-dependent changes in lifespan and fat metabolism in Drosophila. Iyer, J. et al. Quantitative assessment of eye phenotypes for functional genetic studies using Drosophila melanogaster. Sohn, P. D. et al. Pathogenic tau impairs axon initial segment plasticity and excitability homeostasis. Meyer, J. G. & Schilling, B. Clinical applications of quantitative proteomics using targeted and untargeted data-independent acquisition techniques. Schilling, B., Gibson, B. W. & Hunter, C. L. Generation of high-quality SWATH® acquisition data for label-free quantitative proteomics studies using TripleTOF® mass spectrometers. Kurup, K. et al. Calorie restriction prevents age-related changes in the intestinal microbiota. Meador, J. P., Bettcher, L. F., Ellenberger, M. C. & Senn, T. D. Metabolomic profiling for juvenile Chinook salmon exposed to contaminants of emerging concern. Xia, J., Psychogios, N., Young, N. & Wishart, D. S. MetaboAnalyst: a web server for metabolomic data analysis and interpretation. Nucleic Acids Res. Babicki, S. et al. Heatmapper: web-enabled heat mapping for all. Nucleic Acids Res. Bar, S. et al. Neuronal glycogen breakdown mitigates tauopathy via pentose-phosphate-pathway-mediated oxidative stress reduction. figshare https://figshare.com/s/ad5c367a220c6cca66f8 (2025). We would like to thank the members of the Kapahi, Ellerby, Schilling and Tracy labs for their comments throughout the paper and experiments. is supported by Larry L. Hillblom Foundation fellowship 2019-A-026-FEL. K. A. W. was supported by NIH grant T32AG000266-23 and the CatalystX award from Alex and Bob Griswold. was supported by NIH and NIA award F31AG062112, NIH/NIA training grant T32AG000266-24. This work was supported by the NIH and American Federation of Aging Research grants, NIH grants R01AG038688, R21AG054121, AG045835 and RO1-R01AG071995 and the Larry L. Hillblom Foundation to P.K. is supported by NIH grant R01AG070193, L.M.E. is supported by R01AG061879 and P01AG066591. We acknowledge the support of instrumentation for the TripleTOF 6600 from the NIH shared instrumentation grant 1S10 OD016281 (Buck Institute) and a Hevolution Foundation grant to the Buck Institute for Research on Aging (HF-PART-23-1422047). We thank the lab of S. Pletcher, University of Michigan Medical School, for sharing 3XEGS. We thank the Northwest Metabolomics Research Center, University of Washington, for the metabolomics analysis. We thank C. Karch for providing iPSC and isogenic control lines. Support to generate the iPSC was provided by Knight Alzheimer Disease Research Center at Washington University (NIH P30 AG066444, P01 AG03991, and P01 AG026276), NIH AG046374 (C.M.K. ), and the Rainwater Charitable Organization. Schematic diagrams were generated with BioRender.com (https://biorender.com/582r1x5, https://biorender.com/xn3s24o, https://biorender.com/f73h247). We thank M. B. Feany and Scott D. Pletcher for fly strains. personally acknowledges G. T. Meyerhof, M. Dhara, P. Singh, V. Tanwar, L. Gann, W. Gutierrez, M. Gupta, V. P. Narayan, K. R. Kaneshiro, M. M. Shanmugam, G. Chawla, D. Sellegounder, K. Patel, L. Enriquez, E. Morazan, I. Emma and A. K. Cayton for their support. Buck Institute for Research on Aging, Novato, CA, USA Sudipta Bar, Kenneth A. Wilson, Tyler A. U. Hilsabeck, Sydney Alderfer, Jordan B. Burton, Samah Shah, Anja Holtz, Enrique M. Carrera, Jennifer N. Beck, Jackson H. Chen, Grant Kauwe, Tara E. Tracy, Birgit Schilling, Lisa M. Ellerby & Pankaj Kapahi Emory Center for Neurodegenerative Disease, Emory University School of Medicine, Atlanta, GA, USA Eric B. Dammer, Fatemeh Seifar & Nicholas T. Seyfried Emory University, School of Medicine Core Labs, Atlanta, GA, USA Department of Biochemistry, Emory University School of Medicine, Atlanta, GA, USA Fatemeh Seifar, Ananth Shantaraman & Nicholas T. Seyfried Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Pankaj Kapahi. is a founder and a member of the scientific advisory board at Juvify Bio. The other authors have no competing interests. Nature Metabolism thanks Thomas Preat and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Alfredo Giménez-Cassina, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Lifespan of control flies (W1118 crossed with tauR406W) shows DR-mediated extension (p = 2.8164e-99) (204 and 192 flies were studied for AL and DR diets respectively). b, Lifespan of control flies (W1118 crossed with tauWT) shows DR-mediated extension (p = 9.80751e-51). For lifespan 203 and 151 flies were studied for AL and DR diets respectively. c, Mean lifespan of controls, tauR406W, and tauWT of 3 independent experiments show lifespan extension by DR. TauWT and tauR406W flies have the shortest lifespan (**p < 0.0001) than controls (n > 100 for each group). d, Lifespan of tauR406W flies shorter on both AL (p = 6.48275e-38) and DR (p = 6.03135e-57) diets than control flies (elav-Gal4 crossed with mito-GFP). For lifespan 156, 171 172 and 169 flies were studied for tauR406W AL, tauR406W DR, mito-GFP AL and mito-GFP DR were studied respectively. e, Lifespan of tauS11A is shorter (p = 1.64475e-19) than control flies (elav-Gal4 crossed with W1118) and shows DR-mediated extension (p = 3.40185e-24). For lifespan 138, 136, 96 and 85 flies were studied for control AL, control DR, tauS11A AL and tauS11A DR respectively. f, bar plot represents lethality during developmental phase in tauR406W flies and control elav-gal4 crossed with W1118. Each dot represents separate vials of embryo generated from 2:1 female: male cross. g, Lifespan of tauR406W driven by 3X Elav-Gene switch induced by RU486 is reduced (p = 9.09456e-09) than control and shows DR mediated extension (p = 4.02488e-28). For lifespan 188, 166, 185 and 168 flies were studied for the group of AL RU-, AL RU + , DR RU- and DR RU+ respectively. ‘RU-‘ and ‘RU + ‘represent vehicles control or RU486 added in food to drive the construct. h, Images of the whole-mount brain with DCFDA staining of control and tauR406W on AL and DR. i, Quantification shows increased (p < 0.0001) ROS stained by DCFDA in tauR406W flies' brain which is rescued (p < 0.002) by DR. For ROS staining 11, 10, 14 and 12 fly brains were studied for AL RU-, AL RU + DR RU- and DR RU+ groups respectively. An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks p < 0.05 for *, p < 0.01 for **, p < 0.001 for *** and p < 0.0001 for **** by log-rank test (a, b, d, e and g) or by oneway ANOVA with Tukey's multiple comparison test (c and i) or by two-tailed Student's t-test (f). Data in bar graphs are presented as mean ± s.e.m. a, Western blot using anti tau antibody in both control and tauR406W flies in AL and DR diets (n = 3 biological replicates each with 25 fly head). b, Densiometric analysis of western blot (n = 3 biological replicates each with 25 fly head). c, Table shows the result of Cox Proportional hazard ratio test. The coefficient (coef) is that term's beta value, with values > 0 meaning that term increases the hazard (risk of death), and values of <0 decreasing the hazard (risk of death). The term exp(coef) is that term's hazard ratio (HR), with the following breakdown: HR = 1: No effect, HR < 1: Reduction in the hazard, HR > 1: Increase in Hazard. The term se(coef) is the standard error for that term. The z term gives the Wald statistic value. It corresponds to the ratio of each regression coefficient to its standard error (z = coef/se(coef)). The Wald statistics evaluate whether the coefficient is statistically different from 0. The final term, Pr(>|z | ), is that row's p-value. The rest of the table gives the confidence intervals for each variable. d, Apoptotic cell death detected by TUNEL staining in tauR406W fly whole brain shows increased TUNEL staining in tauR406W (at least 33 fly brain were studied per group). An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks p < 0.05 for *, p < 0.01 for **, p < 0.001 for *** and p < 0.0001 for **** by oneway ANOVA with Tukey's multiple comparison test (b). Data in bar graphs are presented as mean ± s.e.m. a, Venn diagram of the number of proteins upregulated in tauwt (red circle) as well as in control on AL diets (blue circle) (p < 0.0001). Dots represent enriched pathways of overlapping proteins, including fatty acid and glycogen metabolism. b, Venn diagram shows numbers of proteins downregulated in tauwt (red circle) and control on AL diets (blue circle) (p < 0.0001). Dots represent enriched pathways of overlapping proteins, including oxidative phosphorylation and glutathione metabolism. c, Venn diagram shows common upregulated protein between tauwt(red) and tauR406W(blue) (p = 0.001). d, Venn diagram shows common downregulated protein between tauwt(red) and tauR406W(blue) (p < 0.0001). e, Relative mRNA expression of GlyP in tauR406W flies and controls under AL and DR diets shows upregulation of GlyP in tauR406W on DR diets compared to control AL (p = 0.0262) and control DR (p = 0.0199) (n = 6 groups, each with 25 fly heads). f, Relative mRNA expression of AGBE in tauR406W flies and controls under AL and DR diets shows upregulation of AGBE in tauR406W on DR diets compared to control DR (p < 0.0001) or tauR406W AL (p = 0.0001) (n = 6 groups, each with 25 fly heads). g, Relative mRNA expression of GlyS in tauR406W flies and controls under AL and DR diets shows upregulation (*p = 0.111 and **p = 0.0011) of GlyS in tauR406W on DR diet (n = 6 groups each with 25 fly heads). h, Relative mRNA expression of pgm in tauR406W flies and control under AL and DR diets shows upregulation (p < 0.0001) of pgm in tauR406W on AL diet (n = 6 groups each with 25 fly head). i, Relative mRNA expression of AGBE, GlyP and pgm in RNAi flies and GlyP overexpressing flies (***p < 0.0001, **p = 0.0029, ***p < 0.0001 and **p = 0.0066 respectively). The experiments were performed using 3-4 groups each with 25 fly heads. j, Western blot shows tau expression in GlyPS15A; tauR406W, and GlyPWT (n = 25 fly heads for each tissue lysate). k, Densitometric analysis of tau expression is not changed between GlyPS15A; tauR406W and GlyPWT; tauR406W. l, GlyP enzyme activity between GlyPS15A; tauR406W (kinase dead mutant control) control and GlyPWT; tauR406W shows significant (p = 0.0002) increased activity in GlyPWT (data were collected using 320 fly heads in 4 biological replicates and two technical replicates for each group). m, Lifespan of GlyPWT; tauR406W (n = 93 flies) and GlyPS15A; tauR406W (n = 95 flies) shows no significant difference. n, Quantification of glycogen shows significant (p = 0.0058) accumulation of glycogen in the head of elav-Gal4; tauWT in compared to the control elav-Gal4xW1118 (n = 4 groups, 25 fly head in each). o, Western blot shows expression of autophagic marker Atg8(LC3) and Ref(2)P/(p62) in GlyPS15A; tauR406W, and GlyPWT (n = 25 fly heads for each tissue lysate). Samples were derive from the same experiment and that blots were processed in parallel. p, Densiometric analysis of ATG8 expression in GlyPS15A; tauR406W, and GlyPWT (n = 3) shows reduced (p = 0.0108) expression of ATG8. q, Densiometric analysis of Ref(2)P expression in GlyPS15A; tauR406W, and GlyPWT (n = 3). An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks p < 0.05 for *, p < 0.01 for **, p < 0.001 for *** and p < 0.0001 for **** by Fisher's exact (a-d) by one-way ANOVA with Tukey's multiple comparison test(e-h) by two tailed Student's t-test (i, k, l, n, p and q). Data in bar graphs are presented as mean ± s.e.m. a, Pathway analysis of metabolites shows amino acids metabolism, urea cycle, and PPP are the few most common pathways. in GlyPWT; tauR406W compared to GlyPS15A; tauR406W (3 replicates, each with 30 fly heads, were used). b, Quantification of reduced and oxidized glutathione ratio shows significantly (p = 0.0383) increased reduced glutathione in GlyPWT; tauR406W than GlyPS15A; tauR406W (175 flies head for 7 biological replicates were used). c, Volcano plot shows 546 transcripts (red) are upregulated and 473 transcripts (green) are downregulated (3 replicates each with 30 fly heads were used to isolate RNA). d, Pathway analysis of altered RNA shows oxidative phosphorylation, glycolysis or gluconeogenesis, ECM- receptor interaction and TCA cycle are the most common. e, TUNEL staining of GlyPWT; tauR406W with 6-AN treatment shows GlyPWT mediated reversal of apoptotic death of brains abrogate by 6-AN treatment (at least 21 fly brain were analyzed per group). An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks* for p < 0.05 by two-tailed Student's t-test (b). Data in bar graphs are presented as mean ± s.e.m. a, Western blot of PKA-C1 and actin of brain tissue lysate of control and tauR406W on AL and DR (n = 3 each tissue lysate with 25 fly heads, experiment was repeated three times independently and observed similar results). b, Normalized densitometric analysis of western blot shows decreased (p = 0.0022) abundance of PKA-C1 of tauR406W on AL. c, Survival curve shows lifespan extension (p = 3.73621e-07) of tauR406W on AL diet with 8-Br-cAMP treatment. For lifespan assay 92 to 97 flies were studied. d, Lifespan of tauR406W on DR diet with 8-Br-cAMP shows no further DR-mediated extension. For lifespan assay 92 to 97 flies were studied. e, TUNEL staining shows cAMP treatment rescues apoptotic cells in tauR406W fly brain (at least 25 fly brain were stained for TUNEL). f, Images of the whole-mount brain with DCFDA staining of control and tauR406W on AL and DR. g, Quantification shows increased (p < 0.0001) ROS stained by DCFDA in tauR406W flies' brain which is rescued (p < 0.0001) by DR. For ROS staining 23, 24, 25 and 25 fly brains were used for control AL, control DR, tauR406W AL and tauR406W DR. h, GlyP activity in GlyP mutant flies shows reduction compared to wild type (data were collected using 320 fly heads in 4 biological replicates). An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks p < 0.05 for *, p < 0.01 for **, p < 0.001 for *** and p < 0.0001 for **** by log-rank test (c, d) or by oneway ANOVA with Tukey's multiple comparison test (b, g), or by two-tailed Student's t-test (h). Data in bar graphs are presented as mean ± s.e.m. a, Plot showing correlation between protein abundance of tau (MTBR) and PYGB (n = 980). b, Plot showing correlation between protein abundance of tau (MTBR) and PGM (n = 980) c, Images show glycogen staining in iPSC-derived tauR406W and isogenic control (Iso-tauR406R) neurons by 2-NBDG transduced with GlyP lentivirus. d, Quantification shows glycogen staining increase (p < 0.0001) in control virus-infected tauR406W than isogenic control infected with control lentivirus, and the glycogen staining is reduced (p < 0.0001) in tauR406W cells with PYGB overexpression. e, TOM20 staining of iPSC-generated neurons shows reduced (p < 0.0001) mitochondrial density in tauR406W than isogenic control cells. f, Quantification of TOM20 staining shows reduced (p < 0.0001) mitochondrial density in tauR406W than isogenic control cells. g, TOM20 staining in PYGB overexpressing tauR406W cells show reduced mitochondrial staining in control lentivirus transduction while mitochondrial density is rescued by PYGB containing lentivirus. h, Glycogen staining with 2-NBDG in iPSC-generated tauV337M neurons and its isogenic control. i, Quantification shows increased (p < 0.0001) glycogen staining in tauV337M cells. j, Quantification of glycogen in cell lysate of tauV337M shows significantly increased (p = 0.0008) quantity than isogenic control. k, Quantification of mean fluorescence intensity of glycogenin significantly increases (p = 0.0018) in tauV337M compared to isogenic control. l, Glycogen staining in tauV337M cells and its isogenic control. m, Quantification shows significant increase (p = 0.0007) of glycogen staining in tauV337M neurons while PYGB overexpression reduces (p < 0.0001) glycogen staining in tauV337M cells compared to tauV337M treated with control lentivirus. Each dot represents an image field from n = 3 coverslips per condition for d, f, i, k and m. An asterisk (*) indicates a significant difference between experimental groups and controls, with the level of significance denoted by the number of asterisks p < 0.05 for *, p < 0.01 for **, p < 0.001 for *** and p < 0.0001 for **** by two tailed Student's t-Test (f, i, j and k) or by two way ANOVA (d and m). Data in bar graphs are presented as mean ± s.e.m. a, c, Co-localization study between tau and glycogenin 1 and across the line intensity plot for tauwt neurons. b, d, Co-localization study between tau and glycogenin 1 and across the line intensity plot for tauR406W neurons. e, Western blot of elute sediment from glycogen co-sedimentation assay with either tauWT or tauP301S using anti tau antibody showed tau sedimented with glycogen f, Western blot of elute sediment from glycogen co-sedimentation assay with BSA showed BSA did not sediment with glycogen. Co-sedimentation assay was repeated 3 times with similar results. Proteomic analysis of tauR406W and tauWT on both AL and DR diets, along with a list of significantly altered proteins and GO term analysis. Metabolomic analysis of GlyPS15A;tauR406W vs GlyPWT; tauR406W fly heads. Transcriptomics analysis of GlyPS15A;tauR406W vs GlyPWT; tauR406W fly heads. Human orthologue of common proteins altered in tau fly and people with AD. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Statistical source data for Extended Fig. Uncropped western blots of all western blot figure. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Bar, S., Wilson, K.A., Hilsabeck, T.A.U. et al. Neuronal glycogen breakdown mitigates tauopathy via pentose-phosphate-pathway-mediated oxidative stress reduction. Nat Metab  (2025). Download citation Published: 30 June 2025 DOI: https://doi.org/10.1038/s42255-025-01314-w Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Metab) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02041-4'>Intraspecies warfare restricts strain coexistence in human skin microbiomes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 09:28:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Microbiology (2025)Cite this article Determining why only a fraction of encountered or applied strains engraft in a given person's microbiome is crucial for understanding and engineering these communities. Previous work has established that metabolic competition between bacteria can restrict colonization success in vivo, but other mechanisms may also prevent successful engraftment. Here we combine genomic analysis and high-throughput agar competition assays to demonstrate that intraspecies warfare presents a significant barrier to strain coexistence in the human skin microbiome by profiling 14,884 pairwise interactions between Staphylococcus epidermidis isolates cultured from 18 people from 6 families. We find that intraspecies antagonisms are abundant, mechanistically diverse, independent of strain relatedness and consistent with rapid evolution via horizontal gene transfer. Critically, these antagonisms are significantly depleted among strains residing on the same person relative to random assemblages, indicating a significant in vivo role. Wide variation in antimicrobial production and resistance suggests trade-offs between these factors and other fitness determinants. Together, our results emphasize that accounting for intraspecies warfare may be essential to the design of long-lasting probiotic therapeutics. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All sequencing data generated in previous work6 are available on the NCBI Sequence Read Archive under Bioproject PRJNA1052084. All sequencing data generated in this work are available under Bioproject PRJNA1215987. All images from the interaction screen are available on Figshare at https://doi.org/10.6084/m9.figshare.25726482 (ref. All other data are included in the manuscript, supplementary tables, or code base (‘Code availability'). Source data are provided with this paper. All code needed to perform the bioinformatic pipeline, image analysis pipeline, and reproduce the figures and analysis in this manuscript is available on GitHub at https://github.com/cpmancuso/Sepidermidis-antagonism (ref. Garud, N. R., Good, B. H., Hallatschek, O. & Pollard, K. S. Evolutionary dynamics of bacteria in the gut microbiome within and across hosts. Google Scholar Ianiro, G. et al. Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases. Google Scholar Valles-Colomer, M. et al. The person-to-person transmission landscape of the gut and oral microbiomes. France, M., Ma, B. & Ravel, J. Persistence and in vivo evolution of vaginal bacterial strains over a multiyear time period. Oh, J., Byrd, A. L., Park, M., Kong, H. H. & Segre, J. A. Temporal stability of the human skin microbiome. Baker, J. S. et al. Intraspecies dynamics underlie the apparent stability of two important skin microbiome species. Cell Host Microbe https://doi.org/10.1016/j.chom.2025.04.010 (2025). Dill-McFarland, K. A. et al. Close social relationships correlate with human gut microbiota composition. Koo, H., Hakim, J. A., Crossman, D. K., Lefkowitz, E. J. & Morrow, C. D. Sharing of gut microbial strains between selected individual sets of twins cohabitating for decades. Sprockett, D., Fukami, T. & Relman, D. A. Role of priority effects in the early-life assembly of the gut microbiota. Debray, R. et al. Priority effects in microbiome assembly. Levy, R. & Borenstein, E. Metabolic modeling of species interaction in the human microbiome elucidates community-level assembly rules. Caballero-Flores, G., Pickard, J. M., Fukuda, S., Inohara, N. & Núñez, G. An enteric pathogen subverts colonization resistance by evading competition for amino acids in the gut. Cell Host Microbe https://doi.org/10.1016/j.chom.2020.06.018 (2020). Watson, A. R. et al. Metabolic independence drives gut microbial colonization and resilience in health and disease. Swaney, M. H., Nelsen, A., Sandstrom, S. & Kalan, L. R. Sweat and sebum preferences of the human skin microbiota. Joglekar, P. et al. Integrated genomic and functional analyses of human skin–associated Staphylococcus reveal extensive inter- and intra-species diversity. Tropini, C., Earle, K. A., Huang, K. C. & Sonnenburg, J. L. The gut microbiome: connecting spatial organization to function. Cell Host Microbe 21, 433–442 (2017). Conwill, A. et al. Anatomy promotes neutral coexistence of strains in the human skin microbiome. Cell Host Microbe 30, 171–182.e7 (2022). Zeng, M. Y. et al. Gut microbiota-induced immunoglobulin G controls systemic infection by symbiotic bacteria and pathogens. Barroso-Batista, J., Demengeot, J. & Gordo, I. Adaptive immunity increases the pace and predictability of evolutionary change in commensal gut bacteria. Kitamoto, S. et al. Dietary l-serine confers a competitive fitness advantage to Enterobacteriaceae in the inflamed gut. Bouslimani, A. et al. The impact of skin care products on skin chemistry and microbiome dynamics. Granato, E. T., Meiller-Legrand, T. A. & Foster, K. R. The evolution and ecology of bacterial warfare. & Gordon, D. M. The ecological role of bacteriocins in bacterial competition. Trends Microbiol. Vetsigian, K., Jajoo, R. & Kishony, R. Structure and evolution of Streptomyces interaction networks in soil and in silico. Verster, A. J. et al. The landscape of type VI secretion across human gut microbiomes reveals its role in community composition. Cell Host Microbe 22, 411–419.e4 (2017). & Griffin, A. S. Bacteriocins and the assembly of natural Pseudomonas fluorescens populations. Detection of bacteriocins produced by lactic acid bacteria. Cordero, O. X. et al. Ecological populations of bacteria act as socially cohesive units of antibiotic production and resistance. Nakatsuji, T. et al. Antimicrobials from human skin commensal bacteria protect against Staphylococcus aureus and are deficient in atopic dermatitis. Janek, D., Zipperer, A., Kulik, A., Krismer, B. High frequency and diversity of antimicrobial activities produced by nasal Staphylococcus strains against bacterial competitors. PLoS Pathog. Peterson, S. B., Bertolli, S. K. & Mougous, J. D. Interbacterial antagonism: at the center of bacterial life. Coyte, K. Z. et al. Horizontal gene transfer and ecological interactions jointly control microbiome stability. Ross, B. D. et al. Human gut bacteria contain acquired interbacterial defence systems. Russel, J., Røder, H. L., Madsen, J. S., Burmølle, M. & Sørensen, S. J. Antagonism correlates with metabolic similarity in diverse bacteria. Kim, S. G. et al. Microbiota-derived lantibiotic restores resistance against vancomycin-resistant Enterococcus. Chesson, P. Mechanisms of maintenance of species diversity. Christensen, G. J. M. et al. Antagonism between Staphylococcus epidermidis and Propionibacterium acnes and its genomic basis. Kellner, R. et al. Gallidermin: a new lanthionine-containing polypeptide antibiotic. Heidrich, C. et al. Isolation, characterization, and heterologous expression of the novel lantibiotic epicidin 280 and analysis of its biosynthetic gene cluster. Zipperer, A. et al. Human commensals producing a novel antibiotic impair pathogen colonization. Libberton, B., Coates, R. E., Brockhurst, M. A. Evidence that intraspecific trait variation among nasal bacteria shapes the distribution of Staphylococcus aureus. Libberton, B., Horsburgh, M. J. The effects of spatial structure, frequency dependence and resistance evolution on the dynamics of toxin-mediated microbial invasions. Charron-Lamoureux, V. et al. Pulcherriminic acid modulates iron availability and protects against oxidative stress during microbial interactions. Costa, F. G., Mills, K. B., Crosby, H. A. & Horswill, A. R. The Staphylococcus aureus regulatory program in a human skin-like environment. Article PubMed Chia, M. et al. Skin metatranscriptomics reveals landscape of variation in microbial activity and gene expression across the human body. Preprint at bioRxiv https://doi.org/10.1101/2024.12.02.626500 (2024). Zhou, W. et al. Host-specific evolutionary and transmission dynamics shape the functional diversification of Staphylococcus epidermidis in human skin. Ebner, P. et al. Lantibiotic production is a burden for the producing staphylococci. Liu, Q. et al. Crosstalk between skin microbiota and immune system in health and disease. Kerr, B., Riley, M. A., Feldman, M. W. & Bohannan, B. J. M. Local dispersal promotes biodiversity in a real-life game of rock–paper–scissors. Costa, S. K., Cho, J. & Cheung, A. L. GraS sensory activity in Staphylococcus epidermidis is modulated by the “guard loop” of VraG and the ATPase activity of VraF. Peschel, A. et al. Staphylococcus aureus resistance to human defensins and evasion of neutrophil killing via the novel virulence factor Mprf is based on modification of membrane lipids with l-lysine. Peschel, A. et al. Inactivation of the dlt operon in Staphylococcus aureus confers sensitivity to defensins, protegrins, and other antimicrobial peptides. Beck, C. et al. Wall teichoic acid substitution with glucose governs phage susceptibility of Staphylococcus epidermidis. Beck, C., Krusche, J., Elsherbini, A. M. A., Du, X. & Peschel, A. Phage susceptibility determinants of the opportunistic pathogen Staphylococcus epidermidis. Oh, J. et al. Biogeography and individuality shape function in the human skin metagenome. Burmeister, A. R. et al. Pleiotropy complicates a trade-off between phage resistance and antibiotic resistance. Acosta, E. M. et al. Bacterial DNA on the skin surface overrepresents the viable skin microbiome. Sheth, R. U. et al. Spatial metagenomic characterization of microbial biogeography in the gut. Claesen, J. et al. A Cutibacterium acnes antibiotic modulates human skin microbiota composition in hair follicles. Wei, M. et al. Harnessing diversity and antagonism within the pig skin microbiota to identify novel mediators of colonization resistance to methicillin-resistant Staphylococcus aureus. Midorikawa, K. et al. Staphylococcus aureus susceptibility to innate antimicrobial peptides, β-defensins and CAP18, expressed by human keratinocytes. & Gallo, R. L. AMPed up immunity: how antimicrobial peptides have multiple roles in immune defense. Ma, Y. et al. Identification of antimicrobial peptides from the human gut microbiome using deep learning. Huang, J. et al. Identification of potent antimicrobial peptides via a machine-learning pipeline that mines the entire space of peptide sequences. Maasch, J. R. M. A., Torres, M. D. T., Melo, M. C. R. & de la Fuente-Nunez, C. Molecular de-extinction of ancient antimicrobial peptides enabled by machine learning. Cell Host Microbe 31, 1260–1274.e6 (2023). Salamzade, R. et al. Evolutionary investigations of the biosynthetic diversity in the skin microbiome using lsaBGC. Shepherd, E. S., Deloache, W. C., Pruss, K. M., Whitaker, W. R. & Sonnenburg, J. L. An exclusive metabolic niche enables strain engraftment in the gut microbiota. Qu, E. B. et al. Intraspecies associations from strain-rich metagenome samples. 2025.02.07.636498 Preprint at bioRxiv https://doi.org/10.1101/2025.02.07.636498 (2025). Gaio, D. et al. Hackflex: low-cost, high-throughput, Illumina Nextera Flex library construction. Baym, M. et al. Inexpensive multiplexed library preparation for megabase-sized genomes. Bankevich, A. et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. Schwengers, O. et al. Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification. Page, A. J. et al. Roary: rapid large-scale prokaryote pan genome analysis. Feldgarden, M. et al. Validating the AMRFinder tool and resistance gene database by using antimicrobial resistance genotype-phenotype correlations in a collection of isolates. Liu, B., Zheng, D., Zhou, S., Chen, L. & Yang, J. VFDB 2022: a general classification scheme for bacterial virulence factors. Nucleic Acids Res. Tesson, F. et al. Systematic and quantitative view of the antiviral arsenal of prokaryotes. Cantalapiedra, C. P., Hernández-Plaza, A., Letunic, I., Bork, P. & Huerta-Cepas, J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Blin, K. et al. antiSMASH 7.0: new and improved predictions for detection, regulation, chemical structures and visualisation. Nucleic Acids Res. Navarro-Muñoz, J. C. et al. A computational framework to explore large-scale biosynthetic diversity. Article PubMed Deatherage, D. E. & Barrick, J. E. Identification of mutations in laboratory-evolved microbes from next-generation sequencing data using breseq. Gutiérrez, D., Martínez, B., Rodríguez, A. & García, P. Genomic characterization of two Staphylococcus epidermidis bacteriophages with anti-biofilm potential. Mancuso, C. Staphylococcus epidermidis antagonism screen photos. We thank the participants of the original study, and C. Dickinson, S. Friedhoff, M. Hirsch, S. Zuckermann and J. Schuler for assistance with study recruitment; J. Zhou, Z. Zhang, S. Zhang and S. Levine for advice on experimental design; and all members of the Lieberman Lab for advice on this project and feedback on the manuscript. This work was funded by The James H. Ferry, Jr Fund for Innovation in Research Education, Colgate-Palmolive Company, and NIH grant 1DP2GM140922 (all to T.D.L. Institute for Medical Engineering and Sciences, Massachusetts Institute of Technology, Cambridge, MA, USA Christopher P. Mancuso, Jacob S. Baker, Evan B. Qu, A. Delphine Tripp, Ishaq O. Balogun & Tami D. Lieberman Department of Civil and Environmental Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA Christopher P. Mancuso, Jacob S. Baker, Evan B. Qu, A. Delphine Tripp, Ishaq O. Balogun & Tami D. Lieberman Department of Systems Biology, Harvard University, Cambridge, MA, USA Broad Institute of MIT and Harvard, Cambridge, MA, USA Ragon Institute of MGH, MIT and Harvard, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project. collected and sequenced the samples. designed the experiments. carried out the experiments and conducted image processing and statistical analyses. conducted bioinformatic analyses. secured funding for the project. The original manuscript was drafted by C.P.M. ; all authors reviewed and edited the final manuscript. Correspondence to Tami D. Lieberman. The authors declare no competing interests. Nature Microbiology thanks Kevin Foster and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) We constructed a pangenome for S. epidermidis isolates in our collection and annotated genes and biosynthetic gene clusters (BGCs). Antagonism-related BGCs were more frequent in accessory genomes, whereas metabolism-related BGCs were found in the core genome. (b) We identified regions of the genome with copy number variation within a lineage, indicating recently gained or lost regions. We compared the relative enrichment of Clusters of Orthologous Gene (COG) terms and hits from functional annotation databases among genes in the core genome and recent gain/loss regions. (c) As in B, but comparing genes in the core and accessory genomes. COG categories denoted in blue are metabolism associated, whereas antimicrobial production and resistance genes may be found in red categories. Significant enrichment in a two-sided Fisher's Exact test is indicated by an asterisk (one for naive P-value < 0.05, two for Bonferroni corrected P-value < 0.05), and the number of genes annotated in each category is indicated to the right of the figure. In a previous study, up to 96 isolates of S. epidermidis were cultured from each of 23 participants at each timepoint sampled during a 1.5-year time period. Six families were selected for inclusion in this study. Relative lineage abundance, as determined from isolates, is plotted for each sample, organized by participant and family. PHLAME was applied to metagenomic samples from the same participant (denoted as MG), which confirmed that lineage abundances estimated by culturing did not have significant culture bias; note that uncalled portions of metagenomic samples (space above bars) can reflect uncertainty in PHLAME due low coverage rather than the true absence of lineages. We used isolate-based lineage abundances because deep metagenomic coverage of S. epidermidis was not available for every sample. Sample labels indicate the family, participant, and sampling timepoint (for example 1AA3 indicates Family 1, participant 1AA, timepoint 3). The number of S. epidermidis colonies isolated from each person is indicated above each bar. Lineages represented in the TSA antagonism screen in this study are indicated in color, while unrepresented lineages are indicated in greyscale. Some additional lineages were included in a follow-up screen M9 + S screen (see Methods); these are indicated by darker grey colors (denoted by *). Colors do not carry over across families, lineages are generally restricted to a family. Across all samples, the average proportion of lineage composition represented by isolates included in the TSA antagonism screen was 96%. Participant 8PB reported antibiotic use between timepoints, which may explain the difference between timepoints. We measured 21,025 pairwise interactions between 145 isolates that passed inclusion criteria (15 replicate cultures were also included, totalling 25,600 interactions). The heatmap depicts the AUC of the ZOI calculated for pairwise interactions between S. epidermidis and other skin microbiome isolates in TSA. Each row and column indicate an isolate (in contrast to Fig. 2a where rows indicate lineages). Multiple isolates from a lineage are denoted by decimals (for example 20.1, 20.2, 20.3). Isolates are sorted by phylogenetic similarity, with dashed lines to indicate species boundaries. Each isolate of species other than S. epidermidis (labels > 200, see x-axis for details) was treated as a separate lineage. Interactions were shared among all isolates of the same lineage in ~97% of cases, so S. epidermidis isolates were dereplicated into lineages for other analyses unless otherwise noted. Nine isolates were added after the preliminary TSA screen, and thus are lacking some data points, indicated by brown squares. Unless otherwise noted, these isolates with partial data were excluded from statistical analysis. Rather than depicting genomic relatedness using a tree (Fig. 2a), we calculated pairwise distances by different sequence similarities to understand whether antagonism and relatedness are correlated. (a) We binned interactions between pairs of lineages with similar average amino acid identities, then plotted the frequency of antagonism among pairs in each bin. (b) As in A, but binned according to average nucleotide identity. (c) As in B, but only for lineages of S. epidermidis, not other species. Antagonism frequency does not significantly vary with genomic distance in our collection, as both R2 and P are weak. (d) We plotted the strength of antagonism (AUC of ZOI) for each antagonistic interaction (excluding non-antagonistic interactions) against the average amino acid identity of the pair. (e) As in D, but against the average nucleotide identity. (f) As in E, but only for lineages of S. epidermidis, not other species. There is a significant but small increase in antagonism strength at high nucleotide identities among S. epidermidis. All P-values and R2 values report the fit of a linear model evaluated by a two-sided ANOVA. (g) The frequency of antagonism does not significantly differ between members of the same or different S. epidermidis phylogroup, according to a two-sided permutation test with shuffled group labels. The phylogeny at right depicts S. epidermidis phylogroup boundaries. Taken together, these results indicate that antagonism occurs at all levels of genomic relatedness, even between closely related lineages. (a) In order to check for shared mechanisms, the binary antagonism matrix was clustered by constructing dendrograms for antagonism and sensitivity (Methods). White boxes indicate antagonism by the producer spot against the receiver lawn. Red arrows indicate the lineages in our collection chosen for our mechanism screen, which include 18 lineages with high antagonism frequencies plus 2 negative controls (which lacked antimicrobial production genes but contained suspected prophage, denoted by yellow arrows). We chose representative receiver bait lineages using the dendrogram of sensitivity to ensure that all interaction patterns were captured. Note that for this analysis, unlike other analyses in this study (Methods), antagonisms from idiosyncratic isolates that exhibit intra-lineage variation were not excluded. Lineages denoted by cyan arrows did not inhibit many S. epidermidis, with the exception of idiosyncratic lineages 20, 37, and 58, which cluster with some S. hominis and S. capitis isolates, denoted by a cyan line. It is possible that the antimicrobials behind these interactions are intended to target non-epidermidis species. (b) We screened cells and filtered supernatants against a representative array of bait lawns chosen based on interaction profiles from above. Filtrates treated with heat or proteinase to degrade proteins and peptides, desalting to remove small molecules, or 1:10 dilution were also tested. Filtrates exhibited varied patterns, including antagonisms likely mediated by small peptide effectors (isolate 30.1) and some cell-associated antagonisms induced by neighbours (isolate 34.2) or both. (c) Some cell-associated antagonisms are iron-mediated, as indicated by the elimination of the interaction in media supplemented with excess FeCl3 (see isolate 51.1). All tested isolates displaying iron-suppressible antagonisms cells contained a biosynthetic gene cluster (BGC) for a product similar to pulcherriminic acid, an iron-sequestering pigment molecule which sabotages the iron supply. (d) We annotated putative bacteriocin BGCs and found BGCs for known compounds (including gallidermin-like, epicidin-like, and lactococcin-like BGCs) as well as unknown compounds. However, we did not find a clear correspondence between BGC presence and the phenotype classified in our mechanism screen, potentially due to multiple inducible effectors being encoded in the genome, incomplete expression of BGCs in different media or environments, or targets not represented in our screen (for example Cutibacterium acnes). One isolate, 32.1, had an active prophage that occasionally produced distinct plaques in this assay, and was later isolated using mitomycin-C induction. (a) We overlaid interactions from TSA onto the S. epidermidis interaction matrix generated in M9 + S media, using color to indicate which media the antagonism appeared in. In total, 91% of interactions are the same in both media conditions, though antagonisms did vary between the two media, such that 23% of antagonisms observed in TSA were also observed in M9 + S, and 61% of antagonisms observed in M9 + S were observed in TSA. (b-e) Despite these differences, we observed a significant depletion of on-person antagonism (ΔAF, see Fig. 3c, Methods) in permutation tests that compare our observed data compared to simulated data with shuffled lineage compositions on each participant. Significance P-values are derived from two-sided permutation tests with shuffled lineage labels (Methods), without multiple hypothesis correction. (a) Antagonism frequencies (AF) between co-resident lineages in samples appears depleted relative to the frequency across all S. epidermidis lineages (red). When lineages were present in a sample but were absent from our screen, we calculated the range of AF that would be expected based on the average rate of antagonism between any two isolates. Error bars depict the upper bound of the AF range in a given sample. This is likely an overestimate of the range, since on-person antagonism frequencies are lower than population-wide antagonism frequency. (b) We took the mean AF across samples of the same participant (subject), and still found that mean within sample AF appeared to be depleted relative to the expected AF. Reproduced from Fig. (c-f) Since t-tests are invalid for non-independent samples, we performed permutation testing by simulating scenarios that retained the structure in the interaction matrix and the composition of samples (for example uneven lineage distribution). For each simulation, we shuffled lineage labels in the relative abundance matrix and the p-value represents the fraction of simulations with less on-person antagonism than observed (black arrow indicates observed value of ΔAF). (c) The rows of the composition matrix were shuffled once across all lineages in our collection. This simulation tests for depletion of antagonism while maintaining the same degree of lineage sharing seen between individuals in the same family. Reproduced from Fig. (d) The rows of the composition matrix were shuffled, AF values were calculated for samples of the first participant, then rows were shuffled again before each new subject. These simulations broke the relatedness between individuals in the same family, while retaining similarity between samples from the same participant, thereby testing depletion of antagonism under the assumption that lineages are acquired from the environment independent of family. (e) To test for depletion of antagonism under the assumption that one can only acquire strains from a family member, shuffling was only performed on submatrices composed only of lineages on a family were shuffled, by row, as in C. (f) As in F, but without maintenance of lineage sharing between family members; submatrices shuffled but by once per participant. Note that less shuffled conditions (e,f) will definitionally have higher p-values because of fewer possible permutations. Under all tested assumptions, antagonism is depleted among strains present on the same person at the same time. Significance P-values are derived from two-sided permutation tests with shuffled lineage labels (Methods), without multiple hypothesis correction. Relative abundance plots where each lineage is colored by the proportion of other lineages it antagonizes on TSA, with blues representing non-antagonistic lineages and reds representing highly antagonistic lineages. Lineages are labelled by number in white text and are plotted in the same order as in Extended Data Fig. Grey lineages have missing data, and therefore an antagonism proportion cannot be calculated. Note that antagonism proportion is relative to other lineages across the study, not just within a family. Some families contain few antagonistic lineages (for example family 1), whereas most families contain at least one highly antagonistic lineage. (a) Lineages were grouped into phylogroups according to phylogenetic structure as described previously, see Extended Data Fig. On many participants, multiple phylogroups coexist on the face at the same time. (b) The agrD sequence was annotated for each lineage on each participant and classified into quorum sensing types. Again, we found that on many participants, multiple agr types coexist on the face at the same time. Co-residence between different phylogroups and between incompatible quorum sensing types suggest that there is not competitive exclusion on the basis of phylogenic dissimilarity or quorum sensing. (c) We measured growth rate in liquid culture in TSB for all isolates in our collection (Methods). We calculated the median growth rate for each isolate in a lineage. Bars denote standard deviation (from technical duplicates of biological triplicates). (d) Lineages that grow faster than other lineages in vitro (green) reach higher abundance on individuals (blue), Spearman's rank correlation. Note that growth in vivo may vary significantly due to differences between nutrient composition of skin and TSB. (e) Lineages that antagonize a higher fraction of other lineages (red) do not grow faster or slower than other lineages (green), Spearman's rank correlation. (f) The growth rate for each isolate of idiosyncratic lineages 20, 37, and 58 (technical duplicates of biological quadruplicates) is shown. Growth rate varies between vraFG mutants (indicated with *) and wild-type isolates of each lineage, but without a directional trend. Reversion mutants of 37 have faster growth rates than 37.3, leaving the basis of the vraFG mutations unresolved. Four “Idiosyncratic” isolates (20.3, 37.3, 49.5, and 70.3) exhibited large differences in antimicrobial sensitivity compared to other members of their respective lineages. Two idiosyncratic isolates shared a sensitivity pattern with lineage 58 (Fig. 4a, Extended Data Fig. 5), prompting a search for parallel mutations. We plotted the sensitivity phenotype against whole genome single nucleotide polymorphism trees constructed for each lineage. Each isolate on the tree was annotated with any vraFG and dlt operon mutations and/or genomic region gains or losses (colored by copy number relative to core genome). The sensitive phenotype (red/orange) correlated with vraF and vraG mutations in (a) lineage 20, (b) lineage 37, and (c) lineage 58. Presence or absence of genomic regions have no apparent trend in regards to the idiosyncratic sensitivity phenotype. While the vraFG mutants form a small fraction of their respective lineages, these isolates were found at multiple timepoints over six months apart, indicating that they are not a transient part of the population. (d) Since each represented a minority of their lineages, idiosyncratic isolates were excluded from other analyses presented in this study. However, on-person antagonisms remain depleted even if these isolates are retained (using simulations as in Fig. Significance P-value is derived from a two-sided permutation test with shuffled lineage labels (Methods), without multiple hypothesis correction. (e) Despite other sensitivities (Fig. 4d), the vraF-frameshift isolate (37.3) was less susceptible to phage infection than isolates with full-length vraF. However, we did not find this phage on participants carrying lineage 37 (Methods). Data are presented as mean +/- standard deviation, n = 6, P values were calculated by unpaired t-test (two-sided). (f) Nevertheless, together with prior results from literature, these results are consistent with a conceptual model in which the vraFG complex could reduce cell envelope modifications, weakening resistance to cationic antimicrobials but providing protection against some phage. (g) Heatmap depicting the Minimum Inhibitory Concentration for antibiotics, lysis enzymes, and antimicrobial fatty acids, based on a screen with one replicate per condition. Polymyxin B ( + 5) and kanamycin ( + 4) are more positively charged than clindamycin ( + 1), ciprofloxacin (0), or cephalothin (-1). Idiosyncratic isolates 20.3 and 37.3 are more sensitive to cationic antimicrobials and lysostaphin than other isolates in their lineages. Both vraFG and dlt operon genotypes in lineage 58 share a similar MIC pattern to these idiosyncratic isolates, especially in response to Polymyxin B. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Mancuso, C.P., Baker, J.S., Qu, E.B. et al. Intraspecies warfare restricts strain coexistence in human skin microbiomes. Nat Microbiol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Microbiol) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250629033506.htm'>Scientists discover ‘off switch' enzyme that could stop heart disease and diabetes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 08:00:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists at The University of Texas at Arlington have identified a new enzyme that can be "turned off" to help the body maintain healthy cholesterol levels -- a significant development that could lead to new treatments for diseases that affect millions of Americans. "We found that by blocking the enzyme IDO1, we are able to control the inflammation in immune cells called macrophages," said Subhrangsu S. Mandal, lead author of a new peer-reviewed study and professor of chemistry at UT Arlington. By better understanding IDO1 and how to block it, we have the potential to better control inflammation and restore proper cholesterol processing, stopping many of these diseases in their tracks." But when inflammation becomes abnormal -- due to stress, injury or infection -- it can damage cells, disrupt normal functions and increase the risk of serious diseases. During these periods, macrophages can't absorb cholesterol properly, which can lead to chronic disease. The team -- Dr. Mandal, postdoctoral researcher Avisankar Chini; doctoral students Prarthana Guha, Ashcharya Rishi and Nagashree Bhat; master's student Angel Covarrubias; and undergraduate researchers Valeria Martinez, Lucine Devejian and Bao Nhi Nguyen -- found that the enzyme IDO1 becomes activated during inflammation, producing a substance called kynurenine that interferes with how macrophages process cholesterol. When IDO1 is blocked, however, macrophages regain their ability to absorb cholesterol. This suggests that reducing IDO1 activity could offer a new way to help prevent heart disease by keeping cholesterol levels in check. They believe that inhibiting NOS could provide another potential therapy for managing cholesterol problems driven by inflammation. "These findings are important because we know too much cholesterol buildup in macrophages can lead to clogged arteries, heart disease and a host of other illnesses," Mandal said. Next, the research team plans to dig deeper into how IDO1 interacts with cholesterol regulation and whether other enzymes play a role. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250629033459.htm'>Quantum computers just beat classical ones — Exponentially and unconditionally</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 06:32:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Quantum computers have the potential to speed up computation, help design new medicines, break codes, and discover exotic new materials -- but that's only when they are truly functional. One key thing that gets in the way: noise or the errors that are produced during computations on a quantum machine -- which in fact makes them less powerful than classical computers - until recently. Daniel Lidar, holder of the Viterbi Professorship in Engineering and Professor of Electrical & Computing Engineering at the USC Viterbi School of Engineering, has been iterating on quantum error correction, and in a new study along with collaborators at USC and Johns Hopkins, has been able to demonstrate a quantum exponential scaling advantage, using two 127-qubit IBM Quantum Eagle processor-powered quantum computers, over the cloud. The paper, "Demonstration of Algorithmic Quantum Speedup for an Abelian Hidden Subgroup Problem," was published in APS flagship journal Physical Review X. The key milestone for quantum computing, Lidar says, has always been to demonstrate that we can execute entire algorithms with a scaling speedup relative to ordinary "classical" computers. And an exponential speedup means that the performance gap roughly doubles for every additional variable. What makes a speedup "unconditional," Lidar explains, is that it doesn't rely on any unproven assumptions. Here, the team led by Lidar used an algorithm they modified for the quantum computer to solve a variation of "Simon's problem," an early example of quantum algorithms that can, in theory, solve a task exponentially faster than any classical counterpart, unconditionally. Simon's problem involves finding a hidden repeating pattern in a mathematical function and is considered the precursor to what's known as Shor's factoring algorithm, which can be used to break codes and launched the entire field of quantum computing. So, how did the team achieve their exponential speedup? Phattharaporn Singkanipa, USC doctoral researcher and first author, says, "The key was squeezing every ounce of performance from the hardware: shorter circuits, smarter pulse sequences, and statistical error mitigation." This resulted in fewer quantum logic operations than would be needed otherwise, which reduced the opportunity for error buildup. Dynamical decoupling had the most dramatic impact on their ability to demonstrate a quantum speedup. Finally, they applied "measurement error mitigation," a method that finds and corrects certain errors that are left over after dynamical decoupling due to imperfections in measuring the qubits' state at the end of the algorithm. Says Lidar, who is also a professor of Chemistry and Physics at the USC Dornsife College of Letters, Arts and Science, "The quantum computing community is showing how quantum processors are beginning to outperform their classical counterparts in targeted tasks, and are stepping into a territory classical computing simply can't reach., Our result shows that already today's quantum computers firmly lie on the side of a scaling quantum advantage." In other words, the quantum performance advantage is becoming increasingly difficult to dispute. Lidar cautions that "this result doesn't have practical applications beyond winning guessing games, and much more work remains to be done before quantum computers can be claimed to have solved a practical real-world problem." Nevertheless, quantum computers' previously "on-paper promise" to provide exponential speedups has now been firmly demonstrated. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250629033449.htm'>Fire smoke exposure leaves toxic metals and lasting immune changes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-30 05:37:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Exposure to fire smoke -- which can be composed of particulate matter, gases, materials from buildings such as perfluoroalkyl and polyfluoroalkyl substances (PFAS), toxic metals, and carcinogenic compounds -- may alter the immune system on a cellular level, according to a new study led by researchers at Harvard T.H. "We've known that smoke exposure causes poor respiratory, cardiac, neurological, and pregnancy outcomes, but we haven't understood how," said corresponding author Kari Nadeau, John Rock Professor of Climate and Population Studies and chair of the Department of Environmental Health. "Our study fills in this knowledge gap, so that clinicians and public health leaders are better equipped to respond to the growing threat of difficult to contain, toxic wildfires." The study was published pn June 26 in Nature Medicine. None of the participants had an acute or chronic condition, and none were taking immunomodulatory drugs at or before the time of the blood draw, which took place within one month after participants had been exposed to fire smoke. Smoke-exposed individuals showed an increase in memory CD8+ T cells (a type of immune cell critical to long-term immunity against pathogens) and elevated activation and chemokine receptor biomarkers (indicators of inflammation and immune activity) within multiple cell types. Additionally, those who had been exposed to smoke showed changes in133 genes related to allergies and asthma, and more of their immune cells were bound with toxic metals, including mercury and cadmium. "Our findings demonstrate that the immune system is extremely sensitive to environmental exposures like fire smoke, even in healthy individuals," said lead author Mary Johnson, principal research scientist in the Department of Environmental Health. "Knowing exactly how may help us detect immune dysfunction from smoke exposure earlier and could pave the way for new therapeutics to mitigate, or prevent altogether, the health effects of smoke exposure and environmental contaminants." "Knowing more about exactly how smoke exposure is harming the body, we may increase public health campaigns about the dangers of smoke exposure and the importance of following evacuation procedures during wildfires," Nadeau said. Other Harvard Chan co-authors included Abhinav Kaushik, Olivia Kline, Xiaoying Zhou, and Elisabeth Simonin. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            