
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-20</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250520121544.htm'>New perspectives for wound healing and the treatment of chronic diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 18:42:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These specialised cells are not only involved in tissue repair and remodelling, but also influence the immune system and the development of diseases such as cancer, fibrosis and chronic inflammatory conditions. "Until now, our understanding of fibroblast diversity has been based primarily on studies in animal models. This new review is the first to compare and integrate extensive human studies that have used modern single-cell technologies. This approach makes it possible to combine findings from different human studies, creating a comprehensive picture of the various origins and functions of human fibroblasts," says Professor Sandra Franz, lead author of the study from the University of Leipzig Medical Center. This is particularly significant for regenerative medicine and the treatment of chronic diseases. The authors of the study -- Dr Marta Torregrossa, Professor Jan C. Simon, and Professor Sandra Franz (all from the University of Leipzig Medical Center), together with Dr Yuval Rinkevich (Helmholtz Munich) -- are conducting joint research as part of the ZellTWund project. Their aim is to further characterise regeneration-promoting fibroblast subtypes and their roles in health and disease -- thereby paving the way for new therapeutic approaches. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250520012832.htm'>Eating craved foods with meals lessens cravings, boosts weight loss</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 18:12:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Food cravings are the downfall of many dieters, who feel locked in an eternal battle with their willpower to resist the tempting sweets, snacks and other foods they love. However, researchers in food science and human nutrition at the University of Illinois Urbana-Champaign say in a new study that eating dessert may be the optimal strategy for losing weight, keeping it off and keeping cravings at bay. Dieters in a clinical trial who incorporated craved foods into a balanced meal plan lost more weight during the 12-month weight-loss program and their cravings remained minimal through the subsequent 12 months of maintenance. Then-graduate student Nouf W. Alfouzan and nutrition professor Manabu T. Nakamura, who published their findings in the journal Physiology and Behavior, said that dieters' food cravings decreased while losing weight and remained minimal as long as they did not regain weight. The participants were part of a larger project conducted with physicians at Carle Clinic in Urbana, Illinois, that aimed to broaden the outreach of an in-person weight-loss program called the Individualized Dietary Improvement Program by converting it to an online format called EMPOWER. If they have a lot of cravings, it is very difficult to lose weight. Although many dieters reported in prior research that their cravings decreased while losing weight, Nakamura and Alfouzan said it remained unclear whether these changes persisted when they reached their weight-loss goal or quit dieting while trying to maintain their weight. The dietary program used in the study educates dieters about key nutrients, helping them make informed decisions about their food selections until sustainable dietary changes are achieved, Nakamura said. Dieters used a data visualization tool that plots foods' protein, fiber and calories so they can enhance nutrition while minimizing the calories they consume. During the first year, participants engaged in 22 online nutrition education sessions created by scholars at the university's Center for Innovation in Teaching and Learning that included strategies for dealing with cravings. "If you are eating and snacking randomly, it's very hard to control," Nakamura said. Our plan used an 'inclusion strategy,' in which people incorporated small portions of craved foods within a well-balanced meal." Every six months participants completed a questionnaire about their cravings for specific foods. Using a scale ranging from 1 (never) to 6 (always), individuals rated themselves on statements such as, "Whenever I have food cravings, I find myself making plans to eat" and "I have no willpower to resist my food cravings." The intensity of each dieter's cravings was calculated by adding their scores for all 15 statements. Of these, 20 people completed the yearlong maintenance program, but because a few regained some of what they had lost, their average weight loss was 6.7%, the researchers found. This finding suggests that individuals' reduced cravings correlated with decreased body fat rather than the negative energy balance of dieting, Nakamura said. "This basically debunks the hungry fat cell theory, a longstanding hypothesis that fat cells become starved for energy and trigger cravings, causing dieters to eat and ultimately regain what they lost. Of the 24 participants who remained in the study at 12 months, more than half reported they used the inclusion strategy to manage cravings, some people as frequently as one to three times a day, while others used it once a week or more. Those who used the strategy lost significantly more weight compared with other participants and experienced significant reductions in their cravings for sweet and high-fat foods, the researchers found. Consistency is another key to managing cravings and weight, Nakamura said. Fluctuations in eating patterns, meal times and amounts trigger cravings, too. Materials provided by University of Illinois at Urbana-Champaign, News Bureau. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43018-025-00982-7'>Age-associated nicotinamide adenine dinucleotide decline drives CAR-T cell failure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 13:32:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Chimeric antigen receptor (CAR) T cell therapy is one of the most promising cancer treatments. However, different hurdles are limiting its application and efficacy. In this context, how aging influences CAR-T cell outcomes is largely unknown. Here we show that CAR-T cells generated from aged female mice present a mitochondrial dysfunction derived from nicotinamide adenine dinucleotide (NAD) depletion that leads to poor stem-like properties and limited functionality in vivo. Moreover, human data analysis revealed that both age and NAD metabolism determine the responsiveness to CAR-T cell therapy. Targeting NAD pathways, we were able to recover the mitochondrial fitness and functionality of CAR-T cells derived from older adults. Altogether, our study demonstrates that aging is a limiting factor to successful CAR-T cell responses. Repairing metabolic and functional obstacles derived from age, such as NAD decline, is a promising strategy to improve current and future CAR-T cell therapies. Immunotherapeutic strategies based on adoptive cell transfer (ACT) of chimeric antigen receptor (CAR) T cells are currently among the most promising approaches to treat cancer. Amongst the features that determine successful responses to CAR-T cell therapy, the long-term maintenance of a T cell pool with stem-like properties is fundamental1,2. As a result, the CAR-T cell field is evolving toward finding strategies that favor the quantity and quality of stem-like T cells in CAR-T infusion products1,2,3. Importantly, this stem-like population engages a particular metabolic program that relies mostly on mitochondrial activity4,5,6,7,8. Indeed, recent studies demonstrated that CAR-T cell products with decreased mitochondrial fitness are associated with poor responses9,10 and metabolic interventions boosting mitochondrial metabolism are able to improve the efficacy of CAR-T cell therapy in preclinical models11,12,13,14,15. A key metabolite that ensures mitochondrial health is nicotinamide adenine dinucleotide (NAD). NAD is a well-known cofactor that has a pivotal role in redox balance and energy metabolism by fueling oxidative phosphorylation. Moreover, it serves as a substrate for several enzymes, such as sirtuin deacetylases (SIRTs), that can regulate the expression of peroxisome proliferator-activated receptor-γ coactivator 1α, a transcription cofactor involved in mitochondrial biogenesis16,17,18. NAD metabolism critically regulates T cell fate and function19,20,21. Thus, alterations in NAD homeostasis have been linked to impaired T cell responses20,22, while restoration of mitochondrial dysfunction through NAD-boosting strategies has been shown to prevent exhaustion of tumor-infiltrating lymphocytes (TILs)23. Aging is the first risk factor associated with cancer. Consequently, the majority (~75%) of persons with cancer and persons eligible for cancer immunotherapy are >65 years old. Importantly, in the context of CAR-T cell therapy, the highest response is observed in B cell acute lymphoblastic leukemia (B-ALL) when the median diagnostic age is <20 years old, while the responses decline with increasing age24. However, whether aging is an important limiting factor for CAR-T cell efficacy and its underlying mechanisms is still unknown. Several investigations have reported that aging leads to deficient immune and metabolic functions that result in altered antitumor responses25. Interestingly, mitochondrial dysfunction is a hallmark of aging26 and NAD decline has been described across several tissues including white adipose tissue (WAT), muscle and liver27,28,29. In our study, we demonstrate that age is a limiting factor for effective CAR-T cell responses. We show that aging drives mitochondrial dysfunction in T cells, which impairs their stem-like properties and antitumor capacities when transduced with a tumor-antigen-directed CAR. We then determine the decline in NAD cellular levels as a major factor responsible for this process and the restauration of NAD homeostasis as a strategy to rejuvenate old CAR-T cells. To decipher how aging affects the composition of CAR-T cell infusion products, we transduced CD8+ T cells derived from young (8 weeks old) and old (>80 weeks old) mice with a Thy1.1+ CAR construct targeting the human oncogene human epidermal growth factor receptor 2 (HER2, also known as ERBB2) (refs. CAR-T cells were then expanded under effector-like (TEM) or memory-like (TCM) polarizing conditions (involving the use of interleukin 2 (IL-2) or IL-7 and IL-15, respectively) (Fig. Young and old CAR-T cells displayed comparable differentiation capacity when cultured in the presence of IL-2 (Fig. However, old CAR-T cells were unable to properly acquire a memory-like phenotype when cultured with IL-7 and IL-15, as shown by both the lower proportion of TCM cells (Fig. 1b,c) and the decreased levels of the stemness marker T cell factor 1 (TCF1) when compared to younger counterparts (Fig. To prove that these differences stem not only from the accumulation of experienced (CD44+) T cells with age (Extended Data Fig. 1a) but also from cell-intrinsic defects, we isolated CD8+ naive T cells (CD44−CD62L+) from young and old mice and tested their differentiation capability in vitro. Following initial activation, we expanded under TEM or TCM polarizing conditions and we observed that naive T cells derived from aged mice did not differentiate efficiently toward a TCM phenotype (Extended Data Fig. Accordingly, the TCM population had lower TCF1 expression (Extended Data Fig. These results suggest that the defective memory-like phenotype identified in aged CAR-T cell infusion products is not only because of differences in the initial population heterogeneity but also because of cell-intrinsic defects of CD8+ T cells. On day 3, T cells were further expanded in the presence of IL-2 or IL-7 and IL-15. c, Proportion of CAR-T cells with TEM (CD44+CD62L−) and TCM (CD44+CD62L+) phenotype (n = 3 biologically independent samples). d, Levels of TCF1 within the TCM population of young and old CAR-T cells on day 7 (n = 3 biologically independent samples). e, Killing capacity of young and old CAR-T cells expanded with IL-7 and IL-15 upon coculture with B16-HER2 cells at a 2:1 effector-to-target ratio. The graph is representative of three independent experiments; error bars represent technical replicates. f, Proportion of IFNγ+ CAR-T cells expanded with IL-7 and IL-15 upon coculture with B16-HER2 cells (n = 4 biologically independent samples). g, Proportion of TNF+ CAR-T cells expanded with IL-7 and IL-15 upon coculture with B16-HER2 cells (n = 4 biologically independent samples). Data are presented as the mean values ± s.e.m. Statistical analysis was performed using a two-way analysis of variance (ANOVA) with Tukey's multiple-comparison test (c,d) or unpaired t-test (f,g), as appropriate. To further assess their functionality, we rechallenged young and old CAR-T cells against HER2-overexpressing B16 melanoma cells in vitro. 1f,g) when compared to young CAR-T cells, thus reflecting the acquisition of a TEM phenotype. Of note, no differences were observed between young and old CAR-T cells when cultured in the presence of IL-2 (Extended Data Fig. Despite providing enhanced cytotoxicity, TEM cells exhibit limited proliferative and self-renewing capacities and are more prone to develop an exhausted phenotype. Indeed, old CAR-T cells under chronic stimulation in vitro had lower IFNγ and TNF production (Extended Data Fig. 1f,g), with no differences in the expression of exhaustion markers such as programmed cell death protein 1 (PD1), T cell immunoglobulin and mucin domain-containing protein 3 (TIM3), lymphocyte activation gene 3 (LAG3) or thymocyte selection-associated high-mobility group box protein (TOX) (Extended Data Fig. Altogether, these results indicate that age greatly influences the composition of CAR-T cell infusion products, favoring the accumulation of TEM cells with higher cytotoxic properties but limited functions upon multiple rechallenges. To determine whether aging would be a limiting factor for CAR-T cell efficacy, we next challenged the HER2-directed CAR-T cells in vivo (Fig. Specifically, we adoptively transferred young and old CAR-T cells (CD45.1+) into mice (CD45.2+) bearing HER2+ B16 tumors. We observed that only CAR-T cells derived from young mice and expanded with IL-7 and IL-15 but not those derived from aged mice could control tumor growth (Fig. No differences were observed between young and old CAR-T cells when expanded in IL-2 (Extended Data Fig. Young CAR-T cells are able to persist long-term in vivo, generating a pool of CD44+CD62L+TCF1+ CAR-T cells in the spleen that can potentially generate tumor-specific TEM cells that can migrate to the tumor microenvironment (TME). Strikingly, the numbers of transferred (CD45.1+) and CAR-expressing (Thy1.1+) aged T cells were nearly undetectable 30 days after ACT in the spleen (Fig. Aged CAR-T cells completely lacked a memory or stem phenotype, as shown by the absence of CD44+CD62L+ (Fig. Interestingly, aged CAR-T cells partially improved their persistence when transferred into older hosts, suggesting that age-dependent changes in the microenvironment can also influence CAR-T persistence (Extended Data Fig. 1n; gating strategy in Extended Data Fig. a, HER2-directed young or old CAR-T cells were adoptively transferred into mice bearing B16-HER2 tumors. Tumor growth was monitored and CAR-T persistence was determined on day 30. b, Follow-up of tumor growth over the course of the experiment. d, Number of adoptively transferred T cells (CD45.1+) in spleen after 30 days of tumor engraftment (n = 6 mice). e, Representative CD45.1/CD45.2 dot plots, gated within the CD8+ T cell populations in spleens. f–h, The number of CAR+ (Thy1.1+) (f), CD44+CD62L+ (g) and TCF1+ (h) T cells, gated within the CD3+CD8+CD45.1+CD45.2− population (n = 6 mice). Data are presented as the mean values ± s.e.m. Statistical analysis was performed using a one-way ANOVA with multiple comparisons (c) or unpaired t-test (d,f–h), as appropriate. The development and maintenance of stem-like properties relies on the T cell capacity to boost mitochondrial metabolism5,34,35. Mitochondrial dysfunction has been extensively studied as a hallmark of aging26,36 but whether mitochondrial defects are the main driver of the loss of stemness in aged CD8+ T cells has not yet been elucidated. To this end, we first compared the mitochondrial profile of freshly isolated CD8+ T cells derived from the spleens of young and aged mice by examining mitochondrial membrane potential and mitochondrial mass by staining with tetramethylrhodamine (TMRM) and MitoTracker green, respectively. CD8+ T cells presented an age-dependent drop in both mitochondrial membrane potential and mass (Fig. Moreover, old CD8+ T cells accumulated mitochondrial reactive oxygen species (Extended Data Fig. Overall, these data suggest the acquisition of mitochondrial dysfunction in aged T cells, as previously reported23,37,38,39,40,41. Accordingly, old CD8+ T cells displayed a reduced basal and maximal oxygen consumption rate (OCR) upon T cell receptor stimulation (Extended Data Fig. To demonstrate that these mitochondrial deficiencies were cell-intrinsic defects induced by aging, we proved that mitochondrial activity was dampened in all T cell populations (naive T cell (TN), TEM and TCM) (Extended Data Fig. Similarly, naive CD8+ T cells also presented an age-dependent progressive decrease in mitochondrial activity without altering mitochondrial size (Extended Data Fig. Electron microscopy (EM) confirmed these findings, as no differences were detected in the number or size of mitochondria (Fig. 3c,d) but aged naive CD8+ T cells had a reduced number of cristae (Fig. 3c,e), a feature of mitochondria with reduced activity. a, TMRM and MitoTracker green staining in freshly isolated bulk CD8+ T cells from the spleens of young and old mice. TMRM is a cell-permeable dye that accumulates in active mitochondria with intact membrane potential, while MitoTracker green binds to mitochondrial proteins giving a readout of mitochondrial activity and size, respectively (n = 5 biologically independent samples). b, Representative TMRM and MitoTracker green dot plots of data summarized in a. c, EM images of young and old naive CD8+ T cells. d,e, The number of mitochondria per cell (d) and the number of cristae per mitochondria (e) found by EM (n = 3 biologically independent samples). In d, dots represent the number of cells analyzed (n = 20). In e, dots represent the number of mitochondria analyzed (n = 50). f, Volcano plot representing metabolomic data in young versus old CD8+ T cells (n = 5 biologically independent samples). g, Young (8 weeks old), intermediate (50 weeks old) and old (105 weeks old) CD8+ T cells were activated for 3 days and treated with the NAD precursor NMN (1 mM) for another 2 days, after which mitochondrial activity was assessed by TMRM staining (n = 4 biologically independent samples). h, Young and old CD8+ T cells were activated and expanded until day 7 in the presence of IL-7 and IL-15, after which they further received three rounds of CD3 restimulation every other day to promote an exhausted phenotype. Cells were treated with the NAD-booster NR (1 mM) and levels of the transcription factor TOX were determined on day 12 (n = 3 biologically independent samples). Data are presented as the mean values ± s.e.m. Statistical analysis was performed using an unpaired t-test (a,d,e), two-way ANOVA (g) or paired t-test (h), as appropriate. Statistical analysis of metabolomic data was performed using a two-way ANOVA on log10-transformed data and corrected with the Benjamini–Hochberg method. To identify age-specific mechanisms driving mitochondrial dysfunction and CAR-T cell failure, we performed liquid chromatography with tandem mass spectrometry (LC–MS/MS) to analyze the metabolome of CD8+ T cells derived from young and old mice. Interestingly, aged CD8+ T cells had higher abundance for 15 metabolites, mostly related to purine and pyrimidine synthesis, such as adenosine diphosphate (ADP) riboside, adenosine monophosphate or guanosine monophosphate (Fig. 3f), and lower abundance for 11 metabolites when compared to young CD8+ T cells (Supplementary Table 1). We and others have shown that supplementation with NAD precursors can increase NAD cellular levels and ameliorate mitochondrial function42. To this end, we treated aged and young T cells with the NAD precursor nicotinamide mononucleotide (NMN) and analyzed their mitochondrial activity. Interestingly, we found that, while NMN was able to improve the mitochondrial activity of young T cells, T cells derived from aged mice were irresponsive to NMN treatment (Fig. Similarly, old T cells treated with the NAD precursor nicotinamide riboside (NR), which we previously reported to prevent T cell exhaustion23, were not able to prevent TOX upregulation upon chronic in vitro stimulations, in contrast to younger counterparts (Fig. Overall, our findings demonstrate that aged CD8+ T cells accumulate important mitochondrial defects associated with NAD decline, which cannot be reversed through the administration of conventional NAD precursors. An investigation by Camacho-Pereira et al. revealed that one of the main triggering factors of NAD decline in WAT, muscle and liver during aging is the systemic upregulation of CD38 (ref. CD38 is a multifunctional enzyme that degrades NAD and modulates NAD homeostasis in a variety of immune cells43,44. CD38-mediated NAD-degradation leads to the production of substrates (namely, cyclic ADP-ribose and nicotinic acid adenine dinucleotide phosphate) that are subsequently involved in the regulation of Ca2+ signaling and the production of adenosine, an important metabolite with immunosuppressive functions43,44. Moreover, it has been reported that CD38 can also mediate the degradation of NAD precursors, including NMN and NR45. Thus, we tested whether CD38 could impact aged CD8+ T cells function by (1) limiting NAD metabolism and mitochondrial activity and (2) preventing responsiveness to NAD precursors. We investigated how CD38 levels change during T cell aging and we found that CD38 is expressed to a higher level in old CD8+ T cells, in both resting and activated states (Fig. Importantly, this phenomenon was observed in all CD8+ T cell populations (TN, TEM and TCM) (Extended Data Fig. To determine whether CD38 is a limiting factor of mitochondrial activity in aged CD8+ T cells, we measured mitochondrial membrane potential in T cell subpopulations with high (CD38hi) and low (CD38low) CD38 expression. Importantly, we found that, in aged CD8+ T cells but not in younger controls, only CD38hi T cells displayed reduced mitochondrial activity (Fig. Moreover, young CD8+ T cells transduced with a CD38-overexpressing construct exhibited a limited generation of TCM cells and decreased mitochondrial DNA (mtDNA) content (Extended Data Fig. 4b,c), recapitulating the features of aged CD8+ T cells. These results suggest that CD38 can modulate the mitochondrial activity and fate decision of CD8+ T cells, creating a link among CD38, aging, mitochondrial fitness and maintenance of T cell stemness. a, Levels of CD38 in bulk CD8+ T cells upon isolation or 3 days after activation (n = 5 biologically independent samples). b, Representative TMRM histograms of CD38hi and CD38low population in young and old CD8+ T cells 3 days after activation. c, Proportion of TMRMhi cells in CD38hi and CD38low populations (n = 3 biologically independent samples). d, NAD/NADH ratio on day 7 in old CAR-T cells expanded with IL-7 and IL-15 and treated with NMN and/or 78c, a specific inhibitor of the NADase enzymatic activity of CD38 (n = 3 biologically independent samples). e, OCR on day 7 in old CAR-T cells expanded with IL-7 and IL-15 and treated with NMN and/or 78c were measured using a Seahorse XFe96 Analyzer. During this assay, mitochondrial fitness was assessed upon the sequential addition of oligomycin (oligo, adenosine triphosphate synthetase inhibitor), FCCP (mitochondrial membrane uncoupler) and rotenone + antimycin A (Rot/AA, complex I and III inhibitors, respectively). Seahorse data displayed in f are representative of three independent experiments (n = 6 technical replicates). h, CD62L levels within TCM populations (n = 5 biologically independent samples). In i–k, young and old CAR-T cells treated with or without NMN and 78c were adoptively transferred into mice bearing B16-HER2 tumors. i,j, Tumor growth (i) and tumor size at last time point (day 30) (j) (n = 9 young, n = 8 old and n = 8 old 78c + NMN). k, Number of transferred T cells (CD45.1+) found in spleen 32 days after tumor engraftment (n = 7 mice). In l–n, young and old CAR-T cells treated with or without NMN and 78c were adoptively transferred into mice bearing EL4-mCD19 tumors (n = 8 mice per group). l,m, Tumor growth (l) and tumor size at last time point (day 17) (m). n, Number of transferred T cells (CD45.1+) found in spleen 17 days after tumor engraftment. Data are presented as the mean values ± s.e.m. Statistical analysis was performed using an unpaired t-test (a,k,n), two-way ANOVA (c) or one-way ANOVA with multiple comparisons (d–m), as appropriate. However, CD38 is not the only enzyme impacting NAD cellular levels in aging. For instance, alterations in NAD synthesis pathways (that is, the salvage or de novo synthesis pathways) or the activation of NAD-consuming enzymes can strongly modulate NAD cellular levels28. In particular, poly(ADP-ribose) polymerases (PARPs) have been described as the major NAD-consuming enzymes in the cell29,46, where their activation has often been associated with aging because of the accumulation of age-dependent DNA damage. However, it is not known whether this phenomenon is present in aged CD8+ T cells and can affect NAD levels and mitochondrial metabolism. To this end, we first measured DNA damage by analyzing the phosphorylation levels of γH2AX and found that aged CD8+ T cells exhibited a significant increase (Extended Data Fig. 4b), accumulated more p-γH2AX than CD38low T cells (Extended Data Fig. These results suggest that the mitochondrial defects in aged CD8+ T cells derive from a multifactorial process impacting NAD homeostasis. Next, we aimed to elucidate whether restoration of NAD levels is sufficient to reinstate mitochondrial fitness and functionality of aged CAR-T cells. To this end, we used the small molecule 78c to specifically block the NADase activity of CD38 (ref. 47) and we combined it with NMN supplementation. We observed that, according to the previous data (Fig. 3g,h), NMN alone was not sufficient to increase NAD levels in aged CD8+ T cells (Fig. However, when combined with the CD38 inhibitor 78c, NAD levels were restored to levels seen in younger controls (Fig. Consequently, 78c + NMN supplementation during ex vivo expansion of CAR-T cells repaired the mitochondrial function of aged CAR-T cells, as shown by an increased spare respiratory capacity (SRC) and mitochondrial mass (Fig. Importantly, the effects of the 78c + NMN treatment were observed in aged T cells but not in younger controls, suggesting that this combination strategy is particularly beneficial in contexts where CD38 levels are aberrantly high. Furthermore, combining 78c and NMN improved the phenotype of the CAR-T infusion products, as shown by a heightened expression of CD62L within the TCM population (Fig. Similarly, inhibition of PARP activity using olaparib also rescued the NAD levels and mitochondrial activity of aged CD8+ T cells (Extended Data Fig. To assess whether the recovery of the mitochondrial fitness was sufficient to restore CAR-T cell functionality in vivo, we reinfused young and old CAR-T cells with or without 78c + NMN treatment in mice bearing HER2+ B16 tumors. We observed that, whereas the untreated aged CAR-T cells were unable to control tumor growth, aged CAR-T cells supplemented with 78c + NMN efficiently controlled tumor growth in line with young CAR-T cells (Fig. Importantly, treated aged CAR-T cells prolonged their long-term survival in vivo, as shown by a significant increase in the number of CD44+CD62L+TCF1+ CAR-T cells found in spleens 30 days after ACT (Fig. Supplementing young CAR-T cells with NMN + 78c during the ex vivo expansion did not result in improved tumor growth control (Extended Data Fig. 5a) or increased persistence in vivo (Extended Data Fig. 5b), indicating that NMN + 78c treatment restores CAR-T cell functionality specifically in the context of aging. Additionally, we modified the HER2 CAR construct to integrate a short hairpin RNA (shRNA) for CD38 (hereafter referred as HER2 CAR_shRNA CD38) as a strategy to modulate CD38 enzymatic activity by reducing its expression. Upon transduction, both young and old CD8+ T cells presented a highly significant downregulation of CD38 protein levels (Extended Data Fig. HER2 CAR_shRNA CD38 T cells generated from young CD8+ T cells and expanded for 7 days in the presence of IL-7 and IL-15 showed a similar TCM phenotype. Interestingly, downregulation of CD38 levels in old CAR-T cells resulted in a worsened expansion of TCM cells, even when expanded in the presence of NMN (Extended Data Fig. To assess their functionality, we transferred either old CAR_shRNA CD38 T cells, old CAR_shRNA CD38 T cells expanded with NMN or old control CAR-T cells expanded with NMN + 78c into mice bearing HER2+ B16 tumors. CD38 downregulation led to an increased number of CAR-T cells in the spleens 30 days after ACT (Extended Data Fig. However, the control CAR-T cells supplemented with NMN + 78c but not old CAR_shRNA CD38 T cells were able to successfully control tumor growth (Extended Data Fig. These results underline the importance of specifically targeting the NADase activity of CD38 and suggest that the recovery of CAR-T cell functionality through CD38 is NAD dependent. To validate our findings in alternative CAR-T cell models, we extended our study using anti-CD19_28z (ref. CD19-directed CAR-T cells were generated from young and old CD8+ T cells (CD45.1+) and were adoptively transferred into mice (CD45.2+) engrafted with EL4 lymphoma cancer cells overexpressing mouse CD19 (mCD19) (Fig. Tumor control capacity was monitored over time. Similarly to the B16-HER2 model, young CAR-T cells were able to successfully control tumor growth, whereas old CAR-T cells failed (Fig. Importantly, old CAR-T cells expanded in the presence of NMN and 78c recovered their functionality in vivo, as shown by improved tumor growth control (Fig. Analysis of the spleens at endpoint (day 17) further revealed an increased number of CD45.1+ T cells, indicating improved persistence (Fig. Overall, these data demonstrates that NAD metabolism, which is greatly compromised during aging, is a key determinant of CAR-T cell successful responses. Combined strategies to boost NAD levels, such as NAD precursors together with a CD38 or PARP inhibitor, rejuvenates aged CAR-T cells, ultimately leading to improved therapeutic efficacy. To assess the influence of aging and NAD metabolism on CAR-T clinical application, we reanalyzed recently published anti-CD19 CAR-T data from 31 participants with large B cell lymphoma (LBCL) (15 responders (R) and 16 nonresponders (NoR))49. Importantly, we found that older participants had inferior responses (Fig. Moreover, single-cell transcriptomic analysis on baseline peripheral blood mononuclear cells (PBMCs) of 20 participants with LBCL (10 R and 10 NoR) revealed differences in NAD metabolic signatures of CD8+ T cells from R and NoR participants (Fig. These differences were also detected in natural killer (NK) cells but not in other immune cells (Extended Data Fig. a, Age of the participants at the time of diagnosis of R (n = 16) and NoR (n = 15) participants with LBCL to anti-CD19 CAR-T cell therapy. b,c, NAD metabolism (Gene Ontology: 0019674) signature score (b) and CD38 expression levels (c) in baseline PBMCs from R (n = 10) and NoR (n = 10) participants to anti-CD19 CAR-T cell therapy. e, Mitochondrial activity in CD38hi and CD38low populations of CD8+ T cells from PBMCs derived from participants with melanoma, as assessed by TMRM staining (n = 10). f, Schematic representation of experimental setup to target NAD metabolism in human anti-CD19 CAR-T cells generated from young (<28 years old) and elderly (>66 years old) healthy donors. g, OCR curve of young and old CAR-T cells with or without NMN and 78c treatment analyzed on day 10 with Seahorse XFe96. In h–l, PBMCs from young and old participants with melanoma were used to generate hCD19_BBz CAR-T cells, which were expanded for 10 days in the presence of IL-2, IL-7 and IL-15 with or without 78c and NMN treatment. h, Schematic representation of experimental setup to target NAD metabolism in human anti-CD19 CAR-T cells generated from young (~40 years old) and elderly (~80 years old) participants with melanoma. i, Representative CCR7 and CD45RA dot plots, as assessed by flow cytometry. k, hCD19-BBz CAR-T cells were cocultured with NALM6–GFP cells in a 1:4 effector-to-target ratio to determine killing capacity using Incucyte. Here dots represent technical replicates from one of two independent experiments. Data are presented as the mean values ± s.e.m. Statistical analysis was performed using an unpaired t-test (a–d), paired t-test (e,g,j) or one-way ANOVA (l), as appropriate. Following our previous analysis on NAD-related pathways, we focused on elucidating how CD38 is involved in CAR-T cell outcomes. We found that CD8+ T cells from NoR participants presented a tendency to express higher Cd38 baseline levels (Fig. Similarly to the NAD signature, Cd38 levels were also lower in the NK cells of R participants (Extended Data Fig. In addition, using a transcriptomic atlas (https://tanlab4generegulation.shiyapps.io/Tcell_Atlas/)50 of premanufactured T cells from 71 participants with B-ALL, we found that increased Cd38 baseline levels across several T cell subtypes dampened CAR-T cell persistence upon infusion (Extended Data Fig. 6e), suggesting that CD38 is associated with the stemness potential of CAR-T cells. Previous investigations characterized CD38 as a marker of terminally exhausted T cells in both murine (Extended Data Fig. 7f,g) and human (Extended Data Fig. Consistently, TILs of NoR participants to immune checkpoint blockade (ICB) accumulate CD8+ T cell clusters that present an exhausted signature and higher Cd38 levels51, a phenomenon also observed in CD4+ T cells and myeloid cells (Extended Data Fig. Our findings, however, add to CD38 a role as a non-exhaustion-related predictive marker of CAR-T cell persistence and efficacy. Of note, other NAD-associated pathways such as DNA damage were also predictive of the response (Extended Data Fig. 7j) suggesting that general NAD signatures in baseline PBMCs could be used to predict CAR-T cell responses. To test whether NAD metabolism could also be targeted to rejuvenate human CAR-T cells, we first analyzed PBMCs derived from participants with melanoma and found an age-dependent increased expression of CD38 in CD8+ T cells (Fig. 5d) but not in CD4+ T cells (Extended Data Fig. To determine whether CD38 expression was altering mitochondrial activity, we compared TMRM staining in CD38hi and CD38low CD8+ T cells and observed that mitochondrial potential was particularly restricted in those cells expressing higher levels of CD38 (Fig. Next, we generated human CD19 CAR-T cells containing a 41BB costimulatory domain using PBMCs derived from young (<30 years old) and old (>65 years old) healthy donors, expanded them in the presence of IL-2, IL-7 and IL-15 and compared their mitochondrial fitness (Fig. CAR-T cells generated from older donors displayed a significant reduction in SRC (Fig. Remarkably, old CAR-T cells supplemented with 78c and NMN during ex vivo expansion were able to revert their mitochondrial defects and reestablish a mitochondrial profile of young CAR-T cells (Fig. These results suggest that age-associated metabolic defects in CAR-T cells are also observed in a human setting and can be reverted by modulating NAD levels. To further investigate the importance of aging and NAD metabolism in a clinical setting, we analyzed PBMCs derived from participants with melanoma and non-small cell lung cancer. We showed that older participants gradually accumulate C–C chemokine receptor type 7 (CCR7)lowCD45RAlow T cells while losing CCR7hiCD45RAhi T cells (Extended Data Fig. To test whether these phenotypic differences could also be observed by the end of the expansion of CAR-T cells, we generated human CAR-T cells using PBMCs from young and old participants with melanoma using the same hCD19_BBz model (Fig. We analyzed the phenotype of CAR-T cells and indeed observed that old CAR-T cells had a lower proportion of CCR7hiCD45RAhi T cells when compared to younger controls. Treatment with NMN + 78c was able to increase the proportion of this T cell subset (Fig. Similarly to the CAR-T cells generated from healthy donors, CAR-T cells from older participants with melanoma presented an impaired mitochondrial function that was reverted upon expansion with NMN + 78c (Extended Data Fig. Of note, these differences were also observed in a human CD19 CAR-T cell model containing a CD28 costimulatory domain (Extended Data Fig. Old hCD19_BBz CAR-T cells expanded using only IL-2 did not present metabolic recovery upon NMN + 78c supplementation, while showing an increased proportion of CCR7hiCD45RAhi cells (Extended Data Fig. Lastly, to investigate whether the metabolic and phenotypic changes translated into recovery of the functionality of human CAR-T cells, we cocultured CAR-T cells with NALM6–GFP and tracked their killing capacity upon multiple rechallenges performed every 2–3 days. We observed few differences in killing capacity between young and old CAR-T cells at early stages (Fig. However, old CAR-T cells lost their tumor killing capacity upon multiple rechallenges, whereas young CAR-T cells were able to maintain it. Importantly, old human CAR-T cells treated with NMN + 78c recovered their functionality, as shown by a prolonged maintenance of killing capacity upon multiple rechallenges (Fig. Overall, these data indicate that age determines the phenotype, functionality and metabolic status of CAR-T cells and manipulating NAD metabolism can be used as a strategy to boost their function. Altogether, our data demonstrate that the age-dependent NAD decline leads to mitochondrial defects and loss of stem-like properties in T cells, ultimately resulting in CAR-T cell failure. Moreover, we establish the restoration of NAD cellular levels as a strategy to recover mitochondrial function and rejuvenate CAR-T cells in the context of aging. Several drivers of aging (for example, genomic instability, epigenetic alterations, chronic inflammation or mitochondrial dysfunction) are also common drivers of tumorigenesis, which make aging the first risk factor associated with cancer incidence52. In this context, the field of immuno-oncology has greatly expanded during the last decade; however, few studies have investigated how aging impacts immunotherapy efficacy. In the context of ICB, data acquired from preclinical models and clinical trials are currently unclear and contradictory53. In mice, some studies have shown decreased response to anti-PD1 or anti-cytotoxic T lymphocyte-associated protein 4 therapy in aged mice54,55, while other studies have documented an intact or even superior response to PD1 and its ligands56. These discrepancies might be explained by the fact that the expression of PD1 and its ligands is altered during aging in a cell-specific and organ-specific manner and aging affects mutagenesis burden in the tumor, thus influencing T cell infiltration in the TME57. Overall, in ICB, there are several factors independent of the intrinsic effect of aging on T cells that can determine the response to therapy. Indeed, although it is just beginning to be appreciated24, to date, no evidence supports the notion that aging might be an important limiting factor of CAR-T cell therapy efficacy in the clinic. In this Article, we demonstrated that CAR-T cells generated from aged mice display qualitative defects associated with an inability to maintain stem-like properties. We show that the lack of stem-like properties arises from mitochondrial dysfunction derived from the decline in NAD cellular levels and its recovery is sufficient to rejuvenate the functionality of aged CAR-T cells. Moreover, we provide clinically relevant data showing that NAD metabolic pathways can be both targeted and used as predictive markers of CAR-T cell therapy efficacy. The role of NAD metabolism in T cell function and antitumor responses has been widely explored. A report from Chatterjee et al. (2018) described a potent antitumor T helper 1 and 17 hybrid cell that was able to maintain effector functions while persisting long-term in vivo21. Interestingly, these cells were dependent on a higher NAD-dependent activity of the histone deacetylase SIRT1 (ref. Other studies have also applied NAD-boosting strategies to prevent T cell exhaustion23 or ameliorate CAR-T cell and TIL therapy20,58, providing promising results in preclinical models. As a result, the use of NAD precursors, such as NR or NMN, is gaining relevance in the field of immunotherapy. However, a limitation of current preclinical studies is the lack of aged mice included within the experimental design, which might lead to real-world discrepancies and difficulties to move from bench to bedside. For example, the administration of NAD precursors as a nutritional supplement has been included in several clinical trials to treat cardiovascular, neurodegenerative and metabolic diseases but their efficacy has been limited when applied in older subjects. In our study, we found that the sole administration of NAD precursors is indeed insufficient to improve the fitness of aged cells. These findings emphasize the need to find alternative or complementary strategies to increase NAD cellular levels and benefit from its effects. For CD8+ T cells, we explored the CD38 inhibitor 78c in combination with NAD precursors but we cannot exclude other strategies based on additional mechanisms of NAD homeostasis disturbance (PARP inhibitors in combination with NAD precursors). Nonetheless, not all cell types upregulate CD38 and might benefit from their strategy. Other immune cells, such as macrophages, present decreased activity of quinolinate and nicotinamide phosphoribosyltransferases during aging, both involved in NAD synthesis pathways, and recovery of their activity reinstates macrophage functionality59,60. These results show the importance of maintaining NAD homeostasis during aging for optimal immune function and suggest a tailored NAD-boosting strategy depending on the cell type and context. Although we focused on the intrinsic defects of CAR-T cells during aging in this study, important factors to consider are the extrinsic barriers that might impede CAR-T cell efficacy. Aging also fosters an environment marked by the presence of senescent cells and systemic low-grade chronic inflammation, known as ‘inflammaging'. Several investigations have described changes in the composition of the TME with aging, including an accumulation of fibroblasts and immunosuppressive cell subsets associated with a senescence-associated secretory phenotype such as myeloid-derived suppressor cells and regulatory T cells. However, whether tumor initiation and progression are aggravated during aging is still unclear, as some reports have described faster tumor growth in aged mice while other studies support the opposite. Similarly, it is widely discussed whether senescence might have protumoral or antitumoral roles. Thus, further investigation is required to decipher the importance of an aged environment on the outcome of antitumor and immunotherapy responses. In conclusion, our study found that aging is an important limiting factor for CAR-T cell therapy. Specifically, aged T cells present reduced NAD cellular levels that are linked to decreased mitochondrial fitness, ultimately preventing the maintenance of stem-like properties of CAR-T cells and leading to deficient long-term survival in vivo and tumor growth control. These findings emphasize the importance of using aged models in the field of cancer immunology, which can uncover mechanisms of CAR-T cell failure that are often overlooked in preclinical studies, shedding light on novel strategies that can ameliorate CAR-T cell therapy. For all in vivo experiments, host C57BL/6 CD45.2+ female mice (8 weeks old) were purchased from EnVigo laboratories (C57BL/6OlaHsd). Donors and recipients of adoptive T cell transfers were sex-matched. Mice were housed at 22 °C with 55% relative humidity on a 12-h light–dark cycle. Mice were fed ad libitum with Safe-150 chow. All animal experiments were performed in the animal facility in Epalinges at the University of Lausanne (UNIL), as approved by the veterinary authorities of the canton of Vaud and performed in accordance with Swiss federal law (VD3572). EL4-mCD19 cells were a kind gift from M. L. Davila (Moffitt Cancer Center) and were cultured in RPMI 1640-Glutamax medium supplemented with 10% heat-inactivated FBS and 1% penicillin–streptomycin. HER2-directed CAR containing a 41BBz costimulatory domain was cloned in the MSGV retroviral transfer vector as described previously13,30. For retrovirus production, Phoenix ECO cells were transfected with HER2 CAR plasmid and pCL-Eco-packaging plasmid using TurboFect transfection reagent (LifeTechnologies) in OptiMEM medium (Thermo Fisher). After 48 h and 72 h, supernatants were recovered and virus was collected by ultracentrifugation (Beckman Avanti J-26). Spleens from wild-type CD45.1 or CD45.1 × CD45.2 mice were smashed through a 70-μm cell strainer. CD8+ T cells were purified using the EasySep mouse CD8+ T cell isolation kit (StemCell), according to the manufacturer's instructions. CD8+ T cells were plated at a concentration of 0.5 × 106 cells per ml and activated with Activator CD3/CD28 Dynabeads (Thermo Fisher) at a 2:1 bead-to-cell ratio in the presence of recombinant murine IL-2 (10 IU per ml; PeproTech). T cells were maintained in RPMI 1640-Glutamax medium supplemented with 10% heat-inactivated FBS, 1% penicillin–streptomycin, 5 μM 2-mercaptoethanol (Gibco) and sodium pyruvate (Gibco). T cells were transduced 24 h and 48 h after activation using 48-well plates precoated with RetroNectin (20 μg ml−1; Takara). After overnight coating at 4 °C, 48-well plates were blocked for 30 min with 2% BSA in PBS, followed by a PBS wash before adding the concentrated retroviruses. Retroviruses were centrifuged for 90 min at 2,000 rcf and 32 °C. Then, T cells were added on top of the viruses and centrifuged for 10 min at 300 rcf and 25 °C. On day 3, activation beads were removed and T cells were expanded using either mouse IL-2 or human IL-7 and IL-15 (10 ng ml−1; Miltenyi Biotec). T cell media and cytokines were replaced on day 5 and transduction efficacy was assessed on day 7. Metabolic and phenotype analyses, as well as ACT for in vivo experiments, were always performed on day 7 unless otherwise stated in the figure legend. Where stated, CAR-T cells were treated with NMN (1 mM; Sigma Aldrich), 78c (200 nM; Sigma Aldrich) and/or olaparib (5 μM; Lubio Science) on days 3 and 5 after activation. TCF1 (clone C63D9, 2203S, Cell Signaling, 1/200) was stained with an unconjugated antibody. An additional staining was performed with secondary goat anti-rabbit IgG (4412S, Cell Signaling, 1:250). For assessment of intracellular markers, cells were fixed and permeabilized using FoxP3 fixation and permeabilization buffer (00-5523-00, eBioscience). For live–dead discrimination, live/dead Aqua or live/dead NIR kits were used (L34957 and L10119, LifeTechnologies). To assess mitochondrial activity and mitochondrial size, cells were stained with TMRM (T668; 25 nM) and MitoTracker Green (M7514; 100 nM) for 30 min at 37 °C. For cytokine staining, T cells were restimulated using anti-CD3ε-coated plates for 4 h in the presence of brefeldin A (420601, BioLegend, 1/1,000). For pH2AX staining, cells were stained upon fixation and permeabilization using the PECy7-conjugated anti-H2A.X (S139) antibody for 1 h at 4 °C (613420, Biolegend). The samples were acquired using the CytoFLEX S (Beckman Coulter), CytoFLEX LX (Beckman Coulter) or Aurora (Cytek Biosciences). Data analysis was performed using FlowJo (version 10.9.0). After 9 days, mice were exposed to sublethal irradiation (5 Gy) followed by one round of intravenous CAR-T cell transfer (2–3 × 106 cells per mouse) performed on day 10. Before the transfer, mice were randomized to have comparative tumor volumes. After 6 days, mice were exposed to sublethal irradiation (5 Gy) followed by one round of intravenous CAR-T cell transfer (3 × 106 cells per mouse) performed on day 7. Before the transfer, mice were randomized to have comparative tumor volumes. Mice were monitored three times a week and tumor length (L; greatest longitudinal measurement) and width (W; greatest transverse measurement) were measured with a caliper. Mice were killed at endpoint by CO2 and, where indicated, tumors, spleens and lymph nodes were collected. As permitted by the Swiss federal law, a maximal tumor size of 1,000 m3 was reached. In some cases, this limit was exceeded on the last day of measurement and the mice were immediately killed. No statistical methods were used to predetermine sample sizes but our sample sizes are similar to those reported in previous publications23,61. Murine CD38 complementary DNA was synthesized and flanked with the restriction enzymes Not1 and SalI, which were cloned into a retroviral MSCV vector containing a Thy1.1 promoter. The sequence can be found in Supplementary Table 2. Young and old naive CD8+ T cells were sorted using the EasySep mouse naive CD8+ T cell isolation kit (StemCell). For the analysis, EM was performed as described previously62. Sorted cells were fixed in their culture medium with glutaraldehyde (EM Sciences) at a final concentration of 2.5% in phosphate buffer (0.1 M PB pH 7.4; Sigma) for 10 min at room temperature (RT). As noted previously, they were directly postfixed by a fresh mixture of glutaraldehyde 2.5%, osmium tetroxide 1% (EM Sciences) and potassium ferrocyanide 1.5% (Sigma) in PB for 1 h at RT. The samples were then washed three times in distilled water and spun down in low-melting-point agarose 2% in H2O (Sigma), left to solidify on ice, cut into 1-mm3 cubes and dehydrated in acetone solution (Sigma) at graded concentrations (30%, 40 min; 50%, 40 min; 70%, 40 min; 100%, 1 h twice). Ultrathin sections of 50 nm were cut on a Leica Ultracut (Leica Mikrosysteme) and picked up on a copper slot grid (2 × 1 mm; EM Sciences) coated with a PEI film (Sigma). Sections were poststained with uranyl acetate 2% (Sigma) in H2O for 10 min, rinsed several times with H2O followed by Reynolds lead citrate in H2O (Sigma) for 10 min and rinsed several times with H2O. Micrographs were recorded with a transmission EM instrument (Philips CM100, Thermo Fisher Scientific) with a TemCam-F416 digital camera (TVIPS). Image analysis and quantification were carried out using ImageJ software (version 2.16.0). The number of mitochondria per cell was quantified. For assessing mitochondrial cristae, each dot represents the crista number in one mitochondrion from one high-magnitude EM image of a live cell. OCRs were determined using a Seahorse Bioanalyzer XFe96. During sensor calibration, cells were incubated in a 37 °C non-CO2 incubator for 45 min. Data were analyzed using Seahorse Wave (version 2.4.3). To determine the killing capacity of murine CAR-T cells, the IncuCyte ZOOM system was used. Specifically, 104 B16-HER2 cells containing the red fluorescent protein mK2 were plated in flat-bottom 96-well plates. After 4 h of incubation, young or old CAR-T cells expanded with IL-2 or IL-7 and IL-15 were added on top at a 2:1 effector-to-target ratio. Upon coculture, plates were placed immediately in the IncuCyte ZOOM system for 3 days. Killing capacity was determined by quantifying the red area and normalizing to negative controls (that is, B16-HER2 mK2 cells cultured in the absence of CAR-T cells). To evaluate the killing capacity of human CAR-T cells upon multiple rechallenges, 103 NALM6–GFP cells were seeded in a flat-bottom 96-well plate and CAR-T cells were added at an effector-to-target ratio of 1:4. Subsequently, 105 NALM6–GFP cells were added every 2–3 days for repeated tumor challenges. The cytotoxic activity of CAR-T cells was monitored over 23 days by measuring the GFP signal density (objects per mm²) using the IncuCyte ZOOM system. The intracellular level of different metabolites was determined by performing LC–MS/MS analysis. For metabolite extraction, cell lysates were extracted by the addition of methanol and H2O (4:1), followed by homogenization with ceramic beads in the Cryolys Precellys 24 sample homogenizer. Dried extracts were resuspended in methanol and H2O before LC–MS/MS analysis. Raw LC–MS/MS data were processed using the Agilent Quantitative analysis software. Raw data can be found in Supplementary Table 1. To determine the mtDNA-tonDNA ratio, qPCR was performed using Power SYBR green master mix (Thermo Fisher) and analyzed on the 7900HT system (Applied Biosystems) to estimate the relative values for mtDNA (COX1) and nDNA (NDUFV1). CD8+ T cells were isolated and activated using plates coated with Ultra-LEAF anti-mouse CD3e (3 μg ml−1, Biolegend) and soluble Ultra-LEAF anti-mouse CD28 (1.5 μg ml−1, Biolegend). On days 7, 9 and 11, CD8 T cells were restimulated using CD3-coated plates. Where indicated, cells were treated with NR (1 mM) on days 7, 9 and 11. Phenotypic and functional readouts were performed on day 12. Human blood samples were obtained from healthy young donors (aged 25–28 years) and elderly donors (aged 66–70 years). To generate CAR-T cells, CD3+ T cells were negatively isolated using the RosetteSep human T cell enrichment cocktail (StemCell, 15061). For stimulation, T cells were cultured with CD3/CD28-activating Dynabeads (Gibco, 11132D) at a 1:1 bead-to-cell ratio. The following day, T cells were transduced with the hCD19_41BBz (FMC63 scFv) or hCD19_CD28z lentiviral vector at a multiplicity of infection of 4. Recombinant human IL-2 (Proleukin, Roche) or a cytokine combination of IL-2, IL-7 (Peprotech, 200-07) and IL-15 (Peprotech, 200-15) was added every other day to a final concentration of 30 IU per ml and 10 ng ml−1, respectively. All T cell functional assays were conducted in media without cytokines. For the generation of CAR-T cells from participants with melanoma, 2 × 105 frozen PBMCs from young (aged 40–44 years) and elderly (aged 79–84 years) donors were seeded in a round-bottom 96-well plate and processed as described above. Donations from healthy volunteers were approved by the Comission Cantonale d'Ethique de la Recherche Genève (CCER). Samples from participants with cancer were taken from a biobank supported by the study protocol CCER 2016-01237. Written informed consent was obtained from all individuals. An scRNA-seq dataset of human baseline PBMCs from 20 participants with LBCL was obtained from the Gene Expression Omnibus (GSE197268). Metadata on age and response to therapy were obtained from the supplementary material of the associated publication49. To exclude low-quality cells and outliers, the following quality control filters were applied on the scRNA-seq data: percentage of mitochondrial genes < 25%, number of detected genes = 300–5,000, number of UMIs = 500–20,000 and log10 genes per UMI > 0.6. Raw counts were normalized using a standard log1p normalization implemented in Seurat (version 5.0.1)63. Broad cell types for each scRNA-seq sample were predicted using the scGate tool and its default PBMC model64. CD8 T cell subtypes were annotated using ProjecTILs65 and a published reference map of human CD8 T cell subtypes66. Gene signatures for NAD metabolism and DNA repair were downloaded from MSigDB67 under the ‘Gene Ontology biological process' subset. On the basis of these gene sets, signature scores were calculated using the UCell method with default parameters68. Statistical comparisons (Wilcoxon test) were performed at the sample level by averaging signature scores for each participant and cell type. For each T cell subtype, we calculated the normalized average expression of CD38, HAVCR2, TOX and TCF7. All statistical analyses were performed using GraphPad Prism version 10.0.3. The sample size (n) is stated in each figure legend together with the statistical test adopted. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. scRNA-seq data from the previous study49 are publicly available from the Gene Expression Omnibus under accession number GSE197268. RNA-seq data from the previous study50 are publicly available online (https://tanlab4generegulation.shinyapps.io/Tcell_Atlas/). Source data are provided with this paper. Mehta, P. H. et al. T cell fitness and autologous CAR T cell therapy in haematologic malignancy. Cheng, J. et al. Understanding the mechanisms of resistance to CAR T-cell therapy in malignancies. Shah, N. N. & Fry, T. J. Mechanisms of resistance to CAR T cell therapy. Enhancing CD8 T-cell memory by modulating fatty acid metabolism. van der Windt, G. J. W. et al. Mitochondrial respiratory capacity is a critical regulator of CD8+ T cell memory development. van der Windt, G. J. W. & Pearce, E. L. Metabolic switching and fuel choice during T-cell differentiation and memory development. Klein Geltink, R. I. et al. Mitochondrial Priming by CD28. Chronic lymphocytic leukemia cells impair mitochondrial fitness in CD8+ T cells and impede CAR T-cell efficacy. Author Correction: Determinants of response and resistance to CD19 chimeric antigen receptor (CAR) T cell therapy of chronic lymphocytic leukemia. Rostamian, H. et al. A metabolic switch to memory CAR T cells: implications for cancer treatment. Alizadeh, D. et al. IL15 enhances CAR T cell antitumor activity by reducing mTORC1 activity and preserving their stem cell memory phenotype. The mitochondrial pyruvate carrier regulates memory T cell differentiation and antitumor function. Jaccard, A. et al. Reductive carboxylation epigenetically instructs T cell differentiation. Distinct signaling of coreceptors regulates specific metabolism pathways and impacts memory development in CAR T cells. Enforced PGC-1α expression promotes CD8 T cell fitness, memory formation and antitumor immunity. Katsyuba, E., Romani, M., Hofer, D. & Auwerx, J. NAD+ homeostasis in health and disease. & Auwerx, J. NAD+ metabolism and the control of energy homeostasis: a balancing act between mitochondria and the nucleus. Tullius, S. G. et al. NAD+ protects against EAE by regulating CD4+ T-cell differentiation. Wang, Y. et al. NAD+ supplement potentiates tumor-killing function by rescuing defective TUB-mediated NAMPT transcription in tumor-infiltrated T cells. Chatterjee, S. et al. CD38–NAD+ axis regulates immunotherapeutic anti-tumor T cell response. Wan, J. et al. De novo NAD+ synthesis contributes to CD8+ T cell metabolic fitness and antitumor function. Disturbed mitochondrial dynamics in CD8+ TILs reinforce T cell exhaustion. Beyond youth: understanding CAR T cell fitness in the context of immunological aging. Drijvers, J. M., Sharpe, A. H. & Haigis, M. C.The effects of age and systemic metabolism on anti-tumor T cell responses. López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. Camacho-Pereira, J. et al. CD38 dictates age-related NAD decline and mitochondrial dysfunction through an SIRT3-dependent mechanism. & Verdin, E. NAD+ metabolism and its roles in cellular processes during ageing. The NAD+/Sirtuin pathway modulates longevity through activation of mitochondrial UPR and FOXO signaling. Tschumi, B. O. et al. CART cells are prone to Fas- and DR5-mediated cell death. Pais Ferreira, D. et al. Central memory CD8+ T cells derive from stem-like Tcf7hi effector cells in the absence of cytotoxic differentiation. Siddiqui, I. et al. Intratumoral Tcf1+PD-1+CD8+ T cells with stem-like properties promote tumor control in response to vaccination and checkpoint blockade immunotherapy. Charmoy, M., Wyss, T., Delorenzi, M. & Held, W. PD-1+ Tcf1+ CD8+ T cells from established chronic infection can form memory while retaining a stable imprint of persistent antigen exposure. O'Sullivan, D. et al. Memory CD8+ T cells use cell-intrinsic lipolysis to support the metabolic programming necessary for development. Corrado, M. & Pearce, E. L.Targeting memory T cell metabolism to improve immunity. Desdín-Micó, G. et al. T cells with dysfunctional mitochondria induce multimorbidity and premature senescence. Impaired mitochondrial oxidative phosphorylation limits the self-renewal of T cells exposed to persistent antigen. Scharping, N. E. et al. Mitochondrial stress induced by continuous stimulation under hypoxia rapidly drives T cell exhaustion. Quinn, K. M., Palchaudhuri, R., Palmer, C. S. & La Gruta, N. L. The clock is ticking: the impact of ageing on T cell metabolism. Quinn, K. M. et al. Metabolic characteristics of CD8+ T cell subsets in young and aged individuals are not predictive of functionality. Targeting age-specific changes in CD4+ T cell metabolism ameliorates alloimmune responses and prolongs graft survival. The NAD-booster nicotinamide riboside potently stimulates hematopoiesis through increased mitochondrial clearance. Kar, A., Mehrotra, S. & Chatterjee, S.CD38: T cell immuno-metabolic modulator. & Zhou, Y. CD38: an important regulator of T cell function. Chini, C. C. S. et al. CD38 ecto-enzyme in immune cells is induced during aging and regulates NAD+ and NMN levels. Hurtado-Bagès, S., Knobloch, G., Ladurner, A. G. & Buschbeck, M. The taming of PARP1 and its impact on NAD+ metabolism. Tarrago, M. G. et al. A potent and specific CD38 inhibitor ameliorates age-related metabolic dysfunction by reversing tissue NAD+ decline. Parker, K. R. et al. Single-cell analyses identify brain mural cells expressing CD19 as potential off-tumor targets for CAR-T immunotherapies. Distinct cellular dynamics associated with response to CAR-T therapy for refractory B cell lymphoma. Chen, G. M. et al. Integrative bulk and single-cell profiling of premanufacture T-cell populations reveals factors mediating long-term persistence of CAR T-cell therapy. Defining T cell states associated with response to checkpoint immunotherapy in melanoma. López-Otín, C., Pietrocola, F., Roiz-Valle, D., Galluzzi, L. & Kroemer, G. Meta-hallmarks of aging and cancer. Han, S., Georgiev, P., Ringel, A. E., Sharpe, A. H. & Haigis, M. C. Age-associated remodeling of T cell immunity and metabolism. Padrón, Á. et al. Age effects of distinct immune checkpoint blockade treatments in a mouse melanoma model. Hurez, V. et al. Mitigating age-related immune dysfunction heightens the efficacy of tumor immunotherapy in aged mice. Kugel, C. H. et al. Age correlates with response to anti-PD1, reflecting age-related differences in intratumoral effector and regulatory T-cell populations. Immune checkpoint expression and relationships to anti-PD-L1 immune checkpoint blockade cancer immunotherapy efficacy in aged versus young mice. Huang, Y. et al. Inhibition of CD38 enzymatic activity enhances CAR-T cell immune-therapeutic efficacy by repressing glycolytic metabolism. Inflammatory macrophage dependence on NAD+ salvage is a consequence of reactive oxygen species-mediated DNA damage. Minhas, P. S. et al. Macrophage de novo NAD+ synthesis specifies immune function in aging and inflammation. Ginefra, P. et al. Urolithin-A promotes CD8+ T cell-mediated cancer immunosurveillance via FOXO1 activation. Mühlethaler-Mottet, A. et al. Low number of neurosecretory vesicles in neuroblastoma impairs massive catecholamine release and prevents hypertension. Dictionary learning for integrative, multimodal and scalable single-cell analysis. & Carmona, S. J. scGate: marker-based purification of cell types from heterogeneous single-cell RNA-seq datasets. Interpretation of T cell states from single-cell transcriptomics data using reference atlases. Andreatta, M. et al. Semi-supervised integration of single-cell transcriptomics data. The Molecular Signatures Database Hallmark gene set collection. Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. We thank F. Derouet (UNIL) for animal care. Flow cytometry analysis was performed at Flow Cytometry Facility at UNIL with the help of R. Bedel, F. Sala de Oyanguren, K. Blackney and J.-F. Mayol. EM analysis was performed with the help of J. Daraspe at the EM Facility at UNIL. Metabolomic analysis was performed with the help of M. Girotra, T. Cherbuin, O. Naveiras and J. Ivanisevic at the Metabolomic Unit at UNIL. We thank V. Dutoit, S. Davanture and V. Widmer from University of Geneva, B. Seijo, A. Mulvey and C. Jiménez Luna from UNIL and I. Buondonno from University of Torino for their technical support. The N.V. laboratory and this study were supported by a Leukemia Lymphoma Society TRP award (LLS 6692-25), the Swiss Cancer Research Foundation (KFS-4993-02-2020-R) and the Swiss National Science Foundation (SNSF 10.002.388). laboratory was supported by the Swiss Institute for Experimental Cancer Research Foundation and Fondation Lionel Perrier for this project. acknowledges support from the Swiss National Science Foundation (SNSF 31003A_179435 and Sinergia CRSII5_202302). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Open access funding provided by University of Lausanne. These authors contributed equally: Jana de Sostoa, Pierpaolo Ginefra. Helen Carrasco Hope, Pierpaolo Ginefra, Massimo Andreatta, Yi-Hsuan Chiang, Catherine Ronet, Jesús Corria Osorio, Ping-Chih Ho, Santiago J. Carmona, George Coukos & Nicola Vannini Ludwig Institute for Cancer Research, University of Lausanne, Epalinges, Switzerland Helen Carrasco Hope, Pierpaolo Ginefra, Massimo Andreatta, Yi-Hsuan Chiang, Catherine Ronet, Jesús Corria Osorio, Ping-Chih Ho, Santiago J. Carmona, George Coukos & Nicola Vannini Agora Cancer Research Center, Lausanne, Switzerland Jana de Sostoa, Massimo Andreatta, Jesús Corria Osorio, Santiago J. Carmona, George Coukos & Denis Migliorini Jana de Sostoa, Massimo Andreatta, Jesús Corria Osorio, François Kuonen, Santiago J. Carmona, George Coukos & Denis Migliorini Laboratory of Integrative and Systems Physiology, Institute of Bioengineering, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar All authors edited and reviewed the final manuscript. Correspondence to Helen Carrasco Hope, Denis Migliorini or Nicola Vannini. is an advisor to MetroBiotech International and Mitopower, who develop strategies to enhance NAD levels. is an inventor on patents related to CAR-T cell therapy, filed by the University of Pennsylvania and the University of Geneva, is the scientific cofounder of Cellula Therapeutics and is a consultant for Carthera, Servier Pharma, Limula and MPC Therapeutics. N.V. is a scientific advisor of Clinique La Prairie. The other authors declare no competing interests. Nature Cancer thanks Marco Davila, Brunie Felding and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Proportion of CD8+ Tn (CD44-CD62L+), Tcm (CD44+CD62L+) and Tem (CD44+CD62L-) cells in the spleens of young and old mice. On the right, representative CD44/CD62L dot plot (n = 5 biologically independent samples). (b) Proportion of Tcm cells (CD44+CD62L+) upon activation of young and old naïve CD8+ T cells and expansion under effector-like (IL-2) or memory-like (IL-7 IL-15) conditions (n = 10 biologically independent samples). (c) Mean fluorescence intensity (MFI) of TCF-1 within young and old Tcm populations expanded from naïve CD8+ T cells with IL-7 and IL-15 (n = 4 biologically independent samples). (d) Killing capacity against B16-HER2 tumor cells of young and old CAR-T cells expanded with IL-2 (n = 3 biologically independent samples). (e) Proportion of IFNγ+ and TNFα+ young and old CAR-T cells expanded with IL-2 upon rechallenge with B16-HER2 tumor cells (n = 4 biologically independent samples). (f) Proportion of IFNγ+ and TNFα+ young and old exhausted CAR-T cells generated upon chronic stimulation in vitro (n = 3 biologically independent samples). (g) MFI of IFNγ and TNFα within IFNγ+ and TNFα+ populations, respectively, of young and old exhausted CAR-T cells generated upon chronic stimulation in vitro (n = 3 biologically independent samples). (h-l) Levels of the exhaustion markers PD-1, TOX, TCF-1, LAG3 and TIM3 in young and old exhausted CAR-T cells (n = 3 biologically independent samples). (m) Growth of B16-HER2 tumors after ACT of young and old CAR-T cells expanded with IL-2 (n = 6 mice per group). (n) CAR-T cells (CD45.1+) generated from young or old mice were adoptively transferred into young or old recipient mice bearing B16-HER2 tumors. Data are represented as mean values +/- SEM. Statistical analysis was performed using two-way ANOVA (A-C), unpaired t-test (E, G and M) or one-way ANOVA with multiple comparisons (F, H-I and N), as appropiate. Gating strategy, from top left to bottom right, to identify CD44+CD62L+ CAR-T cells in spleens after tumor engraftment. (a) Mitochondrial ROS levels assessed by MitoSOX staining in young and old CD8+ T cells after 3 days of activation (n = 5 biologically independent samples). In (c), basal OCR and maximal OCR values are represented (n = 3 biologically independent samples, in C, dots represent technical replicates from one representative experiment). (d) Mitochondrial membrane potential, as assessed by TMRM staining, of Tn, Tcm and Tem populations gated within CD8+ T cells of splenocytes derived from young and old mice (n = 5 biologically independent samples). (e) Mitochondrial size, as assessed by MitoTracker Green staining, of Tn, Tcm and Tem populations gated within CD8+ T cells of splenocytes derived from young and old mice (n = 5 biologically independent samples). Naïve CD8+ T cells (CD44-CD62L+) were isolated from splenocytes of young, intermediate and old mice and stained with TMRM (f) or MitoTracker Green (g) (n = 4 biologically independent samples). In (h), representative TMRM/MitoTracker Green dot plots are shown (n = 4 biologically independent samples). Data are represented as mean values +/- SEM. Statistical analysis was performed using unpaired t-test (A and C), two-way ANOVA (D and E), or one-way ANOVA with multiple comparisons (F and G), as appropiate. (a) Levels of CD38 within Tn, Tcm and Tem populations in young and old CD8+ T cells isolated from splenocytes (n = 5 biologically independent samples). (b) Proportion of Tcm cells on day 7 upon activation of naïve CD8+ T cells derived from young mice (8 weeks old) transduced with a CD38 OE vector and expanded under effector (IL-2) or memory (IL-7 IL-15) polarizing conditions (n = 9 biologically independent samples). (c) mtDNA/nDNA ratio in young CD8 + T cells overexpressing CD38 after 7 days of expansion with IL-7 and IL-15 (n = 9 biologically independent samples). (d) DNA damage, as assessed by pH2AX staining, in freshly isolated CD8+ T cells from young and old mice (n = 4 biologically independent samples). (e) pH2AX levels in CD38hi and CD38low CD8+ T cells from young and old mice (n = 4 biologically independent samples). HER2-directed CAR-T cells were generated from old mice and expanded in the presence of IL-7 and IL-15 ± NMN ± 78c ± Olaparib (PARPi). On day 7, NAD/NADH ratio (f) and SRC as assessed by Seahorse XFe96 Analysis (g) were determined (n = 3, in (G) dots represent technical replicates from one representative experiment). Data are represented as mean values +/- SEM. Statistical analysis was performed using unpaired t-test (A and D), paired t-test (B and C), two-way ANOVA (E), or one-way ANOVA with multiple comparisons (F and G), as appropiate. Her2-directed CAR-T cells were generated using isolated CD8+ T cells from young mice (8 weeks old), expanded in the presence of IL-7 and IL-15, with or without NMN and 78c and adoptively transferred at day 10 into mice bearing B16-HER2+ tumors. Tumor growth is displayed in (a), while number of transferred CAR-T cells (CD45.1+Thy1.1+) found in spleen after 41 days post-tumor engraftment is displayed in (b) (n = 6 Young, n = 7 Young NMN + 78c). Data are represented as mean values +/- SEM. Statistical analysis was performed using unpaired t-test. On day 3, after transduction, CAR-T cells were expanded in the presence of IL-7 and IL-15 until day 7. In (a), protein levels of CD38 as assessed by flow cytometry on day 7. A representative histogram is shown in (b) (n = 3 biologically independent samples). (c) CD44/CD62L plots of young and old Her2 CAR T cells (Ctrl vs shRNA CD38) at day 7 of expansion. For (d), (e) and (f), old Her2 CAR-T cells, old Her2_shRNA CD38 CAR-T cells (expanded in the presence of NMN) or old Her2 CAR-T cells expanded in the presence of NMN + 78c were adoptively transferred at day 10 into mice bearing B16-HER2 tumors (n = 8 per group). Number of transferred CAR-T cells (CD45.1+) found in spleen after 32 days post-tumor engraftment is displayed in (D). Tumor growth is displayed in (E), while tumor size at endpoint is shown in (F). Data are represented as mean values +/- SEM. Statistical analysis was performed using one-way ANOVA. NAD metabolism signature score in CD8+ T cell subtypes (a) and other immune cells (b) from baseline PBMCs of responder and non-responder patients to anti-CD19 CAR-T cell therapy. Analysis was performed using the UCell method68 on recently published scRNAseq data from Haradhvala et al (2022)49. From the same dataset, CD38 expression levels in CD8+ T cell subsets and other immune cells are shown in (c) and (d), respectively (n = 15 R, n = 16 NoR). (e) Comparison of CD38 levels in baseline PBMCs of B cell malignancies patients presenting long-term ( > 6 months) vs short-term ( < 6 months) CAR-T cell persistence upon infusion, using publicly available RNAseq data from Chen et al (2021) (n = 71 patients)50. (f-g) Expression levels of Tcf7, Tox, Havcr2 and Cd38 on a reference map of mouse CD8 T cells, in UMAP space (F) and averaged by T cell subtype (G). (h) Comparison of TCF7, TOX, HAVCR2 and CD38 expression levels in human CD8 T cell subsets66. (i) CD38 expression levels in myeloid cell, CD4+ T cells and CD8+ T cells of tumor immune infiltrates of responder (n = 17) and non-responder (n = 31) patients to ICB51. (j) DNA damage signature score in immune cells from baseline PBMCs of responder and non-responder patients to anti-CD19 CAR-T cell therapy (n = 10 patients)49. (k) CD38 protein levels in CD4+ T cells analyzed in PBMCs derived from young (39-49 years old) and old (76-84 years old) melanoma patients (n = 5) (l) Mitochondrial size, as assessed by MitoTracker Green staining, of hCD19_BBz CAR-T cells generated from PBMCs derived from young and old healthy donors and expanded in the presence of IL-2, IL-7 and IL-15 + /- NMN and 78c. Data are represented as mean values +/- SEM. Statistical analyses were performed using unpaired t-test or one-way ANOVA, as appropiate. (a) Phenotype of PBMCs derived from young and old melanoma and non-small cell lung cancer (NSCLC) was assessed by flow cytometry. On day 10, mitochondrial activity was assessed using Seahorse XFe96 Analyzer (n = 3 patients). (C-E) hCD19_28z CAR-T cells were generated from young (39-49 years old) and old (76-84 years old) melanoma patients and expanded using IL-2, IL-7 and IL-15 (n = 4 patients) (c) The proportion of CCR7hiCD45RAhi CAR-T cells within the CD8+ T cell population was assessed by flow cytometry. (d) Mitochondrial activity was assessed performing a MitoStress Test with Seahorse XFe96 Analyzer. (e) SRC of control and treated (78c + NMN) old CAR-T cells. (F-H) hCD19_BBz CAR-T cells were generated from young (39-49 years old) and old (76-84 years old) melanoma patients PBMCs and expanded using IL-2 only (n = 5 patients) (f) The proportion of CCR7hiCD45RAhi CAR-T cells within the CD8+ T cell population was assessed by flow cytometry. (g) Mitochondrial activity was assessed performing a MitoStress Test with Seahorse XFe96 Analyzer. (h) SRC of control and treated (78c + NMN) old CAR-T cells. Data are represented as mean values +/- SEM. Statistical analyses were performed using paired t-test. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Hope, H.C., de Sostoa, J., Ginefra, P. et al. Age-associated nicotinamide adenine dinucleotide decline drives CAR-T cell failure. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64815804/crispr-therapy/'>Scientists Edited Genes Inside a Living Person for the First Time—and Saved His Life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Life's ability to successfully copy three billion distinct letters in the human genome is an absolute biological wonder—but sometimes, mistakes are made. For all of human history, a person born with such a disorder likely had to live with the condition, and depending on the defect, those lives could be brutally short. In a groundbreaking announcement, detailed in a study published in the New England Journal of Medicine, scientists, doctors, and specialists from institutions around the U.S.—including the Children's Hospital of Philadelphia, University of California-Berkeley, and Penn Medicine—successfully saved the life of a newborn patient named KJ, who had been born with a rare genetic disorder. To pull off this incredible medical feat, doctors employed the world's first custom in vivo (i.e. inside a living organism, rather than in a petri dish) CRISPR gene therapy. This technique, developed over decades thanks to U.S.-funded medical research, could help alleviate painful lives for millions of people born every year with now-fixable genetic disorders. “Years and years of progress in gene editing and collaboration between researchers and clinicians made this moment possible, and while KJ is just one patient, we hope he is the first of many to benefit from a methodology that can be scaled to fit an individual patient's needs,” Children's Hospital of Philadelphia's Rebecca Ahrens-Nicklas, a co-author of the study, said in a press statement. The details of this incredible medical intervention play out like a made-for-tv medical drama, but the stakes were incredibly real and deadly serious. A week after his birth, doctors noticed something wasn't quite right with KJ. After ruling out a few possibilities, they stumbled across the unfortunate answer—a rare genetic disorder called severe carbamoyl phosphate synthetase 1 (CPS1) deficiency that affects only one in every 1.3 million babies. This can have deadly consequences, impact brain development, and wreak havoc on the liver. Usually, the treatment for a disorder like this is a liver transplant, but that was not an option for the infant boy, who was still too young to be considered for the surgery. So, once arriving at a diagnosis, Ahrens-Nicklas contacted a gene-editing specialist at the University of Pennsylvania named Kiran Musunuru and “the clock start[ed]in my mind,” he later told The New York Times. Working with a team of specialists across the country for six months, Ahrens-Nicklas and Musunuru developed a targeted gene therapy to fix KJ's specific variant of CPS1. Meanwhile, KJ was kept under medical surveillance at the hospital and subsisted on a diet completely devoid of protein to avoid making his condition worse. Although it's unknown if he will eventually still need a liver transplant, doctors can now safely say that a human life has been saved thanks to the world's first bespoke in vivo gene therapy—a huge testament to decades of a research and experimentation. “We want each and every patient to have the potential to experience the same results we saw in this first patient, and we hope that other academic investigators will replicate this method for many rare diseases and give many patients a fair shot at living a healthy life,” Musunuru said in a press statement. “The promise of gene therapy that we've heard about for decades is coming to fruition, and it's going to utterly transform the way we approach medicine.” Every Human Body Gives Off an Invisible Glow Is the Key to Human Immortality This Sea Creature? A Surprising Reason Why Your Pee May Turn Red</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/can-we-refreeze-the-arctics-ice-scientists-test-new-geoengineering-solutions/'>Inside the Bold Geoengineering Work to Refreeze the Arctic's Disappearing Ice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers are trying to rebuild sea ice above the Arctic Circle so it can reflect the sun's warming rays, slowing climate change A team with Real Ice prepares to drill through sea ice in the Canadian Arctic, having already flooded a nearby patch (darker blue) to thicken it. This story was produced in partnership with the Pulitzer Center's Ocean Reporting Network. A haze of ice crystals in the air created a halo around the low sun as three snowmobiles thundered onto the sea ice on a February morning in far northern Canada. Wisps of snow blew across the white expanse. It was –26 degrees Celsius as we left Cambridge Bay, an Inuit village in a vast archipelago of treeless islands and ice-choked channels. This temperature was relatively warm—six degrees C above average. Scientists predict that within the next 15 years this ice cap will disappear in summer for the first time in millennia, accelerating global warming. The U.K. company Real Ice, whose heavily bundled team was bouncing around on the other two snowmobiles ahead of mine, hopes to prevent that outcome with an effort that has been called extremely ambitious, insane or even dangerous. At a spot seven kilometers from the village, Real Ice co-founder Cían Sherwin, an Irishman with a red beanie and scraggy goatee, hopped off his snowmobile and started drilling with a long electric auger. Sherwin lowered an aluminum pump, which looked like a large coffee urn attached to a curved rubber hose, through the hole. After a few seconds water began pouring out of the hose, spilling onto the ice in an ethereal shade of blue. As it congeals, “the water acts almost like lava,” Sherwin said. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Thin, broad sheets of ice expand from the ice cap's edges in winter, when it's dark and cold, and melt away in summer, when the sun shines 24 hours a day. Ocean water, in contrast, absorbs 90 percent of sunlight. If the ice starts disappearing entirely in summer, global temperatures could rise an extra 0.19 degree C by 2050. Real Ice is trying to thicken seasonal ice so it lasts longer into the warm months, keeping the planet cool. Sherwin hopes pumping could someday refreeze a million square kilometers of both seasonal and multiyear ice—an area the size of Texas and New Mexico combined and about a fifth of what's now left in summer—to stop the ice cap's death spiral. All it would take, Real Ice says, is half a million ice-making robots. Polar geoengineering on such an enormous scale could help slow warming until the world finally weans itself off coal, oil and natural gas. The researchers at Real Ice argue we no longer have any option but to try; studies suggest that even slashing fossil-fuel use may not save summertime sea ice. Cambridge Bay, which British explorers named for a 19th-century Duke of Cambridge, is a town of 1,800 mostly Inuit inhabitants located across from the Canadian mainland on Victoria Island, one of the world's largest islands. Cambridge Bay lies along the Northwest Passage, an icy sea route between Europe and Asia sought by explorers for 400 years. Franklin's two ships, Erebus and Terror, were trapped in the polar sea ice that surges down toward Cambridge Bay in winter, buckling into ridges up to 10 meters high. All 129 men onboard died of cold, starvation or disease. These days cruise ships coast through the passage every year, often visiting grave sites of Franklin expedition members. The Inuit call Cambridge Bay Ikaluktutiak, meaning “good fishing place.” For millennia their nomadic ancestors came here to fish Arctic char, a silvery-orange cousin of the brook trout. The U.S.S.R. discussed destroying sea ice with coal dust or explosions and detonated three nuclear devices to try to excavate an Arctic canal. In the U.S., physicist Edward Teller's Project Plowshare nearly got approval to gouge out a harbor in Alaska with atomic bombs. Some scientists and entrepreneurs are focused on dispersing sulfate particles in the stratosphere to block sunlight, which could lessen heating but also disrupt global weather patterns such as the South Asian monsoon. Mexico recently announced a ban on this solar geoengineering after Silicon Valley start-up Make Sunsets launched two balloons full of sulfur dioxide there. In Iceland and India, Silicon Valley nonprofit Bright Ice Initiative has scattered tiny glass beads on glaciers to try to reflect more sunlight and slow the melting. Chinese agencies have blown chemical smoke into clouds with rockets, planes, drones and chimneys to provoke snowfall over glaciers on the Tibetan plateau. Researchers in Scandinavia are developing giant curtains that could be anchored to the seabed to block warm ocean water from melting the undersides of ice shelves in Antarctica. Billions of dollars would be needed to affect the climate. The idea for thickening ice came from outer space. At a 2012 conference a fractious forum about global warming soured Arizona State University astrophysicist Steve Desch's hopes for quick climate action. Desch, who studies icy bodies such as Pluto's moon Charon, wondered whether we could buy time by making ice in the Arctic. The problem is that sea ice freezes from below. In 2016 Desch published a paper proposing that wind-powered pumps could thicken sea ice by pulling up water from below and spraying it across the top. Around that time, students at Bangor University in Wales were inspired by a documentary on the Arctic to construct a “re-icing machine,” an ungainly spindle of hoses that twirled like a lawn sprinkler. Encouraged by Desch's paper, he and London entrepreneur Simon Woods founded Real Ice in 2022 to see whether sea-ice thickening could scale up. They eventually recruited Desch and several sea-ice scientists as advisers. The company put its first water onto ice in Nome, Alaska, in January 2023, ditching the sprinkler for a commercial pump. As water pooled, the edges crept outward, soaking into the pockmarked snow, which was up to 25 centimeters (10 inches) thick and crusty like hardened white frosting. Within hours the pool would coagulate into electric-blue slush, like a gas station Slurpee. Under a dusting of snow lay flat gray ice. Inuit hunter David Kavanna guides the team to prospective sites seven kilometers away from Cambridge Bay. The ice would thicken further in coming weeks. Because snow is a better insulator than ice—this quality is why igloos work so well—flooding and freezing the snow could allow more cold to penetrate to the ice's underside, creating more ice. After Real Ice thickened 4,100 square meters of ice here in winter 2023–2024, the crew came back in May 2024 to find a significant increase. “Ice growth from below—that's the really efficient part,” Woods told me as he drilled a measuring hole at another refrozen site. But snow is also a better mirror than ice, which could complicate the picture. Real Ice would need snow to accumulate in spring to replenish the snow it flooded in winter, or the process could increase melting. So far this doesn't seem to be happening. On another morning Woods put a hollow red barrel on the drill and bored into the ice at a refrozen site to extract an ice core about as long and thick as his arm. On a different morning I snowmobiled with University of Alaska Fairbanks marine biologist Brendan Kelly to a low ridge formed by two enormous plates of ice pushing together. A polar science adviser in President Barack Obama's administration, Kelly has studied seals and polar bears for more than four decades. In that time he's also watched fossil-fuel emissions march steadily upward. So despite his discomfort with geoengineering, he agreed to advise Real Ice. But as we crunched on foot along the snow caking the ridge, Kelly stopped to point out an Arctic fox footprint. In spring, ringed seals claw holes through snowdrifts. Kelly scooped at the firm snow, tiny icicles swaying from his white mustache, but couldn't find a lair. They excavate dens in larger drifts to warm their cubs, which are born the size of a guinea pig. Most Arctic snow tends to fall in late autumn. It's unknown whether enough new snow would build up after wintertime ice thickening for bears and seals to make dens in spring. Of course, polar bears and seals are already expected to decline as their sea-ice habitat melts away. Is Real Ice doing more harm than good by pumping seawater into this environment, melting the snow? Across two months last winter, Real Ice pumped water through almost 200 holes. The ice cap is losing 300,000 times that area every year. The Sant'Anna School of Advanced Studies in Pisa, Italy, is developing an underwater drone two meters long that will bore through ice from below with a heated pipe and start pumping water up through it. Real Ice hopes to test a prototype this year, says co-CEO Andrea Ceccolini, an Italian computer scientist and investor who joined the company in 2022. A swarm of 50 drones would melt holes in minutes and pump water as their infrared cameras monitored the progress. Technicians on a floating or onshore hub would swap out the drones' batteries, plugging the old ones into chargers powered by wind turbines or by green hydrogen or ammonia brought in by ship. Tapping into electricity from Canada's Nunavut region would contribute to climate change because most of it is generated from diesel fuel. The ultimate goal of thickening one million square kilometers of sea ice would take an estimated 500,000 drones, which would consume two terawatt-hours of electricity and require 20,000 people to service them, according to rough math Ceccolini has done. The drones would vastly exceed the 3,800 Argo robot sensors circulating in oceans worldwide, and drone experts say a revolution in battery technology would be needed. How much global warming could be countered through sea-ice preservation depends on numerous variables affecting sunlight and melt dynamics. Preserving a million square kilometers of sea ice for one additional summer month would cool Earth as much as removing 930 million metric tons of carbon dioxide from the atmosphere over 20 years, Real Ice estimates. For these results, $10 billion is actually cheap, Ceccolini says, and the cooling would be immediate. Capturing that much CO2 from the air with existing machines would currently cost at least $465 billion. For perspective, humanity emits 910 million tons of CO2 every eight days, with no end in sight. Thickening sea ice is a Band-Aid “while you cure the patient—the planet—properly,” Ceccolini says. Every day in Cambridge Bay, after three or four hours of flooding, the team used an ice ax to hack each pump out of its hole. Fat white frost flowers formed on top of the solidifying surface. It was hard to envision hundreds of thousands of drones popping through the ice day after day, all winter long, for decades. In winter and spring, Inuit residents snowmobile hundreds of kilometers over it to go ice fishing and hunt musk ox and seals, local hunter Brandon Langan told me in his living room, a black musk ox hide hanging behind him. Two out of every three meals are fish or game. “Sea ice, to us, it's life,” Langan said. Ice loss has diminished the local caribou herd by 90 percent; the animals go hungry waiting to cross, and when bottled up for too long on the shore, they're easier prey for wolves. Hundreds of caribou have drowned after breaking through the ice. One hunter who had previously been a guide for Real Ice told me at the cultural center in the high school library that he hopes ice thickening could rejuvenate game populations. Throughout history scientists and explorers often ignored Indigenous knowledge of the Arctic. Franklin was dead, and the remaining explorers traded the Inuit beads for seal meat—they apparently didn't know how to hunt seal themselves. Later, native people found mutilated bodies farther south, indicating the explorers had resorted to cannibalism. Charles Dickens dismissed these reports, which were later confirmed, as “the wild tales of a herd of savages” and suggested the Inuit had slain Franklin's men. This colonial mindset would persist as the Arctic came under government control. Canada and Alaska took Indigenous children, including some from Cambridge Bay, away to be reeducated in abusive residential schools, where thousands died. A week after I met with Langan I talked with Inuit Circumpolar Council international chairwoman Sara Olsvig, who has spoken out against testing the seabed-curtain idea in her native Greenland. She says governments need to start regulating geoengineering, and researchers need to seek the free, prior and informed consent of local communities. When somebody claims, “‘We need your piece of land in the name of a greater good,' that's exactly what happened when we were colonized,” Olsvig says. Ceccolini says the operation will shut down if ice thickening proves ineffective or damaging. During a break for black tea and candied Arctic char, I asked three of them about the sea-ice thickening. They would be concerned about the drones if Real Ice conducted its 100-square-kilometer demonstration, a key step to scaling up, in the strait near Cambridge Bay rather than farther north. “If they start doing that under the water, we're going to get no more fish, no more seal, no more nothing,” said Annie Atighioyak, who was born in an igloo on the sea ice in 1940. As global fossil-fuel emissions keep rising, attitudes about geoengineering are starting to change. Two weeks before I arrived in Cambridge Bay I went to an annual Arctic science conference at the Oceanographic Institute of Monaco. Though not a scientist, Paulsen was the first person to reach all eight poles—the geographic, geomagnetic, magnetic and least accessible poles of each hemisphere—and was onboard one of the two submersibles that planted a Russian flag on the North Pole seabed in 2007. In 2023, while flying over Greenland in an ultralight aircraft, Paulsen was startled to notice that the once brilliant ice sheet was turning darker as less fresh snow fell. He said he decided only “more drastic solutions” could avoid catastrophic climate impacts, given our failure to rein in fossil fuels. It's not enough to just study climate, he scolded the scientists. The University of the Arctic, a network of educational institutions that Paulsen chairs, has rated the feasibility of 61 polar interventions, from spraying glaciers with ski-resort snow guns to cables that stop icebergs from drifting south. Also there was Fonger Ypma of Arctic Reflections, a Dutch sea-ice-thickening company that has done field trials in Newfoundland and Svalbard. Frost flowers form when very cold, still air rests on newly formed ice (at left), which is relatively much warmer. The surge of interest has created a schism in polar science. In October 2024 a preprint paper by 42 top glaciologists condemned ice thickening and other polar geoengineering techniques as dangerous and unfeasible. Spreading tiny glass spheres across mountain glaciers could decrease the reflectivity of ice covered in fresh snow. But the overarching concern is that geoengineering fixes are “making decarbonization a lot less attractive,” according to Heidi Sevestre, one of the paper's authors, who visited an Arctic Reflections trial last year. At the Monaco conference, Kim Holmén, a Norwegian Polar Institute climate scientist who has spent more than three decades on Svalbard, the fastest-warming place on Earth, argued that trying to undo the harms of our technology with even more technology was folly. Critics say it would be more effective to put geoengineering funds toward cutting emissions. The amount being spent on Arctic geoengineering is small but growing. It's hard to imagine state agencies allocating $10 billion a year for sea-ice thickening, especially with China, Russia and reportedly the U.S. looking to develop Arctic shipping routes; for them, less ice is better. Brazil's Amazon Fund for rainforest conservation, which Real Ice has held up as a possible funding model, has collected only $780 million from governments. He wants to hold similar “adopt-a-billionaire” gatherings in the U.S. The administration didn't respond to a request for comment. Real Ice and Arctic Reflections would like to eventually sell “cooling credits,” a strategy used by Make Sunsets. Individuals and companies that want to compensate for their CO2 emissions pay that start-up to launch balloons full of sunlight-blocking sulfur dioxide. Some of the biggest buyers are tech firms such as Microsoft, whose emissions are swelling as the company builds AI data centers. Kelly, the former White House adviser, is indeed worried that geoengineering could be “hijacked” by oil or tech companies as an excuse to continue business as usual. Ice companies just have to be willing to shut down their technology if it starts harming nature or undermining climate goals, he told me as we drank coffee at the research station in Cambridge Bay. “We all have to keep asking ourselves and checking ourselves and one another: Have we slid into hubris?” Kelly said. Alec Luhn wrote the feature “Out of Thin Air” about carbon capture in our September 2024 issue and “Rusting Rivers” in our January 2024 issue. He is an award-winning climate journalist who has reported from a town invaded by polar bears, the only floating nuclear power plant and the coldest inhabited place on Earth.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/surprising-genetic-evidence-shows-human-evolution-in-recent-millennia/'>New Genetic Evidence Challenges the Idea That Human Evolution Has Stalled</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>DNA Studies Uncover Unexpected Evolutionary Changes in Modern Humans Mounting evidence from genome studies indicates that, contrary to received wisdom, our species has undergone profound biological adaptation in its recent evolutionary past For thousands of years they have lived at altitudes of more than two miles, where oxygen is about 35 percent lower than at sea level. This type of setting is among the harshest environments humans have ever inhabited. Scientists have recognized for some time that these residents of the Andes Mountains have evolved genetic adaptations to the thin air of their lofty home. Now researchers are learning that they have also evolved another remarkable genetic adaptation since their ancestors first settled the highlands of South America around 10,000 years ago. The dangers it poses are well known: inorganic arsenic is associated with cancers, skin lesions, heart disease, diabetes and infant mortality in other populations. But the biochemistry of Andeans has evolved to efficiently metabolize this notoriously toxic substance. It is a prime example of natural selection, the evolutionary process by which organisms adapt to their environments to survive longer and produce more offspring. The oldest known members of the human family evolved in Africa around six million to seven million years ago and looked apelike in many ways. Our own species, Homo sapiens, arose in Africa a few hundred thousand years ago and began venturing into other parts of the world in significant numbers around 60,000 years ago. By that point our physical appearance seems to have settled into an evolutionary plateau, with only minor differences among human populations around the globe. After natural selection had worked its wonders for millions of years, transforming small-brained quadrupeds into large-brained bipeds, it appeared that biological evolution had slowed to a crawl in our lineage as H. sapiens developed agriculture, founded civilizations and transformed the planet. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Early studies of the DNA of modern people turned up few fixed differences—genetic variants possessed exclusively by one population—which seemed to confirm this apparent stasis. Consequently, many scholars believed that the latest chapter of the human saga revolved around cultural changes rather than biological ones—figuring out more reliable means of obtaining food instead of changing our digestive or metabolic systems, for instance. But advances in the sequencing of ancient and modern DNA have allowed scientists to look more closely at how our genetic code has evolved over time—and the results are startling. Genetic studies suggest that H. sapiens experienced many major episodes of natural selection in the past few thousand years as our ancestors fanned across the globe and entered new environments containing foods, diseases and toxic substances they had never before encountered. We are like rats or cockroaches—extremely adaptable.” This research offers fresh insights into how our species conquered every corner of the planet. We didn't manage this feat through cultural adaptation alone, as some scientists previously supposed. Rather humans continued to evolve biologically to keep pace with the radical changes they were making in their ways of life as they pushed into terra incognita. In general, human populations share most of the same genetic variation and evolutionary history. New research raises the possibility that recent human history involved far more dynamic evolution than previously thought. In Darwinian biology, the classic conception of natural selection is a “hard sweep,” in which a beneficial mutation allows some individuals to survive longer or produce more offspring such that eventually that variant becomes fixed in the population. For instance, around 42,000 years ago a selective sweep changed a protein on the surface of red blood cells in Africans to boost their resistance to malaria. People in the Tibetan Highlands underwent selective sweeps for genes that helped them tolerate low oxygen (intriguingly, populations of the Himalayas, Andes and Ethiopian highlands adapted to high altitude with different assortments of genes, taking different evolutionary paths to solve similar problems). Around 8,500 years ago early farmers spread an allele that helped them synthesize long-chain polyunsaturated fatty acids from plant-based foods. The new genetic variant allowed agricultural populations to synthesize them from short-chain fatty acids found in plants. This variant was rare at first, but now it is present in about 60 percent of Europeans. Likewise, as dairy farming rose, so, too, did a gene variant that helped people consume milk products into adulthood. When Stonehenge was built around 5,000 years ago, virtually no Europeans possessed the genes people need to digest milk as adults. Yet around 4,500 years ago a gene that kept the lactase turned on in adulthood began to spread through Europe and South Asia. Another series of sweeps beginning around 8,000 years ago gave Eurasians their distinctive pale complexion. These changes reduced their production of the dark skin pigment known as melanin, which is believed to have allowed more sunlight to penetrate their skin and help them synthesize vitamin D, a nutrient in short supply among early agriculturalists. These examples of hard sweeps became well known among geneticists, mostly because they seemed so uncommon. Many researchers thus concluded that selective sweeps accounted for only a little of the genetic change our species has undergone over the past several thousand years. Most of the change, they proposed, stemmed not from natural selection but from gene flow (when populations interbreed as a consequence of migration) and genetic drift (when a genetic variant becomes more or less prevalent through random chance). Yet reconstructing the past from genomes of modern-day people is tricky business because evolution often brushes over its own footprints. Early studies relied on the DNA of modern people to make inferences about evolution, but these methods could detect only events that had lasting effects. Episodes of natural selection are sometimes ephemeral, and evidence of them vanishes from our genomes when the selective pressures subside or when populations mix. Now ancient DNA is allowing investigators to find episodes of long-ago selection that have since been overwritten. Since then, the number of ancient genomes has expanded steadily to more than 10,000 today. The lesson: to fully appreciate the extent of natural selection in history, one must look at local populations in narrow windows of time. Modern Europeans descend from three main ancestral populations: hunter-gatherers who colonized the continent by around 40,000 years ago, early farmers from Anatolia who came into Europe about 8,500 years ago, and pastoralists from the Pontic-Caspian steppe who arrived around 5,000 years ago. In 2022 a research team led by Yassine Souilmi of the University of Adelaide's Australian Center for Ancient DNA examined 1,162 ancient DNA samples from these ancestral lineages and captured snapshots of their genetics before and after they mixed. They scanned the genomes for any regions with unusually low- or high-frequency alleles, signs of ancient sweeps. None were shared with a comparative population of sub-Saharan Africans, suggesting they originated after our species began spreading beyond its African birthplace into other parts of the world. One striking finding was a hard sweep on a region of chromosome 6 called major histocompatibility complex class III, or MHC III, in ancient Anatolians. This ensemble of genes encodes proteins involved in immunity, and natural selection usually promotes genetic diversity in that region to defend against an array of potential threats. Again and again the selection pressures relaxed, and the traces of adaptations that had been widespread were “almost entirely erased from descendant populations” through interbreeding with other groups or genetic drift, Souilmi and his co-authors write in their study. “Such strong positive selection events have been much more common in recent human history than previously recognized,” they conclude. This finding contradicts the notion that technological innovation and intelligence exempted later H. sapiens from biological adaptation. One thing nature regularly throws at us is deadly disease. Human populations have long been locked in an evolutionary arms race with pathogens. Even as ancient humans vanquished dangerous predators, they remained vulnerable to these microscopic enemies. In this way pathogens helped shape who we are today. “Where there is mortality, there is selection: individuals who die before reaching reproductive age do not pass on their genes,” says Lluis Quintana-Murci, a population geneticist at the Institut Pasteur in Paris. “Indeed, infectious diseases and pathogens have been major drivers of natural selection throughout human history.” In a 2023 study, Quintana-Murci and his colleagues analyzed 2,879 ancient and modern genomes to see how the DNA of Europeans changed over the past 10,000 years. They found 139 positions on the genome that had been targeted by strong natural selection—either “positive selection” to promote advantageous genetic variants or “negative selection” to purge harmful ones. More than 80 percent of the positive selection events began in the past 4,500 years—a time of swelling urban communities, growing dependence on agriculture, proximity to domesticated animals and a rise in epidemics. “Natural selection has been pervasive throughout this period,” Quintana-Murci says. Some adaptations to infectious pathogens came at a cost, however: strengthening resistance to ancient diseases might have elevated the likelihood of immune overreaction. In other words, a hypervigilant defense system could go haywire and attack one's own body. For example, there was a sharp increase in several genetic variants that protect against infectious illnesses but also raise the risk of inflammatory bowel conditions such as Crohn's disease. Several variants in the MHC (also known as the human leukocyte antigen, or HLA, region in humans) also appear to have undergone selection to resist pathogens. There was a drop in variants that increase the risk of COVID-19, for instance, suggesting that ancient people battled coronaviruses centuries before the recent pandemic. Taken together, the results suggest that our immune system has been repeatedly tweaked by recent selection like a software system that requires constant updates. Despite the plethora of new discoveries, Quintana-Murci believes researchers have uncovered only the most obvious examples of ancient selection, and he suspects many more cases will come to light as analytical methods become more powerful and researchers obtain more ancient DNA from other regions of the world. One big surprise is just how pervasive these adaptations have been. A team led by scientists at Harvard Medical School analyzed more than 8,400 DNA samples from people who lived in western Eurasia during the past 14,000 years. For each SNP, they computed selection coefficients to measure how much natural selection acted to promote or suppress that variant in the next generation. In a pre-peer-review draft of the paper released publicly last year, David Reich and his colleagues report that they found evidence for natural selection at 347 places on the genome—an order of magnitude more than previously known. Native Tibetans have a genetic adaptation to the low oxygen levels of their high-altitude home. In an interview with podcaster Dwarkesh Patel, Reich previewed the findings and described striking shifts in genetic variants over the past 10 millennia: “We think we have many, many hundreds of places where there [have] been very strong changes in frequency over time,” he said. The whole genome is seething with these changes in this period.” Early farming populations underwent strong selection to abandon the “thrifty genes” that promote body-fat storage. These gene variants had been advantageous for hunter-gatherers who endured times of scarcity, but they became liabilities in the more abundant age of agriculture. Other sweeps forged dramatic genetic changes affecting skin pigmentation; blood type; and susceptibility to diseases such as tuberculosis, multiple sclerosis, diabetes, celiac disease, bipolar disorder and schizophrenia. Presumably this allele offered some as yet unknown protective effect that outweighed its attendant risk of celiac disease. With new analytical techniques, researchers can read these erasures like an ancient palimpsest. “That's the holy grail—having a really powerful method to detect locations in the genome that are very likely to be under selection,” says Ray Tobler, a population geneticist and ancient DNA specialist at Australian National University. One promising area of discovery concerns so-called polygenic traits, which are controlled by multiple genes. Human height, for instance, is estimated to be influenced by more than 100,000 positions. This distributed influence can make it hard to identify genetic targets of natural selection. But with technological advances that have made it faster and cheaper than ever to sequence high volumes of DNA, he says, scientists now can spot polygenic adaptations that previously were invisible. These latest genetic studies have opened a new frontier in research into traits that are far more complex than the single-gene-mediated ability to digest milk in adulthood. But not everyone agrees that these traits are necessarily products of natural selection. Perhaps, skeptics have suggested, the observed fluctuations in allele frequencies are just routine oscillations of variants in the gene pool rather than proof positive of natural selection acting to adapt the human body to environmental challenges. Some of the papers have drawn criticism about their statistical methods. Some findings of ancient selection have not been replicated by other studies. Iain Mathieson, a geneticist at the University of Pennsylvania, is circumspect. He thinks new studies such as the Harvard paper are indeed detecting real shifts in gene frequencies, but he notes that quite a few of them appear to be transient. Mathieson suspects that many genetic variants have been subject to only weak or fleeting selection without much lasting effect. The new research raises the possibility that recent human history involved far more dynamic evolution than previously recognized, with repeated episodes of selection followed by reversals. “It's a super interesting question that ancient DNA is opening back up,” he says, even if there isn't yet consensus in the field about the extent to which this kind of evolution has occurred. That consensus may emerge as scientists obtain additional ancient DNA samples and further refine the tools they use to analyze them.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64794868/original-magna-carta/'>A Historian Pulled an Old Document Out of Storage—and Discovered an Original Magna Carta</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It was believed a copy when purchased in 1946 for $27.50. Believed for decades to be a copy, a pair of British scholars recently uncovered that it is, in fact, the real deal. “My reaction was one of amazement and, in a way, awe that I should have managed to find a previously unknown Magna Carta,” David Carpenter, professor of medieval history at King's College London, told the Associated Press. Harvard had purchased what was believed to have been a copy of the Magna Carta created in 1327 from London bookseller Sweet & Maxwell for $27.50 in 1946 . It carried out an inconspicuous existence in the library since then, until Carpenter took notice of it in December of 2023 and sent images to Nicholas Vincent—a medieval history professor at the University of East Anglia in Britain—asking what he thought. “You know jolly well what that is,” Carpenter said Vincent told him, according to Harvard Magazine. Vincent let Harvard know what they had, but their response was “lukewarm,” at best. Still, he and Carpenter worked with the library to authenticate the work, using ultraviolet light and spectral imaging to reveal faded writing. The Magna Carta was first issued in 1215—a powerful document that placed a rule of law on English monarchs. Carpenter said that well over 30 were created in 1300 by King Edward I and sent across the country with his seal, but only six were thought to have survived. “It's one of the world's most valuable documents,” Carpenter said. The scholars wanted to establish how a document sold in a London bookshop in the 1940s could trace back to 1300. The pair was able to find that a World War I flying ace—Air Vice-Marshal Forster Sammy Maynard, CB—sold the document via a Sotheby's auction in 1945. Vincent said that Maynard's grandson told him that the pilot had inherited historical documents from Thomas and John Clarkson—leaders of a campaign against the British slave trade. It was known to still be there in 1762, making this likely the same document. What was “bought for peanuts in 1946” and has sat unceremoniously within the Boston institution will now get its due. Harvard plans to keep its Magna Carta, but now, they know it's worth. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. World's First Patient Saved by Custom Gene Therapy Experts May Have Found a Way to Simplify Gravity Experts Found an Ancient Trove of Unfinished Art Explorers Found Maya Sacrifices in a ‘Blood Cave'</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d44151-025-00092-4'>Jayant Narlikar, visionary astrophysicist and science populariser, dies at 86</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 12:23:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Jayant Vishnu Narlikar, a visionary Indian astrophysicist whose bold cosmological theories and deep dedication to science education reshaped India's scientific landscape, died on 20 May 2025 in Pune. For over six decades, he remained a tenacious challenger of orthodoxy and a luminous bridge between rigorous theory and the public imagination. His early education was at Banaras Hindu University before he headed to the University of Cambridge for a PhD under the British astronomer Fred Hoyle. Though this work remained outside mainstream acceptance, Narlikar never flinched. He valued conceptual clarity and logical coherence above fashionable trends. “Division by zero is not allowed,” he often said, with the moral firmness of a mathematician. Narlikar returned to India in 1972 and joined the Tata Institute of Fundamental Research (TIFR), assembling a group of scientists whose influence would shape Indian astrophysics for generations. Among them were Thanu Padmanabhan, Sanjeev Dhurandhar and Ajit Kembhavi — all leaders in fields from gravitational wave detection to data-intensive observational astronomy. “I worked on off-beat ideas, which he delighted in taking very seriously.” On one occasion, Souradeep submitted a paper with both their names; Narlikar promptly struck his own name off the author list. Conceived as a hub where scholars from across Indian universities could share resources and research time, IUCAA was a pioneering experiment in scientific democracy. He structured it around an “eight-fold way” — a Buddhist-inspired schema outlining education, research, training and outreach — and persuaded architect Charles Correa to design the campus. Correa, coming off a project on classical Indian astronomy, was intrigued by the chance to build for modern stargazers. High-quality telescopes are prohibitively expensive and often underused due to limited observing time. Narlikar saw public engagement as a core scientific duty. Under his leadership, IUCAA became deeply involved in science outreach, especially at the school level. Narlikar is survived by his three daughters — Geeta, Girija and Leelavati — all scientists. It is rare to find a scientist equally at ease formulating cosmological theories, founding institutions, mentoring iconoclasts and penning science fiction. Rarer still to find one who, until the end, remained curious, civil and unshakably principled. Job Title: Associate or Senior Editor (environmental carbon cycle), Nature Communications Location(s): Shanghai, Beijing, Nanjing, New York, Jersey... Shanghai, Beijing, Nanjing, New York, Jersey City, Philadelphia, Washington DC, Milan, Madrid Junior Group Leader position with a focus on applying artificial intelligence (AI) to the field of integrative structural biology. Job Title: Associate/Senior Editor, Scientific Reports (Engineering) Location(s): Shanghai or Beijing – hybrid working model Closing date: 15th Jun... Job Title: Associate/Senior Editor, Scientific Reports (Physical Sciences) Location(s): Shanghai or Beijing – hybrid working model Closing date: 15... Zhejiang University School of Medicine warmly invites talented individuals from around the world to join us! You have full access to this article via your institution. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64814750/unified-theory-gravity/'>Scientists May Have Found a Way to Simplify Gravity. It Could Change Physics as We Know It.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It may be more straightforward than we thought—mathematically speaking, anyway. Two scientists in Finland are claiming to have advanced the cause of a unified theory of gravity, including “a complete, renormalizable theory of quantum gravity.” Physicists have long tried to mesh gravity with the standard model of physics by, in a sense, comparing like with like—how can we describe gravity using measurable things in a way that aligns with how the standard model describes electromagnetic, weak, and strong forces? A gauge is a way to measure something that is comparable to other things, like India's narrow gauge railways. In physics, gauge theory helps scientists take all the measurable things they know and align them in order to find commonalities or definitions. The duck-ness gauge only requires waddling and quacking. Finally, they write, “[f]our symmetries of the components of the space-time dimension field are used to derive a gauge theory, called unified gravity.” Basically, their goal was to find the mathematically smallest model that could still hold up to all the rules required of a theory of unified gravity (one that unites the standard model and quantum physics). One of the keys is that, within a gauge relationship, many terms can simply be canceled out, the same way you may have learned to do in algebra and calculus. Partanen and Tulkki claim that by substituting new (but equivalent) values for parts of their formulae,  they've created a gauge model that no longer relies on a contentious variable. A key term in this research is normalization, or renormalization. The scientists chose a compact model over a noncompact one, meaning that their model doesn't have any missing pieces that they aren't sure how to categorize. There's no quacking fish or waddling giraffe gumming up the works of what a duck must be. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. She's also an enthusiast of just about everything. World's First Patient Saved by Custom Gene Therapy Explorers Found Maya Sacrifices in a ‘Blood Cave'</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-60016-9'>Standardization guidelines and future trends for PET hydrolase research</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 11:56:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 4684 (2025) Cite this article Enzymatic depolymerization of polyethylene terephthalate (PET) towards monomer recycling offers a green route to a circular plastic economy, with scale-up currently underway. Yet, inconsistent assessment methods hinder clear comparisons between various PET hydrolases. This Perspective aims to identify critical gaps in this dynamic research field and outline key principles for selecting and tailoring novel enzymes, such as using uniform PET samples and standardizing reaction settings that mimic industrial conditions. Applying these guidelines will improve enzyme screening efficiency, increase data reproducibility, deepen the understanding of interfacial biocatalysis, and ultimately accelerate the development of more robust and cost-effective bio-based PET recycling methods. Polyethylene terephthalate (PET) is extensively utilized in beverage bottles, textiles, and food packaging, making it one of the most prevalent plastics in global waste streams. Consequently, its significant contribution to plastic pollution has raised urgent demand for innovative recycling technologies from both scientific and industrial sectors. In many countries, PET, especially single-use bottles, may achieve up to 90% recycling rates, making it the most recycled among all mass-produced fossil-fuel-derived plastics1,2. PET in bottles and food packaging is rarely blended with other polymers and typically lacks or contains only trace amounts of chemical additives3. This high purity and homogeneity of its chemical composition render PET the most broadly studied and preferred target for recycling advancements, encompassing mechanical, chemical, and biological approaches4,5,6. PET has a polyester backbone, which is notably susceptible to hydrolytic cleavage. Since the 1970s, scientists have been inspired to investigate the broad variety of natural ester hydrolases for their potential in depolymerizing synthetic polyesters, including PET. In 2005, Müller et al. reported the first widely recognized enzymatic depolymerization of amorphized waste PET bottles7. The last two decades have witnessed the identification, characterization, and engineering of numerous PET hydrolases6,8, as well as the successful demonstration of biological recycling of waste PET on an industrial scale9. This achievement has been recognized as one of the ten emerging chemical technologies in 2023 by IUPAC10. Recent techno-economic analysis (TEA)11 and life cycle assessment (LCA)12 have indicated that although PET bio-recycling holds promise, substantial challenges persist regarding energy consumption, economic feasibility, and overall environmental impact, particularly due to mechanical processing required to amorphize waste PET and pH regulation during the depolymerization reaction. While innovations addressing these process-related bottlenecks are infrequently reported in recent scientific literature13, an increasing number of reports claiming superior enzymes based on limited experimental evidence have emerged. Moreover, redundant studies often appear that merely replicate findings derived from a set of thoroughly examined benchmark enzymes, particularly informed by two seminal studies: the 2016 Science article by Yoshida et al., which identified IsPETase from Ideonella sakaiensis14, and the 2020 Nature publication by Tournier et al., which described several engineered variants of the leaf-branch compost cutinase (LCC) allowing for rapid PET waste depolymerization on an industrial scale9. By contrast, although researchers in recent years have discovered numerous novel PET hydrolases from diverse habitats with potentially distinct characteristics, most are abandoned after initial publications. Another unfavorable scenario in this research domain is that many studies used polymer substrates from unknown origins or performed plastic depolymerization under conditions unrelated to industrial settings. Consequently, it is imperative to standardize and disclose a full array of these essential parameters in future research to prevent overoptimistic assessments of performance, erroneous data interpretations and the formulation of non-reproducible or invalid hypotheses, in alignment with previous expert viewpoints5,11,12. A recent publication by Arnal et al.15 has experimentally highlighted this issue by providing a reliable assessment of four important PET hydrolases under industry-mimicking recycling conditions. Following the identification of significant inconsistencies in depolymerization performance compared to the original studies, the necessity of establishing standards for future enzyme activity evaluations and enabling cross-study comparisons is increasingly evident. This Perspective aims to provide a brief overview of the historical context and current advancements in this research area, focusing on methodologies for the discovery, screening, and characterization of novel PET hydrolases, as well as assessing their viability for industrial-scale plastic recycling, to complement recent expert reviews with a broader scope6,8,16,17. For the former research objective, we intend to emphasize research gaps in method selection and further development concerning potential final applications of PET hydrolases with distinct features. To achieve the latter research goal of developing more effective industrial biocatalysts for plastic recycling, we recommend prioritizing the use of standardized substrates that are most similar to industrial waste PET feedstocks, as well as selecting reaction conditions that resemble industrial environments in lab-scale or pilot-scale reactor experiments. We anticipate the emergence of an improved multi-pronged future research, including (i) the advancement of scientific understanding of interfacial biocatalysis and enzymatic depolymerization mechanisms, (ii) the validation of screening assays that can be readily implemented in most laboratories without the need for specialized equipment, and (iii) the design of novel PET hydrolases that are less related to a limited number of thoroughly investigated benchmark enzymes. Early efforts to discover polyester hydrolases strongly relied on classical microbiological techniques, including the isolation, cultivation, and enrichment of microbial strains capable of hydrolyzing natural polyesters. This led to the identification of fungal and bacterial cutinases, such as those from Fusarium18 and Thermobifida species19, which were later proven to depolymerize synthetic polyesters like PET7,20. Despite over half a century of research advancements, this method remains valuable for identifying novel plastic-degrading strains and enzymes from plastic-contaminated environments. These include the thoroughly studied PET-metabolizing bacterium I. sakaiensis, which secretes the mesophilic IsPETase14, and likewise other microorganisms with similar characteristics isolated from marine sediments21 or wastewater22. Metagenomics has emerged as an efficient approach to identifying novel PET hydrolases since 2012. The earliest and most extensively studied example is LCC, identified from a plant compost metagenome using a fosmid library and functional screening against tributyrin, an esterase model substrate23. However, functional metagenomics has contributed minimally to further significant PET hydrolase discoveries. Instead, most recent PET hydrolases have been identified through sequence-based enzyme mining, regardless of whether metagenomic DNA was physically extracted24,25 or sourced from online databases26,27,28,29. These sequence-based methods have become central, with gene sources extending to annotated yet uncharacterized sequences found in public repositories like NCBI and UniProt16,29,30,31. However, these bioinformatics-based searches have often been biased by employing established enzymes such as LCC, IsPETase, and those from Thermobifida species as templates, limiting exploration to a narrow phylogeny and sequence space27. Interestingly, despite their sequence similarities, many of these newly discovered PET hydrolases were found to stem from geographically and ecologically distinct habitats, including extreme environments such as deep-sea hydrothermal vents28, glaciers32, and even human saliva metagenomes33, calling into question the hypothesis that PET hydrolases evolved naturally in response to environmental plastic pollution, which has only been present for less than 70 years. It is not surprising that many natural carboxylesterases, found in nearly all organisms, are anticipated to exhibit at least minimal hydrolytic activity on the ester bonds in PET. The restricted structural diversity of reported PET hydrolases, attributable to their close phylogenetic relationships, creates an “echo chamber” effect that overrepresents structure-based mechanistic studies toward particular benchmark enzymes. This is evident in the Protein Data Bank (PDB), where out of more than 130 PET hydrolase-associated structures, over one-third represent IsPETase and its variants. Similar redundancy is seen with other benchmark enzymes9,24,34,35,36, with numerous nearly identical apo structures deposited. Instead, co-structures complexed with substrates or analogues may yield significant insights for semi-rational and artificial intelligence (AI)-driven enzyme engineering; nonetheless, they are limited, with less than 16% of PET hydrolase structures in the PDB solved in a substrate (analogue)-bound state16. This highlights the ongoing challenge of obtaining wet-lab data on substrate interactions, essential for advancing functional studies and developing more efficient PET hydrolases. According to the ESTHER database on α/β-hydrolase fold proteins37, most PET hydrolases with known structures belong to the polyesterase-lipase-cutinase (PLC) family sourced from bacteria27,29. These enzymes can be further classified into three types (I, IIa, IIb) based on amino acid adoption at specific surface subsites38. Type I includes thermophilic enzymes like LCC23 and those from Thermobifida species20, while Type IIa is represented by the polyester hydrolase (PE-H) from Pseudomonas aestusnigri39 and Type IIb by IsPETase14. Figure 1 shows that, despite their classification, these enzymes have a highly conserved core fold with only minor variations from the structure of LCC, such as an extra α-helical turn due to additional residues. Archaeal PET hydrolase from the feruloyl esterase family has key structural similarities with bacterial PLC-family PET hydrolases, but with additional α-helices flanking the active site40. This partially covered active site, similar to certain lipase family polyester hydrolases41, is expected to limit its access to polymeric substrates. Eliminating these lid-like structures has demonstrated an improvement in PET hydrolyzing ability41. Fungal cutinase-like PET hydrolases possess the most unique structures compared to LCC, with the lowest root-mean-square deviation (RMSD) value42. Although fungal cutinases were among the first recognized effective PET hydrolases43, they remained largely neglected in this research domain until recently when several studies with significant polymer degradation performance were published44,45. PET hydrolases with alternative structural folds and distinct active site geometries but considerably low depolymerizing activity were recently discovered using bioinformatics and machine-learning-guided enzyme mining in multiple metagenomic libraries27. This study validated the hypothesis that high depolymerization efficiency is likely confined to bacterial PLC-like and fungal cutinase-like PET hydrolases. A more recent study significantly expanded the range of target PET hydrolase sequences by incorporating various blocks from the ESTHER database, although it remains confined to enzymes with an α/β-hydrolase fold. Landscape profiling by sampling a sequence cluster framework revealed a collection of highly active enzymes, including MiPa-P and KuBu-P, the latter of which can even catalyze PET glycolysis in the presence of highly concentrated ethylene glycol (EG)29. Still, the top-performing enzymes, KuBu-P (PDB code 8YTW) and MiPa-P (PDB code 8YTU), have relatively low RMSD values of 0.94 Å and 0.98 Å to LCC, respectively. These findings are complicating the potential for further engineering of distinct scaffolds for industrial applications. Illustrated are selected bacterial polyesterase-lipase-cutinase (PLC)-like27,34,36,136 and fungal cutinase-like PET hydrolases42,137,138, alongside phylogenetically distinct enzymes from archaea40 and bacteria27,139  with minimal depolymerization activity but structural uniqueness. These PET hydrolases are grouped based on their Cα root-mean-square deviation (RMSD) relative to LCC as a reference. Ribbon plots are colored with a gradient from low RMSD (blue) to 4 Å and higher (white). The catalytic triad (serine, histidine, and asparagine in all structures) is shown as yellow sticks. PDB codes, RMSD values, and residues included in the calculations are shown below each structure. Alternatively, emerging AI-driven techniques, such as de novo enzyme design46, have the potential to generate unique PET hydrolases. In a recent study47, novel, mostly α-helical hydrolase scaffolds were obtained using a diffusion model coupled with an ensemble generation method focused on the active site preorganization. Interestingly, new active serine hydrolase scaffolds have been generated, even specifically designed to catalyze PET hydrolysis. However, disclosing only kinetic data against an atypical small-molecule substrate is insufficient to assess its realistic PET depolymerization activity. Another study generated a terminally truncated PET hydrolase using LCC as a template, with a similar catalytic efficiency but markedly lower stability than its parental scaffold46. The next option for creating novel PET hydrolases is to incorporate the classical hydrolase catalytic triad into structurally stable, non-catalytic proteins48. Nonetheless, the resulting PET depolymerization activity is significantly constrained by the inherent protein architecture, which may hinder effective substrate access necessary for interfacial catalysis17,49. Although these studies demonstrated the feasibility of the computational methodology, it is yet to be determined whether AI-assisted design can generate novel PET hydrolase scaffolds with overwhelming activity that surpasses the naturally evolved α/β hydrolase folds. PET is a semi-crystalline thermoplastic composed of crystalline fractions with well-ordered, tightly packed polymer chains and surrounding amorphous fractions with randomly oriented, intertwined molecules50. The crystalline microstructures exhibit significant resistance to the diffusion of water and enzymes, which are essential reactants and catalysts for biocatalytic depolymerization, respectively8,51. In contrast, the amorphous PET fractions are significantly more susceptible to enzymatic hydrolysis, particularly when environmental temperatures reach or exceed the glass transition temperature (Tg). Shorter segments of amorphous PET undergo conformational changes during the glass transition, which is thought to improve enzyme accessibility and subsequent ester bond hydrolysis significantly8,50,52. The Tg of PET under anhydrous conditions has been determined to range between 65 and 81 °C, depending on the analytical methods used8,53. PET substrates are completely immersed in water during biocatalytic depolymerization, and the resulting water plasticization effect can lower the bulk Tg of PET by up to 16 °C8,53. Considering the enhanced mobility of surface-located polymer chains54, the Tg of the PET surface layer (surface Tg) may be as low as ~40 °C53, which explains why significant hydrolysis of highly amorphous PET can be observed with mesophilic bacteria like I. sakaiensis under ambient conditions14. Melt-extrusion-based amorphization9,15,36,55,56,57,58,59 is commonly used as the first step before enzymatic PET decomposition due to the high crystallinity of over 30% in real-world waste PET streams (Fig. Rapidly quenching the hot polymer extrudates in cold water or other solvents is required to achieve a low-crystallinity of ideally less than 15%, allowing for rapid biocatalytic depolymerization50,60,61. Thereafter, the amorphized PET typically undergoes a mechanical size-reduction process, either via cryogenic grinding or ambient temperature crushing9,15,36,57,58,59. The increased surface-to-volume ratios are considered advantageous for interfacial enzyme accessibility, thereby improving polymer conversion efficiency. Nevertheless, this hypothesis has not consistently held true in several recent studies. The smallest particles with the highest specific surface area (SSA) extracted from the same micronization batch of low-crystallinity (or amorphized) PET typically exhibited slightly or moderately enhanced initial hydrolysis rates, particularly at lower enzyme loading, but significantly reduced maximum achievable depolymerization extent compared to the larger ones with lower SSA55,62,63. The latter phenomenon has been ascribed to the increased crystallinity, which typically exhibits an inverse relationship with particle sizes, caused by strain-induced crystallization during micronization36,55,62,64. Larger ground particles and precursor amorphous PET materials prior to micronization (either granules from melt-quenching or commercial PET films) can be enzymatically depolymerized to comparably high extents after an adequate reaction duration. Given the high energy costs and negative environmental effects associated with the micronization process11,12, the trade-offs between fast hydrolysis kinetics using cryomilled plastics and prolonged depolymerization with non-ground plastics necessitate reassessment in industrial settings. A patent for PET pre-treatment aimed at enzymatic recycling suggests that using an underwater pelletizer (Fig. 2) to produce small pellets under 1 mm could replace the contentious and energy-intensive grinding process65. To improve its economic viability, the cryogrinding step is considered optional and omitted here. The figure also displays key material-related parameters and reaction conditions that affect depolymerization performance. The crystallinity, molecular mass, and specific surface area of a particular PET sample are crucial interrelated parameters that influence the achievable degree of depolymerization. A simplified schematic surface plot illustrating the threshold values for optimal depolymerization performance emphasizes the interdependence of these parameters. Additional parameters influencing decomposition performance, such as varying Tg of the surface layer and interior bulk polymers, as well as the effect of water plasticization and polymer recrystallization under biocatalytic conditions, should be considered. Bioreactor parameters for reaction conditions must be consistently documented and standardized to ensure comparable high-level depolymerization results. Polymer chain scission can occur at each stage of the presently performed thermomechanical pre-treatment of PET waste (Fig. Despite being an undesirable consequence when a similar melt-extrusion process is used for PET mechanical recycling66, the decreased molecular weights were found to facilitate the subsequent enzymatic depolymerization36,67. Once PET feedstocks have a number-averaged molecular mass (Mn) of less than 10,000 g mol−1, their rapid and complete enzymatic depolymerization becomes less dependent on the bulk crystallinity, even if it surpasses the previously recommended degradable threshold value of 20%57. By harsher pre-treatment involving microwaves, PET's Mn can be reduced below the entanglement length of 3500 g mol−1 and readily hydrolyzed using an IsPETase variant at ambient temperature67. Within this Mn range, conventional indicators of high crystalline PET, such as the increased ratio of trans to gauche conformation, become irrelevant for describing its enzymatic degradability. These findings highlight the importance of comprehensively documenting all of these polymer properties, including molecular mass, SSA (or particle size), and bulk crystallinity (preferably also the crystallinity of the surface polymer layer), throughout the depolymerization process in future research. This can assist in unambiguously defining standard PET feedstocks (Table 1) and building a standardized amorphization technique that balances optimal material qualities related to degradability and economic viability, reducing the cost of this most expensive step11,12. Currently, research groups lacking comprehensive polymer characterization equipment are advised to utilize commercially available amorphous PET film, such as that provided by Goodfellow Ltd., as a preliminary test material for PET hydrolases in initial laboratory-scale scientific studies. Several studies36,68 have recently published detailed polymer parameters for this material, ranging beyond mere crystallinity, establishing its optimal utility over waste PET from unknown sources with undefined properties. The distinct polymer properties of PET also define the appropriate characteristics of the biocatalysts and the reaction conditions necessary for its fast depolymerization (Table 1). The flexibility of water-plasticized amorphous PET polymer chains enhances as reaction temperatures rise between 40-70 °C, facilitating their accessibility to enzymatic hydrolysis69. Higher temperatures, however, lead to faster polymer recrystallization into non-degradable crystalline microstructures than enzymatic depolymerization can occur9,15,50,60. Consequently, recent studies indicate a preference for reaction temperatures between 65 and 70 °C and the use of thermotolerant enzymes, such as LCC-ICCG9,15, TurboPETase57 and PES-H115,24,36,61,70,71, in pilot-scale processes that resemble industrial settings9,57,71. Accordingly, thermostabilization has been implemented as a primary strategy to enhance the depolymerization capability of mesophilic PET hydrolases, including Kubu-P29 and IsPETase14. Nonetheless, regarding IsPETase, despite substantial research having raised its melting point (Tm) by over 37 °C72,73,74, the most advanced variants remain inferior to LCC-ICCG in large-scale PET depolymerization tests15. This raises the question of whether a high Tm (above 85 °C) can serve as the sole criterion, as commonly employed in previous studies6,8, to determine if a specific enzyme inherently exhibits the necessary long-term stability and activity over several hours for the rapid PET decomposition at elevated temperatures. Instead, future research should evaluate the kinetic stability of an enzyme under operational conditions to provide a broader assessment of its thermostability. The continuous release of terephthalate (TPA) monomer as a result of enzymatic PET hydrolysis lowers the pH value of the reaction supernatant, which may influence enzyme activity and stability, as well as the compositions of the product mixture15,75. Increased concentrations of unwanted oligomeric ester intermediates can hinder subsequent product purification and diminish the yield of pure TPA. Therefore, recently demonstrated depolymerization processes are frequently pH-controlled in the range of 7–8, which is appropriate for most known PET hydrolases9,43,57. Nonetheless, low-concentrated phosphate or Tris buffers are commonly used in lab-scale bioreactors with pH control9,57. This should be avoided in large industrial settings to reduce costs and ensure reactor durability. pH control is considered a concern regarding the economic and environmental impacts of industrial PET recycling12. For future lab-scale studies, these impacts can be mitigated by utilizing ammonia in place of NaOH for pH regulation12,58 or implementing small-scale bioreactors (<50 mL), such as the modified Chi.Bio systems instead of other larger commercial bioreactors76. Nonetheless, because small reactors can only accommodate less than 10% solid loading, it is recommended to validate the process in medium-sized reactors (in litre scales) that allow standard agitation with up to 30% solid loading before final industrial implementation at cubic meter scale (Box 1). A more ambitious long-term vision to address this pH-related issue may involve designing a PET hydrolase that is less sensitive to pH fluctuations or that is optimized for a low pH range to facilitate rapid accumulation of exclusively pure TPA. Alongside the utilization of standardized PET feedstocks, establishing standardized bioreactor conditions by defining optimal solid (substrate) loading, enzyme loading, temperature, pH, agitation, and other parameters can ensure consistent, efficient, and scalable operations while maximizing monomer yield across diverse scales, from research laboratories to industrial environments. Nevertheless, as indicated in Tables 1 and 2, substantial discrepancies in specific key parameters reported to date, coupled with the sporadic reporting of others, highlight the necessity and challenges of conducting further comprehensive studies for standardization. - Test every PET hydrolase (natural or de novo designed) on a uniform PET substrate with all key parameters described (Table 1). - If defined waste PET sample is unavailable, use commercial amorphous PET film (e.g., Goodfellow ES301445) or its cryomilled form. - For thermotolerant enzymes applicable at ≥65 °C, employ PET-based (amorphous PET or PET NP) HTS assays for enzyme engineering under industrial-like conditions (Table 2); small-molecule proxy substrates should be avoided. - Evaluate top enzyme candidates in small bioreactors (<50 mL), followed by a sequential scale-up to 1 L, 100 L, and beyond (Fig. Modify reaction parameters according to Table 2 at each step as required. - Record pH-stat titration, accumulation of UV-absorbing products (e.g., TPA, MHET, and BHET, also known as TPA equivalents with the abbreviation TPAeq. ), and polymer mass loss over time. - Report initial depolymerization rate as molTPAeq. h−1 genzyme−1 or molTPAeq. - Measure all polymer properties outlined in Table 1 before and after reaction—and ideally at intermediate timepoints—to track changes in crystallinity, molecular weight, surface area, etc. Numerous recently discovered PET hydrolases are intrinsically thermolabile, rendering them less suitable candidates for free enzyme-based PET recycling without extensive and expensive thermostabilization engineering. Alternatively, they can be recombinantly expressed in (mostly mesophilic) microbial cell factories that can utilize at least one PET monomer for growth71,77,78,79. This concept of polymer degradation and monomer catabolism by a single microbe was recently coined PETtrophy80,81, analogous to consolidated bioprocessing in cellulose conversion82. However, achieving high enzymatic plastic depolymerization activities that align with the microbe's growth conditions, including pH, temperature, and salt concentration, is challenging. Molecular engineering tools are frequently absent for modifying microbial thermophiles83, leading to the infrequent demonstration of thermotolerant whole-cell catalysts for PET decomposition at its optimal temperature. The only example to date is engineered Clostridium thermocellum expressing LCC and its variants, though lacking monomer metabolism capability63,84. Other PET-metabolizing strains are constructed based on trivial microbial chassis that grow at temperatures below 40 °C85. Consequently, mesophilic PET hydrolases must be expressed at sufficiently high levels and tailored to enable the rapid release of growth-essential monomers into the medium, outpacing their translocation into microbial cells and subsequent metabolism. The efficacy of PET depolymerization under these conditions is expected to be significantly lower than that at elevated temperatures exceeding 65 °C. Thus, an upcycling strategy utilizing these strains to produce value-added chemicals should always be considered to realize potential economic viability. In addition, focusing on “easier” substrates that can be depolymerized faster at ambient temperatures may enhance the utility of these PET-trophic strains for stakeholders. These include PET oligomers that have been rigorously pretreated or pre-degraded67,86, as well as alternative polyesters with similar chemical compositions, such as the commercially available polybutylene adipate terephthalate (PBAT)80. The time and cost savings from bypassing polymer and monomer separation may compensate the typically low depolymerization efficiency of whole-cell catalysts, providing significant benefits compared to free enzyme-based depolymerization. Another extreme example is the in situ remediation of wastewater contaminated with synthetic microfibers, which, according to a current study, are primarily oligomeric PET87, for example, in effluents in textile production regions88 or general communal wastewater treatment plants. PET hydrolases derived from marine environments exhibiting significant salt tolerance28 can ultimately be utilized to construct environmentally resistant whole-cell catalysts when integrated into a comparably robust microbial chassis89,90. Other mesophilic wild-type enzymes may exhibit insufficient activity and stability under these harsh conditions, such as high salinity, extreme pH levels, or the presence of laundry detergents, necessitating further extensive enzyme optimization efforts91. Nonetheless, this may spark a heated debate about releasing genetically modified microbes (GMM)92, which is strictly prohibited in the EU (and other countries). In academic literature, this is referenced so far as a theoretical alternative in regions lacking other solutions93. However, as a result of intensive ongoing research activities, challenges associated with technical implementations and governmental regulations may be addressed shortly to maximize the benefits of GMMs in combating plastic pollution while mitigating risks of environmental release92. Numerous high-throughput screening (HTS) assays have been developed in recent years for the discovery and engineering of PET hydrolases16,94. However, their specific limitations in areas such as technical availability, ease of handling, and reproducibility necessitate re-evaluation before a standard guideline can be established. Several assays employ self-synthesized small-molecule surrogate substrates that resemble PET segments or degradation intermediates, such as mono- or bis(2-hydroxyethyl) terephthalate (MHET or BHET)95,96,97. While these proxy substrates offer practical benefits, including simplified dosing and the compatibility with established HTS platforms, they may result in a skewed identification of short ester hydrolyzing activity without PET depolymerization capability98,99,100, particularly in environmental samples or functional metagenomics. On the other hand, screening of mutant libraries derived from known PET hydrolases can benefit from this assay category96,101, as increased activity on MHET or BHET is frequently linked to the mitigation of their inhibitory effect, thereby improving overall PET decomposition performance. To minimize false positives, it is still highly advisable to consistently prioritize the direct use of PET polymers in HTS experimental setups. Optimal PET substrates must demonstrate uniform polymer properties, such as low-crystallinity and high specific surface area, facilitating rapid decomposition that can be observed within a constrained reaction volume, such as microplate wells or microdroplets, and a short timeframe, ideally under one hour. PET nanoparticles (NP), produced through solvent displacement processes102,103,104, are a polymeric substrate option characterized by a uniform size range of 100 ± 50 nm and lower molecular weights compared to their precursor materials (Table 1). These material properties promote their rapid enzymatic decomposition, even at ambient temperature36,48,102,103,104,105. PET NP can be used to detect microbial depolymerization activities by clear-zone assay when immobilized in agar plates. Similarly, the turbidity variations of homogenously suspended PET NP immobilized in other hydrogels can facilitate the determination of PET hydrolysis kinetics36,70,102. However, PET NP tend to agglomerate in buffered aqueous solutions, complicating their handling in small reaction volumes such as microplate wells or microdroplets. Therefore, when activity is quantified solely through the formation and detection of released monomers104,105, PET NP have no significant advantage over other amorphous PET materials, whether tailored from commercially available products or casted from polymer solution106,107,108. The casting method for preparing polymeric substrates can also incorporate fluorescent dyes, which can be detected in the supernatant during PET decomposition, providing an alternative to quantifying its monomers108. However, the use of toxic PET soluble organic solvents for HTS setups is considered suboptimal due to challenges in handling, high costs, environmental concerns, and laboratory safety issues. TPA, MHET, and BHET are the main hydrolysis products of PET that can be detected at 240 nm. Aside from the standard quantification method of high-performance liquid chromatography (HPLC), the total amount of their formation can be easily estimated by simply measuring the total absorbance at 240 nm for HTS purposes101,106,109. When TPA needs to be precisely quantified as a final monomer, it can be converted to a hydroxylated form and detected by fluorimetric monitoring with a high detection sensitivity96,104,107,110. The quantification of the other monomer, EG, has been overlooked for a considerable period until recently. EG can now be accurately monitored by different fluorimetric assays based on enzymatic conversion111 or a biosensor112. In conjunction with other recently developed techniques for monitoring TPA105,113,114,115, living-cell based biosensors offer various options for assessing PET hydrolyzing activity, from precise endpoint determination of individual monomers to functional screening of metagenomic libraries. Microfluidics-based HTS also involves living cells, facilitating enzyme expression, biocatalysis, and product detection within a single microdroplet or microbead116. These methods may not effectively identify thermotolerant enzyme variants because the optimal temperatures for cell growth typically do not exceed 40 °C, while apparent PET depolymerization activity is generally detectable at temperatures above 50 °C. The same limitation may also pertain to other cell growth-based screening assays, such as enzyme surface display117,118. While these HTS assays primarily evaluate mesophilic PET hydrolases which typically necessitate costly and risky thermostabilization engineering prior to validating their potential for industrial PET recycling, these enzymes should be better suited to the alternative application scenarios at lower temperatures discussed above. Consequently, an automated liquid handling system that incorporates: (i) enzyme expression and cell lysis at optimal cell-growth temperatures, (ii) PET depolymerization at elevated temperatures, and (iii) endpoint assessment of monomer release under optimal assay conditions should serve as an ideal standard workflow for screening thermotolerant PET hydrolases needed for industrial PET recycling (Table 2 and Fig. A few recent studies demonstrated variations of this standard workflow, particularly characterized by the simultaneous detection of hydrolysis products and quantification of soluble recombinant enzymes in cell lysates through various optical techniques101,108. The latter can replace the labor-intensive and costly high-throughput protein purification step while maintaining high data accuracy of specific enzyme activities, which is crucial for machine-learning-guided enzyme engineering. To optimize these assays, proxy substrates like BHET and its derivatives96,101 can be replaced with, for example, cryomilled commercially available amorphous PET with defined characteristics62,68, particularly for HTS using microplate readers. This may enable solid loading of >10% in microplate wells to more accurately resemble standardized industrial conditions (Table 2). The function-based HTS method is an essential component of the workflow. Given that agar-plate or microfluidics-based screening involves living cells, which prevents the detection of PET hydrolysis at elevated temperatures, microplate-based HTS (detailed in the lower part) may have the potential to be prioritized. Colorimetric or fluorometric methods, sensitive at different wavelengths, are optimal for the distinct quantification of functional enzyme concentrations in cell lysates and the monomers released from PET depolymerization. This approach enables rapid screening of large mutant libraries for efficient PET hydrolases while also producing high-quality data at a high throughput that can be used for machine-learning-guided enzyme engineering116. Currently used endpoint measurements for assessing depolymerization efficacy allow for basic comparisons between engineered enzymes and their references, but they lack time-resolved data throughout the degradation process. As a result, molecular insights required to understand the mechanisms underlying interfacial biocatalysis reactions are frequently missing. Computational modeling based on recently resolved complex structures of various PET hydrolases with substrate analogues16 may elucidate the catalytic cycle in atomic detail. Still, it fails to account for other important aspects such as enzyme adsorption/desorption behavior and PET physical properties. Much evidence suggests that these latter aspects, which reflect non-covalent interactions, serve as key bottlenecks for the overall reaction. The most explicit example of this is the resilience of crystalline PET to enzymatic hydrolysis. However, small, soluble PET fragments, similar to those used in computational analyses, can be hydrolyzed very quickly119, which also implies that the main activation barrier for the overall process lies outside catalytic bond breakage. The classical way to single out the importance of different steps in a complex reaction is comparative kinetics, and this approach has two key requirements. Firstly, a simple yet realistic microkinetic reaction scheme, which introduces rate constants for the interconversion of substrate and intermediates, and secondly experimental methods that provide sufficient data to support kinetic modeling. At the current stage, none of these areas are well-developed for PET hydrolases, but some progress has been made. For example, limitations associated with an undefined concentration of the insoluble substrate have been addressed by introducing substrate area60,120, or attack site density119 as proxies for molar concentration. It has also proven useful to supplement the conventional conditions of substrate excess for steady-state kinetic analysis with measurements, where the enzyme is in excess concerning the substrate102,121. This latter experimental condition is the opposite of the conventional kinetic framework, and it has been termed inverse Michaelis–Menten kinetics122. A steady-state approximation under inverse conditions never occurs in homogenous reactions but may be relevant in two-phase systems as new reaction sites on the surface are replenished continuously123. Parameters from inverse kinetic analyses, particularly the maximal rate at high enzyme load, are of interest because they reflect both catalytic efficacy and the density of reaction or attack sites on the solid surface. Specifically, a combination of data obtained with either enzyme- or substrate-excess provides information on the maximal turnover (kcat in s−1), KM (Michaelis constant in M), and the density of hydrolysable sites on the PET surface (in molsites gPET−1)122. If experiments are only conducted under conditions of substrate excess, KM values will be in mass-based units, typically g L−1. This parameter can be useful for comparing different enzymes acting on the same substrate, but as mass-based KM values depend strongly on the SSA of the substrate, it is generally not meaningful to compare values obtained with different types of PET. Work along these lines has identified some general trends in PET hydrolase kinetics, which are listed in Table 2. For a number of the most proficient enzymes acting on insoluble PET at moderate temperature (30–50 °C), kcat was 0.5–2 s−1, and this was 1–2 orders of magnitude slower than their hydrolysis of the same bond in soluble PET fragments119,121,124. The same enzymes had KM values in the hundredths of nM range, and taken together, this implies strong substrate interactions but slow turnover compared to other hydrolases acting on native, soluble substrates125. A molecular description of these characteristics appears to be a promising avenue for further understanding of structure-function relationships for PET hydrolases. To achieve this, we will need better methods to quantify the accumulation and decay of intermediates; both enzyme-substrate complexes and PET fragments. Progress curves for these intermediates will elucidate the complex pathway from insoluble polymer to the monomers TPA and EG, and it will also provide the necessary experimental input for further kinetic modeling. Advanced knowledge in this respect may help to accelerate the late stage of the industrial recycling process, which is currently thought to be hampered by these hydrolysis intermediates. Finally, we encourage more experimental and computational research into the relationship between polymer conformation and enzyme activity. Some reports have pointed out a strong effect on catalysis of the trans-gauche isomerism in the PET chain52,67, and it has been proposed that constraints on isomerization may lessen gradually as PET chains are shortened in the enzymatic process126. Information on this would be important both for the fundamental understanding of the interfacial enzyme reaction and as a guide for enzyme engineering. The increasing demand for sustainable plastic recycling solutions, integral to the bio-based circular economy and climate crisis mitigation, positions the biotechnological valorization of PET as a dynamic research field in the foreseeable future. The successful implementation of several extensively customized PET hydrolases, which allow for nearly complete polymer-to-monomer conversion in less than 12 h on pilot or larger industrial scales9,29,57, suggests that fast kinetics is unlikely to be the primary constraint in this research domain. However, to maintain competitiveness, PET hydrolases, similar to other industrial enzymes, must undergo ongoing performance improvements, including enhanced enzyme kinetics and long-term stability, tailored pH optima, etc. Consequently, standard guidelines derived from research endeavors so far are discussed above and summarized in Box 1 and Tables 1 and 2. Other related research avenues, which have received considerably less attention yet appear to hold significant promise, should be explored with greater focus in the future. For instance, enzyme immobilization may enhance stability and facilitate multiple applications127,128. Also, the development of synergistic enzyme cocktails may enable reduced dosages or address mixed plastic waste129, a more complex challenge within the plastic pollution dilemma. For the latter, a moderate temperature range of 40–50 °C may be considered when extended operation time is acceptable, particularly when not all applied enzymes exhibit adequate half-lives at higher temperatures. Simultaneously, understanding and optimizing process conditions should be afforded equal attention and effort as enzyme engineering in future research. This may involve optimizing melt-extrusion parameters56, investigating the feasibility of omitting energy-intensive micronization stages, or implementing alternative pre-treatment techniques130, all of which could reduce the costs of the current process11,12. A thorough understanding of the material properties associated with degradability is urgently required to assess the feasibility of current research aimed at developing highly efficient enzymes specifically for crystalline PET. These forthcoming research endeavors may address our knowledge gaps concerning the absent information in the standardized parameter set (Tables 1 and 2) that collectively impacts the future development of industrial PET recycling. At the enzyme discovery stage, researchers should move beyond the trivial homology-based database mining and increase the use of novel bioinformatics- or AI-based tools to discover or design new PET hydrolase scaffolds29,47. Moreover, the previously underutilized classical function-based metagenomics or microbiological techniques can still be remarkably beneficial for discovering novel sequence or structural diversity of depolymerizing enzymes that have not yet been included in the digital repository. We propose a more efficient data-sharing approach by characterizing novel enzymes and all known mutants of benchmark enzymes, summarizing their biocatalytic properties in the standardized format shown in Table 2, and making them available online for free access. These will ultimately contribute significantly to the advancement of AI-based PET hydrolase engineering or design, as well as the development of specific AI models tailored for interfacial biocatalysis. Motivated by the successful industrialization of enzymatic PET recycling, other hydrolysable waste plastics, such as polyurethanes, polylactides, and polyamides, are logically emerging as the next wave of substrates for potential biocatalytic depolymerization13,131,132,133. This ambition necessitates considering the lessons and experiences learned from PET, such as using polymer feedstocks with precisely defined properties and the need for standardized conditions for evaluating novel depolymerizing enzymes. Ultimately, we expect this article will inspire researchers to collaborate and develop similar guidelines to standardize the future bio-based plastic recycling industry. Xin, J. et al. Progress in the catalytic glycolysis of polyethylene terephthalate. Google Scholar Aryan, Y., Yadav, P. & Samadder, S. R. Life cycle assessment of the existing and proposed plastic waste management options in India: a case study. Google Scholar Tsochatzis, E. D., Lopes, J. & Corredig, M. Chemical testing of mechanically recycled polyethylene terephthalate for food packaging in the European Union. Google Scholar Ragaert, K., Delva, L. & Van Geem, K. Mechanical and chemical recycling of solid plastic waste. Waste Manag. Google Scholar Ellis, L. D. et al. Chemical and biological catalysis for plastics recycling and upcycling. Google Scholar Tournier, V. et al. Enzymes' power for plastics degradation. Google Scholar Müller, R.-J., Schrader, H., Profe, J., Dresler, K. & Deckwer, W.-D. Enzymatic degradation of poly(ethylene terephthalate): rapid hydrolyse using a hydrolase from T. fusca. Rapid Commun. Google Scholar Wei, R. et al. Mechanism-based design of efficient PET hydrolases. ACS Catal. Google Scholar Tournier, V. et al. An engineered PET depolymerase to break down and recycle plastic bottles. Google Scholar Gomollón-Bel, F. IUPAC's 2023 top ten emerging technologies in chemistry. Google Scholar Singh, A. et al. Techno-economic, life-cycle, and socioeconomic impact analysis of enzymatic recycling of poly(ethylene terephthalate). Google Scholar Uekert, T. et al. Life cycle assessment of enzymatic poly(ethylene terephthalate) recycling. Google Scholar Wei, R., Weber, G., Blank, L. M. & Bornscheuer, U. T. Process insights for harnessing biotechnology for plastic depolymerization. Google Scholar Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). Google Scholar Arnal, G. et al. Assessment of four engineered PET degrading enzymes considering large-scale industrial applications. ACS Catal. Google Scholar Mican, J. et al. Exploring new galaxies: perspectives on the discovery of novel PET-degrading enzymes. Google Scholar Zhou, J., Cui, Z., Wei, R., Dong, W. & Jiang, M. Interfacial catalysis in enzymatic PET plastic depolymerization. Trends Chem. Google Scholar Purdy, R. E. & Kolattukudy, P. E. Depolymerization of a hydroxy fatty acid biopolymer, cutin, by an extracellular enzyme from Fusarium solani f. pisi: Isolation and some properties of the enzyme. Google Scholar Fett, W. F., Wijey, C., Moreau, R. A. & Osman, S. F. Production of cutinase by Thermomonospora fusca ATCC 27730. Google Scholar Herrero Acero, E. et al. Enzymatic surface hydrolysis of PET: effect of structural diversity on kinetic properties of cutinases from Thermobifida. Google Scholar Liu, R. et al. Polyethylene terephthalate (PET)-degrading bacteria in the pelagic deep-sea sediments of the Pacific Ocean. Google Scholar Wilkes, R. A. et al. Mechanisms of polyethylene terephthalate pellet fragmentation into nanoplastics and assimilable carbons by wastewater comamonas. Sulaiman, S. et al. Isolation of a novel cutinase homolog with polyethylene terephthalate-degrading activity from leaf-branch compost by using a metagenomic approach. Google Scholar Sonnendecker, C. et al. Low carbon footprint recycling of post-consumer PET plastic with a metagenomic polyester hydrolase. Google Scholar Meyer Cifuentes, I. E. et al. Molecular and biochemical differences of the tandem and cold-adapted PET hydrolases Ple628 and Ple629, isolated from a marine microbial consortium. Google Scholar Danso, D. et al. New insights into the function and global distribution of polyethylene terephthalate (PET)-degrading bacteria and enzymes in marine and terrestrial metagenomes. Google Scholar Erickson, E. et al. Sourcing thermotolerant poly(ethylene terephthalate) hydrolase scaffolds from natural diversity. Google Scholar Chen, J. et al. Global marine microbial diversity and its potential in bioprospecting. Google Scholar Seo, H. et al. Landscape profiling of PET depolymerases using a natural sequence cluster framework. Google Scholar Xi, X. et al. Secretory expression in Bacillus subtilis and biochemical characterization of a highly thermostable polyethylene terephthalate hydrolase from bacterium HR29. Hong, H. et al. Discovery and rational engineering of PET hydrolase with both mesophilic and thermophilic PET hydrolase properties. Qi, X., Ji, M., Yin, C.-F., Zhou, N.-Y. & Liu, Y. Glacier as a source of novel polyethylene terephthalate hydrolases. Eiamthong, B. et al. Discovery and genetic code expansion of a polyethylene terephthalate (PET) hydrolase from the human saliva metagenome for the degradation and bio-functionalization of PET. Sulaiman, S., You, D.-J., Kanaya, E., Koga, Y. & Kanaya, S. Crystal structure and thermodynamic and kinetic stability of metagenome-derived LC-cutinase. Zeng, W. et al. Substrate-binding mode of a thermophilic PET hydrolase and engineering the enzyme to enhance the hydrolytic efficacy. ACS Catal. Pfaff, L. et al. Multiple substrate binding mode-guided engineering of a thermophilic PET hydrolase. ACS Catal. Chatonnet, A., Perochon, M., Velluet, E. & Marchot, P. The ESTHER database on alpha/beta hydrolase fold proteins—an overview of recent developments. Joo, S. et al. Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation. Bollinger, A. et al. A novel polyester hydrolase from the marine bacterium Pseudomonas aestusnigri —structural and functional insights. Perez-Garcia, P. et al. An archaeal lid-containing feruloyl esterase degrades polyethylene terephthalate. Biundo, A. et al. Polyphenol oxidases exhibit promiscuous proteolytic activity. Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Ronkvist, ÅM., Xie, W., Lu, W. & Gross, R. A. Cutinase-catalyzed hydrolysis of poly(ethylene terephthalate). Lee, S. H. et al. Characterization and engineering of a fungal poly(ethylene terephthalate) hydrolyzing enzyme from Aspergillus fumigatiaffinis. ACS Catal. Brinch-Pedersen, W. et al. Discovery and surface charge engineering of fungal cutinases for enhanced activity on poly(ethylene terephthalate). ACS Sustain. Ding, Y., Zhang, S., Kong, X., Hess, H. & Zhang, Y. Replicating PET hydrolytic activity by positioning active sites with smaller synthetic protein scaffolds. Lauko, A. et al. Computational design of serine hydrolases. Robles-Martín, A. et al. Sub-micro- and nano-sized polyethylene terephthalate deconstruction with engineered protein nanopores. Wei, R. & Bornscheuer, U. T. Designer catalytic nanopores meet PET nanoparticles. Thomsen, T. B., Almdal, K. & Meyer, A. S. Significance of poly(ethylene terephthalate) (PET) substrate crystallinity on enzymatic degradation. Woodard, L. N. & Grunlan, M. A. Hydrolytic degradation and erosion of polyester biomaterials. ACS Macro Lett. Wei, R. et al. Conformational fitting of a flexible oligomeric substrate does not explain the enzymatic PET degradation. Tarazona, N. A. et al. Rapid depolymerization of poly(ethylene terephthalate) thin films by a dual-enzyme system and its impact on material properties. Shinotsuka, K., Bliznyuk, V. N. & Assender, H. E. Near-surface crystallization of PET. Chang, A. C. et al. Understanding consequences and tradeoffs of melt processing as a pretreatment for enzymatic depolymerization of poly(ethylene terephthalate). Rapid Commun. Patel, A. et al. Melt processing pretreatment effects on enzymatic depolymerization of poly(ethylene terephthalate). ACS Sustain. Cui, Y. et al. Computational redesign of a hydrolase for nearly complete PET depolymerization at industrially relevant high-solids loading. Xue, R. et al. Enzymatic upcycling of PET waste to calcium terephthalate for battery anodes. Cheng, Y., Cheng, Y., Zhou, S., Ruzha, Y. & Yang, Y. Closed-loop recycling of PET fabric and bottle waste by tandem pre-amorphization and enzymatic hydrolysis. Wei, R. et al. Biocatalytic degradation efficiency of postconsumer polyethylene terephthalate packaging determined by their polymer microstructures. Thomsen, T. B. et al. Rate response of poly(ethylene terephthalate)-hydrolases to substrate crystallinity: basis for understanding the lag phase. Brizendine, R. K. et al. Particle size reduction of poly(ethylene terephthalate) increases the rate of enzymatic depolymerization but does not increase the overall conversion extent. ACS Sustain. et al. Optimized whole-cell depolymerization of polyethylene terephthalate to monomers using engineered Clostridium thermocellum. Jabarin, S. A. Strain-induced crystallization of poly(ethylene terephthalate). Desrousseaux, M.-L. et al. A process for degrading plastic products. Schyns, Z. O. G. & Shaver, M. P. Mechanical recycling of packaging plastics: a review. Rapid Commun. Guo, B. et al. Conformational selection in biocatalytic plastic degradation by PETase. ACS Catal. Cuthbertson, A. Characterization of polymer properties and identification of additives in commercially available research plastics. Akram, E. et al. On the temperature dependence of enzymatic degradation of poly(ethylene terephthalate). Jäckering, A. et al. From bulk to binding: decoding the entry of PET into hydrolase binding pockets. JACS Au 4, 4000–4012 (2024). Welsing, G. et al. Two-step biocatalytic conversion of post-consumer polyethylene terephthalate into value-added products facilitated by genetic and bioprocess engineering. Cui, Y. et al. Computational redesign of a PETase for plastic biodegradation under ambient condition by the GRAPE strategy. ACS Catal. Lu, H. et al. Machine learning-aided engineering of hydrolases for PET depolymerization. Bell, E. L. et al. Directed evolution of an efficient and thermostable PET depolymerase. Świderek, K. et al. Mechanistic studies of a lipase unveil effect of pH on hydrolysis products of small PET modules. Denton, M. C. R. et al. Integration of pH control into Chi.Bio reactors and demonstration with small-scale enzymatic poly(ethylene terephthalate) hydrolysis. Mückschel, B. et al. Ethylene glycol metabolism by Pseudomonas putida. Tiso, T. et al. Towards bio-upcycling of polyethylene terephthalate. Narancic, T. et al. Genome analysis of the metabolically versatile Pseudomonas umsongensis GO16: the genetic basis for PET monomer upcycling into polyhydroxyalkanoates. Brandenberg, O. F., Schubert, O. T. & Kruglyak, L. Towards synthetic PETtrophy: engineering Pseudomonas putida for concurrent polyethylene terephthalate (PET) monomer metabolism and PET hydrolase expression. Cell Fact. Han, W. et al. Biodegradation of poly(ethylene terephthalate) through PETase surface-display: from function to structure. Li, Z., Waghmare, P. R., Dijkhuizen, L., Meng, X. & Liu, W. Research advances on the consolidated bioprocessing of lignocellulosic biomass. Crosby, J. R. et al. Extreme thermophiles as emerging metabolic engineering platforms. Yan, F., Wei, R., Cui, Q., Bornscheuer, U. T. & Liu, Y.-J. Thermophilic whole-cell degradation of polyethylene terephthalate using engineered Clostridium thermocellum. Tiso, T. et al. The metabolic potential of plastics as biotechnological carbon sources—review and targets for the future. Giraldo-Narcizo, S., Guenani, N., Sánchez-Pérez, A. M. & Guerrero, A. Accelerated polyethylene terephthalate (PET) enzymatic degradation by room temperature alkali pre-treatment for reduced polymer crystallinity. Yang, T., Xu, Y., Liu, G. & Nowack, B. Oligomers are a major fraction of the submicrometre particles released during washing of polyester textiles. Deng, H. et al. Microplastic pollution in water and sediment in a textile industrial area. Li, T., Menegatti, S. & Crook, N. Breakdown of polyethylene therepthalate microplastics under saltwater conditions using engineered Vibrio natriegens. AIChE J. Zhou, Y. et al. Development of a novel “4E” polyethylene terephthalate bio-recycling process with the potential for industrial application: efficient, economical, energy-saving, and eco-friendly. Zurier, H. S. & Goddard, J. M. PETase engineering for enhanced degradation of microplastic fibers in simulated wastewater sludge processing conditions. ACS EST Water 3, 2210–2218 (2023). Chemla, Y., Sweeney, C. J., Wozniak, C. A. Design and regulation of engineered bacteria for environmental release. de Lorenzo, V. Environmental galenics: large-scale fortification of extant microbiomes with engineered bioremediation agents. Pirillo, V., Pollegioni, L. & Molla, G. Analytical methods for the investigation of enzyme-catalyzed degradation of polyethylene terephthalate. FEBS J. Qiao, Y. et al. Fluorescence-activated droplet sorting of PET degrading microorganisms. Shi, L. et al. Complete depolymerization of PET wastes by an evolved PET hydrolase from directed evolution. Taxeidis, G. et al. New labeled PET analogues enable the functional screening and characterization of PET-degrading enzymes. ACS Sustain. Belisário-Ferrari, M. R., Wei, R., Schneider, T., Honak, A. & Zimmermann, W. Fast turbidimetric assay for analyzing the enzymatic hydrolysis of polyethylene terephthalate model substrates. von Haugwitz, G. et al. Structural insights into (tere)phthalate-ester hydrolysis by a carboxylesterase and its role in promoting PET depolymerization. ACS Catal. Li, A. et al. Discovery and mechanism-guided engineering of BHET hydrolases for improved PET recycling and upcycling. Groseclose, T. M. et al. A high-throughput screening platform for engineering poly(ethylene terephthalate) hydrolases. ACS Catal. Wei, R. et al. Turbidimetric analysis of the enzymatic hydrolysis of polyethylene terephthalate nanoparticles. Vogel, K. et al. Enzymatic degradation of polyethylene terephthalate nanoplastics analyzed in real time by isothermal titration calorimetry. Total Environ. Pfaff, L., Breite, D., Badenhorst, C. P. S., Bornscheuer, U. T. & Wei, R. Fluorimetric high-throughput screening method for polyester hydrolase activity using polyethylene terephthalate nanoparticles. In Methods in Enzymology (eds Weber, G. et al.) 648, 253–270 (Academic Press, 2021). Fu, X. et al. Single distal mutation enhances activity of known PETases via stabilisation of PET-binding. Preprint at https://www.biorxiv.org/content/10.1101/2024.09.11.612432v1 (2024). Thomsen, T. B., Schubert, S. W., Hunt, C. J., Westh, P. & Meyer, A. S. A new continuous assay for quantitative assessment of enzymatic degradation of poly(ethylene terephthalate) (PET). Weigert, S., Gagsteiger, A., Menzel, T. & Höcker, B. A versatile assay platform for enzymatic poly(ethylene-terephthalate) degradation. Protein Eng. Liu, K. et al. A dual fluorescence assay enables high-throughput screening for poly(ethylene terephthalate) hydrolases. Zhong-Johnson, E. Z. L., Voigt, C. A. & Sinskey, A. J. An absorbance method for analysis of enzymatic degradation kinetics of poly(ethylene terephthalate) films. Wei, R., Oeser, T., Billig, S. & Zimmermann, W. A high-throughput assay for enzymatic polyester hydrolysis activity by fluorimetric detection. Gimeno-Pérez, M. et al. A coupled ketoreductase-diaphorase assay for the detection of polyethylene terephthalate-hydrolyzing activity. Li, M. et al. Customization of ethylene glycol (EG)-induced BMOR-based biosensor for the directed evolution of PET degrading enzymes. Bayer, T. et al. Biosensor and chemo-enzymatic one-pot cascade applications to detect and transform PET-derived terephthalic acid in living cells. Dierkes, R. F. et al. An ultra-sensitive Comamonas thiooxidans biosensor for the rapid detection of enzymatic polyethylene terephthalate (PET) degradation. Article PubMed Hernández-Sancho, J. M. et al. A versatile microbial platform as a tunable whole-cell chemical sensor. Gantz, M., Mathis, S. V., Nintzel, F. E. H., Lio, P. & Hollfelder, F. On synergy between ultrahigh throughput screening and machine learning in biocatalyst engineering. Heyde, S. A. H., Arnling Bååth, J., Westh, P., Nørholm, M. H. H. & Jensen, K. Surface display as a functional screening platform for detecting enzymes active on PET. Cell Fact. Cribari, M. A. et al. Ultrahigh-throughput directed evolution of polymer-degrading enzymes using yeast display. Baath, J. A., Borch, K., Jensen, K., Brask, J. & Westh, P. Comparative biochemistry of four polyester (PET) hydrolases. Scandola, M., Focarete, M. L. & Frisoni, G. Simple kinetic model for the heterogeneous enzymatic hydrolysis of natural poly(3-hydroxybutyrate). Erickson, E. et al. Comparative performance of PETase as a function of reaction conditions, substrate properties, and product accumulation. Kari, J., Andersen, M., Borch, K. & Westh, P. An inverse Michaelis Menten approach for general description of interfacial enzyme kinetics. ACS Catal. Andersen, M., Kari, J., Borch, K. & Westh, P. Michaelis–Menten equation for degradation of insoluble substrate. Article MathSciNet Arnling Bååth, J., Jensen, K., Borch, K., Westh, P. & Kari, J. Sabatier principle for rationalizing enzymatic hydrolysis of a synthetic polyester. JACS Au 2, 1223–1231 (2022). Sousa, S. F., Ramos, M. J., Lim, C. & Fernandes, P. A. Relationship between enzyme/substrate properties and enzyme efficiency in hydrolases. ACS Catal. Schubert, S. W. et al. Relationships of crystallinity and reaction rates for enzymatic degradation of poly (ethylene terephthalate), PET. Schwaminger, S. P. et al. Immobilization of PETase enzymes on magnetic iron oxide nanoparticles for the decomposition of microplastic PET. Nanoscale Adv. Zurier, H. S. & Goddard, J. M. Directed immobilization of PETase on mesoporous silica enables sustained depolymerase activity in synthetic wastewater conditions. Bio Mat. Branson, Y. et al. One-pot depolymerization of mixed plastics using a dual enzyme system. Yan, Z.-F. et al. Design and construction of chemical-biological module clusters for degradation and assimilation of poly(ethylene terephthalate) waste. Bayer, T. et al. Structural elucidation of a metagenomic urethanase and its engineering towards enhanced hydrolysis profiles. Bell, E. L. et al. Natural diversity screening, assay development, and characterization of nylon-6 enzymatic depolymerization. Li, Z. et al. Structure-guided engineering of a versatile urethanase improves its polyurethane depolymerization activity. Then, J. et al. Ca2+ and Mg2+ binding site engineering increases the degradation of polyethylene terephthalate films by polyester hydrolases from Thermobifida fusca. Schmidt, J. et al. Effect of Tris, MOPS, and phosphate buffers on the hydrolysis of polyethylene terephthalate films by polyester hydrolases. FEBS Open Bio 6, 919–927 (2016). Chen, C.-C. et al. General features to enhance enzymatic activity of poly(ethylene terephthalate) hydrolysis. Suzuki, K., Watanabe, T. & Kitamoto, H. A biodegradable plastic-degrading cutinase-like enzyme from the phyllosphere yeast Pseudozyma antarctica. PDB database entry https://doi.org/10.2210/pdb7CC4/pdb (2020). Suzuki, K. & Koitabashi, M. Crystal structure of a biodegradable plastic-degrading cutinase from Paraphoma sp. PDB database entry https://doi.org/10.2210/pdb7CY3/pdb (2020). Hwang, J. et al. Structural and biochemical insights into bis(2-hydroxyethyl) terephthalate degrading carboxylesterase isolated from psychrotrophic bacterium Exiguobacterium antarcticum. Download references The authors R.W., L.M.B., and U.T.B. acknowledge funding from the European Union's Horizon 2020 research and innovation program under grant agreement no. 870294 for the MIX-UP project and no. 953214 for the upPE-T project. acknowledges the support of the Werner Siemens Foundation in the frame of the WSS Research Centre “catalaix”. acknowledges support from Innovation Fund Denmark (Grant 5150-00020A) and the Novo Nordisk Foundation (Grants NNF15OC0016606 and NNFSA170028392). acknowledges funding from the “Helmholtz Sustainability Challenge” project FINEST and the satellite project PUreValue, funded by the Investment and Networking Fund of the Helmholtz Association under grant agreement no. Department of Biotechnology & Enzyme Catalysis, Institute of Biochemistry, University of Greifswald, Greifswald, Germany Ren Wei & Uwe T. Bornscheuer Department of Biotechnology and Biomedicine, Technical University of Denmark, Lyngby, Denmark Peter Westh Macromolecular Crystallography, Helmholtz-Zentrum Berlin für Materialien und Energie, Berlin, Germany Gert Weber iAMB - Institute of Applied Microbiology. ABBt - Aachen Biology and Biotechnology, BioSC, RWTH Aachen University, Aachen, Germany Lars M. Blank WSS Research Centre “catalaix”, RWTH Aachen University, Aachen, Germany Lars M. Blank You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar collaboratively developed the concept of this article and wrote the manuscript, with R.W. taking the lead. Correspondence to Ren Wei. is a member of the Scientific Advisory Board of Carbios. The remaining authors (R.W., P.W., G.W., and L.M.B.) declare no competing interests. Nature Communications thanks Sierin Lim and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Wei, R., Westh, P., Weber, G. et al. Standardization guidelines and future trends for PET hydrolase research. Nat Commun 16, 4684 (2025). Received: 09 December 2024 Published: 20 May 2025 DOI: https://doi.org/10.1038/s41467-025-60016-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41576-025-00850-1'>Functional synonymous mutations and their evolutionary consequences</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 11:47:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Genetics (2025)Cite this article Metrics details Synonymous mutations are coding mutations that do not alter protein sequences. Commonly thought to have little to no functional consequence, synonymous mutations have been widely used in evolutionary analyses that require neutral markers, including those foundational for the neutral theory. However, recent studies suggest that synonymous mutations can influence nearly every step in the expression of genetic information and may often be strongly non-neutral. We review the extent and mechanisms of these phenotypic and fitness effects and discuss the implications of the functionality and non-neutrality of synonymous mutations for various analyses and conclusions pertinent to genetics, evolution, conservation and disease. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Agashe, D. in Single Nucleotide Polymorphisms (eds Sauna, Z. E. & Kimchi-Sarfaty, C.) 15–36 (Springer, 2022). Kimura, M. The Neutral Theory of Molecular Evolution (Cambridge Univ. Zhang, J. in Evolution Since Darwin: The First 150 Years (eds Bell, M. A. et al.) 87–118 (Sinauer, 2010). Fiers, W. et al. A-protein gene of bacteriophage MS2. Google Scholar Air, G. M. et al. Gene F of bacteriophage X174. Correlation of nucleotide sequences from the DNA and amino acid sequences from the gene product. Google Scholar Efstratiadis, A., Kafatos, F. C. & Maniatis, T. The primary structure of rabbit β-globin mRNA as determined from cloned DNA. Google Scholar Grantham, R., Gautier, C., Gouy, M., Mercier, R. & Pavé, A. Codon catalog usage and the genome hypothesis. Nucleic Acids Res. Google Scholar Ikemura, T. Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system. Google Scholar Gouy, M. & Gautier, C. Codon usage in bacteria: correlation with gene expressivity. Nucleic Acids Res. Google Scholar Hanson, G. & Coller, J. Codon optimality, bias and usage in translation and mRNA decay. Cell Biol. Google Scholar Liu, Y., Yang, Q. & Zhao, F. Synonymous but not silent: the codon usage code for gene expression and protein folding. Google Scholar Stergachis, A. B. et al. Exonic transcription factor binding directs codon choice and affects protein evolution. Google Scholar Birnbaum, R. Y. et al. Coding exons function as tissue-specific enhancers of nearby genes. Genome Res. Google Scholar Doolittle, W. F., Brunet, T. D., Linquist, S. & Gregory, T. R. Distinguishing between “function” and “effect” in genome biology. Genome Biol. Google Scholar Agoglia, R. M. & Fraser, H. B. Disentangling sources of selection on exonic transcriptional enhancers. Google Scholar Xing, K. & He, X. Reassessing the “duon” hypothesis of protein evolution. Google Scholar Chen, J. et al. Prevalent uses and evolution of exonic regulatory sequences in the human genome. Google Scholar Shen, X., Song, S., Li, C. & Zhang, J. Synonymous mutations in representative yeast genes are mostly strongly non-neutral. Google Scholar Rodriguez, A. et al. Synonymous codon substitutions modulate transcription and translation of a divergent upstream gene by modulating antisense RNA production. Natl Acad. Google Scholar Sekinger, E. A., Moqtaderi, Z. & Struhl, K. Intrinsic histone-DNA interactions and low nucleosome density are important for preferential accessibility of promoter regions in yeast. Google Scholar Tillo, D. & Hughes, T. R. G+C content dominates intrinsic nucleosome occupancy. BMC Bioinform. Google Scholar Zhou, Z. et al. Codon usage is an important determinant of gene expression levels largely through its effects on transcription. Natl Acad. Google Scholar Zhao, F. et al. Genome-wide role of codon usage on transcription and identification of potential regulators. Natl Acad. Google Scholar Qian, W. & Zhang, J. Codon usage bias and nuclear mRNA concentration: correlation vs. causation. Natl Acad. Google Scholar Zhou, Z., Dang, Y., Zhou, M., Yuan, H. & Liu, Y. Codon usage biases co-evolve with transcription termination machinery to suppress premature cleavage and polyadenylation. Google Scholar Savisaar, R. & Hurst, L. D. Exonic splice regulation imposes strong selection at synonymous sites. Genome Res. Google Scholar Chamary, J. V., Parmley, J. L. & Hurst, L. D. Hearing silence: non-neutral evolution at synonymous sites in mammals. Google Scholar Lan, Y. et al. Synonymous mutations promote tumorigenesis by disrupting m6A-dependent mRNA metabolism. Google Scholar Courel, M. et al. GC content shapes mRNA storage and decay in human cells. Google Scholar Mordstein, C. et al. Codon usage and splicing jointly influence mRNA localization. Cell Syst. Google Scholar Zuckerman, B., Ron, M., Mikl, M., Segal, E. & Ulitsky, I. Gene architecture and sequence composition underpin selective dependency of nuclear export of long RNAs on NXF1 and the TREX complex. Google Scholar Savisaar, R. & Hurst, L. D. Both maintenance and avoidance of RNA-binding protein interactions constrain coding sequence evolution. Google Scholar Yang, J. R., Chen, X. & Zhang, J. Codon-by-codon modulation of translational speed and accuracy via mRNA folding. PLoS Biol. Google Scholar Kudla, G., Murray, A. W., Tollervey, D. & Plotkin, J. B. Coding-sequence determinants of gene expression in Escherichia coli. Google Scholar Pop, C. et al. Causal signals between codon bias, mRNA structure, and the efficiency of translation and elongation. Google Scholar Lai, W. C. et al. mRNAs and lncRNAs intrinsically form secondary structures with short end-to-end distances. Google Scholar Gaither, J. B. S. et al. Synonymous variants that disrupt messenger RNA structure are significantly constrained in the human population. Google Scholar Park, C., Chen, X., Yang, J. R. & Zhang, J. Differential requirements for mRNA folding partially explain why highly expressed proteins evolve slowly. Natl Acad. Google Scholar Shabalina, S. A., Ogurtsov, A. Y. & Spiridonov, N. A. A periodic pattern of mRNA secondary structure created by the genetic code. Nucleic Acids Res. Google Scholar Tuller, T., Waldman, Y. Y., Kupiec, M. & Ruppin, E. Translation efficiency is determined by both codon bias and folding energy. Natl Acad. Google Scholar Gu, W., Zhou, T. & Wilke, C. O. A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes. PLoS Comput. Google Scholar Gu, W., Wang, X., Zhai, C., Xie, X. & Zhou, T. Selection on synonymous sites for increased accessibility around miRNA binding sites in plants. Google Scholar Barrington, C. L. et al. Synonymous codon usage regulates translation initiation. Google Scholar Li, K., Kong, J., Zhang, S., Zhao, T. & Qian, W. Distance-dependent inhibition of translation initiation by downstream out-of-frame AUGs is consistent with a Brownian ratchet process of ribosome scanning. Genome Biol. Google Scholar Li, G. W., Oh, E. & Weissman, J. S. The anti-Shine–Dalgarno sequence drives translational pausing and codon choice in bacteria. Google Scholar Yang, C., Hockenberry, A. J., Jewett, M. C. & Amaral, L. A. N. Depletion of Shine-Dalgarno sequences within bacterial coding regions is expression dependent. Google Scholar Hockenberry, A. J., Jewett, M. C., Amaral, L. A. N. & Wilke, C. O. Within-gene Shine–Dalgarno sequences are not selected for function. Google Scholar Varenne, S., Buc, J., Lloubes, R. & Lazdunski, C. Translation is a non-uniform process. Effect of tRNA availability on the rate of elongation of nascent polypeptide chains. Google Scholar Ingolia, N. T., Ghaemmaghami, S., Newman, J. R. & Weissman, J. S. Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Google Scholar Qian, W., Yang, J. R., Pearson, N. M., Maclean, C. & Zhang, J. Balanced codon usage optimizes eukaryotic translational efficiency. PLoS Genet. Google Scholar Charneski, C. A. & Hurst, L. D. Positively charged residues are the major determinants of ribosomal velocity. PLoS Biol. Google Scholar Hussmann, J. A., Patchett, S., Johnson, A., Sawyer, S. & Press, W. H. Understanding biases in ribosome profiling experiments reveals signatures of translation dynamics in yeast. PLoS Genet. Google Scholar Weinberg, D. E. et al. Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation. Google Scholar Tuller, T. et al. An evolutionarily conserved mechanism for controlling the efficiency of protein translation. Google Scholar Pelechano, V., Wei, W. & Steinmetz, L. M. Widespread co-translational RNA decay reveals ribosome dynamics. Google Scholar Bentele, K., Saffert, P., Rauscher, R., Ignatova, Z. & Bluthgen, N. Efficient translation initiation dictates codon usage at gene start. Google Scholar Goodman, D. B., Church, G. M. & Kosuri, S. Causes and effects of N-terminal codon bias in bacterial genes. Google Scholar Sejour, R., Leatherwood, J., Yurovsky, A. & Futcher, B. Enrichment of rare codons at 5' ends of genes is a spandrel caused by evolutionary sequence turnover and does not improve translation. Google Scholar Bazzini, A. A. et al. Codon identity regulates mRNA stability and translation efficiency during the maternal-to-zygotic transition. EMBO J. Google Scholar Mishima, Y. & Tomari, Y. Codon usage and 3' UTR length determine maternal mRNA stability in zebrafish. Google Scholar Presnyak, V. et al. Codon optimality is a major determinant of mRNA stability. Google Scholar Chen, S. et al. Codon-resolution analysis reveals a direct and context-dependent impact of individual synonymous mutations on mRNA level. Google Scholar Boel, G. et al. Codon influence on protein expression in E. coli correlates with mRNA levels. Google Scholar Radhakrishnan, A. et al. The DEAD-box protein Dhh1p couples mRNA decay and translation by monitoring codon optimality. Google Scholar Buschauer, R. et al. The Ccr4-Not complex monitors the translating ribosome for codon optimality. Google Scholar Veltri, A. J. et al. Distinct elongation stalls during translation are linked with distinct pathways for mRNA degradation. Google Scholar Zhu, X., Cruz, V. E., Zhang, H., Erzberger, J. P. & Mendell, J. T. Specific tRNAs promote mRNA decay by recruiting the CCR4-NOT complex to translating ribosomes. Google Scholar Guo, X. et al. Selection and mutation on microRNA target sequences during rice evolution. BMC Genomics 9, 454 (2008). Google Scholar Mordret, E. et al. Systematic detection of amino acid substitutions in proteomes reveals mechanistic basis of ribosome errors and selection for translation fidelity. Google Scholar Sun, M. & Zhang, J. Preferred synonymous codons are translated more accurately: proteomic evidence, among-species variation, and mechanistic basis. Google Scholar Wu, X., Xu, M., Yang, J. R. & Lu, J. Genome-wide impact of codon usage bias on translation optimization in Drosophila melanogaster. Google Scholar Sharma, A. K. & O'Brien, E. P. Non-equilibrium coupling of protein structure and function to translation-elongation kinetics. Google Scholar Stein, K. C. & Frydman, J. The stop-and-go traffic regulating protein biogenesis: how translation kinetics controls proteostasis. Moss, M. J., Chamness, L. M. & Clark, P. L. The effects of codon usage on protein structure and folding. Buhr, F. et al. Synonymous codons direct cotranslational folding toward different protein conformations. Zhang, G., Hubalewska, M. & Ignatova, Z. Transient ribosomal attenuation coordinates protein synthesis and co-translational folding. Arpat, A. B. et al. Transcriptome-wide sites of collided ribosomes reveal principles of translational pausing. Genome Res. Zhao, T. et al. Disome-seq reveals widespread ribosome collisions that promote cotranslational protein folding. Genome Biol. Pechmann, S., Chartron, J. W. & Frydman, J. Local slowdown of translation by nonoptimal codons promotes nascent-chain recognition by SRP in vivo. Fernández-Calero, T. et al. The transcriptional activities and cellular localization of the human estrogen receptor alpha are affected by the synonymous Ala87 mutation. J. Steroid Biochem. Article PubMed Hershberg, R. & Petrov, D. A. Selection on codon bias. Callens, M., Pradier, L., Finnegan, M., Rose, C. & Bedhomme, S. Read between the lines: diversity of nontranslational selection pressures on local codon usage. Genome Biol. Chen, S. L., Lee, W., Hottes, A. K., Shapiro, L. & McAdams, H. H. Codon usage between genomes is constrained by genome-wide mutational processes. Natl Acad. Hershberg, R. & Petrov, D. A. General rules for optimal codon choice. PLoS Genet. Plotkin, J. & Kudla, G. Synonymous but not the same: the causes and consequences of codon bias. Duret, L. Evolution of synonymous codon usage in metazoans. Duret, L. & Mouchiroud, D. Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis. Natl Acad. Pouyet, F., Mouchiroud, D., Duret, L. & Semon, M. Recombination, meiotic expression and human codon usage. Bulmer, M. The selection-mutation-drift theory of synonymous codon usage. Chen, F. et al. Dissimilation of synonymous codon usage bias in virus–host coevolution due to translational selection. Andersson, S. G. & Kurland, C. G. Codon preferences in free-living microorganisms. Drummond, D. A. & Wilke, C. O. Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution. Akashi, H. Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy. Stoletzki, N. & Eyre-Walker, A. Synonymous codon usage in Escherichia coli: selection for translational accuracy. Zhou, T., Weems, M. & Wilke, C. O. Translationally optimal codons associate with structurally sensitive sites in proteins. Johansson, M., Zhang, J. & Ehrenberg, M. Genetic code translation displays a linear trade-off between efficiency and accuracy of tRNA selection. Natl Acad. Akashi, H. Inferring weak selection from patterns of polymorphism and divergence at “silent” sites in Drosophila DNA. Lawrie, D. S., Messer, P. W., Hershberg, R. & Petrov, D. A. Strong purifying selection at synonymous sites in D. melanogaster. PLoS Genet. Huang, Y. F. & Siepel, A. Estimation of allele-specific fitness effects across human protein-coding sequences and implications for disease. Genome Res. Dhindsa, R. S., Copeland, B. R., Mustoe, A. M. & Goldstein, D. B. Natural selection shapes codon usage in the human genome. Eyre-Walker, A. & Keightley, P. D. The distribution of fitness effects of new mutations. Eyre-Walker, A., Woolfit, M. & Phelps, T. The distribution of fitness effects of new deleterious amino acid mutations in humans. Agashe, D., Martinez-Gomez, N. C., Drummond, D. A. & Marx, C. J. Good codons, bad transcript: large reductions in gene expression and fitness arising from synonymous mutations in a key enzyme. Walsh, I. M., Bowman, M. A., Soto Santarriaga, I. F., Rodriguez, A. & Clark, P. L. Synonymous codon substitutions perturb cotranslational protein folding in vivo and impair cell fitness. Natl Acad. Frumkin, I. et al. Codon usage of highly expressed genes affects proteome-wide translation efficiency. Natl Acad. Kristofich, J. et al. Synonymous mutations make dramatic contributions to fitness when growth is limited by a weak-link enzyme. PLoS Genet. Carlini, D. B. & Stephan, W. In vivo introduction of unpreferred synonymous codons into the Drosophila Adh gene results in reduced levels of ADH protein. Agashe, D. et al. Large-effect beneficial synonymous mutations mediate rapid and parallel adaptation in a bacterium. Lind, P. A., Berg, O. G. & Andersson, D. I. Mutational robustness of ribosomal protein genes. Bailey, S. F., Alonso Morales, L. A. & Kassen, R. Effects of synonymous mutations beyond codon bias: the evidence for adaptive synonymous substitutions from microbial evolution experiments. Genome Biol. Lebeuf-Taylor, E., McCloskey, N., Bailey, S. F., Hinz, A. & Kassen, R. The distribution of fitness effects among synonymous mutations in a gene under directional selection. Firnberg, E., Labonte, J. W., Gray, J. J. & Ostermeier, M. A comprehensive, high-resolution map of a gene's fitness landscape. Lind, P. A., Arvidsson, L., Berg, O. G. & Andersson, D. I. Variation in mutational robustness between different proteins and the predictability of fitness effects. Sane, M., Diwan, G. D., Bhat, B. A., Wahl, L. M. & Agashe, D. Shifts in mutation spectra enhance access to beneficial mutations. Natl Acad. Carrasco, P., de la Iglesia, F. & Elena, S. F. Distribution of fitness and virulence effects caused by single-nucleotide substitutions in Tobacco etch virus. Domingo-Calap, P., Cuevas, J. M. & Sanjuan, R. The fitness effects of random mutations in single-stranded DNA and RNA bacteriophages. PLoS Genet. Peris, J. B., Davis, P., Cuevas, J. M., Nebot, M. R. & Sanjuan, R. Distribution of fitness effects caused by single-nucleotide substitutions in bacteriophage f1. Sanjuan, R., Moya, A. & Elena, S. F. The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus. Natl Acad. Cuevas, J. M., Domingo-Calap, P. & Sanjuan, R. The fitness effects of synonymous mutations in DNA and RNA viruses. Sane, M., Parveen, S. & Agashe, D. Mutation bias alters the distribution of fitness effects of mutations. Preprint at bioRxiv https://doi.org/10.1101/2024.03.24.586369 (2025). Kruglyak, L. et al. Insufficient evidence for non-neutrality of synonymous mutations. Shen, X., Song, S., Li, C. & Zhang, J. Further evidence for strong nonneutrality of yeast synonymous mutations. Yang, D. D., Rusch, L. M., Widney, K. A., Morgenthaler, A. & Copley, S. D. Synonymous edits in the Escherichia coli genome have substantial and condition-dependent effects on fitness. Natl Acad. Nyerges, A. et al. Synthetic genomes unveil the effects of synonymous recoding. Preprint at bioRxiv https://doi.org/10.1101/2024.06.16.599206 (2024). Sharon, E. et al. Functional genetic variants revealed by massively parallel precise genome editing. She, R. & Jarosz, D. F. Mapping causal variants with single-nucleotide resolution reveals biochemical drivers of phenotypic change. Hietpas, R. T., Jensen, J. D. & Bolon, D. N. Experimental illumination of a fitness landscape. Natl Acad. Amorosi, C. J. et al. Massively parallel characterization of CYP2C9 variant enzyme activity and abundance. Weile, J. et al. A framework for exhaustively mapping functional missense variants. Dhindsa, R. S. et al. A minimal role for synonymous variation in human disease. Shen, X., Song, S., Li, C. & Zhang, J. On the fitness effects and disease relevance of synonymous mutations. Preprint at bioRxiv https://doi.org/10.1101/2022.08.22.504687 (2022). McGrath, K. M. et al. Fitness benefits of a synonymous substitution in an ancient EF-Tu gene depend on the genetic background. Article PubMed Zwart, M. P. et al. Unraveling the causes of adaptive benefits of synonymous mutations in TEM-1-β-lactamase. Faheem, M., Zhang, C. J., Morris, M. N., Pleiss, J. & Oelschlaeger, P. Role of synonymous mutations in the evolution of TEM β-lactamase genes. Agents Chemother. Bailey, S. F., Hinz, A. & Kassen, R. Adaptive synonymous mutations in an experimentally evolved Pseudomonas fluorescens population. Chen, P. & Zhang, J. The loci of environmental adaptation in a model eukaryote. King, J. L. & Jukes, T. H. Non-Darwinian evolution. Miyata, T. & Yasunaga, T. Rapidly evolving mouse alpha-globin-related pseudo gene and its evolutionary history. Natl Acad. Katju, V. & Bergthorsson, U. Old trade, new tricks: insights into the spontaneous mutation process from the partnering of classical mutation accumulation experiments with high-throughput genomic approaches. Genome Biol. Hudson, R. R., Kreitman, M. & Aguade, M. A test of neutral molecular evolution based on nucleotide data. McDonald, J. H. & Kreitman, M. Adaptive protein evolution at the Adh locus in Drosophila. Sauna, Z. E. & Kimchi-Sarfaty, C. Understanding the contribution of synonymous mutations to human disease. Chen, R., Davydov, E. V., Sirota, M. & Butte, A. J. Non-synonymous and synonymous coding SNPs show similar likelihood and effect size of human disease association. PLoS ONE 5, e13574 (2010). Karczewski, K. J. et al. Systematic single-variant and gene-based association testing of thousands of phenotypes in 394,841 UK biobank exomes. Cell Genom. Takata, A., Ionita-Laza, I., Gogos, J. A., Xu, B. & Karayiorgou, M. De novo synonymous mutations in regulatory elements contribute to the genetic etiology of autism and schizophrenia. Supek, F., Minana, B., Valcarcel, J., Gabaldon, T. & Lehner, B. Synonymous mutations frequently act as driver mutations in human cancers. Sharma, Y. et al. A pan-cancer analysis of synonymous mutations. Lu, J. & Wu, C. I. Weak selection revealed by the whole-genome comparison of the X chromosome and autosomes of human and chimpanzee. Natl Acad. Han, P. et al. Genome-wide survey of ribosome collision. Meydan, S. & Guydosh, N. R. Disome and trisome profiling reveal genome-wide targets of ribosome quality control. Guydosh, N. R. & Green, R. Dom34 rescues ribosomes in 3' untranslated regions. Aitken, C. E. & Puglisi, J. D. Following the intersubunit conformation of the ribosome during translation in real time. Rahman, S., Kosakovsky Pond, S. L., Webb, A. & Hey, J. Weak selection on synonymous codons substantially inflates dN/dS estimates in bacteria. Natl Acad. Zhang, J. & Yang, J. R. Determinants of the rate of protein sequence evolution. Chen, P. & Zhang, J. Antagonistic pleiotropy conceals molecular adaptations in changing environments. Bartoszewski, R. A. et al. A synonymous single nucleotide polymorphism in ΔF508 CFTR alters the secondary structure of the mRNA and the expression of the mutant protein. Lazrak, A. et al. The silent codon change I507-ATC→ATT contributes to the severity of the ΔF508 CFTR channel dysfunction. FASEB J. Richard, P. et al. A synonymous CHRNE mutation responsible for an aberrant splicing leading to congenital myasthenic syndrome. Article PubMed Parkes, M. et al. Sequence variants in the autophagy gene IRGM and multiple other replicating loci contribute to Crohn's disease susceptibility. Brest, P. et al. A synonymous variant in IRGM alters a binding site for miR-196 and causes deregulation of IRGM-dependent xenophagy in Crohn's disease. Li, J. et al. Performance evaluation of pathogenicity-computation methods for missense variants. Nucleic Acids Res. Cheng, N. et al. Comparison and integration of computational methods for deleterious synonymous mutation prediction. Lin, B. C., Katneni, U., Jankowska, K. I., Meyer, D. & Kimchi-Sarfaty, C. In silico methods for predicting functional synonymous variants. Genome Biol. Livingstone, M. et al. Investigating DNA-, RNA-, and protein-based features as a means to discriminate pathogenic synonymous variants. Zhang, X. et al. regSNPs-splicing: a tool for prioritizing synonymous single-nucleotide substitution. Wang, L. et al. Deleterious synonymous mutation identification based on selective ensemble strategy. Article PubMed Buske, O. J., Manickaraj, A., Mital, S., Ray, P. N. & Brudno, M. Identification of deleterious synonymous variants in human genomes. Zhang, T. et al. Syntool: a novel region-based intolerance score to single nucleotide substitution for synonymous mutations predictions based on 123,136 individuals. BioMed Res. Gelfman, S. et al. Annotating pathogenic non-coding variants in genic regions. Rentzsch, P., Witten, D., Cooper, G. M., Shendure, J. & Kircher, M. CADD: predicting the deleteriousness of variants throughout the human genome. Nucleic Acids Res. Quang, D., Chen, Y. & Xie, X. DANN: a deep learning approach for annotating the pathogenicity of genetic variants. Shihab, H. A. et al. An integrative approach to predicting the functional effects of non-coding and coding sequence variation. Capriotti, E. & Fariselli, P. PhD-SNPg: a webserver and lightweight tool for scoring single nucleotide variants. Nucleic Acids Res. Bendl, J. et al. PredictSNP2: a unified platform for accurately evaluating SNP effects by exploiting the different characteristics of variants in distinct genomic regions. PLoS Comput. Download references The authors thank S. Song, S. Zhang and J. Kong for technical assistance, and members of our laboratories for valuable comments. was supported by the U.S. National Institutes of Health (R35GM139484) and W.Q. was supported by the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA28030402), the Biological Breeding-National Science and Technology Major Project (2023ZD04073) and Guangdong Provincial Key Laboratory of Synthetic Genomics (2023B1212060054). Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, MI, USA Jianzhi Zhang State Key Laboratory of Seed Innovation, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China Wenfeng Qian University of Chinese Academy of Sciences, Beijing, China Wenfeng Qian You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar The authors contributed equally to all aspects of the manuscript. Correspondence to Jianzhi Zhang or Wenfeng Qian. The authors declare no competing interests. Nature Reviews Genetics thanks Laurent Duret, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. SynMICdb: http://SynMICdb.dkfz.de Enzymatic alteration of RNA molecules consisting of the conversion of adenosines (A) to inosines (I) at specific positions. Competition among genotypes with different beneficial mutations in an asexual population. A mathematical theory of population genetics that traces all alleles of a gene sampled from a population to a single ancestral copy. The phenomenon in which synonymous codons of an amino acid are unequally used in a genome. A fungicide often used to block eukaryotic translational elongation in experiments. An experimental approach for measuring the effects of individual nucleotides in a DNA segment by creating many mutants of the DNA followed by high-throughput functional/fitness assays of the mutants. Number of individuals in an ideal (that is, Wright–Fisher) population that results in the same amount of genetic drift as in the actual population considered. A quantitative representation of the ability of an individual to pass its genome to the next generation. Investigations of genome-wide sets of genetic variants in groups of individuals to find variants associated with traits of interest. Mechanism that allows enzymes, particularly those involved in DNA replication, RNA transcription and protein synthesis, to enhance their fidelity by discriminating between correct and incorrect substrates. The accuracy of this process is higher than expected solely based on the difference in activation energy between forming correct products and incorrect products. Non-random association of alleles of different loci in a population. Methylation at the nitrogen-6 position of adenosines at specific positions in an RNA molecule. Single-stranded, non-coding RNA molecules of 21 to 23 nucleotides that bind to mRNAs to cause mRNA degradation or suppress mRNA translation. Prevailing evolutionary theory developed in the 1930s to the 1940s by combining Darwin's theory of evolution by natural selection with a population-oriented view of Mendelian genetics. Reduction in free energy of a folded mRNA molecule relative to its unfolded form. A genetic experiment in which a population of organisms is propagated through repeated population bottlenecks to minimize the impact of natural selection. A phenomenon in which some types of mutation occur more frequently than other types. tRNAs carrying an amino acid that differs from that encoded by the codon of concern and whose anticodon does not pair with the codon at exactly one nucleotide position. Genetic changes having selection coefficients that are smaller than the inverse of the effective population size. An evolutionary theory positing that most interspecific differences and intraspecific polymorphisms at the DNA or protein sequence level are selectively neutral rather than adaptive. Point mutations in a coding sequence that alter the encoded protein sequence. Number of nonsynonymous nucleotide substitutions per nonsynonymous site between two homologous coding sequences. The basic structural unit of DNA packaging in eukaryotes consisting of a segment of DNA wound around eight histone proteins. Part of the process of mRNA maturation in which a nascent RNA transcript is cleaved at a particular site and subsequently becomes the object of the addition of a poly-A tail. Codons that are used more frequently than the average of all codons of the same amino acid in highly expressed genes of a genome. A short DNA sequence of usually 2–6 nucleotides required for a Cas nuclease in the CRISPR system to cut; it is generally found 3–4 nucleotides downstream from the cut site. A nonfunctional segment of DNA that is derived from a previously functional gene. A method to determine the chromosomal regions or genetic variants affecting the variation of a quantitative trait among individuals of a species. 5'-most of the three tRNA binding sites in a ribosome that allows a deacylated tRNA to exit. The 3'-most of the three tRNA binding sites in a ribosome that selects charged tRNA molecules during protein synthesis is known as A-site. Middle of the three tRNA binding sites in a ribosome that holds the tRNA linked to the nascent polypeptide chain. The difference in relative fitness between a mutant and the wild type. Sequence motif in mRNA that recruits ribosomes through interaction with the anti-Shine–Dalgarno sequence in 16S ribosomal RNA of prokaryotes. Distribution of the allele frequencies of a set of loci such as single-nucleotide polymorphisms in a population or sample of individuals. A step in pre-mRNA processing that removes introns and joins exons to form mature mRNAs. Point mutations in a coding sequence that do not alter the encoded protein sequence. Number of synonymous nucleotide substitutions per synonymous site between two homologous coding sequences. Codons that are used less frequently than the average of all codons of the same amino acid in highly expressed genes of a genome. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zhang, J., Qian, W. Functional synonymous mutations and their evolutionary consequences. Nat Rev Genet  (2025). Download citation Accepted: 23 April 2025 Published: 20 May 2025 DOI: https://doi.org/10.1038/s41576-025-00850-1 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Genetics (Nat Rev Genet) ISSN 1471-0064 (online) ISSN 1471-0056 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01549-3'>Who were the ancient Denisovans? Fossils reveal secrets about the mysterious humans</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-20 10:29:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Michael Marshall is a science journalist in Devon, UK. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. The fragment didn't seem remarkable, but it was well preserved, giving researchers hope that it harboured intact DNA. A team of geneticists led by Johannes Krause at the Max Planck Institute for Evolutionary Anthropology in Leipzig, Germany, removed 30 milligrams of bone and managed to extract enough intact DNA to analyse it. How Denisovans thrived on top of the world: mysterious ancient humans' survival secrets revealed How Denisovans thrived on top of the world: mysterious ancient humans' survival secrets revealed When the team announced this result in March 2010, it caused a sensation. Up to that point, researchers had used preserved bones to identify every species or population of hominin — the group that includes modern and ancient humans and their immediate ancestors. “Denisovans were created from DNA work,” says palaeoanthropologist Chris Stringer at the Natural History Museum in London. Krause and his colleagues had obtained the entire nuclear genome from the finger bone, which yielded much more information. It showed that the Denisovans were a sister group to the Neanderthals, which lived in Europe and western Asia for hundreds of thousands of years. The team also described the discovery of a molar tooth, which, on the basis of its mitochondrial DNA, was Denisovan: it was unusually large, unlike teeth from modern humans or Neanderthals. A replica of an ancient finger bone found in Denisova Cave. DNA extracted from this bone was used to first identify the Denisovan lineage.Credit: Max Planck Institute for Evolutionary Anthropology Even more surprisingly, the team reported that present-day people living on the islands of New Guinea and Bougainville in the southwestern Pacific Ocean have inherited 4–6% of their DNA from Denisovans, despite the fact that these islands are roughly 8,500 kilometres from Denisova Cave. That finding triggered a frantic search for the Denisovans. Had palaeoanthropologists not yet found them, or were they sitting unrecognized in museum collections? Now, 15 years after the first report on this hominin, a handful of fossils have been identified as Denisovan, with varying degrees of certainty. They reveal a diverse, adaptable population that lived in frigid Siberia, at high altitudes and in the tropics — and researchers are scouring Asia for more evidence of them. The earliest one found outside of Africa was Homo erectus, which inhabited what is now Georgia 1.8 million years ago and soon spread as far east as the island of Java in Indonesia3. Later came the Neanderthals (Homo neanderthalensis), which dispersed throughout Europe and western Asia, sometimes venturing as far as Denisova Cave. Our own species, Homo sapiens, was the last to evolve, appearing at least 200,000 years ago in Africa (see ‘All in the family'). One fossil find in Morocco suggests that modern humans might date back more than 300,000 years4. H. sapiens left that continent in several waves, eventually replacing all other hominins. But before they did, some of them mated with Neanderthals and Denisovans. Today, the highest levels of Denisovan DNA are found in the Ayta people of the Philippines, but Denisovan DNA is widespread in people from southeast Asia, including Papua New Guinea5. When researchers set out to find Denisovan fossils, they discovered that one had been collected decades before — and had gone unrecognized. “The fossil was found without context by a Buddhist monk,” says archaeological scientist Katerina Douka at the University of Vienna. For three decades “it sat in someone's office”, says Douka, until it came to the attention of researchers, including geographer Fahu Chen at the Institute of Tibetan Plateau Research, Chinese Academy of Sciences, in Beijing. In a 2019 study, Chen and his colleagues extracted and analysed preserved proteins that identified the bone as Denisovan, and dated it to at least 160,000 years ago by measuring radioactive decay in the sediment stuck to the bone's surface6. Researchers collect sediment samples from Baishiya Karst Cave on the Tibetan plateau in China, where Denisovan bones and DNA have been found.Credit: Han Yuanyuan This indicated that Denisovans were present there around 100,000 and 60,000 years ago, and that they might have stuck around until 45,000 years ago7. These two studies established that Denisovans lived successfully on the high-altitude Tibetan plateau. This fits with a 2014 genetic study, which found that modern Tibetans have inherited a Denisovan genetic variant that helps them to cope with low oxygen levels, presumably from ancient interbreeding between modern humans and Denisovans in that region8. In parallel, another dramatic fossil find emerged more than 2,000 kilometres away, in northeast China: it looked like it might be a Denisovan, but the controversy over its classification has rumbled on ever since. An unidentified Chinese man, working for the Japanese forces who occupied that region of China at the time, discovered a hominin skull, recognized its value and hid it down a well. He kept it secret for decades, only revealing its location on his deathbed. Siberia's ancient ghost clan starts to surrender its secrets Siberia's ancient ghost clan starts to surrender its secrets A team led by palaeoanthropologist Xijun Ni at the Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences in Beijing — and including Stringer — described the Harbin skull in 2021. They found it was at least 146,000 years old. However, its shape does not match any described hominin species9. In a separate paper, Ni and others described this specimen as belonging to a new species: Homo longi (Dragon Man)10. Several features of the Harbin skull, notably its big molar teeth, seemed like they might be a match for Denisovans. The fossil possibilities were starting to mount up. Some researchers propose that this fossilized skull found in Harbin City in northeast China belonged to a Denisovan.Credit: Q. Ji et al./Innovation Meanwhile, other researchers had been re-examining fossils excavated years earlier, such as a set of skull fragments discovered in the 1970s in Xujiayao, northeast China. “They were almost exactly the same,” he says. One more possible Denisovan sample had turned up in 2008, before the group was even identified. Early that year, a man bought a lower jawbone from an antique shop in Tainan City in Taiwan. It had been dredged from the Penghu Channel, 20 kilometres offshore. The man donated it to a museum after researchers realized, on the basis of photographs, that it was a hominin bone. When they finally described it in 2015, the researchers noted that its second molar looked distinctly Denisovan12. This was supported by a study published this April, in which proteins extracted from the fossil indicated that it belonged to a male Denisovan that lived sometime between 10,000 and 190,000 years ago13 (see ‘On the move'). In a preprint first published last May, Stringer and his colleagues tried to bring all these disparate fossils together. The problem they faced was vexing: there are a great many unclassified Homo fossils from eastern Asia, and it's not obvious how many groups they represent. The team analysed 57 hominin fossils, examining up to 521 characteristics of each, depending on how much material was available. The Eurasian hominins clustered into three groups: modern humans, Neanderthals and H. longi. “We would say that the name for Denisovans is going to be longi, if there is a species name,” says Stringer. Siberia's ancient ghost clan starts to surrender its secrets Divided by DNA: The uneasy relationship between archaeology and ancient genomics How Denisovans thrived on top of the world: mysterious ancient humans' survival secrets revealed Biggest Denisovan fossil yet spills ancient human's secrets Picuris Pueblo oral history and genomics reveal continuity in US Southwest Inside the quest to digitally unroll ancient scrolls burnt by Vesuvius Picuris Pueblo oral history and genomics reveal continuity in US Southwest Inside the quest to digitally unroll ancient scrolls burnt by Vesuvius Mother–infant bonds in wild chimpanzees resemble those of humans Genomics reveals zoonotic and sustained human Mpox spread in West Africa Ku limits RNA-induced innate immunity to allow Alu-expansion in primates Job Title: Associate or Senior Editor (environmental carbon cycle), Nature Communications Location(s): Shanghai, Beijing, Nanjing, New York, Jersey... Shanghai, Beijing, Nanjing, New York, Jersey City, Philadelphia, Washington DC, Milan, Madrid Junior Group Leader position with a focus on applying artificial intelligence (AI) to the field of integrative structural biology. Job Title: Associate/Senior Editor, Scientific Reports (Engineering) Location(s): Shanghai or Beijing – hybrid working model Closing date: 15th Jun... Job Title: Associate/Senior Editor, Scientific Reports (Physical Sciences) Location(s): Shanghai or Beijing – hybrid working model Closing date: 15... Zhejiang University School of Medicine warmly invites talented individuals from around the world to join us! Siberia's ancient ghost clan starts to surrender its secrets Divided by DNA: The uneasy relationship between archaeology and ancient genomics How Denisovans thrived on top of the world: mysterious ancient humans' survival secrets revealed Biggest Denisovan fossil yet spills ancient human's secrets An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            