
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-04</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09989-7'>Atlas-guided discovery of transcription factors for T cell programming</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 16:22:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. CD8+ T cells differentiate into diverse states that shape immune outcomes in cancer and chronic infection1,2,3,4. To define systematically the transcription factors (TFs) driving these states, we built a comprehensive atlas integrating transcriptional and epigenetic data across nine CD8+ T cell states and inferred TF activity profiles. Our analysis catalogued TF activity fingerprints, uncovering regulatory mechanisms governing selective cell state differentiation. Leveraging this platform, we focused on two transcriptionally similar but functionally opposing states that are critical in tumour and viral contexts: terminally exhausted T (TEXterm) cells, which are dysfunctional5,6,7,8, and tissue-resident memory T (TRM) cells, which are protective9,10,11,12,13. Global TF community analysis revealed distinct biological pathways and TF-driven networks underlying protective versus dysfunctional states. Through in vivo CRISPR screening integrated with single-cell RNA sequencing (in vivo Perturb-seq) we delineated several TFs that selectively govern TEXterm cell differentiation. We discovered new TEXterm-selective TFs, including ZSCAN20 and JDP2, with no previous known function in T cells. Targeted deletion of these TFs enhanced tumour control and synergized with immune checkpoint blockade but did not interfere with TRM cell formation. Consistently, their depletion in human T cells reduces the expression of inhibitory receptors and improves effector function. By decoupling exhaustion TEX-selective from protective TRM cell programmes, our platform enables more precise engineering of T cell states, accelerating the rational design of more effective cellular immunotherapies. Cell states are the range of cellular phenotypes arising from a defined cell type's interaction with its environment. Within the immune system, T cells possess several differentiation states, particularly as naive T cells differentiate into diverse states with different functionalities and trafficking patterns in various immune environments, such as tumours and virus infections1,2,3,4. As transcription factors (TFs) govern cell state differentiation14, understanding how TFs shape these states is essential for programming beneficial states with therapeutic potential. One promising application of cell state engineering is enhancing CD8+ T cells for adoptive cell transfer therapy (ACT) of tumour-infiltrating lymphocytes (TILs) or chimeric antigen receptor (CAR) T cells. However, identifying TFs that control CD8+ T cell states is difficult owing to substantial heterogeneity and overlapping transcriptomes, even between functionally divergent states. We focused on two transcriptionally similar yet functionally divergent states: the protective functional tissue-resident memory (TRM) cell state and the dysfunctional terminally exhausted (TEXterm) cell state. Conversely, during persistent antigen stimulation scenarios such as chronic virus infection (for example, HIV) or cancer, T cells progressively express diverse inhibitory receptors, including PD1, and lose memory potential and effector functions. This process is known as T cell exhaustion (TEX)5,6,7,8, and cells in this trajectory eventually adopt the TEXterm cell state. TEXterm cells express higher levels of diverse inhibitory receptors (for example, TIM3 and CD101), lack effector and proliferative capacity, and do not respond effectively to immune checkpoint blockade (ICB), such as anti-PD1 monoclonal antibody (mAb) blockade15,16,17. High TEXterm cell marker expression often indicates poor prognosis in solid tumours, although some markers also correlate with ICB response, highlighting their complex role in tumour immunity18,19. Despite their distinct functional effects on cancer outcomes, TEXterm and TRM cells both reside preferentially in tissues1,3 and display remarkable similarities in their transcriptional profiles, including key regulatory TFs such as BLIMP1 (refs. These two cell states even exhibit highly correlated open chromatin regions (Extended Data Fig. 1d), complicating the precise identification of TFs whose disruption may selectively inhibit TEXterm cell development while preserving TRM cell development. Given that many TFs are expressed commonly across different CD8+ T cell states and differentiation trajectories, a sophisticated and precise bioinformatics approach is crucial to pinpoint the bona fide cell-state-specifying TFs that are essential for T cell programming. a, Diagram summarizing CD8+ T cell trajectories during acute and chronic infection or tumour, highlighting differentiation into various effector, memory and exhaustion states, including parallel TRM and TEXterm lineages with overlapping tissue localization. b, Pearson correlation matrix of batch-effect-corrected RNA-seq datasets. Both colour intensity and circle size indicate correlation strength, with red denoting the highest correlation. c, Workflow of the integrative Taiji analysis. Matched RNA-seq and ATAC-seq datasets3,9,17,22,31,32,33,34,35 were used to construct a regulatory network and calculate TF activity scores using PageRank. d–h, TFs (rows) and samples (columns) are displayed as z-normalized PageRank heatmaps. Each column corresponds to a dataset. d,e, PageRank scores of genes encoding 136 single-state TFs (d) and 173 multi-state TFs (e). f–h, Bubble plots show normalized TF PageRank scores and expression for genes encoding TEXterm-selective (f), TRM -selective (g) and multi-state (h) TFs that are active in both cell states. Circle colour represents the normalized PageRank score (red, high) and circle size indicates log mRNA expression across five datasets. i,j, TF ‘waves' associated with exhaustion (i) or TRM cell differentiation (j), indicating coordinated activity of TF groups during cell state transitions. Sample sizes and statistical details for cell state definitions and TF selection criteria are provided in Extended Data Figs. We hypothesized that key TFs controlling selective CD8+ T cell differentiation could be identified through systematic comparison of TF activity across the differentiation landscape. Accurate prediction requires recognizing that TF activity does not necessarily mirror expression, as it depends on post-translational modifications, cofactors and target accessibility27, and that TF effects propagate through genetic networks. We therefore developed a multi-omics atlas integrating transcriptomic and chromatin accessibility data from nine CD8+ T cell states to understand ‘global' influences of TFs in each cell state and to identify ‘selective' or ‘shared' TFs. Our atlas-based platform can map TF communities and their target genes (‘regulatees'), guiding state-specific differentiation. Our initial objective was to create a comprehensive catalogue of TF activity across diverse CD8+ T cell states by integrating our TF activity analysis pipeline, Taiji28,29,30, with comparative statistical analysis. In Taiji, the gene regulatory network (GRN) is a weighted, directed network that models regulatory interactions between TFs and their target genes. In this GRN, each node corresponds to a gene, and its weight is proportional to the gene's expression level. To determine the global influence of each TF within the network, Taiji applies a personalized PageRank algorithm, which assigns an ‘importance' score to each node that is based on both the quantity and quality of incoming connections. This approach yields a measure of TF activity that reflects the influence of each TF in the broader regulatory landscape, accounting for upstream regulators, downstream targets and feedback loops through iterative computation. Although earlier studies provided foundational insights into cell differentiation, a more refined analysis within CD8+ T cells is needed to achieve higher resolution of TF roles. Therefore, leveraging the improved statistical filtering, we aimed to quantify the global influence of TFs across all CD8+ T cell states. To begin, we analysed assay for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing (RNA-seq) datasets from 121 CD8+ T cell samples spanning nine distinct states, using both previously published and newly generated datasets from well-characterized acute and chronic lymphocytic choriomeningitis virus (LCMV) infections3,9,17,22,31,32,33,34,35 (Extended Data Fig. Next, we conducted an unbiased comparative analysis using statistical filtering to understand the specificity of TF activity across the CD8+ T cell states (Extended Data Fig. By contrast, 173 TFs, including Tcf7 and Tbx21, were key regulators in more than one cell state, termed ‘multi-state' TF genes (Fig. TCF7 is a known driver of naive, MP and TEXprog states, all of which are multipotent with high proliferative capacity3,17. Genes encoding multi-state TFs such as Vax2, Batf, Irf8 and Stat1 were more enriched within the exhaustion-associated cell states (TEXprog, TEXeff and TEXterm). Consistent with the similarity between TEXterm and TRM cells (Fig. 1b–d), these two cell states share the most TF genes compared with other cell states (for example, Egr2, Crem and Prdm1; Extended Data Fig. Although Taiji provides a statistically grounded approach for inferring TF activity (Extended Data Fig. 2a), there is no absolute threshold for defining cell state specificity, and some misclassification is expected, particularly for TFs with overlapping functions or modest differences in activity. Still, Taiji is useful to highlight TFs with activity patterns enriched in specific cell states. For instance, although Eomes is classified as a TEXterm single-state TF gene herein, it also functions in effector, TEM, TCM and TRM cell differentiation36,37. This illustrates that more accurate classifications require further investigation and resolution, as performed herein for several TFs. Despite the strong transcriptional overlap between TEXterm and TRM cells, our Taiji pipeline predicted TFs as being selectively active in either of these two cell states. This could aid in developing better immunotherapies, in which one can engineer T cells away from exhaustion and towards more functional effector cell states without negatively affecting TRM cell formation in tissues and tumours. On the basis of statistical criteria (Extended Data Fig. TEXterm single-state TF genes included those for many previously unreported TFs, such as Zscan20, Jdp2, Zfp324, Zfp143, Zbtb49 and Arid3a (Fig. TRM single-state TF genes included Fosb, Zfp692, Atf4, Pbx4, Junb and Klf6 (Fig. Of the TEXterm and TRM multi-state TF genes, some, such as Nr4a2 (ref. 22,31), were well known to function in the development of both cell states, whereas others, such as Hic1 (ref. 39), were not, identifying them as new multi-state TFs to consider (Fig. We analysed previously reported TFs such as cJUN, BATF/BATF3 and TFAP4 that were identified from functional screening of CD8+ T cells40,41,42,43 based on limited phenotypic readouts. These previous screens tended to identify broadly active, multi-state TFs (Fig. By contrast, our platform enabled a computationally guided, multi-state screen that identified TFs predicted to have greater state-selective activity (Extended Data Fig. To evaluate the TFs that were predicted to govern selective T cell differentiation, we identified dynamic activity patterns of TF groups, termed ‘TF waves' (Extended Data Fig. TF waves reveal possible combinations of TFs that coordinate trajectories. Seven TF waves linked to specific biological pathways were identified, such as the TRM TF wave (Fig. 1i), which includes genes encoding several members of the AP-1 family (for example, Atf3, Fosb and Jun) that are associated uniquely with the TGFβ response pathway (Extended Data Fig. To uncover transcriptional programmes governing TRM or TEXterm cell differentiation, we constructed TF–TF association networks capturing functional relationships between TFs (Fig. Analysis of regulatee-based adjacency matrices (that is, predicted TF–target gene circuits) revealed shared and distinct patterns of TF collaboration across the two states. Single-state TFs displayed strong intra-state connectivity. Multi-state TFs (HIC1, PRDM1, FLI1, GFI1) that were active in both states and previously reported TFs (cJUN, BATF and TFAP4) formed distinct partnerships in each cell state, reflecting context-specific regulatory architectures (Fig. a, Overview of TF–TF network analysis encompassing association and community-level organization of TRM and TEXterm regulatory landscapes. b,c, TF–TF association networks focused on the TEXterm single-state TF ZSCAN20 (b) and the multi-state TF HIC1 (c), depicting predicted context-specific interactions in TRM (green) or TEXterm (brown) cells. d–f, Clustering of TF–TF associations identified five distinct TF communities in TRM and TEXterm networks. Shared TFs (grey) shape overall community topology (d), whereas TRM- or TEXterm-specific interactions are represented as green (e) or brown (f) edges, respectively. Pathway gene sets in Supplementary Table 8. h, Gene set enrichment analysis (GSEA) comparing TEXterm versus TRM cell pathways using batch-effect corrected LCMV bulk RNA-seq3,9,17,22,31,32,33,34,35 and human pan-cancer scRNA-seq data sets44,55,61. i–k, Flow cytometry analysis of proteasome activity showing the highest activity in TEXterm cells during LCMV–Clone-13 infection (i) and MCA-205 tumours (j). In dual transfer experiments, antigen-specific (P14) and bystander (OT-1) CD8+ T cells analysed from B16-GP33 tumours (k) show elevated proteasome activity in TEXterm-like populations. l, Functional impact of proteasome activity on tumour growth. Tumour-bearing C57BL/6 mice were infused with proteasomehigh or proteasomelow OT-1 cells pre-stimulated with B16F1-OVA tumour cells for 7 days. Proteasomehigh OT-1 cells exhibit reduced tumour control. Ordinary one-way analysis of variance (ANOVA) (i–k) and two-way ANOVA Tukey's multiple comparison test (l) were performed. We next grouped the TF–TF association networks into distinct ‘TF neighbour communities' in TRM and TEXterm cells (Supplementary Table 5), and each community was linked to specific biological processes (Fig. Although multi-state TFs shaped overall community topology, single-state TFs drove unique interaction patterns specific to TRM or TEXterm cells within each community. Pathway analysis revealed divergent programmes in each state—for instance, TRM community-3 was associated with cell adhesion and TGFβ response (Fig. 2e,g,h), whereas TEXterm community-3 was linked to apoptosis (Fig. Community-1 in TRM cells controlled RNA metabolism (Fig. 2e,g,h), whereas in TEXterm cells, it was tied to catabolism, proteolysis and autophagy (Fig. To assess the functional relevance of state-enriched pathways, we focused on the proteasome pathway, which emerged as a prominent but previously unrecognized feature of TEXterm cells (Fig. Proteasome gene signatures were enriched in TEXterm-like CD8+ T cells from patients with non-small cell lung cancer (NSCLC)44 and mouse MCA-205 TILs (Extended Data Fig. Consistently, proteasome activity—measured by a validated fluorescent probe45—was highest in TEXterm cells from chronic LCMV (Fig. 2j,k) relative to bystander OT-1 cells (Fig. To test whether high proteasome activity correlates with dysfunction, we sorted OT-1 cells by proteasome activity probe intensity and adoptively transferred them into B16F10-OVA tumour-bearing mice. 2l)—a trend also seen in endogenous TILs (Extended Data Fig. These findings support the TF–TF network and pathway predictions and identify the proteasome pathway as a functional hallmark of TEXterm cells. The Taiji pipeline enabled comparative analysis of TF activity and curated sets of single-state TFs specific to TRM versus TEXterm cells (Fig. To assess its accuracy, Perturb-seq, combining in vivo CRISPR screening with single-cell RNA-seq (scRNA-seq), was performed in two animal models for TRM or TEXterm differentiation (Figs. Our Perturb-seq guide RNA (gRNA) library targeted 19 TF genes, including 7 encoding TEXterm and TRM multi-state TFs and 12 encoding TEXterm single-state TFs. The TEXterm TF genes included one known TF (Nfatc1) and 11 others that had high specificity scores but were not previously linked to TEXterm differentiation (grey boxes; Fig. To ensure comprehensive screening, four gRNAs per target were expressed in two dual-gRNA retroviral vectors (Extended Data Fig. This created a library of 40 dual-gRNA vectors, with 76 TF-gRNAs and four gScramble controls (Supplementary Table 6). a, Schematic of the in vivo Perturb-seq strategy. Cas9+P14+ TCR transgenic CD8+ T cells recognizing the LCMV epitope GP33–41 were transduced with retrovirus-expressing gRNA libraries, adoptively transferred into mice infected previously (1 day earlier) with LCMV–Clone-13, and analysed 18–23 days later by scRNA-seq. b, UMAP showing TEXprog, TEXeff, TEXterm and cell cycle clusters; marker expression is in Extended Data Fig. TEXterm single-state TF genes are in bold. Data represent five pooled replicates from three independent experiments; values are shown as mean ± s.e.m. Statistical analysis: two-way ANOVA with Fisher's least significant difference (LSD) test compared with the gScramble control; results for TEXterm and TEXprog clusters are shown; with full comparisons in Supplementary Table 7; ****P < 0.0001, ***P < 0.001, **P < 0.01, *P < 0.05. e, Representative flowplots showing phenotyping of the TF KOs in LCMV–Clone 13-infected mice. f, Quantification of TEXterm (PD1+SLAMF6−CX3CR1−) frequencies in donor CD8+ T cells. g, Differential expression analysis of TEXterm, TEXprog and TEXeff gene signatures46 (Supplementary Table 8) across each TF KO. j, GSEA showing enrichment of effector-associated gene sets in TF KOs versus control. k,l, Functional validation: cytokine production (IFNγ, TNF) and viral titres in mice receiving TF KO versus control CD8+ T cells. Statistical analysis for f, h, i, k, l, mean ± s.e.m., ordinary one-way ANOVA with Dunnett's multiple comparison versus gScramble (f–k, n ≥ 8, at least three biological replicates; i, n ≥ 4, at least two biological replicates). a, Schematic of in vivo Perturb-seq screening during acute LCMV–Armstrong infection to assess memory CD8+ T cell differentiation. Transduced donor Cas9+P14+CD8+ T cells were analysed for TRM, TEM and TCM states in the small intestine and spleen. b, UMAP embedding of 15,211 cells identifying TCM (Il7r, Tcf7, Sell, S1pr1), TEM (Cx3cr1, Klrg1, Klf2), TRM (Cd69, Itgae, Cd160) and TRM- Itgaelow clusters. c, Differential distribution of cells across tissues. d, Kernel density map of gRNA+ cells in UMAP space. Statistical analysis, two-way ANOVA with Fisher's LSD versus gScramble. Ratio of gRNA+ cells in small intestine to spleen (g) and frequency of splenic TCM (CD62L+KLRG1−) and TEM (CD62L−KLRG1+) cells (h). P14+CD8+ T cells transduced with Klf6 or empty vector were co-transferred (approximately 1:1) into LCMV–Armstrong-infected mice. i, Representative plots pre- and post-transfer. j, Quantification of donor CD69+CD103+ TRM cells in the small intestine. Statistical tests, ordinary one-way ANOVA with Dunnett's multiple comparison versus gScramble (g,h), paired t-tests (i,j); n ≥ 4 (g,h) or n ≥ 6 (i–k) from at least two biological replicates. Cas9+ P14 CD8+ T cells were transduced with this library and transferred into mice infected with LCMV–Clone-13—a model of chronic infection and CD8+ T cell exhaustion (recipient mice also expressed Cas9 to prevent rejection of donor cells). Droplet-based sequencing was performed 18 or more days post-transfer to assess sgRNA and transcriptomes of each spleen-derived donor Cas9+ P14 CD8+ T cell (Fig. 3a), analysing 17,257 cells with unique gRNA expression. To determine which TF genes impaired TEXterm cell differentiation, we first used uniform manifold approximation and projection (UMAP). Four primary clusters were identified: TEXprog, TEXeff and TEXterm cells and those in cell cycle (Fig. TEXeff cells expressed effector markers, including Cx3cr1, Klrd1, Klrk1, Klf2 and Zeb2 (ref. The cell cycle cluster was noted for its expression of Birc5, Mki67, Stmn1 and Tuba1b. Next, we evaluated the impact of individual TF depletion by analysing the distribution of gRNA+ cells across exhaustion states (Fig. CRISPR knockout (KO) of most of the 19 TEXterm-driving TF genes led to a reduction in TEXterm cell frequency. Notably, KOs of multi-state TF genes such as Hic1, Stat3, Prdm1 and Ikzf3 (which encodes AIOLOS) resulted in a profound reduction of approximately 90% in TEXterm differentiation. Other new candidates, such as Etv5, Arid3a, Zfp410, Foxd2 and Prdm4, also reduced TEXterm representation by 25–40%, although some did not reach statistical significance. This Perturb-seq analysis highlights the platform's ability to identify TFs that regulate the TEXterm state, with most tested TFs influencing exhaustion to varying degrees. To further assess how KO of TEXterm-driving TF genes affect CD8+ T cell exhaustion, we used flow cytometry and scRNA-seq to analyse TF KO cells during LCMV–Clone 13 infection (Fig. We tested six TF KOs, including known control (Prdm1) and five newly identified TF genes (Zscan20, Jdp2, Zfp324, Stat3, Hic1) that impaired TEXterm state differentiation in Perturb-seq. Disrupting these TFs reduced TEXterm cell (PD1+CX3CR1−SLAMF6−) frequency by around 50% (Fig. All 19 TEXterm-TF gene KOs exhibited a marked decrease in TEXterm-signature genes46, including Cd7, Cxcr6, Nr4a2 and Entpd1 (Fig. Finally, the TEXterm-driving TF gene KOs were grouped according to their effects on TEXprog (PD1+CX3CR1−SLAMF6+; Fig. Loss of Prdm1 and Stat3 markedly increased the frequency of TEXprog cells and upregulated TEXprog signature genes (Fig. 3g,h) whereas loss of Hic1, Zscan20, Zfp324 or Jdp2 expanded primarily the TEXeff cell population and effector signature genes (Fig. Deletion of the Zscan20 and Jdp2 significantly enhanced effector cytokine production (for example, interferon gamma (IFNγ) and tumour necrosis factor (TNF)) and reduced viral loads in recipient mice (Fig. A principal goal of this work was to identify TFs that selectively repress TEXterm cell differentiation without affecting TRM differentiation, thereby enabling more precise programming of CD8+ T cell states. As nearly all the predicted TEXterm single-state TFs impaired TEXterm differentiation to some degree (Fig. 3), the next step was to evaluate their effects on TRM differentiation to confirm their selective activity. We used the same Perturb-seq library as before, but this time included only the eight TEXterm single-state TFs and seven multi-state TFs that impaired TEXterm state development by more than 25% in chronic LCMV infection (Fig. To assess their impact on memory CD8+ T cell development, we isolated retrovirus-transduced Cas9+ P14 CD8+ T cells from the spleen and small intestine of mice 18 days after acute LCMV–Armstrong infection. We then analysed 15,211 cells using scRNA-seq to determine how these perturbations affected the formation of intestinal TRM cells, as well as circulating splenic TCM and TEM cells (Fig. The UMAP analysis identified four primary clusters containing cells with features of TCM (Il7r, Tcf7, Sell and S1pr1), TEM (Cx3cr1, Klrg1 and Klf2) and TRM cells (Cd69, Cd160 and Itgae (encoding CD103)) as well as a small TRM cell population with lower Itgae but higher Ifng and Irf1 expression designated TRM-Itgaelow (refs. Examination of the gRNA+ cells revealed that none of the eight TEXterm single-state TF gene KOs (Zfp324, Irf8, Zfp410, Nfatc1, Zscan20, Jdp2, Arid3a and Etv5) negatively affected TRM formation significantly (bold gene names in Fig. 7c), perturbation of the TEXterm single-state TFs did not substantially alter TRM-signature gene expression. The platform also predicted new multi-state TFs, including those encoded by Hic1 and Gfi1. Disruption of these multi-state TFs significantly reduced TRM cell frequency (Fig. 7c), mirroring the effects of disruption of Prdm1, which encodes a known multi-state TF for TRM and TEXterm cells. To further validate the Perturb-seq data, we depleted the TEXterm single-state TF genes Zscan20 and Jdp2 and the multi-state TF gene Prdm1 individually in Cas9+ P14 CD8+ T cells, transferred them adoptively into LCMV–Armstrong infected animals, and assessed their differentiation into TCM, TEM and TRM cells using flow cytometry (Fig. Deletion of Zscan20 and Jdp2 did not alter the formation of any memory cell subtypes, whereas perturbation of Prdm1 reduced TRM and increased TCM formation significantly, as expected. Altogether, this multi-omics pipeline predicted TEXterm single-state TFs that drive TEXterm differentiation without affecting TRM cell formation and multi-state TFs that influence both cell states. These results demonstrate the accuracy and predictive power of our approach for pinpointing single-state and multi-state TFs. To further demonstrate the utility of our cell-state selective TF identification pipeline in discovering new TRM-associated TFs, we evaluated Klf6, which was identified through our Taiji analysis as a TRM single-state TF gene (Fig. We considered whether overexpressing Klf6 (Klf6-OE) would enhance TRM formation during acute viral infection without worsening terminal exhaustion in chronic infection. When empty-vector control and Klf6-OE P14 CD8+ T cells were co-transferred, Klf6-OE cells robustly outcompeted control cells, resulting in 15-fold enrichment in the small intestine compared with controls (Fig. Furthermore, there werearound 42 times more CD69+CD103+ double-positive TRM-like cells in Klf6-OE than in control donor cells, indicating that Klf6-OE markedly increased TRM development in the small intestine (Fig. Klf6-OE did not increase terminal exhaustion during chronic infection (Fig. This work not only identifies KLF6 as a new TRM-driving TF but also confirms its selectivity. This platform predicted cell-state-selective TF activity and identified TEXterm single-state TFs as targets for engineering T cells that resist exhaustion yet retain effector and memory functions—offering new strategies to improve immunotherapy efficacy. Given that TRM cells are associated with better clinical outcomes in solid tumours9,10,11,12, we hypothesized that KO of exhaustion-selective TF genes such as Zscan20 could be more effective than targeting TRM and TEXterm multi-state TF genes such as Hic1. Using an ACT model, we transferred TF gRNA retrovirus-transduced Cas9+ P14 CD8+ T cells into mice with established melanoma tumours expressing GP33–41 (Fig. Unlike depletion of the multi-state TF gene Hic1, depleting the TEXterm single-state TF gene Zscan20 resulted in improved tumour control (Fig. To control for inter-mouse variability in antigen load, we co-transferred Zscan20 or Hic1 KO cells with control P14 CD8+ T cells into the same B16-GP33 tumour-bearing mice (Fig. Both KOs significantly increased the frequency of PD1+SLAMF6+TIM3− cells and decreased the frequency of TIM3+ exhausted cells and the TEXterm cell state (PD1+SLAMF6−CX3CR1−) compared with controls (Fig. 8f–h), consistent with their predicted activity in TEXterm cells (Fig. However, Zscan20 KO robustly enhanced effector marker expression (CX3CR1), granzyme B and cytokine production in TILs, whereas Hic1 KO did not seem to improve effector function to the same degree (Fig. Thus, despite their similar effects on suppressing TEXterm cell differentiation in tumours, differences in their ability to promote functional effector-like states may underlie the differential tumour control observed. Given that HIC1 functions as a multi-state TF and ZSCAN20 as a single-state TF, these findings support the general rationale for targeting state-specific TFs to enable more selective programming of T cell differentiation. a, Experimental design and tumour outcomes from adoptive transfer of P14 CD8+ T cells carrying CRISPR KOs of Zscan20 (TEXterm single-state TF gene) or Hic1 (multi-state TF gene active in TEXterm and TRM cells) into B16-GP33 melanoma-bearing mice. Tumour volumes and terminal weights are shown. b, Co-transfer design mixing Zscan20-KO or Hic1-KO Cas9+ P14 cells with scramble controls before transfer. e, Human pan-cancer single-cell multi-omics and scRNA-seq datasets48,49,50,51,52,53,54,55 were integrated to assess TF expression and activity across CD8+ T cell states using scTaiji. f, Paired scRNA-seq and scATAC-seq were used to build regulatory networks and compute PageRank TF activity scores. Shown are normalized scores for TEXterm single-state TF genes (Fig. 1f) with conserved DNA-binding motifs in humans. g, mRNA expression of TEXterm TF genes across TEXterm and TRM clusters in human tumours; cross-species conserved TF genes are in bold. h, Human peripheral blood mononuclear cell (PMBC) KO design. ZSCAN20-KO or JDP2-KO CD8+ T cells were stimulated with anti-CD3/CD28 beads for 18 days to model chronic activation. m, Schematic of adoptive transfer and anti-PD1 treatment testing synergy with TEXterm TF gene KO. Cas9+ P14 cells (±TF KO) were transferred into B16-GP33 tumours and treated with anti-PD1 or IgG2a. D7, day 7; D25, day 25. n,o, Tumour growth and weights for Zscan20-KO (n) and Jdp2-KO (o) versus controls. ; n ≥ 6 from at least two biological replicates. Statistics, two-way ANOVA with Tukey's (tumour volume in a, n, o); one-way ANOVA with Dunnett's (i–k, tumour weights in a, n, o); paired t-tests (c, d); two-way ANOVA with Dunnett's (l). To evaluate the relevance of our mouse findings in human T cells—particularly for applications in immunotherapy—we conducted cross-species validation using publicly available single-cell multi-omics and scRNA-seq datasets from human tumour-infiltrating CD8+ T cells (Fig. Leveraging the Taiji TF analysis platform, we mapped mouse TEXterm-associated and TRM-associated TF genes onto a curated human pan-cancer CD8+ T cell atlas encompassing six tumour types48,49,50,51,52,53,54,55 (glioblastoma (GBM), head and neck squamous cell carcinoma (HNSCC), basal cell carcinoma (BCC), hepatocellular carcinoma (HCC), renal cell carcinoma (RCC) and clear cell renal cell carcinoma (ccRCC)). Human CD8+ T cells were clustered into heterogeneous cell states, including TRM and TEXterm clusters (Fig. Taiji analysis revealed strong cross-species conservation: TEXterm TF genes such as JDP2, ZNF410 and FOXD2 exhibited higher activity in TEXterm clusters than in TRM-like cells (Fig. Similarly, TRM-specific TF genes (for example, NR4A1, KLF6 and FOSB) displayed enriched activity in the human TRM cluster (Extended Data Fig. Furthermore, 22 of the 30 mouse TF genes that were active in both TEXterm and TRM states showed similar activity profiles in human datasets (Extended Data Fig. A few TF genes—such as ZSCAN20—could not be assessed in the Taiji analysis because of missing DNA-binding motifs, but comparative RNA profiling across 15 tumour types supported their relevance, with 24 of 34 mouse TEXterm single-state TF genes, including ZSCAN20 and JDP2, showing higher expression in human TEX cells (Fig. Given these correlations between species, we perturbed ZSCAN20 and JDP2 to assess the relevance of TEXterm single-state TFs in human T cells (Extended Data Fig. Following repeated CD3/CD28 stimulation over 18 days to simulate chronic activation (Fig. 5h), ZSCAN20- or JDP2-deficient CD8+ T cells exhibited increased expression of CCR7 (naive/stem cell memory/TCM marker) and decreased levels of inhibitory receptors, including LAG3, PD1 and TIM3 (Fig. These KO cells also produced higher levels of effector cytokines (Fig. 5k,l), indicating that ZSCAN20 and JDP2 contribute to exhaustion-associated features in human CD8+ T cells. Tumours with high TEXterm cell infiltration often exhibit poor responses to ICB therapy16. We considered whether targeting TEXterm single-state TFs could enhance ICB efficacy. Among the TEXterm-associated TF genes, Zscan20 and Jdp2 were prioritized for their conservation and functional relevance in human T cells (Fig. To test synergy with ICB, treatment began 1 day after adoptive transfer of TF-depleted P14 CD8+ T cells (Fig. The combination of Zscan20 or Jdp2-KO with anti-PD1 therapy significantly reduced tumour burden (Fig. 5n,o) and improved survival (Extended Data Fig. These findings suggest that selectively disrupting TEXterm single-state TFs represents a promising strategy to enhance T cell therapy by minimizing dysfunctional states while preserving beneficial T cell phenotypes. Overall, our cross-species multi-omics and functional perturbation approach underscores the translational potential of Taiji-identified TFs for improving ACT. Our study introduces a powerful platform for identifying TFs that are pivotal in guiding specific CD8+ T cell state differentiation during viral infections and tumour progression. Leveraging our comprehensive transcriptional and epigenetic atlas from nine distinct CD8+ T cell states, we developed a detailed map of TF activity, creating a unique TF fingerprint for each context. Furthermore, we developed TaijiChat, a web interface for natural language queries of our datasets and literature (Supplementary Methods). Focusing on two critical cell states TEXterm and TRM T cells, we examined similarities and differences of TF activity and their networks in both states and engineered T cells to resist exhaustion while retaining functionality of TRM cells. Using in vivo Perturb-seq, we validated TF activity for TEXterm and TRM cells in both acute and chronic infection models. Although recent CRISPR screenings in CD8+ T cells have identified TFs that are important in cytotoxicity, memory formation40,41,42, cell enrichment56 and exhaustion57, a systematic and context-dependent understanding of TF roles across several contexts has been lacking. Our study addresses this gap by generating an accurate catalogue of CD8+ T cell state-defining TF genes, enabling cost-effective validation of predicted TF activity and selectivity using Perturb-seq. Furthermore, our study offers broader and new insight into context-dependent TF regulation. Previously, differential TF cooperation in different contexts was reported25,42,43. We extend this by analysing global TF associations across cell states, revealing how TF communities regulate T cell-specific pathways, including protein catabolism in T cell exhaustion, which aligns with previous research on protein homoeostasis45,58,59. These TF networks reveal how various cellular processes are controlled differentially between TRM and TEXterm cells, providing a rationale for their different functional capabilities within tissues. One of the key outcomes of this study was the identification of new TFs, including ZSCAN20 and JDP2, as TEXterm single-state TFs and KLF6 as a TRM single-state TF, and of newly uncovered roles for multi-state TFs such as HIC1 and GFI1. Perturbing TEXterm single-state TFs not only prevented T cell exhaustion but also preserved the ability of these cells to differentiate into effector and memory states. This led to significant improvements in tumour control. To evaluate the clinical importance of the newly discovered TFs and the catalogue of TFs with TEXterm and TRM selectivity, we confirmed cross-species conservation of a substantial number of TFs using Taiji analysis of a human pan-cancer multi-omics atlas, along with comparative expression analysis across pan-cancer scRNA-seq datasets. Furthermore, we demonstrated enhanced human T cell function following perturbation of the TEXterm single-state TFs ZSCAN20 and JDP2. Depletion of these TFs shows synergistic effects with ICB therapy, leading to significant tumour regression. These findings highlight a promising strategy for enhancing antitumour immunity through precise cell-state programming. Our TF atlas-guided platform can offer optimized ‘TF recipes' for cell programming with increased precision, robustness and durability. Future strategies could integrate enforced expression of TFs that promote favourable states, such as KLF6 for TRM differentiation or other TFs identified through systematic gain-of-function screenings40,41,42,60 with targeted depletion of TEXterm TFs. Such recipes can be refined with AI models. In summary, although our study focuses on CD8+ TEXterm and TRM cell differentiation, the pipeline for identifying single-state TFs and ‘TF recipes' can be adapted for other cell types, expanding cell therapy applications. CD8+ T cell samples were collected from ten datasets, including those generated in this study (Extended Data Fig. In total, we analysed 121 experiments, comprising 52 ATAC-seq and 69 RNA-seq datasets, which were integrated to generate paired samples and served as input for the Taiji pipeline. The samples encompassed nine distinct CD8+ T cell subtypes: naive, TE, MP, TRM, TEM, TCM, TEXprog, TEXeff and TEXterm. Cell states were defined on the basis of established surface marker combinations and LCMV-specific tetramers, including IL7R, KLRG1, PD1, SLAMF6, CD101, Tim3, CD69, CD103, H2-Db LCMV GP33–41 and H2-Db LCMV GP276–286 or congenic markers for P14 (T cell receptor (TCR) specific for the LCMV GP33–41 peptide CD8+ T cells), in the context of either acute (LCMV–Armstrong) or chronic (LCMV–Clone 13) infection models. A complete summary of dataset sources, accession numbers, infection conditions and corresponding cell state definitions (sorting gates) is provided in Supplementary Table 1 and Extended Data Fig. To perform integrative analysis of RNA-seq and ATAC-seq data, we developed Taiji v.2.0, which allows visualization of several downstream analysis–TF wave, TF–TF association and TF community analysis. Epitensor was used for the prediction of chromatin interactions. Putative TF binding motifs were curated from the latest CIS-BP database61. In this analysis, 695 TF genes were identified as having binding sites centred around ATAC-seq peak summits. The average number of nodes (genes) and edges (interactions) of the genetic regulatory networks across CD8+ T cell states were 15,845 and 1,325,694, respectively, including 695 (4.38%) TF nodes. We first identified universal TF genes with mean PageRank across nine cell states ranked as top 10% and coefficients of variation less than 0.5. In total, 54 universal TF genes were identified (Supplementary Table 1). To identify single-state TF genes, we divided the samples into two groups: target and background. We then performed the normality test using Shapiro-Wilk's method to determine whether the two groups were distributed normally, and we found that the PageRank scores of most (90%) samples followed a log-normal distribution. A P value cut-off of 0.05 and log2 fold change (log2FC) cut-off of 0.5 were used for calling lineage-specific TFs. In total, 255 specific TF genes were identified (Supplementary Table 2). Depending on whether the TF gene appeared in several cell states, they could be divided further into multi-state TF genes (Fig. Out of 255 single-state TF genes, 84 appear in TEXterm or TRM cells. To identify the truly distinctive TF genes between TEXterm and TRM, we performed a second round of unpaired t-tests, between only TEXterm and TRM cells (Supplementary Table 3). The same cut-offs, that is, P value of 0.05 and log2FC of 0.5, were applied to select TEXterm single-taskers and TRM single-taskers. Out of 84 TF genes that did not pass the cut-off, 30 were identified as TEXterm and TRM multi-taskers. The full workflow is summarized in Extended Data Fig. Combined with previous knowledge of the T cell differentiation path, TF waves are combinations of TFs that are particularly active in certain differentiation stages, revealing possible mechanisms of how TF activities are coordinated during differentiation. To be more specific, we clustered the TFs based on the normalized PageRank scores across samples. First, we performed principal component analysis (PCA) for dimensionality reduction of the TF score matrix. We retained the first ten principal components for further clustering analysis, which explained more than 70% of the variance (Extended Data Fig. We used the k-means algorithm for clustering analysis. To find the optimal number of clusters and similarity metric, we performed Silhouette analysis to evaluate the clustering quality using five distance metrics: Euclidean distance, Manhattan distance, Kendall correlation, Pearson correlation and Spearman correlation (Extended Data Fig. Pearson correlation was the most appropriate distance metric, as the average Silhouette width was highest of all five distance metrics. On the basis of these analyses, we identified seven distinct dynamic patterns of TF activity during immune cell development. We further performed functional enrichment analysis to identify gene ontology (GO) terms for these clusters. To build the TF–TF association networks, we first defined a set of relevant TFs for each context (TEXterm and TRM) by combining cell state-important and single-state TF genes, resulting in 159 TFs for TEXterm and 170 for TRM cells. The analysis was based on a TF–regulatee network derived from Taiji, where we first consolidated sample networks by averaging the edge weights for each TF–regulatee pair. To reduce noise, regulatees with low variation across all TFs (s.d. Subsequently, a TF–TF correlation matrix was generated by calculating the Spearman's correlation of edge weights for each TF pair across their common regulatees. From this matrix, we constructed a graphical model using the R package ‘huge'62, which uses the Graphical Lasso algorithm and a shrunken empirical cumulative distribution function estimator. An edge between two TFs was established if their correlation was deemed significant by the model, controlled by a lasso penalty parameter (lambda) of 0.052. This value was chosen as it represents a local minimum on the sparsity lambda curve, resulting in approximately 15% of TF–TF pairs being connected. To validate this method, we estimated the false discovery rate by generating a null model through random shuffling of the TF–regulatee edge weights. Applying our algorithm to this null data identified zero interactions, confirming that our approach has a very low false discovery rate. Following construction of TF–TF association networks, we identified functionally related TF communities within each network. This procedure identified five distinct communities for each context (TEXterm and TRM). For GSEA, the genes were first ranked by the mean edge weight in corresponding samples and H, C5, C6 and C7 collections from the Molecular Signatures Database were used for annotation. A cut-off of P < 0.05 was used to select the significantly enriched GO terms and KEGG pathways. We reasoned that (1) log2FC in expression due to TF KO and (2) TF–gene regulatory edge weights could be combined to provide heuristic scores for the regulatory effect of a TF on a target gene. For each guide RNA KO, Seurat's FindMarkers() function was used to quantify log2FC in the expression of a gene with respect to the gScramble condition. Regulatees of a TF were annotated as high-confidence if the magnitude log2FC of the regulate exceeded 0.58 (corresponding to a fold change of 1.5 or its reciprocal) and if the edge weight belonged to the upper quantile of all edge weights attributed to the TF. TF activity was inferred using the Taiji pipeline applied to matched scRNA-seq and scATAC-seq datasets from various human cancers, including ccRCC (n = 2; PRJNA768891, GSE240822), GBM (GSE240822), BCC (GSE123814, EGAS00001006141), HNSCC (GSE139324, EGAS00001006141), HCC (GSE125449, EGAS00001006141) and RCC (PMID: 30093597; EGAS00001006141). Cell types were annotated using canonical marker gene expression and categorized into six main CD8+ T cell states: TEX, TEXProg (progenitor exhausted), TEff (effector), TRM, TCM/naive and proliferating. PageRank scores derived from Taiji were log-transformed, averaged within each T cell category, and then standardized using z-score normalization. Results were visualized with a focus on TF gene activity in TRM and TEX populations. To assess TF gene expression across diverse T cell states, raw count matrices from a published pan-cancer CD8+ T cell atlas (GSE156728) were reprocessed using Seurat's standard workflow. The dataset encompassed T cells from 11 tumour types, including BC, BCL, CHOL, ESCA, FTC, MM, OV, PACA, RC, THCA and UCEC. Cell type annotations provided by the original study were retained and mapped to the following broad categories: TRM, TEX, TEM (effector memory), TM (memory), naive and TK (cycling). Seurat's AverageExpression function was used to compute average logCPM expression for each TF gene in each T cell category, followed by z-score normalization. Data visualization emphasized comparisons between TRM and TEX subsets. C57BL/6/J, OT-1 (C57BL/6-Tg(TcraTcrb)1100Mjb/J), B6.Cg-Rag2tm1.1Cgn/J and CD45.1 (B6.SJL-PtprcaPepcb/BoyJ) mice were purchased from Jackson Laboratories. Mice were infected with 2 × 105 plaque-forming units (PFU) LCMV–Armstrong by intraperitoneal injection or 2 × 106 PFU LCMV Clone-13 by retro-orbital injection under anaesthesia. LCMV fluorescence focus unit titration was performed seeding Vero cells at a density of 30,000 cells per 100 µl in a 96-well flat-bottom plate in DMEM + 10% fetal bovine serum (FBS) + 2% HEPES + 1% penicillin-streptomycin. On the next day, tissues were homogenized on ice, spun down at 1,000g for 5 min at 4 °C and supernatants or serum were diluted in tenfold steps. Diluted samples were added to Vero cells and incubated at 37 °C, 5% CO2 for around 20 h. Subsequently, inocula were aspirated and wells were incubated with 4% paraformaldehyde for 30 min at room temperature before washing with PBS. VL-4 antibody (BioXCell) was conjugated using the Invitrogen AF488 conjugation kit and added to the wells in dilution buffer containing 3% BSA and 0.3% Triton (ThermoFisher Scientific) in PBS. Cells were incubated at 4 °C overnight before washing with PBS and counting foci under the microscope. The number of focus forming units was calculated using the formula: focus forming units per millilitre = number of plaques/(dilution× volume of diluted virus added to the plate). Spleens were dissociated mechanically with 1-ml syringe plungers over a 70-µm nylon strainer. Spleens were incubated in ammonium chloride potassium buffer for 5 min. For isolation of small intestinal intraepithelial lymphocytes, Peyer's patches were first removed by dissection. Pieces were incubated in 30 ml HBSS with 10% FBS, 10 mM HEPES and 1 mM dithioerythritol with vigorous shaking at 37 °C for 30 min. Supernatants were collected, washed and isolated further using 40%/67% discontinuous Percoll density centrifugation for 20 min at room temperature with no brake. B16-GP33 melanoma cell lines were cultured in DMEM (Invitrogen) with 10% FBS, 1% penicillin-streptomycin and 250 μg ml−1 G418 (Invitrogen, catalogue no. The MCA-205 tumour line (Sigma) was maintained in RPMI supplemented with 10% FBS, 300 mg l−1 l-glutamine, 100 U ml−1 penicillin, 100 mM sodium pyruvate, 100 μM non-essential amino acids, 1 mM HEPES, 55 μM 2-mercaptoethanol and 0.2% plasmocin mycoplasma prophylactic (InvivoGen). All the tumour cell lines were used for experiments when in the exponential growth phase. For in vitro T cell culture, splenocytes were activated in RPMI 1640 medium (Invitrogen) containing 10% FBS and 1% penicillin-streptomycin, 2 mM l-glutamine, 0.1 mg ml−1 GP33, beta-mercaptoethanol 50 mM and 10 U ml−1 IL-2. 5n,o) were injected subcutaneously in 100 μl PBS. Around 0.5–1 × 106 Cas9+ P14 T cells with CD45.1 markers were transferred to tumour on day 7 without pre-radiation of tumour-bearing mice. Tumours were measured every 2–3 days post-tumour engraftment for indicated treatments and sizes calculated. For antibody-based treatment, tumour-bearing mice were treated with anti-PD1 antibody (200 µg per injection, clone RMP1-14, BioXcell) twice per week from day 7 post-tumour implantation. Tumour growth was measured twice per week with calipers. Survival events were recorded each time a mouse reached the endpoint (tumour volume greater than or equal to 1,500 mm3). Tumour weights were measured on day 23 for Fig. All experiments were conducted according to the Salk Institute Animal Care and Use Committee and the University of North Carolina at Chapel Hill Animal Care and Use Committee. 5, tumours were minced into small pieces in RPMI containing 2% FBS, DNase I (0.5 µg ml−1; Sigma-Aldrich), and collagenase (0.5 mg ml−1; Sigma-Aldrich) and kept for digestion for 30 min at 37 °C with 70-µm cell strainers (VWR). Filtered cells were incubated with ammonium chloride potassium lysis buffer (Invitrogen) to lyse red blood cells, mixed with excess RPMI 1640 medium (Invitrogen) containing 10% FBS and 1% penicillin-streptomycin, and centrifuged at 400g for 5 min to obtain a single-cell suspension. For experiments involving the Proteasome Activity Probe (R&D systems), cells of interest were incubated with the probe at concentration of 2.5 mM for 2 h at 37 °C in PBS. Samples were washed and then stained with Zombie NIR viability dye (Biolegend) in PBS at 4 °C for 15 min. 2, MCA-205 fibrosarcomas (2.5 × 105) were established by subcutaneous injection into the right flank of C57BL/6 mice. After 12–14 days of tumour growth, spleens, draining lymph nodes and tumours from groups of mice were collected and tumours were processed using the Mouse Tumor Dissociation Kit and gentleMACS dissociator (Miltenyi Biotec) according to the manufacturer's protocol. For purification experiments, samples were pre-enriched using the EasySep Mouse CD8+ T Cell Isolation Kit (Stemcell Technologies) according to the manufacturer's protocol, stained with Live-or-Dye PE Fixable Viability Stain (Biotium) and CD8a-APC (Invitrogen) and live CD8+ cells were sorted using the FACSAria II cell sorter. For adoptive cellular therapy experiments, B16-GP33 melanomas were established subcutaneously by injecting 5.0 × 105 cells into the right flank of CD45.1 mice and tumour-bearing hosts were irradiated with 5 Gy 24 h before T cell transfer. 5 were not irradiated before T cell transfer. After 7 days of tumour growth, 1.5 × 106 CD45.2 OT-1 T cells and 1.5 × 106 CD45.1/CD45.2 P14 T cells were infused in 100 μl PBS into the tail vein in tumour-bearing mice. Tumours were collected 14 days after adoptive cell transfer and CD8 TILs were analysed for proteasome activity. All experiments were conducted in accordance with the guidelines of the University of North Carolina at Chapel Hill Animal Care and Use Committee. For the adoptive transfer experiment involving proteasomehigh and proteasomelow tumour-specific OT-1 T cells (Fig. 2l), whole splenocytes from OT-1 mice were activated with 1 μg ml−1 OVA_257–264 peptide and expanded for 7 days in the presence of 200 U ml−1 rhIL-2 (NCI). On day 7, OT-1 cells were FACS-sorted based on proteasome activity to isolate proteasomehigh and proteasomelow OT-1 populations. A total of 2.5 × 105 sorted OT-1 cells were injected into C57BL/6 mice bearing B16F1-OVA melanomas. Tumours were established by subcutaneous injection of 3 × 105 B16F1-OVA cells into the right flank 7 days before T cell transfer. Recipient mice were preconditioned with 5 Gy total body irradiation 24 h before adoptive transfer. Tumour growth was measured every other day with calipers. After 14 days of tumour growth, live CD45+CD8+CD44+PD1+ T cells were sorted from tumours on the basis of proteasome activity (high versus low) using the FACSAria II cell sorter. A total of 2.5 × 104 cells were then injected into the 2-day MCA-205-bearing RAG2−/− hosts (n = 5 per group) and tumour growth was monitored every other day starting on day 4. All experiments were conducted in accordance with the guidelines of the University of North Carolina at Chapel Hill Animal Care and Use Committee. For overexpression of the gRNA retrovirus vector, 293T cells were transfected with the Eco-helper and MSCV gRNA vectors. Donor P14 splenocytes were activated in vitro by 0.1 mg ml−1 GP33 and 10 U ml−1 IL-2 at 37 °C for 24 h, then spin-transduced (1,500g) with fresh retrovirus supernatant from 293T cells for 90 min at 30 °C in the presence of 5 μg ml−1 polybrene. Naive CD8+ T cells were enriched from spleen using the EasySep Mouse CD8+ T cell Isolation Kit (Stemcell Technologies). sgRNAs targeting Zscan20, Jdp2, Etv5, Prdm1 and Hic1 genes or the mouse or human genome non-targeting scramble (control) were obtained from Synthego, Integrated DNA technologies (IDT) and GeneScript (Supplementary Table 5). Nucleofection of naive CD8+ T cells was performed using a Lonza P3 primary cell kit and program DN100 with 4D-Nucleofector (Lonza Bioscience) for mouse and EO115 for human stimulated T cells. The cells were rested at 37 °C for 3 min. For in vivo adoptive transfer, cells were resuspended in PBS at the desired concentration and transferred adoptively into recipient mice. Genomic DNA was isolated from both KO-induced CD8+ T cells and control cells using a Quick-DNA MicroPrep Kit (Zymo). Genomic DNA concentrations were quantified using a NanoDrop One spectrophotometer (ThermoFisher Scientific). The PCR products were resolved on a 2% agarose gel with SYBR Safe DNA Gel Stain (Invitrogen), and the appropriate bands on the gel were extracted and purified with a Gel DNA Recovery Kit (Zymo). Concentrations of purified amplicon samples were measured and then sent for sequencing with primers designed using Benchling's Primer3 tool. The samples with KOs were compared with wild-type controls using EditCo's Ice Analysis software, providing the indel percentages, KO score and the indel distributions used to assess editing efficiency. Indel percent ranged from 56% to 97%, and the KO score throughout experiments ranged from 32 to 74. Both single-cell suspensions were incubated with Fc receptor-blocking anti-CD16/32 (BioLegend) on ice for 10 min before staining. Cell suspensions were first stained with Red Dead Cell Stain Kit (ThermoFisher) for 10 min on ice. Surface proteins were then stained in FACS buffer (PBS containing 2% FBS and 0.1% sodium azide) for 30 min at 4 °C. To detect cytokine production ex vivo, cell suspensions were resuspended in RPMI 1640 containing 10% FBS, stimulated by 50 ng ml−1 phorbol 12-myristate 13-acetate and 3 μM ionomycin in the presence 2.5 μg ml−1 Brefeldin A (BioLegend, catalogue no. Cells were processed for surface marker staining as described above. 554714) for 30 min at 4 °C, then washed with 1× permeabilization buffer (Invitrogen, catalogue no. 00-5521-00) for 30 min at 4 °C, then washed with 1× permeabilization buffer. Cells were then stained with intracellular antibodies for 30 min at 4 °C. Samples were processed on an LSR-II flow cytometer (BD Biosciences) and data were analysed with FlowJo v.10 (TreeStar). Cells were sorted either on a FACSAria III sorter or a Fusion sorter (BD Biosciences). against mouse proteins were used: anti-CD8a (53-6.7), anti-PD1 (29F.1A12), anti-CX3CR1 (SA011F11), anti-SLAMF6 (13G3), anti-CD38 (90), anti-CD39 (24DMS1), anti-CD101 (Moushi101), anti-KRLG1 (2F1), anti-CD69 (H1.2F3), anti-CD103 (M290), anti-CD62L (MEL-14), anti-TIM3 (RMT3-23), anti-Ly5.1 (A20), anti-Ly5.2 (104), anti-IFNγ (XMG1.2) and anti-TNF (MP6-XT22). against human proteins were used: anti-CD8a (RPA-T8), anti-CD4 (SK3), anti-CD45RA (H100), anti-CD45RO (UCHL1), anti-CCR7 (G043H7), anti-CD62L (DREG-56), anti-CD69 (FN50), anti-CD103 (Ber-ACT8), anti-CXCR6 (K041E5), anti-PD1 (EH12.2H7), anti-CD38 (HIT2), anti-CD39 (A1), anti-LAG3 (11C3C65), anti-TIM3 (F38-2E2), anti-TIGIT (A15153G), anti-IFNγ (4S.B3), anti-TNF (MAb11), anti-IL-2 (JES6-5H4), anti-GZMB (QA16A02) and anti-G4S Linker (E7O2V). Antibodies were purchased from Invitrogen, Biolegend, Cell Signaling or eBiosciences. To assess the functional impact of individual TF gene KOs in CD8+ T cells, we used Cas9-expressing P14 donor cells (LCMV-specific TCR transgenic mice, CD45.1 congenic) transduced with green fluorescent protein (GFP)-expressing retroviral vectors encoding individual gRNAs. Transductions were performed on the day of adoptive transfer without previous sorting. Without sorting, transduced donor cells (0.5–1 × 105) were transferred immediately into congenically distinct Cas9-expressing wild-type recipient mice (CD45.2) infected 1 day previously with either LCMV–Clone 13 or LCMV–Armstrong strains. At least day 20 post-infection, spleens from the Clone 13 model and spleens and small intestines from the Armstrong model were collected. Single-cell suspensions were prepared and analysed by flow cytometry. Successfully transduced (gRNA+) cells were identified by GFP expression, which ranged from 10% to 70% of P14 CD8+ T cells across experiments. Because of variability in the number of GFP+ donor P14 CD8+ T cells obtained from different experiments, all phenotypic analyses were performed in the GFP+(gRNA+)CD45.1+CD8+ population. PD1 positive and negative cells, exhaustion subsets (TEXterm:PD1+SLAMF6−CX3CR1− and TEXprog:PD1+SLAMF6+CX3CR1−, TEXeff:PD1+CX3CR1+) or expression of phenotypic markers was reported as a percentage within the gRNA+(GFP+) P14 CD8+ T cell population to ensure consistency across samples. Naive CD8+ T cells were isolated from the spleens and lymph nodes of Cas9-expressing LCMV TCR transgenic (Cas9 P14) or P14 mice using an EasySep Mouse CD8+ T Cell Isolation Kit (STEMCELL Technologies). Purified P14 cells were activated for about 24 h on plates coated with goat anti-hamster IgG (ThermoFisher), followed by 1 μg ml−1 hamster anti-mouse CD3 and 1 μg ml−1 hamster anti-mouse CD28 antibodies (ThermoFisher), in complete T cell medium (RPMI 1640 supplemented with 10% FBS (HyClone), 55 μM 2-mercaptoethanol, 100 IU ml−1 penicillin-streptomycin and 1% HEPES). After activation, cells were transduced with retroviruses encoding Klf6 overexpression or gRNAs targeting Hic1 or Zscan20 and cultured with 20 IU ml−1 IL-2, 2.5 ng ml−1 IL-7 and 2.5 ng ml−1 IL-15 (PeproTech). At 48 h post-transduction, reporter expression was confirmed by flow cytometry. Donor cell mixes were prepared using control versus Klf6-overexpressing cells (Fig. For tumour studies, 5 × 105 to 1  × 106 transduced T cells (gScramble versus gTF) were transferred on day 7 after B16-GP33 tumour implantation. All experiments were conducted according to guidelines of the University of North Carolina at Chapel Hill Animal Care and Use Committee. To generate a dual-guide sgRNA vector (MSCV-hU6-mU6-SV40-EGFP), we replaced the hU6 RNA scaffold region of the previously described retroviral sgRNA vector MG-guide65 with an additional scaffold66 and the mouse U6 promoter. For the curated gene list containing 21 TF genes, a total of four gRNA sequences distributed on two individual constructs were designed for each gene. To construct the library, a customized double-strand DNA fragment pool containing 80 oligonucleotides targeting those 19 TF genes and four scramble gRNAs (each oligonucleotide contains two guides targeting the same gene) (Supplementary Table 5) was ordered from IDT. The dual-guide library was generated using an In-Fusion (Takara) reaction. In brief, the gRNA containing DNA fragment pool was combined in MG-guide vector linearized with BpiI (ThermoFisher). The construct was then transformed into Stellar competent cells (Takara) and amplified, and the resulting intermediate, individual, construct was assessed for quality using Sanger sequencing. Individual dual-gRNA vectors were then combined. For quality control, sgRNA skewing was measured using the MAGeCKFlute67 to monitor how closely sgRNAs are represented in a library. Retrovirus was generated by co-transfecting HEK293 cells with the dual-guide, direct-capture retroviral TF library and the packaging plasmid pCL-Eco. Cas9-expressing P14 CD8+ T cells were transduced with the viral supernatant to achieve a transduction efficiency of 20–30%. For in vivo experiments, 5 × 104 transduced P14 cells were transferred intravenously into Cas9-expressing, puromycin-resistant C57BL/6 recipient mice infected 1 day previously with either LCMV–Clone-13 or LCMV–Armstrong strain. A total of 25 LCMV–Clone-13-infected mice were used for five biological replicates and ten LCMV–Armstrong-infected mice were used for three biological replicates. Each biological replicate was labelled using hashtag antibodies (BioLegend, TotalSeq-C) to enable sample demultiplexing and statistical analysis. At least 18 days post-infection, donor-derived P14 CD8+ T cells were sorted and pooled for Perturb-seq analysis. Preliminary tests indicated that T cells expressing gRNA in vivo exhibit a greater tendency for gRNA silencing over extended periods compared with ex vivo cultured cells, despite initial successful KOs. To mitigate gRNA barcode silencing, we collected tissue between days 18 and 23. Sorted EGFP+ P14 CD8+ T cells were resuspended and diluted in 10% FBS RPMI at a concentration of 1 × 106 cells ml−1. Both the gene expression library and the CRISPR screening library were prepared using a Chromium Next GEM Single Cell 5′ kit with Feature Barcode technology for CRISPR Screening (10x Genomics). In brief, the single-cell suspensions were loaded onto the Chromium Controller according to their respective cell counts to generate 10,000 single-cell gel beads in emulsion per sample. Each sample was loaded into four separate channels. Chromium Next GEM Single Cell 5′ Kit v.2 (catalogue no. 1000263), Chromium 5′ Feature Barcode Kit (catalogue no. 1000451), Chromium Next GEM Chip K Single Cell Kit (catalogue no. 1000287), Dual Index Kit TT Set A (catalogue no. 1000215), Dual Index Kit TN Set A (catalogue no. 1000250) (10x Genomics) in total were used for each reaction. The resulting libraries were quantified and quality checked using TapeStation (Agilent). Samples were diluted and loaded onto a NovaSeq (Illumina) using a 100 cycle kit to obtain a minimum of 20,000 paired-end reads (26 × 91 bp) per cell for the gene expression library and 5,000 paired-end reads per cell for the CRISPR screening library, yielding an average of 42,639; 36,739 and 53,413 reads aligned from cells from in vivo LCMV–Clone-13, in vivo LCMV–Armstrong infection and in vitro donor respectively. Alignments and count aggregation of gene expression and sgRNA reads were completed using Cell Ranger (v.7.0.1). Gene expression and sgRNA reads were aligned using the Cell Ranger multi count command with default settings. The median average of four, two and 33 unique molecular identifiers (UMIs) were detected from cells from in vivo LCMV–Clone 13 and LCMV–Armstrong infection, and in vitro donor, respectively. Droplets with sgRNA UMI passing of default Cell Ranger CRISPR analysis Protospacer UMI threshold were used in further analysis. The filtered feature matrices were imported into Seurat68 (v.4.3.0) to create assays for a Seurat object containing both gene expression and CRISPR guide capture matrices. Cells detected with sgRNAs targeting two or more genes were then removed to avoid interference from multi-sgRNA-transduced cells. A total of 17,257 cells (Clone-13) and 15,211 cells (Armstrong) were passed through quality filtering and were used for downstream analysis. Count data were normalized by a global-scaling normalization method and linear transformed69. Cluster-specific genes were identified using the FindAllMarkers function of Seurat. We used Nebulosa70 to recover signals from sparse features in single-cell data and made gRNA density plots with scCustomize71 based on kernel density estimation. In each biological replicate (Clone-13, n = 5; Armstrong, n = 3), the percentage cluster distribution of cells with each TF gRNA vector was calculated. Among two gRNA vectors per target TF, the gRNA vector with higher TEXterm reduction was shown in Fig. 3d and used for Perturb-seq in LCMV–Armstrong infection (Supplementary Table 5). Two-way ANOVA with Fisher's LSD test was performed to determine statistical significance. Differentially expressed genes were identified using the MAST model72; the results were then used as inputs for GSEA to evaluate the effect on selected pathways. UMAP plots were generated by calculating UMAP embeddings using Seurat and then plotting them as scatter plots using ggplot2. Kernel density calculations for each gRNA were performed on UMAP embeddings using the MASS package using the kde2d function. The kernel density results were plotted as a raster layer with ggplot2 over the UMAP scatter plots. In brief, 5,000–50,000 viable cells were washed with cold PBS, collected by centrifugation, then lysed in resuspension buffer (RSB) (10 mM Tris-HCl, pH 7.4, 10 mM NaCl, 3 mM MgCl2) supplemented with 0.1% NP40, 0.1% Tween-20 and 0.01% digitonin. Samples were incubated on ice for 3 min, then washed out with 1 ml RSB containing 0.1% Tween-20. DNA was purified using a Qiagen MinElute PCR cleanup kit, then amplified by PCR using indexed oligos. The optimal number of amplification cycles for each sample was determined by quantitative PCR. Paired-end 42-bp or paired-end 75-bp reads were aligned to the Mus musculus mm10 genome using Burrow–Wheeler aligner74,75 with parameters ‘bwa mem -M -k 32'. ATAC-seq peaks were called using the MACS2 (ref. Differentially accessible regions were identified using DESeq2 (ref. Heatmap visualization of ATAC-seq data was performed using pheatmap. scRNA-seq data from GSE10898, GSE99254, GSE98638, GSE199565 and GSE181785 were filtered to keep cells with a low percentage of mitochondrial genes in the transcriptome (less than 5%) and between 200 and 3,000 unique genes to exclude poor quality reads and doublets. Cell cycle scores were regressed when scaling gene expression values and TCR genes were regressed during the clustering process, which was performed with the Louvain algorithm within Seurat and visualized with UMAP. To quantify the gene expression patterns, we used Seurat's module score feature to score each cluster based on its per cell expression of TFs. Count data were normalized and transformed by derivation of the residuals from a regularized negative binomial regression model for each gene (SCT normalization method in Seurat68, v.4.1.1), with 5,000 variable features retained for downstream dimensionality reduction techniques. Integration of data was performed on the patient level with Canonical Correlation Analysis as the dimension reduction technique81. PCA and UMAP dimension reduction were performed, with the first 50 principal components used in UMAP generation. Cells were clustered using the Louvain algorithm with multi-level refinement. The data was subset to CD8+ T cells, which were identified using the labels provided by Guo et al.65. Cell type labels were confirmed by (1) SingleR82 (v.1.8.1) annotation using the ImmGen83 database obtained through celda (v.1.10), (2) cluster marker identification and (3) cell type annotation with the ProjecTILs T cell atlas7 (v.2.2.1). After sub-setting to CD8+ T cells, cells were again normalized using SCT normalization, with 3,000 variable features retained for dimension reduction. PCA and UMAP dimensionality reduction were performed as above. Statistical tests for flow cytometry data were performed using Graphpad Prism v.10. P values were calculated using either two-tailed unpaired Student's t-tests, one-way ANOVA or two-way ANOVA as indicated in each figure. Linear regressions were performed using the ordinary least squares method in R (v.3.6.1). All data were presented as the mean ± s.e.m. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. ATAC-seq data from this paper will be deposited in the GEO database (GSE279498). Taiji v.2.0 output of this study (TF activity atlas, TF–TF interaction maps and TF activity on genome browser view) will be available at our CD8+ T cell TF atlas portal (https://wangweilab.shinyapps.io/Tcellstates/) and interactive interface for TF atlas exploration (https://huggingface.co/spaces/taijichat/chat). All other raw data are available from the corresponding author upon request. Source data are provided with this paper. All scripts and the Taiji v.2.0 package are available at GitHub (https://github.com/Wang-lab-UCSD/Taiji2). & Kaech, S. M. The architectural design of CD8+ T cell responses in acute and chronic infection: parallel structures with divergent fates. Kasmani, M. Y. et al. Clonal lineage tracing reveals mechanisms skewing CD8+ T cell fate decisions in chronic infection. Beltra, J.-C. et al. Developmental relationships of four exhausted CD8 T cell subsets reveals underlying transcriptional and epigenetic landscape control mechanisms. Pauken, K. E. et al. Epigenetic stability of exhausted T cells limits durability of reinvigoration by PD-1 blockade. The epigenetic landscape of T cell exhaustion. Wherry, E. J., Blattman, J. N., Murali-Krishna, K., van der Most, R. & Ahmed, R. Viral persistence alters CD8 T-cell immunodominance and tissue distribution and results in distinct stages of functional impairment. Wherry, E. J. et al. Molecular signature of CD8+ T cell exhaustion during chronic viral infection. Milner, J. J. et al. Runx3 programs CD8 T cell residency in non-lymphoid tissues and tumours. Ganesan, A.-P. et al. Tissue-resident memory features are linked to the magnitude of cytotoxic T cell responses in human lung cancer. Djenidi, F. et al. Tumor–infiltrating lymphocytes are tumor-specific tissue-resident memory T cells and a prognostic factor for survival in lung cancer patients. Corgnac, S., Boutet, M., Kfoury, M., Naltet, C. & Mami-Chouaib, F. The emerging role of CD8+ tissue resident memory T (TRM) cells in antitumor immunity: a unique functional contribution of the CD103 integrin. Komdeur, F. L. et al. CD103+ tumor-infiltrating lymphocytes are tumor-reactive intraepithelial CD8+ T cells associated with prognostic benefit and therapy response in cervical cancer. Kaech, S. M. & Cui, W. Transcriptional control of effector and memory CD8+ T cell differentiation. Defining CD8+ T cells that provide the proliferative burst after PD-1 therapy. Miller, B. C. et al. Subsets of exhausted CD8 T cells differentially mediate tumor control and respond to checkpoint blockade. Proliferating transitory T cells with an effector-like transcriptional signature emerge from PD-1 stem-like CD8 T cells during chronic infection. Liu, F. et al. CTLA-4 correlates with immune and clinical characteristics of glioma. Zhang, M. et al. Prognostic values of CD38+CD101+PD1+CD8+ T cells in pancreatic cancer. Shin, H. et al. A role for the transcriptional repressor Blimp-1 in CD8+ T cell exhaustion during chronic viral infection. Behr, F. M. et al. Blimp-1 rather than Hobit drives the formation of tissue-resident memory CD8 T cells in the lungs. Milner, J. J. et al. Heterogenous populations of tissue-resident CD8+ T cells are generated in response to infection and malignancy. The transcription factor Bhlhe40 programs mitochondrial regulation of resident CD8 T cell fitness and functionality. In vitro modeling of CD8 T cell exhaustion enables CRISPR screening to reveal a role for BHLHE40. Chen, J. et al. NR4A transcription factors limit CAR T cell function in solid tumours. The developmental pathway for CD103+CD8+ tissue-resident memory T cells of skin. & Wang, W. Taiji: system-level identification of key transcription factors reveals transcriptional waves in mouse embryonic development. & Wang, W. Taiji-reprogram: a framework to uncover cell-type specific regulators and predict cellular reprogramming cocktails. Liu, C. et al. Systems-level identification of key transcription factors in immune cell specification. Mackay, L. K. et al. Hobit and Blimp1 instruct a universal transcriptional program of tissue residency in lymphocytes. Milner, J. J. et al. Delineation of a molecularly distinct terminally differentiated memory CD8 T cell population. Guan, T. et al. ZEB1, ZEB2, and the miR-200 family form a counterregulatory network to regulate CD8 T cell fates. & Singh, N. Eomesodermin driven IL-10 production in effector CD8 T cells promotes a memory phenotype. Crowl, J. T. et al. Tissue-resident memory CD8 T cells possess unique transcriptional, epigenetic and functional adaptations to different tissue environments. Formation of IL-7Ralphahigh and IL-7Ralphalow CD8 T cells during infection is regulated by the opposing functions of GABPalpha and Gfi-1. Schmidt, R. et al. CRISPR activation and interference screens decode stimulation responses in primary human T cells. Blaeschke, F. et al. Modular pooled discovery of synthetic knockin sequences to program durable cell therapies. McCutcheon, S. R. et al. Transcriptional and epigenetic regulators of human CD8+ T cell function identified through orthogonal CRISPR screens. Seo, H. et al. BATF and IRF4 cooperate to counter exhaustion in tumor-infiltrating CAR T cells. Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. Riesenberg, B. P. et al. Stress-mediated attenuation of translation undermines T-cell activity in cancer. Daniel, B. et al. Divergent clonal differentiation trajectories of T cell exhaustion. Lin, Y. H. et al. Small intestine and colon tissue-resident memory CD8+ T cells exhibit molecular heterogeneity and differential dependence on Eomes. Long, Z. et al. Single-cell multiomics analysis reveals regulatory programs in clear cell renal cell carcinoma. Terekhanova, N. V. et al. Epigenetic regulation during cancer transitions across 11 tumour types. Integrated single-cell profiling dissects cell-state-specific enhancer landscapes of human tumor-infiltrating CD8 T cells. Yost, K. E. et al. Clonal replacement of tumor-specific T cells following PD-1 blockade. Ma, L. et al. Tumor cell biodiversity drives microenvironmental reprogramming in liver cancer. Young, M. D. et al. Single-cell transcriptomes from human kidneys reveal the cellular identity of renal tumors. Zheng, L. et al. Pan-cancer single-cell landscape of tumor-infiltrating T cells. Zhou, P. et al. Single-cell CRISPR screens in vivo map T cell fate regulomes in cancer. The CD8+ T cell tolerance checkpoint triggers a distinct differentiation state defined by protein translation defects. Chang, J. T. et al. Asymmetric proteasome segregation as a mechanism for unequal partitioning of the transcription factor T-bet during T lymphocyte division. Latent regulatory programs generate synthetic T cell states with enhanced therapeutic potential. Determination and inference of eukaryotic transcription factor sequence specificity. & Wasserman, L. The huge package for high-dimensional undirected graph estimation in R. J. Mach. From Louvain to Leiden: guaranteeing well-connected communities. Schönfeld, M. & Pfeffer, J. in Schlüsselwerke der Netzwerkforschung (eds Holzer, B. Author correction: distinct modes of mitochondrial metabolism uncouple T cell differentiation and function. Cong, L. et al. Multiplex genome engineering using CRISPR/Cas systems. Wang, B. et al. Integrative analysis of pooled CRISPR genetic screens using MAGeCKFlute. Stuart, T. et al. Comprehensive integration of single-cell data. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. & Powell, J. E. Nebulosa recovers single-cell gene expression signals by kernel density estimation. Finak, G. et al. MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. McDonald, B. et al. Canonical BAF complex activity shapes the enhancer landscape that licenses CD8 T cell effector and memory fates. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Zhang, Y. et al. Model-based analysis of ChIP–seq (MACS). Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Integrated analysis of multimodal single-cell data. Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Heng, T. S. P., Painter, M. W. & Immunological Genome Project Consortium. The Immunological Genome Project: networks of gene expression in immune cells. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Zhang, L. et al. Single-cell analyses inform mechanisms of myeloid-targeted therapies in colon cancer. Zheng, C. et al. Landscape of infiltrating T cells in liver cancer revealed by single-cell sequencing. We thank members of the Chung, Kaech and Wang laboratories for their advice and assistance with experiments. We appreciate assistance with animal experiments from the UNC Lineberger Preclinical Research Unit at the University of North Carolina at Chapel Hill, which is supported in part by an NCI Center Core Support Grant (CA16086) to the UNC Lineberger Comprehensive Cancer Center. These authors contributed equally: H. Kay Chung, Cong Liu NOMIS Center for Immunobiology and Microbial Pathogenesis, Salk Institute for Biological Studies, La Jolla, CA, USA H. Kay Chung, Eduardo Casillas, Ming Sun, Shixin Ma, Shirong Tan, Victoria Tripple, Bryan Mcdonald, Qiyuan Yang, Timothy Chen, Siva Karthik Varanasi, Michael LaPorte, Thomas H. Mann, Dan Chen, Filipe Hoffmann & Susan M. Kaech Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA H. Kay Chung, Anamika Battu, Brandon M. Pratt, Fucong Xie, Brian P. Riesenberg, Ming Sun, Elisa Landoni, Yanpei Li, Qidang Ye, Daniel Joo, Jarred Green, Zaid Syed, Nolan J. Brown, Matthew Smith, Z. Audrey Wang, Jennifer Modliszewski, Yusha Liu, Ukrae H. Cho, Gianpietro Dotti, Barbara Savoldo, Jessica E. Thaxton & J. Justin Milner Department of Cell Biology and Physiology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA H. Kay Chung, Ukrae H. Cho & Jessica E. Thaxton Bioinformatics and Systems Biology Program, University of California San Diego, La Jolla, CA, USA Department of Pharmacology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Molecular and Cell Biology Laboratory, Salk Institute for Biological Studies, La Jolla, CA, USA Brent Chick, Josephine Ho & Diana C. Hargreaves Peixiang He, Longwei Liu & Yingxiao Wang Razavi Newman Integrative Genomics and Bioinformatics Core, Salk Institute for Biological Studies, La Jolla, CA, USA Halıcıoğlu Data Science Institute, University of California San Diego, La Jolla, CA, USA Zhen Wang, Jieyuan Liu & Zhiting Hu Department of Electrical and Computer Engineering, Texas A&M University, College Station, TX, USA Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Gianpietro Dotti, Barbara Savoldo & J. Justin Milner Department of Cellular and Molecular Medicine, University of California San Diego, La Jolla, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar analysed the sequencing and bioinformatics data. contributed to TaijiChat and portal generation. provided resources and training for ATAC-seq. provided scientific input and acquired funding. Correspondence to H. Kay Chung, Susan M. Kaech or Wei Wang. G.D. serves on the SAB of NanoCell, Estella, Arovella and Outpace Bio. G.D. is a cofounder of Persistence Bio. hold patents in the field of CAR-engineered cells. The other authors declare no conflicts of interest. Nature thanks Tuoqi Wu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Unbiased clustering identified multiple T cell states consistent with those observed murine LCMV infection and tumors. b, TRM marker genes show higher expression in TEXterm cluster from Pan-cancer scRNA-seq in a. c, Both TRM and TEXterm clusters upregulate exhaustion-8 and TRM31-associated gene signatures. d, Pearson correlation matrix of batch-corrected ATAC-seq datasets3,9,33,34. Color and size are both proportional to correlation strength. e, A total of 121 experiments across multiple data sets3,9,17,22,31,32,33,34,35 were utilized to generate an epigenetic and transcriptional atlas of CD8+ T cells under chronic and acute antigen exposure. a, Logic flow of the unbiased PageRank comparison to classify single-state/multi-state TFs. b, Number of TFs catalogued in each cell state. c, Venn Diagrams showing overlap of TFs with the TRM cell state. d, TF activity score (normalized PageRank) of previously reported TEXterm-preventing TFs (TFAP4, JUN, BATF3, and BATF), newly identified TEXterm single-state TFs (JDP2, ZFP324, ZBTB49, ZFP143, ZSCAN20), and NFATC1, a known TEXterm-associated TF. b, Selection of algorithms and parameters for TF wave analysis. Red color indicates normalized PageRank scores. d, List of TF members in each wave. e, Heatmap of biological pathways enriched in each TF wave. Red-blue color scale indicates the p-value. a, Pseudotime analysis of CD8+ TIL scRNA-seq data from NSCLC patients (n = 14), showing a positive correlation between proteasome gene scores (KEGG: M10680) and T cell exhaustion. b, Gene set enrichment analysis (GO:0043161, Proteasome-mediated ubiquitin-dependent protein catabolic process) of RNA-seq from CD8+ splenocytes (black) and TILs (purple), n = 3 each. c, Tumor growth of Rag2−/− mice bearing MCA-205 sarcomas infused with proteasomehigh or proteasomelow CD8+ TILs isolated from C57BL/6 mice bearing MCA-205 tumors. Two-way ANOVA Tukey's multiple comparison test were performed. a, Retroviral vector design for dual-gRNA delivery. b, Feature plot of differentiation markers (Pan-exhaustion: red, TEXprog: pink, TEXeff: blue, Cell cycle: purple, TEXterm: brown). c, Heatmap of marker gene expression across cell state clusters identified by Seurat's Find Markers() function. Statistical analysis: Ordinary one-way ANOVA with Dunnett's multiple comparisons test versus gScramble (n ≥ 5 from ≥2 biological replicates). a, Feature plots of differentiation marker genes (TCM: yellow, TEM: blue, TRM-Itgaelow: dark blue, TRM: green). b, Heatmap of differentially expressed genes between TRM-Itgaelow and TRM clusters. c, Volcano plots of differentially expressed genes in Cas9+ P14 CD8+ T cells expressing gEtv5, gArid3a, gHic1, or gGfi1. d, TRM single-state TF, Klf6 overexpression does not accelerate T cell exhaustion. Experimental setup: Klf6-RV or control-RV transduced P14 CD8+ T cells co-transferred into mice infected with chronic LCMV-Clone-13, Quantification of the frequency of PD1+, TEXprog (PD1+SLAMF6+CX3CR1−), TEXeff (PD1+SLAMF6−CX3CR1+) and TEXterm (PD1+SLAMF6−CX3CR1−) populations. Paired t-tests (n ≥ 6 from ≥2 biological replicates). b, Representative flow plots of PD1+ P14 cells stained for SLAMF6 and TIM3. c, quantification of TEXprog (PD1+SLAMF6+TIM3−) and exhausted (PD1+SLAMF6−TIM3+) subsets. e, Co-transfer of Cas9+ P14 CD8+ T cells transduced with RV-gZscan20 or RV-gHic1, mixed with gRNA control RV transduced cells into B16-GP33 tumor-bearing mice. f, Representative flow plots of PD1+ P14 CD8+ T cells stained for SLAMF6 and CX3CR1. g-i, Quantification of (g) TEXprog (PD1+SLAMF6+TIM3−), (h) TEXterm (PD1+SLAMF6−TIM3+), and (i) IFNγ+TNF+ population. Paired t-tests (n ≥ 6 from ≥2 biological replicates). a, Human pan-cancer datasets utilized in this study48,49,50,51,52,53,54,55. b, Cluster-specific marker mRNA expression across integrated pan-cancer single-cell multi-omics datasets. c, TF activity analysis using Taiji on matched matched scRNA-seq and scATAC-seq datasets from various human cancers, including ccRCC, GBM, BCC, HNSCC, HCC, and RCC. CD8+ T cell states were annotated using canonical marker gene expression. PageRank scores derived from Taiji were log-transformed, averaged per state, and z-score normalized. Results were visualized with a focus on TF activity TRM and TEXterm cell states. d, mRNA expression of TFs were compared across human CD8+ T cell states. The dataset encompassed T cells from 15 tumor types. Cell type annotations provided by the original study were retained and mapped to the following broad categories of clusters: TRM, TEXterm, TEM, Memory, Naive, and cell cycle. Seurat's AverageExpression function was used, followed by z-score normalization. Data visualization emphasized comparisons between TRM and TEXterm cell states. c, Survival of B16-GP33 bearing mice receiving Zscan20 KO or control P14 CD8+ T cells, followed by anti-PD1 or isotype IgG2a treatment. d, Survival of tumor-bearing mice with Jdp2 KO P14 CD8+ T cell transfer. The method of developing TaijiChat: an integrated conversational interface for multi-omics data exploration and a user guide. Summary of all RNA-seq and ATAC-seq datasets (n = 121) integrated into the Taiji analysis pipeline, including dataset sources, accession numbers, infection models (acute LCMV–Armstrong or chronic LCMV Clone-13), tissue origins and defined CD8+ T cell states (naive, MP, TE, TRM, TEM, TCM, TEXprog, TEXeff, TEXterm). TF PageRank scores of multi-state and single-state TFs. PageRank-based TF activity scores calculated from Taiji-derived regulatory networks for each of the nine CD8+ T cell states. The table lists 255 state-enriched TFs, categorized as single-state or multi-state according to statistical filtering (P < 0.05; log2FC > 0.5). Refined statistical comparison of TRM and TEXterm TF activity, identifying 20 TRM single-state TFs, 34 TEXterm single-state TFs and 30 multi-state TFs active in both states. Scores and classification criteria correspond to Extended Data Fig. Comprehensive list of universal and state-important TFs (top 10% PageRank, coefficient of variation < 0.5) across nine CD8+ T cell states, including TF names and activity ranks derived from Taiji network analysis. Catalogue of TF communities in TRM and TEXterm states. Each community includes constituent TFs (related to Fig. Sequences for gRNAs expressed under dual-gRNA retroviral vector targeting 19 TFs and control gRNAs used for in vivo Perturb-seq, gRNA sequences used in RNP-based KO experiments and primers used to validate KO with sequencing. Summary statistics for cluster distributions of gRNA+ cells across TEXprog, TEXeff, TEXterm, TRM, TEM and TCM clusters from in vivo Perturb-seq datasets (Figs. Curated lists of gene signatures used for TEXterm, TRM, Teff and proteasome pathway analyses, derived from published datasets and this study's Taiji-based enrichment (referenced in Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Chung, H.K., Liu, C., Battu, A. et al. Atlas-guided discovery of transcription factors for T cell programming. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/extraordinary-brain-network-discovery-changes-our-understanding-of/'>‘Extraordinary' brain network discovery changes our understanding of Parkinson's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 16:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists discover brain network that may cause Parkinson's disease An “extraordinary” brain network discovery shows that Parkinson's disease may not be a movement disorder after all Its hallmark symptoms include involuntary muscle contractions, tremor and difficulty walking. But the disease can also disrupt sleep, blood pressure regulation, digestion and cognitive function. The movement-related symptoms can worsen when someone with the disease is under stress, for example, but improve while they are listening to music. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “Parkinson's is not just a movement problem involving one body part. This study shows it is a whole-body brain network disorder that links movement, thinking, arousal and internal body control,” says Michael Okun, a neurologist at the University of Florida and medical director of the Parkinson's Foundation, who was not involved in the study. It's an “extraordinary” set of findings, says Todd Herrington, a neurologist at Massachusetts General Hospital who treats and studies Parkinson's. This headband-shaped brain strip extends from ear to ear and contains a “map” of the entire body—often visualized as a distorted humanoid figure called the homunculus. But neurologist Nico Dosenbach of Washington University in St. Louis had observed something strange. These extra spots of activation “just didn't make sense, if all the things I thought I knew were true,” he says. It turns out that neuroscientists had been underestimating M1 for nearly a century. Interspersed between body-part-specific areas are nodes of a network that coordinates higher-level planning for movement. Instead of being a mere foot soldier following orders from more frontal brain regions, M1 helps plan, guide and coordinate action. These findings caught the eye of Hesheng Liu, a neuroscientist at Changping Laboratory in Beijing. For a decade, he'd been studying Parkinson's disease, trying to figure out how a treatment called deep-brain stimulation (DBS) works to alleviate symptoms. His team had noticed the strange patterns in M1, too. “We had no idea what they are,” Liu says. When he saw Dosenbach's paper on SCAN, everything started to make sense. “Probably, that region is behind Parkinson's disease,” he thought. Doctors don't know what sets off the chain of events that cause Parkinson's disease. Stimulating other regions connected to the substantia nigra can alleviate Parkinson's symptoms, suggesting an entire circuit is involved. Using multiple brain-imaging datasets from 863 real people with Parkinson's and healthy individuals, Liu's team found that SCAN was overlyconnected to deep-brain regions in those with Parkinson's but not in healthy people or those with other movement disorders. Individuals with Parkinson's who had higher connectivity in this circuit experienced worse symptoms. Doctors don't yet know if dying neurons in the substantia nigra cause these SCAN disruptions, or vice versa, says Michael D. Fox, a neurologist at Brigham and Women's Hospital in Boston, who was not involved in the study. But it's “not impossible” that the SCAN dysfunction could start early, too, and cause more neurons to die, he says. Brain stimulation treatments for Parkinson's were more effective when doctors specifically targeted SCAN regions, Liu's team also found. In part because of that limitation, Fox says, TMS isn't offered clinically to people with Parkinson's. But focusing TMS on SCAN regions specifically can improve results, Liu's team showed. TMS may be more appealing and accessible to patients than deep-brain stimulation, which requires surgery. “This, in my mind, elevates the potential of noninvasive brain stimulation for helping patients with Parkinson's in a way that wasn't there before,” Fox says. As a multimedia journalist, she contributes to Scientific American's podcast Science Quickly. Parshall's work has also appeared in Quanta Magazine and Inverse. She graduated from New York University's Arthur L. Carter Journalism Institute with a master's degree in science, health and environmental reporting. She has a bachelor's degree in psychology from Georgetown University. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00337-x'>NIH rolls back red tape on some experiments — spurring excitement and concern</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 12:40:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Researchers conducting some classes of research with human participants will have less paperwork under a new federal guideline.Credit: Getty The decision, which was announced on 29 January and is part of a broader NIH effort to reduce administrative burden, should free such research from the heavy bureaucratic requirements that are designed for clinical trials but are sometimes ill-suited to other fields, such as basic psychology and behavioural studies. The clinical-trial designation usually comes with an obligation to preregister experiments and publish results on ClinicalTrials.gov. Reporting results and ensuring they are analyzed properly are particularly important when it comes to research involving human participants who often offer their time and, in some cases, take on personal risk because they are told that their efforts will advance science, says Zarin. A father's fight to help his sons — and fix clinical trials How Facebook, Twitter and other data troves are revolutionizing social science NIH ends support for most human fetal-tissue research – dismaying some scientists Guinea-Bissau suspends US-funded vaccine trial as African scientists question its motives NIH ends support for most human fetal-tissue research – dismaying some scientists APPLICATION CLOSING DATE: 22.02.2026 Human Technopole (HT) is an interdisciplinary life science research institute, created and supported by the It... Drive excellence in Spatial Computing (AR/VR/XR), building XR platforms, neural rendering, & digital twins for industry, healthcare, & education. A father's fight to help his sons — and fix clinical trials How Facebook, Twitter and other data troves are revolutionizing social science An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00309-1'>Biodiversity conservation has an evidence problem — it's time to fix it</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 12:33:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You have full access to this article via your institution. Although there are hundreds of local, regional and international initiatives to conserve and sustainably use species and ecosystems, many conservation scientists worry that measures such as interventions to conserve individual species or incentives to create protected areas are not supported by strong evidence from research1. Scientists are building giant ‘evidence banks' to create policies that actually work Scientists are building giant ‘evidence banks' to create policies that actually work “It always astonishes me how, while drowning in an ocean of information, we still don't have the scientifically based answers to very simple questions,” says Sandra Díaz, an ecologist at the National University of Córdoba in Argentina. How to improve both the availability and use of good evidence in policymaking does not necessarily make headlines, but it is something that needs to be high on the agenda for IPBES, as it is for the IPCC. Last month, conservation scientists and practitioners met in Cambridge, UK, to discuss what one of Europe's largest conservation groups, the UK-based Wildlife Trusts, is calling an “evidence emergency”. First, the quality of evidence used in drawing up conservation policies needs to improve. In northern India, for example, decades of tree-planting schemes have not increased forest canopy cover because of a failure to account for the reasons why cover was being lost to begin with2. “There's a tremendous amount of information in the scientific literature, but it's largely inaccessible,” says Shahid Naeem, an ecologist at Columbia University in New York City, who was not at the meeting. This is changing, thanks to a solution inspired by the synthesis of literature reviews commonly seen in medicine. It is an enormous undertaking involving hundreds of researchers, who have spent two decades working their way through more than 1.2 million research papers in 18 languages to identify studies that test the impact of conservation interventions. Will AI speed up literature reviews or derail them entirely? Will AI speed up literature reviews or derail them entirely? Another initiative, the Collaboration for Environmental Evidence, involves a network of institutions, in countries including Canada, Chile, France, South Africa, Sweden and the United Kingdom. Its members publish evidence reviews and support others who wish to produce their own. The concept is described in a preprint that was published last year3. For all their undoubted benefits, such projects do have some limitations. Foremost is that they are restricted to what is in the existing literature, which tends to be dominated by studies by researchers in high-income countries. The team members say that they are working on how to incorporate sources of Indigenous knowledge, which offers one way to start bridging this gap. Can AI review the scientific literature — and figure out what it all means? Can AI review the scientific literature — and figure out what it all means? Díaz points out that the desire for improved evidence shouldn't delay action in cases for which the appropriate biodiversity interventions are already known. At the same time, governments are scrambling to increase economic growth, and, in so doing, often disregarding the role nature has in sustaining this growth. Its sponsoring governments should ask it to tackle these questions without delay. Will AI speed up literature reviews or derail them entirely? Scientists are building giant ‘evidence banks' to create policies that actually work Can AI review the scientific literature — and figure out what it all means? Machine learning slashes the testing needed to work out battery lifetimes Open-source AI tool beats giant LLMs in literature reviews — and gets citations right ‘It means I can sleep at night': how sensors are helping to solve scientists' problems AI chatbots are infiltrating social-science surveys — and getting better at avoiding detection APPLICATION CLOSING DATE: 22.02.2026 Human Technopole (HT) is an interdisciplinary life science research institute, created and supported by the It... Drive excellence in Spatial Computing (AR/VR/XR), building XR platforms, neural rendering, & digital twins for industry, healthcare, & education. You have full access to this article via your institution. Will AI speed up literature reviews or derail them entirely? Scientists are building giant ‘evidence banks' to create policies that actually work Can AI review the scientific literature — and figure out what it all means? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00361-x'>Daily briefing: What people with no ‘mind's eye' can tell us about consciousness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 11:51:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You have full access to this article via your institution. Hello Nature readers, would you like to get this Briefing in your inbox free every day? Writing by hand activates parts of the brain associated with learning that typing words does not.Credit: Three Lions/Getty Some schools that dropped the requirement to teach cursive — handwriting characterized by flowing, connected letters — to embrace digital learning are re-introducing penmanship into the classroom. But there are also other, cultural reasons for keeping handwriting alive. “I feel that the next generation should be able to write a love letter or a poem by hand, or at least the grocery list, because it's part of being human, really,” says neuroscientist Audrey van der Meer. Biotech company Merge Labs plans to make brain–computer interface (BCI) devices that use ultrasound to read people's minds and treat mental conditions without implanting electrodes deep into the brain. The firm has emerged as a rival to billionaire Elon Musk's Neuralink and is backed by US$252 million in investment from funders that include artificial-intelligence firm OpenAI. The approach is less invasive than Neuralink-style devices and could offer an alternative to deep-brain stimulation therapies for conditions such as epilepsy. When asked to picture something in their minds, around 4% of people can only conjure a faint image, or might see nothing at all. Take Nature's quiz to assess how vividly you see mental imagery. The robot's flexible fingers also enable it to juggle multiple objects at the same time and, if needed, it can simply leave its arm behind — perfect for dangerous or hard-to-reach places. Environmental engineer Nima Shokri argues that international coverage of the civil unrest in Iran focuses on political and economic crises and ignores “a more destabilizing force”: environmental breakdown. Today I'm preparing for six more weeks of winter after Punxsutawney Phil — Pennsylvania's famed groundhog (Marmota monax) — saw his own shadow yesterday morning. I might still put a couple of my jumpers away, however, as Phil doesn't have an outstanding track record at just 35% accuracy, according to data from the US National Oceanic and Atmospheric Administration. Help me prepare upcoming editions of this newsletter by sending your feedback to at briefing@nature.com. • Nature Briefing: Careers — insights, advice and award-winning journalism to help you optimize your working life • Nature Briefing: Translational Research — covers biotechnology, drug discovery and pharma Daily briefing: Brain-immune crosstalk worsens the damage of heart attacks APPLICATION CLOSING DATE: 22.02.2026 Human Technopole (HT) is an interdisciplinary life science research institute, created and supported by the It... Drive excellence in Spatial Computing (AR/VR/XR), building XR platforms, neural rendering, & digital twins for industry, healthcare, & education. You have full access to this article via your institution. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00312-6'>Quantum computers will finally be useful: what's behind the revolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 11:17:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). A quantum computer chip from Oxford Ionics, one of several companies that have made significant advances in this technology. Just a few years ago, many researchers in quantum computing thought it would take several decades to develop machines that could solve complex tasks, such as predicting how chemicals react or cracking encrypted text. But now, there is growing hope that such machines could arrive in the next ten years. A ‘vibe shift' is how Nathalie de Leon, an experimental quantum physicist at Princeton University in New Jersey, describes the change. Teams in academic laboratories, as well as companies ranging from small start-ups to large technology corporations, have drastically reduced the size of errors that notoriously fickle quantum devices tend to produce, by improving both the manufacturing of quantum devices and the techniques used to control them. Meanwhile, theorists better understand how to use quantum devices more efficiently. “At this point, I am much more certain that quantum computation will be realized, and that the timeline is much shorter than people thought,” says Dorit Aharonov, a computer scientist at Hebrew University in Jerusalem. The latest developments are exciting to physicists because they address some of the main bottlenecks preventing development of viable quantum computers. These devices work by encoding information in qubits, which are units of information that can take on not just the values 0 or 1, like the bits in a classical computer, but also a continuum of possibilities in between. Chao-Yang Lu is among those who expect a fault-tolerant quantum computer by 2035.Credit: Dave Tacon for Nature Crucially, a gate can put multiple qubits in collective entangled, or strongly correlated, states — exponentially boosting the amount of information that they can handle. For decades, researchers questioned the viability of this computational paradigm owing to two main reasons. One is that, in practice, quantum states tend to naturally and randomly drift, and after a certain amount of time, the information they store is inevitably lost. The other is that gates and measurements can themselves introduce errors. But over the past year or so, four teams have shown that these problems are ultimately solvable, Aharonov and others say. Just last December, a fourth team, at the University of Science and Technology of China (USTC) in Hefei, also joined this exclusive club4. The four groups implemented — and improved — a technique called quantum error correction, in which a single unit of quantum information, or ‘logical' qubit, is spread across several ‘physical' qubits. This billion-dollar firm plans to build giant quantum computers from light. This billion-dollar firm plans to build giant quantum computers from light. And QuEra's qubits are represented by the alignment of individual neutral atoms confined by beams of light that act as ‘optical tweezers'. Like any operation on qubits, the correction itself introduces errors. In the 1990s, Aharonov and others proved mathematically that, if applied repeatedly, the process can reduce errors by as much as is desired. To many physicists, this watershed moment demonstrated that large-scale, ‘fault tolerant' quantum computing could be viable. Even when it works, quantum error correction is not a panacea. For a long time, scientists estimated that using it to run a fully fault-tolerant quantum algorithm would require an overhead of 1,000:1, or at least 1,000 physical qubits for each logical qubit. The largest quantum computers built so far have just a few thousand qubits — but early estimates suggested that billions might be needed to do things such as factoring into prime numbers. Andrew Houck (pictured), Nathalie de Leon and their colleagues at Princeton University developed a technology that could make quantum computing more accurate.Credit: Matt Raspanti/Princeton University This task has long been a benchmark because quantum computers that can factor large numbers into primes would be powerful enough to solve previously intractable problems, such as predicting the properties of new ‘wonder materials' or making stock trading super efficient. This has reduced estimates of the number of physical qubits it would take to factor large numbers — which would break a common Internet encryption system — by roughly an order of magnitude every five years. Last year, Google researcher Craig Gidney showed that he could cut the number of qubits down from 20 million to one million5, in part by arranging abstract gate diagrams into complex 3D patterns. (“I do use a lot of geometric intuition,” he says.) Gidney says his implementation is probably close to the best possible performance of a standard quantum-error-correction technique. Better ones, however, could bring the overhead down further, he adds. Quantum computing ‘KPIs' could distinguish true breakthroughs from spurious claims Quantum computing ‘KPIs' could distinguish true breakthroughs from spurious claims “The whole name of the game right now is how you can make error correction more efficient,” says de Leon — and there are several possible approaches. Theoreticians can help by developing error-correction techniques that encode the information of a logical qubit more efficiently, thereby requiring fewer physical qubits. Jens Eisert, a physicist at the Free University of Berlin, says he “would be surprised” if physical-qubit overheads did not come down further in the next few years. “I think, mathematically, the theory of quantum error correction is getting richer and more interesting. There's been a huge explosion of papers,” says Barbara Terhal, a theoretical physicist at QuTech, a quantum-technology research institute, supported by the Dutch government, at the Delft University of Technology in the Netherlands. She warns, however, that complex error-correcting codes could have drawbacks because they make it more complicated to perform gates. One such technique, perfected by IBM, promises to encode logical qubits using one-tenth the number of physical qubits as industry-standard approaches, or an overhead of roughly 100:1. QuEra is experimenting with methods that lean on a major strength of its ‘neutral atom' qubits: the flexibility to be moved around to be entangled with one another at will. Their error-correction approach, too, could in principle lower the overhead to 100:1, says Quera founder Mikhail Lukin, a Harvard physicist. De Leon, meanwhile, has focused on studying the weaknesses of qubits using advanced techniques in metrology, the science of precise measurements. Historically, a major drawback of superconducting qubits has been their short lifetimes, which cause stored information to degrade even as the algorithm manipulates physically distant qubits on the same chip. They then tried switching from superconducting loops made of aluminium to ones made of tantalum, and the supporting material from sapphire to insulating silicon. “There are obvious things to try, where I believe we can get to 10 or 15 milliseconds,” de Leon says, although she also warns that often, after removing one source of noise, another unexpected one creeps in. ‘A truly remarkable breakthrough': Google's new quantum chip achieves accuracy milestone This billion-dollar firm plans to build giant quantum computers from light. Quantum computing ‘KPIs' could distinguish true breakthroughs from spurious claims Google claims ‘quantum advantage' again — but researchers are sceptical ‘It means I can sleep at night': how sensors are helping to solve scientists' problems Open-source AI tool beats giant LLMs in literature reviews — and gets citations right APPLICATION CLOSING DATE: 22.02.2026 Human Technopole (HT) is an interdisciplinary life science research institute, created and supported by the It... Drive excellence in Spatial Computing (AR/VR/XR), building XR platforms, neural rendering, & digital twins for industry, healthcare, & education. ‘A truly remarkable breakthrough': Google's new quantum chip achieves accuracy milestone This billion-dollar firm plans to build giant quantum computers from light. Quantum computing ‘KPIs' could distinguish true breakthroughs from spurious claims Google claims ‘quantum advantage' again — but researchers are sceptical An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/daily-misery-why-some-people-cant-burp-and-how-botox-comes-to-the-rescue/'>‘Daily misery'—why some people can't burp, and how Botox comes to the rescue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Some people experience an inability to burp. An expert who treats this little-known disorder explains why For those with retrograde cricopharyngeus dysfunction, daily life can be miserable, with symptoms such as bloating and chest pain. But a simple Botox injection can help By Kendra Pierre-Louis, Sushmita Pathak & Alex Sugiura You've probably been in this situation: you just had a big lunch or a tall carbonated drink, and out of nowhere a burp rises in your throat unbidden. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. If you are someone's annoying brother, you've probably summoned a burp and unleashed it on your sibling's face at least once. But Paras Dhama can't relate to any of that. Paras Dhama: Because [Laughs] I can't burp. Dhama: So my whole chest and stomach, it becomes heavy—it feels like some air is stuck inside. And it becomes so uncomfortable that one time I was driving on a highway, I had to stop my car, get outside, try to vomit on the side of the road in order to get that air out. Pierre-Louis: This inability to burp—he thought it was a personal failing, like how some people can't whistle. But what he's experiencing is actually due to a medical condition that doctors have only recently begun to understand. Thanks for taking the time to chat with us today. Pierre-Louis: So we're here to talk to you—I think in your paper you describe it as retrograde cricopharyngeus dysfunction. Did I say that anywhere near close to accuracy? Pierre-Louis: Can you describe what that is in plain English? But it also must relax in the retrograde direction to let us burp or vomit. And so people with retrograde cricopharyngeus dysfunction have normal antegrade function—they swallow; none of them really have any trouble swallowing—but they can't burp, and some of them can't vomit. Pierre-Louis: Yeah, I think kind of intuitively people understand how it might be a problem that you can't vomit. Many of us have had stomach bugs, and, like, you know, it's better out than in. But the burping side, I think, was really not intuitive to many of us because we sort of do that unconsciously, I think, and so why is it useful to burp? Bastian: Well, every human swallow includes a little component of air because the saliva bubbles. Well, each of those little, tiny bubbles, when it gets to the stomach they begin to coalesce, so they don't stay as foam. They kind of—over time that builds up what's called a gastric air bubble; it's normal in everyone to have that little air bubble. But if that bubble gets to be a certain size, then it needs to be vented as burping. So if you can't do that and now you feel uncomfortable because you've sensed the need to burp, but you can't—typically, I think what happens is people begin to swallow extra, and they just build up more and more air, to a degree, in some, that is ridiculous; I mean, they just blow themselves up with air. And all of the downstream air that hasn't been released upwards, it has nowhere to go except, eventually, as flatulence. Pierre-Louis: It's, I think, so fascinating that some people can't do this because, like, we expect babies to burp, you know, and we know that if babies can't burp, they get a little bit irritated, and, you know, you have to work [Laughs] to get that gas out. How does someone know that they can't burp versus maybe they're more of a, like, a tiny burper, I guess [Laughs] is what I would call it? Bastian: Well, the majority of human burps are silent. Probably in the last week you were around 20 or 30 or 40 burps within five feet, and you knew nothing about it. So people who can't burp, how do they know? But then there are other people who absolutely never burp—they cannot remember a single time in their entire lives. Pierre-Louis: Is there, like, a test for burping? Bastian: Well, yes, the single test that would prove that a person can't burp is called manometry. But if you do standard manometry, as it's done routinely thousands of times around the world, it will not make the diagnosis. What you need is to establish what we call the syndrome 'cause there's a constellation of symptoms that make the diagnosis in a very firm way. And so basically you ask the patient, and the symptoms that they give you are highly, highly diagnostic: “I can't burp.” Probably 90 percent say, “I make gurgling noises.” They can be quiet and internal ... Bastian: But more often they're heard somewhere between a couple of feet away and across a large room. So can't burp, gurgling, bloating, flatulence—world-class, gold-medal, unbelievable kind of flatulence. Now, those are the big four, but there are quite a few less universal but still very common: painful hiccups. There is nausea after eating; that's a common one. So when you talk to patients, and you get enough of those symptoms together, and you combine it with the primary issue—“I can't burp”—and your diagnostic accuracy is practically 100 percent. Pierre-Louis: And this is a relatively new diagnosis, right? Bastian: Yes, well, the index patient for me was in 2015, and the caseload trickled in at first. And so I thought to myself, “For goodness' sakes, this—somebody knows about this. For example, one was: combination of can't burp and chest pain. But it didn't describe the whole syndrome. Nobody had put it together, and there was no treatment. And so I was privileged to be the one to codify—by codify, I mean drawing it all together and creating the complete picture, rather than stabbing at “can't burp” and adding one additional symptom. The full-orbed description of RCPD came from me and was first published in 2019, so really, RCPD became known to the medical community. Pierre-Louis: And do we know if people are born this way, or does this condition, like, develop over time? Bastian: When you talk to people with RCPD there are many who don't have information about their infancy. But of the group that knows, that is able to get any information, we've learned that approximately one out of three had notable inability to burp as infants; two outta three did burp. Out of the two outta three who did burp there were some who had colic and were gassy and things like that, so there may have been some insufficient burpers. And in that one third there's a subset where it was an ordeal for the parents, where they said, “We were taking them to the doctor. He was crying.” One parent said, “Yes, we would measure what we fed her, and we'd cover ourselves with a towel and stand by with a bowl 'cause it, like clockwork, about 20 minutes later she would throw up a huge amount, and we needed to know, ‘Was she keeping anything?' So we measured what went in, and we measured what came out into the bowl.” And we've had a number say, “If this child had been our first child, she would've been our only child. We would've said, ‘Absolutely cannot do that again. How does someone get treated for this? Bastian: Well, the way we do it here is very simple: we meet the patient and validate the diagnosis, and then we go straight to Botox. We go over to a nearby day surgery center. The patient spends about two and a half hours there. And you can go down into the upper esophagus. You can find the ridge; it's like a band. And then you use a tiny needle, and you inject that muscle in two or three places. They don't need pain medicine, except rarely. It's just a little scratchy sore throat. And then within a few days they begin to burp. So in that method we do a little bit of numbing. Then we attach some little electrodes, like EKG pads, and then we use a hollow—a Teflon-coated needle and come in from the side and/or from the front; there are two basic approaches to the muscle. Pierre-Louis: And what does the Botox do, exactly? Bastian: Botox, it causes a chemical paralysis of muscle. So it's a chemical denervation that is temporary, lasting three to five months. So now this sphincter muscle, which has refused to relax in the retrograde direction—it clamps; it won't let go—now it's become limp, and so the burp can get out and vomiting can happen or whatever. They get very mindful about the series of sensations. But the common one, the one that I'm looking for, is—it's a lowering the larynx, so the burp is arriving, they feel it arriving, and they kind of [Lowers voice], you know, like, when you talk like Yogi Bear ... And then the idea is we have pure Botox burps, and then Botox is fading, so it's a training wheels kind of idea. So in theory they don't need to continue doing it. It's sort of training the muscle on how to burp. Pierre-Louis: And I guess the last question that I have is, like, patients who have—who, like, undergo the Botox or gain the ability to burp, how do they, like, react? Bastian: I was fascinated by the number of people who came up with the word “life-changing.” People say things like, “I simply can't believe that this is what other people are like.” Or they'll say, “I knew this was bad, but I didn't realize how bad it was until I got rid of it, and it's, like, unbelievable.” They're very ecstatic about the improvement in their quality of life. RCPD untreated, the way I describe it is severe daily misery. I have patients—I had one who said to me, “Doctor, if I eat lunch—it's summer or winter—I have to go out to my car, put the seat back because I can't tolerate the discomfort, the pain in my stomach. I can't tolerate it,” stuff like that. Pierre-Louis: And such a pretty straightforward treatment. Tune in on Friday, when we'll dive into potential changes in how we define and diagnose mental health conditions. Tell us about your most memorable kiss. Record a voice memo on your phone or computer, and send it over to ScienceQuickly@sciam.com. Be sure to include your name and where you're from. Science Quickly is produced by me, Kendra Pierre-Louis, along with Fonda Mwangi, Sushmita Pathak and Jeff DelViscio. This episode was edited by Alex Sugiura. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Kendra Pierre-Louis. Have a great rest of your week! She has worked for Gimlet, Bloomberg News and Popular Science. Pierre-Louis is based in New York City. She previously worked at NPR and was a regular contributor to The World from PRX and The Christian Science Monitor. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. SciAm always educates and delights me, and inspires a sense of awe for our vast, beautiful universe. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42003-026-09597-9'>Uniparental analysis of Deep Maniot Greeks reveals genetic continuity from the pre-Medieval era</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 10:11:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The Deep Maniots, an isolated population at the southernmost tip of mainland Greece, have drawn scholarly interest for their unique dialect, culture, and patrilineal clan structure. Geographically shielded by the Mani Peninsula, they are thought to have been minimally affected by 6th-century CE migrations that transformed Balkan demography. To investigate their genetic origins, we analysed Y-DNA and mtDNA from 102 Deep Maniots using next-generation sequencing. Paternally, Deep Maniots exhibit an exceptional prevalence (~80%) of West Asian haplogroup J-M172 (J2a), with subclade J-L930 accounting for ~50% of lineages. We identify Bronze Age Greek ancestry in Y-haplogroups nearly absent elsewhere, highlighting their longstanding genetic isolation. The absence of northeast European-related paternal lineages, common in other mainland Greeks, suggests preservation of southern Greece's pre-Medieval genetic landscape. Y-haplogroup phylogeny reveals strong founder effects dated to ~380–670 CE, while the emergence of clan-based social structure is estimated around 1350 CE, centuries earlier than previously thought. In contrast, maternal lineages display greater heterogeneity, primarily originating from ancient Balkan, Levantine, and West Eurasian sources. These results align with historical and anthropological accounts, showcasing Deep Maniots as a genetic snapshot of pre-Medieval southern Greece, offering new perspectives on population continuity and mobility in the Late Antique eastern Mediterranean. At the dawn of Late Antiquity (250 CE), the multiethnic Roman Empire reached its maximal geographical extent, spanning the entire Mediterranean, the Balkans, Asia Minor, parts of the Levant, and most of Western Europe1. Over the next centuries, during the Migration Period (ca. 300–700 CE), sociopolitical factors, epidemics, and the large-scale settlement of new peoples led to the widespread collapse of urban life across the Empire, severely affecting the inhabitants of the Balkan peninsula, including Greece2,3. The invasions and settlements of Slavic peoples into the territory of present-day Greece, which started in the 6th century CE, led to social and demographic upheaval, as far south as the Peloponnese4,5,6,7. This period witnessed a reduction in the Greek-speaking population, a significant proportion of which sought refuge in fortified settlements and inaccessible mountainous regions8, while others learned to coexist with the Slavic newcomers9. Subsequent migrations and settlements by Crusader colonial states, as well as Albanian-speaking groups during the Late Medieval and Early Post-Medieval periods, further complicated the demographic and linguistic landscape of the Peloponnese10. The inhabitants of the Deep (or Mesa/Inner) Mani Peninsula, the Deep Maniots, historically resided in the southernmost tip of mainland Greece, in a region south of a virtual horizontal borderline starting from Areopolis in the west and finishing at Skoutari in the east11,12 (Fig. Historical, linguistic, and archaeological evidence suggests that Deep Maniots were minimally affected by the demographic turmoil of the Migration Period, most likely by blocking the southward advance of Slavic tribes who had settled extensively in the remainder of the Peloponnese5,6,12,13,14. Due to their geographic and cultural isolation in the Late Medieval and subsequent periods, Deep Maniots developed a unique local identity that differentiated them from other Greek speakers in the region11,15. A Summary of traditional subdivisions of Deep Mani and adjacent areas. The blue dashed line indicates the approximate northern boundary of Deep Mani, according to linguistics, social organisation, and the emic perceptions of the region's inhabitants11,12,15. Significant migrations from Deep Mani to Outer Mani and Gytheion after the 17th century transformed the demography of those regions12. Inset map showing an outline of Greece, with Peloponnese highlighted in brown; arrow indicates the location of Deep Mani (inside red square). B Ancestral areas of origin of the newly sequenced present-day Deep Maniots. Each circle corresponds to the origin of each participant. The number of sampling sites (77) does not correspond to the total number of samples (102), as multiple individuals were sequenced from the same or adjacent locations. The satellite image of the Mani Peninsula was created with QGIS 3.40.0.48155 using basemaps from the Esri World Terrain Base Map, ArcGIS Online (Sources: Esri, USGS, NOAA)156,157, while the inset map of Greece is adapted from © Vemaps.com. Due to the absence of a centralised administration and codified judicial system in their region during the Medieval era, the Deep Maniots developed a unique system of customary law that governed every aspect of social life11,13. A characteristic Maniot tradition was a complex, patrilineal clan system, which persisted until the mid-20th century5,11. As a result, Deep Maniot society was highly stratified, characterised by patriarchal clans at the top of the social hierarchy, each claiming descent from a quasi-mythical, often noble, male ancestor who originated outside Mani, typically Constantinople, Asia Minor, Eastern Thrace, other regions of the Peloponnese, or Italy11,15,16. Clans occupied discrete, compartmentalised territories11,13,15,16 and at times engaged in inter-clan warfare and blood feuds, with members protecting themselves by constructing and residing in fortified tower houses that dot Mani's landscape13,15,16. Weaker clans often lived within the territories of powerful clans, making alliances related to economic activities and warfare, while clan-less families occupied the lowest social tier and led a more vulnerable existence11. Clans over time further branched out into subclans, with each subclan often comprising numerous distinct lineages and family names11,15. Most of the Deep Maniot clans surviving today are thought to have originated in the 16th–17th centuries, based on the earliest census of Mani11,12. Whether the Deep Maniot clan system existed prior to this period remains an open question. From the 18th century onward, probably sparked by Western influences on Greek identity supporting direct continuity between the ancient and the modern Greek world, but also due to their militarised culture and the region's historical ties to Sparta, many Deep Maniots increasingly associated themselves with the ancient Spartans17. Despite the plethora of perceived and suggested ancestries, there is no official consensus on the origins of Deep Maniots, puzzling chroniclers even 1100 years ago18. This uncertainty becomes clearer when examining the region's turbulent history. In classical antiquity, Mani was inhabited by poorly known, likely Doric-speaking tribes linked to Sparta and the broader Laconian world19,20. During Roman rule, Mani retained autonomy within the League of the Free Laconians (195 BCE–297 CE)5. After this period, there is practically no historical mention of Mani's population apart from a successful defence against a Vandal invasion in the 5th century CE5,14,21. Following these events, the fate of Deep Mani's inhabitants remains obscure for more than 400 years. Whether present-day Deep Maniots descend from earlier local populations in the Mani Peninsula, resettled Greek-speakers, foreign settlers, or a mix of these, is unknown. Archaeological and linguistic evidence suggests Deep Mani experienced a unique state of isolation compared to other rural Greek regions during this transitional era (5th–9th centuries CE)5,20. This led to the abandonment of classical and Roman architectural styles, replaced by a unique megalithic tradition found exclusively in Deep Mani, preliminarily dated to the pre-Christian and early Christian periods22,23. Remarkably, the interior of Deep Mani may have remained pagan and without a monetary economy until the 9th century CE14,24, despite the introduction of Christianity to coastal Mani during the 5th–8th centuries CE25 and its expansion after 900 CE26,27,28,29. Evidently, Deep Maniots lived in a semi-autonomous enclave with distinct sociocultural traits, paying tribute instead of taxes to the Eastern Roman Empire—a financial arrangement reserved for semi-integrated tribal confederations30. The first mention of the existence of Deep Maniots as a distinct population is historically documented by Eastern Roman Emperor Porphyrogenitus, who noted in the 10th century CE that they ‘are not of the lineage of the [abovementioned] Slavs, but of the Romans of old, who even today are called Hellenes by the locals on account of their former idolatry'18. Over the following centuries, the long-standing isolation of Deep Maniots appears to have persisted and based on historical population estimates, they reached a total population of about 6000 individuals during the 15th century12. Due to the paucity of archaeological data and historical sources for the Early Medieval period, population genetics provide a unique opportunity to elucidate the origins of the Deep Maniots. Previously, an autosomal genetic analysis on Peloponnesian sub-populations, including a small number of Deep Maniots, found that the latter display very low levels of shared ancestry with modern Slavs from northeastern Europe31, which a subsequent study estimated to an average of 30% in other Peloponnesians3,31. Another study examined deep ancestral components (Neolithic, Mesolithic) of Deep Maniots, and their potential shared ancestry with southern Italian populations32. Despite these prior investigations, the actual ancestral origins of Deep Maniots and whether they were affected by demographic events in the region of present-day Greece have remained unstudied. Extensive ancient DNA (aDNA) sampling has elucidated the genetic origins of Bronze and Iron Age populations of central and southern Greece33,34,35,36, although the ancestry of Greece's Roman and Medieval population is largely unknown. Despite this, there is robust evidence that, during the Roman period, the Balkans received a large-scale influx of populations from Asia Minor and the Near East3,37,38,39. Based on their unique historical circumstances and linguistic particularities40,41,42,43,44,45, the Deep Maniots may represent a genetic snapshot of the pre-Migration Period Greek world, providing invaluable insights into human mobility of the post-classical Eastern Mediterranean. Uniparental markers on the Y-chromosome and mitochondrial DNA (mtDNA) help trace human migrations and link present-day humans to ancient populations due to their unique inheritance patterns. These markers reveal demographic changes, migration and its gender dynamics, such as exogamy and sex bias46,47,48,49,50,51,52,53. They also help estimate historical population size, infer the emergence of new peoples and languages33,48,54, and test the reliability of historical records and oral traditions of shared genealogical descent55,56,57,58. Uniparental markers have never been explored among Deep Maniots, leading to a substantial knowledge gap. To elucidate unknown aspects of Deep Maniot history, we studied 102 individuals with confirmed Deep Maniot ancestry on their paternal side, representing all major clans and families. Employing a state-of-the-art targeted enrichment protocol that uses 155,000 probes to sequence over 15 Mbp of the Y chromosome at high depth (35–105×), enabling full haplogroup resolution through analysis of 700 STRs and 750,000 SNPs59,60, we performed deep sequencing of the Y-chromosome on 71 Deep Maniots, supplementing with Y-STR (n = 14) and SNP-testing (n = 17) on the remaining 30 participants (Fig. mtDNA sequence data were also retrieved from 50 of the participants with maternal Deep Maniot ancestry. With this systematic approach, we aimed to uncover the paternal and maternal origins of the Deep Maniots, determine when the founding Deep Maniot population first appeared, and understand how it evolved over time and geographical space. Additionally, we investigate whether the Deep Maniot clan structure affected the distribution of paternal lineages and test the veracity of origin myths and kinship relationships of major clans. Overall, our study provides a comprehensive understanding of the genetic history and social structure of one of the most enigmatic Greek-speaking populations. Our analysis of 102 Deep Maniots revealed the presence of 14 distinct Y-chromosome haplogroups, which are summarised in Supplementary Data 1. Deep Maniots are characterised by an extremely high frequency of macro-haplogroup J-M172 (J2a) at over 80% (Fig. 2A), second only to the Ingush (88%)61, an ethnic group from the Caucasus. Remarkably, the frequency of this haplogroup in mainland Greece does not exceed 16% (Supplementary Data 2), peaking in the island of Crete at 30–35%62,63. In the broader Eastern Mediterranean region, the highest frequencies of J-M172 can be found among Greek Cypriots and the Lebanese, at ∼30%63,64,65, much lower than what observed among Deep Maniots. Comparative distribution of major Y-DNA haplogroups in A Deep Maniots and B other mainland Greeks. C NMDS plot derived from Rogers' pairwise distances of 17 Y-STRs of 94 west Eurasian populations. The list of populations can be found in Supplementary Data 5. The J-M172 lineage traces its origins to the Caucasus-Zagros region ca. 28,000–26,000 BCE, according to the latest FamilyTreeDNA66 and Yfull67 estimates, and its earliest aDNA record is in Mesolithic Caucasus Hunter Gatherers68 and Neolithic farmers from present-day Iran69, during the 8th millennium BCE. Nine of the 14 haplogroups found in Deep Maniots belong to J-M172 (Supplementary Data 1), and the remaining lineages are assigned to macro-haplogroups R1b-Z2103 (R1b1a1b1b; 8%), G-L13 (G2a2b2a1a1a1a; 7%), J1 (2%), E-V13 (E1b1b1a1b1a; 1%) and R1a-Z93 (R1a1a1b2; 1%) (Fig. The phylogeographic history of each Deep Maniot lineage is analysed extensively in subsequent sections of this study. As noted above, Deep Maniot Y-chromosome haplogroup distributions are markedly different compared to a sample of 944 citizens of mainland Greece sourced from the literature (Fig. Lineages associated with Germanic (I-M253) and Slavic (I-CTS10228, R1a-Z282) peoples3,70, who massively settled the Balkans during the Migration Period3, are found in a combined frequency of 22% in present-day mainland Greeks (Fig. Notably, these lineages are entirely absent from the Deep Maniot dataset (Fig. 2A, Supplementary Data 1, 2), suggesting limited to no paternal contribution from Germanic and Slavic peoples. The low incidence of Y-haplogroup J1 (J-M267) (2%) in Deep Mani (Fig. 2A) also indicates limited contribution from Levantine populations, where this haplogroup is found today in significant frequencies (23–66%)49,66 (some J1 lineages are also known from BA Greece)35. The Deep Maniot sample also lacks Y-haplogroups associated with the Central-Northern European (R1b-U152, R1b-L2, R1b-U106)71,72 and Albanian expansions (e.g. R1b-BY611)73 into Greece in the Middle Ages. Remarkably, Y-haplogroup E-V13, the dominant paternal lineage of present-day mainland Greeks (20%; Fig. 2B), is found in very low frequencies in Deep Mani (1%). Although E-V13 was previously thought to have been transmitted by archaic Greeks74, it is so far entirely absent in ancient samples from Bronze and Iron Age central and southern Greece (Supplementary Data 3), with its earliest record in the Hellenistic era75. A recent study has suggested the large-scale dissemination of this haplogroup occurred in at least three major pulses: one in the Iron Age and Roman Period with populations located north of present-day Greece, such as the ancient Daco-Thracians, one with Aromanians and another with Albanians in the Middle Ages73. The disparities in the frequencies of major haplogroups between mainland Greeks and Deep Maniots demonstrate that the latter were minimally affected by demographic events that shaped the paternal genetic landscape of Balkan populations during the Migration and Medieval Periods. To visualise the paternal genetic relationships of Deep Maniots (17 STRs available for 75 individuals) with a large dataset (n = 12,647) of 94 west Eurasian populations grouped into 61 metapopulations, including 405 mainland Greeks (Supplementary Data 4), we applied Non-Metric Multidimensional Scaling (NMDS) on pairwise Rogers distances76 calculated on 17 Y-chromosome STRs (Fig. 2C) based on geography, previously demonstrated genetic relatedness, and the low stress value of 0.1657. Notable examples include Greek and Turkish Cypriots, who form a cohesive group, as expected64,65, while speakers of South Slavic languages cluster together compared to mainland Greeks and North Macedonian Albanians. Notably, Deep Maniots do not cluster close to any Balkan population, instead plotting towards Caucasian groups, especially with the outlying Ingush (Fig. This clustering pattern is evidently the result of the extremely high frequencies of haplogroup J-M172, shared by both populations, and does not reflect actual recent ancestry. Indeed, the J-M172 subclades found in Deep Maniots are entirely different to those of the Ingush, with the only related lineage, J-FTF77337, sharing a common ancestor with Caucasian populations at the root of J-Z1847 (J2a1a1a2b2a), at ca. We should also note that Y-chromosome haplogroups represent a single, non-recombinant marker, and therefore capture only a fraction of an individual's ancestry. However, the paternal haplogroups of Deep Maniots clearly indicate that they are a genetic isolate within Greece, in accordance with previous studies based on autosomal ancestry31, indicating congruence between the two data sources. To quantify genetic diversity within the Deep Maniot population, we estimated Nei's haplotype diversity (H) at two levels of Y-STR resolution: using 111-locus haplotypes (n = 69) and 17-locus haplotypes (n = 75). We found that the 69 Y-111 STR haplotypes are unique (H = 1, i.e. there are no shared haplotypes between any of the Deep Maniot study participants), indicating population divergence over many centuries. Among the 75 Y-17 STR haplotypes, there are 44 unique haplotypes, of which 35 occur only once in the sample (Supplementary Data 6). The lower haplotype diversity at Y-17 (H ≈ 0.97) versus the maximal diversity at Y-111 (H = 1.00) primarily reflects the gain in discriminatory power when moving from 17 to 111 loci. When using only 17 Y-STR loci, the resolution is lower, meaning that distinct male lineages may appear identical due to limited marker variation. This results in a higher number of shared haplotypes within the Deep Maniot population, even among individuals with deep ancestral divergence. Among the 44 unique Deep Maniot haplotypes at the Y-17 STR level, we did not find a single haplotype that was shared with individuals from other populations in our comparative dataset of nearly 13,000 west Eurasians, also including 405 mainland Greeks (Supplementary Data 4). When relaxing the matching criteria and allowing for ‘−1 matches', only 11 matches were found for 5 Deep Maniot haplotypes (Supplementary Data 6). However, these matches reflect deep divergences (ca. 12,000–1,450 BCE; Supplementary Data 6) and are therefore not closely related to any Deep Maniot, except for a single Greek individual, who might belong to Deep Maniot-specific haplogroup J-L930 (see below). Our discovery that no exact haplotype is shared within our dataset of nearly 13,000 individuals from 94 population groups of interest, highlights the remarkable rarity of Deep Maniot haplotypes outside the Mani peninsula. To increase the comparative strength of our search, we queried FamilyTreeDNA's global customer Y-STR and Y-SNP database (>995,000 individuals). We found only 7 high-level STR matches (≥37 STRs) with non-Deep Maniots in the entire database (Supplementary Data 6). Using FamilyTreeDNA's STR-based TMRCA estimation algorithm (FTDNATiP), we find that all these matches are recent—within the last 350–900 years, and 5/7 belong to Deep Maniot-specific lineages (J-L930, J-FTF87157, Fig. 3), probably representing descendants of Deep Maniots immigrating to other areas rather than migrations of outsiders into Deep Mani. We next queried FamilyTreeDNA's >145,000 present-day user records at high sequence resolution59. Once more, we found only two Deep Maniot Y-haplogroups forming a subclade with non-Deep Maniots, at 750 BCE and 282 CE, respectively (Supplementary Data 6). TMRCA estimates were derived using the FTDNATiP™ algorithm, which calculates coalescence dates from Y-STR genetic distances. Accuracy varies depending on the number of STR markers used, with higher-resolution profiles (e.g. 67–111 STRs) yielding narrower and more precise intervals than lower-resolution sets (e.g. 37 STRs). STR resolution for each sample is listed in Supplementary Data 6. Overall, the STR and SNP-based comparative analyses and the outlying position of Deep Maniots on the NMDS plot (Fig. 2C) are consistent with patterns observed in genetically isolated and/or drifted groups, a term referring to populations that experienced random fluctuations in haplotype frequencies over successive generations due to chance variation in reproductive success77,78. The extreme frequency of J-M172 and the absence of shared haplotypes with other populations suggest that genetic drift, likely driven by long-term isolation and small effective population size, has potentially played an important role in shaping the Deep Maniot Y-chromosome landscape. Consequently, Deep Maniots represent a genetic island that has experienced longstanding isolation and has more likely acted as an exporter of genetic diversity to other parts of Greece and beyond, rather than a sink for newcomers. These results are congruent with historical sources that have extensively documented significant migrations out of Deep Mani towards Italy, Corsica, Balearic Islands, Asia Minor, and the Greek-speaking world more broadly (other parts of the Peloponnese, Ionian islands, Crete, Northern Aegean, Dodecanese) from the 16th century onwards12. Although substantial relative to Deep Mani's small population, these migrations may have left a limited demographic imprint on the regions they reached (with the exception of adjacent areas such as Gytheion), as suggested by the low number of genetic matches to non-Deep Maniots. This section presents detailed phylogeographic information on the three most frequent Deep Maniot lineages (J-L930, J-FTF87157 and R-FTE77744), and their possible connections to populations from the past, as determined by aDNA. Each haplogroup is presented in an order that follows the presence of the earliest available related lineages in the aDNA record. A summary of all haplogroups and their phylogeographic associations are provided in Table 1, whereas for an extended discussion on the origins of the less frequent Deep Maniot Y-DNA lineages, refer to the S1 Text. Haplogroup J-SK1363>J-BY759>J-FTF87157 represents 11% of the Deep Maniot patrilines in our dataset. It has so far been found exclusively in the tip of the Mani Peninsula (and nowhere else globally), where it represents the majority of paternal haplogroups (48%) (Fig. This lineage and its parent branch J-BY759 are strongly associated with Bronze and Iron Age Greece35,36, where they account for 15% of Mycenaean and archaic Greek patrilines (Supplementary Data 3). J-BY759 and its daughter branches have also been found in Greek colonies in Bronze Age Cyprus33 and Sicily (especially Doric sites)79, in individuals from Punic colonies with an Aegean Bronze Age autosomal profile80, and in Imperial Rome38 (Fig. Upstream branch J-SK1363 and its numerous daughter lineages have been recorded in Middle Neolithic-Chalcolithic cultures from present-day Bulgaria81, Croatia82,83, Hungary84, Romania36 and the Cycladic Culture of the Aegean85, confirming the presence of this lineage in the Balkan region and the Aegean since at least 6,500 BCE. Based on the above, Deep Maniot haplogroup J-FTF87157 indicates direct descent from the inhabitants of Bronze Age Greece, whose present-day expansions are confined to the Mani Peninsula (Fig. A Frequency of Deep Maniot patrilines in the different traditional subdivisions of Deep Mani. B Phylogeny of haplogroup J-L930, and the spatial geographical distribution of its subclades in the different traditional subdivisions of Deep Mani and adjacent regions. Note that the distributions of different J-L930 subclades are often geographically localised and non-overlapping. Subclades shown in bold represent lineages for which ancient DNA samples have been found downstream. Note that subclade R-Z2106>R-Z2109 is excluded from this analysis. A subset of the newly genotyped Deep Maniots and a Sicilian from Palermo (retrieved from the FamilyTreeDNA Y-STR and Y-SNP database) form a novel branch of R1b-Z2103>Z2106 (R1b1a1b1b3), namely R-FTE77744 (8% frequency in Deep Mani, Fig. Haplogroup R1b-Z2103 (R1b1a1b1b) was the predominant paternal lineage of the Yamnaya culture from the Pontic-Caspian steppe, which is associated with the dissemination of Indo-European languages into the Balkans and Armenia during the Early Bronze Age (EBA)33,36,86,87. Although the ultimate origin of R-FTE77744 clearly lies in the EBA Pontic-Caspian steppe, its more recent origins are puzzling. R-Z2106 has three daughter branches with ancient and present-day samples found almost exclusively in the North Caucasus (Fig. 5), one branch (R-BY44400) in EBA Moldova, Romania, and Ukraine, and another branch, R-Z2108/Z2109, is found overwhelmingly in the Bronze Age cultures of Albania, Greece, and North Macedonia33 (Fig. No STR or SNP matches for R-FTE77744 were found in the FamilyTreeDNA database (>995,000 individuals) and an extensive dataset of populations from the Caucasus (Supplementary Data 4). Although R1b-Z2103>Z2106>Z2108/Z2109 is the predominant steppe-related patriline in EBA-IA Greece, other R1b haplogroups have also been found (R1b-PF7563, R1b-V1636, Supplementary Data 3), suggesting considerable diversity of Eneolithic-EBA Pontic-Caspian steppe-derived lineages in ancient Greece. It is therefore likely that R-FTE77744 is yet another, remarkably rare, relict lineage from EBA-IA Greece. This hypothesis is further supported by the split between Deep Maniots and the Sicilian individual carrying this subclade, which is estimated at ca. 810 BCE—concurrent with the earliest phase of Greek settlement in Sicily during the Iron Age88. The origins of the remaining Y-DNA haplogroups of Deep Maniots are often obscure, although, as we show in Table 1 and S1 Text, a West Asian or Balkan origin during the Roman Period or earlier is likely for almost all of them. We define haplogroup J-L26 (J2a1a)>J-PF5087>J-PF5160>J-L930 as the Deep Maniot Modal Lineage, due to its extraordinarily high frequency across Deep Maniots (51%; Fig. 4A), and near exclusive association with this population. In FamilyTreeDNA's dataset of 673,000 SNP-tested users, only 8 individuals with haplogroup J-L930 and no known Deep Maniot ancestry were located (Supplementary Data 7). Of these, 7 share large (12–43 cM) identity-by-descent (IBD) segments with two autosomally tested Deep Maniots in our dataset, suggesting a recent origin from Deep Mani (Supplementary Data 7). For reference, the two autosomally tested Deep Maniots share 26–62 cM of their ancestry with genealogically unrelated Deep Maniots in FamilyTreeDNA's autosomal (Family Finder) database (Supplementary Data 7). Furthermore, as noted above (Fig. 3), all J-L930 STR-based matches from FamilyTreeDNA's database also share recent (1300–1600 CE) matching with Deep Maniots, further showcasing the association of this lineage with the Mani Peninsula. By reconstructing the phylogeny of J-L930 (Fig. 6A), we show that it has four daughter branches with remarkably localised distributions, namely: J-FTE86410, the most frequent subclade, which dominates the area of Katopangi and Niklianiko (southwest Deep Mani); J-FTG29984, largely found in the area of Xoumero (northwest Deep Mani); J-FTA23105, primarily found in Xoumero-Katopangi; and J-Y251116, which is exclusively found in eastern (Prosiliaki) Deep Mani, with the exception of an islander from Cythera, ca. 40 km off the coast of Eastern Deep Mani—an island known to have received Deep Maniot settlers12, and a single person from the transitional area between Deep and Outer Mani. These branching patterns are also mirrored in an NMDS plot of 111 Y-STRs of all high-level tests of J-L930 (Fig. Therefore, three of the four daughter branches of J-L930 are found in western (Aposkieri) Deep Mani, and the earliest splitting subclade, J-FTE86410 (ca. 860 CE), experiences all its branching there, where it also attains its highest frequency (Fig. Considering these branching patterns, we propose western Deep Mani as the likeliest geographical area for the origin of J-L930. A Phylogenetic tree of J-L930 based on 41 Y-DNA sequences (derived from FamilyTreeDNA's manually curated Y-chromosome DNA haplotree)66,95. Bottom left inset shows main geographical zones of Deep Mani; symbols represent the geographical origin of each sampled individual. The asterisk (*) refers to a Cytherian islander and an unidentified sample from the Harvard Personal Genome Project. B NMDS plot generated from 111 STR values from the main subclades of J-L930. Each point represents an individual sample, with the sample number indicated above the corresponding dot (see Supplementary Data 1 for full sample details). C Frequency of each J-L930 subclade. Note that 19% of the results correspond to low-level SNP testing, which does not allow for final subclade resolution. Despite its present-day abundance in Deep Mani, the distant origins of J-L930 are puzzling, as it has never been found in the aDNA record. Branching patterns of parallel lineages to J-L930 in FamilyTreeDNA's haplotree may indicate an association with the Neolithic and EBA Levant and Cyprus, although parent branch J-Z8072 (at 7500 BCE) has been found in BA North Caucasus (1450–1200 BCE)89 (Fig. 5), potentially indicating an origin in that region. Additionally, a present-day individual from Albania is also known from upstream lineage J-FGC68843/J-Y31950 (at 2800 BCE)67, indicating possible connections with the EBA Balkans. Overall, virtually every Y-DNA haplogroup found in Deep Mani reveals a consistent pattern of paternal ancestry rooted in the ancient Balkans and West Asia (Table 1, Supplementary Data 8, 9), with strong associations to Bronze Age, Iron Age, and Roman-period Greek-speaking populations (S1 Text, Table 1). These lineages, while diverse in origin, are united by their extreme rarity outside Deep Mani and their apparent arrival prior to or during Antiquity. The haplogroup composition of Deep Maniots suggests the presence of longstanding isolation and localised expansion within different regions of Deep Mani, resulting in the frequency patterns observed in the present day. The Y-DNA data presented here, together with previous autosomal analyses31, have identified Deep Maniots as a distinct population isolate. However, when the Deep Maniots arose as a distinct group has remained unanswered. To gain insights into the temporal origins of present-day Deep Maniots, we plot the mean TMRCAs of haplogroups specific to this population for which extensive data is available (J-L930, J-FTF87157, R-FTE77744, J-PH4244) (Fig. Remarkably, the two most frequent Deep Maniot haplogroups (J-L930, J-FTF87157, 62% of all patrilines) show a sudden and steep increase in subclade diversity only after 380–670 CE (Fig. 7), suggesting that this was the beginning of a period of population expansion, after a significant bottleneck. A similar date was provided for J-L930 by Yfull (ca. 800 CE, branch name J-Y239616)67, despite employing a different method of TMRCA estimation90 and using a much smaller dataset (n = 4). These results are consistent with the first mentions of the Bishopric of Mani (901–907 CE), the appearance of Deep Maniots as a distinct group in historic records (ca. 950 CE)13, and putative dating of the earliest megalithic constructions in the Mani Peninsula (ca. Moreover, the branching patterns of the two most frequent Deep Maniot haplogroups, J-L930 and J-FTF87157, can also provide crucial insights into the settlement dynamics of the Mani Peninsula, as they reveal striking geographic structuring that may reflect historical migrational waves and founder effects within Deep Mani (Fig. An asterisk indicates the date during which the Deep Maniot-specific mutation arose for each of the plotted haplogroups. Graph based on Supplementary Data 10. The clan system of Deep Mani is one of the major societal characteristics of this population, which differentiates them from all other mainland Greeks11,12. However, the origins of the Deep Maniot clan system remain obscure, with one major study suggesting an origin in the mid-16th century11. Our sampling strategy included testing of major Deep Maniot clans, typically inhabiting different villages and reporting shared patrilineal descent (Supplementary Data 1). In 11 of these cases, our dataset includes two or more individuals, which enables us to provide a minimum estimate for the chronology of their founders (Fig. Our analysis suggests that founders of key Deep Maniot clans lived between 1350–1600 CE (Fig. 8), at least 200 years earlier than their suggested first appearance by previous historical studies11,12. These dates agree with earlier historical sources, which indicate the destruction of tower houses in Mani by the Imperial administration in 1415 CE, in order to restore order in the region91. Similar tower houses, which are integral to the institution of the clan, were described in 1445 CE by travellers to Deep Mani92. Together, these findings offer the first genetic framework for dating the emergence of the Deep Maniot clan system, revealing a timeline that opens new avenues for understanding the sociopolitical transformations that shaped Deep Mani during the Late Medieval period. We present detailed information for the remarkable oral and genetic history of each analysed clan in S1 Text. The analysis is based on targeted enrichment sequencing, incorporating ~700 Y-STR markers and 750,000 SNPs59. Kinship is only one of the manifestations of the Deep Maniot clan system. Another aspect of this social organisation concerns the semi-mythological origins of the founder of each clan. Among the Deep Maniot families in our dataset, we recorded 15 mythologies suggesting origins from Eastern Roman officials, Emperors, Crusaders, or high-ranking individuals from other parts of the Peloponnese (S1 Text; Supplementary Data 12). Considering that most of these individuals patrilineally descend from haplogroups that are exclusive to Deep Mani (especially J-L930, found in 11/15 of the mythological ancestors, Supplementary Data 12), we suggest that oral histories of noble descent are unsupported by our genetic findings. We stress that genetic relatedness represents only one dimension of descent, and does not encompass the full spectrum of identity, collective memory, and belonging that these origin stories convey. Considering the stark differences in Y-chromosome haplogroup composition between Deep Maniots and other mainland Greeks, the paternal ancestry of geographically adjacent, related populations might be informative on demographic events that shaped the Mani Peninsula as a whole. The inhabitants of west and southeast Taygetos, collectively known as Outer Maniots, have been shown by previous work to be autosomally closely related to the Deep Maniots, as they share large mean pairwise IBD segments (ca. However, they occupy distinct positions on autosomal-based PCAs and are characterised by different K8 ADMIXTURE components in the study of Stamatoyanopoulos et al.31, suggesting additional demographic influences compared to Deep Maniots. While both PCA and ADMIXTURE analyses can be influenced by genetic drift93,94, autosomal distinction is further supported by the findings of Raveane et al.32, who show that populations from Laconia and eastern Taygetos exhibit higher proportions of Bronze Age Pontic-Caspian steppe-related ancestry compared to Deep Maniots, possibly due to higher input from northeast European groups. By exploring the autosomal customer dataset of FamilyTreeDNA, we located paternal lineages for 13 Outer Maniots, primarily from Messenian Mani (n = 11) (Supplementary Data 13). Although our sample size is small, some inferences can be drawn. The principal haplogroup of Outer Maniots is E-V13>E-Z16659 (46%), in contrast to Deep Maniots, where E-V13 is remarkably rare (1%). Remarkably, one sequenced Messenian Maniot belongs to subclade E-V13>E-Z16659>E-Y3183>E-S2972>E-PH3589>E-S2978>E-BY5285>E-BY116895>E-FTF92003, which is different to the Deep Maniot E-V13 lineage (E-V13>E-BY3880>E-Y16729>E-BY202063>E-FT64983*). The Messenian Maniot forms a subclade with a man from Trifylia, in western coastal Messenia, dated to ca. 620 CE, while upstream lineage E-BY202063 (dated to 480 CE), is found in an individual from the Evrotas valley in Laconia, 80 km to the north of Messenian Mani95. The distribution and TMRCA of this lineage suggest a presence of this subclade in the broader southern Peloponnese at least since the Late Roman period. All other lineages found in Outer Mani (e.g. E-M34, G-PF3146, Ι-Μ223, J-S18579; Supplementary Data 13) are not present in our extensive dataset of Deep Maniots, even at the macro-haplogroup level. We also found two lineages associated with the Migration Period (R1a-Z282>R-YP372, I-M423>I-CTS10228; Supplementary Data 13), which account for 15% of the patrilines in Messenian Mani. Based on the above preliminary results, the Outer Maniots represent a population that experienced different influences on paternal ancestry from their southern kin, the Deep Maniots. Considering the autosomal relatedness (based on IBD-sharing) and the similarities in social organisation and dialect of the two populations11,12,31, it is likely that complex demographic and genetic processes may have led to a transfer of the Deep Maniot cultural package to Outer Mani, uniting the two areas into a common zone of influence and interaction, leading to shared features in their identity. Given that Deep Maniot society was strongly patriarchal until recent times11, the histories of its women remain largely obscure. To address this knowledge gap, we examined mtDNA sequences and terminal haplogroups in 50 Deep Maniots of our dataset whose maternal lineages originate from the Mani Peninsula (Supplementary Data 14), by querying FamilyTreeDNA's Mitotree (>260,000 present-day and 10,800 aDNA samples) and GenBank. Our analysis revealed the presence of at least 30 distinct maternal haplogroups in Deep Mani and the transitional zone of Oetylon and Gytheion (Fig. A mtDNA haplogroup frequency in Mani. B Geographical distribution of mtDNA haplogroups in the Mani Peninsula. The satellite image of the Mani Peninsula was created with QGIS 3.40.0.48155 using a basemap from the Esri World Terrain Base Map, ArcGIS Online (Sources: Esri, USGS, NOAA)156,157. Although the limited resolution of mtDNA renders the origins of many of Deep Maniot matrilines obscure, distinct connections with aDNA samples can be made for most maternal lineages. Haplogroups H7c1k1, H35, HV1e1, HV4b2, HV119, N1b1a13a1a, U1a1a23a, U1a1d1d, U5a1a2 and U8b1b1e, collectively accounting for 38% of all matrilines, have broad connections to BA-IA populations from the Balkans, West Asia, Caucasus, and the Levant, based on their matches to ancient and present-day populations (Supplementary Data 14). We mention the phylogeography of certain region-specific lineages below. The origin of H7c1k1 (12%) likely lies in the ancient eastern Mediterranean, as H7c1 lineages have been found in the BA, Roman and Medieval Balkans36,39 (Fig. 10A) and present-day matches include a Romanian and two Saudis, while upstream lineage H7c1k is found in a present-day Lebanese individual (Supplementary Data 14). A Network of haplogroup H7c. B Network for haplogroup U1a1. C Network for haplogroup U5a1b. D Network for haplogroup HV22'65'119-HV119; panel slightly enlarged due to the smaller number of analysed lineages compared to the other haplogroups. Haplogroup H35 (2%) is a particularly rare lineage that in the aDNA record has been found exclusively among individuals whose autosomal profile is similar to populations from Early Iron Age Bulgaria (Supplementary Data 14). Individuals with this autosomal profile and maternal lineage have been found in Iron Age North Macedonia and Moldova, as well as Late Antique Italy and Hungary (Supplementary Data 14). We therefore interpret this lineage as originating from the Iron Age Balkans. Lineage U1a1d1d (4%) has a particularly old presence in the Eastern Mediterranean, having been recovered from remains in Chalcolithic Bulgaria and southeastern West Asia (Fig. HV119 (8%) and its parent haplogroup, HV22'65'119, represent an extremely rare lineage, with ancient samples from Bronze Age Serbia, Iron Age northeastern France, an undated ancient individual from Tarquinia, and present-day matches from Italy (Fig. 10D), Slovenia, Germany and Holland (Supplementary Data 14). This haplogroup likely represents a lineage that ultimately originates from western Europe, especially the alpine region and the Italian Peninsula, which may have entered the Balkans already by the Bronze Age. Two lineages, R1a1g2 and K1a4f1 (4%), show strong connections to ancient and present-day populations of the Caucasus (Supplementary Data 14). Some lineages in Deep Mani have a broad West Eurasian distribution that does not currently permit their association with specific populations after the Bronze Age—these include T2a1a54, U5a1a2b4, U5a1b1, U5a2b, W3a1 + 200, and K1c1f3 (totalling 24% of all matrilines). Although T2a1a54 has been found only in Deep Maniots and two present-day Greeks, T2a1 and its subclades are known from a broad range of aDNA samples and appear to have originated in the Pontic-Caspian steppe with the Yamnaya and other Indo-European groups that followed them in the region (Supplementary Data 14). U5a1b1 (10%) can also be traced to the Yamnaya steppe herders, with daughter lineages across present-day West Eurasia (Fig. A group of haplogroups, namely, H9a + 16519, HV9 + 152, H149″455, T2f2, X2ap (collectively accounting for 12% of all matrilines) are lineages that so far make their appearance in the Balkan aDNA record during the Migration Period, in present-day northeastern Europeans, or, in the case of T2f2, in a Roman Period individual with a nomadic steppe profile (Supplementary Data 14). Some surprising connections to more distant populations have been observed. Haplogroup U6a6a1a1a (6%) originates from the indigenous populations of the Maghreb and the Canary Islands96,97,98, while M5a1b1a1d (2%) represents one of the founding Roma lineages99. A group of haplogroups (H175, H83c, H85i^, HV0c, HV13g1, V2k; 14% of all matrilines), do not have sufficient phylogeographic resolution to elucidate their origins based on current aDNA or present-day distributional patterns. In the present study, we unveiled the uniparental inheritance patterns and ancestry of Deep Maniots for the first time. In agreement with previous research exploring autosomal ancestry31, we show that Deep Maniots represent a genetic island within mainland Greece. Although the previous study suggested limited contribution of Slavic peoples to the ancestry of present-day Deep Maniots, it did not explore the deeper origins of the Maniots or their relationship to ancient populations. Our findings demonstrate that Deep Maniots overwhelmingly descend from paternal lineages associated with the populations of BA-IA and Roman Greece. Remarkably, the complete absence of haplogroups associated with Germanic, Slavic, Aromanian, Albanian, and Western European populations, which contributed (to varying degrees) to the ancestry of mainland Greeks over the past 1400 years (Fig. 2A, B), further supports historical accounts that Deep Maniots were largely shielded from the tumultuous demographic transformations that took place in the Balkans during the Migration Period and for centuries to come, up to the present. We note that the pronounced founder effects observed in Deep Maniots and the near absence of Y-STR matches with non-Deep Maniots in our dataset are key indicators of genetic isolation, very likely associated with drift78. In particular, the high frequency (51%) of J-L930—which we name the Deep Maniot Modal Lineage, and J-FTF87157 (11% frequency), coupled with their restricted geographic distribution, suggest that genetic drift has probably amplified certain lineages in Deep Mani while eliminating others over time. This process likely contributed to the Deep Maniots' genetic distinctiveness relative to other mainland Greeks. However, although genetic drift has likely shaped the Deep Maniot genetic landscape, particularly in terms of haplogroup frequencies, multiple lines of evidence suggest that the ancestry of Deep Maniots is not the result of merely stochastic processes. While drift likely elevated certain haplogroup frequencies and reduced others, its effects should be broadly distributed; the absence of Migration Period lineages suggests they were likely rare or absent in the founding Deep Maniot population. Furthermore, an autosomal study has shown limited influence from present-day Slavic populations31, while in PCA analyses with ancient samples, Deep Maniots cluster with the East-Mediterranean-shifted Imperial Roman individuals32, close to Mycenaeans, suggesting affinity to pre-Slavic southern Balkan3 and Italian populations37 with similar genetic profiles. Taken together, these findings support our interpretation that Deep Maniots represent a snapshot of regional genetic diversity prior to the demographic transformations of the Migration Period, retaining a significant proportion of ancestry dating back to at least the Late Roman era. While the observed founder effects in Deep Mani, dated to the 4th–7th centuries CE, most likely suggest continuity from earlier local populations, introgression from neighbouring regions of individuals with a Roman Period autosomal profile, followed by re-expansion could be suggested as an alternative explanation. Haplogroup J-L930, for instance, which expands during and after the 7th century CE, could reflect such a process. However, the absence of J-L930 from global Y-DNA datasets suggests its directly ancestral lineages are either extremely rare or even extinct outside Deep Mani. Furthermore, the high frequency of J-L930 in our dataset may have obscured finer-scale variation among other Deep Maniot lineages, which were less likely to be encountered during our sampling. Additionally, Mani's tradition of blood feuds may have led to localised lineage extinctions, reducing Y-DNA diversity and erasing lineages that once captured broader haplogroup variation—including earlier branching of J-L930 within Deep Mani. Taken together, these patterns likely reflect repeated cycles of expansion and contraction within an insularised population, shaped by internal dynamics rather than external input. The abovementioned information also enables us to add crucial insights into a vexing question regarding the origins of the Deep Maniots, that is, whether they descend from earlier populations in the Mani Peninsula, or from Greek speakers who sought refuge there from other regions of the Empire during and after the Migration Period. While this question cannot be conclusively resolved without an extensive aDNA transect of Roman and Medieval Deep Mani, our findings point to genetic continuity from local groups, supported by two compelling lines of evidence. Firstly, the temporal analysis of the most frequent Deep Maniot patrilines, J-L930 and J-FTF87157, reveals a pronounced founder effect occurring between 380 and 670 CE, likely following a population bottleneck. This genetic signature aligns with the earliest historical references to Deep Maniots as a distinct Greek-speaking people18 and implies that the foundational paternal lines of this community stem from individuals who endured this pivotal bottleneck event as part of a single, cohesive group. It is therefore very likely that most present-day Deep Maniots derive their paternal ancestry from the 4th–8th century inhabitants of Deep Mani. Secondly, should Deep Maniots descend from an amalgamation of different Greek speakers from nearby or more distant regions over the past 1400 years, one would expect STR and SNP-matching with other populations. The exceptional rarity of most Deep Maniot patrilines outside the Mani Peninsula and the absence of matches with other populations are clear indicators of longstanding isolation that likely predates the Migration Period. In particular, the deep split of J-FTF87157 within Deep Mani (380 CE) predates the 6th century CE invasions and settlements of Slavic peoples into the region of present-day Greece2,3, indicating long-term continuity of Deep Maniot lineages within the Mani Peninsula. The founder effect that defines the two most frequent Deep Maniot patrilines (J-L930, J-FTF87157) around 380–670 CE may have had multiple causes. Key historical stressors during this period include the Justinianic Plague (6th–8th centuries CE)2,6,30, the onset of Slavic migrations into Greece (ca. 580 CE)2,4,100, and a series of Arab maritime incursions affecting the Greek islands and coastline post-650 CE7,30. Each of these events could have reinforced the isolation—both cultural and genetic—of Deep Mani's inhabitants and may have even caused a dramatic reduction in the size of the population. Notably, the four-century-long silence in historical sources concerning Deep Mani and its people likely marked a formative period in which the Deep Maniots developed into a distinct group, giving rise to enduring traits such as their unique megalithic building traditions. Indeed, one of the more striking revelations of our research lies in the interplay between material culture and genetics. The distribution of Mani Peninsula's megalithic architecture—largely dated to the pre-Christian or early Christian era22—precisely mirrors both the historic borders of Deep Mani (south of Areopolis and Skoutari) and the geographical prevalence of Deep Maniot-specific paternal lineages (Fig. In contrast, regions immediately north of Deep Mani exhibit distinct patrilineal compositions, implying divergent demographic histories. This close correspondence suggests that the megalithic structures were the cultural artifacts of a population directly ancestral to today's Deep Maniots—one that remained bound within this unique cultural zone for centuries, if not millennia. Due to their isolation and the diminished role of imperial administration in this corner of the southern Peloponnese, the Deep Maniots developed a unique system of customary law, which involved blood feuds as a last resort in conflict resolution. The Deep Maniots' aptitude for warfare is documented as early as the 13th century, when the medieval feudal Latin states, specifically the Principality of Achaea, attempted to quell their rebellious activities by building fortresses and castles along the northern borders of Deep Mani10,24. After the Eastern Roman Empire regained control of the Peloponnese from the Latin states, many defensive fortresses or tower houses in Mani were allegedly destroyed in 1415 CE, in order to prevent anarchy and eliminate warfare practices used by the Deep Maniots10,24,91. Such historical events point to a militarised culture that likely facilitated the development of clan structures. Indeed, our ability to trace the founders of certain Deep Maniot clans to the 14th-15th centuries CE supports this hypothesis, complementing broader scholarship, which suggests that clan institutions often emerge in regions lacking centralised governance, in order to foster cooperation in challenging circumstances11,101,102,103. Although previous studies suggested an origin of the clan system in the 16th century11,12, our analyses have recovered clan founders two centuries earlier (Fig. This result should be treated as a minimum estimate, as examination of censuses12,104 and Deep Maniot oral traditions (S1 Text) demonstrate constant turnovers in clan distribution in the Mani Peninsula. Indeed, we show that the earliest recorded clans of Deep Mani, who were prominent and widespread in 1514, occupy only a handful of locations in the present day (S1 Text). It is therefore likely that clans existed prior to the 14th century and may have become reduced or even extinct before the earliest censuses took place. Our analyses of the maternal ancestry of Deep Maniots revealed a more complex genetic landscape, with influences ranging from the ancient Balkans and the Levant to Western Europe and the Maghreb. We should highlight, however, that mtDNA haplogroups provide lower resolution for population genetics compared to Y-chromosome haplogroups, due to higher mutation rates and smaller genome size, leading to backmutation and saturation105. As a result, our analyses can help elucidate the phylogeographic origins of Deep Maniot maternal lineages, but in most cases, the arrival date of each mtDNA haplogroup to Deep Mani cannot be determined with confidence. Although the precise timeframe during which many maternal lineages entered the Deep Maniot gene pool cannot be established for all lineages, it is likely that a substantial portion was already present in the founding population. Mirroring overall Y-DNA frequency patterns, several maternal haplogroups, namely H7c1k1, HV119, T2a1a54, U5a1b1, and U6a6a1a1a appear to have undergone founder effects, collectively accounting for ~42% of all matrilines (Fig. Notably, FamilyTreeDNA's mtDNA Time Tree estimates the TMRCA for the HV119 and H7c1k1 founder events to fall between ca. 540 CE and 866 CE, a period that overlaps with the Y-DNA founder-effect timeframe (380–670 CE). As with Y-DNA, several maternal lineages are Deep Maniot-specific, with most showing no close matches to other populations and a distribution confined to the Mani Peninsula. Notably, haplogroup H7c1k1 may represent the maternal analogue of J-L930, as it is found across Deep Mani and extends to several locations in Outer Mani as well (Fig. We also did not find any non-Deep Maniot matches to H7c1k1 other than a single Romanian and two Saudi Arabians (Supplementary Data 14). We should also note that mtDNA lineages of 5 Neolithic samples from Diros Cave in Deep Mani have been previously published83 (Supplementary Data 3). None of these haplogroups have been found in the present-day population of Deep Mani (Fig. 9), which is attributable either to limitations of the sample size of our dataset (n = 50), or to uniparental turnovers during the Neolithic-Bronze Age transition and later periods. Despite these general patterns of isolation, we also detected limited maternal gene flow from non-Deep Maniot populations. We recovered a single M5a1b1a1d sample—a lineage commonly associated with Roma populations, who are known to have arrived in the Balkans by at least the 12th century CE106—as well as haplogroups likely linked to the Migration Period (H9a + 16519, HV9 + 152, H149″455, T2f2, X2ap). In the context of a patriarchal and kinship-oriented society such as that of the Deep Maniots, the integration of a limited number of foreign individuals may have been facilitated primarily through women. Overall, our study demonstrates that Deep Maniots overwhelmingly descend paternally from ancient Greeks and Eastern (Greek-speaking) Romans known today as the Byzantines. This truly remarkable phenomenon may stem from the unique sociocultural dynamics of Deep Maniots and the geography of the Mani Peninsula, which led them to isolate themselves from different waves of migration, settlement and integration taking place in the surrounding regions. As a result, this hardy group represents a snapshot of the genetic landscape of the Greek-speaking world prior to the demographic turmoil of the Migration Period. While we anticipate that the quest for the origins of Deep Maniots will undoubtedly continue, our work provides a fundamental framework that can inform the interpretation of archaeological, historical, anthropological and linguistic history of the Greek-speaking world and southeastern Europe more broadly. We recruited Deep Maniot participants through two complementary approaches: We collected saliva samples from 75 Deep Maniot volunteers, each with uninterrupted patrilineal ancestry from villages across the entire geographic range of Deep Mani. Of these, 68 individuals underwent next-generation targeted enrichment sequencing, while 7 were genotyped for Y-STRs (using 37 and 111 loci). Additionally, 50 participants also reported matrilineal descent from Deep Mani, enabling analysis of their mtDNA. We focused on individuals originating from settlements that have been continuously inhabited for at least the past 500 years. Given that traditionally, Deep Maniot villages were settled by a single or a handful of clans11,15, we ensured sampling of genealogically unrelated clans per settlement wherever possible, as well as clan-less families, if these were present. As a result, our sampling strategy provides a comprehensive and representative dataset of the Deep Maniot genetic landscape. Given that many settlements in Deep Mani no longer support a permanent population and due to the large-scale emigration of Maniots in the 20th century, more than half of the study's participants comprised members of the Deep Maniot diaspora. Our study follows a community-based participatory research (CBPR) approach incorporating research, reflection, and action in a cyclical process between researchers and participants. The Deep Maniot community plays a central role in our research, with participants engaged in every aspect of the research process. Prior to their inclusion in the study, all newly sequenced participants were informed in non-scientific language about the study's aims and provided informed consent for use of their data for the purposes of strict scientific enquiry. We highlighted that participation in this study is completely optional. Individuals would not receive any financial or material rewards for their involvement and they were free to opt out if they wished to do so. Some participants requested that their clan or subclan name be included in the study, a few opted for it to be anonymised, while others did not want to provide their clan's name. We note that any clan name similarities with non-Maniot Greek surnames are the result of synonymy and does not correspond to any genealogical or genetic relationship with our study's participants. Together with the DNA samples, we also collected personal data from newly sequenced individuals solely to facilitate communication with each volunteer, to inform them about their results (in case they opted to receive them), and to involve them in the study's design. To this end, we designed and implemented a robust data management plan that complied with the General Data Protection Regulation (GDPR), as well as FamilyTreeDNA's strict policies that ensure tester privacy and security107. In this way, the results of our research were communicated to the participants prior to submission of the study. This approach allowed us to engage in discussions with the participants, which informed the research questions addressed in this work. Furthermore, oral traditions of descent, kinship, and migration of each participant's patriline and matriline were shared with us during informal discussions with the relevant community members and were combined with relevant data from the literature5,11,15,16,108. A certificate with a detailed explanation of the results of their personal Y-DNA and mtDNA analysis was provided to participants who requested it. These participants were given the contact details of our team should they have further questions regarding the interpretation of their results. We maintain ongoing communication with clan members to ensure that the local community is fully informed about all the scientific articles we publish. An accessible booklet summarising the results of this study will be provided free of charge in Deep Mani. Two of the Deep Maniot volunteers also opted for the Family Finder autosomal genealogical service109, which uses long (>6 cM) IBD segments shared with individuals in FamilyTreeDNA's consumer database to find genealogical connections from across the world, and to provide preliminary uniparental haplogroup determinations for more than 673,000 SNP-tested users. By querying FamilyTreeDNA's private autosomal and STR datasets, we were able to report Y-chromosome haplogroup frequencies for 27 additional individuals with confirmed Deep Maniot origin, bringing the total number of analysed Deep Maniots to 102 (Supplementary Data 1). Among these, 17 had undergone autosomal testing, 7 were genotyped using Y-STR panels (ranging from 12 to 111 markers), and 3 had performed targeted enrichment sequencing and, after providing consent, shared their terminal subclade. We also report haplogroup frequencies for 13 Outer Maniots, an adjacent population with IBD links to Deep Maniots31, and two Peloponnesians belonging to an E-V13 subclade shared with Outer Mani. This work contributes to the East Mediterranean Population Isolates Study (EMPIS), a collaborative effort to characterise the genetic history of culturally and historically distinct communities across the Eastern Mediterranean, approved by the European University Cyprus Bioethics Committee. Our study on Deep Mani was additionally reviewed and approved by the Bioethics Committee of the School of Sciences, European University Cyprus (Reference: 20250213, 2025–24), the Secretariat of the University of Oxford Medical Sciences Interdivisional Research Ethics Committee (Reference: R92782/RE001), the Faculty of Nursing, National and Kapodistrian University of Athens (Reference: 159263), and the Director of the Areopolis Health Centre. The research and experimental protocols were undertaken in accordance with the principles stated in the International Declaration of Helsinki for the protection of human subjects, the ethical standards of the European University Cyprus, the University of Oxford, the National and Kapodistrian University of Athens, as well as the relevant GDPR laws of the Hellenic Republic. All ethical regulations relevant to human research participants were followed. For the purpose of phylogenetic analysis, the study included 71 high-coverage, whole Y chromosome sequences that had not been previously documented in academic literature. All participants were informed about the study's goals and provided informed consent for their data to be utilised in scientific research. Their sequences were obtained using the Illumina NovaSeq 6000 platform, with Y-chromosome capture performed through a proprietary protocol developed by Gene by Gene (FamilyTreeDNA) using their commercially available Big Y-700 service59. This service's targeted enrichment design employs 155,000 capture probes to sequence >15 Mbp of the Y chromosome at coverage levels 35–105× in depth, depending on sample quality51,110,111. This test offers complete Y-chromosome haplogroup resolution, capturing 700 STRs and 750,000 tree SNPs60,111. Alignment was based on human reference genome GRCh38112,113. The sequencing procedures follow Begg et al.110. Haplogroup assignment for aDNA samples was completed by identifying known branch-defining variants, while less weight was given to non-private mutations identified as highly recurrent variants, and to variants occurring in problematic Y chromosome regions (e.g. the centromere, DYZ19 repeat, and Yq12 heterochromatic region)114. The 71 BigY-700 samples were then incorporated in FamilyTreeDNA's human Y-chromosome phylogeny95, which is generated through automated shared variant detection and manual curation110 and employs the Y Chromosome Consortium haplogroup nomenclature115. The method of phylogenetic tree reconstruction and TMRCA estimation are described in Begg et al.51,110 and Palencia-Madrid et al.51. FamilyTreeDNA's phylogeny includes >145,000 present-day user records at high sequence resolution59,111, as well as every publicly available NGS-sequenced aDNA male sample of sufficient coverage. This approach allowed us to compare Deep Maniot Y-chromosomes with populations from across the globe, both past and present. These insights informed our understanding of the phylogeography and putative ethnolinguistic groups who may have contributed into the Y-chromosome ancestry of present-day Deep Maniots. To identify individuals sharing identical Y-STR haplotypes within the Deep Maniot population, multi-locus haplotypes were constructed by concatenating allele values at the 111- and 17-STR level (the most commonly used STR panel in Y-DNA studies, see haplotype matching section), into a single string for each individual. These composite haplotypes were then tabulated to determine their frequency distribution. Haplotypes observed in more than one individual were classified as shared haplotypes, indicating shared paternal ancestry. To quantify the genetic diversity within the Deep Maniot population, we calculated Nei's haplotype diversity (H)116 at two levels of Y-STR resolution: using 111-locus haplotypes (n = 69) and 17-locus haplotypes (n = 75). Haplotype diversity provides a measure of the probability that two randomly selected haplotypes from the population are different and is particularly informative for assessing genetic variation in populations with uniparentally inherited markers such as the Y chromosome. Nei's haplotype diversity was manually computed in R using the total number of unique haplotypes and their frequency, as identified in the step above, based on the following formula: where n is the total number of individuals, pi is the relative frequency of the i-th haplotype, and k is the total number of unique haplotypes. The finite-sample correction factor n/(n – 1), corrects the downward bias caused by finite sampling, ensuring a more accurate estimation of diversity in small datasets, such as ours. Applying Nei's diversity index at both marker resolutions provides complementary insights into Deep Maniots' genetic structure, as the analysis at the 111-marker level provides a high-resolution overview of present diversity, while the 17-marker level allows for the investigation of deep shared ancestry (Supplementary Data 6). To investigate substructure within the predominant Deep Maniot Y-chromosome lineage, we applied NMDS to individual-level Y-STR haplotypes based on 111 loci, for haplogroup J-L930, which represents 51% of the Deep Maniot sample. Allelic data were in numeric format were transformed into a genind object using the adegenet package (v2.1.10)117, specifying haploid codominant markers. A pairwise dissimilarity matrix was computed using the diss.dist() function from the poppr package (v2.9.3)118, calculating multilocus genetic distances between individuals. NMDS was then applied using the metaMDS() function from the vegan package (v2.6-4)119, with two dimensions (k = 2) and up to 200 random starts (trymax = 200) to ensure convergence. The stress value was used to evaluate the fit of the ordination, with values below 0.1 considered indicative of a good representation of the data in two dimensions. NMDS coordinates were extracted and visualised using ggplot2 (v3.4.4)120. Our comparative Y-STR dataset comprised 12,722 individuals from 56 populations, including a focal sample of 74 individuals from Deep Mani, Greece. and the publicly available samples of the Armenian FamilyTreeDNA project (https://www.familytreedna.com/groups/armeniadnaproject/about). To compare FamilyTreeDNA-genotyped individuals with the 56 focal populations, we applied the following conversion: GATA H4.1 + 1. We selected 17 Y-STR markers from our Deep Maniot dataset to ensure broad comparability with our large-scale reference panel of 12,647 previously published individuals, the vast majority of which were genotyped using the same marker set. To ensure consistency, all allele values were first coerced into character strings prior to further analysis. To determine shared haplotypes (i.e. identical multilocus genotypes), the allele data were converted into a genind object using the ‘df2genind' function from the adegenet package in R117, with the ploidy parameter set to 1 to reflect the haploid state of Y-chromosome markers. Multilocus genotype (MLG) groups were then identified with the ‘mlg.id' function from the poppr R package118. Groups with more than one individual were considered as instances of shared (exact) haplotypes. The number of unique haplotypes as well as the distribution of group sizes were summarised, and the shared haplotype groups were exported for further evaluation and manual extraction of Deep Maniot-specific matches. To capture near-exact haplotype matches, that is haplotypes that differ by only one repeat unit at a single STR locus, the allele data were also converted into a numeric matrix. Pairwise Manhattan distances were computed across all individuals using the ‘dist' function in R. Haplotypes exhibiting a Manhattan distance of 1 were flagged as ‘near-identical', implying that they diverged by exactly one repeat at a single STR locus. For each pair, the locus of variation and the magnitude of the repeat difference were recorded (Supplementary Data 6). These near-miss pairs were compiled into a table and matches corresponding to the Deep Maniot haplotypes were manually isolated (Supplementary Data 6). We ran an Analysis of Molecular Variance (AMOVA) implemented in the pegas package (v1.3-1)141 in R (v4.3.2), on 17-locus Y-STR haplotypes (DYS19, DYS389I, DYS389II, DYS390, DYS391, DYS392, DYS393, DYS385a, DYS385b, DYS438, DYS439, DYS437, DYS448, DYS456, DYS458, DYS635, GATA H4), estimating pairwise RST distances between populations included in our comparative dataset (n populations = 56, n individuals = 12.720), accounting for the stepwise mutation model characteristic of microsatellite loci142 (Supplementary Data 5). Allele data were first converted to a numeric matrix representing raw repeat counts at each STR locus. Individuals were grouped by population using a factor vector. For each pair of populations, we extracted the relevant subset of individuals and computed a Euclidean distance on the raw repeat count data using the dist() function, without squaring the differences to retain the stepwise mutation model. We then applied AMOVA with the model specified as dmat∼pop, where ‘dmat' represents the matrix of Euclidean distances and ‘pop' the grouping variable. We extracted variance components with permutation testing (nperm = 1000) to generate p values for statistical significance, on whether the observed variance components (specifically the between-population variance) differ significantly from those expected under the null hypothesis of no genetic structure. From the AMOVA output, the variance components between populations (σbetween) and within populations (σwithin) were extracted and pairwise RST distances were calculated using the formula: where the ratio quantifies the proportion of the total genetic variance due to differences between populations. In cases where total variance was zero or incomputable, RST was recorded as missing. The computed RST distance for each population pair was assigned symmetrically in a distance matrix for visualisation and interpretation (Fig. The fact that 51% of Deep Maniots belong to a specific Y-haplogroup subclade (J-L930), renders this population an extreme outlier in the broader comparative dataset (Fig. Despite this remarkable founder effect, Deep Maniots display high within-population variance, likely resulting from high mutation rate of Y-STRs leading to high haplotype diversity. While AMOVA-derived RST can offer valuable insights on population differentiation under a stepwise mutation framework, the inherent variability and potential for inflated within-population variance in extreme outlier populations like the Deep Maniots can lead to potential bias and limit its interpretability. In particular, inflated within-group variance can dilute the between-population signal, potentially leading to biased RST values. Moreover, the assumption of the strict stepwise mutation model underlying RST estimation might not fully capture complexities arising in such extreme cases. To assess genetic differentiation among populations, we employed an alternative approach, where we estimated pairwise genetic distances based on allele frequency data using the adegenet package (v2.1.10)117 in R (v4.3.2). Individual-level Y-STR data were first converted into a genind object using the df2genind() function, specifying haploid codominant markers and assigning individuals to populations via a population vector. The derived genind object was then transformed into a genpop object using function genind2genpop(), aggregating allele frequencies at the population level. Pairwise genetic distances between populations were computed using the dist.genpop() function, setting method to Rogers' genetic distance (option 4)76,143, providing a Euclidean distance metric based on allele frequencies defined as: where D4((a, b) is the genetic distance between two populations, m is the number of loci, and pkaj and pkbj are the allele frequencies at locus j in populations a and b, respectively. This approach provides a robust approximation of genetic differentiation suitable for multivariate analyses such as MDS. The resulting distance matrix was converted to a full symmetric matrix for downstream visualisation and interpretation. Rogers' distances, by focusing on aggregated allele frequency differences, potentially provide a more robust and reliable measure of genetic divergence compared to RST distances, in this specific context. In particular, by summarising the overall allele frequency profile, this approach is less sensitive to the high variance within a dominant haplogroup and might provide a more robust estimation of genetic differentiation that reflects the cumulative discrepancies in allele frequencies across all loci without heavily penalising extreme within-population variability. This stability becomes especially advantageous when contrasting populations with varying internal diversities and underlying genetic structures, particularly in the presence of extreme outlier populations. This is particularly relevant in our case, given the Deep Maniot population's outlier status. Given this we opted for the use of Rogers' distances as our primary metric for inter-population comparisons. To visualise patterns of genetic similarity among populations, we applied NMDS analysis to the pairwise genetic distance matrix derived from Rogers' distance (see above), using the metaMDS() function from the vegan package (v2.6-4) in R (v4.3.2)119. A two-dimensional solution (k = 2) was specified and we allowed up to 200 random starts (trymax = 200) to ensure convergence on a stable configuration. NMDS is a rank-based ordination method that represents the relative dissimilarities between populations in a reduced-dimensional space, preserving the order of distances rather than their absolute values. To enhance clarity in the ordination plot and minimise potential bias from populations with extremely high genetic distances or small sample sizes, specific populations (e.g. Georgian, Ossets_Iron, Kubachi, and Shapsug) were excluded from the final visualisation. The NMDS plot was generated using the ggplot2 package120, providing a two-dimensional depiction of the genetic landscape across populations. Lower stress values indicate a better representation of the data in reduced dimensions, with values below 0.1 generally considered indicative of a good fit. As an alternative to NMDS, we applied Uniform Manifold Approximation and Projection (UMAP) using the umap package (v0.2.10.0)144 in R (v4.3.2), to visualise genetic relationships among populations in a reduced-dimensional space. UMAP is a non-linear dimensionality reduction technique that preserves both local and global structure in high-dimensional data, making it well-suited for exploring complex genetic distance matrices, offering an alternative visualisation of genetic relationships. The analysis was applied directly to the pairwise genetic distance matrix derived from Rogers' distance using the umap() function with default configuration settings (umap.defaults) to maintain consistency with common usage standards and specifying the input type as ‘dist'. The resulting two-dimensional layout was extracted and used to construct a scatterplot, with each point representing a population and labelled accordingly. NMDS and UMAP represent different approaches of dimensionality reduction. UMAP is designed to preserve local neighbourhood structures. It models data as a high-dimensional graph and optimises a low-dimensional layout that retains connectivity within that graph, therefore clusters are easier to detect and interpret visually. However, UMAP usually does not preserve global distances or the overall shape of the original space, therefore, the spatial placement of outliers may understate their true genetic distances. NMDS optimises a configuration such that the rank order of pairwise distances is preserved. It prioritises getting the relative dissimilarity (i.e. which populations are genetically more distant from each other) correct, even at the expense of local spacing. As a result, genetically close populations may appear crowded in overlapping clusters, while outliers are usually accurately positioned as genetically distinct. Since Deep Maniots represent an outlier population, we opted for NMDS as the main method for the graphical representation of pairwise genetic distances. We should note that UMAP results in nearly identical plots with those generated by NMDS (Fig. We obtained off-target mtDNA data from BigY-700 testing for 50 of the 71 NGS-tested individuals who also descended from Deep Maniot matrilines, while one additional mtDNA sequence was obtained by querying the FamilyTreeDNA's database. Since these are off-target reads (compared to FamilyTreeDNA's targeted mtDNA test), we can expect a slightly higher risk of NUMT reads being included in the analysis. However, these occur at such a low frequency compared to mtDNA that they could at most result in a heteroplasmy call, which would be excluded from the phylogenetic analysis. We use a threshold of 20% allele depth difference from the majority call for a site to be classified as heteroplasmic. Additionally, FamilyTreeDNA employs an mtDNA analysis pipeline similar to the Broad Institute's best practices workflow for Mitochondrial short variant discovery, incorporating normalisation steps such as MergeBamAlignment --UNMAP_CONTAMINANT_READS to ensure data accuracy and integrity. The resulting FASTA files were then uploaded into Haplogrep 3145, which utilised PhyloTree build 17146 to determine mitochondrial haplogroups and identify local private mutations. The phylogeographic affinities of Deep Maniot mtDNA haplogroups were established by comparison against FamilyTreeDNA's mtFull database, which comprises >260,000 mitochondrial genomes, as well as all publicly available mtDNA sequences of the same macro-haplogroup on BLAST and GenBank. To explore the phylogenetic relationships between Maniot mitochondrial lineages and those of modern and ancient populations, we constructed median-joining networks (MJNs) for four major mitochondrial haplogroups observed in the Deep Maniot sample: HV119, H7c1k1, U1a1d1d, and U5a1b1. For modern samples, complete mitochondrial consensus sequences were retrieved directly from GenBank, using accession numbers reported in the literature (Supplementary Data 15). For ancient samples, consensus sequences were retrieved from GenBank and the Allen Ancient DNA Resource (AADR)147. In instances where ancient consensus sequences were not directly available, we obtained the corresponding raw sequence data (BAM files) from the European Nucleotide Archive (ENA). SAMtools (v1.18)113 was initially used to ensure that the BAM file is sorted and indexed and the consensus sequences were then reconstructed using ANGSD (v0.931)148 following a pipeline suitable for lower coverage or fragmented aDNA data, using options -doFasta 2 -doCounts 1 (chooses the most common base at each locus) -minQ 5-25 (sets the minimum base quality discarding bases with quality scores below a given threshold—increments of 5 in this case) -trim 5 (trims 5 bases at read termini to account for the fact that nucleotide misincorporations due to damage are common in aDNA)149. Following the compilation of consensus sequences for each haplogroup, we aligned sequences using MEGA150 and constructed median-joining networks (MJNs) using PopArt v1.7151, for each haplogroup of interest. Sample nodes were annotated by population or geographic origin, using population metadata. These networks were used to assess the degree of haplotype sharing between Maniots and other populations, and to visualise potential phylogeographic structure within each haplogroup. One of the Deep Maniot lineages has been found in an ancient individual (I8216) from Roman Era Empuries in present-day Catalonia, Spain. This individual was previously modelled as having Aegean ancestry152. To model the autosomal ancestry of I8216, we employed qpAdm from ADMIXTOOLS v.2.0.0153 by using a base set of references approach154. A model was accepted as statistically plausible if its p value was ≥0.01, as followed by previous work. Z-scores were automatically estimated by our qpAdm script, and only values to close to or >Z = 3 are reported. An extensive description of our qpAdm models and the rationale behind the chosen source and reference populations can be found in the Supplementary Data. Following consultation with the Deep Maniot community, and in compliance with FamilyTreeDNAs privacy terms, specific sequencing data (BAM, VCF and FASTA files) are restricted and will not be deposited in a public repository. Applications for strict scientific use of the data can be made by a joint request to L.R.D., A.D.M., and A.H. Access will be granted upon consultation and approval by each individual participant who volunteered in this study, or their close family members, in cases where the volunteer is deceased. Pitts, M. & Versluys, M. J. Globalisation and the Roman World: World History, Connectivity and Material Culture (Cambridge University Press, 2014). in Migration Histories of the Medieval Afroeurasian Transition: Aspects of Mobility Between Africa, Asia and Europe, 300-1500 (eds Preiser-Kapeller, J., Reinfandt, L. & Stouraitis, Y.) Olalde, I. et al. A genetic history of the Balkans from Roman frontier to Slavic migrations. Malingoudis, P. Slavs in Medieval Greece (Kyriakidi Publishing, 2013). Le Péloponnèse Du IVe Au VIIIe Siècle. Byzantina Sorbonensia, Éditions de la Sorbonne Vol. The textual sources for the Peloponnese, A.D.582–959: their creative engagement with ancient literature. Charanis, P. The Chronicle of Monemvasia and the Slavonic settlements in Greece. Lambropoulou, Anna, Anagnostakis, A., Konti, V., Panopoulou, A. Contribution to the interpretation of archaeological evidence of the Peloponnese during the Dark Centuries. in The Dark Centuries of Byzantium (7th-9th c.) (ed. Kountoura-Galake, E.) 189–229 (National Hellenic Research Foundation, Institute for Byzantine Research, 2001). Miller, W. The Latins in the Levant: A History of Frankish Greece (1204-1566). Alexakis, E. P. The clans and family in the traditional society of Mani. Komis, K. Population and Settlements of Mani. Le Magne byzantin: problèmes d'histoire et de topographie. Mélanges offerts à Hélène Ahrweiler 49–62 (Université Paris 1 Panthéon-Sorbonne, 1998). Kassis, K. Genealogy or the Clans of Deep Mani (Self Published, 1978). Kassis, K. Lament Songs of Deep Mani. Unpublished Texts and the Unknown Histories of Lament Songs and Families (Self Published, 1979). Gardner, C. The origins and evolution of ancient Spartan identity in the Mani Peninsula, Greece. Greek Text Edited by Gy. Moravcsik, English Translation by R. J. H. Jenkins. Revue des Études Grecques (Dumbarton Oaks, 1967). Gardner, C. A. M. The Mani Peninsula in Antiquity: An Archaeological, Historical, and Epigraphic Investigation into Regional Identity. The University of British Columbia, Vancouver, (2018). Seifried, R. M. & Gardner, C. A. M. Maintaining Distinction: Local Identity in the Remote Mani Peninsula, Greece, in the Classical, Byzantine, and Ottoman Periods. In Joint Annual Meeting of the Archaeological Institute of America and the Society for Classical Studies, 1–14 (New Orleans, Los Angeles, 2015). History of the Wars. Saitas, Y. Megalithic in the Mani. A Contribution to the Settlement and Building Documentation. Seifried, R. M. The Stone-Built Palaiomaniatika of the Mani Peninsula, Greece. in Deserted villages: perspectives from the eastern Mediterranean (eds Seifried, R. M. & Brown Stewart, D. E.) 153–205 (The Digital Press at the University of North Dakota, 2021). The Castles of Maina. A Historical-Geographical and Archaeological Study. Drandakis, N., Gioles, N. & Konstandinidi, X. Excavation in Tigani of Mani. Mexia, A. Byzantine temple construction in the Peloponnese. (National and Kapodistrian University of Athens, 2011). Building Practices in the Helladic province of Mani during the Komnenian period: The assimilation of the prevailing trends in church architecture. in Against Gravity: Building Practices in the Pre-Industrial World. (eds Ousterhout, R., Borbonus, D. & Dumser, E.) 1–21 (University of Pensylvania, 2016). Seifried, R. M. The Legacy of Byzantine Christianity in the Southern Mani Peninsula, Greece, after Imperial Collapse. in Rituals, Collapse, and Radical Transformation in Archaic States 56–76. The Basilica at Tigani in Mesa Mani (Laconia). in Dedication to the academic Panagiotis L. Vokotopoulos. Kaldellis, A. Romanland: Ethnicity and Empire in Byzantium. Stamatoyannopoulos, G. et al. Genetics of the Peloponnesean populations and the theory of extinction of the medieval Peloponnesean Greeks. Raveane, A. et al. Assessing temporal and geographic contacts across the Adriatic Sea through the analysis of genome-wide data from Southern Italy. Ancient genomics support deep divergence between Eastern and Western Mediterranean Indo-European languages. Lazaridis, I. et al. Genetic origins of the Minoans and Mycenaeans. Ancient DNA reveals admixture history and endogamy in the prehistoric Aegean. The genetic history of the Southern Arc: a bridge between West Asia and Europe. Ancient Rome: a genetic crossroads of Europe and the Mediterranean. Stable population structure in Europe since the Iron Age, despite high mobility. Lazaridis, I. et al. A genetic probe into the ancient and medieval history of Southern Europe and West Asia. Pantelidis, N. A southern Greek dialectal zone? Pantelidis, N. Aspects of the history of Ancient Greek/u/ /u:/ (Υ) and /oi/ (ΟΙ): Evidence from Medieval and Modern Greek. in Maiores philologiae pontes. Festschrift für Michael Meier-Brügger zum 70 Geburtstag. (eds Fritz, M., Kitazumi, K. & Veksina, M.) 187–201 (Beech Stave Press, 2020). Pantelidis, N. Contribution to the linguistic history of southern Greece: Aegina. Honorary tome for Christina Basea-Bezantakou (eds Karla, G., Manolessou, I. Pantelidis, N. New evidence for the dialect of old Athens. Pantelidis, N. The Modern Greek variety of old Athens. The Generative Interpretation of Dialect: A Study of Modern Greek Phonology (Cambridge University Press, 1972). García-Fernández, C. et al. Y-chromosome target enrichment reveals rapid expansion of haplogroup R1b-DF27 in Iberia during the Bronze Age transition. Analysis of the human Y-chromosome haplogroup Q characterizes ancient population movements in Eurasia and the Americas. Pathak, A. K. et al. Human Y chromosome haplogroup L1-M22 traces Neolithic expansion in West Asia and supports the Elamite and Dravidian connection. Origin and diffusion of human Y chromosome haplogroup J1-M267. Lall, G. M. et al. Subdividing Y-chromosome haplogroup R1a1 reveals Norse Viking dispersal lineages in Britain. Luis, J. R., Palencia-Madrid, L., Runfeldt, G., Garcia-Bertrand, R. & Herrera, R. J. Delineating the dispersal of Y-chromosome sub-haplogroup O2a2b-P164 among Austronesian-speaking populations. Maier, P. A., Runfeldt, G., Estes, R. J. & Vilar, M. G. African mitochondrial haplogroup L7: a 100,000-year-old maternal human lineage discovered through reassessment and new sequencing. Kristjansson, D., Bohlin, J., Nguyen, T. T., Jugessur, A. & Schurr, T. G. Evolution and dispersal of mitochondrial DNA haplogroup U5 in Northern Europe: insights from an unsupervised learning approach to phylogeography. Platt, D. E. et al. Autosomal genetics and Y-chromosome haplogroup L1b-M317 reveal Mount Lebanon Maronites as a persistently non-emigrating population. The influence of clan structure on the genetic variation in a single Ghanaian village. The genetic or mythical ancestry of descent groups: lessons from the Y chromosome. Zhabagin, M. et al. Y-Chromosomal insights into the paternal genealogy of the Kerey tribe have called into question their descent from the Stepfather of Genghis Khan. Analysis of clan structure of Tuvans by Y-chromosome markers. Davis, C. et al. Big Y-700 White Paper: Powering Discovery in the Field of Paternal Ancestry. FamilyTreeDNA's Y-DNA Haplotree: 90,000 Branches and Counting. Parallel evolution of genes and languages in the caucasus region. Differential Y-chromosome Anatolian Influences on the Greek and Cretan Neolithic. El-Sibai, M. et al. Geographical structure of the Y-chromosomal genetic landscape of the levant: a coastal-inland contrast. Voskarides, K. et al. Y-chromosome phylogeographic analysis of the Greek-Cypriot population reveals elements consistent with Neolithic and Bronze Age settlements. Heraclides, A. et al. Y-chromosomal analysis of Greek Cypriots reveals a primarily common pre-Ottoman paternal ancestry with Turkish Cypriots. Y-Chr Sequence Interpretation Service. Allentoft, M. E. et al. Population genomics of post-glacial western Eurasia. Early Neolithic genomes from the eastern Fertile Crescent. McColl, H. et al. Steppe Ancestry in Western Eurasia and the Spread of the Germanic Languages. Zalloua, P. A. et al. Y-chromosomal diversity in Lebanon is structured by recent historical events. Haber, M. et al. A transient pulse of genetic admixture from the crusaders in the Near East identified from ancient genome sequences. Davranoglou, L.-R., Aristodemou, A., Wesolowski, D. & Heraclides, A. The Greeks in the West: genetic signatures of the Hellenic colonisation in southern Italy and Sicily. Psonis, N. et al. Genetic affinities between an ancient Greek colony and its metropolis: the case of Amvrakia in western Greece. 145–153 (University of Texas Publication 713, 1972). Ilumäe, A.-M. et al. Human Y chromosome haplogroup N: a non-trivial time-resolved phylogeography that cuts across language families. Founders, drift, and infidelity: the relationship between Y chromosome diversity and patrilineal surnames. The diverse genetic origins of a classical period Greek army. Ringbauer, H. et al. Punic people were genetically diverse with almost no Levantine ancestors. Early contact between late farming and pastoralist societies in southeastern Europe. Freilich, S. et al. Reconstructing genetic histories and social organisation in Neolithic and Bronze Age Croatia. The genomic history of southeastern Europe. Gelabert, P. et al. Social and genetic diversity in first farmers of central Europe. Massive migration from the steppe was a source for Indo-European languages in Europe. Domínguez, A. J. Hellenic Identity and Greek Colonisation. in Ancient West & East Vol. Ancient human genome-wide data from a 3000-year interval in the Caucasus corresponds with eco-geographic regions. Adamov, D., Guryanov, V., Karzhavin, S., Tagankin, V. & Urasin, V. Defining a new rate constant for Y-chromosome SNPs based on full sequencing data. Later Travels (Harvard University Press, 2003). & Coop, G. The contribution of gene flow, selection, and genetic drift to five thousand years of human allele frequency change. & Latch, E. K. Nonrandom missing data can bias principal component analysis inference of population genetic structure. Alberto-Barroso, V. et al. Perinatal burials at pre-Hispanic noncemetery sites in Gran Canaria: Tophet, infanticide, or natural mortality? The history of the North African mitochondrial DNA haplogroup U6 gene flow into the African, Eurasian and American continents. The origin of modern North Africans as depicted by a massive survey of mitogenomes. Ena, G. F., Aizpurua-Iraola, J., Font-Porterias, N., Calafell, F. & Comas, D. Population genetics of the European Roma—a review. Malingoudis, P. Toponymy and history. Elsie, R. The Tribes of Albania: History, Society and Culture. Whitaker, I. Tribal structure and national politics in Albania, 1910–1950. in History and Social Anthropology (ed. Kirchhoff, P. The principles of clanship in human society. Katsafados, P. St. Re-Imagining the Mani: Elegy to a Rural Population (11th to 17th c.). The Megali Mani in the Ottoman Tapu Tahrir Defterleri The TD80 with Notes from the TD367, TD390, TD565, TD603 and TD677 (in Addendum) (Paragraphos, 2025). Using Y-chromosome capture enrichment to resolve haplogroup H2 shows new evidence for a two-path Neolithic expansion to Western Europe. Mendizabal, I. et al. Reconstructing the population history of European Romani from genome-wide data. Dimitrakos-Mesisklis, D. The Niklianoi (Dimitriou Dimitrakou, 1949). Hu, R. et al. Family Finder Matching 5.0 Matching Algorithm and Relationship Estimation. A. et al. Genomic analyses of hair from Ludwig van Beethoven. Cavagnino, C. et al. Unearthing who and Y at Harewood Cemetery and inference of George Washington's Y-chromosomal haplotype. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. The sequence alignment/map format and SAMtools. Krzewińska, M. et al. Related in death? Further insights on the curious case of Bishop Peder Winstrup and his Grandchild's Burial. Consortium, Y. C. A nomenclature system for the tree of human Y-chromosomal binary haplogroups. Nei, M. Molecular Evolutionary Genetics (Columbia University Press, 1987). Jombart, T. adegenet: a R package for the multivariate analysis of genetic markers. Kamvar, Z. N., Tabima, J. F. & Grünwald, N. J. Poppr: an R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction. Oksanen, J. et al. vegan: community ecology package. CRAN: Contributed Packages Preprint at https://doi.org/10.32614/CRAN.package.vegan (2025). Aboukhalid, R. et al. Haplotype frequencies for 17 Y-STR loci (AmpFlSTR®Y-filerTM) in a Moroccan population sample. Alakoc, Y. D. et al. Y-chromosome and autosomal STR diversity in four proximate settlements in Central Anatolia. Albarzinji, B. M., Abdulkarim, F. M., Hussein, S. A., Rashid, D. & Lazim, H. Population genetic study of 17 Y-STR Loci of the Sorani Kurds in the Province of Sulaymaniyah, Iraq. Brisighelli, F. et al. Uniparental markers of contemporary Italian population reveals details on its pre-Roman heritage. Dogan, S. et al. A glimpse at the intricate mosaic of ethnicities from Mesopotamia: paternal lineages of the Northern Iraqi Arabs, Kurds, Syriacs, Turkmens and Yazidis. Eskandarion, M. R. et al. Haplotype diversity of 17 Y-STR in the Iranian population. Ferri, G., Alù, M., Corradini, B., Radheshi, E. & Beduschi, G. Slow and fast evolving markers typing in Modena males (North Italy). Fernandes, A. T. et al. Y-chromosomal STRs in two populations from Israel and the Palestinian Authority Area: Christian and Muslim Arabs. Jankova, R., Seidel, M., Videtič Paska, A., Willuweit, S. & Roewer, L. Y-chromosome diversity of the three major ethno-linguistic groups in the Republic of North Macedonia. Karachanak, S. et al. Y-chromosome diversity in modern Bulgarians: new clues about their ancestry. Kovatsi, L., Saunier, J. L. & Irwin, J. Population genetics of Y-chromosome STRs in a population of Northern Greeks. Laouina, A. et al. Allele frequencies and population data for 17 Y-STR loci (The AmpFlSTR® Y-filer™) in Casablanca resident population. Martı́n, P. et al. A Spanish population study of 17 Y-chromosome STR loci. Comparative Y-chromosome analysis among Cypriots in the context of historical events and migrations. Nogueiro, I., Manco, L., Gomes, V., Amorim, A. & Gusmão, L. Phylogeographic analysis of paternal lineages in NE Portuguese Jewish communities. Purps, J. et al. A global analysis of Y-chromosomal haplotype diversity for 23 STR loci. Robledo, R., Piras, I., Beggs, W. & Calò, C. Analysis of 31 STR Loci in the Genetic Isolate of Carloforte (Sardinia, Italy). Schurr, T. G. et al. Genetic analysis of Mingrelians reveals long-term continuity of populations in Western Georgia (Caucasus). Stanciu, F. et al. Population data for Y-chromosome haplotypes defined by 17 STRs in South-East Romania. Valverde, L. et al. Y-STR variation in the Basque diaspora in the Western USA: evolutionary and forensic perspectives. Paradis, E. pegas: an R package for population genetics with an integrated–modular approach. Excoffier, L., Smouse, P. E. & Quattro, J. M. Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data. Avise, J. C. Molecular Markers, Natural History and Evolution (Chapman & Hall, 1994). Konopka, T. umap: Uniform Manifold Approximation and Projection. CRAN: Contributed Packages Preprint at https://doi.org/10.32614/CRAN.package.umap (2018). Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. van Oven, M. PhyloTree Build 17: growing the human mitochondrial DNA tree. The Allen Ancient DNA Resource (AADR) a curated compendium of ancient human genomes. & Nielsen, R. ANGSD: analysis of next generation sequencing data. & Fernández-Domínguez, E. Mitochondrial DNA consensus calling and quality filtering for constructing ancient human mitogenomes: comparison of two widely applied methods. Tamura, K., Stecher, G. & Kumar, S. MEGA11: molecular evolutionary genetics analysis version 11. Leigh, J. W. & Bryant, D. POPART: full-feature software for haplotype network construction. Ancient admixture in human history. Harney, É., Patterson, N., Reich, D. & Wakeley, J. Assessing the performance of qpAdm: a statistical tool for studying population admixture. World Terrain Base Map. Shorthouse, D. P. SimpleMappr, an online tool to produce publication-quality point maps. We would like to wholeheartedly thank the community of Deep Mani for enthusiastically participating in our research, for sharing their oral traditions with us, and for continued feedback on key aspects of our research. This study is dedicated to all Deep Maniots, in Greece and beyond. We are indebted to the Mayor of East Mani, Petros N. Andreakos, for actively fostering meaningful engagement with the local community and for hosting L.R.D during fieldwork in Deep Mani. We are particularly grateful to Deep Maniot sculptor Michalis Kassis and his brothers Vassilis and Kyriakos, for sharing their unparalleled understanding on Deep Maniot genealogy. We thank Lambros Baltsiotis (Panteion University, doctoral supervisor of L.R.D. ), Leonidas Embirikos, Dionysis Mertyris (Academy of Athens), Michalis Kappas (Ephorate of Antiquities of Messinia), Panayotis St. Katsafados and Giannis Saitas for invaluable discussions and insights on the history and demography of Deep Mani and the provision of important literature. We are truly grateful to Vinia Tsopelas and Barbara Stavrianakos for their unwavering support on key logistical aspects of our research. We thank Alexandros Spanos and Michalis Moree for fruitful discussions on Greek demography and genetics. Finally, we are grateful to three anonymous reviewers for their constructive comments on earlier versions of our study. This study was made possible by community-based funding, where Deep Maniots from Greece and abroad, as well as two donors from the UK, all of whom maintain their confidentially, financially supported this research by ordering targeted enrichment and STR kits. Our study was not supported by any public or commercial funding body. The funders had no role in the study's design and declared no competing interests. These authors contributed equally: Leonidas-Romanos Davranoglou, Athanasios Petros Kofinakos. The Steinhardt Museum, Tel Aviv University, Tel Aviv, Israel Faculty of Nursing, National and Kapodistrian University of Athens, Athens, Greece Leonidas-Romanos Davranoglou & Theodoros Mariolis-Sapsakos Independent Researcher, Piraeus, Greece Anargyros D. Mariolis & Panagiota Soulioti Göran Runfeldt, Paul Andrew Maier & Michael Sager Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Leonidas-Romanos Davranoglou or Alexandros Heraclides. All authors affirm that this research was undertaken without any commercial or financial connections that might present a conflict of interest. Communications Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editors: Pavel Flegontov and Tobias Goris. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Davranoglou, LR., Kofinakos, A.P., Mariolis, A.D. et al. Uniparental analysis of Deep Maniot Greeks reveals genetic continuity from the pre-Medieval era. Version of record: 04 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260203030537.htm'>Why heart disease risk in type 2 diabetes looks different for men and women</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-04 07:22:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Women and men with diabetes face different levels of heart disease risk, but the biological reasons behind those differences are still not well understood. A new study led by Johns Hopkins Medicine takes a closer look at whether sex hormones, including testosterone and estradiol, may help explain why these risks vary. "We are very interested in understanding why women who have diabetes have a greater risk for heart disease compared to men," says lead researcher Wendy Bennett, M.D., M.P.H., an associate professor of medicine at Johns Hopkins University School of Medicine. To conduct the study, researchers analyzed data from the Look Ahead study, a long-term project that examined how weight loss affected heart health in people with type 2 diabetes. These samples were taken at the start of the study and again one year after enrollment, giving researchers insight into how hormone levels changed and whether those changes were linked to future heart disease risk. "We were able to see whether the changes in hormones affected their heart disease risk," Bennett says. "We saw that there were differences in the male participants. Among female participants, however, researchers did not observe clear connections between hormone levels and cardiovascular outcomes. This suggests that hormones may influence heart disease risk differently depending on sex, or that other biological and clinical factors may play a larger role for women with diabetes. "Results from this study contribute to our understanding of how tracking sex hormones in people with diabetes could complement what we already know about traditional heart disease risk factors [like smoking and cholesterol levels]," Bennett says. These include studying how weight loss and hormone changes affect bone health, as well as identifying which patients may be at higher risk for fractures and why. Researchers are also preparing new studies focused on hormone declines during the menopausal transition, also known as perimenopause, and how those hormonal changes may influence cardiovascular risk, particularly in people with chronic conditions such as diabetes. The study's coauthors include Teresa Gisinger, M.D., Ph.D., Jiahuan Helen He, M.H.S., Chigolum Oyeka, MBBS, M.P.H., Jianqiao Ma, ScM, Nityasree Srialluri, M.D., M.S., M.H.S., Mark Woodward, Ph.D., Erin D. Michos, M.D., M.H.S., Rita R. Kalyani, M.D., M.H.S., Jeanne M. Clark, M.D., M.P.H., Alexandra Kautzky-Willer, M.D., and Dhananjay Vaidya, MBBS, Ph.D., M.P.H. Clark reports serving as a scientific advisor to Boehringer Ingelheim and receiving writing support from Novo Nordisk in the last three years. Unrelated to this research, Michos has served as a consultant for Amgen, Arrowhead, AstraZeneca, Bayer, Boehringer Ingelheim, Edwards Life Science, Esperion, Ionis, Eli Lilly, Medtronic, Merck, New Amsterdam, Novartis, Novo Nordisk, and Zoll. This One Body Measurement Could Reveal Heart Disease Risk Years Before Symptoms Appear Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            