
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-20</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65784174/imagination-hyperphantasia-aphantasia/'>This Hidden Human ‘Sense' Is Quietly Shaping Your Reality, Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 16:13:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Researchers are trying to understand how people perceive the world in wildly different ways. Apples seem to have become the internet's symbol for imagination—or at least, mental imagery. Every so often, someone takes to X or Reddit and asks what other users see when they imagine an apple. But what nobody seems to appreciate is the assumption that all responders are picturing an apple in their “mind's eye.” In fact, about 1 to 3 percent of people experience an extreme form of “imagery vividness.” It could mean conjuring an image so vivid that it seems to have projected an apple into real life—or, on the opposite end of the spectrum, thinking about an apple in totally non-visual terms. When we see something, our brain starts with the object's basic, concrete aspects, Bainbridge says. In that sense, the brain climbs the ladder of abstraction, assembling more basic concepts before reaching more complex levels that involve meaning. A 2019 paper published in the journal Nature Reviews Neuroscience describes this process as “reverse visual hierarchy.” The apple test also introduces a spectrum of visual imagery, which shows two conditions that fall at either end: aphantasia and hyperphantasia. Those with aphantasia don't see anything in their mind's eye, while those with hyperphantasia imagine elaborately detailed objects and scenarios that seem almost lifelike. Yet, perception far extends beyond sight, pulling together other types of sensory information, like hearing and smell. Understanding the differences in brains of those with atypical imaginative capabilities, like aphantasia and hyperphantasia, can help provide greater insight into how humans create mental pictures. She and her team used functional magnetic resonance imaging (fMRI) to measure blood flow in the brains of a pair of identical twins as they performed mental imagery tasks. In other words, while people with aphantasia don't see images in their mind's eye, they can still imagine spatial relationships between objects. The aphantasic twin accurately reproduced these scenes in both cases, but with “starkly less detail—including no color” than her sister, according to the paper. A potential explanation for aphantasia could come from less functional connectivity between the area of the brain that deals with spatial and color processing and the area that deals with memory storage, Bainbridge and her co-authors suggest. However, aphantasic brains do have increased connectivity among non-visual areas, the paper says. Research has found that people with aphantasia are no less creative or imaginative than those who can picture an apple—just not through vivid imagery. They may recall taste, smell, and sound ahead of sight, or abstractly know an object's shape and color without seeing the object, as one 2021 paper describes. A 2024 paper does stipulate that about 40 percent of people on the aphantasia spectrum report difficulty with recognizing faces, also known as prosopagnosia. This paper also reports that aphantasic people take longer to complete visual processing tasks. They also have more frequently reported synesthesia, or the condition of experiencing two senses at once—perhaps seeing colors spontaneously along with numbers or letters, for instance. People with hyperphantasia also tend to work in creative fields where object imagery is crucial, while people with aphantasia are more prevalent in mathematical and scientific fields where spatial imagery is necessary. Researchers think they know at least one good reason for our powers of imagination, no matter how they manifest for each of us. Prediction is important to anticipating what stimuli we might experience so we'll know how to respond, says Shimon Edelman, Ph.D., a professor of psychology at Cornell University. Yet, all the monsters, places, people, and scenarios we might imagine to be novel are actually simply composites of the vast amount of experiences we've processed during our lives, Bainbridge says. Elana Spivack is a science writer in New York City, where she lives with her tuxedo cat. A graduate of NYU SHERP, she has also written for Scientific American, Popular Science, Inverse, Slate, and others. Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09431-y'>Engineered yeast provides rare but essential pollen sterols for honeybees</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 16:01:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Honeybees are important crop pollinators, but they increasingly face pollen starvation as a result of agricultural intensification and climate change1. Frequent flowering dearth periods and high-density rearing conditions weaken colonies, which often leads to their demise2. Beekeepers provide colonies with pollen substitutes, but these feeds do not sustain brood production because they lack essential sterols found in pollen3,4. Here we describe a technological advance in honeybee nutrition with wide-reaching impacts on global food security. We first measured the quantity and proportion of sterols present in honeybee tissues. Using this information, we genetically engineered a strain of the oleaginous yeast Yarrowia lipolytica to produce a mixture of essential sterols for bees and incorporated this yeast strain into an otherwise nutritionally complete diet. Colonies exclusively fed with this diet reared brood for significantly longer than those fed diets without suitable sterols. The use of this method to incorporate sterol supplements into pollen substitutes will enable honeybee colonies to produce brood in the absence of floral pollen. Optimized diets created using this yeast strain could also reduce competition between bee species for access to natural floral resources and stem the decline in wild bee populations. Managed honeybees (Apis mellifera) are fundamental to modern agricultural systems and global food security because of the crucial pollination services they provide to crops. Honeybees consume floral pollen, which provides proteins, fats, carbohydrates and micronutrients5. Increasingly, a combination of changes in climate, land use and agricultural practices is limiting the access of honeybees to sufficient and diverse floral resources1,6,7. Nutritional deficiencies increase the susceptibility of bees to disease and colony collapse and contribute to the growing rate of honeybee colony losses2,8. Loss of pollinators in turn reduces crop yields and raises the costs of food production9,10. In the past 40 years, beekeepers have adopted the practice of feeding artificial pollen substitutes when natural forage is insufficient or when bees are kept at high densities. However, commercially available pollen substitutes composed of protein flour, sugars and oils are not nutritionally complete feeds for honeybees. Such diets are missing essential sterols found in floral pollen that are necessary for colony health and growth4,11. Sterols are a structurally diverse class of tetracyclic triterpenoids that are ubiquitously important for eukaryotic cell function, including for membrane architecture, hormone biosynthesis and signalling11,12,13. Unlike other animals, insects and marine invertebrates have lost the ability to synthesize endogenous sterols and instead acquire them from the diet12,13. Most insects produce cholesterol (CHOL) by dealkylating dietary sterols13,14. By contrast, honeybees have lost the ability to dealkylate sterols and instead directly use sterols obtained from the diet as membrane inserts and as precursors for ecdysteroid hormones15,16. Most of the pollen sterols used by bees are not available in quantities that could be fed to bee colonies on a commercial scale, which makes it unfeasible to create a nutritionally complete feed that is a substitute for pollen. Sterols are consumed by nurse-aged worker honeybees from stored pollen (bee bread). The sterols accumulate in their mandibular glands and are transferred to worker, queen and drone larvae through glandular secretions17 (Fig. Sterol diversity is high in pollen18, but only a few key sterols seem to be taken up by worker bees in the gut and transferred to the brood19 (Fig. To quantify the proportions of sterols found in honeybees, we analysed the sterol composition in pupal tissue from each pupal type. 1) constituted 60–70% of sterols in the pupae (Fig. Five other minor sterols were consistently present: β-sitosterol (SITO), desmosterol (DESMO), isofucosterol (ISOFUC), campesterol (CAMP) and CHOL (Fig. The relative abundance of each sterol found in pupae varied among pupal types (two-way quasi-binomial generalized linear model (GLM), sterol × pupal type, χ210 = 47.0, P < 0.0001; Fig. CAMP was almost six times more abundant in worker and queen pupae than drone pupae. a, Adult worker honeybees collect and eat floral pollen, which provides proteins, lipids and micronutrients. Pollen foraged from flowers is packed into cells of the wax comb with honey to form bee bread. Nurse bees consume bee bread and create glandular secretions (jelly), which include sterols required for development, to larvae. Larvae develop into pupae and eventually emerge as female workers, queens or drones (males). b, The relative abundance of each of the six key sterols found in pupae varies among pupal types (two-way quasi-binomial GLM, sterol × pupal type, χ210 = 47.0, P < 0.0001, n = 5 biologically independent samples). Error bars represent the mean ± s.d. c, Total sterol concentration per fresh weight (FW) varies significantly among pupal types (one-way Gaussian GLM, χ22 = 91.9, P < 0.0001, n = 5 biologically independent samples). Error bars represent the mean ± s.d. d, The relative abundance of selected sterols in pollen from bee-pollinated flowers (combined total (%), n = 41 biologically independent samples). Floral pollen varies significantly in the relative abundance of these sterols (one-way quasi-binomial GLM, χ25 = 61.0, P < 0.001). Data were obtained from ref. Letters (d) and asterisks (b,c) denote significance (post hoc comparisons of estimated marginal means with Tukey adjustment, *P < 0.05, **P < 0.01, ***P < 0.001). The total sterol concentration in pupal tissues varied significantly among pupal types and ranged from 130 to 260 μg g–1 fresh tissue (Gaussian GLM, χ22 = 91.9, P < 0.0001; Fig. The total sterol concentration in drone pupae was greater than in queen pupae and double that in worker pupae. The concentrations of individual sterols similarly varied among pupal types (Extended Data Fig. Five of these sterols (SITO, CAMP, CHOL, ISOFUC and 24-MC) were among the six most common sterols found in pollen (SITO, CAMP, CHOL, ISOFUC, cycloartenol and 24-MC)18 (Supplementary Data 1). Analyses of previously published data18 showed that floral pollen varied significantly in the relative abundance of these sterols (quasi-binomial GLM, χ25 = 61.0, P < 0.001). SITO had the highest median relative abundance (40.9%), followed by ISOFUC (9.50%) and 24-MC (5.84%), whereas CAMP, CHOL and DESMO were present at low levels (Fig. The coefficient of variation ranged from 0.830 (SITO) to 2.99 (DESMO; Fig. To produce the pollen sterols needed by bees, we used the oleaginous yeast Y. lipolytica as a host for sterol production. This yeast was chosen because of its high capacity for lipid accumulation and abundance of the sterol precursor acetyl-CoA20,21,22. Furthermore, we used a platform strain with improved mevalonate pathway flux towards farnesyl diphosphate to increase the supply of sterol precursors23. The platform strain is derived from the wild-type isolate W29 and it expresses a CRISPR-associated protein (Cas9) to enable genome editing. This strain also overexpresses four genes that encode mevalonate pathway enzymes: HMG1, ERG12, IDI1 and ERG20. Genes involved in acetyl-CoA formation, namely ATP-citrate lyase 1 (ACL1), and acetyl-CoA synthase from Salmonella enterica (ACS) are also included. The platform strain produced almost three times more ergosterol (ERGO, 1.95 mg g–1 dry cell weight (DCW)) than the wild-type strain W29 (0.659 mg g–1 DCW; Extended Data Fig. The endogenous ERGO pathway of yeasts can be remodelled to synthesize alternative sterols (Fig. However, previous attempts to produce non-native sterols in Y. lipolytica have struggled to achieve high specific yields or stereoselectivity (Supplementary Table 1). In Saccharomyces cerevisiae, genes involved in the late stages of the ERGO pathway are nonessential and alternative sterols can be taken up from the environment when ERGO biosynthesis is compromised, such as under anaerobic conditions24,25. However, most non-Saccharomyces yeasts, such as Y. lipolytica, lack sterol uptake transporters and do not exhibit flexible sterol auxotrophy26. For this reason, endogenous ERGO synthesis is essential, which makes deletion of latter ERGO pathway genes such as ERG4 (which encodes Δ24(28)-sterol reductase) challenging in Y. lipolytica27. a, Schematic of the sterol biosynthesis pathway from mevalonate (MVA) in yeast and plants. b, Gene edits used to create Y. lipolytica strains capable of producing non-native sterols that are tailored for honeybees. To overcome this issue, we used a sterol surrogate. The previously characterized STC1 gene from Tetrahymena thermophilia encodes a squalene-tetrahymanol cyclase that converts squalene to the pentacyclic triterpenoid tetrahymanol (TET; Extended Data Fig. 1), which can act as a substitute for ERGO function in yeasts28,29,30. TET is not anticipated to be detrimental to honeybees as pollen also contains diverse sterol intermediates and stanols that do not accumulate in honeybee tissues18. T. thermophilia SCT1 was introduced into the platform yeast strain under the weak PrGPAT promoter31 to drive minimal expression and to limit the diversion of carbon flux away from the sterol pathway. TET accounted for 14% of the sterols produced in the resulting TET strain (Extended Data Fig. By producing TET in Y. lipolytica, we were able to efficiently manipulate the sterol pathway. The subsequent elimination of ERGO production in our strains may have also helped to remove negative feedback inhibition on sterol biosynthesis. We first engineered Y. lipolytica to produce the most important honeybee sterol, 24-MC. Redirection of the Y. lipolytica sterol pathway from ERGO to 24-MC required us to sequentially delete the endogenous genes ERG4 and ERG5 (which encode sterol C-22 desaturase) and introduce a heterologous DWF5 or DHCR7 gene (which encode Δ7-sterol reductase; Fig. The erg4Δerg5Δ strain was used to screen heterologous DWF5 and DHCR7 variants identified in the genomes of plants, animals, fungi and bacteria. Ten diverse variants were selected and expressed under the strong PrTEFintron promoter31 (Fig. The strain that contained DWF5 from Tetraselmis sp. (a green algal phytoplankton) produced the highest content of 24-MC (42.1 mg g–1 DCW) and accumulated less TET and ergosta-5,7,24(28)-trienol intermediates than the next highest producing strain, which expressed DWF5 from Coccomyxa subellipsoidea (a unicellular green alga) (Fig. a, 24-MC production by engineered Y. lipolytica strains with different heterologous DWF5 and DHCR7 variants. Genes were selected from plants (DWF5), animals (DHCR7), algae, fungi and bacteria, including Tetraselmis sp. GSL018 (Tsp), Coccomyxa subellipsoidea (Cs) S. tuberosum (St), Ectocarpus siliculosus (Es), Mortierella vertcillata (Mv), Danio rerio (Dr), Legionella drancourtii (Ld), G. soja (Gs), Waddlia chondrophila (Wc) and ‘Candidatus Protochlamydia amoebophila' (CPa). Strains were compared for the content of 24-MC, TET and the main pathway intermediate ergosta-5,7,24(28)-trienol (n = 3 biologically independent samples). Error bars represent the mean ± s.d. b, Sterol composition of Y. lipolytica strains progressively engineered to produce a mixture of additional sterols required by honeybees (n = 3 biologically independent samples). Error bars represent the mean ± s.d. TET and total sterol content are shown in Extended Data Fig. 3. c, Time course of sterol content, DCW, glucose addition and broth glucose concentration during fed-batch fermentation of the mixed-sterol strain in 5-litre bioreactors (bioreactor cultivations were carried out in duplicate, and measurements from each bioreactor were carried out in technical duplicates). d, Sterol composition of the mixed-sterol strain after 84 h of fermentation (bioreactor cultivations were carried out in duplicate, and sterol measurements from each bioreactor were carried out in technical duplicates). Time courses of 5-litre bioreactor fermentation of the TET and W29 strains and sterol content from 5-litre bioreactor fermentation of the mixed-sterol, TET and W29 strains are shown in Extended Data Fig. The 24-MC-producing strain was subsequently engineered to create a mixed-sterol strain that produced small amounts of all the other sterols we observed in honeybees (Fig. We first introduced the previously characterized Δ24(28)-sterol reductase (encoded by SSR1) from potato Solanum tuberosum; this enzyme reduces the 24-MC side chain to produce CAMP32 (Fig. By using a heterologous plant variant, we could regulate the expression level with the weak promoter PrDGA1 (ref. 31) and obtain the C-24α R epimer of CAMP found in plants rather than the C-24β S epimer as in ERGO (Extended Data Fig. The resulting ‘two-sterol' strain produced 24-MC (19.9 mg g–1 DCW) and a small amount of CAMP (0.445 mg g–1 DCW, Fig. We then constructed a strain that was further capable of producing C-24 ethyl sterols by introducing the C-28 sterol methyl transferase (encoded by SMT) from Chenopodium quinoa (quinoa) under the control of the PrGPAT promoter. Δ24(25)-sterol reductases are typically found in animals and reduce DESMO to CHOL but are also present in Solanaceous plants as part of the glycoalkaloid synthesis pathway32. When expressed in a 24-MC-producing S. cerevisiae strain, both DESMO and CHOL are produced32. This property was similarly observed in our final ‘mixed-sterol' strain, which produced 24-MC (24.6 mg g–1 DCW), CAMP (0.433 mg g–1 DCW), ISOFUC (1.59 mg g–1 DCW), SITO (trace amounts), CHOL (0.396 mg g–1 DCW) and DESMO (0.898 mg g–1 DCW) during small-scale cultivation (Fig. The total detected sterol content decreased with each round of engineering from 60.2 mg g–1 DCW in the 24-MC strain to 28.0 mg g–1 DCW in the mixed-sterol strain (Extended Data Fig. The mixed-sterol strain was used for cultivation in bioreactors. We investigated sterol and biomass production by cultivating the 24-MC strain in fed-batch mode in 250-ml controlled bioreactors. The 24-MC content reached 8.72–12.0 mg g–1 DCW, and the biomass reached 29.0–33.0 g DCW per litre after 150 h of fermentation (Extended Data Fig. After 84 h of fermentation, the sterol content was 8.46–9.76 mg g–1 DCW, and the cell density reached 48.3–49.8 g DCW per litre (Fig. The relative sterol content increased for the first 24 h and then remained stable. 24-MC constituted approximately 65% of the total sterol content (Fig. ISOFUC, CAMP, CHOL, DESMO and SITO together constituted 15% of the total sterol content, and TET contributed the remaining 20% (Fig. We also fermented the W29 strain and the TET strain in 5-litre bioreactors for use as controls in feeding experiments. The sterol content for these two strains reached 0.100–0.118 and 0.099–0.204 mg g–1 DCW, respectively (Extended Data Fig. Previous studies of non-native sterol production in yeast have achieved up to 36 mg g–1 DCW epi-campesterol in Y. lipolytica and 19.3 mg g–1 DCW epi-ergosterol in S. cerevisiae under fermentation conditions (Supplementary Table 1). We initially achieved 42.1 mg g–1 DCW of 24-MC during small-scale cultivation, which is eightfold more sterol per unit biomass than vegetable oils or floral pollen18,33. This result demonstrates the utility of the pre-engineered platform strain and the potential of Y. lipolytica as a host for sterol production. However, the sterol content reduced with each engineering step of the strain and during cultivation scale-up. Reductions in sterol content have been observed in other studies of similar strains during bioreactor cultivation34,35. Nevertheless, pathway optimization, capacity for sterol storage and growth conditions for lipid accumulation can be further improved to increase the sterol content in Y. lipolytica36. Successful strategies in yeast include regulation of cellular sterol homeostasis and enzyme relocalization to lipid droplets37,38. The resulting yeast biomass from fermentation was dried and ground to create diets that could be tested in semi-field feeding trials with honeybee colonies. Dried yeast biomass was incorporated at 20% w/w into an artificial base-diet formulation that contained additional proteins, fats, vitamins and minerals required by honeybees. The following diets were tested: a mixed-sterol strain diet (MxSt), a TET strain diet (Tet), a W29 strain diet (WT) and a no yeast base diet (base). The latter three were used as control diets. Because the mixed-sterol strain produced only trace amounts of SITO, all yeast-based diets were supplemented with a commercial phytosterol mix, which contained approximately 12.0% w/w SITO (Supplementary Table 7). The base diet was also supplemented with a commercial phytosterol mix to match the total sterol concentration in the MxSt diet (0.340% w/w). We used a semi-field method with small colonies of honeybees to measure the impact of the different diets on brood production in the absence of incoming pollen. We conducted feeding trials over 3 months and monitored diet consumption, sterol composition, colony size and brood status (Fig. Every 15 days, we counted broods at the egg, larval and pupal (capped cell) stages in each colony to assess the ability of each treatment to support brood production and development. In the first 45 days, an extreme heatwave resulted in a reduction in bees of all ages across all colonies (Extended Data Fig. Hence, we supplemented colonies with extra nurse bees on days 32 and 45. Nurse bees have endogenous sterols that can be used to feed larvae3,16,19 (Fig. For this reason, we analysed the brood data starting from day 45, when we stopped adding nurse bees. a–c, Number of eggs (a), larvae (b) and capped brood cells (c) counted per colony for each treatment group for the last 2 months of the feeding trials. Eggs, larvae and capped brood cells were counted in each colony at 15-day intervals (n = 6 biologically independent colonies). Statistical analysis was performed by fitting two-way negative binomial GLMMs followed by post hoc comparisons of estimated marginal means at each time point with Tukey adjustment (*P < 0.05, **P < 0.01, ***P < 0.001). The number of capped brood also varied with diet treatment group (diet × time, χ29 = 30.0, P = 0.0004). Brood counts from the first month of the feeding trials are shown in Extended Data Fig. Note that the scales for months 2 and 3 differ, as brood production waned at the end of summer. The number of eggs varied with time but not diet treatment group (two-way negative binomial generalized linear mixed model (GLMM), diet + time, time: χ23 = 48.3, P < 0.0001; diet: χ23 = 0.782, P = 0.854; Fig. Feeding trials continued until mid-September when egg production in colonies begins to decline going into autumn. As such, there was a threefold reduction in egg laying from day 45 to day 91 and a corresponding decrease in the numbers of larvae, capped brood and adults (Fig. Overall, the number of larvae varied among the diet treatment groups, and the effect of the diet became apparent by day 75 (two-way negative binomial GLMM, diet × time, χ29 = 18.4, P = 0.0304; Fig. At day 75, there were significantly more larvae in the MxSt diet treatment group than the Tet diet treatment group (post hoc comparison of estimated marginal means with Tukey adjustment, P = 0.0109). The base diet treatment group also had more larvae than the Tet diet treatment group (P = 0.0295). By day 91, larval counts were low in all colonies. The low brood production at this time point in the MxSt treatment was due to a seasonal effect that is also observed when colonies are fed with pollen (Extended Data Fig. As time passed, the impact of the diet on bees increased, such that colonies fed with the MxSt diet were more likely to rear larvae to the viable pupal stage (two-way negative binomial GLMM, diet × time, χ29 = 30.0, P = 0.0004; Fig. By day 75, there were significantly more pupae in the MxSt diet treatment group than the Tet (P = 0.0333) and WT (P = 0.0230) diet treatment groups. By the end of the experiment (day 91), there were also significantly more pupae in the MxSt diet treatment group than the base (P = 0.0328), Tet (P = 0.020) and WT (P = 0.020) diet treatment groups. There were still 10–40 capped brood cells in all but 1 colony in the MxSt diet treatment group. By contrast, there were 1–3 capped brood cells in 3 colonies in the base diet treatment group and none in the control yeast diet groups. The data for the control diets (which do not have the required sterols) were similar to our observations of colonies starved of pollen, whereby the rate of brood production is a function of the duration of nutrient starvation (Extended Data Fig. Eventually, in all colonies fed with diets that lacked the required sterols (base, Tet and WT diets), the nurse bees did not have sufficient levels of key corporeal sterols necessary for rearing brood. Although the number of pupae correlated with the diet given, sterol levels in the surviving pupae varied less than in the nurse bees and were not significantly different among the treatment groups (Gaussian GLMM, diet × time, time: χ24 = 14.9, P = 0.005; diet: χ23 = 4.39, P = 0.222; Extended Data Fig. Moreover, sterol levels in the bodies of nurse bees showed less variation than the gut contents (Extended Data Fig. In the MxSt and Tet diet treatment groups, TET accumulated in the guts of nurse bees (up to 42.4 μg per bee) but to a lesser extent in the bodies (up to 18.7 μg per bee) and not at all in the pupae (Extended Data Fig. Pollen contains diverse sterols, including cyclopropyl intermediates and stanols18, but our data indicated that only the Δ5-terminal intermediates of the sterol pathway tended to accumulate in honeybees. In yeast-feeding experiments with our mixed-sterol strain, nurse workers transferred only the key sterols produced by the yeast (24-MC, ISOFUC, DESMO, CAMP, SITO and CHOL) to the brood. Two diets contained the sterol surrogate TET, but this was not found in the brood. This finding indicates that bees have selective mechanisms for enriching larval jelly with the key sterols. Our data validate previous findings11 that honeybees probably require multiple sterols for continued brood production. The most important sterols may be 24-MC and CHOL, as colonies fed sterol-free diets supplemented with 24-MC or CHOL produce more pupae than when supplemented with SITO or CAMP11. We observed a clear benefit to colonies that were fed the MxSt diet, which contains less than 0.5% w/w sterol. Understanding the full impact on the health and performance of honeybees fed with our MxSt yeast would require a long-term field study with standard size colonies. Yeast biomass is an additional valuable source of proteins, lipids and vitamins22,39. Hence, it may be preferable to favour biomass over sterol yield when optimizing fermentation parameters to produce biomass for feed. We speculate that further engineering of lipid, protein and terpenoid profiles to produce nutrients such as beneficial fatty acids, antioxidants, carotenoids and fragrances40,41,42 could extend the suitability of Y. lipolytica biomass as a honeybee feed. Y. lipolytica biomass is regarded as safe for inclusion in food and feed, and engineered biomass has been established as feed in aquaculture39,43,44. The direct use of inactivated yeast biomass in the diet circumvents the need for sterol extraction, which is expensive and inefficient, and minimizes downstream processing. Such technology could also provide an avenue for the development of feeds for other farmed arthropods. Escherichia coli strain DH5α was used for plasmid construction. E. coli was grown at 37 °C and 300 r.p.m. in lysogeny broth (LB) liquid medium and at 37 °C on plates of LB solid medium supplemented with 20 g l–1 agar. Ampicillin was supplemented at a concentration of 100 mg l–1 for plasmid selection. The Y. lipolytica W29 strain (MATa, Y-63746 ARS Culture Collection, The National Center for Agricultural Utilization Research) and the W29-derived platform strain ST9100 (MATa ku70∆::PrTEF1-cas9-TTef12::PrGPD-dsdA-TLip2 IntC_2-HMG<-PrGPD-PrTefInt->ERG12 pCfB8823 IntC_3-SeACS<-PrGPD-PrTefInt->YlACL1 IntD_1-IDI1<-PrGPD-PrTefInt->ERG20, a mevalonate-upregulated strain) have been previously described23. Details for all of the strains used in this study are provided in Supplementary Table 2. Y. lipolytica was grown at 30 °C on yeast extract peptone dextrose (YPD) medium containing 10 g l–1 yeast extract, 20 g l–1 peptone and 20 g l–1 glucose, supplemented with 20 g l–1 agar for preparation of solid medium. For selection, either nourseothricin (250 mg l–1) or hygromycin (400 mg l–1) was added to the medium. Cultivation of strains for sterol production was performed in YPD medium containing 80 g l–1 glucose. Chemicals were obtained, unless indicated otherwise, from Sigma-Aldrich or Merck. Nourseothricin was purchased from Jena BioScience. The codon-optimized sequences are listed in Supplementary Table 3. The plasmids, BioBricks and primers used in this study are listed in Supplementary Tables 4–6. BioBricks were amplified by PCR using Phusion U polymerase (Thermo Scientific). BioBricks were assembled into EasyCloneYALI vectors with uracil-specific excision reagent (USER) cloning31. For marker-mediated gene deletion, upstream and downstream homology arms for relevant genes were synthesized as BioBricks by PCR amplification from the genomic DNA of the platform strain ST9100. Knockout constructs were assembled from BioBricks through USER reactions as detailed in Supplementary Table 5. USER reactions were transformed into E. coli, and correct assemblies were verified by Sanger sequencing (Eurofins). The yeast vectors were integrated into different previously characterized intergenic loci in the Y. lipolytica genome as previously described31. Integration vectors were digested with NotI enzyme (New England BioLabs) before lithium acetate transformation, as previously described31. Correct integration was verified by colony PCR using Taq DNA polymerase master mix RED (Ampliqon) with vector-specific primers and primers complementary to the genomic region adjacent to the integration site31. For marker-mediated gene deletion, transformants were selected on YPD plates supplemented with antibiotic, and correct transformants were confirmed by colony PCR. Marker removal was performed by transformation of the strains with a Cre-recombinase episomal vector31. Marker removal was confirmed by colony PCR. Yeast strains were inoculated into 2.5 ml YPD in 24-deep-well plates with air-penetrable lids (EnzyScreen). The plates were incubated at 30 °C with 300 r.p.m. agitation for 24 h. The optical density at 600 nm (OD600) was measured with an Implen P300 NanoPhotometer. The cultures were then diluted to an OD600 of 0.1 in 2.5 ml fresh YPD medium with 80 g l–1 glucose and grown for a further 72 h at 30 °C with 300 r.p.m. All cultivations were performed in triplicate. DCW was measured at the end of cultivation, whereby 1 ml of culture broth was transferred to a preweighed 2 ml microcentrifuge tube, centrifuged (3,000g, 5 min) and the supernatant was discarded. The cells were then washed twice with deionized water (1 ml). The cell pellet was dried at 60 °C for 7 days before the final weight was measured. For sterol extraction from yeast, 1 ml of culture broth was transferred to a 2 ml microcentrifuge tube, centrifuged and the supernatant was discarded. The cells were washed twice with deionized water (1 ml). The cell pellet was resuspended in 10% w/v methanolic potassium hydroxide (500 μl) and transferred to a 1 ml glass vial for saponification. The saponified samples were then vortexed and spiked with 50 μl of internal standard (1 mg ml–1 epicoprostanol in absolute ethanol). Next, 500 μl n-hexane was added to each sample for extraction of the free sterol component. Samples were vortexed and the organic phase transferred to a 2 ml microcentrifuge tube. The extraction step was repeated in a further 500 μl n-hexane. The combined hexane phases were left overnight at room temperature for evaporation of the solvent. Sterol crystals remained in the tube. For sterol analysis of the diets, each diet was sampled three times into preweighed 20 ml glass vials, and the weight of each sample was recorded. For sterol extraction from honeybee tissues, samples were first dried by freeze drying. Samples were dried at −48 °C under vacuum for 4 days. Dried samples were weighed and stored at −80 °C. For saponification, samples were first broken up with a spatula. For gut samples (in 2 ml microcentrifuge tubes), samples were suspended in 500 μl 10% w/v methanolic potassium hydroxide. For honeybee tissue samples (in 20 ml vials; pupae, nurse carcasses and queens), samples were suspended in 2.5 ml 10% w/v methanolic potassium hydroxide. Diet samples (in 20 ml vials) were suspended in 5 ml 10% w/v methanolic potassium hydroxide. Samples were incubated at 70 °C for 2 h in a water bath, with vortexing at 30–60 min intervals. The saponified samples were then spiked with 50 μl (gut samples) or 100 μl (diet, pupae, nurse carcasses and queen samples) of internal standard (1 mg ml–1 epicoprostanol in absolute ethanol). For extraction of the free sterol component, 500 μl (gut samples), 2.5 ml (pupae, nurse bee carcasses and queen samples) or 5 ml (diet samples) n-hexane was added to each sample. Samples were vortexed and the organic phase transferred to a 2 ml (gut samples) or 7 ml (diet, pupae, nurse bee carcasses and queen samples) glass vial. The extraction step was repeated and the combined hexane phases were left overnight at room temperature for evaporation of the solvent. The resulting extracts were resuspended in 500 μl (gut samples) or 1 ml (diet, pupae, nurse bee carcasses and queen samples) n-hexane and vortexed. From each sample, a subsample of 250 μl was transferred to a 1.5 ml microcentrifuge tube and left at room temperature overnight for final drying. Sterols were resuspended in 500 μl pyridine that contained 20 μl N,O-bis(trimethylsilyl)acetamide (Merck) and incubated for 4 h at 50 °C and then briefly vortexed before direct injection into an Agilent Technologies 8860 gas chromatograph connected to an Agilent Technologies 5977 MSD mass spectrometer (for gas chromatography–mass spectrometry). Samples were eluted over an Agilent HP-5MS column using a splitless injection at 250 °C with a standard gas chromatography program at 170 °C for 1 min, ramped to 280 °C at 20 °C min−1 and monitoring between 50 and 550 AMU. Sterols were identified by comparing their retention time relative to CHOL and mass spectra data available from the National Institute of Standards and Technology mass spectral library per a previous study18. Sterol identity in the final strain ST12178 was confirmed through comparison with authentic standards. Compound identification (using target ions) and quantification were carried out using ChemStation Enhanced Data Analysis (v.E.01.00). The Ambr 250 system (Sartorius Stedim Biotech) was used to carry out 250 ml fed-batch fermentation in duplicate. The 24-MC strain ST11064 was re-streaked from glycerol stocks stored at −80 °C onto a YPD agar plate and incubated at 30 °C for 48 h. The preculture was prepared by inoculating strain ST11064 biomass from the plate into 50 ml YPD medium in a 250 ml shake flask and incubating at 30 °C for 24 h with 250 r.p.m. For Ambr 250 cultivation, the batch medium comprised mineral medium supplemented with yeast extract (10 g l–1) and citric acid (20 g l–1). The mineral medium was prepared with 0.5 g l–1 MgSO4⋅7H2O, 14.4 g l–1 KH2PO4, 0.1% (v/v) vitamin solution and 0.2% (v/v) trace metal solution as previously described45, but with 3.4 g l–1 NH4Cl and glycerol as the carbon source (40 g l–1). The temperature was held constant at 30 °C. Dissolved oxygen was maintained above 20% by using a cascade of stirring speed ranging from 600 to 3,000 r.p.m. and aeration up to 1 volume air per volume growth medium per minute. The pH was maintained at 5.5 through the automatic addition of 1 M NaOH and 2.6 M H3PO4. Antifoam 204 (Sigma) was added automatically. Online measurements of acid and base addition, carbon dioxide evolution rate, dissolved oxygen and stirring speed were recorded for each reactor. The feed medium comprised 250 g l–1 glycerol. Feeding was set to a constant rate of 0.9 ml h–1. Samples were taken from each reactor every 6 h for the first 24 h and then every 12 h and immediately frozen until preparation for analyses. DCW and sterol content were determined from 1 ml samples as described above for small-scale cultivation. For larger scale fermentation, strains were cultivated by fed-batch fermentation in a 5-litre bioreactor (BIOSTAT B-DCU, Sartorius). All fermentations were carried out in duplicate. For each of the strains W29, the TET strain ST11005 and the mixed-sterol strain ST12178, the strain was re-streaked from glycerol stocks onto a YPD agar plate and incubated at 30 °C for 24 h. The preculture was prepared by inoculating strain biomass from the plate into 50 ml YPD medium in a 250 ml shake flask and incubating at 30 °C for 24 h with 250 r.p.m. The volume of preculture required to inoculate a 2-litre batch medium to a starting OD600 of 2.5 was centrifuged for 10 min at 4,000g and concentrated to 10 ml volume. This cell suspension was used to inoculate the bioreactors. The bioreactors were equipped with pH, pO2 and temperature probes. The temperature was held constant at 30 °C. Dissolved oxygen was maintained above 20% by adjusting stirring between 600 and 1,200 r.p.m. Antifoam A (Sigma) was added as required. The batch medium comprised mineral medium supplemented with yeast extract (20 g l–1) and peptone (40 g l–1). The mineral medium was prepared with 0.5 g l–1 MgSO4⋅7H2O, 14.4 g l–1 KH2PO4, 0.1% (v/v) vitamin solution and 0.2% (v/v) trace metal solution as previously described45, 40 g l–1 glucose and 1 ml l–1 antifoam A (Sigma). An exponential feeding profile was programmed, and feeding was initiated 24 h after inoculation. After 36 h of exponential feeding, the feed was switched to a constant rate of 75 ml h–1 until the end of fermentation. Duplicate samples from each reactor were taken every 8 h for the first 24 h and then every 12 h to measure DCW, sterol content, OD600 and glucose concentration. DCW and sterol content were determined from 1-ml samples as described above for small-scale cultivation. During fermentation, 1 ml of culture broth was centrifuged, and the supernatant was used to measure the glucose concentration using a glucose HK assay kit (Sigma). The supernatant was then filtered and frozen until further analyses. Glucose was later quantified using a Dionex Ultimate 3000 HPLC system equipped with a RI-101 refractive index detector (Dionex). An Aminex HPX-87H column (7.8 × 300 mm, Bio-Rad) with a Micro-Guard Cation H+ guard column (4.6 × 30 mm) heated to 30 °C was injected with a 10-µl sample. The mobile phase consisted of 5 mM H2SO4 with an isocratic flow rate of 0.6 ml min–1, which was held for 15 min. HPLC data were processed using Chromeleon software (v.7.2.9, Thermo Fisher Scientific). Glucose was identified and quantified using authentic standards. Glucose concentrations were calculated from the peak area by extrapolation from a six-point calibration curve regression. At the end of fermentation, the yeast biomass was recovered from the culture by centrifugation (4,000g, 20 min) and washed with deionized water. The biomass was heat-inactivated and dried (60 °C for a minimum of 24 h). The dried material was ground to a fine powder and stored at −20 °C until further use. The yeast biomass cannot be subject to inactivation by autoclave or chemical treatment, as this will degrade the sterols present in the yeast. Incubation at 60 °C is commonly deemed sufficient to irreversibly inactivate yeast, and heat-inactivation of genetically modified yeast followed by feeding the inactivated yeast to live animals is a method that has been previously used in the United Kingdom47. Irreversible inactivation of the yeast was confirmed using a standard colony-forming unit assay. The heat-inactivated dried yeast was dissolved at 10 mg ml–1 in water. The suspension was plated in serial dilution (100 μl plated of 10, 1, 0.1, 0.01 and 0.001 mg ml–1 suspensions) on YPD agar and the plates were incubated at 30 °C for at least 7 days. No growth of Y. lipolytica colonies was observed. The detection limit is one organism per mg material or 106 viable organisms per kg of material. The yeast biomass was then incorporated into a meridic artificial diet at 20% w/w. Four diet types were prepared: a mixed-sterol yeast diet that contained the mixed-sterol strain ST12178; a wild-type yeast diet that contained strain W29; a platform yeast diet that contained the TET strain ST11005; and a base diet control without yeast supplementation. The base diet control was formulated to maintain total protein, sugar, sterol and fat content at the same level as in the yeast-supplemented diets. The content of this diet was a modified version of a previously described diet48. Specifically, the base diet contained 17% soy protein isolate (Soysol, MyVegan), 69.4% sugars (fructose, glucose, sucrose and maltodextrin), 6% lipids, 6.50% deionized water, 0.100% vitamin and mineral supplement (Latshaw Apiaries), 0.6% commercial phytosterol mix (BulkSupplements; Supplementary Table 7) and 0.400% carrageenan kappa. The diets were divided into 50 g patties and stored at −20 °C until use. Specifically, the yeast-supplemented diets contained the following components: 20.0% dried yeast powder, 7.80% soy protein isolate (Soysol, MyVegan), 63.4% sugars, 0.1% commercial phytosterol mix (BulkSupplements, approximately 55% purity containing a mixture of sterols and stanols; Supplementary Table 7), 4% lipids, 4.20% deionized water, 0.100% vitamin and mineral supplement (Latshaw Apiaries) and 0.400% carrageenan kappa (Special Ingredients). For the sterol analysis of honeybee brood, which is used as a proxy for the natural sterol profile of honeybee pupae, we sampled worker, drone and queen pupae from naturally fed colonies in our apiary (Buckfast queens, John Krebs Field Station, Oxford). Worker pupae were directly collected from capped brood frames. Queen pupae were reared by grafting young larvae (2–3 days after hatching) into Nicot Queen Rearing Cups (Paynes Bee Farms). These were placed in queenless colonies in repurposed Styrofoam mini-nucleus hives (APIDEA) for up to 8 days until development to the capped brood stage. Tissues (3 pupae per replicate, n = 5) were sampled into preweighed 20 ml glass vials and the fresh weight was recorded before storing at −80 °C until further analysis. Feeding trials were conducted using honeybee colonies maintained in repurposed Styrofoam mini-nucleus hives made up of one brood box with five frames and a top feeder with a hole for patty delivery. Hives were maintained in a closed glasshouse environment designed to prevent bee escape during the period between July and October 2022. Hives were distributed across two glasshouse rooms with varying entrance orientation. Feeders with 30% w/v sugar solution and water were distributed inside the glasshouse and replenished as required. The in-hive and ambient temperature and humidity were recorded every 30 min using autonomous in-hive sensors (Supplementary Data 3). Initially, each hive contained 900–1,200 adult bees, 2–3 frames of brood, larvae and eggs and 1–2 frames of honey stores, but no bee bread. Newly mated queens were introduced in cages with beekeeping candy (Candito, PIDA), for slow release, 3 days before the start of the experiment. Feeding trials were conducted over 3 months from June to September 2022 at the John Krebs Field Station, Oxford. Six hives were randomly assigned to each treatment group (n = 6). Hive weight (after removal of the diet patty) and patty weight were measured. The number of bee seams (one seam defined as a continuous line of bees between adjacent frames, observed after initial hive opening) and frames filled with honey (sugar stores) were estimated by visual inspection. The presence of the mated queen, sugar stores, eggs, larvae and capped brood were checked, and brood frames were photographed for subsequent counting. Eggs, larvae and capped brood were counted using the Adobe Photoshop count tool. Full assessments of the hives were conducted every 15 days. Six days after each full assessment, hives were partially assessed with minimal disruption to the colony. The presence of the mated queen, eggs, larvae and capped brood were briefly checked. On days 21 and 45, hives with low populations (fewer than four bee seams) were topped up with orphanized nurse bees from mixed, naturally fed colonies. At every assessment, nurse bee and pupae samples were taken from three hives from each treatment group. Six nurse bees and six pupae were collected from each of the sampled hives. Samples were collected into preweighed 20 ml glass scintillation vials. The gut contents were transferred to a 2 ml microcentrifuge tube and the remaining tissues were returned to the 20 ml vial. Dissected samples were stored at −80 °C until further analysis. A semi-field pollen starvation trial was conducted from August to October 2023 at the John Krebs Field Station, Oxford. Colonies were housed in mini-nucleus hives set up in an identical manner to the yeast-feeding trial and were maintained in one room of a mesh polytunnel purpose-built to prevent bee escape. One week before the start of the treatment, colonies were topped up with nurse bees from mixed, naturally fed colonies so that each box contained at least five bee seams and re-queened where necessary. We used a mix of pre-existing colonies and newly established colonies, but all were fed pollen for at least 1 month before the start of the experiment and were producing brood. Buckets of water and feeders with 30% w/v sugar solution were distributed inside the polytunnel. Colonies were supplied with pollen or candy patties. Candy patties consisted of about 80% beekeeping candy and around 20% maltodextrin, which was added to slow consumption and reduce melting of the patty in the hive. After 1 month of a feeding period, colonies in treatment groups 1, 2 and 3 were deprived of pollen for the corresponding number of weeks and fed with candy only. The control group (0) was fed pollen throughout. Treatment groups were balanced across hive entrance orientations, colony strengths (bee seams) and position in the polytunnel. We performed a partial assessment every week to measure patty weight and hive weight and estimate bee seams. Every 2 weeks, a full assessment recorded the presence of the mated queen, eggs, larvae and capped cells, and the amount of sugar and pollen stores, and every frame was photographed. The photographs were used to count the number of cells with eggs, larvae and pupae in each hive using ImageJ49. All graph plotting and statistical analyses were performed in R50 (except Extended Data Fig. 8, which was created in GraphPad Prism and analysed using SPSS). Data were tested for normality when appropriate. Post hoc analysis was performed using the car52 and emmeans53 packages with Tukey adjustments for family-wise error rates. The mean relative abundance of each of the major sterols in naturally fed honeybee pupae was calculated as a percentage of the total sterol content. We compared sterol relative abundance across pupal types using a GLM with quasi-binomial distribution (relative abundance modeled as a function of (~) sterol type × pupal type). Sterol concentrations in pupae were calculated from the fresh weights of the pupal tissue. We compared sterol concentrations in pupal tissue across types using GLMs with Gaussian distributions (sterol concentration ~ sterol type × pupal type). We compared the relative abundance of sterols in pollen using a GLM with quasi-binomial distribution (relative abundance ~ sterol type). The coefficient of variation for each sterol was calculated by dividing the standard deviation by the mean relative abundance values of each sterol. Counts for each brood type were compared across diet treatment groups using GLMMs, fitted to counts from day 45 onwards, with hive identifier (ID) as a random effect and negative binomial distributions (brood count ~ diet × time + (1|hive ID)). For egg counts, the interaction term was excluded (egg count ~ diet + time + (1|hive ID)). Because the hive weight correlated significantly with bee seams and the consumption rate correlated significantly with the hive weight for all treatment groups (Extended Data Fig. 8), the daily consumption rates in each interval were normalized by hive weight as a proxy for colony size. The normalized consumption rates were compared across diet treatment groups using a GLMM, with hive ID as a random effect and Gaussian distributions (normalized consumption rate ~ diet + time + (1|hive ID)). The weight of each hive was compared across diet treatment groups using a GLMM, with hive ID as a random effect and Gaussian distributions (hive weight ~ diet + time + (1|hive ID)). The number of bee seams in each hive and the number of frames filled with honey were doubled to give integer values and compared across diet treatment groups using GLMMs, with hive ID as a random effect and Poisson distributions (2 × bee seams ~ diet + time + (1|hive ID); 2 × sugar stores ~ diet + time + (1|hive ID)). For the sterol contents in the bodies and guts of nurse bees and of pupae (μg per individual), GLMMs were fitted for each sample type in each sterol, with Gaussian distributions and hive ID as a random effect (sterol content ~ diet × time + (1|hive ID)). We examined the relationships between variables measured during feeding trials by fitting GLMs (response variable ~ predictor variable × diet). All models used Gaussian distributions, apart from when comparing capped brood counts to the total sterol content of bodies of nurse bees from the same colony. In this case, a negative binomial distribution was used. When no significant interaction was found between diet treatment group and the predictor variable, the interaction term was excluded from the models (response variable ~ predictor variable + diet). For each diet treatment group, linear regressions were fitted between the predictor and response variables of interest. All experiments in the study were performed once. Apart from bioreactor cultivations, a minimum of three biologically independent replicates were collected for all experiments. For bioreactor cultivations, two biological replicates with two technical replicates each were performed for measurement of all parameters. Replicates gave similar results for all experiments. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data are available from Figshare (https://doi.org/10.6084/m9.figshare.22820849)54. Source data are provided with this paper. Goulson, D., Nicholls, E., Botías, C. & Rotheray, E. L. Bee declines driven by combined stress from parasites, pesticides, and lack of flowers. Dolezal, A. G. & Toth, A. L. Feedbacks between nutrition and disease in honey bee health. A., Thompson, M. J., Herbert, E. W. & Shimanuki, H. Sterol utilization in honey bees fed a synthetic diet: analysis of prepupal sterols. Haefeker, W. Pollen supplements and substitutes in the EU feed market: a product/market survey for bees and other animal species. Wright, G. A., Nicolson, S. W. & Shafir, S. Nutritional physiology and ecology of honey bees. Global pollinator declines: trends, impacts and drivers. Threats to an ecosystem service: pressures on pollinators. Branchiccela, B. et al. Impact of nutritional stress on the honeybee colony health. & Vaissière, B. E. Economic valuation of the vulnerability of world agriculture confronted with pollinator decline. & Harder, L. D. The global stock of domesticated honey bees is growing slower than agricultural demand for pollination. & Shimanuki, H. Sterol utilization in honey bees fed a synthetic diet: effects on brood rearing. Desmond, E. & Gribaldo, S. Phylogenomics of sterol synthesis: insights into the origin, evolution, and diversity of a key eukaryotic feature. Behmer, S. T. & Nes, W. D. Insect sterol nutrition and physiology: a global overview. Ikekawa, N., Morisaki, M. & Fujimoto, Y. Sterol metabolism in insects: dealkylation of phytosterol to cholesterol. & Thompson, M. J. Biosynthesis of makisterone A and 20‐hydroxyecdysone from labeled sterols by the honey bee, Apis mellifera. & Feldlaufer, M. F. Selective sterol transfer in the honey bee: its significance and relationship to other hymenoptera. A. Mandibular glands secrete 24-methylenecholesterol into honey bee (Apis mellifera) food jelly. Zu, P. et al. Pollen sterols are associated with phylogeny and environment but not with pollinator guilds. Beopoulos, A. et al. Control of lipid accumulation in the yeast Yarrowia lipolytica. Ledesma-Amaro, R. & Nicaud, J. M. Yarrowia lipolytica as a biotechnological chassis to produce usual and unusual fatty acids. Athenstaedt, K. et al. Lipid particle composition of the yeast Yarrowia lipolytica depends on the carbon source. A. et al. Yarrowia lipolytica strains engineered for the production of terpenoids. Wilcox, L. J. et al. Transcriptional profiling identifies two members of the ATP-binding cassette transporter superfamily required for sterol uptake in yeast. Lorenz, R. T. & Parks, L. W. Regulation of ergosterol biosynthesis and sterol uptake in a sterol-auxotrophic yeast. Tesnière, C., Pradal, M., Legras, J.-L. & Legras, J.-L. Sterol uptake analysis in Saccharomyces and non-Saccharomyces wine yeast species. Patterson, K. et al. Functional genomics for the oleaginous yeast Yarrowia lipolytica. Wiersma, S. J., Mooiman, C., Giera, M. & Pronk, J. T. Squalene-tetrahymanol cyclase expression enables sterol-independent growth of Saccharomyces cerevisiae. Dekker, W. J. C. et al. Engineering the thermotolerant industrial yeast Kluyveromyces marxianus for anaerobic growth. Bouwknegt, J. et al. A squalene-hopene cyclase in Schizosaccharomyces japonicus represents a eukaryotic adaptation to sterol-limited anaerobic environments. Holkenbrink, C. et al. EasyCloneYALI: CRISPR/Cas9-based synthetic toolbox for engineering of the yeast Yarrowia lipolytica. Sawai, S. et al. Sterol side chain reductase 2 is a key enzyme in the biosynthesis of cholesterol, the common precursor of toxic steroidal glycoalkaloids in potato. Fernandes, P. & Cabral, J. M. S. Phytosterols: applications and recovery methods. A. et al. Production of abscisic acid in the oleaginous yeast Yarrowia lipolytica. Ma, Y., Li, J., Huang, S. & Stephanopoulos, G. Targeting pathway expression to subcellular organelles improves astaxanthin synthesis in Yarrowia lipolytica. Gu, Y., Jiao, X., Ye, L. & Yu, H. Metabolic engineering strategies for de novo biosynthesis of sterols and steroids in yeast. Guo, X. J. et al. Compartmentalized reconstitution of post-squalene pathway for 7-dehydrocholesterol overproduction in Saccharomyces cerevisiae. Turck, D. et al. Safety of Yarrowia lipolytica yeast biomass as a novel food pursuant to regulation (EU) 2015/2283. Cao, L. et al. Engineering Yarrowia lipolytica to produce nutritional fatty acids: current status and future perspectives. Removal of lycopene substrate inhibition enables high carotenoid productivity in Yarrowia lipolytica. Sustainable source of omega-3 eicosapentaenoic acid from metabolically engineered Yarrowia lipolytica: from fundamental research to commercial production. Groenewald, M. et al. Yarrowia lipolytica: safety assessment of an oleaginous yeast with a great industrial potential. Jensen, N. B. et al. EasyClone: method for iterative chromosomal integration of multiple genes Saccharomyces cerevisiae. Sáez-Sáez, J. et al. Engineering the oleaginous yeast Yarrowia lipolytica for high-level resveratrol production. Goh, S. et al. Development of a potential yeast-based vaccine platform for Theileria parva infection in cattle. Alternative sources of supplements in Africanized honeybees submitted to royal jelly production. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2022). Brooks, M. E. et al. glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling. & Weisberg, S. An R Companion to Applied Regression (Sage, 2019). Wright, G. Engineered yeast provides rare but essential pollen sterols for honeybee colonies. We thank S. Waddell for the initial idea; D. Nes and W. Zu for initial discussions about ways to engineer a yeast to produce sterols; P. Rubaszka for support with Ambr 250 bioreactors; A. F. Jensen for help with bioreactor autoclaving; C. Glitz for assistance with HPLC; D. Stabler and J. Chennells for preliminary feeding trials; H. Bridge and L. Beaupere for beekeeping assistance during feeding trials; S. Furse for help with freeze-drying; and L. Beaupere and K. Walter for data transcription. Temperature and humidity sensors were provided by BeeHero. E.M. was funded by a BBSRC iCASE grant (BB/M011224/1), with Apix Nutrition as an industrial partner. This work was funded in part by BBSRC grant BB/T015292/1 awarded to G.A.W. Elynor Moore, Raquel T. de Sousa, Stella Felsinger, Rui F. S. Gonçalves & Geraldine A. Wright The Novo Nordisk Foundation Centre for Biosustainability, Technical University of Denmark, Kongens Lyngby, Denmark Jonathan A. Arnesen, Jane D. Dyekjær & Irina Borodina Natural Resources Institute, University of Greenwich, Chatham, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out the honeybee feeding trials. performed the heterologous gene searches. performed the mass spectrometry analyses and identified sterols. performed the pollen-feeding control trial. conducted data analyses and produced the figures. are inventors on a pending patent application relating to certain research described in this article (international patent application number PCT/GB2022/052917 assigned to Apix Biosciences). have shareholdings (<2%) in Apix Biosciences. The remaining authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The basic sterol structure with numbering of the carbon atoms, as well as the structures, common names, systematic names, and chemical formulae of the main sterols referred to in this study. There was a significant interaction between sterol and pupal type (two-way Gaussian GLM, χ2(10) = 255, P < 0.0001). 24-MC was more concentrated in drone than queen and worker pupal tissue. Isofucosterol (ISOFUC) was also more concentrated in drone than queen and worker pupal tissue. Sitosterol (SITO) was more concentrated in drone than worker pupae. b, Mean relative abundance of each of the major sterols in nurses from day 0, and orphanised top-up nurses from day 32 and day 45, given as a percentage of the total sterol (six pupae per sample, t0 nurses, n = 24 biologically independent samples); t32 orphanised nurses, n = 10; t45 orphanised nurses, n = 4) Error bars represent ± s.d. a, Sterol content in W29, platform, TET and erg5Δ strains (n = 3 biologically independent samples). b, TET, and total sterol content in Y. lipolytica strains engineered to produce a mixture of non-native sterols (n = 3 biologically independent samples). c, Time course of sterol content, DCW and glycerol addition during fed-batch fermentation of the 24-MC strain in Ambr® 250 bioreactors (n = 2 biologically independent samples). d, Time course of operational parameters during fed-batch fermentation of the 24-MC strain in Ambr® 250 bioreactors (each plot represents one bioreactor). e, Time course of sterol content, DCW, glucose addition and glucose concentration in the broth during fed-batch fermentation of the W29 strain in 5-L bioreactors (data plotted is the average of two independent bioreactor cultivations). f, Time course of sterol content, DCW, glucose addition and glucose concentration in the broth during fed-batch fermentation of the TET strain in 5 L bioreactors (data plotted is the average of two independent bioreactor cultivations). g, Sterol composition of the W29 and TET strains after fed-batch fermentation in 5 L bioreactors (bioreactor cultivations were carried out in duplicate, sterol measurements from each bioreactor were carried out in technical duplicates). a, Number of eggs, larvae and capped brood cells counted per hive for each treatment group for the first month of the feeding trials. Eggs, larvae, and capped brood cells were counted in each colony at 15-day intervals (n = 6 biologically independent colonies). b, The total consumption (one-way Gaussian GLM, χ2(3) = 3.15, P = 0.370) and total diet provided (one-way Gaussian GLM, χ2(3) = 5.74, P = 0.1252) to each colony over the course of the feeding trials did not vary significantly between treatment groups (n = 6 biologically independent colonies). c, Time course of diet consumption rate normalised by hive weight (n = 6 biologically independent colonies). The normalised consumption rate varied with time (two-way Gaussian GLMM, χ2(12) = 526, P < 0.0001) and diet treatment group (χ2(3) = 15.8, P = 0.0012). Post hoc pairwise comparison of estimated marginal means revealed that consumption rate was higher for colonies provided with the MxSt diet than those provided with the Tet (P = 0.0087) or Base (P < 0.0001) diets at t51. At t66, consumption rate for colonies provided with the MxSt diet was significantly greater than those provided with the Tet (P = 0.0388) or WT (P = 0.0385) diets. a, Variation in hive weight during feeding trials (n = 6 biologically independent colonies). Hive weight varied with time but not diet treatment group (two-way Gaussian GLMM, time: χ2(12) = 176, P < 0.0001, diet: χ2(3) = 1.76, P = 0.623). b, Variation in number of bee seams during feeding trials (n = 6 biologically independent colonies). Bee seams varied with time but not diet treatment group (two-way Poisson GLMM (fitted to doubled count values), time: χ2(12) = 144, P < 0.0001; diet: χ2(3) = 1.38, P = 0.710). c, Variation in sugar stores during feeding trials (n = 6 biologically independent colonies). Sugar stores did not vary significantly with time or diet treatment group (two-way Poisson GLMM (fitted to doubled count values), time: χ2(12) = 8.67, P = 0.731; diet: χ2(3) = 3.11, P = 0.375). a, Lines represent the average sterol content per bee (n = 3 biologically independent samples) for each treatment group (‘mixed-sterol strain diet' (MxSt), ‘tetrahymanol strain diet' (Tet), ‘W29 strain diet' (WT), or ‘no yeast base diet' (Base)). Full statistical analysis results are presented in Supplementary Data 2. From t45 to t75, there was significantly more sterol in the guts of nurses from the MxSt treatment group than those from any other treatment group (post hoc pairwise comparison of estimated marginal means, all P values < 0.0133). By t75 there was more sterol in the nurse bodies from the MxSt treatment group than all other treatment groups (all P values < 0.0001). Total sterol content in the pupae varied significantly with time but not diet treatment group (two-way Gaussian GLMM, time: χ2(4) = 13.3, P = 0.0098; diet: χ2(3) = 3.50, P = 0.321). There was more TET in the nurse guts from the MxSt diet treatment group than all other treatment groups from t45-t60 (all P values < 0.0005) and more TET in the nurse bodies from the MxSt diet treatment group than all other treatment groups at from t60-t75 (all P values < 0.0001). Only trace amounts of TET were detected in the pupae. Time course of temperature (a) and humidity (b) in mini-nucleus hives from each treatment group (‘mixed-sterol strain diet' (MxSt; n = 5 independent sensors in different hives), ‘tetrahymanol strain diet' (Tet; n = 6 independent sensors in different hives), ‘W29 strain diet' (WT; n = 5 independent sensors in different hives), or ‘no yeast base diet' (Base; n = 5 independent sensors in different hives)), and in the glasshouse (ambient; n = 5 independent sensors). Lines represent the daily mean recordings. Ribbons depict the average maximum and minimum daily values. Note: extreme high temperatures occurred during days 26–29 and during days 49–55 which impacted colony brood production. a, Brood production (count of capped cells/colony) was equal across all four treatments (0, 1, 2, 3, x-axis) at the start of the experiment (one-way ANOVA, F3, 26 = 0.594, P = 0.625). Colonies starved for 1, 2 or 3 weeks (x-axis) succumbed to seasonal reduction in brood production at a rate faster than the control (two-way ANOVA, time*treatment, F6,77 = 5.06, P < 0.001). The x-axis indicates the treatment (i.e. the number of weeks that the colonies were starved of pollen, 0, 1, 2, 3) over the three periods of the experiment (Start, Starvation period where no pollen was provided, and Recovery period of 4 weeks of pollen feeding or 6 weeks of pollen feeding. These data indicate that even when starved colonies are provided with pollen, it is not sufficient for them to recover from previous nutrient deprivation. * indicates a significant post-hoc comparison within time point of treatment vs control (P < 0.01). n = 8 biologically independent colonies for the 1 and 3 week starvation treatments, n = 7 biologically independent colonies for the pollen and 2 week starvation treatments. a, Hive weight varied significantly with bee seams (two-way Gaussian GLM, χ2(1) = 83.9, P < 0.0001) and diet treatment group (χ2(3) = 12.9, P = 0.0049, ‘mixed-sterol strain diet' (MxSt), ‘tetrahymanol strain diet' (Tet), ‘W29 strain diet' (WT), or ‘no yeast base diet' (Base); n = 6 biologically independent colonies). b, Consumption rate varied significantly with hive weight (two-way Gaussian GLM, χ2(1) = 85.5, P < 0.0001) and diet treatment group (χ2(3) = 15.1, P = 0.0018; n = 6 biologically independent colonies). c, Total sterol per nurse gut varied significantly with normalised consumption rate (two-way Gaussian GLM, χ2(1) = 43.6, P < 0.0001) and diet treatment group (χ2(3) = 36.2, P < 0.0001; n = 6 biologically independent colonies). d, Total sterol per nurse body did not vary significantly with normalised consumption rate (two-way Gaussian GLM, χ2(1) = 0.844, P = 0.358) but did vary with diet treatment group (χ2(3) = 23.1, P < 0.0001; n = 6 biologically independent colonies). There was a significant interaction between total sterol per nurse body and diet treatment group (two-way Gaussian GLM, χ2(3) = 9.06, P = 0.0285; n = 6 biologically independent colonies). f, Capped brood cell counts did not vary with total sterol per nurse (two-way negative binomial GLM, χ2(1) = 1.08, P = 0.300) or diet treatment group (χ2(3) = 1.83, P = 0.609; n = 6 biologically independent colonies). The Pearson correlation coefficient (R) and P-values are shown. Subset of pollen sterol composition data from Zu et al. (2021) from bee-pollinated plants. Statistical analysis on the sterol composition of samples taken from colonies used during the yeast-feeding trial. Hive temperature and humidity recordings taken during the yeast-feeding trial. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Moore, E., de Sousa, R.T., Felsinger, S. et al. Engineered yeast provides rare but essential pollen sterols for honeybees. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09433-w'>Targeting G1–S-checkpoint-compromised cancers with cyclin A/B RxL inhibitors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 15:57:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Small-cell lung cancers (SCLCs) contain near-universal loss-of-function mutations in RB1 and TP53, compromising the G1–S checkpoint and leading to dysregulated E2F activity1. Other cancers similarly disrupt the G1–S checkpoint through loss of CDKN2A or amplification of cyclin D or cyclin E, also resulting in excessive E2F activity2,3. Although E2F activation is essential for cell cycle progression, hyperactivation promotes apoptosis4,5,6,7,8,9, presenting a therapeutic vulnerability. Cyclin proteins use a conserved hydrophobic patch to bind to substrates bearing short linear RxL motifs10,11,12,13. Cyclin A represses E2F through an RxL-dependent interaction10,14, which, when disrupted, hyperactivates E2F15. However, this substrate interface has remained difficult to target. Here we developed cell-permeable, orally bioavailable macrocyclic peptides that inhibit RxL-mediated interactions of cyclins with their substrates. Genetic screens revealed that cyclin A/Bi induces apoptosis through cyclin B- and CDK2-dependent spindle assembly checkpoint activation. Notably, cyclin A/Bi promoted the formation of neomorphic cyclin B–CDK2 complexes, which drive spindle assembly checkpoint activation and mitotic cell death. Finally, orally administered cyclin A/Bi showed robust anti-tumour activity in chemotherapy-resistant SCLC patient-derived xenografts. These findings reveal gain-of-function mechanisms through which cyclin A/Bi triggers apoptosis and support their development for E2F-driven cancers. SCLC is an aggressive lung cancer subtype lacking effective targeted therapies16. It is genomically defined by near-universal loss-of-function (LOF) mutations in RB1 and TP53, without recurrent druggable oncogenic drivers1, making new therapeutic strategies particularly challenging. One approach is to identify synthetic lethal vulnerabilities associated with RB1 and/or TP53 LOF. Although activating E2Fs are necessary for cellular proliferation, too much E2F activity paradoxically induces apoptosis4,5,6,7,8,9, suggesting a Goldilocks phenomenon. E2F1 is further negatively regulated by phosphorylation during S phase through cyclin A–CDK2, which limits its DNA binding14,19,20,21. Studies from budding yeast to mammalian cells demonstrated that cyclin substrate specificity is determined by a hydrophobic patchon cyclins recognizing short linear motifs on cyclin target proteins10,11,12,13. The hydrophobic patch of cyclin A recognizes and binds to an RxL motif on E2F1 to initiate cyclin A–CDK2-mediated E2F phosphorylation10,14. Disrupting this interaction using tool peptides induces apoptosis selectively in cells with deregulated pRB and increased E2F activity15. Despite efforts, small-molecule inhibitors of cyclin A–RxL interactions have not progressed beyond early stages25, reflecting the challenges of targeting intracellular protein–protein interactions. Macrocyclic peptides can overcome these challenges26,27,28,29 but often lack drug-like properties30. Here we report the development of orally bioavailable macrocyclic peptide inhibitors of cyclin A/B–RxL interactions, elucidate their mechanism of inducing apoptosis in SCLCs with high E2F activity and demonstrate their efficacy in SCLC models. To design passively permeable macrocyclic inhibitors targeting RxL-motif interactions with the hydrophobic patch of cyclins (Fig. 1a), we built a structural model using macrocycle-bound (Protein Data Bank (PDB): 1URC) and p27Kip1-bound (PDB: 1JSU) cyclin A2–CDK2 complexes (Extended Data Fig. 1b) was used as the starting scaffold. a, The structure of cyclin A (PDB: 1JSU), coloured by surface charge (blue, basic; red, acidic; white, neutral). e, Hallmark pathways associated with CIRc-004 sensitivity (false discovery rate (FDR) < 0.05) in 42 SCLC cell lines, calculated by GSVA using MSigDb Hallmark RNA-seq data (left). Each column represents a distinct SCLC cell line, ranked from most to least sensitive. Right, comparison of CIRc-004 GI50 values between high versus low E2F targets or G2M pathway groups (threshold, median pathway score; Mann–Whitney t-test). f, Dose–response curves of the indicated SCLC (NCI-H1048, NCI-H446, NCI-H69, NCI-H82), NSCLC (A549, HCC4006, NCI-H1299) and human non-transformed RPE1 cell lines treated with CIRc-004 for 6 days. The arrows indicate DMSO controls (used for normalization). g, Representative flow cytometry plots of cleaved PARP in NCI-H1048 cells treated with CIRc-004 (200 nM) or DMSO for 3 days. i, The cell cycle distribution after 24 h treatment with CIRc-004 or DMSO followed by propidium iodide staining. For f, h and i, n = 3 biological replicates. Scaffold optimization used peptidic and peptidomimetic features, with permeability-enhancing modifications guided by Circle Pharma's synthetic and computational platforms (Methods). This iterative approach yielded a series of passively permeable macrocycles with high affinity and diverse cyclin selectivity profiles (Fig. CIRc-004 bound to both the hydrophobic patch and smaller adjacent hydrophobic pockets on cyclin A2–CDK2, as shown by space-filling and stick representations (Fig. 1d); additional compound properties are shown in Extended Data Fig. The medicinal chemistry effort to identify passively permeable cyclin RxL inhibitors has been described elsewhere31. 1g), we first tested these compounds for anti-proliferative activity in a limited panel of SCLC cell lines. Although we initially hypothesized that cyclin A RxL inhibition alone would be sufficient, cytotoxicity unexpectedly required inhibition of both cyclin A and B RxL interactions. Dual inhibitors, including CIRc-004, showed the strongest anti-proliferative activity, whereas a cyclin-A-selective RxL inhibitor had little effect (Fig. Non-transformed WI-38 fibroblasts were insensitive to cyclin A/B inhibition. We next screened 46 SCLC cell lines with the dual cyclin A/B RxL inhibitor CIRc-004 in anti-proliferation assays (Fig. Nearly all lines were highly sensitive to CIRc-004 at low nanomolar concentrations. Gene set enrichment (GSEA) and variation (GSVA) analyses33 using MSigDb Hallmark gene sets34 showed strong correlation between CIRc-004 sensitivity and expression of E2F targets and G2/M checkpoint pathway genes, independent of the SCLC subtype (Fig. To determine broader applicability and ensure that cyclin A/B RxL inhibitors were not generally toxic, we screened the Horizon OncoSignature panel of 302 cancer cell lines using CIRc-001, a broader cyclin A/B/E RxL inhibitor (Fig. Approximately 180 cell lines, including lung, breast and ovarian models, showed selective sensitivity to CIRc-001 (half-maximum effective concentration (EC50) < 1 µM) (Extended Data Fig. E2F target gene signatures were enriched in sensitive lines across cancer types, whereas epithelial–mesenchymal transition -associated signatures were enriched in resistant lines (Extended Data Fig. Thus, low E2F activity and high expression of genes associated with epithelial–mesenchymal transition may predict resistance to cyclin A/B RxL inhibition, regardless of the tumour type. CIRc-004 sensitivity was not due to differences in proliferation rates (Extended Data Fig. 2c) and all lines were resistant to the inactive enantiomer CIRc-005 even at high micromolar concentrations (Extended Data Fig. In sensitive SCLCs, CIRc-004 induced apoptosis (Fig. These effects required inhibition of both cyclin A and B, as selective inhibitors CIRc-018 (cyclin A) and CIRc-019 (cyclin B) lacked activity in proliferation, apoptosis and cell cycle assays (Extended Data Fig. Normal cells, including RPE1 and CD34+ haematopoietic stem and progenitor cells, were insensitive to CIRc-004, showing no mitotic arrest or apoptosis, even at high doses, consistent with a potentially large therapeutic window (Fig. 1f,h,i and Extended Data Fig. Instead, CIRc-004 induced p53 expression and G1 arrest at micromolar levels (Extended Data Fig. Lastly, NCI-H82, an SCLC line with lower E2F signatures, was also resistant to CIRc-004, with an EC50 comparable to resistant NSCLCs and RPE1 cells (Fig. To elucidate the downstream mechanisms by which cyclin A/B RxL inhibitors induce apoptosis, we conducted a genome-wide CRISPR–Cas9 resistance screen in the highly sensitive NCI-H1048 SCLC cell line. Cells were transduced with the Brunello sgRNA library and, on day 10 (early timepoint, ETP), were pooled into four treatment arms: (1) dual cyclin A/B RxL inhibitor (CIRc-004); (2) broader cyclin A/B/E RxL inhibitor (CIRc-001); (3) selective orthosteric CDK2 inhibitor (CDK2i; PF-07104091); and (4) the inactive enantiomer CIRc-005 as a control (Fig. Compounds were dosed at EC90 concentrations (Extended Data Fig. After 16 days of drug exposure (day 26, late timepoint, LTP), resistant populations emerged and were collected for deep sequencing. Notably, hits from CIRc-004 and CIRc-001 were largely distinct from those enriched under CDK2i, suggesting a distinct mechanism of action (Fig. Among the top enriched hits for CIRc-004 was cyclin B itself (encoded by CCNB1), providing evidence for cyclin B as a likely target of CIRc-004, but also indicating that cyclin B is required for cell death, and may therefore act through a gain-of-function mechanism. CDK2 was the only shared hit across all three treatments, supporting its central role but again suggesting that cell killing may involve a gain of CDK2 function. a, Schematic of the genome-wide CRISPR–Cas9 knockout resistance screen in NCI-H1048 cells transduced with the Brunello sgRNA library. c, The top enriched hits (q < 0.25) across treatments. Right, STRING network (https://string-db.org; CC BY 4.0)66 of shared hits highlights SAC proteins. d–g, Dose–response curves of NCI-H1048 cells expressing two non-targeting sgRNAs (sgCtrl) or two sgRNAs targeting CCNB1 (d), CDK2 (e), KNTC1 (f) or MAD1L1 (g), treated with CIRc-004 for 6 days. h, Flow cytometry quantification of cleaved PARP in cells from d–g treated with CIRc-004 (20 nM). Data are the fold change relative to vehicle. i, Immunoblot of the indicated SCLC and NSCLC cell lines treated for 24 h with increasing doses of CIRc-004. j, RNA-seq analysis of NCI-H1048 cells treated for 24 h with CIRc-004 (200 nM), CIRc-005 (200 nM) or PF-07104091 (500 nM); the top 150 differentially expressed genes are shown (adjusted P (Padj) < 0.05). k, Immunoblot analysis of p-KNL1 in NCI-H1048 cells treated with CIRc-004 (20 nM), BAY-1217389 (MPS1i, 3 nM) or both for 24 h. l,m, CIRc-004 dose–response for proliferation (l) and cleaved PARP FACS assays (m) in the presence or absence of 3 nM BAY-1217389 for 3 days in NCI-H1048 cells. n, Immunoblot analysis of NCI-H1048 cells expressing sgCtrl or CCNB1 sgRNAs treated with CIRc-004 (20 nM) or DMSO for 24 h (n = 3 biological replicates). o, Cyclin B1 immunoprecipitations from NCI-H1048 cells treated for 2 h with CIRc-004 (004), CIRc-028 (028), CIRc-005 (005), CIRc-018 (018) or CIRc-019 (019) (300 nM); MYT1 binding normalized to cyclin B1 is shown. Statistical significance in h was calculated using unpaired two-tailed Student's t-tests. Notably, many hits enriched under CIRc-004 and CIRc-001 mapped to the spindle assembly checkpoint (SAC), including KNTC1, ESPL1, ZWINT, ZWILCH, ZW10 and MAD1L1 (Fig. 3h,j,k and Supplementary Table 3), indicating that SAC integrity is required for cell killing by cyclin A/B RxL inhibition. By contrast, these were not enriched under CDK2i (Extended Data Fig. In contrast to the cyclin A/B RxL inhibitor screen, CCNB1 was depleted in the CDK2i screen, consistent with cyclin B suppression enhancing CDK2i-induced death. To validate the CRISPR screen, we performed knockouts of CCNB1, CDK2, KNTC1, LIN54, MAD1L1 and ZWINT in NCI-H1048 cells, all of which conferred robust resistance to CIRc-004 (Fig. Similar resistance was confirmed in NCI-H446 SCLC cells for a subset of hits (Extended Data Fig. Lastly, knockout of CCNB1, KNTC1, MAD1L1 or ZWINT rescued mitotic arrest and apoptosis induced by CIRc-004 (Fig. To orthogonally validate these findings, we used a forward genetic screen in iHCT116 cells, a colorectal cancer model engineered with inducible degradation of MLH1 using indole acetic acid (IAA), enabling high (mut-high, +IAA) or low (mut-low, −IAA) mutation rates depending on IAA treatment37. CIRc-004 suppressed proliferation in iHCT116 cells (Extended Data Fig. After barcoded iHCT116 cells were cultured for an extended period with CIRc-004, resistant clones were isolated from both mut-low and mut-high populations (Extended Data Fig. These clones were over 100-fold more resistant than parental cells and remained sensitive to an unrelated toxin (MLN4924), ruling out non-specific resistance (Extended Data Fig. Exome sequencing of eight resistant clones revealed CDC20 mutations in five (Extended Data Fig. Notably, two mutations affected Arg445, and all four unique mutations mapped to the same surface of CDC20 in the crystal structure where CDC20 interacts with the mitotic checkpoint complex to promote SAC activation38,39,40,41 (Extended Data Fig. These heterozygous mutations are probably dominant negatives, allowing SAC bypass and anaphase progression in the presence of CIRc-004. Ectopic expression of CDC20(R445Q), but not wild-type CDC20, in both iHCT116 and NCI-H1048 cells conferred resistance to CIRc-004 (Extended Data Fig. Together, these complementary CRISPR knockout and forward genetic screens converge on the spindle assembly checkpoint as the critical mechanism through which cyclin A/B RxL inhibitors induce mitotic arrest and apoptosis in SCLC and other cancer models. To further determine how CIRc-004 induces mitotic arrest and apoptosis, we performed live-cell imaging of NCI-H1048 cells expressing GFP–H2B. CIRc-004 blocked anaphase initiation, consistent with SAC activation causing mitotic cell death (Extended Data Fig. SAC components assemble on the scaffold protein KNL1, which is phosphorylated by MPS1 in early mitosis42. As KNL1 is dephosphorylated after satisfaction of SAC, persistent phosphorylated KNL1 (p-KNL1) serves as a SAC activation marker. CIRc-004 robustly increased p-KNL1 levels in sensitive, but not resistant, cell lines at apoptosis-inducing concentrations (Figs. RNA-sequencing (RNA-seq) analysis of NCI-H1048 cells acutely treated with CIRc-004 revealed upregulation of SAC-related genes (such as CCNB1 and CDC20) and mitotic signatures, while the inactive enantiomer CIRc-005 had minimal transcriptional impact (Fig. 6k,l), consistent with their relative resistance (Fig. 1f,h,i and Extended Data Fig. In contrast to cyclin A/B RxL inhibitors, CDK2 inhibition using PF-07104091 did not activate the SAC in NCI-H1048 cells (Extended Data Fig. 7a), but instead reduced S-phase progression and led to accumulation of 4N cells undergoing DNA replication, consistent with a late S-phase/early G2 arrest (Extended Data Fig. Consistent with this, PF-07104091 treatment upregulated S-phase-associated genes, including CDC6 and CCNE1 (Extended Data Fig. We hypothesized that chemical inhibition of MPS1 would phenocopy genetic SAC disruption. Using a sublethal dose of MPS1 inhibitor BAY-1217389 (3 nM), which blocked KNL1 phosphorylation without affecting proliferation (Fig. Similarly, expression of CDC20(R445Q), but not wild-type CDC20, abolished CIRc-004-induced SAC activation in iHCT116 and NCI-H1048 cells (Extended Data Fig. These findings confirm that SAC activation is essential for the apoptotic activity of cyclin A/B RxL inhibitors. Cyclin B emerged as a top enriched hit in the CIRc-004 resistance screen, suggesting its activity is essential for the apoptotic effect of cyclin A/B RxL inhibitors (Fig. This is paradoxical if CIRc-004 merely blocks cyclin B function, instead indicating a gain of cyclin B activity. Supporting this, increased cyclin B–CDK1 activity can drive SAC activation40,43, and CIRc-004 increased cyclin B activity as measured by phosphorylation at Ser12644,45 (Extended Data Fig. 2n), demonstrating that cyclin B activity is necessary for SAC activation. We therefore hypothesized that CIRc-004 disrupts an interaction between cyclin B and an RxL-containing substrate that normally functions as a negative regulator of cyclin B–CDK1 activity. One such substrate is MYT1 (encoded by PKMYT1), a kinase that inhibits cyclin B–CDK1 activity by phosphorylating CDK1 at Thr1446. Co-immunoprecipitation experiments showed that dual cyclin A/B RxL inhibitors and the cyclin B-selective inhibitor CIRc-019 disrupted the cyclin B–MYT1 interaction, whereas the cyclin-A-selective inhibitor CIRc-018 and inactive enantiomer CIRc-005 did not (Fig. This was selective for CIRc-018 as CIRc-004 and CIRc-019 did not (Extended Data Fig. WEE1 is another kinase that negatively regulates CDK1 activity48. Although a smaller fraction of WEE1 appears to co-immunoprecipitate with cyclin B, CIRc-004 and CIRc-019 also disrupted this interaction (Extended Data Fig. However, the selective WEE1 inhibitor MK1775 did not hypersensitize NCI-H1048 cells to CIRc-018 (Extended Data Fig. Together, these data show that cyclin B RxL inhibition can disrupt either a MYT1 or WEE1 interaction with cyclin B and that disruption of the cyclin B–MYT1 interaction is functionally important for cell killing by cyclin A/B RxL inhibitors leading to CDK activation on the cyclin B complex to promote SAC activation and mitotic cell death. To further investigate how CIRc-004 induces mitotic cell death, we performed a CRISPR–Cas9 base editor screen in NCI-H1048 cells using A>G and C>T base editors49 and a custom sgRNA library tiling CCNB1, CCNA2, CDK2 and CDC20 (Fig. Sequencing confirmed strong replicate correlation and robust depletion of essential gene controls (Supplementary Fig. Consistent with our forward genetic screen (Extended Data Fig. 5f–m), sgRNAs predicted to mutate CDC20 at Arg445 were enriched (Supplementary Fig. Notably, sgRNAs targeting CCNB1 residues 169–177—where cyclin B canonically binds CDK150—were also enriched (Fig. We then repeated the screen and performed focused amplicon sequencing on CCNB1 amino acid residues 117–285. Amplicon sequencing directly validated several CCNB1 mutants (Glu169Lys, Tyr170His and Tyr177Cys) that conferred CIRc-004 resistance, all located within the region where cyclin B interacts with CDK1 (Fig. Additional enriched mutations were located in the hydrophobic patch of cyclin B where CIRc-004 binds (Fig. Fluorescence-associated cell sorting (FACS)-based competition assays confirmed that mutations disrupting either the cyclin B–CDK interface or the hydrophobic patch conferred resistance (Fig. Amplicon sequencing of mixed cell populations from these assays identified the precise base editor-induced CCNB1 mutations, with the fraction of cells carrying Glu169Lys, Tyr170His or Tyr177Cys substantially enriched after CIRc-004 treatment (Fig. a, Schematic of the base-editor (BE) CRISPR resistance screen in NCI-H1048 cells transduced with an sgRNA library tiling CCNB1, CCNA2, CDK2, CDC20 and controls. After selection (day 10), cells were treated with CIRc-004 (200 nM) or inactive enantiomer CIRc-005 (200 nM) and collected at day 26. b, The average z-scored log-transformed fold change (LFC) of sgRNAs tiling CCNB1 in CIRc-004 versus CIRc-005 in A>G (top) and C>T (bottom) base-editor screens. Mutations are labelled by predicted amino acid changes. e, FACS-based competition assays for the indicated CCNB1 sgRNA variants in GFP+ NCI-H1048 base-editor cells mixed with parental base-editor counterparts at 1:4 and treated with CIRc-004 for 13 days; the fold change enrichment over DMSO is shown. Statistical significance was calculated using unpaired two-tailed Student's t-tests. f, The fraction of variant alleles in CCNB1-mutant lines from e after CIRc-004 (200 nM) or DMSO treatment for 13 days was analysed using deep amplicon sequencing. g, Volcano plot of cyclin B Immunoprecipitation–mass spectrometry in NCI-H1048 cells treated for 2 h with CIRc-004 (50 nM) relative to CIRc-005. Statistical significance was calculated using two-sample Student's t-tests corrected for multiple hypothesis testing using permutation-based FDR (5%). h,i, Immunoblots after cyclin B1 immunoprecipitation (IP) from NCI-H1048 cells (h) or HA-tagged wild-type (WT) CCNB1 versus CCNB1 triple-mutant expressed in HEK293T cells (i) treated with CIRc-004 (300 nM) or DMSO. CDK2 band intensities were normalized to cyclin B1. For h and i, the fold increase in CDK2 relative to DMSO is shown. j, Immunoblot of NCI-H1048 cells infected with CDK2 sgRNAs or sgCtrl and treated with CIRc-004 (20 nM) or DMSO for 24 h. k, Immunoblot of NCI-H69 cells synchronized in mitosis with nocodazole (60 ng ml−1) and released into CIRc-004 (20 nM), PF-07104091 (500 nM) or both. Top, propidium iodide (PI) histogram showing 16 h post-nocodazole synchronization. For j and k, n = 3 biological replicates. To examine protein-level effects, we performed cyclin B immunoprecipitation followed by mass spectrometry. Consistent with our experiments above (Fig. 2o), one of the top depleted proteins after treatment with CIRc-004 was MYT1 (encoded by PKMYT1) and known cyclin–CDK RxL-dependent inhibitors including p27Kip1 and p57Kip2 (encoded by CDKN1B and CDKN1C, respectively)14 (Fig. Notably, a top enriched protein after CIRc-004 treatment was CDK2 (Fig. An AlphaFold model supported binding between cyclin B residues 169–177 and CDK2 (Extended Data Fig. Immunoprecipitation experiments for cyclin B in NCI-H1048, NCI-H446 and RPE1 cells confirmed that CIRc-004 increased the co-immunoprecipitation of CDK2 without affecting CDK1 (Fig. CIRc-004 did not alter the immunoprecipitation/co-immunoprecipitation interaction between cyclin A and CDK2 (Extended Data Fig. CIRc-004 also induced phosphorylation of the CDK mitotic RxL-independent substrate stathmin (STMN1)53, which was blocked by a CDK2i, but not by a CDK1i (Extended Data Fig. Supporting this, STMN1 depletion sensitized cells to CIRc-004 in our CRISPR knockout screen (Fig. 3h,j,k), suggesting that CIRc-004-induced CDK2-dependent stathmin phosphorylation promotes sensitivity to cyclin A/B RxL inhibitors. We next tested whether the cyclin B triple mutant (Glu169Lys/Tyr170His/Tyr177Cys) blocked CDK2 binding. Co-immunoprecipitation experiments showed that CIRc-004 induced an interaction between CDK2 and wild-type cyclin B that was abrogated in cells expressing the cyclin B triple mutant (Fig. Moreover, CDK2 inactivation in NCI-H1048 cells partially abrogated CIRc-004-induced SAC activation (Fig. Lastly, acute CDK2i also abrogated SAC activation by CIRc-004 after release from mitotic arrest with nocodazole further validating that CDK2 activity is necessary for SAC activation by CIRc-004 (Fig. These data, together with CDK2 being a top enriched hit in our CRISPR knockout screen, demonstrate that cyclin A/B RxL inhibitors promote the redirection of CDK2 to cyclin B to form cyclin B–CDK2 complexes to promote SAC activation and mitotic cell death. Having established the role of cyclin B RxL inhibition in the CIRc-004 mechanism of action, we next explored the contribution of cyclin A RxL inhibition. Cyclin A is expressed and canonically associates with CDK2 in S phase55. SAC is normally activated during mitosis when there is improper attachment of kinetochores to the mitotic spindles, which can be caused by DNA damage and replication stress that occurs during S phase40. Moreover, CIRc-004 also induced replication stress markers, including γH2AX, p-RPA2 and p-KAP1, specifically in sensitive SCLC lines (Fig. CIRc-004 induced γH2AX irrespective of whether cells had the ability to form SAC complex and this required both cyclin A and cyclin B RxL inhibition (Extended Data Fig. a, FACS analysis of EdU and FxCycle staining in NCI-H1048 and NCI-H446 cells treated for 24 h with increasing CIRc-004 doses; the EdU+ 4C population is indicated. b, Immunoblot of the indicated SCLC and NSCLC lines treated for 3 days with CIRc-004. c, Immunoblot after cyclin A immunoprecipitation in NCI-H1048 cells treated for 2 h with CIRc-004, CIRc-028, CIRc-005, CIRc-018, CIRc-019 or DMSO (300 nM each). The E2F1 band intensity was normalized to cyclin A; the fold change relative to DMSO shown. d–e,g, Immunoblots of NCI-H1048 cells expressing doxycycline (DOX)-inducible E2F1-sgRNA-resistant cDNA and endogenous E2F1-targeting sgRNA, treated with or without DOX and then with CIRc-004 (20 nM) for 24 h (d,g) or 72 h (e). f,h,i, Dose–response assays of NCI-H1048 (f), MDA-MB-231 (h) and Jurkat (i) cells with or without DOX for 24 h, followed by CIRc-004 treatment for 3 or 6 days. j,k, Immunoblots of MDA-MB-231 and Jurkat cells from the experiments in h and i. l, FACS analysis of EdU and 4′,6′-diamidino-2-phenylindole (DAPI) staining in DOX-induced E2F1-expressing NCI-H1048 cells; the indicated populations are shown with the gates. m, Quantification of S-phase and EdU+ 4C populations from l. Data are mean ± s.d. Statistical significance was calculated using unpaired two-tailed Student's t-tests. For a, b and d–m, n = 3 biological replicates. For histone blots in e and k, total histone H3 was run as a sample-processing control on a separate gel. High E2F activity strongly correlated with sensitivity to CIRc-004 in SCLC lines (Fig. 2b), suggesting that heightened E2F activity could enhance cyclin A/B RxL inhibitor sensitivity. We therefore hypothesized that cyclin A RxL inhibition blocks the RxL-dependent interactions of cyclin A with activating E2Fs (E2F1, E2F2 and E2F3), leading to heightened E2F activity. E2F2 was undetected, probably due to low abundance in NCI-H1048 cells. Overexpression of E2F1 in NCI-H1048, MDA-MB-231, NCI-H82 and Jurkat cells enhanced sensitivity to cyclin A/B RxL inhibitors with a corresponding increase in γH2AX and SAC activation after drug treatment in the cell lines examined (Fig. Similarly, E2F2 or E2F3 overexpression also sensitized cells to the cyclin A/B RxL inhibitor (Extended Data Fig. Consistent with this mechanism, both E2F1 overexpression and CDK2 inhibition led to persistent DNA replication during late S phase/early G2 (Fig. However, in contrast to CIRc-004, neither E2F1 overexpression nor CDK2 inhibition triggered mitotic arrest, highlighting the unique contribution of cyclin B RxL inhibition to cell killing. Together, these results support a model in which the cyclin A–E2F RxL interaction is selectively disrupted by cyclin A RxL inhibition causing E2F hyperactivation leading to S-phase dysregulation and replication stress, which is necessary for cell killing by cyclin A/B RxL inhibitors. We next performed in vivo studies first with a cyclin A/B RxL inhibitor tool compound, CIRc-028, with improved pharmacokinetic properties capable of being dosed intravenously (Fig. Administered daily at 100 mg per kg for 14 days, CIRc-028 induced tumour regressions in NCI-H69 xenografts (Fig. 5a), substantially inhibited growth in NCI-H1048 xenografts (Fig. 5b) and was well tolerated without weight loss (Extended Data Fig. Tumour analyses confirmed that CIRc-028 induced SAC activation and apoptosis in vivo (Fig. a,b, Tumour volume curves of NCI-H69 (a) and NCI-H1048 (b) xenografts in athymic nude mice treated with vehicle or cyclin A/B RxL inhibitor (CIRc-028) dosed at 100 mg per kg (mpk) intravenously (i.v.) Statistical analysis was performed using two-way analysis of variance (ANOVA); P < 0.0001. c,d, Representative IHC analysis of p-KNL1 (c) and cleaved caspase-3 (d) in NCI-H69 tumours 18 h after treatment. n = 5 tumours from independent mice per group. Statistical significance calculated using unpaired two-tailed Student's t-tests. g, Schematic for DFCI-393 and DFCI-402 PDX studies with oral cyclin A/B RxL inhibitor CIRc-014. h, mRNA expression of SCLC transcription factors and neuroendocrine markers by RNA-seq. i,j, Tumour volume curves of DFCI-393 (i) and DFCI-402 (j) PDXs treated with CIRc-014 (100 mg per kg orally three times per day (i) or twice per day (j)) or vehicle for 28 days. For j, n = 10 mice per arm. Statistical analysis was performed using two-way ANOVA; P < 0.0001. k,l, Representative IHC analysis of p-KNL1 (k) and cleaved caspase-3 (l) in DFCI-393 tumours treated with CIRc-014 (100 mg per kg orally three times per day) for 4 days. Statistical significance was calculated using unpaired two-tailed students t-tests. o, The proposed mechanism: in cancers with compromised G1–S checkpoint (for example, RB1 and TP53 loss) and consequently high E2F activity, cyclin A binds to activating E2Fs through RxL motifs to dampen E2F activity, and cyclin B binds to MYT1 to restrain CDK1 activity during G2. Cyclin A/B RxL inhibitors disrupt both interactions: (1) blocking cyclin A–E2F1 repression increases replication stress; (2) blocking cyclin B–MYT1 binding allows cyclin B to form a neomorphic complex with CDK2, driving RxL-independent phosphorylation (of, for example, stathmin), SAC activation and mitotic death. The diagrams in g and o were created with BioRender. We next developed a cyclin A/B RxL inhibitor, CIRc-014 (Extended Data Figs. Orally dosed CIRc-014 was tested in two patient-derived xenograft (PDX) models—DFCI-393 and DFCI-402—each derived from a patient with chemotherapy-resistant SCLC (Fig. RNA-seq analysis classified DFCI-393 as ASCL1+ and DFCI-402 as ASCL1+NEUROD1+, with both models clustering with neuroendocrine-high SCLC tumours1 (Fig. Notably, CIRc-014 caused near complete tumour growth inhibition in both chemoresistant SCLC PDX models (Fig. 5i,j) without weight loss relative to vehicle (Extended Data Fig. Pharmacokinetic analysis in mice dosed with CIRc-014 for 4 days in an independent pharmacokinetic/pharmacodynamic study showed that peak concentrations of unbound CIRc-014 (Cmax) were 141.6 nM, aligned with efficacious in vitro levels (Extended Data Fig. In contrast to CIRc-014—but as expected for an intravenously administered compound—the peak unbound concentration of CIRc-028 was higher, with a C0 of 4.9 µM for NCI-H69 and a C0 of 8.3 µM for NCI-H1048 xenografts studies (Extended Data Fig. Four days of CIRc-014 treatment induced SAC activation and apoptosis in DFCI-393 PDX tumours (Fig. RNA-seq analysis of these tumours showed upregulation of G2/M checkpoint and mitotic spindle gene sets, consistent with our in vitro findings and confirming on-target activity (Extended Data Fig. Together, these studies demonstrate that orally dosed CIRc-014 achieves potent, on-target anti-tumour efficacy in chemoresistant SCLC PDX models without overt toxicity. Here we describe the development of cell-permeable and orally bioavailable macrocycles that bind to cyclins to block their RxL-dependent interactions with RxL-containing substrates. We find that SCLCs with high E2F activity due to a compromised G1–S checkpoint from RB1 and TP53 loss are highly sensitive to macrocycles that inhibit the ability of both cyclin A and cyclin B to interact with their corresponding RxL-binding proteins, leading to SAC activation and mitotic cell death. Two unbiased orthogonal genome-wide genetic screens in cell lines from different cancer types both show that cell killing by cyclin A/B RxL inhibitors is achieved through activation of SAC, which is distinct from the mechanism of cell killing by an orthosteric CDK2i. Our data show that the efficacy of cyclin A/B RxL inhibitors is achieved by blocking the ability of cyclins to interact with at least two RxL-containing substrates including: (1) disruption of a cyclin A–E2F1 inhibitory interaction, leading to increased E2F activity, priming cells for SAC-induced apoptosis; and (2) disruption of a cyclin B–MYT1 RxL inhibitory interaction, which hyperactivates cyclin B–CDK activity, thereby increasing downstream SAC activation and mitotic cell death (the proposed mechanism is shown in Fig. While disruption of these two RxL interactions is necessary for cell killing by cyclin A/B RxL inhibitors, there are almost certainly other cyclin–RxL–substrate interactions10,14,56,57,58 disrupted by these macrocycles that contribute to their ability to induce apoptosis. We found that cyclin A/B RxL inhibitors promote a neomorphic interaction between cyclin B and CDK2, forming complexes that phosphorylate RxL-independent substrates such as stathmin54, leading to SAC activation and apoptosis (Fig. This finding explains why CCNB1 and CDK2 were among the top enriched hits in our CRISPR knockout screen. Canonically, cyclin A50,59 binds to and activates CDK2 during S phase, while cyclin B binds to and activates CDK1 during mitosis50. Although cyclin B–CDK2 binding has been observed in vitro51, to our knowledge, this is the first demonstration that such an interaction occurs in cells achieved through a cyclin A/B RxL macrocycle. Future studies will investigate how RxL macrocycles promote this complex and whether CDK2 activity in this setting is regulated by MYT1, which is likely given the high sequence homology between CDK1 and CDK2 at their N termini and the central role of cyclin–substrate interactions in determining specificity12,51,60. Despite cyclin A and cyclin B being essential for proliferation61,62, RxL macrocycles are well tolerated in vivo and selectively toxic to E2Fhigh cancer cells with 1,000-fold lower concentrations required to kill sensitive cancer cells relative to normal cell lines that were inherently resistant indicating a therapeutic window. We focused this study on SCLC as SCLCs have near-universal LOF mutations in RB1 and TP53, compromising cell cycle checkpoints leading to high and dysregulated E2F activity1. However, our findings probably extend beyond SCLC. Genomic alterations that disrupt the G1–S checkpoint—such as RB1 mutation1, CDKN2A deletion, or cyclin D or E amplifications2,3—are common across many cancer types and lead to E2F hyperactivation. Our data show that high E2F activity sensitizes diverse cancer cell lines to cyclin A/B RxL inhibition, suggesting that E2F expression signatures may serve as predictive biomarkers for therapeutic response. Moreover, this work aligns with previous studies showing SCLCs and other cancers with genomic alterations that compromise the G1–S checkpoint can be therapeutically exploited by disrupting mitotic checkpoint control47,63,64,65. In summary, we present cyclin A/B RxL macrocycles as a promising class of anti-cancer agents with a distinct mechanism from orthosteric CDK inhibitors. The orally bioavailable inhibitor CIRc-014 shows potent, on-target activity in chemoresistant SCLC PDX models without overt toxicity. Circle Pharma is advancing CID-078, a related compound, as a first-in-class cyclin A/B RxL inhibitor into phase 1 clinical evaluation (NCT06577987) to test the therapeutic potential of targeting cyclin–substrate interactions in SCLC and other cancers. NCI-H446, NCI-H69 cells, NCI-H1048, NCI-H526, NCI-H82, WI-38, RPE1 and HEK293T cells were originally obtained from American Type Culture Collection (ATCC) and authenticated by ATCC. NCI-H1048, Jurkat, NCI-H82 and MDA-MB-231 cells with DOX-on inducible E2F1 expression were a gift from W. G. Kaelin's laboratory at DFCI and were originally obtained and authenticated by ATCC prior to genetically manipulation. G-CSF mobilized human peripheral blood CD34+ cells were obtained from StemCell Technologies (700060.1, 230472503C) and validated by flow cytometry. No commonly misidentified cell lines were used in this study. NCI-H69, NCI-H446, HCC4006, NCI-H1299, NCI-H526, A549, NCI-H82 and Jurkat cells were maintained in RPMI-1640 medium supplemented with 10% fetal bovine serum (FBS) (Gemini), 100 U ml−1 penicillin and 100 μg ml−1 streptomycin. NCI-H1048 cells were cultured in RPMI-1640 medium supplemented with 10% fetal bovine serum (FBS), penicillin–streptomycin and insulin–transferrin–selenium (Gemini). WI-38 cells were maintained in DMEM supplemented with 10% FBS. HEK293T cells were maintained in DMEM medium with 10% FBS and penicillin–streptomycin. G-CSF mobilized human peripheral blood CD34+ cells were thawed into StepSpan Serum Free Expansion Medium (SFEM II) Hematopoietic Stem Cell Medium (StemCell Technologies, 09655) supplemented with StemSpan CD34+ Expansion Supplement (StemCell Technologies, 02691). Early-passage cells of all of the cell lines listed above were tested for mycoplasma (Lonza, LT07-218) and then were frozen using Bambanker's freezing medium (Bulldog Bio) and maintained in culture no more than 4 months in cases in which early-passage vials were thawed. iHCT116 cells were previously described37 and cultured in 10% fetal bovine serum in DMEM medium (Sigma-Aldrich) supplemented with 2 mM l-glutamine (Sigma-Aldrich). Macrocycle peptides offer a feasible strategy to block broad and shallow protein-protein interaction surfaces that until recently have been considered undruggable. Over the past decade, a growing body of research has demonstrated that peptidic macrocycles can be engineered to be both passively cell permeable67,68,69,70 and orally bioavailable26,28,71,72, therefore offering promise for this modality to access previously undruggable intracellular targets27. Our approach to identify passively permeable cyclin RxL inhibitors included the incorporation of peptidomimetic features based on our previous research67,68,73,74. Detailed descriptions of our medical chemistry efforts to achieve passive permeability, cyclin binding affinity and selectivity of an earlier series of distinct tool cyclin RxL macrocycles not included in this manuscript are reported in ref. Where indicated, the following chemicals (stored at −20 °C or −80 °C) were also added to the medium: CIRc-004 (cyclin A/B RxL inhibitor, stock 10 mM in DMSO, obtained from Circle Pharma), CIRc-005 (inactive enantiomer of CIRc-004, stock 10 mM in DMSO, obtained from Circle Pharma), CIRc-001 (cyclin A/B RxL inhibitor, stock 10 mM in DMSO, obtained from Circle Pharma), CDK2i (PF-07104091, stock 10 mM in DMSO, chemically synthesized and validated to be the same PF-07104091 in the published patent), CIRc-018 (cyclin-A-selective RxL inhibitor, stock 10 mM in DMSO, obtained from Circle Pharma), CIRc-019 (cyclin-B-selective RxL inhibitor, stock 10 mM in DMSO, obtained from Circle Pharma), MPS1 inhibitor (BAY-1217389, Selleck chemicals, S8215, stock 10 mM), doxycycline hydrochloride (Sigma-Aldrich, 3447, stock 1 mg ml−1), RP-6306 (MedChemExpress, HY-145817A, stock 20 mM), roscovitine (Selleck Chemical, S115350MG, stock 10 mM), staurosporine (MedChemExpress, HY-15141, stock 20 mM), nocodazole (Sigma-Aldrich, M1404, stock 2 mg ml−1), RO-3306 (ENZO, ALX-270-463-M001, stock 10 mM) and MK-1775 (Selleck Chemical, NC1097015, stock 10 mM). The binding potency of macrocycles for cyclin–CDK complexes was determined by a fluorescence polarization competitive assay based on previously established protocols25,75 with the following modifications: (1) a higher-affinity fluorescent-labelled probe was identified referred to as CIR7-2706 (Supplementary Methods); (2) cyclin–CDK protein complexes were sourced as follows: cyclin A2–CDK2 (CRELUX Protein Services; www.crelux.com), cyclin B1–CDK1 (Eurofins, Discovery, 14-450) and cyclin E1–CDK2 (Eurofins, Discovery, 14-475). Fluorescence polarization binding assays were performed in 25 mM HEPES pH 7.5, 100 mM NaCl, 1 mM DTT, 0.01% NP-40 and 1 mg ml−1 BSA for all 3 protein complexes in black 96-well plates (Costar, 3356). After experimental plates were set, they were equilibrated by gentle mixing by placing them on an orbital shaker at 100 rpm for 2 h at room temperature and then read on the SpectraMax i3X Multi-Mode Microplate Detection platform. The 50% inhibitory concentration (IC50) of binding of a fluorescence labelled probe CIR7-2706 at 2 nM to the cyclin–CDK complex was determined from an eight-point threefold serial dilution curve of CID-078. Under these conditions, the assay dynamic range was about 120 mP units between 100% binding of CIR7-2706 to the proteins and its complete displacement by an unlabelled competitor compound. All experiments showed a Z′ factor of >0.80. RxL inhibitor affinity determinations were performed in a WaveDelta instrument (Malvern Panalytical). Each of the cyclin–CDK complexes were immobilized to the chip surface by amine coupling on one of four channels of a 4PCP chip (9060001, Malvern Panalytical) in 10 mM MES pH 6.5 buffer for cyclin A2–CDK2 and cyclin E1–CDK2 and 10 mM MOPS pH 7.0 for cyclin B1–CDK1. For coupling, the proteins were diluted at 25 mg ml−1 and applied to the corresponding chip channel for 600 s at a flow rate of 10 ml min−1, followed by quenching by injecting 1 M ethanolamine-HCl pH 8.5 for 420 s at a flow rate of 10 ml min−1. An eight-point 1:3 serial dilution of the compounds was sequentially injected into the chip at flow rate of 100 ml min−1 for 120 s to measure the association rates, followed by injection for 600 s of buffer alone to measure the dissociation rates. Background readings were subtracted using an empty channel on the chip on which no protein was immobilized. The data were analysed using a 1:1 binding kinetic model using the instrument software (Wave Control, v.4.5.17). The reported Kd, ka and kd are the average of two or more experiments as indicated. ClogP values were calculated using the calculator in ChemDraw. MDCK monolayer cell permeability, kinetic solubility (Ksol), and logD experimental values were generated by Quintara Discovery using standard experimental conditions. sgRNA sequences were designed using the Broad Institutes sgRNA designer tool (http://portals.broadinstitute.org/gpp/public/analysis-tools/sgrna-design) and chosen from the Brunello CP0043 sgRNA library and synthesized by IDT technologies. The sense and antisense oligonucleotides were mixed at equimolar ratios (0.25 nmol of each sense and antisense oligonucleotide) and annealed by heating to 100 °C in annealing buffer (1× annealing buffer: 100 mM NaCl, 10 mM Tris-HCl, pH 7.4) followed by slow cooling to 30 °C for 3 h. The annealed oligonucleotides were then diluted at 1:400 in 0.5× annealing buffer. For CRISPR–Cas9 knockout experiments in cells, the annealed oligos were ligated into pLentiCRISPR Puro V2 (Addgene, 52961). Ligations were done with T4 DNA ligase for 2 h at 25 °C. The ligation mixture was transformed into HB101 competent cells. Ampicillin-resistant colonies were screened by restriction digestion of miniprep DNAs and subsequently validated by DNA sequencing. CDC20 (NM_001255.3) or CDC20 with an N-terminal 3×Flag tag was cloned into the pLVX IRES Hygro vector (Clontech). The CDC20 R445Q mutation was generated using site-directed mutagenesis (Pfu polymerase). pTwist Lenti SFFV Puro WPRE lentiviral vectors encoding: (1) CCNB1 wild-type-HA, (2) CCNB1 triple mutant-HA (E169K/Y170H/Y177C) or a (3) negative control that encodes a functionally inactivated EED mutant-HA with mutations to inactivate EED (F97A/Y148A/Y365A) were codon optimized and synthesized by Twist Biosciences. All plasmids were sequence verified by sanger sequencing. Purified E2F1, E2F2 and E2F3 double-stranded DNA fragments containing attB1 and attB2 sites were introduced into the pDONR223 vector by homologous recombination using BP clonase (Life Technologies, 11789020) for 1 h at 25 °C according to the manufacturer's instructions. The reaction mixture was then transformed at a ratio of 1:10 (reaction volume/volume competent cells) into HB101 competent cells (Promega). Spectinomycin-resistant colonies were screened by restriction digestion of miniprep DNA and subsequently validated by DNA sequencing. A homologous recombination reaction was then performed using LR Clonase II (Life Technologies, 11791100) with the pDONR223-E2F1, pDONR223-E2F2 or pDONR223-E2F3 plasmids and DOX-on pTripz-NEO DEST vector that was modified to be used as destination vectors for Gateway recombination cloning. The LR reaction was transformed into HB101 competent cells (Promega). Kanamycin-resistant (pTripz-NEO) colonies were screened by restriction digestion of miniprep DNA and subsequently validated by DNA sequencing. Lentivirus was then generated and then transduced into cells to make stable DOX-on cell lines for inducible E2F1, E2F2 or E2F3 expression. Lentiviruses were made by Lipofectamine-2000-based co-transfection of HEK293FT cells with the respective lentiviral expression vectors and the packaging plasmids psPAX2 (Addgene, 12260) and pMD2.G (Addgene, 12259) at a ratio of 4:3:1. Virus-containing supernatant was collected at 48 and 72 h after transfection, pooled together (7 ml total per 6 cm tissue culture dish), passed through a 0.45-µm filter, aliquoted and frozen at −80 °C until use. The cells were counted using a Vi-Cell XR Cell Counter (Beckman Coulter) and 2 × 106 cells were resuspended in 1 ml lentivirus with 8 μg ml−1 polybrene in individual wells of a 12-well plate. The plates were then centrifuged at 931g (Allegra X-15R Centrifuge (Beckman Coulter), rotor SX4750A, 2,000 rpm) for 2 h at 30 °C. Then, 16 h later, the virus was removed and cells were grown for 72 h before being placed under drug selection. RPE1 cells were first infected with lentivirus encoding an sgRNA targeting RB1 (sgRB1) or a non-targeting sgRNA (sgC0111, sgControl) in LentiCRISPR-V2 Puro vector. RPE1 sgRB1 or sgControl cells were superinfected with lentiviruses encoding an sgRNA targeting TP53 (sgTP53) or a non-targeting sgRNA (sgC0111, sgControl) and a G418 resistance gene in the pLentiCRISPR-V2 vector. Cells were selected in G418 (500 µg ml−1) until all mock-infected cells were killed and immunoblot analysis was performed for RB1 and p53 in the presence or absence of CIRc-004 at the concentrations indicated. For dose–response assays with cyclin A/B RxL inhibitor (CIRc-004), inactive enantiomer of CIRc-004 (CIRc-005) and CDK2i (PF-07104091), the cells were counted on day 0 using a Vi-Cell XR Cell Counter and plated in a tissue-culture-treated six-well plate at 50,000 cells per ml in 2 ml of complete medium. Normalized cell counts were calculated as (day 6 count for treated samples)/(day 6 count for DMSO treated sample). The EC50 was calculated using GraphPad Prism v.10.0.0. For DOX-inducible (DOX-on) E2F1-, E2F2- or E2F3-overexpressing cell lines, cells were counted on day 0 and plated in tissue-culture-treated six-well plates at 250,000 cells per ml for 3-day assays or 50,000 cells per ml in 2 ml of complete medium for 6-day assays. For 6-day cyclin A/B RxL inhibitor assays, E2F1 expression was induced with DOX for 24 h followed by drug treatment for 6 days, with replacement with fresh DOX every 3 days. NCI-H1048, NCI-H446 and WI-38 cells were counted using a Countess II FL (Invitrogen) and plated at 5,000 cells per well in 96-well plates. NCI-H69 cells were grown to confluency in a T150 flask. Cells were collected by centrifugation at 283g (Thermo Fisher Scientific X4R Pro-MD centrifuge, Sorvall TX1000 rotor, 1,100 rpm) and resuspended in 30 ml medium of which 10 ml was used to seed six 96-well plates. NCI-H526 cells were grown to confluency in a T75 flask, collected by centrifugation at 283g and resuspended in 10 ml medium of which 2.5 ml was used to seed six 96-well plates. Roscovitine and staurosporine were used as plate controls to define the top and bottom of the growth inhibition curves, respectively. Inhibitors were dosed in duplicate in either an 8-point (WI-38) or 10-point (SCLC cell lines) 1:3 serial dilution with 10 µM maximum concentration. Staurosporine was dosed in singlet in an 8- or 10-point 1:2 serial dilution with 1 µM maximum concentration. After dosing, plates were maintained in tissue culture incubators (37 °C, 5% CO2) for 3 days (WI-38) or 5 days (SCLC cell lines) to allow for at least two cell doublings before processing in an MTT proliferation assay (R&D systems, 4890-050-K) performed according to the manufacturer's instructions. The average absorbance value obtained with the highest two concentrations dosed for roscovitine and staurosporine was used for background subtraction. The GI50 was determined by nonlinear regression analysis using log(inhibitor) versus response-variable slope (four parameters) using GraphPad Prism (v.10.1.0). For MTT proliferation assays with the MYT1 inhibitor RP-6306, NCI-H1048, NCI-H446 and NCI-H69 cells were seeded in 96-well plates as described above. The next day, cells were dosed in duplicate in a 10-point 1:3 serial dilution with CIRc-018, CIRc-019, or CIRc-004 alone or in combination with RP-6306 at 100 nM for NCI-H1048 cells or 500 or 1,000 nM for NCI-H446 and 1,000 nM for NCI-H69 cells. After 5 days, cells were processed in an MTT proliferation assay and GI50 values were determined using GraphPad Prism (v.10.1.0) as described above. Viability of cells were quantified using Cell Titer Glo according to the manufacturer's instructions (Promega) and the IC50 values were determined using GraphPad Prism (log inhibitor versus response four parameters). Cells were plated at 250,000 cells per ml density and grown in cyclin A/B RxL inhibitor (CIRc-004), MPS1 inhibitor (BAY-1217389) or DMSO at the indicated concentrations for 72 h. Cells were then collected and fixed in 4% paraformaldehyde followed by a PBS wash, and then permeabilized in ice-cold 90% methanol (added dropwise) for 30 min on ice. The cells were then centrifuged at 206g (Eppendorf centrifuge 5804R, A-4-44 rotor, 2,000 rpm) washed once in PBS, centrifuged again at 206g and then incubated with Pacific blue conjugated cleaved PARP antibody (Cell Signaling, Asp214, D64E10, 60068) at a dilution of 1:100 for 1 h at room temperature protected from light. Cells were then washed again in PBS containing 0.5% BSA, resuspended in PBS containing 0.5% BSA and analysed on the LSR Fortessa flow cytometer (Becton Dickinson). Analysis for cleaved-PARP-positive cells was carried out using FlowJo v.10.8.1. The FACS gating strategy is provided in Supplementary Fig. Cells were plated at 500,000 cells per ml and grown in cyclin A/B RxL inhibitor (CIRc-004) or DMSO at the indicated concentrations for 24 h. The cells were then washed once in room temperature PBS and then fixed in 4% paraformaldehyde for 15 min at room temperature. The cells were then centrifuged at 206g (Eppendorf centrifuge, 5804R, A-4-44 rotor, 2,000 rpm) for 5 min at room temperature, washed once in PBS, centrifuged again at 206g and then permeabilized with ice-cold methanol at 4 °C for 30 min. The cells were then washed again in PBS and then incubated with Alexa-647-conjugated phospho-histone H3 (Ser10) antibody (Cell Signaling, 3458) at a dilution of 1:100 for 1 h at room temperature, then washed once in PBS containing 0.5% BSA, centrifuged at 400g and then counterstained with DAPI) (Cell Signaling, 4083) for 15 min at room temperature. FACS analysis was then performed to determine the percentage of positive p-histone H3 (Ser10) cells using FlowJo v.10.8.1. The FACS gating strategy is provided in Supplementary Fig. Cells were plated at 500,000 cells per ml and grown in cyclin A/B RxL inhibitor (CIRc-004) or DMSO at the indicated concentrations for 24 h. After incubation, cells were washed once in ice-cold PBS and then fixed in ice-cold 80% ethanol (added dropwise) for at least 2 h at −20 °C. The cells were then centrifuged at 206g (Eppendorf centrifuge 5804R, A-4-44 rotor, 2,000 rpm) for 5 min, washed once in PBS, centrifuged again at 206g and then washed again in PBS containing 0.5% BSA. Finally, cells were stained with propidium iodide (BD, 550825) for 15 min at room temperature. FACS analysis for PI was then performed on the LSR Fortessa flow cytometer (Becton Dickinson) and analysis was performed using FlowJo v.10.8.1. The FACS gating strategy is provided in Supplementary Fig. 7b) plate and allowed to adhere overnight. Cells were dosed with cyclin A/B RxL inhibitor (CIRc-004) or DMSO for 24 h. Cells were pulsed for 1 h with 10 µM EdU (Click-It Plus EdU Flow Cytometry Assay kit, Invitrogen, C10632) before collection. After incubation, cells were washed once with ice-cold PBS and fixed by incubation in ice-cold 80% ethanol for at least 2 h at −20 °C. EdU was fluorescently tagged with Alexa Fluor 488 by click reaction (Click-it Plus EdU flow cytometry assay kit, Invitrogen, C10632). DNA content was monitored by FxCycle Violet stain (Invitrogen, F10347) or 1 μg ml−1 DAPI (4,6-diamidino-2-phenylindole, Cell Signaling) for 5 min. 4a, flow cytometry analysis for EdU and DNA content was performed by full spectral flow cytometry on the Northern Lights cytometer (Cytek Biosciences) and analysis was carried out using FlowJo v.10.9.0. 7b, flow cytometry analysis for EdU and DNA content was performed using the LSR Fortessa flow cytometer (Becton Dickinson) and analysis was carried out using FlowJo v.10.8.1. FACS gating strategy is given in Supplementary Fig. G-CSF mobilized human peripheral blood CD34+ cells were plated in a 24-well plate at 25,000 cells per ml after counting with a VI-Cell BLU (Beckman Coulter). Cells were then allowed to recover for 48 h before dosing cells with CIRc-004, staurosporine (100 nM) or DMSO for 24 h. Cells were pulsed for 1 h with 10 µM EdU (Click-It Plus EdU Flow Cytometry Assay kit, Invitrogen, C10634) before collection. After incubation, cells were collected, washed once with PBS and resuspended in 1% BSA-PBS before staining with CD34 (8G12) Per-CP (BD, 340666) for 30 min at a 1:10 dilution protected from light. Cells were then washed in 1% BSA-PBS. Cells were then fixed by resuspending cells in 50 µl of Click-it Fixitive (4% PFA) and incubated for 15 min. Cells were then washed with 1% BSA-PBS and resuspended in permeabilization and wash buffer prepared as directed from Click-It Plus kit (Click-It Plus EdU Flow Cytometry Assay kit, Invitrogen, C10634) for 15 min. EdU was fluorescently tagged with Alexa Fluor 647 by click reaction (Click-It Plus EdU Flow Cytometry Assay kit, Invitrogen, C10634). Cells were then resuspended in permeabilization buffer and stained with FITC mouse anti-cleaved PARP (Asp214) (BD, 558576) for 1 h at a 1:5 dilution protected from light. DNA content was monitored by FxCycle Violet stain (Invitrogen, F10347). Flow cytometry analysis for EdU and DNA content was performed by full spectral flow cytometry on a Northern Lights cytometer (Cytek Biosciences) and analysis was carried out using FlowJo v.10.9.0. The FACS gating strategy is given in Supplementary Fig. Cell pellets were lysed in a modified EBC lysis buffer (50 mM Tris-Cl pH 8.0, 250 mM NaCl, 0.5% NP-40, 5 mM EDTA) supplemented with a protease inhibitor cocktail (Complete, Roche Applied Science, 11836153001) and phosphatase inhibitors (PhosSTOP, Sigma-Aldrich, 04906837001). Soluble cell extracts were quantified using the Bradford protein assay. Then, 20 µg of protein per sample was boiled after adding 3× sample buffer (6.7% SDS, 33% glycerol, 300 mM DTT and Bromophenol Blue) to a final concentration of 1×, resolved by SDS–PAGE using either 10% or 6% SDS–PAGE, semi-dry transferred onto nitrocellulose membranes, blocked in 5% milk in Tris-buffered saline with 0.1% Tween-20 (TBS-T) for 1 h, and probed with the indicated primary antibodies overnight at 4 °C. The membranes were then washed three times in TBS-T, probed with the indicated horseradish peroxidase (HRP)-conjugated secondary antibodies for 1 h at room temperature, and washed three times in TBS-T. Bound antibodies were detected with enhanced chemiluminescence (ECL) western blotting detection reagents (Immobilon, Thermo Fisher Scientific, WBKLS0500) or Supersignal West Pico (Thermo Fisher Scientific, PI34078). The secondary antibodies and dilutions were as follows: goat anti-mouse (Jackson ImmunoResearch, 115-035-003, 1:5,0000) and goat anti-rabbit (Jackson ImmunoResearch, 111-035-003, 1:5,000). To detect p-stathmin S38, total stathmin, p-cyclin B1 (Ser126), p-nucleolin (Thr84), p-RPA2 (Ser33) and p-FOXM1 (Thr600), after incubation with the indicated cyclin RxL inhibitors or DMSO control, NCI-H1048 cells were washed once in ice-cold PBS and lysed in 1% Triton X-100 buffer (40 mM HEPES pH 7.5, 1 mM EDTA pH 8.0, 10 mM sodium pyrophosphate, 10 mM β-glycerophosphate, 50 mM sodium fluoride, 120 mM sodium chloride and 1% Triton X-100) supplemented with a protease inhibitor cocktail (cOmplete, Sigma-Aldrich, 5892970001) and phosphatase inhibitors (PhosSTOP, Sigma-Aldrich, 4906837001). Cell lysates were centrifuged at 9,391g (Eppendorf centrifuge 5424R, FA-45-24-11 rotor, 10,000 rpm) for 10 min. The supernatant (cell extract) was removed and the protein content was estimated by performing a Bradford assay. Cell extracts (30 μg of protein) were boiled after adding 4× Laemmli buffer and separated by SDS–PAGE and transferred to PVDF membranes. Membranes were blocked in TBS plus 0.2% Tween-20 (TBS-T) containing 5% skimmed milk and incubated with relevant primary antibodies overnight at 4 °C in blocking buffer containing 5% BSA. After incubation with primary antibodies, the membranes were washed in TBS-T and then incubated with HRP-labelled secondary antibodies for 1–-2 h at room temperature. Membranes were washed again in TBS-T, incubated with ECL reagent (LI-COR, WesternSure, 92695000) and scanned on the Odyssey Fc Analyzer (LI-COR Biosciences). The secondary antibodies used and dilutions were as follows: goat anti-rabbit-IgG-HRP (Cell Signaling, 7074S, 1:5,000). Quantified values from immunoblots are provided in Supplementary Tables 8 and 9. Raw immunoblots are included as Supplementary Fig. 8u and 9d–f,h, NCI-H1048 cells were seeded at 2.5 × 107 cells per 15 cm plate and were allowed to adhere overnight. The next day, cells were incubated with the indicated cyclin RxL inhibitors at 300 nM or DMSO for 2 h before rinsing once in ice-cold PBS and lysing in 1% NP-40 buffer (40 mM HEPES pH 7.5, 1 mM EDTA pH 8.0, 10 mM sodium pyrophosphate, 10 mM β-glycerophosphate, 50 mM sodium fluoride, 120 mM sodium chloride and 1% NP-40) supplemented with a protease inhibitor cocktail (cOmplete, Sigma-Aldrich, 5892970001) and phosphatase inhibitors (PhosSTOP, Sigma-Aldrich, 4906837001). The cell lysates were centrifuged at 9,391g for 10 min. The supernatant (cell extract) was removed and the protein content was estimated by performing a Bradford assay. Then, 2 mg cell extract was rotated overnight at 4 °C with 2 µg anti-cyclin A (Santa Cruz Biotechnology (SCBT) sc-271682) or 2 µg anti-cyclin B (SCBT sc-245) antibodies. The next day, 20 µl packed protein-A- and protein-G-coated magnetic beads (Dynabeads Protein A and Dynabeads Protein G, Invitrogen, 10002D and 10003D, respectively) were added and lysates were rotated for an additional 1 h at 4 °C. The protein A/G beads were collected using a magnet, washed three times with 1% NP-40 cell lysis buffer, boiled and denatured in 1× Laemmli sample buffer, separated by SDS–PAGE and immunoblotting was performed as described above. The primary antibodies and dilutions used were as follows: anti-MYT1 (Fortis Life Sciences, A302-424A, 1:1,000), rabbit anti-WEE1 (Cell Signaling, 4936, 1:1,000), mouse anti-cyclin B (Cell Signaling, 4135 1:1,000), mouse anti-E2F1 (SCBT, sc-251, 1:1,000), mouse anti-cyclin A (Cell Signaling, 4656, 1:1,000), mouse anti-CDK2 (Origene, TA502935), rabbit anti-CDC2 (Cell Signaling 28439). The secondary antibodies used and dilutions were: mouse anti-rabbit (confirmation specific)-IgG HRP (Cell Signaling, 5127, 1:2,000), goat anti-rabbit IgG-HRP (Cell Signaling, 7074S, 1:5,000), horse anti-mouse IgG-HRP (Cell Signaling, 7076S, 1:5,000), goat anti-mouse-IgE-HRP (Southern Biotech, 1110-05, 1:5,000). For immunoprecipitations of cyclin B–HA and CDK2 in Fig. 9k, HEK293T cells were stably infected with pTwist Lenti SFFV Puro WPRE lentiviral vectors (SYNTHESIZED by Twist Biosciences) encoding (1) CCNB1 wild-type-HA, (2) CCNB1 triple mutant-HA (E169K/Y170H/Y177C) or a (3) negative control encoding a functionally inactivated EED mutant-HA with mutations to inactivate EED (F97A/Y148A/Y365A). In total, 2.0 × 107 cells were seeded at 1 × 106 cells per ml density and, the next day, were treated with the cyclin A/B RxL inhibitor (CIRc-004) at 300 nM or DMSO for 2 h before cell collection. Cells were lysed in 700 µl lysis buffer (Cell Signaling, 9803) supplemented with protease inhibitor cocktail (Complete, Roche Applied Science, 11836153001) and phosphatase inhibitors (PhosSTOP, Sigma-Aldrich, 04906837001). Samples were sonicated (Branson Digital Sonifier) three times for 5 s each to ensure homogenization and centrifuged at 14,000g for 10 min at 4 °C (Eppendorf centrifuge 5424R, FA-45-24-11 rotor). The supernatant was precleared by rotating 20 µl protein A agarose beads (Cell Signaling, 9863) for 1 h at 4 °C. The samples were then briefly centrifuged to remove the beads and protein concentration was determined. Then, 60 µl of lysate was removed as input (~10% total for ~2.5% input per immunoblot) and boiled after adding 3× sample buffer (6.7% SDS, 33% glycerol, 300 mM DTT and Bromophenol Blue) to a final concentration of 1×. Then 300 µl of lysate was immunoprecipitated by rotating overnight at 4 °C with rabbit anti-HA epitope (Cell Signaling, 3724, 1:100) for exogenous cyclin B, rabbit anti-CDK2 E8J9T (Cell Signaling, 18048, 1:100) or equivalent amount of rabbit (DA1E) monoclonal antibody IgG XP isotype control (Cell Signaling, 3900). The next day, 20 µl protein A agarose beads was added to each sample tube and incubated for 1 h at 4 °C with rotation. The beads were collected by centrifugation and washed five times with cold lysis buffer followed by boiling in 3× sample buffer. The samples were run on SDS–PAGE and immunoblotting was performed as described above. Primary antibodies used for immunoblotting were as follows: rabbit anti-cyclin B1 D5C10 (Cell Signaling, 12231, 1:1,000), rabbit anti-CDK2 E8J9T (Cell Signaling, 18048, 1:1,000) and mouse anti-HA.11 epitope tag (BioLegend, 901501, 1:1,000). For immunoprecipitation of endogenous cyclin A with E2F3 in Extended Data Fig. The next day, cells were incubated with the indicated cyclin RxL inhibitors at 300 nM or DMSO for 2 h before rinsing once in ice-cold PBS and lysing in 1% Triton buffer (40 mM HEPES pH 7.5, 1 mM EDTA pH 8.0, 10 mM sodium pyrophosphate, 10 mM β-glycerophosphate, 50 mM sodium fluoride, 120 mM sodium chloride and 1% Triton X-100) supplemented with a protease inhibitor cocktail (cOmplete, Sigma-Aldrich, 5892970001) and phosphatase inhibitors (PhosSTOP, Sigma-Aldrich, 4906837001). Cell lysates were centrifuged at 9,391g for 10 min. The supernatant (cell extract) was removed and the protein content estimated by performing a Bradford assay. Anti-cyclin A antibody (SCBT, sc-271682) was covalently coupled to M-270 Expoxy dynabeads (Dynabeads Antibody Coupling Kit, Invitrogen 14311D) at a ratio of 10 μg antibody per mg of beads and rotated overnight at 37 °C. The dynabeads beads were collected using a magnet, washed three times with 1% Triton cell lysis buffer, boiled and denatured in 1× Laemmli sample buffer, separated by SDS–PAGE, and immunoblotting was performed as described above. The secondary antibodies were horse anti-mouse IgG-HRP (Cell Signaling, 7076S, 1:5,000) and goat anti-mouse-IgE-HRP (Southern Biotech, 1110-05, 1:5,000). In brief, 500,000 cells per ml of NCI-H69 cells were seeded in six-well plates and treated with increasing concentrations of nocodazole (60, 120, 240 ng ml−1) or DMSO as a control. After overnight incubation, cells were collected, fixed and subjected to propidium iodide-based FACS to assess mitotic arrest. A nocodazole concentration of 60 ng ml−1 was sufficient to induce mitotic arrest in around 90% of cells. To determine the optimal wash-out time for mitotic exit, cells were plated at 500,000 cells per ml density and treated with 60 ng ml−1 nocodazole. After overnight incubation, nocodazole-containing media was removed, and cells were washed three times with DPBS (Life Technologies, 14190250). Fresh warm medium was then added, and cells were collected at 0, 3, 6 and 12 h after washout. A 6 h incubation period was optimal to allow the majority of cells to complete mitosis and re-enter the subsequent cell cycle. For the immunoblot analysis shown in Fig. 3k, cells were seeded in 6 cm tissue culture dish at a density of 500,000 cells per ml and incubated overnight with nocodazole (60 ng ml−1). The next day, cells were washed three times with PBS and incubated with CDK2i (PF-07104091, 500 nM), cyclin A/B RxL inhibitor (CIRc-004, 20 nM), nocodazole (60 ng ml−1) or the combinations as indicated using DMSO as control. At 6 h after incubation, cells were collected for immunoblot analysis. 4, cells were counted on day 0 using the Vi-Cell XR Cell Counter and plated in 6 cm plates at 250,000 cells per ml in 4 ml of medium per plate for all cell lines along with indicated concentration of cyclin A/B RxL inhibitor. Then, 3 days later, cells were pelleted by centrifugation at 400g for 3 min at 4 °C in a 15 ml conical tube. The cell pellets were then washed once in 1 ml of ice-cold PBS, transferred to a 1.5 ml Eppendorf tube and pelleted by centrifugation at 400g for 3 min at 4 °C. The PBS was removed by gentle aspiration and the soluble proteins from the cell pellets were extracted by incubation in nucleus lysis buffer (250 mM sucrose, 60 mM KCL, 15 nM NaCl, 15 mM Tris pH 7.5, 5 mM MgCl2, 1 mM CaCl2, 1 mM DTT, 0.3% NP-40 supplemented with a cocktail of protease inhibitor cocktail and phosphatase inhibitors) for 10 min at 4 °C. The extracts were then centrifuged at 10,000g for 1 min at 4 °C and the supernatant was removed by aspiration. The cell pellets were again resuspended in nucleus lysis buffer and incubated for 10 min at 4 °C and then centrifuged at 10,000g for 1 min at 4 °C. The supernatant was removed by aspiration and the histones in the insoluble pellet were extracted by overnight incubation in 60 µl of 0.2 N HCl followed by centrifugation at 16,800g for 15 min at 4 °C. The supernatant was transferred to a fresh 1.5 ml Eppendorf tube and quantified using the Bradford protein assay. Then, 1 μg of protein in sample buffer was boiled and resolved by SDS–PAGE on a 15% SDS–PAGE gel. Immunoblot analysis was performed as described above. The primary antibodies were as follows: rabbit anti-histone-3 DH12 (Cell Signaling, 4499, 1:5,000) and rabbit anti-p-histone H2A.X Ser 139 (Cell Signaling, 2577, 1:1,000). For all experiments, total histone H3 protein was run as a sample processing control on a separate gel owing to the low amount of protein lysate loaded onto the gel (1 μg) for histone blots. A panel of 46 SCLC cell lines was tested for their sensitivity to cyclin A/B RxL inhibition in a Cell Titer Glo (CTG) proliferation assay performed at Shanghai ChemPartner (Fig. Staurosporine was used as an assay control per cell line and was dosed in duplicate from 2 μM maximum concentration in a 10-point 1:3 serial dilution. Cells were exposed to inhibitor for 4–8 days depending on the length of time required for at least two cell doublings to occur. A time 0 h read and an end-point DMSO control read were obtained per cell line for calculation of GI50 values. SCLC panel GI50 values are included in Supplementary Table 1. CIRc-001 was profiled in Horizon Discovery's High Throughput OncoSignature screening platform containing 302 cell lines across 18 indications (Extended Data Fig. Assay plates were dosed in triplicate with inhibitor in a 9-point, 1:3 serial dilution dose–response assay with 10 μM maximum concentration. Cells were incubated for 3 days and then analysed using CellTiter-Glo 2.0 (Promega). At the time of treatment, a set of assay plates that did not receive treatment were collected and ATP levels measured by adding CellTiter-Glo 2.0 (Promega). Potency and efficacy metrics were derived from logistics curves fitted to growth inhibition using Horizon's Chalice software. Horizon Discovery OncoSignature panel cell line GI50 values are provided in Supplementary Table 2. To identify differentially expressed gene sets associated with sensitivity and resistance to CIRc-001 across the Horizon Discovery OncoSignature cell line panel, and to CIRc-004 across the SCLC panel (Fig. 1e), GSEA was first performed between sensitive and resistant cell lines using the MSigDB hallmark pathways using mRNA expression data obtained from the Broad Institute's Cancer Cell Line Encyclopaedia (CCLE) database (2019 release: DepMap 2019q3). Sensitive and resistant are defined as the top and bottom quartile of the area over the curve to identify hallmark pathways that were differentially enriched in sensitive versus resistant lines based on FDR < 0.05 For the Horizon Discovery OncoSignature cell line panel, 42 out of 46 SCLC lines and 288 out of 302 cell lines across Horizon Discovery OncoSignature cell line panel had mRNA expression that was available for the analysis. The GSVA method33, using the MSigDb Hallmark collection of RNA-seq data, was then used to calculate GSVA scores for E2F targets and G2/M checkpoint pathway using the GSVA Bioconductor package (v.1.50.5). To visualize the top selected pathways from GSEA analysis, gene expression data were first aggregated as GSVA scores based on gene set annotations obtained from MSigDB. For each pathway, pathway scores were transformed to z scores across samples to standardize the data. The heat map was constructed using the ComplexHeatmaps package (v.2.18.0) in R (v.4.3.2), where rows represented Hallmark pathways, columns represented samples and the colour intensity reflected the mean z score of each pathway for each sample and was hierarchically clustered by pathway to reveal patterns of enrichment. 1e (right), the median E2F target or G2M pathway score across the SCLC panel was used as the threshold to define low versus high groups and P values were calculated using Mann–Whitney t-tests. On day 0, NCI-H1048 cells were counted using the Vi-Cell Counter. In total, 1.50 × 108 cells (which would yield representation of at least 500 cells per sgRNA if multiplicity of infection (MOI) = 0.3) were pelleted and resuspended at 2 × 106 cells per ml in complete medium containing 50 µl ml−1 of human genome-wide Brunello sgRNA library (CP0043) (purchased from the Broad Institute) lentivirus, which contains both the sgRNA and Cas9, and 8 µg ml−1 polybrene. The lentiviral titre was determined empirically in pilot experiments with a goal MOI of 0.3–0.5. The cells mixed with polybrene and lentivirus were then plated in 1 ml aliquots onto 12-well plates and centrifuged at 931g (Allegra X-15R Centrifuge (Beckman Coulter), rotor SX4750A, 2,000 rpm), for 2 h at 30 °C. In parallel, 2 × 106 cells were also centrifuged under the same conditions but without lentivirus as a control for puromycin selection (Mock). Around 16 h later (day 1), the cells were collected, pooled and centrifuged again at 448g to remove the lentivirus and polybrene, and the cell pellet was resuspended in complete medium and plated into twenty-three 15 cm tissue-culture-treated plates at 0.3 × 106 cells per ml or in 6-well plates at the same density for the control (mock) cells. The cells were then cultured for 72 h at which time (day 4) the cells were counted and plated at 0.4 × 106 cells per ml in 15 cm tissue-culture-treated plates with fresh medium containing puromycin (0.25 µg ml−1) to select for puromycin-transduced cells. To do this, cells infected with the sgRNA library or mock-infected cells were plated at 0.4 × 106 cells per ml in 6-well plates in the presence or absence of puromycin (0.25 µg ml−1). After 72 h (day 7), cells were counted using the Vi-Cell XR Cell Counter and the MOI was calculated using the following equation: (number of puromycin-resistant cells infected with the sgRNA library/total number of cells surviving without puromycin after infection with the sgRNA library) − (number of puromycin-resistant mock-infected cells/total number of mock-infected cells). On the same day (day 7), the puromycin-containing medium was exchanged with fresh complete medium not containing puromycin. On day 10, cells were pooled, counted and split into two replicates. CIRc-005 had no effect on cellular proliferation (Extended Data Fig. 2d) and therefore was used as a negative control comparison for all drug treated arms. A total of 4 × 107 cells were plated at 0.2 × 106 cells per ml in 15 cm tissue culture plates in complete medium containing the respective drugs maintaining 500 cells per sgRNA. At the same time, the remaining cells were collected and divided in aliquots of 4 × 107 (again to maintain representation of 500 cells per sgRNA), washed in PBS and cell pellets were frozen for genomic DNA isolation for the ETP before drug selection. Mock-infected cells were maintained at the same density for all drug conditions and carried out in parallel for the entire screen described below. The medium on the plates under drug selection was exchanged every 3 days with complete medium containing fresh drug. Treatment arms were counted and passaged when cells were about 80% confluent and split back to a representation of 4 × 107 cells at a density of 0.2 × 106 cells per ml onto 15 cm tissue culture plates. This was done for the CIRc-005 inactive enantiomer every 3 days as CIRc-005 had no effect on cellular proliferation, and it happened only once or twice during the screen for all of the other drug treatment conditions as the drugs exhibited strong cytotoxicity. All raw data and outputs of each analysis listed above are provided in Supplementary Table 3. For validation studies, we focused on the cyclin A/B RxL inhibitor (CIRc-004) and not CIRc-001, as CIRc-004 spared cyclin E and, perhaps as a result, demonstrated modestly increased cellular potency. NCI-H1048 and NCI-H446 cells were first infected with pLentiCRISPR V2-Puro lentiviruses encoding sgRNAs targeting the selected screen hits and selected with puromycin. Stably infected cells were counted on day 0 using a Vi-Cell XR Cell Counter and plated onto tissue-culture-treated 6-well plates at 50,000 cells per ml in 2 ml of complete medium. Analysis was performed using GraphPad Prism v.10.0.0. The Nijhawan laboratory developed a forward genetic screening system using an engineered colorectal cancer cell line, termed iHCT116, that identifies mutations that confer compound resistance using HCT116 cells engineered for inducible protein degradation (with the addition of IAA) of mismatch repair protein, MLH137. As a result, cells cultured without IAA have a lower mutation rate (mut-low) than cells cultured with IAA (mut-high). These cells are diploid and, therefore, resistance mutations are more likely to be the result of a gain of function. For the forward genetic screen, 5 million iHCT116 cells that had previously been treated with vehicle or indole acetic acid were cultured in 10 μM of CIRc-004 for 17 days, with the medium replenished every 3 days. The surviving clones were analysed for resistance to CIRc-004 in a 3-day proliferation assay in which CIRc-004 was added in dose–response with a final concentration of DMSO of 0.5%. The EC50 for all clones were at least 100-fold higher than parental cells. All clones tested were resistant to an unrelated toxin, MLN4924, suggesting that non-specific mechanisms of CIRc-004 resistance were unlikely (Extended Data Fig. After evaluating the barcode sequence in all 18 clones, 5 and 12 founder clones were identified in the mut-low and mut-high condition, respectively (Extended Data Fig. Eight distinct clones were selected for exome sequencing. Barcode sequences were PCR amplified and sequenced as described previously37. Genomic DNA was collected from select clones (Qiagen) and processed for whole-exome sequencing. Mutations unique to individual clones were identified using the previously described analysis37. Only four genes were mutated in more than 1 of 8 clones and heterozygous CDC20 mutations were found in 5 of 8 clones. For dose–response assays with MPS1 inhibitor (BAY-1217389) to empirically identify concentrations of the MPS1 inhibitor that effectively blocked MPS1 activity (measured by immunoblotting for p-KNL1) without gross effects on cellular proliferation, the cells were counted on day 0 using the Vi-Cell XR Cell Counter and plated in a tissue-culture-treated 6-well plate at 250,000 cells per ml in 2 ml of complete medium. Analysis was performed using GraphPad Prism v.10.0.0. These experiments identified 3 nM of BAY-1217389 blocks p-KNL1 without grossly affecting proliferation. For rescue experiments with the MPS1 inhibitor, cells were counted using the Vi-Cell XR Cell and plated at 250,000 cells per ml for NCI-H1048 cells or at 50,000 cells per ml for NCI-H69 and NCI-H446 cells and then treated with cyclin A/B RxL inhibitor (CIRc-004) at the indicated concentrations with or without MPS1 inhibitor (BAY-1217389 at 3 nM) and collected after 3 days for dose–response assays using the Vi-Cell system. For FACS-based cleaved PARP apoptosis assay, cells were counted and plated at 250,000 cells per ml and then treated with CIRc-004 at the indicated concentrations with or without MPS1 inhibitor (BAY-1217389 at 3 nM) and, 3 days later, were collected, stained and analysed as described above. For immunoblot analysis of p-KNL1 levels after BAY-1217389 treatment, about 2 million cells were counted using the Vi-Cell XR Cell and plated at 500,000 cells per ml and then treated with CIRc-004 at the indicated concentrations with or without BAY-1217389 at 3 nM, and collected 24 h later for immunoblot analysis. Cells were seeded onto glass coverslips at 250,000 cells per ml and treated with CIRc-004 as indicated. Coverslips were fixed with ice-cold 100% methanol for 15 min on ice. Cells were washed with PBS and incubated in blocking buffer (5% BSA in PBS) for 1 h. Coverslips were then incubated with the following primary antibodies diluted in fresh blocking buffer: rabbit anti-p-histone H2A.X Ser139 (Cell Signaling, 2577, 1:400), rabbit anti-p-CASC5/KNL1 Thr943/Thr1155 D8D4N (Cell Signaling 40758, 1:400), ACA (Antibodies, 15-235, 1:500) and incubated overnight at 4 °C. Cells were rinsed three times with PBS + 0.05% Triton X-100 and then incubated with secondary antibodies: Alexa Fluor 568 goat anti-rabbit (Thermo Fisher Scientific, A11011, 1:300) and Alexa Fluor 647 goat anti-human (Thermo Fisher Scientific, A21445, 1:300) for 1 h at room temperature. The coverslips were then washed three times in PBS containing 0.05% Triton X-100. Cells were then incubated with 1 μg ml−1 DAPI (Cell Signaling) for 5 min. After rinsing with PBS coverslips, the coverslips were mounted onto glass slides with ProLong Glass mounting reagent (Invitrogen, P36930). Images were acquired using the Zeiss LSM980 laser-scanning microscope (Oberkochen) with ZEN 2.3 SP1 software. In brief, regions of interest were defined by identifying mitotic cells and drawing an area centred on the DAPI stain. Background values measured in an adjacent region were subtracted and the resulting corrected total cell fluorescence (CTCF) was calculated using the formula CTCF = (integrated density) − (area of selected cell × mean fluorescence of background). Each individual mitotic cell CTCF was plotted in a dot plot and CIRc-004-treated cells were compared with vehicle-treated cells. Two rounds of FACS sorting were performed to obtain a pure population of eGFP and mCherry double-positive cells. Cells were seeded onto 35 mm µ-Dish (iBiDi, 80137) at 500,000 cells per ml and allowed to attach and grow for 48 h. CIRc-004 (20 nM) was added 2 h before live-cell imaging. z stacks (+6 (above) and −4 (below) planes at 0.5 μm spacing) were captured every 5 min for 10 h, using a Zyla 4.2 sCMOS camera (Andor), and a ×20/0.95 NA Plan Apochromat Lambda objective with the correction collar set to 0.17. An environmental enclosure was used to maintain cell culture conditions (37 °C and humidified 5% CO2) for all live-cell confocal imaging. Images were analysed on NIS Elements Viewer v.4.2 (Nikon). The timing of mitosis was measured from DNA condensation (beginning timepoint) to cytokinesis completion (end timepoint). For FUCCI reporter LIC, RPE1 cells were stably transduced with Incucyte cell cycle green/red construct (47741, Sartorius). After selection with puromycin (2 µg ml−1), cells were seeded in a 96-well plate at 1,000 cells per well. CIRc-004 (2,000 nM) was added before initiating live-cell imaging. Plates were imaged every 20 min for 24 h. Imaging and analysis was performed using Incucyte Zoom/S3 live-cell imagers (Essen Biosciences) and quantified using the Incucyte analysis software. To establish the adenosine or cytosine base editor Cas9-expressing cell line for the screen with the tiling library, NCI-H1048 cells were transduced with pRDA_867 (A>G base editor Cas9) and pRDB_092 (C>T base editor Cas9), respectively. Successfully transduced cells were selected with blasticidin (5 µg ml−1) to make stable adenosine or cytosine base editor cell lines. First a pilot experiment was performed to empirically determine the lentiviral titre needed for a goal MOI of 0.3–0.6. Cells were mixed with polybrene and increasing amounts of the lentiviral CP1985 sgRNA library and plated in 1 ml aliquots onto 12-well plates and centrifuged at 931g (Allegra X-15R Centrifuge (Beckman Coulter), rotor SX4750A, 2,000 rpm), for 2 h at 30 °C. In parallel, 2 × 106 cells were also centrifuged under same conditions but without lentivirus as a control for puromycin selection (Mock). Then, about 16 h later (day 1), the cells were collected, pooled and centrifuged again at 448g to remove the lentivirus and polybrene, and the cell pellet was resuspended in complete medium and plated into 15 cm tissue-culture-treated plates at 0.4 × 106 cells per ml or in 6-well plates at the same density for the control (mock) cells. The cells were then cultured for 72 h at which time (day 4) the cells were counted and plated at 0.4 × 106 cells per ml in 15 cm tissue-culture-treated plates with fresh medium containing puromycin (0.5 µg ml−1) to select for puromycin-transduced cells. After 72 h (day 7), cells were counted using the Vi-Cell XR Cell Counter and the MOI was calculated using the following equation: (number of puromycin-resistant cells infected with the sgRNA library/total number of cells surviving without puromycin after infection with the sgRNA library) − (number of puromycin-resistant mock-infected cells/total number of mock-infected cells). These experiments demonstrated that 25 µl ml−1 of CP1985 virus was needed to achieve a MOI of about 0.3–0.6. For the screen, adenosine or cytosine base editor Cas9-expressing NCI-H1048 cell lines were counted using the Vi-Cell counter on day 0. Two replicates of 2.4 × 107 cells (which would yield representation of at least 2,000 cells per sgRNA if MOI = 0.3) were pelleted and resuspended at 2 × 106 cells per ml in 1 ml of complete medium containing 25 µl ml−1 of the pooled sgRNA library (CP1985) lentivirus and 8 µg ml−1 polybrene. On the same day (day 7), puromycin-containing medium was exchanged with fresh complete medium not containing puromycin for all cells in the screen. On day 10, cells from each replicate were pooled, counted and then split into two additional replicates (four replicates in total). In total, 6 × 106 cells from each replicate were collected for an ETP for representation of 2,000 cells per sgRNA. This was done by washing cells in PBS and freezing cell pellets at −80 °C for genomic DNA isolation. The remaining cells from each replicate were then split into two treatment arms: (1) CIRc-004 (cyclin A/B RxL inhibitor, 200 nM) and (2) CIRc-005 as a negative control and comparator (inactive enantiomer of CIRc-004, 200 nM) with 4.5 × 106 cells in each treatment arm plated in 15 cm tissue culture plates at 0.2 × 106 cells per ml in complete medium containing the respective drugs listed above to maintain representation of 1,500 cells per sgRNA. Mock-infected cells were maintained at the same density for all drug conditions and carried in parallel for the entire screen described below. The medium on the plates under drug selection was exchanged every 3 days with complete medium containing fresh drug at the concentrations described above. Treatment arms were counted and passaged when cells were about 80% confluent and split back to a representation of 4.5 × 106 cells at a density of 0.2 × 106 cells per ml onto 15 cm tissue culture plates. This occurred for the CIRc-005 inactive enantiomer every 3 days as CIRc-005 had no effect on cellular proliferation. As the CIRc-004 exhibited strong cytotoxicity, this treatment arm required medium exchange only every 3 days. The remaining cells were divided into aliquots of 6 × 106 cells, washed in PBS and the cell pellets were frozen for genomic DNA isolation. After completion of the screen, genomic DNA was isolated using the Qiagen Genomic DNA midi prep kit (51183) and mini prep kit (51106) according to the manufacturer's protocol. 2a,b were calculated by subtracting the plasmid DNA (pDNA) log-normalized value from the CIRc-005 LTP (day 26) log-normalized value and this was used for replicate reproducibility and dropout of essential sgRNA versus negative-control sgRNAs. To calculate the z-scored LFC values in Fig. This LFC value (CIRc-004 minus CIRc-005) for each sgRNA was z-scored, a transformation that factors in the performance of the control guides, using the following equation: z score = (x − μ)/σ, where x, μ and σ correspond to the LFC of an individual guide, the mean LFC of all negative control guides and the s.d. These z-scored LFC values for each sgRNA were then matched to the annotated CP1985 library where Beagle (Broad Institute) was used to predict the expected nucleotide edit and plotted for each tiled gene as indicated in each figure panel listed above. All raw data and outputs of each analysis listed above are provided in Supplementary Table 5. To validate the exact mutations enriched after CIRc-004 treatment in CCNB1 corresponding to the amino acids 117–285, which contained several enriched CCNB1 variants from our original base editor screen in Fig. 3b, we repeated the screen as in Fig. 3a at scale with a representation of 1,500 cells per sgRNA, and at the LTP (day 26), mRNA was isolated from the cyclin A/B RxL inhibitor (CIRc-004)-treated and enantiomer control (CIRc-005)-treated arms using the Quick-RNA Miniprep kit (Zymo Research) according to the manufacturer's instructions. A cDNA library was synthesized using iScript Reverse Transcription Supermix for RT-qPCR (BioRad, 1708841) according to the manufacturer's instructions with 3,000 ng of RNA. The target site on CCNB1 was amplified by two-step PCR using KOD XTREME HOT START DNA POLYMERASE (Sigma-Aldrich, 71975-3) using 25% of the total cDNA reaction above. For the first PCR step, CCNB1 1F, 5′-GGAACGGCTGTTGGTTTCTG-3′; and CCNB1 1R, 5′-AAGTAAAAGGGGCCACAAGC-3′ primers were used to amplify a 1,514 bp region. Then, 0.2 μl of the first PCR reaction was used for a second round of PCR amplification using the following primers: 2F, 5′-CGCCTGAGCCTATTTTGGTTG-3′; and CCNB1 2R, 5′-CCATCTGTCTGATTTGGTGCTTAG-3′ to yield a 510 bp product. This final PCR product was column purified using the QIAquick Gel Extraction kit (Qiagen, 28704) and sent for amplicon sequencing using next-generation sequencing by the MGH DNA Core Facility and analysed using MGH DNA Core's complete amplicon sequencing pipeline for data analysis of >600 bp amplicons using the following CCNB1 NM_031966.4 refseq amplicon: CGCCTGAGCCTATTTTGGTTGATACTGCCTCTCCAAGCCCAATGGAAACATCTGGATGTGCCCCTGCAGAAGAAGACCTGTGTCAGGCTTTCTCTGATGTAATTCTTGCAGTAAATGATGTGGATGCAGAAGATGGAGCTGATCCAAACCTTTGTAGTGAATATGTGAAAGATATTTATGCTTATCTGAGACAACTTGAGGAAGAGCAAGCAGTCAGACCAAAATACCTACTGGGTCGGGAAGTCACTGGAAACATGAGAGCCATCCTAATTGACTGGCTAGTACAGGTTCAAATGAAATTCAGGTTGTTGCAGGAGACCATGTACATGACTGTCTCCATTATTGATCGGTTCATGCAGAATAATTGTGTGCCCAAGAAGATGCTGCAGCTGGTTGGTGTCACTGCCATGTTTATTGCAAGCAAATATGAAGAAATGTACCCTCCAGAAATTGGTGACTTTGCTTTTGTGACTGACAACACTTATACTAAGCACCAAATCAGACAGATG. To analyse variants from the CCNB1 refseq above, the percentage of mismatched reads at individual base positions were normalized to the percentage of sequencing reads to get the percentage of variants at each position and then averaged across two independent experiments. All data outputs are in Supplementary Table 5. Co-folding of proteins were done by AlphaFold 2 implemented in ColabFold76. The default parameters were used to predict their relative positions (parameters: msa_mode (MMseqs2_UniRef_Environmental), pair_mode (unpaired_paired), model_type (auto), number of cycles (3), recycle_early_stop_tolerance(auto), relax_max_iterations (200), and pairing_strategy (greedy)). The command ‘alphafold contacts /A to /B distance 3' was used to identify residues potentially interacting within a putative predicted distance of less than 3.0 Å. Schrodinger software suite (2024-3 version) was used to conduct modelling studies on cyclin B. Starting from the binding pose in the cyclin A model, CIRc-004 was minimized in the cyclin B's RxL site using Prime MM-GBSA with the VSGB solvent model, OPLS4 force field and a rigid protein, followed by docking using Glide SP with the ‘refine-only' option. The resulting binding model was then used as the input structure for free-energy perturbation (FEP) calculations78,79 to assess the binding of CIRc-004 to various single and double mutants of cyclin B. sgRNA sequences targeting specific base edits in CCNB1 were selected from the CP1985 sgRNA library based on z-scored LFC enrichment after CIRc-004 treatment in our primary screens, synthesized by IDT technologies and cloned using the same approach as for sgRNA oligo cloning described above. These sgRNAs were integrated into the GFP-expressing pMV_AA013 lentiviral vector (Broad Institute) using sgRNA Cloning method described above. NCI-H1048 cells, expressing either pRDA_867 (A>G base editor Cas9) or pRDB_092 (C>T base editor Cas9), were transduced with the pMV_AA013 plasmid carrying the sgRNA sequences and GFP and subsequently subjected to puromycin selection to establish stable edited cell lines. The GFP-expressing stable base-edited cells were combined with their parental base editor counterpart cells (without the sgRNA and GFP) at a ratio of 1:4 with the exact ratio confirmed by FACS analysis for GFP and seeded at a density of 0.1 × 106 cells per ml in 10 cm dishes. The cells were cultured in the presence of CIRc-004 (20 nM or 200 nM) or DMSO. Cells were collected for FACS analysis on days 6 and 13. After each timepoint, cells were replated and cultured in the presence of drug with fresh drug added every 3 days. On day 13, the remaining cells after FACS were collected, and genomic DNA was extracted. Data from the FACS-based competition assay at day 13 is shown in Fig. To verify the exact mutations in CCNB1 made by the base-editing sgRNAs and to determine whether CIRc-004 caused enrichment of these mutations, genomic DNA from the competition assay above at day 13 was collected and PCR amplification of the CCNB1 target region (forward 3F, 5′-GCCCAATGGAAACATCTGGATG-3′; reverse 3R, 5′- GCGATCTCTTAAGAAATGCTGCCC-3′) was performed followed by next-generation sequencing using CRISPR amplicon sequencing conducted at the MGH DNA Core Facility as previously described80. Variant reads are listed in Supplementary Table 5. To identify proteins that are modulated by CIRc-004, we performed cyclin B immunoprecipitation followed by mass spectrometry in NCI-H1048 cells treated with CIRc-004 (50 nM),CIRc-005 (50 nM) or DMSO for 2 h. The cyclin B1 immunoprecipitation was performed as described above with the following modifications: 10 mg cell extract was rotated overnight at 4 °C with 10 μg anti-cyclin B antibodies. After cyclin B1 immunoprecipitation, beads were washed with PBS before freezing at −80 °C. Dry beads were resuspended with 200 μl of Smart digest buffer (Thermo Fisher Scientific, 60113-101) before adding 3 μl of Smart trypsin. Beads were digested in a Thermo block (Thermo Fisher Scientific) for 1 h 30 min at 70 °C with 1,400 rpm (~250g) agitation. Trypsin digestion was then stopped by adding trifluoroacetic acid at 1% final concentration. The samples were then desalted using SOLA HRP SPE cartridges (Thermo Fisher Scientific, 60109-001) according to the manufacturer's instructions. Purified peptide eluates were dried by vacuum centrifugation and re-suspended in 0.1% formic acid and stored at −20 °C until analysis. Liquid chromatography–tandem mass spectrometry (LC–MS/MS) analysis was performed using an Ultimate 3000 HPLC connected to an Orbitrap Ascend Tribrid instrument (Thermo Fisher Scientific) and interfaced with an EASY-spray source. Then, 0.5% of the tryptic peptides were loaded onto an AcclaimPepMap100 trap column (100 µm × 2 cm, PN164750; Thermo Fisher Scientific) and separated on a 50 cm EasySpray column (ES903, Thermo Fisher Scientific) using a 60 min linear gradient from 2 to 35% B (acetonitrile, 0.1% formic acid) and at a 250 nl min−1 flow rate. Both trap and column were kept at 50 °C. The Orbitrap Ascend was operated in data-independent mode (DIA) to automatically switch between MS1 and MS2, with minor changes from previously described method81,82,83. In brief, MS1 scans were collected in the Orbitrap at a resolving power of 45,000 at m/z 200 over an m/z range of 350–1,650 m/z. The MS1 normalized AGC was set at 125% (5 × 105 ions) with a maximum injection time of 91 ms and a RF lens at 30%. DIA MS2 scans were then acquired using the tMSn scan function at 30,000 orbitrap resolution over 40 scan windows with variable width, with a normalized AGC target of 1,000%, maximum injection time set to auto and a 30% collision energy. DIA-NN software (V8.1) was used to analyse the raw data in library-free mode and using the recommended default settings83. In brief, a library was created from human UniProt SwissProt database (downloaded 22 July 2022 containing 20,386 sequences) using deep learning. Trypsin was selected as the enzyme (1 missed cleavage), N-term M excision and methionine were enabled. The identification and quantification of raw data were performed against the in silico library applying 1% FDR at precursor level and match between runs (MBR). Perseus (v.1.6.2.2) was used to further analyse the DIA-NN protein group output. Protein intensities were log2-transformed and normalized by median subtraction (by column) after applying a pre-processing filter based on valid numbers (3 out of 3 in at least one group). Subsequently, missing values were imputed following the normal distribution down shifted. A two-sample Student's t-test combined with multiple-testing correction using permutation-based FDR (5%) was applied. CIRc-004-dependent interactome is included in Supplementary Table 6. All mouse experiments complied with National Institutes of Health guidelines. The PDX studies were approved and performed at the Dana-Farber Cancer Institute Animal Care and Use Committee. Housing conditions for mice include a 12 h–12 h day–night cycle where the temperature is maintained at 72 °F. For NCI-H69 studies, 5 × 106 NCI-H69 cells were suspended in serum-free RPMI1640 medium, combined with an equal volume of ECM gel (Sigma-Aldrich, E1270), and implanted subcutaneously into the flanks of 7–8-week-old female Nude mice (Envigo, Hsd:Athymic Nude-Foxn1nu). After 13 days, when tumours were between 88–200 mm3 in size, mice were randomized and put into treatment groups of CIRc-028 or vehicle (5% DMSO, 10% Solutol HS15, in 85% D5W). injections of 100 mg per kg CIRc-028 or vehicle for 14 days. Tumours were collected 18 h after the last dose, formalin fixed for 24 h and paraffin embedded for IHC analysis. For NCI-H1048 studies, 3 × 106 NCI-H1048 cells were suspended in a 1:1 mixture of HITES medium and Matrigel (Corning, 354234) and implanted subcutaneously into the flanks of 6–8-week-old female BALB/c nude mice (GemPharmatech Biotech, D000521). After 16 days, when tumours were between 180–315 mm3 in size, the mice were randomized and put into treatment groups. Treatment with CIRc-028 and vehicle was carried out as described above for NCI-H69 xenografts. Animal use protocols specified that animals with tumour volumes larger than 2,000 and 2,500 mm3, respectively, or experiencing greater than 20% body weight loss or other severe clinical signs of deteriorating condition would be euthanized and these end-point criteria were not exceeded for any mice on study without prompt euthanasia. RNA-seq was performed on untreated SCLC DFCI-393 and DFCI-402 PDX tumours that were used for the PDX studies. For RNA-seq analysis of experimental PDX, DFCI-393 tumours were treated with the orally bioavailable cyclin A/B RxL inhibitor CIRc-014 at 100 mg per kg orally three times per day or vehicle (30% PEG400, 20% Solutol HS15, 50% Phosal 53 MCT) three times a day for 4 days and tumours were collected 8 h after the morning dose on the last day of treatment and snap-frozen. For RNA-seq of NCI-H1048 cell line, cells were plated at 500,000 cells per ml in the presence of DMSO (vehicle), cyclin A/B RxL inhibitor (CIRc-004 or CIRc-014, 200 nM), inactive enantiomer (CIRc-005, 200 nM) or a CDK2i (PF-07104091, 500 nM). Then, 24 h after drug treatment, cells were collected for RNA. RNA was extracted using the RNeasy mini kit (Qiagen, 74106) including a DNase digestion step according to the manufacturer's instructions. Total RNA was submitted to Novogene. Paired-end 150 bp sequencing was performed on NovaSeq 6000 sequencer using S4 flow cell. Sequencing reads were mapped to the hg38 genome by STAR. Statistics for DEGs were calculated by DESeq2 (v.1.36.0) (Supplementary Tables 4 and 7). Principal component analysis (PCA) in Extended Data Fig. of log-transformed FPKM (fragments per kilobase of transcript per million mapped fragments) values and these genes were subjected to PCA using the removeBatchEffect function in the limma package (v.3.58.1) and prcomp function of R software (v.4.3.3). The log-transformed sum of ASCL1, NEUROD1, CHGA, CHGA, INSM1 and SYP FPKM values was calculated as the neuroendocrine score for each sample. FPKM values are included in Supplementary Table 7. Heat maps were generated by calculating the z score using log-transformed FPKM values. z-score values are provided in Supplementary Tables 4 and 7. For HALLMARK pathway enrichment analysis, differentially expressed genes were tested if over-represented against the HALLMARK pathways from the MSigDB using R packages msigdbr (v.7.5.1) and clusterProfiler (v.4.8.3). List of differentially expressed genes are included in Supplementary Tables 4 and 7. PDXs were generated from either tumour biopsies or pleural effusions of patients with SCLC. All patient samples were collected after obtaining written informed consent. PDX tumour DFCI-393 originated from a surgical biopsy at resistance from a patient who received previous chemotherapy and radiation. The surgical biopsy of DFCI-393 was directly implanted directly into the subrenal capsule of NSG mice for expansion. DFCI-402 was derived from pleural effusion of a patient who was treated with first-line carboplatin/etoposide and then second-line nivolumab and the pleural effusion was removed as part of routine clinical care. RNA-sequencing of DFCI-393 and DFCI-402 with confirmation by immunoblot analysis showed that both PDX models are neuroendocrine-high SCLCs and DFCI-393 dominantly expresses ASCL1 while DFCI-402 expresses both ASCL1 and NEUROD1. RNA-seq data are included in Supplementary Table 7. Studies and procedures were performed at Dana Farber Institute in accordance with Institutional Animal Care and Use Committee (IACUC) guidelines in an AAALAC accredited vivarium. PDX models were generated using 7- to 8-week-old female NSG (NOD.Cg-Prkdcscid Il2rgtm1WjI/SzJ) mice purchased from Jackson Laboratories. After initial implantation of patient samples, all PDX tumours were expanded and passaged continuously in NSG mice as subcutaneous tumours. Female NSG mice (aged 7–8 weeks) were used for all in vivo studies. Tumour growth was measured twice weekly when palpable. Once tumour volumes reached approximately 200 ± 50 mm3, mice were randomized using Studylog software into the treatment groups as: vehicle (30% PEG400, 20% Solutol HS15, 50% Phosal 53 MCT) or orally bioavailable cyclin A/B RxL inhibitor CIRc-014 (100 mg per kg, Circle Pharma). Animals were euthanized if the tumour was necrotic/ulcerated or if the tumour volume exceeded 2,000 mm3. For the PD study in DFCI-393, mice bearing PDX tumours were treated with either vehicle or CIRc-014 (100 mg per kg orally three times per day) for 4 days. All tumour samples were collected 8 h after the morning dose on the last day of treatment. Tumour samples were fixed and snap-frozen for further downstream IHC analysis (see below). Animal-use protocols specified that animals with tumour volumes larger than 2,000 or experiencing >15% body weight loss or other severe clinical signs of deterioration would be euthanized, and these endpoint criteria were not exceeded for any mice on study without prompt euthanasia. After 4 days of CIRc-014 being dosed at 100 mg per kg orally three times per day, plasma timepoint sampling was conducted after the last dose in the CIRc-014 treatment group (n = 6). The concentration of CIRc-014 was quantified in the mouse plasma samples using LC–MS/MS at Charles River Laboratories. CIRc-014 was used as a reference standard and glyburide (Sigma-Aldrich) was used as the internal standard. Blank matrix CD-1 mouse plasma, pooled/mixed gender with K2EDTA anticoagulant (BioIVT) was used to prepare calibration standards and quality-control samples. Sample extraction was performed on ice. A 10 µl aliquot of sample (calibration standards, quality control samples, blanks and study samples) was added to a 96-well plate. Then, 60 µl of internal standard solution (100 ng ml−1 glyburide in acetonitrile) was added to each well, except for wells designated for matrix blanks to which 60 µl of acetonitrile was added. The samples were covered, vortexed and centrifuged for 5 min. Then, 50 µl of supernatant was transferred to a new 96-well plate and the samples were diluted with 50 µl of MilliQ water before analysis. A Waters Acquity UPLC BEH C18, 50 × 2.1 mm (1.7 µm) column was used and maintained at 50 °C during analysis. 5% acetonitrile/water with 0.1% formic acid was used as mobile phase A and 5% acetonitrile/methanol with 0.1% formic acid was used as mobile phase B. The chromatography condition with flow rate at 0.8 ml min−1 was started with 60% B, ramped to 90% B in 1 min and further increased to 95% B in 0.05 min and held for 0.25 min and the column was re-equilibrated at the initial conditions for 0.4 min. Analyte and internal standard were detected using an Applied Biosystems Sciex API 6500 triple quadrupole mass spectrometer. Linear fitting with 1/x2 weighting for a calibration range from 1.00 to 10,000 ng ml−1 was used for data analysis. Analytical results for all samples met acceptance criteria. dosed cyclin A/Bi CIRc-028, NCI-H69- and NCI-H1048-tumour bearing Nude mice were used from the cell line xenograft efficacy studies described above. After 14 days of dosing with CIRc-028 at 100 mg per kg i.v. once per day, plasma timepoint sampling was conducted after the last dose in the CIRc-028 groups. For the NCI-H69 study, animals 1–5 were bled at 5 min followed by terminal bleed by cardiac puncture at 4 h, and animals 6–10 were bled at 0.5 and 2 h followed by terminal bleed by cardiac puncture at 18 h. For the NCI-H1048 study, animals 1–8 were bled at 5 min and at 0.25, 0.5, 1, 2, 4 and 8 h followed by terminal bleed by cardiac puncture at 24 h. Blood was collected in K2EDTA tubes, centrifuged and plasma was collected, frozen and sent for analysis. CIRc-028 was used as a reference standard and dextromethorphan (Sigma-Aldrich) was used as the internal standard. Blank matrix mouse plasma, with K2EDTA anticoagulant (Bioreclamation) was used to prepare the calibration standards and quality-control samples. Sample extraction was performed on ice. A 20 µl aliquot of sample (calibration standards, quality control samples, blanks and study samples) was added to 2 volumes of cold internal standard solution except samples designated for matrix blanks to which 40 µl of acetonitrile was added. The samples were covered, vortexed and centrifuged for 30 min at 6,100g. An aliquot of each supernatant was transferred to an autosampler plate and diluted with 2 volumes of 0.2% formic acid in water before analysis. A Waters Acquity UPLC BEH C18, 50 × 2.1 mm (1.7 µm) column or equivalent was used and maintained at room temperature during analysis. Water with 0.2% formic acid was used as mobile phase A and acetonitrile with 0.2% formic acid was used as mobile phase B. The chromatography condition with a flow rate at 1.2 ml min−1 was started with 5% B, ramped to 95% B in 1.5 min and held for 0.2 min at 95% B with a flow rate of 0.8 ml min−1 and the column was re-equilibrated at initial conditions for 0.5 min. The analyte and internal standard were detected using an Applied Biosystems Sciex API 4000 triple quadrupole mass spectrometer. Power fit regression with no weighting for a calibration range from 25 to 50,000 ng ml−1 was used for data analysis. Analytical results for all samples met acceptance criteria. CIRc-028 was used as a reference standard. Blank BALB/c nude mice matrix mouse plasma with K2EDTA anticoagulant was used to prepare calibration standards and quality-control samples. Sample extraction was performed on ice. A 10 µl aliquot of sample (calibration standards, quality control samples, blanks and study samples) was added to 200 volumes of cold internal standard solution in acetonitrile except samples designated for matrix blanks to which 200 µl of acetonitrile was added. The samples were covered, vortexed and centrifuged for 15 min at 4,000 rpm. An aliquot of each supernatant was transferred to an autosampler plate and diluted with 2 volumes of with water before analysis. A phenomenex Halo ES-CN 2.7 µ (30 × 3 mm) column or equivalent was used for analysis. 95:5 water:acetonitrile with 0.1% formic acid was used as mobile phase A and 95:5 acetonitrile:water with 0.1% formic acid was used as mobile phase B. The chromatography condition with flow rate at 0.6 ml min−1 was started with 10% B, ramped to 100% B in 1.7 min and held for 0.3 min at 95% B and the column was re-equilibrated at initial conditions for 0.4 min. The analyte and internal standard were detected using an Shimadzu LCMS-8050 quadrupole mass spectrometer. An appropriate internal standard was used. An appropriate linear fit was used for calibration range from 0.5 to 2,000 ng ml−1 was used for data analysis. Analytical results for all samples met acceptance criteria. IHC staining was performed with a Bond RX Autostainer (Leica Biosystems) using a Polymer Refine Detection kit (Leica Biosystems). Sections (thickness, 4 μm) were serially cut from formalin-fixed paraffin-embedded mouse tumour samples to perform single IHC studies using antibodies recognizing p-KNL1 (1:100, rabbit monoclonal antibody, Cell Signaling, 40758), and cleaved caspase-3 (1:400, rabbit monoclonal antibody, Cell Signaling, 9664). For each slide, tumour areas were identified by a pathologist (Y.N.L.) and manually annotated using the HALO system. The number of positive and negative tumour cells was then determined using the HALO platform multiplex-IHC v.3.2.5 algorithm (Indica laboratory) and the percentage of positive tumour cells was subsequently calculated. Genes were considered hits if q < 0.25, using the STARS analysis. For all experiments with statistical data, the number of independent biological replicates is indicated in each figure legend. 2k, 5h,j, 6f–h,l, 7a, 8g,i, 9f,g and 10b–e,g,i,k, at least three biological replicates were performed and representative immunoblots are shown. 8h,u, 9d,e,h–k and 10f, at least two biological replicates were performed and representative immunoblots are shown. Raw immunoblots are included in Supplementary Fig. 2d,f, 4a–h, 5k and 8a,c,e, the mean of three independent biological replicates is shown unless otherwise specified. 8j–t, the representative mean of two technical replicates is shown and each experiment was performed in three biological replicates. For dose–response assay shown in Extended Data Fig. 8v–x, the representative mean of two technical replicates is shown and each experiment was performed in two biological replicates. For the live-cell imaging experiment in Extended Data Fig. 6a, representative micrographs are shown from two biological replicates and, for Extended Data Fig. 2l, average data are shown of six technical replicate wells per treatment given, performed in two biological replicates. For the immunofluorescence experiment in Extended Data Figs. 6c and 10a, representative micrographs are shown from three biological replicates. For the IHC experiments in Fig. 5c,d,k,l, representative micrographs are shown from at least five independent tumours per treatment group and the data are quantified in Fig. 5a,b,i,j, two-way ANOVA was used to compare drug treatment versus vehicle. For all other experiments, statistical significance was calculated using unpaired two-tailed Students t-tests. No statistical methods were used to pre-determine sample sizes, but sample sizes are similar to those used previously for in vivo experiments. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The RNA-seq experiments of NCI-H1048 cells treated with cyclin A/B RxL inhibitors or the CDK2i are included in Supplementary Table 4. FPKM values from the bulk RNA-seq experiments of SCLC PDX models DFCI-393 and DFCI-402 are included in Supplementary Table 7. The RNA-seq experiments of DFCI-393 PDXs treated with cyclin A/B RxL inhibitors are included in Supplementary Table 7. The raw RNA-seq data have been deposited at the Gene Expression Omnibus (GEO) under accession GSE291451. Any other data and materials can be requested from the corresponding author on reasonable request. Source data are provided with this paper. Custom code was not used in this study. George, J. et al. Comprehensive genomic profiles of small cell lung cancer. Kent, L. N. & Leone, G. The broken cycle: E2F dysfunction in cancer. Suski, J. M., Braun, M., Strmiska, V. & Sicinski, P. Targeting cell-cycle machinery in cancer. & Levine, A. J. p53 and E2F-1 cooperate to mediate apoptosis. Qin, X. Q., Livingston, D. M., Kaelin, W. G. Jr & Adams, P. D. Deregulated transcription factor E2F-1 expression leads to S-phase entry and p53-mediated apoptosis. Kowalik, T. F., DeGregori, J., Schwarz, J. K. & Nevins, J. R. E2F1 overexpression in quiescent fibroblasts leads to induction of cellular DNA synthesis and apoptosis. Stiewe, T. & Putzer, B. M. Role of the p53-homologue p73 in E2F1-induced apoptosis. Lissy, N. A., Davis, P. K., Irwin, M., Kaelin, W. G. & Dowdy, S. F. A common E2F-1 and p73 pathway mediates cell death induced by TCR activation. Irwin, M. et al. Role for the p53 homologue p73 in E2F-1-induced apoptosis. A., Lindstrom, D. L. & Harlow, E. Substrate recruitment to cyclin-dependent kinase 2 by a multipurpose docking site on cyclin A. Proc. Loog, M. & Morgan, D. O. Cyclin specificity in the phosphorylation of cyclin-dependent kinase substrates. Ord, M. et al. Multisite phosphorylation code of CDK. Identification of a cyclin-cdk2 recognition motif present in substrates and p21-like cyclin-dependent kinase inhibitors. Selective killing of transformed cells by cyclin/cyclin-dependent kinase 2 antagonists. Rudin, C. M., Brambilla, E., Faivre-Finn, C. & Sage, J. Small-cell lung cancer. Cellular mechanisms of tumour suppression by the retinoblastoma gene. & Rubin, S. M. Molecular mechanisms underlying RB protein function. & Harlow, E. Differential regulation of E2F transactivation by cyclin/cdk2 complexes. Xu, M., Sheppard, K. A., Peng, C. Y., Yee, A. S. & Piwnica-Worms, H. Cyclin A/CDK2 binds directly to E2F-1 and inhibits the DNA-binding activity of E2F-1/DP-1 by phosphorylation. Negative regulation of the growth-promoting transcription factor E2F-1 by a stably bound cyclin A-dependent protein kinase. Ord, M., Venta, R., Moll, K., Valk, E. & Loog, M. Cyclin-specific docking mechanisms reveal the complexity of M-CDk function in the cell cycle. Faustova, I. et al. A new linear cyclin docking motif that mediates exclusively S-phase CDK-specific signaling. Ord, M. et al. Proline-rich motifs control G2-CDK target phosphorylation and priming an anchoring protein for polo kinase localization. Nandha Premnath, P., Craig, S. & McInnes, C. Development of inhibitors of protein-protein Interactions through REPLACE: application to the design and development non-ATP competitive CDK inhibitors. Discovery of potent and orally bioavailable macrocyclic peptide-peptoid hybrid CXCR7 modulators. Buyanova, M. & Pei, D. Targeting intracellular protein-protein interactions with macrocyclic peptides. Merz, M. L. et al. De novo development of small cyclic peptides that are orally bioavailable. Rafi, S. B., Hearn, B. R., Vedantham, P., Jacobson, M. P. & Renslo, A. R. Predicting and improving the membrane permeability of peptidic small molecules. Giordanetto, F. & Kihlberg, J. Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties? Discovery of cell-permeable macrocyclic cyclin A/B RxL inhibitors that demonstrate anti-tumor activity in vivo. Febres-Aldana, C. A. et al. Rb tumor suppressor in small cell lung cancer: combined genomic and IHC analysis with a description of a distinct Rb-proficient subset. Hanzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Li, J. et al. Synergistic function of E2F7 and E2F8 is essential for cell survival and embryonic development. Nguyen, T. P. et al. Inducible mismatch repair streamlines forward genetic approaches to target identification of cytotoxic small molecules. How APC/C-Cdc20 changes its substrate specificity in mitosis. Kapanidou, M., Curtis, N. L. & Bolanos-Garcia, V. M. Cdc20: at the crossroads between chromosome segregation and mitotic exit. & Desai, A. Spindle assembly checkpoint activation and silencing at kinetochores. Corbett, K. D. Molecular mechanisms of spindle assembly checkpoint activation and silencing. & Watanabe, Y. MPS1/Mph1 phosphorylates the kinetochore protein KNL1/Spc7 to recruit SAC components. Hayward, D., Alfonso-Perez, T. & Gruneberg, U. Orchestration of the spindle assembly checkpoint by CDK1-cyclin B1. Active cyclin B1-Cdk1 first appears on centrosomes in prophase. Hagting, A., Jackman, M., Simpson, K. & Pines, J. Translocation of cyclin B1 to the nucleus at prophase requires a phosphorylation-dependent nuclear import signal. Liu, F., Rothblum-Oviatt, C., Ryan, C. E. & Piwnica-Worms, H. Overproduction of human Myt1 kinase induces a G2 cell cycle delay by interfering with the intracellular trafficking of Cdc2-cyclin B1 complexes. Gallo, D. et al. CCNE1 amplification is synthetic lethal with PKMYT1 kinase inhibition. McGowan, C. H. & Russell, P. Human Wee1 kinase inhibits cell division by phosphorylating p34cdc2 exclusively on Tyr15. Malumbres, M. & Barbacid, M. Cell cycle, CDKs and cancer: a changing paradigm. Petri, E. T., Errico, A., Escobedo, L., Hunt, T. & Basavappa, R. The crystal structure of human cyclin B. & Gullberg, M. G2/M transition requires multisite phosphorylation of oncoprotein 18 by two distinct protein kinase systems. Cassimeris, L. The oncoprotein 18/stathmin family of microtubule destabilizers. Human cyclin-dependent kinase 2 is activated during the S and G2 phases of the cell cycle and associates with cyclin A. Proc. Chen, J., Saha, P., Kornbluth, S., Dynlacht, B. D. & Dutta, A. Cyclin-binding motifs are essential for the function of p21CIP1. Zhu, L., Harlow, E. & Dynlacht, B. D. p107 uses a p21CIP1-related domain to bind cyclin/cdk2 and regulate interactions with E2F. A., Harlow, E. & Zhu, L. Specific regulation of E2F family members by cyclin-dependent kinases. Ord, M. et al. High-throughput discovery and deep characterization of cyclin-CDK docking motifs. Santamaria, D. et al. Cdk1 is sufficient to drive the mammalian cell cycle. Kalaszczynska, I. et al. Cyclin A is redundant in fibroblasts but essential in hematopoietic and embryonic stem cells. Gong, X. et al. Aurora A kinase inhibition is synthetic lethal with loss of the RB1 tumor suppressor gene. Cells lacking the RB1 tumor suppressor gene are hyperdependent on aurora B kinase for survival. Targeting the vulnerability of RB tumor suppressor loss in triple-negative breast cancer. The STRING database in 2025: protein networks with directionality of regulation. Going out on a limb: delineating the effects of beta-branching, N-methylation, and side chain size on the passive permeability, solubility, and flexibility of sanguinamide A analogues. Leung, S. S., Mijalkovic, J., Borrelli, K. & Jacobson, M. P. Testing physical models of passive membrane permeation. Rezai, T. et al. Conformational flexibility, internal hydrogen bonding, and passive membrane permeability: successful in silico prediction of the relative permeabilities of cyclic peptides. Rezai, T., Yu, B., Millhauser, G. L., Jacobson, M. P. & Lokey, R. S. Testing the conformational hypothesis of passive membrane permeability using synthetic cyclic peptide diastereomers. Bockus, A. T. et al. Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products. Leung, S. S., Sindhikara, D. & Jacobson, M. P. Simple predictive models of passive membrane permeability incorporating size-dependent membrane-water partition. Andrews, M. J. et al. Design, synthesis, biological activity and structural analysis of cyclic peptide inhibitors targeting the substrate recruitment site of cyclin-dependent kinase complexes. Mirdita, M. et al. ColabFold: making protein folding accessible to all. Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. The maximal and current accuracy of rigorous protein-ligand binding free energy calculations. Advancing drug discovery through enhanced free energy calculations. Hong, D. et al. Plasticity in the absence of NOTCH uncovers a RUNX2-dependent pathway in small cell lung cancer. Comparison of protein quantification in a complex background by DIA and TMT workflows with fixed instrument time. Structural premise of selective deubiquitinase USP30 inhibition by small-molecule benzosulfonamides. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. This work was supported by an SRA from Circle Pharma (M.G.O. ), a William Raveis Charitable Fund Damon Runyon Clinical Investigator Award (CI-101-19; M.G.O. and the Deborah A. Connolly Small Cell Lung Cancer Research Fund (M.G.O.). is a UT Presidential Scholar and holds the Joseph F. Sambrook, PhD, Distinguished Chair in Biomedical Science. is supported by the Welch Foundation I-1879 and V-I-0002-20230731, and the Program in Molecular Medicine supported by an anonymous donor. Shilpa Singh, Yavuz T. Durmaz, Kenneth Ngo, Yixiang Li, Muhannad Abu-Remaileh, Xinyue Li, Marie-Anais Locquet, Prafulla C. Gokhale & Matthew G. Oser Circle Pharma, San Francisco, CA, USA Catherine E. Gleason, Ranya Odeh, Frances Hamkins-Indik, Daphne He, Miles W. Membreno, Meisam Nosrati, Nathan N. Gushwa, Siegfried S. F. Leung, Breena Fraga-Walton, Luis Hernandez, Miguel P. Baldomero, Bryan M. Lent, David Spellmeyer, Joshua F. Luna, Dalena Hoang, Yuliana Gritsenko, Manesh Chand, Megan K. DeMart, Sammy Metobo, Chinmay Bhatt, Justin A. Shapiro, Kai Yang, Nathan J. Dupper, Andrew T. Bockus, Jinshu Fang, Ramesh Bambal, Peadar Cremin, James B. Aggen, Li-Fen Liu, Bernard Levin, Evelyn W. Wang, Constantine Kreatsoulas, Marie Evangelista, Rajinder Singh, David J. Earp & Pablo D. Garcia Min Fang, Vishal Khivansara, Shanhai Xie & Deepak Nijhawan Department of Pathology, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA Yasmin N. Laimon, Varunika Savla & Sabina Signoretti Department of Radiation Oncology, Dana-Farber Cancer Institute, Brigham & Women's Hospital, Harvard Medical School, Boston, MA, USA Belfer Center for Applied Cancer Science, Experimental Therapeutics Core, Dana-Farber Cancer Institute, Boston, MA, USA Kenneth Ngo, Bishma Tuladhar & Prafulla C. Gokhale Broad Institute of MIT and Harvard, Cambridge, MA, USA Target Discovery Institute, Centre for Medicines Discovery, Nuffield Department of Medicine, University of Oxford, Oxford, UK Iolanda Vendrell, Roman Fischer & Benedikt Kessler Department of Oncologic Pathology, Dana-Farber Cancer Institute, Boston, MA, USA Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to conceptualization, methodology, validation, formal analysis, investigation, data curation and writing of manuscript. conducted the CRISPR knockout screen, and S. Singh carried out all associated experiments. also performed the CRISPR base-editor screen and all related experiments, including live-cell imaging, dose–response assays, cloning, flow cytometry, immunoprecipitations, immunofluorescence, RNA-seq and analysis of tumour samples. performed the Cancer Hallmark Pathway analysis. A. Sarkar and A. Spektor assisted with the live-cell imaging experiment. conceptualized and performed the forward genetic screen in iHCT-116 cells. supervised, performed and analysed the immunoprecipitation–mass spectrometry experiment. assisted with development and characterization of the macrocycles. supervised the SCLC cell line xenograft studies. assisted with analysis of bulk RNA-seq. assisted with the live-cell imaging using FUCCI reporter system. M.A.-R. generated the DOX-on inducible E2F1 cell lines. provided methodological support for the CRISPR screens. contributed to resources, visualization, supervision and project administration. contributed to conceptualization, methodology, investigation, resources, supervision, project administration, writing and funding acquisition. contributed to conceptualization, methodology, investigation, formal analysis, validation, resources, data curation, visualization, supervision, overall project administration, writing the original and revised manuscript, and funding acquisition. reports grants from Eli Lilly, Takeda, Novartis, BMS, Auron Therapeutics and Circle Pharma. received an SRA from Circle Pharma to fund this work. The other authors declare no competing interests. Nature thanks Christian Heinis and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Overlay of cyclin A/Cdk2 complexes shown in surface representation with bound ligands at the RxL binding site, including a macrocycle (yellow, PDB: 1URC) and p27Kip1 (purple, PDB: 1JSU). b, Computational model of a lariat decapeptide macrocycle bound at the RxL binding site, based on the bound structures of p27Kip1 (purple) and lariat macrocycle (yellow), as a ligand alignment template for binding prediction. Two published cyclin A2 co-crystal structures with ligand bound at the RxL binding site, including the macrocycle, Ace-Arg-Lys-Leu-Phe-Gly (PDB code: 1URC) and the p27Kip1 peptide (PDB code: 1JSU), were used to generate a reference binding mode for our lariat macrocycles (Extended Data Fig. All modelling studies were performed using the Schrodinger software suite (version 2023-3). The bound ligands, aligned at the RxL binding motif, were combined to produce a template for novel ligand alignment. Specifically, the template model (Extended Data Fig. 1babove) is a lariat decapeptide (Lys-Pro-Ser-Ala-Cys-Arg-Lys-Leu-Phe-Gly) that contains the N-terminal hexapeptide segment of p27Kip1 (Lys25-Pro26-Ser27-Ala28-Cys29-Arg30, PDB: 1JSU) and the lariat cyclic pentapeptide (Lys502-Leu503-Phe504-Gly505, PDB: 1URC). To generate binding models of novel peptidic macrocycles, the 3D coordinates were generated by LigPrep and projected onto the template model by performing the maximum common substructure alignment for macrocycles in the ligand alignment module in Maestro (version 13.7.125) with cyclin A crystal structure (PDB: 1URC) specified as the receptor. 10.0) SP docking with the “refine only” option in a docking grid based on the lariat-bound cyclin A crystal structure (PDB: 1URC). d, Detailed representation of modelled hydrogen bonds of CIRc-004 to Ile281, Trp217 and Gln 254 on cyclin A. e, Binding kinetic parameters determined in a WaveDelta instrument (Malvern Pananalytical) to Cyclin/Cdk complexes immobilized by amine coupling on a 4PCP chip (Malvern Pananalytical). f, Compounds physicochemical properties including Molecular Weight (MW), calculated water/oil partition (cLogP), kinetic solubility measured in 5% DMSO at pH 7.4 (mM), measured distribution coefficient between water and octanol (LogD) and MDCK cell apical to basolateral permeability (MDCK Papp; x10-6 cm/sec). g, Hypothesized model: In RB1-deficient cancers (e.g. SCLC) where E2F1 activity is dysregulated, targeting the cyclin A-E2F1 interaction hyperactivates E2F1, which could selectively impair tumour growth. h. Waterfall plot of GI50s for four SCLC and the non-transformed WI-38 fibroblast cell lines for compounds listed in Fig. a, Anti-proliferation GI50 waterfall plot of cyclin RxL A/B/E inhibitor (CIRc-001) tested against Horizon Discovery cancer cell line panel. b, Pathway enrichment scores for the top upregulated and down regulated MSigDb Hallmark gene sets within the differentially expressed genes identified between sensitive and resistant cell lines (n = 288 cell lines) to CIRc-001 using the data in A. Top pathways with FDR corrected <0.05 and Normalized Enrichment Score (NES) >= −/+ 2 are represented as blue (upregulated) or red (downregulated)). c, Average doubling time (hours) of sensitive (blue) and insensitive (red) cell lines using the data the DMSO control from Fig. 1f comparing cell counts at the endpoint (Day 6) compared to start (Day 0) of the dose-response assay. d, Dose response assays of the indicated human SCLC cell lines and insensitive human NSCLC cell lines treated for 6 days with increasing doses of the inactivate enantiomer of the cyclin A/B RxL inhibitor (CIRc-005). e, Representative histograms of flow cytometric analysis of propidium iodide (PI) stained NCI-H1048 cells treated with CIRc-004 (200 nM) or DMSO (vehicle) for 24 h. f, Dose response assays of NCI-H1048 cells treated for 6 days with increasing doses of the selective cyclin A RxL inhibitor (CIRc-018), the selective cyclin B RxL inhibitor (CIRc-019), or the cyclin A/B RxL inhibitor (CIRc-004). g, Quantitation of cleaved PARP positive cells analysed by flow cytometric analysis in NCI-H1048 cells treated for 3 days with the indicated doses of CIRc-018, CIRc-019, CIRc-004 or DMSO. Statistical significance was calculated using unpaired, two-tailed students t-test. h, Representative histograms of cell cycle distribution of NCI-H1048 cells treated with CIRc-018, CIRc-019, CIRc-004 or DMSO and then stained with PI. i) Representative histograms of cell cycle distribution of human hematopoietic stem and progenitor cells (HSPC) treated with CIRc-004, staurosporine (100 nM) or DMSO for 24 h. j) Quantitation of cleaved PARP positive cells analysed by flow cytometric analysis in HSPCs treated with CIRc-004, staurosporine (100 nM) or DMSO for 24 h. For i-j, n = 2 biological replicates. k, Immunoblot analysis of RPE1 cells treated with indicated concentrations of CIRc-004 for 72 h. l, Cell cycle phase progression measured over 27 h by time lapse imaging of RPE1 cells expressing FUCCI cell cycle reporter, treated with a higher concentration of CIRc-004 (2000 nM) which is required to promote cell cycle arrest in RPE1 cells (see Fig. Graph measures quantification of G1 (red FUCCI signal) cell cycle phase per image at progressive time points. m, Representative histograms of cell cycle distribution of RPE1 cells treated with CIRc-004 (2000 nM) or DMSO for 72 h and then stained with PI. Data are mean +/− SD of 6 technical replicate wells. For c-h, k, m, n = 3 biological replicates. For c, f-g and j, data are mean +/- SD. Arrows in d, f indicates DMSO-treated sample which was used for normalization. d-g, Scatter plot of sgRNA abundance represented as Log Fold Change (LFC) comparing the 2 biological replicates of the CRISPR/Cas9 knockout screen for CIRc-004 (d), CIRc-001 (e), PF-07104091 (f), and CIRc-005 (inactive enantiomer of CIRc-004) (g). h-l, Top enriched and depleted hits from Apron analysis of CIRc-001 at the late timepoint (LTP) (day 26) relative to CIRc-005 enantiomer negative control at the LTP (day 26) (h), PF-07104091 LTP vs. CIRc-005 LTP (i), or CIRc-004 LTP vs. early timepoint (ETP) prior to drug treatment (day 10) (j), CIRc-001 vs. ETP (k), PF-07104091 vs. ETP (l). a-b, Dose response assays of NCI-H1048 cells infected with two independent non-targeting sgRNAs (sgCtrl) or two independent sgRNAs against LIN54 (a) or ZWINT (b), and treated for 6 days with increasing doses of the cyclin A/B RxL inhibitor (CIRc-004). c-f, Dose response assays of NCI-H446 cells infected with sgCtrl or sgRNAs against C) CCNB1 (c), CDK2 (d), KNTC1 (e), MAD1L1 (f), treated for 6 days with increasing doses of CIRc-004. g-h, Dose response assays of NCI-H1048 cells infected with two non-targeting sgRNAs (sgCtrl) or two independent sgRNAs against CCNB1 (g) or CDK2 (h), and treated for 6 days with increasing doses of the Cdk2 inhibitor (PF-07104091). i) Representative flow cytometric analysis of phospho-histone H3 in NCI-H1048 cells infected with the indicated sgRNAs and treated with CIRc-004 at 20 nM or DMSO for 24 h. j-k, Quantitation of cells in mitosis by flow cytometry analysis using phospho-histone H3 from i (j) or PI (k) of NCI-H1048 cells transduced with the indicated sgRNAs and then treated with CIRc-004 at 20 nM or DMSO for 24 h. l) Representative flow cytometric analysis from Fig. 2h of cleaved PARP in NCI-H1048 cells infected with the indicated sgRNAs and treated with CIRc-004 at 20 nM or DMSO for 3 days. For i, data are shown as fold change relative to vehicle. For a-h, j, k, n = 3 biological replicates and data are mean +/– SD. Arrows in a-h indicates DMSO-treated sample used for normalization. Statistical significance in j, k was calculated using unpaired, two-tailed students t-test. b-c, Dose-response curves for six clones isolated from Mut-low iHCT116 cells (b) and 12 clones from Mut-high iHCT116 cells treated with increasing doses of CIRc-004 for 3 days (c). d, Dose response curve to MLN4924 of different CIRc-004 resistant iHCT116 clones. e, Barcode sequences identified in 18 different CIRc-004 resistant clones, common sequences are marked in blue. h, Immunoblot analysis of iHCT116 cells stably expressing vector, Flag-CDC20 WT, or Flag-CDC20 R445Q mutant. j, Immunoblot analysis of NCI-H1048 cells stably expressing empty vector, Flag-CDC20 WT, or Flag-CDC20 R445Q; Actin run on separate gel as a sample processing control. k-m, NCI-H1048 cells from j assessed for CIRc-004 dose response (k), cleaved PARP (l), and phospho-histone H3 (m) by flow cytometry following CIRc-004 treatment (20 nM). For k-m, n = 3 biological replicates; data represent mean ± SD. Statistical significance determined by unpaired, two-tailed Student's t-test. a, Representative images from time-lapse fluorescent microscopy of NCI-H1048 cells expressing H2B-GFP treated with 20 nM cyclin A/B RxL inhibitor (CIRc-004) over 10 h (yellow star: arrested mitotic nuclei, purple star: normal mitotic nuclei, red star: apoptotic nuclei). b, Dot plot showing time (mins) from chromosome condensation to cytokinesis completion in CIRc-004 vs vehicle treated cells. Cells that failed to complete cytokinesis by experimental end point are shown in red. n = 50 total mitotic nuclei per condition were counted from 2 independent experiments. Statistical significance was calculated using unpaired, two-tailed students t-test. c, Representative phospho-KNL1, DAPI and anti-centromere antibody (ACA) confocal microscopy images of NCI-H1048 cells treated with cyclin A/B RxL inhibitor (CIRc-004) at 20 nM or DMSO for 24 h. Magnification = 63x, scale bar = 10 µm. d-e, Dot plot measuring fluorescence intensity of phospho-KNL1 per mitotic cell (d) or number of phospho-KNL1 foci per mitotic cell (e) of NCI-H1048 cells from c n = 45 mitotic cells from 3 biological replicates. f, Immunoblot analysis of the indicated human SCLC cell lines (NCI-H1048, NCI-H446, and NCI-H69) and insensitive human NSCLC cell lines (A549, HCC4006, and NCI-H1299) treated for 24 h with CIRc-004 at 200 nM or DMSO. Note, this immunoblot contains the same lysates from Fig. 2i and is included to ensure that the differences in phospho-KNL1 levels after CIRc-004 treatment between cell lines are not a consequence of being loaded on independent immunoblots. g-h, Immunoblot analysis comparing p-KNL1 levels of insensitive NCI-H82 (g) and RPE1 (h) cell lines after treatment with CIRc-004 for 24 h. NCI-H1048 cells (a sensitive model) are included as a benchmark control. i-j, Bar graphs of significantly enriched Hallmark pathways (Padj <0.05) from RNA-seq comparing CIRc-004 vs. DMSO (i) and CIRc-005 vs. DMSO (j). For, i, j, n = 2 biological replicates. k, Representative histograms of flow cytometric analysis of PI stained RPE1 cells transduced with indicated sgRNAs targeting RB1 and TP53 (sgRB1, sgTP53) or a non-targeting control (sgCtrl) and then treated with CIRc-004 or DMSO for 72 h. n = 3 biological independent experiments. l, Immunoblot analysis of indicated RPE1 cells and NCI-H1048 treated with CIRc-004 or DMSO for 72 h. Representative immunoblot from 3 independent experiments is shown. a, Immunoblot analysis of NCI-H1048 cells treated with indicated doses of Cdk2 inhibitor (PF-07104091) for 24 h. CIRc-004 (200 nM) is included as a benchmark control. b, Representative flow cytometric analysis of EdU and DAPI in NCI-H1048 cells treated with PF-07104091 at 500 nM or DMSO (vehicle) for 24 h. c, Quantification of the EdU positive 4c population after drug treatment shown in b. Statistical significance calculated using unpaired, two-tailed students t-test. **=p < 0.01. d, Heat map of z-scores from RNA-seq data of NCI-H1048 cells treated with the Cdk2 inhibitor (PF-07104091 at 500 nM), cyclin A/B RxL inhibitor (CIRc-004 at 200 nM), or DMSO for 24 h. Data is sorted for Log2FoldChange of Cdk2 inhibitor (replicate 1 and 2) vs. DMSO (replicate 1 and 2) showing genes with padj<0.05 which was 110 top up-regulated and 113 down-regulated genes. Relevant up-regulated genes are labelled on right. e, Bar graphs showing top significantly enriched Hallmark pathways (padj<0.05) calculated using differentially expressed genes from bulk RNA-seq experiment in d comparing Cdk2 inhibitor vs. DMSO in NCI-H1048 cells. 2k) without blocking cellular proliferation and hence 3 nM of BAY-1217389 was used for all rescue experiments. b, The raw cell counts of the dose response with BAY-1217389 from a showing that NCI-H1048 cells treated with BAY-1217389 at 3 nM proliferated over the 3-day dose response assay similar to the DMSO untreated control. Arrows in a, c-f indicates DMSO-treated sample used for normalization. g, h, Immunoblot analysis of HCT116 (g) or NCI-H1048 (h) cells stably expressing empty vector, Flag-CDC20 WT, or Flag-CDC20 R445Q mutant treated with CIRc-004 with indicated concentrations for 24 h. For a-b, d-f, n = 3 biological replicates. For c, h, n = 2 biological replicates. i, Immunoblot analysis for cyclin B pS126 in indicated SCLC lines treated with CIRc-004 (300 nM), CIRc-005 (inactive enantiomer, 300 nM), or DMSO for 4 h. n = 3 biological replicates. j-t, NCI-H1048 (j,o,p), NCI-H69 (k,l,q,r), or NCI-H446 (m,n,s,t) cells treated with RP-6306 (Myt1 inhibitor) alone (j,k,m) or in combination with the cyclin A RxL inhibitor (CIRc-018) (l, n), cyclin A/B RxL inhibitor (CIRc-004) (o,q,s), or the cyclin B RxL inhibitor (CIRc-019) (p,r,t) for 5 days. For j-t, data are mean +/− SD of two technical replicates. u, Immunoblot analysis of Wee1 after cyclin B1 IP in NCI-H1048 cells treated with CIRc-004, inactive enantiomer (I.E., CIRc-005), CIRc-018, CIRc-019, or DMSO. All inhibitors were used at 300 nM for 2 h. n = 2 biological independent experiments. v-x, Dose response assay of NCI-H1048 cells treated with Wee1 inhibitor (MK 1775, 10 nM) in combination with CIRc-018 (v), CIRc-004 (w), or CIRc-019 (x) for 5 days. Data are mean +/− SD of two technical replicates. a, Predicted effects of cyclin B mutations identified from the base editor screen on CIRc-004 ligand binding determined by free energy perturbation (FEP) calculations (see Methods). Mutating residues at the RxL binding site, including Ile204 and Trp208 were predicted to have the most significant effect, reducing binding affinity by 1 to 3 kcal/mol relative to cyclin B WT. To supplement this, the Trp208Ala mutation was examined. Consistently, both Trp208Ala and Trp208Arg mutations were predicted to have a high effect on ligand binding. b, Mass spectrometry analysis after IP of endogenous Cyclin B1 in NCI-H1048 cells treated with CIRc-004 (50 nM), CIRc-005 (50 nM) (inactive enantiomer of CIRc-004, I.E) or DMSO for 2 h. Plots show the relative abundance of the indicated proteins as Intensity (MaxLFQ, 3VN 1 group, log 2 Median centred, missing values imputed). Data points indicate values obtained from n = 3 biological replicates. c, AlphaFold2 model of cyclin B1:Cdk2 complex is shown. Amino acids 169-177 of cyclin B are highlighted in yellow showing Cdk interacting region (pink). Solid lines indicate hydrogen bonds between the two proteins with interacting amino acid residues of cyclin B indicated. d, Immunoblot analysis after IP of endogenous cyclin B1 in NCI-H1048 cells treated with CIRc-004 (300 nM), CIRc-005 (I.E), or DMSO for 2 h. Note that this is the same immunoblot from Fig. e, Immunoblot analysis after IP of endogenous cyclin B1 in NCI-H446 cells treated with CIRc-004 (300 nM), CIRc-005 (I.E), or DMSO for 2 h. f, Immunoblot analysis after IP of endogenous cyclin B1 in NCI-H1048 cells treated with CIRc-004 (300 nM), CIRc-005 (I.E), cyclin A RxL inhibitor (CIRc-018, 300 nM), cyclin B RxL inhibitor (CIRc-019, 300 nM) or DMSO for 2 h. g, Immunoblot analysis after IP of endogenous cyclin B1 in RPE1 cells treated with CIRc-004 (2000 nM), paclitaxel (60 nM) or DMSO for 2 h. For d-h, Cdk2 band intensity is normalized to cyclin B1 shown at the bottom. h, Immunoblot analysis after IP of endogenous cyclin A in NCI-H1048 cells treated with CIRc-004 (300 nM), CIRc-005 (I.E), cyclin A RxL inhibitor (CIRc-018, 300 nM), cyclin B RxL inhibitor (CIRc-019, 300 nM) or DMSO for 2 h. Cdk2 band intensity is normalized to cyclin A shown at the bottom. i-j, Immunoblot analysis of NCI-H1048 (i) or NCI-H446 (j) cells treated cyclin A/B RxL inhibitor (CIRc-004) at 200 nM, Cdk1 inhibitor (RO-3306), Cdk2 inhibitor (PF-07104091) at the concentrations indicated or DMSO for 4 h. k, Immunoblot analysis after IP of endogenous Cdk2 in HEK-293T cells expressing CCNB1 WT-HA, CCNB1 triple mutant-(E169K/Y170H/Y177C)-HA, or a negative control vector, and treated with CIRc-004 (300 nM) or DMSO for 2 h. HA band intensity is normalized to Cdk2 shown at the bottom. For f, g, i, representative immunoblots from n = 3 biological independent experiments are shown. For d, e, h, j, k, representative immunoblots from n = 2 biological independent experiments are shown. a, Representative γH2AX and DAPI confocal microscopic images of NCI-H1048 cells treated with the cyclin A/B RxL inhibitor (CIRc-004) at 20 nM or DMSO for 72 h. Magnification = 63x, scale bar = 10 µm. b, c, Immunoblot analysis for phospho-RPA2 S33 (b) and phospho-KAP1 (c) of NCI-H1048 treated with CIRc-004 at indicated concentrations for 24 h. d, Immunoblot analysis of NCI-H1048 cells infected with indicated sgRNAs and then treated CIRc-004 at 20 nM or DMSO for 72 h. e, Immunoblot analysis of histone lysates from NCI-H1048 cells treated with the selective cyclin A RxL inhibitor (CIRc-018), the selective cyclin B RxL inhibitor (CIRc-019), the cyclin A/B RxL inhibitor (CIRc-004), or DMSO (vehicle) for 72 h. f, E2F3 Immunoblot analysis after IP of endogenous cyclin A in NCI-H1048 cells treated with CIRc-004 (300 nM), CIRc-005 (inactive enantiomer of CIRc-004, I.E), cyclin A RxL inhibitor (CIRc-018) or DMSO for 2 h. E2F3 band intensity is normalized to cyclin A shown at the bottom. g, Immunoblot analysis of NCI-H82 cells infected with a doxycycline (DOX) inducible E2F1 sgRNA-resistant cDNA and then superinfected with an sgRNA targeted endogenous E2F1 grown in the presence or absence of DOX for 24 h. h, Dose response assays of NCI-H82 cells from g grown in the presence or absence of DOX for 24 h and then treated with increasing concentrations of CIRc-004 for 6 days. Data are mean +/− SD and arrows indicates DMSO-treated sample which was used for normalization. i,k, Immunoblot analysis of NCI-H1048 (i) and Jurkat (k) cells infected with a doxycycline (DOX) inducible E2F1, E2F2 or E2F3 cDNA grown in the presence or absence of DOX for 24 h. j, l, Quantitation of average half-maximal effective concentration (EC50) without (light blue) or with (dark blue) DOX of NCI-H1048 (j) or Jurkat cells (l) from i, k expressing DOX inducible E2F1, E2F2, E2F3 treated with CIRc-004 at increasing concentrations for 3 days (j) or 6 days (l). For a-e, h, j, l, representative immunoblots from 3 independent experiments are shown. For histone blots in d, e, total histone H3 run as sample processing control on separate gel. a, b, Body weights of mice enrolled in the NCI-H69 (a) or NCI-H1048 (b) xenografts efficacy treatment studies treated for 14 days with vehicle, cyclin A/B RxL inhibitor (CIRc-028, 100 mg/kg IV QD). For a, n = 10 independent mice per arm. c, Representative IHC micrographs of cell pellets from NCI-H1048 cells treated with CIRc-004 at 200 nM or DMSO (vehicle) overnight and then stained for phospho-KNL1 to validate the phospho-KNL1 antibody for IHC. e, Heat map of z-scores from RNA-seq data of NCI-H1048 cells treated with cyclin A/B RxL inhibitor (CIRc-004 or CIRc-014 at 200 nM), inactive enantiomer (CIRc-005 at 200 nM) or DMSO for 24 h. Data is sorted for Log2FoldChange of CIRc-004 (replicate 1 and 2) vs. DMSO (replicate 1 and 2) showing genes with Padj<0.05 which was 150 top up-regulated and down-regulated genes. Relevant up-regulated genes shown on the right. f, Bar graphs showing top significantly enriched Hallmark pathways (padj<0.05) calculated using differentially expressed genes from bulk RNA-seq experiment in e comparing CIRc-014 vs. DMSO in NCI-H1048 cell line. For e, f, n = 2 biological replicates. g, Principal component analysis (PCA) from RNA-seq data integrating RNA-seq data from DFCI-393 and DFCI-402 PDX models with 81 primary SCLC human samples from George et al. Bar scale shows neuroendocrine score (see Methods). h,i, Body weights of mice enrolled in the DFCI-393 SCLC PDX (h) or DFCI-402 SCLC PDX (i) efficacy studies treated for 28 days with CIRc-014 (100 mg/kg PO TID for DFCI-393 and 100 mg/kg PO BID for DFCI-402) or vehicle. j, Plasma concentration of CIRc-014 from NSG mice bearing DFCI-393 PDX tumours treated with CIRc-014 100 mg/kg PO TID in the pharmacodynamic study in Fig. Mice were dosed for 4 days and plasma was collected at the times indicated after the last dose. Unbound concentrations (nM) at indicated time points shown on the right. k, l, Plasma concentration of CIRc-028 at the end of the efficacy studies where CIRc-028 was dosed 100 mg/kg IV QD for 14 days in CDX models of NCI-H69 model (k) shown in Fig. 5a or NCI-H1048 model (l) shown in Fig. Unbound concentrations (nM) at indicated time points shown on the right. For j, study run in NSG mice and NOD SCID % plasma protein binding (PPB) used to estimate unbound fraction. For k-l, studies run in Athymic Nude and BALB/c Nude mice, respectively, and % PPB from each strain was used to estimate unbound fraction. Plasma was collected from mice after the last dose and CIRc-014 or CIRc-028 concentrations were determined by LC-MS/MS. m, Heat map of z-scores from RNA-seq data of DFCI-393 human SCLC PDX from Fig. 5k–n treated with CIRc-014 (100 mg PO TID) or vehicle for 4 days. Values are sorted for Log2FoldChange of CIRc-014 treated tumours (n = 6 tumours from independent mice) vs. vehicle treated tumours values (n = 5 tumours from independent mice) showing genes with Padj<0.05 which was 39 differentially expressed genes. Relevant up-regulated genes shown on the right. The NCI-H1048 cell line treated with cyclin A/B RxL inhibitors (CIRc-004, CIRc-014 at 200 nM), inactive enantiomer (CIRc-005 at 200 nM), or DMSO (vehicle) as shown in Extended Data Fig. 11e is included as a benchmark control showing overlap of genes upregulated in DFCI-393 PDX tumours after CIRc-014 and NCI-H1048 cell lines treated with cyclin A/B RxL inhibitors (CIRc-004 and CIRc-014). n, Bar graphs showing top significantly enriched Hallmark pathways (padj<0.05) calculated using differentially expressed genes from bulk RNA-seq experiment in m comparing CIRc-014 treated DFCI-393 PDX tumours (n = 6 tumours from independent mice) to vehicle treated DFCI-393 PDX tumours (n = 5 tumours from independent mice). Method section detailing the comprehensive synthesis and characterization of macrocycles targeting cyclin–RxL interactions. Live-cell imaging of NCI-H1048 cells treated with CIRc-004. Time-lapse fluorescence microscopy of NCI-H1048 cells expressing H2B–GFP treated with 20 nM cyclin A/B RxL inhibitor (CIRc-004) over 10 h. Magnification, ×20. Live-cell imaging of NCI-H1048 cells treated with vehicle. Time-lapse fluorescence microscopy of NCI-H1048 cells expressing H2B–GFP treated with vehicle (DMSO) over 3 h. Magnification, ×20. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Targeting G1–S-checkpoint-compromised cancers with cyclin A/B RxL inhibitors. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09370-8'>Cancer-induced nerve injury promotes resistance to anti-PD-1 therapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 15:56:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Perineural invasion (PNI) is a well-established factor of poor prognosis in multiple cancer types1, yet its mechanism remains unclear. Here we provide clinical and mechanistic insights into the role of PNI and cancer-induced nerve injury (CINI) in resistance to anti-PD-1 therapy. Our study demonstrates that PNI and CINI of tumour-associated nerves are associated with poor response to anti-PD-1 therapy among patients with cutaneous squamous cell carcinoma, melanoma and gastric cancer. Electron microscopy and electrical conduction analyses reveal that cancer cells degrade the nerve fibre myelin sheets. The injured neurons respond by autonomously initiating IL-6- and type I interferon-mediated inflammation to promote nerve healing and regeneration. Our findings demonstrate the direct immunoregulatory roles of CINI and its therapeutic potential. The development of cancer immunotherapy, specifically of antibodies blocking PD-1 and its ligand (PD-L1), has ushered in a new oncological era. Yet, the majority of patients undergoing anti-PD-1 therapy still do not respond to treatment2,3,4,5. Considerable efforts have been invested in identifying potential resistance mechanisms to anti-PD-1 therapy, including the role of immune regulation by non-immune cells. Tumour-associated nerves (TANs) are peripheral nerve fibres found within, or in close proximity to tumours6. Tumour infiltration into TANs, known as PNI, is a well-established adverse prognostic factor in many cancers1,7, especially in cutaneous squamous cell carcinoma (cSCC)8, melanoma9, gastric cancer10 and pancreatic adenocarcinoma (PDAC)11. While TANs may promote tumour progression through adrenergic signalling12, little is known about the role of TANs in regulating antitumoural immune activity. A healthy PNS supports haematopoiesis, regulates immune responses against infections and participates in the creation of immune memory13,14,15. By contrast, injured peripheral nerves attract immune cells such as immunosuppressive (M2) macrophages16—key players in tumour progression and resistance to anti-PD-1 therapy17—to promote nerve healing and regeneration18. Here we delineated the interaction between cancer cells, TANs and intratumoural immune activity across neurotropic cancers such as cSCC, melanoma, gastric cancer and PDAC. We demonstrated that cancer cells inflict nerve injury through myelin degradation. As the cancer progresses, more TANs are injured, and the nerve-driven inflammation turns into a chronic process, leading to immune exhaustion and resistance against anti-PD-1 therapy. To evaluate the potential role of TAN injury in clinical response to anti-PD-1 therapy, we collected tumour samples from 56 patients with locally advanced (stage II–IVA) cSCC who were enrolled in two anti-PD-1 clinical trials (NCT03565783 and NCT04154943)19,20. In brief, all patients underwent two to four cycles of neoadjuvant anti-PD-1 therapy with cemiplimab, followed by surgery (Fig. None of the patients underwent radiation treatments before anti-PD-1 therapy. Responders (n = 33) were defined as patients with less than 10% viable tumour cells in their neoadjuvant-treated surgical specimens (Fig. 1b); while non-responders (n = 14) were defined as patients with more than 50% viable tumour cells, as previously described19,20. Patients who had 10–50% viable tumour cells in the surgical specimens (n = 9) were excluded from our cohort a priori19,20 as this patient subpopulation has been inconsistently assigned to either the responders or non-responders groups in previous neoadjuvant clinical trials and might also represent mixed biological resistance phenotypes. Our initial step was to assess the presence of PNI in the clinical trial tumour samples, as PNI is the most established and clinically relevant form of cancer–nerve interaction21. At the baseline, non-responders had a significantly higher incidence of PNI than responders (50% versus 15%, respectively; P = 0.012; Fig. As the definition of PNI is histomorphological and not based on functional evidence of nerve damage1, we measured the expression of the canonical neuron injury markers, ATF322,23 and JUN24, using multiplex immunofluorescence (mIF) to test whether cancer cells can injure the invaded neurons (Fig. Analysis of tumour samples (Extended Data Fig. 1; n = 7 responders and n = 6 non-responders) demonstrated that non-responders expressed higher levels of ATF3 in neurons (B3T+GFAP−) compared with responders (P = 0.005; Fig. In healthy neurons, ATF3 is bound to the neuron nucleus; the increased abundance of ATF3 in TAN axons therefore strongly implies the presence of neuronal injury and neuroregeneration attempts25,26,27. While the expression of JUN in non-responder TAN axons was increased compared with responders, this difference did not reach statistical significance (P = 0.55; Fig. 1e), potentially as JUN is transported into the nucleus after neuronal injury28,29, resulting in a lower abundance in the axons. ECOG PS, Eastern Cooperative Oncology Group performance status; HN-cSCC, head and neck cSCC; IF, immunofluorescence; i.v., intravenous. The tumour (T), nodal (N) and metastasis (M) status56 is shown. c, PNI rates in the cSCC clinical trial tumour samples. Pearson's χ2 test, P = 0.012. d,e, Representative images (d) from the cSCC clinical trial tumour samples and box plots (e) illustrating the expression levels of ATF3 and JUN in neurons (B3T+GFAP−), categorized by response status (n = 6 responders, n = 7 non-responders). f,g, Gene set enrichment analysis (GSEA) of the PNI and nerve injury signature30 (Supplementary Table 1) tested on pretreatment samples from our cSCC clinical trial cohort (f), and on two cohorts of patients with metastatic melanoma (ref. 16 (right)) and one cohort of patients with metastatic gastric cancer34 (middle) according to the anti-PD-1 response status (g). Statistical significance was determined by permutation testing (1,000 permutations), and results are reported as normalized enrichment score (NES) and nominal P value. ca., carcinoma; DE, differentially expressed; LE, leading edge; NR, non-responder; R, responder. j,k, Axotomy experimental design (j) and tumour growth plot (k) of the axotomy mouse experiment. For both in vivo experiments (i and k), data are mean ± s.e.m. over time; statistical analysis was performed using a mixed-effects model with restricted maximum likelihood (REML) estimation; post hoc comparisons at individual timepoints were assessed using Tukey's multiple-comparison test. The diagrams in h and j were created using BioRender. To validate the association of TAN injury and anti-PD-1 resistance, we used an independently established and publicly available gene signature developed to assess PNI in cSCC30 (Supplementary Table 1). The signature was enriched among the cSCC clinical trial non-responder pretreatment samples (false-discovery rate (FDR) < 0.01; Fig. PNI is an established adverse prognostic factor in other immunogenic cancers, such as melanoma9 and gastric cancer31. To demonstrate a nerve-driven, tumour-agnostic effect we applied the same PNI/nerve injury signature to three publicly available RNA-sequencing (RNA-seq) cohorts in patients with melanoma32,33 and gastric cancer34 (Supplementary Table 2). The PNI/nerve injury signature was enriched among pretreatment samples of anti-PD-1 non-responders in all three patient cohorts (FDR < 0.01 for all; Fig. In contrast to the patient population of our cSCC clinical trial, which included only patients with locally advanced disease, the patients in the melanoma and gastric cancer cohorts had metastatic disease. To validate the presence of CINI in a metastatic setting, we used the B16F10-OVA melanoma lung metastases mouse model. The lung-innervating nociceptor neurons within the jugular nodose ganglia of the melanoma-bearing mice demonstrated significant upregulation of both Atf3 and Jun compared with the control mice that were injected with PBS vehicle (FDR-adjusted P < 0.001 for both; Extended Data Fig. The functional association between CINI and resistance to anti-PD-1 therapy was validated in vivo. First, we eliminated nerves from the TME by plucking and removing nerves innervating the skin of immunocompetent SKH1-Elite mice (SKH1-Hrhr). This procedure, called denervation, was performed while preserving the skin vasculature35. A sham procedure was performed in the control group (Fig. cSCC cells (B610K, ultraviolet-induced, SKH1-Hrhr derived) were intradermally injected into the denervated skin. Then, 7 days after cancer inoculation, mice were treated with either anti-PD-1 or IgG control. Next, we tested the impact of TANs injury on anti-PD-1 efficacy. Nerve injury was induced using surgical axotomy (Fig. Surgical axotomy did not induce damage to the tumour blood vessels (Extended Data Fig. To increase the experiment's robustness, we used a different, syngeneic, ultraviolet-light-induced, cSCC cell line termed UVcSCC-M4 cells (Methods and Extended Data Fig. Then, 7 days after axotomy, UVcSCC-M4 cells were orthotopically injected into the injured dermatome of C57BL/6 black mice, followed by anti-PD-1 administration. Axotomized mice treated with anti-PD-1 had worse tumour control compared with the anti-PD-1 sham-operated controls (P < 0.001; Fig. 1k), serving as a mirror image to the denervation experiment. To assess the potential drug-specific effects of anti-PD-1 on human immune cells, we repeated this experiment with human-leukocyte-antigen-matched IC8 (human cSCC cells37) that were injected into axotomized or sham-operated skin of humanized CD34+ NOD scid gamma mice (huCD34-NSG; Extended Data Fig. huCD34-NSG mice that underwent axotomy and anti-PD-1 therapy had poorer treatment efficacy compared with the sham controls (P < 0.01). Overall, these results emphasized the association between CINI and anti-PD-1 therapy resistance. To identify the mechanism of CINI, we examined the interaction between cancer cells and neurons in vitro. Dorsal root ganglion (DRG) neurons were co-cultured with mouse SCC cells (Moc1 and B6). As shown in Supplementary Video 1, the SCC cells demonstrated neurotropism and made direct contact with the axon within 72 h. The ultrastructural changes associated with cancer–neuron interaction were assessed using electron microscopy (EM). Scanning EM (SEM) images were obtained on day 5 of co-culture, confirming the cancer cell attachment and invasion to the epineurium (Fig. Compared with naive neurons, cancer-exposed neurons demonstrated myelin breakdown, with myelin debris presenting as large circular aggregates, distinct from the linear appearance of normal compact myelin (Fig. On the basis of transmission EM (TEM), myelin degradation was evident at the endoneurial and axonal level, with disentanglement of the compact myelin lamellae (Fig. After 7 days of co-culture, a complete loss of the myelin sheath was found at the point of contact with cancer cells, as well as proximal loss of compact myelin integrity and axonal mitochondria (Fig. 2b,c), implying the presence of nerve injury. These in vitro findings were validated in vivo using an invaded sciatic nerve mouse model38—human oral SCC cells (HSC-3) were microinjected directly into the right sciatic nerve of mice, therefore minimizing the potential effect of immune cells. The EM images of the invaded sciatic nerve fibres showed similar myelin degradation (Extended Data Fig. Notably, there was no evidence of synaptic cleft formation between cancer cells and neurons, as the EM images did not show electron-dense material and trans-cleft elements between pre- and post-synaptic membranes39,40 (Fig. a,b, SEM (a) and TEM (b) images of DRG nerves without (left) and with (right) SCC cells. The myelin sheath appears normal in controls (left), whereas degraded myelin is observed in the SCC group (right). d, SEM image displaying an SCC cell invading a DRG axon. a–d EM images showing a naive DRG neuron with normal myelin sheath (white arrowheads and yellow pseudocolour; Schwann cells, green; nerve inner layers, yellow arrowheads and purple pseudocolour) compared with the degraded myelin of DRG neurons co-cultured with SCC cells (round cells, red arrowheads). Note the abnormal mitochondria implying axonal degeneration (black arrow heads). e,f, Multielectrode array recordings from the skin of normal (n = 20) and tumour-bearing (n = 21) biologically independent mice showing mean spike amplitude (e) and absolute spike amplitude (f). Statistical analysis was performed using one-way analysis of variance (ANOVA) followed by Tukey–Kramer post hoc test. h, Pearson's correlation plot based on the mIF staining described above. A linear regression line was fitted to the data using least-squares estimation. i, Expression of ATF3 among NeuO+ DRG neurons (living neurons) after co-culture with normal keratinocytes (HEK), SCC (IC8) cells or monoculture (three biologically independent experiments, each with three independent cell cultures). j, Transcriptional differences in DRG neurons co-cultured with IC8 SCC cells versus monoculture controls. The heat map structured by unsupervised hierarchical clustering analysis. The top 12 most significantly enriched pathways based on Hallmark GSEA are shown. EMT, epithelial–mesenchymal transition; NF-κB, nuclear factor-κB; TGFβ, transforming growth factor-β. Post hoc pairwise P values were calculated using the Tukey–Kramer honest significant difference test. Statistical significance was assessed using one-way ANOVA (P < 0.0001) followed by Tukey–Kramer post hoc test. Myelin degradation is associated with decreased nerve conduction velocity and propagation of evoked electrical activity41. To confirm the presence of myelin degradation in CINI, we first conducted high-throughput electrical conduction studies in a mouse cSCC model. Using multimicroelectrode array (MEA), electrophysiological recordings were obtained from 3–5 mm orthotopic cSCC (intradermally injected B6 cells) and from non-tumour-bearing littermate healthy control skin. Simultaneous spatial and temporal continuous recording of the extracellular field potential (FP) revealed that conduction in tumour-bearing skin was significantly lower compared with the normal skin conduction (41.8 µV and 60.2 µV, respectively; P < 0.0001; Fig. Moreover, tumour-bearing skin demonstrated compound potentials, suggestive of myelin degradation42 (Fig. We validated these observations using an in vitro neuron viability assay, which included the following conditions: naive neurons, exogenous stress (no medium exchange for 6 days leading to exhausted neuronal medium), neurons treated with MOC1-conditioned medium, neurons co-cultured with MOC1 cells. Culture medium exchanges did not affect neuron viability, and neurons exposed to MOC1-conditioned medium did not show significantly reduced viability compared with the controls (Extended Data Fig. The assay was validated using cell lines of two other cancer types associated with PNI, melanoma B16 and PDAC MIAPaCa (Extended Data Fig. Moreover, direct cancer–neuron contact resulted in a complete loss of spontaneous and evoked electrical activity after 48 h, while this effect was not observed in the MOC1-conditioned medium or exogenous stress neurons. Overall, these findings implied that CINI results from a direct cancer–neuron contact, not from paracrine secretion or synapse formation. To clinically validate that CINI is associated with myelin degradation, we tested the co-localization of CINI and degraded myelin in an independent cohort of treatment-naive patients with cSCC with localized cancer who underwent surgical resection (wide local excision) of their tumour at the University of Texas MD Anderson Cancer Center between 2013 and 2019 and had archived formalin-fixed paraffin tissue blocks available for analysis. The density of injured nerves (B3T+ATF3+JUN+) correlated with the density of degraded myelin (dMBP+, Pearson's correlation coefficient r = 0.79, P < 0.0001; Fig. CINI's mechanism was further explored in vitro. Induced pluripotent stem (iPS)-cell-derived human DRG neurons were either cultured alone (naive), co-cultured with normal keratinocytes (HEK cell line) or co-cultured with human cSCC cells (IC8) for 48 h. Cells were labelled with a live neuron marker (NeuO). A gene set analysis compared the transcriptomic differences between NeuO+ neurons from the neuron–cancer co-culture and neuron-only controls. Among the top 12 Hallmark gene sets upregulated in cancer-exposed neurons, six were immune related, including tumour necrosis factor (TNF), IFNα, IFNγ and IL-6 signalling (all FDR < 0.01; Fig. By contrast, neither nerve-injury-related nor immune-related transcriptional changes were detected when neuron-only controls were compared with neurons co-cultured with normal keratinocytes (Extended Data Fig. Anti-PD-1-exposed neurons did not demonstrate a significant transcriptional change compared with naive neurons (Extended Data Fig. As an additional validation degree, this experiment used mouse trigeminal ganglia (TG) neurons instead of the iPS cell DRG neurons, as most of the tumours in our trial originated from the head and neck region (an area receiving the majority of its sensory innervation from the TG). TG neurons and MOC1 SCC cells were either cultured alone or co-cultured together for 48 h. IL-6 was not secreted by cancer cells and had the most prominent increase in secretion from neurons after exposure to cancer cells (Fig. Overall, our results demonstrated that cancer cells induced highly specific neuronal injury through myelin degradation, followed by the initiation of an inflammatory response by the injured neurons themselves, supporting the role of neurons as immune regulators. After a peripheral nerve injury, neurons attract immune cells to the perineural niche to promote healing and neuroregeneration36. This healing process is led by immunosuppressive macrophages, inducing local immune exhaustion and suppression16. Mice were intradermally injected with MOC2 SCC cells into their whisker pads, a highly innervated area43, followed by tumour extraction at different timepoints (7, 9, 11 and 13 days after tumour inoculation). Immune activity was assessed by mIF stains (Fig. 3b), using markers of immunosuppressive macrophages (CD68+CD163+PD-L1+) and exhausted CD8+ T cells (CD8+TIM3+). This analysis demonstrated that an increase in the abundance of NFH+ATF3+ injured nerves was coupled with an increase in the intratumoural infiltration of immunosuppressive macrophages and exhausted CD8+ T cells in the entire TME. Our first goal was to demonstrate that the origin of immunosuppression was the injured nerves. To do so, we used mIF to assess immune infiltration in the perineural niches of nerves with CINI (NFH+ATF3+) versus nerves without CINI in cSCC tumours collected from treatment-naive patients. The analysis showed increased infiltration of immunosuppressive macrophages (CD163+PD-L1+) and neutrophil extracellular traps (MPO+C3H+, previously associated with post-nerve injury neuroinflammation44; Fig. 3c (top)) in the perineural niches of CINI nerves. These findings were validated in patients with PDAC45—a highly neurotropic cancer with a well-established deleterious effect of PNI on its clinical outcome46. The CINI mIF pattern in treatment-naive patients with PDAC was similar to that of cSCC (Fig. Next, we validated the mIF findings using an independent cohort. PDAC tumours, which were resected from seven treatment-naive patients, underwent CosMx spatial transcriptomics analysis to profile the immune infiltration in the perineural niches. TANs were divided into three categories based on their ATF3 or JUN expression—injured (expression of both ATF3 and JUN), intermediate (expression of one marker) or non-injured (not expressing either transcript). Perineural niches were defined as areas in the TME within 150 μm from TANs47. Compared with healthy nerves, injured nerves had a higher infiltration of CD163+ macrophages and LAG3+CD8+ T cells (Fig. This consistency across different cancer types, patient cohorts and scientific methods strengthens our initial observations and suggests a common mechanism of CINI-related immune effect across different neurotropic cancers. a, Quantification of nerves invaded by SCC (MOC2 cells) injected into mouse whisker pads (n = 12). mIF shows an invaded nerve (asterisk) lacking MBP, suggesting myelin degradation. b, mIF analysis of the whisker pad tumours collected over time. c, mIF tumours collected from treatment-naive patients with cSCC and PDAC; nerves are indicated by the dashed lines. d, Immune cell density in the perineural niche in treatment-naive PDAC samples (n = 7), based on CosMx analysis. Statistical analysis was performed using two-tailed Mann–Whitney exact tests. e, Heat map of nerve-injury-related proteins in the perineural niches among the patients in our anti-PD-1 cSCC trial, measured using DSP, with data transformed into z scores (Supplementary Table 8). f, Bubble heat map based on a DSP protein matrix showing the Pearson's correlation coefficients between immune and neural proteins expressed in the perineural niches of neoadjuvant-treated tumours (patients in the anti-PD-1 cSCC trial). Statistical analysis was performed using pairwise Pearson correlation. Statistical analysis was performed using two-tailed Student's t-tests with the assumption of equal variances (pooled t-test). h, Spatial transcriptomics of tumour samples from an independent treatment-naive cohort of patients with cSCC (see the main text), assessing the co-localization of three phenotypes: CINI, immunosuppressive inflammation and antitumoural immunity (55-μm resolution per spot). A linear regression line (blue) was created based on the least-squares method; the shaded region shows the 95% CI; statistical analysis was performed using pairwise Pearson's correlation. To spatially test the interaction between CINI and immune activity in the perineural niche in the context of anti-PD-1 therapy, our cSCC neoadjuvant clinical trial samples were assessed using GeoMx digital spatial profiling (DSP). Neural niches were identified by immune labelling of the pan-neuronal markers NFH and B3T (Extended Data Fig. The protein expression in neoadjuvant-treated samples of non-responders (n = 109 ROIs) was compared to those of responders (n = 360 ROIs). Tumours from non-responding patients demonstrated increased expression of neurodegeneration protein markers such as PARK7, PARK5, PINK1 and LRRK2 (all FDR < 0.01; Fig. Analysis of the pretreatment tumour samples showed that the neuroprotective protein APOA-I was overexpressed in the perineural niches of responders versus non-responders (FDR < 0.01; Extended Data Fig. 7e), suggesting a lower degree of CINI among responders. Moreover, the CD31 expression level did not differ between responders and non-responders in the pretreatment samples (FDR = 0.76; Extended Data Fig. 7f), further suggesting that CINI is a nerve-specific process that does not affect nearby blood vessels. Perineural niches of neoadjuvant-treated non-responders showed a direct and significant correlation between markers of neuronal response to injury and both immune activation and suppression markers (Fig. By contrast, the perineural niches of responders showed mostly an inverse correlation between markers of neuronal injury and immune markers. These findings were validated using mIF stains. The perineural niches (defined as an area within 150 μm of the TAN epicentre47) in neoadjuvant-treated samples showed a higher abundance of CD68+CD163+, CD8+PD-1+ and CD8+LAG3+ cells among non-responders compared with responders (P = 0.055, P = 0.078 and P = 0.095, respectively; Fig. To demonstrate that the CINI-related inflammatory response extends beyond perineural niches, we conducted a spatial transcriptomic analysis using fresh tumour samples from an independent, treatment-naive cohort of patients with cSCC (n = 11) who underwent Mohs surgery at The University of Texas MD Anderson Cancer Center (Supplementary Table 4). The spatial analysis revealed a correlation between the CINI and immunosuppressive inflammation phenotypes (r = 0.44), but no correlation with antitumoural immunity (r = 0.13; Extended Data Fig. Figure 3h shows the spatial distribution of these phenotypes in individual tissue sections, visually representing the relationship between CINI and immunosuppressive inflammation across the tumour microenvironment. These findings were further validated using mIF of treatment-naive patients with cSCC in two distinct tumoural regions: the perineural niche (immune cells within 100 μm of the nerve epicentre) and nerve-remote areas (immune cells more than 1,000 μm away from the nerve epicentre). To focus on the active tumour areas, we excluded nerve-remote ROIs that contained non-active tumour sites, such as stroma. This analysis demonstrated that the presence of CD8+LAG3+ and CD163+PD-L1+ immunosuppressive cells in the perineural niche of nerves with PNI corresponded with similar infiltration patterns in nerve-remote areas (r = 0.5, P = 0.011; and r = 0.67, P < 0.001, respectively; Fig. To establish causality between CINI and immunosuppressive activity in the TME, we conducted a series of in vivo experiments. To increase the robustness, we used a third cancer model, melanoma, as PNI has a critical deleterious effect on its clinical outcome9. First, we inoculated either B16F10-OVA melanoma cells or non-tumorigenic keratinocytes into mouse hind paws (Extended Data Fig. Then, 14 days after inoculation, the mice were euthanized, and the L3–L5 DRG neurons innervating the tumour/keratinocytes-inoculated paw were collected and analysed using single-cell RNA-seq (scRNA-seq; n = 32,951 neurons from 8 mice). Cancer-injured neurons were characterized by high expression of neuronal injury markers, most prominently Atf3 (Fig. 9b), and upregulation of multiple immune-related pathways, such as Hallmark inflammatory response and Hallmark IL-6 JAK STAT3 signalling (FDR < 0.001; Extended Data Fig. Atf3 is an adaptive response gene activated by stressful stimuli22. To test the effect of CINI signalling blockade on tumour growth and intratumoural immune activity, we used a conditional cre-lox Atf3 knockout (Atf3-cKO) mouse model, limiting the cKO only to nociceptors neurons (Nav1.8cre::Atf3fl/fl mice versus their permissive littermate controls (Nav1.8WT::Atf3fl/fl, termed wild type (WT)). B16F10-OVA melanoma cells were injected into both Atf3-cKO and WT mice. mIF stains showed that CINI occurred in both mouse groups, as nerves with degraded myelin were present in both the WT and the Atf3-cKO tumours (MBP–dMBP; Extended Data Fig. 9h), and increased tumour infiltration of IFNγ+CD8+ T cells (P = 0.04; Fig. 10a), suggesting a superior antitumoural immune activity within Atf3-cKO tumours. To test this hypothesis, we conducted scRNA-seq analysis of intratumoural immune cells. There was no difference in the proportional abundance of these clusters among the two conditions (Fig. 4e), suggesting a functional difference between the groups rather than immune trafficking differences. Our analysis revealed that CD8+ T cells in Atf3-cKO tumours (for which CINI neuron stress signalling was impaired) had a lower expression of exhaustion markers such as Lag3 (P = 0.05; Fig. Moreover, macrophages from Atf3-cKO tumours demonstrated an M1-like phenotype17 (Fig. 4g), which is associated with reduced immunosuppression and improved tumour control49. By contrast, WT macrophages were enriched with inflammatory pathways such as IL-6 signalling (Fig. Overall, the Atf3-cKO model demonstrated that ameliorating CINI intraneuronal injury signalling resulted in reduced immunosuppressive/exhaustive activity in the general TME of melanoma tumours. a, Uniform manifold approximation and projection (UMAP) plot of scRNA-seq data of DRG neurons innervating mouse paws that were inoculated with either B16F10-OVA melanoma or normal keratinocytes (Extended Data Fig. b, Cancer-injured neurons were characterized by an enrichment of nerve-injury-related genes. c, Tumour growth curve comparing the tumour growth of melanomas inoculated into mice with a cre-flox conditional knockout of Atf3 in nociceptor neurons (Atf3-cKO, n = 7) versus their permissive littermate controls (NaV1.8WT::Atf3fl/fl, WT, n = 6). Tumour growth over time (P < 0.001) was analysed using a mixed-effects model; post hoc comparisons for individual timepoints were conducted using Šídák's multiple-comparison test (P < 0.001). Statistical analysis was performed using non-paired two-tailed Student's t-tests. e, scRNA-seq analysis of intratumoural (melanoma) immune (CD45+) cells in Atf3-cKO versus WT mice (Extended Data Fig. f, Expression of Lag3 and Tox genes in CD8+ T cells from the immune scRNA-seq experiment presented in e. g, Expression of M1 macrophage markers (Il1b+Cd86+Cd80+, associated with enhanced antitumoural immune activity) in intratumoural macrophages. Statistical analysis was performed using two-tailed Wilcoxon rank-sum tests; significance was established through normal approximation. Blinded analysis of immunohistochemical staining demonstrated no differences in CD8+ T cell abundance between responders and non-responders before or after treatment (Fig. 5a), further suggesting a functional impairment of intratumoural CD8+ T cells rather than reduced trafficking. 5c and Supplementary Table 6), such as T cell activation (GO:0042110; FDR = 0.001), and negative regulation of IL-6 production (GO:0032729; FDR = 0.03). NanoString nCounter PanCancer immune profiling demonstrated upregulation of IFNα and IFNβ (IFN-I) signalling in the tumours of neoadjuvant-treated non-responders (Fig. This finding, echoing the results of our in vitro neuron-cancer co-culture (Fig. 2j), might initially seem counterintuitive as IFN-I signalling has been associated with a favourable response to immunotherapy. However, while acute IFN-I signalling promotes antitumoural immunity, chronic IFN-I signalling can result in immunosuppression as a negative feedback loop, leading to immunotherapy resistance50. a,b, Immunohistochemistry-based cell counts of intratumoural cells in the cSCC clinical trial cohort, according to the anti-PD-1 clinical response. MHC, major histocompatibility complex; TCR, T cell receptor. d–f, Nanostring nCounter PanCancer analysis of neoadjuvant-treated samples showing differentially expressed genes (d), T regulatory (Treg) cells and TGFB1 expression (e) and pathway enrichment analysis (f) according to response status. The box plots show the median values (centre line), 25th and 75th percentiles (box limits), and the minimum and maximum values (whiskers); each dot corresponds to a clinical sample. The presented pathways were considered to be significantly enriched at FDR < 0.2. g–i, All three panels represent tumour growth plots for the IFNα receptor subunit 1 knockout (Ifnar1-KO, Ifnar1−/−) mouse experiment. k, Experimental design (created using BioRender) of the IL-6R blockade (anti-IL-6R) experiment. l, Tumour growth curves of the IL-6R blockade experiment. Each point represents an experimental animal. For all tumour growth curves, data are mean ± s.e.m. Statistical analysis was performed using a mixed-effects model with REML estimation. Post hoc comparisons at individual timepoints were evaluated using Tukey's multiple-comparison test. For all bar plots, data are mean ± s.e.m. Statistical significance was evaluated using one-way ANOVA (P < 0.0001) followed by the Tukey–Kramer post hoc test. IL-6 consistently showed a strong link to CINI across our experiments. IFN-I demonstrated a connection to CINI in several of our tests and, crucially, was associated with resistance to anti-PD-1 in our clinical trial tumour samples. On the basis of these findings, we investigated whether blocking IFN-I or IL-6 signalling in a CINI model could enhance anti-PD-1 efficacy. First, we used an IFNα receptor subunit 1 knockout (Ifnar1-KO) mouse model53. Seventeen days after inoculating UVSCC-M4 cells into Ifnar1-KO and WT (C57 black) mice, the mice underwent injection of a demyelinating agent (ethidium bromide, EtBr) into the tumour–normal skin interface versus a sham PBS injection to the same location. Anti-PD-1 or IgG control treatment was initiated 2 days after the sham/EtBr injection. By contrast, Ifnar1-KO mice were unaffected by demyelination, as Ifnar1-KO sham, Ifnar1-KO EtBr and WT sham mice had similar tumour sizes after anti-PD-1 therapy (Fig. The addition of STING agonists, which promote IFN-I signalling, did not improve anti-PD-1 efficacy. As there was no correlation between the tumour size based on imaging with the pathological response to treatment in the neoadjuvant anti-PD-1 cSCC clinical trials (51% pathological complete response, CR, rate versus only 6% of radiological CR)20, histological cancer cell viability was assessed for each collected tumour. On the basis of cancer viability, demyelination adversely affected anti-PD-1 efficacy in WT mice (Fig. Despite the significant difference in tumour size, the differences in tumour viability between demyelinated Ifnar1-KO and WT mice treated with anti-PD-1 did not reach statistical significance. Next, we tested the effect of IL-6 signalling blockade on anti-PD-1 efficacy. One week after the inoculation of B610K cSCC cells, immunocompetent SKH1-Hrhr mice underwent either EtBr injection into the tumour–normal skin interface or a sham PBS injection, followed by initiation of anti-PD-1 treatment 2 days later (Fig. Subsequent anti-PD-1 doses were given alone or in combination with antibodies blocking the IL-6 receptor (IL-6R). The addition of anti-IL-6R antibodies to anti-PD-1 did not reduce the tumour size in a statistically significant manner (Fig. However, similar to the efficacy results of the clinical trial, there was a difference in tumour viability. Compared with anti-PD-1 monotherapy, the combination of anti-PD-1 and anti-IL-6R significantly reduced tumour viability among mice with demyelination (P < 0.001; Fig. The anti-IL-6R reduction in tumour viability did not occur in mice without demyelination, supporting the hypothesis that IL-6 activity in the tumours was not due to an inherent IL-6 secretion from the tumour cells themselves but, rather, the result of an inflammatory response driven by CINI. Although the role of the nervous system as an immune regulator has gained recognition in recent years12,55, little is known about the PNS effect on antitumoural immunity. Our data, based on rigorous and extensive experiments such as multiomics analysis of patient cohorts with multiple PNI-associated cancer types, in vitro assays of different cell lines (including neurons from both mice and humans) and in vivo models using different mice (including genetically modified mice), has consistently suggested that CINI confers resistance to anti-PD-1 therapy. We proposed a mechanism driving this effect. Cancer cells cause myelin degradation in nerves, as evident by myelin structural changes and abnormal nerve electrical conduction studies. The myelin degradation results in nerve injury. ATF3, a response gene for neuronal injury, mediates intraneuronal injury signalling that leads to upregulation of IFN-I and IL-6 pathways. As the tumour progresses, it expands to nearby nerves and causes more CINI, which in turn increases the injured neurons' inflammatory signalling, turning it into a chronic process. Chronic infiltration of pro-healing suppressive immune cells into the perineural niches eventually affects the general immune tone in the TME, dampening overall antitumoural immunity and response to anti-PD-1. Targeting different steps in the CINI pathway, including Atf3-cKO, Ifnar1-KO or antibodies blocking IL-6 signalling, reversed immunosuppression and restored anti-PD-1 efficacy. Our findings suggest that the observed reduction in neuron viability and signalling is specifically associated with the process of CINI/PNI rather than a general cancer-induced effect, highlighting the importance of cancer cell invasive behaviour. This effect may represent an initial phase of cancer–nerve interaction, with the potential for surviving or regenerated neurons to contribute to cancer growth over time. Our co-culture system mimicked the conditions of active PNI, providing insights into direct cancer–neuron interactions. In conclusion, our findings suggest an anti-PD-1 resistance mechanism mediated by CINI and its associated chronic inflammation. These findings might also serve as a basis for identifying biomarkers and the development of therapeutic agents that might overcome anti-PD-1 resistance in patients with multiple cancer types. Supplementary Table 10 summarizes the different clinical cohorts used in this Article, and presents the results of their relevant analysis. The full description of each cohort is depicted in the text and in Supplementary Table 10. The clinical trials protocol, statistical analysis plan, and institutional review board approvals were previously reported19,20. Patients older than 18 were eligible if they had resectable stage II–IV (M0) CSCC, for which primary surgery would be recommended in routine clinical practice. Patients with stage II CSCC should have a primary tumour ≥3 cm in longest diameter to be eligible. Patients were additionally required to have adequate organ function (determined by assessment of complete blood cell count and comprehensive metabolic function), at least one measurable lesion on the basis of the Response Evaluation Criteria in Solid Tumours version 1.1 (RECIST 1.1)57 and an Eastern Cooperative Oncology Group (ECOG) performance status of 0 or 1. After a screening period of up to 28 days, the patients received the neoadjuvant cemiplimab (Regeneron Pharmaceuticals) 350 mg intravenously every 3 weeks until unacceptable toxicity, disease progression or withdrawal of consent. After completion of neoadjuvant treatment, the protocol window for surgery was study days 75–100. Pretreatment biopsy specimens were subjected to histopathological assessment for confirmation of diagnosis and to permit morphologic comparison between tumour tissue before treatment and any residual tumour following therapy. Pathologic response was assessed in the post-treatment surgical specimens according to standard pathologic evaluation recommendations58 and re-reviewed by a dedicated dermatopathologist (P.N.) The primary end point was pathologic complete response, defined as the absence of viable tumour in post-treatment surgical specimens, determined on the basis of independent central pathology review. Major pathologic response was a secondary end point defined by the presence of >0% but ≤10% viable tumour cells in post-treatment surgical specimens. Further clinical information regarding the trial patients is provided in Fig. The allocation of the trial patient tumour samples into different analyses is described in Extended Data Fig. Immunohistochemical analysis was performed using 4-μm-thick formalin-fixed and paraffin-embedded (FFPE) cSCC tissue samples obtained before and after anti-PD-1 therapy. Staining was performed using antibodies against CD8 (Thermo Fisher Scientific, MS-457s), PD-1 (Abcam, ab137132) and PD-L1 (Cell Signaling Technology, 13684S) using a BOND-RX instrument with a Bond Polymer Refine Detection Kit (Leica Biosystems, DS9800). Stained slides were scanned and digitalized using a Scan Scope XT system (Aperio/Leica Biosystems). Single-stain annotations were performed by a pathologist, and staining quantification was done using HALO software (v.2.3.2089.70; Indica Labs). The number of marker-positive cells was calculated and expressed as positive-cell density (number of positive cells per mm2). In brief, FFPE slides were placed in tanks containing BioGenex EZ Elegans AR 2 buffer, enclosed inside a BioGenex EZ Retriever microwave system. After heating, the slides were cooled to room temperature, loaded into the Comet system, and covered and sealed by a fast-fluidic exchange microfluidics chip for staining and imaging in the instrument. The Comet system performs a sequential IF-based staining and imaging followed by an elution of the primary and secondary antibodies. Optimal staining conditions for each antibody panel were determined using the ‘Characterization Part 2' instrument protocol, in which antibodies are titrated for optimal signal and the elution efficiency is verified by an iterative procedure comprised of staining, imaging, elution and re-imaging for elution confirmation for all antibodies. Two panels (m-neuroimmune panel; and m-myelin degradation panel) were used for mouse-derived tissues (Supplementary Table 10). The Comet collected images in OME-Tiff formats, which were scanned and then visualized in Comet Viewer (Lunaphore). Image analysis was performed using Oncotopix Discovery (Visiopharm, v.2023.01)60,61. A deep learning classifier was trained using specific tissue features from annotated reference images to guide the identification of ROIs. In brief, a tissue ROI was created after training two variables (tissue and non-tissue regions). Tissue and ROI boundaries were then delineated using image analysis smoothing. Aberrant signals, such as those caused by dust particles, tissue folds or air bubbles, were excluded manually from the ROIs. Another deep learner classifier was developed and trained for nerve identification following two training variables (nerve and non-nerve structure) within the delineated tissue ROI regions. The following training parameters were used to develop the nerve identification app: input: NFH and B3T; learning parameters: learning rate-1.0e-05; Mini-batch size-2; loss function: cross-entropy; iteration: 300,000. Cells in non-nerve areas (NFH−B3T−) were identified by watershed desegmentation of the nuclear DAPI signal and removal of incomplete nuclei by size exclusion. The resulting nuclear mask was enlarged by 1.25 μm (5 pixels) to capture nuclear and adjacent cytoplasmic fluorescence signals. Cell types were identified using a hierarchical decision tree with empirically determined thresholds visually verified across the tissues. NFH+B3T+ nerves were segmented using the NFH/B3T signal intensity. Nerve nuclei were excluded a priori from the initial cell classification strategy, and only those that exceeded the threshold for NFH+B3T+ levels were incorporated into the nerve mask. Neural niches were determined to have a 150-μm ROI diameter around each nerve (NFH+B3T+). All object-based phenotyping resulting tables were exported as CSV files for downstream analyses. Staining was performed 4–8 min per cycle for primary antibodies, diluted 1:50–1:2000 in a Multistaining Buffer (Lunaphore). Secondary antibodies, AlexaFluor 555 and AlexaFluor 647 (Thermo Fisher Scientific) were diluted 1:200 and 1:400, respectively, in a multistaining buffer. Tissues were counterstained with DAPI solution (Thermo Fisher Scientific) at every cycle. cSCC anti-PD-1 clinical trial FFPE tumour samples were used for DSP of the neural and immune protein expression (see below the description of assays and markers used). First, we mapped the neural niches using haematoxylin and eosin (H&E)-stained tissue sections, including the tumour and surrounding tissue. These slides were scanned using a Scan Scope digital pathology system (Aperio) in SVS format at a 40× magnification and visualized using Image Scope software (Aperio). All nerve profiles in the specimen were annotated, including PNI. The extent of viable tumour or tumour bed and uninvolved tissue was drawn or designated on whole-slide scans. Next, two consecutive 5-μm-thick tissue sections from samples obtained before and after anti-PD-1 therapy were stained according to the semiautomated GeoMx DSP standard protein protocol62 using the BOND-RX system to profile TANs and their perineural microenvironments (neural niches). Both sections were stained for the following morphology biomarkers: panCK, SYTO 13 (GeoMx Solid Tumour TME Morphology Kit, NanoString, 121300301), β-III-tubulin (EP1569Y, AF, Abcam, ab52623; 1:2,000 mg ml−1) and neurofilament (EPR20020, AF, Abcam, ab207176; 1:1,000 mg ml−1; Extended Data Fig. Optimization of IF biomarkers was previously performed with different antibody dilutions using normal colon tissue to achieve the highest signal-to-noise ratio. One tumour section was used to profile TANs with the DSP neural cell profiling core (GMX-PROCO-NCT-HNCP-12) and GeoMx Parkinson's pathology panel (GMX-PROMOD-NCT-HPDP-12). A serial tissue section was used to profile the perineural microenvironment with the following DSP human immuno-oncology protein core panel and modules: GeoMx Immune Cell Profiling (121300101), GeoMx IO Drug Target Assay (121300102), GeoMx Immune Activation Status Assay (121300103) and GeoMx Immune Cell Typing Assay (121300104; 49 protein targets). The full list of targets is described in Supplementary Table 10. After scanning using the GeoMx DSP device, mIF imaging slides were visualized with the adjustment of channel thresholds for each fluorophore. ROIs were selected after pathological evaluation of sequential sections of H&E-stained nerve fibres in tumour tissue and tumour bed tissue identified by an expert pathologist (P.N.). Using a two-step strategy, a polygon and rectangle selection tool was applied to select ROIs of up to 660 × 785 mm. (1) Nerve profiling: a slide labelled with a DSP neuroprotein panel was used to select up to 12 intratumoural ROIs containing tumour-associated nerve fibres. Neural niches were identified and segmented using the β-III-tubulin (+), Neurofilament Heavy (NF-H) (+) and panCK (−) phenotype. (2) Perineural niche immune profiling: a consecutive section labelled with a DSP human immuno-oncology protein panel was used to select up to 12 matching ROIs with TANs. The matching perineural microenvironment compartment, that is, neural niches, were identified using the same morphology markers (β-III-tubulin (+), neurofilament heavy (NF-H) (+) and panCK (−) phenotype). Segmented areas (also known as areas of illumination) were illuminated individually using ultraviolet light with the GeoMx DSP device. Oligonucleotide tags conjugated with antibodies present within each area of illumination were photocleaved. Released tags were quantified using nCounter, and tag counts of the various markers were mapped back to their corresponding tissue locations, yielding a spatially resolved digital profile of analyte abundance. Digital counts were normalized using background correction. DSP data analysis software was used to visualize protein expression patterns and perform statistical analysis62. These data are available in Supplementary Table 8. Nucleic acid extraction, library preparation and sequencing: DNA/RNA extraction was performed using the Mag-Bind FFPE DNA/RNA 96 Kit (Omega Bio-Tek) according to the manufacturer's protocol. Libraries were constructed with KAPA RNA HyperPrep with RiboErase (Roche) and performed based on the manufacturer's recommendations. The final library quantity was measured by KAPA SYBR FAST qPCR, and library quality was evaluated by TapeStation D1000 ScreenTape (Agilent Technologies). The final library size was about 430 bp with an insert size of about 200 bp. Equimolar pooling of libraries was performed based on quality-control values and sequenced on the Illumina NovaSeq system (Illumina) with a read length configuration of 150 PE for 80 million paired-end reads per sample (40 million in each direction). Extracted genomic DNA was then quantified using the Qubit 2.0 DNA HS Assay (Thermo Fisher Scientific), and the quality was assessed using the Tapestation genomic DNA Assay (Agilent Technologies). Library preparation was performed using SureSelectXT Low Input Reagent Kits (Agilent Technologies) according to the manufacturer's recommendations. Library quality and quantity were assessed using the Qubit 2.0 DNA HS Assay (Thermo Fisher Scientific), Tapestation High Sensitivity D1000 Assay (Agilent Technologies) and QuantStudio 5 System (Applied Biosystems). Equimolar pooling of libraries was performed based on quality-control values and sequenced on the Illumina NovaSeq (Illumina) system with a read length configuration of 150 PE for 130 million paired-end reads (65 million in each direction) or 26 million paired-end reads (13 million in each direction). RNA-seq data processing and analysis: raw paired-end reads in FASTQ format were checked for read quality using FastQC (v.0.11.8; http://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Illumina TruSeq adapters were trimmed from the paired-end reads using cutadapt (v.1.18). Genes were annotated using the gene transfer format file for GRCh38. Read counts were normalized using the trimmed mean of M method implemented in the R Bioconductor package edgeR to determine the abundance of each gene. The generalized linear model likelihood ratio test from edgeR was used to identify DEGs between groups. These data have been deposited in the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) under accession umber GSE289743. The PNI signature was based on an independent and previously published gene expression signature associated with PNI in cSCC of the head and neck30. The initial dataset consisted of DEGs from tumours lacking PNI compared with those exhibiting incidental PNI, identified solely through histopathological evaluation (n = 2,412). The second dataset included DEGs from tumours with no PNI versus those with clinical PNI, detected using imaging techniques (n = 7,793). To curate a gene list that captures the molecular features associated with any form of PNI (both incidental and clinical), we focused on DEGs that were upregulated in both datasets. In our analysis, DEGs represented by multiple microarray probes were excluded if one or more probes failed to satisfy our inclusion criteria. Specifically, cases in which one probe suggested significant upregulation of a gene in PNI-positive cases, while another indicated either downregulation in PNI or no significant difference, were not considered. This 46-gene PNI/nerve injury signature was first tested on our anti-PD-1 neoadjuvant cSCC clinical trial bulk RNA-seq data using GSEA. GSEA calculates an enrichment score (ES) for gene sets by ranking genes based on their overrepresentation in two groups. A positive ES indicates enrichment in one group, while a negative ES indicates enrichment in the other. We considered the gene sets to be significantly enriched at a threshold of P < 0.05. We did not apply corrections for multiple-hypothesis testing in this case, as only a single gene signature was assessed. The 46-gene PNI/nerve injury signature was later validated in external, publicly available cohorts of patients who had PNI-associated tumours and were treated with anti-PD-1 therapy. The bulk RNA-seq data of three patient cohorts were available (two melanoma32,33 and one gastric cancer34; Supplementary Table 2): Reference 33: this study included 88 patients with metastatic melanoma, receiving either nivolumab (n = 39) or pembrolizumab (n = 49). For GSEA, patients with CR and PR were classified as responders (n = 41, 46.6%), while those with PD were categorized as non-responders (n = 47, 53.4%). The response distribution was: CR = 4 (9.7%), PR = 15 (36.6%), stable disease (SD) = 6 (14.6%) and PD = 16 (39%). In GSEA analysis, the CR and PR patients were grouped as responders (n = 19, 46.3%), while SD and PD patients were grouped as non-responders (n = 22, 53.7%). Reference 34: this study comprised 45 patients with metastatic gastric cancer who received pembrolizumab as salvage therapy, with available RNA-seq data. Patients were treated with pembrolizumab in second-line or third-line settings for metastatic disease. The response rates recorded were: CR = 3 (6.6%), PR = 9 (20%), SD = 15 (33.3%) and PD = 18 (40%). For GSEA, those with CR and PR were considered responders (n = 12, 26.7%), while patients with SD and PD were classified as non-responders (n = 33, 73.3%). RNA was isolated from FFPE tumour sections by dewaxing using a deparaffinization solution (QIAGEN), and total RNA was extracted using the RecoverAll Total Nucleic Acid Isolation Kit (Ambion) according to the manufacturer's instructions. The RNA purity and quantity were assessed using a NanoDrop ND-1000 spectrometer (Thermo Fisher Scientific). For a NanoString assay, 100 ng of RNA was used to detect immune gene expression using a nCounter PanCancer Immune Profiling Panel along with a custom CodeSet (NanoString). Counts of the reporter probes were tabulated for each sample using a nCounter Digital Analyzer, and raw data output was imported into the nSolver software program (v.4.0; NanoString). Moreover, the nSolver (Advanced Analysis 2.0) data analysis package was used for the normalization of expression levels, cell type and differential gene expression analyses. Gene set enrichment analysis was performed with Qlucore Omics Explorer software (v.3.7). Data were plotted using Prism software (v.9; GraphPad Software) and two-tailed Mann–Whitney U-tests were performed to compare groups. These data are provided in Supplementary Table 9. Non-trial original clinical cohorts were used in this Article: (1) 32 treatment-naive patients with stage I–II cSCC whose tumours were excised at MDACC (The University of Texas MD Anderson Cancer Center) before February 2023. The TMA was used to characterize immune infiltration in perineural niches using mIF. (2) 187 treatment-naive patients with stage I–II PDAC whose tumours were resected at MDACC between 2004 to 2015. Archived tumours from these patients were used for the construction of another TMA, as described before64. This TMA was also used to characterize immune infiltration in perineural niches using mIF as a means of external validation. (3) 11 treatment-naive patients with stage I–II cSCC whose primary tumours were excised by Mohs surgery at MDACC between 2020 and 2022 (Supplementary Table 4). Their fresh-frozen collected tumours were used for spatial transcriptomic analysis. (4) 7 treatment-naive patients with PDAC whose tumours were resected at Massachusetts General Hospital, provided to this study by W. Hwang. In brief, these patients with locally advanced PDAC underwent surgical resection of the primary tumour with or without previous neoadjuvant chemotherapy and radiotherapy. Fresh-frozen collected and FFPE archived cSCC primary tumours were used to demonstrate the spatial relationship between nerve injury and different immune phenotypes. The tumours were prepared for sequencing according to the manufacturer's instructions (10x Genomics, Visium Spatial) using the following kits: Visium Spatial Tissue Optimization Slide & Reagent Kit, 4 slides PN-1000193; Visium Gateway Tissue Optimization Slide & Reagent Kits, 1 slide (PN-1000313, PN-1000314); Visium Accessory Kit, PN-1000194. The prepared libraries were pooled and sequenced on the NovaSeq 6000 (Illumina) system, generating ~40 million 2 × 150 base paired-end reads per sample. The spaceranger (v.1.3.0) mkfastq pipeline was used to convert Illumina sequencer's binary base call (BCL) files into FASTQ format. The samples were then run through the spaceranger count pipeline, which performs alignment, tissue detection, fiducial detection and barcode/unique molecular identifier counting. Reference-sequence alignment of human samples was performed using the GRCh38 Reference 2020-A (23 June 2020). Images were processed using Fiji to divide them into individual capture areas and then rotated to align the fiducial spot patterns65. The pipeline uses Visium spatial barcodes to generate feature-spot matrices, determine clusters and perform gene expression analysis. Functions in the R package Seurat (v.4.1.1) were used for downstream analysis66. Spots that did not overlap with the tissue sections or had ≥20% mitochondrial reads (10% for humanized mouse) or had <200/500 detected genes were removed from the downstream expression analysis. Expression counts were normalized using Seurat's NormalizeData. The normalized expression data were further scaled to mean 0 and variance 1 using the function ScaleData. RunPCA and RunUMAP were used for dimensionality reduction. Sample batch effect correction was performed using RunHarmony. Clustering was done using the functions FindNeighbors and FindClusters. We performed transcriptional phenotyping for each sequenced tissue section based on the molecular characteristics of TANs and tumour immune infiltrate using gene signatures obtained from the literature67. Nerves were scored according to the CINI signature that included the following genes: TAGAP, KCNJ8, COL1A1, PECAM1, TMEM119, ATF3, JUN, KLF6, NOCT, LMO7, CSF1, ENTPD1, UCHL1, PINK1, BHLHE41, ITGAM, CHL1, SNCA, SCPEP1 and VEGFA. Spatial feature expression plots were generated using the SpatialFeaturePlot function. One-way ANOVA with Tukey's post hoc test was used to assess the significance of the group difference. These data have been deposited in the NCBI GEO under accession number GSE289745. Nanostring CosMx spatial molecular imaging (MDACC-external PDAC cohort): this spatial molecular imaging analysis was conducted on human PDAC primary tumours provided by W. Hwang (see above for a description of the clinical cohort). FFPE tissue sections (5 µm) were prepared and mounted onto Superfrost Plus Micro Slides (VWR) under RNase-free conditions. The samples underwent deparaffinization, proteinase K digestion and heat-induced epitope retrieval using the Leica Bond RX system. After preparation, Nanostring CosMx spatial transcriptomics imaging was performed on the sections. Perineural niches were defined as regions within 150 μm of TANs in the TME. RNA in situ hybridization probes targeting a 960-plex base panel, along with 30 additional custom-selected probes, were hybridized overnight at 37 °C. After hybridization, the samples were washed and blocked and protein staining was performed using a fluorophore-conjugated antibody cocktail targeting CD298, B2M, PanCK, CD45 and CD3 proteins, along with DAPI for nuclear staining. Data acquisition and image processing were conducted using an in-house spatial molecular imaging data-processing pipeline45. z-stack images were collected, and cell segmentation was performed using Cellpose (v.2.0.5), with robustness evaluated through comparison to Baysor (v.0.6.0). Transcripts were assigned to individual cells based on location and cell segmentation boundaries. Expression profiles for individual cells were normalized and log-transformed. Cell types were annotated using Insitutype (v.1.0.0) based on protein and RNA expression profiles, with malignant and nonmalignant cells identified using PanCK expression and cytokeratin RNA markers. Batch correction across tissue slides was performed using ComBat (sva, v.3.46.0). These data have been deposited in the NCBI GEO under accession number GSE199102. The different cell types used for in vitro assays in this study, along with the figure panel presenting the results of their relevant analysis, are summarized in Supplementary Table 10. For in vitro experiments in which there were replicate samples, we repeated the experiments at least three times to confirm the findings. RealDRG cryopreserved hPS-cell-derived sensory neurons (nociceptors). Using the Senso-DM directed differentiation process on internal hiPS cell lines, we can generate billions of neurons at scale and at 99% purity—without the need for mitotic inhibitors. Principal component analysis shows that RealDRG neurons mature in the shortest time and are closer to primary human DRG tissue compared with other hiPS-cell-derived sensory neurons (https://www.anatomic.com/realdrg). Donors provided written informed consent to donate tissue samples for the study, for which the protocol was reviewed and approved by The University of Texas MD Anderson Cancer Center Institutional Review Board. In brief, each donor was undergoing surgical treatment that necessitated ligation of spinal nerve roots to facilitate tumour resection or spinal reconstruction. Spinal roots were ligated proximal to the DRG, spinal roots were sharply cut both proximal and distal to the DRG and excised DRGs were transferred immediately into a cold (∼4 °C) and sterile balanced salt solution containing nutrients. DRGs were transported to the laboratory on ice in a sterile, sealed 50 ml centrifuge tube. After arrival at the laboratory, each ganglion was carefully dissected from the surrounding connective tissues and sectioned into several 1–2 mm pieces, digested in 2 ml of a mixed enzyme solution (0.1% trypsin (Sigma-Aldrich, T9201), 0.1% collagenase (Sigma-Aldrich, C1764; w/v, final concentration) and 0.01% DNase (Sigma-Aldrich, D5025) diluted in DMEM/F-12), and transferred to a 37 °C rotator to shake at a speed of 124–128 rpm. Every 20 min, tissue fragments were allowed to settle, and the supernatant/dissociated cells were collected and transferred to DMEM/F-12 with an enzyme inhibitor. The supernatant was replaced with 2 ml of fresh digestion solution. The tissue was returned to the 37 °C rotator, and this process was repeated until tissue fragments were completely digested. Dissociated cells were centrifuged at 180 rpm for 5 min, the supernatant was removed and the cells were gently resuspended in culture medium with DMEM/F-12 supplemented with EV-free 10% serum and 2 mM glutamine. Cells were plated onto laminin-coated µ-Slide 8 Well (Ibidi) and cultured at 37 °C with 5% CO2 for 24–72 h. hIPS-cell-derived motor (RealMOTO) and sensory neurons (RealDRG) were purchased from Anatomic. After TG dissection, tissue was enzymatically digested with papain (40 U ml−1, EMD Millipore) for 20 min at 37 °C followed by 20 min of digestion with collagenase II (4 mg ml−1)/dispase II (4.6 mg ml−1) solution. Using the Percoll gradient, comprising 12.5% and 28% Percoll in complete L-15 medium (L-15 with 5% fetal calf serum, penicillin–streptomycin, HEPES), we separated the myelin and nerve debris from trigeminal neurons. Neurons were labelled with NeuroFluor NeuO (01801) membrane-permeable fluorescent probe for detecting live neurons according to the manufacturer's protocol (StemCell Technologies) and sorted by flow cytometry using the LSR II flow cytometer running FACSDiva 8.0 software and FACSAria (all BD Biosciences). The co-culture system using neurons and cancer cells is based on a technique initially described for prostate cancer cells70 and modified by our group12. Neurons were prepared as described previously12 and cocultured with cancer cells or normal keratinocytes. Human neurons were co-cultured with the human cSCC IC8 cell line (obtained from the laboratory of I.L. and C.H.37, or with the human epidermal keratinocyte HEK cells (ATCC, PCS-200-011). Mouse neurons were co-cultured with the mouse oral squamous cell carcinoma MOC1 cell line (Sigma-Aldrich, scc469) and B6-10K (obtained from the laboratory of K.Y.T.). Sensory or motor neurons were cultured to assess the impact of nerves' degenerative/regenerative status on immunotherapy in vitro. Neurons were plated on laminin-coated (Life Technologies, 23017015) round glass coverslips set in 12-well plates. Then, 2 h after plating, the wells were flooded with 1 ml of warm (37 °C) Ham's F12 culture medium (Sigma-Aldrich) supplemented with 10% EV-depleted fetal bovine serum (Gibco, Thermo Fisher Scientific) and 1% penicillin–streptomycin and incubated at 37 °C in 5% CO2 and 95% air. The carcinoma cells were plated (1:10 neuron: cancer cells ratio), and cultures were grown in RPMI 1640 medium containing 10% EV-depleted fetal bovine serum in 37 °C and 5% CO2 incubation conditions, for 72 h, with or without anti-PD-1 antibody (cemiplimab, 1,000 μg ml−1, or RMP1-14, 100 μg ml−1, InVivoMAb Antibodies, BioXCell, for human and mouse, respectively), before neuron sorting and RNA extraction. TG neurons were obtained from C57BL/6J mice (n = 8), processed using the above protocol, seeded to 12-well culture plates (3 × 105 total cells per well) and cultured for 72 h. The cultured neurons were then divided into four experimental groups (two wells per group) in which each group was exposed to different culture conditions for an additional 72 h. The first group (control) received no medium change; the second group received fresh medium (20% FBS in Ham's 12 + penicillin–streptomycin + 1% vitamin); the third group received conditioned medium from the MOC1 mouse oral squamous cell carcinoma cell line previously cultured (10 × 103 seeded cells in 20% FBS in Ham's 12 + penicillin–streptomycin + 1% vitamin) for 3 days. After 72 h of exposure, neurons were collected by trypsinization, medium was added (F-15 + 30% FBS) and then live neuro cells were stained with NeuO (membrane-permeable fluorescent dye) dye for 1 h and washed. NeuO-positive cells were sorted by flow cytometry using the CytoFLEX SRT Cell Sorter. RealDRG cryopreserved hPS-cell-derived sensory neurons were obtained from Anatomic. RealDRG neurons were seeded on 0.01% poly-l-ornithine-coated (Millipore Sigma, A-004-C) µ-slide 8-well ibidi plates (80827) at a density of 0.15 × 105 total cells per well. The poly-l-ornithine coating was performed overnight at room temperature in the tissue culture hood, followed by three washes with sterile water. Subsequently, iMatrix-511 silk (Takara, T304) diluted 1:50 in Dulbecco's PBS (dPBS) was added as a secondary coating and incubated at 37 °C for 2 h after washing off the poly-l-ornithine. The RealDRG cells were cultured for 10 days to allow for maturation, with half of the Senso MM medium (Anatomic) being replaced every alternate day. After the initial 10-day culture period, the neurons were divided into four experimental groups (two wells per group) and exposed to different culture conditions for an additional 4 days (96 h), aligning with the timeline in the provided PDF. The experimental groups were as follows: fresh medium: cells received fresh Senso MM medium (Anatomic). To obtain conditioned medium, B16-F10 and Mia PaCa2 cells were seeded at 10 × 103 cells in Senso MM medium and cultured for 24 h. The medium was then replaced, and the supernatant was collected after another 24 h, filtered with a 0.22 µm filter and stored at 4 °C. This process was repeated, and the two collections were mixed at a 1:1 ratio. After the 4-day exposure period, the mature neurons from µ-slide 8-well ibidi plates were fixed with 4% buffered formalin. Neurons were then stained with NFH antibody (ab4680, Abcam) followed by a secondary Texas Red-conjugated antibody. Images were acquired using the Cytation 7 imaging reader. Cell counts and neurite morphology were assessed using the neurite outgrowth module of the Gen5 Cytation 7 software. rDRGs were plated in duplicate wells in an 8-well Ibidi plate and assigned to one of four conditions: (1) fresh medium; (2) no medium change; (3) conditioned medium; or (4) direct co-culture with either B16-F10 or Mia PaCa-2 cells. Cell viability was confirmed using NeuO staining in the GFP channel before fixation. Cells were then fixed and stained with neurofilament heavy chain (NFH, Texas Red channel) to assess neuronal morphology and with DAPI to label nuclei. Using a ×20 magnification, multichannel fluorescence images were taken as a 12 × 12 image montage from the centre of each well in an 8-well Ibidi plate. The Agilent BioTek Cytation 5 cell imaging multimode reader was used for all image acquisition. A minimum of ten ROIs per condition was selected based on consistent cell density. ROIs were excluded if they exhibited low cell density or imaging artifacts. Morphological analysis and thresholding for the selection of neurons and assessment of neuronal morphology were completed using the neurite outgrowth module in Gen5 Cytation 7 software. We assessed neurite length and thickness across the four conditions, with two wells per condition. The Gen5 neurite outgrowth module analysis automatically provides results for both image-level metrics, such as total outgrowth length per image and cell-level average values, like the per-cell average neurite length. Default thresholding for soma selection was applied to exclude weak NFH signal in the TexasRed channel, and neurons were selected and distinguished from cancer cells by using the ‘optimize soma using nuclear signal' setting for DAPI staining and by selecting for cells with NFH+ protrusions extending more than 20 µm from the cell, which represent neurites (dendrites and axons). ‘Only keep neurites connected to a soma', ‘Discard short neurites' and ‘Discard short ending branches' were selected in the ‘Neurite' settings window, with short neurites under 20 µm excluded and short ending branches under 5 µm excluded to ensure that only the true intact neurite signal was studied. From these 8–10 final ROIs, we exported morphology data from the manually programmed neurite outgrowth module. The metrics reported from each ROI were the average neurite length (µm) and the average neurite thickness (calculated as NFH+ area divided by neurite length in µm). The 24-well CytoView MEA plate (Axion, M384-tMEA-24W) electrodes were prepared for culturing realDRG according to the Anatomic protocol (https://www.anatomic.com/_files/ugd/a239f4_97800271c1964458aa21c9a9008849a3.pdf). In brief, around 7 ml cell-culture-grade water with penicillin–streptomycin was added to the outer rim of the plate. Then, electrodes were coated with 50 μl per well of 0.01% PLO for 1 h at room temperature. PLO was washed off four times with 500 μl cell-culture-grade water, ensuring complete removal between washes. Then, 50 μl of iMatrix-511 SILK (Takara, T304) diluted 1:50 in dPBS per well was added as a secondary coating and incubated at 37 °C for 2 h. The human realDRG cells were quickly thawed in a water bath, collected in fresh F12 medium (20% FBS, 1% penicillin–streptomycin), centrifuged at 300 rpm for 4 min and resuspended in Senso MM media (Anatomic). The realDRGs were plated at 20,000 and 50,000 cells per well. Basal spontaneous firing rate of iPS-cell-derived sensory neurons assessed by MEA. After 10 days in normal culture, mature, firing neurons were cultured for 6 days under control conditions, stressed conditions (no medium exchange), in MOC1 conditioned medium or in MOC1-co-culture. Spontaneous firing rates were recorded at the baseline, 24 and 72 h. The mean firing rate of iPS-cell-derived sensory neurons co-cultured with MIA PaCa-2 cells (20,000 or 50,000 cells per well) over 5 days was measured by MEA. RealDRGs were plated at 20,000 and 50,000 cells per well on 24-well CytoView MEA plates, with medium changes every 48 h. After 8–9 days in culture, mature neurons started firing spontaneously and, at day 10, all neurons established stable spontaneous activity. After 10 days in normal culture, mature firing neurons were cultured for 6 days under control conditions, stressed conditions (no medium exchange), in Mia-PaCa2-conditioned medium or in Mia-PaCa2 co-culture. The spontaneous functional activity of the realDRGs was recorded for a minimum of 10 min and every 24 h for 6 days (days 10, 11, 12, 13, 14 and 15). Statistical analysis was performed using GraphPad Prism. One-way ANOVA with Tukey's multiple -comparisons test was used to determine statistical significance between groups, with a significance level of P < 0.05. RNA was isolated from cultured neurons using total RNA was extracted using the RecoverALL Total Nucleic Acid Isolation kit (Ambion) according to the manufacturer's instructions. RNA libraries were prepared and sequenced at the University of Houston Sequencing and Editing Core according to standard protocols. Enrichment for mRNA from total RNA was performed by coding-region-specific biotinylated capture probes, which were selected using streptavidin magnetic beads. mRNA-enriched libraries were prepared using the QIAseq Stranded Total RNA Kit (Qiagen) with 200 ng of input RNA. RNA was fragmented, reverse-transcribed into cDNA, ligated with sequence adaptors and amplified by PCR. Size selection for libraries was performed using SPRIselect beads (Beckman Coulter). The prepared libraries were pooled and sequenced on the NovaSeq 6000 system (Illumina), using an S4 flow cell at paired-end 150 bp configuration at 20–30 million read depth for each sample. The raw sequencing data (FASTQ files) were imported into Qiagen CLC Genomics Workbench 20.0.4. Initial quality control of the sequencing reads was performed using the built-in tools, including the Quality Control function, to evaluate read quality metrics, such as read length distribution, quality scores and potential adapter contamination. Low-quality reads and contaminants were trimmed or filtered out. Gene expression levels were quantified using the Count Features tool. The output generated a count table indicating the raw read counts for each gene across all samples. Read counts were normalized using the trimmed mean of M method implemented in the R Bioconductor package edgeR (v.4.2.1)71 to determine the abundance of each gene. Next, we fit a negative binomial generalized log-linear model to estimate the quasi-dispersions with empirical Bayes moderation (glmQLFit function), followed by hypothesis testing with empirical Bayes quasi-likelihood F-tests. P-value adjustment was performed using the Bonferroni correction method. These data have been deposited in the NCBI GEO under accession number GSE289744. GSEA was conducted to investigate the enrichment of gene sets of mouse neurons cultivated in vitro. GSEA analysis was performed considering the Hallmark gene set (MH) from the Mouse MSigDB Collections (mh.all.v2024.1.Mm.symbols.gmt) and performing 1,000 permutations63,72. Fixed samples containing 3% glutaraldehyde plus 2% paraformaldehyde in 0.1 M cacodylate buffer, pH 7.3, were washed with 0.1 M cacodylate buffer, pH 7.3, post fixed with 1% cacodylate-buffered osmium tetroxide, washed with 0.1 M cacodylate buffer, then in distilled water. Next, the samples were sequentially treated with Millipore-filtered 1% aqueous tannic acid, washed in distilled water, treated with Millipore-filtered 1% aqueous uranyl acetate, then rinsed thoroughly with distilled water. The samples were mounted on double-stick carbon tabs (Ted Pella), previously mounted onto glass microscope slides. The samples were then coated under vacuum using a Balzer MED 010 evaporator (Technotrade) with platinum alloy for a thickness of 25 nm, then immediately flash-carbon-coated under vacuum. The samples were examined/imaged in a JSM-5900 scanning electron microscope (JEOL) at an accelerating voltage of 5 kV. Samples were fixed with a solution containing 3% glutaraldehyde plus 2% paraformaldehyde in 0.1 M cacodylate buffer, pH 7.3, then washed in 0.1 M sodium cacodylate buffer and treated with 0.1% Millipore-filtered cacodylate buffered tannic acid, post fixed with 1% buffered osmium tetroxide and stained en bloc with 1% Millipore-filtered uranyl acetate. The samples were polymerized in a 60 °C oven for approximately 3 days. Ultrathin sections were cut in a Leica Ultracut microtome (Leica), stained with uranyl acetate and lead citrate and examined in a JEM 1010 transmission electron microscope (JEOL) at an accelerating voltage of 80 kV. Digital images were obtained using AMT Imaging System (Advanced Microscopy Techniques). The samples were not diluted before use in the assay and were assessed in triplicate. Then, 50 µl of capture beads were added per well to the assay plate, followed by 50 μl of cell culture supernatant. After three washes, 25 µl of detection antibody was added per well and incubated for 1 h at room temperature with shaking at 500 rpm. Then, 50 µl of streptavidin-PE solution was added per well and incubated for 30 min at room temperature with shaking at 500 rpm. After three washes, 120 µl of reading buffer was added per well and incubated for 5 min. Data were acquired using the Luminex 200 analyzer and xPONENT v.4.2 software. Analysis was performed using Bio-Plex Manager v.6.1 software. The different cell types used for in vivo experiments in this study, along with the figure panel presenting the results of their relevant analysis, are summarized in Supplementary Table 10. A minimum of five mice per experimental group was used to ensure that appropriate statistical analyses could be conducted. In each experiment conducted, all efforts to replicate the results were successful. All animal studies were carried out according to protocols approved by The University of Texas MD Anderson Cancer Center (protocols 00000950-RN03, 00001522-RN01, 00001522-RN02, 00002342-RN00), NYU, Queen's University (protocols 2380 and 2393), and Université de Montréal (protocols 21046 and 21047) Institutional Animal Care and Use Committee. Disease development and progression were closely monitored. According to our approved protocol, mice with metastases were euthanized as soon as we noticed signs of discomfort in the mice or when the largest dimension of a tumour reached 20 mm. In none of the experiments were these limits exceeded. Mice were housed in standard environmental conditions (12 h–12 h light–dark cycle; 23 °C; food and water ad libitum) at facilities accredited by the Canadian Council of Animal Care (UdeM and Queen's University). As per our IACUC-approved protocol, the following end points have been used in all the experiments and have not been exceeded. Along with excessive body weight loss (maximum of 10%), the end points include excessive tumour volume (10% of animal's body weight; ~20 mm × 20 mm, not exceeded in any experiment), skin ulceration, necrosis, bleeding, infection and self-inflicted injury, prostration, lethargy, unresponsiveness to stimulation and/or lack of grooming. To generate murine skin SCC cell lines, p53-deficient (K14Cre; p53R172H/flox) C57BL/6 mice (4–6-week-old, both males and females) were exposed to 7 kJ m−2 s−1 of ultraviolet B light, 3 doses per week for 100 days. The hair on the mice's dorsal surface was shaved twice or thrice a week before UV exposure. This model has been shown to effectively simulate human cuSCC development, producing tumours with a genomic profile similar to those found in humans with cuSCC73. Skin tumours (minimum 5 mm in diameter and persistent for at least 2 weeks) started to develop 9 months after the end of the UV-exposure protocol. Each tumour was split in half. One section was processed for FFPE, H&E stained and IF staining with anti-pan-keratin (type I) was performed for confirmation of cSCC diagnosis. The remaining tissue section was macrodissected and incubated in cell culture medium for 7–10 days. The cultured cells were sorted using the CD326 (EPCAM) antibody, and the resulting isolated epithelial cells were again incubated in cell culture medium, IF stained with anti-pan-keratin (type I) (E6S1S) antibody and DAPI to confirm their epithelial origin. The isolated malignant keratinocytes were injected intradermally (200,000 cells per 60 μl) into C57BL/6 WT mice, and their tumorigenicity was recorded. The newly developed tumours were assessed using H&E staining to confirm the histopathological diagnosis of cSCC. This process resulted in five distinct UVSCC cell lines (M2, M3, M4, M5 and M6) (Extended Data Fig. We used denervation and surgical axotomy to assess in vivo nerve injury modulation of the immune response. For denervation, we used 6–8-week-old male and female SKH1-Elite mice (SKH1-Hrhr, Charles River). For axotomy, we used 6–8-week-old male and female C57BL/6J mice (strain 000664, Jackson Laboratory) and 4–6-week-old female HU-NCG CD34+ humanized mice (Charles River). Then, we removed any nerves from the skin flap. Next, 7 days after denervation, the mice were injected with the mouse (SKH1-Hrhr derived) ultraviolet-induced cSCC B610K cells (mouse cells generated by primary culture of UV-induced skin tumours in SKH1-Elite, hairless, euthymic and immunocompetent mice (Charles River; https://www.criver.com/products-services/find-model/skh1-elite-mouse?region=3646)74. Then, 7 days after axotomy, C57BL mice were injected with the mouse ultraviolet-induced cSCC M4 cells (Extended Data Fig. Hu-NCG CD34+ Humanized Mice were injected with the human IC8 cuSCC cell line (C.H.)37. The human cell line HLA was tested at the Histocompatibility Typing Laboratory at the Hematopoietic Stem Cell Transplant program of MD Anderson Cancer Center. Then, 1 week after tumour implantation, the mice were treated biweekly with either murine (intraperitoneal (i.p. We used the rat IgG2a (BE0089, 2A3, BioXCell) as a control. An Ifnar1-KO mouse model (B6(Cg)-Ifnar1^tm1.2Ees/J, 028288, Jackson Laboratory) and WT C57BL/6J mice (000664, Jackson Laboratory), aged 10–12 weeks, both male and female, were used in this experiment. On day 0, UVSCC-M4 cells (200,000 cells in 30 µl PBS) were inoculated into the footpads of Ifnar1-KO and WT mice. Then, 17 days after inoculation (day 17), the mice underwent either injection of ethidium bromide (25 µl of 0.5% solution) into the tibial nerve at the tumour–normal skin interface to induce demyelination or a sham procedure involving PBS injection into the same location. Anti-PD-1 treatment (BioXCell Clone RMP1-14, BP0146; 250 µg per mouse, i.p. in sterile PBS) was initiated 2 days after demyelination (day 19) and administered twice weekly for a total of 6 doses. WT mouse groups, both with and without demyelination, were treated with anti-PD-1 alone or in combination with the STING agonist c-di-GMP (CDG; Invivogen, 25 µg per mouse, intratumoural), administered according to the same schedule to enhance IFN type I signalling. The tumour volumes were measured every 3 days using callipers, and after euthanizing the mice, the footpads were excised, weighed and fixed in 10% buffered formalin for histopathological viability analysis. In brief, the tumour bed was evaluated, and the average percentages of viable tumour, necrosis, anucleate keratin, inflammatory infiltrate or fibrosis were estimated for each tumour sample. We evaluated the effect of IL-6 signalling blockade on anti-PD-1 efficacy in 6–8-week-old male C57BL mice (000664, Jackson Laboratory). One week after the orthotopic inoculation of mouse UV-induced cSCC M4 cells into the skin, mice received either an injection of ethidium bromide into the tumour periphery at the tumour-normal skin interface to induce demyelination or a sham injection. was initiated 2 days after demyelination and administered twice weekly for 8 doses over 4 weeks. Subsequent anti-PD-1 doses were administered either alone or in combination with an anti-IL-6 receptor antibody (InVivoMAb anti-mouse IL-6, BioXCell, MP5-20F3, BE0046; 200 µg per mouse, twice weekly for 3 weeks). Tumour volumes and immune response outcomes were monitored throughout the experiment. In brief, the tumour bed was evaluated, and the average percentages of viable tumour, necrosis, anucleate keratin, inflammatory infiltrate or fibrosis were estimated for each tumour sample. PNI over time (whiskers pad model): 8–10-week-old female C57BL mice (000664, Jackson Laboratory) were used. All procedures were approved by the New York University Institutional Animal Care. Animals were housed in a temperature-controlled, pathogen-free room under a 12 h–12 h light–dark cycle (06:00–18:00) with ad libitum access to food and water. The MOC2 mouse oral squamous cell carcinoma cell lines (Sigma Chemicals) were cultured in IMDM/F12 (2:1 mixture) with 5% FBS (Thermo Fisher Scientific), 1% penicillin–streptomycin, 1% amphotericin, 5 ng ml−1 EGF (Millipore), 400 ng ml−1 hydrocortisone and 5 mg ml−1 insulin. Under 3% isoflurane anaesthesia, 20 μl of MOC2 cells (5,000 cells total) in serum-free culture medium was injected into the left whisker pad using a 10 μl Hamilton syringe. The mice were randomly allocated to four groups. The tumours were processed for histological assessment and mIF staining, focusing on identifying PNI by tumour cells and characterizing the perineural immune infiltrate. Mouse tumours were quickly dissected from euthanized animals (orthotopic B610K intradermal model in SKH1-Elite animals and normal skin controls) and analysed by P.D.V. Tissues were immediately sectioned to generate tissue slices. One tumour slice/sample was formalin-fixed, paraffin-embedded and processed for histological staining (H&E) to assess tumour presence. Electrical recordings were captured using a MEA1060-Inv-BC microelectrode array system (Multichannel Systems) with a perforated microelectrode array, 60pMEA100/30iR-Ti MEAs (890335, Harvard Apparatus). The MEAs used contained 60 electrodes, of which one is a reference electrode. These electrodes are spaced 100 µm apart in a 6 × 10 grid (ten electrodes per row arranged in six columns generating a rectangle of electrodes onto which tissue slice is placed); each electrode is 30 µm in diameter. The perforated MEAs allow for a gentle suction (applied by a vacuum pump) to keep the tissue slice in contact with electrodes for the duration of recordings. This enables us to capture the electrical activity from every electrode for every second of each recording. As such, recordings from 59 electrodes per second per slice were collected (one electrode is a reference electrode). Recordings occurred at room temperature using a 25-kHz sampling frequency and a Butterworth second-order digital filter set to high-pass with a cut-off frequency of 10 Hz (to eliminate slow FPs). A STG4000 stimulus generator (Multichannel Systems) was used for stimulation, which consisted of biphasic voltage, −0.5 V and +0.5 V each for 100 µs and repeated after a 23 ms interval. Stimulation was applied to selected electrodes; evoked spike responses were recorded. Two independent types of stimulations were used on each slice. The same electrode sets were stimulated for all slices analysed and the stimulation parameters were identical. Recordings continued after electrical stimulations were turned off to allow for capturing of activity back to baseline. In all cases, continuous electrical activity per second per electrode recordings were captured and analysed using the MC_Rach4.6.2 software (Multichannel Systems). TEM analysis was conducted on mouse tissue samples from a previously established model of PNI in the sciatic nerve78. Sciatic nerves were dissected and fixed for 2 h. Fixation was performed using a modified reduced osmium impregnation method. The samples were embedded in fresh 100% durcupan. The processed samples were sectioned (1 µm) and stained with 1% toluidine blue to identify the areas of interest. Thinner sections (70 nm) were mounted onto slot grids and loaded onto the Zeiss grid holder for EM imaging. Images were taken with a beam acceleration of 20.0 kV and a working distance of approximately 2.0 mm, capturing 6,144 px by 4,608 px images with a pixel size of 100 nm to 200 nm. A Talos120C transmission electron microscope with the Gatan (4k × 4k) OneView Camera was used for high-resolution EM imaging. Bulk RNA-seq analysis of lung-metastasis-innervating neurons was performed after intravenous injection of 5 × 105 B16F10-OVA melanoma cells or PBS vehicle into nociceptor neuron reporter mice (Trpv1cre::TdTomatofl/WT). Then, 2 weeks later, the mice were euthanized, and lung metastases were visually confirmed in the B16F10-eGFP-inoculated group. Jugular nodose ganglia (JNC) were dissected into ice-cold HEPES-buffered DMEM (Thermo Fisher Scientific, 12430062). For each biological replicate, JNC from four mice (two male, two female) were pooled and transferred into HEPES-buffered DMEM containing 1 mg ml−1 collagenase IV (Sigma-Aldrich, C5138) and 2.4 U ml−1 dispase II (Sigma-Aldrich, 04942078001), then incubated at 37 °C for 70 min. After washing with DMEM, ganglia were gently triturated using glass Pasteur pipettes of decreasing diameter. Cells were centrifuged at 200g over a 15% BSA gradient in PBS to remove debris, stained with SYTO 40 (10 μM, Thermo Fisher Scientific, S11351) for 5 min at room temperature to distinguish cells from axonal debris, washed with PBS and resuspended in sterile flow cytometry buffer (PBS with 2% FBS and 1 mM EDTA). The cell suspension was filtered through a 70 µm mesh (VWR, 10204-924) and nociceptor neurons (tdTomato+) were FACS-enriched on a BD FACSAria cell sorter, collected directly into 500 µl TRIzol reagent (Invitrogen, 15596026), and stored at −80 °C until RNA extraction, according to established protocols79. Library preparation was conducted at the Institut de Recherche en Cancérologie et en Immunologie (IRIC), Université de Montréal. The RNA quality was confirmed on an Agilent Bioanalyzer, with all samples achieving an RNA integrity number of ≥7.5. Libraries were prepared using a poly(A)-enrichment, single-stranded RNA-seq method (KapaBiosystems, KAPA RNA Hyperprep Kit, KR1352) and sequenced on the Illumina NextSeq500 platform with 75-cycle single-end reads. Basecalling was performed using Illumina RTA v.2.4.11, and demultiplexing was done using bcl2fastq v.2.20 (allowing for one mismatch in the index). Trimmomatic was used to remove adapter sequences and low-quality bases from the 3′ end of each read, and the resulting high-quality reads were aligned to the GRCm38 mouse genome using STAR v.2.5.11, which also generated gene-level read counts. Differential expression analysis was performed using edgeR. The generalized linear model likelihood ratio test from edgeR was used to identify significant DEGs between groups. Genes were considered to be differentially expressed if they had an FDR-adjusted P < 0.05. log2-transformed fold changes and −log10-transformed P values were calculated from the normalized data, and additional data analysis and visualization were carried out in RStudio. These data have been deposited in the NCBI GEO under accession number GSE292089. All lines were backcrossed >6 generations on the C57BL6/J background (H2-Kb). Littermates with no Cre (that is, Trpv1WT::tdTomatofl/WT, NaV1.8WT::Atf3fl/fl) were used as controls referred to as WT for the Atf3 gene in the relevant figures. Mice of both sexes were used for these crosses. All Cre driver lines used were viable and fertile, and abnormal phenotypes were not detected. Offspring were tail-clipped; tissue was used to assess the presence of transgene by standard PCR, as described by Jackson Laboratory or the donating investigators. B16F10-mCherry-OVA83 (M. F. Krummel), B16F10-OVA (Biocytogen, 311537), B16F10-eGFP (Imanis, CL053) or non-tumorigenic keratinocytes (CellnTEC, MPEK-BL6100) were cultured in complete Dulbecco's modified Eagle's medium high glucose (DMEM, Corning, 10-013-CV) supplemented with 10% FBS (Seradigm, 3100) and 1% penicillin–streptomycin (Corning, MT-3001-Cl), and maintained at 37 °C in a humidified incubator under 5% CO2. All of the cell lines tested negative for mycoplasma, and none are listed by the International Cell Line Authentication Committee registry (v.11). Non-commercial cell lines (B16F10-OVA-mCherry) were authenticated using antibody (against OVA, mCherry) and/or imaging as well as morphology and growth property. Commercial cell lines have not been further authenticated. Growth was assessed daily using handheld digital calipers and the tumour volume was determined using the formula L × W2 × 0.52 (ref. Six-week-old male and female littermate control (Trpv1WT::Diptheria-Toxinfl/WT) mice were intradermally injected into the left paw with 2 × 105 B16F10-OVA cells or non-tumorigenic keratinocytes (MPEK-BL6). Then, 7 days later, tumour-infiltrating neurons (or neurons in MPEK-injected skin) were retrogradely labelled by an acute intratumour injection of AAV9-CAG-mCherry-WPRE-SV40p (10 µl, 5.5 × 109). Then, 14 days after inoculation, the mice were euthanized, and the L3–L5 DRG was collected and digested. Most CD45+ cells were depleted using the MagniSort mouse CD45 depletion kit (Invitrogen, 8804-6864). The remaining neurons were resuspended in FACS buffer (PBS with 2% FCS and EDTA) and stained with Viability Dye eFluor 780 (eBioscience, 65-0865-14) at 4 °C for 15 min. Subsequently, nuclei were stained with SYTO 40 (10 µM, Thermo Fisher Scientific, S11351) for 5 min at room temperature to distinguish cells from axonal debris. Cells were then purified by FACS on the BD FACSAria IIu cell sorter and subjected to single-cell barcoding using the Chromium Next GEM Single Cell 3′ GEM, Library and Gene Expression v3.1 system (10x Genomics). Libraries were sequenced on an Illumina NovaSeq X platform, generating 120 million total reads. Reads were aligned to the mouse reference genome using Cell Ranger (10x Genomics) and analysed with Seurat. AAV9 expression was not detected and therefore subsequent analysis was performed using principal component analysis, which identified 19 clusters, which were assigned cell types using known markers and the Tabula Muris reference atlas. These data have been deposited in the NCBI GEO under accession number GSE292522. Enrichment of gene signatures at the single-cell level was evaluated by performing single-cell GSEA using the R package irGSEA85. Enrichment calculation was performed using the ssGSEA86 method using a rank-normalized approach and generating an empirical cumulative distribution function for each cell. For this analysis, we considered the 50 gene expression signatures from the Mouse MSigDB MH collection: hallmark gene sets72. B16F10-mCherry-OVA melanoma cells (5 × 105) were inoculated intradermally into the flank of 8-week-old male and female mice of which the nociceptor neurons were either permissive (NaV1.8WT::Atf3fl/fl) or resistant (NaV1.8cre::Atf3fl/fl) to injury. Then, 10 days after tumour inoculation, the tumours were excised, minced with a razor blade and digested for 30 min in DMEM containing 10 mM HEPES, 1.6 mg ml−1 collagenase IV (Sigma-Aldrich, C5138) and 10 μg ml−1 DNase I (Sigma-Aldrich, 4942078001). The resulting cell suspensions were filtered through a 40 μm mesh, and samples from two mice were pooled for each experiment. After red blood cell lysis (Gibco, A10492-01), cells were stained with Zombie Aqua viability dye (BioLegend, 423105) and a FITC anti-mouse CD45 antibody (BioLegend, 103107). Approximately 500,000 live immune cells, which included around 80% CD45+ cells, were sorted on the BD FACSAria III cell sorter into a collection buffer (PBS, 0.04% BSA and 50% FBS; Sigma-Aldrich, F4135), then fixed and prepared for single-cell library construction using the Chromium 10x Fixed RNA kits (PN-1000414 and PN-1000497). Libraries were sequenced on the Illumina NovaSeq X system and reads were aligned to the mouse reference genome using Cell Ranger (10x Genomics). Count matrices were generated and analysed in Seurat. Next, low-quality cells, potential doublets and multiplets were removed. Batch effect was corrected using canonical correlation analysis in Seurat. Identification of the major immune phenotypes was performed based on the expression of key phenotypical markers (Extended Data Fig. Differences in the frequency of tumour-associated leukocytes between groups were assessed using the Wilcoxon rank-sum test. Over-representation analysis was performed using the Enrichr R package87. These data have been deposited in the NCBI GEO under accession number GSE292090. Tumours were harvested and enzymatically digested in DMEM + 5% FBS (Seradigm, 3100) + 2 mg ml−1 collagenase D (Sigma, 11088866001) + 1 mg ml−1 Collagenase IV (Sigma, C5138-1G) + 40 µg ml−1 DNAse I (Sigma, 10104159001) under constant shaking (40 min, 37 °C). The cell suspension was centrifuged at 400g for 5 min. The pellet was resuspended in 70% Percoll gradient (GE Healthcare), overlaid with 40% Percoll and centrifuged at 500g for 20 min at room temperature with acceleration and deceleration at 1. Single-cell suspensions were prepared in FACS buffer (PBS containing 2% FCS and EDTA) and stained with either Zombie Aqua (BioLegend, 423102) for 15 min at room temperature or Viability Dye eFluor 780 (eBioscience, 65-0865-14) for 15 min at 4 °C. After washing, cells were incubated with Fc Block (0.5 mg ml−1; BD Biosciences, 553141) for 20 min at 4 °C, then stained for 30 min at 4 °C with anti-CD45-FITC (1:100; BioLegend, 103107) and anti-CD8-PerCP/Cyanine5.5 (1:100; BioLegend, 100733). For cytokine expression analysis, cells were stimulated in vitro for 3 h with PMA (50 ng ml−1; Sigma-Aldrich, P1585), ionomycin (1 μg ml−1; Sigma-Aldrich, I3909) and GolgiStop (1:100; BD Biosciences, 554724). They were then fixed and permeabilized (1:100; BD Biosciences, 554714), stained with anti-IFN-γ-APC (1:100; BioLegend, 505810) and analysed on a Cytoflex (Beckman) flow cytometer. A total of 2 × 105 B16F10-OVA-mCherry melanoma cells were injected intradermally into the right hindpaw of C57BL/6 mice or Atf3GFP/GFP mice. Then, 14 days after inoculation, the mice were anaesthetized with urethane (Sigma-Aldrich, U2500, 20%, 250 μl, i.p.) The ipsilateral (tumour-bearing paw) and contralateral (control paw) L3–L5 DRGs were collected, post-fixed in 4% PFA for 24 h at 4 °C and then immersed sequentially in 10%, 20% and 30% sucrose (Sigma-Aldrich, S8501) for 24 h each. The tissues were embedded in Fisher Healthcare Tissue-Plus O.C.T. Compound (Thermo Fisher Scientific, 23-730-571), frozen at –80 °C and cryosectioned into 15–30 μm sections. The sections were blocked in PBS containing 0.2% Triton X-100, 5% BSA and 5% goat serum (Sigma-Aldrich, G9023) for 2 h at room temperature, then incubated at 4 °C for 48 h with rabbit anti-mouse ATF3 (1:100, Abcam, ab207434) and chicken anti-mouse TUBB3 (1:500, Abcam, ab41489) in staining solution (PBS with 0.2% Triton X-100 and 3% goat serum). After three 15-minute washes in PBS containing 0.2% Triton X-100, the sections were incubated at 4 °C for 16 h with AF488 anti-chicken (1:1,000, Abcam, ab150169) and AF647 anti-rabbit (1:1,000, Invitrogen, A-21245) antibodies. The samples were then washed and stained with DAPI (2 μg ml−1, Invitrogen, D1306) for 10 min at room temperature. Images were acquired using the Nikon Ti2 microscope equipped with a Photometrics Prime 95B camera. Using ImageJ or Nikon Elements software, circular ROIs were drawn around each neuron and nucleus. The mean grey values for ATF3 and TUBB3 fluorescence (in a.u.) within each ROI were exported for further analysis in Microsoft Excel. The ratio of ATF3+ neurons to TUBB3+ neurons was determined, indicating increased ATF3 expression in ipsilateral (tumour-bearing) L3–L5 DRGs compared with the contralateral side. In parallel, a threshold-based method was also applied to define the relative number of ATF3+ nuclei, which also show increased relative expression in ipsilateral (tumour-bearing) L3–L5 DRGs neurons. Values from Atf3GFP/GFP mice independently validated these findings and are not shown. Unpaired Wilcoxon rank-sum tests were conducted to analyse immunohistochemical, in vitro and mouse data. P values of less than 0.05 were considered to indicate nominal statistical significance. Owing to the variance of xenograft growth in control mice, we used at least three mice per genotype to give 80% power to detect an effect size of 20% with a significance level of 0.05. The number of independent mice used is listed in the figure legend for all mouse experiments. No statistical methods were used to predetermine the sample size. The experiments were not randomized, and investigators were not blinded to allocation during experiments and outcome assessment. To assess differences in cytokine levels between groups, we used a nested one-way analysis of variance (ANOVA) to account for both technical and biological replicates, followed by Tukey's post hoc test. The cytokine levels were standardized by calculating z scores based on the mean for plotting purposes. To compare tumour growth in mouse models under different interventions, tumour volume was recorded at multiple timepoints. In this way, we used a mixed-effects model (REML) to compare tumour growth curves and account for repeated tumour measures. Intervention groups, time and their interaction were considered to be fixed effects, and individual animals were considered to be random factors. Moreover, post hoc Tukey's multiple-comparison test was conducted for pairwise comparisons between groups at specific timepoints. Analyses were performed using JMP Pro v.17.0 software and R. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. RNA-seq data derived from clinical trial samples are accessible at the GEO under accession number GSE289743. The nCounter PanCancer Immune Profiling Panel, along with the GeoMX DSP gene and protein expression data, are provided in Supplementary Tables 8 and 9, respectively. Visium spatial transcriptomics data (GSE289745), gene expression data from neuron co-cultures (GSE289744), scRNA-seq data from murine ganglia (GSE292522) and tumours (GSE292090), as well as bulk RNA-seq data from mouse ganglia (GSE292089), are all available in the GEO. The spatial molecular imaging for PDAC has been previously published45 and is also accessible from the GEO (GSE199102). Source data are provided with this paper. Four-year survival with nivolumab in patients with previously treated advanced non-small-cell lung cancer: a pooled analysis. Robert, C. et al. Pembrolizumab versus ipilimumab in advanced melanoma. Seiwert, T. Y. et al. Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial. Chen, S. H. et al. Perineural invasion of cancer: a complex crosstalk between cells and molecules in the perineural niche. B., Johnson, M. M., Chua, T. L., Karia, P. S. & Schmults, C. D. Outcomes of primary cutaneous squamous cell carcinoma with perineural invasion: an 11-year cohort study. Namikawa, K., Aung, P. P., Gershenwald, J. E., Milton, D. R. & Prieto, V. G. Clinical impact of ulceration width, lymphovascular invasion, microscopic satellitosis, perineural invasion, and mitotic rate in patients undergoing sentinel lymph node biopsy for cutaneous melanoma: a retrospective observational study at a comprehensive cancer center. Deng, J. et al. Prognostic value of perineural invasion in gastric cancer: a systematic review and meta-analysis. The key genes for perineural invasion in pancreatic ductal adenocarcinoma identified with monte-carlo feature selection method. Amit, M. et al. Loss of p53 drives neuron reprogramming in head and neck cancer. Jacobson, A., Yang, D., Vella, M. & Chiu, I. M. The intestinal neuro-immune axis: crosstalk between neurons, immune cells, and microbes. Koren, T. et al. Insular cortex neurons encode and retrieve specific immune responses. The regulation of immunological processes by peripheral neurons in homeostasis and disease. Liu, P. et al. Role of macrophages in peripheral nerve injury and repair. & Cheung, C. W. Immune actions on the peripheral nervous system in pain. Pilot phase II trial of neoadjuvant immunotherapy in locoregionally advanced, resectable cutaneous squamous cell carcinoma of the head and neck. Gross, N. D. et al. Neoadjuvant cemiplimab for stage II to IV cutaneous squamous-cell carcinoma. Amit, M., Na'ara, S. & Gil, Z. Mechanisms of cancer dissemination along nerves. Anderson, P. N. Activating transcription factor 3 and the nervous system. Seijffers, R., Mills, C. D. & Woolf, C. J. ATF3 increases the intrinsic growth state of DRG neurons to enhance peripheral nerve regeneration. Ruff, C. A. et al. Neuronal c-Jun is required for successful axonal regeneration, but the effects of phosphorylation of its N-terminus are moderate. & Morgan, J. R. Activating transcription factor 3 (ATF3) is a highly conserved pro-regenerative transcription factor in the vertebrate nervous system. Expression of ATF3 and axonal outgrowth are impaired after delayed nerve repair. Seijffers, R. et al. ATF3 expression improves motor function in the ALS mouse model by promoting motor neuron survival and retaining muscle innervation. Jin, Y., Tay, D., So, K. F. & Wu, W. Expression of c-jun in the lateral vestibular nucleus following spinal cord injury and peripheral nerve graft transplantation in adult rats. Schreck, I. et al. c-Jun localizes to the nucleus independent of its phosphorylation by and interaction with JNK and vice versa promotes nuclear accumulation of JNK. Expression profiling of cutaneous squamous cell carcinoma with perineural invasion implicates the p53 pathway in the process. Zhang, F. et al. Lymphovascular or perineural invasion is associated with lymph node metastasis and survival outcomes in patients with gastric cancer. Distinct immune cell populations define response to anti-PD-1 monotherapy and anti-PD-1/anti-CTLA-4 combined therapy. Liu, D. et al. Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade in patients with metastatic melanoma. Kim, S. T. et al. Comprehensive molecular characterization of clinical responses to PD-1 inhibition in metastatic gastric cancer. & Wong, S. Y. Cutaneous surgical denervation: a method for testing the requirement for nerves in mouse models of skin disease. Rotshenker, S. Wallerian degeneration: the innate-immune response to traumatic nerve injury. Hassan, S. et al. A unique panel of patient-derived cutaneous squamous cell carcinoma cell lines provides a preclinical pathway for therapeutic testing. Deborde, S. et al. An in vivo murine sciatic nerve model of perineural invasion. & Reese, T. S. Electron microscopic tomography reveals discrete transcleft elements at excitatory and inhibitory synapses. Rollenhagen, A. et al. Synaptic organization of the human temporal lobe neocortex as revealed by high-resolution transmission, focused ion beam scanning, and electron microscopic tomography. & Smith, K. J. Conduction in segmentally demyelinated mammalian central axons. Linking demyelination to compound action potential dispersion with a spike-diffuse-spike approach. Feng, Z. et al. Neutrophil extracellular traps exacerbate secondary injury via promoting neuroinflammation and blood-spinal cord barrier disruption in spinal cord injury. Shiau, C. et al. Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment. Hirai, I. et al. Perineural invasion in pancreatic cancer. Magnon, C. et al. Autonomic nerve development contributes to prostate cancer progression. Z. et al. ATF3 is a neuron-specific biomarker for spinal cord injury and ischaemic stroke. Mantovani, A., Allavena, P., Marchesi, F. & Garlanda, C. Macrophages as tools and targets in cancer therapy. & Kroemer, G. Type I interferons in anticancer immunity. The cytokines interleukin-6 and interferon-α induce distinct microglia phenotypes. Type I interferon response drives neuroinflammation and synapse loss in Alzheimer disease. & Donlin, L. T. Regulation of type I interferon responses. Monje, M. et al. Roadmap for the emerging field of cancer neuroscience. The Eighth Edition AJCC Cancer Staging Manual: continuing to build a bridge from a population-based to a more “personalized” approach to cancer staging. Eisenhauer, E. A. et al. New response evaluation criteria in solid tumours: revised RECIST guideline (version 1.1). Tetzlaff, M. T. et al. Pathological assessment of resection specimens after neoadjuvant therapy for metastatic melanoma. Cottrell, T. R. et al. Pathologic features of response to neoadjuvant anti-PD-1 in resected non-small-cell lung carcinoma: a proposal for quantitative immune-related pathologic response criteria (irPRC). Lindskrog, S. et al. An integrated multi-omics analysis identifies prognostic molecular subtypes of non-muscle-invasive bladder cancer. Taber, A. et al. Molecular correlates of cisplatin-based chemotherapy response in muscle invasive bladder cancer by integrated multi-omics analysis. GeoMx DSP automated slide preparation user manual. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Nguyen Kovochich, A. et al. HOXB7 promotes invasion and predicts survival in pancreatic adenocarcinoma. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Renthal, W. et al. Transcriptional reprogramming of distinct peripheral sensory neuron subtypes after axonal injury. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Malin, S. A., Davis, B. M. & Molliver, D. C. Production of dissociated sensory neuron cultures and considerations for their use in studying neuronal function and plasticity. In vitro dorsal root ganglia and human prostate cell line interaction: redefining perineural invasion in prostate cancer. Chen, Y., Chen, L., Lun, A. T. L., Baldoni, P. L. & Smyth, G. K. edgeR v4: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Nguyen, T. N. et al. Integrative transcriptomic analysis for linking acute stress responses to squamous cell carcinoma development. Enhanced UV-induced skin carcinogenesis in transgenic mice overexpressing proprotein convertases. Burova, E. et al. Preclinical development of the anti-LAG-3 antibody REGN3767: characterization and activity in combination with the Anti-PD-1 antibody cemiplimab in human PD-1xLAG-3-knockin mice. Restaino, A. C. et al. Functional neuronal circuits promote disease progression in cancer. Peripheral nerve injury and sensitization underlie pain associated with oral cancer perineural invasion. Crosson, T. et al. Cytokines reprogram airway sensory neurons in asthma. Crosson, T. et al. FcepsilonR1-expressing nociceptors trigger allergic airway inflammation. Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Voehringer, D., Liang, H. E. & Locksley, R. M. Homeostasis and effector function of lymphopenia-induced “memory-like” T cells in constitutively T cell-depleted mice. Headley, M. B. et al. Visualization of immediate immune responses to pioneer metastatic cells in the lung. Radiation and dual checkpoint blockade activate non-redundant immune mechanisms in cancer. Fan, C. et al. irGSEA: the integration of single-cell rank-based gene set enrichment analysis. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Xie, Z. et al. Gene set knowledge discovery with Enrichr. We thank D. Aten for the artistic work; D. A. Cobanoglu (J.P.A. This research was performed in the Flow Cytometry & Cellular Imaging Core Facility (J.K.B.) ), which is supported in part by the National Institutes of Health through M.D. CCSG grant NIH P30CA016672 supports the High Resolution Electron Microscopy Facility. is the Felix L. Haas Endowed Professor in Basic Science and the Charles B. Barker Endowed Chair at The University of Texas Anderson Cancer Center, Houston, TX, USA. is supported by the Institutional Start-up from UT MD Anderson Cancer Center. Work in the Calin laboratory is supported in part by the NIDCR grant R01DE032018, Team DOD grant in Gastric Cancer CA200990P2 (W81XWH-21-1-0715), The G. Harold & Leila Y. Mathers Foundation, a Development Grant associated with the Brain SPORE 2P50CA127001 and by the Ben and Catherine Ivy Foundation grant. Work in the Watowich laboratory is supported by NIH grant R01AI133822; S.S.W. is the Vivian L. Smith Distinguished Chair in Immunology. Work in the Dougherty laboratory is supported by NIH grant CA200263, Thompson Family Foundation Initiative. has funding from the NCI, Research Specialist 1 R50 CA243707-01A1. ), the Burroughs Wellcome Fund Career Award for Medical Scientists (W.L.H. ), the Pancreatic Cancer Action Network Career Development Award (W.L.H.) and the Krantz Family Center for Cancer Research Quantum Award (W.L.H.). Work in the Ye laboratory is supported by NIH grant NIDCR R01DE029493, NIDCR HEAL Initiative R01DE032501 and RM1 DE033491. Work in the D'Silva laboratory is supported by NIH grants R35DE027551 and R01 CA250214. Work in the Bunimovich laboratory is supported by NIH grants R01 CA266529 and R01 CA272946. Work in the Talbot laboratory is supported by the Canadian Institutes of Health Research (CIHR; grants 407016, 461274, 461275), the Canada Foundation for Innovation (44135), a Canadian Cancer Society Emerging Scholar Research Grant (708096), the Knut and Alice Wallenberg Foundation (KAW 2021.0141, KAW 2022.0327), the Swedish Research Council (2022-01661), the Natural Sciences and Engineering Research Council of Canada (RGPIN-2019-06824) and NIH/NIDCR (R01DE032712). is supported by a Brain Canada Rising Star award from the Henri and Berenice Kaufmann Foundation. is supported by the MD Anderson Cancer Center Institutional Research Grant (600382 30 124924 2). J.N.M is the Alando J. Ballantyne Distinguished Chair of Head and Neck Surgery, The University of Texas Anderson Cancer Center, Houston, TX, USA. These authors contributed equally: Erez N. Baruch, Frederico O. Gleber-Netto, Priyadharsini Nagarajan, Xiayu Rao, Shamima Akhter, Tuany Eichwald Division of Cancer Medicine, Hematology and Oncology Fellowship program, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Erez N. Baruch, Paulino Tallón de Lara, Derrick L. Tao & Tara M. Davidson Platform for Innovative Microbiome and Translational Research, Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Erez N. Baruch, Nadim J. Ajami, Ryan P. Goepfert & Jennifer A. Wargo Department of Head and Neck Surgery, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Frederico O. Gleber-Netto, Shamima Akhter, Tongxin Xie, Adebayo Adewale, Shorook Naara, Hinduja N. Sathishkumar, Shajedul Islam, William McCarthy, Dan Yaniv, Jeffrey N. Myers, Neil D. Gross & Moran Amit Department of Anatomical Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Physiology and Pharmacology, Karolinska Institutet, Stockholm, Sweden Department of Biomedical and Molecular Sciences, Queen's University, Kingston, Ontario, Canada Tuany Eichwald, Mohammad Balood, Karine Roversi, Amin Reza Nikpoor, Maryam Ahmadi, Ali Ahmadi & Sebastien Talbot The Neurodegeneration Consortium, Therapeutics Discovery Division, University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Head and Neck Thoracic Medical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA James P. Allison Institute, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Sonali Jindal, Sreyashi Basu, James P. Allison & Padmanee Sharma Centre for Cell Biology and Cutaneous Research, Blizard Institute, Faculty of Medicine and Dentistry, Queen Mary University of London, London, UK Center for Systems Biology, Krantz Family Center for Cancer Research, Department of Radiation Oncology, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Broad Institute of MIT and Harvard, Cambridge, MA, USA Massachusetts Institute of Technology, Cambridge, MA, USA Department of Cancer Biology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Paulino Tallón de Lara & George A. Calin Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Andrew Futreal, Kunal Rai, Veena Kochat, Jennifer A. Wargo & Moran Amit Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Cutaneous Oncology, H. Lee Moffitt Cancer Center & Research Institute, Tampa, FL, USA Mona Yuan, Naijiang Liu & Yi Ye Pain Research Center, Department of Molecular Pathobiology, New York University College of Dentistry, New York, NY, USA Mona Yuan, Naijiang Liu & Yi Ye Cancer Biology and Immunotherapies Group, Sanford Research, Sioux Falls, SD, USA Department of Pathology, H. Lee Moffitt Cancer Center & Research Institute, Tampa, FL, USA Department of Tumor Microenvironment & Metastasis, H. Lee Moffitt Cancer Center & Research Institute, Tampa, FL, USA Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA UTHealth Graduate School of Biomedical Sciences, Houston, TX, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Jing Wang, Kenneth Y. Tsai, Sebastien Talbot, Neil D. Gross or Moran Amit. serves as a consultant to NFlection Therapeutics, Sun Pharma, DXB Biosciences. reports a consulting or advisory role at Regeneron, Sanofi, Elevar Therapeutics, Remix, Eisai, Bioatlas, Coherus in the past 24 months and research Funds (Inst) from Prelude, Ayala, Merck, Pfizer, Rakuten, EMD Serono, ISA, Viracta and Gilead in the past 24 months. has been a consultant to Replimmune, Nurix, Roche/Genentech, Array, Pfizer, Novartis, BMS, GSK, Sanofi-Aventis, Vaccinex, Apexigen, Eisai, Iovance, Merck and ABM Therapeutics, and has been the principal investigator of research grants to MD Anderson by Roche/Genentech, GSK, Sanofi-Aventis, Merck, Myriad, Oncothyreon, Pfizer, ABM Therapeutics and LEAD Pharma. received a reagent from Eterna Therapeutics, unrelated to the work presented here. is listed as an inventor on a US patent application (PCT/US17/53.717) submitted by the University of Texas MD Anderson Cancer Center that covers methods to enhance immune checkpoint blockade responses by modulating the microbiome; she reports compensation for speaker's bureau and honoraria from PeerView and serves as a consultant and/or advisory board member for Gustave Roussy Cancer Center, OSE Immunotherapeutics, Bayer Therapeutics, James Cancer Center OSU and Daiichi Sankyo. is an advisory board member for Regeneron, Merck, Replimune, Immunocore, Iovance Biotherapeutics, Novartis, IO Biotech, MyCareGorithm and HUYABIO International; received travel support from Castle Biosciences and Regeneron; is a data safety monitoring board member for Incyte and AstraZeneca; a scientific advisory board member for T-Knife Therapeutics; a study steering committee member for BMS, Nektar, Regeneron and Replimune; holds common stock in Bellicum Pharmaceuticals and Amarin; and has received research funding (to institution) from BMS, Merck, Regeneron, Replimune, GSK, Celgene, Novartis, IDEAYA Biosciences, Modulation Therapeutics and HUYABIO International. is a scientific advisory committee member for Adaptive Biotechnologies, Akoya Biosciences, Apricity, Asher Bio, BioAtla, BioNTech, Catalio, C-Reveal Therapeutics, Dragonfly Therapeutics, Earli, Enable Medicine, Glympse, Henlius/Hengenix, Hummingbird, ImaginAb, InterVenn Biosciences, JSL Health, Lytix Biopharma, Marker Therapeutics, Matrisome, NTx, Oncolytics, Osteologic, PBM Capital, Phenomic Al, Polaris Pharma, Soley Therapeutics, Sporos, Time Bioventures, Two Bear Capital, Vironexis and Xilis; holds private investments in Affini-T, Candel Therapeutics, LAVA Therapeutics, Spotlight and Trained Therapeutix Discovery. declares no competing interests related to ‘A Randomized Phase II Trial of Induction Pembrolizumab Followed by Surgery Versus Surgery Alone for Resectable Cutaneous Squamous Cell Carcinoma'. has participated in advisory boards for Regeneron, Castle Biosciences, Replimune, Incyte and Sun Pharma. The other authors declare no competing interests. Nature thanks Shawn Hervey-Jumper and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Patient tumour samples were collected as part of two clinical trials of neo-adjuvant immunotherapy in cutaneous squamous cell carcinoma patients - NCT03565783 and NCT04154943. This figure details the number of available patients and tumour samples per each analysis reported in this manuscript. For patients who had a large neo-adjuvant-treated tumour, several samples from the same patient tumour were used to minimize a potential tumour heterogeneity bias. Gene expression of c-Jun and Atf3 in the jugular-nodose ganglia neurons from mice with melanoma lung metastases (B16F10-OVA) was compared to PBS vehicle controls. P-values (FDR) were calculated using the empirical Bayes quasi-likelihood F-test implemented in edgeR and were corrected for multiple hypothesis testing using the Benjamini–Hochberg method. (a) Immunohistochemistry stain against ERG, an endothelial cell marker, was conducted on tumour samples of axotomized cutaneous squamous cell carcinoma (cSCC) bearing mice (see Fig. This representative ERG stain showed intact intra-tumoral vasculature (red arrowheads, see vascular lumen integrity) despite perineural inflammation (asterisks), suggesting that the tumour-associated nerve injury (nerve marked with black arrowheads) and perineural inflammation were not accompanied by vascular damage. p53-deficient (K14Cre; Trp53R172H/flox) C57BL/6 mice (4–6 weeks old) were exposed to 7 Kilojoules/m2/S of ultraviolet B (UVB) light, 3 doses/week for 100 days. The hair on the mice's dorsal surface was shaved twice or thrice a week before UV exposure. Skin tumours (minimum 5 mm in diameter and persistent for at least 2 weeks) started to develop 9 months after the end of the UV-exposure protocol. Each tumour was split in half. One section was formalin-fixed and paraffin-embedded (FFPE), H&E stained, and immunofluorescence (IF) staining with anti-Pan-Keratin (Type I) was performed for confirmation of cSCC diagnosis. The remaining tissue section was macro-dissected and incubated in cell culture media for 7–10 days. The cultured cells were sorted using the CD326 (EpCAM) antibody, and the resulting isolated epithelial cells were again incubated in cell culture media, IF stained with anti-Pan-Keratin (Type I) (E6S1S) antibody and DAPI to confirm their epithelial origin. The isolated malignant keratinocytes were injected intradermally (200 K cells/60ul) into C57BL/6 wild-type mice, and their tumorigenicity was recorded. The newly developed tumours were assessed using H&E staining to confirm the histopathological diagnosis of cSCC. This process resulted in five distinct UVSCC cell lines (M2, M3, M4, M5 and M6). (Created with BioRender.com) (c) The susceptibility of UV-cSCC cells to anti-PD-1 therapy was tested in vivo. The UVcSCC cells were intradermally injected into C57BL/6 mice and subsequently treated with either IgG or anti-PD-1 antibody. Among the five cell lines tested, only UVcSCC-M4 exhibited susceptibility to anti-PD-1 treatment. Statistical analysis was performed using a mixed-effects model with restricted maximum likelihood (REML) estimation, and post hoc comparisons at specific time points were conducted using Šidák's multiple comparisons test. (Created with BioRender.com) (e) Tumour growth curve of the huCD34+ NSG axotomy experiment. Initially, both groups demonstrated a response to anti-PD-1 therapy (Cemiplimab, Cem). However, only sham-operated mice had a durable response (417.7 mm3 versus 105.4 mm3 in axotomized and sham-operated mice, respectively, on day 40, p-value < 0.01). over time, reflecting results from 7 biologically independent animals per group. A mixed-effects model utilizing restricted maximum likelihood (REML) estimation was used for statistical analysis. Post hoc comparisons at specific time points were conducted using Šidák's multiple comparisons test, with an adjusted p-value of <0.001 at day 40. Representative electron microscopy (EM) images of nerves from mice who underwent micro-injection of human oral SCC cells (HSC-3) directly into the right sciatic nerve, simulating PNI, versus sham controls (PBS injection). Top row (PBS control): Nerves from sham mice show large numbers of normal-appearing myelinated axons, both large and small. The overall structure appears intact and unremarkable. Bottom row (HSC-3 SCC cells): Nerves from mice with PNI exhibit several pathological changes. Myelin invaginations (red asterisks) demonstrated an excessive in-folding of the myelin sheath into the axons. Intra-axonal accumulation of dark organelles is observed, often associated with a narrow myelin sheath. Some axons (yellow asterisks) show signs of dilatation with accumulated organelles and thinning of the myelin sheath, possibly reflecting compensatory changes due to axonal damage. (a) Representative images of RealDRG neurons (stained with NeuO) after 72 h in standard sensory neuron media (Senso-MM) followed by different treatments: no media exchange, fresh media exchange, co-culture with MOC1 squamous cell carcinoma cells, or culture in MOC1-conditioned media for an additional 72 h. Initial seeding density: 0.15 × 105 cells/well in poly-l-ornithine and iMatrix-511 coated 8-well ibidi plates. (b) Bar plot showing the viability of NeuO+ neurons, normalized to the “No media change” control. Neuronal viability was reduced only in the co-culture group. Data are represented as mean ± s.e.m., based on 6 biologically independent samples for each group. (c) Neurite analysis pipeline demonstration showing the methodology for quantifying neuronal morphology. Representative multichannel fluorescence images demonstrate the automated analysis approach using the Neurite Outgrowth Module in Gen5 Cytation 7 software, with neurons identified by neurofilament heavy chain (NFH) staining and DAPI nuclear labelling. (d) Representative monochrome immunofluorescence images of neurofilament heavy chain (NFH) staining showing neuronal morphology under different experimental conditions used for neurite metric quantification. Images were acquired using an Agilent BioTek Cytation 5 cell imaging multimode reader at 20x magnification as 12 × 12 image montages from the centre of each well. (e) Quantification of average neurite length showing a trending decrease in B16 melanoma co-culture conditions (left, p = 0.054) and no significant difference in MIA PaCa-2 (PDAC) co-culture (right, p = 0.606). Analysis performed using automated neurite outgrowth module with short ending branches <5 μm excluded. (f) Quantification of neurite thickness (calculated as NFH+ area divided by neurite length) showing a significant decrease in both B16 melanoma (left, p = 0.006) and MIA PaCa-2 (right, p = 0.014) co-culture conditions compared to controls. Data represent mean ± standard deviation from a minimum of 10 regions of interest per condition, with duplicate wells per condition. (g) Basal spontaneous firing rate of iPSC-derived sensory neurons assessed by multi-microelectrode array (MEA). After 10 days in normal culture, mature, firing neurons were cultured for 6 days under control conditions, stressed conditions (no media exchange), in MOC1-conditioned media, or co-cultured with MOC1 cells. derived from two biologically independent samples for each group. (h) Mean firing rate of iPSC-derived sensory neurons co-cultured with MIA PaCa-2 cells over 5 days, measured by MEA. RealDRGs were plated at 20 K cells/well (n = 3 biologically independent samples, left panel) and 50 K cells/well (n = 2 biologically independent samples, right panel) on 24-well CytoView MEA plates, with media changes every 48 h. After 10 days in normal culture establishing stable spontaneous activity, mature neurons were cultured for 6 days under control conditions, stressed conditions (no media exchange), in MIA PaCa-2-conditioned media, or in MIA PaCa-2 co-culture. Spontaneous functional activity was recorded for a minimum of 10 min every 24 h from day 10 through day 15. Volcano plots showing the differentially expressed (DE) genes between (a) monoculture of human dorsal root ganglia (DRG) neurons vs. DRG neurons co-cultured with IC8 human squamous cell carcinoma (SCC) cells; see Fig. 2 for gene set enrichment analysis (b) monoculture of human DRG neurons vs. DRG neurons co-cultured with normal keratinocytes (HEK). Only eight genes were DE in the presence of normal keratinocytes. Only one gene was DE in the presence of anti-PD-1 – SCN10A, which encodes the alpha subunit of a sodium channel found in sensory nerves. Only four genes were DE between the conditions. Differential expression was determined with the empirical Bayes quasi-likelihood F-tests (two-sided) followed by multiple comparisons adjustment with Bonferroni correction. (a-c) Representative images of the nerve identification process for digital spatial profiling (DSP) analysis. Nerves were identified using NFH (yellow) and B3-tubulin (red) staining in neoadjuvant-treated tumour samples of a responder (a) and a non-responder patient (b-c); note the pan-cytokeratin (PanCK)-positive cancer cells (in pink) surrounding the nerve. A total of 553 neural niches (84 ROI for pre-treatment samples and 469 ROI for neoadjuvant-treated) were identified and analysed. (d) Geometric ROI (white rectangle) showing a representative digital capture of the nerve (middle, shadowed, nerve capture panel) and the surrounding peri-neural niche, which was assessed for immune markers (right, shadowed, immune capture panel). The former was analysed for neural phenotypic markers and the latter for immune markers. (e) Box plots showing expression levels of neuro-protective marker ApoA-I in tumour-associated nerves (TANs) among pre-treatment tumour samples according to clinical response status. A higher expression among responders suggests a lower abundance of injured nerves compared to non-responders prior to anti-PD-1 therapy. (f) The box plot demonstrates the expression level of CD31, a marker of blood-nerve barrier reactivity and angiogenesis. The lack of difference in CD31 expression between responders and non-responders suggests that endothelial cells or blood vessels are not affected by CINI. n = 118 perineural niches; FDR, false discovery rate. Spatial transcriptomic analysis of tumour samples from an independent treatment naïve cutaneous squamous cell carcinoma patient cohort (n = 11). We examined the spatial relationships among three functional phenotypes: CINI, anti-tumoral immunity, and immunosuppressive inflammation. The anti-tumoral immunity phenotype included CD8A+GZMB+PRF1+ and CD4+IL2+ T cells, as well as CD86+IRF8+TNF+ and CD68+PSMB10+HLADQA1+HLADRA+HLADRB1+ antigen-presenting cells. The immunosuppressive inflammation phenotype comprised CD204+CD206+CD163+ and CD68+IL10+ tumour-associated macrophages, along with CD4+FOXP3+ T regulatory cells. The CINI signature included general markers for nerve detection (NEFL, NEFH, NEFM, NEUROD1, MRGPRD, TAC1, SSTR2, HAPLN4, SST) and nerve injury markers (ATF3, JUN, SOX1, SMAD1, BHLHE41, KLF7, KLF6). (b) Representative scatterplots showing correlations for individual patients. (a) Illustration of experimental design: B16F10-OVA melanoma cells (2 × 105 cells) or non-tumorigenic keratinocytes (MPEK-BL6) were injected into the right hind paw of 8-week-old male and female mice. Fourteen days post-inoculation, the mice were euthanized, and the L3–L5 DRG neurons, which innervate the paw inoculated with tumour/keratinocytes, were harvested based on anatomy and underwent single-cell RNA sequencing (scRNA-seq). (Created with BioRender.com) (b) Neuron scRNA-seq cluster abundance bar plot – please see main text Fig. 4a for the Uniform Manifold Approximation and Projection (UMAP). (c) Neuron single-cell gene set enrichment analysis (sc-GSEA), based on the Hallmark dataset. Bubble size represents the pairwise post hoc comparisons (Tukey test). (d) Density heatmaps showing the distribution of scGSEA scores for inflammatory pathways across neurons in the KIN, MIN, and CIN groups. Notably, CIN neurons show a greater density at higher enrichment scores across all pathways. (e-f) Representative immunofluorescence images (e) and corresponding quantification (f) of ATF3 expression in L3–L5 dorsal-root-ganglion neurons innervating the melanoma-bearing paw (ipsilateral) versus the contralateral tumour-free paw (control). Data are from n = 5 mice per group, and an unpaired two-tailed Student's t-test confirmed a significant difference between groups (p = 0.005). Box-and-whisker plots display the median (centre line), interquartile range (box), and full data range (whiskers). (g) Multiplex IF staining of melanoma (S100+) tumours from mice with a cre-flox conditional knockout of Atf3 in nociceptors neurons (NaV1.8cre::Atf3fl/fl, Atf3-cKO) versus their permissive littermate controls (NaV1.8 wt::Atf3fl/fl). (h) Tumour weight plot of the experiment detailed in main text Fig. 4, comparing the weight (mg) of melanoma B16F10-OVA tumours which were extracted from littermate control (Nav1.8WT::Atf3fl/fl) (n = 5) or mice whose nociceptor neurons are cKO for Atf3 (Atf3-cKO) (n = 4) (Nav1.8cre::Atf3fl/fl). The statistical significance was determined using an unpaired two-tailed t-test (p = 0.033). (a) Gating strategy used to assess the percentage of IFNγ+CD8+ T-cells in melanoma tumours from littermate controls and Atf3-cKO animals, shown in Fig. (b) Uniform Manifold Approximation and Projection (UMAP) of scRNA-seq data demonstrates the distribution of CD45+ cells from littermate controls and Atf3 cKO animals (left) and their phenotypes (right). 4, for the abundance of each major immune subtype. Gene expression signature associated with PNI and nerve injury. Metadata information from cohorts used for validation of PNI signature. Clinical characteristics of the independent sample cohort. Results from pathway enrichment analysis, focusing on the differentially expressed genes in macrophages from Atf3-cKO and Atf3-cKO animal models. GO enrichment results for genes that are differentially expressed between neoadjuvant-treated samples from responders and non-responders. Clinicopathological characteristics of anti-PD-1-treated cSCC clinical trial samples. nCounter PanCancer Immune Profiling Panel data. In vitro PNI: time-lapse microscopy visualization of cancer cells co-cultured with dorsal root ganglion (DRG) neurons. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Baruch, E.N., Gleber-Netto, F.O., Nagarajan, P. et al. Cancer-induced nerve injury promotes resistance to anti-PD-1 therapy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09428-7'>Structural basis for the dynamic regulation of mTORC1 by amino acids</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 15:49:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The mechanistic target of rapamycin complex 1 (mTORC1) anchors a conserved signalling pathway that regulates growth in response to nutrient availability1,2,3,4,5. Amino acids activate mTORC1 through the Rag GTPases, which are regulated by GATOR, a supercomplex consisting of GATOR1, KICSTOR and the nutrient-sensing hub GATOR2 (refs. The mechanisms through which these sensors regulate GATOR2 and how they detach from it upon binding their cognate amino acids remain unknown. Here, using cryo-electron microscopy, we determined the structures of a stabilized GATOR2 bound to either Sestrin2 or CASTOR1. The sensors occupy distinct and non-overlapping binding sites, disruption of which selectively impairs the ability of mTORC1 to sense individual amino acids. We also resolved the apo (leucine-free) structure of Sestrin2 and characterized the amino acid-induced structural rearrangements within Sestrin2 and CASTOR1 that trigger their dissociation from GATOR2. Binding of either sensor restricts the dynamic WDR24 β-propeller of GATOR2, a domain essential for nutrient-dependent mTORC1 activation. These findings reveal the allosteric mechanisms that convey amino acid sufficiency to GATOR2 and the ensuing structural changes that lead to mTORC1 activation. The large eukaryotic protein kinase mTORC1 regulates mass accumulation, metabolism and ageing, and dysregulation of mTORC1 is associated with various common diseases, such as cancer and neurodegeneration1,2,3,4,5,11. Nutrients activate mTORC1 by promoting its translocation to the lysosomal surface. This process depends on the heterodimeric Rag GTPases (RagA or RagB bound to RagC or RagD) and several protein complexes that control their localization and nucleotide state, including Ragulator and GATOR, an approximately 2 MDa supercomplex comprising two subcomplexes: GATOR1–KICSTOR and GATOR2 (refs. Although the biochemical function of GATOR2 is unknown, genetic experiments have shown that it opposes GATOR1, an inhibitor of mTORC1 signalling that serves as a GTPase-activating protein (GAP) towards RagA/B7. In human cells, GATOR2 serves as the primary conduit through which nutrients signal to mTORC1 via the leucine sensors Sestrin1 and Sestrin2 and the arginine sensor CASTOR1, which directly interact with GATOR2 in the absence of their respective ligands7,10,16,17. Although structures of CASTOR1 in the apo and arginine-bound forms have been determined18,19,20, only the leucine-bound structure of Sestrin2 has been resolved, with its apo form remaining unsolved21,22. Recently, the structures of human and budding yeast GATOR2 (SEACAT in yeast) were determined and revealed that human GATOR2 forms a large cage decorated with WD40 domain β-propellers that interact with Sestrin2 and CASTOR1 to convey nutrient cues10,23. However, several major mechanistic questions remain unanswered, including how the nutrient sensors impact GATOR2 and how ligand binding mechanistically relieves their inhibition. Here we present the structures of GATOR2 bound to Sestrin2 or CASTOR1 and shed light on these questions. We previously determined the cryo-electron microscopy (cryo-EM) structure of the apo GATOR2 complex, produced by co-expressing its five core subunits (WDR24, WDR59, MIOS, SEC13 and SEH1L), which form heterodimer pairs consisting of MIOS–SEH1L, WDR24–SEH1L and WDR59–SEC13 (ref. During single-particle analysis, however, it became apparent that most GATOR2 particles were incomplete, missing at least one of the smaller SEH1L or SEC13 subunits required for GATOR2 to interact with partners, including Sestrin2 and CASTOR1. To address this problem, we generated ‘single-chain' variants of WDR24, MIOS and WDR59, in which SEH1L or SEC13 were inserted in frame into their cognate heterodimer partners, each of which donates a three-stranded β-blade to complete the β-propellers of SEH1L or SEC13. Thus, the single-chain variants each encode two complete WD40 β-propellers, ensuring full subunit occupancy of GATOR2 particles (Fig. a, Domain organization of GATOR2 and sc-GATOR2 components. The grey trapezoids indicate β-blade donation by WDR24, MIOS or WDR59 to complete the β-propellers of SEH1L or SEC13. b, sc-GATOR2 interacts with GATOR1, KICSTOR and the nutrient sensors. Anti-Flag immunoprecipitates (IPs) were prepared from MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs and were analysed by immunoblotting for the indicated proteins. Data are representative of two independent experiments. c, Size-exclusion chromatography profiles of sc-GATOR2, sc-GATOR2–Sestrin2 and sc-GATOR2–CASTOR1 complexes. d, Coomassie blue-stained SDS–PAGE analysis of sc-GATOR2, sc-GATOR2–Sestrin2 and sc-GATOR2–CASTOR1 complexes. e, Cryo-EM structures of the human apo sc-GATOR2 (centre), sc-GATOR2–CASTOR1 (left) and sc-GATOR2–Sestrin2 (right) complexes. Views of the experimental maps are shown. Sestrin2 and CASTOR1 bind to distinct, non-overlapping sites on sc-GATOR2. For gel source data, see Supplementary Fig. We verified that individual single-chain variants sc-WDR24–SEH1L, sc-MIOS–SEH1L and sc-WDR59–SEC13 restore amino acid-dependent mTORC1 activation to knockout cell lines comparably with separately expressed GATOR2 components (Extended Data Fig. We also confirmed that co-expressed single-chain constructs assemble into a sc-GATOR2 variant that binds to Sestrin2, CASTOR1, GATOR1 and KICSTOR, and that more effectively reactivates mTORC1 signalling in MIOS-null cells than wild-type GATOR2 (Fig. These results also indicate that dissociation of SEH1L or SEC13 is not a regulatory mechanism required for nutrient signalling through GATOR2. We transiently expressed sc-GATOR2 in HEK293F cells and purified it by affinity chromatography followed by size-exclusion chromatography (Fig. The resulting elution profile and subunit stoichiometry of sc-GATOR2 matched those of wild-type GATOR2, leading us to a successful cryo-EM structure determination at 3.47 Å resolution (Fig. sc-GATOR2 exhibits an architecture that closely mirrors that of wild-type GATOR2, with four MIOS–SEH1L, two WDR24–SEH1L and two WDR59–SEC13 heterodimeric subunits assembling into a symmetric cage built on an octagonal scaffold10 (Fig. The scaffolds of sc-GATOR2 and wild-type GATOR2 were nearly superimposable with a root mean square deviation (RMSD) of 1.1 Å (Extended Data Fig. Local resolution analysis confirmed complete occupancy of SEH1L and SEC13, as expected given their covalent fusion to their core GATOR2 partners (Supplementary Figs. Consistent with greater particle integrity in sc-GATOR2, we were able to model an additional region of WDR24 (residues 610–630), extending along WDR59 and its associated SEC13 subunit (Extended Data Fig. 2a–c), as well as a portion of the engineered sc-WDR59–SEC13 linker (Extended Data Fig. As with wild-type GATOR2, we were unable to visualize the N-terminal β-propeller and RWD domains of WDR59 (Extended Data Fig. Similarly, the local resolution corresponding to the WDR24 β-propeller was limited, suggesting that this domain is either flexible or conformationally dynamic (Supplementary Figs. We conclude that sc-GATOR2 recapitulates the structure and function of wild-type GATOR2. To obtain a GATOR2–Sestrin2 co-complex, we transiently co-expressed sc-GATOR2 with a Sestrin2 mutant (Glu451Gln) that does not bind to leucine and constitutively associates with GATOR2 (ref. We further supplemented the affinity-purified sc-GATOR2–Sestrin2 complex with additional purified Sestrin2 before size-exclusion chromatography. Compared with apo sc-GATOR2, the sc-GATOR2–Sestrin2 complex eluted with a larger apparent hydrodynamic radius, indicating a conformational rearrangement upon Sestrin2 binding (Fig. We determined the structure of the sc-GATOR2–Sestrin2 complex using single-particle cryo-EM at a resolution of 3.36 Å (Figs. 6–9 and Supplementary Video 1) and found that Sestrin2 binding did not induce any major structural rearrangements involving the GATOR2 α-solenoidal domain or C-termainal domain (CTD). Among the sc-GATOR2–Sestrin2 complex particles, the vast majority were doubly occupied, indicating that two polypeptides of Sestrin2, and probably Sestrin1 and Sestrin3, can simultaneously bind to the same GATOR2 particle (Supplementary Fig. a, Overview of the GATOR2–Sestrin2 co-complex. Each GATOR2 particle can bind two copies of Sestrin2. Sestrin2 primarily interacts with the WDR24 and SEH1L(WDR24) β-propeller but makes additional contacts with the MIOS CTD. c,d, Views of the interfaces between WDR24 and Sestrin2. e, View of the interface between Sestrin2 and SEH1L(WDR24). g, Validation of the GATOR2–Sestrin2 interface. Anti-Flag IPs were prepared from WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs and were analysed by immunoblotting for the indicated proteins. h, Validation of the GATOR2–Sestrin2 interface. Anti-HA IPs were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min and were analysed as in g. i–k, Leucine triggers a conformational change that results in the release of Sestrin2 from GATOR2. i, The binding of leucine to Sestrin2 promotes the formation of helices αL1 and αL2 atop Sestrin2. k, The formation of the Sestrin2 helix αL2 repositions critical residues, including Arg404 and Arg338, that repel GATOR2. Data are representative of two independent experiments (g,h). For gel source data, see Supplementary Fig. The interaction is driven by the negatively charged surface of Sestrin2 helices αC1 and αC2 (Fig. 2d), which are recognized by a positively charged surface formed by multiple WDR24 β-propeller loops. In addition, SEH1L enhances the recruitment of Sestrin2 via its blades 2 and 3, with critical involvement of residue Asp111 (Fig. We confirmed that these residues (particularly the WDR24 residues Arg46, Arg121 and Arg228) are critical for the GATOR2–Sestrin2 interaction by co-immunoprecipitation after complementation of WDR24-deficient cells (Fig. We mutagenized SEH1L within the WDR24–SEH1L single-chain construct so as not to disrupt its roles in other cellular pathways and found that disruption of the Sestrin2 interaction surface on SEH1L was sufficient to disrupt the GATOR2–Sestrin2 interaction (Extended Data Fig. These results establish that the GATOR2–Sestrin2 interaction is principally driven by electrostatic interactions between a basic surface on WDR24 and a complementary, acidic surface on Sestrin2 (Fig. As previously reported, Sestrin2 must bind to GATOR2 to inhibit mTORC1 signalling when overexpressed21 (Extended Data Fig. Sestrin2 also makes auxiliary contacts with the MIOS CTD, including through the Sestrin2 residue Asp104, which is positioned against SEH1L(WDR24) in the MIOS–WDR24 CTD–CTD junction (Fig. Disruption of this interface weakened the association between GATOR2 and Sestrin2 (Fig. GATOR2 variants that cannot bind to Sestrin2 were also unable to interact with Sestrin1 or Sestrin3, indicating that the Sestrin proteins share a common binding site, consistent with the conservation of residues involved in GATOR2 binding (Extended Data Fig. Although mutations of some residues critical for Sestrin2 binding (for example, Arg121) abolished mTORC1 signalling, substitution of other residues (for example, Arg46 and Met273) did not affect it (Extended Data Fig. Thus, GATOR2 uses partially overlapping surfaces for binding Sestrin2 and for activating mTORC1, suggesting that the Sestrins directly occlude a crucial GATOR2 function. The GATOR2–Sestrin2 interaction was required for cells to sense the absence of leucine, but not arginine (Extended Data Fig. 3f), similar to cells lacking all Sestrins16. Likewise, overexpression of Sestrin2 variants deficient in GATOR2 binding failed to inhibit mTORC1 signalling (Extended Data Fig. These results provide further evidence that mTORC1 senses leucine availability through the Sestrin1/2–GATOR2 axis. Until now, the structure of leucine-free Sestrin2 has remained unsolved because apo Sestrin2 is unstable and not amenable to purification and structural studies22. Furthermore, artificial intelligence-based structure prediction algorithms have yet to model conformational differences between wild-type Sestrin2 and variants that cannot bind to leucine24. Here we observed no density corresponding to leucine in the Sestrin2 ligand binding pocket. Thus, we have captured the apo state of Sestrin2 (Fig. These observations suggest that Sestrin2 binds mutually exclusively to either GATOR2 or to leucine. The leucine-binding pocket of Sestrin2 does not contribute to the GATOR2-binding surface and is solvent accessible, suggesting that Sestrin2 can encounter leucine without needing to first dissociate from GATOR2 (Fig. There is a high degree of core structural similarity between the leucine-bound Sestrin2 structure (Protein Data Bank (PDB) ID: 5DJ4) and our structure of apo Sestrin2 in complex with sc-GATOR2 (RMSD of approximately 1.15 Å)21. However, we identified two critical features that differentiate these two structures and suggest a mechanism for leucine-dependent dissociation of Sestrin2 from GATOR2 (Fig. First, we observed that the Sestrin2 loop αC2–αC3 adopts a fully extended configuration when bound to GATOR2 (Fig. 2j), unlike the compacted two-stranded β-sheet ‘lid' covering the pocket entrance in leucine-bound Sestrin2. Second, a significant proportion of the ‘linker' region (residues 220–338), which connects the N-terminal and C-terminal lobes of Sestrin2, is either absent in our structure or undergoes substantial conformational changes. Specifically, a 14-residue segment within the linker region (residues 324–338, the ‘transducer segment') exhibits a dramatic shift in both backbone and side-chain positions. In addition, the characteristic helix–loop–helix motif (αL1–αL2) seen in the leucine-bound Sestrin2 structure is flexible and missing from our structure. Sestrin2 αL1 contains Leu261, which is required for leucine binding and dissociation from GATOR2 (ref. To test this hypothesis, we generated variants of Sestrin2 containing substitutions in αL2 (Glu288Ala and Lys291Ala) designed to disrupt its packing against the Sestrin2 linker residues 333–336 (Fig. 2k), and we found that leucine failed to break the complex formed between GATOR2 and these variants (Extended Data Fig. Thus, intramolecular interactions distant from the ligand-binding pocket are required for leucine to disrupt the GATOR2–Sestrin2 interaction. The leucine-binding pocket and the GATOR2-binding surfaces are located on distinct faces of Sestrin2, suggesting that bound leucine does not directly break the GATOR2–Sestrin2 complex. To clarify the mechanism of Sestrin2 release from GATOR2, we compared leucine-bound Sestrin2 and the GATOR2-bound apo Sestrin2 (Fig. Neither the rigidified αL1 and αL2 helices, nor their connecting linker, directly clash with GATOR2, indicating that their leucine-dependent formation does not preclude the GATOR2–Sestrin2 interaction (Fig. We then considered whether leucine binding may allosterically reorganize the GATOR2–Sestrin2 interface. We found that leucine binding is associated with a molecular cascade that reorients the Arg338 side chain. Specifically, leucine binding reconfigures the ligand-binding pocket lid and allows for rigidification of αL1 and αL2. Residues within the transducer segment, including Thr327, must reposition to resolve clashes caused by αL1 and αL2. Of note, after a clash with Phe287 in αL2, the side chain of the Sestrin2 residue Arg404 shifts towards Arg338, which in turn shifts towards the GATOR2 interface and clashes with the WDR24 residue Arg46. We propose that this allosteric cascade disrupts the GATOR2–Sestrin2 interaction through a repulsive interaction between the Sestrin2 residue Arg338 and the WDR24 residue Arg46 (Fig. Indeed, we found that leucine could not dissociate Sestrin2(Arg404Gly) or Sestrin2(Arg338Ala), variants deficient in the allosteric cascade, from GATOR2 in vitro (Extended Data Fig. These findings suggest that leucine increases the off-rate of Sestrin2 from GATOR2. As we could not determine the precise positions of all Sestrin2 αL1–αL2 linker segment side chains, it is possible that additional residues may contribute to the leucine-stimulated dissociation of Sestrin2 from GATOR2. As the structure of apo Sestrin2 in its free state remains unknown, we also note that some structural features observed in our sc-GATOR2–Sestrin2 complex may be induced by GATOR2 binding rather than loss of leucine. Using a similar approach as with Sestrin2, we prepared the sc-GATOR2–CASTOR1 complex and analysed its structure using cryo-EM (Fig. We determined the structure of the CASTOR1-bound sc-GATOR2 complex using single-particle cryo-EM, achieving a resolution of 3.40 Å (Figs. a, Overview of the GATOR2–CASTOR1 co-complex. CASTOR1 binds to a dimeric interface formed by the MIOS brace region. Additional copies of CASTOR1 interact with the MIOS gloves via a single binding site but are probably not physiologically relevant. Each protomer of CASTOR1 contacts one of the MIOS β-propellers that constitute the brace. c,d, Views of the interfaces between MIOS and CASTOR1. e, Validation of the GATOR2–CASTOR1 interface. Anti-Flag IPs were prepared from MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs and analysed by immunoblotting for the indicated proteins. f, Validation of the GATOR2–CASTOR1 interface. Anti-HA IPs were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min and analysed as in e. g–i, Arginine triggers a conformational change that results in the release of CASTOR1 from GATOR2. Comparison of the structures of arginine-bound CASTOR1 (light coral; 5I2C rebuilt) with arginine-free, GATOR2-bound CASTOR1 (salmon). h, The binding of arginine to CASTOR1 reconfigures the loop enclosing the arginine-binding pocket and displaces helix α7 towards GATOR2 by one-half turn. i, Arginine binding, and translation of the CASTOR1 helix α7, reorients helix α3 and the ‘release loop', which clashes with and displaces CASTOR1 from GATOR2. Data are representative of two independent experiments (e,f). For gel source data, see Supplementary Fig. CASTOR1 is a homodimer, and both protomers are required to engage GATOR2 and inhibit mTORC1 signalling17,19. We discovered that the CASTOR1 dimer binds to the GATOR2 ‘brace', with each CASTOR1 protomer interacting in a structurally equivalent manner with the bottom face of one MIOS β-propeller (Fig. In a subset of particles, we detected a CASTOR1 dimer associated with the opposite ‘glove' side of GATOR2 via only a single MIOS β-propeller (Fig. The inhibitory function of CASTOR1 requires two GATOR2-binding sites19, so we suspect that these glove-associated CASTOR1 molecules are technical artefacts due to supplementation of excess free CASTOR1. The middle MIOS loop β3.2–β3.3 projects its Arg137 side chain into a cleft formed by the CASTOR1 ACT2–ACT4 interface, where it makes a salt bridge with the CASTOR1 residue Asp121, which is essential for the GATOR2–CASTOR1 interaction19 (Fig. The flanking MIOS loops provide additional cross-loop hydrogen bonding that supports their fully outstretched configurations (Arg114, His136, Asp139 and Asn184). The MIOS loop β2.4–β3.1 interacts with CASTOR1 via a combination of backbone and side-chain hydrogen bonding, driven by the MIOS residues Lys111, His112 and Ala113 and the CASTOR1 residue Ser258, as well as Tyr118, a residue previously implicated in GATOR2 binding19 (Fig. Aside from weak interactions with the MIOS residues Phe221 and Gln218 (Fig. 3c), this loop lacks a strong electrostatic anchor point, as indicated by its low resolution in our structure. We validated the GATOR2–CASTOR1 interaction through complementation of MIOS-deficient cell lines and found that the MIOS residue Arg137 is essential for GATOR2 to bind to CASTOR1 (Fig. GATOR2 was also unable to interact with CASTOR2 in cells expressing the MIOS(Arg137Ala) variant, indicating that CASTORs share a binding mode (Extended Data Fig. Similarly, we found that the CASTOR1 residues Tyr118, Gln119 and Asp121 are required for the interaction with GATOR2 (Fig. In agreement, wild-type CASTOR1, but not the CASTOR1(Tyr118Ala/Gln119Ala) or CASTOR1(Asp121Ala) mutants, suppressed mTORC1 signalling when overexpressed (Extended Data Fig. We previously found that the MIOS β-propeller is necessary for arginine deprivation to inhibit mTORC1 (ref. Likewise, mTORC1 signalling in cells expressing MIOS(Arg137Ala) is resistant to regulation by arginine but could be suppressed by leucine starvation (Extended Data Fig. These findings reinforce that the GATOR2–CASTOR1 interaction transmits arginine availability to mTORC1. Previous crystallographic studies have reported nearly identical configurations of apo and arginine-bound CASTOR1 in the crystal18,19,20. Here, however, we identified several structural features unique to GATOR2-bound CASTOR1. First, we observed a subtle rotation (approximately 15°) in the relative angle between the two protomers of CASTOR1. Of note, this angled conformation is not unique to the double-engaged MIOS brace β-propellers but also appears on the ‘glove' side of GATOR2, where CASTOR1 engages a single MIOS β-propeller. We also resolved the CASTOR1 β6–α3 loop (residues 82–90), which adopts a conformation different from those determined in the crystal structures of CASTOR1 in either the apo or arginine-bound forms18,19,20 (Fig. We suspect that these apparent differences may be artefacts of crystal packing that limited previous studies. The CASTOR1 arginine-binding pocket and GATOR2-interacting surface are located on distinct surfaces, so we reasoned that arginine may allosterically regulate the GATOR2–CASTOR1 interaction. We analysed our sc-GATOR2–CASTOR1 structure alongside published structures of free CASTOR1. This loop is absent in the apo CASTOR1 structure and inconsistently modelled in arginine-bound structures18,19,20. By re-evaluating the density maps of all protein chains, we found that many CASTOR1 protomers did exhibit a common position of the β6–α3 loop, distinct from that seen in our sc-GATOR2–CASTOR1 structure, that was not attributable to crystal packing. In fact, we were able to build the missing β6–α3 loop into the 1.8 Å density map of arginine-bound CASTOR1 (PDB ID: 5I2C)19. We propose that arginine binding triggers a series of structural rearrangements that reposition the β6–α3 loop to promote release of CASTOR1 from GATOR2 (Supplementary Video 3). First, arginine binding triggers compaction of the CASTOR1 ligand-binding pocket lid (loop β14–α7), which repositions several residues essential for engaging arginine (Fig. This compaction creates a hydrophobic repulsion between Leu273 and Ile280, displacing helix α7 by approximately one-half turn towards MIOS (Fig. This shift alters the local chemical environment at the interface between helix α7 and the adjacent discontinuous helix α3, causing helix α3 to kink dramatically by around 60° (Fig. To accommodate this kink, Tyr297 reorients its side chain, and the new position of helix α3 is stabilized by hydrophobic interactions involving Val94 and hydrogen bonding via Thr95. As helix α3 kinks, it pulls along the β6–α3 loop, causing the Ala92 residue in this loop to collide with the MIOS residue Arg137 (Fig. We propose that this series of allosteric events disrupts the GATOR2–CASTOR1 interaction by creating a direct clash between the CASTOR1 residue Ala92 and the MIOS residue Arg137 (Fig. We generated a CASTOR1 variant with a shortened loop β6–α3 (∆85–90), which we hypothesized might preclude its reorientation and clash with GATOR2, and found that this truncation mutant more effectively immunoprecipitated GATOR2 from arginine-starved cells than wild-type CASTOR1 (Extended Data Fig. Likewise, arginine did not disrupt the CASTOR1(∆85–90)–GATOR2 complex in vitro (Extended Data Fig. We also introduced a Val94Asp mutation into helix α3 of CASTOR1 and found that this substitution partially impaired the arginine-mediated release of CASTOR1 from GATOR2 in vitro (Extended Data Fig. Thus, we conclude that the reconfiguration of the CASTOR1 helix α3 is critical for its arginine-stimulated dissociation from GATOR2. Through molecular dynamics simulations, we found that the ligand-binding lid of CASTOR1 is tightly closed and does not open over long simulation times (approximately 2.5 µs; Extended Data Fig. We wondered whether a multistep binding process might be required in which pocket lid opening is followed by translocation of arginine into the pocket. During dissociation from CASTOR1, we found that arginine pauses near helix α7 for 20 ns on average, maintaining hydrogen bond interactions with residues Ser111 and Glu277, before fully dissociating. We propose that arginine also enters CASTOR1 through this two-step route and that an interaction with helix α7 is an intermediate required to release CASTOR1 from GATOR2. Indeed, we mutated the CASTOR1 residue Glu277, which does not participate in arginine binding at the orthosteric site, to alanine and observed that this mutation strengthened the GATOR2–CASTOR1 interaction and blunted the ability of arginine to disrupt the complex in vitro (Extended Data Fig. Thus, CASTOR1 binds to arginine through a multistep process with crucial transient intermediate states. CASTOR1 binds to GATOR2 at the brace, formed by dimerization of two MIOS β-propellers (Fig. Compared with apo sc-GATOR2, we observed that CASTOR1 binding causes the two MIOS brace β-propellers to move apart, leading to the relative translation and rotation of the adjacent SEH1L(MIOS) (Fig. In apo sc-GATOR2, as in wild-type GATOR2, SEH1L(MIOS) rigidifies the scaffold by binding the WDR59 zinc-finger (ZnF) domain. However, in CASTOR1-bound sc-GATOR2, we were unable to detect density corresponding to the WDR59 ZnF (Extended Data Fig. This effectively creates a hinge at that position, enabling remodelling of the shape of the scaffold. We found that the forced separation of the MIOS brace β-propellers, driven by CASTOR1 binding, results in a more compact and squared GATOR2 scaffold than that of apo GATOR2 (Fig. This mutant GATOR2 showed reduced interaction with GATOR1–KICSTOR and blunted activation of mTORC1 signalling compared with wild-type WDR59 (Extended Data Fig. Unlike Sestrin2, which co-immunoprecipitated with all GATOR subcomplexes, or SAMTOR, which exclusively co-immunoprecipitated with GATOR1–KICSTOR, CASTOR1 predominantly captured GATOR2 (Extended Data Fig. Thus, CASTOR1 influences subcomplex interactions within GATOR. These results suggest that CASTOR1 allosterically reshapes the GATOR2 scaffold, which weakens the GATOR2–GATOR1 interaction and dampens mTORC1 activation. We were unable to detect the WDR24 β-propellers in most particles used to determine the structure of apo sc-GATOR2, suggesting that this domain is flexible (Supplementary Fig. Both Sestrin2 and CASTOR1 increased the apparent hydrodynamic radius of sc-GATOR2 (Fig. 1c), possibly consistent with rigidification of the WDR24 β-propeller, so we wondered whether stabilization of this domain could reflect a common mechanism of GATOR2 regulation. a, Association of either CASTOR1 or Sestrin2 stabilizes the dynamic WDR24 β-propellers and promotes their attachment to the GATOR2 scaffold. b, Quantification of the WDR24 β-propeller position in apo GATOR2, sc-GATOR2–CASTOR1 and sc-GATOR2–Sestrin2 particles used for structural determination. Both Sestrin2 and CASTOR1 rigidify the WDR24 β-propeller, but Sestrin2 does so more effectively. c, Epitope tagging of the WDR24 N-terminal β-propeller impairs mTORC1 activation in a tag size-dependent manner. WDR24-deficient cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before collection. Anti-HA IPs were prepared and analysed by immunoblotting for the indicated proteins. Data are representative of two independent experiments. For gel source data, see Supplementary Fig. We generated a mask covering the positions of the two WDR24 β-propellers and we reanalysed our single-particle datasets to quantify their occupancy (Extended Data Fig. Most apo sc-GATOR2 particles exhibited no detectable WDR24 β-propeller density (Fig. By contrast, most of the sc-GATOR2–CASTOR1 particles had at least one detectable WDR24 β-propeller and both WDR24 β-propellers were resolved in the overwhelming majority of sc-GATOR2–Sestrin2 particles (Fig. Principal component analysis of conformational heterogeneity within the sc-GATOR2 particle revealed two major internal motions: lateral and longitudinal (Supplementary Video 5). Both motions compress the two halves of sc-GATOR2, bringing the MIOS glove β-propellers closer together. Of note, the occupancy of the WDR24 β-propeller remains stable across longitudinal motion states but improves dramatically in the fully outstretched configuration of the lateral motion trajectory. Nutrient sensors modulate these dynamics in distinct ways: CASTOR1 stabilizes WDR24 occupancy across the entire lateral trajectory by gently squaring the GATOR2 scaffold, whereas Sestrin2 restricts longitudinal motion through direct association with the WDR24 β-propeller. In general, we found that mTORC1 signalling is more sensitive to deprivation of leucine than arginine25, which may reflect the greater degree of WDR24 β-propeller restraint imparted by Sestrin2 than by CASTOR1. In agreement, we found that fusion of an epitope tag to the WDR24 β-propeller inhibited mTORC1 activation in a manner that correlated with its size (Fig. Collectively, these results suggest that WDR24 β-propeller flexibility is required for GATOR2 to stimulate mTORC1 signalling, and that both Sestrin2 and CASTOR1 antagonize the function of this domain. The GATOR2 complex is a critical nutrient-sensing hub and regulator of mTORC1 signalling7,10. Despite its importance, the mechanisms by which amino acids influence GATOR2 through their cognate sensors have remained elusive. Here we determined the structure of GATOR2 when bound to its nutrient-gated regulators: Sestrin2 and CASTOR1. We have revealed that the Sestrin and CASTOR protein families occupy non-overlapping binding sites and antagonize GATOR2 function through both distinct and common mechanisms. The ligand-binding pockets of both Sestrin2 and CASTOR1 do not directly overlap with their respective GATOR2-interacting surfaces, suggesting that ligand binding does not first require dissociation from GATOR2. Our results suggest that cognate ligand binding triggers allosteric changes in the nutrient sensors, which reconfigure their GATOR2-binding surfaces to form steric clashes that dissociate them from GATOR2. Thus, GATOR2 can ‘read' the ligand state of a nutrient sensor by interacting with a dynamic portion of the sensor, distant from its ligand-binding pocket. We suspect that nutrient-stimulated allosteric rearrangements probably also prevent nutrient sensor reassociation until cytosolic nutrient levels fall and the sensors dissociate from their ligands. We also found that auxiliary structural features of the nutrient sensors, distant from the ligand-binding pockets, are required for stable association with a sensed metabolite. We speculate that the evolution of these allosterically regulated intramolecular interactions, which stabilize the ligand-bound form of the sensor, might have enabled the adaptation of ancestral enzymes with intrinsically weaker ligand affinities into nutrient sensors capable of stable engagement with their cognate metabolites. These sensors could then have been assimilated into the mTORC1 pathway through the evolution of an interaction between the sensor and a critical GATOR2 region. Given the role for allostery in ligand binding, divergent residues located at surfaces other than the ligand-binding pocket or the GATOR2 interaction surface, such as the poorly resolved Sestrin N termini, may explain the different ligand-binding affinities of Sestrin1–3 and CASTOR1–2 (refs. Despite interacting with distinct surfaces on GATOR2, we found that both Sestrin2 and CASTOR1 promote WDR24 β-propeller rigidification, suggesting that this may be a shared mechanism of GATOR2 antagonism. Our results also suggest that the extent of WDR24 β-propeller interference dictates the potency of mTORC1 inhibition. CASTOR1-dependent allosteric stabilization of the WDR24 β-propeller may also potentiate the GATOR2–Sestrin2 interaction, hinting at synergistic regulation of mTORC1 signalling by distinct amino acids. We also found that the nutrients sensors impact GATOR2 through unique mechanisms: Sestrin2 obstructs an essential GATOR2 functional surface and CASTOR1 reshapes the GATOR2 scaffold and weakens assembly of the GATOR supercomplex. Previously, WDR24 was proposed to function as an E3 ubiquitin ligase regulated by direct interaction between Sestrin2 and its RING domain26. We report, however, that Sestrin2 does not affect the fold or solvent accessibility of the WDR24 RING domain, which does not interact with Sestrin2. The Drosophila melanogaster protein Unmet was identified as a lineage-specific S-adenosyl methionine sensor for the dTORC1 pathway27. Like CASTOR1, Unmet is a dimer and requires the MIOS β-propeller to interact with GATOR2, which suggests that CASTOR1 and Unmet may share a binding site and mechanism of GATOR2 inhibition. Yet-to-be-discovered nutrient sensors may also function through GATOR2 β-propellers. The study of amino acid sensing by mTORC1 in vivo is complicated by the presence of multiple members of the Sestrin and CASTOR families in mammalian genomes, necessitating generation of simultaneous loss-of-function variants28. Here we generated single amino acid substitution variants in GATOR2 that rendered mTORC1 signalling resistant to either Sestrins or CASTORs. We anticipate that these GATOR2 variants will facilitate delineation of distinct physiological roles for leucine and arginine sensing by mTORC1 in vivo. As of submission, artificial intelligence-driven structural prediction tools, trained on the PDB, cannot adequately model the conformational changes that occur in the nutrient sensors upon ligand or GATOR2 binding24,29. We anticipate that empirical structural determination of the GATOR complex, including its unresolved component KICSTOR and regulator SAMTOR, will be critical to unveil the regulatory logic of the mTORC1 nutrient-sensing pathway and to rationally design therapeutics to modulate its function. Reagents were obtained from the following sources: antibodies to S6K pT389 (9234), S6K (2708), MIOS (13557), WDR59 (53385), Sestrin2 (8487), MYC epitope tag (2276 and 2278), Flag epitope tag (14793), HA epitope tag (3724) and horseradish peroxidase-linked anti-rabbit secondary antibody (7074) were from Cell Signaling Technology; antibodies to SEH1L (ab218531) and DEPDC5 (ab213181 and ab185565) were from Abcam; antibodies to SEC13 (15397-1-AP), WDR24 (20778-1-AP) and KPTN (16094-1-AP) were from Proteintech; antibodies to Raptor (09-217) and Flag-M2 (F1804) used for preparing anti-Flag magnetic beads were from MilliporeSigma; and antibody to NPRL3 was from Novus Biologicals (NBP1-88447). InstantBlue Coomassie Protein Stain was from Abcam; anti-Flag-M2 affinity gels, ATP and amino acids were from MilliporeSigma; DMEM, Expi293 Expression Medium, inactivated fetal serum, Dynabeads M-270 epoxy, Dynabeads protein G and anti-HA magnetic beads were from Thermo Fisher Scientific; XtremeGene9, PhosSTOP and cOmplete Protease Inhibitor were from Roche; amino acid-deficient RPMI (R8999-03A and R9010-01) was from US Biologicals; and PEI MAX 40 kDa was from Polysciences. Adherent HEK293T cells were cultured in DMEM (Thermo Fisher Scientific) with 10% inactivated fetal serum (Thermo Fisher Scientific) and 4.5 g l−1 glucose containing 2 mM GlutaMAX (Thermo Fisher Scientific), 100 IU ml−1 penicillin and 100 µg ml−1 streptomycin. Adherent cell lines were maintained at 37 °C and 5% CO2. Expi293F cells were grown in Expi293 medium (Thermo Fisher Scientific) supplemented with 100 IU ml−1 penicillin and 100 µg ml−1 streptomycin. Suspension cells were grown in an INFORS Multitron Pro shaker operating at 37 °C, 8% CO2, 80% humidity and 90–125 rpm. HEK293T cells were obtained from the American Type Culture Collection, and Expi293F cells were obtained from Thermo Fisher Scientific. All cell lines were validated and tested for mycoplasma. Transfection, cell lysis and immunoprecipitations were performed as previously described10. To harvest samples, cells were washed once with ice-cold PBS and then lysed with lysis buffer (1% Triton X-100, 40 mM HEPES pH 7.4, 10 mM β-glycerol phosphate, 10 mM pyrophosphate and 2.5 mM MgCl2) and 1 tablet of EDTA-free protease cocktail (Roche) per 25 ml buffer. For anti-Flag immunoprecipitations, magnetic beads bound to antibody recognizing the Flag epitope tag were prepared in-house by coupling Dynabeads M-270 epoxy (Thermo Fisher Scientific) to Flag-M2 antibody (MilliporeSigma) as previously described10. For anti-HA immunoprecipitations, anti-HA-coupled magnetic beads (Thermo Fisher Scientific) were used. Before use, beads were washed three times with Triton X-100 lysis buffer and then incubated with the supernatant of each clarified lysate for 1 h at 4 °C. Each immunoprecipitation used 10 µl of anti-Flag or 30 µl of anti-HA magnetic beads. For anti-MYC immunoprecipitations (Extended Data Fig. 9b), 4 µl of anti-MYC antibody was added to each sample of clarified lysate and incubated at 4 °C for 1 h. Subsequently, 30 µl of pre-washed protein G magnetic beads was added to each sample and incubated for an additional 1 h at 4 °C. Following immunoprecipitation, beads were washed one time with Triton X-100 lysis buffer and two times with Triton X-100 lysis buffer supplemented to contain 150–500 mM NaCl. Immunoprecipitated proteins were denatured by addition of SDS–PAGE sample buffer and boiling for 5 min at 95 °C. For experiments requiring transfection of cDNAs into HEK293T cells, 2 × 106 cells were plated in 10-cm culture dishes or 5 × 106 cells in 15-cm culture dishes. Twenty-four hours later, cells were transfected with the appropriate pRK5-based cDNA expression plasmids using the polyethylenimine method, as previously described30. The total amount of plasmid DNA in each transfection was normalized to 5 μg with empty pRK5 (10-cm plates) or 20 µg of empty pRK5 (15-cm plates). Thirty-six hours following transfection, cells were lysed as described above. For experiments that required amino acid starvation, cells were incubated in amino acid-free RPMI for 60 min, as previously described31. To restimulate cells following starvation, an amino acid mixture prepared from individual powders of amino acids (MilliporeSigma) was added to cell culture media for 15 min. Starvations for individual amino acids were performed similarly. Experiments evaluating the in vitro dissociation of GATOR2–sensors complexes were performed as previously described16,17,19,21, with the following modifications. Following immunoprecipitation, washing and equilibration in cytosolic buffer, the beads incubated with either 300 µM leucine or 400 µM arginine for 20 min at 4 °C with gentle rocking. The beads were then washed twice more, after which the samples were denatured by addition of sample buffer and analysed by immunoblotting, as described above. For purification of sc-GATOR2, 5 l of Expi293F cells grown in Expi293 medium (Thermo Fisher Scientific) was transiently transfected with cDNAs encoding N-terminally Flag-tagged sc-WDR59–SEC13 (Addgene 237416), tag-free sc-MIOS–SEH1L (Addgene 237417) and tag-free sc-WDR24–SEH1L (Addgene 237418) in the pRK5 vector. To prepare sc-GATOR2 in complex with Sestrin2 or CASTOR1, the cDNA mixture was supplemented with HA–Sestrin2(E451Q) (Addgene 237420) or HA–CASTOR1(D304A) (Addgene 237419) in the pRK5 vector, respectively. Cells were transfected at a density of 3 × 106 per millilitre with 1 mg total cDNA and 4 mg PEI MAX 40 K (Polysciences) per litre culture. Seventy-two hours after transfection, cells were harvested, washed in ice-cold PBS and then lysed in Triton lysis buffer (TLB; 1% Triton X-100, 50 mM HEPES pH 7.4, 100 mM NaCl, 50 mM arginine, 50 mM glutamic acid and 2.5 mM MgCl2) with EDTA-free protease inhibitor cocktail (Roche; 1 tablet per 25 ml buffer) and 1 mM ATP (100 ml of lysis buffer was used per litre cell culture). The lysate was cleared by ultracentrifugation at 50,000g for 20 min. Pre-washed anti-FLAG-M2 affinity gel (MilliporeSigma) was added to the clarified lysate (2 ml slurry per litre culture) and incubated for 2 h at 4 °C on a nutator. The GATOR2 complex was eluted from the beads by competitive elution in CHAPS buffer supplemented with 0.5 mg ml−1 3× Flag peptide (sequence DYKDHDGDYKDHDIDYKDDDDK). Eluate was collected with bead separator columns, concentrated with a regenerated-cellulose centrifugal filter (100 kDa MWCO; MilliporeSigma), and clarified by centrifugation at 21,000g for 10 min at 4 °C to remove any debris. Further purification was performed by size-exclusion chromatography (SEC) using a TSKgel G4000SWxl column (Tosoh) pre-equilibrated in CHAPS buffer supplemented with 2 mM dithiothreitol. Before SEC purification of the sc-GATOR2–Sestrin2 or sc-GATOR2–CASTOR1 complexes, the eluates were supplemented with fivefold molar excess of purified wild-type Sestrin2 or CASTOR1(D304A), respectively. Elution fractions were resolved by SDS–PAGE (4–12% Bolt gels, Thermo Fisher Scientific) and stained with InstantBlue Coomassie Protein Stain (Abcam). Pure protein fractions were pooled, concentrated by centrifugal ultrafiltration, supplemented with 10% glycerol and snap frozen in liquid nitrogen before storage at −80 °C. For analysis by cryo-EM, GATOR2 samples were prepared immediately following SEC. Immediately before grid preparation of the sc-GATOR2–Sestrin2 or sc-GATOR2–CASTOR1 complexes, the purified samples were supplemented with threefold molar excess of purified wild-type Sestrin2 or CASTOR1(D304A), respectively. To generate single-chain constructs, fragments with the following boundaries were prepared by PCR amplification using Q5 High-Fidelity Polymerase (New England Biolabs) and assembled into pRK5 by Gibson Assembly (New England Biolabs): To generate point mutants in cDNAs for structural validation, site-directed mutagenesis was performed using the KLD Enzyme Mix (New England Biolabs) according to the manufacturer's instructions. Cells that stably expressed cDNAs were generated as previously described8. The lentiviral expression plasmids used were: pLJM60-Flag-METAP2 and pLJM1-WDR24, with the latter containing site-directed mutations. Molecular dynamics simulations were performed using GROMACS (2024-rc)32. The initial model was constructed using the coordinates of dimeric CASTOR1 from our cryo-EM structure of sc-GATOR2–CASTOR1, with an arginine ligand positioned in the binding pocket according to its placement in the crystal structure of arginine-bound CASTOR1 (PDB ID: 5I2C)19. To accurately quantify the relocation of the arginine-binding loop relative to the initial input state, simulations specifically targeting loop dynamics were conducted using a single CASTOR1 protomer. By contrast, simulations investigating the mechanism of arginine release and the swimming molecular dynamics binding study were performed using the CASTOR1 dimer to account for potential allosteric effects. Seven independent unbinding events were analysed, five of which exhibited the transient state described in the article. Missing atoms were added, and protonation states of titratable residues were adjusted to reflect physiological pH using the Protein Repair and Analysis Server33. Protein chain termini were modelled in their charged ionic states. The arginine ligand topology was generated using ACPYPE (antexhamber Python parser interface)36. The system was solvated in a triclinic box of explicit TIP3P water molecules, with a 20 Å buffer between the protein and box edges. Na+ and Cl− ions were added to neutralize the system and achieve a physiological ionic strength of 150 mM. Energy minimization was performed using the steepest descent algorithm until the maximum force on any atom fell below 1,000 kJ mol nm−1, ensuring the absence of steric clashes or inappropriate geometry. The system was then equilibrated in two stages: first, isothermal-isochoric (NVT) equilibration at 300 K for 100 ps using the V-rescale thermostat with a time constant of 0.1 ps; second, isothermal-isobaric (NPT) equilibration at 1 bar for 1 ns using the Parrinello–Rahman barostat with a time constant of 2.0 ps. Production molecular dynamics simulations were typically run for 1,000 ns (2-fs time step) at 300 K and 1 bar. Periodic boundary conditions were applied in all three dimensions. Non-bonded interactions were calculated using a 1.0-nm cut-off for van der Waals forces, and long-range electrostatics were computed using the particle-mesh Ewald summation method with a Fourier spacing of 0.12 nm. Bonds involving hydrogen atoms were constrained using the LINCS algorithm. Simulations were run on the Stanford University high-performance computing cluster (SHERLOCK), utilizing 1× GPU and 4× CPU per simulation. Trajectory analyses of RMSD were performed using the RMSD Trajectory Tool in VMD37. In accordance with the reliability and reproducibility standards for molecular dynamics simulations for Nature38, additional details have been provided in Supplementary Table 4. Coordinate files corresponding to critical simulation outputs (Extended Data Fig. 7c–e) are available as Supplementary Information 1 and 2. Cryo-EM specimens were prepared using a Vitrobot Mark IV (Thermo Fisher Scientific). All three purified sc-GATOR2 complexes — apo state, Sestrin2 bound and CASTOR1 bound — were concentrated to 3 mg ml−1 in a final volume of 3 µl. These samples were applied to glow-discharged gold 300 square-mesh Quantifoil R 1.2/1.3 holey carbon grids (Quantifoil) and blotted from both sides for 3 s at 95% humidity before plunge freezing in liquid ethane. Datasets were collected on a Titan Krios G3 electron microscope (Thermo Fisher Scientific) operated at 300 kV, equipped with a Gatan K3 direct detection camera (super-resolution counting mode) and a BioQuantum energy filters (slit width of 20 eV). Specific microscope parameters for each dataset are provided in Supplementary Tables 1–3. Fully automated data collection was performed using the EPU software. Data processing workflows for reconstitution of the three sc-GATOR2 complexes are illustrated in Supplementary Figs. Details for each dataset are summarized below. Contrast transfer function (CTF) parameters were determined using CTFFIND (v4.1.14)42. Motion-corrected micrographs were denoised using Topaz-Denoise43 with pretrained models. For each dataset, particle picking was performed in Topaz (v0.2.5)44 using two distinct search models: (1) a default pre-calculated model provided by the software developers, and (2) a custom-trained model generated from a manually picked subset of 1,000 GATOR2 particles. Reference-based 2D classifications45 were applied to effectively eliminate contaminating non-GATOR2 particles and partially disassembled GATOR2 complexes. The deposited global consensus map from our previously published apo GATOR2 state (EMD-26519) served as the reference input map10. Remaining particles were reclassified in 3D, and the best classes from different Topaz sets (default and trained) were combined using a strict distance cut-off (150 Å) to remove duplicates. The resulting cleaned particle sets (758,113 for apo, 287,922 for Sestrin2 and 537,062 for CASTOR1) were then used to train a refined Topaz search model for a final round of reference-based particle picking. The collected data included a minor subset of CCT–TRiC complex particles, which co-eluted with GATOR2 during SEC due to their similar size of approximately 1 MDa. These particles were identified and removed during the initial 2D classification of each respective dataset. The resulting cleaned refined particle sets were then merged with the clean default and trained sets, using a strict distance cut-off (150 Å) to remove duplicates. These combined particle sets (1,148,926 for apo, 605,308 for Sestrin2 bound and 865,410 for CASTOR1 bound) were re-extracted at full size and refined through iterative cycles of CTF and aberration refinement in RELION (per-particle defocus, per-micrograph astigmatism, beam tilt and higher-order aberrations)46. Alignment of the GATOR2 maps to C2 symmetry and correction of per-particle motion in RELION40 further improved the resolution of the reconstructions. These particle sets were then transferred to cryoSPARC (v4.5)47 and refined using iterative cycles of non-uniform refinement48 and per-particle CTF refinement46. The final consensus maps, generated with C2 symmetry applied, reached resolutions of 3.47 Å (apo state), 3.36 Å (Sestrin2-bound state) and 3.40 Å (CASTOR1-bound state), as determined by Fourier shell correlation at the 0.143 threshold between independently refined half-maps. All sc-GATOR2 particle sets exhibited significant heterogeneity. In contrast to our previous GATOR2 dataset (EMD-26519), in which subunit dissociation complicated the analysis of compositional and conformational heterogeneity10, the newly designed single-chain constructs guaranteed compositional homogeneity. This allowed us to specifically focus on characterizing the intrinsic conformational flexibility of the GATOR2 complex. All particle sets were expanded using C2 symmetry (2,297,852 for apo, 1,210,616 for Sestrin2 and 1,730,820 for CASTOR1) and subjected to masking and local refinement in cryoSPARC (without particle subtraction). This allowed improved visualization of multiple overlapping regions within the GATOR2 complex: four regions for sc-GATOR2–Sestrin2 and five regions each for apo sc-GATOR2 and sc-GATOR2–CASTOR1 (Supplementary Figs. The resulting particles were further analysed for heterogeneity using 3D variability analysis (3DVA)49 in cryoSPARC (filter resolution = 5 Å and iterations = 40) to isolate particle subsets with the highest occupancy in each masked region and to identify rigid sections that move as semi-independent bodies. These selected subsets underwent additional local refinement in cryoSPARC, followed by final sharpening using automated post-processing routines in DeepEMhancer (v0.16)50. High-resolution features were enhanced using the high-res algorithm (resolution cut-off = 2.5 Å), whereas low-resolution features were improved using the wide-target algorithm (resolution cut-off = 4.5 Å). Finally, high-occupancy composite maps (both unsharpened and DeepEMhancer sharpened) were generated in phenix.combine_focused_maps (Phenix v2.0)51 by combining rigid bodies within the respective sc-GATOR2 complexes (Supplementary Figs. The resulting composite sc-GATOR2 maps were generated as pseudosymmetric, with C2 symmetry applied to all structural regions except the brace MIOS–SEH1L β-propellers, which directly span the C2 symmetry axis. Because these β-propellers adopt two distinct orientations — one symmetric and one asymmetric (Supplementary Fig. 15) — we treated them as a C1 feature for consistency. All model building tasks were performed using the C2-symmetric composite maps of the respective sc-GATOR2 complexes. Previously published structures — human apo GATOR2 (PDB ID: 7UHY)10, human leucine-bound Sestrin2 (PDB ID: 5DJ4)21 and human arginine-bound CASTOR1 (PDB ID: 5I2C)19 — were docked into the cryo-EM reconstructions using UCSF ChimeraX (v1.8)52. These initial models were iteratively rebuilt through cycles of interactive adjustments in Coot (v0.9.8)53 and refinement in phenix.real_space_refine (Phenix v2.0)54, incorporating AlphaFold 2 (refs. 55,56) predictions to improve backbone geometry in regions of lower map resolution. For each sc-GATOR2 complex, only one protomer (the asymmetric unit) was built and refined as described above; the full complex was generated by applying C2 symmetry operators. Model refinement was carried out with restraints applied to secondary structure elements and zinc-coordinating residues. Model quality was evaluated using MolProbity (v4.5.2)57. The final models consist of the following: apo sc-GATOR2 (7,588 amino acids and 32 zinc ions), Sestrin2-bound sc-GATOR2 (8,426 amino acids and 32 zinc ions) and CASTOR1-bound sc-GATOR2 (9,594 amino acids and 30 zinc ions). The atomic model of leucine-bound Sestrin2 (originally PDB ID: 5DJ4) was rebuilt based on insights from the sc-GATOR2–Sestrin2 cryo-EM structure. In addition, residues 249–255, 270–280 and 296–297 (the αL1–linker–αL2 segment) were added to the model based on weaker but interpretable electron density. The updated model was improved through iterative cycles of interactive model building in Coot and refinement using phenix.refine (Phenix v2.0)58. After each refinement cycle, the model was evaluated, geometry corrected and adjusted for density fit until convergence was reached. Similarly, the atomic model of arginine-bound CASTOR1 (originally PDB ID: 5I2C) was rebuilt using insights from the sc-GATOR2–CASTOR1 cryo-EM structure. Residues 82–89, corresponding to the release loop, were added to the model based on weaker but interpretable electron density, and the model was iteratively refined as described above. The final updated arginine-bound CASTOR1 model was deposited in the PDB under accession code 9PDO. To evaluate the effects of inhibitor (CASTOR1 and Sestrin2) binding on the dynamic behaviour of the WDR24 β-propeller, particle subsets with full inhibitor occupancy were first selected. Non-symmetry-expanded particle sets for sc-GATOR2–Sestrin2 (605,308 particles) and sc-GATOR2–CASTOR1 (865,410 particles) were analysed for heterogeneity using cryoSPARC 3DVA47,49 (filter resolution = 5 Å and iterations = 40). Masks were applied to cover Sestrin2, SEH1L(WDR24) and the WDR24 β-propeller for sc-GATOR2–Sestrin2, and the brace-associated CASTOR1, SEH1L(MIOS) and the MIOS β-propeller for sc-GATOR2–CASTOR1. The resulting particle subsets with full inhibitor occupancy (307,953 for sc-GATOR2–Sestrin2 and 559,052 for sc-GATOR2–CASTOR1) were locally refined using relaxed C2 symmetry parameters and the masks from 3DVA. To evaluate the flexibility of the WDR24 β-propeller, 3D classifications were performed using a global mask covering the entire GATOR2 complex and a targeted mask focusing on the two WDR24 β-propellers (Extended Data Fig. Hard classification was enforced (convergence criterion = 1% and filter resolution = 10 Å) to strictly distinguish between the presence or absence of the WDR24 β-propeller. Initial testing confirmed that 15,000 particles are sufficient to generate a high-quality sc-GATOR2 map. To ensure robustness and reproducibility, particles were repeatedly classified into subsets of 20,000, 22,000, 25,000, 28,000 and 30,000 particles per class. The resulting classes were categorized based on the presence of double, single or no WDR24 β-propellers on the GATOR2 complex surface (Extended Data Fig. Occupancy ratios were calculated and plotted, with error bars representing the range of values obtained across repeated classifications for each GATOR2 complex. The same WDR24 β-propeller masks and classification protocols were applied to the control particle set of apo sc-GATOR2 (1,148,926 particles). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. & Sabatini, D. M. mTOR at the nexus of nutrition, growth, ageing and disease. & Guan, K.-L. mTOR as a central hub of nutrient signalling and cell growth. & Manning, B. D. Molecular logic of mTORC1 signalling as a metabolic rheostat. Melick, C. H. & Jewell, J. L. Regulation of mTORC1 by upstream stimuli. The Rag GTPases bind raptor and mediate amino acid signaling to mTORC1. Bar-Peled, L. et al. A tumor suppressor complex with GAP activity for the Rag GTPases that signal amino acid sufficiency to mTORC1. Wolfson, R. L. et al. KICSTOR recruits GATOR1 to the lysosome and is necessary for nutrients to regulate mTORC1. Structure of the nutrient-sensing hub GATOR2. Structural mechanisms of the mTOR pathway. Sancak, Y. et al. Ragulator–Rag complex targets mTORC1 to the lysosomal surface and is necessary for its activation by amino acids. Architecture of human Rag GTPase heterodimers and their complex with mTORC1. Architecture of the human GATOR1 and GATOR1–Rag GTPases complexes. Wolfson, R. L. et al. Sestrin2 is a leucine sensor for the mTORC1 pathway. & Guo, L. Crystal structures of arginine sensor CASTOR1 in arginine-bound and ligand free states. Saxton, R. A., Chantranupong, L., Knockenhauer, K. E., Schwartz, T. U. Structural mechanism for the arginine sensing and regulation of CASTOR1 in the mTORC1 signaling pathway. Structural basis for leucine sensing by the Sestrin2–mTORC1 pathway. Tafur, L. et al. Cryo-EM structure of the SEA complex. Agarwal, V. & McShan, A. C. The power and pitfalls of AlphaFold2 for structure prediction beyond rigid globular proteins. Gu, X. et al. SAMTOR is an S-adenosylmethionine sensor for the mTORC1 pathway. Jiang, C. et al. Ring domains are essential for GATOR2-dependent mTORC1 activation. Liu, G. Y., Jouandin, P., Bahng, R. E., Perrimon, N. & Sabatini, D. M. An evolutionary mechanism to assimilate new nutrient sensors into the mTORC1 pathway. Cangelosi, A. L. et al. Zonated leucine sensing by Sestrin–mTORC1 in the liver controls the response to dietary leucine. & Wu, G. New insights into GATOR2-dependent interactions and its conformational changes in amino acid sensing. Boussif, O. et al. A versatile vector for gene and oligonucleotide transfer into cells in culture and in vivo: polyethylenimine. The folliculin tumor suppressor is a GAP for the RagC/D GTPases that signal amino acid levels to mTORC1. Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. Nnyigide, O. S., Nnyigide, T. O., Lee, S.-G. & Hyun, K. Protein Repair and Analysis Server: a web server to repair PDB structures, add missing heavy atoms and hydrogen atoms, and assign secondary structures by amide interactions. Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. & Klein, M. L. Comparison of simple potential functions for simulating liquid water. Sousa da Silva, A. W. & Vranken, W. F. ACPYPE — antechamber Python parser interface. & Schulten, K. VMD: visual molecular dynamics. Reliability and reproducibility checklist for molecular dynamics simulations. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Zivanov, J., Nakane, T. & Scheres, S. H. W. A Bayesian approach to beam-induced motion correction in cryo-EM single-particle analysis. Burt, A. et al. An image processing pipeline for electron cryo‐tomography in RELION‐5. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. & Berger, B. Topaz-Denoise: general deep denoising models for cryoEM and cryoET. Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Kimanius, D., Dong, L., Sharov, G., Nakane, T. & Scheres, S. H. W. New tools for automated cryo-EM single-particle analysis in RELION-4.0. Zivanov, J., Nakane, T. & Scheres, S. H. W. Estimation of high-order aberrations and anisotropic magnification from cryo-EM data sets in RELION-3.1. & Brubaker, M. A.cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Punjani, A., Zhang, H. & Fleet, D. J. Non-uniform refinement: adaptive regularization improves single-particle cryo-EM reconstruction. 3D variability analysis: resolving continuous flexibility and discrete heterogeneity from single particle cryo-EM. Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. Afonine, P. V. et al. New tools for the analysis and validation of cryo-EM maps and atomic models. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. & Emsley, P. Current developments in Coot for macromolecular model building of electron cryo‐microscopy and crystallographic data. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Highly accurate protein structure prediction with AlphaFold. Improved AlphaFold modeling with implicit experimental information. Prisant, M. G., Williams, C. J., Chen, V. B., Richardson, J. S. & Richardson, D. C. New tools in MolProbity validation: CaBLAM for CryoEM backbone, UnDowser to rethink “waters,” and NGL Viewer to recapture online 3D graphics. Liebschner, D. et al. Macromolecular structure determination using X‐rays, neutrons and electrons: recent developments in Phenix. & Swindells, M. B. LigPlot+: multiple ligand–protein interaction diagrams for drug discovery. We thank all members of the Rogala and Sabatini laboratories for helpful insights; E. Brignole and C. Borsa for assistance in cryo-EM data collection; E. Alemany, M. Meloche and the Stanford Research Computing Center for high-performance computing support; T. Terwilliger for implementing fixes in Phenix tools; and T. Schwartz for helpful discussions regarding the crystal structures of Sestrin2 and CASTOR1. This work was supported by grants from the US National Institutes of Health (R35 GM150935, R00 CA255926 and P30 CA124435 to K.B.R. ); the US Department of Defense (CA220856 to K.B.R. (EMBL-Stanford Life Science Alliance), K.Y.L.-G. (National Institutes of Health T32 CA009302), P.V.L. (Burroughs Wellcome Fund Career Award for Medical Scientists Award, the Ellison Foundation, the Smith Family Award for Excellence in Biomedical Research, the MGH Department of Medicine Transformative Scholar Award, and the Chen Institute Department of Medicine Transformative Scholar Award). These authors contributed equally: Max L. Valenstein, Maximilian Wranik Department of Medicine, Massachusetts General Hospital, Boston, MA, USA Whitehead Institute for Biomedical Research, Cambridge, MA, USA Harvard Medical School, Boston, MA, USA Department of Structural Biology, Stanford University School of Medicine, Stanford, CA, USA Maximilian Wranik, Karen Y. Linde-Garelli, Yuri Choi & Kacper B. Rogala Department of Chemical and Systems Biology, Stanford University School of Medicine, Stanford, CA, USA Maximilian Wranik, Karen Y. Linde-Garelli, Yuri Choi & Kacper B. Rogala Maximilian Wranik, Karen Y. Linde-Garelli, Yuri Choi & Kacper B. Rogala Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA Department of Surgery, Massachusetts General Hospital, Boston, MA, USA Broad Institute of MIT and Harvard, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar purified the proteins with assistance from P.V.L. vitrified samples for cryo-EM analysis. updated the crystal structures of Sestrin2 and CASTOR1. designed and performed the biochemical experiments with assistance from P.V.L. performed and analysed the molecular dynamics simulations. prepared the figures and illustrations. All authors edited the manuscript. Correspondence to David M. Sabatini or Kacper B. Rogala. The authors declare no competing interests. Nature thanks Adam Frost and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Wild-type WDR24 and sc-WDR24-SEH1L comparably activate mTORC1 signaling. WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed by immunoblotting for the indicated proteins. (b) Wild-type MIOS and sc-MIOS-SEH1L comparably activate mTORC1 signaling. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (c) Wild-type WDR59 and sc-WDR59-SEC13 comparably activate mTORC1 signaling. WDR59-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (d) Wild-type GATOR2 and sc-GATOR2 comparably activate mTORC1 signaling. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (e) Leucine comparably regulates the interaction between Sestrin2 and wild-type GATOR2 or sc-GATOR2. WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of leucine for 60 min and restimulated with leucine for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (f) Arginine comparably regulates the interaction between CASTOR1 and wild-type GATOR2 or sc-GATOR2. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of arginine for 60 min and restimulated with arginine for 15 min before harvest. Anti-FLAG immunoprecipitates were prepared and analyzed as in (a). (g) Wild-type and sc-GATOR2 comparably activate mTORC1 signaling. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (h) Size-exclusion chromatography profiles of wild-type and sc-GATOR2 complexes. (i) Coomassie blue-stained SDS–PAGE analysis of wild-type and sc-GATOR2 complexes. (j) Comparison of the structures of wild-type GATOR2 (PDB ID: 7UHY) and apo sc-GATOR2 (this study). The MIOS brace region is highly flexible and its position within the sc-GATOR2 structure reflects one possible conformation. Data in (a)-(g) are representative of two independent experiments. For gel source data, see Supplementary Fig. (c) Cartoon representation of the apo sc-GATOR2 model illustrating the reassignment of chains I and J from wild-type GATOR2 (PDB ID: 7UHY), which we were previously unable to assign to a GATOR2 component, to WDR59 (chain D of apo sc-GATOR2). (f) Comparison of the sc-WDR59-SEC13 linker region from wild-type GATOR2 (PDB ID: 7UHY) and apo sc-GATOR2. (a) Validation of the GATOR2-Sestrin2 interface. Anti-FLAG immunoprecipitates (IPs) were prepared from WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs and were analyzed by immunoblotting for the indicated proteins. (b) The GATOR2-Sestrin2 interaction is required for overexpression of Sestrin2 to inhibit mTORC1 signaling. Anti-FLAG immunoprecipitates (IPs) were prepared from HEK293T cells transiently expressing the indicated cDNAs and were analyzed as in (a). Anti-FLAG immunoprecipitates (IPs) were prepared from WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs and were analyzed as in (a). (e) Identification of WDR24 mutants that interfere with the GATOR2-Sestrin2 interaction but not activation of mTORC1 by nutrients. WDR24-deficient HEK293T cells stably expressing the indicated cDNAs were starved of all amino acids for 60 min and restimulated with all amino acids for 15 min before harvest. Cell lysates were analyzed as in (a). (f) The GATOR2-Sestrin2 interaction is required for mTORC1 to sense leucine, but not arginine, deprivation. WDR24-deficient HEK293T cells stably expressing the indicated cDNAs were starved of either leucine or arginine for 60 min and restimulated with leucine or arginine for 15 min before harvest. Cell lysates were analyzed as in (a). Data in (a)-(f) are representative of two independent experiments. For gel source data, see Supplementary Fig. (a) Interactions involving the Sestrin2 αL1 and αL2 helices are required for leucine-induced disruption of the GATOR2-Sestrin2 complex. Anti-FLAG immunoprecipitates were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min. Where indicated, leucine (300 μM) was added directly to the immunoprecipitates, which, after re-washing, were analyzed by immunoblotting for the indicated proteins. Anti-FLAG immunoprecipitates were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min. Data in (a) and (b) are representative of two independent experiments. For gel source data, see Supplementary Fig. (a) Validation of the CASTOR2-GATOR2 interaction. Anti-FLAG immunoprecipitates (IPs) were prepared from MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs and were analyzed by immunoblotting for the indicated proteins. (b) The GATOR2-CASTOR1 interaction is required for overexpression of CASTOR1 to inhibit mTORC1 signaling. Anti-FLAG immunoprecipitates (IPs) were prepared from HEK293T cells transiently expressing the indicated cDNAs and were analyzed as in (a). (c) The MIOS N-terminal β-propeller is not required for amino acids to activate mTORC1 signaling. Anti-FLAG immunoprecipitates (IPs) were prepared from HEK293T cells transiently expressing the indicated cDNAs and were analyzed as in (a). (d) The GATOR2-CASTOR1 interaction is required for arginine deprivation to inhibit mTORC1 signaling. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of arginine for 60 min and restimulated with arginine for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). (e) The GATOR2-CASTOR1 interaction is necessary for arginine deprivation, but not for leucine deprivation, to inhibit mTORC1 signaling. MIOS-deficient HEK293T cells transiently expressing the indicated cDNAs were starved of either leucine or arginine for 60 min and restimulated with leucine or arginine for 15 min before harvest. Anti-HA immunoprecipitates were prepared and analyzed as in (a). Data in (a)-(e) are representative of two independent experiments. For gel source data, see Supplementary Fig. Anti-HA immunoprecipitates (IPs) were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min and were analyzed by immunoblotting for the indicated proteins. (b) The CASTOR1 loop β6-α3 and α3 helix are required for arginine-induced disruption of the GATOR2-CASTOR1 complex. Anti-FLAG immunoprecipitates were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min. Where indicated, arginine (400 μM) was added directly to the immunoprecipitates, which, after re-washing, were analyzed as in (a). Data in (a) and (b) are representative of two independent experiments. For gel source data, see Supplementary Fig. (a) Accessibility of the CASTOR1 active site. In both states, arginine-bound (PDB ID: 5I2C, yellow) and arginine-free (blue) CASTOR1, the arginine-binding loop (residues 269 – 280), which is highlighted in darker colors, fully covers the CASTOR1 surface area restricting the accessibility of soluble arginine towards the CASTOR1 active site. (b) Left: Simulated dynamics of the arginine-binding loop (residues 269 – 280) in the absence of arginine. The general flexibility is measured in r.m.s.d compared to the energy equilibrated input state (step 7) over a simulation run time of 1000 ns. The general flexibility is measured in r.m.s.d compared to the energy equilibrated input state (step 7) over a simulation run time of 1000 ns. (c) Transient coordination states during the arginine unbinding reaction. Left: arginine binding mode obtained by crystallography (PDB ID: 5I2C). Middle and right: Examples for transient arginine coordination states with highlighted critical side chain resides during the simulated arginine-unbinding reaction. (d) Key interactions of the shown transient states determined by LigPlot59. Hydrogen bonds are shown as green dotted lines, while the spoked arcs represent residues making nonbonded contacts with the arginine. (e) Protein surface exposition of the transient arginine coordination states. (f) Disruption of the CASTOR1 residues involved in transient arginine binding stabilizes the GATOR2-CASTOR1 interaction. Anti-HA immunoprecipitates (IPs) were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min and analyzed by immunoblotting for the indicated proteins. (g) CASTOR1 residues involved in transient arginine binding are required for arginine-induced disruption of the GATOR2-CASTOR1 complex. Anti-FLAG immunoprecipitates were prepared from HEK293T cells transiently expressing the indicated cDNAs and deprived of all amino acids for 60 min. Where indicated, arginine (400 μM) was added directly to the immunoprecipitates, which, after re-washing, were analyzed as in (f). Data in (f) and (g) are representative of two independent experiments. For gel source data, see Supplementary Fig. (a) Association of CASTOR1 to GATOR2 disrupts the WDR59 zinc finger (ZnF). Comparison of the WDR59 C-terminal domain (CTD) from apo sc-GATOR2 and sc-GATOR2-CASTOR1. (c) The WDR59 ZnF is required for full activation of mTORC1 signaling and GATOR2-GATOR1-KICSTOR assembly. Anti-FLAG and anti-HA immunoprecipitates were prepared from WDR59-deficient HEK293T cells transiently expressing the indicated cDNAs and analyzed by immunoblotting for the indicated proteins. (d) CASTOR1 selectively associates with GATOR2 and not the entire GATOR supercomplex. Anti-FLAG immunoprecipitates were prepared from HEK293T cells transiently expressing the indicated cDNAs and analyzed as in (c). Data in (c) and (d) are representative of two independent experiments. For gel source data, see Supplementary Fig. The solvent masks applied for particle subset selection in 3D classification are shown in green and applied focus masks are shown in blue. (b) Epitope tagging of the WDR24 N-terminal β-propeller does not disrupt interactions between GATOR2-GATOR1-KICSTOR. Anti-MYC immunoprecipitates from WDR24-deficient HEK293T cells transiently expressing the indicated cDNAs were prepared and analyzed by immunoblotting for the indicated proteins. Data in (b) are representative of two independent experiments. For gel source data, see Supplementary Fig. PDB coordinates for CASTOR1 transition state #1 presented in Extended Data Figure 7d. PDB coordinates for CASTOR1 transition state #2 presented in Extended Data Figure 7d. Flexibility analysis of the apo sc-GATOR2, sc-GATOR2-Sestrin2, and sc-GATOR2-CASTOR1 complexes. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Structural basis for the dynamic regulation of mTORC1 by amino acids. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09421-0'>TCF1 and LEF1 promote B-1a cell homeostasis and regulatory function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 15:47:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. B-1 cells are innate-like immune cells abundant in serosal cavities with antibodies enriched in bacterial recognition, yet their existence in humans has been controversial1,2,3. Unlike conventional B cells that are continuously replenished, B-1a cells are produced early in life and maintained through self-renewal7. Here we show that the transcription factors TCF1 and LEF1 are critical regulators of B-1a cells. LEF1 expression is highest in fetal and bone marrow B-1 progenitors, whereas the levels of TCF1 are higher in splenic and peritoneal B-1 cells than in B-1 progenitors. TCF1–LEF1 double deficient mice have reduced B-1a cells and defective B-1a cell maintenance. These transcription factors promote MYC-dependent metabolic pathways and induce a stem-like population upon activation, partly via IL-10 production. Furthermore, adoptive transfer of B-1 cells lacking TCF1 and LEF1 fails to suppress brain inflammation. These transcription factors are also expressed in human chronic lymphocytic leukaemia B cells and in a B-1-like population that is abundant in pleural fluid and circulation of some patients with pleural infection. Our findings define a TCF1–LEF1-driven transcriptional program that integrates stemness and regulatory function in B-1a cells. B-1a cell development is instructed by B cell receptor (BCR) signal strength and selection is based on BCR recognition of components of bacterial and senescent red blood cell membranes, as well as apoptotic cells and oxidized lipids8. B-1a cells have a restricted BCR repertoire9 and persist through adulthood due to their capacity to self-renew, a process that requires sustained but limited proliferation to maintain a stable cell pool over time10. Because of these properties, B-1a cells have been referred to by some as ‘natural memory' cells11. B-1a cells in the peritoneal cavity have slow turnover rates compared with B-2 cells12, and can completely reconstitute host tissues lacking these cells after adoptive transfer, persisting for many months with little contribution from host-derived cells13. B-1a cells are potent repressors of autoreactive and inflammatory immune responses and tissue damage, largely due to their ability to produce IL-10 (refs. The existence of B-1 cells in humans has, however, been controversial. Here we show that TCF1 and LEF1—known to confer stem-cell like properties in various cell types including CD8+ T cells16—are expressed in mouse B-1 cells and human B-1-like cells. In vivo experiments in mice showed that TCF1 and LEF1 promote B-1 maintenance and regulatory ability via enhancing IL-10 production. To understand the transcriptional program governing B-1 cell homeostasis, we performed single-cell RNA sequencing (scRNA-seq) on sorted peritoneal CD19+ cells from adult mice. Tcf7 (encoding TCF1) was expressed in the Cd5 subcluster, together with Bhlhe41, a regulator of B-1a cell development17 (Fig. ImmGen also revealed high Tcf7 expression in B-1 cells (Extended Data Fig. 17,18) expressed TCF1 by flow cytometry (Extended Data Fig. LEF1 is a transcription factor that interacts with TCF1 (ref. 19), sharing overlapping yet not redundant functions in many biological processes20. Although peritoneal cavity B-1a (PC B-1a) cells expressed low amounts of Lef1 mRNA by scRNA-seq, flow cytometric analysis revealed that the LEF1 protein shared the same pattern of expression as TCF1 (Fig. Lef1 was also expressed in splenic T3 B cells, and both transcription factors were also highly expressed in memory B cells according to Immgen (Extended Data Fig. B-1a cells are enriched in phosphatidylcholine (PtC) reactivity1. We found that peritoneal B-1a cells expressing the lowest amount of B220 contained more PtC-binding B cells and expressed the highest amounts of TCF1 and LEF1 (Fig. TCF1 and LEF1 were also detected in bone marrow pro-B cells that give rise to both B-1 and B-2 cell progenitors, as well as B-1 progenitor (B-1P) cells21 (Extended Data Fig. a, Gene expression in the different clusters projected on uniform manifold approximation and projection (UMAP) of scRNA-seq from total peritoneal B cells. e–g, Flow cytometry plots showing CD38+ and CD38− B cells, CD43, CD5, TCF1 and LEF1 expression from the indicated compartments (e,f) and quantification of the percentage of CD43+CD5+ cells within the CD19+ cells in healthy donor (HD; n = 7), CLL (n = 11), pleural infection (n = 3) and sleep disordered breathing/otitis media with effusion ((SDB/OME); n = 6; g). CTV, CellTrace Violet; FSC, forward scatter; PBMC, peripheral blood mononuclear cell; PF, pleural fluid. h, Contour plots (top) and quantification (bottom) showing PtC reactivity in the indicated subsets from freshly processed PBMC of HD (n = 3). i, Fold change of expression of TCF1 and LEF1 between CD43+CD5+ cells and CD43−CD5− cells in patients with CLL. j–m, Contour plots and quantification of the percentage of B-1, B-2, B-1a and B-1b cells from the peritoneal cavity (j,k); B-1 and follicular B cells from the spleen (l,m) from TCF1WTLEF1WT (n = 4 or 6), TCF1Δ (n = 4), LEF1Δ (n = 6) or TCF1ΔLEF1Δ (n = 4 or 5). n, Frequency of the indicated peritoneal and splenic B cell populations in sublethally irradiated Rag1−/− recipient mice 6 weeks after reconstitution with fetal embryonic day 14.5 liver cells from mice of TCF1WTLEF1WT (n = 4) or TCF1ΔLEF1Δ (n = 7). Each symbol represents an individual mouse and bars represent median values. Statistical analysis was performed using one-way analysis of variance (ANOVA) with Tukey multiple-comparison test (g,k,m) and two-tailed Mann–Whitney U-test (n) or two-way ANOVA (h). The exact P values are shown. In humans, rare circulating B cells bearing B-1 markers have been thought to be pre-plasmablasts22. A recent study sequencing human fetal tissues has identified a B-1-like cell subset expressing CD5 and CD43 (ref. In search for a human adult B-1-like population that might co-express TCF1 and LEF1, we stained cells from pleural effusions drained from patients with bacterial pleural infection. Paired CTV-labelled peripheral blood mononuclear cells and unlabelled pleural fluid cells from the same donor were stained in the same tube, and gatings were applied to exclude CD38+ plasmablasts (Fig. We found a population bearing the B-1 markers CD43 and CD5 and co-expressing TCF and LEF1 that constituted over 80% of all B cells in pleural fluid and up to 60% of peripheral blood B cells in some patients (Fig. Despite this B-1-like population being rare in blood from healthy donors (Fig. 1f,h), it was still enriched in PtC reactivity and expressed higher levels of TCF1 and LEF1 than CD43−CD5+ cells and the bulk of CD43−CD5− cells (Fig. Expression of CD27 and CD24 was heterogenous in human CD43+CD5+ B-1-like cells (Extended Data Fig. Although the majority of CD43+CD5+ cells were IgM+, up to 6% expressed IgA (Extended Data Fig. Of note, PtC-reactive cells were also found in the CD38+ population (Extended Data Fig. A CD43+CD5+ phenotype also characterized the neoplastic chronic lymphocytic leukaemia (CLL) B cells24, which also co-expressed LEF1, a diagnostic marker for CLL25 and/or TCF1 (Fig. Together, these data show that human CD38− B cells that co-express both CD43 and CD5 express higher levels of LEF1 and TCF1 than other mature B cell subsets and are enriched in PtC reactivity, thus displaying a phenotype that resembles mouse B-1 cells. To determine whether TCF1 and LEF1 have a role in B-1 cell homeostasis, we generated mice lacking these transcription factors only in B cells by crossing Tcf7-floxed and Lef1-floxed mice to mice expressing Cre under the control of the Mb1 (Cd79a) promoter (CreMb1)26. Although B-2 and B-1b cells were comparable in all groups, mice double deficient in TCF1 and LEF1 (TCF1ΔLEF1Δ) had a 45% reduction in total peritoneal B-1 cells, a 71% reduction in peritoneal B-1a cells and a 67% reduction in splenic B-1 cells compared with CreMb1-expressing littermate controls (TCF1WTLEF1WT; Fig. Deficiency of TCF1 alone only decreased splenic B-1 cells. TCF1–LEF1 deficiency reduced CD5 expression in B-1a cells but these remained identifiable (Extended Data Fig. To establish whether TCF1 and LEF1 influence homeostasis of both fetal liver-derived and bone marrow-derived B-1a cells, we reconstituted sublethally irradiated Rag1−/− mice with either embryonic day 14.5 fetal liver cells or adult bone marrow cells from mice sufficient or deficient in TCF1 and/or LEF1. B-1a cells were decreased by 70% and 60% in mice receiving TCF1ΔLEF1Δ fetal liver and bone marrow cells, respectively, compared with recipients of wild-type cells (Fig. B-1P cells identified as Lin−CD93+IgM−CD19+B220neg-low peak during late gestation and differentiate into both B-1a and B-1b cells or mature in the spleen through a transitional (TrB-1a) cell stage27 (Extended Data Fig. TCF1 and LEF1 deficiency resulted in an increased frequency of B-1P cells in embryonic day 18.5 fetal liver and neonatal (days 1, 3 and 9) bone marrow (Extended Data Fig. 3b,c), yet splenic transitional B-1a cells (TrB-1a; CD93+IgM+CD19+B220lowCD5+) known to exclusively generate B-1a cells27, as well as their likely counterparts in the bone marrow28—B-1Ps (BM B-1Ps)—appeared significantly reduced (Extended Data Fig. Although this suggests that TCF1–LEF1 are involved in B-1a cell development, the decrease in CD5 expression observed in the absence of these transcription factors may underestimate the number of progenitors. Together, our results indicate that TCF1 and LEF1 are required for the formation of a replete peripheral mature B-1a cell pool. Both the proliferative rate and the expression of TCF1 and LEF1 were significantly higher in young (4 weeks of age) than adult (10–16 weeks of age) mice (Fig. Peritoneal B-1a cells lacking TCF1 and LEF1 failed to expand in adult life compared with the steady increase seen in littermate controls (Fig. To formally evaluate self-renewal, we adoptively transferred peritoneal cells from TCF1WTLEF1WT or TCF1ΔLEF1Δ (CD45.2) donors into congenic CD45.1 recipients, who then received 5-bromodeoxyuridine (BrdU) in the drinking water for 12 days. Deficiency in TCF1 and LEF1 led to a greater reduction in the proportion of donor-derived BrdU-labelled B-1a cells (P = 0.0210) than B-2 cells (P = 0.0948; Fig. In a complementary approach, we adoptively transferred 50:50 mixes of CD45.2 TCF1ΔLEF1Δ:CD45.1 TCF1WTLEF1WT-sorted or control CD45.2 TCF1WTLEF1WT:CD45.1 TCF1WTLEF1WT-sorted peritoneal B-1a cells into unirradiated Rag1−/− mice that cannot repopulate the B-1a cell pool. A smaller fraction of donor-derived CD45.2 B-1a cells was found in both the peritoneal cavity and the spleen when donor B-1a cells lacked TCF1 and LEF1 (Extended Data Fig. a,b, Contour plots (a) and quantification (b) of positive cells and TCF1 or LEF1 expression in peritoneal B-1a cells from 4 (n = 5), 8 (n = 4) and 16 (n = 3) weeks of age mice. c, Contour plots showing B-1a cells from TCF1WTLEF1WT and TCF1ΔLEF1Δ mice at 4 (top) and 14 (bottom) weeks of age, gated on peritoneal CD19+B220− cells (left) and quantification (right). d, Experimental diagram (top left), dot plots (right) and quantification (bottom left) of BrdU-labelled donor-derived B-1a or B-2 cells: TCF1WTLEF1WT (n = 3) and TCF1ΔLEF1Δ (n = 4). e,f, Circular plots show cell numbers of the different clonotypes (e) with corresponding heatmap (number of clones) and quantification of peritoneal B cell clones with a particular VH–VL pairing from TCF1WTLEF1WT and TCF1ΔLEF1Δ mice. g, Contour plots (left) and quantification (right) of PtC liposome-binding CD5+ PC B-1 cell numbers from TCF1WTLEF1WT (n = 6) and TCF1ΔLEF1Δ (n = 4) mice. h, Quantification of the percentage of peritoneal and splenic B-1 cells from ETS1WT (Ets1+/+.CreCd19; n = 5) and ETS1Δ (Ets1flox/flox.CreCd19; n = 4) mice. Each symbol represents an individual mouse, and the bars indicate median values. Results are representative of n = 2 (a–d,g,h) experiments. Statistical analysis was performed using two-tailed Welch's t-test (d), Mann–Whitney U-test (g,h), one-way ANOVA with Tukey multiple-comparison test (b) or two-way ANOVA (c). The exact P values are shown. Consistent with the well-described constitutive BCR signalling in B-1a cells31, phosphorylated SYK, BTK and BLNK could be readily detected ex vivo in B-1a cells in the absence of exogenous stimulation; this was diminished in TCF1ΔLEF1Δ B-1a cells (Extended Data Fig. Downregulation of κ-light and λ-light chains that characterizes B-1a cells was also increased in TCF1ΔLEF1Δ B-1a cells (Extended Data Fig. CD19 remained unchanged in both fetal liver B-1P (FL B1P) and PC B1a cells lacking TCF1 and LEF1 (Extended Data Fig. BCR scRNA-seq of peritoneal CD19+ cells from adult TCF1–LEF1 sufficient versus deficient mice revealed three highly expanded clonal clusters (Fig. 2e) expressing B-1a-associated immunoglobulin V genes composed of heavy-chain and light-chain pairs Ighv11-2/Igkv14-126 (c1), Ighv12-3/Igkv4-91 (c2) and Ighv9-3/Iglv2 (c3) in mice from both genotypes, albeit reduced in TCF1–LEF1-deficient mice (Fig. Although the BCR repertoire of the total peritoneal CD19+ B cell population from TCF1–LEF1-deficient mice appeared more diverse (Extended Data Fig. 5a, top), this was probably only due to the overall decrease in the more clonal B-1a cells and overrepresentation of sequenced B-1b and B-2 cells. By contrast, the B-1 cell BCR repertoire of TCF1–LEF1-deficient peritoneal cavity B cells did appear less diverse than that of wild-type counterparts (Extended Data Fig. 5a, bottom, and 5c) despite reduced CD5 expression, which has been associated with clonal dominance32. Consistent with a decrease in the absolute numbers of B-1a cells, the number of PtC-binding peritoneal B cells was decreased in TCF1–LEF1-deficient mice (Fig. 2g), while still representing approximately 42% of B-1a cells, comparable with wild-type littermates (Extended Data Fig. We did not detect decreased serum antibodies in mice lacking TCF1 and LEF1 (Extended Data Fig. 5d), suggesting compensation by B-1 cell-derived plasma cells33. Indeed, B-1 plasma cells were comparable with those of wild-type mice when quantified as a fraction of total splenic cells, but B-1 plasmablasts were increased when quantified as a fraction of splenic B-1 cells (Extended Data Fig. An approximately sevenfold increase in IgG3+ B-1a cells was seen in mice lacking TCF1–LEF1 (Extended Data Fig. To obtain insights into signalling pathways downstream of TCF1–LEF1 in B-1a cells, we performed RNA-seq on purified TCF1WTLEF1WT and TCF1ΔLEF1Δ B-1a cells, and expression of B-1a signature genes was superimposed for comparison34. B-1a cell identity was largely maintained within the residual B-1a subset in the absence of TCF1–LEF1 (Extended Data Fig. Transcripts encoding the transcriptional factors Ets1 and Irf4 were downregulated in single and double TCF1–LEF1-deficient B-1a cells (Extended Data Fig. Given a previous report showing the B cell intrinsic roles of ETS1 on B cell activation and differentiation35, we investigated whether this transcription factor regulates B-1a cell numbers and found that B-1a cells were reduced in the peritoneal cavity, but not the spleen, of adult mice lacking ETS1 in B cells (Fig. 2h); albeit, this was not evident in young mice (data not shown), suggesting that ETS1 may act downstream of TCF1–LEF1 to promote maintenance of B-1a cells with age. Gene set enrichment analysis revealed that TCF1–LEF1-dependent pathways were linked to cell cycle, including G2–M checkpoint (Cdk1 and Ccnb2) and E2F targets (E2f8 and Cenpm; Extended Data Fig. The hallmark ‘MYC targets' pathway was downregulated in TCF1ΔLEF1Δ B-1a cells (Fig. There was also a strong and significant correlation between the expression of MYC—a master regulator of metabolism—as well as its target gene Bcl2 (ref. B-1 cells depend on glycolysis, acquisition of exogenous fatty acids and autophagy for self-renewal37. Consistent with this, inhibition of fatty acid metabolism, oxidative phosphorylation and glycolysis pathways were also prominent pathways in the residual TCF1ΔLEF1Δ B-1a cells (Fig. a, Gene set enrichment analysis of bulk RNA-seq from TCF1WTLEF1WT and TCF1ΔLEF1Δ B-1a cells shows enrichment for the indicated signalling pathways. The black dotted lines indicate neutral enrichment. b, Correlation analysis between expression of TCF1 or LEF1 and expression of MYC (top) or BCL2 (bottom) in peritoneal B-1a, B-1b and B-2 cells (n = 6). gMFI, geometric mean fluorescence intensity. c, UMAP of scRNA-seq profiles from the four indicated sorted cell subsets (FL B1P and BM B1P), B-1 cells from the spleen (SP B1) and peritoneal cavity (PC B1) barcoded and pooled for sequencing. d,e, Developmental trajectory of the combined (d) or individual (e) FL B1P, BM B1P, SP B1 and PC B1 populations constructed by Monocle2. f, Comb plots displaying the incidence and amplitude of the indicated genes in each subset shown in panel d. g, Dot plot presentation of hallmark gene sets upon gene set enrichment analysis for differentially expressed genes in B-1a cells of the indicated genotypes. h, Dot plot presentation of expression of the indicated genes in human prenatal B-1 cells according to single, double or no TCF1 and LEF1 expression. Panel h was adapted with permission from ref. Data are representative of n = 2 experiments. The significant hallmark gene sets in MSigDB (adjusted P < 0.05) for each contrast based on the hypergeometric testing using cluster profiler package (g). To delineate unique versus overlapping or shared roles of TCF1 and LEF1 in B-1a cell maturation, we generated scRNA-seq data from four distinct B-1 developmental stages, individually enriched, barcoded and then pooled: (1) FL B1Ps, (2) adult bone marrow B-1 progenitors (BM B1Ps), (3) PC B1 cells, and (4) spleen B-1 cells (SP B1s; Fig. Trajectory analysis revealed a continuous, well-populated path with FL B1Ps at one terminus of the trajectory, SP B1 and PC B1 clusters positioned close to the other end, and BM B1Ps dispersed between FL B1Ps and SP B1s or PC B1s (Fig. A small number of SP B1 and PC B1 cells located within the FL B1P cluster (Fig. Lef1 was expressed strongly in B-1Ps from both fetal liver and bone marrow and maintained expression, albeit at low levels, in peripheral B-1 cells. Tcf7 by contrast was low at the progenitor stages, and expression increased in peripheral splenic and peritoneal cavity B-1 cells (Fig. Molecules associated with B-1 cell development including Lin28b39 and Arid3a40 as well as Il7r were highly expressed in fetal B-1Ps, whereas Bhlhe41 was expressed in BM B1Ps, and more strongly in SP B1 and PC B1 cells. The latter also expressed Ctla4 (ref. Interrogation of pathways revealed that IL-6–STAT3, TGFβ and TNF signalling were dysregulated in LEF1-deficient B-1a cells, whereas TCF1 deficiency predominantly affected cell cycle via regulation of E2F targets and G2–M checkpoints. By contrast, deficiency of either transcription factor caused changes in the IL-2–STAT5 signalling pathway (Fig. Reanalysis of a scRNA-seq atlas of human prenatal tissues23 revealed that most cells expressing the mouse B-1a marker genes CD5 and SPN (encoding CD43) co-expressed TCF1 and LEF1 alone or in combination. Cells co-expressing both TCF1 and LEF1 also expressed the highest amounts of CD5, SPN, IL2RG and IL7RA (Fig. As seen in mice (Extended Data Fig. 1g), TCF1 and LEF1 were also expressed in human progenitor B cells, particularly in ProB cells, as well as in cycling B cells, as indicated by Mki67 expression23 (Extended Data Fig. Given that IL-2 signalling is a TCF1–LEF1-dependent hallmark pathway in mouse B-1a cells (Fig. 3h), we asked whether these receptors influence B-1 cell development in humans. We evaluated the B-1-like cells in the circulation of patients with paediatric severe combined immunodeficiency (SCID) with genetic loss-of-function variants in IL2RG and IL7RA, and sufficient events in the CD19+CD38− gate. Patients with IL-2Rγ deficiency had an 80% reduction in B-1-like cells out of total B cells (Extended Data Fig. Given that the median age of the healthy donors was 24 years compared with 6 years in the patients, and that adult mice have twofold more B-1a cells than young mice, further work is needed to reach definitive conclusions. Next, we set out to investigate the potential targets of TCF1 and LEF1 in B-1a cells. Owing to low cell numbers and low-level expression of these transcription factors in B-1a cells, CUT&Tag was not successful. Analysis of TCF1 peaks from chromatin immunoprecipitation followed by sequencing datasets of mouse thymocytes42 and comparison with open chromatin regions identified by assay for transposase-accessible chromatin using sequencing (ATAC-seq) in B-1a cells from Vh12/Vk4 transgenic mice17 revealed TCF1 peaks at promoters or enhancers of Cd5, Myc and Ets1 that also harboured open chromatin regions in B-1a cells, suggesting that they are TCF1 targets (Extended Data Fig. B-1a cells have been described to exert regulatory roles via production of IL-10 (ref. CD21highCD1dhigh marginal-zone B cells44 and plasma cells can also produce IL-10 (ref. We scored the expression of ‘B10-associated genes' and the related ‘negative regulation of immune system genes'46 in our scRNA-seq analysis of peritoneal B cells. Both gene signatures were upregulated in the B-1a c1, c2 and c3 clonal clusters (Figs. These signatures included CTLA4 and PDL1, known to be expressed by B-1a cells and to contribute to various immunoregulatory processes, including promoting regulatory T cell responses, dampening inflammatory macrophage activity, limiting T cell-mediated central nervous system (CNS) damage and maintaining B cell tolerance4,5,6,47,48. Indeed, the PDL1 protein was highly expressed in peritoneal B-1 cells compared with conventional B cells (Fig. B-1 cells with high TCF1–LEF1 expression produced twofold higher IL-10 upon activation than those expressing low amounts of TCF1–LEF1 (Fig. Intravenous lipopolysaccharide (LPS) injection induces IL-10-producing LAG3+CD138+ cells49; the number of these cells was also decreased in mice lacking TCF1ΔLEF1Δ in B cells (Extended Data Fig. a, UMAP showing clonal BCR usage in clusters of peritoneal B cells and expression of Tcf7, Cd5 and Ctla4. c, PDL1 histograms and gMFI in the indicated cell subsets (n = 8). d,e, Flow cytometry plots and quantification of IL-10+ cells in TCF1high or TCF1low PC B1 cells (n = 6; d) and IL-10+ cells within PC B1 cells from TCF1WTLEF1WT (n = 11) and TCF1ΔLEF1Δ mice (n = 5; e). f, Expression of CD5, CD21, CD1d and B220 by CD19lowIL-10+ (blue), CD19highIL-10+ (red) and IL-10− (grey) cells with or without stimulation. g, TCF1 and LEF1 histograms (left) and quantification (right) in the cell subset shown in panel f (n = 3). h, Pseudocolour plots and quantification of IL-10+ splenic B cells from TCF1WTLEF1WT (n = 6) and TCF1ΔLEF1Δ (n = 4) mice. i,j, Correlation analysis between TCF1 or LEF1 expression and the percentage of IL-10+ cells (n = 6; i) or expression of CD5 and the percentage of IL-10+ in peritoneal B-1a cells from TCF1WTLEF1WT (n = 5) and TCF1ΔLEF1Δ (n = 5; j). k, Mean clinical scores of mice treated with either PBS (n = 5) or adoptively transferred with peritoneal B-1 cells from TCF1WTLEF1WT (n = 5) or TCF1ΔLEF1Δ (n = 5). o, Identification of GRM2A-positive B-1 cells in the EAE mouse brain; representative cells are from the area labelled ‘i'. Representative cells from the areas labelled ‘ii–iv' are shown in Extended Data Fig. Each symbol represents an individual mouse, and the bars represent the median values (c–e,g,h). Data are presented as mean ± s.e.m. Statistical analysis was performed using two-tailed Pearson correlation analysis (i,j), two-tailed Mann–Whitney t-test (h), one-way ANOVA with Tukey multiple-comparison test (c–e,g) and two-way ANOVA (k). The exact P values are shown. Among splenic B cells, only CD19low cells exhibiting a CD5+CD1d+ phenotype characteristic of B10 cells43 produced IL-10 in the steady state, whereas upon LPS stimulation, we observed an increase in IL-10 production among CD19high cells that displayed a B-1a phenotype (B220lowCD5highCD21lowCD1dlow) and higher expression of TCF1 and LEF1 (Fig. Only the latter were significantly reduced in TCF1ΔLEF1Δ mice (Fig. There was a strong correlation between the increase in IL-10-producing cells and TCF1 and LEF1 expression. A positive correlation, albeit weak, was also seen with CD5 expression, consistent with the known role for CD5 in promoting IL-10 production50, and the observed reduction in CD5 expression in B-1a cells lacking TCF1–LEF1 (Fig. As TCF1+ and LEF1+ fetal B cells express minimal IL-10 (Fig. 3h), it is likely that these transcription factors promote IL-10 production upon postnatal exposure to commensal bacteria bearing Toll-like receptor (TLR) and B-1a BCR ligands (for example, PtC) that coincides with the largest wave of B-1 cells51. B cells have been shown to have protective roles in human multiple sclerosis52 and its experimental autoimmune encephalomyelitis (EAE) mouse model53. In EAE, such regulation requires B-1a cells rather than recirculating follicular B-2 cells48,54 and is dependent on IL-10 (ref. To test whether TCF1 and LEF1 were required for B-1a-mediated regulation in EAE, an equal number of activated peritoneal B-1 cells from TCF1–LEF1-sufficient or TCF1–LEF1-deficient mice was adoptively transferred 3.5 days after recipient mice were immunized with myelin oligodendrocyte glycoprotein (MOG35–55). Clinical disease severity was significantly reduced in mice receiving TCF1WTLEF1WT B-1 cells and disease onset was also delayed compared with mice receiving TCF1ΔLEF1Δ B-1 cells (Fig. We asked whether adoptively transferred B-1 cells can migrate into the CNS. Activation of B-1 cells modulates markers used for their identification55. We thus generated Gramd2amApple-cre mice in which B-1 cells can be tracked due to expression of the mApple fluorochrome driven by Gramd2a (encoding GRM2A) that we found to be selectively expressed in peripheral B-1 cells (Fig. Immunofluorescence of brain sections 21 days after B-1 cell adoptive transfer in the EAE model (Fig. 7h) revealed GRM2A+ donor-derived B-1 cells in the brain. These were predominantly located underneath the meninges or nearby parenchyma. Although this suggests that B-1a cells can enter the brain and may thus exert local regulatory effects, we cannot exclude additional distal effects in secondary lymphoid organs. Finally, we considered the possibility that TCF1 and/or LEF1 control B-1 cell expansion and self-renewal via boosting B cell-derived IL-10, as B-1 cell-derived IL-10 has been shown to repress proliferation56. In the absence of TCF1–LEF1, LPS-activated B-1 cells proliferated extensively and the ratio of CD138+ plasma cells to non-plasma cells was decreased (Fig. IL-10 blockade using IL-10R-Fc antibody in wild-type B-1 cell cultures mimicked the excessive proliferation seen in TCF1–LEF1-deficient B-1 cells (Extended Data Fig. 7i), whereas the addition of IL-10 to cultures of TCF1–LEF1-deficient B-1 cells partially rescued this phenotype (Fig. Purified peritoneal cavity TCF1WTLEF1WT or TCF1ΔLEF1Δ B-1 cells were cultured with LPS, IL-10R-Fc antibody or IgG1 isotype for 3 days. c, Percentage of non-plasma cells undergone more than two divisions. d, Density plots (left) and mean ratio of plasma cells:non-plasma cells (right) after 3 days of LPS stimulation with or without IL-10. e, Representative gating strategy of MYChigh and MYClow cells at division 3 (D3). f,g, Density plots of MYC expression in proliferating cells (f) and quantification of the percentage of MYChigh and MYClow in each division measured by CTV dilution (g). h, Representative gating strategy of MYChigh and MYClow cells at undivided (D0) and D3 (top) and the ratio of MYClow:MYChigh (bottom). j, Histograms (left) and quantification (right) of CD86, FCRL5 and CD19 expression on TCF1WTLEF1WT or TCF1ΔLEF1Δ B-1 cells stimulated with LPS for 3 days that are either D0 or have undergone D3 or D5. k, Histogram of PDL1 expression in D0 and D1 of TCF1WTLEF1WT or TCF1ΔLEF1Δ B-1 cells stimulated with or without LPS for 3 days. l, Chromatin immunoprecipitation followed by sequencing (ChIP–seq) analysis of TCF1 binding in CD4+CD8+ thymocytes42 and ATAC-seq analysis at the Pdl1 locus on B-1a cells from Vh12/Vk4 transgenic mice17. Panel l was adapted from refs. 17,42, Springer Nature Ltd. Data are presented as mean ± s.d. Data are representative of n = 2 experiments with a total of 9 mice per genotype, with peritoneal cavity cells from 3 mice being pooled together. Statistical analysis was performed using one-way ANOVA with Tukey multiple-comparison test (b–d,h) or two-way ANOVA (j). The exact P values are shown. In CD8+ T cells, TCF1 expression allows the emergence of resting memory ‘stem-like' MYClow cells from cycling effector MYChigh cells via regulation of proliferation, metabolism and differentiation30. A comparable MYClow population appeared among B-1a but not B-2 cells around the third division post-LPS stimulation, and MYClow cells exhibited limited proliferation (Fig. MYClow cells were not generated in B-1 cell cultures lacking TCF1–LEF1 where cells continued to proliferate extensively without downregulating CD19 (Fig. Consistent with being a stem/memory-like population, emerging MYClow B-1a cells at the third division downregulated CD19 and markers associated with B cell activation and exhaustion including CD86, FCRL5 and CD11b (Fig. 5i); this was not seen in TCF1–LEF1-deficient cells (Fig. Expression of CD11c and ZEB2 that identify ‘aged-associated B cells'57 remained unchanged in TCF1–LEF1-deficient B-1a cells (Extended Data Fig. In the absence of TCF1–LEF1, PDL1 was strongly downregulated in activated B-1 cells independently of cell division (Fig. Reanalysis of published ATAC-seq from B-1a cells17 revealed two chromatin-accessible regions in the first two exons of Cd274 (encoding PDL1; Fig. 5l), suggesting that PDL1 is a direct target of TCF1 as reported in double-positive thymocytes42. Thus, TCF1–LEF1 are likely to also promote the regulatory function of B-1a cells by maintaining PDL1 expression. Together, these data suggest that TCF1–LEF1 promote PDL1 and IL-10 expression that contribute to the regulatory function of B-1a cells while limiting excessive proliferation and exhaustion to allow emergence of a MYC1low stem-like population essential for B-1a maintenance. Our findings suggest that TCF1 and LEF1 are important for the generation of IL-10-producing regulatory B-1 cells in the periphery. As the major wave of B-1 cell expansion occurs postnatally upon microbiome exposure58 and bacterial CpG is a potent inducer of IL-10 secretion59, B-1a regulatory function is probably key to help prevent inappropriate immune responses to commensal bacteria and viruses. Gut IgA plasma cells have been shown to arise from the same haematopoietic progenitors as B-1a cells during ontogeny and produce IgA clones in response to neonatal gut viral infection60. Whether early-life B-1a education and their antimicrobial IgA production determines the outcome of protective versus pathogenic or autoimmune reactions to microbial exposures later in life remains to be determined. Our identification of expanded B-1-like cells in the blood of some patients with pleural infection based on CD5, CD43, TCF1 and LEF1 expression suggests that these cells may also serve as a biomarker for early detection of serosal infections (for example, pleuritis and peritonitis) and possibly sepsis. Given the observed gradation in PtC reactivity across CD43+CD5+, CD43–CD5+ and CD43–CD5– circulating populations in healthy donors—and the fact that a fraction of CD38+ cells also exhibit PtC reactivity—it is likely that surface markers such as CD5, CD43 and CD38 change dynamically as B-1-like cells mature. Similarly, an inter-relationship may exist between CD38+ and CD38– B-1-like cells, with CD38 expression changing as these cells mature and undergo activation. Our data also suggest that chronic activation of B-1 cells—known to be selected for self-reactivities early in life32–or loss of IL-10 or PDL1-mediated regulatory function as observed in the absence of TCF1–LEF1, may contribute to autoimmunity. Although TCF1 and LEF1 deficiency had more pronounced effects on B-1 cells, it is possible that these transcription factors also contribute to the homeostasis of long-lived B-2 cell subsets. We observed a trend towards reduced B-2 cell turnover and diminished surface BCR expression in the absence of TCF1 and LEF1. This effect may be particularly relevant for B-2 cells that rely on self-renewal for their maintenance, such as memory B cells, which exhibit the highest expression levels of both transcription factors among B-2 cell populations. Together, our work paves the way for studies that can harness the TCF1–LEF1 axis to control B cell stemness and regulatory ability, and to further investigate the function of human B-1-like cells in infection, autoimmune diseases and cancer. Mb1Cre mice, Rag1−/− and CD45.1 mice were maintained on a C57BL/6 background and housed in specific pathogen-free conditions at the Australian National University (ANU) Bioscience Facility. Tcf7flox/floxLef1flox/flox mice were backcrossed at least six times onto the C57BL/6 background, then were subsequently crossed to Mb1Cre mice to generate conditional Tcf1;Lef1-knockout mice. Mice were used from 8 to 12 weeks, and they were age matched when comparing the effect of two genotypes, except for ageing assessment (4–21 weeks). Following adeno-associated virus (AAV) induction of a packaged rAAV donor molecule CRIPSR–Cas9 reagents were electroporated into fertilized single-cell C57BL/6 zygotes. The donor vector contained an in-frame T2A iCre P2A mApple cassette flanked by 5′ and 3′ regions of homology (400 bp, respectively) designed to insert the donor template in-frame immediately upstream of the Gramd2a stop codon in exon 12, such that expression of iCre and mApple was controlled by the endogenous Gramd2a machinery. The AAV donor vector was synthesized and packaged into AAV serotype 1 by VectorBuilder. The guide sequence used was 5′-ATGTGCAGGTAACAGTCACT-3′ and this was synthesized as a synthetic guide RNA by IDT. Mouse procedures were approved by the ANU's Animal Experimentation Ethics Committee or UK Home Office under project licence (PP2867252). Ets1flox/flox mice crossed more than 12 generations onto a C57BL/6 background were obtained from B. Kee at the University of Chicago. Mice were used at both 9 weeks and 20 weeks of age to compare phenotypes. Mouse procedures involving Ets1flox/flox mice were approved by the University at Buffalo Institutional Animal Care and Use Committee. All patients and healthy controls provided written informed consent. Patients with confirmed bacterial pleural infection were invited to donate samples (Extended Data Table 1); all patients had pleural neutrophil counts of more than 10 million per litre, a neutrophil:lymphocyte ratio of more than 4 and were on antibiotic treatment. All clinical specimens (pleural fluid and blood) were collected for Oxford Radcliffe Pleural Biobank (ethical approval reference: 19/SC/0173, South Central-Oxford C Research Ethics Committee) and the study was approved by the Oxford Radcliffe Biobank Tissue Access Committee (reference number: 22/A093). All donors provided written informed consent. Patients and healthy donors were also recruited following favourable study approval on 4 March 2024 by the London-Brent Research Ethics Committee (REC reference 24/PR/0218, IRAS 330505) and as part of the CAPTURE (NCT03226886) study, a prospective longitudinal cohort study investigating COVID vaccine responses in patients with cancer (Extended Data Table 2). CAPTURE is sponsored by the Royal Marsden Hospital and was approved as a substudy of TRACERx Renal (NCT03226886) by the National Research Ethics Service Committee London, Fulham on 1 May 2020 (REC 11/LO/1996). Baseline samples before COVID vaccination from patients with untreated CLL were selected. Collected as part of the EXACT study, a prospective longitudinal cohort study investigating response to immune checkpoint inhibitor therapy in patients with cancer. EXACT is sponsored by the Royal Marsden Hospital and was approved by the National Research Ethics Service Committee, West Midlands-Black Country on 22 November 2021 (REC/WM/0251). A baseline sample from one patient with untreated CLL and before immune checkpoint inhibitor therapy was selected. Patients and healthy donor peripheral blood samples (Extended Data Table 3) were obtained upon written informed consent, according to protocols NCT03394053 (www.clinicaltrials.gov) approved by the Institutional Review Boards of Children's National Hospital and of the National Institutes of Health and 93-I-0119, approved by the Institutional Review Board of the National Institutes of Health. Adenoid tissue samples were collected from children (2–6 years of age, two girls and four boys) undergoing elective adenoidectomy surgery for either otitis media with effusion (OME) or sleep disordered breathing (SDB) at the Great North Children's Hospital. The parents of all donors provided written consent for tissue donation for research and this project was approved by Newcastle University Biobank (project code: NRTB-1). Ethical approval to obtain blood from healthy individuals was provided by the London-Brent Regional Ethics Committee (REC: 21/LO/0682). Pleural fluid and blood specimens were transferred and processed in the laboratory within hours after collection. Pleural fluid specimens were centrifuged at 800g for 10 min. The acellular supernatant was removed and stored in −80 °C. Then, the cellular pellet was resuspended in red blood cell lysis buffer (5–10 ml; J62990.AK, Thermo Scientific) and incubated for 5 min. The sample was centrifuged at 500g for 5 min and the supernatant was discarded. If there were red blood cells, the process was repeated, otherwise the cells were washed in 5 ml of PBS and centrifuged at 500g for 5 min. The supernatant was discarded, and the cells were resuspended in 5 ml of RPMI enriched with 10% FBS, passed through a 70-µm filter and counted. Whole blood was collected in EDTA tubes (VWR) and stored at 4 °C until processing or processed freshly. All samples were processed within 24 h. Before processing, tubes were brought to room temperature. Peripheral blood mononuclear cells and plasma were isolated by density-gradient centrifugation using centrifugation tubes (SepMate, StemCell) per the manufacturer's instructions. Peripheral blood mononuclear cells either proceeded to be stained with antibodies directly or resuspended in recovery cell culture freezing medium (Fisher Scientific) containing 10% DMSO, placed overnight in CoolCell freezing containers (Corning) at −80 °C and then transferred to liquid nitrogen for long-term storage. Adenoid tissue samples were collected in PBS and immediately mechanically dissociated into a single-cell suspension using scalpel blades and a 100-µm filter. Cells were then cryopreserved in freezing media (10% DMSO and 90% FBS) in liquid nitrogen before flow cytometry experiments. Single-cell suspension from mouse peritoneal cavity, spleens and bone marrow were treated with TruStain FcX rat anti-mouse CD16/32 antibodies (101320, BioLegend). The cells were then stained with primary antibodies used for mouse samples along with LIVE/DEAD Fixable Aqua Dead Cell strain (Invitrogen) or eBioscience Fixable eFluor780 viability dye (Invitrogen). DOPC/CHOL/fluorescein-DHPE (54:45:1) and rhodamine-DH PE liposomes (DOPC/CHOL/rhodamine-DHPE(54:45:1)) were used (F60103F2-R, FormumMax). For cytokine intracellular staining, the cells were stimulated with phorbol 12-myristate 13-acetate (PMA), ionomycin, brefeldin A and LPS for the terminal 5 h of culture. Cells were harvested and stained for surface markers, including ebioscience Fixable eFluor780 viability dye (Invitrogen) to exclude dead cells before cells were fixed. Intracellular staining was performed with the Cytofix/Cytoperm kit (BD) with IL-10-PE (JES5-16E3; 505008, BioLegend) and IL-10-BV421 (JES5-16E3; 563276, BD) as recommended. For transcription factors, the eBioscience FOXP3/transcription factor staining buffer set (Invitrogen) was used per the manufacturer's instructions. For staining phosphorylated BCR signalling components, peritoneal cells were isolated using ice-cold FACS buffer (2% FBS in PBS) and stained on ice with CD19, B220 and CD5 antibodies, followed by immediately being fixed with pre-warmed 1.6% paraformaldehyde for 10 min at 37 °C. Cells were then permeabilized with PERM III buffer (558050, BD) for 30 min on ice and subsequently stained with phosphorylated BCR signalling components for 40 min at room temperature. Human samples were treated with human TruStain FcX CD16, CD32 and CD64 antibodies (BioLegend) and then stained with the following anti-human antibodies: CD3-BV785 (HIT3a; 740961, BD Bioscience), CD19-BUV737 (SJ25C1; 612756, BD Bioscience), CD19-PerCP/Cyanine5.5 (SJ25C1; 363016, BioLegend), CD27-PECy7 (M-T271; 560609, BD Bioscience), CD38-BV605 (HIT2; 569699, BD Bioscience), CD43-BV510 (1G10; 563377, BD Horizon), CD43-FITC (1G10; 555475, BD), CD38-BV510 (HIT2; 303540, BioLegend), CD5-APC (UCHT2; 300612, BioLegend), CD5-PE/Cyanine7 (UCHT2; 300622, BioLegend), IgA-PerCP-Vio700 (IS11-8E10; 130-113-478, MACS), CD24-BV711 (ML5; 563401, BD), IgM-BUV395 (G20-127; 563903, BD), TCF1/TCF7 (C63D9; 2203, Cell Signaling Technology), LEF1 (EPR2029Y; ab137872, Abcam), Alexa Fluor-488 donkey anti-rabbit IgG (minimal x-reactivity Poly4064; 406416, BioLegend) and Alexa Fluor-647 donkey anti-rabbit IgG (minimal x-reactivity Poly4064; 406414, BioLegend). Flow cytometers (LSRFortessa X-20, FACSAria II and LSR II; BD) and software (CellQuest and FACSDiva; BD) were used for the acquisition of flow cytometric data, and FlowJo software (Tree Star) was used for analysis. To generate fetal liver chimera, 1 × 106 fetal liver cells from Tcf7+/floxLef1+/floxCreMb1 or Tcf7flox/floxLef1flox/floxCreMb1 mice (at embryonic day 14.5) were transferred intravenously into sublethally irradiated (500 rad) Rag1−/− recipients. To generate bone marrow chimeras, 2 × 106 bone marrow- derived haematopoietic stem cells from 8-week-old mice with aforementioned genotypes were transferred intravenously into sublethally irradiated Rag1−/− recipients. Mice were given Bactrim in their drinking water for 48 h before injection and for 6 weeks after injection and housed in sterile cages. After 6 weeks of reconstitution, mice were taken down for phenotyping by flow cytometry. Equal numbers (1 × 105 cells) of flow-cytometry-sorted peritoneal CD19+B220−CD5+CD23− B-1a cells from Tcf7+/+Lef1+/+ (CD45.2+) or Tcf7flox/floxLef1flox/floxCreMb1 (CD45.2+) and wild type (CD45.1+) were adoptively transferred into Rag1−/− recipients intraperitoneally. The frequency of donor B cells among total peritoneal and splenic B cells was analysed at 2 months after transfer. Peritoneal cells were harvested from Tcf7+/+Lef1+/+ (CD45.2+) or Tcf7flox/floxLef1flox/floxCreMb1 (CD45.2+) donors by injecting 5 ml of serum-free, DMEM medium without l-glutamine (Gibco) into the peritoneal cavity and withdrew as much fluid as possible, followed by centrifuging to harvest the cells; 3 × 106 cells were then injected intraperitoneally into the CD45.1 recipients. The recipients were fed with water containing 0.8 mg ml−1 BrdU for 12 days. Mice were then killed; the frequency of BrdU-positive cells among indicated donor cells in the peritoneal cavity was analysed using a BrdU staining kit (8811-6600-42, Thermo Fisher) and analysed by flow cytometry. Flat-bottom 96-well ELISA plates (3855, Thermo Fisher) were coated with goat anti-mouse κ-UNLB (1050-01, Southern Biotech) overnight. The plates were subsequently washed and blocked using 1% bovine serum albumin in 1× PBS for 1.5 h at 37 °C. Serial dilution of the mouse serum was added to the wells and incubated overnight at 4 °C. Targeted antibodies were detected using AP-conjugated goat anti-mouse IgM (1020-04, Southern Biotech) and AP-conjugated goat anti-mouse IgG3 (1100-04, Southern Biotech). Plates were developed using 1 mg ml−1 phosphatase substrate tablets (S0942, Sigma-Aldrich), and the absorbances at 405 nm and 605 nm were measured using an Infinite 200 PRO plate reader (Tecan) equipped with i-control (v1.9) software. Serum was added in serial dilution before the addition of secondary antibody: alkaline phosphatase (AP)-conjugated goat anti-mouse IgM (1020-04, Southern Biotech) and AP-conjugated goat anti-mouse IgG3 (1100-04, Southern Biotech). In brief, isolated leukocytes or purified cells were resuspended (1 × 106 cells per millilitre) with LPS (10 μg ml−1), PMA, ionomycin and brefeldin A (1:500; BioLegend) for 5 h. For IL-10 detection, cells were stained with surface markers followed by fixation, permeabilization with the Cytofix/Cytoperm kit (BD Bioscience) and staining with IL-10-PE (JES5-16E3, BioLegend) or IL10-BV421 (JES5-16E3, BD) according to the manufacturer's instructions. Total RNA was purified from sorted peritoneal B-1a cells (CD19+CD3−B220−CD5+CD23−7AAD−) using the PicoPure RNA isolation kit. Library construction and sequencing were performed in the Biomolecular Resource Facility, the John Curtin School of Medical Research, ANU. The single end reads of 76-bp sequencing were generated on a HiSeq2000 machine with a depth of more than 30 million reads per sample. 63) and the mapped reads were assigned with FeatureCounts (v2.4)64 based on the genome-build GRCm38.p4 annotation and NCBI Refseq gene mode by removing ribosomal genes and non-coding RNA, respectively. Differential expression analyses were performed with voom-limma65, after removal of lowly expressed genes and normalized using the trimmed mean of M-values method66,67. Significantly differentially expressed genes were identified by applying a Benjamini–Hochberg adjusted P value threshold of 0.05 (ref. Gene set enrichment or pathway analysis were performed using clusterProfiler69 and Camera70 against the Gene Ontology database, KEGG database and HALLMARK C2 and C7 gene sets in the MSigDB (v7.5). Peritoneal B cells (CD19+CD3−7AAD−) from three 8-week-old mice, B-1Ps (Lin−CD93+IgM−CD19+B220low/−) from embryonic day 18.5 fetal livers (three fetal livers were pooled together as one replicate) or bone marrow from three 8-week-old mice, and B-1 cells (CD19+B220low/−) from the peritoneal cavity and spleen from three 8-week-old mice were sorted by the FACS Aria II cell sorting system (BD Immunocytometry Systems). Cells (n = 10,000–20,000) per sample were run on the 10X Chromium platform (10X Genomics). Library preparation and sequencing were performed by the Australian Cancer Research Foundation Biomolecular Resource Facility, the John Curtin School of Medical Research, ANU or Genomics STP, The Francis Crick Institute according to the manufacturer's instructions for the Chromium Next GEM Single Cell 5′ Kit v2 or v3. Two libraries were generated and the mRNA transcript expression and BCR repertoire were measured. Data were processed and analysed using the Seurat package (v5.0.3) in R (v4.4.1). Raw gene expression matrices were filtered to exclude low-quality cells based on the thresholds for mitochondrial gene content (set at 5%) and the number of detected genes per cell, using the PercentageFeatureSet() and subset() functions. For multiplexed samples labelled with cell hashing antibodies, demultiplexing was performed using the MULTIseqDemux() function in Seurat. Hashtag oligonucleotide counts were normalized using centred log-ratio (CLR) transformation via the NormalizeData() function with normalization.method = ‘CLR', and cells were classified as singlets, doublets or negatives based on the hashtag oligonucleotide signal. Only singlets were retained for downstream analysis. Normalization and integration were performed using the scTransform() workflow, followed by the identification of highly variable features. Dimensionality reduction was performed using principal component analysis via the RunPCA() function, and the top 20–30 principal components were selected to construct a shared nearest neighbour graph using FindNeighbours(). Clustering was conducted using the FindClusters() function. Clusters were visualized using UMAP via the RunUMAP() function. Differential expressed genes for each cluster were identified using the FindAllMarkers() function with test.use set as either ‘wilcox' or ‘MAST'. Where applicable, cell types were annotated based on canonical marker expression or informed by publicly available datasets. Cell trajectory analysis was performed using Monocle2 (v2.32.0) in R (v4.4.1). To filter relevant genes, those expressed in at least 10 cells and with a mean expression value of 0.1 or greater were selected. Next, highly variable genes across cell subsets were identified using the differentialGeneTest() function with the formula fullModelFormulaStr = ‘~cell_subset'. On the basis of their q-value, the top 2,000 genes were selected, and setOrderingFilter() was applied to prioritize them for ordering in pseudo-time analysis. Dimensionality reduction was performed using reduceDimension() with the parameter reduction_method = ‘DDRTree'. Cells were subsequently ordered along the trajectory using the orderCells() function and visualized with plot_cell_trajectory(). All other parameters were set to their default values. BCR repertoire analysis was conducted with the Immcantation framework, following the guidelines provided by its developers (http://immcantation.org/). This was implemented in a Python environment (v3.11.10). We began with 10X Genomics BCR sequencing data processed through the CellRanger pipeline (v7.0.1) to obtain annotated clonotype and contig sequences. For V(D)J gene assignment, the AssignGenes.py command from the Change-O toolkit was used, referencing the mouse IMGT and IgBLAST (v1.22.0) databases. Cells expressing more than one heavy chain or only light chains were removed. Clonotypes were assigned using the DefineClones.py tool based on heavy chain sequences, using the default Hamming distance substitution model. Subsequently, light_cluster.py was applied to refine clonotype grouping by incorporating light chain information. Finally, the resulting datasets were merged with the annotated Seurat file and key metrics, including V(D)J gene usage, CDR and FWR sequences, lengths and mutational content were extracted for downstream analysis and visualization. For diversity analysis, we used the iNEXT package (v3.0.1) to perform coverage-based rarefaction and extrapolation analyses. The resulting sampling curves, which illustrate diversity estimates with respect to sample coverage, were visualized using ggiNEXT() function. In addition, we constructed circle packing plots using the packcircles package (v0.3.6) to visualize the immune repertoire composition of individual samples. These analyses were conducted in R (v4.4.1). Processed and annotated scRNA-seq data of fetal immune cells, specifically the haematopoietic stem and progenitor cells and the B cell lineages, in Suo et al.23 were downloaded from the developmental cell atlas portal (https://developmentcellatlas.cellgeni.sanger.ac.uk/fetal-immune/lymphoid/). The B-1 cells were classified into four clusters based on the expression of LEF1 and TCF7. All the analyses were performed using scanpy (v1.9.8). FASTQ files were downloaded from the NCBI's Gene Expression Omnibus database for TCF1 and LEF1 chromatin immunoprecipitation followed by sequencing (ChIP–seq) data (SRP142342)42. FASTQ files were aligned to Enembl's mouse GRCm38 genome using BWA (v0.7.15). The resulting BAM files were sorted, duplicates marked and indexed using Picard (v2.1.1). Peaks were called using MAC2 (v2.1.1) that were enriched in TCF1 or LEF1 relative to input using default parameters. Peaks were annotated using HOMER (v4.8). BAM files were normalized to 10 million reads, and IGVTools (v2.3.75) was used to generate coverage files. FASTQ files were downloaded from the NCBI's Gene Expression Omnibus database for ATAC-seq samples in B-1a cells (GSM2461745)17. FASTQ files were aligned to Ensembl's mouse GRCm38 genome using BWA (v0.7.15). The resulting BAM files were sorted, duplicates marked and indexed using Picard (v2.1.1). IGVTools (v2.3.75) was used to generate coverage file for visualization. Cells were cultured in RPMI 1640 medium supplemented with 10% FBS, 2 mM l-glutamine, 100 U penicillin–streptomycin, 0.1 mM non-essential amino acids, 100 mM HEPES and 55 μM 2-mercaptoethanol at 37 °C in 5% CO2. B-1 cells were magnetically purified from the peritoneal cavity using the Pan B cell isolation kit II (130-104-443, Miltenyi Biotec) with anti-mouse CD45R (B220) antibody (130-110-707, Miltenyi Biotec) and labelled with CellTrace Violet (C34557, Thermo Fisher) followed by stimulating with or without 5 µg ml−1 LPS (O111:B4; L4391, Sigma), 0.4 µg ml−1 IL-10 (210-10-10UG, Thermo Fisher), 10 µg ml−1 InVivoMAb anti-mouse IL-10R (CD210; BE0050, BioXCell) and its isotype control, InVivoMAb rat IgG1 isotype control, anti-horseradish peroxidase (HRP; BE0088, BioXCell) for 72 h. B-1 cells were magnetically purified from the peritoneal cavity using the Pan B cell isolation kit II (130-104-443, Miltenyi Biotec) with anti-mouse CD45R (B220) antibody (130-110-707, Miltenyi Biotec) and cultured in RPMI 1640 medium supplemented with 10% FBS, 2 mM l-glutamine, 100 U penicillin–streptomycin, 0.1 mM non-essential amino acids, 100 mM HEPES, 55 μM 2-mercaptoethanol and 5 µg ml−1 LPS (O111:B4; L4391, Sigma) for 3 days at 37 °C in 5% CO2. Supernatant was collected from cell culture, and IL-10 production was measured by the cytometric bead array mouse IL-10 Flex set (BD). EAE was induced by immunization with MOG35–55 peptide, emulsified in Complete Freund's adjuvant (Sigma-Aldrich) and pertussis toxin (Sigma-Aldrich). On day 3.5, 2 × 105 B-1 cells that had been previously activated with R848 (0.1 μg ml−1) for 48 h were transferred intravenously. Sections (3 µm) were staining on the Leica Bond Rx automated staining platform using sequential application of antibodies with Opal TSA fluorophore detection followed by heat stripping of the antibody complex between stainings using ER1 or ER2 antigen retrieval solution (Leica) for 20 min at 95 °C. The following antibodies, secondaries and opal TSA reagents were used: mApple (1:100; STJ140269, St John's Laboratory), CD45R/B220 (1:750; 553086, BD Biosciences), CD19 (1:200; ab245235, Abcam), Immpress HRP Horse anti-goat IgG polymer (MP-7405-50, Vector), streptomycin-HRP (1:500; P0397, Dako), NovoLink Max Polymer (RE7260-CE, Leica) and Opal 520/570/690 (FP1487001KT/FP1488001KT/FP1497001KT, Akoya Biosciences). Slides were counterstained with DAPI and imaged on the PhenoImager HT (Akoya Biosciences). Statistical methods used in RNA-seq, scRNA-seq and single-cell BCR-seq are described above. Comparison between groups was performed using parametric t-test, two-tailed Welch's t-test, Mann–Whitney t-test, one-way ANOVA with Tukey multiple-comparison test, two-way ANOVA and two-tailed Pearson correlation analysis from GraphPad Prism10 (GraphPad Software). Sample sizes are provided in the figures, and statistically significant differences are indicated as exact P values. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data supporting the findings of this study are available within the paper or its supplementary material. Datasets that support the findings of this study are accessible at the following repositories: RNA-seq data (GSE290505), and scRNA-seq and scBCR-seq data (GSE294717 and GSE298030). Processed and annotated scRNA-seq data of fetal immune cells, specifically the haematopoietic stem and progenitor cells and the B cell lineages in Suo et al.23 were downloaded from the developmental cell atlas portal (https://developmentcellatlas.cellgeni.sanger.ac.uk/fetal-immune/lymphoid/). Source data are provided with this paper. No custom code was used in this study. Baumgarth, N. The double life of a B-1 cell: self-reactivity selects for protective effector functions. Smith, F. L. & Baumgarth, N. B-1 cell responses to infections. Yang, Y. et al. Antigen-specific memory in B-1a and its relationship to natural immunity. Yang, Y. et al. CTLA-4 expression by B-1a B cells is essential for immune tolerance. Popi, A. F., Lopes, J. D. & Mariano, M. Interleukin-10 secreted by B-1 cells modulates the phagocytic activity of murine macrophages in vitro. Khan, A. R. et al. PD-L1hi B cells are critical regulators of humoral immunity. Positive selection of anti-Thy-1 autoreactive B-1 cells and natural serum autoantibody production independent from bone marrow B cell development. Distinct mechanisms define murine B cell lineage immunoglobulin heavy chain (IgH) repertoires. Differential development of progenitor activity for three B-cell lineages. Martin, F. & Kearney, J. F. B-cell subsets and the mature preimmune repertoire. Marginal zone and B1 B cells as part of a “natural immune memory”. Baumgarth, N. A hard(y) look at B-1 cell development and function. Aziz, M., Holodick, N. E., Rothstein, T. L. & Wang, P. The role of B-1 cells in inflammation. O'garra, A. et al. Ly-1 B (B-1) cells are the main source of B cell-derived interleukin 10. Johnson, J. L. et al. Lineage-determining transcription factor TCF-1 initiates the epigenetic identity of T cells. Essential role for the transcription factor Bhlhe41 in regulating the development, self-renewal and BCR repertoire of B-1a cells. Shan, Q. et al. Tcf1-CTCF cooperativity shapes genomic architecture to promote CD8+ T cell homeostasis. Montecino-Rodriguez, E. & Dorshkind, K. B-1 B cell development in the fetus and adult. Characterization of proposed human B-1 cells reveals pre-plasmablast phenotype. Suo, C. et al. Mapping the developing human immune system across organs. Durrieu, F. et al. Normal levels of peripheral CD19+CD5+ CLL-like cells: toward a defined threshold for CLL follow-up — a GEIL-GOELAMS study. Menter, T., Dirnhofer, S. & Tzankov, A. LEF1: a highly specific marker for the diagnosis of chronic lymphocytic B cell leukaemia/small lymphocytic B cell lymphoma. Pedersen, G. K. et al. B-1a transitional cells are phenotypically distinct and are lacking in mice deficient in IκBNS. Identification of novel B-1 transitional progenitors by B-1 lymphocyte fate-mapping transgenic mouse model Bhlhe41 dTomato-Cre. Moreira, S. et al. A single TCF transcription factor, regardless of its activation capacity, is sufficient for effective trilineage differentiation of ESCs. Lin, W. H. W. et al. CD8+ T lymphocyte self-renewal during effector cell determination. Wong, S. C. et al. Peritoneal CD5+ B-1 cells have signaling properties similar to tolerant B cells. Vanhee, S. et al. Lin28b controls a neonatal to adult switch in B cell positive selection. Smith, F. L. et al. B-1 plasma cells require non-cognate CD4 T cell help to generate a unique repertoire of natural IgM. Sun, J. et al. Transcriptomics identify CD9 as a marker of murine IL-10-competent regulatory B cells. Russell, L. et al. Requirement for transcription factor Ets1 in B cell tolerance to self-antigens. & Cleveland, J. L. Myc pathways provoking cell suicide and cancer. Clarke, A. J., Riffelmacher, T., Braas, D., Cornall, R. J. & Simon, A. K. B1a B cells require autophagy for metabolic homeostasis and self-renewal. The Hox11 gene is essential for cell survival during spleen development. Yuan, J., Nguyen, C. K., Liu, X., Kanellopoulou, C. & Muljo, S. A. Lin28b reprograms adult bone marrow hematopoietic progenitors to mediate fetal-like lymphopoiesis. Zhou, Y. et al. Lin28b promotes fetal B lymphopoiesis through the transcription factor Arid3a. Kobayashi, M. et al. Bmi1 maintains the self-renewal property of innate-like B lymphocytes. Emmanuel, A. O. et al. TCF-1 and HEB cooperate to establish the epigenetic and transcription profiles of CD4+CD8+ thymocytes. Yanaba, K. et al. A regulatory B cell subset with a unique CD1dhiCD5+ phenotype controls T cell-dependent inflammatory responses. Gray, M., Miles, K., Salter, D., Gray, D. & Savill, J. Apoptotic cells protect mice from autoimmune inflammation by the induction of regulatory B cells. Signaling via the MyD88 adaptor protein in B cells suppresses protective immunity during Salmonella typhimurium infection. Characterization of organ-specific regulatory B cells using single-cell RNA sequencing. & Haas, K. M. The PD-1 regulatory axis inhibits T cell-independent B cell memory generation and reactivation. Immune tolerance to apoptotic self is mediated primarily by regulatory B1a cells. Lino, A. C. et al. LAG-3 inhibitory receptor expression identifies immunosuppressive natural regulatory plasma cells. Gary-Gouy, H. et al. Human CD5 promotes B-cell survival through stimulation of autocrine IL-10 production. Gu, Q. et al. Intestinal newborn regulatory B cell antibodies modulate microbiota communities. Distinct effector cytokine profiles of memory and naive human B cell subsets and implication in multiple sclerosis. Fillatreau, S., Sweenie, C. H., McGeachy, M. J., Gray, D. & Anderton, S. M. B cells regulate autoimmunity by provision of IL-10. Kurnellas, M. P. et al. Amyloid fibrils activate B-1a lymphocytes to ameliorate inflammatory brain disease. Kreuk, L. S. M. et al. B cell receptor and toll-like receptor signaling coordinate to control distinct B-1 responses to both self and the microbiota. Heine, G. et al. Autocrine IL-10 promotes human B-cell differentiation into IgM- or IgG-secreting plasmablasts. Vergani, S. & Yuan, J. B-1 cells carry the memory of neonatal immune imprinting. Tedder, T. F. B10 cells: a functionally defined regulatory B cell subset. Vergani, S. et al. A self-sustaining layer of early-life-origin B cells drives steady-state IgA responses in the adult gut. Steinke, F. C. et al. TCF-1 and LEF-1 act upstream of Th-POK to promote CD4+ T cell lineage choice and cooperate with Runx3 to silence the Cd4 gene in CD8+ T cells. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Bourgon, R., Gentleman, R. & Huber, W. Independent filtering increases detection power for high-throughput experiments. A scaling normalization method for differential expression analysis of RNA-seq data. Controlling the false discovery rate: a practical and powerful approach to multiple testing. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Wu, D. & Smyth, G. K. Camera: a competitive gene set test accounting for inter-gene correlation. Wang, R. X. et al. Interleukin-35 induces regulatory B cells that suppress autoimmune disease. Chao, A. et al. Rarefaction and extrapolation with Hill numbers: a framework for sampling and estimation in species diversity studies. We thank D. Calado, T. Chen and A. Xu for expert advice; D. Schneider-Luftman, M. Llorian Sopenia and Science Technology Platorms for help with next-generation sequencing analysis; S. Varsani-Brown and J. Olsen from Genetic Modification Service for generating the mouse model; Biological Research Facility for animal care; Experimental Histopathology for experimental tissue analysis in the Francis Crick Institute; the Australian Phenomics Facility for animal care; the CAPTURE Consortium, H. Vohra and M. Devoy from the CHASM facility for assistance with FACS sorting; J. Darcy for sample preparation from the National Institute for Health and Care Research Birmingham Biomedical Research Centre; the personnel of the Australian Cancer Research Foundation Biomolecular Resource Facility for Sanger sequencing; and P. Lane for critical reading of our manuscript. This work was supported by the Francis Crick Institute (CC2228), which receives its core funding from Cancer Research UK, the UK Medical Research Council and the Wellcome Trust; and has also received funding by a Royal Society Wolfson Fellowship, a Wellcome Trust Discovery grant to C.G.V., UK Research and Innovation/BCUK (HP MR/X006891/1) to H.M.P., funds from the National Institute of Allergy and Infectious Diseases Division of Intramural Research to L.D.N., and the Chinese Academy of Medical Sciences Innovation Fund for Medical Science, China (grant number 2024-I2M-2-001-1) to N.I.K. Open Access funding provided by The Francis Crick Institute. These authors contributed equally: Hao Wang, Jonathan A. Roco, Xiangpeng Meng Francis Crick Institute, London, UK Qian Shen, Hao Wang, Jonathan A. Roco, Probir Chakravarty, Amalie Grenov, Max Emmerich, Ilenia Papa, James C. Lee, Paula Gonzalez-Figueroa, Samra Turajlic & Carola G. Vinuesa Division of Immunology and Infectious Disease, John Curtin School of Medical Research, Australian National University, Canberra, Australian Capital Territory, Australia Jonathan A. Roco, Xiangpeng Meng, Paula Gonzalez-Figueroa & Yaoyuan Zhang Laboratory of Clinical Immunology and Microbiology, National Institute of Allergy and Infectious Disease, National Institutes of Health, Bethesda, MD, USA Marita Bosticardo, Ottavia M. Delmonte, Alexandra F. Freeman & Luigi D. Notarangelo Department of Biochemistry, State University of New York at Buffalo, Buffalo, NY, USA Michael Battaglia & Lee Ann Garrett-Sinha ANU Bioinformatics Consultancy, John Curtin School of Medical Research, Australian National University, Canberra, Australian Capital Territory, Australia Benjamin James Talks & Jason Powell Vijaya Baskar Mahalingam Shanmugiah & Muzlifah Haniffa Renal and Skin Unit, The Royal Marsden Hospital, London, UK Division of Allergy and Immunology, Children's National Hospital, Washington, DC, USA Frazer Institute, Faculty of Medicine, The University of Queensland, Brisbane, Queensland, Australia Ian Frazer Centre for Children's Immunotherapy Research, Child Health Research Centre, Faculty of Medicine, The University of Queensland, Brisbane, Queensland, Australia Centre for Discovery and Innovation, Hackensack University Medical Center, Nutley, NJ, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Qian Shen or Carola G. Vinuesa. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) UMAP plots showing the expression of Bhlhe41 after removing all BCR genes in peritoneal total B cells. (b) Expression of Tcf7 and Lef1 across the ImmGen Database. (c) Flow cytometry plots present TCF1 or LEF1 positive B cells in spleen. (f-g) Contour plots (f) and gMFI quantification of TCF1 and LEF1 in B cells subsets from bone marrow (n = 5) (g). (h) Contour plots showing gating strategy and TCF1 and LEF1 expression in CD43+CD5+ B cells from PBMC and pleural fluid (PF) of patient. (i) Fold change of expression of TCF1 and LEF1 between CD43+CD5+ cells and CD43−CD5− cells in each donor (n = 3). Fold change of expression of TCF1 and LEF1 between CD43+CD5int cells or CD43lo/−CD5+ and CD43-CD5- cells in HD (n = 6). Each symbol represents an individual mouse or donor, and bars represent median values. Data are representative of n = 2 experiments. Statistical analysis was performed using one way ANOVA with Tukey multiple-comparison test (e,g), two way ANOVA (j), Paired t test (i). The exact P values are shown. (c) gMFI of CD5 expression within B-1a cells in peritoneal cavity between TCF1WTLEF1WT (n = 7) and TCF1ΔLEF1Δ (n = 7) mice. (d) Frequency of bone marrow (BM) B cell populations from lethally irradiated Rag1–/– recipient mice 6 weeks after transferring foetal liver cells (E14.5) from TCF1WTLEF1WT (n = 4) and TCF1ΔLEF1Δ (n = 7). (e) Frequency of the indicated peritoneal and splenic B cell populations in sub-lethally irradiated Rag1–/– recipient mice 7 weeks after reconstitution with bone marrow cells from mice with TCF1WTLEF1WT (n = 5), TCF1Δ (n = 6), LEF1Δ (n = 7), TCF1ΔLEF1Δ (n = 6) mice. Each symbol represents an individual mouse and bars represent median values. Data are representative of n = 3 (a-c), n = 4 (d-e) experiments, Statistical analysis was performed using two-tailed Mann-Whitney U test (b-d), one way ANOVA with Tukey multiple-comparison test (a,e). The exact P values are shown. (a) Diagram depicting key stages of B-1a development. (h) Flow cytometry plots of Ki67 positive cells and histogram of TCF1 or LEF1 in splenic B-1a cells in 4 weeks old mice (Young, n = 5) and 8-16 weeks old mice (Adult, n = 7). (i) Treatment protocol of mixed sorted B-1a cells of (50% ly5b TCF1WTLEF1WT + 50% ly5ba TCF1WTLEF1WT) and (50% ly5b TCF1ΔLEF1Δ + 50% ly5a TCF1WTLEF1WT) in Rag1–/– recipient mice at 2 months after intraperitoneal injection (left), Flow cytometry plots (middle) and quantification of percentage of B-1a cells in peritoneal cavity (TCF1WTLEF1WT n = 5; TCF1ΔLEF1Δ n = 5) and spleen (TCF1WTLEF1WT n = 4; TCF1ΔLEF1Δ n = 5) (right). Each symbol represents an individual mouse and bars represent median values. Data are representative of n = 2 experiments (c-e,g-i). Statistical analysis was performed using two-tailed Mann-Whitney t test (c-e,g-i). The exact P values are shown. Each symbol represents an individual mouse and bars represent median values. Data are representative of n = 2 experiments (a-c). Statistical analysis was performed using two-tailed Mann-Whitney t test (a,c), one way ANOVA with Tukey multiple-comparison test (b). The exact P values are shown. (a) Coverage-based diversity accumulation curve for Hill numbers of order (q = 0, 1 and 2), i.e., species richness, Shannon entropy and Simpson index, respectively72 in peritoneal B cell (top) or B-1 cells (bottom). (b) Circular plot of cell number of different clonotypes in TCF1WTLEF1WT and TCF1ΔLEF1Δ peritoneal B-1 cells. (c) Flow cytometry plots and quantification of frequency of PtC+B-1a cells in TCF1WTLEF1WT (n = 12) and TCF1ΔLEF1Δ (n = 8) mice. (d) Total serum IgM and IgG3 were determined by ELISA between TCF1WTLEF1WT (n = 5) and TCF1ΔLEF1Δ (n = 7) mice. (f) Flow cytometry plots and quantification of percentage of IgG3+ from peritoneal B-1a cells in TCF1WTLEF1WT (n = 5) and TCF1ΔLEF1Δ (n = 5) mice. Each symbol represents an individual mouse and bars represent median values. Data from n = 3 experiments in total (c, e), or representative of n = 3 experiments (d-f). Statistical analysis was performed using two-tailed Mann-Whitney U test (c,e-f), one way ANOVA with Tukey multiple-comparison test (d). The exact P values are shown. Superimposed B-1a signature genes from published RNA-seq data which over- or under-represented in peritoneal B-1a versus B-1b cells34. (b) Expression heatmap of B-1a signature genes, with the colour representing normalized RPM for each gene across different samples. (c) Dot plot presentation of Hallmark gene sets according to GSEA for differentially expressed genes in TCF1WTLEF1WT and TCF1ΔLEF1Δ B-1a cells. (d) Cnet plot presentation of enriched Hallmark pathways (P < 0.05, FDR < 0.05) upon GSEA. Size = the number of differentially expressed gene which belongs to the enriched Hallmark pathway. Fold change difference between TCF1WTLEF1WT and TCF1ΔLEF1Δ B-1a cells is shown. (e) Frequency of TCF1 and LEF1 expressed cells in human B-1 clusters23. (f) Dot plot representation of Tcf7 and Lef1 mRNA expression in different subsets of human prenatal tissues. Panels e,f were adapted from ref. (g) Flow cytometry plots and quantification of B-1-enriched cells from PBMCs from 4 SCID patients and 3 HD. (h) ChIP-seq analysis of TCF1 binding in CD4+CD8+ thymocytes42 and ATAC-seq analysis at Cd5, Myc and Ets1 on B-1a cells from Vh12/Vk4 transgenic mice17. Panel h was adapted from refs. The significant hallmark gene sets in MSigDB (adjustment p-value < 0.05) for each contrast based on the hypergeometric testing using cluster profiler package (c). (a) IL-10 secretion measured by cytometric bead Arrays (CBA) from TCF1WTLEF1WT (n = 7) and TCF1ΔLEF1Δ (n = 7) B-1 cells cultured in media with LPS for 3 days. (b) Flow cytometry plots and quantification of percentage of CD138+Lag3+cells in the spleen of TCF1WTLEF1WT (n = 5) or TCF1ΔLEF1Δ (n = 5) mice 3 days after injection with LPS. (c) gMFI quantification of TCF1 and LEF1 in B cells subsets from B-1a (CD19+ B220– CD5+ CD23–), B10 (CD1dhi CD5+), immature transitional-2 (T2-MZP) (B220+ IgMhi CD21hi CD23+), marginal zone B cells (MZB) (CD19+ B220+ CD23– CD21+), Follicular B cells (FO) (CD19+ B220+ CD23+ CD21−) from spleen of C57BL/6J mice (n = 6). (d) Flowchart of EAE immunization. (e) Expression of Gramd2a across the ImmGen Database. The figure was created using ImmGen (https://immgen.org). (h) Identification of GRM2A positive B-1 cells in EAE mouse brain. 4o (i) Flow cytometry plots of plasma cell (PC) (CD138+) and non-plasma cell (CD138−) cells from purified peritoneal cavity TCF1WTLEF1WT B-1 cells cultured with LPS and isotype control or anti-mouse IL-10R for 3 days. (j) Flow cytometry plots of CD138+ cells after magnetically purified splenic TCF1WTLEF1WT or TCF1ΔLEF1Δ B-2 cells were cultured with LPS and anti-mouse IL-10R or isotype control for 3 days. (l) Quantification of Zeb2 expression in TCF1WTLEF1WT and TCF1ΔLEF1Δ B-1a cells by RNA-seq. Each symbol represents an individual mouse, Data are presented as mean valued +/– S.D. Data are representative of n = 2 experiments (b-c,g), peritoneal cavity cells from three mice are pooled together (a,k). Images are representative of n = 3 mice (l). Statistical analysis was performed using two-tailed Mann-Whitney U test (a-b,l) and one way ANOVA (c,g). The exact P values are shown. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/tiny-probes-can-surf-sunlight-to-explore-earths-mesosphere-and-mars/'>These Tiny Disks Could Explore the ‘Ignorosphere' That Planes and Satellites Can't Reach</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These Tiny Disks Will Sail on Sunlight into Earth's Mysterious ‘Ignorosphere' With no fuel or engines, tiny explorers will surf sun-warmed air alone to explore high in the skies of Earth and Mars This artist's impression shows multiple small devices soaring on sunlight at the edges of Earth's atmosphere. Scientists have devised tiny featherweight disks that could float on sunlight in Earth's mesosphere or the thin air of Mars, theoretically even while carrying payloads. Extending about 50 to 85 kilometers above Earth's surface, the mesosphere is too high to be reached by aircraft and weather balloons but too low for access by satellites, making it one of our planet's least-studied regions. The new centimeter-wide disks are made from two thin perforated membranes of ceramic alumina connected by tiny vertical supports. They are kept aloft by a force called photophoresis: the light-induced movement of small particles at very low atmospheric pressures. In lab experiments simulating mesospheric air pressure and illumination, the researchers showed that their devices could float passively, without any power source. Their paper reporting the results was published in Nature on August 13. Thus, gas molecules bouncing off the lower part gained more momentum than those at the top, generating lift similar to how a rocket's jet produces upward thrust. This enhancement boosted the disks' performance to surpass photophoretic flyers previously demonstrated by other groups, which had required illumination several times brighter than that of sunlight. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. He developed what came to be known as a Crookes radiometer, a toylike device that spins its metallic fins when exposed to sunlight. Designing the disks to utilize thermal transpiration and optimizing them to carry the biggest possible payloads required careful computational modeling of mesospheric conditions and photophoretic forces, Schafer says. This “gave us really good insight into what would make an efficient structure” for mesospheric flight, he says. The team calculates that this larger version of the disk could stay aloft at an altitude of 75 km during daytime; in summertime at polar latitudes, it could even achieve constant mesopheric flight, sustained after sundown by the photophoretic force of infrared light emanating from Earth's gradually cooling surface. Ruth Lieberman, a heliophysicist who worked on earlier attempts at photophoretic technology but wasn't involved in this project, calls it a brilliant design. Once you get past the prototype phase and can figure out how to manufacture [at scale], it strikes me as a really potentially fantastic solution for observing the atmosphere at very low cost in a way that gets you very good spatial temporal coverage.” Schafer has co-founded a company based in part on the new research that is developing new iterations of the disks and planning to launch payload-free atmospheric test flights as early as 2026. Creating disks truly capable of pulling their own weight in the mesosphere or beyond is a more formidable task—akin to a five- to 10-year project, Schafer says. “If you wanted to put a payload on board these devices, I think it's certainly doable, but it's going to take a lot of time and work,” he says. Scaling up is going to be a materials manufacturing challenge, he adds, that warrants more research. Payal Dhar (she/they) is a freelance journalist who covers science, technology and society. They write about AI, robotics, biotech, space, online communities, games and any shiny new technology that catches their eye.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/flight/a65664697/the-worst-air-disaster-in-american-history-occurred-in-broad-daylight-why-do-similar-mistakes-keep-happening/'>The Worst Air Disaster in American History Occurred in Broad Daylight. Mistakes Keep Happening.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Not a soul witnessed what we can only imagine were violent fireballs and streaming plumes of black smoke. No one even knew that the canyon's mighty peace had been disturbed—save, of course, the 128 people on board both aircraft, who lived through mere seconds of terror no human should ever know, the last seconds of their lives. And despite the fact that almost every railed ledge in the tourist area afforded unobstructed views, so the folks visiting from Waukegan and Sacramento and Hartford, their new Kodak Brownie cameras hanging from their necks, could see for miles across the massive chasm. Despite all that, two planes collided spectacularly and nobody saw; in the quiet after the crash, it was as if it had never happened. That's how vast the Grand Canyon truly is. In fact, nobody anywhere knew anything was wrong until about an hour later, when both flights failed to check in at their shared radio waypoint, known as Painted Desert. There were no radar blips that suddenly disappeared from an air traffic controller's screen, no frantic bystanders rushing to the scene, no chilling mayday calls that spurred rescue teams into action. Though one such call was sent—a calm, defeated declaration that “Salt Lake, United 718 . . . ah . . . we're going in” from United copilot Robert Harms, transmitted to the Salt Lake City control center—it was garbled and not properly identified until weeks later, during the investigation. The only sign of trouble was a lingering wisp of smoke that a private pilot noticed while flying over the canyon. On that day its walls would have been bathed in pure southwestern sunlight speckled with scrolling cloud shadows—a good day to fly. Within hours, the world would learn of it. It's easy now for us to look back at a 70-year-old calamity and think, That was early days. The midair collision of American Airlines Flight 5342 and a U.S. Army Black Hawk helicopter over the Potomac River in Washington, D.C., killed 67 people. It happened barely three miles from the White House. Air travel in the 1950s was at once regal and primitive. Piston-powered airliners crisscrossed largely empty skies at what today would seem a leisurely pace. Transcontinental flights would often make multiple stops, in rustic Old West outposts like Oklahoma City and bustling heartland metropolises like Chicago. Most airlines, in fact, operated dedicated first-class-only flights on popular routes. These flights had excellent cuisine, cocktail lounges, curtains on the windows, and roomy seats. The thunderous roar of four 3,400-horsepower, 18-cylinder radial engines furiously spinning their 14-foot-diameter Hamilton Standard Hydromatic props still wasn't as brain-meltingly loud as the narrow, screeching jet engines travelers would be subjected to once the Boeing 707 and Douglas DC-8 debuted in 1958 and 1959, respectively. Air travel in the '50s was a new, exhilarating alternate reality. The United and TWA flights that left Los Angeles International Airport on June 30, 1956, were among those special first-class-only flights. After boarding mere feet from each other at adjacent gates, the two aircraft taxied to the runway. TWA 2 took off first, at 9:01 a.m. local time, on its way to Kansas City Downtown Airport with 70 on board. At the controls: Captain Jack Gandy, First Officer James Ritner, and Flight Engineer Forrest Breyfogle. United 718 rolled down the same runway at 9:04 a.m., headed for Chicago's Midway Airport with 58 on board, including Captain Robert Shirley, First Officer Robert Harms, and Flight Engineer Gerard Fiore. Both aircraft had two flight attendants on duty. Neither had flight data recorders or cockpit voice recorders, which were not in widespread use at the time. The airplanes were supposed to take off about 15 minutes apart, but both had minor delays that separated their departures by just three minutes. The airplanes were cleared to travel routes set by their airlines—TWA 2 past Daggett, California, at an altitude of 19,000 feet, and United 718 through Palm Springs, California, at 21,000 feet. Both were operating under visual flight rules (VFR), meaning the crews were obligated to scan for weather challenges and other aircraft visually. However, their staggered timing, different speeds (TWA 2 at 270 knots, United 718 at 288 knots), and 2,000-foot vertical separation should have kept them out of each other's way. Both flight crews, however, opted to fly “off-airways” after getting their initial clearances, meaning they would take more direct paths to their next waypoints than they would have by staying on prescribed airline routes, which are meant to accommodate a wider range of flights. They'd also be in uncontrolled airspace, with no electronic tracking by any air traffic controllers (ATC), and were required to stay in visible airspace so as to avoid other aircraft. (Airline dispatchers often relayed communications for their pilots if the aircraft were out of radio range of controllers farther down the route.) Controllers at Los Angeles Center further checked with the Salt Lake City control center where TWA 2 was headed, and that controller, in Salt Lake, flagged the altitude as being assigned to United 718. A subsequent request for “1,000 on top”—meaning the plane would stay 1,000 feet above the cloud tops—was approved, but TWA 2 still had the responsibility of maintaining visual separation from other aircraft, and it was told of United 718's altitude. TWA 2 then settled in at 21,000 feet, matching United's altitude. Investigations indicated that sharing this information could have prevented the accident, but didn't implicate the controller because he was managing a multitude of flights in the area and was correct to assume the flights would “see and avoid” each other, per ATC procedures. With no active radar tracking above the Grand Canyon, the flight crews were effectively on their own. These included less favorable winds than expected, course deviations around clouds, and the chance that the pilots wanted to give passengers better views of the Grand Canyon below. In his book about the tragedy, We Are Going In, Mike Nelson, whose uncle, Jack Groshans, was killed aboard United 718, explores this prospect of sightseeing diversions in detail. Both flights appeared to have been chasing such prospects, as they had both made distinct, last-minute northward changes to their paths. The consequences of these deviations were horrific. “It was as though their ends were predestined,” Nelson writes, citing their shared delays taking off, the converging paths of their original flight plans, TWA 2's bump to 21,000 feet cruising altitude, and their diversions north to the same square inches of Grand Canyon airspace. It dove like a missile for the ground and impacted Temple Butte, a prominent formation in the eastern canyon, at more than 400 mph. United 718 had slightly more controllability, but the tip of its left wing was mangled, reducing lift. It staggered on for just a mile before crashing into nearby Chuar Butte, copilot Harms fighting for control the whole way in. This was an airline tradition at the time, particularly on those dedicated first-class flights. Nelson considers it unlikely that Shirley could have done any better had he still been at the controls. The airplanes had slammed into separate canyon walls, exploding into flames and scattering debris across craggy terrain accessible only by experienced climbers, and even they had to be brought in via helicopter. The crash debris and victim remains had settled over two wide areas, with much of it falling into deep gullies and crevices. Though recovery crews gathered as much as possible from the site over the following months, many aircraft fragments were left at the remote and largely inaccessible site for years—some to this day. It happened during a dramatic surge in civilian air travel, at the cusp of the coming Jet Age. Piston-powered propeller aircraft were able to stretch the speed and distances of air travel while their own obsolescence lurked on the horizon, as those faster and longer-range jet-powered airliners began to enter service. Much of the industry's momentum could be traced to World War II, a conflict dominated by the development and large-scale deployment of innovative new aircraft, many of which would be quickly adapted from transporting soldiers and bombs to carrying paying passengers wearing heels and drinking martinis. For one thing, there were shortfalls in the federal budget that left the airlines hanging, says aviation historian Robert van der Linden, curator of air transportation at the Smithsonian's National Air and Space Museum. Active, radar-aided control, he says, was available only around major cities, and most air traffic control systems became antiquated and susceptible to human error. The area where the airliners collided was a region where commercial pilots were simply supposed to “see and be seen” by other aircraft. “Pilots were allowed to deviate from their flight paths at the pilot's discretion to avoid weather or, in the case of this accident, fly over scenic attractions,” van der Linden says. This list doesn't include multiple midair collisions and close calls among small private or commuter aircraft. These, along with revelations about antiquated technology causing a recent communications blackout at Newark International Airport, are prompting the public and the aviation industry to wonder if our air traffic control system needs to modernize far more aggressively. The FAA recently reported a shortage of 3,000 air traffic controllers. A February headline from PBS News put it this way: “Aviation disasters and close calls have people worried about flying.” As search crews—including a team of five Swiss mountain climbers—were airlifted into the canyon to begin recovering evidence and victim remains, news crews descended on Grand Canyon Village on the South Rim and in nearby Flagstaff, Arizona, where the investigation would be headquartered. Though flying was still expensive and a relative luxury, airlines were already lofting thousands of flights per day in the United States alone. Crashes and midair collisions were actually somewhat common amid this surge in flying—though rarely at cruising altitude—as safety steps struggled to keep up with progress. Led by Parshall, the investigative work focused on radio communications, common practices of aircraft while flying off-airways, and weather—the likely culprits for any aviation accident. But he also explored critical nuances as questions emerged not only about what led the airplanes to the same point in space and time, but also about why they couldn't be stopped. Why didn't they see each other? At the time, cockpit windows in pressurized aircraft were notoriously small, and angles of visibility might have inhibited detection of other aircraft. They also noted a possible illusion scenario: Pilots sometimes can see objects ahead of them on a potential collision course, but the objects register no relative motion from the observing aircraft until they suddenly converge in an instant. (This is similar to the feeling of being a driver in a car stopped at a traffic light and sensing an adjacent vehicle nudging forward, but thinking it's your own vehicle drifting backward—or vice versa.) Investigators also looked into that question of whether crews were chasing a better view of the Grand Canyon for their premier customers, who were paying what would today cost up to $3,000 for first-class transcontinental flights. The former was a more precise system used for military operations, while the latter was more affordable and readily adaptable globally. It took time to make such determinations, and there wasn't a single authority forcing the decision. “Airlines were delaying the purchase of navigational equipment as there was uncertainty regarding what standard would be implemented,” an FAA summary reported in hindsight. Shortly after the collision, in August 1956, the deadlock was broken and the details of a compromise agreed upon.” The solution, dubbed VORTAC, incorporated key advantages from both systems in distance measuring and directional guidance, and it would provide a critical melding of civilian and military navigation systems. Though midair collisions were indeed more frequent at the time, they typically occurred at low altitude near airports, not in high, wide-open skies. Nelson places this disaster in the same category as another familiar, and perhaps equally unlikely, transportation tragedy. “The Titanic was confidently held to be unsinkable, and then, on its maiden voyage, it sank. No accident like it had ever happened before,” he writes. That conversation is ongoing, as is the broader one about upgrading current air traffic control technologies once again. A digitally based upgrade, with modern high-speed data communication systems and eventually automated tracking and collision-avoidance systems long in planning, may now arrive sooner than expected: In May, Transportation Secretary Sean P. Duffy announced an effort to radically overhaul the nation's air traffic control system, including recruiting more air traffic controllers and upgrading networks. It became a National Historic Landmark in 2014, the first such designation to commemorate something that happened in the air. At the ceremony, attended by Nelson, victim relatives, and others affected by the disaster, park officials unveiled a plaque. It reads: “This tragic accident site represents a watershed moment in the modernization of America's airways leading to the establishment of the Federal Aviation Administration and national standards for aviation safety. The Potomac River collision in January 2025 was the first major incident on U.S. soil in 16 years, but to a lot of people, that 16-year stretch isn't good enough. In 1956, mass air travel was not yet fully realized—the accident then, while tragic, somehow feels explainable. But today millions of people fly every day, every hour, and helicopters aren't supposed to crash into planes. The investigation of that crash will take a while. Yes, we have vastly better technology now. But inexplicably, we somehow don't have enough air traffic controllers, not even close, nor do we have the required technology to do this job in their absence. As in the terrible quiet moments that followed the Grand Canyon crash that no one saw, it's as if it never happened, so little have we learned from it. And so the questions haven't changed: Eric Adams is a writer and photographer who focuses on technology, transportation, science, travel, and other subjects for a wide range of outlets, including Wired, The Drive, Gear Patrol, Men's Health, Popular Science, Forbes, and others. A Pilot Is Pretty Sure He Found Amelia Earhart World's Largest Passenger Aircraft Will Fly Again Watch These 7 Airplane Wings Pushed to the Brink and Beyond The Worst Air Disaster in American History The 30 Most Important Planes of All Time</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65821818/breath-hold-record/'>A Freediver Held His Breath For Almost Half an Hour—and Obliterated a World Record</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The attempt was only possible thanks to some pre-gaming with pure oxygen, but still eye-popping. On March 27, 2021, Šobat held his breath underwater for an astounding 24 minutes and 37 seconds—a feat that was only possible thanks to minutes of huffing pure oxygen before making the attempt. Now, four years, fellow countryman and freediver Vitomir Maričić exceeded this world record—and it wasn't even close. On June 14, 2025 in Opatija, Croatia, Maričić performed an oxygen-assisted breath hold that shattered Šobat's by nearly a full five minutes, clocking in at 29 min and 3 seconds. He says he took on this record as both a personal challenge and a way to raise awareness for ocean conservation. “After the 20-minute mark, everything became easier, at least mentally,” Maričić told Divernet. Denying your body life-supporting oxygen isn't something to be taken lightly. When someone holds their breath, CO2 rises and drives an increased desire in the brain to breathe. Eventually, the diaphragm will begin contracting involuntarily, which is typically when untrained apneists will break down and start gulping down air. However, Maričić pre-gamed his world-record-breaking attempt by inhaling pure oxygen for 10 minutes before submersion. This offsets that initial oxygen-carbon dioxide balance, allowing freedivers to hold their breath for significantly longer than is possible for our unaided biology. That's why the world record for unaided breath holding is a much shorter (but still very impressive) 11 minutes and 35 seconds. However, humans have zero hope of ever snatching any world record from marine mammals entirely—the Cuvier's beaked whale can stay submerged underwater for more than three hours. Why One Side of Earth Is Rapidly Getting Colder Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65812259/engagement-diamond-find/'>A Woman Digging for the Perfect Engagement Stone Found a 2 Carat Diamond—Right Under Her Shoe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>She spent weeks searching, and plans to give the stone a very special place in her life. When Micherre Fox made the trek from New York City to the Crater of Diamonds State Park in Pike County, Arkansas, she was hoping to find a rock worthy of her engagement ring. Staff at the park's Diamond Discovery entry confirmed that she had indeed come upon not just a diamond, but the third largest found in park this year. 366 diamonds have been registered at the park since January, and of those, only 11 that surfaced from its literal crater of diamonds were over one carat in size. This is why some lucky parkgoers, like Fox, may not even have to dig for a diamond. 75,000 diamonds have been found on park grounds since 1906 (long before it was officially a state park), the largest of which is now owned by the Smithsonian and on display at the National Museum of Natural History. Diamonds don't normally materialize at the feet of tourists. Unlike most rocks and minerals, which form relatively close to the surface, they are born deep in the mantle and tend to be anywhere from a billion to 3.5 billion years old (with the oldest being almost as ancient as Earth and older than many stars). When gobs of magma erupt from volcanoes, they take diamonds and other minerals with them. It is common for diamonds to be found embedded in magma that has hardened into a substance known as kimberlite. Known as much for their toughness as they are for their beauty, diamonds make it through being carried by magma rivers and belched out. “Having never seen an actual diamond in my hands, I didn't know for sure, but it was the most ‘diamond-y diamond' I had seen,” she said in a recent park press release. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Why One Side of Earth Is Rapidly Getting Colder Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65792312/ancient-teeth-fossils/'>Scientists Found 13 Ancient Teeth That May Rewrite Human Evolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. A new discovery of fossilized teeth in an Ethiopian field has researchers theorizing that they came from a new species of Australopithecus. “This new research shows that the image many of us have in our minds of an ape to a Neanderthal to a modern human is not correct—evolution doesn't work like that,” Kaye Reed, Arizona State University paleoecologist, said in a statement. “Here we have two hominin species that are together.” Researchers discovered 13 of the Australopithecus teeth in the field that also gave us a Homo jaw, which has been dated to 2.8 million years ago. The team has been able to date these fossils by analyzing signs of volcanic and tectonic activity, including the presence of feldspar crystal within ash. “We can date the eruptions that were happening on the landscape when they're deposited,” Christopher Campisano, an ASU geologist, said in a statement. “And we know that these fossils are interbed between those eruptions, so we can date units above and below the fossils. The team's next step is examining tooth enamel to understand what members of this species was eating. They can then compare their findings about this Australopithecus specimen to what is already known about early Homo species, hoping to determine just how closely linked the two were. More fossils will help us tell the story of what happened to our ancestors a long time ago.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Why One Side of Earth Is Rapidly Getting Colder Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250820000754.htm'>Genetic evidence confirms early puberty accelerates aging and disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 09:23:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Reproductive timing matters when it comes to aging and age-related disease. "Even though women are routinely asked about their menstrual and childbirth history when they receive medical care, this information has rarely factored into the care they receive outside of OB/GYN," he says. "These risk factors, whether positive or negative, clearly have significant influence on a variety of age-related diseases and should be considered in the larger context of overall health." "We identified 126 genetic markers that mediate the effects of early puberty and childbirth on aging," said postdoctoral fellow Yifan Xiang, MD, who led the research. "Many of these markers are involved in well-known longevity pathways, such as IGF-1, growth hormone, AMPK and mTOR signaling, key regulators of metabolism and aging." "Our study provides some of the strongest human evidence for this theory," Kapahi says. "We show that genetic factors favoring early reproduction come with the significant cost later in life including accelerated aging and disease. The role of BMI in aging and disease risk Kapahi says the study highlights the role of Body Mass Index (BMI) as a critical mediator of this process, finding that early reproductive events contribute to a higher BMI, which in turn increases the risk of metabolic disease. Kapahi says understanding the long-term impact of reproductive timing allows for the development of personalized healthcare strategies that could help mitigate the risks associated with early puberty and early childbirth, adding that lifestyle modifications, metabolic screenings and tailored dietary recommendations could improve long-term health in women. No specific causes for the phenomena have been identified, but research suggests obesity may play a role. While updated research guidelines call for the use of both sexes in preclinical research in mice, Kapahi says this current study still challenges traditional experimental design, noting that most disease models use virgin female mice, which may not accurately represent real-world aging patterns. "If evolution has shaped us to prioritize early reproduction at the cost of aging, how can we leverage this knowledge to extend healthspan in modern society? "While we cannot change our genetic inheritance, understanding these genetic tradeoffs empowers us to make informed choices about health, lifestyle and medical care." The study also identifies several genetic pathways that can be manipulated to optimize health for mothers as well as her offspring Kapahi says. Other Buck researchers involved in the study include: Vineeta Tanwar, Parminder Singh, and Lizellen La Follette. What a 30-Year Study Just Found About Your French Fries and Diabetes Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250820000752.htm'>Scientists uncover sugar pathway driving aggressive childhood cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 09:23:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new study delving into the genetic drivers of a rare and aggressive childhood cancer called Malignant Peripheral Nerve Sheath Tumor (MPNST) has revealed metabolic frailties in the cancer cells that might be exploited to improved treatments for patients. These tumors grow quickly, spread easily, and don't respond well to current treatments. Metastasis is the leading cause of death in MPNST and there currently are no targeted treatments for metastatic disease. The study, led by University of Iowa researchers Eric Taylor, PhD, professor of molecular physiology and biophysics, and Rebecca Dodd, PhD, associate professor of internal medicine, identifies a specific metabolic pathway that is critical for MPNST cells' survival and growth, which could potentially be targeted with future therapies. To better understand these tumors, the UI researchers used gene editing to create new research models that closely match the cancer-driving mutations found in patients. Using these models, they then applied state-of-the-art genomic and metabolomic tools to map the metabolic pathways that fuel tumor growth in MPNST. The study, published in Science Advances, found that these cancers rely on a key metabolic pathway to help them survive oxidative stress and drive tumor growth. This pathway, known as the Pentose Phosphate Pathway (PPP), metabolizes sugar to produce a critical antioxidant molecule that the cancer cells require to combat oxidative stress. "It opens the door to treatment strategies that haven't been explored before and could lead to more effective treatments and better outcomes for patients who urgently need new options." The lead author was UI graduate student Gavin McGivney, PhD, from Bayard in Guthrie County, Iowa, who was co-mentored by Dodd and Taylor. McGivney graduated from the UI Cancer Biology graduate program in 2024, and is now a postdoctoral scholar at University of Chicago. Surprising Discovery Links Brain's Immune Cells to Alzheimer's Disease What a 30-Year Study Just Found About Your French Fries and Diabetes Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250818102129.htm'>Astronomers discover a hidden engine inside space's “Eye of Sauron”</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-20 03:49:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A look into the throat of an active galaxy reveals a ring-shaped magnetic field that may explain extreme gamma radiation and neutrinos. Located billions of light-years away, the blazar PKS 1424+240 had long baffled astronomers. It stood out as the brightest known neutrino-emitting blazar in the sky -- as identified by the IceCube Neutrino Observatory -- and was also glowing in very high-energy gamma rays ob-served by ground-based Cherenkov telescopes. Yet, oddly, its radio jet appeared to move sluggishly, contradicting expectations that only the fastest jets can power such intense high-energy emissions. Now, thanks to 15 years of ultra-precise radio observations from the Very Long Baseline Ar-ray (VLBA), researchers have stitched together a deep image of this jet at unparalleled resolution. This structure plays a key role in launching and collimating the plasma flow, and may be essential for accelerating particles to extreme energies. "When we started MOJAVE, the idea of one day directly connecting distant black hole jets to cosmic neutrinos felt like science fiction. Today, our observations are making it real," says Anton Zensus, Director at MPIfR and co-founder of the program. This result strengthens the link between relativistic jets, high-energy neutrinos, and the role of magnetic fields in shaping cosmic accelerators -- marking a milestone in multimessenger astronomy. This alignment makes blazars appear bright across the electromagnetic spectrum and allows scientists to study extreme physical processes -- including the acceleration of particles to energies far beyond those achieved in human-made accelerators. The VLBA (Very Long Baseline Array) is an array of ten antennas, at locations across the continental United States and in Hawaii and St Croix, which operates in the very long baseline interferometry (VLBI) mode. Spacings between the antennas vary up to approximately ten thousand kilometers, providing angular resolution on the sky as fine as 50 micro-arcseconds. MOJAVE (Monitoring Of Jets in Active galactic nuclei with VLBA Experiments) is a long-term program to monitor radio brightness and polarization variations in jets associated with active galaxies visible in the northern sky. We are using these data to better understand the complex evolution and magnetic field structures of jets on light-year scales, close to where they originate in the active nucleus, and how this activity is correlated with a high energy electromagnetic and neutrino emission. MuSES, which stands for Multi-messenger Studies of Energetic Sources, is a pioneering initiative in astrophysics. MuSES aims to address these fundamental questions by exploiting recent advances in multi-messenger astronomy. Views and opinions expressed are however those of the author(s) only and do not necessarily reflect those of the European Union or European Research Council Executive Agency (ERCEA). Materials provided by Max Planck Institute for Radio Astronomy. Note: Content may be edited for style and length. Astronomers Capture the “Eye of Sauron” Beaming at Earth Surprising Discovery Links Brain's Immune Cells to Alzheimer's Disease What a 30-Year Study Just Found About Your French Fries and Diabetes Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            