
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69937639/egyptian-monastery/'>Archaeologists Uncovered a Lost Monastery Buried In the Egyptian Desert</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The monastery complex discovery comprised the remains of a church, residential halls, and communal dining spaces, in addition to numerous artifacts. When archaeologists stripped away the desert sand in Upper Egypt's Sohag province, they uncovered a Byzantine-era monastery, complete with a church, residential halls, communal dining spaces, and artifacts up to 1,400 years old. The discovery wasn't a one-off structure, but an entire integrated residential complex for monastic living, complete with a variety of mudbrick structures, according to a translated statement from Egypt's Ministry of Tourism and Antiquities, part of the government's Supreme Council of Antiquities. Buildings with rectangular halls also featured a semicircular vaulted apse on the eastern end and small, brick-vaulted alcoves throughout, likely used as individual worship retreats for monks. Badii said the structures house trusses and wall beams, while including layers of tiling on the walls and floors. These lost structures may have been the monk's communal dining areas. The excavation revealed ruins of buildings featuring red brick and limestone plaster basins covered with a layer of red tiles for waterproofing, which were likely used to store water or for an industrial activity that took place on the site. Amphorae found were once used for storage, some inscribed with Coptic letters, numbers, or names that could have delineated the type of writings stored inside. Sharif Fathi, minister of tourism and antiquities, said such a find can deepen our understanding of Egyptian history across eras while helping develop a tourist attraction for those interested in the history of civilization and religions. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Find Missing Piece of Ramesses II Statue This Is How Humans Can Recover Lost Vision The Maya Kingdom Collapsed Due to Burning Events Experts Found an Odd Carving of a Mythical Figure A Colossal Squid Might Have Shown Its Secret Face</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260109080211.htm'>Common food preservatives linked to higher risk of type 2 diabetes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 13:17:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>People who consume higher amounts of food preservatives may face a greater risk of developing type 2 diabetes, according to a large new study. The research was conducted by scientists from Inserm, INRAE, Sorbonne Paris Nord University, Paris Cité University and Cnam as part of the Nutritional Epidemiology Research Team (CRESS-EREN). More than 700,000 of those products contained at least one preservative. In their analysis, Inserm researchers divided preservative additives into two main groups. The first group includes non-antioxidant preservatives, which slow spoilage by limiting microbial growth or slowing chemical reactions in food. The second group consists of antioxidant additives, which help preserve foods by reducing or controlling exposure to oxygen in packaging. Earlier experimental research has raised concerns that some preservatives may harm cells or DNA and interfere with normal metabolic processes. However, direct evidence linking preservative intake to type 2 diabetes in large human populations has been limited until now. To better understand this potential connection, a research team led by Mathilde Touvier, Inserm Research Director, examined long-term exposure to food preservatives and the incidence of type 2 diabetes using detailed data from the NutriNet-Santé study. Tracking Diet and Health Over More Than a Decade Participants regularly provided information about their medical history, socio-demographic background, physical activity, lifestyle habits, and overall health. They also submitted detailed food records covering multiple 24-hour periods. These records included the names and brands of industrial food products they consumed. Researchers cross-referenced this information with several databases (Open Food Facts, Oqali, EFSA) and combined it with measurements of additives in foods and beverages. Across all food records, researchers identified a total of 58 preservative-related additives. This included 33 preservatives in the strict sense and 27 antioxidant additives. The analysis accounted for many factors that could influence diabetes risk, including age, sex, education, smoking, alcohol use, and overall diet quality (calories, sugar, salt, saturated fats, fibre, etc. Compared with people who consumed the lowest levels of preservatives, those with higher intake showed a markedly increased risk of developing type 2 diabetes. "More broadly, these new data add to others in favor of a reassessment of the regulations governing the general use of food additives by the food industry in order to improve consumer protection," adds Anaïs Hasenböhler, a doctoral student at EREN who conducted these studies. Note: Content may be edited for style and length. Microscopic Robots That Swim Think and Act on Their Own Diabetes Doesn't Just Raise Sugar It Can Change Your Blood A Once-in-a-Generation Discovery Could Transform the Future of Dairy Farming Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260108231333.htm'>Scientists Found a Way to Supercharge the Immune System Against Cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 13:01:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The approach aims to help immune cells recognize and attack tumors more effectively. These antibodies work by grabbing and clustering several immune cell receptors at once, which increases the strength of the signal that tells a T cell to attack cancer. This receptor needs a matching key (ligand) to activate T cells. During infections, the body naturally produces this ligand, but cancer cells do not. Without it, T cells receive only a weak activation signal and struggle to attack tumors effectively. Antibodies can sometimes act like a master key, helping trigger immune responses. However, most antibodies used in medicine today have a Y shaped structure with two arms, which limits them to binding just two receptors at a time. Although antibody based treatments have transformed cancer care, they do not work for every patient. In some cancers, T cells still fail to become fully active because they are missing the combination of signals needed to mount a strong attack. The antibodies developed in this study were built with four binding arms instead of two. They also recruit a second immune cell, which forces all the CD27 receptors being held to gather together. However, the natural antibody format was not powerful enough, so we had to create a more effective version." Laboratory tests using mice and human immune cells showed that the new antibodies were much better at activating CD8+ T cells than standard Y shaped antibodies. The enhanced activation led to a stronger anti tumor response. By making CD27 easier to target with therapy, the research offers a roadmap for developing new immunotherapy treatments that better harness the immune system's natural power. Diabetes Doesn't Just Raise Sugar It Can Change Your Blood A Once-in-a-Generation Discovery Could Transform the Future of Dairy Farming Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02451-4'>De novo and inherited dominant variants in U4 and U6 snRNA genes cause retinitis pigmentosa</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 11:11:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Small nuclear RNAs (snRNAs) combine with specific proteins to generate small nuclear ribonucleoproteins (snRNPs), the building blocks of the spliceosome. Here we show that heterozygous inherited and de novo variants in RNU4-2 and in four RNU6 paralogs (RNU6-1, RNU6-2, RNU6-8 and RNU6-9), which encode U6, recur in individuals with nonsyndromic retinitis pigmentosa (RP), a genetic disorder causing progressive blindness. These variants cluster within the three-way junction of the U4/U6 duplex, a site that interacts with tri-snRNP splicing factors also known to cause RP (PRPF3, PRPF8, PRPF31), and seem to affect snRNP biogenesis. Based on our cohort, deleterious variants in RNU4-2 and RNU6 paralogs may explain up to ~1.4% of otherwise undiagnosed RP cases. This study highlights the contribution of noncoding RNA genes to Mendelian disease and reveals pleiotropy in RNU4-2, where distinct variants underlie neurodevelopmental disorder and retinal degeneration. While approximately2 million individuals worldwide are affected by retinitis pigmentosa (RP), it is estimated that 30% to 50% remain without a conclusive genetic diagnosis, even after exome or genome sequencing is performed1,2,3,4. This reflects high genetic heterogeneity, limited testing access and as-yet-unidentified disease genes, which in general carry pathogenic variants that are exceedingly rare in the control population5,6,7. Noncoding RNAs are essential to many cellular processes, including pre-messenger RNA (pre-mRNA) splicing, which is ensured by the spliceosome, a macromolecular complex that in its major form is composed of five small nuclear RNAs (snRNAs), U1, U2, U4, U5 and U6, and ~300 proteins8. Each snRNA associates with a specific set of proteins to form a small nuclear ribonucleoprotein (snRNP), the functional unit of the spliceosome. Variants in RNU4-2, one of the two paralogs encoding U4, have been linked to a common neurodevelopmental disorder (NDD) known as ReNU syndrome (OMIM: 620851). These variants account for up to 0.4% of all NDD cases and lead to systematic misrecognition of donor splice sites by the spliceosome9,10,11. Likewise, RNU2-2 and RNU5B-1 have been recently associated with NDDs11,12,13. Several spliceosomal proteins are also known to be involved in a wide range of hereditary diseases, including RP, as first noted by McKie and colleagues14. Specifically, of the ~100 genes that are currently associated with nonsyndromic RP5, the tri-snRNP splicing factor genes PRPF3, PRPF4, PRPF8, PRPF31 and SNRNP200 underlie the autosomal dominant form of the condition (adRP), with variants in PRPF31 accounting for 10–20% of all adRP cases3,15. Here, we identify both inherited and de novo variants in RNU4-2 and four paralogs of RNU6, encoding the U6 snRNA, as the molecular cause of adRP in 153 individuals across 67 families. Furthermore, we show that such variants increase the association of U4 and U6 snRNAs with the splicing factors SART3 and PRPF31, suggesting impaired snRNP biogenesis. 1), in which seven of eight siblings (II:1–II:7) and their father (I:1) displayed classical RP features (Supplementary Fig. Genome sequencing was negative for pathogenic variants in known retinal disease-associated genes, but selective DNA variant filtering and shared haplotype analysis revealed a total of 55 variants that were absent from gnomAD v.4.17,18 and co-segregated with RP. Of these, none was predicted to impact splicing (SpliceAI > 0.2)19 and only one was evolutionarily conserved (GERP = 4.03 and phyloP-vertebrate = 3.18)20,21, a single-nucleotide insertion in the gene RNU4-2 (NR_003137.2:n.18_19insA; Fig. This DNA change was present in one individual from the All of Us database22. a, Two-dimensional structure of the U4/U6 duplex, with recurrent variants identified in RP cases (in red for U4 and in green for U6), all clustering within the three-way junction. Nucleotides affected by variants previously observed in NDD cases are underlined. d, Same as b for all five RNU6 paralogs combined, with recurrent causative variants displayed in green. Significant P values for variants enriched in RP cases versus controls from gnomAD are indicated (two-sided Fisher's test with Bonferroni correction). To find additional families, we first screened by Sanger sequencing a cohort of 1,891 individuals from the European Retinal Disease Consortium (www.erdc.info) with RP or Leber congenital amaurosis who remained undiagnosed after a large high-throughput screening using single molecule Molecular Inversion Probes23. This analysis led to the identification of three additional families comprising 15 affected individuals segregating the same pathogenic variant (Supplementary Fig. The n.18_19insA allele was significantly enriched in the RP cohort compared with both the gnomAD and the All of Us databases (analyzed control genomes: 76,215 and 414,000, respectively; Bonferroni-corrected P values = 2.6 × 10−3 and 6.9 × 10−5, respectively, by two-sided Fisher's test; Supplementary Table 3). Additional screening of the RNU4-2 sequence in the same cohort led to the identification of 28 other variants, one of which (n.56T>C) recurred in eight individuals from four families (Fig. Additional screening of 2,830 RP cases without previous genetic diagnosis from our respective institutions' cohorts, the UK National Genomic Research Library (hosting data from the Genomics England 100,000 Genomes Project24 and from the NHS Genomic Medicine Service) uncovered an additional patient harboring n.18_19insA (for whom the variant was de novo) and six families (nine affected individuals) carrying the n.56T>C variant (Supplementary Fig. Of note, incomplete penetrance was observed for nine obligate carriers, without visual symptoms (Supplementary Fig. One carrier of n.56T>C was asymptomatic, with subnormal electroretinogram, diffuse atrophic changes in the periphery and attenuated vessels. Another individual with the same variant showed no clinical signs of disease upon examination, and seven (among whom four were deceased) were not clinically evaluated to determine their disease status. Our combined screening of RNU4-2 also revealed 24 other unique rare DNA changes in 27 families, which were classified as variants of uncertain significance (VUS), as well as three benign changes (Supplementary Table 3). Because U4 snRNA can also be transcribed from its paralog RNU4-1, which differs from RNU4-2 at only four positions (n.37, n.88, n.99 and n.113; Supplementary Table 4), we next examined its sequence in our initial cohort and identified 63 variants, none of which were significantly enriched in cases compared with controls; also, these changes did not include variants at sites corresponding to n.18_19 and n.56 of RNU4-2 (Fig. Notably, RNU4-1 appears to be more tolerant to variation compared with RNU4-2, as evidenced by the numerous and frequent variants that are present in genomes from the general population (cumulative allele frequency of 20.4% in RNU4-1 versus 1.2% in RNU4-2; gnomAD v.4.1) (Fig. We therefore hypothesized that variants in U6 could also underlie adRP and extended our analysis to all five identical paralogous genes producing the U6 snRNA, scattered across the genome (RNU6-1, RNU6-2, RNU6-7, RNU6-8 and RNU6-9; Supplementary Table 4). A screening of these genes by Sanger sequencing in our initial cohort of 1,891 RP families revealed 94 DNA changes in total. The n.55_56insG insertion recurred at the exact relative position in RNU6-2, RNU6-8 and RNU6-9 (four families per gene, 34 cases in total; Supplementary Fig. Since this variant was identical in three U6 genes, we reasoned that the specific DNA change, rather than any particular paralog, was relevant to the etiology of the disease. We therefore repeated our analysis by collapsing the five RNU6 genes and detected 66 unique variants. 1 and Supplementary Table 2) and found to be significantly enriched in cases versus controls (Bonferroni-corrected P value = 1.8 × 10−3 (gnomAD, a single RNU6-2 positive individual of unknown status) and 2.1 × 10−5 (All of Us, no positive individuals); Supplementary Table 3). We then extended our analysis to the same international cohorts of patients that were previously analyzed (n = 2,830) and identified 74 additional cases from 38 families who were positive for either n.55_56insG or n.56_57insG (Supplementary Table 2). In total, these two variants were detected in 112 affected individuals from 52 families, involving all RNU6 paralogs except RNU6-7. The n.55_56insG insertion was present in most cases (102 individuals from 47 families), occurring in four of the five RNU6 paralogs: RNU6-1, RNU6-2, RNU6-8 and RNU6-9, while n.56_57insG was present in ten individuals from five families, in RNU6-1, RNU6-2 and RNU6-9 (Supplementary Tables 1 and 2 and Supplementary Figs. Notably, n.55_56insG was confirmed to be a de novo event in eight individuals, clinically identified as sporadic cases. In 14 additional pedigrees, it was also observed in individuals born to unaffected parents, for which de novo inheritance was suspected but could not be confirmed, due to the lack of parental DNA. In contrast, no de novo events could be detected for n.56_57insG, which was identified exclusively in families with adRP (Supplementary Fig. Similar to the screening of the RNU4 paralogs, our analysis of RNU6 paralogs revealed 66 VUSs and 23 benign variants, validated by Sanger sequencing (Supplementary Table 3). The overall phenotype across all cases was consistent with classical RP, based on clinical examination and electrophysiological testing, with symptomatic onset predominantly in adolescence (Supplementary Table 5). In addition, other concurrent ocular disease features were noted across individuals in the cohort: cystoid macular edema (55.9%), non-age-related lens opacities (23.6%) and various vitreomacular complications (30.6%) (Supplementary Table 5). Based on our data from these 4,722 RP cases, mostly of European descent and lacking a genetic diagnosis, we estimate that RNU4- and RNU6-associated RP could be responsible for ~1.4% of all molecularly undiagnosed individuals with this disease. Furthermore, considering that approximately 30% of RP diagnoses correspond to adRP25,26 and that our positive families include 24 isolated individuals, we can further infer that these variants may account for approximately 3.0% of undiagnosed adRP families. All RP variants are predicted to map in spatial proximity with each other, within the three-way junction delimited by stem-I and stem-II of the U4/U6 duplex and the 5′ stem-loop of U4 (Figs. In particular, they are located in a different region compared with those underlying NDD (Fig. In silico two-dimensional modeling of RNA secondary structure predicted as well that the RNU4-2 variant n.18_19insA inserts a nucleotide between stem-II and the U4 5′ stem-loop (Supplementary Fig. In addition, n.18_19insA slightly modifies the orientation of the 5′ stem-loop relative to stem-I and stem-II (Supplementary Fig. Interestingly, we observed that a benign insertion at the same position, n.55_56insT, was present in gnomAD v.4.1 in all five RNU6 paralogs with a cumulative frequency of 0.12% (n = 181) (Supplementary Fig. While these models provide a coherent structural rationale for the observed clustering, the precise effects of the variants on U4/U6 architecture remain to be experimentally verified. Analysis of cryo-electron microscopy data (PDB 6QW6)27 confirmed that all RP variants identified reside in a region critical for binding of the U4/U6 duplex to the splicing factors PRPF31, PRPF3 and PRPF8, all previously associated with adRP16,17 (Fig. Specifically, this region first engages PRPF31 or the PRPF3/PRPF4 complex, initiating the assembly interface, and is subsequently stabilized in its native orientation upon the coordinated binding of additional tri-snRNP components, including PRPF6 and PRPF828. 2c,d), via hydrogen bonds with eight and three residues of these proteins, respectively. Notably, by querying the ClinVar database29, we detected a missense variant affecting one of these residues, p.(Arg449Gly) of PRPF3, identified in a three-generation family with seven affected individuals having clinical features similar to those observed in most cases from our study30. b, Same as in a with interacting PRPF proteins. d, Same as c but for PRPF3. Since the human genome contains several RNU4 and RNU6 pseudogenes31, we investigated whether any of these might be incorrectly annotated and could instead produce functional RNA, potentially contributing to the disease. In addition, we sought to understand why the various U4 and U6 paralogs appear to be differentially mutated, with RNU4-1 and RNU6-7 displaying none of the recurrent pathogenic variants. We used RNA sequencing (RNA-seq) data from human neurosensory retina (NSR), retinal pigment epithelium (RPE) and choroid that were enriched for small RNAs, applying stringent and paralog-aware bioinformatics analyses designed to mitigate the complexities associated with reads aligning against multiple paralogs and/or pseudogenes (Methods). RNU4-2 was more highly expressed than RNU4-1 in all tissues (average ratio: 1.63; Fig. Conversely, individual expression of RNU6 genes and pseudogenes in the retina could not be reliably quantified by RNA-seq, since their sequences are identical, except for the last nucleotide. Therefore, we compared the total expression of RNU4 and RNU6, regardless of their respective paralogs and pseudogenes. RNU6 expression was on average 3.39× higher across the three tissues, compared with RNU4 (Fig. Of note, NSR and RPE had higher expression of RNU4 (2.51×) and RNU6 (6.09×) with respect to the choroid, an ocular tissue not directly involved in vision, used as a control (Fig. This observation is in agreement with previous data showing that snRNA expression in the retina is approximately sixfold higher compared with muscle, testis, heart and brain32, indicating a high demand for snRNAs in these two retinal layers. In addition, we analyzed the Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq) and H3K27ac chromatin immunoprecipitation followed by sequencing (ChIP–seq) data from retinal tissues33 in genomic regions spanning all RNU4 and RNU6 sequences. These data, combined, indicate potential active transcription at promoter regions. Our analysis showed clear transcription marks in all paralogous RNU4 and RNU6 genes in the retina (Fig. Conversely, these signatures were absent from the 105 U4 pseudogenes and the 1,312 U6 pseudogenes, except for RNU4-8P, which displayed strong signals, but probably by virtue of its close proximity to the ACTR1B promoter. Of note, RNU6-92P and RNU6-656P had high ATAC-seq signals but very low H3K27ac signals at their respective promoters (Fig. We performed the same analysis for other snRNA genes present in the human genome, which revealed a similar trend: all RNU genes, with the exception of RNU5F-1, had marks of active transcription and only a few among the thousands of RNU pseudogenes displayed signals compatible with potential expression, therefore representing plausible candidate genes for retinal disease (Supplementary Fig. Interestingly, the same type of analysis, based on conservation and expression data from GTEx, was recently performed by others, showing similar results34. For RNU6-7, both ATAC-seq and H3K27ac signals were within the same range as those observed for other RNU6 genes (Fig. 3d), and, therefore, the absence of pathogenic variants could not be explained by a potential differential expression. We thus analyzed the genetic landscape of variations in healthy individuals in all five U6 paralogs and observed that RNU6-7 displayed a lower number of variants, compared with the others (Supplementary Fig. These seemingly contradictory observations merit further investigations in future studies. We performed transcriptome analysis following the collection of RNA from circulating leukocytes in nine affected individuals carrying variants in RNU4-2, RNU6-1 and RNU6-9 (three individuals per gene), as well as from 14 healthy controls (Supplementary Table 6). To avoid systematic errors linked to the use of different collection kits35 across our cohort (Methods), we performed independent case–control tests for samples collected with PAXgene kits (three RNU4-2 cases, six controls) or Tempus kits (six RNU6 cases, eight controls). We identified 27 and eight differentially expressed genes in the two datasets, respectively, with no gene overlap (fold-change > 2 and false discovery rate (FDR) P value < 0.05; Supplementary Table 7), indicating no major differences in global gene expression in leukocytes in cases versus controls. We then further explored the data by investigating potential bias in pre-mRNA splicing, as performed in ref. This analysis led to the identification of 107 upregulated and 67 downregulated 5′ splice sites in the PAXgene set, and 37 upregulated and 13 downregulated 5′ splice sites in the Tempus set, with only two sites in common between the datasets (in the genes CLEC2D and SNHG29; Supplementary Table 8). At these two sites the expression differences, although statistically significant, were below 10%. We then examined the DNA sequences of the 224 differentially expressed splice sites, focusing on the occurrence of the ‘AG' dinucleotide at positions −2/−1, which was previously reported to be enriched in sites with increased splicing in RNU4-2 in patients with NDD11. No differences in ‘AG' frequency were observed between sites with increased versus decreased usage in either the PAXgene or Tempus samples (two-tailed Fisher's test, P = 0.63 and P = 0.52, respectively). We also compared nucleotide frequencies at each position between upregulated and downregulated splice sites, separately for the two groups, and found no significant differences (two-tailed Fisher's test with FDR correction, P < 0.05). A similar analysis of overlapping dinucleotides across the region flanking the 5′ splice site (for example, positions −4/−3, −3/−2, up to +7/+8) revealed no significant differences. Since PRPF variants associated with RP affect primarily spliceosomal assembly32, we investigated whether the same phenomenon could be driven by the variants detected in this work. For comparison, we included in our assays the U4 n.64_65insT variant, which causes NDD, and observed no significant alteration in the association with any of the proteins tested, compared with wild type (Fig. Additionally, no significant differences were detected between NDD and RP variants, pointing to the need for targeted functional studies to delineate their respective impacts on spliceosome dynamics. Similarly, U6 RNA bearing the n.55_56insT and n.57T>G variants, observed in healthy control individuals, presumably did not affect spliceosome formation, since the low amount of protein associated with them implies that they entered the spliceosome assembly process only minimally (Fig. Taken together, the results indicate that RP pathogenic variants have potentially a specific dominant effect on snRNP biogenesis and delay the assembly process at the di-snRNP stage. snRNPs were immunoprecipitated via MS2-YFP by anti-GFP antibodies and co-precipitated proteins were detected by western blotting. Immunoprecipitated proteins are normalized to input and U4 or U6 WT controls. Full-length blots and antibody validation are provided as Source Data. Ctrl, control; IP, immunoprecipitation; WT, wild type. The numerous genes associated with RP and allied diseases belong to a wide range of functional classes, from retina-specific biochemical pathways to ubiquitous cellular processes5. Yet, how these defects ultimately lead to retinal degeneration often remains unclear. The link between pathogenic variants in splicing factors of the tri-snRNP complex (RP-PRPFs), essential for survival in all eukaryotes, and RP, a phenotype limited to the eye, represents perhaps the most intriguing of these biological enigmas. Interestingly, these snRNAs are also an integral part of the di- and tri-snRNP and directly interact with some RP-PRPF proteins. In addition, similar to RP-PRPFs, they are also associated with the same specific phenotype: de novo or inherited adRP, with reduced penetrance for RNU4-2 variants. Importantly, the clinical presentation of patients with RNU4-2 and RNU6 variants overlaps with that of other spliceosome-related forms of adRP, particularly showing an earlier onset—contrasting with the generally milder prognosis observed in most other adRP types36,37—and a relatively high co-occurrence of features such as cataracts and cystoid macular edema, found in cases with PRPF3138,39, PRPF840 and SNRNP20041 variants. Prevalence estimations indicate that these snRNA pathogenic changes may account for an elevated number of undiagnosed cases, and it is therefore surprising that the RNU4 and RNU6 genes have escaped disease association until now. A partial explanation for this phenomenon is that mainstream sequencing approaches are biased towards DNA-capturing procedures that do not include snRNA genes. Furthermore, although genome sequencing is increasingly being adopted in routine diagnostics, variants in snRNA genes may have remained undetected because they affect noncoding transcripts, which are more challenging to interpret and are often overlooked or deprioritized by standard analytical pipelines. An intriguing feature of pathogenic changes in RNU4-2 is their pleiotropy with respect to NDD (ReNU syndrome) and RP. Chen et al.9 described that more than half of the patients with ReNU also display some visual abnormalities, although only three were documented as having retinal phenotypes (one had an abnormal electroretinogram response, one had Leber congenital amaurosis and one presented with macular dysfunction). However, most cases were too young to display the typical symptoms of RP, which usually manifest during adolescence or early adulthood42. Although the exact mechanism for this phenotypic selectivity is unknown, RNU4-2 variants represent a clear and new allelic series involving noncoding RNA genes. A recent preprint highlighted a strong effect of ReNU variants in an RNU4-2 saturation genome editing experiment, while the RP variant n.56T>C in the same gene did not show any statistically significant effect43. In addition, the region of RNU4-2 containing RP variants showed function scores within the neutral range of the saturation genome editing assay43, suggesting a potentially milder pathogenic effect compared with ReNU changes. It is therefore plausible that the RP variants identified in RNU4-2 and RNU6 paralogs could lead to photoreceptor death and subsequent visual loss, while having no influence on the development of the brain. Additionally, ReNU variants are located in the stem-III and the T-loop of the U4/U6 duplex and interfere with the proper recognition of intronic 5′ splice signals, likely because these regions are involved in pairing pre-mRNA with U6 (ref. Consistent with this evidence, we did not observe any major splicing anomalies in transcripts from patients with RP, with the only two significant events showing differences below 10% in expression. Conversely, our biochemical assays support a role for RP-associated variants in altering spliceosomal assembly. As the magnitude of the observed changes was in all instances rather moderate (less than 1.5-fold with respect to controls), we interpret these data as indicating that snRNA variants associated with RP are unlikely to prevent the assembly of spliceosomal complexes. Rather, they may cause a subtle alteration in snRNP dynamics, possibly affecting the efficiency of their biogenesis or recycling steps. In particular, the increased association with SART3 and, to a lesser extent, with PRPF31, together with unchanged or slightly reduced interaction with SNRNP200, may indicate a modest delay in the transition from the di-snRNP to the tri-snRNP form. Moreover, the pathogenic variants identified in this study lie within regions of the U4/U6 duplex that directly contact the PRPF3 and PRPF31 proteins, two splicing factors linked to adRP whose mutations also delay spliceosomal complex assembly32. In terms of specific molecular effect, our functional data show that snRNAs bearing RP variants display enhanced interaction with di-snRNP protein markers, suggesting that pathogenesis could result from a gain-of-function or dominant-negative mechanism, rather than from haploinsufficiency. This hypothesis is strengthened by the evidence that molecularly similar but benign variants, commonly observed in the general population, seem not to bind efficiently to di-snRNP markers and potentially not to be incorporated into the spliceosome, supporting the idea that spliceosomal functions could be haplosufficient with respect to heterozygous and snRNA-depleting variants. Although DNA changes associating RNU4-2 to ReNU syndrome have been primarily reported as de novo events9,10, in our study most families with RP (61%) bore RNU4-2 and RNU6 changes as inherited variants. In part, this difference can be explained by the reduced reproductive fitness associated with NDD versus RP. Unlike ReNU syndrome, symptomatic onset (night-blindness and peripheral vision loss) in nonsyndromic adRP begins later in life, with severe central vision loss usually occurring after the onset of reproductive age. Another difference involves the inheritance of dominant variants, which in ReNU seems to be almost exclusively of maternal origin9. We did not observe the same trend for RP, with variants being inherited from either of the parents, possibly indicating the absence of any sex-specific negative selection during gametogenesis or at the embryonic stage. This indicates that, assuming equal expression within paralogs, the presence of only ~25% of mutant U4 (heterozygous genotype, over two copies) or ~10% of mutant U6 (heterozygous genotype, over five copies) is sufficient to lead to a disease phenotype, again in support of a gain-of-function or dominant-negative molecular mechanism. This could be a crucial consideration for the development of potential gene-based therapies, as gene-augmentation strategies may be suboptimal compared with gene correction or antisense oligonucleotide approaches. Our data also highlight the existence of mutational hotspots outside the coding regions of the human genome, emphasizing the need for further research into these parts of our genetic material, and show that the clustering of de novo pathogenic variants is not restricted to severe diseases with childhood onset44, but may extend to milder pathologies, such as RP. In conclusion, we identified four recurrent pathogenic variants in RNU4-2 and in four of the five paralogs of the U6 snRNA as a frequent cause of de novo or inherited adRP. The immediate impact of these findings involves improved diagnosis and genetic counseling for patients with hereditary visual loss, especially for isolated cases who could potentially bear heterozygous de novo events. More fundamentally, this work substantially broadens our understanding of the genetic landscape of human disease, paving the way for the development of new molecular therapeutic approaches. This study adhered to the tenets of the Declaration of Helsinki, and signed, informed consent was obtained from all participants. 2009-32), and the local ethics committees of all other participating institutions. Complete ophthalmic examinations were performed by a retinal specialist, which included measurement of best-corrected visual acuity and intraocular pressures, and examination of anterior segment and fundus (dilated). Color fundus photographs and montages were captured using the FF450plus Fundus Camera (Carl Zeiss Meditec) and Optos 200 Tx (Optos). Fundus autofluorescence images (488-nm excitation) and high-resolution spectral-domain optical coherence tomography (SD-OCT) scans were acquired using the Spectralis HRA+OCT module (Heidelberg Engineering). Hyper-autofluorescent ring contours were analyzed using a custom program in FIJI software (National Institute of Mental Health)45. Progression rates were calculated using linear mixed-effects regression in R (v.4.0.4) with time (years) since baseline as the primary independent variable, baseline ring size as a covariate and inter-ocular differences as a random effect. Photoreceptor+ thickness was assessed on horizontal SD-OCT scans through the fovea using a semi-automated procedure46. Layer segmentation was performed in a semi-automated manner using a custom software in MatLab (MathWorks). Full-field electroretinogram recordings were conducted using the Espion Visual Electrophysiology System (Diagnosys) according to International Society for Clinical Electrophysiology of Vision (ISCEV) standards47. Sequencing was performed by BGI Tech Solutions using the DNBseq Sequencing Technology, with a minimal median coverage per genome of 30×. The processing of the sequencing data was performed by using BWA mem (v.0.7.17)48, Picard (v.2.14.0-SNAPSHOT) (http://broadinstitute.github.io/picard) and GATK (v.4.1.4.1)49 for mapping to the human genome reference sequence (build hg19/GRCh37) and variant calling2. For variant annotation, we used ANNOVAR50 with the addition of splicing predictions by MaxEntScan51 and SpliceAI19. Human Genome Variation Society (HGVS) notations of the variants were retrieved using VariantValidator52 and American College of Medical Genetics and Genomics (ACMG) classification53 was applied according to the ACGS Best Practice Guidelines for Variant Classification in Rare Disease 202354. In particular, we used the PS4_strong criterion for variants significantly enriched in cases versus controls (gnomAD v.4.1 and All of Us), as assessed for each variant by two-tailed Fisher's exact test in R (fisher.test function), in agreement with the ACMG recommendations53 (odds ratio > 5.0, lower bound of the confidence interval > 1.0, corrected P value < 0.05), but only for variants present in at least three probands to avoid any bias from imbalanced case–control sets. This assessment was made using probands only (n = 1,891) for all variants, except for those in Supplementary Table 1, which were assessed in 4,722 individuals. PM6 and PP1 were applied according to ClinGen Sequence Variant Interpretation (SVI) recommendations. Specifically, PM6_sup was applied when two unrelated families had de novo variants without parental confirmation, given that RP is a ‘phenotype consistent with gene but not highly specific and high genetic heterogeneity'. We defined thresholds for PM2_sup and BS2 based on the frequency of RHO p.(Pro23His), the most prevalent variant causing adRP, which was detected once in gnomAD v.4.1 and 13 times in All of Us. Specifically, PM2_sup was assigned to variants that were present fewer than two times in gnomAD v.4.1 and fewer than 14 times in All of Us. BS2 was applied to variants that were observed more than four times in gnomAD v.4.1 or more than 28 times in All of Us, that is, twice the values of p.(Pro23His). BA1 was considered for variants with allele frequency >5% in gnomAD v.4.1 or the All of Us databases, whereas BS1 was assigned to variants with allele frequencies greater than expected for disease (1/2,000 = 0.05%). RNU4-1, RNU4-2, RNU6-1, RNU6-2, RNU6-7, RNU6-8 and RNU6-9 PCR fragments were sequenced using Sanger sequencing and screened for novel variants in these genes. We utilized RNAfold WebServer to model the effect of variants with default parameters56 and RNAcanvas was used for drawing the structure57. We used ChimeraX with PDB file, using PDB file 6QW6 to draw three-dimensional representation of the U4/U6 duplex with and without surrounding PRPF proteins. RNA was isolated from human donor eye tissue, which was collected and dissected according a reported procedure58 from an ethically approved Research Tissue Bank (UK NHS Health Research Authority reference no. Total RNA was isolated from four NSR samples, 16 pelleted RPE samples and 13 choroid samples that had been stored in RNAlater (Thermo Fisher Scientific), using an Animal Tissue RNA Purification kit (Norgen Biotek), as per manufacturer's instructions. Sequencing libraries were prepared using the NEBnext multiplex small RNA library preparation kit, as per manufacturer's protocols, with size selection performed using Ampure beads. Paired-end sequencing (2 × 75 base pairs (bp)) was performed on an Illumina HiSeq 4000. NEBnext adapters were removed from sequencing reads using trimmomatic (v.0.39) before alignment against the GRCh38 reference genome with bowtie59 (v.1.3). No mismatches between sequencing reads and the reference genome were allowed, and no restriction was set on multi-mapping reads. Sequence read counts were restricted to primary alignments using samtools (v.1.21)60, and therefore only counted once if they aligned to multiple RNU4 (n = 90) or RNU6 (n = 1,277) genes or pseudogenes. Calculations were drawn from read 1 datasets and normalized for the total read count achieved for the sample. 61 (eight different experiments) and H3K27ac ChIP–seq data from ref. 62 (five different experiments) were downloaded as bigwig files from the RegRet database (http://genome.ucsc.edu/s/stvdsomp/RegRet)63. For both data types, the signal (the genes and 500 bp on each side) was extracted using bedtools (v.2.27.1) after conversion using bigWigToWig (v.469). We quantified the signal for all RNU genes and pseudogenes first by normalizing the signal of each experiment to the maximum and then summing them. Peripheral blood samples were collected from affected individuals and controls using either Tempus Blood RNA tubes (Applied Biosystems) or PAXgene Blood RNA tubes (Qiagen). Total leukocyte RNA was extracted with the Tempus Spin RNA Isolation Kit (Applied Biosystems) or the Preserved Blood RNA Purification Kit II (Norgen Biotek), respectively, following the manufacturers' protocols. Following the quality assessment of RNA integrity and concentration, 100 ng of input RNA per sample was subsequently processed for library preparation using the KAPA RNA HyperPrep Kit with RiboErase (HMR) and KAPA Globin Depletion Hybridization Oligos (Roche). Sequencing was performed on an Illumina NovaSeq 6000 platform with 2 × 101-bp paired-end reads. To improve quality score calculations for the final base, one additional base was sequenced in both read 1 and read 2. The Q30 value for all RNA-seq data was ≥91.1%. Reads were aligned to reference transcripts from Ensembl (v.110, GRCh38) using STAR (v.2.7.11a) with the option --twopassMode Basic. DESeq2 (v.1.46.0) with default options was used for differential expression analysis between different groups according to sample origin (Tempus or PAXgene tubes) and presence/absence of the pathogenic RNU genotypes, with fold-change > 2 and FDR P value < 0.05. We used rMATS65 to assess differential alternative splicing, separately for the Tempus and PAXgene sets and with specific options (--allow-clipping --variable-read-length --anchorLength 1 --novelSS --task both --libType fr-unstranded -t paired --readLength 101). We further used the Python scripts from ref. 11 to process the rMATS output and filter the data according to a mean coverage > 7, an FDR P value < 0.1 and a deltaPSI value > 0.05. The R function fisher.test with default parameters was used to assess differences in base compositions at splicing sites, at each position, as well as differences for 2-mers (for example, positions −4/−3 to +7/+8). U4 n.18_19insA, n.56T>C and n.64_65insT variants were introduced by site-directed mutagenesis into the plasmid expressing U4-MS266. The MS2 loop was inserted between nucleotides 10 and 11. U6 n.55_56insG, n.55_56insT, n.56_57insG and n.57G>T variants were introduced by site-directed mutagenesis. U4- and U6-expressing plasmids were transfected into HeLa cells stably expressing MS2-YFP protein. At 24 h after transfection, snRNAs were immunoprecipitated using anti-GFP antibodies and co-precipitated proteins were analyzed by western blotting66. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Research on the de-identified patient data used in this publication from the Genomics England 100,000 Genomes Project and the NHS GMS dataset can be carried out in the Genomics England Research Environment subject to a collaborative agreement that adheres to patient-led governance. Sharing of additional sequencing or blood RNA-seq data is subject to the European General Data Protection Regulation (GDPR) applicable in the countries of residence of the tested individuals and may become available upon a data transfer agreement approved by local ethical committees. Patient sample identifiers from this study can be released upon reasonable request from ‘M1-A to M9-B' to the corresponding local ‘DNA-number'. Specific variant requests or other data are available from the corresponding author (S.R.) The data generated during this study (causative variants from Supplementary Table 1) are submitted to the Leiden Open (source) Variation Database (LOVD) (http://www.lovd.nl) and ClinVar (accession codes SCV006562526 to SCV006562534). Additional details regarding PCR conditions or primer design are available upon request. Detailed information on antibodies used are provided in Supplementary Table 10. Small RNA-seq datasets analyzed in this study are available at the NCBI Sequence Read Archive through accession PRJNA1256119 (https://www.ncbi.nlm.nih.gov/sra/PRJNA1256119). The genes and pseudogenes analyzed are present in Supplementary Table 11 and the read counts are available in Supplementary Table 12. Source data are provided with this paper. The first genetic landscape of inherited retinal dystrophies in Portuguese patients identifies recurrent homozygous mutations as a frequent cause of pathogenesis. Perea-Romero, I. et al. Genetic landscape of 6089 inherited retinal dystrophies affected cases in Spain and their therapeutic and extended epidemiological implications. Conti, G. M. et al. Genetics of retinitis pigmentosa and other hereditary retinal disorders in western Switzerland. Rivolta, C. et al. RetiGene, a comprehensive gene atlas for inherited retinal diseases. Hanany, M., Rivolta, C. & Sharon, D. Worldwide carrier frequency and genetic prevalence of autosomal recessive inherited retinal diseases. The mutational constraint spectrum quantified from variation in 141,456 humans. Rogalska, M. E. et al. Transcriptome-wide splicing network reveals specialized regulatory functions of the core spliceosome. Chen, Y. et al. De novo variants in the RNU4-2 snRNA cause a frequent neurodevelopmental syndrome. Greene, D. et al. Mutations in the U4 snRNA gene RNU4-2 cause one of the most prevalent monogenic neurodevelopmental disorders. Dominant variants in major spliceosome U4 and U5 small nuclear RNA genes cause neurodevelopmental disorders through splicing disruption. Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes. Greene, D. et al. Mutations in the small nuclear RNA gene RNU2-2 cause a severe neurodevelopmental disorder with prominent epilepsy. Binding of the human Prp31 Nop domain to a composite RNA-protein platform in U4 snRNP. Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. Predicting splicing from primary sequence with deep learning. Detection of nonneutral substitution rates on mammalian phylogenies. Davydov, E. V. et al. Identifying a high fraction of the human genome to be under selective constraint using GERP++. Cost-effective sequence analysis of 113 genes in 1,192 probands with retinitis pigmentosa and Leber congenital amaurosis. Caulfield, M. et al. National Genomic Research Library. & Sullivan, L. S. Genes and mutations causing autosomal dominant retinitis pigmentosa. Prevalence of disease-causing mutations in families with autosomal dominant retinitis pigmentosa: a screen of known genes in 200 families. Charenton, C., Wilkinson, M. E. & Nagai, K. Mechanism of 5′ splice site transfer for human spliceosome activation. Hardin, J. W., Warnasooriya, C., Kondo, Y., Nagai, K. & Rueda, D. Assembly and dynamics of the U4/U6 di-snRNP by single-molecule FRET. Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Zhong, Z. et al. Two novel mutations in PRPF3 causing autosomal dominant retinitis pigmentosa. & Weiner, A. M. Abundant pseudogenes for small nuclear RNAs are dispersed in the human genome. Tanackovic, G. et al. PRPF mutations are associated with generalized defects in spliceosome formation and pre-mRNA splicing in patients with retinitis pigmentosa. Comparative 3D genome analysis between neural retina and retinal pigment epithelium reveals differential cis-regulatory interactions at retinal disease loci. Prasetyo, N. K. & Gardner, P. P. Assessing the robustness of human ncRNA notation at HGNC. Skogholt, A. H. et al. Gene expression differences between PAXgene and Tempus blood RNA tubes are highly reproducible between independent samples and biobanks. Visual acuity impairment in patients with retinitis pigmentosa at age 45 years or older. Bodenbender, J. P. et al. Clinical and genetic findings in a cohort of patients with PRPF31-associated retinal dystrophy. Waseem, N. H. et al. Mutations in the gene coding for the pre-mRNA splicing factor, PRPF31, in patients with autosomal dominant retinitis pigmentosa. Maubaret, C. G. et al. Autosomal dominant retinitis pigmentosa with intrafamilial variability and incomplete penetrance in two families carrying mutations in PRPF8. Yusuf, I. H. et al. Clinical characterization of retinitis pigmentosa associated with variants in SNRNP200. Saturation genome editing of RNU4-2 reveals distinct dominant and recessive neurodevelopmental disorders. & Brunner, H. G. De novo mutations in human genetic disease. Patients and animal models of CNGβ1-deficient retinitis pigmentosa support gene augmentation approach. Robson, A. G. et al. ISCEV Standard for full-field clinical electroretinography (2022 update). Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. DePristo, M. A. et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals. & Dalgleish, R. VariantValidator: accurate validation, mapping, and formatting of sequence variation descriptions. Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Ellard, S. et al. ACGS Best Practice Guidelines for Variant Classification 2019. Biesecker, L. G. et al. ClinGen guidance for use of the PP1/BS4 co-segregation and PP4 phenotype specificity criteria for sequence variant pathogenicity classification. Gruber, A. R., Lorenz, R., Bernhart, S. H., Neuböck, R. & Hofacker, I. L. The Vienna RNA websuite. & Simon, A. E. RNAcanvas: interactive drawing and exploration of nucleic acid structures. McHarg, S. et al. Mast cell infiltration of the choroid and protease release are early events in age-related macular degeneration associated with genetic risk at both chromosomes 1q32 and 10q26. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Wang, J. et al. ATAC-Seq analysis reveals a widespread decrease of chromatin accessibility in age-related macular degeneration. Cherry, T. J. et al. Mapping the cis-regulatory architecture of the human retina reveals noncoding genetic variation in disease. Van de Sompele, S. et al. Multi-omics approach dissects cis-regulatory mechanisms underlying North Carolina macular dystrophy, a retinal enhanceropathy. Jiang, H., Lei, R., Ding, S. W. & Zhu, S. Skewer: a fast and accurate adapter trimmer for next-generation sequencing paired-end reads. Shen, S. et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-seq data. The Sm-core mediates the retention of partially-assembled spliceosomal snRNPs in Cajal bodies until their full maturation. We thank all patients and their family members for their help and participation in this study. The title page was formatted using AuthorArranger, a tool developed at the National Cancer Institute (National Institutes of Health, NIH). was supported by the Swiss National Science Foundation (SNSF) Grant No. 310030_204285 entitled ‘Genomics of inherited retinal diseases'. was supported by the Foundation Fighting Blindness Career Development Award (grant no. and D. Stanek were supported by HORIZON-MSCA-2022-DN (grant no. This work has been funded by a Foundation Fighting Blindness Program Project Award (grant no. – Jonas Children's Vision Care (JCVC) is supported by the National Institute of Health grant nos. U01EY030580, U01EY034590 R24EY028758, R24EY027285, 5P30EY019007, R01EY033770 and R01EY018213, R01EY024698; the Foundation Fighting Blindness grant no. was supported by Instituto de Salud Carlos III (ISCIII) of the Ministerio de Ciencia e Innovación and Unión Europea – European Regional Development Fund (FEDER) (grant nos. PI22/00321 and IMP/00009), Centro de Investigación Biomédica en Red Enfermedades Raras (CIBERER, grant no. This work was performed by using the data contained in the ‘Programa Infraestructura de Medicina de Precisión asociada a la Ciencia y la Tecnología en Medicina Genómica (IMPaCT-GENóMICA)', coordinated by the CIBERER and founded by ISCIII. L.F.-C. was supported by Centro de Investigación Biomédica en Red (CIBER). R.A. was supported by the National Eye Institute (NEI) (grant nos. was supported by the Combined Ophthalmic Research Rotterdam grant no. was supported by ZonMw Vidi (grant no. and N.C. were supported by Fighting Blindness Ireland (grant nos. MRCG-2016-14) and the Science Foundation Ireland (grant nos. was supported by the Macular Society (United Kingdom), the National Institute for Health and Care Research (NIHR) Manchester Biomedical Research Centre (BRC) (grant no. NIHR203308) and the University of Manchester Core Genomics Technology Facility. was supported by the Israel Science Foundation (grant no. is supported by the University of Melbourne Research Fellowship. was supported by the National Eye Institute (NEI) (grant nos. were supported by grants from the Foundation Fighting Blindness (grant no. were supported by Ghent University Special Research Fund (grant no. 1803816N) are Senior Clinical Investigators of the Research Foundation-Flanders (FWO). N.M. and S.S. are Ph.D. fellows of HORIZON-MSCA-2022-DN ProgRET (grant no. R.A. was supported by the Foundation Fighting Blindness. P30 EY002162, the Foundation Fighting Blindness, an unrestricted grant from RPB and the All May See Foundation. was supported by research grants from the Japan Agency for Medical Research and Development (AMED) (grant nos. R01 EY030499-01, Dr. Lentz), The Canadian Institutes for Health Research (CIHR), Fighting Blindness Canada (FBC) and Fonds de Recherche du Québec - Santé (FRQS). participates in the NAC Attack clinical trial, which is funded by the NIH via grant nos. were supported by Retina Australia (awarded to the Australian Inherited Retinal Disease Registry and DNA Bank). was supported by the Wellcome Trust (grant no. was supported by the BrightFocus Foundation (grant no. was supported by the National Eye Institute (NEI) (grant no. were supported by the Italian Ministry of Health (grant no. D. Stanek was supported by the Project P JAC grant no. CZ.02.01.01/00/22_008/0004575 RNA for therapy, Co-Funded by the European Union. was supported by the European Commission (Recon4IMD – grant no. GAP-101080997) and the Deutsche Forschungsgemeinschaft (German Research Foundation, DFG, grant nos. were supported by a research grant (no. were supported by the Velux Stiftung Grant no. are supported by Retina UK and Fight for Sight UK (RP Genome Project Grant no. were supported by the National Institute for Health Research Biomedical Research Centre at Moorfields Eye Hospital and UCL Institute of Ophthalmology. was funded by a Medical Research Council (MRC) Clinician Scientist Fellowship (grant no. was supported by Instituto de Salud Carlos III (ISCIII) of the Spanish Ministry of Health (grant no. CIPROM/2023/26 from the Generalitat Valenciana and IMPaCT-GENOMICA (grant no. and A.G. were supported by the Dutch Ministry of Education, Culture and Sciences, Gravitation grant no. This work is supported by partners of the European Reference Network for Rare Eye Diseases ERN-EYE (Grant Agreement No. Novartis contributed funding for the preceding RP-LCA smMIPs panel design and subsequent sequencing (to F.P.M.C., S.R. Novartis was not involved in the study design; collection, analysis or interpretation of data; the writing of this article; or the decision to submit it for publication. This research was made possible through access to data generated by the 2025 French Genomic Medicine initiative and present in the National Genomic Research Library, which is managed by Genomics England Limited (a wholly owned company of the Department of Health and Social Care). The National Genomic Research Library holds data provided by patients and collected by the NHS as part of their care, and data collected as part of their participation in research. The National Genomic Research Library is funded by the National Institute for Health Research and NHS England. These authors contributed equally: Mathieu Quinodoz, Kim Rodenburg. These authors jointly supervised this work: Susanne Roosing, Carlo Rivolta. Mathieu Quinodoz, Karolina Kaminska, Ana Belén Iglesias-Romero, Mukhtar Ullah, Marc Folcher, Francesca Cancellieri & Carlo Rivolta Mathieu Quinodoz, Karolina Kaminska, Ana Belén Iglesias-Romero, Marc Folcher, Georg Ansari, Francesca Cancellieri, Nicolas Feltgen, Maximilian Pfau & Carlo Rivolta Department of Human Genetics, Radboud University Medical Center, Nijmegen, the Netherlands Kim Rodenburg, Suzanne E. de Bruijn, Erica G. M. Boonen, Nick Zomer, Lara K. Holtes, Zelia Corradi, Stefanida Shliaga, Daan M. Panneman, Rebekkah J. Hitti-Malin, Galuh D. N. Astuti, Ronny Derks, Alejandro Garanto, Christian Gilissen, Lonneke Haer-Wigman, Alexander Hoischen, Kornelia Neveling, Michiel Oorsprong, Frans P. M. Cremers & Susanne Roosing Department of Otorhinolaryngology, Hearing & Genes, Radboud University Medical Center, Nijmegen, the Netherlands Erica G. M. Boonen & L. Ingeborgh van den Born Department of Ophthalmology, Rothschild Foundation Hospital, Paris, France Departments of Ophthalmology, Pathology & Cell Biology, Columbia Stem Cell Initiative, Vagelos College of Physicians and Surgeons Columbia University Irving Medical Center, New York, NY, USA Edward S. Harkness Eye Institute, Jonas Children's Vision Care (JCVC), Columbia University Irving Medical Center, New York-Presbyterian Hospital, New York, NY, USA Manir Ali, Chris F. Inglehearn, Martin McKibbin & Carmel Toomes Molecular Biology Research Unit, St John Eye Hospital Group, Jerusalem, Palestine Gavin Arno, Jessica C. Gardner, Alison J. Hardcastle, Omar A. Mahroo, Michel Michaelides & Andrew R. Webster Gavin Arno, Omar A. Mahroo, Michel Michaelides, Andrew R. Webster & Siying Lin Department of Genetics & Genomics, Instituto de Investigación Sanitaria-Fundación Jiménez Díaz University Hospital, Universidad Autónoma de Madrid (IIS-FJD, UAM), Madrid, Spain Carmen Ayuso, Fiona Blanco-Kelly, Lidia Fernández-Caballero & Ana Isabel Sánchez-Barbero Center for Biomedical Network Research on Rare Diseases (CIBERER), Instituto de Salud Carlos III, Madrid, Spain Carmen Ayuso, Fiona Blanco-Kelly, Lidia Fernández-Caballero & Ana Isabel Sánchez-Barbero Department of Ophthalmology, Shiley Eye Institute, University of California San Diego, La Jolla, CA, USA Sandro Banfi, Marianthi Karali, Margherita Scarpato & Roberta Zeuli Eyal Banin, Asodu Sandeep Sarma & Dror Sharon Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam, the Netherlands Tahsin Stefan Barakat, Lies H. Hoefsloot & Virginie J. M. Verhoeven Department of Ophthalmology, Semmelweis University, Budapest, Hungary Mirella T. S. Barboni, Krisztina Knézy, Zoltán Zsolt Nagy & Viktória Szabó Miriam Bauwens, Elfride De Baere, Marieke De Bruyne, Bart P. Leroy, Quinten Mahieu, Nelson Martins & Mattias Van Heetvelde Miriam Bauwens, Elfride De Baere, Marieke De Bruyne, Quinten Mahieu, Nelson Martins & Mattias Van Heetvelde Virginie Bernard, Luke Mansard & Francis Ramond Retina Foundation of the Southwest, Dallas, TX, USA Beatrice Bocquet, Vasiliki Kalatzis, Gaël Manes, Luke Mansard & Isabelle Meunier Beatrice Bocquet, Vasiliki Kalatzis & Isabelle Meunier Dominique Bremond-Gignac, Cyril Burin des Roziers & Sophie Valleix Ocular Genomics Institute, Massachusetts Eye and Ear, Harvard Medical School, Boston, MA, USA Service de Médecine Génomique des Maladies de Système et d'Organe, Hôpital Cochin, AP-HP Centre Université Paris Cité, Paris, France Cyril Burin des Roziers & Sophie Valleix Laboratoire de Biologie Médicale SeqOIA, Site Broussais, Paris, France Cyril Burin des Roziers & Sophie Valleix Elizabeth L. Cadena, Stephen P. Daiger & Lori S. Sullivan Vista Vision Eye Clinic, Brescia, Italy Neurogenetics Research Center, IRCCS Mondino Foundation, Pavia, Italy Luca Cattaneo, Simone Gana & Enza Maria Valente Naomi Chadderton, G. Jane Farrar & Laura Whelan Department of Cell Therapy and Regenerative Medicine, Andalusian Molecular Biology and Regenerative Medicine Centre, CABIMER, Seville, Spain Australian Inherited Retinal Disease Registry and DNA Bank, Department of Medical Technology and Physics, Sir Charles Gairdner Hospital, Perth, Western Australia, Australia John N. De Roach, Tina M. Lamey & Terri L. McLaren Department of Paediatrics and Inherited Metabolic Disorders, First Faculty of Medicine, Charles University and General University Hospital in Prague, Prague, Czech Republic Lubica Dudakova, Petra Liskova & Marie Vajter Department of Ophthalmology, University of California, San Francisco, Wayne and Gladys Valley Center for Vision, San Francisco, CA, USA Department of Pediatrics, Amalia Children's Hospital, Radboud University Medical Center, Nijmegen, the Netherlands Kensuke Goto, Taro Kominami & Ai Fujita Sajiki Genomics for Health in Africa (GHA), Africa-Europe Cluster of Research Excellence (CoRE), Ghana, South Africa Department of Ophthalmology and Vision Sciences, Ocular Genetics Program, The Hospital for Sick Children, Toronto, Ontario, Canada Department of Internal Medicine, Radboud University Medical Center, Nijmegen, the Netherlands Department of Ophthalmology, Radboud University Medical Center, Nijmegen, the Netherlands Department of Ophthalmology, Section of Pediatric Ophthalmology, Strabismus, and Ocular Genetics, DOH Eye Center, East Avenue Medical Center, Quezon City, Philippines Division of Molecular and Cellular Biology, National Institute of Sensory Organs, NHO Tokyo Medical Center, Tokyo, Japan Takeshi Iwata, Yang Pan, Akiko Suga & Kazutoshi Yoshitake Brynjar O. Jensson, Kari Stefansson & Patrick Sulem Department of Inherited Retinal Dystrophies, Ophthalmic Genetics Group, OMMA, Ophthalmological Institute of Athens, Athens, Greece Eye Clinic, Multidisciplinary Department of Medical, Surgical and Dental Sciences, University of Campania “Luigi Vanvitelli”, Naples, Italy Department of Ophthalmology, Erasmus MC University Medical Center, Rotterdam, the Netherlands Department of Pediatric Surgery, Division of Pediatric Ophthalmology, Montreal Children's Hospital, McGill University Health Center (MUHC), Montreal, Quebec, Canada Department of Ophthalmology, Rambam Health Care Campus, Haifa, Israel Department of Ophthalmology, First Faculty of Medicine, Charles University and General University Hospital in Prague, Prague, Czech Republic Department of Ophthalmology, Fundación Jiménez Díaz University Hospital, Madrid, Spain Anastasia Papachristou, Miltiadis K. Tsilimbaris & Pavlina Tsoka Raj Ramesar, Lisa Roberts & Casey Valentine Department of Ophthalmology, HUB-Erasme Hospital, Brussels, Belgium Department of Neuroscience, Biodonostia Health Research Institute, Donostia-San Sebastián, Spain Department of Genetic, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV/EHU), Leioa, Spain iNOVA4Health, NOVA Medical School, Faculdade de Ciências Médicas, NMS, FCM, Universidade NOVA de Lisboa, Lisboa, Portugal Department of Child Neurology and Psychiatry, IRCCS Mondino Foundation, Pavia, Italy Department of Medical Genetics, ULS Santa Maria, Lisboa, Portugal Medical Genetics and Prenatal Diagnostics Clinic, Children's Clinical University Hospital, Riga, Latvia Department of Ophthalmology; Oculogenetics Unit, Jules Gonin University Hospital; University of Lausanne, Lausanne, Switzerland School of Pharmacy and Biomolecular Sciences (PBS), RCSI University of Medicine and Health Sciences, Dublin, Ireland Bonei Olam - Center for Rare Jewish Genetic Diseases, Brooklyn, NY, USA Department of Ophthalmology, Columbia University Irving Medical Center, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to the collection of material and clinical data for family M1-A. coordinated the collection of DNA samples. carried out the molecular biology experiments. Z. Cvackova and D. Stanek conducted all in vitro experiments. M.A., A.A., S.A., G. Ansari, G. Arno, G.D.N.A., C.A., R.A., S.B., E.B., T.S.B., M.T.S.B., M.B., T.B.-Y., V.B., D.G.B., P.B., F.B.-K., B.B., C.J.F.B., K.B., D.B.-G., A.C.B.-J., K.M.B., C.B.d.R., E.L.C., G.C., F.C., L.C., N.C., P.C.I., L.C.-S., S.P.D., E.D.B., M.D.B., B.d.l.C., J.N.D.R., J.D.Z., R.D., C.-M.D., L.D., J.L.D., G.J.F., N.F., B.J.F., L.F.-C., J.M.F.S., S.G., A.G., J.C.G., C.G., R.G.-D., K.G., S.G.-J., T.B.H., L.H.-W., A.J.H., T.H., E.H., L.H.H., A.H., J.P.H., C.B.H., M.B.B.I., C.F.I., T.I., B.O.J., K.J., V.K., S. Kamakari, M.K., U.K., C.C.W.K., K.K., R.K.K., S. Kohl, T.K., L.K., T.M.L., R.L., B.P.L., S.L., P.L., I.L., V.R.d.J.L.-R., Q.M., O.A.M., G.M., L. Mansard, M.P.M.-G., N.M., L. Mauring, M. McKibbin, T.L.M., I.M., M. Michaelides, J.M.M., K.M., R.M., Z.Z.N., K.N., M. Ołdak, M. Oorsprong, Y.P., A. Papachristou, A. Percesepe, M.P., E.A.P., E.P., R.R., F.R., F.A.R., G.I.R., L.R., M.R.-H., J.R.-E., A.H.S., A.F.S., A.I.S.-B., A.S.S., R.S., C.M.S., M. Scarpato, H.P.N.S., D. Sharon, S.G.S., F.S., A.B.S., M. Stefaniotou, K. Stefansson, K. Stingl, A.S., P.S., L.S.S., V.S., J.P.S., G.T., A.A.H.J.T., C.T., V.H.T., M.K.T., P.T., V.V., M.V., S. Valeina, E.M.V., C.V., R.V., S. Valleix, J.v.A., L.I.v.d.B., M.V.H., V.J.M.V., A.L.V., A.R.W., L.W., B.W., G.G.Y., K.Y., J.C.Z., R.Z. were instrumental in acquiring funding, clinical data, validation of genetic or molecular data, and reviewing and editing the final manuscript. All authors approved the final content of this work. The authors affiliated with deCODE genetics/Amgen Inc. The other authors declare no competing interests. Nature Genetics thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Quinodoz, M., Rodenburg, K., Cvackova, Z. et al. De novo and inherited dominant variants in U4 and U6 snRNA genes cause retinitis pigmentosa. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02217-y'>Large-scale testing of antimicrobial lethality at single-cell resolution predicts mycobacterial infection outcomes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 11:03:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In vitro antibiotic testing is important for guiding therapy and drug development. Current methods are focused on growth inhibition in bulk bacterial populations but often fail to accurately predict treatment responses. Here we introduce Antimicrobial Single-Cell Testing (ASCT), a large-scale live-cell imaging approach that quantifies bacterial killing in real time at single-cell resolution. By tracking over 140 million mycobacteria and analysing ~20,000 time–kill curves, we identify key determinants of antibiotic killing and its clinical relevance. For Mycobacterium tuberculosis, we found that drug-specific killing dynamics in starved bacteria, rather than growth inhibition or killing of growing cells, predict regimen efficacy in mice and humans. Extending this approach to Mycobacterium abscessus and comparing 405 bacterial strains, we show that antibiotic killing is also a genetically encoded bacterial trait (drug tolerance). We demonstrate that tolerance patterns cluster by antibiotic targets, identify a phage protein that modulates antibiotic killing, and show that strain-specific killing dynamics are associated with individual patient outcomes independent of drug resistance. Together, these findings establish a framework that reveals how drug properties and bacterial diversity shape treatment responses, offering a path to more effective and personalized therapies. Bacteria have evolved diverse strategies to overcome toxic exposures. This phenomenon is known as drug tolerance, referring to delayed killing of the overall population, and persistence, when a highly tolerant subpopulation is present2. Despite decades of recognition and the identification of multiple tolerance mechanisms3,4,5,6, key aspects of tolerance biology, especially in human infections, and its impact on antibiotic treatment remain poorly understood7. Even in the absence of antibiotic resistance, treatment outcomes for bacterial infections, including urinary tract, respiratory and bloodstream infections, are often poor8,9,10,11. This challenge is highlighted in mycobacterial infections, where limited treatment efficacy necessitates prolonged multidrug therapy lasting months to years. For drug-susceptible Mycobacterium tuberculosis, which caused more than 1 million deaths in 202312, it took decades of research to shorten treatment from 6 to 4 months13. These extended treatment durations are costly, associated with treatment-related toxicity and increase the risk of non-adherence, potentially driving relapses and transmission. The therapeutic challenge is even more acute in Mycobacterium abscessus infections, now one of the most prevalent mycobacterial infections in developed countries14. For M. abscessus, no consistently effective drug regimen exists, and treatment outcomes are often poor, with failure rates frequently exceeding 50% despite months or years of treatment with multiple in vitro active drugs15. This gap between in vitro growth inhibition and in vivo efficacy motivated us to develop strategies beyond standard susceptibility testing to better predict treatment outcomes. Bacterial killing could be such a factor, which is not captured by minimum inhibitory concentrations (MICs) and has been challenging to study due to the labour-intensive, low-throughput nature of colony-forming unit (c.f.u.) To overcome this gap, we established Antimicrobial Single-Cell Testing (ASCT), a highly scalable method for quantifying bacterial viability at single-cell resolution. We first applied ASCT to Mycobacterium tuberculosis to examine whether the killing properties of different drug regimens are linked to in vivo treatment responses in mouse and human populations. Extending this approach to Mycobacterium abscessus, we then compared hundreds of clinical isolates to determine how bacterial strain variation shapes antibiotic killing dynamics and individual patient outcomes. To quantify bacterial growth and antibiotic killing in bacterial populations and subpopulations at a scale comparable to that of MIC assessments, we developed ASCT (Fig. 1 and Supplementary Video 1), a workflow that simultaneously tracks the behaviour and viability of millions of single cells across diverse conditions in 1,536-well plates. In ASCT, bacteria are dispensed into multiwell plates, immobilized in agar pads containing the viability dye propidium iodide (PI) and exposed to drugs (Fig. High-content live-cell imaging captures brightfield and fluorescence images of over 10,000 fields every 2–4 h for up to 7 days, generating up to one million images per experiment. Time-lapse images of every field are sequentially analysed using sparse and low-rank decomposition to correct for background fluorescence19, supervised random forest classifiers for bacterial segmentation and viability classification20, drift correction and single-cell tracking based on object position and homology (Fig. To analyse time–kill kinetics at the population level, single-cell data are pooled, and overall killing is quantified using the area under the time–kill curve (AUC; Fig. ASCT showed no detectable effect on bacterial viability in antibiotic-free conditions (Extended Data Fig. Although ASCT can also quantify homogeneous and heterogenous bacterial growth (for example, resistance or hetero-resistance), single-cell tracking of growing cells is constrained by the small well size and overlapping bacteria. This study therefore focuses on the determinants and consequences of antibiotic killing. a, Illustration of the ASCT setup. Bacteria (dead cells in red) are immobilized in agar pads containing the viability dye PI at the bottom of wells of a 1,536-well plate and exposed to drugs. Live-cell imaging (brightfield and fluorescence) of 9 fields per well and up to 1,536 wells was performed at 2–4-h intervals from the plate bottom. b, Schematic of the automated image analysis. Raw brightfield images (to assess cell morphology) and fluorescence images (Cy3 settings, to quantify PI accumulation) were processed to segment cells (lines indicate cell borders), classify bacterial viability (white, viable; red, dead) and track individual cells (track IDs indicated by numbers). c–e, ASCT-derived time–kill curves of M. abscessus (isolate number 328). c, Example of 4,212 single-cell trajectories during moxifloxacin exposure from a single imaging well, with mean PI intensity over time highlighted. Bacterial trajectories are ordered by the time of PI positivity. The white line represents the proportion of live bacteria across 29 timepoints and reveals population time–kill kinetics. The AUC quantifies overall killing, integrating drug tolerance and persistence phenotypes. barely exceeds the line thickness) across multiple antibiotics (d) and tigecycline concentrations (e). AMK, amikacin; AZM, azithromycin; FOX, cefoxitin; IPM, imipenem; LZD, linezolid; MIN, minocycline; MXF, moxifloxacin; TGC, tigecycline. The number of tracked bacteria per imaging well was 4,246 ± 1,227 (mean ± s.d.). f,g, Green fluorescent protein (GFP) induction after 12-h cefoxitin treatment and washout with PI fluorescence. f, Representative brightfield, GFP and PI fluorescence images with merged overlays acquired ~1 h after antibiotic removal and GFP induction. The experiment was repeated twice with similar results. g, Corresponding single-cell fluorescence intensities, with gating of GFP- and PI-positive and negative populations. Bacterial viability was assessed using PI, a cell-impermeable fluorescent dye that accumulates in cells upon membrane disruption and serves as a proxy for bacterial death, as previously demonstrated in multiple bacterial species21,22,23. To validate this method, we treated M. abscessus with a number of antibiotics for 24 h and tracked more than 30,000 single bacteria for a further 24 h post antibiotic washout. Only one PI-positive bacterium resumed growth (a misclassified doublet of a live and dead bacterium), while 1–11% of the PI-negative bacteria regrew in antibiotic-free media (Extended Data Fig. 2a and Supplementary Video 2), confirming that PI accumulation is a reliable marker of bacterial death. To further assess the viability of PI-negative bacteria, we used an inducible M. abscessus reporter strain to test their capacity to synthesize new proteins. After 12 h of antibiotic exposure, drugs were removed and GFP expression was induced with anhydrotetracycline. Following cefoxitin or moxifloxacin treatment, most PI-negative bacteria expressed GFP shortly after induction (Fig. 2b,c), indicating that they remained viable. Bacteria treated with amikacin, a direct inhibitor of protein synthesis, showed delayed GFP expression after washout. Importantly, the fraction of bacteria capable of GFP synthesis far exceeded the fraction of regrowing bacteria (4.2% for cefoxitin, 2.7% for moxifloxacin and 3.4% for amikacin), demonstrating that regrowth assays underestimate survival and that most metabolically competent cells fail to regrow in standard media conditions. Moreover, repeated assessments after antibiotic removal revealed a progressively increasing PI-positive population (Extended Data Fig. Using ASCT, we first investigated whether the drug's bactericidal properties correspond to in vivo outcomes in mice and humans. Specifically, we analysed the potential of ASCT to identify more effective treatments for tuberculosis. We tested antibiotic killing of 65 drug regimens in the two avirulent M. tuberculosis strains H37Ra and mc27000, in nutrient-rich and starvation conditions. Each drug was dosed at the maximum blood concentration (Cmax) achievable during therapeutic dosing in humans. Drug regimens containing isoniazid, rifampicin or ethambutol (all part of the standard M. tuberculosis regimen) were most effective in killing growing M. tuberculosis (Extended Data Fig. In contrast, combinations including bedaquiline, clofazimine or pretomanid, were more effective under starvation conditions, consistent with previous reports that these drugs retain activity in non-replicating M. tuberculosis (Fig. Their efficacy during starvation is probably due to a disruption of energy metabolism, respiration or redox balance, which are essential processes for survival in nutrient-limited environments. a, ASCT-based time–kill kinetics of PBS-starved M. tuberculosis (mc27000) exposed to 65 drug regimens, with the following regimens highlighted: isoniazid-rifampicin (HR, dark blue), isoniazid-rifampicin-ethambutol-pyrazinamide (HREZ, light blue), bedaquiline-pretomanid-linezolid (BPaL, red) and bedaquiline-pretomanid-pyrazinamide (BPaZ, purple). b, Areas under the kill curve for M. tuberculosis drug regimens, averaged across three M. tuberculosis starvation conditions (mean ± s.e.m. c, Drug regimens were previously classified as similar to standard-of-care (SOC) or better than SOC on the basis of their performance in relapsing mouse models (RMM), bactericidal mouse models (BMM) of common mouse strains, the granulomatous C3HeB/FeJ mouse strain and in clinical studies30,31. Time–kill curves (AUC averaged across three M. tuberculosis starvation models) of similar-to-SOC drug regimens were compared with better-than-SOC regimens using a two-sided Mann–Whitney U-test. Boxplots show the median, interquartile range and total range (central line, box and whiskers, respectively) of AUC values (RMM: n = 46, P = 2.7 × 10−5; BMM common strains: n = 48, P = 0.0015; BMM C3HeB/FeJ: n = 15, P = 0.0047; clinical bactericidal activity: n = 14, P = 0.020). d, Median MICs of drug regimens, calculated from the MICs of individual drugs (averaged from mc27000 and H37Ra strains), were compared between similar-to-SOC and better-than-SOC regimens. Sample sizes, statistical tests and box plots are identical to those described in c. BgMM, BMM C3HeB/FeJ. To evaluate ASCT's ability to predict infection outcomes, we used established classifications of M. tuberculosis drug regimen efficacy, categorizing them as either similar to standard-of-care or better than SOC, on the basis of mouse and clinical studies30,31. While time–kill kinetics of growing M. tuberculosis did not discriminate between similar-to-SOC and better-than-SOC drug combinations, we found strong associations between in vivo outcomes and in vitro killing under starvation conditions, independent of whether quantified by AUC or live-cell fractions at single timepoints (Fig. These findings suggest that non-growing, metabolically less active bacteria, commonly found in intra- and extracellular infection niches, are more challenging to clear and probably responsible for long-term infection outcomes32,33,34,35. For example, killing of pantothenate-starved mc27000 (the auxotrophic M. tuberculosis strain H37Rv ΔpanCD ΔRD1) was highly associated with outcomes in multiple mouse models, including the relapsing mouse model, the bactericidal model of common mouse strains and the C3HeB/FeJ strain, mirroring different outcomes (bacteriologic burden and relapses) and host pathobiology36,37. Moreover, in vitro mc27000 killing upon pantothenate starvation was associated with clinical outcomes in phase 2 clinical studies, as measured by bactericidal activity during treatment31. Similar results were obtained with 14-day phosphate-buffered saline (PBS) starvation of the mc27000 and H37Ra M. tuberculosis strains. Overall, 11 out of 12 associations between M. tuberculosis killing under starvation and clinical outcomes reached statistical significance. In contrast, c.f.u.-based killing assessments were not associated with in vivo outcome measures (Extended Data Fig. Because MICs within combinations may shift due to drug–drug interactions, we systematically measured all possible pairwise combinations among the 13 drugs tested (9 × 9 concentration matrices in quadruplicate for each of the 78 drug pairs, ~25,000 assays in total; Extended Data Fig. From these data we predicted high-order drug effects and compared them with Bliss expectations derived from single-drug activities38. Interactions were generally weak and not associated with in vivo outcomes, except for clinical trial data where antagonism correlated with regimens classified as better-than-SOC (Extended Data Fig. Logistic regression models combining MIC with interactions metrics showed only moderate predictive performance in the relapsing mouse model and no predictive value in the bactericidal mouse model (Extended Data Fig. Notably, all combinations contained at least one drug administered at concentrations above the MIC, highlighting the value of MICs for identifying active agents, even though regimens with very low MICs did not translate into improved outcomes. To complement ASCT, we also performed population-level resazurin assays as an alternative measure of antibiotic killing. As expected for a metabolic readout, starved M. tuberculosis showed lower fluorescence levels than growing cultures. Across all regimens, relative fluorescence changes correlated with ASCT-derived killing only under starvation, where modest associations with in vivo outcomes were also observed (Supplementary Fig. The performance of ASCT in predicting clinical study outcomes using M. tuberculosis starvation models yielded areas under the receiver operating curve (AUC-ROC) between 0.76 and 0.94 (Extended Data Fig. 4b), suggesting that the killing activity of M. tuberculosis regimens upon starvation is a critical in vitro marker that corresponds to in vivo efficacy. These analyses, based on non-clinical M. tuberculosis strains, show that time–kill dynamics in starved conditions can quantify regimen performance at the population level. While these findings highlight the potential of ASCT to prioritize antibiotic treatments in development, they do not extend to predicting individual patient outcomes. To investigate bacterial factors that influence antibiotic killing and individual infection outcomes, we next focused on different bacterial isolates. Due to the biosafety constraints of M. tuberculosis live-cell imaging, we studied M. abscessus, an emerging concern in developed countries. To evaluate how phenotypic and genetic diversity within a naturally evolved bacterial species correlate with antibiotic killing (that is, drug tolerance), we obtained a single bacterial isolate from each of 405 patients with M. abscessus lung infection across Europe and Australia39,40 (Fig. Using ASCT, we generated time–kill profiles for each clinical isolate and the M. abscessus laboratory strain against 8 antibiotics that are commonly used and recommended in clinical treatment41. In total, we assessed 18,244 time–kill curves and tracked ~130 million bacterial cells over 29 timepoints and 72 h. Because all clinical M. abscessus isolates of a given antibiotic condition were assessed simultaneously, they were frozen in logarithmic phase and exposed to drugs shortly after thawing, when they were probably in a lag phase. Isolate–drug pairs that exhibited bacterial growth (that is, drug resistance) or did not meet quality criteria were excluded from analyses (Methods). Our findings revealed highly diverse but reproducible time–kill kinetics under identical drug exposure, indicating that specific strain characteristics modulate antibiotic killing (Fig. a, Study centres in the United Kingdom, Ireland, mainland Europe and Australia. Maximum-likelihood phylogenetic tree of 376 M. abscessus isolates obtained from different patients, with each centre indicated by colour. b, Time–kill curves of 406 M. abscessus isolates across 8 antibiotics (the lower drug concentrations are shown), with viability distributions at 72 h, excluding growing isolates. c, Gompertz function fitting to optical density (OD) measurements (black dots), illustrating the determination of bacterial growth rates and lag times. The insert highlights lag time estimation from the Gompertz function and inoculum size on a logarithmic scale. d, Distributions of lag times (top) and growth rates (bottom) from 226 fitted M. abscessus growth curves. e, Pearson correlation analyses between drug tolerance (high drug concentration, continuous line; low concentration, dashed line), bacterial growth rate, lag time and the corresponding MIC. Drug tolerance and persistence have been linked to slow bacterial growth or prolonged lag times, associated with metabolic dormancy42,43. To evaluate how bacterial replication affects antibiotic killing, we measured optical densities of 226 M. abscessus strains during planktonic growth in nutrient-rich conditions and fitted Gompertz functions to provide model-based estimates of each isolate's lag time and growth rate (Fig. Apart from revealing a strong direct correlation between growth rate and lag time, reflecting a trade-off between bacterial growth and adaptation44, we identified correlations between growth rates or lag times and drug tolerance phenotypes (Fig. For instance, drug tolerance to imipenem and cefoxitin, which target cell wall synthesis specifically during bacterial replication, correlated with slow growth but not lag time. In contrast, moxifloxacin killing (targeting the DNA) was associated with longer lag times only. These associations were weak, explaining only up to 13% of the variation in the drug tolerance phenotype, suggesting that replication patterns were not the main driver of drug tolerance. However, potential technical factors, such as variability in growth assessments, curve fitting or isolate clumping, may have reduced the strength of these associations. We also found weak associations (R2 up to 0.11) between drug tolerance and resistance phenotypes (especially for cefoxitin and minocycline), mainly driven by outliers with MICs close to the tested concentration. Using a non-parametric test (Spearman correlation), most of these associations disappeared (Supplementary Fig. 2), supporting the idea that drug tolerance is a marker of antibiotic efficacy independent of drug resistance45. Drug tolerance phenotypes, especially for antibiotics targeting protein synthesis, were strongly associated, with some correlating weakly with MICs (Extended Data Fig. 8a), suggesting shared mechanisms or adaptive evolution reminiscent of collateral sensitivity and cross-resistance. While drug resistance is well established as a genetic trait, drug tolerance has traditionally been viewed as a largely phenotypic characteristic. Recent studies, however, have identified multiple genetic mechanisms underlying drug tolerance in mycobacteria and other bacterial species42,46,47,48. We employed whole-genome sequencing and estimated the proportion of phenotypic variance attributable to genetic variability, known as heritability49. Specifically, we mapped 1.3 million M. abscessus unitigs (sequences of variable length), which capture most genomic variation arising from vertical inheritance, horizontal gene transfer and mutations including single nucleotide polymorphisms, insertions, deletions and gene presence–absence, to respective phenotypes using linear mixed models50,51. Resistance phenotypes for several antibiotics (particularly macrolide MICs) were strongly determined by bacterial genetics, whereas others, such as imipenem and cefoxitin, showed low heritability, probably reflecting chemical instability and limited biological variation in MIC measurements (Fig. Across all drugs, tolerance phenotypes demonstrated substantial heritability (32–97%), far exceeding random chance (1.1%, Student's t-test P < 2.2 × 10−16), indicating that killing phenotypes are strain-specific and largely genetically determined. a, Heritability estimates for drug resistance (MIC; n = 221) and drug tolerance phenotypes (AUC; n = 371, excluding growing isolates) based on 1.3 million genetic M. abscessus variants (unitigs) derived from whole-genome sequencing. CLR, clarithromycin; CLR*, inducible clarithromycin resistance; L, low drug concentration; H, high drug concentration. b, Maximum-likelihood phylogenetic tree of 353 M. abscessus isolates, aligned with the tigecycline tolerance heat map (high concentration) and mutational macrolide and aminoglycoside (Aminogl.) Purple lines mark clades with reduced tigecycline tolerance. Mapping these phenotypes to the M. abscessus phylogeny (Fig. 8b), we observed that some of the high- or low-tolerance phenotypes evolved in parallel, resulting in homoplastic traits, while others were conserved in phylogenetically related isolates, indicating inherited clades. One such example is a tigecycline low-tolerance clade within the dominant circulating clone of M. abscessus massiliense52. Many of these isolates carry high-level mutational aminoglycoside and macrolide resistance and have been linked to increased virulence40. Our finding of low tigecycline tolerance highlights a potential vulnerability in otherwise highly drug-resistant M. abscessus, where treatment success rates can fall below 20%. We then investigated whether strain-specific kill kinetics influence individual patient outcomes, given the limited direct evidence linking drug tolerance to treatment failures7. Our analysis showed that, among eight tested drugs, only macrolide MICs were associated with clinical outcomes (Fig. However, tolerance to amikacin, cefoxitin and imipenem, measured at low and high drug concentrations, correlated with M. abscessus clearance in patients (Extended Data Fig. Specifically, rapid bacterial killing was associated with favourable clinical outcomes, whereas high tolerance was linked to poor outcomes. Notably, the effect sizes of drug tolerance in predicting infection outcomes were comparable to those of macrolide MICs, currently the main in vitro measure guiding M. abscessus treatment decisions. Incorporating a single drug tolerance measure alongside macrolide resistance improved clinical outcome prediction, increasing the AUC-ROC from 0.69 to 0.78 (Fig. These findings underscore drug tolerance as an independent and clinically relevant marker of antibiotic efficacy. a, Comparison of M. abscessus drug tolerance (AUC, low concentration) and drug resistance (MICs) between isolates from patients with poor and favourable clinical outcomes, using a two-sided Mann–Whitney U-test (n = 135, excluding growing isolates). Boxplots show the median, interquartile range and total range (central line, box and whiskers, respectively) of AUC values. **P < 0.01, ***P < 0.001. b, Logistic regression predicting clinical outcomes from clarithromycin resistance (MIC) and imipenem tolerance (low concentration), individually and combined. Fifty ROC curves were generated by randomly selecting 80% of the samples for each condition. The mean AUC-ROC and 95% confidence intervals are shown. By applying principal component analysis to 5,056 M. abscessus time–kill curves and mapping antibiotics in drug tolerance space, we identified drug clusters related to the drug's mode of action (Fig. These clusters indicate that bacterial survival mechanisms are linked to the drug's target, probably driven by shared downstream effects, such as conserved stress responses that modulate antibiotic killing53. To explore the genetic basis of drug tolerance, we mapped ~280,000 M. abscessus genotypes (single nucleotide polymorphisms, insertions and deletions) extracted from whole-genome sequences of each isolate to drug tolerance phenotypes, using mixed-effects models corrected for population structure. This analysis identified multiple genotypes and genes strongly associated with drug tolerance phenotypes (Fig. Tolerance-associated genes grouped by the antibiotic target, with specific gene sets linked to protein synthesis, DNA and cell wall-active drugs. a, Principal component analysis of 8 drugs at 2 concentrations in drug tolerance space, based on Spearman correlations from 350 clinical M. abscessus isolates. b, Genome-wide association results for 16 drug tolerance phenotypes, highlighting the top 5 associated genes (moderate- or high-effect genotypes) per phenotype. The heat map shows association of 43 genes (top genotype per gene) with tolerance phenotypes using a mixed-effects model corrected for population structure (two-sided Wald test). Associations with P > 0.0001 are in white. c, Manhattan plot of 272,351 M. abscessus genotypes and their association with amikacin (low concentration) kill kinetics (two-sided Wald test), with the Bonferroni correction threshold (black line, 1.7 × 10−7). Multiple variants in MAB_0233 were strongly linked to amikacin killing (insert). d, Time–kill kinetics of control, ΔMAB_0233 and complemented strains (mean ± s.e.m., nine replicates per condition). AUC values of ΔMAB_0233 were compared with control or complemented strains using a two-sided Mann–Whitney U-test. P values for amikacin, linezolid and tigecycline were 4 × 10−5, except for tigecycline time–kill kinetics between ΔMAB_0233 and the control strain (P = 8 × 10−5). We further examined MAB_0233, a putative phage tail tape measure protein, in which several deletions were associated with drug tolerance to drugs targeting protein synthesis (Fig. As such, MAB_0233 could facilitate intracellular drug accumulation, enhance metabolic activity to potentiate drug effects, interfere with stress responses, or directly synergize with the drug. Using oligonucleotide-mediated recombineering followed by Bxb1 integrase targeting (ORBIT54), we generated a MAB_0233 knockout strain in M. abscessus, which showed increased tolerance for antibiotics targeting translation (amikacin, tigecycline and linezolid), but not the cell wall, while MICs remained stable (Fig. MAB_0233 complementation restored the low-tolerance phenotype for amikacin and tigecycline. However, for linezolid, the complemented strain retained an altered time–kill profile, suggesting partial complementation or the involvement of additional mechanisms influencing linezolid tolerance. Antibiotics that demonstrate activity in vitro may fail to achieve clinical success, highlighting a critical disconnect between microbiological testing and patient outcomes. This gap represents a major challenge in antibiotic development and patient care. To address it, we developed antimicrobial single-cell testing, a strategy that quantifies bacterial viability in real time and at large scale, providing new insights into an underexplored dimension of antibiotic activity. Applying ASCT, we demonstrate that drug-specific killing (in M. tuberculosis) and strain-specific killing (in M. abscessus) are key predictors of drug trial failures and poor clinical outcomes, and thus determinants of in vivo efficacy. This dual perspective of conserved drug effects and variable strain-specific responses shows how both components shape antibiotic killing and ultimately affect treatment outcomes. Several large-scale approaches to quantify bacterial viability have been recently developed, typically coupling a viability marker with an appropriate readout technology. Markers include conventional regrowth, metabolic dyes, biosensors and fluorescent reporters, while readouts range from bulk optical or fluorescent measurements to live-cell imaging, flow cytometry and label-free biophysical methods16,17. Although these strategies move beyond classical culture assays, most remain constrained by limited throughput, sensitivity or single-cell resolution, leaving critical aspects of bacterial survival unresolved. To overcome these limitations, we established a platform for repetitive, large-scale, real-time, single-cell assessments of bacterial viability across diverse clinical isolates. We used PI as a viability marker and confirmed that PI-positive cells do not regrow and can therefore be considered dead23. To further characterize PI-negative cells, we employed a reporter strain with inducible GFP expression and found that most PI-negative bacteria retained the ability to synthesize new proteins, in contrast to the very small fraction of bacteria that regrew. Notably, many PI-negative cells that initially produced GFP later died after drug washout. This post-exposure killing probably reflects incomplete drug removal, classical post-antibiotic effects or nutrient shock24,55,56. These findings indicate that conventional methods underestimate bacterial survival and highlight the importance of real-time, single-cell measures of bacterial viability. Still, defining true bacterial viability remains challenging, and PI may only partially capture it. We therefore focused on identifying phenotypes predictive of in vivo outcomes, a goal achieved with greater performance and throughput by ASCT through real-time single-cell PI quantifications. We applied ASCT to investigate the time–kill kinetics of multiple M. tuberculosis drug regimens across two strains and various experimental conditions. While combinations including the standard drugs isoniazid, rifampicin and ethambutol were most lethal under growth conditions, only killing under starvation predicted in vivo population outcomes. Antibiotic killing in these starvation conditions, reflective of metabolic inactivity in intra- and extracellular infection niches57, correlated strongly with outcomes in multiple in vivo models, including the relapsing and bactericidal mouse models as well as humans during phase 2 clinical trials. Thus, even using non-clinical, avirulent M. tuberculosis strains, we were able to predict in vivo outcomes of virulent tuberculosis at the population level, something not achievable with c.f.u. counts, MICs, drug interaction measures or resazurin-based assessments. Importantly, our data also show that not only the extent of killing matters, but also the physiological bacterial state in which killing is assessed, highlighting the importance of infection-relevant contexts when evaluating killing dynamics. Extending this approach to more host-like conditions may further improve its predictive performance. Given the high attrition rates in antibiotic development and clinical trials58, there is an urgent need for more efficient strategies to prioritize therapeutic regimens, which is a challenging task, considering the growing number of active lead compounds and possible combinations59. By enabling the evaluation of thousands of drug combinations in vitro—with demonstrated relevance in vivo—ASCT has the potential to identify more effective and much shorter treatment regimens against M. tuberculosis infection and disease. This advancement could accelerate global health initiatives, such as the World Health Organization's goal to eliminate most tuberculosis burden by 203512,60. Highly effective antibiotics are the cornerstone of successful antibiotic treatments, yet clinical outcomes vary widely, with some patients responding to short courses and others failing despite prolonged multidrug treatment61. We show that antibiotic killing is highly variable across clinical M. abscessus isolates, further challenging the prevailing clinical view that bacterial killing is solely determined by the drug's bactericidal properties62. This variability in killing, different from drug resistance, highlights drug tolerance as a fundamental bacterial trait that may greatly affect treatment efficacy. Notably, there exists scarce evidence that drug tolerance, rather than limited drug penetration or reinfections, underlies antibiotic treatment failures63,64,65. We found that M. abscessus strains poorly killed by amikacin, cefoxitin or imipenem (in high and low drug concentrations) were linked to worse clinical outcomes in individual patients, providing some of the most compelling evidence so far that drug tolerance is a critical determinant of treatment success. These findings indicate that tolerance phenotypes could complement conventional drug susceptibility testing to improve outcome prediction and clinical decision-making. We also show that drug tolerance (such as resistance) is largely determined by the bacterial genetic background and, thus, a heritable and evolvable bacterial trait. For example, within the subspecies M. abscessus massiliense, we identified a low-tolerance clade among highly resistant strains, revealing vulnerabilities that could be exploited to enhance bacterial clearance. Tolerance phenotypes clustered with the drug's mode of action, suggesting conserved survival pathways among drugs targeting similar cellular functions. Using phenogenomic analysis, we identified numerous genes and genotypes associated with high- or low-tolerance phenotypes. Through gene knockout and complementation experiments, we further validated a phage protein that modulates bacterial killing. These tolerance mechanisms represent potential targets for sterilizing antibiotic treatments, while tolerance genotypes could be integrated into diagnostics, analogous to molecular susceptibility testing. Presumably, molecular testing, or phenotyping via ASCT, could facilitate individualized, tolerance-tailored antibiotic regimens and help improve patient outcomes beyond the development of new drugs and regimens. Because PI measures membrane compromise, it directly reflects killing by cell wall damage but only indirectly captures other killing mechanisms. Since all dead cells eventually lose membrane integrity, PI therefore detects killing by non-lytic antibiotics only after a delay. In M. abscessus (but not in M. tuberculosis) we observed a delay of ~3 h for such drugs. Given the extended duration of our experiments and in vivo validation, this effect is probably minimal in mycobacteria but may become more relevant when applying ASCT to other bacterial species. While our findings highlight bacterial killing as a key determinant of antibiotic activity, clinical treatment outcomes are influenced by many additional factors. These include host immune responses, fluctuating drug exposures and penetration into tissues, drug interactions, toxicity and adherence, as well as the complexity of infection environments, such as intracellular niches and granulomas in mycobacterial infections, or acidic compartments with altered drug activity (for example, pyrazinamide)34,66,67. ASCT does not capture these non-bacterial determinants and therefore cannot fully predict treatment outcomes, but it provides a scalable and biologically meaningful readout of bacterial susceptibility that complements existing tools. We validated ASCT in mycobacteria, but this approach is probably applicable to other bacterial species and multidrug-resistant pathogens. However, adjustments in experimental design, imaging setup and analysis will be required to account for the species-specific morphology and physiology, particularly in regard to viability assessments. The in vivo relevance of drug tolerance and nutrient limitation is also likely to vary across pathogens and infection sites. In our study we interrogated antibiotic killing at two levels: first, whether drug-specific killing in M. tuberculosis correlates with in vivo outcomes at the population level; and second, whether strain-specific variation in killing across clinical M. abscessus isolates associates with individual patient responses. Although these approaches differ (drug regimen versus strain), both converge on the same principle that killing dynamics are critical determinants of treatment success. While we examined these factors independently, clinical treatment responses are shaped by the interplay of drug properties and bacterial behaviours. By studying millions of single-cell fates across hundreds of conditions, our approach provides a scalable framework to translate in vitro killing into in vivo efficacy, opening new avenues for drug development and personalized therapy. Clinical M. abscessus isolates were obtained from respiratory samples (sputum or bronchoalveolar lavage fluid) collected from patients with pulmonary M. abscessus infection as described previously40. All work with clinical M. abscessus isolates and avirulent M. tuberculosis strains was conducted in a biosafety level 2 (BSL-2) facility, in accordance with institutional biosafety regulations. Included patients came from all major cystic fibrosis centres in the United Kingdom, the Republic of Ireland (Dublin), Sweden (Gothenburg), Denmark (Copenhagen and Skejby), the Netherlands (Nijmegen) and Australia (Queensland). M. abscessus isolates were retrieved from the original mycobacterial growth indicator tubes (MGIT) or, if otherwise unavailable, from subcultured isolates. The study was approved in England and Wales by the National Research Ethics Service (12/EE/0158) and the National Information Governance Board (ECC 3-03 (f)/2012) and in other centres by respective local review boards. All consumables are provided in Supplementary Table 4. Planktonic growth rates and the lag times of M. abscessus isolates were assessed following suspension in nutrient-rich media. Approximately 105 c.f.u.s of M. abscessus were inoculated in 2.5 ml of Middlebrook 7H9 medium (supplemented with 0.4% glycerol, 10% OADC [oleic acid, albumin, dextrose, catalase] and 0.05% Tween 80) in 15 ml glass tubes, which were then incubated at 37 °C with 150 r.p.m. Optical densities (OD565) were measured with a densitometer (DEN-1B; Biosan) and quantified in McFarland units. OD565 readings were performed every 12 h for a minimum of 5 days until the readings stabilized for at least 24 h (change in McFarland less than 0.5). Background-corrected OD565 measurements were used to fit Gompertz functions to estimate growth rates, where N is the number of bacteria at a given time (t), with A representing carrying capacity (maximum population size), B the initial growth factor affecting displacement along the y axis and C, the growth rate constant. Lag times were calculated from the fitted Gompertz function and the known inoculum size. The lag time (λ) was calculated as: Antibiotic resistance was quantified using MICs, following the Clinical Laboratory Standards Institute (CLSI; M24 3rd edition) guidelines70. received training at the National Center for Mycobacteria, Zürich (Switzerland), to align with standard clinical microbiology practices. M. abscessus isolates were cultured in Middlebrook 7H9, supplemented with 0.4% glycerol, 10% OADC and 0.05% Tween 80, for 3 days and then diluted in PBS to a McFarland standard of 0.5. A 140 µl aliquot of this suspension was mixed into 14 ml cation-adjusted Mueller–Hinton broth (CAMHB) to achieve an approximate density of 106 c.f.u.s ml−1. For each experiment CAMHB was freshly prepared. Antibiotics and CAMHB were added to 96-well plates to reach a volume of 50 µl per well. After mixing, the 50 µl bacterial suspension was added to each well to obtain a final bacterial concentration of ~5 × 105 c.f.u.s ml−1. Each plate included growth and media control wells. Mycobacterium peregrinum (ATCC-700686) MICs were assessed for quality control in each experimental batch. Azithromycin and clarithromycin conditions were reassessed after 14 days of incubation to evaluate inducible macrolide resistance. MICs were recorded as the lowest drug concentration preventing visible mycobacterial growth. M. tuberculosis drug susceptibility testing was done analogous to M. abscessus, except that M. tuberculosis isolates were cultured in 5 ml of Middlebrook 7H9 broth, supplemented with 0.4% glycerol, 10% OADC and 0.05% Tween 80 in 50 ml tubes (M. tuberculosis mc27000 was additionally supplemented with 100 µg ml−1 pantothenate), and that MIC plates were assessed after 14 days. M. tuberculosis mc27000 was cultured in 10 ml of Middlebrook 7H9 broth supplemented with 0.4% glycerol, 10% OADC, 0.05% Tween 80 and 100 µg ml−1 pantothenic acid. Cultures were incubated at 37 °C with shaking (150 r.p.m.) until mid-log phase, then washed and resuspended in PBS before incubation at 37 °C for 14 days. PBS-starved bacteria where standardized to ~107 c.f.u.s ml−1, and 180 µl was dispensed into 96-well U-bottom-plates. Each condition was tested in triplicate. Serial dilutions were performed, and 5 µl of each dilution were spotted onto 7H11 agar plates supplemented with 100 µg ml−1 pantothenic acid (in duplicate). In addition, 50 µl of undiluted samples were washed and directly plated on pantothenate-supplemented 7H11 agar. Plates were incubated at 37 °C until visible colonies emerged. The in vivo classifications of M. tuberculosis drug regimens ‘as good or worse than SOC (where isoniazid-rifampicin-pyrazinamide-ethambutol or isoniazid-rifampicin-pyrazinamide were considered SOC)' or ‘better than SOC' were obtained from two previous studies. In vivo classifications based on mouse models were obtained from ref. 30, and classifications based on phase 2a and 2b clinical trials (assessing bactericidal activity) were obtained from ref. All similar-to-SOC or better-than-SOC drug combinations from the relapsing mouse model (n = 46) and clinical studies (n = 14) were assessed with ASCT. These drug combinations and classifications of single drugs were also used to evaluate the performance of ASCT in bactericidal mouse models (common mouse strains: n = 48; C3HeB/FeJ mouse strain: n = 15). Clinical metadata of patients with respiratory M. abscessus infection were available for a subset of patients. Treatment outcomes were assessed in patients meeting the ATS/IDSA criteria of NTM pulmonary disease72. Patients with positive cultures after 6 months of M. abscessus treatment were considered as having persisting infections, and patients with sustained culture conversion (microbiological clearance) as cleared infections. Growing isolates were excluded from the analysis of bacterial killing. The discriminative performance for predicting the success of drug regimens in mouse or patient populations (M. tuberculosis) and the clinical outcome in individual patients (M. abscessus) was assessed with logistic regression and quantified with AUC-ROC. For each condition, 50 bootstrap iterations were performed by randomly selecting 80% of the samples. Sensitivity, specificity, positive and negative predictive values, and F1 scores were calculated at the optimal cut-off determined by the Youden index. M. abscessus and M. tuberculosis isolates were cultured in 5 ml of Middlebrook 7H9 broth, supplemented with 0.4% glycerol, 10% OADC and 0.05% Tween 80 in 50 ml tubes (M. tuberculosis mc27000 was additionally supplemented with 100 µg ml−1 pantothenate). For PBS starvation, mid-log phase bacteria (H37Ra or M. tuberculosis mc27000) were washed and resuspended in PBS, the 50 ml tube filled with PBS and incubated for 14 days at 37 °C without shaking. For pantothenate starvation, M. tuberculosis mc27000 (H37Rv ΔpanCD ΔRD1) was incubated for 14 days with Middlebrook 7H9, supplemented with 0.4% glycerol, but not pantothenate. After growth or starvation, mycobacteria were centrifuged at 3,000 g for 10 min, and the resulting pellet was resuspended in Middlebrook 7H9 (for growth conditions and pantothenate starvation), in Middlebrook 7H9 with pantothenate supplementation (for growth conditions in M. tuberculosis mc27000) or in PBS (for PBS starvation). To achieve single-cell suspensions, large clumps were removed by low-speed centrifugation (200 g, 3 min), followed by serial filtration of the supernatant through 5 µm and 1.2 µm filters (Sartorius Minisart). In every sample, bacterial densities were assessed using OD565 measurements, and the live-cell fraction was quantified using propidium iodide staining and imaging. Bacterial samples with a live-cell fraction below 95% were discarded and reprocessed. Single-cell suspensions were immediately used (M. tuberculosis) or frozen at −80 °C (M. abscessus clinical isolates). ASCT uses a dual-layer approach: the first layer consists of an agar pad containing and immobilizing bacteria, and the second layer comprises drug-containing solutions (Fig. To prepare the agar pad, ultra-low gelling temperature agarose (ULGA; Lonza SeaPrep) and Middlebrook 7H9 were dissolved in hot dH2O. Additional dissolution was facilitated through heating, where any evaporated water was replaced to maintain constant 7H9 and agarose concentrations. Once the solution cooled to 50 °C or lower, OADC and glycerol were added. The mixture was then filtered through a 0.22 µm filter and propidium iodide was added. This agarose solution was aliquoted as needed. The final gel pad solution contained 0.4 agarose, 1× Middlebrook 7H9, 10% OADC, 0.4% glycerol, 8 µg ml−1 propidium iodide and ~5 × 106 bacteria per ml. For non-starving M. tuberculosis mc27000, the agar pad also contained 100 µg ml−1 pantothenate. In pantothenate-starvation conditions, the gel pad did not contain pantothenate, and in PBS-starvation media, all supplements were replaced with 1× PBS. All preparations were made in 96-well plates using a 96-channel pipette (Mini96, Integra Biosciences). Of the isolate–agar–PI solution, 7 µl was dispensed, using liquid handling (I.DOT, Dispendix), into each well of a 1,536-well plate (Greiner, Screenstar). At least one column and row were left as buffer wells at the edges. All materials used in the agarose preparation, including chemicals, pipette tips, filters, the plate and so on, were preheated to 37 °C. This step was followed by cold centrifugation at 4 °C and 1,500 g for 20 min to solidify the ULGA. After leaving the plate for 30 min at room temperature, a defined drug solution (4 µl) dissolved in Middlebrook 7H9 or PBS (for PBS starvation) was added to each well. The plate was then sealed with parafilm, covered with a lid and remained for another 60 min at room temperature before being placed into the microscope's preheated live-cell chamber (37 °C; Life Imaging Services). Images were captured with a Nikon Ti2-E inverted microscope using a CFI Plan Apo Lambda ×40 NA 0.95 or a CFI Plan Apo Lambda ×100 NA 1.45 objective, and the Perfect Focus System for maintenance of the focus over time. PI fluorescence was excited with a Spectra III Light Engine (Lumencor) at 555 nm and collected with a penta-edge 408/504/581/667/762 dichroic beam splitter and a 440/40, 520/21, 606/34, 694/34, 809/81 penta-bandpass filter. For ×40, the fluorescence excitation light intensity was set to 10% and the camera exposure time to 20 ms, and the brightfield illumination light intensity was set to 50% and the camera exposure time to 8 ms. For ×100 acquisition, the excitation light intensity and camera exposure were adjusted accordingly. The NIS JOBS module was used for image acquisition automation. Briefly, 9 equally spaced fields of view for each well and imaging timepoint were acquired. For M. abscessus, images were captured every 2.5 h for a total duration of 72 h and 29 timepoints; for M. tuberculosis, images were acquired every 4 h for a duration of 168 h (42 timepoints in total). Imaging one timepoint took ~130 min (1,260 wells with 9 fields of view per well). At the end of image acquisition, imaging data (M. abscessus: up to 11,340 movies and 7.8 TB per experiment; M. tuberculosis: up to 11 TB per experiment) were transferred to the high-performance computing cluster at the University of Basel (sciCORE) for subsequent image and data analysis. For drug tolerance profiling, ASCT was performed on all clinical M. abscessus isolates using a standardized panel of eight antibiotics, regardless of the treatment each patient received. M. abscessus isolates were assessed within a single ASCT plate for a single antibiotic condition in triplicate. The drug concentrations used were based on the MICs for ATCC-19977, determined similar to CLSI criteria71. In contrast to CLSI, they were assessed in 7H9 supplemented with 0.4% glycerol, 10% OADC and 0.05% Tween 80 and quantified using OD600. Growth inhibition at increasing drug concentrations was fitted with a four-parameter log-logistic model using the R drc package73. Antibiotic time–kill kinetics for all 8 drugs were evaluated at 2 different drug concentrations: a lower concentration (10-fold the MIC of ATCC-19977) and a higher concentration (20-fold the MIC of ATCC-19977). Image processing of every image time-lapse (all imaging timepoints of a single field) consists of five individual image analysis steps: background control, cell segmentation, cell classification, drift correction and cell tracking. To increase the accuracy of fluorescence intensity quantifications, we applied BaSiC, a method that automates background correction through low-rank and sparse decomposition via Fiji19,74. This method adjusts for spatial and temporal variations in background fluorescence, which commonly arise from uneven and repetitive illumination. Employing the default BaSiC settings, we achieved consistent fluorescence signals across different conditions, well positions and time (Extended Data Fig. Bacteria were segmented using a combined pixel and object classification approach implemented in ilastik20. In the pixel classification task, each pixel was subjected to supervised random forest classifiers (100 trees) to differentiate between the background and cellular structures. Within the object classification task, the pixel classification map, estimating the probability of each pixel belonging to the background or cellular class, was smoothed and thresholds were applied to generate object features such as intensity statistics and shape descriptors. These features were used to train the following object classifiers: PI+ single cells, PI− single cells and clumps. Pixel and object classifiers were trained on ~30 time-lapses, including different M. abscessus strains and antibiotic conditions. These classifiers were then applied to all data (over 200,000 time-lapses) in batch mode. To assess the fate of PI+ and PI− bacteria upon antibiotic washout, we trained bacteria with ambiguous PI signals as an additional object class. This class was excluded from single-cell growth assessments. To validate segmentation accuracy, 3 randomly selected time-lapse datasets were manually annotated (with automated support) to generate ground-truth pixel prediction maps. The same time-lapses were then processed by ASCT pixel classification. Ground-truth and ASCT pixel classifications were compared using the ImageJ CLIJ2 plugin74. Segmentation accuracy (that is, object overlap) was quantified with the Jaccard Index (Extended Data Fig. To quantify PI classification accuracy, 1,600 PI-positive and 1,600 PI-negative single cells were manually annotated within 5 timelapse datasets, including different M. abscessus isolates and treatment conditions. The ASCT PI classification algorithm was than compared with manually annotated objects using the caret R package75 (Extended Data Fig. To optimize downstream analysis and especially bacterial tracking, we corrected for any drift in imaging fields, which mainly occurred at the beginning of each experiment due to thermal changes (thermal drift). Drift correction for large-scale time-lapse data is computationally expensive. We extracted a window of 600 × 600 pixels from the binary map of segmented objects in frame 0 (centred at c1 = (800,800)), the very first image of a time-lapse. We also extracted a window of 600 × 600 pixels from frame 2 and shifted this section, one pixel at a time, up to 200 pixels away from c1 in each direction, creating 401 × 401 windows. The windows from frame 1 were overlaid on the frame 0 window to find maximum alignment. Given that cells are not always uniformly distributed or can lose focus, the same strategy was performed with two other windows (c2 = (1,300,1,300) and c2 = (1,700,1,700)). From frame 3 onward, windows were shifted by fewer pixels to achieve a 75% alignment likelihood between the frames. The shifts were initially set to 25 pixels and consecutively adjusted to 25, 50, 100 and 200 pixels if the alignment likelihood was not achieved. To automatically follow the behaviours of individual cells during the experiment, we established a custom script to track segmented objects across imaging frames. For each frame, we extracted object features from the segmentation output obtained via brightfield imaging using the MATLAB function ‘regionprops'. Our cell-tracking algorithm aims to link corresponding objects between the initial frame 0 and subsequent frames, with nt denoting an object n at timeframe t. Object homology H between frames was determined by comparing features of individual objects. A graph of linking objects (nt, n0) at frame t was represented by a 3-dimensonal tensor H(n0, n0 + N, t), where N0 is the population of objects at frame 0. All possible homology values were integrated into an object matrix. Rather than comparing each frame's object to find the closest match, we employed a comprehensive local minimum strategy. This process was iteratively repeated to identify and remove further links until a homology threshold of 1 × 108 was reached. This local minimum strategy allowed individual objects to be linked throughout the time-lapse to generate single-cell trajectories. Wells with less than 1,000 tracked bacteria for M. abscessus and less than 500 tracked bacteria for M. tuberculosis were discarded. After quantifying the morphology and viability trajectories of each bacterium, the imaging data were further analysed. Bacterial growth in ASCT was defined as an increase in total object area greater than 3.1-fold during antibiotic exposure. This threshold was first defined by comparing growing and non-growing cells and then validated in M. abscessus isolates with and without inducible macrolide resistance during azithromycin exposure (Extended Data Fig. If growth was detected in two-thirds or more of the replicates for a given isolate–drug condition, the condition was considered growing and removed from the killing analyses. The reproducibility of M. abscessus time–kill kinetics was assessed by comparing live-cell fractions at distinct timepoints (3, 6, 9, 12, 24, 36, 48, 60 and 72 h). Exact live-cell fractions at given timepoints were interpolated from overall time–kill kinetics. The coefficient of variation (CoV) was calculated for each triplicate at every timepoint. All triplicates were included in downstream analyses if the mean CoV of all 3 replicates fell within 3 s.d. of the overall mean triplicate CoV distribution per drug condition. Triplicates exceeding this threshold were reduced to the best-performing duplicate. These duplicates were retained if their mean duplicate CoV was within 3 s.d. of the overall mean duplicate CoV per drug condition; otherwise, the isolate–drug pair was excluded from further analyses. A minimum of 2 reproducible time–kill curves per isolate–drug condition was required for downstream analyses of M. abscessus kill analyses. To assess for outliers in our M. abscessus drug tolerance assessment, we performed principal component analyses for every antibiotic condition on the basis of live-cell fractions at 3, 6, 9, 12, 24, 36, 48, 60 and 72 h. Isolate–drug conditions with a distance of more than 3 s.d. from the mean, assessed with Mahalanobis distance and calculated using the first 2 principal components, were excluded from further analyses. In total, 957 (14.7%) M. abscessus isolate–drug pairs were excluded from time–kill analysis due to bacterial growth (mostly azithromycin and minocycline) and 156 (2.4%) due to poor reproducibility or being outliers. Imaging data were acquired consecutively for each well at different timepoints. The live-cell fraction at 0 h (timepoint 0) was extrapolated by fitting a logistic function to the live-cell fractions from the first 5 imaging frames, corresponding to ~13 h (all conditions except imipenem and cefoxitin). Due to the rapid killing observed in imipenem and cefoxitin conditions, resulting in inaccurate fitting, the maximum live-cell fraction observed for each isolate in any of the other drug conditions was used as the isolate-specific live-cell fraction at 0 h. For M. abscessus, isolates with a live-cell fraction below 80% and isolates that had less than 1,000 cells per well across multiple wells were excluded from further analyses. Using the 0-h live-cell fraction, all frames were normalized to an initial live-cell fraction of 100%. Overall, antibiotic killing was quantified as the arithmetic mean of the area under the 72-h (M. abscessus) or 168-h (M. tuberculosis) time–kill curve, ranging from 1 to 0 (maximum survival to most rapid killing, respectively). We assessed the relationship between M. abscessus drug tolerance phenotypes, growth rates, lag times and MICs using Pearson correlation and R2 values. Spearman correlation was employed to examine the contribution of outliers. For drug clustering in drug tolerance space, a Spearman correlation matrix was generated on the basis of pairwise comparisons of the area under the time–kill curves from 350 clinical M. abscessus isolates. Only isolates with 4 or less missing drug tolerance values (out of 16) were used. We also applied the tracking algorithm to identify whether single cells are able to grow (form microcolonies) after antibiotic washout. To achieve tracking accuracy and computational efficacy, we employed a backward approach. We identified and categorized objects with a size 5–15 times the median size of single cells as microcolonies and recorded the area, appearance time and centroids. Analogous to the homology matrix to track single non-growing cells, we used the homology index to track microcolonies backwards to the final single-cell object, that is, the initial originating cell. We focused on two features: the object area, which is important particularly to track large objects, which grow and potentially move; and centroids, which are critical to track single non-moving cells. With this approach, we could determine whether specific objects generated microcolonies. DNA libraries were constructed with unique identifiers for each isolate and sequenced using multiplexed paired-end sequencing. De novo genome assemblies were assessed for quality. Assemblies with a length longer than 6 Mb, more than 300 contigs, an average depth below 30×, a coverage of the reference genome below 50%, or presumed mixed infection were discarded. Sequence reads were mapped to the M. abscessus ATCC-19977 genome using BWA, followed by INDEL realignment77,78. Single nucleotide polymorphisms (SNPs) and small insertions/deletions (INDELs) were identified using bcftools and annotated with SNPeff79,80. SNPs were filtered to require a minimum base call quality of 50, a minimum mapping quality of 20, and at least 8 matching reads covering an SNP (3 per strand). The coverage of these regions in clinical isolates was assessed with sambamba81. Large deletions were defined as 2 consecutive windows with a mean coverage of 5× or below, occurring in at least 5% of all genomes. Variants with identical distributions were collapsed into a single variant. Maximum-likelihood trees were generated using FastTree, inferred from core SNPs and visualized with iTOL82,83. Mutational aminoglycoside resistance was evaluated with mutations in the rrs genotype and macrolide resistance with mutations in rrl84,85. Unitigs for all isolates were extracted using the unitig-caller tool, which utilizes an FM-index built around the Bifrost API51,86. A similarity matrix was generated from phylogenetic distances, which was then used to correct for population structure. Using pyseer, FaST linear mixed models were applied to estimate narrow-sense heritability (h2), representing the proportion of variance in the phenotype attributable to genetic variation50. Random chance was assessed by shuffling each drug tolerance phenotype across M. abscessus isolates and calculating heritability (10 times for each drug tolerance phenotype; 160 times in total). We performed genome-wide association studies (GWAS) to analyse ~300,000 M. abscessus genetic variants, including SNPs, INDELs and large deletions, in relation to drug tolerance phenotypes. Variants were classified by presumed genetic effects: low effect (intergenic variants, synonymous SNPs), moderate effect (non-synonymous SNPs, inframe INDELs) and high effect (frameshift variants, start/stop alterations, large deletions). We applied linear mixed models to account for population structure, integrating a relatedness matrix, and quantified associations using the Wald test39,87. We used a Bonferroni threshold of 1.7 × 10−7 to control for multiple hypothesis testing. To summarize GWAS hits, we extracted the top 5 genes showing the strongest association per phenotype (for moderate- or high-effect variants). The top associations of these genes with all tolerance phenotypes were then shown in a heat map (moderate- or high-effect variants). M. abscessus ATCC-19977 was transformed with pUV15tetORm (Addgene, plasmid 17975; gift from Sabine Ehrt) to generate a reporter strain expressing GFP under the control of the tetracycline-inducible promoter Pmyc1tetO89. Broth cultures were grown in Middlebrook 7H9 supplemented with hygromycin (1 mg ml−1) until mid-log phase, after which hygromycin was removed. Single-cell suspensions were generated as described for ASCT and treated for 12 h in non-shaking culture conditions with amikacin (26 µg ml−1), cefoxitin (63.1 µg ml−1) or moxifloxacin (8.5 µg ml−1). Following treatment, antibiotics were washed off and bacteria were transferred to the ASCT platform containing 2 µg ml−1 anhydrotetracycline to induce GFP expression. Brightfield and fluorescence images (PI and GFP) were acquired for 12 h. An untreated, induced control sample was used for GFP and PI gating. For each channel, log10-transformed fluorescence intensities were fitted with a 2-component Gaussian mixture model using the Mclust function. Data were visualized as hexbin density plots (100 bins) of PI versus GFP intensity with overlaid thresholds. To assess M. tuberculosis drug combinations, 13 drugs were combined in every pairwise combination using 9 × 9 checkerboards with log√2-fold dilution steps. Each condition was assessed using 4 technical replicates. Drugs were diluted in 7H9 medium, transferred to 384-well plates and inoculated with ~1.2 × 105 c.f.u.s of the H37Ra strain per well. After 21 days, OD600 was assessed and the bacterial fitness was determined as the background-corrected OD600 divided by the median OD600 of all growth control wells per plate. Single drugs and pairwise dose–response curves were fitted with a 4-parametric log-logistic model (drc R package). Pairwise interaction scores were determined using only the equipotent concentrations90 and further used to predict high-order interactions38. Deviations were expressed as log2 interaction strengths. To evaluate the combined predictive performance of interaction and MIC parameters (median MIC), a logistic regression model was implemented using the glm function in R. Models predicted binary outcomes (similar or better than SOC) in relapsing and bactericidal mouse models. Predictive performance was evaluated across 50 bootstrap iterations (randomly sampling 80% of the conditions), with mean ROC-AUC values assigned to each predictor. Statistical significance was evaluated using a z-test against the null hypothesis of random performance (AUC = 0.5). As an alternative proxy of antibiotic killing, we measured the reduction of resazurin to resorufin in M. tuberculosis mc27000 across all drug regimens. M. tuberculosis growth and starvation conditions were prepared as described for ASCT. Approximately 5 × 106 bacteria suspended in 49.5 µl 7H9 were dispensed per well into clear, flat-bottom 384-well plates and exposed to 0.5 µl drug solution. Drug regimens and concentrations were identical to those tested in ASCT. Each condition was tested in quadruplicate and included medium only and untreated bacterial controls. Plates were sealed with parafilm and incubated at 37 °C in a non-shaking incubator. At days 0, 7 and 14, selected plates were removed and 5 µl of a 0.13% resazurin stock solution was added to each well. After further incubation at 37 °C for 12 h, plates were briefly opened to reduce condensation before fluorescence was measured (excitation 560 nm, emission 590 nm) on a Synergy H1 plate reader (BioTek). Mean relative fluorescence units (RFUs) were background corrected. Antibiotic killing was quantified as the RFU change between days 0 and 14, which was then compared with ASCT-derived killing and in vivo outcomes. MAB_0233 knockout mutants (ΔMAB_0233) were generated on the M. abscessus ATCC-19977 background using ORBIT54. pKM444 (RecT-Int-expressing plasmid, Kanr) and pKM496 (for gene deletion by integration into target gene, Zeor) were gifted by Kenan Murphy (Addgene, plasmid 108319 and 109301)54. pMV261 (replicative vector for gene complementation) was obtained from NovoPro Biosciences91. To amplify plasmids, Escherichia coli DH5α cells were grown on LB agar plates supplemented with 50 μg ml−1 kanamycin (pKM444 or pMV261) or 50 μg ml−1 zeocin (pKM496). Liquid cultures of M. abscessus ATCC-19977 were incubated at 37 °C and 150 r.p.m. Cells were diluted to an OD600 of 0.03 in 20 ml of Middlebrook 7H9, supplemented with 0.4% glycerol, 10% OADC and 0.05% Tween 80 in a 250 ml baffled flask. The culture was placed at 30 °C and 100 r.p.m. for 18–20 h. The culture was then washed 3× with 20 ml ice-cold sterile 10% glycerol supplemented with 0.05% Tween 80 (washing solution). Following the third wash, the cells were collected by centrifugation and resuspended in 200 μl of washing solution. The ORBIT plasmid pKM444 (200 ng) was mixed with competent M. abscessus and rested on ice for 5 min. The cells were electroporated (2.5 kV, 1,000 Ω and 25 μF) using a 2-mm-gap-width electroporation cuvette. for 18–20 h. The recovered cells were pelleted at 3,000 ɡ (25 °C, 5 min), resuspended in 100 µl Middlebrook 7H9 medium and plated on 7H11 plates supplemented with 0.5% glycerol, 10% OADC and 250 μg ml−1 kanamycin. The plates were incubated at 30 °C for 3–4 days. Colonies resulting from pKM444 transformation were picked and grown on Middlebrook 7H11 agar plates supplemented with 0.5% glycerol, 10% OADC and 250 μg ml−1 kanamycin. Cells were collected by centrifugation and resuspended in TRIzol (Thermo Fisher). Cell disruption was performed by bead beating using Lysing Matrix B beads (200 µl) in a FastPrep-24 Classic instrument at 6.5 m s−1 for 60 s, followed by 60 s incubation on ice. This process was repeated for 3 cycles. The presence of the pKM444 plasmid was verified by PCR, using DreamTaq DNA polymerase (Thermo Fisher). Primers specific to the plasmid (forward: CACGTTGTGTCTCAAA ATCTC; reverse: CGATAACGTTCTCGGCTC) were used to amplify a target segment of the plasmid. PCR products were resolved by agarose gel electrophoresis, and bands of the expected size were excised and submitted for Sanger sequencing to verify plasmid presence. The targeting oligonucleotide (sequence for MAB_0233: CGGCCACATGTTTTGTGCCGCT AGGGGAAATCAGCTCGGCATCGTCCGTGTGCCGTTGTTGGTTTGTACCGTACACCACTGAGACCGCGGTGGTTGACCAGACAAACCCACATACCCTGATGCGAGTTCAACTGCCATCTGTTGCCCCCTTCGGGCAATCGGTGCAGC) was acquired from IDT with 4 nmole Ultramer DNA Oligo property and diluted in nuclease-free water to 1 μg μl−1. M. abscessus ATCC-19977, harbouring the pKM444 plasmid, was cultured in Middlebrook 7H9 medium with 5 µg ml−1 anhydrotetracycline (ATc). A 20 ml culture was prepared in a 250 ml baffled flask wrapped in aluminum, washed as described above and resuspended in 200 μl washing solution. Oligonucleotide (2 μl, 1 μg μl−1) guiding the payload plasmid pKM496 to the target gene was aliquoted into 1.5 ml microcentrifuge tubes and denatured at 95 °C for 5 min to prevent secondary structure formation. The denatured oligonucleotides were cooled on ice for at least 5 min before adding 200 ng pKM496 (ref. Subsequently, 200 μl of cells were added to the tube, gently mixed by pipetting and incubated for 5 min. Electroporation, recovery and plating were performed as described above, except that recovered cells were plated on 7H11 plates supplemented with 100 μg ml−1 zeocin. DNA extraction and PCR were performed as described using primers flanking MAB_0233 and pKM496 (forward: CGCTCACAACTGAATACCC; reverse: CCTGGTATCTTTATAGTCCTGTC). To complement ΔMAB_0233, the gene was amplified by PCR (KAPA HiFi, high-fidelity enzyme) and cloned into pMV261 downstream of the mycobacterial strong constitutive promoter, pHSP60 (ref. In addition, the T2 terminating region from E. coli rrnB was added to the end of the open reading frame (ORF) as an efficient terminator of transcription. To eliminate the pKM444 plasmid, which confers kanamycin resistance, the M. abscessus knockout strain ΔMAB_0233 was subjected to serial dilutions and growth cycles until cured. Cured strains were transformed with pMV261 following the same transformation protocol as described for pKM444. Complementary (c)DNA was prepared using the High-Capacity cDNA Reverse Transcription kit (Applied Biosystems). cDNA levels were quantified by quantitative real-time PCR (qPCR) on an Applied Biosystems qPCR machine using a PowerUp SYBR Green Master Mix (Thermo Fisher) and analysed by the ΔΔCt method. Gene expression was controlled using the housekeeping genes MAB_3009 (sigA) and MAB_ 3869c. Two sets of primers covering ~150 bp sequences of the beginning and the end of the MAB_0233 ORF were designed (primer_1_FWD: GCGAAGCCTTCGCCAAAGCTC, primer_1_REV: CCTCGTTGAGCTTTTCCAGCGC, primer_2_FWD: CGCAACGTCTCCGATGGGAAACC, primer_2_REV: GAGTTGAGCGGCGTCCATGC). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Whole-genome sequence data are available at the European Nucleotide Archive (ENA). Representative images to test ASCT are available on Zenodo at https://doi.org/10.5281/zenodo.17232777 (ref. The full raw imaging dataset is not publicly deposited due to size constraints but is available from the lead contact upon request. All original code has been deposited on GitHub at https://github.com/BoeckLab/ASCT and archived at Zenodo under https://doi.org/10.5281/zenodo.17775709 (ref. Darby, E. M. et al. Molecular mechanisms of antibiotic resistance revisited. Balaban, N. Q. et al. Definitions and guidelines for research on antibiotic persistence. Bigger, J. W. Treatment of staphylococcal infections with penicillin by intermittent sterilisation. Horne, D. & Tomasz, A. Tolerant response of Streptococcus sanguis to beta-lactams and other cell wall inhibitors. Balaban, N. Q., Merrin, J., Chait, R., Kowalik, L. & Leibler, S. Bacterial persistence as a phenotypic switch. & Helaine, S. Impact of bacterial persisters on their host. Kalil, A. C., Schooneveld, T. C. V., Fey, P. D. & Rupp, M. E. Association between vancomycin minimum inhibitory concentration and mortality among patients with Staphylococcus aureus bloodstream infections: a systematic review and meta-analysis. Somayaji, R. et al. Antimicrobial susceptibility testing (AST) and associated clinical outcomes in individuals with cystic fibrosis: a systematic review. Impact of antibiotic MIC on infection outcome in patients with susceptible Gram-negative bacteria: a systematic review and meta-analysis. Choi, S.-H. et al. Risk factors for recurrent Staphylococcus aureus bacteremia. Global Tuberculosis Report 2024 (World Health Organization, 2024). Four-month rifapentine regimens with or without moxifloxacin for tuberculosis. Marshall, J. E., Mercaldo, R. A., Lipner, E. M. & Prevots, D. R. Incidence of nontuberculous mycobacteria infections among persons with cystic fibrosis in the United States (2010–2019). Kwak, N. et al. Mycobacterium abscessus pulmonary disease: individual patient data meta-analysis. Stokes, J. M. et al. A multiplexable assay for screening antibiotic lethality against drug-tolerant bacteria. & Balaban, N. Q. TDtest: easy detection of bacterial tolerance and persistence in clinical isolates by a modified disk-diffusion assay. Lázár, V., Snitser, O., Barkan, D. & Kishony, R. Antibiotic combinations reduce Staphylococcus aureus clearance. Peng, T. et al. A BaSiC tool for background and shading correction of optical microscopy images. Berg, S. et al. ilastik: interactive machine learning for (bio)image analysis. & Desjardins, R. LIVE/DEAD® BacLightTM: application of a new rapid staining method for direct enumeration of viable and total bacteria in drinking water. & Nyström, T. Sorting out bacterial viability with optical tweezers. Maglica, Ž, Özdemir, E. & McKinney, J. D. Single-cell tracking reveals antibiotic-induced changes in mycobacterial energy metabolism. Fanous, J. et al. Limited impact of Salmonella stress and persisters on antibiotic clearance. Stover, C. K. et al. A small-molecule nitroimidazopyran drug candidate for the treatment of tuberculosis. Koul, A. et al. Diarylquinolines are bactericidal for dormant mycobacteria as a result of disturbed ATP homeostasis. Tweed, C. D. et al. Bedaquiline, moxifloxacin, pretomanid, and pyrazinamide during the first 8 weeks of treatment of patients with drug-susceptible or drug-resistant pulmonary tuberculosis: a multicentre, open-label, partially randomised, phase 2b trial. Cevik, M. et al. Bedaquiline-pretomanid-moxifloxacin-pyrazinamide for drug-sensitive and drug-resistant pulmonary tuberculosis treatment: a phase 2c, open-label, multicentre, partially randomised controlled trial. Systematic measurement of combination-drug landscapes to predict in vivo treatment outcomes for tuberculosis. Design principles to assemble drug combinations for effective tuberculosis therapy using interpretable pairwise drug response measurements. Active starvation responses mediate antibiotic tolerance in biofilms and nutrient-limited bacteria. The association between sterilizing activity and drug distribution into tuberculosis lesions. Extreme drug tolerance of Mycobacterium tuberculosis in caseum. Discovery of antibiotics that selectively kill metabolically dormant bacteria. Pan, H. et al. Ipr1 gene mediates innate immunity to tuberculosis. & Orme, I. M. Animal models of tuberculosis: an overview. Prediction of ultra-high-order antibiotic combinations based on pairwise interactions. Boeck, L. et al. Mycobacterium abscessus pathogenesis identified by phenogenomic analyses. Emergence and spread of a human-transmissible multidrug-resistant nontuberculous mycobacterium. US Cystic Fibrosis Foundation and European Cystic Fibrosis Society consensus recommendations for the management of non-tuberculous mycobacteria in individuals with cystic fibrosis. Fridman, O., Goldberg, A., Ronin, I., Shoresh, N. & Balaban, N. Q. Optimization of lag time underlies antibiotic tolerance in evolved bacterial populations. Basan, M. et al. A universal trade-off between growth and lag in fluctuating environments. An experimental framework for quantifying bacterial tolerance. Kreutzfeldt, K. M. et al. CinA mediates multidrug tolerance in Mycobacterium tuberculosis. Hicks, N. D. et al. Clinically prevalent mutations in Mycobacterium tuberculosis alter propionate metabolism and mediate multidrug tolerance. Antibiotic tolerance facilitates the evolution of resistance. Boosting heritability: estimating the genetic component of phenotypic variation with multiple sample splitting. Improved prediction of bacterial genotype–phenotype associations using interpretable pangenome-spanning regressions. A., Galardini, M., Bentley, S. D., Weiser, J. N. & Corander, J. pyseer: a comprehensive tool for microbial pangenome-wide association studies. Dissemination of Mycobacterium abscessus via global transmission networks. Murphy, K. C. et al. ORBIT: a new paradigm for genetic engineering of mycobacterial chromosomes. Hong, Y., Zeng, J., Wang, X., Drlica, K. & Zhao, X. Post-stress bacterial cell death mediated by reactive oxygen species. & Duncan, K. Evaluation of a nutrient starvation model of Mycobacterium tuberculosis persistence by gene and protein expression profiling. Wong, C. H., Siah, K. W. & Lo, A. W. Estimation of clinical trial success rates and related parameters. & Blaskovich, M. A. T. Antibiotics in the clinical pipeline as of December 2022. Silva, S., Arinaminpathy, N., Atun, R., Goosby, E. & Reid, M. Economic impact of tuberculosis mortality in 120 countries and the cost of not achieving the Sustainable Development Goals tuberculosis targets: a full-income analysis. Gelband, H. Regimens of less than six months for treating tuberculosis. Maier, L. et al. Unravelling the collateral damage of antibiotics on gut bacteria. Mulcahy, L. R., Burns, J. L., Lory, S. & Lewis, K. Emergence of Pseudomonas aeruginosa strains producing high levels of persister cells in patients with cystic fibrosis. Dissemination of persistent intestinal bacteria via the mesenteric lymph nodes causes typhoid relapse. Prevalence of antibiotic tolerance and risk for reinfection among Escherichia coli bloodstream isolates: a prospective cohort study. Adams, K. N. et al. Drug tolerance in replicating mycobacteria mediated by a macrophage-induced efflux mechanism. Santucci, P. et al. Intracellular localisation of Mycobacterium tuberculosis affects efficacy of the antibiotic pyrazinamide. Sambandamurthy, V. K. et al. Mycobacterium tuberculosis DeltaRD1 DeltapanCD: a safe and limited replicating mutant strain that protects immunocompetent and immunocompromised mice against experimental tuberculosis. Tjørve, K. M. C. & Tjørve, E. The use of Gompertz models in growth analyses, and new Gompertz-model approach: an addition to the Unified-Richards family. Woods, G. L. et al. Susceptibility testing of Mycobacteria, Nocardiae, and other aerobic actinomycetes. & Woods, G. L. Antimycobacterial susceptibility testing of nontuberculous mycobacteria. Griffith, D. E. et al. An official ATS/IDSA statement: diagnosis, treatment, and prevention of nontuberculous mycobacterial diseases. Ritz, C., Baty, F., Streibig, J. C. & Gerhard, D. Dose–response analysis using R. PLoS ONE 10, e0146021 (2015). Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Kuhn, M. Building predictive models in R using the caret package. Bryant, J. M. et al. Whole-genome sequencing to identify transmission of Mycobacterium abscessus between patients with cystic fibrosis: a retrospective cohort study. Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff. & Prins, P. Sambamba: fast processing of NGS alignment formats. Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. & Bork, P. Interactive Tree Of Life (iTOL) v4: recent updates and new developments. Genetic analysis of new 16S rRNA mutations conferring aminoglycoside resistance in Mycobacterium abscessus. Bastian, S. et al. Assessment of clarithromycin susceptibility in strains belonging to the Mycobacterium abscessus group by erm(41) and rrl sequencing. Holley, G. & Melsted, P. Bifrost: highly parallel construction and indexing of colored and compacted de Bruijn graphs. & Stephens, M. Genome-wide efficient mixed-model analysis for association studies. Pruim, R. J. et al. LocusZoom: regional visualization of genome-wide association scan results. Guo, X. V. et al. Silencing essential protein secretion in Mycobacterium smegmatis by using tetracycline repressors. Efficient measurement and factorization of high-order drug interactions in Mycobacterium tuberculosis. Stover, C. K. et al. New use of BCG for recombinant vaccines. Jovanovic, A. Demo data set for ASCT analysis. Jovanovic, A., Sadeghi, A., Muniz, S. E. C. & Boeck, L. Antimicrobial single-cell testing analysis workflows. We thank B. Schulthess and P. Sander for training in MIC assessments; A. Trauner, D. Pinschewer, M. Zampieri, A. Harms, D. Portevin, B. Aldridge, D. Bumann and U. Jenal for expert input; the scientific computing center at the University of Basel (sciCORE) and specifically R. M. Cabezón, I. M. de Ilarduya, G. Losilla and M. Jacquot for computational support; J. Sollier for critical reading and editing of the paper; the DBM microscopy core facility; and M. Roth and M. Tamm for lab space. Icons were obtained from PhyloPic (http://phylopic.org). received funding from the Wellcome Discovery award 226602/Z/22/Z and LifeArc/CF Trust Innovation Hub THUB01. These authors contributed equally: Alexander Jovanovic, Frederick K. Bright, Ahmad Sadeghi. Alexander Jovanovic, Frederick K. Bright, Ahmad Sadeghi, Basil Wicki, Santiago E. Caño Muñiz, Greta C. Giannini, Sara Toprak, Loïc Sauteur, Anna Rodoni, Andreas Wüst, Philippe Dehio, Michael Abanto & Lucas Boeck Swiss Tropical and Public Health Institute, Allschwil, Switzerland Gallipoli Medical Research, Greenslopes Private Hospital, Brisbane, Queensland, Australia Greenslopes Clinical School, Faculty of Health, Medicine and Behavioural Sciences, The University of Queensland, Brisbane, Queensland, Australia School of Medicine and Dentistry, Griffith University, Southport, Queensland, Australia Gold Coast University Hospital, Southport, Queensland, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors declare no competing interests. Nature Microbiology thanks Minsu Kim and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Overlay of manually annotated “ground truth” bacterial segmentation with automated object segmentation (pixel classification) of M. abscessus brightfield images. The Jaccard index quantifies overlay accuracy across imaging time points during time-lapse acquisition of 5,887 bacteria. Boxplots show the median, interquartile range, and data range within 1.5 x IQR (central line, box, and whiskers, respectively) of Jaccard indices. The dashed line indicates high segmentation accuracy (Jaccard index of 0.8). (b) Changes in dynamic range following BaSiC background correction. Mean PI values per single M. abscessus bacterium before and after correction over 72-hour time-lapses (first imaging time point: blue, last time point: red) shown in density plots. (c) Accuracy of automated PI classification in predicting “ground-truth” classes of PI-positive and PI-negative single cells. (d) Changes in single-cell count (which declines during growth) and object area (assessed with ASCT) over 72 hours under azithromycin treatment across 406 M. abscessus isolates. Isolates with (red) and without (black) inducible macrolide resistance are highlighted. (e) Number of single M. abscessus bacteria tracked over 72 hours per well within 1536-plates. Data from 406 M. abscessus isolates, and about 130 million tracked bacteria across eight drugs and two concentrations (data from Fig. (f) Fraction of viable M. abscessus immobilised in ASCT under nutrient-rich and starvation conditions without antibiotic exposure (dividing and PI-negative cells were considered viable). Yellow dots indicate the fraction of PI-negative bacteria in starved broth culture at 24 and 48 hours (no single-cell tracking). Effect of different (g) gel pad volumes and (h) gel pad agarose concentrations on M. abscessus time-kill kinetics. (a) Fraction of bacteria (including numbers) showing regrowth after 24-hour antibiotic treatment and antibiotic washout across PI-negative and PI-positive bacteria. (b-c) Induced GFP expression after antibiotic washout. (b) Image gating strategy for GFP and PI in untreated M. abscessus controls. (c) GFP and PI fluorescence intensities of M. abscessus populations treated with amikacin (AMK), cefoxitin (FOX), or moxifloxacin (MXF) following antibiotic removal. Time (hours) indicates imaging time points, with 0 set to the start of imaging ( ~ 1 hour after GFP induction). (a) ASCT-based time-kill kinetics of growing M. tuberculosis (H37Ra) exposed to 65 drug regimens, with the following regimens highlighted: isoniazid-rifampicin (HR; dark blue), isoniazid-rifampicin-ethambutol-pyrazinamide (HREZ; light blue), bedaquiline-pretomanid-linezolid (BPaL; red), and bedaquiline-pretomanid-pyrazinamide (BPaZ; purple). (b) Mean area under the kill curve for M. tuberculosis drug regimens under growth conditions, averaged across the two M. tuberculosis strains (H37Ra and mc27000). Each drug regimen was tested in nine technical replicates (poor-quality curves were excluded), and data are shown as mean ± SEM. (c-d) Killing dynamics (AUCs) of M. tuberculosis drug combinations were assessed in regard to the presence and absence of individual drugs. Drugs that increased killing under (c) growth conditions (averaged from mc27000 and H37Ra strains) or (d) starvation conditions (averaged from three starvation conditions) are shown. Groups were compared using a two-sided Mann-Whitney U test. Each dot indicates a drug regimen. Boxplots show the median, interquartile range, and total range (central line, box, and whiskers, respectively) of AUC values. (a) Time-kill curves (AUCs) upon growth and starvation conditions of two M. tuberculosis isolates were used to compare similar-to-SOC versus better-than-SOC classifications, based on outcomes in relapsing mouse models (RMM), bactericidal mouse models (BMM) of common mouse strains, the granulomatous C3HeB/FeJ strain and clinical studies30,31. Comparisons were performed using a two-sided Mann-Whitney U test (P values indicated). Each dot represents a drug regimen. Boxplots show the median, interquartile range and total range (central line, box, and whiskers, respectively) of AUC values. PA indicates pantothenate, PBS phosphate-buffered saline. (b) Performance of time-kill kinetics (AUC averaged across three M. tuberculosis starvation models) for predicting in vivo M. tuberculosis outcomes (similar-to-SOC versus better-than-SOC drug regimens). Using logistic regression, fifty ROC curves were generated for every condition by randomly selecting 80% of the samples. Mean area under the ROC curves and 95% confidence intervals are presented. (a) Colony forming unit (CFU) change of PBS-starved M. tuberculosis mc27000 during drug treatment. (b) Correlation of CFU-based killing of 65 drug regimens with ASCT-based time-kill kinetics, both assessed in the PBS-starved M. tuberculosis mc27000 strain and quantified with two-sided Pearson correlation. (c) CFU-based killing of PBS-starved M. tuberculosis mc27000 at day 3 and 7 was used to compare similar-to-SOC versus better-than-SOC classifications, based on outcomes in relapsing mouse models (RMM), bactericidal mouse models (BMM) of common mouse strains, the granulomatous C3HeB/FeJ strain and clinical studies30,31. Comparisons were performed using a two-sided Mann-Whitney U test (P values indicated). Each dot represents a drug regimen. The minimum MIC or median MIC of all drugs within each combination, or the mean MIC of the two most potent drugs (mean*), was assessed and compared between similar-to-SOC and better-than-SOC, based on outcomes in relapsing mouse models (RMM), bactericidal mouse models (BMM) of common mouse strains, the granulomatous C3HeB/FeJ strain and clinical studies30,31. Comparisons were performed using a two-sided Mann-Whitney U test (P values indicated). Each dot represents a drug regimen. (a) Pairwise M. tuberculosis drug-drug interactions of 13 x 12 drugs, each performed in 9x9 checkerboards, highlighting expected and observed growth inhibition. Pairwise interactions are quantified by the Loewe interaction score. B indicates bedaquiline, C clofazimine, D delamanid, E ethambutol, H isoniazid, L linezolid, M moxifloxacin, Pa pretomanid, Z pyrazinamide, R rifampicin, P rifapentine, Sq SQ109 and Su sutezolid. (c) Drug interactions were compared across similar-to-SOC versus better-than-SOC classifications, based on outcomes in relapsing mouse models (RMM), bactericidal mouse models (BMM) of common mouse strains, the granulomatous C3HeB/FeJ strain and clinical studies30,31. Comparisons were performed using a two-sided Mann-Whitney U test (P values indicated). (d) The performance of combined MIC (median regimen MIC) and drug interaction measures for predicting in vivo outcomes was assessed using logistic regression models. Fifty ROC curves were generated for every condition by randomly selecting 80% of the samples. Mean area under the ROC curves and 95% confidence intervals are presented. (a) Pearson correlation of M. abscessus drug resistance and tolerance phenotypes. White central dots highlight significant correlations (two-sided Pearson test, p < 0.05). AUC indicates the area under the kill curve, MIC minimum inhibitory concentration, AMK amikacin, AZM azithromycin, FOX cefoxitin, IPM imipenem, LZD linezolid, MIN minocycline, MXF moxifloxacin, TGC tigecycline, L refers to low drug concentration, and H to high drug concentration. (a) MIC distributions of 229 clinical isolates and the frequency of isolates associated with cleared (n = 46) and persisting infection (n = 89). (b) Odds ratios of M. abscessus minimum inhibitory concentrations for predicting treatment failures (lack of culture conversion) in 135 patients. Odds ratios of individual MIC measures were assessed with logistic regression (clarithromycin: P = 0.0019; clarithromycin inducible: P = 0.0045). Comparison of drug tolerance between M. abscessus isolates from patients with poor versus favourable clinical outcomes (n = 135, excluding growing isolates), using a two-sided Mann-Whitney U test (P indicated). Boxplots show the median, interquartile range and total range (central line, box, and whiskers, respectively) of AUC values. Automated image analysis in Antimicrobial Single-Cell Testing. Raw brightfield images (cell morphology; 100x objective) and fluorescence images (quantifying propidium iodide [PI] accumulation) are processed to segment cells (lines indicate cell borders), classify bacterial viability (white: viable; red: dead) and track individual cells (track indicated by colours). The video shows brightfield / fluorescence images of M. abscessus exposed to imipenem (high concentration), as well as cell segmentation, classification and tracking over 72 hours. Growth of propidium iodide negative bacteria after antibiotic washout. M. abscessus ATCC-19977 was treated for 24 hours with imipenem (low concentration), washed and then imaged (100x objective) without antibiotic treatment for 24 hours. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Large-scale testing of antimicrobial lethality at single-cell resolution predicts mycobacterial infection outcomes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01852-1'>RPA exhaustion activates SLFN11 to eliminate cells with heightened replication stress</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 10:27:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. SLFN11 is epigenetically silenced and confers chemoresistance in half of all cancers. In response to replication stress, SLFN11 triggers translation shutdown and p53-independent apoptosis, but how DNA damage activates SLFN11 remains unclear. Here through CRISPR-based screens we implicate SLFN11 as the critical determinant of cisplatin sensitivity in cells lacking primase–polymerase (PrimPol)-mediated repriming. SLFN11 and the downstream integrated stress response uniquely promote cisplatin-driven apoptosis in PrimPol-deficient cells. We demonstrate that replication protein A (RPA) exhaustion and single-stranded DNA exposure trigger SLFN11 activation and cell death when PrimPol is inactivated. We further identify the USP1–WDR48 deubiquitinase complex as a positive modulator of SLFN11 activation in PrimPol-deficient cells, revealing an addiction to the Fanconi anaemia pathway to resolve cisplatin lesions. Finally, we demonstrate that rapid RPA exhaustion on chemical inhibition of DNA polymerase α activates SLFN11-dependent cell death. Together, our results implicate RPA exhaustion as a general mechanism to activate SLFN11 in response to heightened replication stress. DNA replication is constantly challenged by numerous endogenous and exogenous sources of DNA damage yet is an inherently accurate and efficient process1,2,3,4. Replication stress invoked by DNA lesions is sensed by the ataxia telangiectasia and Rad3-related (ATR)-dependent intra-S-phase checkpoint, which is activated by the accumulation of RPA-bound single-stranded (ss)DNA at stalled forks and triggers a cascade of events that regulate origin firing, replication fork stabilization and lesion repair or bypass5,6,7,8. Indeed, multiple interdependent DNA damage, repair and tolerance pathways recognize, respond to and repair impediments at stalled replication forks. Recent work also uncovered the phenomenon of RPA exhaustion at stalled forks during heightened replication stress, which leads to exposure of ssDNA, fork breakage and irreversible replication catastrophe9,10. How cells respond to RPA exhaustion independently of ATR signalling is currently unknown. Owing to their inherent plasticity, stalled replication forks can be remodelled or components of the replisome modified to differentially recruit factors required to cope with specific types of fork blockages2,11. Stalled replication forks can also be directly restarted by DNA primer synthesis downstream of fork blockages via the primase–polymerase PrimPol12,13,14,15,16,17. In vitro, PrimPol-mediated repriming can occur in as little as 14 nt downstream of a replication-blocking lesion15, which could limit ssDNA accumulation at stalled forks caused by helicase–polymerase uncoupling18,19. PrimPol reprimes downstream of diverse sources of replication fork blockage ranging from endogenous G-quadruplexes and R-loops to chain-terminating nucleotides, bulky base adducts induced by ultraviolet light, benzo[a]pyrene-diol-epoxide (BPDE) and interstrand and intrastrand crosslinks induced by mitomycin C (MMC) and cisplatin14,15,18,20,21,22,23,24,25,26,27. Cisplatin is a commonly used chemotherapeutic that targets rapidly dividing cancer cells by creating intrastrand and interstrand DNA crosslinks that stall active replication forks28. The major pathway responsible for repairing cisplatin-induced interstrand crosslinks is the Fanconi anaemia (FA) pathway, which comprises over 20 gene products, many of which are shared with homologous recombination (HR) and translesion DNA synthesis (TLS)29. Initiation of the FA pathway depends on recruitment of the FA core complex comprising 14 proteins, followed by ATR-dependent phosphorylation and FANCL-dependent ubiquitination of the FANCD2–FANCI complex30,31. FANCD2–FANCI ubiquitination (and FA pathway activation) is tightly regulated by the deubiquitinase (DUB) complex USP1–WDR48 (also known as USP1–UAF1) (ref. Intriguingly, a recent study proposed a potential role for PrimPol repriming in an interstrand crosslink tolerance pathway25. Paradoxically, despite evidence that PrimPol can reprime synthesis downstream of cisplatin DNA lesions in cells, loss of PrimPol alone does not broadly confer sensitivity to cisplatin14,20,22,33. A satisfactory explanation for this discrepancy is currently lacking and highlights the current challenge in understanding how PrimPol activity cooperates with other pathways at stalled forks. Innate and acquired resistance to cisplatin occurs frequently in patients through various mechanisms, including increased drug efflux, increased DNA repair capacity and loss of proapoptotic pathways34,35. Indeed, previous work demonstrated a role for increased expression of PrimPol in the adaptive response to cisplatin treatment in BRCA-deficient tumours20. One of the strongest prognostic markers of cisplatin efficacy is the expression of Schlafen 11 (SLFN11), a transfer (t)RNA nuclease that induces p53-independent apoptosis in response to DNA damage36,37,38,39,40,41. Intriguingly, SLFN11 is epigenetically silenced in half of all treatment-naive cancer cell lines, leading to chemoresistance42,43,44. SLFN11 induces irreversible replication fork arrest through a range of proposed mechanisms, including prevention of origin firing via CDT1 degradation, increasing local chromatin accessibility and inhibition of HR or checkpoint responses37,45,46,47. After fork arrest, SLFN11 cleaves type II tRNAs, leading to ribosome stalling at rare UUA leucine codons38,48,49. Ribosome stalling results in activation of the general control non-derepressible 2 (GCN2)-dependent integrated stress response (ISR) and JNK-mediated ribotoxic stress response, triggering p53-independent apoptosis38,39. However, despite understanding how SLFN11 activation triggers apoptosis, the DNA lesion that activates SLFN11 remains unknown. Here we sought to identify drivers of cisplatin cytotoxicity using the chronic myelogenous leukaemia eHAP cell line as a model system. Surprisingly, our targeted CRISPR–Cas9 screens identified loss of PrimPol as conferring cisplatin sensitivity. In contrast to FA, HR and TLS deficiencies, cisplatin sensitivity of PrimPol knockout (KO) cells uniquely depends on SLFN11. We show that cells deficient in PrimPol-mediated repriming or drugs that induce ssDNA accumulation at replication forks potently activate an SLFN11-dependent and GCN2-dependent cell death. Using PrimPol deficiency as a model system to study SLFN11 activation, we identified RPA exhaustion as the primary mechanism of SLFN11 activation in response to DNA damage. Furthermore, we implicated USP1-dependent downregulation of the FA pathway as the mechanism that induces RPA exhaustion and SLFN11-dependent cell death, specifically when PrimPol is inactivated. Finally, we identified DNA polymerase α inhibition as a potent inducer of RPA exhaustion and SLFN11-dependent cell death. We proposed that, through its activation by RPA exhaustion, SLFN11 maintains genome stability by triggering the elimination of cells experiencing heightened replication stress, a hallmark of early cancer cells. To investigate the pathways required for cisplatin lesion metabolism at stalled replication forks, we performed a targeted CRISPR–Cas9 dropout screen in the diploid inducible Cas9 (iCas9) eHAP cell line (Fig. Two single guide (sg)RNA libraries (Supplementary Table 1) were utilized to interrogate genes associated with chromatin and telomere maintenance (pool 1) and the DNA damage response (DDR; pool 2). To determine the efficacy of our screens, we first compared sgRNA counts in wild-type (WT) cells infected with pool 2 treated with cisplatin to an untreated condition using the MaGeCK algorithm50 (Fig. As observed previously33, sgRNAs targeting genes belonging to the FA, nucleotide excision repair, TLS, DNA end-resection and HR pathways resulted in marked sensitivity to cisplatin (Fig. Surprisingly, we also identified loss of PrimPol, the primase–polymerase that catalyses repriming downstream of leading strand fork blockages, as also conferring sensitivity to cisplatin. This contrasts with previously published reports suggesting that loss of PrimPol alone does not confer sensitivity to cisplatin, despite its well-established role in DNA damage tolerance14,20,22,33. a, An experimental scheme depicting how targeted CRISPR–Cas9 screens performed. Two separate screens were performed in biological triplicates utilizing pool 1 sgRNA library (1,117 genes targeted) or pool 2 sgRNA library (288 genes targeted). b, A volcano plot depicting a measure of statistical significance (−log10(MaGeCK score)) plotted against log2(fold-change) in abundance of sgRNAs targeting indicated genes in cisplatin-treated versus untreated conditions in eHAP iCas9 WT cells infected with the pool 2 sgRNA library. Labelled genes are coloured based on the DDR pathway in which they operate. 33. d, A western blot depicting loss of PrimPol protein in an isogenic eHAP iCas9 PrimPol KO clone. e, Population doublings of eHAP iCas9 WT (grey) and PrimPol KO (red) cells plotted against time to demonstrate cell growth in the absence (squares) or presence of 450 nM (circles) or 550 nM (triangles) cisplatin. Growth curve experiments were performed in n = 2 biological replicates. f, Experimental scheme for a Cell Titer Glo viability assay to determine the cisplatin sensitivity of eHAP iCas9 WT or PrimPol KO cells complemented with WT PrimPol, PrimPol AxA (catalytic mutant Asp114Ala, Glu116Ala) or PrimPol CH (primase mutant Cys419Gly, His426Tyr). Exogenous protein levels were downregulated using the misFIT expression tuner system. h, Cell viability measured using the Cell Titer Glo assay on treatment of the indicated doses of cisplatin over 5 d. These experiments were performed in n = 4 biological replicates. i, Experimental scheme for a Cell Titer Glo assay to determine cell viability of eHAP iCas9 WT or PrimPol KO cells challenged with the indicated drugs. j, Cell viability measured after challenging eHAP iCas9 WT or PrimPol KO cells with the indicated doses of BPDE. Experiments were performed in n = 3 biological replicates. k, Cell viability measured after challenging eHAP iCas9 WT or PrimPol KO cells with the indicated doses of MMC. Experiments were performed in n = 3 biological replicates. l, A heatmap depicting sensitivity of eHAP iCas9 PrimPol KO versus WT cells to various DNA-damaging agents and inhibitors to DDR proteins. Sensitivities were determined by computing the log2(fold-change) between half-maximal inhibitory concentration (IC50) values of PrimPol KO and WT cells. As UBE2A and UBE2N have well-established roles in protecting against cisplatin cytotoxicity51,52, we focused on understanding why loss of PrimPol confers cisplatin sensitivity specifically in eHAP cells. To this end, we generated an isogenic PrimPol KO clone in the eHAP iCas9 cell line, which retained doxycycline-induced Cas9 cutting as measured by a flow cytometry-based assay (Fig. We validated our screen results by challenging eHAP WT and PrimPol KO cells with cisplatin in a growth curve experiment. PrimPol KO cells exhibited a marked loss in cell fitness when challenged with cisplatin compared with WT cells (Fig. To ensure that this phenotype was specific to loss of PrimPol and not due to off-target effects from gene editing or clonal isolation, we complemented PrimPol KO cells with complementary DNAs expressing eGFP, WT PrimPol, catalytically dead PrimPol (AxA: Asp114Ala, Glu116Ala) or primase-dead PrimPol (CH: Cys419Gly, His426Tyr) (Extended Data Fig. To avoid artifacts associated with protein overexpression, we utilized the microRNA silencing-mediated fine-tuners (misFIT) system53 to decrease cellular expression of all cDNA constructs (Extended Data Fig. We selected a misFIT sequence that lowered WT PrimPol expression levels to near-endogenous levels (Extended Data Fig. These constructs were stably expressed in the eHAP iCas9 PrimPol KO cell line and subsequently challenged with cisplatin (Fig. Only the WT PrimPol construct was able to complement cisplatin sensitivity observed in the PrimPol KO cell line, indicating that this phenotype is specific to loss of PrimPol (Fig. These data suggest that the repriming activity of PrimPol is required to tolerate cisplatin-induced DNA damage. Next, we sought to understand whether PrimPol activity protects against other types of DNA damage in human cells. Previous reports have suggested that loss of PrimPol confers sensitivity to the bulky base adduct BPDE and interstrand crosslinking agent MMC21,25,33. We challenged both eHAP iCas9 WT and PrimPol KO cells with BPDE, MMC and ten other genotoxins or inhibitors of DDR proteins (Fig. As previously reported, loss of PrimPol conferred cellular toxicity to both BPDE and MMC (Fig. However, no robust phenotype was observed when PrimPol KO cells were challenged with other types of DNA damage or inhibitors of DDR proteins (Fig. Thus, loss of PrimPol-mediated repriming specifically confers cellular sensitivity to bulky base adducts induced by cisplatin, BPDE and MMC. To identify drivers of cisplatin sensitivity in cells deficient in PrimPol-mediated repriming, we performed two targeted CRISPR–Cas9 dropout screens as previously described (Fig. We compared sgRNA counts between PrimPol KO and WT eHAP iCas9 cells in cisplatin-treated conditions, which identified loss of the tRNase SLFN11 as the top hit conferring cisplatin resistance in PrimPol KO cells in the pool 1 screen (Fig. In addition, we identified loss of the deubiquitinase USP1 as the top hit conferring cisplatin resistance in PrimPol KO cells in the pool 2 screen (Fig. Our screens also identified loss of FANCD2, RAD18 and the RPA trimeric complex as synthetic lethal with loss of PrimPol in cisplatin-treated cells (Fig. a, A volcano plot measuring statistical significance against log2(fold-change) in sgRNA counts in eHAP iCas9 PrimPol KO cells versus WT cells in the cisplatin-treated arm of the pool 1 sgRNA screen on day 6. b, A volcano plot measuring statistical significance against log2(fold-change) in sgRNA counts in eHAP iCas9 PrimPol KO cells versus WT cells in the cisplatin-treated arm of the pool 2 sgRNA screen on day 6. Genes labelled in blue exhibited increased sgRNA counts whereas genes labelled in red exhibited decreased sgRNA counts in PrimPol KO cells versus WT cells. c, A histogram of curated messenger RNA sequencing data depicting the expression of SLFN11 in human cancer cell lines curated from the DepMap repository. d, A western blot (WB) showing SLFN11 and PrimPol expression levels in HeLa, U2OS, HCT116, A549, HEK293A, DU145, eHAP and A673 cells infected with lentiviruses harbouring either Cas9–AAVS1 or Cas9–PrimPol. e, Experimental scheme showing how HeLa, U2OS, HCT116, A549, HEK293A, DU145, eHAP and A673 cells infected with Cas9–sgAAVS1 or Cas9–sgPrimPol were challenged with cisplatin and cell viability measured using the Cell Titer Glo assay. f–n, Cisplatin dose–response curves for HeLa Kyoto (f), U2OS (g), HCT116 (h), RPE-1 p53 KO (i), A549 (j), DU145 (k), HEK293A (l), A673 (m) and eHAP (n) cell lines harbouring Cas9–AAVS1 or Cas9–PrimPol. All experiments were performed in n = 2 biological replicates. We first sought to characterize the relationship between SLFN11 and PrimPol-mediated repriming in response to cisplatin treatment. Previous work identified SLFN11 activity as a potent sensitizer to a wide range of chemotherapies, including cisplatin36,42. Importantly, over half of cell lines catalogued in the DepMap repository do not express SLFN1144, including widely used non-cancerous RPE-1 and cancerous HeLa, U2OS and HCT116 (Fig. To explore whether loss of SLFN11 expression masks cisplatin sensitivity in PrimPol KO cells, we introduced a Cas9–sgRNA cassette targeting the AAVS1 safe harbour site or PRIMPOL in nine cell lines with varied tissue origin and differing PrimPol and SLFN11 expression levels (Fig. Loss of PrimPol in cell lines expressing no or low levels of SLFN11 did not confer cisplatin sensitivity (Fig. Conversely, loss of PrimPol in the three cell lines expressing high levels of SLFN11—HEK293A, eHAP and A673—conferred mild-to-severe cisplatin sensitivity (Fig. Importantly, we also generated PrimPol KO clones in cell lines expressing low (HT-1080), medium (NCIH-460) or high (A673) levels of SLFN11. Loss of PrimPol did not confer cisplatin sensitivity in the HT-1080 cell line (Extended Data Fig. 2a–c) but did confer moderate-to-severe cisplatin sensitivity in NCIH-460 and A673 cell lines (Extended Data Fig. Together, these data suggest a correlation between SLFN11 expression levels and cisplatin sensitivity when PrimPol-mediated repriming is inactivated. We sought to determine whether SLFN11 directly promotes cisplatin sensitivity on PrimPol inactivation (Fig. Inducible loss of SLFN11 conferred a growth benefit in both WT and PrimPol KO cells when challenged with cisplatin (Fig. Strikingly, loss of SLFN11 completely rescued cisplatin sensitivity observed in PrimPol KO cells to levels indistinguishable from WT cells (Fig. These results were recapitulated in an orthogonal colony formation assay (Extended Data Fig. 3a–c) and in the SLFN11-expressing A673 cell line (Extended Data Fig. b, A western blot demonstrating transient knockout of SLFN11 in eHAP iCas9 WT or PrimPol KO cells. These experiments were performed in n = 3 biological replicates. These experiments were performed using the same WT sgNTC and PrimPol KO sgNTC samples depicted in Fig. d, An experimental scheme for challenging eHAP iCas9 WT or PrimPol KO cells with cisplatin in the presence or absence of A-92, a chemical inhibitor of GCN2. e, A schematic depicting the signalling cascade connecting DNA damage to the ISR factor GCN2 and subsequent ribosome stalling and cell death. Inhibition of GCN2 prevents ribosome stalling and downstream cell death. f, Cisplatin dose–response curves for eHAP iCas9 WT or PrimPol KO cells in the presence or absence of 750 nM GCN2 inhibitor A-92. Experiments were performed in n = 3 biological replicates. g, An experimental scheme for measuring apoptosis via cleaved caspase-3 using flow cytometry. h, A bar plot depicting the percentage of cleaved caspase-3 positive cells determined using flow cytometry as shown in g. Experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to determine biological significance. i, An experimental scheme for measuring cell viability in response to cisplatin. j, A western blot showing SLFN11, PrimPol and GFP protein expression in SLFN11 WT, SLFN11 E209A (tRNA nuclease mutant), SLFN11 Lys652Asp (ssDNA-binding mutant) or SLFN11 Glu669Gln (helicase mutant) re-expression cell lines. k, Dose–response curves for SLFN11 re-expression cell lines challenged with cisplatin. These experiments were performed in n = 3 biological replicates. l, An experimental setup depicting how chromatin-fractionation immunofluorescence experiments were performed. m, The eHAP iCas9 WT or PrimPol KO cells transiently depleted of SLFN11 mock treated or treated with 450 nM cisplatin for 24 h. Representative micrographs of chromatin-bound immunofluorescence of phospho-RPA32 Ser33, DAPI or merged are shown. Data were normalized in each biological replicate to untreated WT samples. These experiments were performed in n = 5 biological replicates. A two-way analysis of variance (ANOVA) was performed to assess biological significance. The pRPA S33 staining was performed as a co-stain with γH2AX phospho-Ser139 depicted in Extended Data Fig. o, A representative western blot depicting levels of RPA phospho-Ser33, SLFN11 and GCN2 phospho-Thr899 in whole-cell extracts and chromatin fractions after treatment with 1 μM cisplatin for 24 h. This blot was performed in n = 2 biological replicates. A recent landmark study implicated SLFN11 ssDNA-dependent tRNase activity as a trigger for downstream ribosome stalling and GCN2 (gene name EIF2AK4) activation38. To determine whether SLFN11-dependent activation of GCN2 confers cisplatin cytotoxicity in PrimPol KO cells, we challenged eHAP iCas9 WT or PrimPol KO cells with cisplatin in the presence of a fixed dose of A-92, a specific inhibitor of GCN2 (GCN2i)54 (Fig. Inhibition of GCN2 completely rescued cisplatin cytotoxicity in PrimPol KO cells to levels indistinguishable from WT cells (Fig. Importantly, this fixed dose of GCN2 inhibitor did not affect cell growth in untreated conditions, ruling out any effect on cell-cycle progression (Extended Data Fig. In addition, we confirmed that SLFN11 and GCN2 act epistatically to confer cisplatin sensitivity in PrimPol KO cells because concurrent loss of SLFN11 and GCN2 inhibition did not confer additive resistance to cisplatin in any genotype (Extended Data Fig. Next, we examined whether SLFN11–GCN2-dependent cell death selectively confers cisplatin sensitivity in PrimPol KO cells or whether this is a generalized mechanism of cell death when other DDR pathways are compromised. Thus, we generated FANCD2 and RAD18 KO clones in eHAP iCas9 cells, then challenged these cell lines with cisplatin in the presence or absence of a fixed dose of GCN2i (Extended Data Fig. Inhibition of GCN2 completely rescued cisplatin sensitivity in PrimPol KO cells but did not provide any significant growth benefit in FANCD2 or RAD18 KO cells (Extended Data Fig. Similarly, loss of SLFN11 did not rescue cisplatin sensitivity when HR was inhibited using B02, a RAD51 inhibitor55 (Extended Data Fig. Together, these data suggest that the SLFN11–GCN2 axis selectively confers cisplatin sensitivity when repriming is inactivated. Previous studies have determined that SLFN11 triggers p53-independent apoptosis in response to DNA damage, either through activation of the ribotoxic stress response or downregulation of the anti-apoptotic factor MCL138,39. We sought to determine whether the SLFN11–GCN2-dependent cell death that we observed was due to apoptosis (Fig. In the absence of PrimPol, cisplatin robustly induced apoptosis that was completely dependent on SLFN11 and GCN2 (Fig. Together, these data demonstrate that cisplatin induces SLFN11-dependent and GCN2-dependent apoptotic cell death when PrimPol-mediated repriming is inactivated. The ssDNA-binding, tRNA nuclease and helicase activities of SLFN11 have all been implicated as sensitizing cells to genotoxic stress37,38,39. To determine which functions of SLFN11 were required to confer cisplatin sensitivity in PrimPol KO cells, we generated an eHAP iCas9 PrimPol–SLFN11 double KO clone and re-expressed eGFP, WT SLFN11, SLFN11 Glu209Ala (tRNA nuclease mutant), SLFN11 Lys652Asp (ssDNA-binding mutant) or SLFN11 Glu669Gln (Walker B–helicase mutant) proteins to near-endogenous levels (Fig. All three activities of SLFN11 were required to confer cisplatin sensitivity in PrimPol KO cells (Fig. As ssDNA binding was required to induce SLFN11-dependent cell death, we hypothesized that ssDNA accumulation at cisplatin-stalled replication forks could trigger SLFN11 activation. Levels of replication checkpoint activation and fork-associated ssDNA accumulation were determined by staining for ATR-dependent phosphorylation of RPA2 at Ser33 (RPA pSer33) in response to cisplatin (Fig. We observed a twofold increase in the mean focal intensity of RPA pSer33 in PrimPol KO cells treated with cisplatin compared with WT cells (Fig. This robust activation of the replication checkpoint was not completely suppressed on deletion of SLFN11 (Fig. We observed a similar induction of RPA pSer33 using western blotting techniques (Extended Data Fig. Neither γH2AX–pSer139 nor RAD51 levels increased after low doses of cisplatin treatment in PrimPol KO cells when compared with WT cells (Extended Data Fig. Finally, we observed a robust accumulation of SLFN11 on chromatin and activation of the ISR (as measured by phosphorylation of GCN2 at Thr899) after cisplatin treatment, particularly in PrimPol KO cells (Fig. Together, these data suggest that ssDNA accumulates at cisplatin-stalled replication forks in the absence of repriming, which could serve as a lesion for SLFN11 and GCN2 activation. Having established a correlation between cisplatin-induced ssDNA accumulation at stalled forks and the induction of SLFN11-dependent cell death in PrimPol KO cells (Fig. Our screens revealed that cells lacking repriming are particularly sensitive to loss of the RPA heterotrimeric complex when challenged with cisplatin (Fig. We hypothesized that RPA exhaustion could result in uncoated ssDNA, which could serve as a template for SLFN11 binding to stalled forks, activating its tRNA nuclease activity and downstream apoptosis. To determine whether cisplatin induces RPA exhaustion, we utilized quantitative image-based cytometry (QIBC) to simultaneously measure the levels of ssDNA (native 5-bromo-2ʹ-deoxyuridine (BrdU)) and chromatin-bound RPA at a cellular level, as previously described9 (Fig. Cells undergoing RPA exhaustion were quantified by determining those cells where the ratio of ssDNA to chromatin-bound RPA signal deviated from linearity (Fig. In untreated conditions, <1.5% of cells were undergoing RPA exhaustion, with no significant differences in each genotype tested (Fig. On challenge with 2.5 μM cisplatin, the percentage of cells undergoing RPA exhaustion increased twofold in WT cells. Strikingly, the percentage of cells undergoing RPA exhaustion increased more than tenfold in cells deficient in PrimPol when compared with untreated conditions (Fig. Together, these data suggest that cells deficient in repriming are more prone to RPA exhaustion in response to cisplatin. a, A diagram depicting how ssDNA was measured by detecting incorporated BrdU under non-denaturing conditions. b, An experimental scheme for measuring RPA exhaustion in eHAP iCas9 WT or PrimPol KO cells in which SLFN11 has been transiently knocked out using QIBC. c, Representative scatter plots depicting mean BrdU signal intensity per cell (y axis) plotted against mean chromatin-bound RPA32 (x axis). The linear relationship between BrdU and RPA32 signals is depicted as a light-blue line. Cells undergoing RPA exhaustion are depicted within the dashed box and coloured red. The percentage of cells undergoing RPA exhaustion is indicated in each panel. d, A bar plot depicting the percentage of cells undergoing RPA exhaustion in the experiments shown in a and b. These experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess biological significance. e, Representative scatter plots depicting mean γH2AX–pSer139 signal intensity per cell (y axis) plotted against mean chromatin-bound RPA32 (x axis). Cells with high RPA32 and γH2AX–pSer139 signals are coloured in red. The percentage of cells with high RPA32 and γH2AX–pSer139 signals is indicated in each panel. f, A bar plot depicting the percentage of cells exhibiting high RPA32–γH2AX–pSer139 signals in the experiments shown in a and e. These experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess biological significance. g, Experimental scheme for challenging eHAP iCas9 WT or PrimPol KO cells with cisplatin in the presence or absence of GCN2i after siRNA knockdown of RPA2. h, A western blot depicting RPA2, PrimPol, vinculin and total protein levels in eHAP iCas9 WT or PrimPol KO cells after transient knockdown of RPA2. i, A dose–response curve depicting cell survival in eHAP iCas9 WT or PrimPol KO cells (normalized to untransfected cells) after siRNA knockdown of RPA at indicated concentrations in the presence or absence of cisplatin and GCN2i. This experiment was performed in n = 3 biological replicates. j, An experimental scheme for challenging eHAP iCas9 WT or PrimPol KO cells with two fixed doses of cisplatin in the presence or absence of DNA-PK inhibitor (DNA-PKi) after transient knockout of SLFN11. k, A bar plot depicting cell survival in the presence of 450 nM cisplatin with or without treatment with DNA-PKi. The experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess statistical significance. l, A bar plot depicting cell survival in the presence of 1.5 μM cisplatin with or without treatment with DNA-PKi. The experiments shown were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess statistical significance. One consequence of RPA exhaustion is the generation of double-stranded breaks (DSBs) that manifest as high γH2AX–pSer139 signal in cells with a high burden of chromatin-bound RPA9. In untreated conditions, <1% of cells contained high γH2AX–pSer139 and chromatin-bound RPA signals (Fig. Although cisplatin treatment modestly increased the percentage of WT cells with high γH2AX–RPA signal, loss of SLFN11 resulted in a robust accumulation of cells with high γH2AX–RPA signal in both WT and PrimPol KO cells (Fig. Although SLFN11-proficient, PrimPol-deficient cells also exhibited a higher percentage of γH2AX–RPA double-positive cells, this is probably due to apoptosis56, because these cells are the most sensitive to cisplatin treatment (Fig. These data suggest that, although RPA exhaustion can occur in both SLFN11-proficient and SLFN11-deficient cells, the latter undergo replication catastrophe and DSB formation at higher rates when compared with the former. Having established that cisplatin-induced RPA exhaustion occurs more readily when PrimPol is inactivated, we sought to understand whether RPA exhaustion could directly activate SLFN11–GCN2-dependent cell death. To test this possibility, we reduced RPA pools in cells by performing a titration of small interfering (si)RNA targeting the RPA2 gene in eHAP iCas9 WT or PrimPol KO cells (Fig. In the absence of cisplatin, siRNA-mediated depletion of RPA2 resulted in a dose-dependent loss in cell viability that was independent of SLFN11–GCN2 activity (Extended Data Fig. Strikingly, PrimPol KO cells exhibited an siRNA dose-dependent sensitization to cisplatin, which was completely rescued on inhibition of GCN2 (Fig. These results also explain why higher levels of RPA exhaustion are observed in SLFN11-negative cells, probably due to these cells no longer undergoing RPA exhaustion-induced apoptosis (Fig. Together, these data suggest that induced RPA exhaustion confers a SLFN11–GCN2-dependent cisplatin sensitivity when repriming is inactivated. Having established that SLFN11-deficient cells undergo replication catastrophe and DSB formation at a higher rate after RPA exhaustion than SLFN11-proficient cells, we hypothesized that these DSBs could be suitable substrates for toxic repair via the non-homologous end-joining (NHEJ) repair pathway. To test this hypothesis, we transiently knocked out SLFN11 in eHAP iCas9 WT or PrimPol KO cells and subsequently challenged these cells with cisplatin in the presence or absence of a chemical inhibitor of the catalytic subunit of DNA-dependent protein kinase (DNA-PK), NU744157 (Fig. At a low dose of cisplatin, PrimPol KO cells exhibited an SLFN11-dependent loss in cell viability that was independent of DNA-PK activity (Fig. At a high dose of cisplatin, SLFN11-proficient WT or PrimPol KO cells exhibited a SLFN11-independent and DNA-PK-independent sensitization to cisplatin (Fig. However, in SLFN11-deficient cells, inhibition of DNA-PK resulted in a substantial rescue of cell viability in response to cisplatin, independent of PrimPol status (Fig. Together, these results suggest that DSBs that form after RPA exhaustion and replication catastrophe in SLFN11-deficient cells are suitable substrates for toxic DNA-PK activity. Having established SLFN11 as a key determinant of cisplatin sensitivity in PrimPol KO cells, we sought to leverage this phenotype to identify modulators of SLFN11 activation in the absence of repriming. Our targeted CRISPR–Cas9 screens identified loss of the deubiquitinase USP1 as conferring resistance to cisplatin treatment in PrimPol KO cells (Fig. Loss of USP1 or WDR48 rescued cisplatin sensitivity observed in PrimPol KO cells but had no effect in WT cells, as predicted by our screens (Fig. These results were recapitulated using a fixed dose of ML323, a specific inhibitor of the USP1–WDR48 complex58 (Extended Data Fig. These results were also observed in a second SLFN11-expressing cell line, A673 (Extended Data Fig. Importantly, WT or PrimPol KO cells treated with a fixed dose of USP1 inhibitor did not impact cell growth or viability in unchallenged conditions (Extended Data Fig. Together, these data suggest that loss of USP1–WDR48 complex activity confers resistance to cisplatin specifically in cells deficient in repriming. a, A schematic depicting how cell viability assays were performed in eHAP iCas9 WT or PrimPol KO cells challenged with cisplatin after transient knockout of USP1 or WDR48. b, A representative western blot depicting PrimPol, USP1 and WDR48 protein levels on loss of USP1 or WDR48. This western blot was performed in n = 2 biological replicates. c, A cisplatin dose–response curve for eHAP iCas9 WT or PrimPol cells on loss of USP1. These experiments were performed in n = 3 biological replicates. d, A cisplatin dose–response curve for eHAP iCas9 WT or PrimPol cells on loss of WDR48. These experiments were performed in n = 3 biological replicates. These experiments were performed using the same WT sgNTC and PrimPol KO sgNTC samples depicted in Fig. e, An experimental scheme for measuring RPA exhaustion in cell pools after transient knockout of USP1 and SLFN11 using QIBC. f, A representative western blot showing USP1 and SLFN11 protein levels after transient knockout of each protein. This western blot was performed in n = 2 biological replicates. g, Representative scatter plots depicting mean ssDNA (BrdU) intensity (y axis) plotted against mean chromatin-bound RPA32 intensity (x axis) for each genotype and treatment depicted. The dashed boxes indicate RPA-exhausted cells where the ratio of mean ssDNA signal against mean chromatin-bound RPA32 signal has deviated from linearity (blue line). The percentage of cells undergoing RPA exhaustion in each genotype and treatment is shown. h, A bar plot depicting the percentage of cells undergoing RPA exhaustion in each genotype and treatment tested. These experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess biological significance. i, An experimental scheme depicting how apoptosis (cleaved caspase-3 signal) was measured using flow cytometry in eHAP iCas9 cells after transient knockout of USP1 or SLFN11. j, A bar plot depicting the percentage of cells undergoing apoptosis, as assessed by measuring the cleaved caspase-3 signal using flow cytometry. These experiments were performed in n = 3 biological replicates. A two-way ANOVA was performed to assess biological significance. As loss of USP1–WDR48 complex activity rescued cisplatin sensitivity in PrimPol KO cells and this phenotype is entirely dependent on SLFN11-mediated and GCN2-mediated apoptosis, we hypothesized that USP1 promotes SLFN11 activation and induction of apoptosis by inducing RPA exhaustion. To test this hypothesis, we performed QIBC experiments to determine the levels of RPA exhaustion in eHAP iCas9 WT or PrimPol KO cells that have had USP1 or SLFN11 transiently knocked out (Fig. To avoid any effects of SLFN11-dependent apoptosis on masking RPA exhaustion levels, we performed all experiments in SLFN11-deficient cells. The level of cisplatin-induced RPA exhaustion in PrimPol KO cells was threefold to fourfold higher than in WT cells (Fig. Strikingly, loss of USP1 in PrimPol KO cells reduced the level of cisplatin-induced RPA exhaustion by twofold to threefold when compared with USP1-proficient PrimPol KO cells (Fig. Together, our data indicate that loss of USP1 significantly reduces the levels of cisplatin-induced RPA exhaustion in cells lacking repriming. We hypothesized that the rescue of RPA exhaustion observed on loss of USP1 would lead to a reduction in SLFN11-dependent apoptosis and cell death. Loss of USP1 significantly rescued SLFN11-dependent apoptosis in PrimPol KO cells challenged with cisplatin and provided no benefit in SLFN11-deficient cells (Fig. As observed with RPA exhaustion and apoptosis, USP1 inhibition significantly rescued cell viability of PrimPol KO cells challenged with cisplatin in an SLFN11-dependent manner (Extended Data Fig. USP1 targets two major proteins in the DDR for deubiquitination: FANCD2–FANCI and PCNA59,60. Deubiquitination of FANCD2–FANCI and PCNA leads to downregulation of the FA and TLS pathways, respectively. As USP1 also drove cisplatin sensitivity in an SLFN11-dependent manner in PrimPol KO cells, we sought to determine whether downregulation of the FA or TLS pathways was responsible for driving RPA exhaustion and SLFN11-dependent cell death. First, we wanted to determine whether loss of the FA or TLS pathway was synthetic lethal with loss of repriming in response to cisplatin as predicted in our targeted CRISPR–Cas9 screens (Fig. Indeed, loss of either RAD18 or FANCL led to an additive cisplatin sensitivity in cells lacking repriming (Extended Data Fig. Second, we wanted to determine whether upregulation of the FA or TLS pathway was responsible for rescuing cisplatin sensitivity in cells lacking repriming after USP1 depletion (Extended Data Fig. Loss of RAD18 did not reverse the cisplatin resistance observed on knockout of USP1 in PrimPol KO cells, whereas loss of FANCL completely reversed this phenotype (Extended Data Fig. Strikingly, loss of USP1 partially rescued the exquisite cisplatin sensitivity observed in double RAD18–PrimPol KO cells, underlining the importance of the FA pathway in repairing cisplatin lesions, particularly when TLS and repriming are inactivated (Extended Data Fig. Indeed, we observed an increase in the number of FANCD2 foci present in PrimPol KO cells challenged with cisplatin, suggesting a compensatory upregulation of the FA pathway when repriming is inactivated (Extended Data Fig. Together, our data implicate USP1-driven downregulation of the FA pathway as the mechanism behind the increased cisplatin-induced RPA exhaustion and SLFN11-dependent apoptosis in cells lacking repriming. Having established that cells lacking repriming undergo RPA exhaustion and SLFN11-dependent cell death when challenged with cisplatin, we wanted to determine whether this mechanism is broadly applicable to other types of DNA damage, independent of PrimPol status. To address this, we transiently knocked out SLFN11 in WT eHAP iCas9 cells and challenged them with 12 different genotoxic agents or inhibitors of DDR pathways (Fig. Strikingly, cells treated with ST1926, a selective chemical inhibitor of DNA polymerase α61, underwent a potent SLFN11-dependent death response (Fig. Other drugs, such as the poly(ADP ribose polymerase) inhibitor, olaparib, or the topoisomerase I inhibitor, camptothecin, also induced a strong SLFN11-dependent death response (Fig. Conversely, drugs targeting DSB repair pathways (RAD51 or ataxia-telangiectasia mutated (ATM) inhibitor) or inducing DSBs (bleomycin) did not elicit a strong SLFN11-dependent cell death (Fig. Surprisingly, hydroxyurea, a potent inducer of ssDNA at replication forks, failed to robustly activate SLFN11-dependent cell death (Fig. We hypothesized that this could be due to ST1926 and hydroxyurea inducing RPA exhaustion at different levels, as previously described9,10. a, An experimental scheme depicting how cell survival was measured in cells with indicated genotypes after challenge with 12 different drugs targeting the DDR. b, A Forest plot depicting the log2(fold-change) of IC50 values derived from dose–response curves of the indicated drugs when SLFN11 was transiently knocked out versus a non-targeting control. Coloured dots represent calculated mean IC50 values, whereas black lines indicate 95% confidence intervals of each calculated mean IC50 value. c, An experimental scheme depicting how RPA exhaustion was assessed using QIBC after treatment with DNA polymerase α inhibitor (ST1926). d, A western blot depicting SLFN11 protein levels in eHAP iCas9 WT and an isogenic SLFN11 KO clone. e, A scheme of how SLFN11 activation was measured by staining cells for chromatin-bound ATF4 after ST1926-induced RPA exhaustion. f, Representative scatter plots measuring mean ssDNA or BrdU (y axis) versus mean chromatin-bound RPA32 signals (x axis). Linearity between the ssDNA and chromatin-bound RPA32 signals is depicted as a light-blue line. Cells undergoing RPA exhaustion are highlighted within the dashed box, with representative percentages of cells undergoing RPA exhaustion indicated. Cells with high levels of chromatin-bound ATF4 (ATF4high) are coloured red, whereas ATF4− cells are shown in grey. g, A bar plot depicting the percentage of cells with indicated genotype and treatment with ATF4high (high ATF4 activation). These experiments were performed in n = 3 biological replicates. A one-way ANOVA was performed to assess biological significance. h, A bar plot depicting the percentage of eHAP iCas9 WT cells with high ATF4 activation that are undergoing RPA exhaustion. These experiments were performed in n = 3 biological replicates. A one-way ANOVA was performed to assess biological significance. i, A bar plot depicting the percentage of cells undergoing RPA exhaustion that exhibit ATF4 activation (ATF4high). These experiments were performed in n = 3 biological replicates. A one-way ANOVA was performed to assess biological significance. As ST1926 potently induces RPA exhaustion and SLFN11-dependent cell death, we wanted to determine whether ST1926-induced RPA exhaustion could activate SLFN11. We utilized QIBC to simultaneously determine the levels of RPA exhaustion and SLFN11 activation in eHAP iCas9 WT cells or an isogenic SLFN11 KO clone (Fig. We were unable to detect SLFN11 chromatin binding under the rigorous pre-extraction conditions required for native BrdU or chromatin-bound RPA QIBC experiments. Thus, we stained for chromatin-bound ATF4, the master transcription factor activated by the GCN2-dependent ISR62, as a proxy for SLFN11 activation (Fig. Indeed, eHAP cells treated with ST1926 potently activated ATF4 in an SLFN11-dependent manner, providing confidence in our approach (Extended Data Fig. We utilized a conservative threshold for ATF4 activation by only scoring the top 25% of ATF4+ cells as ATF4high (Extended Data Fig. Given that only high or chronic levels of ISR activation are proapoptotic62,63, we wanted to ensure that both replicate reproducibility and accurate scoring of only the cells with the highest levels of ATF4. Low doses of ST1926 rapidly induced RPA exhaustion after 1 h of treatment in both WT and SLFN11 KO cells (Fig. Consistent with our model that RPA exhaustion activates SLFN11, >60% of WT cells with ATF4high were undergoing RPA exhaustion while 10% of WT cells undergoing RPA exhaustion exhibited high levels of ATF4 (Fig. Together, these data suggest that inhibition of DNA polymerase α results in robust RPA exhaustion that can activate the ISR in an SLFN11-dependent manner. Having established ST1926 as a potent inducer of both RPA exhaustion and SLFN11-dependent cell death, we investigated whether this was broadly applicable to other SLFN11-expressing cell lines. We generated five cancer cell lines harbouring an inducible Cas9 that express different levels of SLFN11: HeLa Kyoto, HT-1080, TOV112D, A673 and eHAP. These cells were challenged with ST1926 treatment after transient knockout of SLFN11 (Fig. Together, these data suggest that inhibition of DNA polymerase α induces SLFN11-dependent cell death through a mechanism that is conserved across different cancer cell types. a, An experimental scheme depicting how cell viability was measured in indicated cell lines after transient knockout of SLFN11 and challenge with DNA polymerase α inhibitor (ST1926). b, A western blot depicting SLFN11 protein levels after transient knockout of SLFN11 in the indicated cell lines. c–g, Dose–response curves of HeLa Kyoto iCas9 (c), HT-1080 iCas9 (d), TOV112D iCas9 (e), A673 iCas9 (f) and eHAP iCas9 (g) cells challenged with the indicated doses of DNA polymerase α inhibitor (ST1926). Experiments were performed in n = 3 biological replicates. h, In WT cells expressing SLFN11 (blue panel), PrimPol repriming activity restricts accumulation of ssDNA at cisplatin-stalled replication forks caused by uncoupling of the helicase and DNA polymerase, thereby preventing SLFN11 activation and cell death. Similarly, inhibition of DNA polymerase α uncouples DNA replication on each strand to drive ssDNA accumulation at forks in a PrimPol-independent manner. In PrimPol-deficient cells (green panel), cisplatin-induced DNA damage drives accumulation of ssDNA at replication forks in a USP1-dependent manner, leading to RPA exhaustion and SLFN11 activation. USP1-dependent downregulation of the FA pathway allows for full SLFN11 activation and subsequent ribosome stalling, ISR activation and cell death, as previously described38. In SLFN11-deficient cells (red panel), cisplatin-induced DNA damage or inhibition of DNA polymerase α causes ssDNA accumulation at forks and RPA exhaustion. Loss of SLFN11 prevents signal transduction through ribosome stalling and GCN2, preventing SLFN11-dependent cell death. Instead, prolonged RPA exhaustion leads to fork breakage and replication catastrophe as previously described9, leading to DSBs that are repaired by the NHEJ pathway, leading to cell death. Our findings demonstrate that RPA exhaustion and exposure of ssDNA are the trigger for SLFN11 activation in response to replication-stalling lesions. Our study also reconciles a long-standing discrepancy between mechanistic observations that PrimPol reprimes downstream of DNA lesions while also being dispensable for cell viability in response to those same DNA lesions, which we attribute to the expression status of SLFN11 in cells. Our serendipitous finding that chronic myelogenous leukaemia eHAP cancer cells lacking PrimPol-mediated repriming are sensitive to cisplatin (Fig. 1a–i) was initially surprising, because previous studies showed that other cell lines such as RPE-1, UWB1.289, HEK293T and U2OS do not exhibit this phenotype14,20,22,33. Our targeted CRISPR–Cas9 screens identified SLFN11 as a driver of cisplatin sensitivity, particularly in cells deficient in PrimPol-mediated repriming (Fig. Using a panel of 12 cancer and non-cancerous cell lines, we showed that SLFN11 expression level is broadly correlated with cisplatin sensitivity when PrimPol-mediated repriming is inactivated (Fig. The lack of cisplatin sensitivity observed previously in other cell lines lacking repriming can be explained by the silencing of SLFN11 in these cell lines. Indeed, targeted knockout of SLFN11 or inhibition of the downstream ISR kinase GCN2 rescued both apoptosis and cisplatin sensitivity observed in eHAP PrimPol KO cells to levels indistinguishable from WT cells (Fig. Importantly, our study implicates the GCN2-mediated ISR as being primarily responsible for SLFN11-dependent apoptosis, in contrast with previous work, which implicated the JNK-mediated ribotoxic stress response or impaired ribosome biogenesis38,39. This is probably due to a difference in dose and treatment time of genotoxic agents used in our work versus those used in other studies. Future work is needed to understand how different burdens of replication stress induce differential SLFN11-dependent responses. Intriguingly, we found that the tRNA nuclease, ssDNA-binding and ATPase or helicase activities of SLFN11 were all required to confer cisplatin sensitivity in PrimPol KO cells (Fig. We favour a model whereby the ATPase or helicase activity of SLFN11 is required to release SLFN11 from DNA to effectively cleave tRNAs. However, future studies focusing on further dissecting the distinct functions of SLFN11 and how each promotes p53-independent apoptosis are warranted. SLFN11 preferentially binds ssDNA in vitro64 and this activity is required for translation shutdown, p53-independent apoptosis38,39, ssDNA-induced cell death65 and cisplatin sensitivity in PrimPol KO cells (Fig. However, the endogenous source of ssDNA that activates SLFN11 was previously unknown. It is unlikely that every stalled replication fork would activate SLFN11-dependent cell death, because many drugs, such as hydroxyurea, induce fork stalling but do not robustly activate the SLFN11–GCN2 cell death pathway (Fig. We hypothesized that a potential source of ssDNA under heightened replication stress was RPA exhaustion, a rare yet catastrophic event that serves as a potent source of ssDNA in the cell9. Indeed, we observed more cells undergoing RPA exhaustion under conditions that induce SLFN11-dependent cell death (Figs. Importantly, RPA exhaustion induced by siRNA targeting of RPA2 led to a SLFN11–GCN2-dependent sensitization to cisplatin when repriming was inactivated (Fig. RPA exhaustion triggering SLFN11-dependent apoptosis also explains why drugs like hydroxyurea do not potently activate an SLFN11-dependent response, whereas drugs like ST1926 do (Fig. Hydroxyurea alone does not induce RPA exhaustion even though it induces ssDNA accumulation at forks, probably through its potent induction of ATR and subsequent prevention of origin firing10. However, inhibition of DNA polymerase α induces rapid ssDNA accumulation, potent induction of RPA exhaustion10 and SLFN11 activation (Fig. Another hallmark of RPA exhaustion is the generation of double-stranded DNA breaks after a replication fork catastrophe and fork breakage. Previous work connecting RPA exhaustion and replication fork catastrophe has been performed in U2OS, a SLFN11-negative cell line9. Strikingly, we observed RPA exhaustion-induced DSBs predominantly in SLFN11-negative cells that were preferentially acted on by DNA-PK, leading to cell death (Figs. Our interpretation of these data is that RPA exhaustion triggers SLFN11-dependent apoptosis before replication fork catastrophe (Fig. When SLFN11 is not expressed, cells undergoing RPA exhaustion do not undergo apoptosis but instead undergo canonical, replication fork catastrophe, DSB formation, potentially engaging the NHEJ pathway (Fig. Our work also identified the USP1–WDR48 DUB complex as a modulator of SLFN11 activation at the replication fork. In cells lacking repriming, USP1 drives cisplatin-induced RPA exhaustion, resulting in induction of SLFN11-dependent apoptosis and cell death (Fig. We identified upregulation of the FA pathway as responsible for preventing SLFN11-dependent apoptosis when USP1 is knocked out (Extended Data Fig. Our interpretation of these data is that cells lacking PrimPol become addicted to the FA pathway to resolve cisplatin lesions. However, we cannot rule out that FA-mediated repair of cisplatin lesions is delayed in the absence of PrimPol25, potentially leading to a temporary increase in FANCD2 foci. However, as the FA pathway and PrimPol operate additively in response to cisplatin (Extended Data Fig. 7e) and upregulation of the FA pathway rescues cisplatin sensitivity in PrimPol KO cells (Extended Data Fig. 7j), we favour the model that PrimPol and the FA pathway act separately in response to cisplatin. In conclusion, our findings describe a fundamental insight into the mechanism of SLFN11 activation at replication-stalling lesions, revealing a critical role in sensing and eradicating cells experiencing RPA exhaustion. We propose that the ATR-dependent, intra-S-phase checkpoint senses and signals replication stress by virtue of the targeting of ATR to RPA-coated ssDNA by ATR-interacting protein (ATRIP), which is required to stabilize the replisome, invoke repair and regulate origin firing. If this response is overwhelmed or RPA becomes limiting due to RPA exhaustion, this would create the dangerous scenario of unprotected ssDNA prone to nucleolytic attack and replication catastrophe. Our work reveals that SLFN11 has evolved to sense RPA exhaustion and, on activation, triggers a ‘pathway of last resort' to eliminate cells with heightened replication stress. Given that elevated replication stress is a hallmark of cancers, RPA exhaustion would create a strong selective pressure to epigenetically silence SLFN11 to allow cancer cell survival, which would explain its frequent inactivation in >50% of treatment-naive tumours. The eHAP-inducible Cas9 (iCas9) cells were generated previously66 and maintained in Isocove's modified Dulbecco's medium (Gibco, catalogue number 12440053) supplemented with 10% tetracycline-free fetal bovine serum (FBS; Pan Biotech, catalogue number P30-3602) and penicillin–streptomycin (Gibco, catalogue number 15140122). HeLa Kyoto, U2OS, DU145, A549, HEK293A, HEK293 FT, HCT116, TOV112D and A673 cells were maintained in Dulbecco's modified Eagle's medium (Gibco, catalogue number 41966029) supplemented with 10% tetracycline-free FBS and penicillin–streptomycin. RPE-1 p53 KO cells were a kind gift of K. Vousden and maintained in Dulbecco's modified Eagle's medium/F12 medium (Gibco, catalogue number 11320033) supplemented with 10% tetracycline-free FBS and penicillin–streptomycin. HT-1080 iCas9 cells were generated as described in ‘Inducible Cas9 parental cell line generation' and maintained in Eagle's minimum essential medium (LGC, catalogue number 30-2003) supplemented with 10% FBS and penicillin–streptomycin. NCI-H460 iCas9 cells were generated as described in ‘Inducible Cas9 parental cell line generation' and maintained in Roswell Park Memorial Institute 1640 medium (Gibco, catalogue nnumber 11530586) supplemented with 10% FBS and penicillin–streptomycin. All cells utilized in this study were grown at 37 °C and 5% CO2 in humidified cell culture incubators. All cell lines in this study were obtained from the Francis Crick Institute Cell Science Scientific Technology Platform. Cisplatin (Sigma-Aldrich, catalogue number P4394) was dissolved in phosphate-buffered saline (PBS) to a stock concentration of 5 mM and stored at room temperature, protected from light, for 2–3 weeks before making a fresh stock. Hydroxyurea (Sigma-Aldrich, catalogue number H627) was dissolved fresh in sterile water to a stock concentration of 100 mM before each experiment. ON-TARGETplus SMARTPool non-targeting siRNA (catalogue number D-001810-10-05) or siRNA targeting human RPA2 (catalogue number L-017058-01-0005) was purchased from Horizon Discovery and resuspended in siRNA dilution buffer (Dharmacon, catalogue number B-002000-UB-100) to a concentration of 20 μM. The eHAP iCas9 WT or PrimPol KO cells were transfected with the indicated concentrations of siRNA targeting the RPA2 gene using lipofectamine RNAiMAX (Thermo Fisher Scientific, catalogue number 13778150) while keeping total siRNA concentration constant by buffering with non-targeting siRNA. PrimPol cDNA was purchased from OriGene (catalogue number SC100629). PrimPol cDNA was amplified using Gateway primers PrimPol-gateway-F and PrimPol-gateway-R (Supplementary Table 2) and cloned into the pDONR221 plasmid (Thermo Fisher Scientific, catalogue number 12536017). PrimPol was subcloned into the pLEX_307 (pLenti EF1a) destination vector (Addgene, catalogue number 41392). PrimPol AxA (Asp114Ala, Glu116Ala) and CH (Cys419Gly, His426Tyr) mutants were introduced into this plasmid using the Q5 Site-Directed Mutagenesis kit (NEB), with primers shown in Supplementary Table 2. SLFN11 cDNA was amplified and cloned into a pDONR221 plasmid as described above. SLFN11 Glu209Ala, Lys652Asp and Glu669Gln mutants were generated using the Q5 Site-Directed Mutagenesis kit with the primers shown in Supplementary Table 2. Finally, WT or mutant SLFN11 cDNAs were cloned into a Piggyback EF1a-driven Gateway plasmid. These plasmids were transfected into the indicated cells alongside the PiggyBac Transposase plasmid to generate stable cell lines expressing SLFN11. PrimPol expression was downregulated by cloning misFIT sequences53 complementary to the mIR-17 microRNA with indicated mutations and spacer sequences into the 3ʹ-UTR downstream of WT PrimPol, PrimPol AxA or PrimPol CH mutants previously cloned into pLEX_307. A complete list of oligonucleotides used for misFIT cloning is shown in Supplementary Table 2. The sgRNA sequences were sourced from the Brunello sgRNA library67 or designed in house (Supplementary Table 1). Lentiviruses were generated by transfecting HEK293 FT cells with third-generation packaging plasmids VSVG, pLP1 (gag or pol) and pLP2 (Rev) and a lentiviral vector of interest. The medium was changed on transfected cells 18 h post-transfection; cells were left for a further 48 h, after which viral supernatants were harvested, filtered with a 0.45-μm filter, aliquoted and frozen at −80 °C. Cells were prepared for transduction by changing the medium to a low volume of complete medium with added polybrene (Sigma-Aldrich, catalogue number H9268) to a final concentration of 8 μg ml−1. Cells were then transduced by adding viral supernatants and either left overnight or immediately spinfected by spinning cell or viral supernatant mixtures at 500g and 37 °C for 60 min. HT-1080 iCas9, NCIH-460 iCas9, TOV112D iCas9 and A673 iCas9 parental cell lines were generated by transducing lentiviral particles generated as described in ‘Lentivirus preparation and infection' harbouring the Edit-R Inducible Lentiviral Cas9 plasmid purchased from Dharmacon or Horizon (Horizon, catalogue number CAS11229). Transduced cells were enriched by selecting with blasticidin (8 μg ml−1) for 2–3 d until untransduced cells died. Single-cell clones were isolated from these pools of transduced cells and iCas9 activity was measured as described in ‘Inducible Cas9 activity assay'. Clones exhibiting <5% Cas9 activity in uninduced (no doxycycline) and >90% Cas9 activity in induced (with doxycycline) were used as parental iCas9 cell lines. Inducible Cas9 clones were assessed for Cas9 activity by transducing lentiviral particles harbouring the pKLV2-U6gRNA5(gGFP)-PGKBFP2AGFP-W or pKLV2-U6gRNA5(gGFP)-PGKmCherry2AGFP-W plasmids (Addgene, catalogue nos. Cells were split into two populations in the absence or presence of 1 μg ml−1 of doxycycline to induce Cas9 expression 24 h post-transduction. Cells were then maintained with or without doxycycline for 3 d (changing doxycycline as needed every 2 d). To generate eHAP iCas9 PrimPol, FANCD2, SLFN11, RAD18 or USP1 KO clones, an sgRNA targeting PrimPol, FANCD2, SLFN11, RAD18 or USP1 was cloned into pLenti-sgRNA-puro vector. This vector was transfected into eHAP iCas9 WT or PrimPol KO parental cells while Cas9 was induced on addition of 1 μg ml−1 of doxycycline. Cells were enriched for those harbouring the vector by selecting with puromycin for 1–2 d while maintaining Cas9 expression with doxycycline. The resulting pool of transfected cells was seeded by limiting dilution and clones were picked and screened by purifying genomic DNA using the PureLink Genomic DNA Mini Kit (Thermo Fisher Scientific, catalogue number K182001) and PCR amplifying and Sanger sequencing for the sgRNA cut site. Positive clones were further screened by western blotting to assess protein levels. The indicated parental cell lines were transduced with lentiviral particles harbouring a vector containing either a constitutively expressed sgRNA (pLenti-sgRNA-puro or pLenti-sgRNA-hygro) or an all-in-one Cas9–sgRNA cassette (pLentiCRISPRv2-puro). Cells were selected with appropriate selection agents at the concentrations listed above. Selected pools of cells were then used directly in experiments or single-cell clones were isolated as indicated. Whole-cell extracts were isolated by harvesting cells by centrifugation, washing once with PBS, spinning down again and freezing cell pellets at −80 °C. Cells were thawed and resuspended in radioimmunoprecipitation buffer (50 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1% Triton X-100, 0.1% sodium dodecylsulfate (SDS), 0.5% deoxycholate, 1× phosphatase (Phos-Stop, Roche, catalogue number 4906845001) and 1× protease (cOmplete EDTA-free, Roche, catalogue number 11836170001) inhibitor cocktails). Cells were allowed to lyse for 15–20 min on ice, briefly sonicated to shear chromatin, then centrifuged at high speed (30,000g) to remove cellular debris. Cell lysates were normalized to equal protein concentration and mixed with appropriate volumes of NuPAGE LDS sample buffer (4×; Thermo Fisher Scientific, catalogue number NP0008) supplemented with 100 mM 2-mercaptoethanol. Lysates were boiled at 90 °C for 10 min and frozen at −20 °C until ready to run on SDS–polyacrylamide gel electrophoresis (PAGE). Protein extracts were thawed at room temperature before loading 30–40 μg of total protein on to 4–12% Bis–Tris NuPAGE SDS–PAGE gradient gels or NuPAGE 3–8% Tris-acetate gradient gels (Thermo Fisher Scientific) with an appropriate number of wells. SDS–PAGE gels were run with an applied voltage of 120–180 V for 1–2.5 h. For chromatin fractionations, whole-cell lysates were split in two and pelleted as above. One half of each sample was resuspended in radioimmunoprecipitation buffer as above. Chromatin pellets were harvested by centrifuging at maximum speed for 15 s, after which pellets were washed in CSK buffer and centrifuged again. Chromatin pellets were resuspended in 1× NuPAGE LDS sample buffer, sonicated and boiled as with whole-cell lysates. Proteins were transferred to nitrocellulose western blotting membranes (GE Healthcare or Amersham) by applying a constant amperage of 400 mA for 1 h. Membranes were stained with Ponceau S stain (Thermo Fisher Scientific) and imaged to assess protein loading. Membranes were then blocked using 5% non-fat milk powder dissolved in Tris-buffered saline with Tween 20 (TBS-T) for 1 h at room temperature. Non-specific proteins were washed away with TBS-T 3× for at least 5 min per wash. Secondary horseradish peroxidase-conjugated antibodies were diluted 1:2,000 in 5% non-fat milk powder dissolved in TBS-T and applied to membranes for 1 h at room temperature. Membranes were then thoroughly washed at least 5× with TBS-T before the addition of chemiluminescence reagent (BioRad, Clarity or ClarityMAX). Membranes were then imaged using a BioRad ChemDoc imaging system. Indicated cell lines were grown for a total of 10 d in complete medium with the indicated concentrations of cisplatin, splitting every 2 d into fresh cisplatin as needed. Cells were counted every 2 d using a Countess 3 cell counter. Population doubling rates were determined using the following formula: where n is population doubling, Xf the final cell count and Xi the initial cell count. Cells were then exposed to selected drugs at indicated concentrations and grown for a further 5 d without changing the medium (7 d for A673). To read luminescence activity, growth medium was removed and replaced with a 1:1 ratio of 90 μl of complete medium: 90 μl of Cell Titer Glo One Solution Assay reagent (Promega, catalogue number G8462). The 96-well plates were agitated for 5 min at 120 rpm, after which the luminescence readings were read using a BMG LabTech ClarioStar plate reader. Dose–response curves were generated by normalizing the luminescence reading of all samples to an untreated internal control for each cell line. Two customized sgRNA libraries containing four sgRNAs per gene were designed in house and ordered from Sigma-Aldrich (Supplementary Table 1). Pool 1 was designed to target genes found at human telomeres whereas pool 2 was designed as a supplemental sgRNA library to broaden the scope of pool 1 to cover additional targets involved in the DDR. Lentiviral particle preparations were obtained from Sigma-Aldrich and titered by infecting an equal number of eHAP iCas9 WT cells with increasing volumes of virus and determining the multiplicity of infection. The eHAP iCas9 WT and PrimPol KO cells were infected with either pool 1 or pool 2 lentivirus in biological triplicate at a multiplicity of infection = 0.2 and an sgRNA representation of at least 500 cells per sgRNA. Cells harbouring the sgRNA libraries were enriched by selecting with puromycin for 2 d. Doxycycline was added to the medium to induce Cas9 expression over 6 d, splitting every 2 d with fresh doxycycline. Cells were then split into two arms per screen: untreated or treated with 450 nM cisplatin. Cells were grown for 6 d more, splitting every 2 d with fresh cisplatin in the treatment arm. Cell pellets were collected throughout the screens with an appropriate number of cells to maintain 500 cells per sgRNA representation. Genomic DNA was isolated from these cells and subsequently used as templates for single-step PCR amplification of the sgRNA sequences present in each sample. PCR products were cleaned up using AMPure XP beads (Beckman Coulter, catalogue number A63800), according to the manufacturer's instructions and quantified using a QuBit system. PCR products were deep sequenced using Illumina Sequencing. Sequencing reads were trimmed, sgRNA counts were calculated (Supplementary Tables 3 and 4) and comparisons between samples were performed using the MaGeCK algorithm50 with default settings (Supplementary Table 5). The eHAP iCas9 cells were plated at a concentration of 200 cells per well in a 24-well plate in technical triplicates for each condition in each biological replicate. Cells were drugged with the indicated doses of cisplatin 18 h after plating. Colonies were allowed to form for 5 d after drugging, after which the medium was aspirated, colonies were washed with PBS, then stained with Crystal Violet stain (0.5% Crystal Violet in 20% methanol) for 15–30 min at room temperature. Excess Crystal Violet stain was removed and cells were washed with water and allowed to dry completely before imaging, using an Oxford Optronix GelCount colony counter. Colonies were quantified by automatically masking wells and adjusting CHARM settings to accurately detect colonies of varied sizes. The eHAP iCas9 cells were seeded in 12-well plates and allowed to grow for 24 h before mock, cisplatin or GCN2 inhibitor treatment. After 40 h, cells were trypsinized and harvested in PBS, then transferred to 96-well round-bottomed plates (Falcon, catalogue number 353263). Cells were centrifuged at 500g for 5 min, after which the supernatant was aspirated. Cells were fixed in 100 μl of BD PhosFlow Fix Buffer I (BD Biosciences, catalogue number 557870) for 15 min at 37 °C. Cells were centrifuged again then permeabilized in 100 μl of BD PhosFlow Perm Buffer III (BD Biosciences, catalogue number 558050) at 4 °C for 30 min. Cells were centrifuged then blocked in 100 μl of blocking buffer (PBS + 10% goat serum) (Thermo Fisher Scientific, catalogue number G6767) for 30 min at room temperature. A 2× primary antibody dilution was made using rabbit anti-cleaved caspase-3 (BD Biosciences, catalogue number 570524) and added to blocked cells so that the final concentration of cleaved caspase-3 antibody was 1:1,000. After primary antibody incubation for 1 h, cells were washed once in PBS + 0.1% goat serum, then stained using goat anti-rabbit Alexa Fluor-647 (Thermo Fisher Scientific, catalogue number A-21244) at a final concentration of 1:1,000. After incubation with secondary antibody for 1 h at room temperature, cells were washed once in PBS + 0.1% goat serum, then resuspended in 200 μl of PBS + 0.1% goat serum. Cells were strained through a 35-μm mesh into round-bottomed flow cytometry tubes (Thermo Fisher Scientific, catalogue number 10585801). Doublets were removed from analysis by gating for forward scattering height versus area, after which cleaved caspase-3-positive cells were quantified. The eHAP iCas9 cells were plated at a concentration of 3,000 cells per well in a black, 96-well PerkinElmer (now Revvity) PhenoPlate. Cells were allowed to attach overnight, after which they were mock treated or treated with the indicated doses of cisplatin for 24 h. Medium was aspirated and cells were washed once in PBS. Chromatin fractions were obtained by pre-extracting soluble proteins at the same time as fixation with 4% paraformaldehyde in PBS + 0.2% Triton X-100 for 15 min at room temperature. Fixed and extracted cells were washed twice with PBS and stored at 4 °C until ready to be stained. When possible, all biological replicates were stained at the same time. PBS was aspirated from the 96-well plates, after which the cells were permeabilized by treating with PBS + 0.2% Triton X-100 for 15 min at room temperature. Cells were washed once in PBS and then blocked in blocking buffer (PBS + 3% bovine serum albumin + 0.1% Triton X-100) for 1 h at room temperature. Blocking buffer was removed and primary antibodies were applied at the following concentrations in blocking buffer for 1 h at room temperature: RPA32 phospho-Ser33 (Bethyl, catalogue number A300-246A, 1:2,500); RAD51 (Millipore, catalogue number ABE257, 1:500); γH2AX phospho-Ser139 (Millipore clone JBW301, catalogue number 05-636, 1:2,500); and FANCD2 (abcam, catalogue number ab108928, 1:2,000). After incubation with primary antibodies, cells were washed 3× for at least 5 min with PBS + 0.1% Triton X-100. Secondary antibodies and DAPI were diluted in blocking buffer to final concentrations of 1:1,000 (secondary antibodies) and 0.5 μg ml−1 of (DAPI), then incubated with cells for 1 h at room temperature. Goat anti-mouse Alexa Fluor-488 (Invitrogen, catalogue number A-11029) and goat anti-rabbit Alexa Fluor-647 (Invitrogen, catalogue number A-21245) were used in all experiments. After incubation with secondary antibodies, cells were washed 3× with PBS + 0.1% Triton X-100 for at least 5 min each. Cells were then washed with PBS twice and stored in PBS at 4 °C until ready to image. Stained plates were scanned using the Operetta CLS confocal high-content system (×40). Maximum intensity projections were used for phenotypic analysis on 17 fields of view (FOVs); within each FOV, 5 planes were taken within each z-stack. Cell nuclei were identified using Harmony 5.1 software, followed by foci identification in the relevant imaging channels. The eHAP iCas9 WT, PrimPol KO or SLFN11 KO cells with indicated genotypes were plated at a concentration of 10,000 (cisplatin treatment for 24 h) or 15,000 (ST1926 treatment time course) cells per well in a black, 96-well PhenoPlate in the presence of 10 μM BrdU. Treatments were added to cells as described in each figure. After completion of each treatment, cells were washed with PBS once. Chromatin-bound proteins were pre-extracted on ice in CSK buffer for 2 min. When possible, biological replicates for experiments were stained together. Cells were permeabilised in PBS + 0.2% Triton X-100 for 10 min and then blocked in blocking buffer (PBS + 3% BSA + 0.01% Triton X-100) for 45 min at room temperature. Primary antibodies targeting BrdU (mouse, BD Biosciences, catalogue number 555627, 1:250), RPA32 (rat, CST, catalogue number 2208 1:500) and either ATF4 (rabbit, CST, catalogue number 11815, 1:500) or γH2AX–pSer139 (rabbit, CST, catalogue number 2577, 1:500) were diluted at the indicated concentrations in blocking buffer and incubated with cells for 1 h at room temperature. Cells were washed 3× in PBS + 0.01% Triton X-100, after which secondary antibodies (goat anti-rat Alexa Fluor-647: Thermo Fisher Scientific, catalogue number A-21247, 1:1,000; HCA goat anti-mouse Alexa Fluor-488: Thermo Fisher Scientific, catalogue number A-11029, 1:1,000; HCA donkey anti-rabbit Alexa Fluor-568: Thermo Fisher Scientific, catalogue number A10042, 1:1,000) and DAPI (0.5 μg ml−1) were incubated with cells for 1 h at room temperature in the dark. Stained plates were scanned using the Operetta CLS confocal high-content system (×20). Maximum intensity projections were used for phenotypic analysis on 49 FOVs. Cell nuclei were identified by DAPI staining using Harmony 5.1 software, followed by quantification of nuclear sum and mean intensities for each staining. Raw mean BrdU–γH2AX–pSer139 and RPA32 intensities were plotted against one another in R Studio on log2(transformed) axes. Linear relationships between BrdU and RPA32 signals was derived from a linear transformation of untreated cells. RPA-exhausted cells were quantified by observing at which intensities the signal began to deviate from linearity. A threshold for all ATF4+ cells was drawn for each condition, after which the top quartile (25%) of ATF4+ cells was scored as ATF4high. All statistical tests were performed using GraphPad Prism v.10.6.1 (799). The number of biological replicates and statistical test utilized for each figure panel are available in the corresponding figure legend. No statistical method was used to predetermine sample size. No data were excluded from the analyses. The experiments were not randomized and the investigators were not blinded to allocation during experiments and outcome assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw Illumina sequencing FASTQ files related to the CRISPR–Cas9 screens performed in this study are publicly available in the National Center for Biotechnology Information Sequence Read Archive and BioProject Databases with accession number PRJNA1214387. Source data are provided with this paper. Causes and consequences of replication stress. Saxena, S. & Zou, L. Hallmarks of DNA replication stress. Magdalou, I., Lopez, B. S., Pasero, P. & Lambert, S. A. E. The causes of replication stress and their consequences on genome stability and cell fate. Zou, L. & Elledge, S. J. Sensing DNA damage through ATRIP recognition of RPA-ssDNA complexes. Berti, M. & Vindigni, A. Replication stress: getting back on track. & Cortez, D. ATR: an essential regulator of genome integrity. Toledo, L. I. et al. ATR prohibits replication catastrophe by preventing global exhaustion of RPA. Ercilla, A. et al. Physiological tolerance to ssDNA enables strand uncoupling during DNA replication. Berti, M., Cortez, D. & Lopes, M. The plasticity of DNA replication forks in response to clinically relevant genotoxic stress. García-Gómez, S. et al. PrimPol, an archaic primase/polymerase operating in human cells. Bianchi, J. et al. PrimPol bypasses UV photoproducts during eukaryotic chromosomal DNA replication. & Doherty, A. J. PrimPol is required for the maintenance of efficient nuclear and mitochondrial DNA replication in human cells. Kobayashi, K. et al. Repriming by PrimPol is critical for DNA replication restart downstream of lesions and chain-terminating nucleosides. Wan, L. et al. hPrimpol1/CCDC111 is a human DNA primase-polymerase required for the maintenance of genome integrity. Mourón, S. et al. Repriming of DNA synthesis at stalled replication forks by human PrimPol. Schiavone, D. et al. PrimPol Is required for replicative tolerance of G Quadruplexes in vertebrate cells. Bainbridge, L. J., Teague, R. & Doherty, A. J. Repriming DNA synthesis: an intrinsic restart pathway that maintains efficient genome replication. Quinet, A. et al. PRIMPOL-mediated adaptive response suppresses replication fork reversal in BRCA-deficient cells. Piberger, A. L. et al. PrimPol-dependent single-stranded gap formation mediates homologous recombination at bulky DNA adducts. Mellor, C., Nassar, J., Šviković, S. & Sale, J. E. PRIMPOL ensures robust handoff between on-the-fly and post-replicative DNA lesion bypass. Li, T., Tang, L., Kou, H. & Wang, F. PRIMPOL competes with RAD51 to resolve G-quadruplex-induced replication stress via its interaction with RPA. Šviković, S. et al. R-loop formation during S phase is restricted by PrimPol-mediated repriming. González-Acosta, D. et al. PrimPol-mediated repriming facilitates replication traverse of DNA interstrand crosslinks. Washif, M., Kawasumi, R. & Hirota, K. PrimPol-mediated repriming elicits gap-filling by template switching and promotes cellular tolerance to cidofovir. Kawale, A. S. et al. APOBEC3A induces DNA gaps through PRIMPOL and confers gap-associated therapeutic vulnerability. Dasari, S. & Tchounwou, P. B. Cisplatin in cancer therapy: molecular mechanisms of action. Sims, A. E. et al. FANCI is a second monoubiquitinated member of the Fanconi anemia pathway. Identification of the FANCI protein, a monoubiquitinated FANCD2 paralog required for DNA repair. Cohn, M. A. et al. A UAF1-containing multisubunit protein complex regulates the Fanconi anemia pathway. Olivieri, M. et al. A genetic map of the response to DNA damage in human cells. Shen, D.-W., Pouliot, L. M., Hall, M. D., Gottesman, M. M. & Sibley, D. R. Cisplatin resistance: a cellular self-defense mechanism resulting from multiple epigenetic and genetic changes. New Insights into mechanisms of cisplatin resistance: from tumor cell to microenvironment. Zoppoli, G. et al. Putative DNA/RNA helicase Schlafen-11 (SLFN11) sensitizes cancer cells to DNA-damaging agents. Murai, J. et al. SLFN11 blocks stressed replication forks independently of ATR. Boon, N. J. et al. DNA damage induces p53-independent apoptosis through ribosome stalling. Ogawa, A. et al. SLFN11-mediated ribosome biogenesis impairment induces TP53-independent apoptosis. Dynamic variations in epithelial-to-mesenchymal transition (EMT), ATM, and SLFN11 govern response to PARP inhibitors and cisplatin in small cell lung cancer. He, T. et al. Methylation of SLFN11 is a marker of poor prognosis and cisplatin resistance in colorectal cancer. Nogales, V. et al. Epigenetic inactivation of the putative DNA/RNA helicase SLFN11 in human cancer confers resistance to platinum drugs. Murai, J., Thomas, A., Miettinen, M. & Pommier, Y. Schlafen 11 (SLFN11), a restriction factor for replicative stress induced by DNA-targeting anti-cancer therapies. Mu, Y. et al. SLFN11 inhibits checkpoint maintenance and homologous recombination repair. Jo, U. et al. SLFN11 promotes CDT1 degradation by CUL4 in response to replicative DNA damage, while its absence leads to synthetic lethality with ATR/CHK1 inhibitors. Murai, J. et al. Chromatin remodeling and immediate early gene activation by SLFN11 in response to replication stress. Li, M. et al. DNA damage-Induced cell death relies on SLFN11-dependent cleavage of distinct type II tRNAs. Li, M. et al. Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Chronic treatment with cisplatin induces replication-dependent sister chromatid recombination to confer cisplatin-resistant phenotype in nasopharyngeal carcinoma. & Shekhar, M. P. Pharmacological targeting of RAD6 enzyme-mediated translesion synthesis overcomes resistance to platinum-based drugs. Precise tuning of gene expression levels in mammalian cells. Identification of a highly potent and selective DNA-dependent protein kinase (DNA-PK) inhibitor (NU7441) by screening of chromenone libraries. Liang, Q. et al. A selective USP1–UAF1 inhibitor links deubiquitination to DNA damage responses. Huang, T. T. et al. Regulation of monoubiquitinated PCNA by DUB autocleavage. Mechanism of action of the atypical retinoid ST1926 in colorectal cancer: DNA damage and DNA polymerase α. Costa-Mattioli, M. & Walter, P. The integrated stress response: from mechanism to disease. Metzner, F. J. et al. Mechanistic understanding of human SLFN11. Zhang, P. et al. Schlafen 11 triggers innate immune responses through its ribonuclease activity upon detection of single-stranded DNA. Defective ALC1 nucleosome remodeling confers PARPi sensitization and synthetic lethality with HRD. Sanson, K. R. et al. Optimized libraries for CRISPR-Cas9 genetic screens with multiple modalities. 's lab for technical assistance, careful reading of the paper and helpful discussions. This work was supported by the Cell Science (grant number CC1079), Flow Cytometry (grant number CC1062), Genomics (grant number CC1064) and Screening and Automated Science (grant number CC1071) Scientific Technology Platforms at the Francis Crick Institute. We thank K. Vousden for providing the RPE-1 p53 KO cell line and J. Méndez for providing a PrimPol antibody. This work was supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (grant number CC2057), the UK Medical Research Council (grant number CC2057) and the Wellcome Trust (grant number CC2057). laboratory is also funded by a Wellcome Trust Senior Investigator Grant (220808/Z/20/Z) and Advanced Investigator grants from the European Research Council (TelMetab 742437 and ChrEndProt 101053876). DSB Repair Laboratory, The Francis Crick Institute, London, UK Tyler H. Stanage, Shudong Li, Sandra Segura-Bayona, Aurora I. Idilli, Rhona Millar, Graeme Hewitt & Simon J. Boulton Cancer Research UK Radnet City of London Centre, UCL Cancer Institute, London, UK School of Cancer and Pharmaceutical Sciences, Comprehensive Cancer Centre, Kings's College London, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed QIBC experimental design and conceptualization. helped with CRISPR–Cas9 screening design, execution and troubleshooting. is co-founder and shareholder in Artios Pharma Limited, unrelated to this work. The other authors declare no competing interests. Nature Cell Biology thanks Juan Mendez, Annabel Quinet and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) DECODR analysis of Sanger sequencing of the PRIMPOL PR sgRNA cut site in eHAP iCas9 PRIMPOL KO C1 cells showing the cells habour a -2 bp deletion. (b) Experimental scheme for assessing inducible Cas9 activity using a lentiviral BFP/GFP flow cytometry reporter assay. (c) Representative gating strategy for all flow cytometry experiments conducted in this study. (d) Flow cytometry of eHAP iCas9 WT or PRIMPOL KO C1 cells infected with the BFP/GFP reporter shown in (B) in the presence or absence of doxycycline. (e) Sanger sequencing of the PRIMPOL AxA mutant (D114A/E116A) cDNA. (f) Sanger sequencing of the PRIMPOL CH mutant (C419H/H426Y) cDNA. (g) Scheme for introducing microRNA response elements (MREs) into the 3' UTRs of PRIMPOL expression constructs to fine-tune protein expression levels. (h) A Western blot showing PRIMPOL expression levels in eHAP iCas9 WT (lane 1) or PRIMPOL KO cells transiently transfected with PRIMPOL expression constructs containing indicated MREs. (i) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the PARP inhibitor Olaparib. (j) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the DNA polymerase inhibitor aphidicolin. (k) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the DNA polymerase alpha inhibitor ST1926. (l) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of bleomycin. (m) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the methyl methane sulfonate (MMS). (n) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the RAD51 inhibitor B02. (o) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the ribonucleotide reductase inhibitor hydroxyurea (HU). (p) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the ATR inhibitor AZD6738. (q) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of the topoisomerase I inhibitor camptothecin (CPT). (r) Dose response curves for eHAP iCas9 WT or PRIMPOL KO cells challenged with indicated concentrations of ATM inhibitor KU-5593. All experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (a) An experimental scheme demonstrating how viability assays were performed in indicated PRIMPOL KO cell lines. (b) A Western blot depicting PRIMPOL and SLFN11 protein levels in indicated cell lines and PRIMPOL KO clones. (c) A cisplatin dose response curve for HT-1080 ICas9 WT or PRIMPOL KO cells. Experiments were performed in N = 2 biological replicates. Data are represented as means ± SD. (d) Cisplatin dose response curves for NCIH-460 iCas9 WT or PRIMPOL KO clones. Experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (e) A cisplatin dose response curve for A673 Cas9-AAVS1 or A673 Cas9-PRIMPOL KO clones. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (b) Representative images of colony formation assays used to assess viability of eHAP iCas9 WT or PRIMPOL KO cells in response to cisplatin upon loss of SLFN11. (c) Quantitation of colony formation assays as depicted in (b). These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was used to calculate biological significance. (d) Experimental scheme for a lentiviral-based mCherry/GFP Cas9 flow cytometry reporter. (f) Experimental scheme for challenging A673 Cas9-AAVS1 C2 or A673 Cas9-PRIMPOL C2 cells transiently depleted of SLFN11 protein with cisplatin. (g) A Western blot showing PRIMPOL and SLFN11 protein levels in A673 Cas9-AAVS1 C2 and A673 Cas9-PRIMPOL cells transiently depleted of SLFN11 protein. Experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A one-way ANOVA was performed to assess biological significance. (i) Bar plots showing cell survival of eHAP iCas9 WT or PRIMPOL KO cells in response to treatment with 750 nM GCN2 inhibitor in the absence of cisplatin. Experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess biological significance. (j) Experimental scheme for challenging eHAP iCas9 WT or PRIMPOL KO cells with cisplatin in the presence or absence of GCN2 inhibitor following transient knockout of SLFN11. (k) A dose response curve depicting cells with indicated genotypes challenged with cisplatin in the absence (solid lines, circles) or presence (dashed lines, triangled) of 750 nM GCN2 inhibitor. Experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (l) An experimental scheme for challenging eHAP iCas9 WT, PRIMPOL KO, FANCD2 KO, or RAD18 KO cells with cisplatin in the presence or absence of 750 nM GCN2 inhibitor. (n) A bar plot depicting cell survival of eHAP iCas9 WT, PRIMPOL KO, FANCD2 KO, and RAD18 KO cells at 450 nM cisplatin compared to an untreated control in each cell line in the absence (darker bars) or presence (lighter bars) of GCN2 inhibitor. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (o) An experimental scheme for challenging eHAP iCas9 WT cells with cisplatin in the presence or absence of RAD51 inhibitor (B02) following transient knockout of SLFN11. (p) A bar plot depicting cell survival in untreated conditions in the absence or presence of RAD51 inhibitor following transient knockout of SLFN11. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (q) A bar plot depicting cell survival in cells treated with 450 nM cisplatin in the absence or presence of RAD51 inhibitor following transient knockout of SLFN11. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (a) Representative flow cytometry experiments depicting cleaved caspase-3 signal plotted against forward scatter area. Cleaved caspase-3 positive cells were gated and quantitated as depicted. (b) A scheme depicting how apoptosis was measured in eHAP iCas9 WT or PRIMPOL KO cells using cleaved caspase-3 positive cells in flow cyomtetry experiments. (c) Representative dot plots depicting how the percentage of cleaved caspase-3 positive cells were determined using flow cytometry after challenge with cisplatin in the absence or presence of 650 nM GCN2 inhibitor. (d) A bar plot depicting the percentage of cleaved caspase-3 positive cells in each indicated genotype following challenge with cisplatin in the absence or presence of GCN2 inhibitor determined using flow cytometry in (b). These experiments were performed in N = 4 biological replicates. (e) Representative Western blots demonstrating induction of DNA damage response markers in eHAP iCas9 WT or PRIMPOL KO cells in response to indicated doses of cisplatin for 24 or 48 h. (f) Experimental setup of measuring chromatin-bound phospho-gamma H2AX serine 139 using immunofluorescence in eHAP iCas9 WT or PRIMPOL KO cells depleted of SLFN11 protein (top). (g) eHAP iCas9 WT or PRIMPOL KO Cells transiently depleted of SLFN11 were mock treated or treated with 450 nM cisplatin for 24 h. Representative micrographs of chromatin-bound immunofluorescence of γH2AX phospho serine 139, DAPI, or merged are shown. (h) Bar plots showing phospho-gamma H2AX serine 139 foci signal normalised to WT untreated cells in eHAP iCas9 WT or PRIMPOL KO cells depleted of SLFN11 protein. Experiments were performed in N = 5 biological replicates. Data are represented as means ± SD. A two-way ANOVA was used to assess biological significance. γH2AX phospho serine 139 staining was performed as a co-stain with pRPA S33 depicted in Fig. 4j-l. (i) eHAP iCas9 WT or PRIMPOL KO Cells transiently depleted of SLFN11 were mock treated or treated with 450 nM cisplatin for 24 h. Representative micrographs of chromatin-bound immunofluorescence of RAD51, DAPI, or merged are shown (j) Quantification of RAD51 foci in indicated cell lines. These experiments were performed in N = 5 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess biological significance. (a) Representative micrographs depicting BrdU, RPA, and DAPI signals generated in QIBC experiments shown in Fig. (b) Representative micrographs depicting γH2AX pS139, RPA, and DAPI signals generated in QIBC experiments shown in Fig. (c) A dose response curve depicting cell viability in eHAP iCas9 WT or PRIMPOL KO cells treated with indicated doses of siRNA targeting the RPA2 gene in the presence or absence of 750 nM GCN2 inhibitor. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (a) An experimental schematic describing how cell viability assays were performed in the presence of cisplatin and ML323 (USP1 inhibitor). (b) A cisplatin dose response curve for eHAP iCas9 WT or PRIMPOL KO cells with or without 4.5 μM ML323. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (c) A bar plot showing how eHAP iCas9 WT (gray bars) or PRIMPOL KO cells (red bars) respond to 4.5 μM ML323. Survival is normalised to untreated conditions within each genotype. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (d) A schematic showing how cell viability experiments were performed in A673 Cas9-AAVS1 or A673 Cas9-PRIMPOL cells in the presence of cisplatin and ML323 (16.5 μM). (e) A bar blot depicting cell survival for A673 Cas9-AAVS1 or A673 Cas9-PRIMPOL cells in the presence of 200 nM cisplatin -/+ 16.5 μM ML323. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (f) An experimental scheme depicting how cell viability was measured after challenging cells with cisplatin and USP1 inhibitor (ML323) in eHAP iCas9 cells with indicated genotypes. (g) A bar plot depicting cell survival of eHAP iCas9 WT or PRIMPOL KO cells following challenge with 670 nM cisplatin in the absence (darker bars) or presence (light bars) of 4.5 μM USP1 inhibitor following transient knockout of SLFN11. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (a) A series of representative dot plots depicting how apoptosis was measured in cells with indicated genotypes by quantifying the percentage of cells that stained positively for cleaved caspase-3. These dot plots are related to the quantification of apoptosis shown in Fig. (a) A schematic showing how RAD18 or FANCL was knocked out in eHAP iCas9 WT or PRIMPOL KO cells and subsequently challenged with cisplatin. (b) A Western blot showing PRIMPOL and RAD18 protein levels in eHAP iCas9 WT or PRIMPOL KO cells upon loss of RAD18 (R18). (c) A cisplatin dose response curve for eHAP iCas9 WT or PRIMPOL cells upon loss of RAD18. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. These experiments were performed using the same WT sgNTC and PRIMPOL KO sgNTC samples depicted in (m). (d) A Western blot showing PRIMPOL and FANCL protein levels in eHAP iCas9 WT or PRIMPOL KO cells upon loss of FANCL (FL). (e) A cisplatin dose response curve for eHAP iCas9 WT or PRIMPOL KO cells upon loss of FANCL. eHAP iCas9 WT and PRIMPOL KO curves are the same as those shown in (d), as these experiments were performed at the same time. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (f) A series of Western blots showing indicated protein levels in eHAP iCas9 WT or PRIMPOL cells upon loss of USP1 and/or RAD18 (R18). (g) A series of Western blots showing indicated protein levels in eHAP iCas9 WT or PRIMPOL KO cells upon loss of USP1 and/or FANCL (FL). (h) An experimental scheme for how USP1/RAD18/FANCL proteins were knocked out in eHAP iCas9 WT or PRIMPOL KO cells and subsequently challenged with cisplatin in a colony formation assay. (i) Representative images of colony formation assays in indicated cell backgrounds in untreated conditions or upon treatment with 450 nM cisplatin. (j) Quantification of colony survival in the colony formation assays represented in (d) normalised to untreated samples within each genotype. These experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to test statistical significance. (k) An experimental schematic describing how chromatin-bound immunofluorescence was performed. (m) A bar plot showing chromatin-bound FANCD2 foci numbers in eHAP iCas9 WT or PRIMPOL KO cells treated with 450 nM cisplatin. These experiments were performed in N = 5 biological replicates. Data are represented as means ± SD. A two-way ANOVA was performed to assess statistical significance. (b) A dose response curve measuring cell viability in the presence of indicated concentrations of olaparib, a PARP inhibitor. (c) A dose response curve measuring cell viability in the presence of indicated concentrations of camptothecin (CPT), a topoisomerase I inhibitor. (d) A dose response curve measuring cell viability in the presence of indicated concentrations of benzo(a)pyrene-7,8-dihydrodiol-9,10-epoxide (BPDE). (e) A dose response curve measuring cell viability in the presence of indicated concentrations of methyl methanesulfonate (MMS). (f) A dose response curve measuring cell viability in the presence of indicated concentrations cisplatin. (g) A dose response curve measuring cell viability in the presence of indicated concentrations of mitomycin C (MMC). (h) A dose response curve measuring cell viability in the presence of indicated concentrations of AZD6738, an ATR inhibitor. (j) A dose response curve measuring cell viability in the presence of indicated concentrations of bleomycin. (k) A dose response curve measuring cell viability in the presence of indicated concentrations of KU-5593, an ATM inhibitor. All experiments were performed in N = 3 biological replicates. Data are represented as means ± SD. (l) A dose response curve measuring cell viability in the presence of indicated concentrations of B02, a RAD51 inhibitor. (a) Representative micrographs related to QIBC experiments shown in Fig. (b) Representative histograms depicting mean chromatin-bound ATF4 signal in indicated conditions. Supplementary Table 3 CRISPR screen, pool 1, raw counts. Supplementary Table 4 CRISPR screen, pool 2, raw counts. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Stanage, T.H., Li, S., Segura-Bayona, S. et al. RPA exhaustion activates SLFN11 to eliminate cells with heightened replication stress. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260109023114.htm'>Stanford's AI spots hidden disease warnings that show up while you sleep</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-09 07:40:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A restless night often leads to fatigue the next day, but it may also signal health problems that emerge much later. Scientists at Stanford Medicine and their collaborators have developed an artificial intelligence system that can examine body signals from a single night of sleep and estimate a person's risk of developing more than 100 different medical conditions. The system, called SleepFM, was trained using almost 600,000 hours of sleep recordings from 65,000 individuals. These recordings came from polysomnography, an in-depth sleep test that uses multiple sensors to track brain activity, heart function, breathing patterns, eye movement, leg motion, and other physical signals during sleep. While it is widely used to diagnose sleep disorders, researchers realized it also captures a vast amount of physiological information that has rarely been fully analyzed. "We record an amazing number of signals when we study sleep," said Emmanual Mignot, MD, PhD, the Craig Reynolds Professor in Sleep Medicine and co-senior author of the new study, which will publish Jan. 6 in Nature Medicine. In routine clinical practice, only a small portion of this information is examined. Recent advances in artificial intelligence now allow researchers to analyze these large and complex datasets more thoroughly. Large language models like ChatGPT use a similar approach, though they are trained on text rather than biological signals. SleepFM was trained on 585,000 hours of polysomnography data collected from patients evaluated at sleep clinics. Each sleep recording was divided into five-second segments, which function much like words used to train language-based AI systems. "SleepFM is essentially learning the language of sleep," Zou said. The model integrates multiple streams of information, including brain signals, heart rhythms, muscle activity, pulse measurements, and airflow during breathing, and learns how these signals interact. The team then pursued a more ambitious objective: determining whether sleep data could predict future disease. To do this, they linked polysomnography records with long-term health outcomes from the same individuals. This was possible because the researchers had access to decades of medical records from a single sleep clinic. Their sleep studies were recorded at the clinic between 1999 and 2024 and paired with electronic health records that followed some patients for as long as 25 years. Using this combined dataset, SleepFM reviewed more than 1,000 disease categories and identified 130 conditions that could be predicted with reasonable accuracy using sleep data alone. The strongest results were seen for cancers, pregnancy complications, circulatory diseases, and mental health disorders, with prediction scores above a C-index of 0.8. The C-index, or concordance index, measures how well a model can rank people by risk. It reflects how often the model correctly predicts which of two individuals will experience a health event first. "For all possible pairs of individuals, the model gives a ranking of who's more likely to experience an event -- a heart attack, for instance -- earlier. "We were pleasantly surprised that for a pretty diverse set of conditions, the model is able to make informative predictions," Zou said. Zou also noted that models with lower accuracy, often around a C-index of 0.7, are already used in medical practice, such as tools that help predict how patients might respond to certain cancer treatments. Future versions may incorporate data from wearable devices to expand the range of physiological signals. "It doesn't explain that to us in English," Zou said. "But we have developed different interpretation techniques to figure out what the model is looking at when it's making a specific disease prediction." The team found that while heart-related signals were more influential in predicting cardiovascular disease and brain-related signals played a larger role in mental health predictions, the most accurate results came from combining all types of data. Rahul Thapa, a PhD student in biomedical data science, and Magnus Ruud Kjaer, a PhD student at Technical University of Denmark, are co-lead authors of the study. Microscopic Robots That Swim Think and Act on Their Own Diabetes Doesn't Just Raise Sugar It Can Change Your Blood A Once-in-a-Generation Discovery Could Transform the Future of Dairy Farming Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            