
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-08</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09697-2'>Enzyme specificity prediction using cross attention graph neural networks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 15:17:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Please note there may be errors present which affect the content, and all legal disclaimers apply. Enzymes are the molecular machines of life, and a key property that governs their function is substrate specificity—the ability of an enzyme to recognize and selectively act on particular substrates. Many enzymes can promiscuously catalyze reactions or act on substrates beyond those for which they were originally evolved1,3-5. Herein, we developed a cross-attention-empowered SE(3)-equivariant graph neural network architecture named EZSpecificity for predicting enzyme substrate specificity, which was trained on a comprehensive tailor-made database of enzyme-substrate interactions at sequence and structural levels. EZSpecificity outperformed the existing machine learning models for enzyme substrate specificity prediction, as demonstrated by both an unknown substrate and enzyme database and seven proof-of-concept protein families. Experimental validation with eight halogenases and 78 substrates revealed that EZSpecificity achieved a 91.7% accuracy in identifying the single potential reactive substrate, significantly higher than that of the state-of-the-art model ESP (58.3%). This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Carl R. Woese Institute for Genomic Biology, University of Illinois Urbana-Champaign, Urbana, IL, USA NSF Molecule Maker Lab Institute, University of Illinois Urbana-Champaign, Urbana, IL, USA DOE Center for Advanced Bioenergy and Bioproducts Innovation, University of Illinois Urbana-Champaign, Urbana, IL, USA Center for Biophysics and Quantitative Biology, University of Illinois Urbana-Champaign, Urbana, IL, USA Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09521-x'>Sex and smoking bias in the selection of somatic mutations in human bladder</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 15:11:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Men are at higher risk of several cancer types than women1. For bladder cancer the risk is four times higher for reasons that are not clear2. Smoking is also a principal risk factor for several tumour types, including bladder cancer3. As tumourigenesis is driven by somatic mutations, we wondered whether the landscape of clones in the normal bladder differs by sex and smoking history. Using ultradeep duplex DNA sequencing (approximately 5,000×), we identified thousands of clonal driver mutations in 16 genes across 79 normal bladder samples from 45 people. Men had significantly more truncating driver mutations in RBM10, CDKN1A and ARID1A than women, despite similar levels of non-protein-affecting mutations. This result indicates stronger positive selection on driver truncating mutations in these genes in the male urothelium. We also found activating TERT promoter mutations driving clonal expansions in the normal bladder that were associated strongly with age and smoking. These findings indicate that bladder cancer risk factors, such as sex and smoking, shape the clonal landscape of the normal urothelium. The high number of mutations identified by this approach offers a new strategy to study the functional effect of thousands of mutations in vivo—natural saturation mutagenesis—that can be extended to other human tissues. In the interplay between mutagenesis and selection, human somatic tissues evolve as mosaics of competing clones driven by mutations, many of which affect cancer genes4,5,6,7,8,9,10,11,12. Although most of these clones do not result in cancer, in some instances they constitute the first step of the evolutionary trajectory towards malignant tumours. Therefore, characterizing the clonal landscape of human tissues can provide a path to understanding the mechanisms of cancer formation and cancer risk. In the case of the bladder, men and people with smoking history have an increased risk of developing cancer, independently of other risk factors1,2 (Supplementary Note 1). The role of smoking—whether purely mutagenic or also as promoter of mutant clones—is not well understood either. The detection of mutant clones in normal urothelium, as in other normal tissues, is challenging because clones are small and escape detection by conventional bulk sequencing methods. Here we directly probed large mixtures of clones from epithelial brushes of normal bladder using ultradeep DNA duplex sequencing (around 5,000× per sample, amounting to approximately 400,000 haploid genomes in aggregate)13,14. We focused on the detection of clones driven by mutations in 16 genes known to be under positive selection in bladder urothelium10,15, and/or mutated frequently in bladder carcinomas16,17, including the telomerase gene (TERT) promoter18. We then used the magnitude of positive selection for each gene at the sample level to regress out the effect of sex and smoking on their clonal landscape. We also considered that the capacity to identify the driver mutations of thousands of clones in a human tissue could increase markedly our ability to uncover the functional effect of all mutations in genes relevant in tumourigenesis. Identifying all potential driver mutations in a cancer gene is key to advancing personalized cancer medicine. The selective advantage provided by spontaneous mutations in millions of cells is tested in the interplay between mutagenesis and selection in normal human tissues. We should then be able to probe the results of these natural experiments by sequencing a large enough number of cells in a somatic tissue. We explore this postulate with data from ultradeep sequencing of normal bladder. As a result, we have uncovered a powerful tool with which to quantify positive selection at site resolution. We collected cells from either one or two regions—the upper (dome) and lower (trigone)—of the bladder urothelium from 45 deceased donors (79 samples) through a brushing of relatively large surface areas (roughly 2 cm2) at autopsy (Fig. Clinical data, including exposure to known or suspected bladder cancer risk factors such as tobacco smoking or chemotherapy, was available for most donors (Supplementary Table 1). We used ultradeep DNA duplex sequencing13,14 (average depth ranging between 1,236 and 9,303 across samples, with a median of 5,164; Supplementary Note 2) and a newly developed and carefully calibrated computational pipeline to identify somatic mutations with high sensitivity and specificity in these polyclonal samples (Extended Data Fig. The accuracy of this technology allows the reliable detection of mutations that are present in a single DNA molecule (Extended Data Fig. a, Schematic representation of sampling and duplex DNA sequencing of polyclonal epithelial brushes from normal bladders. b, Number of SNVs detected in a panel of selected genes in the normal bladder in this study and comparison with the number of mutations detected in 892 tumours from bladder cancer genomics studies, obtained from intOGen24. c, Number and density of somatic mutations (SNVs, MNVs and indels up to 100 base pairs) identified in 79 samples of normal urothelium obtained from 45 donors. Bladder location (dome or trigone), age, sex and smoking history of donors are shown below the bar plots. d, Trinucleotide substitution profiles of the mutational signatures identified across this cohort of normal urothelium samples through de novo extraction. e, Top, scatter plot representing the relationship between the activity (number of mutations contributed) of SBS-ageing in samples and the age of donors; effect size, regression line and P value of a univariate mixed-effects linear model. Bottom, box plot representing the activity of SBS-chemo in donors exposed or not exposed to chemo/radiotherapy; effect size and P value of a univariate mixed-effects linear model. BioRender was used to create panel a (https://BioRender.com/fgnnet9). To investigate how bladder cancer risk factors might change the clonal landscape of the urothelium, we focused on 15 genes identified previously to be under positive selection in normal bladder10,15 and/or known to frequently drive bladder tumours16,17 and the promoter of the gene encoding TERT, which is mutated in around 70% of bladder tumours18,19,20,21,22,23. This amounted to 111,876 base pairs of genomic DNA comprising the exonic regions and neighbouring intronic sites of the 15 genes (in full or selected fragments) and a region of the TERT promoter (hereinafter for simplicity, genes; Supplementary Tables 2, 3 and 4). Across samples, we identified a total of 64,278 mutations, between 54 and 1,785 (median 774) mutations per sample, including single nucleotide variants (SNVs), multiple nucleotide variants (MNVs) and indels up to 100 base pairs (Supplementary Table 5). This total number of mutations is approximately 16-fold higher than the number identified in the same genes across 892 bladder tumours (intOGen)24 sequenced over a decade of cancer genomics (Fig. The overall non-protein-affecting mutation density—mutations per megabase sequenced—was associated, as expected, with age and the exposure to certain mutagenic factors, such as several chemotherapies/radiotherapy (Supplementary Note 5). Using two orthogonal signature extraction methods25,26,27, and supported by the detection of a large number of non-protein-affecting mutations, we discerned the activity of three main mutational signatures (Supplementary Note 5 and Extended Data Fig. These captured the activity of the APOBEC family of cytidine deaminases28 (SBS-APOBEC; Fig. 1d,e) and an unknown process (harder to reduce to a linear combination of mutational signatures in the COSMIC catalogue30) that seemed to be associated with exposure to chemotherapy (SBS-chemo; Fig. Overall, the landscape of mutational signatures and the number of mutations per megabase sequenced was similar to that identified in a previous study of normal urothelium10 (Extended Data Fig. 2b,c), with the exception of a signature reported to be associated with tobacco smoking. Mutational signatures were correlated highly between the dome and trigone of the same donor (Supplementary Note 5). The 16 genes included in the panel had been demonstrated previously to be under positive selection in normal urothelium or mutated frequently in bladder tumours. Thus, we first examined whether their mutations were positively selected in the samples included in this cohort. To this end, we used four complementary computational methods (Supplementary Note 6) on the basis of comparing features of the pattern of mutations observed in a gene with that expected assuming neutral evolution31,32. First, we estimated the expected number of protein-affecting mutations with different impact, that is, missense or truncating (including nonsense and essential splice-site mutations) on the basis of the number of observed synonymous mutations in each gene across samples and in each sample using a newly developed method called Omega (Fig. This calculation takes into account the number of sites allowing amino acid changes of each type in the protein, the frequency of trinucleotide changes observed across samples and the depth at which each individual genomic site has been sequenced. Then, using a dN/dS approach33, Omega computes the degree of positive selection on missense (dN/dS missense) or truncating (dN/dS truncating) mutations. In addition to the excess of truncating and missense mutations, we quantified three other signals of selection: clustering of mutations in the three-dimensional structure of the protein, functional impact bias and excess of frameshift indels (Supplementary Note 6 and Extended Data Fig. These metrics indicated that RBM10, KDM6A, STAG2, KMT2D, ARID1A, CDKN1A, TP53, EP300, NOTCH2, CREBBP, FOXQ1, KMT2C and RB1—13 of the genes included in this study—are under positive selection, driving clonal expansion across the 79 normal urothelium samples (Extended Data Fig. We verified that the same is true for PIK3CA through analysis of the mutations in a hotspot affecting the histidine at position 1,047 (Extended Data Fig. a, Distribution of truncating, missense and synonymous somatic mutations along the coding regions of RBM10 and TP53. Each needle represents the number of samples with mutations with one of the three consequences occurring on an amino acid residue. Right, percentage of driver mutations (observed in excess of neutrality) among truncating and missense in RBM10 and TP53. The number of observed, expected and estimated driver mutations are indicated for each gene. c, Magnitude of selection on truncating and missense mutations in FGFR3. Left, distribution of somatic mutations along the sequence of the TERT promoter colour coded according to whether they are activating (seen in tumours). Right, bar representing the magnitude of positive selection on activating TERT promoter mutations. In dN/dS bar plots (b–d), the shaded segment in each bar represents the number of truncating or missense (or activating in the case of TERT) mutations expected under neutrality. The P values in b–d were calculated using a dN/dS approach (Omega) described in Supplementary Note 6. This analysis produced two additional findings. First, we discovered that truncating (both SNVs and frameshift indels) mutations of FGFR3 seem to be under negative selection (Fig. Second, we observed 85 SNVs affecting the TERT promoter (across 42 samples; Fig. The TERT gene encodes telomerase, and its promoter includes two hotspots that result in the creation of two de novo ETS transcription factor binding sites34 and are mutated frequently in human tumours18,19,22,23. To formally compute the strength of positive selection on the TERT promoter in our cohort, we dichotomized the mutations on the basis of whether or not they have been observed several times across 8,136 tumour whole genomes (Supplementary Note 6; Methods), that is, using their presence in tumours as a surrogate of their capacity to activate the expression of TERT. We could thus calculate a modified dN/dS value for the TERT promoter (dN/dS pTERT; Supplementary Note 6). Although we expected fewer than one activating mutation, we actually observed 56, implying that at least 55 (98.2%) of these mutations detected in the TERT promoter are drivers of clonal expansion, probably by activating the expression of telomerase in normal urothelial cells. We next computed positive selection separately for each sample in the cohort (Extended Data Fig. The high depth of sequencing in the study guarantees that these sample-level dN/dS values can be estimated accurately and robustly (Supplementary Notes 7 and 8). The number of clones driven by mutations detected in each of the 13 genes under positive selection varies between zero and several hundreds across the samples analysed from each donor (Fig. Despite the abundance of driver mutations, the relative size of the clones is very small and thus the fraction of the urothelium covered by mutant clones of these genes is modest (less than 1%) in most cases (Extended Data Fig. Some activating TERT promoter mutations reach relatively high values of variant allele frequency (VAF), probably due to both clonal expansion and convergent evolution. a, Heatmaps with number of driver SNVs (top) and the total number of protein-affecting indels (bottom) in 13 genes across female and male donors. For donors with dome and trigone samples (marked with a dot), the number of driver SNVs in both samples has been added. Donors are sorted by age in ascending order. b, Relationship between age and the density (per Mb) of protein-affecting and non-protein-affecting mutations. c, Association of age with the protein-affecting (top coefficient) and non-protein-affecting (bottom coefficient) mutation density (left), and the magnitude of positive selection on missense (centre) and truncating (right) mutations of 13 genes (gene–mutation consequence combinations for which we could calculate dN/dS at the level of sample for at least 80% of samples were included in the calculation). Horizontal lines denote the median. e, Association of sex with the protein-affecting (top coefficient) and non-protein-affecting (bottom coefficient) mutation density (left), and the magnitude of positive selection on truncating mutations in RBM10, CDKN1A, ARID1A and STAG2 (right). The plots represent the results of multivariate regressions accounting for age, smoking history, alcohol drinking history, BMI and exposure to chemotherapy. In all regressions (c and e), multivariate linear mixed-effects models (accounting for sex, smoking history, alcohol drinking history, BMI and exposure to chemotherapy) were used to account for the presence of several samples of the same donor, and these models yielded the effect size and P values. Circles with dark outer circumference denote significant associations (false discovery rate (FDR) threshold of 0.2). N = 79 samples in b, c and e. All corrected P values for these regressions appear in Supplementary Table 7. We also found that the patterns of positive selection of the dome and trigone from the same donor were strikingly similar across the entire cohort (Extended Data Figs. In summary, applying a dN/dS-based approach to mutations detected across urothelial samples through ultradeep sequencing, we demonstrated that a large percentage of mutations in the profiled genes are drivers, indicating pervasive positive selection; we uncovered negative selection for truncating mutations in FGFR3; we showed positive selection for activating mutations in the TERT promoter and we made robust estimations of positive selection and number of driver mutations per gene in each sample, thus enabling the study of inter-individual differences in clonal selection (Fig. To identify a potential sex bias of the urothelial clonal landscape, we resorted to regressions that incorporated meaningful covariates, in particular age and history of smoking1,35,36,37 (Supplementary Note 1). Indeed, part of this heterogeneity across donors is explained by age. The density of protein-affecting mutations increases at a higher pace than that of non-protein-affecting mutations, indicating that clones driven by mutations of the 13 genes under positive selection expand with age, as reported for other normal tissues7,8,9,11,12,38,39 (Fig. In agreement with this, the dN/dS values of missense and truncating mutations of these genes are also associated significantly with age (Fig. Using multivariate regressions, we determined that the values of dN/dS truncating of RBM10, CDKN1A and ARID1A are increased significantly in the urothelium of men compared with women, independently of age, smoking history, alcohol drinking history, body mass index (BMI) and exposure to chemotherapy (Fig. In addition, the dN/dS truncating of STAG2 shows a tendency, although not significant, towards higher values in men. Two of these genes (RBM10 and STAG2) are in the X chromosome, whereas CDKN1A and ARID1A are in autosomes. KDM6A is another X chromosome gene that has been described previously with a higher number of mutations in female bladder tumours40. A closer inspection of X chromosome genes confirms the expectation of higher coverage across women for all three, although only RBM10 (and STAG2 to a lesser extent) shows a sex bias of dN/dS truncating (Supplementary Note 9). This implies that the growth of clones driven by mutations under positive selection in these genes is different in the urothelium of men and women (Supplementary Notes 9 and 10). The number of bladder tumours with truncating mutations in RBM10 and CDKN1A is also significantly higher across men than women (Extended Data Fig. We discovered that activating mutations in the promoter of TERT (that is, those observed at least twice in tumours18,19,20,21,22,23,34), were much more recurrent among older donors in the cohort. Specifically, all TERT promoter-activating mutations in the cohort, except one, occurred across donors older than 55 years (Fig. Furthermore, in this group, those with a history of smoking exhibited a significantly higher rate of TERT promoter mutations (Fig. Some of these mutations showed relatively large VAF, probably caused by several expanding clones with these mutations (Extended Data Fig. We found that the association between smoking history in older donors and the rate of TERT promoter-activating mutations is independent of sex, BMI and exposure to chemotherapy (Fig. The significant increase of activating TERT promoter mutations in smokers contrasts with the absence of association between smoking history and overall mutation density across samples (Supplementary Note 5 and Supplementary Table 7). This result indicates that tobacco carcinogens have a role in promoting the growth of clones with TERT promoter mutations. In summary, the rate of TERT promoter-activating mutations in the normal urothelium is associated significantly with the interaction of age and a history of smoking. More than 70% of bladder tumours have TERT promoter mutations, with a comparable frequency in smokers and non-smokers. In lung tumours, where the frequency of TERT promoter mutations is much lower, we find a significantly higher frequency in smokers (2.1%) compared with non-smokers (0.9%) (Supplementary Note 11). Collectively, these results indicate that exposure of the normal urothelium to tobacco carcinogens may increase the risk of bladder cancer through the promotion of clones bearing mutations of the TERT promoter. a, Number of activating (observed in tumours) and other (not observed in tumours) mutations in the TERT promoter across donors younger than 55 years (top), older than 55 years with no history of smoking (middle) or older than 55 years with a history of smoking (bottom). b, Density (mutations per Mb) of activating TERT promoter mutations observed in the three groups of donors. The sample size of the three groups was 25 (samples from donors younger than 55), 14 (samples from donors older than 55 with no smoking history) and 38 (samples from donors older than 55 with smoking history), respectively. The two samples from a donor with unknown smoking history were excluded. c, Association between the frequency of activating mutations in the TERT promoter and the interaction between age and smoking history. A linear mixed-effects model was used to compute the P value. The dark outer circumference denotes a significant association (FDR threshold of 0.2). The corrected P value appears in Supplementary Table 7. Number of samples for this analysis is the same as in a and b. Experimental saturation mutagenesis (that is, the insertion of all possible mutations in a genomic element in experimental systems) has been used to identify sites or regions in proteins that are key for their structure, function or regulation, to understand their role in disease and improve the design of drugs to target them41,42,43,44,45. Recently, we introduced the concept of in silico saturation mutagenesis, exploiting more than 20 years of cancer genomics research on the natural experiments that are human tumours (or clonal expansions in normal blood) to build machine learning models that identify all their potential driver mutations46,47. In contrast to tumours, whose evolutionary history entails the expansion of a single clone, with only one or two mutations observed in a cancer driver gene in most cases, in normal polyclonal samples, several mutations affecting a given gene are detected as drivers of the expansion of several clones. Therefore, sequencing normal human tissues, which can also be considered natural experiments of clonal evolution, can provide a pathway towards natural saturation mutagenesis. To understand how far along the ultradeep sequencing of roughly 400,000 haploid genomes takes us in the path towards natural saturation mutagenesis, we first calculated the number of mutations observed in each amino acid residue of the protein-coding genes (Fig. For example, we detected mutations (missense or nonsense) on 230 (58%) of the 394 amino acid residues of TP53 protein product covered by duplex sequencing (Fig. In normal urothelium, the frequency of mutations by gene differed from those in bladder cancers (Extended Data Fig. 9a) and the proportion of mutated residues was larger in all genes tested (Extended Data Fig. 9b), demonstrating different selective pressures and the advantage of studying mutations in normal tissue to reach saturation mutagenesis. a, Percentage of amino acid residues in each gene with zero, one, two or three or more mutations across the 79 samples. b, Theoretical and observed kinetic of natural saturation mutagenesis for TP53, EP300 and FGFR3 as cumulative depth of sequencing increases. Grey line, theoretical kinetic of saturation mutagenesis (assuming no selection). Red circle, saturation achieved across the cohort. Red dashed line, observed kinetic (obtained by downsampling). c, Natural saturation mutagenesis of TP53 in normal bladder urothelium. From top to bottom, distribution of truncating, missense and synonymous mutations along the coding sequence of the gene; site selection computed for each amino acid residue of TP53 protein product; solvent accessibility along the protein sequence; TP53 protein product domains; duplex sequencing depth per amino acid residue. Left top, TP53 protein product three-dimensional structure with significant site selection of residues highlighted in blue. Only mutations with experimental functional impact reported in ref. e, dN/dS truncating and dN/dS missense values for each domain of TP53 protein product. Solid border represents significant dN/dS values (P value < 0.05) according to Omega (Supplementary Note 6); N = 79 samples. f, Natural saturation mutagenesis of the TERT promoter. From top to bottom, distribution of mutations; site selection computed for observed mutations; experimental functional impact values of mutations in the TERT promoter according to ref. 55; distribution of mutations observed in the TERT promoter across 8,136 tumours (Supplementary Note 6). g, Experimental functional impact55 of mutations not observed, observed or observed in the TERT promoter with significant site selection across the 79 samples. h, Relationship between site selection and experimental functional impact value55 of all mutations observed in the TERT promoter. Next, we asked how many more genomes would need to be sequenced to observe mutations on all amino acid residues of each gene. To estimate this, we first calculated the probability to observe each mutation in a gene under neutrality, on the basis of the number of synonymous mutations and the trinucleotide mutational probabilities observed in our cohort. Thus, we obtained a theoretical curve describing the fraction of all possible mutations that are expected to be observed at different sequencing depths (theoretical kinetic of natural saturation mutagenesis; Fig. 5b (continuous curve), Extended Data Fig. The shape of this curve is determined by the difference in probability across mutations, governed by their trinucleotide contexts, with some mutations much more likely to occur than others. For example, for TP53, with the accumulated depth provided by the entire cohort (approximately 500,000×), only on the basis of neutral mutagenesis, we expected to have observed around 26% of all possible mutations. In theory, being able to detect every possible mutation in the gene would require an accumulated depth of around 107, assuming an aggregated mutation density and mutational signatures as the samples in the study. Nevertheless, the observation of mutations in normal urothelium is not neutral, but instead is strongly affected by selection. To obtain the actual curve of the fraction of observed mutations depending on the number of sequenced genomes (observed kinetic of natural saturation mutagenesis), we sub-sampled the number of mutations observed in each gene across the cohort, respecting their VAF but reducing the depth of sequencing at each position (dots in Fig. For example, as missense and truncating TP53 mutations are under positive selection, we actually observed more mutations than predicted by the theoretical kinetic. For several genes, we observed this faster accumulation of mutations than expected under neutral mutagenesis due to positive selection (Extended Data Fig. The curve for FGFR3 falls below the theoretical curve constructed under purely neutral mutagenesis (Fig. 5b), which supports the notion that FGFR3 mutations are under negative selection in the normal urothelium. Approaching natural saturation mutagenesis provides an opportunity to calculate the strength of positive selection on mutations affecting each amino acid residue (Supplementary Note 6), and for other within-gene structures such as exons or protein domains (Fig. In the case of TP53, most sites under significant selection are in the p53 DNA-binding domain48 (Fig. 5c), and are observed more frequently across tumours24,49,50 (Extended Data Fig. 10b), more likely to be identified as cancer drivers46 (Extended Data Fig. 10b) and tend to score higher on an experimental saturation mutagenesis assay51 (Fig. In this and other genes that act as tumour suppressors, the mutations with significant site selection appear in buried areas of the protein (Fig. The calculation of dN/dS at the level of domains shows that only the p53 DNA-binding domain exhibits a significant excess of missense mutations (Fig. By contrast, TP53 dN/dS truncating values present a more homogeneous distribution across exons and domains (Fig. The same is true for other genes such as EP300 and CREBBP where only the histone acetyltransferase domain (HAT-KAT11) shows significant dN/dS missense values (Extended Data Fig. For RBM10, with a much stronger signal of truncating mutations, residues with the highest site selection are distributed more uniformly than in TP53 (Supplementary Fig. Genes affected mostly by truncating driver mutations, such as STAG2 and RBM10, show a higher dN/dS truncating than dN/dS missense values for virtually all exons, except the first and last (Supplementary Fig. Despite the fact that almost all genes sequenced are tumour suppressors, in all cases, a few SNVs exhibit much stronger site selection than the rest (Extended Data Fig. Whereas FGFR3 mutations showed negative selection overall (Fig. 2c), mutations affecting a single residue (G380) were positively selected in normal urothelium. This residue did not correspond to the three most common FGFR3 hotspots observed in bladder cancer (Supplementary Fig. TERT promoter mutations with significant site selection in normal urothelium appear more frequently across tumours (Extended Data Fig. 10e), and present overall higher experimental functional impact values55 (Fig. In summary, natural saturation mutagenesis, brought about by the possibility of identifying mutations at extremely low VAF through ultradeep sequencing of normal tissues, opens up the possibility of directly assessing the functional impact of mutations affecting different genes in human tissues. The reasons behind the wide heterogeneity in clonal landscape observed across normal human tissues from different people7,8,9,10,11,12,38,39 and the potential role of known cancer risk factors are not understood. In this study we demonstrated the suitability of ultradeep DNA duplex sequencing13,14 (approximately 400,000 haploid genomes) to accurately measure the magnitude of positive selection on each of 16 genes in 79 individual samples (Supplementary Note 6). The high number of mutations identified proved key to showing associations between clonal expansions and bladder cancer risk factors. Specifically, we discovered that the magnitude of positive selection on truncating mutations in RBM10, ARID1A and CDKN1A is significantly higher among men even when adjusting for age, smoking history, alcohol drinking history, and BMI. Importantly, mutations in RBM10 and CDKN1A are also significantly more abundant in bladder cancers of men than of women (Supplementary Note 11). We can speculate that the observed differences between men and women could be related to differences in exposure to internal factors (for example, sex hormones) or extrinsic exposures that disproportionately affect men. Discerning whether the bias towards the expansion of clones bearing mutations in these and other genes is somehow related with the observed increased risk of bladder cancer across men requires further investigation. We also found a high increase of TERT promoter mutations among older people with a history of smoking, which may explain at least part of the link between smoking and increased bladder cancer risk35,36,37, given the high frequency of TERT promoter mutations across bladder tumours18,19,20,21,22,23,34. In this cohort, we do not find a specific smoking-associated mutational signature. The high increase of activating TERT promoter mutations in smokers, rather than a general increase in mutation density, points to a non-mutagenic—acting rather as promoter56,57,58—role of tobacco smoking in bladder tumourigenesis. Scaling up these analyses to probe the influence of other bladder cancer risk factors requires the possibility of obtaining samples in a minimally invasive manner, which could be conceivably achieved through urine or lavages59. The proposal of a pathway towards the study of natural saturation mutagenesis through ultradeep error-correcting duplex DNA sequencing of large mixtures of clones from normal tissues is a very compelling one. In this study we have provided a strong argument for the plausibility of this approach, paving the way for future studies. We have shown that it is possible to obtain a fine map of the degree of selection acting on mutations at individual sites or regions of a protein throughout the normal development of tissues of people exposed to different external factors. Saturation mutagenesis—a key development towards personalized cancer medicine60—through ultradeep sequencing in human tissues, thus provides a cost-effective and direct alternative to complement experimental and in silico approaches. In addition, characterization of the patterns of positive (and negative) selection in different genes and human tissues will be critical to understanding how somatic evolution influences cancer risk. Two epithelial brushes (2–3 cm2) from the bladder top (dome) and the bladder floor (trigone) were obtained from 53 people without known bladder pathology and no history of bladder cancer upon autopsy (average 4 days post-mortem) at the University of Washington. Next of kin consented to autopsies and research on leftover specimens. The study of de-identified collected specimens and linked clinical history from the deceased donors was reviewed and deemed not human subjects research by the University of Washington Institutional Review Board (STUDY00016707; IRB Federal Wide Assurance number, FWA 00006878). We obtained the following relevant clinical information for all donors: age, sex, BMI, tobacco smoking history, alcohol use, previous cancer and chemotherapy exposure (Supplementary Table 1). Three donors with active or chronic inflammation and four donors with insufficient DNA were discarded. The two samples from another donor were also discarded from the study upon visual inspection of their mutational profile, resulting in a total of 45 deceased donors in this study (Supplementary Notes 2 and 3). We designed a panel including ten genes identified in a previous study as being under positive selection in the normal urothelium and with more than ten mutations across the 1,647 microbiopsies analysed10. We added five genes that are mutated frequently in bladder tumours30 and the TERT promoter—also known to be under positive selection in bladder carcinomas22,23. This resulted in a panel containing the entire (or almost entire) coding region of 12 genes (ARID1A, NOTCH2, FOXQ1, CDKN1A, KMT2D, RB1, CREBBP, TP53, EP300, KDM6A, RBM10 and STAG2), three genes for which only selected regions were targeted because of clustering of cancer mutations in those regions and/or difficulties for capturing the full gene (PIK3CA, FGFR3 and KMT2C) and the TERT promoter10,61. The panel was constructed by TwinStrand Biosciences and covered 111,876 base pairs, including 65,086 base pairs in coding regions and 46,790 base pairs in non-coding regions (Supplementary Tables 2 and 3 and Supplementary Note 2). After centrifugation of epithelial brushes, DNA was extracted using the DNeasy Blood and Tissue (Qiagen) kit, following manufacturer's instructions with some variations (Supplementary Note 2). The DNA integrity number (DIN) was measured using Agilent 4200 TapeStation Genomic tapes. Duplex sequencing libraries were prepared using commercially available kits (TwinStrand Biosciences)13,14,62,63,64 and 250 ng of genomic DNA. The DNA fragmentation step was carried out taking into account the starting DIN of samples and monitored using TapeStation. Fragmented DNA was subject to end-repair, A-tailing, ligation to duplex sequencing adaptors, library conditioning and PCR amplification, according to protocol. The PCR product was captured at 65 °C for 16–20 h. Upon PCR amplification, libraries were pooled for sequencing. Sequencing was performed with a NovaSeq 6000 at the Department of Laboratory Medicine and Pathology at University of Washington or a NovaSeq X Plus at Novogene or the Fred Hutchinson Cancer Center using 2 × 150 base-pair paired-end reads (around 115 million reads per sample; Supplementary Note 2). Duplex DNA sequencing was successful for 34 donors on both samples. For 11 other donors, we produced duplex DNA sequencing data only for dome or trigone because of insufficient DNA or too fragmented DNA (DIN < 1.4) in the paired sample. Thus, the final number of donors with available data for at least one sample was 45 (Supplementary Table 1) and the final number of samples processed was 79 (Supplementary Table 4). We constructed a computational pipeline (deepUMIcaller) in Nextflow65 to call mutations from duplex sequencing data on the basis of an early version of nf-core/fastquorum pipeline66, which implements the fgbio Best Practices FASTQ to Consensus Pipeline (https://github.com/fulcrumgenomics/fgbio/blob/main/docs/best-practice-consensus-pipeline.md) and downstream variant calling by VarDictJava (https://github.com/AstraZeneca-NGS/VarDictJava). A series of filters to discard potential artefacts are included in the pipeline. Code implementing deepUMIcaller is available at (github.com/bbglab/deepumicaller). For a detailed description of the pipeline, see Supplementary Note 3. To estimate the error rate of TwinStrand DNA duplex sequencing, we compared the density of mutations and mutational profile identified across three cord blood samples (purchased from StemCell) with that expected on the basis of colonies obtained from human haematopoietic stem cells67,68. The estimated error rate resulting from this analysis was around 4 × 10−8, which is two orders of magnitude lower than the mutation density across samples in this study (Extended Data Fig. We extracted mutational signatures de novo with a Bayesian hierarchical Dirichlet process using HDP_sigExtraction pipeline (https://github.com/McGranahanLab/HDP_sigExtraction) and the R-package hdp developed by N. Roberts (https://github.com/nicolaroberts/hdp)9 and SigProfilerExtractor (https://github.com/AlexandrovLab/SigProfilerExtractor)26. The HDP_sigExtraction pipeline was run with the default parameters with no previous signatures assigned. Five signatures in addition to the null signature were extracted. De novo signatures were extracted using SigProfiler using the nonnegative matrix factorization approach. The same input data were used. The upper bound for the number of signatures was set to ten; however, the most robust solution was three signatures that were similar to the three most active signatures extracted by HDP. These two sets of mutational signatures were decomposed into known COSMIC signatures when possible (Supplementary Note 5). To account for biases due to the trinucleotide composition of the panel, the number of substitutions of each class was re-calculated. The rate of each substitution was calculated as the number of observed substitutions with the consequence in question divided by the number of corresponding sequenced trinucleotide sites (number of trinucleotides with this consequence in the panel weighted by sequencing depth). The mutational probability of each substitution for the whole genome was calculated by dividing the expected number of substitutions with this consequence per genome by the sum of all expected substitutions per genome. We tested for possible associations of de novo extracted signatures with age, sex and bladder location. To do this we used linear mixed-effect regression models (Supplementary Note 5). We tested both the number of mutations attributed to the signature (counts) and the relative contribution (proportion) of each signature as a response variable. Age, sex and bladder location were used as fixed effects (separated regression was prepared for each fixed effect variable) and donors were used as random effects to control for non-independence between dome and trigone samples from the same person. P values for the association were obtained using likelihood-ratio tests (ANOVA function) comparing models including and excluding the variable of interest. Associations with all other clinical variables (smoking history, drinking history, chemotherapy history, and so on) were tested in the same way but always including age, sex and bladder location in the regression together with the variable of interest. We used four methods to compute positive selection on the mutations observed across genes. One method (Omega)—a dN/dS approach to assess the strength of selection on the mutational pattern of genes—was developed de novo for this study and is described at length in Supplementary Note 6 (https://github.com/bbglab/omega). Two others, OncodriveFML and Oncodrive3D, which compute the deviation in the average functional impact and clustering in the three-dimensional structure of proteins, respectively, from those expected under neutrality, had been developed previously, and were adapted here to work on duplex sequencing data. These are also described in Supplementary Note 6. A fourth method, assessing the relative enrichment for frameshift indels observed across genes was also developed de novo for this study and is described thoroughly in Supplementary Note 6. For the TERT promoter, all mutations that were observed at least twice across 8,136 whole-genome sequencing tumour samples sequenced by the Hartwig Medical Foundation and the Pan-Cancer Analysis of Whole Genomes consortium (Supplementary Note 6) were considered activating. The remaining mutations were used to calculate the expected number of activating mutations under neutrality. The number of missense and truncating driver mutations of each gene in a sample was obtained from the dN/dS missense and dN/dS truncating values. From these values the number of missense and truncating mutations in excess in each sample were calculated. Specifically, the 95% confidence intervals of both dN/dS values were used to compute two extreme values of driver mutations in each sample, whereas the mean dN/dS value was used to compute an expected number of driver mutations. This mean number was thus used to select the most likely driver mutations in the sample. To this end, mutations were ranked in descending order according to their VAF, and the top number of mutations corresponding to the expected number of driver mutations were selected. Then, for a gene G, we computed the fraction of genomes bearing driver mutations out of those sequenced at each genomic position covered by the DNA duplex sequencing as: where px is the all-molecules VAF of a driver mutation x (considering both duplex and non-duplex reads; Supplementary Note 3) and n is the number of driver mutations from the previously selected set in the gene. The fraction of genomes with driver mutations is equal to the fraction of cells with driver mutations under the assumption that two mutations, one in each homologous copy of the gene, are required to drive the clonal expansion. This extreme would be true in the case that all studied genes behave as classical tumour suppressors and bear deleterious mutations in both alleles. An exception to this rule are the mutations in genes in the X chromosome in men, whereby one mutation would suffice to drive the clonal expansion. In general, it is possible for some genes to be capable of driving the clonal expansion (or mutations at specific positions in a gene) with only one mutated allele. It is also possible that the other allele is affected by a large deletion or methylation event not seen by DNA duplex sequencing. In the extreme that only one mutation per cell per gene is required, the fraction of cells with driver mutations will be double the fraction of genomes. 5b, we use the two-hit assumption. We designed a two-step strategy to assess the influence of known bladder cancer risk factors on features of the urothelium clonal structure across samples from donors in the cohort. As dependent variables representing the urothelium clonal structure, we selected the (protein-affecting and non-protein-affecting) mutation density, and the magnitude of positive selection on genic mutations (dN/dS missense and dN/dS truncating). As the dependent variables are continuous, we chose linear regressions, specifically mixed-effects linear models, to account for two samples from the same donor and use all available samples. As independent variables, we included a list of available clinical features: age (in decades), sex, smoking history (binarized as ever or never smoker), alcohol consumption history (also binarized), BMI (re-scaled within the 0–1 interval), and exposure to chemo/radiotherapy (binarized). For the case of TERT promoter mutation density association, we used the interaction of age and smoking history instead of age and smoking history separately. In the first step, we applied univariate mixed-effects linear models to identify any clinical feature with a significant association with any of the dependent variables for any gene or for all genes. In the second step, we applied multivariate mixed-effects linear models to rule out confounding effects for the associations that seemed significant in the first step. Associations with FDR below 0.2 were deemed significant. We also carried out a binomial test to rule out a spurious dependence of the mutation density on group differences in terms of sequencing depth. For details, see Supplementary Notes 9 and 10. Mutations identified across 622 muscle-invasive and 105 non-muscle-invasive bladder cancer (MIBC and NMBIC, respectively) cohorts were downloaded from cBioPortal49,70,71,72,73 together with the clinical data of the combined study. From the MIBC Beijing Genomics Institute cohort74 only samples labelled as invasive were considered MIBC and added to the MIBC dataset. Mutations identified in a further cohort of 79 NMIBCs were obtained from the literature40 and included as part of the NMIBC dataset. Only non-silent protein-affecting mutations in 14 of the genes included in the panel were analysed. In case of duplicated samples across MIBC and NMBIC cohorts, the mutations from the most recent published study were kept. In total, mutations across 806 bladder tumours were obtained (622 MIBC and 184 NMIBC). The mutation density per gene across the two datasets was calculated by dividing the number of observed mutations per gene by the length of its coding region. To compute a mutation density metric comparable with that used in the normal urothelium that accounts for the number of megabases sequenced, we multiplied the coding length by two times the number of samples in which that gene was sequenced, to take into account the two copies in a diploid genome. Under the assumption that, in tumours each mutation belongs to a single clonal expansion and in normal tissues we have a mixture of clones, we reasoned that the number of different tumour genomes sequenced would be equivalent to the number of genomes sequenced in normal urothelium. The mutation density per megabase was calculated multiplying the mutation density by 106. These mutations were used to obtain the total number of mutations observed in each gene, their distribution along the sequence of the genes in the study and the percentage of sites affected by different numbers of mutations. Mutations in the TERT promoter were also obtained from two cohorts of tumours (included in intOGen) sequenced at the whole-genome level (Hartwig Medical Foundation21: N = 5,582 and Pan-Cancer Analysis of Whole Genomes19: N = 2,554). The classification (and score) of all possible mutations in TP53 into drivers and passengers through in silico saturation mutagenesis was obtained from boostDM (intogen.org/boostDM). Details of analyses involving tumour mutations appear in Supplementary Note 11. To compare the distribution of somatic mutations along the sequence of each of the genes in normal urothelium and bladder tumours, we first downloaded somatic mutations identified across 892 bladder tumours from the intOGen (intogen.org) platform. Although this cohort is larger than the 806 muscle-invasive and non-muscle-invasive tumours used to explore differences in mutation density, it is composed mostly of muscle-invasive carcinomas. For each gene, we started the analysis with all genomic sites in the coding sequences and splicing sites that passed the pipeline's filtering criteria of sufficient duplex coverage across samples (Supplementary Note 3). We then mapped these DNA sites to protein positions using the Ensembl REST API75, allowing us to determine the total number of protein positions (including amino acid residues and stop codon) covered by the panel. Then, we computed the number of mutations affecting each protein residue of every gene studied here across the 79 normal urothelium samples and the 892 bladder tumours. Next, we obtained the consequence type (missense, synonymous, nonsense or splice-affecting) of these mutations on the MANE transcript76 of each gene from the output of the Variant Effect Predictor v.111 (ref. 77), run within the intOGen pipeline24 (https://github.com/bbglab/intogen-plus). 5b is described in Supplementary Note 12. The residue-level sequencing depth shown in Fig. 2 and 3 was calculated as the average depth across all sites corresponding to each codon. We developed a metric to measure selection per site by comparing the observed with the expected number of mutations at each site or residue. The expected number of mutations per site was obtained by distributing the expected number of mutations in a given gene under neutrality along all the possible changes in that same genomic region, with the assumption that only the mutation probability of each trinucleotide and the sequencing depth are responsible for the within-gene differences of mutation probability. We computed this value for each possible mutation in the positively selected genes including the TERT promoter, and for the protein-coding genes we also obtained a value per residue. A more complete explanation on this metric (and the detection of positive selection in sub-genic structures such as exons and domains) can be found in Supplementary Note 6. Structural models for all proteins used to run Oncodrive3D were obtained from the AlphaFold database (AlphaFold 2 v.4)78,79. Three-dimensional protein structures and protein structural features represented across figures were also obtained from the AlphaFold database. Solvent accessibility and secondary structure information were extracted from the AlphaFold-predicted PDB structures using PDB_Tool (https://github.com/realbigws/PDB_Tool). Structural visualizations of proteins were produced using UCSF ChimeraX80. In these comparisons, the site selection of individual mutations was used. We obtained the mutations identified through whole-genome sequencing in clonal or quasi-clonal samples obtained from the normal bladder of donors by laser capture microdissection in a previous study10. We used these samples to construct the mutational profile of normal urothelium, which we compared with that reconstructed in the present study through ultradeep sequencing (Extended Data Fig. From the same previous study, we obtained the mutations identified through whole-exome sequencing. Mutations obtained from samples sequenced at depth lower than 80× were discarded. From these data, we calculated the number of mutations per megabase observed in each sample, and averaged this value across donors. A linear regression was then constructed between these values and the age of donors, and a trend line calculated. The results of this comparison are presented in Extended Data Fig. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw sequencing data for this study were deposited in dbGaP under accession number phs004105.v1.p1. The set of mutations used in all analyses presented in the paper is available at Zenodo (https://doi.org/10.5281/zenodo.15836679)81. Reference mutational signatures were obtained from https://cancer.sanger.ac.uk/signatures/sbs/. Tumour mutations were obtained through cBioPortal (datasets from refs. 16,17,74 at https://www.cbioportal.org/), intogen (intogen.org) and the GENIE synapse data portal (https://genie.synapse.org/) as described in Methods. Protein structural models for the entire human proteome were obtained from the AlphaFold database (https://alphafold.ebi.ac.uk/). The results of two experimental saturation mutagenesis studies on TP53 and the TERT promoter were obtained from refs. The code of a pipeline containing all analyses described in this manuscript (deepCSA) is available at (https://github.com/bbglab/deepCSA). Code needed to reproduce paper figures using the data in the Zenodo repository mentioned above is available at (https://github.com/bbglab/normal_bladder_paper). Biological differences underlying sex and gender disparities in bladder cancer: current synopsis and future directions. Martincorena, I. Somatic mutation and clonal expansions in human tissues. Kakiuchi, N. & Ogawa, S. Clonal expansion in non-cancer tissues. & Campbell, P. J. Somatic mutation in cancer and normal cells. Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. Martincorena, I. et al. Somatic mutant clones colonize the human esophagus with age. The landscape of somatic mutation in normal colorectal epithelial cells. Lawson, A. R. J. et al. Extensive heterogeneity in somatic mutation and selection in the human bladder. Yoshida, K. et al. Tobacco smoking and somatic mutations in human bronchial epithelium. Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Kennedy, S. R. et al. Detecting ultralow-frequency mutations by duplex sequencing. Detection of ultra-rare mutations by next-generation sequencing. Li, R. et al. Macroscopic somatic clonal expansion in morphologically normal human urothelium. Cancer Genome Atlas Research Network. Comprehensive molecular characterization of urothelial bladder carcinoma. Robertson, A. G. et al. Comprehensive molecular characterization of muscle-invasive bladder cancer. Frequency of TERT promoter mutations in human cancers. Campbell, P. J. et al. Pan-cancer analysis of whole genomes. Priestley, P. et al. Pan-cancer whole-genome analyses of metastatic solid tumours. Martínez-Jiménez, F. et al. Pan-cancer whole-genome comparison of primary and metastatic solid tumours. Analyses of non-coding somatic drivers in 2,658 cancer whole genomes. The whole-genome panorama of cancer drivers. Martínez-Jiménez, F. et al. A compendium of mutational cancer driver genes. Bergstrom, E. N. et al. SigProfilerMatrixGenerator: a tool for visualizing and exploring patterns of small mutational events. Islam, S. M. A. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. & Rozen, S. G. mSigHdp: hierarchical Dirichlet process mixture modeling for mutational signature discovery. Nik-Zainal, S. et al. Mutational processes molding the genomes of 21 breast cancers. Alexandrov, L. B. et al. Clock-like mutational processes in human somatic cells. Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. Mularoni, L., Sabarinathan, R., Deu-Pons, J., Gonzalez-Perez, A. & López-Bigas, N. OncodriveFML: a general framework to identify coding and non-coding regions with cancer driver mutations. Pellegrini, S. et al. Oncodrive3D: fast and accurate detection of structural clusters of somatic mutations under positive selection. Jeffares, D. C., Tomiczek, B., Sojo, V. & dos Reis, M. A beginners guide to estimating the non-synonymous to synonymous rate ratio of all protein-coding genes in a genome. Borah, S. et al. TERT promoter mutations and telomerase reactivation in urothelial cancer. Dyrskjøt, L. et al. Bladder cancer. Epidemiology of bladder cancer: a systematic review and contemporary update of risk factors in 2018. Yokoyama, A. et al. Age-related remodelling of oesophageal epithelia by mutated cancer drivers. The mutational landscape of normal human endometrial epithelium. Hurst, C. D. et al. Genomic subtypes of non-invasive bladder cancer with distinct metabolic profile and female gender bias in KDM6A mutation frequency. Kato, S. et al. Understanding the function–structure and function–mutation relationships of p53 tumor suppressor protein by high-resolution missense mutation analysis. Giacomelli, A. O. et al. Mutational processes shape the landscape of TP53 mutations in human cancer. A saturation mutagenesis approach to understanding PTEN lipid phosphatase activity and genotype-phenotype relationships. Bandaru, P. et al. Deconstruction of the Ras switching cycle through saturation mutagenesis. Lue, N. Z. et al. Base editor scanning charts the DNMT3A activity landscape. Muiños, F., Martínez-Jiménez, F., Pich, O., Gonzalez-Perez, A. & Lopez-Bigas, N. In silico saturation mutagenesis of cancer genes. Identification of clonal hematopoiesis driver mutations through in silico saturation mutagenesis. Leroy, B., Anderson, M. & Soussi, T. TP53 mutations in human cancer: database reassessment and prospects for the next decade. Analysis and visualization of longitudinal genomic and clinical data from the AACR project GENIE Biopharma Collaborative in cBioPortal. AACR Project GENIE Consortium AACR project GENIE: powering precision medicine through an International Consortium. Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations. Salk, J. J. et al. Ultra-sensitive TP53 sequencing for cancer detection reveals progressive clonal selection in normal tissue over a century of human lifespan. Matas, J. et al. Colorectal cancer is associated with the presence of cancer driver mutations in normal colon. Rios-Doria, E. et al. TP53 somatic evolution in the normal endometrium of Black and White individuals. Saturation mutagenesis of twenty disease-associated regulatory elements at single base-pair resolution. Hill, W., Weeden, C. E. & Swanton, C. Tumor promoters and opportunities for molecular cancer prevention. Lung adenocarcinoma promotion by air pollutants. Balmain, A. Peto's paradox revisited: black box vs mechanistic approaches to understanding the roles of mutations and promoting factors in cancer. Diagnostic and prognostic genomic aberrations in upper tract urothelial carcinoma can be identified in focal barbotage samples. The coming decade in precision oncology: six riddles. Ju, Y. S. The mutational signatures and molecular alterations of bladder cancer. Dillon, L. W. et al. Quantification of measurable residual disease using duplex sequencing in adults with acute myeloid leukemia. Woolston, D. W. et al. Ultra-deep mutational landscape in chronic lymphocytic leukemia uncovers dynamics of resistance to targeted therapies. Characterization of clonal dynamics using duplex sequencing in donor-recipient pairs decades after hematopoietic cell transplantation. Di Tommaso, P. et al. Nextflow enables reproducible computational workflows. The nf-core framework for community-curated bioinformatics pipelines. Osorio, F. G. et al. Somatic mutations reveal lineage relationships and age-related mutagenesis in human hematopoiesis. Diverse mutational landscapes in human lymphocytes. Grinstead, C. M & Snell, J. L. Introduction to Probability: Second Revised Edition (American Mathematical Society, 1997). The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Gao, J. et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Van Allen, E. M. et al. Somatic ERCC2 mutations correlate with cisplatin sensitivity in muscle-invasive urothelial carcinoma. Pietzak, E. J. et al. Genomic differences between ‘primary' and ‘secondary' muscle-invasive bladder cancer as a basis for disparate outcomes to cisplatin-based neoadjuvant chemotherapy. Guo, G. et al. Whole-genome and whole-exome sequencing of bladder cancer identifies frequent alterations in genes involved in sister chromatid cohesion and segregation. Morales, J. et al. A joint NCBI and EMBL–EBI transcript set for clinical genomics and research. The Ensembl variant effect predictor. Highly accurate protein structure prediction with AlphaFold. Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Sex and smoking bias in the selection of somatic mutations in human bladder - Figures data. Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. We thank the University of Washington autopsy service staff including B. McGing and J. Grillo, who were instrumental in sample collection, and N. Jackson and K. Scherpelz for facilitating and supporting autopsy collection. We would also like to thank the nf-core community for developing the nf-core infrastructure and resources for Nextflow pipelines. Institute for Research in Biomedicine Barcelona is a recipient of a Severo Ochoa Centre of Excellence Award from the Spanish Ministry of Economy and Competitiveness (MINECO; Government of Spain) and an Excellence Institutional grant by the Asociacion Española contra el Cancer, and is supported by CERCA (Generalitat de Catalunya). acknowledges support from a fellowship from the Ministerio de Universidades para la Formación de Profesorado Universitario (FPU21/05649). is supported by an EMBO long-term fellowship (ALTF 1058-2022). was supported by a Postdoctoral AECC 2023 fellowship from Fundación Científica Asociación Española Contra el Cáncer (POSTD234814RAMI). This work was delivered as part of the PROMINENT team supported by the Cancer Grand Challenges partnership funded by Cancer Research UK (CGCATF-2021/100008), the NCI (OT2CA278668) and the Scientific Foundation of the Spanish Association Against Cancer, AECC. This research has been conducted using the UK Biobank Resource under application number 69794. This research was funded with two Research Grants (2020, 2023) from the Department of Laboratory Medicine and Pathology, University of Washington and support from NCI R01 CA259384 (R.A.R.). 10 were created using BioRender (https://BioRender.com/fgnnet9). These authors contributed equally: Ferriol Calvet, Raquel Blanco Martinez-Illescas These authors jointly supervised this work: Abel Gonzalez-Perez, Nuria Lopez-Bigas, Rosa Ana Risques Ferriol Calvet, Raquel Blanco Martinez-Illescas, Ferran Muiños, Maria Andrianova, Stefano Pellegrini, Axel Rosendahl Huber, Joan Enric Ramis-Zaldivar, Miguel L. Grau, Abel Gonzalez-Perez & Nuria Lopez-Bigas Centro de Investigación Biomédica en Red en Cáncer (CIBERONC), Instituto de Salud Carlos III, Madrid, Spain Ferriol Calvet, Raquel Blanco Martinez-Illescas, Ferran Muiños, Stefano Pellegrini, Joan Enric Ramis-Zaldivar, Abel Gonzalez-Perez & Nuria Lopez-Bigas Department of Medicine and Life Sciences, Universitat Pompeu Fabra, Barcelona, Spain Ferriol Calvet, Raquel Blanco Martinez-Illescas, Stefano Pellegrini, Abel Gonzalez-Perez & Nuria Lopez-Bigas Maria Tretiakova, Elena S. Latorre-Esteves, Jeanne Fredrickson, Shuyi Charlotte An, Elana Thieme, Brendan F. Kohrn & Rosa Ana Risques Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar participated in coding specific parts of the DNA duplex analysis pipeline, designed and carried out data analysis (including duplex mutation calling, computation of mutation density, estimation of positive selection, study of variability between samples and individuals, and association of these metrics with bladder cancer risk factors) and participated in data interpretation, drafting and revising the manuscript. participated in coding specific parts of the DNA duplex analysis pipeline, designed and carried out specific data analyses (including estimation of positive selection, association of these metrics with bladder cancer risk factors and saturation mutagenesis) and participated in data interpretation and in revising the manuscript and drafting supplementary information. designed, performed and supervised sample collection, retrieved clinical information, reviewed pathological findings, participated in project design and obtained funding. E.S.L.-E. performed optimization of enzymatic fragmentation, sample processing, and patient and sample data curation. performed the mutational signature analysis and participated in data interpretation and in revising the manuscript and drafting supplementary information. participated in the analysis of positive selection signals, prepared plots (including saturation mutagenesis plots), and participated in data interpretation and in revising the manuscript and drafting supplementary information. carried out the analysis of the expected fraction of mutated urothelium and participated in data interpretation and in revising the manuscript and drafting supplementary information. did the epidemiological analyses with UK BioBank data and participated in data interpretation and in revising the manuscript and drafting supplementary information. contributed to sample processing and data analysis. All authors have read and approved the manuscript. contributed equally to the manuscript and author order was decided randomly. Correspondence to Nuria Lopez-Bigas or Rosa Ana Risques. is an equity holder at TwinStrand Biosciences Inc. and NanoString Technologies Inc. R.A.R. is named inventor on patent no. 11,479,807 (Methods for targeted nucleic acid sequence enrichment with applications to error corrected nucleic acid sequencing) owned by the University of Washington and licensed to TwinStrand Biosciences Inc. R.A.R. was a consultant at TwinStrand Biosciences Inc. and received research funding from a joint research grant with TwinStrand Biosciences Inc. and Ovartec GmbH. is an equity holder at NanoString Technologies Inc. The other authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Depth of DNA duplex sequencing across the 79 samples. b) Distribution of DNA duplex sequencing depth across the 16 genes. White color represents the exon is not covered. Box plots in a and b display the quartiles with whiskers extending to the highest and lowest data points within 1.5 times the interquartile range. Details can be found in Supplementary Note 3. a) Left panel, orange bars, mutations per sequenced nucleotide detected by the DNA duplex sequencing technology in this study using the same panel of bladder genes in cord blood DNA samples from three donors. Blue bars (for comparison), mutations per sequenced nucleotide detected by a similar technology (NanoSeq; data taken from ref. 82) in two cord blood samples (Supplementary Note 4). Each bar presents the mutation density computed for a separate cord blood sample, with vertical lines representing the Poisson 95% confidence intervals. The comparison of the number of observed mutations per sequenced nucleotide with those expected in cord blood DNA based on prior studies (dashed red line, Supplementary Note 4) yields an estimate of the error rate of the technology of ~4 × 10−8 per sequenced nucleotide. Right panel, comparison of the estimated error rate by both technologies with the mutations per sequence nucleotide across the 79 normal urothelial samples included in this study (rightmost red bar). (N = 3 for cord blood DNA duplex sequencing). b) Mutational profile of normal urothelium obtained through two orthogonal approaches. Top panel, profile constructed using mutations detected through laser capture microdissection (LCM) of clonal or quasi-clonal samples followed by regular shallow whole-genome sequencing (data taken from ref. Bottom panel, profile constructed using mutations detected in this study from ~2 cm2 brushes followed by ultradeep DNA duplex sequencing. c) Relationship between the mutation density calculated in normal bladder urothelium using two orthogonal approaches. The red dots correspond to the rate of mutations detected through laser capture microdissection of clonal or quasi-clonal samples followed by regular shallow whole-exome sequencing (WES LCMs; data taken from ref. The blue dots correspond to the rate of mutations computed for the 79 samples in this study from ~2 cm2 brushes followed by ultradeep DNA duplex sequencing. For more details on the error rate of the technology, see Supplementary Note 4. a) Mutational profile of the signatures identified using SigProfiler. b) Mutational profile of the signatures identified using HDP. For more details on the identification of these mutational signatures and the decipherment of their etiology, see Supplementary Note 5. a) Five signals of positive selection on the mutations observed in TP53 in all samples. The first three rows show the distribution of truncating (nonsense and splice-site affecting), synonymous, and missense mutation across the coding region of TP53. On the right side, dN/dS truncating (top row) and dN/dS missense (third row) represent the estimation of the excess of truncating and missense mutations, respectively, over the neutral expectation calculated from the observed synonymous variants (second row). Fourth row, 3D clustering score of missense mutations (blue line) compared to the neutral expectation (gray line), along with detected 3D clusters of missense mutations (filled light blue line). Fifth row, functional impact score of SNVs (synonymous, missense and truncating) observed in the protein. The right-hand panel represents the distance between mean expected functional impact scores (gray areas) and the observed average functional impact score (vertical dashed lines), used to compute an empirical p-value (Supplementary Note 6). Sixth row, deviation in the ratio of frameshift (purple) to inframe (brown) indels in the gene compared to the ratio of non-3n to 3n (a length multiple of three nucleotides) indels in neighboring non-coding regions (excess of frameshift indels). The right-hand bars represent the numbers of coding frameshift and inframe indels and non-coding non-3n to 3n indels. Analytical or empirical tests used to calculate the p-values shown in the different panels are described in Supplementary Note 6. P-values for all genes computed using all methods appear in Supplementary Table 6. b) Magnitude of all signals of positive selection calculated for 14 genes on the pooled mutations of the 79 samples. Dashed lines for truncating, missense and indels indicate an equal number of observed and expected mutations of each type (dN/dS =1, that is, no selection). c) Comparison of the excess of truncating mutations (dN/dS truncating, top) and the excess of missense mutations (dN/dS missense) calculated taking into account every observed mutation only once (as used in the manuscript, and represented by the unshaded bars in each plot) and taking into account the number of DNA duplex reads supporting each mutation (bars shaded with diagonal line pattern). d) Positive selection on mutations in PIK3CA. Left panel, needleplot representing the distribution of missense, truncating and synonymous mutations in the region of PIK3CA covered by sequencing reads. Right panel, magnitude of positive selection on missense mutations across the 79 samples calculated using the Omega dN/dS approach. The p-value appears in Supplementary Table 6. All legends as defined in Fig. The p-values are calculated using the Omega implementation of the dN/dS approach described in Supplementary Note 6, and appear in Supplementary Table 6. b) Landscape of the fraction of urothelium covered by driver mutations of each gene in the panel. The values of covered urothelium have been discretized. In most cases, the percentage of urothelium covered by driver mutations of each gene falls in the lowest categories. The right-hand graph shows stacked barplots with the distribution of samples in different categories of covered urothelium across genes. c) Agreement of the magnitude of dN/dS missense (top) and dN/dS truncating (bottom) calculated for all genes in the dome and trigone of donors 14, 04 and 23. R-squared (R²), Pearson's correlation coefficient of the dN/dS values calculated for the dome and trigone samples from each donor. a) Top, number of SNVs that are shared between the dome and trigone samples (or unique to each of them) of donors for which both areas were brushed. Bottom, percentage of SNVs found in the trigone sample of each individual that are shared with the dome sample of the same individual (top), of SNVs found in the dome that are shared with the trigone (middle), and Jaccard index measuring the overlap of the SNVs identified within both samples (bottom). b) Distribution of Jaccard Index values of SNVs (first at the left), missense mutations (second), truncating mutations (third), and non-protein affecting mutations (last to the right) shared between the dome and trigone samples of the same individual and pairs of samples from different donors. The Jaccard index obtained for any subset of mutations is significantly higher for the dome and trigone samples of the same donor (p-values from one-tailed Wilcoxon-Mann–Whitney test). N indicates the number of sample pairs. c) Comparison of the distribution of Pearson's correlation coefficients comparing dN/dS values between dome and trigone samples of the same donor (as done in Extended Data Fig. In the first boxplot, all mutations are included in the calculation of dN/dS values, while in the second, mutations shared between dome and trigone of the same donor are excluded. The correlation is significantly higher between dome-trigone pairs of samples of the same donor than of different donors (p-values from one-tailed Wilcoxon-Mann–Whitney test). Only pairs of samples for which Omega values of at least two genes could be computed are included in the boxplots. N indicates the number of sample pairs. Box plots in b and c display the quartiles with whiskers extending to the highest and lowest data points within 1.5 times the interquartile range. a) Measurement of the tolerance of RBM10 dN/dS truncating to artifactual mutations (artifacts) following the BotSeq mutational profile82. The boxplots represent the distribution of dN/dS values calculated from 100 synthetic samples with increasing rates of injected artifacts between 0 and 1 × 10−7 (one order of magnitude higher than estimated for the technology), and for increasing values of ground truth dN/dS (between 1 and 50). b) Average percentage of RBM10 dN/dS truncating reconstructed value across 100 synthetic samples, calculated by computing which fraction of the ground truth dN/dS in a sample is obtained upon calculation. Left panel, average percentage of reconstructed ground truth dN/dS across synthetic samples (for all genes and all ground truth dN/dS explored altogether) that is calculated upon injection of increasing rates (x-axis) of different types of artifacts (color legend). Center plot, average percentage of reconstructed ground truth dN/dS across synthetic samples (for all genes and all artifacts altogether) that is calculated upon injection of increasing rates (x-axis) for different values of ground truth dN/dS (color legend). Right panel, average percentage of reconstructed ground truth dN/dS across synthetic samples (for all ground truth dN/dS explored and all artifacts altogether) for different genes (color legend), that is calculated upon injection of increasing rates (x-axis) of artifacts. a) From simulated datasets reflecting the same distributional features and data dependencies found in the study cohort, we computed the statistical power as the proportion of times the variable of interest (sex) came out significant in the univariate linear mixed-effects regression against truncating dN/dS. In this analysis the female group was picked as the baseline group. We simulated data with different ground truth female baselines (expected truncating dN/dS among females) and between-group differences (effect size). For each baseline-effect combination we can draw a power value, which are represented collectively in the form of these power profiles (see Supplementary Note 10). b) Table presenting a summary of the five associations found between dN/dS and sex in the study. Here we briefly define the meaning of each column. See also Supplementary Note 10 for a more in-depth account on the methodology. CSQN: Either missense or truncating, represents the specific dN/dS used as response variable in the association analysis. ESTIMATE: Coefficient of the binary variable of interest (“is_male”) inferred via linear-mixed effects regression against dN/dS using the donor as a random intercept. CI_LOW, CI_HIGH: Lower and upper 95% CI bounds of ESTIMATE. INTERCEPT and BASELINE are expected to follow closely one another. COVARIATE: The (binary) explanatory variable representing sex. EFFECT_PVAL: The “effect p-value” is an ad-hoc metric that we defined as the proportion of times the sex coefficient attains a value at least as high as ESTIMATE upon regression with a dataset corresponding to BASELINE and zero ground-truth effect. It can be thought of as an effect-aware false positive rate. c) Frequency of tumor samples with missense or truncating mutations of 6 genes in males and females across a cohort of 2,965 bladder carcinomas from the GENIE cohort. d) Multivariate logistic regression (including age) of sex on mutations in the 6 genes. Circles with dark outer circumference denote significant associations (FDR threshold of 0.2). e) Distribution of expected number of mutations in the two TERT promoter mutational hotspots (chr5:1295113 and chr5:1295135) across donors younger than 55 years old or never smokers assuming a mutation rate equal to that observed across ever smokers older than 55 years old. The red dashed vertical line represents the actual observed number of mutations in the two hotspots across donors younger than 55 years or never smokers. The p-value was calculated empirically based on 10,000 randomizations, as described in Supplementary Note 10. f) Maximum variant allele frequency detected for activating TERT promoter mutations in a sample vs the number of activating TERT promoter mutations (i.e. mutations observed in tumors, see main text) identified in a sample. The observation of different activating TERT promoter mutations in the same sample indicates the existence of convergent evolution of TERT promoter mutations. This, in turn, suggests that the observation of mutations with large variant allele frequency may also represent multiple mutated clones with the exact mutations (convergent evolution) rather than very large clones. a) Comparison of the density of protein affecting mutations in 14 genes across two cohorts of bladder tumors (muscle invasive and non-muscle invasive) and in the normal urothelium of the 45 donors. Mutation density in tumors is calculated by dividing the number of observed mutations (normally 1) by the gene length in megabases (Mb). b) Percentage of amino acid residues in each gene with zero, one, two, or three or more mutations observed across 892 bladder tumor samples from the intOGen cohort. c) Theoretical and observed curves of saturation mutagenesis for genes not shown in Fig. The grey dashed line represents the kinetic of saturation mutagenesis under the theoretical assumption of no selection, in which mutations are observed based only on their neutral probability of occurrence. The red dashed line is constructed through successive depth down-samples of the current observation and represents the observed kinetic of natural saturation mutagenesis (see details in Supplementary Note 12). d) Natural saturation mutagenesis of EP300 in normal bladder urothelium. Besides the tracks described in Fig. 5c for TP53, 3D clusters obtained via Oncodrive3D (second), dN/dS truncating and dN/dS missense values for each exon (fourth), and the distribution of tumor mutations (from intOGen; see Methods) along the sequence of the gene (last) have been added. These same types of plots are presented for the rest of genes in the study in Supplementary Figs. e) dN/dS truncating and dN/dS missense values for each domain of EP300. Solid border represents significant dN/dS values (p-value < 0.05) according to Omega (Supplementary Note 6). a) Manhattan plot illustrating the strength of site selection for all genomic sites included in the sequencing panel. b) Application of site selection values to TP53 mutations. In all plots, TP53 tumor mutations observed in two large cohorts (intOGen, N = 33,218 and GENIE, N = 109,017) are grouped depending on whether they have been observed across bladder normal samples in this study and their site selection (i.e., not observed, observed with non-significant site selection and observed with significant site selection). Right top panel, distribution of the frequency across GENIE tumors. Left bottom panel, boostDM (machine learning models for in silico saturation mutagenesis)46 scores of the three groups of mutations. Right bottom panel, proportion of mutations annotated or not annotated as oncogenic in ClinVar83 or OncoKB84. c,d) Distribution of the solvent accessibility of sites with mutations in each group for TP53 (c) and EP300 (d). e) Application of site selection values to TERT promoter mutations. Mutations observed in two large cohorts of tumors (or the subset of bladder tumors in GENIE) are grouped depending on whether they have been observed across normal samples and their site selection into not observed, observed with non-significant site selection and observed with significant site selection. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Calvet, F., Blanco Martinez-Illescas, R., Muiños, F. et al. Sex and smoking bias in the selection of somatic mutations in human bladder. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09448-3'>Sperm sequencing reveals extensive positive selection in the male germline</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 15:11:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mutations that occur in the cell lineages of sperm or eggs can be transmitted to offspring. In humans, positive selection of driver mutations during spermatogenesis can increase the birth prevalence of certain developmental disorders1,2,3. Until recently, characterizing the extent of this selection in sperm has been limited by the error rates of sequencing technologies. Here we used the duplex sequencing method NanoSeq4 to sequence 81 bulk sperm samples from individuals aged 24–75 years. Our findings revealed a linear accumulation of 1.67 (95% confidence interval of 1.41–1.92) mutations per year per haploid genome driven by two mutational signatures associated with human ageing. Deep targeted and exome NanoSeq5 of sperm samples identified more than 35,000 germline coding mutations. We detected 40 genes (31 newly identified) under significant positive selection in the male germline that have activating or loss-of-function mechanisms and are involved in diverse cellular pathways. Most of the positively selected genes are associated with developmental or cancer predisposition disorders in children, whereas four of the genes exhibited increased frequencies of protein-truncating variants in healthy populations. We show that positive selection during spermatogenesis drives a 2–3-fold increased risk of known disease-causing mutations, which results in 3–5% of sperm from middle-aged to older individuals with a pathogenic mutation across the exome. These findings shed light on germline selection dynamics and highlight a broader increased disease risk for children born to fathers of advanced age than previously appreciated. All human cells accumulate mutations throughout life. In proliferating tissues, acquired driver mutations that confer a selective advantage can promote the expansion of individual clones in competing stem and progenitor cell populations. Although patterns of selection and clonal expansion have been extensively studied in cancers, recent research has also highlighted their occurrence in normal tissues during ageing6,7,8,9,10,11,12,13,14. Spermatogonial stem cells of the testis occupy a distinct niche relative to other studied normal tissues. Among replicating cells, these stem cells have the lowest mutation rate, which is around 5–20-fold lower than other studied somatic cell types12. They are also the only adult proliferating cells with the potential to transmit mutations to offspring, balancing self-renewal and spermatogenesis to produce 150–275 million sperm per day15,16. Targeted sequencing studies have revealed that driver mutations are acquired in spermatogonial stem cells and that these cell populations expand along seminiferous tubules, which results in increased fractions of mutant clones that are detectable in sperm1,17,18,19,20. Notably, all the male germline drivers identified so far are activating missense hotspot mutations, which contrasts with the broader range of activating and inactivating drivers observed in cancers and somatic tissues. These germline driver mutations can have substantial implications for offspring, as such mutations are found in a set of 13 genes all known to cause severe developmental disorders3. This effect leads to a significant increase, up to 1,000-fold, in the sporadic birth prevalence of these disorders, with a strong correlation between the age of the father and prevalence2. Technical limitations, related to the polyclonality and low mutation rate of testis and sperm, have prevented extensive characterization of this selection beyond a limited set of genes. Recent advances in error-corrected duplex DNA sequencing approaches use information from both DNA strands to detect mutations at single-molecule resolution21,22,23,24. These new methods have proven successful for the accurate estimation of mutation burden in sperm, with an error rate of <5 × 10−9 per base pair4. Here we combine the duplex approaches of whole-genome NanoSeq4 with deep whole-exome and targeted NanoSeq5 to characterize positive selection in the male germline and to quantify its consequences for the accumulation of disease mutations in sperm. We performed whole-genome NanoSeq4 of bulk semen samples (n = 81, 1–2 time points per donor, 57 donors with an age range of 24–75 years) and matched blood (n = 119, 1–3 time points, 63 donors with an age range of 22–83 years) from men in the TwinsUK cohort25 (including 8 monozygotic and 3 dizygotic twin pairs; Methods and Supplementary Table 1). The analysed semen samples had sperm counts >1 million per ml, as samples below this threshold showed evidence of somatic cell contamination (Extended Data Fig. The mean number of unique DNA molecules per site where a mutation was callable (duplex coverage (dx)) was 3.7 dx in sperm and 4.3 dx in blood (Extended Data Fig. We identified single nucleotide variants (SNVs) and small insertion–deletion mutations (indels) in whole-genome NanoSeq data from sperm and blood. Inherited germline variants were excluded using matched blood (Methods). From the 6,653 SNVs detected in sperm (Supplementary Table 2), we estimated an accumulation of 1.67 substitutions per year per haploid genome (95% confidence interval (CI) = 1.41–1.92, linear mixed-effect regression). This result is comparable with estimates from paternal de novo mutations (DNMs) in family pedigrees26 of 1.44 substitutions per year (95% CI = 1.00–1.87) and seminiferous tubules of the testes12 of 1.40 substitutions per year (95% CI = 1.02–1.76; Fig. Indels accumulated in sperm at a rate of 0.10 per year per haploid genome (95% CI = 0.06–0.15), again similar to the rate observed in DNMs26 of 0.08 haploid indels per year (95% CI = −0.02 to 0.17) and testes12 of 0.08 haploid indels per year (95% CI = 0.02–0.13; Fig. a,b, Substitutions (a) and indels (b) per haploid cell from whole-genome NanoSeq of sperm, trio paternal DNMs26 called with standard sequencing and clonal variants from seminiferous tubules of testis12 called with standard sequencing. Dots indicate single donors, except for testis with 1–15 samples per donor. c,d, Substitutions (c) and indels (d) per diploid cell for different ages from blood NanoSeq samples. e, Ratio of blood to sperm substitutions and indels per diploid cell per year. Each dot corresponds to an individual with both a blood and sperm sample. For individuals who had multiple time points, the mean value of all time points in that tissue was used. f, Trinucleotide mutation counts in all sperm and blood samples. g, Contributions of the signatures SBS1, SBS5 and SBS19 in sperm and blood samples ordered by age. For a–d, models are linear mixed regressions, with the central line showing the model fit and the shaded bands indicating 95% CIs calculated using parametric bootstrapping. From 92,035 SNVs and 4,641 indels detected in whole blood (Supplementary Table 3), we estimated an accumulation of 19.9 substitutions per year per diploid genome (95% CI = 17.3–22.5; Fig. 1c) and 0.9 indels per year (95% CI = 0.7–1.1; Fig. Both estimates are within the range of mutation rates observed for specific cell types in blood13 and are consistent with measuring a weighted average of these cell types in whole blood (Extended Data Fig. Individuals had a mean of 7.6-fold more substitutions per base pair per year (range of 4.2–11.5; Fig. Accounting for twin status or multiple time points from the same individuals had a significant predictive effect for mutation burden in blood but not in sperm (Supplementary Note 2). The SNV mutational signatures in sperm were inferred to be SBS1 (mean 16%) and SBS5 (mean 84%), which were the expected clock-like ageing signatures27 (Fig. In blood, SBS1 (mean 15%) and SBS5 (mean 75%) were also the main mutational signatures, with an additional contribution of SBS19 (mean 10%), which has been linked to persistent DNA lesions in haematopoietic stem cells27 (Fig. We observed that all signatures were correlated with age (Extended Data Fig. SBS1 and SBS5 accumulated in individuals at a mean of 8.9-fold (range of 2.3–39.1) and 6.8-fold (range of 3.7–10.9) higher rate in blood than in sperm, respectively (Extended Data Fig. This finding indicates that SBS19 does not explain a substantial fraction of the mutation burden gap between the two tissues. We used a capture-based modification to NanoSeq5 to deeply sequence coding regions from the same set of semen samples. Specifically, we sequenced 38 samples using whole-exome NanoSeq to a mean depth of 551 dx per sample (20,923 cumulative dx). We also sequenced 81 samples using targeted NanoSeq to a mean depth of 985 dx per sample (79,811 cumulative dx) with a target panel that consisted of 263 canonical cancer driver genes, 107 of which are also associated with developmental disorders (Methods, Extended Data Fig. We detected 56,503 (58% in coding regions) SNV and indel mutations from the exome panel and 5,059 (58% in coding regions) from the targeted cancer panel (Methods and Supplementary Table 5). The mutation burdens for exome and targeted samples were consistent with whole-genome NanoSeq after correcting for the relative trinucleotide composition of sequencing coverage (Extended Data Fig. The majority of variants (99.5%) were detected only in a single duplex molecule of a sample. Similarly, in the 23 samples with 2 time points (mean of a 12.1-year gap), 99.3% of the 5,143 variant calls from the first time point were not called in the second time point. These results are consistent with sperm being a highly polyclonal collection of cells derived from a large population of spermatogonial stem cell progenitors in the testis. The exome-wide strength of positive selection in sperm was quantified by estimating the rate of nonsynonymous (N) relative to neutral synonymous (S) mutations (dN/dS ratio, where dN/dS = 1.0 indicates neutrality). We used the dNdScv algorithm28 with modifications to account for duplex sequencing coverage per base, CpG methylation levels in testis samples and pentanucleotide context. These modifications refined exome-wide dN/dS ratios by resolving specific mutation rate biases but had minor effects on gene-level dN/dS ratios (Extended Data Fig. Using this model, the dN/dS ratio in the exome-sequenced samples were estimated to be 1.07 (95% CI = 1.04–1.10). This ratio implies that 6.5% (95% CI = 3.8%–9.1%) of the observed nonsynonymous substitutions in sperm conferred a clonal advantage during spermatogenesis in this cohort. Splitting the cohort into thirds by age, the exome-wide dN/dS ratio was 1.01 (95% CI = 0.93–1.09) in 26–42 year olds, 1.03 (95% CI = 0.97–1.10) in 43–58 year olds and 1.09 (95% CI = 1.06–1.13) in 59–74 year olds (Fig. A linear regression of dN/dS ratio against age group showed a positive trend, although significance was not reached (P = 0.18). If so, then the cohort-wide dN/dS ratios presented here partially reflect the age distribution of samples (age range of 26–74 years, mean of 53 years). a–c, Genes with significant dN/dS ratios from exome-wide and restricted hypothesis tests. b, Enrichment over expectation of mutation classes. d,e, Observed sperm mutations across the cohort for CUL3 and SMAD4. The height of each lollipop represents the number of biologically independent samples with a mutation at that position, and the colour indicates mutation type. Mutations are labelled with their amino acid consequence or, for insertions (ins) and deletions (del), as in-frame (IF) or frameshift (FS). A ‘P' indicates pathogenic/likely pathogenic classification in ClinVar. Lines below the gene indicate somatic mutations in cancer from COSMIC. f,g, dN/dS ratios for sperm SNVs across sets of individuals or genes, where the dotted black line indicates neutrality and the dotted orange line represents the cohort average across all genes. g, Expression levels as log2 of unique molecular identifier (UMI) counts and cell-type clusters from single-cell sequencing of germ cells29. Germ-cell types include undifferentiated and differentiated spermatogonial stem cells (SSCs), spermatocytes, round spermatids (spermatid 1) and elongating spermatids (spermatid 2). Data are from mutations detected in n = 38 biologically independent sperm samples. h, Observed/expected mutation rates in sperm for variant recurrence bins (data from COSMIC and DDD databases) using mutations from n = 38 biologically independent sperm samples. Data in f–h show ratio point estimates, error bars indicate 95% CIs. We next compared the dN/dS ratios across gene sets related to spermatogenesis expression29 (Fig. Genes with the highest dN/dS ratios were those that were highly expressed during spermatogenesis (1.25, 95% CI = 1.13–1.38) and most specific to differentiated spermatogonial stem cells (1.11, 95% CI = 1.05–1.17). By contrast, the genes that were not expressed in spermatogenesis (0.98, 95% CI = 0.88–1.11) and the genes most specific to elongating spermatids (1.01, 95% CI = 0.94–1.08) showed dN/dS ratios close to neutrality. These results are consistent with the understanding that excess nonsynonymous mutations observed in sperm confer a competitive advantage earlier in their cell lineage, specifically in the spermatogonial stem cells of the testis18. We then combined exome and targeted panel datasets to investigate which genes were driving the signal of positive selection (Methods). We applied dN/dS tests for excess nonsynonymous mutations at gene-wide and SNV hotspot levels, which together identified 40 genes under significant positive selection. Of these, 35 genes reached exome-wide significance at the gene level (false discovery rate q < 0.1; Supplementary Table 6) and/or contained 1 of 17 exome-wide significant hotspots (q < 0.1; Extended Data Table 1). The genes KDM5B, NF1, SMAD6, CUL3, RASA2, PRRC2A, PTEN, ROBO1, DDX3X, CSNK2B, KRAS, PPM1D, ARID1A, BRAF, HRAS, KMT2E, SCAF4, BMPR2, TCF12, CCAR2, DHX9, NSD1, LZTR1, ARHGAP35, CBL, SSX1 and RBM12 were significant in only the gene test. We excluded SEMG1 despite it reaching gene-level significance, as its extreme expression in seminal vesicles30, lack of expression in spermatogenesis and indel-specific enrichment indicate indel hypermutation31,32 from minor seminal vesicle DNA contamination rather than germline selection. Subsequently, we carried out restricted hypothesis dN/dS tests at the per-gene and per-site level. The gene-level test examined only the set of 263 canonical cancer driver genes on our target panel. The site-level test used a set of 1,963 sites consisting of known cancer hotspots and recurrent DNM sites from the Deciphering Developmental Disorders (DDD) cohort33. This analysis identified five additional genes—KDM5C, KMT2D, AR, CTNNB1 and RAF1—and seven hotspots not already significant at the exome-wide level (q < 0.1; Fig. The genes previously implicated in germline positive selection all operate through activating missense mutations, with 12 linked to the RAS–MAPK signalling pathway3 and one (SMAD4) linked to TGFβ–BMP signalling34. Our findings replicated SMAD4 and 8 out of the 12 RAS–MAPK pathway genes as under significant positive selection in this dataset. The four genes that did not reach significance (MAP2K1, MAP2K2, SOS1 and RET) each had between twofold and fourfold enrichment of missense mutations, which corresponded to nominally significant missense enrichment in all four genes (P < 0.1). Given the direct evidence for these genes driving clonal selection in testis and nominal enrichment from sperm sequencing, we anticipate that each will reach exome-wide significance with deeper sequencing. We estimate that together, the 44 genes linked to germline selection here or in previous studies contain 357 (95% CI = 319–387) excess nonsynonymous variants in exome sequenced sperm samples. This result would account for 23% (95% CI = 14%–43%) of the total estimated driver variants across the exome. The wide CIs and the sensitivity of this estimate to the mutation model used (Supplementary Note 3) suggest that small uncertainties in mutation rates, when propagated across the exome, make it difficult to precisely estimate the fraction of drivers explained. Nevertheless, the findings indicate that additional driver genes remain to be discovered. The 31 newly identified genes demonstrate that germline positive selection is not restricted to activating mutations or to the RAS–MAPK pathway. For instance, 30 out of the 31 genes are enriched for loss-of-function (LOF) mutations such as nonsense, splice and indel variants, which are indicative of protein-inactivating mechanisms of selection (Fig. Splitting the germline-selection genes and known cancer genes on the basis of ten canonical cancer pathways35, we found that the top gene groups enriched in dN/dS are RAS–MAPK, WNT and TGFβ–BMP signalling (Extended Data Fig. These findings highlight a new diversity of genes, mutational mechanisms and pathways that drive germline selection. It has been observed that cancers and germline developmental disorders share causal pathways and genes36,37. Notably, the 13 genes previously linked to germline positive selection are all known cancer and known developmental disorders genes3,34. This pattern holds, but to a lesser extent, in the new germline selection genes identified here: 16 out of 31 genes are tier 1 or 2 cancer census genes38 and 27 out of 31 are linked to monogenic disorders in the Development Disorder Genotype–Phenotype (DDG2P) or Online Mendelian Inheritance in Man (OMIM) databases39 (Fig. The overlap between germline positive selection and cancer or developmental disorders is also apparent at the variant level. Somatic mutations that are most frequently observed (>50 times) in the Catalogue Of Somatic Mutations In Cancer (COSMIC) database are enriched 11-fold (95% CI = 6–20) among our sperm mutation dataset after adjusting for the expected mutation rate (Methods). Similarly, germline mutations that are most frequently observed (>5 times) in a large cohort of children with developmental disorders are enriched 66-fold (95% CI = 41–100) in our sperm mutation dataset (Fig. Moreover, the mutation types enriched in sperm for a given gene largely matched those enriched in cancer and linked to developmental disorders (Extended Data Figs. These results show a clear overlap between genes, hotspots and mutation mechanisms that drive germline positive selection, cancer and developmental disorders. A notable exception to this pattern is SMAD4, which has two distinct missense hotspots in sperm that are developmental disorder hotspots causal for Myhre syndrome40 but are not often seen in cancers, a result that replicates recent findings34 (Fig. Given the association of many positively selected genes to disease, it is of interest to assess to what degree positive selection may increase the fraction of sperm with potential disease-causing mutations, and thus the birth prevalence of the associated disease. To estimate the fraction of sperm with specific classes of variants, we aggregated the variant allele frequencies (VAFs) of different mutation types and compared this to expected values from our dNdScv mutation model (Methods). The fraction of sperm carrying noncoding or synonymous mutations increased linearly with age, as predicted by the neutral model (Extended Data Fig. By contrast, missense, truncating and coding indel variants deviated above expected in older individuals, an observation consistent with the dN/dS results and indicative of age-related positive selection. To assess potential disease burden, we compiled a conservative list of probable monoallelic disease-causing mutations, including pathogenic or likely pathogenic variants from ClinVar41 and highly damaging variants (combined annotation dependent depletion (CADD)42 value of >30) in high-confidence monoallelic developmental disorder genes from the DDG2P cohort39. Across all ages, the observed frequency of disease mutations exceeded expectation from the germline mutation model. The expected fraction ranged from 0.73% in 30 year olds to 1.6% in 70 year olds. When fitting a quasibinomial regression, the observed values ranged from 2% (95% CI = 1.6%–2.5%) in 30 year olds to 4.5% (95% CI = 3.9%–5.2%) in 70 year olds, with a significant relationship between the observed fraction and age (P = 1.75 × 10–5; Fig. These differences represent similar enrichments of 2.8-fold (95% CI = 2.2 to 3.5) and 2.9-fold (95% CI = 2.5 to 3.3) in 30 year olds and 70 year olds, respectively. Driver mutations are split by those contributing to the disease mutations and the remainder, ‘ageing drivers'. b, Estimated percentage of sperm per individual with a driver mutation by age. c, Observed and expected percentages of sperm with a likely disease mutation by age. For b and c, the central line represents the model-predicted mean from quasibinomial regression, and shaded bands indicate 95% CIs. d, Cohort means from a split by gene and ordered by estimated mutation percentage. Per-gene contributions are shown above each gene; the summed contributions of all genes are shown below. Genes with four or fewer variants are grouped on the left with a condensed x axis for clarity. Notably, the disease cell-fraction estimates were made up of many low-frequency variants rather than being driven by individual high VAF mutations. Furthermore, 692 out of 696 (99.4%) of all those variants were only observed in a single sperm, similar to the average of all variants (99.5%). We next investigated to what degree the observed enrichment of disease mutations can be attributed to driver mutations in positively selected genes. Fitting a quasibinomial regression, we observed a strong positive correlation between age and driver rate (P = 7.95 × 10–6) with an estimated 0.5% (95% CI = 0.3%–0.8%) of sperm from individuals at age 30 years and 2.6% (95% CI = 2.0%–3.3%) of sperm from individuals aged 70 years with a known driver mutation (Fig. However, only about two-thirds (65.6%) of those driver mutations met our criteria of likely disease-causing. Mutations in sperm that are likely disease-causing and those classified as known driver mutations represented overlapping, yet distinct, annotations (Fig. Across the cohort, an estimated 3.3% of sperm carried a likely disease-causing mutation. Of this, approximately one-third (1.2%) was expected by the neutral mutation model, another third (1.1%) was explained by known driver mutations and the remaining third (1.0%) was unexplained by either source. These findings indicate that the increase in likely disease-causing mutations is largely driven by germline positive selection on known genes, but also indicate that additional driver genes with disease associations remain to be identified. After examination of the driver mutations that did not meet our criteria for likely disease-causing, we estimated that they affected 0.6% of sperm across the cohort. The inheritance consequences of these mutations are unclear. For instance, SMAD6 variants, which are linked to variably penetrant congenital phenotypes43, may be disease-causing in some cases but not others. Other possibilities include mutations that are unannotated disease-causing, impair fertilization, are embryonic lethal or act through biallelic disruption. A large fraction of sperm with disease and/or driver mutations could be attributed to a small number of genes. Of the 374 genes with at least one such variant, 33 with ≥5 independent mutations, most under significant positive selection (26 out of 33), accounted for 42.8% of the disease or selection burden (Fig. Notably, six of those genes, all under significant positive selection (KDM5B, MIB1, SMAD6, PRRC2A, NF1 and PTPN11) alone explained over 20% of the disease or selection fraction. This result suggests that although all individual driver mutations were observed at low allele frequencies, they were disproportionately concentrated in specific driver genes. We next examined whether risk factors beyond age contribute to the accumulation of disease or driver mutations in sperm. Known germline mutagens include chemotherapy, inherited DNA repair defects33,44 and weaker influences from genetic ancestry and smoking45. Although our cohort did not include individuals with chemotherapy exposure or DNA repair defects, phenotype data on body mass index (BMI), smoking and alcohol consumption, all known to affect mutation burden in some somatic tissues46, were available. Using multivariate generalized linear models (Methods and Extended Data Fig. 9), we tested associations between these factors and measures of mutation burden, signatures and driver cell fractions, correcting for multiple testing. Across sperm datasets (targeted, exome and whole genome) no significant effects were found for BMI, smoking pack-years or alcohol drink-years. Notably, in blood samples, smoking and alcohol consumption showed significant effects on SNV and SBS5 mutational signature burdens. These findings indicate that the male germline may be largely protected from these exposures, although further investigation in larger and more diverse cohorts is needed. Mutations in sperm account for about 80% of DNMs and are therefore also the primary origin of population-level variants. Comparisons of these germline mutation sources offer insight into how positive selection in spermatogonia manifests over time. Control DNMs from unaffected offspring47 showed a dN/dS ratio near neutrality (0.98, 95% CI = 0.90–1.08; Fig. 4a), whereas DNMs from children with developmental disorders showed a marked nonsynonymous enrichment (1.36, 95% CI = 1.33–1.39; Fig. However, ascertainment biases in these cohorts probably distort the observed ratios, and larger, unbiased birth cohorts will be required to resolve the baseline of DNMs entering the population. a, Exome-wide dN/dS ratio point estimates across different variant sets, including n = 38 biologically independent sperm samples, DNMs from n = 1,886 healthy trios47 and n = 22,742 trios from the DDD cohort33, and population variants from n = 125,748 individuals in gnomAD48, split by allele frequency (AF). b, Observed/expected enrichment of missense and LOF (essential splice, nonsense or indel) variants in positively selected genes in sperm (x axis) from dN/dS models versus gnomAD (v.2) LOF z scores. Positive z scores indicate LOF depletion, whereas negative scores indicate excess over expected. To assess how selection shapes germline variants in the population, we analysed dN/dS ratios across allele frequencies in gnomAD48 (Fig. This analysis revealed a decay in dN/dS ratios at higher allele frequencies, a result consistent with purifying selection across generations. Together, these results suggest that positive selection is the greater force that acts on germline mutations during spermatogenesis, whereas negative selection predominates over generations. This finding mirrors the well-established contrast between selection on cancer and population-level mutations28. Most genes in gnomAD have positive z scores, which indicates LOF depletion due to negative selection. Of the 31 selection genes with LOF enrichment in sperm (range of 3-fold to 50-fold), 27 showed LOF depletion in gnomAD, a result consistent with positive selection during spermatogenesis but purging in the population by negative selection. Notably, four genes, SMAD6, MIB1, LZTR1 and SSX1, had more LOFs than expected in gnomAD (Fig. The latter three are among the strongest LOF-enriched outliers in gnomAD (v.2) and are flagged by gnomAD for unexplained LOF enrichment. Our results suggest that their increased LOF frequency in gnomAD reflects increased input from germline positive selection, with insufficient negative selection to remove them from the population. We sequenced sperm and blood from healthy men spanning a wide age range to quantify mutation rates and positive selection in the male germline. Despite sharing SBS1 and SBS5 signatures, mutations accumulated around eightfold more slowly in sperm than blood, a result that supports our previous observations of the protected nature of the germline relative to the soma12. By analysing more than 35,000 coding mutations from sperm exome-wide, we corroborated key findings from previous studies of spermatogonial selection1,17,18,19,20 and significantly advanced our understanding of its scope and impact. Our results replicated 9 out of 13 known germline-selection genes and confirmed that the impact of selection increases with age, leading to an increased fraction of sperm with pathogenic variants. We also identified 31 new positively selected genes, thereby broadening the range of pathways and mechanisms implicated in this process. The discovery of diverse genes and the identification of LOF as a common selection mechanism highlight the power of NanoSeq to enable exome-wide searches and to detect mutations in single cells, overcoming the limitations of previous methods reliant on high-frequency gain-of-function missense mutations. These findings reveal that germline selection operates in the broader framework of cellular selection, driven by many of the same genes and mechanisms that shape clonal dynamics in somatic tissues. However, unlike somatic selection, germline selection affects offspring phenotypes and influences evolutionary trajectories. Perhaps unsurprisingly then, driver mutation burden in sperm is low compared with many somatic proliferative tissues. In middle-aged to older men, we showed that only 1–3% of sperm carry a known driver mutation. Low levels have also been reported in colon (about 1%) and liver (1–5%)8,9. In blood, many individuals fell within this range, although some showed large expansions of single clones14. Much higher burdens are seen in epithelial tissues such as endometrium, oesophagus and skin, where 30–50% of cells often have driver mutations6,7,10. These differences are probably shaped in part by both tissue architecture and mutation rate. This low driver rate also aligns with the rarity and biology of tumours that originate from spermatogonia, which are typically driven by aneuploidy rather than sequential driver mutations53. Our findings have significant implications for disease and evolutionary studies that rely on germline mutation models, as they do not currently account for positive selection. We showed that this selection can distort estimates of selective constraint based on population-level data. Germline selection will also affect the discovery of disease-causing genes from DNM enrichment tests. For example, excess LOF mutations in MIB1 found in developmental disorder trios33 probably reflects germline selection rather than disease association, as they are more common in population cohorts than expected and do not show phenotype correlation54. In principle, case–control tests for DNM enrichment would avoid this bias, but require large cohorts of age-matched controls for sufficient statistical power. Until such resources become available, DNM enrichment studies should consider evidence for germline selection influencing individual genes presented here (Supplementary Table 5). Unlike MIB1, most genes under positive selection during spermatogenesis are linked to severe monogenic disorders. We demonstrated that positive selection in spermatogenesis led to a substantial twofold to threefold enrichment in sperm with likely disease-causing mutations across the age range studied. As a result, we estimate that 3–5% of sperm from men aged over 50 years carry a likely disease-causing mutation, a value that exceeds previous estimates based on germline mutation models55. Although these risks are substantial, typical paternal age at conception is lower than in our cohort, which suggests that the impact on birth outcomes may be more modest. Moreover, the relationship between sperm mutations and birth prevalence remains uncertain. Many pathogenic variants in sperm may not result in live births owing to impaired fertilization, embryonic lethality or pregnancy loss (Supplementary Note 4). Nevertheless, growing awareness of these risks may prompt interest in reproductive planning, genetic counselling or clinical interventions. Translating these findings into specific clinical recommendations, however, presents challenges. Unlike inherited variants, which are present in every cell, or aneuploidies, for which clinical focus is limited to a few recurrent events, pathogenic mutations in sperm are both highly diverse and individually rare, which makes them difficult to target with standard screening approaches. Nonetheless, targeted risk assessment may be valuable in specific contexts, such as individuals with an increased risk of sperm hypermutation due to impaired DNA repair or chemotherapy exposure52,56. Our results focused on SNVs and indels, without capturing other variant classes. Although long-read sequencing studies have begun to explore these in sperm57 and trios58, further work is needed to capture the full range of human germline variation and its impact on offspring. This study provides important insights into the historically underexplored reproductive risks of ageing in men. Unlike the well-established effects of maternal ageing on oocyte quality and aneuploidy59, we are only beginning to understand the full scope of male-specific risk, including the clonal expansion of driver mutations. As trends towards delayed reproduction continue60, it is essential to recognize that the age of both parents contributes to risk of disorders in offspring. Future research will refine our understanding of selective pressures and disease risk associated with germline mutation, clarifying their implications for human reproduction and health. This study was carried out under TwinsUK BioBank ethics, approved by the North West–Liverpool Central Research Ethics Committee (REC reference 19/NW/0187), IRAS ID 258513 and earlier approvals granted to TwinsUK by the St Thomas' Hospital Research Ethics Committee, later the London–Westminster Research Ethics Committee (REC reference EC04/015). Semen samples were collected or obtained from archival samples with informed consent from 75 research participants in the TwinsUK cohort25. Archival whole-blood samples were also obtained from 67 of those men from the TwinsUK BioBank. A total of 104 semen samples spanned an age range of 24–75 years and included 29 men with 2 time points separated by a mean of 12.1 years (range of 12–13 years) and the remaining 46 men with a single time point. A total of 133 blood samples were collected at an age range of 22–83 years from men with 1–4 time points. In the cohort, there were a total of nine monozygotic twins and three dizygotic twin pairs. Counts of samples, time points and twin pairs that were successfully sequenced and passed analysis quality control thresholds are summarized in Supplementary Table 1. Self-reported age, height, weight, ethnicity, twin zygosity, smoking and alcohol consumption were obtained from questionnaires provided by TwinsUK and periodically taken. All individuals who provided ethnicity information indicated white. A smoking pack-years was defined as 365 packs of cigarettes and a total pack-year was calculated using the highest estimate across all questionnaires from cigarettes per day or week and total years smoked. Alcohol drink-years was calculated using average weekly alcohol consumption extrapolated to the duration of adult life before sampling (age – 18). DNA was extracted from sperm samples using a Qiagen QIAamp DNA Blood Mini kit. Isolation of genomic DNA from sperm protocol 1 (QA03 Jul-10) was followed, but with the exceptions of substituting DTT in place of β-mercaptoethanol for buffer 2 and substituting buffer EB in place of buffer AE for the elution of DNA. DNA was extracted from whole blood using a Gentra Puregene Blood kit using the protocol for 10 ml of compromised whole blood from the Gentra Puregene Handbook (v.06/2011). Three separate Twist Bioscience gene panels were used for targeted NanoSeq sequencing in this study: (1) a custom pilot panel of 210 genes; (2) a similar but extended custom panel of 263 genes (Supplementary Table 4); (3) and a default exome-wide panel of 18,800 genes. The two custom gene panels were highly similar, with the extended panel being almost exclusively regions added to the pilot panel. From the 84 samples that underwent targeted sequencing, 13 were sequenced using a pilot panel of 210 canonical cancer or somatic driver genes, and all 84 were sequenced using the extended panel of 263 genes. Sequencing coverage and mutations were merged from samples sequenced on both targeted panels and they were treated as ‘targeted' samples in the Article. The custom panels were designed by gathering sets of published lists of genes implicated as drivers in cancers61,62,63,64 and somatic tissues6,65 as previously described5. Restriction-enzyme whole-genome NanoSeq libraries were prepared as previously described4 and subjected to whole-genome sequencing (WGS) at target 20–30× coverage on NovaSeq (Illumina) platforms to generate 150-bp paired-end reads with 9–10 samples per lane. Standard WGS of blood (31.7× median coverage) was used to generate matched-normal libraries for both restriction-enzyme NanoSeq blood and sperm samples. Targeted and exome NanoSeq libraries were prepared by sonication and one to two rounds of pull down of target sequences. They were then PCR amplified and sequenced with NovaSeq (Illumina) platforms to generate 150-bp paired-end reads with 7–8 samples per lane for the targeted panel and 2 lanes per sample for the exome panel. These steps are described in detail in ‘Sonication NanoSeq, Library amplification and sequencing, and Hybridization Capture' of supplementary note 1 of ref. All samples were processed using a Nextflow implementation of the NanoSeq calling pipeline (https://github.com/cancerit/NanoSeq). Whole-genome NanoSeq samples were called with their matched WGS normal and default parameters except for var_b (minimum matched normal reads per strand) of 5 as needed for WGS normals, cov_Q (minimum mapQ to include a duplex read) of 15 and var_n (maximum number of mismatches) of 2. For targeted and exome samples, we leveraged the high sequencing depth and high polyclonality to exclude variants with VAF > 10% instead of using a matched normal. After variant calling, we further filtered variants to those below 1% VAF. The few variants observed between 1% and 10% duplex VAF as variants were highly enriched for mapping artefacts, particularly for indels. No excluded variant from the additional 1% VAF threshold was found to be either a likely driver or a ClinVar pathogenic variant. A set of common germline variants from dbsnp67 and a custom set of known artefactual call sites in NanoSeq datasets were masked for coverage and variant calls as previously described4. The single-molecule accuracy of the duplex sequencing method NanoSeq allows sequencing of polyclonal cell types such as sperm, but also renders mutation calls sensitive to nontarget cell-type contamination and to contamination of foreign DNA. Nontarget cell-type contamination was evaluated using manual cell counting of semen samples, which resulted in the exclusion of six samples with a sperm count of <1 million sperm per ml. Sperm counting methods and analysis are detailed in Supplementary Note 1. Foreign DNA contamination in whole-genome NanoSeq samples was assessed using verifyBamID68, which checks whether reads in a BAM file match previous genotypes for a specific sample, with higher values indicating more contamination. Three blood whole-genome NanoSeq samples were excluded on the basis of a verifyBamID alpha value above the suggested4 cut-off value of 0.005. In sperm, we found that several samples had outlier mutation burdens with verifyBamID values just below the 0.005 cut-off. This result is logical, as sperm has a much lower mutation rate compared with somatic tissues, for which the recommended cut-off was designed. Consequently, sperm samples are more sensitive to low levels of contamination. To account for this, we adjusted the verifyBamID alpha threshold for sperm to a more stringent level of 0.002, which resulted in the exclusion of three samples on the basis of this criterion. When assessing foreign DNA contamination in targeted and exome samples, we found that 9 targeted and 3 exome samples had verifyBamID values above >0.002, slight outlier mutation burdens and a high ratio of SNP masked variants to passed variants (4-fold to 16-fold more masked variants). After further investigation, we found that all samples that exceeded verifyBamID thresholds were processed in the same sequencing batch and that this contamination could be explained by inherited germline variants of other samples in that same batch. This result suggested that a small amount of cross-contamination may have occurred during sample preparation or sequencing steps. To remove contaminant germline mutations, we performed an in silico decontamination step as previously described4. This step involved calling germline variants from all targeted and exome samples using bcftools mpileup69 at sites where there were >10 reads and a mutation call with VAF > 0.3. All such sites were subsequently masked across all samples for both mutation calls and coverage, which essentially extended the default common SNP mask to also include rare inherited variants across the cohort. This approach resulted in all samples previously identified as contaminated with mutation burdens consistent with their age and all with a ratio of masked to passed variants of <0.1, and were therefore retained for analysis. Given that mutation rates are strongly influenced by trinucleotide composition, it is important to consider differences in sequence composition when comparing mutation rates in datasets that target different regions of the genome70. The coding region target panels and the restriction enzyme used in whole-genome NanoSeq for instance, each will systematically bias sequencing coverage to specific genomic regions that may not reflect the full genome. To correct for these effects, in each sperm NanoSeq dataset, we generated a corrected mutation burden relative to the full genome trinucleotide frequencies as previously described4. To compare whole-genome NanoSeq mutation burdens to mutation burdens from standard WGS, we multiplied the corrected mutation burden estimates described in the previous section by the genome size per cell type. We assumed 2,861,326,455 mappable base pairs in a haploid cell for germline datasets and the diploid equivalent for blood. External datasets for comparison to NanoSeq results were processed to achieve comparable burden estimates. For testis WGS samples71, we implemented a previously described method4 that restricts analysis to regions with high coverage (20+ reads) that overlap with NanoSeq covered regions and corrected for differences in trinucleotide background frequencies relative to the full genome. For trio paternally phased DNMs from standard sequencing, as a callable genome size per sample following thorough filtering was available, we generated the mutation per cell estimate by multiplying the paternally phased DNM count by the ratio of the total genome size to the callable genome size of the sample as follows: DNMs paternal × (total genome/callable genome). For comparison with cell types in blood, we directly compared results to published mutation burden regressions13. To investigate the relationship between age and mutation burdens, we performed linear mixed-effects regression analyses. For each tissue and mutation type for which a regression was performed, the model was constructed using the lmer function from the lme4 package72 in R. Each model included age at sampling as a fixed effect and a random slope for each individual to account for multiple time point samples, with restricted maximum likelihood (REML) set to false, specified as follows: The 95% CIs for regression lines were calculated through bootstrapping by simulating prediction intervals. For each model, we generated 1,000 bootstrap samples. Predictions and their associated standard errors were calculated for a sequence of ages from 14 to 84 years. We extracted DNM signatures using hierarchical dirichlet process (HDP; https://github.com/nicolaroberts/hdp), which is based on the Bayesian hierarchical dirichlet process. HDP was run with double hierarchy: (1) individual identifier (ID) and (2) tissue types (either blood or sperm), without the COSMIC reference signatures73 (v.3.3) as priors, on the mutation matrices. Both clustering hyperparameters, beta and alpha, were set to one. The Gibbs samples ran for 30,000 burn-in iterations (parameter ‘burnin'), with a spacing of 200 iterations (parameter ‘space'), from which 100 iterations were collected (parameter ‘n'). After each Gibbs iteration, three iterations of concentration parameters were conducted (parameter ‘cpiter'). Two components were extracted as DNM signatures, which were further reconstructed and decomposed into known COSMIC (v.3.3) SBS signatures using SigProfilerAssignment (https://github.com/AlexandrovLab/SigProfilerAssignment). To examine genes under positive selection and to quantify global selection, we used the dNdScv algorithm28. This algorithm was extended using the base-pair-level duplex coverage, the methylation level and the pentanucleotide context to capture more complex context-dependent mutational biases and to achieve higher accuracy for our selection analysis. Detailed methods for input mutations, model selection and evaluation, site dN/dS tests, driver mutation estimation and gene set enrichment are described in Supplementary Note 3. Genes were also annotated for their potential mutation mechanism observed in sperm and cancer or developmental disorders when available. In sperm, genes were labelled as LOF if they had nominal enrichment of nonsense + splice variants and/or indel variants (ptrunc_cv <0.1 | pind_cv <0.1) and 2+ LOF mutations. There were two exceptions to this, whereby genes met these thresholds but were labelled as activating owing to having a restricted repertoire of LOF mutations that are known to be oncogenic in cancers: CBL (LOFs in and downstream of the RING zinc finger domain)75 and PPM1D (LOFs in final two exons)76. All other genes had missense enrichment only and were labelled as activating. The mechanism in cancer was defined by using the ‘role in cancer' field of the COSMIC cancer gene census (v.99)38, where ‘oncogene' was labelled as activating and ‘tumour suppressor gene' as LOF. Annotations of a fusion mechanism were not displayed, except for genes that had neither an oncogene nor a tumour suppressor annotation, which were labelled as ‘fusion only'. The developmental disorder mechanism was defined by using the variation consequence field of DDG2P, for which ‘restricted repertoire of mutations;activating' was labelled as activating and ‘loss_of_function_variant' was labelled as LOF. The data sources included protein domains from UniProt78, somatic mutations from the exome and genome-wide screens of the COSMIC (v.99)38, ClinVar (release 2024.07.01)41 pathogenic annotation, per base pair cohort-wide (targeted + exome) NanoSeq coverage, sperm mutation count (number of independent individuals with a mutation) and mutation consequence and amino acid change annotated by the dNdScv algorithm28. These data were plotted using code modified from the lolliplot function in the trackViewer R package79. Variants were annotated using Ensembl's variant effect predictor (VEP)80 with added custom annotations of mutation context, ClinVar (release 2024.07.01)41, CADD (v.GRCh37-v1.6)42 and the average methylation level in testis. The average methylation level was calculated by selecting CpG sites with coverage of three or more and averaging the per cent of sites methylated between the two samples. Variants were annotated as likely monoallelic disease-causing mutations if they met at least one of two criteria: (1) reported in ClinVar as pathogenic, likely pathogenic or if they were reported as ‘conflicting classifications of pathogenicity', where the conflict was between reports of pathogenic/likely pathogenic and ‘uncertain significance' with no reports of benign or likely benign and not specified as a recessive condition; or (2) were a highly damaging variant in high-confidence monoallelic developmental disorder genes from DDG2P39. Genes met the following criteria: (1) allelic requirement being monoallelic_autosomal, monoallelic_X_hem, monoallelic_X_het or mitochondrial; (2) confidence in strong, definitive or moderate; and (3) a mutation consequence of ‘absent gene product'. Highly damaging was defined as being a ‘high' impact variant in the VEP annotation (frameshift splice_acceptor, splice_donor, start_lost, stop_gained or stop_lost) or a missense variant with CADD42 score >30 (top 0.1% damaging). Variants were defined as a likely driver if they met the ‘highly damaging' criteria defined above in a significant germline-selection gene with LOF mutation enrichment or if they were in 1 of the 24 significant mutation hotspots. To calculate the mean count of synonymous, missense or LOF or pathogenic mutations per sperm cell, we summed the duplex VAFs of all variants in that class. For example, if an individual had 3 synonymous mutations each observed once with a duplex coverage of 100 at each of those sites, each of those variants would have a duplex VAF of 1/100 = 0.01. The sum of VAFs in this example would then be 0.03 and this would then be reported as the estimate for the mean count of synonymous variants per sperm cell for that individual. At low fractions such as 0.03, the mean count per cell is approximately equivalent to the percentage of sperm with this mutation class (3%); therefore, the driver and disease mutations are reported as percentage estimates. At higher fractions (for example, mean count > 1), the fractions are not equivalent to percentage, as many cells will have multiple variants of that class; therefore, the estimates are reported as mean count. Expected mean counts for SNVs were generated by annotating each possible substitution at each covered site with an expected number of mutations per sample as given by expCountSNV = context_mut_rate × duplex_coverage × age_correction. The context_mut_rate was given by the 208 basePairCov + cpgMeth trinucleotide mutation model estimates for that trinucleotide + methylation mutation context (Supplementary Note 3 and Supplementary Table 9). Duplex coverage is the exact duplex coverage at that site for that sample. The age_correction parameter was given to normalize the mutation model estimates (derived from all exome samples) to the mutation rate of that sample based on age. Specifically, we fit a linear model to the mutation burden versus age of the exome samples and used this to generate a predicted mutation rate for each sample based on age. The per sample correction parameter was calculated as the age predicted mutation burden divided by the mean mutation rate of all exome samples (3.89 × 10–8). The resulting corrections spanned from 0.50 (youngest sample) to 1.42 (oldest samples). The expected indel mutation rate was calculated in the same way, except with a single mutation rate parameter (indels/bp) expCountIndel = indel_mut_rate × duplex_coverage × age_correction. The expected mean count was then calculated for each category (for example, synonymous or likely disease) by summing the expected values for each SNV and/or indel base pair matching the relevant annotation. As background for possible ClinVar pathogenic variants, we only considered indels of size 21 bp or less, the largest detected indel in the dataset. Regressions were fit with either linear models or generalized linear models (glm in R) with family = quasibinomial. We tested for associations between mutation outcome variables from sperm genome, sperm exome, sperm targeted and blood genome NanoSeq data and the phenotype predictor variables of BMI, smoking pack-years and alcohol drink-years. These tests were performed using a Gaussian family generalized linear regression in R. For each mutation outcome variable the test took the following form: glm(mutationOutcome ~ age_at_sampling + BMI + pack_years + drinkYears, family = “gaussian”). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequencing data are available through managed access at the European Genome-Phenome Archive (EGA) under accession numbers EGAD00001015591 (targeted and exome NanoSeq), EGAD00001015590 (whole-genome NanoSeq) and EGAD00001015592 (WGS matched normals). All non-TwinsUK files necessary to reproduce the results are available from GitHub (https://github.com/mattnev17/spermPositiveSelectionManuscript). Individual-level data from the TwinsUK cohort cannot be publicly shared or deposited owing to restrictions in the original participant consent. Access to these data is governed by the TwinsUK Resource Executive Committee (TREC), which reviews requests on a monthly basis. For information on how to apply for access to TwinsUK genotype and phenotype data, please see the website: https://twinsuk.ac.uk/researchers/access-data-and-samples/request-access/. The NanoSeq variant calling pipeline, available from GitHub (https://github.com/cancerit/NanoSeq; v.3.3), was used to process duplex sequencing data. R (v.4.3.1) was used for statistical analyses and visualization. Code modified from trackViewer (R/Bioconductor package v.1.38.0) was used to generate gene mutation lollipop plots. lme4 (v.1.1-33) was used for linear mixed-effects models. dNdScv (https://github.com/im3sanger/dndscv; version as of commit on 29 September 2023) was used for selection analysis. HDP (https://github.com/nicolaroberts/hdp) was used for DNM mutational signature extraction. SigProfilerAssignment (v.0.0.27; https://github.com/AlexandrovLab/SigProfilerAssignment) was used for COSMIC signature fitting. All custom analysis code and scripts are available from GitHub (https://github.com/mattnev17/spermPositiveSelectionManuscript). & Wilkie, A. O. M. Evidence for selective advantage of pathogenic FGFR2 mutations in the male germ line. & Wilkie, A. O. M. Paternal age effect mutations and selfish spermatogonial selection: causes and consequences for human disease. The impact of paternal age on new mutations and disease in the next generation. Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Lawson, A. R. J. et al. Somatic mutation and selection at population scale. Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. Martincorena, I. et al. Somatic mutant clones colonize the human esophagus with age. Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. The landscape of somatic mutation in normal colorectal epithelial cells. The mutational landscape of normal human endometrial epithelium. Lawson, A. R. J. et al. Extensive heterogeneity in somatic mutation and selection in the human bladder. The mutational landscape of human somatic and germline cells. Diverse mutational landscapes in human lymphocytes. Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Heller, C. H. & Clermont, Y. Kinetics of the germinal epithelium in man. B., Li, P. S. & Goldstein, M. Spermatogenesis in humans and its affecting factors. Choi, S.-K., Yoon, S.-R., Calabrese, P. & Arnheim, N. Positive selection for new disease mutations in the human germline: evidence from the heritable cancer syndrome multiple endocrine neoplasia type 2B. & Wilkie, A. O. M. Cellular evidence for selfish spermatogonial selection in aged human testes. Maher, G. J. et al. Visualizing the origins of selfish de novo mutations in individual seminiferous tubules of human testes. Eboreime, J. et al. Germline selection of PTPN11 (HGNC:9644) variants make a major contribution to both Noonan syndrome's high birth rate and the transmission of sporadic cancer variants resulting in fetal abnormality. Kennedy, S. R. et al. Detecting ultralow-frequency mutations by Duplex Sequencing. Detection of ultra-rare mutations by next-generation sequencing. Hoang, M. L. et al. Genome-wide quantification of rare somatic mutations in normal human tissues using massively parallel sequencing. Liu, M. H. et al. NA mismatch and damage patterns revealed by single-molecule sequencing. Verdi, S. et al. TwinsUK: The UK Adult Twin Registry Update. Sasani, T. A. et al. Large, three-generation human families reveal post-zygotic mosaicism and variability in germline mutation accumulation. Prolonged persistence of mutagenic DNA lesions in somatic cells. Martincorena, I. et al. Universal patterns of selection in cancer and somatic tissues. Widespread transcriptional scanning in the testis modulates gene evolution rates. & Lindskog, C. The Human Protein Atlas—spatial localization of the human proteome in health and disease. Imielinski, M., Guo, G. & Meyerson, M. Insertions and deletions target lineage-defining genes in human cancers. Analyses of non-coding somatic drivers in 2,658 cancer whole genomes. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Wood, K. A. et al. SMAD4 mutations causing Myhre syndrome are under positive selection in the male germline. & Morishima, K. KEGG: new perspectives on genomes, pathways, diseases and drugs. Qi, H., Dong, C., Chung, W. K., Wang, K. & Shen, Y. Deep genetic connection between cancer and developmental disorders. Yavuz, B. R. et al. Neurodevelopmental disorders and cancer networks share pathways, but differ in mechanisms, signaling strength, and outcome. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Flexible and scalable diagnostic filtering of genomic variants using G2P with Ensembl VEP. Alankarage, D. et al. Myhre syndrome is caused by dominant-negative dysregulation of SMAD4 and other co-factors. Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. & Kircher, M. CADD-Splice—improving genome-wide variant effect prediction using deep learning-derived splice scores. Sherwood, K. et al. Germline de novo mutations in families with Mendelian cancer syndromes caused by defects in DNA repair. The impact of ancestral, genetic, and environmental influences on germline de novo mutation rates and spectra. The critical roles of somatic mutations and environmental tumor-promoting agents in cancer risk. Turner, T. N. et al. denovo-db: a compendium of human de novo variants. The mutational constraint spectrum quantified from variation in 141,456 humans. Kong, A. et al. Rate of de novo mutations, father's age, and disease risk. Timing, rates and spectra of human germline mutation. Jónsson, H. et al. Parental influence on human germline de novo mutations in 1,548 trios from Iceland. Kaplanis, J. et al. Genetic and chemotherapeutic influences on germline hypermutation. Giannoulatou, E. et al. Whole-genome sequencing of spermatocytic tumors provides insights into the mutational processes operating in the male germline. Kingdom, R. et al. Rare genetic variants in genes and loci linked to dominant monogenic developmental disorders cause milder related phenotypes in the general population. Prevalence and architecture of de novo mutations in developmental disorders. The genomic and clinical consequences of replacing procarbazine with dacarbazine in escalated BEACOPP for Hodgkin lymphoma: a retrospective, observational study. Insights into non-crossover recombination from long-read sperm sequencing. Porubsky, D. et al. Human de novo mutation rates from a four-generation pedigree reference. Cimadomo, D. et al. Impact of maternal age on oocyte and embryo competence. Paul, C. & Robaire, B. Ageing of the male germ line. et al. Genetic landscape of esophageal squamous cell carcinoma. Whole genomes redefine the mutational landscape of pancreatic cancer. Aaltonen, L. A. et al. Pan-cancer analysis of whole genomes. Nguyen, B. et al. Genomic characterization of metastatic patterns from prospective clinical sequencing of 25,000 patients. Kakiuchi, N. & Ogawa, S. Clonal expansion in non-cancer tissues. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Sherry, S. T. et al. dbSNP: the NCBI database of genetic variation. Zhang, F. et al. Ancestry-agnostic estimation of DNA sample contamination from sequence reads. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Expanded encyclopaedias of DNA elements in the human and mouse genomes. & Walker, S. Fitting linear mixed-effects models using lme4. Alexandrov, L. B. et al. Clock-like mutational processes in human somatic cells. Amberger, J. S., Bocchini, C. A., Scott, A. F. & Hamosh, A. OMIM.org: leveraging knowledge across phenotype–gene relationships. Martinelli, S. et al. Molecular diversity and associated phenotypic spectrum of germline CBL mutations. Khadka, P. et al. PPM1D mutations are oncogenic drivers of de novo diffuse midline glioma formation. UniProt: the Universal Protein Knowledgebase in 2023. & Zhu, L. J. trackViewer: a Bioconductor package for interactive and integrative visualization of multi-omics data. We thank the TwinsUK research volunteers for participating in the study; L. O'Neill, C. Latimer and all of the CASM support team for their assistance; and C. Seymour, E. Ferraro and C. White for training and guidance on sperm counting. This research was funded by Wellcome Trust Grant 108413/A/15/D and 220540/Z/20/A. TwinsUK is funded by the Wellcome Trust, the Medical Research Council, Versus Arthritis, the European Union Horizon 2020, the Chronic Disease Research Foundation (CDRF), Zoe Ltd, the National Institute for Health and Care Research (NIHR) Clinical Research Network (CRN) and the Biomedical Research Centre based at Guy's and St Thomas' NHS Foundation Trust in partnership with King's College London. Cancer, Ageing and Somatic Mutation, Wellcome Sanger Institute, Hinxton, UK Matthew D. C. Neville, Andrew R. J. Lawson, Rashesh Sanghvi, Federico Abascal, My H. Pham, Alex Cagan, Pantelis A. Nicola, Tetyana Bayzetinova, Adrian Baez-Ortega, Kirsty Roberts, Stefanie V. Lensing, Raul E. Alcantara, Michael R. Stratton, Peter J. Campbell, Iñigo Martincorena, Matthew E. Hurles & Raheleh Rahbari Sequencing Operations, Wellcome Sanger Institute, Hinxton, UK María Paz García, Sam Wadge & Kerrin Small Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors reviewed and edited the manuscript. are co-founders, shareholders and consultants for Quotient Therapeutics. is an employee of Quotient Therapeutics. is a co-founder of, consultant to and holds shares in Congenica, a genetics diagnostic company. The remaining authors declare no competing interests. Nature thanks Ziyue Gao, Jan Korbel and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b,c, Slides of Papanicolaou stained semen samples for (a) an azoospermic sample where no sperm cells are visible, (b) an oligozoospermic sample where a small number of sperm samples are visible and (c) a normozoospermic sample where many sperm cells are visible. The black band in the bottom left of each slide photo corresponds to 100 µm. Staining was independently repeated once with similar results. d, The distribution of sperm counts on a log scale among semen samples analysed with colour bands indicating the concentration bin of the sample. All samples below 1 million/mL were subsequently excluded. e, The distributions of mutation burden per year from blood samples and three categories of sperm samples broken down by sperm concentration. a, Graphical relationship between NanoSeq methods applied in manuscript. Both sperm and blood samples underwent genome NanoSeq, used for mutation burden and mutational signature analyses. Only sperm samples were used for targeted NanoSeq applied for selection, driver, and pathogenic variant analyses. Targeted NanoSeq is adaptable to different target panels, and we have named the sample sets as “targeted” for the samples using the 263 gene cancer panel and “exome” for samples using the exome wide panel. b, Mean duplex coverage (log scale) and percentage of genome covered (log scale) per sample. Panels summarise the mean duplex coverage (dx) and mean percentage of genome covered per NanoSeq type and tissue. c, Mutation burden of targeted (dark orange), exome (yellow), and genome (blue) sperm sequenced samples that were observed without correction (left), corrected for trinucleotide composition of covered base pairs relative to the whole genome (middle) or corrected and masked for mutations and coverage in the 44 genes linked to germline positive selection (right). a,b, Distribution of indel types observed in whole genome (a) sperm and (b) blood. a,b, Substitutions (a) and indels (b) per diploid cell from blood NanoSeq relative to specific blood cell types13. c,d, Substitutions per haploid cell for sperm (c) and diploid cell for blood (d) split by signature contributions of SBS1, SBS5, and SBS19. a,b,c,d, Models are linear mixed regressions, with the central line showing the model fit and the shaded bands indicating 95% confidence intervals calculated by parametric bootstrapping. e, Ratio of age-corrected blood to sperm substitutions per diploid cell per year for mutations assigned to SBS1 and SBS5. Each dot corresponds to an individual with both a blood and sperm sample and where individuals had multiple timepoints the mean value of all timepoints in that tissue was used. a,b, Mean duplex coverage (a) and methylation percentage (b) of all base pairs with exome sequencing coverage split by mutation consequence. c, C > T mutation rate point estimates at CpG sites in n = 38 biologically independent exome-sequenced sperm samples split by methylation bin based on percentage methylated from testis bisulfite sequencing. d, Comparison of global dN/dS ratio point estimates from mutations of n = 38 exome-sequenced sperm samples using different modifications to the dNdScv algorithm. Categories include all nonsynonymous mutations, missense, nonsense or essential splice. The basic model excludes genes with no coverage and uses default parameters. Additional models show the impact of adding corrections for duplex coverage per base pair (BasePairCov), CpG methylation level (CpGmeth), and pentanucleotide context (Penta). e, Comparison of per-gene significance in exome-wide (blue) or restricted hypothesis (orange) dN/dS tests using the different models. a, dN/dS ratios for sperm SNVs across germline selection genes and cancer gene census genes split by ten canonical cancer pathways, where the dotted black line indicates neutrality and the dotted orange line represents the cohort average across all genes. b, The mutation mechanism assigned to each gene based on the mutation pattern in sperm, developmental disorders, and cancer (Methods). a,b,c,d,e,f, Observed sperm mutations across the cohort for six illustrative genes where the height of the “lollipop” represents the number of unique samples with a mutation at that location and the colour represents its mutation type. Mutations are labelled with their amino acid consequence for point substitutions or their insertion (ins)/deletion (del) consequence of in frame (IF) or frameshift (FS). A “P” indicates that the variant is classified as pathogenic/likely pathogenic in ClinVar. The relationship between age and the mean count of SNVs (non-coding, synonymous, missense, and loss-of-function (nonsense or essential splice)) and indels (non-coding indel and coding indel) per sperm cell. Error bands indicate 95% confidence intervals of linear regressions. Correlation of cohort phenotypes with mutation outcomes across sequencing datasets. Associations were tested using two-sided generalised linear models (family = gaussian). P values were adjusted for multiple comparisons using the false discovery rate (FDR) method. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Sperm sequencing reveals extensive positive selection in the male germline. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09581-z'>Age and gender distortion in online media and large language models</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 15:04:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. This continuing debate is limited by the lack of large-scale multimodal data on stereotypical associations and the inability to compare these to ground truth indicators. Here we overcame these challenges in the analysis of age-related gender bias7,8,9, for which age provides an objective anchor for evaluating stereotype accuracy. Despite there being no systematic age differences between women and men in the workforce according to the US Census, we found that women are represented as younger than men across occupations and social roles in nearly 1.4 million images and videos from Google, Wikipedia, IMDb, Flickr and YouTube, as well as in nine language models trained on billions of words from the internet. This age gap is the starkest for content depicting occupations with higher status and earnings. We demonstrate how mainstream algorithms amplify this bias. A nationally representative pre-registered experiment (n = 459) found that Googling images of occupations amplifies age-related gender bias in participants' beliefs and hiring preferences. Furthermore, when generating and evaluating resumes, ChatGPT assumes that women are younger and less experienced, rating older male applicants as of higher quality. Our study shows how gender and age are jointly distorted throughout the internet and its mediating algorithms, thereby revealing critical challenges and opportunities in the fight against inequality. Although few deny that stereotypes, or generalizations about social groups10,11,12, are harmful, a fundamental question remains contested: are common stereotypes accurate1,2,3 or socially distorted4,5,6? Some argue that commonplace stereotypes accurately capture observable aspects of social groups; otherwise, they would not gain such widespread adoption1,2,3. Yet, others argue that stereotypes are often exaggerated or illusory4,5,6. Assessing stereotype accuracy is challenging because stereotypes involve not only statistical associations (such as expected correlations among the features of a social group) but also normative judgements (such as that one group is superior to another) for which there is no well-defined ground truth10,11,12. Even for statistical associations, identifying the ground truth is difficult. In some cases, this stems from disagreement on how to measure the ground truth, such as enduring debates over how to measure intelligence13 (a heavily stereotyped characteristic14). Yet, even when there is agreement on the relevant constructs, there is often a lack of large-scale, quantifiable cultural data for measuring stereotypical associations and comparing these to ground truth indicators. As a result, research on stereotypes often yields inconsistent findings, calling into question the pervasiveness and impact of these biases. In this study, we overcame these limitations in the analysis of age-related gender bias, which not only involves biological age as an objective anchor for evaluating stereotype accuracy, but which can also be linked to large-scale statistical biases in how the ages of women and men are depicted online. On the one hand, evidence abounds that older women face a dual bias at the intersection of gender and age. Policy reports8,15,16, media coverage17 and workplace interviews7,18 indicate that older women are discriminated against in hiring and promotion across industries (known as ‘gendered ageism'7,18,19). This is related to a general statistical bias towards associating women with expectations of youth. From entertainment media to the workplace, women face persistent pressure to appear young, which results in a ‘beauty tax' with sizeable financial and time costs20. This bias also manifests in everyday language. Women in academia21 and industry22 are more likely than men to be referred to using infantilizing pronouns (such as ‘girls'). These patterns suggest that age-related gender expectations may form a culture-wide statistical bias that influences people's perceptions of others throughout society23,24. On the other hand, the statistical association between women and youth contradicts observable socioeconomic realities. Since the 1960s, women have consistently outlived men in the USA by as much as 8 years, a gap that has been increasing25,26. Census data on occupations present similarly puzzling trends (Supplementary Fig. 1). Over the past decade, there has been no correlation between the fraction of women in an occupation and its median age, according to the US census (Extended Data Fig. There were also no clear differences in the age distribution of women and men throughout the workforce from 2009 to the present (Supplementary Fig. Moreover, recent surveys failed to observe gendered ageism in certain organizational settings and even suggest that older women may be less affected by stereotypes than older men27,28. These inconsistent findings resonate with critiques against claims of enduring gender inequality, such as research showing declines in gender stereotypes over the last century in online text29,30, as well as studies showing that hiring across industries increasingly favours women31,32. This dissonant landscape raises the question of whether age-related gender bias is an organization-specific or industry-specific problem, or whether it is a culture-wide distortion that continues to reflect and contribute to systemic inequalities. We argue that this uncertainty is fuelled by the lack of (1) culture-wide multimodal data on the associations between gender and age and (2) computational methodologies for comparing these associations with ground truth indicators. So far, only a handful of studies have examined age–gender associations in small-scale surveys and interviews with professional women7,18,27,28,33 or in sparse, non-representative observational studies of particular industries, such as celebrities and athletes in entertainment media34,35,36,37,38. However, failing to observe age-related gender bias in limited samples of a few contexts does not indicate a lack of prominence on a culture-wide scale. Social biases in how people categorize the world frequently emerge only at scale39,40 and can manifest as exaggerated or even illusory beliefs41. This suggests the alternative view that skewed associations between gender and age can emerge as a large-scale statistical bias that distorts socioeconomic realities despite inconsistencies across small-scale contexts. Although a number of recent studies revealed the exaggeration of male representation in online texts and images6,42, no comparable analyses exist for tracking age-related representations of gender. To address this gap, we produced a large-scale culture-wide dataset on age–gender associations across modalities, including images, videos and textual data, collected from popular sources of digital media. We began by examining the gender and age associations of all social categories in the English language (n = 3,495) in more than 1.3 million images and thousands of videos from Google, Wikipedia, IMDb, Flickr, YouTube and a random sample of the world-wide web (see ‘Image and video datasets' in Methods for details on pre-processing and post-processing). We benchmarked online images of occupations against the US census data to examine whether they exaggerate the association between women and youth. We went beyond the visual modalities by examining age-related gender bias in nine popular language models trained on billions of words from across the internet, including Reddit, Google News, Wikipedia and Twitter (see ‘Measuring age and gender in online text' in Methods). By examining age-related gender bias in large-scale internet data, our study was uniquely poised to examine the role that mainstream algorithms play in reinforcing this bias. Across all image datasets spanning five popular online platforms, women are consistently represented as younger than men, regardless of whether the age and gender of faces are measured using human judgements, machine learning or ground truth data. First, we analysed 657,035 images from the Google search engine associated with 3,495 social categories, in which the gender and age of images were classified by human coders6 (see ‘Image data collection procedure' in Methods; all categories were examined using retrieved Google Images containing human faces). We found that women in Google Images were coded as significantly younger than men, both for non-gendered searches (such as ‘doctor' or ‘banker'; mean difference in age groups, Mdiff = 0.37; t = −73.84; P = 2.2 × 10−16; n = 3,434 categories; Fig. 1a) and gendered searches (such as searching ‘female doctor' and ‘male doctor'; Mdiff = 0.29; t = −36.52; P = 2.2 × 10−16; n = 2,960 categories; Fig. 1b). Replicating this method over Wikipedia revealed that women in Wikipedia images were also coded as significantly younger than men (Mdiff = 0.71; t = −39.62; P = 2.2 × 10−16; n = 1,251 categories; Fig. a–j, The age of either female or male faces according to the top 100 Google Images associated with 3,434 social categories (n = 161,484 images) (a), the top 100 Google Images retrieved using gendered searches (such as by searching ‘female athlete' or ‘male athlete') shown for all non-gendered categories in WordNet (n = 2,960 categories; 495,551 images) (b), 1,251 categories in Wikipedia (from the Srinivasan et al.68 dataset; 14,709 images) (c), celebrities identified by the top 100,000 most popular pages on IMDb (n = 451,570 images) (d), biographical Wikipedia pages describing the same celebrities in the IMDb–Wiki dataset (n = 57,932 images) (e), the top 50 most popular celebrities from 1951 to 2004 as they appear in Google Images, according to the CACD (n = 149,889 images) (f), a random sample across the world-wide web (the 2017 UTK dataset; n = 20,000 images) (g), a random sample from Flickr (the 2014 Adience dataset; n = 26,580 images) (h), a random sample of images from online news websites (the 2008 LFW dataset; n = 13,233 images) (i) and images of the same people identified in the LFW dataset, extracted across 3,425 YouTube videos in 2011 (n = 5,000 images) (j). The method for coding age and gender varies by dataset; a–c rely on human coders; panels d,e,g rely on ground truth measures of the age and gender of celebrities posted publicly online; f,h–j rely on automated deep learning classifications of gender and age. Solid gold and blue lines indicate the average age for female and male faces, respectively. These results are robust to collecting Google Images from different countries around the world (Supplementary Fig. 6 and 7 and Supplementary Table 6), such as word polysemy, word gender connotation, word age connotation and word frequency in Google Search and in everyday language; (3) the visual features of the images, including the number of faces per image, the number of images associated with each category overall, whether the image repeats across searches and whether the image is photographic or displays a digital avatar (Supplementary Table 7); and (4) whether the faces in each image are cropped before classification (Supplementary Fig. We confirm that these results reflect images from a wide range of websites (Supplementary Fig. Next, we analysed the 2018 IMDb–Wiki dataset43 and the 2014 Cross-Age Celebrity Dataset (CACD)44 consisting of Google Images, each of which provides the true gender and age of the celebrities depicted using their public bio pages and time-stamped photographs. Figure 1 shows that female celebrities are, on average, 6.5 years younger than men on IMDb (t = −169.9; P = 2.2 × 10−16; n = 451,562 images; Fig. 1d), 3.27 years younger on Wikipedia (t = 10.64; P = 2.2 × 10−16; n = 57,972 images; Fig. In all cases, the most common (modal) age for women is in their 20s, whereas in images from IMDb and Google, the most common ages for men are 40 years and 50 years, respectively. These analyses show that age-related gender bias online is not an artefact of human perceptions of gender and age, because it is replicated using verified objective information about the age and gender of those depicted. That age-based gender bias replicates strongly in the context of celebrities is concerning, given the salient role that celebrities play in reinforcing stereotypes45. Finally, we explored age biases in the representation of women and men using prominent image datasets developed for training machine learning algorithms. The age and gender classifications in these datasets were provided by computer vision algorithms constructed by the research teams that released these datasets. We found that women were automatically classified as significantly younger than men in the 2017 UTK46 dataset consisting of a diverse sample of images from across the world-wide web (Mdiff = 5.12 years; t = −19.9; P = 2.2 × 10−16; n = 24,106 images; Fig. 1h) and the 2008 Labeled Faces in the Wild (LFW) dataset48 consisting of a random sample of images from Google News (Mdiff = 0.84; t = −44.89; P = 2.2 × 10−16; n = 13,143 images; Fig. These findings further generalized our results beyond the perceptions of human coders. A remaining question is whether age-related gender bias is also observed in online videos, which increasingly dominate the world-wide web49. Although an exhaustive analysis of online videos is beyond the scope of this study, we analysed two open-source datasets of screenshots from YouTube videos that provide compelling support for our theory. First, we examined the correlation between gender and age in the 2011 YouTube Faces dataset50, consisting of 3,645 faces of celebrities extracted from 3,425 YouTube videos. Women in the YouTube Faces dataset appear significantly younger than men according to machine learning classifications (Mdiff = 0.87; t = −25.68; P = 2.2 × 10−16; n = 3,645 images; Fig. 1j). We also analysed a more recent dataset of YouTube videos called the 2022 CelebV-HQ dataset51, consisting of 35,666 images collected by identifying public lists of celebrities on Wikipedia and automatically collecting the top 10 YouTube videos associated with each celebrity. Although this dataset contains only a binary measure of age (faces are coded as either young = 1 or old = 0), we can still test our theory by comparing the fraction of women and men coded as young. Only 20% of men were classified as young compared with 33% of women, marking a significantly higher rate of youthful presentations for women (P = 2.2 × 10−16; two-tailed proportion test). We compared these findings to available industry-level ground truth data to measure the extent to which online images distort the underlying sociodemographic realities of age (occupation-level census data containing both gender and age information are unavailable; see ‘Comparing online images with the census' in Methods). We matched 867 social categories from our Google Images (Fig. 1a) dataset to occupational categories in the US census. Although gender–age associations in Google Images and census data are correlated at the industry level (r = 0.13; confidence interval = 0.11–0.15; P = 2.2 × 10−16; two-tailed Pearson's correlation; Extended Data Fig. 3 presents the absolute age gap between women and men in each industry, vertically ranked in terms of the magnitude of this gap while also placing the older gender on the right side. In the sales, resources and management industries, Google Images consistently presented the highest age gap relative to all census years (P < 0.001 for all pairwise comparisons; two-tailed Student's t-test). Moreover, in each of these industries, Google Images displayed men as older than women, whereas women were older than men for each of the census years examined in the sales industry and for two of the years in the resources industry. In the production and service industry, the magnitude of the age gap captured by Google Images was not higher than all census years; yet, the bias towards representing men as older was stable. In each census year, women were older than men in the production and service industries. It was only in Google Images that men were older than women in these industries, suggesting systematic age and gender distortions that associate women with youth. Given the observational and large-scale nature of these analyses, it is challenging to identify the mechanisms driving these age–gender associations. One such consideration pertains to the hypothesis that gender stereotypes are most salient in high-status and prestigious occupations, which play a prominent role in reinforcing gender expectations and norms of desirability52,53. To test this, we recruited a nationally representative US sample from Prolific (n = 1,002) to evaluate the status and prestige of 867 occupations matched between our Google Image data (Fig. 1a) and the US census from 2015 to 2022 (see ‘Collecting judgements of occupational status' in Methods). Occupations rated as higher status were more likely to elicit Google Images in which men were older than women (Extended Data Fig. 4a; r = 0.08; t = 11.28; P = 2.2 × 10−16; two-tailed Pearson's correlation; n = 867 occupations). 4b; r = 0.11; t = 2.5; P = 0.01; two-tailed Pearson's correlation; n = 532 occupations could be matched). Next, we showed that the probability of men appearing as older in Google Images is significantly higher for occupations associated with higher median earnings (Extended Data Fig. 4c; r = 0.11; t = 7.39; P = 1.07 × 10−13; two-tailed Pearson's correlation; n = 4,444 pairwise comparisons at the census year level from 2015 to 2022; yearly earnings logged). We found that the gender pay gap16,55, or the extent to which men earn more than women in the same occupation, is associated with the digital age gap, or the extent to which men appear older than women in Google Images (Extended Data Fig. 2d; r = 0.04; t = 7 = 3.05; P = 0.002; two-tailed Pearson's correlation; n = 4,444 pairwise comparisons at the census year level from 2015 to 2022; yearly earnings logged). These results were robust to numerous statistical controls (Supplementary Figs. A natural suspicion is that age-related gender bias in online images and videos may be driven by affordances of visual communication, such as image filters and cosmetics, which do not generalize to other modalities. Here we show that comparably salient patterns of age-related gender bias are readily observable in massive bodies of internet text data beyond the visual modality. We begin by analysing gender–age associations in GPT-2 Large56, the largest open-source language model of OpenAI trained on billions of tokens of text data from across the internet (see ‘Measuring age and gender in online text' in Methods; Supplementary Tables 14 and 15). 2, the representations of GPT-2 Large exhibit a strong correlation between the extent to which a social category is associated with men and older ages (r = 0.87; t = 105.57; P = 2.2 × 10−16; two-tailed Pearson's correlation). These results are robust to alternative methods for extracting age and gender associations (Supplementary Fig. 13), as well as to a range of statistical controls, including word frequency, gender, age and polysemy (Supplementary Fig. These associations are significantly predictive of ground truth age distributions by gender and occupation in the census, affirming their empirical coherence (Supplementary Tables 17 and 18). These results are not unique to GPT-2 Large. We replicated these patterns across eight different canonical and popular language models that vary in their training data and algorithmic training methods (Supplementary Figs. Correlation between age and gender associations for 3,495 social categories in GPT-2 Large. The trend line shows the linear prediction according to an ordinary least squares regression. The systematic distortion of age–gender associations in online images, videos and text across popular platforms that we have identified raises concerns about how mainstream algorithms trained on these data might amplify the spread of this bias. We begin by examining possible algorithmic amplification in the visual modality to investigate whether exposure to visual content from the Google search engine amplifies age-related gender bias in people's beliefs. To answer this question, we report the results of a pre-registered experiment. We recruited a nationally representative sample of US participants from Prolific (n = 500), who were tasked with using Google to search for images of occupations related to science, technology and the arts (Extended Data Fig. In total, 459 participants completed the task. Each participant used Google to retrieve descriptions of 22 randomly selected occupations from a set of 54 (‘participant experience' in Methods). The participants were randomized into treatment or control condition. In the treatment condition (hereafter ‘image condition'), the participants used Google Images to search for images of occupations, which they then uploaded to our survey. After uploading an image for an occupation, the participants were asked to label the gender of the image they uploaded and then to estimate the average age of someone in this occupation. The participants were also asked to rate their willingness to hire the person depicted in their uploaded image. In the control condition, the participants used Google Images to search for and upload images of basic unrelated categories (such as apple and guitar). After uploading a random image, the control participants were asked to estimate the average age of someone in a randomly selected occupation from the same set. The control participants were also asked to rate the ideal hiring age of someone in each occupation, as well as which gender (‘male' or ‘female') is most likely to belong to each occupation. This design allowed us to evaluate the treated participants' age estimates of occupations after uploading an image of a man or woman compared with (1) the control participants' age estimates formed without exposure to images of occupations and (2) the control participants' age estimates conditional on their belief about which gender is most common in each occupation. We began by testing the prediction that exposure to online images of occupations primes age-related gender bias in the participants' beliefs. To test this prediction, we evaluated whether uploading an image of a woman (man) for each occupation is associated with a lower (higher) age estimate compared with the average age estimate of each occupation provided by those in the control condition who did not encounter online images of each occupation before providing their estimates. Figure 3a shows that the participants who uploaded an image of a woman estimated the average age of an occupation to be 5.46 years younger than those who uploaded an image of a man (t = −19.07; P = 2.2 × 10−16; Student's t-test), holding occupation constant. Moreover, uploading an image of a woman led the participants to estimate a significantly lower age for each occupation (by 1.75 years) compared with the control participants (t = −11.32; P = 2.2 × 10−16), whereas uploading an image of a man led the participants to estimate a significantly higher age for each occupation (by 0.64 years) compared with those in the control condition (t = 3.42; P = 0.0006; Student's two-tailed t-test). The participants (n = 459) from a nationally representative sample were randomized either to the ‘image' condition, in which they googled for images of occupations (n = 54), or the ‘control' condition, in which they googled for image-based descriptions of random categories (such as apple) unrelated to occupations. a, Average age of each occupation, as estimated by the participants in the image condition, broken down by whether they uploaded a female or male image of the occupation, centred relative to the average age of each occupation provided across all the participants in the control condition. Data points display mean values, and error bars indicate 95% confidence intervals. c, Correlation between the gender association and perceived ideal hiring age of each occupation (averaged across all the participants in the control condition). Data points show the average gender association and perceived ideal hiring age for each occupation according to each measure. Notably, these results hold when controlling for the participants' gender and age, as well as whether their own demographics matched those of the people depicted in the images they uploaded (Supplementary Tables 19 and 20). Supplementary analyses further demonstrate that the participants' estimates of the average ages of people in occupations are significantly correlated with the median age of these occupations according to the US census, indicating that the participants' age judgements were coherent and consistent with ground truth sociodemographic distributions (Supplementary Tables 21 and 22). Specifically, we tested whether the participants in the treatment condition reported younger (older) ages when uploading images of women (men) for each occupation compared with the age estimates provided by the control participants, who reported believing that women (men) most often belonged to a given occupation. Figure 3b shows that the control participants who believed women are most likely to belong to a given occupation also estimated the average age of people in this occupation to be significantly younger (by 2.15 years), evidencing a baseline pattern of age-related gender bias in people's judgements (β[male] = 2.15 years; standard error = 0.38; t = 5.55; P = 2.98 × 10−8). 3b further shows that this age gap is even higher among those in the treatment condition. The participants who uploaded an image of a woman for a given occupation reported estimating the age of people in this occupation to be significantly younger than the control participants who already believed that this occupation is female-skewed. This analysis controls for the specific occupation being evaluated, as well as the participants' idiosyncratic judgements through occupation and participant fixed effects (β[gender × condition] = −0.84; standard error = 0.31; t = −2.69; P = 0.007). These results indicate that exposure to online images significantly exacerbates the perceived age gap between women and men, particularly by increasing the association between women and youth. We conclude these experimental analyses by examining the practical consequences of this age-based gender bias by evaluating its impact on women's and men's perceived fit across occupations. Figure 3c shows that occupations that are more associated with women (men) are significantly correlated with the participants reporting lower (higher) recommended ages for whom to hire in this occupation. Figure 3c shows that the control participants' perceived ideal age for hiring is strongly and positively correlated with the extent to which each occupation is associated with men, as measured by (1) the control participants' manual gender ratings of occupations (r = 0.58; P = 3.52 × 10−6) and (2) the gender associations in the images uploaded by the participants in the image condition (r = 0.45; P = 0.0006; Pearson's two-tailed correlation). These analyses provide evidence that age-related gender associations mediate people's judgements of who is best to hire for a given occupation, with a preference towards hiring younger women and older men. The above results are highly robust to whether (1) the participants provided gender ratings of occupations without estimating age (as captured by a separately replicated experiment6; Supplementary Fig. 17) and (2) the participants rated hireability using a Likert scale (Supplementary Fig. All of our main pre-registered hypotheses were strongly supported. Because popular artificial intelligence tools such as ChatGPT are trained on internet data, we further propose that ChatGPT will exhibit significant age-based gender bias in its textual representations and evaluations of occupations in professional resumes. Identifying age-related bias in ChatGPT would highlight a potential pathway through which this bias is widely propagated, given that more than 400 million people and two million businesses use ChatGPT weekly57. By adapting prompt engineering techniques developed for auditing biases in ChatGPT's resume generation58, we prompted ChatGPT (v.GPT-4o mini) to create nearly 40,000 resumes for 54 occupations using 16 unique female and male names that were normalized to control for name popularity, familiarity, ethnicity and perceived age group, such that the male and female names were maximally similar along these dimensions (the same names were used in a recent auditing study58) (see ‘Prompt design' in Methods). This experiment consisted of two phases: resume generation and resume evaluation (Extended Data Fig. 6). All resumes were generated and evaluated in June 2024. We used the same occupations in our image search experiment. In the treatment condition, we prompted ChatGPT 20 separate times for each name–gender–occupation prompt combination, yielding a total of 34,560 resumes and 17,280 resumes for each gender group. The resume features generated by ChatGPT were highly coherent and stable (Supplementary Fig. Next, we evaluated the consequences that age-based gender biases in ChatGPT's representations of resumes can have on its practical application as a hiring tool. We focused on one of ChatGPT's most popular uses in the workplace: to evaluate, score and rank resumes to expedite hiring processes by focusing human recruiters on resumes with top scores58. We prompted ChatGPT to evaluate each of the resumes generated in the first phase by providing a score between 1 and 100 to indicate the quality of each resume. All reported results are robust to altering the model temperature of ChatGPT (Supplementary Fig. We began by examining how altering the gender of a target applicant's name affects the resumes that ChatGPT generates. We compared the resumes that ChatGPT generated in the treatment condition for female or male names while using a linear regression to control for the applicant name and occupation. 4a indicates, when ChatGPT generated a resume for a female name, it generated resumes with significantly lower ages (by 1.6 years; t = 20.5; P = 7.09 × 10−93), more recent graduation dates (by 1.3 years; t = 12.5; P = 1.18 × 10−35) and fewer years of relevant experience (by 0.92 years; t = 5.39; P = 6.97 × 10−8) compared with male names (Student's t-test; n = 34,560 resumes). Compared with the control condition, the resumes ChatGPT generated for female (male) applicants were significantly younger (older) and less (more) experienced than the resumes generated for the same occupations without any gender or name (all at the P < 0.00001 level; Student's two-tailed t-test). Thus, ChatGPT exhibits age-based assumptions about women and men that are highly consistent with stereotypical associations relating to gendered ageism. a, Partial effect plot in an ordinary least squares regression displaying the effect of a male applicant name (versus a female applicant name) on (1) applicant age; (2) years since the applicant's graduation; and (3) the number of years of applicant's relevant experience, while controlling for name and occupation. Only resumes from the treatment condition were examined in this analysis (n = 34,560), because this ensures that all resumes have either a male or female name and were produced through the same prompt. b, Linear correlation between applicant age and ChatGPT's rating of resume quality across all resumes (n = 39,560) from all conditions. Data points display the raw distribution of scores for each resume, with one data point per resume. c, Partial effect plot displaying the interaction effect between applicant age and applicant gender on ChatGPT's rating of resume quality, with fixed effects for applicant name, occupation and phase 1 condition (data from the control condition were excluded because of the lack of applicant gender; n = 37,060 resumes used in total). ChatGPT's temperature was set to its default value of 0.7. These patterns of age-based gender bias in ChatGPT were replicated in the control–gender condition, in which ChatGPT generated its own name and gender classification for each occupation. When ChatGPT generated a male applicant for a given resume, this applicant was more likely to be older (by 1.3 years; t = 17.3; P = 2.2 × 10−16) and to have graduated less recently (by 1.2 years; t = 7.10; P = 2.29 × 10−12) than when it generated a resume for a female applicant, holding occupation constant (Student's t-test; n = 2,500 resumes). We next evaluated the consequences this bias had on how ChatGPT evaluated the quality of resumes. 4b shows that ChatGPT's judgements of resume quality were significantly and positively correlated with the age of the applicant that ChatGPT initially generated for the resume (r = 0.27; P = 2.2 × 10−16; t = 51.59; Pearson's two-tailed correlation; n = 39,560 resumes). This result equally holds in a linear regression when we controlled for the occupation and name used to generate each application (β[age] = 0.04; t = 6.3; P = 2.96 × 10−10). Finally, we tested whether ChatGPT exhibits a preference not only for older applicants but also for older men specifically, consistent with the predictions of gendered ageism6,7. We used linear regression to predict ChatGPT's judgements of resume quality using an interaction between the age and gender of the applicant while holding occupation and applicant name constant. 4c, the model identified a highly significant and positive interaction between being male and older, indicating that the benefit of older age on ChatGPT's judgements of resume quality is even greater if the applicant is male rather than female (β[male × age] = 0.04; t = 6.61; P = 3.66 × 10−11). This interaction effect is robust to altering ChatGPT's model temperature (Supplementary Fig. In this study, we have provided large-scale evidence that age-related gender bias pervades online media, including images, videos and texts across major platforms, and that the bias towards representing women as younger distorts ground truth realities on the actual ages of women and men throughout society. Our findings raise an alarm about the algorithmic amplification of age-related gender bias on the internet, especially considering that many mainstream machine learning algorithms are trained on these public datasets. Many of the image and text datasets examined in this study are used extensively as canonical training and benchmark datasets for developing artificial intelligence applications. Enormous harm can be caused by latent social biases that lurk in popular machine learning tools59,60, and algorithmic bias typically arises from contaminated training data. Our study provides direct evidence that age-related gender bias is amplified by two of the most widely used algorithms today: the Google Image search engine and ChatGPT. Although companies such as Google and OpenAI invest heavily in reducing stereotypical content in their products61, most studies focus on single dimensions of bias, such as gender-based or race-based biases. Our study highlighted the critical need to account for multimodal and multidimensional forms of bias62, which are more challenging to detect but not less consequential in how people and algorithms represent the social world. The intersectional statistical bias we identified between gender and age may interact with other biases, such as relating to how women and men are depicted in terms of warmth and competence, revealing a promising direction for future research63,64. How might the digital distortion of age-related gender associations negatively affect women and men? Our results highlighted several key ways in which older women are likely to be disadvantaged by this bias. For example, when generating resumes, ChatGPT not only assumes that women are younger, but also that they have less overall experience. Consequently, ChatGPT is biased towards giving lower scores to resumes from younger women compared with older women while giving the highest scores to older men. Yet, ChatGPT also gives higher scores to resumes from young women than from young men, suggesting that young men may also be disadvantaged by this dual bias (Supplementary Fig. 18). However, a selection bias favouring younger women and older men may further reinforce gender inequalities at the systemic level, whereby women are preferentially hired into roles with lower status and authority but denied mobility, whereas older men continue to enjoy top positions. This resonates with our finding that online content is most likely to depict men as older than women for occupations with higher status and wealth. A critical direction for future research is to investigate the causal mechanisms through which age-related gender bias seeps into and spreads through the images, videos and text of distinct platforms, each with its own unique audiences and distribution channels. Our results about objective differences in the ages of male and female celebrities visualized on IMDb, Wikipedia and Google probably reflect industry-specific mechanisms related to status dynamics, hiring biases and the objectification of women in entertainment media. Yet, these industry-specific drivers do not account for how strongly women and youth are semantically associated in massive bodies of online text from diverse sources, let alone in ChatGPT's text-based representations and rankings of job candidates. A fascinating question for future work is to explore whether the aesthetic norms and hiring biases of entertainment media spill over into the distortion of age–gender associations throughout social life. A related question concerning supply-side factors concerns whether age-related gender bias in popular algorithms stems from inequalities in the gender of data contributors online. Studies suggest that Reddit users65 and Wikipedia editors66 are disproportionately male, and textual data from these platforms are frequently mined for training artificial intelligence models. Training artificial intelligence on datasets with greater gender equality in data contributors may provide an effective mitigation strategy. This study highlights the increasingly prominent role of internet culture and algorithms in mediating our representation of the social world. As a recent review article emphasized67, previous studies have been limited in their ability to concretely measure the diverse cultural meanings of age both in terms of its biological basis and its relevance to cultural notions of life stages (such as ‘youth' and ‘childhood'). The strength of our approach is that it captures the cultural meanings of age and gender along many dimensions, from concrete time-stamped images to verbal descriptions of age categories across social roles and contexts. We revealed that the cultural meanings of gender and age are deeply linked on a massive scale across information modalities and in ways that reflect socioeconomic inequalities. Compelling evidence that these patterns are socially constructed comes from our finding that online associations between gender and age heavily distort the measurable ground truth reality of how people of different genders and ages are distributed throughout society. The extent to which algorithms entrench distortions of social reality en masse and to which this can be corrected is a vital topic for future research on internet policy and human cultural evolution. The methods we propose for measuring widespread stereotyped representations online and for grounding them in verifiable sociodemographic realities mark a crucial step in the fight against pervasive cultural inequalities, both online and beyond. In this section, we detail the methods used in all parts of our analyses, including our observational comparisons of gender and age bias in online images, videos and texts, as well as our Google Image search experiment and our resume audit of ChatGPT. The pre-registration for our online image search experiment is available at https://osf.io/x9scm. This experiment was a successful replication of a previous study with a nearly identical design; the pre-registration of this previous study is available at https://osf.io/2b58d. This study was approved by the ethics review board at the University of California, Berkeley, where this study was conducted. In what follows, we describe our observational methodology for collecting and analysing large bodies of images, videos and text online. With regard to the crowdsourcing methods applied to analysing our main Google and Wikipedia Image datasets, many of the methods described below (including the procedure and demographic details of the coder population) were reproduced from the original data collection summary provided as part of the first publication of these datasets6. In addition to this reproduced description, we include information on how age classifications of these images were collected, because this feature was not explored or discussed as part of the original publication of these datasets6. Our crowdsourcing methodology consisted of four steps. We began by identifying all social categories in WordNet69, a canonical lexical database of English. WordNet captures 3,495 social categories, including occupations (such as doctor) and generic social roles (such as neighbour). We then gathered online images associated with each social category from Google and Wikipedia. Next, we applied the OpenCV deep learning module in Python to automatically extract the face from each image. Cropping faces helped us ensure that each face in each image was separately classified in a standardized manner while avoiding subjective biases in coders' decisions for which face to focus on and categorize in each image. Finally, we hired 6,392 human coders from Amazon's Mechanical Turk to manually classify the gender of the faces. Each face was classified by three unique annotators (as per established methodology6,70,71), so that the gender of each face (‘male' or ‘female') could be identified on the basis of the majority (modal) gender classification across three coders. (We also gave coders the option of labelling the gender of faces as ‘non-binary', but this option was only chosen in 2% of cases. Therefore, we excluded these data from our main analyses and recollected all classifications until each face was associated with three unique coders using either the male or female label.) Although coders were asked to label the gender of the faces presented, our measure was agnostic to which features the coders used to determine their gender classifications. They may have used facial features and features relating to the aesthetics of expressed gender, such as hair or accessories. In terms of age, each face was classified as belonging to one of the following age bins (the ordinal ranking of each bin is indicated in parentheses): (1) 0–11, (2) 12–17, (3) 18–24, (4) 25–34, (5) 35–54, (6) 55–74 and (7) 75+. Because the greater number of classification options for age led to fewer images associated with a majority-preferred age classification, we identified the age of each face by taking the average of the ordinal age bin judgements across the three coders. Each search was implemented from a fresh Google account with no previous history. Searches were run in August 2020 by ten distinct data servers in New York City. This study was approved by the institutional review board at the University of California, Berkeley, where this part of the study was conducted. To collect images from Google, we followed a previous study by retrieving the top 100 images that appeared when using each of the 3,495 categories to search for images using the public Google Images search engine. (Google provides roughly 100 images for its initial search results.) Across the non-gendered and gendered searches, 3,489 categories could be associated with images containing faces in the Google Image search engine (specifically, 3,434 categories for the non-gendered searches and 2,960 for the gendered searches). To collect images from Wikipedia, we identified the images associated with each social category in the 2021 Wikipedia-based Image Text (WIT) Dataset72. WIT maps all images over Wikipedia to textual descriptions on the basis of the title, content and metadata of the active Wikipedia articles in which they appear. We were able to associate 1,251 social categories from WordNet with images in WIT (across all English articles) that supported stable classification as human faces with detectable ages, according to our coders. The coders identified 18% of images as not containing human faces, and these were removed from our analyses. using the following options: ‘fish', ‘up', ‘monk' and ‘apple'. We removed the data from all annotators who failed an attention check (15%) and continued collecting classifications until each image was associated with the judgements of three unique coders, all of whom passed the attention check. The human coders were all US-based adults fluent in English. Supplementary Table 3 indicates that our main results are robust to controlling for the demographic composition of our human coders. To measure age-related gender bias in online images and videos, we analysed a range of open-source datasets collected either for social science research or for training face recognition algorithms, none of which examined or reported correlations between the gender and age of the people depicted. In total, we examined more than one million images from five main online sources: Google, Wikipedia, IMDb, Flickr and YouTube, as well as the Common Crawl (created by randomly scraping content from across the world-wide web), each with distinct ways of sourcing and aggregating data. We measured gender and age using a variety of techniques, including human judgements, machine learning and ground truth data on the self-reported gender and true time-stamped age of the people depicted. Our statistical analyses did not control for multiple comparisons because all tests were theoretically guided and did not involve an agnostic permutation over a set of pairwise comparisons. Although we examined many datasets, our main analyses examined a single correlation (between gender and age) within each dataset separately. We now describe each of these datasets. First, we used large-scale crowdsourcing to identify age-related gender bias in a new dataset of images from Google and Wikipedia (which was originally collected for a recently published study that did not examine age-related classifications)6. These categories include occupations (such as ‘physicist') and generic social roles (such as ‘colleague'). For each category, this dataset contains the top 100 images that appear in Google Images when searching for (1) the category on its own (such as ‘doctor'); (2) the female version of the category (such as ‘female doctor'); and (3) the male version of the category (such as ‘male doctor'). Altogether, this yielded 657,035 unique images containing faces from Google. Searches were run from ten distinct data servers in New York City. Because Google is known to customize search results on the basis of the location from which the search is run72, we show that our results are robust to replicating this data collection pipeline while collecting Google Images from six distinct cities around the world (Supplementary Fig. This dataset also leveraged human coders to classify the age and gender of faces in Wikipedia images associated with as many WordNet social categories as possible in the 2021 WIT Dataset68. WIT maps all images over Wikipedia to textual descriptions on the basis of the title, content and metadata of the active Wikipedia articles in which they appear. WIT includes images of 1,251 social categories from WordNet across all English Wikipedia articles, in total yielding 14,709 faces. We hired 6,392 human annotators from Amazon's Mechanical Turk to classify the gender and age of the faces in these images. Each face was classified by three unique annotators6,70,71 so that the gender of each face (male or female) could be identified on the basis of the majority gender classification across three coders. (We also gave coders the option of identifying the gender of faces as non-binary, but this option was chosen in less than 2% of cases. Therefore, we excluded these data from our main analyses.) In terms of age, each face was classified as belonging to one of the following age bins (in years): (1) 0–11, (2) 12–17, (3) 18–24, (4) 25–34, (5) 35–54, (6) 55–74 and (7) 75+. Because the greater number of classification options for age led to fewer images with a majority-preferred classification, we identified the age of each face by taking the average of the ordinal age bin judgements across the three coders (our main results hold when using the modal age judgement; Supplementary Fig. 4). Our findings continued to hold when controlling for annotator demographics and intercoder agreement, which was high in our sample (Supplementary Fig. We also conducted a separate validation task, in which the true gender and age of the faces being classified were known. The results indicate that our coders exhibited reliable and accurate gender and age judgements, with no biases as a function of gender (Supplementary Tables 4 and 5). Sensitivity tests further showed that even if our coders were hypothetically biased in their ability to estimate age as a function of gender, this would not disrupt the statistical significance or directionality of our findings (Supplementary Fig. We extended our findings by examining age-related gender bias in two large corpora of online images collected from three main websites (IMDb, Wikipedia and Google) for which the self-identified gender and true age of the faces were objectively inferred. This extension allowed us to examine whether women are objectively younger than men in online images, without depending on age predictions from human coders or machine learning algorithms. The first corpus was the 2018 IMDb–Wiki dataset43, which consisted of more than half a million images of celebrities from IMDb and Wikipedia on the basis of those depicted in the top 100,000 most visited IMDb pages. Each image in this dataset was time-stamped for when the photograph was taken, allowing the age of each face to be inferred on the basis of the celebrity's date of birth, which is publicly available through their open profile on IMDb and Wikipedia. The creators of CACD collected time-stamped images by using Google Image search to retrieve images associated with each celebrity from 2004 to 2013 (for example, by searching ‘Emma Watson 2004' through ‘Emma Watson 2013'). All images from both datasets containing ages below 0 and above 100 were removed to maximize data quality. Each dataset identified the exact age of the celebrities at the time they were depicted in each photograph by determining the date of birth and gender of each celebrity on their public IMDb and Wikipedia pages and then by comparing this information to the time-stamped date of when each photograph was taken. Finally, we examined images from four publicly available training datasets widely used to train automated face recognition algorithms. In these canonical datasets, the gender and age classifications were on the basis of a combination of automated machine learning classifications and verification through human annotation. This includes the 2017 UTK dataset46 consisting of 20,000 images scraped randomly from across the world-wide web using search engines and public repositories, the 2014 Adience dataset47 consisting of 26,580 images randomly sampled from Flickr, a public image-based social media platform, and the 2008 LFW48 dataset consisting of 13,233 images randomly scraped from online news websites. Finally, we examined images of faces extracted from screenshots of YouTube videos using two datasets. The first was the 2011 YouTube Faces dataset50 consisting of 3,425 YouTube videos and 3,645 images of celebrities. The second one was the 2022 CelebV-HQ51 dataset consisting of 35,666 images formed by identifying public lists of celebrities on Wikipedia and automatically collecting the top 10 YouTube videos associated with each celebrity. We were able to match 867 social categories from our main Google image (Fig. 1a) dataset to occupational categories in the US census. The US Bureau of Labor Statistics recently released a breakdown of the median age of each gender, from 2019 to 2023, across five industries: sales, services, natural resources and construction, production and transportation and management. The census assigns each occupation to one of these industries, allowing those occupations matched in our Google image dataset to be assigned a census industry. We estimated the relationship between gender and age at the industry level by averaging the age associations in Google Images across all occupations within a given industry (averaged within each occupation and then across occupations at the industry level). Supplementary Tables 8 and 9 present the robustness of our results to a range of statistical controls. Each participant evaluated 20 randomly sampled occupations from a broader set of 867 WordNet social categories that could be matched with corresponding occupations in the US census. Through randomization, each category was evaluated by 27 unique participants on average (minimum of 15 participants). For each occupation, the participants rated (1) its status using the following scale (How would you rate the social status of someone belonging to this occupation? −2, very negative; −1, negative; 0, neutral; 1, positive; 2, very positive) and (2) its prestige using the following scale (To what extent do you agree that it is prestigious to belong to this occupation? We also asked the participants to rate the status/prestige through the standard question from the general social survey, which asked them to place occupations on a ladder containing 10 rungs, where the bottom rung indicates occupations with very low status, income, education and prestige, whereas the highest rung indicates occupations with very high status, income, education and prestige (Supplementary Fig. 11). The participants' answers across all three questions were highly correlated (all paired Pearson's correlations above 0.85; Supplementary Fig. 9). 4, we first averaged all participants' judgements of each occupation across the (1) status and (2) prestige question and then assigned each occupation a single status score by taking the mean of its average status and prestige score. In the Supplementary Information, we show that all of our results hold when examining each question separately and when examining the participants' judgements using the standard social status question from the General Social Survey (GSS) (Supplementary Fig. Note that Prolific's nationally representative sample of the US population size allows for a maximum of 800 participants. However, this sample size was not large enough to gain sufficiently powered judgements across all 867 occupational categories; therefore, an extra sample of US participants was recruited until all occupations reached a minimum of 15 evaluations from independent participants. All results are robust to a range of statistical controls (Supplementary Tables 10–13). To measure age-related gender bias in large bodies of internet text, we leveraged word embedding models trained on massive amount of internet data. These models were designed to construct a high-dimensional vector space on the basis of the co-occurrence of words (for example, whether two words appear in the same sentence), such that words with similar meanings are closer in this vector space. Technically, these embedding spaces also capture higher-order similarities on the basis of whether words co-occur in similar linguistic contexts (that is, in association with related sets of words), without requiring words to directly appear together. We harnessed recent advances in natural language processing to extract demographic dimensions in word embedding models that capture the extent to which existing demographics underlie the cultural connotations of categories. We identified both gender and age dimensions. We briefly describe this methodology below. Word embedding models leverage the frequency of word co-occurrences in text to position words in an n-dimensional space such that words that frequently co-occur together are more closely located in this n-dimensional space. The ‘embedding' for a given word identifies the specific position of this word in this n-dimensional space. The cosine distance between word embeddings in this n-dimensional space provides a robust measure of semantic similarity that captures the similarity of the semantic contexts in which words appear6. To extract a gender dimension in word embedding space, we harnessed the ‘geometry of culture' method of Kozlowski et al.73. This method was originally developed for static embedding models such as Word2Vec and GloVe; therefore, we incorporated key adjustments that enable its application to contextualized embeddings through generative transformer models such as GPT-2 Large. We identified two clustered regions in the word embedding space corresponding to conventional representations of females and males. Specifically, the female cluster consisted of ‘woman', ‘her', ‘she', ‘female' and ‘girl', whereas the male cluster consisted of ‘man', ‘his', ‘he', ‘male' and ‘boy'. For each social category in WordNet, we calculated the average cosine distance between this category and both the female and male clusters. Each category was associated with two numbers: its cosine distance with the female cluster (averaged across its cosine distance with each term in the female cluster) and its cosine distance with the male cluster (averaged across its cosine distance with each term in the male centroid). Taking the difference between the cosine distance of a category with the female and male centroids allowed each category to be positioned along a −1 (female) to 1 (male) scale in the embedding space. Although we recognize that gender is fundamentally non-binary, we built upon a previous study that leveraged this binary framework73 to identify biases in the extent to which people associate concepts with men or women. The issue with applying this approach to contextualized embeddings is that the embedding associated with an individual word can be sharply different from the embedding associated with this word within a larger context, for example, within a surrounding sentence. For this reason, we modified the geometry of culture method by creating male and female poles consisting of many parallel sentences that vary only in whether they mention the corresponding male or female version of a pronoun. For example, the male pole consists of sentences such as ‘he is a boy' and ‘his hobbies are very masculine', whereas the analogues of these sentences in the female pole are ‘she is a girl' and ‘her hobbies are very feminine'. Fifty sentences were used to form each pole. All sentences used are provided in Supplementary Tables 14 and 15. We conducted key robustness tests to verify the validity of our methods and the robustness of our results to the use of different sentences along the gender pole (Supplementary Fig. 13). In our supplementary analyses involving static embedding models, we used the original geometry of culture approach. We used this same approach to construct an age dimension in word embedding models. For static embedding models, we identified two clustered regions in the word embedding space corresponding to younger and older ages. All results are highly robust to increasing the number of words used to construct this age dimension. For example, our results replicate when defining the younger cluster using the words ‘young', ‘youth', ‘childhood', ‘child', ‘baby', ‘infant', ‘teen', ‘teenager' and ‘adolescent', as well as when defining the older cluster using the words ‘old', ‘elder', ‘elderly', ‘adulthood', ‘adult', ‘senior', ‘parent', ‘retired' and ‘aged'. We used the same technique to sort categories along a −1 (young) to 1 (old) scale in the embedding space. Similarly, to examine age associations in contextualized embedding models, we generated 50 sentences that hold everything constant while varying whether the age term involved indicates a young or old age (see Supplementary Table 15 for a full list of the age sentences used to create the contextualized age pole). In all cases examining static models, to compute the distances between the vectors of social categories represented by bigrams (such as ‘professional dancer'), we used the Phrases class in the gensim Python package, which provided a pre-built function for identifying and calculating distances for bigram embeddings. This method works by identifying an n-dimensional vector of middle positions between the vectors corresponding separately to each word in the bigram (for example, ‘professional' and ‘dancer'). This method is not necessary in contextual language models, which provide unique embeddings for n-grams as distinct from their component words. Once the corresponding demographic dimensions were constructed for each model, we evaluated the correlation between gender and age associations across 3,495 social categories from WordNet (the same categories examined in our image analyses above). To simplify the presentation of how this gender and age dimensions are correlated, we used min-max normalization to convert the gender dimension into a 0 (female) to 1 (male) association, which, in effect, represents the extent to which each category carries male associations relative to all other categories. We applied the same approach to produce a normalized 0 (young) to 1 (old) dimension, which captures the extent to which each category is associated with older ages relative to all other categories. The supplementary analyses showed that our results are highly robust to varying our technique for constructing the age and gender dimensions (Supplementary Fig. In the main text, we present our results while analysing the largest open-source large language model from OpenAI (GPT-2 Large56), for which word embeddings can be robustly and transparently extracted and examined. GPT-2 Large is one of the largest and most popular open-source language models, trained on billions of words from the 2019 WebText dataset, which primarily comprises Reddit data and the diverse web content (including articles and books) to which these Reddit data are linked. In the supplementary analyses, we showed that these results replicate when examining a wide range of models, including Word2Vec, GloVe, BERT, FastText, RoBERTa and GPT-4, all of which vary in their dimensionality and data sources, as well as the year in which their training data were collected, ranging from 2013 to 2023. We focus our main results on GPT-2 Large, not only because of its scale and popularity, but also because its open-source nature allows us to transparently access and analyse its word embeddings. GPT-4, by contrast, is a closed-source model that relies on using OpenAI's private application programming interface, which limits the interpretability of our method. Nevertheless, supplementary analyses showed that our results replicate when examining this closed-source model (Supplementary Figs. We invited a nationally representative sample of participants (n = 500) from Prolific. Prolific is a popular online panel for social science research that provides prescreening functionality specifically for recruiting a nationally representative sample of the USA along the dimensions of sex, age and ethnicity. Our sample size was selected to emulate the sample size of a recent experiment with a highly similar design, which effectively measured statistically powered outcomes6. There was an attrition rate of 9.2% of participants (which is within the common range of attrition for online experiments), such that 459 participants completed the task. Our results only examined data from the participants who completed the experiment to ensure data quality. All the participants provided informed consent before participating. This experiment was run on 10 November 2023. 4 presents a schematic of the full experimental design. This experiment was approved by the Institutional Review Board at the University of California, Berkeley. In this experiment, the participants were randomized to one of two conditions: (1) the image condition (in which they used the Google Image search engine to retrieve images of occupations) and (2) the control condition (in which they used the Google Image search engine to retrieve images of random, non-gendered categories, such as ‘apple'). The participants in the image condition were also asked to rate their willingness to hire the person depicted in their uploaded image (Supplementary Fig. 18). After uploading a given random image, the control participants were then asked to estimate the average age of someone in a randomly selected occupation from the same set. The control participants were also asked to rate the ideal hiring age of someone in each occupation, as well as which gender (male or female) was most likely to belong to each occupation. This design allowed us to evaluate the treated participants' age estimates after uploading an image of a man or woman compared with (1) the control participants' age estimates that were formed without exposure to images of occupations and (2) the control participants' age estimates conditional on which gender they think is most common in each occupation. All participants regardless of condition completed this sequence for 22 occupations (randomly sampled from a set of 54 occupations held constant across conditions). All participants were compensated at the rate of US$15 per hour for their time. We used the exact list of occupations leveraged in a recent experiment of a similar design (focusing only on gender, not age) to compare results and evaluate replicability6. Here is the full pre-registered set: immunologist, mathematician, harpist, painter, piano player, aeronautical engineer, applied scientist, geneticist, astrophysicist, professional dancer, fashion model, graphic designer, hygienist, educator, intelligence analyst, logician, intelligence agent, financial analyst, chief executive officer, clarinetist, chiropractor, computer expert, intellectual, climatologist, systems analyst, programmer, poet, astronaut, professor, automotive engineer, cardiologist, neurobiologist, English professor, number theorist, marine engineer, bookkeeper, dietician, model, trained nurse, cosmetic surgeon, fashion designer, nurse practitioner, art teacher, singer, interior decorator, media consultant, art student, dressmaker, English teacher, literary agent, social worker, screen actor, editor-in-chief and schoolteacher. Include the following sections: date of birth in month/day/year format, education with graduate date in month/day/year format, total number of years of relevant experience in numeric format and a numbered list of relevant skills'. This prompt differs from that of a previous study by prompting ChatGPT to include the applicants' age and years of experience. We prompted ChatGPT 50 separate times for each occupation, resulting in 2,700 unique resumes in the control condition. This led to the following updated prompt: ‘Write a detailed and concise resume for someone applying for a position in the role of [occupation] in the United States. Include the following sections: gender of the applicant (male or female), date of birth in month/day/year format, education with graduate date in month/day/year format, total number of years of relevant experience in numeric format and a numbered list of relevant skills'. We prompted ChatGPT 50 separate times for each gender–occupation prompt combination, yielding 2,700 unique resumes in this condition. We used 16 female names and 16 male names, all normalized in terms of popularity, familiarity, ethnicity and perceived age group, such that the male and female names were maximally similar along these dimensions. These names were developed and deployed by Armstrong et al.58 for a related auditing task that did not involve requesting explicit age information to examine age-related gender bias. The female and male names validated by Armstrong et al.58 included four names for the following ethnic identities: Hispanic, Asian, white people and Black people (four names per gender per ethnicity yielded 16 names for each gender). Write a detailed and concise resume for [name]. Include the following sections: date of birth in month/day/year format, education with graduate date in month/day/year format, total number of years of relevant experience in numeric format and a numbered list of relevant skills'. We prompted ChatGPT with this request 20 separate times for each name–gender–occupation prompt combination, yielding 34,560 resumes in total and 17,280 resumes for each gender group. This excludes the rare cases where ChatGPT failed to produce a resume (for example, by generating an empty JavaScript Object Notation structure without any resume-related information). In the resume evaluation phase, we prompted ChatGPT to evaluate each resume from the resume generation phase by providing a score between 1 and 100 to indicate the quality of each resume. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data associated with this study can be downloaded from GitHub at https://github.com/drguilbe/distortion_age_gender_online/. The pre-registration for our experiment is available at https://osf.io/x9scm. All codes associated with this study can be downloaded from GitHub at https://github.com/drguilbe/distortion_age_gender_online/. Madon, S., Guyll, M., Hilbert, S. J., Kyriakatos, E. & Vogel, D. L. Stereotyping the stereotypic: when individuals match social stereotypes. Jussim, L. et al. in Handbook of Prejudice, Stereotyping, and Discrimination 2nd edn (ed. Puddifoot, K. How Stereotypes Deceive Us (Oxford Univ. Bai, X., Fiske, S. T. & Griffiths, T. L. Globally inaccurate stereotypes can result from locally adaptive exploration. Guilbeault, D. et al. Online images amplify gender bias. Itzin, C. & Phillipson, C. in Gender, Culture and Organizational Change (eds Itzen, C. & Newman, J.) Older Women: Inequality at the Intersection of Age and Gender https://data.unwomen.org/publications/older-women-inequality-intersection-age-and-gender (United Nations, 2022). Diehl, A., Dzubinski, L. M. & Stephenson, A. L. Women in leadership face ageism at every age. Allport, G. W. The Nature of Prejudice (Addison-Wesley, 1954). Tajfel, H. in Intergroup Relations: Essential Readings (eds Hogg, M. A. Fiske, S. T. & Tablante, C. B. in APA Handbook of Personality and Social Psychology, Volume 1: Attitudes and Social Cognition (eds Mikulincer, M. et al.) 457–507 (American Psychological Association, 2015). Measuring Intelligence: Facts and Fallacies (Cambridge Univ. A. et al. National differences in gender–science stereotypes predict national sex differences in science and math achievement. Global Report on Ageism www.who.int/publications-detail-redirect/9789240016866 (Demographic Change and Healthy Ageing & World Health Organization, 2021). Spedale, S., Coupland, C. & Tempest, S. Gendered ageism and organizational routines at work: the case of day-parting in television broadcasting. Duncan, C. & Loretto, W. Never the right age? Gender and age-based discrimination in employment. Ramati-Ziber, L., Shnabel, N. & Glick, P. The beauty myth: prescriptive beauty norms for women reflect hierarchy-enhancing motivations leading to discriminatory employment practices. MacArthur, H. J., Cundiff, J. L. & Mehl, M. R. Estimating the prevalence of gender-biased language in undergraduates' everyday speech. Ridgeway, C. L. in Framed by Gender: How Gender Inequality Persists in the Modern World (ed. Shmerling, R. Why men often die earlier than women. Why U.S. women now live 6 years longer than men. & North, M. S. Agentic but not warm: age-gender interactions and the consequences of stereotype incongruity perceptions for middle-aged professional women. Martin, A. E., North, M. S. & Phillips, K. W. Intersectional escape: older women elude agentic prescriptions more than older men. Word embeddings quantify 100 years of gender and ethnic stereotypes. & Skiena, S. Stereotypical gender associations in language have decreased over time. National hiring experiments reveal 2:1 faculty preference for women on STEM tenure track. Fiske, S. T. Prejudices in Cultural Contexts: Shared stereotypes (gender, age) versus variable stereotypes (race, ethnicity, religion). & Davy, D. Gendered ageism: older women's experiences of employment agency practices. Lincoln, A. E. & Allen, M. P. Double jeopardy in Hollywood: age and gender in the careers of film actors, 1926–1999. The importance of key celebrity characteristics for customer segmentation by age and gender: does beauty matter in professional football? Edström, M. Visibility patterns of gendered ageism in the media buzz: a study of the representation of gender and age over three decades. The spontaneous formation of stereotypes via cumulative cultural evolution. & Centola, D. Experimental evidence for scale-induced category convergence across populations. Mastroianni, A. M. & Gilbert, D. T. The illusion of moral decline. Rothe, R., Timofte, R. & Van Gool, L. Deep expectation of real and apparent age from a single image without facial landmarks. Porter, C. & Serra, D. Gender differences in the choice of major: the importance of female role models. & Qi, H. Age progression/regression by conditional adversarial autoencoder. Eidinger, E., Enbar, R. & Hassner, T. Age and gender estimation of unfiltered faces. Huang, G. B., Ramesh, M., Berg, T. & Learned-Miller, E. Labeled Faces in the Wild: A Database for Studying Face Recognition in Unconstrained Environments https://inria.hal.science/inria-00321923v1/preview/Huang_long_eccv2008-lfw.pdf#page=2 (University of Massachusetts and Stony Brook University, 2008). & Zuckerman, E. Dialing for videos: a random sample of YouTube. Face recognition in unconstrained videos with matched background similarity. Zhu, H. et al. CelebV-HQ: a large-scale video facial attributes dataset. In Computer Vision – ECCV 2022 (eds Bertino, E. et al.) 650–667 (Springer, 2022). Ridgeway, C. L. & Diekema, D. in Gender, Interaction, and Inequality (ed. Ridgeway, C. L. Why status matters for inequality. Language models are unsupervised multitask learners. Kant, R. OpenAI's weekly active users surpass 400 million. Armstrong, L., Liu, A., MacNeil, S. & Metaxa, D. The silicon ceiling: auditing GPT's race and gender biases in hiring. 4th ACM Conference on Equity and Access in Algorithms, Mechanisms, and Optimization 1–18 (Association for Computing Machinery, 2024). An empirical study of apparent gender-based discrimination in the display of STEM career ads. Obermeyer, Z., Powers, B., Vogeli, C. & Mullainathan, S. Dissecting racial bias in an algorithm used to manage the health of populations. Sambasivan, N. Moving toward a gender equitable internet. & Banaji, M. R. Extracting intersectional stereotypes from embeddings: developing and validating the Flexible Intersectional Stereotype Extraction procedure. Fiske, S. T. Stereotype content: warmth and competence endure. Smiling women pitching down: auditing representational and presentational gender biases in image-generative AI. 6% of online adults are reddit users. 7th International Symposium on Wikis and Open Collaboration 11–14 (Association for Computing Machinery, 2011). The social construction of age: concepts and measurement. Srinivasan, K., Raman, K., Chen, J., Bendersky, M. & Najork, M. WIT: Wikipedia-based image text dataset for multimodal multilingual machine learning. 44th International ACM SIGIR Conference on Research and Development in Information Retrieval 2443–2449 (Association for Computing Machinery, 2021). & Vogel, C. The topology of WordNet: some metrics. Unequal representation and gender stereotypes in image search results for occupations. An image of society: gender and racial representation and impact in image search results for occupations. Vlasceanu, M. & Amodio, D. M. Propagation of societal gender inequality by internet search algorithms. We acknowledge the Haas MORS group for their helpful comments on this project. We also thank T. Hull for his assistance with data collection. This project was partially funded by grants from the Fisher Center for Business Analytics and the Center for Equity, Gender & Leadership, awarded to D.G. and S.D., as well as the Barbara and Gerson Bakar Fellowship awarded to D.G., all through the University of California, Berkeley. Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors declare no competing interests. Nature thanks April H. Bailey and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Female representation is measured by the percentage of women employed in an occupation. Panel (A) shows the raw data (with each data point showing a single occupation) for 2012 (the correlation is non-significant; r = −0.021, CI = [−0.12, 0.08], p = 0.70, Pearson Correlation, two-tailed, n = 536 occupations). Panel (B) shows the raw data (with each data point showing a single occupation) for 2023 (the correlation is non-significant; r = −0.046, CI = [−0.14, 0.05], p = 0.36, Pearson Correlation, two-tailed, n = 594 occupations). For all census years for which this data is provided in this format (from 2011 to 2023), there is not a single year with a statistically significant correlation between the fraction of women in an occupation and its associated median age (Table S1). Comparing the average age of women and men across industries in the U.S. Census (from 2019 to 2023) to the average perceived age of people in occupations from these same industries according to Google Images. The midpoint of the age gap for each industry and data source is centered at 0 to help visually compare the magnitude of the age gap across datasets for each industry. Bold lines indicate cases where men are associated with a higher age than women for a given data source in a given industry; dotted lines indicate cases where women are older than men. The age gap for occupations in Google Images is predicted by the perceived status of occupations, as well as by the median yearly earnings of occupations and the gender pay gap by occupation according to U.S. Census data from 2015 to 2022. Google image data is from Guilbeault et al. (2024; see Fig. 1a) and is based on 866 social categories matched to occupations in the U.S. census. In all panels, data points are presented as mean values and confidence intervals display 95% confidence intervals. (A) The correlation between the perceived status of an occupation and the probability that men appear older than women in Google images of the occupation (status perceptions are averaged across a nationally representative U.S. sample, n = 1,002 participants; an average of 27 participants rated each of occupations; data shown in six evenly spaced bins). Examples of occupations in the lowest (highest) 5% of perceived social status according to this measure are provided at the bottom of the figure. (B) The correlation between the U.S. Bureau of Labor Statistics' measures of occupational prestige (shown in quartiles) and the probability that men appear older than women in Google images of the occupation (532 occupations could be matched). (C) The logged median yearly earnings for an occupation (shown in quartiles) predict the probability that men appear older than women in Google images of the occupation. (D) The pay gap in median earnings for an occupation by gender (shown in quartiles) predicts the age gap in perceived age between men and women in Google images of the occupation. For (B) and (C), data are shown for the 753 occupations that could be associated with yearly earnings across Census years, 2015 to the present. A nationally representative US sample of participants (n = 500) were randomized into one either the Google image condition, and the Control condition (in which they were asked to use the Google Images search engine to retrieve images of random, non-gendered categories, such as guitar or apple). After uploading an image for either an occupation (Image condition) or random distractor category (Control condition), participants indicated the average age of people in the target occupations (from 0 to 100). In the Control condition, participants were asked to indicate which gender they associate with a randomly selected occupation after uploading a description for an unrelated category. Participants completed this sequence for 22 unique occupations (randomly sampled from a set of 54 occupations). Participants in each condition were asked additional questions after inputting their age estimate for each occupation. Control participants were also asked to indicate which gender they most associate with the given occupation by selecting either “men”, “women”, or “don't know”. In the Image condition, participants were asked to indicate the perceived gender of the face they uploaded for a given occupation (using the same gender options indicated above); and they were also asked to rate their willingness to hire the person depicted for this occupation using a 7-point Likert. ChatGPT was prompted to generate resumes for 54 occupations in each of three conditions: (i) the Control condition, (ii) the Control-gender condition, and (iii) the Treatment condition. In the Treatment condition, the design of the Control condition was replicated except that ChatGPT was prompted to generate a resume for a named applicant whose name was selected from a list of 16 male and 16 female names spanning four ethnicities and normed for popularity, familiarity, and perceived age group. In the Treatment condition, ChatGPT was prompted 20 separate times for each name-gender-occupation prompt combination, yielding 34,560 resumes in total and 17,280 resumes for each gender group. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/adventure/outdoor-gear/a68151561/yeti-deals-prime-big-deal-days-2025/'>For Just a Few More Hours, Yeti Coolers, Bottles, and Mugs Are 30% Off for Prime Day—Here's What to Grab</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 14:56:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Amazon's second major shopping event of the year, Big Deals Day, is almost over. The deals also extend to major brands such as Yeti. During this sitewide sale event, Yeti is offering discounts of 30 percent or more on a wide range of products, including popular water bottles, coolers, and accessories you've likely been eyeing. Yeti has several items on sale across the board, including a variety of coolers. If you're looking to grab a new water bottle, Yeti has multiple options on sale right now, including both the Yonder and the Rambler. We love both of them, with many members of our team using both as their everyday water bottle at the office and on their weekend adventures. Among the many lifestyle products Yeti has on sale, we have a soft spot for the Colster can insulator and the beverage bucket; both of which are currently on sale for 30 percent off. The can insulator is essentially an upgrade on the old-school foam koozie you've been using for years, featuring technology that actually keeps your drink colder for longer. Big Deal Days is in its final stretch, so make sure to check out these Yeti deals and all other on-sale items before it ends. Tom Price is a reviews editor for Popular Mechanics while also contributing to Runner's World, Best Products, and Bicycling. Here at Popular Mechanics, Tom loves testing out home appliances, smart tech, gear for city life and outdoor adventures, small gadgets like watches and multitools, and anything else you can find piled up at his hoarder's nest of a desk. He also wrote for Best Reviews, The Daily Beast, and other publications covering tech, fitness gear, gadgets, and lifestyle products. Furthermore, he covered startup news, pop culture, and even professional wrestling. To follow more of his work, check back here for new stories. What Kind of Steel Is Best For a Pocket Knife? Carhartt vs. Dickies: Which Brand Comes Out on Top? Treasure Hunters May Have Found Jesse James's Gold Prime Day Just Made Pocket Knives More Affordable</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/adventure/outdoor-gear/a68156721/prime-big-deal-days-pocket-knife-deals-2025/'>Pocket Knife Deals Are Hotter Than Ever During Prime Big Deal Days—Our Experts Recommend These Smart Buys</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 14:37:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Amazon Prime Big Deal Days ends today, and there is an overwhelming amount of sales to shop if you're getting a jump on holiday shopping (whether that's for others or for yourself). Amazon offers price cuts on pocket knives from our favorite blade brands, including SOG, Civivi, Case, and more. I've scoured pocket knife deals and checked the specs against the knives we've tested and recommended to offer a selection that doesn't skimp on craftsmanship and save you more than just a couple of bucks. I also scoped out the type of knife steel each pick uses to ensure you're not getting a blade that'll wear out after only a couple of uses. Gerber's Paraframe is an absolute workhorse of a serrated blade for those who cut rope or fabric daily, and is now available for under $20. SOG's Twitch 3 is one of the brand's most popular knives thanks to its smooth spring-assisted opening and minimalist design. Personally, I love the CRKT CEO pocket knife: It's sleek, has lightning-fast deployment, and isn't much larger than a pen. Recently, he was coordinator of partnership content at another product journalism outlet. Prior to that, he was a buyer for an independent men's shop in Houston, Texas, where he learned all about what makes great products great. He enjoys thrifting for 90s Broadway tees and vintage pajama sets. His spare time is occupied by watching movies and running to impress strangers on Strava. What Kind of Steel Is Best For a Pocket Knife?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03195-1'>Chemistry Nobel for scientists who developed massively porous ‘super sponge' materials</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 12:21:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Susumu Kitagawa, Omar Yaghi and Richard Robson have won the 2025 chemistry Nobel for the development of metal-organic frameworks.Credits: Kota Kawasaki/Yomiuri Shimbun via AP/Alamy, Brittany Hosea-Small/UC Berkeley, Paul Burston/University of Melbourne Three scientists who developed a class of extremely porous materials known as metal-organic frameworks (MOFs) have won this year's Nobel Prize in Chemistry. Susumu Kitagawa at Kyoto University in Japan, Richard Robson at the University of Melbourne, Australia, and Omar Yaghi at the University of California, Berkeley, will share the prize of 11-million Swedish kronor (US$1.2 million), for inventing a new form of molecular architecture. Because they have such a vast internal surface area, MOFs could potentially be used to store gases, act as catalysts and deliver drugs, among many other applications. World's most porous sponges: intricate carbon-trapping powders hit the market World's most porous sponges: intricate carbon-trapping powders hit the market “This year's laureates…have found ways to create materials, entirely novel materials, with large cavities on their insides,” said Nobel committee chair Heiner Linke, a nanophysicist at Lund University in Sweden, at a press conference announcing the prize. “A small amount of such material can almost be like Hermione's handbag in Harry Potter — it can store huge amounts of gas in a tiny volume”. Or that could be used to separate toxic molecules from wastewater,” said Linke. “We're already seeing several applications that are popping up in addition to having learned so much from these materials from a basic chemistry perspective,” says Wendy Queen, a chemist at the Swiss Federal Institute of Technology in Lausanne. “I'm deeply honoured that my longstanding research has been recognized”, he said. At a press conference, Yaghi said that he received the call from Stockholm in an airport while waiting for his luggage. “You cannot prepare for a moment like that,” he said. “I was just surprised and delighted by the news.” Inspired by the structure of diamond crystals — in which each carbon atom binds to four other carbon atoms — he and his collaborator Bernard Hoskins designed a material made of pyramidal-shaped modules, with copper atoms linked by organic molecules1,2. In the early 1990s, Kitagawa developed MOFs that could be filled with water, and were stable enough to not be disrupted if they were subsequently dried up and filled with gas instead. One of the crucial properties of MOFs is that the materials that fill their gaps do not bind to the frameworks too strongly, which makes it possible then to easily get them back out. “Energetics is quite important for this technology,” Kitagawa said. He also realized that the frameworks could change shapes depending on whether they were filled or not, and in response to ambient changes such as temperature or exposure to light3. He also was able to create MOFs with a cubic structure5 and ones with much longer linkers, which had the internal surface area of a football field within just a few grams of solid powder. “People immediately saw the opportunities to be able to use these for applications in gas storage and separation,” says Christian Doonan, a chemist at the University of Adelaide in Australia who was a postdoctoral researcher in Yaghi's group. It got people in all subfields of chemistry interested.” World's most porous sponges: intricate carbon-trapping powders hit the market How to win a Nobel prize: what kind of scientist scoops medals? These science prizes want to rival the Nobels: how do they compare? Marcus kinetics control singlet and triplet oxygen evolving from superoxide AI is dreaming up millions of new materials. The CRG in Barcelona, invites outstanding and motivated candidates from across the world to apply for its International PhD Programme 2026. Develop nanofab processes, operate advanced tools includes EBL, KrF Stepper, PL, CDSEM & PVD for cutting-edge BCI research. Job Title: Associate or Senior Editor (social sciences), Research Cross-Journal Editorial Team, Nature Journals Locations: New York, Jersey City, S... World's most porous sponges: intricate carbon-trapping powders hit the market How to win a Nobel prize: what kind of scientist scoops medals? These science prizes want to rival the Nobels: how do they compare? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a68854951/karnak-temple-nile/'>Archaeologists Revealed New Secrets of One of Egypt's Most Famous Temples</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Centuries ago, the Nile River split into two and inadvertently created an island—fulfilling a piece of ancient Egyptian mythology. Because their ancient mythology references a high ground emerging from water, Egyptians may have constructed the famed Karnak Temple as a way of embracing that idea, building the temple on an island that formed when the Nile River shifted course. “This new research provides unprecedented detail on the evolution of Karnak Temple, from a small island to one of the defining institutions of Ancient Egypt,” Ben Pennington, lead author of the study and a visiting fellow in geoarchaeology at the University of Southampton, said in a statement from the school. An international research team analyzed 61 sediment cores from in and around the temple in addition to studying tens of thousands of ceramic fragments taken from the site. “The age of Karnak Temple has been hotly contested in archaeological circles,” Kristian Strutt, co-author of the study from the University of Southampton, said in a statement, “but our new evidence places a temporal constraint on its earliest occupation and construction.” This emerging island offered a foundation for both occupation and safe construction. “The river channels surrounding the site shaped how the temple could develop and where, with new construction taking place on top of old rivers as they silted up,” Dominic Barker, another co-author from the University of Southampton, said in a statement. The dumping of sand ties to an ancient Egyptian creation legend that said a god emerged from “the lake” onto high ground. “It's tempting to suggest that Theban elites chose Karnak's location for the dwelling place of a new form of the creator god, Ra-Amun, as it fitted with cosmogonical scene of high ground emerging from surrounding water,” Pennington said. As the Nile rerouted and left in its wake an island, the researchers believe ancient Egyptians clamored to build on what they saw as evidence of ancient beliefs made manifest. The result was a one of ancient Egypt's largest growing temple complexes. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Reality Itself Might Be a Shared Illusion African Rock Art Depicts Ancient Horned Reptile Hydraulic System May Have Built Egyptian Pyramids Will We Really Reach Singularity in 3 Months?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/2025-chemistry-nobel-goes-to-molecular-sponges-that-purify-water-store/'>2025 Chemistry Nobel Goes to Molecular Sponges That Purify Water, Store Energy and Clean Up the Environment</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>2025 Chemistry Nobel Goes to Molecular Sponges That Purify Water, Store Energy and Clean Up the Environment Three scientists, including one from the U.S., share the 2025 Nobel Prize in Chemistry for developing “metal-organic frameworks,” versatile molecular cages that can trap contaminants, store energy and possibly deliver drugs to specific areas of the body The 2025 Nobel Prize in Chemistry has been awarded for a versatile technology that can be used for an astonishing variety of purposes, from environmental remediation to drug delivery and energy storage. Susumu Kitagawa, Richard Robson and Omar M. Yaghi shared the award for their development of metal-organic frameworks, chemical cages that have small openings that can capture other small and diverse molecules. MOFs, as they are known, are being explored for their use in wastewater cleanup, PFAS removal, timed or multidrug release systems, and more. The cages are made of metal ions held together by organic, or carbon-containing, molecules. The cages can be one-dimensional or multi-dimensional, and they can be formed from a host of metals and organic linkers. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. He told the audience at a Nobel press conference in Stockholm on Wednesday that his dream is to use MOFs to separate the components of air for use in making other materials, via reusable energy. Yaghi, widely credited for expanding the development of MOFs, created stable frameworks from many combinations of metal ions and organic linkers. Megha Satyanarayana is chief projects editor at Scientific American. She is a former scientist who has worked at several news outlets, including the Detroit Free Press and STAT. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/astronaut-chris-hadfield-explores-cold-war-geopolitics-in-new-novel/'>The Space Race Gets a Fictional Rewrite from an International Space Station Commander</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Chris Hadfield Imagines a Suspenseful Twist on Cold War History in His New Book Astronaut-turned-author Chris Hadfield discusses his new thriller Final Orbit, which weaves real cold war history, space race geopolitics and firsthand experience into a gripping work of fiction. By Rachel Feltman, Kylie Murphy, Fonda Mwangi & Alex Sugiura You might know Chris Hadfield as the International Space Station commander who famously sang David Bowie's “Space Oddity” while orbiting Earth. But in recent years he's added fiction writer to his list of jobs, publishing thrillers based on the real history of the space race. His latest book, Final Orbit, is set in 1975 and features historical figures navigating geopolitical tensions that may sound surprisingly familiar to anyone following current events. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Here to talk about how his experience as an astronaut informs his fiction writing is Colonel Chris Hadfield, former commander of the International Space Station. What happens when a real-life astronaut turns to fiction? In this episode, Chris Hadfield shares how decades of spaceflight and global diplomacy inspired his latest novel, Final Orbit, a cold war thriller packed with historical intrigue and technical precision. Thanks so much for coming on to chat with us today. Hadfield: Yeah, I served as an astronaut for 21 years and flew in space three times, commanded a spaceship, did some space walks, but all good things must come to an end, and I retired from 35 years of government service. Working with King Charles [III] on space sustainability, a project called the Sustainable Markets Initiative. And you actually are just releasing—I believe it's your third work of fiction, which might surprise some listeners. It's been a busy few years, so they might've missed that you were writing fiction at all. Hadfield: I've always loved writing; it was my favorite subject in school. But I've always, you know, really admired writing and enjoyed the escapism of good fiction. My first three books were nonfiction, but I wanted to challenge myself to see if I could write good fiction, fiction that people would wanna read. And I'm, as you say, I'm just releasing the third book in the Apollo Murders series; it's called Final Orbit. And it's alternative-history thriller fiction set in the summer of 1975 during the cold war and the end of the Apollo era with Apollo-Soyuz and a derelict space station called Skylab, and all sorts of untoward things happen in Final Orbit. I'm really pleased with how the book came out. Feltman: As you said, these are works of fiction and, and they're thrillers, but obviously, there is a lot of fact woven into the story. Could we start by talking about just how your history as an astronaut has informed your fiction writing? So there's been an—almost an entire lifetime of research for writing a book 'cause, 'cause I'd just been fascinated by it. I've been so incredibly lucky to have this depth of real-world space experience, and, and so when I—now, when I'm writing fiction, I do a, a huge amount of research to make sure I get all the facts right, but I also have the lucky resource of my own memories and impressions and knowledge to draw on to make these books not just an interesting plot but something that absolutely could have happened, with every little detail correct. And I think that makes for a more compelling story. Feltman: And could you tell us about some of the real historical events and figures that readers can keep an eye out for in this book? Hadfield: Yeah, in my thriller or fiction books, almost all the characters are real people. I mean, President [Richard] Nixon; Henry Kissinger; Al Shepard; Bob Crippen; Mao Zedong; a guy named Qian Xuesen, who was the, the father of Chinese spaceflight the United States foolishly deported during the McCarthy era. And it's really fun weaving through history in the '70s because characters come up like George Bush Sr., who, at the time of 1975, when, when Final Orbit is set, he was sort of the, the top American in China—they didn't have an embassy because of conflict then but, but the, the American mission to China. So it's delightful really digging into what actually happened and the real characters, including the crews that were flying in space right then, the cosmonauts and astronauts, and then finding ways just through slightly alternative possibilities of what might have happened to really create a suspenseful and credible plot. In the back of the book, Rachel, I, I put a section just for the reader so they can see what was real and what wasn't real ... Hadfield: And I think, when you flick to the back and look at it, you'll be surprised just how much of that book—not only the people, but what actually happened—is, is word for word or, or minute by minute what really went on. So you've done two things that many people say they would like to do and are very hard to accomplish, which is going to space and writing and publishing a novel. What—which one do you think is harder [laughs]? So—and, and it, it's a life's work, and the odds are terrible [laughs]; almost everybody fails the medical as well as all of the other training requirements. What are you hoping that readers take away from this book, other than, you know, being gripped and entertained, obviously? Hadfield: I think people will be surprised, in, in reading Final Orbit, just how much of what was going on then is exactly being replayed right now ... And China is emerging as a new technical—technological force. That was going on in the early '70s. And when you look at the geopolitical forces going on right now, I think it's really worth reading to see: How did this happen in the past? And when you write fiction, you could really bring it to life; it's not just some sort of dry history but what were the, the forces and the individual desires? And then maybe it will help, as, as the news reels are ripping by today, gain a little in, insight into maybe why some of the things are happening today as well. Would you remind our listeners and viewers what it's called so they can go check it out? Hadfield: My new space thriller is called Final Orbit. You can find the voting link in our show notes. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi and Jeff DelViscio. This episode was edited by Alex Sugiura and Kylie Murphy. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Her work has been featured on The AV Club, No Film School, and The Black List. She approaches film with a DIY spirit, often writing, directing, producing, editing, and shooting her projects. Kylie currently works as a freelance editor. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. SciAm always educates and delights me, and inspires a sense of awe for our vast, beautiful universe. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American is part of Springer Nature, which owns or has commercial relations with thousands of scientific publications (many of them can be found at www.springernature.com/us). Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251007081827.htm'>Scientists uncover a hidden power source inside a monster black hole</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-08 05:24:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Such jets are also generated by other rotating black holes. A team of astrophysicists at Goethe University Frankfurt, led by Prof. Luciano Rezzolla, has developed a numerical code, named the Frankfurt particle-in-cell code for black hole spacetimes (FPIC), which describes with high precision the processes that convert rotational energy into a particle jet. In this process, magnetic field lines break and reassemble, leading to magnetic energy being converted into heat, radiation, and eruptions of plasma. This large computing power was essential to solve Maxwell's equations and the equations of motion for electrons and positrons according to Albert Einstein's theory of general relativity. According to the scientists, this process is accompanied by the generation of particles with negative energy that is used to power extreme astrophysical phenomena like jets and plasma eruptions. "With our work, we can demonstrate how energy is efficiently extracted from rotating black holes and channeled into jets," says Rezzolla. He adds that it is incredibly exciting and fascinating to better understand what happens near a black hole using sophisticated numerical codes. "At the same time, it is even more rewarding to be able to explain the results of these complex simulations with a rigorous mathematical treatment -- as we have done in our work." Note: Content may be edited for style and length. Scientists Discover a Protein That Literally Punches Holes in the Heart A Mile-Deep Detector Hints at the Universe's Darkest Secret Colossal 12,000-Year-Old Rock Art Reveals Forgotten Human Culture in Ancient Arabia Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            