
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-01</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09588-6'>Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 15:09:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease characterized by a progressive loss of motor neurons. Neuroinflammation is apparent in affected tissues, including increased T cell infiltration and activation of microglia, particularly in the spinal cord1,2. Autoimmune responses are thought to have a key role in ALS pathology, and it is hypothesized that T cells contribute to the rapid loss of neurons during disease progression3,4. However, until now there has been no reported target for such an autoimmune response. Here we show that ALS is associated with recognition of the C9orf72 antigen, and we map the specific epitopes that are recognized. We show that these responses are mediated by CD4+ T cells that preferentially release IL-5 and IL-10, and that IL-10-mediated T cell responses are significantly greater in donors who have a longer predicted survival time. Our results reinforce the previous hypothesis that neuroinflammation has an important role in ALS disease progression, possibly because of a disrupted balance of inflammatory and counter-inflammatory T cell responses4. These findings highlight the potential of therapeutic strategies aimed at enhancing regulatory T cells5, and identify a key target for antigen-specific T cell responses that could enable precision therapeutics in ALS. ALS is a neurodegenerative disease with a mean life expectancy of around four years after diagnosis. It is characterized by extensive damage to oligodendrocytes and a loss of motor neurons, leading to paralysis. Neuroinflammation is apparent in the affected tissues, and includes high levels of T cell infiltration—mainly by CD8+ cytotoxic cells but also by CD4+ T helper cells—and increased expression of human leukocyte antigen (HLA) class I and II molecules by microglia, particularly in the spinal cord1,2. It has long been hypothesized that an autoimmune process, including specific T cells, mediates the rapid loss of neurons during the disease3,4, but no direct evidence has been reported for such a response. A hallmark of ALS, like many neurodegenerative disorders, is the cytoplasmic inclusion of proteins such as TAR-DNA binding protein (TDP-43), particularly in oligodendrocytes and motor neurons6. In Parkinson's disease (PD) and related Parkinsonian disorders, alpha-synuclein (α-syn), the main component of Lewy body inclusions, has been found to be a target of autoreactive T cells7. However, two previous studies have been unable to identify a similar response to TDP-43 in individuals with ALS8,9. More than 25 genetic mutations have been associated with ALS, including mutant alleles of TDP-43 and of superoxide dismutase 1 (SOD1). Mutations in C9orf72 are present in 40% of familial ALS cases and around 10% of sporadic ALS cases, and are also associated with frontotemporal dementia10,11. The disease-associated mutations of C9orf72 produce an expansion of a hexanucleotide repeat in a nominally non-coding region, although these repeats are thought to encode smaller peptides composed of dipeptide repeats that might affect proteolytic processing12. Expansions of nucleotide repeats are also found in genes associated with other developmental and degenerative disorders, including Huntington's disease, Fragile X syndrome, spinocerebellar ataxias and Friedreich's ataxia. It is widely thought that ALS is associated with decreased expression of C9orf72, and that loss of C9orf72 function contributes to disease pathogenesis13, suggesting that an autoimmune attack on C9orf72 might result in decreased expression and influence pathogenesis. Here we report an association of ALS with recognition of the C9orf72 antigen, and map the specific epitopes that are recognized. These responses are mediated by CD4+ T cells that preferentially release IL-5 and IL-10. We found that the frequency of IL-10-releasing C9orf72-specific T cells was significantly higher in donors with ALS who had a longer predicted survival time, raising the possibility that the T cells might have a regulatory and protective influence on disease progression. We analysed peripheral blood mononuclear cells (PBMCs) from whole-blood donations from 40 individuals with ALS and 28 age- and sex-matched healthy controls. Study participants were recruited from four collaborating clinic sites across the USA: Emory University (Atlanta, Georgia); the National Institute for Neurological Disorders and Stroke at the National Institutes of Health (Bethesda, Maryland); the University of California, Irvine (Irvine, California); and Harvard Medical School/Massachusetts General Hospital (Boston, Massachusetts). In addition, nine of the study participants with ALS were recruited through Sanguine Biosciences (Woburn, Massachusetts), a contract research organization. Demographics and clinical characteristics of the cohort are shown in Extended Data Table 1. The cohort was age-matched (control, median 59.0, range 29–82; ALS, median 61.0, range 45–81; P = 0.680 by two-tailed Mann–Whitney test, which was used here and throughout the rest of the manuscript). The median time of blood donation after symptom onset was 36 months (24.0 for male individuals and 48.0 for female individuals; P = 0.023). The median time after ALS diagnosis was 12.0 months (12.0 for male individuals and 18.0 for females individuals; P = 0.599). The revised ALS functional rating scale (ALSFRS-R) score—a measure of disease severity in ALS that captures aspects of bodily function including speech, handwriting, walking and breathing—was recorded at the time of diagnosis and at follow-up to measure disease progression. ALSFRS-R scores were recorded at the time of blood donation; scores were 38.0 (median) for donors with ALS (n = 33) (39.0 for male individuals and 37.0 for female individuals; P = 0.329). Using flow cytometry, we determined the frequencies of different cell subsets in the ALS and control cohorts16 (Extended Data Fig. 1), and identified a reduction in the frequency of mucosal-associated invariant T (MAIT) cells (P = 0.047; Fig. 1a), defined by the co-expression of CD3, CD161 and CD26. The comparisons of these cell subsets were not corrected for multiple testing because they are independent of each other, but the statistical significance in MAIT frequency would be likely to be lost if correction for multiple testing were to be applied. We found no further significant differences in the major T cell subsets (Fig. 1a) or in the CD4+ and CD8+ memory T cell subsets (Fig. 1b,c), defined on the basis of CD45RA and CCR7 expression as naive T cells (TN; CD45RA+CCR7+), central memory T cells (TCM; CD45RA−CCR7+), effector memory T cells (TEM; CD45RA−CCR7−) and effector memory T cells expressing CD45RA (TEMRA; CD45RA+CCR7−). However, within the functional CD4+ T helper cell subset, defined by the expression of CCR6, CXCR3 and CCR4 subset markers17, we found significant decreases in the frequency of TH1* cells—a subset of T helper cells with a phenotype similar to that of both TH17 and TH1 cells16—and in the frequency of TH1 cell subsets, in individuals with ALS compared with healthy control donors (P = 0.002 and P = 0.006, respectively; Fig. a–c, Broad representation of lymphocyte subsets (a) and CD4 (b) or CD8 (c) memory T cell subsets in individuals with ALS (n = 19) and in healthy control (HC) donors (n = 20). d, Composition of functional T helper cell subsets defined on the basis of expression of the CCR6, CXCR3 and CCR4 subset markers, in the same donors as in a–c. P values from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence intervals are shown. We then investigated whether there were autoimmune responses against possible antigens associated with ALS, including TDP-43, SOD1 and C9orf72, following an approach that was previously used to define autoreactive epitope sequences from individuals with PD7,18. Owing to the low frequency of autoreactive T cells7,18,20, PBMCs were stimulated with the peptide sets to expand autoantigen-specific T cells, to obtain sufficient cells for reliable phenotyping of the T cell responses. After two weeks of in vitro expansion, FluoroSpot assays were used to measure T cell responses to the peptide pools by measuring the release of interferon-γ (IFNγ), a classical response by pro-inflammatory TH1 and TH1* CD4+ T cells, and interleukin-5 (IL-5) and IL-10, which are associated with TH2 and/or regulatory CD4+ cells. In addition to the TDP-43, SOD1 and C9orf72 pools, we included a positive control pool for T cell reactivity, encompassing epitopes derived from Epstein–Barr virus (EBV), for which nearly 100% of the US population is seropositive21,22, and which has been used extensively as a positive control owing to the high-magnitude T cell response detected in most individuals19. Overall antigenic responses, defined as the total sum of IFNγ, IL-5 and IL-10 responses detected against each pool, are presented in Fig. Notably, a significant and selective increase in overall responses to the C9orf72 peptide pool was observed in participants with ALS, with the geometric mean responses approximately 4.15-fold higher in the ALS group compared with the controls (P = 0.001). These data suggest that C9orf72 is preferentially targeted by autoreactive T cells in the context of ALS. a, Total IFNγ, IL-5 and IL-10 responses detected against each pool in HC donors (n = 28) and individuals with ALS (n = 28). b–d, Production of IFNγ (b), IL-5 (c) and IL-10 (d) individual cytokines, in the same donors as in a. SFCs, spot-forming cells. e, Relative balance of average IFNγ, IL-5 and IL-10 responses in HC individuals and in individuals with ALS. f, IFNγ-mediated (left), IL-5-mediated (middle) and IL-10-mediated (right) T cell responses towards serially diluted concentrations of the C9orf72 peptide pool in HC individuals (n = 5) and individuals with ALS (n = 5). g, Corresponding peptide concentration required to achieve 50% of the maximum response (EC50) in HC donors (n = 5) and those with ALS (n = 5). For one HC donor, the lack of IL-5 responses prevented the development of a dose–response curve. P values from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence intervals are shown. All three cytokines were detected in ALS and control cohorts (Fig. 2b–d), and the differences were most prominent for IL-5 production (fold change = 5.13; P = 0.002). Differences in the production of IL-10 (fold change = 3.08; P = 0.004) and IFNγ (fold change = 2.08; P = 0.037) were smaller but still significant. These comparisons were not adjusted for multiple comparisons, because they are independent of each other. Because the difference in IFNγ-mediated T cell responses was close to the limit of significance, statistical significance would be likely to be lost if correction for multiple comparisons were to be applied. Thus, although autoimmune T cell responses were of higher magnitude in people with ALS, the proportion of the autoreactive T cells that released IL-5 was higher in individuals with ALS than it was in healthy controls (P = 0.014), and, by contrast, the proportion of IFNγ responses was higher in healthy controls than it was in those with ALS (P = 0.007; Extended Data Fig. In line with an IL-5 biased response, we found that the IFNγ/IL-5 ratio was significantly lower in individuals with ALS (P = 0.047; Extended Data Fig. Of note, we found that the IFNγ-releasing cells in individuals with ALS had an approximately twofold lower avidity compared with those of healthy controls (P = 0.016; Fig. 2f,g), whereas no difference was observed for IL-5- or IL-10-releasing cells. We conclude that ALS is associated with a skewing of autoreactive T cell responses towards the TH2-like phenotype, consistent with the overall reduction in TH1/TH1*-like subsets (Fig. To assess whether C9orf72-specific T cell responses are specific for ALS or represent a general feature of neurodegenerative disorders, we measured responses of T cells towards C9orf72 peptides in participants diagnosed with PD or Alzheimer's disease (AD)23,24 (Extended Data Table 2). Notably, when comparing these individuals with the previously screened ALS and control cohorts, we found that participants with PD or AD had significantly increased total and IFNγ-mediated T cell responses towards C9orf72, compared with control individuals (Extended Data Fig. However, similar to the control group, their responses were found to be mediated mainly by IFNγ, and the proportion of the total response mediated by IFNγ and IL-5 differed significantly between individuals with ALS and those with PD or AD (Extended Data Fig. We next assessed the representation of CD4+ versus CD8+ T cell subsets in PBMC cell cultures after two weeks of restimulation with the C9orf72 peptide pool, using flow cytometry (Extended Data Fig. In both ALS and control samples, analysis of live single CD3+ cells revealed that CD4+ T cells were predominant (control: CD4+, 83.5% of live CD3+ cells; CD8+, 8.9%; P < 0.001 (Extended Data Fig. To further characterize the phenotype of C9orf72-specific T cells, we performed intracellular cytokine staining (ICS) assays to quantify cells producing IFNγ, IL-4, IL-10 and IL-17 in response to antigen stimulation after the two-week culture period (Extended Data Fig. IL-4 was chosen as a representative TH2 cytokine because this cytokine is more readily detected by ICS than is IL-5. Cytokine-producing cells in both control and ALS cohorts were mostly associated with a CD4+ phenotype (control: CD4+, median 2.0; CD8+, 0.25; P < 0.001 (Extended Data Fig. 4c); ALS: CD4+, median 1.4; CD8+, 0.07; P = 0.003 (Extended Data Fig. Cytokine production by CD4+ T cells from healthy controls was highest for IFNγ, and for individuals with ALS it was highest for the TH2 cytokine IL-4 (Extended Data Fig. 4e,f), reflecting a trend for a reduced proportion of IFNγ responses in individuals with ALS compared with controls (P = 0.078; Extended Data Fig. Indeed, we found that the IFNγ/IL-4, IFNγ/IL-10 and IFNγ/IL-4 + IL-10 ratios were all significantly lower in individuals with ALS (P = 0.032, P = 0.049 and P = 0.008, respectively; Extended Data Fig. 4h–j), confirming a T cell response skewed away from IFNγ in ALS, as seen in Fig. In conclusion, C9orf72 responses were mediated mostly by CD4+ T cells. These responses were associated with a relative bias in favour of IL-4 and IL-10 production in the case of ALS samples and an IFNγ bias in the case of control samples. To identify the specific epitopes recognized within C9orf72, we assayed responses by 28 individuals with ALS. After two weeks of in vitro culture, responses were deconvoluted by assaying ‘mesopools' consisting of pools of around ten individual peptides, which were then deconvoluted to identify individual epitopes. High immunogenicity was associated with regions spanning residues 41–95, 266–295 and 361–445. Most (n = 20 of 28) individuals with ALS recognized multiple epitopes (median 3, range 0–26; Fig. a, Distribution of recognized epitopes among the 100 peptides spanning the C9orf72 protein sequence, identified in 28 participants with ALS. c, Number of predicted binding events to the 27 most common HLA II allelic variants, by peptides recognized in three or more patients (n = 21), one or two patients (n = 51) or no patients (n = 23). Binding was predicted using the NetMHCIIpan 4.1 EL prediction method, using a cut-off for a binding event as a predicted binding percentile score threshold of 20% or less26. P value from two-tailed Mann–Whitney test and mean ± 95% confidence intervals are shown. Twenty-three C9orf72-derived peptides were recognized by three or more individuals, highlighting the broad immunogenicity of the C9orf72 protein. The most frequently recognized epitopes are listed in Table 1, including sequences, position, overall response frequency and average response magnitude for responding individuals. To identify potential HLA-restricted peptides, we next HLA-typed the participants with ALS included in the deconvolution experiment. Using the restrictor analysis tool for epitopes (RATE) algorithm25, we identified 13 HLA–peptide combinations that were significantly associated with a detectable T cell response. Using the NetMHCIIpan-4.1 EL algorithm26, we verified that significant peptide binding to the corresponding HLA molecule was predicted for seven of these interactions. The C9orf72 protein can contain multiple phosphorylated residues. We compared the immunogenicity of phosphorylated or non-phosphorylated peptides at positions 1–15, 6–20, 426–440, 431–445 and 436–450. In general, similar reactivity was noted for the phosphorylated and non-phosphorylated peptides (Supplementary Table 1). The 23 most dominant epitopes recognized in three or more participants with ALS were associated with predicted HLA class II binding to 9 ± 7.8 of the 27 HLA alleles. By contrast, epitopes recognized by only one or two patients were associated with binding to 5 ± 5.3 of the alleles, whereas peptides recognized by no one were associated with binding to 3 ± 5.6 predicted alleles (Fig. The number of predicted binding events was significantly higher for the peptides recognized in three or more patients than for the peptides recognized in no individuals (P = 0.001). Similarly, we observed a positive correlation between the response frequency and the number of the 27 HLA alleles predicted to bind to each peptide (r = 0.339, P = 0.001; Extended Data Fig. We further observed that the predicted binding strength, represented as a normalized rank value in which a small value indicates high affinity25, was significantly lower for the 72 peptides recognized in one or more individuals, compared with the peptides not recognized in any individual (P = 0.010; Extended Data Fig. Overall, these results indicate that HLA binding influences C9orf72 epitope reactivity and that the most dominant epitopes are likely to be associated with presentation by multiple common HLA alleles; that is, exhibit a promiscuous HLA-binding capacity. We next investigated whether SOD1 and TDP-43 peptides were also predicted to bind HLA II. As shown in Extended Data Fig. 5c, TDP-43 and SOD1 peptides are predicted to bind the most common 27 HLA-DR, -DQ, and -DP HLA class II allelic variants, with no statistically significant difference (P = 0.266) in the number of HLA class II allelic variants predicted to bind to TDP-43 (median 2, range 0–22), SOD1 (median 1.5, range 0–23) or C9orf72 (median 1, range = 0–27)). Thus, the increased response to C9orf72 is not due to a lack of HLA class II binding for SOD1 and TDP-43. No loss of expression of SOD1 or TDP-43 has been reported in peripheral cells, which could explain why the tolerance to these proteins has not been affected. Alternatively, this phenomenon could be related to an autophagy-related increase in the secretion of C9orf7229,30; because TDP-43 and SOD1 are not involved in autophagy, they might be absent or present at low levels in exosomes. We examined the correlation between C9orf72 reactivity and biological and clinical variables for individuals with ALS. No significant difference in total C9orf72 reactivity was found between male and female individuals (Extended Data Fig. In addition, no significant correlation was found between total C9orf72 reactivity and age (Extended Data Fig. 6b), time from symptom onset (Extended Data Fig. 6c) or time from diagnosis (Extended Data Fig. No significant difference was also noted for any individual cytokine considered separately (not shown). A trend for a positive correlation was found between IL-5 responses and ALSFRS-R score (r = 0.279, P = 0.119), whereas no trend between IFNγ and IL-10 responses and ALSFRS-R was detected (Extended Data Fig. Thus, this suggests that C9orf72 reactivity is present at disease onset and diagnosis. Because ALS is associated with multiple genes, we investigated whether differences in C9orf72 reactivity were associated with specific mutant alleles that have been linked to the disease. Notably, C9orf72 CD4+ T cell reactivity was approximately sixfold higher in individuals carrying mutations in the non-coding regions of C9orf72 than it was in individuals with ALS who had other mutations (ataxin-2, CHCHD10, FIG4, NEK1, SOD1 and TBK1), or in those who did not carry known mutations or for whom genetic data were not available (unknown mutation status) (Extended Data Fig. The difference was most pronounced for IL-5 and IL-10; IFNγ showed a non-significant trend (Extended Data Fig. We validated these findings in a second cohort of seven participants with ALS carrying the C9orf72 mutation and seven participants with ALS who did not carry any known mutations linked with ALS (Extended Data Table 3). T cell responses towards C9orf72 were increased in the C9orf72 mutation carriers for all three cytokines in the validation cohort (Fig. a–d, Total (a), IFNγ-mediated (b), IL-5-mediated (c) and IL-10-mediated (d) T cell responses towards C9orf72 in a validation cohort of participants with ALS carrying the C9orf72 mutation (n = 7) or not carrying any mutation linked to an increased risk of ALS (n = 7). e–h, Total (e), IFNγ-mediated (f), IL-5-mediated (g) and IL-10-mediated (h) T cell responses towards C9orf72 in individuals with ALS with a short (n = 6) or long (n = 11) predicted survival time. P values from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence intervals are shown. Using this approach, we categorized patients according to their predicted survival time: very short (one individual), short (one individual), intermediate (four individuals), long (three individuals) or very long (eight individuals). Notably, we observed an approximately fivefold higher IL-10-mediated T cell response towards C9orf72 in individuals with ALS who had a longer predicted survival time (long to very long; 43.7 and 91.0 months median predicted survival time, respectively, in the original study31), compared with patients who had a shorter predicted survival time (very short to intermediate; 17.7, 25.3 and 32.2 months median predicted survival time, respectively, in the original study31) (P = 0.033; Fig. Here we identify C9orf72 as a major target of ALS-associated T cell autoreactivity. This is, to our knowledge, the first study showing the recognition by human T cells of a specific autoantigen associated with ALS. Notably, although the autoreactive T cell response is found broadly in individuals with ALS, it is particularly high in those who are carriers of C9orf72 mutant alleles in classically non-coding regions of the open reading frame. We find that the T cell subset associated with C9orf72 reactivity predominantly comprises CD4+ T cells associated with both inflammatory and anti-inflammatory phenotypes. The anti-inflammatory IL-10-mediated responses were associated with a longer predicted survival time, raising the possibility that enhancing this anti-inflammatory component might provide therapeutic benefits. The responses were broad, encompassing several unique T cell epitopes. Neurodegenerative diseases such as PD and AD were classically not considered to have autoimmune features. However, multiple sclerosis (MS) is associated with high levels of brain infiltration of T cells suspected to react with epitopes derived from viral-related or myelin-related proteins32. Adaptive immune responses related to herpes viruses have been implicated in AD pathogenesis33. We have also reported that people with PD have CD4+ and CD8+ T cells that react with α-syn and PINK1, products of PD-associated genes7,18,20. Thus, the involvement of adaptive immune responses in neurodegenerative disease might be common rather than exceptional. We identify autoimmune responses to epitopes derived from C9orf72, a protein corresponding to the most common genetic cause of ALS10. C9orf72 is thought to have roles in regulating autophagy and secretory pathways13. In PD, AD and ALS, disruption of autophagy and intracellular transport has been found to induce increased autophagic organelle fusion with the plasma membrane and secretion of contents into the extracellular space by neurons through a process known as secretory autophagy29,34,35. These secreted exosomes have been found to be enriched for proteins involved in autophagy30. Because C9orf72 is involved in autophagy, it is possible that HLA-II-expressing cells of the monocyte lineage, such as astrocytes, microglia and border-associated macrophages, could internalize extracellular C9orf72-containing autophagosome content, and present C9orf72 antigens for recognition by CD4+ T cells. Alternatively, C9orf72 could be released during neuronal cell death, and thereafter internalized by glial cells or central nervous system (CNS)-associated macrophages and presented to CD4+ T cells. Because both neuronal cell death and secretory autophagy are increased in PD, AD and ALS, this could explain why individuals with PD and AD were also found to have increased T cell responses towards C9orf72. However, the phenotype of C9orf72-reactive T cells differed between individuals with ALS, those with PD and those with AD, in that individuals with PD or AD had a significantly higher proportion of pro-inflammatory IFNγ-releasing cells and lower proportion of IL-5-releasing cells than individuals with ALS. The underlying mechanisms that drive this difference in T cell phenotype are not known, but it will be of interest in future studies to investigate whether IL-10-releasing C9orf72-specific T cells could provide a therapeutic approach in PD and AD, as the results here suggest for ALS. Future studies will aim to establish whether and how regulatory C9orf72-specific T cells can slow the progression of ALS in preclinical animal models; for example, considering whether regulatory C9orf72-specific T cells promote a neuroprotective phenotype in antigen-presenting glial cells36. However, to the best of our knowledge, such a model is not at present available. We did not detect significant differences in SOD1-specific T cell responses, and observed only a non-significant trend in total reactivity for TDP-43, consistent with two previous studies that found a lack of T cell reactivity with TDP-43 in individuals with ALS8,9. We did, however, detect a small but significant increase in IL-5-mediated T cell responses towards TDP-43. Thus, although TDP-43 does not seem to be a major target of autoreactive T cells in ALS, it could potentially be a target of autoreactive T cells in a subset of patients. Our results, therefore, do not exclude the possibility of additional autoimmune targets in ALS, and we note that individuals with disorders such as PD and MS have been found to have multiple autoimmune antigens7,18,37. Previous studies have reported that individuals with ALS have increased frequencies of clonally expanded T cells8,38, including TH2-like T cells in the cerebrospinal fluid38. However, because no previous studies have reported a target of these expanded cells, it is not known whether this expansion is driven by an autoimmune process, or through bystander activation owing to increased levels of inflammatory cytokines. We find that the targets of T cell reactivity cover the length of the C9orf72 protein, with regions spanning residues 41–95, 266–295 and 361–445 associated with high immunogenicity. Notably, the finding that the reactivity spans across C9orf72 argues against a molecular mimicry effect, as suggested for MS, in which T cells reactive for EBV also cross-react with myelin autoantigens32. Moreover, the observation that individuals with ALS have T cell reactivity towards a median of 3 epitopes, and as many as 26 peptides, indicates that they typically have an oligoclonal—and, in some individuals, polyclonal—T cell response to C9orf72. Analysis of predicted HLA-binding capacity reveals a possible mechanism for the high immunogenicity of C9orf72, in that these regions contain peptides with the capacity to bind multiple HLA types that are commonly encountered in human populations39. As C9orf72 is highly expressed in CD14+ conventional monocytes13,42, reduced expression might result in reduced peripheral tolerance towards C9orf72. The loss of peripheral tolerance would allow C9orf72-specific T cells to persist and later expand in individuals with ALS when extracellular C9orf72-containing autophagosome content is internalized and presented to T cells by microglia or other antigen-presenting cells in the CNS. Consistent with this hypothesis, we found, in two independent cohorts, that individuals with ALS who carry the C9orf72 intron mutation have the highest autoimmune C9orf72-specific T cell responses. Individuals with long predicted survival times exhibited the highest IL-10-mediated T cell responses, suggesting that T cell responses skewed away from IL-10 are associated with a more rapid disease progression. As ALS is associated with decreased C9orf72 expression13, it is possible that a pro-inflammatory autoimmune attack on C9orf72 results in decreased expression and thereby influences pathogenesis. Although neurons and oligodendrocytes do not typically present antigens, and CNS parenchymal T cells are rare in healthy conditions, rat motor neurons have been shown to express MHC class I RNA and to upregulate this expression with exposure to IFNγ (ref. CD8 T cells expressing mutant SOD1 have been reported to kill motor neurons in mice44. In the mouse CNS, IFNγ rapidly promotes a high level of MHC class I synthesis in oligodendrocytes and microglia, with lower levels in neurons and astrocytes45, and HLA class II expression has been reported in classes of human oligodendrocytes46. The T cells that recognize C9orf72 are predominantly CD4+, with a tendency for polarization toward a TH2 cell or regulatory T (Treg) cell-like phenotype. Individuals with ALS have substantially more C9orf72-reactive T cells than healthy controls, but they have a lower proportion of TH1- and TH1*-like CD4+ T cells and a higher frequency of TH2- and Treg-like T cells than control individuals. Of note, this was specific for the C9orf72-reactive T cells, because no difference in the frequency of IL-5- or IL-10-releasing cells was observed in response to the control antigen EBV. IL-5 and IL-10 are associated with anti-inflammatory features47, and it is possible that the autoimmune reactivity detected might be multifaceted, with an IFNγ component having an inflammatory role and an IL-10 component a counter-inflammatory one. Although IL-5 and other TH2-associated cytokines have classically been associated with allergy and thought to act principally on basophils, these cytokines have also been found to have crucial roles in the regulation of cognitive function and brain inflammatory processes47. We found that the IFNγ-releasing T cells had a lower functional avidity to the C9orf72 peptides in participants with ALS than in healthy controls. It is tempting to speculate that this reduced avidity reflects an increased peripheral tolerance towards C9orf72 mediated by the IL-10-releasing T cells, but this needs to be further investigated. The observation that IL-10-mediated C9orf72-specific T cell responses were significantly higher in participants with ALS who had longer predicted survival times aligns with previous reports suggesting that ALS is characterized by an initial neuroprotective TH2 and Treg cell response, which is subsequently replaced by a pro-inflammatory TH1 response38,48,49. Those studies were limited to bulk phenotyping of T cells38,49,50, and the identification of C9orf72 as a major target of ALS-associated T cell autoreactivity promises to help define the specific changes in the adaptive immune system that occur over the course of the disease. Overall, our findings support suggestions that neuronal and oligodendrocyte death in ALS could be due to an imbalance of neuroinflammatory and counter-inflammatory pathways4, and are in line with reports of high levels of effector T cells38,50. Given this, clinical trials intended to enhance Treg cells should be considered5—in particular, trials of antigen-specific T cell responses, which could provide targeted therapies for people with ALS. Because of the low frequency of autoreactive T cells, our experiments required an in vitro restimulation step to expand the relatively rare responding T cells7,18,20 so as to obtain sufficient cells for a reliable phenotyping. The use of this technique is therefore limited to epitopes that are already well characterized. Furthermore, historically, producing effective HLA class II tetramers, which are required to measure responses by CD4 T cells, has been technically challenging. However, analysing potential HLA restrictions for selected epitopes could enable the design of tetramer reagents in future studies. Owing to the low responses of control individuals to the C9orf72 peptide pool, we did not perform peptide–response deconvolution for these participants. The study did not address phenotypic subtypes of ALS or related disorders, including frontotemporal dementia. As mentioned, as with most autoimmune disorders, there are likely to be additional targets of autoimmune reactivity. This study examined a limited number of candidate proteins because of their established genetic connections with ALS. It would be interesting to perform an unbiased analysis of candidate proteins. In our approach, each candidate antigen is tested with sets of overlapping 15-mer peptides, and to test, for example, the top 1,000 candidate open reading frames expressed in nervous tissues, hundreds of thousands of peptides would be required. Because about two million to three million cells (corresponding to 2–3 ml of blood) are required to measure the rare autoimmune T cells that respond to any given peptide pool, testing even 1,000 candidates would require the donation of an impractical amount of blood. T cell-receptor analysis of antigen-specific T cells in the CNS would be valuable, but T cells are present at very low numbers in the CNS of people with ALS38, and a successful single-cell characterization of antigen-specific T cells in tissue requires thousands of cells. All participants provided written informed consent for participation in the study. Research activities with samples from Sanguine Biosciences (a fee-for-service, contract research organization) are not considered human participants research under Health and Human Services regulations. Individuals with ALS (n = 40) and healthy control individuals (n = 28) were recruited by Massachusetts General Hospital, Emory University, University of California, Irvine, Sanguine Biosciences and the National Institutes of Health, referred to as the study cohort in the Methods. An additional 14 participants with ALS were included from the University of California, San Diego and served as a validation cohort. Samples from Massachusetts General Hospital were used to perform preliminary experiments and are not included in the data presented in this study. Inclusion criteria for patients with ALS consisted of: (i) a diagnosis of familial or sporadic ALS according to the El Escorial criteria; (ii) aged 20 years or older at the time of symptom onset; and (iii) the ability to provide informed consent. Exclusion criteria included: (i) other major neurological or medical diseases that could cause progressive weakness or cognitive dysfunction; and (ii) an unstable medical condition that would make participation unsafe. Inclusion criteria for age- and sex-matched control individuals consisted of: (i) age including 53–82 to match the ages of individuals with ALS; (ii) sex matching that of individuals with ALS; and (iii) the ability to provide informed consent. Exclusion criteria for control participants were the same as those for ALS, with the addition of self-reported ALS genetic risk factors (that is, ALS in first-degree blood relatives). ALS diagnosis was based on clinical criteria (symptoms and examination findings). Typically, the diagnosis was associated with clinical findings of upper and lower motor dysfunction, a progressive course and a failure of diagnostic tests such as brain and spinal cord magnetic resonance imaging (MRI), electrophysiological testing and routine laboratory studies to identify an alternative cause. If the clinical presentation deviated from these typical features, a neuromuscular specialist with extensive experience in diagnosing ALS was consulted to adjudicate the diagnosis. ALSFRS-R data were used to evaluate the functional status of participants with ALS and monitor functional changes in patients over time. ALSFRS-R data were missing for 7 participants with ALS from the study cohort and all 14 patients from the validation cohort; these were excluded from analyses in which the ALSFRS-R was used. Predicted survival time was obtained using the online ENCALS prediction model31 (http://tool.encalssurvivalmodel.org/), a well-established and widely used survival prediction tool for participants with ALS, originally developed using data from more than 10,000 individuals across several ALS centres in Europe. The classification is based on diagnostic delay (months), progression rate (points decrease on ALSFRS-R per month), forced vital capacity (percentage of predicted capacity based on normative values for sex, age and height), definite ALS (according to El Escorial criteria), frontotemporal dementia (yes or no), C9orf72 repeat expansion (yes or no) and site of onset. The prediction model classifies patients into one of five categories on the basis of predicted survival time: very short, short, intermediate, long or very long. Twenty-four individuals with ALS were HLA-typed at an American Society for Histocompatibility and Immunogenetics (ASHI)–accredited laboratory at Murdoch University in Western Australia. Typing for HLA class II (DQA1, DQB1, DRB1, DRB3, DRB4, DRB5 and DPB1) was performed using locus-specific PCR amplification, sequencing was done using the Illumina MiSeq platform and the alleles were called using an ASHI-accredited HLA allele caller software pipeline, IIID HLA analysis suite (http://www.iiid.com.au/laboratory-testing/). The full HLA typing data for these patients are provided in Supplementary Table 2. Individuals with PD (n = 15) were recruited by the Movement Disorders Clinic at the Department of Neurology at CUMC. Inclusion criteria for the patients consisted of (i) clinically diagnosed PD with the presence of bradykinesia and either resting tremor or rigidity; (ii) PD diagnosis between the age of 35 and 80; (iii) history establishing the benefit of dopaminergic medication; and (iv) ability to provide informed consent. Exclusion criteria for PD were: (i) atypical parkinsonism or other neurological disorders; (ii) history of cancer within the past three years; (iii) autoimmune disease; and (iv) chronic immune-modulatory therapy. The recruited individuals with PD all met the UK Parkinson's Disease Society Brain Bank criteria for PD. Individuals with AD (n = 15) were recruited from the Alzheimer's Disease Research Center at CUMC or from PrecisionMed (a fee-for-service, contract research organization). Patients recruited from CUMC were diagnosed by neurologists according to the National Institute of Aging and Alzheimer's Association criteria24; those recruited from PrecisionMed were diagnosed according to NINCDS-ADRDA criteria23, by a neurologist or internist. Venous blood was collected from each participant in either heparin or EDTA-containing blood bags or tubes, as previously reported and described. PBMCs were isolated from whole blood by density-gradient centrifugation using Ficoll Paque Plus (GE). In brief, blood was first spun at 800g for 15 min with brakes off to remove plasma. Plasma-depleted blood was then diluted with RPMI, and 35 ml of blood was carefully layered on tubes containing 15 ml Ficoll Paque Plus. These tubes were centrifuged at 800g for 25 min with the brakes off. The interphase cell layer resulting from this spin was collected, washed with RPMI, counted and cryopreserved in 90% v/v fetal bovine serum (FBS) and 10% v/v dimethyl sulfoxide (DMSO), and stored in liquid nitrogen until tested. The detailed protocol for PBMC isolation can be found at protocols.io (https://doi.org/10.17504/protocols.io.bw2ipgce). Peptides were synthesized commercially as crude material on a 1-mg scale by TC Peptide Lab. Lyophilized peptide products were dissolved in 100% DMSO at a concentration of 20 mg ml−1, and their quality was spot-checked by mass spectrometry. These peptides were also combined in ‘megapools' as described19 and used in the antigen screening experiments. C9orf72 peptides were also tested individually, as described in the main text. In vitro expansion and subsequent FluoroSpot assays were performed as previously described20. An equal number of ALS and control samples were included per experiment, to reduce the risk of technical biases. In brief, PBMCs were thawed and then stimulated with neuroantigen or EBV peptide pools (5 μg ml−1) for four days. After four days, cells were supplemented with fresh RPMI and IL-2 (10 U ml−1, ProspecBio), and fed again every three days until day 11. After two weeks of culture, T cell responses to neuroantigen pools were measured by IFNγ, IL-5 and IL-10 FluoroSpot assays. A total of 1 × 105 of the collected cells were plated in each well of the coated FluoroSpot plates along with each respective antigen (5 μg ml−1), and incubated at 37 °C in 5% CO2 for 22 h. Cells were also stimulated with 10 μg ml−1 phytohaemagglutinin (positive control) and DMSO corresponding to the concentration in the peptide pool dilutions (negative control) to assess non-specific cytokine production. All conditions were tested in triplicate. An antibody cocktail containing IFNγ (7-B6-1-FS-BAM), IL-5 (5A10-WASP) and IL-10 (12G8-biotin) prepared in phosphate-buffered saline (PBS) with 0.1% bovine serum albumin was added and incubated for two hours at room temperature. Membranes were then washed again, and secondary antibodies (anti-BAM-490, anti-WASP-640 and SA-550) were incubated for one hour at room temperature. Then, membranes were washed, incubated with fluorescence enhancer (Mabtech) and air-dried for reading. Spots were read and counted using the Mabtech Apex (v.2.0) software on the Mabtech IRIS system. The dilutions of all antibodies used are provided in Supplementary Table 3. Responses were considered positive if they met all three of the following criteria: (i) DMSO background-subtracted SFCs per 106 PBMCs ≥ 100; (ii) stimulation index ≥ 2 compared with DMSO controls; and (iii) P ≤ 0.05 by Student's t-test or Poisson distribution test. Negative responses were assigned as 100 SFCs per 106 PBMCs, the lower limit of detection of the assay. The total SFCs per donor measurement was obtained by combining the individual IFNγ, IL-5 and IL-10 SFC values for all samples with a positive response. If no positive response was detected for any of the cytokines, the total cytokine response was assigned as 100 SFCs per 106 PBMCs. The number of predicted binding events of peptides to the 27 most common HLA-DR, HLA-DQ and HLA-DP class II allelic variants was analysed using the MHC-II Binding Prediction Results tool available at the Immune Epitope Database (IEDB)28. As suggested by the IEDB, we used the NetMHCIIpan-4.1 EL prediction method, and set the cut-off for a binding event as a predicted binding percentile score threshold of 20% or less18. Predicted binding strength was obtained using the NetMHCIIpan-4.1 EL prediction method, and is reported as a normalized rank value compared with peptides from 10,000 randomly selected peptides, with a small value indicating high affinity25. HLA restrictions were inferred using RATE, hosted by the IEDB25. In brief, RATE infers HLA restrictions by considering the presence or absence of a response to a given epitope as the biological outcome. Because spurious results can be observed for HLA alleles in linkage disequilibrium, peptide–HLA associations were validated using the NetMHCIIpan-4.1 EL method, with a percentile score threshold of 30% or less, to validate that the peptides were predicted to bind to their corresponding HLA allele. Cells were washed, counted and plated in a 96-well plate at a density of 1 × 106 cells per well. The dilutions of all antibodies used are provided in Supplementary Table 3. Stained cells were washed twice and resuspended in 100 µl PBS to be run on the Cytek Aurora System (Cytek). FCS files produced from the Cytek Aurora flow cytometer, using the SpectroFlo flow cytometry software (Cytek,v.3.3), were then analysed using FlowJo software (v.10.10.0, Tree Star). Antigen-specific cells were expanded for two weeks as described above. On day 14, cells were washed, counted, plated in a 96-well plate at a density of 1 × 106 cells per well, and restimulated with the C9orf72 megapool. After a two-hour incubation, cells were treated for an additional 4 h with GolgiPlug (BD) and GolgiStop (BD). Cells were then washed and stained with a mixture of the following antibodies: fixable viability dye eFluor 506 (Thermo Fisher Scientific), CD4-APC-eF780 (clone: RPA-T4, Thermo Fisher Scientific), CD3-AF700 (clone: UCHT1, BD) and CD8-BV650 (clone: RPA-T8, BioLegend). Cells were incubated for 30 min at 4 °C. Cells were then washed and fixed and permeabilized using the Cyto-Fast Fix/Perm buffer (BioLegend) for 20 min at room temperature. The dilutions of all antibodies used are provided in Supplementary Table 3. Stained cells were then washed twice and resuspended in 100 µl PBS to be run on the Fortessa X-20 flow cytometer (BD). The average proportion of each cytokine for each donor was determined by dividing the frequency of cells releasing each cytokine by the total number of cells releasing any of the three cytokines for each donor. Statistical analyses were performed and graphs were created using GraphPad Prism's descriptive statistics, two-tailed Mann–Whitney test, one-way ANOVA with Dunnett's multiple comparisons test, two-tailed Fisher exact test and Spearman's r test as applicable (GraphPad Prism, v.10). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data for all figures are provided with this paper. Source data are provided with this paper. No codes were used within this study. Kawamata, T., Akiyama, H., Yamada, T. & McGeer, P. L. Immunologic reactions in amyotrophic lateral sclerosis brain and spinal cord tissue. Rodrigues Lima-Junior, J., Sulzer, D., Lindestam Arlehamn, C. S. & Sette, A. The role of immune-mediated alterations and disorders in ALS disease. Appel, S. H., Beers, D. R. & Zhao, W. Amyotrophic lateral sclerosis is a systemic disease: peripheral contributions to inflammation-mediated neurodegeneration. Combined regulatory T-lymphocyte and IL-2 treatment Is safe, tolerable, and biologically active for 1 year in persons with amyotrophic lateral sclerosis. Murray, M. E. et al. Clinical and neuropathologic heterogeneity of c9FTD/ALS associated with hexanucleotide repeat expansion in C9ORF72. Sulzer, D. et al. T cells from patients with Parkinson's disease recognize α-synuclein peptides. Campisi, L. et al. Clonally expanded CD8 T cells characterize amyotrophic lateral sclerosis-4. Ramachandran, S. et al. Low T-cell reactivity to TDP-43 peptides in ALS. Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. Renton, A. E. et al. A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. Sultana, J. et al. C9orf72-associated dipeptide repeat expansions perturb ER–Golgi vesicular trafficking, inducing Golgi fragmentation and ER stress, in ALS/FTD. McCombe, P. A., Lee, J. D., Woodruff, T. M. & Henderson, R. D. The peripheral immune system and amyotrophic lateral sclerosis. Lindestam Arlehamn, C. S. et al. T-cell deficiency and hyperinflammatory monocyte responses associate with Mycobacterium avium complex lung disease. Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells. Williams, G. P. et al. PINK1 is a target of T cell responses in Parkinson's disease. The MegaPool approach to characterize adaptive CD4+ and CD8+ T cell responses. Lindestam Arlehamn, C. S. et al. α-Synuclein-specific T cell reactivity is associated with preclinical and early Parkinson's disease. Dan, J. M. et al. A cytokine-independent approach to identify antigen-specific human germinal center T follicular helper cells and rare antigen-specific CD4+ T cells in blood. Dunmire, S. K., Verghese, P. S. & Balfour, H. H. Primary Epstein–Barr virus infection. The diagnosis of dementia due to Alzheimer's disease: recommendations from the National Institute on Aging–Alzheimer's Association workgroups on diagnostic guidelines for Alzheimer's disease. & Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Accurate prediction of HLA class II antigen presentation across all loci using tailored data acquisition and refined machine learning. Greenbaum, J. et al. Functional classification of class II human leukocyte antigen (HLA) molecules reveals seven different supertypes and a surprising degree of repertoire sharing across supertypes. Hartmann, J. et al. SKA2 regulated hyperactive secretory autophagy drives neuroinflammation-induced neurodegeneration. & Holzbaur, E. L. F. Autophagic stress activates distinct compensatory secretory pathways in neurons. Westeneng, H. J. et al. Prognosis for patients with amyotrophic lateral sclerosis: development and validation of a personalised prediction model. & Ascherio, A. Epstein–Barr virus as a leading cause of multiple sclerosis: mechanisms and implications. Gate, D. et al. Clonally expanded CD8 T cells patrol the cerebrospinal fluid in Alzheimer's disease. Iguchi, Y. et al. Exosome secretion is a key pathway for clearance of pathological TDP-43. Sepúlveda, D., Cisternas-Olmedo, M., Arcos, J., Nassif, M. & Vidal, R. L. Contribution of autophagy–lysosomal pathway in the exosomal secretion of α-synuclein and its impact in the progression of Parkinson's disease. Calafatti, M., Cocozza, G., Limatola, C. & Garofalo, S. Microglial crosstalk with astrocytes and immune cells in amyotrophic lateral sclerosis. Identification of four novel T cell autoantigens and personal autoreactive profiles in multiple sclerosis. Yazdani, S. et al. T cell responses at diagnosis of amyotrophic lateral sclerosis predict disease progression. Oseroff, C. et al. Molecular determinants of T cell epitope recognition to the common Timothy grass allergen. Balendra, R. & Isaacs, A. M. C9orf72-mediated ALS and FTD: multiple pathways to disease. Cook, C. N. et al. C9orf72 poly(GR) aggregation induces TDP-43 proteinopathy. Linda, H., Hammarberg, H., Piehl, F., Khademi, M. & Olsson, T. Expression of MHC class I heavy chain and β2-microglobulin in rat brainstem motoneurons and nigral dopaminergic neurons. Coque, E. et al. Cytotoxic CD8+ T lymphocytes expressing ALS-causing SOD1 mutant selectively trigger death of spinal motoneurons. Hobson, B. D. et al. Conserved and cell type-specific transcriptional responses to IFN-γ in the ventral midbrain. Jakel, S. et al. Altered human oligodendrocyte heterogeneity in multiple sclerosis. Beers, D. R. et al. Neuroinflammation modulates distinct regional and temporal clinical responses in ALS mice. & Ziemssen, T. Peripheral proinflammatory Th1/Th17 immune cell shift is linked to disease severity in amyotrophic lateral sclerosis. Rolfes, L. et al. Amyotrophic lateral sclerosis patients show increased peripheral and intrathecal T-cell activation. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis - Spectral data. Michaelis, T., Johansson, E., Sulzer, D. & Sette, A. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis - ICS data Screen 1. Michaelis, T., Johansson, E., Sulzer, D. & Sette, A. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis - ICS data Screen 2. Michaelis, T., Johansson, E., Sulzer, D. & Sette, A. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis - ICS data Screen 3. Michaelis, T., Johansson, E., Sulzer, D. & Sette, A. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis - ICS data Screen 4. This work was supported by La Jolla Institute for Immunology, Kyowa Kirin North America, the Swedish Research Council (salary for E.J., grant reference 2024-00175) and the Freedom Together Foundation (to D.S. The contributions of the NIH authors were made as part of their official duties as NIH federal employees, are in compliance with agency policy requirements, and are considered works of the US Government. These authors contributed equally: Cecilia S. Lindestam Arlehamn, Emil Johansson Center for Autoimmunity and Inflammation, La Jolla Institute for Immunology, La Jolla, CA, USA Tanner Michaelis, Cecilia S. Lindestam Arlehamn, Emil Johansson, April Frazier, John Sidney & Alessandro Sette Center for Vaccine Research, Department of Infectious Disease Immunology, Statens Serum Institut, Copenhagen, Denmark Sean M. Healey and AMG Center for ALS, Department of Neurology, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Department of Neurology, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Department of Neurology, Emory University, Atlanta, GA, USA National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Vanderbilt University School of Medicine, Nashville, TN, USA Department of Neurology, Columbia University Irving Medical Center, New York, NY, USA Departments of Psychiatry, Neurology and Pharmacology, Columbia University, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar recruited participants and performed clinical evaluations. All authors read, edited and approved the manuscript before submission. Correspondence to David Sulzer or Alessandro Sette. The authors have declared that no competing interests exist. Nature thanks Aaron Gitler, Lawrence Stern and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Gating strategies for broad immunophenotyping of T cell subsets in PBMCs. Proportion of IFNγ (a) and IL-5 (b) releasing T cells among all cytokine-releasing cells in response to C9orf72 in healthy controls (HC; n = 28) and donors with ALS (n = 28). Total (d), IFNγ- (e), IL-5- (f), and IL-10-mediated (g) T cell responses towards C9orf72 in ALS, HC, Parkinson's disease (PD; n = 15), and Alzheimer's disease (AD; n = 15) cohorts (ALS (n = 28) and HC (n = 28) are the same as in Fig. Proportion of IFNγ- (h) and IL-5-releasing (i) T cells among all cytokine-releasing cells, and ratio of the number of cells releasing IFNγ and IL-5 (j), in response to C9orf72 in HC, ALS, AD, and PD cohorts. P-values from two-tailed Mann–Whitney tests from comparisons of two groups, and from Kruskal-Wallis tests followed by Dunn's multiple comparisons test for comparison of four groups. Only significant p-values are shown, the complete list of p-values can be found in the Source Data file. Geometric mean ± 95% confidence interval is shown. Gating strategies for the identification of cytokine-expressing T cells 24 h post C9orf72 peptide pool restimulation of 2-week cell culture, in healthy controls (HC; n = 30) and donors with ALS (n = 29). Frequency of CD4 and CD8 T cells in HC (a) and ALS (b), as well as cytokine-expressing CD4 and CD8 T cells in HC individuals (n = 15) (c) and patients with ALS (n = 15) (d) 6 h post C9orf72 peptide pool restimulation of 2-week culture. Cytokine expression in CD4 (e) and CD8 (f) T cells in patients with ALS and HC individuals after C9orf72 peptide pool restimulation. (g) Overall sum of % CD4 and CD8 responses by the different cytokines. P-values are from two-tailed Mann–Whitney test, and geometric mean ± 95% confidence interval and fold change (FC) values are shown. (a) Correlation between response frequency and number of predicted binding events for each peptide (n = 95) to the 27 most common HLA II allelic variants. Correlation is indicated by Spearman r and p-value. (b) Comparison of median binding rank to the 27 most common HLA II allelic variants for peptides recognized (n = 72) or not recognized (n = 23) in 28 patients with ALS. The most recognized peptide, aa56–DGEITFLANHTLNGE, is highlighted in red. P-value from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence interval are shown. (c) Predicted binding events for each peptide from C9orf72, TDP-43, and SOD1, to the 27 most common HLA II allelic variants. P-values from two-tailed Kruskal-Wallis tests followed by Dunn's multiple comparison test, and median ± 95% confidence interval are shown. (a) Difference in magnitude of response between ALS samples from male (n = 24) and female (n = 16) individuals. P-value from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence interval are shown. Correlation between C9orf72 response and age (n = 40) (b), time since onset (n = 31) (c), and time since diagnosis (n = 40) (d) against the magnitude of C9orf72-specific reactivity. Correlation between ALSFRS-R score and Total (e), IFNγ- (f), IL-5- (g), and IL-10- (h) mediated C9orf72 responses in donors with ALS (n = 33). Correlation is indicated by Spearman r and p-value. Total (i), IFNγ- (j), IL-5- (k), and IL-10-mediated (l) T cell responses towards C9orf72 for participants with ALS carrying the C9orf72 mutation (n = 3), compared to the remaining participants with ALS, either carrying other mutations (ataxin-2, CHCHD10, FIG4, NEK1, SOD1, and TBK1) (n = 12), or carrying none of those mutations (n = 10), or those for which the genetic data were not available (unknown; n = 15). P-values from two-tailed Mann–Whitney tests and geometric mean ± 95% confidence interval are shown for histograms. Supplementary Table 2 contains HLA type information for ALS donors included in peptide response deconvolution experiments, related to Fig. Supplementary Table 3 lists the antibodies used in the study, and how they were diluted for each type of experiment. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Michaelis, T., Lindestam Arlehamn, C.S., Johansson, E. et al. Autoimmune response to C9orf72 protein in amyotrophic lateral sclerosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09542-6'>Polygenic and developmental profiles of autism differ by age at diagnosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 15:05:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Although autism has historically been conceptualized as a condition that emerges in early childhood1,2, many autistic people are diagnosed later in life3,4,5. Using longitudinal data from four independent birth cohorts, we demonstrate that two different socioemotional and behavioural trajectories are associated with age at diagnosis. In independent cohorts of autistic individuals, common genetic variants account for approximately 11% of the variance in age at autism diagnosis, similar to the contribution of individual sociodemographic and clinical factors, which typically explain less than 15% of this variance. We further demonstrate that the polygenic architecture of autism can be broken down into two modestly genetically correlated (rg = 0.38, s.e. One of these factors is associated with earlier autism diagnosis and lower social and communication abilities in early childhood, but is only moderately genetically correlated with attention deficit–hyperactivity disorder (ADHD) and mental-health conditions. Our findings have important implications for how we conceptualize autism and provide a model to explain some of the diversity found in autism. Ever since its earliest descriptions in the 1940s1,2, autism has been thought of as a condition that emerges in early childhood. However, a greater proportion of autistic individuals are now receiving an autism diagnosis from mid-childhood onwards than in early childhood3,4,5. Supporting this, some studies have demonstrated that a subset of children who do not initially meet the criteria for an autism diagnosis receive a diagnosis later7,10,11,12,13,14. Later autism diagnosis is associated with elevated co-occurring mental-health conditions15,16, highlighting the need to understand why some autistic people are not diagnosed until later in life. Several social, demographic and clinical factors have been linked to age at autism diagnosis17. However, previous studies have shown that individual clinical and sociodemographic factors explain only a small proportion (typically less than 15%) of the variance in age at autism diagnosis (Extended Data Fig. This indicates that other factors contribute to age at autism diagnosis. One of these additional factors could be genetic differences between autistic individuals. Despite the relatively high heritability of autism18, the role of genetics in age at autism diagnosis has not to our knowledge been previously studied. Two theoretical models can explain how genetics affect age at autism diagnosis. In the first model, autism has a single polygenic aetiology, with the same set of genetic variants underlying autism, regardless of age at diagnosis (the ‘unitary model'; Extended Data Fig. In this model, later-diagnosed autism may have subtle clinical features that are harder to recognize early in life, so individuals do not cross a diagnostic threshold earlier in life, perhaps because they have a lower genetic predisposition to autism. As these people get older, environmental factors may alter their clinical features, eventually bringing individuals above the clinical threshold to receive an autism diagnosis later in life. An alternative model is that earlier- and later-diagnosed autism have different underlying developmental trajectories and polygenic aetiologies (the ‘developmental model'; Extended Data Fig. This model aligns with existing evidence that the genetic influences on traits related to autism vary across development19,20,21. This model does not preclude a role for environmental factors influencing when someone receives an autism diagnosis but implies that different sets of genetic variants are associated with earlier- and later-diagnosed autism. Here we examined the evidence for these two models through four linked aims (Extended Data Fig. First, we investigated whether the trajectories of socioemotional and behavioural development are associated with age at autism diagnosis in birth cohorts. Although variable developmental trajectories have been observed among autistic individuals and their younger siblings22, it is unclear whether these differences in trajectories are associated with age at diagnosis. Second, we estimated the proportion of variance in age at autism diagnosis that is explained by common single nucleotide polymorphisms (SNP-based heritability). We then tested whether this is attenuated by a range of clinical and demographic factors, as predicted by the unitary model. Third, we investigated whether different polygenic factors are associated with earlier and later autism diagnosis, as predicted by the developmental model. Finally, we estimated the genetic correlation between the autism polygenic factors related to age at diagnosis and other mental-health and developmental phenotypes. In Aim 1, we investigated whether autistic individuals have varying socioemotional and behavioural trajectories, and whether these are associated with age at autism diagnosis in three birth cohorts (n = 89 to 188 autistic individuals with recorded age at diagnosis between 5 years and 17 years). These are the Millennium Cohort Study (MCS, participants born in 2000) and the Longitudinal Study of Australian Children: Kindergarten cohort (LSAC-K, 1999) and Birth cohort (LSAC-B, 2003) (Extended Data Fig. All three cohorts collected longitudinal information on socioemotional and behavioural development using the carer-reported Strengths and Difficulties Questionnaire (SDQ)23. The SDQ is widely used, has excellent psychometric properties24,25,26 and is largely invariant across age, sex and different populations27,28,29, indicating that it is measuring the same latent trait across these demographic variables. Furthermore, the SDQ is moderately correlated with autism-specific measures30,31,32,33, although it does not capture all of the core diagnostic features of autism. Because not all cohorts recorded the exact age when children received their autism diagnosis, we used the child's age during the study data collection when carers first reported the diagnosis as an approximation of age at autism diagnosis. To identify latent trajectories, we used growth mixture models of the SDQ total difficulties and subscale scores among autistic individuals in all three cohorts. Growth mixture models do not require grouping based on an a priori hypothesis but can identify latent subgroups based on longitudinal differences in SDQ scores. Across all three birth cohorts, growth mixture modelling identified a two-trajectory model as being optimal for SDQ total difficulties and most subscale scores (Supplementary Table 3 and Supplementary Figs. The first latent trajectory was characterized by difficulties in early childhood that remained stable or modestly attenuated in adolescence (termed ‘early childhood emergent latent trajectory'). The second latent trajectory was characterized by fewer difficulties in early childhood that increased in late childhood and adolescence (termed ‘late childhood emergent latent trajectory') (Fig. a–c, Longitudinal growth mixture models of SDQ total scores in autistic individuals, demonstrating the presence of two groups in the MCS (a), LSAC-B (b) and LSAC-K (c) cohorts. d–f, Stacked bar charts show the proportion of individuals who had been diagnosed as autistic at specific ages, categorized by membership in the latent trajectories identified from the growth mixture models in MCS (d), LSAC-B (e), and LSAC-K (f) cohorts. Autistic individuals in the early childhood emergent latent trajectory were more likely to be diagnosed as autistic in childhood than autistic individuals in the late childhood emergent latent trajectory in MCS (P = 1.42 × 10−4, χ2 test) and LSAC-B (P = 2.24 × 10−2, χ2 test) (Fig. Sensitivity analyses in MCS confirmed the robustness of the two latent trajectories and their association with age at diagnosis among autistic children. We also obtained consistent results when restricting the analyses to only male individuals (n = 136; Supplementary Tables 3 and 4), indicating that these results were not driven by sex differences in age of diagnosis. To assess the specificity of this result to autism, we tested whether similar latent trajectories were also observed in children with ADHD but not autism (n = 89, imputed n = 325) in MCS using growth mixture models. Two latent SDQ trajectory classes emerged, but these were not significantly linked to age of ADHD diagnosis, except for the SDQ hyperactivity/inattention and conduct problems subscales in the imputed sample (Supplementary Figs. 7 and 8 and Supplementary Table 6), indicating that the findings are relatively specific to autism, rather than to neurodevelopmental conditions more broadly. Although female individuals receive an autism diagnosis later than male individuals on average34, in all three cohorts, the sex ratio was similar between the two latent trajectories (Supplementary Table 4), possibly because of their relatively small sample sizes. Individuals in the late childhood emergent latent trajectories were more likely to report mental-health conditions (Supplementary Table 7), consistent with previous epidemiological observations among later-diagnosed autistic individuals15,16. Using multiple regression models, we then examined the extent to which these two latent trajectories contributed to differences in age at autism diagnosis over and above sociodemographic and cognitive characteristics (Supplementary Table 8). In these models, SDQ latent trajectories explained 11.7% (LSAC-B) to 30.3% (MCS) of the variance in age of autism diagnosis. By contrast, sociodemographic variables explained 4.8% (LSAC-B) to 5.5% (MCS) of the total variance across cohorts, consistent with previous reports (Extended Data Fig. In the imputed MCS sample (n = 623; Supplementary Table 5 and Supplementary Note 2), the SDQ latent trajectories and sociodemographic variables explained 56.6% and 3.2% of the variance, respectively. The associations between different SDQ trajectories and age at autism diagnosis were also supported by latent growth curve models fitted on earlier- and later-diagnosed autistic individuals (Supplementary Note 5, Supplementary Table 9 and Supplementary Figs. These results confirm that the association between SDQ trajectories and age at autism diagnosis is robust to methodological choices The above analyses demonstrate that variation in socioemotional and behavioural trajectories, measured using the SDQ, is associated with age at autism diagnosis. Previous research has demonstrated that developmental variation traits related to autism are partly explained by genetic factors19,20,35,36,37,38. A corollary of this is that genetic factors may also be associated with age at autism diagnosis. Subsequently, in Aim 2, we tested whether age at autism diagnosis is heritable in two large cohorts of autistic individuals using genetic data and information on age at autism diagnosis. This includes: first, the Danish-based iPSYCH cohort (ntotal = 18,965), a population-based sample derived from the Danish national registries that includes autistic individuals; and second, the US-based cohort of autistic individuals (SPARK39; ntotal = 28,165; Extended Data Figs. 5 and 6), which recruits families with at least one autistic individual through online platforms and medical centres across the United States. In SPARK, we conducted initial analyses in a discovery subset of 18,809 autistic individuals (SPARK Discovery), and replicated key findings in a second sample of 9,356 autistic individuals that became available only after the initial analyses were completed (SPARK Replication). SPARK and iPSYCH differed in the diagnostic classification system (iPSYCH: International Classification of Diseases (ICD) and SPARK: Diagnostic and Statistical Manual of Mental Disorders (DSM)) and median age at diagnosis (iPSYCH, median = 10 years, median absolute deviation = 4 years; SPARK, median = 4 years, median absolute deviation = 2.7 years). We conducted a genome-wide association study (GWAS) in iPSYCH and across both the Discovery and Replication samples of SPARK, using age at autism diagnosis as a quantitative trait. In all three samples, we identified significant and consistent SNP-based heritability of approximately 11% for age at autism diagnosis (Fig. This is larger than, or similar to, the variance explained by several other clinical and sociodemographic factors tested in SPARK (Extended Data Fig. a, SNP-based heritability (h2) for age at autism diagnosis in the SPARK cohorts, calculated using single-component genome-wide complex trait analysis with a genomic-relatedness-based restricted maximum-likelihood approach (GCTA-GREML) for the SPARK Discovery cohort (orange dashed line, n = 16,786), SPARK Replication cohort (purple dashed line, n = 8,558) and a meta-analysis of the two (light blue solid line, n = 25,344), and iPSYCH, calculated using linkage disequilibrium score regression coefficient (LDSC) (solid green line, n = 18,965). b, SNP-based heritability (GCTA-GREML) in the SPARK cohorts after accounting for various clinical and sociodemographic factors. The x axis has been truncated at 0 and 0.25. In a and b, central points represent SNP-based heritability estimates and error bars indicate 95% confidence intervals. Sample sizes for b are provided in Supplementary Table 10. PC, genetic principal component; RBS-R, Repetitive Behavior Scale-Revised; SCQ, Social Communication Questionnaire; SES, socioeconomic status. In contrast to the effect of common genetic variants, in a subsample of SPARK with available data for both parents and their autistic child (n = 6,206 trios), we observed no association between age at autism diagnosis and rare de novo variants or inherited protein truncating or missense variants in highly constrained genes (Supplementary Table 11). This may possibly be the result of low statistical power or reflect later autism diagnosis in some carriers of de novo mutations owing to diagnostic overshadowing by co-occurring intellectual disability or global developmental delay40,41. We next tested whether this SNP-based heritability of age at autism diagnosis is consistent with either of the two theoretical models outlined earlier. The unitary model assumes that later diagnosis reflects subtle or less-severe clinical features, so the SNP-based heritability of age at autism diagnosis may simply reflect the severity of autism features. Alternatively, the SNP-based heritability may reflect additional genetic influences associated with co-occurring developmental delays, developmental regression or intellectual disability, which may lead to an earlier diagnosis. It may also reflect the heritable component of parental socioeconomic status and neighbourhood deprivation. which are proxies for parental awareness and healthcare access that affect diagnostic timing. Controlling for any of these measures should attenuate the SNP-based heritability. By contrast, the developmental model assumes that SNP-based heritability of age at autism diagnosis reflects a mixture of different polygenic factors that are correlated with age at diagnosis but are independent of these covariates. Under this model, a significant SNP-based heritability should persist after controlling for clinical, developmental and sociodemographic measures. In line with the developmental model, we found that the SNP-based heritability did not significantly attenuate after controlling for parental sociodemographic measures, clinical features or co-occurring developmental delays and conditions (Fig. This is inconsistent with the unitary model, although imperfect and incomplete measurement of clinical and developmental phenotypes may limit this conclusion. A second prediction of the unitary model is that earlier diagnosis is associated with a greater polygenic propensity for autism compared with later diagnosis (Extended Data Fig. In this model, all autistic individuals would have a higher polygenic propensity for autism compared with non-autistic controls. This would result in negative genetic correlations between GWAS of age at autism diagnosis and GWAS of autism, with the magnitude of this negative correlation decreasing as the median age at diagnosis in the autism GWAS samples increases. We tested this prediction using 13 different but partly overlapping autism GWASs, including six GWASs stratified by age at diagnosis and two GWASs stratified by sex (Box 1 and Fig. The genetic correlation between age at autism diagnosis and different autism GWASs varied systematically, becoming increasingly positive as the median age at diagnosis increased (Fig. However, contrary to the expectation under the unitary model, we observed positive genetic correlations between age at autism diagnosis and autism GWAS comprising later-diagnosed autistic individuals. These findings support the existence of different genetic architectures across diagnostic age groups, aligning with the developmental model. Circle size indicates the number of autistic individuals (cases) in the GWAS, and exact sample sizes are provided in Box 1. Beige circles represent GWAS unstratified by age at diagnosis; red circles represent GWAS stratified by age at diagnosis. Lighter (more transparent) circles indicate studies with no information about age at autism diagnosis, and the median ages have been inferred from other available information (PGC-2017 (ref. Right, genetic correlations with age at autism diagnosis for both SPARK (blue, meta-analysis, n = 28,165) and iPSYCH (green, n = 18,965) datasets, with error bars representing 95% confidence intervals. Furthermore, the male- and female-stratified autism GWAS from iPSYCH had a similar genetic correlation with age at autism diagnosis. This reflects the fact that these two age at diagnosis GWASs are only moderately genetically correlated with each other (genetic correlation (rg) = 0.51, standard error (s.e.) = 0.19, P = 7.56 × 10−3), which may be due to the different recruitment strategies and resulting differences in the median age at autism diagnosis in the two cohorts (Supplementary Note 6). Autism GWAS not stratified by age at diagnosis SPARK (Matoba et al.49): case-pseudocontrol design (4,535 pairs) with family-based ascertainment across the United States. FinnGen (Data Release r10): population-based sample from Finland (646 cases and 301,879 controls). Most of the cases met the diagnostic criteria for autism under DSM-IV-TR/ICD-10 or earlier (onset of features before age 3) after screening using the Autism Diagnostic Observation Schedule and the Autism Diagnostic Interview-Revised), making this a clinically well-characterized cohort. Although age at diagnosis is unavailable, most of the participants were recruited as trios through medical or research centres in the United States. iPSYCHmales50: males-only subset of iPSYCH (15,025 autistic and 19,763 controls). iPSYCHfemales50: females-only subset of iPSYCH (4,845 autistic and 19,315 controls). Age at diagnosis for PGC is unavailable, so we calculated an estimated age by weighing the median age at diagnosis of iPSYCH with the estimated median age in PGC by their respective sample size. Autism GWAS stratified by age at diagnosis SPARK (using unaffected family members as controls), meta-analysed from the Discovery and Replication subsets. This cut-off period was chosen to reflect the cut-off used in the latent growth curve models and represents a time window characterized by the onset of puberty, the transition from primary to secondary school and an increase in the number of autistic girls being diagnosed. Diagnosed before age 6 (SPARKbefore6): 21,435 autistic individuals; categorized as ‘early-diagnosed' based on previous research8; median age at diagnosis = 3 (1.23) years. These findings indicate that the age at autism diagnosis reflects a mixture of different age-dependent polygenic traits (developmental model), rather than a single polygenic trait (unitary model). In Aim 3, we tested this by estimating genetic correlations among the 13 autism GWASs (Box 1). We observed a gradient in the genetic correlations related to the similarity in median age at diagnosis between cohorts. Cohorts with the most similar median ages at diagnosis (differing by a maximum of 2 years) showed the highest genetic correlations (rg = 1, s.e. = 0.04), and correlations progressively decreased as age differences increased. Asterisks indicate significant genetic correlations after Benjamini–Yekutieli adjustment. For both panels, GWASs have been ordered based on hierarchical clustering of the genetic correlations. b, Structural equation model illustrating the two-correlated genetic-factor models for autism, using six minimally overlapping autism GWAS datasets. Residual variances for each GWAS dataset are represented using a two-headed arrow connecting the residual variable (u) to itself. Hierarchical clustering of the genetic correlations identified two broad, overlapping clusters that differed by age at autism diagnosis. One cluster comprised GWAS of autism in cohorts with predominantly childhood-diagnosed individuals, and the other comprised GWAS of autism in cohorts with a large fraction of individuals diagnosed in adolescence or later, consistent with predictions from the developmental model. We formally tested this by modelling the genetic covariance using the structural equation models in GenomicSEM42, testing six theoretical models (Supplementary Table 14). We used six minimally overlapping GWASs for autism with wide variation in age at autism diagnosis among those listed in Box 1. We found that a correlated two-factor model was the most parsimonious and fit the data best (Akaike information criterion, 38.64; confirmatory fit index, 0.99; standardized root mean residual, 0.08; Fig. Factor 1 (earlier-diagnosed autism factor) was defined by the GWAS with predominantly early childhood-diagnosed individuals (PGC-2017, SPARKbefore6, with a median age at diagnosis of 3). Factor 2 (later-diagnosed autism factor) was defined primarily by GWASs with adolescent- or adult-diagnosed individuals (iPSYCHafter10, FinnGen and SPARKafter10). The cross-loading of iPSYCHbefore9 (median age at diagnosis of around 5.7) indicates that factor 2 may impact behaviours in mid-to-late childhood as well. The two factors had a small genetic correlation (rg = 0.38, s.e. = 0.06). The later-diagnosed autism factor was positively genetically correlated only with age at autism diagnosis from SPARK. Genetic correlation with the autism GWAS stratified by sex showed that both autism factors had stronger genetic correlations with autism in male than in female individuals, with a larger difference for the earlier-diagnosed autism factor (Fig. 5), consistent with established sex differences in age at autism diagnosis. Sample sizes are shown in Supplementary Table 18. Further analyses using polygenic scores confirmed that the association between the two polygenic autism factors and age at autism diagnosis is not due to several confounding factors using within-family approaches, nor to differences in clinical and demographic factors, and co-occurring developmental conditions (Supplementary Note 7 and Supplementary Tables 15–17). Taken together, the above findings demonstrate that earlier- and later-diagnosed autism have different polygenic aetiologies, supporting the developmental model. The above analyses indicate that there are at least two polygenic factors associated with age at autism diagnosis. Because later-diagnosed autistic individuals have higher rates of mental-health conditions15,16, we proposed that this might partly be because earlier- and later-diagnosed autism factors have differing genetic correlations with mental health and cognitive phenotypes. In Aim 4, we investigated this hypothesis using genetic correlation analyses. The earlier-diagnosed autism factor (factor 1) had a low (rg of around 0.1–0.2) but significant genetic correlation with educational attainment, cognitive aptitude, ADHD and various mental-health and related conditions (Fig 5 and Supplementary Table 18). The later-diagnosed autism factor (factor 2) showed a statistically similar genetic correlation with educational attainment but significantly higher genetic correlations (rg of around 0.5–0.7) with ADHD and a range of other mental-health and related conditions, including depression, post-traumatic stress disorder (PTSD), childhood maltreatment and self-harm. After accounting for genetic effects on ADHD, we saw an attenuated but significant moderate genetic correlation between later-diagnosed autism (factor 2) and mental-health conditions, indicating that shared genetics with ADHD do not fully explain the elevated correlation between later-diagnosed autism and mental-health phenotypes (Supplementary Table 18 and Supplementary Fig. Sensitivity analyses using age at diagnosis-stratified GWASs from iPSYCH and SPARK yielded largely consistent genetic correlation results, indicating that these results are not due to cohort differences (Supplementary Table 19 and Extended Data Fig. The higher genetic correlation between later-diagnosed autism and other mental-health conditions may indicate diagnostic misclassification (in which individuals with other conditions incorrectly receive an autism diagnosis) or diagnostic overshadowing (where the presence of co-occurring mental-health conditions can delay an autism diagnosis). However, decomposition of the autism genetic signal using genomicSEM indicated that later-diagnosed autism cannot be entirely attributed to the polygenic effects of earlier-diagnosed autism and six other mental-health conditions tested: schizophrenia, ADHD, anorexia nervosa, depression, bipolar disorder and PTSD (Supplementary Note 8). Thus, the later-diagnosed autism genetic factor does not represent the additive genetic effects of earlier-diagnosed autism and mental-health conditions. Finally, we investigated whether the two autism polygenic factors related to age at diagnosis differed in their associations with developmental traits. Cross-sectionally, these genetic factors showed few significant differences in genetic correlation or polygenic score association with developmental phenotypes measured at age 3 or earlier. The exceptions were age at onset of walking43 and expressive vocabulary at an age of 2–3 years44, with which the earlier-diagnosed autism factor was positively genetically correlated but the later-diagnosed factor was not (Supplementary Note 9 and Supplementary Tables 20–22). However, longitudinal polygenic score analyses across two birth cohorts revealed differential genetic effects of earlier- versus later-diagnosed autism factors on SDQ total difficulties scores over time (Supplementary Note 9 and Supplementary Table 23). The results of this study indicate that earlier- and later-diagnosed autism are associated with different developmental trajectories, and are only moderately genetically correlated with each other. This possible framework provides one axis of heterogeneity to describe the widely acknowledged clinical and genetic heterogeneity within autism that thus far has been challenging to identify. This finding of two or more developmentally variable polygenic latent traits for autism is robust to various observed clinical and demographic factors (Supplementary Note 10), including sex and intellectual disability. Consistent with the wider literature on developmental variation in autism22, our analyses of socioemotional and behavioural trajectories across multiple birth cohorts converge with the genetic findings: polygenic scores for earlier- and later-diagnosed autism have different associations with developmental changes in SDQ total difficulties scores (Supplementary Note 9). These results indicate that the timing of autism diagnosis may partly reflect aetiologically different developmental pathways, rather than purely environmental or diagnostic factors. Our findings are consistent with the wider literature that demonstrates that genetic influences on traits related to autism vary across development in the general population19,20. This two-polygenic-trait genetic model provides one framework to understand genetic heterogeneity in autism and the varying patterns of genetic correlations between different GWASs of autism and other phenotypes. For example, previous GWASs of autism (including PGC-2017 (ref. 45)) found limited genetic correlation with ADHD, contrary to findings from more-recent autism GWAS (such as Grove et al.46). We show that this is explained by the different average age at diagnosis across these GWASs (Box 1 and Fig. 3), because the genetic correlation between autism and ADHD increases with later age at autism diagnosis (Fig. These findings were confirmed using within-family analyses that demonstrated over-transmission of ADHD polygenic scores, mainly to individuals with a later autism diagnosis (Supplementary Table 24). Both the later-diagnosed genetic autism factor (Fig. 4) and the late childhood emergent latent trajectory of SDQ total difficulties scores (Fig. 1) are associated with greater mental-health problems (Fig. This indicates that epidemiological findings of greater mental-health difficulties among later-diagnosed autistic individuals15,16 may be partly explained by the developmental model of autism. Given that autistic female individuals are, on average, diagnosed later in life, research that investigates sex and gender differences in both autism and co-occurring conditions16,47 needs to account for genetic confounding associated with age at autism diagnosis. Findings that may seem to reflect sex differences may also partly reflect differences associated with age at diagnosis. For example, the higher prevalence of mental-health problems in autistic female individuals16,47 compared with male individuals attenuates when restricting to autistic individuals diagnosed before age the age of 5 (ref. These findings must be interpreted considering several limitations. First, the SNP-based heritability for age at autism diagnosis is only about 11% (Fig. 2), and other observed developmental and demographic factors typically explain less than 15% of the variance (Extended Data Fig. We find that the genetic effects on age at autism diagnosis are not mediated by several of these measured developmental and demographic factors, but we acknowledge that there may be several unmeasured factors that may mediate the genetic effects. Furthermore, the substantial variation across the datasets explored highlights that age at autism diagnosis is immensely complex and varies across geography and time. Local cultural factors, access to health care, gender bias, stigma, ethnicity and camouflaging probably have an effect on who receives a diagnosis and when. 1) were built using only the SDQ, which measures a wide range of parent-reported neurodevelopmental and mental-health traits. Although the SDQ is correlated with an autism diagnosis, it does not fully capture core autistic traits, and other measures of autistic traits were not available in the birth cohorts. Third, it is likely that other dimensions contribute to the genetic heterogeneity in autism. Fourth, we use earlier- and later-diagnosed autism as relative terms, reflecting that developmental and polygenic differences represent a gradient (Fig. 10), rather than being discrete categories, and because there is no consensus on age thresholds for early versus late diagnosis48. Fifth, autism diagnoses in the birth cohorts used in the current study rely on community-based carer or self-reporting, rather than standardized clinical assessments. As such, there may be varying delays between the emergence of autistic features and a formal autism diagnosis. However, our findings can guide future research using longitudinal cohorts, and particularly sibling studies, that systematically track the emergence of autism features over time. Finally, our genetic analyses focused on common genetic variants in genetically inferred European ancestries, because we had only limited GWAS data from other populations, highlighting the need for future research to examine the transferability of these findings across diverse genetic ancestries. In conclusion, we find that the developmental trajectories and polygenic architecture of autism varies with age at diagnosis. These findings partly explain the varying genetic correlations among the different GWASs of autism and between autism and various mental-health conditions. These findings provide further support for the hypothesis that the umbrella term ‘autism' describes multiple phenomena with differing aetiologies, developmental trajectories and correlations with mental-health conditions. These findings have implications for how we conceptualize neurodevelopment more broadly, and for understanding diagnosis, sex and gender differences, and co-occurring health profiles in autism. We use the term autistic and non-autistic to refer to people with and without an autism diagnosis51. This search resulted in more than 1,700 studies. A manual review identified 184 studies that investigated factors associated with age at autism diagnosis. We focused on the variance explained by sociodemographic factors (sex, reported race, household income, mother's education and father's education), cognitive and developmental factors (reported IQ score, reported intellectual disability, age at walking independently, age at first words, language regression and other regression) and autism severity (scores on the Social Communication Questionnaire and Repetitive Behaviour Scale-Revised). After excluding individuals with missing data, we quantified the variance explained by these factors using relative importance analysis (see method in ref. 52) for 5,773 autistic individuals diagnosed before the age of 22. Thereafter, analyses were done using the relaimpo (v.2.2-7) package in R, which allowed us to examine the contributions of all variables simultaneously53. Further details about the cohorts are provided in Supplementary Note 1. We used data from MCS, LSAC-B and LSAC-K for the growth mixture models. We did not use data from GUI for growth mixture models because there were only three time points, which is not enough to identify two or more trajectories. All four cohorts were used for latent growth curve models. As indicated in Supplementary Table 2, these cohorts were selected because they were longitudinal in nature, were nationally representative and included key data on behavioural profiles and neurodevelopmental diagnosis. These overlapping features across datasets allowed for cross-country comparisons and generalization57. In all cohorts, across multiple sweeps, the main carer was asked whether the participant had a diagnosis of autism (Extended Data Fig. For age at diagnosis, we used the age at the sweep when carers first reported their child's autism diagnosis in every cohort, to maximize sample sizes and ensure consistency across cohorts for effective comparisons. Although the specific age at diagnosis was provided for LSAC-B and LSAC-K, we opted not to use this, because we identified errors in some reports in which months and years of diagnosis were swapped or not reported. For our primary analyses, we included a narrowly defined sample of children with consistently reported autism diagnoses by primary and proxy carers (when both were available) and no other reported neurodevelopmental diagnosis (particularly ADHD). To assess the generalizability of our results and increase the sample size, we then expanded the sample to include all children with any reported diagnosis of autism (results are reported in Supplementary Note 3). This expanded sample included cases regardless of whether the diagnoses were consistent across sweeps or carers, and included those with co-occurring ADHD. Furthermore, we imputed the independent variables and covariates for autistic individuals with missing information, as detailed below (more details are in Supplementary Note 2 and Supplementary Note 3). Finally, to assess the specificity of the trajectories for autism, we conducted analyses among children who had a consistent ADHD diagnosis but no diagnosis of autism. We used the SDQ to obtain social, emotional and behavioural profiles of participants, with repeated measures from 3 years to 18 years across cohorts. SDQ comprises 25 statements that carers were asked to rate on a three-point Likert scale (‘not true', ‘somewhat true' and ‘certainly true') based on the child's symptoms or behaviours over the past six months. There were five subscales, each containing five items, which assessed emotional symptoms, conduct problems, hyperactivity−inattention, peer relationship problems and prosocial behaviours23. The first four subscales assessed difficulties, and their combined total score ranged from 0 to 40, with higher scores indicating greater difficulties. The fifth subscale (prosocial behaviours) represented strengths and ranged from 0 to 10, with higher scores indicating more prosocial behaviours. We analysed the total score and each subscale separately. The SDQ demonstrates good test–retest reliability and criterion validity across countries24,25,26. Its five-factor structure (each subscale as a factor) has shown consistency and invariance across age, sex and ethnic background24,28. Only children with complete data of SDQ across all sweeps were included in the analyses, except for the imputation analyses. Sociodemographic measures were included as covariates to account for their impact on age at diagnosis in each cohort (Supplementary Table 25). Specific measures and available information vary across cohorts, but we generally included sex, ethnic background, maternal age at delivery, child's cognitive aptitude, household SES and deprivation level of the living area, to account for factors that may affect the age when someone received an autism diagnosis59,60. Only subsets of children in the complete-SDQ samples, with complete data for these sociodemographic factors, were included in the analyses. For MCS, although various census classifications for ethnic groups were available, we opted to use a binary indicator to identify non-white ethnic minorities. Ethnicity data were not collected in either LSAC cohort. Instead, visible ethnic minority status was determined mainly by parental country of birth and the language(s) spoken at home61. Maternal age at delivery was collected only for MCS. For other cohorts, we used maternal age (in years) at the first sweep of data collection to reflect the variation in maternal age at delivery. For MCS, we identified multiple variables linked to cognitive ability, SES and area deprivation. Similarly, for LSAC-B, we identified multiple measures linked to SES, although there were no measures with sufficient sample size linked to cognitive ability or area deprivation. Subsequently, we conducted principal component analysis (PCA) for cognitive abilities, SES and area deprivation in MCS, and for SES in LSAC-B. PCA was done using a wide range of measures collected across sweeps (Supplementary Table 25), with one variable excluded from any pair with a correlation coefficient greater than 0.70 to address multicollinearity. By contrast, subsequent components contributed substantially less, supporting the use of the respective PC1 as the summary measure for cognitive ability, SES and area deprivation (Supplementary Table 25). No autistic children in the MCS or LSAC cohorts who had measures of cognitive aptitude met the criteria for intellectual disability, probably because of participation bias. All PCA analyses were done in R using the prcomp() function62. We used two methods to model the longitudinal trajectories of SDQ total and subscale scores. First, we used growth mixture models to identify whether there were latent groups of autistic individuals, based on their trajectories of SDQ total and subscale scores. Growth mixture models assume that the sample consists of multiple mixed effects models, each capturing a subgroup trajectory with shared intercept and slope63. We fitted models with one to four groups for each subscale and SDQ total scores in each cohort, using the lcmm (v.2.1.0) package in R64. The optimal number of latent trajectories were then determined by comparing fit indices, including Bayesian information criterion values, classification quality measure (entropy) and substantive interpretation. Models with lower Bayesian information criterion values and higher entropy were favoured65. Models identifying subgroups with less than 5% of the sample size were not considered, owing to poor statistical reliability and limited practical significance66. Second, we used linear latent growth curve models to identify the latent trajectories of SDQ total and subscale scores in the three groups (childhood diagnosed, adolescent diagnosed and the general population) for all cohorts. Each linear model included a latent intercept to represent the initial level of the outcome variable, and a linear latent slope to represent the mean rate of change over time. We chose this 9–11 age window as our cut-off because it aligns with the onset of puberty and the transition from primary to secondary school, and aligns with epidemiological evidence showing increased autism incidence in female individuals during this window34. An earlier cut-off was not feasible because only MCS (ages 5–7) and GUI (age 7) recorded autism diagnoses before this window. A later cut-off was not possible because there were no autism diagnoses in MCS after age 14. To further examine the relationship between age at diagnosis and socioemotional and behavioural outcomes, we conducted further latent growth curve models for autistic children using stepwise groupings by age at diagnosis in MCS and LSAC-B (see Supplementary Note 5 and Supplementary Fig. All individuals who lacked an autism diagnosis (and also an ADHD diagnosis in MCS) were included in the general-population group. Given the sex differences in age at autism diagnosis34, we also applied the same models stratified by sex, estimating latent intercept and slope for each sex, within the autistic samples. All latent growth curve models were fitted under the structural equation modelling framework using the lavaan (v.0.6-19) package in R67. To examine the association between growth mixture models-derived SDQ latent trajectories and mental-health phenotypes in MCS, LSAC-B and LSAC-K, adjusting for sex, we used multiple regression in autistic individuals. Multiple regression analyses were conducted in MCS, LSAC-B and LSAC-K to investigate the association between age at autism diagnosis (the outcome variable), SDQ total difficulties and subscale latent trajectories memberships identified in optimal growth mixture models, and also accounting for other sociodemographic covariates. We did not detect any multicollinearity among the variables using variance inflation factors. The relative importance of each predictor was assessed using dominance analysis68. This approach leverages the correlation matrix to consider not only individual predictors, but also the correlations between them, providing a more comprehensive assessment of their relative importance70. To examine potential causal pathways, mediation analyses were done, allowing sociodemographic factors to indirectly influence the age at diagnosis through their effects on SDQ latent trajectory memberships identified in the optimal growth mixture model. Using structural equation modelling in the lavaan (v.0.6-19) package67, both direct and indirect effects were assessed, with their significance calculated using bootstrapping analysis. Further details are provided in Supplementary Note 4. To investigate the specificity of our findings to autism, we used growth mixture models, latent growth curve models, regression and mediation analyses in individuals with ADHD, but without a co-occurring autism diagnosis in the MCS cohort (n = 89, Supplementary Table 6, with results presented in Supplementary Note 5). Carers were asked the following question: “Has a doctor or other health professional ever told you that <child's name> had attention deficit hyperactivity disorder (ADHD)? To assess the impact of missingness, we used softImpute (v.1.4-1) to impute missing data for all children with an autism or ADHD diagnosis reported by any carer in any sweep in the MCS cohort (autism: n = 623, Supplementary Table 5; ADHD: n = 325, Supplementary Table 6). We chose softImpute because of its computational efficiency in handling large-scale matrices through low-rank approximation, effectively preserving underlying structure of input data. Further information is provided in Supplementary Note 2. We used data from the SPARK cohort39 iWES2 v.1 dataset (released in Feb 2022), which included data from 70,487 autistic individuals and their families as the SPARK Discovery cohort. Data from SPARK iWES v.3 (released in August 2024), which included an additional 71,267 autistic individuals and their families, was included in the SPARK Replication cohort. To avoid false positives caused by fine-scale population stratification, we restricted the analyses to individuals of genetically inferred European ancestries (Discovery, n = 51,869; Replication, n = 50,211 autistic and non-autistic participants), which was provided by the SPARK consortium. From this, we excluded individuals with genotyping rate of less than 98%, individuals with sex mismatches and those with excess heterozygosity (3 standard deviations from the mean heterozygosity). Where trio data were available, trios with greater than 5% Mendelian errors were excluded, resulting in 47,170 (Discovery) and 48,750 (Replication) autistic and non-autistic individuals. We included genetic variants with minor allele frequency (MAF) greater than 1%, genotyping rate more than 95% and that were in Hardy–Weinberg equilibrium (P > 1 × 10−6), resulting in 518,189 (Discovery) and 1,225,308 (Replication) SNPs. We used these quality-controlled genotype data for imputation, calculating genetic principal components and inferring relatedness among individuals. For genetic PCA, we pruned SNPs for linkage disequilibrium (maximum r2 = 0.1) and removed the human leukocyte antigen region. Using PC-AiR72 in GENESIS (v.2.22.2), we first calculated PCs in genetically unrelated individuals and then projected the PCs onto related individuals. After imputation, variants were converted from GRCh38/hg38 to GRCh37/hg19 using liftOver. Polygenic scores (PGSs) were calculated using PRScs76 (v.1.1.0), which has a Bayesian shrinkage prior. PGSs were calculated for autism diagnosed before age 11 (iPSYCHbefore11) and autism diagnosed after age 10 (iPSYCHafter10), generated using the iPSYCH2015 (ref. 77) cohort, details of which are provided below. For simplicity, we refer to this cohort as iPSYCH throughout. We ran separate linear regression analyses between each of the two PGSs and age at autism diagnosis (converted to years in all analyses) in the quality-controlled dataset. We excluded individuals older than 22 to focus on those who had an autism diagnosis using either the DSM-IV79 or DSM-5, retaining a maximum of 18,809 (Discovery cohort) and 9,383 (Replication cohort) autistic individuals for PGS analyses. This criteria also allowed us to focus on individuals who received their diagnosis in childhood or adolescence, because older adults may have missed an earlier diagnosis of autism owing to secular changes in societal attitudes towards autism. The baseline model included intellectual disability (16.34% had carer-reported intellectual disability), sex and the first 10 genetic PCs as covariates. We ran eight different sensitivity analyses by including various covariates as well as the covariates included in the baseline model. First, we ran three models to account for developmental and clinical covariates: age at walking and age at first words (model 2), age at walking and first words, autism severity (SCQ and RBS-R total scores), carer-reported IQ scores, and language or other regression (model 3), and stratified analyses restricted to individuals without intellectual disability who can speak in longer sentences (model 4). Next, we ran two models accounting for sociodemographic factors: parental SES (model 5) and also area deprivation (model 6). Finally, we also ran sensitivity analyses after stratifying by sex. In the SPARK cohort, we obtained data for age at achieving nine developmental milestones (in months) for autistic individuals. For all milestones, we excluded individuals who were more than five median absolute deviations from the median. We ran multiple linear regression with PGS for iPSYCHbefore11 and iPSYCHafter10 GWASs with sex and the first ten genetic PCs as covariates. Yet again, we ran the analyses for both the Discovery and Replication cohorts and meta-analysed it using inverse-variance-weighted meta-analysis. We identified rare (MAF < 0.1%) de novo and inherited variants in complete trios from SPARK, as previously described40. We identified high-impact protein-truncating variants by restricting it to variants in loss-of-function observed/expected upper bound fraction (LOEUF)80; highly constrained decile (LOEUF < 0.37) that were annotated as either frameshift, stop gained or start lost; and that had a loss-of-function transcript effect estimator (LOFTEE) high-confidence annotation. To identify high-impact de novo missense variants, we restricted it to variants in LOEUF highly constrained genes (LOEUF < 0.37) that had an MPC (missense badness, PolyPhen-2, and constraint) score81 > 2. We included sex and the first ten genetic PCs as covariates. We generated a GWAS of age at autism diagnosis (in years) in the quality-controlled dataset from SPARK, restricting it to autistic individuals who were under 22 years of age (Discovery, n = 18,809; Replication, n = 9,356) and SNPs with MAF > 1%. GWAS was generated using FastGWA82 with sex, intellectual disability and the first ten genetic PCs as covariates. In iPSYCH, we generated an additional GWAS of age at autism diagnosis (in years) in a quality-controlled dataset of unrelated individuals with sex and intellectual disability included as covariates using FastGWA82, restricting it to SNPs with MAF > 1%. To keep it consistent with SPARK, we excluded individuals who were diagnosed after age 22, leaving a total sample of 18,965 individuals. Further details of quality control and imputation are provided in ref. We generated three age at autism diagnosis stratified GWASs in SPARK using (unscreened) non-autistic parents and siblings as controls (Discovery, ncontrol = 24,965; Replication, ncontrol = 33,302). For these analyses, we did not restrict it to individuals under 22, to increase the sample size. We included age at recruitment in the study (to account for the use of parents as controls, who potentially lack an autism diagnosis owing to secular changes in attitudes and diagnosis), sex and the first ten genetic PCs as covariates. Fast-GWA GLMM can account for relatedness and fine-scale population stratification, even in family-based samples such as SPARK. Although inclusion of unscreened related individuals as controls can decrease heritability and statistical power to identify loci90, we used the GWAS to primarily conduct genetic correlation and related analyses. To ensure the robustness of these models we did the following: first, we confirmed that the attenuation ratio for all GWASs was not significantly greater than 1; second, we generated an additional GWAS of SPARK without stratifying by age at autism diagnosis using the same methods and confirmed a high genetic correlation (rg = 0.92, s.e. = 0.17) with a previous SPARK GWAS49, which used a case-pseudocontrol approach; and third, in the genomicSEM analyses, we ran sensitivity analyses using a trio-based SPARK GWAS49 in lieu of the age at diagnosis stratified GWAS from SPARK and confirmed our findings. We chose to subdivide the iPSYCH cohort at age 10 because we observed an increase in SDQ scores in birth cohorts at this age, and because age 10 is associated with an increase in diagnosis of female individuals in epidemiological samples34. We also conducted a GWAS of autism diagnosed before age nine (iPSYCHbefore9, n = 5,451 autistic and n = 36,667 non-autistic individuals). All individuals included in these GWASs from iPSYCH were born between May 1980 and December 2008 to mothers who were living in Denmark. The GWAS was done for unrelated individuals of European ancestry, with the first ten genetic PCs included as covariates using logistic regression as provided in PLINK. Heritability analyses for age at autism diagnosis were conducted using a single-component genome-wide complex trait analysis with a genomic-relatedness-based restricted maximum likelihood approach (GCTA-GREML v1.94.1)91,92 in unrelated autistic individuals using the quality-controlled genetic data in SPARK. We estimated SNP-based heritability first after including sex and the first ten genetic PCs as covariates, and then also with intellectual disability as a covariate (baseline model). We conducted genetic correlation analyses using LDSC, with linkage disequilibrium scores from the northwest European populations. We did genetic correlation analyses among different autism GWASs using LDSC (v.1.0.1). For genomicSEM42 (v.0.0.5) analyses, we restricted it to six GWASs with minimal sample overlap, without high genetic correlation (rg > 0.95), and with wide variation in age at diagnosis to conduct genomicSEM analyses using autosomes. Using the patterns of genetic correlations observed, we tested an age at diagnosis-related correlated two-factor model. We also tested: first, a single-factor model; second, a correlated two-factor ‘geography' model, in which three US-based autism GWASs loaded onto one factor, and three Europe-based autism GWASs loaded onto a second factor; third, a bifactor model based on age at diagnosis; fourth, a bifactor model based on the geography of the cohorts; and finally, a hierarchical factor model based on age at diagnosis. The two-factor model was chosen because it had lower root mean square error of approximation and higher comparative fit index and was more parsimonious than the bifactor model. We ran sensitivity analyses using different GWASs of autism as input and confirmed that the two-correlated-factor model was the best-fitting model of those tested. We obtained quality-controlled and imputed genotype data from ALSPAC94,95,96. Further details about the cohort are provided in Supplementary Note 1. In brief, ALSPAC children were genotyped using the Illumina HumanHap550 quad chip genotyping platforms by 23andme. Some individuals were excluded owing to sex mismatches, excess heterozygosity, missingness greater than 3% and insufficient sample replication (identical by descent score of less than 0.8). After multidimensional scaling and comparison with Hapmap II (release 22), only individuals of genetically inferred European ancestries were retained. SNPs with low frequency (MAF < 1%), poor genotyping (call rate < 95%) and deviations from Hardy–Weinberg equilibrium (P < 5 × 10−7) were removed; 9,115 subjects and 500,527 SNPs passed quality control. After imputation, we further removed low-frequency SNPs (MAF < 1%). Genome-wide genotype data were generated by the Sample Logistics and Genotyping facilities at the Wellcome Sanger Institute and LabCorp (Laboratory Corporation of America) using support from 23andMe. We also obtained quality-controlled and imputed data from MCS. In brief, MCS samples were genotyped using the Illumina Global Screening Array97. Some individuals were excluded owing to sex mismatches, excess heterozygosity and missingness greater than 2%. We identified European samples using the GenoPred pipeline98 (https://github.com/opain/GenoPred). SNPs with low frequency (MAF < 1%), poor genotyping (call rate < 97%) and deviations from Hardy–Weinberg equilibrium (P < 1 × 10−6) were removed. Genetic PCs were calculated for both cohorts using PC-AiR, as described earlier. We obtained scores on the SDQ total and subscales for six ages in the MCS and five ages in ALSPAC. We ran cross-sectional analysis at each age using multiple linear regression with PGS for iPSYCHbefore11 and iPSYCHafter10, with sex, age and the first ten genetic PCs as covariates. We also ran multiple linear mixed effects regression using the lme4 (v.1.1.27.1) package in R99, fitting a PGS by age interaction term to investigate whether the effects of PGS on SDQ change over time. To investigate whether the differences in association between MCS and ALSPAC were due to differences in ascertainment between the two cohorts, we matched ALSPAC to MCS using entropy balancing100 and re-ran the PGS association analyses. Entropy balancing is a reweighting technique that ensures the covariate distributions are identical between groups. This method uses optimization algorithms to assign weights to individuals such that the weighted average of the covariates in ALSPAC (the larger genotyped cohort) matches that of MCS (the smaller genotyped cohort), minimizing confounding biases and increasing comparability. We used the child's biological sex, maternal age at delivery and maternal highest educational qualification at first data collection in each cohort as matching factors. Entropy balancing was done using the ebal (v.0.1-8) package in R101. For ALSPAC, we obtained understanding of simple phrases (such as “Do you want that?” or “Come here”) and gesture scores from the Macarthur-Bates Communicative Development Inventories102 at 15 months of age (Supplementary Note 9). We conducted multiple linear regression using PGS for iPSYCHbefore11 and iPSYCHafter10, with sex, age and the first ten genetic PCs as covariates. Autism diagnosis for the MCS was obtained using parent/carer reports of autism/Asperger's syndrome diagnosis by a doctor at ages 5, 7, 11 and 14. We conducted Firth's bias-reduced multiple logistic regression (logistf v.1.26.0 package in R) using PGS for iPSYCHbefore11 and iPSYCHafter10, with sex, age and the first ten genetic PCs as covariates. Ethical approval for individual cohorts was obtained independently of the current study. Ethical approval for ALSPAC was obtained from the ALSPAC Ethics and Law Committee and the local research ethics committees. Ethical approval for each sweep of MCS was obtained from NHS research ethics committees (MREC). Ethical approval for LSAC was obtained from the Australian Institute of Family Studies Human Research Ethics Committee. Ethical approval for GUI was obtained from a dedicated research ethics committee set up by the Department of Children, Equality, Disability, Integration and Youth. Ethical approval for SPARK was obtained from the Western Institutional Review Board Copernicus Group (IRB protocol 20151664). Ethical approval for the analyses of de-identified data used in this study was obtained from the Cambridge Human Biology Research Ethics Committee (HBREC.2020.07). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. • The iPSYCH autism GWAS (unstratified, sex stratified and age at diagnosis stratified, age at diagnosis) can be obtained from J. Grove (grove@biomed.au.dk) or A.D.B. • For ALSPAC, the study website contains details of all the available data through a fully searchable data dictionary and variable search tool: http://www.bristol.ac.uk/alspac/researchers/our-data/ • SPARK quality control, imputation and GWAS: https://github.com/vwarrier/SPARK_iWES2_imputation/ Schendel, D. E. & Thorsteinsson, E. Cumulative incidence of autism into adulthood for birth cohorts in Denmark, 1980–2012. Russell, G. et al. Time trends in autism diagnosis over 20 years: a UK population-based cohort study. Time trends over 16 years in incidence – rates of autism spectrum disorders across the lifespan based on nationwide Danish register data. Diagnosis of autism spectrum disorder after age 5 in children evaluated longitudinally since infancy. & Williams, K. Parent-reported autism diagnostic stability and trajectories in the Longitudinal Study of Australian Children. Allison, C. et al. Quantitative Checklist for Autism in Toddlers (Q-CHAT). Davidovitch, M., Levit-Binnun, N., Golan, D. & Manning-Courtney, P. Late diagnosis of autism spectrum disorder after initial negative assessment by a multidisciplinary team. Avlund, S. H. et al. Factors associated with a delayed autism spectrum disorder diagnosis in children previously assessed on suspicion of autism. Landa, R. J., Reetzke, R., Holingue, C. B., Herman, D. & Hess, C. R. Diagnostic stability and phenotypic differences among school-age children diagnosed with ASD before age 2. Jadav, N. & Bal, V. H. Associations between co-occurring conditions and age of autism diagnosis: implications for mental health training and adult autism research. Rødgaard, E.-M., Jensen, K., Miskowiak, K. W. & Mottron, L. Autism comorbidities show elevated female-to-male odds ratios and are associated with the age of first autism diagnosis. Daniels, A. M. & Mandell, D. S. Explaining differences in age at autism spectrum disorder diagnosis: a critical review. Havdahl, A. et al. Genetic contributions to autism spectrum disorder. St Pourcain, B. et al. Developmental changes within the genetic architecture of social communication behavior: a multivariate study of genetic variance in unrelated individuals. St Pourcain, B. et al. ASD and schizophrenia show distinct developmental profiles in common genetic overlap with population-based social communication difficulties. Martini, M. I. et al. Age effects on autism heritability and etiological stability of autistic traits. Gentles, S. J. et al. Trajectory research in children with an autism diagnosis: a scoping review. Goodman, R. Strengths and Difficulties Questionnaire (American Psychological Association, 1997); https://doi.org/10.1037/t00540-000. The Strengths and Difficulties Questionnaire: psychometric properties of the parent and teacher version in children aged 4-7. Kovacs, S. & Sharp, C. Criterion validity of the Strengths and Difficulties Questionnaire (SDQ) with inpatient adolescents. Borg, A.-M., Kaukonen, P., Salmelin, R., Joukamaa, M. & Tamminen, T. Reliability of the strengths and difficulties questionnaire among Finnish 4–9-year-old children. & Murray, A. L. Longitudinal invariance of the Strengths and Difficulties Questionnaire across ages 4 to 16 in the ALSPAC sample. Association between intellectual functioning and autistic traits in the general population of children. Skuse, D. H. et al. Social communication competence and functional adaptation in a general population of children: preliminary evidence for sex-by-verbal IQ differential risk. & Hughes, J. C. Emotional and behavioural problems of children with autism spectrum disorder attending mainstream schools. Incidence rates and cumulative incidences of the full spectrum of diagnosed mental disorders in childhood and adolescence. St Pourcain, B. et al. Heritability and genome-wide analyses of problematic peer relationships during childhood and adolescence. & Plomin, R. Prosocial behavior from early to middle childhood: genetic and environmental influences on stability and change. Jami, E. S. et al. Genome-wide association meta-analysis of childhood and adolescent internalizing symptoms. Contribution of autosomal rare and de novo variants to sex differences in autism. Aizenberg, D., Shalev, I., Uzefovsky, F. & Eran, A. Data-driven characterization of individuals with delayed autism diagnosis. Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Verhoef, E. et al. Genome-wide analyses of vocabulary size in infancy and toddlerhood: associations with attention-deficit/hyperactivity disorder, literacy, and cognition-related traits. Meta-analysis of GWAS of over 16,000 individuals with autism spectrum disorder highlights a novel locus at 10q24.32 and a significant overlap with schizophrenia. Identification of common genetic risk variants for autism spectrum disorder. Russell, A. S. et al. Who, when, where, and why: a systematic review of “late diagnosis” in autism. Matoba, N. et al. Common genetic risk variants identified in the SPARK cohort support DDHD2 as a candidate risk gene for autism. Warrier, V. et al. Genetic correlates of phenotypic heterogeneity in autism. Roman-Urrestarazu, A., Dumas, G. & Warrier, V. Naming autism in the right context. Relative importance for linear regression in R: the package relaimpo. Clifford, S. A., Davies, S., Wake, M. & Child Health CheckPoint Team. Child Health CheckPoint: cohort summary and methodology of a physical health and biospecimen module for the Longitudinal Study of Australian Children. Layte, R. & McCrory, C. Growing Up in Ireland: National Longitudinal Study of Children. Maternal Health Behaviours and Child Growth in Infancy (The Stationery Office, Dublin, 2015). Anderson, E. R. Analyzing change in short-term longitudinal research using cohort-sequential designs. Grasso, M., Lazzaro, G., Demaria, F., Menghini, D. & Vicari, S. The Strengths and Difficulties Questionnaire as a valuable screening tool for identifying core symptoms and behavioural and emotional problems in children with neuropsychiatric disorders. Brett, D., Warnell, F., McConachie, H. & Parr, J. R. Factors affecting age at ASD diagnosis in UK: no evidence that diagnosis age has decreased between 2004 and 2014. Roman-Urrestarazu, A. et al. Association of race/ethnicity and social disadvantage with autism prevalence in 7 million school children in England. & Kelderman, H. On growth curves and mixture models. Proust-Lima, C., Philipps, V. & Liquet, B. Estimation of extended mixed models using latent classes and latent processes: the R package lcmm. Growth mixture modeling: a method for identifying differences in longitudinal change among unobserved groups. Grimm, K. J., Mazza, G. L. & Davoudzadeh, P. Model selection in finite mixture models: a k-fold cross-validation approach. Rosseel, Y. Iavaan: An R package for structural equation modeling. B. et al. Increasing social skills and pro-social behavior for three children diagnosed with autism through the use of a teaching package. Azen, R. & Budescu, D. V. Comparing predictors in multivariate regression models: an extension of dominance analysis. Robust relationship inference in genome-wide association studies. Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Das, S. et al. Next-generation genotype imputation service and methods. Loh, P.-R., Palamara, P. F. & Price, A. L. Fast and accurate long-range phasing in a UK Biobank cohort. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. The iPSYCH2015 Case-Cohort sample: updated directions for unravelling genetic and environmental architectures of severe mental disorders. Diagnostic and Statistical Manual of Mental Disorders 4th edn (American Psychiatric Association, 1994). The mutational constraint spectrum quantified from variation in 141,456 humans. Regional missense constraint improves variant deleteriousness prediction. Jiang, L. et al. A resource-efficient tool for mixed model association analysis of large-scale data. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Lam, M. et al. RICOPILI: Rapid Imputation for COnsortias PIpeLIne. & Abecasis, G. R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Als, T. D. et al. Depression pathophysiology, risk prediction of recurrence and comorbid psychiatric disorders using genome-wide analyses. A generalized linear mixed model association tool for biobank-scale data. Peyrot, W. J., Boomsma, D. I., Penninx, B. W. J. H. & Wray, N. R. Disease and polygenic architecture: avoid trio design and appropriately account for unscreened control subjects for common disease. Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Yang, J. et al. Common SNPs explain a large proportion of the heritability for human height. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. ALSPAC–the Avon Longitudinal Study of Parents and Children. Fraser, A. et al. Cohort profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort. Fitzsimons, E. et al. Collection of genetic data at scale for a nationally representative population: the UK Millennium Cohort Study. Evaluation of polygenic prediction methodology within a reference-standardized framework. Hainmueller, J. Entropy balancing for causal effects: a multivariate reweighting method to produce balanced samples in observational studies. Fenson, L. et al. MacArthur-Bates Communicative Development Inventories, 2nd edn (American Psychological Association, 2006); https://doi.org/10.1037/t11538-000. This research was supported by funding from the Simons Foundation for Autism Research Initiative, the Wellcome Trust (214322\Z\18\Z and 226392/Z/22/Z), Horizon-Europe R2D2-MH (grant agreement 101057385) and UKRI (10063472). For the purpose of open access, we have applied a CC BY public copyright licence to any author-accepted manuscript version arising from this submission. Any views expressed are those of the author(s) and not necessarily those of the funder. and the European Union's Horizon Europe program (R2D2-MH; grant agreement 101057385 to A.D.B.). High-performance computer capacity for handling and statistical analysis of iPSYCH data on the GenomeDK HPC facility was provided by the Center for Genomics and Personalized Medicine and the Centre for Integrative Sequencing, iSEQ, Aarhus University, Denmark (grant to A.D.B.). The UK Medical Research Council and Wellcome (grant 217065/Z/19/Z) and the University of Bristol provide core support for ALSPAC. A comprehensive list of grants funding is available on the ALSPAC website (http://www.bristol.ac.uk/alspac/external/documents/grant-acknowledgements.pdf). R2D2-MH has been funded by Horizon Europe (grant agreement 101057385), by UK Research and Innovation (UKRI) under the UK government's Horizon Europe funding guarantee (grant 10039383) and by the Swiss State Secretariat for Education, Research and Innovation (SERI) under contract 22.00277. are funded by the Max Planck Society. We thank the Centre for Longitudinal Studies (CLS), UCL Social Research Institute, for the use of these data and the UK Data Service for making them available. However, neither CLS nor the UK Data Service bear any responsibility for the analysis or interpretation of these data. This paper uses unit record data from Growing Up in Australia, the Longitudinal Study of Australian Children. Growing Up in Ireland (GUI) was funded by the Government of Ireland through the Department of Children, Equality, Disability, Integration and Youth (DCEDIY) and the Central Statistics Office (CSO). Results in this report are based on analysis of data from research microdata files provided by the Central Statistics Office (CSO). We thank all the families who took part in this study, the midwives for their help in recruiting them and the ALSPAC team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists and nurses. This study includes data from the Norwegian Mother, Father and Child Cohort Study, conducted by the Norwegian Institute of Public Health. Xinhe Zhang, Yuanjun Gu, Sharon A. S. Neufeld, Simon Baron-Cohen & Varun Warrier Xinhe Zhang, Yuanjun Gu, Carrie Allison, Alex Tsompanidis, Deep Adhya, Rosemary Holt, Omar Al-Rubaie, Marcin Radecki, Meng-Chuan Lai, Yeshaya David Greenberg, Elizabeth Weir, Yumnah T. Khan, Simon Baron-Cohen & Varun Warrier iPSYCH, The Lundbeck Foundation Initiative for Integrative Psychiatric Research, Aarhus, Denmark Jakob Grove, Ole Mors, Preben Bo Mortensen, Merete Nordentoft, Thomas Werge, Marie Bækvad-Hansen, David Michael Hougaard, Ole Mors & Anders D. Børglum Jakob Grove, Cornelia K. Buus, Lea K. Nielsen & Anders D. Børglum Jakob Grove, Cornelia K. Buus, Lea K. Nielsen, Preben Bo Mortensen & Anders D. Børglum Mahmoud Koko, Daniel S. Malawsky, Emma M. Wade & Hilary C. Martin Language and Genetics Department, Max Planck Institute for Psycholinguistics, Nijmegen, The Netherlands Centre for Brain and Cognitive Development, Department of Psychological Sciences, Birkbeck University of London, London, UK Research Department, Lovisenberg Diaconal Hospital, Oslo, Norway Program in Neurobehavioral Genetics and Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA, USA Donders Institute for Brain, Cognition and Behaviour, Radboud University, Nijmegen, The Netherlands Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA The Broad Institute of MIT and Harvard, Cambridge, MA, USA Deepak P. Srivastava, Lucia Dutan-Polit, Adam Pavlinek, Laura Sichlinger & Roland Nagy Department of Obstetrics and Gynecology, Meir Medical Center, Kefar Sava, Israel Faculty of Medical & Health Sciences, Tel-Aviv University, Tel Aviv, Israel Laboratory for Autism and Neurodevelopmental Disorders, Center for Neuroscience and Cognitive Systems, Istituto Italiano Di Tecnologia, Rovereto, Italy Margaret and Wallace McCain Centre for Child, Youth & Family Mental Health and Azrieli Adult Neurodevelopmental Centre, Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, Canada Department of Psychiatry, The Hospital for Sick Children, Toronto, Ontario, Canada Psychology Department, Ben-Gurion University of the Negev, Be'er Sheva, Israel Danish Center for Neonatal Screening, Department of Congenital Disorders, Statens Serum Institut, Copenhagen, Denmark Institute of Biological Psychiatry, Mental Health Services, Copenhagen University Hospital, Roskilde, Denmark Caroline M. Nievergelt, Adam X. Maihofer, Dewleen G. Baker, Carol E. Franz, Kelleigh Hogan, William S. Kremen, Elizabeth A. Mikita, Sonya B. Norman, Matthew S. Panizzon, Victoria B. Risbrough & Murray B. Stein Center of Excellence for Stress and Mental Health, Veterans Affairs San Diego Healthcare System, San Diego, CA, USA Caroline M. Nievergelt, Adam X. Maihofer, Dewleen G. Baker, Kelleigh Hogan, Elizabeth A. Mikita, Sonya B. Norman & Victoria B. Risbrough Caroline M. Nievergelt, Adam X. Maihofer, Kelleigh Hogan, Elizabeth A. Mikita & Victoria B. Risbrough Chan School of Public Health, Boston, MA, USA Karmel W. Choi, Bizu Gelaye, Andrew Ratanatharathorn, Laura Sampson, Michelle A. Williams & Karestan C. Koenen Department of Psychiatry, Massachusetts General Hospital, Boston, MA, USA Psychology and Neuroscience, Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, London, UK Jonathan R. I. Coleman, Gerome Breen, Chiara Fabbri & Saskia Hagenaars Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Nikolaos P. Daskalakis, Chris Chatzinakos, Christopher P. DiPietro, Hailiang Huang, Jordan W. Smoller, Yan Xia, Clement C. Zai & Karestan C. Koenen Department of Psychiatry, Harvard Medical School, Boston, MA, USA Nikolaos P. Daskalakis, Elizabeth A. Bolger, Chris Chatzinakos, Milissa L. Kaufman, Alaptagin Khan, Lauren A. M. Lebois, Martin H. Teicher, Sherry Winternitz & Kerry J. Ressler Center of Excellence in Depression and Anxiety Disorders, McLean Hospital, Belmont, MA, USA VA Connecticut Healthcare Center, West Haven, CT, USA Renato Polimanti, Daniel F. Levey, Gita A. Pathak & Robert H. Pietrzak Cindy Aaronson, Leah Cahn, Frank Desarnaud, Adriana Feder, Janine D. Flory, Iouri Makotkine & Rachel Yehuda Department of Psychiatry Virginia Institute for Psychiatric and Behavioral Genetics, Richmond, VI, USA Department of Pediatrics, Harvard Medical School, Boston, MA, USA Department of Social and Behavioral Sciences, Harvard T.H. Chan School of Public Health, Boston, MA, USA Department of Psychiatry, Virginia Commonwealth University, Richmond, VA, USA Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, USA Anthony Batzler, Joanna M. Biernacka, Brandon J. Coombes & Gregory D. Jenkins Jean C. Beckham, Michelle F. Dennis, Michael A. Hauser, Nathan A. Kimbrel & Douglas E. Williamson Genetics Research Laboratory, VA Mid-Atlantic Mental Illness Research, Education, and Clinical Center (MIRECC), Durham, NC, USA Laboratory of Integrative Neuroscience, Department of Psychiatry, Universidade Federal de São Paulo, São Paulo, Brazil Instituto Nacional de Psiquiatraía Ramón de la Fuente Muñiz, Center for Global Mental Health, Mexico City, Mexico Comprehensive Injury Center, Medical College of Wisconsin, Milwaukee, WI, USA Department of Psychiatry, James J. Peters VA Medical Center, Bronx, NY, USA Department of Psychiatry and Psychology, Mayo Clinic, Rochester, MN, USA Department of Psychiatry, Washington University in Saint Louis School of Medicine, Saint Louis, MO, USA MRC Centre for Psychiatric Genetics and Genomics, Cardiff University, National Centre for Mental Health, Cardiff, UK Elizabeth A. Bolger, Milissa L. Kaufman, Alaptagin Khan, Sherry Winternitz & Kerry J. Ressler Department for Congenital Disorders, Statens Serum Institut, Copenhagen, Denmark K. G. Jebsen Center for Genetic Epidemiology, Department of Public Health and Nursing, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology, Trondheim, Norway Sigrid Børte, Kristian Hveem, Bendik S. Winsvold & John-Anker Zwart Department of Research, Innovation and Education, Division of Clinical Neuroscience, Oslo University Hospital, Oslo, Norway Sigrid Børte, Synne Stensland, Bendik S. Winsvold & John-Anker Zwart School of Medicine, Case Western Reserve University, Cleveland, OH, USA Department of Psychiatry, University Hospitals, Cleveland, OH, USA Division of Depression and Anxiety Disorders, McLean Hospital, Belmont, MA, USA Mental Health & Neuroscience Program, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Lucía Colodro-Conde, Scott D. Gordon, Michelle K. Lupton, Kerrie McAloney & Sarah E. Medland Huntsman Mental Health Institute, Salt Lake City, UT, USA Norwegian Centre for Violence and Traumatic Stress Studies, Oslo, Norway Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN, USA Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, USA Department of Psychiatry and Behavioral Sciences, Emory University, Atlanta, GA, USA Negar Fani, Charles F. Gillespie, Tanja Jovanovic, Adriana Lori, Jessica L. Maples-Keller, Vasiliki Michopoulos, Abigail Powers, Barbara O. Rothbaum, Alicia K. Smith, Jennifer S. Stevens, Sanne J. H. van Rooij & Kerry J. Ressler Department of Medicine (Biomedical Genetics), Boston University Chobanian & Avedisian School of Medicine, Boston, MA, USA Department of Psychological Sciences, Case Western Reserve University, Cleveland, OH, USA Psychiatry Service, VA Connecticut Healthcare Center, West Haven, CT, USA Psychiatry and Behavioral Neurosciences, Wayne State University School of Medicine, Detroit, MI, USA Cohen Veterans Bioscience, New York, NY, USA Department of Genetics, Washington University in Saint Louis School of Medicine, Saint Louis, MO, USA Sian M. J. Hemmings, Soraya Seedat & Leigh L. van den Heuvel SAMRC Genomics of Brain Disorders Research Unit, Stellenbosch University, Cape Town, South Africa Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA Department of Psychiatry, Yale University, New Haven, CT, USA Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Centre for Addiction and Mental Health, Neurogenetics Section, Molecular Brain Science Department, Campbell Family Mental Health Research Institute, Toronto, Ontario, Canada Department of Health Care Policy, Harvard Medical School, Boston, MA, USA Department of Psychiatry & Neuroscience Institute, SA MRC Unit on Risk & Resilience in Mental Disorders, University of Cape Town, Cape Town, South Africa Mental Illness Research, Education and Clinical Center, Crescenz VAMC, Philadelphia, PA, USA Bruce R. Lawford, Divya Mehta, Charles Phillip Morris & Joanne Voisey Department of Psychiatry and Behavioral Sciences, Texas A&M University College of Medicine, Bryan, TX, USA Department of Anesthesiology, UNC Institute for Trauma Recovery, Chapel Hill, NC, USA Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden Department of Genetic and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Genetics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Department of Psychology, Harvard University, Boston, MA, USA Department of Emergency Medicine, UNC Institute for Trauma Recovery, Chapel Hill, NC, USA National Center for Post Traumatic Stress Disorder, Executive Division, White River Junction, VT, USA Department of Emergency Medicine, Alpert Brown Medical School, Providence, RI, USA Department of Pediatrics, Alpert Brown Medical School, Providence, RI, USA Department of Psychiatry and Human Behavior, Alpert Brown Medical School, Providence, RI, USA Department of Psychology, Northern Illinois University, DeKalb, IL, USA Research and Development Service, South Texas Veterans Health Care System, San Antonio, TX, USA Department of Psychiatry and Behavioral Sciences, University of Texas Health Science Center at San Antonio, San Antonio, TX, USA U.S. Department of Veterans Affairs National Center for Posttraumatic Stress Disorder, West Haven, CT, USA Center for Care Delivery and Outcomes Research (CCDOR), Minneapolis, MN, USA Chan School of Public Health, Boston, MA, USA Department of Psychological Sciences, Emory University, Atlanta, GA, USA Department of Research and Outcomes, Skyland Trail, Atlanta, GA, USA School of Public Health and Department of Epidemiology, Louisiana State University Health Sciences Center, New Orleans, LA, USA School for Mental Health and Neuroscience, Department of Psychiatry and Neuropsychology, Maastricht Universitair Medisch Centrum, Maastricht, The Netherlands Child Mind Institute, New York, NY, USA Department of Medicine, Universidad Peruana de Ciencias Aplicadas, Lima, Peru SAMRC Extramural Genomics of Brain Disorders Research Unit, Stellenbosch University, Cape Town, South Africa Institute for Research on Women and Gender, University of Michigan, Ann Arbor, MI, USA Psychiatric and Neurodevelopmental Genetics Unit (PNGU), Massachusetts General Hospital, Boston, MA, USA Developmental Biopsychiatry Research Program, McLean Hospital, Belmont, MA, USA Hans, Institute of Biological Psychiatry, Roskilde, Denmark Herbert Wertheim School of Public Health and Human Longevity Science, University of California, San Diego, La Jolla, CA, USA Department of Psychiatry, Uniformed Services University, Bethesda, MD, USA Lifespan Changes in Brain and Cognition (LCBC), Department of Psychology, University of Oslo, Oslo, Norway Department of Mental Health, Ralph H Johnson VA Medical Center, Charleston, SC, USA Department of Mental Health, James J. Peters VA Medical Center, Bronx, NY, USA Research Service, Central Texas Veterans Health Care System, Temple, TX, USA Department of Psychiatry and Behavioral Sciences, Texas A&M University School of Medicine, Bryan, TX, USA Centre for Addiction and Mental Health, General Adult Psychiatry and Health Systems Division, Toronto, Ontario, Canada Department of Biostatistics, Yale University, New Haven, CT, USA wrote the initial draft with input from H.C.M. All authors read and commented on the final manuscript. Correspondence to Xinhe Zhang or Varun Warrier. received a speakers' fee from the Lundbeck Foundation. The other authors declare no competing interests. Nature thanks Lonnie Zwaigenbaum and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Variance explained (R² or η²) in age at autism diagnosis by clinical and sociodemographic factors, identified from the review of literature (1998–2023). Variables are grouped into sociodemographic (MAD, SES), socioemotional-behavioural (SDQ scores), sex, clinical (e.g., IQ, regression, language ability), and autism severity (e.g, SCQ, ADOS, ADI-R, RBS-R) categories. Colored points denote variables analysed in the current study. Inset shows factors that explain greater than 10% of the variance in age at autism diagnosis. Note: None of these studies account for additional family, service access, and contextual factors known to influence diagnostic timing. Abbreviations: MAD, Maternal Age at Delivery; IQ, Intelligence Quotient; SES, Socio-economic Status; ADOS, Autism Diagnostic Observation Schedule; ADI-R, Autism Diagnostic Interview-Revised; RBS-R, Repetitive Behavior Scale-Revised; SCQ, Social Communication Questionnaire; SDQ, Strengths and Difficulties Questionnaire; SDQ Total, SDQ Total Difficulties. In this model, autism emerges from a unitary polygenic aetiology. The study consists of four linked aims to understand whether the developmental trajectories and polygenic etiology of autism differs by age at diagnosis. In Aim 1, we modelled socioemotional and behavioural trajectories among autistic individuals in birth cohorts and investigated their association with age at autism diagnosis. In Aim 2, estimated the SNP-based heritability of age at autism diagnosis and whether it attenuates when accounting for various clinical and demographic factors. In Aim 3, we investigated whether the varying patterns of genetic correlations observed among different GWAS of autism can be explained by different polygenic factors associated with age at diagnosis. In Aim 4, we investigated the genetic relationship between the two autism polygenic factors and mental health and developmental phenotypes. Schematic diagram of the cohorts included in the study and the ages when data was collected for SDQ scores (dots) and autism diagnosis (in boxes). Reports of autism diagnosis were available at ages: MCS - 5,7,11,14; GUI - 9,13,17; LSAC-B - 7,9,11,13,15; and LSAC-K: 11,13,15,17. Sample sizes and the year of initial SDQ data collection for each cohort are shown on the ordinate axis. The age cutoff used in the Latent Growth Curve Models for each cohort is indicated by a red line. GUI was used only for Latent Growth Curve Models and excluded from Growth Mixture Models. We conducted two age at autism diagnosis GWAS. In addition, we conducted six case-control GWAS, where autistic individuals were stratified based on their age at autism diagnosis. Frequency histograms of age at autism diagnosis in iPSYCH and SPARK. Median and median absolute deviation (MAD) for age at diagnosis, and sample sizes (N) have been provided. Points represent genetic correlation estimates and whiskers indicate 95% confidence intervals. Green represents the earlier diagnosed autism GWAS (iPSYCH before 9 and SPARK before 6), and purple represents later diagnosed autism GWAS (iPSYCH and SPARK after 10). Asterisk (*) indicates significantly different genetic correlation between the earlier and later diagnosed GWAS (P < 0.05, two-tailed Z test). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Zhang, X., Grove, J., Gu, Y. et al. Polygenic and developmental profiles of autism differ by age at diagnosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250930034243.htm'>Blocking one protein supercharges the immune system against cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 13:25:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have discovered a way to make the immune system's T cells significantly more effective at fighting cancer. By blocking a protein called Ant2, they were able to reprogram how these cells consume and generate energy -- essentially rewiring their internal power supply. This shift makes T cells more active, resilient, and better at attacking tumors. A new study may pave the way for a new generation of cancer therapies -- by training the body's own immune system to work smarter and hit harder. "By disabling Ant2, we triggered a complete shift in how T cells produce and use energy," explains Prof. Berger. "This reprogramming made them significantly better at recognizing and killing cancer cells." In simpler terms, blocking this protein forces the immune cells to adapt their metabolism, turning them into stronger, faster, and more aggressive cancer fighters. The altered T cells exhibited greater stamina, faster replication, and sharper targeting of cancerous threats. Perhaps most importantly, the researchers showed that this metabolic rewiring can be triggered not only through genetic modifications but also with drugs -- opening the door for potential clinical applications. This discovery is part of a growing movement in cancer immunotherapy that focuses not only on guiding the immune system but upgrading its inner machinery. While more studies and clinical trials are needed, the implications of this breakthrough are promising: new treatments that harness the body's own defenses, fine-tuned for peak performance. "This work highlights how deeply interconnected metabolism and immunity truly are," says Prof. Berger. "By learning how to control the power source of our immune cells, we may be able to unlock therapies that are both more natural and more effective." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68096646/humans-emerged/'>Humans May Have Emerged Hundreds of Thousands of Years Earlier Than We Thought</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Homo sapiens were thought to have appeared about 600,000 years ago, but two closely related million-year-old skulls suggest that our species might be even more ancient. Homo sapiens are thought to have first emerged 600,000 years ago alongside Neanderthals and Denisovans, when these species all diverged from Homo erectus. When a 1.1 million-year-old skull unearthed near the Hanjiang River in China's Hubei Province back in 1990, it was assumed to be Homo erectus, as it predated the earliest known Homo sapiens bones. It turns out, however, that this might be a case of mistaken identity. Hominin remains from the Middle Pleistocene (between about 774,000 to 129,000 years ago)—found in places as distant from each other as China, South Africa and the Philippines—have shown so much morphological variation that some are even thought to be previously unknown species. What makes the skull especially confusing is its merging of more primitive Homo erectus features with more recognizable Homo sapiens features that arose later in human evolution. Known as Yunxian 2, it was found near another cranium, Yunxian 1 (not as well preserved), and both have now been revisited by an international team of researchers from Fudan University in China and the UK's Natural History Museum. Yunxian 2 in particular was almost completely reconstructed (except for a few missing fragments and teeth), but which species it belonged to was not immediately obvious. The Yunxian skulls remain the oldest Homo longi fossils to be uncovered, but they are still more evolved than some other specimens. As seen in Yunxian 2, features that Homo longi have in common with modern humans include a larger brain, longer and lower foreheads, and eyes that are spaced closer together. Homo sapiens fossils from that far back in time have not surfaced yet—if they had, it would be hard evidence for the much earlier evolution of both our species (and hominins with larger brains in general)—but there might still be a million-year-old Homo sapiens skull out there. The most ancient Homo sapiens fossils discovered in Africa are only 300,000 years old. They added that some features may mark an evolutionary transition, and that the “narrow temporal gap between Yunxian and deeper longi nodes suggests rapid early diversification of the longi clade, as in the sapiens and Neanderthal clades.” Have we really been around for over a million years? That could mean our species was walking with Neanderthals and Denisovans for 800,000 years. There was one more skull found at the same site as Yunxian 1 and 2—Yunxian 3—which needs further study. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63790-8'>Unifying machine learning and interpolation theory via interpolating neural networks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 12:21:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Computational science and engineering are shifting toward data-centric, optimization-based, and self-correcting solvers with artificial intelligence. This transition faces challenges such as low accuracy with sparse data, poor scalability, and high computational cost in complex system design. This work introduces Interpolating Neural Network (INN)-a network architecture blending interpolation theory and tensor decomposition. INN significantly reduces computational effort and memory requirements while maintaining high accuracy. Thus, it outperforms traditional partial differential equation (PDE) solvers, machine learning (ML) models, and physics-informed neural networks (PINNs). It also efficiently handles sparse data and enables dynamic updates of nonlinear activation. Demonstrated in metal additive manufacturing, INN rapidly constructs an accurate surrogate model of Laser Powder Bed Fusion (L-PBF) heat transfer simulation. It achieves sub-10-micrometer resolution for a 10 mm path in under 15 minutes on a single GPU, which is 5-8 orders of magnitude faster than competing ML models. This offers a new perspective for addressing challenges in computational science and engineering. Emerging scientific computational methods are moving from relying on explicitly defined and modular programming to the adoption of neural network-based self-corrective algorithms. The shift towards software 2.0 partially resolves the issue of labor-intensive programming in Software 1.0 and has significantly advanced the domain of large language models (LLMs) and other foundational models2,3. However, adopting a similar technology in the fields of computational engineering and science presents unique challenges: generality, data-hunger, scalability, and sustainability. Machine learning (ML)-based partial differential equation (PDE) solvers do not generalize with the same level of accuracy as traditional numerical methods4,5,6,7,8. The success of purely data-driven ML models on sparse datasets is always limited10. The scalability of the neural network-based solver is of major concern when dealing with extremely high-dimensional inverse system design. High-fidelity numerical methods with over millions of degrees of freedom (DoFs) are often required for simulating the additive manufacturing (AM) process or integrated circuit (IC) at extremely fine resolution and multi-physics/multiscale setup11,12,13. Furthermore, the solution is often unusable for different materials or processing scenarios and requires repeated solutions of extremely large systems. Finally, the energy cost associated with very large-scale ML models is becoming prohibitive. The energy consumption required to train the largest ML models increased by six orders of magnitude from 2012 to 201814, and there are growing concerns about fresh water consumption for cooling massive AI data centers15. Hence, the sustainability of this new generation of computational tools will become a major concern. In this context, this paper introduces an Interpolating Neural Network (INN), a computational framework for constructing machine learning models that emulate the behavior of PDE solvers, such as the finite element method (FEM)16,17,18, and a purely data-driven tool, such as deep neural networks (DNNs) with the same architecture. The development of INNs was inspired by the hierarchical deep-learning neural network (HiDeNN) framework and its variants7,17,18,19,20,21,22, which were originally designed for solving PDEs. INNs build on this foundation by reinterpreting components of numerical analysis as machine learning parameters, thereby generalizing their use for training, calibration, and the solution of various scientific and engineering problems. The goal of an INN is to create generalizable, transferable, scalable, and sustainable tools for the next generation of engineering software. Curiously, although FEM and DNNs have different origins and purposes, they share one thing in common: they approximate functions. The former approximates a function with interpolation functions (or shape functions—the frequently used terminology in FEM) generated from discrete nodal coordinates in the input domain and with the corresponding nodal values in the output domain, while the latter constructs a black-box function space using neural networks parameterized with weights, biases, and activation functions. Nevertheless, it is possible to view a DNN as a general case of interpolation-based methods, such as FEM where the nodes are shifted from space-time input domain to a more general input domain that can comprise any quantity (word embeddings, pixel values, etc. ), interconnectivity of FEM nodes is expressed as computational graph (connectivity in graph-based networks), and interpolation functions act as the nonlinear activation functions. On top of this, separating each input parameter dimension with tensor decomposition (TD)19,20,21,23 can make INNs scalable. Since the interpolation functions are interpretable and adaptable (i.e., one can adjust the basis functions that construct interpolation functions), and TD is scalable, INNs fuse these ideas to create accurate and sustainable computational tools. INNs follow three steps: 1) discretize an input domain into non-overlapping segments whose bounds are denoted by interpolation nodes, 2) construct a computational graph with the interpolation nodes, and formulate interpolation functions similar to message passing operations in graph-based neural networks24,25,26, and 3) optimize the values and coordinates of the interpolation nodes for a given loss function. When the input domain is discretized with a regular mesh (see the special case in Fig. 1), INNs leverage TD to convert the growth rate of the computational cost to the problem dimension from exponential to linear. INNs are very flexible in performing 1) data training, 2) PDE solving, or 3) parameter calibration while using the same architecture. The computer codes can be found at https://github.com/hachanook/pyinn. The INN box illustrates the graph representation of the input domain discretized with an arbitrary Euclidean mesh (left) and a regular mesh as a special case (right). Node and edge features are given from the mesh. After Q-step message passing, each node j will store an interpolation function N[Q](j)(x). Finally, the readout operation sums the product of the interpolation functions N[Q](j)(x) and nodal values u(j). The superscripts with square brackets [] and parentheses () denote the message passing step and graph node index, respectively. The INN trainer employs data-driven loss functions (e.g., mean squared error loss for regression), while the INN solver adopts a residual loss of a partial differential equation (PDE). This article discusses the details of constructing an INN and implementing it to solve several benchmark problems in a purely data-driven setting, and solving both forward and inverse problems when a PDE is known. As a demonstrative engineering application of INNs, a simulation of process variation in laser powder bed fusion (L-PBF) AM is considered. In L-PBF, a laser is used to melt and fuse layers of metal powders to build a metal component. As the laser spot size ranges from 50 to 100 μm11,27, physical simulations of AM require a sub-10-micron resolution that challenges most traditional numerical methods12,28. The required computational resources become prohibitive, particularly when they are employed to generate training data for a surrogate model within a vast parametric space. 2, the INN presents a new direction toward part-scale AM simulations for online manufacturing control, demonstrating performance gains by orders of magnitude. The article also conducts various numerical experiments that cover computer science and engineering domains to demonstrate the superior capabilities of INNs in training, solving, and calibrating. Detailed problem definition and explanation can be found in the Discussion section. We compare the single-scale finite-difference method (FDM) solver28, the variational multiscale FEM solver12, and the INN solver with CANDECOMP/PARAFAC (CP) decomposition and Q = 2. The data points for the first two methods with dashed marker edges are estimated as they are intractable with a single GPU, while those of INN with a solid marker edge are computed. All benchmarks were conducted with a single GPU, NVIDIA RTX A6000 GPU with 48 GB VRAM. Consider a regression problem that relates I inputs and L outputs. The first step of an INN is to discretize the input domain in the I-dimensional Euclidean space into a mesh, which can be as general as an unstructured irregular mesh or as specific as a structured regular mesh. In any case, a mesh in the Euclidean space can be readily represented as a graph—the most general form of a discretized input—as illustrated in Fig. The graph nodes in an INN are arbitrary points in the input domain, which are irrelevant to the data structure. This is a key distinction of an INN from the graph neural network (GNN), which will be elaborated upon in the following discussion. Suppose there are J nodes (or vertices) and E edges in the graph, where the input domain is discretized with C non-overlapping segments. Each segment occupies a subspace of the I-dimensional input domain. An edge that links node j and k has a feature e(j, k) that stores indices of segments connected to the edge. For example, when I = 3, L = 2, node 10 in Fig. The edge connecting nodes 30 and 12 stores e(30, 12) = {5, 7}. Once the input domain is properly discretized, INN message passing is conducted to build an interpolation function on each graph node. In contrast, the INN message passing returns a function (i.e., interpolation function N(j)(x), or often called shape function in FEM) for each node (j = 1, …, J) as a hidden state. A general Q-step message passing of an INN, \({{{\mathcal{M}}}}^{[q]}(*)\), can be expressed as: where \({{{\mathcal{N}}}}^{s}(j)\) is a set of neighboring nodes of the center node j with s connections (i.e., s-hops, see Fig. 3 and Supplementary Information (SI) Section 1.1 for high-dimensional cases). The operation \({{{\mathcal{M}}}}^{[q]}(*)\) constructs an interpolation function N[q](j)(x) for a graph node x(j) using neighboring nodal coordinates and edge information, x(k), e(j, k), and the interpolation function of the previous message passing, N[q−1](j)(x). The function N[Q](j)(x) is the INN interpolation function at node j after Q message passing. The \({{{\mathcal{N}}}}^{s}(j)\) is a set of neighboring nodes of the center node j with s connections (or s-hops). It is worth noting that the interpolation functions used throughout this article satisfy the Kronecker delta property, i.e., N[q](j)(x(k)) = δjk, meaning that the INN function space exactly passes the interpolation points. However, this condition may be relaxed depending on the specific problem and the chosen interpolation method. For instance, if the training data is highly noisy or spline basis functions are used, this condition may no longer be necessary. With a single message passing (Q = 1) and s = 1 hop, the INN message passing degenerates to the standard FEM linear shape function (or Lagrange polynomials of order P = 1). 3 (left), it is the most localized approximation of an INN. In this case, the message passing operation at node j = 3 is written as: One can progressively enlarge the support domain and the approximation capability of an interpolation function by adding message passing with a higher hop. For instance, the second message passing with s = 2 hop constructs compact-supported interpolation functions with higher nonlinearity. This is theoretically equivalent to the generalized finite element method (GFEM)29 and convolution hierarchical deep-learning neural networks (C-HiDeNN)7,20. The message passing with a max-hop (Fig. 3, right) can even make it a non-local approximation where the support of interpolation functions occupies the entire input domain, mimicking meshfree methods that exhibit superconvergence (i.e., faster convergence rate than the complete order of polynomial basis)30,31,32. Depending on the choice of interpolation technique, other hyperparameters can be involved in \({{{\mathcal{M}}}}^{[q]}(*)\), such as the activation (or basis) function and dilation parameter7,20. This enables adaptive update of INN interpolation functions during training to improve accuracy (see SI Section 2.2). A more comprehensive discussion on functional approximation through interpolation theory and subsequent solvers can be found in refs. See SI Section 1 for various choices of the message passing operation. Regardless of the choice of interpolation technique for the message passing, the global DoFs (i.e., number of nodes) remain constant. INNs adapt nodal connectivity through the adjacency matrix and basis functions to reproduce almost any interpolation technique available in numerical methods. The compact-supported interpolation functions enable INNs to be optimized locally and promptly, making INNs distinguishable from MLPs. Since an MLP is a global approximator, INNs with compact-supported interpolation functions 1) converge faster than MLPs with the same number of trainable parameters, and 2) facilitate training on a sparse dataset. During the forward propagation, an input variable \({{\boldsymbol{x}}}\in {{\mathbb{R}}}^{I}\) enters each graph node's interpolation function N[Q](j)(x), followed by a graph-level readout operation: where the superscript [Q] is dropped for brevity (i.e., N[Q](j)(x) = N(j)(x)). The interpolation operator \({{\mathcal{J}}}\) will designate an interpolated field output throughout this paper. The readout operation can be written as a matrix multiplication: The matrix U is a horizontal stack of nodal values u(j), while \({{\bf{{{\mathcal{X}}}}}}(x)\) is a vectorized function of x that is parameterized with nodal coordinates x(j) during the message passing operation. The graph node features (i.e., coordinate (x(j)) and value (u(j))) are trainable parameters of the INN. If the nodal coordinates (x(j)) are fixed, one can find nodal values (u(j)) without changing the discretization of the input domain. Once the forward propagation is defined, the loss function is chosen based on the problem type: training, solving, or calibrating (see Method Section for details). When the input domain is discretized with a regular mesh, as illustrated in the special case of Fig. 1, we can significantly reduce the trainable parameters (or degrees of freedom, DoFs) by leveraging TD23 that makes INN scalable. Here, we introduce the two widely accepted TD methods: Tucker decomposition36,37 and CANDECOMP/PARAFAC (CP) decomposition38,39,40. One of the widely used TD methods is Tucker decomposition36,37. It approximates a high-order tensor with Ji nodes in i-th dimension as a tensor contraction between dimension-wise matrices and a core tensor \({{\bf{{{\mathcal{G}}}}}}\), which has the same order of the original tensor but with smaller nodes Mi (<Ji). The Mi is often called a “mode” to distinguish it from the original node Ji. Consider a three-input (I = 3) and one output (L = 1) system, and assume the input domain is discretized with J = J1 × J2 × J3 nodes, as shown in the special case box of Fig. To facilitate tensor notation, the nodal values will be denoted with left/right super/sub scripts, \(\begin{array}{l}m\\ i\end{array}{u}_{l}^{(j)}\), where \(i\in {{\mathbb{N}}}^{I}\) is the input index, \(m\in {{\mathbb{N}}}^{{M}_{i}}\) is the mode index, \(l\in {{\mathbb{N}}}^{L}\) is the output index, and \(j\in {{\mathbb{N}}}^{{J}_{i}}\) is the nodal index. The interpolated field \({{\mathcal{J}}}u({{\boldsymbol{x}}})\in {{\mathbb{R}}}^{L=1}\) can be represented as a Tucker product: where the core tensor \({{\bf{{{\mathcal{G}}}}}}\in {{\mathbb{R}}}^{{M}_{1}\times {M}_{2}\times {M}_{3}}\) is a trainable full matrix typically smaller than the original tensor that compresses the data. Here, \({{\bf{{{\mathcal{A}}}}}}{\times }_{a}^{b}{{\bf{{{\mathcal{B}}}}}}\) denotes the tensor contraction operation between the a-th dimension of tensor \({{\bf{{{\mathcal{A}}}}}}\) and the b-th dimension of tensor \({{\bf{{{\mathcal{B}}}}}}\). The \({{\mathcal{J}}}\begin{array}{l}\\ i\end{array}{{\boldsymbol{u}}}({x}_{i})\in {{\mathbb{R}}}^{{M}_{i}\times 1}\) is one-dimensional (1D) interpolated output of ith input dimension over Mi modes represented as: As illustrated in the special case box of Fig. 1, the message passing (blue arrow) only happens in the axial directions, yielding 1D interpolation functions: \({N}^{({J}_{i})}({x}_{i})\). When there is more than one output (L > 1), the interpolated field becomes a vector of L elements: The trainable parameters are the core tensors \({{{\mathcal{G}}}}_{l}\in {{\mathbb{R}}}^{{M}_{1}\times {M}_{2}\times {M}_{3}},l=1,\ldots,L\), and the nodal values \(\begin{array}{l}\\ i\end{array}{{{\boldsymbol{U}}}}_{l}\in {{\mathbb{R}}}^{{M}_{i}\times {J}_{i}},l=1,\ldots,L\), yielding a total count of \(L({\prod }_{i}^{I}{M}_{i}+\mathop{\sum }_{i}^{I}{M}_{i}{J}_{i})\) that scales linearly with the nodal discretization Ji, the number of modes M, and the number of outputs L. Tucker decomposition can be further simplified to CANDECOMP/PARAFAC (CP) decomposition38,39,40 by setting the core tensor \({{\bf{{{\mathcal{G}}}}}}\) as an order-I super diagonal tensor: \({{\bf{{{\mathcal{G}}}}}}\in {{\mathbb{R}}}^{{M}^{I}}\), M = M1 = ⋯ = MI, all zero entries except the diagonal elements. If we further set the diagonal elements of \({{\bf{{{\mathcal{G}}}}}}\) to be 1, the Tucker decomposition in Eq. where \({{\mathcal{J}}}\begin{array}{l}m\\ i\end{array}{{\boldsymbol{u}}}({x}_{i})\in {{\mathbb{R}}}^{L}\) and ⊙ represents multiplication in elements. In CP decomposition, the core tensor is no longer trainable; thus, the total trainable parameter becomes \(ML{\sum }_{i}^{I}\,{J}_{i}\). If the 1D interpolated fields \({{\mathcal{J}}}\begin{array}{l}m\\ i\end{array}{{\boldsymbol{u}}}({x}_{i})\) are constructed with Q = 1 message passing, Eq. The Q = 2 nonlinear INN approximation fields are called C-HiDeNN-TD20. It is important to note that both Tucker and CP decomposition replace a high-dimensional interpolation with one-dimensional interpolations that dramatically reduce the number of trainable parameters (or DoFs). As given in Table 1, the number of trainable parameters of the full interpolation scales exponentially with the input dimension I, whereas the Tucker and CP decomposition scales linearly with I. Considering the number of trainable parameters of multi-layer perceptrons (MLPs) scales quadratically with the number of hidden neurons, while that of INNs always scales linearly with the mode M, input I, output L, and discretization J, INNs with TD may dramatically reduce the model complexity and computing requirements. Therefore, INNs can be a sustainable alternative to traditional AI models. However, the reconstructed basis using TD loses expressibility because of the separated variables. There is no cross-term in the functional space of TD. Nevertheless, we can transform this into an advantage by enriching the 1D approximations with adaptive activation (see SI Section 2.2) for the Q-step message passing. Therefore, we can find a hybrid architecture that preserves the advantages of numerical methods (i.e., FEM) and neural networks while acknowledging that it loses something from both. GNNs have been widely applied to various machine learning tasks involving graph-structured data41,42. These tasks are typically categorized into three levels: node-level, edge-level, and graph-level learning. INNs, on the other hand, leverage the graph structure to construct a network architecture, but the graph nodes do not have to conform to the data structure. They are arbitrary interpolation points in the input domain. Furthermore, an INN generally performs graph-level predictions; given an input x, it predicts an output u by summing the product of interpolation functions and interpolation values (see Eq. In SI Section 2.7, we present a benchmark involving the training of 3D FEA simulation results and compare the problem formulations and training outcomes across MLP, GNN, and INN models. INNs with CP TD have remarkable similarity with the proper generalized decomposition (PGD), which will be elucidated in two different aspects: function approximation and solution schemes. PGDs can utilize any functional space that admits the CP decomposition form (see Eq. Thus, an INN employing CP decomposition becomes functionally equivalent to the PGD. However, the INN framework was developed from a broader perspective: starting from general, non-separable function approximations and later incorporating structured decompositions such as Tucker and CP decomposition to effectively handle input domains discretized on regular grids. For PDE solving, the general rule of PGD is to find solutions mode by mode and in a decomposed space (1D or low-dimensional space) for each mode. This is often referred to as subspace iteration43. The INN solver, on the other hand, generally solves for all modes and dimensions at once. Detailed solution scheme of the INN solver can be found in SI Section 4.2 and44. Similarly, the general rule of an INN trainer is to optimize the entire trainable parameters simultaneously, akin to how weights and biases are jointly updated in MLPs during backpropagation. However, alternative optimization strategies are certainly possible. For example, one could train the INN sequentially by mode or by input dimension (similar to the way the PGD solves a PDE), or optimize a set of modes together, followed by optimizing another set of modes. Exploring such customized training schemes is a promising direction for future work. To highlight INN's advantages in speed (epoch at convergence) and storage (number of parameters) over MLP, we introduce a benchmark problem: a 10-input 5-output physical function45, (see SI Section 2.4 for details). Using this equation, we randomly generate 100,000 data points using a Latin hypercube sampling. MLPs with two and three hidden layers and with a sigmoid activation are tested, while INNs with CP decomposition, two (Q = 2) message passing, and s = 2, P = 2 polynomial activation are adopted. To investigate the convergence behavior depending on the number of parameters, we set the stopping criteria as training mean squared error (MSE): 4e-4 and counted the epoch at convergence. Since the physical function is deterministic (no noise) and we drew a sufficiently large number of data, no considerable overfitting was observed. Figure 4a reveals that given the same number of trainable parameters, INNs converge significantly faster than MLPs. MLPs with more parameters tend to stop at earlier epochs, however, even the largest MLP (3 layers, 100 neurons) with 21,805 parameters converged at the 45th epoch, while that of INN (10 seg., 14 modes) with only 7700 parameters converges at the 14th epoch. Notice that all benchmarks were conducted using NVIDIA RTX A6000 GPU with 48GB VRAM, and the training time per epoch for INN and MLP was indistinguishable (around 1 second per epoch). This benchmark demonstrates that the INN trainer is lightweight and fast-converging compared to traditional MLPs, underlining the sustainability and scalability of INNs. Other benchmarks of the INN trainer are elucidated in SI Section 2: SI Section 2.2: Adaptive INN activation function, SI Section 2.3: Training one-input two-output function, SI Section 2.5: Four-input one output battery manufacturing data, SI Section 2.6: Spiral classification, and SI Section 2.7: Learning FEA simulation results. We highlight SI Section 2.2, where the INN interpolation functions are updated during training to enhance model accuracy. This demonstrates INN's capability of adaptive activation function, a widely accepted concept in AI and the scientific machine learning literature46,47. An in-depth discussion on the choice of INN hyperparameters is provided in SI Section 2.10. We set the same optimization condition: ADAM optimizer; learning rate: 1e-3; batch size: 128. The number of neurons per hidden layer is denoted for each MLP data point, while the number of segments and modes are denoted for each INN data point. MLP used the sigmoid activation function, while INN with CP decomposition adopted Q = 2, s = 2, P = 2. b A 1D Poisson's equation is solved with PINN and INN. PINN is made of a 1-layer MLP with a varying number of neurons. Randomly selected 10k collocation points are used to compute the PDE loss. With a batch size of 128, 2000 epochs are conducted using an ADAM optimizer with a learning rate of 1e-1. INN Q = 1, s = 1, P = 1 with the galerkin formulation is equivalent to FEM with linear elements. All solvers and trainers are graphics processing units (GPU) optimized with the JAX library65. INNs can achieve well-known theoretical convergence rates when solving a PDE7. This benchmark solves a 1D Poisson's equation defined in the Method Section with the H1 norm error estimator. It is mathematically proven that a numerical solution of FEM or any interpolation-based solution method exhibits a convergence rate of P, which is the complete order of the polynomial basis used in the interpolation20. INNs with different hyperparameters are studied. As expected from the convergence theory, the H1 error of INNs with different P converges at a rate of P. On the other hand, PINNs constructed with MLP do not reveal a convergent behavior. Only a few theoretical works have proved the convergence rate of PINNs under specific conditions48. A PDE solver needs to have a stable and predictable convergence rate because it guides an engineer in choosing the mesh resolution and other hyperparameters for achieving the desired level of accuracy. We also demonstrate the convergent behavior of INN solvers with and without CP decomposition for a 3D linear elasticity equation in SI Sections 4.5 and 4.6. Now we illustrate how INNs can be applied to advance various facets of metal AM. Note that the INN with CP decomposition will be referred to as INN in this section for brevity. L-PBF is a type of metal AM in which metal powders are layered and melted by a laser beam to create objects that match the provided CAD designs49. The added flexibility of L-PBF comes with a huge design space and spatially varying microstructure after manufacturing. Therefore, considerable research is devoted to employing in situ monitoring data for real-time control of L-PBF processes to minimize the variability and optimize the target properties50. Computational modeling of L-PBF spans the simulation of the manufacturing process with a high-dimensional design space (solving), the identification of model parameters from sparse experimental data (calibrating), and the online monitoring and control of the L-PBF (training). The governing equation for modeling the manufacturing process is given by a heat conduction equation with a moving laser heat source Sh: where the equation involves spatial variables x, temporal variable t, and four variable parameters: thermal conductivity k, laser power P, absorptivity η, and laser depth d. For the simplest case of a single-track laser path, the initial condition is T(x, 0) = T0(x) and the boundary conditions (BCs) are \(T=\widetilde{T}\) in ΓD for Dirichlet BC and \(\frac{\partial T}{\partial {{\bf{x}}}}\cdot {{\bf{n}}}=\bar{q}\) in ΓN for Neumann BC. The \(\bar{q}\) includes contributions from convection, radiation, and evaporation on the boundary surfaces. The spatial domain boundary Γ = ΓD ∪ ΓN. Details of the initial and boundary conditions can be found in SI Section 4. The first application is fully data-driven, where the data are generated from numerical methods, and the INN trainer is used to develop an online control system for L-PBF AM. The goal is to maintain a homogeneous temperature of the melt pool (i.e., a molten region near the laser spot) across each layer using real-time sensor data, under the hypothesis that a homogeneous melt pool temperature across each layer of material deposition will reduce the variability in the microstructure and mechanical properties. The laser power at each point of a certain layer needs to be controlled (increased/decreased by a certain amount compared to the previous layer) to achieve this homogeneity in the melt pool temperature. The fundamental challenge of implementing a model predictive control system for such an application is the computational resources required for forward prediction (involving finite element or computational fluid dynamics models) and inverse prediction. 5 on the row for training, the INN trainer aims to provide a reliable and memory-efficient surrogate model that can replace computation-heavy mechanistic models in predictive control systems. In this application, the database is generated by fusing experiments and computational methods (see SI Section 2.8 for details) for aluminum alloy. First, a data-driven real-time online monitoring and feedback control tool is formulated with an INN that uses only 18% training parameters of MLP and is 18-31 times faster than MLP. The calibration problem develops a reduced-order model of laser powder bed fusion (L-PBF) AM to calibrate the heat source parameters from experimental data. Finally, the INN solver solves a space-time-parameter heat transfer equation, resulting in a significant storage reduction and faster simulations (see Table 2 for numerical comparisons). The mechanistic features are precomputed based on the toolpath and geometry of the build to generalize a data-driven model for unseen toolpath and geometric features51. Finally, the control software calculates how much change in laser power is required at each point, and sends the information to the machine. The forward model is trained on experimentally observed data, while the inverse model is trained on computational data coming from a finite-difference solver28. The INN model is applied for both forward and inverse prediction and is compared with the MLP. Figure 5 shows that for the inverse model, INN is at least 18 times faster to train compared to MLP to reach the same level of training error, and it reduced the number of trainable parameters by a staggering amount of 82%. In SI Section 2.8, the INN inverse model is dynamically updated with three different amounts of data. This example highlights the dynamic update capability of INNs. In this example, an INN calibrates the heat source function Sh in Eq. A Gaussian beam profile is modeled as a volumetric heat source52, which is written as: (9), and rb is the Gaussian profile's standard deviation that controls the beam's width. Here, V is the laser scan speed, and RHF is a predetermined residual heat factor based on the laser toolpath53. The major challenge of this problem is to achieve a highly accurate surrogate model given a sparse set of training data. INNs outperform MLPs in training these scarce data because they are rooted in the interpolation theory. For the experimental data, small cylindrical samples, ~2 mm in diameter and 10 mm in height, are fabricated using a Gaussian laser beam at varying power levels and scanning speeds. During the process, real-time temperature field data recorded as TEP are collected for calibration purposes (details of the experiments can be found in SI Section 3.1). An INN with 1760 trainable parameters is trained on 28 sets of melt pool temperature data generated by the finite-difference method (FDM)28 using 4 different power-scanning speed setups. Later, this trained INN is used as a surrogate to optimize the three parameters (p1, p2, p3) such that the heat source in Eq. For comparison, a 3-layer MLP with 2689 trainable parameters (roughly the same order of magnitude as the INN) is trained using the same data. We observed that INNs converge well within 100 epochs, while MLPs require over 1000 epochs to converge to a test MSE 10 times larger than the INN, given all other setups are the same. Details on this training and calibration process are provided in SI Section 3.1. The INN- (definition of the INN calibrator can be found in the Method Section) and MLP-based calibrations are repeated 50 times to provide a statistical comparison (see Fig. Even with only 28 sets of simulation data, the calibrated parameters using the INN produce a mean melt pool temperature within a 7.9% difference of the experimental data. In contrast, the parameters calibrated from the MLP surrogate produce a difference of 44.1%, which is unsatisfactory. This is expected since MLP is a general model requiring a large amount of data to achieve a reliable representation of the problem. On the other hand, the INN achieves a much lower difference, showing that the model can learn underlying physics even with a scarce dataset. INNs can also be utilized as a solver to directly obtain the surrogate model of the AM process involving Space-Parameter-Time (S-P-T) dependencies without generating training data. In this case, an INN is used as a parametric interpolation function in the S-P-T solution space that satisfies the governing physical equation and corresponding boundary/initial conditions. Here, we showcase the power of INN by solving a single-track scan in the L-PBF process. We are interested in obtaining the temperature field, which not only depends on space x and time t but also on the laser and material parameters p. In other words, the surrogate model f is a mapping from the S-P-T continuum to the temperature field: \(f:={{\mathbb{R}}}^{8}\to {\mathbb{R}}\). The surrogate model f can also be obtained using the standard data-driven approach. In the offline stage, training data are generated by repeatedly running numerical solvers (such as FDM or variational multiscale method (VMS)) in the Space-Time domain with different sets of parameters p. Then, a surrogate model is trained with the data. The data-driven approach suffers from the curse of dimensionality when the parametric space is high-dimensional, resulting in expensive computation for repetitive data generation, huge memory costs for running full-scale simulations, and excessive disk storage of offline training data. We estimate the time and storage required for the data-driven approaches with FDM and VMS, as shown in Table 2 and illustrated in Fig. Unlike the data-driven approaches, INN solvers treat the parameters p as additional parametric inputs. As a result, INNs obtain a parametric surrogate model f = u(x, p, t) directly from the governing equation without going through the cumbersome offline data generation and surrogate model training. Different solution schemes can be used to obtain the PDE solution, such as PGD54, which solves the solution mode-by-mode, or TD, which solves all modes simultaneously19,44 (see SI Section 4.2 for details). As can be seen from Table 2, INN is projected to be 108 and 105 faster than FDM and VMS, respectively, for offline data generation. Moreover, INN requires significantly smaller storage as opposed to offline-online data-driven methods. Therefore, INN can exhibit good sustainability for high-dimensional, large-scale PDEs. Furthermore, our results prove that the INN solution (or the S-P-T surrogate model) achieves an exceptionally high accuracy of R2 = 0.9969, considering that most data-driven approaches for S-P-T problems trained on numerical simulation data suffer from low model accuracy, typically below R2 = 0.955. It is worth mentioning that the physics-informed neural networks (PINNs) can handle the S-P-T problem similarly to the INN solver. However, the number of collocation points scales exponentially with the number of inputs, and our preliminary study revealed that PINN fails to converge for the same 8-dimensional S-P-T problem after consuming considerable computational resources (see SI Section 4.4). This certifies that the INN has good generalizability to achieve high accuracy, as classical numerical algorithms can solve high-dimensional problems that classical numerical methods cannot solve. This article demonstrates that INNs can train, solve, and calibrate scientific and engineering problems that are extremely challenging or prohibitive for existing numerical methods and machine learning models. The keys to INN's success are 1) it interpolates nodal values with well-established interpolation theories and 2) it leverages the TD for model order reduction. Due to the reduced number of parameters without compromising predictive accuracy, INNs become an efficient substitute for MLPs or PINNs. There are numerous research questions that could generate significant interest across various fields, including, but not limited to, scientific machine learning, applied mathematics, computer vision, and data science. Thus far, the superior efficiency of INNs over MLPs and PINNs has been clearly demonstrated in learning and solving deterministic problems, particularly in cases where the number of input and output variables is relatively low (e.g., fewer than 20). However, it remains uncertain whether INNs can outperform established AI models in addressing problems characterized by significant uncertainty or involving hundreds to thousands of input and output variables. Indeed, no single AI model consistently outperforms others across all problem domains. A comprehensive investigation into the applicability and limitations of INNs across a broader range of scenarios is required. Depending on the problem, INN's superior training efficiency may overfit data. Multi-resolution aspect (i.e., a varying mesh resolution or INN activation functions across TD modes) of INNs needs to be investigated. The interpretability of INNs should be investigated. INNs may address the challenge of model-based interpretability (a terminology defined from56), which involves developing models that are both simple enough to be easily understood by developers and capable of maintaining high predictive accuracy. Similar to INN solvers, the convergence of INN trainers can be studied both mathematically and numerically. Since INNs can be used as both a solver for physics-based problems and a function approximator for data-driven problems, it is of interest to combine the two behaviors into one single model to solve large-scale problems involving both complex physics and scarce data, or problems with incomplete mathematical models. The superior performance of INN solvers may facilitate multiscale analysis in a vast parametric space. One can integrate the S-P-T INN solver with a concurrent multiscale analysis framework such as self-consistent clustering analysis57,58 and open a new direction towards parameterized multiscale analysis. The current INN code has been optimized to run on a single GPU. Since the INN forward pass over multiple modes can be parallelized, multi-GPU programming will further speed up the code. Although Tucker decomposition is introduced in the manuscript, the numerical experiments conducted in this article mostly focus on the CP decomposition. In this paper, the authors have proposed a formidable research problem with a lot of avenues to explore in future research, for example, how to autonomously design the interpolation structure or how to decide when to use data-driven versus equation solving approach. The article tackles some of the aspects of the proposed research, illustrated by concrete examples and programming concepts, but much research remains to be done to obtain its full general solution and a complete mathematical understanding. A general INN forward propagation is provided in Eq. INN can be used for data training as any other neural network architecture. While MLPs optimize weights and biases during training, INN trainers optimize nodal values \({{\boldsymbol{U}}}={\{{{{\boldsymbol{u}}}}^{(j)}\}}_{j=1,\cdots,J}\) (and, if needed, nodal coordinates (\({{\boldsymbol{X}}}={\{{{{\boldsymbol{x}}}}^{(j)}\}}_{j=1,\cdots,J}\))) under a given loss function and training data. An MSE loss function for this regression problem is defined as: Finally, an optimization is formulated as follows. Here, we introduce a formulation with CP decomposition to solve the generalized space (x)—parameter (θ)—time (t) (S-P-T) PDE, whose computational cost is prohibitively high for most numerical methods and machine learning approaches54. where \({{\mathcal{L}}}\) is the general partial differential operator (can be linear or nonlinear), \({{\boldsymbol{x}}}\in {{\mathbb{R}}}^{d}\) is the spatial input, \({{\boldsymbol{p}}}\in {{\mathbb{R}}}^{k}\) is the parametric input, and \(t\in {\mathbb{R}}\) is the temporal input. An INN solution field in the S-P-T domain can be written as: where d and k are the spatial dimension and the number of parameters, respectively. Similar to the trainer, the goal is to find nodal values U (and nodal coordinates X, if one wants to adapt the mesh). where δu is the weight function that varies with the choice of formulation: Galerkin, Petrov-Galerkin, collocation, least square, etc. Due to the Kronecker delta property of INN interpolation functions, Dirichlet boundary conditions and initial conditions can be strongly imposed. As a result, only the weighted sum residual is considered in Eq. This distinguishes INNs from most data-driven PDE solvers that weakly impose these conditions63. In this paper, we focus on the Galerkin formulation where the same function space is used for both trial (solution) and test (weight) functions. (15) and leveraging the integration by parts, integrals of higher order differential operators \({{\mathcal{L}}}\) can be transformed into the integration of 1st order derivatives. This integral can be efficiently computed using 1D Gaussian integration thanks to CP decomposition. (15) depend on the type of PDEs. In this paper, INN solvers are applied to solve a heat transfer equation in metal AM and a linear elasticity equation in solid mechanics problems. (15) for the parametric transient heat transfer equation is introduced in SI Section 4. The word calibration in mathematics refers to the reverse process of regression, where a known or measured observation of the output variables (u*) is used to predict the corresponding input variables (x*). By definition, it is analogous to solving an inverse problem in engineering design. To build a good calibrator, having an accurate forward model is of paramount importance, followed by building a good optimizer for the inverse problem. A trained or solved INN has the potential to be a superior candidate for the forward model inside a calibrator because it is fully differentiable and accurate, equipped with fast inference time (~milliseconds). A general formulation of the INN calibrator is described below: where \({{\mathcal{J}}}{{\boldsymbol{u}}}({{\boldsymbol{x}}})\) is the trained or solved forward model, \({{{\boldsymbol{u}}}}_{k}^{*}\) is the k-th measured observation and x* is the calibrated input variable. The INN calibrator applied to the heat source calibration task in metal AM can be found in the Discussion Section, while other benchmarks can be found in SI Section 3. We borrow the manufactured problem first introduced in ref. The 1D Poisson's equation is defined as: where Δ is the Laplace operator. When the body force is given as: The H1 norm error estimator is defined as: where uh(x) is the approximated solution. The benchmark data that support the findings of this study are available from https://github.com/hachanook/pyinn or a released version64. The computer codes used for the benchmarking can be found at https://github.com/hachanook/pyinn or a released version64. Codes for AM examples can be provided upon a reasonable request. Moor, M. et al. Foundation models for generalist medical artificial intelligence. Lu, L., Jin, P., Pang, G., Zhang, Z. & Karniadakis, G. E. Learning nonlinear operators via deeponet based on the universal approximation theorem of operators. Zhang, C. & Shafieezadeh, A. Simulation-free reliability analysis with active learning and physics-informed neural network. Goswami, S., Kontolati, K., Shields, M. D. & Karniadakis, G. E. Deep transfer operator learning for partial differential equations under conditional shift. Park, C. et al. Convolution hierarchical deep-learning neural network (c-hidenn) with graphics processing unit (GPU) acceleration. Can physics-informed neural networks beat the finite element method? Weak baselines and reporting biases lead to overoptimism in machine learning for fluid-related partial differential equations. Poulinakis, K., Drikakis, D., Kokkinakis, I. W. & Spottswood, S. M. Machine-learning methods on noisy and sparse data. A. et al. Physics guided heat source for quantitative prediction of IN18 laser additive manufacturing processes. Leonor, J. P. & Wagner, G. J. Go-melt: GPU-optimized multilevel execution of lpbf thermal simulations. Shih, M., Chen, K., Lee, T., Tarng, D. & Hung, C. Fe simulation model for warpage evaluation of glass interposer substrate packages. Crawford, K. Generative AI's environmental costs are soaring–and mostly secret. Liu, W. K., Li, S. & Park, H. S. Eighty years of the finite element method: birth, evolution, and future. Zhang, L. et al. Hierarchical deep-learning neural networks: finite elements and beyond. Saha, S. et al. Hierarchical deep learning neural network (hidenn): an artificial intelligence (ai) framework for computational science and engineering. Zhang, L., Lu, Y., Tang, S. & Liu, W. K. Hidenn-td: reduced-order hierarchical deep learning neural networks. Lu, Y. et al. Convolution hierarchical deep-learning neural networks (c-hidenn): finite elements, isogeometric analysis, tensor decomposition, and beyond. Li, H. et al. Convolution hierarchical deep-learning neural network tensor decomposition (c-hidenn-td) for high-resolution topology optimization. Liu, Y. et al. Hidenn-fem: a seamless machine learning approach to nonlinear finite element analysis. Kolda, T. G. & Bader, B. W. Tensor decompositions and applications. Wu, Z. et al. A comprehensive survey on graph neural networks. Zhou, J. et al. Graph neural networks: a review of methods and applications. Ju, W. et al. A comprehensive survey on deep graph representation learning. Selective laser melting of in situ titanium–titanium boride composites: processing, microstructure and mechanical properties. Efficient gpu-accelerated thermomechanical solver for residual stress prediction in additive manufacturing. Tian, R. Extra-dof-free and linearly independent enrichments in gfem. Liu, W. K., Jun, S. & Zhang, Y. F. Reproducing kernel particle methods. & Belytschko, T. Reproducing kernel particle methods for structural dynamics. & Foster, J. T. Super-convergence of reproducing kernel approximation. Davis, P. J. Interpolation and approximation (Courier Corporation, 1975). Brenner, S. C.The mathematical theory of finite element methods (Springer, 2008). Kunoth, A. et al. Splines and PDEs: From approximation theory to numerical linear algebra (Springer, 2018). Tucker, L. R. Some mathematical notes on three-mode factor analysis. Foundations of the parafac procedure: models and conditions for an “explanatory” multi-modal factor analysis. Analysis of individual differences in multidimensional scaling via an n-way generalization of “eckart-young” decomposition. Towards a standardized notation and terminology in multiway analysis. Scarselli, F., Gori, M., Tsoi, A. C., Hagenbuchner, M. & Monfardini, G. The graph neural network model. Zhao, Y. et al. A review of graph neural network applications in mechanics-related domains. Chinesta, F. & Ladevèze, P. Proper generalized decomposition. In: Model Order Reduction, Snapshot-Based Methods and Algorithms, vol. Guo, J. et al. Tensor-decomposition-based a priori surrogate (taps) modeling for ultra large-scale simulations. Surjanovic, S. & Bingham, D. Virtual library of simulation experiments: test functions and datasets." Simon Fraser University, Burnaby, BC, Canada, accessed May 13 (2015). Lau, M. M. & Lim, K. H. Review of adaptive activation function in deep neural network. In 2018 IEEE-EMBS Conference on Biomedical Engineering and Sciences (IECBES), 686–690 (IEEE, 2018). Jagtap, A. D., Kawaguchi, K. & Karniadakis, G. E. Adaptive activation functions accelerate convergence in deep and physics-informed neural networks. & Karniadakis, G. E. On the convergence of physics informed neural networks for linear second-order elliptic and parabolic type PDEs. Bhavar, V. et al. A review on powder bed fusion technology of metal additive manufacturing. Cai, Y., Xiong, J., Chen, H. & Zhang, G. A review of in-situ monitoring and process control system in metal-based laser additive manufacturing. Kozjek, D. et al. Data-driven prediction of next-layer melt pool temperatures in laser powder bed fusion based on co-axial high-resolution planck thermometry measurements. Li, Y. et al. Statistical parameterized physics-based machine learning digital shadow models for laser powder bed fusion process. A residual heat compensation based scan strategy for powder bed fusion additive manufacturing. Chinesta, F., Ladeveze, P. & Cueto, E. A short review on model order reduction based on proper generalized decomposition. Towards a digital twin framework in additive manufacturing: machine learning and bayesian optimization for time series process optimization. Murdoch, W. J., Singh, C., Kumbier, K., Abbasi-Asl, R. & Yu, B. Definitions, methods, and applications in interpretable machine learning. Liu, Z., Bessa, M. & Liu, W. K. Self-consistent clustering analysis: an efficient multi-scale scheme for inelastic heterogeneous materials. Yu, C., Kafka, O. L. & Liu, W. K. Self-consistent clustering analysis for multiscale modeling at finite strains. Guo, J. et al. Convolutional hierarchical deep learning neural networks-tensor decomposition (c-hidenn-td): a scalable surrogate modeling approach for large-scale physical systems. On the deterministic solution of multidimensional parametric models using the proper generalized decomposition. & Karniadakis, G. E. Variational physics-informed neural networks for solving partial differential equations. & Karniadakis, G. E. hp-vpinns: Variational physics-informed neural networks with domain decomposition. Raissi, M., Perdikaris, P. & Karniadakis, G. E. Physics-informed neural networks: a deep learning framework for solving forward and inverse problems involving nonlinear partial differential equations. Bradbury, J. et al. JAX: composable transformations of Python+NumPy programs http://github.com/jax-ml/jax (2018). We express our sincere gratitude to Dr. Gino Domel, Mr. Joseph P. Leonor, Mr. Stefan Knapik, and Mr. Vispi Karkaria from Northwestern University, and Dr. Yingjian Liu and Mr. Monish Yadav Pabbala from UT Dallas for their invaluable comments and support throughout this work. These authors contributed equally: Chanwook Park, Sourav Saha. Department of Mechanical Engineering, Northwestern University, Evanston, IL, USA Chanwook Park, Xiaoyu Xie, Wei Chen, Gregory J. Wanger, Jian Cao & Wing Kam Liu HIDENN-AI, LLC, Evanston, IL, USA Chanwook Park, Jiachen Guo, Dong Qian & Wing Kam Liu Kevin T. Crofton Department of Aerospace and Ocean Engineering, Virginia Polytechnic Institute and State University, Blacksburg, VA, USA Theoretical and Applied Mechanics Program, Northwestern University, Evanston, IL, USA School of Engineering, Brown University, Providence, RI, USA Oden Institute for Computational Engineering and Sciences, University of Texas at Austin, Austin, TX, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and J.C. participated in developing the original ideas, provided feedback, and revised the manuscript. provided key motivations and critical insights to the development of INN. supervised the entire project, suggested revisions, and acquired funding. The authors declare no competing interests. Nature Communications thanks Massimiliano Lupo Pasini, Chady Ghnatios, Nicola Demo and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Park, C., Saha, S., Guo, J. et al. Unifying machine learning and interpolation theory via interpolating neural networks. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01496-4'>Large-scale visualization of α-synuclein oligomers in Parkinson's disease brain tissue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 11:29:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Parkinson's disease (PD) is a neurodegenerative condition characterized by the presence of intraneuronal aggregates containing fibrillar ɑ-synuclein known as Lewy bodies. These large end-stage species are formed by smaller soluble protein nanoscale assemblies, often termed oligomers, which are proposed as early drivers of pathogenesis. Until now, this hypothesis has remained controversial, at least in part because it has not been possible to directly visualize nanoscale assemblies in human brain tissue. Here we present Advanced Sensing of Aggregates—Parkinson's Disease, an imaging method to generate large-scale α-synuclein aggregate maps in post-mortem human brain tissue. We combined autofluorescence suppression with single-molecule fluorescence microscopy, which together enable the detection of nanoscale α-synuclein aggregates. To demonstrate the use of this platform, we analysed ~1.2 million nanoscale aggregates from the anterior cingulate cortex in human post-mortem brain samples from patients with PD and healthy controls. Our data reveal a disease-specific shift in a subpopulation of nanoscale assemblies that represent an early feature of the proteinopathy that underlies PD. We anticipate that quantitative information about this distribution provided by Advanced Sensing of Aggregates—Parkinson's Disease will enable mechanistic studies to reveal the pathological processes caused by α-synuclein aggregation. Parkinson's disease (PD) is a progressive neurodegenerative disorder that initially causes the loss of dopaminergic neurons in the substantia nigra, resulting in a movement disorder consisting of tremors, bradykinesia and rigidity1. The disease spreads over several years, affecting multiple brain regions and resulting in dementia, neuropsychiatric, autonomic and sleep disturbances2. Pathologically, PD is characterized by neuronal loss accompanied by the accumulation of microscale α-synuclein aggregates called Lewy bodies and Lewy neurites. The morphologies of these structures are typically either neuritic (~5–10 µm in length) or round (~5–20 µm diameter)3, and have been observed in varying densities in different brain regions depending on the disease stage4. Such structures form the basis of PD diagnostic staging criteria4,5. Further evidence implicating α-synuclein in PD arises from the observation that mutations or gene rearrangements in SNCA6,7,8,9,10,11,12,13, the gene encoding the α-synuclein protein, cause early-onset autosomal dominant PD and variants in the SNCA gene increase the risk of sporadic PD. Protein aggregation occurs through the self-assembly of monomeric α-synuclein into small protein assemblies, which undergo growth and structural conversion to soluble intermediate species, gradually acquiring cross β-sheet structure14,15,16. The smaller intermediary structures, including small fibrillar species and amorphous oligomers17, bridge monomers and the much larger fibrillar structures found in Lewy bodies18, are expected to contain tens to hundreds of monomeric protein units19. Oligomers can have a variety of post-translational modifications and conformations and contain α-helices and/or β-pleated sheets20,21,22. In cell culture and animal models, it has been shown that oligomers cause neurotoxicity and neuronal death consistent with PD23,24,25,26,27,28,29,30. Oligomers have mainly been studied using recombinant protein, but these aggregates are not identical to the oligomeric assemblies found in human tissue, and different preparation protocols for recombinant oligomers can lead to a variety of characteristics18,31,32,33, motivating studies on native proteins; proteins that are in their natural structure and functional confirmation. Detecting endogenous, small aggregates in a post-mortem human brain has long remained elusive, primarily owing to a lack of sensitivity. Proximity-ligation assays (PLA) have verified the presence of small α-synuclein aggregates by signal amplification16,34. However, direct visualization of nanoscale assemblies in brain tissue has, so far, not been possible, hindering our understanding of how these species are distributed spatially and by size. Here, we present an optical detection and analysis platform, Advanced Sensing of Aggregates–Parkinson's Disease (ASA–PD), that can be used to quantify aggregate density, distribution and size directly in fluorescently labelled post-mortem human brain tissue. We applied ASA–PD to characterize α-synuclein assemblies in large areas of post-mortem tissue sections from patients with PD and matched healthy controls (HCs). By enhancing the sensitivity of traditional immunofluorescence techniques, combined with our analytical approach, we have been able to detect and characterize over 1.2 million α-synuclein aggregates. The entire dataset, metadata and analysis toolset have been made available online (Data Availability35,36). In addition to the microscale aggregates described by classical Lewy pathology37, our data show that assemblies are present in both PD and HC samples. Notably, PD samples contained a shifted subpopulation of bright nanoscale assemblies largely absent from the HCs. The presence of this ‘disease-specific' shift was detected in PD cases from different brain banks, disease stages, immunofluorescent labels and antigen-retrieval methods. The presence of this subpopulation was confirmed using a range of orthogonal analytical approaches. Our findings are consistent with the hypothesis that misfolded α-synuclein readily form a continuum of larger nanoscale aggregates that eventually give rise to the microscale structures traditionally associated with the disease. We visualized and quantified the distributions of measured species in brain tissue, characterized their biochemical properties and determined their distributions in relation to several specific cell types in the human brain. An overview of the ASA–PD pipeline is shown in Fig. In brief, the aim is to capture spatial data over the entire scale of structure sizes most critical in PD, from individual cells to small aggregates (Fig. Detailed descriptions of the sample preparation steps are described in the Methods. First, 8-μm-thick brain tissue sections were mounted on glass slides, stained and then processed in the five stages illustrated in Fig. 1b: (1) background suppression, (2) enhanced imaging, (3) feature detection, (4) analytical computation and (5) spatial distribution analysis, where the first two steps contain the experimental portion of our workflow and steps 3–5 perform image-processing tasks and analysis. a, ASA-PD is an imaging and analysis method for detecting protein aggregates in tissue down to nanoscale aggregates. b, The five main steps for imaging and analysis. Background suppression and enhanced imaging improve the signal-to-noise ratio such that aggregates can be detected and quantified in the analysis pipeline, including the spatial distributions relative to cell-specific markers. Panels a and b created with BioRender.com. Observing protein aggregates is relatively routine in in vitro conditions38,39,40, but detecting small species in vivo poses a challenge owing to the poor signal-to-noise ratio in tissue. High background intensity, caused by tissue autofluorescence, acts as a noise floor that obscures the presence of dim objects such as oligomeric species. This noise effectively implements a brightness filter that leaves only large protein aggregates, with many attached fluorescent antibodies, as detectable species. To reduce the high autofluorescence of human brain tissue that inhibits sensitive imaging, we deployed Sudan Black B (Fig. 2a), a fat-soluble diazo dye and well-known autofluorescence quencher on untreated brain tissue sections41. Under optimized conditions, 10 min of incubation with 0.1% Sudan Black led to a 93% reduction in background autofluorescence for 561 nm laser excitation (26 W cm−2 illumination intensity), corresponding to a decrease in median detected photon counts from 4,400 ± 1,040 photons ± median absolute deviation (MAD) to 333 ± 47 photons (Fig. 2b) (the background reduction for other excitation colours (488, 561 nm), treatment times and concentrations are shown in Supplementary Fig. Next, we repeated this background suppression step on antibody-labelled samples and evaluated various antibodies against multiple forms of α-synuclein for specificity and detectability (Supplementary Figs. The background reduction by Sudan Black facilitated the reliable detection of small features in images with a vastly improved signal-to-noise ratio for some of the antibodies tested, as shown in Fig. a, Background suppression is achieved with the autofluorescence quencher Sudan Black (SB). b, Box plots showing the IQR and 5th–95th percentile bounds of autofluorescence (AF) intensity before (median of 4,400) and after treatment with 0.1% SB (median of 333). c, Before quenching, the fluorescence from Alexa Fluor 568 labelled small aggregates is masked by the background autofluorescence. After quenching, small aggregates can be easily visualized (both images excited at 561 nm, 26 W cm−2). d, A high NA objective collects a larger amount of light from the sample. e, The modelled signal-to-noise ratio for imaging punctate objects in post-quenched tissue background at 100× magnification across a range of NAs of objectives. Only at high NA (>1) large aggregates and oligomers become detectable. f, Images of p-syn stained PD tissue with 40× magnification, NA = 0.75 (top) and 100×, NA = 1.49 (bottom). Close-ups show that the same small aggregate is clearly visible at high magnification and high NA. g, Images of p-syn stained PD tissue with no background suppression and low NA (left), background suppression implemented and low NA (middle) and background suppression implemented and high NA (right). Several example puncta are shown in the closeups (oligomers) after background suppression is implemented and a high NA objective is used. To visualize the α-synuclein aggregates most associated with PD, we used an antibody targeting phosphorylated α-synuclein at serine 129 (hereafter called p-syn). This post-translational modification promotes inclusion formation and/or toxicity in human cells42, Drosophila43 and rodent models44,45. Further, p-syn forms the vast majority of all insoluble α-synuclein aggregates in the PD brain46. Given this link between p-syn and pathology in synucleinopathies, we tested a variety of antibodies (Supplementary Table 1), including two complementary antibodies targeting the pS129 epitope of α-synuclein raised in two species (rabbit, AB_2270761, and mouse, AB_2819037). The AB_2819037 antibody showed characteristic Lewy pathology in both DAB and immunofluorescence staining and were shown to be specific through substantial co-localization with a second antibody for total α-synuclein (AB_2832854) (Supplementary Fig. The final p-syn antibody selection, AB_2819037, was selected because of (1) the degree of coincidence of the p-syn antibody compared with total α-synuclein; (2) co-localization with other disease-related proteins, such as ubiquitin and p62; and (3) the demonstration of antibody specificity for human α-synuclein based on mouse tissue with the overexpression or knockout of human α-synuclein47. Furthermore, we confirmed (via electron microscopy and fluorescence imaging) that purified p-syn can aggregate in vitro, and form β-sheet rich 10-nm assemblies48. These recombinant p-syn protein aggregates can be detected by the same AB_2819037 antibody (Supplementary Fig. One way to improve the signal-to-noise ratio beyond reducing the overall background intensity is by improving the light-collection efficiency of the imaging system. In most microscopes, the least efficient step in light collection occurs at the objective lens of the microscope and is encoded in the numerical aperture (NA). Using a high NA objective lens has two main impacts: first, the NA scales with the collection angle of collected light49 (Fig. 2d and Supplementary equation (3)), and thus, more photons from the sample are collected at high NA. For imaging in tissue, we deployed a 1.49 NA oil-immersion, 100× microscope objective lens often used in single-molecule fluorescence applications51. The result is an overall increase in the signal-to-noise ratio for all objects, which is particularly important for the nanoscale assemblies that fall below the detectability range for lower NA objectives (Fig. 2e), such as the air objectives most often used in slide scanners for clinical applications52. Figure 2f compares a 0.75 NA 40× air objective lens (top) with the 1.49 NA 100× oil objective lens used in this study (bottom) for the same tissue sample stained for phosphorylated α-synuclein and quenched with 0.1% Sudan Black. The effect of background suppression and increased light collection using a high NA are shown in Fig. In these images, a wide variety of object sizes become visible; which we divide into two classes on the basis of their apparent size relative to the diffraction limit. Specifically, we define ‘large' as greater than the optical diffraction limit of visible light, spanning ~200 nm to tens of microns, and ‘nanoscale' as objects below the optical diffraction limit (<200 nm). Three examples of the nanoscale objects that become visible via ASA–PD are highlighted in Fig. We refer to the latter objects as protein assemblies. Applying the ASA–PD protocol within tissue revealed hundreds of detectable fluorescent puncta per field of view (FOV) (55 × 55 µm2) in both PD and HC samples (Supplementary Fig. To perform statistically robust comparisons between samples, we developed a computationally efficient method for detecting and quantifying fluorescent species. This open-source analysis pipeline53 facilitates the rapid processing of large image libraries, facilitating transparent, shareable and verifiable results. A schematic illustrating the analysis method and its validation is shown in Fig. In brief, the analysis pipeline identifies features in an image, classifies them as either large aggregates or protein assemblies and quantifies details such as brightness, size and position53. A detailed description of the analysis is provided in Supplementary Information Note 3.1 and Supplementary Figs. Figure 3a shows a typical PD image containing nano and microscale features. Microscale aggregates, such as Lewy bodies and Lewy neurites, are extremely bright in the dataset. These objects can be segmented with a simple intensity threshold after a background subtraction step (large-object pipeline in Fig. As large objects sometimes extend over multiple z-slices, the mask in each plane is multiplied by a segmented maximum-intensity projection from the z-stack. Smaller aggregates appear as dim, diffraction-limited puncta, and it is crucial to account for local background heterogeneity for detection (small-aggregate pipeline in Fig. To do so, we applied a bandpass filter to each image which selects features on the scale of the diffraction limit54 (Supplementary Figs. Next, a threshold was used to create a mask containing only small objects. Objects with a footprint larger than the diffraction limit49 were reclassified as ‘large' for subsequent analysis. Finally, the large and small aggregate masks were compared, and overlapping objects were removed from the nanoscale object dataset. Figure 3c shows an overlay of the detected objects on the original image, and a gallery of diffraction-limited puncta is shown in Fig. a, A typical sample image containing features of various sizes and intensities, that is, Lewy neurites, micron-scale aggregates and sub-diffraction-limit oligomers. b, The aggregate detection pipeline for measuring large aggregates (top) and subdiffraction-sized features (bottom). The large object pipeline combines the z-projected intensity data with background-subtracted and threshold individual slices to generate a binary mask. Small aggregates are identified using a Ricker wavelet filter that acts as a bandpass, emphasizing small spots, which are then measured with a threshold and sorted by the number of pixels above the background. Features larger than the diffraction limit are reclassified as ‘large' and features overlapping between the two masks are removed from the small aggregate pool. c, The large (green) and oligomer (blue) masks shown over the original image. d, The representative oligomers detected from c. e–h, The quantification of the pipeline performance using simulated images of diffraction-limited spots on a noisy background at various signal-to-noise ratios. The grey shaded region represents the lower quartile determined from experimental conditions, while the green and pink shaded area represents the mean ± s.d. The intensity and average background values for all detected peaks in simulated images were estimated by quantifying the pixel values around the detected peaks (pink curve) and by fitting a symmetric two-dimensional-Gaussian function with nonlinear least squares fitting (green). The presented values were obtained by averaging the mean and s.d. To evaluate the performance of the pipeline for detecting and characterizing nanoscale assemblies, we simulated images of puncta on noisy backgrounds at various signal-to-noise levels based on empirically determined parameters (Supplementary Note 3.2, Simulations). In the signal-to-noise range of our data, approximately ~4–8, the algorithm's sensitivity was >82%, with a precision of >94% (Fig. At the same time, the relative error for estimating the local background per puncta outperformed nonlinear least-squares Gaussian fitting in this low signal-to-noise regime where Gaussian fitting performed poorly on aberrantly detected pixels, that is, false positives (Fig. To characterize the distributions of α-synuclein in brain tissue, we selected three PD brains (Braak stage 6) and three HC brains for imaging (Supplementary Table 2). Tissue sections from the anterior cingulate cortical gyrus were put through the ASA–PD process. At this point, three principal regions within the grey matter were selected for investigation. At each of these regions, nine FOVs were captured in a 3 × 3 grid with a lateral separation of 150 μm to avoid any spatial overlap (each image covers 55 × 55 µm2). In total, 17 axial planes were recorded using a 500 nm step size (Fig. This process generated 13,770 high-resolution images (>41.6 mm2) that were manually validated to ensure the sample was in focus and the tissue contained no notable tears or defects. After this verification step, 12,028 images remained, 87.5% of the original dataset (5,954 PD and 6,074 HC images). These images were analysed as described in the previous section to map large aggregates (Fig. 4c,d) and nanoscale assemblies (Fig. Negative control samples, lacking primary antibodies, were also tested using PD tissue to quantify the degree of false positives caused by residual autofluorescence and unbound secondary antibodies (Supplementary Fig. a, The imaging of grey matter was performed in three areas, each area being a 3 × 3 grid of z stacks (17 slices) spaced 150 µm apart. b, The number of HCs and patients with PD (n = 3), number of tissue sections (n = 15) and number of images taken NHC = 6,074 and NPD = 5,954. c, Examples of analysed FOVs showing only the detected large aggregates. d, The number of large aggregates detected per patient over 1,800 FOVs (5.4 mm2). for large aggregates was 3,866 ± 408 in HCs and 26,314 ± 7712 in PD, the means were compared by a two-tailed two-sample t-test, with P = 0.0147. e,f, Example FOVs of detected α-synuclein oligomers in HC and PD (Braak stage 6), respectively. g, The total number of α-synuclein aggregates in HC and PD tissues. The left panel shows oligomers (<0.04 μm2). The right panel shows large aggregates (>0.04 μm2). The typical Lewy neurites sizes (~5–30 μm2) and Lewy bodies (~30–300 μm2) are shown for reference. h, Violin plot of brightness of detected oligomers truncated at 1.5× IQR. The bright subpopulation of oligomers is shown in red for PD. i, The total number of detected oligomers per patient above this brightness threshold, 10,280 photons. Error bars are variation in boundary rejection percentage per patient, propagated. From the ~12,000 images recorded across 30 tissue sections, we obtained a dataset containing more than 125,000 large aggregates and 1,260,000 nanoscale assemblies35,36. From the ~400 FOVs (~1.2 mm2), from each patient sample, the average number of large aggregates detected was ~tenfold higher in patients with PD than in the HC, with 26,314 ± 7712 in PD and 3,866 ± 408 in HC, respectively (Fig. These aggregates were distributed over a broad range of sizes from 0.04 to 100 µm2 in PD and 0.04 to 1 µm2 in HC (Fig. 4g), where the aggregate sizes associated with Lewy pathology essentially exclusively found in PD samples, consistent with the original tissue classifications (Supplementary Table 2). The total number of detected nanoscale objects in PD and HCs were much more similar (Fig. 4g), with 682,826 and 582,026 objects, respectively (with densities 0.082 objects per μm2 for PD and 0.067 objects per μm2 for HC). While image resolution remains fundamentally diffraction-limited, the high sensitivity of ASA–PD to dim puncta, coupled with their relative sparsity, allows the detection of ~10 nm objects—far below the diffraction limit. At this scale, the resolution obscures aggregate sizes; however, for larger aggregates, where the size and brightness can be measured, the two were strongly linearly proportional, R2 > 0.99 (Supplementary Fig. We, therefore, characterized the distribution of nanoscale-object intensities as a proxy for the approximate size of these assemblies (Fig. The brightness distributions for all measured objects are shown in Supplementary Fig. 13b together with the estimated number of bound secondary antibodies, assuming each antibody contributes ~700 photons in our imaging conditions (Supplementary Fig. Relative to the HC, the median was larger for PD samples, 3,700 photons (MAD of 1,690) and 2,750 photons (MAD of 1,060), and the distribution of brightnesses in PD samples was also broader, characterized by its interquartile range (IQR) IQRPD = 4,280 photons compared with IQRHC = 2,690 for HCs. To determine if the distribution tail was reproducibly different between PD and HC samples, we defined a brightness threshold using the HC measurements (Fig. The number of nanoscale objects above this threshold (10,280 photons, equating to ~15 bound secondary antibodies) is shown in Fig. This data represents ~10% of all measured PD assemblies but only 0.26% of those in HC (totalling 68,569 in PD and 1,518 in HC). The existence of this bright, disease-specific shift in the nanoscale population was highly robust by ASA–PD and was observed consistently when testing different α-synuclein antibodies, two different brain banks (Queen Square Brain Bank for Neurological Disorders (QSBB) and Multiple Sclerosis and Parkinson's Brain Bank (Imperial), 12 individuals (6 PD and 6 HC), and using different antigen retrieval methods (formic acid and heat mediated epitope retrieval) (Supplementary Note 3.3, Supplementary Figs. Our approach enables the direct visualization of nanoscale aggregates that are typically challenging to detect. To further investigate the nanoscale assemblies revealed by our approach and determine if the fraction of bright nanoscale aggregates found in disease PD tissue could be detected with orthogonal methods, we performed PLA, enzyme-linked immunosorbent assays (ELISA), size exclusion chromatography (SEC) and seed amplification assays (SAAs) in brain tissue samples. PLA can detect protein–protein interactions using antibody-linked DNA probes34. When these probes are adjacent, they produce an amplified signal that can be visualized using fluorescence microscopy (Supplementary Fig. This approach can be used to detect nanoscale assemblies of α-synuclein by amplifying the signal from α-synuclein 211 antibodies that are in close proximity. PLA was performed on 13 brain sections from the anterior cingulate gyrus, comprising six late Braak stage 5–6 PD and seven HC samples (Supplementary Table 3), and imaged (20× magnification, NA 0.75). Four images of the cingulate cortex were taken per sample and fluorescent puncta were quantified based on their intensity and a minimum size threshold of 0.9 µm. Representative images and the puncta density is shown in Supplementary Fig. The average number of puncta revealed an enriched population of aggregated α-synuclein in PD samples compared with HC samples, consistent with the ASA-PD data. Next, brain lysates were fractionated using SEC to separate soluble α-synuclein species by apparent molecular weight from the Braak stage five or six PD, and HC donors (three PD and three HC; Supplementary Fig. This was followed by ELISA on the fractions to quantify the absolute amount of αSyn per fraction. Fractions were then subjected to the SAA, which allows the amplification and detection of small amounts of aggregates present in a sample. Recombinant human monomeric α-synuclein was used for the amplification of templated aggregation from pre-existing aggregates, and the kinetics of the amplification reaction was monitored by Thioflavin T (ThT) binding and increase in its fluorescence intensity. The SEC-ELISA analysis revealed that total α-synuclein concentrations differed across high and low molecular weight fractions by two orders of magnitude but did not reveal statistical differences in the total α-synuclein in PD compared with controls, suggesting a similar amount of total protein (Supplementary Fig. The SAA revealed that high molecular weight fractions (200 kDa to 5 MDa) from PD brains had significantly shorter lag times, indicating the presence of seed competent αSyn aggregates, compared with samples derived from healthy brain tissue. Physiological (low molecular weight) fractions, in contrast, showed no statistical differences in lag times between PD and HC samples (Supplementary Fig. Comparisons between high molecular weight and low molecular weight fractions could not be made owing to altered α-synuclein concentrations. Therefore, there is a relatively small amount of aggregated α-synuclein (according to ELISA approx. 0.5% of total) both in physiological as well as disease tissue. In PD tissue, however, there is an apparent conversion of non-seed competent, or physiological, α-synuclein aggregates into seed competent aggregates. Finally, we validated the presence of Proteinase K-resistant fluorescent puncta to assess whether the observed aggregates exhibited distinct chemicophysical properties (Supplementary Fig. Notably, some bright puncta persisted after treatment, indicating a measurable degree of resistance to Proteinase K. Collectively, the ASA–PD and amplification assay data confirm the presence of a population of small, soluble protein assemblies in the brain, which changes in PD samples, namely a subpopulation of bright assemblies has distinctive chemicophysical properties, such as their size and seed competence, and Proteinase K resistance. In addition to measuring object densities and size distributions, ASA–PD can be used to analyse spatial distributions (Fig. To determine if the heterogeneity reported for larger α-synuclein aggregates4,55 extends to nanoscale species in PD samples, we performed a spatial-clustering test, which compares the likelihood of encountering assemblies as a function of the distance from it53 (Fig. Over 400 FoVs, 682,826 nanoscale objects were detected and characterized. On average, these species were found to cluster relative to a complete spatial random distribution. 5f, blue); however, the disease-specific populations exhibited a substantially higher degree of clustering (68,569 aggregates; Fig. a, An illustration showing example spot patterns with random, clustered and dispersed underlying spatial distributions. b, The equation used here for likelihood calculation. c, An example likelihood plot showing likelihood as a function of radius for random, clustered and dispersed spatial distributions. e, The same analysed FOV with detected oligomers in blue and red, with red serving to highlight the disease-specific oligomers. f, A plot showing the likelihood of oligomer–oligomer distances, with 95% confidence interval presented as the shaded region, shows that all oligomers tend to spatially cluster, and that disease-specific oligomers have a higher clustering tendency. g–r, Example FOVs stained with antibodies for neurofilaments, P2RY12, GFAP and Olig2 (g, j, m and p), the same images with labelled cells shown in green and disease-specific oligomers highlighted in red (h, k, n and p) and a plot showing the likelihood of oligomer–cell distances, with the 95% confidence interval presented as the shaded region, which shows that disease-specific oligomers tend to cluster in or around neurons, microglia and astrocytes, while being dispersed from oligodendrocyte nuclei (i, l, o and r). ASA–PD can also be used to interrogate the distance of objects to cell-specific markers in co-stained samples. To do so, we co-stained samples with α-synuclein and various cell markers, minimizing photobleaching and optimizing the signal of the nanoscale species. For some cell types, namely neurofilament (neurons), P2RY12 (microglia) and GFAP (astrocytes), the disease-specific species showed an increased probability of being localized in proximity to neurofilament (neurons), P2RY12 (microglia) and GFAP (astrocytes). By contrast, disease specific species have a low probability of being localized to the Olig2 marker (oligodendrocyte nuclei). This approach can thus enable quantitative statistical analysis of differences of disease-specific species density over large data sets. We have demonstrated the direct detection of single α-synuclein assemblies in human post-mortem brain tissue and performed quantitative analysis of more than 1.2 million objects across 30 tissue sections (682,826 nanoscale assemblies detected across 18 PD tissue sections, compared with 582,026 nanoscale assemblies detected across 18 HC sections). The acquisition of this large-scale dataset was made possible by the imaging component of ASA–PD, which is a combination of background suppression and high-NA collection of light that improves the signal-to-noise sufficiently to visualize the dim signal from individual nanoscale aggregates. The analysis pipeline that forms the detection step of ASA–PD allowed the sensitive and precise detection of these dim signals, as well as the determination of cell-specificity from co-staining and cell segmentation. The detection of small protein assemblies in post-mortem tissue is sensitive to many experimental fluctuations, and therefore ASA–PD was tested in a range of conditions, including the source of the brains (from two independent brain banks, Supplementary Table 3) and the traditional antigen retrieval methods (formic acid and heat mediated epitope retrieval). We found that the most consistent detection of oligomers was obtained using formalin-fixed paraffin-embedded (FFPE) Braak stage 6 tissue sections from a single brain bank, without any formic acid antigen retrieval; however, broadly similar trends were observed in all conditions tested (Supplementary Note 3.3 and Supplementary Fig. In agreement with the classical studies of neuropathology in PD post-mortem brains, we observed a six-fold increase in the number of aggregates above the diffraction limit in PD with respect to HC samples, and a 21-fold increase for aggregates >5 µm2, that is, aggregate sizes in line with Lewy pathology4. Lewy pathology, consisting of Lewy bodies and Lewy neurites, are microscale aggregates that are always found in sporadic PD cases (and have been used to define the disease4,56) and are sometimes found in HC tissue, where it has been described as incidental Lewy Body Disease57. Such studies have described the presence or absence of large Lewy aggregates, reporting an estimated level of abundance as a marker of severity of Lewy pathology, which lacks detailed quantitation. The technical advances implemented in ASA–PD have allowed us to capture the broad size range of protein aggregates containing phosphorylated α-synuclein and the quantification of their sizes and frequency in human PD tissue. With this large-scale dataset, we established that there is a continuum of aggregate sizes in the human brain, ranging from large (microscale) to very small (nanoscale). Using ASA–PD, each stage of the protein aggregation pathway present in disease brain can now be measured. Importantly, there are notably fewer large aggregates compared with small aggregates, as we measure a 16-fold increase in small aggregates below the optical diffraction limit compared with larger ones in disease. Therefore, small aggregates are by far the more abundant aggregate species, underscoring the need to determine the nature of their link with PD, a long-standing unanswered question in the field. Small aggregates have been previously shown to be increased in brains with Lewy pathology compared with controls58,59,60, and elevated levels of oligomers have been detected in the cerebrospinal fluid of patients with PD compared with controls, increasing with disease severity61,62,63,64,65. Furthermore, a PLA-based approach revealed a widespread distribution of α-synuclein oligomers in disease brain tissue, in contrast to canonical microscale Lewy-related pathology66. ASA–PD revealed an abundance of nanoscale assemblies in both control and PD tissue (equating to hundreds of points per cell). Their presence in control and disease suggests that small α-synuclein aggregates form under physiological conditions, where their formation and clearance are kept in balance by the protein homoeostasis system. For instance, the existence of α-synuclein as a tetramer or higher-order conformations has been previously reported in healthy-control samples using biochemical approaches67. Furthermore, serine 129 phosphorylation (pS129) may arise under homoeostatic conditions in response to synapse activity, being a reversible event that may regulate neuronal activity68. Activity-induced pS129 leads to conformational changes that facilitate interactions with new binding partners at the synapse, enabling α-synuclein to attenuate neurotransmission throughout regulating neurotransmitter release69. The abundance of small assemblies of phosphorylated α-synuclein in both healthy and PD brains would be in keeping with these proposed physiological roles for pS129 α-synuclein. Given the abundance of small assemblies in ASA–PD, and the known physiological role of α-synuclein, it is possible that ASA–PD is detecting α-synuclein assemblies at synapses, highlighting the potential for this approach to localize different subpopulations of assemblies in future. The large throughput enabled by ASA–PD makes it possible to characterize the subpopulation of disease-specific species, which represent just 9.7% of the total nanoscale assemblies. The existence of a disease specific subpopulation in late-stage PD brain was supported by amplification-based methods that further verified their presence. Furthermore, characteristics of this disease-specific soluble population, in particular, their ability to exhibit prion-like activity and therefore be seed competent was demonstrated for the higher molecular weight disease specific species. Our results suggest that, in the progression of PD, a fraction of the physiological assemblies detectable in HC undergo a transition to the disease-specific species detected in PD, consistent with prior in vitro studies that identified ‘type A' and ‘type B' oligomers, respectively17,19. Once this transition has occurred, these objects, which were detected at 21-fold higher frequency than the large aggregates, may then ultimately aggregate further and become the fibrillar structures that are found in Lewy bodies and Lewy neurites. However, as ASA–PD or SAA cannot resolve the underlying structural differences between pre-fibrillar oligomers and short fibril fragments, further investigation would be needed to confirm this theory. Nevertheless this hypothesis is consistent with previous findings of aggregation kinetics in vitro38,70,71,72, where a small proportion of the total α-synuclein population, under pro-aggregation conditions, will give rise to disease-specific, more toxic oligomers. In in vitro systems and human neurons, the earliest oligomers formed are Proteinase K sensitive, relatively inert and non-toxic; during aggregation, these oligomers undergo a transition to larger, Proteinase K resistant forms15,17,37,73, consistent with our observation of resistant aggregates in tissue. This transition is associated with structural conversion from relatively disordered assemblies to highly ordered and toxic species with the acquisition of β-sheet structure. It is the acquisition of this β-sheet structure that is associated with the propensity for these in vitro formed pathological oligomers to disrupt membranes and induce toxicity in human neuronal systems74. It is not yet clear how a structural conversion to the disease-specific species may occur in the brain. However, ASA–PD's ability to detect a broad spectrum of aggregates including the disease-specific subpopulation opens the door to further investigate their structural properties, specifically changes in their order, sensitivity to degradation and β-sheet structure75. While microscale α-synuclein inclusions have been found predominantly in neurons in PD3,55, less is known about smaller species. Using a spatial analysis that captures both the distances between assemblies and assemblies to cell boundaries, our data suggest that the disease-specific species cluster inside or around cell types, including neurons, astrocytes and microglia. As large (Lewy) aggregates are principally located within neurons, it is possible that the observed protein assemblies transition from their physiological state into the pathological state (either by size or by structural conversion), where they may precede the formation of the later stage Lewy bodies. How the brighter, disease-specific species ultimately form Lewy bodies containing not only fibrillar α-synuclein but also neuronal lipid membranes and organelles76,77, is not known in vivo. Detailed cellular and subcellular imaging of these assemblies as they transition from the bright disease-specific species to fibrillar structures are needed to provide information on the trajectory of Lewy body formation within neurons in the human brain, their heterogeneity and the components within them. While this study has involved the analysis of over 1 million aggregates, several limitations still pose further challenges: First, the determination of aggregate sizes: we have not directly measured the physical dimensions of the assemblies but inferred the distribution of sizes on the basis of their relative integrated brightnesses. Above the diffraction limit (around ~200 nm), we observed a strong linear relationship between the integrated brightness of aggregates and their measured area. We hypothesize that this pattern persists below the diffraction limit as well. The linear correlation with area suggests that microscale aggregates have numerous exposed epitopes on their surfaces, making it unlikely for antibodies to penetrate deeply inside the aggregates. Second, the detection of a distinct aggregate class: we validated the use of multiple antibodies and primarily used an antibody to the serine 129 C-terminal phosphorylated form of α-synuclein. Therefore, we cannot be certain that the shifted distributions and densities observed in PD tissue are conserved for all α-synuclein assemblies. Indeed, antibodies to multiple epitopes, including those targeting the N terminus, have been shown to provide more detailed information on other aggregate subtypes and locations in the brain78,79. Third, the detailed characterization of the disease-specific subpopulation: this will require super-resolution methods or electron microscopy to visualize nanoscale structures. To use super-resolution methods, further signal-to-noise ratio improvement is needed. This could include background reduction (for example, optical-clearing techniques80) combined with better sectioning (for example, confocal microscopy or thinner tissue slicing). Fourth, the extended coverage of brain regions: higher throughput methods are also needed to increase statistical confidence in biological findings at the macroscopic scale of the brain. While high-throughput methods with single-molecule sensitivity are possible, there are currently no commercially available solutions, necessitating the use of custom instruments. We acknowledge the limited sample numbers in this study but estimate from these data that an approximate two-order-of-magnitude increase in throughput of the ASA–PD method will enable the exploration of different brain regions, encompass more cases and facilitate large-scale automation, ultimately establishing a foundational aggregation map of the PD brain. We designed ASA–PD, a platform technology for the large-scale imaging of protein aggregates in brain tissue. We then demonstrated its application to generate the largest dataset so far describing the distribution of α-synuclein aggregates, their prevalence, and their spatial location in the PD brain. Without this quantitative information, it has been challenging to establish the nature of the link between α-synuclein aggregation and PD. Although substantial evidence from model systems has implicated small α-synuclein assemblies in pathological processes, it has remained unclear whether such processes are truly relevant in the disease. With ASA–PD, we identified a subpopulation of small assemblies present in disease tissue. The ASA–PD platform can thus be used to design mechanistic studies to understand how these species are created. These studies will build on the ability of ASA–PD to address the regional and cellular microenvironments that promote the development of disease-specific species, as well as the temporal evolution of the end-stage Lewy body pathology from these objects. Furthermore, integrating the ASA–PD dataset with other single-cell and spatial RNA and protein technologies will allow the identification of the key pathways and mechanisms associated with the cellular environments that promote protein aggregation and Lewy body formation. We also note that the ASA–PD method is widely applicable to other neurodegenerative diseases, where the role of protein aggregation remains largely unresolved. The detailed protocols and software packages for single-molecule slides for fluorescence microscopy81, free-floating mouse brain immunohistochemistry82, preparing tissue staining and imaging83 and feature-detection software84 are available online. Post-mortem brain tissue was obtained from Queen Square Brain Bank for Neurological Disorders (QSBB), University College London, and Multiple Sclerosis and Parkinson's Brain Bank, Imperial College London (Imperial). Braak stage 3/4 PD cases (n = 3) were obtained from Imperial, Braak stage 6 PD cases (n = 4; 1 for technical controls, 3 for main study) from QSBB and three HCs from each brain bank to control for brain bank processing effects (n = 6; 3 from Imperial, 3 from QSBB) as highlighted in Supplementary Table 2. Standard diagnostic criteria were used to determine the pathological diagnosis. Case demographics for each case are described in Supplementary Table 3. Then, 8-µm-thick FFPE tissue sections from the cingulate cortex were cut from the cases summarized in Supplementary Table 2 and described in detail in Supplementary Table 3. These sections were loaded onto Superfrost Plus microscope slides. FFPE sections were baked at 37 °C for 24 h and then 60 °C overnight. They were deparaffinized in xylene and rehydrated using graded alcohols. Endogenous peroxidase activity was blocked in 0.3% H2O2 in methanol for 10 min to suppress autofluorescence85,86. All sections underwent heat-mediated epitope retrieval for 10 min in citrate buffer (pH 6.0). Half of the sections were additionally incubated in formic acid for 10 min before heat-mediated epitope retrieval to test the optimal antigen retrieval conditions as these conditions are used routinely for diagnostic work and have worked previously in our experiments87. Non-specific binding was blocked with 10% dried milk solution in phosphate buffered saline (PBS). Tissue sections were incubated with primary antibodies anti-α-synuclein (LB509, AB_2832854 1:100; phospho S129 rabbit polyclonal, AB_2270761, 1:200; phospho S129 mouse monoclonal, AB_2819037, 1:500), anti-P2RY12 (AB_2669027, 1:100), anti-neurofilament (RT-97, AB_2941917, 1:200), anti-Glial fibrillary acidic protein (GFAP) (5C10, AB_2747779, 1:1000) and anti-Olig2 (AB_570666 1:100) for 1 h at room temperature, washed three times for five minutes in PBS followed by the corresponding AlexaFluor (anti-mouse 488, AB_2534069/anti-mouse 568, AB_144696/anti-rabbit 488, AB_143165/anti-rabbit 568, AB_143157 all at 1:200) for 1 h at room temperature. Sections were kept in the dark from this point onwards. Sections were then washed three times for 5 min in PBS and incubated in Sudan Black (multiple concentrations and incubation times were tested, as described in ‘Background suppression' section). Sudan Black was removed with three washes of 30% ethanol before they were mounted with Vectashield PLUS (Vector Laboratories, H-1900), coverslipped (22x50 mm #1, VWR, 631-0137) and sealed with CoverGrip sealant (Biotium, 23005) for imaging. Sections were stored at 4 °C until imaging. Coverslips (24 × 50 mm, #1, VWR, 48404-453) were argon plasma cleaned (Ar plasma cleaner, PDC-002, Harrick Plasma) for 30 min before a trimmed gasket was placed on top (CultureWell™ Reusable Gasket, 6 mm diameter, Grace Bio-Labs, 103280). Poly-l-Lysine (PLL) (0.01% w/v PLL, Sigma-Aldrich, P4707) was placed in the wells for 30 min. The PLL was removed, the wells washed three times with PBS (pH 7.4, 1x Gibco, Thermo Fisher Scientific, 10010023), and the secondary antibody of choice was added (Alexa Fluor 568 goat anti-mouse—AB_144696 or Alexa Fluor 568 goat anti-rabbit—AB_143157) at a dilution of 1:10,000 in PBS from 2 mg ml−1 stock to a final concentration of 0.2 μg ml−1. The antibodies were left in the wells for 5–10 s for sufficient surface density before the wells were washed three times with PBS. PBS (30 μl) was left in the wells for imaging. Images were taken over two slides with 25 FOVs per slide. Images of human post-mortem brain and single-molecule antibodies were taken using a custom-built widefield fluorescence microscope that has been described previously88. Illumination of the sample was by a 488 nm laser (iBeam-SMART, Toptica) and a 561 nm laser (LaserBoxx, DPSS, Oxxius), both of which had the same excitation alignment. Both lasers were circularly polarized using quarter-wave plates, collimated and expanded to minimize field variation. The laser lines were aligned and focused on the back focal plane of the objective lens (100× Plan Apo TIRF, NA 1.49 oil-immersion, Nikon) to allow for the sample to be illuminated by a highly inclined and laminated optical sheet (HILO). Emitted fluorescence was collected by the objective lens before passing through a dichroic mirror (Di01-R405/488/561/635, Semrock). The collected fluorescence then passed through emission filters dependent on the excitation wavelength (FF01-520/44-25 + BLP01-488R for 488 nm excitation, LP02-568RS-25 + FF01-587/35-25 for 561 nm excitation, Semrock). The fluorescence was then expanded (1.5×) during projection onto an electron-multiplying charge-coupled device (EMCCD, Evolve 512 Delta, Photometrics). The EMCCD was operating in frame transfer mode with an electron multiplication gain of 250 analogue-to-digital units per photon. Z-stacks of images were taken through the samples in 0.5 µm steps with 17 steps per FoV covering a depth of 8 µm. To reduce bias in FoV selection, nine FoVs were recorded in a grid formation of 3 × 3 FoVs with 150 µm spacing, implemented with the ImageJ289 Micromanager plugin90. Three 3 × 3 grids were recorded at three random locations within the grey matter of each section, resulting in 27 z-stacks per section. Images were recorded with 1 s exposure time of the EMCCD. Images were recorded with a power density of 2.4 W cm−2 (488 nm excitation) and 25.9 W cm−2 (561 nm excitation). To convert the pixel values from counts in analogue-to-digital units to photons, a series of images were recorded at different illumination intensities, including one taken under no light to measure the camera (EMCCD, Photometrics, Evolve 512 Delta) offset91. Each intensity was captured in 500 frames, resulting in a total of 3,000 frames across the six different illumination levels. For every pixel, the mean and variance was calculated across the 500 frames, generating six different variance and mean values corresponding to the six illumination intensities. The camera gain per pixel, expressed in photoelectrons per count, was determined by calculating the slope between the six variance and mean values per pixel, subtracting the dark frame offset. Custom code was written to detect both cells and α-synuclein aggregates. Firstly, each z-stack was manually inspected to remove out-of-focus images. For the aggregate channel, two binary masks representing nanoscale and larger fluorescent objects images respectively were created for each image using the following four steps: (1) the positions of large objects were detected with a difference-of-Gaussian kernel (σ1 = 2 px, σ2 = 40 px) and Otsu's threshold, (2) cellular autofluorescence was removed using high-frequency filtering (σhigh-pass = 5 px), and diffraction-limited-sized features were enhanced through Ricker wavelet filtering (σRicker = 1.1 px); (3) bright spots within the image were identified using an intensity threshold on ther basis of the top 2.5th percentile of the image; and (4) spurious pixels were removed from the union of the two binary masks in step 1 and 3 via morphological opening operation with a disk shape (radius of 1 px). The resulting mask is then separated into nanoscale and non-nanoscale fluorescent objects, where the threshold was 19 px, determined by 40 nm subdiffraction limited fluorescent beads (40 nm, FluoSphere F10720). For the cell channel, cell segmentation was achieved using the function described in the first step of the aggregate-detection pipeline but using different kernel sizes (σ) dependent on cell type. For specific details of σ values used, see Supplementary Information Note 3.1. All imaging data were viewed in ImageJ and analysed and plotted using custom MATLAB code and Origin. This corresponds to the ~99.7 brightness percentile of the nanoscale assemblies identified in HC patients. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All unprocessed data35, processed data (binary masks, density and intensity information)36, methods and detailed protocols are available online. Analysis code is available online. Sveinbjornsdottir, S. The clinical symptoms of Parkinson's disease. Chaudhuri, K. R., Healy, D. G. & Schapira, A. H. Non-motor symptoms of Parkinson's disease: diagnosis and management. Spillantini, M. G., Crowther, R. A., Jakes, R., Hasegawa, M. & Goedert, M. α-Synuclein in filamentous inclusions of Lewy bodies from Parkinson's disease and dementia with Lewy bodies. Staging of brain pathology related to sporadic Parkinson's disease. Attems, J. et al. Neuropathological consensus criteria for the evaluation of Lewy pathology in post-mortem brains: a multi-centre study. A., Puschmann, A., Dickson, D. W. & Wszolek, Z. K. Autosomal dominant Parkinson's disease caused by SNCA duplications. B. et al. α-Synuclein locus triplication causes Parkinson's disease. Krüger, R. et al. AlaSOPro mutation in the gene encoding α-synuclein in Parkinson's disease. The new mutation, E46K, of α-synuclein causes parkinson and Lewy body dementia. Appel-Cresswell, S. et al. Alpha-synuclein p.H50Q, a novel pathogenic mutation for Parkinson's disease. Lesage, S. et al. G51D α-synuclein mutation causes a novel Parkinsonian–pyramidal syndrome. Polymeropoulos, M. H. et al. Mutation in the α-synuclein gene identified in families with Parkinson's disease. Guo, Y. et al. Genetic analysis and literature review of SNCA variants in Parkinson's disease. Bisi, N. et al. α-Synuclein: an all-inclusive trip around its structure, influencing factors and applied techniques. Choi, M. L. et al. Pathological structural conversion of α-synuclein at the mitochondria induces neuronal toxicity. Direct visualization of alpha-synuclein oligomers reveals previously undetected pathology in Parkinson's disease brain. Structural basis of membrane disruption and cellular toxicity by α-synuclein oligomers. In vivo demonstration that α-synuclein oligomers are toxic. Villar-Piqué, A., Lopes da Fonseca, T. & Outeiro, T. F. Structure, function and toxicity of alpha-synuclein: the Bermuda triangle in synucleinopathies. & Liu, R. The role of α-synuclein oligomers in Parkinson's disease. Stöckl, M. T., Zijlstra, N. & Subramaniam, V. α-Synuclein oligomers: an amyloid pore? Different species of α-synuclein oligomers induce calcium influx and seeding. Tetzlaff, J. E. et al. CHIP targets toxic α-synuclein oligomers for degradation. The lipid peroxidation products 4-oxo-2-nonenal and 4-hydroxy-2-nonenal promote the formation of α-synuclein oligomers with distinct biochemical, morphological, and functional properties. Rockenstein, E. et al. Accumulation of oligomer-prone α-synuclein exacerbates synaptic and neuronal degeneration in vivo. Luth, E. S., Stavrovskaya, I. G., Bartels, T., Kristal, B. S. & Selkoe, D. J. Soluble, prefibrillar α-synuclein oligomers promote complex I-dependent, Ca2+-induced mitochondrial dysfunction. Structural characterization of toxic oligomers that are kinetically trapped during α-synuclein fibril formation. Bengoa-Vergniory, N., Roberts, R. F., Wade-Martins, R. & Alegre-Abarrategui, J. Alpha-synuclein oligomers: a new hope. Hughes, C. D. et al. Picomolar concentrations of oligomeric alpha-synuclein sensitizes TLR4 to play an initiating role in Parkinson's disease pathogenesis. Direct detection of alpha synuclein oligomers in vivo. Outeiro, T. F. Alpha-synuclein antibody characterization: why semantics matters. Kumar, S. T. et al. How specific are the conformation-specific α-synuclein antibodies? Characterization and validation of 16 α-synuclein conformation-specific antibodies using well-characterized preparations of α-synuclein monomers, fibrils and oligomers with distinct structures and morphology. Wide distribution of alpha-synuclein oligomers in multiple system atrophy brain detected by proximity ligation. Direct visualisation of alpha-synuclein oligomers in human brain tissue—raw data. Aligning Science Across Parkinsons https://doi.org/10.5281/ZENODO.10610924 (2024). Andrews, R., Fu, B., Toomey, C. E., Breiter, J. C. & Lee, S. F. Direct visualisation of alpha-synuclein oligomers in human brain tissue—processed data. Aligning Science Across Parkinsons https://doi.org/10.5281/ZENODO.10610657 (2024). Goedert, M., Spillantini, M. G., Del Tredici, K. & Braak, H. 100 years of Lewy pathology. Lee, J.-E. et al. Mapping surface hydrophobicity of α-synuclein oligomers at the nanoscale. Whiten, D. R. et al. Nanoscopic characterisation of individual endogenous protein aggregates in human neuronal cells. Delayed emergence of subdiffraction-sized mutant huntingtin fibrils following inclusion body formation. Oliveira, V. C. et al. Sudan Black B treatment reduces autofluorescence and improves resolution of in situ hybridization specific fluorescent signals of brain sections. Luk, K. C. et al. Exogenous α-synuclein fibrils seed the formation of Lewy body-like intracellular inclusions in cultured cells. Chen, L. & Feany, M. B. α-Synuclein phosphorylation controls neurotoxicity and inclusion formation in a Drosophila model of Parkinson disease. Kahle, P. J. et al. Hyperphosphorylation and insolubility of α‐synuclein in transgenic mouse oligodendrocytes. The phosphorylation state of Ser-129 in human α-synuclein determines neurodegeneration in a rat model of Parkinson disease. Fujiwara, H. et al. α-Synuclein is phosphorylated in synucleinopathy lesions. Deficits in dopaminergic transmission precede neuron loss and dysfunction in a new Parkinson model. Needham, L.-M. et al. ThX—a next-generation probe for the early detection of amyloid aggregates. beiträge zur theorie des mikroskops und der mikroskopischen wahrnehmung. Single-Molecule Techniques: a Laboratory Manual (Cold Spring Harbor Laboratory Press, 2008). Whole slide imaging equivalency and efficiency study: experience at a large academic center. Fu, B. et al. RASP: optimal single puncta detection in complex cellular backgrounds. Hosken, J. W. J. Ricker wavelets in their various guises. Spillantini, M. G. et al. α-Synuclein in Lewy bodies. A critical reappraisal of current staging of Lewy-related pathology in human brain. Jellinger, K. A. α-Synuclein pathology in Parkinson's and Alzheimer's disease brain: incidence and topographic distribution—a pilot study. The formation of highly soluble oligomers of α-synuclein is regulated by fatty acids and enhanced in Parkinson's disease. Kramer, M. L. & Schulz-Schaeffer, W. J. Presynaptic α-synuclein aggregates, not lewy bodies, cause neurodegeneration in dementia with lewy bodies. Detection of elevated levels of soluble -synuclein oligomers in post-mortem brain extracts from patients with dementia with Lewy bodies. Decreased α-synuclein in cerebrospinal fluid of aged individuals and subjects with Parkinson's disease. Park, M. J., Cheon, S.-M., Bae, H.-R., Kim, S.-H. & Kim, J. W. Elevated levels of α-synuclein oligomer in the cerebrospinal fluid of drug-naïve patients with parkinson's disease. Aasly, J. O. et al. Elevated levels of cerebrospinal fluid α-synuclein oligomers in healthy asymptomatic LRRK2 mutation carriers. Parnetti, L. et al. Cerebrospinal fluid lysosomal enzymes and alpha‐synuclein in Parkinson's disease. Majbour, N. K. et al. Cerebrospinal α-synuclein oligomers reflect disease motor severity in DeNoPa longitudinal cohort. Sekiya, H. et al. Discrepancy between distribution of alpha-synuclein oligomers and Lewy-related pathology in Parkinson's disease. Dettmer, U. et al. Parkinson-causing α-synuclein missense mutations shift native tetramers to monomers as a mechanism for disease initiation. Dynamic physiological α-synuclein S129 phosphorylation is driven by neuronal activity. Parra-Rivas, L. A. et al. Serine-129 phosphorylation of α-synuclein is an activity-dependent trigger for physiologic protein-protein interactions and synaptic function. Horrocks, M. H. et al. Single-molecule imaging of individual amyloid protein aggregates in human biofluids. Bongiovanni, M. N. et al. Multi-dimensional super-resolution imaging enables surface hydrophobicity mapping. Shaib, A. H. et al. One-step nanoscale expansion microscopy reveals individual protein shapes. Angelova, P. R. et al. Alpha synuclein aggregation drives ferroptosis: an interplay of iron, calcium and lipid peroxidation. Toxic prefibrillar α-synuclein amyloid oligomers adopt a distinctive antiparallel β-sheet structure. A. et al. Optical structural analysis of individual α‐synuclein oligomers. Shahmoradian, S. H. et al. Lewy pathology in Parkinson's disease consists of crowded organelles and lipid membranes. The process of Lewy body formation, rather than simply α-synuclein fibrillization, is one of the major drivers of neurodegeneration. A. et al. Aggregate-prone brain regions in Parkinson's disease are rich in unique N-terminus α-synuclein conformers with high proteolysis susceptibility. Altay, M. F. et al. Development and validation of an expanded antibody toolset that captures alpha-synuclein pathological diversity in Lewy body diseases. Structural and molecular interrogation of intact biological systems. Andrews, R. Single-molecule antibody slides for fluorescence microscopy V1. Breiter, J. Free-floating mouse brain immunohistochemistry V1. Andrews, R. & Lachica, J. Single-molecule immunofluorescence tissue staining protocol for oligomer imaging V2. Direct visualisation of alpha-synuclein oligomers in human brain. Tang, X., Ibanez, F. & Tamborindeguy, C. Quenching autofluorescence in the alimentary canal tissues of Bactericera cockerelli (Hemiptera: Triozidae) for immunofluorescence labeling. Koga, R., Tsuchida, T. & Fukatsu, T. Quenching autofluorescence of insect tissues for in situ detection of endosymbionts. Toomey, C. E. et al. Mitochondrial dysfunction is a key pathological driver of early stage Parkinson's. Jenkins, E. et al. Antigen discrimination by T cells relies on size-constrained microvillar contact. Rueden, C. T. et al. ImageJ2: ImageJ for the next generation of scientific image data. Edelstein, A., Amodaj, N., Hoover, K., Vale, R. & Stuurman, N. Computer control of microscopes using µManager. Thompson, M. A., Lew, M. D., Badieirostami, M. & Moerner, W. E. Localizing and tracking single nanoscale emitters in three dimensions with high spatiotemporal resolution using a double-helix point spread function. The authors thank H. Greer and the Cambridge Electron Microscopy Staff for their assistance and P. Mrovecova for their Proximity Ligation Assay Biorender template. This research was funded in part by Aligning Science Across Parkinson's, ASAP-000509 (M.R., N.W., M.V., S.G. and S.F.L. Fox Foundation for Parkinson's Research; Astra Zeneca Postdoctoral Fellowship (R.A.); The Medical Research Council, MC_UU_00003/5 (C.L. ), which receives funding from the UK Medical Research Council; the US National Institute of Neurological Disorders and Stroke grants, R01-NS109209, RF1NS133979 and R01-NS078165 (T.B. ); the Chan Zuckerberg Collaborative Pairs Initiative Phase 2 (T.B. ); Project PID2021-128210OA-I00 (N.B.-V.) funded by MICIU/AEI/10.13039/501100011033 and by FEDER, EU; project RYC2021-034659-I (N.B.-V.) funded by MICIU/AEI /10.13039/501100011033 and by the European Union NextGenerationEU/PRTR; the Ikerbasque Basque Foundation for Science, EU COFUND H2020-MSCA-COFUND-2020-101034228-WOLFRAM2 (N.B.-V.); Achucarro Basque Center for Neuroscience (N.B.-V.); TransMedTech Institute (L.E.W. ), which receives funding from the Canada First Research Excellence Fund; Wellcome, 100172/Z/12/2 (S.G.); and MRC Senior Clinical Fellow, MR/T008199/1 (S.G.). Present address: Department of Chemistry, University of Zurich, Zurich, Switzerland Present address: MSD R&D Innovation Centre, London, UK These authors contributed equally: Rebecca Andrews, Bin Fu, Christina E. Toomey. Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK Rebecca Andrews, Bin Fu, Jonathan C. Breiter, Joseph S. Beckwith, Ru Tian, Emma E. Brock, Lisa-Maria Needham, Gregory J. Chant & Steven F. Lee Aligning Science Across Parkinson's Collaborative Research Network, Chevy Chase, MD, USA Rebecca Andrews, Bin Fu, Christina E. Toomey, Jonathan C. Breiter, Joanne Lachica, Joseph S. Beckwith, Ru Tian, Emma E. Brock, Gregory J. Chant, Camille Loiseau, Angèle Deconfin, Peter J. Magill, Zane Jaunmuktane, John Hardy, Tammaryn Lashley, Mina Ryten, Michele Vendruscolo, Nicholas W. Wood, Lucien E. Weiss, Sonia Gandhi & Steven F. Lee Oncology R&D, AstraZeneca, Cambridge, UK Department of Clinical and Movement Neurosciences, UCL Queen Square Institute of Neurology, London, UK Christina E. Toomey, Joanne Lachica, James Evans, Zane Jaunmuktane, Nicholas W. Wood & Sonia Gandhi The Francis Crick Institute, London, UK Christina E. Toomey, Joanne Lachica, Rebeka Popovic, James Evans, Yen Goh & Sonia Gandhi Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK Jonathan C. Breiter, Ru Tian & Michele Vendruscolo Medical Research Council Brain Network Dynamics Unit, Nuffield Department of Clinical Neurosciences, University of Oxford, Oxford, UK Angèle Deconfin, Kenza Baspin & Lucien E. Weiss Achucarro Basque Center for Neuroscience, Leioa, Spain Lenart Senicar, Marisa Edmonds & Tim Bartels Ikerbasque-Basque Foundation for Science, Bilbao, Spain Department of Neuroscience, University of the Basque Country, Leioa, Spain Division of Neuropathology, National Hospital for Neurology and Neurosurgery, University College London NHS Foundation Trust, London, UK Neuroscience BioPharmaceuticals R&D, AstraZeneca, Cambridge, UK Department of Neurodegenerative Diseases, UCL Queen Square Institute of Neurology, London, UK Great Ormond Street Institute of Child Health, University College London, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and S.G. conducted the sample preparation. carried out the material preparation. and N.B.-V. carried out data collection. carried out the data analysis. carried out the data interpretation. carried out the manuscript writing. All authors read and approved of the final manuscript. Correspondence to Lucien E. Weiss, Sonia Gandhi or Steven F. Lee. Ethical approval for the study was granted by the Local Research Ethics committee of the National Hospital for Neurology and Neurosurgery. is currently employed by MSD. All other authors declare no conflicts of interest. Nature Biomedical Engineering thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Large-scale visualization of α-synuclein oligomers in Parkinson's disease brain tissue. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03120-6'>Will your study change the world? This AI tool predicts the impact of your research</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 10:53:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. The benefits of doing science can feel abstract, but a new tool can trace how research projects had broad societal impacts on policies, medicines or products.Credit: Daniel Leal/AFP via Getty Scientific research offers many benefits to society, but how do you trace the impact of specific projects? It's easy to track which papers result from a grant, but much harder to follow how research has broader societal impacts on policies, medicines or products. “Those are much more exciting [impacts], and provide greater public good,” says Dashun Wang, director of the Center for Science of Science and Innovation at Northwestern University in Evanston, Illinois. Staša Milojević, who studies the science of science at Indiana University in Bloomington and was not involved in developing the tool, says that it could help to fill an important gap when it comes to translating studies of how science works into useful data. “However, their practical impact is often limited because they lack the tools that stakeholders can easily use to obtain useful insights.” “Even without its predictive aspect, having a tool that allows one to look up PIs [principal investigators] or grants from wide areas of science and evaluate them in terms of their various research metrics normalized for variations over fields and time is extremely useful.” Wang expects that many people will be interested in the tool. “There's an infinite amount of questions you could ask,” he says. The preprint, which is yet to be peer reviewed, includes case studies involving anonymous potential users. One project manager at a large US funding agency used it to examine the broad social impacts of the agency's grants, which he said he had never been able to see before. But he also identified a previously unknown gender disparity — namely that, in some fields, female PIs were under-represented — which he planned further investigate. Technology leaders should ‘pay back' society to support the common good NIH races to spend its 2025 grant money — but fewer projects win funding US autism research gets $50-million funding boost — amid row over Tylenol End GDP mania: how the world should really measure prosperity Robert Wood Johnson Medical School at Rutgers University invites applications The SiRIUS Institute of Medical Research at Tohoku University invites applications for 5 tenure-track Lecturer and Assistant Professor positions. The Director is initially appointed for a three-year term, renewable for a second term, and concurrently holds the title of Research Fellow. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/engineered-microbes-pull-critical-minerals-from-mining-waste/'>These Microbes Are Eating Mining Waste—And Pulling Out Valuable Metals</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At the northern edge of Chile's Atacama Desert sits a pile of rocks that's so big that you can see it from space—and it's teeming with invisible activity. Billions of microbes are hard at work dissolving compounds in this giant mound of crushed ore from Escondida, the biggest copper mine on the planet. “Microbes are the world's oldest miners,” says Liz Dennett, founder and CEO of the start-up Endolith Mining, based near Denver, Colo. “They've had billions of years to become incredibly good at eating rocks.” Scientists at Endolith and elsewhere are engineering microbes to get even better at this process, called biomining—to work faster, extract more copper and even pull out other kinds of minerals. “Think of it like a superhero training camp,” Dennett says. In May the company's engineered microbes demonstrated copper extraction superior to microbes found in nature; its first field deployments are scheduled for later this year. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Biomining, if it can be scaled up, could make it possible to decrease reliance on global supply chains, which are becoming ever more fragile. “If we can make biomining work, we can break the monopoly that states like China have on critical metals,” says Buz Barstow, a biological and environmental engineer at Cornell University. Barstow is leading a project called the Microbe-Mineral Atlas that catalogs microorganisms, their genes and how they interact with minerals. The project's goal is to build genetically engineered microorganisms that can effectively mine critical metals. As many countries transition to renewable energy, they will require fewer fossil fuels but more minerals such as lithium, cobalt, copper, nickel and zinc. These are not only needed for wind turbines, solar panels and batteries; they're also crucial for the laptops and cell phones we use every day. Forecasts show we'll need more copper in the next 30 years than has been mined throughout all of history. Much of the low-hanging fruit—that is, high-grade ore—has been picked, and mines have to work a lot harder than before to get the same quantity. Conventional mining techniques are resource-intensive, expensive and harmful to the environment. After using explosives and heavy machinery to extract ore from the earth, mining companies must isolate and purify the minerals in question. Smelting requires extremely high temperatures, reached by burning fossil fuels or using a lot of electricity. The sulfuric acid used for this process can cause harm before it even reaches mines. “There's a lot of secondary and tertiary effects on the environment.” “Microbes use at least six different mechanisms for biomining,” Barstow says. The most common is an oxidation-reduction reaction, or redox reaction, in which microbes break the chemical bonds in a sulfide ore by “eating” their sulfur and iron. The mineral-rich solution is collected in a pond after it is drained from the rock and is then exposed to solvents and electricity that attract the minerals like a magnet while leaving water, acid and impurities behind. “Replacing [most] sulfuric acid is a big economic benefit, as it can often be the largest operating expense for a mine,” says Sasha Milshteyn, founder of Transition Biomining, a company that analyzes the DNA of microbes found in ores to develop custom additives for increased copper recovery. Though the process avoids toxic gas emissions, uses less energy and water than conventional methods and minimizes hazardous chemicals, it has its limitations. It's slower than traditional mining: while smelting can take hours to days, and acid leaching takes days to weeks, microbes do their work over several months. As Barstow puts it, “Biomining won't be an environmental panacea; it will just be quite a bit better than what we do now.” “Modern mining technologies ‘skim the cream' of economically valuable metals from a deposit and leave everything else behind in [waste rock called] tailings,” Barstow says. That waste is worth far more than it usually gets credit for. A study recently published in Science found that recovering the minerals in waste from existing U.S. mines could meet nearly all of the country's critical mineral needs; recovering just 1 percent would substantially reduce import reliance for many elements. “If large mines just added additional recovery circuits to their process, this could bring needed minerals into production relatively quickly,” says the study's lead author Elizabeth Holley, a mining engineer at the Colorado School of Mines. Microbes may be the world's oldest miners, but biomining as a technology is still new, Milshteyn notes, and doesn't yet leverage the full complexity of microbial ecosystems. “I think the next generation of biomining has to contend with that complexity.” Her work has appeared in Time magazine, Forbes, AI Frontiers, Scientific American and SingularityHub, among other outlets. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/is-life-inside-enceladus-cassini-uncovers-complex-organic-chemistry-in/'>Is Life inside Enceladus? Saturn's Ocean Moon Is Awash with Biology's Raw Ingredients</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-01 09:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Artist's concept of the Cassini spacecraft passing through plumes venting from Saturn's ocean moon, Enceladus. For nearly two decades Enceladus, a 500-kilometer-wide moon of Saturn, has been a top target in the hunt for extraterrestrial life. In 2005, shortly after arriving in orbit around the ringed planet, the joint NASA–European Space Agency (ESA) Cassini mission found clinching evidence that Enceladus harbored a liquid-water ocean beneath its bright-white icy crust—plumes of seawater spraying up from the moon's south pole. Astrobiologists have become ever more enthralled by Enceladus ever since, as further studies of the ice grains in the plumes have revealed multiple molecular building blocks of life blasting out from the hidden ocean. Now scientists revisiting data from Cassini—which ended its mission in 2017—have spied even more tantalizing ingredients in the plumes: suites of complex organic molecules which, on Earth, are involved in the chemistry associated with even bigger molecules considered essential for biology. Published Wednesday in Nature Astronomy, the discovery bolsters the case for follow-up missions to search for signs of life within Saturn's enigmatic, ocean-bearing moon. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “These new results are very intriguing and raise the question of what, exactly, is the true nature and origin of organics within Enceladus's ocean,” says Kevin Hand, a planetary scientist and director of the Ocean Worlds Lab at NASA's Jet Propulsion Laboratory, who was not involved in the study. Besides its remoteness from Earth, Enceladus has kept so many of its secrets for so long because the Cassini orbiter wasn't really designed for such deep scrutiny of a single, specific object. Cassini launched nearly 30 years ago carrying instruments built in the 1980s or 1990s, back when the moon's subsurface ocean and south polar plumes were unknown. Repurposing that vintage kit for in-depth astrobiology was difficult—not least of all because of how hard the resulting data were to work with. “Cassini's instruments were made to analyze the chemical composition of dust and ice particles, but they weren't meant to explore the subsurface material of Enceladus,” Khawaja says. The result was that countless different types of particles with similar masses blurred together in the CDA's detections, making it nearly impossible for scientists back on Earth to discern them. They could clearly see that ordinary water molecules comprised the vast majority of collected material—nearly 98 percent, Khawaja says. Cassini's speed during E5 was nearly 18 kilometers per second (km/s)—about 6 km/s faster than most other flybys—which translated into massive improvements in the CDA data. “The impact speed was higher, and at such high speeds, water molecules shatter. E5 was also lucky because it sent Cassini plowing through a plume that had been ejected mere minutes beforehand. This ensured the material came fresh out of Enceladus's subsurface and had not been altered or degraded by cosmic radiation. But years of painstaking data analysis were still to come. “I think it makes a lot of sense that it would take diligence and patience to fully understand the CDA data. I applaud them for taking such care in their analyses,” says Shannon MacKenzie, a planetary scientist at the Johns Hopkins University Applied Physics Laboratory, who wasn't involved in the study. The team's work revealed the plumes contain several chemical compounds Cassini previously detected in the E ring, a torus of ice and dust Enceladus makes as it spews material in its orbit around Saturn. The analysis found other materials that had been seen in the E ring as well: amines, aromatics and oxygen-bearing molecules. “In these fresh grains, we've got molecules like esters and ethers, which were carrying oxygen in themselves and had double bonds,” Khawaja says. Another new finding was the presence of compounds where oxygen and nitrogen were probably combined. “We suspect these are sort of intermediates to make further, complex organics, maybe potentially organics that are biologically relevant,” he adds. Certainty is elusive because the organics collected by CDA were shattered into multiple tiny fragments; researchers are still figuring out how to piece these fragments back together. “This work shows that some of the fragments are indeed derived from quite large and complex organic compounds,” Hand says. Khawaja already has ideas about what follow-up missions might find by delving deeper with better, state-of-the-art instruments. This network of reactions could yield lipids, too—molecules that can arrange themselves into cell membranes. Even so, Khawaja notes, “we don't have a clue about any actual biological relevance yet.” There's also some room left for discovery in the Cassini data. “There are still certain spectral types that I see and don't understand,” Khawaja says. But not all are equally sold out on the lander idea. “The results of this study corroborate mission concepts that don't even need to land—we could just continually fly through the plumes and collect fresh material,” Hand says. “Why risk landing when Enceladus is handing out free samples?” Regardless of such logistical debates, what's clear is that Enceladus remains one of the most alluring destinations to look for extraterrestrial life. “Water, energy and the right chemicals—all three keystones of habitability are there,” Khawaja says. Even if future studies fail to find life, he argues, the implications would be enormous. Jacek Krywko is a freelance writer who covers space exploration, artificial intelligence, computer science and all sorts of engineering wizardry. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            