
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-29</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/did-inhaling-xenon-gas-really-help-mount-everest-climbers-reach-the-summit/'>Did Inhaling Xenon Gas Really Help Mount Everest Climbers Reach the Summit in Record Time?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 16:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These Climbers Summited Mount Everest in Record Time. British climbers recently reached the top of Mount Everest in record time. Part of the Himalayan mountains, Mount Everest is considered the highest point on Earth, reaching a height of more than 8.8 kilometers. Last week a quartet of British climbers made it to the top of Mount Everest—and spent less than a week on the total round trip from London. That's weeks fewer than it usually takes to acclimate to the high elevation, scale the world's highest peak and head home. But experts on the medical uses of xenon are uncertain that it was a decisive factor. We just don't know,” says Andrew Subudhi, a professor of human physiology and nutrition at the University of Colorado Colorado Springs, who studies human performance in low-oxygen environments. “From the scientific evidence, I can't see anything that is definitive or even proof-of-concept yet.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. It's been used as an anesthetic on occasion since the 1950s, says Robert Dickinson, a senior lecturer in medicine at Imperial College London. This protective quality has been demonstrated in many animal studies and a handful of small human trials, Dickinson says. Activating these receptors has an excitatory effect on neurons, but xenon tamps down NMDA activity. After a brain injury, NMDA receptors can become overexcited, causing further cell death, so quieting these receptors might prevent additional damage. Those are xenon's best-studied effects on human health. Before attempting Everest's summit, climbers must hang out in Kathmandu, Nepal, and then Everest Base Camp for weeks, lest they fall prey to altitude sickness, which is marked by fatigue, headache, nausea and confusion. Xenon, a noble gas, is getting attention in the sports medicine world for its potential to increase the production of oxygen-carrying red blood cells. The problem is: no one really knows if the EPO boost provided by xenon is enough to make a real difference in how someone handles a high elevation. Davide Cattano, an anesthesiologist at the McGovern Medical School at the University of Texas Health Science Center at Houston, did some of the animal research that has shown that xenon increases a blood factor called hypoxia-inducible factor 1–alpha (HIF-1α), which in turn can increase EPO. He's skeptical that the recent Everest climbers saw much benefit. “The level of HIF that you're inducing does not justify this superhuman capability,” Cattano says. One 2019 study published in the Journal of Applied Physiology tested 12 runners who were randomly assigned to inhale air that contained 70 percent xenon or a sham gas for two minutes each day for several weeks before they ran three kilometers. The runners who inhaled xenon saw an increase in EPO in their blood, but they didn't show any improvement in fitness or athletic performance, as measured by their running speed and their heart rate and respiration during exercise. Even dosing people with EPO directly with injections may not prevent altitude sickness or improve performance at high elevation, Subudhi says. In a study that is currently in review for publication in a scientific journal, he and his colleagues tested EPO injections on a small group of mountain-climbing athletes, and these subjects didn't see any benefits. It is possible xenon improved the climbers' oxygen-carrying capacity by boosting their EPO, experts say. Just the act of breathing a heavy gas like xenon might also result in some change to lung capacity, he says, even if the EPO effect is small. These tents create a low-oxygen environment, which definitely increases EPO and red blood cell production. Xenon is expensive, which has limited its use as an anesthetic and in athletics. “People are literally fighting for their lives at high altitudes, and if you're doing things that may give you a small chance of improving your rate of success, yeah, it might be worth it to some people,” he says. “Not everybody is going to sit there and make a completely scientific decision about their life.” Stephanie Pappas is a freelance science journalist based in Denver, Colo.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09183-9'>Mouse liver assembloids model periportal architecture and biliary fibrosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 15:41:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Current tissue-derived organoids fail to reproduce the complex cellular composition and tissue architecture observed in vivo3. Here, we describe a multicellular organoid system composed of adult hepatocytes, cholangiocytes and mesenchymal cells that recapitulates the architecture of the liver periportal region and, when manipulated, models aspects of cholestatic injury and biliary fibrosis. We first generate reproducible hepatocyte organoids with functional bile canaliculi network that retain morphological features of in vivo tissue. By combining these with cholangiocytes and portal fibroblasts, we generate assembloids that mimic the cellular interactions of the periportal region. Strikingly, manipulating the relative number of portal mesenchymal cells is sufficient to induce a fibrotic-like state, independently of an immune compartment. By generating chimeric assembloids of mutant and wild-type cells, or after gene knockdown, we show proof-of-concept that our system is amenable to investigating gene function and cell-autonomous mechanisms. Taken together, we demonstrate that liver assembloids represent a suitable in vitro system to study bile canaliculi formation, bile drainage, and how different cell types contribute to cholestatic disease and biliary fibrosis, in an all-in-one model. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Present address: Institut Curie, Département de Génétique et Biologie du Développement, Paris, France Present address: Department of Mathematics, UCLA, Los Angeles, California, USA Present address: Center for Organoid Systems, Technische Universität München, Munich, Germany These authors contributed equally: Anna M. Dowbaj, Aleksandra Sljukic Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany Anna M. Dowbaj, Aleksandra Sljukic, Armin Niksic, Cedric Landerer, Julien Delpierre, Haochen Yang, Aparajita Lahree, Ariane C. Kühn, Sarah Seifert, Heather A. Harrington, Marino Zerial & Meritxell Huch Center for Systems Biology (CSBD), Dresden, Germany Cedric Landerer, Haochen Yang, Heather A. Harrington, Marino Zerial & Meritxell Huch Haochen Yang, David Beers, Helen M. Byrne & Heather A. Harrington You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Cytokine array blots from 2 biological replicates, presented as raw scans of the signal, and overlay with the blot brightfield image. Images are annotated for the sample name. TPM values from bulk RNA sequencing for each gene were used for heatmaps in Extended Data Figure 3b (HM-Wnt HepOrg vs freshly isolated hepatocytes); Extended Data Figure 5a (HM-Wnt and MM media HepOrg vs freshly isolated hepatocytes); Extended Data Figure 10a (7 days or 2.5 weeks homeostasis and fibrotic-like assembloids). Gene set enrichment analysis (GSEA) of hepatocytes, cholangiocytes and portal mesenchyme; Related to Extended Data Figure 7k-m. Related to Figure 4e; GSEA Homeostasis and Fibrosis-like Assembloids using KEGG, reactome and MSigDB_Hallmarks databases; Related to Extended Data Figure 10e-g; GSEA of cells from fibrosis-like versus homeostasis assembloids for hepatocytes (e), cholangiocytes (f), and mesenchyme (g). Differential gene expression (DGE) between HepOrg and freshly isolated hepatocytes. Differential gene expression for comparison of HepOrg in HM-Wnt or MM media cultured at passage 2 for 1 week or 2 weeks, compared to freshly isolated hepatocytes. HepOrg was treated with 5-CFDA (pseudo-coloured Royal LUT). HepOrg was treated with 5-CFDA (pseudo-coloured Royal LUT). HepOrg transport CLF, CMFDA and fluorescent Phosphatidylcholine (PC). Maximum intensity projection of a live cell imaging of a bubbly-shaped HepOrg in MM media at passage 2, showing functional update and transport of CLF (Video 3), CMFDA (Video 4) and fluorescent PC (Video 5). HepOrg transport CLF, CMFDA and fluorescent Phosphatidylcholine (PC). Maximum intensity projection of a live cell imaging of a bubbly-shaped HepOrg in MM media at passage 2, showing functional update and transport of CLF (Video 3), CMFDA (Video 4) and fluorescent PC (Video 5). HepOrg transport CLF, CMFDA and fluorescent Phosphatidylcholine (PC). Maximum intensity projection of a live cell imaging of a bubbly-shaped HepOrg in MM media at passage 2, showing functional update and transport of CLF (Video 3), CMFDA (Video 4) and fluorescent PC (Video 5). Maximum intensity projection of assembloid formation 2 days after seeding. Assembloids can still recruit cells after being formed. Maximum intensity projection of assembloid formation 2 days after seeding. Assembloids can still recruit cells after being formed. Representative 3D-reconstruction showing the connection between bile canaliculi-bile duct in liver tissue. Reconstruction was generated from multiphoton image showing a continuous lumen of bile canaliculi (CD13, green) entering bile duct (PCK, magenta). Hepatocytes, which connect their bile canaliculi to the bile duct lumen are visualised in different colours (red, yellow, cyan). Representative 3D-reconstruction showing the connection between bile canaliculi-bile duct in a periportal liver assembloid. Reconstruction was generated from high-resolution Airyscan 3D image showing a continuous lumen of bile canaliculi (ZO-1, green) entering bile duct (KRT19, magenta). Hepatocytes, which connect their bile canaliculi to the bile duct lumen are visualised in different colours (red, yellow). Representative 3D-reconstruction showing the connection between bile canaliculi-bile duct in liver tissue. Reconstruction was generated from multiphoton image showing a continuous lumen of bile canaliculi (CD13, green) entering bile duct (PCK, magenta). Hepatocytes, which connect their bile canaliculi to the bile duct lumen are visualised in different colours (red, yellow, cyan). Representative 3D-reconstruction showing the connection between bile canaliculi-bile duct in a periportal liver assembloid Animation of a 3D reconstruction showing interface of bile canaliculi-bile duct in a mouse periportal assembloid. Reconstruction was generated from confocal image showing a continuous lumen of bile canaliculi (CD13, green) entering bile duct (nuc-tdTomato, magenta). Hepatocytes, which connect their bile canaliculi to the bile duct lumen are visualised in different colours (red, yellow, cyan). Live imaging of the uptake and flow of the bile acid analogue cholyl-L-lysyl-fluorescein (CLF, pseudo-colour range) in assembloids shows functional transport of bile salts from bile canaliculi into the lumen of the bile duct lined by cholangiocytes (magenta). Live imaging of the uptake and flow of the bile acid analogue cholyl-L-lysyl-fluorescein (CLF, pseudo-colour range) in assembloids shows functional transport of bile salts from bile canaliculi into the lumen of the bile duct lined by cholangiocytes (magenta). Structures with aberrant cholangiocyte ratio and non-physiological BC-BD connection do not transport bile acid analogue. CLF uptake is not observed in structures with aberrant architecture where cholangiocytes are not embedded in the organoid (mem-tdTomato, magenta). Fibrotic-like assembloids show disruption of cell integrity. Maximum intensity projection of a 3D assembloid showing big bursts of cell-free DNA signal, imaging from 2 days after seeding. Fibrotic-like assembloids show no functional bile duct-bile canaliculi connection. Live imaging of the bile acid analogue cholyl-L-lysyl-fluorescein (CLF, pseudo-colour range) in fibrotic-like assembloids is not observed. Mouse liver assembloids model periportal architecture and biliary fibrosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41579-025-01193-8'>Pseudomonas aeruginosa : ecology, evolution, pathogenesis and antimicrobial susceptibility</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 13:28:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Microbiology (2025)Cite this article Metrics details Pseudomonas aeruginosa has long served as a model organism in microbiology, particularly for studies on gene expression, quorum sensing, antibiotic resistance, virulence and biofilm formation. Its genetic tractability has advanced the understanding of complex regulatory networks and experimental evolution. The versatility of this bacterium stems from its genomic variability, metabolic flexibility and phenotypic diversity, enabling it to thrive in diverse environments, both as a harmless saprophyte and an opportunistic human pathogen. P. aeruginosa can cause acute and chronic human infections, particularly in patients with underlying immune deficiencies. Its intrinsic antibiotic tolerance and resistance, together with its ability to produce multiple virulence factors while rapidly adapting to infection conditions, pose a major clinical challenge. In this Review, we explore key features contributing to the ecological and pathogenic versatility of P. aeruginosa. We examine the molecular mechanisms and ecological and evolutionary implications of quorum sensing and biofilm formation. We explore the virulence strategies and in vivo fitness determinants, as well as the evolutionary dynamics and global epidemiology of P. aeruginosa, with a focus on antimicrobial resistance. Finally, we discuss emerging strategies to control P. aeruginosa infections and address outstanding questions in the field. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Gessard, C. Classics in infectious diseases. On the blue and green coloration that appears on bandages. By Carle Gessard (1850–1925). Google Scholar Crone, S. et al. The environmental occurrence of Pseudomonas aeruginosa. In this study, the authors reevaluate the ubiquitous presence of P. aeruginosa, which is generally rare in pristine environments but shows a higher prevalence in areas impacted by human activities. Google Scholar Micek, S. T. et al. Pseudomonas aeruginosa bloodstream infection: importance of appropriate initial antimicrobial treatment. Agents Chemother. Google Scholar Newman, J. N., Floyd, R. V. & Fothergill, J. L. Invasion and diversity in Pseudomonas aeruginosa urinary tract infections. Google Scholar Gitter, A., Mena, K. D., Mendez, K. S., Wu, F. & Gerba, C. P. Eye infection risks from Pseudomonas aeruginosa via hand soap and eye drops. Article PubMed Google Scholar Turner, K. H., Everett, J., Trivedi, U., Rumbaugh, K. P. & Whiteley, M. Requirements for Pseudomonas aeruginosa acute burn and chronic surgical wound infection. PLoS Genet. Google Scholar Rossi, E. et al. Pseudomonas aeruginosa adaptation and evolution in patients with cystic fibrosis. Google Scholar Martinez-Solano, L., Macia, M. D., Fajardo, A., Oliver, A. & Martinez, J. L. Chronic Pseudomonas aeruginosa infection in chronic obstructive pulmonary disease. Google Scholar Fernandez-Barat, L. et al. Intensive care unit-acquired pneumonia due to Pseudomonas aeruginosa with and without multidrug resistance. Article PubMed Google Scholar Cao, P. et al. A Pseudomonas aeruginosa small RNA regulates chronic and acute infection. In this study, the authors identify the oxygen-responsive small RNA SicX as the chronic-to-acute switch in P. aeruginosa during mammalian infection. Google Scholar Centers for Disease Control and Prevention. Antibiotic resistance threats in the United States. CDC https://www.cdc.gov/drugresistance/biggest-threats.html (2019). Magiorakos, A. P. et al. Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: an international expert proposal for interim standard definitions for acquired resistance. Google Scholar Woodford, N., Turton, J. F. & Livermore, D. M. Multiresistant Gram-negative bacteria: the role of high-risk clones in the dissemination of antibiotic resistance. FEMS Microbiol. Google Scholar Stover, C. K. et al. Complete genome sequence of Pseudomonas aeruginosa PAO1, an opportunistic pathogen. Google Scholar Taylor, P. K., Van Kessel, A. T. M., Colavita, A., Hancock, R. E. W. & Mah, T. F. A novel small RNA is important for biofilm formation and pathogenicity in Pseudomonas aeruginosa. PLoS ONE 12, e0182582 (2017). Google Scholar Gomez-Lozano, M., Marvig, R. L., Molin, S. & Long, K. S. Genome-wide identification of novel small RNAs in Pseudomonas aeruginosa. Google Scholar Wurtzel, O. et al. The single-nucleotide resolution transcriptome of Pseudomonas aeruginosa grown in body temperature. PLoS Pathog. Google Scholar Green, S. K., Schroth, M. N., Cho, J. J., Kominos, S. K. & Vitanza-jack, V. B. Agricultural plants and soil as a reservoir for Pseudomonas aeruginosa. Google Scholar Khan, N. H. et al. Isolation of Pseudomonas aeruginosa from open ocean and comparison with freshwater, clinical, and animal isolates. Google Scholar Haritash, A. K. & Kaushik, C. P. Biodegradation aspects of polycyclic aromatic hydrocarbons (PAHs): a review. Google Scholar Rybtke, M., Hultqvist, L. D., Givskov, M. & Tolker-Nielsen, T. Pseudomonas aeruginosa biofilm infections: community structure, antimicrobial tolerance and immune response. Google Scholar Favero, M. S., Carson, L. A., Bond, W. W. & Petersen, N. J. Pseudomonas aeruginosa: growth in distilled water from hospitals. Google Scholar Ringen, L. M. & Drake, C. H. A study of the incidence of Pseudomonas aeruginosa from various natural sources. Google Scholar Botzenhart, K. & Döring, G. in Pseudomonas aeruginosa as an Opportunistic Pathogen (eds Campa, M., Bendinelli, M. & Friedman, H.) 1–18 (Springer, 1993). Qin, S. et al. Pseudomonas aeruginosa: pathogenesis, virulence factors, antibiotic resistance, interaction with host, technology advances and emerging therapeutics. Here, the authors present a comprehensive review of the virulence factors of P. aeruginosa. Google Scholar Sikdar, R. & Elias, M. H. Evidence for complex interplay between quorum sensing and antibiotic resistance in Pseudomonas aeruginosa. Article PubMed Google Scholar Shrout, J. D. et al. The impact of quorum sensing and swarming motility on Pseudomonas aeruginosa biofilm formation is nutritionally conditional. Google Scholar West, S. A., Winzer, K., Gardner, A. & Diggle, S. P. Quorum sensing and the confusion about diffusion. Trends Microbiol. Google Scholar Whiteley, M., Diggle, S. P. & Greenberg, E. P. Progress in and promise of bacterial quorum sensing research. Google Scholar Wagner, V. E., Bushnell, D., Passador, L., Brooks, A. I. & Iglewski, B. H. Microarray analysis of Pseudomonas aeruginosa quorum-sensing regulons: effects of growth phase and environment. Google Scholar Whiteley, M., Lee, K. M. & Greenberg, E. P. Identification of genes controlled by quorum sensing in Pseudomonas aeruginosa. Natl Acad. Google Scholar Schuster, M., Lostroh, C. P., Ogi, T. & Greenberg, E. P. Identification, timing, and signal specificity of Pseudomonas aeruginosa quorum-controlled genes: a transcriptome analysis. Google Scholar Letizia, M. et al. PqsE expands and differentially modulates the RhlR quorum sensing regulon in Pseudomonas aeruginosa. Article PubMed Google Scholar Rampioni, G. et al. Unravelling the genome-wide contributions of specific 2-alkyl-4-quinolones and PqsE to quorum sensing in Pseudomonas aeruginosa. PLoS Pathog. Google Scholar Schuster, M. & Greenberg, E. P. A network of networks: quorum-sensing gene regulation in Pseudomonas aeruginosa. Google Scholar Borgert, S. R. et al. Moonlighting chaperone activity of the enzyme PqsE contributes to RhlR-controlled virulence of Pseudomonas aeruginosa. Google Scholar Taylor, I. R. et al. Inhibitor mimetic mutations in the Pseudomonas aeruginosa PqsE enzyme reveal a protein–protein interaction with the quorum-sensing receptor RhlR that is vital for virulence factor production. ACS Chem. Google Scholar Pearson, J. P., Feldman, M., Iglewski, B. H. & Prince, A. Pseudomonas aeruginosa cell-to-cell signaling is required for virulence in a model of acute pulmonary infection. Google Scholar Rumbaugh, K. P., Griswold, J. A., Iglewski, B. H. & Hamood, A. N. Contribution of quorum sensing to the virulence of Pseudomonas aeruginosa in burn wound infections. Google Scholar Rampioni, G. et al. Transcriptomic analysis reveals a global alkyl-quinolone-independent regulatory role for PqsE in facilitating the environmental adaptation of Pseudomonas aeruginosa to plant and animal hosts. Google Scholar Latifi, A., Foglino, M., Tanaka, K., Williams, P. & Lazdunski, A. A hierarchical quorum-sensing cascade in Pseudomonas aeruginosa links the transcriptional activators LasR and RhIR (VsmR) to expression of the stationary-phase sigma factor RpoS. Google Scholar McGrath, S., Wade, D. S. & Pesci, E. C. Dueling quorum sensing systems in Pseudomonas aeruginosa control the production of the Pseudomonas quinolone signal (PQS). FEMS Microbiol. Google Scholar Wade, D. S. et al. Regulation of Pseudomonas quinolone signal synthesis in Pseudomonas aeruginosa. Google Scholar McKnight, S. L., Iglewski, B. H. & Pesci, E. C. The Pseudomonas quinolone signal regulates rhl quorum sensing in Pseudomonas aeruginosa. Google Scholar Feltner, J. B. et al. LasR variant cystic fibrosis isolates reveal an adaptable quorum-sensing hierarchy in Pseudomonas aeruginosa. Google Scholar Kostylev, M. et al. Evolution of the Pseudomonas aeruginosa quorum-sensing hierarchy. Natl Acad. Google Scholar Cruz, R. L. et al. RhlR-regulated acyl-homoserine lactone quorum sensing in a cystic fibrosis isolate of Pseudomonas aeruginosa. Google Scholar Dekimpe, V. & Deziel, E. Revisiting the quorum-sensing hierarchy in Pseudomonas aeruginosa: the transcriptional regulator RhlR regulates LasR-specific factors. Google Scholar Oshri, R. D., Zrihen, K. S., Shner, I., Omer Bendori, S. & Eldar, A. Selection for increased quorum-sensing cooperation in Pseudomonas aeruginosa through the shut-down of a drug resistance pump. ISME J. Google Scholar O'Loughlin, C. T. et al. A quorum-sensing inhibitor blocks Pseudomonas aeruginosa virulence and biofilm formation. Natl Acad. Google Scholar O'Connor, K., Zhao, C. Y., Mei, M. & Diggle, S. P. Frequency of quorum-sensing mutations in Pseudomonas aeruginosa strains isolated from different environments. Google Scholar Trottier, M. C. et al. The end of the reign of a “master regulator”? A defect in function of the LasR quorum sensing regulator is a common feature of Pseudomonas aeruginosa isolates. Article PubMed Google Scholar Groleau, M. C., Taillefer, H., Vincent, A. T., Constant, P. & Deziel, E. Pseudomonas aeruginosa isolates defective in function of the LasR quorum sensing regulator are frequent in diverse environmental niches. Google Scholar Vanderwoude, J. et al. The evolution of virulence in Pseudomonas aeruginosa during chronic wound infection. Google Scholar Azimi, S., Klementiev, A. D., Whiteley, M. & Diggle, S. P. Bacterial quorum sensing during infection. Google Scholar Diggle, S. P., Griffin, A. S., Campbell, G. S. & West, S. A. Cooperation and conflict in quorum-sensing bacterial populations. This study experimentally demonstrates that quorum sensing in P. aeruginosa can be exploited by non-cooperative individuals, with kin selection mitigating this issue. Google Scholar Zhao, K. et al. Evolution of lasR mutants in polymorphic Pseudomonas aeruginosa populations facilitates chronic infection of the lung. Google Scholar Sandoz, K. M., Mitzimberg, S. M. & Schuster, M. Social cheating in Pseudomonas aeruginosa quorum sensing. Natl Acad. Google Scholar Rumbaugh, K. P. et al. Quorum sensing and the social evolution of bacterial virulence. Google Scholar D'Argenio, D. A. et al. Growth phenotypes of Pseudomonas aeruginosa lasR mutants adapted to the airways of cystic fibrosis patients. Google Scholar Hoffman, L. R. et al. Nutrient availability as a mechanism for selection of antibiotic tolerant Pseudomonas aeruginosa within the CF airway. PLoS Pathog. Google Scholar Barth, A. L. & Pitt, T. L. The high amino-acid content of sputum from cystic fibrosis patients promotes growth of auxotrophic Pseudomonas aeruginosa. Google Scholar Clay, M. E. et al. Pseudomonas aeruginosa lasR mutant fitness in microoxia is supported by an Anr-regulated oxygen-binding hemerythrin. Natl Acad. Google Scholar Azimi, S. et al. Allelic polymorphism shapes community function in evolving Pseudomonas aeruginosa populations. ISME J. Google Scholar Hoffman, L. R. et al. Pseudomonas aeruginosa lasR mutants are associated with cystic fibrosis lung disease progression. Google Scholar Chugani, S. A. et al. QscR, a modulator of quorum-sensing signal synthesis and virulence in Pseudomonas aeruginosa. Natl Acad. Google Scholar Ding, F. et al. The Pseudomonas aeruginosa orphan quorum sensing signal receptor QscR regulates global quorum sensing gene expression by activating a single linked operon. mBio https://doi.org/10.1128/mBio.01274-18 (2018). Smith, P. & Schuster, M. Antiactivators prevent self-sensing in Pseudomonas aeruginosa quorum sensing. Natl Acad. Google Scholar Rattray, J. B. et al. Bacterial quorum sensing allows graded and bimodal cellular responses to variations in population density. Article PubMed Google Scholar Mellini, M. et al. RsaL-driven negative regulation promotes heterogeneity in Pseudomonas aeruginosa quorum sensing. Article PubMed Google Scholar Bondi, R. et al. The multi-output incoherent feedforward loop constituted by the transcriptional regulators LasR and RsaL confers robustness to a subset of quorum sensing genes in Pseudomonas aeruginosa. Google Scholar Ackermann, M. A functional perspective on phenotypic heterogeneity in microorganisms. Google Scholar Sindeldecker, D. & Stoodley, P. The many antibiotic resistance and tolerance strategies of Pseudomonas aeruginosa. Google Scholar Ciofu, O., Moser, C., Jensen, P. O. & Hoiby, N. Tolerance and resistance of microbial biofilms. Google Scholar Franklin, M. J., Nivens, D. E., Weadge, J. T. & Howell, P. L. Biosynthesis of the Pseudomonas aeruginosa extracellular polysaccharides, alginate, Pel, and Psl. Google Scholar Colvin, K. M. et al. The Pel and Psl polysaccharides provide Pseudomonas aeruginosa structural redundancy within the biofilm matrix. Google Scholar Martin, D. W. et al. Mechanism of conversion to mucoidy in Pseudomonas aeruginosa infecting cystic fibrosis patients. Natl Acad. Google Scholar Hentzer, M. et al. Alginate overproduction affects Pseudomonas aeruginosa biofilm structure and function. Google Scholar Colvin, K. M. et al. The Pel polysaccharide can serve a structural and protective role in the biofilm matrix of Pseudomonas aeruginosa. PLoS Pathog. Google Scholar Le Mauff, F. et al. The Pel polysaccharide is predominantly composed of a dimeric repeat of alpha-1,4 linked galactosamine and N-acetylgalactosamine. Google Scholar Murakami, K. et al. Role of psl genes in antibiotic tolerance of adherent Pseudomonas aeruginosa. Agents Chemother. Google Scholar Jennings, L. K. et al. Pel is a cationic exopolysaccharide that cross-links extracellular DNA in the Pseudomonas aeruginosa biofilm matrix. Natl Acad. Google Scholar Wang, S. et al. The exopolysaccharide Psl–eDNA interaction enables the formation of a biofilm skeleton in Pseudomonas aeruginosa. Google Scholar Whitchurch, C. B., Tolker-Nielsen, T., Ragas, P. C. & Mattick, J. S. Extracellular DNA required for bacterial biofilm formation. Google Scholar Turnbull, L. et al. Explosive cell lysis as a mechanism for the biogenesis of bacterial membrane vesicles and biofilms. Google Scholar Nazeer, R. R., Wang, M. & Welch, M. More than just a gel: the extracellular matrixome of Pseudomonas aeruginosa. Google Scholar Couto, N., Schooling, S. R., Dutcher, J. R. & Barber, J. Proteome profiles of outer membrane vesicles and extracellular matrix of Pseudomonas aeruginosa biofilms. J. Proteome Res. Google Scholar Diggle, S. P. et al. The galactophilic lectin, LecA, contributes to biofilm development in Pseudomonas aeruginosa. Google Scholar Passos da Silva, D. et al. The Pseudomonas aeruginosa lectin LecB binds to the exopolysaccharide Psl and stabilizes the biofilm matrix. Google Scholar Reichhardt, C. The Pseudomonas aeruginosa biofilm matrix protein CdrA has similarities to other fibrillar adhesin proteins. Article PubMed Google Scholar Ha, D. G. & O'Toole, G. A. c-di-GMP and its effects on biofilm formation and dispersion: a Pseudomonas aeruginosa review. Google Scholar Zheng, X. et al. The surface interface and swimming motility influence surface-sensing responses in Pseudomonas aeruginosa. Natl Acad. Google Scholar Rumbaugh, K. P. & Bjarnsholt, T. Microbial primer: in vivo biofilm. Google Scholar Bjarnsholt, T. et al. The in vivo biofilm. Trends Microbiol. Google Scholar Klausen, M., Aaes-Jorgensen, A., Molin, S. & Tolker-Nielsen, T. Involvement of bacterial migration in the development of complex multicellular structures in Pseudomonas aeruginosa biofilms. Google Scholar Nickel, J. C., Downey, J. & Costerton, J. W. Ultrastructural study of microbiologic colonization of urinary catheters. Google Scholar Costerton, J. W. et al. New methods for the detection of orthopedic and other biofilm infections. FEMS Immunol. Google Scholar Azimi, S. et al. O-specific antigen-dependent surface hydrophobicity mediates aggregate assembly type in Pseudomonas aeruginosa. Article PubMed Google Scholar Secor, P. R., Michaels, L. A., Bublitz, D. C., Jennings, L. K. & Singh, P. K. The depletion mechanism actuates bacterial aggregation by exopolysaccharides and determines species distribution and composition in bacterial aggregates. Cell Infect. Google Scholar Secor, P. R., Michaels, L. A., Ratjen, A., Jennings, L. K. & Singh, P. K. Entropically driven aggregation of bacteria by host polymers promotes antibiotic tolerance in Pseudomonas aeruginosa. Natl Acad. Google Scholar Bjarnsholt, T. et al. Pseudomonas aeruginosa biofilms in the respiratory tract of cystic fibrosis patients. Article PubMed Google Scholar Homøe, P., Bjarnsholt, T., Wessman, M., Sørensen, H. C. & Johansen, H. K. Morphological evidence of biofilm formation in Greenlanders with chronic suppurative otitis media. Article PubMed Google Scholar Kirketerp-Moller, K. et al. Distribution, organization, and ecology of bacteria in chronic wounds. Google Scholar Kragh, K. N., Tolker-Nielsen, T. & Lichtenberg, M. The non-attached biofilm aggregate. Google Scholar Kolpen, M. et al. Bacterial biofilms predominate in both acute and chronic human lung infections. Article PubMed Google Scholar Jennings, L. K. et al. Pseudomonas aeruginosa aggregates in cystic fibrosis sputum produce exopolysaccharides that likely impede current therapies. This study demonstrates that the morphology of P. aeruginosa biofilm aggregate in cystic fibrosis sputum is consistent with a polysaccharide-dependent aggregation mechanism. Google Scholar Rossy, T. et al. Pseudomonas aeruginosa type IV pili actively induce mucus contraction to form biofilms in tissue-engineered human airways. PLoS Biol. Google Scholar Rumbaugh, K. P. & Whiteley, M. Towards improved biofilm models. Liao, C., Huang, X., Wang, Q., Yao, D. & Lu, W. Virulence factors of Pseudomonas aeruginosa and antivirulence strategies to combat its drug resistance. Cell Infect. Google Scholar Hauser, A. R. The type III secretion system of Pseudomonas aeruginosa: infection by injection. Google Scholar Yahr, T. L., Goranson, J. & Frank, D. W. Exoenzyme S of Pseudomonas aeruginosa is secreted by a type III pathway. Google Scholar Liu, P. V. The roles of various fractions of Pseudomonas aeruginosa in its pathogenesis. Identity of the lethal toxins produced in vitro and in vivo. Google Scholar Michalska, M. & Wolf, P. Pseudomonas exotoxin A: optimized by evolution for effective killing. Google Scholar Armstrong, S., Yates, S. P. & Merrill, A. R. Insight into the catalytic mechanism of Pseudomonas aeruginosa exotoxin A. Studies of toxin interaction with eukaryotic elongation factor-2. Google Scholar Jorgensen, R. et al. Exotoxin A–eEF2 complex structure indicates ADP ribosylation by ribosome mimicry. Google Scholar Ochsner, U. A., Johnson, Z., Lamont, I. L., Cunliffe, H. E. & Vasil, M. L. Exotoxin A production in Pseudomonas aeruginosa requires the iron-regulated pvdS gene encoding an alternative sigma factor. Google Scholar Farajnia, S. et al. Protective efficacy of recombinant exotoxin A-flagellin fusion protein against Pseudomonas aeruginosa infection. Google Scholar Galdino, A. C. M., Branquinha, M. H., Santos, A. L. S. & Viganor, L. in Pathophysiological Aspects of Proteases (eds Chakraborti, S. & Dhalla, N. S.) 381–397 (Springer, 2017). Soberon-Chavez, G., Lepine, F. & Deziel, E. Production of rhamnolipids by Pseudomonas aeruginosa. Google Scholar Hall, S. et al. Cellular effects of pyocyanin, a secreted virulence factor of Pseudomonas aeruginosa. Google Scholar Allen, L. et al. Pyocyanin production by Pseudomonas aeruginosa induces neutrophil apoptosis and impairs neutrophil-mediated host defenses in vivo. Google Scholar Gallagher, L. A. & Manoil, C. Pseudomonas aeruginosa PAO1 kills Caenorhabditis elegans by cyanide poisoning. Google Scholar Zuhra, K. & Szabo, C. The two faces of cyanide: an environmental toxin and a potential novel mammalian gasotransmitter. FEBS J. Google Scholar Cornelis, P. & Dingemans, J. Pseudomonas aeruginosa adapts its iron uptake strategies in function of the type of infections. Cell Infect. Google Scholar Lamont, I. L., Beare, P. A., Ochsner, U., Vasil, A. I. & Vasil, M. L. Siderophore-mediated signaling regulates virulence factor production in Pseudomonas aeruginosa. Natl Acad. Google Scholar Janet-Maitre, M. et al. Genome-wide screen in human plasma identifies multifaceted complement evasion of Pseudomonas aeruginosa. PLoS Pathog. This genome-wide screen in human plasma identifies novel factors that contribute to P. aeruginosa's multifaceted evasion of complement-mediated killing. Google Scholar Turner, K. H., Wessel, A. K., Palmer, G. C., Murray, J. L. & Whiteley, M. Essential genome of Pseudomonas aeruginosa in cystic fibrosis sputum. Natl Acad. Google Scholar Vincent, J. L. et al. Prevalence and outcomes of infection among patients in intensive care units in 2017. Google Scholar Pham, T. M. et al. Routes of transmission of VIM-positive Pseudomonas aeruginosa in the adult intensive care unit — analysis of 9 years of surveillance at a university hospital using a mathematical model. Google Scholar Garvey, M. I., Bradley, C. W., Tracey, J. & Oppenheim, B. Continued transmission of Pseudomonas aeruginosa from a wash hand basin tap in a critical care unit. Google Scholar Costa, D. et al. Nosocomial outbreak of Pseudomonas aeruginosa associated with a drinking water fountain. Jones, A. M. et al. Spread of a multiresistant strain of Pseudomonas aeruginosa in an adult cystic fibrosis clinic. Milczewska, J. et al. Clinical outcomes for cystic fibrosis patients with Pseudomonas aeruginosa cross-infections. Article PubMed Doring, G. et al. Generation of Pseudomonas aeruginosa aerosols during handwashing from contaminated sink drains, transmission to hands of hospital personnel, and its prevention by use of a new heating device. Saitou, K., Furuhata, K., Kawakami, Y. & Fukuyama, M. Biofilm formation abilities and disinfectant-resistance of Pseudomonas aeruginosa isolated from cockroaches captured in hospitals. Biocontrol Sci. Okuda, J. et al. Translocation of Pseudomonas aeruginosa from the intestinal tract is mediated by the binding of ExoS to an Na,K-ATPase regulator, FXYD3. Pirnay, J. P. et al. Pseudomonas aeruginosa population structure revisited. PLoS ONE 4, e7740 (2009). Curran, B., Jonas, D., Grundmann, H., Pitt, T. & Dowson, C. G. Development of a multilocus sequence typing scheme for the opportunistic pathogen Pseudomonas aeruginosa. Botelho, J., Grosso, F. & Peixe, L. Antibiotic resistance in Pseudomonas aeruginosa — mechanisms, epidemiology and evolution. Drug Resist. Article PubMed Oliver, A., Mulet, X., Lopez-Causape, C. & Juan, C. The increasing threat of Pseudomonas aeruginosa high-risk clones. Drug Resist. Article PubMed Winstanley, C. et al. Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the liverpool epidemic strain of Pseudomonas aeruginosa. Genome Res. Lee, C. et al. Why? — Successful Pseudomonas aeruginosa clones with a focus on clone C. FEMS Microbiol. Fischer, S., Dethlefsen, S., Klockgether, J. & Tummler, B. Phenotypic and genomic comparison of the two most common ExoU-positive Pseudomonas aeruginosa clones, PA14 and ST235. Treepong, P. et al. Global emergence of the widespread Pseudomonas aeruginosa ST235 clone. Del Barrio-Tofino, E., Lopez-Causape, C. & Oliver, A. Pseudomonas aeruginosa epidemic high-risk clones and their association with horizontally-acquired β-lactamases: 2020 update. Article PubMed Viedma, E. et al. VIM-2-producing multidrug-resistant Pseudomonas aeruginosa ST175 clone, Spain. Breidenstein, E. B., de la Fuente-Nunez, C. & Hancock, R. E. Pseudomonas aeruginosa: all roads lead to resistance. Trends Microbiol. Shepherd, M. J. et al. Ecological and evolutionary mechanisms driving within-patient emergence of antimicrobial resistance. Lorusso, A. B., Carrara, J. A., Barroso, C. D. N., Tuon, F. F. & Faoro, H. Role of efflux pumps on antimicrobial resistance in Pseudomonas aeruginosa. Diaz Caballero, J. et al. Mixed strain pathogen populations accelerate the evolution of antibiotic resistance in patients. Williams, D. et al. Divergent, coexisting Pseudomonas aeruginosa lineages in chronic cystic fibrosis lung infections. Care Med. Wheatley, R. et al. Rapid evolution and host immunity drive the rise and fall of carbapenem resistance during an acute Pseudomonas aeruginosa infection. Meirelles, L. A. et al. Pseudomonas aeruginosa faces a fitness trade-off between mucosal colonization and antibiotic tolerance during airway infection. Andersson, D. I., Nicoloff, H. & Hjort, K. Mechanisms and clinical relevance of bacterial heteroresistance. Ikonomidis, A. et al. Efflux system overexpression and decreased OprD contribute to the carbapenem heterogeneity in Pseudomonas aeruginosa. FEMS Microbiol. Mei, S., Gao, Y., Zhu, C., Dong, C. & Chen, Y. Research of the heteroresistance of Pseudomonas aeruginosa to imipenem. Xu, Y. et al. Mechanisms of heteroresistance and resistance to imipenem in Pseudomonas aeruginosa. Drug Resist. Li, W. R. et al. Pseudomonas aeruginosa heteroresistance to levofloxacin caused by upregulated expression of essential genes for DNA replication and repair. Brauncajs, M., Bielec, F., Macieja, A. & Pastuszak-Lewandoska, D. Cefiderocol — an effective antimicrobial for MDR infections but a challenge for routine antimicrobial susceptibility testing. Hahn, A. et al. Bacteriophage therapy for pan-drug-resistant Pseudomonas aeruginosa in two persons with cystic fibrosis. High Impact Case Rep. 11, 23247096231188243 (2023). Jault, P. et al. Efficacy and tolerability of a cocktail of bacteriophages to treat burn wounds infected by Pseudomonas aeruginosa (PhagoBurn): a randomised, controlled, double-blind phase 1/2 trial. Lancet Infect. Article PubMed Rumbaugh, K. P. & Sauer, K. Biofilm dispersion. Rezzoagli, C., Archetti, M., Mignot, I., Baumgartner, M. & Kummerli, R. Combining antibiotics with antivirulence compounds can have synergistic effects and reverse selection for antibiotic resistance in Pseudomonas aeruginosa. PLoS Biol. Allen, R. C., Popat, R., Diggle, S. P. & Brown, S. P. Targeting virulence: can we make evolution-proof drugs? Mei, M., Estrada, I., Diggle, S. P. & Goldberg, J. B. R-pyocins as targeted antimicrobials against Pseudomonas aeruginosa. npj Antimicrob. Lee, D. G. et al. Genomic analysis reveals that Pseudomonas aeruginosa virulence is combinatorial. Genome Biol. Freschi, L. et al. The Pseudomonas aeruginosa pan-genome provides new insights on its population structure, horizontal gene transfer, and pathogenicity. Genome Biol. Rudra, B., Duncan, L., Shah, A. J., Shah, H. N. & Gupta, R. S. Phylogenomic and comparative genomic studies robustly demarcate two distinct clades of Pseudomonas aeruginosa strains: proposal to transfer the strains from an outlier clade to a novel species Pseudomonas paraeruginosa sp. Abram, K. Z., Jun, S. R. & Udaondo, Z. Pseudomonas aeruginosa pangenome: core and accessory genes of a highly resourceful opportunistic pathogen. Kung, V. L., Ozer, E. A. & Hauser, A. R. The accessory genome of Pseudomonas aeruginosa. Darch, S. E. et al. Recombination is a key driver of genomic and phenotypic diversity in a Pseudomonas aeruginosa population during cystic fibrosis infection. Jorth, P. et al. Regional isolation drives bacterial diversification within cystic fibrosis lungs. Cell Host Microbe 18, 307–319 (2015). Vanderwoude, J., Azimi, S., Read, T. D. & Diggle, S. P. The role of hypermutation and collateral sensitivity in antimicrobial resistance diversity of Pseudomonas aeruginosa populations in cystic fibrosis lung infection. Article PubMed Whiteley, M. et al. Gene expression in Pseudomonas aeruginosa biofilms. Palmer, K. L., Aye, L. M. & Whiteley, M. Nutritional cues control Pseudomonas aeruginosa multicellular behavior in cystic fibrosis sputum. Arai, H. Regulation and function of versatile aerobic and anaerobic respiratory metabolism in Pseudomonas aeruginosa. Palmer, G. C. & Whiteley, M. Metabolism and pathogenicity of Pseudomonas aeruginosa infections in the lungs of individuals with cystic fibrosis. Sauvage, S. et al. Impact of carbon source supplementations on Pseudomonas aeruginosa physiology. J. Proteome Res. Cox, C. D. & Parker, J. Use of 2-aminoacetophenone production in identification of Pseudomonas aeruginosa. Dunphy, L. J. et al. Multidimensional clinical surveillance of Pseudomonas aeruginosa reveals complex relationships between isolate source, morphology, and antimicrobial resistance. Article PubMed da Cruz Nizer, W. S. et al. Oxidative stress response in Pseudomonas aeruginosa. Neves, P. R., McCulloch, J. A., Mamizuka, E. M. & Lincopan, N. in Encyclopedia of Food Microbiology 2nd edn (eds Batt, C. A. & Tortorello, M. L.) 253–260 (Academic Press, 2014). Visca, P., Imperi, F. & Lamont, I. L. Pyoverdine siderophores: from biogenesis to biosignificance. Trends Microbiol. Lau, G. W., Hassett, D. J., Ran, H. & Kong, F. The role of pyocyanin in Pseudomonas aeruginosa infection. Trends Mol. Ogunnariwo, J. & Hamilton-Miller, J. M. Brown- and red-pigmented Pseudomonas aeruginosa: differentiation between melanin and pyorubrin. Michel-Briand, Y. & Baysse, C. The pyocins of Pseudomonas aeruginosa. Besse, A., Groleau, M. C. & Deziel, E. Emergence of small colony variants is an adaptive strategy used by Pseudomonas aeruginosa to mitigate the effects of redox imbalance. Article PubMed Dietrich, L. E. et al. Bacterial community morphogenesis is intimately linked to the intracellular redox state. Giallonardi, G. et al. Alkyl-quinolone-dependent quorum sensing controls prophage-mediated autolysis in Pseudomonas aeruginosa colony biofilms. Cell Infect. Rashid, M. H. & Kornberg, A. Inorganic polyphosphate is needed for swimming, swarming, and twitching motilities of Pseudomonas aeruginosa. Natl Acad. Craig, L., Forest, K. T. & Maier, B. Type IV pili: dynamics, biophysics and functional consequences. Holloway, B. W. Genetic recombination in Pseudomonas aeruginosa. Rahme, L. G. et al. Common virulence factors for bacterial pathogenicity in plants and animals. Parsek, M. R. & Greenberg, E. P. Sociomicrobiology: the connections between quorum sensing and biofilms. Trends Microbiol. Cornforth, D. M., Diggle, F. L., Melvin, J. A., Bomberger, J. M. & Whiteley, M. Quantitative framework for model evaluation in microbiology research using Pseudomonas aeruginosa and cystic fibrosis infection as a test case. In this study, the authors provide a framework for quantifying the accuracy of preclinical models relative to human infections. Lewin, G. R. et al. Application of a quantitative framework to improve the accuracy of a bacterial infection model. Natl Acad. Guan, J. et al. Bacteriophage genome engineering with CRISPR–Cas13a. This study develops a genetic editing tool using RNA-targeting CRISPR–Cas13a and P. aeruginosa to manipulate jumbo phages, overcoming challenges posed by their protective phage nucleus. Lewin, G. R., Stocke, K. S., Lamont, R. J. & Whiteley, M. A quantitative framework reveals traditional laboratory growth is a highly accurate model of human oral infection. Natl Acad. Duncan, R. P. et al. Improvement of a mouse infection model to capture Pseudomonas aeruginosa chronic physiology in cystic fibrosis. Natl Acad. Chaikeeratisak, V. et al. Assembly of a nucleus-like structure during viral replication in bacteria. Download references The authors thank the Cystic Fibrosis foundation for grants (WHITEL20A0 and WHITEL22G0) to M.W. and a postdoctoral fellowship to M.L. (LETIZI24G0-BASBAUM); the National Institutes of Health (NIH) and the National Institute of Allergy and Infectious Diseases (NIAID) for funding to S.P.D. School of Biological Sciences, Center for Microbial Dynamics and Infection, Georgia Institute of Technology, Atlanta, GA, USA Morgana Letizia, Stephen P. Diggle & Marvin Whiteley Emory Children's Cystic Fibrosis Center, Atlanta, GA, USA Morgana Letizia, Stephen P. Diggle & Marvin Whiteley You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar The authors contributed equally to all aspects of the article. Correspondence to Marvin Whiteley. is the co-founder and CSO of SynthBiome, Inc. M.L. declare no competing interests. Nature Reviews Microbiology thanks Niels Hoiby, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Stochastic switching between phenotypic states to enhance population long-term fitness in fluctuating environmental conditions. The spatial assembly and distribution of various organisms in an environment through time. A mechanism of bacterial aggregation driven by electrostatic interactions between bacterial cell surfaces and polymers present in the environment. A process in which the reduction of free energy through increased entropy of the whole system induces the stacked aggregation of bacterial cells in polymer-rich environments. Cooperating individuals specialize in carrying out specific tasks, providing an inclusive fitness benefit to all individuals involved. Cell-autonomous and density-independent reception of signals produced by the same cell. Studies on the group behaviours of microorganisms. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Letizia, M., Diggle, S.P. & Whiteley, M. Pseudomonas aeruginosa: ecology, evolution, pathogenesis and antimicrobial susceptibility. Nat Rev Microbiol  (2025). Download citation Accepted: 12 May 2025 Published: 29 May 2025 DOI: https://doi.org/10.1038/s41579-025-01193-8 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Microbiology (Nat Rev Microbiol) ISSN 1740-1534 (online) ISSN 1740-1526 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a64906716/rna-world-replication/'>Scientists Are Pretty Close to Replicating the First Thing That Ever Lived</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At some point, RNA must have replicated itself before the evolutionary arrival of DNA and proteins, and now we might know how. We may earn commission if you buy from a link. It's a question that has lit fires of curiosity in philosophers, theologians, and more recently (at least, historically speaking) scientists for millennia. While the the older guard of high thought used stories or metaphors to derive life's origin story, scientists instead learn about the inner workings of life's smallest building blocks in an attempt to understand how they first formed life billions of years ago. This long scientific exploration has led most evolutionary biologists to the conclusion that, for at least 400 million years, Earth was an “RNA World.” The hypothesis suggests that life first took form due to self-replicating RNA, before the evolutionary arrival of DNA or even proteins. While scientists are very much still on the hunt for evidence that validates the first of these two issues, a team from University College of London (UCL) is closing in on solving the second problem. Published in the journal Nature Chemistry, a team of UCL scientists (along with experts from the MRC Laboratory of Molecular Biology in Cambridge) used three-letter “triplet” RNA building blocks subjected to acid and heat in water. What occurred next is possibly an intimate glimpse of how life first formed on Earth—between the liquid gaps of the ice crystals, these building blocks coated the RNA strands and prevented them from zipping back together. Eventually, the strand was so long that these structures could perform biological functions. “The triplet or three-letter building blocks of RNA we used, called trinucleotides, do not occur in biology today, but they allow for much easier replication. The earliest forms of life are likely to have been quite different from any life that we know about,” James Attwater, lead author of the study from UCL, said in a press statement. “The changing conditions we engineered can occur naturally, for instance with night and day cycles of temperature, or in geothermal environments where hot rocks meet a cold atmosphere.” UCL has long been involved in constructing the play-by-play of life's origins on Earth. This new study now attempts to understand, in a lab setting, how those ancient RNA first began replication, a process that's essential to understanding the foundation of life. “Life is separated from pure chemistry by information, a molecular memory encoded in the genetic material that is transmitted from one generation to the next,” Philipp Holliger, the senior author of the study from MRC Laboratory of Molecular Biology, said in a press statement. Currently, the researchers have only been able to replicate roughly 17 percent of the RNA strand (roughly 30 out of 180 letters), but the team says there's no reason they won't achieve complete replication with improved enzyme efficiency. Although many questions remain, Earth's ancient RNA World could have actually had the capacity for self-replication. It's an intriguing step forward, but the scientific journey continues. Hidden Hydrogen Can Power The World if We Find It This Ancient Larva Still Has Its Brain And Guts</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02707-1'>A visual–omics foundation model to bridge histopathology with spatial transcriptomics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 10:05:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Artificial intelligence has revolutionized computational biology. Recent developments in omics technologies, including single-cell RNA sequencing and spatial transcriptomics, provide detailed genomic data alongside tissue histology. However, current computational models focus on either omics or image analysis, lacking their integration. To address this, we developed OmiCLIP, a visual–omics foundation model linking hematoxylin and eosin images and transcriptomics using tissue patches from Visium data. We transformed transcriptomic data into ‘sentences' by concatenating top-expressed gene symbols from each patch. We curated a dataset of 2.2 million paired tissue images and transcriptomic data across 32 organs to train OmiCLIP integrating histology and transcriptomics. Building on OmiCLIP, our Loki platform offers five key functions: tissue alignment, annotation via bulk RNA sequencing or marker genes, cell-type decomposition, image–transcriptomics retrieval and spatial transcriptomics gene expression prediction from hematoxylin and eosin-stained images. Computational biology has advanced notably with artificial intelligence (AI) for tasks such as gene expression enhancement, single-cell perturbation prediction, tissue annotation, diagnosis, primary tumor origin predictions and image retrieval from hematoxylin and eosin (H&E)-stained images1,2,3,4,5,6,7. These visual–language foundation models support applications like text-to-image and image-to-text retrieval, histology image classification, captioning and diagnosis improvement. Omics data, including transcriptomics and genetics, provide crucial insights into cell types in health and disease, enhancing our understanding of cellular heterogeneity, lineage tracing and disease mechanisms13,14,15,16,17,18,19,20,21,22. Combining omics data with histology images offers complementary information for both research and clinical applications, and has been used for predicting cancer outcomes, prognosis and response to neoadjuvant chemotherapy3. However, existing methods remain task specific and lack a unified multimodal AI model to integrate histology and omics data. Additionally, challenges remain in developing infrastructure to efficiently analyze sequencing data and pathology images together. To address these gaps, we introduce omics and image pretraining, OmiCLIP, a transcriptomic–image dual-encoder foundation model and Loki platform, an infrastructure of multimodal analysis using OmiCLIP as a backbone. To train OmiCLIP, we curated the ST-bank dataset with 2.2 million tissue patches from 1,007 samples across 32 organs with paired whole-slide images (WSIs) and 10x Visium spatial transcriptomics (ST) data. Inspired by large language model-based single-cell models like GenePT23 and Cell2Sentence24, we represented transcriptomics of a tissue patch by a ‘sentence' of top-ranking highly expressed genes, separated by spaces (‘ '). Using this large-scale set of transcriptomics–histology image pairs, we trained the CLIP-based foundation model, integrating both genomic and image data. Building upon OmiCLIP, the Loki platform offers five core functions: tissue alignment, tissue annotation, cell-type decomposition, image–transcriptomics retrieval and ST gene expression prediction (Fig. Loki provides several distinctive features, including aligning H&E images with ST data, annotating tissue H&E images based on bulk RNA sequencing (RNA-seq) or marker genes and decomposing cell types from H&E images with reference to single-cell RNA sequencing (scRNA-seq). We evaluated Loki's functions against 22 state-of-the-art (SOTA) methods on 5 simulation datasets, 19 publicly available experimental datasets and 4 in-house experimental datasets, showing Loki's consistent accuracy and robustness across tasks. We also investigated OmiCLIP's embeddings for clustering and annotating scRNA-seq data and predicting The Cancer Genome Atlas (TCGA) participants' risk levels (Supplementary Notes 1 and 2). a, The workflow of pretraining the OmiCLIP model with paired image–transcriptomics dataset via contrastive learning. b, Workflow of the Loki platform using the OmiCLIP foundation model as an engine. Right diagram lists the existing modules of the Loki platform, including tissue alignment, cell-type decomposition, tissue annotation, ST gene expression prediction and histology image–transcriptomics retrieval. c, The heat map represents image embeddings and transcriptomic embeddings similarity across various organs and disease conditions. HCM, hypertrophic cardiomyopathy; HBV, hepatitis B virus infection. d, Schematic illustration of Loki platform with transfer learning for 3D tissue analysis. Transcriptomics provides insights into cellular diversity within tissues, making it a natural indicator of tissue diversity25. ST technologies bridge histopathology images and transcriptomics data, enabling the development of a foundation model that integrates both. We introduce OmiCLIP, a visual–transcriptomics foundation model trained on ST-bank, which includes diverse histopathology images and over 2.2 million paired transcriptomics from 113 studies (Fig. ST-bank covers 32 organ types, including conditions like health, cancer, heart failure and Alzheimer's disease (Fig. We applied a quality-control pipeline to retain ST data with high-resolution H&E images. As the batch effects may strongly affect the generalization ability of the model, the adopted rank-based strategies inspired by recent single-cell foundation models such as GeneFormer26 and scFoundation27 successfully eliminate batch effects through rank-based approaches rather than relying directly on raw read counts or normalized gene expression values. Specifically, we standardized text descriptions of the associated images by converting all Ensembl gene IDs to gene symbols and removing housekeeping genes. To format transcriptomics for language models, genes symbols were ranked from high to low by expression levels and structured into sentences for the text encoder (Fig. The image and transcriptomics modalities were aligned in a common representation space utilizing contrastive learning (Fig. In this dual-modality space, paired image and transcriptomic embedding vectors were optimized to be similar. To evaluate OmiCLIP's reliability to image quality variability across samples due to technological limitations, we simulated low-quality H&E images by adding Gaussian noise and compared the similarity scores between the paired transcriptomic and original image embeddings, with paired transcriptomic and simulated low-quality image embeddings, which were encoded by OmiCLIP's image and transcriptomic encoders. PLIP and OpenAI CLIP served as benchmarks (Extended Data Fig. 3a,b), and results demonstrated that OmiCLIP is robust to variations in image quality. For sequencing depth variability across technologies, we first analyzed the sequencing depth ranges in ST-bank and categorized samples into high, medium and low sequencing depth groups, identified as 11,792 unique molecular identifier (UMI) counts, 4,512 UMI counts and 615 UMI counts, respectively. Second, we generated low sequencing depth ST simulations using the downsampling function implemented in scuttle28. We compared similarity scores between paired images and original transcriptomic embeddings, with paired images and downsampled transcriptomic embeddings. These embeddings were encoded using OmiCLIP's image and transcriptomic encoders, using PLIP and OpenAI CLIP as benchmarks (Extended Data Fig. Results demonstrated OmiCLIP's robustness across sequencing depths, highlighting its adaptability to datasets generated across different technologies. The key advantage of contrastive-aligned visual–transcriptomics pretraining is its unique capability to drive the development of cross-modality tissue analysis tools. As a proof of concept, we developed Loki, a unified AI platform for multimodal analysis. In Loki, five modules were implemented, including Loki Align for multi-section tissue alignment, Loki Annotate for multimodal tissue annotation, Loki Decompose for cell-type decomposition from transcriptomics or histology, Loki Retrieve for histology image–transcriptomics retrieval and Loki PredEx for ST gene expression prediction from histology images (Fig. While these initial modules demonstrate its potential, Loki is designed to expand, supporting the development of more tools to further enhance multimodal tissue reconstruction and analysis. Loki could serve as the infrastructure that efficiently transfers transcriptomics such as scRNA-seq, bulk RNA-seq data and even marker genes into pathology image analysis via the pretrained model (OmiCLIP) (Fig. 1d), streamlining workflows, accelerating analysis and minimizing sequencing cost in research areas such as three-dimensional (3D) tissue studies and pathology diagnosis. OmiCLIP's image embeddings capture the morphology of tissues, while its transcriptomic embeddings represent genomic characteristics. Since OmiCLIP includes both transcriptomics and image encoders, here we evaluated whether contrastive learning enhances the ability of each encoder to represent tissue types better than the initial encoders. To assess clustering performance, we moved beyond qualitative visualizations and introduced quantitative metrics to assess the quality of the clustering. The uniform manifold approximation and projection (UMAP) visualizations showed that both embeddings clustered similar tissue types (Extended Data Fig. 2); however, the results were limited in their ability to quantify clustering quality and may have appeared unstable in some cases. Therefore, we computed the Calinski–Harabasz (CH) score29, a widely used clustering validation metric, which balances the dispersion between clusters with the cohesion within clusters (Methods). Higher CH scores reflect better clustering performance by indicating more distinct and internally consistent clusters. First, we calculated CH scores across 95 tissue samples from the ST-bank dataset, which included expert-annotated cell types from breast, healthy heart, kidney cancer and lung tissues and heart tissue with myocardial infarction (Supplementary Table 2). These annotated cell types served as ground-truth cluster labels. Our results showed a significant increase (P value < 0.001; Extended Data Fig. 1) in CH scores for embeddings after contrastive learning compared to before, demonstrating improved clustering performance. After contrastive learning, CH scores significantly increased in all organ types (P value < 0.05; Extended Data Fig. OmiCLIP's image embeddings also outperformed SOTA models like UNI7 and GigaPath30 by aligning image and transcriptomic data, not just image–image interactions. The results demonstrated OmiCLIP's ability to capture tissue heterogeneity. Researchers recently began investigating spatial biology in 3D, revealing new insights into tissue organization and cellular interactions. This requires tools to align multiple H&E images or ST sections, and even cross-align H&E images with ST slides. However, spatial distortions and biological variations between sections make alignment challenging. To address this, we developed the module Loki Align to align ST-to-ST data, H&E image-to-H&E image, and H&E image-to-ST data. Loki Align first embeds patch-level transcriptomics or H&E images into a 768-dimension space using OmiCLIP, and then applies the adapted coherent point drift (CPD) method31 to align two embeddings, preserving probability distribution and topology (Fig. We evaluated Loki Align on four datasets including two simulation datasets, a set of eight adjacent small intestine tissue sections, and a set of two adjacent ovarian carcinosarcoma sections. To ensure compatibility with datasets that may not be represented in the ST-bank, we used fine-tuning as a default setting for the Loki Align in the alignment tasks. Fine-tuning minimized contrastive loss between image embeddings and the paired text embeddings of the top-expressed gene name sentence (Methods). We further evaluated the zero-shot performance on an ovarian carcinosarcoma dataset. a, Schematic illustration of tissue alignment using ST and histology image with Loki Align. P values were calculated using a one-sided Wilcoxon test. We colored the samples using the top three PCA components of OmiCLIP transcriptomic embeddings, mapped to red, green and blue color channels, respectively. For visualization, we stacked the eight samples together along the perpendicular axis before and after different alignment methods, respectively, and visualized from the side view. The source2 that has no spatial variable gene selected by GPSA to run it is marked as ‘not applicable' (NA). Box plots show the comparison of tissue alignment performances on these seven source samples respectively and combined, represented by the PCC (and Kendall's tau coefficient in Supplementary Fig. 1) of highly variable gene expression between target and source samples after alignment at the same location, using Loki and baseline methods (PASTE, GPSA and CPD using PCA embeddings as input), respectively. We colored the samples as described in c. e, Alignment performance comparison using PCC and Kendall's tau coefficient of the highly expressed gene expression between the target sample and the source sample at aligned locations, using Loki (ST-to-ST and image-to-ST) and baseline methods PASTE (ST-to-ST), GPSA (ST-to-ST) and CAST (ST-to-ST), respectively. First, we simulated paired H&E images and ST data by perturbing gene expression and spatial locations with varying noise levels, covering diverse tissue types and disease types (Methods). We measured the distance between Loki-aligned data and the ground truth, and compared Loki Align with PASTE and GPSA, which are designed for ST section alignment32,33. At both high and low noise levels, Loki ST-to-ST alignment and Loki image-to-ST alignment ranked first and second, respectively, among the four methods (Fig. 2b), significantly outperforming PASTE and GPSA (P values < 0.001, Wilcoxon test). This superiority likely stems from PASTE's design for linear transformations, which maintains topological integrity but struggles with spatial warping32, while GPSA aims to map readouts to a common coordinate system, risking topological fidelity33. Second, we tested Loki Align on eight adjacent human small intestine tissues sections34. Real-world datasets often present challenges due to distortions such as rotation, tilt, uneven slicing and missing fragments. For better performance, we fine-tuned OmiCLIP using the target slide's H&E image and ST data. We aligned seven source ST datasets to target ST data and seven source H&E images to target ST data using Loki Align and applied PASTE and GPSA to align seven source ST datasets to target ST data. Loki Align successfully aligned all source sections to the target section. To evaluate the performance, we calculated the Pearson correlation coefficient (PCC) and Kendall's tau coefficient. For ST-to-ST scenarios, we compared the aligned ST data and the target ST data. The median PCC for Loki's image-to-ST and ST-to-ST alignment ranged from 0.67 to 0.80 and 0.62 to 0.83, respectively (Fig. The median Kendall's tau coefficient ranged from 0.16 to 0.27 for Loki's image-to-ST and 0.18 to 0.27 for ST-to-ST alignment (Supplementary Fig. On the vertical plane, Loki correctly aligned the same tissue types by image-to-ST and ST-to-ST alignment, while PASTE and GPSA twisted the tissues. PASTE rotated three source sections (sources 1–3; Fig. GPSA found common coordinates in six of the seven slices but introduced tremendous distortions, resulting in a PCC of 0.27 to 0.56 and Kendall's tau coefficient of 0.06 to 0.13. Overall, Loki ST-to-ST and image-to-ST alignments outperformed the SOTA methods. To isolate the contributions of OmiCLIP embeddings versus the superior registration method (CPD), we applied CPD to both OmiCLIP embeddings and transcriptomic embeddings that was reduced to two principal components using principal component analysis (PCA; Fig. OmiCLIP embeddings significantly improved the performance of alignment compared to PCA embeddings (P value < 0.001, Wilcoxon test). Third, we assessed Loki Align's performance on two adjacent human ovarian carcinosarcoma sections35 (Fig. The spatial expression patterns of representative genes are shown in Supplementary Fig. Fourth, we evaluated Loki Align on a human breast cancer dataset37 with paired 10x Visium and Xenium slides (Extended Data Fig. To perform the alignment, we first calculated transcriptomic embeddings for the Visium slide using gene sentences derived from Visium transcriptomic data. For the Xenium slide, we created pseudo-Visium data by averaging gene expression values across pseudo-spots. These pseudo-Visium data were then used to calculate transcriptomic embeddings via the transcriptomic encoder of OmiCLIP. The resulting distance between the aligned Xenium slide and the target Visium slide was 0.08 mm, demonstrating that Loki Align effectively aligns Visium and Xenium slides with high precision. Fifth, we evaluated the performance of three training strategies: pretraining plus fine-tuning, pure pretraining and pure training from scratch on ovarian carcinosarcoma samples (Supplementary Fig. The best performance was achieved with pretraining plus fine-tuning, resulting in a median PCC of 0.86 and a Kendall's tau coefficient of 0.17. Pure pretraining showed comparable performance, with a median PCC of 0.85 and a Kendall's tau coefficient of 0.18. In contrast, training from scratch exhibited the lowest performance, with a median PCC of 0.53 and a Kendall's tau coefficient of 0.06. Overall, we recommend fine-tuning as a default setting for Loki Align, as it ensures compatibility with datasets underrepresented in the ST-bank. Lastly, we examined whether Loki Align could leverage both modalities simultaneously for alignment over a single modality. To evaluate this, we integrated image embeddings and transcriptomic embeddings by averaging them and used the combined embeddings to align two adjacent ovarian carcinosarcoma samples. We then calculated the PCC and Kendall's tau coefficient for the image embeddings, transcriptomic embeddings and averaged embeddings to assess performance (Supplementary Fig. Altogether, by addressing spatial distortions and biological variability, Loki Align enables the accurate alignment of multiple H&E images and ST sections, thereby supporting advanced 3D reconstructions of tissue organization, particularly for cross-modality studies that combine H&E images and ST data. Next, we evaluated Loki's capability to analyze H&E images using bulk RNA-seq data, which is commonly used in both basic research and clinical practice. During OmiCLIP pretraining, the cosine similarities between paired ST and histology images were maximized, allowing the similarity between the H&E image of tissue patches and tissue-type-specific bulk RNA-seq data to indicate tissue-type enrichment. We developed Loki Annotate to annotate H&E images using tissue-type-specific bulk RNA-seq data as a reference. We used OmiCLIP to encode tissue patches from a WSI and the tissue-specific bulk RNA-seq data, then calculated the cosine similarity between the encoded embeddings (Fig. Higher similarity values indicate greater presence of the tissue type. a, Schematic illustration of tissue annotation using H&E image and reference bulk RNA-seq data from different sources, with OmiCLIP paired image and transcriptomic embeddings. Heat map shows the similarity of WSIs to the corresponding reference bulk RNA-seq of tumor, fibroblast and adipose, respectively. The color of the heat map reflects the similarities between WSIs and reference bulk RNA-seq data, with red indicating high similarity and blue indicating low similarity. In three breast cancer tissues, H&E regions corresponding to tumor tissue showed high similarity with the bulk RNA-seq data from tumor biopsies, which include tumor-related markers such as COL1A1 (ref. Similarity scores within the tumor regions were significantly higher than those outside (P value < 0.05, Wilcoxon test). Additionally, higher similarity scores were consistent with higher diagnostic values of tumors calculated by clustering-constrained-attention multiple-instance learning40 (CLAM, a SOTA WSI tumor analysis model; Fig. Next, we tested the similarity between H&E images of heart failure tissues and fibroblast RNA-seq data, as well as between H&E images of normal breast tissues and adipose RNA-seq data. In summary, Loki Annotate effectively annotates H&E images by using tissue-type-specific bulk RNA-seq data as a reference. When bulk RNA-seq is unavailable, Loki Annotate can also annotate tissues using predefined marker genes, similar to the workflow of using bulk RNA-seq data without fine-tuning. We created tissue-specific gene lists using well-established markers, such as ‘TP53, EPCAM, KRAS, …, DSP' for tumors (Fig. As with the bulk RNA-seq approach, we used OmiCLIP to encode tissue patches from histology images and the gene name sentence composed from the marker gene list. Tissue-type annotation was determined by cosine similarity derived from the dot product of normalized text embeddings and H&E image embeddings, with the highest cosine similarity score assigned as the predicted tissue to the query image. Based on these annotations, precision was defined as the proportion of correctly predicted tissues (true positives) of all predicted tissues, while recall was defined as the proportion of correctly predicted tissues of all actual tissues. The F1 score was calculated as the harmonic mean of precision and recall, which was used to measure classification performance. We measured annotation performance using F1 score and compared our results to the OpenAI CLIP model. Our analysis showed that Loki consistently outperformed OpenAI CLIP across all four datasets (Fig. The F1 scores of Loki ranged from 0.59 to 0.96, while the F1 scores of OpenAI CLIP ranged from 0.03 to 0.34 (Fig. a, Schematic illustration of tissue annotation using H&E image and reference marker genes. For Loki, we used the text content of marker gene symbols of each tissue type. For the PLIP model, we used the text content of natural language description of each tissue type. b, Examples of similarity scores of images and texts calculated by Loki and OpenAI CLIP model, respectively. c, Comparison of zero-shot performances, represented by weighted F1 scores, across four datasets using Loki and OpenAI CLIP, respectively. d, Comparison of zero-shot performances, represented by weighted F1 scores, across four datasets using Loki, PLIP and incorporating Loki and PLIP models by average similarity (shown in a; Methods), respectively. e, Comparison of zero-shot performances, represented by weighted F1 scores of each tissue type in the CRC7K dataset using OpenAI CLIP model, Loki, PLIP model and incorporating Loki and PLIP models, respectively. f, Confusion matrix of the CRC7K dataset using Loki (left), PLIP model (middle) and incorporating Loki and PLIP models (right), respectively. The ground-truth labels are presented in rows and the predicted labels are presented in columns. ADI, adipose tissue; NOR, normal colon mucosa; TUM, colorectal carcinoma epithelium; LYM, lymphocytes; MUC, mucus; DEB, debris; MUS, smooth muscle; STR, cancer-associated stroma. Several studies have developed visual–language foundation models using paired histopathology images and captions11,12. Given that transcriptomics and natural language provide complementary information, we investigated whether their combination could improve annotation performance without additional training. We applied PLIP, a visual–language foundation model for pathology image analysis, to annotate the tissue images by descriptive prompts, such as converting ‘tumor' to ‘an H&E image patch of colorectal adenocarcinoma epithelium' in the CRC7K dataset. We then combined Loki and PLIP by averaging their similarity scores of an H&E image and a given tissue type (Fig. 4a and Methods), resulting in the best performance across all four benchmark datasets (Fig. In CRC7K, PLIP misclassified 63% of colorectal adenocarcinoma epithelium images as cancer-associated stroma, while Loki misclassified 15% of tumor images as normal colon mucosa. Notably, combining Loki and PLIP achieved a 93% recall rate, demonstrating that combining transcriptomic and natural language enhances overall performance compared to each modality alone (Fig. Since OmiCLIP can project the Visium ST data and H&E images to a shared embedding space, we developed Loki Decompose, a feature to decompose cell types in both ST data and H&E images, using scRNA-seq as a reference. Inspired by ST decomposition models like Tangram and CytoSPACE45,46, we used OmiCLIP to encode the patches (the same size as a Visium spot) of an H&E image and scRNA-seq transcriptomic profile into this embedding space. As an application of Tangram with OmiCLIP embeddings instead of gene expression data, Loki Decompose applied Tangram's nonconvex optimization algorithm47 to deconvolute the OmiCLIP embeddings of an H&E image patch or the embeddings of a Visium spot's transcriptomic profile rather than raw gene expression data, providing the cell-type composition of an image patch or a Visium spot (Fig. We assessed Loki Decompose on our in-house triple-negative breast cancer (TNBC) dataset, a human colorectal cancer dataset48 and a brain dataset49,50. a, Schematic illustration of tissue alignment using ST, reference scRNA-seq data and histology images with OmiCLIP paired transcriptomic and image embeddings after fine-tuning. b, H&E image of our in-house TNBC sample, characterized by Xenium into three major cell types: cancer epithelial, immune and stromal cells. c, Performance comparison of 12 decomposition methods using JS divergence, SSIM and impact scores. z-scores of JS divergence (or SSIM) across methods were calculated based on the average JS divergence (or SSIM) among cell types. The green color indicates decomposition tools. The blue color indicates the performance of replacing OmiCLIP embeddings with other transcriptomic foundation models' embeddings. d, Cell-type decomposition results on three major cell types of the TNBC sample using the image by Loki and using ST by Tangram, with Xenium data as ground truth. e, H&E image of the human colorectal cancer sample and cell-type distribution within the Visium-HD capture area. f, Bar plot shows the accuracy of decomposition on four major cell types by Loki using ST or image mode, and by Tangram using ST. Error bars indicate the standard deviation and the center values represent the mean. For both JS divergence and SSIM, adjusted P value > 0.1 using a two-sided Wilcoxon test. Different tissue regions are annotated by the pathologist as ground truth. Heat map shows the cell-type distribution of fibroblast, tumor, intestinal epithelial, smooth muscle and immune/inflammatory cells, with color reflecting the density of each cell type. h, Cell-type decomposition results on the brain sample. Left, brain anatomic references with zoom-in H&E image patches of L1 (VLMCs, astrocytes), L2/3, L4/5, L6 and white matter (WM; oligodendrocytes), respectively. Right, heat map shows the cell-type distribution of VLMCs, astrocytes, L2/3, L4/5, L6 and oligodendrocytes, with color reflecting the distribution of each cell type. First, we performed a Xenium experiment on the in-house TNBC sample and captured paired H&E images. We generated pseudo-Visium data from the Xenium data as a benchmark for evaluating Loki Decompose, using publicly available scRNA-seq data as a ref. We used Loki to decompose pseudo-Visium spots and H&E images, using paired sequencing and image data from one-fourth of a WSI for fine-tuning followed by cross-validation (Methods). Decomposition accuracy was evaluated using Jensen–Shannon (JS) divergence and the structural similarity index measure (SSIM). Since JS divergence and SSIM operate on different scales, we standardized their values by calculating z-scores among different methods (details in Methods). The z-score for JS divergence was inverted (that is, multiplied by −1), as lower values indicate better performance. Finally, we averaged the z-scores of JS divergence and SSIM to calculate an overall impact score, which provides a unified metric for comparison across methods. Loki Decompose in ST mode and image mode ranked as the top two methods with impact scores of 1.32 and 1.11, respectively, outperforming other SOTA methods52 including Tangram, Spatial Seurat53, CARD54, CytoSPACE, Cell2location50, SpatialDWLS55 and RCTD56, with impact scores ranging from 0.87 to −1.82 (Fig. As single-cell foundation models such as GeneFormer26, scGPT57 and scFoundation27 can also provide the transcriptomic embeddings, to further evaluate the approach, we replaced OmiCLIP gene expression embeddings with those from single-cell foundation models GeneFormer, scGPT and scFoundation. Second, we evaluated Loki Decompose using pseudo-Visium data generated from whole-genome sequencing Visium-HD data of human colorectal cancer as a benchmark (Fig. We fine-tuned OmiCLIP on regions with paired sequencing and image data (Methods). Loki Decompose successfully predicted the spatial distribution of key cell types (Extended Data Fig. We developed a technique inspired by non-maximum suppression (NMS)58 to refine spatial probabilistic maps, enhancing decomposition performance by reducing ambiguity in complex spatial scenarios and focusing predictions on the most confident cell-type assignments. Using JS divergence and SSIM scores, Loki Decompose based on either the ST data or the H&E images was comparable to Tangram, which used gene expression as input (Fig. 5g), segmenting it into image patches matching Visium spot size. Similarly, we used OmiCLIP to encode image patches and transcriptomics of scRNA-seq and then decomposed those using scRNA-seq data. Loki Decompose accurately predicted densities of tumor, fibroblast, intestinal epithelial, smooth muscle, immune and inflammatory cells, aligning closely with pathology annotations (Fig. Additionally, our predicted tumor cell density matched that of CLAM40, further validating Loki Decompose's robustness (Fig. Fourth, to test Loki Decompose in a more challenging scenario, we applied it to a brain tissue, where neurons share similar morphology. VLMCs and astrocytes are concentrated near the cortical surface and pial borders (for example, layer 1), while oligodendrocytes are more prevalent in deeper layers and within white matter tracts49. To decompose the mouse brain cortex slice, we applied a workflow similar to the one for other decomposition tasks. First, we fine-tuned OmiCLIP using adjacent Visium data and H&E images, then segmented the WSI into patches, corresponding to Visium spot size. Finally, Loki Decompose was applied to predict cell-type distributions within the brain cortex H&E image. Loki Decompose accurately predicted the distribution of VLMCs, astrocytes, neurons from L2/3, L4/5 and L6 and oligodendrocytes, aligning closely with brain anatomic ref. Lastly, we tested the performance of decomposition using three training strategies: pretraining plus fine-tuning, pure pretraining and pure training from scratch on TNBC samples (Extended Data Fig. The analysis showed that pretraining plus fine-tuning had the best performance, achieving a mean SSIM score of 0.30 and a mean JS divergence of 0.40. In contrast, pure pretraining resulted in a mean SSIM score of 0.13 and a mean JS divergence of 0.43, while pure training from scratch performed the worst, with a mean SSIM score of 0.00070 and a mean JS divergence of 0.44. Although pure pretraining achieved a comparable JS divergence score to the pretraining plus fine-tuning method (0.43 versus 0.40), it showed a notable decline in the SSIM (0.13 versus 0.30), underscoring the importance of fine-tuning for optimal performance. Therefore, we strongly recommend fine-tuning the model for this task to achieve optimal results. Altogether, Loki Decompose effectively inferred cell-type fractions from H&E images and ST data, demonstrating its potential to enhance spatial tissue analysis by utilizing H&E images to reduce experimental costs and processing time, particularly in multi-section tissue studies. Leveraging such ability of OmiCLIP, we developed Loki Retrieve to identify and retrieve transcriptomics data corresponding to a given H&E image. Using OmiCLIP's image encoder, query images were encoded to embeddings to retrieve the most similar transcriptomic entries from the ST-bank dataset in the aligned latent space (Fig. Then, we systematically evaluated our model on diverse datasets including four independent histopathology datasets of colorectal cancer, lung cancer and lymph node metastasis, along with eight in-house tissues of heart failure, Alzheimer's disease and breast cancer human tissues (Supplementary Fig. Because ground-truth transcriptomics data were unavailable, retrieval accuracy was assessed by measuring similarity between the query image and the retrieved transcriptomics-paired images. Overall, Loki Retrieve significantly outperformed OpenAI CLIP and PLIP by a large margin (Fig. 6c,d; P value < 0.05), achieving median similarity scores ranging from 0.7 to 0.9. a, Schematic illustration of image-to-transcriptomics retrieval on the ST-bank dataset. For each example image from adipose tissue, colorectal adenocarcinoma epithelium, lymphocytes, smooth muscle and normal colon mucosa, the retrieved top 50 most similar transcriptomics are shown by the paired image from the ST-bank dataset. c, Image-to-transcriptomics retrieval similarity scores across the four validation datasets—CRC7K, WSSS4LUAD, LC25000 and PatchCamelyon—using Loki, OpenAI CLIP and PLIP. d, Image-to-transcriptomics retrieval similarity scores across the eight in-house human tissues: heart failure (HF), Alzheimer's disease (AD), metaplastic breast cancer (MPBC) and TNBC, using Loki, OpenAI CLIP and PLIP. e, Image-to-transcriptomics retrieval evaluation across four validation datasets and one test dataset using Loki, OpenAI CLIP and PLIP, with random baseline. The top-K quantile most similar transcriptomics were retrieved. We further evaluated image-to-transcriptomics retrieval performance by calculating the rank of the correct pair using Recall@K (5% and 10%). We used four reserved samples from ST-bank as validation datasets including brain, heart, kidney and breast tissue samples and four independent ST studies as a test dataset, including desmoplastic small round cell tumor, colorectal cancer, vascular and colon samples (Supplementary Table 4). Results demonstrated that Loki notably outperformed both OpenAI CLIP and PLIP across all validation datasets. Specifically, Loki achieved Recall@5% of 0.125 and Recall@10% of 0.227 for brain (average 2.3-fold higher than OpenAI CLIP and 2.5-fold higher than PLIP), Recall@5% of 0.186 and Recall@10% of 0.291 for heart (average 3.2-fold higher than OpenAI CLIP and 3.1-fold higher than PLIP), Recall@5% of 0.173 and Recall@10% of 0.297 for kidney (average 3.2-fold higher than OpenAI CLIP and PLIP) and Recall@5% of 0.140 and Recall@10% of 0.240 for breast (average 2.6-fold higher than OpenAI CLIP and PLIP; Fig. On the test dataset, Loki further demonstrated substantial improvements, achieving Recall@5% of 0.117 and Recall@10% of 0.208 (average 3.1-fold higher than OpenAI CLIP and 3.0-fold higher than PLIP; Fig. Together, these results confirm Loki's superior performance in accurately retrieving paired transcriptomic information from images. Building on the success of Loki Align, Annotate and Decompose in analyzing tissue across the H&E image and transcriptomics data, we developed Loki PredEx to predict gene expression for image patches. Loki PredEx computes a weighted sum of gene expression from reference ST spots where weights are determined by similarity scores between the query image and ST data, both encoded by OmiCLIP (Supplementary Fig. Several studies have explored predicting gene expression from H&E images using AI models59,60,61,62. We compared Loki PredEx with them on a normal human heart dataset comprising 39 samples. Loki accurately predicted highly variable gene expression, as demonstrated by the spatial distribution of the predicted gene expression (Extended Data Fig. To evaluate the performance, we used mean squared error (MSE) and PCC as two metrics. Loki PredEx demonstrated superior performance, achieving the best results based on MSE scores in 28 of 39 cases, and ranking as the best in 16 of 39 samples based on PCC compared to Hist2ST, HisToGene, BLEEP and mclSTExp (Extended Data Fig. These results showed the robustness of OmiCLIP in predicting ST data across diverse datasets (Extended Data Fig. A major limitation of deep learning models like HisToGene is their heavy hardware requirements. Models like HisToGene and Hist2ST were optimized for smaller legacy ST datasets, with fewer spots. For instance, HisToGene is typically trained on less than 7,000 spot–image pairs. However, with modern ST technologies such as Visium, slides contain over 4,000 spots, pushing memory demands above 300 GB and complicating GPU-based training. In our experiments, training HisToGene on over 80,000 spots from 35 tissues required 4 h on 16 2.60 GHz Intel Xeon Gold 6348 CPUs for 100 epochs and Hist2ST took 31 h under similar conditions. Loki PredEx avoids these resource-intensive training needs, providing a more efficient alternative. Together, Loki PredEx delivers accurate ST gene expression predictions, and avoids these resource-intensive training needs, providing a more efficient alternative based upon the use of pretrained weights, highlighting its potential as a scalable infrastructure. Existing dual-modality foundation models in computational biology11,12 primarily combine images with textual descriptions, proving their utility in histopathology annotation and analysis. However, the natural language descriptions lack molecular insights for disease characterization. Our study first suggests that publicly available ST datasets provide sufficient volume and diversity to pretrain a foundation model bridging tissue morphology with genomics. The success of the development of our foundation model could represent a substantial step toward understanding molecular mechanisms regulating tissue phenotypes in health and disease. We presented OmiCLIP, a high-performance histopathology image–omics foundation model by contrastive learning. Unlike visual–language foundation models, OmiCLIP integrates molecular insights with pathology images, complementing language descriptions. Benchmark results indicate that OmiCLIP performs comparably to, and in some cases surpasses visual–language foundation models in tissue annotation, suggesting that marker genes could serve as effective tissue labels independent of language. Notably, our annotation of tissue types incorporating both language description and marker genes shows promise for triple-modal foundation modeling of image, transcriptomics and language. Using marker genes as a label could potentially facilitate molecular investigation-related studies such as drug repurposing, immune response prediction and disease mechanism discovery. A key question is whether OmiCLIP's transcriptomic encoder generalizes to other sequencing techniques like bulk RNA-seq and scRNA-seq. We evaluated the information of transcriptomic embeddings by cell annotation of scRNA-seq data (Supplementary Note 1) and tumor classification of bulk RNA-seq data (Supplementary Note 2). Our results show that OmiCLIP's transcriptomic embeddings efficiently cluster participants with cancer without specific training and accurately annotate cell types with even 1% of labeled cells. Loki could potentially enhance 3D tissue analysis by integrating imaging and molecular modalities in a scalable and efficient manner. Emerging 3D histology and omics techniques already show promise in improving diagnostic accuracy by preserving native 3D tissue morphology, leading to better prognostic predictions and ultimately improved patient care63,64,65,66. However, challenges remain in spatial distortions and aligning molecular data across different modalities. Loki addresses these by aligning tissue slices and integrating ST, histology and scRNA-seq data, enabling a more comprehensive understanding of tissue architecture and cellular interactions, which is crucial for 3D tissue analysis. Incorporating Loki into workflows facilitates detailed molecular and spatial features analysis across tissue sections, supporting automated, scalable and high-resolution 3D tissue analysis. Loki provides an AI-powered platform supporting the expansion of additional tools in a unified framework. Among the existing modules, Loki Annotate automates annotation and interpretation of molecular and spatial tissue features using associated or external RNA-seq data or marker genes. Loki PredEx predicts spatial gene expression from histology images, reducing reliance on costly and laborious ST experiments. These modules, leveraging contrastively aligned embeddings, enable efficient multimodal tissue reconstruction and analysis, providing a scalable solution to the growing demand for high-resolution tissue studies. Loki's ability to integrate diverse data types across tissue sections minimizes cost and complexity while accelerating workflows in enabling deeper insights into biological systems. Compared to billion-scale datasets for developing visual–language models in the general machine-learning domain, the major limitation of this study is pretraining data size. We expect that continued use of training datasets may further improve the zero-shot performance. However, several biomedical multimodal foundation models were efficiently trained on million-scale datasets by removing duplicates and noise11,12,67, a strategy we used to optimize training efficiency. Notably, as a contrastive learning framework, OmiCLIP is not generative and cannot directly generate the accurate transcriptomic profile of the query image. Instead, it retrieves tissues with the most similar transcriptomic profiles to the query tissue. While it effectively embeds transcriptomic and histology data at the patch level, it does not inherently generate new data, such as reconstructing a WSI with gene expression patterns. However, OmiCLIP's patch-level embeddings could support generative approaches, such as diffusion models, to reconstruct WSIs with ST details. Future studies could refine the transcriptomic encoder using RNA-seq datasets like scRNA-seq and bulk RNA-seq data. Although ST-bank includes 32 organ types, rare conditions may be underrepresented. We suggest fine-tuning alignment and decomposition tasks to ensure compatibility with datasets that are not covered in ST-bank (Extended Data Fig. Unlike single-cell foundation models like scGPT57, Geneformer26 and scFoundation27, our approach models omics data as text, effectively bridging molecular and visual modalities. Representing gene expression data as text leverages natural language processing models to embed biological information into a high-dimensional space, offering several advantages over using gene expression values directly. First, text embeddings integrate omics data with various biological entities such as pathways, functional annotations68 and cell types69, extending the model's capabilities beyond tissue alignment and decomposition, making it adaptable to a broader range of biological tasks. Second, this approach aligns with other multimodal foundation models, and allows incorporation of proteomics, metabolomics and DAPI images into the same unified space. In contrast, raw gene expression values lack flexibility for such integrations and require additional preprocessing. Third, text-based foundation models trained on billions of tokens provided robust text embeddings, like GenePT23, demonstrating that gene embeddings from textual descriptions can match or surpass models trained on extensive gene expression datasets. This supports our approach of utilizing text-based embeddings to capture rich biological information efficiently. While integrating two modalities enhances information capture, it may also introduce noise or misalignment, potentially overshadowing benefits. If one modality dominates, performance gains from dual-modality fusion may be minimal. Loki Decompose is valuable in scenarios where sequencing costs limit transcriptomic profiling. By estimating cell-type proportions from images, researchers can preselect, screen or perform batch processing of samples cost-effectively for exploratory studies and large-scale screenings. Loki Retrieve utilizes curated reference images for ground-truth comparisons, aiding validation and interpretation, especially when training data for prediction models like Loki PredEx are scarce. Together, our approach contributes to a unified, scalable framework for multimodal analysis. To conclude, we created ST-bank, a dataset of over 2 million pathology-specific image–transcriptomics pairs. We developed OmiCLIP to integrate these data, forming a visual–omics foundation model. Leveraging OmiCLIP, we built Loki, an infrastructure enabling multimodal analysis for tissue alignment, tissue annotation, cell-type decomposition, histology image–transcriptomics retrieval and ST gene expression prediction. These capabilities represent a fundamental step toward bridging and applying foundation models in genomics for histopathology. We curated a large dataset of histopathology image–transcriptomics pairs using publicly available 10x Visium datasets (Supplementary Table 1). H&E images were cropped to match ST spot sizes, and text sentences were generated by combining the top 50 expressed genes per spot into sentences. For example, the top-expressed genes in one spot, for example, SNAP25, ENO2, CKB, GRIN2C and CAMK4, will be combined into a sentence: ‘SNAP25 ENO2 CKB GRIN2C CAMK4 … MTOR VPS13D'. Data preprocessing involved removing duplicates and excluding low-resolution H&E images (<2,000 × 2,000 pixels), and normalizing raw count matrices following standard protocols using Seurat70 and Scanpy71. For datasets in transcripts per million or fragments per kilobase of transcript per million fragments mapped formats, which cannot be normalized to standard gene expression profiles, were retained unchanged. Quality control was applied to filter out contaminated, extremely low-quality or damaged cells, retaining only those with over 200 expressed genes. Housekeeping genes were removed to ensure a more biologically relevant analysis. These steps resulted in ST-bank, a pathology-specific image–transcriptomics caption dataset of 2,185,571 pairs. We simulated new ST experiments by perturbing both gene expression and spatial locations at different levels of noise, generating 10 simulated datasets per real dataset, totaling 200 datasets (100 low-noise, 100 high-noise). Real-world data tests used a normal human small intestine Visium dataset34 of eight adjacent tissue slices, a human ovarian carcinosarcoma Visium dataset35 of two adjacent tissue slices and a human breast cancer Visium and Xenium dataset37. Bulk RNA-seq data-based annotation used three normal human breast and three human heart failure histology images79,80 and three breast cancer histology images from TCGA. Pathology experts annotated different tissue regions. Bulk RNA-seq datasets including 663 human adipose and 504 fibroblast samples from the Genotype-Tissue Expression Portal and three paired tumor biopsy samples from TCGA. We downloaded a human colorectal cancer dataset48 to create pseudo-Visium spots in the Visium-HD capture area. Pathology experts annotated different tissue regions. We collected an in-house TNBC patient-derived xenograft for processing on Xenium slides, to create pseudo-Visium spots with an external scRNA-seq reference of TNBC51 for decomposition. We also downloaded a mouse brain Visium dataset50 and a scRNA-seq dataset49 from the Allen Institute. We collected our in-house heart failure patient tissue, paraffin-embedded Alzheimer's disease patient tissue, and metaplastic breast cancer and TNBC patient-derived xenografts. The validation datasets included brain, heart, kidney and breast samples, and the test dataset included desmoplastic small round cell tumor, colorectal cancer, vascular and colon samples (Supplementary Table 4). We used a normal human heart sample dataset81 of paired ST data and H&E images including 39 samples. The image encoder was based on a standard vision transformer (ViT)82 with an input image size of 224 × 224 pixels. The text encoder was based on a causal masking transformer with input text length of 76 tokens. Regarding the initial embeddings of ST data, the initial text encoder was not trained from scratch but on LAION-5B83, including biological literature, which may explain its tendency to cluster similar tissue patches. The model was trained for 20 epochs, using one NVIDIA A100 80-GB GPU with a local batch size of 64. The output vectors of the image and text encoders with dimensions of 768 were optimized by minimizing the contrastive loss on a given batch. All experiments were run in Python v.3.9. Detailed software versions are: CUDA v.12.2; torch v.2.3.1; torchvision v.0.18.1; scipy v.1.13.1; pillow v.10.4.0; scikit-learn v.1.5.2; pandas v.2.2.3; numpy v.1.25.0; and scanpy v.1.10.3. To improve performance on downstream tasks, OmiCLIP allows fine-tuning with user datasets. The fine-tuning dataset is created by preprocessing Visium data using a standard 10x Space Ranger pipeline and generating gene name sentences as describe in ‘Training dataset curation', ensuring compatibility with the pretraining dataset format. Fine-tuning is done using contrastive loss9 between image embeddings and paired text embeddings of the top-expressed gene sentences. The contrastive loss is calculated according to equation (1): where \({x}_{i}\) and \({y}_{j}\) denote the normalized image and text embeddings, respectively. The pretrained model was fine-tuned for ten epochs for the tissue alignment task and five epochs for the cell-type decomposition task, using a local batch size of 64, minimizing the contrastive loss. We first fine-tuned OmiCLIP using paired ST data and H&E image of the target sample. The fine-tuned OmiCLIP text encoder and image encoder then encoded ST data and image, respectively. We used a nonrigid point set registration algorithm based on the CPD method31, which iteratively aligns two point sets by minimizing the statistical discrepancies. Where \(D\) is the point's dimensionality, \(M,N\) are the number of points in each set, and \(x,y\) are the source and target points in sets \(X\) and \(Y\), respectively. Point sets are modeled as Gaussian mixture samples, with correspondence probability matrix \(G\) computed as shown in equation (3): This forms the basis for expectation–maximization steps, which iterate until convergence. During the E-step, posterior probabilities \(P\) of correspondences update as given by equation (4): In the M-step, \(W\) updates according to equations (5)–(7): where the transformation weights \(W\) are constrained to \(0\le W\le 1\). We optimized CPD by adding the first two principal components of embeddings generated by OmiCLIP image encoder or text encoder, along with the original two-dimensional coordinates. We further calculated the homography matrix with translation and rotation between spots before and after alignment to avoid tremendous distortion. For PASTE, GPSA and CAST, we used their default configuration for tissue preparation and alignment in Visium data. OmiCLIP enables zero-shot annotation by learning an aligned latent space for image and transcriptomic embeddings, eliminating the need for retraining. We used OmiCLIP text encoder to encode bulk RNA-seq data and image encoder for H&E images, then calculated cosine similarity between transcriptomic and image embeddings at spot level. Annotation was determined by selecting candidate texts with the highest similarity score to image query. We evaluate this using four datasets: CRC7K, LC25000, PatchCamelyon and WSSS4LUAD. For Loki, text candidates were generated according to marker genes of each tissue type (Supplementary Table 3). For the PLIP model, text candidates were generated from tissue-type descriptions (Supplementary Table 3). The OmiCLIP image encoder encoded images resized to 20 × 20 pixels, consistent with its pretraining. For jointly using Loki and PLIP, we summed their normalized similarity scores. Let, \({s}_{{Loki}}\left(I,T\;\right)\) and \({s}_{{PLIP}}\left(I,T\;\right)\) represent the similarity scores between an image \(I\) and text \(T\) computed by Loki and PLIP, respectively. Normalized scores were obtained according to equations (8)–(10): The candidate text \({T}^{* }\) with the highest combined similarity score was identified as given by equation (11): To decompose human colorectal cancer slices, we fine-tuned OmiCLIP using paired Visium ST data and H&E images. We then used fine-tuned OmiCLIP text encoder to encode scRNA-seq data and pseudo-Visium ST data, and image encoder to encode H&E images. For in-house TNBC human samples, we fine-tuned OmiCLIP using a quarter of a region (top-right, top-left, bottom-right or bottom-left) of pseudo-Visium ST data and H&E images, then encoded scRNA-seq data and ST data via the text encoder and H&E images via the image encoder. Similarly, for mouse brain cortex slices, we fine-tuned OmiCLIP using adjacent Visium ST data and H&E images, then encoded scRNA-seq data and H&E images accordingly. We used a nonconvex optimization algorithm implemented by Tangram to co-register OmiCLIP embeddings of scRNA-seq data with those of ST data or H&E images. We aimed to obtain a probabilistic mapping matrix \(M\) aligning single cells to specific spots based on embedding similarities between scRNA-seq and ST data or scRNA-seq and H&E images. The mapping matrix \(M\) of dimensions spots-by-cells quantifies the likelihood that a given single cell is located within a particular spot. The scRNA-seq data matrix \(S\) is structured as cells-by-embeddings, while the ST data or H&E image matrix \(G\) is formatted as spots-by-embeddings. The optimal mapping matrix \(M\) is derived by minimizing the loss function \(L\left(S,M\right)\) as shown in equation (12): Here, \({\cos }_{{\rm{distance}}}\) denotes the cosine distance between OmiCLIP embeddings of the mapped single cells and those of ST data or H&E images. For Tangram, we used a uniform density prior for each spot without target count, aligning with Loki Decompose. The same settings were used for Loki, while Spatial Seurat, CARD, CytoSPACE, RCTD, Cell2location and spatialDWLS utilized their default configurations and tissue preparation and decomposition functions. For scGPT, scFoundation and GeneFormer, we used default configuration and tissue preparation functions before using the Tangram method with same default configurations to decompose cell types. To evaluate their performance, we used cell-type information from Xenium, Visium-HD and pathology annotation as ground truth. To improve decomposition performance in regions with complex cellular heterogeneity, we developed a refinement strategy inspired by NMS58. This method prioritizes the most probable cell type within each spot, reducing overlapping or ambiguous assignments when multiple cell types have comparable probabilities. This refined method is recommended in complex spatial scenarios, such as colorectal cancer. For \(N\) total spots (indexed by \(i=1,\ldots ,N\)), and \(C\) cell types, we defined \({P}_{i,c}\) as the original probability of cell type \(c\) at spot i. The NMS-based refinement follows two steps: selecting the highest probability cell type and suppressing others. The most likely cell type at each spot i was determined as given by equation (13): Then refined probabilities \({P}_{i,c}^{({\rm{NMS}})}\) was defined according to equation (14): This NMS-based refinement ensured that only the cell type with the highest likelihood remained at each spot, eliminating competing probabilities and improving spatial decomposition accuracy. Similarly to Loki Annotate, the retrieval results were decided by choosing candidate transcriptomics with the highest similarity score to the image query. Here, as shown in equation (15), \(K\) indicates the set of all pairs, \({I}_{q}\) indicates the image embeddings of a given query, \({I}_{k}\) indicates the transcriptomics embeddings and \({k}^{* }\) indicates the candidate transcriptomics with the highest similarity score. We applied 10-fold cross-validation to evaluate Loki PredEx's performance. In each fold, OmiCLIP was fine-tuned on the training set for ten epochs, and then we used the fine-tuned OmiCLIP text encoder to encode the ST data of training sets and the image encoder to encode the image of validation sets. For each spot in the validation set, cosine similarity between its image embeddings and all the transcriptomic embeddings in the training set was computed, and these weights were used to generate ST gene expression prediction for validation set spots via a weighted average as given by equation (16): where \(T\) is the set of all spots in the training set, Xi is the predicted gene expression for validation spot i, wi,j is the similarity score between validation spot i and training spot j, and Xj is the gene expression for training spot \(j\). To benchmark performance, we compared Loki PredEx against HisToGene, Hist2ST, BLEEP and mclSTExp, on the same dataset. In each fold, the top 300 expressed genes in the validation set were selected for prediction. By applying the same cross-validation procedure and evaluating the top 300 expressed genes in each fold, we ensured a fair comparison between Loki PredEx and baseline models. In ‘OmiCLIP improves image and transcriptomics representations', we used the Leiden algorithm in Scanpy71 to cluster ST with default parameters including a resolution of 1 and a sparse adjacency matrix derived from neighbor connectivity. We then calculated the UMAP embeddings with an effective minimum distance of 0.5 and three dimensions. The CH score, also referred to as the variance ratio criterion, was used to evaluate clustering quality for a given dataset by comparing between-cluster dispersion and within-cluster dispersion. It was computed using two sets of ground truth, a benchmarked dataset containing 95 tissue samples from the ST-bank, which included expert-annotated cell types (Supplementary Table 2) and the Leiden clustering (described above) labels for samples without cell-type annotations. For a dataset with \(n\) points \(\left\{{x}_{1},\ldots ,{x}_{n}\right\}\) divided into \(k\) clusters \(\left\{{C}_{1},\ldots ,{C}_{k}\right\}\), CH score is the ratio normalized by the number of degrees of freedom for between-cluster and within-cluster dispersions, respectively, as given by equation (17): Between-cluster sum of squares (BCSS) is calculated as the weighted sum of squared Euclidean distances from each cluster's centroid to overall centroid, as given by equation (18): Within-cluster sum of squares (WCSS) measures the cohesion of the clusters with smaller values indicating tighter clustering and is the total squared Euclidean distances from each data point to its cluster centroid, as given by equation (19): The PCC, which ranges from −1 to 1, assessed tissue alignment and gene expression prediction. Given paired data \(\left\{\left({x}_{1},{y}_{1}\right),\ldots ,\left({x}_{n},{y}_{n}\right)\right\}\) consisting of n pairs, PCC represented by \({r}_{{xy}}\) is defined in equation (20): Kendall's tau coefficient, which ranges from −1 to 1, assessed tissue alignment, as given by equation (21): where \(P\) denotes the number of concordant pairs, \(Q\) is the number of discordant pairs, while \(T\) and \(U\) represent ties occurring solely in \(x\) or solely in \(y\), respectively. JS divergence, which ranges from 0 to 1, assessed cell-type decomposition. To calculate JS divergence between two probability distributions \(P\) and \(Q\), we first computed the pointwise average distribution, as given by equation (22): KL divergence is a measure of how one probability distribution diverges from a second distribution, as given by equation (23): JS divergence is the average of these two KL divergences as given by equation (24): The SSIM, which ranges from −1 to 1, assessed cell-type decomposition, where we considered the cell-type distribution in spatial as image. For two images \(x\) and \(y\), as shown in equation (25): where \({u}_{x}\) and \({u}_{y}\) are the mean intensities of images \(x\) and \(y\), \({\sigma }_{x}^{2}\) and \({\sigma }_{y}^{2}\) are the variances of \(x\) and \(y\), \({\sigma }_{{xy}}\) is the covariance between \(x\) and \(y\), \({C}_{1}\) and \({C}_{2}\) are small constants to stabilize the division when the denominators are close to zero. MSE assessed ST gene expression prediction by comparing the Euclidean distance of the highly expressed gene expression between ground truth and prediction for each method within the same location. For each decomposition method \(m\), we computed the mean JS divergence, \({\rm{JS}}_{m}\), and the mean SSIM, \({\rm{{SSIM}}}_{m}\), across all cell types as given by equations (26) and (27): where \({p}_{c}\) and \({q}_{c}\) represent the ground truth and predicted proportions, respectively. \(N\) represents the total number of cell types. We standardized SSIM and JS divergence across methods to enable direct comparison, as they operate on different scales. The standardized metrics \({Z}_{{\rm{SSIM}}_{m}}\) and \({Z}_{{{JS}}_{m}}\) are calculated according to equation (28): Because lower JS divergence indicates better performance, we inverted the standardized JS divergence values by multiplying them by −1, as given by equation (29): To generate a unified metric for decomposition accuracy, we averaged the inverted JS divergence z-scores and the SSIM z-scores for each method as given by equation (30): F1 score, which ranges from 0 to 1, assessed zero-shot and linear probing methods as given by equation (31): A higher F1 score indicates better overall performance in classification tasks. The weighted F1 score was calculated by averaging the F1 scores for each class, with each class's contribution weighted based on its frequency in the data. Let \(Q\) be the set of all queries, and \(N\) be the total number of queries. For each query \(q\in Q\), the retrieval model outputs a ranked list of candidate targets as given by equation (32): where \({c}_{q,i}\) is the \({i}^{{th}}\) highest-ranked candidate for query \(q\) based on cosine similarity, and \({\rm{quantile}}\left(q\right)\) is the quantile of the smallest index i of the ground-truth target. \({\rm{Recall@K}}\) is defined as the fraction of queries for which the ground-truth target occurs at rank \(K\) or better as given by equation (33): where \({\rm{I}}\left[\bullet \right]\) is an indicator function that takes the value of 1 if \({\rm{quantile}}\left(q\right)\le K\) and 0 otherwise. Two-sided Student's t-test and Wilcoxon rank-sum test were used to assess statistical significance between models. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The normal human small intestine dataset used for the tissue alignment task can be found in https://doi.org/10.1038/s41467-023-36071-5 (ref. The human ovarian carcinosarcoma dataset used for the tissue alignment task can be found at https://doi.org/10.1016/j.xgen.2021.100065 (ref. The human breast cancer dataset used for the tissue alignment task can be found at https://doi.org/10.1038/s41467-023-43458-x (ref. The human colorectal cancer dataset including Visium, Visium-HD and scRNA-seq data of serial slices used for cell-type decomposition task can be found at https://doi.org/10.1101/2024.06.04.597233 (ref. The TNBC scRNA-seq data used for the cell-type decomposition task can be found at https://doi.org/10.1038/s41467-018-06052-0 (ref. The TNBC Xenium data generated in this study have been deposited in the Gene Expression Omnibus database under accession code GSE293199. The brain dataset including Visium data of serial slices used for cell-type decomposition task can be found at https://doi.org/10.1038/s41587-021-01139-4 (ref. The brain scRNA-seq dataset used for cell-type decomposition task can be found at https://doi.org/10.1038/s41586-018-0654-5 (ref. The histology images of TCGA BRCA dataset used for the tissue annotation task are available from the NIH Genomic Data Commons (https://portal.gdc.cancer.gov/). The bulk RNA-seq data used for tissue annotation task are available from the Genotype-Tissue Expression Portal (https://gtexportal.org/home/) and TCGA (https://portal.gdc.cancer.gov/). CRC7k image patch data and labels can be found at Zenodo via https://doi.org/10.5281/zenodo.1214456 (ref. WSSS4LUAD image patches and labels can be found at https://wsss4luad.grand-challenge.org/. LC25000 image patches and labels can be found at https://github.com/tampapath/lung_colon_image_set/. PatchCamelyon image patches and labels can be found at https://patchcamelyon.grand-challenge.org/. The validation and test datasets used for the image–transcriptomics retrieval task can be found in Supplementary Table 4. The normal human heart samples used for the ST gene expression prediction task can be found at https://doi.org/10.1038/s41586-023-06311-1 (ref. The ST-bank database is available at https://github.com/GuangyuWangLab2021/Loki/. The pretrained OmiCLIP weights are available via https://huggingface.co/WangGuangyuLab/Loki/. Similar image search for histopathology: SMILY. Chen, C. et al. Fast and scalable search of whole-slide images via self-supervised deep learning. Artificial intelligence reveals features associated with breast cancer neoadjuvant chemotherapy responses from multi-stain histopathologic images. Lu, M. Y. et al. AI-based pathology predicts origins for cancers of unknown primary. Zhu, L. et al. An accurate prediction of the origin for bone metastatic cancer using deep learning on digital pathological images. Chen, R. J. et al. Pan-cancer integrative histology-genomic analysis via multimodal deep learning. Towards a general-purpose foundation model for computational pathology. Radford, A. et al. Learning transferable visual models from natural language supervision. 38th International Conference on Machine Learning 8748–8763 (PMLR, 2021). Yu, J. et al. CoCa: contrastive captioners are image-text foundation models. Supervision exists everywhere: a data efficient contrastive language-image pre-training paradigm. Lu, M. Y. et al. A visual-language foundation model for computational pathology. A visual–language foundation model for pathology image analysis using medical twitter. et al. An ERK5-NRF2 axis mediates senescence-associated stemness and atherosclerosis. Mao, H. et al. CRAT links cholesterol metabolism to innate immune responses in the heart. Chau, K. M. et al. TNIK regulation of interferon signaling and endothelial cell response to virus infection. Nguyen, M. T. et al. Endothelial activation and fibrotic changes are impeded by laminar flow-induced CHK1-SENP2 activity through mechanisms distinct from endothelial-to-mesenchymal cell transition. Silverman, A. D., Karim, A. S. & Jewett, M. C. Cell-free gene expression: an expanded repertoire of applications. Ding, J., Sharon, N. & Bar-Joseph, Z. Temporal modelling using single-cell transcriptomics. Chakraborty, A. et al. Epigenetic induction of smooth muscle cell phenotypic alterations in aortic aneurysms and dissections. & Ren, B. Characterizing cis-regulatory elements using single-cell epigenomics. Wagner, D. E. & Klein, A. M. Lineage tracing meets single-cell omics: opportunities and challenges. Simple and effective embedding model for single-cell biology built from ChatGPT. 41st International Conference on Machine Learning 27299–27325 (PMLR, 2024). Szabo, P. A. et al. Single-cell transcriptomics of human T cells reveals tissue and activation signatures in health and disease. Transfer learning enables predictions in network biology. Large-scale foundation model on single-cell transcriptomics. McCarthy, D. J., Campbell, K. R., Lun, A. T. & Wills, Q. F. Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R. Bioinformatics 33, 1179–1186 (2017). A dendrite method for cluster analysis. Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. & Raphael, B. J. Alignment and integration of spatial transcriptomics data. Jones, A., Townes, F. W., Li, D. & Engelhardt, B. E. Alignment of spatial genomics data using deep Gaussian processes. Mirzazadeh, R. et al. Spatially resolved transcriptomic profiling of degraded and challenging fresh frozen samples. Villacampa, E. G. et al. Genome-wide spatial expression profiling in formalin-fixed tissues. Tang, Z. et al. Search and match across spatial omics samples at single-cell resolution. Janesick, A. et al. High resolution mapping of the tumor microenvironment using integrated single-cell, spatial and in situ analysis. Identification of the collagen type 1 alpha 1 gene (COL1A1) as a candidate survival-related factor associated with hepatocellular carcinoma. Gu, Y. et al. A pan-cancer analysis of the prognostic and immunological role of β-actin (ACTB) in human cancers. Lu, M. Y. et al. Data-efficient and weakly supervised computational pathology on whole-slide images. Predicting survival from colorectal cancer histology slides using deep learning: a retrospective multicenter study. Han, C. et al. WSSS4LUAD: grand challenge on weakly-supervised tissue semantic segmentation for lung adenocarcinoma. Veeling, B. S., Linmans, J., Winkens, J., Cohen, T. & Welling, M. Rotation equivariant CNNs for digital pathology. In Medical Image Computing and Computer Assisted Intervention – MICCAI 2018: 21st International Conference, Granada, Spain, September 16–20, 2018, Proceedings, Part II 210–218 (Springer, 2018). Lung and colon cancer histopathological image dataset (LC25000). Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram. Vahid, M. R. et al. High-resolution alignment of single-cell and spatial transcriptomes with CytoSPACE. Jain, P. & Kar, P. Non-convex optimization for machine learning. Characterization of immune cell populations in the tumor microenvironment of colorectal cancer using high definition spatial profiling. Shared and distinct transcriptomic cell types across neocortical areas. Kleshchevnikov, V. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Karaayvaz, M. et al. Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single-cell RNA-seq. Li, B. et al. Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution. Stuart, T. et al. Comprehensive integration of single-cell data. & Zhou, X. Spatially informed cell-type deconvolution for spatial transcriptomics. Dong, R. & Yuan, G.-C. SpatialDWLS: accurate deconvolution of spatial transcriptomic data. Robust decomposition of cell type mixtures in spatial transcriptomics. Cui, H. et al. scGPT: toward building a foundation model for single-cell multi-omics using generative AI. In 18th International Conference on Pattern Recognition (ICPR'06) 850–855 (IEEE, 2006). Pang, M., Su, K. & Li, M. Leveraging information in spatial transcriptomics to predict super-resolution gene expression from histology images in tumors. Zeng, Y. et al. Spatial transcriptomics prediction from histology jointly through transformer and graph neural networks. Xie, R. et al. Spatially resolved gene expression prediction from histology images via bi-modal contrastive learning. Advances in Neural Information Processing Systems 36 (2024). & Wang, C. Multimodal contrastive learning for spatial gene expression prediction using histology images. Kuett, L. et al. Three-dimensional imaging mass cytometry for highly multiplexed molecular and cellular mapping of tissues and the tumor microenvironment. Xie, W. et al. Prostate cancer risk stratification via nondestructive 3D pathology with deep learning–assisted gland analysis. Analysis of 3D pathology samples using weakly supervised AI. Schott, M. et al. Open-ST: high-resolution spatial transcriptomics in 3D. Christensen, M., Vukadinovic, M., Yuan, N. & Ouyang, D. Vision–language foundation model for echocardiogram interpretation. Hou, W. & Ji, Z. Assessing GPT-4 for cell type annotation in single-cell RNA-seq analysis. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. He, S. et al. Starfysh reveals heterogeneous spatial dynamics in the breast tumor microenvironment. Barkley, D. et al. Cancer cell states recur across tumor types and form specific interactions with the tumor microenvironment. Garbarino, O. et al. Spatial resolution of cellular senescence dynamics in human colorectal liver metastasis. Wu, R. et al. Comprehensive analysis of spatial architecture in primary liver cancer. Figiel, S. et al. Spatial transcriptomic analysis of virtual prostate biopsy reveals confounding effect of tissue heterogeneity on genomic signatures. Sountoulidis, A. et al. A topographic atlas defines developmental origins of cell heterogeneity in the human embryonic lung. Engblom, C. et al. Spatial transcriptomics of B cell and T cell receptors reveals lymphocyte clonal dynamics. Kumar, T. et al. A spatially resolved single-cell genomic atlas of the adult human breast. Kuppe, C. et al. Spatial multi-omic map of human myocardial infarction. Kanemaru, K. et al. Spatially resolved multiomics of human cardiac niches. Dosovitskiy, A. et al. An image is worth 16x16 words: transformers for image recognition at scale. In International Conference on Learning Representations (OpenReview.net, 2020). Schuhmann, C. et al. Laion-5b: an open large-scale dataset for training next generation image-text models. 100,000 histological images of human colorectal cancer and healthy tissue. This work was supported in part by the grant R35GM150460 (to G.W.) from the National Institute of General Medical Sciences (NIGMS) and grant R01HL169204-01A1 (to L.L.) from the National Institutes of Health (NIH)-National Heart, Lung, and Blood Institute (NHLBI). is supported by NIH/National Institute of Neurological Disorders and Stroke (NINDS) award K22 NS112678, NIH/National Cancer Institute (NCI) award R01 CA284315 and Cancer Prevention and Research Institute of Texas (CPRIT) award RR220017. We acknowledge J. Chang in Houston Methodist Research Institute for support and assistance in facilitating access to clinical resources essential to this study. These authors contributed equally: Weiqing Chen, Pengzhi Zhang. Weiqing Chen, Pengzhi Zhang, Tu N Tran, Yiwei Xiao, Shengyu Li & Guangyu Wang Department of Physiology, Biophysics & Systems Biology, Weill Cornell Graduate School of Medical Science, Cornell University, New York, NY, USA Pengzhi Zhang, Tu N Tran, Yiwei Xiao, Shengyu Li, Keith Youker, Li Lai, Longhou Fang, Nhat-Tu Le, John P. Cooke & Guangyu Wang Pengzhi Zhang, Tu N Tran, Yiwei Xiao, Shengyu Li, Vrutant V. Shah, Kristopher W. Brannan, John P. Cooke & Guangyu Wang Pengzhi Zhang, Tu N Tran, Yiwei Xiao, Shengyu Li, Keith Youker, Li Lai, Longhou Fang, Nhat-Tu Le, John P. Cooke & Guangyu Wang Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH, USA Department of Cardiology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Center for Immunotherapy, Neal Cancer Center, Houston Methodist Research Institute, Houston, TX, USA Department of Bioinformatics and Computational Biology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar The authors declare no competing interests. Nature Methods thanks Spencer Krieger and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Rita Strack, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Clustering performance on ST-bank data with cell type annotation. Left: clustering performance using transcriptomic embeddings generated from OmiCLIP model before and after training. Right: clustering performance usings image embeddings from OmiCLIP model before and after training. The Calinski-Harabasz scores were calculated on the embeddings (Methods) using the pretrained OmiCLIP transcriptomic (left) and image (right) encoders, evaluated for each organ type. Higher Calinski-Harabasz scores indicate better separation capability between clusters of the embeddings. b, Image and transcriptomic embeddings of the lung, kidney cancer, healthy heart, and Myocardial Infarction (MI) heart samples. Each row corresponds to a WSI and showcases information from two modalities. a, Clustering performance on all ST-bank data. Top: clustering performance using transcriptomic embeddings generated from OmiCLIP model before and after training. Bottom: clustering performance usings image embeddings from OmiCLIP model before and after training, and image embeddings generated from UNI and Pro-GigaPath, respectively. The Calinski-Harabasz scores were calculated on the embeddings using the pre-trained OmiCLIP transcriptomic (top) and image (bottom) encoders, evaluated for each organ type. Higher Calinski-Harabasz scores indicate better separation capability between clusters of the embeddings. Sample sizes are skin: 163, brain: 119, breast: 97, heart: 73, kidney: 73, embryo: 73, others: 64, liver: 57, prostate: 49, spinal cord: 44, ovary: 32, colon: 29, pancreas: 25, lung: 22, tonsil: 18, uterus: 17, adipose: 15, small intestine: 14, and stomach: 12. b, Image and transcriptomic embeddings of the spinal cord, liver cancer, brain cancer, kidney cancer and skin cancer samples. Each row corresponds to a WSI and showcases information from two modalities. a, Example image with low-quality region marked in red line and simulated low-quality image by adding Gaussian noise. Sample sizes are 10 for each simulated condition. c, Cosine similarity of the paired image with transcriptomic embeddings using OmiCLIP (original transcriptomes and down sampled transcriptome from high sequencing depth to middle sequencing depth, middle sequencing depth to low sequencing depth, and high sequencing depth to low sequencing depth, respectively), PLIP (original transcriptome), and OpenAI CLIP (original transcriptome). Tissue alignment results on breast cancer sample using Loki Align. c, Xenium ST data after Loki alignment. a, Xenium data from our in-house TNBC patient sample, colored by Louvain clusters and cell types, respectively. b, H&E image, marker gene expression (KRT7, ATCG2, RORC), and cell type distribution in an example zoom-in region of the TNBC sample. c, Cell type decomposition results on 3 major cell types of the TNBC sample using ST by RCTD, CARD, scGPT, Spatial Seurat, scFoundation, GeneFormer, CytoSPACE, Cell2location, and SpatialDWLS, respectively. a, UMAP representation of the OmiCLIP transcriptomic embeddings colored by cell types, where each dot represents a spot. b, Cell type decomposition result using Loki ST decomposition and Loki image decomposition respectively on human colorectal sample within the Visium HD capture area, and ground truth. Heatmap shows the cell type distribution of tumor, fibroblast, smooth muscle, and intestinal epithelial, respectively, with color reflecting the probability of each cell type. a, Cell type decomposition results on 3 major cell types of the TNBC sample using Loki Decompose Image-to-ST (fine-tuning, pre-training, and train from scratch). b, Bar plot shows the accuracy of decomposition of 3 major cell types by Loki Decompose Image-to-ST (fine-tuning, pre-training, and train from scratch). Error bar is standard deviation with center measured by mean. H&E images, ground truth ST gene expression, and ST gene expression predicted by Loki, Hist2ST, HisToGene, BLEEP, and mclSTExp, respectively. a, Comparison of ST gene expression prediction performances, represented by MSE and PCC respectively on 39 normal heart tissues using Loki, Hist2ST, HisToGene, BLEEP, and mclSTExp, respectively. b, Summarized comparison of ST gene expression prediction performances, represented by MSE and PCC respectively across all samples using Loki, HisToGene, mclSTExp, BLEEP, and Hist2ST respectively. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. A visual–omics foundation model to bridge histopathology with spatial transcriptomics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02258-y'>Superconductivity and normal-state transport in compressively strained La</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 09:58:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Materials (2025)Cite this article Metrics details The discovery of superconductivity under high pressure in Ruddlesden–Popper phases of bulk nickelates has sparked great interest in stabilizing ambient-pressure superconductivity in the thin-film form using epitaxial strain. Recently, signs of superconductivity have been observed in compressively strained bilayer nickelate thin films with an onset temperature exceeding 40 K, although with broad, two-step-like transitions. Here we report the intrinsic superconductivity and normal-state transport properties in compressively strained La2PrNi2O7 thin films, achieved through a combination of isovalent Pr substitution, growth optimization and precision ozone annealing. The superconducting onset occurs above 48 K, with zero resistance reached above 30 K, and the critical current density at 1.4 K is 100-fold larger than previous reports. The normal-state resistivity exhibits quadratic temperature dependence indicative of Fermi liquid behaviour, and other phenomenological similarities to transport in overdoped cuprates suggest parallels in their emergent properties. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Any additional data that support the findings of this study are available from the corresponding authors upon reasonable request. Source data are provided with this paper. Sun, H. et al. Signatures of superconductivity near 80 K in a nickelate under high pressure. Google Scholar Zhang, Y. et al. High-temperature superconductivity with zero resistance and strange-metal behaviour in La3Ni2O7–δ. Wang, G. et al. Pressure-induced superconductivity in polycrystalline La3Ni2O7–δ. Zhu, Y. et al. Superconductivity in pressurized trilayer La4Ni3O10−δ single crystals. Wang, N. et al. Bulk high-temperature superconductivity in pressurized tetragonal La2PrNi2O7. Wang, M., Wen, H.-H., Wu, T., Yao, D.-X. & Xiang, T. Normal and superconducting properties of La3Ni2O7. Chinese Phys. Locquet, J.-P. et al. Doubling the critical temperature of La1.9Sr0.1CuO4 using epitaxial strain. Bozovic, I., Logvenov, G., Belca, I., Narimbetov, B. & Sveklo, I. Epitaxial strain and superconductivity in La2–xSrxCuO4 thin films. Ko, E. K. et al. Signatures of ambient pressure superconductivity in thin film La3Ni2O7. Zhou, G. et al. Ambient-pressure superconductivity onset above 40 K in (La,Pr)3Ni2O7 films. Puphal, P. et al. Unconventional crystal structure of the high-pressure superconductor La3Ni2O7. Chen, X. et al. Polymorphism in the Ruddlesden–Popper nickelate La3Ni2O7: discovery of a hidden phase with distinctive layer stacking. Wang, H., Chen, L., Rutherford, A., Zhou, H. & Xie, W. Long-range structural order in a hidden phase of Ruddlesden–Popper bilayer nickelate La3Ni2O7. Zinkevich, M., Solak, N., Nitsche, H., Ahrens, M. & Aldinger, F. Stability and thermodynamic functions of lanthanum nickelates. Alloys Compd. Cui, T. et al. Strain-mediated phase crossover in Ruddlesden–Popper nickelates. Li, D. et al. Superconductivity in an infinite-layer nickelate. Smith, J. A., Cima, M. J. & Sonnenberg, N. High critical current density thick MOD-derived YBCO films. IEEE Trans. Wang, L. et al. Structure responsible for the superconducting state in La3Ni2O7 at high-pressure and low-temperature conditions. Li, J. et al. Identification of the superconductivity in bilayer nickelate La3Ni2O7 upon 100 GPa. Preprint at https://arxiv.org/abs/2404.11369 (2025). Dong, Z. et al. Visualization of oxygen vacancies and self-doped ligand holes in La3Ni2O7–δ. Batakliev, T., Georgiev, V., Anachkov, M., Rakovsky, S. & Rakovsky, S. Ozone decomposition. Wiesmann, H. et al. Simple model for characterizing the electrical resistivity in A – 15 superconductors. Cooper, R. A. et al. Anomalous criticality in the electrical resistivity of La2–xSrxCuO4. Hussey, N. E. Phenomenology of the normal state in-plane transport properties of high-Tc cuprates. Hwang, H. Y. et al. Scaling of the temperature dependent Hall effect in La2–xSrxCuO4. Zhou, Y. et al. Investigations of key issues on the reproducibility of high-Tc superconductivity emerging from compressed La3Ni2O7. Matter Radiat. Gu, Y., Le, C., Yang, Z., Wu, X. & Hu, J. Effective model and pairing tendency in the bilayer Ni-based superconductor La3Ni2O7. Zhang, Y., Lin, L.-F., Moreo, A. & Dagotto, E. Electronic structure, dimer physics, orbital-selective behavior, and magnetic tendencies in the bilayer nickelate superconductor La3Ni2O7 under pressure. Christiansson, V., Petocchi, F. & Werner, P. Correlated electronic structure of La3Ni2O7 under pressure. Lechermann, F., Gondolf, J., Bötzel, S. & Eremin, I. M. Electronic correlations and superconducting instability in La3Ni2O7 under high pressure. Yang, Q.-G., Wang, D. & Wang, Q.-H. Possible s±-wave superconductivity in La3Ni2O7. Liu, Y.-B., Mei, J.-W., Ye, F., Chen, W.-Q. & Yang, F. s±-wave pairing and the destructive role of apical-oxygen deficiencies in La3Ni2O7 under pressure. Yang, Y., Zhang, G.-M. & Zhang, F.-C. Interlayer valence bonds and two-component theory for high-Tc superconductivity of La3Ni2O7 under pressure. Shen, Y., Qin, M. & Zhang, G.-M. Effective bi-layer model Hamiltonian and density-matrix renormalization group study for the high-Tc superconductivity in La3Ni2O7 under high pressure. Chinese Phys. Qin, Q. & Yang, Y. High-Tc superconductivity by mobilizing local spin singlets and possible route to higher Tc in pressurized La3Ni2O7. Luo, Z., Lv, B., Wang, M., Wú, W. & Yao, D.-X. High-Tc superconductivity in La3Ni2O7 based on the bilayer two-orbital t-J model. npj Quantum Mater. Sakakibara, H., Kitamine, N., Ochi, M. & Kuroki, K. Possible high Tc superconductivity in La3Ni2O7 under high pressure through manifestation of a nearly half-filled bilayer Hubbard model. Geisler, B., Hamlin, J. J., Stewart, G. R., Hennig, R. G. & Hirschfeld, P. J. Fermi surface reconstruction in strained La3Ni2O7 on LaAlO3(001) and SrTiO3(001). Preprint at https://arxiv.org/abs/2411.14600 (2024). Zhao, Y.-F. & Botana, A. S. Electronic structure of Ruddlesden-Popper nickelates: Strain to mimic the effects of pressure. Lu, C., Pan, Z., Yang, F. & Wu, C. Interlayer-coupling-driven high-temperature superconductivity in La3Ni2O7 under pressure. Oh, H. & Zhang, Y.-H. Type-II t-J model and shared superexchange coupling from Hund's rule in superconducting La3Ni2O7. Cao, Y. & Yang, Y. Flat bands promoted by Hund's rule coupling in the candidate double-layer high-temperature superconductor La3Ni2O7 under high pressure. et al. Bilayer t-J-J⊥ model and magnetically mediated pairing in the pressurized nickelate La3Ni2O7. Ouyang, Z. et al. Hund electronic correlation in La3Ni2O7 under high pressure. Wang, Z., Jiang, K. & Zhang, F.-C. Self-doped molecular Mott insulator for bilayer high-temperature superconducting La3Ni2O7. Preprint at https://arxiv.org/abs/2412.18469 (2025). Botana, A. S., Lee, K.-W., Norman, M. R., Pardo, V. & Pickett, W. E. Low valence nickelates: launching the nickel age of superconductivity. Luo, Z., Hu, X., Wang, M., Wú, W. & Yao, D.-X. Bilayer two-orbital model of La3Ni2O7 under pressure. Fan, Z. et al. Superconductivity in nickelate and cuprate superconductors with strong bilayer coupling. Jiang, K., Wang, Z. & Zhang, F.-C. High-temperature superconductivity in La3Ni2O7. Chinese Phys. Jiang, R., Hou, J., Fan, Z., Lang, Z.-J. & Ku, W. Pressure driven fractionalization of ionic spins results in cupratelike high-Tc superconductivity in La3Ni2O7. Harper, F. E. & Tinkham, M. The mixed state in superconducting thin films. Harvey, S. P. et al. Evidence for nodal superconductivity in infinite-layer nickelates. Preprint at https://arxiv.org/abs/2201.12971 (2022). Mihaly, L., Kendziora, C., Hartge, J., Mandrus, D. & Forro, L. High-pressure cell for oxygen annealing at elevated temperatures. We thank Y. Deng, K. Lee, Y. Lee, Y. Lv, J. May-Mann, F. Theuss, B. Y. Wang, Y.-M. Wu and J. J. Yu for discussions and assistance. acknowledge support from the US Department of Energy, Office of Basic Energy Sciences, Division of Materials Sciences and Engineering (contract no. DE-AC02-76SF00515), as well as SuperC and the Kavli Foundation (aspects of magnetic characterization). Work at the Stanford Nano Shared Facilities (SNSF) RRID:SCR_023230 is supported by the National Science Foundation under grant ECCS-1542152. acknowledge support from the National Science Foundation (DMR-1719875), NSF PARADIM (DMR-2039380) and the Weill Institute and the Kavli Institute at Cornell University. X-ray measurements were carried out at the SSRL, SLAC National Accelerator Laboratory, supported by the US Department of Energy, Office of Science, Office of Basic Energy Sciences (contract no. acknowledges support from the Max Planck Society and Schmidt Science Fellows in partnership with the Rhodes Trust. Stanford Institute for Materials and Energy Sciences, SLAC National Accelerator Laboratory, Menlo Park, CA, USA Yidi Liu, Eun Kyo Ko, Yaoju Tarn, Jiarui Li, Srinivas Raghu, Yijun Yu & Harold Y. Hwang Department of Physics, Stanford University, Stanford, CA, USA Yidi Liu & Srinivas Raghu Department of Applied Physics, Stanford University, Stanford, CA, USA Eun Kyo Ko, Yaoju Tarn, Jiarui Li, Yijun Yu & Harold Y. Hwang School of Applied and Engineering Physics, Cornell University, Ithaca, NY, USA Lopa Bhatt & David A. Muller Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, Menlo Park, CA, USA Vivek Thampy Max Planck Institute for Chemical Physics of Solids, Dresden, Germany Berit H. Goodge Kavli Institute at Cornell for Nanoscale Technology, Cornell University, Ithaca, NY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the project. synthesised the films. built the ozone annealing setup and studied the ozone annealing effects with Y.L. characterized and studied the transport properties of films with the assistance of Y.T. measured and analysed the STEM images. performed the reciprocal-space mapping measurements. analysed the data and wrote the paper with input from all authors. Correspondence to Yijun Yu or Harold Y. Hwang. The authors declare no competing interests. Nature Materials thanks Jinguang Cheng and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Normalized real part (Re(Vp)) and b, imaginary (Im(Vp)) part of the voltage in the pickup coil as a function of temperature, measured using a two-coil mutual-inductance technique on sample P236. c, Resistivity of the sample. For comparison, corresponding data from refs. 9 and 10, measured on La3Ni2O7 and (La,Pr)3Ni2O7 films, are also plotted. The red triangles in b and c mark the onset temperature of diamagnetic response and zero resistance Tc, respectively. a, Synchrotron x-ray RSM of sample P75 on substrate SLAO(001). The (1117) Bragg peak of the film is measured along with the SLAO(109) Bragg peak. b, Tc,onset versus in-plane lattice constant. The bulk and thin film studies from ref. 9 are plotted for comparison. a, ρ(T) of films grown at different p(O2), with ozone annealing conditions as described in ref. 9. b, ρ(T) of films annealed at varying Tanneal but the same w(O3). c, ρ(T) of films annealed at varying w(O3) but the same Tanneal. In each plot, the sample with the highest Tc,onset is highlighted with a thick line. a, Schematic of the ozone annealing setup. b, Photograph of the setup, with an inset showing a film in the chip carrier inside the quartz tube. To the right of the chip carrier, a PT1000 sensor is positioned to measure the temperature at the chip carrier. c, Interior of the ozone generator, consisting of a tunable high voltage module (front), an ozone quartz generator with inlet and outlet (centre), and a cooling fan (back). A power meter is connected to the high voltage module to monitor the power which regulates w(O3). a-c, Time-dependent profiles of ρ, w(O3), and Tanneal during optimized ozone annealing. The inset of a shows the saturation of ρ upon oxygenation completion. Blue vertical dashed lines mark changes in w(O3), while red vertical dashed lines indicate the start of warming, reaching Tanneal, and the completion of annealing, respectively. The complex evolution of ρ with time can be understood by changes in the oxidation environment during the annealing process. This is due to both the time required for the gas environment inside the quartz tube to respond after manually adjusting w(O3), and the temperature change. a-b, time-dependent resistivity measurements during ozone annealing of two specimens from the same growth (Sample P237). The triangles indicate the points at which the samples began cooling. In a, resistivity did not reach saturation before cooling, whereas in constrast to b. c, ρ(T) of the specimens shown in a and b. a, XRD θ–2θ symmetric scan of as-grown films right after growth and five months later. b, Resistivity versus temperature ρ(T) for two specimens of sample P75: One piece was ozone-annealed immediately after growth, while the other was ozone-annealed five months later. c and d, Time-dependent ρ(T) plots of ozone-annealed samples P80 (stored in ambient conditions) and P75 (stored in liquid nitrogen dewar), respectively. e, Time-dependent resistivity at 300 K, normalized by the resistivity at day 0, for samples stored under different conditions. f, A photograph of Swagelok® cap and nut that is used as a gas-tight container to store samples in a nitrogen dewar. Data are anti-symmetrised. Data below 60 K shows non-linear behaviour due to the onset of superconductivity. a-c correspond to samples P75, P78, and P108, respectively. Data from Fig. 4b in the main text. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Liu, Y., Ko, E.K., Tarn, Y. et al. Superconductivity and normal-state transport in compressively strained La2PrNi2O7 thin films. Received: 14 January 2025 Accepted: 05 May 2025 Published: 29 May 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Materials © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02683-z'>Anatomy of the 23andMe fall and implications for consumer genomics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 09:41:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. 23andMe's bankruptcy serves as a moment of reflection for the direct-to-consumer (DTC) genomics industry. We analyzed 23andMe financial data and business practices to reveal the factors behind the fall of the company, once valued at US $6 billion and now being considered for acquisition by Regeneron for merely $250 million. Key challenges faced by 23andMe in monetizing its genomic data reveal that this information, at least in a typical DTC setting, is simply not worth that much. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Davies, K. The $1,000 Genome: The Revolution in DNA Sequencing and the New Era of Personalized Medicine (Simon and Schuster, 2015). Larkin, L. DNA tests. The DNA Geek https://thednageek.com/dna-tests/ (2023). Brown, S. N., Jouni, H., Marroush, T. S. & Kullo, I. J. Circ. Polygenic Risk Score Task Force of the International Common Disease Alliance. Minikel, E. V., Painter, J. L., Dong, C. C. & Nelson, M. R. Nature 629, 624–629 (2024). Phares, S., Phillip, K. & Trusheim, M. Nat. Sudlow, C. et al. PLoS Med. Benjamin, J. S. et al. Cancer Res. Fenaux, J. et al. Oncoimmunology 12, 2217737 (2023). Khan, R. & Mittelman, D. Genome Biol. Eleven Therapeutics, Tel Aviv, Israel WhiteLab Genomics, Paris, France You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Yaniv Erlich or Dina Zielinski. is an employee and shareholder of Eleven Therapeutics, an RNA Therapeutic start-up. is an employee and shareholder of WhiteLab Genomics, a gene and cell therapy company. Quarterly analysis of public financial information Erlich, Y., Zielinski, D. Anatomy of the 23andMe fall and implications for consumer genomics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250528131827.htm'>Researchers engineer a herpes virus to turn on T cells for immunotherapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 02:13:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The idea: the virus has evolved to commandeer cellular machinery in order activate signaling pathways inside cells and these strategies can be repurposed to bolster immunotherapy against diseases like cancer. T cells are front line defenders against pathogens, like viruses, and cancer because they can kill infected or malignant cells. CAR-T therapy is one such example of prompting the body's own immune system to attack certain forms of cancer using T cells. However, the therapeutic potential of T cells can be limited by the suppressive environment present within tumors that impairs T cell survival and function. The University of Michigan team identified herpes virus saimiri, which infects the T cells of squirrel monkeys, as a source of proteins that activate pathways in T cells that are needed to promote T cell survival. Their findings hint at a new approach -- using genes from organisms with proven ability to modulate human cells -- to enhance the power of immunotherapy. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250528131824.htm'>New AI tool reveals single-cell structure of chromosomes -- in 3D</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 02:13:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In a major leap forward for genetic and biomedical research, two scientists at the University of Missouri have developed a powerful new artificial intelligence tool that can predict the 3D shape of chromosomes inside individual cells -- helping researchers gain a new view of how our genes work. Chromosomes are the tiny storage boxes that hold our DNA. This folding not only saves space -- it also controls which genes are active or inactive. But when the DNA doesn't fold the right way, it can disrupt normal cell functions and lead to serious diseases, including cancer. Historically, scientists have relied on data that averaged results from millions of cells at once. That makes it almost impossible to see the unique differences between individual cells. But the new AI model developed by Yanli Wang and Jianlin "Jack" Cheng at Mizzou's College of Engineering changes that. "That folding controls which genes are turned on or off." But the new AI tool is specially designed to work with those challenges. It's smart enough to spot weak patterns in noisy data, and it knows how to estimate a chromosome's 3D shape even when some information is missing. It also understands how to "see" biological structures correctly, even when they're rotated. Compared to a previous deep learning AI method, Mizzou's tool is more than twice as accurate when analyzing human single-cell data. "Our tool helps scientists study those differences in detail -- which can lead to new insights into health and disease." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250528131636.htm'>Mother's warmth in childhood influences teen health by shaping perceptions of social safety</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-29 01:58:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The study, published in JAMA Psychiatry, found that children who experience more maternal warmth at age 3 have more positive perceptions of social safety at age 14, which in turn predicts better physical and mental health outcomes at age 17. Greater maternal warmth, defined as more praise, positive tone of voice and acts of affection, has previously been shown to predict better health across the lifespan. One possibility is that interpersonal experiences early in life affect whether children perceive the social world as safe vs. threatening, accepting vs. rejecting and supportive vs. dismissive. Over time, these perceptions develop into mental frameworks, called social safety schemas, which help individuals interpret, organize, and make predictions about social situations and relationships. "In a way, these schemas represent your core beliefs about the world, what you can expect from it, and how you fit in." The UCLA Health study is the first longitudinal research to track how maternal warmth in early childhood is related to perceptions of social safety in mid adolescence, and how perceptions of social safety influence physical and mental health outcomes as youth near adulthood. Parental warmth care has been historically overlooked in research, Alley said, although preliminary research suggests that the quality of care that fathers provide also predicts child outcomes and should thus be a focus of future research. Researchers used data from more than 8,500 children who were assessed as part of long-term Millennium Cohort Study in the United Kingdom. Independent evaluators visited the children's homes at age 3 and assessed their mother's warmth (praise, positive tone of voice) and harshness (physically restraining or grabbing the child). At age 14, social safety schemas were measured with questions such as "Do I have family and friends who help me feel safe, secure and happy?" The children then reported on their overall physical health, psychiatric problems and psychological distress at age 17. "That is a powerful message, because although early-life circumstances are not always easy to change, we can help youth view others and their future in a more positive light," said Slavich. Alley said the fact that maternal warmth was found to more strongly affect adolescent health than maternal harshness was important because it has implications for how to best intervene. Based on the study findings, for example, enhancing a teenager's sense of safety, by way of a public health campaign or intervention, may be more effective than focusing on reducing perceptions of harshness, and it can potentially have a positive impact on health outcomes for years to come, even after poor maternal care has been experienced. Namely, it's not just about stopping the negative things like poor care but about putting effort toward enhancing the positives like warmth and safety," Alley said. "It also important to know that people who have experienced poor care during childhood are not doomed; if we focus on their perceptions of the world, we can greatly improve their lives." "Perceiving the social world as a socially safe, inclusive place to be really matters for physical and mental health, and this knowledge can be used to develop better interventions and public health campaigns designed to enhance resilience across the lifespan." Additional studies are needed to determine how maternal warmth affects children in other contexts outside the United Kingdom, as well as how health care providers and policymakers may improve perceptions of social safety to enhance youth health outcomes. Materials provided by University of California - Los Angeles Health Sciences. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            