
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-10</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251210092019.htm'>Blood tests reveal obesity rapidly accelerates Alzheimer's progression</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 17:26:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have carried out the first investigation of how obesity affects Alzheimer's disease blood biomarkers (BBMs). To explore this connection, the team drew on five years of data from 407 volunteers enrolled in the Alzheimer's Disease Neuroimaging Initiative, which provided both amyloid positron emission tomography (PET) scans and blood samples. Researchers analyzed plasma samples for several BBMs related to Alzheimer's disease, including pTau217 levels (a biomarker used in the diagnosis and monitoring of Alzheimer's disease), neurofilament light chain (NfL) -- a protein fragment released from damaged or dying neurons -- and plasma GFAP -- a protein expressed primarily in astrocytes (cells that support and protect neurons in the brain and spinal cord) using six leading commercial tests. "We believe the reduced BBMs in obese individuals was due to dilution from the higher blood volume," said study lead author Soheil Mohammadi, M.D., M.P.H., postdoctoral research associate at MIR. "In fact, by relying on the baseline measurements, you could be fooled into thinking that the people with obesity had a lower pathology of Alzheimer's disease. A longitudinal study involves repeatedly collecting data from the same group over an extended period, tracking changes and trends over a period of time. As the years passed, both Alzheimer's disease BBMs and brain PET scans showed a greater build-up of Alzheimer's-related pathology in participants with obesity compared with those without obesity. People with obesity experienced a 29% to 95% faster increase in plasma pTau217 ratio levels. Dr. Raji noted that their results showed blood tests offered greater sensitivity than PET scans for detecting the influence of obesity on Alzheimer's-related brain changes. "The fact that we can track the predictive influence of obesity on rising blood biomarkers more sensitively than PET is what astonished me in this study," he said. According to Dr. Mohammadi, the way obesity shapes the progression of amyloid burden and related shifts in Alzheimer's blood biomarkers has important implications for how clinicians assess and manage risk. Dr. Raji anticipates that repeated measurements of blood biomarkers, combined with brain imaging, will increasingly be used to track treatment strategies involving anti-amyloid drugs. "This is such profound science to follow right now because we have drugs that can treat obesity quite powerfully, which means we could track the effect of weight loss drugs on Alzheimer's biomarkers in future studies," he said. This work is foundational for future studies and treatment trials." Other co-authors are Farzaneh Rahmani, M.D., M.P.H., Mahsa Dolatshahi, M.D., M.P.H., and Suzanne E. Schindler, M.D., Ph.D. ER Doctors Are Sounding the Alarm on a Fast-Growing Cannabis Illness Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-04037-w'>Huge genetic study reveals hidden links between psychiatric conditions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 17:04:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Psychiatrists have long relied on diagnostic manuals that regard most mental-health conditions as distinct from one another — depression, for instance, is listed as a separate disorder from anxiety. But a genetic analysis of more than one million people suggests that a host of psychiatric conditions have common biological roots. The results, published today in Nature1, reveal that people with seemingly disparate conditions often share many of the same disease-linked genetic variants. The analysis found that 14 major psychiatric disorders cluster into five categories, each characterized by a common set of genetic risk factors. The neurodevelopmental category, for example, includes both attention deficit hyperactivity disorder and autism, which psychiatric handbooks classify as separate conditions. Many supposedly individual conditions are “ultimately more overlapping than they are distinct, which should offer patients hope”, says study co-author Andrew Grotzinger, a psychiatric geneticist at the University of Colorado Boulder. Grotzinger says his team's work was motivated by the finding that people diagnosed with one mental-health condition are highly likely to be diagnosed with another one. For example, previous research has shown that most people diagnosed with depression have also been diagnosed with a condition called generalized anxiety disorder, and vice versa2. To learn whether there is a biological explanation for these correlations, Grotzinger and his colleagues aggregated genomic data from more than one million people with psychiatric conditions and from millions of healthy controls. The researchers found that the 14 mental-health conditions they studied generally fall into five distinct buckets, each with its own genetic profile. A final category includes substance-use disorders such as alcohol-use disorder and nicotine dependence. People whose genetic profile corresponds to a given bucket are at elevated risk of any of the conditions in that bucket. The team then worked backwards from the categories that they had identified and found 238 specific genomic regions that are associated with at least one of these shared categories. Read the related News & Views, ‘Shared genetic risk in psychiatric disorders'. A journey into the causes and effects of depression An RNA splicing system that excises DNA transposons from animal mRNAs The ‘silent' brain cells that shape our behaviour, memory and health Laboratory of Regenerative Biology and Medicine (Dr. Yuval Rinkevich), CIMR Title: Associate or Senior Editor, Nature Computational Science  Organization: Nature Portfolio  Location: Shanghai, Beijing, New York or New Jerse... A journey into the causes and effects of depression An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69687688/structural-battery-unlock-massless-energy-storage-science/'>Scientists Built the Battery That Will Finally Unlock Massless Energy Storage</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 17:01:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The term refers to an energy storage device that can also bear weight as part of a structure—like if the studs in your home were all batteries, or if an electric fence also held up a wall. In the paper, researchers from Chalmers University of Technology and KTH Royal Institute of Technology in Sweden reveal how their “massless” structural battery works. The main use case is for electric cars, where a literally massive amount of batteries take up a ton of room and don't contribute to the actual structure of the car. “Due to their multifunctionality, structural battery composites are often referred to as ‘massless energy storage' and have the potential to revolutionize the future design of electric vehicles and devices,” the researchers explain. To make the structural battery, the scientists layered a buffer glass “fabric” between a positive and negative electrode, then packed it with a space-age polymer electrolyte and cured it in the oven. What results is a tough, flat battery cell that conducts well and holds up to tensile tests in all directions. The battery's combined qualities (or “multifunctionality”) make it 10 times better than any previous massless battery—a project scientists have worked on since 2007. Chalmers University of Technology writes in a press release: This could result in a battery that produces 75 Wh/kg of energy and 75 GPa of stiffness, setting more records for massless batteries and also greatly reducing their weight. Besides electric cars, the study team mentions e-bikes, satellites, and laptops as technologies that could use massless batteries. There could be further applications that we don't think of as electric at all today. They could even combine massless structural batteries with solar panels in order to store what they soak up for later use. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. Divers Couldn't Believe a Lost Shipwreck's Age Scientists Just Discovered a New Law of Physics</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09854-7'>Gut micro-organisms associated with health, nutrition and dietary interventions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 16:40:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The incidence of cardiometabolic diseases is increasing globally, and both poor diet and the human gut microbiome have been implicated1. However, the field lacks large-scale, comprehensive studies exploring these links in diverse populations2. Here, in over 34,000 US and UK participants with metagenomic, diet, anthropometric and host health data, we identified known and yet-to-be-cultured gut microbiome species associated significantly with different diets and risk factors. We developed a ranking of species most favourably and unfavourably associated with human health markers, called the ‘ZOE Microbiome Health Ranking 2025'. This system showed strong and reproducible associations between the ranking of microbial species and both body mass index and host disease conditions on more than 7,800 additional public samples. In an additional 746 people from two dietary interventional clinical trials, favourably ranked species increased in abundance and prevalence, and unfavourably ranked species reduced over time. In conclusion, these analyses provide strong support for the association of both diet and microbiome with health markers, and the summary system can be used to inform the basis for future causal and mechanistic studies. It should be emphasized, however, that causal inference is not possible without prospective cohort studies and interventional clinical trials. Cardiometabolic diseases (CMDs) are the leading causes of morbidity and mortality in Western countries and constitute a heavy burden on global healthcare systems1. The most predominant CMDs are cardiovascular disease (CVD) and type 2 diabetes (T2D)3, which are connected with the increased consumption of calorie-dense, high-risk processed foods observed over the past few decades4. Habitual diet is not only among the known risk factors associated with CMDs, but also the primary modifiable target for prevention and treatment5. Well established anthropometric and intermediary measures of CMDs, ranging from clinical measurements (for example, blood pressure) to lipid profiles (such as triglycerides, cholesterol and lipoproteins), glucose levels (for example, fasting and postprandial glucose, and haemoglobin A1c (HbA1c)), inflammatory markers (for example, glycosylated proteins, the systemic inflammation marker GlycA21 and high-sensitivity C-reactive protein), and known risk factors such as body mass index (BMI), can be used to study the diet–CMD axis6,7,8 but do not consider the biochemical mechanisms occurring in the human gut. The human gut microbiome has emerged as a cofactor on the same axis as it is associated with diet and cardiometabolic conditions9,10,11,12 and is a modifiable element13,14,15. A change in dietary patterns can shift the species-level composition of the microbiome, with knock-on effects on host health16. However, individual responses to dietary interventions vary, and precision nutrition aims at identifying host-specific factors that modulate the interaction between diet and host health17, but it is currently not possible to disentangle the effects diet plays to improve cardiometabolic health via the microbiome. Furthermore, the composition of the gut microbiome displays high individuality and variation depending on different demographics, ethnicity, sex and age; hence, defining or identifying universal biomarkers of a healthy gut microbiome has proven difficult18,19,20. Nutritional intervention studies usually involve low sample-size cohorts at the population level and are often limited by their statistical power and specificity to local lifestyle and dietary habits, which are all critical aspects, especially given the microbiome's complexity and variability. Large-scale comprehensive studies with multi-national populations can help disentangle some of the complex interplays between dietary patterns and the gut microbiome to develop personalized interventions to prevent and treat CMDs. Accordingly, we collated, sampled and analysed five of the largest metagenomic cohorts available to date, comprising more than 34,000 people and spanning two continents, paired with dietary data, detailed anthropometric and health markers. We identified microbiome species consistently associated with more favourable and (inversely) unfavourable health markers across continents. These species were organized into two microbiome rankings, representing host health and diet quality, respectively, that can be the basis for future causal and mechanistic studies. We used four large-scale microbiome cohorts from the ZOE PREDICT studies (n = 33,596; Fig. 1, Supplementary Table 1 and Methods) to assemble an extensive microbiome dataset of people with detailed dietary records along with anthropometric measures. Collected data comprise common health risk factors such as BMI, triglycerides, blood glucose and HbA1c, as well as several dietary indices and clinical markers that are intermediary measures of cardiometabolic health, such as the atherosclerotic CVD (ASCVD) risk, high-density lipoproteins (HDL) and GlycA21 (Supplementary Table 2 and Methods). a, In this study, we considered and harmonized five cross-sectional ZOE PREDICT cohorts with participants from the UK and the USA (Supplementary Fig. For each cohort, sample size and the percentage of female participants (% F) are reported in the upper bar plots, with sequencing depth (left-hand columns, darker colour, average size in gigabases) and the total number of detected species (right-hand columns, lighter colour) are reported in the middle bar plots, showing that cohorts with lower sequencing depths do not have fewer total numbers of detected species. Bottom box plots, distributions of age (left-hand columns, darker colour) and BMI (right-hand columns, lighter colour) in the five PREDICT cohorts (the PREDICT 1 (P1) cohort is split into its UK and US parts, but considered as a single cohort). Box plots show first and third quartiles (boxes) and the median (middle line); whiskers extend up to 1.5 × interquartile range (IQR). b, Random forest classification (discriminating the first three quartiles against the fourth quartile) and regression machine learning models (Methods) trained on the whole microbiome SGB-level relative abundance values with a cross-validation approach, show moderately strong and consistent associations with different categories of clinical data available across the five cross-sectional ZOE PREDICT cohorts (full machine learning results are reported in Extended Data Fig. A systematic machine learning validated approach9,22 (Methods) revealed strong associations consistent across the five ZOE PREDICT cohorts between the microbiome and surrogate health markers and nutrition (Fig. Markers that were classified accurately by the gut microbiome included glycemia, blood cholesterol, triglycerides and inflammation (both fasting and postprandial; Extended Data Fig. 1), with age, BMI, the healthy eating index23 and the healthful plant-based diet index (PDI)24 also correlated with microbiome machine learning regression estimates (Spearman's correlation > 0.4; Fig. The top predicted markers from both machine learning regression and classification showed consistent associations across PREDICT cohorts, with average area under the receiver operating characteristic curve (AUC) ranging from 0.64 to 0.73, and an average Spearman's correlation ranging from 0.30 to 0.46 for regression (Fig. We next set out to identify which gut microbial species were most responsible for the microbiome's associations with host markers. To do so, we grouped the 37 markers into three categories: (1) anthropometric-derived and accessible health-related measures (hereafter called ‘personal' and including, for example, ASCVD and blood pressure), (2) fasting (for example, GlycA, triglycerides, HDL, cholesterol and glucose) and (3) postprandial markers, which are surrogate measures of cardiometabolic health. As expected, some markers tended to correlate quantitatively (Supplementary Table 4 and Methods). We considered 661 non-rare microbial species (greater than 20% prevalence; Methods) according to the definition of species-level genome bins (SGBs)19,20, and computed the partial Spearman's correlations (corrected for sex, age and BMI) between the relative abundance of each micro-organism and the value of each marker. Correlations were ranked, and correlations' ranks were averaged within each category and then averaged among the three categories in each cohort (Methods). The five resulting cohort-level average rankings were averaged to derive a single ranking that we called the ‘ZOE Microbiome Health Ranking 2025' (ZOE MB health-rank). This resulted in a ranking for 661 microbial species in which the lowest ranking (closer to 0) species are the most positively associated with the considered panel of host markers and vice versa for the highest ranking (closer to 1) species (Fig. a, Average percentiles for the 15 most favourably (top) and unfavourably (bottom) ranked SGBs (selected for visualization purposes) across all five PREDICT cohorts. Percentiles are computed from the ranking of the correlations between SGBs and the different markers in each clinical data category. Percentiles close to 0 reflect SGBs consistently correlated positively with positive markers and negatively with negative markers, and vice versa for percentiles close to 1. For each cohort, the average percentiles for three clinical data categories are shown (personal, fasting and postprandial) and the cohort-level average. Box plots as in Fig. 1. b,c, Detailed percentiles for the 15 most favourably and unfavourably ranked SGBs against the markers of the three clinical data categories of the PREDICT 1 (UK) (b) and PREDICT 3 US22A (US) (c) cohorts. Detailed panels of the percentiles for the other three cohorts can be found in Extended Data Fig. 2. iAUC, incremental area under the curve; PUFA, polyunsaturated fatty acid; QUICKI, quantitative insulin sensitivity check index; THR, total-cholesterol-to-HDL ratio; VLDL, very-low-density lipoprotein. Most SGBs ranked within the 50 most favourably or unfavourably linked to host anthropometry belong to the Firmicutes phylum (92 out of 100) and, in particular, to the Clostridia class (n = 80; Supplementary Table 7). Within this class, in the ZOE MB health-ranks, most SGBs belonged to the Clostridiales order, with 32 unfavourably ranked SGBs (of which n = 27 Lachnospiraceae out of 50) and 31 favourably ranked SGBs (n = 13 Lachnospiraceae and n = 12 Ruminococcaceae) assigned to this order. Collectively, the average total relative abundance of the 50 most favourably ranked SGBs is 5.98%, whereas the 50 most unfavourably ranked SGBs account for 13.64% (Supplementary Table 7). A large portion of the 50 most favourably ZOE MB health-ranked SGBs are unknown (n = 22), meaning that these are uncultured species represented solely by microbial genomes reconstructed from metagenomic data. Of the 28 known SGBs (with available isolate genomes), 24 are still uncharacterized species without phenotypic descriptions and recognized taxonomic names (Supplementary Table 7). Eubacterium siraeum (SGB4198) and Faecalibacterium prausnitzii (SGB15317) are among the few exceptions with previous support for their favourable role9,25. By contrast, the 50 unfavourably ZOE MB health-ranked SGBs are generally species with cultured isolates and established taxonomic labels (Supplementary Table 7). Among the 44 known SGBs, several species were already linked with detrimental effects on the host, including Ruminococcus gnavus26, Flavonifractor plautii27, Ruminococcus torques28,29 and Enterocloster bolteae30. Overall, the most prevalent favourably ranked health-associated micro-organisms in the human gut belong to under-investigated species, highlighting gaps in our knowledge of the potential beneficial role of the human microbiome in promoting and maintaining non-pathogenic conditions. Similarly to the ZOE MB health-ranks, we defined a species ranking on the basis only of dietary markers across all five PREDICT cohorts, which we called the ‘ZOE Microbiome Diet Ranking 2025' (ZOE MB diet-rank; Supplementary Table 5). As markers of a generally healthier diet, we adopted five validated indices (Methods) computed starting from validated food frequency questionnaires (FFQs) or logged diet data (logged using a mobile phone app), reflecting long- and short-term dietary habits, respectively (Extended Data Figs. The ZOE MB health- and diet-rankings showed, as expected, general concordance (Spearman's ρ = 0.72; Extended Data Fig. Although the large majority of the SGBs highlighted by high or low ZOE MB health-ranks and diet-ranks belong to unknown taxa, reported phenotypic characteristics of known species agree with our analysis. For example, R. torques (SGB4608) and F. plautii (SGB15132), discussed previously as unfavourable species according to the ZOE MB health-ranks, were also concordantly unfavourably ranked in the ZOE MB diet-ranks (0.991–0.904 and 0.981–0.901, respectively). On the other hand, the favourably ranked Blautia glucerasea (SGB4816) was described to reduce visceral fat accumulation, blood glucose and triglycerides in mice31 (ZOE MB health-ranks and diet-ranks of 0.267 and 0.062, respectively). As another example, in a dietary fibre supplementation trial involving individuals with T2D, Lachnospira eligens (SGB5082) was increased selectively and associated negatively with postprandial glucose and insulin, body weight and waist circumference32 (ZOE MB health-ranks and diet-ranks of 0.276 and 0.115, respectively), indicating that precise dietary interventions aimed at stimulating beneficial bacterial growth can contribute to treating or managing metabolic disorders symptoms. Despite the overall agreement between the ZOE MB health- and diet-rankings, 65 out of the 661 ranked SGBs showed discordant rankings (absolute rank difference at least 0.3; Extended Data Fig. Generally, the different trends may be due to the different capacities of certain bacteria (for example, generalists) to use a variety of substrates, including those derived from unhealthy diets, while releasing functional metabolites with protective or health-promoting effects. Among these, for example, Harryflintia acetispora (SGB14838) was found associated with favourable cardiometabolic markers and unfavourable diets (ZOE MB health-rank = 0.363 and ZOE MB diet-rank = 0.879) in this study. This strict anaerobe can use readily available monosaccharides such as maltose, glucose and fructose, but can also produce short-chain fatty acids33, which are regulatory and anti-inflammatory mediators34. Across the US and UK populations, the ZOE MB health-rankings showed high consistency (Spearman's ρ = 0.61; Extended Data Fig. 6b), whereas country-specific ZOE MB diet-rankings were more heterogeneous (Spearman's ρ = 0.26; Extended Data Fig. The intraclass correlation coefficients (ICC)35 also suggest that the ZOE MB health-ranks are more consistent across countries than the ZOE MB diet-ranks (ICC = 0.5929 and 0.2623, respectively; Extended Data Fig. Across cohorts, we obtained an ICC = 0.63 and 0.46 for the ZOE MB health-ranks and diet-ranks, respectively, indicating that health rankings were more able to capture cohorts and countries differences, whereas the most favourably ranked species appeared to match across populations with similar levels of industrialization and lifestyle. BMI is an imperfect but widely adopted and easy-to-obtain anthropometric marker of health risk. As BMI was not included among the markers of the ZOE MB health- and diet-rankings, and we corrected for it in the partial correlation analysis, we set out to evaluate how the two rankings can stratify people according to their BMI to assess how health signatures in the gut microbiome are reflected in body mass. We correlated the 661 ZOE MB health-ranked species with BMI (corrected for sex and age), in each PREDICT cohort, and found that, overall, the ranks were associated positively with BMI (Spearman's ρ = 0.72), with the favourably ranked SGBs correlated negatively with BMI, whereas unfavourably ranked SGBs correlated positively with BMI (Fig. These results were confirmed when considering the ZOE MB diet-ranks and discrete BMI categories (Extended Data Fig. 7a–c; all intra-dataset comparisons statistically significant at Q < 0.2 and all 30 except 7 at Q < 0.01) as well as the cumulative abundance of the species in the two 50-species sets (Fig. 3b,c; all intra-dataset comparisons statistically significant at Q < 0.2 and all 30 except 5 at Q < 0.01). a, Concordance of ZOE MB health-ranks with partial Spearman's correlations against BMI (corrected for sex and age) across PREDICT cohorts. Favourably ranked SGBs correlate negatively with BMI; unfavourably ranked SGBs correlate positively (ZOE MB diet-ranks in Extended Data Fig. b,c, Cumulative relative abundance of favourably (b) and unfavourably (c) ranked SGBs across BMI categories. As BMI increases, reflecting higher health risks, the abundance of favourable SGBs decreases whereas that of unfavourable SGBs increases. Similar patterns were seen for SGB richness (Extended Data Fig. Only non-significant (NS) false discovery rate (FDR)-corrected P values (Q > 0.01, two-sided Mann–Whitney U-test) are annotated. Box plots as in Fig. 1. d, Meta-analysis of the 50 most favourable and unfavourable SGBs comparing participants of healthy weight with those with obesity from public cohorts. Lower BMI is associated with more favourable SGBs; people with higher BMI carry more unfavourable SGBs. Meta-analysis on ranks defined on UK and US participants shows reproducibility across countries. Other comparisons are in Extended Data Fig. 8 and the diet-ranked SGBs meta-analysis in Extended Data Fig. Country codes: ARG, Argentina; AUT, Austria; DEU, Germany; DNK, Denmark; FRA, France; GBR, United Kingdom of Great Britain and Northern Ireland; IRL, Ireland; ISR, Israel; KAZ, Kazakhstan; NLD, Netherlands; USA, United States of America. e, Meta-analysis of disease group (adjusted by sex, age and BMI) on standardized mean differences (SMD) of cumulative relative abundance of the 50 most favourable (left) and unfavourable (right) SGBs from both rankings (meta-analysis on SGB richness in Supplementary Fig. f, Meta-analysis of normalized ZOE MB health-ranks and diet-ranks, weighted by arcsin square-root of relative abundance values (right, weighted score sum; left, score sum (unweighted)). Horizontal lines in meta-analysis plots represent 95% confidence intervals. CRC, colorectal cancer; IBD, inflammatory bowel disease; IGT, impaired glucose tolerance. To generalize these associations, we leveraged a total of 5,348 healthy individuals from 27 public cohorts divided into three BMI categories, healthy weight (n = 2,837), overweight (n = 1,562) and obese (n = 949; Supplementary Table 9 and Methods). In 47 pairwise comparisons, 34 had a higher median richness for the 50 most favourably ranked ZOE MB health SGBs in lower BMI groups versus higher BMI groups (binomial P = 0.003; Supplementary Table 10 and Supplementary Fig. 3a), and this was not dependent on country effects or sequencing depth (Supplementary Table 11), highlighting the generalization of the identified ranks. Meta-analysis based on linear regression on single cohorts (Methods) showed that individuals with healthy weight carried, on average, 5.2 more of the 50 favourably ZOE MB health-ranked SGBs than people with obesity (P = 0.0003; Fig. 3d and Supplementary Table 12), which corresponded to a normalized difference in the cumulative abundances of unfavourably and favourably ranked SGBs of Cohen's d = −0.59 (P < 0.0001; Supplementary Tables 10 and 13 and Methods). Correspondingly, individuals with obesity carried, on average, 1.95 more of the unfavourably ranked SGBs than people of healthy weight (P = 0.0005; Fig. 3d, Supplementary Tables 14 and 15; Cohen's d on cumulative relative abundances = 0.29; P = 0.0001). Pairwise analysis of the other BMI categories confirmed these results (Extended Data Fig. Similarly, we tested the association of the 50 most favourably and unfavourably ZOE MB diet-ranked SGBs with BMI, and found similar but milder signals compared with the ZOE MB health-ranks (average Spearman's correlations between the two ranks and BMI of 0.61 and 0.72, respectively; Fig. Using public datasets, 36 intra-dataset comparisons out of 47 showed a higher median cumulative abundance and a higher median richness of the 50 most favourable SGBs in lower BMI classes compared with higher BMIs (binomial P = 0.0003; Supplementary Fig. Conversely, 36 comparisons showed a higher median count of the least favourable 50 SGBs for the higher BMI classes compared with the lower BMI groups (binomial P = 0.0003; Supplementary Table 10). The contribution of diet-ranked SGBs in different BMI categories similarly showed a decreasing number and cumulative relative abundance of favourably ranked SGBs and an increase in unfavourably ranked SGBs (Extended Data Fig. In meta-analysis, healthy weight and overweight participants carried 3.5 and 1.5 more favourable diet-ranked SGBs, and 1.25 and 0.88 fewer unfavourable ZOE MB diet-ranked SGBs than obesity participants, respectively (Extended Data Fig. All these analyses were confirmed when rankings were computed without adjusting for BMI (Extended Data Fig. 7h–k) and, altogether, these results suggest that the ZOE MB health- and diet-ranks can stratify people based on their obesity status regardless of geography. Next, we assessed whether the ZOE MB health-ranked SGBs had a differential presence or abundance in control participants compared with participants with a defined disease condition, exploiting 25 case–control, publicly available microbiome studies (4,816 samples in total with n = 2,707 controls and n = 2,109 cases; Methods) investigating five diseases with variable levels of association with the gut microbiome (Supplementary Table 20). The number of the 50 most favourably ZOE MB health-ranked SGBs was higher in controls than cases for 21 of the 25 cohorts, whereas the count of the 50 most unfavourably ranked SGBs was correspondingly higher in cases for the same number of cohorts (binomial P = 0.0004). We performed a meta-analysis on the count of the 50 most favourable and unfavourable SGBs from the ZOE MB health- and diet-rankings. Control samples carried, on average, 3.6 more favourably ranked SGBs than participants with disease (random-effect model, P = 0.0002; Methods) and 1.6 fewer unfavourable SGBs (P = 0.0004; Supplementary Fig. Similarly, for the ZOE MB diet-ranked SGBs, controls carried, on average, 3.8 more favourable SGBs and 1.3 fewer unfavourable SGBs, P = 9.5×10−6 and P = 0.0006, respectively; Supplementary Fig. Furthermore, meta-analyses of the cumulative abundance of the 50 most favourable and unfavourable SGBs confirmed a greater contribution from favourable species in control groups and of unfavourable SGBs in the corresponding disease groups (meta-analysis Cohen's d = −0.29, P = 7.1 × 10−6 and d = 0.21, P = 0.054 for the ZOE MB health-ranks; d = −0.24, P = 3.1 × 10−6 and d = 0.28, P = 0.0002 for the ZOE MB diet-ranks; Fig. To assess how informative the rankings are in summarizing the health-associated status of a single sample, we scored all metagenomes from diseased and control participants by summing the normalized ZOE MB health-ranks of the SGBs present in the sample (Methods). We found a strong separation between diseased and control participants (meta-analysis Cohen's d = −0.37, P = 8.3 × 10−8), improving over the simple counting of the number of most favourable and unfavourable SGBs (Fig. Notably, T2D showed the strongest disease-specific association (meta-analysis Cohen's d = −0.47, P = 6.78 × 10−5; Fig. 3f and Supplementary Table 23) with the weighted version of this score showing an even stronger effect for T2D (meta-analysis Cohen's d = −0.51, P = 0.0002). People were also scored using the ZOE MB diet-ranks, and similar links with their health status emerged (Fig. Notably, standard alpha diversity measures such as gut SGBs richness and Shannon's entropy measures showed weaker and less consistent associations, with significant links only in the IBD and T2D comparisons (Supplementary Fig. Although the ranking-based scoring of single samples cannot have the same predictive power for host phenotypes compared with condition-specific supervised learning approaches relying directly on labelled training data, our results showed how embedding the ranking system into a simple one-dimensional microbiome index provides a meaningful evaluation of microbiome health conditions. To validate the effect of dietary changes on the presence and abundance of gut microbial species according to their ZOE MB health-rankings, we analysed two dietary intervention studies, namely ZOE METHOD36 and BIOME37 (ClinicalTrials.gov registrations, NCT05273268 and NCT06231706, respectively). In brief, the ZOE METHOD cohort comprised n = 347 people assigned to a personalized dietary intervention programme (PDP; n = 177) arm versus an arm with general diet advice following the US Department of Agriculture recommendations (control, n = 170). People assigned to the PDP group showed lower energy intake and a significant decrease in triglycerides, HbA1c, weight and waist circumference after 18 weeks36. The ZOE BIOME cohort comprised n = 349 healthy adults (intention-to-treat) randomized into the primary intervention group (receiving a defined prebiotic blend, n = 116), the functional control group (receiving bread croutons to match the calories in the control group, n = 120) and the daily probiotic group (supplemented with 15 billion colony-forming units of Lacticaseibacillus rhamnosus per day, n = 113). Overall, weight, waist circumference, metabolites and gastrointestinal symptoms did not differ significantly between groups37. We identified which microbiome species were impacted significantly by the dietary interventions in the two cohorts. In the ZOE BIOME cohort, 57, 4 and 14 prevalent SGBs showed significant changes at the endpoint (Q < 0.01) for the prebiotic blend, probiotic and control arm, respectively (Fig. Among the species with a significant change in the prebiotic arm were beneficial fibre-degrading Bifidobacterium adolescentis (SGB17244), Bifidobacterium longum (SGB17248) and Blautia obeum (SGB4811)38,39,40, as well as butyrate-producing Agathobaculum butyriciproducens (SGB14993), Anaerobutyricum hallii (SGB4532) and Coprococcus catus (SGB4670)41,42. By contrast, the species Dysosmobacter welbionis (SGB15078), among the top unfavourably associated SGBs in our study, was decreased significantly by the same dietary intervention (Supplementary Table 25). 4b and Supplementary Table 25; Wilcoxon signed-rank test Q < 0.1). Of note, the prominent butyrate producers Roseburia hominis (SGB4936) and A. butyriciproducens (SGB14993) were also found to increase in the PDP intervention. a,b, Pre–post dietary intervention variations in prevalent gut microbial SGBs (at least 10% at both time points). The plots show the effect size (log2-transformed ratio of mean relative SGB abundance at endpoint over baseline) against the significance (Q values, FDR–Benjamini–Hochberg-corrected P values). a, BIOME cohort (ClinicalTrials.gov NCT06231706) with n = 321 healthy adults from the UK (n = 106 prebiotic blend, n = 106 probiotic and n = 109 control), significance threshold set to Q < 0.01. b, METHOD cohort (ClinicalTrials.gov NCT05273268) with n = 347 US individuals (n = 177 PDP, n = 170 control), and significance threshold set to Q < 0.1. c, Change in relative abundance for the significant SGBs in the intervention arms of BIOME (prebiotic blend, n = 57). d, Change in relative abundance of METHOD (PDP, n = 46), separated into those that increase from those that decrease from baseline (B) to endpoint (E). 10a–d reports the change in relative abundance and prevalence of the control and prebiotic arms. Two-sided Wilcoxon test; box plots as in Fig. The dietary intervention groups of both clinical trials that aimed at improving diet using different approaches (prebiotic blend for BIOME and PDP for METHOD) showed the highest number of significantly changing SGBs (Fig. Focusing on the most significant gut microbial SGBs with largest change in relative abundance after dietary interventions, we found increasing Bifidobacterium animalis (SGB17278)—a bacterium present in dairy-based foods and in the microbiome of people consuming larger amounts of them43,44 (Fig. 5a,b and Supplementary Table 25), an unknown Lachnospiraceae bacterium (SGB4953, BIOME; Fig. 5b) both previously associated with a vegan diet43, and another unknown Lachnospiraceae bacterium (SGB5200, BIOME; Fig. 5b), Phocea massiliensis (SGB14837), a currently uncharacterized Ruminococcaceae species (SGB14899) and Candidatus Pararuminococcus gallinarum (SGB63327) (all found in the BIOME cohort; Fig. 5a), were instead decreasing in the intervention and reported to be associated with a mixed diet43. The Streptococcus salivarius (SGB8007, METHOD; Fig. 5b) species was also found in food microbiomes45 and, together with an unknown Ruminococcaceae species (SGB14899, BIOME; Fig. 4a (left), paired with their ZOE MB health-ranks and diet-ranks, if available (right). b, The 20 most significant gut microbial SGBs with the greatest effect sizes following the METHOD personalized dietary intervention programme from Fig. 4b (left), paired with their ZOE MB health-ranks and diet-ranks, if available (right). The x axis shows the log2-transformed ratio of mean relative abundance SGB values at endpoint over baseline. All values are reported in Supplementary Table 25. c,d, The distributions of the ZOE MB health-ranks (c) and diet-ranks (d) for the prebiotic blend arm of the BIOME cohort (n = 57 of tested SGBs). The distributions show that SGBs increasing in relative abundance have significantly more favourable ranks, whereas decreasing SGBs have more unfavourable ranks (two-sided Mann–Whitney U-test, P = 7.78 × 10−3, P = 3.00 × 10−5, P = 5.20 × 10−5 and P = 2.03 × 10−5, respectively). Distributions of the ZOE MB health-ranks and diet-ranks for the other arms are reported in Extended Data Fig. Box plots as in Fig. We found that the SGBs with increased relative abundance at endpoint in the prebiotic blend arm of the BIOME trial (Fig. 4a and Supplementary Table 25) showed significantly more favourable ZOE MB health-ranks and diet-ranks than the decreasing SGBs (Fig. No significant enrichment for ZOE MB health- and diet-rankings was instead detected for the significantly changing SGBs for the probiotic group of the BIOME cohort or the control groups of both BIOME and METHOD cohorts (Extended Data Fig. Together, these results show how dietary interventions or tailored prebiotic blends, both aiming at improving diet quality, positively modulate the microbiome composition. The SGBs' rankings (ZOE MB health and diet), which were defined on cross-sectional independent cohorts, were strongly and consistently predictive of the SGBs most associated with the dietary interventions in independent cohorts and countries, supporting the direct, reproducible and actionable link between diet and microbiome composition. Defining the baseline composition of the human gut microbiome in ‘healthy' host conditions has been a long-standing challenge. This area has several problems, including defining general host health across age, as well as the inter-population microbiome variability, the existence of several distinct health-associated microbiome configurations19,46, the personalized nature of diet's impact on the gut microbiome16,47, the diversity of dietary regimes48,49 and the effect of social interaction on microbiome transmission44. To address this challenge, we reformulated the question of what a health-associated microbiome is by scoring gut microbiome species for their tendency to be correlated with healthy diet scores and with the continuum of a panel of intermediate markers of cardiometabolic health, in large and generally healthy populations. By leveraging diet scores such as the healthy eating index or the healthful PDI, and health estimators such as blood glucose, HDL and triglycerides, we identified species that are expected to characterize hosts in healthier conditions, as well as other species that are enriched in hosts with more unfavourable health risk factors. Most of the key health-associated species were from previously uncharacterized species, underlining the wide knowledge gap of the microbiome composition in non-diseased conditions. These rankings, named ZOE MB health-ranks and diet-ranks, are released and maintained publicly (Supplementary Table 5 and ‘Data availability' section) and can be adopted by the research community to evaluate whether a given human gut microbiome sample is characterized by a more favourable or unfavourable diet and health-associated species. Several factors were crucial in the robust definition of the proposed microbiome species ranking systems. First, the scale of our combined cohorts with consistent experimental protocols for metagenomic sequencing and analysis is unprecedented. Second, the geographic diversity spanning all US states and UK regions, although confined to typical Westernized lifestyles and diets, allowed us to overcome local lifestyle-associated microbiome configurations. Third, consistent long- and short-term diet logging data processed in an integrative quantitative approach and validated markers that are intermediary measures of cardiometabolic health, and more advanced postprandial metabolomic-derived markers, enabled a fine-grained definition of relative health gradients across the surveyed populations. Fourth, publicly available datasets that were processed and curated uniformly, permitted independent validation and generalization of the results and showed the relevance of the species rankings toward additional conditions and diseases not evaluated in the original populations. We acknowledge that the demographic composition of the cohorts may influence some associations, and we are continuing to expand in both population scale and precision of each host-associated readout. Our microbiome species ranking system proved accurate in reflecting changes induced by large-scale dietary intervention trials with associated host marker improvements (Figs. Indeed, the cross-sectional associations were reflected in a significant and substantial increase of health-associated microbiome species and a reduction or depletion of unfavourably ranked species. Many health-associated host markers are co-correlated because they are nutritional indicators, and disentangling their direct interactions from those mediated by the microbiome will remain elusive until large-scale microbiome interventions become possible in humans. In this respect, one key limitation of our study design is that it does not allow directly disentangling of the effect that diet exerts on the microbiome to improve cardiometabolic health from the impact of diet only. This is particularly important as diet-based ranks were more dependent on country-related differences compared with health ranks, and further studies should explore food-specific links with gut microbial species and cardiometabolic outcomes in greater detail50. This would entail designing large-scale interventions in which both the introduction of single foods and alterations of specific microbiome characteristics (for example, by administration of specific microbiome members) are tested, which are ultimately required to provide causal evidence that personalized nutritional interventions targeting the microbiota have a robust and reproducible impact on cardiometabolic health. By providing the full list of ranked microbial species, this work can be exploited in future research on microbiome-powered precision nutrition and can be expanded in the future to more diverse populations and lifestyles that are currently underrepresented in microbiome, nutritional and health studies. The ZOE PREDICT programme comprises several distinct studies that together constitute one of the largest multi-omic health initiatives, linking diet, person-specific metabolic responses to foods, and the gut microbiome. The PREDICT 1 cohort (NCT03479866) was described previously9,51. In brief, PREDICT 1 enrolled 1,098 participants (n = 1,001 from the UK and n = 97 from the USA) who underwent a clinical visit to collect anthropometric information and blood samples, followed by an at-home phase during which postprandial responses to both standardized tests and ad libitum meals were recorded. Stool samples were collected at home before the in-person clinical visit. The PREDICT 2 study (NCT03983733) had a similar collection protocol to PREDICT 1 but was conducted entirely remotely and included data from 975 people from 48 US states (including the federal District of Columbia and without participants from North Dakota and Hawaii). The PREDICT 3 cohorts (US21, US22A and UK22A) are research cohorts (NCT04735835) embedded within the ZOE commercial product. Participants provide informed written consent for their data to be used for scientific research purposes. Cardiometabolic markers were collected as described below. Furthermore, we considered and analysed two registered clinical nutritional intervention studies, namely METHOD36 (NCT05273268) and BIOME37 (NCT06231706), focusing on the microbiome changes and their links with the two derived SGB-level rankings (ZOE MB health-ranks and diet-ranks). All study protocols are registered and available on clinicaltrials.gov through the clinical trials number and link affiliated with each trial. For the PREDICT 1 cohort, sample collection, DNA extraction and sequencing were described previously9. The PREDICT 2 samples were collected in Zymo buffer, DNA extraction was performed at QIAGEN Genomic Services using DNeasy 96 PowerSoil Pro, and sequencing was performed on the Illumina NovaSeq 6000 platform using the S4 flow cell and targeting 7.5 Gb per sample. Sample processing was performed by Zymo and Prebiomics. In brief, DNA extraction by Zymo used the ZymoBIOMICS-96 MagBead DNA kit, whereas Prebiomics used the DNeasy 96 PowerSoil Pro kits. Sequencing libraries were prepared using the Illumina DNA Prep Tagmentation kit, following the manufacturer's guidelines. Whole-genome shotgun metagenomic sequencing on the Illumina NovaSeq 6000 platform used the S4 flow cell and targetted 3.75 Gb per sample. All raw sequenced data were quality controlled using the preprocessing pipeline available at https://github.com/SegataLab/preprocessing, which comprises three steps: (1) removal of reads with low-quality (Q < 20), too short (length under 75 nt), or with more than two ambiguous bases; (2) removal of host contaminant DNAs (Illumina's spike-in phiX 174 and human genomes, hg19); and (3) synchronization of paired-end and unpaired reads. In the PREDICT cohorts, we assessed long-term food intakes using FFQs, which were largely consistent across cohorts. Specifically, for PREDICT 1 participants (UK), we used a modified 131-item European Prospective Investigation into Cancer and Nutrition (EPIC) FFQ52. Participants in PREDICT 2 (USA) were surveyed using a similarly validated Diet History Questionnaire-III FFQ, including 135 items about food and beverages, as well as 26 questions about dietary supplements53. In PREDICT 3 UK22A and US22A, we developed and used a 264-item FFQ adapted from the EPIC-Norfolk Study FFQ and the Diet History Questionnaire-III. Consequently, there is a large overlap between the food items collected across the FFQs; for example, 90% of questions in the EPIC FFQ are included in the PREDICT 3 FFQ. This FFQ also includes additional food items to accurately capture modern eating habits—a limitation of older FFQ versions54. In the PREDICT 3 US21 cohort, FFQs were not collected, and only short-term logged dietary data collected using the ZOE mobile phone app were used instead. Starting from both long- and short-term dietary data, we computed three versions of the PDI55, namely, the overall PDI, the healthful PDI (measuring the adherence to a healthier plant-based foods diet) and the unhealthy PDI (measuring the intake of unhealthful plant-based foods), as well as the healthy eating index23 (measuring how consumed foods align with dietary guidelines), the alternative Mediterranean diet score (measuring the adherence to a Mediterranean diet)56 and the Healthy Food Diversity (HFD) index (measuring the number, distribution and health value of consumed foods)57. Specifically, to calculate PDIs and the healthy eating index, food items were first assembled into food groups by mapping them onto a ‘food tree' consisting of a database of nutrient information arranged according to a hierarchical tree structure: level 1 (9 food groups), level 2 (52 food groups) and level 3 (195 food groups). UK foods were mapped onto the Composition of Foods Integrated Dataset (CoFID)58 using food categories or sub-group codes, whereas US foods were similarly mapped onto the US Department of Agriculture Food and Nutrient Database for Dietary Studies database. Level 3 foods were aggregated and harmonized by nutrition scientists to allow for comparisons across cohorts. The Mediterranean diet and HFD scores were calculated as described previously9. In PREDICT 1, sex and age were self-reported, whereas height, weight and blood pressure were measured at a clinic visit (day 0). At the clinic visit, participants were also fitted with wearable continuous glucose monitor CGM) devices (Abbott Freestyle Libre Pro (FSL)), visceral fat mass was measured using dual-energy X-ray absorptiometry scans following standard manufacturer's recommendations (DXA; Hologic QDR 4500 plus) and fasting GlycA was measured using a high-throughput NMR metabolomics (Nightingale Health) 2016 panel. Fasting and postprandial venous blood samples were also collected at the clinic; plasma glucose and serum total cholesterol, HDL-C and triglycerides were measured using Affinity 1.0, and whole blood HbA1c% was measured using Viapath. The ten-year ASCVD risk was calculated as per the 2019 American College of Cardiology (ACC) and American Heart Association (AHA) clinical guidelines59. Additional data were collected over the subsequent 13-day period at home; postprandial responses to eight standardized meals (seven in duplicate) of differing macronutrient (fat, carbohydrate, protein and fibre) content were measured using CGMs and dried-blood-spot analysis as described previously13. T2D and hyperlipidemia were self-reported via health questionnaires. Sex, age, height, weight and blood pressure were self-reported, and fasting and postprandial responses for total cholesterol, HDL-C, triglycerides and HbA1c were assessed using whole blood finger-prick samples collected at home using dried-blood-spot analysis by commercial laboratories (CRL, Eurofins Biomnis). CGMs were fitted at home by participants. A selection of standardized meals smaller than in PREDICT 1 was tested in PREDICT 2 and PREDICT 3 (a metabolic challenge meal, and medium-fat and carbohydrate breakfast and lunch meals). Some of the considered markers represent the same metabolic function over time and showed positive correlations between their fasting and postprandial measurements, whereas others represent opposite types of the same biomolecular pathway and showed negative correlations among them (Supplementary Table 4). We leveraged 27,011 public metagenomic samples from 107 cohorts available through the curated MetagenomicData 3 (cMD3) resource60,61 to define the cohorts used for the meta-analyses on BMI and healthy–diseased comparison (‘Statistical and meta-analyses'). For the meta-analysis on BMI, we selected cohorts with stool microbiome samples from healthy participants (self-assessed, not reporting a diagnosis), aged at least 16 years, BMI ≥ 18.5 and sex information available. Cohorts with fewer than 30 people were excluded. Furthermore, the ThomasAM_2018_c and LeChatelier_2013 cohorts were excluded as duplicates in the YachidaS_2019 and NiesenHB_2014 cohorts, respectively. Overall, 6,182 samples from 34 different cohorts and 20 countries were retrieved. Then, each combination of country, dataset and two BMI categories was tested if at least 15 samples were retained. These led to analysing a total of 5,348 samples from 27 cohorts (2,837 healthy weight, 1,562 overweight and 949 obese participants; Supplementary Table 9). For the health–diseased meta-analyses, we selected from cMD3 participants aged at least 16 years, BMI ≥ 18.5 and the sex information available that were part of a case–control study of one of the following diseases: CRC, IBD (including ulcerative colitis and Crohn's disease), T2D, IGT and ASCVD. Studies with fewer than 30 people were excluded. In total, we considered ten datasets of CRC (650 cases and 645 controls), two datasets of IGT (273 cases and 492 controls), five datasets of T2D (775 cases and 900 controls), three datasets of IBD (103 controls, 59 of which used in two different comparisons, 60 individuals with Crohn's disease and 68 individuals with ulcerative colitis) and three datasets of CVD (283 cases and 508 controls). Notably, German and French participants of the MetaCardis cohort were separated, and this led to a set of 449 controls used in both the T2D and the IGT analyses, whereas only the 176 controls from France were used in the CVD analysis. Overall, the total number of samples analysed was N = 4,816 (2,707 controls and 2,109 cases) from 25 cohorts and 10 countries (Supplementary Table 20). All microbiome samples from the PREDICT cohorts were profiled using MetaPhlAn 4 (v.4.beta.2, database vJan21_CHOCOPhlAnSGB_202103), without performing read subsampling, as the benefit of occasionally detecting a few additional low-abundance species in samples with a higher number of reads outweighs the potential noise from uneven sequencing depth. Samples retrieved from cMD3 (described in ‘Public human microbiome datasets') were profiled with MetaPhlAn 4 (v.4.beta.1, database vJan21_CHOCOPhlAnSGB_202103) using default parameters in both cases (among default parameters, the stat_q is set to 0.2 by default, which defines the quantiles for the robust average coverage calculation), which precludes the necessity for additional prevalence filters considering its default parameters are tailored for the taxonomic profiling of human microbiome samples19. MetaPhlAn 4 is a publicly available taxonomic profiler for metagenomic samples (Github repository: https://github.com/biobakery/MetaPhlAn) that leverages medium and high-quality genomes from isolates and metagenome-assembled genomes (MAGs). Isolate genomes and MAGs are clustered at 95% average nucleotide identity to define SGBs, as described previously20. If an SGB cluster contains a genome isolate, then it is referred to by that isolate's taxonomic label. If an SGB contains only MAGs, then it represents an unknown species cluster and is assigned the taxonomic label of a genus, family or phylum, according to which is the genomically closest to a taxonomic label from isolate genomes. As the taxonomic classification of MetaPhlAn depends on species-specific marker genes, sometimes there are several SGBs of very closely related genomes for which the identification of SGB-specific markers is not feasible. In this case, more than one SGB can be considered together, and the label ‘_group' is appended to the representative SGB ID. We first identified a subset of prevalent SGBs to ensure a minimum number of non-zero relative abundance values. In each PREDICT cohort, we selected markers that are intermediary measures of host health or diet health, and they were organized into four categories: personal, dietary, fasting and postprandial (Supplementary Table 2). Second, we calculated the partial Spearman's correlation between each SGB and health markers, adjusting for sex, age and BMI, using the ‘pingouin' Python package (v.0.5.4, https://github.com/raphaelvallat/pingouin) (Extended Data Figs. The relative abundance values of SGBs (including zeros) were used as input for the correlations. Third, the SGB-marker partial correlations were sorted ascending if the marker was considered as positive with respect to health, or descending if the marker was considered as negative. These sorted partial correlations were ranked and normalized according to cohort sample sizes into percentiles ranging from 0 to 1 (function pandas.DataFrame.rank with param pct=True from pandas v.2.1.3) (Fig. 2b,c and Extended Data Figs. Fourth, for each category of markers, we computed the average percentiles across markers (Fig. SGBs were retained in the overall rankings if they were ranked in at least two different cohorts, leading to a final ranking of 661 SGBs. Finally, the ZOE MB health-rank 2025 was defined by first averaging the personal, fasting and postprandial category percentiles within each cohort, and then averaging these cohort-specific averages. The ZOE MB diet-rank 2025 instead was defined by averaging the dietary percentiles across all cohorts (Fig. The ZOE Microbiome Rankings are also available at https://zoe.com/our-science/microbiome-ranking. To assess the link to the human gut microbiome composition, we developed and used a machine learning framework based on random forest classification and regression algorithms from the scikit-learn (v.1.3.2) Python package (as implemented in the RandomForestClassifier and RandomForestRegressor functions, respectively), both with ‘n_estimators=1000' and ‘max_features=sqrt' parameters63. We trained random forest classifiers and regressors on MetaPhlAn 4-estimated SGB-level relative abundances (arcsine square-root transformed) to assess the extent to which the outcome variable was predictable from the microbiome as a proxy of the strength of the microbiome–variable association. This framework was used and described originally in ref. 9 and accounts for the presence of twin pairs in the data, which avoids biases due to identical values in twins. In brief, the framework uses a cross-validation approach, splitting the dataset randomly into training and testing folds with an 80:20 ratio, respectively, and repeated 100 times (as implemented in the StratifiedShuffleSplit function). Folds are also constructed to maintain a similar ratio of the two classes to predict as they appear in the full data. We performed a meta-analysis to determine the possible links between BMI (categorized into ‘healthy weight', ‘overweight' and ‘obese') and our ranked SGBs across various publicly available studies comprising a total of 5,348 people who were not diagnosed with any specific disease. We first evaluated the ZOE MB health- and diet-ranks by assessing the cumulative relative abundance and richness of the 50 most favourable and the 50 least favourable SGBs in each dataset in each BMI category: healthy weight, overweight and obese (see ‘Public human microbiome datasets' for the specific cut-offs). Specifically, we assessed the number of intra-dataset, between-BMI groups pairwise comparisons in which the group median abundance or the group median count was higher in the lower BMI group (when considering most favourable SGBs from both ranks) or higher in the higher BMI group (when looking at least favourable SGBs). Next, we fit linear models for each dataset and pair of BMI categories: healthy weight versus overweight, healthy weight versus obese, and overweight versus obese. In the first model, we looked at the count of the 50 most favourable and unfavourable ZOE MB health- and diet-ranked SGBs. A second model was fitted on the cumulative relative abundance (arcsine square-root transformed) of the 50 most favourable and unfavourable SGBs in the two rankings. All models were adjusted by sex and age. Cohen's d was used to estimate the effect size of the normalized difference between unfavourable and favourable ranked SGBs when considering cumulative abundances. This quantifies the difference between the means of two groups in terms of standard deviations. Specifically, as originally defined, a ‘small' effect size corresponds to d = 0.2, a ‘medium' effect size to d = 0.5 and a ‘large' effect size to d = 0.8 (ref. In these models, the lower BMI category of each comparison was used as the negative control, so negative coefficients reflect a higher count of SGBs in the lower BMI category, whereas positive coefficients reflect a higher count of SGBs in the higher BMI category. Effect sizes were summarized through meta-analysis, computed as a random-effect model using the Paule–Mandel heterogeneity on adjusted mean differences from the linear regression models (standardized for cumulative abundances). We assessed the presence of the 50 most favourable and most unfavourable SGBs from both the ZOE MB health- and diet-ranks among the countries considered in these analyses (18 in total) and when considering only people of healthy weight (n = 2,837). We used ordinary least squares adjusted by sequencing depth when comparing two datasets from different countries, and linear mixed model blocked by dataset ID and adjusted by sequencing depth when comparing pairs of countries in which at least one country was represented by more than one dataset (country- and sequencing depth-adjusted P values are presented in Supplementary Table 11). A second meta-analysis tested the associations between our ZOE MB health- and diet-ranked SGBs and five gut-associated diseases (CVD, T2D, IBD, CRC and IGT) across studies, for a total of 4,816 samples (‘Public human microbiome datasets'). Linear models were used to predict the binary disease outcome (healthy versus diseased) for each disease, using the cumulative abundances (arcsine square-root transformed) of the 50 most favourable or unfavourable SGBs, adjusting by sex, age and BMI. We also defined models to predict healthy versus diseased using the sum of the SGB ranks normalized between −1 and 1, considering all 661 SGBs for the ZOE MB health- and diet-ranks, once using the direct sum of the SGB ranks and once weighting ranks by the relative abundance of each SGB in each sample (transformed using the arcsine and square-root function to avoid overestimating the ranks of highly abundant species due to compositionality). SMDs were calculated similarly to those in the previous case. In all meta-analytical models, the set of cohorts considered comprised studies encompassing several diseases with a shared control group that we analysed separately. To account for the overlaps in the studies considered, we computed weights based on the inverse effect sizes variance-covariance matrix, as suggested previously66,67. Of note, in the comparisons of controls versus T2D, IGT and CVD, the MetaCardis French and German sub-cohorts were considered as different datasets, and their controls were meta-analysed as different cohorts. In particular, only French control samples were used in the CVD analysis, which included only French cases. Finally, meta-analysis summaries were computed using the same technique. Analyses we carried out with Python (v.3.12.0), using also the following libraries: numpy (v.1.26.2), scipy (v.1.11.4), statsmodels (v.0.14.0), and matplotlib (v.3.8.2) and seabron (v.0.11.2) for visualization. All study protocols are registered on clinicaltrials.gov and procedures are compliant with all relevant ethical regulations. Ethical approval for the PREDICT 1 study was obtained in the UK from the King's College London Research Ethics Committee (REC) and Integrated Research Application System (IRAS 236407), and in the USA from the institutional review board (Partners Healthcare Institutional Review Board (IRB) 2018P002078). Ethical approval for the PREDICT 2 study (Pro00033432) was obtained from Advarra IRB. Ethical approval for the PREDICT 3 study (Pro00044316, HR/DP-21/22-28300 and HR/DP-24/25-45829) was obtained from Advarra IRB and King's College London REC. Ethical approval for the METHOD study (Pro00044316; protocol no. 00044316) was obtained from Advarra IRB. Ethical approval for the BIOME study (HR/DP-23/24-39673) was obtained through King's College London REC. All participants provided written informed consent and all studies were carried out in accordance with the Declaration of Helsinki and Good Clinical Practice. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw metagenomic samples, along with metadata information (sex, age, BMI and country) and microbiome profiles for all participants of the ZOE PREDICT Studies, are publicly available. Metagenomes from PREDICT 1 are publicly available as previously reported9, whereas the PREDICT 2 and PREDICT 3 cohorts (US21, US22A and UK22A) are deposited in the European Nucleotide Archive (ENA) of the European Bioinformatics Institute (EBI) under accession numbers PRJEB75460, PRJEB75462, PRJEB75463 and PRJEB75464, and are publicly accessible. Sex, age, BMI, country and quantitative taxonomic profiles for each sample are publicly available within the curated MetagenomicData package60 and at Zenodo (https://doi.org/10.5281/zenodo.15307999)68. The full list of species for the ZOE Microbiome Rankings are publicly available at https://zoe.com/our-science/microbiome-ranking, where future updates will also be made available. To protect participant privacy, individual participant clinical data are not publicly available and cannot be deposited in public repositories. Researchers may request access to the restricted data by submitting a research proposal via email to data.papers@joinzoe.com. All proposals will be reviewed by a sub-panel of the ZOE Scientific Advisory Board within 4 working weeks. Proposals, researchers or institutions requesting data will be approved if they meet the standard criteria related to ethics, privacy and data protection regulations. Approved researchers are required to enter into a data-sharing agreement with ZOE. The requested host parameters will be provided as ordered data points without loss of reproducibility, as the analysis of this work (including deriving the ranks) was performed using non-parametric statistics. These data are available at Zenodo (https://doi.org/10.5281/zenodo.17236382)69 and are encrypted; access to the data will be granted to researchers whose proposals are approved. All data from non-PREDICT external public cohorts used to validate the rankings are available in full at Zenodo (https://doi.org/10.5281/zenodo.17236261)70. The custom Python code developed for the meta-analyses performed on public data and included in this work is available at GitHub (https://github.com/SegataLab/inverse_var_weight) and at Zenodo (https://doi.org/10.5281/zenodo.17236261)70. Global Health Estimates: Leading Causes of Death https://go.nature.com/48bFXYT (accessed 3 June 2024). Precision nutrition for cardiometabolic diseases. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Rauber, F. et al. Ultra-processed foods and excessive free sugar intake in the UK: a nationally representative cross-sectional study. Estruch, R. et al. Primary prevention of cardiovascular disease with a Mediterranean diet supplemented with extra-virgin olive oil or nuts. Yamada, T., Kimura-Koyanagi, M., Sakaguchi, K., Ogawa, W. & Tamori, Y. Obesity and risk for its comorbidities diabetes, hypertension, and dyslipidemia in Japanese individuals aged 65 years. Definition and diagnostic criteria of clinical obesity. Asnicar, F. et al. Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals. Gut metagenome in European women with normal, impaired and diabetic glucose control. The gut microbiome modulates the protective association between a Mediterranean diet and cardiometabolic disease risk. Valles-Colomer, M. et al. Cardiometabolic health, diet and the gut microbiome: a meta-omics perspective. Berry, S. E. et al. Human postprandial responses to food and potential for precision nutrition. Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases. Response of gut microbiota to metabolite changes induced by endurance exercise. Diet rapidly and reproducibly alters the human gut microbiome. Yatsunenko, T. et al. Human gut microbiome viewed across age and geography. Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Connelly, M. A., Otvos, J. D., Shalaurova, I., Playford, M. P. & Mehta, N. N. GlycA, a novel biomarker of systemic inflammation and cardiovascular disease risk. Asnicar, F., Thomas, A. M., Passerini, A., Waldron, L. & Segata, N. Machine learning for microbiologists. Krebs-Smith, S. M. et al. Update of the healthy eating index: HEI-2015. Satija, A. et al. Plant-based dietary patterns and incidence of type 2 diabetes in US men and women: results from three prospective cohort studies. Yassour, M. et al. Sub-clinical detection of gut microbial biomarkers of obesity and type 2 diabetes. Zhai, L. et al. Ruminococcus gnavus plays a pathogenic role in diarrhea-predominant irritable bowel syndrome by increasing serotonin biosynthesis. Gut microbiota composition in patients with newly diagnosed bipolar disorder and their unaffected first-degree relatives. Lloyd-Price, J. et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. Gut microbiome structure and metabolic activity in inflammatory bowel disease. Increased fecal ethanol and enriched ethanol-producing gut bacteria Limosilactobacillus fermentum, Enterocloster bolteae, Mediterraneibacter gnavus and Streptococcus mutans in nonalcoholic steatohepatitis. Caroline de Oliveira Melo, N. et al. Oral administration of viable or heat-inactivated Lacticaseibacillus rhamnosus GG influences on metabolic outcomes and gut microbiota in rodents fed a high-fat high-fructose diet. Gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes. Petzoldt, D., Breves, G., Rautenschlein, S. & Taras, D. Harryflintia acetispora gen. nov., sp. Koh, A., De Vadder, F., Kovatcheva-Datchary, P. & Bäckhed, F. From dietary fiber to host physiology: short-chain fatty acids as key bacterial metabolites. Carrasco, J. L. & Jover, L. Estimating the generalized concordance correlation coefficient through variance components. Effects of a personalized nutrition program on cardiometabolic health: a randomized controlled trial. Creedon, A. C. et al. A diverse high-fibre plant-based dietary intervention improves gut microbiome composition, gut symptoms, energy and hunger in healthy adults: a randomised controlled trial. Gut microbiota modulation with long-chain corn bran arabinoxylan in adults with overweight and obesity is linked to an individualized temporal increase in fecal propionate. Belenguer, A. et al. Two routes of metabolic cross-feeding between Bifidobacterium adolescentis and butyrate-producing anaerobes from the human gut. Falony, G., Vlachou, A., Verbrugghe, K. & De Vuyst, L. Cross-feeding between Bifidobacterium longum BB536 and acetate-converting, butyrate-producing colon bacteria during growth on oligofructose. Duncan, S. H., Louis, P. & Flint, H. J. Lactate-utilizing bacteria, isolated from human feces, that produce butyrate as a major fermentation product. Hj, L. P. F. Diversity, metabolism and microbial ecology of butyrate-producing bacteria from the human large intestine. Gut microbiome signatures of vegan, vegetarian and omnivore diets and associated health outcomes across 21,561 individuals. Carlino, N. et al. Unexplored microbial diversity from 2,500 food metagenomes and links with the human microbiome. Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with bioBakery 3. eLife 10, e65088 (2021). Zeevi, D. et al. Personalized nutrition by prediction of glycemic responses. Gut microbiota richness promotes its stability upon increased dietary fibre intake in healthy adults. Oliver, A. et al. High-fiber, whole-food dietary intervention alters the human gut microbiome but not fecal short-chain fatty acids. Manghi, P. et al. Coffee consumption is associated with intestinal Lawsonibacter asaccharolyticus abundance and prevalence across multiple cohorts. Berry, S. et al. Personalised REsponses to DIetary Composition Trial (PREDICT): an intervention study to determine inter-individual differences in postprandial response to foods. Bingham, S. A. et al. Nutritional methods in the European Prospective Investigation of Cancer in Norfolk. Comparative validation of the Block, Willett, and National Cancer Institute food frequency questionnaires: the Eating at America's Table Study. Wennberg, M., Kastenbom, L., Eriksson, L., Winkvist, A. & Johansson, I. Validation of a digital food frequency questionnaire for the Northern Sweden Diet Database. Satija, A. et al. Healthful and unhealthful plant-based diets and the risk of coronary heart disease in US adults. Fung, T. T. et al. Diet-quality scores and plasma concentrations of markers of inflammation and endothelial dysfunction. Vadiveloo, M., Dixon, L. B., Mijanovich, T., Elbel, B. & Parekh, N. Development and evaluation of the US Healthy Food Diversity index. Roe, M., Pinchen, H., Church, S. & Finglas, P. McCance and Widdowson's The composition of Foods Seventh summary edition and updated composition of foods integrated dataset. Goff, D. C. Jr. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Accessible, curated metagenomic data through ExperimentHub. Manghi, P. et al. Meta-analysis of 22,710 human metagenomes defines an index of oral to gut microbial introgression and associations with age, sex, BMI, and diseases. Manghi, P. et al. MetaPhlAn 4 profiling of unknown species-level genome bins improves the characterization of diet-associated microbiome changes in mice. Pedregosa, F. et al. Scikit-learn: machine learning in Python. Statistical Power Analysis for the Behavioral Sciences 2nd edn (Routledge, 1988). Nakagawa, S. & Cuthill, I. C. Effect size, confidence interval and statistical significance: a practical guide for biologists. & Sullivan, P. F. Meta-analysis of genome-wide association studies with overlapping subjects. & Olkin, I. Stochastically Dependent Effect Sizes. OLK NSF 289 (Department of Statistics, Stanford Univ., 1992). Gut microbes associated with health, nutrition and dietary interventions. Asnicar, F. Gut microbes associated with health, nutrition and dietary interventions. Asnicar, F. Gut microbes associated with health, nutrition and dietary interventions. This work was supported by Zoe Ltd. and TwinsUK, which is funded by the Wellcome Trust, Medical Research Council, Versus Arthritis, European Union Horizon 2020, Chronic Disease Research Foundation (CDRF), the National Institute for Health Research (NIHR) Clinical Research Network (CRN) and Biomedical Research Centre based at Guy's and St Thomas' NHS Foundation Trust in partnership with King's College London. It was also supported by the European Research Council (ERC-CoG microTOUCH-101045015) to N.S., by the European Union NextGenerationEU (Interconnected Nord-Est Innovation programme, INEST) to N.S., by the National Cancer Institute of the National Institutes of Health (1U01CA230551) to N.S. and by the Premio Internazionale Lombardia e Ricerca 2019 to N.S. Present address: Research and Innovation Center, Fondazione Edmund Mach, San Michele all'Adige, Italy These authors contributed equally: Sarah E. Berry, Tim D. Spector, Nicola Segata Francesco Asnicar, Paolo Manghi, Gloria Fackelmann, Gabriel Baldanzi, Liviana Ricci, Gianmarco Piccinno, Elisa Piperni, Katarina Mladenovic & Nicola Segata Elco Bakker, Federica Amati, Alberto Arrè, Sajaysurya Ganesh, Francesca Giordano, Richard Davies, Jonathan Wolf, Kate M. Bermingham, Sarah E. Berry & Tim D. Spector IEO, Istituto Europeo di Oncologia IRCSS, Milan, Italy Federica Amati, Tim D. Spector & Nicola Segata Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and supervized the study. and A.A. performed the analyses: F. Asnicar collected the data, performed microbiome analyses and species rankings; P.M. performed meta-analyses on public data; E.P. supported the analysis of longitudinal data; K.M. helped with microbiome profiling; A.A. supported the analysis of the two clinical trial, nutritional intervention studies. set up the interface to retrieve clinical data information. is co-founder of ZOE Ltd. and made the assembly of these large cohorts possible. All authors reviewed and edited the manuscript. Correspondence to Francesco Asnicar or Nicola Segata. are, or have been, employees of Zoe Ltd. F. Asnicar, S.E.B., T.D.S. are consultants to ZOE Ltd. F. Asnicar, R.D., J.W., S.E.B., T.D.S. receive options with ZOE Ltd. All other authors declare no competing interests. Zoe Ltd. holds the following patent applications on the SGBs ranking: PCT (World) patent pending applications PCT/EP2024/058262, PCT/EP2024/058286 and PCT/EP2024/058290. Nature thanks Johanna M. Geleijnse, Reiner Jumpertz-von Schwartzenberg, Matthew Olm, Daniel Tancredi and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Distributions of the random forest median AUCs (a) and median Spearman's correlation coefficients (b) (Methods) in the five cross-sectional PREDICT studies for the different clinical data divided into four categories: ‘Personal', ‘Dietary', ‘Fasting', and ‘Postprandial'. The AUC and Spearman's index thresholds of 0.7 and 0.3, respectively, are indicated with a dashed line. The single-marker percentiles, divided into the three categories (‘Personal', ‘Fasting', and ‘Postprandial') for the 15 most favorable and unfavorable ZOE MB Health-ranked SGBs the other three PREDICT cohorts not reported in Fig. Heatmaps with the single Spearman's partial correlations for all PREDICT cohorts are available in Supplementary Fig. a-e) Spearman's partial correlations (corrected for age, sex, and BMI) between SGB relative abundance and single marker values show consistency across the five PREDICT cohorts. These partial correlations were ranked and averaged first within and then across the three data categories (‘Personal', ‘Fasting', and ‘Postprandial', reported in Supplementary Fig. The cohorts' averages were then used to define the cardiometabolic rank (for those SGBs analyzed in at least two cohorts). a-e) For each PREDICT cohort, we computed Spearman's partial correlation between the SGBs' relative abundances and different diet indexes. Associations were ranked and averaged in each cohort separately. f) The ZOE MB Diet-ranking was computed for SGBs ranked in at least two PREDICT cohorts. The raw Spearman's partial correlations are available in Supplementary Fig. a-e) Study-wise Spearman's partial correlation coefficients (corrected for sex, age, and BMI) for the 15 most favorable and unfavorable ZOE MB Diet-ranked SGBs in different diet indexes. The associations appear consistent across cohorts. a) The ZOE MB Health and Diet ranks are overall in agreement (Spearman's correlation = 0.72), albeit some SGBs show discordant rankings (absolute difference between the two ranks ≥ 0.3). These SGBs are highlighted in orange, and their ranks and taxonomy assignment are reported in Supplementary Table 6. b,c) Comparison of the ZOE MB Health (b) and Diet (c) ranks computed only on the PREDICT UK and US cohorts (Spearman's correlations of 0.61 and 0.26, respectively). The top and right-side histograms depict the x and y-axis marginal distributions in each plot. a) Comparison of the ZOE MB Diet-ranks (x-axis) with the Spearman's partial correlations (corrected for sex and age, y-axis) for the 661 ranked SGBs in the five PREDICT cohorts. b) The number of the 50 most-favorably ranked SGBs (ZOE MB Health-rank, Richness) detected in different BMI categories, showed that increasing BMI, linked with increasing health risks, is reflected by a lower presence of favorable SGBs. On the other hand, c) unfavorably-ranked SGBs show an increasing count in higher-risk BMI categories. d,e) The box plots report the number of the 50 most favorable and unfavorable ZOE MB Diet-ranked SGBs of individuals stratified into three BMI categories (healthy-weight, overweight, and obese) in each PREDICT cohort. f,g) Similarly, the box plots represent the cumulative relative abundance of the 50 most favorable and unfavorable ZOE MB Diet-ranked SGBs in individuals categorized into the three BMI categories in each cohort. h,i) The box plots report the number of the 50 most favorably and most unfavorably ranked SGBs, ranked using the same markers and categories as in the ZOE MB Health-ranks (Methods), but partial correlations were corrected only for sex and age. j,k) Similarly, the box plots report the count of the 50 most favorable and unfavorable SGBs in the three BMI categories, with SGBs ranked according to their partial correlation with BMI, adjusted by sex and age. Only non-significant FDR-corrected P values (ns, P value > 0.01) from the Mann-Whitney U test are reported. a) Overweight individuals tend to carry a higher number of the 50 most favorably ZOE MB Health-ranked SGBs than obese individuals (left); the 50 most unfavorably ranked SGBs are increased in obese individuals vs overweight individuals (Methods). b) Healthy-weight individuals tend to carry a higher number of the 50 most favorably ZOE MB Health-ranked SGBs than overweight individuals (left); the 50 most unfavorably ranked SGBs are found in similar amounts in healthy-weight and overweight individuals (Methods). Error bars represent the 95% confidence interval. Healthy-weight and overweight individuals tend to have a higher number of favorably-ranked SGBs than obese individuals (Methods). Obese individuals tend to have a higher number of unfavorably-ranked SGBs (Methods). Error bars represent the 95% confidence interval. 4a) and d) of the METHOD cohort (relative to Fig. SGBs are separated into “increasing” and “decreasing”, depending on their trend in relative abundance values, showing that SGBs found to be increased in relative abundance are also more prevalent, while the opposite is observed for SGBs decreasing in relative abundance. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Asnicar, F., Manghi, P., Fackelmann, G. et al. Gut micro-organisms associated with health, nutrition and dietary interventions. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69622331/upside-down-crown/'>Archaeologists Opened an Ancient Tomb—and Found a Story of Royal Tragedy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A crown was left with the deceased, but its position indicated her reign may not have ended the way she would have liked. According to a translated statement from the Greek Ministry of Culture, a team of archaeologists found a cemetery in east-central Greece, about 60 miles northwest of Athens, full of tombs dated from about 800 to 320 B.C.E., during the Archaic and Classical periods. A preliminary examination of dental remains showed she was an adult woman between 20 and 30 years old. The striking part of the find, though, was a bronze-banded diadem (or crown) ceremoniously placed on her head upside down as a symbol of her superiority or rank. While lions symbolized royal power and authority, when the crown was flipped upside down, the lions were seen lying down. That superiority of rank and high class may not have lasted beyond the death of the noblewoman. marked a particularly tumultuous time for any monarchs in Greece, as the traditional hereditary kingdom regimes and the rise of nobles had started to give way to aristocratic regimes and the movement to Athenian democracy by the early sixth century B.C.E. The grave goods buried with the young woman featured plenty of bronze items, including two oversized pins with engraved geometric-style horses, a necklace with a pendant, bone and ivory beads, amber tokens, and bronze earrings. She was buried wearing bracelets and rings. In the same cluster of tombs, the team found a young girl, likely around 4 years old, crowned with a similar bronze diadem with inlaid rosettes. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Say a Physical Warp Drive Is Possible Divers Couldn't Believe a Lost Shipwreck's Age Archaeologists Found a Medieval Knight's Odd Skull Tiny Humans Are Hiding in Indonesia, Scientist Claims A Whole New Cat Color Has Emerged Scientists Just Discovered a New Law of Physics</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69597133/lost-observatory-oldest-in-the-americas/'>A Lost Observatory Just Emerged From the Desert—And It May Be the Oldest in the Americas</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Studying the sun, moon, and stars was obviously a more complex endeavor in the ancient world than it is now, but thanks to a fresh discovery at an already famous astrological site on the north-central coast of Peru, we have a better idea of just how complex. and is designated as a UNESCO World Heritage Site. “It is thus a testimony to the culmination of a long historical evolution of astronomical practices in the Casma Valley.” The dual purpose of both solar and lunar observations—lunar observation is considered a trickier endeavor than solar tracking—signify a more intense astronomical understanding than previously believed. Archaeologists also located a Patazca-style ceremonial vessel within the new find. The vessel, about three feet tall with clay figures of warriors in combat positions, suggests that cultural elites were involved with goings-on at the complex, perhaps combined astronomical knowledge with military leadership. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Say a Physical Warp Drive Is Possible Divers Couldn't Believe a Lost Shipwreck's Age Tiny Humans Are Hiding in Indonesia, Scientist Claims Scientists Just Discovered a New Law of Physics</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69649788/china-hanging-coffins/'>Ancient Cliffside Coffins Held the Remains of a Lost People. DNA Evidence Says They're Still Here.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The modern-day Bo people of southeastern China have long maintained that they had no connection to those inside the hanging coffins. We may earn commission if you buy from a link. An unexpected sight can be seen high on sheer rock walls in parts of southern China and Southeast Asia. What may look like rock protrusions are actually wooden coffins, hewn from tree trunks and hung on the cliffside by peoples who were thought to have vanished hundreds of years ago. Hanging coffins are a funerary custom once practiced by the Bo people of Southwest China. Shrouded in mystery, this ancient ethnic group was often hidden from the outside world. ), it was written in The Brief Chronicles of Yunnan that “Coffins set high are considered auspicious. Many of these coffins have remained suspended for hundreds or thousands of years, partly because those who laid them to rest made sure to support them with wooden stakes wedged into cracks and holes in the rock. Hanging coffins were widespread in the region during ancient times, but eventually faded out. While they thrived until about 400 years ago, a small Bo community still survives in the mountains of southeastern Yunnan Province. “Although the hanging coffin practice ceased to appear in historical records, the genetic traces left behind provide compelling evidence of a shared origin and cultural continuity that transcends modern national boundaries,” a team of researchers said in a study recently published in Nature Communications. “From this region, the practice spread to other parts of China, eventually moving southward and westward across various cultural zones.” While modern Bo do not believe they are related to the people who were once thought to be capable of flight because of their mortuary practices, the researchers analyzed samples of human remains from individuals found at Hanging Coffin sites in Yunnan, Guangxi, and Thailand. They then sequenced the genomes of the living Bo population in She De Village and compared their findings to see whether there was any genetic connection. Most of the individuals from Southwest China were genetically similar, with ancestry originating on the coast of southeastern China. After the team viewed their findings from anthropological and archaeological perspectives, Hanging Coffins are now thought to have first appeared among coastal populations in southeastern China around 3,000 years ago. Coffins of similar age, dated to about 1,200 years ago, also showed evidence of a population shift from southern China to northwestern Thailand, supporting these findings. Some ancient individuals from Hanging Coffins also had Northeast Asian ancestry. “Further exploration with additional human remains and archaeological content from these regions, incorporating interdisciplinary scientific perspectives, could contribute to a more comprehensive understanding of the history of suspended wooden coffin burial customs in the future,” the researchers said. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Tiny Humans Are Hiding in Indonesia, Scientist Claims</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/can-nasa-bring-mars-rocks-back-to-earth/'>NASA Struggles to Bring Perseverance's Mars Discoveries Back Home</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA's Perseverance Rover Faces New Uncertainty on Mars NASA's Perseverance rover has gathered groundbreaking Mars samples, but the mission to bring them home is facing serious challenges. By Kendra Pierre-Louis, Lee Billings, Fonda Mwangi & Alex Sugiura In July of 2020, NASA engineers sent a rover named Perseverance hurtling into space. And in pictures it kind of looked like the diminutive robot in the Disney-Pixar film WALL-E—just much, much larger. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. But that project, known as Mars Sample Return, is hanging on by a thread. To walk us through what's happening we are joined today by Lee Billings, a senior desk editor here at SciAm. Lee Billings: Kendra, it's great to be here, as always. Pierre-Louis: You know, I think to begin can you tell us broadly about the mission that Perseverance was tasked with completing on Mars? And it was a follow-up to Curiosity, another NASA Mars rover. But the point is, is that Perseverance's goal, it's kind of the apex of planning for something called Mars Sample Return, which has been going on for decades ... And the whole core idea is: we can study rocks better here on Earth than we can on Mars because it's hard to lift or build a huge wet lab on another planet ... Billings: It's easier to bring smaller little bits of stuff back here. And so Perseverance was crucial to that—in, again, kind of a very incremental way—building on a lot of the missions that had come before, in that Perseverance is actually kind of where the rubber meets the road, so to speak, where NASA scientists and other scientists from other institutions had narrowed down a short list of places to go on Mars to look for potential signs of life. And then from that short list they actually found a place, called Jezero Crater, this 30-mile-wide—almost 30-mile-wide depression in Mars that used to harbor an ancient lake and river delta and all this stuff. And that's where Perseverance landed—had a rather nail-biting landing with the Sky Crane from JPL [Jet Propulsion Laboratory] that involved rockets and wires [delivering] this thing down to the surface. And it's been there, again, for the past several years, gathering samples, using a lot of its onboard instrumentation to figure out which of the rocks, which of the parts of the environment around there are most interesting to bring back to Earth. And it has 43, I believe, sample tubes—that's how many it carries—and the idea is that, in each one of these sample tubes, it can deposit a little sample, a little bit of Mars, to bring back to Earth someday. Billings: So the big question, of course, is: “Are we alone?” And Mars is right next door, relatively speaking, and it has a lot of indications that suggest that it really wasn't so bad for life as we know it billions of years ago. If you think about the Mars of four or three billion years ago, this was a time when we see abundant signs on the planet of flowing water, of thicker atmosphere. Right now, let's be clear, Mars is a frozen, terrible place to live. Pierre-Louis: It feels like you're describing my apartment. Or parts of my house, for that matter. It's not a good place to live, despite what some people might say. But long ago you could imagine it actually not being so bad of a place to live, to the degree that we really don't necessarily know of someplace that has a relatively Earth-like—or Earth-adjacent, we might call [it]—environment in which we could imagine life as we know it being able to exist. And of course, “life as we know it” doesn't just include podcasters talking to each other into microphones but also little, tiny, microscopic single-celled organisms, microbes, which is really what most people would think would be on Mars rather than, you know, little green men or some kind of strange polar bear that's gonna eat you. And so the, the core idea really driving all this is trying to figure out whether or not we're alone. If we can look at the planet right next door to us and we can see that it was a true so-called independent origin or second genesis that would be huge. There is this one sample, or this one singular rock, that Perseverance has found on Mars: a big hunk of mudstone on the rim of this crater that used to harbor, billions of years ago, a big lake and, and riverine delta. [The rock is] called “Cheyava Falls,” and it's filled with organic matter. That's a pretty important thing because where you see organics on Earth, you tend to see life. It also displays very tantalizing patterns on its surface and in the matrix of the rock itself; they're called “poppy seeds” and “leopard spots.” And in short we see rocks that have those same types of features on Earth, and they can be produced abiotically, through lifeless processes over lots of time via a little bit of heating and some chemistry, but most of the time, when we see them on Earth, they're due to little microbes, little bugs in the rock, that use iron minerals to get energy to drive their metabolism. And so based on everything that Perseverance has seen of this rock and has studied from this rock it looks like the rock formed in conditions that are closer to what we'd expect for life to be there. So it's not something that happened via volcanic eruption or some sudden event like that it; it's rather like, okay, this mudstone was deposited on the bottom of this ancient lake billions of years ago and sat there, and at some point, somehow, these really curious marks and speckles got all over it. So the evidence for life beyond Earth may already literally be in hand—almost. All we need to do is go there and get it. So that is the big thing, but it's not the only thing—a lot of people get hung up on this. Even if Mars Sample Return reveals no life at this site or no life anywhere on the planet after we've exhaustively looked everywhere somehow, we would still learn so much about Mars, and that's important because Mars is kind of giving us an idea of the limits of habitability that we will potentially someday encounter on our own planet. And studying the mechanics of that and knowing how that happened is valuable for being good stewards of our own planet. Because of course, we've never launched anything from Mars before, and that's how you have to get the samples back to Earth. I mean, you know, a billion here, a billion there pretty much adds up to real money. But obviously, that is a lot of money. What happened was that this plan, it involved first retrieving the samples with a second rover that goes there to retrieve the samples and then that rover bringing the samples from Perseverance, or from a cache where they've been dropped on the ground, back to a launch site or maybe just launching from that same platform; launching it up into orbit, where it would rendezvous with an orbiting spacecraft that would take in the sample, store it, secure it, and then blast it back to Earth, on a long journey back to Earth, where it would then, you know, land somewhere in the U.S. high desert, probably, of the Southwest U.S., and be collected and taken to specialized facilities that are all biohazarded up, right? You think of, like, The Hot Zone or other things, you know, the place that they go to study Ebola, stuff like that, that's the same kind of facility they would take this stuff to, just 'cause we don't really know for sure. An analysis that was done that basically showed it was behind schedule and over cost, and the costs were looking to be upwards of $11 billion, so, you know, nearly a double in the estimated price tag. And the former administrator of NASA, former senator and shuttle astronaut Bill Nelson, outgoing from the Biden administration, said, “You know, that's just too long to wait. We gotta find a faster, better, cheaper way to do it.” But he left the core decision-making on that to the Trump administration. Why would it take 15 years for the samples to get back? Billings: Well, one thing to think about is that planets revolve around the sun—“Oh, well, of course they do.” But what that means is that they don't necessarily all revolve at the same rate, in lockstep with each other. About every two years you have an alignment between Earth and Mars that allows us to get there more easily, with less fuel from a rocket. You know, everything you take off the Earth into orbit costs money to get up there, and it's a pretty exorbitant price, even though the price is falling. So every two years is one thing to think about. The other thing to think about is that we're still talking about huge pieces of hardware that are either being built or not fully built. So that includes things like, you know, the rover that's going to retrieve the samples from Mars, from wherever Perseverance has stored them or wherever Perseverance is. It may include, you know, the Earth-return vehicle—all these components. It even includes the facilities on the Earth, you know, biohazard-style facilities that I mentioned earlier, that are needed to make people feel safe to have this stuff on our planet. And it's the kind of thing where if you threw money at it in some kind of Apollo program “let's beat the Russians to the moon”–style race, you could do it much faster, yes. NASA hasn't been that for a long time, so singularly focused. It's also the broader problem that because NASA's different, because it has so much more stuff in its portfolio, there was and is a real risk that throwing too much money at Mars Sample Return would come at the very direct cost of much less money for pretty much everything else, for instance, in planetary science. And there's so many cool things out there in the solar system to see, whether you're thinking about the moon of Jupiter Europa or various moons around Saturn, for instance, or even Venus, that it's a pretty hard ask, I think, to say, “Well, let's just put all our eggs in the Mars basket.” Pierre-Louis: So my understanding is the Biden administration, when they were like, “We're gonna look for alternative methods,” they were really looking towards commercial space. And can we talk a little bit about kind of the push for Mars from the commercial side? The rise of commercial space, the new era that we are in and advancing into, was a key factor, I think, in the political decision-making that happened that brought us to this crossroads moment for Mars Sample Return because it's undeniable that the rates of launches are soaring and there are more players, more capabilities than ever before, more competition than ever before, more ways to potentially get this done. And not all of them necessarily have to rely on tried-and-true legacy ways that have certain price tags and costs associated with them. So for instance, people love to talk about SpaceX, and most of the focus is on how SpaceX might help NASA astronauts return to the moon as part of the Artemis program, and the cornerstone of that is this huge, immensely gigantic megarocket that's intended to be fully reusable, which is the first time in history we've ever tried to do that, really, called Starship, that they keep testing and that it keeps blowing up or falling apart, right? So one idea, for instance, was: maybe we can just, if Starship works out really well getting people to the moon, we can just co-opt one of those Starships and send it to Mars and plop it down, and it can do all sorts of things. But SpaceX isn't the only game in town, right? So Blue Origin, the company from Jeff Bezos of Amazon fame, is also a launch provider. Another example would be a dark horse, in that most people don't really know about it as much, is the company called Rocket Lab. The plan from Rocket Lab that they've publicized claims to be able to deliver the samples from Perseverance for a price of about $4 billion, rather than something like $6 billion or $11 billion or more. And we don't have those numbers for all the proposals, and there's more proposals than I'm mentioning from other commercial outfits. But the point is is that there is a very strong, compelling case to be made, I think, that we can indeed do much of this, if not all of this, for much cheaper than what was previously the plan of record. Don't forget to tune in on Friday, when our associate books editor, Bri Kane, digs into whether we should be thinking about AI in terms of empires and colonialism with Karen Hao. Science Quickly is produced by me, Kendra Pierre-Louis, along with Fonda Mwangi and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. She has worked for Gimlet, the Bloomberg and Popular Science. Pierre-Louis is based in New York City. He is author of a critically acclaimed book, Five Billion Years of Solitude: The Search for Life Among the Stars, which in 2014 won a Science Communication Award from the American Institute of Physics. In addition to his work for Scientific American, Billings's writing has appeared in the New York Times, the Wall Street Journal, the Boston Globe, Wired, New Scientist, Popular Science and many other publications. Billings joined Scientific American in 2014 and previously worked as a staff editor at SEED magazine. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, the Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251209234139.htm'>Scientists reveal a tiny brain chip that streams thoughts in real time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-10 04:56:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new brain implant could significantly reshape how people interact with computers while offering new treatment possibilities for conditions such as epilepsy, spinal cord injury, ALS, stroke, and blindness. By creating a minimally invasive, high-throughput communication path to the brain, it has the potential to support seizure control and help restore motor, speech, and visual abilities. This chip forms a wireless, high-bandwidth link between the brain and external computers. Shepard worked closely with senior and co-corresponding author Andreas S. Tolias, PhD, professor at the Byers Eye Institute at Stanford University and co-founding director of the Enigma Project. Tolias's extensive experience training AI systems on large-scale neural recordings, including those collected with BISC, helped the team analyze how well the implant could decode brain activity. "BISC turns the cortical surface into an effective portal, delivering high-bandwidth, minimally invasive read-write communication with AI and external devices," Tolias says. "Its single-chip scalability paves the way for adaptive neuroprosthetics and brain-AI interfaces to treat many neuropsychiatric disorders, such as epilepsy." Dr. Brett Youngerman, assistant professor of neurological surgery at Columbia University and neurosurgeon at NewYork-Presbyterian/Columbia University Irving Medical Center, served as the project's main clinical collaborator. BISC surpasses previous technology on both fronts," Youngerman adds. "Semiconductor technology has made this possible, allowing the computing power of room-sized computers to now fit in your pocket," Shepard says. "We are now doing the same for medical implantables, allowing complex electronics to exist in the body while taking up almost no space." Current medical-grade BCIs typically rely on multiple separate microelectronic components, such as amplifiers, data converters, and radio transmitters. The entire system resides on a single complementary metal-oxide-semiconductor (CMOS) integrated circuit that has been thinned to 50 μm and occupies less than 1/1000th the volume of a standard implant. The chip integrates a radio transceiver, a wireless power circuit, digital control electronics, power management, data converters, and the analog components necessary for both recording and stimulation. The external relay station provides power and data communication through a custom ultrawideband radio link that reaches 100 Mbps, a throughput at least 100 times higher than any other wireless BCI currently available. The high-bandwidth recording demonstrated in this study allows brain signals to be processed by advanced machine-learning and deep-learning algorithms, which can interpret complex intentions, perceptual experiences, and brain states. "By integrating everything on one piece of silicon, we've shown how brain interfaces can become smaller, safer, and dramatically more powerful," Shepard says. The BISC implant was fabricated using TSMC's 0.13-μm Bipolar-CMOS-DMOS (BCD) technology. This fabrication method combines three semiconductor technologies into one chip to produce mixed-signal integrated circuits (ICs). They developed surgical procedures to place the thin implant safely in a preclinical model and confirmed that the device produced high-quality, stable recordings. "These initial studies give us invaluable data about how the device performs in a real surgical setting," Youngerman says. "The extreme miniaturization by BISC is very exciting as a platform for new generations of implantable technologies that also interface with the brain with other modalities such as light and sound," Pesaran says. To move the technology closer to practical use, researchers at Columbia and Stanford created Kampto Neurotech, a startup founded by Columbia electrical engineering alumnus Dr. Nanyu Zeng, one of the project's lead engineers. "This is a fundamentally different way of building BCI devices," Zeng says. As artificial intelligence continues to advance, BCIs are gaining momentum both for restoring lost abilities in people with neurological disorders and for potential future applications that enhance normal function through direct brain-to-computer communication. "By combining ultra-high resolution neural recording with fully wireless operation, and pairing that with advanced decoding and stimulation algorithms, we are moving toward a future where the brain and AI systems can interact seamlessly -- not just for research, but for human benefit," Shepard says. Materials provided by Columbia University School of Engineering and Applied Science. Note: Content may be edited for style and length. Gravitational Waves Expose Hidden Dark Matter Around Black Holes Scientists Find a Hidden Brain Switch That Makes Habits Form Fast New Discovery Exposes the Hidden Weak Spot Cancer Uses to Survive DNA Damage Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            