
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-22</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09655-y'>SARS-CoV-2 mRNA vaccines sensitize tumours to immune checkpoint blockade</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 15:36:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Immune checkpoint inhibitors (ICIs) extend survival in many patients with cancer but are ineffective in patients without pre-existing immunity1,2,3,4,5,6,7,8,9. Although personalized mRNA cancer vaccines sensitize tumours to ICIs by directing immune attacks against preselected antigens, personalized vaccines are limited by complex and time-intensive manufacturing processes10,11,12,13,14. Here we show that mRNA vaccines targeting SARS-CoV-2 also sensitize tumours to ICIs. In preclinical models, SARS-CoV-2 mRNA vaccines led to a substantial increase in type I interferon, enabling innate immune cells to prime CD8+ T cells that target tumour-associated antigens. Concomitant ICI treatment is required for maximal efficacy in immunologically cold tumours, which respond by increasing PD-L1 expression. Moreover, receipt of SARS-CoV-2 mRNA vaccines within 100 days of initiating ICI is associated with significantly improved median and three-year overall survival in multiple large retrospective cohorts. This benefit is similar among patients with immunologically cold tumours. Together, these results demonstrate that clinically available mRNA vaccines targeting non-tumour-related antigens are potent immune modulators capable of sensitizing tumours to ICIs. These poor responses are attributed to immunosuppressive tumour microenvironments (TMEs) characterized by tolerogenic dendritic cells (DCs), myeloid suppressor cells and regulatory T cells, and may be predicted in some histologies by low intratumoural PD-L1 before immunotherapy initiation. However, there are currently no clinically available methods to improve responses to ICI by modifying the TME. We recently reported that systemic administration of highly immunogenic mRNA nanoparticles induces a viraemia-like cytokine/chemokine response that resets the systemic and intratumoural immune milieu, sensitizing resistant tumours to ICIs10,11,12,14. Although the personalized mRNA vaccines that we and others are developing remain in clinical evaluation (NCT04573140)12, COVID-19 mRNA vaccines also induce robust stimulation of cytokine secretion15, and there are now multiple case reports of patients whose tumours spontaneously resolved after COVID-19 mRNA vaccines16,17. However, the impact of COVID-19 mRNA vaccines on immune therapy is unknown. Here we report that the innate immune response to SARS-CoV-2 spike mRNA vaccination resets the cancer immunotherapy cycle and primes adaptive immunity for synergy with ICIs. We found that receipt of a SARS-CoV-2 mRNA vaccine within 100 days of ICI initiation was associated with substantial improvements in overall survival (OS) in patients with non-small cell lung cancer (NSCLC) and melanoma. In preclinical models, we found that this effect required a surge in type-I interferon (IFN) that enhanced antigen-presenting cell (APC) priming of T cells in lymphoid organs. Although tumour cells subvert these primed responses by increasing PD-L1 expression, co-administration of ICIs sustains T cell responses and elicits epitope spreading against tumour-associated antigens. We revealed analogous response correlates for humans receiving COVID-19 mRNA vaccines, including heightened IFNα production, innate/adaptive immune activation and increases in tumour PD-L1 expression. Together, our results demonstrate that clinically available mRNA vaccines targeting non-tumour antigens are potent immune modulators capable of sensitizing tumours to ICIs. To determine whether COVID-19 mRNA vaccines were associated with improved responses to immune checkpoint blockade, we first compared the OS among a cohort of patients with stage III/IV NSCLC treated at The University of Texas MD Anderson Cancer Center (MDACC) between January 2015 and September 2022 (Supplementary Table 1a). We identified 180 patients who received a COVID mRNA vaccine within 100 days of ICI initiation, and 704 patients who were treated with ICI and did not receive a COVID vaccine (Supplementary Table 1a). After controlling for 39 covariables with Cox proportional hazards regression, including clinical stage, histology, steroid use, performance status, mutation status, comorbidities and treatment year, we found that receipt of a COVID-19 mRNA vaccine within 100 days of initiation of ICI was associated with significantly improved median OS (20.6 months versus 37.3 months) and 3 year OS (30.8% versus 55.7%, adjusted hazard ratio (HRadj = 0.51, 95% confidence interval (CI) = 0.37–0.71, P < 0.0001) (Fig. This survival advantage was similar for patients with stage III unresectable NSCLC (HRadj = 0.37, 95% CI = 0.16–0.89, P = 0.0268) (Fig. Patients who received two vaccines in the 100 days surrounding initiation of ICI experienced similar OS compared with those who received only one vaccine (Extended Data Fig. These results were also consistent when considering only those patients whose closest mRNA vaccine was within 100 days before their first ICI (Extended Data Fig. 1d), when narrowing the vaccination window to 50 instead of 100 days (Extended Data Fig. 1e), when restricting to only those patients treated during the pandemic (Extended Data Fig. 1f), after correcting for immortal time bias (Extended Data Fig. 1g) and with propensity score matching (PSM; Extended Data Fig. Patients who received a COVID-19 vaccine within 100 days of chemotherapy (a group that did not include targeted therapies owing to significant heterogeneity and limited patient numbers within different drug cohorts) but did not receive ICI had no detectable survival benefit (Extended Data Fig. Likewise, patients who received a pneumonia or influenza vaccine within 100 days of initiating ICI (Extended Data Fig. 2b–e) and those with resectable stage III tumours (Extended Data Fig. a–e, Survival for patients with NSCLC (a–c) or metastatic melanoma (d,e) treated with ICI who received a COVID-19 mRNA vaccine within 100 days of initiating ICI or did not receive a COVID-19 mRNA vaccine. P values and HRadj were calculated using two-sided Cox proportional hazards regression (Supplementary Tables 3, 5, 7, 9 and 11), including all variables that were significantly associated with survival on univariable analysis (Supplementary Tables 2, 4, 6, 8 and 10). We then repeated this analysis in a separate cohort of patients treated with their first round of ICI for metastatic melanoma, including 43 patients who received a COVID-19 mRNA vaccine within 100 days of initiating ICI and 167 who did not receive a COVID-19 vaccine (Supplementary Table 1b). After accounting for covariables, including histology, steroid use, performance status, mutation status, comorbidities and treatment year with Cox proportional hazards regression, we found that receipt of a COVID-19 mRNA vaccine within 100 days of initiating ICI was associated with substantially improved OS (median OS, 26.67 months versus unmet; 36-month OS, 44.1% versus 67.6%; HRadj = 0.37, 95% CI = 0.18–0.74, P = 0.0048) (Fig. These effects were again similar for both vaccine manufacturers (Extended Data Fig. 3a), prime and boost vaccines (Extended Data Fig. 3b), patients receiving single or multiple vaccine doses (Extended Data Fig. 3c), when limiting the analysis to only patients who received their vaccine prior to initiating ICI (Extended Data Fig. 3e,f) and when expanding the analysis to include patients on second- or third-line therapy and those without distant metastases (Extended Data Fig. Moreover, the survival advantage was magnified with PSM (OS: HR = 0.44, 95% CI = 0.18–0.77, P = 0.0063; PFS: HR = 0.46, 95% CI = 0.27–0.77, P = 0.0022) (Extended Data Fig. To demonstrate whether effects observed in humans could be modelled in animals, we recreated commercial preparations of COVID mRNA vaccines for administration to tumour-bearing animals in conjunction with ICIs. As SARS-CoV-2 spike protein has been fully sequenced, we synthesized the published mRNA construct used for the Pfizer/BioNTech vaccine (BNT162b2) and validated the fidelity of synthesis based on mRNA size (Extended Data Fig. 4a,b) and the ability to elicit neutralizing antibodies after in vivo administration to animals (Extended Data Fig. mRNA was encapsulated in lipid nanoparticles (LNPs) and met the specification range delineated for clinical preparations of BNT162b2 based on encapsulation efficiency, size distribution, polydispersity and charge18 (Extended Data Fig. 4d–g), the latter of which we found could be disproportionately affected by buffer conditions leading to a net positive charge (Extended Data Fig. We chose mouse B16F0 melanoma and Lewis lung carcinoma (LLC) as models to test immunogenicity and efficacy of spike RNA-LNP vaccines due to the effects of the vaccine in these clinical settings and because both tumour models are poorly responsive to ICIs. We found this regimen to be superior to either monotherapy alone in mice with established B16F0 tumours with tumour volumes of about 80 mm3 (Fig. In mice with established LLC, we identified metastatic lesions in the lungs of untreated mice that were similar for monotherapy but significantly reduced with the combination treatment (Extended Data Fig. We also found that starting RNA-LNPs before ICI produced similar effects relative to concomitant treatment (Extended Data Fig. We next tested whether RNA-LNPs could reduce the growth of orthotopic intrapulmonary tumours. We implanted LLC cells orthotopically and administered treatment starting on day 3. In this model, the combination of RNA-LNPs and ICI resulted in superior inhibition of tumour growth as measured by lung weights (Extended Data Fig. a, The experimental design and tumour volume of mice inoculated with B16F0 cells. b, The experimental design and tumour volume for mice inoculated with LLC cells. c, The experimental design and tumour volume measurements for mice inoculated with LLC cells. d, The experimental design and tumour volume for mice inoculated with B16F0 cells. e, IFNα plasma enzyme linked immunosorbent assay (ELISA) from B16F0-tumour-bearing mice (n = 8 per group) 24 h after one RNA-LNP vaccine (day 3). j–o, Characterization of antigen presentation among myeloid cells in tumour draining lymph nodes (tdLNs) and spleens of mice bearing B16F10-ova tumours 24 h after vaccine 2 (days 10 and 13). n indicates the number of biologically independent samples. We next sought to understand the mechanism by which RNA-LNPs targeting the spike protein mediate these antitumour effects. To do this, we used an early treatment model of B16F0 with or without cytokine-blocking antibodies. In this model, RNA-LNPs and PD-L1 blockade each provided numerical but statistically insignificant survival benefits. However, combination therapy with RNA-LNPs and PD-L1 blockade strongly inhibited tumour growth (Fig. Although we previously demonstrated a role for IFNα in the response to mRNA vaccines, recent evidence suggests a dominant role for IL-1 signalling in responses to the specific RNA-LNPs targeting the COVID-19 spike protein15. While blockade of IL-1R had no effect on tumour growth, anti-tumour responses were completely abrogated when blocking type I interferon signalling with IFNAR1 monoclonal antibodies (Fig. Moreover, direct administration of supraphysiologic doses of type I IFN recapitulated the antitumour effects (Extended Data Fig. However, stimulation of type I IFN signalling with low molecular mass (LMW) poly(I:C) did not elicit similar immunity (Extended Data Fig. These results highlight the importance of IFN in driving innate and adaptive immunity, and the ability to manipulate and reset the immune setpoint away from tolerance toward effector immune responses in the presence of ICIs using a commercially available mRNA preparation. Next, we sought to assess whether other mRNA species would elicit similar antitumour effects. We found no significant difference in antitumour activity between these two groups, suggesting that innate immune sensing of the mRNA itself is the primary driver of antitumour activity from mRNA vaccines (Extended Data Fig. As N1-methyl-pseudouridine is used in place of uridine in COVID-19 mRNA vaccines to reduce innate immune activation, we hypothesized that replacing N1-methyl-pseudouridine with uridine to activate pattern recognition receptors would provide even more robust innate immune activation and antitumour activity. As expected, replacing N1-methyl-pseudouridine with uridine in pp65 mRNA resulted in further synergy with ICI (Extended Data Fig. However, replacing N1-methyl-pseudouridine with uridine in spike mRNA provided only numerical improvement in antitumour response (Extended Data Fig. 6e,f), suggesting that innate immune sensing is a multifactorial process that may be influenced by specific mRNA constructs. We then completed additional studies to identify how RNA-LNPs elicit antitumour immunity. Previous work has established that RNA-LNPs stimulate type I IFN production by stimulating the intracellular double-stranded RNA (dsRNA) sensor MDA519. However, the mechanism by which RNA-LNPs activate this sensor is unclear. To understand this further, we first ruled out a role for dsRNA in our vaccines by measuring the level of dsRNA contamination in our manufactured ssRNA product. We found the dsRNA/ssRNA ratio in our mRNA to be 0.011% (Extended Data Fig. We then repeated our tumour growth curve incorporating an additional dsRNA-removal protocol20. With this step, we successfully removed all detectable dsRNA contamination (Extended Data Fig. Importantly, we found no change in antitumour efficacy with the complete elimination of dsRNA using this method (Extended Data Fig. Finally, we found no abrogation of type I IFN after RNA-LNP administration in mice lacking RIG-I, a critical sensor of dsRNA and activator of type I IFN response (Extended Data Fig. Together, these data suggest that dsRNA is not a major cause of the antitumour effects in our preclinical models. We found that anionic lipoplexes formulated as lipid particle aggregates (LPA) do not elicit similar antitumour effects (Extended Data Fig. Consistent with this hypothesis, we found that RNA extracted from our RNA-LNPs contained high-molecular-mass secondary structures similar in mass to dsRNA (Extended Data Fig. Combined with previous reports of MDA5 activation with RNA-LNPs but not with ssRNA alone, these data may suggest a mechanism by which encapsulation with LNPs forms high-molecular-mass secondary structures enabling activation of double-stranded sensing machinery (that is, MDA5) for induction of type I interferons. We next sought to better understand the impact of RNA-LNPs on innate immune cells. IFNα was significantly elevated in both RNA-LNP and combination groups in conjunction with other T helper 1 chemokines (Fig. After treatment with spike-encoding RNA-LNP, there was precipitous activation of APCs, including DCs, macrophages and Ly6C+ cells expressing MHC class II (MHC-II) in lymphoreticular organs, that was abrogated by IFNAR1 blockade (Fig. Although this response was dependent on interferon signalling, the magnitude of the response could not be achieved by simply administering systemic IFNα in combination with ICI (Fig. Myeloid activation extended to the tumour, where receipt of RNA-LNPs was associated with IFNAR1-dependent increases in activated Ly6C+ myeloid cells (Extended Data Fig. To determine whether increased myeloid activation corresponded to enhanced presentation of tumour antigens, we treated mice with B16F10 tumours expressing chicken ovalbumin (B16F10-OVA) at days 10 and 13 and evaluated presentation of ovalbumin on MHC-I using flow cytometry. We found that spike-encoding RNA-LNPs stimulate APCs to present tumour antigens in lymphoid organs in the presence of costimulatory molecules (Fig. Notably, presentation of tumour antigens in the presence of costimulatory molecules was particularly enriched in Ly6C+MHC-II+ cells (Fig. In addition to the precipitous activation of APCs, spike-encoding RNA-LNPs in combination with ICIs also elicited expansion of CD8+ T lymphocytes with a marked increase in activation within the effector and effector memory compartments (Fig. Concomitant with these findings, PD-1 expression was increased in T cells and effector/effector memory CD8+ cell subsets, underscoring the potent ability of combination therapy to rapidly prime T cells (Extended Data Fig. Overall, these results illustrate a role for systemic immunomodulation in the recruitment of myeloid cells to lymphoid organs for tumor antigen presentation to cognate T cells. a,b, The percentage of activated effector (a) and effector memory (b) T cells in the spleens of tumour-bearing mice on day 21 (vaccine days 3, 6, 17) (n = 5 per group). c, The percentage of tetramer+ cells of splenic CD8+ T cells collected from mice bearing B16F0 tumours on day 21 (vaccination days 14 and 17). d, The normalized percentage of AIM+ T cells (n = 5 mice per peptide; exceptions are shown in the Supplementary Information) after splenocyte co-culture with overlapping peptide pools. e,f, Representative images (e) and blinded manual counting (n = 4 tumours per group with 4 counts per tumour) (f) of PD-1+CD3+ cells by immunofluorescence 24 h after vaccine 3 (days 3, 6 and 20) from s.c. tumors of B16F0-tumour-bearing mice treated with or without anti-PD-L1. AF647, Alexa Fluor 647. g, The percentage PD-1+CD8+ cells of CD3+ T cells in tumours of B16F0-bearing mice vaccinated with RNA-LNPs (days 14 and 17). h, Pooled tetramer positivity (%) among CD8+ T cells in B16F0 tumours. i, PD-L1 expression on B16F0 tumour cells (CD45−FSC-Ahigh) isolated from mice 24 h after vaccine 3 (days 3, 6 and 17) as determined using flow cytometry. j,k, Blinded manual counting (n = 6 tumours per group with 4 counts per tumour) (j) and representative images (k) of PD-L1+ tumour cells (SOX10) by immunofluorescence 24 h after vaccine 3 (days 3, 6, 21) from B16F0-tumour-bearing mice. For f and j, the colours represent individual tumours. n values indicate biologically independent samples unless indicated otherwise. To confirm that spike RNA-LNPs were mediating the expansion of tumour reactive T cells, we isolated CD8+ cells from spleens of treated mice and stained them with tetramers targeting peptides with high predicted binding affinity from six melanoma-associated antigens: GP-100 EGSRNQDWL, GP-100 KVPRNQDWL, claudin 6 (CLDN6), survivin, WT1 and Trp2. We found that combination therapy with RNA-LNPs and ICIs stimulated expansion of tetramer reactive T cells targeting each of these antigens (Fig. To confirm that these T cells were truly tumour reactive, we used an activation-inducible marker (AIM) assay22,23. In this assay, antigen-reactive T cells cultured ex vivo with overlapping peptide pools from the same tumour antigens are identified through tandem expression of the AIMs CD69 and 4-1BB (Supplementary Fig. Using this approach, we detected a substantial increase in peptide-reactive (AIM+) CD8+ T cells from mice treated with the combination of RNA-LNPs and ICI, further supporting tumour reactivity (Fig. These data confirm that spike RNA-LNPs prime the immune response for activation, presentation and recognition of tumour-associated antigens in a manner that can be significantly expanded through concomitant treatment with ICI. We next evaluated the T cell compartment in treated tumours. In mice with B16F0 tumours, we identified substantial infiltration of PD-1+CD8+ T cells using both immunofluorescence (Fig. Importantly, we found that treatment with RNA-LNPs and ICIs increased PD-1 expression on total tumour-infiltrating CD8+ cells by greater than twentyfold (2.39% versus 51.363%, P < 0.0001) (Fig. 3f), and that PD-1+CD8+ T cells dominated the total CD3+ T cell compartment in treated mice while representing only a small minority of total CD3+ T cells in untreated mice (5.69% versus 60.6%, P < 0.001) (Fig. We next evaluated the antigen specificity of these cells. As there were many fewer CD8+ T cells in tumours compared with in spleens after enriching CD8+ cells, we pooled the six tetramers described above for a single pan-tetramer stain containing all six targets. Using this approach, we found that CD8+ tumour-infiltrating lymphocytes from mice treated with RNA-LNPs and ICIs were twice as likely to be tetramer reactive compared with non-RNA-LNP controls (3.10% versus 7.98%, P = 0.0229) (Fig. Commensurate with the increase in tumour-reactive T lymphocytes in the TME, we found that spike-encoding RNA-LNPs significantly increased PD-L1 expression on tumour cells (Fig. Blockade of IFNα signalling abrogated PD-L1 expression, confirming its importance in initiating the immunotherapy response (Extended Data Fig. Together, these data suggest that spike RNA-LNPs stimulate the production and infiltration of activated, tumour-reactive CD8+ T cells that overcome compensatory expression of PD-1 and PD-L1 in the presence of ICIs. As there are species-specific differences in how humans and mice respond to mRNA, we next sought to confirm that the pathways identified in mouse models are relevant in humans. To do this, we collected blood and plasma samples from five healthy volunteers at the baseline and at 6 h, 24 h, 7 days and 14  days after receipt of mRNA-1273 Spikevax Monovalent XBB.1.5 (COVID-19 mRNA vaccine, 2023–2024 formulation, 50 μg mRNA) (Fig. We first evaluated plasma from these volunteers using the NULISA-Seq Inflammation Panel (Alamar Biosciences)—a multiplex assay designed to sensitively detect over 250 immune-related cytokines. As IFNα was found to be essential for antitumour activity in mice, we were particularly interested in the kinetics of this protein at 24 h. Notably, not only was IFNα upregulated, but it was the most upregulated cytokine at any timepoint, increasing by an average of around 280-fold relative to the baseline to a final plasma concentration between 1 and 10 pg ml−1 (Fig. More broadly, IL-6 and IFNγ were the only cytokines that were significantly elevated at 6 h after immunization (Fig. By contrast, a multitude of inflammatory cytokines surged at 24 h after immunization, including IFNα, IFNγ, IFNω and IFNγ-inducible protein-10 (CXCL10) (Fig. Although IL-1 was not found to be significantly increased at any timepoint, IL-1R antagonist (IL1RN) was significantly elevated at 24 h, suggesting a compensatory response to a surge in IL-1 (Fig. As expected, cytokine responses to COVID-19 mRNA vaccination were short-lived in healthy individuals, with all cytokines returning to the baseline levels by 7 days (Fig. a, Schematic of the experimental design in which blood was drawn from five healthy individuals at baseline and 6 h, 24 h, 7 days and 14 days after Spikevax (mRNA-1273) COVID-19 mRNA immunization. b,c, Individual datapoints highlighting changes in expression of IFNα from baseline to 24 h for each of five healthy volunteers. The concentration was also measured separately with NULISAseq absolute quantification (AQ) (c). Adjusted P values were calculated using moderated two-tailed t-tests with false-discovery rate (FDR) correction for multiple testing. e,f, PD-L1 expression on circulating myeloid cells (CD3−CD19−CD56−CD11b+) (n = 5) (e) and DCs (CD3−CD19−CD56−CD11c+MHC-II+) (n = 5) (f) at 6 h, 24 h and 7 days after immunization. P values were calculated using two-tailed paired t-tests. To evaluate the impact of COVID-19 mRNA-induced antiviral cytokines on immune cells, we next evaluated the phenotypes of circulating myeloid cells using flow cytometry. Consistent with our findings in mouse models, we found that mRNA immunization drove innate immune activation exemplified by increased expression of PD-L1 on circulating CD11b+ myeloid cells and CD11c+ DCs (Fig. Phenotypes for all cells normalized by 7 days, which may be a result of the lack of target in these healthy participants or lymphoreticular localization of activated T cells for polarization into memory cells. We next repeated this study in 11 healthy volunteers who received the Comirnaty vaccine (BNT162b2, COVID-19 mRNA vaccine, 2024–2025 formula, with 30 μg mRNA) (Extended Data Fig. Although we found similar changes to the cytokine profile overall, the magnitude of the increase in type I IFN and innate immune activation by flow cytometry was significantly reduced with BNT162b2, which contains less mRNA relative to mRNA-1273 (Extended Data Fig. As infiltration of antigen-specific T cells in tumours is known to be associated with increases in PD-L1 expression on tumour cells and was correlated with antitumour response in our preclinical models, we hypothesized that patients who received a COVID-19 mRNA vaccine would exhibit higher PD-L1 expression on their tumours. To test this hypothesis, we assembled two cohorts. We first evaluated 2,315 pathology reports from patients with NSCLC with biopsies reporting tumour proportion score (TPS), separating patients into three groups based on the timing between their biopsy and their most recent mRNA vaccine (Fig. We found that patients who had received a COVID-19 mRNA vaccine less than 100 days before biopsy exhibited a 24% increase in mean TPS of PD-L1 compared with patients who had not received any COVID-19 mRNA vaccines before biopsy (31% versus 25%, P = 0.0450) and a 41% increase in mean TPS relative to patients who received an mRNA vaccine 100 or more days before biopsy (31% versus 22%, P = 0.0099) (Fig. As a TPS of 50% is a clinically important threshold to determine whether patients with NSCLC are eligible for single-agent immunotherapy instead of chemoimmunotherapy, we next evaluated these data as a binary outcome around this threshold. Notably, we found that patients who received a COVID-19 mRNA vaccine were 29% more likely to meet or exceed the 50% TPS threshold over unvaccinated patients (36% versus 28%, P = 0.0295) (Fig. 5c), suggesting that mRNA vaccines have a sufficient impact on TPS to modify treatment decisions. By contrast, pre-biopsy influenza and pneumonia vaccines were not associated with TPS changes (Fig. a, Schematic of patients with NSCLC biopsies documenting PD-L1 TPS. b, TPS stratified by COVID-19 mRNA vaccination timing. c, The distribution of samples with TPS ≥ 50%. d, TPS stratified by influenza (left) or pneumonia (right) vaccination timing. e, Schematic of biopsies documenting TPS or combined positive score (CPS) of PD-L1 at our institution (January 2020 to October 2023). f, Primary tumour locations from this diverse cohort. g, TPS in the tissue-agnostic cohort stratified by COVID-19 mRNA immunization timing. h, TPS stratified by timing of influenza vaccination. P values were calculated using two-tailed unpaired t-tests (b, d and h), two-tailed unpaired t-tests with Welch's correction for unequal variance (g) and two-sided Fisher's exact test evaluating the likelihood of TPS greater than 50% (c). i, Survival of patients in the tissue-agnostic cohort treated with ICI who received any COVID-19 vaccine within 100 days of initiating ICI or did not receive any COVID-19 vaccine. k, Survival for patients in i who started ICI in the pandemic era (since 2 September 2020, 100 days before mRNA vaccine approval). Survival analyses in the tissue-agnostic cohort were not limited to only those patients with a clear TPS value. l–o, The OS for patients with metastatic stage IV NSCLC treated with ICI who received a COVID-19 mRNA vaccine within 100 days of initiating ICI or did not receive a COVID-19 vaccine who had baseline PD-L1 expression at baseline biopsy TPS < 1% (l), 1–49.9% (m) or ≥50% (n). To evaluate the impact of vaccination in each clinical setting, patients were excluded if they received a COVID-19 mRNA vaccine before their biopsy. o, OS of unvaccinated patients with stage IV NSCLC stratified by era of ICI start who had baseline TPS < 1% at biopsy. P values and HRs were calculated using log-rank (Mantel–Cox, two-sided) tests (i–o). To expand our findings beyond NSCLC and melanoma, we assembled a separate cohort including all patients at our quaternary referral centre with pathology reports including the term ‘PD-L1' in a four-year period including the pandemic era (Fig. Together, we identified 5,317 unique pathology reports from January 2020 to October 2023 with the term ‘PD-L1' including 2,831 reporting TPS from a diverse array of patients representing a variety of primary sites and histologies (Fig. In this cohort, receipt of a COVID-19 mRNA vaccine within 100 days before biopsy was associated with a 37% increase in TPS (13.3% versus 9.7%, P = 0.0364) (Fig. 10k), similar to the effect seen in patients with NSCLC. We next repeated the survival analysis in this broad patient cohort, including patients at our institution with a biopsy for PD-L1 during our dates of interest who received ICI (n = 888). In this cohort, patients who received any COVID-19 vaccine within 100 days of initiating ICI experienced significantly improved survival relative to their unvaccinated peers (P = 0.0038, HR = 0.73, 95% CI = 0.60–0.90) (Fig. This effect was consistent when limited to patients who received their vaccine within 100 days before the start of ICI (P = 0.0311, HR = 0.76, 95% CI = 0.60–0.96) (Fig. 5j) and patients who started their ICI during the pandemic era (P = 0.0056, HR = 0.74, 95% CI = 0.60–0.91) (Fig. We next sought to evaluate whether vaccination could restore immune sensitivity in patients with immunologically ‘cold' tumors. We used pre-vaccine TPS as a surrogate for immune sensitivity, as NSCLC patients with TPS <1% have reduced benefit from ICIs compared to patients with baseline TPS >1%. Among patients with stage IV NSCLC and baseline TPS <1%, those who received a COVID-19 mRNA vaccine within 100 days of initiating ICI exhibited OS similar to that of patients with baseline TPS >1%, suggesting restored sensitivity to ICIs (Fig. In addition, the association between COVID-19 mRNA vaccination and OS was similar for patients with TPS <1% relative to those patients with a TPS at biopsy of 1–49.9% or ≥50% (Fig. This effect was not explained by changes in patient management during the pandemic period, as patients with stage IV NSCLC with TPS < 1% at biopsy who did not receive a vaccine had similar outcomes before and during the pandemic era (Fig. Together, these data suggest a model in which mRNA vaccines targeting non-tumour-related antigens stimulate robust antitumour immune responses that sensitize tumors to ICIs (Extended Data Fig. mRNA vaccines first stimulate a surge in antiviral cytokines, including IFNα, that drive systemic innate immune activation. Tumour-resident innate immune cells activated by this cytokine surge prime T cells, which become activated and infiltrate tumours. Although tumour cells evade attack by upregulating PD-L1 expression, combination with ICI enables COVID-19 mRNA vaccines to overcome this compensatory response, eliciting tumour regression and improved survival. Immunotherapy promises to deliver systemic anti-cancer therapy with long-term memory preventing recurrence. However, immunotherapy with ICIs relies on pre-existing anti-cancer immunity, which is absent in most patients. mRNA vaccines have recently emerged as a promising strategy to generate anti-cancer immunity to magnify the effects of ICIs24,25,26,27. Here we used mRNA vaccines targeting the COVID-19 spike protein to demonstrate a mechanism by which mRNA vaccines improve survival in combination with ICI, even when the mRNA does not encode tumour antigens. Spike RNA-LNPs elicit body-wide APC activation, leading to the expansion of highly activated tumour specific T cells. When combined with ICIs, these T cells mediate tumour regression. As personalized neoantigen vaccines require considerable manufacturing time, off-the-shelf RNA-LNPs targeting tumour-associated or even infectious disease antigens may represent widely available, low-cost alternatives for patients waiting for personalized neoantigen vaccines or in settings in which personalized neoantigen vaccines are not available. Although the local injection of RNA-LNPs for infectious disease or personalized cancer vaccines does not aim to directly modify the tumour immune microenvironment, we show here that even local injection of RNA-LNPs containing N1-methyl-pseudouridine to minimize innate immune activation produces sufficient stimulus to reprogram TMEs and sensitize tumours to ICI. Moreover, we find that augmenting innate immune activation by replacing N1-methyl-pseudouridine-modified mRNA with unmodified mRNA may further enhance the antitumour effects of this approach. The surprising effectiveness of this approach may explain the relative success of mRNA vaccines compared with other methodologies that also generate T cells reactive against targeted neoepitopes10,11,12,13,28,29. These results reveal that the timing of a routine immunization might influence treatment trajectories, and provide an over-the-counter means to overcome intrinsic ICI resistance. More broadly, they establish systemic innate immune modulation as a strategy to sensitize tumours to ICIs, and position off-the-shelf RNA therapeutics targeting infectious disease antigens as universal modulators of antitumour immunity. Although we focus on a single therapeutic due to its wide availability, these data could pave the way for other universal mRNA therapeutics specifically designed to reset patient immune systems for enhanced response to immunotherapy. We completed a non-interventional, retrospective review of patient data using the MDACC electronic health record system, which contains a record of the patients who are treated at the primary campus of MDACC, a large quaternary cancer hospital in Houston, Texas. The chart review for this study involved three groups of patients: (1) patients with tumour biopsies confirming stage III or stage IV NSCLC between January 2017 and September 2022; (2) patients with melanoma of any stage who received single- or multi-agent immune checkpoint blockade between January 2019 and December 2022; and (3) a tissue-agnostic cohort, which included all patients with pathology results for PD-L1 from January 2020 to October 2023 at our institution across a wide range of histologies. This study was approved by the MD Anderson Cancer Center institutional review board. In the NSCLC dataset, patient information was collected regarding patient demographics (such as age at immunotherapy start, sex, ethnicity), primary tumour histology, clinical stage, known tumour mutations (such as those in EGFR, KRAS, HER2, ALK, MET, p53 and RET), metastatic burden at immunotherapy start (brain, liver), Eastern Cooperative Oncology Group (ECOG) performance status (PS; range, 0–5) near the initiation of immunotherapy, radiation therapy in the time around immunotherapy start, chemotherapy history, immunodeficiency, comorbidities (heart disease, kidney disease, liver disease, respiratory disease), history of other primary tumours, steroid use around immunotherapy initiation, date of last follow-up, date of death, date of first recurrence or progression, ICI agent names and start dates, COVID-19, influenza and pneumococcal vaccination dates, and TPS. For patients with multiple biopsies, the PD-L1 expression recorded from the closest biopsy to ICI start was used for analysis. In the melanoma dataset, patient information was collected regarding patient demographics (including age at immunotherapy start, sex, ethnicity), primary tumour histology, clinical stage, known tumour mutations (such as those in EGFR, KRAS, HER2, ALK, MET, p53 and RET), metastatic burden at immunotherapy start (for example, brain, liver), PS (range, 0–5) at the initiation of immunotherapy, chemotherapy history, immunodeficiency, comorbidities (for example, heart disease, kidney disease, liver disease, respiratory disease), other primary tumour data, steroid use around COVID-19 vaccination and immunotherapy initiation, date of last follow-up, date of death, date of first recurrence/progression, ICI agent names and start dates, and COVID-19 vaccination dates. Although our dataset did not include specific vaccine formulations administered to each patient, vaccine formulations administered during the study period included the original monovalent mRNA-1273 vaccine from Moderna (100 µg mRNA prime, 50 µg mRNA booster) released on 18 December 2020; the bivalent Moderna vaccine targeting the original strain and Omicron BA.4/BA.5 (50 µg mRNA) released on 1 September 2022; the original monovalent vaccine from Pfizer/BioNTech (30 µg mRNA prime and booster) released on 11 December 2020; and the Pfizer/BioNTech bivalent formulation (30 µg mRNA) released on 31 August 2022. Patients were separated into two groups: (1) patients who received a COVID-19 mRNA vaccination within 100 days of ICI start; and (2) patients who did not receive a COVID-19 vaccination. For patients who did not receive COVID-19 mRNA vaccination, OS was calculated as the time between the initiation of their first ICI start and the date of death or last follow-up. For patients who received COVID-19 mRNA vaccination, PFS was calculated as the time between the initiation of ICI closest to mRNA vaccination and the first incidence of either pathology-confirmed recurrence or imaging-confirmed progression, whichever occurred earlier, that was declared progression in their primary medical oncologist's clinical notes. For patients who did not receive COVID-19 mRNA vaccination, PFS was calculated as the time between their first ICI start date and clinician-confirmed progression as described above. Patients who progressed before the receipt of mRNA vaccination were included in the vaccination group for this analysis. For Cox proportional hazards regression, time-dependent variables were defined as described in the figure captions. Binary and categorical variables (for example, stage, gender, mutational status, comorbidities) were converted into factors. For each variable, we constructed individual Cox proportional hazards models. HRs, 95% CIs (Wald) and P values were extracted from model summaries. Variables with P < 0.05 in univariable analysis were considered for multivariable modelling. Multivariable Cox proportional hazard regressions were similarly generated with significant variables from univariable analysis. For multivariable Cox proportional hazards regressions, patients with missing values were excluded from analyses. Categories with fewer than five cases are reported for completeness but were considered not statistically meaningful due to convergence and insufficient data for reliable inference. Variables with significant relationships with survival were included in multivariable analyses (MVAs), and those significant after MVA (P < 0.05) were included in PSM. Certain variables (for example, steroid use within 1 month of vaccine) represented subgroups of the treatment cohort by definition. These were evaluated descriptively and in univariable models but were not included as covariables in MVAs, as they do not represent baseline confounders and were not significant on univariable analysis. We assessed the proportional hazards assumption in multivariate models by evaluating Schoenfeld residuals and found no major violations. A small number of patients in the NSCLC dataset did not have documented ECOG scores, which were an important predictor of outcomes in Cox proportional hazards regression. BMI had missingness in initial univariable analyses but, because it was not required for PSM, imputation was not performed to address the missingness. Multivariable logistic regression and ridge regression were used to evaluate associations between ECOG missingness and clinically relevant pretreatment covariables. Ridge regression was selected due to multicollinearity between important variables that could not be excluded from the model. Variables not included in models predicting ECOG missingness were vaccination status, steroid usage within a timeframe of vaccination or ICI, PD-L1, concurrent chemotherapy, immunotherapy agent and BMI (as mentioned above). Subsequent ridge regression analysis suggested that ECOG scores in this analysis had a high probability of exhibiting missingness at random. Given the small sample size in the stage III NSCLC dataset treatment group, multiple imputation was performed to estimate missing ECOG values using chained equations and the R package MICE, where predictive mean matching (PMM) (with set.seed(2025)) was used to create five imputed datasets. Variables used for imputation were chosen based on large absolute ridge regression coefficients (|β| > 0.3) and clinical relevance to ensure that prediction of ECOG missingness occurred with accuracy. After selection using regression coefficients, age and previous cycles of systemic therapy were included due to clinical importance. To facilitate PSM while maintaining power in patients with stage III NSCLC, the mode of the five imputed ECOG values was used as a single value for each patient (n = 5). This approach was necessary to minimize variance inflation and avoid diluting treatment effects in a very small sample. As PMM is random, small deviations in variable definition, factor levels and row ordering may result in slightly different imputed values. However, sensitivity analyses with slightly different mode selection in the case of ties, different seeds and an increased number of datasets (n = 20 or n = 10 rather than n = 5) revealed similar outcomes (P < 0.05) after subsequent matching. The plausibility of imputed values was assessed using density plots, which demonstrated appropriate range and overlap between observed and imputed ECOG values. After imputation, PSM was performed using the R MatchIt package. A logistic regression model was used to estimate propensity scores, predicting treatment assignment of ‘vaccine within 100 days' or ‘no vaccine' with covariables based on significant factors associated with survival from MVA, including original or imputed ECOG scores. Across all propensity score analyses, nearest-neighbour matching was performed on the previously estimated propensity scores to identify a balanced cohort. In the case of stage III unresectable NSCLC PSM, nearest-neighbour PSM was performed with a caliper of 0.1 and a 2:1 ratio of control patients to treated patients in stage III UR NSCLC to improve covariable balance and maximize statistical power. Sensitivity analysis with removal of ECOG from the PSM model revealed similar trends. In stage IV NSCLC, patients with missing ECOG were excluded due to sufficient remaining sample size. PSM was once again performed with nearest-neighbour matching, with a caliper of 0.1 and a 1:1 ratio of control patients to treated patients. Sensitivity analyses with removal of ECOG from PSM rather than imputation and with the same methods as reported in stage III NSCLC (with the same caliper of 0.1 but with a ratio of 1:1 rather than 2:1) also each revealed statistically significant results. No imputation was performed in the melanoma cohort due to completeness of variables. The melanoma PSM analyses were run with nearest-neighbour matching with a caliper of 0.1 and a 1:1 ratio of control patients to treated patients. Across all matched cohorts, absolute standardized mean differences (|SMD|) of pscores were consistently less than 0.05 after matching, indicating excellent balance between groups. Balance diagnostics were confirmed visually using jitter plots and histograms of propensity scores. Log-rank tests were conducted to assess the differences between treatment groups. GraphPad Prism was used to visualize Kaplan–Meier curves, 95% CIs and at-risk numbers at 0, 10, 20, 30 and 40 months, including after extraction of propensity score-matched data from R. Comparisons between groups in primary datasets (Fig. 1) were reported as adjusted HRs calculated with Cox proportional hazard regression. Comparisons between groups in subgroup analyses were reported as unadjusted HRs calculated with log-rank tests to preserve robustness. P values are reported for primary survival analyses. In settings in which Kaplan–Meier curves crossed, the restricted mean survival time was calculated using area under the curve analysis of the absolute differences between arms at 12 and 24 months. All statistical analyses were conducted in R v.4.4.2 (2024-10-31) and GraphPad Prism. For patients in both the tissue-agnostic cohort and the NSCLC dataset, biopsy date, pathology reports, histological information and diagnosis codes were collected for patients who had received a biopsy with pathology evaluating PD-L1 at MD Anderson (from January 2020 to October 2023 for the tissue-agnostic cohort and August 2016 to August 2022 for the NSCLC cohort). Data were curated according to the following principles: (1) simple quantitative values were replicated exactly; (2) TPS values of “<1%” were reported as 0%; (3) for patients with a CPS of 0% or CPS <1% with no TPS value, TPS was reported as 0%, as CPS of 0 or <1% implies a lack of expression of PD-L1 on both tumour cells and myeloid cells; (4) TPS values between 1.1 and 49.9 reported as “<x%” or “>x%” were excluded due to lack of interpretability; (5) TPS values of >50% were recorded as 50% and TPS values >60% were recorded as 60%, and so on, as these values indicate high PD-L1 expression. Replicate patient biopsies (repeat biopsies performed on the same patient within 2 months of a previous biopsy) were excluded, and the earliest biopsy was included. Separately, survival was analysed in the tumour-agnostic dataset with any COVID-19 mRNA vaccine within 100 days of ICI compared with patients who had no recorded history of a COVID-19 vaccine. TPS values were not required for inclusion. For group-wise analysis, patients in these cohorts were grouped into (1) those who received their closest COVID-19 mRNA vaccination less than 100 days before tissue biopsy; (2) patients who received their closest COVID-19 mRNA vaccine more than or exactly 100 days before tissue biopsy; and (3) patients who did not receive a COVID-19 mRNA vaccination before biopsy. Patients who received non-mRNA-based COVID-19 vaccines within 100 days before biopsy were included in the no-vaccine group. Studies in healthy volunteers were approved by the MD Anderson Institutional Review Board. Frozen plasma was thawed and analysed using the NULISAseq Inflammation Panel 250 by Alamar Biosciences. Absolute quantification of IFNα concentrations was performed using the NULISAseq Inflammation Panel AQ. Statistical analysis was performed using R, in which we evaluated significance using a linear model with fixed effects based on the time from vaccine and controlling for patient ID (paired), including accommodation of biological variance between the baselines using the limma package's duplicateCorrelation function. P values were calculated using moderated t-tests with FDR correction for multiple testing. For comparisons between Pfizer and Moderna vaccines, we used a linear modelling approach (limma) to assess the log2-transformed fold change in cytokine concentrations per-patient from the baseline to 6 h and 24 h. Moderated t-tests with empirical Bayes shrinkage were applied, and P values were corrected using the Benjamini–Hochberg FDR method. Contrasts were constructed to directly compare Moderna versus Pfizer for each timepoint. Cytokines were visualized if they demonstrated significant differences between Moderna and Pfizer from the baseline to 6 h or 24 h (absolute value of log2-transformed fold change greater than 0.5 and FDR-corrected P < 0.05). Mean expression values per condition were normalized to the baseline and visualized. Volcano plots were generated using R. Heat maps and graphs of cytokines with significant changes in relative expression in the plasma at 24 h were generated in GraphPad Prism. Peripheral blood mononuclear cells were thawed in a 37 °C water bath and immediately transferred into a 50 ml tube containing 9 ml of complete RPMI medium (RPMI + 10% FBS) at a ratio of 1 part cells to 9 parts medium. Cells were centrifuged at 400g for 10 min, and the supernatant was decanted. The cell pellet was resuspended in 1 ml of PBS, and incubated with live/dead marker (1 µl per 1,000 µl of cell suspension, containing 1–10 million cells per ml) for 30 min at 4 °C. After incubation, 3 ml of PBS + 2% FBS was added, and the cells were centrifuged again at 400g for 8 min and the supernatant was decanted. Cells were then pre-incubated with 5 µl of Human TruStain FcX (BioLegend, 422302) per 100 µl of cell suspension for 5 min at room temperature. The cells were then washed with PBS and centrifuged at 400g for 6 min. After decanting the supernatant, 10 µl of Brilliant buffer was added to each tube, and the mixture was allowed to sit for 5 min before addition of antibodies targeting extracellular markers and the cells were incubated for 20 min at room temperature in the dark. After incubation, cells were washed and fixed in the dark for 45 min at 4 °C with 500 µl of fixation solution (BD). Cells were then permeabilized with 2 ml permeabilization buffer. Intracellular stains were added in permeabilization buffer, and cells were incubated for approximately 40 min at room temperature. After incubation, cells were washed with permeabilization buffer and resuspended in 200 µl of PBS + 2% FBS. Analysis was then conducted using Cytek Aurora Spectral Flow Cytometer. All mouse experiments and procedures were approved by the University of Florida or the University of Texas MD Anderson Institutional Animal Care and Use Committee (IACUC). Mice (aged 4–10 weeks) were maintained at 21 ± 1 °C and 35% humidity under a 14 h–10 h light–dark cycle. Tumour measurements were performed blinded to the treatment group. Orthotopic LLC models were implemented by implanting 100,000 LLC cells into the left lung of C57BL6 mice through direct injection below the 9th rib. A mix of male and female mice was used for experiments with B16F0, and male mice were used for experiments with LLC. All mice were vaccinated intramuscularly with 25 μg per dose of RNA-LNP or RNA-LPA (mRNA fraction)12. We administered anti-mouse IFNα receptor (aIFNAR1, Bio X Cell, BE0241) and anti-mouse IL-1R (Bio X Cell, BE0256) antibodies intraperitoneally at an initial dose of 500 μg per mouse for the first dose, followed by a maintenance dose of 250 μg per mouse twice a week for the remaining treatment period. Anti-PD-L1 (Bio X Cell, BE0101) checkpoint inhibitor was administered at 400 μg per mouse for the initial dose, followed by a maintenance dose of 200 μg per mouse twice a week for the remaining treatment period. Anti-PD-1 (Bio X Cell, BE0146) checkpoint inhibitor was administered at an initial dose of 400 μg per mouse, followed by a maintenance dose of 200 μg per mouse. LMW poly(I:C) (InvivoGen, tlrl-picw) was administered intramuscularly with 25 μg per mice for two doses. Tumours were measured at a frequency of three times a week starting on day 8 until more than 20% of mice reached the end point. Mice were euthanized after reaching the humane end point. SARS-CoV-2 spike coding sequence with K986P and V987P mutations was inserted into a pGEM-4Z backbone downstream of the T7 promoter with previously published UTRs and poly(A) signal analogous to the mRNA in BNT162b230,31. The T7 promoter was changed to have an AGG initiator sequence by site-directed mutagenesis (NEB, E0554) according to the company's recommended protocol. Plasmids were grown in NEB5a competent Escherichia coli and purified using the RNeasy Maxi kit (Qiagen, 75162) and sequenced using whole-plasmid sequencing by Genewiz. The in vitro transcription reaction was performed at 20 °C for 10 h. DNA was removed according to the kit instructions and RNA was purified using the RNeasy maxi kit (Qiagen, 75162). RNA was eluted in purified RNase-free water and stored at −80°C until use. mRNA was checked for quality using an Agilent TapeStation 4150. After thawing, RNA was diluted to 100 µg ml−1 in RNase-free water followed by incubation at room temperature or 72 °C for 3 min. The samples were run using an RNA ScreenTape (Agilent, 5067-5576). For dsRNA removal, the method described previously was followed20. Cellulose fibres (Sigma-Aldrich, C6288-100G) suspended in chromatography buffer were added to Nucleospin filter units (Macherey-Nagel, 740606) followed by washing. Up to 500 μg of RNA per filter unit was added to cellulose and incubated by rapid mixing for 30 min. RNA was then filtered through a 0.45 µm syringe filter (Pall, 4604) to remove any cellulose particulate before being precipitated and reconstituted in RNase-free water20. Before complexation, 0.5 M of RNase-free citrate buffer (pH 3.75) (Teknova, Custom order) was added to the RNA for a final concentration of 0.1 M. ALC-0159 (Avanti, 880155 P), ALC-0315 (Avanti, 890900 O), cholesterol (Sigma-Aldrich, C8667) and DSPC (Avanti, 850365P) were reconstituted in 100% ethanol at a ratio of 1.7:47.5:40.8:10. Lipids and RNA were mixed at a 3:1 FRR on a Nanonsemblr Ignite (Precision NanoSystems, now part of Cytiva) for an N/P ratio of 6. RNA-LNPs were dialysed overnight at 4 °C with two buffer exchanges at the 3 h and 6 h timepoints using a 3.5 kDA dialysis cassette (Thermo Fisher Scientific, A52967). After dialysis, LNPs were filtered through the 0.2 µM Supor EX ECV filter (Pall, KS2ECV2S). LNPs were concentrated using 30 kDA Amicon filters (Millipore, UFC903024) by centrifugation at 2,000g. In brief, DOTAP liposomes 2.5 mg ml−1 were mixed with mRNA at a 1:1 mass to mass ratio and complexed for 15 min at room temperature before administration. RNA concentration was determined in LNPs using the RiboGreen assay (Invitrogen, R11490) using the BioTek Cytation 3 plate reader. The encapsulation efficiency was determined by comparing readings to LNPs in TE buffer versus LNPs in 1% Triton X-100. LNPs were diluted up to 500 μg ml−1 with PBS before injection. Empty LNPs were used at a volume consistent with the dose of RNA-LNPs fabricated using the same steps and same amounts of lipids. Fluorescence values were plotted in a standard curve with an extrapolation factor set to 1.1. The values from wells with Triton X-100 (total mRNA) and TE buffer (free mRNA) was entered into the following equation: EE% = [1 – (Free RNA/Total RNA)] × 100%. Total mRNA RNA-LNPs were also loaded onto a 1% agarose gel made in 1× TAE buffer (Quality Biological, 351-008-131) with GelRed (Biotium, 41003-T). Then 500 ng of RNA was loaded into LNPs and free RNA was mixed with 6× SDS-free (NEB, B7025S) loading dye and run at 90 V for 45 min. Particle size was acquired using a Malvern Zetasizer Ultra and NanoSight's NS300. Before particle-size, PDI, concentration and surface ζ potential analysis, LNPs were diluted 100-fold with HyClone HyPure water (Cytiva, SH30538.02). They were and run in triplicate at 25 °C on the Zetasizer. Orthogonal particle-size distribution and concentration measurements were carried out using Malvern's NanoSight NS300 NTA equipment. Each LNP sample was diluted 500-fold with PBS and pumped through the equipment at constant speed for five captures with optimized camera settings. An optimized detection threshold was set for analysis. mRNA was extracted from LNPs by dissolving 750 μg ml−1 with 5 volumes of ethanol or 2 volumes of isopropanol followed by 2 washes with cold 70% ethanol. RNA pellets were dissolved in RNase-free H2O before analysis using the TapeStation. Mice were bled through the retroorbital route using heparinized capillaries (Thermo Fisher Scientific, 22-362566) collecting a maximum of 200 µl into EDTA coated tubes (Thermo Fisher Scientific, NC9414041). Whole blood was centrifuged at 1,200g for 15 min. Plasma was collected and stored at −80 °C until use. Tumours were dissected from euthanized mice with the external fibrous sac left intact. Tumours were bisected using a scalpel and half of each sample was placed into a GentleMacs column GentleMACS C Tube (130-096-334) with an enzyme mix containing 10 mg ml−1 of collagenase (Sigma-Aldrich, C5138-1G), 1 mg ml−1 of hyaluronidase (StemCell Technologies, 07461) and 200 mg ml−1 of DNase (Sigma-Aldrich, D5025-150KU) or using the tumour dissociation kit (Miltenyi, 130-096-730) followed by debris removal (Miltenyi, 130-109-398). Samples were run on a m_TDK_1 cycle and centrifuged at 300g for 5 min after completion. The pellets were resuspended in cold PBS and filtered through a 70-µm cell strainer. Whole spleens were collected from euthanized mice and placed into cold RPMI medium. The spleens were then teased through a 70-µm filter and lysed for 5 min at 37 °C using 1× BD Pharmalyse buffer (BD, 555899). Lysis buffer was quenched with medium and centrifuged at 500g for 5 min. Splenocytes were resuspended in cold PBS and filtered through a 70-µm cell strainer and washed once with PBS before being counted using the Beckman Coulter Vi-cell XR or haemocytometer. In total, 1 × 106 cells from either tumours and spleens were placed into 96-well V-bottom plates. Unless stated otherwise, the washing steps were performed by centrifugation at 500g for 5 min at 4 °C followed by resuspension of cells with 200 µl of buffer, with mixing done by pipetting. Cells were washed with cold PBS and stained with 100 µl of live/dead stain (Thermo Fisher Scientific, L10119) for 30 min at 4 °C. Cells were centrifuged and resuspended with 10 µl True stain FCX buffer (BioLegend, 422302) diluted to 100 µg ml−1 with FACS buffer for 10 min. Next, 100 µl of FACS buffer was added to each well and the cells were washed twice with cold FACS buffer. Cells were fixed for 15 min with 100 µl of Cytofix buffer (BD, 554655) at 4 °C. Next, 100 µl of PBS was added to each sample and cells were washed twice and stored in FACS buffer at 4 °C in the dark until analysis. Initial compensation was acquired using Ultracomp eBeads (Thermo Fisher Scientific, 01-2222-42) and ArC amine reactive compensation beads (Thermo Fisher Scientific, A10346). Peptides selected to be high-affinity binders for Survivin (ATFKNWPFL), GP100A (EGSRNQDWL), GP100B (KVPRNQDWL), WT1 (RMFPNAPYL), CLDN6 or claudin 6 (KVYDSLLAL) and TRP2 (SVYDFFVWL) were purchased from GenScript. In brief, peptides were reconstituted to 1 mg ml−1 in deionized water, further diluted to 75 μM and incubated with easYmer (Eagle Biosciences, 5004-01, 5001-01) at 18 °C for 48 h. The resulting complexes were tetramerized by gradually mixing with streptavidin-APC (BD Biosciences, 554067). After incubation, a final stock concentration of 500 nM for each tetramer was achieved. Cells were left to stain in the dark at room temperature for 1 h, washed once with FACS buffer, co-stained with surface antibodies (CD3, CD4) at room temperature for 20 min, washed twice with FACS buffer and analysed on a Cytek Aurora flow cytometer. The AIM assay was performed as previously described12. In this iteration, whole splenocytes were used rather than isolated T cells. In brief, 24 h after the last RNA-LNP vaccine, splenocytes were collected for antigen recall assay. Spleens were collected and processed as described above. Peptides were chosen based on their suggested upregulation in melanoma tumours12. Selected peptides were purchased from JPT Peptide Technologies and reconstituted according to the manufacturer's guidelines. Co-culture was maintained for 48 h in an incubator at 37 °C under 5% CO2. Cells were next collected and stained for the AIM assay for evaluating activation of antigen-specific T cells. Cells were centrifuged and resuspended in FACS buffer containing antibodies mix (Supplementary Table 14). Cells were washed three times and fixed using fixation medium (Thermo Fisher Scientific, GAS001S100) for 15 min at room temperature. Cells were washed once with PBS and were stored in FACS buffer at 4 °C in the dark until analysis. Results were acquired using a BD Fortessa III. Values were normalized by subtracting the percentage of AIM+ cells of tbe DMSO-only controls from each treatment condition. Statistical analysis was performed using Brown–Forsythe and Welch ANOVA, followed by Dunnett's T3 multiple-comparison test. A portion of the plasma samples were assessed for IFNα through ELISA (Invitrogen, BMS6027). For quantification of spike IgG, mice that received three vaccines/vehicle doses during tumour studies were bled once the humane end point was reached. Plasma was diluted 100,000-fold and run according to the company's recommended parameters for semiquantitative analysis. Half of bisected tumours were fixed with 4% paraformaldehyde at 4 °C overnight. The samples were washed three times using PBS and then immersed successively into 10%, 20% and 30% sucrose for cryopreservation. Blocks of tissue were moved to −20 °C for 24 h and were sectioned at a thickness of 30 µm using the Leica cryostat, and the sections were placed onto microscope slides. Before staining, the slides were brought up to room temperature for 15 min and washed three times with PBS. Tissue was blocked at room temperature for 1 h using a blocking buffer containing 2% goat or donkey serum, 1% BSA and 0.1% Triton X-100 in PBS. The sections were then stained with primary antibodies (Supplementary Table 15) in blocking buffer overnight at 4 °C. After incubation, the sections were washed three times with PBS and stained with secondary antibodies (Supplementary Table 15) diluted in blocking buffer for 1 h at room temperature. After three 5 min PBS washes, the sections were incubated with DAPI (1:1,000 in PBS) for 10 min at room temperature. After a triple PBS wash, the sections were mounted with Prolong Glass Antifade Mountant (Thermo Fisher Scientific, P36984) and covered with a cover glass. Images were acquired using a Leica Stellaris 8 WLL Spectral Confocal Microscope. Image processing was performed using Fiji ImageJ software (NIH) and Imaris (Oxford Instruments). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All clinical data are de-identified to protect patient privacy. Source data are provided with this paper. Pires da Silva, I. et al. Ipilimumab alone or ipilimumab plus anti-PD-1 therapy in patients with metastatic melanoma resistant to anti-PD-(L)1 monotherapy: a multicentre, retrospective, cohort study. Geoerger, B. et al. Pembrolizumab in paediatric patients with advanced melanoma or a PD-L1-positive, advanced, relapsed, or refractory solid tumour or lymphoma (KEYNOTE-051): interim analysis of an open-label, single-arm, phase 1-2 trial. Schachter, J. et al. Pembrolizumab versus ipilimumab for advanced melanoma: final overall survival results of a multicentre, randomised, open-label phase 3 study (KEYNOTE-006). Borghaei, H. et al. Nivolumab versus docetaxel in advanced nonsquamous non-small-cell lung cancer. Carbone, D. P. et al. First-line nivolumab in stage IV or recurrent non-small-cell lung cancer. Fehrenbacher, L. et al. Atezolizumab versus docetaxel for patients with previously treated non-small-cell lung cancer (POPLAR): a multicentre, open-label, phase 2 randomised controlled trial. Garon, E. B. et al. Pembrolizumab for the treatment of non-small-cell lung cancer. Herbst, R. S. et al. Pembrolizumab versus docetaxel for previously treated, PD-L1-positive, advanced non-small-cell lung cancer (KEYNOTE-010): a randomised controlled trial. Rittmeyer, A. et al. Atezolizumab versus docetaxel in patients with previously treated non-small-cell lung cancer (OAK): a phase 3, open-label, multicentre randomised controlled trial. Sayour, E. J. et al. Personalized tumor rna loaded lipid-nanoparticles prime the systemic and intratumoral milieu for response to cancer immunotherapy. Systemic activation of antigen presenting cells via RNA-loaded nanoparticles. Mendez-Gomez, H. R. et al. RNA aggregates harness the danger response for potent cancer immunotherapy. Systemic RNA delivery to dendritic cells exploits antiviral defence for cancer immunotherapy. Qdaisat S. et al. Sensitization of tumours to immunotherapy by boosting early type-I interferon responses enables epitope spreading. Tahtinen, S. et al. IL-1 and IL-1ra are key regulators of the inflammatory response to RNA vaccines. Sousa L. G. et al. Spontaneous tumor regression following COVID-19 vaccination. Therapeutic effect of mRNA SARS-CoV-2 vaccine on melanoma skin metastases. Insights into the structure of comirnaty COVID-19 vaccine: a theory on soft, partially bilayer-covered nanoparticles with hydrogen bond-stabilized mRNA-lipid complexes. Li, C. et al. Mechanisms of innate and adaptive immunity to the Pfizer-BioNTech BNT162b2 vaccine. Baiersdorfer, M. et al. A facile method for the removal of dsRNA contaminant from in vitro-transcribed mRNA. Pichlmair, A. et al. Activation of MDA5 requires higher-order RNA structures generated during virus infection. Balmas E. et al. Islet-autoreactive CD4+ T cells are linked with response to alefacept in type 1 diabetes. Poloni, C. et al. T-cell activation-induced marker assays in health and disease. Treatment options in primary mediastinal B cell lymphoma patients, retrospective multicentric analysis; a Turkish oncology group study. Simoes, E. A. F. et al. Immunogenicity and safety of a third COVID-19 BNT162b2 mRNA vaccine dose in 5- to 11-year olds. Systemic vaccination induces CD8+ T cells and remodels the tumor microenvironment. Ramirez-Valdez, R. A. et al. Intravenous heterologous prime-boost vaccination activates innate and adaptive immunity to promote tumor regression. Research reported in this publication was supported by the National Cancer (NCI) Institute under the University of Texas MD Anderson Cancer Center SPORE in Melanoma P50CA221703, the University of Texas MD Anderson Cancer Center Support Grant P30CA016672, T32-CA196561-08 (A.J.G. This research was also supported by an FDA orphan disease award R01FD007268 (E.J.S. ), foundation grants from the American Brain Tumour Association, the Radiological Society of North America, Conquer Cancer Foundation, the Cullen M. Taniguchi Immuno-Radiotherapy Symposium Seed Award, ASTRO-LUNGevity, Live Like Bella Childhood Cancer Foundation/Florida Department of Health, CureSearch (Catapult Award), Stop Children's Cancer and the Bonnie R. Freeman Professorship, Danny's Dream, Ian's Friend's Foundation, Alex's Lemonade Stand (R Accelerated Award), the Rally Foundation for Childhood Cancer Research, the Medulloblastoma Initiative (MBI) and Cure Group 4 medulloblastoma consortium, The National Pediatric Cancer Foundation and philanthropic contributions to The University of Texas MD Anderson Lung Moon Shot Program. We thank D. Montgomery, L. Saldarriaga, X. Ma, C. von Roemeling and D. Zhang for their help with animals experiments; D. Li for his help with statistical analysis; and the staff at the Melanoma Informatics, Tissue Resource and Translational Pathology Core (MelCore) and the Flow Cytometry and Cellular Imaging Core Facility at The University of Texas MD Anderson Cancer Center and the ICBR Cytometry Core (RRID: SCR_019119) and animal care facilities at the University of Florida. Images provided in this Article were created using BioRender. These authors contributed equally: Adam J. Grippin, Christiano Marconi, Sage Copling, Nan Li These authors jointly supervised this work: Elias J. Sayour, Steven H. Lin Adam J. Grippin, Nan Li, Chen Braun, Cole Woody, Priti Gupta, Min Wang, Annette Wu, Michael K. Rooney, Benjamin R. Schrank, Yifan Wang, Yifan Ma, Enoch Chang, Ramez Kouzy, Ada Argueta, Diana Amaya, Joe Chen, Gabriel Alatrash, William Dennis, Richard Liang, Wen Jiang & Steven H. Lin Lillian S. Wells Department of Neurosurgery, Preston A. Christiano Marconi, Dhruvkumar Soni, Frances Weidert, Chao Xie, Eden Goldenberg, Chong Zhao, Anna DeVries, Paul Castillo, Nagheme Thomas, Andria Doty, Hector R. Mendez-Gomez & Elias J. Sayour Sage Copling, Andrew Kim, Jordan Jafarnia & Nina Nariman Department of Enterprise Data Engineering and Analytics, The University of Texas MD Anderson Cancer Center, Houston, TX, USA University of North Carolina, Chapel Hill, NC, USA Department of Surgical Oncology, Division Surgery, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Julie Simon, Alicia Bea Davies & Jennifer Wargo Jeff Lewis, Xiong Wei, Waree Rinsurongkawong, Hai Tran, Jianjun Zhang, Don L. Gibbons & John V. Heymach Department of Thoracic and Cardiovascular Surgery, The University of Texas MD Anderson Cancer Center, Houston, TX, USA James P. Allison Institute, The University of Texas MD Anderson Cancer Center, Houston, TX, USA The University of Texas MD Anderson Cancer Center, Houston, TX, USA Ashley Aaroe, Sanu Abraham, Lee Andrews II, Kiran K. Badami, Janna A. Baganz, Pratibha Bajwa, Gregory R. Barbosa, Hannah C. Beird, Kristy Brock, Elizabeth M. Burton, Juan Cata, Caroline Chung, Catherine Claussen, John Crommett, Michael Cutherell, Bouthaina Dabaja, Hiba Dagher, Kevin M. Daniels, Mary Domask, Giulio Draetta, Paul Edelkamp Jr, Sarah Fisher, Katy Elizabeth French, Andrew Futreal, Maria Gaeta, Myrna Godoy, Drew Goldstein, Jillian Gunther, Kate Hutcheson, David Jaffray, Jeff Jin, Teny Matthew John, Trey Kell, Mark Knafl, Rayson C. Kwan, J. Jack Lee, Jennifer Litton, Kevin W. McEnery, Mary McGuire, Benjamin Mescher, Tejo Musunuru, Mayoora Muthu, Joseph Nates, Craig S. Owen, Priyadharshini Padmakumar, Nicholas Palaskas, Jay J. Patel, Sabitha Prabhakaran, Lucas Ramsey, Vinod Ravi, Cristhiam Rojas Hernandez, Bilja Sajith, Paul A. Scheet, Stephanie Schmidt, Kenna R. Shaw, Sanjay Shete, Daniel P. Shoenthal, Lessley J. Stoltenberg, Hussein Tawbi, Anastasia Turin, Samir Unni, Benju Vicknamparampil, Max C. Weber, John Weinstein, Scott Eric Woodman, Mark C. Wozny, Carol Wu, Jia Wu, James C. Yao, Chingyi Young, Emily Yu & Steven Zatorski Cancer Care Equity and Professional Wellness, Sarah Cannon Research Institute (SCRI), Nashville, TN, USA Sir Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia conducted, directed, supervised and analysed data for preclinical studies. S.C. conducted, directed, supervised and analysed data for clinical studies. contributed to overall statistical strategy and data analysis. conducted chart review and analysed data for retrospective clinical studies. assisted in design and execution of translational studies. J.S., A.B.D., X.W., W.R., J. Lewis, J. Lee, A.A.V., D.L.G., H.T., J.Z., J.W., J.V.H and all of the members of the Data-Driven Determinants for Covid-19 Oncology Discovery Effort (D3CODE) Team analysed, aggregated and maintained clinical databases required for retrospective clinical studies. Correspondence to Adam J. Grippin, Elias J. Sayour or Steven H. Lin. has received travel funding from Alamar Biosciences and is a scientific advisor for Sift Biosciences and BeyondSpring Pharmaceuticals. reports grants from Merck, grants and personal fees from Johnson & Johnson and Novartis, personal fees from Bristol Myers Squibb, AstraZeneca, GenePlus, Innovent and Hengrui outside the submitted work. is a scientific advisor for AstraZeneca, Eli Lilly, Menarini Richerche, 4D Pharma, Onconova Therapeutics and Sanofi. is on the scientific advisory committee for Achelois, Affini-T, Akoya Biosciences, Apricity, Asher Bio, BioAtla LLC, Candel Therapeutics, Catalio, C-Reveal Therapeutics, Dragonfly Therapeutics, Earli Inc, Enable Medicine, Henlius/Hengenix, Hummingbird, ImaginAb, InterVenn Biosciences, LAVA Therapeutics, Lytix Biopharma, Marker Therapeutics, Matrisome, Oncolytics, Osteologic, PBM Capital, Phenomic AI, Polaris Pharma, Soley Therapeutics, Spotlight, Trained Therapeutix Discovery, Two Bear Capital and Xilis, Inc., and reports private investments in Adaptive Biotechnologies, BioNTech, JSL Health, Sporos and Time Bioventures. reports being on advisory committees for BioNTech, Genentech, Mirati Therapeutics, Eli Lilly, Janssen, Boehringer Ingelheim, Regeneron, Takeda, BerGenBio, Jazz, Curio Science, Novartis, AstraZeneca, BioAlta, Sanofi, Spectrum, GlaxoSmithKline, EMD Serono, BluePrint Medicine and Chugai; support from AstraZeneca, Boehringer Ingelheim, Spectrum, Mirati, Bristol Myers Squibb and Takeda; and licensing or royalties from Spectrum. is a paid consultant for Siren Biotechnology, an external advisory board member at Nature's Toolbox (NTX) with stock options, and a scientific advisor for iOncologi, Inc. The Article discusses patented technologies related to RNA therapeutics from A.J.G., C.M., S.H.L, D.S., H.R.M.-G. and E.J.S. Some of these technologies are licensed or under option to license by iOncologi, Inc. H.R.M.-G. and E.J.S. receive royalty payments from patents licensed to iOncologi. The other authors declare no competing interests. Nature thanks Mariano Provencio, Siri Tahtinen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-c, Overall survival for NSCLC patients who received immune therapy and obtained a COVID mRNA vaccine differentiated by vaccine manufacturer (a), whether the patient received their first vaccine during this period (“Prime only”), a booster (“Boost only”), or both a priming vaccine and a booster vaccine within the 100-day period (“Prime and Boost”) (b), and number of vaccines received within 100 days of ICI initiation (c). One patient who received 3 vaccines within 100 days is not represented. d, Overall survival among patients with NSCLC receiving their first round of ICI, stratified by receipt of COVID mRNA vaccine in the 100 days prior to ICI initiation. e, Overall survival for NSCLC patients who received immune therapy and obtained a COVID mRNA vaccine within 50 days of initiating immunotherapy. f, Overall survival for NSCLC patients receiving ICI starting on or after 9/2/2020, stratified by receipt of COVID mRNA vaccination within 100 days surrounding ICI initiation. g, Overall survival for NSCLC patients stratified by receipt of COVID mRNA vaccines with all events occurring in the first 100 days after initiating ICI removed to correct for immortal time bias. h-i, Propensity score matching for overall survival in patients with Stage III Unresectable NSCLC (h) and metastatic NSCLC (i) treated with ICI who received a COVID mRNA vaccine within 100 days of initiating ICI or did not receive a COVID mRNA vaccine. Hazard ratios and p values were calculated by log-rank (Mantel-Cox, two-sided) tests. a, Overall survival for NSCLC patients who did not receive immune checkpoint inhibition and received a COVID vaccine within 100 days of initiating chemotherapy or did not receive a COVID vaccine. b-e, Overall survival for NSCLC patients stratified by receipt of influenza vaccines (b-c), or pneumonia vaccines (d-e) with all events included (b, d) or, to correct for immortal time bias, including only events that occured greater than 100 days after initiating ICI (c, e). Patients who also received COVID vaccination were excluded from the influenza and pneumonia vaccine analyses. f-g, Overall survival for (f) all patients with Stage III NSCLC and (g) patients with Resectable Stage III NSCLC treated with ICI who received a COVID mRNA vaccine within 100 days of initiating ICI or did not receive a COVID mRNA vaccine. Hazard ratios and p values were calculated by log-rank (Mantel-Cox, two-sided) tests. a-c, Overall survival for Stage IV melanoma patients who received their first round of immune therapy and obtained a COVID mRNA vaccine differentiated by vaccine manufacturer (a), whether the patient received their first vaccine during this period (“Prime only”), a booster (“Boost only”), or both a priming vaccine and a booster vaccine within the 100-day period (“Prime and Boost”) (b), and the number of vaccines received within 100 days of ICI initiation (c). d, Overall survival among patients with Stage IV melanoma who are receiving their first round of ICI stratified by receipt of COVID mRNA vaccine in the 100 days prior to ICI initiation. Hazard ratios are reported using log-rank tests. e-f, Overall survival for Stage IV Melanoma receiving ICI starting on or after 9/2/2020, stratified by receipt of COVID mRNA vaccination within 100 days surrounding ICI initiation. For crossing survival curves as in e, RMST was calculated rather than logrank (Mantel-Cox) testing (see Methods). f, Restricted Mean Survival Time (RMST) at 12 and 24 months. Absolute differences between arms are compared with a two-tailed non-parametric area under the curve (AUC) analysis. g-i, Overall survival for patients in the Melanoma dataset treated with ICI. g, Survival for patients in the Melanoma cohort treated with ICI who received a COVID mRNA vaccine within 100 days of initiating any line of ICI or did not receive a COVID mRNA vaccine. h, Survival for Stage III patients in the Melanoma cohort treated with ICI who received a COVID mRNA vaccine within 100 days of initiating any line of ICI or did not receive a COVID mRNA vaccine. i, Survival for all Stage IV patients in the Melanoma cohort treated with ICI who received a COVID mRNA vaccine within 100 days of initiating any line of ICI or did not receive a COVID mRNA vaccine. j-k, Propensity score matching for overall survival (j) and progression-free survival (k) in patients with metastatic melanoma treated with ICI who received a COVID mRNA vaccine within 100 days of initiating ICI or did not receive a COVID mRNA vaccine. Matching was performed using all variables significantly associated with survival on multivariable analysis. Hazard ratios and p values were calculated by log-rank (Mantel-Cox, two-sided) tests unless otherwise specified. b, Quality of mRNA assessed on a BioAnalyzer. c, Total anti-Spike IgG generated by C57Bl/6 mice after 3 doses of vaccine (n = 3 biological replicates). Data are displayed as mean with SEM. d, Visualization of mRNA loading in LNPs via gel electrophoresis. e, Size distribution of LNPs assessed by DLS with 3 technical replicates. Data are displayed as mean with SD. f, Size distribution determined via nanoparticle tracking analysis. h, pH and zeta potential at biologically relevant levels of sodium bicarbonate. a, Graphical experimental design and individual tumour growth curves of C57Bl6 mice inoculated with B16F0 (50,000 cells) and vaccinated with RNA-LNP i.m. b, Graphical experimental design and individual tumour growth curves for C57Bl/6 mice inoculated with LLC (200,000 cells) and vaccinated with spike RNA-LNP (Day 3,6) with and without anti-PD1 mAbs. c, Graphical experimental design of individual tumour growth curves (c), boxplots of tumours weights (d) and counts of metastatic tumours in lungs on Day 17 (e) in C57Bl/6 mice inoculated with LLC (200,000 cells) and vaccinated with spike RNA-LNP i.m. Significance was determined by two-tailed Mann-Whitney U test (d, e, g) and two-way ANOVA/mixed-effect analysis with Geisser-greenhouse correction (f). f, data are represented as mean +/- SEM. c, Tumour growth for C57Bl/6 mice with subcutaneous B16F0 tumours (50,000 cells) treated with anti-PD-L1 and either RNA-LNPs or exogenous IFN-α (Days 3, 6, and 20) (n = 8/group). Early differences in tumour growth volumes were lost by day 20 without continued treatment. e, Tumour growth for C57Bl/6 mice with B16F0 tumours (50,000 cells) treated with anti-PD-L1(Days 3/6/10/13/17/20) with or without RNA-LNPs (Days 3,6,20) containing mRNA coding for the Spike or the CMV antigen pp65 incorporating N1-methyl pseudouridine (“modified”) or wild-type uridine (“unmodified”) and (f) boxplots of day 17 and day 20 tumour volumes (n = 8/group). Significance was determined by two-tailed Mann-Whitney U test (b,f) and two-way ANOVA/mixed-effect analysis with Geisser-Greenhouse correction (c,d,e). Data are displayed as means with standard error. Boxplots in b and f display whiskers extending to the highest and lowest values from a box drawn between the 1st and 3rd quartiles with a line centred at the median. For c-e, data are represented as mean +/- SEM. a, QC analysis of dsRNA contamination before and after purification (analysis performed by Genscript's dsRNA residue assay). In this experiment, anionic LPA was synthesized by mixing DOTAP liposomes with mRNA at a 1:1 mass to mass ratio to formulate lipid particle aggregates (see methods). Data are displayed as means with standard error. Significance was determined by two-way ANOVA/mixed-effect analysis with Geisser-greenhouse correction (b), and two-tailed unpaired t tests (c). For b, data are represented as mean +/- SEM. Values are represented as pg/mL (a-j) for samples within the standard curve, and as fluorescent intensity values for cytokines above the standard curve (k-l). Plasma from subcutaneous B16F0 (50,000 cells) bearing C57Bl/6 animals (n = 5/group) 24 h after one RNA-LNP vaccine i.m. Whiskers extend to highest and lowest values, with a box shown between 1st and 3rd quartiles with a line centred at median. (Days 3, 6, 20) from spleens of C57Bl/6 animals bearing subcutaneous B16F0 (50,000 cells, n = 5/group). Wild type and IFN-gamma KO mice with s.c. B16F0 tumours (50,000 cells) were treated with three doses of mRNA vaccines (Days 3, 6, and 17) with or without twice weekly antibodies blocking the IFN-a receptor (IFNAR1) (n = 4 untreated, n = 5 for all other groups). Whiskers extend to highest and lowest values from a box drawn between 1st and 3rd quartiles with a line centred at median. Significance was determined using two-tailed unpaired t tests. Adjusted p values were calculated using moderated two-tailed t-tests with FDR correction for multiple testing. c-d, Individual data points highlighting changes in expression of IFN-α from baseline to 24 h for healthy volunteers (n = 11) expressed as fold change from baseline (c) and concentration (d). Data are presented as means with standard error. p values in e-h are results of two-tailed paired t tests. Moderated t tests were performed on per-patient log2 fold change differences between cytokines at baseline vs 6 h or 24 h, with direct comparison of fold change from baseline in volunteers treated with either Moderna or Pfizer at each timepoint. Relative fold change for Moderna compared to Pfizer was displayed for differences that were significant with |log2FC | > 0.5 and p < 0.05 at either 6 h or 24 h after multiple comparisons testing. Cumulative moving average of PDL1 expression for patients in the NSCLC (j) and Tissue Agnostic (k) cohorts stratified by the time from each patient's most recent COVID mRNA vaccine. Data indicates the average of all TPS measurements from patients who received biopsy within each period from COVID mRNA immunization. Blue lines indicate unvaccinated patient average TPS. RNA-LNPs stimulate production of IFN-α, leading APCs to prime T cells in lymphoid organs. Combination therapy with RNA-LNPs and ICIs overcomes this resistance mechanism, leading to tumour rejection. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Grippin, A.J., Marconi, C., Copling, S. et al. SARS-CoV-2 mRNA vaccines sensitize tumours to immune checkpoint blockade. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03300-4'>Google claims ‘quantum advantage' again — but researchers are sceptical</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 15:35:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Google researchers have made a fresh claim of quantum advantage — the ability of quantum computers to radically speed up calculations compared with their classical counterparts. It could also, in theory, be replicated on another quantum computer. “This algorithm offers the opportunity for real-world applications,” said Hartmut Neven, who heads Google's quantum-computing lab, in Santa Barbara, California, at a briefing for journalists ahead of the announcement. The firm is optimistic that in five years there will be practical uses for quantum computers, he added. Google uncovers how quantum computers can beat today's best supercomputers Google uncovers how quantum computers can beat today's best supercomputers “Personally I don't think that's enough to make such a big claim,” he says. Others say that the promise of practical use so soon is premature. Google researchers and their collaborators fleshed out how they could apply the algorithm to simple molecules in a preprint study that they have submitted to arXiv. They were able to predict certain features of the molecules' structures using quantum simulations, and confirm their findings with nuclear magnetic resonance (NMR) measurements. Applying the quantum echoes algorithm to more complex systems will require less noisy hardware or methods to correct for errors that are still being worked on, said Tom O'Brien, a research scientist at Google Quantum AI in Munich, Germany, at the briefing. The demonstrations used Google's Willow chip, which harnesses 105 tiny superconducting circuits to store information as quantum bits, or qubits — the quantum equivalent of classical bits of information. Google's algorithm is able to detect subtle quantum links between distant parts of the computer, which otherwise get scrambled and lost owing to interactions between the device's many quantum parts. To apply the algorithm to molecules, researchers make qubits simulate ‘spins' of atomic nuclei, the quantum property that makes each nucleus act like a tiny bar magnet. By simulating spins using qubits, the quantum echoes algorithm can tease out long-distance interactions to give more structural information than is possible using NMR alone, said O'Brien. Google uncovers how quantum computers can beat today's best supercomputers AI discovers learning algorithm that outperforms those designed by humans We are seeking a highly motivated Research Assistant to join our dynamic team at i-BRAIN. The School of Engineering at Westlake University is seeking to fill multiple tenured or tenure-track faculty positions in all ranks. Google uncovers how quantum computers can beat today's best supercomputers An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/google-measures-quantum-echoes-on-willow-quantum-computer-chip/'>‘Quantum Echoes' on Google's Willow Chip Hint at Future Breakthroughs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 15:25:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“Quantum echoes” rippling through Google's quantum computer chip Willow could lead to advances in molecular chemistry and the physics of black holes Presently limited to 100 or so error-prone qubits, the quantum equivalents to classical bits, they are far from supreme calculating machines that are capable of exactly simulating chemical reactions and more. Quantum theorists and experimenters are working within these constraints to find feasible tasks for their devices to show an advantage over the performance of classical computers. Researchers at Google have spent years developing one such task: measuring how quantum information gets jumbled up over time. “Different systems scramble things in different ways,” says Shenglong Xu, a quantum information theorist at Texas A&M University, who was not involved with the Google research. “How the information gets processed tells us about the nature of the system.” Insight from scrambling could even provide details that would allow quantum technologists to achieve precise molecular simulations with their machines. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In a preprint paper, posted online in June, the Google team reported the most in-depth measurement of information scrambling to date, which was achieved with their 105-qubit Willow chip. The team's measurement was so complex, in fact, that using current algorithms a classical supercomputer would be far slower than Willow. The Nature paper's three referees were broadly positive about the technical accomplishment in anonymized peer review reports that Google shared with Scientific American. One of these reviewers praised the work as “truly impressive” for “experimentally accessing such subtle quantum interference effects.” But the referees split over the degree to which Google had demonstrated bona fide quantum advantage. The Google team's achievement “seems like it is beyond what we can do right now using classical methods,” Xu says. “It's a very interesting contribution to the field.” Chaos is common to the classical world because classical systems can be highly sensitive to small changes in their initial conditions. So what happens to the proverbial butterfly in a quantum system? “There's always going to be small [quantum] fluctuations,” says Pieter Claeys, a physicist at the Max Planck Institute for the Physics of Complex Systems in Dresden, Germany, who was not involved with the new study. Like the classical butterfly, these fluctuations can also have downstream effects on the scrambling of information in a quantum system. The OTOC protocol goes something like this: take a solved Rubik's cube and scramble it with a set sequence of twists. Then add an additional twist and perform the first sequence in reverse. In 2021 Google demonstrated the OTOC protocol on its Sycamore chip—a more error-prone, 53-qubit predecessor to Willow. In a press call discussing the new Willow results, Hartmut Neven, head of Google's quantum computing effort, described OTOCs as a “measure of how quickly information travels in a highly entangled system.” In their 2021 work, the Google researchers were able to observe in detail how information spread across Sycamore's grid of qubits, rippling outward from an initial state. To test the limits of Willow, with its doubled number of qubits and roughly doubled qubit fidelity, the Google researchers ran it through a doubled OTOC protocol: scramble, butterfly, unscramble, scramble, butterfly, unscramble. Google researchers estimate it would take three years for a classical supercomputer to do what Willow did in two hours. The researchers applied an OTOC protocol to Willow to estimate a key property of a system of organic molecules: the distance between two hydrogen atoms. Though the simulation technique is still nascent and not faster than classical approaches, it does seem to agree with experimental results. “People have started thinking about information scrambling and information dynamics in the context of black hole physics,” says Laura Cui, a Ph.D. student researching quantum information at the California Institute of Technology. “We're very much on the way to resolving it using these tools from information theory.” If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09618-3'>Cryogenic X-ray photoelectron spectroscopy for battery interfaces</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 15:08:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 646, pages 850–855 (2025)Cite this article Understanding the chemical environment of pristine interfaces is a long-sought goal in electrochemistry, materials science and surface science. A substantial understanding of one such interface, the solid electrolyte interphase (SEI) in lithium anodes, originates from X-ray photoelectron spectroscopy (XPS)1,2. However, room temperature (RT) combined with ultrahigh vacuum (UHV) can induce major SEI evolution from reactions and volatilization during XPS1,2. Thus, a technique is necessary for SEI stabilization. Here we develop cryogenic (cryo)-XPS with immediate plunge freezing and demonstrate SEI preservation. We discover substantially different SEI speciation and a thicker pristine SEI with cryo-XPS, free from RT-associated thickness reduction and alterations to important species, including LiF and Li2O, in UHV. This new access to pristine SEI composition enables performance correlations across diverse electrolyte chemistries. Primarily, we highlight the necessity of studying sensitive interfaces under cryogenic conditions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access $199.00 per year only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data needed to evaluate the conclusions are present in the paper or the Supplementary Information. Source data are provided with this paper. Oyakhire, S. T., Gong, H., Cui, Y., Bao, Z. & Bent, S. F. An X-ray photoelectron spectroscopy primer for solid electrolyte interphase characterization in lithium metal anodes. ACS Energy Lett. Google Scholar & Bao, Z. Degradation and speciation of Li salts during XPS analysis for battery research. ACS Energy Lett. Bard, A. J. et al. ChemInform Abstract: the electrode/electrolyte interface - a status report. & Manthiram, A. Electrode-electrolyte interfaces in lithium-based batteries. Energy Environ. Stamenkovic, V. R., Strmcnik, D., Lopes, P. P. & Markovic, N. M. Energy and fuels from electrochemical interfaces. Lin, D., Liu, Y. & Cui, Y. Reviving the lithium metal anode for high-energy batteries. Li, Y., Leung, K. & Qi, Y. Computational exploration of the Li-electrode|electrolyte interface in the presence of a nanometer thick solid-electrolyte interphase layer. Winter, M. The solid electrolyte interphase – the most important and the least understood solid electrolyte in rechargeable Li batteries. Dedryvère, R. et al. XPS identification of the organic and inorganic components of the electrode/electrolyte interface formed on a metallic cathode. Kanamura, K., Tamura, H., Shiraishi, S. & Takehara, Z.-I. XPS analysis for the lithium surface immersed in γ-butyrolactone containing various salts. Andersson, A. M. & Edström, K. Chemical composition and morphology of the elevated temperature SEI on graphite. Li, Y. et al. Atomic structure of sensitive battery materials and interfaces revealed by cryo-electron microscopy. Boyle, D. T. et al. Corrosion of lithium metal anodes during calendar ageing and its microscopic origins. Steinrück, H. G. et al. Interfacial speciation determines interfacial chemistry: X-ray-induced lithium fluoride formation from water-in-salt electrolytes on solid surfaces. Zhang, Z. et al. Capturing the swelling of solid-electrolyte interphase in lithium metal batteries. Wang, X. et al. New insights on the structure of electrochemically deposited lithium metal and its solid electrolyte interphases via cryogenic TEM. Nano Lett. Zachman, M. J., Tu, Z., Choudhury, S., Archer, L. A. & Kourkoutis, L. F. Cryo-STEM mapping of solid–liquid interfaces and dendrites in lithium-metal batteries. Zhang, Z. et al. Cryogenic electron microscopy for energy materials. Bai, X., McMullan, G. & Scheres, S. H. W. How cryo-EM is revolutionizing structural biology. On the development of electron cryo-microscopy (Nobel Lecture). Henderson, R. From electron crystallography to single particle CryoEM (Nobel Lecture). Taylor, K. A. & Glaeser, R. M. Electron diffraction of frozen, hydrated protein crystals. McDowall, A. W. et al. Electron microscopy of frozen hydrated sections of vitreous ice and vitrified biological samples. Henderson, R., Baldwin, J. M., Downing, K. H., Lepault, J. & Zemlin, F. Structure of purple membrane from Halobacterium halobium: recording, measurement and evaluation of electron micrographs at 3.5 Å resolution. Fernandez-Moran, H. Cell-membrane ultrastructure: low-temperature electron microscopy and X-ray diffraction studies of lipoprotein components in lamellar systems. Tan, J., Matz, J., Dong, P., Shen, J. & Ye, M. A. A growing appreciation for the role of LiF in the solid electrolyte interphase. Energy Mater. Li, T., Zhang, X.-Q., Shi, P. & Zhang, Q. Fluorinated solid-electrolyte interphase in high-voltage lithium metal batteries. Wang, C., Meng, Y. S. & Xu, K. Perspective—fluorinating Interphases. Hobold, G. M., Wang, C., Steinberg, K., Li, Y. & Gallant, B. M. High lithium oxide prevalence in the lithium solid–electrolyte interphase for high Coulombic efficiency. Kim, M. S. et al. Suspension electrolyte with modified Li+ solvation environment for lithium metal batteries. Lin, D. et al. Fast galvanic lithium corrosion involving a Kirkendall-type mechanism. Arkel, A. E., Spitsbergen, U. & Heyding, R. D. Note on the volatility of lithium oxide. Kudo, H., Wu, C. H. & Ihle, H. R. Mass-spectrometric study of the vaporization of Li2O(s) and thermochemistry of gaseous LiO, Li2O, Li3O, and Li2O2. Seah, M. P. & Dench, W. A. Quantitative electron spectroscopy of surfaces: a standard data base for electron inelastic mean free paths in solids. D'Acunto, G. et al. Atomic layer deposition of hafnium oxide on InAs: insight from time-resolved in situ studies. Electron Mater 2, 3915–3922 (2020). Walther, T. in Microscopy Methods in Nanomaterials Characterization 105–134 (Elsevier, 2017). García De Abajo, F. J. & Di Giulio, V. Optical excitations with electron beams: challenges and opportunities. ACS Photon. & Pasta, M. Solid electrolyte interphases in lithium metal batteries. He, M., Guo, R., Hobold, G. M., Gao, H. & Gallant, B. M. The intrinsic behavior of lithium fluoride in solid electrolyte interphases on lithium. Kim, M. S. et al. Revealing the multifunctions of Li3N in the suspension electrolyte for lithium metal batteries. Spearman, C. The proof and measurement of association between two things. Oyakhire, S. T. et al. Proximity matters: interfacial solvation dictates solid electrolyte interphase composition. Nano Lett. Oyakhire, S. T. & Bent, S. F. Interfacial engineering of lithium metal anodes: what is left to uncover? Yu, Z. et al. Rational solvent molecule tuning for high-performance lithium metal battery electrolytes. Peled, E., Golodnitsky, D. & Ardel, G. Advanced model for solid electrolyte interphase electrodes in liquid and polymer electrolytes. Cui, Z. et al. Molecular anchoring of free solvents for high-voltage and high-safety lithium metal batteries. Zhang, W. et al. Recovery of isolated lithium through discharged state calendar ageing. Otto, S.-K. et al. In-depth characterization of lithium-metal surfaces with XPS and ToF-SIMS: toward better understanding of the passivation layer. Baer, D. R. XPS guide: charge neutralization and binding energy referencing for insulating samples. Greczynski, G. & Hultman, L. Compromising science by ignorant instrument calibration—need to revisit half a century of published XPS data. Liu, Q. et al. A fluorinated cation introduces new interphasial chemistries to enable high-voltage lithium metal batteries. Ge, S. et al. High safety and cycling stability of ultrahigh energy lithium ion batteries. Cell Rep. Phys. Wood, K. N. & Teeter, G. XPS on Li-battery-related compounds: analysis of inorganic SEI phases and a methodology for charge correction. ACS Appl. Energy Mater. Rustomji, C. S. et al. Liquefied gas electrolytes for electrochemical energy storage devices. Ren, X. et al. Enabling high-voltage lithium-metal batteries under practical conditions. Templeton, D. M. et al. Guidelines for terms related to chemical speciation and fractionation of elements. Definitions, structural aspects, and methodological approaches (IUPAC recommendations 2000). Feldmann, J. et al. Microwave-assisted sample preparation for element speciation. in Microwave-Assisted Sample Preparation for Trace Element Determination 281–312 (Elsevier, 2014). Greczynski, G. & Hultman, L. Towards reliable X-ray photoelectron spectroscopy: sputter-damage effects in transition metal borides, carbides, nitrides, and oxides. acknowledges support from the TomKat Center Graduate Fellowship for Translational Research and the Link Foundation Energy Fellowship. acknowledges support from the Wallenberg Foundation Postdoctoral Scholarship Program. acknowledges support from the Schmidt Science Postdoctoral Fellowship. acknowledges support from NSF GRFP. acknowledges the cryo-EM research support from the Department of Energy, Office of Basic Energy Sciences, Division of Materials Science and Engineering under contract DE-AC02-76SF00515, and the support from Assistant Secretary for Energy Efficiency and Renewable Energy, Office of Vehicle Technologies, of the US Department of Energy under the Battery Materials Research Program, the Battery500 Consortium program, and the CEI consortium program. acknowledges support from the Stanford StorageX initiative seed grant award. Part of this work was performed at the Stanford Nano Shared Facilities supported by the National Science Foundation under award no. acknowledges the discussions with J. Lee and W. Zhang on cryo-XPS transfer. appreciates D. Li and J.H. Lee for providing the LHCE electrolyte. Present address: Department of Materials Science and Engineering, University of California Berkeley, Berkeley, CA, USA Department of Chemical Engineering, Stanford University, Stanford, CA, USA Sanzeeda Baig Shuchi, Giulio D'Acunto, Solomon T. Oyakhire, Kenzie M. Sanroman Gutierrez & Stacey F. Bent Department of Chemistry, Stanford University, Stanford, CA, USA Stanford Nano Shared Facilities, Stanford University, Stanford, CA, USA Department of Materials Science and Engineering, Stanford University, Stanford, CA, USA Il Rok Choi & Yi Cui Stanford Institute for Materials Energy and Energy Sciences, SLAC National Laboratory, Menlo Park, CA, USA Yi Cui Department of Energy Science and Engineering, Stanford University, Stanford, CA, USA Yi Cui & Stacey F. Bent Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the idea and designed experiments. performed all the electrochemical experiments and conducted all the RT-XPS and cryo-XPS experiments. and G.D. developed the cryo-XPS transfer process. G.D. assisted with cryo-XPS experiments during the initial phase of the study. assisted with experimental designs. assisted with data interpretation discussions and manuscript editing. helped with cryo-XPS transfer development and instrument-related discussions. analysed and interpreted the data with assistance and feedback from all the authors. wrote the first draft of the manuscript, which was edited and revised by all the authors. assisted with electrochemical tests for CEI characterization. supervised the project in all aspects. Correspondence to Yi Cui or Stacey F. Bent. The authors declare no competing interests. Nature thanks Zhouguang Lu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-f, All high-resolution spectra for (a) F 1 s, (b) Li 1 s, (c) N 1 s, (d) C 1 s, (e) O 1 s, (f) S 2p with cryo-XPS at 0 min and 60 min. The measurements are carried out in 1 M LiFSI/DME (1 M) electrolyte. Global normalization of all the peaks is performed with respect to the C-F/SOxFy peak in F 1 s high-resolution spectra in (a). The measurements are performed on one analysis spot to capture the SEI evolution decoupled from spatial heterogeneities in the SEI. a-f, All high-resolution spectra for (a) F 1 s, (b) Li 1 s, (c) N 1 s, (d) C 1 s, (e) O 1 s, (f) S 2p with cryo heated to RT-XPS at 0 min and 60 min. The measurements are carried out in 1 M LiFSI/DME (1 M) electrolyte. Global normalization of all the peaks is performed with respect to the C-F peak in F 1 s high-resolution spectra in (a). The measurements are performed on one analysis spot to capture the SEI evolution decoupled from spatial heterogeneities in the SEI. The analysis spot is the same as the one shown in Extended Data Fig. a-b, High-resolution spectra for F 1 s peaks with (a) cryo-XPS and (b) cryo heated to RT-XPS at 0 min and at 60 min. c, Atomic concentration for unwashed SEI with cryo-XPS and cryo heated to RT-XPS at 0 min and at 60 min. The intensities of F 1 s spectra shown in (a) and (b) are normalized with respect to the SOxFy/ C-F peak. The measurements are performed on one analysis spot to capture the SEI evolution decoupled from spatial heterogeneities in the SEI. The measurements are carried out in 1 M LiFSI/DME (1 M) electrolyte. High-resolution F 1 s spectra for with cryo-XPS (blue, top), cryo-heated to RT-XPS (pink, middle), cryo-heated to RT, and then re-cooled to cryo-XPS (black, bottom), respectively. The measurements are performed on one analysis spot to capture the SEI evolution decoupled from spatial heterogeneities in the SEI. The measurements are carried out in 1 M LiPF6/EC-DEC (LP) electrolyte. An unwashed SEI sample is used as the composition is found to be more sensitive to temperature change in Extended Data Fig. High-resolution spectra for F 1 s peaks of SEI formed on Cu in 1 M LiPF6/EC-DEC (LP) electrolyte using a protocol of holding the Cu current collector at 10 mV above Li electrodeposition potential for 30 min using cryo-XPS and RT-XPS. This protocol ensures no Li is electrodeposited on Cu so that only electrolyte decomposition prior to Li plating can be studied. a-b, High-resolution spectra for (a) F 1 s peaks and (b) O 1 s peaks collected using cryo-XPS (blue, top), cryo heated to RT-XPS (pink, middle), and RT-XPS (yellow, bottom) with schematics showing SEI evolution. Global normalization across F 1 s and O 1 s is performed for the same electrolyte with respect to the C-F peak in F 1 s high-resolution spectra shown in (a). The measurements are carried out in 1 M LiPF6/EC-DEC (LP) electrolyte. a, c, High-resolution spectra for (a) Li 1 s peaks and (c) C 1 s peaks with cryo-XPS and RT-XPS. b, Histogram showing thickness distribution of the SEI using cryo-TEM. d, C K-edge fine structure of Li dendrites using cryo-STEM EELS. e, Summary of key findings from cryo-XPS, cryo-STEM EELS and RT-XPS. The intensities of C 1 s and Li 1 s spectra shown in (a) and (c) are normalized to exhibit the same maximum intensity for both cryo-XPS and RT-XPS data. The measurements in (a)-(d) are carried out on (a), (c) 0.5 mAh cm–2 and (b), (d) 0.1 mAh cm–2 Li deposited on (a), (c) Cu foil and (b), (d) Cu-evaporated TEM grid at a current density of 1 mA cm–2 in 1 M LiPF6/EC-DEC (LP) electrolyte and rinsed by 60 µL DEC. tSEI refers to thickness of the SEI. The data shown in (b) and (d) are adapted with permission from Zhang, Z. et al., Science, DOI number: 10.1126/science.abi8703 [2022], AAAS15. a-b, High-resolution spectra for (a) Li 1 s and (b) N 1 s peaks with cryo-XPS (blue, top), cryo heated to RT-XPS (pink, middle), and RT-XPS (yellow, bottom). Global normalization is performed with respect to the C-F peak in F 1 s high-resolution spectra shown in Fig. The measurements are carried out in LiFSI-1.2 DME-3 TTE by molar ratio (LHCE) electrolyte. We identify the peak near ~52 eV with cryo-XPS as Li3N (not Li0 metal) because it appears and disappears in conjunction with the same species peak in N 1 s region. Besides, Li is highly electropositive, so it is more favorable for Li to exist in its compound forms. Average % atomic concentration change with sputtering for cryo-XPS and cryo heated to RT-XPS. The measurements are carried out in residual SEI from 4 M LiFSI/DME (4 M) electrolyte after 5 cycles. The error bars of XPS results are calculated from the analysis of three different spots of the same sample. Ar+ ion sputtering is destructive toward surface chemical composition1,2,58. So, the degree of composition change before and after Ar+ ion sputtering is used to understand the relative amount of stable species in the SEI. a-b, CE (Supplementary Table 2) from Aurbach's method15,46 in relation to the F/C ratio of SEI from (a) RT-XPS and (b) cryo-XPS. The measurements are carried out on 0.5 mAh cm–2 of Li deposited on Cu at a current density of 1 mA cm–2 in 1 M LiPF6/EC-DEC (LP), 90 vol% 1 M LiPF6/EC-DEC with 10 vol% FEC (LPF), 1 M LiFSI/DME (1 M), 4 M LiFSI/DME (4 M), and LiFSI-1.2 DME-3 TTE by molar ratio (LHCE) electrolytes. The error bars of XPS results are calculated from the analysis of three different spots of the same sample. Spearman rank correlation coefficient41 (ρ) is calculated from the average values of the CE and F/C ratio. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Shuchi, S.B., D'Acunto, G., Sayavong, P. et al. Cryogenic X-ray photoelectron spectroscopy for battery interfaces. Issue date: 23 October 2025 DOI: https://doi.org/10.1038/s41586-025-09618-3 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09526-6'>Observation of constructive interference at the edge of quantum ergodicity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 15:06:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 646, pages 825–830 (2025)Cite this article The dynamics of quantum many-body systems is characterized by quantum observables that are reconstructed from correlation functions at separate points in space and time1,2,3. In dynamics with fast entanglement generation, however, quantum observables generally become insensitive to the details of the underlying dynamics at long times due to the effects of scrambling. To circumvent this limitation and enable access to relevant dynamics in experimental systems, repeated time-reversal protocols have been successfully implemented4. Here we experimentally measure the second-order out-of-time-order correlators (OTOC(2))5,6,7,8,9,10,11,12,13,14,15,16,17,18 on a superconducting quantum processor and find that they remain sensitive to the underlying dynamics at long timescales. Furthermore, OTOC(2) manifests quantum correlations in a highly entangled quantum many-body system that are inaccessible without time-reversal techniques. This is demonstrated through an experimental protocol that randomizes the phases of Pauli strings in the Heisenberg picture by inserting Pauli operators during quantum evolution. The measured values of OTOC(2) are substantially changed by the protocol, thereby revealing constructive interference between Pauli strings that form large loops in the configuration space. The observed interference mechanism also endows OTOC(2) with high degrees of classical simulation complexity. These results, combined with the capability of OTOC(2) in unravelling useful details of quantum dynamics, as shown through an example of Hamiltonian learning, indicate a viable path to practical quantum advantage. Identifying complex correlations between the many-body degrees of freedom in a quantum system is a central goal for the simulation of quantum dynamics. Even spectroscopic questions can be formulated in terms of few-point dynamical correlations. As entanglement grows with system size or evolution time, the resulting dynamics are often ergodic. Consequently, the sensitivities to the details of the quantum dynamics decay exponentially for most quantum observables, limiting their utility in revealing many-body correlations. Numerical or analytical studies of correlations are also hindered by the difficulty of identifying subtle contributing processes, which undermine common simplifying assumptions. Moreover, the linearity of the Schrödinger equation precludes the use of classical techniques based on sensitivity to initial conditions, methods that have proven effective in detecting the butterfly effect and characterizing classical chaos. As a solution to the above challenge, experimental protocols that use refocusing to echo out nearly all evolution have become essential for probing highly entangled dynamics. These protocols have proven indispensable in quantum metrology and sensing19,20 as well as in studies of chaos, black holes and thermalization6,8,21,22,23. Dynamical sequences that include time reversal are most naturally described in the Heisenberg picture of operator evolution (Fig. The sequence can be conceptualized as an interference problem, where correlations reflect coherent interference across many-body trajectories. Computing an observable can, thus, be expressed as a sum over distinct trajectories. In this conceptual framework, each time reversal corresponds to the addition of two interference arms and also other cross-terms contributing to experimental observables, which are formally known as out-of-time-order correlators (OTOCs)5,6,7,8,9,10,11,12,13,14,15,16,17,18. a, When dynamical protocols involve echoing, the Heisenberg picture of the operator evolution is the natural framework for studying dynamics. b, OTOC and OTOC(2) can be viewed as time interferometers, which highlights their capability of refocusing on desired details and echoing out unwanted dynamics. See text for the definition of parameters. In our work, we perform a family of OTOC experiments and leverage the interference framework to understand how different paths and their combinations reveal quantum correlations inaccessible without time reversal or with numerical methods. More specifically, we use the unique programmability of a digital quantum processor to change the number of interference arms (Fig. 2) and insert either noisy (Fig. 5) phase shifters into each arm. In response, we find that OTOCs are more sensitive to these perturbations compared with observables in the absence of time reversal. Furthermore, we discover that this sensitivity is enhanced as the order k of OTOC(k) (the number of interference arms) increases. In particular, OTOC(2) reveals constructive interference between Pauli strings that is invisible in lower-order observables. a, Top, quantum circuit schematic for measuring OTOCs of different orders, OTOC(k). Here, \(| {\psi }_{M}\rangle \) is an eigenstate of the measurement operator M (realized as Z in this work). The operator B is realized as X. Bottom, implementation of the unitary U as t cycles of single- and two-qubit gates. Each single-qubit gate is \(\exp \left(-{\rm{i}}\frac{\theta }{2}(\cos (\phi )X+\sin (\phi )Y)\right)\), where θ/π ∈ {0.25, 0.5, 0.75} and ϕ/π is chosen randomly from the interval [−1, 1]. Each iSWAP-like gate is equivalent to an iSWAP followed by a CPHASE gate with a conditional phase of approximately 0.35 rad. The colour at each qubit site indicates data collected with B applied to the given qubit. Purple dots indicate the fixed location of qm. Cyan lines represent the light cone of qm. For \({{\mathcal{C}}}^{(2)}\), \({{\mathcal{C}}}^{(4)}\) and \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\), qm has the same fixed location as in b whereas qb gradually moves further from qm as the number of circuit cycles increases, such that the OTOC mean \({\overline{{\mathcal{C}}}}^{(2)}\approx 0.5\) is maintained. \({{\mathcal{C}}}^{(1)}\) corresponds to \(\langle Z(t)Z\rangle \) measured at a qubit close to the centre of the lattice. a, In the Heisenberg picture, the time-evolved B(t) branches into a superposition of multi-qubit Pauli strings. For \({{\mathcal{C}}}^{(2)}\), in which only two copies of B(t) are present, the final strings Pα and Pβ need to be identical to contribute. For \({{\mathcal{C}}}^{(4)}\), the strings (Pα, Pβ, Pγ, Pδ) contribute a ‘diagonal' component \({{\mathcal{C}}}_{{\rm{diag}}}^{(4)}\) when Pα = Pβ and Pγ = Pδ, or an ‘off-diagonal' component \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) when Pα ≠ Pβ ≠ Pγ ≠ Pδ. b, Protocol for probing quantum interference. Random Pauli operators are inserted at one circuit cycle, which changes the signs of the Pauli string coefficients. c, Relative signal change, characterized by 1 − ρ, as a function of the cycle at which Paulis are inserted. ρ refers to the Pearson correlation between experimental data from 50 different 40-qubit circuits (t = 22 cycles), obtained with and without Pauli insertion. Error bars denote standard errors estimated from resampling the experimental data. Insets, Data at cycle 11. d, Comparison of experimental \({{\mathcal{C}}}^{(2)}\) values against exactly simulated \({{\mathcal{C}}}^{(2)}\) for a set of 40-qubit circuit instances. Values computed using CMC heuristic algorithms are shown for comparison, achieving an SNR of 5.3, like that of the quantum processor (SNR = 5.4). e, Experimental \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) values on the same set of 40 qubits, alongside exact and CMC simulations. Here the experimental SNR is 3.9 whereas the SNR from CMC is 1.1. Error bars on experimental data are based on an empirical error model discussed in Supplementary Information sections II.F.3 and II.F.4. Exp, experiment; MC, Monte Carlo; sim, simulation. To understand how repeated time-reversal restores the sensitivity to quantum dynamics, we first consider measuring the Pauli operator M ∈ {X, Y, Z} of a qubit qm in a square lattice of qubits and initialized in an eigenstate of M. The measurement at a time t is equivalent to the time-ordered correlator (TOC), ⟨M(t)M⟩, where M(t) = U†(t)MU(t) denotes the time-evolved M in the Heisenberg picture, U is a many-body unitary, and ⟨…⟩ denotes the expectation value over a particular initial state. As observed in previous experiments24,25,26,27, ⟨M(t)M⟩ decays exponentially over time when U is ergodic. The above decay can, however, be partially refocused through the evolution outlined in Fig. Here, the dynamics U is replaced with the nested echo sequence Uk(t) = B(t)[MB(t)]k−1, where B(t) = U†(t)BU(t) is the time-evolved operator of another Pauli B acting on qubit(s) qb some distance away from qm and k ≥ 1 is an integer. The action of Uk may be understood as dispersing the information injected by M, modifying it by B, reversing it back to M and repeating this process k − 1 times. As \({U}_{k}^{\dagger }(t)={U}_{k}(t)\), the expectation value (denoted as \({{\mathcal{C}}}^{(2k)}\) in this case) may be written as: Equation (1) yields two key insights. First, if the information originating from qm has not yet reached qb, B(t) commutes with M and the information coming back to qm is identical to its initial value. We, therefore, expect the existence of a wavefront across which \({{\mathcal{C}}}^{(2k)}\) decays. By increasing the separation between qm and qb, this front may be pushed later in time, thus allowing large signals to be measured where TOCs are approximately 0. Second, provided U is not a Clifford sequence, information starting from M and returning to M can take several different paths in configuration space. Correlations between Pauli strings within B(t) may, therefore, manifest through constructive interference between different paths for \({{\mathcal{C}}}^{(2k)}\) with k ≥ 2. We begin by characterizing the sensitivity of OTOC(2) to the microscopic details of quantum dynamics. Figure 2a schematically shows our quantum circuits, which are composed of random single-qubit and fixed two-qubit gates. The experiment is done by first fixing the choices of qm and qb. A circuit instance i is then generated by varying the random parameters of the single-qubit gates that interleave the deterministic set of two-qubit gates. For a fixed number of circuit cycles t within U, the quantity \({{\mathcal{C}}}^{(2k)}(t,{q}_{{\rm{m}}},{q}_{{\rm{b}}},i)\) is repeatedly measured until the statistical noise of the measurement is less than  10% of its average value. This protocol is then repeated by varying t, qm and qb, as well as the circuit instance i, which is sampled 50 to 250 times. Last, all experimental \({{\mathcal{C}}}^{(4)}\) or \({{\mathcal{C}}}^{(2)}\) values are normalized by a global rescaling factor obtained through error-mitigation strategies (Supplementary Information sections II.E.1 and II.F.1). The top row of Fig. 2b displays the values of \({\overline{{\mathcal{C}}}}^{(4)}(t,{q}_{{\rm{b}}})\) for different circuit cycles and choices of qb, where the overline denotes averaging over circuit instances. The location of qm is fixed throughout these measurements. For each t, there is a boundary for qb beyond which \({{\mathcal{C}}}^{(4)}\) is  approximately 1. This boundary defines the light cone of qm, which corresponds to the set of qubits that have been entangled with qm. Moreover, we find that the circuit-to-circuit fluctuation of \({{\mathcal{C}}}^{(4)}(t,{q}_{{\rm{b}}})\), defined as the standard deviation σ of its value over circuit instances, is of the same order of magnitude as the average value near the information front (bottom row of Fig. This observation indicates that \({{\mathcal{C}}}^{(4)}\) is, indeed, highly sensitive to the details of the underlying evolution U, an effect that we will later use to demonstrate its application in Hamiltonian learning. To study more systematically the decay of OTOC sensitivity over time, we measure the standard deviations of various OTOCs as functions of t (Fig. We observe that the standard deviations of \({{\mathcal{C}}}^{(2)}\), \({{\mathcal{C}}}^{(4)}\) and the off-diagonal part of OTOC(2) (\({{\mathcal{C}}}_{{\rm{off-diag}}}^{(4)}\), see ‘Large-loop interference in OTOC(2)' for a definition) decay algebraically and remain over 0.01 beyond t = 20. The standard deviation of a TOC, which does not have the echo-like structure of OTOCs, decays exponentially over time and becomes less than 0.01 at t = 9. The stark contrast between TOC and OTOCs indicates that the interferometric nature of the latter is crucial for enhancing sensitivities to quantum dynamics. In Supplementary Information section IV, we provide further studies of OTOC(k) fluctuations using one-dimensional Haar random circuits, finding that these observables decay as a power law consistent with the two-dimensional experimental data. The experiments performed in the preceding section, which change all details of U between OTOC measurements, is analogous to shifting the arms of an interferometer entirely (Fig. In this section, we demonstrate that higher-order OTOCs also become increasingly more sensitive to the phases of the interferometer arms, a hallmark of interference phenomena. Conceptually, many-body interference may be understood by first noting that equation (1) can, in the ergodic limit, be expressed as: For a given circuit instance, B(t) can be decomposed in the basis of the 4N Pauli strings {Pn} of our N-qubit system: where {bn} is a set of real-valued time-dependent coefficients. The time evolution of B is visualized in the bottom left panel of Fig. 3a, where B is seen to branch out in Pauli space over time due to the action of non-Clifford gates in our circuits, creating the so-called operator entanglement18,29,30. In the schematic, the trajectories of some Pauli strings (shaded areas) are also seen to recombine during the time evolution. This mechanism, which affects both \({{\mathcal{C}}}^{(4)}\) and \({{\mathcal{C}}}^{(2)}\), will be referred to as small-loop interference. It is present only in \({{\mathcal{C}}}^{(4)}\), as schematically shown in the upper panels of Fig. Here each cαβγδ is also a real-valued coefficient. Each Pauli string in this expression is represented as a coloured segment in the diagrams within the top panels of Fig. Multiplying two Pauli strings joins them at one end and forms a new Pauli string connecting the two new terminal points. This condition is satisfied through two distinct means. If α = β and γ = δ, we obtain the so-called diagonal contribution \({{\mathcal{C}}}_{\text{diag}}^{(4)}\) to \({{\mathcal{C}}}^{(4)}\) in which the loops enclose zero area. If α ≠ β ≠ γ ≠ δ, we obtain the off-diagonal contribution \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\), named because each term may be thought of as an off-diagonal element in the 44N × 44N density matrix formed by the Paulis. Diagrammatically, \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) consists of a superposition of operator loops, each of which contains three unconstrained Pauli strings and, therefore, encloses an arbitrarily large area. For comparison, large-loop interference is absent in \({{\mathcal{C}}}^{(2)}\), as only Pauli strings with α = β contribute, like \({{\mathcal{C}}}_{\text{diag}}^{(4)}\). To characterize the effects of quantum interference, we insert random Pauli operators at different cycles within each application of U and U† (Fig. This is analogous to shifting the phase around an ordinary interference loop without changing its intensity. By ensemble-averaging over random Paulis and probing the changes in \({{\mathcal{C}}}^{(4)}\) or \({{\mathcal{C}}}^{(2)}\), the contribution of quantum interference to each observable can then be quantified. Figure 3c shows the experimentally measured 1 − ρ, where ρ is the Pearson correlation between circuits with and without inserted Paulis. The inserted Paulis produce substantial changes in \({{\mathcal{C}}}^{(4)}\), indicating that the large-loop interference effects (\({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\)) are a dominant contribution to \({{\mathcal{C}}}^{(4)}\). By contrast, the signal change for \({{\mathcal{C}}}^{(2)}\) is much weaker overall, owing to the presence of only small-loop interference. We also note that 1 − ρ is slightly reduced at later insertion cycles for \({{\mathcal{C}}}^{(4)}\), which may arise from external decoherence processes that tend to reduce the visibility of quantum interference. Last, by subtracting the \({{\mathcal{C}}}^{(4)}\) of each circuit after Pauli insertion from its original value, the off-diagonal contribution \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) can be experimentally extracted. We find that the observed interference effects in \({{\mathcal{C}}}^{(2)}\) and \({{\mathcal{C}}}^{(4)}\) are closely connected to their complexities with respect to approximate classical simulation algorithms, as the degree of quantum interference sets the level of allowed classical approximation. For OTOC, we find that it is sometimes approximated well by combining exact wavefunction evolution with a Monte Carlo simulation that ignores the effects of small interference loops. Two such algorithms, cached Monte Carlo (CMC) and tensor networks Monte Carlo, are described in Supplementary Information section III.B. Figure 3d shows experimental \({{\mathcal{C}}}^{(2)}\) values from a set of circuits comprising 40 qubits, along with values of \({{\mathcal{C}}}^{(2)}\) computed using CMC. To quantify the accuracy of each dataset, we define a signal-to-noise ratio (SNR; Methods) against exactly simulated \({{\mathcal{C}}}^{(2)}\) values shown in the same figure. The SNR achieved by CMC (5.3) is close to the experimental SNR of 5.4. For the off-diagonal OTOC(2) (\({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\)), classical algorithms are far less accurate and achieve a much lower SNR (1.1) compared with the experiment (3.9), as shown in Fig. In Supplementary Information section III.C, we review all classical simulation algorithms attempted as part of this work and show that none succeeds in approximating \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\). The combination of large sensitivity and high classical simulation complexity makes higher-order OTOCs such as OTOC(2) a prime candidate for achieving the long-standing goal of practical quantum advantage. To illustrate this potential, we perform two more experiments, which show: (1) OTOC(2) can be accurately resolved in regimes that are, at present, intractable with classical supercomputers. (2) A specific example wherein OTOC(2) is used to accomplish a practical task. Figure 4a shows a set of \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) measurements performed with a 65-qubit geometry with B applied simultaneously to three different qubits, chosen to maximize the effective quantum volume (corresponding to the number of two-qubit gates falling within the light cones of the B and M operators31). To estimate the accuracy of these measurements, we next characterize the experimental error (SNR) across six different system sizes of up to 40 qubits (Fig. Here we observe that the SNR degrades only weakly as the system size increases and is also captured by the confidence interval of an empirical error model detailed in Supplementary Information sections II.F.3 and II.F.4. Given that classical heuristic algorithms are unable to achieve this accuracy (Supplementary Information section III.C), tensor-network contraction is the most effective approach toward classically simulating the same circuits. Figure 4c shows the estimated cost of simulating \({{\mathcal{C}}}_{\text{diag}}^{(4)}\) through tensor-network contraction on the Frontier supercomputer, which converges to approximately 3.2 years. This is a factor of approximately 13,000 longer than the experimental data collection time of 2.1 h per circuit, indicating that this experiment is, at present, in the beyond-classical regime of quantum computation. a, \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) measured on a set of 65-qubit circuits each having t = 23 cycles. B acts simultaneously on three different qubits. b, Experimental SNRs for circuits measured with system sizes ranging from 18 to 40 qubits. Error bars correspond to the 95% confidence interval of an empirical error model (Supplementary Information sections II.F.3 and II.F.4). c, Estimated time to compute \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) of a single circuit in a on the Frontier supercomputer using tensor-network contraction. The estimate was obtained by running a specially designed optimization algorithm46,47,48,49 on 20 Google Cloud virtual machines (totalling 1,200 CPUs) up to a period of 24 h (x axis). Estimates using a publicly available library cotengra47 lead to costs that are ten times higher after the same optimization time. To apply OTOCs in real-world applications, we consider a physical system of interest characterized by a Hamiltonian with a set of unknown parameters. The physical system supplies a collection of OTOC(2) data, which is compared against a quantum simulation of the same Hamiltonian. The slowly decaying signal size and sensitivity of OTOC(2), as demonstrated in Figs. 2 and  3, make it a particularly suitable candidate for accomplishing this task, which is known as Hamiltonian learning32,33,34,35. a, Scheme for applying OTOC(2) in Hamiltonian learning. OTOC(2) measured in a physical system of interest is compared with a quantum simulation of OTOC(2) using a parameterized Hamiltonian of the same system. Hamiltonian parameters are then optimized to minimize the difference between the two datasets. b, Demonstrating a one-parameter learning experiment. A collection of classically simulated \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) values from 20 circuit instances having a 34-qubit geometry (bottom left panel) are treated as data from a physical system of interest. The goal is to learn a particular phase ξ/π = 0.6 of the two-qubit gate unitary U2Q belonging to one pair of qubits (green bar in the top and bottom left panels). c, Experimentally measured \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) (quantum processor data) as a function of ξ for three different circuit instances. Blue lines indicate the ideal values of \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) from a classical simulation. These lines intersect all three datasets close to the target value of ξ (vertical dashed line). d, An optimization cost function, corresponding to the root-mean-square difference between the quantum processor data and the classical simulation data of all 20 circuit instances as a function of ξ. To demonstrate the proposed scheme in practice, we construct a one-parameter learning example, as shown in Fig. A set of \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) values from 20 random circuit instances, produced by a classical simulation to mimic the role of the ‘physical system' in Fig. All details of U, except the phase ξ of one two-qubit gate located along the passage between qm and qb, are also given. To learn the unknown parameter ξ, we measure \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) on the quantum processor while varying ξ. Results for three circuit instances are shown in Fig. 5c, where we see smooth oscillations of experimental signals that are distinct between different instances. Importantly, all oscillations intersect the classically simulated values of \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\) at the target value of ξ. This is further reflected in Fig. 5d, where we have constructed a cost function between the classically simulated and experimentally measured values of \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\). In this work, we have shown that OTOCs have quantum interference effects that endow them with a high sensitivity to details of the quantum dynamics and, for OTOC(2), also high levels of classical simulation complexity. As such, OTOCs are viable candidates for realizing practical quantum advantage, a chief milestone sought by recent experiments36,37,38. Generally, practical quantum advantage can be formulated as the task of measuring the expectation values of low-rank observables, for example, energy or correlations3,39, such that: The observable can be experimentally measured with the proper accuracy, in our case with an SNR above unity. The observable lies beyond the reach of both exact classical simulation and heuristic methods that trade accuracy for efficiency31,41,42,43,44. Satisfying both defines a ‘Goldilocks zone' for quantum advantage. To demonstrate practical quantum advantage, one more criterion is required: Here, by measuring a many-body observable with SNR > 2 and showing that it is beyond the reach of currently known classical simulation algorithms, we have made progress towards (1) and (2). Although the random circuits used in the dynamic learning demonstration remain a toy model for Hamiltonians that are of practical relevance, the scheme is readily applicable to real physical systems. One such example is solid-state nuclear magnetic resonance systems, where dipolar couplings between spin pairs can be inverted without complete knowledge of their strength45. Comparing experimental data from such systems with quantum simulation outcomes may allow more accurate estimates of these couplings. We leave this exciting real-world application for future work. The quantum processor originally comprised 105 frequency-tunable superconducting transmon qubits connected by tunable couplers. On cooling down, we found that two qubits were inoperable due to broken coupler bias lines. They were, therefore, excluded from the experiment. Details of the processor, including gate and read-out errors, qubit coherence times, anharmonicities and frequencies, can be found in Supplementary Information section II.A. Owing to improved single-qubit relaxation times T1 (median of 106 μs) compared with our earlier quantum processors, a median two-qubit gate (iSWAP-like) error of 0.15% was achieved for this device. As a system-wide benchmark, we performed a random circuit sampling experiment and estimated the overall fidelity to be 0.001 at 40 circuit cycles, which doubles the circuit volume compared with previous records and corresponds to approximately 1025 years of simulation run-time on the Frontier supercomputer using tensor-network contraction algorithms (Supplementary Information section II.B). The SNR is defined as follows. This is done to both the set of experimentally measured (\(\{{{\mathcal{C}}}_{\text{exp}}\}\)) and the set of numerically simulated quantities (\(\{{{\mathcal{C}}}_{{\rm{sim}}}\}\)). The SNR is then \(1/\sqrt{\overline{{({{\mathcal{C}}}_{\text{exp}}^{(s)}-{{\mathcal{C}}}_{{\rm{sim}}}^{(s)})}^{2}}}\), where the overline denotes averaging over all circuit instances. This definition of SNR is directly related to the Pearson correlation coefficient ρ as SNR \(=1/\sqrt{2(1-\rho )}\). For measurements of \({{\mathcal{C}}}^{(2k)}\) with respect to a single initial state, an auxiliary qubit may be entangled with qm after preparing the system in the desired initial state and then read out at the end. In our present work, we simplify this scheme by initializing qm in the \(| 0\rangle \) state, which is an eigenstate of the measurement operator M = Z. Measuring qm in the Z basis directly yields \({{\mathcal{C}}}^{(2k)}\). It is also possible to measure \({{\mathcal{C}}}^{(2k)}\) of an infinite-temperature initial state, that is a maximally mixed state with the density matrix I/2N, where I is the identity matrix and N is the number of qubits. This is done by averaging over different initial states. In Supplementary Information section II.E.4, we show measurements of infinite-temperature \({{\mathcal{C}}}^{(2)}\) after averaging over initial states in a set of 66-qubit circuits. A sizeable circuit-to-circuit fluctuation is still found in this case, indicating that the signal size of \({{\mathcal{C}}}^{(2k)}\) remains large even for infinite-temperature states. The various circuit geometries studied throughout this work were chosen with the following considerations: (1) For each system size, we placed qb and qm close to the opposing edges of a lattice. (2) The total number of circuit cycles for each system size was adjusted to be the maximum beyond which the average signal size for \({{\mathcal{C}}}_{{\rm{off-diag}}}^{(4)}\) was reduced below 0.01. These two considerations were implemented such that each small-scale geometry (for example, the six different geometries used in Fig. Each 40-qubit circuit instance in Fig. 3d,e required 3 h to simulate exactly on a Google Cloud virtual machine with 11.5 Tb of RAM and 416 CPUs. 3d using a CMC simulation required a cache size of 1 billion (see Supplementary Information section III.B.2 for a definition) and 6 days of simulation time (per circuit) on a single NVIDIA H100 GPU to gather sufficient statistics. In Supplementary Information section IV we analyse the moments of \({{\mathcal{C}}}^{(2)}\) and \({{\mathcal{C}}}^{(4)}\) theoretically and numerically for the one-dimensional bricklayer circuit of two-qubit Haar random gates. For \({{\mathcal{C}}}^{(2)}\), we used a combination of perturbation theory and large-scale matrix product state simulations to relate the power-law circuit-to-circuit fluctuations to the small interference loops introduced in the main text. For \({{\mathcal{C}}}^{(4)}\) and \({{\mathcal{C}}}_{\text{off-diag}}^{(4)}\), a similar analysis is challenging and, instead, we verified the polynomial scaling of fluctuations with exact numerics. We also demonstrate the existence of a dynamical quantum phase transition in the spectrum of the operator MB(t) (equation (2)). The properties of this transition were captured in an analytically tractable random matrix model, which allowed us to compute moments k ≥ 1. This quantum criticality is consistent with the classically intractable power-law correlations in \({{\mathcal{C}}}^{(2k)}\). 3a), which have coefficients with random signs. In Supplementary Information section III.C.3, we further argue theoretically that this interference is not an artefact of the Pauli representation but is intrinsic to \({{\mathcal{C}}}^{(4)}\). We accomplished this for the circuit-averaged \({{\mathcal{C}}}^{(4)}\), which is strictly easier to compute classically. We used a mapping between the average \({{\mathcal{C}}}^{(4)}\) and a model of a magnet that has the structure of the symmetric group of order 4, which reflects universal symmetries of the random circuit ensemble. We showed numerically that the sign problem in this model is severe. We, therefore, argue that the severe sign problem is an inevitable feature that presents a barrier for classical sampling algorithms when computing \({{\mathcal{C}}}^{(4)}\). The quantum circuits and data generated and analysed for this study are available at Zenodo (https://doi.org/10.5281/zenodo.15640502)50. The source code used for the tensor contraction cost estimates are available under an open-source licence at GitHub (https://github.com/google-research/tnco). Feynman, R. P. Simulating physics with computers. Altman, E. et al. Quantum simulators: architectures and opportunities. Baum, J., Munowitz, M., Garroway, A. N. & Pines, A. Multiple–quantum dynamics in solid state NMR. & Ovchinnikov, Y. N. Quasiclassical method in the theory of superconductivity. Shenker, S. H. & Stanford, D. Black holes and the butterfly effect. Maldacena, J., Shenker, S. H. & Stanford, D. A bound on chaos. Measuring the scrambling of quantum information. Microscopic model of quantum butterfly effect: out-of-time-order correlators and traveling combustion waves. Chaos and complexity by design. Measuring out-of-time-order correlations and multiple quantum spectra in a trapped-ion quantum magnet. Operator spreading in random unitary circuits. Operator hydrodynamics, OTOCs, and entanglement growth in systems without conservation laws. Rakovszky, T. et al. Diffusive hydrodynamics of out-of-time-ordered correlators with charge conservation. Operator spreading and the emergence of dissipative hydrodynamics under unitary evolution with conservation laws. Mi, X. et al. Information scrambling in quantum circuits. Approaching the Heisenberg limit without single-particle detection. Improving metrology with quantum scrambling. Black holes as mirrors: quantum information in random subsystems. Locality, quantum fluctuations, and scrambling. Kaufman, A. M. et al. Quantum thermalization through entanglement in an isolated many-body system. Zhou, Z. et al. Thermalization dynamics of a gauge theory on a quantum simulator. Zhang, X. et al. A superconducting quantum simulator based on a photonic-bandgap metamaterial. Andersen, T. I. et al. Thermalization and criticality on an analogue–digital quantum simulator. Braumüller, J. et al. Probing quantum information propagation with out-of-time-ordered correlators. Zanardi, P. Entanglement of quantum evolutions. Resource theory of quantum scrambling. Effective quantum volume, fidelity and computational cost of noisy quantum processing experiments. O'Brien, T. E. et al. Quantum computation of molecular structure using data from challenging-to-classically-simulate nuclear magnetic resonance experiments. Cotler, J., Schuster, T. & Mohseni, M. Information-theoretic hardness of out-of-time-order correlators. Schuster, T. et al. Learning quantum systems via out-of-time-order correlators. Evidence for the utility of quantum computing before fault tolerance. King, A. D. et al. Beyond-classical computation in quantum simulation. Haghshenas, R. et al. Digital quantum magnetism at the frontier of classical simulations. Bernstein, E. & Vazirani, U. Quantum complexity theory. Tindall, J., Fishman, M., Stoudenmire, E. M. & Sels, D. Efficient tensor network simulation of IBM's Eagle kicked Ising experiment. Begušić, T. et al. Fast and converged classical simulations of evidence for the utility of quantum computing before fault tolerance. Tindall, J. et al. Dynamics of disordered quantum systems with two- and three-dimensional tensor networks. Mauron, L. & Carleo, G. Challenging the quantum advantage frontier with large-scale classical simulations of annealing dynamics. Rhim, W.-K. et al. Time-reversal experiments in dipolar-coupled spin systems. Boixo, S. et al. Simulation of low-depth quantum circuits as complex undirected graphical models. & Kourtis, S. Hyper-optimized tensor network contraction. Kalachev, G. et al. Multi-tensor contraction for XEB verification of quantum circuits. Circuits for the main text of ‘Constructive interference at the edge of quantum ergodic dynamics'. We are grateful to A. Ashkenazi, S. Brin, S. Pichai and R. Porat for their executive sponsorship of the Google Quantum AI team and for their continued engagement and support. Google Research, Mountain View, CA, USA Dmitry A. Abanin, Rajeev Acharya, Laleh Aghababaie-Beni, Georg Aigeldinger, Ross Alcaraz, Igor Aleiner, Trond I. Andersen, Markus Ansmann, Frank Arute, Kunal Arya, Abraham Asfaw, Nikita Astrakhantsev, Juan Atalaya, Ryan Babbush, Dave Bacon, Brian Ballard, Joseph C. Bardin, Andreas Bengtsson, Alexander Bilmes, Sergio Boixo, Gina Bortoli, Alexandre Bourassa, Jenna Bovaird, Dylan Bowers, Leon Brill, Michael Broughton, David A. Browne, Brett Buchea, Bob B. Buckley, David A. Buell, Tim Burger, Brian Burkett, Nicholas Bushnell, Anthony Cabrera, Juan Campero, Hung-Shen Chang, Yu Chen, Zijun Chen, Ben Chiaro, Liang-Ying Chih, Desmond Chik, Charina Chou, Jahan Claes, Agnetta Y. Cleland, Josh Cogan, Roberto Collins, Paul Conner, William Courtney, Alexander L. Crook, Ben Curtin, Sayan Das, Laura De Lorenzo, Dripto M. Debroy, Sean Demura, Michel Devoret, Agustin Di Paolo, Paul Donohoe, Ilya Drozdov, Andrew Dunsworth, Clint Earle, Alec Eickbusch, Aviv Moshe Elbag, Mahmoud Elzouka, Catherine Erickson, Lara Faoro, Edward Farhi, Vinicius S. Ferreira, Leslie Flores Burgos, Ebrahim Forati, Austin G. Fowler, Brooks Foxen, Suhas Ganjam, Gonzalo Garcia, Robert Gasca, Élie Genois, William Giang, Craig Gidney, Dar Gilboa, Raja Gosula, Alejandro Grajales Dau, Dietrich Graumann, Alex Greene, Jonathan A. Gross, Steve Habegger, John Hall, Michael C. Hamilton, Monica Hansen, Matthew P. Harrigan, Sean D. Harrington, Stephen Heslin, Paula Heu, Oscar Higgott, Gordon Hill, Jeremy Hilton, Sabrina Hong, Hsin-Yuan Huang, Ashley Huff, William J. Huggins, Lev B. Ioffe, Sergei V. Isakov, Justin Iveland, Evan Jeffrey, Zhang Jiang, Xiaoxuan Jin, Cody Jones, Stephen Jordan, Chaitali Joshi, Pavol Juhas, Andreas Kabel, Dvir Kafri, Hui Kang, Amir H. Karamlou, Kostyantyn Kechedzhi, Julian Kelly, Trupti Khaire, Tanuj Khattar, Mostafa Khezri, Seon Kim, Robbie King, Paul V. Klimov, Andrey R. Klots, Bryce Kobrin, Alexander N. Korotkov, Fedor Kostritsa, Robin Kothari, John Mark Kreikebaum, Vladislav D. Kurilovich, David Landhuis, Tiano Lange-Dei, Brandon W. Langley, Pavel Laptev, Kim-Ming Lau, Loïck Le Guevel, Justin Ledford, Joonho Lee, Kenny Lee, Yuri D. Lensky, Shannon Leon, Brian J. Lester, Wing Yan Li, Alexander T. Lill, Wayne Liu, William P. Livingston, Aditya Locharla, Erik Lucero, Daniel Lundahl, Aaron Lunt, Sid Madhuk, Fionn D. Malone, Ashley Maloney, Salvatore Mandrà, James M. Manyika, Leigh S. Martin, Orion Martin, Steven Martin, Yossi Matias, Cameron Maxfield, Jarrod R. McClean, Matt McEwen, Seneca Meeks, Anthony Megrant, Xiao Mi, Kevin C. Miao, Amanda Mieszala, Zlatko Minev, Reza Molavi, Sebastian Molina, Shirin Montazeri, Alexis Morvan, Ramis Movassagh, Wojciech Mruczkiewicz, Ofer Naaman, Matthew Neeley, Charles Neill, Ani Nersisyan, Hartmut Neven, Michael Newman, Jiun How Ng, Anthony Nguyen, Murray Nguyen, Chia-Hung Ni, Murphy Yuezhen Niu, Logan Oas, Thomas E. O'Brien, William D. Oliver, Alex Opremcak, Kristoffer Ottosson, Andre Petukhov, Alex Pizzuto, John Platt, Rebecca Potter, Orion Pritchard, Leonid P. Pryadko, Chris Quintana, Ganesh Ramachandran, Matthew J. Reagor, John Redding, David M. Rhodes, Gabrielle Roberts, Eliott Rosenberg, Emma Rosenfeld, Pedram Roushan, Nicholas C. Rubin, Negar Saei, Daniel Sank, Kannan Sankaragomathi, Kevin J. Satzinger, Alexander Schmidhuber, Henry F. Schurkus, Christopher Schuster, Michael J. Shearn, Aaron Shorter, Noah Shutty, Vladimir Shvarts, Volodymyr Sivak, Jindra Skruzny, Spencer Small, Vadim Smelyanskiy, W. Clarke Smith, Rolando D. Somma, Sofia Springer, George Sterling, Doug Strain, Jordan Suchard, Philippe Suchsland, Aaron Szasz, Alex Sztein, Douglas Thor, Eifu Tomita, Alfredo Torres, M. Mert Torunbalci, Abeer Vaishnav, Justin Vargas, Sergey Vdovichev, Guifre Vidal, Benjamin Villalonga, Catherine Vollgraff Heidweiller, Steven Waltman, Shannon X. Wang, Brayden Ware, Kate Weber, Travis Weidel, Tom Westerhout, Theodore White, Kristi Wong, Bryan W. K. Woo, Cheng Xing, Z. Jamie Yao, Ping Yeh, Bicheng Ying, Juhwan Yoo, Noureldin Yosri, Grayson Young, Adam Zalcman, Yaxing Zhang, Ningfeng Zhu & Nicholas Zobrist Ashok Ajoy, Christian Bengs & Chongwei Zhang Department of Electrical and Computer Engineering, University of Massachusetts, Amherst, MA, USA NVIDIA Corporation, Santa Clara, CA, USA Saul Cohen, Hanfeng Gu, Ikko Hamamura & Elica Kyoseva Institute for Quantum Information and Matter, Caltech, Pasadena, CA, USA Computational and Mathematical Sciences, Caltech, Pasadena, CA, USA Department of Chemistry, Harvard University, Cambridge, MA, USA Quantum Artificial Intelligence Laboratory, NASA Ames Research Center, Moffett Field, CA, USA Department of Electrical Engineering and Computer Science, Massachusetts Institute of Technology, Cambridge, MA, USA Department of Physics and Astronomy, Dartmouth College, Hanover, NH, USA Walter Burke Institute for Theoretical Physics, Caltech, Pasadena, CA, USA Max Planck Institute for the Physics of Complex Systems, Dresden, Germany D.A.A., R. Acharya, L.A.-B., G.A., R. Alcaraz, I.A., T.I.A., M.A., F.A., K.A., A. Asfaw, N.A., J.A., R.B., D. Bacon, B. Ballard, J.C.B., A. Bengtsson, A. Bilmes, S.B., G.B., A. Bourassa, J.B., D. Bowers, L.B., M.B., D.A. Buell, T.B., B. Burkett, N.B., A.C., J. Campero, H.-S.C., Y.C., Z.C., B. Chiaro, L.-Y.C., D.C., C.C., J. Cogan, A.Y.C., J. Claes, R.C., P.C., W.C., A.L.C., B. Curtin, S. Das, L.D.L., D.M.D., S. Demura, M.D., A.D.P., P.D., I.D., A.D., C. Earle, A.E., A.M.E., M.E., C. Erickson, L.F., E. Farhi, V.S.F., L.F.B., E. Forati, A.G.F., B.F., S.G., G.G., R. Gasca, É.G., W.G., C.G., D. Gilboa, R. Gosula, A.G.D., D. Graumann, A.G., J.A.G., S. Heslin, J. Hall, M.C.H., M.H., M.P.H., S.D.H., S. Habegger, P.H., O.H., G.H., J. Hilton, S. Hong, H.-Y.H., A.H., W.J.H., L.B.I., S.V.I., J.I., E.J., Z.J., X.J., C. Jones, S.J., C. Joshi, P.J., A.K., D.K., H.K., A.H.K., K.K., J.K., T. Khaire, T. Khattar, M.K., S.K., R. King, P.V.K., R. Kothari, B.K., A.N.K., F.K., R. Kothari, J.M.K., V.D.K., D. Landhuis, T.L.-D., B.W.L., P.L., K.-M.L., L.L.G., J. Ledford, J. Lee, K.L., Y.D.L., S.L., B.J.L., W.Y.L., A.T.L., W.L., W.P.L., A. Locharla, E.L., D. Lundahl, A. Lunt, S. Madhuk, F.D.M., A. Maloney, S. Mandrà, J.M.M., L.S.M., O.M., S. Molina, Y.M., C.M., J.R.M., M.M., S. Montazeri, A. Megrant, X.M., K.C.M., A. Mieszala, Z.M., R. Movassagh, S. Meeks, S. Martin, A. Morvan, R. Molavi, W.M., O.N., M. Neeley, C.N., A. Nersisyan, H.N., M. Newman, J.H.N., A. Nguyen, M. Nguyen, C.-H.N., M.Y.N., L.O., T.E.O., W.D.O., A.O., K.O., A. Petukhov, A. Pizzuto, J.P., R.P., O.P., L.P.P., C.Q., G. Ramachandran, M.J.R., J.R., D.M.R., G. Roberts, E. Rosenberg, E. Rosenfeld, P.R., N.C.R., N. Saei, D. Strain, K.S., K.J.S., A. contributed to the classical simulation of OTOCs. All authors participated in writing and revising the paper and Supplementary Information. The authors declare no competing interests. Nature thanks Morten Kjaergaard and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Google Quantum AI and Collaborators. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251022023113.htm'>A “toxic duo” may be the hidden trigger behind Alzheimer's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 13:20:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For decades, scientists have known that Alzheimer's disease is marked by sticky plaques and tangled proteins in the brain. Yet despite decades of progress, this deeper understanding has not led to fully effective treatments. When Aβ attaches to fibrinogen, it produces unusual clots that resist breakdown. These stubborn clots are linked to inflammation and damage in blood vessels, and even very small amounts of the complex appear to set off early signs of Alzheimer's, including loss of synapses, swelling in the brain, and leaks in the blood-brain barrier. "It takes a larger amount of Aβ or fibrinogen alone to cause serious damage in the Alzheimer's brain," says Erin Norris, research associate professor in the laboratory of Sidney Strickland at Rockefeller. "But when the two complex together, you only need very small amounts of each to cause damage. "Only recently, with a number of breakthroughs in the field, did people begin to believe that the vascular system is involved in AD pathogenesis," Norris says. "Since our initial findings, we've been focused on studying the mechanisms that explain how a dysfunctional vascular system impacts AD." The researchers wanted to know how much damage it could cause on its own. They recreated low concentrations of the Aβ/fibrinogen complex in the lab and applied it to thin slices of mouse brain tissue as well as to living mice. This allowed them to observe its effects under tightly controlled conditions. "We wanted to really show the damage -- to zoom in on exactly how pre- and post-synaptic terminals were being harmed," says Research Associate Elisa Nicoloso Simões-Pires. Their experiments revealed that while Aβ and fibrinogen alone caused little harm, even small amounts of the combined complex led to major problems. It damaged synapses, increased inflammation, and disrupted the blood-brain barrier -- all signature features of Alzheimer's disease. One strength of the study is that it used both isolated brain tissue and live mice. Mice exposed to the complex, for instance, also showed elevated levels of phospho-tau181, a biomarker used in humans to detect Alzheimer's years before symptoms arise. While many mechanisms contribute to Alzheimer's, the team believes this particular pathway deserves more attention. "A lot of other factors can induce neurotoxicity, and we certainly do not propose that inhibiting this complex formation would cure AD. But perhaps targeting this complex would alleviate some of the pathologies and be even more effective in combination with other therapies." These discoveries bring researchers one step closer to understanding how damage spreads in the Alzheimer's brain -- and how stopping a single toxic interaction could make a difference. Note: Content may be edited for style and length. These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds A Strange Glow in the Milky Way May Be Our First Glimpse of Dark Matter Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a69031033/electric-car-equation-toyota/'>The Electric-Car Equation No One Wants to Talk About: One EV = 90 Hybrids</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It's a question that has plagued the automotive industry for decades as we continue to understand the environmental impact of internal combustion engines. And that's a major catalyst behind Toyota's strategy to prioritize the construction and sale of hybrid vehicles over electric vehicles. Toyota's internal strategy suggests that the raw materials for one electric vehicle could be used to build six plug-in hybrids or 90 traditional hybrids; Kerry Creech, the president of TMMK, said during a roundtable that the main motivation was to optimize resource efficiency. That's also not to discount that the carbon reduction achieved by building (and driving) 90 hybrids would not be insignificant compared to selling just one electric vehicle. The biggest brains in automotive engineering have been working to reduce carbon emissions for decades. It's not rocket science that EVs emit zero carbon, but they never became mainstream in the way many expected them to. Range anxiety remains a significant concern for many motorists who are on the fence about purchasing an electric vehicle as their next automobile. Hybrids and plug-in hybrids are infinitely more attractive, as they give customers a taste of all-electric power that's supplemented by good old-fashioned gasoline. While many brands like Tesla are engineering cutting-edge tech, Toyota's more nuanced approach is selling customers what they actually want. The latter stance resonates with a greater number of buyers who aren't quite ready to make the jump to an electric vehicle. For context, let's take a look at Toyota's RAV4 Prime plug-in hybrid. Unlike conventional hybrids, which use electric drive to supplement the internal combustion engine, plug-in hybrids can also travel a limited range on electric power alone. For those of you who are still driving into an office five days a week, research says the average American drives approximately 30 miles a day. We should clarify that while Toyota's manufacturing initiative prioritizes the production of hybrids, these vehicles are actually quite complicated—even compared to modern EVs. In fact, getting an internal combustion engine and electric propulsion system to behave is just the tip of the iceberg when it comes to making these vehicles tick. As you may know, hybrid systems have the engine clicking on and off constantly; while that's great for efficiency, it's generally not enough runtime to get the fluids properly warmed up. So much so, in fact, that hybrids often use lower viscosity oils to cope with the lower operating temperatures induced by these stop-start scenarios. If it wasn't abundantly clear, hybrid vehicles still make a ton of sense. If you're in the market for a new vehicle, we'd still find it hard to recommend an EV over a straightforward hybrid. Scout's latest vehicles will have an available Extended-Range Electric Vehicle (EREV) powertrain featuring electric drive that's supported by an internal combustion “range extender.” It's much simpler than a “conventional” hybrid system as the internal combustion engine isn't connected to the wheels—it's merely acting as a generator. However, Scout's EREV architecture appears to be a considerable advancement over the half-baked powertrain found in BMW's kitschy hatchback. Assuming the numbers are right, it could potentially offer more than 500 miles of range on a single charge. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. The Air Force Is Looking To Blow Up a Cybertruck Tesla's Suspension Can Blow If You Skip This Step</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69102298/frequency-comb-chip/'>A Freak Accident Might Have Just Unlocked Faster Internet for Everyone</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It could also help AI data centers move massive amounts of information more effectively. We may earn commission if you buy from a link. In many ways, science is a story of happy accidents. This one may not rewrite our understanding of gravity or usher in a new era of medicine, but it could be a huge boon for another world changing technology: artificial intelligence. In the study, published earlier this month in the journal Nature Photonics, scientists detail how exploring ways to improve Light Detection and Ranging (LiDAR) accidentally led to the discovery of a chip-sized frequency comb, a special kind of light where color bands are lined up next to each other and bright frequencies appear as spikes on spectrogram—sort of like teeth on a comb. “That means you can replace racks of individual lasers with one compact device, cutting cost, saving space, and opening the door to much faster, more energy-efficient systems.” To pull this off, Gil-Molina and the research team relied on multimode laser diodes, which are typically used in medical devices and laser-cutting tools. Not exactly known for being neat and orderly beams of light, researchers used a technique known as a “locking mechanism” to essentially “purify” the noisy light, Gil-Molina says. This method uses silicon photonics to clean up the laser's output so that it displays high coherence. Then the chip splits this rarified laser light into dozens of colors. Because these colors don't interfere with one another, each optical hue is its own channel, allowing for the possibility of sending dozens of data streams simultaneously. What's different is that frequency combs typically require powerful and expensive lasers and amplifiers, but this study shows that they can achieve the same trick on just a single chip. “This is about bringing lab-grade light sources into real-world devices,” says Gil-Molina. The most obvious application for this new kind of chip is AI data centers, which are increasingly feeling the strain to move information fast enough, especially between processors and memory. While many data centers still use single-wavelength lasers along fiber optic cables, a compact frequency comb can send multiple data streams along that same cable. “This research marks another milestone in our mission to advance silicon photonics,” Columbia University's Michal Lipson, the senior author of the study, said in a press statement. The Universe Keeps Teaching Itself New Physics, a Controversial Theory Says</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02361-5'>Pathogenic UNC13A variants cause a neurodevelopmental syndrome by impairing synaptic function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 09:11:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The UNC13A gene encodes a presynaptic protein that is crucial for setting the strength and dynamics of information transfer between neurons. Here we describe a neurodevelopmental syndrome caused by germline coding or splice-site variants in UNC13A. The syndrome presents with variable degrees of developmental delay and intellectual disability, seizures of different types, tremor and dyskinetic movements and, in some cases, death in early childhood. Using assays with expression of UNC13A variants in mouse hippocampal neurons and in Caenorhabditis elegans, we identify three mechanisms of pathogenicity, including reduction in synaptic strength caused by reduced UNC13A protein expression, increased neurotransmission caused by UNC13A gain-of-function and impaired regulation of neurotransmission by second messenger signalling. Based on a strong genotype–phenotype-functional correlation, we classify three UNC13A syndrome subtypes (types A–C). We conclude that the precise regulation of neurotransmitter release by UNC13A is critical for human nervous system function. UNC13A, the major UNC13 paralog in mammals, is a highly conserved presynaptic protein that is essential for chemical synaptic transmission. It is required for assembly of the SNARE complex that bridges between neurotransmitter-filled synaptic vesicles (SVs) and the plasma membrane, placing SVs in a fusion-competent, ‘primed' state1,2,3,4,5. During periods of high-frequency synaptic activity, UNC13A supports the resupply of SVs to counterbalance SV consumption10,11,12,13,14, sustaining neurotransmission. UNC13A activity is enhanced through direct binding of Ca2+, Ca2+-calmodulin, Ca2+-phospholipids and diacylglycerol (DAG) to specific regulatory domains. The acceleration of UNC13A activity results in dynamic changes in neurotransmission strength, a phenomenon known as synaptic plasticity10,11,12,13,15,16. Most neuronal subtypes additionally express the paralogs Unc13b and/or Unc13c17, which have similar functions. Mice that constitutively lack Unc13a die shortly after birth19,20 owing to a 90% loss of the RRP at excitatory synapses19. The combined elimination of Unc13a/Unc13b results in a complete block of neurotransmission at both excitatory and inhibitory synapses owing to complete RRP loss20. Consequently, newborn Unc13a/b double knockout (DKO) mice are paralyzed but have intact nervous system development20,21. A pivotal role for UNC13A in human neurological disorders is emerging. Two patients harboring homozygous truncating variants in UNC13A showed profound developmental delay and died during early childhood22,23. Another patient harboring a de novo missense variant (P814L) in UNC13A exhibited mild intellectual disability, dyskinesia and intention tremor24. Certain deep intronic single-nucleotide polymorphisms contribute to genetic risk for the neurodegenerative conditions amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD)25,26,27, and promote ALS/FTD disease progression26,27,28,29,30,31,32,33. Recent studies identified reduced UNC13A expression in neurons with ALS/FTD pathology owing to TDP-43-mediated alterations in splicing34,35, but the mechanisms linking reduced UNC13A expression to disease progression remain unknown33,36. We identified a neurodevelopmental syndrome caused by variants in the UNC13A gene. Systematic patient and variant characterization enabled classification of three disease subtypes. Patients with a type A condition present with profound global developmental delay (GDD) and early-onset seizures. These patients harbor biallelic loss-of-function missense, truncating or splice-site variants that lead to a >50% reduction of UNC13A expression and to a severe reduction in neurotransmission strength in experimental models. Patients with a type B condition exhibit developmental delay, particularly speech delay, and ataxia, tremor or dyskinetic movements as hallmarks of the condition. These patients harbor de novo missense variants that result in a gain of UNC13A function, leading to enhanced neurotransmission. The type C condition is caused by a familial heterozygous missense variant that results in altered regulation of UNC13A function. The patients are mildly affected, exhibiting learning difficulties to mild–moderate intellectual disability and seizures. Overall, we demonstrate that UNC13A disease-causing variants operate in a cell-autonomous fashion to change synaptic strength and plasticity. Importantly, we present evidence that reduction of UNC13A protein expression to 20–30% of wild-type (WT) levels strongly impairs synaptic transmission and plasticity and has severe consequences in humans, a finding that may be important to understand the role of UNC13A in ALS and FTD. We identified diverse UNC13A variants in 48 index patients presenting with neurodevelopmental deficiencies (Supplementary Data 1). Based on genotype–phenotype-functional assessment, we classified variants in 20 patients as pathogenic (Fig. 1a) and in eight patients as (likely) benign. a, Location of validated pathogenic variants identified in 20 patients (Supplementary Data 1) mapped on a schematic representation of the UNC13A gene (NM_001080421.2). Variants with biallelic inheritance are in magenta (p.T117Rfs*18 and p.R202H, and c.767+1 G > T and c.4073+1 G > A are in a compound heterozygous state; others are in a homozygous state), heterozygous de novo variants are in black and a heterozygous, inherited variant is in brown. Exon colors correspond to the color of protein domains in b. b, Validated pathogenic missense variants overlaid on a schematic representation of the UNC13A protein. c, UNC13A tolerance landscape generated based on the observed missense and synonymous variants in gnomAD and corrected for the sequence composition (MetaDome database37). The degree of tolerance for missense variants is color-coded (lowest scores, red, highly intolerant; highest scores, blue, highly tolerant). d, AlphaMissense scores38 plotted for all missense variants (pathogenic, of uncertain significance (VUS) and (likely) benign) reported in this study (Supplementary Data 1; heterozygous de novo missense variants, bold typeface; heterozygous inherited or biallelic variants, non-bold typeface; validated pathogenic variants, red; VUS, gray; validated (likely) benign, blue). Notably, all validated pathogenic variants reported in this study are within the pathogenic AlphaMissense score range, and three out of four validated benign variants fall within the benign score range. We consider VUS variants in the pathogenic AlphaMissense score range as hot VUS (14 variants) and those within the benign/ambiguous score range as cold VUS (12 variants). e, Left: age of patients (with median and interquartile range; error bars, s.d.) at the latest physician visit, stratified by inheritance pattern. Middle: major clinical features of 20 patients with pathogenic variants (Supplementary Data 1), stratified by inheritance pattern. Right: standard deviation scores for growth parameters of patients with pathogenic variants at birth and at the latest visit are illustrated by inheritance pattern (HC, head circumference). A first group of six patients (18 months to ~15 years old; Fig. 2 and 3) presented with severe-to-profound GDD or intellectual disability, hypotonia and seizures of different types (largely controllable with medication) or death in early childhood caused by respiratory failure after pneumonia in one case. UNC13A variants in these patients were homozygous or compound heterozygous missense, insertion–deletion or splice-site variants with gene-disrupting splicing effects proven by minigene assays (Supplementary Data 1 and Extended Data Figs. Notably, parents of patients with biallelic variants in this study who were heterozygous carriers of loss-of-function missense or gene-disrupting splice-site variants, and parents of an individual in a previously published case22 with a homozygous gene-disrupting variant were all reportedly healthy, demonstrating possible tolerance to heterozygous loss-of-function UNC13A variants, as has also been shown in mice19. A second group of 13 patients (21 months to 32 years old; Fig. 1a,b (black) and Supplementary Data 1) harbored pathogenic, heterozygous de novo missense variants with multiple substitutions at amino acids 808, 811 and 814. They presented with variable degrees of GDD, hypotonia, seizures of different types (mainly refractory to treatment) and typically exhibited ataxia, tremor or dyskinetic movements rarely observed in other patients (Supplementary Data 1 and Supplementary Videos 1 and 2). The protein region spanning the seven highly conserved amino acids 808–814 forms a hotspot for an autosomal dominant pattern of inheritance and is predicted to be highly intolerant to variation based on the MetaDome database37 and AlphaMissense scores38 (Fig. We term this region the ‘UNC13 hinge' because it links a regulatory domain cluster with the MUN domain, which mediates SNARE complex assembly39,40 (Fig. The third group of patients consisted of a family with at least four affected members across two generations (4 years to 35 years old; Fig. 1a,b (brown) and Supplementary Data 1) harboring a pathogenic heterozygous missense variant (C587F) that caused learning difficulties to mild–moderate intellectual disability as well as controlled seizures. The relatively mild presentation of this variant led to autosomal dominant familial heritability. Altogether, pathogenic variants in UNC13A cause a spectrum of neurodevelopmental deficiencies (Fig. 1e; mainly GDD or intellectual disability, hypotonia, seizures and abnormal movement features) with both autosomal dominant (de novo or inherited) and recessive inheritance, depending on the type and location of the variants. Additionally, we identified de novo heterozygous or biallelic variants of uncertain significance scattered throughout the UNC13A protein sequence (Extended Data Fig. Although many of these variants are high-level (hot) candidates for impacting protein function based on in silico predictions (Fig. 1d and Supplementary Data 1), reliable readouts proving their pathogenicity are currently unavailable. We selected a subset of variants with substitutions at key UNC13A domains (Fig. 2a) and comprehensively characterized their impact on synapse function. We show that variants E52K, R202H, C587F, G808D/C and P814L are pathogenic, while our assays did not detect phenotypic changes for R799Q and N1013S, which we therefore classify as (likely) benign (Supplementary Data 1–3 and Extended Data Fig. A comparative summary of all genetic, clinical and experimental data is presented in Extended Data Fig. a, Location of studied variants within the UNC13A protein domains (not drawn to scale). b, Schematic representation of the autaptic cell culture system used in this study. c, Left: example images of cultured autaptic hippocampal neurons from Unc13a/b DKO mice that were transfected on DIV 2–3 with UNC13A–GFP cDNA encoding WT or disease-related variants and stained with antibodies against the presynaptic marker VGLUT1, the postsynaptic marker Shank2, the dendritic marker MAP2 and with an antibody against GFP (n = 24–37). d, Quantification of VGLUT1–Shank2 juxtapositioned puncta per neuron. e, Fraction of VGLUT1–Shank2 juxtapositioned puncta where UNC13A–GFP co-staining was detected. The P values in e refer to pairwise comparisons of mean fractions with WT using Dunn's multiple comparison test. f, Cumulative probability distributions of UNC13A–GFP intensities at VGLUT1–Shank2 juxtapositioned puncta, reporting relative UNC13A expression levels at individual synapses. For each cell analyzed, the mean Munc13 intensity over all synapses was obtained and, subsequently, the pooled mean was calculated for each genotype (Supplementary Data 2). The P values in f refer to pairwise comparisons of mean intensities with WT using Tukey's multiple comparisons test. See Supplementary Data 2 for further details. Neurodevelopmental disorders have been linked to deficiencies in synapse development41. To examine whether UNC13A variants affect synaptogenesis or presynaptic UNC13A abundance, we cultured hippocampal neurons from Unc13a/b DKO mice in a microisland culture system42,43 (Fig. 2b), where single neurons grow on astrocytic islands, making synapses onto themselves (‘autapses'). We re-introduced WT rat Unc13a cDNA (UNC13AWT; ~94% identity with human UNC13A) or UNC13A cDNA encoding disease-related variants using lentiviral transduction44 at day in vitro (DIV) 2–3. Following immunocytochemical labeling, synapses were defined as VGLUT1-positive puncta (a presynaptic marker) colocalizing with Shank2-positive puncta (a postsynaptic marker). We found no significant change in the number of formed synapses between the different conditions (Fig. We then quantified the fraction of VGLUT1–Shank2 puncta exhibiting UNC13A immunoreactivity (Fig. UNC13AWT was detected in 93.4 ± 1.75% of the VGLUT1–Shank2-positive puncta, and similar colocalization levels were detected for UNC13A with variants in the Ca2+-phospholipid binding C2B domain (UNC13AR799Q), UNC13A hinge (UNC13AG808D) and MUN domain (UNC13AN1013S). Neurons expressing a UNC13A variant in the DAG-binding C1 domain (UNC13AC587F) exhibited a mild but statistically significant reduction in the degree of colocalization. Remarkably, neurons expressing either of the two amino-terminal variants, UNC13AE52K and UNC13AR202H, displayed a strong reduction in the fraction of synapses co-labeled for UNC13A (35 ± 3.9% and 32.5 ± 4% of WT levels, respectively). Moreover, substantially weaker UNC13A labeling intensities were evident (UNC13AE52K, 23 ± 11% of WT levels; UNC13AR202H, 20 ± 13% of WT levels; Fig. Taken together, UNC13A disease variants do not affect synapse number in vitro, but the E52K and R202H substitutions cause a strong reduction of synaptic UNC13A levels. 3a,b), while R202 is part of an intrinsically disordered region (IDR; amino acids 151 to ~450; Fig. The E52K and R202H disease variants are inherited in an autosomal recessive manner and cause a severe disorder (Fig. Expression of UNC13AE52K or UNC13AR202H in HEK293 cells, which lack UNC13A but express the misfolded protein detection machinery, showed no evidence that these variants cause inherent protein instability (Extended Data Fig. a, Amino acid sequence alignment of the C2A domain and adjacent region for UNC13A homologs in different species (disease variants are highlighted by a bar; E52K, orange; R202H, green; HS, Homo sapiens; RN, Rattus norvegicus; CE, C. elegans). b, Structure of the UNC13A C2A domain (orange) in complex with the RIM zinc-finger domain (blue) (PDB 2CJS)72. E52 is shown in a space-filled presentation. c, Pictures of three patients with pathogenic homozygous amino-terminal variants, showing narrow forehead, highly arched eyebrows, impression of hypertelorism, depressed nasal bridge, small nasal tip and underdeveloped, anteverted ala nasi, long philtrum, impression of retrognathia and large, mildly forward-facing earlobes. d, Representative pedigrees of patients with pathogenic biallelic amino-terminal variants, illustrating an autosomal recessive pattern of inheritance with apparently healthy carrier parents. e, Schematic illustration of electrophysiological recordings in autaptic neurons. g–m, Characterization of synaptic transmission in autaptic hippocampal neurons expressing UNC13AWT (black), UNC13AE52K (orange) or GFP-NLS (gray). Example traces (g) and summary data showing mean frequencies (h) of mEPSCs. Example traces (i) and summary data showing mean charges (j) of eEPSCs. Example traces (k) and summary data showing mean charges (l) of sucrose-evoked EPSCs as a measure of RRP size. No eEPSCs were elicited in UNC13AE52K-expressing neurons during high-frequency AP trains (40 Hz) (m). Example traces (o), mean amplitudes (p) and mean frequencies (q) of spontaneously occurring mEPSCs in neurons expressing UNC13AWT (black) and UNC13AR202H (green). Smaller sucrose-evoked EPSCs (t) indicate a reduced RRP size in UNC13AE52K-expressing neurons (u), while the mean vesicular release probability was unchanged (v). Data were obtained from at least three independent cultures per condition. See Supplementary Data 3 for further details. To assess how variants affect synaptic transmission, we conducted whole-cell voltage-clamp recordings in autaptic Unc13a/b DKO glutamatergic hippocampal neurons (Fig. 3e)20,42 transfected with UNC13AWT or UNC13A carrying disease variants. 3f), we could not detect spontaneous miniature excitatory postsynaptic currents (mEPSCs) or action potential (AP)-evoked excitatory postsynaptic currents (eEPSCs; Fig. Given that high-frequency AP firing can enhance UNC13A activity, we delivered trains of 40 APs at 40 Hz frequency but did not detect synaptic currents throughout the trains (Fig. Together, we conclude that UNC13AE52K does not rescue synaptic transmission to measurable levels. 3n) rescued synaptic transmission, albeit with lower efficacy than UNC13AWT. Mean frequency and amplitude of mEPSCs were similar to those measured in neurons expressing UNC13AWT (Fig. 3o–q), but the average eEPSC size (Fig. The mean vesicular release probability (\({\bar{p}}_{\mathrm{vr}}\)), calculated as the charge transfer during a single AP-induced eEPSC divided by the charge transfer in response to sucrose-evoked RRP release, was unaltered (Fig. Taken together, these results demonstrate that two variants cause lower synaptic UNC13A abundance and reduced synaptic strength in mouse neurons. Affected patients presented with profound GDD and no motor development (Figs. 1e and 3c), consistent with decreased synaptic efficacy observed in vitro. Interestingly, although both substitutions reduce UNC13A levels to similar extents (Fig. The substantial neurotransmission in UNC13R202H-expressing neurons indicates that very low UNC13A levels are sufficient for synapse function. We identified 13 patients with de novo variants in the UNC13A hinge (Fig. We previously reported one patient with an UNC13A P814L variant24, causing a gain-of-function in synaptic transmission. To consolidate variant pathogenicity and establish functional causality between the expression of hinge variants and synaptic transmission properties, we studied the frequently affected residue G808 (Fig. a, A sequence alignment of UNC13A homologs illustrating that the UNC13 hinge domain is conserved (residues found mutated in patients in bold). b, Facial portraits of patients with pathogenic de novo hinge variants, displaying high anterior hairline with an appearance of tall forehead, impression of hypertelorism, deeply set eyes, attached earlobes, smooth, rather short philtrum, thin upper lip vermilion, everted lower lip vermilion, bulbous nasal tip and retrognathia. c, Representative pedigrees of patients with pathogenic de novo variants, illustrating an autosomal dominant pattern of inheritance. d, Crystal structure of the UNC13A (PDB 5UE8) identifies the UNC13A hinge as non-structured (purple), connecting two structured domains. e–l, Characterization of synaptic transmission in autaptic hippocampal neurons expressing UNC13AWT (gray) or UNC13AG808D (blue). Example traces (e) and summary data showing mean amplitudes (f) and mean frequencies (g) of mEPSCs. Example traces (h) of sucrose-evoked EPSCs and summary plots of the measured RRP size (i). Example traces (j) of eEPSCs and summary plots of eEPSC charge (k) and vesicular release probability (l). m, Schematic illustration of the aldicarb assay in C. elegans. Aldicarb-induced accumulation of ACh in the extrasynaptic space leads to receptor desensitization and to worm paralysis. Synaptic phenotypes associated with ACh hypersecretion lead to faster ACh accumulation and to faster paralysis. Reduced secretion results in slower paralysis. n, CRISPR-generated heterozygous (green) and homozygous (blue) knock-in UNC-13P814L (UNC-13P956L) worms exhibit faster aldicarb-induced paralysis than WT worms. o, Similar results obtained in heterozygous (green) or homozygous (blue) UNC-13G808C (UNC-13G950C) knock-in worms. Data were obtained from at least three independent cultures per condition. Circles in n and o represent averaged data from >10 worms. See Supplementary Data 3 for further details. In autaptic hippocampal Unc13a/b DKO neurons expressing UNC13AG808D–GFP, we observed a greater than twofold increase in the frequency of mEPSCs compared to neurons expressing the WT variant, whereas mEPSC amplitudes were unaltered (Fig. Given that larger RRP size could underlie an increased mEPSC frequency, we estimated the RRP sizes and found similar values in UNC13AG808D-expressing and UNC13AWT-expressing neurons (Fig. Together, these findings are consistent with the notion that RRP SVs in resting UNC13AG808D-expressing neurons are more likely to fuse spontaneously, possibly indicating higher SV fusogenicity49 or a larger fraction of ‘tightly-docked' SVs50. UNC13AG808D-expressing neurons also showed significantly larger eEPSC charges (Fig. Given that the average mEPSC charge and RRP size were unaltered (Fig. 4f,i), the increased eEPSC charge is indicative of a larger fraction of the RRP released by single APs (higher \({\bar{p}}_{\mathrm{vr}}\); Fig. We conclude that UNC13AG808D leads to a gain-of-function in spontaneous and evoked neurotransmission. Given that all UNC13A hinge variants were heterozygous, we tested whether a gain-of-function phenotype prevails in the presence of a WT allele. We generated homozygous and heterozygous C. elegans strains harboring the G808C and P814L substitutions using CRISPR–Cas9 targeting of the unc-13 gene. The C. elegans and human UNC13A protein products are highly homologous (~50% identity, ~70% sequence similarity) and serve similar functions51,52,53,54. As a behavioral readout of synaptic function, we assayed loss of movement in response to exposure to the acetylcholinesterase inhibitor aldicarb (Fig. 4m), with faster paralysis indicating acetylcholine hypersecretion. These data are consistent with a gain-of-function in acetylcholine secretion, also in the presence of a WT allele. Based on these results, we conclude that UNC13A hinge variations increase the degree of spontaneous and evoked neurotransmission. 5), we conclude that UNC13A hinge variations are pathogenic and act through a similar mechanism. We identified four patients harboring a heterozygous missense variant (c.1760_1761delinsTT; C587F) in the UNC13A DAG-binding C1 domain (Fig. In cultured WT synapses, the membrane-permeable DAG analog Phorbol 12,13-dibutyrate (PDBu) causes a greater than twofold increase in synaptic strength within seconds after application15,56. To test whether C587F hampers DAG sensing, we used a protein-translocation assay. In HEK293 cells, overexpressed UNC13AWT–GFP undergoes translocation to the plasma membrane upon PDBu application (Fig. A C1 domain point mutation, H554K, blocks this translocation14 (Fig. We found that the disease variant C587F blocks UNC13A translocation, while other variants in this study exhibit WT-like behavior (Fig. a, Sequence alignment for UNC13 paralogs highlighting H554 (blue) and C587 (bold). b, Facial portraits of index patient 6, his father and aunt, carrying the pathogenic variant, displaying a high, broad forehead, impression of hypertelorism, deeply set eyes, fullness of the upper lateral eyelid, depressed nasal bridge, underdeveloped ala nasi and small nasal tip, smooth, broad philtrum and everted vermilion of the upper lip. c, A four-generation pedigree, with four individuals confirmed as harboring c.1760 G > T, p.(C587F). d, The C1 domain structure (top) and a zoomed-in view of one Zn2+-binding pocket. e, HEK293FT cells transiently transfected with UNC13A–GFP constructs encoding WT or patient variants, treated with 1 µM PDBu for 1 h (scale bar, 5 µm; n = 10–15 per condition; white arrowheads highlight UNC13A location (green)). g–n, Characterization of synaptic transmission in autaptic hippocampal neurons expressing UNC13AWT (gray) or UNC13AC587F (red). Example traces (g) and summary plots showing mean amplitude (h) and mean mEPSC frequency (i). Example traces (j) and summary plots showing mean eEPSC charge (k). Example traces (l) and summary plots showing mean sucrose-evoked EPSC charge (m). n, Quantification of vesicular release probability. o, Time to paralysis in CRISPR-targeted C. elegans worms carrying a homozygous C587F (C729F) variation and exposed to aldicarb. p,q, Example eEPSCs before and after application of PDBu in neurons expressing UNC13AWT (p) or UNC13AC587F (q). r, The average time courses of eEPSC amplitude change. In o and r–t, circles represent an average from >10 worms or an average of the indicated number of neurons. Data were obtained from at least three independent cultures per condition; error bars, s.e.m. Statistical analysis was performed by using a two-tailed Mann–Whitney test (f, h, i, k, m, n) or Wilcoxon test (o). See Supplementary Data 3 for further details. We then characterized synaptic transmission and DAG responsiveness in autaptic neurons expressing UNC13AC587F (Fig. Basal spontaneous and AP-evoked synaptic transmission were largely similar in Unc13a/b DKO neurons expressing UNC13AC587F or UNC13AWT. This is consistent with a similar time course of aldicarb-induced paralysis in homozygous C587F worms and WT worms (Fig. However, PDBu failed to potentiate eEPSC amplitudes in autaptic neurons (Fig. 5p–r), in agreement with the absence of PDBu-induced translocation (Fig. We conclude that the UNC13AC587F variation acutely changes the responsiveness of UNC13A to DAG signaling, while not strongly interfering with basal synaptic transmission (see Extended Data Fig. Finally, we tested whether other disease variants impair the PDBu-induced synaptic potentiation. Expression of UNC13AG808D, with a presumably intact C1 domain (Fig. 5e), resulted in little PDBu-induced potentiation (~20% of WT; Fig. This is probably a consequence of already altered basal synaptic transmission properties: synaptic strength is enhanced in neurons expressing UNC13AG808D, which may partially occlude PDBu potentiation. By contrast, neurons expressing UNC13AR202H exhibit lower basal synaptic strength, which may allow stronger potentiation. These data suggest an unbalanced DAG signaling in UNC13AG808D-expressing and UNC13AR202H-expressing neurons. During repetitive activation, synapses exhibit short-term plasticity (STP)57, a use-dependent alteration of synaptic strength on a time scale of milliseconds to minutes. STP occurs when fusion-competent SVs are progressively consumed. We next studied how disease variants in UNC13A affect this process. We recorded eEPSCs during high-frequency trains, embedded in low-frequency stimulation (Fig. 6b–d,f–h): in neurons expressing UNC13AC587F or UNC13AG808D, prominent short-term synaptic depression of eEPSC amplitudes was observed (Fig. 6c,d,g,h (gray arrow)), whereas UNC13AR202H-expressing neurons typically exhibited short-term synaptic enhancement (Fig. To explore underlying mechanisms, we quantified the replenishment rate constants of the RRP during 40 Hz AP trains, based on a single-pool model of SV priming12,58. RRP replenishment rate constants were reduced for neurons expressing UNC13AC587F or UNC13AG808D but tended to be accelerated in UNC13AR202H-expressing neurons (Fig. a–d, Stimulation protocol (a) and average time courses of normalized eEPSC amplitudes recorded in autaptic hippocampal neurons expressing UNC13AWT (gray) or UNC13AR202H (b, green), UNC13AC587F (c, red) or UNC13AG808D (d, blue), before, during and after a 10 Hz train consisting of 30 APs. e–h, Average time courses of normalized eEPSC amplitudes recorded before, during and after a 40 Hz train consisting of 40 APs (e) in neurons expressing UNC13AWT (gray) or UNC13AR202H (f, green), UNC13AC587F (g, red), or UNC13AG808D (h, blue). i, Replenishment rate constants calculated based on a single-pool model of SV priming for neurons of the indicated genotypes. j,k, Magnitude of eEPSC augmentation after conditioning 10 Hz (j) or 40 Hz (k) stimulation. Data were obtained from at least three independent cultures per condition. Statistical analysis was performed by using a two-tailed Mann–Whitney test (i–k). Following AP trains, synaptic strength can transiently increase (Fig. We quantified such augmentation by measuring eEPSCs 2 s after the cessation of AP trains. Collectively, these data demonstrate that UNC13A disease variants have a complex impact on STP. Reduced UNC13A expression favors facilitation of synaptic responses during and strong augmentation after an AP train, while gain of UNC13A function or diminished UNC13A PDBu/DAG regulation leads to increased depression during and reduced augmentation after AP trains. We report the identification of a neurodevelopmental syndrome caused by pathogenic UNC13A variants that cause cell-autonomous changes to synaptic transmission and plasticity. STXBP1 encodes the MUNC18-1 protein, a functional interactor of UNC13A in SNARE complex assembly61,62,63. Based on the genetic, clinical and functional evidence presented here (Extended Data Fig. 5c), we propose a categorization of the UNC13A syndrome. Type A comprises variants that cause full or partial UNC13A protein loss. This type is caused by biallelic missense (Fig. These variants lead to profound GDD in all cases, and in some, cause death in early childhood owing to respiratory failure after pneumonia (Fig. All patients develop early-onset seizures that mostly respond to treatment. Based on a previously reported case23 in which an individual was diagnosed with fatal myasthenia according to electromyography findings, and based on patients 3 and 7 in our study (Supplementary Data 1 and Extended Data Figs. 2 and 3), we suggest considering myasthenic presentation in type A patients. Type A variants are associated with eliminated or strongly reduced UNC13A expression, weaker synaptic transmission under basal and low-frequency activity (Fig. 3) and a tendency towards enhanced synaptic responses during and after AP trains, or by DAG signaling (Figs. We speculate that these variants impair UNC13A transport and/or active zone recruitment. The type B UNC13A condition comprises variants that cause an UNC13A gain-of-function. This type is caused by de novo heterozygous missense variants in the UNC13A hinge (G808D, G808C, G808V, K811E, P814L) (Figs. The overall severity of the developmental delay is less than that observed in type A, and no death in early childhood is reported. Epileptic seizures are severe and become refractory to treatment in most cases. A hallmark of this condition is ataxia and tremor or dyskinetic movements24 (Fig. As UNC13A is also expressed in motoneurons, it remains to be determined whether the imbalanced synaptic transmission at the neuromuscular junction contributes to this presentation. Functionally, type B variants cause a gain-of-function of synaptic transmission during spontaneous and low-frequency AP activity (Fig. Finally, the type C condition comprises a variant that results in UNC13A dysregulation. This type is caused by a familial heterozygous missense variant in the UNC13A C1 domain (C587F; Figs. 1 and 5), presenting with mild delay in speech development, learning difficulties to mild–moderate intellectual disability and controlled seizures (Supplementary Data 1). Neurons expressing this variant show near-normal synaptic transmission in response to low-frequency AP activity, but absent potentiation by DAG signaling (Fig. We anticipate additional pathogenic variants and disease subtypes to emerge. In particular, loss of function without loss of protein is expected for variants that interfere with UNC13A function in SNARE complex assembly (for example, in the MUN or C2C domain), which may lead to a type A-like condition. Based on AlphaMissense scores and the rareness of variants in the hinge region (frequency in gnomADv4.1 database), almost all possible hinge amino acid exchanges (42 out of 45; 93%) show pathogenic predictions, and we anticipate additional variants to arise. UNC13A hyperfunction has been described in structure–function studies46,53,64, and variants in these regions may also lead to a type B condition. Our functional assays were designed to investigate deficits caused by UNC13A variants and were therefore performed in the absence of a WT UNC13A allele (which is relevant for heterozygous variants) and in the absence of UNC13B/C. Co-expression of UNC13A/B/C is expected to modify the functional phenotypes we describe in a synapse-specific fashion, but to what extent is hard to predict and requires further experimentation. Interestingly, UNC13B expression can fully compensate for UNC13A loss in hippocampal inhibitory neurons, but UNC13B/C can only marginally compensate for UNC13A loss (<10%) in excitatory hippocampal synapses20,65. Reasons for the limited redundancy remain unclear and may involve active zone recruitment mechanisms66,67. Nonetheless, following full or partial loss of UNC13A (type A), co-expression of UNC13B may support synaptic transmission in inhibitory neurons to a larger extent than in excitatory neurons, shaping the inhibition-to-excitation ratio within neuronal networks. It is tempting to speculate that this might underlie the favorable response of epileptic seizures to medication in type A patients. In the type B condition, we demonstrated that UNC13AP814L exerts a similar effect in excitatory hippocampal and inhibitory striatal neurons24 and that the magnitude of the functional gain is similar in the presence or absence of UNC13AWT (Fig. A key finding in this study is the identification of variants that reduced UNC13A expression levels below 50% of WT levels (E52K, R202H and, probably, some splice-site variants). Reduction of UNC13A expression levels has been recently highlighted as a pathogenic mechanism in ALS and FTD. In healthy neurons, TDP-43 acts as a splicing regulator for UNC13A mRNA34,35,68, in which it represses the inclusion of a cryptic exon. Deep intronic UNC13A variants that possibly change the cryptic exon's splicing propensity pose a strong genetic risk for ALS/FTD disease and disease progression25,27,34,35. How reduced UNC13A levels enhance ALS/FTD pathology remains to be established, and the minimal UNC13A levels required for proper synaptic function have not yet been determined. Several observations made here shed light on these issues. We report that the R202H substitution severely interferes with human motor function (Fig. Although we cannot exclude that R202H has additional effects on UNC13A function in addition to reducing its expression levels, these data suggest that altered neurotransmission may accompany ALS/FTD cellular pathology as UNC13A levels decline and potentially exacerbate disease symptoms. However, we also note that 20–30% of UNC13A expression levels are already sufficient to support ~60% of neurotransmitter release in cultured neurons and that patients with low UNC13A expression levels have improved survival chances compared with patients with no functional UNC13A, who die in early childhood22,23. Together, we propose that therapeutic strategies that stabilize 20–30% as minimal UNC13A expression may already be beneficial. Given that UNC13A haploinsufficiency appears to be tolerated in humans (refs. 22,23 and this study) and in mice19, restoration of UNC13A to levels approaching 50% appears to be sufficient as an upper target. Antisense oligonucleotide therapies71, for example, are approved for clinical use and could be well-used in ALS/FTD and in patients with type B and C neurodevelopmental disorders. Our study underscores the importance of an experimentally validated mechanistic understanding of pathogenicity for diverse variants in the same gene, as different therapeutic strategies should be used to combat them. In addition, for each patient, ethical approvals and informed consent forms from parents or guardians were obtained by the respective research teams and institutions for data use and publication, including photographs or videos where applicable. The use of Unc13a/b knockout mice was approved by the responsible local government organization (Niedersächsisches Landesamt für Verbraucherschutz und Lebensmittelsicherheit; 33.19-42502-04-15/1817 and 33.19-42502-04-20/3589, and Landesamt für Gesundheit und Soziales; G106/20). Mice (Mus musculus, backcrossed to a C57BL/6N background) of embryonic day 18 or postnatal day 0–1 and of both sexes were used in electrophysiological experiments. Each culture was obtained from one or more mice, and two to four independent cultures were used per condition. Adult mice were kept under IVC/SPF conditions, on a 12 h light, 12 h dark cycle, at room temperature (22 ± 2 °C) and humidity levels of 55 ± 10%. Individuals included in this study underwent exome or genome sequencing on diverse sequencing platforms on a clinical or research basis in academic institutes or diagnostic labs worldwide. Using GeneMatcher73 and personal communication with colleagues enabled us to assemble clinical and genetic details on a total of 48 index patients with de novo or inherited heterozygous or biallelic variants in UNC13A (NM_001080421.3), including two previously published cases harboring c.154 G > A, p.(E52K) (case 74 described in ref. 74) and c.4379 C > T, p.(A1460V) (case UPN-0740 described in ref. Standard deviation of growth parameters was calculated as previously described76. De-identified clinical data from collaborating institutions were obtained with local institutional review board approval and informed consent from parents or guardians of affected individuals, including consent for publication of facial photographs for individuals in Figs. For splicing assays, the pSplicePOLR2G3 vector was digested and purified as described previously78. Inserts of UNC13A exons and their flanking intronic regions were amplified using the Q5 HotStar High-Fidelity Taq Polymerase kit (New England Biolabs), and cloning reactions were performed with the In-Fusion HD Cloning kit (Clontech) following the manufacturer's recommendations. Reverse transcription was performed with the ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs), and fluorescent PCR was performed as described previously79. PCR products were size-separated on a 3130xl Genetic Analyzer (Applied Biosystems). Output data were analyzed with Peak Scanner software (Applied Biosystems). Unless otherwise indicated, cDNA encoding rat UNC13A tagged with EGFP at the carboxy terminus80 was used. The FUGW vector81, in which the ubiquitin promoter was exchanged with the human synapsin 1 promoter, was used according to a previously published81, modified protocol82, using helper plasmids (Addgene, 8454 and 8455; ref. In a subset of experiments, constructs encoding NLS-GFP-P2A-UNC13A-Flag under the synapsin promoter were used82. Viral transfection of neurons was carried out on DIV 2–3. Plasmid and primer sequence information are summarized in Supplementary Table 1. Neurons were prepared from the brains of E18 Unc13a/b DKO mice and plated on WT astrocyte microisland cultures according to published protocols83. Whole-cell voltage-clamp data were acquired using an Axon Multiclamp 700B amplifier, Digidata 1440A data acquisition system and pCLAMP 10 software (Molecular Devices). Basal eEPSCs were recorded at a frequency of 0.1 Hz. Rapid external solution exchange around individual neurons was achieved by using a custom-made fast-flow system controlled by a stepper motor. Recordings were made at room temperature (~22 °C). The size of the RRP was estimated from the integrated charge of EPSCs evoked by application of 500 mM sucrose (7 s or 5 s in Fig. 3k–l) after subtraction of the steady-state current to correct for ongoing SV pool replenishment. mEPSCs were recorded for 100 s in the presence of 300 nM tetrodotoxin. 3g,h, additional recordings in the presence of NBQX (3 μM, Tocris Bioscience) were used to subtract false-positive events from traces recorded in extracellular solution. mEPSC traces were filtered at 1 kHz, and miniature events were identified using a sliding template function in Axograph or in Igor Pro (5–200 pA; rise time, 0.15–1.5 ms; half-width, 0.5–5 ms). PDBu (1 μM) was applied for 1 min. Analyses were performed using Axograph (v.1.4.3) (Molecular Devices) or Igor Pro (Wavemetrics). Data were obtained from at least two cultures, and for the majority of data, from four cultures. Quantifications of synapse number, signal intensity and colocalization were made by immunostaining of microisland hippocampal cultures. After three PBS washes, the remaining PFA was quenched with 50 mM glycine for 10 min, and samples were washed again with PBS. Before permeabilization, samples were treated with four drops of Image-iT FX signal enhancer in 1.5–2 ml PBS per well (in a six-well plate) and maintained shaking for 20 min. Samples were permeabilized for 30 min with 0.1% Triton X-100 and 2.5% normal goat serum (NGS) diluted in PBS while gently shaking. Blocking was performed with 2.5% NGS in PBS, twice for a total of 15 min. The primary antibodies were diluted in blocking solution, and coverslips were mounted upside down on 150 μl antibody solution placed on parafilm and kept for 2.5 h at room temperature or overnight at 4 °C, both in a wet chamber. Coverslips were washed three times with 0.1% Triton X-100 and 2.5% NGS in PBS and subsequently mounted upside down in 150 μl secondary antibody solution in a dark, wet chamber for 45 min at room temperature. Finally, the coverslips were washed five times with 0.1% Triton X-100 and 2.5% NGS in PBS for 2–5 min. The coverslips were briefly washed in double-distilled water, excess water was removed and the coverslips were mounted on a glass slide using Aqua-Poly/Mount. Microislands containing one hippocampal neuron were efficiently localized by first acquiring a low-magnification overview image of the entire coverslip using a Zeiss LSM 880 confocal laser scanning microscope. Specifically, upon 405 nm excitation, a ×10 air objective (0.45 NA) was used to image Alexa Fluor 405-stained MAP2 (410–470 nm). The overview image served as a map based on which the motorized stage of the Zeiss LSM 880 was moved to selected autaptic neurons, from which high-resolution (x, y, z: 208, 208, 309 nm) z-stacks of VGLUT1 (637–758 nm), Shank2 (570–624 nm), UNC13A–eGFP (490–552 nm) and MAP2 (410–470 nm) were acquired with a ×40 oil immersion objective (1.40 NA). Overlapping fields of view for subsequent image stitching were acquired with the above settings, in case the autaptic neuron did not fit in one field of view. All image processing was conducted with the same automated Imaris (v.9.10.0) batch process. Overlapping Shank2 and VGLUT1 surfaces were used to estimate the total number of synapses contacting individual autaptic neurons. Subsequently, the synapses overlapping with the UNC13A–eGFP signal were identified, and in a reciprocal manner, UNC13A–eGFP surfaces that overlap with synapses were identified. For illustration, representative images were processed with the FIJI software package84 (v.2.14.0/1.54f) and assembled in Adobe Illustrator. HEK293T cells were seeded on poly-L-lysine-coated glass coverslips and cultured in DMEM supplemented with 1% MEM, 1% GlutaMAX, 1% penicillin–streptomycin and 10% FBS at 37 °C under 5% CO2. Cells were transfected with UNC13A–GFP cDNA in pEGFP–N1 vector80 carrying the respective disease-related variations under the CMV promoter using Lipofectamine 2000 and 1 μg DNA per well in a six-well plate, according to the manufacturer's recommendations. After 3.5 h of incubation, the medium was replaced with culture medium, and the cells were incubated overnight. Translocation was induced by applying 1 µM PDBu for 1 h. Cells were washed with PBS, fixed with 4% PFA in PBS at room temperature for 10 min, washed with PBS and treated with 50 mM glycine in PBS for 10 min to quench remaining PFA. Cells were washed in PBS and incubated with 1 µg ml−1 DAPI in PBS at room temperature for 15 min. Cells were then washed again with PBS and mounted using Aqua-Poly/Mount. Fluorescence imaging was performed on a confocal laser scanning microscope (Zeiss LSM710-CONFOCOR3). Strains were maintained and genetically manipulated as previously described85. Animals were raised at 20 °C on nematode growth media seeded with OP50. unc-13(nu641) harbors a C-terminal mScarlet in the unc-13 locus as described previously86. UNC-13 residue numbering is based on the UNC-13L isoform accession number NP_001021874. CRISPR–Cas9-mediated genome editing was performed to generate the desired genetic modifications in C. elegans. The CRISPR–Cas9 system was implemented using a co-injection strategy with Cas9 protein, CRISPR RNA (crRNAs) and repair templates. For crRNA design, crRNAs targeting the desired genomic loci were initially designed using the Benchling online tool and custom crRNA design software from Integrated DNA Technologies (IDT). crRNAs were selected based on proximity of the cut site to the desired mutation, high on-target efficiency and minimal predicted off-target mutations. crRNAs were ordered from IDT, resuspended in 4 μg μl−1 in nuclease-free water and diluted to a working concentration of 0.4 μg μl−1 before adding to the injection mix. For Cas9 preparation, recombinant Cas9 protein purchased from IDT (Alt-R S.p. Trans-activating CRISPR RNA (tracrRNA) was purchased from IDT, diluted to 4 μg μl−1 in IDT nuclease-free duplex buffer and aliquots were stored at −20 °C. For precise edits, single-stranded oligodeoxynucleotides were used as repair templates. These templates included the desired modification flanked by 20–50 bp homology arms and an engineered restriction site. Single-stranded oligodeoxynucleotides were ordered from IDT (Ultramer DNA Oligonucleotides). The mixture was incubated at 37 °C for 15 min to allow ribonucleoprotein complex formation. Following ribonucleoprotein formation, 2 µl of mutant repair oligonucleotide (100 µM stock) and 2 µl of co-CRISPR repair oligonucleotide were added. Young adult worms (either N2 or unc-13(nu641)) were immobilized on 2% agarose pads in halocarbon oil. The CRISPR–Cas9 injection mix was manually injected into the gonads of P0 worms using a Picospritzer III microinjector (Parker). The worm gonad was visualized using an Olympus IX51 inverted microscope with a ×40 air objective. Injected worms were recovered on seeded nematode growth medium plates at 20 °C for 3–4 days. F1 progeny expressing the co-injection marker (unc-58) were selected for screening and grown to the young adult stage. Genomic DNA was extracted from worm lysates, and the targeted locus was amplified by PCR. PCR products were digested with the engineered restriction site to identify potential edits. Worms containing the restriction site and lacking the unc-58 gain-of-function phenotype were singled out, and homozygous progeny were isolated. Genomic DNA was then re-extracted and sequenced using nanopore sequencing (Plasmidsaurus) to confirm the presence of the desired edits. Animals were then outcrossed at least four times with the parent strain. Sequence information for C. elegans strains can be found in Supplementary Note 1. To measure aldicarb sensitivity in C. elegans, 20–25 animals were placed on OP50-seeded agar plates containing 1 mM aldicarb (CarboSynth) as described previously87,88. During an assay, the experimenter was blind to all genotypes, and each genotype was assayed at least ten times. Paralysis curves for each genotype were generated by averaging individual time courses. A model of UNC13A consisting of the C1, C2B and MUN domains (residues 505–985) was predicted with AlphaFold (v.2.0; 01-JUL-21)89 using the UniProt sequence Q9UPW8 as input. The resulting model showed, as expected, a high similarity (root mean squared deviation, 1.08 Å) to the known 3D structure of rat MUNC-13 (PDB 5UE8)90. All other amino acids were protonated at pH 7.0. Subsequently, WT UNC13A and all single mutants were solvated in cubic boxes of SPC/E water91 with the dimension of approximately 12.8 nm3. The net charges of the models were neutralized with K+Cl−, and an ionic strength of 150 mM K+Cl− was added. Additionally, five Ca2+ ions were inserted, replacing water molecules. To prevent ion aggregation, a previously published parameter set92 was applied. After energy minimization and thermal equilibration with gradually decreasing position restraints, all periodic models were allowed to move freely during the following 1 µs-long production runs. To ensure reasonable statistics, all simulations were repeated five times. Constraining all bonds containing hydrogen atoms with the Lincs algorithm96 allowed a time step of 2 fs. Short-range electrostatics and van der Waals interactions were truncated at 1.2 nm, while long-range electrostatics were calculated with the Particle Ewald summation97. Images of 3D structures were generated with VMD98, graphs were drawn with the Matplotlib library included in Python99, and dipole moments were calculated with the MDAnalysis tool100. As a negative control, non-transfected cells were used. Cells were transfected with Lipofectamine 2000 (according to the manufacturer's manual) and 1 µg of the plasmid. Then, 24 h after transfection, cells were lysed in RIPA buffer (150 mM NaCl, 1% IGEPAL CA 630, 1% sodium-deoxycholate, 0.1% SDS and 30 mM Tris-Cl, pH 7.4, supplemented with aprotinin, leupeptin and phenylmethylsulfonyl fluoride). Supernatants were transferred into a fresh tube, and protein concentration was determined. Samples were diluted in Laemmli buffer and incubated for 5 min at 95 °C. Next, 10 μg of the samples were separated on 8% Bis–Tris gel in cold MOPS buffer at 150 V, blotted on a nitrocellulose membrane (Protran 0.2 NC, Amersham) and stained with the Pierce reversible protein stain and destain kit (according to the manufacturer's manual; Thermo Fisher Scientific, 46430) for a sample loading control. The membranes were cut and incubated with an anti-Flag antibody, anti-GFP antibody (see Supplementary Table 2) and corresponding secondary antibodies, and the chemiluminescence signal was detected on an ECL ChemoStar imager (Intas). Signal intensity quantification was done in Fiji101. Protein levels were first normalized to the total protein staining of the corresponding lane, and ratios between the GFP and Flag signals were calculated. Electrophysiological data are presented as means ± s.e.m., and statistical significance was determined using the non-parametric two-tailed Mann–Whitney test. For the C. elegans paralysis assay, data were analyzed using a non-parametric Kruskal–Wallis test followed by Dunn's test for all pairwise comparisons. Western blots and imaging data are presented as means ± s.e.m., and statistical significance was evaluated using a non-parametric Kruskal–Wallis test. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Molecular dynamics simulation data are available at https://doi.org/10.5281/zenodo.14554776 (ref. All other primary data will be made available on request from the corresponding authors. Source data are provided with this paper. No code was developed for this study. Lai, Y. et al. Molecular mechanisms of synaptic vesicle priming by Munc13 and Munc18. & Zhou, Q. Molecular mechanisms of fast neurotransmitter release. The morphological and molecular nature of synaptic vesicle priming at presynaptic active zones. Siksou, L. et al. A common molecular basis for membrane docking and functional priming of synaptic vesicles. Sakamoto, H. et al. Synaptic weight set by Munc13-1 supramolecular assemblies. Rapid active zone remodeling consolidates presynaptic potentiation. Increased vesicle fusion competence underlies long-term potentiation at hippocampal mossy fiber synapses. Junge, H. J. et al. Calmodulin and Munc13 form a Ca2+ sensor/effector complex that controls short-term synaptic plasticity. Shin, O. H. et al. Munc13 C2B domain is an activity-dependent Ca2+ regulator of synaptic exocytosis. Lipstein, N. et al. Munc13-1 is a Ca2+-phospholipid-dependent vesicle priming hub that shapes synaptic short-term plasticity and enables sustained neurotransmission. Dynamic control of synaptic vesicle replenishment and short-term plasticity by Ca2+-calmodulin-Munc13-1 signaling. Betz, A. et al. Munc13-1 is a presynaptic phorbol ester receptor that enhances neurotransmitter release. Beta phorbol ester- and diacylglycerol-induced augmentation of transmitter release is mediated by Munc13s and not by PKCs. Differential control of vesicle priming and short-term plasticity by Munc13 isoforms. Augustin, I., Betz, A., Herrmann, C., Jo, T. & Brose, N. Differential expression of two novel Munc13 proteins in rat brain. Augustin, I., Rosenmund, C., Sudhof, T. C. & Brose, N. Munc13-1 is essential for fusion competence of glutamatergic synaptic vesicles. Varoqueaux, F. et al. Total arrest of spontaneous and evoked synaptic transmission but normal synaptogenesis in the absence of Munc13-mediated vesicle priming. Mullins, J. R., McFadden, K., Snow, N. & Oviedo, A. Homozygous UNC13A variant in an infant with congenital encephalopathy and severe neuromuscular phenotype: a case report with detailed central nervous system neuropathologic findings. Engel, A. G., Selcen, D., Shen, X. M., Milone, M. & Harper, C. M. Loss of MUNC13-1 function causes microcephaly, cortical hyperexcitability, and fatal myasthenia. Lipstein, N. et al. Synaptic UNC13A protein variant causes increased neurotransmission and dyskinetic movement disorder. Diekstra, F. P. et al. C9orf72 and UNC13A are shared risk loci for amyotrophic lateral sclerosis and frontotemporal dementia: a genome-wide meta-analysis. Diekstra, F. P. et al. UNC13A is a modifier of survival in amyotrophic lateral sclerosis. van Es, M. A. et al. Genome-wide association study identifies 19p13.3 (UNC13A) and 9p21.2 as susceptibility loci for sporadic amyotrophic lateral sclerosis. Analysis of the UNC13A gene as a risk factor for sporadic amyotrophic lateral sclerosis. Chio, A. et al. UNC13A influences survival in Italian amyotrophic lateral sclerosis patients: a population-based study. Vidal-Taboada, J. M. et al. UNC13A confers risk for sporadic ALS and influences survival in a Spanish cohort. Placek, K. et al. UNC13A polymorphism contributes to frontotemporal disease in sporadic amyotrophic lateral sclerosis. Willemse, S. W. et al. UNC13A in amyotrophic lateral sclerosis: from genetic association to therapeutic target. Ma, X. R. et al. TDP-43 represses cryptic exon inclusion in the FTD-ALS gene UNC13A. Akiyama, T., Koike, Y., Petrucelli, L. & Gitler, A. D. Cracking the cryptic code in amyotrophic lateral sclerosis and frontotemporal dementia: towards therapeutic targets and biomarkers. Wiel, L. et al. MetaDome: pathogenicity analysis of genetic variants through aggregation of homologous human protein domains. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Betz, A., Okamoto, M., Benseler, F. & Brose, N. Direct interaction of the rat unc-13 homologue Munc13-1 with the N terminus of syntaxin. Basu, J. et al. A minimal domain responsible for Munc13 activity. Marin, O. Developmental timing and critical windows for the treatment of psychiatric disorders. Bekkers, J. M. & Stevens, C. F. Excitatory and inhibitory autaptic currents in isolated hippocampal neurons maintained in cell culture. Furshpan, E. J., MacLeish, P. R., O'Lague, P. H. & Potter, D. D. Chemical transmission between rat sympathetic neurons and cardiac myocytes developing in microcultures: evidence for cholinergic, adrenergic, and dual-function neurons. Stewart, S. A. et al. Lentivirus-delivered stable gene silencing by RNAi in primary cells. Dulubova, I. et al. A Munc13/RIM/Rab3 tripartite complex: From priming to plasticity? Camacho, M. et al. Heterodimerization of Munc13 C2A domain with RIM regulates synaptic vesicle docking and priming. Deng, L., Kaeser, P. S., Xu, W. & Sudhof, T. C. RIM proteins activate vesicle priming by reversing autoinhibitory homodimerization of Munc13. Betz, A. et al. Functional interaction of the active zone proteins Munc13-1 and RIM1 in synaptic vesicle priming. Basu, J., Betz, A., Brose, N. & Rosenmund, C. Munc13-1 C1 domain activation lowers the energy barrier for synaptic vesicle fusion. Lin, K. H., Taschenberger, H. & Neher, E. A sequential two-step priming scheme reproduces diversity in synaptic strength and short-term plasticity. Richmond, J. E., Davis, W. S. & Jorgensen, E. M. UNC-13 is required for synaptic vesicle fusion in C. elegans. Richmond, J. E., Weimer, R. M. & Jorgensen, E. M. An open form of syntaxin bypasses the requirement for UNC-13 in vesicle priming. & Dittman, J. S. A C1–C2 module in Munc13 inhibits calcium-dependent neurotransmitter release. Liu, H. et al. Heterodimerization of UNC-13/RIM regulates synaptic vesicle release probability but not priming in C. elegans. Quest, A. F., Bardes, E. S. & Bell, R. M. A phorbol ester binding domain of protein kinase C gamma. High affinity binding to a glutathione-S-transferase/Cys2 fusion protein. Malenka, R. C., Ayoub, G. S. & Nicoll, R. A. Phorbol esters enhance transmitter release in rat hippocampal slices. Wesseling, J. F. & Lo, D. C. Limit on the role of activity in controlling the release-ready supply of synaptic vesicles. Xian, J. et al. Assessing the landscape of STXBP1-related disorders in 534 individuals. A. et al. A catalogue of new incidence estimates of monogenic neurodevelopmental disorders caused by de novo variants. He, E. et al. Munc13-1 and Munc18-1 together prevent NSF-dependent de-priming of synaptic vesicles. Meijer, M. et al. Munc18-1 mutations that strongly impair SNARE-complex binding support normal synaptic transmission. Li, L. et al. A hyperactive form of unc-13 enhances Ca2+ sensitivity and synaptic vesicle release probability in C. elegans. Lipstein, N. et al. Nonconserved Ca2+/calmodulin binding sites in Munc13s differentially control synaptic short-term plasticity. Kawabe, H. et al. ELKS1 localizes the synaptic vesicle priming protein bMunc13-2 to a specific subset of active zones. Active zone scaffolds differentially accumulate Unc13 isoforms to tune Ca2+ channel-vesicle coupling. Ling, J. P., Pletnikova, O., Troncoso, J. C. & Wong, P. C. TDP-43 repression of nonconserved cryptic exons is compromised in ALS-FTD. Neumann, M. et al. Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Zarebidaki, F. et al. Disentangling the roles of RIM and Munc13 in synaptic vesicle localization and neurotransmission. Bennett, C. F., Krainer, A. R. & Cleveland, D. W. Antisense oligonucleotide therapies for neurodegenerative diseases. Structural basis for a Munc13-1 homodimer to Munc13-1/RIM heterodimer switch. Sobreira, N., Schiettecatte, F., Valle, D. & Hamosh, A. GeneMatcher: a matching tool for connecting investigators with an interest in the same gene. Improved diagnostic yield compared with targeted gene sequencing panels suggests a role for whole-genome sequencing as a first-tier genetic test. Lessons learned from large-scale, first-tier clinical exome sequencing in a highly consanguineous population. Boonsawat, P. et al. Elucidation of the phenotypic spectrum and genetic landscape in primary and secondary microcephaly. Freeman, P. J. et al. Standardizing variant naming in literature with VariantValidator to increase diagnostic rates. Callebaut, I. et al. Comprehensive functional annotation of 18 missense mutations found in suspected hemochromatosis type 4 patients. Raud, L. et al. Functional analysis of novel RHD variants: splicing disruption is likely to be a common mechanism of variant D phenotype. & Baltimore, D. Germline transmission and tissue-specific expression of transgenes delivered by lentiviral vectors. Camacho, M. et al. Control of neurotransmitter release by two distinct membrane-binding faces of the Munc13-1 C1C2B region. Analysis of neurotransmitter release mechanisms by photolysis of caged Ca2+ in an autaptic neuron culture system. Rueden, C. T. et al. ImageJ2: ImageJ for the next generation of scientific image data. Brenner, S. The genetics of Caenorhabditis elegans. Padmanarayana, M. et al. A unique C2 domain at the C terminus of Munc13 promotes synaptic vesicle priming. Dittman, J. S. & Kaplan, J. M. Behavioral impact of neurotransmitter-activated G-protein-coupled receptors: muscarinic and GABAB receptors regulate Caenorhabditis elegans locomotion. Nurrish, S., Segalat, L. & Kaplan, J. M. Serotonin inhibition of synaptic transmission: Gαo decreases the abundance of UNC-13 at release sites. Highly accurate protein structure prediction with AlphaFold. Xu, J. et al. Mechanistic insights into neurotransmitter release and presynaptic plasticity from the crystal structure of Munc13-1 C1C2BMUN. Berendsen, H. J. C., Grigera, J. R. & Straatsma, T. P. The missing term in effective pair potentials. Determination of alkali and halide monovalent ion parameters for use in explicitly solvated biomolecular simulations. Improved side-chain torsion potentials for the Amber ff99SB protein force field. Bussi, G., Donadio, D. & Parrinello, M. Canonical sampling through velocity rescaling. Parrinello, M. & Rahman, A. Polymorphic transitions in single crystals: a new molecular dynamics method. Hess, B., Bekker, H., Berendsen, H. J. C. & Fraaije, J. G. E. M. LINCS: a linear constraint solver for molecular simulations. Darden, T., York, D. & Pedersen, L. Particle mesh Ewald: an N⋅log(N) method for Ewald sums in large systems. & Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Lipstein, N., Sun, H. & Utesch, T. Pathogenic UNC13A variants cause a neurodevelopmental syndrome by impairing synaptic function. Quest, A. F., Bardes, E. S. & Bell, R. M. A phorbol ester binding domain of protein kinase C gamma. Deletion analysis of the Cys2 domain defines a minimal 43-amino acid peptide. We thank all involved families for their participation and permission to publish this study. We thank A. Günther, K. Steinhagen, the AGCT lab and animal facility staff at the Max Planck Institute of Multidisciplinary Sciences for technical help, and B. van Rossum for illustrations. We thank the Viral Core Facility, Charité–Universitätsmedizin Berlin, for virus production, and B. Weinberg for the contribution of lentiviral vectors (Addgene, 8454 and 8455). This work was supported by the German Research Foundation Excellence Strategy EXC-2049-390688087 (N.L., C.R., A.A.K. ), the North-German Supercomputing Alliance HLRN (H. Sun and T.U. ), the University of Zurich Clinical Research Priority Program Praeclare (A. Rauch), the University of Greenwich QR Funding (R.A.), NIH NS 116747 (to J.S.D.) Additional acknowledgments are listed in Supplementary Note 2. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Open access funding provided by Leibniz-Forschungsinstitut für Molekulare Pharmakologie im Forschungsverbund Berlin e.V. Reza Asadollahi, Paranchai Boonsawat, Dennis Kraemer & Anita Rauch Faculty of Engineering and Science, University of Greenwich London, Medway Campus, Chatham Maritime, London, UK Department of Molecular Neurobiology, Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany Aisha Ahmad, Mareike Lohse, Mrinalini Ranjan, Jeong Seop Rhee, Holger Taschenberger, Nils Brose & Noa Lipstein Leibniz-Forschungsinstitut für Molekulare Pharmakologie (FMP), Berlin, Germany Aisha Ahmad, Mareike Lohse, Siqi Sun, Tillmann Utesch, Jonas D. Sommer, Dragana Ilic, Kati Fischermanns, Moritz Boll, Han Sun & Noa Lipstein Department of Biochemistry, Weill Cornell Medicine, New York, NY, USA Jasmine Shahanoor Hinzen, Murugesh Padmanarayana & Jeremy S. Dittman Chandran Ka, Sylvia Redon, Kevin Uguen & Gerald Le Gac Chandran Ka, Sylvia Redon, Kevin Uguen & Gerald Le Gac Divisions of Newborn Medicine and Genetics and Genomics, Department of Pediatrics, Boston Children's Hospital, Harvard Medical School, Boston, MA, USA Manton Center for Orphan Disease Research, Division of Genetics and Genomics, Department of Pediatrics, Boston Children's Hospital, Harvard Medical School, Boston, MA, USA Genome Diagnostics, Department of Paediatric Laboratory Medicine, The Hospital for Sick Children and University of Toronto, Toronto, Ontario, Canada Department of Human Genetics, Research Institute, National Center for Global Health and Medicine, Tokyo, Japan Clinical Genetics Unit, Instituto da Crianca, Hospital das Clinicas HCFMUSP, Faculdade de Medicina, Universidade de São Paulo, São Paulo, Brazil Bruno de Oliveira Stephan, Rachel Sayuri Honjo, Debora R. Bertola & Chong Ae Kim Division of Clinical Genetics, Joe DiMaggio Children's Hospital, Hollywood, FL, USA Center for Pediatric Neurological Disease Research, St. Jude Children's Hospital, Memphis, TN, USA Department of Metabolic Medicine, The Royal Children's Hospital, Melbourne, Victoria, Australia Victorian Clinical Genetics Services, Murdoch Children's Research Institute, Melbourne, Victoria, Australia Joy Lee, Oliver Heath, Natasha J. Brown, Naomi Baker, Zornitza Stark & Martin Delatycki Department of Clinical Genetics, Erasmus MC, Rotterdam, The Netherlands Farrah Rajabi, Lance H. Rodan & Stephanie A. Coury Division of Genetics, Brigham and Women's Hospital, Boston, MA, USA Konrad Platzer, Henry Oppermann & Rami Abou Jamra Division of Genetics and Rare Diseases, Ann and Robert H. Lurie Children's Hospital of Chicago, Chicago, IL, USA Division of Genetics and Metabolism, Nicklaus Children's Hospital, Miami, FL, USA Division of Biochemical Genetics, BC Children's Hospital, Vancouver, British Columbia, Canada Stephanie Efthymiou, Reza Maroofian & Henry Houlden Centre de Référence Neuromusculaire and Paediatric Neurology Department, Hôpital Universitaire des Enfants Reine Fabiola, Hôpital Universitaire de Bruxelles (HUB), Université Libre de Bruxelles, Brussels, Belgium Indira Gandhi Institute of Child Health, Bangalore, India Department of Medical Genetics, Next Generation Genetic Polyclinic, Mashhad, Iran Human Genetics, Medical Faculty, School of Medicine and Health Sciences, Carl von Ossietzky Universität Oldenburg, Oldenburg, Germany Ingrid M. Wentzensen, Kristin G. Monaghan & Yanmin Chen The Hospital for Sick Children, Toronto, Ontario, Canada Department of Neuropediatrics, Faculty of Medicine and University Hospital Carl Gustav Carus, Technische Universität Dresden, Dresden, Germany Division of Genetic and Genomic Medicine, Nationwide Children's Hospital, Columbus, OH, USA The Steve and Cindy Rasmussen Institute for Genomic Medicine, Nationwide Children's Hospital, Columbus, OH, USA Department of Pediatrics, Women's and Children's Institute, Geisinger Medical Center, Danville, PA, USA Laboratory of Medical Genetics, Translational Cytogenomics Research Unit, Bambino Gesù Children's Hospital, IRCCS, Rome, Italy Corewell Health West Helen DeVos Children's Hospital, Grand Rapids, MI, USA The Institute for Rare Diseases, Edmond and Lily Safra Children's Hospital, Sheba Medical Center, Ramat Gan, Israel School of Medicine, Faculty of Medical and Health Sciences, Tel Aviv University, Tel Aviv, Israel Pediatric Neurology Unit, The Edmond and Lily Safra Children's Hospital, Sheba Medical Center, Ramat Gan, Israel Amy Margolin, Jennifer Morrison & Aditi Dagli Elliot Stolerman, Raymond J. Louie & Camerun Washington Department of Translational Genomics, Center for Genomic Medicine, King Faisal Specialist Hospital and Research Center, Riyadh, Saudi Arabia College of Medicine, Alfaisal University, Riyadh, Saudi Arabia Institute of Human Genetics, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Department of Pediatrics, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Medical Department, Mendelics Genomic Analysis, São Paulo, Brazil Federal University of Ceará - UFC and Hospital Infantil Albert Sabin, Fortaleza, Brazil André Luiz Santos Pessoa & Antonio Edvan Camelo-Filho Département de Génétique, Centre de référence des malformations et maladies congénitales du cervelet, Hôpital Trousseau, AP-HP.Sorbonne Université, Paris, France Service de Neuropédiatrie-Pathologie du Développement, Centre de Référence Neurogénétique, Hôpital Trousseau, FHU I2-D2, AP-HP.Sorbonne Université, Paris, France Cyril Mignot, Boris Keren, Perrine Charles & Caroline Nava Trace Neuroscience, South San Francisco, CA, USA Ryan D. Morrie, Shila Mekhoubad & Eric M. Green Neuroproteomics Group, Department of Molecular Neurobiology, Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany City Campus Light Microscopy Facility, Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany Institute of Biochemistry, Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU), Erlangen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar A.A.K., A. Rad, A.D., A.D.G., A.E.C.-F., A.H.S., A.L., A. Rosen, A.L.S.P., A.M., A.N., A.P., A.R.-R., A.S., A.T., B.A., B.B.Z., B.d.O.S., B.G., B.H., B.I., B.K., C.A.K., C. Maxton, C.B., C.C.d.K., C.E.P., C.H., C.N., C.R.M., C.W., C. Mignot, D.C.K., D.D., D.R.B., D. Koeberl, E.A., E.G.K., E.M.G., E.S., F.B., F.E., F.K., F.M., F.P.B., F.P.M., F.R., F.S.A., G.C.K., G.O., H.C.M., H.D., H.H., H.O., H.U., I.H., I.M.W., J. Lisfeld, J.C., J. Lefranc, J.M., J.N., J. Lee, J.T., K. Riley, K. Reinson, K.G.M., K.L.H., K.K.B., K.M., K.Õ., K.P., K.U., L.B., L.G.H., L.H.R., L.Z., M. Heijligers, M.D., M.H.W., M.J.B., M. Heidari, M.O.-L., M.P.F., N. Baker, N. Matsumoto, N.D., N.J.B., N.J.L., N. Miyake, O.H., P. Charles, P. Carmelo, P.J., P.M.C., P.V., P.Y.B.A., R.A.J., R.C., R.M., R.S.H., R.J.L., R.Ś., R.Y., R.D.M., S.A.C., S.C., S.B., S.E., S. Race, S.J.B., S.J.C.S., S.M., S.M.-A., S.M.M., S.M.R., S. Redon, S.Z., V.R.G.K., Y.C. acquired resources (genetic, diagnostic and clinical information). All authors read, reviewed, edited and approved the paper. Correspondence to Reza Asadollahi or Noa Lipstein. is a scientific founder of Trace Neuroscience. All the above may own stock in Trace Neuroscience. and A. Rad are employees of Arcensus GmbH. are employees of and may own stock in GeneDx. D. Koeberl has received grant support from Viking Therapeutics, Genzyme Sanofi, Roivant Rare Diseases and Amicus and has held equity in Asklepios Biopharmaceutical (AskBio). The other authors declare no competing interests. Nature Genetics thanks Stephan Sigrist, Jan Veldink and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. b, UNC13A c.523+6 T > C variant exhibits a strong, deleterious splicing impact leading to exon 7 skipping (a.u., arbitrary units). c, d, Neurophysiologic findings at Nasalis muscle in repetitive nerve stimulation (RNS) recordings. c, Decrementing response (19.7%) induced by low-frequency (3 Hz) repetitive nerve stimulation. d, Incrementing response (17.8%) induced by high-frequency (20 Hz) repetitive nerve stimulation. The findings are compatible with a compromised presynaptic neuromuscular junction function. e, A three-generation pedigree illustrating two affected cousins, patients 4a and 4b, harboring identical pathogenic homozygous variants p.(R202H) in UNC13A. a, b, UNC13A c.767+1 G > T (intron 9) and c.4073+1 G > A (intron 34) variants inherited in compound heterozygous state exert strong, deleterious impact on UNC13A exon 9 (a) and exon 34 splicing (b), respectively. c, an UNC13A c.2186+5 G > A variant (intron 18) inherited in a homozygous state revealed strong, deleterious impact on UNC13A exon 18 splicing. All events are predicted to cause mis-splicing and downstream nonsense-mediated mRNA decay. a-l, Data obtained from neurons expressing UNC13AWT (gray) or UNC13AR799Q (brown). a, Example traces of spontaneous miniature excitatory postsynaptic currents (mEPSCs), and b, quantification of mEPSC frequency and c, mEPSC amplitude. d, Example traces and e, quantification of the charge transfer during a single action-potential evoked excitatory postsynaptic currents (eEPSCs). h, Average vesicular release probability \({\bar{p}}_{{vr}}\). j,k, Averaged time courses of normalized eEPSC amplitudes, before, during and after a high-frequency action potential train at 10 Hz (j) or 40 Hz (k), for the indicated genotypes. l, Replenishment rate constant calculated based on a single-pool model58 for neurons of the indicated genotypes. m-x, similar data for neurons expressing UNC13AWT (gray) or UNC13AN1013S (purple). See Supplemental table 2 and 3 for further details. b, Spectrum of pathogenic variants, patterns of inheritance and associated phenotypes reported for genes encoding neuronal SNAREs and associated AZ proteins. Frequency of diverse disease-causing variants and their patterns of inheritance are shown based on the data in the Human Gene Mutation Database (HGMDv.4.1). Size of circles corresponds to the frequencies of reports in the literature with respective numbers labelled. Spectrum of associated phenotypes derived from clinical synopses in OMIM, GeneReviews and HGMD is shown with light blue boxes for each gene. AD, autosomal dominant; AR, autosomal recessive; ASD, autism spectrum disorder; CNV, copy number variant; DD, developmental delay; ID, intellectual disability; LGD, likely gene disrupting. The data is normalized to the respective control (dotted vertical lines) to allow a comparison of magnitudes and directions of variant-associated changes. SEMs were calculated by Gaussian error propagation. Data for the P814L variant are obtained from24. b, An example blot including total protein stain (up), an anti-FLAG immunoblot (IB; middle), and an anti-GFP immunoblot (down) from one of four experiments used to determine the expression level of UNC13A-FLAG with or without disease variants in comparison to that of GFP. c, Changes in the ratio of UNC13A-FLAG to GFP intensity were statistically not significant, which agrees with the notion that disease variants do not interfere with UNC13A stability. Circles represent four independent biological replications. The C1 domain structure is maintained by a zinc-cluster104. Two Zn2+ ions are coordinated each by one histidine and three cysteine residues105. The finding that the C587F variant has a minor effect on basal synaptic transmission properties is surprising, because knock-in mouse neurons expressing a H567K mutation that abolishes PDBu/DAG binding14,15,49 have impaired synaptic transmission properties. We ran five independent 1 µs-long MD simulations for the WT protein, H554K, C587F, and for other disease variants as controls (see materials and methods). a, Averaged dipole moments of the C1 domain during the last 250 ns of five simulations (black dots). The Zn2+ ion is represented by as a blue sphere. c, The averaged minimal distances between Zn2+ and E586 of the last 250 ns of five independent MD simulations (black dots). The structural and electrostatic changes observed in MD simulations support the notion that C587F exerts a different effect on the C1 domain structure than H554K, although both abolish PDBu sensitivity. In a and c the standard deviation is indicated by black bars. Averages are highlighted as gray dots. a, EPSC charge, b, RRP size and c, the average vesicular release probability \({\bar{p}}_{{vr}}\) are reduced in UNC13AR202H-expressing neurons. Note that \({\bar{p}}_{{vr}}\) was unchanged under the experimental conditions in Fig. 3. d, Potentiation of the EPSC amplitude by the DAG analog PDBu is stronger in UNC13AR202H-expressing neurons. e,f, Stronger facilitation during and larger augmentation after AP trains at 1 Hz or 10 Hz are observed in neurons expressing UNC13AR202H. Note that before and after the high frequency trains, EPSCs were probed at frequency of 0.2 Hz, and not at 0.1 Hz as in the rest of the manuscript. Supplementary Table 1: plasmid and primer sequence information. Supplementary Note 1: sequence information for C. elegnas strains. Pt.19 with pathogenic de novo P814L variant, 2y6m old, demonstrating upper limb tremor during fine motor activities as well as minor head titubation. Pt.19 with pathogenic de novo P814L variant, 2y9m old, demonstrating continuous motor movements involving trunk, head titubation and upper limb tremor during fine motor activities. Summary of genetic and clinical findings in individuals with pathogenic, VUS or (likely) benign variants in UNC13A. Immunocytochemical analysis of disease-related UNC13A variant expression at synapses. Electrophysiological characterization of synaptic transmission in cultured hippocampal neurons expressing UNC13A disease variants. Unprocessed immunoblots for Extended Data Fig. Source data used to generate Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Asadollahi, R., Ahmad, A., Boonsawat, P. et al. Pathogenic UNC13A variants cause a neurodevelopmental syndrome by impairing synaptic function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251021083650.htm'>A giant wave is rippling through the Milky Way, and scientists don't know why</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 06:30:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Data collected by the European Space Agency's Gaia space telescope show that our galaxy is not only rotating and wobbling but also sending out a vast wave that travels outward from its center. For about a century, astronomers have known that the Milky Way's stars orbit its core, and Gaia has precisely tracked their speeds and trajectories. Then in 2020, Gaia uncovered that this warped disc slowly oscillates over time, similar to the motion of a spinning top. Now, researchers have identified an enormous wave that moves through the Milky Way, influencing stars tens of thousands of light-years from the Sun. Thousands of bright stars, shown in red and blue, are overlaid on Gaia's detailed map of the Milky Way. The red and blue regions mark the newly discovered wave: red areas indicate stars located above the warped plane, while blue areas show stars lying below it. Although no spacecraft can venture beyond the galaxy, Gaia's remarkably precise measurements -- covering all three spatial dimensions (3D) and three components of motion (toward and away from us, and across the sky) -- allow scientists to construct these top-down and edge-on views of the Milky Way. "This observed behavior is consistent with what we would expect from a wave," Eloisa explains. If we could freeze that moment in time, some people would be standing upright, others would just have sat down (after the wave passed), and some would be about to stand (as the wave approaches). Galactic timescales are far longer, but the principle is similar. Eloisa and her team detected this remarkable motion by carefully studying young giant stars and Cepheid stars, both of which vary in brightness in predictable ways that make them easy for Gaia to observe across large distances. Newly formed stars could retain information from the gas they were born from, preserving a kind of "memory" of the wave. One possibility is that the Milky Way experienced a past encounter or collision with a smaller, dwarf galaxy, but further analysis is needed to confirm this. This newly found "great wave" might also have some link to a smaller undulating structure known as the Radcliffe Wave, which lies roughly 500 light-years from the Sun and extends about 9,000 light-years across. That's why we would like to do more research," Eloisa adds. "The upcoming fourth data release from Gaia will include even better positions and motions for Milky Way stars, including variable stars like Cepheids. This will help scientists to make even better maps, and thereby advance our understanding of these characteristic features in our home galaxy," says Johannes Sahlmann, ESA's Gaia Project Scientist. Note: Content may be edited for style and length. Scientists Reveal the Surprising Innermost Secrets of Spaghetti These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds A Strange Glow in the Milky Way May Be Our First Glimpse of Dark Matter Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251021083647.htm'>Atlantic dolphins are dying much younger. Scientists sound the alarm</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-22 05:48:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>"There is an urgent need to manage the population better," said Etienne Rouby, a postdoctoral researcher at the Institute of Arctic and Alpine Research (INSTAAR). Roughly 6 million common dolphins inhabit tropical and temperate waters worldwide, making them the most numerous members of the cetacean family, which includes whales and porpoises. However, this same region is also one of Europe's busiest fishing zones. While dolphins are not the intended catch, many become accidental victims of fishing operations. Known as "bycatch," this phenomenon causes thousands of dolphin deaths each year. Despite such figures, past surveys suggested that dolphin numbers in the area were stable. Traditional monitoring methods rely on counting dolphins spotted from ships or aircraft. Because the animals are constantly moving, these surveys can overlook population changes until they become severe. For species like dolphins, which reproduce slowly and live for decades, that delay can make recovery extremely difficult once the decline becomes visible, Rouby explained. To better track survival trends, the research team took a different approach by examining stranded dolphins that had washed ashore along the Bay of Biscay. Dolphins usually beach themselves when sick, injured, or disoriented, and most do not survive. Although stranded animals represent only about 10% of total deaths, their condition over time can reveal important patterns in population health. "We wanted to capture changes in the population's survival and fertility rates. This decrease has also resulted in fewer calves being born, signaling a broader reproductive decline. The study found that population growth has slowed by 2.4% since 1997. Under ideal conditions, common dolphin numbers can increase by about 4% per year, meaning that growth in 2019 was likely only around 1.6%. "The numbers are likely to be lower in reality," Rouby said. He warned that if this trend continues, growth could eventually turn negative, marking the beginning of an overall population decline. Although initial reports show the measure may be helping, Rouby suggested that adjusting the ban's timing based on dolphin migration patterns could make it more effective. Other North Atlantic cetaceans, including harbor porpoises and bottlenose dolphins, may be facing similar pressures. "Without these predators, fish populations could become out of control, and they would in turn consume too much plankton and vegetation until the system collapses." "As humans, we should make conscious decisions to protect the living and non-living things around us. Facing evidence of viability loss, we need to act before it is too late." These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds A Strange Glow in the Milky Way May Be Our First Glimpse of Dark Matter Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            