
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-27</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09419-8'>Maternal stress triggers early-life eczema through fetal mast cell programming</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 15:37:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Prenatal stress (PS) is a repeated exposure to aversive situations during pregnancy, including high emotional strain, which is suspected to affect homeostatic systems in infants. Paediatric eczema develops quickly after birth at flexural sites subjected to continuous mechanical constraints1,2. Although epidemiological studies have suggested an association between PS and a higher risk of eczema in children3,4,5,6, no causative biological link has yet been identified. Here we show that eczema at birth originates from molecular dysregulations of neuroimmune circuits in utero, triggered by fluctuations in the maternal hypothalamic–pituitary–adrenal axis. We found that offspring of stressed pregnant dams have dysregulated mast cells and skin-projecting neurons and quickly develop eczema in response to harmless mechanical friction. We demonstrated that PS transiently modulates amniotic fluid corticosterone concentrations, which directly alters the activation program of skin mast cells expressing the glucocorticoid receptor Nr3c1 and the adjacent sensory neurons conveying mechanosensation. Therapeutic normalization of maternal corticosterone concentrations or genetic depletion of Mcpt5+ mast cells during stressed gestation prevents fetal immune dysregulation and protects against eczema development after birth. Our findings support a new model in which early-onset paediatric eczema originates from dysregulations in the fetal immune system, caused by fluctuations in maternal glucocorticoids induced by stress. To study the effect of prenatal stress (PS) on atopic dermatitis at birth, we used a non-infectious mouse model of PS7, in which pregnant dams were restrained and exposed to bright light for 30 min, 3 times a day, from embryonic day (E)13 to E18 (Fig. 1a). The skin of 8-week-old (W8) PS offspring was visually and histologically comparable to that of control (CT) offspring, with no detectable changes in key macroscopic and microscopic features, including in the distribution pattern and abundance of key barrier proteins (Fig. However, both W3 and W8 offspring displayed a relatively high transepidermal water loss (TEWL) (Fig. 1f) at steady state, a well-known functional indicator of loose barrier in atopic dermatitis, especially in children8. These findings indicate that, although appearing to have normal skin, PS offspring exhibit subtle alterations in skin barrier and underlying indications of a predisposition to type 2 immune responses at steady state. d, Baseline TEWL measurements (in g m−2 h−1; n = 11). e, Serum cytokine concentrations (in pg ml−1) in CT (n = 7) and PS (n = 6) W8 offspring at steady state. f, Mild mechanical skin injury protocol (tape stripping (T/S)). Skin injury was induced by low repeats of tape stripping. i, Back skin of CT (n = 15) and PS (n = 17) W8 offspring was patched with PBS-filled gauze pads occluded in a dressing to favour innocuous mechanical trigger. j,k, Representative photographs and H&E staining of CT and PS back skin sections from Stim. W8 offspring (j) and epidermal thickness (in µm; n = 8; k). l, Spleen weight (in mg; n = 14 (CT) and 17 (PS)). Data were obtained from at least three independent experiments. Bars represent mean values, and one dot or square corresponds to a single male or female mouse. IgE, immunoglobulin E; NS, nonsignificant; stim., mechanical stimulation. Illustrations in f and i were created using BioRender (https://biorender.com). Compared with adult atopic dermatitis, paediatric atopic dermatitis often develops at flexural sites exposed to continuous mechanical constraints. We established a model of soft mechanical skin damage induced by tape stripping (Fig. 1f) using a pattern that did not induce any sign of inflammation in CT offspring. Conversely, both W3 and W8 PS offspring rapidly developed eruptions of eczematous lesions, revealing pronounced inflammation, with a significant increase in epidermal thickness (Fig. We next patched the back skin of the mice with sterile PBS-filled gauze pads occluded with a transparent dressing to induce a light but continuous wet friction (Fig. 1i) to mimic the wet flexural locations of babies. Although the W8 CT offspring did not show any signs of inflammation, the PS offspring rapidly developed severe lesions characterized by outbreaks of eczema with significant increase in epidermal thickness (Fig. 1j,k). However, compared with conventional allergen-induced models of adult atopic dermatitis9, there was no evidence of systemic type 2 inflammation (Fig. However, we found a strong expression of keratin 6, a marker of inflammation in keratinocytes (Extended Data Fig. 1i,j). Finally, a relatively mixed inflammatory signature of genes encoding type 1 (such as Ifng and Stat4), type 2 (such as Tslp, Il13 and Stat6), regulatory (such as Tgfb1) and type 17 (such as Il17a, Il17f, Ccl20 and Stat3) immune responses (Extended Data Fig. 1k) was observed in the skin of the PS offspring, a common clinical feature of atopic dermatitis often found in paediatric patients10,11. This phenotype also seemed to be restricted to skin tissue because no differences were observed in models of mast cell-dependent MrgprB2-mediated passive cutaneous anaphylaxis (Extended Data Fig. 1m) and house dust mite (HDM)/lipopolysaccharide-induced acute asthma (Extended Data Fig. Longitudinal clinical studies have consistently documented instances of spontaneous remission in paediatric atopic dermatitis, indicating a favourable prognosis for many children as they transition into adolescence2. Thus, we investigated whether the W24 PS offspring would also be susceptible to our protocol of light continuous wet friction (Extended Data Fig. 1r). Compared with their W8 counterpart, the W24 PS offspring displayed only a quickly resolving dryness, without any signs of eczematous lesions or epidermal thickening (Extended Data Fig. 1s). Overall, these data demonstrate that this model of PS is associated with the development of eczema-like skin lesions in progeny in response to otherwise harmless mechanical friction. The lesions exhibit different pathological features than adult atopic dermatitis, and this phenotype can naturally resolve with age. Beyond tissue inflammation, increased mechanical sensory sensitivity, which can worsen itch, disrupt sleep and impact the quality of life, is an important factor that participates in the full spectrum of discomfort experienced by children with atopic dermatitis12,13,14. Mechanoception is notably encoded by specialized populations of dorsal root ganglia (DRG) neurons that respond to different types of mechanical stimuli: Aβ and Aδ low-threshold mechanoreceptors (LTMRs), c-LTMR responding to light touch and non-peptidergic nociceptors, notably responding to noxious mechanical stimuli15,16,17,18. We assessed the W8 PS offspring mechanical sensitivity using three complementary behavioural assays (Fig. 2a–c): an alloknesis experiment by applying a series of calibrated monofilaments with increasing forces (von Frey filaments) to the back skin and recording of the elicited responses, a sticky-tape response assay measuring the success rate of adhesive tape removal and a mechanoception assay using the von Frey filament test applied to the glabrous skin and recording the threshold at which a sensory response is elicited. 1f–k, we found that PS offspring displayed significantly higher somatic mechanical sensitivity than CT offspring in all tested behavioural experiments (Fig. 2a–c), suggesting a development of mechanical hypersensitivity upon exposure to PS. We next isolated DRG of W8 CT and PS offspring and performed a comparative transcriptomic analysis using bulk RNA sequencing (RNA-seq) (Fig. 2d). We found more than 300 differentially expressed genes (DEGs) between CT and PS offspring with a minimum fold change more than 1.25 and an adjusted P value (Padj) < 0.05 (Fig. 2e,f). Among gene ontology terms enriched in PS DRG neurons at W8, we observed various pathways associated with metabolism and energy regulation, neuron development/differentiation and axon guidance (Fig. 2g). We next examined the expression of genes reported to be expressed by either peptidergic (PEP), non-peptidergic or c-LTMR (Fig. 2h). Genes associated with PEP nociceptors were globally downregulated in PS offspring, including preprotachykinin 1, Trpv1 and Calca (Fig. 2h). We observed a significant upregulation of genes associated with non-peptidergic neurons, including MrgprD and plexin-C1, as well as with c-LTMRs, such as Th, Cacna1h and P2Y purinoceptor 1 (Fig. 2h), with both neuronal populations being specialized in conveying cutaneous mechanical sensation. To investigate potential modified functional responses in the different populations, we next dissociated and cultured DRG neurons in vitro (Extended Data Fig. 2d). We monitored intracellular calcium concentrations in DRG neurons after the addition of 10 nM capsaicin (an agonist of TRPV1 mainly expressed on PEP neurons), 100 µM β-alanine (an agonist of MrgprD in non-peptidergic neurons) or 2 nM MRS2365 (a synthetic agonist of P2YR1 expressed on LTMRs). We observed that a significantly higher proportion of neurons responded to MRS2365 in DRG from PS offspring, whereas no significant differences were observed in response to β-alanine and capsaicin (Extended Data Fig. 2d). b, Sticky-tape assay and success rate in CT versus PS groups (n = 15). c, von Frey filament test and thresholds (in g) of CT (n = 10) and PS (n = 13) W8 offspring. d, DRG from CT and PS offspring (n = 3) were collected at W8 and bulk RNA sequenced. e, Volcano plot of the significantly upregulated (red) and downregulated (green) DEGs in PS W8 offspring. Minimum fold change greater than 1.25 and Padj < 0.05. f, Heat map of the top 50 DEGs between the DRG neurons of PS and CT W8 offspring. g, Gene ontology terms enriched in DRG of PS W8 offspring (top and bottom panels for upregulated and downregulated genes, respectively). h, Suggested markers for the identification of PEP, non-peptidergic (NP) and c-LTMRs (left panel). Volcano plot representing specific genes of PEP (orange), NP (blue) neurons and c-LTMRs (red) DEGs in PS W8 offspring (right panel). i,j, Representative confocal microscopy images of Tubβ3 (green; n = 8 (CT) and 7 (PS)), TH (red; n = 8) and IB4 (blue; n = 6) (i) and filament length (in µm; j) in hair follicles in back skin sections of W8 CT and PS offspring. Data were obtained from two or three independent experiments. Bars represent mean values, and each dot or square corresponds to a single mouse (a,c,j). Illustrations in a–c were created using BioRender (https://biorender.com). Finally, we used three-dimensional (3D) confocal microscopy to count the number of sensory neurons in DRG and map the axonal density of back and glabrous skin-projecting sensory neurons in W8 CT versus PS offspring at steady state. Compared with CT offspring, PS offspring displayed a significantly higher density of tubulin β3 (Tubβ3)+ fibres (pan neuronal marker), Tubβ3+TH+ fibres (identifies mainly c-LTMR and dopaminergic neurons in the skin) and Tubβ3+ isolectin B4+ fibres (IB4; identifies non-peptidergic neurons) in the back skin (Fig. 2i,j). In accordance, PS offspring also showed a higher density of Tubβ3+ neurofilament H (NFH)+ fibres (stains all myelinated LTMR neurons) and glial cell line-derived neurotrophic factor family receptor alpha 2 (Gfrα2)+ (identifies non-peptidergic neurons) in the glabrous skin (Extended Data Fig. 2h–j). In DRG, although IB4+ non-peptidergic neurons remained unchanged, PS offspring displayed a higher percentage of TH+ neurons and a reduced percentage of substance P+ PEP neurons (Extended Data Fig. 2k,l). In line with these findings, the chemical ablation of TRPV1+ PEP neurons using resiniferatoxin (RTX)9 did not impact the development of inflammation in response to soft mechanical skin damage (Extended Data Fig. 3a–c). We also used Nav1.8-cre;Dtafl/fl mice (Nav1.8 is also known as Scn10a), in which sensory neurons, including c-LTMRs and, possibly, a subset of sympathetic neurons, are genetically depleted to assess whether PS-induced neuronal alterations also contribute to the inflammatory phenotype or are instead more restricted to heightened skin sensitivity. Compared with CT offspring, both PS Nav1.8-cre;Dtafl/fl and PS littermate CTs developed comparable eczema-like skin lesions in response to otherwise harmless mechanical friction (Extended Data Fig. 3d), suggesting that sensory neurons are not required for inflammation onset. Finally, peripheral chemical sympathectomy using neurotoxin 6-hydroxydopamine (6-OHDA) also did not abrogate the hypersensitivity phenotype because PS offspring still exhibited reduced sensitivity thresholds and increased c-LTMR filament length (Extended Data Fig. These results show that PS was associated with transcriptomic, functional and anatomical modifications in non-peptidergic and c-LTMR neuronal compartments, along with abnormal responsiveness to mechanical stimuli. Although not involved in the development of the observed inflammatory reaction, such heightened sensitivity may help explain, at least in part, the sensory discomfort frequently reported in paediatric eczema, underscoring the need to address both immune and sensory dimensions of the disease. We, along with others, have previously reported that tissue-resident immune cells, including mast cells, play an important role in maintaining skin homeostasis9,19,20,21. We used flow cytometry to investigate the abundance of skin-resident immune cells (Extended Data Fig. 4a–c) in W8 CT versus PS offspring and found no significant differences in lymphoid and myeloid compartments (Extended Data Fig. 4d,e). To understand the potential molecular impact of stress during gestation on skin-resident immune cells, we used fluorescence-activated cell sorting (FACS) to isolate CD45+ immune cells from the skin of W8 and W24 mice (previously shown to be responsive and unresponsive to light continuous wet friction, respectively; Extended Data Fig. 1r,s) and performed a single-cell RNA sequencing (scRNA-seq) to generate transcriptional profiles for each individual cell (Fig. By exploring canonical markers of immune populations in the different Seurat clusters, 11 immune populations were identified (Fig. We next analysed the DEGs between the two conditions at W8 and found that skin mast cells (expressing Cpa3, Tpsb2 and Mrgprb2) presented the most significant number of transcriptomic modifications with 530 DEGs (Fig. 3d). Among the identified DEGs enriched in PS mast cells, many of them were found to be involved in granule formation/trafficking, such as Vamp8, syndecan 4 and Rac1, as well as in cell activation/metabolism, such as apolipoprotein E, S100a6 and S100a11, Atp5e and neuropeptide receptor Ramp1 (Fig. 3e). Using fluorescent avidin, which specifically binds to mast cell granules in the skin23, we found that skin mast cells from PS offspring were already highly activated/degranulated (presence of exteriorized cytoplasmic granules) at steady state (Fig. 3f,g). Among other gene ontology terms enriched in PS mast cells, we found stress-associated neuroendocrine pathways, oxidative phosphorylation and abnormal immune system physiology (Fig. 3h). Skin-resident mast cells of stressed pregnant dams were not more degranulated (Extended Data Fig. 5a), indicating that the observed phenotype is restricted to fetal tissue. a, Single-cell RNA sequencing of W8 and W24 (n = 3) CD45+ immune cells from back skin. c, Heat map of main canonical marker expression. d, Number of DEGs in PS compared with CT W8 offspring. f, Representative confocal microscopy images of avidin SRho+ mast cells (red) in the back skin of W8 CT and PS offspring. g, Mast cell (MC) counts per square millimetre (n = 7 (CT) and 9 (PS)). h, Gene ontology terms enriched in PS W8 mast cells. i, Mating pairs were designed as follows: KitWsh/Wsh females with KitWsh/Wsh males and iDTAfl/fl females with Mcpt5-cre+ males. j–m, Representative photographs and H&E staining of CT and PS back skin sections from W8 stim. n, Heat map of the top 100 W8 DEGs and normalized gene expression. o, Representative confocal microscopy images of avidin SRho+ mast cells (red) in the back skin of W24 CT and PS offspring. p, MC counts per square millimetre (n = 3 (CT) and 7 (PS)). Data were obtained from three independent experiments. Each dot or square corresponds to a single mouse (g,k,m,p). Illustrations in a were created using BioRender (https://biorender.com). We then investigated the role played by mast cells in PS-associated eczematous lesions in response to our model of light continuous wet friction using two complementary transgenic models (Fig. 3i): KitWSh/WSh mice (deficient in mast cells but have other abnormalities) and Mcpt5-cre;Dta mice (selectively lack most of the skin mast cells; Extended Data Fig. 5b). Compared with their respective littermate CTs, we found that both mast cell-deficient strains were almost completely protected from the development of PS-associated eczematous lesions in this model (Fig. 3j–m). However, Mcpt5-cre+/−;Dta mice did not show signs of improvement in von Frey mechanical sensitivity and neuron density in the skin (Extended Data Fig. 5c–e), suggesting that the mast cell and sensory dimensions of the disease might not be directly interconnected in this model. Compared with W8 PS mast cells, both the transcriptomic program and activation/degranulation status of W24 PS mast cells were back to normal, that is, similar to that found in W8 and W24 CT mast cells (Fig. 3n–p). We next investigated the presence of epigenetic modifications in W8 and W24 PS versus CT mast cells using a single-nucleus assay for transposase-accessible chromatin sequencing (snATAC-seq) on CD45+ cells isolated from skin tissue (Extended Data Fig. 5f–h). We found that W8 PS mast cells showed a relatively high number of differentially accessible regions (DARs) compared with W8 CT mast cells, indicating epigenetic modifications induced by PS. However, such epigenetic imprinting was strongly reduced at W24 (Extended Data Fig. 5i). These data demonstrate that PS-associated eczematous lesions in response to light wet friction depend on Mcpt5+ skin mast cells for full development through profound dysregulation of their transcriptomic/epigenetic program. However, such dysregulations are no longer present in skin mast cells isolated at W24, a stage at which the pathogenic phenotype has naturally reversed. We, along with others, previously reported that skin mast cells are of fetal origin, long-lived and independent of bone marrow for renewal22,24,25. We next investigated whether PS could impact fetal skin mast cell programming in utero (Fig. 4a). We first assessed the number and degranulation status of skin mast cells in whole-mount fetuses by 3D confocal microscopy at E18.5, after the last session of PS, a time point at which the vast majority of them are suspected to be yolk-sac-derived24. Although the number and distribution of mast cells were unchanged (Fig. 4b,c), these cells were already highly activated/degranulated at steady state in PS fetal skin compared with CT (Fig. 4b,d). We next investigated which population of fetal mast cells would be the most affected by PS. We administered a single dose of 4-hydroxytamoxifen (4OHT) to pregnant Cdh5-creERT2+/−;mTmG (cell membrane-localized tdTomato (mT) and cell membrane-localized EGFP (mG)) dams at E7.5 or E10.5 to fate-map yolk-sac-derived or haematopoietic stem cell (HSC)-derived mast cells, respectively (Extended Data Fig. 5j). We next used Cdh5-creERT2+/−;Tdt mice to fate-map yolk-sac-derived mast cells during postnatal development (Extended Data Fig. 5j). We found that the majority of mast cells were yolk-sac-derived at W8, but at W24, these cells were not detectable anymore (Extended Data Fig. 5n,o). These data are in line with reports suggesting that the natural turnover of long-lived skin mast cells is at least 24 weeks22,24,25. Altogether, these results could explain, at least in part, why the PS-hypersensitive skin phenotype naturally reversed at W24. a–f, Fetal skin from E18.5 embryos was collected (a) and analysed using confocal microscopy (b–f). g–k, Fetal mast cells were then sorted and analysed using bulk RNA-seq. b, Representative microscopy images of avidin SRho+ mast cells in whole embryo sagittal section (left) and 3D back skin (right) in CT and PS E18.5 offspring. c,d, Total (c) and degranulated (d) mast cell counts per square millimetre of back skin (n = 7). e,f, Representative 3D confocal microscopy images of Tubβ3 staining (green) in fetal skin of CT and PS E18.5 offspring (e) and associated filament length (in µm; n = 5 (CT) and 6 (PS); f). Data were obtained from three independent experiments. Bars represent mean values, and each dot corresponds to a single mouse (c,d,f). ; **P < 0.01; Mann–Whitney U-test (c,d,f), two-sided. i, Volcano plot of the significantly upregulated (red) and downregulated (green) DEGs in PS E18.5 offspring. j, Heat map of the top 50 DEGs between the mast cells of CT and PS E18.5 offspring. k, Gene ontology terms enriched in mast cells from PS versus CT (blue, related to mast cell biology; red, related to response to glucocorticoid). The left upper panel is for upregulated genes, whereas the left lower panel is for downregulated genes. Examples of genes enriched in PS mast cells related to granule formation, remodelling, maturation and activation are shown in the right panels. The neuronal branching was found unchanged at E18.5 (Fig. 4e,f), which was consistent with previous studies describing that full branching of sensory neurons occurs over several weeks after birth26,27. To analyse putative molecular changes responsible for such a phenotype, we sorted fetal skin mast cells (as CD45+CD64−F480−CD117+ST2+ skin cells) from three pools of four E18.5 CT and PS fetuses (Fig. Using a principal component analysis (PCA), we found that skin mast cells from CT and PS fetuses clustered separately (Fig. 4h), indicating distinct transcriptomic signatures in PS and CT offspring. Fetal mast cells from PS fetuses exhibited 1,284 DEGs (minimum fold change greater than 1.25; Padj < 0.05), with 874 upregulated genes and 478 downregulated genes (Fig. 4i,j). In line with our scRNA-seq dataset at W8 (Fig. We also observed a strong enrichment in genes associated with glucocorticoid signalling and neuronal synaptic plasticity (Fig. 4k). In conclusion, such a large transcriptomic modification upon exposure to early-life stress highlights a significant perturbation in the development/biology of fetal mast cells in utero, including genes inherent to granule formation, cell activation/maturation and neuronal modulation. We investigated in silico whether embryonic sensory neurons and/or mast cells could express the glucocorticoid receptor Nr3c1 or the β2-adrenergic receptor Adrb2 during development, potentially integrating signals from maternal HPA or sympathetic–adrenal–medullary axes, respectively. We used publicly available scRNA-seq datasets of mouse skin29 and DRG development15 (Fig. We found that sensory neurons expressed Nr3c1 but not Adrb2 during development (Extended Data Fig. 6b). We isolated mast cells and macrophages (as another yolk-sac-derived myeloid cell control) from fetal skin identified by their respective expression of Cpa3 and Csf1r (Fig. 5b). At E13.5, we found that macrophages, but not mast cells, expressed the gene Adrb2. Compared with macrophages, E13.5 skin mast cells expressed high levels of Nr3c1, which stayed relatively stable over time (Fig. 5c,d). In line with these findings, when exploring our previously described sequencing datasets (Figs. 3 and 4), we found that PS (but not CT) E18.5 skin mast cells expressed a strong signature of genes well known to be activated in response to glucocorticoid signalling (Fig. 5e). As fetal skin is directly in contact with the amniotic fluid, we sought to determine corticosterone concentrations at the maternal–fetal interface. Corticosterone concentrations were found significantly upregulated in the amniotic fluid and maternal blood of stressed dams (Fig. However, this increase was not observed in E18.5 embryos (Fig. We found a significant enrichment of pro-inflammatory cytokines in the amniotic fluid from PS yolk sacs compared with CT (Fig. 5g). In line with these observations, we sought to investigate whether corticosterone and/or corticosterone-enriched amniotic fluid could directly modulate fetal mast cell activation. We sorted CD117+ mast cells from E18.5 fetal skin and monitored mast cell degranulation dynamics upon the addition of different stimuli in vitro23 (Fig. 5h). We first validated our experimental framework and observed efficient activation of fetal mast cells upon the addition of substance P as a positive CT (Fig. 5i). Corticosterone addition resulted in a significant increase in mean fluorescence intensity (MFI) compared with vehicle CT, showing that it can directly act on fetal mast cell activation program (Fig. 5j). In line with this finding, the amniotic fluid from PS yolk sacs, and not from CT, also induced fetal skin mast cells to degranulate (Fig. 5j). We also monitored the intracellular calcium concentrations of W8 cultured DRG neurons after the addition of corticosterone. Although not being statistically significant, there was a strong tendency to observe calcium flux in DRG neurons from both CT and PS offspring (Extended Data Fig. Dotted lines define the mast cell and macrophages populations. g, Heat map of cytokines significantly enriched in AF. h, Single-cell analysis of mast cell degranulation. i,j, Representative confocal microscopy pictures of avidin SRho staining (left) and associated MFI (right) (n = 476 (Tyrode), 335 (substance P (SP)), 651 (Tyrode–chloroform), 475 (corticosterone (CORT)), 353 (AF CT) and 361 (AF PS)) of fetal mast cells after stimulation with classical stimuli (i), corticosterone or amniotic fluids (j). l, Corticosterone concentrations (in ng ml−1) in dams serum (n = 13 (CT and PS), 12 (CT-Met) and 10 (PS-Met)). m, Heat map of AF cytokine concentrations. Upper rows show average values from g. n, Representative photographs and H&E after stim. o, von Frey thresholds (n = 9 (CT-Met) and 15 (PS-Met)). p, Representative confocal microscopy images of avidin SRho in back skin. q, Representative confocal microscopy pictures of avidin SRho and MFI (n = 771 (AF CT-Met) and 752 (AF PS-Met)). Data were obtained from at least three independent experiments. Investigating the role of corticosterone in PS-associated eczematous lesions is complex because glucocorticoids are essential during gestation for ensuring the structural and functional maturation of the fetal organs30,31. This led us to develop a protocol on the basis of repeated intraperitoneal injections of 50 mg kg−1 of metyrapone (a selective inhibitor of CYP11B1 responsible for corticosterone synthesis) before each stress session to restore optimal corticosterone concentrations in stressed pregnant dams (Fig. 5k). This protocol significantly blocked the increase in corticosterone triggered by PS (Fig. 5l). Such a ‘normalization' of corticosterone using metyrapone did not impact the weight of pregnant dams, the litter size or the weight of male and female offspring after birth (Extended Data Fig. 6f). It did not dampen the concentrations of cytokines found in the amniotic fluid after PS (Fig. However, W8 PS offspring born from metyrapone-treated dams were almost completely protected from the development of eczematous lesions upon mild continuous wet friction, with restored normal epidermal thickness (Fig. 5n) and mechanosensation, according to von Frey threshold analyses (Fig. 5o). Although this reversal was not associated with the reestablishment of a normal density of sensory neurons in the skin, neurons from CT-Met and PS-Met offspring exhibited similar responses to in vitro stimulation with MRS2365 and β-alanine (Extended Data Fig. 6g–i). Transient normalization of corticosterone concentrations prevented abnormal activation programs in E18.5 fetal and W8 adult skin mast cells (Fig. 5p), a phenomenon required for PS-associated phenotype (Fig. 3i–m). The amniotic fluid of metyrapone-treated PS dams failed to activate fetal mast cells in vitro, thereby reversing the previously observed phenotype and confirming that the presence of high concentrations of corticosterone, but not of inflammatory cytokines, is responsible for fetal mast cell activation (Fig. 5q). Finally, to confirm that the observed effect of corticosterone on mast cell biology was not only inherent to our mouse model of PS, we used transgenic AdKOv2 mice, in which the adrenal-specific ablation of Prkar1a (also known as Cnc1) reproduces the phenotype of adrenocorticotropic hormone-independent Cushing's syndrome32 (hypercorticosteronaemia; Extended Data Fig. 6j). Compared with littermate wild-type CTs, we found that the skin of AdKOv2 mice exhibited abnormally activated/degranulated mast cells, a phenotype that resembles that of PS offspring (Extended Data Fig. 6k). Taken together, our findings in mice demonstrate that enhanced maternal corticosterone production induces transient alterations of fetal immune programming and predisposition to develop eczematous lesions at birth. In humans, atopic individuals frequently exhibit heightened anxiety and are prone to stress-induced exacerbations of their condition, as well as an increased risk of eczema in their offspring3,4,5,6. Although inherited genetic factors are well documented, growing evidence suggests that psychological stress and emotional states contribute to atopic disease development from birth. For example, newborns with a family history of atopy display an enhanced HPA axis response to a feet heel prick stressor33,34. We used publicly available scRNA-seq datasets from human embryos at different stages of development to investigate the presence of NR3C1 and ADRB2 in sensory neurons, mast cells and macrophages35,36 (Extended Data Fig. 7a,d). Mirroring our findings in mice, we found that developing sensory neurons and skin embryonic mast cells expressed high levels of NR3C1 but very low levels of ADRB2, whereas embryonic skin macrophages expressed high levels of ADRB2 but not NR3C1 (Extended Data Fig. 7b,c,e,f). We next had access to maternal blood collected from atopic (here clinically defined as having an HDM-positive basophil activation test (BAT)) and non-atopic (HDM-negative BAT) 58 pregnant women from the Healthy Early Life Moments in Singapore (HELMS) cohort37 at two distinct time points: 6–10 gestational weeks and during delivery (Extended Data Fig. 7g). We found that atopic pregnant women had significantly higher circulating concentrations of cortisol than non-atopic pregnant women during the gestational period (Extended Data Fig. 7h). However, by the time of delivery, cortisol concentrations in both groups converged, consistent with previous findings of a physiological glucocorticoid surge during labour38,39 (Extended Data Fig. These results strongly suggest a potential translational relevance of our findings obtained in mouse models, linking elevated blood glucocorticoid concentrations with enhanced atopic predisposition. Our findings support a new model in which the inflammatory and sensory features of early-onset paediatric eczema originate from molecular dysregulations of neuronal and immune compartments in utero triggered by fluctuations in the maternal HPA axis. These results provide a unique view on potent tissue-specific imprinting by maternal glucocorticoids and identify fetal skin mast cells as critical for full development of eczematous lesions at birth in response to otherwise innocuous mechanical friction (Extended Data Fig. 8). Previous studies have examined the effects of postnatal stress on organ physiology40,41,42,43, whereas others have explored how maternal IgE can sensitize fetal mast cells44. However, little is known about the impact of psychological PS on eczema outcome. Mast cell–sensory neuron units in the skin are key regulators of models of allergic skin inflammation in adult mice9,45,46,47,48,49,50. Other findings have also implicated neuroimmune interactions in neurodevelopment and behavioural response48,51,52,53. In this study, we did not find evidence of bidirectional neuroimmune communication. Our results show a parallel impact of maternal stress on fetal mast cells and somatosensory systems, potentially influencing the inflammatory and sensory aspects of the disease through distinct mechanisms. We found that Mcpt5-cre;Dta mice showed a persistent alteration of neuronal branching. However, those mice are still mast cell-sufficient in embryos; thus, it is difficult to strictly rule out a possible role of dysregulated fetal mast cells in the alteration of neuronal branching. Other models of stress have been linked to PS-associated microbiota dysbiosis, particularly at the gut level7,55,56,57. In addition, the introduction of a new solid diet several weeks after birth is often accompanied by a weaning reaction, which has been found to enhance the maturation of the immune system54,58,59,60. Although most of the observed perturbations in our PS model arise in utero, before colonization at birth or weaning, we cannot totally exclude a potential role of the skin or gut microbiota. It is also attractive to place our study in perspective with the ‘epithelial barrier hypothesis', which proposes that the increase in epithelial cell-damaging agents could loosen barriers61,62. In this context, it would be interesting to investigate whether PS offspring could be hypersensitive to even very low doses of polluting agents at birth. In summary, our results bring a new concept, in which early-onset eczema with remission could have a prenatal origin. It is interesting to speculate that such PS-associated ‘fragilized' skin barrier at birth could then favour skin colonization by pathogenic microbiota54, sensitization by domestic allergens and thus pave the way towards the so-called atopic march63,64,65. A deeper understanding of the delicate equilibrium at the maternal–fetal interface could lead to new interventions to alleviate the impact of environmental factors during pregnancy, ultimately reducing the extremely high incidence of paediatric eczema and eventually other allergic disorders. Mice were bred and housed in the local animal facilities of the Centre Régional d'Exploration Fonctionnelle et de Ressources Expérimentales (CREFRE) in pathogen-free sanitary conditions, with free access to water and food, and were submitted to alternating cycles of 12 h of light and darkness at an ambient temperature of 22 °C. C57BL6j mice were purchased from Janvier Labs or Charles River laboratories. Mcpt5-cre66, iDTA, AdKOv2 and Cdh5-creERT2;mTmG mice were kindly provided by A. Roers (Heidelberg University Hospital), J.-C. Guéry, P. Val and F. Lenfant, respectively. On the basis of the results obtained in our laboratory in models of atopic dermatitis9, we estimate that, on the basis of the difference in the mean values of the clinical scores and with a predicted 3.5 standard deviation for individual measurements, we need six mice per group to give us 90% power to detect a difference in clinical score at statistical significance of *P ≤ 0.05 using a two-tailed unpaired t-test. Sample sizes for the behavioural tests were chosen to ensure equal numbers of male and female animals and equal group sizes, consistent with previous publications67. Sample sizes for the in vitro culture experiments (DRG neuron cultures and mast cell activation assays) were also selected to ensure equal numbers of replicates per experimental condition. The number of biological replicates was on the basis of previous publications using similar methods9,23, providing sufficient power to detect meaningful effects while accounting for biological variability. To minimize bias, after allocating CT and PS groups (to ensure equal representation of groups), the investigators were blinded to the group and/or genotype of the mice whenever possible during animal model and data analysis. Although we did not expect to find differences in responses of male versus female mice, we have examined the data by sex. Both age-matched male and female mice were used in experiments. Littermate CT mice were used in all experiments in which transgenic mice were used. All animal care and experimentation were conducted in France in compliance with the guidelines of the European Union (86/609/EEC) and the French Committee of Ethics (87/848) policies and with the specific approval from the local ministry-approved committee on ethics in animal experimentation (Ethics Committee UMS006 CEEA-122; project no. Upon arrival in the animal facility, the animals were housed in accordance with European Directive 2010/63/EU. The mice were kept in cages at 20–24 °C with 45–65% humidity, in groups of three for females and two for males, and litters were housed in groups of five. Successful mating was judged by the presence of vaginal plugs, which defined E0.5 post-conception. Female weight was monitored all along the gestation period, and gestation was confirmed by a significant increase in body weight, where a weight gain of greater than 3.5 g from E0 to E13 was indicative of gestation. Briefly, pregnant dams were randomly attributed to CT and PS groups. The PS group was then subjected to chronic restraint stress in transparent drilled 50-ml Falcon Tubes (up to 6 mice per cage), put under a bright light (200 W), 3 times a day for 30 min each, from E13 until E18. The CT pregnant females were left undisturbed. Female dams were monitored every day during pregnancy and kept housed in groups of two or three animals until E18.5 after the last stress session when they were put in individual cages to prepare their nests. After birth, all pups were kept in the same cage with their biological mother. The pups were then weaned at the age of 21–28 days in distinct sex groups in new cages and introduced to a maintenance diet (SAFE A04; 84.1% cereals, 4% animal proteins, 8% plant proteins and 3.9% vitamins and minerals). Experiments were conducted at the desired age of offspring. The injection pattern and dose were determined to obtain an effective corticosterone synthesis inhibition while keeping the required levels for gestation. Serum for corticosterone dosage was collected immediately after the last session of stress on E18.5 and assessed for corticosterone concentrations. For induction of reporter recombination in the offspring, a single dose of 4OHT was delivered by intraperitoneal injections to pregnant females at E7.5 or E10.5. To counteract the adverse effects of 4OHT on pregnancies, 4OHT solutions were supplemented with progesterone. Females received a dose of 1.2-mg 4OHT and 0.6-mg progesterone. In cases when females could not give birth naturally, pups were delivered by C-section and cross fostered with lactating CD1 females. Mouse back skin was shaved and treated with a PBS-filled gauze pad occluded in Tegaderm Transparent Dressing (3M Health Care) every 2 days. On day 5, the gauze pads were removed, and the mice were left without any treatment for 2 days. This cycle was repeated for extra 2 weeks. Two days after the last cycle of treatment, the mice were euthanized, and back skin specimens corresponding to the treated areas were obtained for analysis. Skin inflammation was graded by unbiased measurement of the epidermal thickness. Only skin lesions found in the patching area were taken into consideration. The back skins of W3 and W8–W12 mice were shaved using an electrical razor. To disrupt the skin barrier, tape strips (autoclave tapes) were pressed firmly against the shaved area and removed immediately after application. Tweezers were used to remove the tape in a fluent stroke. The tape strip application was always conducted by the same person to avoid any bias owing to strength and applied pressure differences. Barrier permeability was monitored using TEWL measurements (in g m−2 h−1) that were recorded in optimal room conditions (temperature 20 °C; 40–60% relative humidity). Skin sections were disrupted in RIPA Lysis Buffer (1:10; EMD Millipore Sigma-Aldrich) supplemented with protease inhibitor cocktail tablets (cOmplete Mini EDTA-free tablets; EASYpack; Roche) and homogenized using CKMix beads (Bertin) and Precellys 24 homogenizer. Skin sections were then processed with two cycles of 6,500 rpm for 20 s. The protein concentrations of clarified tissue were measured by Pierce BCA Protein Assay Kits (Thermo Fisher Scientific). Samples were subsequently normalized to a concentration of 0.5 mg ml−1 with the 1× RIPA Lysis Buffer. The obtained skin lysates, amniotic fluid and serum samples were analysed using a proximity extension assay using the Olink Target 48 Mouse Cytokine panel in the presence of internal (incubation, extension and detection CTs) and external CTs. Data were analysed using a preprocessing normalization procedure. For each sample and data point, the corresponding quantification cycle (Cq) value for the extension CT was subtracted, thus normalizing for technical variation in one run. The protein concentration in standard concentration units (in pg ml−1) was obtained by fitting the normalized protein expression value to a standard curve. Each mouse received five innocuous mechanical stimuli at the nape of the neck from two von Frey filaments (0.07 g and 0.4 g; Bioseb BIO-VF-M). W8–W12 mice were acclimated for 30 min in the experimental room and then in Plexiglas boxes for 20 min. A white 12.7-mm circular adhesive tape was gently placed to the upper back of the mice between shoulders. The tape-directed responses of the mice were monitored as follows67: wet-dog shake, one bout; trying to reach tape with snout, one bout; burst of directed scratching with hind leg, one bout; neck grooming with front paw, one bout; grooming of head, lower back, paws, legs and ventral areas or tail, no bout. The following parameters were analysed: tape riddance (success/timeout), total number of bouts over the total trial time and bouts per minute, no response time (sum of time throughout which no bout occurred for 15 s or more) and time course of bouts represented as area under the curve of the time course plot (cumulative bouts versus time). Fisher's exact test was used to test whether PS affects the rate of mice managing to get the tape off in time. Time courses of bouts were plotted per second and then averaged for each time point. The area under these averaged time course curves (baseline at 0 bouts) was calculated using the trapezoid rule for every Δx = 1 s using GraphPad Prism. and the averages of the total bouts and total bouts per minute were then statistically analysed by two-way ANOVA followed by Holm–Šidák multiple comparison tests. W8–W12 were acclimated for 1 h in the testing arena. Increasing diameter filaments delivering increasing predetermined forces were applied perpendicularly to the hind paw surface of the mouse until they buckle. Any brisk behaviour, including paw withdrawal, licking or shaking during the probing or immediately after the filament was removed, was considered as a positive response. For each mouse, the paw withdrawal threshold was determined as the filament force that induced a positive response on three of five trials. The ears of W8 offspring were injected intradermally with 8-µg ciprofloxacine, an MrgprB2 agonist in 20-μl PBS. Ear swelling was measured every 10 min under light anaesthesia (2% isoflurane), starting just before injection (T0) and ending 2 h after injection (T120). Mice were sensitized with 1 mg of peanut extract (clinical-grade preparations used for skin testing; Stallergenes Laboratories) along with 10 μg of cholera toxin (Sigma-Aldrich) in 100-μl water (HCO3−) administered by means of oral gavage once a week for 4 weeks. One week after the last sensitization with peanut extract, the mice were challenged with the intraperitoneal injection of 1 mg of crude peanut extract (Greer Laboratories) in 100-μl PBS. Rectal temperature measurements were performed every 30 min, starting immediately before challenge (T0) and ending 2 h after challenge (T120). W8 offspring were sensitized by intranasal administration of lipopolysaccharide on day 0 (D0) and either PBS or 20 µg (30 µl) of the HDM strain Dermatophagoides pteronyssinus on D1 and D8, under general anaesthesia (not too long so that the mouse can breathe through its nose and inhale the liquid). Briefly, lung lobes were collected in RPMI medium in Miltenyi C Tubes and then digested with 0.5 mg ml−1 of DNase I (Sigma-Aldrich) and 0.25 mg ml−1 of Liberase (Roche) using the Miltenyi gentleMACS (protocol: mouse lungs). Cell suspensions were then filtered, and red blood cells were lysed using ammonium chloride–potassium lysis buffer for 5 min. The pellet was recovered and stained for FACS analysis. To ablate TRPV1+ nociceptors, W4 CT or PS mice were subcutaneously injected with increasing RTX doses of 30, 70 and 100 μg kg−1 in 100-μl PBS for three consecutive days. Some mice were injected with similar volumes of dimethyl sulfoxide (DMSO) in 100-μl PBS. Four weeks later, denervation was assessed using the classical tail flick assay. CT and PS mice received two intraperitoneal injections of 6-OHDA (150 mg kg−1) at 24-h interval. The mice were sacrificed 2 days after the first injection, and back skin was collected for imaging analysis. Frozen or paraffin-embedded mouse skin sections pretreated using a heat-induced epitope retrieval method (in 10 mM sodium citrate buffer (pH 6.0)) were permeabilized for 30 min in PBS 0.5% (w/v)% BSA and 0.3% Triton X-100 and incubated overnight at 4 °C with fluorophore-coupled antibodies or unconjugated antibodies: anti-Filaggrin (Ozyme; BLE905801; 1:50), anti-Loricrin (Ozyme; BLE905101; 1:50), anti-mouse Keratin 6A (Ozyme; BLE905701; 1:50), Anti-Claudin 1 antibody (Abcam; ab15098; 1:50), Alexa Fluor 647 anti-mouse CD45 (BioLegend; 103124; 1:400), anti-TH (Sigma; AB1542; 1:100), Alexa Fluor 647 anti-Tubulin β3 (BioLegend; 801210; 1:50), anti-human/mouse GFRα2 (R&D Systems; AF429; 1:200), anti-mouse NFH (Sigma; AB1989; 1:1,000), anti-GFP (Proteintech; pabg1; 1:200), anti-RFP (Takara; 632496; 1:200), Anti-Substance P (Merck Millipore; MAB356; 1:200) and anti-αSMA (Thermo Fisher Scientific; 14-9760-82; Clone 1A4; 1/100). The sections were then washed 3 times in PBS 0.5% (w/v)% BSA and 0.3% Triton X-100 and incubated, if needed, with the following secondary antibodies in PBS 0.5% (w/v)% BSA and 0.3% Triton X-100 for 2 h at room temperature in the dark: Donkey anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647 (Invitrogen; A31571; 1:200), Goat anti-Rabbit IgG (H+L) Cross-Adsorbed Secondary Antibody, Alexa Fluor 594 (Invitrogen; A11012; 1:200), Donkey anti-Sheep IgG (H+L) Cross-Adsorbed Secondary Antibody, Alexa Fluor 568 (Invitrogen, A21099; 1:200), Goat anti-Rabbit IgG (H+L) Cross-Adsorbed Secondary Antibody, Alexa Fluor 488 (Invitrogen; A11008; 1:200) and Goat anti-Rat IgG (H+L) Cross-Adsorbed Secondary Antibody, Alexa Fluor 647 (Thermo Fisher Scientific; A21247; 1:200). The slides were then washed, mounted in Mowiol medium and sealed with a coverslip. Z-stack images (512 × 512) were acquired using Leica STELLARIS, Leica SP8 and Zeiss LSM 710 Meta inverted confocal laser scanning microscopes and were processed using ImageJ (v.2.16.0) and Imaris (Bitplane; v.9.5.1) software. The cell bodies of neurons were counted on at least three images of three non-overlapping DRG sections (from lower thoracic to upper lumbar DRG) per mouse. Data were presented as percentage of Tdtomato+ neurons. Mouse back skin sections were stained according to standard procedures at the histopathology department CREFRE U006. Slides were then scanned using a Pannoramic Digital Slide Scanner (3DHISTECH). The 3D high-resolution images were taken using a Leica TCS SP8 MP Meta inverted confocal microscope, as described above. Cell suspensions from embryonic and adult mouse tissues were obtained through mechanical dissociation and enzymatic digest. Briefly, adult back skin from W8 or W24 mice was collected in pre-digestion medium, finely minced and incubated at 37 °C on a rotating plate to remove epithelial cells and other impurities. The samples were then digested for 45 min on a rotating plate with 1.25 mg of Liberase (Sigma) and 2.5 mg of DNAse I (Sigma) to disaggregate the tissue. The samples were further dissociated with the Miltenyi gentleMACS Dissociator, and cells were then enriched with a Percoll gradient. Fetal skin was collected, finely minced and digested in RPMI medium containing 0.1 mg ml−1 of DNAse I (Sigma), 400 U ml−1 of Collagenase I (Sigma), 0.1 mg ml−1 of Liberase DL (Roche) and 1 mg ml−1 of Dispase II (Roche) for 45 min on a rotating plate at 37 °C. Single-cell suspensions were then used for flow cytometry. The cell suspensions were blocked with anti-mouse CD16/32 (S17011E; BioLegend) for 15 min at 4 °C. Data were acquired on a BD FACSymphony cytometer and were analysed using FlowJo (Tree Star; v.10.10.0) software. Cell sorting was performed on BD FACSAria. Immune cell populations were gated as follows: All cells were pregated as viable (using SYTOX viability dye) singlets. Skin samples or DRG from adult mice were stored immediately in RNAlater then dissociated in TRIzol with the Miltenyi gentleMACS Dissociator. Tissue RNA was extracted using the RNeasy Qiagen Mini Kit according to the manufacturer's guidelines. Fetal mast cells were sorted, as described above. RNA was then extracted using RNeasy Qiagen Micro Kit according to the manufacturer's guidelines. RNA quality and titres were analysed using NanoDrop. RNA was either used for bulk RNA-seq or real-time quantitative polymerase chain reaction (qPCR). Sample quality CT and libraries were prepared at the GeT-Santé facility38,39. RNA concentration and purity were determined using a NanoDrop 2000 spectrophotometer (Thermo Fisher Scientific). The integrity of RNA was checked with a Fragment Analyzer (Agilent Technologies) using the RNA standard sensitivity kit. The 260/280 purity ratios were all greater than 1.8. The RNA-seq paired-end libraries were prepared at the GeT-Santé facility according to Illumina's protocol with some adjustments, using the TruSeq Stranded mRNA library prep Kit (Illumina). Briefly, messenger RNAs were first selected from either 3,000 ng (DRG sequencing) or 500 ng (sorted fetal mast cell sequencing) of total RNA using poly-T beads. RNAs were then fragmented for 2′ and retrotranscribed to generate double-stranded complementary DNA (cDNA). Eleven cycles of polymerase chain reaction (PCR) were applied to amplify libraries, and an extra double-sized purification step allowed to obtain 280–1,000 pb fragments. The quantification and sequencing of libraries were performed at the GeT-PlaGe core facility (Institut National de Recherche pour l'Agriculture, l'Alimentation et l'Environnement). Libraries were quantified by means of qPCR using the KAPA Library Quantification Kit (Roche) to obtain an accurate quantification. Libraries were equimolarly pooled, and RNA-seq was then performed on one S prime lane of the Illumina NovaSeq 6000 instrument (Illumina) using the NovaSeq 6000 S prime v.1 Reagent Kit (300 cycles) and a paired-end 2 × 150-pb strategy. FastQC was performed on sequence reads to evaluate the sequencing quality. Reads were then aligned on GRCm39 (mm10) as a reference genome using STAR (v.2.4.0)68 aligner and counted with HTseq (v.0.9.1)69, and a counting matrix was generated. Count matrix values were transformed using regularized log transformation implemented in DESeq2 (v.1.34.0)70. Approximately 16,000 cells per sample were encapsulated into droplets using Chromium Next GEM Single Cell 3′ Reagent Kits v.3.1 according to the manufacturer's protocol (10x Genomics). Briefly, after generation of nanolitre-scale Gel bead-in-EMulsion (GEM) using Next GEM Chip G, GEMs were reverse transcribed in a C1000 Touch Thermal Cycler (Bio-Rad) programmed at 53 °C for 45 min and 85 °C for 5 min and held at 4 °C. After reverse transcription, single-cell droplets were broken, and cDNA was isolated and cleaned with Clean-Up Mix containing Dynabeads (Thermo Fisher Scientific). Subsequently, the amplified cDNA was fragmented, end-repaired, A-tailed, index adaptor ligated and cleaned with Clean-Up Mix containing SPRIselect Reagent Kit (Beckman Coulter) in between steps. The sequencing-ready libraries were cleaned up with SPRIselect beads. Libraries prepared using 10x Genomics technology were pooled and charged with 1% PhiX on one S prime lane of the NovaSeq 6000 instrument (Illumina) using the NovaSeq 6000 S prime Reagent Kit v.1.5 (200 cycles) and the following sequencing parameters: 28 bp read 1, 8 bp index 1 (i7) and 150 bp read 2. The S prime lane generated a total of 434 × 106 reads. Transcript reads were demultiplexed, aligned to reference mouse genome (mm10 - GENCODE vM23/Ensembl 98) and quantified using the Cell Ranger v.7.0.1 (ref. 71). 73) was preferred as the main suite of analysis. Cells with more than 10% of their reads were assigned to mitochondrial genes, and cells with less than 100 detected genes were excluded from the dataset. We then ran the Seurat standard analysis pipeline: normalization, finding highly variable genes, scaling and PCA. Variance explainability of the principal component dimensions was observed, and the first 20 principal components were chosen for downstream analysis. For the annotated cell types, the Poisson test within Seurat's FindMarkers() function was run to compare one group to another. Genes with Padj < 0.05 and an absolute average log2(fold change) greater than 0.58 were determined to be significant in the given DEG test. Briefly, fresh cells were counted using a Countess 3 Automated Cell Counter (Thermo Fisher Scientific), and 150,000 cells were used. The cell pellet was homogenized in 100 µl of chilled 0.025% IGEPAL lysis buffer (10x Genomics). Lysate was incubated on ice for 2 min. At the lysis end, 1 ml of chilled wash buffer (10x Genomics) was added. After centrifugation (500 relative centrifugal force; 5 min; 4 °C), the supernatant was carefully removed, and the nuclei were resuspended in 1 ml of wash buffer. The nuclei suspension was then filtered through a 40-μm strainer and centrifuged (500 relative centrifugal force; 5 min; 4 °C) for a total of three washes to remove debris. Nuclei were immediately treated using the 10x Genomics Chromium Next GEM Single Nuclei Multiome ATAC + Gene Expression protocol (CG000338-Rev F). After transposition, the nuclei were diluted to optimal concentration and loaded into the Chromium Next GEM Chip J, followed by partitioning into GEMs using the Chromium X (10x Genomics). After GEM rupture, cDNA and transposed DNA fragments were purified using Dynabeads MyOne Silane (Thermo Fisher Scientific). Transposed DNA fragments underwent seven PCR cycle amplification to generate chromatin accessibility libraries, whereas cDNA fragments were further processed through end-repair, adaptor ligation and ten indexing PCR cycles to generate enriched gene expression libraries. The resulting gene expression and ATAC libraries were quantified using a Qubit Fluorometer (Thermo Fisher Scientific) and analysed for fragment size distribution using an Agilent TapeStation before sequencing. The libraries were pooled at the recommended ratio and sequenced on an Illumina NovaSeq 6000 platform at the Technology Cluster of the Toulouse Cancer Research Centre (CRCT) using paired-end reads (read 1, 50 bp; read 2, 49 bp; index 1, 8 bp; index 2, 24 bp) and using paired-end reads (read 1, 28 bp; read 2, 90 bp; index 1, 10 bp; index 2, 10 bp), targeting a sequencing depth of approximately 50,000 read pairs per nucleus for gene expression to achieve a sequencing depth of 3,350 median fragments per cell for 19,946 cells post-aggregation. Single-nucleus multiome sequencing data, including ATAC-seq and single-nucleus RNA-seq reads, were processed using Cell Ranger ARC (v.2.0.2). Reads were demultiplexed, aligned to the reference mouse genome (mm10 - GENCODE vM23/Ensembl 98) and quantified to generate filtered feature matrices and ATAC fragment files. The resulting datasets were loaded into Seurat (v.5.1.0) and Signac (v.1.14.0) for downstream analysis. Raw read files from all samples were merged into a single Seurat object, followed by quality control filtering to remove low-quality or multiple nuclei. Nuclei were retained if they met the following criteria: (single-nucleus RNA sequencing) 300 or more RNA features, total RNA counts of less than 5,000 and less than 10% mitochondrial RNA content; (snATAC-seq) ATAC peak counts between 100 and 20,000, nucleosome signal less than 0.75 and transcription start site enrichment score above 2. Gene expression values were log-normalized and scaled. Highly variable genes were identified; PCA was performed, retaining 50 principal components. Batch correction was applied using Harmony, and the corrected embeddings were stored for downstream analysis. For chromatin accessibility data, peaks with a minimum of five reads were retained. Term frequency–inverse document frequency normalization was applied, followed by singular value decomposition, keeping 50 latent semantic indexing components. Batch correction was applied using Harmony, and corrected embeddings were stored. A weighted nearest neighbour graph was constructed using Harmony-corrected embeddings from RNA and ATAC data. A joint UMAP was generated, and clustering was performed using the Louvain algorithm with a resolution of 0.1. For cell identification, similar markers to those listed above were used. To identify DARs of chromatin, pairwise comparisons were performed across experimental conditions. Only regions with a minimum detection rate of 5% across cells were considered. Genes associated with DARs were annotated using the ClosestFeature() function of Signac package (v.1.14.0). Summary statistics were recorded for each comparison, and the results were visualized using dot plots displaying the number of genes associated with DARs. RNA (1 μg) preparations from mouse skin were used for total RNA reverse transcription with SuperScript III First-Strand Synthesis System kit (Invitrogen) using random hexamers (Thermo Fisher Scientific) for priming. Transcripts encoding chemokine (C–C motif) ligand 20, thymic stromal lymphopoietin, interferon-γ, ILs (IL-10, IL-12p35, IL-12p40, IL-13, IL-17A, IL-17F, IL-22, IL-4, IL-5 and IL-6), oncostatin M, retinoic acid receptor-related orphan nuclear receptor-γt, signal transducer and activator of transcription (Stat; Stat3, Stat4 and Stat6) and transforming growth factor-β1 were quantified using real-time PCR using specific forward and reverse primers, the Takyon SYBR 2X MasterMix blue dTTP kit (Eurogentec) and quantified using the LightCycler 480 II (Roche Diagnostics). Blood samples were collected and centrifuged at 15,000g for 5 min. Sera were then collected and stored at −20 °C until use. Total IgE titres and corticosterone concentrations were quantified using an enzyme-linked immunosorbent assay (ELISA) kit (Invitrogen and Abcam, respectively) following the manufacturer's recommendations. Mast cells from E18.5 fetal skin (n = 24) were sorted and plated into poly-d-lysine-coated (5 µg ml−1; Sigma-Aldrich) six-well glass bottom plates with coverglass (IBL Baustoff + Labor) in Tyrode's buffer at 37 °C for 30 min, supplemented with 5 µg ml−1 avidin–sulforhodamine 101 (avidin SRho; Sigma-Aldrich). Cells were then stimulated with vehicle (Tyrode ± chloroform (negative CT)), 100 µM substance P (positive CT), 100 ng ml−1 of corticosterone and amniotic fluid (from CT, PS, CT-Met and PS-Met yolk sacs). Fluorescence was recorded after 30 min in a controlled atmosphere (using a ZEISS stage incubation system with objective heater; 37 °C) using Leica Sp8 confocal laser scanning microscope. MFI was quantified using the measurement function of ImageJ software for each defined single cell. DRG from all spinal levels were collected in Hanks' balanced salt solution without Ca and Mg and were digested with a mixture of 5 mg ml−1 of dispase and 1 mg ml−1 of collagenase type I at 37 °C for 45 min. DRG were then washed and further triturated with increasing gauge syringes in DMEM supplemented with 0.15 mg ml−1 of DNAse I. After dissociation, cells were spun at 300g and resuspended in medium before being plated in Nunc Lab-Tek II CC2 Chamber Slides with cover (Thermo Fisher Scientific). DRG were cultured for 24 h in DMEM supplemented with 50 ng ml−1 of nerve growth factor and 1:50 NeuroCult (STEMCELL) at 37 °C. The cells were loaded with Fluo-4 for 30 min in the dark at 37 °C in HEPES/Hanks' balanced salt solution immediately before imaging. The cells were imaged for 20 s to establish a baseline. At the end of every imaging trial, 50 mM KCl was added as a positive CT. Damaged, detached, high-baseline and motion-activated cells were excluded from analysis. This study was registered with the identifier NCT05207059 at ClinicalTrials.gov37. All subjects were recruited after obtaining necessary ethical approvals from the SingHealth Centralised Institutional Review Board in Singapore, and informed consent was obtained for every participant. Eligible participants were women aged 21–40 years with a body mass index of 25–40 kg m−2; intending to reside in Singapore for the next 4 years; identifying as Chinese, Malay, Indian or a combination of these ethnic groups; and planning to conceive within a year. Exclusion criteria included current pregnancy; known type 1 or type 2 diabetes; and recent use (past month) of anticonvulsants, oral steroids, certain contraceptives, fertility treatments or medications for human immunodeficiency virus or hepatitis B/C. Maternal blood was collected from pregnant women at two distinct time points: 6–10 gestational weeks and during delivery. BAT was performed, as described previously75, using HDM allergen, and responders were defined as those exhibiting greater than 38% degranulation upon HDM stimulation. Cortisol measurements in plasma were performed using ELISA kit (Abcam) following the manufacturer's recommendations. Kruskal–Wallis test for multiple comparisons using Dunn's correction, two-way ANOVA for multiple comparisons using Šidák corrections, Fisher's exact test and Mann–Whitney U-test were performed on samples, as noted in the respective figure legends. A P value of less than 0.05 was considered statistically significant. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The reference genome used in this study was mm10 - GENCODE vM23/Ensembl 98. The raw and processed datasets are available in the Gene Expression Omnibus under the following accession numbers: GSE271943 (bulk RNA-seq of fetal mast cells), GSE272046 (bulk RNA-seq of adult DRG), GSE272169 (single-cell RNA-seq of adult CD45+ cells) and GSE290848 (Multiome ATAC + Gene Expression of adult skin). Reanalysed publicly available data were from previously published studies15,29,35,36. Source data are provided with this paper. Bieber, T. et al. Clinical phenotypes and endophenotypes of atopic dermatitis: where are we, and where should we go? Smejda, K. et al. Maternal stress during pregnancy and allergic diseases in children during the first year of life. Exposure to psychosocial job strain during pregnancy and odds of asthma and atopic dermatitis among 7-year old children—a prospective cohort study. Letourneau, N. L. et al. Maternal sensitivity and social support protect against childhood atopic dermatitis. Sausenthaler, S. et al. Stress-related maternal factors during pregnancy in relation to childhood eczema: results from the LISA study. Identification of bacterial lipopeptides as key players in IBS. Skin barrier function and infant tidal flow-volubme loops—a population-based observational study. Serhan, N. et al. House dust mites activate nociceptor–mast cell clusters to drive type 2 skin inflammation. Esaki, H. et al. Early-onset pediatric atopic dermatitis is TH2 but also TH17 polarized in skin. Differences between pediatric and adult atopic dermatitis. Effects of atopic dermatitis on young American children and their families. The sensory profile of children with atopic dermatitis as determined by the sensory profile questionnaire. The relationship between sensory hypersensitivity and sleep quality of children with atopic dermatitis. The emergence of transcriptional identity in somatosensory neurons. Usoskin, D. et al. Unbiased classification of sensory neuron types by large-scale single-cell RNA sequencing. Qi, L. et al. A mouse DRG genetic toolkit reveals morphological and physiological diversity of somatosensory neuron subtypes. & Gaudenzio, N. Bidirectional sensory neuron-immune interactions: a new vision in the understanding of allergic inflammation. Feng, X., Zhan, H. & Sokol, C. L. Sensory neuronal control of skin barrier immunity. Zhang, S. et al. Nonpeptidergic neurons suppress mast cells via glutamate to maintain skin homeostasis. Tauber, M. et al. Landscape of mast cell populations across organs in mice and humans. Different activation signals induce distinct mast cell degranulation strategies. Gentek, R. et al. Hemogenic endothelial fate mapping reveals dual developmental origin of mast cells. Adult connective tissue-resident mast cells originate from late erythro-myeloid progenitors. Meltzer, S., Santiago, C., Sharma, N. & Ginty, D. D. The cellular and molecular basis of somatosensory neuron development. Koutsioumpa, C. et al. Skin-type-dependent development of murine mechanosensory neurons. Godoy, L. D., Rossignoli, M. T., Delfino-Pereira, P., Garcia-Cairasco, N. & de Lima Umeoka, E. H. A comprehensive overview on stress neurobiology: basic concepts and clinical implications. Lee, H. et al. Single-cell and spatial transcriptome analysis of dermal fibroblast development in perinatal mouse skin: dynamic lineage differentiation and key driver genes. Sahut-Barnola, I. et al. Cushing's syndrome and fetal features resurgence in adrenal cortex-specific Prkar1a knockout mice. & Hellhammer, D. Increased responsiveness of the hypothalamus-pituitary-adrenal (HPA) axis to stress in newborns with atopic disposition. Wright, R. J., Cohen, R. T. & Cohen, S. The impact of stress on the development and expression of atopy. Lu, T. et al. Decoding transcriptional identity in developing human sensory neurons and organoid modeling. Effects of an integrated mobile health lifestyle intervention among overweight and obese women planning for pregnancy in Singapore: protocol for the single-arm Healthy Early Life Moments in Singapore (HELMS) study. Chai, Y. et al. Changes of serum cortisol during pregnancy and labor initiation: an onsite cross-sectional study. Wang, W.-S., Guo, C.-M. & Sun, K. Cortisol regeneration in the fetal membranes, a coincidental or requisite event in human parturition? McClain, J. L. et al. Histamine-dependent interactions between mast cells, glia, and neurons are altered following early-life adversity in mice and humans. Early weaning stress induces chronic functional diarrhea, intestinal barrier defects, and increased mast cell activity in a porcine model of early life adversity. Pállinger, E., Tóthfalusi, L. & Csaba, G. Prolonged effect of endorphin treatment during pregnancy in the rat on the histamine content of immune cells of F1 and F2 offspring generations. & Bueno, L. Nerve growth factor mediates alterations of colonic sensitivity and mucosal barrier induced by neonatal stress in rats. Msallam, R. et al. Fetal mast cells mediate postnatal allergic responses dependent on maternal IgE. & Gaudenzio, N. Nociceptor–mast cell sensory clusters as regulators of skin homeostasis. Bao, C. & Abraham, S. N. Mast cell–sensory neuron crosstalk in allergic diseases. Green, D. P., Limjunyawong, N., Gour, N., Pundir, P. & Dong, X. A mast-cell-specific receptor mediates neurogenic inflammation and pain. Kim, E., Huh, J. R. & Choi, G. B. Prenatal and postnatal neuroimmune interactions in neurodevelopmental disorders. Veiga-Fernandes, H. & Mucida, D. Neuro-immune interactions at barrier surfaces. Chu, C., Artis, D. & Chiu, I. M. Neuro-immune interactions in the tissues. & Huh, J. R. Maternal inflammation and its ramifications on fetal neurodevelopment. Brain control of humoral immune responses amenable to behavioural modulation. Plum, T. et al. Mast cells link immune sensing to antigen-avoidance behaviour. Unique maternal immune and functional microbial profiles during prenatal stress. Zhang, Z. et al. Prenatal stress leads to deficits in brain development, mood related behaviors and gut microbiota in offspring. Lim, A. I. et al. Prenatal maternal infection promotes tissue-specific immunity and inflammation in offspring. Gensollen, T., Iyer, S. S., Kasper, D. L. & Blumberg, R. S. How colonization by microbiota in early life shapes the immune system. Differential peripheral immune signatures elicited by vegan versus ketogenic diets in humans. Al Nabhani, Z. et al. A weaning reaction to microbiota is required for resistance to immunopathologies in the adult. Does the epithelial barrier hypothesis explain the increase in allergy, autoimmunity and other chronic conditions? Ozdemir, C. et al. Lifestyle changes and industrialization in the development of allergic diseases. de Marco, R. et al. Foetal exposure to maternal stressful events increases the risk of having asthma and atopic diseases in childhood. & Spergel, J. M. The atopic march: critical evidence and clinical relevance. Oetjen, L. K. & Kim, B. S. Interactions of the immune and sensory nervous systems in atopy. Dudeck, A. et al. Mast cells are key promoters of contact allergy that mediate the adjuvant effects of haptens. & Schmidt, M. Maturational changes in mouse cutaneous touch and piezo2-mediated mechanotransduction. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Putri, G. H., Anders, S., Pyl, P. T., Pimanda, J. E. & Zanini, F. Analysing high-throughput sequencing data in Python with HTSeq 2.0. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Decontamination of ambient RNA in single-cell RNA-seq with DecontX. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. This study was supported by the Agence Nationale pour la Recherche, FONDATION ACTERIA and the European Research Council (ERC-2023-COG 101124255) to N. Gaudenzio. We extend our gratitude to all the members of the Gaudenzio laboratory at Infinity for their discussions and technical assistance; J. Martin for his help in the initial phase of the project; S. Allart, L. Lobjois and S. Lachambre for their technical support at the imaging facility of Infinity; F. A. Faqihi and H. Garnier for their assistance with the flow cytometry platform of Infinity; S. Milia, F. Abella and A. Aubry from the histopathology platform; R. Balouzat, E. Debon and F. Chaboud from the animal facility for their invaluable assistance at the experimental platform US06/CREFRE; E. Lhuillier and F. Martin from the Genotoul platform; A. Heider and M. Li for technical assistance in the proximity extension assay; and J. Ong, C. H. Jun and the SIgN Immunomonitoring platform for help with processing and analysis of the data from samples collected in Singapore. These authors contributed equally: Nasser S. Abdullah, Nadine Gheziel Nadine Serhan, Nasser S. Abdullah, Nadine Gheziel, Alexia Loste, Rüçhan Ekren, Elodie Labit, Anne-Alicia Gonzalez, Laurent Reber, Lilian Basso & Nicolas Gaudenzio Institut de Recherche en Santé Digestive, University of Toulouse, INSERM, INRAe, ENVT, UPS, Toulouse, France Nadine Gheziel, Camille Petitfils, Gaëlle Payros & Nicolas Cénac Holly Freya Grace Tinsley & Rebecca Gentek Unité de Recherche 1268 Biopolymères Interactions Assemblages, Institut National de Recherche pour l'Agriculture, l'Alimentation et l'Environnement, Nantes, France Inserm U1015, Gustave Roussy Cancer Campus, Villejuif, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and N. Gaudenzio were involved in experimental design. N. Gaudenzio is (or was) collaborating, consulting or serving as a member of the scientific advisory board for Genoskin (where he is CSO and a shareholder), Escient pharmaceuticals, Aikium, CEVA, MaxiVAX, Boehringer Ingelheim, Novartis, Sanofi, Allegria and argenx. The other authors declare no competing interests. Nature thanks Soman Abraham, Brian Kim and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Dams weight follow-up (n = 22 [CT], 18 [PS]). f, Baseline TEWL (g/m2/h; n = 13 [CT], 7 [PS]). h, Representative photographs and H&E in W3 skin, epidermal thickness (µm, CT [n = 10], PS [n = 9]). i,j, Representative confocal microscopy images of Stim. p, Representative confocal microscopy images of α-smooth muscle actin (αSMA) in W8 lungs. s, Representative photographs and H&E of Stim. W24 skin and epidermal thickness (µm; n = 4 [CT], 12 [PS]). Data are from at least two/three independent experiments. Each dot represents one mouse (b, c, f, h, q-s). Illustrations in l,m,n were created using BioRender (https://biorender.com). b,c, Averaged time courses ± SEM of tape-directed bouts (b) and AUC (c) of CT (n = 8 M/7 F) and PS (n = 9 M/6 F) offspring. d, Ex vivo DRG neuron stimulation and calcium imaging. e-g, Traces of the calcium-induced changes of fluorescence exclusively in responding neurons after addition of capsaicine (e, n = 35 [CT], 21 [PS]), b-alanine (f, n = 97 [CT], 57 [PS]) and MRS2365 (g, n = 87 [CT], 131 [PS]), and amplitude of the neuronal response measured as AUC of the highlighted duration (for 20 s after the addition of the stimulus), normalized per cell (upper panel) or per experiment (lower panel). Each triangle is a responding neuron, and each dot represents an experiment including 2 mice per group (d-g). h-j, Representative confocal microscopy images of neurofilament H (NFH, h) and GDNF Family Receptor Alpha 2 (Gfrα2, i) in glabrous skin sections and filament length (µm, j) per mm2 in steady state W8 CT (n = 6) and PS (n = 7) offspring. k,l, Representative confocal microscopy images of Tubβ3, TH, IB4 and Substance P staining (k) and associated percentages (l) among all Tdt+ neurons, in DRG from Nav1.8-Cre+;Tdt CT (n = 6) and PS (n = 6 [TH+], 7 [IB4+, SP+]) at steady state. Data are from at least two or three independent experiments. Bars represent mean values, and each dot or square corresponds to a single mouse (a,j,l). Illustrations in d were created using BioRender (https://biorender.com). W4 offspring was subcutaneously injected with increasing resiniferatoxin (RTX) doses of 30 μg/kg, 70 μg/kg and 100 μg/kg for three consecutive days. 4 weeks later, denervation was assessed using the classical tail flick assay, and a mild model of skin damage (T/S) was conducted. b, Tail flick latency (s) for CT and PS (n = 9 [DMSO], 10 [RTX]) offspring. c, Representative H&E staining and corresponding epidermal thickness (µm) of CT and PS (n = 4 [DMSO], 5 [RTX]) back skin sections of W8 offspring. d, Representative H&E staining and corresponding epidermal thickness (µm) of CT (n = 6 [Nav1.8-Cre-;DTAfl/fl], 8 [Nav1.8-Cre+;DTAfl/fl]) and PS (n = 5 [Nav1.8-Cre-;DTAfl/fl], 9 [Nav1.8-Cre+;DTAfl/fl]) back skin sections of W8 offspring. W8 offspring were intraperitoneally injected with the neurotoxin 6-hydroxydopamine (6-OHDA, 150 mg/kg). g,h, Representative confocal microscopy images of Tyrosine Hydroxylase (TH, red) (g) and associated filament length (µm, h) in back skin sections of 6-OHDA-treated W8 CT (n = 7) and PS (n = 9) offspring. Data are from one or two independent experiments. Bars represent mean values, and each dot, triangle or square corresponds to a single mouse (c, d, f, h). Immune phenotyping was then conducted using flow cytometry. b-e, Gating strategies of lymphoid (b) and myeloid (c) populations and corresponding cell proportions (d,e; n = 7). f, Enrichment of immune cells and sorting strategy. g, W8 and W24 immune cells transcriptome (n = 27,501) visualized with UMAP, colored according to unsupervised Seurat clustering. a,b, Representative confocal microscopy images of Avidin SRho+ mast cells in back skin of pregnant dams (n = 7, a), Mcpt5-Cre+/-;iDTAfl/fl (n = 12 [Mcpt5-Cre-/-;iDTAfl/fl], 13 [Mcpt5-Cre+/-;iDTAfl/fl], b), and counts per mm2. d, Representative confocal microscopy images and filament length (µm) of Tubβ3 (n = 8 [Cre-], 12 [Cre+]), TH (n = 6 [Cre-], 9 [Cre+]) and IB4 (n = 4 [Cre-], 8 [Cre+]) in PS back skin sections. f, Single nuclei Multiome ATAC seq. h, Heatmap of main canonical markers expression. i, Number of differentially accessible chromatin regions (DARs). k-m, Representative confocal microscopy images of Avidin SRho+ mast cells and GFP (k), percentages (%) of mG (GFP)+ mast cells among total mast cells (l) and degranulated mast cells (m), in fetal back skin of PS offspring, fate-mapped at E7.5 (n = 8) or E10.5 (n = 12). n,o, Representative confocal microscopy images of Avidin+ mast cells and Tdt (n) and percentages (%) of Tdt+ mast cells among total mast cells (o), in back skin of CT Cdh5-CreERT2+/-;Tdt W8 (n = 5) and W24 (6) offspring, fate-mapped at E7.5. Arrows indicate YS-derived mast cells (k, n). Data are from at least two or three independent experiments. Bars represent mean values, and each dot/square corresponds to a single mouse (a-d, m, o). a, UMAP plot of E12.5, E15.5, P0, P5 and adult DRG scRNAseq. Weight of male (n = 10 [CT], 7 [CT-Met], 10 [PS], 11 [PS-Met]) and female (n = 8 [CT], 12 [CT-Met], 18 [PS/PS-Met]) offspring (lower). g, Representative confocal microscopy images in W8 back skin. k, Representative confocal microscopy images of CD45 and AvidinSRho staining in back skin. Data are from at least three independent experiments. Each dot/square corresponds to a single mouse (d, f, h, j, k) or one experiment (2 mice/experiment, e, i). Dotted blue/red lines indicate the mean percentage from CT/PS, respectively. Mean + SEM; *P < 0.05, **P < 0.01, ***P < 0.001, ns: not significant; 2-way ANOVA for multiple comparisons using Šidák correction (c [P = 0.0022, F = 15.00], f [upper left, P = 0.2029, F = 1.785]), Mann-Whitney test (d, f [upper right], h, j, k), Kruskal Wallis for multiple comparison with Dunn's correction (e, f [lower]), two-sided. a, UMAP plot of the publicly available scRNAseq dataset for human fetal DRG performed at gestational week (GW)7-10, 12, 14, 15, 17 and 20. b, Feature plot of the expression of NR3C1 and ADRB2. c, Expression levels of NR3C1 and ADRB2 in neural crest cells, sensory neuron progenitors and nociceptors during development. d, UMAP plot of the publicly available scRNAseq dataset for human skin performed at GW 7.5, 10, 12.5, 15 and 17. e, Feature plot of the expression of NR3C1 and ADRB2. g, Description of the study cohort, including demographic and clinical characteristics, immune sensitization status (non atopic: NA, atopic: R), and cortisol levels. h-j, Cortisol levels (ng/ml) at GW6-10 (h, n = 16 [NR], 11 [R]) and during delivery (i, n = 17 [NR], 8 [R]), and associated averages (j). Mean + SEM; *P < 0.05, ns: not significant; Mann-Whitney test (h, i), two-sided. Illustration in g was created using BioRender (https://biorender.com). Activation of the HPA axis in pregnant dams upon exposure to chronic stress leads to increase in corticosterone-enriched and inflammatory in utero environment (1), which subsequently disrupts the development of long-lived yolk sac (YS)-derived fetal skin mast cells (2). Upon exposure to innocuous mechanical trigger (i.e. mild wet friction), the offspring develops severe eczematous lesions (5) that resolve with age, after the natural replacement of imprinted YS-derived mast cells (6). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Serhan, N., Abdullah, N.S., Gheziel, N. et al. Maternal stress triggers early-life eczema through fetal mast cell programming. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09427-8'>Cocaine chemogenetics blunts drug-seeking by synthetic physiology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 15:37:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Chemical feedback is ubiquitous in physiology but is challenging to study without perturbing basal functions. One example is addictive drugs, which elicit a positive-feedback cycle of drug-seeking and ingestion by acting on the brain to increase dopamine signalling1,2,3. However, interfering with this process by altering basal dopamine also adversely affects learning, movement, attention and wakefulness4. Here, inspired by physiological control systems, we developed a highly selective synthetic physiology approach to interfere with the positive-feedback cycle of addiction by installing a cocaine-dependent opposing signalling process into this body–brain signalling loop. We used protein engineering to create cocaine-gated ion channels that are selective for cocaine over other drugs and endogenous molecules. Expression of an excitatory cocaine-gated channel in the rat lateral habenula, a brain region that is normally inhibited by cocaine, suppressed cocaine self-administration without affecting food motivation. This artificial cocaine-activated chemogenetic process reduced the cocaine-induced extracellular dopamine rise in the nucleus accumbens. Our results show that cocaine chemogenetics is a selective approach for countering drug reinforcement by clamping dopamine release in the presence of cocaine. In the future, chemogenetic receptors could be developed for additional addictive drugs or hormones and metabolites, which would facilitate efforts to probe their neural circuit mechanisms using a synthetic physiology approach. As these chemogenetic ion channels are specific for cocaine over natural rewards, they may also offer a route towards gene therapies for cocaine addiction. The addictive properties of drugs are dependent on the temporal dynamics of their brain exposure, which is strongly affected by dose, route and reinforcement schedule5,6,7. Traditional approaches for addiction research and therapy use open-loop neural perturbations, such as pharmacology and chemogenetics8, that are not directly coupled to the addictive drug brain concentrations, and even fast-timescale optogenetic or electrical stimulation are not yoked to changes in addictive drug exposure levels9,10. As the neural signalling pathways from addictive drugs are sensitive to the drug-exposure time course, it is desirable to tailor neural perturbations to the temporal dynamics of addictive drug exposure. To address this limitation of open-loop techniques, we set out to develop specialized closed-loop chemogenetic receptors gated by specific addictive drugs. This is an approach to alter neuron activity in a manner that precisely mirrors the variable dynamics of addictive drug ingestion and brain pharmacokinetics. Inspired by physiological systems, we envisioned that engineered receptors could be used to create artificial opposing signalling processes between drug ingestion and drug reinforcement to selectively suppress drug-seeking (Fig. 1a) without affecting responses for natural rewards. a, Synthetic physiology schematic for using cocaine chemogenetics to reverse reinforcement after ingestion of an addictive drug that binds to a chemogenetic receptor targeted to a neural circuit. The diagram was adapted with permission from ref. b, The chemical structures of cocaine and ACh. c, Cocaine (yellow) binds to AChBP at the interface of two protomer subunits (cyan and pink). Homologous α7 nAChR pre-protein residues and numbering are shown (modified from Protein Data Bank (PDB): 2pgz). d, The potency of cocaine and ACh agonism of chimeric channels comprising single-mutation α7 nAChR LBDs and IPDs from either 5HT3 or GlyR. The red asterisks highlight chimeric channels with cocaine agonism and reduced ACh potency. e, The relationship between ACh potency and cocaine potency at α7–5HT3 mutant chimeric channels: L141G (1), L141G G175K (2), L141G G175K Y217F (3), L141G G175K Y115F (4), L141G G175K Y210F (5), L141G G175K Y210F Y217F (6). We initially focused on chemogenetic receptors for cocaine to add to the tools available for investigating psychostimulant mechanisms, which may also have implications for treating cocaine addiction. The reinforcing effects of cocaine are mediated by blockade of the dopamine reuptake transporter (DAT, encoded by Slc6a3), leading to elevated extracellular dopamine levels2. Cocaine-activated chemogenetic receptors would allow individual neural populations to be systematically examined for their role in sustaining cocaine-seeking. Cocaine is an especially challenging molecule for developing chemogenetic receptors because DAT is a cocaine-antagonized co-transporter protein not readily adaptable to creating a cocaine-gated ion channel that can directly modulate neural activity. However, cocaine has an amine-containing pharmacophore that is chemically analogous to molecules that have agonist activities at ion channels. Thus, we pursued a protein-engineering approach to develop cocaine-activated ion channels. α7 nicotinic acetylcholine (ACh) receptors (nAChRs) are ligand-gated ion channels, and they have been engineered to possess new pharmacological properties by altering the ligand-binding domain (LBD)11,12. The LBD of α7 nAChR can be spliced to the ion pore domain (IPD) of the cation-selective serotonin receptor or the anion-selective glycine receptor, and the resultant α7–5HT3 or α7–GlyR chimeric channel subunits self-assemble into pentameric ion channels for neuron activation or inhibition, respectively11,12,13,14 (Extended Data Fig. Importantly, agonists for nAChRs have pharmacophores with structural similarity to addictive drugs (Extended Data Fig. 1b) is a low-affinity antagonist of multiple nAChR subtypes15,16,17. A crystal structure of the homologous ACh-binding protein (AChBP)18 in a complex with cocaine (Fig. 1c) shows a binding mode similar to canonical nAChR agonists, such as nicotine. Moreover, tropisetron, a molecule with even greater structural similarity to cocaine that shares the N-methyl tropane ring structure (Extended Data Fig. We therefore reasoned that the carboxymethyl ester of cocaine (Extended Data Fig. 1b) may be preventing cocaine agonist activity by colliding with bulky amino acids on the complementary face of the ligand-binding pocket that is formed at the protein–protein interface of two α7 nAChR LBD subunits. We investigated whether we could identify mutations that would both enhance cocaine affinity for the α7 nAChR LBD as well as confer novel cocaine agonist activity for the corresponding chimeric ion channels. To identify mutated ion channels with cocaine agonist activity, we used a fluorescence membrane potential assay to screen the activity of chimeric α7–5HT3 and α7–GlyR ion channels with amino acid substitutions in the LBD that are in proximity to cocaine (Fig. Trp77Phe and Leu131Asn showed some cocaine agonist activity but also had undesired activity with the endogenous agonist ACh. Substitution of Leu141 with the smaller amino acid residues Gly or Ala produced a marked induction of cocaine agonism and reduced ACh potency. The bulky side chain of Leu141 is proximal to the carboxymethyl ester of cocaine (Fig. 1c), indicating that reduction of steric clash in the ligand-binding pocket promotes cocaine binding and agonist activity. To develop a cocaine-based chemogenetic platform for use in vivo, we performed additional optimization of the LBD to further improve cocaine potency and reduce ACh potency. ACh binding is facilitated by cation–pi interactions in the electron rich ‘aromatic cage' from multiple Tyr residues21,22,23. A reduction in the electron density by mutation of Tyr210 and Tyr217 (Extended Data Fig. 1c) each to Phe right-shifted the ACh potency but also reduced the cocaine potency (Fig. The additional mutation of Gly175Lys (Extended Data Fig. 1c), which was previously reported to increase nicotine binding24, improved the cocaine potency at α7(L141G,G175K,Y210F,Y217F)–5HT3, with the cocaine half-maximum effective concentration (EC50) = 1.5 ± 0.3 µM (Fig. This EC50 is below brain cocaine concentrations (~2.5–20 μM) estimated from self-administration6,25. The channel showed low potency activity of the endogenous nAChR agonists, ACh and choline (ACh EC50 = 216 ± 35 µM, choline EC50 = 1,212 ± 61 µM; Fig. Cocaine binding to α7(L141G,G175K,Y210F,Y217F)–5HT3 was also measured by displacement of the α7 nAChR antagonist [3H]ASEM28 (2 nM). This showed a biphasic binding curve with 45 ± 10% of high-affinity binding sites (high-affinity inhibition constant (KiH) = 1.6 ± 0.9 nM, low-affinity inhibition constant (KiL) = 10 ± 7 μM) (Fig. The KiH of α7(L141G,G175K,Y210F,Y217F)–5HT3 for cocaine is greater than the range of cocaine affinities (0.23–2.0 μM) reported for its endogenous target, DAT29,30,31. We named this cocaine-gated cation channel coca-5HT3. Importantly, coca-5HT3 was not activated by the main cocaine metabolites (ecgonine and benzoyl ecgonine; Extended Data Fig. 2b), probably due to unfavourable interactions with the free carboxylate group in these molecules. A panel of other addictive drugs that have an amine pharmacophore (amphetamine, methamphetamine, morphine, heroin and oxycodone) did not activate this receptor (Extended Data Fig. Even nicotine, which is an agonist of the unmutated α7–5HT3 chimeric channel, showed greater than 300-fold reduced potency for coca-5HT3 (Extended Data Fig. Finally, coca-5HT3 was not activated by a larger panel of endogenous signalling molecules at physiologically relevant concentrations (Extended Data Fig. Thus, coca-5HT3 exhibited highly selective pharmacology for cocaine, with no or low activity for other physiologically relevant amine-containing molecules that we tested. Some clinically used drugs have a tropane pharmacophore similar to cocaine, but most showed no or low potency activity at coca-5HT3 (Extended Data Fig. Coca-5HT3 showed robust cocaine-activated currents in whole-cell voltage-clamp recordings in HEK293 cells in which the steady-state window currents associated with prolonged agonist applications corresponded to the EC50 values from the membrane potential assay (Fig. The electrophysiological response to ACh (Fig. c, Depolarization in hippocampal neurons expressing coca-5HT3 (n = 11 cells) or GFP (n = 9 cells). Downward deflections indicate current injections to monitor membrane properties. e, The fold change in current necessary to elicit an action potential (rheobase) in neurons expressing coca-5HT3 (n = 9 cells) or GFP (n = 4 cells) in response to 3 µM cocaine. Statistical analysis was performed using a two-sided t-test; P = 0.023. f,g, Cocaine-activated (f) and ACh-activated (g) currents from coca-GlyR in HEK293 cells. h, Membrane potential assay of coca-GlyR potency for cocaine (left; n = 7 separate dose–responses) and ACh (right; n = 8 separate dose–responses) compared with unmodified α7–GlyR (n = 3 separate dose–responses each). i, Cocaine inhibits excitability to current injection of a hippocampal neuron expressing coca-GlyR. j, The input resistance of hippocampal neurons expressing coca-GlyR (n = 8 cells) or GFP (n = 13 cells) in the presence of cocaine. k, The fold change of current that elicits an action potential (rheobase) in neurons expressing coca-GlyR (n = 10 (3 µM) and n = 6 (10 µM) cells) or GFP (n = 4 (3 µM) and n = 7 (10 µM) cells) in response to cocaine (two-way ANOVA, transgene, F1,23 = 6.14, P = 0.02; dose, F1,23 = 0.69; P = 0.41; transgene × dose, F1,23 = 0.01, P = 0.93). To examine neuromodulation by these cocaine-gated channels, we transfected cultured hippocampal neurons with vectors that co-expressed the fluorescent GFP reporter (pCAG::coca-5HT3-IRES-GFP). Current-clamp recordings of the coca-5HT3 cation channel in cultured hippocampal neurons depolarized neurons in a dose-dependent manner (Fig. 2c) but did not affect control neurons. Neuronal firing was increased at cocaine concentrations of 1–3 µM (Fig. Consistent with this, the amplitude of current required to elicit an action potential (rheobase) was reduced by 3 µM cocaine (Fig. We obtained analogous results for a corresponding cocaine-gated chemogenetic chloride channel by swapping a similar α7 nAChR LBD (Leu141Gly, Gly175Lys and Tyr217Phe) onto the IPD of the glycine receptor (hereafter, coca-GlyR) (Fig. This channel showed a biphasic cocaine competition binding curve with 37.9 ± 0.5% high-affinity binding sites (KiH = 0.014 ± 0.01 nM; KiL = 850 ± 500 nM) (Extended Data Fig. Functional analysis showed large, sustained currents in whole-cell voltage-clamp recordings in transfected HEK293 cells (Fig. Coca-GlyR had a cocaine EC50 of 1.2 ± 0.1 µM (n = 7) and an ACh EC50 of 63 ± 12 µM (n = 8) (Fig. Cultured hippocampal neurons transduced with coca-GlyR fired normally in response to current injection, but firing was inhibited by cocaine (Fig. 2i), which was associated with a dose-dependent reduction of membrane input resistance (Fig. Correspondingly, the rheobase was increased (Fig. The passive membrane electrical properties of control neurons and neurons expressing coca-5HT3 or coca-GlyR did not show statistically significant differences (Fig. We next tested the use of cocaine chemogenetics to counteract the reinforcement processes modulated by cocaine. The lateral habenula (LHb) has been reported to have an anti-reward function that blunts excessive reward-seeking32 by activating rostromedial tegmental nucleus (RMTg) GABA neurons and ventral tegmental area (VTA) GABA neurons that both inhibit VTA dopamine neurons33 (Fig. The LHb is normally inhibited by cocaine in vivo, in part through elevated dopamine acting on the dopamine type 2 (D2) receptor in LHb neurons34, and LHb inhibition promotes cocaine self-administration35. Cocaine-mediated LHb inhibition is transient (0–15 min)34, tracking the similarly short brain-exposure time course of cocaine6, and is associated with reward in this timeframe6,34,36. We reasoned that an artificial opposing response produced by expressing coca-5HT3 excitatory channels in LHb neurons would reverse the sign of the cocaine-elicited inhibition response at this addiction node (Fig. This operation enabled us to install an artificial cocaine-dependent signalling node in the brain and to test the capacity of direct cocaine-mediated activation of LHb neurons to interfere with cocaine reinforcement solely during the dynamic drug-exposure time course after cocaine self-administration. a, Schematic of brain reinforcement circuits engineered to produce artificial feedback through cocaine-gated excitatory channels that oppose the natural suppression of LHb activity resulting from elevated dopamine (DA). Blue, dopamine; red, glutamate; green, GABA. DSt, dorsal striatum; NAc, nucleus accumbens. The diagram was created in BioRender. b, Schematic of dose–response curves for drug self-administration and the potential effects of interfering with reinforcement mechanisms. c, Organization of the cocaine self-administration experiment. d, Left, image of a coronal rat brain section expressing coca-5HT3-IRES-mCherry in the LHb (scale bar, 1 mm). Right, magnified image of the region indicated by the dashed box (scale bar, 200 µm). e,f, Acquisition of food (e; number of pellets per 3 h) and cocaine infusions per 3 h (f; 0.5 mg per kg per infusion) in rats with coca-5HT3 in the LHb (n = 11 rats) and controls (n = 15 rats). g,h, Rats with coca-5HT3 in the LHb (n = 11 rats) showed significantly lower cocaine infusions per 3 h at 0.125 mg per kg and 0.25 mg per kg (g; P = 0.01 and P = 0.04, respectively, Holm–Šidák post hoc comparison) and intake per 3 h at doses that maintain high-effort self-administration compared with the controls (h; n = 15 rats; mixed-effects ANOVA, dose × transgene interaction, P = 0.02). In the IVSA model of addiction, voluntary cocaine intake is dependent on the infusion dose, which approximates an inverted U-shaped curve. Interventions in animal models that shift down the dose–response curve are associated with clinical efficacy for addiction therapies, whereas treatments that right-shift the dose–response curve are more likely to result in compensatory increased drug administration37,38,39 (Fig. The LHb was targeted in rats by stereotaxic injection of the adeno-associated virus AAV5-Syn::coca-5HT3-IRES-mCherry (Fig. Control rats had sham surgery without AAV injection. LHb neurons expressing coca-5HT3 had similar basal electrophysiological properties to control neurons in the absence of cocaine (Extended Data Fig. Rats were initially trained for food self-administration. All of the rats showed acquisition of food self-administration and rats injected with coca-5HT3 did not differ significantly from the controls (Fig. 3e) (two-way repeated-measures analysis of variance (RM-ANOVA), transgene main effect (F1,24 = 2.21, P = 0.15), transgene × session interaction (F9,216 = 0.52, P = 0.85)), showing that food-seeking was unaffected by coca-5HT3 expression in the LHb. catheter was implanted into the jugular vein of both experimental groups, and the rats were exposed to the cocaine IVSA procedure using a reinforcing dose of cocaine (0.5 mg per kg per infusion). We found that rats expressing coca-5HT3 and control rats acquired cocaine IVSA similarly (Fig. 3f) (two-way RM-ANOVA, transgene × session interaction, F9,216 = 1.17, P = 0.31). This shows that coca-5HT3 does not significantly interfere with cocaine self-administration using a high infusion dose that does not require high instrumental responding to maintain exposure to reinforcing levels of cocaine. We performed a dose–response assessment in the coca-5HT3 and control groups after acquisition of cocaine IVSA. The dose–response curve for cocaine infusions in the control group showed a prominent peak at unit dose 0.125 mg per kg per infusion. Notably, none of the rats reached the infusion limit (60 infusions per session at 0.5 and 1 mg per kg unit doses). A mixed-effects ANOVA showed significant effects of dose (F4,94 = 29.82, P < 0.001) and dose × transgene interaction (F4,94 = 2.87, P = 0.02). In a follow up Holm–Šidák post hoc comparison, the rats injected with AAV5-Syn::coca-5HT3-IRES-mCherry had significantly fewer infusions than the control rats at the 0.125 mg per kg (P = 0.01) and 0.25 mg per kg (P = 0.04) unit doses (Fig. Coca-5HT3 expression did not significantly affect preference for sucrose solution (two-way RM-ANOVA, transgene main effect (F1,10 = 0.78, P = 0.39), session main effect (F1,10 = 60.8, P < 0.001), transgene × session interaction (F1,10 = 1.97, P = 0.19)) or operant responding for sucrose pellets (two-way RM-ANOVA, transgene main effect (F1,11 = 0.24, P = 0.63), session main effect (F5,55 = 13.64, P < 0.001), transgene × session interaction (F5,55 = 0.68, P = 0.63)) (Extended Data Fig. Moreover, coca-5HT3 expression did not significantly affect the locomotor response to a novel environment (two-way RM-ANOVA, transgene main effect (F1,10 = 0.04, P = 0.83), session main effect (F59,590 = 11.89, P < 0.001), transgene × session interaction (F59,590 = 0.86, P = 0.75)) (Extended Data Fig. These experiments confirm that, in the absence of cocaine, coca-5HT3 expression in the LHb does not significantly affect locomotion, hedonic food preference or cocaine-independent operant responding for sucrose. Cocaine's reinforcing and addictive properties are mediated by increases in extracellular dopamine in the nucleus accumbens2. Elevated LHb neuron activity negatively regulates dopamine release in the nucleus accumbens35,40,41,42,43,44,45. We hypothesized that cocaine-mediated activation of coca-5HT3 in the LHb would decrease cocaine-induced dopamine release in the nucleus accumbens. To investigate this, we first used small animal positron emission tomography (PET) and the dopamine D2/D3 receptor antagonist radiotracer [18F]fallypride. In vivo [18F]fallypride binding at D2/D3 receptors is displaceable by pharmacological or task-induced elevations of extracellular dopamine and constitutes a non-invasive approach to measure brain-wide changes in dopamine release in rodents and humans46,47. We hypothesized that cocaine would decrease [18F]fallypride binding through displacement by elevated extracellular dopamine in control rats and that this effect would be decreased in rats injected with coca-5HT3 in the LHb (Fig. Using a voxel-wise statistical parametric mapping analysis, we found that cocaine significantly decreased [18F]fallypride binding in control rats in a set of voxels encompassing the medial shell and core of the nucleus accumbens and the ventromedial division of the dorsal striatum (paired t-test, cocaine versus saline, t = 2.89, P = 0.01) (Fig. Rats expressing coca-5HT3 in the LHb and treated with cocaine showed less [18F]fallypride displacement in the ventral striatum compared with the control rats (three-way RM-ANOVA, time × transgene × brain region interaction, F25,300 = 3.83, P < 0.001) (Fig. In a separate group of rats, we confirmed that expression of coca-5HT3 in the LHb did not affect the striatal dopamine concentration (unpaired t-test, coca-5HT3 versus control, t = 1.07, P = 0.31) (Extended Data Fig. Overall, this indicates that cocaine activation of coca-5HT3 in the LHb suppresses cocaine-induced dopamine elevation in the ventral striatum. a, Schematic showing the stereotaxic delivery of AAV5-Syn::coca-5HT3-IRES-mCherry into the LHb of rats and [18F]fallypride PET procedures. The diagram was created in BioRender. b, Schematic showing the displacement of [18F]fallypride by cocaine detected using PET. The diagram was created in BioRender. c, Coronal PET brain images co-registered to a magnetic resonance imaging template showing significant cocaine-induced voxel-wise decreases in [18F]fallypride binding in the ventral striatum of control rats (n = 9 rats) and in rats with coca-5HT3 in the LHb (n = 5 rats). d, Time–activity curves of [18F]fallypride binding (standardized uptake value (SUV)) in the ventral striatum (squares) and cerebellum (triangles) calculated as the percentage of the baseline (60–90 min) for control (n = 9 rats, black) and coca-5HT3 (n = 5 rats, red) rats, showing that cocaine selectively displaces [18F]fallypride binding (that is, increases dopamine) in the ventral striatum of control rats (three-way RM-ANOVA, time × transgene × brain region interaction, F25,300 = 3.83, P < 0.001). The diagram was created in BioRender. f, Left, image of a coronal rat brain section showing the optic fibre tract and GRABDA expression (scale bar, 1 mm). g,h, GRABDA response time course with saline and cocaine (5 mg per kg) injection in control rats (g) (n = 5 rats; two-way RM-ANOVA, treatment main effect, F1,5 = 47.77, P = 0.001; time main effect, F2.2,11.1 = 1.84, P = 0.2; treatment × time interaction, F2.1,10.4 = 10.86, P = 0.002) and rats with coca-5HT3 in the LHb (h) (n = 6 rats; two-way RM-ANOVA, treatment main effect, F1,4 = 3.58, P = 0.13; time main effect, F1.5,6.1 = 0.73, P = 0.48; treatment × time interaction, F2.5,10.2 = 2.03, P = 0.17). i, The AUC of time-course measurements with saline- and cocaine-induced (5, 10 and 20 mg per kg) GRABDA responses. We used the same [18F]fallypride displacement procedure in rats expressing Cre recombinase in tyrosine hydroxylase (TH)-expressing dopamine neurons that were transduced by intracranial injection of AAV1-Syn::FLEX-coca-GlyR-IRES-mCherry to test the in vivo efficacy of coca-GlyR. Th-cre rats expressing coca-GlyR in VTA dopamine neurons showed less [18F]fallypride displacement in the ventral striatum compared with control rats (three-way RM-ANOVA, time × transgene × brain region interaction, F12,169 = 2.44, P = 0.006) (Extended Data Fig. 8) indicating that cocaine-gated inhibition of VTA dopamine neurons by coca-GlyR suppressed the typical cocaine-induced dopamine elevation in the ventral striatum. To further investigate dopamine levels in the nucleus accumbens during cocaine chemogenetics, we performed fibre photometry in rats using a genetically encoded, fluorescent G-protein-coupled receptor (GPCR)-activation-based dopamine (GRABDA) sensor48. Rats injected with AAV5-Syn::coca-5HT3-IRES-mCherry into the LHb and controls were implanted with an optical fibre coated with AAV9-Syn::GRABDA1m and targeted to the shell portion of the nucleus accumbens (Fig. Rats were anaesthetized and their optical fibre was connected to the patch cord. GRABDA signals were then collected for 5 min, followed by a saline intraperitoneal (i.p.) injection, and additional GRABDA signal measurement for 20 min. at either 20, 10 or 5 mg per kg, and the GRABDA signal was measured for an additional 20 min. Each rat was tested three times with the cocaine dose administered in a randomized order. Compared with saline, all doses of cocaine significantly increased the GRABDA response in control rats (Fig. 9) (two-way RM-ANOVA, 5 mg per kg: treatment × time interaction (F2.1,10.4 = 10.86, P = 0.002); 10 mg per kg: treatment × time interaction (F1.7,8.8 = 21.45, P = 0.005); 20 mg per kg: treatment × time interaction (F1.4,7.2 = 5.09, P = 0.04)). By contrast, cocaine did not produce statistically significant increases in the GRABDA response in coca-5HT3 rats at any dose (Fig. 10) (two-way RM-ANOVA, 5 mg per kg: treatment × time interaction (F2.5,10.2 = 2.04, P = 0.17); 10 mg per kg: treatment × time interaction (F1.1,8.8 = 1.24, P = 0.33); 20 mg per kg: treatment × time interaction (F1.3,5.4 = 0.37, P = 0.63)). Next, we calculated the area under the curve (AUC) to test for differences in the cocaine-induced GRABDA response between coca-5HT3 and control rats. Rats with coca-5HT3 had a lower GRABDA response compared with control rats that was statistically significant (two-way RM-ANOVA: transgene effect (F1,9 = 10.03, P = 0.01); dose × transgene interaction (F2,18 = 1.05, P = 0.37)) (Fig. Taken together, the lack of cocaine-induced [18F]fallypride displacement using PET and the lack of statistically significant cocaine-induced GRABDA responses in coca-5HT3 rats indicate that activation of coca-5HT3 in the LHb by cocaine inhibits cocaine-induced extracellular increases in nucleus accumbens dopamine, consistent with the lower cocaine self-administration behaviour that we observed in rats expressing coca-5HT3 in the LHb. Here we demonstrate a chemogenetic approach inspired by physiology that responds directly to cocaine and therefore enables the introduction of an artificial feedback process that is engaged when cocaine enters the brain and can suppress cocaine-seeking. This was implemented by engineering cocaine-specific activation of a circuit that is normally inhibited by cocaine; the activation of this circuit blunts cocaine-mediated elevation of extracellular dopamine. For this, we used protein engineering to generate ion channels that are activated by cocaine. These ion channels are highly sensitive to cocaine, and not sensitive to endogenous molecules or other addictive drugs, including nicotine, at physiologically relevant doses. Accordingly, these engineered cocaine-gated channels can be used to create artificial signalling processes opposing drug-seeking solely during drug exposure. Chemical feedback processes are ubiquitous in biology but are challenging to study because interference with receptors involved in chemical signalling (such as dopamine receptors) typically perturbs basal functions. Cocaine chemogenetics is a biologically inspired approach to study and perturb feedback processes in organisms. By using an addictive drug as the agonist for chemogenetic neuromodulation, this approach intrinsically couples chemogenetic neuromodulation to drug self-administration and pharmacokinetic exposure in the brain. This enables a unique form of temporal control that can be used to investigate and to interfere with the drug-induced reinforcement processes of addiction. Our findings show that activation of LHb neurons solely during cocaine exposure blunts high-effort cocaine self-administration. LHb activation did not interfere with the motor actions of cocaine-seeking or the ability of cocaine to reinforce these actions at high doses involving low effort to maintain a reinforcing dose. Instead, artificial cocaine-mediated LHb activation selectively suppressed the higher response rates necessary to maintain reinforcing exposure to cocaine at lower dose infusions. This was probably due to a reduction in the release of dopamine in the nucleus accumbens. In support of this, open-loop chemogenetic inhibition of VTA dopamine neurons similarly suppresses cocaine self-administration at intermediate infusion doses but not high doses49. The effect of LHb activation on cocaine self-administration is consistent with related investigations of LHb functions on reward-seeking. Reward administration elicits phasic reduction of LHb neuron activity and omission leads to LHb activation50. Following the rewarding spike in cocaine after i.v. administration, brain cocaine levels quickly fall within 15 min (ref. Cocaine activation of coca-5HT3 in the LHb artificially induces the anti-rewarding effect of LHb activation during the cocaine-exposure period, therefore opposing high-effort cocaine self-administration. This perturbation is intrinsically linked to the rise and fall of the cocaine concentration. The specific LHb subpopulations that are modulated by cocaine are not established; therefore, further investigation is needed to precisely target cocaine chemogenetics as a negative-feedback control system for cocaine reinforcement. The behavioural effect of reduced cocaine self-administration is consistent with other studies showing that LHb activation reduces reward-seeking behaviours51,52 and is associated with depression53. Here cocaine-induced activation of the LHb blunted the cocaine-induced dopamine rise without suppressing dopamine below the baseline levels, which is desirable to counteract the reinforcing effects of cocaine without gross disruptions to other functions of dopamine signalling. The relationship between motor vigour and elevated dopamine signalling54,55,56 probably contributes to the large effects of this chemogenetic process to suppress high effort responding for cocaine. One limitation of this study is that it focused solely on male rats and included the use of two different rat strains. Further investigation is needed to examine the extent to which these effects generalize to female rats and other rat strains or species. Cocaine chemogenetics could be further deployed to investigate the contributions to addiction behaviours by other neural populations. For example, cocaine-induced chemogenetic activation of a pain- or sickness-inducing neuron population may also alter drug self-administration, which would be analogous to the mechanism of the alcohol-use-disorder treatment drug disulfiram. Importantly, cocaine chemogenetics is a means to selectively perturb the outcome evaluation phase after drug ingestion, which is challenging even with rapid optogenetic methods because these perturbations are difficult to precisely yoke to drug exposure in the brain. In the future, chemogenetic receptors can be developed with selectivity for additional addictive drugs or even drug combinations, which would facilitate efforts to study their neural circuit mechanisms using drug-mediated chemogenetic neuromodulation. Based on what is learned with addictive drugs, these closed-loop chemogenetic approaches may be further applied to other body–brain signalling processes, such as those involving hormones or nutrients. The chemogenetic receptors reported here are specific for cocaine over food rewards. This is important because cocaine alters signalling pathways that are also used for processing natural rewards. Indeed, activation of the LHb suppresses food consumption51,52; it is therefore desirable to activate the LHb only during cocaine taking. The shared role of drug-seeking circuits with motivation for healthy physiologically relevant rewards has been a major impediment to developing pharmacotherapies for addictive drugs. We found that cocaine-gated chemogenetic LHb activation suppresses cocaine intake in rats using a well-validated preclinical model of cocaine abuse liability. As these chemogenetic ion channels are specific for cocaine, they may eventually lead to gene therapies selective for cocaine addiction without affecting enjoyment of natural rewards. Site-directed mutagenesis was performed using QuickChange Lightning SDM (Agilent). For these constructs, the α7 nAChR LBD was based on the sequence of human CHRNA7. For testing in human embryonic kidney (HEK) cells, channels were expressed from pcDNA3.1, (Invitrogen/Thermo Fisher Scientific). For Cre-dependent rAAV, constructs containing Coca-GlyR followed by IRES-mCherry were cloned into an AAV2-Synapsin(Syn)-FLEX vector57 in the reverse orientation with respect to the Syn promoter. Using these constructs, rAAV serotype 1 was produced by the Janelia Molecular Biology Core Facility. Coca-5HT3 (α7(L141G,G175K,Y210F,Y217F)–5HT3) or coca-GlyR (α7(L141G,G175K,Y217F)–GlyR) followed by IRES-mCherry were cloned into an AAV2 vector downstream of the Syn promoter and were also packaged as AAV serotype 5 (Janelia). For electroporation and in vitro electrophysiology in CA1 pyramidal cells, mutant channel cDNAs were cloned into a pCAG expression vector with an upstream CAG promoter and a downstream IRES-GFP sequence. The numbering scheme used for ion channel amino acid residue positions is based on the unprocessed pre-protein sequence derived from the cDNA sequences. The crystal structure of cocaine in a complex with AChBP was obtained from the Protein Data Bank (PDB: 2PGZ). Amino acid substitutions were made in PyMol for residues near cocaine in the crystal structure for the homologous residues in the human protein sequence for display purposes. The model was used to guide exploration of residues to be mutated for effects on cocaine agonism. ATCC CRL-1573 cells (HEK293, passages 40–49) were plated into Corning CellBIND plates (10 cm diameter), at 3 × 106 cells per plate and grown for 18–24 h to achieve about 80% confluency. Although the HEK293 cell line is frequently misidentified, we used obtained it directly from the ATCC and used these cells because the chimeric ion channels constructs are efficiently transfected and express well in this cell line. Cell lines were tested for mycoplasma contamination. To transfect cells, Opti-MEM (500 µl, Invitrogen), DNA plasmid (10 µg) and FuGene HD (30 µl, Roche) were combined, incubated for 15 min at room temperature and the mixture was then added dropwise to cells in a 10 cm2 plate. After an additional 18–20 h, the medium was aspirated and cells were rinsed (1× PBS, 5 ml), treated with 0.05% trypsin-EDTA (3.0 ml, 5 min) and neutralized with DMEM + 10%FBS (7.0 ml). Cells were pooled, centrifuged (100g, 2 min) and resuspended in DMEM + 10% FBS then counted (Beckman Coulter Vi-Cell XR cell analyzer). Cells were then plated in 96 well BIOCOAT poly-d-lysine black/clear cellware with a cell density of 6 × 104 live cells in 200 µl per well. Cells were incubated another 22–26 h (37 °C in 8% CO2) then medium aspirated, and the FLIPR membrane potential assay solution was added and incubated at room temperature for approximately 20 min. Membrane potential assay solution was prepared with a bottle of Molecular Devices membrane potential assay kit–Blue (R8034) dissolved in 200 ml of HEPES-buffered Earle's balanced salt solution (HEBSS). Drug compound plates were prepared in Nalge Nunc 96-well conical-bottom plates in HEBSS ( < 0.5% DMSO) and tested at concentration intervals from 0.1 µM to 100 µM. Amphetamine and cocaine (Sigma-Aldrich) were dissolved in saline to a 100 mM stock before making the dilution series. Nicotine was dissolved to 100 mM stock in saline from >99% liquid (Sigma-Aldrich). All of the other controlled substances were isolated in 1 mg quantities using the SpeedVac (Thermo Fisher Scientific) from Cerilliant ACS Chemical Standard solutions in methanol (Sigma-Aldrich) and dissolved in DMSO stock to 100 mM. DMSO alone was also tested to ensure that there was no response at these dilutions. Endogenous amines for testing at coca-5HT3 (Extended Data Fig. 3) were dissolved in saline to 100 mM stocks. To measure ion-channel activation, we used the Hamamatsu FDSS 6000 plate reader and liquid handling system. Compound solutions (50 µl) were simultaneously delivered from a 96-well compound plate after 10 baseline scans, which were followed by 170 additional scans. To calculate the MP EC50, the maximum response for each well was extracted, normalized to the maximum response for each compound and sigmoidal dose–response curves were calculated using software written in MATLAB (MathWorks Central: ec50.m, C. Evangelista, v.1.0, 14 January 2004) and the nlinfit function from the Statistics Toolbox. Channel mutants and agonists were further characterized in whole-cell voltage-clamp experiments performed on HEK293 cells. At passage 44–48, cells were plated at low density (1 × 103 per cm2) on poly-d-lysine-coated glass coverslips, then transfected in 10 cm2 wells using the Fugene 6 transfection reagent (Roche) with DNA plasmids containing both an ion channel cDNA (pcDNA3.1, 2 µg per well) and cDNA for GFP (pCMV, 0.1 μg per well), followed by replacement of the medium after 3 h. Recordings were made 24–72 h after transfection. Electrophysiological recordings were performed in whole-cell voltage-clamp mode, on an inverted fluorescence microscope (Olympus IX-51) using pClamp software and a Multiclamp 200B amplifier (Molecular Devices). Electrophysiological data was filtered at 4 kHz, digitized at 10 kHz using a Digidata 1400 interface (Molecular Devices), recorded on a PC (Dell) and analysed using custom MATLAB software (MathWorks). Solutions were delivered from a gravity fed perfusion system through a capillary tube placed approximately 100 mm from the recorded cell using an eight-way manifold pinch valve system (Nanion), which was controlled from the acquisition software. The duration of agonist application was based on pilot experiments to determine the time required to reach a steady-state current response for the channel. Osmolarities were within 285–305 mOsm l−1 and the pH was adjusted to 7.3–7.4 with NaOH. Hippocampal neuron cultures were prepared by modification of a previously reported method58. Hippocampi were dissected from postnatal day 0 neonatal rat pups (Charles River) in HEPES buffered Hank's solution containing 100 U ml−1 penicillin and 10 μg ml−1 streptomycin. After digestion, papain was removed and tissue was dissociated in plating medium (MEM plus 10% fetal bovine serum, 28 mM glucose, 2.4 mM NaHCO3, 100 µg ml−1 transferrin, 25 μg ml−1 insulin, 2 mM l-glutamine, 100 U ml−1 penicillin and 10 µg ml−1 streptomycin). The cell suspension was passed through a 70 µm filter and centrifuged at 90g for 7 min. The resulting cell pellet was resuspended in plating medium, counted and assayed for cell viability. pCAG::coca-GlyR-IRES-GFP or pCAG::coca-5HT3-IRES-GFP plasmids were transfected by electroporation using the Lonza Nucleofector system (P3 Primary Cell 96-well Kit, Lonza) then cells were plated onto poly-d-lysine-coated 13 mm dia #1 glass coverslips in the wells of a 24-well plate at 50,000 cells per well. Each well received 60 µl of cell suspension plus 60 µl of NbActiv4 medium (BrainBits) and was incubated at 37 C in 5% CO2 for 4 h for initial cell attachment. NbActiv4 medium (1 ml) was then added to each well and cultures kept for the duration of the experiment. Weekly medium changes were made by replacing 0.5 ml medium from each well with fresh NbActiv4. Tight seals (>4 GΩ) were made in voltage-clamp mode and, immediately after patch disruption to establish the whole-cell recording configuration, the resting membrane potential was measured by switching briefly to the ‘I = 0' mode on the Multiclamp Commander (Molecular Devices). After switching back to voltage-clamp mode, the other cell membrane properties were measured using the automated cell capacitance and membrane resistance compensation circuitry on the same device. Series resistance compensation was not applied. Membrane properties during current-clamp recordings were monitored by small current injections (−20 pA, 200 ms, 1 Hz). Thresholds for activation were estimated in current-clamp mode by injection of sequentially increasing depolarizing current steps (either 5 pA or 50 pA steps depending on whether a large shunt was observed due to chemogenetic receptor activation by cocaine). For analysis of coca-5HT3 effects on the passive membrane properties of LHb neurons, C57BL/6J mice (male, aged 4–5 weeks) were injected with AAV5-Syn::coca-5HT3-IRES-mCherry (titre, 4 × 1012 viral genomes (vg) per ml) or with a control virus AAV5-Syn::IRES-mCherry (titre: 2 × 1012 vg per ml) in the LHb (anteroposterior (AP): −1.55 mm relative to bregma; mediolateral (ML): +0.45 mm; dorsoventral (DV): −2.65 mm, −2.75 mm, −2.85 mm, 100 nl at each depth). After 2–5 weeks, brain slices were prepared. In brief, animals were deeply anaesthetized with isoflurane, decapitated and the brains removed in ice-cold sectioning solution (described below). The brains were then mounted onto a stage using Krazy Glue, and coronal brain slices (250 µm) were cut using a cooled tissue slicer (Leica 1200S). N-Methyl-d-glucamine was used as a sodium replacement. Slices were held down in a glass-bottomed chamber, perfused and viewed on the stage of an upright Olympus BX-51 fluorescence microscope using infrared gradient contrast optics. Transfected neurons were identified by their mCherry fluorescence. Electrophysiological recordings were made in whole-cell current-clamp mode, using a Multiclamp 200B amplifier (Molecular Devices) and MATLAB Ephus software for acquisition59. Data were filtered at 4 kHz, digitized at 10 kHz using the National Instruments interface hardware and recorded on a PC. HEK293 cells were transfected with 5 µg per dish of plasmids encoding coca-5HT3 or coca-GlyR (in pcDNA3.1) and collected 48 h after transfection. Cells were suspended in Tris-HCl 50 mM pH 7.4 supplemented with protease inhibitor cocktail (1:100, Sigma-Aldrich). HEK293 cells were disrupted with a 6 Polytron homogenizer (Kinematica). Homogenates were centrifuged at 48,000g (50 min, 4 °C) and washed twice in the same conditions to isolate the membrane fraction. Protein was quantified using the bicinchoninic acid method (Pierce). Membrane suspensions (50 µg of protein per ml) were incubated in 50 mM Tris-HCl (pH 7.4) containing 8 mM CaCl2, 2 nM of [3H]ASEM (26 Ci mmol−1, Novandi Chemistry) and increasing concentrations of the tested compounds during 2 h at room temperature as previously described12. Free and membrane-bound radioligands were separated by rapid filtration of 500 μl aliquots in a 96-well plate harvester (Brandel) and washed with 2 ml of ice-cold Tris-HCl buffer. Microscint-20 scintillation liquid (65 μl per well, PerkinElmer) was added to the filter plates, plates were incubated overnight at room temperature and radioactivity counts were determined in the MicroBeta2 plate counter (PerkinElmer) with an efficiency of 41%. Two-site competition curves were fitted using Prism 10 (GraphPad). Ki values were calculated using the Cheng–Prusoff equation. Rats were single housed on a reverse light cycle (lights off 07:00; lights on 19:00). Sample sizes were estimated using G*Power or based on experience from past work. Adult male Sprague–Dawley rats were bilaterally injected (0.5 µl per side) with AAV5-Syn::coca-5HT3-IRES-mCherry in the LHb (10° angle, AP relative to bregma: −3.8 mm; ML, ±1.7 mm; DV, −5.2 mm) (n = 6) or exposed to sham surgery (n = 6). Then, 4 weeks later, the rats were assessed for open-field locomotion. Each rat was placed in a clear Plexiglas box (40 cm × 40 cm × 30 cm) and activity was recorded by infrared beam brakes for 60 min (Opto-varimex ATM3, Columbus Instruments). The distance travelled was plotted as mean ± s.e.m. centimetres between the control and coca-5HT3 rats. Rats from the open-field locomotor experiments were presented with two bottles of water in their home cage to acclimatize them to the procedure for 1 day. The next day, one bottle was replaced with a 1% sucrose solution dissolved in water. After 24 h, the consumed volume was noted in ml and the position of the bottles was switched to control for potential side preference. On day 3 the consumed volume was calculated for both water and sucrose 24 h after position switch. This volume was then plotted as percentage mean ± s.e.m. Adult male Sprague–Dawley rats were bilaterally injected (0.5 µl per side) with AAV5-Syn::coca-5HT3-IRES-mCherry in the LHb (10° angle; AP relative to bregma, −3.8 mm; ML, ±1.7 mm; DV, −5.2 mm) (n = 8) or exposed to sham surgery (n = 5). Then, 4 weeks later, the rats were restricted to 18–20 g of chow per day and trained to press a lever for 20 mg sucrose pellets on an FR1 schedule (1 h per day for 7 days). Presses on this lever activated the food magazine to dispense one pellet (Dustless Precision Pellets, F07595, 20 mg, Sucrose; Bio-Serv) accompanied by a 20 s cue light and timeout. Responses during the timeout period were recorded but did not result in pellets being dispensed. Rats from the sucrose preference experiment were bilaterally injected (0.5 µl per side) with AAV5-Syn::coca-5HT3-IRES-mCherry in the LHb (10° angle; AP relative to bregma, −3.8 mm; ML, ±1.7 mm; DV, −5.2 mm) (n = 6) or exposed to sham surgery (n = 6). Then, 6 weeks later, the rats were euthanized, and brain tissue was extracted and flash-frozen in isopentane. The tissue that contained the bilateral striatum was then homogenized and analysed for dopamine levels using an enzyme-linked immunosorbent assay (ELISA; Abcam, ab285238). Dopamine levels were plotted as the mean ± s.e.m. pg per mg between control and coca-5HT3 rats. Male Long Evans rats were anaesthetized with isoflurane (1.5–2%). The rats were then placed onto a stereotaxic frame (Kopf) and injected with the AAV vector AAV5-Syn::coca-5HT3-IRES-mCherry (0.5 µl, 4.95 × 1012 vg per ml) into the LHb (AP relative to bregma, −3.8 mm; ML, ±0.7 mm; DV, −5.25 mm, at a 10° angle) (n = 11) or underwent sham surgery (n = 15). Approximately 1 week later, the rats were trained to self-administer food in daily operant sessions. For food training, rats were restricted to four large chow pellets per day and the body weight was monitored daily. Rats were tested for 10 days, and the sessions lasted for 3 h. For the first 5 days, the rats were tested using a fixed ratio 1 (FR1) schedule of reinforcement and were then switched to an FR5 schedule for the 5 subsequent days. Each lever press led to delivery of a 45 mg food pellet (maximum of 200 pellets per session) and activation of a cue light above the lever for 1 s. Each reinforced lever press was followed by a 20 s timeout during which lever responses were recorded but had no programmed consequences. After these operant sessions, the rats underwent surgery for jugular vein catheterization. The rats were anaesthetized with isoflurane (1.5–2%). Catheters made from Silastic tubing attached to a modified 22-gauge cannula (Plastics One, C313G-5up) and cemented to polypropylene mesh (Elko Filtering, 05–1000/45) were inserted into the jugular vein, and the mesh was fixed to the mid-scapular region of the rat. Rats were injected with carprofen (2.5 mg per kg, subcutaneous, Norbrook) after surgery and on the next day. Rats recovered for 6 days before self-administration procedures. Catheters were flushed daily with gentamicin (4.25 mg ml−1, Fresenius Kabi, 1002) dissolved in sterile saline. If catheter failure was suspected, catheter patency was tested with a short-acting barbiturate anaesthetic Brevital (methohexital sodium, 10 mg ml−1 in buffered saline, 0.1–0.2 ml injection volume, i.v. After approximately 1 week, the rats began daily cocaine IVSA operant training sessions. Rats were tested using an FR1 schedule of reinforcement and were then switched to an FR5 schedule. In these sessions, a response on the lever previously paired with food had no programmed consequences (that is, inactive lever). A response on the cocaine-paired lever (that is, active lever) led to the infusion of a 0.5 mg per kg dose of cocaine (maximum of 60 infusions per session) and coincided with activation of a cue light located above the active lever for 20 s (that is, timeout period). During this time, lever responses were recorded but had no programmed consequences. After cocaine training, the rats were tested for IVSA using different cocaine unit doses (0.0625, 0.125, 0.25, 0.5 and 1 mg per kg per infusion). The order of unit dose testing was randomized across rats. Rats could commit a maximum of 60 infusions when tested at the 0.5 and 1 mg per kg unit doses but no infusion limit was present at other unit doses. Statistical analyses were performed using Prism 10 (GraphPad). Male Sprague–Dawley rats were anaesthetized with isoflurane (1.5-2%) and then placed onto a stereotaxic frame (Kopf) and injected with AAV5-Syn::coca-5HT3-IRES-mCherry (0.5 µl, 4.95 × 1012 vg per ml) into the LHb (AP relative to bregma, −3.8 mm; ML, ±0.7 mm; DV, −5.25 mm, at a 10° angle) (n = 5) or underwent sham surgery (n = 9). Male Th-cre rats were anaesthetized with isoflurane (1.5–2%) and then placed onto a stereotaxic frame (Kopf) and injected with AAV1-Syn::FLEX-coca-GlyR-IRES-mCherry (0.5 µl, 6 × 1012 vg per ml) into the VTA (AP, −6.0 mm relative to bregma; ML, ±1.9 mm; DV, −8.3 mm at a 10° angle) (n = 9) or underwent sham surgery (n = 6). Approximately 3 weeks later, the rats were anaesthetized, placed inside the PET scanner (Mediso NanoScan PET/CT), injected with [18F]fallypride (~13 MBq) and then scanned using PET. [18F]fallypride reaches steady-state by around 45 min after its injection and then remains stable for at least 150 min60. We therefore defined the baseline [18F]fallypride binding from 60 to 90 min after its injection and then, to elicit an increase in extracellular dopamine, at 90 min after [18F]fallypride injection, we injected rats with cocaine (3 mg per kg, i.v.) The PET data were reconstructed and corrected for dead-time and radioactive decay and then co-registered to an magnetic resonance imaging template as previously described46,60,61. Co-registered images were analysed using one-way ANOVA and the resulting parametric images were filtered for statistically significant (P  < 0.01) clusters larger than 100 contiguous voxels. All statistical parametric mapping (SPM) analyses were performed using MATLAB R2016 (MathWorks) and SPM12 (University College London)62. Next, additional quantitative assessments of PET images were performed using the PMOD software environment (PMOD Technologies). Statistical analyses were performed using Prism 10 (GraphPad). Male Sprague–Dawley rats were anaesthetized with isoflurane (1.5–2%) and then placed onto a stereotaxic frame (Kopf) and injected with AAV5-Syn::coca-5HT3-IRES-mCherry (0.5 µl, 4.95 × 1012 vg per ml) into the LHb (AP, −3.8 mm relative to bregma; ML, ±0.7 mm; DV, −5.25 mm, at a 10° angle) (n = 5) or underwent sham surgery (n = 6). Rats were also implanted with an optic fibre (400 µm core, 0.50 NA) attached to a 2.5 mm diameter ferrule (Thorlabs) directed to the medial shell of the nucleus accumbens (AP, +1.7 mm relative to bregma; ML, ±0.9 mm; DV, −7.0 mm). To ensure that AAV9-Syn::GRABDA1m was transduced at the direct vicinity of the fibre tip, we coated the fibre with the AAV (0.4 µl, 5 × 1012 vg per ml) using a silk fibroin film before its implantation as previously described63. For fibre photometry, dopamine (GRABDA) dynamics were measured using a Tucker-Davis Technologies (TDT) system in anaesthetized rats. The dopamine and isosbestic signals were excited by 465 (~40–50 µW) and 405 nm (~20 µW) LEDs, respectively. On the testing day, rats were anaesthetized (isoflurane: induction, 5%; maintenance, 3%) and attached to a photometry patch cord (400 μm core, 0.50 NA). GRABDA signals were then collected for 5 min, followed by a saline i.p. injection, and additional GRABDA signal measurement for 20 min. at 20, 10 or 5 mg per kg, and the GRABDA signal was measured for an additional 20 min. Each rat was tested three times with cocaine doses administered in randomized order. Fibre photometry analyses were performed using scripts in MATLAB R2023b (MathWorks) based on codes from the Barker (https://github.com/djamesbarker)64 and Lerner (https://github.com/talialerner/)65 laboratories. The dopamine signal was fitted with the isosbestic signal [ΔF/F] and transformed to a z score for comparison of the dopamine signal between rats64. The GRABDA signal was normalized to values at 1 min before saline or cocaine injection and shown as the average z-scored activity per min. Fibre photometry data are represented as mean ± s.e.m. Statistical analyses were performed using Prism 10 (GraphPad). Rats were anaesthetized with isoflurane and transcardially perfused with PBS, followed by 4% PFA. Brains were stored overnight in 4% PFA at 4 °C and then transferred to PBS with 30% sucrose for 3 to 4 days at 4 °C. Brains were frozen and sectioned on a cryostat (30 μm to 50 μm; Leica) and slices were collected in PBS. On the first day, the sections were blocked with 0.1% saponin and 1% BSA in physiological saline for 30 min at room temperature and incubated with specific antibodies: rat monoclonal anti-mCherry (1:1,000 diluted in 0.01% saponin and 0.1% BSA in physiological saline, Invitrogen, M11217) or chicken polyclonal anti-GFP antibody (1:2,000 diluted in 0.01% saponin and 1% BSA in physiological saline, Abcam, ab13970) at 4 °C overnight. The slices were soaked in physiological saline (three times for 5 min at room temperature), then incubated with a secondary antibody in 0.01% saponin and 0.1% BSA dissolved in physiological saline: goat-anti rat immunoglobulin G (IgG) (H+L) cross-absorbed, Alexa Fluor 594 (1:500, Invitrogen, A11007), or goat-anti chicken immunoglobulin Y (IgY) H&L (Alexa Fluor 488) (1:1,000, Abcam, ab150173) for 2 h at room temperature (protected from light). After secondary antibody incubation, the slices were washed with physiological saline (three 5 min washes) and then mounted onto SuperFrost Plus Slides, counterstained with Fluoromount-G medium (Southern Biotech) and coverslipped. Images were taken using a Leica LMD6 microscope equipped with the Leica DFC7000T camera and processed using LasX software. Statistical analysis was performed using SigmaStat or GraphPad Prism. We used mixed, two-way or three-way ANOVA taking repeated measures into account when appropriate or independent t-tests or paired-sample t-tests, as defined in the text. Post hoc tests were performed using Holm–Šidák multiple-comparison correction. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. DNA constructs will be available at Addgene (https://www.addgene.org/Scott_Sternson/) under a materials transfer agreement. Source data are provided with this paper. Lüscher, C. The emergence of a circuit model for addiction. Koob, G. F. & Volkow, N. D. Neurobiology of addiction: a neurocircuitry analysis. Phillips, K. A., Epstein, D. H. & Preston, K. L. Psychostimulant addiction treatment. Jones, R. T. The pharmacology of cocaine smoking in humans. Volkow, N. D. & Swanson, J. M. Variables that affect the clinical use and abuse of methylphenidate in the treatment of ADHD. Sternson, S. M. & Roth, B. L. Chemogenetic tools to interrogate brain functions. Yu, J. et al. Nucleus accumbens feedforward inhibition circuit promotes cocaine self-administration. Refining deep brain stimulation to emulate optogenetic treatment of synaptic pathology. Magnus, C. J. et al. Chemical and genetic engineering of selective ion channel-ligand interactions. Magnus, C. J. et al. Ultrapotent chemogenetics for research and potential clinical applications. Grutter, T. et al. Molecular tuning of fast gating in pentameric ligand-gated ion channels. Eisele, J. L. et al. Chimaeric nicotinic-serotonergic receptor combines distinct ligand binding and channel specificities. Francis, M. M., Vazquez, R. W., Papke, R. L. & Oswald, R. E. Subtype-selective inhibition of neuronal nicotinic acetylcholine receptors by cocaine is determined by the α4 and β4 subunits. Cocaine potently blocks neuronal α3β4 nicotinic acetylcholine receptors in SH-SY5Y cells. & Taylor, P. Galanthamine and non-competitive inhibitor binding to ACh-binding protein: evidence for a binding site on non-α-subunit interfaces of heteromeric neuronal nicotinic receptors. The 5-HT3 antagonist tropisetron (ICS 205-930) is a potent and selective α7 nicotinic receptor partial agonist. & Horenstein, N. A. Molecular dissection of tropisetron, an α7 nicotinic acetylcholine receptor-selective partial agonist. Structural basis of human α7 nicotinic acetylcholine receptor activation. Zhong, W. et al. From ab initio quantum mechanics to molecular neurobiology: a cation-pi binding site in the nicotinic receptor. Beene, D. L. et al. Cation-pi interactions in ligand recognition by serotonergic (5-HT3A) and nicotinic acetylcholine receptors: the anomalous binding properties of nicotine. Grutter, T. et al. An H-bond between two residues from different loops of the acetylcholine binding site contributes to the activation mechanism of nicotinic receptors. A., Dobrin, C. V. & Roberts, D. C. Brain-cocaine concentrations determine the dose self-administered by rats on a novel behaviorally dependent dosing schedule. Parikh, V., Kozak, R., Martinez, V. & Sarter, M. Prefrontal acetylcholine release controls cue detection on multiple timescales. Griffith, C. A., Owen, L. J., Body, R., McDowell, G. & Keevil, B. G. Development of a method to measure plasma and whole blood choline by liquid chromatography tandem mass spectrometry. Horti, A. G. et al. 18F-ASEM, a radiolabeled antagonist for imaging the alpha7-nicotinic acetylcholine receptor with PET. Han, D. D. & Gu, H. H. Comparison of the monoamine transporters from human and mouse in their sensitivities to psychostimulant drugs. Kilty, J. E., Lorang, D. & Amara, S. G. Cloning and expression of a cocaine-sensitive rat dopamine transporter. & Caron, M. G. Molecular characterization of the dopamine transporter. Koob, G. F. & Le Moal, M. Addiction and the brain antireward system. Groos, D. & Helmchen, F. The lateral habenula: a hub for value-guided behavior. Cocaine drives aversive conditioning via delayed activation of dopamine-responsive habenular and midbrain pathways. Nair, S. G. et al. Effect of chemogenetic inhibition of lateral habenula neuronal activity on cocaine- and food-seeking behaviors in the rat. & Necessary, B. D. Evidence for opponent-process actions of intravenous cocaine. O'Connor, E. C., Chapman, K., Butler, P. & Mead, A. N. The predictive validity of the rat self-administration model for abuse liability. Carter, L. P. & Griffiths, R. R. Principles of laboratory assessment of drug abuse liability and implications for clinical development. Mello, N. K. & Negus, S. S. Preclinical evaluation of pharmacotherapies for treatment of cocaine and opioid abuse using drug self-administration procedures. Brinschwitz, K. et al. Glutamatergic axons from the lateral habenula mainly terminate on GABAergic neurons of the ventral midbrain. & Holland, P. C. The rostromedial tegmental nucleus (RMTg), a GABAergic afferent to midbrain dopamine neurons, encodes aversive stimuli and inhibits motor responses. Ji, H. & Shepard, P. D. Lateral habenula stimulation inhibits rat midbrain dopamine neurons through a GABAA receptor-mediated mechanism. Differential tonic influence of lateral habenula on prefrontal cortex and nucleus accumbens dopamine release. Evidence for, and nature of, the tonic inhibitory influence of habenulointerpeduncular pathways upon cerebral dopaminergic transmission in the rat. Bonaventura, J. et al. Pharmacological and behavioral divergence of ketamine enantiomers: implications for abuse liability. Cocaine cue-induced dopamine release in amygdala and hippocampus: a high-resolution PET [18F]fallypride Study in cocaine dependent participants. Sun, F. et al. Next-generation GRAB sensors for monitoring dopaminergic activity in vivo. Liu, X. et al. Epac2 in midbrain dopamine neurons contributes to cocaine reinforcement via enhancement of dopamine release. Matsumoto, M. & Hikosaka, O. Lateral habenula as a source of negative reward signals in dopamine neurons. Stamatakis, A. M. et al. Lateral hypothalamic area glutamatergic neurons and their projections to the lateral habenula regulate feeding and reward. Stamatakis, A. M. & Stuber, G. D. Activation of lateral habenula inputs to the ventral midbrain promotes behavioral avoidance. Yang, Y. et al. Ketamine blocks bursting in the lateral habenula to rapidly relieve depression. Beierholm, U. et al. Dopamine modulates reward-related vigor. Panigrahi, B. et al. Dopamine is required for the neural representation and control of movement vigor. A. et al. Mesolimbic dopamine signals the value of work. Atasoy, D., Aponte, Y., Su, H. H. & Sternson, S. M. A FLEX switch targets channelrhodopsin-2 to multiple cell types for imaging and long-range circuit mapping. Isolation and culture of adult rat hippocampal neurons. A. et al. Ephus: multipurpose data acquisition software for neuroscience experiments. Mukherjee, J. et al. Dopamine D3 receptor binding of 18F-fallypride: evaluation using in vitro and in vivo PET imaging studies. Ceccarini, J. et al. Optimized in vivo detection of dopamine release using 18F-fallypride PET. Expression of the excitatory opsin ChRERα can be traced longitudinally in rat and nonhuman primate brains with PET imaging. Silk fibroin films facilitate single-step targeted expression of optogenetic proteins. Bruno, C. A. et al. pMAT: An open-source software suite for the analysis of fiber photometry data. Sherathiya, V. N., Schaid, M. D., Seiler, J. L., Lopez, G. C. & Lerner, T. N. GuPPy, a Python toolbox for the analysis of fiber photometry data. Solis, O. Gomez et al/F-Phot-Analysis: v1.0.0 initial release – F-Phot-Analysis. We thank the staff at the Janelia Research Campus core facility staff (M. Ramirez performed mutagenesis and subcloning for this research) for support; M. Hiramoto for performing some electrophysiology characterization; and the staff at the National Institute on Drug Abuse Translational & Analytical Core (S. Jackson and L. Kryszak) and the National Institute on Drug Abuse Genetic Engineering and Viral Vector Core (C. Richie) for support. This research was supported by HHMI (S.M.S. ); and National Institute on Drug Abuse Intramural Research Program ZIA DA000069 (M.M.). HHMI laboratory heads have previously granted a non-exclusive CC BY 4.0 license to the public and a sublicensable license to HHMI in their research articles. Pursuant to those licenses, the author-accepted manuscript of this Article can be made freely available under a CC BY 4.0 license immediately on publication. These authors contributed equally: Juan L. Gomez, Christopher J. Magnus Biobehavioral Imaging and Molecular Neuropsychopharmacology Section, National Institute on Drug Abuse Intramural Research Program, Baltimore, MD, USA Juan L. Gomez, Jordi Bonaventura, Oscar Solis, Fallon P. Curry, Marjorie R. Levinstein, Reece C. Budinich, Meghan L. Carlton, Emilya N. Ventriglia, Sherry Lam, Ingrid Schoenborn, William Dunne & Michael Michaelides Howard Hughes Medical Institute; Janelia Research Campus, Ashburn, VA, USA Christopher J. Magnus, Le Wang & Scott M. Sternson Howard Hughes Medical Institute, University of California San Diego, La Jolla, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Michael Michaelides or Scott M. Sternson. have a pending patent application on this technology (US patent application 20240002463-A1) as well as additional issued patents on related chemogenetic technologies. is a consultant for Kriya Therapeutics, which is focused on therapeutic applications of chemogenetics. is a principal investigator on a cooperative research and development agreement between NIDA and Kriya Therapeutics. The other authors declare no competing interests. Nature thanks Brandon Henderson, Paul Kenny and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Chimeric ion channels were created from the ligand binding domain (LBD) of the α7 nAChR that was spliced to either the IPD of 5HT3 or GlyR and produced channels for neuron activation or inhibition, respectively. Mutated chimeric channel subunits homopentamerize to form LGICs. Mutations in the LBD conferred novel cocaine agonism and reduced ACh sensitivity. b, Structural elements that correspond to the nicotine pharmacophore are highlighted in red. Acetoxymethyl ester portion of cocaine that is proposed to adversely affect cocaine agonism of the unmodified α7 nAChR is highlighted in blue. Structural relationship to tropisetron, an α7 nAChR agonist, is shown on the right. c, Cocaine binding to AChBP (modified from PDB: 2pgz). Two views of cocaine (yellow) bound to AChBP at the interface of two protomer subunits (cyan & pink) showing 4 amino acid side chains in grey that are from homologous α7 nAChR human sequence and are mutated in coca-5HT3. Homologous α7 nAChR pre-protein numbering is shown. a, Dose response of choline on coca-5HT3 chimeric ion channel (n = 4 separate dose responses). b, Dose response for cocaine and two cocaine metabolites on coca-5HT3 (n = 1 dose response each). c, Dose response for cocaine and other amine-containing addictive drugs on coca-5HT3 (n = 1 dose response each). Data is shown as mean ± sem. a, Endogenous amines (n = 1 dose response per amine). b, Clinically used drugs with structural similarity to cocaine and two common anaesthetics (cocaine, n = 5 separate dose responses; benztropine, n = 3 separate dose responses; methylatropine, n = 3 separate dose responses; all other drugs, n = 1 dose response). Data is shown as mean ± sem. Dose-response curves were fitted using a two-site binding model. Data are shown as mean (n = 3 replicate curves for ASEM and n = 2 replicate curves for cocaine). LHb neurons from mice expressing coca-5HT3-IRES-mCherry (n = 16 cells) or negative control mCherry (n = 25 cells) showing a, Capacitance (Shapiro Wilk normality test P < 0.05, two-sided Mann-Whitney Rank Sum Test), b, Membrane Resistance (Shapiro Wilk normality test P < 0.05, two-sided Mann-Whitney Rank Sum Test), c, and Resting Membrane Potential (Shapiro Wilk normality test P = 0.97, two-sided t-test). Data is shown as mean ± sem, statistical tests are two-sided. b, Total sucrose intake (g/kg) (two-way RM ANOVA; transgene main effect (F(1, 11) = 0.18, p = 0.68), session main effect (F(5, 55) = 13.41, p < 0.001), transgene × session interaction (F(5, 55) = 0.61, p = 0.69)), (c) active lever presses (two-way RM ANOVA; transgene main effect (F(1, 11) = 0.24, p = 0.63), session main effect (F(5, 55) = 10.37, p < 0.001), transgene × session interaction (F(5, 55) = 0.72, p = 0.61)), and (d) number of sucrose pellets delivered during operant experiments in rats with coca-5HT3 expression in LHb (n = 5 rats) and in controls (n = 8 rats). All experiments are in the absence of cocaine. e,f, Open-field locomotor activity in rats with coca-5HT3 expression in LHb (n = 6 rats) and in controls (n = 6 rats) showing (e) total distance travelled over the 1 h session (two-sided unpaired t-test, t = 0.21; p = 0.84) and (f) within session time course of distance travelled. Experiment is in the absence of cocaine. Data is shown as mean ± sem. a, Time-activity curves of [18F]fallypride binding (SUV; standardized uptake value) in the ventral striatum and cerebellum for control (n = 9 rats) and coca-5HT3 expressing rats (n = 9 rats). Data is shown as mean ± sem. a, Schematic showing the stereotaxic delivery of AAV1-Syn::FLEX-Coca-GlyR-IRES-mCherry into the VTA of TH-cre rats and [18F]fallypride PET procedures. b, Coronal PET brain images co-registered to MRI showing significant cocaine-induced voxel-wise decreases in [18F]fallypride binding in the ventral striatum of control rats (n = 6 rats) and in rats with coca-GlyR in VTA dopamine neurons (n = 9 rats). c, Time-activity curves of [18F]fallypride binding (SUV; standardized uptake value) in the ventral striatum (squares) and cerebellum (triangles) calculated as the percent of baseline (60–90 min) for control (n = 6 rats, black) and coca-GlyR rats (n = 9 rats, orange) showing that cocaine selectively displaces [18F]fallypride binding (i.e., increases dopamine) in the ventral striatum of control rats. Data is shown as mean ± sem. a-d, Time course (z-score) of GRABDA response in control rats treated with saline or cocaine (a, c) (10 mg/kg: two-way RM ANOVA; treatment main effect (F(1, 5) = 32.2, p = 0.002), time main effect (F(2.6, 13.1) = 4.56, p = 0.2), treatment × time interaction (F(1.7, 8.8) = 21.45, p = 0.005, 20 mg/kg: two-way RM ANOVA; treatment main effect (F(1, 5) = 13.61, p = 0.014), time main effect (F(1.4, 7.1) = 2.3, p = 0.17), treatment × time interaction (F(1.4, 7.2) = 5.09, p = 0.04) and coca-5HT3 rats treated with saline or cocaine (b, d) (10 mg/kg: two-way RM ANOVA; treatment main effect (F(1, 4) = 5.05, p = 0.08), time main effect (F(1.4, 5.6) = 1.12, p = 0.36), treatment × time interaction (F(1.1, 8.8) = 1.24, p = 0.33, 20 mg/kg: two-way RM ANOVA; treatment main effect (F(1, 4) = 0.41, p = 0.55), time main effect (F(2.4, 9.7) = 0.52, p = 0.63), treatment × time interaction (F(1.3, 5.4) = 0.37, p = 0.63). Data is shown as mean ± sem. a,b, Raw photometry data from a representative control (a) and LHb-coca-5HT3 rat (b) in response to saline and 5 mg/kg cocaine injections. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Cocaine chemogenetics blunts drug-seeking by synthetic physiology. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02739-9'>An abiding mystery of the French Revolution is solved — by epidemiology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 15:21:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Mariana Lenharo is a news reporter for Nature based in New York City. In 1789, rumours spread like a virus across France: gangs of bandits were attacking villages, destroying crops and terrorizing peasants, mobilized by nobles trying to suppress political unrest. Did a deliberate effort to advance revolution drive the rumours? Drawing on historical records and models developed to trace epidemics, researchers conclude that the fearmongering had rational, not emotional, roots1. Using data to objectively measure societal events “is the new paradigm for studying human dynamics”, says Walter Quattrociocchi, a computer scientist specializing in complex systems at Sapienza University of Rome, who has studied the spread of misinformation online. The study focuses on “something that we can measure”, he notes. The conspiratorial hearsay that spread across France in July and August of 1789 led peasants to invade castles and destroy land registers, which, in some places, were crucial for the local lord's claim to authority over estates. Parent, whose work involves applying complex system analysis to the study of history, met his co-authors, physicist Stefano Zapperi and pathologist Caterina La Porta at the University of Milan, Italy, at a conference. They also used historical road maps to identify the possible dissemination routes. “We treated it as a transmission network for an epidemic,” Zapperi says. Listen to the related podcast, ‘Viral spread: how rumours surged in revolutionary France'. Epidemiology models explain rumour spreading during France's Great Fear of 1789 Immigrant–native pay gap driven by lack of access to high-paying jobs Epidemiology models explain rumour spreading during France's Great Fear of 1789 Scientific meetings debate the effect of climate change on future food production David Nabarro obituary: global-health leader who fought malnutrition, malaria, Ebola and COVID-19 IOP is China's premier research institution in condensed matter physics and related fields. SLS invites applications for multiple tenure-track/tenured faculty positions at all academic ranks. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09388-y'>Thioester-mediated RNA aminoacylation and peptidyl-RNA synthesis in water</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 15:11:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 644, pages 933–944 (2025)Cite this article To orchestrate ribosomal peptide synthesis, transfer RNAs (tRNAs) must be aminoacylated, with activated amino acids, at their 2′,3′-diol moiety1,2, and so the selective aminoacylation of RNA in water is a key challenge that must be resolved to explain the origin of protein biosynthesis. So far, there have been no chemical methods to effectively and selectively aminoacylate RNA-2′,3′-diols with the breadth of proteinogenic amino acids in water3,4,5. Here we demonstrate that (biological) aminoacyl-thiols (1) react selectively with RNA diols over amine nucleophiles, promoting aminoacylation over adventitious (non-coded) peptide bond formation. Broad side-chain scope is demonstrated, including Ala, Arg, Asp, Glu, Gln, Gly, His, Leu, Lys, Met, Phe, Pro, Ser and Val, and Arg aminoacylation is enhanced by unprecedented side-chain nucleophilic catalysis. Duplex formation directs chemoselective 2′,3′-aminoacylation of RNA. We demonstrate that prebiotic nitriles, N-carboxyanhydrides and amino acid anhydrides, as well as biological aminoacyl-adenylates, all react with thiols (including coenzymes A and M) to selectively yield aminoacyl-thiols (1) in water. Finally, we demonstrate that the switch from thioester to thioacid activation inverts diol/amine selectivity, promoting peptide synthesis in excellent yield. Two-step, one-pot, chemically controlled formation of peptidyl-RNA is observed in water at neutral pH. Our results indicate an important role for thiol cofactors in RNA aminoacylation before the evolution of proteinaceous synthetase enzymes. At life's functional core, there is a complex and inseparable interplay between nucleic acids and proteins, but the origin of this relationship remains a mystery. Although nucleic acids store, replicate and transmit sequence information through their inherent structural capacity for molecular (self-)recognition6,7,8,9, proteins are the molecular, structural and catalytic workhorses of life. Unlike nucleic acids, peptides do not innately replicate in a sequence-specific manner10,11, so life must control and transmit the peptide sequences that are essential to its survival through nucleic acid encoding12. Understanding how nucleotide-controlled peptide biosynthesis could have first emerged is a notable gap in our understanding of life but is a formidable challenge owing to the immense complexity and antiquity of protein synthesis1. However, all proteins are built through the iteration of a universally conserved two-stage process, ribosomal peptide synthesis (RPS). In the first stage of RPS, transfer RNAs (tRNAs) are aminoacylated, loading an activated amino acid onto the 2′,3′-diol moiety of the tRNA, before these subsequently undergo ribosome-catalysed peptide synthesis (Fig. Notably, throughout RPS, the activated amino acids are orchestrated and controlled through their covalent attachment to RNA and so the selective formation of RNA esters underpins, and must predate, RPS. a, Schematic showing the two-step mechanism of RPS. RNA aminoacylation is controlled by aminoacyl-tRNA synthetase enzymes, which are themselves produced by RPS. This causal paradox obfuscates the origins of protein biosynthesis. The formation of aminoacyl-thiol (1) blocks the spontaneous formation of peptides, favouring the non-enzymatic synthesis of aminoacyl-RNAs that are required for RPS. R = amino acid side chain. Chemical structures are drawn neutral for clarity. (2) Selective and high-yielding non-enzymatic 2′,3′-aminoacylation of adenosine 16A (2 mM) with aminoacyl-thiol \({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) (120 mM; superscript e = ethanethiyl moiety) in water at pH 6.5 and room temperature. Despite the large excess of 1, and its free amine, peptide synthesis is not observed. Thioester-activated amino acids react selectively with RNA to yield aminoacyl-RNA at neutral pH. RNA aminoacylation is controlled in biology by a unique family of synthetase enzymes2. However, despite its obvious importance, the discovery of a non-enzymatic synthesis of aminoacyl-RNA is a long-standing and hereto unresolved challenge3,4,5. RNA amino acid esters are not known to form selectively in water, posing the question of their provenance, and so we set out to develop an effective and selective RNA aminoacylation in water. Previous studies of nucleoside aminoacylation have focused on aminoacyl phosphates 2, aminoacyl imidazole 3 and N-carboxyanhydrides (NCAs; 4)13,14,15,16 but all of these electrophiles are highly unstable in water and difficult to prepare without substantial hydrolysis. Even more problematically, these highly activated amino acids do not react selectively with the 2′,3′-diol of RNA, as is required for tRNA aminoacylation, and so uncontrolled background amide formation is a substantial problem15. It is important that the formation of aminoacyl-RNAs (exploited in RPS) must occur without uncontrolled amino acid polymerization: this is a prerequisite for controlled RPS, which must outcompete random peptide synthesis to rise above the noise threshold17. Therefore, despite decades of research, there remains no effective means of chemically aminoacylating RNA strands terminating in a vicinal diol, with hydrolysis and polymerization occurring more readily than RNA aminoacylation5. Considering the poor selectivity exhibited by highly reactive electrophiles, we suspected that the solution to selective RNA aminoacylation would be found through a milder activation strategy and specifically reasoned that biological thioesters may be the ideal mode of activation to achieve selective RNA aminoacylation in water. Thioesters are a crucial component of central metabolism, driving a wide range of anabolic pathways, including fatty acid, polyketide and, notably, non-ribosomal peptide syntheses18. Thioesters, like RNA aminoacylation, have ancient roots in biochemistry and both predate the last universal common ancestor of all life on Earth. The role of thioesters in central metabolism suggests that thioester-based metabolism (a ‘thioester world') may have laid the foundations for extant life18,19 and so we considered the role that thioesters might play in RNA aminoacylation. The aminoacylation of RNA is just one class of biological acyl transfer that is catalysed by synthetase enzymes. However, most biological acyl-transfer reactions exploit acyl-thiols and pantetheine 5a as a cofactor20. 5a, the functional fragment of coenzyme A, is universally conserved across all organisms and is life's most prominent acyl-transfer cofactor. Although it has been proposed that aminoacyl-thiols (1) may hold the key to the early evolution of peptide synthesis18, it has always previously been predicted that 1 would directly yield peptides. However, this ignores the essential and ancient role of RNA aminoacylation in protein synthesis, as well as the inherent aqueous reactivity of thioesters. Here we demonstrate that (biological) aminoacyl-thiols (1) chemoselectively aminoacylate RNA at neutral pH in water. This non-enzymatic RNA aminoacylation highlights an important chemical distinction between the two separate stages of protein synthesis, that is, between RNA aminoacylation and peptide synthesis, and demonstrates how to chemically control these two steps under the same reaction conditions in water. Aminoacyl-thiol (1) and (biological) thioester activation suppress adventitious peptide synthesis while delivering the activation required to achieve the first stage of RPS (Fig. 1), conceptually uniting biological thioester energy with functional RNAs to allow chemoselective diol acylation in water. We also demonstrate that thioacid activation provides the reactivity required to orthogonally achieve the second step of RPS, that is, peptidyl-RNA synthesis, in water under the same reaction conditions. These two closely related modes of activation, thioester and thioacid activation, can direct the two chemical steps required for RPS under the same conditions in water and proceed without evolved catalysts, intramolecular or Watson–Crick templated reactivity and without purification of intermediate aminoacyl-RNAs. Together, these two reactions provide a mechanism to realize high-yielding, chemoselective, one-pot synthesis of peptidyl-RNA in water at neutral pH. To begin our investigation, it was first essential to determine whether ambiphilic aminoacyl-thiol 1, which is an electrophilically activated thioester and a nucleophilic amine (pKaH 7.5), would spontaneously polymerize21,22. However, 1 was found to be very stable in water (Fig. Peptide formation from 1 was ineffective at all pHs and the main pathway was the slow hydrolysis of 1. For example, alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) reacts sluggishly at pH 7.0 to afford a very low yield of diketopiperazine (6AlaAla, <3%), alongside alanine (7Ala, 48%) after 5 days (Fig. More 6 was observed at higher pH but the maximum yield (21%) was observed at pH 8.0 (Supplementary Fig. 45) and hydrolysis dominated at pH 9.0–10.0 (Supplementary Fig. Notably, we found that peptide formation was further suppressed at lower pHs (Fig. a, Alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (100 mM) reacted ineffectually with amine, imidazole (18) and phosphate nucleophiles (red) at neutral pH but in excellent-to-quantitative yield with sulfur nucleophiles (blue). Maximum per cent conversion with specified nucleophile (150 mM)† at pH 7.0 and room temperature is given. b, The stability of l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) at room temperature in 2-(N-morpholino)ethanesulfonic acid (MES; 1 M, pH 5.0–6.5) or 3-(N-morpholino)propanesulfonic acid (MOPS; 1 M, pH 8.0) buffer. c, (1) Selective aminoacylation of ribonucleosides (16). (2) 2′,3′-Aminoacyl nucleoside 17 yield observed on incubating alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) with adenosine 16A (20 mM) at room temperature in MES (1 M, pH 5.0–6.5) or MOPS (1 M, pH 8.0) buffer. 70 for aminoacylation yields at pH 5.0–10.0 over 100 h. (3) Alcohol-aminoacylation yield for specified nucleosides (20 mM) with l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) at room temperature and pH 6.5 after 18 h. *2 mM 16G; 2 mM 21G. 1 for an expanded scope of nucleosides. d, (1) Nucleoside aminoacylation at low initial concentration. (2) Yield of uridine 16U (2 mM) aminoacylation observed with l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (5 mM or 10 mM) when frozen, incubated at −7 °C for 5 days and then thawed at room temperature (r.t.). The yield of 17Ala is substantially augmented by freezing and comparable with aminoacylation of 16U (20 mM) and l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (600 mM) at pH 6.5 and room temperature after 24 h. Surprised by the stability of ambiphilic thioester 1, we next investigated its reaction with glycine nitrile 8Gly (pKaH 5.4), the most nucleophilic α-aminonitrile, which would be predominately free-base amine at pH 7.0. 8Gly also reacted ineffectively with l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\), yielding dipeptide nitrile 9AlaGly in very low (8%) yield. Moreover, other proteinogenic aminonitriles (for example, 8Leu), amino acids (7Ala) and amino acid amides (10Ala), did not react with thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) at all (Fig. This thioester reactivity strongly contrasts with the near-quantitative ligations observed with activated peptide thioacids (for example, Ac-(Aaa)n-SH; 11) under the same, neutral pH, conditions23,24, despite the close structural and chemical relationship between 1 and 11. This demonstrated a marked difference in the reactivity of thioesters and underlined an important principle: highly activated electrophiles react effectively with amines in water to yield peptides24 but weakly activated thioesters do not21. We therefore considered that coordination of these two types of activation may hold the key to differentiating RNA aminoacylation and peptide bond formation in water under the same conditions and therefore the means to realizing effective non-enzymatic synthesis of peptidyl-RNA in water. Surprised by the poor reactivity of aminoacyl-thiol 1 with amines, we next investigated their reaction with sulfide nucleophiles. In contrast to amines, hydrogen sulfide (H2S) reacted with 1 to afford thioacids (12) in excellent-to-quantitative yield (Fig. Cysteine (7Cys) also reacted with thioester 1 to yield dipeptides (13) in near-quantitative yield, by sulfide exchange and intramolecular S-to-N acyl shift. Similarly, thiopropanol 5f yielded ester 14 in good yield and this facile intramolecular esterification, by means of S-to-O acyl shift, led us to consider how aminoacyl-thiols (1) could selectively aminoacylate nucleosides at neutral pH. We next incubated 5′-mercapto-adenosine 15 with thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) and observed 2′,3′-diol aminoacylation (54%), which we initially ascribed to intramolecular S-to-O acyl transfer. Notably, however, addition of cytidine (16C) as a competitor nucleophile led to intermolecular aminoacylation of 16C (Extended Data Fig. Ribonucleoside-2′,3′-aminoacylation with thioester 1 was observed despite the large (about 3,000-fold) excess of water and tenfold excess of amine relative to the RNA 2′,3′-diol (Fig. This efficient and selective nucleoside aminoacylation stands in sharp contrast to the reaction of acyl-imidazoles (3), reported by Weber and Orgel, which form dipeptides more efficiently than aminoacyl-RNAs4, and previous reports in which evolved catalysts or intramolecular tethering were required to enable RNA aminoacylation in water25,26, demonstrating that biological thioesters are predisposed to aminoacylate RNA while suppressing peptide bond formation. To further investigate the selectivity of aminoacyl transfer, we next challenged thioester 1 to react with phosphate and imidazole (18), which are common nucleophiles for activated carboxylates in water. We found that neither reacted with thioester 1 (Fig. Perhaps even more surprisingly, glycerol (19), which contains three alcohols and a vicinal diol, such as ribonucleosides (16), did not react with 1. This demonstrated an important but very fine balance between nucleophile and electrophile that enables ribonucleosides (16) to react selectively with aminoacyl-thiols (1). Nucleoside aminoacylation was most rapid at pH 8.0–9.0 but was most 2′,3′-diol selective and highest yielding at pH 6.0–7.0. The aminoacyl nucleoside products (17) were also most stable at lower pH and so, overall, ribonucleoside aminoacylation was most effective at pH 6.5 (Fig. Aminoacylation was observed across a wide range of nucleoside concentrations (2–100 mM) and aminoacyl-thiol concentrations (20–600 mM) at room temperature. As expected, a higher (steady-state) concentration of aminoacyl nucleoside (17) was observed when fuelled by a higher concentration of aminoacyl-thiol 1. However, notably, on freezing, the initial conditions could be extremely dilute (1–10 mM) and result in a very high yield of aminoacyl-RNA (17). 2d and Supplementary Table 12), despite the initial and final very low concentration of the solution. It is of note that eutectic phase conditions also promote ribozyme activity27, warranting further investigation of eutectic phase aminoacyl-thiol reactions. Aminoacylation was kinetically selective for the furanosyl-2′,3′-diol, probably because of its increased acidity with respect to other alcohols. To explore this selectivity further, we investigated the reaction of l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) with a range of nucleosides (Fig. As expected, aminoacylation was nearly completely suppressed for 2′,3′-dideoxyadenosine 20A, which only has a primary 5′-alcohol and no diol. Canonical 2′-deoxynucleosides (DNA; 21) also furnished very low yields of aminoacylation. However, non-canonical 3′-deoxyadenosine 22A and 5′-deoxyuridine 23U both underwent substantial aminoacylation. This initially suggested that RNA may react through its 2′-alcohol, followed by equilibration of the 2′-ester and the 3′-ester. However, further investigation revealed that thioester 1 reacted effectively and selectively with 3′-O-methoxy-adenosine 24A and 2′-O-methoxy-adenosine 25A at their secondary alcohols, which indicated that the 2′-alcohol and 3′-alcohol of RNA are comparably reactive. Together, these results demonstrate a strong kinetic preference for reaction at the secondary alcohol of ribonucleosides, rather than at the sterically more accessible primary alcohol, in sharp contrast to the well-known selectivity observed in organic synthesis. Next, we investigated the effect of nucleoside phosphorylation on aminoacylation. Nucleoside-5′-monophosphates (NMP; 26), nucleoside-5′-diphosphates (NDP; 27) and nucleotide-5′-triphosphates (NTP; 28) were all much less reactive than nucleosides (16), whereas overall levels of thioester turnover (to amino acid) remained similar. This indicated that nucleoside phosphate (NXP) aminoacylation was slower, rather than aminoacyl-ester hydrolysis being faster. We suspected that NXP aminoacylation was inhibited by the elevated diol pKa following phosphorylation. In line with this reasoning, negligible aminoacylation of adenosine-3′-phosphate (29A) and adenosine-2′-phosphate (30A) was observed; consistent with the high pKa of their 2′-hydroxyl and 3′-hydroxyl relative to a 2′,3′-diol. Notably, this phosphate-suppressed reactivity was found to be specific to phosphate monoesters and polyphosphates. Therefore, adenosine-3′,5′-cyclic phosphate (31A) underwent highly effective 2′-aminoacylation, whereas 5′-aminoacylation of adenosine-2′,3′-cyclic phosphate 32A remained ineffective, further demonstrating the pronounced kinetic selectivity for reaction at the secondary over the primary alcohol in nucleotides (Extended Data Fig. The suppressed aminoacylation of 2′,3′-diols in NXPs (but not phosphodiesters) could play an important role in directing aminoacylation to oligonucleotide-2′,3′-diols, which are necessarily 5′-phosphodiesters, under conditions in which phosphorylated NXP monomers are concurrently available for RNA synthesis. The difference between phospho-monoester and phospho-diester aminoacylation would (correctly) direct aminoacylation within a network of reactions (that is, in a cell) that can simultaneously synthesize RNA and exploit RNA aminoacylation in peptide synthesis. Control over the site of aminoacylation in oligonucleotides is essential for optimal positioning of aminoacyl esters within RNA. Life specifically aminoacylates RNA at the 3′-terminus in the first step of protein synthesis (Fig. 1a) and therefore we next turned our attention to nucleic acid aminoacylation, for which internal 2′-aminoacylation introduced a further challenge to selectivity28. We first investigated the aminoacylation of DNA oligonucleotides (ON1–ON3). Minimal DNA aminoacylation was observed, as a single-gel band, consistent with a single site of aminoacylation at the DNA 3′-terminus (ON2; Fig. As expected, aminoacylation was blocked by a terminal 2′,3′-dideoxynucleotide (ON1; Fig. 3c (1)) but strongly (sixfold) enhanced by a terminal 2′,3′-diol (ON3; Fig. As well as enhanced yield with the 2′,3′-diol terminus, a second minor bis-aminoacylation product was observed, demonstrating the greatly enhanced reactivity of the (RNA) 2′,3′-diol over the (DNA) 3′-hydroxyl in oligonucleotides. a, Aminoacylation of single-stranded (ss) RNA at terminal 2′,3′-diol and internal 2′-alcohols. b, Selective 2′,3′-aminoacylation of duplex (ds) RNA by aminoacyl-thiols (1a, -SR1 = pantetheinyl) in water. We next investigated (single-strand) ss-RNA (ON4 and ON5). Extensive and non-specific ss-RNA aminoacylation was observed, with aminoacylation at several hydroxyls throughout the RNA oligomer (ON5; Fig. Multiple-site ss-RNA aminoacylation was not blocked by a 3′-terminal deoxynucleotide (ON4; Fig. 3c (4)) and little or no selectivity was observed for aminoacylation at the 2′,3′-diol of ss-RNAs, even in a short nucleic acid. Longer RNAs have more internal 2′-hydroxyls and further diminish the selectivity of aminoacylation. Concerned by the poor selectivity of ss-RNA, and considering that nucleic acid replication must furnish nucleic acid duplex, we next investigated (double-stranded) ds-RNA. We were pleased to find that the formation of Watson–Crick duplex resurrected selective aminoacylation. The yield of 2′,3′-diol aminoacylation was equal in ds-RNA (ON5) and ss-oligomer ON3, a chimeric oligonucleotide with a single RNA at the 3′-terminus (Fig. 3c), providing a clear demonstration that duplex formation did not inhibit aminoacylation of the RNA 2′,3′-diol. However, at the same time, duplex formation strongly inhibits aminoacylation of internal 2′-hydroxyls29 (Fig. 3c (6)) and therefore aminoacylation is highly selective for the terminal 2′,3′-diol in ds-RNA. Surprised by this simple and inherent mechanism by which RNA duplex directs selective aminoacylation, we next investigated a more congested nicked duplex, with an adjacent downstream 5′-phosphorylated oligomer. The presence of this downstream oligomer did not block diol aminoacylation (Fig. 3c), demonstrating that selective 2′,3′-aminoacylation can occur despite a proximal 5′-phosphate. These results indicate that RNA's ability to form duplexes may be essential to directing the site of RNA aminoacylation, as well as ordering the alignment of tRNAs on mRNA templates during protein synthesis. Given the marked efficiency of 3′-selective ds-RNA aminoacylation, we next investigated whether changing the aminoacyl-thiol side chain would enable a broad scope of chemical aminoacylation of RNAs in water with proteinogenic amino acids. Crucially, thioester-mediated ribonucleoside aminoacylation was extremely efficient relative to the poor reactivity shown with amines (Fig. This led us to consider that side-chain compatibility, which has not previously been examined in other aminoacylation studies, would be high. Notably, no serine acylation was observed despite a tenfold excess of the (Ser) α,β-amino-alcohol over nucleoside-diol, further demonstrating high selectivity for the 2′,3′-diol. ; n = 3) for the reaction of: black, 16U (20 mM) in water at pH 6.5 and room temperature after 24 h; blue, ss-ON5 (0.5 µM) in MES buffer (400 mM, pH 6.5) and KCl (1 M) with l-\({{\bf{1}}}_{{\bf{A}}{\bf{a}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) at room temperature after 16 h; and green, ds-ON5/ON7 (0.5 µM) in MES buffer (400 mM, pH 6.5) and KCl (1 M) with l-\({{\bf{1}}}_{{\bf{A}}{\bf{a}}{\bf{a}}}^{{\bf{e}}}\)(200 mM) at room temperature after 16 h. †l-\({{\bf{1}}}_{{\bf{P}}{\bf{h}}{\bf{e}}}^{{\bf{e}}}\) (100 mM). *l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) (60 mM). 5c for l-\({{\bf{1}}}_{{\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{e}}}\) and Supplementary Figs. b, The reaction of arginine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) was enhanced by side-chain catalysis. 1H NMR spectra show: (1) adenosine 16A (2 mM); (2) the reaction of 16A (2 mM) with l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) (120 mM) at pH 6.5 and room temperature after 6 h, which formed arginyl-adenosine \({{\bf{17}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{A}}}\) (76%) in excellent yield. 1b (2) for l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) reaction time course. The Arg side chain (pKaH 13.8) is drawn neutral but will be fully protonated at pH 6.0–7.0. c, Minizyme-enhanced aminoacylation of oligonucleotide 5′-FAM-UGAGAGAGCCU (ON10). ; n = 3) of ON10 as a blunt end duplex (+5′-AGGCUCUCUCA (ON11) 0.5 µM), underhang (+5′-GGCUCUCUCA (ON12) 0.5 µM) and ‘minizyme' complex (+5′-GUGGCUCUCUCA (ON13) 0.5 µM) with l-\({{\bf{1}}}_{{\bf{P}}{\bf{h}}{\bf{e}}}^{{\bf{e}}}\) or l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (100 mM) in MES buffer (200 mM, pH 6.5) and KCl (500 mM) after 16 h at room temperature. Notably, arginine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) was an extremely effective aminoacylating agent, furnishing arginyl-adenosine \({{\bf{17}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{A}}}\) in up to 76% yield (Fig. Quantitative ss-RNA and 64% 3′-selective ds-RNA arginylation was observed following reaction of l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\) with oligomeric RNAs. Further analysis indicated that cyclic arginine 33 was formed in situ and 33 aminoacylated RNA even more effectively than thioester 1 (Supplementary Table 13), demonstrating pronounced intramolecular nucleophilic catalysis. To our knowledge, this mode of nucleophilic Arg catalysis has not previously been reported in enzyme catalysis but it is highly likely that acyl-transfer enzymes would have exploited this catalytic strategy. The enhanced aminoacylation of RNA with arginine, as well as the side-chain-specific affinity of Arg-peptides for RNAs30,31,32, warrant further investigation of the structure and function of arginylated RNAs. Given this unexpected cyclization with Arg, a side chain normally considered non-nucleophilic, we might expect lysine thioester l-\({{\bf{1}}}_{{\bf{L}}{\bf{y}}{\bf{s}}}^{{\bf{e}}}\) to cyclize rather than aminoacylate. However, aminoacylation with l-\({{\bf{1}}}_{{\bf{L}}{\bf{y}}{\bf{s}}}^{{\bf{e}}}\) proceeded just as effectively as aminoacylation with alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{r}}{\bf{g}}}^{{\bf{e}}}\). Overall, excellent proteinogenic side-chain tolerance was observed during nucleoside aminoacylation, suggesting that aminoacyl-thiol (1) is a privileged substrate for the aminoacylation of RNA in water. Comparable aminoacylation yields were observed for ds-RNAs terminating in all four canonical ribonucleotides (A, U, C and G; ON5, ON14–ON17) with the same thioester (Supplementary Fig. 112), indicating the propensity of thioester-mediated aminoacylation to charge a variety of proteinogenic amino acids onto ds-RNA in a sequence-independent manner. However, notably, 2′,3′-aminoacylation was found to be more effective (about 1.3–1.5-fold) with a template overhang at the 3′-terminus, relative to a nicked duplex, suggesting a potential catalytic role for the template strand. Indeed, we found that ribozymes augmented thioester-mediated side-chain-selective aminoacylation. Specifically, an RNA duplex (ON10 + ON13) with a mispaired 3′-U/5′-U and 5′-G overhang15, amplified 3′-aminoacylation yields (Fig. 4c) with respect to ds-RNA (ON10 + ON11) and enabled substantial aminoacylation with phenylalanine thioester (l-\({{\bf{1}}}_{{\bf{P}}{\bf{h}}{\bf{e}}}^{{\bf{e}}}\)), even at extremely low (2 mM) concentrations (Supplementary Fig. These results indicate that enhancing the 2′,3′-diol-selective reactivity of specific amino thioesters (1) with specific RNA catalysts has the potential to direct amino acid sequence pairings, paving the way for a (primitive) coding system. Ideally, RNA aminoacylation would be intrinsically directed to the (controlled) products of RNA biosynthesis (that is, RNAs with 2′,3′-diols) and adventitious aminoacylation of RNA degradation products would be inhibited. So we next considered the effect that RNA degradation would have on aminoacylation. RNA hydrolysis specifically furnishes RNAs that terminate in 2′-phosphate or 3′-phosphate, rather than the 5′-phosphates that are produced by RNA biosynthesis. Therefore we next incubated ON5-3′p with thioester 1 and were pleased to observe that the 3′-phosphate very strongly suppressed ds-oligonucleotide aminoacylation (Extended Data Fig. Notably, this inhibition offers an innate mechanism for RNA hydrolysis to prevent adventitious aminoacylation, which would be highly beneficial within a system (or cell) that is continuously generating and degrading (catalytic) RNAs. In a preliminary investigation of the diastereoselectivity of thioester-mediated 2′,3′-aminoacylation, oligonucleotides terminating in pyrimidines were observed to react with negligible selectivity (ON15, ON16: <0.06 d.e. This selectivity was not affected by duplex formation and FAM-(dN)9N chimeric nucleic acids (for example, ON2) underwent aminoacylation with similar diastereoselectivity (<0.2 d.e.). This selectivity seems to be governed at the monomeric level, with comparable selectivity observed during the aminoacylation of nucleosides; pyrimidines 16C/16U = <0.05 d.e., purines 16A/16G = 0.3 d.e. Given the transient nature of aminoacyl esters, as only one intermediate of nucleic-acid-mediated peptide synthesis, the impact of this selectivity on the overall multistep scheme cannot yet be known, but it is inevitable that (several) subsequent steps will affect diastereoselectivity33 and even evolved synthetase enzymes may not achieve strictly L-selective aminoacylation34,35, despite the presence of D-amino acids in the cell. Encoding amino acids during aminoacylation will inevitably affect stereochemical selection but we believe that stereochemically agnostic tRNAs are likely to be important to enable the comprehensive side-chain compatibility necessary for uniformly effective synthesis of diverse peptides, with sterically diverse amino acid side chains, which are encoded only by nucleic acid information (rather than being intrinsically favoured by peptide structures) during RPS. Therefore, we have not attempted to augment or reverse the innate (lack of) diastereoselectivity observed here but catalytic aminoacylation would pave the way to address amino acid coding and selection in the future, for which amino acid chirality can be set during the synthesis of the aminoacyl-thiol and so we next turned our attention to thiol-catalysed aminoacyl transfer. Given the unprecedented selectivity for nucleoside aminoacylation by aminoacyl-thiols (1), we next investigated their potential provenance, as although aminoacyl-thiols (1) are biological, no reasonable non-enzymatic route had yet been found to form these fascinating compounds. We suspected that activated carboxylates, such as the biological aminoacyl adenylate 2 or prebiotic NCAs 4, would react with thiols (5) to form thioester 1 in situ, which could then aminoacylate nucleic acids with a selectivity, efficiency and side-chain scope that would not be possible without a thiol cofactor. To test this hypothesis, we incubated a range of prebiotically plausible NCAs 4 with thiols 5. We observed rapid and highly efficient synthesis of aminoacyl-thiol 1, even under highly dilute (1 mM) conditions (Fig. Simple and hydrophobic NCAs (4Gly, l-4Phe, l-4Val) furnished their respective aminoacyl-thiol (1) in near-quantitative yield on reaction with coenzyme M (thiol 5c). Biological aminoacyl-phosphate 2Ala (20 mM) also reacted with 3-mercaptopropionic acid (thiol 5b) to furnish aminoacyl-thiol \({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\) in good unoptimized yield (42%; Extended Data Fig. Notably, without thiols, NCAs (4) underwent self-condensation and hydrolysis36, which completely blocked nucleoside aminoacylation, but spontaneous self-condensation of activated amino acids (2, 4) was strongly suppressed by thiol 5. 1H NMR spectra showing various reactions. a, NCA l-4Ala (20 mM) in MES buffer (200 mM; 3.99 ppm), at pH 5.0 and room temperature, which yields: (1) alanine l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{c}}}\) (>90%) without (coenzyme M) thiol 5c, and (2) alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{c}}}\) (86%) with thiol 5c (100 mM), the aminoacyl-thiol product is highly stable at pH 5.0. b, Adenosine 16A (20 mM) and NCA l-4Ala (600 mM) at pH 6.5 and room temperature, for which no nucleoside aminoacylation was observed. Adenosine 16A (20 mM), NCA l-4Ala (600 mM) and thiol 5c (1 M) at pH 6.5 and room temperature, which furnished alanyl-adenosine \({{\bf{17}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{A}}}\) (35%) from the reaction of adenosine (16A) with aminoacyl-thiol (l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{c}}}\)) formed in situ in water. c, Adenosine 16A (20 mM), aspartate-anhydride 34Asp (600 mM) and thiol 3c (1 M) at pH 6.5 and room temperature after (1) 0.5 days and (2) 5 days, which furnished aspartyl-adenosine \({{\bf{17}}}_{{\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{A}}}\) from the reaction of adenosine (16A) with aminoacyl-thiol l-\({{\bf{1}}}_{{\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{c}}}\) formed in situ in water. d, Incubation of prebiotically plausible α-aminonitrile 8Ala (2 mM) and thiol 5b (10 mM) in frozen sodium phosphate solution (pH 7.0, 50 mM) at −7 °C for 30 days yields aminoacyl-thiol \({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\) in 48% yield at neutral pH. Thiol-suppressed NCA self-condensation suggested that thiol-catalysed aminoacylation of RNA-2′,3′-diols might be feasible, so next we incubated Ala-NCA (l-4Ala) with thiol 5c and adenosine (16A) in water. After 16 h, the resultant solution was pH 6.0 and we observed ribonucleoside aminoacylation furnishing alanyl-adenosine \({{\bf{17}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{A}}}\) in 35% yield (Fig. This effective thiol-mediated aminoacylation stands in sharp contrast to the reaction in the absence of the thiol cofactor; with no thiol, no nucleoside aminoacylation was observed under otherwise identical conditions (Fig. As expected, on incubation of oligonucleotides with NCAs (4) no oligonucleotide aminoacylation was observed. However, the addition of thiol 5b, alongside NCA 4Ala, led to effective oligonucleotide aminoacylation, through in situ formation of aminoacyl-thiol 1. For example, oligonucleotide (ON5) reacted with l-4Ala (400 mM) and thiol 5b (200 mM) in MES buffer (pH 6.0) to yield aminoacyl-RNA (23%). 127), demonstrating the marked enhancement of aminoacylation by thiols (5). We next turned our attention to glutamate (Glu) and aspartate (Asp), which have two carboxylate moieties. Glutamate thioester l-\({{\bf{1}}}_{{\bf{G}}{\bf{l}}{\bf{u}}}^{{\bf{e}}}\) (and Glu-NCA 4Glu + thiol 5c) were observed to only yield α-aminoacylation of nucleosides (11% \({{\bf{17}}}_{{\bf{G}}{\bf{l}}{\bf{u}}}^{{\bf{A}}}\) from 4Glu through in situ formation l-\({{\bf{1}}}_{{\bf{G}}{\bf{l}}{\bf{u}}}^{{\bf{c}}}\); Supplementary Fig. Glu-isomerization rapidly quenched γ-activation through pyroglutamate formation, which completely blocked γ-aminoacylation. Aspartate has two adjacent carboxylates that cannot lactamize and therefore Asp is the most challenging proteinogenic amino acid disposition for chemoselective activation and α-selective aminoacylation. However, Asp-NCA (4Asp) reacted with coenzyme M (thiol 5c) to yield aminoacyl-thiol l-\({{\bf{1}}}_{\alpha {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{c}}}\) (49%) with excellent α-selectivity (α/β 11:1) and in situ reaction with adenosine (16A) yielded aspartyl-adenosine \({{\bf{17}}}_{\alpha {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{A}}}\) also with excellent α-selectivity after 18 h (Supplementary Fig. Aspartyl-thioester l-\({{\bf{1}}}_{\alpha {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{c}}}\) slowly isomerized to its β-isomer l-\({{\bf{1}}}_{\beta {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{c}}}\), which then led to selective β-aminoacylation after 5 days. However, initial α-selectivity was strongly favoured. Moreover, the α-selectivity of Asp was not only favoured by the inherent (α-amine-tethered) α-carboxylate activation of NCA 4 but also for Asp-anhydride 34Asp, despite dual activation of both the α-carboxylate and β-carboxylate. Notably, Asp-anhydride 34Asp reacted with thiols (5) to selectively yield α-aspartyl-thioester l-\({{\bf{1}}}_{\alpha {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{c}}}\) (45%) with excellent α-selectivity (α/β 12:1) in water after 30 min (Supplementary Fig. Therefore, incubating Asp-anhydride 34Asp, thiol 5c and adenosine 16A in water at pH 6.5 resulted in selective α-aminoacylation of 16A and furnished \({{\bf{17}}}_{\alpha {\bf{A}}{\bf{s}}{\bf{p}}}^{{\bf{A}}}\) (12%) after 12 h (Fig. 5c (1)), demonstrating a profound kinetic selectivity for natural (proteinogenic) α-aminoacylation, irrespective of α-amine-tethered activation. The reaction of thiols 5 with activated amino acids (2, 4), which have previously been suggested to be relevant for prebiotic aminoacylation15,16, illustrates the role that thiol cofactors could have played in enabling catalytically controlled aminoacylation. These results demonstrate that there are several high-yielding prebiotic pathways to aminoacyl-thiols 1, as well as for in situ thioester-mediated ribonucleoside-2′,3′-aminoacylation at near-neutral pH. Aminoacyl-thiols 1 seem to be very well suited to the spontaneous and selective aminoacylation of nucleosides in water. Both aminoacyl adenylate 2 and NCAs 4 are derived from amino acids by means of electrophilic activation and neither has a long lifetime in water. Although this makes their efficient reaction with thiols more noteworthy, we envisaged that a prebiotically plausible route to aminoacyl-thiols (1) might be feasible by the reaction of aminonitriles (8) with thiols (5). Aminonitriles (8) are considered probable precursors to amino acids on the early Earth but their high-energy nitrile has sufficient energy to be directly transformed into a thioester, avoiding fully hydrolysed amino acids and the resultant need for an external source of chemical energy. Aminonitriles (8) are kinetically stable but we have found that thiols (5) can unlock the energy in nitriles to enable formation of peptide bonds37,38,39. Furthermore, we have recently discovered a selective route by which pantetheine (5a), the universal biological thioester-bearing moiety that is central to non-ribosomal peptide synthesis, forms in water from nitriles20. This link between nitriles, biological peptides and biological thiols led us to consider the direct link between nitriles and activated amino acid thioesters more closely. We began by studying the reaction of alanine nitrile (8Ala) with 3-mercaptopropionic acid (5b)33,38. 185) but under mildly acidic conditions (pH 5.0), alanine thioester (\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\)) was formed in 20% yield (Extended Data Fig. A range of simple and biological thiols, including pantetheine (5a) and coenzyme M (5c), reacted with aminonitriles (8) to yield aminoacyl-thiols (1) under acidic conditions (Extended Data Fig. The formation of thioester 1 from aminonitrile 8 was observed at prebiotically plausible pH (that is, pH 5.0), but it was most effective under slightly more acidic conditions. At pH 3.0–4.0, the formation of aminoacyl-thiol 1 was observed in 42–48% yield. However, initially, this low pH (<5.0) seemed prebiotically unrealistic. We reasoned that the selectivity for aminoacyl-thiol 1 would be improved by general-acid catalysis40 and phosphate was found to modestly increase the yield of thioester \({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\) (56%; Extended Data Fig. However, phosphate, as well as being a catalyst for thioester formation, could drive (reversible) pH change from neutral pH to mildly acidic conditions by a highly plausible prebiotic mechanism, because freezing phosphate solutions leads to an ideal drop in pH (ref. 41), to the sweet spot for aminoacyl-thiol 1 formation. Accordingly, we found that solutions of nitrile 8Ala and thiol 5b, buffered by phosphate at pH 7.0, yielded alanine thioester \({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\) efficiently when frozen and incubated at −7 °C. Furthermore, freezing allows the initial conditions to be, in principle, almost indefinitely dilute owing to the concentrating effects of forming the eutectic phase. For example, we observed that incubating 2 mM alanine nitrile 8Ala with 10 mM thiol 5b at −7 °C led to effective synthesis of alanine thioester \({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{b}}}\) (48%) after 30 days and a neutral pH solution of thioester 1 on thawing the frozen solution (Fig. 195), demonstrating low concentration, prebiotically plausible, synthesis of thioester 1 from a neutral pH solution of aminonitrile 8. However, the conversion of 8Phe (21%) and 8Val (14%) were lower, probably because of thioimidate tautomerization (Supplementary Discussion). Proteinogenic α-aminonitriles (8) were readily converted in comparable yield to their respective aminoacyl-thiols (1). Notably, however, non-proteinogenic α-hydroxy nitrile 36, β-aminonitrile 37 and acylated α-aminonitrile 38 all underwent <5% reaction with thiol 5b under the same conditions (Extended Data Fig. Nucleoside aminoacylation with aminoacyl-thiols (1) results from judiciously balanced reactivity of the nucleoside-2′,3′-diol and aminoacyl-thiol 1, allowing (biologically) activated amino acids (that is, thioester 1) to chemoselectively deliver aminoacyl-RNA in water (Fig. Notably, thioester activation is orthogonal to peptide bond formation at neutral pH. To demonstrate further the importance of thioester activation for this selectivity, we set out to undertake an investigation of peptide synthesis and the formation of peptidyl-RNA (40) in water under the same conditions that yield aminoacyl-RNA (17). Notably, reaction at the RNA-2′,3′-diol was blocked by N-acylation of thioester 1 and no peptidyl-RNA (40) was observed on incubating nucleoside 16A with N-acetyl-aminoacyl-thiol Ac-Ala-SEt \({{\bf{39}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) at pH 6.5 (Extended Data Fig. This further demonstrates the importance of the amine moiety for promoting selective reaction at the RNA-2′,3′-diol but, more importantly, selective reaction of the free amine (that is, thioester 1) provides a mechanism to promote peptide formation by means of the nucleophilicity of the aminoacyl-RNA (17) as required for nucleic-acid-orchestrated peptide synthesis (that is, RPS). Aminoacylated nucleotides (for example, 17) have been demonstrated to form peptides by iterative, non-enzymatic cycles of 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC)-mediated condensation and so the selective formation of aminoacyl-RNAs (17) from aminoacyl-thiols (1) completes the non-enzymatic, protecting-group-free RNA-mediated synthesis of peptides from thioesters and RNA in water42,43. However, we considered it to be more important that, contrary to these EDC-activated results, in our reactions, aminoacyl-RNAs (17) are not observed to yield peptidyl-RNA (40) at all. Despite the aminoacyl-RNA (17) having a free amine and there being a large excess of thioester available, which could, in principle, react to yield peptides, no peptidyl-RNA (40) synthesis was observed, only the selective aminoacylation of RNA. This difference suggested that there must be an underlying chemical switch that would turn on peptide synthesis and this might be exploited to chemically control stepwise biomimetic synthesis of peptidyl-RNA in water. EDC-activated carboxylic acids are highly activated and so undergo reaction with amines to yield peptides in water. This stands in sharp contrast to the reactivity of aminoacyl-thiols (1) that are weakly activated and do not yield peptides under the same conditions. On reflection, we reasoned that peptide synthesis would be most effectively promoted by a change in the characteristics of acyl-activation, rather than by nucleic acid templating, and to test this hypothesis, we next studied the synthesis of peptidyl-RNA from (prebiotically plausible) thioacids (11). Thioacids are readily accessible from thioesters, on reaction with hydrogen sulfide (H2S) at neutral pH (Fig. 2a), and can be activated selectively in water by ferricyanide, copper salts or cyanoacetylene (HC3N) to undergo chemoselective, high-yielding peptide ligation. These ligations are extremely selective for peptide coupling and tolerate all of the 20 proteinogenic amino acid residues24, therefore we suspected that they would be ideally suited to peptidyl-RNA synthesis in water. Notably, aminoacyl-RNA (17) reacted with peptide thioacids 11 and ferricyanide at pH 6.0–6.5 to furnish peptidyl-RNA 40 in excellent to near-quantitative yield (Fig. Unlike thioester 1, activated thioacid 11 reacted very selectively with the amine moiety of 17. Notably, unlike EDC activation, thioacid activation is orthogonal to peptide side chains and nucleoside functionalization24 and so peptidyl-RNA 40 was observed to form very efficiently, without (thioacid-mediated) acylation of nucleoside alcohols or nucleobases (Extended Data Fig. Moreover, owing to the mild (neutral) conditions, the stereochemistry of aminoacyl-thiols (1), aminoacyl nucleosides (17) and peptide thioacids (11) were preserved during both aminoacylation and peptidyl-RNA synthesis in water; racemization was not observed and so peptidyl-RNAs (40) were formed as a single isomer from homochiral substrates. a, Incubation of 16 with thioester 1 and thioacid 11 yields aminoacyl-RNA 17, and in situ oxidation of 11 converts 17 into peptidyl-RNA 40 in near-quantitative yield. R and R1 = peptide side chain. b, 1H NMR spectra of: (1) 16A (20 mM) and 33 (80 mM) in MES buffer (500 mM, pH 6.5) after 2 h; (2) after addition of K3Fe(CN)6 (150 mM) and thioacid 11Gly (50 mM), which yields \({{\bf{40}}}_{{\bf{A}}{\bf{r}}{\bf{g}}{\bf{G}}{\bf{l}}{\bf{y}}{\bf{A}}{\bf{c}}}^{{\bf{A}}}\) (93%); (3) 16A (20 mM) and 33 (80 mM) in MES buffer (500 mM, pH 6.0) after 3 h; (4) after addition of K3Fe(CN)6 (450 mM) and Ac-Ala-Ala-SH (11AlaAla, 150 mM), which furnished \({{\bf{40}}}_{{\bf{A}}{\bf{r}}{\bf{g}}{\bf{A}}{\bf{l}}{\bf{a}}{\bf{A}}{\bf{l}}{\bf{a}}{\bf{A}}{\bf{c}}}^{{\bf{A}}}\) (89%); (5) adenosine-3′,5′-cyclic phosphate 31A (20 mM) and 33 (80 mM) in MES buffer (500 mM, pH 6.5) after 1 h, which yields aminoacyl-RNA 41; (6) after addition of K3Fe(CN)6 (300 mM) and Ac-Gly-Gly-Gly-SH (11GlyGlyGly, 100 mM), which furnished \({{\bf{42}}}_{{\bf{A}}{\bf{r}}{\bf{g}}{\bf{G}}{\bf{l}}{\bf{y}}{\bf{G}}{\bf{l}}{\bf{y}}{\bf{G}}{\bf{l}}{\bf{y}}{\bf{A}}{\bf{c}}}^{{\bf{A}}}\) (95%); see Extended Data Figs. 1 and 8 for structures of 31, 41 and 42; (7) 16A (20 mM) and l-\({{\bf{1}}}_{{\bf{L}}{\bf{e}}{\bf{u}}}^{{\bf{e}}}\) (200 mM) in MES buffer (1 M, pH 6.5) after 24 h; (8) after addition of K3Fe(CN)6 (1.05 M) and 11Gly (350 mM), which furnished peptidyl-RNA \({{\bf{40}}}_{{\bf{L}}{\bf{e}}{\bf{u}}{\bf{G}}{\bf{l}}{\bf{y}}{\bf{A}}{\bf{c}}}^{{\bf{A}}}\) (95%). 5.81 ppm (s, 1H) resonance = glycyl-thioacid by-product (see ref. 24). For a comparable one-pot peptidyl-RNA synthesis, in which 16A, 33 and K3Fe(CN)6 are incubated to yield 17 and then thioacid 11Gly was added at 2 h to furnish 40, see Supplementary Fig. Finally, owing to the orthogonal nature of thioester and thioacid activation and their capability to direct the two steps of RPS, we found that the one-pot synthesis of aminoacyl-RNA (17) in the presence of thioacid 11 was observed, at pH 6.5. No hydrolysis of 11 or peptide bond formation was observed (Fig. 6c (2)) and so subsequent in situ, one-pot oxidation of 11 then converted aminoacyl-RNA (17) into peptidyl-RNA (40) in near-quantitative yield (Fig. The selective formation of peptidyl-RNA 40 in water demonstrates the subtle, but important, difference in reactivity between aminoacyl-thiols 1 and peptide thioacids 11, and shows conclusively that the selective reaction of thioester (1) with nucleoside alcohols cannot be explained by amine protonation at neutral pH. Thioesters and thioacids together provide the chemical differentiation required to control RNA aminoacylation and peptide synthesis, through orthogonal reactions, under the same (pH and concentration) conditions. The highly selective non-enzymatic synthesis of aminoacyl-thiols (1), coupled with our recent selective synthesis of pantetheine (5a)20, completes a direct synthetic chain from simple nitriles to the aminoacyl-thiols used by life to facilitate non-ribosomal peptide synthesis (that is, pantetheinyl aminoacyl-thiol \({{\bf{1}}}_{{\bf{A}}{\bf{a}}{\bf{a}}}^{{\bf{a}}}\)) and, from there, selective formation of aminoacyl nucleotides (17) is observed in water at neutral pH. The emergence of the biochemical intermediates of both ribosomal and non-ribosomal peptide synthesis from one chemical route, in water and without protecting groups, is distinct and unexpected. Coupling nitrile reactivity with the physical (eutectic) behaviour of phosphate solutions41 makes aminoacyl-thiol synthesis prebiotically plausible and warrants further (geochemical) investigation of this process in model (arctic or polar) soda lakes that are known to accumulate high concentrations of phosphate44. We have discovered that aminoacyl-thiols 1 and biological thiol cofactors (for example, pantetheine 5a and coenzyme M 5c), make RNA aminoacylation a predisposed chemical process in water. The formation of aminoacyl-RNAs (17) is the first step of RPS but had proved to be extremely challenging under prebiotically realistic conditions4,5. Thioester-mediated RNA aminoacylation resolves this problem, occurring with unprecedented efficiency and side-chain tolerance in water at neutral pH to yield the substrates that are required for RPS in water. Our results indicate that thioesters (and thiol cofactors) may have played a key role during the early biochemical evolution of protein synthesis. Although the de novo development of a ‘ribosome' and the iteration of the second step of protein synthesis are beyond the scope of the present work, we have demonstrated that a simple switch in acyl activation, from thioester (1) to thioacid (11) activation, switches on peptide synthesis and switches off diol aminoacylation. Together, these reactions demonstrate that the two steps of RPS can be chemically orchestrated without evolved catalysts. Future work will focus on developing peptidyl-RNA coupling to enable iteration and catalytic control over the second step of RPS. The specified aminoacyl-thiol (\({{\bf{1}}}_{{\bf{A}}{\bf{a}}{\bf{a}}}^{{\bf{a}}{\boldsymbol{-}}{\bf{e}}}\)) and pentaerythritol (internal standard) were dissolved in degassed H2O/D2O (98:2) at the specified pH and added to the specified nucleoside or nucleotide (16, 20–32). The pH of the resultant solution was measured and, if necessary, corrected to the stated pH with HCl/NaOH and then this solution was diluted with degassed H2O/D2O (98:2) to give the specified reactant concentrations. The solution was then incubated at room temperature and nuclear magnetic resonance (NMR) spectra were periodically acquired. The specified gel-purified FAM-oligonucleotide (ON#; 10 µM, 0.5 µl), +/−10-mer or 15-mer complement ON (10 µM, 0.6 µl) and +/−5-mer downstream ON (10 µM, 0.6 µl) were added to the reaction buffer (2 µl, 2.5 M KCl, 1–2 M MES, pH 6.5). The resultant solution (which contained 0.5 µM RNA substrate, ±0.6 µM 10-mer or 15-mer-complement, ±0.6 µM 5-mer, 500 mM KCl, 200–400 mM MES pH 6.5 and 200 mM thioester) was vortexed, centrifuged and incubated at room temperature. At each required time point, an aliquot (1–2 µl) was removed. If the reaction contains ds-RNA, a competitor oligomer (30 equiv. with respect to FAM oligonucleotide) was then added to the aliquot and it was incubated for 5–10 min leading to the formation of a single-stranded FAM-tagged oligomer. Then a quenching buffer (16 µl; 10 mM EDTA, 150 mM NaOAc and 93% (vol/vol) formamide) was added to the aliquot and then it was analysed by polyacrylamide gel electrophoresis (PAGE; 36% wt/vol urea, 20% (19:1) acrylamide, 10% 10 × EA) in 1 × EA buffer (100 mM NaOAc, 2 mM EDTA, pH 5.2) pre-cooled to 1 °C. The gel was then shaken in 1 × TBE buffer (89 mM Tris, 89 mM borate, 2 mM EDTA, pH 8.3) for 5–10 min and the fluorescent (473 nm excitation) bands were quantified. The specified aminoacyl-thiol (\({{\bf{1}}}_{{\bf{A}}{\bf{a}}{\bf{a}}}^{{\bf{a}}{\boldsymbol{-}}{\bf{e}}}\); 100 μmol) or 33 (40 μmol) was dissolved in degassed D2O (0.4 ml) or H2O/D2O (98:2, 0.4 ml) +/− MES buffer (0.5–2.0 M) and the solution was carefully adjusted to pH 6.0–6.5 with NaOH or HCl as required. The solution was adjusted again to the specified pH value and diluted to 0.5 ml with degassed D2O or H2O/D2O (98:2). The solution was then incubated at room temperature and NMR spectra were periodically acquired over 2–24 h. If MES buffer was not present in solution, MES buffer (1 M, 500 μmol, pH 6.5) was then added. K3Fe(CN)6 (150 or 450 μmol) and thioacid (11; 50 or 150 μmol) were then added, and after 10 min, the reaction mixture was centrifuged. The supernatant was incubated at room temperature and NMR spectra were acquired. Data generated and analysed during this study, including experimental data, spectroscopic and PAGE images, are included in this article and its Supplementary Information. Weber, A. L. & Lacey, J. C. Aminoacyl transfer from an adenylate anhydride to polyribonucleotides. Weber, A. L. & Orgel, L. E. Amino acid activation with adenosine 5′-phosphorimidazolide. Radakovic, A. et al. A potential role for RNA aminoacylation prior to its role in peptide synthesis. B., Beaudreau, G. & Mills, D. The synthesis of a self-propagating and infectious nucleic acid with a purified enzyme. Bartel, D. P. & Szostak, J. W. Isolation of new ribozymes from a large pool of random sequences. & Joyce, G. F. Emergence of a fast-reacting ribozyme that is capable of undergoing continuous evolution. Ichihashi, N. et al. Darwinian evolution in a translation-coupled RNA replication system within a cell-like compartment. A., Severin, K. & Ghadiri, M. R. A self-replicating peptide. Rout, S. K., Friedmann, M. P., Riek, R. & Greenwald, J. A prebiotic template-directed peptide synthesis based on amyloids. Breaker, R. R. & Joyce, G. F. The expanding view of RNA and DNA function. Tamura, K. & Schimmel, P. Chiral-selective aminoacylation of an RNA minihelix. Wu, L.-F., Su, M., Liu, Z., Bjork, S. J. & Sutherland, J. D. Interstrand aminoacyl transfer in a tRNA acceptor stem-overhang mimic. Turk, R. M., Illangasekare, M. & Yarus, M. Catalyzed and spontaneous reactions on ribozyme ribose. Biron, J.-P., Parkes, A. L., Pascal, R. & Sutherland, J. D. Expeditious, potentially primordial, aminoacylation of nucleotides. Shannon, C. E. A mathematical theory of communication. De Duve, C. in The Molecular Origins of Life: Assembling Pieces of the Puzzle (ed. Goldford, J. E., Hartman, H., Smith, T. F. & Segrè, D. Remnants of an ancient metabolism without phosphate. Fairchild, J., Islam, S., Singh, J., Bučar, D. K. & Powner, M. W. Prebiotically plausible chemoselective pantetheine synthesis in water. Gless, B. H., Schmied, S. H., Bejder, B. S. & Olsen, C. A. Förster resonance energy transfer assay for investigating the reactivity of thioesters in biochemistry and native chemical ligation. Weber, A. L. & Orgel, L. E. The formation of peptides from glycine thioesters. & Powner, M. W. Selective synthesis of lysine peptides and the prebiotically plausible synthesis of catalytically active diaminopropionic acid peptide nitriles in water. Canavelli, P., Islam, S. & Powner, M. W. Peptide ligation by chemoselective aminonitrile coupling in water. Ninomiya, K., Minohata, T., Nishimura, M. & Sisido, M. In situ chemical aminoacylation with amino acid thioesters linked to a peptide nucleic acid. Li, N. & Huang, F. Ribozyme-catalyzed aminoacylation from CoA thioesters. & Holliger, P. In-ice evolution of RNA polymerase ribozyme activity. Merino, E. J., Wilkinson, K. A., Coughlan, J. L. & Weeks, K. M. RNA structure analysis at single nucleotide resolution by selective 2′-hydroxyl acylation and primer extension (SHAPE). Fang, L., Xiao, L., Jun, Y. W., Onishi, Y. & Kool, E. T. Reversible 2′-OH acylation enhances RNA stability. Yarus, M. A specific amino acid binding site composed of RNA. Aumiller, W. & Keating, C. Phosphorylation-mediated RNA/peptide complex coacervation as a model for intracellular liquid organelles. & Blackmond, D. G. Symmetry breaking and chiral amplification in prebiotic ligation reactions. Calendar, R. & Berg, P. d-tyrosyl RNA: formation, hydrolysis and utilization for protein synthesis. Pawar, K. I. et al. Role of D-aminoacyl-tRNA deacylase beyond chiral proofreading as a cellular defense against glycine mischarging by AlaRS. Hadjichristidis, N., Iatrous, H., Pitsikalis, M. & Sakellariou, G. Synthesis of well-defined polypeptide-based materials via the ring-opening polymerization of α-amino acid N-carboxyanhydrides. Foden, C. S. et al. Prebiotic synthesis of cysteine peptides that catalyze peptide ligation in neutral water. Singh, J. et al. Prebiotic catalytic peptide ligation yields proteinogenic peptides by intramolecular amide catalyzed hydrolysis facilitating regioselective lysine ligation in neutral water. & Powner, M. W. Spontaneous peptide ligation mediated by cysteamine. Chaturvedi, R. K., MacMahon, A. E. & Schmir, G. L. The hydrolysis of thioimidate esters. Tetrahedral intermediates and general acid catalysis. Van den Berg, L. The effect of addition of sodium and potassium chloride to the reciprocal system: KH2PO4-Na2HPO4-H2O on pH and composition during freezing. Jash, B., Tremmel, P., Jovanovic, D. & Richert, C. Single nucleotide translation without ribosomes. Reußwig, S. G. & Richert, C. Ribosome-free translation up to pentapeptides via template walk on RNA sequences. Toner, J. D. & Catling, D. C. A carbonate-rich lake solution to the phosphate problem of the origin of life. We thank the Engineering and Physical Sciences Research Council (EP/X011755/1 to M.W.P. Jyoti Singh, Benjamin Thoma, Daniel Whitaker, Max Satterly Webley, Yuan Yao & Matthew W. Powner Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and D.W. contributed equally to the development of RNA aminoacylation. The authors declare no competing interests. Nature thanks Thomas Carell and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, 1H NMR spectra show the reaction of: (i) l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) and cytidine 16C (20 mM) at pH 6.5, which yields aminoacyl-cytidine \({{\bf{17}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{C}}}\) (25%) after 24 h; (ii) l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (100 mM), 16C (20 mM) and 5′-mercapto-adenosine 15 (15 mM) at pH 6.5 after 24 h; and (iii) l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (100 mM) and 15 (20 mM) at pH 6.5, in which 2′,3′-diol aminoacylation (54%) is observed after 24 h. b, Structure of non-canonical nucleosides (20, 22–25) and ribonucleotides (26–32). c, Alcohol-aminoacylation yield for specified nucleosides (20 mM) with l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM) at room temperature and pH 6.5 after 24 h. See Fig. 2c for canonical RNA and DNA nucleosides. For more experimental details, see Supplementary Figs. RNA hydrolysis proceeds by means of fragmentation, which yields 2′-phosphate and 3′-phosphate through 2′,3′-phosphate intermediate. The high pKa of these alcohols inhibits aminoacylation and prevents the adventitious aminoacylation of RNA degradation products. PAGE images show the reaction of ds-oligomeric RNA (0.5 µM) with aminoacyl-thiol l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (200 mM, R = Me, R1 = Et) in MES buffer (200 mM, pH 6.5) and KCl (500 mM) after 16 h at room temperature: left, 5′-FAM-UAGGAGACCA (ON5) + 5′-GCAGUUGGUCUCCUA (ON7); right, 5′-FAM-UAGGAGACCA-3′p (ON5-3′p) + ON7. a, Yield of aminoacyl-thiol \({{\bf{1}}}_{{\bf{Aaa}}}^{{\bf{c}}}\) from NCA (4) and coenzyme M (5c) in MES buffer (200 mM) at room temperature after 10–30 min. For further data and other thiols (5), see Supplementary Figs. For yield of \({{\bf{1}}}_{{\bf{Ala}}}^{{\bf{b}}}\) observed from reaction of aminoacyl-adenylate 2 and thiol 5b at alternative pHs (pH 2.0–5.0), see Supplementary Table 39. a–d, 1H NMR spectra show the selective synthesis of alanine thioester \({{\bf{1}}}_{{\bf{Ala}}}^{{\bf{b}}}\) (50–55%) from alanine nitrile 8Ala (200 mM) and thiol 5b (9 equiv., 1.8 M) at pH 3.0 and 5.0 and room temperature, in the presence of: lactonitrile (36) (b); β-alanine nitrile (37) (c); and N-acetyl alanine nitrile (38Ala) (d). 5 for low-concentration, neutral pH reactions. at specified pH and room temperature. PB, phosphate; CB, citrate; n.d., not detected. b, The yield of aminoacyl-thiol \({{\bf{1}}}_{{\bf{Aaa}}}^{{\bf{b}}}\) from the reaction of specified α-aminonitrile 8 (0.2 M) and thiol 5b (0.2–1.8 M, 1–9 equiv.) in PB at pH 3.0 and room temperature. An expanded dataset is provided in Supplementary Tables 41–46. Stoichiometric competition between amidoacyl-thiol (39) and aminoacyl-thiol (1) demonstrates the selective aminoacylation of RNA at neutral pH. The reaction of adenosine 16A (20 mM), N-acetyl-glycine thioacid 11Gly (240 mM) and alanine thioester l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) (240 mM) in MES buffer (500 mM, pH 6.5) at room temperature yields: (i) no amidoacyl-RNA, when K3Fe(CN)6 (600 mM) was added after 30 min; l-\({{\bf{1}}}_{{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) was the only amine after 30 min, so Ac-Gly-Ala-SEt (\({{\bf{39}}}_{{\bf{G}}{\bf{l}}{\bf{y}}{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\)) was observed as the only peptide product and \({{\bf{39}}}_{{\bf{G}}{\bf{l}}{\bf{y}}{\bf{A}}{\bf{l}}{\bf{a}}}^{{\bf{e}}}\) does not react with 16A (also see Extended Data Fig. Yield of peptidyl-RNA 40 and 42 from the reaction of aminoacyl-RNA 17 or 41 with thioacid 11 (100–300 mM) and K3Fe(CN)6 (3 equiv.) in MES buffer (0.5–1.0 M, pD 6.0) at room temperature. All aminoacyl-RNAs were formed from their respective nucleoside (A, C, U = 20 mM, G = 2 mM) and thioester 1 (200 mM) or cyclic arginine 33 (80 mM) and peptide synthesis implemented in situ, without isolation or any purification. 1–264 (nuclear magnetic resonance (NMR) spectra, gel images, reaction time course graphs), Supplementary Tables 1–51 (reaction scope, optimization, mass spectrometry (MS) data) and Supplementary References. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Singh, J., Thoma, B., Whitaker, D. et al. Thioester-mediated RNA aminoacylation and peptidyl-RNA synthesis in water. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250827010732.htm'>Astronomers capture breathtaking first look at a planet being born</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 14:52:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>However, in the case of this star, we instead detected a completely unexpected and exceptionally beautiful multi-ringed dust disk. The first one was discovered in 2018, by a research team also involving Dr Ginski. WISPIT 2b is also the first unambiguous planet detection in a multi-ringed disk, making it the ideal laboratory to study planet-disk interaction and subsequent evolution. The planet was captured in near infrared light - the type of view that someone would see when using night-vision goggles -- as it is still glowing and hot after its initial formation phase. The team at Leiden University and University of Galway captured a spectacular clear image of the young proto-planet embedded in a disk gap. They also confirmed that the planet is orbiting its host star. This detection at a specific wavelength or color of light indicates that the planet is still actively accreting gas as it is forming its atmosphere. WISPIT 2b was detected as part of a five-year observational research project during which the international team sought to establish whether wide orbit gas giant planets are more common around younger or older stars. Dust and gas rich disks around young stars are the birth cradles of planets. They can look quite spectacular with many different structures such as rings and spiral arms, which researchers believe are related to planets forming within them. Dr Ginski added:"Capturing an image of these forming planets has proven extremely challenging and it gives us a real chance to understand why the many thousands of older exoplanet systems out there look so diverse and so different from our own solar system. I think many of our colleagues who study planet formation will take a close look at this system in the years to come." The study was led by an early career PhD student, Richelle van Capelleveen from Leiden University and co-led by a graduate student team at University of Galway. The research findings were co-authored by Dr Ginski and three Physics graduates students who are specializing in Astrophysics at University of Galway. Richelle van Capelleveen said: "Discovering this planet was an amazing experience -- we were incredibly lucky. This system will likely be a benchmark for years to come." Dr Ginski said: "We were so fortunate to have these incredible young researchers on the case. WISPIT 2b, with its position within its birth disk, is a beautiful example of a planet that can be used to explore current planet formation models. I am certain this will become a landmark paper, owing particularly to the work of Richelle van Capelleveen and her exceptional team." I could hardly believe it was a real detection when Dr Ginski first showed me the image. It's a big one -- that's sure to spark discussion within the research community and advance our understanding of planet formation. Contributing to something this impactful, and doing so alongside international collaborators, is exactly the kind of opportunity early-career researchers like Chloe, Dan and I dream of." I feel so well treated by the University of Galway Physics department and especially my supervisor Dr Christian Ginski to have provided me with the opportunity to be part of such an exciting project." After 60 Years, Scientists Uncover Hidden Brain Pathway Behind Diabetes Drug Metformin Common Pesticide Linked to “Remarkably Widespread” Brain Abnormalities in Children Revolutionary Cortisol Test Lets You “See” Stress With a Smartphone Camera How B Vitamins Could Slow Cognitive Decline and Protect Against Dementia Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/deep-sea-nodules-may-produce-oxygen-raising-concerns-over-ocean-mining/'>Deep-Sea Nodules May Produce Oxygen—Raising Concerns over Ocean Mining</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Mining the Deep Sea Could Threaten a Source of Ocean Oxygen Deep-sea rocks packed with valuable metals may also be making oxygen in the deep, dark ocean—raising new questions about the cost of mining them. By Rachel Feltman, Jeffery DelViscio & Alex Sugiura Scattered across the deep ocean floor are trillions of potato-sized black rocks packed with valuable metals such cobalt and copper. Mining companies want to harvest these nodules to get materials for electric vehicle batteries and other clean energy tech. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Today we're joined by Clare Fieseler and Jason Jaacks, who recently explored these mysterious deep-sea rocks in a mini documentary for Scientific American. Jason is a documentary filmmaker and an associate professor of journalism at the University of Rhode Island, and Clare is a scientist as well as a journalist for Canary Media, a nonprofit news outlet focused on clean energy and climate change. Thank you both so much for coming on to chat. Feltman: So let's start with a, a basic question: What are nodules, and how do scientists think that they're formed? At first they were just on display in Victorian museums like moon rocks, and then in, like, the 1950s and '60s scientists started to kind of realize that—or at least they hypothesized—that these had been formed over millions and millions of years, that they likely started off as little, tiny bits of what they call anuclei, and it's, like, a shark tooth or a whale ear bone or a piece of pumice from, from an exploded volcano. Parts of debris fall to the ocean floor, and then through a process called sorption, they begin to almost accumulate minerals from the seawater in these layers. And actually, when you can cut into one you'll see these rings that look like tree rings, and you can age a nodule by counting those rings. And so they're infinitely fascinating from a scientific perspective, and now they're also fascinating from an economic perspective, which we'll probably talk about. Feltman: Definitely gonna get into that. But like you said these aren't really much to look at at first glance, but they can apparently upend our understanding of how life started on Earth. Jaacks: One of the remarkable things about these nodules is that they provide habitat for all kinds of different creatures, and those are creatures that we can see, but they also provide habitat for all kinds of interesting microbial life. And so one of our sources described those nodules to us as a “Manhattan ...” They basically noticed that oxygen levels within this chamber were going up. So if that is, in fact, correct, then the rocks themselves were producing oxygen. And that might still be the case, but the source we interviewed for this film, Dr. Jeff Marlow, put in kind of like this chemical cocktail that acts as a poison to kill those microbes, and after they introduced that to their experiment the nodules were still producing oxygen from in that chamber. And so if they successfully killed all the microbes that were living in that nodule, then it must be something else. And so the paper is very preliminary and exploratory, and the scientists even admit, like, “We need to do a lot more studies, but we think it's not the microbes; we think it's actually the rock itself.” Like, this is the mind-blowing side of things. The scientists are still trying to understand what that process is, but they think it's something called seawater electrolysis: essentially where the nodule is acting like a battery to power a process that produces this oxygen. Feltman: Wow, I mean, even just “Manhattan for microbes” is so evocative, and then the oxygen stuff is, like, almost a little spooky in a very cool way [laughs]. Fieseler: These nodules contain minerals that are absolutely essential for electric vehicle batteries that are made today. This week I bought a Kia EV9, which is an electric vehicle, and in my new car, in the giant battery underneath my feet as I drive, there's enough cobalt where you want a [reliable] supply chain of cobalt if you want kind of a clean-car revolution for the United States and for the world. We get most of our cobalt from the Democratic Republic of the Congo, which has its own issues with human rights abuses and so forth. Fieseler: And we can talk about that later. Feltman: Yeah, well, that's actually—it's a great segue. My next question was just gonna be: you know, you guys made a film about the nodules, so I would love to hear more about what tensions exist around them and, and who the players are in this conversation. Jaacks: Yeah, so in making this film we discovered this kind of core tension between trying to understand this environment, this place that we really don't know much about, and these little potato-sized kind of alien rocks from the bottom of the sea it turns out have this, you know, economic potential that we didn't really know about prior. So especially as we try to electrify our economy these are really valuable minerals. So the tension that we really were exploring in the film was: How do we understand these nodules? You know, in one way they are the substrate and they provide this structure for life at the bottom of the sea, and do we want to try to understand that, or do we use those resources kind of in the short term to electrify our economies so that we can avert some of the worst impacts of, of climate change? So that tension between: What are these rocks for? Do they have, you know, value laying on the seafloor because they might be potentially producing oxygen or because they provide this entire habitat for what is, ultimately, the default habitat on Earth? And Clare stumbled across this incredible story that actually led to the rediscovery of the very first deep-sea mining site, which happened off the coast of South Carolina. Fieseler: You might be hearing about deep-sea mining for the first time now—deep-sea mining has been around for about 50 years. And so a wealthy shipbuilder poured a bunch of money into this endeavor, and in 1970 they tested it successfully over 100 miles off the coast of South Carolina. But then, you know, it never was really economically profitable. The technology was cool, you know, they wanted to prove that they could do it, but yet a lot of these early deep-sea mining companies went bankrupt because the economics were never there. But what was so interesting is that they left this legacy of experimentation on the bottom of the seafloor. And one of the characters in our film, Dr. Jason Chaytor, he kind of stumbled upon mention of the world's first deep-sea mining experiment site, which had been completely lost by the U.S. government. And he spent years trying to, like, piece together what happened and where it was, and he got money from the U.S. government to go back to just see, like, “What does this place look like [more than] 50 years later?” They didn't know what they were gonna see. And the very first time I spoke to him about this, I said, “Well, what did it look like when you brought robots, you know, to the bottom of the sea [more than] 50 years after mining?” He said, “It looked like they were there yesterday.” That really kind of, like, gave me goose bumps, and I knew I kind of wanted to pursue this further. Like, let's start following this.” And that's kind of how this film came to be. Fieseler: So the Metals Company, in pursuit of a license to extract minerals in an area beyond national jurisdiction, which just means, like, an area of the ocean that doesn't belong to any country—it's, it's the high seas, right? Fieseler: And technically, under a global treaty that most of the world's countries have signed, that area is designated under international law as the “common heritage” of all mankind: the [more than] eight billion people alive today and all future generations to come. And countries haven't yet agreed on how to do that—they're, they're very close. And so in preparation for getting approval, in theory, this company got [exploration permits], so they're kind of doing the, the studies, and they had to pay scientists to kind of understand what's down there, right? “If we go and mine this area with the approval of all these countries under this treaty called the Law of the Sea, we need to make sure that it's gonna have—you know, understand our impact down there.” They had to pay for this science, and this discovery of oxygen production was completely unexpected, and it just happens that the results are just not great for the company's sales pitch. These rocks, they're just kind of sitting out there like golf balls on a fairway—scoop 'em up and no harm, no foul, right? If that's not the case, as this science is suggesting, if they're producing oxygen then, you know, removing them, it is, like, way more risky than we thought. Fieseler: There's a rebuttal online that people can go and find—if you look at the Metals Company “dark oxygen” rebuttal—but it's worth mentioning that there's a peer-review process here, right? Fieseler: If you're refuting a peer-reviewed study, then you have to submit your arguments and your rebuttal to the peer-reviewed journal where it was published. So it has not kind of gotten the stamp of approval from other peers that their rebuttal has scientific merit. But Andrew Sweetman, who was the lead co-author of the study, he actually got quite a bit of money to continue research on this. So research on “dark oxygen” is moving forward; that's one thing. We're waiting to see if the rebuttal passes peer review, what that looks like. But, you know, there's a new player in this space, and that is the Trump administration. The Trump administration has put out an executive order to essentially support companies to mine the deep sea in areas outside of U.S. waters, and this is really controversial. Jaacks: So the International Seabed Authority, they're this international body that is designated to regulate how mining companies will move forward. We were just at their 30th meeting in Kingston, Jamaica, where their organization is headquartered, for the assembly in which all the delegates from the countries that have signed on, they were all there to continue this global conversation about how to move forward or whether to move forward. They didn't come to a conclusion at the end of this meeting; they meet twice a year. There's also this tension between an administration that is trying to move unilaterally and move in a different direction than a global organization that was specifically established to regulate and come up with rules about how we might exploit these kinds of resources. So, you know, there's going to be this continuing battle between the American position on deep-sea mining and a global one, so we'll continue to watch that as it develops. Thank you both so much for coming on to talk us through this. Jaacks: Thank you so much for having us, yeah. For more on these mysterious nodules, check out Clare and Jason's documentary about the subject over on Scientific American's YouTube channel. You can find a link to that in our show notes. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02411-0'>Tropical deforestation is associated with considerable heat-related mortality</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 09:19:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Tropical deforestation induces local warming and is a potential human health risk, having been linked to elevated human heat stress and reduced safe outdoor working hours. Here we show deforestation-induced local warming is associated with 28,000 (95% confidence interval: 23,610–33,560) heat-related deaths per year using a pan-tropical assessment. Analysis of satellite data shows tropical deforestation during 2001–2020 exposed 345 million people to local warming with population-weighted daytime land surface warming of 0.27 °C. Estimated heat-related mortality rates are greatest in Southeast Asia (8–11 deaths for every 100,000 people living in deforested areas) followed by tropical regions of Africa and the Americas. In regions of forest loss, local warming from deforestation could account for over one third of total climate heat-related mortality, highlighting the important contribution of tropical deforestation to ongoing warming and heat-related health risks within the context of climate change. Over recent decades, tropical forests have faced extensive deforestation and degradation1,2, driven primarily by agricultural expansion and logging3. The impacts of tropical forest loss are profound, affecting biodiversity4, global climate5,6 the hydrological cycle7 and human communities8,9,10,11,12. Previous studies have demonstrated a strong association between tropical forest loss and increases in surface temperature on both local6,12,13,14,15,16,17,18,19 and regional20,21 scales. The local warming associated with deforestation can be immediate and of substantial magnitude22, equivalent to or larger than that projected from a century of global climate change under a high emissions scenario12,20. Human exposure to elevated temperatures presents a major potential health risk. Heat stress can negatively affect mood and mental health23, impair physical performance24 and reduce labour productivity25,26. Furthermore, exposure to heat is associated with an increased risk of morbidity and mortality from cardiovascular diseases and other causes27,28. Considerable heat-related mortality has been attributed to recent climate change29. The effects of climate change on human health are compounded by socioeconomic and demographic factors30. Population vulnerability to climate change is linked to health expenditure and proportions of obese and elderly populations31. Along with these factors, heat-related human health effects can be modulated by the degree of technological adaptation32, which can be strongly linked to socioeconomic status33. In lower-income countries with limited adaptive capacity, including many countries in the tropics, heat-stress-related labour capacity losses may have substantial economic consequences34 and increase poverty35. Tropical deforestation is associated with increases in human heat exposure12,22,36. In field experiments, deforestation-induced heat exposure has been demonstrated to reduce cognitive performance37 and labour productivity26. Across the tropics, local warming from deforestation between 2003 and 2018 was associated with losses in safe thermal working conditions for 2.8 million outdoor workers12. Little is known about the potential for deforestation-induced warming to lead to additional deaths at the pan-tropical scale. Wolff et al.10 showed that in Berau Regency in Indonesia, the combination of global climate change and deforestation between 2002 and 2018 increased population-weighted mean temperatures by 0.86 °C, accounting for an estimated 7.3–8.5% of all-cause mortality (or 101–118 additional deaths per year). Wider assessments of the population-weighted warming due to deforestation or the potential impacts on health and mortality are lacking but are urgently needed to inform land use policy and climate adaptation strategies. Here, we make the first pan-tropical assessment of the population-weighted warming due to tropical deforestation and the associated heat-related mortality burden. We focus our analysis on tropical deforestation that occurred from 2001 to 2020. We use spatially explicit satellite datasets of annual tree cover change1 to identify forest loss at a ~1 km2 spatial scale. We use satellite datasets of land surface temperature38 to quantify surface warming that has occurred over this period. Through comparing land surface temperature change over locations of forest loss to neighbouring locations without forest loss, we isolate the warming due to deforestation. We then use human population distribution data39 to map population-weighted exposure to this warming. Finally, we use data on non-accidental mortality40 combined with relationships between heat exposure and excess mortality from the literature31 to estimate the heat-attributable excess mortality due to nearby tropical deforestation. Our analysis provides important evidence of the negative potential human health impacts of tropical deforestation at local, regional and national scales. During 2001–2020, a total of 1.6 million km2 of forest area (with greater than 10% forest canopy cover) was lost across the tropics. The greatest forest loss occurred across Tropical Central and South America (~760,000 km2), with extensive forest loss also occurring across Southeast Asia (~490,000 km2) and Tropical Africa (~340,000 km2). Surface temperatures in the tropics have generally increased over this time period due to a combination of global climate change and deforestation, with regional annual mean warming (ΔT) of +0.34 °C in Tropical Central and South America, +0.10 °C in Tropical Africa and +0.72 °C in Southeast Asia. Areas of forest loss coincide with areas of strong positive ΔT across many regions of the tropics (Fig. We defined 1 km2 pixels that experienced at least two percentage points of forest loss during 2001–2020 as pixels that experienced deforestation (Methods). A total of 60% of pixels with warming of greater than four standard deviations from the mean (4.4 °C) experienced deforestation during 2001–2020, while deforested pixels make up only 17% of pixels in the tropics. The average ΔT in deforested locations across the tropics (+0.70 °C) exceeds warming that has occurred over areas that have maintained forest cover (+0.20 °C) (Table 1). Regionally, average ΔT in deforested locations (+0.73 °C in Tropical Central and South America, +0.75 °C in Tropical Africa and +0.61 °C in Southeast Asia) are 2–9× the average ΔT in locations that maintained forest cover (+0.30 °C in Tropical Central and South America, +0.08 °C in Tropical Africa and +0.31 °C in Southeast Asia). Similar to previous studies10,12,17, we assess the warming due to deforestation using land surface temperature data because near-surface air temperature data are not available at sufficiently high spatial resolution in the tropics. We compared land surface temperature and near-surface air temperature from the European Centre for Medium-Range Weather Forecasts Reanalysis v5 (ERA5 (ref. 41)) over the same period and find they agree to within 2% (Extended Data Table 1), confirming our approach is justified (Methods). a, The percentage-point change in forest cover from 2001 to 2020. b, The change in annual mean land surface temperature (ΔT) between the 2001–2003 mean and the 2018–2020 mean. c, The spatial distribution of population count in 2020. To estimate the ΔT due to deforestation only, excluding the contribution of global climate change and regional climate variability, we compared ΔT over locations of forest loss with neighbouring locations without forest loss. Over 2001–2020, deforestation is associated with 0.45 °C of warming on average across deforested areas of the tropics (Table 1). The regional mean deforestation-induced ΔT is greatest in Tropical Central and South America (+0.53 °C), with slightly lower ΔT in Tropical Africa (+0.39 °C) and Southeast Asia (+0.37 °C). This means that deforestation accounts for over half of the regional average warming in deforested areas of the tropics (and ~70% in Tropical Central and South America). In general, the regions with the greatest deforestation-induced warming correspond to the regions with the greatest forest loss (Fig. 2), particularly in the Arc of Deforestation in the southern Amazon and Sumatra and Kalimantan in Indonesia. Strong deforestation-induced warming is also visible in other areas that have experienced extensive forest loss1, including in Central America (Guatemala and Nicaragua), southern West Africa (Cote d'Ivoire, Ghana and Nigeria), Central and Eastern Africa (Democratic Republic of the Congo, Uganda and Tanzania) and Mainland Southeast Asia (Cambodia and Vietnam). a–f, The maps show regions of Tropical Central and South America (a and b), Tropical Africa (c and d), and Southeast Asia (e and f). The maps in a, c and e show the deforestation-induced ΔT at the ~1 km2 pixel level. The maps in b, d and f show the population-weighted mean deforestation-induced ΔT for second-level administration divisions for populations located in areas of forest loss. The division boundaries are from GADM (https://gadm.org/index.html). The areas with no data points are shown in white. This local warming associated with deforestation has important implications for large populations that live across tropical forest regions (Fig. The tropics (25° S – 25° N) are home to over 3.5 billion people, with 13% (452 million people) living in regions that have experienced forest loss during the 2001–2020 period. We estimate that 345 million people were exposed to warming due to deforestation during 2001–2020 (Table 2), with substantial numbers of people exposed to deforestation-induced ΔT of greater than +1 °C (33 million people), +2 °C (8.0 million people) and +3 °C (2.6 million people). Across the tropics, 76% of people living in regions of tropical forest loss were exposed to deforestation-induced warming, with a similar proportion in the different tropical regions (74–80%). In Tropical Africa, 148 million people were exposed to warming from deforestation, compared with 122 million people in Southeast Asia and 67 million people in Tropical Central and South America. Over 2001–2020, the populations in areas of tropical forest loss were exposed to a mean ΔT of +0.27 °C due to deforestation (Table 1). Regional population-weighted mean ΔT is greatest in Tropical Africa (+0.32 °C), where deforestation-induced warming overlaps with more heavily populated areas, and lowest in Tropical Central and South America (+0.22 °C), where deforested areas with strong warming are more sparsely populated (Fig. Across all regions, population-weighted warming due to deforestation is less than area-weighted warming (Table 1) due to lower population densities in rural areas, which typically experience greater forest loss (Extended Data Figs. We estimated the heat-related excess mortality burden associated with deforestation by combining our estimates of deforestation-induced ΔT with region-specific heat vulnerability indices31 and non-accidental mortality rates (including mortality from all causes apart from external causes)40. In general, areas with the greatest mortality burden correspond to regions with moderate to high levels of forest loss and deforestation-induced warming (Fig. However, because the mortality estimates also depend on other factors (population count (Fig. 1c), non-accidental mortality rates and population heat vulnerability (Extended Data Table 2)), the spatial pattern differs from that of the deforestation-induced ΔT (Fig. The spatial distribution of the deforestation-associated heat-related mortality rate (that is, deaths per 100,000 people) (Extended Data Fig. 4) is more consistent with deforestation-induced ΔT (Fig. 2), demonstrating that the heat-related mortality burden is strongly dependent on the magnitude of the population within areas of forest loss. a–c, The maps show regions of Tropical Central and South America (a), Tropical Africa (b), and Southeast Asia (c). The colours show the number of deaths per year (central estimate) for populations located in areas of forest loss, aggregated by second-level administration divisions (boundaries from GADM, https://gadm.org/index.html). Note that the colour scales are different for each region. The administration divisions containing no data points are shown in white. Overall, we estimate that warming due to deforestation over 2001–2020 is associated with an additional 28,330 (95% confidence interval (CI): 23,610–33,560) non-accidental deaths per year (Table 2). The estimated deforestation-associated heat-related mortality burden is greatest in Southeast Asia (15,680 (95% CI: 13,000–18,470) annual excess deaths) due to a relatively large exposed population, predominantly in Indonesia, with greater heat vulnerability. In Tropical Africa we estimate 9,890 (95% CI: 8,250–11,850) annual excess deaths associated with deforestation. This is lower than in Southeast Asia, despite greater population exposure to deforestation-induced warming and higher underlying non-accidental mortality rates, because of the lower heat vulnerability that we have assumed for populations in this region (Methods). Although areas of forest loss in Tropical Central and South America experience some of the largest deforestation-induced warming in the tropics, populations in these areas are relatively low, leading to lower population exposure and heat-related mortality (2,520 (95% CI: 2,160–2,950) annual excess deaths) than in the other tropical regions. Tropical forest regions have warmed substantially over the last two decades due to a combination of climate change and land-use change. We show that over the period 2001–2020, tropical deforestation has caused annual mean land surface warming of 0.7 °C, in line with previous assessments12,13,14,15,16,17,18. In our analysis, deforestation-induced warming accounts for 64% of the total warming experienced over regions with tropical forest loss (Table 1), demonstrating that deforestation is a major driver of local warming. Deforestation caused 39% of the population-weighted warming experienced over regions with tropical forest loss (Table 1). Our findings suggest that warming from tropical deforestation impacts large numbers of people and could result in a substantial health burden. We estimate that 345 million people were exposed to local warming from forest loss during 2001–2020. We estimate this warming is associated with an annual heat-related mortality burden of 28,330 (95% CI: 23,610–33,560), equivalent to 39% of the total heat-related mortality (from global climate change and deforestation combined) over locations of forest loss (Table 2). The Global Burden of Disease (GBD) Study42 estimated annual excess deaths attributable to high temperature in 2019 to be 14,400 (95% CI: 10,800–18,300) in Latin America and the Caribbean, 50,800 (95% CI: 36,400–66,300) in Sub-Saharan Africa and 41,200 (95% CI: 34,600–49,300) in Southeast Asia. We estimate the total annual heat-related excess deaths (associated with warming from both global climate change and deforestation during 2001–2020) for forest loss regions to be 5,800 (95% CI: 5,000–6,700) in Tropical Central and South America, 28,000 (95% CI: 23,300–33,800) in Tropical Africa and 37,000 (95% CI: 30,700–43,600) in Southeast Asia. Calculating these mortality burdens for the total population, our estimates are comparable with those from the GBD for Latin America and the Caribbean and Sub-Saharan Africa but notably higher for Southeast Asia. However, large differences in methodology, timescale, spatial resolution and temperature datasets complicate direct comparison. Notably, GBD estimates rely on ERA5 (ref. 41) near-surface air temperature, which lacks recent and dynamic land cover data (using a monthly climatological vegetation map for the years 2000–200843). Therefore, in regions with sparse weather station coverage, ERA5 does not account for land cover change impacts on local temperature and does not capture increases in temperature with increasing deforestation44. Heat-related mortality from deforestation accounts for 1.1% of non-accidental mortality over regions of tropical forest loss, increasing to 1.6% over Southeast Asia (Table 2). Wolff et al.10 estimated that warming from the combination of global climate change and deforestation during 2002–2018 accounted for 7.3–8.5% of all-cause mortality in the Berau Regency in East Kalimantan, Indonesia. In comparison, we estimate warming from global climate change and deforestation during 2001–2020 accounts for 6.7% of all-cause mortality in Berau Regency and 8.0% for populations in locations of forest loss (Extended Data Table 3). We estimate greater population-weighted ΔT, due to the inclusion of more years of surface temperature data, but we apply a lower heat vulnerability index to be conservative and to be consistent with the regional average value for Southeast Asia from Lee et al.31. We estimate that warming from deforestation during 2001–2020 accounts for 2.0% of all-cause mortality in areas of forest loss in Berau Regency and 34.5% of the total heat-attributable mortality (from global climate change and deforestation combined). We find that 42% of the tropics-wide heat-related mortality burden associated with deforestation occurs in regions which had greater than 50% forest canopy cover in 2001 compared with 58% in regions of intermediate canopy cover (10–50%) (Extended Data Table 4). Most forest loss occurs in regions that have already been partially deforested such as the Arc of Deforestation (Fig. 1a), which also overlap regions with greater population density (Fig. 1c) and heat-related mortality (Fig. In Africa, forest loss in areas of intermediate canopy cover accounts for 70% of the heat-related mortality burden associated with forest loss. This highlights the impact of forest loss within regions of naturally occurring intermediate canopy cover such as the miombo woodlands of Africa and the Cerrado and Chaco in South America. Our work highlights the importance of sustainable management of these biomes to reduce potential negative health impacts on local populations. To contextualize the heat-related mortality burden, it is important to consider other health risks associated with tropical deforestation. Smoke pollution from deforestation-related fires can degrade regional air quality45,46 leading to adverse health impacts47,48. In 2015, peatland fires in Indonesia resulted in exposure of 69 million people in Equatorial Asia to unhealthy air quality conditions49. Long-term exposure to particulate air pollution from forest and vegetation fires is estimated to cause ~3,000–16,800 premature deaths annually in South America50,51,52, ~6,000–59,000 in Southeast Asia49,53,54 and ~43,000 in Africa55. Our estimates of the heat-related mortality burden associated with deforestation-induced warming are comparable with the lower end of estimates of these fire-related mortality estimates. Tropical deforestation is also linked to increased malaria risk in some contexts11,56, with evidence suggesting it is an important driver of childhood malaria risk in poor households57. With deforestation-driven malaria incidence estimates only available for the Brazilian Amazon11, we compare our results to national-level malaria mortality estimates from the GBD42, noting that these may underestimate the burden in deforested areas. In 2019, malaria accounted for 0.04% (95% CI: 0.02–0.07%) of total deaths in Latin America and Caribbean, 7.98% (95% CI: 3.24–15.17%) in Sub-Saharan Africa, and 0.04% (95% CI: 0.01–0.16%) in Southeast Asia42. We estimate that heat-related mortality associated with deforestation during 2001–2020 accounts for a smaller proportion of total deaths than malaria in Tropical Africa (0.13%) but greater proportions in Tropical Central and South America (0.08%) and Southeast Asia (0.23%). These results suggest that the deforestation-associated heat-related mortality burden is comparable with other major health burdens linked to tropical deforestation in the Americas and Southeast Asia. There are no country-specific heat vulnerability indices available for African countries due to lack of data31, and so we have used the continental-average heat vulnerability index for South America. Lee et al.31 report that the heat vulnerability index shows a large negative dependence on the total health expenditure per capita and a large positive dependence on proportions of obese and elderly populations. A comparison of the key factors that affect heat vulnerability31 for Sub-Saharan Africa and other regions and countries (Extended Data Table 5) suggests that using the South American heat vulnerability index for African countries may be conservative. Increasing the heat vulnerability index used for African countries from the South American index (2.34%p °C−1) to the Southeast Asian index (5.29%p °C−1) more than doubles the deforestation-associated heat-attributable mortality burden estimate for Tropical Africa. Our analysis highlights the critical need for increased climate and health data to improve understanding of heat-health relationships in tropical African countries and other understudied tropical regions. Our study focused on the local biophysical warming impacts of deforestation (that is, warming within the same 1 km2 pixel as the forest loss). Butt et al.21 demonstrated that deforestation of the Amazon caused strong warming at distances up to 100 km away from the forest loss. We do not include potential health effects of associated regional temperature changes on populations located outside deforested regions, suggesting we may have underestimated the impacts on human health. Tropical deforestation also releases CO2 that contributes to global warming. During 2014–2023, deforestation released 1.7 GtC yr−1, accounting for 15% of total anthropogenic CO2 emissions58. The global warming from release of CO2 is not included in our study, meaning that our estimates of heat-related mortality associated with deforestation are likely to be conservative. We estimate that non-deforested regions of the tropics have warmed by 0.2 °C over 2001–2020 (Table 1). Based on the contribution to global emissions, CO2 from tropical deforestation may have accounted for 0.03 °C (15%) of this warming, which is substantially less than the biophysical warming (0.45 °C) in regions of tropical deforestation. Nevertheless, the warming from CO2 emissions acts globally and the contribution of tropical deforestation to the global burden of heat-related mortality29 may be substantial. This highlights the strong potential for reducing tropical deforestation to contribute to reduced warming at both local and global scales. Future global climate change is projected to increase heat-related mortality in the tropics31,59 and severely impact outdoor worker wellbeing and health60. Tropical deforestation and its associated local warming are likely to amplify these impacts. Climate model simulations indicate that cropland expansion into tropical forests could elevate local near-surface air temperature and increase heat stress in low-latitude regions61. In deforested areas of the tropics, future global climate change is projected to decrease safe outdoor working hours for millions of people12. In the Amazon region, scenarios of future climate warming and deforestation project substantial increases in heat stress and reductions in work capacity62. Furthermore, compound climate extremes including drought-heatwave episodes, which may be associated with enhanced fire activity and heat-related mortality in Brazil63, are increasing in a warming world64. In 2023 and 2024, critical episodes of prolonged drought occurred in the Brazilian Amazon, illustrating how climate change intensifies health threats through direct impacts, such as heatwaves and water scarcity, and indirect impacts, such as air pollution from fires. Lower-income populations in the tropics are already experiencing a greater increase in the frequency of extreme temperatures compared with higher income populations65 and are projected to be disproportionately impacted under future global warming66. Tropical nations have particularly high heat vulnerability indices, suggesting their populations may be at higher mortality risk due to climate change31. This higher heat vulnerability is linked to lower per-capita health expenditure31, reflecting persistent socioeconomic disparities in healthcare access67. Vulnerable populations, particularly traditional and indigenous communities, often live near deforested areas and face limited access to resources and infrastructure needed to cope with the combination of rising temperatures and environmental changes caused by deforestation and climate change62. Our findings suggest that for populations in areas of tropical forest loss, deforestation-driven warming may exacerbate the heat-related health effects of global climate change, with critical implications for outdoor labour10,12. Furthermore, these populations may also suffer from disproportionate exposure to other deforestation-related health risks, such as malaria57 and fire-sourced air pollution54. Our results underscore the role of deforestation in intensifying local warming within the broader context of climate change. Conservation of tropical forests can help mitigate local warming and strengthen the adaptive capacity of vulnerable populations. Future work should explore how protected areas function as buffers against temperature extremes, particularly for surrounding populations. Overall, our findings highlight the urgent need for targeted policy interventions to reduce tropical forest loss, alongside improved adaptation strategies and access to healthcare, to protect vulnerable populations in the tropics from the health risks associated with deforestation. As in previous work17,21, we used land surface temperature (LST) data from NASA MODIS, specifically the MOD11A2 8-d LST data (here using the latest version 6.1)38 at 0.01° × 0.01° spatial resolution. We excluded data where the estimated emissivity error was greater than 0.02 and where the LST error was greater than 1 K, following Li et al.13. We used MODIS data from the Terra satellite (10:30 local overpass time) because of its longer sampling period compared with the Aqua satellite (February 2000 onwards versus July 2002 onwards) (13:30 overpass time) and due to lower cloud cover in the morning. We first aggregated the 8-day LST data by month ignoring any 8-day period where data were missing due to clouds or because of the quality screening process. We then calculated 3-year means for two periods at the start (2001–2003) and end (2018–2020) of the study period. Using 3-year averages reduces the influences of climate variability. The LST dataset was regridded using bilinear interpolation (using Python package xESMF68) to match the finer resolution of the population data grid (30 arc-seconds). The ΔT in multiannual mean LST was then calculated by subtracting the 2001–2003 mean LST from the 2018–2020 mean LST. Our final ΔT dataset comprised over 64 million pixels (~1 km2 in extent) over the tropics (25° S to 25° N). To assess the impacts of using morning LST data in place of daily mean air temperature data (as used by Lee et al.31 to derive the heat vulnerability indices), we used 2-m temperature (t2m) and LST data from the ERA5 (ref. We downloaded the hourly Analysis-Ready, Cloud Optimized (ARCO) ERA5 data from the Google Cloud Public Dataset Program69 for the years 2001–2003 and 2018–2020. First, we calculated daily mean t2m and extracted the ERA5 LST data at 10:00–11:00 local time. Second, we aggregated both variables by month and calculated 6-year means, regridding to match the population data grid (30 arc-seconds). Third, we calculated the relationship between ERA5 LST and ERA5 t2m over tropical forest pixels (using 2001 forest cover) of the main tropical forest regions (Extended Data Table 1). Forest cover data were taken from the Global Forest Change (GFC) V1.9 dataset1 at 30 m × 30 m spatial resolution. Annual forest cover for the period 2000–2020 was calculated by taking tree cover in the year 2000, defined as canopy closure for all vegetation taller than 5 m in height, and subjecting it to annual forest loss, defined as a disturbance from a forest to non-forest state. Forest cover was calculated at the native 30 m × 30 m spatial resolution and then converted to match the resolution of the population data by first calculating the forest cover fraction in each larger pixel (0.008° × 0.008°) and then converting to a 30 arc-second resolution using area-weighted regridding (using Python package Iris70). The percentage-point change in forest cover between 2001 and 2020 was calculated as the difference between the annual forest cover fractions (in per cent) in 2001 and 2020. Forest cover gain can lead to cooling that can offset some of the warming due to forest loss71. To identify and exclude areas where forest regrowth has occurred, we used forest extent data at 30 m × 30 m spatial resolution from the Global Land Analysis and Discovery (GLAD) Global Land Cover and Land Use Change dataset72,73, available for the years 2000 and 2020. In the forest extent dataset, forest presence is indicated for pixels with 5 m or greater forest height. We converted this dataset to match the resolution of the population data using the same method as for the GFC data and calculated the percentage-point difference in forest extent between 2000 and 2020. We then removed any 30-arc-second pixels with greater than a 50%-point net increase in forest extent between 2000 and 2020 from the ΔT data. We used elevation data from the Global Multi-resolution Terrain Elevation Data (GMTED2010)74 at 7.5-arc-second spatial resolution, which was regridded using bilinear interpolation to match the resolution of the population data. Spatially explicit population data for the year 2020 was taken from LandScan75,39 at 30-arc-second spatial resolution representing an average population over 24 h (Fig. We used LandScan data for 2001 to test the sensitivity of our results to changing population density and distribution (‘Methodological limitations and justification' section). To explore exposure of rural and urban populations, we used the Settlement Model Layers data from the Global Human Settlement Layer (GHSL) 2023 Data Package76,77. Annual all-cause mortality and non-accidental mortality rates for all ages for the year 2019 were taken from the Global Burden of Disease Study (GBD)40. Non-accidental mortality (or non-external mortality) includes mortality from all causes except external causes such as accidents, suicides and homicides. We used data from 2019 instead of 2020 to exclude possible effects of coronavirus disease 2019 on the mortality rates. We used the highest resolution data available from the GBD40: province/state level data for Brazil and Indonesia and national-level data for all other countries in the tropics. We used GBD non-accidental mortality rates from 2001 to test the sensitivity of our results to changes in the annual mortality rates (‘Methodological limitations and justification' section). To estimate total annual all-cause and non-accidental mortality burdens at the pixel level, we multiplied the GBD mortality rates by the gridded total population data (pop) where nonacc_mortratecountry is the national (or subnational) non-accidental mortality rate and nonacc_morti is the total annual non-accidental mortality burden per pixel (i). To estimate the heat-related excess mortality burden, we used country- or continent-specific heat vulnerability indices from Lee et al.31 following Wolff et al.10. The heat vulnerability index is the percentage-point increase of regional excess mortality per 1 °C increase of regional temperature (%p °C−1). Lee et al.31 first estimated location-specific temperature-mortality relationships for 459 locations in 28 countries using daily mean temperature, then estimated the projected excess mortality attributable to temperature change over 2010–2099 using a calibrated multimodel mean temperature time-series under different scenarios. The heat vulnerability indices were obtained for each location by applying linear regressions to the projection data31. The heat vulnerability indices were estimated by Lee et al.31 for 10 continental regions and 28 countries; those used in this study are shown in Extended Data Table 2. Where country-specific indices were not available, we applied a continent-specific value. Following Lee et al.31, for countries located in Central and South Asia, we applied the heat vulnerability index from Iran; for countries located in Central America, we applied the index from Mexico. Lee et al.31 were unable to derive indices for African countries due to a lack of data; therefore, we have applied the lowest continental-average index (from South America) for the whole of the Tropical Africa. We estimated the local ΔT due to forest loss between 2001 and 2020 at the 30-arc-second (~1 km2) pixel level across the tropics (25° S to 25° N). To remove the influence of global climate change over the study period on ΔT, we used a moving-window nearest-neighbour approach7,17. In our analysis we used all 30-arc-second pixels within the tropics (25° S to 25° N) with 10% or greater forest cover fraction in 2001. Pixels with less than 0.5%-point forest cover loss between 2001 and 2020 were classified as ‘non-deforested' pixels. Pixels with 2%-point or greater forest cover loss between 2001 and 2020 were classified as ‘deforested' pixels. Our final ΔT dataset comprised ~16.8 million non-deforested pixels and ~11.3 million deforested pixels across the tropics (25° S to 25° N). For each deforested pixel, we selected all surrounding pixels that maintained forest cover (‘non-deforested' pixels) within a circle of 0.25° radius (~27 km at the equator) and within an elevation of ±50 m. If no surrounding non-deforested pixels were available within the 0.25° radius circle (3% of pixels), the radius of the circle was extended to 0.50°, with the same elevation constraints of within ±50 m. We used 0.25° and 0.5° radius circles similar to those used in previous work (for example, refs. These previous studies—and ours—assume the background climate to be uniform over this distance. We calculated the mean ΔT over all selected non-deforested (‘control') pixels and subtracted this from the ΔT of the central deforested pixel to get the deforestation-induced ΔT. A total of 0.58% of deforested pixels across the tropics were without surrounding non-deforested pixels within the specified spatial circle and elevation range; these pixels were retained in the data array but stored with a missing data indicator. We then divided the resulting deforestation-induced ΔT data array by the percentage-point change in forest cover to calculate the change in temperature per percentage-point forest cover loss. At low levels of forest cover loss (<10%-point), the variability in ΔT per percentage-point forest cover loss exceeds the range in values found at greater levels of forest cover loss. To reduce this variability, we used a moving average smoothing approach. For each deforested pixel, we calculated the mean ΔT per percentage-point forest cover loss over all surrounding deforested pixels within a 0.10° rolling circle, excluding any pixel with less than 5%-point forest cover loss or with a z-score greater than 3 from the mean. For deforested pixels without surrounding deforested pixels with ≥5%-point forest cover loss within the 0.10° spatial circle (0.4% of all deforested pixels across the tropics), we used a regional mean value for ΔT per percentage-point forest cover loss. Finally, we multiplied the ‘smoothed' pixel-level ΔT-per-percentage-point-forest-cover-loss values by the pixel-level percentage-point change in forest cover to obtain a more robust estimate of the pixel-level deforestation-induced ΔT. To estimate heat-related excess mortality, we followed the approach of Wolff et al.10, using the relationships between heat-attributable excess mortality and temperature estimated by Lee et al.31. Wolff et al.10 estimated heat-related mortality due to global climate change and deforestation combined. In this study, we estimated the heat-related mortality due specifically to local deforestation-induced warming. First, we calculated the pixel-level deforestation-induced ΔT over 2001–2020 in all deforested pixels across the tropics (as described above). Second, to estimate the percentage-point increase in heat-attributable excess mortality (%pincrease_mort) per deforested pixel (i), we multiplied the country/continent-specific heat vulnerability indices (HVI) by the pixel-level ΔT in all deforested pixels with a positive temperature change (79% of all deforested pixels across the tropics) (ΔTdefor) The mortality impact of the 2001–2020 deforestation-induced warming will already be included in the non-accidental mortality burden data from 2019 (‘Population and mortality datasets' section). Thus, following Wolff et al.10, we next calculated the corrected counterfactual mortality burden (CF_mort) (that is, without the excess mortality associated with the deforestation-induced ΔT that would already be included in the total non-accidental mortality burden) per deforested pixel (i) Lastly, we calculated the mortality attributable to warming from deforestation for each deforested pixel, as the difference between the counterfactual non-accidental mortality and total non-accidental mortality The results are estimated using annual non-accidental mortality rates and therefore represent an annual excess mortality burden associated with exposure to a ~20-year temperature change. Following Wolff et al.10, we refer to the heat-health impact results as an annual excess heat-attributable mortality burden. The uncertainty range in the heat-attributable mortality estimates was calculated based on the uncertainty range (the 95% CI) of the GBD all-cause/non-accidental mortality rates40. For comparison, we also estimated the total heat-related mortality attributable to warming over 2001–2020 from global climate change and deforestation combined. This was done using the same method as above but replacing the deforestation-induced ΔT with the total ΔT. We note that where a deforested pixel experiences warming but surrounding non-deforested (‘control') pixels experience an overall cooling, the deforestation-induced ΔT will be greater than the total ΔT, and the number of deforestation-associated heat-related deaths will exceed the number of total heat-related deaths in that pixel. We defined forested locations in the tropics as pixels with 10% forest canopy cover of vegetation taller than 5 m in height, which is consistent with the definition from the Food and Agriculture Organization of the United Nations78. We defined deforestation to be a net loss in forest cover of 2%-points or greater between 2001 and 2020. We accounted for dynamics of forest cover change within the given time period, by using the GLAD forest extent dataset to remove pixels with a >50%-point net increase in forest extent between 2000 and 2020. We tested the sensitivity to these assumptions. Defining deforestation as net forest loss greater than 1%-point, increased the number of data pixels and population included in our analysis, increasing our estimated mortality burden by 9% (Extended Data Table 4), but remains within our cited uncertainty range. Applying a lower threshold to the net increase in forest extent (20%-point) reduced the population exposed to deforestation-associated warming by only 1% (Extended Data Table 4). We analysed the change in forest cover and LST between 2001 and 2020. The years of data included in the analysis were selected to maximize the study period with the available data, while also avoiding strong ENSO years in our 3-year averages of the LST data (2001–2003 and 2018–2020). We note that our analysis method attempts to remove the impact of climate variability through comparing temperature change over regions of forest loss with nearby regions with no forest loss. For example, during El Nino periods, both regions of forest loss and regions of no forest loss will experience a warming due to the impacts of El Nino. Our analysis pulls out the impact of forest loss during the El Nino by subtracting the temperature of surrounding forested pixels. We tested the sensitivity to the selected years of LST data by recalculating for a different 3-year period (from 2003–2005 to 2018–2020), 5-year periods (from 2001–2005 to 2016–2020) and for single years (2001–2019 and 2001–2020). Estimated deforestation-associated heat-related mortality burdens for these different time periods lie within our 95% CIs (varying by −13% to +15%) (Extended Data Table 4), showing that our method is robust to these assumptions. Averaging the forest loss data to match the same years as the LST data (from 2001–2003 to 2018–2020) reduces the heat-related mortality burden associated with deforestation by only 2% (Extended Data Table 4). We used LST data to estimate human exposure and heat-related health impacts of deforestation-induced warming rather than near-surface air temperature, because air temperature data are not available at sufficiently high spatial resolution and coverage for the study time period. The use of LST data for exploring fine-scale temperature changes under tropical forest loss has been discussed and justified in previous studies10,12,17. For tropical forest regions, there is a particularly uneven and sparse distribution of ground-based meteorological stations79,80, which means that air temperature measurements from these stations (or datasets that spatially interpolate these measurements) are not appropriate for a tropics-wide assessment of fine-scale deforestation-induced temperature changes. Where meteorological stations exist, previous studies have shown a tight connection between LST and near-surface air temperature14,81,82,83. Estimates of near-surface air temperature from climate reanalyses, such as the ERA5 (ref. 41), have been used extensively to estimate human exposure to extreme heat and the associated health impacts40,84,85,86. While the relatively coarse resolution of ERA5 data make it unsuitable for examining fine-scale temperature changes driven by deforestation, it is reported to well capture changes in air temperature where meteorological station density is high79. To assess the impacts of using LST data in place of air temperature data, we used the relationship between ERA5 daily mean t2m and ERA5 10:00–11:00 local time LST (Extended Data Table 1) to adjust MODIS LST to daily mean 2-m air temperature. We then recalculated the impacts of forest loss using this adjusted ‘air temperature data', which reduces the deforestation-associated heat-attributable mortality burden by only 1% (Extended Data Table 4). We used morning measurements of LST from MODIS-Terra for its greater data availability compared with MODIS-Aqua (an overpass of 13:30 local time). Furthermore, MODIS-Terra tends to sense a cooler land surface than MODIS-Aqua and is therefore more representative of the daily mean temperature data used to derive the heat vulnerability indices by Lee et al.31. Using afternoon LST data from MODIS-Aqua (for 2003–2005 mean to 2018–2020 mean) increased the population-weighted warming associated with deforestation to +0.29 °C and increased the estimated deforestation-associated heat-attributable mortality burden to 30,590 (95% CI: 25,520–36,230) (Extended Data Table 4). This mortality burden is 19% greater than with MODIS-Terra LST for the same period and within our stated 95% CI. In this work, we have not assessed the potential impact of deforestation on humidity. In situ measurements87,88 and reanalysis data89 show a general reduction in humidity in cleared areas of the tropics compared with forested areas, although this difference can be seasonally dependent. Results of a modelling study suggest that changes in humidity under simulated cropland expansion into low-latitude forested areas could moderate combined impacts on moist heat and heat stress61. However, Masuda et al.26 and Parsons et al.12 show that despite potential deforestation-induced changes in humidity, humid heat exposure increases substantially in areas of tropical forest loss compared with nearby forested areas, negatively impacting outdoor workers. There is a strong need for field measurements of humidity in tropical forest areas to enable further research into impacts of deforestation on humidity and the associated health impacts Following Wolff et al.10, we have not included an estimation of the change in cold-related mortality under higher temperatures. For tropical countries, the reduction in cold-related mortality under increasing temperatures is small and is outweighed by the associated increase in heat-related mortality31. The heat vulnerability indices were developed by Lee et al.31 using either non-accidental or all-cause mortality rates (Extended Data Table 2), depending on the data available for each country. For simplicity we report results estimated using non-accidental mortality rates. Using all-cause mortality rates in all locations increases the total deforestation-associated heat-attributable mortality burden across the tropics by ~8% (with a 15% increase in Tropical Central and South America where differences between non-accidental and all-cause mortality rates are greatest). We report heat-related mortality estimates that are relevant for a near-present-day population, using population count and spatial distribution for 2020 and non-accidental (or all-cause) mortality rates for 2019. During 2001–2020, population count has increased across the tropics, while changes in non-accidental mortality rates are mixed. Using population count and spatial distribution and non-accidental mortality rates for the start of the study period (2001) reduces the total deforestation-associated heat-attributable mortality burden across the tropics by ~10%. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The datasets used in the analyses for this study are freely available to download from the following locations: MODIS land surface temperature data at https://www.earthdata.nasa.gov/; European Centre for Medium-Range Weather Forecasts Reanalysis v5 (ERA5) Analysis-Ready, Cloud Optimized (ARCO) dataset on GitHub at https://github.com/google-research/arco-era5; Global Forest Change (GFC) dataset at https://storage.googleapis.com/earthenginepartners-hansen/GFC-2023-v1.11/download.html; Global Land Analysis and Discovery (GLAD) Global Land Cover and Land Use Change dataset at https://glad.umd.edu/dataset/GLCLUC2020; Global Multi-resolution Terrain Elevation Data (GMTED2010) at https://earthexplorer.usgs.gov/; LandScan population data at https://landscan.ornl.gov/; Global Human Settlement Layer dataset at https://human-settlement.emergency.copernicus.eu/ghs_smod2023.php; GBD Study cause-specific mortality rates at https://gbd2019.healthdata.org/gbd-results/ or https://vizhub.healthdata.org/gbd-results/; and digital geospatial data for plotting administrative boundaries from Database of Global Administrative Boundaries (GADM) at https://gadm.org/index.html. The processed data files produced and used in the analyses of this study are available through Code Ocean90. The data were analysed using open-source Python packages (including xESMF68, Iris70, xarray91, NumPy92, Matplotlib93, Pandas94 and Cartopy95) that are freely available online. The Python code used to produce the central results of this study is available as a Compute Capsule through Code Ocean90. Any other Python code used is available upon request from the corresponding author. Hansen, M. C. et al. High-resolution global maps of 21st-century forest cover change. Vancutsem, C. et al. Long-term (1990–2019) monitoring of forest cover changes in the humid tropics. Pendrill, F. et al. Disentangling the numbers behind agriculture-driven tropical deforestation. Global biodiversity loss from tropical deforestation. The tropical forest carbon cycle and climate change. Lawrence, D., Coe, M., Walker, W., Verchot, L. & Vandecar, K. The unseen effects of deforestation: biophysical effects on climate. & Spracklen, D. V. Tropical deforestation causes large reductions in observed precipitation. Pienkowski, T., Dickens, B. L., Sun, H. & Carrasco, L. R. Empirical evidence of the public health benefits of tropical forest conservation in Cambodia: a generalised linear mixed-effects model analysis. Wolff, N. H., Masuda, Y. J., Meijaard, E., Wells, J. & Game, E. T. Impacts of tropical deforestation on local temperature and human well-being perceptions. Wolff, N. H. et al. The effect of deforestation and climate change on all-cause mortality and unsafe work conditions due to heat exposure in Berau, Indonesia: a modelling study. & Mordecai, E. A. Amazon deforestation drives malaria transmission, and malaria burden reduces forest clearing. Tropical deforestation accelerates local warming and loss of safe outdoor working hours. Li, Y. et al. Local cooling and warming effects of forests based on satellite observations. Alkama, R. & Cescatti, A. Biophysical climate impacts of recent changes in global forest cover. Bright, R. M. et al. Local temperature response to land cover and management change driven by non-radiative processes. The mark of vegetation change on Earth's surface energy balance. & Spracklen, D. V. Climate benefits of intact amazon forests and the biophysical consequences of disturbance. A., Winck, G. R., Weber, M. M., Nichols, E. & Sinervo, B. Impacts of forestation and deforestation on local temperature across the globe. Abera, T. A. et al. Deforestation amplifies climate change effects on warming and cloud level rise in African montane forests. Zeppetello, L. R. V. et al. Large scale tropical deforestation drives extreme warming. Butt, E. W. et al. Amazon deforestation causes strong regional warming. Masuda, Y. J. et al. How are healthy, working populations affected by increasing temperatures in the tropics? Implications for climate change adaptation policies. Rony, M. K. K. & Alamgir, H. M. High temperatures on mental health: recognizing the association and the need for proactive strategies—a perspective. Périard, J. D., Eijsvogels, T. M. H. & Daanen, H. A. M. Exercise under heat stress: thermoregulation, hydration, performance implications, and mitigation strategies. Dasgupta, S. et al. Effects of climate change on combined labour productivity and supply: an empirical, multi-model study. Masuda, Y. J. et al. Warming from tropical deforestation reduces worker productivity in rural communities. Green, H. et al. Impact of heat on mortality and morbidity in low and middle income countries: a review of the epidemiological evidence and considerations for future research. Silveira, I. H. et al. Heat waves and mortality in the Brazilian Amazon: effect modification by heat wave characteristics, population subgroup, and cause of death. The burden of heat-related mortality attributable to recent human-induced climate change. & Kenny, G. P. Heat health planning: the importance of social and community factors. Predicted temperature-increase-induced global health burden and its regional variability. Barreca, A., Clay, K., Deschenes, O., Greenstone, M. & Shapiro, J. S. Adapting to climate change: the remarkable decline in the US temperature-mortality relationship over the twentieth century. Davis, L., Gertler, P., Jarvis, S. & Wolfram, C. Air conditioning and global inequality. Global labor loss due to humid heat exposure underestimated for outdoor workers. Saeed, W. et al. The poverty impacts of labor heat stress in West Africa under a warming climate. Suter, M. K. et al. Association between work in deforested, compared to forested, areas and human heat strain: an experimental study in a rural tropical environment. Masuda, Y. et al. Heat exposure from tropical deforestation decreases cognitive performance of rural workers: an experimental study. Wan, Z., Hook, S. & Hulley, G. MODIS/Terra Land Surface Temperature/Emissivity 8-Day L3 Global 1km SIN Grid V061. NASA Land Processes Distributed Active Archive Center https://doi.org/10.5067/MODIS/MOD11A2.061 (2021). Rose, A. et al. LandScan Global 2020 [dataset]. Oak Ridge National Laboratory https://doi.org/10.48690/1523378 (2021). Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the global burden of disease study 2019. Hersbach, H. et al. Global Burden of Disease Collaborative Network. Global Burden of Disease Study 2021 (GBD 2021) data resources. Institute for Health Metrics and Evaluation https://ghdx.healthdata.org/gbd-2021 (2024). Boussetta, S., Balsamo, G., Beljaars, A., Kral, T. & Jarlan, L. Impact of a satellite-derived leaf area index monthly climatology in a global numerical weather prediction model. da Silva, Q. P., Moreira, D. S., de Freitas, H. C. & Domingues, L. M. Discrepancies between observation and ERA5 reanalysis in the Amazon deforestation context: a case study. Reddington, C. L. et al. Contribution of vegetation and peat fires to particulate air pollution in South-east Asia. de Oliveira, I. N. et al. Air pollution from forest burning as environmental risk for millions of inhabitants of the Brazilian Amazon: an exposure indicator for human health. Cadernos de Saúde Pública 39, e00131422 (2023). Jacobson, L. S. V. et al. Acute effects of particulate matter and black carbon from seasonal fires on peak expiratory flow of school children in the Brazilian Amazon. Xue, T. et al. Associations between exposure to landscape fire smoke and child mortality in low-income and middle-income countries: a matched case-control study. Crippa, P. et al. Population exposure to hazardous air quality due to the 2015 fires in Equatorial Asia. Reddington, C. L. et al. Air quality and human health improvements from reductions in deforestation-related fire in Brazil. Butt, E. W. et al. Large air quality and human health impacts due to Amazon forest and vegetation fires. Butt, E. W., Conibear, L., Knote, C. & Spracklen, D. V. Large air quality and public health impacts due to Amazonian deforestation fires in 2019. Kiely, L. et al. Air quality and health impacts of vegetation and peatfires in Equatorial Asia during 2004–2015. Reddington, C. L. et al. Air pollution from forest and vegetation fires in Southeast Asia disproportionately impacts the poor. Bauer, S. E., Im, U., Mezuman, K. & Gao, C. Y. Desert dust, industrialization, and agricultural fires: health impacts of outdoor air pollution in Africa. Fornace, K. M. et al. Association between landscape factors and spatial patterns of Plasmodium knowlesi infections in Sabah, Malaysia. Estifanos, T. K., Fisher, B., Galford, G. L. & Ricketts, T. H. Impacts of deforestation on childhood malaria depend on wealth and vector biology. Global carbon budget 2024. Mora, C. et al. Global risk of deadly heat. Masuda, Y. J. et al. Impacts of warming on outdoor worker well-being in the tropics and adaptation options. Orlov, A. et al. Changes in land cover and management affect heat stress and labor capacity. de Oliveira, B. F. A. et al. Deforestation and climate change are projected to increase heat stress risk in the Brazilian Amazon. Libonati, R. et al. Drought–heatwave nexus in Brazil and related impacts on health and fires: a comprehensive review. Mukherjee, S. & Mishra, A. K. Increase in compound drought and heatwaves in a warming world. Herold, N., Alexander, L., Green, D. & Donat, M. Greater increases in temperature extremes in low versus high income countries. The inequality of climate change from 1.5 to 2 °C of global warming. & Neelsen, S. A comprehensive assessment of universal health coverage in 111 countries: a retrospective observational study. Zhuang, J. et al. pangeo-data/xESMF: v0.8.2 (v0.8.2). Carver, R. W., and Merose, A. ARCO-ERA5: an analysis-ready cloud-optimized reanalysis dataset. American Meteorological Society https://ams.confex.com/ams/103ANNUAL/meetingapp.cgi/Paper/415842 (2023). Met Office Iris: a Python package for analysing and visualising meteorological and oceanographic data sets, v3.1.0. Su, Y. et al. Asymmetric influence of forest cover gain and loss on land surface temperature. The global 2000–2020 land cover and land use change dataset derived from the Landsat archive: first results. Potapov, P. et al. Mapping and monitoring global forest canopy height through integration of GEDI and Landsat data. Global Multi-resolution Terrain Elevation Data 2010 (GMTED2010) Open-File Report 2011–1073 (US Geological Survey, 2011). Bhaduri, B. et al. LandScan USA: a high-resolution geospatial and temporal modeling approach for population distribution and dynamics. Schiavina, M., Melchiorri, M. & Pesaresi, M. GHS-SMOD R2023A—GHS settlement layers, application of the Degree of Urbanisation methodology (stage I) to GHS-POP R2023A and GHS-BUILT-S R2023A, multitemporal (1975–2030). European Commission Joint Research Centre https://doi.org/10.2905/A0DF7A6F-49DE-46EA-9BDE-563437A6E2BA (2023). European Commission GHSL Data Package 2023 (Publications Office of the European Union, 2024); https://data.europa.eu/doi/10.2760/098587 Global Forest Resources Assessment. Terms and definitions—FRA 2020 Forest Resources Assessment Working Paper 188 (Food and Agriculture Organization of the United Nations, 2020); https://www.fao.org/3/I8661EN/i8661en.pdf Liu, R. et al. Global-scale ERA5 product precipitation and temperature evaluation. Mistry, M. N. et al. Comparison of weather station and climate reanalysis data for modelling temperature-related mortality. An analysis of spatio-temporal relationship between satellite-based land surface temperature and station-based near-surface air temperature over Brazil. Gallo, K., Hale, R., Tarpley, D. & Yu, Y. Evaluation of the relationship between air and land surface temperature under clear- and cloudy-sky conditions. Mildrexler, D. J., Zhao, M. & Running, S. W. A global comparison between station air temperatures and MODIS land surface temperatures reveals the cooling role of forests. The most at-risk regions in the world for high-impact heatwaves. Effects of ambient heat exposure on risk of all-cause mortality in children younger than 5 years in Africa: a pooled time-series analysis. Romanello, M. et al. The 2024 report of the Lancet Countdown on health and climate change: facing record-breaking threats from delayed action. The convective boundary layer over pasture and forest in Amazonia. Dubreuil, V. et al. Impact of land-cover change in the Southern Amazonia climate: a case study for the region of Alta Floresta, Mato Grosso, Brazil. Xu, X. et al. Deforestation triggering irreversible transition in Amazon hydrological cycle. Reddington, C. L. et al. Data files and code for manuscript ‘Tropical deforestation is associated with considerable heat-related mortality'. Hoyer, S. & Hamman, J. xarray: N-D labeled arrays and datasets in Python. Array programming with NumPy. Hunter, J. D. Matplotlib: a 2D graphics environment. The Pandas development team. Elson et al. SciTools/cartopy: REL: v0.24.1 (v0.24.1). Gasparrini, A. et al. Projections of temperature-related excess mortality under climate change scenarios. World Bank World Development Indicators: Current Health Expenditure per Capita (Current US$) (World Bank Group, 2024); https://data.worldbank.org/indicator/SH.XPD.CHEX.PC.CD World Bank Gender Statistics: Prevalence of Overweight (% of Adults) (World Bank DataBank, 2016); https://databank.worldbank.org/source/gender-statistics/Series/SH.STA.OWAD.MA.ZS United Nations World Development Indicators: Population Ages 65 and Above (% of Total Population) World Population Prospects: 2022 Revision (United Nations Population Division, 2022); https://data.worldbank.org/indicator/sp.pop.65up.to.zs The research has been supported by funding from the European Research Council under the European Union's Horizon 2020 research and innovation programme (DECAF project; grant agreement no. ), the International Science Partnerships Fund (ISPF), through the Met Office Climate Science for Service Partnership (CSSP) Brazil (E.W.B., C.S., J.C.A.B., D.V.S. ), the Natural Environmental Research Council (NE/Z00005X/1) (E.W.B., D.V.S. ), and the UK Research and Innovation Future Leaders Fellowship (MR/X034097/1) (J.C.A.B. Institute for Climate and Atmospheric Science, School of Earth and Environment, University of Leeds, Leeds, UK C. L. Reddington, C. Smith, E. W. Butt, J. C. A. Baker & D. V. Spracklen Fiocruz Regional Office of Piauí, National School of Public Health, Oswaldo Cruz Foundation, Teresina, Brazil Department of Meteorology and Climate Science, Kwame Nkrumah University of Science and Technology, Kumasi, Ghana Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar provided data and analysis tools; C.L.R. performed the research and analysed data; B.F.A.O. contributed region-specific expertise; and all authors contributed to experimental design, interpretation of results and writing of the paper. The authors declare no competing interests. Nature Climate Change thanks Jun Ge, Jihoon Jung and the other, anonymous, reviewer for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Population count in 2020 versus percentage-point forest cover loss between 2001 and 2020. Green points show data for Tropical Central and South America, red points show data for Tropical Africa, and blue points show data for Southeast Asia. Results are shown for populations in locations of forest loss (see Methods) in a. Tropical Central and South America, b. Tropical Africa, and c. Southeast Asia. Dashed vertical lines show the area- and population-weighted mean deforestation-induced ΔT for each region. Results are shown for populations in locations of forest loss in a. Tropical Central and South America, b. Tropical Africa, and c. Southeast Asia. Dashed vertical lines show the population-weighted mean deforestation-induced ΔT for each region. The maps show regions of Tropical Central and South America (a), Tropical Africa (b), and Southeast Asia (c). Colours show number of deaths per year per 100,000 people located in areas of forest loss (central estimate), aggregated by second-level administration divisions. Boundaries for second-level administration divisions are from GADM (https://gadm.org/index.html). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tropical deforestation is associated with considerable heat-related mortality. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250826005233.htm'>Scientists finally pinpoint Jupiter's birth using “molten rock raindrops”</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 03:38:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Four and a half billion years ago Jupiter rapidly grew to its massive size. This caused them to smash into each other at such high speeds that the rocks and dust they contained melted on impact and created floating molten rock droplets, or chondrules, that we find preserved in meteorites today. Chondrules, small spheres approximately 0.1-2 millimeters wide, were incorporated into asteroids as the solar system formed. How chondrules came to have their round shape has puzzled scientists for decades. "When planetesimals collided with each other, water instantly vaporized into expanding steam. This acted like tiny explosions and broke apart the molten silicate rock into the tiny droplets we see in meteorites today," co-lead author Professor Sin-iti Sirono from Nagoya University's Graduate School of Earth and Environmental Sciences explained. "Previous formation theories couldn't explain chondrule characteristics without requiring very specific conditions, while this model requires conditions that naturally occurred in the early solar system when Jupiter was born." The model also shows that chondrule production coincides with Jupiter's intense accumulation of nebular gas to reach its massive size. As meteorite data tell us that peak chondrule formation took place 1.8 million years after the solar system began, this is also the time at which Jupiter was born," Dr. Diego Turrini, co-lead author and senior researcher at the Italian National Institute for Astrophysics (INAF) said. This study provides a clearer picture of how our solar system formed. The most likely explanation is that other giant planets like Saturn also triggered chondrule formation when they were born. After 60 Years, Scientists Uncover Hidden Brain Pathway Behind Diabetes Drug Metformin Revolutionary Cortisol Test Lets You “See” Stress With a Smartphone Camera How B Vitamins Could Slow Cognitive Decline and Protect Against Dementia Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/spacex-successfully-launches-starship-spacecraft-after-string-of-mishaps/'>SpaceX Successfully Launches Starship Spacecraft after String of Mishaps</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-27 01:35:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>SpaceX Successfully Launches Starship Spacecraft after String of Mishaps Overcoming three recent failed tries, Elon Musk's rocket company successfully flew its reusable jumbo booster and upper-stage Starship spacecraft An earlier SpaceX Starship and Super Heavy booster are moved to the launch pad at Starbase in Boca Chica, Texas. SpaceX successfully test launched its Starship rocket on Tuesday, overcoming past failures that had raised questions about the capabilities—and planetary aspirations—of the pioneering space launch firm. The 6:30 P.M. CDT liftoff from Texas carried a massive “Super Heavy” booster to the edge of space, some 119 miles high, where it dropped off from Starship's upper-stage spacecraft, somersaulted and fired two of its engines to soft land in the Gulf of Mexico. The upper stage continued into orbit, released eight test satellites and then braked to make its own splashdown in the Indian Ocean about an hour and six minutes after launch. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. SpaceX chief Elon Musk has promised a successful Starship will cut the cost of shooting satellites into space significantly. Starship's upper-stage spacecraft is also intended to land NASA astronauts on the moon for the Artemis III mission, which is scheduled for 2027 but will almost certainly be delayed. This year two previous Starship test flights exploded over the Caribbean, and a third spun out of control over the Indian Ocean—mishaps that were all tied to explosions in rocket fuel systems. That has spurred concern that the rocket's design fundamentally lacks the stability needed to dampen damaging vibrations from its multiple engines. Experts have speculated that adding stabilizing mass to Starship's fuel system may be necessary, but such a move would cut into its cargo capacity. The successful launch and planned water landings of both the Super Heavy booster and Starship's upper-stage spacecraft reaffirms SpaceX's policy of rapid testing spacecraft prototypes. This version is intended to travel into low-Earth orbit to refuel other rockets in space, a new technology that is needed for moon missions. Dan Vergano is a senior editor at Scientific American. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            