
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-08</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250408121923.htm'>Genetic predisposition to sedentary behavior increases the risk of cardiovascular diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 23:28:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Although sedentary behaviour may be an evolutionarily selected trait, it is still important to try to be physically active, says a new study conducted at the University of Jyväskylä, Finland. Researchers have shown for the first time that genetic predisposition to sedentary behaviour is associated with a higher risk of developing the most common cardiovascular diseases. Genetics are known to be associated with both cardiovascular diseases and low levels of physical activity. Genetic predisposition can be determined using modern genome-wide polygenic scores. The researchers developed a polygenic score to describe genetic predisposition to leisure screen time, the most common type of voluntary sedentary behaviour. The analyses were repeated in a separate reference group, which consisted of about 35,000 Norwegians. "People with the highest predisposition to sedentary behaviour accumulated about half an hour more daily sedentary time and had about a 20% higher risk of developing the most common cardiovascular diseases, compared to those with the lowest genetic predisposition." It is worth getting moving, even if you feel lazy Sedentary behaviour may be a trait selected by evolution. Paleoanthropologists have previously argued that physical activity has long been a survival need in human history, which is why humans would not have evolved to move voluntarily. "Our results support previous theories that the human trait to be sedentary has a genetic basis and illustrate its health effects," says Joensuu, "and these findings should be taken into account when promoting the health of the population. However, since physical activity has positive effects on our overall wellbeing, it's important to ignore these types of negative feelings." One effective way to do this is by fostering a sense of community and the joy that comes with physical activity." The group is led by Elina Sillanpää, Associate Professor of Health Promotion. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250408121657.htm'>Viral 'backbone' underlies variation in rotavirus vaccine effectiveness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 19:22:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have shown that differences in the entire rotavirus genome -- not just its two surface proteins -- affect how well vaccines work, helping to explain why some strains are more likely to infect vaccinated individuals. They say the novel approach to estimating rotavirus vaccine effectiveness provides convincing evidence that rotavirus vaccines should be designed based on the whole genome of circulating strains, rather than the previous use of two surface proteins. Rotavirus is a contagious gastrointestinal infection that causes inflammation of the stomach and intestines (gastroenteritis), marked by severe dehydrating diarrhea. Two main vaccines, Rotarix (RV1) and RotaTeq (RV5), have significantly reduced cases of severe illness from rotavirus in the US, but they do not provide perfect protection. Traditionally, these differences have been quantified using two proteins on the virus' outer shell -- VP7 and VP4. "We set out to investigate why some vaccinated children still get sick with rotavirus," says lead author Jiye Kwon, a PhD student at the Department of Epidemiology of Microbial Diseases, and the Public Health Modeling Unit, Yale School of Public Health, New Haven, US. "Previous research has focused on just the outer proteins of the virus, but rotavirus has a total of 11 genetic segments. They sought to identify whether vaccine effectiveness decreased as the genetic distance between virus and vaccine strains increased, as well as to examine how the genetic diversity of rotavirus changes in areas with higher vaccine coverage. Their results revealed that individuals vaccinated with Rotarix (RV1) were more likely to be infected by rotavirus strains that were significantly genetically different from the vaccine -- more than 9.6% different in their full genome. On the other hand, the genetically distant strains tended to have a different viral backbone called genogroup 2 (DS-1-like) or have mix-and-match variants known as reassortant strains. The Rotarix (RV1) vaccine provided strong protection against genetically similar viral strains, but its protection dropped significantly for more genetically distant strains. The RotaTeq (RV5) vaccine followed a similar pattern, but differences in its effectiveness were less pronounced. This was also observed in areas with high usage of RotaTeq (RV5). "Current vaccines still provide strong protection against severe illness in rotavirus, but these findings highlight the need to continually monitor viral evolution to maintain vaccine effectiveness in the long term," says Kwon. The team caution that their study is limited by a relatively small sample size of cases, due to the requirement of whole genome sequencing data. They call for future studies to further validate their findings in other settings where whole genome sequencing data is more widely available. There are now four rotavirus vaccines currently available and more in the pipeline; our framework for using whole genome sequencing data to understand how all gene segments contribute to immune protection could be crucial for maintaining their long-term success." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250408121322.htm'>CVD and obesity: When protective lipids decline, health risks increase</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 17:43:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research from Weill Cornell Medicine has uncovered a surprising culprit underlying cardiovascular diseases in obesity and diabetes -- not the presence of certain fats, but their suppression. The study, published Feb. 25 in Nature Communications, challenges the conventional belief that a type of fat called ceramides accumulates in blood vessels causing inflammation and health risks. Instead, their findings reveal that when ceramides decrease in endothelial cells lining blood vessels, it can be damaging and cause chronic illnesses. Ironically, the findings could ultimately lead to therapies that maintain high levels of these protective lipids in patients with obesity. Also working on this research are co-first authors Dr. Onorina L. Manzo, postdoctoral associate and Luisa Rubinelli, both in Dr. Di Lorenzo's lab. Dr. Di Lorenzo and her team discovered the importance of ceramides in blood vessels two years ago. Together with Dr. Giuseppe Faraco, assistant professor of neuroscience at Weill Cornell Medicine, they found that decreased levels of ceramides in otherwise healthy mice causes severe blood vessel inflammation in the brain, clot formation and death. Ultimately, when ceramide is broken down by the body it produces a compound called sphingosine-1-phosphate (S1P), which builds up and protects mice against cardiovascular disease. But when this process doesn't work the mice are left vulnerable. This decrease leads to increased blood pressure, impaired vascular regulation and higher glucose levels -- all of which contribute to cardiometabolic conditions that affect the heart (cardiovascular system) and energy processing (metabolism), like diabetes like diabetes, hypertension, coronary artery disease and stroke. To understand how these different molecules interact, the researchers tested what happens in animal models. These mice showed signs of inflammation, diabetes and high blood pressure. But what happens if the Nogo-B inhibitor wasn't present? "These mice have the same body weight and diabetes as controls, but their blood vessel health is much better," said Dr. Di Lorenzo. This also showed that the regulation of ceramide metabolism causes vascular dysfunction and inflammation in obesity." "Nogo suppresses biosynthesis of ceramides, so if we can identify a drug that can block Nogo-B, we could restore ceramide levels to a healthy balance and this would fight not only obesity and diabetes, but would directly keep blood vessels functioning properly," she said. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250408121324.htm'>ALS drug effectively treats Alzheimer's disease in new animal study</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 17:43:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Experimental drug NU-9 -- a small molecule compound approved by the U.S. Food and Drug Administration (FDA) for clinical trials for the treatment of amyotrophic lateral sclerosis (ALS) -- improves neuron health in animal models of Alzheimer's disease, according to a new Northwestern University study. Like ALS, Alzheimer's disease also results from misfolded proteins that damage brain health. Rather than treating symptoms from specific diseases, NU-9 instead addresses the underlying mechanisms of disease. "This drug is quite remarkable that it works in these multiple systems," said Northwestern's Richard B. Silverman, who invented NU-9. But how well upper motor neurons function in mice is similar to how well they function in humans. So, it seems to me, NU-9 really should work." "What our study demonstrates is that the same mechanism affects two totally different proteins in two totally different diseases," said Northwestern's William Klein, the study's co-corresponding author. "In both diseases, cells suffer from toxic protein buildup. It appears there is a common mechanism that gets rid of these proteins to prevent them from clustering. Silverman, who previously invented pregabalin (Lyrica) to treat fibromyalgia, nerve pain and epilepsy, is the Patrick G. Ryan/Aon Professor of Chemistry at Northwestern's Weinberg College of Arts and Sciences and founder of Akava Therapeutics, a startup company which is commercializing NU-9. An expert on Alzheimer's disease, Klein is a professor of neurobiology at Weinberg and cofounder of Acumen Pharmaceuticals, which has a therapeutic monoclonal antibody to treat Alzheimer's disease currently in clinical trials. In patients with neurodegenerative diseases, misfolded proteins clump together inside brain cells. This accumulation of misfolded proteins leads to toxicity that disrupts normal brain function and eventually triggers brain cells to die. "These are good proteins gone bad," Klein said. In previous studies, Silverman and longtime collaborator P. Hande Ozdinler, an associate professor of neurology at Northwestern University Feinberg School of Medicine, discovered that NU-9 helped cells remove the protein clumps caused by two unrelated mutated proteins to restore neuron function in animal models with ALS. Silverman and Klein wanted to explore whether NU-9 might have a similar effect on Alzheimer's disease. In one experiment, the scientists added a form of amyloid beta to these cells. The amyloid beta oligomers quickly formed and stuck to cells. They administered an oral dose of NU-9 to a mouse model of Alzheimer's disease and found the animals' performance on memory tests improved. In promising follow-up studies, the team also found NU-9 reduces brain inflammation associated with Alzheimer's disease. So, the drug is very powerful on two levels: cellular and whole animal." Although the researchers are still trying to fully understand how NU-9 works, they made some key discoveries. They found NU-9 specifically prevents the buildup of amyloid beta oligomers that form inside cells but doesn't prevent these proteins from forming outside cells. This means NU-9 must be working on a process within the cell to prevent harmful protein clusters. In Alzheimer's disease, this recycling system is disrupted, causing amyloid beta to accumulate. "They collect junk and other components that are not useful to the cell, chew them up and get rid of them. We found the proteasome wasn't involved at all. It's the lysosome that plays a role in how NU-9 works. "It's like a relay race for moving these toxic clustered proteins around the cell. The proteins are clustered in one vesicle and then another vesicle and then finally handed off to the lysosome. We think NU-9 targets something in the early stage of that relay, but we don't know exactly what the target is." The team also plans to explore the effectiveness on NU-9 in other neurodegenerative diseases, like Parkinson's disease and Huntington's disease. "It has long been thought that every neurodegenerative disease is a completely separate disease, but our findings suggest that common mechanisms might connect them," Silverman said. "This discovery opens the door to a new family of therapeutic compounds that, like NU-9, could control multiple degenerative diseases at a point before major damage to cells begins." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-dire-wolf-isnt-back-but-heres-what-de-extinction-tech-can-actually-do/'>The Dire Wolf Isn't Back—But Here's What ‘De-Extinction' Tech Can Actually Do</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 16:25:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Colossal Bioscience says it has “de-extincted” the dire wolf, but other scientists disagree and say more important conservation science is being lost in all the hype One of the "dire wolves" created by Colossal Biosciences. The television series Game of Thrones helped popularize dire wolves, but the creatures don't just represent a figment of science fiction: the dire wolf was a real animal that went extinct around 10,000 years ago. On Monday Colossal Biosciences, a Dallas, Tex.–based biotechnology company, announced that it brought the species back with the birth of two pups last October and a third this past January. (The company had previously announced the development of a “woolly mouse,” or a mouse whose genome was edited to give it brown, shaggy fur like that of the extinct woolly mammoth.) But many scientists say what Colossal produced this time is not, in fact, the dire wolf. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Dire wolves (Aenocyon dirus) are an extinct carnivore that lived throughout what are today North and South America during the Pleistocene and early Holocene epochs (about 250,000 to 10,000 years ago). First described in the 1850s, their fossils have been found all over the Americas, perhaps most famously in Los Angeles' La Brea Tar Pits. It is thought that dire wolves died out as their prey did. The resemblance between dire wolves and gray wolves is an example of convergent evolution, when species separately evolve similar adaptations because they lead a similar lifestyle, the study's researchers said at the time. This is a genetically modified gray wolf.”—Jacquelyn Gill, paleoecologist Beth Shapiro, who is now Colossal's chief science officer and was a co-author of the 2021 paper, says the company's recent work builds on those findings. After examining the dire wolf genome, Shapiro and her team edited 20 sites on 14 genes in the genome of the modern gray wolf (Canis lupus), introducing what they say are 15 extinct dire wolf variants. No ancient dire wolf genes themselves were directly inserted into the genome, however. The scientists created embryos that were implanted in surrogate dogs. Colossal targeted genes that affected phenotype, or the observable characteristics of an organism—in this case, largely its appearance. We can't create that many edits at once,” she says. “But it's also not the goal.” Instead, Shapiro adds, “we want to create functional versions of extinct species. The "dire wolves" are being kept on a preserve in an undisclosed location. This is a genetically modified gray wolf,” says Jacquelyn Gill, a paleoecologist at the University of Maine, who has worked with Shapiro in the past but was not involved in this project. The pups “don't have any traits that would allow us to understand the dire wolf any better than we did yesterday,” Gill says, adding that understanding ice age organisms isn't just a matter of knowing what they looked like or what they ate—but also about knowing what they did in those ancient ecosystems. “Some of those things are coded genetically; some of those are cultural” and passed down from generation to generation. At best, this would be an “incremental step” toward de-extinction, she says. And even if scientists could clone a dire wolf that was completely identical to its extinct ancestor, Gill adds, that would raise a very important question: “What are we going to do with it?” “There is cool science here, I just wish it wasn't getting lost in hype.”—Gill It currently has no plans to breed these wolves. “We have downsized planet Earth,” she says, noting that conservationists already struggle to maintain and support populations of large predators, such as gray wolves. “It's hard to imagine a practical application here.” Colossal's "dire wolves," Romulus and Remus, at three months of age, born October 1, 2024. Red wolves, once found from Texas to Pennsylvania, saw their numbers nose-dive in the 20th century because of hunting and habitat loss. But as with many small, endangered populations, the gene pool is limited. Shapiro says Colossal's technique, which uses routine blood draws, yields cells that are easier to reprogram than those that come from skin—and could thus be a better way of diversifying the red wolf gene pool. “It's actually using technology to prevent species from going extinct,” says Matt James, Colossal's chief animal officer. “I just wish it wasn't getting lost in hype.” Andrea Thompson is an associate editor covering the environment, energy and earth sciences. Prior to joining Scientific American, she was a senior writer covering climate science at Climate Central and a reporter and editor at Live Science, where she primarily covered earth science and the environment. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. in atmospheric chemistry from the Georgia Institute of Technology.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64308954/consciousness-shapes-reality-science/'>Consciousness Isn't Just in Your Head—It May Be Altering Reality Itself, Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 13:52:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But lately, we've really tried to dive deep into the world's most complex machine: you. Discover the most mind-blowing new science stories from Pop Mech Pro! In the process, we've had to ask some mind-bending questions like: Just bring an open mind, and prepare to reconsider the very nature of consciousness ... starting with the possibility that it's not just all in your head. Is consciousness confined to your brain, or does every part of your body contain a kind of consciousness, down to the cellular level? That's what William B. Miller, Ph.D., an evolutionary biologist and physician, is trying to figure out. Miller and his team have proposed ideas that flip Charles Darwin's concept of natural selection on its head, including how consciousness may play a critical part in the way life evolves. Miller is among a small, but growing group of scientists who believe that your cells aren't “passive robots that automatically follow a code of instructions, carrying out orders from our genome like mindless drones.” Instead, these scientists think the roughly 37 trillion cells that make up your body are themselves conscious—and that life and consciousness began exactly at the same time. This concept of cellular consciousness has gained further credence with the advent of a new class of AI-designed multicellular organisms known as “xenobots.” These are cells that form new roles beyond their original biological function, such as using hairlike cilia for locomotion rather than transporting mucus. This is why scientists like Peter Noble, Ph.D., and Alex Pozhitkov, Ph.D., say that xenobots form a kind of “third state” of life, “wherein cells can reorganize after the death of an organism to form something new.” In recent decades, some experts have developed a theory that says when your brain performs quantum operations, it generates consciousness. One scientist, Michael Pravica, Ph.D., for example, believes that your brain can access hidden dimensions during moments of heightened awareness. (You have to read his explanation for what happens during a “Eureka” moment.) In the study, scientists administered anesthesia to rats, some of whom also received molecules that stabilized a microscopic part of their brain known as “microtubules,” or hollow tubes that experts believe “perform incredible operations in the quantum realm.” The stabilized rats actually remained conscious for longer, suggesting a tantalizing link between these quantum brain components and the very essence of consciousness itself. If this all sounds trippy, you're right—and, as it turns out, taking “trips” on psychedelics might actually unlock new levels of consciousness. But let's go a step further: What if psychedelics like magic mushrooms aren't just a new way to expand your consciousness? What if they're the very thing that helped shape it? A research review from the Miguel Lillo Foundation in Argentina concluded that, “psilocybin not only influenced an individual's perceptions while under its effects, but shaped human consciousness as a whole over the thousands of generations that humans had been eating psychedelic fungi.” Consciousness remains one of the most profound, mind-blowing mysteries in all of science, and we've covered its every twist and turn, including: If you're even remotely curious about embarking upon the fascinating journey into the nature of you, join Pop Mech Pro today, and you'll instantly unlock every single exclusive story, project plan, guide, video, and gadget review on PopularMechanics.com. Michael Natale is a news editor for the Hearst Enthusiast Group. His stories have appeared in Popular Mechanics, Best Products, and Runner's World. A Student Sniffed Out an Ancient Circle of Stones This “Battery In a Rock” Changes the Energy Game</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/breakthrough-prize-winner-gerard-t-hooft-says-quantum-mechanics-is-nonsense/'>Breakthrough Prize Winner Gerard 't Hooft Says Quantum Mechanics Is ‘Nonsense'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Quantum Physics Is on the Wrong Track, Says Breakthrough Prize Winner Gerard 't Hooft After netting the world's highest-paying science award, preeminent theoretical physicist Gerard 't Hooft reflects on his legacy and the future of physics But 't Hooft's unassuming, soft-spoken manner belies his towering scientific stature, which is better revealed by the mathematical rigor and deep physical insights that define his work—and by the prodigious numbers of prestigious prizes he has accrued, which include a Nobel Prize, a Wolf Prize, a Franklin Medal, and many more. His latest accolade, announced on April 5, is the most lucrative in all of science: a Special Breakthrough Prize in Fundamental Physics, worth $3 million, in recognition of 't Hooft's myriad contributions to physics across his long career. His most celebrated discovery—and the one that earned him, along with his former Ph.D. thesis adviser, the late Martinus Veltman, the 1999 Nobel Prize in Physics—showed how to make sense of non-Abelian gauge theories, which are complex mathematical frameworks that describe how elementary particles interact. Together, 't Hooft and Veltman demonstrated that these theories could be renormalized, meaning that intractable infinite quantities that cropped up in calculations could be tamed in a consistent and precise way. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. But beyond this, 't Hooft has made many other breakthroughs, which are too numerous—and, in most cases, too technical—to thoroughly describe here. The idea has since become central to many efforts to unify quantum mechanics with Einstein's general theory of relativity in an all-encompassing theory of quantum gravity. In a conversation with Scientific American, 't Hooft spoke about his Breakthrough Prize, his optimism for the future of particle physics, his dissatisfaction with quantum mechanics, and the scientific and cultural effects that have arisen from some of his most provocative ideas. [An edited transcript of the interview follows.] It seems you've won practically all the big physics prizes at this point. But, yeah, I've won quite a few prizes. What worries me a little bit is that most of them were for the same thing. You get prize after prize for something that has already been recognized as such, whereas I've done other things in science that are not as well known—not by the general public, at least. But anyway, the Breakthrough Foundation has made a summary of my work for which they gave this prize, and that contains practically everything! But given how many prizes you have won, does this one feel like just another notch on your belt? Has this all become routine for you, or is it still exciting? The climax really was the Nobel Prize itself, which is only granted to a very few people every year. But this one is also very special. And as you mentioned, this one recognizes the full sweep of your scientific career rather than just one facet of it, such as your work in the 1970s with Veltman to explain the electroweak interaction that led to you both sharing the 1999 Nobel Prize in Physics. That work, of course, was fundamental to the subsequent formulation of the Standard Model of particle physics, now celebrated as the most well tested and successful scientific theory ever devised. But in some respects, the Standard Model has become notorious, too, because its many myriad experimental validations have contributed to a crisis in particle physics wherein progress has slowed down as researchers have seen no obvious path forward to further breakthroughs. Does this aspect of the Standard Model's decades-long dominance worry you? I think it is natural for science that we cannot always have an infinitely continuous stream of discoveries and new insights. There will be periods, like the one we are in now in particle physics, where things seem to be quieter. But I think history shows it won't be always like this. A few centuries ago, when [James Clerk] Maxwell joined electricity and magnetism, and after that, when Max Planck made the first observations about energy being quantized, there were long periods in which very little seemed to be happening. In reality, of course, many things did happen in other fields, such as statistical physics and other fundamental branches of science. Look at astronomy right now; the astronomers have their great moments all the time, and you can't say there's a dull moment at all! They're discovering many new things in the universe as their telescopes become bigger and more accurate and as they use more and more fundamental scientific techniques to enhance their resolution. You can say much the same thing about biophysics or medicine, where discoveries are made nearly every day. But in my field, you're right, it seems to be that nothing is happening. Things are happening, just at a more modest scale. Are you optimistic, then, that this situation will change, and we'll see a resurgence in big particle physics discoveries? That's a very good question because it looks as if there's nothing we can do. If the situation proceeds in such a way that every new breakthrough requires a 10-fold, or even larger, increase in the machines' size, power and costs, then clearly we won't get much beyond where we are now. I cannot exclude such obstacles standing in the way of progress, but the history of science suggests, in such a case, progress will simply go in different directions. But I would like to advise to the new generation of scientists: don't worry about that, because the real reason why there's nothing new coming is that everybody's thinking the same way! I'm a bit puzzled and disappointed about this. People don't seem to want to make the daring new steps that I think are really necessary. There are insights, of course, but not the ones that are needed to make a basic new breakthrough in our field. I think we have to start thinking in a different way. And particularly in the 1970s, there was a very efficient way of making further progress: think differently from what your friends are doing, and then you find something new! I think that is still true; however, I'm getting old now and am no longer getting brilliant new ideas every week. And to my mind, people thinking in such novel ways is not happening enough. My way of thinking about the world, about physics, about the other disciplines related to physics is that everything should be much more logical, much more direct, much more “down to Earth.” Many people who write papers on quantum mechanics like to keep some sense of mysticism about it, as if there's something strange, something almost religious about the subject. Quantum mechanics is based on a mathematical method used to describe very ordinary physical effects. But in this completely classical world, there are still too many things that we don't know today, there are “steps” we're basically missing on our path to deeper understanding. I'm talking about steps that would exploit the fact that the whole world is very simple and straightforward. The trouble is, the world still appears complicated to us now, which is why we're in this situation. The Standard Model is based on quantum mechanics, and quantum mechanics tells you what happens when particles approach one another and scatter. But they can scatter in many different ways; they have a large number of choices of ways in which they scatter against each other, and the Standard Model doesn't give any sound prediction there. But the theory never tells you which choice nature makes; it only tells you that these different possibilities are there at a certain probability amplitude. That is the world as we know it. But it's not the laws of nature themselves. What's missing is our understanding as to what it is that sometimes makes a particle go this way, sometimes that way. They don't hit each other directly head-on but hit at some angle, and then they scatter away from some angle. But what the theory today is not saying is what I should actually be looking at if two particles approach each other to predict how they'll scatter ahead of time. Imagine if you knew the way such interactions would go as precisely as you could know what will happen when two grand pianos hit each other. In principle, for the pianos, you could say exactly which wire will hit each other wire; you could predict exactly what happens when two grand pianos collide. In practice, such predictions for particles are considered to be too hard, and you turn to statistics, and you conclude that your piano particles can scatter in all directions, and that's all there is to be said. Well, for looking at pianos, maybe you can say something more. If you know exactly where and at which angle they will hit each other, you can predict ahead of time how they will scatter. And that should be in our theories of the elementary particles as well—and it isn't. I'm saying we should start to think in these ways. And people refuse that because they think quantum mechanics is too beautiful to be wrong. Whereas I believe that quantum mechanics is not the right way of ultimately saying what basic laws objects obey when they hit each other. Incidentally, while I was preparing for this interview, I found a conversation you had in 2013 with one of my predecessors here at Scientific American, George Musser. You said that you considered locality to “be an essential ingredient for any simple, ultimate law governing the universe.” It sounds like that's still your view. I think, in fact, that you can understand and explain quantum mechanics very well if you only assume that the laws are local laws. And if it does matter, then you have what we call “nonlocality.” But nonlocality would be a disaster for most solid scientific theories! We don't know exactly what to do when two particles collide because we don't know whether particles look like grand pianos or like pure points. But, then again, they can't be pure points because pure points can't do anything. There's something in there, and we should be able to write down all the laws on what's in there, in these particles: How can they collide against each other? We should be able to phrase such things as solid laws, and we are not even close to that. And this is why I think other breakthroughs should still be possible—many of them!—to help us get closer to this level of understanding for particles that we simply don't have today, not even as something approximate. You know, in my talks with theoretical physicists, I've noticed that the greater and more accomplished the individual is, the more likely they are to say, “The real challenge is not in answering old questions but rather in finding new, better questions for whatever problem you're addressing.” I think that's because there's this temptation for optimism about what can be known—this feeling that by asking the “right” questions, meaningful answers must emerge. Do you really think the problem is that we're not asking the right questions, or is it instead that we've been asking the right ones, and their answers are, against our hopes, simply beyond our reach? And of course, that was the wrong answer each time. Before Maxwell, nobody understood how exactly electric and magnetic fields hang together, and they thought, “Oh, this is impossible to find out because it's weird!” But then Maxwell said, no, you just need this one term, and then it all straightens out! If you believe right from the beginning that quantum mechanics is a theory that only gives you statistical answers and never anything better than that, then I think you're on the wrong track. And people refuse to drop the idea that quantum mechanics is some strange sort of supernatural feature of the particles that we will never understand. We will understand, but we need to step backward first, and that's always my message in science in general: before you understand something, just take a few steps back. Just imagine: What would your basic laws possibly be if you didn't have quantum mechanics? Answering that, of course, requires saying what quantum mechanics is. Quantum mechanics is the possibility that you can consider superpositions of states. And I'd argue that superpositions of states are not real. If you look very carefully, things never superimpose. [Erwin] Schrödinger asked the right questions here: You know, take my cat, it can be dead; it can be alive. That's complete nonsense—yet, at that level, it seems to be the only correct answer to say exactly where the particle is, what its velocity is, what its spin is, and so on. These would be variables in terms of which you can't move a cat, you can't say whether it's dead or alive, unless you would make more nonlocal changes. There must be ways to describe all states for alive cats and for dead cats, but these states will mix with states that don't describe cats at all. Using superpositions is then just a trick that works at first but doesn't get at the states we want to understand. If superpositions are illusory in that they are purely mathematical concepts that have no basis in physical reality, how does that square with the ongoing success of quantum information science and quantum computing, where it seems as if superpositions are a real physical phenomenon that can be leveraged, for instance, to do things that can't be done classically? Well, I think quantum technology is just what you get if you assume the reality of superimposed systems. We know superpositions in the macroscopic world are nonsense. And I believe, in the microscopic world, it's clearly nonsense, too, even though it may seem we have nothing besides superpositions to use for understanding atoms. Instead I think what they should be doing is trying to remove the quantum mechanics from the description, trying to use more fundamental degrees of freedom, like those discrete states I mentioned. They're not asking the right questions, and that failure to do so makes things look more and more complicated—more and more quantum-mechanical—whereas, in reality, it shouldn't be interpreted that way. Weren't we just discussing the tendency of eminent theorists to talk about not asking the right questions? Well, let me say that, yes, they do the right experiments. Yes, they try to make the right things. And yes, their quantum computers may be more powerful than anything else for certain applications because they understand “quantum mechanics”—by which I mean they understand how these microscopic systems actually act, in great detail, because this is something that actually came out of studying the quantum world. Yes, we know how small objects react and interact. But our problem is that, at present, we can only make statistical predictions. And as soon as a quantum computer gives you statistical distributions instead of correct answers, well, that's the end of your “computer”; you can't use it for most applications anymore. For instance, you want to decipher a secret code or something like that. You want to have the exact answer: “This is what it means, not that!” And let's not equate this answer to a superposition of those two possibilities—again, that's nonsense. What I'm saying is: we must unwind quantum mechanics, so to speak, as to see what happens underneath. And until the quantum technologists start doing that, I believe they won't make really big progress. But to have this realization is apparently very difficult. This is my feeling as to why we don't make breakthroughs. We should think about things in a different manner. It seems you're saying we must live in a clockwork universe, one in which things must be purely deterministic at a very fundamental level, and thus there's very little room for any sort of quasi-mystical speculation. You mentioned earlier the stubborn persistence of an almost religious approach to quantum mechanics within the scientific community, not to mention in popular culture. Perhaps this attitude endures because, for so many people, it preserves something ineffable about all that we experience in the world rather than assuming everything can be known by filling in the right equations. So if you do believe in this sort of clockwork universe, I wonder what you'd say its most mysterious aspect would be. And this deterministic universe we discuss is something that could only be fully understood by someone with a much bigger mind, a much bigger brain, than I have because they'll have to consider all possibilities. And as soon as you make some wrong assumption, then you again get this quantum-mechanical situation in which things get to superimpose each other. And in one of my last [preprint] papers on arXiv.org, I wrote a little simple model—too simple to be useful in a real world. And because of this, I call it my grandfather's clock model. They are just showing a time with infinite precision, say. And the pendulum is really a quantum pendulum; it can be quantized; we can write quantum equations for it. But I got very few reactions to this. Now we understand how to continue!” But instead they've said, “Okay, right, 't Hooft has another hot idea, another crazy idea. And he has many of those crazy ideas. Let him be happy with it; we're going to do our own thing.” And that's the most common reaction I've gotten. I'd suspect the reasons for that reaction are, in some sense, not scientific and rather more “cultural,” right? Arguably because of this idea, there are people—mostly nonscientists, I'd imagine—who truly believe that the cosmos is in fact within a black hole or that it's all some simulation in a higher-dimensional computer. The idea being for this “simulation hypothesis” that perhaps nothing is “real” besides information itself, as everything else could just be a projection of patterns of 1's and 0's encoded on the outermost boundary of the observable universe. I wonder what you think about this phenomenon in which you put forth a provocative theoretical insight more than 30 years ago, and it has somehow led to the world's richest man seriously suggesting on a popular podcast that “we are most likely” all just avatars in some cosmic-scale video game. Maybe I should have never talked about the holographic principle because, yes, some people are galloping away into nonsense, linking this idea with supernatural features and poorly defined dimensionality, all to sound very mysterious. And I have a big problem with that. I think you shouldn't phrase the laws of nature in more complicated terms than strictly necessary. Even Einstein once said something like this, that you have to simplify things as much as possible but not beyond reality, not beyond the truth. I am a bit worried that the holographic principle has only invited people to be more mysterious because I want the extreme opposite. For me, even quantum mechanics is already too far away from reason. And you know, if you rephrase quantum mechanics to treat Hilbert space [a type of vector space that allows for infinite dimensions] as something used for practical purposes rather than Hilbert space being a fundamental property of nature, you don't even need this sort of holography anymore! Lee Billings is a science journalist specializing in astronomy, physics, planetary science, and spaceflight, and is a senior editor at Scientific American. In addition to his work for Scientific American, Billings's writing has appeared in the New York Times, the Wall Street Journal, the Boston Globe, Wired, New Scientist, Popular Science, and many other publications.Billings joined Scientific American in 2014, and previously worked as a staff editor at SEED magazine.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/people-likely-arent-as-susceptible-to-false-memories-as-researchers-thought/'>People Likely Aren't as Susceptible to False Memories as Researchers Thought</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 12:02:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New studies underscore the difficulty of implanting entirely fictional events in a person's recollection We know that our mind keeps an imperfect record of the past. We can forget or misremember details with frustrating consequences. Our attention can be diverted in ways that make it all too easy to miss key events. This concept is often used to cast doubt on the reliability of a plaintiff's testimony in a court case, suggesting it is easy to create false memories of entire events. For example, lawyers representing Harvey Weinstein cited this idea to raise questions about several women's allegations against him. Recently we had the opportunity to take a closer look at this concept by analyzing data from a study that intended to replicate one of the most iconic experiments on false memories to date. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The experiment was published by American psychologists Elizabeth Loftus and Jacqueline Pickrell in 1995. She then wanted to see whether it was possible to implant an entire false memory for a childhood event that had never happened. To that end, in the 1995 study, she and Pickrell misled participants into believing that, according to their parents or older sibling, around age five, they had been lost in a shopping mall and then found by an older woman. Loftus had previously claimed some therapists could implant false memories of childhood sexual abuse in their clients. Her “lost in the mall” experiment therefore offered evidence that such a thing might be possible. In a 2017 paper we identified two big questions that have been hanging over these studies. For example, would the participants themselves agree that they not only believed in the false event on their relative's say-so but had an actual memory of it? And secondly, what exactly was it that the participants remembered? Could some of those recollections have been true memories? And what does a “partial” false memory consist of? Our new analysis digs into these questions and suggests that the body of research on false memory induction must be treated with caution: it is likely much more difficult to convince someone of a false memory than past work has suggested. They used a larger sample of 123 people and reported that 35 percent of participants had a false memory, 10 percent more than in the original study. Murphy's team's data and transcriptions of what participants actually said were made freely available to other researchers, reflecting a move toward greater transparency in psychological research. Those rated as having a full false memory recalled fewer than three of the details on average, while those described as having a partial false memory recalled about one detail. In all cases, these participants described real events that they clearly distinguished from the suggested fake event. One participant said, “My memory is completely different to the other [suggested] memory.” Another said, “I don't really remember that one.... But like me getting lost in the shop was like a regular occurrence.” Others were so uncertain about the suggested details in the fake story that their testimony would have little value in court. Taking everything into account, we estimated that only five participants could reasonably be claimed to have a false memory rather than the 43 that were originally claimed. Their comments revealed, for example, that they compared the scenario with other episodes of being lost, thinking about who would have been present and considering if the mall was as suggested. The great majority of these so-called false memories were much more limited, and held with much less conviction, than the claims made about this type of experiment led us to expect. While these questions remain, psychologists should be very cautious about how they present the findings on memory implantation to others. It is easy to overstate the relevance or generalizability of scientific evidence. Though memory is limited and sometimes mistaken, completely false memories are not easy to implant. Most of the time, memory does a good enough job. And while it is valuable to bring critical distance and skepticism when considering the reliability of memory—particularly in legal contexts—we should not be too quick to throw out a person's testimony simply because it could be imperfect. Are you a scientist who specializes in neuroscience, cognitive science or psychology? Please send suggestions to Scientific American's Mind Matters editor Daisy Yuhas at dyuhas@sciam.com. This is an opinion and analysis article, and the views expressed by the author or authors are not necessarily those of Scientific American. She studies memory, adverse life experiences, post-traumatic stress disorder and research methodology. Chris R. Brewin is an emeritus professor of clinical psychology at University College London.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43016-025-01157-4'>Statistical approaches are inadequate for accurate estimation of yield potential and gaps at regional level</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 11:55:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Metrics details Accurate spatial information on yield potential and gaps is key to determine crop production potential. Although statistical methods are widely used to estimate these parameters at regional to global levels, a rigorous evaluation of their performance is lacking. Here we compared outcomes derived from four published statistical approaches based on highest average farmer yields over time and space against those derived from a ‘bottom-up' approach based on crop modelling and local weather and soil data for major rain-fed crops in the United States. Statistical methods failed to capture spatial variation in water-limited yield potential, consistently under- or overestimating yield gaps across regions. Statistical methods led to conflicting results, with production potential almost doubling from one method to another. We emphasize the need for well-validated crop models coupled with local data, robust spatial frameworks and extrapolation methods to provide more reliable assessments of production potential from local to regional scales. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Data on water-limited yield potential and actual yield from GYGA are available via the yield gap atlas website at www.yieldgap.org. Data on water-limited yield potential and gaps from the GQ, GR and CB statistical methods can be accessed via Zenodo at https://doi.org/10.5281/zenodo.10234041 (ref. 57), via Github at https://github.com/ramiadantsoa/Yield_trends and via Earthstat at http://www.earthstat.org/yield-gaps-climate-bins-major-crops/, respectively. Data on actual farmer yields were retrieved from USDA-NASS Quick Stats at https://www.nass.usda.gov/Quick_Stats/. Soil data for GYGA were retrieved from the gSSURGO soil database at https://www.nrcs.usda.gov/resources/data-and-reports/gridded-soil-survey-geographic-gssurgo-database. Harvested-area data are available via USDA CDL at https://www.nass.usda.gov/Research_and_Science/Cropland/Release/index.php. Source data are provided with this paper. The R code for the current study is publicly available via GitHub at https://github.com/ACouedel/Yield_Gaps_Methods_Comparison. OECD-FAO Agricultural Outlook 2024-2033. OECD-FAO Agricultural Outlook 2024-2033 (OECD-FAO, 2024); https://doi.org/10.4060/cd0991en Cassman, K. G. & Grassini, P. A global perspective on sustainable intensification research. Google Scholar van Ittersum, M. K. et al. Yield gap analysis with local to global relevance—a review. Field Crops Res. Google Scholar Cassman, K. G., Dobermann, A., Walters, D. T. & Yang, H. Meeting cereal demand while protecting natural resources and improving environmental quality. Google Scholar Lobell, D. B., Cassman, K. G. & Field, C. B. Crop yield gaps: their importance, magnitudes, and causes. Van Ittersum, M. K. et al. Can sub-Saharan Africa feed itself? Schils, R. et al. Cereal yield gaps across Europe. Grassini, P. et al. Robust spatial frameworks for leveraging research on sustainable crop intensification. Rattalino Edreira, J. I. et al. Spatial frameworks for robust estimation of yield gaps. Article PubMed Yuan, S. et al. Southeast Asia must narrow down the yield gap to continue to be a major rice bowl. Article PubMed Monzon, J. P. et al. Fostering a climate-smart intensification for oil palm. Grassini, P. et al. How good is good enough? Data requirements for reliable crop yield simulations and yield-gap analysis. Field Crops Res. van Bussel, L. G. J. et al. From field to atlas: upscaling of location-specific yield gap estimates. Field Crops Res. Deng, N. et al. Closing yield gaps for rice self-sufficiency in China. Licker, R. et al. Mind the gap: how do climate and agricultural management explain the ‘yield gap' of croplands around the world? Solutions for a cultivated planet. Neumann, K., Verburg, P. H., Stehfest, E. & Müller, C. The yield gap of global grain production: a spatial analysis. & Hatfield, J. L. Yield gaps and yield relationships in central U.S. soybean production systems. Hatfield, J. L., Wright-Morton, L. & Hall, B. Vulnerability of grain crops and croplands in the Midwest to climatic variability and adaptation strategies. & Hatfield, J. L. Yield and yield gaps in central U.S. corn production systems. Kucharik, C. J., Ramiadantsoa, T., Zhang, J. & Ives, A. R. Spatiotemporal trends in crop yields, yield variability, and yield gaps across the USA. Mueller, N. D. et al. Closing yield gaps through nutrient and water management. Gerber, J. S. et al. Global yield gap time trends reveal regions at risk of future crop yield stagnation. Hatfield, J. L. & Beres, B. L. Yield gaps in wheat: path to enhancing productivity. FAOSTAT (FAO, 2018); https://www.fao.org/faostat/en/#home Couëdel, A. et al. Assessing environment types for maize, soybean, and wheat in the United States as determined by spatio-temporal variation in drought and heat stress. USDA-National Agricultural Statistics Service (NASS) (USDA, 2018). Lollato, R. P., Edwards, J. T. & Ochsner, T. E. Meteorological limits to winter wheat productivity in the U.S. southern Great Plains. Field Crops Res. Rattalino Edreira, J. I. et al. Assessing causes of yield gaps in agricultural areas with diversity in climate and soils. Rizzo, G. et al. Climate and agronomy, not genetics, underpin recent maize yield gains in favorable environments. Grassini, P. et al. in Crop Physiology Ch. 2 (eds Sadras, V. O. & Calderini, D. F.) (Academic Press, 2015); https://doi.org/10.1016/B978-0-12-417104-6.00002-9 Aramburu-Merlos, F., van Loon, M. P., van Ittersum, M. K. & Grassini, P. High-resolution global maps of yield potential with local relevance for targeted crop production improvement. Article PubMed Cassman, K. G. Ecological intensification of cereal production systems: yield potential, soil quality, and precision agriculture. Aramburu Merlos, F. et al. Potential for crop production increase in Argentina through closure of existing yield gaps. Field Crops Res. Schils, R. L. M., van Voorn, G. A. K., Grassini, P. & van Ittersum, M. K. Uncertainty is more than a number or colour: Involving experts in uncertainty assessments of yield gaps. Liu, B. et al. Global wheat production with 1.5 and 2.0 °C above pre-industrial warming. Bassu, S. et al. How do various maize crop models vary in their responses to climate change factors? Couëdel, A. et al. Long term soil organic carbon and crop yield feedbacks differ between 16 soil-crop models in sub-Saharan Africa. & Schultz, J. E. Water use efficiency of wheat in a Mediterranean-type environment. I. the relation between yield, water use and climate. Rattalino Edreira, J. I. et al. Water productivity of rainfed maize and wheat: a local to global perspective. Silva, J. V. et al. Revisiting yield gaps and the scope for sustainable intensification for irrigated lowland rice in Southeast Asia. van Dijk, M. et al. Disentangling agronomic and economic yield gaps: an integrated framework and application. Xie, Y., Lark, T. J., Brown, J. F. & Gibbs, H. K. Mapping irrigated cropland extent across the conterminous United States at 30 m resolution using a semi-automatic training approach on Google Earth Engine. USDA Cropland Data Layer (CDL) (USDA, 2017). van Wart, J. et al. Use of agro-climatic zones to upscale simulated crop yield potential. Field Crops Res. Evans, L. T. Crop Evolution, Adaptation and Yield (Cambridge Univ. van Wart, J., Kersebaum, K. C., Peng, S., Milner, M. & Cassman, K. G. Estimating crop yield potential at regional to national scales. Field Crops Res. Yang, H., Grassini, P., Cassman, K. G., Aiken, R. M. & Coyne, P. I. Improvements to the hybrid-maize model for simulating maize yields in harsh rainfed environments. Field Crops Res. A. et al. An overview of APSIM, a model designed for farming systems simulation. Holzworth, D. P. et al. APSIM—evolution towards a new generation of agricultural systems simulation. & Sinclair, T. R. Modeling Physiology of Crop Development, Growth and Yield (CAB International, 2012). Archontoulis, S. V. et al. Predicting crop yields and soil-plant nitrogen dynamics in the US Corn Belt. Elli, E. F. et al. Climate change and management impacts on soybean N fixation, soil N mineralization, N2O emissions, and seed yield. Jamieson, P. D., Porter, J. R. & Wilson, D. R. A test of the computer simulation model ARCWHEAT1 on wheat crops grown in New Zealand. Field Crops Res. Van Wart, J. et al. Creating long-term weather data from thin air for crop simulation modeling. R Core Team R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2023). Gerber, J. S. et al. Global spatially explicit yield gap time trends reveal regions at risk of future crop yield stagnation. This study was supported by the National Science Foundation (grant number 2214604 to P.G. ), the National Institute of Food and Agriculture of the United States Department of Agriculture (Hatch NEB-22-373 and Agriculture and Food Research Initiative grant number 12431808 to P.G. ), the Foundation for Food and Agricultural Research (grant number 534264 to S.V.A.) and the Kansas Wheat Commission (grant number 20-1898 to R.P.L. Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE, USA Antoine Couëdel, Fatima A. Tenorio, Fernando Aramburu-Merlos, Juan I. Rattalino Edreira & Patricio Grassini AIDA, University Montpellier, CIRAD, Montpellier, France CIRAD, UPR AIDA, Montpellier, France Department of Agronomy, Kansas State University, Manhattan, KS, USA Department of Agronomy, Iowa State University, Ames, IA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Research was conceived by A.C. and P.G. ran model simulations. Data acquisition, data processing and statistical analysis were performed by A.C. The paper was written by A.C. and P.G. with input from all authors. Correspondence to Patricio Grassini. The authors declare no competing interests. Nature Food thanks Paul Struik and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. State acronyms: Colorado (CO), Iowa (IA), Illinois (IL), Indiana (IN), Kansas (KS), Kentucky (KY), Michigan (MI), Missouri (MO), Montana (MT), Nebraska (NE), Ohio (OH), Oklahoma (OK), North Dakota (ND), New Mexico (NM), South Dakota (SD), Texas (TX), and Wyoming (WY). Source of harvested area is USDA CDL (2022) at 30×30 m resolution. Simulated yield potential plotted against measured yields in well-managed experiments conducted across a wide range of environments. Details are provided elsewhere 28,48,52,53. Root mean square error (RMSE), relative RMSE as percentage of the average measured yield (rRMSE, in %), and mean error (ME, Mg ha-1) are indicated. Diagonal line indicates y = x. Methods to estimate yield potential include bottom-up approach of GYGA based on well-validated crop models and high-quality weather and soil data, quantile regression based on Ya at local or global levels (LQ and GQ, respectively), Ya distribution across a large geographic region (GR), and Ya distribution within global climate bins (CB). Each datapoint represents the average yield potential estimated at each site following each method. Red crosses represent the weighted US average according to the crop area at each site. Diagonal line indicates y = x. Note that range of NASS Ya are not identical across comparisons as they were selected to match the time period for which Yw was available for each method. Likewise, sites from two states (Nebraska and Kansas) with substantial rainfed and irrigated maize and soybean area were not included for the GQ method because associated yield potential was not disaggregated by water regime. Methods to estimate yield potential include the bottom-up modelling approach of the Global Yield Gap Atlas (GYGA) based on well-validated crop models and high-quality weather and soil data, quantile regression based on Ya at local or global levels (LQ and GQ, respectively), Ya distribution across a large geographic region (GR), and Ya distribution within global climate bins (CB). The yield gap was estimated as the differences between average yield potential and average farmer yield. Each circle represents the average long-term yield gap at each site as estimated by each method. Sites from two states (Nebraska and Kansas) with substantial rainfed and irrigated maize and soybean area were not included for the GQ method because associated yield potential was not disaggregated by water regime. Maize and soybean maps show the US Corn Belt wheat maps show the US Central Great Plains. More details about the US Corn Belt and Great Plains are available in Extended Data Fig. Supplementary Tables 1 and 2. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Couëdel, A., Lollato, R.P., Archontoulis, S.V. et al. Statistical approaches are inadequate for accurate estimation of yield potential and gaps at regional level. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/surgeons-can-permanently-change-eye-color-with-experimental/'>Surgeons Can Now Permanently Change Human Eye Color—But Should They?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 11:01:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An Experimental Eye-Color-Changing Surgery Is Gaining Popularity—Here's What We Know about It A new surgical procedure to permanently change eye color is gaining traction in the U.S. Some people who have always wanted a different eye color, like this vibrant green shade above, are considering a new surgery called cosmetic keratopigmentation to permanently change them. Miranda, a now 49-year-old sales associate, had dreamed since childhood of having hazel or green eyes. For decades she'd worn colored contacts and admired how the lenses “softened” her facial features. She sometimes wore them to bed or the beach despite doctors' warnings. For the procedure, Miranda lay flat on an operating table opposite a large, machine-guided laser. Speculums bared her eyes wide as numbing drops desensitized the tissues. In under 20 minutes and with minimal discomfort, the transformation was complete: Miranda's honey-colored eyes looked “surprisingly natural,” she says. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. Yet public demand has recently surged, in part because of viral social media videos. Miranda's surgeon and corneal specialist, Brian Boxer Wachler, has performed hundreds ofcosmetic keratopigmentation procedures since adding it to his practice in early 2024. Another well-established practitioner, Alexander Movshovich, told the Wall Street Journal last November that he was on track to perform more than 400 surgeries in 2024. Some, like Miranda, believe different colors better suit their appearance. Others, Boxer Wachler says, find solace in having an eye color that resembles a family member's, living or deceased. After receiving their new eye color, “I've had patients be giddy with laughter or cry of happiness,” Boxer Wachler says. In September 2024, surgeon Boxer Wachler transformed this client's eye color from brown to a “medium-intensity evergreen.” But iris implantation routinely causes chronic inflammation, glaucoma, cataracts and vision loss. (At around $6,000 per eye, however, cosmetic keratopigmentation costs more than twice that of LASIK.) In December 2024, Boxer Wachler transformed another client's eye color from brown to a “medium-intensity emerald green.” Early studies indeed suggest that serious adverse effects of cosmetic keratopigmentation are rare, and practitioners market the surgery as safe. But many ophthalmologists emphasize that research on the procedure features small sample sizes and short follow-ups, leaving crucial questions about long-term effects. In fact, it is not standardized or regulated by any American medical organization. “I tell my students, ‘You want to be on the cutting edge, not the bleeding edge,'” says Roberto Pineda, a corneal specialist at Massachusetts Eye and Ear. Scarring of the cornea may also occur from potential infection. Boxer Wachler, Movshovich and two doctors who pioneered cosmetic keratopigmentation promptly wrote to the academy, urging it to withdraw its warning and citing research they felt demonstrated the procedure's safety. The academy has stood by its statement, says AAO spokesperson Thomas Steinemann, an ophthalmologist at MetroHealth. Boxer Wachler says few lasting adverse effects have been reported in the procedure's medical literature. Of the 29 people who developed complications, 49 percent suffered light sensitivity that tended to resolve after six months; 19 percent saw their new eye color fade or change; and 4 and 2 percent, respectively, experienced slight visual field limitations or pain in magnetic resonance imaging (MRI) machines. A 2021 study conducted by some of the same co-authors, who surveyed 40 cosmetic keratopigmentation clients two and a half years after their operation, reported similar but less frequent complications. More recent research documented five cosmetic keratopigmentation recipients who later developed ectasia, a corneal bulge that can distort vision without treatment. He and Movshovich say they've pioneered techniques to remove some of a client's dye if needed, but the pigment generally persists in eyes. Some serious complications build gradually, as with iris implantation, she notes. This treatment, also performed off-label, is an option for people born without irises or missing parts of them, which causes serious visual glare that can make daytime activities such as driving difficult or painful; in medical keratopigmentation, surgeons inject dye over missing regions of the iris to help block excessive light. Pineda, who began performing the therapeutic procedure in the late 1990s, notes that reports of side effects are limited but can include pigment fading and, in rare circumstances, spreading. A decades-long randomized controlled trial with hundreds or thousands of people who have received cosmetic keratopigmentation could best help doctors understand the procedure's long-term implications. But the logistics would likely be too expensive or burdensome for test subjects, says ophthalmologist Kevin Miller of UCLA Health. Doctors who perform the surgery can voluntarily document people's reports of severe complications or conduct limited survey-based studies—but they are not required to. Boxer Wachler says he declines to operate on people with a history of LASIK, eye inflammation, present or prior autoimmune conditions or other conditions that might raise the risk of complications. He treats clients at risk for glaucoma on a case-by-case basis. Steinemann advises anyone with a strong family history of eye disease to steer clear of the procedure, noting that it might obstruct surgeries needed to treat any future conditions. Boxer Wachler encourages prospective cosmetic keratopigmentation receivers to seek out experienced providers for the safest and most natural-looking results. Many ophthalmologists recommend that people interested in changing their eye color stick to colored contacts—a cheaper option they say poses minimal risk when used appropriately. The AAO urges consumers to get their cosmetic contact fittings, prescriptions and use instructions from eye care professionals. “I'd ask people to remember,” Steinemann says, “that with your eyes, you don't get a second chance.”</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01064-5'>Cells are swapping their mitochondria. What does this mean for our health?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 10:32:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Gemma Conroy is a freelance science journalist based in Mexico City. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Ever since they were discovered in the mid-nineteenth century, mitochondria have been known as organelles that reside inside cells. But that textbook picture now seems to be wrong. An explosion of research is challenging mitochondria's long-standing image as exclusively cellular organelles. “They may be a multicellular organelle,” says Jonathan Brestoff, an immunologist who studies metabolism at Washington University in St. Louis, Missouri. In other words, the supposedly static energy factories now seem to be expert travellers, skipping from one cell to another on demand. Cancer cells ‘poison' the immune system with tainted mitochondria Cancer cells ‘poison' the immune system with tainted mitochondria This ‘mitochondrial transfer' has been observed in a wide variety of cells and in organisms as diverse as yeast, molluscs and rodents. “It's really exciting to see,” says Jeffrey Spees, a stem-cell biologist at the University of Vermont in Burlington. It's not yet clear why mitochondria are so mobile. In cellular emergencies, newly arrived mitochondria might kick-start tissue repair, fire up the immune system or rescue distressed cells from death. But what this means for human health is still a mystery. Researchers haven't yet captured the process inside the human body and so don't know for sure if it happens in people, says Daniel Davis, an immunologist at Imperial College London. “We don't have the technology yet to witness this happening,” he says. That fact hasn't stopped researchers from exploring how to leverage mitochondrial transfer to treat a variety of diseases, including cancer and stroke. Over the past three decades, research has revealed that mitochondria are much more than cellular powerhouses that turn nutrients from food into energy. Last year, researchers found that mitochondria divide themselves into two distinct forms to help cells survive nutrient starvation1. All mitochondria — in whatever organism, in whatever part of the body — are thought to have sprung from the same ancient bacterium. Around 1.5 billion years ago, this drifting bacterium was swallowed up by the microbe that eventually gave rise to eukaryotes — the large group of organisms, including us, whose cells have an enclosed nucleus. Scientists make precise gene edits to mitochondrial DNA for first time Scientists make precise gene edits to mitochondrial DNA for first time After several evolutionary twists and turns, this borrowed bacterium became the organelle that drives metabolism. Mitochondria's microbial origins probably help to explain why they are more dynamic than they first appear, says Kazuhide Hayakawa, a neuroscientist at Massachusetts General Hospital in Boston, who studies how mitochondrial transfer could help to treat stroke. In 2006, Spees and his colleagues captured the first glimpse of mitochondria skipping from one cell to another3. The team had been trying to understand a perplexing behaviour of stem cells in laboratory dishes. To investigate mitochondria's role, the researchers cultured human lung cancer cells, which lacked the organelles, with stem cells sourced from bone marrow. Since then, researchers have observed mitochondria zipping between several types of cell — lung, heart, brain, fat, bone and more. Sometimes the mitochondria are travelling down ephemeral highways known as tunnelling nanotubes that form between cells and transport other cellular cargo. Researchers are learning that the process is often a form of cellular damage control, says Clair Crewe, a cell biologist at Washington University. Some studies suggest, for example, that mitochondrial transfer might help cells to weather neurological storms. In 2016, Hayakawa and his colleagues found that in mice that have had a stroke, support cells called astrocytes deliver their mitochondria to faltering neurons. Lung cells might also benefit from a mitochondrial boost during a crisis, says Jahar Bhattacharya at Columbia University in New York City, who specializes in a severe inflammatory condition known as acute lung injury. He and his colleagues have found that in mice with this inflammation, stromal cells — which make up connective tissues that support organs — transfer their mitochondria to lung cells6. Cells with loaned organelles had higher concentrations of the cellular fuel ATP, which ended up being distributed to nearby cells that did not receive new mitochondria. Bhattacharya was amazed when he and his team witnessed mitochondrial transfer in action. “I don't think we slept for the next few nights, it was so exciting,” he says. Other research hints that transferred mitochondria might supercharge wound healing. In 2021, Anne-Marie Rodriguez, a cell biologist at Sorbonne University in Paris, and her colleagues found that platelets isolated from human blood shuttled their mitochondria to stem cells when researchers put the two cell types into a dish together. When the cells were placed onto skin wounds in mice, those injuries healed faster than did injuries in rodents that had received either stem cells or platelets alone7. Researchers suspect that cells with dysfunctional mitochondria might even have ways to request healthy mitochondria from their neighbours, although the exact mechanisms underlying this process remain murky. “We're only beginning to understand the signalling that's involved,” says Crewe. Beyond its role in recovery, researchers want to know whether mitochondrial transfer is an essential part of everyday biology. Initial evidence suggests it might help to maintain healthy tissues. Last year, Minghao Zheng, a regenerative biologist at the University of Western Australia in Perth, and his colleagues discovered that some types of astrocyte donate their mitochondria to cells that line blood vessels in the mouse brain8. When the researchers disrupted this process, the blood–brain barrier became leaky, suggesting that mitochondrial transfer helps to maintain this protective membrane shield. Zheng and his team had already reported that mitochondrial transfer in the bones of mice can accelerate the formation of new blood vessels9. In healthy mice, Brestoff and his colleagues reported, white fat cells transfer their mitochondria to macrophages — white blood cells that hoover up cellular debris. The number of shuttled organelles was reduced in obese mice. The obese mice also burnt less energy than their healthy counterparts10. These organelles might help macrophages to function when their metabolism is disrupted, says Brestoff. In the labyrinthine world of the immune system, donated mitochondria might have an anti-inflammatory effect, especially when they are taken up by T cells — white blood cells that stave off infections and disease. In studies in cell cultures, Patricia Alejandra Luz-Crawford, an immunologist at the University of the Andes in Santiago, Chile, and her colleagues found that some types of T cell that receive mitochondria from stem cells produce fewer inflammatory molecules. Stem cells cultured from people with rheumatoid arthritis pass fewer mitochondria along to T cells than do stem cells from healthy individuals, which she says might contribute to the chronic inflammation associated with the disease11. But there are many unanswered questions about mitochondrial transfer, including what the organelles might be doing after they enter cells and how long they last, says Luz-Crawford. The lack of detail about why cells transfer their mitochondria makes it hard to know what specific role these cellular exchanges might have in conditions such as cardiovascular disease and obesity, says Rodriguez. In vivo studies have tracked mitochondria in only a handful of tissue types, making it difficult to build a bigger picture of the wider impacts these transfers have on health. Scientists make precise gene edits to mitochondrial DNA for first time Cancer cells ‘poison' the immune system with tainted mitochondria The contribution of de novo coding mutations to meningomyelocele 154 million lives and counting: 5 charts reveal the power of vaccines An animal source of mpox emerges — and it's a squirrel Long COVID activists fought Trump team's research cuts and won ― for now Mutations that accrue through life set the stage for stomach cancer Zhejiang University (ZJU) International Campus in Haining, China, about 120km southwest of Shanghai Wuxi Medical College of Jiangnan University (Affiliated Hospital) invites you to join us! Scientists make precise gene edits to mitochondrial DNA for first time Cancer cells ‘poison' the immune system with tainted mitochondria An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02155-9'>Genetic regulation of gene expression across multiple tissues in chickens</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 09:49:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The chicken is a valuable model for understanding fundamental biology and vertebrate evolution and is a major global source of nutrient-dense and lean protein. Despite being the first non-mammalian amniote to have its genome sequenced, a systematic characterization of functional variation on the chicken genome remains lacking. Here, we integrated bulk RNA sequencing (RNA-seq) data from 7,015 samples, single-cell RNA-seq data from 127,598 cells and 2,869 whole-genome sequences to present a pilot atlas of regulatory variants across 28 chicken tissues. This atlas reveals millions of regulatory effects on primary expression (protein-coding genes, long non-coding RNA and exons) and post-transcriptional modifications (alternative splicing and 3′-untranslated region alternative polyadenylation). We highlighted distinct molecular mechanisms underlying these regulatory variants, their context-dependent behavior and their utility in interpreting genome-wide associations for 39 chicken complex traits. Finally, our comparative analyses of gene regulation between chickens and mammals demonstrate how this resource can facilitate cross-species gene mapping of complex traits. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout All raw data analyzed in this study are publicly available for download without restrictions from the Sequence Read Archive (https://www.ncbi.nlm.nih.gov/sra) and National Genomics Data Center (NGDC) BioProject (https://bigd.big.ac.cn/bioproject) databases. The GRCg6a chicken reference genome (v.102) is available at Ensembl (https://www.ensembl.org). Details of RNA-seq, WGS, chromatin immunoprecipitation sequencing peaks and single-cell RNA-seq can be found in Supplementary Tables. All processed data, including metadata, the genotype imputation reference panel, molecular phenotypes, imputed genotypes and the summary statistics of molQTL and GWAS, are available on Zenodo (https://doi.org/10.5281/zenodo.14902956)102 and http://chicken.farmgtex.org. Source data are provided with this paper. All the computational scripts and codes for RNA-seq, WGS, single-cell RNA-seq and Hi-C dataset analyses as well as the respective quality control, molecular phenotype normalization, genotype imputation, molQTL mapping, functional enrichment, colocalization, SMR and TWAS are available on Zenodo (https://doi.org/10.5281/zenodo.14902956)102 and the FarmGTEx GitHub website (https://github.com/guandailu/ChickenGTEx_pilot_phase). Hillier, L. W. & Miller, W. Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution. Burt, D. W. Emergence of the chicken as a model organism: implications for agriculture and biology. Beacon, T. H. & Davie, J. R. The chicken model organism for epigenomic research. Garcia, P., Wang, Y., Viallet, J. The chicken embryo model: a novel and relevant model for immune-based studies. The genetic architecture of domestication in the chicken: effects of pleiotropy and linkage. & Burggren, W. W. Beyond the chicken: alternative avian models for developmental physiological research. Brown, W. R. A., Hubbard, S. J., Tickle, C. & Wilson, S. A. The chicken as a model for large-scale analysis of vertebrate gene function. Heterogeneity of a dwarf phenotype in Dutch traditional chicken breeds revealed by genomic analyses. Wang, M.-S. et al. An evolutionary genomic perspective on the breeding of dwarf chickens. van der Eijk, J. A. J. et al. Chicken lines divergently selected on feather pecking differ in immune characteristics. Lillie, M. et al. Genomic signatures of 60 years of bidirectional selection for 8-week body weight in chickens. Smith, J. et al. Fourth report on chicken genes and chromosomes 2022. Kern, C. et al. Functional annotations of three domestic animal genomes provide vital resources for comparative and agricultural research. Pan, Z. et al. An atlas of regulatory elements in chicken: a resource for chicken genetics and genomics. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Aguet, F. et al. Genetic effects on gene expression across human tissues. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Liu, S. & Fang, L. The CattleGTEx atlas reveals regulatory mechanisms underlying complex traits. Liu, S. et al. A multi-tissue atlas of regulatory variants in cattle. Teng, J. et al. A compendium of genetic regulatory effects across pig tissues. Faced with inequality: chicken do not have a general dosage compensation of sex-linked genes. Nicholas, F. W. Online Mendelian inheritance in animals (OMIA): a comparative knowledgebase of genetic disorders and other familial traits in non-laboratory animals. Wang, Z. et al. An EAV-HP Insertion in 5′ flanking region of SLCO1B3 causes blue eggshell in the chicken. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Guan, L., Yang, Q., Gu, M., Chen, L. & Zhang, X. Exon expression QTL (eeQTL) analysis highlights distant genomic variations associated with splicing regulation. Li, Y. I. et al. Annotation-free quantification of RNA splicing using LeafCutter. Improving fine-mapping by modeling infinitesimal effects. Storey, J. D. & Tibshirani, R. Statistical significance for genomewide studies. & Troyanskaya, O. G. Predicting effects of noncoding variants with deep learning-based sequence model. Robins, C. et al. Genetic control of the human brain proteome. Guan, D. et al. Profiling chromatin contacts at micro-scale in the chicken genome. Noda, D. et al. ELAC2, a putative prostate cancer susceptibility gene product, potentiates TGF-β/Smad-induced growth arrest of prostate cells. Hu, H. et al. AnimalTFDB 3.0: a comprehensive resource for annotation and prediction of animal transcription factors. Urbut, S. M., Wang, G., Carbonetto, P. & Stephens, M. Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions. Wang, Y. et al. Genetic dissection of growth traits in a unique chicken advanced intercross line. Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Gamazon, E. R. et al. A gene-based association method for mapping traits using reference transcriptome data. Integrating predicted transcriptome from multiple tissues improves association detection. Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics. Barbeira, A. N. et al. Exploiting the GTEx resources to decipher the mechanisms at GWAS loci. Mostafavi, H., Spence, J. P., Naqvi, S. & Pritchard, J. K. Systematic differences in discovery of genetic effects on gene expression and complex traits. The missing link between genetic association and regulatory function. Sowa, A. S. et al. Karyopherin α-3 is a key protein in the pathogenesis of spinocerebellar ataxia type 3 controlling the nuclear localization of ataxin-3. Cao, J. et al. Effect of combinations of monochromatic lights on growth and productive performance of broilers. Artificial polychromatic light affects growth and physiology in chicks. Dominant KPNA3 mutations cause infantile-onset hereditary spastic paraplegia. Santhanam, N. et al. RatXcan: a framework for cross-species integration of genome-wide association and gene expression data. Naqvi, S. et al. Conservation, acquisition, and functional impact of sex-biased gene expression in mammals. Li, J. et al. Genome-wide association studies for egg quality traits in White Leghorn layers using low-pass sequencing and SNP chip data. Qi, T. et al. Genetic control of RNA splicing and its distinct role in complex trait variation. Li, L. et al. An atlas of alternative polyadenylation quantitative trait loci contributing to complex trait and disease heritability. Munro, D. et al. Multimodal analysis of RNA sequencing data powers discovery of complex trait genetics. Prediction of transcript isoforms in 19 chicken tissues by Oxford Nanopore long-read sequencing. Kuo, R. I. et al. Normalized long read RNA sequencing in chicken reveals transcriptome complexity similar to human. Thomas, S., Underwood, J. G., Tseng, E., & Holloway, A. K. Long-read sequencing of chicken transcripts and identification of new transcript isoforms. Zhang, J. et al. Comprehensive analysis of structural variants in chickens using PacBio sequencing. Mobile element variation contributes to population-specific genome diversification, gene regulation and disease risk. Li, M. et al. De novo assembly of 20 chicken genomes reveals the undetectable phenomenon for thousands of core genes on microchromosomes and subtelomeric regions. Discovery of target genes and pathways at GWAS loci by pooled single-cell CRISPR screens. Implications of gene inheritance patterns on the heterosis of abdominal fat deposition in chickens. Yi, G. et al. In-depth duodenal transcriptome survey in chickens with divergent feed efficiency using RNA-Seq. Integrated analysis of lncRNA and mRNA repertoires in Marek's disease infected spleens identifies genes relevant to resistance. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Jehl, F. et al. An integrative atlas of chicken long non-coding genes and their annotations across 25 tissues. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Kovaka, S. et al. Transcriptome assembly from long-read RNA-seq alignments with StringTie2. Chen, C. et al. TBtools: an integrative toolkit developed for interactive analyses of big biological data. Yu, G., Smith, D. K., Zhu, H., Guan, Y. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. van der Maaten, L. & Hinton, G. Visualizing data using t-SNE. Dynamic analyses of alternative polyadenylation from RNA-seq reveal a 3′-UTR landscape across seven tumour types. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Zhong, C. et al. Age-dependent genetic architectures of chicken body weight explored by multidimensional GWAS and molQTL analyses. Runs of homozygosity and selection signature analyses reveal putative genomic regions for artificial selection in layer breeding. Jin, J. et al. Calcium deposition in chicken eggshells: role of host genetics and gut microbiota. Degalez, F. et al. Enriched atlas of lncRNA and protein-coding genes for the GRCg7b chicken assembly and its functional annotation across 47 tissues. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. The Sequence Alignment/Map format and SAMtools. Danecek, P. et al. Twelve years of SAMtools and BCFtools. & Browning, S. R. A one-penny imputed genome from next-generation reference panels. Stegle, O., Parts, L., Piipari, M., Winn, J. & Durbin, R. Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Principal components analysis corrects for stratification in genome-wide association studies. Scaling computational genomics to millions of individuals with GPUs. Fast and efficient QTL mapper for thousands of molecular phenotypes. Controlling the false discovery rate: a practical and powerful approach to multiple testing. R: a language and environment for statistical computing (R Foundation for Statistical Computing, 2022). Yin, L. et al. rMVP: a memory-efficient, visualization-enhanced, and parallel-accelerated tool for genome-wide association study. VanRaden, P. M. Efficient methods to compute genomic predictions. Zou, Y., Carbonetto, P., Wang, G. & Stephens, M. Fine-mapping from summary data with the “sum of single effects” model. Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. Applying meta-analysis to genotype-tissue expression data from multiple tissues to identify eQTLs and increase the number of eGenes. Speagle, J. S. A conceptual introduction to Markov chain Monte Carlo methods. & Robinson, D. qvalue: Q-value estimation for false discovery rate control. Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. Lee, Y., Luca, F., Pique-Regi, R. & Wen, X. Bayesian multi-SNP genetic association analysis: control of FDR and use of summary statistics. Wen, X., Lee, Y., Luca, F. & Pique-Regi, R. Efficient integrative multi-SNP association analysis via deterministic approximation of posteriors. Wen, X. Molecular QTL discovery incorporating genomic annotations using Bayesian false discovery rate control. Bhattacharya, A. et al. Best practices for multi-ancestry, meta-analytic transcriptome-wide association studies: lessons from the Global Biobank Meta-analysis Initiative. Guan, D. Chicken genotype-tissue expression (ChickenGTEx) project. We thank all the researchers who have contributed to the publicly available data used in this research. We extend special acknowledgment to the human GTEx consortium15 for their sharing of computational codes for some data analyses and figure generation. L.F. was supported by Agriculture and Food Research Initiative Competitive grants no. and L.F.) from the US Department of Agriculture (USDA) National Institute of Food and Agriculture, and seed-funding from CellFood Hub (Aarhus University Foundation, AUFF). was supported by Agriculture and Food Research Initiative Competitive grants nos. and L.F.) from the USDA National Institute of Food and Agriculture. acknowledges funding from Agriculture and Food Research Initiative Competitive grant nos. N.Y. acknowledges fundings from the National Key Research and Development Program of China (2021YFD1300600 and 2022YFF1000204). acknowledges funding from the National Natural Science Foundation of China (31961133003) and the support of the high-performance computing platform of the National Research Facility for Phenotypic and Genotypic Analysis of Model Animals (Beijing). Yuzhe Wang was supported by the National Natural Science Foundation of China (32272862). S. Rong acknowledges funding from Jiangsu Agricultural Industry Technology System (JATS[2022]406). Zhe Zhang acknowledges fundings from the National Natural Science Foundation of China (32022078 to Zhe Zhang), the Local Innovative and Research Teams Project of Guangdong Province (2019BT02N630 to Q.N. Zhang Zhang acknowledges funding from National Natural Science Foundation of China (32030021), National Key Research and Development Program of China (2021YFF0703702) and Technical Support Talent Program of Chinese Academy of Sciences (awarded to D.Z.). acknowledges funding from the National Key Research and Development Program of China (2024YFF1000100) and the Science and Technology Innovation 2030–Major Project (2022ZD04017). were supported by Science and Technology Planning Project of Guangzhou City (201504010017) and Natural Scientific Foundation of China (31402067). X. Zhao acknowledges funding from the Natural Sciences and Engineering Research Council of Canada Discovery grant (RGPIN-2022-03884). acknowledges funding from the Yunnan Fundamental Research Projects (202301AW070012, 202401AV070007). was supported in part by USDA NIFA AFRI grant nos. 2019-67015-29321 and 2021-67015-33409 and the appropriated project 8042-31000-112-00-D of the USDA Agricultural Research Service. These authors jointly supervised this work: Ning Yang, Xiaoxiang Hu, Huaijun Zhou, Lingzhao Fang. Department of Animal Science, University of California-Davis, Davis, CA, USA Dailu Guan, Ying Wang & Huaijun Zhou Center for Quantitative Genetics and Genomics (QGG), Aarhus University, Aarhus, Denmark Zhonghao Bai, Di Zhu, Houcheng Li, Albert Johannes Buitenhuis, Goutam Sahana, Mogens Sandø Lund & Lingzhao Fang State Key Laboratory of Animal Biotech Breeding, College of Biological Sciences, China Agricultural University, Beijing, China State Key Laboratory of Animal Biotech Breeding and Frontier Science Center of Molecular Design Breeding, College of Animal Science and Technology, China Agricultural University, Beijing, China Conghao Zhong, Fangren Lan, Xiaochang Li, Congjiao Sun & Ning Yang Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing, China Yali Hou, Zhangyuan Pan & Wei Si State Key Laboratory of Swine and Poultry Breeding Industry, Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou, China Shuqi Diao, Yahui Gao, Jinyan Teng, Zhiting Xu, Qing Lin, Zhenhui Li, Qinghua Nie, Xiquan Zhang & Zhe Zhang MRC Human Genetics Unit at the Institute of Genetics and Cancer, The University of Edinburgh, Edinburgh, UK Jiangsu Livestock Embryo Engineering Laboratory, College of Animal Science and Technology, Nanjing Agricultural University, Nanjing, China Animal Genomics and Improvement Laboratory, Henry A. Wallace Beltsville Agricultural Research Center, Agricultural Research Service, USDA, Beltsville, MD, USA Yahui Gao & George E. Liu Department of Animal and Avian Sciences, University of Maryland, College Park, MD, USA Beijing Institute of Genomics, Chinese Academy of Sciences and China National Center for Bioinformation, Beijing, China Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Livestock and Poultry Multi-omics of MARA, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China Poultry Institute, Chinese Academy of Agricultural Sciences, Yangzhou, China Shourong Shi & Dan Shao PEGASE, INRAE, Institut Agro, Saint Gilles, France Fabien Degalez & Sandrine Lagarrigue State Key Laboratory of Genetic Evolution & Animal Models and Yunnan Key Laboratory of Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, China INRAE, GABI, AgroParisTech, Université Paris-Saclay, Jouy‐en‐Josas, France Dominique Rocha & Mathieu Charles The Roslin Institute, Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Midlothian, UK Jacqueline Smith & Kellie Watson Department of Animal Sciences, Data Science and Informatics Institute, University of Missouri, Columbia, MO, USA Palaeogenomics Group, Department of Veterinary Sciences, Ludwig Maximilian University, Munich, Germany School of Biological and Behavioural Sciences, Queen Mary University of London, London, UK The Palaeogenomics & Bio-Archaeology Research Network, School of Archaeology, University of Oxford, Oxford, UK Department of Animal Science, Iowa State University, Ames, IA, USA Department of Animal Science, McGill University, Montreal, Quebec, Canada Scotland's Rural College (SRUC), Roslin Institute Building, Midlothian, UK College of Animal Science and Technology, Hunan Agricultural University, Changsha, China State Key Laboratory of Swine and Poultry Breeding Industry, Guangdong Key Laboratory of Animal Breeding and Nutrition, Institute of Animal Science, Guangdong Academy of Agricultural Sciences, Guangzhou, China Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Laboratory of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, College of Animal Science, South China Agricultural University, Guangzhou, China Zhenhui Li, Qinghua Nie & Xiquan Zhang Cambridge-Baker Systems Genomics Initiative, Baker Heart and Diabetes Institute, Melbourne, Victoria, Australia School of Agriculture, Food and Ecosystem Sciences, The University of Melbourne, Parkville, Victoria, Australia School of Life Sciences, Westlake University, Hangzhou, China Avian Disease and Oncology Laboratory, USDA, ARS, USNPRC, East Lansing, MI, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed bioinformatic analyses of RNA-seq data analysis. conducted WGS data analysis. D. Zhu performed the deep-learning analysis. performed multi-omics and single-cell RNA-seq data analysis. performed GWAS integrative analysis. led the comparison of GTEx between chickens and mammals. L.F., H. Zhou, D.G., X. Zhu, Q.L., C.Z., Y.H., Yuzhe Wang, C.S., J.T., F.D., S.L., Ying Wang, M.W., M.P., D.R., M.C., J.S., K.W., A.J.B., W.W., L. Frantz, G.L., M.S.L., G.S., S.S., D.S., S.J.L., X. Zhao, X.R., S.L., B.L., H. Zhang and H.C. contributed to the critical interpretation of analytical results before and during manuscript preparation. H. Zhou, L. Fang, N.Y., X.H., G.E.L., Zhe Zhang, S.S., D.S., X. Zhao, Q.N., Z.L., W.L., H.Q., W.S. contributed to the data and computational resources. and L. Fang drafted the manuscript. All authors read, edited and approved the final manuscript. Correspondence to Ning Yang, Xiaoxiang Hu, Huaijun Zhou or Lingzhao Fang. The authors declare no competing interests. Nature Genetics thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Gene expression levels categorized by the number of tissues in which genes are expressed (defined as > 0.1 TPM in >80% of RNA-Seq samples). TPM: Transcripts per Million. (b) Functional enrichment analysis of tissue-specific genes using the Gene Ontology (GO) database. (c) Number of tissue-specific genes associated with at least one epigenetic regulator in the respective tissue. The annotation of epigenetic regulators was retrieved from Pan et al.14. (d) Proportion of tissue-specific genes linked to at least one regulator (Regulator-specific) across tissues. (e) Differentially expressed liver genes (n = 327, shown at x-axis) by sex. Genes were identified according to expression differences between males and females. Each bar represents a gene being tested. (f) Ideogram depicting the chromosomal locations of 17 genes (triangles) exhibiting sex-biased expression across 18 tissues. The color scale represents gene density within a 1 Mb window. (g) An example illustrating the expression of SLCO1B3 across 27 tissues (n = 5,273 samples) and the chromatin states surrounding it across 23 tissues14. SLCO1B3 expression in the liver (n = 903) is significantly higher than in the retina (Student's two-sided t-test, Bonferroni-adjusted P = 1.12 × 10-239). Colors ranging from light to dark indicate increasing chromatin interaction strength. TssA: strongly active promoters/transcripts, TssAHet: flanking active TSS without ATAC, TxFlnk: transcribed at gene, TxFlnkWk: weak transcribed at gene, TxFlnkHet: transcribed region without ATAC, EnhA: strong active enhancer, EnhAMe: medium enhancer with ATAC, EnhAWk: weak active enhancer, EnhAHet: active enhancer no ATAC (hetero), EnhPois: poised enhancer, ATAC_Is: ATAC island, TssBiv: bivalent/poised TSS, Repr: repressed polycomb, ReprWk: weak repressed polycomb. (a) Correlation between the number of high-quality SNPs within cis-windows ( ± 1 Mb of the transcriptional start site (TSS) and the corresponding gene expression level (log2 scaled). (b) Number of high-quality SNPs as a function of tissue specificity of gene expression, measured by the Tau value. P values were calculated via the asymptotic t approximation. (c) Percentage of high-quality SNPs within epigenetic regulators14. (d) Percentage of epigenomic regulators containing at least one high-quality SNP. (e) Proportion of imputed and reference genotypes categorized by variant type. (f) Genotype concordance and Spearman's correlation between paired whole genome sequences (WGS) and RNA-Seq samples across 6 independent populations. (g) Genotype concordance and Spearman's correlation (mean ± s.d.) between paired WGS and RNA-Seq samples. Significance was assessed using a two-sided Student's t-test. (i) Relationship between the median expression levels of genes and the ratio of imputed to observed SNPs. Significance was obtained with two-sided Student's t-test. (j) Number of imputed and observed SNPs as a function of distance to the TSS, stratified by the median expression levels of genes across samples. Significance was assessed using a two-sided Student's t-test. (b) Gene expression levels, measured as Transcripts per Million (TPM) for eGenes detected in at least one tissue and non-eGenes across 28 tissues. (c) Spearman correlations between lead eQTL effect sizes and gene expression across 28 tissues. ): not tested due to low expression levels; Uncorrelated: tested but not significantly correlated; Uninterpretable: significant but ambiguous correlation direction due to effect sizes crossing zero; Positive corr. : positively correlated; Negative corr. (d) Conditionally independent eQTLs across 28 tissues. The proportion of eGenes with different numbers of independent eQTLs is shown as blue stacked bars (left y-axis), and the mean number of independent eQTLs per eGene is represented by red dots (right y-axis). Tissues are sorted by increasing sample size. A tissue color legend is provided at the right. The figure is generated using human GTEx project codes15. (e) Linear regression slope as a function of allelic fold change (aFC) at log2 scale of eQTLs in the liver (n = 741). (f) Proportion of detected eQTLs across different effect size categories (from left to right panels) as a function of tissue sample size. Dot colors correspond to the legend in panel (d). (g)-(i) Down-sampling analyses of eGene and eQTLs. The panel (g) depicts the number of eGenes (left y-axis) detected and mean eQTLs per eGene (right y-axis) at different sample sizes. The middle panel (h) shows the proportion of detected eQTLs with large effects (absolute log2aFC ≥ 1, left y-axis) and small effects (absolute log2aFC ≤ 0.25, right y-axis). The right panel (i) presents the number of eGenes detected when the effect size of lead eQTL is large (absolute log2aFC ≥ 1, left y-axis) or small (absolute log2aFC ≤ 0.25, right y-axis). We carried out down-sampling analyses (10 replications per sample size) in liver and muscle. Error bars represent standard errors across the replications. (a) The overlap of eGenes detected using imputed and observed genotypes (directly called from whole-genome sequences). (b) Comparison of significance (-log10P) between eGenes detected from observed and imputed genotypes. (c) Effect size (that is, slope) of lead variants across different categories: Both (n = 10,047): eGenes detected by both imputed and observed genotypes; Imputed genotypes only (n = 380): eGenes detected only by imputed genotypes; Observed SNPs only (n = 571): eGenes detected only by observed genotypes; Neither (n = 3,120): Non-eGenes in both approaches. Statistical significance was assessed using a two-sided Student's t-test. (d) The overlap of lead variants identified by imputed and observed genotypes. (e) Comparison of distribution of lead variants relative to gene transcription start sites (TSS) for imputed and observed genotypes. (f) Correlation of effect sizes for lead variants of eGenes identified using imputed and observed genotypes. Same lead: lead variants of shared eGenes in panel (a) are identical. P values were computed via the asymptotic t approximation. (g) Linkage disequilibrium (LD, r2) of different lead variants for the same eGenes (n = 2,307) detected by imputed and observed genotypes. The “Distance-matched random set” includes an equal number of SNP pairs with similar physical distances to lead variants. Significance was assessed using a two-sided Student's t-test. (h) Percentage of eGenes detected as a function of LD (r2) between two different lead variants detected by observed and imputed genotypes. The exact number of eGenes is depicted on each bar. (i) Functional enrichment (log2Fold change, mean ± s.d.) of eQTLs detected using observed versus imputed genotypes across chromatin states14. (j) Venn diagram depicting the overlap of eGene-independent SNP pairs identified using imputed and observed genotypes. (k) Number of eGenes stratified by different number of credible sizes. (l) Percentage of fine-mapped variants detected using observed and imputed genotypes, shown as a function of different posterior inclusion probability thresholds. Manhattan plots for imputed (bottom) and observed SNPs (top) of three eGenes: EPHB2 (m), UBXN6 (n), and ENSGALG00000006465 (o). (a) Correlations of lead variant effects (upper triangle) and cis-h2 (lower triangle) between chickens and mammals. P values of Pearson's correlations were computed using the asymptotic t approximation. (b) Internal validation of eQTLs. Bars in light blue represent the Spearman correlation coefficients of eQTL effect sizes between validation and discovery groups (left y-axis), while red dots represent the π1 statistic, estimating the replication rate of eQTL between groups (right y-axis). Each of the 15 tissues with over 100 individuals was randomly and evenly split into discovery and validation groups. (c) Correlation between eQTL effect sizes (x-axis, n = 2,396) and effect sizes from allele-specific expression (ASE) analysis (y-axis) in the liver. P values were computed via the asymptotic t approximation. (d) The proportion of regulatory variants predicted by DeepSEA (prediction score > 0.7) based on 310 functional profiles in chickens (Table S8). molQTL_set: conditionally independent molQTL across tissues; Random_set: randomly selected variants matched for minor allele frequency (MAF) with molQTL; Background: all 1.5 million tested variants. *** P < 0.001, based on a two-sided Student's t-test. (a) linear regression model (LRM)-based validation. (b) linear mixed model (LMM)-based validation. Validation was carried out in three tissues: hypothalamus (upper row), liver (middle row) and pituitary gland (bottom row). ChickenGTEx served as the discovery population, while an independent validation population consisted of commercial White Plymouth Rock chickens. P values were computed via the asymptotic t approximation. (c) Comparison of SNPs between the ChickenGTEx discovery population and validation populations. Only SNPs common (minor allele frequency, MAF > 0.05) in both discovery and validation populations were included. (d) The number of SNPs used in π1 calculation shown in panel (a). (e) Sample sizes of the ChickenGTEx discovery and validation populations. (f) The number of eGenes detected in the discovery and validation populations. (g) Effect size distribution of replicated and not-replicated eQTLs across tissues. Replicated eQTLs: SNPs significant in the discovery population that also meet the significant threshold in the validation population. Statistical significance was obtained using a two-sided Student's t-test. (h) The π1 value plotted as a function of eQTL effect size (log2 transformed allelic fold change, log2aFC) in the discovery population (liver). (i) Histogram of eQTL nominal P values in the validation population. The nearest variant in the validation population to the corresponding lead variant in the discovery population was selected for each eGene. (j) Distribution of eQTL nominal P values in the validation population. For each eGene identified in the discovery population, the top lead variant within the same LD block was selected. (a) Fold enrichment (mean ± s.d.) of molQTL in strong enhancers (E6) and super-enhancers. Error bars represent the standard errors of enrichment across 17 tissues common to this study and Pan et al.14. (b) Fraction of eGene-eVariant pairs overlapping with regulatory elements-target gene pairs retrieved from Pan et al.14. (c) Percentage of eGene-eVariant pairs located within the same topologically associating domains (TAD) predicted from Hi-C data. (d) Enrichment of ePhenotype-molQTL pairs (log2 transformed odds ratio) within the same TAD, analyzed as a function of different distances to TSS. The error bars indicate standard errors of enrichment across 28 tissues. ePhenotype: molecular phenotypes regulated by at least one genetic variant. Odds ratio was obtained by fitting the linear model: SameTAD = eQTL + |TSS distance|+ eQTL* |TSS distance|, where SameTAD represents whether the eGene-eVariant pair is within the same TAD (coded as 1) or not (coded as 0). The symbols “–“ and “+” denote upstream and downstream eVariants relative to TSS, respectively. (e) Manhattan plot displaying SNP associations with TIMM17B gene expression in the brain. The lead SNP (rs317368746) is highlighted with a diamond. The bottom panel depicts regulatory elements annotations retrieved from Pan et al.14. TssA: strongly active promoters/transcripts, TssAHet: flanking active TSS without ATAC, TxFlnk: transcribed at gene, TxFlnkWk: weak transcribed at gene, TxFlnkHet: transcribed region without ATAC, EnhA: strong active enhancer, EnhAMe: medium enhancer with ATAC, EnhAWk: weak active enhancer, EnhAHet: active enhancer no ATAC (hetero), EnhPois: poised enhancer, ATAC_Is: ATAC island, TssBiv: bivalent/poised TSS, Repr: repressed polycomb, ReprWk: weak repressed polycomb. Absolute effect size (allelic fold change, aFC) of eQTLs (a), distance of eQTL to TSS (b), and minor allele frequency (c) as a function of the number of tissues in which the eGene is expressed. P values were computed via the asymptotic t-approximation. (d) Fraction of eQTLs around transcription start site (TSS) based on the number of tissues in which they are active in. (e) Fold enrichment (log2 scaled, mean ± s.d., y-axis) of tissue-specific and -shared eQTLs across 15 chromatin states. Error bars indicates standard errors of enrichment, with E1–E15 representing chromatin states defined previously14. (f) Manhattan plot for eQTL mapping of ALG3 in the liver, muscle, brain, and blood. (g) and (h) Comparison of blood-specific eQTLs (Specific, n = 11,884) and the rest of blood eQTLs (Common, n = 9,089) in terms of Minor Allele Frequency (MAF) and effect size, respectively. Statistical significance was determined using a two-tailed Student's t-test. (i) Opposite effects of lead variants on FBXO5 and ELAC2 in different tissues. Top panel: The effect of lead variant rs315639985 on FBXO5 in blood is opposite to that of rs312482960 in spleen. Bottom panel: The effect of lead variant rs313608694 on ELAC2 in embryo is opposite to that in spleen. (k) Heatmap depicting eQTL effect sharing between breeds. (l) Expression of ENSGALG00000028174 (PRKCDBP) regulated by rs314795649 genotypes, consistent across all four breeds (n = 286 samples). Examples of sex-biased eGene TCLF5 (a), CPS1 (b) and SNAI2 (c), respectively, in the liver (n = 137). (d) Dot plots of eGene ATP6V1A expression (y-axis) against the transcription factor TCF25 expression across three genotypes of rs313600592. (e) Number of cell-type interaction QTL (ci-eQTL) detected in each tissue-cell combination (FDR < 5%). (f) Distance of eQTLs and ci-eQTLs from the transcript start site (TSS). (g) Fold enrichment of eQTLs and ci-eQTLs across 15 chromatin states (log2 scaled, mean ± s.d.). Fold enrichment mean (dot) and standard deviation (error bars) were obtained from enrichment tests in five tissues (liver, muscle, heart, bursa and spleen)14. TssA: strongly active promoters/transcripts, TssAHet: flanking active TSS without ATAC, TxFlnk: transcribed at gene, TxFlnkWk: weak transcribed at gene, TxFlnkHet: transcribed region without ATAC, EnhA: strong active enhancer, EnhAMe: medium enhancer with ATAC, EnhAWk: weak active enhancer, EnhAHet: active enhancer no ATAC (hetero), EnhPois: poised enhancer, ATAC_Is: ATAC island, TssBiv: bivalent/poised TSS, Repr: repressed polycomb, ReprWk: weak repressed polycomb. (h) Histogram of linkage disequilibrium (LD) between lead variants of eQTLs and ci-eQTLs targeting the same genes. The corresponding bulk tissue eQTL is shown on the right (tested in 517 samples). (j) Heatmap of Spearman's correlation of ci-eQTL effect sizes across tissue-cell type combinations. Tissues are clustered based on dissimilarities (that is 1-d), where d is Euclidean distance calculated from ci-eQTL effects using the complete linkage method. The color legend of tissues and cell types is shown at the bottom. (a) Associations of gene expression and chicken body weight via single-species TWAS (upper panel) in chickens and cross-species (chicken, pig, and human) meta-TWAS (bottom panel) in muscle tissue. TWAS: transcriptome-wide association study. The cross-species meta-TWAS analysis was conducted by integrating chicken body weight TWAS data with corresponding TWAS results for mammalian growth-related traits in muscle. Physiologically similar traits to chicken body weight were defined arbitrarily as growth-related traits, such as backfat thickness and average daily gain (ADG) in pigs, and body weight and height in humans. Non-physiologically similar traits included growth-unrelated phenotypes, such as the number of mummified pigs and number of weaned piglets in pigs, and type 2 diabetes and heart failure in humans. Quantile–quantile (Q-Q) plots for TWAS and meta-TWAS results are depicted on the right. (b) Nominal TWAS associations of 14 genes identified by cross-species meta-TWAS (panel a) in independent chicken populations. Phenome-wide associations (that is PheWAS) of GOLGA3 with pig traits (c) and human traits (d). y-axis shows the negative base-10 logarithm of the P-value, and x-axis indicates traits tested. Each dot represents a trait, colored by trait categories, and the red dash line indicates the FDR threshold of 0.05. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Guan, D., Bai, Z., Zhu, X. et al. Genetic regulation of gene expression across multiple tissues in chickens. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Genetics (Nat Genet) Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-57846-y'>Daytime eating during simulated night work mitigates changes in cardiovascular risk factors: secondary analyses of a randomized controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 09:38:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Effective countermeasures against the adverse cardiovascular effects of circadian misalignment, such as effects experienced due to night work or jet lag, remain to be established in humans. Here, we aim to test whether eating only during daytime can mitigate such adverse effects vs. eating during the night and day (typical for night shift workers) under simulated night work (secondary analysis of NCT02291952). This single-blind, parallel-arm trial randomized 20 healthy participants (non-shift workers) to simulated night work with meals consumed during night and day (Nighttime Meal Control Group) or only during daytime (Daytime Meal Intervention Group). The primary outcomes were pNN50 (percentage consecutive heartbeat intervals >50 ms), RMSSD (root mean square of successive heartbeat differences), and LF/HF (low/high cardiac frequency). The secondary outcome was blood concentrations of prothrombotic factor plasminogen activator inhibitor-1 (PAI-1). These measures were assessed under Constant Routine conditions, before (baseline) and after (postmisalignment) simulated night work. The meal timing intervention significantly modified the impact of simulated night work on cardiac vagal modulation and PAI-1 (pFDR = 0.001). In the Control Group, the postmisalignment Constant Routine showed a decrease in pNN50 by 25.7% (pFDR = 0.008) and RMMSD by 14.3% (pFDR = 0.02), and an increase in LF/HF by 5.5% (pFDR = 0.04) and PAI-1 by 23.9% (pFDR = 0.04), vs. the baseline Constant Routine. For exploratory outcomes, the intervention significantly modified the impact of simulated night work on blood pressure (P < 0.05), with no significant change in the Control Group, and a significant reduction by 6-8% (P < 0.01) in the Intervention Group; without significant effects for heart rate or cortisol. These findings indicate that daytime eating, despite mistimed sleep, may mitigate changes in cardiovascular risk factors and offer translational evidence for developing a behavioral strategy to help minimize the adverse changes in cardiovascular risk factors in individuals exposed to circadian misalignment, such as shift workers. Shift work is prevalent worldwide, with ~15% of the workforce performing night shift work in industrialized countries1, and it increases the risk for cardiovascular disease (CVD)2,3,4,5,6,7. For instance, longer duration of night shift work was associated with increased coronary heart disease (CHD) risk in individuals followed over 24 years8. Importantly, this increased risk cannot be fully explained by differences in lifestyle and socioeconomic status9. Such observations highlight the urgent need to understand the underlying mechanisms and develop evidence-based countermeasures to mitigate these adverse health effects. Circadian misalignment, the misalignment between the central circadian pacemaker and the behavioral sleep/wake cycle (typical in shift workers) causes negative cardiometabolic changes10,11,12,13,14. The adverse effects of circadian misalignment on cardiovascular risk factors in humans can be determined using stringently controlled laboratory protocols, including Forced Desynchrony (FD) and simulated shift work protocols15. In a FD protocol, in which participants were exposed to seven 28-h “days” under dim light (<3 lx), waketime blood pressure was higher during circadian misalignment. These misalignment-induced effects on blood pressure were not explained by changes in waketime heart rate, cortisol, nor sympathoadrenal measures (e.g., urinary epinephrine and norepinephrine), which remained unchanged11. While FD protocols provide experimental evidence for the adverse circadian misalignment effects on cardiovascular function, its translational value may be limited because shift workers do not live on 28-h “days”. Therefore, simulated shift work protocols offer a closer approximation to what shift workers typically experience while controlling for behavioral/environmental factors that directly affect cardiovascular function. For instance, a randomized crossover clinical trial showed that circadian misalignment increased 24-h systolic and diastolic blood pressure. At the same time, it decreased waketime cardiac vagal modulation (e.g., pNN50, percentage of consecutive heartbeat intervals differing by >50 ms, and RMSSD, root mean square of successive differences between normal heartbeats)16. These experimental human studies indicate that exposure to circadian misalignment may act as an underlying mechanism for the increased risk of cardiovascular disease in shift workers17. Since shift work is not foreseen to disappear, treatment interventions against the long-term adverse cardiovascular events are urgently needed. Preclinical work indicates that a well-consolidated 24-hour cycle of feeding and fasting can sustain cardiac health18. Food access limited to daytime protected against cardiac tissue aging in flies on either a normal or a fat-supplemented diet. Therefore, circadian organization is fundamental for normal health and longevity, while circadian disruption is implicated in the etiology of cardiac disease. Despite these recent breakthroughs, effective countermeasures against adverse cardiovascular effects due to circadian misalignment remain to be established in humans. Here, we aim to identify whether circadian alignment of eating (despite the mistiming of sleep and of other behaviors) prevents the adverse effects of circadian misalignment on cardiovascular risk factors. Healthy participants [7 females, 12 males; age: 26.5 ± 4.1 y; body-mass index: 22.7 ± 2.1 kg/m2] (Table 1) underwent a randomized single-blind parallel clinical trial (Fig. 1) consisting of a 14-day circadian laboratory protocol to test the effects of simulated night work with either nighttime and daytime or only daytime eating on circadian rhythms of cardiovascular function compared to simulated day work (Fig. No important harms nor unintended adverse effects were observed in the meal timing groups. The simulated night work was induced using a Forced Desynchrony protocol (FD; four 28-h “days”, see Methods). In the Nighttime Meal Control (NMC) Group, participants had a typical 28-h FD protocol with all behaviors scheduled across a 28-h cycle, including the fasting/eating cycle, which resulted in meals consumed during the daytime and nighttime (typical among shift workers). In the Daytime Meal Intervention (DMI) Group, participants underwent a modified 28-h FD protocol, with all behaviors scheduled on a 28-h cycle, except for the fasting/eating cycle, which was scheduled according to a 24-h cycle, resulting in meals consumed only during the daytime. We assessed the cardiovascular outcomes during Constant Routine protocols (CR, i.e., at least 32-hours of constant wakefulness, semi-recumbent posture, dim light conditions [<3 lx], and hourly isocaloric snacks19), before (baseline CR) and after simulated night work (post-misalignment CR). Because the effects of behaviors and environment are minimized during the CR protocol, it is ideally suited to assess the cardiovascular outcomes, which could have otherwise been affected during the FD study segment. Throughout the baseline and post-misalignment CR protocols, we repeatedly assessed cardiac vagal modulation by heart rate variability analyses (pNN50, RMSSD, LF/HF ratio, which are sensitive to circadian misalignment effects14), circulating concentrations of plasminogen activator inhibitor-1 (PAI-1, a key prothrombotic factor under circadian control20), blood pressure, heart rate, and plasma cortisol. The 14-day stringently controlled circadian laboratory study included a parallel-design, randomized controlled trial with participants randomized to the Nighttime Meal Control (NMC) group or the Daytime Meal Intervention (DMI) group (see SI appendix for details). The study design is presented in relative clock time (for a participant with a habitual wake-up time of 7 a.m.). Our results showed that the meal timing intervention significantly modified the impact of simulated night work on pNN50 (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: pFDR = 0.008). The pNN50 levels decreased in the NMC group by 25.7% after exposure to simulated night work (post-misalignment CR), compared to that before (i.e., baseline CR) (95% confidence intervals [CI], −33.9% to −17.5%; Tukey-Kramer post-hoc test adjusted for multiple comparisons, P = 0.001; Fig. Conversely, no significant change was observed in the DMI group (95% CI, −8.9% to 4.9%; P = 0.23; Fig. Similarly, the meal timing intervention significantly modified the impact of simulated night work on RMSSD (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work intervention: pFDR = 0.02). The RMSSD levels significantly decreased in the NMC group by 14.3% (95% CI, −18.4% to −10.1%; P = 0.001; Fig. 3C), after simulated night work compared to baseline. Conversely, no significant change was observed in the DMI group (95% CI, −2.1% to 2.2%; P = 0.31; Fig. Likewise, the meal timing intervention significantly modified the impact of simulated night work on LF/HF (i.e., the ratio of low frequency and high cardiac frequency heart rate variability, a measure to estimate cardiac autonomic modulation) (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: pFDR = 0.04). The LF/HF ratio levels significantly increased in the NMC group by 5.5% (95%CI, 3.5% to 6.9%; P = 0.03; Fig. 3E), after simulated night work compared to baseline. Conversely, no significant change was observed in the DMI group (95% CI, −0.3% to 1.7%; P = 0.29; Fig. Of note, the meal timing intervention did not significantly modify the impact of simulated night work on the endogenous circadian rhythms (i.e., circadian phase and amplitude) of pNN50, RMSSD, and LF/HF ratio (cosinor mixed-model analyses; interaction of meal timing group, pre/post simulated night work, and circadian effect: pFDR = 0.34). The meal timing intervention significantly modified the impact of simulated night work on pNN50 (A, B), RMSSD (C, D), LF/HF (E, F) and PAI-1 (G, H) levels. For all results, cosinor mixed-model analyses were used, and the interaction of meal timing group vs. pre/post simulated night work is reported in the figures. Cosine curves make use of precise circadian phase of each measurement for each participant. Circles indicate data grouped into 15°-circadian windows (~1-hour resolution, except for PAI-1 that is presented with 2-h resolution) with SEM error bars and the top x axes were scaled to the approximate group-averaged clock time of the circadian CBT minimum (reflected by 0o) for reference (i.e., relative clock time). Data correspond to the average (mean ± SEM) across participants per pre/post simulated night work condition and per meal timing group (n = 10 in the NMC group and n = 9 in the DMI group). The meal timing intervention significantly modified the impact of simulated night work on circulating PAI-1 concentrations (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: pFDR = 0.02). The PAI-1 concentrations significantly increased in the NMC group by 23.9% (95%CI, −2.3% to −45.6%; P = 0.001; Fig. 3G), after simulated night work compared to baseline. Conversely, no significant change was observed in the DMI group (95% CI, −2.1% to 2.2%; P = 0.24; Fig. The meal timing intervention did not significantly modify the effect of simulated night work on cortisol levels (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: P = 0.31; Fig. Neither in the NMC group (95% CI, −5.4% to 0.9%) nor in the DMI group (95% CI, −2.1% to 5.6%) was there a significant change in cortisol levels (Tukey-Kramer post-hoc test adjusted for multiple comparisons, P = 0.29 and P = 0.33, respectively). Similarly, the meal timing intervention did not significantly modify the effect of simulated night work on the heart rate (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: P = 0.36; Fig. Neither in the NMC group (95% CI, −0.4% to 2.8%) nor in the DMI group (95% CI, −1.2% to 2.1%) was there a significant change in heart rate (Tukey-Kramer post-hoc test adjusted for multiple comparisons, P = 0.39 and P = 0.32, respectively). The meal timing intervention significantly modified the impact of simulated night work on the SBP and DBP levels (cosinor mixed-model analyses; meal timing group vs. pre/post simulated night work interaction: respectively, P = 0.03 and P = 0.02, respectively; Fig. In the NMC group, SBP and DBP levels did not significantly change after exposure to the simulated night work, compared to baseline (SBP: 95% CI, −1.3% to 2.8%; DBP: 95% CI, −1.1% to 0.7%; Tukey-Kramer post-hoc test adjusted for multiple comparisons, P = 0.22). Conversely, in the DMI group, blood pressure levels significantly decreased by 6-8% after exposure to the simulated night work, compared to baseline (SBP: −6.1%, 95% CI, −8.9% to −4.5%; DBP: −8.0%, 95% CI, −9.1% to −6.9%; Tukey-Kramer post-hoc test adjusted for multiple comparisons, P = 0.005 and P = 0.001). Of note, the meal timing intervention did not significantly modify the impact of simulated night work on the endogenous circadian rhythms (i.e., circadian phase and amplitude) of cortisol, heart rate, SBP, and DBP (cosinor mixed-model analyses; interaction of meal timing group, pre/post simulated night work, and circadian effect: P = 0.45, P = 0.51, P = 0.31 and P = 0.34, respectively). We found evidence for an increase in cardiovascular risk factors after simulated night work with nighttime and daytime eating, whereas a Daytime Meal Intervention minimized the adverse effects of simulated night shift work on cardiovascular risk factors. Exposure to simulated night work with nightime and daytime eating decreased cardiac vagal modulation, with vagal (parasympathetic) activity typically considered cardioprotective16, and these findings are consistent with a previous clinical trial that induced circadian misalignment, i.e., 12-h inverted behavioral and environmental cycles for 3 days16. Circadian clocks within the suprachiasmatic nucleus in the hypothalamus (SCN) and the heart set daily rhythms in sinoatrial and atrioventricular node activity, with a consequential time-of-day dependent vulnerability to ventricular arrhythmia21. Exposure to circadian misalignment might alter the electrophysiological cardiac milieu, potentially increasing the susceptibility to adverse cardiac events. This may provide a mechanism underlying the elevated risk of cardiovascular disease and cardiac events in shift workers22,23 Additionally, circadian misalignment decreased PAI-1 concentration by 11% during misalignment in a previous simulated shift work study16, while we show an increase by 24% after misalignment24. Mice exposed to chronic circadian phase shifts had an increase in PAI-1 mRNA expression and PAI-1 levels and decreased tissue-type plasminogen activator mRNA expression in the liver25. Such findings and ours suggest that circadian disruption might induce hypofibrinolysis and increase cardiovascular risk by inducing PAI-1 gene expression following circadian misalignment. Our results indicate no significant circadian misalignment nor meal timing effects on cortisol levels (Fig. S1), in line with previous results where acute11,12, and chronic26 circadian misalignment had a limited impact on cortisol levels. Similarly, we observed no significant circadian misalignment nor meal timing effects on heart rate (Fig. While not directly comparable to our current study, time-restricted eating (i.e., 8-hour eating window from 7 a.m. to 3 p.m.) did not significantly affect heart rate, compared with a control eating pattern (i.e., a self-selected ≥12-hour window)27. Of note, the daytime meal intervention significantly reduced blood pressure levels by 6-8% after exposure to the simulated night work, compared to baseline, an effect that was not observed in the nighttime meal control (Fig. The beneficial effect of appropriately aligned meal timing has been shown in a randomized control trial with shift workers who underwent 12 weeks of 10-h time-restricted eating compared to a standard of care (i.e., nutritional counseling and Mediterranean diet at usual eating times)28. Accordingly, participants consistently showed decreased diastolic blood pressure, compared to those who received standard of care. Similarly, a 14-week parallel-arm, randomized clinical trial weight-loss program with either early TRE (8-hour eating window from 7a.m.–3p.m.) or control eating (i.e., a self-selected ≥12-hour window) showed decreased diastolic blood pressure27. While our DMI findings bear resemblance to these TRE clinical trials, direct comparisons are challenging, as those studies, e.g., modified the fasting durations as part of the intervention, recruited participants who were overweight, included different sample sizes, and were long-term field studies. Of note, we did not observe endogenous circadian rhythms in blood pressure during baseline CR in either group. Circadian blood pressure rhythms have been described in previous laboratory studies29,30,31,32,33. While those previous studies used a Dynamap blood pressure monitor typically used for in-hospital spot-checks, our study used a Spacelabs blood pressure monitor typically used for 24-h automated blood pressure monitoring at home, which limits a direct comparison. Preclinical work suggests dramatic effects of circadian rhythm disorganization on cardiovascular integrity34. A point mutation in the circadian regulatory gene, casein kinase-1, results in an intrinsic circadian period of 22 h (instead of approximately 24 h in wildtypes), early-onset cardiomyopathy, extensive fibrosis, and severely impaired contractility34. Intriguingly, under light cycles of 22 h, which match their intrinsic period (i.e., in resonance), their cyclic behavioral patterns and heart structure and function normalized, indicating the importance of synchrony between the internal circadian clock with environmental and behavioral cycles34. Moreover, preclinical work suggests a robust 24-h feeding/fasting cycle can maintain cardiac health in a high-fat fed aging Drosophila model, as compared to ad libitum fed flies that showed drastic deterioration of cardiac function, including increased arrhythmia index and reduced heart contractility18. In mice models, time-restricted feeding with food available only during the active nighttime reduced sympathetic activity during the resting daytime35. Here, we show that daytime meal timing avoids the adverse effects of simulated night work on cardiovascular function in humans. Critically, in our laboratory protocol, the design of the meal timing groups was identical (i.e., caloric and macronutrient intake, physical activity, posture, scheduled sleep schedule, and lighting conditions) except for the timing of meals. Moreover, our findings were unlikely due to group differences in participant sociodemographics and study-related characteristics (Table 1). Short sleep duration is associated with increased morbidity and mortality, particularly from cardiovascular disorders, such as CHD, arrhythmias, and hypertension36. As sleep structure before the baseline and simulated night work conditions did not differ between the meal timing groups24, differences in prior sleep are unlikely to mediate our reported effects. Typical breakfast, lunch, and dinner meals during the daytime are no longer the social norm because skipping meals and snacking are becoming increasingly prevalent, as well as eating during the night37. Nighttime eating was associated with increased CHD risk compared with those who did not eat during the night after adjustment for sociodemographic data, diet, lifestyle, and CHD risk factors38. Data from 103,389 adults in the NutriNet-Santé study indicated that a later first meal (after 9 a.m. relative to before 8 a.m.) and a later last meal (after 9 p.m. relative to before 8 p.m.) was associated with elevated CVD risk39. Night shift workers show changes in meal patterns, skipping meals and consuming them during the night40. Therefore, meal timing is a potential modifiable lifestyle factor involved in the higher CVD risk in night workers. In this context, our study demonstrates that a meal-timing-based behavioral intervention can help mitigate the adverse cardiovascular effects induced by a mistimed sleep/wake schedule, as typically experienced by night workers. Limitations in our study include the restricted age range of participants (18–30 years), which limits its relevance to middle-aged and older populations. Moreover, caution is warranted as our study has a limited sample size (due to the stringent participant selection criteria and the intensive and controlled study design) and includes individuals who do not engage in actual shift work. Additionally, we performed an FD protocol to induce circadian misalignment, and while laboratory approaches offer the greatest experimental control, its findings have limited direct translation to night shift workers, as they do not live under such carefully controlled behavioral and environmental conditions. Future clinical trials with larger samples, including individuals undergoing real-life shift work schedules (e.g., permanent, rotating or irregular night shifts, morning shifts, and evening shifts), are needed to establish whether our reported beneficial effects on cardiovascular risk factors translate to the shift work population. The protocol was approved by the Partners HealthCare's institutional review board (IRB) and performed in accordance with the principles of the Declaration of Helsinki, and participants provided written informed consent. Laboratory protocols were conducted at the Center for Clinical Investigation at Brigham and Women's Hospital, Boston, United States, between 19 March 2015 and 29 August 2018. Due to the complex study design, this is a single study performed in a human clinical trial. This randomized clinical trial provides proof-of-evidence for the beneficial effect of meal timing as a countermeasure against the adverse cardiovascular effects of circadian misalignment in humans, and future studies are needed to replicate our findings. Participants admitted to the laboratory protocols were free from medical conditions, including current and previous history of cardiometabolic disorders. Participants were not engaged in shift work, had not crossed more than one time zone in the previous 3 months, and did not have sleep disorders, including insomnia. Biochemical blood panels at screening included a comprehensive metabolic panel, TSH, complete blood count, and HbA1c, all of which had to be within typical ranges for study inclusion. Participants were not taking medications (excepting oral contraceptives), caffeine, smoking, or using recreational drugs (verified with urine toxicological panel). Participants underwent a randomized, parallel, controlled, single-blinded trial24, in which they were randomly assigned to one of two meal timing groups. The Nighttime Meal Control Group (NMC) completed a protocol that included simulated day work with day eating [baseline] followed by simulated night work with daytime and nighttime eating, typical for shift workers. The Daytime Meal Intervention Group (DMI) completed a protocol that included simulated day work with day eating (baseline) followed by simulated night work with daytime eating. Participants were randomized using minimization (Minim.exe, MS-DOS free access program for minimizing participants into the arms of a clinical trial) to minimize imbalance between meal timing groups. Minimization was performed—in decreasing sequence of importance—by sex, body mass index (BMI), and age (these factors were dichotomized, i.e., female or male, 18.5–24.9 kg/m2 or 25–29.9 kg/m2, and 18-26 years or 27-35 years, respectively). Twenty healthy normotensive participants (mean age, 26.6 y [SD, 4.2 y, range: 18–35 y], eight females, BMI range: 18.5–29.9 kg/m2, hemoglobin A1C range: 4.9–5.4%) were randomized to the laboratory protocol: ten were allocated to the NMC Group and ten to the DMI Group (Fig. Four females commenced the laboratory protocol on menstrual cycle days 1-5 (two per meal timing group) and four during days 14–19 (two per meal timing group). We excluded data from one participant in the DMI Group due to their inability to consume all food during the simulated night work. The final study sample included ten participants for the NMC Group (mean age, 27.0 years [SD, 4.4 y], 4 females, BMI: 22.5 kg/m2 [SD, 3.5]), and nine for the DMI Group (mean age, 26.2 years [SD, 4.1 y], 3 females, BMI: 23.1 kg/m2 [SD, 3.1]). Additional study-related characteristics between groups (e.g., diet, chronotype, among others) are shown in Table 1. No statistical differences in participant sociodemographics and study-related characteristics occurred between groups. Other aspects of this study, that was designed to test separate, independent hypotheses, have previously been published24,41,42,43. The personalized laboratory sleep/wake schedule was determined as the participants' habitual bedtime with 8-h time in bed. To ensure compliance with this schedule, each participant was monitored with at-home wristworn wearables (Actiwatch, Respironics). Participants also called into a time-stamped voice-mailbox when they were going to bed and getting out of bed. If more than one deviation (>1 h) per week from the target sleep/wake time was detected, participants were excluded from the study. In addition, during the three days before the laboratory study, participants received all meals (three meals and one snack) from the Metabolic Kitchen to meet dietary requirements (Harris-Benedict formula with activity factor 1.4) and controlled macronutrient distribution (45–50% carbohydrate, 15–20% protein, 30–35% fat, with 150 mEq Na+ (±20%), 100 mEq K+ (±20%)), to match the subsequent laboratory diet. Participants had to consume the provided meals at their habitual eating times to standardize the amount, type, and timing of food intake before the laboratory protocol. The latter was ensured by participants calling into a time-stamped voice-mailbox when they began each meal and by a daily food log that included food content and times. Participants were requested to consume their meals at the same time as in the laboratory relative to waketime (+/−1h). Participants who did not comply with the dietary requirements were excluded from the laboratory study. Moreover, participants were required to refrain from exercise (running/jogging, swimming, cycling, weight lifting, circuit training, and yoga) for 3 days prior to the laboratory protocol. Participants remained in individual laboratory suites in an environment free of time cues. Throughout the study, when participants were not involved in a study task, they could undertake leisure activities, such as reading, writing, watching movies, crafts, etc. We monitored each participant's activity for compliance by means of closed-circuit TV and wrist-worn actigraphy. Days 3-4 included a baseline Constant Routine (CR) protocol (Fig. 2), during which participants spent 32 h continuously awake in a constant semi-recumbent body posture, without physical exertion, in dim light (~3 lx in the horizontal angle of gaze) and eating hourly isocaloric snacks. This allowed a baseline assessment of endogenous circadian rhythms of physiologic markers by eliminating or minimizing the influences of behavioral and environmental factors on a given rhythm19. Following the baseline CR, participants had a 12-h sleep opportunity to recover. On Days 5 and 6, participants had further recovery from the baseline CR. On Day 7, participants underwent a 28-h Forced Desynchrony protocol (FD) to induce circadian misalignment, with 28-h sleep/wake cycles under dim light (~3 lx), to which the central circadian pacemaker in humans cannot entrain44. We used a 28-h FD protocol to assess the impact of circadian misalignment on cardiometabolic function11. During each 28-h cycle, the ratio of scheduled wakefulness (18 h:40 min) and sleep (9 h:20 min) was maintained at 2:1, to match the self-selected 8-h habitual time in bed per 24-h. The participants' sleep episodes (0 lx) were split into three identical blocks, each separated by 1 h of wakefulness in dim light (~3 lx) while remaining at rest in a semi-recumbent posture in bed. This allowed the participants to consume food during the circadian day when otherwise they would be sleeping. Importantly, participants woke during each sleep episode irrespective of meal consumption to ensure that both study groups had three equal sleep blocks during each of the FD'days'. On the first 28-h sleep/wake cycle, participants had normal circadian alignment (waking up at their habitual wake-time, e.g., 7a.m. In contrast, on the fourth sleep/wake cycle, participants were 12-h misaligned (wake up at, e.g., 7p.m. ; simulated night work) as compared to the first cycle in both meal timing groups. The NMC Group had a typical 28-h FD protocol with all behaviors, including the fasting-eating cycle, maintained on a 28-h cycle. Because of that, three meals and a snack were scheduled at fixed times relative to scheduled wake-time (at 0 h:10 min, 4 h:10 min, 8 h:10 min, and 12 h:10 min since scheduled awakening, during baseline and simulated night work). Thus, each meal was shifted 4 h later each cycle, in alignment with the sleep/wake cycle. Participants in the NMC Group thus consumed food during both the daytime and nighttime, which is typical behavior of shift workers. In contrast, the DMI Group had a modified 28-h FD protocol, with all behaviors identically scheduled on a 28-h cycle, except for the fasting/eating cycle that was maintained on a 24-h cycle. This meal timing approach in the DMI Group allowed alignment of the fasting/eating cycle to the ~24-h central circadian cycle and ensured meal consumption only during the daytime and at the same clock time during each FD cycle. After the four “days” of a 28-h FD protocol, participants underwent a post-misalignment CR (Days 11/12/13) that allowed assessing the impact of prior circadian misalignment on endogenous circadian rhythms. Thereafter, participants were scheduled to a 12-h sleep opportunity to allow them to recover partially from the post-misalignment CR protocol, and then were discharged from the study. Participants received an isocaloric diet (i.e., CR snacks) calculated according to the Harris-Benedict equation with an activity factor of 1.2 (as participants had decreased activity). The diet consisted of 45–50% carbohydrate, 15–20% protein, 30–35% fat, with 150 mEq Na+ (±20%), 100 mEq K+ (±20%), and at least 2.5 L of water per 24 h. CR snacks comprised two alternating CR options (e.g., CR snack A, then CR snack B, then CR snack A, and so forth). This was based on a food preference form for each participantʼs two CR preselected CR snack choices (2 of 6 snack choices with different ingredients but the same macronutrient composition). CR snacks were calculated with the same two snack options and the same caloric level throughout both CR protocols per participant. Participants had 10–15 minutes to consume the CR snacks and were instructed to consume all food provided (verified by checking their food trays). During the CR protocol, actual energy consumption in the NMC Group was 99.9% (SEM, 0.01) and 99.9% (SEM, 0.04%) during baseline CR and post-misalignment CR, respectively. In the DMI Group, it was 98.6% (SEM, 1.9%) and 99.9% (SEM, 0.01%) during baseline CR and post-misalignment CR, respectively. During the FD protocol, participants received meals (breakfast, lunch, snack and dinner) standardized across days based on a food preference form for each participant. Meals were calculated according to a 28-h day for the NMC Group and to a 24-h day for the DMI Group, during the four “days” in the FD protocol. Diet was calculated according to the Harris-Benedict equation with an activity factor of 1.4, and consisted of 45–50% carbohydrate, 15–20% protein, 30–35% fat, with 150 mEq Na+ (±20%), 100 mEq K+ (±20%), and at least 2.5 L of water per 24 h. The energy content of the meals/snack (% of total “day's” calorie intake) was as follows: Breakfast: 33.3% (±35 kcals); Lunch: 23.4% (±20 kcals); Snack: 10% ( ± 10 kcals); Dinner: 33.3% (±35 kcals). Breakfast and Dinner Test meals (used for glucose tolerance assessment published separately24) were preselected (one of two test meal choices, based on a food preference form for each participant). During the other segments of the laboratory protocol, participants received an isocaloric diet, calculated according to the Harris–Benedict equation with an activity factor of 1.4. The diet had the same macronutrient composition as for the CR and FD segments, which was 45–50% carbohydrate, 15–20% protein, 30–35% fat, with 150 mEq Na+ (±20%), 100 mEq K+ (±20%), and at least 2.5 L of water per 24 h. Participants were instructed to consume all food provided (verified by checking their food trays). The primary and secondary outcomes of this study are reported in clinicaltrials.gov (clinical trial registration: NCT02291952). Accordingly, the primary outcomes of this study as reported in ref. These primary outcomes were published in24, except for the leptin results, due to funding constraints and issues regarding preprocessing of the blood assays. The secondary outcomes of this study were: changes in circadian phase markers, such as from core body temperature and cortisol, in the circadian rhythm in resting energy expenditure, in hunger and appetite, mood, and cognitive performance, in microbiota, gene expression, epigenetic or proteomic markers, and in sleep. Most of these secondary outcomes have been published16,41,42,43. The study is a secondary analysis of our randomized clinical trial, and here we assessed the following cardiovascular measurements: Heart rate variability (pNN50, RMSSD, and LF/HF ratio). Electrocardiographic (ECG) measurements were performed continuously during the CR protocols, and data were binned into every 10-min. Assays were obtained using blood samples every 2 hours during the baseline and post-misalignment CR protocols. Measurements were obtained with a Spacelabs 90217 ambulatory blood pressure monitor (Spacelabs Medical, Inc.) placed on the non-dominate arm. Assays were obtained using blood samples every hour during the baseline and post-misalignment CR protocols. Electrocardiographic (ECG) measurements were performed continuously during the CR protocols, and data were binned into 10-min bins. R-wave peak detection was done using ARISTOTLE45 following the same procedures as outlined in previously published work46,47. We established an analysis pipeline using MATLAB (Ver. R2022a; The MathWorks Inc., Natick, MA, USA) for HRV analyses. Specifically, consecutive R peaks were used to construct the RR interval time series. As the R peak detection was based on an automated approach, anomalous intervals due to false detections could not be avoided. They are usually manifested as spikes in the time series. To diminish the influence of these spikes on subsequent analyses, an impulse rejection filter was first applied, and if spikes were ever identified, they were replaced by the median value of the surrounding five samples48. Time- and frequency-domain analyses of RR interval time series were done per 10-min fixed time window. Time-domain measures included the root mean square of the successive differences in RR intervals (RMSSD) and the proportion of the number of times that successive normal sinus intervals differed greater than 50 ms (pNN50). Greater values of RMSSD or pNN50 mean higher vagal modulation. No results were rendered if the 10-min time series contained less than 20 RR intervals. For frequency-domain analysis, each 10-min time series was first evenly resampled. The power spectral density (psd) was computed using the Welch's method with an added Hanning window. The power of low-frequency (LF; 0.04–0.15 Hz) and high-frequency (HF; 0.15–0.4 Hz) bands were then calculated. Note that for a reliable estimation of the psd, at least two consecutive segments, each with 150-s data, were required. The PSD was performed separately to each 150-s data, and the final psd for the 10-min time series was estimated by averaging all available psd's. If less than two consecutive 150-s data were available, no frequency-domain results were rendered. Blood pressure and heart rate measurements started shortly after constant routine protocol started until scheduled recovery sleep time. Measurements were obtained with a Spacelabs 90217 ambulatory blood pressure monitor (Spacelabs Medical, Inc.) placed on the non-dominate arm. Such method has been validated according to the Association for the Advancement of Medical Instrumentation's standards49. Due to data loss, we had full time-series data for seven participants in the NMC group and six participants in the DMI group for the blood pressure and heart rate measurements. On admission to the laboratory, an 18-gauge intravenous catheter was inserted into the participant's forearm. The catheter was connected to a triple-stopcock manifold (Cobe Laboratories Inc., Lakewood, CO) via an intravenous loop with a 12-foot small-lumen extension cable (Sorex Pharmaceuticals, Salt Lake City, UT) through which blood sampling could continue in the next room without disturbing sleep. Between samples, infusion of a solution of 0.45% saline with 5000 IU/liter heparin at one drop every 5 to 10 sec maintained patency. Blood was transferred to 5-cm3 vacutainer tubes and centrifuged at 4 °C, pipetted into polystyrene tubes, and frozen at −80 °C until analysis. Participants' hematocrit and hemoglobin were measured on each CR day and on FD day 1 and day 4 (when blood measurements took place) to assess whether levels remained within normal range. The sample size derived from the difference in the effect of misalignment on glucose tolerance (primary endpoint: 3 h postprandial plasma glucose profiles; for details, see24) between the meal timing groups. To determine a large effect size (d = 1.5) with −80% power, eight participants per group were required (total sample = 16). We compared participants' characteristics with Yates's chi-squared tests or two-tailed t-tests for independent groups, and their demographics and study-related characteristics did not significantly differ between the meal timing groups (Table 1). Females and males were included in the study with a 1:1 ratio. This study was not designed to test for sex-differences and the sample size calculation did not include sex as a potential factor of interest, as there are no sex differences reported for meal timing studies in humans. Thus, we did not test for interaction effects of sex or menstrual phase because of the limited sample size. Regarding race, we included 1 Black, 1 Asian American, and 8 White participants in the control group, and 1 Black and 8 White participants in the intervention group. Regarding ethnicity, we included 10 non-Hispanics in the control group, and 7 non-Hispanics and 2 Hispanics in the intervention group. The race and ethnicity ratio are in accordance with the population demographics census of 2014 for the state of Massachusetts, United States. This study was not designed to test for race and ethnicity differences and the sample size calculation did not factor these as a potential covariate of interest, as there are no race and ethnicity differences reported for meal timing studies in humans. Thus, we did not test for such differences in the effects of the intervention because of the limited sample size. We assessed cardiac vagal modulation by heart rate variability analyses, blood pressure, and blood assays of cortisol and PAI-1 before and after the simulated night work (i.e., pre/post night work, i.e., baseline CR and post-misalignment CR, respectively). This CR approach allowed disentangling the contribution of endogenous circadian control from the acute effects of fasting/eating, sleep/wake, rest/activity, and dark/light cycles19. As the outcomes had non-normal data distribution, we normalized the cardiovascular data using an average of each participant's levels measured throughout the Baseline CR. This approach also allowed minimizing interindividual differences in physiologic measures. We applied a mean ± 3 SD (standard deviation) filter to remove outliers on the raw data. Data from the first 5 h after starting the CRs were excluded from analysis, as is standard, to allow for stabilization of circadian rhythms. The effects of the circadian cycle and circadian alignment condition were assessed by cosinor analyses using mixed model analyses of variance (PROC MIXED, SAS), which were applied to all cardiovascular outcomes. These cosinor mixed-models included “Circadian effect” (a fundamental circadian component of ∼24-h), “Time since scheduled waketime” (a linear effect of hours into the CR protocol), “pre/post simulated night work” (baseline CR vs. post-misalignment CR), “meal timing group” (NMC and DMI), and the interaction of meal timing group and pre/post night work. Additionally, we tested whether the meal timing intervention modified the impact of simulated night work on the endogenous circadian cardiovascular rhythms (i.e., circadian phase and amplitude) using cosinor mixed-model analyses that included the interaction of meal timing group, pre/post night work, and circadian effect. In the results section, we also report the percentage of change from baseline to post-simulated night work, per group. This corresponds to the post-hoc results from the interaction of meal timing group and pre/post night work, and are presented for each group separately, as we used a parallel design (i.e., while each participant serves as their baseline per meal timing condition, different participants were enrolled for the two meal timing groups). Missing data were not included in the cosinor mixed-model analyses (pNN50: 0.81%; RMSSD: 0.81%; LF/HF: 0.82%, and PAI-1: 1.12%). Participant was included as a random factor. Post-hoc comparisons used the Tukey-Kramer test to adjust for multiple testing. To control overall type I error in null hypothesis testing when conducting multiple comparisons, P-values from the mixed-model analysis were adjusted using False Discovery Rates (pFDR) (PROC MULTTEST, SAS) for the primary (i.e., pNN50, RMSSD, LF/HF ratio) and secondary (PAI-1) cardiovascular outcomes. The significance for all statistical tests was set as P < 0.05. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. As per the NIH Policy on Data Sharing, we will make the datasets that underlie the results reported in this article available to other investigators following publication of the final study results. All proposals for data will be reviewed and will ensure that the proposals are complete and valid and that the data are available, consistent with participant privacy and informed consent by the corresponding author. Responses will be provided within three months. Such datasets will not contain identifying information per the regulations outlined in HIPAA. Per standard Partners HealthCare System policies, we will require from any investigator or entity requesting the data a data-sharing agreement that provides for: (1) a commitment to using the data only for research purposes and not to identify any individual participant; (2) a commitment to securing the data using appropriate computer technology; and (3) a commitment to destroying or returning the data after analyses are completed. Ha, M. & Park, J. Shiftwork and metabolic risk factors of cardiovascular disease. & Gillander Gadin, K. Shift work, parental cardiovascular disease and myocardial infarction in males. McNamee, R. et al. Shiftwork and mortality from ischaemic heart disease. Oishi, M. et al. A longitudinal study on the relationship between shift work and the progression of hypertension in male Japanese workers. & Kolbe-Alexander, T. Shift work and the risk of cardiovascular disease. A systematic review and meta-analysis including dose-response relationship. Vetter, C. et al. Association Between Rotating Night Shift Work And Risk Of Coronary Heart Disease Among Women. Is there an association between shift work and having a metabolic syndrome? Results from a population based study of 27,485 people. Morris, C. J. et al. Endogenous circadian system and circadian misalignment impact glucose tolerance via separate mechanisms in humans. Adverse metabolic and cardiovascular consequences of circadian misalignment. Effects of the internal Circadian system and circadian misalignment on glucose tolerance in chronic shift workers. Grimaldi, D., Carter, J. R., Van Cauter, E. & Leproult, R. Adverse impact of sleep restriction and circadian misalignment on autonomic function in healthy young adults. Chellappa, S. L., Vujovic, N., Williams, J. S. & Scheer, F. Impact of Circadian disruption on cardiovascular function and disease. Morris, C. J., Purvis, T. E., Hu, K. & Scheer, F. A. Circadian misalignment increases cardiovascular disease risk factors in humans. Morris, C. J., Purvis, T. E., Mistretta, J., Hu, K. & Scheer, F. Circadian misalignment increases C-reactive protein and blood pressure in chronic shift workers. Gill, S., Le, H. D., Melkani, G. C. & Panda, S. Time-restricted feeding attenuates age-related cardiac decline in Drosophila. Getting through to circadian oscillators: why use constant routines? Human circadian system causes a morning peak in prothrombotic plasminogen activator inhibitor-1 (PAI-1) independent of the sleep/wake cycle. Distinct circadian mechanisms govern cardiac rhythms and susceptibility to arrhythmia. Meloni, M., Setzu, D., Del Rio, A., Campagna, M. & Cocco, P. QTc interval and electrocardiographic changes by type of shift work. Chellappa, S. L. et al. Daytime eating prevents internal circadian misalignment and glucose intolerance in night work. Oishi, K. & Ohkura, N. Chronic circadian clock disruption induces expression of the cardiovascular risk factor plasminogen activator inhibitor-1 in mice. Influence of sleep deprivation and circadian misalignment on cortisol, inflammatory markers, and cytokine balance. Effectiveness of early time-restricted eating for weight loss, fat loss, and cardiometabolic health in adults with obesity: a randomized clinical trial. Manoogian, E. N. C. et al. Feasibility of time-restricted eating and impacts on cardiometabolic health in 24-h shift workers: The Healthy Heroes randomized control trial. Scheer, F. A. et al. Impact of the human circadian system, exercise, and their interaction on cardiovascular function. Hu, K., Scheer, F. A., Laker, M., Smales, C. & Shea, S. A. Endogenous circadian rhythm in vasovagal response to head-up tilt. Existence of an endogenous circadian blood pressure rhythm in humans that peaks in the evening. Impact of mental stress, the circadian system and their interaction on human cardiovascular function. The circadian system modulates the rate of recovery of systolic blood pressure after exercise in humans. Martino, T. A. et al. Circadian rhythm disorganization produces profound cardiovascular and renal disease in hamsters. T. et al. Time-restricted feeding protects the blood pressure circadian rhythm in diabetic mice. Tobaldini, E. et al. Short sleep duration and cardiometabolic risk: from pathophysiology to clinical evidence. Meal timing and frequency: implications for cardiovascular disease prevention: a scientific statement From the American Heart Association. Prospective study of breakfast eating and incident coronary heart disease in a cohort of male US health professionals. Palomar-Cros, A. et al. Dietary circadian rhythms and cardiovascular disease risk in the prospective NutriNet-Sante cohort. Effects of shift work on the eating behavior of police officers on patrol. Chellappa, S. L. et al. Proof-of-principle demonstration of endogenous circadian system and circadian misalignment effects on human oral microbiota. Qian, J. et al. Daytime eating prevents mood vulnerability in night work. Scheer, F. & Chellappa, S. L. Endogenous circadian rhythms in mood and well-being. Stability, precision, and near-24-hour period of the human circadian pacemaker. Moody, G. & Mark, R. G. Development and evaluation of a 2-lead ECG analysis program. Resting Heartbeat Complexity Predicts All-Cause and Cardiorespiratory Mortality in Middle- to Older-Aged Adults From the UK Biobank. Gao, L. et al. Nocturnal heart rate variability moderates the association between sleep-wake regularity and mood in young adults. Shi, B., Motin, M. A., Wang, X., Karmakar, C. & Li, P. Bivariate entropy analysis of electrocardiographic RR-QT Time Series. Accuracy of the SpaceLabs Medical 90217 ambulatory blood pressure monitor. was supported by R01HL118601 and the Alexander Von Humboldt Foundation. Medical Chronobiology Program, Division of Sleep and Circadian Disorders, Departments of Medicine and Neurology, Brigham and Women's Hospital, Boston, MA, USA Sarah L. Chellappa, Jingyi Qian, Nina Vujovic & Frank A.J.L. Division of Sleep Medicine, Harvard Medical School, Boston, MA, USA Sarah L. Chellappa, Lei Gao, Jingyi Qian, Nina Vujovic, Peng Li, Kun Hu & Frank A.J.L. Department of Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Lei Gao, Peng Li & Kun Hu Medical Biodynamics Program, Division of Sleep and Circadian Disorders, Departments of Medicine and Neurology, Brigham and Women's Hospital, Boston, MA, USA Lei Gao, Peng Li & Kun Hu Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Sarah L. Chellappa or Frank A.J.L. served on the Board of Directors for the Sleep Research Society and has received consulting fees from the University of Alabama at Birmingham and Morehouse University. consultancies are not related to the current work. The other authors declare that they have no competing interests. Nature Communications thanks Tami Martino and the other anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher'snote Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Chellappa, S.L., Gao, L., Qian, J. et al. Daytime eating during simulated night work mitigates changes in cardiovascular risk factors: secondary analyses of a randomized controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01093-0'>Daily briefing: The physicist behind baseball's new ‘torpedo' bat</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-08 09:34:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Hello Nature readers, would you like to get this Briefing in your inbox free every day? One of the fundamental-physics prizes was awarded to 13,508 physicists for work done at CERN, Europe's particle physics laboratory. But it disintegrates in a few days when exposed to sea star wasting disease, whose cause is unknown. Reference: Proceedings of the Royal Society B: Biological Sciences paper Multiple organizations representing university faculty members filed a lawsuit on 25 March challenging the actions. The team led by Robert F. Kennedy Jr pressured the country's top vaccine regulator to deliver non-existent anti-vaccine data and weaken regulation of unproved stem-cell treatments, and forced him to resign when he refused, reports The Wall Street Journal. In his resignation letter, hematologist-oncologist Peter Marks said that “it has become clear that truth and transparency are not desired by the secretary, but rather he wishes subservient confirmation of his misinformation and lies”. He downplayed his past statements, but has since made moves that worry public-health specialists, such as reportedly tipping an anti-vaccine activist to pursue evidence in support of a discredited claim that vaccines are linked to autism. Andrew Robinson's pick of the top five science books to read this week includes a look at an outbreak of overdiagnosis, a natural history of the neck and a thought-provoking right-wing perspective on the problems with academia. The growing role of artificial intelligence (AI) in science could do more harm than good if modelling-based approaches are wielded without nuance or understanding, argue computer scientists Arvind Narayanan and Sayash Kapoor. And, even if it's error-free, “it is easy for researchers to overestimate the predictive capabilities of an AI model, thereby creating the illusion of progress while stalling real advancements”. On Friday, our penguin-seeking puzzle brought us to the stunning moss-covered banks of Coal Creek in the Coeur d' Alene National Forest in Idaho. • Nature Briefing: Careers — insights, advice and award-winning journalism to help you optimize your working life • Nature Briefing: Translational Research — covers biotechnology, drug discovery and pharma Daily briefing: This brain structure filters which thoughts we become aware of Daily briefing: Women's work is missing from the scientific literature Zhejiang University (ZJU) International Campus in Haining, China, about 120km southwest of Shanghai Wuxi Medical College of Jiangnan University (Affiliated Hospital) invites you to join us! You have full access to this article via your institution. Daily briefing: This brain structure filters which thoughts we become aware of Daily briefing: Women's work is missing from the scientific literature An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            