
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-29</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69700710/etruscan-ancient-face/'>Archaeologists Unearthed an Ancient Face Where It Wasn't Supposed to Be</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At an archaeological site in what used to be the ancient Etruscan city of Vulci, the disembodied head of a young woman began to emerge from the ground, her empty eyes staring through millennia of history. Her marble face had not seen daylight in 2,500 years. It was a rare glimpse into a larger sculpture that shows the extent of Greek influence on Etruscan art and culture. The face of the nameless woman most likely gazed out from a monumental temple that was discovered in 2020 near Tempio Grande. With an eerily lifelike smile, she wears a diadem over a crown of hair that has been meticulously carved down to every strand, and while the paint that once brought her to life has long since eroded, she was preserved so well that traces of original paint can still be seen on the marble. While it used to be thought that most of the cultural exchange between Etruria and the Attica region of Greece (also famous for its honey) was evidenced in the form of painted pottery, the head of this kore shows that Hellenic influence reached beyond clay vessels. Whoever commissioned this statue must have been one of the Etruscan elite. Most previous Etruscan excavations have focused on tombs and necropolis complexes without much investigation of everyday urban life. Unlike the broken columns of ancient Greek and Roman cities that are still standing, no buildings survive in that area. This kore is now being restored at Rome's Instituto Centrale per il Restauro. It was also built the same time as another temple that was excavated during the 1950s and dated based on artifacts that were unearthed around its foundations. Its images of forest nymphs and satyrs were finally determined to be revelers of Dionysus, the Greek deity of wine, fertility, ecstasy, mysticism and theatre, who appears as a horned god in ornamental figures that surround them. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Gravity Proves We Live in a Simulation, Scientist Says This “Battery In a Rock” Changes the Energy Game Scientists Made Glue So Strong, It Can Tow A Car Black Market Coins Led to a Bronze Age Fortress Could We Have Evidence That Cells Are Conscious?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69688804/ancient-city-first-wall/'>Archaeologists Uncovered an Unparalleled Segment of an Ancient City's ‘First Wall'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“The newly uncovered section, known in ancient historical sources as the ‘First Wall,' is particularly impressive in both its size and its degree of preservation,” Amit Re'im and Marion Zindel, excavation directors for the Israel Antiquities Authority, wrote in a statement. Located nearby the Tower of David Jerusalem Museum, which dates to King Herod's time, this wall would have also surrounded Mount Zion during the second temple period. Details of the wall are preserved through historical accounts—Josephus, a first century C.E. historian who chronicled its route and gates, called it “impregnable.” That said, just because it was called the First Wall, doesn't make it so: Josephus also noted an old wall that may have been built by David and Solomon around the 10th century B.C.E. This First Wall's impressively robust 16 feet of thickness was found intact, but its current height indicates that the wall was systematically and deliberately dismantled. “There is much more to this wall than meets the eye,” Re'im and Zindel wrote. This raised questions about who was responsible for the wall's destruction.” But there is a second theory of destruction—one that rests the blame on King Herod. The Roman-appointed ruler was well-known for destroying Jewish buildings in his bid to become a king over the Jews, and systematically removing the First Wall would have fit with his intentions, according to Re'im and Zindel. “This segment of Jerusalem's ancient city wall uncovered by the Israel Antiquities Authority is tangible and moving evidence of Jerusalem's might and stature during the Hasmonean period,” Amichai Eliyahu, Israeli minister of heritage, said in a statement. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. This Is the Oldest Evidence of Human-Made Fire Archaeologists Found an Ancient Face of Mixed Cultures Experts Found a Village from John Smith's Writings Black Market Coins Led to a Bronze Age Fortress</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/russia-scales-back-ambitions-for-its-next-space-station/'>Russia Quietly Changed Its Space Station Plans. Here's What That Means</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The proposed Russian Orbital Station will stick to a familiar path instead of an ambitious polar orbit Russia's Nauka multipurpose laboratory module (MLM) on the International Space Station. The International Space Station (ISS) is due to close down by 2030, and Russia, one of its main partners, is designing its own replacement orbital laboratory. After a decade of planning to place its future space station into a high-latitude polar orbit for Arctic observation, Russian authorities have changed their minds. The decision goes beyond a routine adjustment to flight plans. Roscosmos officials did not respond to a request for comment. “We are building our own national orbital station, and India is building its own,” Roscosmos general director Dmitry Bakanov said during a visit to New Delhi, according to Russian news source RBC. India has announced plans to launch the first part of its Bharatiya Antariksh Station in 2028. “We are negotiating to place them in the same orbital plane.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Experts, however, are skeptical that alignment with India played more than a rhetorical role. “Compatibility with India in terms of inclination became a conveniently timed additional justification for a decision driven by different considerations,” Payson says. Sticking with the same orbit will allow Roscosmos to reuse designs and technology from ISS and its existing Soyuz spacecraft, he adds. In December 2014 Oleg Ostapenko, then head of Roscosmos, said the agency was considering a high-latitude station that would allow observation of most of Russia's territory and could potentially serve as a staging ground for future lunar missions. Early concepts also envisioned a suite of novel components, including the OKA-T autonomous module for ultrasensitive materials science experiments and an expandable module—Russia's counterpart to the ISS's BEAM—equipped with a centrifuge. The station was repeatedly described as a test bed for prospective elements of lunar infrastructure. Additional docking hardware and follow-on modules would have to be sent up before even a minimal crew could visit. These constraints ultimately pushed the polar orbit station project to a dead end, says military and space analyst Pavel Luzin, a senior fellow at the Saratoga Foundation. “NEM appears impossible to redesign into the core of a future station,” Luzin says. “Even if basic crew systems can be added, there would be little room left for scientific payloads—it would essentially become an enlarged spacecraft.” And there would be no need to upgrade the Vostochny launch pad because crewed launches could rely on proven Soyuz spacecraft from the fully equipped Baikonur Cosmodrome. NEM would retain its original role as a science-and-power module and could be launched aboard the reliable Proton-M rocket. In late 2028 a nearly identical Universal Node Module (UUM) will be launched from Baikonur and attached to Nauka. In 2029 NEM will be launched aboard a Proton rocket from Baikonur and docked to the UUM's lower port. An airlock module, ShM, will follow in 2030, attaching to the side port. Once assembled, the ROS will detach from the rest of ISS, which will have served as a building berth for its Russian “successor.” The new Russian station will then fly independently, with its attitude and orbit controlled by two specially modified Progress cargo spacecraft docked for that purpose. Even these streamlined plans, however, will require substantial work to make the ROS operate independently. The relatively new Nauka module, launched in 2021, already needs repairs, including a fix to its leaking cooling system. All tasks must be completed before June 2030, when the ISS is expected to begin a rapid descent. There may be a gap before Russian cosmonauts can visit the new station. “I believe Russia faces, at the very least, a hiatus in its crewed spaceflight program,” Luzin says. Ilya Ferapontov is a science journalist based in Russia. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67782-6'>Geometry-preserving vector field reconstruction of high-dimensional cell-state dynamics using ddHodge</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 11:06:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The differentiation potency of cells is governed by dynamic changes in gene expression, which can be inferred from single-cell RNA sequencing (scRNA-seq) data. While velocity-based approaches have been used to analyze cell state changes as vector fields, extracting acceleration (change of change) information remains challenging because of the sparsity and high-dimensionality of the data. Here, we develop ddHodge, a framework based on Hodge decomposition for precise vector-field reconstruction. ddHodge accurately recovers all basic components of the vector field, namely, the gradient, curl, and divergence, including the acceleration of the cell state, as second-order derivatives, even from biased and sparse samples. Furthermore, we extend the method to approximate high-dimensional gene expression dynamics on lower-dimensional data manifolds. By applying ddHodge to scRNA-seq data from mouse embryogenesis, we reveal that the gene expression dynamics during development follow a gradient system shaped by potential landscapes, which has not previously been validated with real data. Furthermore, we quantify differentiation potency as cell state stability on the basis of the divergence and identify key genes that drive potency. Our general computational framework for analyzing complex biological systems can elucidate cell fate decisions in developmental processes. The dynamics of cell states, namely, the temporal changes in cell states, underpin all biological phenomena, including development, regeneration, differentiation, and pathology. One of the primary goals of life sciences is to understand these cell state dynamics. However, these dynamics represent a high-dimensional, multibody system involving interactions among tens of thousands of proteins, nucleic acids, and lipid molecules, constituting a complex, nonequilibrium system1,2. Consequently, traditional hypothesis-driven approaches have focused on small subsystems formed by the interactions among the molecules of interest, creating building blocks for understanding the whole system. Recent advancements in single-cell omics3,4,5,6 have enabled comprehensive evaluations of cell states represented by all gene expression profiles, which could allow a data-driven understanding of complex cell state dynamics. However, single-cell measurement techniques are invasive, and the resulting data are static, lacking temporal changes for each cell. Therefore, pseudotime analysis techniques7,8,9 have been employed to construct temporal information based on of large-scale static snapshot data to analyze gene expression dynamics. Nevertheless, conventional pseudotime analysis methods provide only an order of time, and the speed and acceleration/deceleration cannot be evaluated, making it challenging to analyze detailed dynamics, including the stability of states that represent sensitivity of gene expression responses to perturbations, as well as more diverse types of state changes. Therefore, RNA velocity10,11 has frequently been used to trace changes in cell states as trajectories in gene expression space by utilizing the time derivative of the gene expression (velocity) of each cell. Based on this time-derivative inference approach involving gene expression data, methods for reverse engineering these dynamics (vector-field reconstruction) have been proposed, allowing analyses of the interactions among genes that drive local dynamics and the least action paths of the cell state12. However, owing to the lack of a sufficient theoretical foundation to ensure accurate reconstruction of vector fields on the basis of high-dimensional data, a methodology for precisely reconstructing cell state dynamics, appropriately summarizing the dynamics occurring in high-dimensional space, and evaluating the stability of each cell state has yet to be developed. Here, we developed a high-dimensional dynamics analysis framework called ddHodge. Our framework is based on the mathematical theorem of Hodge decomposition, which provides the three fundamental components of vector fields on manifolds13,14. ddHodge offers a unified perspective on the problem of high-dimensional vector-field reconstruction on manifold and provides a practical method to precisely reconstruct dynamics in unprecedented detail. Through simulations and validation using single-cell RNA sequencing (scRNA-seq) data, we demonstrated that ddHodge can provide a summary of the overall unknown high-dimensional dynamics and identify genes associated with fate determination by affecting the stability of cell states. The cell states represented by each data point are generated from an unknown differential equation system F(x, y) (Fig. These states are pairs of observed positions (x, y) and their corresponding velocities F(x, y). Each velocity is given as the gradient of an unknown potential function U. The convex region in the potential landscape, indicates the stability of states that are diversifying; branching is likely to occur, and the concave region is considered to be converged; the cell states adopt specific lineages (Fig. Therefore, the shape of the potential landscape is a critical indicator of dynamics. Here, we used a 2D gradient system as an illustrative example to explain the interpretation of potential. Note that gradient systems are the special cases where vector fields are represented solely by the gradients of a scalar function, and only the potential determines the dynamics. Generally, dynamics are non-gradient systems. The underlying dynamics of the cell state generates single-cell data (left), and the dynamics are represented by a potential landscape (middle). A negative divergence (concave shape) indicates a stable state change in which velocities converge toward the center. A positive divergence (convex shape) indicates an unstable state change in which the cell state trajectories become more diverse. B Complex dynamics in high-dimensional space (left) contain both rotational and gradient components, shown here, consist of spiral source (S1) and sink (S2) components. After sampling scRNA-seq data, conventional approaches using nonlinear dimensionality reduction (UMAP) and velocity embedding lose geometric information, distorting the original dynamics. C ddHodge strategy: Reconstruct dynamics in the low-dimensional tangent space patches and combine these patches smoothly. The low-dimensional tangent spaces at each data point are estimated using a local PCA approach. D ddHodge provides rich information about high-dimensional dynamics: the stability of states (Divergence), the best section to visualize (Schur vector), and the changes of involved genes (Grassmann). Reconstructing cell state dynamics after nonlinear dimensionality reduction involving data deformation is a considerably difficult problem15,16,17. Nevertheless, nonlinear dimensionality reduction is useful for visualization. We set the original high-dimensional (3D) dynamical system to have a region of instability (S1) with a large positive divergence and a region of stability (S2) with a large negative divergence (Fig. However, in commonly used nonlinear dimensionality reduction methods such as UMAP18, the density of the data points in original space is equalized in the reduced space; as a result, geometric features such as length and volume are lost. This deformation affects the landscape shape, making it difficult to identify regions S1 and S2 existed in the original space (Fig. Additionally, the velocity embedding approach10,15 used for visualizing velocity information can misrepresent dynamics by embedding high-dimensional velocity data with neighboring data points in the reduced space, potentially presenting nonexistent closed trajectories at the data boundaries. Hu et al. also showed that trajectories on cylindrical manifolds—formed by combining circular cell cycles trajectory with straight path trajectory—lose essential geometric information under nonlinear dimensionality reduction, which makes appropriate dynamical interpretation difficult19. To address the challenge of reconstructing high-dimensional dynamics, we propose a combination of two approaches. First, we introduce a high-precision vector-field reconstruction method based on Hodge decomposition. The goal is to completely reconstruct these three components. Next, to make this method applicable in high-dimensional spaces, we consider approximating the Hodge components on a low-dimensional manifold (Fig. 1C; dimensionality reduction preserving data geometry). By using local principal component analysis (PCA) to approximate tangent spaces20 and patchworking these spaces, we build a low-dimensional curved space embedded in the high-dimensional space (hereafter referred to as the data manifold) (Fig. 1C, top), enabling the reconstruction of the potential, divergence, and curl in high dimensions (Fig. We implemented this framework as ddHodge. ddHodge allows us to evaluate the temporal order and stability of data points on the basis of the potential and divergence, respectively. The periodic and oscillatory behavior of dynamic processes, such as the cell cycle and circadian rhythms, can be evaluated using the curl and harmonic components, independent of the gradient component. Even after dimensionality reduction, the Grassmann distance (the degree of bending when tangent spaces are patchworked) allows us to identify the timing of significant changes of dimensions (genes). The Jacobian matrix reconstructed on the data manifold can be used to select cross-sections of the gene expression space via Schur decomposition, where the trajectories undergo significant changes. By extracting and visualizing these rich dynamic indicators (Fig. 1D), we can identify genes associated with the diversification and convergence of dynamic cell states. We evaluated the performance of ddHodge with synthetic data (Fig. Additionally, we assessed the ability of ddHodge to reconstruct the shape of the potential landscape (divergence) and compared it with that of conventional methods. A Workflow of the simulation validation approach. We used the nonlinear system F(x, y), which is completely determined by the gradient of a predefined potential U(x, y) in a low-dimensional flat plane (2D Euclidean space) (left). The initial values were set around the highest potential peak (2nd to 3rd panels). Using 100 data points sampled after random time intervals from 0 to 4 seconds (3rd to 4th panels), the potential values were reconstructed (right). These two landscapes (middle column, top and bottom rows) have different shapes with the same potential values. The flat shape with zero divergence (top-right) indicates constant changes of states, whereas the bumpy shape with large positive and negative divergence (bottom-right) indicates unstable states. C Comparison with the DEC method in terms of the divergence reconstruction performance: DEC with Delaunay triangular mesh vs. ddHodge with a k-NN graph. E Toggle-switch model: A nonlinear mathematical model of cell differentiation for generating multistable states is visualized with streamline plots. F ddHodge produces substantially more accurate reconstruction results than does Dynamo: the divergence, curl, and Jacobian (detJ) results obtained using Dynamo and ddHodge are shown. G Accurate recovery of the Jacobian matrix elements with ddHodge. Each panel indicates the estimated value of the Jacobian matrix elements (J11 J12 and J21 J22). To evaluate the performance of ddHodge in reconstructing dynamic information without dimensionality reduction, we conducted simulations of a dynamical system in which the trajectory of each point on a 2D plane was determined solely by the gradient of the potential U, defined as F(x, y) = −∇U(x, y) (Fig. We randomly initialized points near the peak of the potential in the lower left region and simulated their trajectories for random durations between 0-4 seconds, obtaining 100 pairs of position and velocity data (Fig. This result demonstrates that the proposed method can provide high-precision potential reconstruction results even with small datasets and partial sampling of data points. We viewed the least squares potential estimation method as fitting a piecewise quadratic function to the edges of a graph with given slopes at the ends (Supplementary Notes). The shape is approximated by collecting fitted piecewise quadratic functions of each cross-section to the neighboring vertex (Fig. 2B) or experiences acceleration, deceleration, or passes through branching points (changing slope; lower case in Fig. Next, we verified the reliability of ddHodge's results. We compared the divergence reconstruction performance of ddHodge with that of the discrete exterior calculus (DEC) method21,22, which can be realized as a type of finite element method (FEM), a well-established numerical analysis method (Fig. The results showed that ddHodge achieved a reconstruction accuracy comparable to or better than that of the DEC method, even when the data boundary, where the DEC method had lower reproducibility, was excluded (yellow triangles in Fig. Next, to evaluate the reconstruction performance of ddHodge, we compared ddHodge with Dynamo, which has similar vector-field reconstruction capabilities. In this comparison, we used simulation data from the toggle-switch model, a 2D nonlinear differential equation model (Fig. We compared the reconstruction results of the two methods using N = 5000 uniformly sampled pairs of position (x, y) and velocity F(x, y) data from the toggle-switch model12. The results showed that ddHodge significantly outperformed Dynamo in all metrics: the divergence (ddHodge vs. Dynamo: 0.0027 vs. 0.1142), curl (0.0027 vs. 0.0742), and Jacobian (0.0051 vs. 0.1050) (Fig. The vector-field reconstruction function in Dynamo, utilizing SparseVFC23, is based on the theory of reproducing kernel Hilbert spaces (RKHS). In the context of numerical analysis, as explained in the comparison of DEC and ddHodge, this kernel method can also be viewed as a variant of the numerical method. Specifically, this is a variant of the Galerkin method that utilizes radial basis functions, which applies to higher dimensions. Hence, we further verified whether both could achieve equivalent reconstruction performance in terms of the numerical method with appropriate parameter settings. We thus anticipated that increasing the number of kernel basis functions (parameter M) in SparseVFC would improve accuracy. By varying M in the 2D toggle-switch model simulation, we confirmed that Dynamo's reconstruction accuracy is comparable to ddHodge's around M = 1000 (Supplementary Figure S1). In contrast, ddHodge retained advantages in computational efficiency and exhibited a scaling law with respect to the sampling density (cell number), owing to its use of sparse matrices derived from the local k-nearest-neighbor structure, combined with the efficient Krylov subspace solver. We also evaluated the robustness of ddHodge to noise by varying the smoothing parameter λ and neighbor count k at different noise levels (SNR = 1000 to 10; see Supplementary Figure S2 and Methods for the definition of SNR). In high-noise environments (SNR = 10), increasing λ from 0.1 to 10.0 improved MSE from 22.3 to 1.27, and increasing k from 6 to 30 under the same conditions dramatically improved MSE from 1.27 to 0.23. These results suggest that increasing k is effective for noisy data. Furthermore, the Jacobian matrix J can be reconstructed from the accurately reconstructed Hessian matrix and the curl (Methods, Supplementary Notes). We then evaluated the applicability of the proposed method for interaction analyses of gene pairs using J. The clear red and blue bands in the reconstruction results of each element of the Jacobian matrix (∂ẋ/∂x, ∂ẏ/∂x, ∂ẋ/∂y, and ∂ẏ/∂y where ẋ represents dx/dt) faithfully reproduced the regions that respond sensitively to perturbations in genes x and y (Fig. These results demonstrate the ability of ddHodge to reconstruct unknown nonlinear dynamics with high accuracy and efficiency, even from partially sampled data. Next, we aimed to develop a dimensionality reduction method suitable for high-dimensional dynamical systems (Fig. For example, the surface of a sphere in three dimensions can be represented as a collection of small two-dimensional planes (Fig. Based on this concept, we hypothesized that by reconstructing and stitching dynamics within local lower-dimensional spaces (tangent spaces) using a local PCA approach20, both dimensionality reduction and faithful reproductions of high-dimensional dynamics could be achieved while preserving the geometry of the data manifold. We also considered smoothing (regularization) along the curved axes on the data manifold. In the smoothing process, the axes of adjacent tangent spaces should be matched (Fig. 3A, middle; xu and xv, yu and yv); therefore, using idea of parallel transport in differential geometry20,24,25, we developed a method for designing near-parallel coordinate axes for high-dimensional data using a discrete connection Laplacian (sheaf Laplacian)25,26,27,28 (Fig. A Designing near-parallel coordinates on curved spaces: The surface of a sphere, as an example of a curved 2D space embedded in 3D, is represented by small patches of flat tangent planes (top). The arrows (local x- and y-axes, xu and yu) to the adjacent tangent planes involves the problem of parallel transport (middle). The solution is to find the minimum eigenvalue of a sheaf Laplacian (bottom). B Near-parallel coordinates on a sphere: A triangulated spherical mesh is used as the input graph. The insets show two examples of designed near-parallel coordinates. C Near-perfect reconstruction of dynamics on a sphere: Scatter plot of N = 2000 random points comparing the reconstructed (horizontal) and analytical (vertical) values of the potential, divergence, and curl on a sphere. D Sampling scheme from the Lorenz system: N = 2000 points and velocities sampled at random intervals from 0 to 5 s, starting from a random initial point. E Lower-dimensional representation of Lorenz system: The dynamic indices (potential, curl, and divergence) reconstructed in estimated tangent planes via a local PCA approach, visualized on a global (i.e., including all sampled points) PCA plane. F Grassmann distance reveals hinge points: The distance indicates the folding angle when patching local PCA planes. G Schur decomposition: The section that highlights critical changes at a specified point is selected. First, we present a simple example of designing near-parallel coordinates on a sphere (Fig. We used the graph structure of a triangulated mesh as the input. We use the term ‘near'-parallel to indicate directions of movement close to parallel transport on curved space. Singularities, the points that consolidate and resolve coordinate distortions across the entire sphere, appeared to be vortices24 (Fig. The leftmost six identical eigenvalues indicate that the total amount of coordinate distortion remains unchanged, regardless of which vector in the 6-dimensional eigenspace is used to construct the axis. Next, using velocities sampled from the artificial dynamics simulated on a sphere (Methods), we reconstructed the dynamics with smoothing on the data manifold (Fig. 3C; inset images, MSEs: 1.0 × 10−3, 4.0 × 10−4, and 1.0 × 10−3, respectively). We further verified the importance of the subspace alignment procedure using an asymmetric saddle-shaped potential example (Supplementary Figure S3). While smoothing results with unaligned axes significantly deviate from theoretical values of the Hessian matrix, subspace alignment using the sheaf Laplacian enables estimation that perfectly matches the theoretical values. The dynamics of the Lorenz system are characterized by trajectories that irregularly move between two disks in three-dimensional space. Therefore, we attempted to approximate the dynamics in the space of two joint planes as a data manifold and capture the characteristics of the original dynamics (Fig. As expected, the main characteristics of rotational behavior were identified by the positive (anticlockwise) and negative (clockwise) curl values on the left and right sides of this space. In summary, as indicated by the smaller divergence values compared with the curl values, the dynamics of this system were considered dominated by rotations rather than gradients. Here, we used the ddHodge results in three dimensions without approximating the Lorenz system in 2D space (Fig. As expected, prominent rotational dynamics were observed on the SV1-SV2 plane at the point at which the curl has its maximum value (Fig. We also verified the accuracy of Jacobian matrix reconstruction in tangent space using the Lorenz system and a feedback-loop system (Supplementary Fig. Despite information loss due to dimensionality reduction, qualitative consistency with the true Jacobian matrix will be maintained, demonstrating its applicability to gene interaction analysis (Supplementary Figure S4D–F). Before scRNA-seq data analysis, we conducted a 100-gene dynamics simulation based on the RNA velocity model31 to verify ddHodge's ability to detect changes in gene-expression dynamics (Supplementary Figure S5A-D). Note that, in standard scRNA-seq analysis, thousands of genes are reduced to less than 100 dimensions through PCA preprocessing, so ddHodge's local PCA approach naturally integrates with existing analysis workflows. Unlike the Lorenz example, this RNA-velocity simulation presents a difficult setting for ddHodge: because 100 genes evolve independently, no low-dimensional data manifold exists. Nevertheless, after low-dimensional approximation (\(r=4\)), decreases in potential and changes in divergence (Supplementary Figure S5C-D) captured transitions to equilibrium states, the presence of perturbations (rate changes), and points of fate diversification. Next, the scRNA-seq data were analyzed using ddHodge to quantify the potential, which represents the order of cell state transitions, and divergence, which reflects the stability of each cell state (Fig. In our analysis of scRNA-seq data, the following assumptions are adopted: (1) cells progress through state changes governed by an autonomous dynamical system, (2) both expression levels and velocity measurements are obtainable via existing methods, and (3) the resulting dynamics can be represented on a low-dimensional data manifold. These assumptions are widely used in existing RNA velocity and general scRNA-seq data analysis, including dimensionality reduction processes, and ddHodge operates faithfully within these constraints. A scRNA-seq data analysis framework: ddHodge takes scRNA-seq data paired with velocity information (e.g., RNA velocity) as input (left). The recovered dynamic indices are overlaid on visualizations such as those obtained via UMAP (middle), providing a global view of high-dimensional dynamics through landscape representations. From this global view, cells of interest can be determined by the extracted dynamical indicators (right), allowing detailed investigation of specified regions (local view) in which gene expression state changes are prominent, as well as the genes involved. B Dynamical indicators extracted during mouse embryogenesis. In addition to clusters and embryonic stages, four dynamical indicators extracted by ddHodge are shown. For comparison, the results of ddHodge based on the FUCCI dataset, which is expected to have a dominant cyclic component, are also shown. D Changes in the stability of the cell state during development: Box plots show the distribution of divergence values for each developmental stage (Q1, median and Q3; whiskers extend to 1.5 × IQR). The sample sizes (n) are 1,277, 7,134, 22,808 and 14,905 individual cells, respectively. The right image shows a canalization-type landscape, where cells transition from diversification (unstable) to convergence (stable). F Simplified landscape view of development: A potential-divergence plot of the surface ectoderm lineages extracted from (E), represented as thin lines. In this analysis, we focused on VASA-seq data, which provide highly accurate velocity information due to their high degree of intron coverage6. The VASA-seq data contained N = 46,124 cells from mouse embryos in the stage of E6.5–9.5. Using the gene expression levels and RNA velocity as inputs, the components of the Hodge decomposition—potential, divergence, curl, and Grassmann distance—were reconstructed and mapped to a low-dimensional UMAP representation using different colors (Fig. The small Grassmann distance can be interpreted as indicating that the trajectory was confined to almost a flat space, meaning there were no significant changes in the utilized gene sets (dimensions). Conversely, if one finds hinge-like points as shown in the Lorenz system, it can be interpreted that there were points where the utilized gene groups changed significantly. This result suggests that the trajectories of the gene expression dynamics associated with embryonic development reside in a nearly four-dimensional manifold. We also found a correlation between the potential value and developmental stage, i.e., the potential progressively decreases as development progresses, indicating its utility as a pseudotime indicator12 (Supplementary Figure S6). Similarly, the divergence became increasingly negative as development progressed. These findings suggest that the majority of the gene expression dynamics during embryogenesis can be described by a gradient system governed by a potential landscape. To further confirm that the dynamics of embryogenesis is mainly driven by a gradient system, we quantified the proportion of cyclic or periodic components separated from the gradient component (Fig. In this dataset, the gradient component accounted for 88.2% of the gene expression dynamics, whereas the divergence-free cyclic component accounted for only 11.7% of the gene expression dynamics. In the FUCCI data, the cyclic component was dominant (Supplementary Figure S7A, B). In regions where the divergence-free flow dominates, potential cannot provide a faithful temporal ordering (Supplementary Figure S7C). For circular component visualization, subtracting the gradient component from edge weights reveals the distribution of curl and harmonic components, and we observed strong divergence-free components in the G2M region (Supplementary Figure S7D). These findings suggest that embryonic gene expression dynamics can be explained mainly by the potential rather than by periodic components such as the cell cycle. To further elucidate the biological significance of this divergence, we analyzed its changes over the course of embryogenesis (Fig. During the early stages (E6.5–E7.5), most cells exhibited positive divergence, indicating an undifferentiated and unstable (divergent) state. In contrast, from E8.5 to E9.5, the divergence values became increasingly negative, suggesting that the gene expression states of the cells tended to converge and stabilize as development progressed. This finding indicates that early embryonic cells exhibit a high degree of plasticity, which subsequently decreases as differentiation becomes more defined and cell states stabilize. Thus, we attempted to describe the dynamics of embryogenesis using only the potential and divergence (Fig. The epiblast, which possesses pluripotency with high divergence, maintained its cellular identity with a shallow slope (gradient) between E6.5 and E7.5 but was no longer detectable after E8.5. In contrast, cell populations that had diverged into distinct lineages, including the splanchnic mesoderm, emerged beneath the epiblast from E7.5 onward and progressively descended along the landscape toward E9.5, differentiating into more stable cardiac and epithelial lineages. These findings suggest that we successfully visualized the fate decision process that occurs during development according to the height and shape of the potential landscape during gastrulation. Next, we focused on the surface ectoderm lineage (Fig. The surface ectoderm, a nonneural ectoderm, gives rise to the epithelium and epidermis of sensory organs. Even at E8.5, pre-epidermal keratinocytes appeared at a higher positions in the plot (potential > 0.5) than the placodal area, suggesting that the epidermal lineage may be specified earlier than the epithelial lineage33. Furthermore, pre-epidermal keratinocytes and the otic epithelium exhibited stable cell states at E9.5, as indicated by the highly negative divergence values (approximately −0.01). In contrast, the divergence of the fusing epithelium, which exhibits gene expression characteristics associated with the amniotic ectoderm, remained close to zero, indicating that the cell state changed with a constant speed, which is a unique property of this type of cell. Taken together, our results demonstrate that the developmental trajectory of the surface ectoderm lineage is consistent with a recent single-cell study of embryogenesis34,35. Moreover, we successfully quantified the potency of differentiation at each developmental stage using the potential and divergence. As nonlinear dimensionality reduction is not performed in ddHodge, the genes that drive cell state changes can be tracked. We thus focused on individual points of interest (POIs) at which cell states undergo significant changes and investigated the local dynamics (Fig. A (Case 1) Stabilization of a cell state with a highly negative divergence in axon elongation: 2000 cells surrounding the selected spinal cord cells that had minimal negative divergence values in cluster 18, including the neuromesodermal progenitor (NMP) and forebrain/midbrain cell types, were selected (inset circle; enlarged image of the UMAP plot near the POI). C (Case 2) Diversification of the cell state with a highly positive divergence in the amniochorionic mesoderm: 2000 cells surrounding amniochorionic mesoderm cells that had maximal positive divergence values in cluster 14, including allantois, extraembryonic mesoderm, and splanchnic mesoderm cell types, were selected. To identify cells that become more stable, namely, those with negative divergence values, we selected the POI with the smallest divergence within cluster 18 (Fig. To determine the most related genes in this section, we identified candidate genes using the correlation coefficients between the divergence and gene expression levels for 1000 neighboring cells in the POI (Supplementary Figure S8). The velocities projected onto the SV2-SV4 planes demonstrated that the cell states converged and stabilized (negative divergence); furthermore, Pax6 expression was identified36 (Fig. Additionally, Nrcam and Robo2, which are genes related to axon guidance37, exhibited strong negative correlations with the divergence values, similar to Pax6, suggesting their involvement in the stabilization of cell states during axial elongation. In contrast, another cell population annotated as neuromesodermal progenitors (NMPs) expressing Cdx2 was positively correlated with divergence (Fig. 5B, top and bottom left panels, Supplementary Figure S8). Cdx2, which is regulated by multiple extracellular signals, has been reported to play a role in maintaining bipotent NMP populations38,39. These findings suggest that the mechanism maintaining undifferentiated NMPs may involve destabilization of cell states through Cdx2 expression. In summary, through this analysis, we identified several genes associated with the spinal cord lineage involved in regulating cell state stability. To identify genes associated with cell state diversification, we selected another POI with the highest divergence from cluster 14 (Fig. The neighboring cells of this POI primarily consisted of two amnion-like cell groups (annotated as A and B) (Fig. Therefore, only group A, which expresses Ptch1, may undergo de-repression via Shh signaling and transition to a distinct cell state. Additionally, Slc2a1, which encodes a glucose transporter of Glut1, presented the highest loading value on the SV3 axis; however, its expression level increased with developmental progression in both A and B. In summary, this axis-based approach can provide a biological interpretation similar to that of PCA. As an extra verification with real data, we applied ddHodge to a muscle regeneration dataset41. ddHodge-derived metrics can be visualized by coloring points on UMAP, focusing on muscle satellite-cell activation and muscle differentiation processes (Supplementary Figure S10A). High-divergence regions (red) were observed on the upper side of the MuSCs cluster, suggesting the need for a detailed analysis of presumable fate-bifurcation points. This high-divergence region was selected as a POI. Schur decomposition analysis was performed (Supplementary Figure S10B). By visualizing divergence values and velocity-vector projections on the SV1–SV2 plane, we succeeded in extracting the moment of fate-determination in asymmetric-division, which is composed of Myog (muscle-differentiation marker) and Cdkn1c (cell-cycle control) expression axes (Supplementary Figure S10C). Furthermore, compared to ordinal differential gene expression (DEG) analysis aimed at detecting marker genes selectively expressed in each cluster, ddHodge has the characteristic of detecting genes that change rapidly (accelerate/decelerate or diverge/contraction) within clusters, so the importance scores of both do not necessarily match (Supplementary Figure S11). ddHodge's exploratory analysis using divergence as an indicator successfully captures the behavior of cellular state destabilization at this fate bifurcation point, that tend to be overlooked by conventional RNA velocity analysis and DEG analysis. In addition to divergence, other dynamical indicators calculated by ddHodge can be exported in anndata format, making it compatible with many single-cell data visualization tools (Supplementary Figure S12). This enables interactive POI selection considering biological context while confirming gene expression and cell-type annotations within each cell type. Here, we developed ddHodge as a method for reconstructing the original vector field using pairs of sampling points and their velocities obtained from an unknown system. By utilizing RNA velocities as inputs, ddHodge extends the capabilities of existing methods by allowing evaluations of both the direction of cell state changes and the stability of each state, which indicates a potency of change, which has not previously been evaluated. We further confirmed that the majority of the gene expression dynamics in embryogenesis can be described as a gradient system (i.e., a potential landscape), which was previously unclear. Furthermore, we quantified the difficulty of lineage transitions from the divergence values, complementing traditional trajectory analyses of cell fate conceptualized by the epigenetic landscape. Additionally, we identified genes that influence the stability of the cell state. In this study, we demonstrated that the potential and divergence reconstructed from RNA velocity data can serve as indicators for identifying points at which cell states converge and diversify. Furthermore, ddHodge can be used to extract cross-sections of high-dimensional space and identify key regulatory genes that govern local dynamics. Since this framework is general and requires only a weak assumption that the velocity is sampled from an autonomous system, ddHodge can be broadly applied to various types of velocity data, including chromatin velocity data, protein velocity data, time-resolved measurements of cellular positions, and even simulation data from theoretical models of high-dimensional dynamical systems. Many methods have been proposed for scRNA-seq dynamics analysis, each with different interests and goals42. Methods such as DeepVelo43 and GraphVelo44 primarily focus on improving the estimation or embedding of RNA velocity, with the explicit vector-field reconstruction falling within the scope of Dynamo and ddHodge. By contrast, Graph-Dynamo17 takes a different conceptual approach, modeling stochastic cellular-state dynamics governed by a Fokker–Planck equation, focusing on transition probabilities among metastable states and exit times. ddHodge performs component-wise and precise reconstruction of vector fields—potential, divergence, and curl–on manifold via Hodge decomposition in a DEC–like, structure-preserving manner. It further provides principled cross-sectional analysis in tangent space (via Schur decomposition) while ensuring geometry-preserving dimensional reduction. Reconstructing a high-dimensional Jacobian that encodes all gene interactions is fundamentally impossible after dimensionality reduction. A similar limitation arises when the dynamics are intrinsically confined to a low-dimensional data manifold, such as the disk-like structures in the Lorenz and feedback-loop examples, because velocity components orthogonal to this low-dimensional structure are never observed. In these situations, just as after dimensionality reduction, the full ambient-space Jacobian cannot be identified by any method—including ddHodge or Dynamo—unless additional assumptions or biases are imposed on the unobserved directions. The Schur vectors of the tangent-space Jacobian can be interpreted as composite axes, analogous to PCA loading vectors. 5, this enables identification and interpretation of genes that drive fate transitions, including those exhibiting rapid expression changes. ddHodge reconstructs the fundamental components of vector fields, including the potential, divergence, curl, and harmonic by overcoming several difficulties associated with analyses performed in high-dimensional spaces. For example, in the DEC method, these difficulties include constructing the mesh, evaluating areas and volumes, which are required to determine the densities and constructing discrete Hodge stars21,22. With our approach, we could accurately estimate the divergence by applying previously intractable numerical analysis methods to higher-dimensional spaces without considering boundary conditions. Additionally, we developed a method for estimating the curl in spaces of three or more dimensions. To address concerns regarding the reliability of the results due to noise and limited sample sizes—challenges inherent in data analysis—we incorporated a dedicated smoothing strategy in ddHodge. However, due to computational constraints, we have not yet produced a strict reconstruction of the curl component with ddHodge; instead, an approximation strategy that recovers the rotational features only within the PC1-2 plane is employed in the current version of ddHodge. We observed regions that exhibited higher curl values in the mouse gastrulation data, specifically in areas associated with hematopoietic processes. These regions may represent dynamics like strong perturbation components orthogonal to the gradient direction, rather than gentle periodic/cyclical dynamics. In this paper, we did not focus on the interpretation of strong curl components. However, in the future, we need to pursue a deeper biological interpretation, as well as a complete reconstruction of the curl components. Further interpretation of divergence-free periodic components may be feasible by developing efficient methods for computing curl by Hodge duals in high-dimensional spaces and establishing appropriate interpretative frameworks, such as optimal cycles45, to obtain a succinct representation of the harmonic components in higher-dimensional spaces. Owing to its scalability to high-dimensional data without the need for mesh construction, this method has potential as a tool for supporting data analysis involving high-dimensional complex dynamic systems. This section outlines the ddHodge workflow. Detailed explanations and illustrations are provided in the Supplemental Notes. Additionally, a \(k\)-nearest neighbors (k-NN) graph \({{\mathcal{G}}}=\left(V,E\right)\) is constructed from \({{\mathcal{X}}}\). Since the tangent spaces of neighboring vertices generally have different subspaces, we construct a discrete connection Laplacian \({\Delta }_{s}\) using sheaves on graphs to align the local coordinates within the tangent spaces. The alignment is performed by minimizing the following loss function \(L\) over the set of tangent vectors \({{{\boldsymbol{z}}}}_{i}\in {T}_{i}{{\mathcal{M}}}\): The potential \({{\boldsymbol{u}}}\in {{\mathbb{R}}}^{N}\) corresponding to the gradient component of the vector field is estimated using the gradient operator \({\delta }_{0}={{\rm{grad}}}\), which is an \(M\times N\) matrix, and the edge weights \({{\boldsymbol{w}}}\in {{\mathbb{R}}}^{M}\) of graph \({{\mathcal{G}}}\). where \(N\) and \(M\) represent the number of vertices and edges in \({{\mathcal{G}}}\), respectively. In this study, the edge weights \({w}_{\left(i,j\right)}\) are defined as: In this definition, the integral of the flux is approximated along the edge direction and is equivalent to fitting a quadratic function to the cross-section of potential function \(U\) (see Supplementary Notes for more details). For a point \({{{\boldsymbol{x}}}}_{i}\), let \({{\boldsymbol{\eta }}}={{{\boldsymbol{x}}}}_{j}-{{{\boldsymbol{x}}}}_{i}\) be the vector indicating the direction from this point to a neighboring point \({{{\boldsymbol{x}}}}_{j}\). The second-order directional differential coefficient of the fitted quadratic function is given by: The Hessian matrix \(H\) in the original coordinate system can be obtained via a quadratic form. For example, in two dimensions of x-y plane, the relationship is expressed as: By gathering the known second-order differential coefficients such as \({U}_{\eta \eta }\) in independent directions \({\bf{\varphi}},{\boldsymbol{\psi}},\ldots\), the linear equation can be constructed to estimate the elements of \(H\). We denote the above equation as \(C{{\boldsymbol{\beta }}}={{\boldsymbol{y}}}\). The divergence is obtained from \(-{{\rm{tr}}}H\) after estimating \(H\). In practice, the problem is better solved as an optimization problem \({\min }_{{{\boldsymbol{\beta }}}}\parallel C{{\boldsymbol{\beta }}}-{{\boldsymbol{y}}}{\parallel }^{2}\) to mitigate issues in data analysis, such as those related to noise and the limited sample size. Additionally, since the number of Hessian matrix elements (number of estimated parameters) increases on the order of \({d}^{2}\), the Hessian matrix is estimated in the reduced-dimensional space. To enable the regularization of the estimation process between neighboring vertices, we merged the individual Hessian matrix estimation problems as a single linear equation: This allows us to introduce a penalty (Dirichlet energy on the graph) to the squares of the differences \({{{\boldsymbol{\beta }}}}_{i}-{{{\boldsymbol{\beta }}}}_{j}\) between adjacent vertices as: Here, assume that the \(p\) parameters are arranged in a certain order as \({{{\boldsymbol{\beta }}}}_{i}=\left({U}_{{xx}}^{i},{U}_{{xy}}^{i},{U}_{{yy}}^{i}\right)\), such that the differences between each parameter can be individually evaluated using the extended gradient matrix \({D}_{0}={\delta }_{0}\otimes {I}_{p}\) with the Kronecker product and \({L}_{0}={D}_{0}^{{{\top }}}{D}_{0}\). The optimization problem with this regularization term is then solved as: Given the large \({Np}\times {Np}\) size of the sparse matrix \({L}_{0}\), efficient solvers such as the Krylov subspace method are recommended. For differential forms in the Euclidean space \({{\mathbb{R}}}^{2}\), the relationship between \({{\rm{div}}}\) and \({{\rm{curl}}}\) is given by: According to the theory of Hodge decomposition, a vector field can be expressed as the sum of a gradient part \(-\nabla U\) and a divergence-free residual part \(r\): Since \(D\) is linear and \(D\nabla U\) gives the Hessian matrix \(H\) of \(U\), \(D\left(f\right)\) has the following decomposition: If \(r\) consists of only rotational components (i.e., there are no holes in the space), \(D\left(r\right)\) can be expressed using the matrix \(Q\), representing a \(\pi /2\) rotation in the PC1-2 plane, as \(D\left(r\right)=\frac{\theta }{2}Q\). ddHodge was implemented in Julia, a high-performance scientific programming language. The most computationally intensive parts are computing eigenvalues and solving least-squares problems with large sparse matrices. To optimize these calculations, we employ the Krylov subspace method (KrylovKit.jl), which efficiently handles sparse matrices by leveraging repeated matrix‒vector multiplication operations. With GPU acceleration (we used NVIDIA RTX A2000 12 GB), the computation takes approximately 35 seconds for 46,124 cells with the parameter: k = 12 of k-NN graph and r = 4 of reduced dimension, and approximately 2 minutes for another dataset with 98,316 cells with the same parameter (Figs. To create a gradient system in a 2D plane, we use the Gaussian function as follows: We constructed a potential function with peaks and valleys as follows: The simulation is performed by adding noise (Gaussian noise with a standard deviation of 0.2) to the initial positions at the center of the potential peak and running random simulations for 0 to 4 s using the Runge–Kutta method. We sampled \(N=100\) points and their velocities. The toggle-switch model is defined by the following set of differential equations: The parameters are set to \({K}_{1}={K}_{2}=1/2,n=4\), and \({a}_{1}={a}_{2}={b}_{1}={b}_{2}=1\). The parameters are set to \(\sigma=10\), \(\rho=28\), and \(\beta=8/3\). The simulation is performed by initializing \(\left(x,y,z\right)=\left(10,10,10\right)\) with added Gaussian noise \(N\left(3,0\right)\) and sampling points after a random time \(T\), which was generated from a uniform distribution \(T\sim U\left(0.5,5\right)\). We consider RNA-velocity data with scaled degradation rate \(\widetilde{\gamma }=\gamma /\beta=1.0\). Such a condition emulates a scenario in which the steady states lie on the diagonal \({u}_{s}={s}_{s}\) in the spliced–unspliced plane (i.e., the ratio of spliced to unspliced counts is 1:1). We generated a proxy for the unspliced count \(u=\widetilde{\gamma }s+v\) because \({ds}/{dt}\propto v=u-\widetilde{\gamma }s\), where \(v\) is the (scaled) velocity. Then, SNR is defined by \({\sigma }_{s}/{\sigma }_{n}\) when the noise is drawn from \({{\mathscr{N}}}((0,0),{\sigma }_{s}^{2}I)\). The edge weights \({{\boldsymbol{w}}}\) are determined by velocity (see the Potential estimation section in Methods). Then, we calculate the vector \({{\boldsymbol{u}}}\) representing potential values at vertices, which allows us to determine the gradient component at edges as \(-{\delta }_{0}{{\boldsymbol{u}}}\). Next, the cyclic (divergence-free) component \({{\boldsymbol{r}}}={{\boldsymbol{w}}}+{\delta }_{0}{{\boldsymbol{u}}}\) obtained by subtracting the gradient \(-{\delta }_{0}{{\boldsymbol{u}}}\) from weight \({{\boldsymbol{w}}}\). These quantities are summarized as follows: The gradient and divergence-free components are orthogonal, and they satisfy \({{\langle }}{\delta }_{0}{{\boldsymbol{u}}},{{\boldsymbol{r}}}{{\rangle }}=0\). Velocities were estimated using scvelo (version 0.3.2). For cell-type annotation, the scRNA-seq data of mouse embryos34 were used as training data for Celltypist (version 1.6.3), and the cell-type labels were given for the VASA-seq data. The analysis using ddHodge was performed with a \(k\)-NN graph where \(k=12\) and the reduced dimensionality was \(r=4\). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All datasets analyzed in this study are publicly available. For VASA-seq data, spliced and unspliced read counts were obtained from GEO (accession: GSE176588). The processed loom file of FUCCI data was obtained from GitHub (CellProfiling/SingleCellProteogenomics [https://github.com/CellProfiling/SingleCellProteogenomics]). For the regenerating muscle tissue dataset, the raw sequence reads were obtained from NCBI (accession: SRP241205). The processed data from the VASA-seq and muscle regeneration dataset, along with various dynamic indicators calculated by ddHodge, are deposited in Figshare (https://doi.org/10.6084/m9.figshare.28723652). Source data for all figures are available in the Figshare repository and in the GitHub repository described below. The Jupyter notebooks used to reproduce the figures and the related processed data in this manuscript are also available at https://github.com/kazumits/ddHodge_figures. Furusawa, C. & Kaneko, K. A dynamical-systems view of stem cell biology. Fang, X., Kruse, K., Lu, T. & Wang, J. Nonequilibrium physics in biology. Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Hashimshony, T. et al. CEL-Seq2: sensitive highly-multiplexed single-cell RNA-Seq. Sasagawa, Y. et al. Quartz-Seq2: a high-throughput single-cell RNA-sequencing method that effectively uses limited sequence reads. Salmen, F. et al. High-throughput total RNA sequencing in single cells using VASA-seq. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Haghverdi, L., Buettner, F. & Theis, F. J. Diffusion maps for high-dimensional single-cell analysis of differentiation data. Street, K. et al. Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics. La Manno, G. et al. RNA velocity of single cells. Bergen, V., Lange, M., Peidli, S., Wolf, F. A. & Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Qiu, X. et al. Mapping transcriptomic vector fields of single cells. Statistical ranking and combinatorial Hodge theory. Lim, L.-H. Hodge Laplacians on Graphs. Lange, M. et al. CellRank for directed single-cell fate mapping. Towards reliable quantification of cell state velocities. Zhang, Y. et al. Graph-Dynamo: Learning stochastic cellular state transition dynamics from single cell data. McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. Hu, S. et al. Epithelial-mesenchymal transition couples with cell cycle arrest at various stages. & Wu, H.-T. Vector diffusion maps and the connection Laplacian. https://doi.org/10.7907/ZHY8-V329 (California Institute of Technology, 2003). Desbrun, M., Hirani, A. N., Leok, M. & Marsden, J. E. Discrete Exterior Calculus. Ma, J., Zhao, J., Tian, J., Bai, X. & Tu, Z. Regularized vector field learning with sparse approximation for mismatch removal. Crane, K., Desbrun, M. & Schröder, P. Trivial connections on discrete surfaces. & Racanière, S. Disentangling by subspace diffusion. & Ghrist, R. Toward a spectral theory of cellular sheaves. & Ghrist, R. Opinion dynamics on discourse sheaves. Bodnar, C., Di Giovanni, F., Chamberlain, B. P., Liò, P. & Bronstein, M. Neural sheaf diffusion: a topological perspective on heterophily and oversmoothing in GNNs. Berwick-Evans, D., Hirani, A. N. & Schubel, M. D. Discrete Vector Bundles with Connection and the Bianchi Identity. Reuter, B., Fackeldey, K. & Weber, M. Generalized Markov modeling of nonreversible molecular kinetics. Gorin, G., Fang, M., Chari, T. & Pachter, L. RNA velocity unraveled. Mahdessian, D. et al. Spatiotemporal dissection of the cell cycle with single-cell proteogenomics. A., Taketo, M. M., Dufort, D. & Groves, A. K. Wnt signals mediate a fate decision between otic placode and epidermis. Systematic reconstruction of cellular trajectories across mouse embryogenesis. Qiu, C. et al. A single-cell time-lapse of mouse prenatal development from gastrula to birth. Ericson, J. et al. Pax6 Controls Progenitor Cell Identity and Neuronal Fate in Response to Graded Shh Signaling. Thompson, H., Andrews, W., Parnavelas, J. G. & Erskine, L. Robo2 is required for Slit-mediated intraretinal axon guidance. Henrique, D., Abranches, E., Verrier, L. & Storey, K. G. Neuromesodermal progenitors and the making of the spinal cord. Gouti, M. et al. A Gene regulatory network balances neural and mesoderm specification during vertebrate trunk development. De Micheli, A. J. et al. Single-cell analysis of the muscle stem cell hierarchy identifies heterotypic communication signals involved in skeletal muscle regeneration. Zhang, Z., Sun, Y., Peng, Q., Li, T. & Zhou, P. Integrating dynamical systems modeling with spatiotemporal scRNA-seq data analysis. Zhanlin, Chen., William, C. K., Aheyon, H., Mark, G. & Jing, Z. DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations. Chen, Y. et al. GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells. Dey, T. K., Hirani, A. N. & Krishnamoorthy, B. Optimal Homologous Cycles, Total Unimodularity, and Linear Programming. Maehara, K. Geometry-preserving vector field reconstruction of high-dimensional cell-state dynamics using ddHodge. Yusuke Imoto and Yasuaki Hiraoka at Kyoto University for insightful discussions, and Advanced Research Initiative, Research Promotion Unit, Medical Institute of Bioregulation, Kyushu University, for their support. This work was supported by JST PRESTO JPMJPR2026, JST CREST JPMJCR23N3, JPMJCR24Q1, JSPS KAKENHI JP22H04696, JP23H04288 and JP25H01484 to K.M., AMED BINDS JP22ama121017j0001, JSPS KAKENHI JP24H02323, JP23H00372, JP22H04676 and JP22K19275 to Y.O. This work was also supported in part by the MEXT Promotion of Development of a Joint Usage / Research System Project: Pan-Omics DDRIC, MRCI for High Depth Omics, CURE:JPMXP1323015486 for MIB and RIIT and Autonomous Medical Research Center in Kyushu University. Department of Multi-Omics, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar built the theoretical framework of ddHodge and developed software, K.M. The authors declare no competing interests. Nature Communications thanks Jianhua Xing, who co-reviewed with Yan Zhang, Xiaojie Qiu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Maehara, K., Ohkawa, Y. Geometry-preserving vector field reconstruction of high-dimensional cell-state dynamics using ddHodge. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02446-1'>p53 inactivation drives breast cancer metastasis to the brain through SCD1 upregulation and increased fatty acid metabolism</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 11:06:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Brain metastasis (BM) carries a poor prognosis, yet the molecular basis of brain tropism remains unclear. Analysis of breast cancer BM (BCBM) revealed pervasive p53 inactivation through mutations and/or aneuploidy, with pathway disruption already present in primary tumors. Functionally, p53 inactivation markedly increased BCBM formation and growth in vivo, causally linking p53 perturbation to BM. Mechanistically, p53 inactivation upregulated SCD1 and fatty acid synthesis (FAS), essential for brain-metastasizing cells; SCD1 knockout abolished the p53-dependent growth advantage. Molecularly, p53 suppressed SCD1 directly through promoter binding and indirectly by downregulating its co-activator DEPDC1. Astrocytes further enhanced FAS by secreting factors that were metabolized in a p53-dependent manner, promoting tumor survival, proliferation and migration. Finally, p53-deficient tumors were sensitive to FAS inhibition ex vivo and in vivo. Thus, we identify p53 inactivation as a driver of BCBM, reveal p53-dependent and astrocyte-dependent FAS modulation and highlight FAS as a therapeutically targetable BCBM vulnerability. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Buy this article Prices may be subject to local taxes which are calculated during checkout TCGA4,15, METABRIC5, MBC17,29,30 and MSK16,17,19 data were obtained from cBioPortal73. 75) were obtained from the GEO. CAL51 TP53-KO RNA sequencing data were from ref. RNA sequencing data for the hTERT-HMEC engineered cell lines were deposited to SRA under accession PRJNA1246204. RNA sequencing data of metastases were deposited to GEO under accession GSE245414. RNA sequencing data from patient autopsy samples are available upon request from BROCADE, subject to Access Committee and ethical approvals. These restrictions are in place to ensure that the data are used in compliance with ethical guidelines for patient confidentiality and informed consent. Application details are provided at https://www.petermac.org/research/cohort-studies/brocade/information-on-brocade-for-researchers. Applications are typically reviewed within 4–6 weeks. RNA sequencing data of MDA-MB231 cells after siRNA-mediated knockdown of DEPDC1 were deposited to GEO under accession GSE281411. P53 CUT&Tag data were deposited to SRA, under accession PRJNA1247024. Source data are provided with this paper. Chen, W., Hoffmann, A. D., Liu, H. & Liu, X. Organotropism: new insights into molecular mechanisms of breast cancer metastasis. Cancer Genome Atlas Network Comprehensive molecular portraits of human breast tumours. The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Liu, Y. et al. Deletions linked to TP53 loss drive cancer through p53-independent mechanisms. Ferraro, G. B. et al. Fatty acid synthesis is required for breast cancer brain metastasis. Lorger, M. Tumor microenvironment in the brain. Reciprocal interactions between innate immune cells and astrocytes facilitate neuroinflammation and brain metastasis via lipocalin-2. Schwartz, H. et al. Incipient melanoma brain metastases instigate astrogliosis and neuroinflammation. Priego, N. et al. STAT3 labels a subpopulation of reactive astrocytes required for brain metastasis article. Inflammatory activation of astrocytes facilitates melanoma brain tropism via the CXCL10-CXCR3 signaling axis. Zou, Y. et al. Polyunsaturated fatty acids from astrocytes activate PPARγ signaling in cancer cells to promote brain metastasis. Jin, X. et al. A metastasis map of human cancer cell lines. Aaltonen, L. A. et al. Pan-cancer analysis of whole genomes. The genomic landscape of endocrine-resistant advanced breast cancers. Zehir, A. et al. Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients. The somatic mutation profiles of 2,433 breast cancers refine their genomic and transcriptomic landscapes. Nguyen, B. et al. Genomic characterization of metastatic patterns from prospective clinical sequencing of 25,000 patients. De Mattos-Arruda, L. et al. The genomic and immune landscapes of lethal metastatic breast cancer. Schrijver, W. A. M. E. et al. Mutation profiling of key cancer genes in primary breast cancers and their distant metastases. Integrated RNA and DNA sequencing reveals early drivers of metastatic breast cancer. Integrated genomic and transcriptomic analysis of human brain metastases identifies alterations of potential clinical significance. Ding, L. et al. Genome remodelling in a basal-like breast cancer metastasis and xenograft. Piccirilli, C. et al. Allelic deletions on chromosome-17 and mutations in the p53 gene in tumors metastatic to brain. Wang, Z. et al. Loss-of-function but not gain-of-function properties of mutant TP53 are critical for the proliferation, survival and metastasis of a broad range of cancer cells. Xu, J. et al. 14-3-3ζ turns TGF-β's function from tumor suppressor to metastasis promoter in breast cancer by contextual changes of Smad partners from p53 to Gli2. Latent bone metastasis in breast cancer tied to Src-dependent survival signals. Jain, E. The Metastatic Breast Cancer Project: leveraging patient-partnered research to expand the clinical and genomic landscape of metastatic breast cancer and accelerate discoveries. Preprint at medRxiv https://doi.org/10.1101/2023.06.07.23291117 (2023). Parry, M. Introducing the Metastatic Breast Cancer Project: a novel patient-partnered initiative to accelerate understanding of MBC. & Hemminki, K. Clinical landscape of cancer metastases. Cancer Genome Atlas Research Network Comprehensive molecular profiling of lung adenocarcinoma. Xiong, Z., Gharagozlou, S., Vengco, I., Chen, W. & Ohlfest, J. R. Effective CpG immunotherapy of breast carcinoma prevents but fails to eradicate established brain metastasis. Rockwell, S. C., Kallman, R. F. & Fajardo, L. F. Characteristics of a serially transplanted mouse mammary tumor and its tissue-culture-adapted derivative. Establishment and characterisation of a new tumorigenic cell line with a normal karyotype derived from a human breast adenocarcinoma. Redman-Rivera, L. N. et al. Acquisition of aneuploidy drives mutant p53-associated gain-of-function phenotypes. The microenvironmental landscape of brain tumors. Savino, A. M. et al. Metabolic adaptation of acute lymphoblastic leukemia to the central nervous system microenvironment depends on stearoyl-CoA desaturase. Schild, T., Low, V., Blenis, J. & Gomes, A. P. Unique metabolic adaptations dictate distal organ-specific metastatic colonization. Perelroizen, R. et al. Astrocyte immunometabolic regulation of the tumour microenvironment drives glioblastoma pathogenicity. Elahi, L. S. et al. Valproic acid targets IDH1 mutants through alteration of lipid metabolism. Ferris, H. A. et al. Loss of astrocyte cholesterol synthesis disrupts neuronal function and alters whole-body metabolism. Moore, S. A. Polyunsaturated fatty acid synthesis and release. Medina, J. M. & Tabernero, A. Astrocyte-synthesized oleic acid behaves as a neurotrophic factor for neurons. Bernoud, N. et al. Astrocytes are mainly responsible for the polyunsaturated fatty acid enrichment in blood–brain barrier endothelial cells in vitro. Altea-Manzano, P. et al. A palmitate-rich metastatic niche enables metastasis growth via p65 acetylation resulting in pro-metastatic NF-κB signaling. Koundouros, N. & Poulogiannis, G. Reprogramming of fatty acid metabolism in cancer. Guthmann, F., Haupt, R., Looman, A. C., Spener, F. & Rüstow, B. Fatty acid translocase/CD36 mediates the uptake of palmitate by type II pneumocytes. Targeting metastasis-initiating cells through the fatty acid receptor CD36. Vallvé, J.-C. et al. Unsaturated fatty acids and their oxidation products stimulate CD36 gene expression in human macrophages. Yang, P. et al. Dietary oleic acid-induced CD36 promotes cervical cancer cell growth and metastasis via up-regulation Src/ERK pathway. Feng, W. W., Zuppe, H. T. & Kurokawa, M. The role of CD36 in cancer progression and its value as a therapeutic target. The multifaceted roles of fatty acid synthesis in cancer. Sen, U., Coleman, C. & Sen, T. Stearoyl coenzyme A desaturase-1: multitasker in cancer, metabolism, and ferroptosis. The landscape of cancer cell line metabolism. Kaya-Okur, H. S., Janssens, D. H., Henikoff, J. G., Ahmad, K. & Henikoff, S. Efficient low-cost chromatin profiling with CUT&Tag. Kirschner, K. et al. Phenotype specific analyses reveal distinct regulatory mechanism for chronically activated p53. Global transcriptional program of p53 target genes during the process of apoptosis and cell cycle progression. Bené, H., Lasky, D. & Ntambi, J. M. Cloning and characterization of the human stearoyl-CoA desaturase gene promoter: transcriptional activation by sterol regulatory element binding protein and repression by polyunsaturated fatty acids and cholesterol. Girardini, J. E. et al. A Pin1/mutant p53 axis promotes aggressiveness in breast cancer. Zhao, H. et al. High expression of DEPDC1 promotes malignant phenotypes of breast cancer cells and predicts poor prognosis in patients with breast cancer. Brigandi, R. A., Zhu, J., Murnane, A. & Shakib, S. A phase 1 randomized, placebo-controlled trial with a topical inhibitor of stearoyl-coenzyme A desaturase 1 under occluded and nonoccluded conditions. & Lupu, R. Fatty acid synthase (FASN) as a therapeutic target in breast cancer. & Dobrzyn, P. Stearoyl-CoA desaturase 1 as a therapeutic target for the treatment of cancer. Cheng, Y.-J., Fan, F., Zhang, Z. & Zhang, H. Lipid metabolism in malignant tumor brain metastasis: reprogramming and therapeutic potential. Defining a cancer dependency map. Sivanand, S. et al. Cancer tissue of origin constrains the growth and metabolism of metastases. Ramaswamy, S., Ross, K. N., Lander, E. S. & Golub, T. R. A molecular signature of metastasis in primary solid tumors. Sanghvi, N. et al. Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues. Targeting SCD triggers lipotoxicity of cancer cells and enhances anti-tumor immunity in breast cancer brain metastasis mouse models. Zou, Z., Ohta, T. & Oki, S. ChIP-Atlas 3.0: a data-mining suite to explore chromosome architecture together with large-scale regulome data. Zou, Z., Ohta, T., Miura, F. & Oki, S. ChIP-Atlas 2021 update: a data-mining suite for exploring epigenomic landscapes by fully integrating ChIP-seq, ATAC-seq and Bisulfite-seq data. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Selection of bone metastasis seeds by mesenchymal signals in the primary tumor stroma. Brain metastasis prediction by transcriptomic profiling in triple-negative breast cancer. Distinct gene expression profiles between primary breast cancers and brain metastases from pair-matched samples. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Reich, M. et al. GenePattern 2.0. Smid, M. et al. Subtypes of breast cancer show preferential site of relapse. The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. Van Neerven, S. et al. Inflammatory cytokine release of astrocytes in vitro is reduced by all-trans retinoic acid. Agnese, S. T., Spierto, F. W. & Hannon, W. H. Evaluation of four reagents for delipidation of serum. Targeting angiogenesis-dependent calcified neoplasms using combined polymer therapeutics. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Shalem, O. et al. Genome-scale CRISPR-Cas9 knockout screening in human cells. Wang, B. et al. ATXN1L, CIC, and ETS transcription factors modulate sensitivity to MAPK pathway inhibition. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. Tanaka, A., To, J., O'Brien, B., Donnelly, S. & Lund, M. Selection of reliable reference genes for the normalisation of gene expression levels following time course LPS stimulation of murine bone marrow derived macrophages. Pozzi, S. et al. MCP-1/CCR2 axis inhibition sensitizes the brain microenvironment against melanoma brain metastasis progression. Folch, J., Lees, M. & Sloane Stanley, G. H. A simple method for the isolation and purification of total lipides from animal tissues. Leikin-Frenkel, A. et al. Dietary α linolenic acid in pregnant mice and during weaning increases brain docosahexaenoic acid and improves recognition memory in the offspring. Leikin-Frenkel, A. et al. The effect of α-linolenic acid enrichment in perinatal diets in preventing high fat diet-induced SCD1 increased activity and lipid disarray in adult offspring of low density lipoprotein receptor knockout (LDLRKO) mice. Viral infection of the marine alga Emiliania huxleyi triggers lipidome remodeling and induces the production of highly saturated triacylglycerol. Zheng, L. et al. Fumarate induces redox-dependent senescence by modifying glutathione metabolism. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Zhang, Y. et al. Model-based Analysis of ChIP–Seq (MACS). Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Stuart, T., Srivastava, A., Madad, S., Lareau, C. A. & Satija, R. Single-cell chromatin state analysis with Signac. Zerbib, J. et al. Human aneuploid cells depend on the RAF/MEK/ERK pathway for overcoming increased DNA damage. Discovery of tumor-specific irreversible inhibitors of stearoyl CoA desaturase. Zhu, L. et al. A clinically compatible drug-screening platform based on organotypic cultures identifies vulnerabilities to prevent and treat brain metastasis. Iorio, F. et al. A landscape of pharmacogenomic interactions in cancer. The authors thank J. Pietenpol (Vanderbilt University Medical Center) for generously sharing isogenic CAL51 cells with various TP53 perturbations; N. Erez (Tel Aviv University), A. Sonneblick (Schneider Children's Medical Center), S. Izraeli (Tel Aviv University) and O. Shoshani (Weizmann Institute) for helpful discussions; M. Itkin and A. Felder from the Life Science Core facilities at WIS for their assistance with metabolomics; P. Ofek, E. Yeini and Y. Zilberstein (Tel Aviv University) for assistance with intracranial and intracardiac injections, monitoring and imaging of mice; I. Shur and D. Makarovsky (Tel Aviv University) for assistance with FACS sorting; S. Israeli (Schneider Children's Medical Center), A. Savino (Schneider Children's Medical Center), A. Ruban (Tel Aviv University) and Y. Goldshmit (Tel Aviv University) for the sharing of reagents; P. Cassoni (University of Turin) for funding support of the Bertero lab; and R. Serwa from the IMol Proteomic Core Facility for assistance with proteomics. The authors acknowledge the patients who donated samples, as well as their families, who supported their wish to donate to the BROCADE Rapid Autopsy Program. The authors are grateful for the support of BROCADE, initially supported by the Australian National Breast Cancer Foundation (NBCF) under an infrastructure grant (IF-14-001). The authors would like to thank the study co-PIs, R. Anderson (ONJCRI/PMCC) and A. Swarbrick (Garvan Institute of Medical Research) and L. Devereux, who manages BROCADE and helped with the identification, collection and processing of samples. The authors also acknowledge the Victorian Institute of Forensic Medicine that conducted the rapid autopsies; P. MacCallum, Kinghorn Centre, Garvan Institute and Tobin Brothers Funeral Directors for providing transport of donors. The BROCADE study operates under Ethics and Governance oversight at the PMCC. This work was supported by the European Research Council Starting Grant (945674 to U.B.-D.), the Israel Cancer Research Fund Project Award (to U.B.-D.), the Azrieli Foundation Faculty Fellowship (to U.B.-D.), the Israel Science Foundation (grants 1805/21 and 3755/21 to U.B.-D.), the BSF (project grant 2019228 to U.B.-D.), the Schreiber Foundation of Tel Aviv University Faculty of Medicine (to U.B.-D.) and the Tel Aviv University/LaTrobe University/ONJCRI Joint Collaboration Research Program in Cancer Research (to U.B.-D. and D.M.). is supported by the Love Your Sister Foundation and the Victorian Cancer Agency Mid-Career Research Fellowship (MCRF21011) and the Australian NBCF (IIRS0049). is supported by the Breast Cancer Alliance Young Investigator Grant. acknowledge support from LOEWE Center Frankfurt Cancer Institute, supported by the Hessen State Ministry for Higher Education, Research, and the Arts (III L 5-519/03/03.001-(0015)). H.M. additionally acknowledges support from the Federal Ministry of Education and Research and is cofunded by the European Commission (01KT2304A). acknowledges support from the Fondazione AIRC IG (grants 22174 and 30570), the Fondazione AIRC Special Program Molecular Clinical Oncology ‘5 per 1,000' (grant 22759), Worldwide Cancer Research (grant 24-0361), the NRRP NextGenerationEU Project CN00000041-National Center for Gene Therapy and Drugs based on RNA Technology to GDS. was supported by an AIRC fellowship for Italy. R.A.-P. is supported by Israel Science Foundation (700/24), DFG (706375), the Yoran Institute for Human Genome Research, the Zimin Institute, Claire and Amédée Maratier Institute for the Study of Blindness and Tel Aviv University. R.S.-F. is supported by the European Research Council (ERC) Advanced Grant (835227), 3DBrainStrom and the ERC PoC grant (862580), 3DCanPredict, the Israel Science Foundation (ISF 1969/18, ISF 3706/24), the Israel Cancer Research Fund Professorship Award (PROF-18-602) and the Morris Kahn Foundation. These authors contributed equally: Kathrin Laue, Sabina Pozzi. These authors jointly supervised this work: Ronit Satchi-Fainaro, Uri Ben-David. Department of Human Molecular Genetics and Biochemistry, Gray Faculty of Medical and Health Sciences, Tel Aviv University, Tel Aviv, Israel Kathrin Laue, Johanna Zerbib, Yonatan Eliezer, Yael Cohen-Sharir, Tom Winkler, Rachel Slutsky, Katharina Lange, Ron Saad & Uri Ben-David Department of Physiology and Pharmacology, Gray Faculty of Medical and Health Sciences, Tel Aviv University, Tel Aviv, Israel Sabina Pozzi, Rami Khoury, Giuseppe Longobardi, Sahar Israeli Dangoor & Ronit Satchi-Fainaro International Centre for Genetic Engineering and Biotechnology (ICGEB), Area Science Park—Padriciano Campus, Trieste, Italy Rebecca Bertolio, Manuel Caputo, Alessandra Rustighi, Silvano Piazza & Giannino Del Sal Pathology Unit, Department of Medical Sciences, University of Turin, Turin, Italy Alessia A. Ricci & Luca Bertero Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel Lital Adler & Ayelet Erez Gray Faculty of Medical and Health Sciences, Tel Aviv University, Tel Aviv, Israel Alicia I. Leikin-Frenkel & Iris Barshack The Strassburger Lipid Center, Sheba Medical Center, Tel Hashomer, Ramat Gan, Israel Department of Human Molecular Genetics and Biochemistry, Faculty of Medical & Health Sciences and Sagol School of Neuroscience, Tel Aviv University, Tel Aviv, Israel Shai Ovadia & Ruth Ashery-Padan European Molecular Biology Laboratory (EMBL), Genome Biology Unit, Heidelberg, Germany Department of Life Sciences, University of Trieste, Trieste, Italy Alessandra Rustighi & Giannino Del Sal Translational Oncology Research Unit, Regina Elena National Cancer Institute, Rome, Italy Andrea Sacconi & Giovanni Blandino Department of Systems Biology, University of Massachusetts Chan Medical School, Worcester, MA, USA Rayna Y. Magesh, Faith N. Keller & Emma V. Watson Olivia Newton-John Cancer Research Institute, Melbourne, Victoria, Australia Jean Berthelet, Yang Liao, Wei Shi & Delphine Merino School of Cancer Medicine, La Trobe University, Melbourne, Victoria, Australia Jean Berthelet & Delphine Merino Georg-Speyer-Haus, Institute for Tumor Biology and Experimental Therapy, Frankfurt am Main, Germany Alexander Schäffer & Hind Medyouf IMol Polish Academy of Sciences, Warsaw, Poland Institute of Pathology, Sheba Medical Center, Tel HaShomer, Ramat Gan, Israel Life Science Core Facility, Weizmann Institute of Science, Rehovot, Israel Sergey Malitsky & Alexander Brandis Department of Neurosurgery, University Hospital Frankfurt, Goethe University Frankfurt, Frankfurt am Main, Germany Thomas Broggini & Marcus Czabanka Thomas Broggini & Marcus Czabanka Immunology Division, Walter and Eliza Hall Institute of Medical Research, Melbourne, Victoria, Australia Department of Medical Biology, Faculty of Medicine, Dentistry, and Health Science, The University of Melbourne, Melbourne, Victoria, Australia IFOM ETS, The AIRC Institute of Molecular Oncology, Milan, Italy Sagol School of Neurosciences, Tel Aviv University, Tel Aviv, Israel Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and U.B.-D. conceived the study. performed most of the computational analyses and in vitro experiments, assisted with in vivo experiments and led the data analysis of all experiments. S. Pozzi performed most of the in vivo animal experiments, spheroid experiments and astrocyte derivation. assisted with animal experiments. and R. Slutsky performed in vitro experiments. and R. Saad assisted with genomic data analyses. contributed the hTERT-HMEC model and its gene expression analysis. shared proteomics data of MCF7 cells. R.K. and A.I.L.-F. performed the GC experiments. performed the isotope tracing experiments with the help of S.M. contributed clinical samples of BCBM and performed IHC experiments. contributed clinical samples of BCBM. provided RNA-seq of clinical samples of metastases from various organs. and R.A.-P. performed CUT&Tag experiments. and H.M. contributed clinical BM samples and performed drug treatments of patient-derived organotypic cultures. S. Piazza performed bioinformatic analyses of the DEPDC1 RNA-seq data. and M. Caputo performed cell culture and biochemical analyses. oversaw the RNA-seq and biochemical analyses of DEPDC1 in BC cells. R.S.-F. directed and supervised the ex vivo spheroid experiments and in vivo work. U.B.-D. directed and supervised the overall study. and U.B.-D. wrote the manuscript with input from all co-authors. Correspondence to Ronit Satchi-Fainaro or Uri Ben-David. R.S.-F. is a board director at Teva Pharmaceutical Industries Ltd. and receives unrelated research funding from Merck KGaA. U.B.-D. received consulting fees from Accent Therapeutics and received unrelated research funding from Novocure. The remaining authors declare no competing interests. Nature Genetics thanks S. Bakhoum and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Distribution of TP53 perturbation modes in primary BC across three clinical datasets (TCGA, METABRIC, MSK-IMPACT). del17p prevalence: TCGA 86%, METABRIC 81%, MSK-IMPACT 71%. See Supplementary Table 1. b, Distribution of TP53 biallelic perturbation modes in primary BC from the TCGA cohort: 92% of biallelic inactivation cases combine del17p and a point mutation. c, The effect of TP53 status on its gene expression. TP53 mRNA levels are lower in BCs with TP53 copy-number losses (regardless of TP53 mutation status) in TCGA (left) and METABRIC (right) datasets. Deletions of TP53 occurred almost exclusively through del17p, whereas focal deletions were rare (2%). Boxplots: bar = median; box = 25–75th percentile; whiskers = range. d, Gene set enrichment analysis (GSEA) confirms that the p53 pathway activity is lower in TP53-perturbed vs. TP53-WT primary BCs, irrespective of TP53 perturbation mode (METABRIC, ‘P53_DN.V2.DN'; p-values and FDR-corrected q-values indicated). e, Proteomics-based ssGSEA scores of p53 targets in MCF7 cell strains show lower p53 activity in del17p vs. 17p-WT MCF7 cells. TMT multiplexing mass spectrometry-based quantification, ‘Oncogenic signatures, P53_DN.V2_DN', two-sided t-test *p = 0.03. 17p-WT: n = 2, del17p: n = 3. f, Generation of human mammary epithelial (HMEC) cells with TP53 alterations: diploid HMECs spontaneously gained a heterozygous p53 mutation (Y163S) in culture, impairing p53 function. The TP53-WT/mut clone subsequently lost the remaining TP53-WT allele through del17p, resulting in TP53 biallelic inactivation (TP53-mut/del17p). In parallel, CRISPR/Cas9 was used to generate a knockout line on the TP53-WT/mut background, also resulting in TP53 biallelic inactivation (TP53-mut/KO). g, RNA-seq-based comparison of mRNA levels of several p53 targets (CDKN1A, MDM2, TP53INP1) shows their decrease in TP53-mut/del17p cells and TP53-mut/KO vs. TP53-WT; intermediate levels were observed in TP53-WT/mut cells. h, GSEA of hTERT-HMECs with various TP53 alterations (TP53-WT/mut, TP53-mut/del17p, or TP53-mut/KO) reveals reduced p53 pathway expression (‘p53-targets Nutlin_UP, U2OS' gene set) in comparison to TP53-WT cells, which is lower in cells with biallelic vs. monoallelic inactivation. P-values and FDR-corrected q-values indicated. i, Comparison of the prevalence of TP53 alterations in primary breast tumors and across metastatic sites. TP53 alterations are more frequent in BMs vs. primary tumors or other metastatic sites (MSK 2018). Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BMs vs. primary tumors, ****p = 1 × 10−4; BMs vs. other metastatic sites, ****p = 4 × 10−5; primary tumors vs. others, p = 0.19. Panels f and i were created with BioRender.com. j, Comparison of the prevalence of TP53 biallelic perturbations in primary breast tumors and across metastatic sites. TP53 biallelic perturbations are more frequent in BMs vs. primary breast tumors or other metastatic sites (MSK 2018). Two-sided Fisher's exact test, TP53 biallelic inactivation vs. TP53-WT: BMs vs. primary tumors, ***p = 1.8 × 10−4; BMs vs. others, ***p = 4 × 10−4; primary tumors vs. others, p = 0.33. Primary n = 477, brain n = 26, lung n = 66, liver n = 218, bone n = 116, others n = 322. k, ssGSEA scores of the p53 pathway in brain metastases (brain) and metastases from other organs (others), derived from the same single patient. p53 pathway activity was lower in brain vs. other metastatic sites from the same patient (brain: n = 1, others: n = 6; gray: ‘P53_DN.V1_DN', purple: ‘P53_DN.V1_UP'). a–c, GSEA comparing primary tumors of patients who developed BMs and those who developed metastases in other organs. a, GSEA plot showing downregulation of a p53 expression signature in primary tumors that metastasized to the brain vs. bone (MBC cohort). Oncogenic signature ‘p53_DN.V1_DN', brain vs. bone p = 0.005. Brain n = 7, bone = 24. b, GSEA plot showing upregulation of a signature indicative of decreased p53 signaling in primary BCs that metastasized to the brain in comparison to other sites in a second dataset (GSE12276). Brain vs. others, oncogenic signature ‘P53_DN.V2_UP', p = 0.003. Brain n = 7, others n = 121. c, GSEA plots showing downregulation of a signature indicative of increased p53 signaling (left) and upregulation of a signature indicative of decreased p53 signaling (right) in triple-negative primary breast tumors that metastasized to the brain vs. those that did not (data source GSE76714)75. Brain vs. others, oncogenic signature: ‘P53_DN.V1_DN' p = 0.018, oncogenic signatures: P53_DN.V2_UP p = 0.035, brain n = 21, others n = 38. See Supplementary Table 3. d, GSEA of primary BCs (TCGA cohort), comparing tumors with TP53 inactivation to TP53-WT tumors. TP53 inactivation: n = 623, TP53-WT: n = 259. e, ssGSEA of brain-relapse signatures in primary BCs with TP53 inactivation compared to TP53-WT tumors (MBC cohort). Tumors with TP53 inactivation showed increased expression of signatures positively correlated with BM. See Supplementary Table 4. f, Correlation between BM potential and p53 pathway activity levels in human BC. High potential for BM is strongly correlated with low p53 pathway activity (METABRIC cohort). Readout for p53 pathway activity: ssGSEA scores for ‘Kannan_TP53_targets' gene set. Readout for brain-metastatic potential: ssGSEA scores for ‘Smid_breast_cancer_relapse_in_brain' gene set. Two-sided Pearson's correlation r = −0.41, p = 2.9 × 10−54. g, ssGSEA of brain-relapse signatures in primary BCs that metastasized to the brain compared to those that metastasized elsewhere. See Supplementary Table 5. h,i, GSEA of BM gene expression gene sets in tumors with TP53 mutations, hotspot mutations or copy-number loss, compared to TP53-WT tumors, in the METABRIC cohort. GSEA showed an increase in gene expression that is upregulated in BM and a decrease in gene expression that is downregulated in BM, in (h) tumors with monoallelic TP53 inactivation vs. TP53-WT tumors; and (i) tumors with biallelic TP53 inactivation vs. TP53-WT tumors, independent of the mode of p53 inactivation. P-values and q-values indicated in the respective panels. a, Correlation of TP53 inactivation with BM prevalence across carcinomas, for female (left) and male (right) tumors. Each dot represents a primary carcinoma (for lung and kidney, the highest-ranking subtypes were considered). Female: one-sided Spearman's ρ = 0.58, p = 0.04. Male: one-sided Spearman's ρ = 0.80, p = 0.003. b, Pan-cancer analysis of TP53 perturbation prevalence: TP53 alterations, and biallelic TP53 inactivation in particular, are enriched in BM in comparison to primary tumors and to other metastatic sites (MSK-IMPACT cohort17). Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BM vs. primary tumor, ****p = 1 × 10−5, BM vs. other metastasis, ****p = 1 × 10−5. TP53 biallelic inactivation vs. TP53-WT: BM vs. primary tumor, ****p = 1 × 10−5, BM vs. other metastasis ****p = 1 × 10−5. c, In total, 367 carcinoma cell lines were grouped by their BM potential (top vs. low 25%). Panel c was created with BioRender.com. d, High-BM carcinoma cell lines show enrichment of TP53 biallelic inactivation (two-sided Fisher's exact test *p = 0.03). e, High-BM carcinoma cell lines show decreased p53 signaling, as evaluated by GSEA. Oncogenic signatures: ‘P53_DN.V1_DN' p = 0.015, rank #3. See Supplementary Table 3. f–i, TP53 alterations are enriched in BM vs. other metastases from primary lung tumors. f, Non-small cell lung cancer metastasis. Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BM vs. primary tumors ****p = 1 × 10−5, BM vs. other metastasis ***p = 8 × 10−4. TP53 biallelic inactivation vs. TP53-WT: BM vs. primary tumors ****p = 1 × 10−5, BM vs. other metastasis ****p = 1 × 10−5. Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BM vs. primary tumors ****p = 1 × 10−5, BM vs. other metastasis **p = 0.0017. TP53 biallelic inactivation vs. TP53-WT: BM vs. primary tumors ****p = 1 × 10−5, BM vs. other metastasis ***p = 1 × 10−4. Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BM vs. primary tumors p = 0.13, BM vs. other metastasis p = 0.55. TP53 biallelic inactivation vs. TP53-WT: BM vs. primary tumor p = 0.57, BM vs. other metastasis **p = 0.01. Two-sided Fisher's exact test, TP53 inactivation vs. TP53-WT: BM vs. primary tumors p = 1, BM vs. other metastasis p = 1. TP53 biallelic inactivation vs. TP53-WT: BM vs. primary tumors p = 0.4, BM vs. other metastasis *p = 0.045. j, Comparison of tumors based on their TP53 status. Panel j was created with BioRender.com. k, TP53-deficient primary lung adenocarcinomas show increased expression of BM signatures (TCGA PanCancer Atlas). l, ssGSEA comparison shows that TP53 inactivation is associated with increased BM expression signatures. m, ssGSEA comparison shows that TP53 inactivation is associated with decreased bone-metastasis expression signatures. Two-sided t-test, ‘Smid_breast_cancer_relapse_in_bone' ****p = 1.2 × 10−9, ‘Smid_breast_cancer_relapse_in_bone_up' ****p = 4.8 × 10−8, ‘Smid_breast_cancer_relapse_in_bone_dn' ****p = 5.9 × 10−5.. n, TP53-deficient primary lung adenocarcinomas with biallelic inactivation show increased expression of BM signatures (TCGA PanCancer Atlas). TP53-null n = 169, TP53-WT n = 185. o, ssGSEA comparison shows that TP53 biallelic inactivation is associated with increased BM expression signatures. p, ssGSEA comparison shows that TP53 biallelic inactivation is associated with decreased bone-metastasis expression signatures. a, Experimental design and timeline of an intra-cardiac injection competition assay of EMT6 Trp53-WT and Trp53-null isogenic cells. Fluorescently-labeled Trp53-WT (red) and Trp53-null (green) EMT6 isogenic cells were injected into the same mice. The entire cohort was sacrificed after the first mouse died. In a first experiment only brains were analyzed, whereas in an additional experiment 5 organs (brain, lung, liver, bone and spleen) were collected and analyzed for fluorescence. b, Fluorescence images of dissected organs analyzed for the presence of Trp53-WT (red) and Trp53-null (green) cells. Fluorescence was calibrated to an organ from an uninjected control mouse to account for autofluorescence. Trp53-null tumors were present in 4/5 brains, but only one brain contained a tumor derived from Trp53-WT cells. The opposite trend was observed in bone samples (femur and tibia): Trp53-WT tumors were present in the bone of all mice, whereas no bone contained a tumor derived from Trp53-null cells. c, Fluorescence-based quantification of metastasis incidence to brain, lung, liver, bone and spleen following intra-cardiac injection of Trp53-WT and Trp53-null cells into the same mice. Increased number of brain lesions was detected with Trp53-null eGFP-labeled cells in comparison to Trp53-WT (sgRNA control) mCherry-labeled cells. Two-sided Fisher's exact test, brain *p = 0.02. n = 10 mice. The opposite trend was observed for bone tumors that only formed from Trp53-WT cells, two-sided Fisher's exact test, bone **p = 0.007. n = 5 mice. d, Increased tumor burden in the brain in mice from Trp53-null cells following intra-cardiac co-injection of Trp53-WT and Trp53-null BC cells. Tumor burden was defined as fluorescent signal over area. Trp53-WT: n = 5, Trp53-null: n = 5. e, Increased number of fluorescent foci in brain, but not in other organs, from Trp53-null cells in mice co-injected with isogenic Trp53-WT and Trp53-null BC cells. Number of fluorescent foci per organ. Trp53-WT n = 5, Trp53-null n = 5. f, Mouse brains were processed, and tissue sections were obtained from 5 areas of each brain. Panels a and f were created with BioRender.com. g, Immunofluorescence images of brain sections visualizing isogenic cancer cells. Trp53-null are labeled in green and Trp53-WT cells in red, sections were counterstained with DAPI to visualize cell nuclei. h–j, Fluorescence-based quantification of micrometastases derived from Trp53-WT and Trp53-null cells. h, The number of Trp53-null cells throughout the brain was significantly higher. Two-sided t-test **p = 0.006. i, The number of fluorescent Trp53-null cells per slide was significantly higher. Two-sided t-test **p = 0.003. j, Mouse brains contained bigger cell clusters of Trp53-null cells following competitive injection of labeled Trp53-null BC cells and their isogenic WT controls. Quantification of Trp53-null and Trp53-WT cell cluster size in brain sections from five brain areas. Fold change Trp53-null over Trp53-WT as % cell population. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a, Fluorescence-based quantification of microglia in the EMT6-derived tumors. Iba1 staining intensity was similar in tumors growing from Trp53-null and Trp53-WT BC cells. Mice n = 3. b, Fluorescence-based quantification of endothelial cells in the EMT6-derived tumors. CD31 staining intensity was similar in tumors growing from Trp53-null and Trp53-WT BC cells. Two-sided t-test p = 0.23, Tumor sections evaluated: Trp53-WT n = 11, Trp53-null n = 13. Mice n = 3. c, Live imaging-based proliferation curves of EMT6 BC cells co-cultured with mouse astrocytes. The effect of co-culture on the growth of Trp53-null BC cells is stronger than that of Trp53-WT cells. d, Live imaging-based proliferation curves of BC EMT6 cells cultured with ACM. The effect of ACM on the growth of Trp53-null BC cells is stronger than its effect on Trp53-WT cells. e, Live imaging-based proliferation curves of EMT6 BC cells cultured with ACM. The effect of ACM on the growth of Trp53-null BC cells is stronger than its effect on Trp53-WT cells (shown as cell numbers). f, Flow cytometry analysis of BC cells cultured with ACM. The fraction of cells in apoptosis is lower in Trp53-null than in Trp53-WT cells. Fraction of AnnexinV+ cells is shown as fold change relative to SFM controls. Two-sided t-test, Trp53-null vs. Trp53-WT, **p = 0.01, n = 4. g, Transwell migration assay to assess the migration of BC cells towards ACM. BC cells were seeded in SFM in the upper compartment and ACM or astrocytes were placed in the lower compartment. h, Quantification of cell migration in the transwell assay after 20 h of co-culture of BC cells and astrocytes, showing all conditions analyzed (see Fig. The effect of astrocytes on cell migration was stronger in Trp53-null cells. i, Quantification of cell migration in the transwell assay after 20 h of co-culture of BC cells and ACM. The effect of ACM on cell migration was significantly stronger in Trp53-null cells. The fold increase of migrating cells in ACM over SFM was significant at **p = 0.005. Fields counted TP53-WT n = 17, TP53-null n = 12. j, Live imaging-based quantification of the effect of ACM on cell velocity of Trp53-WT and Trp53-null cells. Cell velocity shown as fold increase in ACM over SFM. Each data point represents a velocity measurement; n = 47. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a–d, ssGSEA comparisons show upregulation of adipogenesis, FA synthesis, uptake, and activation in BMs vs. other metastatic sites in several cohorts: a, GSE14018; b, GSE245414; c, BROCADE patient 6071 autopsy data; and d, in BMs compared to their matched primary breast tumors (GSE125989)76. For the list of gene sets and exact p-values, see Supplementary Tables 7–9. Panels a–d were created with BioRender.com. e, ssGSEA comparisons show strong upregulation of FA synthesis/uptake in tumors with biallelic TP53 inactivation; tumors with monoallelic TP53 perturbation show weaker but significant effects (METABRIC cohort). For the list of gene sets and exact p-values, see Supplementary Table 7. f, GSEA comparing gene expression signatures related to lipid metabolism in HMEC cells with monoallelic (TP53-WT/mut) or biallelic (TP53-mut/del17p and TP53-mut/KO) TP53 inactivation, in comparison to their parental TP53-WT cells. Shown as a heatmap, upregulated signatures in red and downregulated signatures in blue. TP53-WT/mut cells showed intermediate p53 pathway expression levels and elevated lipid metabolism expression levels. g, Metabolomics analysis of OA-containing triglycerides reveals enrichment in ACM vs. SFM (two-sided t-test ****p = 2.3 × 10−5). See Supplementary Table 13. h, Live-imaging-based proliferation curves show that Trp53-null EMT6 cells grow better than their isogenic Trp53-WT controls when lipid levels are limited, in line with increased FA synthesis in Trp53-null cells. i, Viability quantification shows that Trp53-null cells survive better under lipid starvation conditions (two-sided t-test ***p = 3 × 10−4). Trp53-WT n = 4, Trp53-null n = 4. j, High PA (50 µM) was better tolerated by Trp53-null vs. -WT BC cells (doubling time relative to control). Paired one-sided t-test *p = 0.03). n = 3. k,l, CD36 protein quantification. k, Representative gel. l, Western blot quantification shows CD36 protein upregulation in TP53-null CAL51 cells, especially when cultured in ACM (one-sided t-test *p = 0.02, n = 3). CD36 protein levels are shown as fold change of ACM over WT SFM conditions. Bar = median; whiskers = range highest to lowest data points. m, Comparison of CD36 mRNA expression levels in primary BCs of the luminal B subtype, with monoallelic or biallelic TP53 inactivation vs. TP53-WT tumors, independent of inactivation mode (TCGA and METABRIC cohorts combined). Tumors with biallelic TP53 inactivation expressed lower levels of CD36, whereas tumors with monoallelic TP53 inactivation showed intermediate CD36 mRNA levels. TP53-WT n = 159, TP53 monoallelic inactivation n = 179, TP53 biallelic inactivation n = 126, TP53-WT/mut: n = 20, TP53-WT/del: n = 159. n, Comparison of CD36 mRNA expression levels in human primary BCs with TP53 biallelic inactivation vs TP53-WT of the Luminal A molecular subtype (TCGA). TP53-null tumors exhibited higher CD36 mRNA expression levels. Shown as fold-increase CD36 expression relative to the average expression of TP53-WT tumors. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a, SCD1 expression is higher in BMs vs. their matched primary tumors (GSE125989). Paired two-sided t-test *p = 0.04. Matched primary tumors and BMs: n = 16. b,c, SCD1 and FASN mRNA expression levels are elevated in BMs vs. other metastases. BM: n = 7, metastases in other sites: n = 29. c, GSE245414: one-sided t-test, SCD1 **p = 0.01, FASN **p = 0.01. BM: n = 6, other metastatic sites: n = 26. d,e, SCD1 mRNA levels were higher in human BCs with monoallelic or biallelic TP53 inactivation, irrespective of their mode of p53 inactivation (mutation or copy-number loss). TP53-WT: n = 266, TP53-WT/del: n = 287, TP53-WT/mut: n = 50, biallelic TP53 inactivation: n = 279. e, METABRIC, one-sided t-test: TP53-monoallelic inactivation vs. TP53-WT **p = 0.009; TP53-WT/del vs. TP53-WT *p = 0.02; TP53-WT/mut vs. TP53-WT *p = 0.01; biallelic TP53 inactivation vs. TP53-WT ***p = 6 × 10−4. TP53-WT: n = 592, biallelic TP53 inactivation: n = 324, TP53-WT/del: n = 280, TP53-WT/mut: n = 138. f, FASN mRNA levels were higher in primary BCs with biallelic TP53 inactivation vs. TP53-WT tumors (TCGA; LumB subtype). TP53-WT: n = 31, biallelic TP53 inactivation: n = 63. g, FASN mRNA levels were higher in tumors with biallelic TP53 inactivation vs. TP53-WT tumors (METABRIC; LumB subtype). TP53-WT: n = 128, TP53-mut/del: n = 63. h, FASN mRNA levels were higher in human BCs with monoallelic or biallelic TP53 inactivation, irrespective of their mode of p53 inactivation (TCGA and METABRIC cohorts combined; LumA subtype). Low p53 pathway activity (ssGSEA score, ‘Kannan_p53_targets') correlates with high FASN mRNA levels across primary BCs that metastasized to the brain (GSE125989). Two-sided Spearman's ρ = –0.47, p = 0.035, n = 16. j, Scd1 mRNA levels by qPCR: Scd1 expression was higher in Trp53-null vs. -WT EMT6 cells grown in full medium, delipidated medium, or SFM (One-sided t-test *p = 0.01, **p = 0.003, **p = 0.003). Full: n = 4, delipidated: n = 3, SFM: n = 4. k, Fasn mRNA levels by qPCR. Fasn expression was higher in Trp53-null EMT6 grown in full medium, delipidated medium or ACM. l,m, ACM increased SCD1 protein levels more in TP53-null and TP53-mut (R175H, R273H) vs. TP53-WT cells in an additional CAL51 isogenic cell system. l, Representative gel. One-sided one-sample t-test, TP53-WT ACM vs. SFM *p = 0.02; TP53-null ACM vs. SFM **p = 0.002; TP53-mut (R175H) ACM vs. SFM **p = 0.01; TP53-mut (273H) ACM vs. SFM *p = 0.02. TP53-mut (R175H): n = 9, TP53-mut (R273H): n = 6. n,o, ACM increased FASN protein levels more in TP53-null and TP53-mut (R175H, R273H) vs. TP53-WT cells in an additional CAL51 isogenic cell system. One-sided one-sample t-test, TP53-WT ACM vs. SFM p = 0.32; TP53-null ACM vs. SFM *p = 0.01; TP53-mut (R175H) ACM vs. SFM **p = 0.002; TP53-mut (273H) ACM vs. SFM *p = 0.04. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a, Quantification of FA content using gas chromatography, assessing POA over PA enrichment in EMT6 Trp53-WT and Trp53-null cells after ACM exposure for 20 h. Two-sided t-test, ***p = 8 × 10−4, n = 3. b, 13C-tracing shows higher POA M + 16 fractional enrichment in Trp53-null cells after 13C-PA incubation (20 µM, 25 h). One-sided t-test **p = 0.004, n = 4. c, 13C-PA tracing shows higher POA isotope mean enrichment in Trp53-null vs. WT after 13C-PA incubation. One-sided t-test **p = 0.002, n = 4. d, SCD1 and CD36 mRNA are positively correlated in BCBM (GSE14017). Two-sided Spearman's ρ = 0.66, p = 0.008, n = 15. e, Schematic illustration of the FA synthesis pathway, with potential carbon sources. f, LC-MS analysis of polar metabolites in ACM shows elevated glutamate but unchanged glucose in ACM vs. SFM. See Supplementary Table 13. g, Metabolic profiling55 of human BC cell lines reveals lower glutamate levels in TP53-null vs. TP53-WT BC cells. TP53-WT: n = 4, TP53-null: n = 20. h, Representative images of GFP+ Trp53-WT and Trp53-null cells after 48 h in SFM, ACM, or glutamate-supplemented SFM (75 mM). Glutamate improved proliferation, more strongly in Trp53-null. i, Live-cell imaging proliferation curves under the same conditions show that glutamate partially phenocopied ACM effect (two-sided t-test *p = 0.02 (36 h); *p = 0.04 (48 h); n = 3). j–l, SCD1 and FASN protein levels in isogenic Trp53-WT and Trp53-null cells cultured in SFM with or without glutamate (75 mM). k,l, Western blot quantification shows that ACM exposure elevated SCD1 and FASN protein levels in Trp53-null cells. Trp53-WT: n = 4, Trp53-null: n = 4. m, 13C-glutamate tracing (24 h) into PA (M + 2) shows higher enrichment in Trp53-null vs. WT. Two-sided t-test *p = 0.03, n = 3. Bar = median; whiskers = range highest to lowest data points. n, 13C-glutamate tracing (24 h) into POA (M + 2) shows higher enrichment in Trp53-null vs. WT. One-sided t-test *p = 0.02, n = 3. Bar = median; whiskers = range highest to lowest data points. o, PA mean enrichment from 13C-glutamate (left) or 13C-glucose (right) after 24-h incubation. Isotope-labeled PA was significantly enriched in Trp53-null cells in comparison to the Trp53-WT controls after 13C-glutamate incubation. One-sided t-test, Trp53-null vs. Trp53-WT, *p = 0.02, n = 3. No difference in PA mean enrichment in Trp53-null vs. Trp53-WT cells was observed after 13C-glucose incubation. Bar = median; whiskers = range highest to lowest data points. p, ¹³C-glutamate tracing (24 h) into POA showed higher enrichment in Trp53-null vs. Trp53-WT. One-sided t-test *p = 0.03, n = 3. Bar = median; whiskers = range highest to lowest data points. q, Schematic illustration of OA isotopologues after 13C-glutamate labeling. Panels a and q were created with BioRender.com. r, Quantification of OA isotopologue distribution after 13C-glutamate incubation (48 h) shows higher M + 2 and M + 4 fractions in Trp53-null vs. Trp53-WT. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a, Overlap of CAL51 p53 CUT&Tag peaks with ChIP Atlas p53-bound genes72. A total of 3,524 peaks (54%) overlap; two-sided chi-squared test, p = 1 × 10−5. See Supplementary Table 16. b, CUT&Tag read coverage plot showing p53 binding at CDKN1A promoter in TP53-WT CAL51 cells. c, Overlap of CAL51 p53 CUT&Tag peaks with ChIP Atlas SREBP1-bound genes72. A total of 823 peaks (14%) overlap; two-sided chi-squared test, p = 1 × 10−5. See Supplementary Table 16. d, SREBF1 mRNA by qPCR: Higher SREBF1 mRNA levels in TP53-null vs. TP53-WT CAL51 cells (SFM, two-sided t-test, ***p = 2 × 10−4. e, TP53 mRNA by qPCR: siRNA-mediated TP53 knockdown resulted in >90% reduction in TP53 mRNA levels in MCF10A and MCF7 cells (paired one-sided t-test, MCF10A ***p = 9.9 × 10−4, MCF7 ***p = 1.2 × 10−4, n = 3). f, CDKN1A mRNA by qPCR: elevated CDKN1A mRNA levels following exposure to nutlin-3 in MCF10A and MCF7 cells (paired one-sided t-test, MCF10A *p = 0.02, MCF7 *p = 0.03, n = 3). g, Cdkn1a mRNA qPCR: elevated Cdkn1a mRNA levels following Trp53 overexpression in 4T1 mouse BC cells (paired one-sided t-test, ***p = 6 × 10−4, n = 5). h, DEPDC1 mRNA (TCGA BC cohort): highest DEPDC1 expression in primary BCs with biallelic TP53 inactivation, intermediate expression in tumors with monoallelic TP53 inactivation, and lowest expression in TP53-WT tumors. DEPDC1-high: n = 90, DEPDC1-low n = 90. j, Transcriptional signatures associated with DEPDC1 by GSEA. DEPDC1-high tumors were enriched for adipogenesis: ‘Burton_Adipogenesis-3' (red), rank #13/5844; ****p = 5.8 × 10−88). See Supplementary Table 17. k, DEPDC1 mRNA by qPCR: siRNA-mediated DEPDC1 knockdown reduced its mRNA by >90% in MDA-MB-231 cells (paired one-sided t-test, siDEPDC1(I) ****p = 2.5 × 10−7, siDEPDC1(II) ****p = 8.2 × 10−7, n = 4). l,m, DEPDC1 protein by western blot: siRNA-mediated DEPDC1 knockdown resulted in a reduction in SCD1 protein levels in four human BC cell lines. l, Representative gel. n, DEPDC1 Chromatin IP (ChIP) in MCF10A cells: enrichment for DEPDC1 binding at the proximal promoter of SCD1. ChIP-fold enrichment over IgG, region C (paired one-sided t-test *p = 0.049 n = 3). o,p, mRNA of SREBP1 targets. o, Higher expression of SREBP1 targets in DEPDC1-high (top 10%) vs. DEPDC1-low (bottom 10%) primary BCs (TCGA). DEPDC1-high n = 90, -low n = 90. p, Highest expression of SREBP1 targets in primary BCs with biallelic TP53 inactivation, and intermediate expression in tumors with monoallelic TP53 alterations compared to TP53-WT BCs. q, Reduced bone-metastasis signatures in DEPDC1-high tumors (top 10%, TCGA). DEPDC1-high n = 90, DEPDC1-low n = 90. r, Increased brain metastasis signatures in DEPDC1-high tumors (top 25%). DEPDC1-high n = 223, DEPDC1-low n = 223. s, Srebf1 mRNA by qPCR: Higher Srebf1 mRNA levels in ACM-exposed Trp53-null vs. Trp53-WT EMT6 cells. Paired one-sided t-test *p = 0.02. TP53-WT: n = 6, TP53-null: n = 6. t, DEPDC1 mRNA by qPCR: Higher DEPDC1 mRNA levels in ACM-exposed TP53-null vs. TP53-WT CAL51 cells. Paired one-sided t-test *p = 0.01. TP53-WT: n = 11, TP53-null: n = 11. u,v, DEPDC1 protein by western blot. v, Elevated DEPDC1 protein levels in TP53-null CAL51 cells cultured in SFM, with further increase following ACM exposure. Paired one-sided t-test, TP53-null ACM/SFM vs. TP53-WT ACM/SFM *p = 0.04. n = 4. Boxplots: bar = median; box = 25–75th percentile; whiskers = range. a, Correlation between p53 pathway activity (ssGSEA score for MSigDB signature ‘Kannan_p53_targets_dn') and sensitivity of human BC cell lines to CRISPR/Cas9 knockout of SCD1 (negative values represent increased sensitivity). Two-sided Spearman's rho = 0.29, p = 0.048. BC cell lines: n = 34. b, Comparison of drug sensitivity: Trp53-null vs. Trp53-WT EMT6 cells treated with the SCD1 inhibitor SW203668 (72 h, MTT viability readout). Paired one-sided t-test, *p = 0.02. Trp53-WT: n = 6, Trp53-null: n = 6. c, Relative viability of Trp53-WT and Trp53-null EMT6 cells treated with SCD1 inhibitor A939572 (75 nM, 72 h). Trp53-null cells were more sensitive to the drug. Trp53-WT: n = 3, Trp53-null: n = 3. d, Representative images of mCherry-labeled TP53-WT and TP53-null CAL51 cells following treatment with SW203668 (100 nM, 48 h) or vehicle control. TP53-null cells showed increased sensitivity to the drug. e, Proliferation curves of TP53-WT and TP53-null human CAL51 BC cells following treatment with SW203668 (100 nM, 72 h) or vehicle control. TP53-null cells were more sensitive to the drug. One-sided t-test: 48 h, *p = 0.03; 60 h *p = 0.02; 68 h **p = 0.01; 15 fields per experiment. f,g, Relative viability of TP53-WT and TP53-null CAL51 cells treated with the SCD1 inhibitors SW203668 (100 nM, 72 h; f) or A939572 (75 nM, 72 h; g). TP53-null cells were more sensitive to the drug. TP53-WT: n = 3, TP53-null: n = 3. h, Proliferation curves of Trp53-WT and Trp53-null EMT6 cells treated with SW203668 (1 µM) with or without POA (20 µM). i, Sensitivity of TP53-WT and TP53-null BC lines to RNAi-mediated knockdown of FASN. TP53-null cells are more sensitive to FASN knockdown. BC cell lines: TP53-WT: n = 4, TP53-null: n = 14. j, Comparison of drug sensitivity between TP53-WT and TP53-null BC lines treated with the FASN inhibitor C75 (AUC values, GDSC102). TP53-null cells were more sensitive to the drug. BC cell lines, TP53-WT: n = 3, TP53-null: n = 2. k,l, Representative images (k) and proliferation curves (l) of mCherry-labeled Trp53-WT and Trp53-null EMT6 cells treated with C75 (20 µM, 28 h). Trp53-null cells were more sensitive to the drug. One-sided t-test **p = 0.01 at 24 h, **p = 0.005 at 28 h. Error bars, mean ± s.e.m. m, Comparison of drug sensitivity between TP53-WT and TP53-null EMT6 cells treated with C75 (72 h, MTT viability readout). Trp53-null cells were more sensitive to the drug. Paired one-sided t-test, *p = 0.02, n = 6. n, Proliferation curves of Trp53-WT and Trp53-null EMT6 cells cultured as 3D spheroids and treated with SW203668 (0.1 µM or 3 µM) or vehicle control. Trp53-null spheroids were more sensitive to the drug. o, Quantification of spheroid area of Trp53-WT and Trp53-null EMT6 cells treated with C75 (20 µM, 48 h) or vehicle control. Trp53-null spheroids were smaller following drug treatment. Trp53-WT: n = 3, Trp53-null: n = 3. p, Fraction of apoptotic cells, measured by staining for cleaved-caspase-3, in human BCBM organotypic cultures treated with the SCD1i SW203668 (3 µM) or vehicle control. Treated organotypic cultures showed a higher fraction of apoptotic cells. Paired one-sided t-test, *p = 0.02, n = 3 BCBM samples. Fraction of Ki67+ cells in human BM slices treated with the SCD1i SW203668 (3 µM) or vehicle control. Trend towards reduced proliferation was observed (17% decrease, paired one-sided t-test, p = 0.13, n = 5). Boxplots: bar = median; box = 25–75th percentile; whiskers = range. 1–16, Supplementary Methods and supporting data for Supplementary Figs. Supporting data for Supplementary Figs. Source data for Figs. 1a–e,i,m,n, 2a,c,g–i,k,n,p, 3b,d,g,i,l,n,p,r, 4a–f,h,i,l,n–p,r,t–v,x,z, 5a–d,f,g,i,k,l,n–r,t,u,w, 6a,d–h,j,k,m–s,u and 7a,b,d,e,g,j,k,n, and Extended Data Figs. 1a–c,e,g,i–k, 2e–g, 3a,b,d,f–i,l,m,o,p, 4c–e,h–j, 5a–f,h,i,g, 6a–h,j,l–n, 7a–k,m,o, 8a–d,f,g,i,k–p,r, 9d–k,m–t and 10a–c,e–j,l–q. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Laue, K., Pozzi, S., Zerbib, J. et al. p53 inactivation drives breast cancer metastasis to the brain through SCD1 upregulation and increased fatty acid metabolism. Version of record: 29 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-04104-2'>Why academics should do more consulting — and how to make it work</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 10:18:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Emma Walker is a senior managing consultant at Wellspring, London, UK. The number of intellectual property licences has increased and more spin-out companies are being formed. By contrast, academic consulting work, one of the most direct and scalable means by which academics can shape industry, government and civil society, remains underdeveloped (see ‘Consulting gap'). Consulting work and other routes for exchanging knowledge are often treated as administrative functions. At a time when universities are struggling financially and academic research is often seen as distant from people's everyday concerns, that must change. If the work is properly supported, it can diversify income, build enduring partnerships and ensure that research delivers a tangible impact. For example, one small software team at the University of Southampton, UK, turned open-source verification tools into more than £600,000 (US$799,000) of consulting income by advising on safety-critical railway systems between 2014 and 2021 (see go.nature.com/44ueoch). For academics, consulting also offers professional development, wider networks, financial reward and real-world experience. It creates career porosity between academia, industry and policy, opening routes into leadership roles. For example, consulting can underpin the spin-out-company model: an academic who created intellectual property might be engaged as a ‘fractional executive' through a consulting contract, allowing them to contribute to the company and develop leadership skills. Consulting work also enriches the research culture of higher-education institutions, giving academics direct feedback on how their work is used in practice, informing grant proposals and strengthening career progression. For society, consulting accelerates the application of knowledge to national priorities, such as clean energy, health and social policy. It provides a mechanism for organizations to benefit from publicly funded research. It also ensures university expertise is applied beyond the campus walls, and can feed back into teaching with real-world case studies that better prepare graduates for the workforce. In one example, industry-led consulting projects were developed at Henley Business School, UK, to give master's students hands-on experience. Consulting, therefore, is one of the most immediate channels through which universities can deliver societal impact. Using data from the UK Higher Education Statistics Agency on the number of consultancy contracts and academic staff at nine universities, we found that less than 10% of academic staff, on average, engaged in consulting work. Higher average values of contracts have driven some growth, with the linear trend in total academic consultancy income rising by 31% from 2014–15 to 2023–24. Medical parasitology consultant Wellington Oyibo reports on anti-malaria strategies.Credit: Adekunle Ajayi/NurPhoto via Getty Data are less available for institutions in other countries; limited information (including personal communications with US technology-transfer offices) suggests that consulting income typically does not make up a large share of institutional turnover. This suggests a global underutilization of academic expertise through consulting, even among the world's most research-intensive universities — and a lost revenue stream for higher education. Private consultancy firms are increasingly moving into this underexploited market, capturing opportunities that universities neglect. Small-scale (under £5,000), fast-turnaround, one-off projects are commonly sidelined by university contract offices because they represent too small an income for strained institutional resources, creating space for contract-research organizations and technical consultancies with the agility to capture this demand. If universities do not act, global professional services firms will continue to dominate the space in which academic insight could deliver direct societal benefit. Weak institutional policies on consulting are one reason for this missed opportunity. Our analysis reflects practices across a subset of research-intensive institutions, including our own, and is intended to inform sector-wide discussion rather than prescribe a single model. When we conducted interviews with staff who manage academic consulting services across multiple UK universities, administrative delays and inconsistent incentives or recognition of consulting were often cited as common issues. The financial outcomes for academics vary widely, from flexible fees agreed between client and academic to university-set day rates. Some academics choose to put their consulting income into research accounts at their institution to support their group or department, but whether this is permitted, and on what terms, varies between universities and even across departments in the same institution. At some organizations, up to 50% of an individual's earnings can be diverted into institutional research funds. Contract approvals range from a 24-hour turnaround to months of negotiations. Faced with these obstacles, some academics turn to private consulting conducted outside their employment contracts, often without legal cover and sometimes in breach of institutional terms. Independent Review of University Spin-out Companies (Department for Science, Innovation and Technology & HM Treasury, 2023). Universities under fire must harness more of the financial value they create How I'm using AI tools to help universities maximize research impacts Point of no returns: researchers are crossing a threshold in the fight for funding AI and quantum science take centre stage under Trump — but with little new proposed funding Point of no returns: researchers are crossing a threshold in the fight for funding Grant cuts, arrests, lay-offs: Trump made 2025 a tumultuous year for science US–Africa bilateral health deals won't help against diseases that ignore borders Flu's link to cardiovascular disease shows why vaccination is essential A universal flu vaccine has proved challenging — could it finally be possible? Nankai University welcomes global outstanding talents to join for common development. IOP is China's premier research institution in condensed matter physics and related fields. SLS invites applications for multiple tenure-track/tenured faculty positions at all academic ranks. Universities under fire must harness more of the financial value they create How I'm using AI tools to help universities maximize research impacts An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67076-x'>Biomimetic model of corticostriatal micro-assemblies discovers a neural code</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 02:06:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications , Article number:  (2025) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Although computational models have deepened our understanding of neuroscience, it is still highly challenging to link actual low-level physiological activity (spiking, field potentials) and biochemistry (transmitters and receptors) directly with high-level cognitive abilities (decision-making, working memory) and associated disorders. Here, we introduce a mechanistically accurate multi-scale model directly generating simulated physiology from which extended neural and cognitive phenomena emerge. The model produces spiking, fields, phase synchronies, and synaptic change, directly generating working memory, decisions, and categorization. These were then validated on extensive experimental macaque data from which the model received no prior training of any kind. Moreover, the simulation uncovered a previously unknown neural code (“incongruent neurons”) that specifically predicts upcoming erroneous behaviors, also subsequently confirmed in empirical data. The biomimetic model thus directly and predictively links decision and reinforcement signals, of computational interest, with spiking and field codes, of neurobiological importance. Electrophysiological recordings from macaque brain under category learning paradigm that was used for validation in this paper can be accessed at : https://github.com/Neuroblox/Neuroblox.jl/blob/main/RESOURCES.md. This includes preprocessed LFP signals and spikes from PFC and Striatum. Source data for the figures are provided with this paper. Source data are provided with this paper. The model shown here is now a part of Neuroblox computational neuroscience platform (https://www.neuroblox.ai/), which includes an easy-to-use GUI as well as documentation and tutorials. A standalone Julia implementation of the model can be found at https://github.com/Neuroblox/Neuroblox.jl/blob/main/RESOURCES.md. Shepherd, G. M. G. Corticostriatal connectivity and its role in disease. Google Scholar Bekhbat, M. et al. Functional connectivity in reward circuitry and symptoms of anhedonia as therapeutic targets in depression with high inflammation: evidence from a dopamine challenge study. Google Scholar Rappaport, B. I., Kandala, S., Luby, J. L. & Barch, D. M. Brain reward system dysfunction in adolescence: current, cumulative, and developmental periods of depression. Google Scholar Roltsch Hellard, E. et al. Optogenetic control of alcohol-seeking behavior via the dorsomedial striatal circuit. Google Scholar Bariselli, S. & Lovinger, D. M. Corticostriatal circuit models of cognitive impairments induced by fetal exposure to alcohol. Google Scholar Sabaroedin, K., Tiego, J. & Fornito, A. Circuit-based approaches to understanding corticostriatothalamic dysfunction across the psychosis continuum. Google Scholar Izhikevich, E. M. & Edelman, G. M. Large-scale model of mammalian thalamocortical systems. Natl Acad. Google Scholar Jirsa, V. et al. Personalised virtual brain models in epilepsy. Google Scholar Leblois, A., Boraud, T., Meissner, W., Bergman, H. & Hansel, D. Competition between feedback loops underlies normal and pathological dynamics in the basal ganglia. Google Scholar Gurney, K., Prescott, T. J. & Redgrave, P. A computational model of action selection in the basal ganglia. A new functional anatomy. Google Scholar Gurney, K., Prescott, T. J. & Redgrave, P. A computational model of action selection in the basal ganglia. Analysis and simulation of behaviour. Google Scholar Frank, M. J. Dynamic dopamine modulation in the basal ganglia: a neurocomputational account of cognitive deficits in medicated and nonmedicated Parkinsonism. Google Scholar Frank, M. J. Hold your horses: a dynamic computational role for the subthalamic nucleus in decision making. Neural Netw. Google Scholar DiCarlo, J. J. & Cox, D. D. Untangling invariant object recognition. Trends Cogn. Google Scholar DiCarlo, J. J., Zoccolan, D. & Rust, N. C. How does the brain solve visual object recognition? Google Scholar Humphries, M. D., Stewart, R. D. & Gurney, K. N. A physiologically plausible model of action selection and oscillatory activity in the basal ganglia. Google Scholar Ashby, F. G., Ennis, J. M. & Spiering, B. J. A neurobiological theory of automaticity in perceptual categorization. Google Scholar Ashby, F. G. & Crossley, M. J. A computational model of how cholinergic interneurons protect striatal-dependent learning. Google Scholar Goldman, M. S., Compte, A. & Wang, X. J. Neural Integrator Models. In Larry R. Squire, editor, Encyclopedia of Neuroscience, pages 165–178. (Academic Press, 2009). Wang, Xiao-Jing Theory of the multiregional neocortex: large-scale neural dynamics and distributed cognition. Google Scholar Wang, Xiao-Jing Decision making in recurrent neuronal circuits. Google Scholar Hill, S. & Tononi, G. Modeling sleep and wakefulness in the thalamocortical system. Google Scholar Wagatsuma, N., Potjans, T. C.Diesmann, M. & Fukai, T. Layer-Dependent Attentional Processing by Top-down Signals in a Visual Cortical Microcircuit Model. Frontiers in Computational Neuroscience, 5, July 2011. O'Reilly, R. C. & Munakata, Y.Computational Explorations in Cognitive Neuroscience: Understanding the Mind by Simulating the Brain. The MIT Press, August 2000. O'Reilly, R. C. & Frank, M. J. Making working memory work: a computational model of learning in the prefrontal cortex and basal ganglia. Neural Comput. Google Scholar Eliasmith, C. et al. A large-scale model of the functioning brain. Google Scholar Antzoulatos, E. G. & Miller, E. K. Differences between neural activity in prefrontal cortex and striatum during learning of novel abstract categories. Google Scholar Antzoulatos, E. G. & Miller, E. K. Increases in functional connectivity between prefrontal cortex and striatum during category learning. Google Scholar Coultrip, R., Granger, R. & Lynch, G. A cortical model of winner-take-all competition via lateral inhibition. Neural Netw. Google Scholar Mao, Zhi-Hong & Massaquoi, S. G. Dynamics of winner-take-all competition in recurrent neural networks with lateral inhibition. IEEE Trans. Neural Netw. Google Scholar Kaski, S. & Kohonen, T. Winner-take-all networks for physiological models of competitive learning. Neural Netw. Google Scholar A Mathematical Approach to Neural Systems. In Jacqueline Metzler, editor, Systems Neuroscience, pages 67–117. Academic Press, January 1977. & Ermentrout, G. B. Spatially structured activity in synaptically coupled neuronal networks: II. Lateral inhibition and standing pulses. SIAM J. Appl. Google Scholar Oja, E. Neural networks, principal components, and subspaces. J. Neural Syst. Google Scholar Grossberg, S. Competitive learning: from interactive activation to adaptive resonance. Google Scholar von der Malsburg, Chr Self-organization of orientation sensitive cells in the striate cortex. Google Scholar Felch, A. C. & Granger, R. H. The hypergeometric connectivity hypothesis: divergent performance of brain circuits with different synaptic connectivity distributions. Google Scholar Abeles, M.Corticonics: Neural Circuits of the Cerebral Cortex. Fitzpatrick, D., Lund, J. S., Schmechel, D. E. & Towles, A. C. Distribution of GABAergic neurons and axon terminals in the macaque striate cortex. Google Scholar Hendry, S. H., Schwark, H. D., Jones, E. G. & Yan, J. Numbers and proportions of GABA-immunoreactive neurons in different areas of monkey cerebral cortex. Google Scholar Douglas, R. J. & Martin, K. A. C. Neuronal circuits of the neocortex. Google Scholar Pfeffer, C. K., Xue, M., He, M., Huang, Z. J. & Scanziani, M. Inhibition of inhibition in visual cortex: the logic of connections between molecularly distinct interneurons. Google Scholar Campagnola, L. et al. Local connectivity and synaptic dynamics in mouse and human neocortex. Google Scholar Rudy, B., Fishell, G., Lee, SooHyun & Hjerling-Leffler, J. Three groups of interneurons account for nearly 100% of neocortical GABAergic neurons. Developmental Neurobiol. Google Scholar Tremblay, R., Lee, S. & Rudy, B. GABAergic interneurons in the neocortex: from cellular properties to circuits. Google Scholar Coombes, S. & Byrne, A. Next Generation Neural Mass Models. In Fernando Corinto and Alessandro Torcini, editors, Nonlinear Dynamics in Computational Neuroscience, pages 1–16. Springer International Publishing, Cham, 2019. Byrne, A., O'Dea, R. D., Forrester, M., Ross, J. & Coombes, S. Next-generation neural mass and field modeling. B., Guzman, J. N., Peterson, J. D., Goldberg, J. & Surmeier, D. J. Thalamic gating of corticostriatal signaling by cholinergic interneurons. Carandini, M., Mechler, F., Leonard, C. S. & Movshon, J. A. Spike train encoding by regular-spiking cells of the visual cortex. Cash, S. & Yuste, R. Input summation by cultured pyramidal neurons is linear and position-independent. Cash, S. & Yuste, R. Linear summation of excitatory inputs by CA1 pyramidal neurons. Tamás, G. ábor, Szabadics, J. ános & Somogyi, P. Cell type- and subcellular position-dependent summation of unitary postsynaptic potentials in neocortical neurons. Gasparini, S. & Magee, J. C. State-dependent dendritic computation in hippocampal CA1 pyramidal neurons. Silver, R. A. Neuronal arithmetic. Koch, C. Biophysics of Computation: Information Processing in Single Neurons. (Oxford University Press, 2004). Bernander, O., Koch, C. & Douglas, R. J. Amplification and linearization of distal synaptic input to cortical pyramidal cells. Granger, R. Engines of the brain: the computational instruction set of human cognition. AI Mag. Bhattacharya, S., Brincat, S. L., Lundqvist, M. & Miller, E. K. Traveling waves in the prefrontal cortex during working memory. PLOS Computational Biol. Davis, Z. W. et al. Spontaneous traveling waves naturally emerge from horizontal fiber time delays and travel through locally asynchronous-irregular states. Stephen, E. P. et al. Broadband slow-wave modulation in posterior and anterior cortex tracks distinct states of propofol-induced unconsciousness. Lubenov, E. V. & Siapas, A. G. Hippocampal theta oscillations are travelling waves. Aisa, B., Mingus, B. & O'Reilly, R. The emergent neural modeling system. Chaudhuri, R., Bernacchia, A. A diversity of localized timescales in network activity, January 2014. https://elifesciences.org/articles/01239/figures. Hasson, U., Yang, E., Vallines, I., Heeger, D. J. & Rubin, N. A hierarchy of temporal receptive windows in human cortex. Dayan, P. & Abbott, L. F. Theoretical Neuroscience: Computational and Mathematical Modeling of Neural Systems. Gerstner, W., Kistler, W. M., Naud, R. & Paninski, L. Neuronal Dynamics: From Single Neurons to Networks and Models of Cognition. (Cambridge University Press, 2014). & Poo, M.-m. Synaptic modifications in cultured hippocampal neurons: dependence on spike timing, synaptic strength, and postsynaptic cell type. Song, S., Miller, K. D. & Abbott, L. F. Competitive Hebbian learning through spike-timing-dependent synaptic plasticity. LeCun, Y., Bengio, Y. & Hinton, G. Deep learning. Elman, J. L. Finding structure in time. Knill, D. C. & Pouget, A. The Bayesian brain: the role of uncertainty in neural coding and computation. Trends Neurosci. Sutton, R. S. & Barto, A. G. Reinforcement Learning: An Introduction. Hopfield, J. J. Neural networks and physical systems with emergent collective computational abilities. & Nakahara, H. Population coding and decoding in a neural field: a computational study. Neural Comput. Rumelhart, D. E., Hinton, G. E. & Williams, R. J. Learning representations by back-propagating errors. Milo, R. et al. Network motifs: simple building blocks of complex networks. & Kötter, R. Motifs in brain networks. PLOS Biol. Georgopoulos, A. P., Schwartz, A. & Kettner, R. E. Neuronal population coding of movement direction. Schurger, A., Sitt, J. D. & Dehaene, S. An accumulator model for spontaneous neural activity prior to self-initiated movement. Libet, B., Gleason, C. A., Wright, E. W. & Pearl, D. K. Time of conscious intention to act in relation to onset of cerebral activity (readiness-potential): the unconscious initiation of a freely voluntary act. Soon, C. S., Brass, M., Heinze, H.-J. & Haynes, J.-D. Unconscious determinants of free decisions in the human brain. Haynes, J.-D. et al. Reading hidden intentions in the human brain. Jeong, H. et al. Mesolimbic dopamine release conveys causal associations. Jiang, X. et al. Principles of connectivity among morphologically defined cell types in adult neocortex. Adesnik, H. & Scanziani, M. Lateral competition for cortical space by layer-specific horizontal circuits. Naka, A. et al. Complementary networks of cortical somatostatin interneurons enforce layer specific control. Atallah, B. V., Bruns, W., Carandini, M. & Scanziani, M. Parvalbumin-expressing interneurons linearly transform cortical responses to visual stimuli. Kim, D. et al. Distinct roles of parvalbumin- and somatostatin-expressing interneurons in working memory. Hodgkin, A. L. & Huxley, A. F. A quantitative description of membrane current and its application to conduction and excitation in nerve. Wicks, S. R., Roehrig, C. J. & Rankin, C. H. A dynamic network simulation of the nematode tap withdrawal circuit: predictions concerning synaptic function using behavioral criteria. Jones, B. E. Arousal systems. Feldmeyer, D., Lubke, J. & Sakmann, B. Efficacy and connectivity of intracolumnar pairs of layer 2/3 pyramidal cells in the barrel cortex of juvenile rats. Kawaguchi, Y., Karube, F. & Kubota, Y. Dendritic branch typing and spine expression patterns in cortical nonpyramidal cells. Lund, J. S., Yoshioka, T. & Levitt, J. Comparison of intrinsic connectivity in different areas of macaque monkey cerebral cortex. Braitenberg, V. and Schuz, A.Cortex: statistics and geometry of neuronal connectivity. Springer, Berlin, Heidelberg, 1998. Shimo, Y. Role of tonically active neurons in primate caudate in reward-oriented saccadic eye movement. & Papa, S. M. Striatal oscillations in parkinsonian non-human primates. Deffains, M. et al. Subthalamic, not striatal, activity correlates with basal ganglia downstream activity in normal and parkinsonian monkeys. Vazey, E. M., Moorman, D. E. & Aston-Jones, G. Phasic locus coeruleus activity regulates cortical encoding of salience information. Schneider, J. S. & Olazabal, U. E. Behaviorally specific limb use deficits following globus pallidus lesions in rats. Hauber, W., Lutz, S. & Münkle, M. The effects of globus pallidus lesions on dopamine-dependent motor behaviour in rats. Lai, Yuan-Yang, Kodama, T., Hsieh, K.-C., Nguyen, D. & Siegel, J. M. Substantia nigra pars reticulata-mediated sleep and motor activity regulation. Calabresi, P., Mercuri, N. B., De Murtas, M. & Bernardi, G. Endogenous GABA mediates presynaptic inhibition of spontaneous and evoked excitatory synaptic potentials in the rat neostriatum. Kreitzer, A. C. & Malenka, R. C. Striatal plasticity and basal ganglia circuit function. Reynolds, J. N. J. & Wickens, J. R. Dopamine-dependent plasticity of corticostriatal synapses. Intrator, N. & Cooper, L. N. Objective function formulation of the BCM theory of visual cortical plasticity: statistical connections, stability conditions. Bienenstock, E. L., Cooper, L. N. & Munro, P. W. Theory for the development of neuron selectivity: orientation specificity and binocular interaction in visual cortex. Larson, J., Xiao, P. & Lynch, G. Reversal of LTP by theta frequency stimulation. Larson, J. & Munkacsy, E. Theta-burst LTP. Granger, R., Whitson, J., Larson, J. & Lynch, G. Non-Hebbian properties of long-term potentiation enable high-capacity encoding of temporal sequences. Castro-Alamancos, M. A. & Connors, B. W. Cellular mechanisms of the augmenting response: short-term plasticity in a thalamocortical pathway. Castro-Alamancos, M. A. & Connors, B. W. Short-term plasticity of a thalamocortical pathway dynamically modulated by behavioral state. Mongillo, G., Barak, O. & Tsodyks, M. Synaptic theory of working memory. Kamiya, H. & Zucker, R. S. Residual Ca2+ and short-term synaptic plasticity. Ibata, K., Sun, Q. & Turrigiano, G. G. Rapid synaptic scaling induced by changes in postsynaptic firing. Bouchacourt, F. & Buschman, T. J. A flexible model of working memory. Carnevale, N. T. & Hines, M. L. The NEURON Book. Cambridge University Press, 1 edition, January 2006. Goodman, D. Brian: a simulator for spiking neural networks in Python. Gewaltig, M.-O., Morrison, A. & Plesser, H. E. NEST by Example: An Introduction to the Neural Simulation Tool NEST. In N. Le Novere, editor, Computational Systems Neurobiology, pages 533–558. Sanz-Leon, P. et al. The virtual brain: a simulator of primate brain network dynamics. Mazzoni, A. et al. Computing the local field potential (LFP) from integrate-and-fire network models. PLOS Comput. We thank Alex Driussi for his valuable support with graphics. We thank Anthony Chesebro for useful discussions. The research presented here was funded by the Baszucki Brain Research Fund, United States (LRMP). This work was supported in part by the Office of Naval Research, United States : N00014-21-1-2290 (RG). The experiments were funded by Office of Naval Research, United States : MURI N00014-23-1-2768 and MURI W911NF2410228, National Institute of Mental Health : 1R01MH131715-01, Freedom Together Foundation, and The Picower Institute for Learning and Memory (EKM). of Psychological and Brain Sciences, Dartmouth College, Hanover, NH, USA Anand Pathak & Richard Granger The Picower Institute for Learning and Memory and Department of Brain and Cognitive Sciences, Massachusetts Institute of Technology, Cambridge, MA, USA Scott L. Brincat, Evan G. Antzoulatos & Earl K. Miller of Biomedical Engineering, Laufer Center for Physical and Quantitative Biology, State University of New York at Stony Brook, Stony Brook, NY, USA Haris Organtzidis, Helmut H. Strey, Sageanne Senneff & Lilianne R. Mujica-Parodi Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Santa Fe Institute, Santa Fe, NM, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Data curation: S.L.B., E.G.A., and E.K.M. Formal analysis: A.P. Funding acquisition: H.H.S., L.R.M.P., E.K.M., and R.G. Project administration: L.R.M.P. Software: A.P., H.O., S.S., and H.H.S. Visualization: A.P., L.R.M.P., and R.G. Writing - original draft: A.P. Writing - review & editing: A.P., S.L.B., H.H.S., L.R.M.P., E.K.M., and R.G. Correspondence to Richard Granger. The authors declare the following competing interests: authors RG, EKM, LRMP, and HHS are co-Founders of Neuroblox Inc., a company spun out of SUNYSB, MIT, and Dartmouth to develop a commercial-grade software platform for multi-scale computational neuroscience with applications to diagnosis and treatment of brain-based disorders. The remaining authors declare no competing interests. Nature Communications thanks Garrison Cottrell, Tomoki Fukai, Jung Lee and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Pathak, A., Brincat, S.L., Organtzidis, H. et al. Biomimetic model of corticostriatal micro-assemblies discovers a neural code. Nat Commun  (2025). Received: 31 July 2024 Accepted: 19 November 2025 Published: 29 December 2025 DOI: https://doi.org/10.1038/s41467-025-67076-x Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Commun) ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67919-7'>Ultrafast dynamics of vibronically dressed core excitons in graphite: a femtosecond RIXS perspective</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-29 02:06:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications , Article number:  (2025) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Time-resolved resonant inelastic X-ray scattering (tr-RIXS) is a powerful technique for probing quasiparticle interactions in quantum materials under nonequilibrium conditions. Here, we implement tr-RIXS at the carbon K-edge to investigate the ultrafast dynamics of core excitons coupled to vibrational modes in graphite. Using femtosecond X-ray pulses from a free-electron laser, we monitor the temporal evolution of vibronically dressed excitons and their interaction with symmetry-selective optical phonons. By tuning the incident photon energy across the 1s → σ* resonance and analyzing the integrated inelastic sideband intensity, we reveal a detuning-controlled crossover between two complementary dynamical regimes. Phenomenological modeling and first-principles calculations reproduce both the magnitude and detuning dependence of the spectral-weight changes. In this work, enabled by the unique capabilities of X-ray free-electron lasers, we demonstrate how tr-RIXS can access coupled electronic and lattice dynamics with elemental and symmetry specificity, opening new routes to control vibronic interactions in light-element and low-dimensional quantum materials. The data that support the findings of this study, including raw and processed RIXS spectra, XAS profiles, and analysis outputs, are publicly available in the Elettra Sincrotrone Trieste S.C.p.A. repository under the title Dataset of Ultrafast dynamics of vibronically dressed core excitons in graphite at https://doi.org/10.34965/i53456. Source data for Figs. 1-4 and Supplementary Figs. are provided with the paper. All analysis scripts and fitting routines used in this study are available in the same repository at https://doi.org/10.34965/i53456under a CC-BY-SA-4.0 license. Yu, P. Y. & Cardona, M. Fundamentals of semiconductors, physics and materials properties. Texts Phys. Ahuja, R. et al. Theoretical and experimental study of the graphite 1s X-ray absorption edges. Google Scholar Zhang, L. et al. Electronic band structure of graphene from resonant soft X-ray spectroscopy: the role of core-hole effects. Google Scholar Wessely, O., Katsnelson, M. I. & Eriksson, O. Ab initio theory of dynamical core-hole screening in graphite from X-ray absorption spectra. Google Scholar Veenendaal, M. V. & Carra, P. Excitons and resonant inelastic X-ray scattering in graphite. Google Scholar Brühwiler, P. A. et al. π* and σ* excitons in C 1s absorption of graphite. Google Scholar Skytt, P. & Ma, Y. Angle-resolved soft-x-ray fluorescence and absorption study of graphite. Google Scholar Ma, Y. et al. Core excitons and vibronic coupling in diamond and graphite. Google Scholar Sette, F. et al. Lifetime and screening of the C 1s photoemission in graphite. Google Scholar Weng, X., Rez, P. & Ma, H. Carbon K-shell near-edge structure: multiple scattering and band-theory calculations. Google Scholar Mele, E. J. & Ritsko, J. J. Fermi-level lowering and the core exciton spectrum of intercalated graphite. Google Scholar Feng, X. et al. Disparate exciton-phonon couplings for zone-center and boundary phonons in solid-state graphite. Google Scholar Gilmore, K. Quantifying vibronic coupling with resonant inelastic X-ray scattering. Google Scholar Dashwood, C. D. et al. Probing electron-phonon interactions away from the Fermi level with resonant inelastic X-ray scattering. Google Scholar Geondzhian, A. & Gilmore, K. Generalization of the Franck-Condon model for phonon excitations by resonant inelastic x-ray scattering. Google Scholar Harada, Y. et al. Dynamical symmetry breaking under core excitation in graphite: polarization correlation in soft X-ray recombination emission. Google Scholar Attekum, P. M. T. M. V. & Wertheim, G. K. Excitonic effects in core-hole screening. Google Scholar Dean, M. P. M. et al. Ultrafast energy- and momentum-resolved dynamics of magnetic correlations in the photo-doped Mott insulator Sr2IrO4. Google Scholar Cao, Y. et al. Ultrafast dynamics of spin and orbital correlations in quantum materials: an energy- and momentum-resolved perspective. Google Scholar Mazzone, D. G. et al. Laser-induced transient magnons in Sr3Ir2O7 throughout the Brillouin zone. Natl Acad. Google Scholar Paris, E. et al. Probing the interplay between lattice dynamics and short-range magnetic correlations in CuGeO3 with femtosecond RIXS. npj Quantum Mater. Google Scholar Mitrano, M., Johnston, S., Kim, Y.-J. & Dean, M. P. M. Exploring quantum materials with resonant inelastic X-Ray scattering. Google Scholar Thielemann-Kühn, N. et al. Optical control of 4f orbital state in rare-earth metals. Google Scholar Xu, C. & Zong, A. Time-domain study of coupled collective excitations in quantum materials. npj Quantum Mater. Google Scholar Freibert, A., Mendive-Tapia, D., Huse, N. & Vendrell, O. Time-dependent resonant inelastic X-ray scattering of pyrazine at the nitrogen K-edge: a quantum dynamics approach. Theory Comput. Google Scholar Monney, C., Patthey, L., Razzoli, E. & Schmitt, T. Static and time-resolved resonant inelastic X-ray scattering: Recent results and future prospects. X Ray Spectrom. Google Scholar Ishioka, K. et al. Ultrafast electron-phonon decoupling in graphite. Google Scholar Pomarico, E. et al. Enhanced electron-phonon coupling in graphene with periodically distorted lattice. Google Scholar Erhardt, N. G. et al. Ultrafast Nonequilibrium Enhancement of Electron-Phonon Interaction in 2H-MoTe2. Pan, Y. et al. Momentum-resolved signatures of carrier screening effects on electron-phonon coupling in MoS2. ACS Nano 19, 11381–11389 (2025). Google Scholar Ament, L. J. P., Veenendaal, M. V., Devereaux, T. P., Hill, J. P. & Brink, J. V. D. Resonant inelastic x-ray scattering studies of elementary excitations. Google Scholar Carlisle, J. A. et al. Probing the graphite band structure with resonant soft-X-ray fluorescence. Google Scholar Gel'mukhanov, F., Sałek, P., Privalov, T. & Ågren, H. Duration of x-ray Raman scattering. Google Scholar Yavaş, H. et al. Observation of phonons with resonant inelastic x-ray scattering. Google Scholar Mainwood, A. & Stoneham, A. M. A comparison of the core exciton and nitrogen donor in diamond. Google Scholar Gel'mukhanov, F. & Ågren, H. Resonant inelastic x-ray scattering with symmetry-selective excitation. Google Scholar Geondzhian, A. & Gilmore, K. Demonstration of resonant inelastic x-ray scattering as a probe of exciton-phonon coupling. Google Scholar Malvestuto, M. et al. The MagneDyn beamline at the FERMI free electron laser. Google Scholar Xu, S. et al. Energy dependence of electron lifetime in graphite observed with femtosecond photoemission spectroscopy. Google Scholar Breusing, M. et al. Ultrafast nonequilibrium carrier dynamics in a single graphene layer. Google Scholar Spataru, C. D. et al. Anomalous quasiparticle lifetime in graphite: band structure effects. Google Scholar Piscanec, S., Lazzeri, M., Mauri, F., Ferrari, A. C. & Robertson, J. Kohn anomalies and electron-phonon interactions in graphite. Google Scholar Park, C.-H., Giustino, F., Spataru, C. D., Cohen, M. L. & Louie, S. G. First-principles study of electron linewidths in graphene. Google Scholar Park, C.-H., Giustino, F., Cohen, M. L. & Louie, S. G. Velocity renormalization and carrier lifetime in graphene from the electron-phonon interaction. Google Scholar Johannsen, J. C. et al. Direct view of hot carrier dynamics in graphene. Google Scholar Caruso, F., Novko, D. & Draxl, C. Photoemission signatures of nonequilibrium carrier dynamics from first principles. Google Scholar Novko, D. & Kralj, M. Phonon-assisted processes in the ultraviolet-transient optical response of graphene. npj 2D Mater. Google Scholar Duvel, M. et al. Far-from-equilibrium electron-phonon interactions in optically excited graphene. Nano Lett. Google Scholar Girotto, N. & Novko, D. Dynamical phonons following electron relaxation stages in photoexcited graphene. Google Scholar Sidiropoulos, T. P. H. et al. Probing the energy conversion pathways between light, carriers, and lattice in real time with attosecond core-level spectroscopy. Google Scholar Breusing, M., Ropers, C. & Elsaesser, T. Ultrafast carrier dynamics in graphite. Google Scholar Kampfrath, T., Perfetti, L., Schapper, F., Frischkorn, C. & Wolf, M. Strongly coupled optical phonons in the ultrafast dynamics of the electronic energy and current relaxation in graphite. Google Scholar Ritsko, J. J. Valence- and core-electronic excitations in potassium-intercalated graphite. Google Scholar Carlisle, J. A. et al. Crystal-momentum-resolved electronic structure of solids using resonant soft-X-ray fluorescence spectroscopy. J. Electron Spectrosc. Google Scholar Skytt, P. et al. Probing symmetry breaking upon core excitation with resonant x-ray fluorescence. Google Scholar Skytt, P. et al. Quenching of symmetry breaking in resonant inelastic X-ray scattering by detuned excitation. Google Scholar Ma, Y. et al. Soft-x-ray resonant inelastic scattering at the C K edge of diamond. Google Scholar Brühwiler, P. A., Kuiper, P., Eriksson, O., Ahuja, R. & Svensson, S. Core hole effects in resonant inelastic X-ray scattering of graphite. Google Scholar Margine, E. R. & Giustino, F. Two-gap superconductivity in heavily n-doped graphene: Ab initio Migdal-Eliashberg theory. Google Scholar Mazzola, F. et al. Strong electron-phonon coupling in the σ band of graphene. Google Scholar Gel'mukhanov, F., Privalov, T. & Agren, H. Collapse of vibrational structure in spectra of resonant x-ray Raman scattering. Google Scholar Yang, J.-A., Parham, S., Dessau, D. & Reznik, D. Novel electron-phonon relaxation pathway in graphite revealed by time-resolved Raman scattering and angle-resolved photoemission spectroscopy. Google Scholar Ulstrup, S. et al. Ultrafast electron dynamics in epitaxial graphene investigated with time- and angle-resolved photoemission spectroscopy. Google Scholar Gierz, I. et al. Snapshots of non-equilibrium Dirac carrier distributions in graphene. Google Scholar Na, M. X. et al. Direct determination of mode-projected electron-phonon coupling in the time domain. Google Scholar Harb, M. et al. Picosecond dynamics of laser-induced strain in graphite. Google Scholar Carbone, F., Baum, P., Rudolf, P. & Zewail, A. H. Structural preablation dynamics of graphite observed by ultrafast electron crystallography. Google Scholar Carbone, F. The interplay between structure and orbitals in the chemical bonding of graphite. Google Scholar Stern, M. J. et al. Mapping momentum-dependent electron-phonon coupling and nonequilibrium phonon dynamics with ultrafast electron diffuse scattering. Nordgren, J. & Guo, J. Instrumentation for soft X-ray emission spectroscopy. J. Electron Spectrosc. Danailov, M. B. et al. Towards jitter-free pump-probe measurements at seeded free electron laser facilities. Sigalotti, P. et al. Ultrafast laser synchronization at the FERMI@Elettra FEL. In Proc. SPIE - The International Society for Optical Engineering 8778 (SPIE, 2013). Giannozzi, P. et al. Advanced capabilities for materials modelling with quantum ESPRESSO. Baroni, S., Gironcoli, S. D., Corso, A. D. & Giannozzi, P. Phonons and related crystal properties from density-functional perturbation theory. Lee, H. et al. Electron-phonon physics from first principles using the EPW code. npj Comput. Marzari, N., Mostofi, A. A., Yates, J. R., Souza, I. & Vanderbilt, D. Maximally localized Wannier functions: theory and applications. Giustino, F. Electron-phonon interactions from first principles. Pagliara, S. et al. Photoinduced π − pi* band gap renormalization in graphite. Williams, P. F., Rousseau, D. L. & Dworetsky, S. H. Resonance fluorescence and resonance raman scattering: lifetimes in molecular iodine. Download references We acknowledge the assistance of the staff at FERMI during the beamtimes 20209081 and 20214052. thanks Dr. Carlo Alberto Brondin (ISM-CNR) for the preparation of graphite crystals. Elettra Sincrotrone Trieste S.C.p.A, Trieste, Italy Marco Malvestuto, Richa Bhardwaj, Antonio Caretta, Simone Laterza, Fulvio Parmigiani, Michele Manfredda, Alberto Simoncig, Marco Zangrando, Alexander Demidovich, Peter Susnjar, Enrico Massimiliano Allaria, Alexander Darius Brynes, David Garzella, Luca Giannessi, Primož Rebernik & Filippo Sottocorona CNR - Istituto Officina dei Materiali (IOM), Trieste, Italy Department of Physics, University of Trieste, Trieste, Italy Beatrice Volpato, Elena Babici & Fulvio Parmigiani Science and Technology Facilities Council (STFC), Daresbury Laboratory, Warrington, UK Alexander Darius Brynes Centre for Advanced Laser Techniques, Institute of Physics, Zagreb, Croatia Dino Novko Donostia International Physics Center (DIPC), Donostia-San Sebastian, Spain Dino Novko Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out the experiment, collected the data, and contributed to the preliminary analysis. participated in the data analysis. The theoretical investigation was conducted by D.N., M.M., and A.C. were responsible for the MagneDyn endstation. optimized the accelerator and provided the FEL beam. optimized the optical transport of the MagneDyn beamline and characterized the FEL pulses. optimized the optical laser system. contributed to the revision of the manuscript. wrote the manuscript, which all authors discussed. proposed and led the project. Correspondence to Marco Malvestuto. The authors declare no competing interests. Nature Communications thanks MengXing (Ketty) Na, and the other anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Malvestuto, M., Volpato, B., Babici, E. et al. Ultrafast dynamics of vibronically dressed core excitons in graphite: a femtosecond RIXS perspective. Nat Commun  (2025). Download citation Received: 24 March 2025 Accepted: 11 December 2025 Published: 29 December 2025 DOI: https://doi.org/10.1038/s41467-025-67919-7 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            