
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-31</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a69893444/colossal-squid-secret-face-discovered/'>For the First Time Ever, the Colossal Squid Might Have Shown Its Secret Face</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-31 14:08:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Humans spend nearly their entire lives on land, but the Earth we call home is really a water world. With 71 percent of the Earth's surface covered by water, this expansive ecosystem has been difficult to study, and many animals of the deep ocean remain a complete mystery. Believed to live in the Southern Ocean surrounding Antarctica, this immense cephalopod—the largest invertebrate species on the planet— has never been observed in its natural habitat. Scientists only get a good look at these animals when trawlers accidentally catch them in their nets. “Antarctica is experiencing rapid and complex change, and it is critical to have a better understanding of these changes for the region's ocean ecosystems,” the Kolossal team wrote in a paper detailing the method in 2024. Yet, the tourism industry in Antarctica is growing rapidly, and collaboration between tourism companies and researchers provides important access to the region.” While leveraging tourism for marine exploration is a logistical win-win, finding verified footage of a colossal squid is as hard as ever. The video is being peer-reviewed by experts, but it's unlikely that scientists will ever know for sure. But because footage of any squid species in the Southern Ocean is rare, the footage is a huge victory for marine biologists studying these famously elusive animals regardless of its contents. Although the team's stated goal is to capture footage of an adult colossal squid in its natural environment, according to IFLScience, the camera filmed nearly 80 species. Among them were giant volcano sponges, Antarctic sunflower stars, and many other marine invertebrates. But as marine biologists team up with tourism vessels to explore the oceans, some of the our water planet's biggest questions could slowly be answered. Gravity Proves We Live in a Simulation, Scientist Says Humans May Be Able to Grow New Teeth in 4 Years A New Alloy Is Defying the Limits of Metal Scientists Made Glue So Strong, It Can Tow A Car</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/nasa-telescopes-capture-colliding-spiral-galaxies-in-sparkling-detail/'>NASA Telescopes Capture Colliding Spiral Galaxies in Sparkling Detail</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-31 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Astronomers combined data from NASA's JWST and Chandra X-ray Observatory to create a stunning new image of two merging spiral galaxies Two space telescopes really are better than one. This month NASA released a new image that combines observations from the James Webb Space Telescope (JWST) and the Chandra X-ray Observatory of two spiral galaxies on a cosmic collision course. The pair are located some 120 million light-years from Earth. Together they are joined in a slow, gravitational dance that will, billions of years from now, end in their merger into a single galaxy. In the image, mid-infrared data from JWST shows dust and other cooler matter in white, gray and red, while x-ray data from Chandra shows high-energy areas, including regions of intense star formation, in blue. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. When galaxies merge, they can trigger explosive bursts of star formation, and astronomers are keen to observe these collisions to understand how galaxies evolve over time. Claire Cameron is the Breaking News Chief at Scientific American. Originally from Scotland, she moved to New York in 2012. Her work has appeared in National Geographic, Slate, Inc. Magazine, Nautilus, Semafor, and elsewhere. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42256-025-01170-z'>Harnessing the power of single-cell large language models with parameter-efficient fine-tuning using scPEFT</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-31 11:17:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Machine Intelligence (2025)Cite this article A preprint version of the article is available at bioRxiv. Single-cell large language models (scLLMs) capture essential biological insights from vast single-cell atlases but struggle in out-of-context applications, where zero-shot predictions can be unreliable. To address this, here we introduce a single-cell parameter-efficient fine-tuning (scPEFT) framework that integrates learnable, low-dimensional adapters into scLLMs. By freezing the backbone model and updating only the adapter parameters, scPEFT efficiently adapts to specific tasks using limited custom data. This approach mitigates catastrophic forgetting, reduces parameter tuning by over 96% and decreases GPU memory usage by more than half, thus substantially enhancing the accessibility of scLLMs for resource-constrained researchers. When validated across diverse datasets, scPEFT outperformed zero-shot models and traditional fine-tuning in disease-specific, cross-species and undercharacterized cell population tasks. Its attention-mechanism analysis identified COVID-related genes associated with specific cell states and uncovered unique blood cell subpopulations, demonstrating the capacity of scPEFT for condition-specific interpretations. These findings position scPEFT as an efficient solution for enhancing the utility of scLLMs in general single-cell analyses. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Prices may be subject to local taxes which are calculated during checkout All datasets used in this study are publicly available. The NSCLC, BMMC and CD34+, Mouse, Mouse-10x/smart-seq, and macaque datasets can be accessed under numbers GSE179994, GSE139369, GSE115746, GSE185862 and GSE142585 of the Gene Expression Omnibus (GEO), respectively. The MS dataset is available at https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-35. The COVID-19 dataset can be accessed via Figshare at https://doi.org/10.6084/m9.figshare.16922467.v1 (ref. The C. elegans dataset is available from Calico Research at https://c.elegans.aging.atlas.research.calicolabs.com/. The processed Lung atlas can be downloaded as a h5ad file from https://cellxgene.cziscience.com/collections/edb893ee-4066-4128-9aec-5eb2b03f8287. The Bivalent promoters, long-range transcription factor and NOTCH1 network datasets were preprocessed and published by Geneformer in its Hugging Face repository at https://huggingface.co/datasets/ctheodoris/Genecorpus-30M/tree/main/example_input_files/gene_classification. The Adamson, Norman, Replogle_k562 and Replogle_rpe1 datasets can be extracted and preprocessed using GEAR's PerData class load function by specifying the data_name argument as ‘norman', ‘adamson', ‘replogle_k562_essential' and ‘replogle_rpe1_essential', respectively. The PBMC 10K dataset can be extracted via scvi's CLI: scvi.data.pbmc_dataset(). The Prirhinal cortex and COVID-BATCH datasets are available via GitHub at https://github.com/bowang-lab/scGPT/tree/main/data. All processed datasets can be accessed via GitHub at https://github.com/coffee19850519/scPEFT and via Figshare at https://doi.org/10.6084/m9.figshare.30763886 (ref. The source code is freely available via GitHub at https://github.com/coffee19850519/scPEFT and via Zenodo at https://doi.org/10.5281/zenodo.17781912 (ref. 44) with the MIT licence. Paik, D. T. et al. Single-cell RNA sequencing in cardiovascular development, disease and medicine. The history and advances in cancer immunotherapy: understanding the characteristics of tumor-infiltrating immune cells and their therapeutic implications. Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis. Yang, F. et al. scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data. Transfer learning enables predictions in network biology. Cui, H. et al. scGPT: toward building a foundation model for single-cell multi-omics using generative AI. Large-scale foundation model on single-cell transcriptomics. Heimberg, G. et al. A cell atlas foundation model for scalable search of similar human cells. Yang, X. et al. GeneCompass: deciphering universal gene regulatory mechanisms with a knowledge-informed cross-species foundation model. Fischer, F. et al. scTab: scaling cross-tissue single-cell annotation models. Harnessing the deep learning power of foundation models in single-cell omics. Kedzierska, K. Z. et al. Zero-shot evaluation reveals limitations of single-cell foundation models. Deeper evaluation of a single-cell foundation model. Chen, H. et al. Quantized multi-task learning for context-specific representations of gene network dynamics. Preprint at bioRxiv https://doi.org/10.1101/2024.08.16.608180 (2024). Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Liu, B. et al. Temporal single-cell tracing reveals clonal revival and expansion of precursor exhausted T cells during anti-PD-1 therapy in lung cancer. Chang, H. et al. C/EBPδ drives interactions between human MAIT cells and endothelial cells that are important for extravasation. Differences in the molecular signatures of mucosal-associated invariant T cells and conventional T cells. Teng, X. et al. SIGIRR deficiency contributes to CD4 T cell abnormalities by facilitating the IL1/C/EBPβ/TNF-α signaling axis in rheumatoid arthritis. Yang, R. et al. Distinct epigenetic features of tumor-reactive CD8+ T cells in colorectal cancer patients revealed by genome-wide DNA methylation analysis. Hodge, R. D. et al. Conserved cell types with divergent features in human versus mouse cortex. Yao, Z. et al. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. Traag, V. A., Waltman, L. & Van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Deep generative modeling for single-cell transcriptomics. Hie, B. L. et al. Scanorama: integrating large and diverse single-cell transcriptomic datasets. Roohani, Y., Huang, K. & Leskovec, J. Predicting transcriptional outcomes of novel multigene perturbations with GEARS. Ahlmann-Eltze, C., Huber, W. & Anders, S. Deep-learning-based gene perturbation effect prediction does not yet outperform simple linear baselines. Hong, L. et al. Fast, sensitive detection of protein homologs using deep dense retrieval. Predicting cellular responses to perturbation across diverse contexts with State. Preprint at bioRxiv https://doi.org/10.1101/2025.06.26.661135 (2025). Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. IEEE/CVF International Conference on Computer Vision 4015–4026 (IEEE, 2023). Attention is all you need. Advances in Neural Information Processing Systems, Vol. 30 (eds Guyon, I. et al.) (Curran Associates, 2017). Kingma, D. P. & Ba, J. Adam: a method for stochastic optimization. CZI Cell Science Program et al. CZ CELLxGENE Discover: a single-cell data platform for scalable exploration, analysis and modeling of aggregated data. Franzén, O., Gan, L.-M. & Björkegren, J. L. M. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data. Li, X. L. & Liang, P. Prefix-tuning: optimizing continuous prompts for generation. Hu, E. J. et al. LoRA: Low-rank adaptation of large language models. In The 10th International Conference on Learning Representations (ICLR, 2022). Exploring and retrieving sequence and metadata for species across the tree of life with NCBI datasets. He, F. Processed datasets used in scPEFT study. He, F. Codebase for scPEFT: harnessing the power of single-cell large language models with parameter-efficient fine-tuning (v1.0.0). We thank M. Zhao for technical support. This work was funded by the National Institutes of Health (NIH; R35GM126985). is supported in part by the NIH (R01GM152585, P01CA278732, P01AI177687, U54AG075931 and R01DK138504) and by the Pelotonia Institute of Immuno-Oncology. Department of Electrical Engineering and Computer Science, Bond Life Sciences Center, University of Missouri, Columbia, MO, USA Fei He, Ruixin Fei, Yang Yu, Xinyu Zhang, Xianyu Wang, Mingyue Gao, Li Su, Yibo Chen, Jinpu Li, Baichuan Jin & Dong Xu Department of Biomedical Informatics, The Ohio State University, Columbus, OH, USA Jordan E. Krull, Hao Cheng, Yuzhou Chang, Anjun Ma & Qin Ma Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH, USA Jordan E. Krull, Hao Cheng, Yuzhou Chang, Anjun Ma & Qin Ma Institute for Data Science and Informatics, University of Missouri, Columbia, MO, USA Li Su, Yibo Chen, Jinpu Li & Dong Xu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the study. collected the datasets and designed the case studies. performed the experiments and contributed analytical tools. revised and improved the paper. Y. Chen, J.L., Y. Chang and A.M. analysed and visualized the experimental results. supervised the study. All authors approved the final version of the paper. Correspondence to Qin Ma or Dong Xu. The authors declare no competing interests. Nature Machine Intelligence thanks Qi Liu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The violin plots benchmark native, finetuned scGPT, and scPEFT using scGPT pre-trained weights from whole human data, pan-cancer data, and lung-specific data. Statistical significance between scPEFT and other models was assessed using a paired two-sided Student's t-test across the five-fold validation results (n = 5 independent validation tests). On each box, the central mark indicates the median, and the bottom and top edges of the box indicate the 25th and 75th percentiles. The whiskers extend to the most extreme data points without outliers, and the outliers are plotted individually as circles. a-d, Confusion matrices for native, finetuned scLLMs, and scPEFT with (a) scBERT, (b) Geneformer, (c) scFoundation and (d) scGPT as respective backbones. Each cell reflects the percentage of instances from the row-defined cell type that are predicted as the column-defined cell type. High values along the diagonal indicate accurate predictions, while off-diagonal values represent misclassifications. The results exhibit the superior performance of scPEFT, particularly in identifying rare cell types (proportion < 5%). Notably, CD4+ enriched proliferation T cells were accurately identified by native scLLMs but misclassified by finetuned models, demonstrating an instance of catastrophic forgetting. This issue was not observed by scPEFT and was marked by the red boxes. e, Bar plot depicting the proportion of each cell type in the dataset. a-d, Confusion matrices for native, finetuned scLLMs, and scPEFT with (a) scBERT, (b) Geneformer, (c) scFoundation and (d) scGPT as respective backbones. Notably, multiple cell types were accurately identified by native scLLMs but misclassified by their finetuned models, demonstrating clear instances of catastrophic forgetting. This issue was not observed by scPEFT and was marked by the red boxes. e, Bar plot depicting the proportion of each cell type in the dataset. a-d, Confusion matrices for native, finetuned scLLMs, and scPEFT with (a) scBERT, (b) Geneformer, (c) scFoundation and (d) scGPT as respective backbones. Notably, the excitatory neuron cells at cortex layer 2-3 were accurately identified by native scGPT but misclassified by its finetuned model, demonstrating an instance of catastrophic forgetting. This issue was not observed by scPEFT and was marked by the red boxes. e, Bar plot depicting the proportion of each cell type in the dataset. Histograms of differential attention scores were derived from (a) native, (b) finetuned scGPT, and (c) scPEFT models, respectively, in analysis of COVID-related cell-state-specific genes in Memory CD8+ T cells versus Naïve Memory CD8+ T cells. Histograms were generated for the top, middle, and last Transformer layers in these models. Red crosses mark the bins where genes of interest (CCL5, GZMK, and CST7) were located. The COVID-related cell-state-specific genes were analyzed in the comparisons of (a) Memory CD8+ T cells versus Naïve Memory CD8+ T cells, and (b) Effector Memory CD8+ T cells versus Memory CD8+ T cells, respectively. Volcano plots of Differential Gene Expression (DEG) analysis are colored by differential attention scores from native scGPT, finetuned scGPT, and scPEFT trained on the COVID dataset. The top attention genes of interest were highlighted with their names. Dot size reflects the genes' adjusted p-values from DEG. Statistical significance was determined using the two-sided Wilcoxon rank-sum test with Benjamini-Yekutieli false discovery rate control. n = 1950 and 973 cells were used for the test cases, respectively. n = 5 independent validation tests were conducted. On each box, the central mark indicates the median, and the bottom and top edges of the box indicate the 25th and 75th percentiles. The whiskers extend to the most extreme data points without outliers, and the outliers are plotted individually as circles. a-g, UMAP visualizations of corrected cell embeddings, color-coded by cell type and batch, for scPEFT (using Encoder adapter, Token adapter, LoRA, and Prefix adapter), finetuned scGPT, scVI, and Scanorama. h, Comparative table of batch correction performance across these methods. a-g, UMAP visualizations of corrected cell embeddings, color-coded by cell type and batch, for scPEFT (using Encoder adapter, Token adapter, LoRA, and Prefix adapter), finetuned scGPT, scVI, and Scanorama. h, Comparative table of batch correction performance across these methods. a-g, UMAP visualizations of corrected cell embeddings, color-coded by cell type and batch, for scPEFT (using Encoder adapter, Token adapter, LoRA, and Prefix adapter), finetuned scGPT, scVI, and Scanorama. h, Comparative table of batch correction performance across these methods. Supplementary Notes 1–13, Tables 1–7 and Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Harnessing the power of single-cell large language models with parameter-efficient fine-tuning using scPEFT. Nat Mach Intell  (2025). Version of record: 31 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Machine Intelligence (Nat Mach Intell) Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-04102-4'>Some of your cells are not genetically yours — what can they tell us about life and death?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-31 10:47:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). People carry populations of cells from their biological relatives. Hidden Guests: Migrating Cells and How the New Science of Microchimerism is Redefining Human Identity Lise Barnéoud, transl. Could baby's first bacteria take root before birth? Likewise, children carry cells that were transferred to them in utero from their mothers — some of which might even be from the child's maternal grandmothers, older siblings or twin. But they are also rare — much rarer, for example, than the trillions of microorganisms that reside in our guts, on our skin and in many other organs. We carry only one microchimeric cell for every 10,000 to 1 million of our own cells. In Hidden Guests, the author invites readers to learn about the pioneering “microchimerists” — scientists who discovered these fascinating shared cells. Microchimeric cells were, we learn, discovered through a series of accidental observations. In the late 1800s, pathologist Georg Schmorl described ‘giant cells' in the lungs of people who had died from eclampsia — a life-threatening inflammatory condition that can occur during pregnancy. For more than two decades, it was presumed that these microchimeric cells were a temporary feature of pregnancy. It wasn't until 1993 that geneticist Diana Bianchi found cells with Y chromosomes in women who had given birth to sons between one and 27 years earlier2. This finding overturned the dogma that children inherit genes from their parents and never the other way around — these transferred fetal cells move through the family tree, travelling ‘backwards in time' from children to their mothers. Bianchi and others would go on to show that these cells have remarkable regenerative properties — promoting wound healing in mothers by transforming into blood vessels or skin cells. Microchimerism also calls into question a central tenet of immunology: that the immune system works by classifying cells in a binary fashion, as ‘self' or ‘non-self'. Under this simplistic model, microchimeric cells should trigger an immune response and be rejected by the body — but they do not. At around the same time that Bianchi was making her groundbreaking discovery, rheumatologist Lee Nelson also found Y-chromosome-containing cells in people who had previously given birth to sons3. Nelson was studying autoimmune diseases, which disproportionately affect middle-aged women, and at the time were thought to be caused by hormone imbalances. Cells pass from mother to baby, and vice versa, across the placenta.Credit: Jeffrey Davis/Getty These immune cells won Nobel fame — can they solve autoimmune disease? Annual flu vaccines are far from ideal — this is why Spatiotemporal cellular map of the developing human reproductive tract Restoring youth to old immune cells: mRNA therapy turns back the clock Physicists disagree wildly on what quantum mechanics says about reality, Nature survey shows Physicists should revel in the diversity of ways to understand quantum mechanics Part of the DFG-funded Research Training Group ‘GRK 2989: Targeting Cellular Interfaces in Reperfused Acute Myocardial Infarction (TCI repAMI)' The Westlake Fellowship provides funding for global talent to study PhD programs in 9 areas of science, engineering, and technology. Seeking exceptional Senior/Junior PIs, Postdocs, and Core Specialists globally year-round Seeking scientists in pathobiology, immunology, vaccinology, epidemiology, drug discovery, focusing on microbial infections and inflammations. PIs & Postdocs for cancer, immunology, neuroscience, stem cell, cardiovascular and related fields These immune cells won Nobel fame — can they solve autoimmune disease? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            