
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-07</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250507125830.htm'>Childhood brain tumors develop early in highly specialized nerve cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 19:15:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Medulloblastomas enlarge rapidly, often grow into surrounding tissue and can also form metastases. A team of researchers at the Hopp Children's Cancer Center Heidelberg (KiTZ), the German Cancer Research Center (DKFZ) and Heidelberg University Hospital (UKHD) has now investigated exactly where and when the most aggressive subgroups of these tumors develop in so-called single cell analyses. "This technology allows us to obtain a high-resolution picture of the genetic make-up within a tumor, so that we can reconstruct its developmental history fairly accurately." Accordingly, there are "early" and "late" genetic changes within a tumor. Large rearrangements of entire chromosomes or chromosome arms take place in these cells, presumably initially by chance, as the present results show: The cells loose or gain certain chromosomes. "We assume that the early loss or gain of certain chromosomes is the first step in tumor development and that these occur many years before the clinical symptoms appear," says Lena Kutscher from KiTZ and DKFZ, who led the study together with Stefan Pfister, Director at KiTZ, Head of Department at DKFZ and paediatric oncologist at Heidelberg University Hospital (UKHD). Lena Kutscher explains what these results could mean for diagnosis and treatment: "If we succeed in developing sufficiently sensitive methods in the future to detect these early changes, for example as DNA fragments in the blood, this could form the basis for possible early detection in newborns and infants." Materials provided by German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ). Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250507125656.htm'>New vacuum-assisted extrusion technique improves performance of polymer prints</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 18:51:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists at Oak Ridge National Laboratory have developed a vacuum-assisted extrusion method that reduces internal porosity -- by up to 75% -- in large-scale 3D-printed polymer parts. Large-format additive manufacturing, or LFAM, enables the direct printing of meter-scale structures used in aerospace, automotive and defense tooling. But widespread adoption has been hindered by internal porosity, or voids, that weaken printed components. ORNL researchers tackled this challenge with a novel approach: integrating a vacuum hopper during the extrusion process to remove trapped gases and minimize void formation in fiber-reinforced materials. These materials are widely used in LFAM for their stiffness and low thermal expansion but often suffer from intra-bead porosity that limits part quality. The new system reduced porosity to under 2%, even with varying fiber content. "Using this innovative technique, we are not only addressing the critical issue of porosity in large-scale polymer prints but also paving the way for stronger composites," said ORNL's Vipin Kumar. While the current method is designed for batch processing, ORNL has developed a patent-pending concept for continuous deposition systems, which will be the focus of upcoming research. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250507130503.htm'>Scientists discover the genes that influence when babies start walking</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 18:01:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The age at which babies take their first steps is strongly influenced by their genes, according to new research from the University of Surrey. They identified 11 genetic markers influencing when babies start walking, thus offering multiple targets for future in-depth biological investigation. For years, researchers knew that environmental factors could influence when babies begin to walk, but this new finding shows that genetics also has a major impact. It suggests that, just like with other traits such as height, some children may naturally start walking earlier or later because of their genetic propensity. "Most babies take their first step sometime between ages 8 months and 24 months, so it is a wide window in which this exciting milestone happens. "Until now, we didn't understand what causes the wide differences between children in when they take their first step. We see that genetics play a considerable role in influencing the timing of this milestone." Moreover, walking later within the typical range was linked genetically with less chance of developing ADHD. Finally, the study showed that relatively later onset of walking was influenced by some of the same genes involved in higher educational attainment. Starting to walk independently is a major milestone for young children. We hope these new genetic findings can advance fundamental understanding about the causes of walking and be used to better support children with motor disorders and learning disabilities. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250507125835.htm'>Teens driving older vehicles have increased risk for fatal crashes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 17:31:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Motor vehicle crashes are the leading cause of death for U.S. teens. In a study published today in JAMA Network Open, researchers found that driving vehicles 6-15 years old had a 19% greater risk of driver death in fatal crashes, and driving vehicles older than 15 years had a 31% greater risk, both compared to driving vehicles 5 years old or newer, regardless of driver age. "Because every family wants their teen to arrive anywhere safely, teen driver fatalities are a serious public health concern," said Jingzhen Ginger Yang, PhD, MPH, lead author of the study and principal investigator in the Center for Injury Research and Policy of the Abigail Wexner Research Institute at Nationwide Children's. "Given teen drivers' high crash rates compared to other age groups and their increasing involvement in fatal crashes, it is crucial for teen drivers to operate the safest vehicles available." Older vehicles and vehicles with fewer driver assistance technologies were associated with a higher risk of death for drivers involved in fatal crashes, regardless of driver age. "Our findings, along with those from other studies, underscore the importance of safe vehicle strategies, education for families and ensuring teens drive safer cars whenever possible," said Fangda Zhang, PhD, research scientist in the Center for Injury Research and Policy at Nationwide Children's and co-lead author of the study. While it is an exciting milestone for families with new drivers, this practice increases teens' vulnerability to vehicle malfunctions, making their driving less safe." Families should be advised to prioritize safety features when choosing the first car for teens, ensuring it is newer and safer, given the increased involvement of teen drivers in motor vehicle crashes and motor vehicle crash-related fatalities. Data for this study were obtained from Fatality Analysis Reporting System (FARS), a comprehensive crash database widely recognized in traffic safety research for its detailed information on drivers, vehicles, and crash environments involved in US fatal crashes. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08960-w'>Chromatin loops are an ancestral hallmark of the animal regulatory genome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 15:41:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In bilaterian animals, gene regulation is shaped by a combination of linear and spatial regulatory information. Regulatory elements along the genome are integrated into gene regulatory landscapes through chromatin compartmentalization1,2, insulation of neighbouring genomic regions3,4 and chromatin looping that brings together distal cis-regulatory sequences5. However, the evolution of these regulatory features is unknown because the three-dimensional genome architecture of most animal lineages remains unexplored6,7. To trace the evolutionary origins of animal genome regulation, here we characterized the physical organization of the genome in non-bilaterian animals (sponges, ctenophores, placozoans and cnidarians)8,9 and their closest unicellular relatives (ichthyosporeans, filastereans and choanoflagellates)10 by combining high-resolution chromosome conformation capture11,12 with epigenomic marks and gene expression data. Our comparative analysis showed that chromatin looping is a conserved feature of genome architecture in ctenophores, placozoans and cnidarians. These sequence-determined distal contacts involve both promoter–enhancer and promoter–promoter interactions. By contrast, chromatin loops are absent in the unicellular relatives of animals. Our findings indicate that spatial genome regulation emerged early in animal evolution. This evolutionary innovation introduced regulatory complexity, ultimately facilitating the diversification of animal developmental programmes and cell type repertoires. A fundamental characteristic of animal multicellularity is the existence of specialized cell types. These cell types result from differential access to genomic information in each cell. Thus, evolutionary changes in genome regulation are proposed to be a major innovation linked to the emergence of complex multicellularity with stable cell differentiation6,10. This idea is supported by comparative genomic analyses showing that gene innovation at the origin of animals was less extensive than previously thought10,13, thus suggesting that an important animal innovation was the ability to coregulate existing genes in different combinations. In bilaterian animals, genome spatial compartmentalization mediates the organization of gene neighbourhoods that can be independently regulated3,4,6 and that are specific to different cell types14. Another mechanism contributing to elaborate gene regulation in bilaterians is the combinatorial interaction of distal cis-regulatory elements and gene promoters by means of chromatin loops that bring distant regions into spatial proximity through genome folding, contrasting with the predominant regulation by proximal promoter elements in unicellular eukaryotes10. Comparative analyses of histone posttranslational modifications have shown that candidate distal enhancer elements, as defined by chromatin features, predate the origin of bilaterian animals8,15,16, whereas such enhancers are absent in the closest unicellular relatives of animals17. However, it is still unclear whether distal regulation in early-branching metazoans is mediated by physical interactions with gene promoters or linked to the existence of insulated gene regulatory landscapes. To investigate the origins of animal gene regulation, here we comparatively studied chromatin architecture at subkilobase resolution in non-bilaterian animal lineages and their closest unicellular relatives of animals (Fig. This includes the two phyla proposed as the sister group to all other animals9,18: ctenophores, which are mostly pelagic, marine predators that swim using ciliated comb cells and have complex nerve nets19,20; and sponges, which are sessile, benthic organisms that filter-feed using collared choanocyte cells8,21. We also examined placozoans, which are millimetre-sized, flat animals that feed on microbial mats by gliding using ciliary movement and mucus secretion, controlled by peptidergic secretory cells22, and cnidarians, the sister group to bilaterians that includes jellyfishes, corals and anemones23. Finally, we studied three unicellular relatives of animals, known as unicellular holozoans: ichthyosporeans, which are osmotrophic unicellular eukaryotes that reproduce through multinucleated coenocytes24; filastereans, which are heterotrophic protists with complex life cycles, including aggregative multicellular stages17,25 and choanoflagellates, which are heterotrophic flagellates that show both single-cell and colonial forms and are the closest living relatives to animals26. The comparative analysis of chromatin maps across these lineages allows us to reconstruct the evolutionary history of genome regulation in animals. a, Comparison of genomic features across metazoans and unicellular holozoans. For H. sapiens, we used previously published mCG methylation percentage data from H1 ESCs cells. Of note, although distal cis-regulatory elements (dCRE) were identified in Amphimedon queenslandica15, their presence in E. muelleri had not been reported previously. mCG, CG methylation; TEs, transposable elements. b, Top left, phylogenetic tree showing the taxon sampling in this study, along with the number of profiled species per clade. Top right and below, Micro-C interaction maps of specific genomic regions (S. arctica, chr. 3L: 20480000–20820000, bin 800 bp; and H. sapiens, chr. 12: 69000000–71000000, bin 5 kb), showing examples of insulation boundaries or chromatin loops. All interaction maps were balanced using ICE normalization. We used Micro-C11,12 to map genome-wide chromatin contacts at single-nucleosome resolution in representatives of non-bilaterian animal lineages (Fig. As outgroup species, we studied chromatin architecture in three unicellular holozoans: the ichthyosporean Sphaeroforma arctica24, unicellular filasterean amoeba Capsaspora owczarzaki17,25 and the choanoflagellate Salpingoeca rosetta26. We also compared our chromatin maps with existing datasets from two bilaterians: Drosophila melanogaster27 and Homo sapiens12. To analyse our chromatin contact experiments, we first resequenced de novo and assembled to chromosome-scale the genomes of M. leidyi, E. muelleri and C. owczarzaki using a combination of Nanopore (Oxford Nanopore Technology) long-read sequencing and Micro-C data (Extended Data Fig. For S. arctica, S. rosetta and T. adhaerens, we rescaffolded existing genomes22,24,26 to chromosome level using Micro-C data. In addition, to interpret the observed contact features, we generated genome-wide maps of chromatin accessibility (assay for transposase-accessible chromatin with high-throughput sequencing or ATAC-seq), chromatin modifications (chromatin immunoprecipitation with sequencing (ChIP–seq) for H3K4me3, H3K4me2, H3K4me1) and gene expression (RNA sequencing or RNA-seq), or used published datasets when available (Supplementary Table 2). We integrated three-dimensional (3D) chromatin data with linear chromatin marks to systematically compare genome architectural features at different resolutions3,4,7 (compartmentalization, insulation and chromatin looping) and across phylogenetically distant species with diverse genome sizes, gene densities and transposable element content (Fig. We first analysed global chromosomal compartmentalization, which results from the spatial segregation of distinct chromatin states genome-wide (active, A; inactive, B) and is influenced by histone marks, DNA methylation and gene transcription, among other phenomena28,29. As such, compartmentalization is often considered an intrinsic biophysical property of the chromatin driven by phase separation30,31. To compare the degree of self-affinity and segregation between major chromatin compartments, we defined A/B compartment limits in each species. We then calculated the intensity of compartmentalization in genomic bins with compartment A and B interaction frequency in the top 20th percentile (Fig. Compartmentalization strength in each species was quantified as the ratio of homotypic (AA, BB) to heterotypic (AB) interactions (Fig. The relative resolutions were obtained by partitioning genomes into equal number of bins across species (Extended Data Fig. 3a,b), but the differences between species remained consistent regardless of the number of bins used (Fig. Furthermore, we assigned an intermediate compartment (I) to regions with weak spatial separation (Extended Data Fig. a, Saddle plots showing contact interactions between A and B compartments in each species, organized by eigenvector ranking. To obtain the distance-normalized matrix, the ratio of observed-over expected interactions is calculated, followed by eigenvector decomposition. The observed (O) and expected (E) values are averaged to create a saddle plot. Cowc, C. owczarzaki; Dmel, D. melanogaster; Emue, E. muelleri; Hsap, H. sapiens; Mlei, M. leidyi; Nvec, N. vectensis; Sarc, S. arctica; Sros, S. rosetta; Tadh, T. adhaerens. b, Compartment strength quantification at different relative resolutions. The barplot below shows the contribution of homotypical chromatin interactions within active (AA) and inactive (BB) chromatin states. c, Aggregate plots showing contact enrichment within a rescaled region between two insulation boundaries. The boundaries are identified using the sliding diamond window to detect the changes in contact frequencies in each genomic bin. To plot pile-ups, regions between insulation boundaries are rescaled and their normalized observed and expected contact frequencies are averaged. d, Insulation score distributions illustrating the degree of isolation between linear genomic neighbourhoods. Number of annotated strong boundaries is indicated in blue, with a vertical line representing the median value of each distribution. e, Classification of insulation boundaries using hierarchical assignment of structural and genomic features. f, Size distribution of annotated chromatin loops in each species. The boxplots show the median (centre line), 25th and 75th percentiles (box limits) and the whiskers show the range of variability, excluding outliers, which are shown as individual points. g, Annotation of chromatin loop anchors with promoter (P) and enhancer (E) signatures based on normalized H3K4me3 and H3K4me2 or H3K4me1 ChIP–seq coverage. Chromatin loop anchors with undefined (U) epigenetic signature are shown in grey. Our analysis revealed that, with exception of M. leidyi, animal genomes were globally segregated into transcriptionally active, gene-dense compartments and transcriptionally inactive, transposable element-rich compartments, similar to what is observed in bilaterian animals (Extended Data Fig. In these species, we detected a strong separation of A and B compartments in saddle plots (Fig. 2a) and the compartment strength values above 1.8 (Fig. Moreover, these compartments encompass relatively large contiguous regions across the genome (Extended Data Fig. By contrast, unicellular holozoans and M. leidyi did not show strong separation of large A and B compartments (Fig. 2a,b), similar to what is observed in yeast32 and other protists33. The absence of large-scale chromatin compartments in M. leidyi is unusual among animals, although it has been previously reported in certain species34. This lack of compartmentalization may be due to the absence of constitutively silenced regions across different cell types. Overall, our results indicate that A/B chromosomal compartmentalization is a phylogenetically conserved feature across animal genomes. We next characterized small-scale chromosomal features across species by defining spatial insulation boundaries between neighbouring loci. The boundary elements that partition genome into domains can arise from active transcription, silenced repetitive regions or binding of sequence-specific architectural proteins at insulator or tethering elements5,27,35,36. Thus, our first goal was to identify the occurrence of insulation boundaries in each species (Fig. 4), and then classify these points into different regulatory or structural features (domain boundaries, gene bodies, regulatory loops and so on) (Fig. To this end, we calculated insulation scores for each species, representing the difference in contact frequencies between each genomic bin and its neighbouring bins. We used different resolutions and sliding window sizes (Extended Data Fig. 4a,b) and, for each species, we selected the resolution and two window sizes that yielded the maximal insulation signal, indicating the strongest partitioning of the genome into isolated structural and functional domains. The median distance between successive identified boundary elements varied between 6.4 kilobases (kb) in S. rosetta and 190 kb in H. sapiens, yet the median number of genes per interval was consistently similar across species, with two to four genes (Extended Data Fig. The presence of self-interacting domains, contiguous regions of the genome with enriched interactions, was assessed by examining the average pile-up plots between insulation boundaries (Fig. We observed weak contact enrichment between pairs of insulated boundaries in unicellular holozoans and E. muelleri. In M. leidyi, boundary elements were tethered through strong focal contacts and without intradomain interactions, contrary to what would be expected within topologically associating domains (TADs)3. By contrast, D. melanogaster showed intradomain enrichment without focal contacts, in agreement with previously reported domains37. T. adhaerens and N. vectensis showed a certain degree of self-affinity within insulated neighbourhoods, as well as focal point enrichment (Fig. M. leidyi, T. adhaerens, N. vectensis, H. sapiens and S. arctica genomes contained strong boundary elements in comparison with E. muelleri and, especially, the weakly insulated genomes of C. owczarzaki and S. rosetta (Fig. After identifying insulation points, we investigated the genomic features associated with these boundaries (Fig. We first assigned insulation boundaries to annotated chromatin loops, followed by the transcription start sites (TSSs) of genes not involved in chromatin looping and then accessible chromatin regions that may represent other regulatory elements. Remaining boundaries were assigned to A/B compartment limits. This analysis revealed that most insulation boundaries in unicellular holozoans and E. muelleri were associated with active TSSs (Fig. 2e), suggesting that active transcription is the main factor defining insulation in these species37. By contrast, many insulation boundaries could be assigned to chromatin loop anchors in M. leidyi (77%; compared to 78% in H. sapiens human embryonic stem cells) and in T. adhaerens (38%), whereas in N. vectensis, we identified 166 chromatin loops that represented only 1.6% of insulation boundaries. The number of chromatin loops in M. leidyi (4,261) and T. adhaerens (3,065) was much higher than those found in N. vectensis (166) and D. melanogaster (313)27, despite their similar genome sizes and gene densities (Fig. Loop sizes were comparable in these four species (median 21–28 kb), but much smaller than in H. sapiens (median 140 kb) with a genome 15–30 times larger (Fig. To further characterize these distal contacts, we examined genome-wide H3K4me3, H3K4me2 and H3K4me1 to classify many of the identified loop anchor sites as promoter-like elements (Fig. By contrast, 79% of loops in T. adhaerens connected promoters to other promoters, similarly to what is observed in D. melanogaster (49%)38. Our results show that enhancer–promoter and promoter–promoter long-range chromatin loops are shared between bilaterians and early-branching animal lineages, and possibly date back to the origin of animal multicellularity. In unicellular holozoans, we did not observe any spatial contact patterns indicative of chromatin loops. However, manual inspection revealed a few regions enriched in distal contacts. For example, in S. arctica, we could identify 296 self-interacting insulated domains that also contact each other (Extended Data Fig. These regions were depleted of active histone marks and were enriched in transposable elements, probably representing repressed chromatin domains that cosegregate (Extended Data Fig. In S. rosetta, there were 183 distally interacting regions that contained lowly expressed genes (Extended Data Fig. 5d,e) and were enriched in H3K4me1 and H3K27me3 or lacked profiled marks (Extended Data Fig. These may also represent repressed regions39, albeit they do not form well-defined domains like in S. arctica. In C. owczarzaki, we observed a plaid pattern indicative of chromatin microcompartments (Extended Data Fig. 5g), reflecting the spatial cosegregation of active promoters of highly transcribed genes with a strong H3K4me3 signal (Extended Data Fig. These microcompartment contacts form a regional small-scale checkerboard pattern with alternating loci of high and low interactions. Furthermore, we also detected high-frequency contact domains over gene bodies of highly expressed genes (Extended Data Fig. In the sponge E. muelleri, we identified local interactions perpendicular to the main diagonal, and visually reminiscent to fountains observed in mouse, zebrafish and C. elegans40 (Extended Data Fig. Manual inspection further revealed 84 focal contacts between distal genomic loci (Extended Data Fig. 6c), including gene promoters interacting with other regions showing promoter or enhancer-like chromatin signatures (Extended Data Fig. These weak distal interactions occurred between extended genomic regions, in contrast to the point-to-point contacts typical of chromatin loops (Extended Data Fig. Although chromatin loops were absent in E. muelleri, we identified 243 distal cis-regulatory elements, consistent with findings in other sponge species15. These elements were characterized by chromatin accessibility, with surrounding regions showing high H3K4me1 and low H3K4me3 signals, and were mostly intergenic but close to annotated TSS (median 3.8 kb) (Extended Data Fig. This distance-to-TSS distribution was similar to that of annotated enhancer elements in M. leidyi, T. adhaerens, N. vectensis and D. melanogaster that do not form loops (median 5.6 kb, compared to 31 kb in loop-forming enhancers) (Extended Data Fig. 6f), suggesting that sponges' enhancer elements might function by proximity without the need for stable looping41. Genome folding in the cnidarian N. vectensis was characterized by the presence of chromatin loops, as well as weakly insulated self-interacting domains (Extended Data Fig. We identified 166 chromatin loops forming both promoter–promoter and promoter–enhancer contacts, and with some loops spanning nearly 1 megabase (Mb) (Extended Data Fig. Chromatin loops have also been reported in the hydrozoan Hydra vulgaris42, suggesting they are a conserved feature in cnidarians. Notably, some of the identified chromatin loops in N. vectensis showed a one-sided stripe pattern similar to those observed in other species, which are generated by cohesin extrusion43. Moreover, we identified an enriched GTGT motif (FC = 327, P = 1 × 10−40) present in 32% of loop anchors (Extended Data Fig. This motif resembles sequences with G-quadruplex-forming potential44, which have been shown to stabilize enhancer–promoter interactions in other species45. Beyond chromatin loops, we also observed self-interacting domains in N. vectensis (Extended Data Fig. The insulation boundaries of these domains were enriched for the YY1 motif (FC = 9,016, P = 1 × 10−87) (Extended Data Fig. 7f), which is known to mediate chromatin interactions35,45. These regions represent high-frequency contacts within the same gene regulatory landscape, but are not stabilized by chromatin loops as in vertebrate TADs3, nor are they as strongly insulated as the domains defined by insulator elements in D. melanogaster27. Our high-resolution chromatin contract maps revealed a complex 3D genome organization in the placozoan T. adhaerens, characterized by many loop contacts forming 3D interaction hubs (Fig. To confirm this observation, we profiled chromatin contacts in a distantly related placozoan species, Cladtertia collaboinventa, which showed a very similar pattern (Fig. Most of these interactions are promoter–promoter hubs (n = 2,413 for T. adhaerens and n = 3,239 for C. collaboinventa) (Extended Data Fig. Notably, 7–10% of chromatin contacts (n = 241 for T. adhaerens, n = 394 for C. collaboinventa) connected promoters with intronic or intergenic enhancer regions (Extended Data Fig. 8a,b), revealing the presence of distal cis-regulatory elements in placozoans. a, Example of syntenic genomic regions in placozoans T. adhaerens (TadhH1_4: 3860000–4060000, bin 800 bp) and C. collaboinventa (chr. b, Gaudí plots projecting ATAC-seq, H3K4me3 and exon annotation signals onto a two-dimensional Kamada–Kawai graph layout (top left) represented by the top 20% of contact pairs with solid colours highlighting statistically significant regions (P < 0.05) identified using a one-sided permutation test. The high–high (HH) signal marks genomic bins enriched in signal and that are in spatial proximity with other bins enriched in signal; low–low (LL) bins are depleted in signal as well as neighbourhoods are depleted in signal; high–low (HL) and low–high (LH) are bins that are enriched in signal, but not their neighbourhood, and in reverse. c, Classification of T. adhaerens genes into three categories (GP1, GP2 and GP3) on the basis of structural and epigenetic features. Top, example regions containing genes classified into GP1, GP2 and GP3 groups. The resolution of Micro-C maps is 800 bp, maximum intensity value of ICE normalized Micro-C maps is as in a. Bottom, average loop strength between promoter regions of the genes from each groups is measured with APA. The colour bar of pile-up plots shows enrichment of observed over expected values. d, Sequence motif found in loop regions, which are also overlapping GP1 promoter regions (left panel), is present in promoter regions of orthologous GP1 genes in other placozoan species (right panel). The total number of shared orthologues is indicated. e, Heatmaps showing CPM normalized ATAC-seq and ChIP–seq coverage, motif scores and Mutator transposable element density ±5 kb around the TSS of GP1, GP2 and GP3 genes. Each heatmap scale starts at zero. To further reconstruct the 3D organization of these hubs, we used METALoci to calculate spatial correlation between genome folding and epigenetic (ATAC, H3K4me3 ChIP) or genomic (exon annotation) features (Fig. This analysis revealed a nested structure where accessible promoter regions were central to the 3D interactions tightly clustered in space (ATAC-seq in Fig. 3b), whereas gene bodies and the first nucleosome (H3K4me3) occupied more peripheral locations (Fig. Furthermore, genes within spatial promoter hubs were linearly grouped along the genome, resembling the arrangement of housekeeping genes observed in mouse embryonic stem cells46. Alternatively, these structures could be associated with active transcription and the formation of micro-compartmentalized RNA polymerase II-driven transcription hotspots47. Notably, not all collinear genes formed promoter hubs. Following this observation, we categorized genes into three groups based on their spatial and epigenetic organization (Fig. This includes group 1 genes (GP1, n = 2,978 in T. adhaerens, n = 3,973 in C. collaboinventa) that had both ATAC and H3K4me3 peaks and formed chromatin loops, with an average interaction strength in aggregate peak analysis (APA) of 1.32, indicating the enrichment of Micro-C signal at loop anchors. Group 2 genes (GP2: n = 3,681 in T. adhaerens, n = 3,119 in C. collaboinventa) also showed ATAC and H3K4me3 peaks, but lacked strong distal contacts (APA = 1.12). Last, group 3 genes (GP3: n = 3,851 in T. adhaerens, n = 4,238 in C. collaboinventa) had neither chromatin loops (APA = 0.968) nor active chromatin marks (Fig. On average, GP1 genes showed a stronger H3K4me3 ChIP–seq signal and higher expression levels compared to genes in GP2 and GP3 (Extended Data Fig. 8d) and were associated with housekeeping functions, including intracellular trafficking, translation and messenger RNA processing (Extended Data Fig. By contrast, GP3 genes were enriched in cell type-specific functions related to peptidergic cells (Extended Data Fig. 8g,h), potentially explaining the lack of chromatin features in our bulk epigenomic experiments. To understand what distinguishes placozoan GP1 genes, we analysed loop anchor sequences in both species using genomic sequences as background. We identified an enriched motif at chromatin loop anchor regions in both placozoan species (Fig. 8i,j) and found that GP1 promoters frequently contained insertions of Mutator DNA transposable elements (Fig. 8e), with the terminal inverted repeat (TIR) sequence of this transposon containing the identified sequence motif. To further explore this association, we constructed a phylogenetic tree including all intact Mutator TIR sequences in four placozoan species (Extended Data Fig. This analysis revealed a Mutator family shared across species and with consensus TIR sequences resembling the motif found in chromatin loops anchors (Extended Data Fig. The connection between chromatin loops and the Mutator transposable element suggests a potential evolutionary and functional relationship. One possibility is that an architectural protein in placozoans evolved to recognize the sequence motif within the Mutator TIRs, leading to ‘domestication' of these sites as regulatory elements. Alternatively, the presence of the motif and Mutator TIR sequences may indicate targeted integration of Mutator transposons into promoter regions of highly expressed genes. Overall, our analyses showed that roughly one-third of T. adhaerens and C. collaboinventa genes are part of promoter hubs mediated by chromatin loops and that these contacts are associated with the presence of conserved Mutator DNA transposons harbouring a specific sequence motif. The physical architecture of M. leidyi genome is dominated by thousands of chromatin loops (n = 4,261) (Fig. In total, we identified 916 gene promoters participating in chromatin loops, with each promoter contacting between one (50%) and up to 15 enhancers (Fig. These enhancers are mainly located in intronic (69%) and intergenic (24%) regions at one to eight genes from the contacted promoters. We also observed the accumulation of cohesin at loop anchor sites using ChIP–seq against SMC1 cohesin subunit (Extended Data Fig. To assess whether these features are conserved across ctenophores, we profiled chromatin contacts, albeit at lower resolution, in the cydippid ctenophore Hormiphora californensis (Extended Data Fig. 9d), which diverged from the lobate ctenophore M. leidyi roughly 180 million years ago9. At the sampled resolution, we detected 239 strong chromatin loops in H. californensis. In both ctenophores, genes involved in chromatin loop formation showed higher expression (Extended Data Fig. a, Example genomic region showing chromatin loops between promoters and enhancers at 400 bp resolution. b, Left, histogram of enhancer contacts per promoter. Right, genomic location of enhancers. c, Sequence motif enriched in loop anchors. d, DNA methylation profiles centred around motifs located at promoter and enhancer loop regions, or outside loops. e, Chromatin-bound proteome of M. leidyi, showing identified proteins sorted by abundance with architectural proteins CTEP1 and CTEP2 as the most abundant zf-C2H2s. f, DAP-seq signals around GC-motif sites with high (left) versus low (right) methylation levels, and sites located within (top) or outside (bottom) of loop anchors. CTEP1 showed higher affinity for unmethylated GC-rich motifs in DAP-seq assays with native or PCR amplified gDNA (lacking methylation). g, Boxplots showing PhastCons conservation scores across three ctenophore species (B. microptera, P. bachei and H. californensis). Two-sided Wilcoxon rank sum test showed significant conservation differences between intergenic enhancers (n = 969) and promoters in loops (n = 778) (***P = 1.3 × 10−15) and between promoters in loops and promoters outside loops (n = 14,996) (***P < 2.22 × 10−16), whereas intergenic enhancers and promoters outside loops showed no significant difference (not significant (NS), P = 0.88). h, Syntenic conservation within M. leidyi chromatin loops compared to H. californensis. Left plot, barplot showing the fraction of conserved orthologues (OGs) in all alignable genomic regions across ctenophore species (***P = 5.5 × 10−4, chi-squared test for given probabilities). Right plot, boxplot of shared orthologues between individual genomic regions within chromatin loops (n = 115) versus in random genomic regions (n = 259) of similar size (***P = 2.4 × 10−5, Wilcoxon rank sum test with continuity correction). Boxplot limits as in g. Silhouette of H. californensis in h reproduced from PhyloPic (https://www.phylopic.org/), created by S. Haddock and K. Wothe under a CC0 1.0 Universal Public Domain licence. To investigate whether the chromatin loops in ctenophores are formed in a sequence-specific manner, we searched for the enriched motif in loop anchors of both species, using GC-normalized genomic random sequences as a background. We identified GC-rich motif (FC = 8,522; P = 1 × 10−497) that was present in over 75% of loop anchors (Fig. 9f) and at both promoter (79%) and enhancer sites (74%) involved in chromatin loops (Extended Data Fig. In addition, this motif was found in an extra 3,348 gene promoters (21% of all genes) with no chromatin loops detected (Extended Data Fig. As the identified GC-rich motif contains two CpG dinucleotides, we examined DNA methylation using long-read Nanopore sequencing data. The overall methylation level in M. leidyi was low (6.8%), in agreement with previous reports using whole-genome bisulfite sequencing48. However, at loop anchor sites motifs showed low cytosine methylation, whereas motif occurrences outside loop anchor points showed high methylation (Fig. Thus, we propose that DNA methylation of this GC-rich motif serves as a regulatory mechanism of loop formation in M. leidyi, potentially controlling the binding of an unknown, methylation-sensitive architectural DNA-binding factor, similar to mechanisms described for CCCTC-binding factor (CTCF) and other transcription factors49. The presence of DNA-binding proteins was further supported by the ATAC-seq footprint profile at motif regions in loop anchors (Extended Data Fig. To identify these potential architectural proteins, we profiled the chromatin-bound proteome of M. leidyi (Fig. We then selected the most abundant zf-C2H2 domain-containing proteins and analysed their DNA-binding specificity using DAP-seq, as zf-C2H2 factors are often associated with chromatin looping in other species2,5,35,50. This analysis identified two proteins, named here CTEP1 (Ctenophore-specific Tethering Protein 1) and CTEP2, which overlapped with 80% of detected loop anchor regions and showed strong affinity for the same GC-rich motif we had previously identified (Extended Data Fig. Moreover, DAP-seq confirmed that the binding of both proteins was inhibited at sites with high DNA methylation (Fig. Thus, we conclude that CTEP1 and CTEP2 bind unmethylated GC-rich motif sites at chromatin loops. Notably, these proteins are conserved across ctenophore species (Extended Data Fig. 9n and Supplementary Table 3), but are absent from genomes of other metazoans. Finally, we analysed evolutionary conservation of the sequences at the loop anchor points. To this end, we calculated genome-wide conservation scores from alignments of M. leidyi genome with three other ctenophore species (Bolinopsis microptera9, Pleurobrachia bachei20 and H. californensis51). Chromatin loop anchors, both at intronic and intergenic regions, showed higher sequence conservation compared to other introns or random genomic regions, respectively (Fig. The promoters of genes involved in distal contacts showed lower conservation score compared to other promoters (Fig. 4g), with conservation levels similar to those of random intergenic regions. Moreover, these promoters had a high frequency of transposable element integrations and elevated DNA methylation (Fig. Furthermore, we found that genes located within enhancer–promoter loop regions in M. leidyi have higher syntenic conservation across ctenophore species compared to other genomic regions of similar size (Fig. Overall, the conservation of loop anchor regions across ctenophore species and the increased syntenic linkage of genes suggest that gene positioning is constrained by genome architecture. These findings indicate that the distal chromatin contacts identified in M. leidyi represent an evolutionary conserved mechanism of genome regulation present in both lobate and cydippid ctenophores. Genome architecture is the result of both physicochemical and regulatory processes3,4,31. In unicellular organisms, chromatin contact patterns are shaped by the polymer nature of the chromatin fibre32 and by gene transcriptional states52. Also, insulation boundaries resulting from highly transcribed genes in divergent orientations are described in dinoflagellate genomes33. In unicellular holozoans, we observed similar insulation patterns around TSSs, but without evidence of further regulatory features or sequence-specific determinants associated with insulation boundaries. We also found cosegregating inactive chromatin regions in the large genome of S. arctica, and to a lesser extent in S. rosetta39. By contrast, these structures are absent in unicellular organisms such as C. owczarzaki or S. cerevisiae, which both have gene-dense genomes without heterochromatic regions. In bilaterian species, extra chromatin structures involved in gene regulation have been observed, often mediated by architectural proteins binding to specific sequences2,5,35,50. These include discrete chromatin loops between cis-regulatory elements and promoters, mediated by tethering elements27, as well as insulated gene regulatory landscapes, such as loop TADs bounded by convergent CTCF sites in vertebrates3. Notably, TAD-like domain structures can also result from the passive cosegregation of active versus inactive chromatin states37,54, rather than being determined by sequence-specific insulation elements. Examples of these are Polycomb bodies55 and other heterochromatic compartment domains28,29. In early-branching animals we did not identify loop-bound TADs or any evidence of sequence-defined insulated TADs. However, we did detect chromatin loops spanning tens of kilobases and linking distal cis-regulatory elements and promoters in cnidarians, ctenophores and placozoans. In the case of ctenophores, thousands of chromatin loops link enhancers and promoters, showing that distal loops can be extremely frequent even in small genomes (roughly 200 Mb). Another example is the thousands of chromatin loops in placozoans, with even smaller genomes (roughly 100 Mb). Both placozoans and ctenophores complex looping architectures are associated with transposable elements. Although the causal relationship between transposable elements and chromatin loops is unclear, this observation suggests that complex 3D genome architectures might be influenced by lineage-specific transposable element invasion histories56. The mechanisms and factors responsible for loop formation in non-bilaterians and most invertebrates remain unknown7. The zf-C2H2 protein CTCF is the main architectural protein in vertebrates and is conserved across bilaterians. In annelids it has been associated to open chromatin regions57 and in cephalopods it defines TAD boundaries58. Given that CTCF is absent in non-bilaterians36, other factors, possibly from the zf-C2H2 family (Extended Data Fig. 9q), might be involved in the formation of these loops. In fact, a variety of architectural proteins other than CTCF have been described in Drosophila, many of which are zf-C2H2 proteins with restricted phylogenetic distributions such as the insect-specific CP190 factor2,50,59. Similarly, we identified two ctenophore-specific zf-C2H2 proteins (CTEP1 and CTEP2) associated with loop anchor regions in M. leidyi. It is possible that other, yet unidentified, lineage-specific zf-C2H2 proteins contribute to chromatin architecture in different animal lineages. Globally, our findings suggest an evolutionary scenario (Fig. 5) in which chromatin compartment domains defined by transcriptional states28 (but lacking sequence-specific insulation or tethering elements) were present in the unicellular ancestor of animals, as seen in extant unicellular holozoans. At the origin of animals, distal cis-regulatory elements evolved, requiring sequence-determined, stable chromatin looping mechanisms to link these enhancers with gene promoters (at least at certain distances41). This added an extra layer of regulatory complexity to cell type-specific gene regulation. The origin of this distal gene regulation would also explain the existence of regulatory-linked genomic regions showing conserved synteny60, as observed in ctenophore regions between loop anchor points. Moreover, domains insulated by sequence elements probably originated at the root of bilaterian animals, as they are observed in vertebrates, insects and probably spiralians57,58. In the specific case of vertebrates these domains are formed by a mechanism of CTCF-dependent loop extrusion so far not observed in any other lineage7, which further exemplifies the potential diversity of mechanisms involved in chromatin architecture across metazoans. Future extended taxon sampling will further refine this evolutionary scenario and help solve open questions such as whether there are conserved or lineage-specific factors involved in the establishment of chromatin loops across animals, how dynamic these structures are in development and across cell types or when did sequence-determined, insulated TADs first emerged in animal evolution. a, Phylogenetic tree illustrating the taxonomic distribution of 3D-chromatin features. b, Schematic depicting major innovations in animal genome regulation at different ancestral nodes. S. arctica coenocytic culture was grown in marine broth (Difco, 3704 g l−1) at 12 °C in 25 cm2 flasks. Cells were passaged every 7 days using a 1:100 dilution. To synchronize cells in the G1/early S phase, an 8-day old culture was treated with 200 mM hydroxyurea (Sigma-Aldrich, catalogue no. H8627) for 18 h in the presence of 0.3% dimethylsulfoxide (DMSO). Frozen cells were then reconstituted in CMFSW and crosslinked with 1% formaldehyde (Thermo Scientific, catalogue no. The crosslinking reaction was quenched with 128 mM glycine for 5 min in the vacuum desiccator, followed by a 15 min incubation on ice. Cells were pelleted at 4 °C for 10 min at 2,000g, washed once with CMFSW, reconstituted in CMFSW to the concentration of 2 M ml−1 and crosslinked with 3 mM DSG (Thermo Scientific, catalogue no. A35392) for 40 min at room temperature on a rotating wheel. The reaction was quenched with 400 mM glycine for 5 min. Double-crosslinked cells were pelleted at 4 °C for 15 min at 2,000g and flash-frozen in liquid nitrogen. C. owczarzaki strain ATCC30864 was maintained in axenic culture at 23 °C in the ATCC (American Type Culture Collection) medium 1034 (modified PYNFH medium) in 25 cm2 flasks. For subculture, filopodial cells were passaged every 2–3 days using a dilution of 1:100. Before collection, filopodial cells were synchronized in G1 or early S phase by treating a filopodial culture of 70–80% confluency with 100 mM hydroxyurea (Sigma-Aldrich, catalogue no. Synchronized cells were scraped off the surface and pelleted at 2,200g for 5 min at room temperature. Collected cells were crosslinked as described in ref. Briefly, cells were crosslinked with 1% formaldehyde (Thermo Scientific, catalogue no. 28906) in PBS for 10 min on a rotating wheel at room temperature. The crosslinking reaction was quenched with 128 mM glycine for 5 min at room temperature followed by extra incubation on ice for 15 min. The crosslinked cells were pelleted at 4 °C for 10 min at 2,000g and washed once with ice-cold PBS. Cells were diluted in PBS to the concentration of 2 M ml−1 and also crosslinked with 3 mM DSG (Thermo Scientific, catalogue no. A35392) for 40 min at room temperature on a rotating wheel. The crosslinking was quenched with 400 mM glycine for 5 min. Cells were pelleted at 4 °C for 15 min at 2,000g and flash-frozen in aliquots of 2 million cells. S. rosetta was cocultured with Echinicola pacifica bacteria in artificial sea water supplemented with 20% cereal grass media (CGM3) at 23 °C. To synchronize the cell culture in the G1 or early S phase, cells from a 3-day-old culture were pelleted at 2,000g for 10 min and diluted in 4% CGM3 in artificial sea water to the concentration of 300,000 cells per ml. Cells were treated with 0.05 mM aphidicolin (Sigma-Aldrich, catalogue no. 178273) in the presence of 0.3% DMSO. After 18 h of incubation, cells, including chain colonies, fast and slow swimmers, were pelleted at 2,000g for 15 min. To remove bacteria from the choanoflagellate culture, collected cells reconstituted in 1 ml of culture media were passed through a Ficoll layer (1.6% Ficoll (Sigma-Aldrich, catalogue no. F5415), 0.5 M sorbitol, 50 mM Tris-HCl (pH 8.8), 15 mM MgCl2, 1% artificial sea water) by centrifugation at 1,000g for 10 min at 4 °C. Pelleted choanoflagellate cells were then double-crosslinked with 1% formaldehyde in CMFSW and 3 mM DSG in CMFSW as described above for C. owczarzaki. The crosslinked cells were pelleted at 4 °C for 15 min at 2,000g and flash-frozen in liquid nitrogen. E. muelleri sponges gemmules were hatched and grown for 1 week in Strekal's media63 in 150 × 25 mm culture dishes (Corning, catalogue no. To isolate phagocytic choanocyte cell population, specimens were fed for 10 min with 0.5 µm fluorescent carboxylate-modified FluoSpheres (Invitrogen, catalogue no. Sponges were washed once with Strekal's media, and 1% formaldehyde solution in Strekal's media was added to crosslink specimens for 10 min at room temperature with occasional mixing. To quench formaldehyde, 128 mM glycine was added and incubated for 5 min at room temperature and 15 min on ice. Crosslinked sponge specimens were washed twice with ice-cold Strekal's media. Roughly 80 specimens were transferred in 5 ml of the Strekal's media and dissociated by trituration until all tissue was removed from the gemmule husks (roughly ten trituration passages). The dissociated cell suspension was filtered through a 40-µm cell strainer, and cells were diluted to 2 M ml−1 concentration. The second crosslinking was performed with 3 mM DSG (Thermo Scientific, catalogue no. 20593) in Strekal's media for 40 min at room temperature on a rotating wheel. The reaction was quenched with 400 mM glycine for 5 min at room temperature. Crosslinked cells were pelleted at 4 °C for 15 min at 2,000g, and then resuspended in 2 ml of ice-cold Strekal's media with 2 µg ml−1 Hoechst 33342 (Thermo Scientific, catalogue no. Choanocytes were isolated using a BD FACS Aria II sorter with BD FACSDiva v.6.1.3 (BD Biosciences) as cells showing both FluoSphere fluorescence and Hoechst nuclei staining. Fluorescence-activated cell sorting (FACS) profiles were analysed with FlowJo v.10.7 (Extended Data Fig. M. leidyi specimens were kept in 300-ml glass beakers with 5–10 individuals at 21 °C in artificial sea water (Red Sea, catalogue no. Ctenophores were fed daily with a mixture of living rotifers (Brachionus sp.) and brine shrimps (Artemia salina). The water was exchanged once a week. To dissociate animal tissue, roughly five adult animals (10 mm long) were transferred into CMFSW and washed twice to exchange the buffer. Animal tissue was dissociated into single cells in 5 ml of fresh CMFSW by triturating every 2 min for a total of 10 min. Dissociated cells were filtered through a 40-µm cell strainer and diluted to 2 M ml−1 for the subsequent formaldehyde crosslinking. The reaction was stopped with 128 mM glycine for 5 min at room temperature and 15 min on ice. Crosslinked cells were pelleted at 4 °C for 10 min at 2,000g, washed once with CMFSW and resuspended to 2 M ml−1 for a second crosslinking with 3 mM DSG in CMFSW. The crosslinking reaction was stopped after 40 min of incubation at room temperature on a rotating wheel with 400 mM glycine for 5 min. H. californensis specimens from the first generation (F1) of a laboratory-reared culture at the Monterey Bay Aquarium (USA) were flash-frozen and pulverized in liquid nitrogen. Extracted cells and nuclei were filtered through a 40-µm cell strainer and pelleted by centrifugation at 4 °C for 10 min at 2,000g. Cells were double-crosslinked with 1% formaldehyde in CMFSW and 3 mM DSG in CMFSW as described for M. leidyi. T. adhaerens and C. collaboinventa colonies were grown in 200 × 30 mm glass Petri dishes at 21 °C in artificial sea water (Red Sea, catalogue no. Placozoans were fed once a week with unicellular algae (Pyrenomonas sp. ), the water was exchanged every second week. To prepare single-cell suspension, roughly 500 animals were collected, washed twice with CMFSW and resuspended in 1 ml of CMFSW supplemented with 2 mM EDTA. Animal tissue was triturated every 2 min for a total of 10 min at room temperature. Dissociated cells were filtered through a 40-µm cell strainer, diluted to 2 M ml−1 and crosslinked with 1% formaldehyde in CMFSW for 10 min at room temperature on a rotating wheel. The reaction was quenched with 128 mM glycine for 5 min at room temperature and 15 min on ice. Cells were pelleted at 4 °C for 10 min at 2,000g, washed once with CMFSW and resuspended in 3 mM DSG in CMFSW for a second crosslinking. After 40 min of incubation at room temperature on a rotating wheel, 400 mM glycine was added to stop the reaction and cells were pelleted at 4 °C for 15 min at 2,000g. N. vectensis NvElav1::mOrange transgenic line65 was maintained in one-third artificial sea water (Red Sea, catalogue no. To isolate NvElav1::mOrange positive cells, 1.5–2-month-old animals starved for 1 day before the experiment were crosslinked with 1% formaldehyde in Ca2+/Mg2+-free one-third sea water (one-third CMF: 17 mM HEPES (pH 7.4), 167 mM NaCl, 9 mM NaHCO3, 3.3 mM KCl) for 10 min under vacuum. The crosslinking reaction was stopped by adding 128 mM glycine and incubating the tissue under vacuum for 5 min, followed by a 15 min incubation on ice. The crosslinked tissue was dissociated into single cells by incubating the tissue with 10 mg ml−1 of Protease XIV (Sigma-Aldrich, catalogue no. P5147) in one-third CMF and 1 mM CaCl2 for 5 min at 24 °C triturating the tissue every 1 min. The digested tissue was pelleted at 800g for 5 min, reconstituted in one-third CMF supplemented with 2 mM EDTA and 2 µg ml−1 Hoechst 33342 (Thermo Scientific, catalogue no. 62249), and the trituration continued for another 5–10 min. Dissociated cells were filtered through a 40-µm cell strainer, and neurons were isolated using a BD FACS Aria II as cells showing both the mOrange signal and Hoechst nuclei staining (Extended Data Fig. Isolated NvElav1::mOrange positive cells were also crosslinked with 3 mM DSG for 40 min at room temperature. Micro-C libraries were prepared as previously described11,12 with the following modification. To digest chromatin to a 80% monomers to 20% dimer and oligomers nucleosome ratio, an appropriate amount of MNase (Takara Bio, catalogue no. 2910a) was added (Extended Data Fig. The digestion reaction was stopped with 4 mM EGTA (pH 8.0) followed by incubation at 65 °C for 10 min without agitation. Cells were washed twice with ice-cold MB2 buffer (10 mM Tris-HCl (pH 7.4), 50 mM NaCl, 10 mM MgCl2, 0.1% BSA) and pelleted at 4,500g for 5 min at 4 °C. Next, to repair the fragment ends after MNase digestion, pelleted cells were resuspended in the repair reaction mix (5 µl of 10× NEBuffer 2.1, 34 µl of nuclease-free water, 1 µl of 100 mM ATP, 2.5 µl of 100 mM DTT) supplemented with 2.5 µl of 10 U µl−1 T4 PNK (NEB, catalogue no. After 15 min of incubation at 37 °C with 850 rpm agitation, 5 µl of 5 U µl−1 Klenow Fragment (NEB, catalogue no. M0210) was added to generate 3′–5′ overhangs in the absence of dNTPs for a subsequent incorporation of biotin-labelled dNTPs. To biotinylate DNA fragment ends, the mixture of dNTPs was added to the reaction mix (2.5 µl of 10× T4 DNA Ligase buffer, 11.875 µl of nuclease-free water, 5 µl of 1 mM Biotin-dATP (Jena Bioscience, catalogue no. NU-835-BIO14), 5 µl of 1 mM Biotin-dCTP (Jena Bioscience, catalogue no. After 45 min of incubation at room temperature with interval mixing at 850 rpm, the reaction was stopped with 30 mM EDTA (pH 8.0) followed by incubation at 65 °C for 20 min without agitation. The chromatin from lysed cells and nuclei was pelleted at 10,000g for 10 min at 4 °C and washed twice with MB3 buffer (50 mM Tris-HCl (pH 7.5), 10 mM MgCl2). EL0012)) and incubated at room temperature for at least 2.5 h. To remove biotin from unligated ends, pelleted chromatin was treated with 2 µl of 100 U µl−1 Exonuclease III (NEB, catalogue no. M0206) for 5 min at 37 °C and agitation 850 rpm. Then, chromatin was decrosslinked and deproteinased overnight at 65 °C at 850 rpm in the presence of 350 mM NaCl, 1% SDS and 1 mg ml−1 proteinase K (Roche, catalogue no. D4014) and eluted in 50 µl of 10 mM Tris-HCl (pH 8.0) (Extended Data Fig. Next, biotinylated proximity ligated DNA fragments were captured with Dynabeads MyOne Streptavidin (Life Technologies, catalogue no. DNA ends were prepared for adapter ligation and dA-tailed using NEBNext End repair/dA-tailing mix (NEB, catalogue no. The Y-shaped Illumina adapters were ligated with NEBNext Ultra II Ligation Module (NEB, catalogue no. E7595S), and the final library was amplified using NEBNext High-Fidelity 2× PCR Master Mix (NEB, catalogue no. The final libraries were double-size selected with Ampure XP (Beckman Coulter, catalogue no. The detailed Micro-C stepwise protocol is reported in Supplementary Text 1. Genomic DNA (gDNA) from C. owczarzaki (Cowc) strain ATCC30864 was extracted with Blood & Cell Culture DNA Mini Kit (Qiagen, catalogue no. The library was constructed by the use of Ligation Sequencing Kit (Oxford Nanopore, catalogue no. SQK-LSK109) and NEBNext Companion Module (NEB, catalogue no. E7180), and sequenced with the R9.4.1 Flow Cell set on a MinION device (Oxford Nanopore). We obtained 4.3 M reads with an estimated Oxford Nanopore N50 of 5.4 kb. E. muelleri gDNA was isolated using the Nanobind Tissue (Circulomics, catalogue no. NB-900-701-01) from 177 mg of frozen tissue of clonal juvenile sponges hatched from overwintering cysts (gemmules). Gemmules were obtained from the head tank of the Kapoor Tunnel (Sooke Reservoir), part of the drinking water system of the city of Victoria, British Columbia, Canada21. Short DNA fragments of less than 10 kb were removed with Short Read Eliminator Kit (Circulomics, catalogue no. gDNA was quantified with a Qubit fluorometer and sequenced on an Oxford Nanopore using a PromethION flow cell (R9.4), producing 5.31 million reads with an estimated Oxford Nanopore N50 of 18.97 kb. To reduce the level of heterozygosity during the assembly of M. leidyi genome (below), an animal culture was established from a single individual through self-fertilization. High molecular weight DNA was isolated from 5–8 animals (3–5 cm) starved for 24 h before flash-freezing. Frozen tissues were powdered with mortar and pestle, dissolved in 10 ml of urea extraction buffer (50 mM Tris-HCl (pH 8.0), 7 M Urea, 312.5 mM NaCl, 20 mM EDTA (pH 8.0), 1% w/v N-lauroylsarcosine sodium salt) as described in ref. 66 and incubated for 10 min at room temperature on a rocking platform 20 rpm. gDNA was then purified twice with a phenol-chloroform-isoamyl alcohol mixture pH 7.7–8.3 (Sigma-Aldrich, catalogue no. Finally, the isolated DNA was subjected to another round of purification with Nanobind Tissue kit (Circulomics, catalogue no. NB-900-701-01), followed by short-read elimination with the Short Read Eliminator Kit (Circulomics, catalogue no. Sequencing was performed on Oxford Nanopore using PromethION flow cell (R9.4). We obtained 4.54 million reads with an estimated Oxford Nanopore N50 of 36.84 kb. ATAC-seq libraries from M. leidyi and from sorted choanocytes of E. muelleri were prepared using Omni-ATAC protocol as described previously67. Briefly, two M. leidyi adult specimens were dissociated using CMFSW with 0.25% α-Chymotrypsin (Sigma-Aldrich, catalogue no. To isolate nuclei, dissociated cells were transferred into cold hypotonic ATAC lysis buffer adjusted for marine animals (10 mM Tris-HCl (pH 7.5), 35 mM NaCl, 3 mM MgCl2, 0.1% Tween-20, 0.01% NP-40, 0.01% digitonin, 70 µM Pitstop (Abcam, AB1206875MG)). Nuclei were then pelleted and resuspended in cold PBS buffer with 0.8 M Sorbitol. We used 50,000 nuclei per each tagmentation reaction. To sort choanocytes of E. muelleri, 7 days posthatching sponges were fed with 0.5 µm fluorescent carboxylate-modified FluoSpheres (Invitrogen, catalogue no. After 10 min of incubation, sponges were washed twice with Strekal's media, collected and dissociated for 15 min at 28 °C using Protease XIV (Sigma-Aldrich, catalogue no. P5147) in Strekal's media and 1 mM CaCl2. Cells were pelleted at 800g for 5 min at room temperature and resuspended in Strekal's media with 2 mM EDTA (pH 8.0). Further dissociation and trituration of sponge tissue continued for another 15 min at room temperature. Cells were filtered through a 40-µm cell strainer, stained with 2 µg ml−1 Hoechst 33342 and sorted using FACS. For each tagmentation reaction we used 100,000 nuclei. ATAC-seq libraries were prepared as described previously67 and sequenced on Illumina NextSeq 500 using High-Output 75 cycles. For S. arctica, C. owczarzaki, S. rosetta, M. leidyi, E. muellleri, T. adhaerens, C. collaboinventa and N. vectensis, double-crosslinked cells, as above, were washed with PBS, resuspended in 500 µl of cell lysis buffer (20 mM HEPES (pH 7.5), 10 mM NaCl, 0.2% IGEPAL CA-630, 5 mM EDTA, protease inhibitors cocktail) and incubated on ice for 10 min. The resulting pellets were resuspended in bead beating buffer (20 mM HEPES (pH 7.5), 10 mM NaCl, 5 mM EDTA, protease inhibitors cocktail), and then transferred to 0.2 ml tubes containing acid-washed glass beads (Sigma-Aldrich, G8772). Cells were lysed by vortexing five times for 30 s. The supernatant was transferred to a 1.5-ml sonication tube, SDS was added to 0.6% and samples were sonicated 3–5 cycles of 30 s on, 30 s off in a Bioruptor Pico (Diagenode) to generate 200–300 bp fragments. Chromatin was diluted with 5 volumes of dilution buffer (20 mM HEPES (pH 7.5), 140 mM NaCl), centrifuged at 16,000g for 10 min at 4 °C and stored at −80 °C before use. Chromatin immunoprecipitation was performed as previously described68 with the following modifications. The pool of chromatin was incubated for 14–16 h at 4 °C with 5 µl (1:50 dilution) of anti-H3K4me1 (Cell Signaling, catalogue no. 5326), 6 µl (3.4 µg) of anti-H3K4me2 (Abcam, catalogue no. ab32356), 2.5 µl (1:100 dilution) of anti-H3K4me3 (Millipore, catalogue no. 07-473), 5 µl (5 µg) of anti-SMC1 (Thermo Fisher, A300-055A) or 2 µl (2 µg) of anti-H3 (Abcam, catalogue no. 16-661) and Protein G (Sigma-Aldrich, catalogue no. Immunoprecipitated complexes were washed, reverse crosslinked for 3 h at 68 °C, deproteinased and then purified using Ampure XP beads (Beckman Coulter, catalogue no. Final libraries were prepared using the NEBNext Ultra II DNA Library Prep Kit (New England BioLabs) according to the manufacturer's protocol. ChIP–seq libraries were sequenced on Illumina NextSeq 500 sequencer using High-Output 75 cycles. Single-cell libraries were prepared from freshly dissociated and sorted choanocytes of E. muelleri as previously described8. To collect cells for MARS-seq libraries, 7 days posthatching, sponges were fed with 0.5 µm fluorescent carboxylate-modified FluoSpheres (Invitrogen, catalogue no. Animal tissues were dissociated and prepared for sorting as described above for ATAC-seq. Dissociated cells were sorted through FACS into four 384-well MARS-seq plates. In total, 1,536 single-cell libraries were prepared and sequenced on an Illumina NextSeq 500 using High-Output 75 cycles. Chromatin proteomics samples were prepared as previously descibed69 with minor modifications. Briefly, double-crosslinked cells of M. leidyi (1 million per replicate) were solubilized in 1 ml of lysis buffer (4 M guanidine thiocyanate, 100 mM Tris-HCl (pH 8.0), 10 mM EDTA, 2% N-lauroylsarcosine sodium salt) and incubated for 10 min. Next, before adding DNA-binding beads (Invitrogen, catalogue no. 37002D), cell lysate was mixed with 1 ml of 2-propanol. The beads were washed using 1 ml of wash buffer (1:1 lysis buffer to 2-propanol ratio), transferred to a 1.5-ml sonication tube and washed again with 1 ml of 80% ethanol. The chromatin was then eluted in 200 µl of 10 mM Tris-HCl (pH 8.0) containing proteinase inhibitors (Roche, catalogue no. 04693132001) and sonicated using a Bioruptor Pico at 4 °C for 3 cycles (30 s ON, 30 s OFF). To remove RNA-binding proteins, RNase A (Roche, catalogue no. 10109142001) was added to the sonicated samples, which were then incubated at 37 °C with agitation in a thermomixer. Afterwards, chromatin was re-bound to the beads by adding 250 µl of lysis buffer, vortexing and then sequentially adding 300 µl of 2-propanol. The beads were washed twice with 1 ml of 80% ethanol, and proteins were digested on the beads using trypsin (Promega, catalogue no. Samples were analysed using an Orbitrap Eclipse mass spectrometer (Thermo Fisher Scientific) coupled to an EASY-nLC 1200 (Thermo Fisher Scientific (Proxeon)). Peptides were loaded directly onto the analytical column and were separated by reversed-phase chromatography using a 50-cm column with an inner diameter of 75 μm, packed with 2-μm C18 particles (Thermo Fisher Scientific, catalogue no. Chromatographic gradients started at 95% buffer A (0.1% formic acid in water) and 5% buffer B (0.1% formic acid in 80% acetonitrile) with a flow rate of 300 nl min−1 and gradually increased to 25% buffer B and 75% A in 52 min and then to 40% buffer B and 60% A in 8 min. After each analysis, the column was washed for 10 min with 100% buffer B. The mass spectrometer was operated in positive ionization mode with nanospray voltage set at 2.4 kV and source temperature at 305 °C. The acquisition was performed in data-dependent acquisition mode and full mass spectrometry scans with one micro-scan at resolution of 120,000 were used over a mass range of m/z 350–1,400 with detection in the Orbitrap mass analyser. In each cycle of data-dependent acquisition analysis, following each survey scan, the most intense ions above a threshold ion count of 10,000 were selected for fragmentation. The number of selected precursor ions for fragmentation was determined by the ‘Top Speed' acquisition algorithm and a dynamic exclusion of 60 s. Fragment ion spectra were produced by means of high-energy collision dissociation at normalized collision energy of 28% and they were acquired in the ion trap mass analyser. Automatic gain control and injection time were set to ‘Standard' and ‘Dynamic', respectively, and an isolation window of 1.4 m/z was used. Digested bovine serum albumin (NEB, catalogue no. P8108S) was analysed between each sample to avoid sample carryover and to assure stability of the instrument, and Qcloud70 was used to control instrument longitudinal performance during the project. Acquired spectra were analysed using the Proteome Discoverer software suite (v.2.5, Thermo Fisher Scientific) and the Mascot search engine (v.2.6, Matrix Science71). The data were searches against M. leidyi database and a list of common contaminants (16,042 entries)72 as well as all the corresponding decoy entries. For the peptide identification a precursor ion mass tolerance of 7 ppm was used for the MS1 level, trypsin was chosen as enzyme and up to three missed cleavages were allowed. The fragment ion mass tolerance was set to 0.5 Da for MS2 spectra. Oxidation of methionine and N-terminal protein acetylation were used as variable modifications whereas carbamidomethylation on cysteines was set as a fixed modification. False discovery rate in peptide identification was set to a maximum of 1%. Peptide quantification data were retrieved from the ‘Precursor ions quantifier' node from Proteome Discoverer (v.2.5) using 2-ppm mass tolerance for the peptide extracted ion current. The obtained values were used to calculate protein fold-changes and their corresponding P value and adjusted P values. The DNA-binding domains of candidate zf-C2H2 proteins were cloned from complementary DNA (cDNA) library of M. leidyi into the pIX-HALO vector using NEBuilder HiFi DNA Assembly Master Mix (NEB, catalogue no. The obtained HALO-fusion constructs were translated using the TnT SP6High-Yield Wheat Germ Protein Expression System (Promega, catalogue no. Next, an adapter-ligated DNA library was prepared from native gDNA of M. leidyi using NEBNext Ultra II FS DNA library prep kit (NEB, catalogue no. The binding to HALO-zf-C2H2 fusion proteins and recovery of adapter-ligated gDNA libraries was performed as described in ref. The generated DAP-seq libraries were sequenced in paired-end mode on an Illumina NextSeq 500 using High-Output 75 cycles. We made preliminary genome assemblies of C. owczarzaki from Oxford Nanopore reads basecalled by Guppy v.6.0.1 using NextDenovo v.2.5.0 (ref. 76), which produced 20, 141 and 56 contigs including the mitochondrial genome, respectively. For the Flye assembly, we only used 5,000 bp or longer reads. We then integrated the three assemblies by manually comparing them to each other, with a help of reciprocal large-scale alignments generated with minimap2 (ref. The integrated assembly was polished with the Nanopore reads using Flye ten times, and with Illumina reads78 using HyPo79 twice. A chromosome-scale duplication, which was in the end included in chromosome 15 after the 3D assembly (Extended Data Fig. 2d), was temporarily removed before annotating the genome. Finally, we manually inspected the whole assembly sequence together with the mapped Illumina data, Nanopore data and the previous Sanger sequence data25, and navigated them on the Integrative Genomic Viewer (IGV)80 to find and fix errors occurred during the consensus calling. We also manually phased chimeric haplotypes for some genes using the long reads. In total, 7,937 nucleotides were manually inserted or deleted at 430 sites and 1,193 nucleotides were substituted at 1,081 sites. We produced two new genome assemblies for E. muelleri and M. leidyi. In both cases, we used Oxford Nanopore reads after base call correction using Guppy v.5.0.17 (using the dna_r9.4.1_450bps_sup_prom.cfg configuration the super-accurate base calling model, and a filtering reads with min_qscore=10). Then, we used two different long-read assemblers (Flye v.2.9-b1768, ref. 81) and various assembly strategies (filtering by read length at 0, 10 and 50 kb), and selected the best resulting draft assemblies for each species. To that end, we evaluated the contiguity (measured using the contig N50), completeness and occurrence of uncollapsed haplotypes for each draft (Extended Data Fig. Contiguity was evaluated using total assembly length and contig N50. Completeness was measured with the fraction of conserved orthologues recovered by BUSCO v.5.1.2 (ref. The presence of uncollapsed haplotypes was assessed with the distribution of per-base sequencing depths, calculated using the pbcstat utility in purge_dups v.1.2.5 (ref. 77), using the -x map-ont preset for long-read mapping) (Extended Data Fig. The best drafts for each species were produced using the following parameter combinations: (1) for E. muelleri, we used the Shasta assembler with the Nanopore configuration (--config Nanopore-Oct2021 flag), without filtering by read length (estimated sequencing depth roughly 100×) and (2) for M. leidyi, we used Flye with reads filtered at 50 kb (estimated sequencing depth roughly 150×), the raw Nanopore read configuration (--nano-raw flag) and an estimated total assembly size of 200 Mb. Then, we used purge_dups to collapse putative uncollapsed haplotypes in each assembly, in the following manner: (1) we split the assembly into contigs with the split_fa utility; (2) we aligned the genome to itself with minimap2 and the -x asm5 preset; (3) we used the read alignments to the unsplit assembly (produced with minimap2 -x map-ont) to obtain the sequencing depth histogram and calculate coverage cutoffs with pbcstat and calcuts, respectively; (4) we used these cutoffs and the mapped reads to remove haplotigs and overlaps for the draft, with purge_dups proper and using two rounds of alignment chaining (-2 flag) and finally (5) we reevaluated the assembly quality using per-base sequencing depth distributions (above) and reductions in the fraction of duplicated BUSCO orthologues. To obtain chromosome-level genome assemblies, generated Micro-C libraries were mapped to de novo draft genome assemblies (C. owczarzaki, E. muelleri and M. leidyi) or current genome assemblies (T. adhaerens ASM15027v1, ref. The resulting assemblies were manually reviewed and corrected with Juicebox Assembly Tools88 (Extended Data Fig. Finally, chromosome-level genome assemblies were polished with Medaka (v.1.5.0) to correct possible sequence errors such as indels and mismatches, as follows: (1) first, we mapped the Nanopore reads to the chromosome-level assembly using the minimap2-based mini_align utility; (2) we then used Medaka consensus to obtain consensus sequences, specifying a batch size of 200 (--batch 200 flag) and the r941_prom_sup_g507 configuration (--model flag) and (3) we merged the consensus and variant calls for all chromosomes into a polished assembly using Medaka stitch. To annotate the C. owczarzaki genome, we did not mask the repeats because the intergenic regions are very small25 and, thus, masking only increased annotation failure on duplicated genes. The three preliminary annotations, evidenced by metazoan proteins, protozoan proteins and RNA-seq data, were combined with TSEBRA91, giving rise to 9,069 annotated transcripts. Finally, we manually searched and fixed wrong annotations by navigating the assembly on IGV80, comparing the combined annotation with the three preliminary annotations together with the mapped RNA-seq data. By this careful inspection, we modified or newly annotated 1,871 transcripts including alternatively spliced ones. Compared to the previously published proteome25 (v.2), only 4,076 out of 8,792 proteins (including alternatively spliced ones) had completely matched sequences to the those predicted in this study, allowing simple amino acid mismatches probably accounting for polymorphisms. To annotate M. leidyi genome we first downloaded developmental Illumina RNA-seq samples (GSE93977), trimmed them with fastp and built a de novo Trinity assembly, which was mapped to the genome using gmap92. The RNA-seq was also directly mapped to genome using HISAT2 (ref. 93) with the –dta parameter, and genome-based transcriptomes were built for each sample using StringTie94. Merged mapped RNA-seq samples were then used to find high-quality intron junctions using Portcullis. The combination of Trinity, StringTie and Portcullis intron junctions were then fed to Mikado for transcript selection. The best resulting gene models based on mapping to UniProt were then used to train an Augustus model for M. leidyi. Augustus was used for an ab initio gene prediction, using exonic hints from Mikado, intron hints from Portcullis and coding sequence hints from a MetaEuk95 run with query fasta files combining proteins from H. californensis and UniProt. Mikado transcripts and Augustus gene models were then merged using EVidenceModeler (scores of 10 for Mikado transcripts and 2 for Augustus gene models). To annotate S. arctica, S. rosetta, E. muelleri, T. adhaerens and C. collaboinventa genome assemblies, gene models from previous assemblies were mapped onto new coordinates using Liftoff (v.1.6.1)83 with -overlap 1 -flank 1 options. Repetitive sequences and transposable elements were annotated using EDTA (v.2.1.0)97 with the following parameters: --sensitive 1 --anno 1 (Extended Data Fig. For H. sapiens, we used RepeatMasker (v.open-4-0-3) annotation of GRCh38 genome released by UCSC. The fast5 files obtained from the PromethION were used as input for Megalodon (v.2.5), with the Remora model dna_r9.4.1_e8 sup for 5hmc_5mc modification only on CG dinucleotides. We then built bigwig files using the bedGraphToBigWig tool from UCSC. The Megalodon CG methylation calls were compared to previously published Whole-Genome Bisulfite Sequencing remapped to the new reference genomes using Bismark (SRR8346013 and SRR10356110)21,48. Both data sources were congruent, yet Nanopore had deeper and broader coverage, we used Megalodon methylation data for subsequent analysis. Micro-C data were processed using the 4D Nucleome processing pipeline98. Briefly, raw reads were mapped to the reference genome using bwa mem (v.0.7.17-r1188) with the -SP5M option. The mapped reads were sorted and filtered with pairtools (v.0.3.0)99. Pairs that mapped within a 2-bp distance from each other were considered duplicates. We also discarded reads mapping within the distance of 200 bp, which eliminates self-ligated pairs and reads mapping to adjacent nucleosomes. Only uniquely mapping pairs and 5′ most unique alignments of multiple ligations pairs were aggregated into 200-bp bin contact matrices and multiresolution .cool or .hic files (Extended Data Fig. Contact matrices were normalized with cooler (v.0.8.11)100 using the iterative correction and eigenvector (ICE) balancing method101 for .cool files or with Juicer tools86 using Knight–Ruiz balancing102 for .hic files. All contact heatmaps were visualized with either Cooltools (v.0.5.1)103 or Coolbox (v.0.3.8)104 and genome assembly heatmaps were visualized using HiGlass105. Reproducibility between replicates was estimated using the stratum-adjusted correlation coefficient (SCC) implemented in HiCRep106 at resolutions of 1, 2, 5, 10, 25 and 50 kb (Extended Data Fig. The SCC scores were averaged across chromosomes. Technical replicates were first merged, deduplicated and only then combined into the final contact maps. The decay of the average contact frequency over genomic distance from 1 kb to 100 Mb was calculated using Cooltools (v.0.5.1)103. The decay curves were calculated for each chromosome separately, and then averaged across chromosomes (Extended Data Fig. Compartment analysis was performed on observed-over-expected contact maps at resolutions equivalent to 5,000, 10,000, 20,000 and 50,000 bins per species genome (Extended Data Fig. We visually examined calculated eigenvectors, and, for each organism, the E1 vector corresponded to the compartmentalization pattern of contact maps. Active (A) and inactive (B) compartment types were assigned by GC content (for all species except C. owczarzaki) or H3K4me3 chromatin signal for C. owczarzaki, such that higher GC regions or positively correlated with H3K4me3 signal regions correspond to A compartment. Specifically, the eigenvectors were sorted from lowest to highest value and combined into 40 groups according to their eigenvector value. The first (bottom 2.5% E1 values) and last (top 2.5% E1 values) groups were ignored to exclude potential outliers. The observed-over-expected value of the remaining 38 groups was averaged across all bins and chromosomes and visualized as saddle plots. Compartment strength was calculated as the ratio of homotypic (AA + BB) over heterotypic (AB + BA) compartment contacts. We choose the top 20% of observed-over-expected values for both homotypic and heterotypic interactions. To assess the error in estimating compartment strength, we compared the compartments strength across different resolutions as well as performed visual inspection of the contact maps (Extended Data Fig. The latter showed varying degrees of accuracy in identifying compartment types between species, with the algorithm performing particularly poorly on M. leidyi due to the lack of well-defined chromatin compartmentalization in this species in our sample. Therefore, we assigned an extra intermediate compartment I to the intermediate eigenvalues close to zero. To that end, we modelled the genome-wide eigenvalues distribution as a Gaussian mixture with three components using the normalmixEM function from the mixtools R package (v.2.0.0) as described107. The B–I and I–A thresholds were defined as intersection points between components (Extended Data Fig. To characterize the distribution of genomic features in the A, I and B compartments, we calculated cumulative H3K4me3 chromatin signals and RNA-seq expression values for each compartment region. Furthermore, we estimated the percentage of bases annotated as transposable elements or coding gene regions within these compartments. To compute the insulation profiles, we first determined the optimal resolution and window sizes for a target genome. To that end, we calculated insulation scores using Cooltools insulation module103 at resolutions roughly equivalent to 50,000, 100,000, 200,000 and 400,000 genomic bins per species genome with a sliding window for each resolution that is ×5, ×10 and ×25 the applied resolution (Extended Data Fig. The resolution and two window sizes with maximum average insulation scores were considered optimal because they reflected the strongest partitioning of genomes into isolated domains. Insulation boundaries located within two bins of unmappable genomic region were removed. Identified insulation boundaries were categorized into strong and weak using the peak prominence of their boundary strength distributions (Li threshold) as implemented in the Cooltools insulation score module. Strong boundaries were further annotated with overlapping genomic features that fall within one bin of the annotated feature from the insulation boundary. For example, if compartment boundaries were called at the resolution 5 kb, then the maximum distance to the closest insulation boundary is ±10 kb. To estimate internal interactions within contact domains, rescaled pile-ups were generated using coolpup.py108. Contact domains were defined as valleys between two strongly insulated regions, which were not further from each other than 100 kb. Chromatin loops were identified using SIP v.1.6.1 (ref. In M. leidyi, the SIP peak caller was applied with the following parameters: -norm KR -g 3.0 -min 2.0 -max 2.0 -mat 5000 -d 10 -res 400 -sat 0.01 -t 2000 -nbZero 6 -factor 4 -fdr 0.05 -isDroso false. For T. adhaerens, chromatin loops were called with the following parameters: -norm KR -g 5.0 -min 4.0 -max 4.0 -mat 5000 -d 20 -res 100 -sat 0.01 -t 2000 -nbZero 6 -factor 4 -fdr 0.05 -isDroso false; for C. collaboinventa: -norm VC_SQRT -g 1.5 -min 3.0 -max 3.0 -mat 5000 -d 20 -res 500 -sat 0.01 -t 2000 -nbZero 6 -factor 2 -fdr 0.05 -isDroso false; for H. californensis: -norm KR -g 2.5 -min 3.0 -max 3.0 -mat 5000 -d 10 -res 1000 -sat 0.01 -t 2000 -nbZero 6 -factor 4 -fdr 0.05 -isDroso false. Identified loops were then filtered based on APSscore, removing high-intensity signals outside the normal distribution of APSscore values. This threshold ensured accurate removal of false positive regions that corresponded to structural genomic rearrangements, such as inversions or assembly artefacts. For H. californensis, we kept only annotated loops with values greater than ten. Chromatin loops in N. vectensis and focal chromatin contacts in E. muelleri were annotated manually. Each loop anchor was assigned a promoter or enhancer identity based on their epigenetic signature. We calculated quantile normalized counts per million (CPM) coverage of H3K4me3, H3K4me2 and H3K4me1 ChIP signals in 1-kb (T. adhaerens, C. collaboinventa, N. vectensis), 2-kb (M. leidyi, E. muelleri, D. melanogaster) or 10-kb (H. sapiens) windows from a centre of a loop anchor. METALoci110 (v.0.3.0) analysis was applied to explore the spatial distribution and autocorrelation of epigenetic signal in T. adhaerens contact maps. For each region of interest at 800 bp resolution, the top 20% pairs of contacts were used to create a two-dimensional graph layout by means of the Kamada–Kawai algorithm111 (Fig. 3b, top left panel) using the ‘metaloci layout' with default parameters. Next, the signal of interest (H3K4me3 ChIP, ATAC, genic exon annotation) measured in the 800-bp genomic bin was mapped onto the built graph layout using ‘metaloci lm' with default parameters. Spatial and epigenetic signals were embedded into Voronoi diagrams for enhanced visualization as a Gaudí plot (Extended Data Fig. 8c), and the local Moran's index (LMI) analysis112,113 was applied for each bin of the Gaudí plot. According to LMI analysis, each bin is assigned to one of the four distinct groups, called LMI quadrants, based on the signal value in a bin and average signal value in its neighbourhood. If a bin and its neighbourhood have similar amounts of signal (low or high), then this bin is assigned to a low–low (blue) or high–high (red) quadrant. If a bin and its neighbourhood have different amounts of signal, then the bin is assigned to a low–high (cyan) or high–low (orange) quadrants, respectively. Significantly colocalized bins according to LMI, in which a P value is obtained using a permutation test, are highlighted by colour in the LMI scatterplots (Extended Data Fig. An analogous colouring scheme is applied to the Voronoi diagrams of the Gaudí plots. Hence, the highlighted blue and red bins on a Gaudí plot represent bins in which the signal is significantly colocalized in the space. Thus, ATAC-seq, H3K4me3 and motif score signals are significantly enriched inside the nested focal contacts (Fig. 8c), whereas exons are significantly enriched outside loop contacts (Fig. METALoci code is available at the GitHub repository (https://github.com/3DGenomes/METALoci). Loop anchor regions of M. leidyi (n = 8,523) and H. californensis (n = 478) were scanned for enriched motifs with HOMER114 in de novo motif discovery mode. As background sequences, we used random genomic regions of equivalent size and GC content (n = 38,810 in M. leidyi and n = 49,097 in H. californensis). For motif enrichment analysis in T. adhaerens (n = 3,557) and C. collaboinventa (n = 4,037), we scanned loop anchor regions using random genomic sequences of equivalent size (n = 32,004 in T. adhaerens and n = 36,178 in C. collaboinventa) as background. Loop anchor regions of N. vectensis (n = 327) were scanned for enriched motif using random genomic sequences (n = 45,268) of equivalent size and GC content as background. In addition, we used ATAC-seq accessible neuronal promoter regions (n = 22,961) as background to scan for enriched motifs in genomic regions that overlap ATAC-seq peaks located at the non-loop insulation boundaries (n = 9,016). To annotate genomic regions with identified motifs, we used the monaLisa package115, selecting percentile threshold of motif scores by comparing the motif score distributions in the target regions with genome-wide motif score distributions (Extended Data Figs. We evaluated the degree of sequence conservation of the M. leidyi genome by comparing it to other ctenophores (B. microptera, P. bachei and Hormiphora californiensis). To that end, we first aligned all genomes to each other using Cactus v.2.6.4 (ref. 116), following a progressive approach guided by the species trees of ctenophores, namely: ((M. leidyi, B. microptera), (H. californiensis, P. bachei)). 117) to create MAF (multiple alignment format) alignments of each chromosome, using M. leidyi as reference. To identify conserved regions in these genomes, we used the rphast v.1.6.1 implementation of the Phast toolkit118, as follows: (1) we used phyloFit119 to create an initial null model of neutral change based on the fourfold degenerate codon positions of each genome's coding regions, using a general reversible nucleotide transition matrix and the predefined species tree; (2) we used phastCons to optimize this model using the expectation–maximization procedure, re-estimating transition probabilities and tree parameters at each step (the optimization step was performed using only the longest chromosome in each genome). We evaluated the degree of syntenic conservation of the loop regions in M. leidyi compared to the other ctenophore genomes, and compared it to that of length-matched regions not involved in loops. To that end, we first identified orthologous genes across the four ctenophore species (M. leidyi, B. microptera, P. bachei and H. californensis) using Broccoli v.1.2 (ref. 120) to obtain orthologous gene pairs (step 4), using predicted peptide sequences as input (longest isoform per gene only). Within Broccoli, we used the maximum-likelihood gene tree inference algorithm (based on IQ-TREE121) and set a k-mer length of 10,000 to avoid the removal of paralogous sequences from the analysis. Second, we mapped pairs of loop anchor regions from M. leidyi (2,353 pairs of promoter–enhancer and 99 promoter–promoter loops, n = 2,452 in total) to their closest overlapping genes, and used these genes and their orthologs as anchors to map these regions to the other ctenophore genomes. In parallel, we randomly selected length-matched, non-loop overlapping regions from the M. leidyi genome to compare their synteny conservation with that of loop regions (using the randomizeRegions function in the regioneR R package122 to select 3× background regions, n = 7,356). Then, for each pair of species, we evaluated the synteny conservation of the foreground (loop) and background regions (random non-loops) from the point of view of the flanking synteny-anchoring genes, using two different metrics: (1) the fraction of shared orthologous genes between the flanking genes across all regions in the foreground and background sets (testing the significance of the difference using a χ2 test for given probabilities) and (2) the distributions of per-region shared orthologs (tested using the one-sided Wilcoxon rank sum test with continuity correction). We used previously published datasets of ATAC-seq for C. owczarzaki17, S. rosetta39, T. adhaerens123, C. collaboinventa123, D. melanogaster124 and H. sapiens125 as well as newly generated datasets for N. vectensis, M. leidyi and E. muelleri. Sequenced reads were demultiplexed and converted to fastq files using bcl2fastq v.2.20 Illumina. Raw reads were filtered and trimmed with Trimmomatic (v.0.39)126 before mapping to the reference genome with bwa mem (v.0.7.17-r1188) and duplicates were marked with bamsormadup from biobambam2 (https://github.com/gt1/biobambam2). Using deeptools alignmentSieve aligned reads were filtered and shifted with -ATACshift, which corresponds to mate reads being shifted +4 and −5 bp for positive and negative strands, respectively. To generate nucleosome-position data tracks, nucleosome-free and nucleosome-bound regions were defined using the following length thresholds 0–120 and 150–240 bp, respectively. ATAC peaks were called with MACS2 (ref. Footrpint ATAC score was calculated using TOBIAS v.0.13.3 (ref. We analysed publicly available dataset for D. melanogaster129 and H. sapiens130 and 34 newly generated ChIP–seq datasets as described below. Raw reads after removal of 3′-adapters and quality filtering with Trimmomatic (v.0.39)126 were aligned to the reference genome with bwa mem (v.0.7.17-r1188). Aggregated density plots were visualized with deeptools (v.3.1.3)131. Raw reads from amplified and native gDNA fragments bound by HALO-zf-C2H2 protein fusions were analysed as described for ChIP–seq. Motif enrichment analysis was performed using HOMER114 in de novo motif discovery mode for MACS2 identified narrow peaks resized to 300 bp (for CTEP1 n = 14,638; for CTEP2 n = 10,615). GC- and size-normalized random genomic regions were used as background (for CTEP1 n = 25,964; for CTEP2 n = 30,744). We used previously published datasets of bulk poly-A enriched RNA-seq for S. arctica132, C. owczarzaki61, S. rosetta39, D. melanogaster124 and H. sapiens133 (Supplementary Table 2). To process data, raw reads were aligned to the reference genome using STAR (v.020201)134 in --quantMode to estimate the number of read counts per gene. Single-cell MARS-seq libraries generated previously8,135 were aligned to new reference genomes of E. muelleri (GCA_049114765.1), M. leidyi (GCA_048537945.1) and N. vectensis (GCA_932526225.1) using Liftoff or de novo annotated gene models. To improve single-cell RNA-seq quantification, gene annotations for E. muelleri and M. leidyi have been extended using GeneExt136. Briefly, MARS-seq alignment files have been subsampled to 100 M reads and MACS2 (ref. 127) was used to call peaks using default parameters. Intergenic peaks were filtered based on the 20th percentile of the genic peak coverage and each gene was extended to the most distant peak within 5,000 nucleotides. Metacell and clustering analyses were performed as previously described8. The single-cell expression atlas for T. adhaerens was obtained from a previously published dataset123. ATAC-seq, ChIP–seq and RNA-seq datasets were analysed as described above. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw and processed high-throughput sequencing data are available in a Genome Expression Omnibus (GEO) repository under accession number GEO GSE260572. Raw proteomics data have been deposited to the PRIDE101 repository with the dataset identifier PXD056500. Furthermore, sequenced and assembled genome sequences, genome annotations and genomic intervals used in this study, such as chromatin loop anchors, insulation boundaries and compartmentalization domains are also available on GitHub (https://github.com/sebepedroslab/early-metazoa-3D-chromatin). In addition, datasets can be explored in interactive genome browsers137 for each species at A.S.-P.'s laboratory site (https://sebelab.crg.eu/3d-genomes-arc-jb2). Scripts to reproduce the data processing and downstream analysis are available at GitHub (https://github.com/sebepedroslab/early-metazoa-3D-chromatin). Unless otherwise specified, scripts are based on R v.4.3.2 and Python v.3.7.7, and the language-specific libraries specified in the Methods section. Lieberman-Aiden, E. et al. Comprehensive mapping of long range interactions reveals folding principles of the human genome. Sexton, T. et al. Three-dimensional folding and functional organization principles of the Drosophila genome. & Phillips-Cremins, J. E. On the existence and functionality of topologically associating domains. Szabo, Q., Bantignies, F. & Cavalli, G. Principles of genome folding into topologically associating domains. Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. & Cavalli, G. Chromosomal domains: epigenetic contexts and functional implications of genomic compartmentalization. Acemel, R. D. & Lupiáñez, D. G. Evolution of 3D chromatin organization at different scales. Early metazoan cell type diversity and the evolution of multicellular gene regulation. Ancient gene linkages support ctenophores as sister to other animals. The origin of Metazoa: a unicellular perspective. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Brunet, T. & King, N. The origin of animal multicellularity and cell differentiation. Winick-Ng, W. et al. Cell-type specialization is encoded by specific chromatin topologies. Gaiti, F. et al. Landscape of histone modifications in a sponge reveals the origin of animal cis-regulatory complexity. Evolutionary conservation of the eumetazoan gene regulatory landscape. The dynamic regulatory genome of capsaspora and the origin of animal multicellularity. Simion, P. et al. A large and consistent phylogenomic dataset supports sponges as the sister group to all other animals. The genome of the ctenophore Mnemiopsis leidyi and its implications for cell type evolution. Kenny, N. J. et al. Tracing animal genomic evolution with the chromosomal-level assembly of the freshwater sponge Ephydatia muelleri. Putnam, N. H. et al. Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization. Dudin, O. et al. A unicellular relative of animals generates a layer of polarized cells by actomyosin-dependent cellularization. The Capsaspora genome reveals a complex unicellular prehistory of animals. Fairclough, S. R. et al. Premetazoan genome evolution and the regulation of cell differentiation in the choanoflagellate Salpingoeca rosetta. Batut, P. J. et al. Genome organization controls transcriptional dynamics during development. Diverse silent chromatin states modulate genome compartmentalization and loop extrusion barriers. Schwarzer, W. et al. Two independent modes of chromatin organization revealed by cohesin removal. Phase separation drives heterochromatin domain formation. Duan, Z. et al. A three-dimensional model of the yeast genome. Nand, A. et al. Genetic and spatial organization of the unusual chromosomes of the dinoflagellate Symbiodinium microadriaticum. Harris, H. L. et al. Chromatin alternates between A and B compartments at kilobase scale for subgenic organization. Weintraub, A. S. et al. YY1 is a structural regulator of enhancer-promoter loops. Heger, P., Marin, B., Bartkuhn, M., Schierenberg, E. & Wiehe, T. The chromatin insulator CTCF and the emergence of metazoan diversity. Rowley, M. J. et al. Evolutionarily conserved principles predict 3D chromatin organization. Levo, M. et al. Transcriptional coupling of distant regulatory genes in living embryos. Gahan, J. M. et al. Chromatin profiling identifies putative dual roles for H3K27me3 in regulating transposons and cell type-specific genes in choanoflagellates. Galitsyna, A. et al. Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation. Hansen, K. L. et al. Synergy between cis-regulatory elements can render cohesin dispensable for distal enhancer function. Cazet, J. F. et al. A chromosome-scale epigenetic map of the Hydra genome reveals conserved regulators of cell state. The energetics and physiological impact of cohesin extrusion. Lam, E. Y. N., Beraldi, D., Tannahill, D. & Balasubramanian, S. G-quadruplex structures are stable and detectable in human genomic DNA. Li, L. et al. YY1 interacts with guanine quadruplexes to regulate DNA looping and gene expression. Dejosez, M. et al. Regulatory architecture of housekeeping genes is driven by promoter assemblies. Goel, V. Y., Huseyin, M. K. & Hansen, A. S. Region Capture Micro-C reveals coalescence of enhancers and promoters into nested microcompartments. de Mendoza, A. et al. Convergent evolution of a vertebrate-like methylome in a marine sponge. Yin, Y. et al. Impact of cytosine methylation on DNA binding specificities of human transcription factors. Architectural proteins Pita, Zw5, and ZIPIC contain homodimerization domain and support specific long-range interactions in Drosophila. Schultz, D. T. et al. A chromosome-scale genome assembly and karyotype of the ctenophore Hormiphora californensis. & Rando, O. J. Micro-C XL: assaying chromosome conformation from the nucleosome to the entire genome. Sun, L. et al. Mapping nucleosome-resolution chromatin organization and enhancer-promoter loops in plants using Micro-C-XL. & Mirny, L. Spandrels of the cell nucleus. Waves of retrotransposon expansion remodel genome organization and CTCF binding in multiple mammalian lineages. Martín-Durán, J. M. et al. Conservative route to genome compaction in a miniature annelid. Emergence of novel cephalopod gene regulation and expression through large-scale genome reorganization. & Gambetta, M. C. The role of insulation in patterning gene expression. Harmston, N. et al. Topologically associating domains are ancient features that coincide with Metazoan clusters of extreme noncoding conservation. Regulated aggregative multicellularity in a close unicellular relative of metazoa. & Gibcus, J. H. Hi-C 3.0: improved protocol for genome-wide chromosome conformation capture. Curtis, A. S. G. & Vyver, G. The control of cell adhesion in a morphogenetic system. The ancestral gene repertoire of animal stem cells. & Rentzsch, F. Nervous systems of the sea anemone Nematostella vectensis are generated by ectoderm and endoderm and shaped by distinct mechanisms. Zimmermann, B. et al. Topological structures and syntenic conservation in sea anemone genomes. Corces, M. R. et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Chromatin state dynamics during blood formation. Rafiee, M.-R. et al. Chromatin-contact atlas reveals disorder-mediated protein interactions and moonlighting chromatin-associated RBPs. Chiva, C. et al. QCloud: a cloud-based quality control system for mass spectrometry-based proteomics laboratories. Perkins, D. N., Pappin, D. J. C., Creasy, D. M. & Cottrell, J. S. Probability-based protein identification by searching sequence databases using mass spectrometry data. Beer, L. A., Liu, P., Ky, B., Barnhart, K. T. & Speicher, D. W. in Serum/Plasma Proteomics: Methods and Protocols (eds Greening, D. W. & Simpson, R. Bartlett, A. et al. Mapping genome-wide transcription-factor binding sites using DAP-seq. Hu, J. et al. NextDenovo: an efficient error correction and accurate assembly tool for noisy long reads. Assembly of long, error-prone reads using repeat graphs. Efficient assembly of nanopore reads via highly accurate and intact error correction. Li, H. Minimap2: pairwise alignment for nucleotide sequences. Revision of the Capsaspora genome using read mating information adjusts the view on Premetazoan genome. & Sung, W.-K. HyPo: Super fast & accurate polisher for long read genome assemblies. Robinson, J. T. et al. Integrative genomics viewer. Shafin, K. et al. Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes. Manni, M., Berkeley, M. R., Seppey, M., Simão, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. & Salzberg, S. L. Liftoff: accurate mapping of gene annotations. Guan, D. et al. Identifying and removing haplotypic duplication in primary genome assemblies. Tessler, M. et al. Phylogenomics and the first higher taxonomy of Placozoa, an ancient and enigmatic animal phylum. Durand, N. C. et al. Juicer provides a one-click system for analyzing loop-resolution Hi-C experiments. Dudchenko, O. et al. De novo assembly of the Aedes aegypti genome using Hi-C yields chromosome-length scaffolds. Durand, N. C. et al. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. Brůna, T., Hoff, K. J., Lomsadze, A., Stanke, M. & Borodovsky, M. BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database. Kuznetsov, D. et al. OrthoDB v11: annotation of orthologs in the widest sampling of organismal diversity. Gabriel, L., Hoff, K. J., Brůna, T., Borodovsky, M. & Stanke, M. TSEBRA: transcript selector for BRAKER. Wu, T. D. & Watanabe, C. K. GMAP: a genomic mapping and alignment program for mRNA and EST sequences. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Shumate, A., Wong, B., Pertea, G. & Pertea, M. Improved transcriptome assembly using a hybrid of long and short reads with StringTie. Levy Karin, E., Mirdita, M. & Söding, J. MetaEuk—sensitive, high-throughput gene discovery, and annotation for large-scale eukaryotic metagenomics. Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies. Ou, S. et al. Benchmarking transposable element annotation methods for creation of a streamlined, comprehensive pipeline. The 4D Nucleome Data Portal as a resource for searching and visualizing curated nucleomics data. Open2C et al. Pairtools: from sequencing data to chromosome contacts. Abdennur, N. & Mirny, L. A. Cooler: scalable storage for Hi-C data and other genomically labeled arrays. Imakaev, M. et al. Iterative correction of Hi-C data reveals hallmarks of chromosome organization. & Ruiz, D. A fast algorithm for matrix balancing. Open2C et al. Cooltools: enabling high-resolution Hi-C analysis in Python. Xu, W. et al. CoolBox: a flexible toolkit for visual analysis of genomics data. Kerpedjiev, P. et al. HiGlass: web-based visual exploration and analysis of genome interaction maps. & Noble, W. S. HiCRep.py: fast comparison of Hi-C contact matrices in Python. Vilarrasa-Blasi, R. et al. Dynamics of genome architecture and chromatin function during human B cell differentiation and neoplastic transformation. Flyamer, I. M., Illingworth, R. S. & Bickmore, W. A. Coolpup.py: versatile pile-up analysis of Hi-C data. Analysis of Hi-C data using SIP effectively identifies loops in organisms from C. elegans to mammals. Mota-Gómez, I. et al. Sex-determining 3D regulatory hubs revealed by genome spatial auto-correlation analysis. Kamada, T. & Kawai, S. An algorithm for drawing general undirected graphs. Moran, P. A. P. Notes on continuous stochastic phenomena. Anselin, L. Local indicators of spatial association—LISA. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Machlab, D. et al. monaLisa: an R/Bioconductor package for identifying regulatory motifs. Armstrong, J. et al. Progressive Cactus is a multiple-genome aligner for the thousand-genome era. Hickey, G., Paten, B., Earl, D., Zerbino, D. & Haussler, D. HAL: a hierarchical format for storing and analyzing multiple genome alignments. Hubisz, M. J., Pollard, K. S. & Siepel, A. PHAST and RPHAST: phylogenetic analysis with space/time models. Pollard, K. S., Hubisz, M. J., Rosenbloom, K. R. & Siepel, A. Detection of nonneutral substitution rates on mammalian phylogenies. Derelle, R., Philippe, H. & Colbourne, J. K. Broccoli: combining phylogenetic and network analyses for orthology assignment. Minh, B. Q. et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. Gel, B. et al. regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests. Najle, S. R. et al. Stepwise emergence of the neuronal gene expression program in early animal evolution. Koromila, T. et al. Odd-paired is a pioneer-like factor that coordinates with Zelda to control gene expression in embryos. Systematic evaluation of chromosome conformation capture assays. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Zhang, Y. et al. Model-based analysis of ChIP–seq (MACS). Bentsen, M. et al. ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Li, X.-Y., Harrison, M. M., Villalta, J. E., Kaplan, T. & Eisen, M. B. Establishment of regions of genomic activity during the Drosophila maternal to zygotic transition. Rajagopal, N. et al. RFECS: a random-forest based algorithm for enhancer identification from chromatin state. Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Zhang, P. et al. On the origin and evolution of RNA editing in metazoans. Dunham, I. et al. An integrated encyclopedia of DNA elements in the human genome. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Sebé-Pedrós, A. et al. Cnidarian cell type diversity and regulation revealed by whole-organism single-cell RNA-seq. & Sebé-Pedrós, A. GeneExt: a gene model extension tool for enhanced single-cell RNA-seq analysis. Diesh, C. et al. JBrowse 2: a modular genome browser with views of synteny and structural variation. Hoencamp, C. et al. 3D genomics across the tree of life identifies condensin II as a determinant of architecture type. Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Guo, Y. et al. Chromatin jets define the properties of cohesin-driven in vivo loop extrusion. Isiaka, B. N. et al. Cohesin forms fountains at active enhancers in C. elegans. We thank I. Maeso and M. Irimia for critical comments on the manuscript. We thank A. Hino and S. Denbo for greatly helping us to re-sequence and annotate the Capsaspora genome de novo. We are very grateful to S. B. Johnson, S. Haddock, MBARI and the David and Lucile Packard Foundation for sending us H. californensis samples. was supported by a Juan de la Cierva postdoctoral fellowship (grant no. FJC2020-043131-I) from the Spanish Ministry of Science and Innovation. is supported by the European Union's H2020 research and innovation programme under Marie Sklodowska-Curie grant agreement no. are supported by FPI PhD fellowships from the Spanish Ministry of Science and Innovation. is supported by the EMBO postdoctoral fellowship grant no. acknowledges the financing support from MEXT/JSPS KAKENHI grant no. was supported by NSERC Discovery grant (no. were supported by the Michael Sars Centre core budget from the University of Bergen. is supported by the European Research Council (grant no. M.A.M.-R. acknowledges support by the Spanish Ministerio de Ciencia e Innovación (grant no. These authors contributed equally: Cristina Navarrete, Xavier Grau-Bové Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain Iana V. Kim, Cristina Navarrete, Xavier Grau-Bové, Marta Iglesias, Anamaria Elek, Grygoriy Zolotarov, Sean A. Montgomery, Ewa Ksiezopolska, Didac Cañas-Armenteros, Marc A. Marti-Renom & Arnau Sebé-Pedrós Centre Nacional d'Anàlisis Genòmic (CNAG), Barcelona, Spain Iana V. Kim, Nikolai S. Bykov & Marc A. Marti-Renom Universitat Pompeu Fabra (UPF), Barcelona, Spain Cristina Navarrete, Marc A. Marti-Renom & Arnau Sebé-Pedrós School of Biological and Behavioral Sciences, Queen Mary University of London, London, UK You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar sequenced and assembled genomes to chromosome level. collected biological material, performed Micro-C experiments and conducted bioinformatic analysis. performed phylogenetic and sequence conservation analyses. and A.S.-P. wrote the manuscript with contributions from all authors. Correspondence to Iana V. Kim or Arnau Sebé-Pedrós. The authors declare no competing interests. Nature thanks Giacomo Cavalli, Ferdinand Marlétaz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Overview of the input material for Micro-C experiments, library preparation strategy, and sequencing statistics in each species. The D. melanogaster dataset27 was subsampled to match the coverage of generated Micro-C maps in this study. b, Top, fluorescence-activated cell sorting (FACS) profile of crosslinked phagocytic choanocytes from E. muelleri labelled by feeding sponges with fluorescent microspheres. Only cells positive for nuclei Hoechst 33342 staining together with fluorescent beads were sorted. The sorted cell population (P3) was selected using sequential gating strategy through P1 and P2. Right, fluorescent microscopy image of sorted choanocytes, where PH stands for phase contrast, nuclei are in blue, FluoSpheres beads are in green. Scale bar (top right corner) is 50 µm. Below, the sequential FACS gating strategy (P1 - P3) to sort mOrange::NvElav+ neuronal cells (P4) from the N. vectensis transgenic line65. Wild type animals lacking the mOrange fluorescent protein were used to verify the gating strategy. The FACS sorting experiment data for E. muelleri and N. vectensis are representative of at least 6 independent experiments. c, The quality of chromatin digestion with MNase and followed proximity ligation was assessed with High sensitivity D5000 ScreenTapes using the Agilent 2200 TapeStation systems. The optimal chromatin fragmentation with Mnase results in up to 80% mononucleosomes profile. d, Barplots showing the percentage of reads mapped to the genome of each species. e, Barplots illustrating the distibution of intrachromosomal (cis) and interchromosomal (trans) interactions in each replicate experiment. The percentage of trans-contacts observed is species-specific but can be influenced by several factors: (i) the type of nuclear organization, such as Rabl-like configuration or the presence of chromosome territories, (ii) a high chromosome count, as seen in S. arctica and S. rosetta, (iii) and the reduction in nuclear diameter during the growth of coenocytes in S. arctica. f, Heatmap showing pairwise similarity scores between biological and technical replicates calculated as the stratum adjusted correlation coefficient (SCC). Differences in pairwise comparisons between experimental replicates are shown as mean ± s.d. The number of replicates per species as in (d-e). g, Top, cis-decay plots showing the rate of decay of contact frequency over genomic distance. The contact probability is averaged over all chromosomes. For C. owczarzaki, samples obtained from mitotic (blue) and synchronised G1/S stage (in orange) show different contact frequency behavior at short (below 10 Kb) and long (over 1 Mb) genomic distances. Bottom, log-derivative of cis-decay plots that predicts the folding of DNA into genomic structures and their size, most commonly chromatin loops, which tend to be the dominant micro-scale contacts. The first pronounced peak and dip at log-derivative cis-decay plots (highlighted in grey) in M. leidyi is observed at the scale from 10 Kb to 100 Kb. BUSCO completeness score was calculated using genome mode or protein mode against metazoan BUSCO dataset for all species except unicellular holozoans, where eukaryotic dataset was used. c, Chromosome-level re-assembly of S. arctica, S. rosetta, T. adhaerens and C. collaboinventa genomes using Micro-C data resulting in total of 27, 36, 6 and 6 chromosomes, respectively. Both S. arctica and S. rosetta posses genome-wide telomere clustering, whereas placozoans display strong interchromosomal compartmentalization signal. d, Left, genome-wide Micro-C contact map showing the chromosome-level assembly of C. owczarzaki. Right, chromosomal rearrangements in C. owczarzaki. Visual inspection of chromatin interaction maps revealed heterozygous deletions on C. owczarzaki chromosome 2, which is also confirmed by the uneven distribution of anti-H3 ChIP-seq coverage. In addition, one arm of the chromosome 13 exhibits genome-wide increase in the interaction frequency with other chromosomes, as well as two-fold coverage of H3 ChIP-seq, suggesting the gain of a chromosome arm pair. Finally, chromosome 15 v shares one arm with chromosome 15 and appears to be whole-arm translocation. e, Same as (d) for M. leidyi. The presence of uncollapsed haplotypes was estimated by distribution of per-base sequencing depth (left). Chromosomes in M. leidyi exhibit telomere clustering as well as increased intrachromosomal interactions similar to chromosome territories. f, Same as (d) for E. muelleri genome assembly with chromosome organisation similar to chromosome territories. g, Repeat content for each assembled genome, annotated using EDTA97. a, Table translating relative resolutions of contact maps that were used to calculate compartmentalisation signal into base-pair resolutions. b, Example genomic regions showing eigenvector coefficients E1 and compartment annotation into A (active, red), B (inactive, blue) and I (intermediate, yellow). c, Density plots showing genome-wide distribution of E1 eigenvalues and the relative abundance of each defined compartments (stacked barplot on top). Compartments were defined using fitting of Gaussian mixture with three components (k = 3). A Bayesian Information Criterion (BIC) was computed for the specified mixture (bottom plot) (see Method section). d, Association between chromatin compartments and different genomic features calculated per genomic bin at a relative resolution of 20,000 bins per genome, as in (a). The proportion of features in each compartment category follows the classification as in (c). The mean value of distributions is shown as the center line on the boxplots, with interquartile range (IQR) as the box limits and whiskers extending to 1.5x IQR. Insulation score profiles were calculated for multiple resolutions with window sizes corresponding to 5x, 10x and 25x the chosen resolution. Parameters showing two strongest average insulation scores were considered optimal. For each of our studied species, an example contact map region with calculated insulation profile is shown. b, Left, overlap between regions annotated as strong boundaries using strategy described in this paper (see Method section) and previously published datasets12,27. c, Left, distribution of boundary strength values per species. Insulation boundaries (marked in blue) were selected using Li threshold as implemented in cooltools103. Middle, distribution of linear distances (Kb) between successive boundaries, with the number of examined region between boundaries indicated. Right, boxplots showing the number of genes located between insulation boundaries (same number of examined regions as in the previous plot). Boxplots center line shows the median value, with box limits indicating the IQR and whiskers as 1.5x IQR. d, Epigenetic, structural and gene features associated to insulation boundaries in each species. Note that a boundary can be annotated with multiple features (e.g. TSS, ATAC and H3K4me3 peaks). a, Example genomic regions in S. arctica illustrating the co-segregation of inactive chromatin regions. The interacting regions, highlighted in grey, fold into chromatin domains that exhibit local compartmentalised interactions. b, The manually annotated 296 compartment domains have a median size of 18 Kb. Middle, the observed long-distance interactions within the domains display a local checkerboard pattern, where contacts are enriched within certain set of loci, while contacts between them are depleted. To quantify the contact distribution, we calculated the sum of ICE (iterative correction and eigenvector decomposition)-normalized contacts within the segregated regions and their flanking regions (30 Kb) at a resolution of 2,800 bp, across the size range of 50 Kb to 5 Mb. The contact interaction pattern observed over the silenced regions showed a reduced interaction frequency across the region body compared to flanking loci. This interaction pattern is typical for checkerboard compartmentalisation, in contrast to loop interactions, which manifest as local peaks in interaction frequencies. Right, genes located within the compartment domains are lowly expressed or silenced (*** p-value < 2.2e−16, Wilcoxon rank sum test). Boxplots center line shows the median value, with box limits indicating the IQR and whiskers as 1.5x IQR. c, Distribution of epigenetic signals across compartment domains. The regions within the annotated domains were located within the inactive B compartment and were enriched in transposable elements, predominantly Gypsy LTRs, which accounted for 63% of the total TEs in these regions. d, Size distribution of manually annotated 183 contact regions in S. rosetta that harbour lowly expressed genes (*** p-value = 4.8e−6, Wilcoxon rank sum test), boxplots as in (b). e, Example genomic regions in S. rosetta forming distal interactions. f, Same as (c) for S. rosetta. The interacting regions show weak enrichment in H3K4me1 and H3K27me3 signals compared to random genomic regions. g, Same as (e) for C. owczarzaki. h, Distal contacts in C. owczarzaki connect promoter regions of highly expressed genes (*** p-value = 5.3e−12, Wilcoxon rank sum test), boxplots as in (b). i, Distal contacts in C. owczarzaki are indicative of micro-compartmentalisation signal because of the characteristic alternating contact pattern and the decreased cumulative interactions in the promoter regions of the target genes compared to the concentration typically seen in chromatin loops annotated in T. adhaerens, M. leidyi, N. vectensis, D. melanogaster and H. sapiens. To quantify the distribution of contact interactions around TSS-TES sites, we calculated and compared the sum of ICE (iterative correction and eigenvector decomposition)-normalized contacts at species-specific resolutions (400 bp for C. owczarzaki, T. adhaerens and D. melanogaster, 500 bp for N. vectensis, 800 bp for M. leidyi and 5 Kb for H. sapiens). To eliminate confounding signals from distal compartmentalisation pattern or other long-distance interaction patterns, the sum of considered interactions was restricted to contacts that fall within the range size of annotated loops or interacting regions (4–100 Kb for C. owczarzaki, T. adhaerens, 5–250 Kb for D. melanogaster, 10–360 Kb for N. vectensis, 5–150 Kb for M. leidyi and 50–1,060 Kb for H. sapiens). The TSS-TES regions were rescaled into 50 bins with flanking regions of 10 Kb for each species except H. sapiens with 100 Kb flanking regions. Additionally, we flipped the TSS-TES regions and their corresponding flanking regions for negative-stranded genes. Notice that in C. owczarzaki, the sum of interactions around the TSS was lower than average interactions within the gene body. This is due to a small-scale local checkerboard pattern, where regions between interaction loci showed low contact frequency. As a result, cumulative interactions at promoters were even lower than average background signal and signal over gene bodies. In contrast, in other examined species, including T. adhaerens, M. leidyi, N. vectensis, D. melanogaster and H. sapiens, where chromatin loops connected examined promoter regions to cis-regulatory elements, the contact frequency at loop anchor regions was enriched and higher than the average across gene bodies. These differences highlight distinct modes of chromatin organization of C. owczarzaki with other species. j, In C. owczarzaki, a subset of highly expressed genes (274) exhibit increased interaction frequencies between TSS and TES forming gene body interaction domains. a, Example E. muelleri genomic regions showing contact patterns perpendicular to the diagonal of the Micro-C matrix and visually resembling flares139, jets140, or fountains40,141. b, Aggregated contact strength around the midpoints of flare regions. Random genomic regions anchored at the TSS of expression-matched genes were used as a control. Boxplots center line shows the median value, with box limits indicating the IQR and whiskers as 1.5x IQR. c, Example E. muelleri genomic regions showing distal interactions connecting promoter and enhancer-like anchor regions. Unlike typical chromatin loops, the preferential contact interactions in E. muelleri are diffuse and do not form a conspicuous dot contact pattern. d, A total of 84 manually annotated focal contacts connecting distal regulatory elements were classified as enhancers or promoters based on their H3K4me3 to H3K4me1 ratio. e, Aggregate plots demonstrating contact enrichment within rescaled contact regions, compared to random genomic regions anchored at TSS of expression-matched genes on one side and distance-matched random points on the other side. Boxplot limits are as in (b). f, Non-promoter cis-regulatory elements were identified based on chromatin state, defined by low H3K4me3 and high H3K4me1 enrichment around regions of accessible chromatin. The plots illustrate the distribution of these elements and their proximity to the nearest transcription start site (TSS) or other contact anchors within loop-forming enhancers. Notice the distance-to-TSS distribution of E. muelleri enhancer-like elements is similar to that of enhancers that do not form stable loops in other species. a, Example genomic region in N. vectensis showing chromatin loop contacts with loop anchors highlighted in grey. b, Loop anchor regions were classified as promoter-side if characterized by high H3K4me3 ChIP-seq signal levels and low H3K4me2 or low H3K4me1 signal. Enhancer-side loop anchors were defined as regions with low H3K4me3 and high H3K4me2 or H3K4me1. c, Most loop anchors retained their original classification, regardless of whether the H3K4me3/H3K4me2 or H3K4me3/H3K4me1 ratio was used. For N. vectensis, the ratio of H3K4me3/H3K4me1 outperformed H3K4me3/H3K4me2 in classifying loop anchors, as most of the disputed loops anchors annotated as promoters with H3K4me3/H3K4me2 were predominantly located in intronic and intergenic regions (pie chart). d, Left, aggregated contact strength of chromatin loop interactions, showing the overall intensity and frequency of chromatin contacts across loop anchor points. Right, loop anchors in N. vectensis show GTGT-motif enrichment (FC = 327, p-value = 1e−40) compared to GC-normalised background genomic regions. e, Genomic regions in N. vectensis displaying non-loop self-interacting domains. f, Same as (d), but for regions between insulation boundaries that also harbour self-interacting domains. Right, motif enrichment analysis was focused on accessible chromatin regions at the insulation boundaries. Accessible promoter regions in neuronal Elav+ cells were used as the background for comparison. a, Annotation of chromatin loop anchors with promoter and enhancer chromatin signatures for T. adhaerens and C. collaboinventa. Loop anchors annotated as enhancers were mostly located within promoter regions of other genes. To resolve this ambiguity, such loop anchors were classified as promoters based on their genomic context. b, Example contact map regions depicting promoter-enhancer distal interactions highlighted in grey in syntenic regions of placozoans. c, Local Moran's Index scatterplot visualises assignment of genomic bins to four distinct groups: High-High (HH), where examined signal (ATAC or H3K4me3) spatially co-localises in a neighbourhood of other bins with high signal; Low-Low (LL) bin has low examined signal and located in a neighbourhood of bins with low signal; when bin and its neighbourhood have different levels of signal, then the bin is assigned to Low-High (LH) or High-Low (HL) quadrants. Statistically significant values are in solid colors. Right panel illustrates intensity of examined signal layered over the two-dimensional Kamada-Kawai representation of top 20% contact interactions. p-values and r-values (Pearson correlation coefficients) were determined using a one-sided permutation test. A linear least-squares regression was then performed between z-scores of ATAC or H3K4me3 values and the signal's spatial lag. d, Boxplots showing relative gene expression (RNA-seq) and peak intensity (H3K4me3) at promoter regions of genes from GP1, GP2, and GP3 groups. Boxplots center line shows the median value, with box limits indicating the IQR and whiskers as 1.5x IQR. e, Left, heatmaps showing CPM normalised ATAC-seq and ChIP-seq coverage, motif scores and Mutator transposable element density within 5 Kb of the TSSs of GP1, GP2, and GP3 genes in C. collaboinventa. Each heatmap scale starts at zero. Middle, aggregate peak analysis displaying the contact strength between gene promoters within each annotated group. Right: Genes in C. collaboinventa from various gene groups, classified based on the presence of chromatin loops and their epigenetic states, demonstrate overlap with orthologous genes from GP1, GP2, and GP3 in T. adhaerens. f, GO-term enrichment analysis of GP1 genes with p-values determined using Fisher's exact test. g, Barplots showing the cell type (from previously published dataset123) in which genes belonging to each group are maximally expressed. Only variable genes (with a fold-change higher than 1.8) are included. h, Scatterplot showing total gene expression (x-axis) versus gene expression variability (y-axis) across cell types. i, Distribution of motif scores in loop anchor regions compared to the genome-wide background. C. collaboinventa harbour similar motif to T. adhaerens (similarity score = 0.93) in 60% of annotated loop anchor regions. j, Local Moran's Index scatterplot and Gaudí plots demonstrate spatial co-localisation of sequence motif identified in promoters of GP1 genes of T. adhaerens (motif score above 80th percentile). Statistically significant values are calculated as in (c). k, Schematic phylogenetic tree of TIR sequences of Mutator DNA transposons from four placozoan species (Trichoplax adhaerens, Trichoplax sp. Placozoan Mutator DNA TIRs can be classified into 5 clades with consensus sequences. Pie charts shows the proportion of Mutator transposons harbouring the consensus TIR sequences. a, Scatter plot showing the normalised H3K4me3 and H3K4me2 ChIP-seq coverage in 2 Kb region around loop anchor. b, Comparison of loop anchor annotation using either H3K4me3/H3K4me2 or H3K4me3/H3K4me1 ratios. For M. leidyi, H3K4me3/H3K4me2 ratio were more effective in annotating loop anchors, as many loop anchors classified as promoters using H3K4me3/H3K4me1 were found within intergenic or intronic regions (pie chart). c, Normalised coverage for different chromatin features around loop anchors classified as promoters and enhancers. d, Genomic regions in H. californensis showing chromatin loops. In total, we annotated 239 chromatin loops, with 51% of loop anchors located within intronic or intergenic regions. High-resolution chromatin maps are expected to significantly increase the number of reported loops in H. californensis. e, Boxplots showing the total expression in scRNA-seq data8 for M. leidyi or RNA-seq data51 for C. californensis of genes with a loop anchor at their promoter regions, in their introns (enhancer sites), and genes not involved in distal chromatin interactions (outside loops). *** stands for p-value < 2.22e−16 of two-sided Wilcoxon rank sum test. Boxplots center line shows the median value, with box limits indicating the IQR and whiskers as 1.5x IQR. f, Motif score distributions at loop anchors (max score in 2,000 bp window around the center of a loop anchor) compared to genomic background. In H. californensis, we detected similar to M. leidyi GC-rich motif (similarity score = 0.96) enriched in 38% of loop anchors. g, Fraction of loop anchor sites containing the identified GC-rich motif at promoter sites (in orange), at enhancer sites (green) or at the promoters of genes not involved in chromatin loops (cyan). h, Scatterplot showing total gene expression (x-axis) versus gene expression variability (y-axis) across cell types, highlighting genes with their promoter involved in chromatin loops (orange) and also genes containing the GC-rich motif in their promoters but not involved in loops (cyan). These motif-containing genes without detected loops showed lower and more variable expression across cell types than genes with detected loops, suggesting the former could be forming loops in low-abundance cell types that we are unable to detect in bulk Micro-C experiments. i, DNA methylation levels at GC-motif sites located at chromatin loops (left) compared to methylation levels in motif occurrences outside detected chromatin loops (right). j, Bias-corrected ATAC footprint profiles centered around motifs located at loop anchors. k, Distribution of CTEP1 and CTEP2 bound DAP-seq peaks across genomic regions with varying DNA methylation levels and within annotated loop anchors. l, Number of loop anchor regions that contain CTEP1 and CTEP2 DAP-seq peaks. m, DAP-seq quantile normalized CPM coverage around GC-rich motif from CTEP2 binding assay using native genomic DNA fragments or unmethylated PCR amplified genomic DNA. 4f) exhibited higher affinity for the unmethylated GC-rich motif. n, Multiple sequence alignments of CTEP1 and CTEP2 genes were performed against the dataset of 358 metazoan genomes (Supplementary Table 3). The significant hits against CTEP proteins, exhibiting sequence identity above 50%, were found exclusively within ctenophores. o, Left, boxplots showing the number of transposable element insertions per promoter region of genes involved in chromatin loops compare to genes that are outside loops (*** indicates p-value < 2.22e−16, two-sided Wilcoxon rank sum test). Right, barplots showing the fraction of promoters in loops containing TE insertions compared to promoters not involved in loops and random genomic regions. Over 90% of promoter regions involved in distal interactions harbour insertion of DNA transposon. Additionally, promoters in loops have higher frequency of insertions of LTR and Unknown type transposons. Boxplot limits as in (e). p, Syntenic conservation within M. leidyi chromatin loops compared to Pleurobrachia bachei or Bolinopsis microptera. Left, barplot showing the fraction of conserved orthologs in all alignable genomic regions across ctenophore species (chi-squared test for given probabilities). Right, boxplot showing the fraction of shared orthologs between individual genomic regions within chromatin loops (P. bachei: n = 105; B. microptera: n = 332) versus in random genomic regions of similar size (P. bachei: n = 198; B. microptera: n = 945). p-value significance was calculated using two-sided Wilcoxon rank sum test. Boxplot limits as in (e). q, Number of predicted genes with zf-C2H2 protein domain in the different species studied included in this study. Detailed Micro-C protocol used in this study. Quality control metrics for each Micro-C replicate. List of publicly available datasets and their corresponding sequence read archive accession numbers used in this study. This file contains a list of orthologous genes for CTEP1 and CTEP2 architectural proteins in other ctenophore species, as well as a list of 358 publicly available metazoan genome datasets used for multiple sequence alignments of CTEP1 and CTEP2. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Kim, I.V., Navarrete, C., Grau-Bové, X. et al. Chromatin loops are an ancestral hallmark of the animal regulatory genome. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08985-1'>Light-microscopy-based connectomic reconstruction of mammalian brain tissue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 15:10:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mapping neurons and resolving their individual synaptic connections can be achieved by volumetric imaging at nanoscale resolution1,2 with dense cellular labelling. Light microscopy is uniquely positioned to visualize specific molecules, but dense, synapse-level circuit reconstruction by light microscopy has been out of reach, owing to limitations in resolution, contrast and volumetric imaging capability. Here we describe light-microscopy-based connectomics (LICONN). We integrated specifically engineered hydrogel embedding and expansion with comprehensive deep-learning-based segmentation and analysis of connectivity, thereby directly incorporating molecular information into synapse-level reconstructions of brain tissue. LICONN will allow synapse-level phenotyping of brain tissue in biological experiments in a readily adoptable manner. The brain is made up of an incredibly dense, complex and fine-grained arrangement of neurons with support cells, which together constitute a functional network that enables brain function. Determining how neurons are connected and reconstructing the circuitry that underlies information processing—that is, determining connectomes—demands accurate tracing of cellular circuit components including axons and dendritic spines, resolving synaptic connections and assigning them to specific neurons. Light microscopy holds considerable potential for unifying synapse-level circuit reconstruction with in-depth molecular characterization. However, its resolution is conventionally limited to several hundred nanometres (best case 200–300 nm laterally, and typically substantially worse (around 1,000 nm) along the optical (z) axis)—much too coarse to distinguish densely labelled cellular structures. Electron microscopy (EM), with its nanometre-scale resolution and comprehensive structural contrast, is at present the only technology that allows dense connectomic analysis (that is, comprehensive reconstruction of cellular circuit components1,2,3), and enormous strides have been made in using EM to map connectivity in organisms as diverse as worms4, flies5,6,7, mice8 and humans9,10. These advances were facilitated by technological progress in automated data collection and deep-learning analysis, which has made the challenge of densely annotating all cellular structures tractable2,3. EM sample preparation and readout are not directly compatible with visualizing specific molecules in circuit reconstruction, and require correlation with light microscopy to obtain molecular information11,12,13. EM reconstructions allow connectivity through chemical synapses to be inferred from structural features14. However, synapses cannot be further differentiated molecularly, and information related to signalling between cells, such as the distribution of receptor molecules that have key roles beyond classical synaptic transmission, remains lacking. Super-resolution optical imaging offers resolution beyond the diffraction limit by increasing instrument resolution15 or by expanding samples to increase distances between features16, but it has been limited mostly to sparse subsets of cells or molecule distributions devoid of cellular context. For example, multicolour ‘Brainbow' labelling with expansion and synaptic-marker detection allowed the connectivity of cellular subsets to be inferred17. To visualize living brain tissue comprehensively, fluorophores have been applied extracellularly, casting super-resolved cellular ‘shadows' when imaged with stimulated emission depletion (STED) microscopy18 in super-resolution shadow imaging (SUSHI)19. Combining this with two-stage machine learning enabled reconstruction at three-dimensional (3D) nanoscale resolution with LIONESS (live information-optimized nanoscopy enabling saturated segmentation)20. Fixation-compatible extracellular labelling with CATS (comprehensive analysis of tissues across scales)21 visualized tissue architecture using STED or expansion microscopy (ExM). However, LIONESS and CATS have not provided the traceability and accuracy required for synapse-level circuit reconstruction. ExM16 increases effective resolution by embedding tissue in a swellable hydrogel, disrupting the tissue's mechanical cohesiveness and expanding it. Expansion factor (exF) and corresponding resolution enhancement have been increased from around fourfold16,22 to around eight-to-tenfold23,24,25,26,27 in single-step approaches, and to around 16× and beyond with iterative application28,29,30 of two swellable hydrogels. Indiscriminate (‘pan-') labelling for protein density using amine-reactive fluorophore derivatives, such as N-hydroxysuccinimidyl (NHS) esters, revealed cellular ultrastructure with single-step25,31 and iterative30 expansion and visualized the complexity of brain tissue25,32. However, it has not been possible to achieve light-microscopy imaging at the resolution and signal-to-noise ratio that are required for dense connectomic reconstruction. Here we present a technology that can be used to densely reconstruct brain circuitry with light microscopy at synaptic resolution. We engineered a high-fidelity iterative hydrogel expansion scheme paired with protein-density staining and high-speed diffraction-limited readout that enables manual neuronal tracing and deep-learning-based cellular segmentation (Fig. We show traceability of the finest neuronal structures, including axons and dendritic spines; simultaneous molecular measurement; deep-learning prediction of molecule locations; and connectivity analysis at single-synapse resolution. We validate the technology using independent ground truth from sparse positive labelling and quantification of spine traceability. We furthermore provide comparisons of statistical data on neuronal connectivity with previous EM measurements, a method that has been used to cross-validate EM datasets33. This technology, which we term light-microscopy-based connectomics (LICONN), offers molecularly informed reconstruction of brain tissue with broad accessibility. a, LICONN volume of around 1 × 106 μm3 (native tissue scale) of mouse primary somatosensory cortex (layers II/III–IV, 396 × 109 × 22 µm3 original tissue scale, 0.95 × 106 µm3 before and 3.5 × 109 µm3 after hydrogel expansion of approximately 16×). Seventy-nine example cells from dense reconstruction with FFN. Right, dendrite from pyramidal neuron (box at top of left panel), with deep-learning predictions of the synaptic molecules bassoon (cyan, pre-synapses) and SHANK2 (magenta, excitatory post-synapses) and synaptically connected axons (bottom). Length scales and scale bars refer to biological size before expansion throughout. b, Subregion (single plane) of a. Enlarged views: top, intracellular structure of pyramidal neuron; bottom, primary cilium. Spinning-disk confocal imaging data with contrast-limited adaptive histogram equalization (CLAHE) (for comparison of raw versus CLAHE, see Supplementary Fig. Small panels: top, excitatory synapse with presynaptic protein-rich punctate features and bar-like feature at post-synapse (without CLAHE). Line: direction of intensity profiles (top right), measuring distance between pre- and postsynaptic features (DP: dense projection; mean ± s.d.) Middle, single bouton contacting two spines and DP–DP distance (mean ± s.d.). Scale bars, 2 μm (main image); 100 nm (right images). d, Periodic, protein-dense structure at circumference of neurite subset (without CLAHE). Example line profile and periodicity (mean ± s.d.). Top left, single plane (magnified) of LICONN volume (around 19 × 19 × 19 µm3) from primary somatosensory cortex in a Thy1-eGFP mouse with cytosolic eGFP expression in an axon (arrow). Top right, overlay with immunolabelling for eGFP. Bottom, renderings (green) and skeletons (black) of eight eGFP-expressing axons (ground truth, based on eGFP and structural LICONN channels), and skeletons generated by two annotators blinded to eGFP signal (magenta, consensus, offset for clarity). For additional datasets, see Supplementary Figs. Top, single plane (magnified) from LICONN volume (around 19 × 19 × 19 µm3, hippocampus, CA1; Thy1-eGFP mouse) with eGFP-expressing dendrite (arrow) and overlay with eGFP (green). Middle, cross-sections of eGFP-expressing dendrite with spines. Bottom, dendrite skeleton (black) generated from eGFP and structural channels (ground truth), within 3D rendering (green). Additional skeleton (magenta) generated from structural channel by two annotators blinded to eGFP. For additional datasets, see Supplementary Fig. g, Left, single-tile LICONN volume (hippocampus, CA1) with manual cellular annotations (colour, 658 structures) and 3D rendering. Right, neurites and magnified synaptic connections (different camera position). Our strategy for dense light-microscopy-based connectomics achieves an increase in resolution through hydrogel expansion rather than optical super-resolution, exploiting the speed, optical sectioning and availability of standard, diffraction-limited, spinning-disk confocal microscopes. Hydrogel embedding34 and expansion homogenize the refractive index to that of water. This facilitates the acquisition of volumes extended both laterally and along the z axis, which is notoriously difficult with other super-resolution techniques because of aberrations, scattering and photobleaching. It also allows facile multicolour readout of specific molecules at a resolution essentially identical to that of the structural channel. To achieve high-fidelity tissue preservation and neuronal traceability, we developed an iterative expansion technology based on independent, interpenetrating hydrogel networks including tailored chemical fixation, retention of cellular proteins and hydrogel chemistry that obviated hydrogel cleavage and signal handover steps. Available single-step or iterative approaches provided insufficient performance and straightforward modifications to increase exF resulted in unstable hydrogels (Supplementary Figs. We transcardially perfused mice with hydrogel monomer (acrylamide, AA)-containing fixative solution, equipping cellular molecules with vinyl residues, which subsequently co-polymerized with the hydrogel. Optimization of perfusion, with a lower monomer concentration (10% AA) than that used previously22,32, improved cellular preservation (Supplementary Fig. Chemical fixation does not preserve the precise shape of extracellular space19,33. However, LICONN maintains low-signal-intensity regions around cells, which is advantageous for tracing and segmentation, similar to extracellular-space-preserving protocols35 in EM connectomics. We collected and sliced brains, and exploited the broad reactivity of multi-functional epoxide compounds36,37—specifically, glycidyl methacrylate (GMA) and glycerol triglycidyl ether (TGE, bearing three epoxide rings)—to functionalize proteins more broadly with acrylate groups for hydrogel anchoring than common amine-reactive compounds, and to further fix and stabilize biomolecules, respectively. Alternatively, amine-reactive anchoring38,39 using N-acryloxysuccinimide (NAS)27 resulted in traceable datasets (Supplementary Figs. 2, 5, 6), but epoxide use improved cellular ultrastructure and emphasized synaptic features. We polymerized an expandable acrylamide–sodium acrylate hydrogel, integrating functionalized cellular molecules into the hydrogel network, and disrupted mechanical cohesiveness using heat and chemical denaturation22 (Supplementary Fig. After around fourfold expansion, we optionally applied immunolabelling to visualize specific proteins. A non-expandable stabilizing hydrogel prevented shrinkage during the application of a second swellable hydrogel intercalating with the first two hydrogels. To achieve structural preservation and homogeneous expansion, we optimized the composition of the hydrogels (Supplementary Figs. 8, 9), and found that chemically neutralizing unreacted groups after each polymerization step improved high-fidelity expansion by abolishing cross-links between hydrogels, ensuring their independence. Finally, protein-density (‘pan-protein') staining with fluorophore NHS esters comprehensively visualized cellular structures, mapping (primary) amines that were abundant on proteins. These triple-hydrogel–sample hybrids yielded an expansion of around 16-fold and were mechanically robust, facilitating handling and extended imaging (exF = 15.44 ± 1.68, mean ± s.d., n = 4 technical replicates across n = 3 mice; Supplementary Fig. 10; unless otherwise stated, we give length measures in original tissue size by scaling measured post-expansion lengths with this exF). Expansion-induced distortions were similar to those in previous work28,29,32 (Supplementary Fig. Spinning-disc confocal imaging with a high-numerical-aperture (NA = 1.15) water-immersion objective lens in the green spectral range yielded an expected resolution of around 280 nm laterally and around 730 nm axially. With an exF of around 16, this translated into effective resolutions of around 20 nm and and around 50 nm, respectively, demanding an effective voxel size of about 10 × 10 × 25 nm3 for adequate sampling. Overall, this workflow was robust and provided the resolution, contrast and throughput required for connectomic tissue reconstruction. We analysed a tissue volume of around 1 × 106 µm3 (native tissue scale, 396 × 109 × 22 µm3, 0.95 × 106 µm3 pre-expansion, effective voxel size 9.7 × 9.7 × 25.9 nm3, 3.5 × 109 µm3 post-expansion; Fig. 1a), spanning layers II/III–IV of the primary somatosensory cortex (Supplementary Fig. A total of 132 partially overlapping subvolumes were imaged (arranged on a 6 × 22 grid) and an automated algorithm40 (scalable optical flow-based image montaging and alignment; SOFIMA) was used for seamless volume fusion. With high parallelization in spinning-disc confocal microscopy (around 300 focal points in our system), the approximately 4.2 × 109-µm3 post-expansion size (0.47 teravoxels, including tile overlap) was imaged within 6.5 h. This corresponded to an effective voxel rate of 17 × 106 voxels per second (17 MHz), including overhead from sample stage movement and tile overlap. Individual neurons with their axons and dendrites were clearly delineated from each other in densely packed neuropil and showed rich subcellular structures (Fig. 12), including mitochondria, Golgi apparatus and primary cilia. When we inspected dendrites and their spines—postsynaptic structures that are typical of excitatory synapses (Fig. 1c)—we found putative synaptic transmission sites highlighted by protein-rich, high-intensity features, akin to postsynaptic densities (PSDs) in EM data on chemically fixed specimens14. Similarly, presynaptic sites exhibited protein-dense nanoscale features, arranged in a lattice-like pattern spaced at 97 ± 28 nm, at a distance of 139 ± 19 nm from PSDs (192 synapses, 3 technical replicates across n = 2 mice) (Fig. 1c), again similar to features seen in EM. We also observed prominent ring-like periodic patterns at the circumference of a subset of neurites (Fig. The periodicity of 89 ± 12 nm (32 distance measurements, n = 2 mice) was highly suggestive of the actin and β-spectrin cytoskeletal lattice that organizes specific proteins41 below the plasma membrane. Periodicity was consistent with the value of 182 nm that was previously reported when labelling only one component41. Together, these results indicate that our expansion and imaging procedure reports the cellular constituents of brain tissue with high fidelity from the tissue scale to the nanoscale. Thin, tortuous axons in dense neuropil and dendritic spines with their thin necks are among the most challenging structures for connectomic tracing. To evaluate the reliability of manual tracing, we compared human consensus skeletons with sparse fluorescent labelling. Specifically, we obtained ‘ground-truth' neurites from cytosolically expressed enhanced green fluorescent protein (eGFP, detected by immunolabelling) in a subset of neurons in Thy1-eGFP mice. We compared those with independently traced skeletons of the same objects, manually generated exclusively from the LICONN structural channel (Fig. After training (Methods), 2 tracers received 12 LICONN datasets with 37 axon stretches (880 µm cumulative length, n = 3 technical replicates across n = 2 mice, cortex), with a seed point in each eGFP-expressing axon. Tracers were blinded to the eGFP signal itself. They independently traced the indicated axons, compared results and found consensus at locations of disagreement (Fig. Of 37 axons analysed, the consensus skeletons followed a wrong path in one case, compared with eGFP ground truth (1.1 errors per mm; Fig. In a similar analysis of eGFP-expressing dendrites in the hippocampus, the blinded tracers correctly identified 259 out of 289 spines (Fig. Encouraged by the consistency between LICONN-derived skeletons and their eGFP ground truth, we returned to the 1 × 106-µm3 cortical dataset in Fig. We validated the traceability of axons and dendrites across tile borders (Extended Data Fig. We sampled 38 subvolumes of 2 × 2 × 2 µm3 at random locations. An expert annotator marked all spine heads on the basis of morphology and PSDs; spine density was 1.0 ± 0.3 per µm3 (mean ± s.d. The annotator then manually attached spine heads to parent dendrites. Of 306 spine heads, 285 (93.1%) were unambiguously traced to a dendrite. To test whether LICONN enabled volumetric annotation, we manually reconstructed neuronal structures in a 19.3 × 19.3 × 8.1-µm3 volume (imaged at an effective voxel size of 9.7 × 9.7 × 13.0 nm3) from the hippocampal CA1 stratum oriens (Fig. We reconstructed 658 structures, revealing their complex shapes and interwoven arrangement. This showed that LICONN is suitable for detailed volumetric annotation. However, manual reconstruction scales poorly, and would be difficult to apply comprehensively for the volume in Fig. Having manually validated traceability and segmentability, we analysed larger volumes by adopting deep-learning segmentation algorithms from EM connectomics. Specifically, we trained flood-filling networks (FFNs)42, which have achieved state-of-the-art segmentation accuracy on diverse connectomic datasets. We imaged a 109 × 74 × 22-µm3 region in the hippocampal CA1 (Fig. 2) in a 4 × 6 tile arrangement with an effective voxel size of 9.7 × 9.7 × 13.0 nm3. Within this volume, we chose an 83,825-µm3 bounding box (85 × 69 × 14 µm3, 68.6 gigavoxels at native imaging resolution) and produced ground-truth annotations by iterative model predictions and manual proofreading (Methods). a, Rendering of 85 × 69 × 14-µm3 LICONN volume (native tissue scale) from hippocampal neuropil in CA1, overlaid with dense FFN-based segmentation of neuronal structures in the bottom corner. Neuronal structures were comprehensively proofread in this volume by correction of split and merge errors (without any manual painting of voxels; see https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid82.json for original data and proofread segmentation). b, Magnified view from a single plane with (top to bottom) raw structural data (CLAHE applied), dense segmentation after proofreading and overlay. c, Rendering of 5.8% of axons contained in the volume in a (see https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid82_fig2_axons.json for browsable data). f, Spatial arrangement of selected axonal and dendritic structures, highlighting various types of contact. g, Segment size (number of voxels on a logarithmic scale) of neuronal structures for the base FFN segmentation (blue), after automated agglomeration (white) and after full manual proofreading of the automated agglomeration (yellow) for the dataset in a (n = 1 dataset). h, Edge accuracy for the base segmentation (blue), after automated agglomeration (white) and after manual proofreading (yellow). i, Distribution of spine-head volumes in the same segmentation volume, analysed for 59,332 spines. An FFN was trained on the ground-truth annotations, applied to the whole bounding box (Fig. 2) and evaluated on 99 manually skeletonized neurites (69 axons, 1.8 mm cumulative path length, 30 dendrites with 1,041 dendritic spines; Supplementary Figs. The mean spine density along skeletonized dendrites (1.6 ± 0.3 per µm, nine dendrite stretches) was similar to that in previous EM data43. We optimized the FFN base segmentation (Methods) to minimize merge errors, as confirmed by comparison with the skeletons (0 mergers, 413 splits, 80.1% edge accuracy; see ref. We applied automated agglomeration42, which increased the edge accuracy to 92.8% (Fig. 2h), reducing splits by 92.5% (from 413 to 31), with some misattached spines but no mergers between axons or dendrites. We then attempted to eliminate remaining errors in the automated reconstruction through comprehensive manual proofreading of the entire 83,825-µm3 volume (correcting object-level split and merge errors; Methods). We labelled objects in the proofread segmentation as axons, dendrites or glia using an automated classifier (Methods) and found 18,268 axons with a cumulative length of 342.3 mm, of which 5.8% (by length) are shown in Fig. 2c (segmentation: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid82_fig2_axons.json), and 1,643 dendrites with a cumulative length of 119.1 mm (Fig. 2d,e), of which 27.3% are shown in Fig. Using the manually generated skeletons, we evaluated the contributions of base segmentation, automated agglomeration and manual proofreading to increasing segment size and edge accuracy (Fig. The proofread reconstruction yielded an edge accuracy of 95.6% with one morphological merger, 29 incorrectly attached spines (2.8% of spines), 14 uncorrected spine splits (1.4%) and zero splits involving axons or dendrite trunks. We did not process glial segments and blood vessels further, because the FFN models were trained exclusively on neuronal structures. Dense segmentation (browsable at https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid82.json) thus revealed the complex 3D arrangements of neuronal structures in nanoscale detail (Fig. Visual inspection suggested that signal-containing regions that were not covered by segments corresponded mainly to slight deviations in segment shape. Areas that were not captured in automated segmentation contained mostly intracellular regions or spines. We therefore quantified the traceability of spines in the volume, randomly sampling 40 subvolumes of 2 × 2 × 2 µm3. An annotator traced 281 of 301 spines (93.4%) to a parent dendrite, using only raw imaging data. The spine density was 0.9 ± 0.3 per µm3, similar to previous reports in CA1 (ref. We next compared ground-truth skeletons with the FFN segmentation, in which the remaining error was dominated by spines that were not labelled by the FFN (83 of 1,041 spines in the skeletons, 8.0%). Nearly all spine necks in the automated segmentation were attached to a dendrite; occasionally the FFN neglected to segment voxels at spine necks, but agglomeration and proofreading were mostly still able to attach spine heads to the correct dendrite (1,000 of 1,041 spines on the manually generated skeletons). 2i; 53% within 0.01 and 0.05 µm3) were consistent with EM data44. Overall, LICONN enabled FFN-based segmentation with automated accuracy comparable to state-of-the-art EM results7,42 and manual correction of remaining errors using standard connectomic proofreading workflows. We next sought to take advantage of the ability of light microscopy to visualize specific molecules, to directly verify cellular, subcellular and synaptic identities and place molecules in the context of the tissue's 3D architecture and connectivity. Post-expansion immunolabelling (applied here after the first expansion) avoided extra tissue processing before expansion and promoted epitope accessibility22,29 in the expanded, molecularly decrowded tissue–hydrogel hybrid. This approach also renders the displacement of fluorophores from biological targets irrelevant by effectively ‘shrinking' antibodies from their physical size of around 10 nm to less than 1 nm. Although certain epitopes are denatured during expansion, our limited screen identified a range of commercial antibodies that are compatible with LICONN. Iterative expansion with multicolour readout in standard spinning-disc confocal microscopes directly provided super-resolution measurements of structural and molecular channels, which is difficult when correlating light microscopy with EM connectomics (Supplementary Fig. We first focused on synaptic proteins by immunolabelling bassoon, a presynaptic active-zone scaffolding protein, and PSD95, a postsynaptic scaffolding protein at excitatory synapses, and visualizing them in structural context in triple-colour measurements (Fig. At the high 3D resolution achieved here, bassoon labelling revealed lattice-like arrangements of nanoscale spots, recapitulating the protein-dense features in the structural channel (Fig. Both PSD95 and SHANK2, another postsynaptic scaffolding protein, were arranged in more compact, disc-like arrangements mirroring PSDs in the structural channel. Distance measurements between bassoon and SHANK2 yielded 154 ± 19 nm (Fig. 3b; mean ± s.d., 106 synapses, n = 2 mice), comparable to previous measurements from cultured neurons45. Applying the FFN-segmentation model from Fig. 2a–f located the synaptic molecular machinery to the respective neuronal structures, such as spiny dendrites (Fig. We labelled additional synaptic proteins in LICONN's super-resolved structural framework (Fig. Labelling for the vesicular glutamate transporter VGLUT1 highlighted synaptic vesicles, and co-labelling with RIM1/2 concomitantly demarcated active zones. The NMDA glutamate receptor GLUN1 overlapped with SHANK2 at post-synapses, with the centres of mass of their 3D distributions spaced 16 ± 5 nm apart (185 synapses, n = 1 mouse; Fig. a, Immunolabelling for presynaptic bassoon (cyan) and postsynaptic PSD95 (magenta) in LICONN volume (somatosensory cortex). Right, magnified views of synapses (boxes in left panel) showing immunolabelling and structural channels separately and overlaid. Scale bars, 2 μm (left); 500 nm (right). b, Distance between bassoon and SHANK2 signals (mean ± s.d., violin plot including median and quartiles, 106 synapses). c, Three-dimensional renderings of bassoon and SHANK2 immunolabelling mapped onto a dendrite from FFN segmentation of the volume in a. d, Top, LICONN with immunolabelling for synaptic markers. Single planes from volumes in hippocampal CA3 (stratum lucidum, leftmost panel) and CA1. e, Left, dendritic spine with SHANK2 (magenta) and GLUN1 (cyan) immunolabelling, showing maximum intensity projections of respective immunolabellings (points: centres of mass). f, Illustration of excitatory-synapse detection through bassoon (cyan) and SHANK2 (magenta) immunolabellings, converted to point annotations. Examples include 1:1 and 1:2 presynaptic to postsynaptic connections. g, Immunolabelling renderings with detected synapses (2:1, 1:1 and 1:2 presynaptic to postsynaptic connections). h, Ground-truth (proofread) immunolabelling-based excitatory-synapse detections in a 913-µm3 volume (hippocampus, CA1, stratum radiatum). i, Three-dimensional rendering of dendrite (hippocampus, FFN segmentation) with excitatory synapses (bars), detected through bassoon (pre-synapses, cyan) and SHANK2 (post-synapses, red) immunolabelling. Magnified views include synaptically connected boutons. j, LICONN with immunolabelling for gephyrin (yellow, inhibitory post-synapses) and SHANK2 (magenta, excitatory post-synapses), with immunolabelling shown separately and overlaid for the boxed region. Scale bars, 2 μm (left); 500 nm (right). k, Left, LICONN with immunolabelling for VGAT (cyan, inhibitory pre-synapses) and gephyrin (yellow), with channels shown separately for the boxed region. Right, similar measurement with immunolabelling for bassoon (cyan, excitatory and inhibitory pre-synapses) and gephyrin (yellow). Scale bars, 1 μm (main); 500 nm (enlarged boxes). Proximity between neurites is a weak predictor of synaptic connectivity1. We therefore used molecular information as ground truth for connectivity and developed an automated synapse identification pipeline based on immunolabelling (Fig. We first computationally annotated pre-synapses and excitatory post-synapses as defined by the presence of bassoon (excitatory and inhibitory pre-synapses) and SHANK2 (excitatory post-synapses), respectively (Fig. 3f); sampling intensity in the structural channel facilitated distinguishing synaptic signal from unavoidable immunolabelling background (Methods). We then automatically matched corresponding pre- and post-synapses to full synapses, including both one-to-one and one-to-many connections (Fig. Unpaired pre-synapses corresponded to inhibitory synapses, missed SHANK2 detections (for example, owing to low copy number or partial epitope degradation resulting in sub-threshold signal) or infrequent synapses lacking SHANK2 (but identifiable as excitatory because of a prominent PSD). We therefore revisited unpaired pre-synapses and classified neighbouring post-synapses as excitatory if a prominent PSD was present in the structural channel. When comparing purely automated synapse detections with manually validated synapse locations in a test dataset (1,059 excitatory synapses), we obtained 95% accuracy for the detection of pre- and post-synapses, and 90% for fully assembled synapses (Supplementary Fig. 19) (913 µm3 volume, F1: range 0–1, combining precision and recall; pre-synapses: F1 = 0.94; post-synapses: F1 = 0.95; full synapses: F1 = 0.90). We found detection to be robust against variation in imaging parameters and applicable in both hippocampus and cortex (Supplementary Fig. Finally, by integrating FFN-based neuron segmentation with automated synapse detection, we inferred excitatory axonal inputs onto a specific dendrite (Fig. LICONN thus allowed us to map molecularly defined synaptic connectivity onto automated morphological reconstructions. To identify inhibitory synapses, we labelled gephyrin, a postsynaptic scaffolding protein. Gephyrin-positive, inhibitory post-synapses were less conspicuous structurally than their excitatory, SHANK2-positive counterparts, and these molecules were mutually exclusive at individual synapses (Fig. As a consistency check, we also co-labelled vesicular GABA transporter (VGAT) and gephyrin, identifying pre- and postsynaptic compartments in the same inhibitory synapses. As expected, presynaptic bassoon and postsynaptic gephyrin were closely juxtaposed (Fig. Molecular labelling thus distinguished excitatory and inhibitory synapses in LICONN volumes. Next, we related molecular properties to neurite tracings for characterizing fundamental parameters of synaptic wiring, including inhibition and excitation. We imaged two LICONN volumes in primary somatosensory cortex (143 × 20 × 24 µm3 and 179 × 20× 24 µm3, one mouse), with immunolabelling for excitatory (SHANK2) and inhibitory (gephyrin) post-synapses (Fig. We manually analysed 11 spiny dendrites (Fig. 4a; total length 123 µm, 322 spines) and found 2.8 ± 1.2 (mean ± s.d. throughout) synaptic inputs per µm length (Fig. Synapses were found mostly on spine heads (90.3%), whereas 1.6% and 8.1% located to spine necks and shafts, respectively (Fig. As expected, the vast majority of spine heads were positive only for SHANK2 (95.7 ± 3.0%); on rare occasions, they had only a gephyrin-positive connection (0.6%) or a gephyrin-positive connection in addition to the usual excitatory input (1.7%). A similarly small number of spine heads (2.1%) had neither SHANK2- nor gephyrin-positive connections, but had a PSD in the structural channel, with SHANK2 either not expressed or not labelled (Fig. As expected, these molecules were mutually exclusive at any given synapse (Fig. Immunolabelling directly yielded the balance of excitatory versus inhibitory inputs to dendrites (90% versus 10%, respectively; Fig. The results were consistent when the analysis was repeated using structural information—that is, the locations of synapses in axon outputs (shaft, spine head or neck)—to classify axons into excitatory or inhibitory, as is commonly done in EM-based connectomics (Supplementary Fig. a, Spiny dendrite with SHANK2-expressing excitatory (magenta) and gephyrin-expressing inhibitory post-synapses (yellow), connected axons (presynaptic bassoon: cyan) and seed locations. b, Input density onto spiny dendrites (primary somatosensory cortex), defined by immunolabelling: excitatory (SHANK2+); inhibitory (gephyrin+); IF+: total immunofluorescence. Data points: individual dendrites (11; 123 µm total, 351 IF+ synapses, b–f). d, Molecular properties of spine heads, with synapses identified by immunolabelling or PSDs. e, Molecular properties of shaft synapses. f, Excitatory (E) versus inhibitory (I) synapses onto dendrites. g, Spine seeding by excitatory axon outputs. h, Output density of spine-seeded axons (19 axons (data points) with two or more outputs, total length 990 µm). i, Spine fraction traced to parent dendrite for spines contacted by the same 19 axons (128 spines, mean ± s.d. j, Inhibitory axons seeded at an AIS (gephyrin+, no PSD). k, Output fraction onto shafts of AIS-seeded axons (eight axons with three or more outputs). l, Deep-learning prediction of bassoon and SHANK2 location from structural LICONN channel. Single plane from LICONN volume (CA1, stratum radiatum, not included in training), comparing prediction with immunolabelling. n, Excitatory input and output for a pyramidal neuron from the dataset in Fig. Synapse detections through bassoon and SHANK2 prediction mapped onto FFN segmentation. Magenta numbers: detected synapses between indicated branch points. Magnified views: structural channel, molecule predictions and cellular segments (partial proofreading, eliminating false-positive detections without adding missed detections). o, Connectivity prediction in hippocampal dataset from Fig. Middle, axon (black) with connected dendrites. Right, dendrite (black) with connected axons. p, Integration of structural, immunolabelling and deep-learning analysis. Dendrite (primary somatosensory cortex) with immunolabelling-based detection of excitatory (501) and inhibitory (80) post-synapses. Insets: presynaptic partners identified by deep-learning bassoon prediction. We next classified axons as excitatory according to structural and molecular criteria: we selected a spine and identified an axon synaptically connected to it (‘spine-seeded'), by requiring (i) a PSD in the structural channel and (ii) SHANK2 at the synaptic seed site. When we manually traced 19 such axons (total length 990 µm), they exhibited 0.16 ± 0.07 synaptic outputs per µm length (Fig. 4g,h), with 0.15 ± 0.07 per µm onto spines and 0.01 ± 0.01 per µm onto shafts, recapitulating the known preference of excitatory outputs for spines10. We used these axons as seeds for a further independent test of spine traceability (‘axon-seeded' spine tracing), again allowing us to trace the vast majority of spine necks (96%, 123 out of 128) to the parent dendrite (Fig. For inhibitory axons (defined by (i) seeding from a synapse at a dendritic shaft; (ii) lack of PSD; and (iii) presence of gephyrin), we found an inverse target preference (Supplementary Fig. Finally, we selected three cells in which the entire axon initial segment (AIS) was contained in the imaging volumes (Fig. AISs were characterized by a pronounced periodic labelling pattern in the structural channel, akin to the actin- and spectrin-induced lattice in neurites (Figs. We detected 0.25 ± 0.15 gephyrin-positive inputs per µm AIS. We selected axons that provided inhibitory input to AISs (gephyrin-positive, no PSD) and traced their outputs. The 8 axon stretches analysed with 3 or more outputs formed 44 synapses at dendrites (synapses onto soma or AIS not analysed), with a strong preference for shafts (92.5%; Fig. a, Molecular identification of interneurons through somatostatin immunolabelling (SST+, cyan). b, Ankyrin G immunolabelling (red) (cortex), with periodic protein-density modulation in structural channel, highlighting the AIS. Low-protein-density voids (straight arrows) around axons indicate myelination (Extended Data Fig. c,d, Primary cilia in LICONN (cortex, hippocampus; c) with intensely labelled centrioles (schematic; d) in the basal body. e, Immunolabelling for acetylated tubulin (magenta) and adenylate cyclase 3 (cyan) at a primary cilium (hippocampus, CA1) and overlay with the structural LICONN channel, with the membrane-bound adenylate cyclase signal ensheathing axoneme. f, Length of primary cilia (mean ± s.d. throughout) according to cell type in dataset from Fig. 1a (78 cells; box plots: median; lower and upper quartiles; whiskers: minimum and maximum (throughout); points: individual cells). g, Length of primary cilia in wild-type and Hnrnpu+/− mice (hippocampus, CA1, pyramidal layer; Supplementary Fig. h, LICONN volume (border of corpus callosum and alveus; Extended Data Fig. 9) with immunolabelling for acetylated tubulin (red), revealing multi-ciliated cells. i, Cross-section of cilium with ninefold symmetry in protein density, probably reflecting microtubule doublets, with ring diameter and doublet distance. j, GFAP (red, astrocytes) and glutamate receptor GLUN1 (magenta) immunolabelling with deep-learning bassoon prediction (cyan) in LICONN volume (hippocampus). Magnified images: astrocytic primary cilium at different z-planes, apposed to synaptic boutons. Scale bars, 1 μm (main image); 500 nm (right images). k, Immunolabelling for the astrocytic gap-junction protein connexin-43 (orange) and the inhibitory-synapse marker gephyrin (yellow) in LICONN (cortex). Magnified images, gap junction between astrocytes. Scale bars, 1 μm (main image); 500 nm (right images). LICONN with connexin-43 (orange) and (gephyrin, yellow) immunolabelling, deep-learning prediction of pre-synapses (bassoon, cyan) and excitatory post-synapses (SHANK2, magenta). Channels shown separately for boxed region. m, Density of gephyrin-positive inhibitory synapses and connexin-43-positive gap junctions (four volumes; cortex, hippocampal CA1). LICONN thus provided a natural means to integrate structural and molecular information to derive fundamental neuronal network parameters related to excitatory and inhibitory connectivity. To overcome limitations in microscopy hardware and imaging time from adding colour channels for further molecular targets, we used the correspondence between molecular and structural features to predict, rather than measure, the locations of synaptic molecules. We restricted prediction to excitatory synapses, which have more pronounced structural features than inhibitory synapses. Using deep learning, we predicted both bassoon at presynaptic sites and SHANK2 at excitatory post-synapses (Fig. Instead of human annotation, measurements of bassoon and SHANK2 (16,699 µm3, around 16,250 excitatory synapses), paired with the structural LICONN channel, were used to train neural networks21,46 that predicted the locations of molecules from the structural channel. When evaluated on datasets not included in training, predicted signals were highly consistent with ground-truth immunolabelling data (Fig. We converted molecule predictions to presynaptic and postsynaptic annotations and implemented post-processing (similar to that used in immunolabelling-based detection; Methods) to increase accuracy. Comparison of deep-learning-based synapse prediction with human ground-truth annotations yielded F1 scores higher than 0.9 for detecting synapses (hippocampus, CA1, stratum radiatum; pre-synapses: F1 = 0.94; post-synapses: F1 = 0.95; full synapses: F1 = 0.92, using 14,207 µm3 (around 13,650 synapses) for training and 180 µm3 for testing). We obtained a similar accuracy on test datasets with varied imaging parameters (z-step size) or brain region (cortex) (Supplementary Fig. These results show that the locations of excitatory synapses can be predicted with high fidelity from structural LICONN data. We then applied deep-learning-based synapse prediction to map the synaptic input field of an identified neuron in a dataset devoid of immunolabelling. We chose a pyramidal neuron from the FFN-based segmentation of the cortical dataset in Fig. 1a and related both pre-synapse (bassoon) and excitatory post-synapse (SHANK2) predictions to the cellular segmentation (Fig. The imaging volume contained 821 µm of this cell's neurites. The volume also contained 367 µm of the neuron's axonal output, with eight predicted pre-synapses. We next applied the prediction of bassoon and SHANK2 to map excitatory synaptic connectivity onto the densely proofread neuronal segmentation in the hippocampus from Fig. We detected 79,291 pre-synapses (0.95 per µm3) and 71,976 excitatory post-synapses (0.86 per µm3) and mapped them onto individual neurite segments (Fig. Visualizing one axon with its synaptic partners shows rich connectivity, with 31 synapses onto 28 different dendrites. Similarly, focusing on a single spiny dendrite, 284 axons established 303 synaptic contacts with it (267 single connections, 16 axons with 2 synapses and one with 4 synapses; Fig. We further created a ‘virtual' four-colour connectomic volume combining the LICONN structural channel, deep-learning bassoon prediction (for detecting pre-synapses) and super-resolved immunolabelling of gephyrin and SHANK2 (for molecular synapse differentiation), again in the somatosensory cortex (subset of dataset in Fig. We found 501 excitatory and 80 inhibitory synapses on a dendrite reconstructed over a 162-µm length (Fig. By contrast, on cell somata, inhibitory connections were dominant (Extended Data Fig. We further verified the robustness of deep-learning synapse prediction by replacing measured SHANK2 locations with predicted SHANK2 on the same dendrites and axons as in Fig. In this virtual four-colour volume (structure, measured gephyrin, predicted bassoon and predicted SHANK2), we obtained similar connectivity (Supplementary Fig. 22) to that found with SHANK2 immunolabelling, and quantifications of the balance of excitation and inhibition were in line with previous EM connectomics data8,10,47. We now sought to use both structural and molecular information in LICONN to characterize cell types and subcellular specializations. Similar to EM connectomics, analysing cell shape enabled an expert to classify cells in the cortical dataset in Fig. The relative proportions were comparable to those in EM data from the mouse visual cortex48. Adding specific immunolabelling molecularly identified cell types, including inhibitory interneurons expressing somatostatin (Fig. 23) or the voltage-gated potassium channel KV3.1b (Supplementary Fig. We then used molecular labelling to clarify the identity of subcellular structures and characterize their structural appearance. Labelling for ankyrin G molecularly highlighted AISs. We discovered that ankyrin G showed a lattice-like pattern at AISs, mirrored by a similar pattern in the protein-density map (Fig. This differentiated AISs from dendrites, which were mostly devoid of protein-density stripes near the soma. Similarly, labelling for myelin basic protein (Extended Data Fig. 5) clarified the identity of halo regions of low protein density around certain axons (Fig. 5b), unambiguously identifying them as myelinated axons. We corroborated their identity by immunolabelling for acetylated tubulin, a common post-translational modification of stable microtubules, and adenylate cyclase 3 (Fig. We quantified cilia length according to cell type (Fig. Ciliary function is essential for many aspects of brain development. In humans, mutations in the heterogeneous nuclear ribonucleoprotein U (HNRNPU) gene cause early-onset epilepsy, autistic features and intellectual disability. In the developing mouse brain, HNRNPU has been localized on primary cilia51. In addition, changes in the expression of genes associated with cilia organization have been observed in a human model system of induced-pluripotent-stem-cell-derived neurons from patients with HNRNPU mutations52. We hypothesized that mutations in Hnrnpu affect primary cilia length, and tested this in a haploinsufficient (Hnrnpu+/−) mouse model. We skeletonized primary cilia of pyramidal neurons in the hippocampal CA1 (Supplementary Fig. 5g), and concluded that, at least for hippocampal neurons, Hnrnpu mutations do not result in obvious defects in primary cilia (length (wild type): 10.3 ± 1.6 µm, length (mutant): 10.5 ± 1.6 µm, no statistically significant difference (t-test, P = 0.64)). Finally, we applied LICONN more broadly throughout the brain to assess tissue characteristics beyond synaptic connectivity. We confirmed its applicability in various regions, including the hypothalamus, piriform area and cerebellum (Extended Data Fig. When analysing white matter, including the corpus callosum—a key structure connecting the two hemispheres—and the alveus, we found layers with a high density of myelinated axons (Extended Data Fig. We were intrigued by nearby clusters of cells with multiple prominent processes (Fig. 5i), we realized that they corresponded to multiple cilia, originating from basal bodies, again with ninefold internal symmetry. Cilia were prominently highlighted when immunolabelling for acetylated tubulin (Fig. The microtubule ring had a diameter of 214 ± 27 nm (mean ± s.d. ), with individual doublets spaced 63 ± 9 nm apart (Fig. The ability to map electrical connectivity through gap junctions would be especially useful. This information is typically missing from connectomics datasets, because reliable detection of gap junctions requires particularly high resolution in EM53. We immunolabelled connexin-43, a gap-junction protein expressed in astrocytes54, and identified astrocytes by the expression of glial fibrillary acidic protein (GFAP) (Fig. GFAP labelling also molecularly assigned cellular structures bordering capillaries to astrocytes (Extended Data Fig. Complementing this with excitatory synaptic connections through deep-learning prediction of bassoon and SHANK2 (Fig. 5l) enabled simultaneous mapping of excitatory, inhibitory and electrical connectivity in the same circuit. Similarly, specific immunolabelling allowed us to determine the density of inhibitory (0.039 ± 0.010 per µm3, gephyrin) and electrical (0.054 ± 0.015 per µm3, astrocyte-associated connexin-43) connections (four imaging volumes across hippocampus and cortex, two each) (Fig. To axially extend LICONN volumes beyond the imaging depth accessible with high-NA objective lenses (0.6-mm working distance, in our case), we implemented a block-face imaging method to obtain LICONN volumes built from partially overlapping subvolumes arranged on a 3D grid, allowing seamless fusion without gaps (Extended Data Fig. We first imaged partially overlapping volumetric tiles arranged in two dimensions (2D) to obtain a first slab of LICONN data. We then used a conventional vibratome to slice off most of the imaged hydrogel layer, while placing the cut within the imaged slab. In a subsequent imaging round, we obtained an axially offset multi-tile volume situated more deeply in the tissue, featuring a continuous region of axial overlap with the first multi-tile volume. We then computationally fused the individual imaging volumes in 3D in a voxel-exact manner (Methods), enabling tracing of neuronal structures, including thin axons, across imaging slabs (Extended Data Fig. 10; original data for fused volume: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid146.json). To demonstrate the potential for further scaling, we iteratively applied this block-face imaging and sectioning approach across 12 rounds, with 108 volumetric imaging tiles arranged on a 3 × 3 × 12 grid in 3D. After voxel-exact fusion, the resulting volume covered 205 µm axially and allowed tracing of neuronal structures across tile borders in 3D (Extended Data Fig. 10 and Supplementary Video 4; see https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/multiround_fusion.json). In summary, LICONN is a straightforward technology that directly provides integrated structural and molecular characterization across brain regions, cell types and spatial scales. The development of connectomics methods has been driven by the ambition of simultaneously achieving dense 3D reconstruction of neurites, synapse-level connectivity, diverse molecular annotations and cost-effective scaling to large volumes (cubic millimetres or more). Successive generations of EM-based technologies have enabled enormous progress2, but important limitations remain, particularly in the ability to extract molecular details. By contrast, current light-microscopy techniques for visualizing cellular tissue architecture19,20,21,25,32, including LIONESS20 and CATS21, do not reach the accuracy and traceability required for connectomic reconstruction. Here we introduce LICONN, which, like EM, enables reliable manual tracing of dendrites, axons and spines (Fig. 1), high-accuracy automated reconstruction of those structures (edge accuracy 92.8%; Fig. However, unlike EM, LICONN enables direct and simultaneous measurement of spatially resolved molecular information, including specific proteins that reveal chemical synapse subtypes (Fig. 3), electrical synapses and key subcellular features (Fig. We used LICONN to acquire volumes of up to roughly 1 × 106 μm3 (native tissue scale; Fig. 1a), which are similar volumes (albeit with more limited axial extent) to those of previous EM-based connectomics datasets, on which biological analysis of neural circuits has been successfully performed1,10,47. We achieved reasonable acquisition rates (6.5-h acquisition time for around 1 × 106  μm3 native scale, 0.39 teravoxels, 17 × 106 voxels per s (17 MHz) effective voxel rate including overhead from tile overlap and sample stage positioning) even without optimizations for imaging speed20. Within the axial range of the working distance of the objective lens (here 600 µm), simple optical sectioning replaced the ultrathin (nanometre to tens of nanometres) physical sectioning or milling in EM reconstruction. However, the largest EM datasets span a cubic millimetre, and a major long-term goal is mapping the hundreds of cubic millimetres of an entire mouse brain55. One promising strategy for scaling LICONN volumes includes hydrogel sectioning to parallelize readout. LICONN hydrogels are mechanically robust, facilitating handling and sectioning. We typically expanded 50-µm-thick slices for experimental convenience, but this does not constitute a fundamental limitation. We expect that scaling LICONN expansion to larger samples will be possible by adjusting denaturation, polymerization and labelling parameters. 10), we obtained LICONN imaging volumes that appeared, judged from the structural channel, equivalent in resolution, signal-to-noise ratio and structural preservation to datasets used for extensive validation of traceability. We further showed that consecutive, overlapping imaging volumes in the axial direction can be acquired and computationally fused by removing slabs of the expanded LICONN hydrogel already imaged. Such axial extension of volumes will be required for comprehensive analysis of cellular connectivity, whereas lateral tiling is more straightforward (for example, around 3,000 tiles, approximately 1-mm2 native scale in Extended Data Fig. We have also shown the detection of specific proteins by immunolabelling. Using post-expansion labelling, LICONN benefits from improved epitope access22,29 and avoids fluorophore-to-epitope linkage error. Further refinement will expand LICONN's molecular information content. For example, hydrogel-compatible spatial transcriptomics methods have emerged56,57,58,59 that measure gene expression directly in tissue. By integrating connectivity with in situ molecular information from individual cells, LICONN presents a viable path towards multimodal descriptions of mammalian brain cells, including morphology, connectivity (including electrical connections), physiology and gene expression60. We further applied LICONN to quantify cellular properties beyond synaptic connectivity, focusing on cilia. Analysing primary cilia in a model of neurodevelopmental disease exemplifies how LICONN can be used to study genotype-to-phenotype relationships and cellular alterations in diseased brains. LICONN was developed to reconstruct arguably the most complex tissue structure, specifically identifying and tracing the finest neuronal processes, such as axons and spines, in the brain; we expect the technology to be broadly useful in other organs and systems in which high-resolution tissue analysis is desirable. Finally, we note that LICONN is highly accessible. Acquisition is driven by broadly available conventional light-microscopy hardware (here, spinning-disc confocal), and, although LICONN sample preparation introduces new strategies to achieve high-fidelity tissue expansion, the protocol is not fundamentally more complex than previous expansion techniques that have been widely adopted. Deep-learning-based analysis used FFNs and other deep-learning frameworks that have previously been applied in neuronal segmentation and synapse prediction. These and the custom code we developed here are available open source. In conclusion, LICONN forms a technological basis for the routine adoption of connectomic studies in non-specialized neuroscience labs, as well as enabling high-resolution studies in organs other than the brain. Mice were housed in groups of three to four under controlled laboratory conditions (12:12-h light–dark cycle with lights on at 07:00; 21 ± 1 °C; 55 ± 10% humidity) with food (pellets, 10 mm) and autoclaved water ad libitum. Mice were housed in commercially available individually ventilated cages made from polysulfone with a solid cage floor, dust-free bedding (woodchips) and nesting material. For all experiments, male and female mice were used interchangeably to demonstrate the technology. Adult mice (aged typically two to three months, unless otherwise noted) were used as indicated with the following genotypes: C57BL/6J wild-type mice, Thy1-eGFP (STOCK Tg(Thy1-eGFP)MJrs/J mice, 007788, RRID:IMSR_JAX:007788 hemizygous) and haploinsufficient Hnrnpu+/− mice (deletion of one allele of Hnrnpu spanning axons 4 to 14, generated by crossing the HnrnpUWT/flox line (Hnrnpu<tm1.1Tman>/J, strain: 032187, RRID: IMSR_JAX:032187) with the CMV-CreCre/Cre line (B6.C-Tg(CMV-cre)1Cgn/J, strain: 006054, RRID: IMSR_JAX:006054). Solutions were prepared on the same day and kept at room temperature. Mice were first anaesthetized with isoflurane (1–2% (volume/volume; v/v) and then with ketamine (80–100 mg per kg body weight) and xylazine (10 mg per kg) intraperitoneally, combined with metamizol (200 mg per kg) subcutaneously for analgesia. After checking for deep anaesthesia by toe pinch, they were transcardially perfused at a flow rate of 7 ml min−1 using a peristaltic pump, first with room-temperature 1× phosphate-buffered saline (PBS) for 2 min and then with a solution (room temperature) containing 4% (weight/volume; w/v) paraformaldehyde (pH 7.4) and 10% acrylamide (AA, w/v) in 1× PBS for 6 min. Brains were collected and post-fixed for 8–12 h at 4 °C with gentle agitation, using the same fixative solution. Brains were washed three times in cold (4 °C) 1× PBS for around 1 min each, with gentle agitation. Where indicated, sections were prepared at a 300-µm thickness. Sections were placed in ice-cold 1× PBS supplemented with 100 mM glycine to quench PFA-reactive groups for 6–8 h at 4 °C and washed three times for around 1 min in 4 °C 1× PBS. They were stored at 4 °C in 1× PBS supplemented with 0.015% NaN3 for up to three months, typically. Brain sections were washed twice for 15 min each in 1× PBS at room temperature with gentle agitation. They were then washed twice for 15 min each in 100 mM sodium bicarbonate in milli-Q water, pH 8.0, at room temperature followed by incubation with 0.1% (w/v) TGE and 0.1% (w/v) GMA in 100 mM sodium bicarbonate in milli-Q water (pH 8.0) for 3 h at 37 °C with gentle agitation in a chemical hood. For 300-µm-thick slices, the TGE and GMA concentration was increased to 1% (w/v). Samples were then washed with 1× PBS for 1 h with gentle agitation at room temperature. Pre-expansion immunolabelling was performed only for distortion analysis measurements. All other immunolabellings were performed after expansion. Brain slices were permeabilized for 60–80 min with 0.2% Tween-20 in 1× PBS at room temperature with gentle agitation. Labelling with primary anti-GFP antibody was performed in 5% BSA and 0.2% Tween-20 in 1× PBS overnight at 4 °C with gentle agitation. Samples were then washed three to five times for around 30 min each in 1× PBS at room temperature with gentle agitation, followed by staining with secondary antibody overnight at 4 °C, in the same buffer as for the primary antibody labelling, with gentle agitation, followed by three to five washes for around 30 min each in 1× PBS with gentle agitation. Pre-expansion images were acquired on a spinning-disc confocal microscope after the first gelation step. For preparation of the first hydrogel solution, AA and sodium acrylate (SA) were mixed in milli-Q water, vortexed and centrifuged at 4,500g for 5 min. Supernatant was transferred to a fresh 50-ml tube and supplemented with N,N′-methylenebisacrylamide (BIS) stock solution (prepared in milli-Q water) to a final BIS concentration of 0.075% w/v. After adjustment to the final volume, solutions were aliquoted in 2-ml tubes and stored at −20 °C, typically for up to one month. Solutions for the stabilizing and second expandable hydrogels were prepared similarly, usually freshly before experiments. We observed that the quality of sodium acrylate was crucial for successful expansion, mirrored in a batch dependency for this specific compound. In an ice-water bath, the first hydrogel monomer solution (Supplementary Table 5) was supplemented first with 0.001% 4-hydroxy-2,2,6,6-tetramethylpiperidin-1 (TEMPO, w/v) and then with 0.15% ammonium persulfate (APS, w/v) and 0.15% N,N,N′,N′-tetramethyl-ethylenediamine (TEMED, volume/volume; v/v) and vortexed. The following sample-handling steps were performed in an ice-water bath. Brain sections were pre-incubated with the hydrogel monomer solution for 30–45 min (75–90 min for 300-µm-thick slices) with gentle agitation in a 24-well plate. A gelation chamber was assembled from two cover glasses on bottom and top and #1 cover-glass strips on either side as spacers (for 300-µm sections: one #1 and one #1.5 cover-glass strip). Gelation was then performed at 37 °C in a humidified chamber for two hours. For optionally acquiring pre-expansion images, the sample was imaged on a spinning-disc confocal microscope (Oxford Instruments, Andor Dragonfly) after the first hour of gelation for around 15 min, and then further incubated at 37 °C for another approximately 45 min. Together with the bottom coverslip, it was then transferred to a 5-ml beaker with 2 ml denaturation buffer (200 mM SDS, 200 mM NaCl and 50 mM Tris-HCl in milli-Q water, pH adjusted to 9.0 with NaOH). The beaker was placed in a larger vessel containing pre-heated water and transferred to a water bath at 95 °C for 100 min. For 300-µm-thick samples, the denaturation time was extended to four hours, starting at 85 °C and ramping up to 95 °C within about 15 min. For expansion, samples were placed in milli-Q water, with the water being changed every 20 min (approximately) until no further increase in gel size was observed. We neutralized unreacted double bonds of the divinyl cross-linker (BIS) remaining after polymerization of the first hydrogel to ensure that it would not react with the following stabilizing and second expandable hydrogels. For this, we incubated with 0.2% APS plus 0.2% TEMED in milli-Q water for 2.5 h at 37 °C with gentle agitation, followed by one washing step with agitation in milli-Q water for 30 min at 37 °C and a similar washing step at room temperature. Expanded hydrogels were again trimmed to the region of interest and incubated in 1× PBS for 10 min at room temperature, shrinking the gel in a 12-well plate. It was then incubated with primary antibodies (Supplementary Table 3) in 1× PBS at 4 °C overnight with gentle agitation in a 24-well plate, followed by at least three washes for one hour each in 1× PBS with gentle agitation at room temperature. Secondary antibody incubation (Supplementary Table 4) was also performed in 1× PBS at 4 °C overnight with gentle agitation, followed by at least three washes for one hour each in 1× PBS with gentle agitation at room temperature. Post-fixation was conducted for 15 min at room temperature with 4% PFA in 1× PBS, followed by quenching with 100 mM glycine in 1× PBS for 10 min at room temperature with gentle agitation and subsequent washing three times in 1× PBS for around 3 min each. Hydrogels were re-expanded in milli-Q water in a 12-well plate before further processing, and labelling was optionally evaluated by imaging. During the LICONN procedure, cellular proteins are anchored to the hydrogel and epitopes can undergo denaturation and structural modification. We tested further primary antibodies by imaging directly after the immunolabelling; that is, without applying the second expansion step, as summarized in Supplementary Table 6. Note that not all of the antibodies that produce labelling in the first hydrogel may be retained throughout the entire sample preparation procedure. Dedicated optimization of heat and chemical denaturation and antibody incubation conditions may improve performance for individual antibodies. For somatostatin labelling, we applied a mixture of primary antibodies to increase the signal strength. Expanded hydrogels were pre-incubated on ice-water for 3–3.5 h or 5 h for 50-µm-thick and 300-µm-thick slices, respectively, with the monomer solution of the stabilizing hydrogel (10% (w/v) AA, 0.025% (w/v) BIS, 0.05% (v/v) TEMED and 0.05% (w/v) APS in milli-Q water) with gentle agitation in a 12-well plate. After removing excess monomer solution, the hydrogel was sandwiched between a 22 × 22-mm2 coverslip placed on a microscopy slide and an 18-mm round coverslip on top of the hydrogel without spacers, and surrounded by monomer solution. Gelation was then performed for two hours at 37 °C in a pre-warmed (37 °C) humidified chamber. Gels were washed with milli-Q water for around 30–60 min at room temperature. To neutralize remaining unreacted double bonds of the BIS cross-linker in the stabilizing hydrogel network and ensure that the hydrogel network did not react with the following second expandable hydrogel network, the hydrogel–tissue hybrid was incubated with 0.2% APS and 0.2% TEMED in milli-Q water for 2.5 h at 37 °C with gentle agitation, followed by washing with agitation in milli-Q water for 30 min at 37 °C and a second washing step at room temperature. Stabilized and neutralized hydrogels were pre-incubated on ice-water with the monomer solution for the second expandable hydrogel (19% sodium acrylate, 10% AA, 0.025% (w/v) BIS, 0.05% TEMED and 0.05% APS in milli-Q water) in a 12-well plate for 3–3.5 h (50-µm slices), or 5 h (300-µm slices). After removing excess monomer solution, the hydrogel was again sandwiched between a 22 × 22-mm2 coverslip placed on a microscopy slide and an 18 × 18-mm2 coverslip on top of the hydrogel without spacers, and surrounded by monomer solution. Gelation was then performed for two hours at 37 °C in a pre-warmed (37 °C) humidified chamber. Hydrogels were washed with 1× PBS for around 30 min at room temperature. Pan-protein staining was performed with either 40 µM ATTO 488 NHS ester or 40 µM Alexa Fluor 488 NHS ester (Supplementary Table 7) in 1× PBS overnight at 4 °C with gentle agitation. Hydrogels were optionally washed three times for one hour in total with 1× PBS, and expanded for up to five hours before imaging in milli-Q water, with fluid exchange approximately every hour. Before mounting for imaging, the region of interest was located with the spinning-disc confocal microscope in the expanded hydrogel after placing it onto a plastic culture dish with the bottom replaced with a 50-mm #1 cover-glass. Round coverslips (40 mm #1.5; Bioptechs) were rinsed with milli-Q water, covered with poly-l-lysine (PLL) and incubated at 37 °C for up to three hours. Coated coverslips were stored at 4 °C in 1× PBS for up to one week. Right before mounting, coverslips were washed with milli-Q water and placed in a home-built imaging chamber made from aluminium. The hydrogel was placed in the centre of the coverslip and further stabilized with two-component dental silicone (Twinsil extrahart, Picodent), and immersed in milli-Q water. We imaged for up to eight hours, typically. Imaging was performed on an Andor Dragonfly microscope based on a Nikon Ti2E inverted stand with motorized stage and an Andor Zyla 4.2 Megapixel sCMOS camera (2,048 × 2,048 pixels). Data were acquired using Andor Fusion software v. 2.2. Imaging of non-expanded and expanded samples was performed with a 40× water-immersion objective (Nikon Apochromat LWD 40× lambda S, NA 1.15, water, working distance (WD) 0.6 mm), using the 40-µm disc pattern. 23, 24), a Nikon CFI P-Apochromat 20×, NA 0.95, WD 0.95-mm objective lens was used. The structural channel was imaged with a 488-nm excitation wavelength with a 521/38-nm detection bandpass filter. Typical exposure times were 110–150 ms and the laser power was set to 10–15% of available laser power. The lateral field of view (FOV) was typically 307 × 307 µm2, corresponding to a biological tissue scale of around 20 × 20 µm2. For tiled measurements, the ‘field' mode was used with 10% lateral overlap between individual subvolumes. For additional imaging of immunolabelling channels, colour channels (with excitation wavelengths of 488 nm (structural channel), 637 nm and 561 nm) were recorded sequentially in a frame-wise manner with exposure times of 120–200 ms per immunolabelling channel and typical laser power settings of 10–20%. Images for distortion analysis were acquired with a 150 × 150 × 200-nm3 voxel size with the 40×, NA 1.15 water-immersion objective lens. To seamlessly stitch imaging volumes not only laterally but also in the axial direction, we performed the following ‘lossless' sectioning procedure, ensuring overlap of imaging volumes both laterally and axially. We expanded 300-µm-thick brain tissue sections and mounted them on PLL-coated coverslips for imaging as described above. The vibratome chamber was partially filled with water, surrounding but not covering the hydrogel. We zeroed the vertical blade position on the hydrogel surface and lowered it to the desired cutting position, chosen to fall within the already imaged region (for example, cutting 320 µm below the hydrogel surface after imaging an axial range of 466-µm physical imaging range in the two-round measurement in Extended Data Fig. Cutting was performed at 0.2 mm s−1 blade advancement. For the multi-round measurement, we iteratively repeated this procedure. When establishing the procedure, we suggest that the protocol is adhered to closely, including details such as the precise chemicals and providers listed, the timing of steps, concentrations, temperatures at which individual steps are performed (for example, on ice-water versus at room temperature) and preparation of fresh solutions directly before experiments where indicated. 1–5, 7–9 and Supplementary Table 8), we suggest checking the following points in the case of suboptimal results. Quality of transcardial fixative perfusion is crucial to achieve adequate tissue preservation, including, for example, cell shape. If tissue preservation is suboptimal, basic parameters such as perfusion speed and duration, and composition of solutions, should be checked. We anecdotally observed that when setting the pH of the perfusion solution, large pH variations should be avoided and it was preferable to carefully titrate in one direction until the final pH was reached. If the final expansion factor is lower than expected, over-fixation might be a factor (owing to an excessive duration of the post-fixation step, for example). Similarly, excessive protein anchoring might interfere with full expansion. Variation of cross-linker (BIS) concentration is also expected to have a direct effect on the expansion factor. For sodium acrylate, we observed a batch-to-batch variation in compound quality (for example, impurities), which could have an effect on the final expansion factor and overall outcome. For the other chemicals used, we did not make obvious similar observations. Shortening the time for expansion (for example, one hour instead of four to five hours for the final hydrogel) might also compromise the final expansion factor. Premature gelation of the second or third hydrogel might take place if the same reaction well is used for performing the preceding neutralizing step (removing unreacted double bonds) involving APS and TEMED, even if the well is washed. If the signal-to-noise ratio in the structural labelling is unsatisfactory, protein anchoring and labelling with NHS esters of fluorophores merit scrutiny. NHS esters are prone to hydrolysis. NHS esters of fluorophores should be dissolved only once needed in high-quality (water-free) dimethyl sulfoxide and aliquoted in single-use quantities (10-µl aliquots of 20 mM stock solution stored at −20 °C) to avoid freeze-thaw cycles, and should be used within two to three months. To monitor success at the various steps of the procedure during the set-up phase, it might be useful to perform imaging at various check-points, including after the first expansion step (applying NHS labelling), after immunolabelling and after full expansion. Observing these guidelines produced highly consistent results. We followed the outcome of all transcardial fixative perfusions performed within a certain time period (n = 15 mice) from perfusion to expansion and final imaging. In total, the 15 perfusion experiments yielded 50 experimental runs that typically contained several technical replicates (hydrogel embedding and expansion experiments). In one out of 15 transcardial fixative perfusion experiments, the fixative perfusion itself failed, producing poor tissue preservation in the resulting samples, which is likely to be related to the suboptimal quality of the perfusion solution. In one technical replicate, we observed premature gelation of the stabilizing hydrogel, probably owing to remnants of APS and TEMED in the same well plate used during the neutralization step. The other replicates produced useful imaging data either for demonstrating various technical aspects (for example, determining the expansion factor, distortion analysis and testing of immunolabelling) or for the full LICONN procedure, as required. We selected a subset of these for further analysis and display, on the basis of visual inspection for factors such as the signal-to-noise ratio. Raw imaging data were obtained in IMS format, as generated by the spinning-disc confocal microscope (Andor Dragonfly), which is a hierarchical data format with six resolution levels, from which we extracted the highest resolution level (level 0). Data conversion for downstream analysis was performed with custom Python scripts implemented in Python v.3.8 or higher, including the Imaris-ims-file-reader, zarr, webKnossos and tifffile packages. Custom scripts are available at https://github.com/danzllab/LICONN, with individual package versions specified in the respective configuration (.yml) files. Datasets are available through Neuroglancer (see Data availability section) or through the data repository of the Institute of Science and Technology Austria (ISTA)61 at https://research-explorer.ista.ac.at/record/18697. At various processing steps, conversion of file formats was necessary, and this was performed on ISTA's high-performance computing cluster, using either a single node or SLURM (slurm-wlm 22.05.8) for task allocation, according to the resource requirement based on file size. We transferred files to cluster storage and converted from the respective source format to tiff, zarr, n5 or webKnossos (wk) format according to processing needs. Imaging data were processed using Fiji v.1.54f, including the CLAHE, BigWarp and BigData viewer plug-ins. For visualization, display ranges were adjusted, also accounting for a camera background signal of around 100. Intensity lookup tables were linear unless otherwise noted. The background in immunolabellings was set to transparent (alpha = black) and immunolabellings were overlaid with the structural channel. The output was saved in PNG format. CLAHE (contrast-limited adaptive histogram equalization) was applied as indicated for individual figure panels, using either Fiji or a Python-based workflow as described below. The maximum slope was set to 3 and the standard (not fast) processing option was chosen. Three-dimensional renderings of cilia were done with Imaris v.9.3. We analysed a total of 14 imaging volumes of around 20 × 20 µm2 laterally and 9–15 µm axially, recorded across 4 technical replicates in n = 3 Thy1-eGFP mice, exhibiting cytosolic expression of eGFP in a sparse subset of neurons. Imaging volumes corresponded to single tile measurements (no stitching or fusion of subvolumes). For distortion analysis, immunolabelling against eGFP was performed before subjecting the samples to the LICONN expansion procedure, and the eGFP channel was used for analysis. Pre-expansion images were acquired after around one hour of polymerization of the first hydrogel. Imaging volumes containing the same structures were acquired after expansion of the first hydrogel (before application of the stabilizing hydrogel) and after expansion of the second expandable hydrogel, with a spinning-disc confocal microscope and a 40×, NA 1.15 water-immersion objective lens, as in the LICONN imaging. This allowed us to assess distortions beyond the approximately 200-nm spatial scale. Using the BigWarp63 plug-in in Fiji, we first manually placed around 25 landmarks at corresponding neuronal features in maximum intensity projections of the respective pre- and post-expansion imaging volumes. Using BigWarp, we then applied either a similarity transformation (including isotropic scaling, translation and rotation) or an affine transformation (including scaling, translation, rotation and shearing) to the pre-expansion images to best match landmarks in pre- and post-expansion images. We smoothed pre- and post-expansion projection images with Gaussian filters of different widths to account for the different resolution in the respective images. For individual expansion steps, σ was set to 1 and 6 pixels for pre- and post-expansion images, respectively. For total (iterative) expansion, σ was set to 1 and 16 for pre- and post-expansion images, respectively, resulting in a comparable appearance as judged by visual inspection. We now used the imregdemons.m function in MATLAB (v.R2022b, MathWorks) to compute distortion vector fields62,64 from the aligned pre- and post-expansion images. For this, we used the images before the respective expansion step as a reference and evaluated distortions arising from that expansion step. For display purposes, we used the quiver function in MATLAB with vector scaling set to 1.5. We then computed and plotted distortions (measurement error) for different measurement lengths, as previously described16. For this, we generated binary masks of neuronal structures of interest by thresholding pre- and post-expansion images, and then combining, dilating and eroding the resulting masks using parameters as judged by visual inspection. We then randomly sampled 2 × 105 pairs of points restricted to the masked area of interest. We then found a corresponding set of transformed vectors by applying the distortion field generated with imregdemons.m to the respective pairs of points. For each measurement length, we then computed the measurement errors as lengths of the difference vectors between the pre-expansion and transformed vectors combined from three or four different datasets within one experiment. For plotting, measurement errors at a given measurement length were averaged and the mean was plotted together with the standard deviation. From these measurement errors, the root mean square (RMS) error was calculated across different measurement scales. Finally, the mean and standard deviation of RMS errors across the n = 4 technical replicates were plotted for individual expansion cycles and transformations. For display purposes, we smoothed the measurement error curve with a one-dimensional median filter. We extended SOFIMA (https://github.com/google-research/sofima, git hash 64d5c7c) to support seamless stitching of 3D tiles. In brief, we processed every volumetric tile with CLAHE, applied independently to every in-plane (xy) section of the tile. The tiles had an in-plane size of 2,048 × 2,048 voxel2 and an axial extent varying depending on the experiment. We laid out the tiles on a 2D grid according to imaging metadata, and used a coarse positioning step to find their initial location in the coordinate system of the output volume. This was followed by fine alignment to elastically deform them to obtain seamless matches between adjacent tiles and form 3D sections, around 20–30-μm thick. The tiles were then rendered with linear-distance-weighted blending of overlapping image content. For the coarse positioning step, we extracted a small (50-voxel)3 cube of image content from the ‘preceding' tile of every adjacent tile pair, and used cross-correlation to identify the corresponding image content in the ‘following' tile, forming an offset vector between the tiles. For fine alignment, we modelled every tile as an elastic mesh of unit masses placed on a regular grid with a 3D stride of 20 voxels and nearest neighbour masses connected with Hookean springs. Coarse positioning resulted in the neighbouring tiles partially overlapping with each other, and for any mass located in the overlapping image region, we again used cross-correlation to identify the position within the neighbouring tile of a an (80 voxel)3 image patch centred on the mass (PATCH_SIZE=80). These formed point correspondences between the tiles, which we modelled as 0-length Hookean springs, and allowed the whole system to relax. For datasets comprising more than one thick section (Extended Data Fig. 10), we first stitched tiles into thick sections as described above, and then aligned them to each other sequentially. We then modelled the following section as a 3D spring mesh, and allowed it to relax according to the flow field. This is similar to the ‘fine alignment' of individual tiles, and the overall strategy is a direct 3D analogue of how alignment of 2D sections is done with SOFIMA for volume EM datasets. Details of processing were as follows. In all cases, we based stitching and fusion on the structural imaging channel. We first transferred individual raw IMS files (uint16 format) to cluster storage and extracted the highest resolution level (level 0). We then used a BigStitcher-based Fiji macro to concatenate the data and convert to n5 format. We then applied CLAHE tile-by-tile with Python. Data handling was done with the TensorStore library (https://google.github.io/tensorstore/, v.0.1.33). We used the numpy.clip function to clip the intensity range plane by plane typically between 120 and 350, determined by visual inspection. For CLAHE, we used skimage.exposure.equalize_adapthist to apply histogram equalization, with a clip limit of 0.03. We multiplied the output (ranging from 0 to 1) with 255 and saved as uint8 format. We set the QUERY_R_ORTHO and QUERY_R_OVERLAP parameters to 25. The second, fine-stitching, step was used to warp tiles and achieve a smooth transition at tile borders, using a stride of (20, 20, 20) voxels. This step outputs a mesh that ensures a globally consistent warping and relative arrangement of individual tiles. In the last step, we applied the obtained mesh to either the CLAHE-modified structural imaging data for downstream (automated) segmentation or skeletonization, or the raw structural imaging data, as well as to immunolabelling channels, resulting in the respective fused imaging volumes in n5 format. We performed these processing steps on ISTA's high-performance computing cluster and/or, in the case of the measurement in Fig. 10, on Google's high-performance computing infrastructure, requesting resources according to the processing task. Typically, for CLAHE, we requested 16 CPUs (Intel Xeon CPU E5–2680 v4, 2.40 GHz or similar) and 150 GB of RAM. For coarse stitching, we typically requested 3 GPUs (GeForce RTX 2080 Ti or similar), 32 CPUs and 100 GB or RAM. Fine stitching was done with 1 GPU, 16 CPUs and up to 970 GB of RAM. Rendering was typically done requesting 32 CPUs and up to 700 GB of RAM. For manual segmentation, we used VAST v.1.4.0 (downloaded from https://lichtman.rc.fas.harvard.edu/vast/) with the data in eight-bit format. Segmentation of the data in Fig. 1g was originally performed by one person, and the resulting segmentation was used for training. 1g, segmentation was partially proofread by another person using VAST v.1.4.1. We used webKnossos v.22.05.1 for manual skeletonization, running on a local computer with 2× AMD EPIC MILAN 75F3 processor, 32-core 32C/64T, 2.95 GHz with 512 GB of memory, 2 TB NVMe-SSD and an additional 27 TB NVMe-SSD, as well as 4 NVIDIA A6000 GPUs. Before conversion, we clipped the intensity range according to visual inspection, to account for intensity outliers and avoid signal overflow. For convenient exploration of the datasets across spatial scales, we made use of ten precomputed resolution levels, using the webKnossos downsample function. When comparing automated segmentation of the volume in Fig. 2a with manually generated skeletons, we checked sites of discrepancy and corrected skeletons as appropriate. For testing traceability by comparison with an independently generated ground truth, we compared manual tracings of neurites in the structural LICONN channel with their structure revealed by cytosolic expression of eGFP in Thy1-EGFP mice. The LICONN structural channel and the eGFP channel were recorded at a similar spatial resolution after tissue expansion with a spinning-disc confocal microscope. The structural channel was recorded in the 488-nm excitation channel, whereas eGFP was detected by post-expansion immunolabelling as described above with imaging at 561 nm excitation. Voxel sizes were either 150 × 150 × 200 nm3 or 150 × 150 × 300 nm3, with the largest step size referring to the z direction. Data were converted to wk format for tracing. Using the webKnossos access permissions feature, we assigned permissions to ensure blinding as required. For generating ‘ground-truth' skeletons of eGFP-expressing neurons, both the structural and the eGFP channels were loaded and considered jointly. Two novice human tracers were trained in a non-blinded manner with both the eGFP and the structural LICONN channels of four datasets (single volumetric imaging tiles, around 19 × 19 × 19 µm3), containing, in total, two eGFP-expressing axons and two eGFP-expressing dendrite stretches. These datasets were excluded from further analysis. For dendritic spines, annotators were asked to indicate these as short branches diverging from the main dendrite trunk. Human consensus skeletons were then overlaid with and compared to the eGFP-based ground truth using webKnossos. We used FFNs (available at https://github.com/google/ffn, git hash 12d680e)42 to automatically segment the datasets. We used the original convolutional stack architecture with a 33 × 33 × 33-voxel FOV, a step size of 8 voxels and the network depth extended to 20 residual modules. The models were applied to data with 19.4 × 19.4 × 25.9-nm3 voxel size (in original tissue scale). For this, CLAHE-processed volumetric imaging data were 2× downsampled laterally relative to the imaging voxel size using volume averaging in Python. Axially, datasets with a 200-nm axial-imaging step size (effective voxel size 9.7 × 9.7 ×13.0 nm3, obtained by scaling with an exF of 15.44; Fig. 2a) were also downsampled twofold in the z direction, whereas datasets with a 400-nm axial step size (effective voxel size 9.7 × 9.7 × 25.9 nm3; Fig. 1a) were not downsampled in the z direction. Training was performed with a batch size of 128 and a learning rate of 10−4 using the AdamW65 optimizer with 32 NVIDIA V100 GPUs, for up to 30 days, which corresponded to about 8.8 million training steps. A Jupyter notebook for performing FFN inference on LICONN data is available at https://github.com/google/ffn/blob/master/notebooks/ (git hash: 12d680e). We used an iterative approach to collect the volumetric ground-truth annotations for training the FFNs. First, 162 neurite fragments covering 110 Mvx (megavoxels, at full imaging resolution) were manually annotated as described above, using VAST in the dataset shown in Fig. Model 1 trained with these data on full-resolution imaging data was used to segment the multi-tile volume in Fig. We then manually agglomerated 103 neurite fragments covering 447 Mvx (voxel number referring to full imaging resolution) within that segmentation (for a description of manual agglomeration, see below). We used this to train another FFN model (model 2), using both the newly collected data and the pre-existing manually painted annotations for training with voxels 2× downsampled in every direction (19.4 × 19.4 × 25.9-nm3 effective voxel size). Within the segmentation results of model 2, we manually proofread a larger set of axons (540 axons, 920 Mvx at full imaging resolution, 115 Mvx downsampled, 17.8-mm cumulative path length) and dendrites (314 dendrites, 3,344 Mvx at full imaging resolution, 418 Mvx downsampled, 23.8-mm cumulative path length), and used these data to train the final (model 3) FFN model that was used to produce all the automated neurite segmentations in the manuscript. We followed a segmentation strategy optimized for first reducing the number of segments affected by merge errors (‘base segmentation') and then agglomerating these mostly merge-free supervoxels into larger neurites, as described in the following sections. For the base segmentation, we applied the seed-order oversegmentation consensus procedure42, followed by oversegmentation consensus with two more segmentations generated with alternative FFN check-points (snapshots of weights saved at regular time intervals during training), which were manually selected for having a small number of wrong merges. 2a to zero, whereas the overall volume contained at least five mergers. For determining edge accuracy42, a skeleton edge was counted as ‘accurate' when both nodes were labelled identically and the label did not correspond to a segment overlapping more than one skeleton. We automatically skeletonized the base segmentation for downstream use in agglomeration and proofreading using the TEASAR algorithm66 (https://github.com/seung-lab/kimimaro, version b390a9abdde60ec81472f1901e7980a962ecd847) and heuristically identified all nodes of degree 1 in the resulting skeleton graphs as ‘end-points'. To automatically classify segments into distinct subclasses—in particular, axons and dendrites—we developed a semantic segmentation pipeline. For this, an experimenter classified manually agglomerated objects in a segmentation of a 512 × 512 × 334-voxel subvolume (at the downsampled 19.4 × 19.4 × 25.9-nm3 effective voxel size) of the dataset in Fig. We used these annotations to train a neural network model to perform semantic segmentation (voxel-wise multi-class prediction) of the structural LICONN data. The model was trained with asynchronous stochastic gradient descent at a learning rate of 10−4, using 16 NVIDIA V100 GPUs and a batch size of 16. Training lasted approximately 38 h, during which 40.7 million steps were executed. Training points from all five classes were sampled at equal frequencies. We used the trained model to generate a semantic segmentation of the entire volume in Fig. These voxel-wise classifications were then aggregated over all voxels of every instance segment, and the class predicted most frequently was associated with that segment. The instance segmentation pipeline described above is optimized to make it unlikely that an individual super-voxel incorrectly covers more than one neurite. This happens at the cost of an increased number of split errors; that is, assigning multiple smaller segments to the same neurite. To mitigate this, we applied a multistep agglomeration procedure in which the FFN model was used to evaluate whether a pair of geometrically proximal segments should be merged together42 and how a segment could be further extended from a heuristically detected end-point in the skeletons generated from the base segmentation9. We used the same training data as for the main instance segmentation model, but split the data into two non-overlapping sets, so that one model was trained exclusively on axons, and the other exclusively on dendrites. Within the prediction results for every end-point, we determined the segment S with the highest recovery fraction (other than the segment E containing the end-point) and agglomerated S and E if that fraction was at least 50%, and if S had an end-point within 100 voxels of the end-point of E. We applied the dendrite model to all skeleton nodes of dendritic trunks (skeletons generated from the base segmentation), taking any dendritic segment of at least 100,000 voxels to be a trunk. For every evaluated point, we collected any putative spine (dendritic segment of less than 100,000 voxels) associated with at least a 50% recovery fraction. We then agglomerated all spines to the trunk for which it had the highest recovery fraction. The agglomeration procedure resulted in a graph with 516,437 edges, in which each edge represents one pair of automatically joined base segments. The semantic segmentation model allowed us to associate per-class voxel counts with all base and agglomerated segments. Specifically, we sorted the edge list in decreasing order of the associated scores, and discarded any edge that would cause a merge between inconsistent classes. An (agglomerated) segment was considered to belong to one of these composite classes when it had a total size of at least 100,000 voxels, and at least 75% of its voxels were classified as the target class. This criterion was relaxed to 10,000 voxels and 50%, respectively, for the end-point agglomeration step, in which we also disallowed merging together objects larger than 100,000 voxels. The automated segmentation of the dataset obtained as a result of the agglomeration procedure was expected to have remaining split and merge errors, which we decided to correct in a structured manual proofreading workflow, leading to a marked increase in proofreading speed. Focusing on topological split and merge errors rather than painting voxels, the workflow was implemented as a Python script driving the Neuroglancer viewer67 (https://github.com/google/neuroglancer, c466b24), within which an expert annotator interactively inspected segments and the structural channel images in both 2D cross-sections and as 3D mesh rendering. The tool had features for automatically moving the 3D cursor to predetermined locations (such as end-points of segments), marking segments for future view and directly editing the agglomeration graph by adding or removing edges. All axon segments (n = 2,674) of at least 10,000 voxels and touching a single face of the bounding box of the volume were inspected for splits and manually connected to appropriate partner segments where necessary. The tool was configured to automatically navigate to heuristically detected end-points of the axon segments. All axons within the volume (n = 18,667) were reviewed for shape implausibilities resulting from putative merge errors, which were then inspected and corrected. Dendritic spines not connected to a trunk (n = 2,782) were reviewed and connected to a trunk when one could be unambiguously identified. All dendritic branches (n = 1,643) were reviewed for incorrectly associated spines, which were separated, and subsequently manually connected to the correct trunk. To automatically detect pre- and postsynaptic sites in volumetric datasets comprising the LICONN structural channel and immunolabelling channels for bassoon and SHANK2, respectively, we loaded the multicolour image stacks in tiff format. We converted image stacks for molecular signals to 32-bit floating numbers and adjusted the contrast to a minimum of 101 to account for camera background, and the maximum to the 99.5th and 99th percentile for the dataset in Fig. Background was subtracted from signal and negative values were set to zero. We then applied Otsu thresholding and used the skimage.measure.label function for converting the resulting binary mask into instance segmentations. The list of maximum intensity values had a bimodal distribution, which we used to distinguish between synaptic regions (high intensity in structural channel) and unspecific immunolabelling (low intensity in structural channel). To separate the two populations, we again used Otsu thresholding. To remove further non-specific detections, we performed additional size filtering. Segments that occupied less than 60 voxels were removed, as well as objects that spanned fewer than a certain number of planes in the z direction (300-nm z-step size: <13; 200-nm z-step size: <14 for bassoon and <12 for SHANK2; 400-nm z-step size: <12 for SHANK2). Listed parameters were determined from a grid search on a separate validation dataset or a subvolume of the analysed dataset for each of the z-step sizes, choosing the parameter set that resulted in the highest F1 score. For detecting gephyrin-positive post-synapses, we applied a simplified algorithm involving background removal, Otsu thresholding, morphological opening and size filtering, and manually proofread the output. For validating automated synapse detection, a human annotator created ground-truth synapse annotation in a dataset comprising the LICONN structural data, bassoon immunolabelling and SHANK2 immunolabelling. Bassoon signals without corresponding SHANK2 signals, which are likely, in large part, to correspond to inhibitory synapses, were annotated as single source points. Similarly, at volume edges, instances of SHANK2 signals without corresponding bassoon signals occurred, which were annotated as single target points. The human annotator also indicated synaptic connections that lacked SHANK2 signal but showed a clear PSD in the structural channel as excitatory post-synapse. The special cases of a single post-synapse connected to more than one pre-synapse or a single pre-synapse connected to multiple post-synapses were handled by text comments. In total, three datasets were annotated. One test dataset from the hippocampus had a 300-nm z-step size and comprised 700 × 700 × 200 voxels of size 9.7 × 9.7 × 19.4 nm3. This dataset contained 261 pre-synapses and 247 post-synapses, forming 242 full synapses (with one-to-many connections contributing a count of 1 with each edge), with 25 occurrences of unpaired pre-synapses and 7 unpaired post-synapses. The second test dataset from the hippocampus had a 200-nm z-step size and comprised 1,000 × 1,000 × 600 voxels of size 9.7 × 9.7 × 13.0 nm3. This dataset contained 1,116 pre-synapses and 1,084 post-synapses, forming 1,059 full synapses (with one-to-many connections contributing a count of 1 with each edge), with 57 occurrences of unpaired pre-synapses and 25 unpaired post-synapses. The third test dataset was from cortex, had a 300-nm z-step size and comprised 1,322 × 1,322 × 170 voxels of size 9.7 × 9.7 × 19.4 nm3. This dataset contained 306 pre-synapses and 262 post-synapses, forming 261 full synapses (with one-to-many connections contributing a count of 1 with each edge), with 48 occurrences of unpaired pre-synapses and one unpaired post-synapse. We generated further validation datasets for parameter tuning, which were not included in the test data. To compare the automatically generated point annotations of bassoon or SHANK2 signals with the corresponding human ground-truth point annotations, we performed spatial matching of the respective point annotations. For this, we defined a cost matrix with entry Cij corresponding to the Euclidean distance between automatically generated point annotation i and manually generated point annotation j. We defined a matching limit rmatching; that is, an upper distance beyond which matching was not allowed, and set it to rmatching = 30 voxels after stepping this parameter in a test dataset. For Cij > rmatching, we set this matrix entry to an arbitrary large number. We now used the scipy.optimize.linear_sum_assignment function to solve the linear assignment problem of finding corresponding points; that is, the single-point to single-point matches with minimum Euclidean distances. Pairs matching between automatically and manually generated annotations were classified as true-positive automated detections. Automated detections without corresponding ground-truth element within the matching radius rmatching were classified as false-positive detections. Ground-truth detections without a corresponding match in the automated detection were classified as false-negative detections. To connect corresponding presynaptic sites automatically identified by bassoon immunolabelling with postsynaptic sites automatically identified by SHANK2 immunolabelling, we first defined a cost matrix with entries Sij corresponding to the Euclidean distances between presynaptic annotations j and postsynaptic annotations i. We then defined a binary matching matrix, Mij, with Mij = 1 indicating a connection between the respective pre- and postsynaptic sites (else 0). Solving this as a linear assignment problem as above would result exclusively in 1:1 matches, which would not capture cases in which one pre-synapse is connected to multiple post-synapses or vice versa. We therefore developed a pipeline that gave priority to the spatially closest occurrences of bassoon and SHANK2 during the presynaptic–postsynaptic pairing procedure but that also allowed for one-to-many connections. In a first pass, we paired every postsynaptic site with the closest presynaptic sites; that is, we identified the smallest Sij within each row of the cost matrix and set the corresponding Mij to 1 if Sij was below a maximum matching distance of dmatching = 30 voxels. This value was taken from the distance distribution between manually generated pre- and postsynaptic annotations, choosing a value close to the observed maximum distance of 31 voxels. Hence, each post-synapse was at most associated with one pre-synapse, whereas each pre-synapse could be associated to zero, one or more post-synapses. In a second pass, we now exclusively considered those bassoon signals that were left unpaired in the previous run (‘floating' pre-synapses); that is, bassoon occurrences j that did not have Mij = 1 after the first pass. This identified the previously unpaired pre-synapses that had SHANK2 signal in close spatial proximity; that is, within the matching limit. First associating the closest pre- and postsynaptic sites in this way did not allow additional connections between pre- and postsynaptic detections that were already assembled into a synaptic connection in a more closely matching pair, which would occur if we simply matched all pre- and postsynaptic occurrences within the maximum matching distance dmatching. We classified automated detections of synaptic connections (that is, a connection between an individual presynaptic annotation and an individual postsynaptic annotation) as true positive if the following criteria were fulfilled: (i) true-positive detection of a bassoon occurrence; (ii) true-positive detection of a SHANK2 occurrence; and (iii) an automatically detected connection between these two that was also present in the manually annotated connections. We classified automated detections of synaptic connections as false positive if (i) a presynaptic and a postsynaptic detection were automatically connected but were not connected in the manual annotation or (ii) a false-positive molecule detection (bassoon or SHANK2 prediction) was assigned a synaptic connection. We classified detections of synaptic connections as false negative if a manually annotated synaptic connection was not present in the automated detection of synaptic connections. A typical scenario for this was a missed automated detection of pre- or postsynaptic sites, which can also happen if two neighbouring bassoon occurrences (or SHANK2 occurrences) are blurred into each other during the smoothing step and thus undersegmented. For this, we automatically generated two lists of detections: one that included objects spanning fewer than the specified number of planes in the z direction and a second in which these structures were removed. To account for those cases in which a particular synapse was not retained during one of the processing steps for automated synapse detection, we implemented an automated post-processing step for the respective test datasets (Supplementary Fig. Such cases included failed detection in cases in which the synaptic immunolabelling was either weak or absent at a particular synapse and therefore did not meet the cut-off threshold (for example, owing to lack of SHANK2 expression), or the sampling of the structural LICONN feature did not pass the global threshold, or size filtering. Here, we identified unpaired pre- and postsynaptic detections, defined a local search area around those and checked whether bright structural features were present in the LICONN channel around those unpaired detections. For cases in which a prominent PSD or bright presynaptic feature was present, we classified them as additional pre- or postsynaptic detections, and added them to the respective lists of pre- and post-synapses, including them in the presynaptic–postsynaptic matching algorithm. In brief, we checked individual rows and columns of the matching matrix Mij that only contained zeros, corresponding to unmatched post- and pre-synapses, respectively. We then defined a 100 × 100 × 100-voxel3 bounding box around those locations (or smaller at imaging volume edges). We then applied intensity rescaling to the structural LICONN channel between p1 and p99.95, where p1 refers to the first percentile and p99.95 to the 99.95th percentile. We then removed background using Gaussian filters of two different widths at σ = 2 voxels and σ = 10 voxels, subtraction and clipping of negative values to zero. For the dataset in Supplementary Fig. We then removed previously detected pre- and post-synapses by multiplying the resulting mask with a mask corresponding to an inverted semantic segmentation of the previously detected pre- and post-synapses. We then did a binary opening (eroding and then dilating) with a ball of two-voxel radius. We then performed instance segmentation with skimage.measure.label, removed structures smaller than 20 voxels and determined the weighted centroid on the intensity distribution in the underlying structural LICONN channel. We classified the resulting objects as a new pre- or postsynaptic detection when their Euclidian distance from the originally unpaired pre- or postsynaptic site was smaller than the matching limit rmatching. We used these to update the list of pre- and postsynaptic detections used for the association to full synapses. We devised a strategy for identifying the locations of pre-synapses and post-synapses from the structural imaging channel alone. We used deep-learning-based prediction of synapse location with an approach that did not require human input for the generation of training data. For this, we implemented a U-net architecture for image translation to predict the locations of synaptic molecules from structural data, using measured molecule locations as input to the training. We then converted the predicted molecule locations to automatically generated synapse detections, for this step using a pipeline that was similar to the immunolabelling-based pipeline above. We converted 3D super-resolved measurements of synaptic-molecule locations (based on specific immunolabelling), paired with the structural imaging channel, to training data with the following workflow. We first converted TIFF images to 32-bit floating format and split them into individual datasets for the structural and each of the immunolabelling channels, maintaining the original voxel size. We then implemented a preprocessing pipeline to automatically remove unspecific immunolabelling signals, similar to the approach taken for the immunolabelling-based synapse detection described above. However, unlike in that pipeline (in which we performed a strong smoothing step to avoid oversegmentation of the spatially structured bassoon and SHANK2 distributions), we here preserved the spatial distribution of the bassoon and SHANK2 signals more closely, allowing the deep-learning network to recapitulate the molecule arrangement in the respective synaptic compartments, including the lattice-like arrangement of bassoon, paralleled by high-intensity (protein-rich) structural features. We then removed background by applying isotropic Gaussian filters of two different widths using scipy.ndimage.gaussian_filter, with σ = 0.5 voxels and σ = 10 voxels, corresponding to signal and background, respectively, subtracting background from signal and clipping negative values to 0. We applied Otsu thresholding to generate a binary mask and used the skimage.measure.label function to derive an instance segmentation. We applied Otsu thresholding to distinguish segments with underlying high-intensity LICONN features, which we classified as specific immunolabellings, and without high-intensity features, which we classified as non-specific immunolabellings. We next removed the latter and converted the remaining segments back to an overall binary mask. For the structural channel, we applied simple intensity normalization to image pixel values im (im' = (im − min)/(max − min)), where min is the 0th percentile and max is the 100th percentile of the intensity distribution. We saved the paired immunolabelling and structural data in zarr format, allowing resource-efficient handling of large datasets. We implemented deep-learning pipelines with Pytorch v.1.12.1 (https://pytorch.org) and used the Gunpowder framework v.1.2.2 (https://github.com/funkelab/gunpowder) to implement our data loading, augmentation, training and prediction pipeline that conveniently allowed processing of big datasets. For predicting molecule location, we used the 3D U-net architecture from a previous study46. We used Adam's optimizer (torch.optim.Adam) with a 10−4 learning rate. We performed network training on ISTA's high-performance computing cluster, using SLURM for task allocation. We typically requested 8 CPUs, graphics acceleration by 1 GPU (NVIDIA GeForce) and 700 GB of RAM. We trained on datasets with a 300-nm z-step size (6 individual tiles, total volume of 16,699 µm3, containing around 16,250 excitatory synapses) and applied the resulting model to datasets with a 200-nm, 300-nm or 400-nm z-step size. For selecting a model for the different z-step sizes, we applied models from various check-points and took the best-performing check-point when evaluating F1 on the respective test datasets. For the 300-nm z-step size, we used the 20,000 iterations check-point for both bassoon and SHANK2 prediction. During the validation phase for the 300-nm z-step size prediction, training data comprised 5 of the 6 tiles (total volume of 14,207 µm3, with around 13,650 excitatory synapses), and one dataset was used for testing. For final training, all six datasets were included. During training, patches of training data (normalized between 0 and 1 with gp.normalize) were randomly sampled from the training datasets with equal probability and used in training if at least 0.1% of voxels were included in the binary mask generated during preprocessing of molecular signals (that is, 1,048.6 voxels of the 64 × 128 × 128-voxel patches). Training data were augmented by mirroring and transposing in the xy direction on structural and molecular channels. In addition, we applied intensity augmentation on the structural channel with (0.9, 1.1, −0.1, 0.1) for the two scaling and the two intensity shifting parameters, respectively, and enabled range clipping. Prediction was performed on ISTA's high-performance computing cluster using SLURM for task allocation. We requested 8 CPUs, graphics acceleration by 1 GPU (NVIDIA GeForce RTX 2080 Ti or similar) and 100 GB of RAM. Prediction was performed in a piece-wise, ‘scanning' mode, again with a patch size of 64 × 128 × 128 voxels after converting LICONN structural data to zarr format while applying the same normalization procedure as for the training data. We then applied 3D Gaussian filtering with σ = 5 for bassoon and SHANK2. We applied Otsu thresholding to derive a binary mask and skimage.measure.label to extract instance segmentations. We erased volume segments smaller than 60 voxels and those that covered fewer than 13 imaging planes. Association of pre- and postsynaptic sites and validation of deep-learning-based synapse prediction were performed with the same pipeline as was used for immunolabelling-based synapse detection. Neurite and synaptic segmentations, skeletons and imaging volumes were visualized in 3D with Blender versions 2.92, 3.2 or 3.51 (https://www.blender.org/). Video files were also generated with Blender. For visualizing neurites on an overview scale (Figs. 1a, 2c,d), precomputed meshes from Neuroglancer were used. 1e,f, 3c) were based on meshes generated with the marching cubes implementation in scikit-image. For 3D rendering of molecular signals in the context of neuronal segmentations, immunolabelling channels and predicted molecule distributions were intensity scaled according to visual inspection, background was removed with σ ≈ 0.5 and σ ≈ 5 to preserve shapes and then Otsu thresholding was applied. To associate these signals with a particular neuronal segment, the segment was expanded with a ball of radius 6 and any molecular signals with at least 1 voxel overlap were retained. Meshes were again generated with the marching cubes algorithm. To map detected synapses onto selected neuronal segments, we first expanded the segment with a ball of radius of 5 or 6 voxels. To map postsynaptic sites onto dendrites, we selected those detections whose centre coordinates were located within the expanded neuronal segment. Owing to the large size of the imaging volume, computations were performed on manually defined patches, and results were combined for visualization and further analysis. 4p, we first performed deep-learning-based synapse prediction on the imaging volume. For this, we predicted bassoon and SHANK2 molecule locations in smaller overlapping patches of the volume, and combined them into the final full-size prediction volume. Next, we converted the predicted molecule locations into synaptic point annotations using the algorithm described above. Because of the large size of the volume, pre-synapse and post-synapse annotations were computed on manually defined non-overlapping patches. Association of pre- and post-synapses—that is, computation of the matching matrix Mij—was performed on the entire volume to avoid errors in full synapse assembly owing to separation by a patch boundary. We then associated pre- and postsynaptic annotations with the neuronal segmentation. For each pre-synapse (or post-synapse) location, we identified the underlying segment ID from the FFN-based neurite segmentation. If no segment was present at the coordinates of a specific pre- or postsynaptic detection, we performed an automated search in a single plane for the nearest non-zero segment in a 20 × 20-voxel neighbourhood. For this, we iterated through the matching matrix Mij and added a connection between the segment associated with post-synapse i and the segment associated with pre-synapse j. GraphPad Prism v.10.1.2 was used for statistical analysis and plotting. 21 was performed as described previously68, based on code available at https://gitlab.mpcdf.mpg.de/connectomics/human_primate. This is a proof-of-concept study focusing on the development of new technology and its applicability. Accordingly, no prior determination of statistical sample size was performed. Once the technology was established and the data quality was to our satisfaction, experiments were performed in multiple replicates to ensure and demonstrate reproducibility. For the comparison of primary cilia length in wild-type versus Hnrnpu+/− mice (Fig. 5g), we chose n = 3 biological replicates in each group, recorded across a total of n = 7 technical replicates to demonstrate that a biologically meaningful number of replicates can be analysed with LICONN. No randomization between experimental groups was performed. The comparison of cilia length between wild-type and mutant mice (Fig. 5g) was performed with an unpaired two-tailed t-test (P = 0.64; not significant). Measurements of cellular parameters (for example, distance measurements within synapses, periodicity and connectivity) were performed to show that quantitative information that is consistent with previous measurements can be extracted from LICONN data. Quantification of segmentation accuracy and synapse-detection fidelity were performed to evaluate how well computational algorithms could reproduce human ground-truth annotations. In these cases, blinding was not possible or appropriate. For determining the accuracy of human tracings relative to independent ground-truth information from sparse eGFP expression, blinding to that independent ground-truth information was performed. Blinding was implemented through the permissions settings in webKnossos. To confirm the reproducibility of the technology, we performed a series of technical replicates, which were typically recorded across several biological specimens, as indicated below. For analyses, only datasets of high labelling and imaging quality were pursued. Typically, during the development phase, a number of experiments with some variation of parameters were performed. In all images, representative data from single experiments are shown. For some of the experiments that we performed routinely, the stated numbers give lower bounds and we did not count further replicates beyond n = 10. 1a,b: imaging data representative of acquisition of expansion and multi-tile imaging in n = 10 replicates across n = 3 mice. 1b represents subregion of dataset from Fig. 1c: 192 synapses analysed with multiple line profiles per synapse, in n = 3 technical replicates across n = 2 mice. 6: single-tile LICONN measurements were performed with a large number of technical and biological replicates (n ≫ 20). 16: multi-tile LICONN measurements were performed in n = 10 replicates. FFN-based segmentation was applied to n = 6 datasets. Comprehensive proofreading of all neuronal structures (Fig. 2a–g) and quantification of segmentation results using manually generated reference skeletons (Fig. 3c: visualization from the dataset in Fig. Bassoon and SHANK2 labelling was performed in n = 5 replicates across 3 mice. 3d: synaptic immunolabellings were replicated n = 2 times, with further immunolabellings for synaptic targets in various contexts. 3e: 185 synapses analysed in 3 imaging volumes from n = 1 technical replicate. 19: immunolabelling-based, automated synapse-detection accuracy was evaluated against manually generated ground truth in n = 3 technical replicates recorded across n = 2 mice. 3i: visualization of synapse detections was performed for n = 1 dendrite. 3j,k: immunolabelling for inhibitory synapses was performed in n = 5 technical replicates. 20, 21: manual connectivity analysis was performed across 2 imaging volumes (total 154,560 µm3) in n = 1 technical replicate (n = 1 mouse). 4h,i: output analysis and spine tracing was performed for n = 19 axons. 4k: output was analysed for n = 8 AIS-seeded axons. 19: accuracy of deep-learning-based synapse prediction was evaluated against manually generated ground-truth annotations in n = 3 technical replicates recorded across n = 2 mice. 4n: mapping of synaptic inputs and outputs was performed in n = 1 cell and represents further analysis of the dataset in Fig. 4p: visualization of excitatory and inhibitory inputs was done for n = 1 dendrite. 5f: analysis of 78 cells from dataset in Fig. 5i: Cilia diameter: 52 cross-sections; microtubule doublet distance: 54 distance measurements, analysed across n = 2 technical replicates in n = 1 mouse (5 imaging volumes). 5: immunolabellings were reproduced in n = 2 replicates. 5l: representative of immunolabelling in n = 3 replicates. 5m: gap-junction and inhibitory-synapse density evaluated across n = 4 imaging volumes in cortex and hippocampus from n = 1 mouse. 8: imaging of the various layers was performed in n = 1 mouse. 9: data representative of replicates in n = 2 mice. 10c–f: post-expansion hydrogel sectioning was performed in n = 3 technical replicates. 10h: block-face imaging and sectioning over 12 rounds was performed in n = 1 specimen. 1: data points correspond to individual technical replicates. 1f: a single hydrogel was subdivided after the first gelation step. Expansion factors for the second expandable hydrogel were determined in the following technical replicates: replicate 11: n = 3; replicates 13, 7, 14 and 1: n = 4. 2: data representative of n = 3 technical replicates. 3: each condition was technically replicated n = 2 times. Hydrogels 14 and 15 were replicated n = 5 times. Number of mice analysed: Supplementary Fig. 5: data representative of n = 3 technical replicates. 7: data representative of n = 3 technical replicates. 8: data representative of n = 2 technical replicates. 9: data representative of n = 3 technical replicates for each condition. at given measurement length from sampling 2 × 105 pairs of points in pre- and post-expansion images (see Methods) from the specific single measurement (n = 1 imaging volume) shown in the figure panel. 10b,c: RMS measurement error over different measurement lengths for the first and second expansion steps and for the overall LICONN procedure were evaluated as mean ± s.d. across n = 4 technical replicates (using a total of 14 imaging volumes), recorded in cortex across n = 3 mice. 18: data representative of n = 10 technical replicates. Visual illustration of resolution improvement by iterative expansion as shown in the figure was performed in n = 1 specimen. 20, 21: manual tracings were performed across 2 imaging volumes from n = 1 specimen (n = 1 technical replicate) imaged in the high-resolution overview scan. 22: tracing with deep-learning prediction of synapses was performed in n = 1 technical replicate. 23, 24: immunolabellings were technically replicated n = 2 times. 25: high-resolution overview imaging of the region analysed in Fig. Figures were created in Adobe Illustrator v.27.5 and v.27.7. 5d and brain cross-section schematics in Supplementary Fig. Hnrnpu+/− mice are available from the authors upon request. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Original data and/or segmentations for example datasets are available in browsable format through Neuroglancer, as follows. Full segmentation with comprehensive proofreading of neuronal structures: Fig. Additional datasets with lateral and/or axial fusion: Fig. 1a,b: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/fig1a.json; Extended Data Fig. 10c–f: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/expid146.json; and Extended Data Fig. Source data are provided with this paper. Custom code for synapse detection, connectivity analysis and distortion analysis is available at https://github.com/danzllab/LICONN. FFNs are available at https://github.com/google/ffn, including a Jupyter notebook for performing FFN inference on LICONN data (https://github.com/google/ffn/blob/master/notebooks/jax_ffn_inference_liconn.ipynb, git hash: 12d680e). SOFIMA is available at https://github.com/google-research/sofima (git hash 64d5c7c). Jefferis, G., Collinson, L., Bosch, C., Costa, M. & Schlegel, P. Scaling up Connectomics: The Road to a Whole Mouse Brain Connectome. & Denk, W. Progress and remaining challenges in high-throughput volume electron microscopy. Scheffer, L. K. et al. A connectome and analysis of the adult Drosophila central brain. The connectome of an insect brain. Dorkenwald, S. et al. Neuronal wiring diagram of an adult brain. Functional connectomics spanning multiple areas of mouse visual cortex. Shapson-Coe, A. et al. A petavoxel fragment of human cerebral cortex reconstructed at nanoscale resolution. Loomba, S. et al. Connectomic comparison of mouse and human cortex. Fang, T. et al. Nanobody immunostaining for correlated light and electron microscopy with preservation of ultrastructure. & Briggman, K. L. Permeabilization-free en bloc immunohistochemistry for correlative microscopy. Han, X. et al. Multiplexed volumetric CLEM enabled by scFvs provides insights into the cytology of cerebellar cortex. Holler, S., Köstinger, G., Martin, K. A. C., Schuhknecht, G. F. P. & Stratford, K. J. Structure and function of a neocortical synapse. Sahl, S. J., Hell, S. W. & Jakobs, S. Fluorescence nanoscopy in cell biology. Shen, F. Y. et al. Light microscopy based approach for mapping connectivity with molecular specificity. & Hell, S. W. Fluorescence microscopy with diffraction resolution barrier broken by stimulated emission. Tønnesen, J., Inavalli, V. V. G. K. & Nägerl, U. V. Super-resolution imaging of the extracellular space in living brain tissue. Dense 4D nanoscale reconstruction of living brain tissue. Imaging brain tissue architecture across millimeter to nanometer scales. Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. Truckenbrodt, S. et al. X10 expansion microscopy enables 25 nm resolution on conventional microscopes. & Danzl, J. G. A practical guide to optimization in X10 expansion microscopy. Damstra, H. G. et al. Visualizing cellular and tissue ultrastructure using ten-fold robust expansion microscopy (TREx). Klimas, A. et al. Magnify is a universal molecular anchoring strategy for expansion microscopy. Park, H. E. et al. Scalable and isotropic expansion of tissues with simply tunable expansion ratio. Revealing nanostructures in brain tissue via protein decrowding by iterative expansion microscopy. Light microscopy of proteins in their ultrastructural context. Mao, C. et al. Feature-rich covalent stains for super-resolution and cleared tissue fluorescence microscopy. M'Saad, O. et al. All-optical visualization of specific molecules in the ultrastructural context of brain tissue. Korogod, N., Petersen, C. C. & Knott, G. W. Ultrastructural analysis of adult mouse neocortex comparing aldehyde perfusion with cryo fixation. Structural and molecular interrogation of intact biological systems. Pallotto, M., Watkins, P. V., Fubara, B., Singer, J. H. & Briggman, K. L. Extracellular space preservation aids the connectomic analysis of neural circuits. Park, Y.-G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. Expansion microscopy using a single anchor molecule for high-yield multiplexed imaging of proteins and RNAs. Tillberg, P. W. et al. Protein-retention expansion microscopy of cells and tissues labeled using standard fluorescent proteins and antibodies. Expansion microscopy with conventional antibodies and fluorescent proteins. Januszewski, M., Blakely, T. & Lueckmann, J.-M. SOFIMA: Scalable optical flow-based image montaging and alignment. Xu, K., Zhong, G. & Zhuang, X. Actin, spectrin, and associated proteins form a periodic cytoskeletal structure in axons. Januszewski, M. et al. High-precision automated reconstruction of neurons with flood-filling networks. Mishchenko, Y. et al. Ultrastructural analysis of hippocampal neuropil from the connectomics perspective. Bartol, T. M. Jr et al. Nanoconnectomic upper bound on the variability of synaptic plasticity. Dani, A., Huang, B., Bergan, J., Dulac, C. & Zhuang, X. Superresolution imaging of chemical synapses in the brain. Ounkomol, C., Seshamani, S., Maleckar, M. M., Collman, F. & Johnson, G. R. Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy. Dense connectomic reconstruction in layer 4 of the somatosensory cortex. Elabbady, L. et al. Perisomatic ultrastructure efficiently classifies cells in mouse cortex. Sheu, S.-H. et al. A serotonergic axon–cilium synapse drives nuclear signaling to alter chromatin accessibility. Ott, C. M. et al. Ultrastructural differences impact cilia shape and external exposure across cell classes in the visual cortex. Sapir, T. et al. Heterogeneous nuclear ribonucleoprotein U (HNRNPU) safeguards the developing mouse cortex. Deficiency of the heterogeneous nuclear ribonucleoprotein U locus leads to delayed hindbrain neurogenesis. Lin, J. H.-C. et al. Gap-junction-mediated propagation and amplification of cell injury. Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Eng, C.-H. L. et al. Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+. Wang, G., Moffitt, J. R. & Zhuang, X. Multiplexed imaging of high-density libraries of RNAs with MERFISH and expansion microscopy. Scaled, high fidelity electrophysiological, morphological, and transcriptomic cell characterization. Danzl, J. et al. Research data for publication ‘Light-microscopy-based connectomic reconstruction of mammalian brain tissue'. Institute of Science and Technology Austria https://doi.org/10.15479/AT:ISTA:18697 (2025). Sun, D. et al. Click-ExM enables expansion microscopy for all biomolecules. & Saalfeld, S. Robust registration of calcium images by learned contrast synthesis. Image matching as a diffusion process: an analogy with Maxwell's demons. In International Conference on Learning Representations (ICLR 2019) https://openreview.net/forum?id=Bkg6RiCqY7 (OpenReview, 2019). Sato, M., Bitter, I., Bender, M. A., Kaufman, A. E. & Nakajima, M. TEASAR: tree-structure extraction algorithm for accurate and robust skeletons. 8th Pacific Conference on Computer Graphics and Applications 281–449 (IEEE, 2000). Loomba, S. Connectomic Comparison of the Human and Non-Human Primate Cortex to Mouse. We acknowledge support from ISTA's scientific service units: Imaging and Optics, Lab Support, Scientific Computing, the preclinical facility, the Miba Machine Shop and the library. We acknowledge funding from the following sources: Austrian Science Fund (FWF) grant DK W1232 (J.G.D. ); Austrian Academy of Sciences DOC fellowship 26137 (M.R.T. ); Gesellschaft für Forschungsförderung NÖ (NFB) grant LSC18-022 (J.G.D. Open access funding provided by Institute of Science and Technology (IST Austria). Mojtaba R. Tavakoli, Julia Lyudchik, Vitali Vistunou, Nathalie Agudelo Dueñas, Jakob Vorlaufer, Christoph Sommer, Caroline Kreuzinger, Bárbara Oliveira, Alban Cenameri, Gaia Novarino & Johann G. Danzl Google Research, Mountain View, CA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar and M.J. developed FFN-based segmentation methods, which were applied by M.J. M.J. analysed segmentations, developed volume fusion methods and processed data. ISTA filed a patent application covering the expansion technology, with M.R.T. are employed by Google, which sells cloud computing services. The remaining authors declare no competing interests. Nature thanks Narayanan Kasthuri, Joergen Kornfeld, Valentin Nägerl and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Magnified views indicate examples of glial cells, primary cilia used in further analysis (Fig. Spines of a particular dendrite were only comprehensively traced in the right part of the dendrite indicated in the magnified panel ii. Hippocampal overview composed of 2,969 high-resolution tiles from spinning-disc confocal microscopy with delineation of subregions and layers. a, Mossy fibre boutons forming synapses with complex spines (thorny excrescences) at proximal dendrites of CA3 pyramidal neurons in hippocampus, interspersed with bundles of mossy fibres (non-myelinated axons of DG granule cells). Single plane of a LICONN imaging volume including the structural channel (grey) and immunolabelling for VGLUT1 (vesicular glutamate transporter 1, magenta), a synaptic vesicle marker in excitatory synapses, and the active-zone marker RIM1/2 (cyan). Bottom: Immunolabelling and structural channels displayed separately and as overlay for the two mossy fibre boutons and their postsynaptic partners in the box in the top panel. b, Similar measurement with immunolabelling for presynaptic bassoon and the postsynaptic scaffolding protein PSD95, present at excitatory synapses. Reconstruction of four example cell somata from LICONN datasets in somatosensory cortex with immunolabelling for SHANK2 and gephyrin. Excitatory (magenta) and inhibitory (yellow) synaptic inputs are mapped onto cell body reconstructions. Synapses were classified as excitatory if (1) a typical presynaptic structure, (2) a PSD and (3) SHANK2 were present. Synapses were classified as inhibitory if (1) a presynaptic structure was present and (2) a PSD was missing, with or without gephyrin. For each cell body, the number of excitatory and inhibitory connections are indicated, showing a much higher number of inhibitory inputs. We analysed 13 cell bodies (contained ≥50% within the imaging volume) across 8 datasets and counted 48.7 ± 19.8 (mean ± s.d.) inhibitory synapses per soma (excluding the one cell with reversed input preference displayed on the right). Excitatory synapses were sparse, with most somata receiving 0 and one soma receiving 1 excitatory input. a, Single plane of a LICONN imaging volume in the hippocampal CA1 stratum radiatum, including the structural channel (grey) and immunolabelling for myelin basic protein (magenta) and the astrocytic gap-junction protein connexin-43 (cyan). Magnified views highlight structures expressing the respective molecular markers: (i) Immunolabelling for myelin basic protein identifies myelin sheaths, where the regions largely devoid of signal in the structural LICONN channel correspond to lipid-rich myelination. b, Single plane from LICONN imaging volume in hippocampus, showing the structural channel (grey) and immunolabelling for GFAP (orange) and connexin-43 (cyan). GFAP is an intermediate filament protein, commonly used as astrocyte marker. c, Single plane from a LICONN imaging volume in the CA3 stratum lucidum, with additional immunolabelling for vimentin (orange, here highlighting the endothelial lining of a blood capillary) and the peroxisomal marker PMP70 (70 kDa peroxisomal membrane protein). Note that peroxisomes do not produce pronounced contrast in the structural (protein density) channel but can be identified by specific labelling. Single example planes from LICONN imaging volumes in hypothalamus, piriform area and cerebellum. In the cerebellum, additional immunolabelling for bassoon and SHANK2 was applied. Single-plane overview image with magnified views, illustrating the complex arrangement of cellular structures including a primary cilium and intracellular organelle diversity in (i) and mossy fibre bundles and an example of multiple shaft synapses in (ii). b, Single planes from individual LICONN imaging volumes spanning deep cortex, corpus callosum, alveus and hippocampus according to the schematic, revealing the distinct organization in the various regions. Imaging of the various layers was performed in n = 1 mouse. b, 3D rendering of a LICONN imaging volume (top, recorded with a 20× objective lens) of the region indicated by the box in the overview, highlighting multi-ciliated cells. The bright structures correspond to basal bodies. Right: Single imaging plane showing multi-ciliated cells and magnified view of cilia with intensity lookup table adjusted to visualize the axonemes of cilia. Representative of replicates in n = 2 mice. a, LICONN in 300 µm-thick brain slices (hippocampus, CA1, stratum radiatum). Single imaging planes in subregions of multi-tile volume, with data quality equivalent to thinner tissue sections. In round 1, a multi-tile volume is acquired, then most of the imaged hydrogel layer is removed with a vibratome, and in round 2 a second multi-tile imaging volume deeper in the tissue is acquired. A region of continuous overlap recorded in both imaging rounds enables lossless stitching and fusion of imaging volumes across rounds. c, Axial (xz-)view of the LICONN volume of panel a, fused from 72 subvolumes arranged on a 6x6x2 3D grid. Data were acquired in two imaging rounds at increasing depth (data displayed after coarse stitching). d, Single-plane overview of the same volume in lateral (xy-)direction (109x109 µm2 native scale) after volume fusion. e, Single-plane axial (xz-)view after volume fusion in the same dataset, with seamless fusion across the two imaging rounds. f, Single-plane lateral (xy-)view of a subarea in the axial overlap region between imaging rounds, after volume fusion. Homogeneous colouring indicates voxel-exact matching from individual imaging rounds after fusion. g, 3D-illustration of manually traced axons in the final stitched and fused volume, demonstrating traceability across borders between imaging rounds. h, Single axial plane of LICONN volume fused from 108 subvolumes arranged on 3x3x12 grid in 3D, acquired in 12 consecutive block-face imaging/sectioning rounds, and 3D-illustration of manually traced axons. Axons extend across tile borders in 3D, from axon entering to axon leaving the imaging volume. Axial extent of final fused volume: 205 µm original tissue scale (7926 optical sections, 400 nm z-steps in post-expansion volume). Original data (CLAHE-processed) of final fused volume: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/multiround_fusion.json. 1–25, providing further information on LICONN optimization, analysis and application. Fly-through of the original data in Fig. 1g, with step-by-step appearance of manually annotated segments. Rendering of original data in Fig. 2a (hippocampus) with a subset of neurite segments from the FFN segmentation, with comprehensive proofreading of neuronal structures. Excitatory synaptic inputs and outputs of a cortical neuron based on deep-learning prediction of synapses. 10h, covering 205 µm along the axial direction after fusion of the imaging subvolumes arranged on a 3×3×12 grid. Voxels are downsampled in the xy direction owing to file-size limitations. Original data after CLAHE processing and fusion of the multi-round data are available in browsable format at: https://neuroglancer-demo.appspot.com/#!gs://liconn-public/ng_states/multiround_fusion.json. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tavakoli, M.R., Lyudchik, J., Januszewski, M. et al. Light-microscopy-based connectomic reconstruction of mammalian brain tissue. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08957-5'>Dopamine D1–D2 signalling in hippocampus arbitrates approach and avoidance</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 15:10:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The hippocampus1,2,3,4,5,6, as well as dopamine circuits7,8,9, coordinates decision-making in anxiety-eliciting situations. Yet, little is known about how dopamine modulates hippocampal representations of emotionally salient stimuli to inform appropriate resolution of approach versus avoidance conflicts. Here we studied dopaminoceptive neurons in the male mouse ventral hippocampus (vHipp), molecularly distinguished by their expression of dopamine D1 or D2 receptors. We show that these neurons are transcriptionally distinct and topographically organized across vHipp subfields and cell types. In the ventral subiculum where they are enriched, both D1 and D2 neurons are recruited during anxiogenic exploration, yet with distinct profiles related to investigation and behavioural selection. In turn, they mediate opposite approach–avoidance responses, and are differentially modulated by dopaminergic transmission in that region. Together, these results suggest that vHipp dopamine dynamics gate exploratory behaviours under contextual uncertainty, implicating dopaminoception in the complex computation engaged in the vHipp to govern emotional states. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access Buy this article Prices may be subject to local taxes which are calculated during checkout All snRNA-seq data reported in this study have been deposited in the Gene Expression Omnibus under the accession number GSE227313. Source data for all statistical analyses are provided with this article. All other data, including raw photometry data, are available on request from the corresponding authors. Source data are provided with this paper. Sample code and data for the fibre photometry signal analysis are publicly available on GitHub (https://github.com/arthurgodino/Photometry-analysis). All other custom scripts and code utilized in this study, including for statistical analysis, are available on request from the corresponding authors. A. Précis of the neuropsychology of anxiety: an enquiry into the functions of the septo-hippocampal system. & Moser, E. I. Functional differentiation in the hippocampus. Fanselow, M. S. & Dong, H.-W. Are the dorsal and ventral hippocampus functionally distinct structures? A., Witter, M. P., Lein, E. S. & Moser, E. I. Functional organization of the hippocampal longitudinal axis. Bryant, K. G. & Barker, J. M. Arbitration of approach-avoidance conflict by ventral hippocampus. Linking external stimuli with internal drives: a role for the ventral hippocampus. Zweifel, L. S. et al. Activation of dopamine neurons is critical for aversive conditioning and prevention of generalized anxiety. Lammel, S., Lim, B. K. & Malenka, R. C. Reward and aversion in a heterogeneous midbrain dopamine system. Morel, C. et al. Midbrain projection to the basolateral amygdala encodes anxiety-like but not depression-like behaviors. Robinson, O. J., Pike, A. C., Cornwell, B. & Grillon, C. The translational neural circuitry of anxiety. Gergues, M. M. et al. Circuit and molecular architecture of a ventral hippocampal network. Wee, R. W. S. & MacAskill, A. F. Biased connectivity of brain-wide inputs to ventral subiculum output neurons. Royer, S., Sirota, A., Patel, J. & Buzsáki, G. Distinct representations and theta dynamics in dorsal and ventral hippocampus. Ciocchi, S., Passecker, J., Malagon-Vina, H., Mikus, N. & Klausberger, T. Brain computation. Selective information routing by ventral hippocampal CA1 projection neurons. Jimenez, J. C. et al. Anxiety cells in a hippocampal-hypothalamic Circuit. Verney, C. et al. Morphological evidence for a dopaminergic terminal field in the hippocampal formation of young and adult rat. Gasbarri, A., Verney, C., Innocenzi, R., Campana, E. & Pacitti, C. Mesolimbic dopaminergic neurons innervating the hippocampal formation in the rat: a combined retrograde tracing and immunohistochemical study. Park, A. J. et al. Reset of hippocampal-prefrontal circuitry facilitates learning. Witter, M. P. & Amaral, D. G. in The Rat Nervous System 3rd edn (ed. Bentivoglio, M. & Morelli, M. in Handbook of Chemical Neuroanatomy Vol. 21 (eds Dunnett, S. B. et al.) 1–107 (Elsevier, 2005). Dubovyk, V. & Manahan-Vaughan, D. Less means more: the magnitude of synaptic plasticity along the hippocampal dorso-ventral axis is inversely related to the expression levels of plasticity-related neurotransmitter receptors. Dubovyk, V. & Manahan-Vaughan, D. Gradient of expression of dopamine D2 receptors along the dorso-ventral axis of the hippocampus. Wei, X. et al. Dopamine D1 or D2 receptor-expressing neurons in the central nervous system. Salery, M., Trifilieff, P., Caboche, J. & Vanhoutte, P. From signaling molecules to circuits and behaviors: cell-type-specific adaptations to psychostimulant exposure in the striatum. Gerfen, C. R. Segregation of D1 and D2 dopamine receptors in the striatal direct and indirect pathways: an historical perspective. Berke, J. D. What does dopamine mean? Bromberg-Martin, E. S., Matsumoto, M. & Hikosaka, O. Dopamine in motivational control: rewarding, aversive, and alerting. Berry, A. S. et al. Dopaminergic mechanisms underlying normal variation in trait anxiety. Biology and bias in cell type-specific RNAseq of nucleus accumbens medium spiny neurons. Gangarossa, G. et al. Characterization of dopamine D1 and D2 receptor-expressing neurons in the mouse hippocampus. Puighermanal, E. et al. drd2-cre:ribotag Mouse line unravels the possible diversity of dopamine d2 receptor-expressing cells of the dorsal mouse hippocampus. Puighermanal, E. et al. Anatomical and molecular characterization of dopamine D1 receptor-expressing neurons of the mouse CA1 dorsal hippocampus. Cembrowski, M. S., Wang, L., Sugino, K., Shields, B. C. & Spruston, N. Hipposeq: a comprehensive RNA-seq database of gene expression in hippocampal principal neurons. Cembrowski, M. S. et al. Spatial gene-expression gradients underlie prominent heterogeneity of CA1 pyramidal neurons. The subiculum is a patchwork of discrete subregions. Classes and continua of hippocampal CA1 inhibitory neurons revealed by single-cell transcriptomics. Yao, Z. et al. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. & von Zastrow, M. A molecular landscape of mouse hippocampal neuromodulation. Fenno, L. E. et al. Comprehensive dual- and triple-feature intersectional single-vector delivery of diverse functional payloads to cells of behaving mammals. Biane, J. S. et al. Neural dynamics underlying associative learning in the dorsal and ventral hippocampus. Labouesse, M. A., Cola, R. B. & Patriarchi, T. GPCR-based dopamine sensors — a detailed guide to inform sensor choice for in vivo imaging. Patriarchi, T. et al. Ultrafast neuronal imaging of dopamine dynamics with designed genetically encoded sensors. Patriarchi, T. et al. An expanded palette of dopamine sensors for multiplex imaging in vivo. Sun, F. et al. A genetically encoded fluorescent sensor enables rapid and specific detection of dopamine in flies, fish, and mice. & Sabatini, B. L. Dopaminergic modulation of synaptic transmission in cortex and striatum. Bravo-Rivera, C., Roman-Ortiz, C., Brignoni-Perez, E., Sotres-Bayon, F. & Quirk, G. J. Neural structures mediating expression and extinction of platform-mediated avoidance. Diehl, M. M., Bravo-Rivera, C. & Quirk, G. J. The study of active avoidance: a platform for discussion. The hippocampal-VTA loop: controlling the entry of information into long-term memory. Duszkiewicz, A. J., McNamara, C. G., Takeuchi, T. & Genzel, L. Novelty and dopaminergic modulation of memory persistence: a tale of two systems. Forro, T. et al. Anxiety-related activity of ventral hippocampal interneurons. Volitaki, E., Forro, T., Li, K., Nevian, T. & Ciocchi, S. Activity of ventral hippocampal parvalbumin interneurons during anxiety. Li, K., Koukoutselos, K., Sakaguchi, M. & Ciocchi, S. Distinct ventral hippocampal inhibitory microcircuits regulating anxiety and fear behaviors. Direct ventral hippocampal-prefrontal input is required for anxiety-related neural activity and behavior. Glangetas, C. et al. NMDA-receptor-dependent plasticity in the bed nucleus of the stria terminalis triggers long-term anxiolysis. Sánchez-Bellot, C., AlSubaie, R., Mishchanchuk, K., Wee, R. W. S. & MacAskill, A. F. Two opposing hippocampus to prefrontal cortex pathways for the control of approach and avoidance behaviour. Edelmann, E. & Lessmann, V. Dopaminergic innervation and modulation of hippocampal networks. A. Limbic and cortical information processing in the nucleus accumbens. Morales, M. & Margolis, E. B. Ventral tegmental area: cellular heterogeneity, connectivity and behaviour. Takeuchi, T. et al. Locus coeruleus and dopaminergic consolidation of everyday memory. Kempadoo, K. A., Mosharov, E. V., Choi, S. J., Sulzer, D. & Kandel, E. R. Dopamine release from the locus coeruleus to the dorsal hippocampus promotes spatial learning and memory. Godino, A. et al. Transcriptional control of nucleus accumbens neuronal excitability by retinoid X receptor α tunes sensitivity to drug rewards. Mi, H. et al. Protocol update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0). In vivo imaging identifies temporal signature of D1 and D2 medium spiny neurons in cocaine reward. Martianova, E., Aronson, S. & Proulx, C. D. Multi-fiber photometry to record neural activity in freely-moving animals. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Controlling the false discovery rate: a practical and powerful approach to multiple testing. We thank S. Pirpinias, K. Beach, C. McManus, K. Schmidt, N. Pulido, K. Rosen, C. Blaschke and E. Mouzon for transgenics breeding and genotyping; and X. Qiao and E. Aritzia from the Dean's Flow Cytometry CoRE at the Icahn School of Medicine at Mount Sinai for assistance in nuclei sorting. This work was supported by grants from the Boehringer Ingelheim Fonds (PhD Fellowship to A.G.), the US National Institutes of Health (RF1MH128970 and U01MH116442 to P.R., and R01DA014133 and R01MH051399 to E.J.N.) and the Hope for Depression Research Foundation (to E.J.N. Present address: Aix-Marseille Université, INSERM, INMED, Marseille, France These authors contributed equally: Arthur Godino, Marine Salery Nash Family Department of Neuroscience and Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Arthur Godino, Marine Salery, Angelica M. Minier-Toribio, Vishwendra Patel, Veronika Kondev, Eric M. Parise, Freddyson J. Martinez-Rivera, Carole Morel & Eric J. Nestler Department of Psychiatry and Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Vishwendra Patel, John F. Fullard, Panos Roussos, Robert D. Blitzer & Eric J. Nestler Department of Genetics and Genomic Sciences and Icahn Genomics Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA John F. Fullard & Panos Roussos Center for Disease Neurogenomics, Icahn School of Medicine at Mount Sinai, New York, NY, USA John F. Fullard & Panos Roussos Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Carole Morel & Robert D. Blitzer Mental Illness Research, Education and Clinical Centers, James J. Peters VA Medical Center, Bronx, NY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar A.G. developed all methodology, except for the PMA task, which was developed by A.M.M.-T. and F.J.M.-R. A.G. performed all behavioural experiments and photometry recordings, with help from A.M.M.-T. and V.K. for PMA testing and E.M.P. performed nuclei sorting, FISH and confocal imaging. performed the electrophysiological recordings, with supervision by R.D.B. performed snRNA-seq library preparation. contributed resources for snRNA-seq and software for analysis. contributed software for photometry analysis. A.G. performed all formal analysis and visualization. A.G. wrote the manuscript, which was reviewed and edited by M.S., A.M.M.-T., J.F.F., F.J.M.-R., C.M., P.R., R.D.B. Correspondence to Arthur Godino or Eric J. Nestler. The authors declare no competing interests. Nature thanks Anna Beyeler and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Representative images after RNA fluorescence in situ hybridization (FISH) for eGFP, Drd1 and Drd2 transcripts in the DG (top) or vSub (bottom) of a D1-Cre (left) or D2-Cre (right) x fl/fleGFP::L10a male mouse. b, Drd1 and Drd2 combinatorial expression in eGFP-positive cells in the DG (top) or vSub (bottom) of D1-Cre (n = 4 male mice, 2 hemisections/mouse) and D2-Cre (n = 5 male mice, 2 hemisections/mouse) x fl/fleGFP::L10a male mice. Cells circled in yellow are Drd1+/Drd2+. c, Representative Fluorescence-Activated Nuclei Sorting (FANS) gating strategy from a D1-Cre x fl/fleGFP::L10a male sample. d, Representative sorting summary. e, Percent of GFP-positive nuclei for all D1-Cre (n = 4) and D2-Cre (n = 4) x fl/fleGFP::L10a sorted samples. Data represented as mean ± sem. Scale bars 100 µm. Same data as in Fig. 1: n = 23,809 GFP-positive cells from D1-Cre x fl/fleGFP::L10a mice (3 female, 4 male); n = 20,136 GFP-positive cells from D2-Cre x fl/fleGFP::L10a mice (4 female, 4 male), 3-5 hemisections/mouse. a, Density heatmaps of D1 (red) and D2 (green) cells along dorso-ventral (DV) and antero-posterior (AP) coordinates in the subiculum and CA1 pyramidal and stratum oriens layers (left) and in the DG hilus (right). b, Distribution of D1 and D2 GFP-positive cells, as a proportion of the total D1 or D2 GFP-positive cells detected. LMM-ANOVA: region F11,121 = 55.51 p < 0.0001, region x D1/D2 F11,121 = 13.95 p < 0.0001; followed by FDR-adjusted post-hoc tests. c, Segregation of D1 (left) and D2 (right) vHipp cells between GAD67+|CaMKIIα− inhibitory cells and GAD67−|CaMKIIα+ excitatory cells, as a proportion of the total D1 or D2 cells detected in each subregion. LMM-ANOVA: region F11,253 = 4.00 p < 0.0001, inh/exc F1,253 = 1320.13 p < 0.0001, D1/D2 x inh/exc F1,253 = 34.36 p < 0.0001, region x sex x inh/exc F11,253 = 2.34 p = 0.0094, region x D1/D2 x inh/exc F11,253 = 15.03 p < 0.0001; followed by FDR-adjusted post-hoc tests. Data represented as mean ± sem. a, Workflow for snRNAseq of vHipp from male D1-Cre (n = 4) and D2-Cre (n = 4) x fl/fleGFP::L10a. c, Proportion of all nuclei in individual clusters. d, D1-sorted or D2-sorted origin of individual nuclei, quantified in Fig. e, Expression of dopamine receptor Drd1 and Drd2 genes in individual nuclei. Nuclei are considered as co-expressing D1 and D2 receptors (yellow) either if both Drd1 and Drd2 are detected, if Drd1 is detected in a D2-sorted nucleus or if Drd2 is detected in a D1-sorted nucleus. Insert shows the repartition of D1-D2 co-expressing nuclei across clusters. f, Expression of published marker genes for different cell types across clusters. Full lists of cluster marker genes available in Supplementary Table 1. a, Representative spread, fiber placements (red lines) and representative image of AAV-DIO-GCaMP6s expression in the vSub of D1-Cre (left) and D2-Cre (right) male mice. b, Representative spread, fiber placements (red lines) and representative image of AAV-dLight-1.1, AAV-GRABDA-1h expression in the vSub. Full hemisection illustrates spread of a control AAV-GFP virus, while zoomed insets depict faint membrane expression patterns of dopamine sensors. Note the difference in GFP signal intensity with neighboring entorhinal cortex (EC). c, Representative spread and representative image of AAV-DIO-mCherry, -hM3Dq and -hM4Di expression in the vSub of D1-Cre (left) and D2-Cre (right) male mice. Placement was verified in all mice either by confocal imaging or by dissection of fresh tissue under fluorescent illumination. d, Representative spread, fiber placements (red lines) and representative image of AAV-DIO-EYFP and -ChR2 expression in the vSub of D1-Cre (left) and D2-Cre (right) male mice. vSub was primarily targeted (dark shaded areas), but unavoidable viral spread encompassed various degrees of vCA3 and vDG (light shaded areas). Only animals with notable somatic expression in neighboring entorhinal cortex (<10%) were removed from analysis. Scale bars 500 µm. Data are obtained from the same mice as in Fig. a, Distance travelled (left; t-test: t14.63 = −0.73 p = 0.4783) and velocity (right; t-test: t14.70 = −0.71 p = 0.4885). b, Open arm (OA) exploration time (left; LMM-ANOVA: cell type F1,19 = 0.0013 p = 0.9713, compartment F2,38 = 498.02 p < 0.0001, cell type x compartment F2,38 = 8.12 p = 0.0012; followed by FDR-adjusted post-hoc tests), total number of OA entries (middle; LMM-ANOVA: cell type F1,19 = 1.87 p = 0.1876, compartment F2,38 = 64.02 p < 0.0001, cell type x compartment F2,38 = 2.51 p = 0.0950; followed by FDR-adjusted post-hoc tests) and average OA exploration bout length (right; LMM-ANOVA: cell type F1,19 = 0.011 p = 0.9167, compartment F2,38 = 36.85 p < 0.0001, cell type x compartment F2,38 = 0.14 p = 0.8673; followed by FDR-adjusted post-hoc tests). c, Number of head-dips (total: t-test: t18.945 = 0.83 p = 0.4170; per behavioral outcome: LMM-ANOVA: cell type F1,19 = 0.69 p = 0.4180, outcome F2,38 = 44.21 p < 0.0001, cell type x outcome F2,38 = 4.04 p = 0.0256; followed by FDR-adjusted post-hoc tests). d, Correlation of GCaMP6s signal intensity with individual OA (left, D1: n = 163; Pearson's r = 0.0499, p = 0.5273; D2: n = 107; Pearson's r = 0.3242, p = 0.0007) or closed arm (CA, right, D1: n = 197; Pearson's r = −0.2383, p = 0.0007; D2: n = 198; Pearson's r = −0.1984, p = 0.0051) exploration bout lengths. e, Average GCaMP6s signal in D1-Cre (n = 27 head-dips) and D2-Cre (n = 36 head-dips) mice around head-dips events with unclear outcome. Boxplot represents the median time ± inter-quartile range of the times of maximal acceleration (max. after each head-dip. f, Maximum (peak) GCaMP6s signal before (pre) or after (post) each head-dip event. LMM-ANOVA: cell type F1,22.22 = 0.27 p = 0.6069, outcome F2,375.77 = 7.06 p = 0.0010, pre-post F1,385 = 84.64 p < 0.0001, cell type x outcome F2,375.77 = 16.74 p < 0.0001, cell type x pre-post F1,385 = 0.17 p = 0.6775, outcome x pre-post F2,385 = 11.69 p < 0.0001, cell type x outcome x pre-post F2,385 = 34.37 p < 0.0001; followed by FDR-adjusted post-hoc tests. g, GCaMP6s signal change slope after head-dip events. LMM-ANOVA: cell type F1,23.29 = 1.29 p = 0.2679, outcome F2,380.68 = 6.90 p = 0.0011, cell type x outcome F2,380.68 = 28.50 p < 0.0001; followed by FDR-adjusted post-hoc tests. h, Time delay between the time of maximal GCaMP6s signal change (max. slope) and the time of movement initiation (max. LMM-summary: intercept ≠ 0 t67.07 = −3.68 p = 0.0005, and LMM-ANOVA: cell type F1,23.10 = 0.0053 p = 0.9427, outcome F2,383.84 = 0.19 p = 0.8241, cell type x outcome F2,383.84 = 0.46 p = 0.6310; followed by individual LMM-summary for intercept ≠ 0 for each cell type x outcome combination and FDR adjustment of p-values. i, SVM workflow schematic. j, Classification of unclear-outcome head-dips using a SVM trained on manually-annotated avoid/explore D1 and D2 time-series. Both average GCaMP6s signal (top) and heatmaps of individual time-series (bottom) are represented for both D1 and D2 signals, split by SVM outcome prediction. Data represented as mean ± sem, regression lines with their 95% confidence intervals. Calcium imaging of vSub D1 and D2 excitatory neuron activity during EPM testing. b, Representative images at the fiber tip (FT) in a D1-Cre (left) and D2-Cre (right) mouse vSub. c, Quantification in D1-Cre (top, 1 hemisection/mouse) and D2-Cre (bottom, 1 hemisection/mouse) animals indicating GCaMP6f expression largely restricted to excitatory CaMKIIα+|GAD67− cells. d, Representative D1-Cre (top) and D2-Cre (bottom) GCaMP6f signal. e, Heatmaps of average GCaMP6f signal. f, Average GCaMP6f signal in during entries in the open arm (OA) or closed arm (CA). g, Signal intensity (AUC) by EPM compartment. LMM-ANOVA: compartment F2,20 = 58.21 p < 0.0001, cell type F1,10 = 1.12 p = 0.3142, compartment x cell type F2,20 = 1.04 p = 0.3724; followed by FDR-adjusted post-hoc tests. Data represented as mean ± sem. Scale bars 100 µm. b, Representative D1-Cre (top) and D2-Cre (bottom) GCaMP6s signal. c, Heatmaps of average GCaMP6s signal. d, Average GCaMP6s signal during entries in the arena center (D1: n = 50; D2: n = 149 entries). e, Average GCaMP6s signal around exits from immobility in arena corners (D1: n = 231; D2: n = 178 events). Dotted line indicates movement initiation. f, Signal intensity (AUC) by OFT zone. LMM-ANOVA: zone F3,39 = 6.40 p = 0.0012, cell type F1,13 = 0.33 p = 0.5742, zone x cell type F3,39 = 7.74 p = 0.0004; followed by FDR-adjusted post-hoc tests. h, Representative D1-Cre (top) and D2-Cre (bottom) GCaMP6s signal. The first (1) feeding bout is denoted. i, Heatmaps of average GCaMP6s signal. j, Average GCaMP6s signal around the initiation of feeding bouts (D1: n = 68; D2: n = 88 bouts). k, Average GCaMP6s signal around exits from immobility in arena corners (D1: n = 87; D2: n = 32 events). Dotted line indicates movement initiation. l, Signal intensity (AUC) by NSF arena zone and feeding activity in the center zone. LMM-ANOVA: zone F4,44 = 10.36 p < 0.0001, cell type F1,11 = 0.64 p = 0.4405, zone x cell type F4,44 = 1.65 p = 0.1785; followed by FDR-adjusted post-hoc tests. Data represented as mean ± sem. Effects of CNO itself (n = 10 CNO, n = 10 vehicle male mice) on a, OA exploration time (t-test: t14.65 = −0.061 p = 0.9523); b, total number of open arm entries (left; t-test: t17.99 = −0.53 p = 0.6045) and average OA exploration bout length (right; t-test: t17.26 = −0.057 p = 0.9548). Data are obtained from the same mice as in Fig. d, Total number of OA entries (left; t-test: t8.16 = 0.73 p = 0.4857) and average OA exploration bout length (right; t-test: t16.42 = 1.05 p = 0.3095) for D1-Cre mice. e, Total number of OA entries (left; t-test: t20.68 = −2.16 p = 0.0423) and average OA exploration bout length (right; t-test: t14.63 = −2.18 p = 0.0463) for D2-Cre mice. f, Experimental schematic. Data are obtained from the same mice as in Fig. g, Total number of OA entries (left; t-test: t20.94 = 0.64 p = 0.5284) and average OA exploration bout length (right; t-test: t20.60 = −1.89 p = 0.0723) for D1-Cre mice. h, Total number of OA entries (left; t-test: t21.68 = 1.55 p = 0.1354) and average OA exploration bout length (right; t-test: t21.93 = 1.80 p = 0.0858) for D2-Cre mice. j, Total locomotor activity in OFT for D1-Cre (left; t-test: t13.25 = −2.52 p = 0.0253) and D2-Cre (right; t-test: t20.99 = 0.08 p = 0.9374) mice. k, Time spent in the OFT center zone for D1-Cre (left; t-test: t18.84 = 2.6052 p = 0.0175) and D2-Cre (right; t-test: t19.62 = −0.63 p = 0.5379) mice. l, Latency to the first feeding bout during NSF testing for D1-Cre (left; t-test: t18.31 = 0.51 p = 0.6171) and D2-Cre (right; t-test: t21.82 = −2.32 p = 0.0303) mice. Data are obtained from the same mice as in Fig. n, Total number of OA entries (left; LMM-ANOVA: stimulation zone F1,17 = 0.67 p = 0.4232, virus F1,17 = 2.63 p = 0.1234, stimulation zone x virus F1,17 = 2.55 p = 0.1284; followed by FDR-adjusted post-hoc tests) and OA exploration bout length (right; LMM-ANOVA: stimulation zone F1,17 = 8.88 p = 0.0084, virus F1,17 = 0.58 p = 0.4580, stimulation zone x virus F1,17 = 5.8529 p = 0.0271; followed by FDR-adjusted post-hoc tests) for D1-Cre mice. o, Total number of open arm entries (left; LMM-ANOVA: stimulation zone F1,19 = 8.70 p = 0.0082, virus F1,19 = 7.94 p = 0.0110, stimulation zone x virus F1,19 = 0.036 p = 0.8526; followed by FDR-adjusted post-hoc tests) and OA exploration bout length (right; LMM-ANOVA: stimulation zone F1,19 = 2.62 p = 0.1221, virus F1,19 = 6.1645 p = 0.0225, stimulation zone x virus F1,19 = 9.55 p = 0.0060; followed by FDR-adjusted post-hoc tests) for D2-Cre mice. Data represented as mean ± sem. b, Representative RdLight-1 (top) and GRABDA−1h (bottom) signal. c, Heatmaps of average signal. d, Average RdLight−1 signal during entries to (left) and exits from (right) the open arm (OA) or closed arm (CA) (entry: n = 24 OA, n = 99 CA, exit: n = 42 OA, n = 97 CA). e, Average GRABDA-1h signal during entries to (left) and exits from (right) the OA or CA (entry: n = 28 OA, n = 92 CA, exit: n = 44 OA, n = 77 CA). f, Average control signal during entries to (left) and exits from (right) the OA or CA (entry: n = 33 OA, n = 112 CA, exit: n = 45 OA, n = 105 CA). g, Signal intensity by EPM compartment. Data analyzed together with Fig. 5e data, statistics in Fig. Data represented as mean ± sem. Data are obtained from the same mice as in Fig. LM-ANOVA: virus F2,32 = 3.60 p = 0.0389; followed by FDR-adjusted post-hoc tests. LM-ANOVA: virus F2,32 = 7.14 p = 0.0027; followed by FDR-adjusted post-hoc tests. d, Total number of OA entries in EPM. LM-ANOVA: virus F2,33 = 0.12 p = 0.8839. e, Total locomotor activity in OFT. LM-ANOVA: virus F2,33 = 0.67 p = 0.5190. f, Time spent in OFT center zone. LM-ANOVA: virus F2,33 = 1.66 p = 0.2049; followed by FDR-adjusted post-hoc tests. g, Latency to the first feeding bout during NSF testing. LM-ANOVA: virus F2,33 = 3.33 p = 0.0482; followed by FDR-adjusted post-hoc tests. Data represented as mean ± sem. c, Quantification of peri-event signal changes. LMM-ANOVAs (Foot-shock, DA: F7,147 = 18.64 p < 0.0001. First platform exit, DA: F7,119 = 7.64 p < 0.0001. First platform exit, D1: F7,294 = 1.19 p = 0.3109. First platform exit, D2: F7,273 = 9.72 p < 0.0001); followed by FDR-adjusted post-hoc tests (all pairwise comparisons were computed but only the ones significant vs the first 1.5 s time-bin are represented, p < 0.05*, <0.01**, <0.001***). d, Times of peak photometry RdLight−1 and D2-GCaMP6s signals after foot-shock (left, LMM-ANOVA: sensor F1,22.94 = 4.99 p = 0.0355) and first platform exit (right, LMM-ANOVA: sensor F1,19.468 = 1.49 p = 0.2376). e, Correlation of RdLight-1 and D2-GCaMP6s signal (n = 1 dual-injected male mouse) around foot-shocks (left; Pearson's r = −0.18 p < 0.0001), reward consumption (Pearson's r = −0.37 p < 0.0001) and first platform exit after tone end (right; Pearson's r = −0.19, p < 0.0001). Data represented as mean ± sem, regression lines with their 95% confidence intervals. b, Approach behavior (lever presses) during extinction sessions, during tone presentation (left; LMM-ANOVA: session F4,60 = 6.50 p = 0.0002, DREADD F1,15 = 0.47 p = 0.5023, session x DREADD F4,60 = 0.31 p = 0.8692) and inter-trial intervals (ITI, right; LMM-ANOVA: session F4,60 = 1.10 p = 0.3644, DREADD F1,15 = 1.45 p = 0.2466, session x DREADD F4,60 = 1.35 p = 0.2623). c, Avoidance behavior (time on platform) during extinction sessions. LMM-ANOVA: session F4,60 = 21.56, p < 0.0001, DREADD F1,15 = 0.003 p = 0.9568, session x DREADD F4,60 = 0.36 p = 0.8348. d, Freezing upon tone presentation during extinction sessions. LMM-ANOVA: session F4,60 = 2.92 p = 0.0283, DREADD F1,15 = 2.73 p = 0.1191, session x DREADD F4,60 = 0.19 p = 0.9424. e, Approach behavior (lever presses) during conditioning sessions for D1-Cre mice during tone presentation (left; LMM-ANOVA: session F9,135 = 3.05 p = 0.0024, DREADD F1,15 = 0.04 p = 0.8533, session x DREADD F9,135 = 0.27 p = 0.9808) or during inter-tone intervals (ITI; right; LMM-ANOVA: session F9,135 = 5.61 p < 0.0001, DREADD F1,15 = 0.003 p = 0.9608, session x DREADD F9,135 = 1.08 p = 0.3846). f, Avoidance (time on platform) behavior during conditioning sessions for D1-Cre mice. LMM-ANOVA: session F9,133.1 = 15.04 p < 0.0001, DREADD F1,15.006 = 2.79 p = 0.1156, session x DREADD F9,133.1 = 1.67 p = 0.1027. g, Experimental schematic (n = 11 D2-mCherry, n = 9 D2-hM4Di male mice). h, Approach behavior (lever presses) during extinction sessions, during tone presentation (left; LMM-ANOVA: session F4,72 = 64.75 p < 0.0001, DREADD F1,18 = 5.27 p = 0.0339, session x DREADD F4,72 = 0.83 p = 0.5099) and inter-trial intervals (ITI, right; LMM-ANOVA: session F4,72 = 22.15 p < 0.0001, DREADD F1,18 = 0.006 p = 0.9379, session x DREADD F4,72 = 2.03 p = 0.0994). i, Avoidance behavior (time on platform) during extinction sessions. LMM-ANOVA: session F4,72 = 26.69, p < 0.0001, DREADD F1,18 = 6.13 p = 0.0234, session x DREADD F4,72 = 0.57 p = 0.6858. j, Freezing upon tone presentation during extinction sessions. LMM-ANOVA: session F4,72 = 4.25 p = 0.0038, DREADD F1,18 = 0.003 p = 0.9570, session x DREADD F4,72 = 1.45 p = 0.2278. k, Approach behavior (lever presses) during conditioning sessions for D2-Cre mice during tone presentation (left; LMM-ANOVA: session F9,162 = 10.30 p < 0.0001, DREADD F1,18 = 1.03 p = 0.3227, session x DREADD F9,162 = 1.54 p = 0.1373) or during inter-tone intervals (ITI; right; LMM-ANOVA: session F9,162 = 5.50 p < 0.0001, DREADD F1,18 = 4.12 p = 0.0575, session x DREADD F9,162 = 1.11 p = 0.3589). l, Avoidance behavior (time on platform) during conditioning sessions for D2-Cre mice (LMM-ANOVA: session F9,162 = 26.31 p < 0.0001, DREADD F1,18 = 0.44 p = 0.5170, session x DREADD F9,162 = 1.91 p = 0.05385). Data represented as mean ± sem. List of marker genes for snRNAseq cell type clusters List of D1 vs D2 DEGs for each snRNAseq cell type cluster List of D1 vs D2 DEGs grouped by gene expression pattern List of gene ontology terms associated with DEG lists in each gene expression pattern Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Godino, A., Salery, M., Minier-Toribio, A.M. et al. Dopamine D1–D2 signalling in hippocampus arbitrates approach and avoidance. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08907-1'>Global emergence of unprecedented lifetime exposure to climate extremes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 15:03:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 641, pages 374–379 (2025)Cite this article Yet, how this translates into unprecedented cumulative extreme event exposure in a person's lifetime remains unclear. Here we use climate models, impact models and demographic data to project the number of people experiencing cumulative lifetime exposure to climate extremes above the 99.99th percentile of exposure expected in a pre-industrial climate. We project that the birth cohort fraction facing this unprecedented lifetime exposure to heatwaves, crop failures, river floods, droughts, wildfires and tropical cyclones will at least double from 1960 to 2020 under current mitigation policies aligned with a global warming pathway reaching 2.7 °C above pre-industrial temperatures by 2100. Under a 1.5 °C pathway, 52% of people born in 2020 will experience unprecedented lifetime exposure to heatwaves. The chance of facing unprecedented lifetime exposure to heatwaves is substantially larger among population groups characterized by high socioeconomic vulnerabilities. Our results call for deep and sustained greenhouse gas emissions reductions to lower the burden of climate change on current young generations. Climate extremes have detrimental effects on society and are a foremost concern around climate change1. Anthropogenic influences have been identified in heatwaves, river floods, droughts, crop failures and certain aspects of wildfires and tropical cyclones2,3. With continued atmospheric warming, the intensity, frequency and duration of some of these events are projected to increase further4,5,6,7,8,9, with varying levels and spread depending on the event considered3. Current policies could warm global mean temperature (GMT) to +2.7 °C (+2.2–3.4 °C) above pre-industrial levels by the end of the century10. As this warming is expected to increase human exposure to climate extremes3, young generations will reap the consequences of the present-day mitigation of greenhouse gas emissions. The above climate extremes are projected to occur most frequently across the lifetimes of current young generations11. As such, the number of climate extremes experienced across a person's lifetime can far exceed the expected exposure under a pre-industrial climate. Yet, the number of people who will experience this unprecedented lifetime exposure (ULE) to climate extremes remains unclear. Here we cross an extensive portfolio of multi-model projections of climate extremes with demographic data, GMT trajectories and two measures of vulnerability. We evaluate the emergence of ULE to extreme events at the grid scale to estimate the global membership of birth cohorts that will face ULE (Methods). Then, we show how this sub-population is stratified in terms of vulnerability. This is one of the first estimates of the number of people projected to experience ULE across a multidimensional framework, including birth year, warming scenario and vulnerability. We illustrate what ULE means for extreme heatwaves in one grid cell (0.5° × 0.5°) located over Brussels, Belgium, for three GMT pathways in which warming above pre-industrial temperatures reaches 1.5 °C, 2.5 °C and 3.5 °C by the year 2100. People born in 1960 and spending their life in Brussels are projected to experience three heatwaves in their lifetime, showing little sensitivity to the GMT pathway (Fig. In this location, the 1960 birth cohort does not exceed the threshold of ULE, which we define as the 99.99th percentile of a large sample of lifetime exposures in a pre-industrial control climate and which is six heatwaves here (Fig. 1b, grey histogram and dashed line). By contrast, the 1990 birth cohort emerges into ULE for the two warmest GMT pathways shown (Fig. This implies that, under temperature pathways reaching 2.5 °C or higher warming by 2100, this cohort will face more heatwaves than they would have been expected to experience with a one in ten thousand chance in the absence of climate change. In the 1.5 °C pathway, the 2020 birth cohort is projected to experience nearly 11 heatwaves, yet this increases to 18 and 26 heatwaves in pathways reaching 2.5 °C and 3.5 °C, respectively, by the end of the century. This by far exceeds the ULE threshold under each GMT pathway, with an age of emergence already around 40 years old for the 2.5 °C and 3.5 °C pathways (Fig. We then count the number of people per birth cohort that eventually reach ULE, using absolute population estimates at the grid scale and relative cohort sizes at the country level. In this location, a best estimate of 21,000 people from the 1990 birth cohort and 24,000 people from the 2020 birth cohort are projected to experience ULE (except for the 1990 birth cohort under the 1.5 °C pathway). Under a 1.5 °C pathway, all cohorts born in Brussels after 1990 reach ULE, totalling 665,000 people. For a 3.5 °C pathway, ULE begins for people born in 1978, increasing this total to 941,000 people. For cohorts that emerge, it is virtually certain (at least  >99.99% chance) that their lifetime heatwave exposure cannot be explained by internal climate variability. a,c,e, Multi-model mean time series of cumulative heatwave exposure for people born in 1960 (a), 1990 (c) and 2020 (e) in 1.5 °C (blue line), 2.5 °C (gold line) and 3.5 °C (red line) pathways. b,d,f, Histograms for 1960 (b), 1990 (d) and 2020 (f) birth cohorts show the pre-industrial sample density of 40,000 bootstrapped lifetime exposures overlaid with final lifetime exposures from the time series of the birth cohort. Dashed lines show the 99.99th percentile of the pre-industrial sample distribution, that is, the threshold of unprecedented lifetime exposure (ULE) for this location, cohort and climate extreme. We now repeat this analysis for every land grid cell and project the population fraction of each birth cohort experiencing ULE to heatwaves across the globe (CFheatwaves for cohort fraction reaching ULE to heatwaves). Of the 81 million people born in 1960, on average, around 16% (13 million people) face ULE to heatwaves regardless of the scenario. This fraction rises towards younger generations, and from the 1980 birth cohort onwards, CFheatwaves begins to depend on GMT pathways (Fig. In a 1.5 °C pathway, CFheatwaves stabilizes for recent birth cohorts, reaching an average of 52% for the 2020 birth cohort (62 million people). Comparatively, CFheatwaves of the 2020 birth cohort is almost doubled in a 3.5 °C pathway, reaching 92%. This implies that 111 million children born in 2020 will live an unprecedented life in terms of heatwave exposure in a world that warms to 3.5 °C compared with 62 million in a 1.5 °C pathway. a, Box plots show the cohort fraction reaching ULE to heatwaves (CFheatwaves) for 1.5 °C (blue), 2.5 °C (gold) and 3.5 °C (red) pathways for global birth cohorts between 1960 and 2020 (middle line, median; box limits, upper and lower quartiles; whiskers, extend to the full range of the model ensemble). b, Bars show global cohort sizes in millions, with totals in grey and median numbers of people reaching ULE to heatwaves for 1.5 °C (blue), 2.5 °C (gold) and 3.5 °C (red) pathways. c–e, Maps display country-level CFheatwaves of the 2020 birth cohort for 1.5 °C (c), 2.5 °C (d) and 3.5 °C (e) pathways. At the country level, CFheatwaves for the 2020 birth cohort is the highest in the tropics under low GMT pathways, yet this pattern disappears as heatwaves become widespread under high GMT pathways (Fig. Under a 1.5 °C pathway, equatorial regions have relatively high CFheatwaves; of the 177 countries in this analysis, 104 have most of the population of 2020 birth cohort living with unprecedented exposure to heatwaves (CFheatwaves ≥ 50%; Fig. This latitudinal pattern is less apparent in a 2.5 °C pathway (Fig. In a 3.5 °C pathway, 167 countries have CFheatwaves ≥ 50%, 155 countries have CFheatwaves ≥ 90% and in 113 countries the entire birth cohort faces unprecedented heatwave exposure (CFheatwaves = 100%; Fig. We then expand the analysis to a total of six climate extremes12 and 21 warming pathways (Fig. For every combination of birth cohort, climate extreme and warming pathway, we quantify the number of people experiencing ULE at the grid scale and subsequently aggregate to the country or global level. Cohort fraction (CF) for climate extremes other than heatwaves is lower across all birth years and GMT pathways because they are generally less widespread than heatwaves; however, they still affect a large population fraction (Fig. In a 3.5 °C pathway, 29% of those born in 2020 will live through unprecedented exposure to crop failures (Fig. This is followed by river floods, in which 14% will face unprecedented exposure to this extreme (Fig. As not all climate projections reach high warming levels, the ensemble size shrinks towards higher warming levels. Consequently, crop failures, droughts, river floods and tropical cyclones, which are more dependent on changes in the water cycle than heatwaves, exhibit discontinuities in CF at some GMT intervals (Fig. These sampling artefacts disappear when visualizing CFs for a smaller subset of simulations that are available for all GMT trajectories (Supplementary Note 1 and Supplementary Fig. Although model uncertainties are larger for extremes other than heatwaves, differences in CF across birth cohorts are statistically significant for all six climate extremes (Supplementary Note 2 and Supplementary Figs. a–d, Cohort fraction (CF) across all birth years (1960–2020) and GMT pathways (1.5–3.5 °C) for heatwaves (CFheatwaves; a), crop failures (CFcrop failures; b), wildfires (CFwildfires; c), droughts (CFdroughts; d), river floods (CFfloods; e) and tropical cyclones (CFtropical cyclones; f). Each extreme event panel has its colour bar range. Across all projections available for the 2.7 °C pathway aligned with current policies10, ULE to heatwaves occurs in the Americas, Africa, the Middle East and Australia already for the 1960 birth cohort and globally for the 2020 birth cohort (Supplementary Figs. The ULE to crop failures expands around the United States, South America, Sub-Saharan Africa and East Asia between 1960 and 2020 cohorts (Supplementary Figs. The ULE to river floods occurs in northern latitudes for the 1960 cohort, in line with the observations and model projections for precipitation changes13,14,15 and expands southwards into much of the world for the 2020 cohort (Supplementary Fig. The lower CF of some extremes, such as tropical cyclones, is expected given the geographical constraints of these events and their distinct meteorological drivers. Tropical cyclones can, therefore, be re-evaluated by limiting the analysis to regions that can experience them. We consider these regions to be any grid cells exposed at least once to the event across our whole ensemble of exposure projections (Supplementary Fig. CFtropical cyclones nearly doubles when constraining total birth cohort size to exposed regions. For the 2020 birth cohort, this estimate changes from 6% to 11% in a 1.5 °C pathway and from 10% to 19% in a 3.5 °C pathway. Finally, we cross our grid-scale projections for ULE to heatwaves against two grid-scale indicators of socioeconomic vulnerability (Methods): (1) the Global Gridded Relative Deprivation Index v.1 (GRDI; ref. 16), which expresses relative deprivation according to six socioeconomic indicators; and (2) lifetime mean GDP per capita (denoted as GDP; ref. 17). Binning our birth cohort members into the top and bottom 20% of GRDI (Fig. 7a) enables a grid-scale comparison of ULE for population groups with high and low socioeconomic vulnerability. Using GRDI, we find that the most vulnerable subset of each birth cohort projected to experience ULE to heatwaves under current policies is substantially larger than the least vulnerable subset. This implies that socioeconomically vulnerable people have a consistently higher chance of facing unprecedented lifetime heatwave exposure compared with the least vulnerable members of their generation (Fig. For example, of the 2020 birth cohort, 95% or 23 million members of the high deprivation (high socioeconomic vulnerability) group face ULE to heatwaves, whereas this is 78% (19 million) for the low deprivation group. This disparity is similar when using GDP, but with only 1974 and later birth years having significant differences across vulnerability strata (Supplementary Fig. Here, for the 2020 birth year, 92% (22 million) of the low-income group face ULE under current policies, whereas this is 79% (19 million) for the high-income group. Under alternative warming pathways of 1.5 °C and 3.5 °C, although the same direction of disparities remains across vulnerability strata, the lowest vulnerability groups (low deprivation and high GDP) benefit the most from a low warming pathway (Fig. Socioeconomically vulnerable groups have lower adaptive capacity and face more constraints when it comes to implementing effective adaptation measures18,19. Our results highlight that precisely these groups with the highest socioeconomic vulnerability and lowest adaptation potential face the highest chance for unprecedented heatwave exposure (Fig. This underlines the disproportionate risk for deprived communities in light of past and future climate extremes. a, Geographic distribution of the 20% highest (brown markers) and 20% lowest (green markers) scoring 2020 birth cohort members (with roughly equal population) in the GRDI16. Grid cell marker sizes and colours are scaled by their population. b, Fraction of these two groups projected to experience ULE to heatwaves under the current policies pathway of 2.7 °C warming by 2100 for every fifth birth year. Light-coloured bars show total cohort sizes per birth year and vulnerability group, whereas dark colours indicate the affected fraction. Error bars show the standard deviation across projections. Asterisks indicate that a low- or high-vulnerability group from a given birth cohort has significantly more members with ULE to heatwaves than the alternative vulnerability group of the same birth cohort (at the 5% level). c,d, The high deprivation (c) and low deprivation (d) share of the birth cohort that is projected to experience ULE under the 1.5 °C (blue) and 3.5 °C (red) pathways. Our analysis only quantifies local exposure by design; yet in reality, the effects of climate extremes cascade non-locally. For example, in 2023, smoke from an active wildfire season in Canada was transported south along the east coast of the United States, exposing millions of people to hazardous air quality20 and causing an increased cardiopulmonary disease burden21. Climate extremes also affect society through economic impacts, including the rising cost of living due to supply chain disruptions22 and taxation to recover public infrastructure23. For instance, climate change endangers staple crop production in the main breadbasket countries that supply most of our caloric intake globally24, forcing market instabilities that only the wealthiest can cope with25. These missing non-local impacts make our estimates conservative. By contrast, we do not capture how people adapt to extremes and thereby potentially reduce their exposure or vulnerability. For example, exposure to heatwaves can be reduced for population groups that can afford access to air conditioning18. However, maladaptive responses to climate extremes can instead create lock-ins of vulnerability and exposure1. Therefore, our lifetime exposure estimates omit beneficial adaptation outcomes as well as detrimental non-local and maladaptation effects. Finally, opting for a threshold below 99.99% would lower the bar and increase ULE estimates, and vice versa. Yet this effect is limited because the reference distribution is typically composed of small integers. By contrast, using thresholds above 99.99% risks redundancy in our bootstrapped data sample (Methods). Some demographic realities are not accounted for here. Factors such as within-country migration, fertility and mortality respond in reality to the climate extremes considered here11. In the United States, where the population faces exposure to all extremes analysed in this study, city centres attract young people26 and disparities in life expectancy have been found across race–county combinations27 and rural–urban residency28. For instance, life expectancy is longer for those living in cities, yet here we apply country-average life expectancy and cohort size distribution uniformly within each country. Furthermore, we do not account for within-grid-cell heterogeneity, that is, we miss some fine-scale variations in socioeconomic vulnerability and exposure in socioeconomically diverse regions such as cities. Finally, we focus on the socioeconomic dimension of vulnerability, thereby neglecting that vulnerability to climate extremes may also vary with, for instance, age, gender or disability status. As demographic and multidimensional vulnerability information becomes available at ever higher spatial resolution and explicitly accounts for climate impact projections, it will become possible to deepen the analysis of the interaction between climate change and population dynamics. Hydrological variables have high internal climate variability29 and projecting these events requires an additional impact-modelling step relative to heatwaves, which are computed directly from global climate model output (Methods). Furthermore, these events have sensitivities to input data quality and process representation across the modelling chain (Supplementary Note 2). Other uncertainties, such as demographic representation, are not captured in this analysis. Finally, we opt for assessing ULE at the grid scale instead of at the country level. In doing so, we downscale demographic data instead of upscaling climate data, thereby projecting lifetime exposure based on the local climate of individual birth cohort members. This incurs a trade-off for accepting natural variability in locations at which ULE occurs, yet minimizing year-to-year variability in country- and global-scale CF estimates (Supplementary Note 3 and Supplementary Fig. In summary, we find that large fractions of global birth cohorts are projected to live unprecedented exposure to heatwaves, river floods, droughts, crop failures, wildfires and tropical cyclones. As the frequency of these six climate extremes increases with warming, so does the fraction of people who will face ULE to these events. More ambitious policies are needed to achieve the goal of the Paris Agreement of limiting global warming to 1.5 °C by 2100 relative to the 2.7 °C warming expected under current policies, especially as the most vulnerable groups have more members projected to face unprecedented exposure to heatwaves. Children would reap the direct benefits of this increased ambition: a total of 613 million children born between 2003 and 2020 would then avoid ULE to heatwaves. This underlines the urgent need for deep and sustained greenhouse gas emission reductions to safeguard the future of current young generations. The Inter-Sectoral Impact Model Intercomparison Project (ISIMIP) provides a simulation protocol for projecting the impacts of climate change across sectors such as biomes, agriculture, lakes, water, fisheries, marine ecosystems and permafrost (www.isimip.org). In ISIMIP2b, impact models representing these sectors are run using atmospheric boundary conditions from a consistent set of bias-adjusted global climate models (GCMs) from phase 5 of the Coupled Model Intercomparison Project (CMIP5) that were selected based on their availability of daily data and ability to represent a range of climate sensitivities17; the Geophysical Fluid Dynamics Laboratory Earth System Model (GFDL-ESM2M; ref. 30), the earth system configuration of the Hadley Centre Global Environmental Model (HadGEM2-ES; ref. 31), the general circulation model from the Institut Pierre-Simon Laplace Coupled Model (IPSL-CM5A-LR; ref. 32) and the Model for Interdisciplinary Research on Climate (MIROC5; ref. Impact simulations are run for pre-industrial control (286 ppm CO2; 1666–2099), historical (1861–2005) and future (2006–2099) periods. Future simulations are based on Representative Concentration Pathways (RCPs) 2.6, 6.0 and 8.5 of GCM input datasets. Global projections of annual, grid-scale fractions of exposure to each extreme event category are calculated from ISIMIP2b impact simulations and GCM input data. For the full details of these computations, we refer to ref. 12, but we summarize extreme event definitions below. For heatwaves, droughts, crop failures and river floods, we use localized pre-industrial thresholds to determine event occurrences, whereas for tropical cyclones, we use a single absolute threshold, and wildfires are modelled explicitly (Supplementary Table 20). Heatwaves affect an entire grid cell if the Heat Wave Magnitude Index daily (HWMId; refs. 34,35) of that year exceeds a threshold in the pre-industrial control HWMId distribution in that grid cell11. Although we refer to heatwaves throughout the paper, our definition technically refers to a 3-day extreme heat event that is expected on average once per century under pre-industrial climate conditions. These extreme heat events occur, by definition, everywhere across the world, but with different associated absolute temperature values. Previous analysis highlighted that intergenerational inequalities in lifetime heatwave exposure are robust across a range of heatwave definitions11. Crop failures are based on the sum of the area occupied by maize, wheat, soy or rice within a grid cell when their simulated yield falls below a threshold of their pre-industrial reference yield. Droughts, such as heatwaves, affect an entire grid cell if, for 7 months, monthly soil moisture remains below a threshold of pre-industrial soil moisture levels. Floods only correspond to river flooding, and the flooded area is derived from comparing daily discharge simulations from models of the global water sector to pre-industrial discharge. CaMa-Flood, a global river-routing model36, is used to convert these discharge values to flooded areas. Tropical cyclones occur if a grid cell sustains hurricane-force winds (≥64 knots) at least once a year37,38. Exposure to tropical cyclones does not encompass the flood hazards typically associated with tropical cyclones. Wildfires occur when the burnt area is simulated in a grid cell. Burnt area is either taken directly from annual burnt area calculations or as the annual sum of monthly burnt area in cases in which impact models simulate burnt area sub-annually, capped at 100% of a grid cell. We reiterate that all exposure definitions here neglect potential exposure reduction measures and non-local effects. We subsequently quantify human exposure to climate extremes in a way that facilitates comparison and aggregation across extreme event categories. We consider all people in a grid cell exposed to a climate extreme in a particular year if the climate extreme occurs in that year. We thereby assume that if such a river flood or wildfire occurs somewhere in a 0.5° × 0.5° grid cell, this is sufficiently close to any person located in that grid cell to be considered affected by this extreme event. Using demographic data (see below), we subsequently convert this annual human exposure to lifetime exposure of birth cohorts by summing annual grid fractions of individual event categories across their lifetimes. Demographic data for population totals, cohort sizes and life expectancy enable our projection of the CF experiencing ULE to these six extremes. 17) and originate from population estimates from v.3.2 of the History Database of the Global Environment (HYDE3.2; refs. 39,40) for the historical period (1860–2000) and population projections from middle-of-the-road Shared Socioeconomic Pathway (SSP2; refs. We note that these datasets at present do not account for the impact of climate on population dynamics, for example, through changes in migration, fertility and mortality, although these feedbacks may substantially alter the demographic data. Cohort sizes from the Wittgenstein Centre for Demography and Global Human Capital43 provide estimates of country-level population totals every 5 years (between 1950 and 2100) for each 5-year age group (0- to 4-year-olds, 5- to 9-year-olds, and so on, until 95- to 99-year-olds and a final age group for those 100 years and older). Life expectancy data come from the United Nations World Population Prospects (UNWPP; ref. 44) and describe the life expectancy of 5-year-olds at the country level for 5-year blocks (1950–1955 to 2015–2020). In this dataset, life expectancy is reported for 5-year-olds to exclude biases from infant mortality. Countries that can be spatially resolved at the ISIMIP grid scale and have cohort and life expectancy estimates in these datasets meet the requirements of this study and total 177. We refer to the supplementary material of ref. 11 for a broader discussion of these datasets but explain our application of them in this analysis below. Life expectancies for each country are first linearly interpolated to annual values by assuming that the values of the original 5-year groups are representative of the middle of that group. Furthermore, we add 5 years to annual life expectancies to capture the life expectancy of each cohort since birth, as the original data begin at age 5. As the maximum UNWPP life expectancy for people born in 2020 prescribes the final year in this analysis (2113), annual population totals must be extrapolated to reach this year. For population totals, we take each year beyond 2100 as the mean of the preceding 10 years of the dataset, such that population numbers for 2101 are the mean of 2091–2100. For cohort sizes in each country, we interpolate annual cohort sizes and age groups from the original 5-year age groups and divide age totals by 5 to maintain original population sizes in this dataset and linearly extrapolate these estimates to 2113. This provides the absolute numbers of 0- to 100-year-olds for each year across 1960–2113. To downscale this demographic information to the grid scale, we assume spatially homogeneous cohort representation and life expectancy. Birth cohort size is represented as the number of people of age 0 of a given birth year in a given grid cell. Spatial variability in age structure and life expectancy within a country is therefore ignored in this study. To project CF across different warming pathways by 2100, we construct a series of incrementally warming GMT pathways between 1960 and 2113 based on GMT trajectories taken from the AR6 Scenario Explorer45. The time series from the AR6 scenario explorer were chosen as anchor points for interpolation to produce a range of plausible GMT time series. Furthermore, they were selected to minimize overshooting in the early years of low GMT pathways over higher GMT pathways, which can skew lifetime exposure estimates for early birth cohorts (Supplementary Fig. The upper bound of this subset was limited to 3.5 °C in favour of sampling more simulations for higher GMT projections, which we discuss further below. For the lower bound, 1.5 °C was chosen because it is a more realistic minimal warming scenario than 1.0 °C. Note that the 1.5 °C anchor scenario maximally reaches 1.57 °C before reducing to 1.5 °C by 2100. It is, therefore, referred to as 1.5 °C throughout this analysis. This yields a total of 21 GMT pathways for which we project CF. Our dataset of extreme event exposures represents occurrences of these extremes forced by GCM-modelled climates. These climates have unique GMT warming pathways that depend on their radiative forcing scenario (historical or RCP), as prescribed by the ISIMIP2b modelling protocol. To project these exposure maps along even intervals of warming scenarios, which the original simulations do not provide, we use the 21 GMT pathways described above. The GMT warming levels behind the exposure projections are first smoothed with a 21-year rolling mean before GMT mapping is undertaken. In cases in which our constructed GMT pathways exceed the GMT warming levels of GCM simulations by too much, this mapping erroneously resamples the year of exposures corresponding to the maximum warming level of their forcing GCM. To this end, we implement a constraint in this sampling procedure such that GMT-mapped series are only used if the maximum difference across all GMT pairs is no larger than 0.2 °C. This constraint incrementally reduces ensemble sizes of exposure projections for higher GMT pathways (Supplementary Table 19). Estimating lifetime exposure to extreme events requires crossing life expectancy data at the country level with grid-scale exposure projections. For each GMT trajectory (1.5–3.5 °C, 0.1 °C intervals), birth year (1960–2020) and country (177), exposures are summed across lifetimes at the grid scale. This assumes life expectancy to be spatially homogeneous across each country. Exposure during the death years is also included in this sum by multiplying these exposure projections by the fraction of the final year lived. This produces country-wide maps of lifetime exposure at the grid scale for each GMT trajectory and birth year in this analysis. To generate a baseline distribution of lifetime exposure in a world without climate change, large samples of pre-industrial lifetime exposures are bootstrapped assuming 1960 life expectancy in each country. Here, for each exposure projection originating from a simulation under a pre-industrial climate, 10,000 lifetime exposures are estimated by resampling exposure years with replacement. Depending on ISIMIP2b data availability, pre-industrial exposure projections have a length of 239–639 years per simulation from which to resample from11. This process generates 40,000–310,000 country-wide maps of lifetime exposure, depending on the extreme event considered and its underlying data availability, enabling exposure projections in a pre-industrial climate. Using the pre-industrial period as a baseline enables (1) our GMT mapping procedure; (2) bootstrapping a stationary time series to achieve a large reference dataset; and (3) the production of a reference dataset with information that is independent of the projections forming our ULE estimates. We define an emergence threshold for ULE to extreme events as the 99.99th percentile of our grid-scale samples of pre-industrial lifetime exposure. When it comes to the selection of this percentile, we went as extreme as possible given the bootstrapping of the pre-industrial control runs. This choice was based on a sensitivity analysis for different percentile values that showed a levelling off of lifetime exposure for percentiles more extreme than 99.99%. For each extreme event, birth year, GMT pathway and grid cell, we assess if lifetime exposure emerges or passes this threshold of extreme exposure in a pre-industrial climate. If this threshold is passed, we consider the whole birth cohort in this grid cell to have emerged, tallying its size among a global pool of the same birth cohort and GMT trajectory of people projected to live ULE. This means that, in some locations, even if the sum of exposed grid cell fractions across a pre-industrial lifetime does not cover the entire grid cell, we still extract the entire birth cohort size associated with that grid cell. We sum the number of emerged people in each birth cohort globally, although this birth cohort has a different life expectancy in each country. Once the number of people who have emerged globally is tallied, we divide this by the respective total cohort sizes to estimate CF per birth cohort. Note that ULE, therefore, does not refer to unprecedented in terms of the magnitude of assets or people exposed, but rather in terms of the number of events accumulated across an average person's lifespan in comparison with what they would face in a pre-industrial climate. We use two grid-scale indicators of vulnerability to compare with our estimates of ULE to heat waves. The first is an ISIMIP2b GDP input dataset using concatenated historical and SSP2 time series covering 1860–2099 annually17. This dataset was disaggregated from the country to grid level using spatial and socioeconomic interactions among cities, land cover and road network information and SSP-prescribed estimates of rural and urban expansion46. 16), which communicates relative levels of multidimensional deprivation and poverty (0–100, least to most deprived). This deprivation score uses six input components. This can indicate vulnerability, for which high ratios indicate a dependency of supposed consumers and non-producers on the working-age (producing) population47. Second, infant mortality rates (IMR), taken as the deaths in children younger than 1 year of age per 1,000 live births annually, are a signal of population health and form a long-term Sustainable Development Goal of the United Nations48. Third, the Subnational Human Development Index (SHDI), an assessment of human well-being across education, health and standard of living, originates from the Human Development Index, the latter of which is considered one of the most popular indices to assess country-level well-being. The SHDI improves on the HDI in terms of spatial scale and in representing 161 countries across all world regions and development levels49. Fourth, as rural populations are generally prone to multidimensional poverty50, low values in the ratio of built-up to non-built-up area (BUILT) signal high deprivation. The fifth and sixth components use the mean (of 2020; VNL 2020) and slope (2012–2020; VNL Slope) of nighttime light intensity, a proxy for human activity, economic output and infrastructure development51, to indicate deprivation for areas of low nighttime light intensity. These input components range from 30 arc seconds (roughly 1 km) resolution to subnational regions and are harmonized in an ArcGIS Fishnet feature class for aggregation onto a 0–100 range representing low to high deprivation. The GRDI, therefore, encapsulates multiple dimensions through which generations face deprivation and therewith socioeconomic vulnerability to climate extremes. Although our approach does not explicitly account for actual or potential adaptation to climate change, this multidimensional approach to vulnerability provides relevant information on the current adaptation potential of local populations. We preprocess GDP and GRDI products to enable their comparison with our ULE estimates across birth years. We then use our ISIMIP population totals to compute GDP per capita at the grid scale. Using the GDP per capita metric, we calculate lifetime mean GDP per capita using our life expectancy information for the 1960–2020 birth cohorts. For GRDI, we conservatively regrid the original grid cells of  about 1 km to the 0.5° ISIMIP grid. Although GRDI is a map composed of data spanning 2010–2020, we assume this to be representative of 2020, but nonetheless compare it with the 1960–2020 birth cohort range, similar to the rest of the analysis. We then identify 20% quantile ranges (that is, (0–20], (20–40], … (80–100]) for the lifetime GDP of each birth year and for the singular GRDI map (assumed to align with 2020 population totals). To this end, we rank the vulnerability indicators and apply these ranks to our birth cohort totals on the same grid and for the matching year. Finally, we bin the ranked vulnerability indicators by their associated population totals into five groups of nearly equal population (as it is not possible to achieve perfect bin sizes given the sums of grid-scale population totals). The quantile range of each vulnerability indicator is then a map that can be used to mask the existing locations of ULE, such as birth years and all GMT pathways. 12), we compare the lowest and the highest 20% of each indicator by population. The data for this analysis originate from multiple sources and are hereby listed. Model inputs, raw impact model simulations and post-processed extremes (the latter as Derived Output Data) from ISIMIP2b, as well as GDP data, are accessible at the ISIMIP repository here (https://data.isimip.org). Cohort sizes are taken from the Wittgenstein Centre for Demography and Global Human Capital (https://dataexplorer.wittgensteincentre.org/wcde-v2). Global mean temperatures are extracted from the AR6 scenario explorer45 (https://doi.org/10.5281/zenodo.7197970). GRDI is hosted on the NASA EARTHDATA platform (https://doi.org/10.7927/3xxe-ap97). Maps in this analysis contain base map information made with Natural Earth (naturalearthdata.com). The code for this analysis can be found at GitHub (https://github.com/VUB-HYDR/2025_Grant_etal_Nature/tree/main). Climate Change 2022: Impacts, Adaptation and Vulnerability. Contribution of Working Group II to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (Cambridge Univ. Global burned area increasingly explained by climate change. Seneviratne, S. I. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) (Cambridge Univ. Cook, B. I. et al. Twenty-first century drought projections in the CMIP6 forcing scenarios. Prediction and projection of heatwaves. Gaupp, F., Hall, J., Mitchell, D. & Dadson, S. Increasing risks of multiple breadbasket failure under 1.5 and 2 °C global warming. Yu, Y. et al. Machine learning-based observation-constrained projections reveal elevated global socioeconomic risks from wildfire. Global flood risk under climate change. Tropical cyclones and climate change. Thiery, W. et al. Intergenerational inequities in exposure to climate extremes. Projecting exposure to extreme climate impact events across six event categories and three spatial scales. Wan, H., Zhang, X., Zwiers, F. & Min, S.-K. Attributing northern high-latitude precipitation change over the period 1966-2005 to human influence. Knutson, T. R. & Zeng, F. Model assessment of observed precipitation trends over land regions: detectable human influences and Possible low bias in model trends. Influence of anthropogenic and natural forcings on future changes in precipitation projected by the CMIP6-DAMIP models. Center for International Earth Science Information Network-CIESIN-Columbia University. Global Gridded Relative Deprivation Index (GRDI), Version 1 (NASA Socioeconomic Data and Applications Center, 2022). Frieler, K. et al. Assessing the impacts of 1.5 °C global warming – simulation protocol of the Inter-Sectoral Impact Model Intercomparison Project (ISIMIP2b). Andrijevic, M., Byers, E., Mastrucci, A., Smits, J. & Fuss, S. Future cooling gap in shared socioeconomic pathways. Towards scenario representation of adaptive capacity for global climate change assessments. Drivers and impacts of the record-breaking 2023 wildfire season in Canada. Maldarelli, M. E. et al. Polluted air from Canadian wildfires and cardiopulmonary disease in the Eastern US. Franzke, C. L. E. Impacts of a changing climate on economic damages and insurance. A. U.S. floods: the necessity of mitigation. & Schwalm, C. R. Increasing risks of crop failure and water scarcity in global breadbaskets by 2030. Tigchelaar, M., Battisti, D. S., Naylor, R. L. & Ray, D. K. Future warming increases probability of globally synchronized maize production shocks. & Shubho, M. T. H. Urban revival by millennials? Intraurban net migration patterns of young adults, 1980–2010. Eight Americas: investigating mortality disparities across races, counties, and race-counties in the United States. Singh, G. K. & Siahpush, M. Widening rural–urban disparities in life expectancy, U.S., 1969–2009. Douville, H. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) (Cambridge Univ. Dunne, J. P. et al. GFDL's ESM2 global coupled climate-carbon Earth system models. Part I: physical formulation and baseline simulation characteristics. The HadGEM2-ES implementation of CMIP5 centennial simulations. Climate change projections using the IPSL-CM5 Earth System Model: from CMIP3 to CMIP5. & Fischer, E. M. Top ten European heatwaves since 1950 and their occurrence in the coming decades. & Sterl, A. Humid heat waves at different warming levels. Yamazaki, D., Kanae, S., Kim, H. & Oki, T. A physically based description of floodplain inundation dynamics in a global river routing model. Emanuel, K. Downscaling CMIP5 climate models shows increased tropical cyclone activity over the 21st century. Holland, G. A revised hurricane pressure-wind model. & Janssen, P. Long-term dynamic modeling of global population and built-up area in a spatially explicit way: HYDE 3.1. & Drecht, G. V. The HYDE 3.1 spatially explicit database of human-induced global land-use change over the past 12,000 years. Samir, K. C. & Lutz, W. The human core of the shared socioeconomic pathways: population scenarios by age, sex and level of education for all countries to 2100. Lutz, W., Goujon, A., Samir, K. C., Stonawski, M. & Stilianakis, N. (eds) Demographic and Human Capital Scenarios for the 21st Century: 2018 Assessment for 201 Countries (Publications Office of the European Union, 2018). United Nations Department of Economic and Social Affairs. World Population Prospects 2019 (United Nations, 2019). Byers, E. et al. AR6 Scenarios Database v.1.1. Murakami, D. & Yamagata, Y. Estimation of gridded population and GDP scenarios with spatially explicit statistical downscaling. Dependency ratios: useful policy-making tools? Global, regional, and national trends in under-5 mortality between 1990 and 2019 with scenario-based projections until 2030: a systematic analysis by the UN Inter-agency Group for Child Mortality Estimation. The Subnational Human Development Database. Laborde Debucquet, D. & Martin, W. Implications of the global growth slowdown for rural poverty. & Hodler, R. Nighttime lights as a proxy for human development at the local level. We acknowledge the Potsdam Institute for Climate Impact Research (PIK) for their coordination of ISIMIP and the modellers contributing runs to ISIMIP2b. We thank M. Andrijevic, C.-F. Schleussner and E. Byers for their discussions regarding this paper. acknowledges funding from the European Research Council (ERC) under the Horizon Framework research and innovation programme of the European Union (grant agreement no. We acknowledge further funding from the European Union under the grant agreement no. The computational resources and services used in this work were provided by the VSC (Flemish Supercomputer Center), funded by the Research Foundation – Flanders (FWO) and the Flemish Government, department EWI. This work was supported by the Wellcome Trust 309498/Z/24/Z. Department of Water and Climate, Vrije Universiteit Brussel, Brussels, Belgium Luke Grant, Inne Vanderkelen & Wim Thiery Canadian Centre for Climate Modelling and Analysis, Environment and Climate Change Canada, Victoria, British Columbia, Canada Lukas Gudmundsson, Erich Fischer & Sonia I. Seneviratne You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar L. Grant helped to conceive this study, performed the analysis and wrote the paper. helped with coding parts of this analysis and writing the paper. The authors declare no competing interests. Nature thanks Stephane Hallegatte, Damon Matthews, Raya Muttarak, Dáithí Stone and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Grant, L., Vanderkelen, I., Gudmundsson, L. et al. Global emergence of unprecedented lifetime exposure to climate extremes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64690613/cyprus-statue-sanctuary/'>Archaeologists Were Searching For a Lost Sanctuary—and Re-Discovered a Treasure Trove of Ancient Statues</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A team of German archaeologists recently uncovered an ancient statue sanctuary in Cyprus dedicated to Apollo. Additionally, the team found entirely new types of statutes previously unknown to have existed in Frangissa. “Such larger-than-life figures were previously only known here in Frangissa made from terracotta, including the famous ‘Colossus of Tamassos' in the Cyprus Museum exhibition.” The site also contained marbled glass beads and Egyptian amulets made of faience—materials that show cultural exchanges. One features local Cypro-Syllabic characters, while the other refers to the Ptolemies—the Hellenistic rulers of Egypt who also controlled Cyprus at one time—in Greek. While the larger pieces were captured by German researcher Max Ohnefalsch-Richter in 1885—and have ended up in museums in Canada, the United States, Great Britain, Ireland, Cyprus, and possibly even Russia—numerous fragments of limestone figures and large-format terracottas were left behind. Luckily, today, we see them for the treasures they are. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Spotted the Milky Way on a Sarcophagus Hikers Found an Old Can Filled With Gold Coins Scientists Found an Ancient Figurine With No Face Experts Found an Ancient  Tomb Full of Treasure This Culture's Language Is Global, But Not Its DNA</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/blood-of-man-whos-had-200-snakebites-helps-make-a-potent-antivenom/'>A Snake Collector Volunteered for 200 Snakebites. His Blood Is Now Being Studied in Antivenom</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Blood of Man Who's Had 200 Snakebites Helps Make a Potent Antivenom A new snakebite treatment combines an existing drug with antibodies from a hyperimmune reptile collector, raising both hopes and ethical concerns Scientists have made a potent antivenom using antibodies from a man who has been bitten hundreds of times by venomous snakes. The antivenom combines the existing drug varespladib with antibodies that are copies of those in the blood of Tim Friede, a US snake collector who has given himself more than 600 doses of venom to build up his immunity. He has also been bitten roughly 200 times by venomous snakes. Scientists say that the research could lead to direly needed treatments, but that its reliance on material from a person who performed dangerous experiments on himself makes it ethically murky. The paper's authors say they played no part in Friede's self-exposure to venom. “We did not advise Friede to do this and no one else needs to do this again — we have all the molecules we need,” says co-author Jacob Glanville, chief executive of biomedical firm Centivax in South San Francisco, California. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Each antivenom protects against the venom of at most a few species. “Considering the advanced technologies available in immunology today, it is unacceptable to continue relying on these outdated methods for treating snake bites,” says Kartik Sunagar, a biologist who develops antivenoms at the Indian Institute of Science in Bengaluru. The paper's authors sought to make an antivenom that would protect against a wide range of the world's 600-plus venomous snake species. Both types of peptide bind to the same receptors on nerve cells, impairing communication between neurons and potentially causing muscle paralysis and respiratory failure. Glanville and his co-author Peter Kwong, a biochemist at Columbia University in New York City, had read news coverage about Friede, who took careful notes about his venom exposure. After receiving approval from an ethics review board, getting informed consent from Friede, and supplying him with documents about the dangers of snake venom, the team collected two vials of Friede's blood. Seeking to add protection against even more species, the researchers tested a third element: varespladib, which inhibits a snake-venom enzyme that breaks down muscle and nerve tissue. Antibodies from snake collector Tim Friede, here with a water cobra (Hydrodynastes gigas), have been used to make a broad-spectrum antivenom. But he also says that the study is well executed, and shows that combinations of small-molecule drugs such as varespladib and monoclonal antibodies — copies of human antibodies — hold promise. However, it's not clear whether these antibodies can be produced at an industrial scale for an affordable price, he says. “There is no reason to believe that the new generation of broad-spectrum antibodies will achieve these results.” Glanville says he is thinking about ways to make these therapies more portable and affordable. He also says it's important to provide proof that the cocktail works in the real world before initiating any human trials. Dogs will first be given the experimental treatment; if it doesn't work after a few minutes, they will receive conventional antivenom. This article is reproduced with permission and was first published on May 2, 2025. Katherine Bourzac is a journalist based in San Francisco, who covers environment, climate, chemistry, health and computing for Nature, Science News, and other publications. First published in 1869, Nature is the world's leading multidisciplinary science journal. Nature publishes the finest peer-reviewed research that drives ground-breaking discovery, and is read by thought-leaders and decision-makers around the world.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64679079/human-lifespans-increasing/'>Human Lifespans Keep Increasing—and Scientists Say They're Not Slowing Down Anytime Soon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. From iconic songs by Queen and Alphaville to Isaac Asimov's radical sci-fi vision of the essence of life eternally encoded into computers, immortality has been a human desire that has persisted since the first deathless gods emerged thousands and thousands of years ago. Could we really live forever if there was no physical or biological force to strike us down? Adaptogen beverages brewed from mushrooms have become the newest New-Age immortality tonic. Walt Disney had himself cryogenically frozen (well, this one may be a stretch). Several studies have hypothesized a hard limit to the human lifespan. But recently, researchers Lucio Vincius and Andrea Migliano from the University of Zurich have argued against that—they do not see the lifespan of our species plateauing just yet. Olshansky acknowledges that advances in medicine and public health have steadily increased the human life expectancy over time, but at least in the world's wealthiest (and longest-lived) nations, increases have slowed down drastically in the past 30 years. “Both female and male lifespans continue to linearly increase at a global scale,” Vinicius and Migliano said their study, which is in the process of being peer-reviewed and has been uploaded as a preprint to the server bioRxiv. “This remarkably long trend observed since 1840 remains at odds with our expectation that human lifespans must at some point hit a biologically imposed ceiling.” In fact, they caution that previous claims of such a ceiling are only premature predictions. Now, Olshansky and his colleagues are doubtful that any more than 15% of females and 5% of males will reach their hundredth birthday, and stated “radical human life extension is implausible.” However, these findings are limited to ten countries in which the rise in life expectancy has slowed down, presumably with an end in sight. Referring to the Human Mortality Database—whose longevity data covers 41 countries—what Vinicius and Migliano see is a continuation in the improvement of life expectancy on a global scale. Male lifespans have increased by an average of 2.03 years per decade since 1840, and female lifespans increased by 2.31 years per decade in that same time frame. Between 2000 and 2020, male lifespans have increased by 1.96 years per decade for males and 1.45 years per decade for females, neither of which is that far off from previous decades. Men are catching up to women, and countries with lower life expectancies are catching up to those with higher life expectancies. Whether lifespans in those countries will reach (or even exceed) what was studied by Olshansky remains unknown. If there is a longevity ceiling, nobody has hit it yet. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. A Man Let Deadly Snakes Bite Him For Science</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64689931/somlo-volcano-hill-treasure/'>Archaeologists Found 300 Ancient Treasures on the Hilltop of a Volcano</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Whether people buried their wealth as ritual offerings to the gods or to hide it from greedy leaders, the ancient hoards they left behind are extremely valuable to modern-day scientists. Using metal detectors and LiDAR surveys, one archaeological team recently found over 300 artifacts from at least six different hoards in Hungary, including weapons, amber beads, and tusks of domestic boars. Researchers published their findings in the journal Antiquity. The excavations took place at Somló—a wine region in western Hungary located mostly on the slopes of an extinct volcano—and were part of a larger research project by the National Institute of Archaeology of the Hungarian National Museum Public Collection Centre. One notable find at the site is an elaborate Alpine-style spearhead, but archeologists also discovered fabric and leather remains buried in the area, along with countless other objects associated with bronze working. Modern-day Somló is known for its wine production, and in the 19th century, local farmers began accidentally unearthing artifacts on the hilltop. However, researchers do know that the farmers discovered artifacts like jewelry, weapons, and bronze vessels. Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. Scientists Spotted the Milky Way on a Sarcophagus Hikers Found an Old Can Filled With Gold Coins Scientists Found an Ancient Figurine With No Face Experts Found an Ancient  Tomb Full of Treasure This Culture's Language Is Global, But Not Its DNA</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/chemistry-of-the-papal-conclaves-black-and-white-smoke/'>The Pyrotechnic Chemistry of the Vatican's Pope Smoke Signals Explained</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Pyrotechnic chemistry drives the Vatican's recipes for the black and white smoke used to announce papal election outcomes Although the ritual may feel like something from another era, the papal conclave's colored smoke signals only came into use in 1903, according to NBC News—a relatively recent development for a process that's been going on for nearly a millennium. (The Holy See Press Office did not respond to requests for confirmation that the same recipes remain in use this year.) If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In combustion reactions, a fuel reacts with an oxygen-rich substance called an oxidizer, creating waste products that often include water and carbon dioxide. Smoke is made up of small particles suspended in air, says John Steinberg, a pyrotechnics expert. The smoke's color is dictated by the particle color, which means white and black colored smoke result from waste products other than carbon dioxide, which is colorless. “You don't want complete combustion because if we burn carbon completely, we get carbon dioxide,” he says. These principles underlie both smokes produced by the Vatican conclave. Here the pyrotechnics at play are straightforward because black particles are simple to create—they result from most types of organic fuels. “Black smoke is the easiest” to make, Steinberg says. In this reaction, potassium perchlorate is the oxygen-rich oxidizer, a role it frequently serves in fireworks because it is fairly safe and easily gives up its oxygen; anthracene is the fuel. But anthracene is relatively difficult to burn, Steinberg explains—hence the addition of a little sulfur. Chemically speaking, white smoke is more challenging, Steinberg says. Some sources of white particles, such as zinc chloride or ammonium chloride, can be strongly irritating—which the Vatican seems to have wanted to avoid. Instead the 2013 recipe calls for potassium chlorate, the common sugar lactose and a type of pine resin sometimes known as Greek pitch. The first two ingredients are a common pair, Steinberg says: “Any pyrotechnician knows that if you want smoke, you start with potassium chlorate and sugar.” Potassium chlorate includes one fewer atom of oxygen per molecule than the potassium perchlorate used in black smoke and is an even stronger oxidizer—so strong it's dangerous to use with combustion-prone sulfur. The powerful oxidizer is required in this case, however, because neither sugar nor pine resin are particularly eager to burn, Steinberg says. Pine resin is relatively rich in water and long chains of oxygen-containing hydrocarbons called carboxylic acids. “That's basically what this smoke is,” Steinberg says. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-brood-xiv-zombie-cicadas-are-coming/'>Prepare for ‘Sex, Drugs and Zombies'—Brood XIV Cicadas Are Coming</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Prepare for ‘Sex, Drugs and Zombies'—Brood XIV Cicadas Are Coming As 17-year cicadas emerge this spring, a zombifying fungus is waiting for them Above ground, flightless cicada nymphs transform into black-bodied, winged adults, ready for a month-long bacchanal of song and sex. But for many cicadas—possibly tens of millions—mating will be a gruesome parody of procreation in which their body is turned into a disintegrating puppet by the deadly fungus Massospora cicadina, which only infects 13-year and 17-year cicadas. An infected insect will try to mate even though its genitals have been consumed by the fungus and replaced by a plug of fungal structures called conidiospores, which spread their “zombification” effect on contact. M. cicadina makes male cicadas flick their wings like amorous females do; healthy males become infected when they try to mate with the imposters. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “It's sex, drugs and zombies,” says John Cooley, an evolutionary biologist at the University of Connecticut. “Nature is stranger than any science fiction that's ever been written.” Every brood emergence helps address questions that can't be answered in a lab, such as when the fungus invades nymphs' bodies. When M. cicadina infects adults, it produces durable, thick-walled “resting spores” that drop from its host's crumbling abdomen onto the ground. Resting spores infect other nymphs, which, after metamorphosis, develop their own plug of stalklike conidiospores—the spores that sexually transmit the fungus to other adults. But scientists don't know if resting spores infect nymphs after they hatch or when they surface more than a decade later. In fact, the fungus may have more than these two spore types; they can possibly produce others that kill nymphs underground, Kasson says. Researchers recently found that M. cicadina has the largest genome in the fungus kingdom, meaning that certain aspects of its biology—such as its reproductive cycle—could be quite complex. The only other fungi with a comparable genome size are rust fungi: plant pathogens with up to five life cycle stages. Given that rust fungi and M. cicadina both have unusually large genomes, M. cicadina might share other features with rust fungi, such as multiple spore varieties, Kasson suggests. According to Cooley, periodical cicadas' unusually long nymph stage has led to a lack of specialized predators of these insects, with one exception: M. cicadina. Because M. cicadina prevents its hosts from reproducing, the fungus may also affect cicada populations and brood distribution; that relationship, Cooley adds, is another piece of the periodical cicada puzzle. For Cooley, periodical cicadas offer a window into species distribution and how populations shift over time. “They turn out to be one of the world's best study organisms for asking really big evolutionary questions about species and speciation.” “What I'm going after directly is the question of range change,” Cooley says. “I'm also looking for overlaps between this and other broods.” Within the broods, patterns of waxing and waning zombie infections can reveal how cicada populations change over time. Places with more cicadas will almost certainly have more zombies, says entomologist Chris Alice Kratzer, author and illustrator of the field guide The Cicadas of North America. “I would expect to see a lot more Massospora in Kentucky and Tennessee this year than in some places like Pennsylvania or Massachusetts,” Kratzer says. Based on prior records from the crowdsourcing app iNaturalist for other broods, Kasson predicts that approximately two to four percent of Brood XIV will be zombified. Kratzer, who has previously confirmed sightings of cicadas and M. cicadina for iNaturalist, is also verifying Brood XIV sightings for the platform. If cicada-spotters are patient, they could pinpoint a zombie or two. But even if they don't find any, the sheer number of periodical cicadas is impressive to behold. With predictions of as many as 1.5 million insects per square acre in some places, this year's Brood XIV emergence will be a sight that observers won't soon forget. Mindy Weisberger is a science writer covering biology, paleontology, climate change and space.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02325-x'>High-income groups disproportionately contribute to climate extremes worldwide</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-07 09:53:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Climate injustice persists as those least responsible often bear the greatest impacts, both between and within countries. Here we show how GHG emissions from consumption and investments attributable to the wealthiest population groups have disproportionately influenced present-day climate change. We link emissions inequality over the period 1990–2020 to regional climate extremes using an emulator-based framework. We find that two-thirds (one-fifth) of warming is attributable to the wealthiest 10% (1%), meaning that individual contributions are 6.5 (20) times the average per capita contribution. For extreme events, the top 10% (1%) contributed 7 (26) times the average to increases in monthly 1-in-100-year heat extremes globally and 6 (17) times more to Amazon droughts. Emissions from the wealthiest 10% in the United States and China led to a two- to threefold increase in heat extremes across vulnerable regions. Quantifying the link between wealth disparities and climate impacts can assist in the discourse on climate equity and justice. Over the past two decades, extreme events attributable to climate change resulted in an annual average of US$143 billion in damages1. How these costs could and should be covered—both between and within countries—is a matter of debate2. Central to this debate is the stark disparity between those responsible for emissions and those affected by their impacts. The wealthiest 10% of the global population accounted for nearly half of global emissions in 2019 through private consumption and investments, whereas the poorest 50% accounted for only one-tenth of global emissions3. At the same time, regions with low historic emissions and income levels are typically more frequently and severely exposed to climate impacts4,5 and have limited resources for adaptation6. This cause-and-effect injustice is widely acknowledged7, yet a quantification of how emissions inequality translates into unequal accountability for the resulting global temperature levels and extreme climate events is missing. This translation should account for the individual warming contributions of emissions of non-CO2 GHGs, such as methane (CH4), given their major role in recent warming8. In this study we combine wealth-based carbon inequality assessments3 with an emulator-based climate modelling framework9 to systematically attribute changes in global mean temperature (GMT) levels and grid-cell-level climate extremes to emissions from different wealth groups. We use the Model for the Assessment of the Greenhouse Gas Induced Climate Change (MAGICC)10, a simple climate model, in conjunction with the Modular Earth System Model Emulator for Monthly Temperature and Precipitation (MESMER-M-TP)11, a model that is able to generate large ensembles of spatially explicit monthly temperature and precipitation data that closely resemble those of complex Earth system models at a fraction of the cost. We use attribution science frameworks to link human-induced GHG emissions to changes in the modelled frequency and intensity of extremes12. These frameworks were originally developed to attribute changes in total human emissions13,14. Today, they are increasingly applied for source attribution; that is, to quantify the relative contributions of individual emitter groups, such as companies or countries15,16. When attributing impacts among multiple emitters, various approaches exist and serve distinct purposes17,18. Here we assess the changes in the characteristics of monthly extremes but for the emissions attributable to a specific emitter group15,18. We generate counterfactual emission pathways by subtracting the 1990–2019 emissions of specific emitter groups, namely the wealthiest 10%, 1% and 0.1% globally, as well as in the United States, EU27, India and China (Fig. Emissions data are drawn from ref. These emissions are attributed primarily to consumers, except emissions from capital formation in production sectors, which are attributed to firm owners3. Emissions are reported as a basket of GHGs with warming expressed in CO2 equivalents. We convert these counterfactual pathways into GMT levels and gridded climatic variables (Fig. 1a), allowing us to compare the 2020 climate against the hypothetical 2020 climate state that we would observe if these groups had not emitted. Specifically, we attribute GMT levels and changes in the probability and intensity of extremely hot and dry months (Fig. Prolonged heat poses a significant burden on human health19 and sustained precipitation deficits impair crop yields and threaten water resources20, meaning that both climate extremes are highly relevant in driving climate impacts. We measure meteorological droughts with the standardized precipitation index computed over 3-month periods (SPI-3)20. Counterfactual emissions were converted into GMT using the simple climate model MAGICC and subsequently translated into grid-cell-level realizations of climatic variables using MESMER-M-TP. Historic emissions with and without contributions from selected emitter groups after 1990 (orange). b, Median GMT levels for historic and counterfactual emissions pathways (solid lines) along with 5th–95th confidence intervals (shaded envelopes) derived from 600 ensemble members. c, Reference, present-day and counterfactual distributions at a single grid-cell using temperature as an example. We quantify climate impacts associated with wealthy emitters and compare them to the global average per capita contribution. We do not assess what would constitute fair or just emissions, nor do we assign direct responsibility for the resulting impacts. For illustration, we also provide counterfactual warming outcomes based on rescaling global emissions according to the per capita profile of individual income percentiles. Our modelling framework depicts natural variability and uncertainty in the global response to emission changes (Methods). Unless mentioned otherwise, we provide median results with the 5th–95th confidence intervals in parentheses. All results are statistically significant (established via a one-sample t-tests, Methods) unless explicitly marked otherwise. As our database provides only basket emissions, we derived the main results by assuming that emissions for each GHG scale proportionally with the globally aggregated emissions (Methods). We explore the sensitivity to this assumption in Supplementary Section 2. We found that about 65% (0.40 °C (0.27–0.56 °C)) of this increase is attributable to the global top 10%, 20% (0.12 °C (0.09–0.17 °C)) to the top 1% and 8% (0.05 °C (0.03–0.07 °C)) to the top 0.1% (Fig. These warming contributions are higher (by about one-fifth) than the respective group's contributions to aggregated GHG basket emissions (Supplementary Table 2), underscoring the importance of non-CO2 GHGs21 (see also Supplementary Section 2). a, Median GMT increase over 1990–2020 and the shares attributed to global top 10%, 1% and 0.1%. Hatched areas indicate the warming for each group based on an equal per capita contribution to warming. CIFs indicating the group's contribution to global warming relative to the average contribution are given above the bars. Vertical lines represent the 5th–95th confidence intervals from natural variability and uncertainty in the global temperature response. Circles highlight median values from the sensitivity analysis (Methods; the lower circle is CO2-based emissions and the upper circle is non-CO2-based emissions). Estimates are based on 600 ensemble members each. b, Median hypothetical GMT increase from 1990–2020 if everyone emitted like the given income groups, with the 5th–95th confidence intervals represented as vertical lines. Estimates are based on 600 ensemble members. c, Regional breakdown of the global top 10%,1% and 0.1% in 2019. d, Global (solid line) and regional (symbols) income distributions in 2019. e, Same as a but for the regional top 10%, 1% and 0.1% in the United States, the EU27, India and China. Grey bars highlight the median GMT increase attributable to each region as a whole. Two CIFs are given: the lighter (darker) value is relative to the country's equal share (actual emissions) and measures global (regional) inequality. Vertical lines represent the 5th–95th confidence intervals from natural variability and uncertainty in the global temperature response. Circles highlight median values from the sensitivity analysis. Estimates are based on 600 ensemble members each. To put these numbers into perspective, we defined a group's equal share as the contribution to warming they would have if their per capita impact matched the global average. We then derived climate inequality factors (CIFs) as the group's actual contribution to global warming relative to their equal share. CIFs increase from 6.5 for the top 10% to 20 (77) for the top 1% (0.1%), indicating an amplification of climate inequality with increasing wealth. The full depth of the disproportion in contributions to GMT level becomes tangible when global emissions are rescaled according to the per capita profile of global income groups (Fig. If the entire world population had emitted like the bottom 50%, there would have been minimal additional warming since 1990. However, if the entire world population had emitted like the top 10%, 1% or 0.1%, the GMT increase since 1990 would have been 2.9 °C, 6.7 °C or 12.2 °C. Note that the composition of the global top 10% shifts over time (Supplementary Fig. 1), and while our focus is on wealthy individuals from the world's largest economies, those from smaller (wealthier, as well as less wealthy) countries also contribute disproportionately3. Income levels from regional top emitters deviate from their global counterparts: the top 10% and 1% in the United States and the EU27 (India and China) are wealthier (poorer) than the globally wealthiest 10% and 1% (Fig. Attributed GMT shares by regional emitter groups combine within- and between-region inequality. In the United States (EU27), the top 10% contribute 3.1 (2.8) times more to global warming than the average citizen, but 17 (8) times more than the global average. This relative inequality increases with increasing wealth: the top 1% in the United States (EU27) contribute 53 (21) times their equal shares, and the top 0.1% contribute 190 (64) times their equal shares. In China, where the overall CIF is near 1, the top 10%, 1% and 0.1% emit 4, 13 and 50 times their equal shares, showing even greater regional influence from societal elites. Similarly, in India, where the national CIF is 0.3 (implying that the countries per capita average emissions are below the global average), the top 10%, 1% and 0.1% emit 1.2, 4 and 10 times the global average. We attribute regional increases in the frequency of extremely hot (dry) months, here defined as 1-in-100-year events in a pre-industrial climate, to the emissions of the global top 10% (Fig. 3a), so throughout this Article extremes or extreme events refer to a month being at least as hot (dry) as the hottest (driest) 1% of months in a pre-industrial climate. We state attribution results as additional event counts over a modelled 100-year period of current climate conditions compared with those in 1990 (Methods). For example, 10 additional attributable events indicate that a pre-industrial 1-in-100-year event happens an additional 10 times within 100 years, meaning that its probability has increased tenfold. a,b, Monthly distributions of heat extremes (a) and drought extremes in the Amazon (b) across grid cells attributable to the global top 10%. Distributions were derived by first computing the median attribution results at each grid cell (estimated from 15,000 ensemble members each) and then computing statistics across all 2,652 grid cells. c,d, Spatial distribution of the median number of heat (c) and drought (d) extremes during peak temperature months attributable to the global top 10%. Median estimates are derived from 15,000 ensemble members. Hatched regions indicate insignificant results and/or insufficient model agreement. e,f, Median number of additional heat (e) and drought (f) extremes by region (highlighted on the maps in c and d). CIFs indicating the group's contribution to extremes relative to the average contribution are given above the bars. Vertical lines correspond to 5th–95th confidence intervals. We omit showing global drought estimates because of insufficient agreement among grid-cells. Distributions were derived by first computing median attribution results at each grid cell (estimated from 15,000 ensemble members each) and then computing statistics across all grid cells within each region. For heat extremes, changes are most (least) pronounced in August (February), where 11.5 [8.5–15.4] (3.5 [2.4–4.3]) additional events are attributable to the global top 10%. These changes refer to the global median over land that is predominantly located on the Northern Hemisphere. Areas in strongly affected regions, such as the Amazon region, Southeast Asia or central Africa, face months with up to 30-fold increases in probability (see also Supplementary Figs. To account for differences in seasonality between the Northern Hemisphere and Southern Hemisphere, we assessed heat extremes during the month with peak temperatures (Supplementary Fig. Emissions from 1990–2020 led to a 12.3-fold (9.7–17.9) increase in the probability of heat extremes in the peak temperature month. The top 10% (1%) contribute 7.3 (25.7) times the global average to this (Supplementary Figs. In addition, the intensities of these extremes increased by 0.83 °C (0.75–0.97 °C) since 1990. Of this increase, 0.55 °C (0.50–0.67 °C) (0.17 °C (0.16–0.21 °C)) is attributable to the top 10% (1%), suggesting that their contribution is 6.7 (21.1) times higher than the global average (Supplementary Figs. Attribution results for meteorological drought extremes strongly depend on region and month, with robust drying found mainly in Central America, the Amazon, the Mediterranean, southern Africa and northern Australia (Fig. This is expected, given the considerable intermodel disagreement and large uncertainties in regional precipitation projections22,23,24. We found the strongest attributable drying trend in the Amazon region in October. Overall, the region faces a threefold increase (1.7–4.2) in extreme probability compared with 1990 (Fig. The Amazon region is of global importance, given its uniquely biodiverse system and its major role in the global carbon cycle. Drought events over the past century have already negatively impacted carbon storage in Amazonian rainforests25. Overall, the spatial disparity in attributable changes at the grid-cell-level (Fig. 2) face smaller increases than regions that have contributed very little (for example, western North America and west and central Europe compared with the Amazon region and west southern Africa in Fig. The relatively small (attributable) changes over India and parts of China and the simultaneously high intermodel disagreement are worth noting, given that they are inconsistent with the increasing number of climate-related disasters India is already facing26. We assume that this is related to the climate effects of air pollution, which we discuss in greater detail in Supplementary Section 3. The inequality in warming contributions from affluent groups in high-emitting regions exceeds the inequality in their global counterparts (Fig. This disparity also appears at the grid-cell-level: in the global median, emissions from the top 10% (1%) in the United States are associated with 1.3 (1.0–1.8) (0.3 (0.3–0.5)) additional 1-in-100-year heat events during peak temperature months. This impact represents 23 (60) times the global average contribution and about 3 (2) times the relative contribution of the global top 10% (1%) (Fig. The increase in extreme heat is unevenly distributed across regions. For example, in heat-affected areas such as the Amazon and southeast Africa, emissions from the top 10% in China (the United States) are linked to 2.7 (1.9–3.2) (2.5 (1.7–2.8)) and 2.7 (1.1–3.4) (2.5 (1.0–3.2)) additional occurrences of extreme heat during months with peak temperatures (Fig. In these areas, we can also robustly attribute heat extremes to the regional top 1%. In the Amazon, 0.8 (0.7–1.0) additional heat extremes are attributable to the top 1% in China. This corresponds to an increase in occurrence frequency of 80%. Left: median number of additional heat extremes in selected regions that are attributable to the top 10% of emitters in China, the United States, the EU27 and India. Right: same as on the left but for the top 1% of emitters. Wider bars indicate that more events are attributable to a given emitter group. Median estimates were derived by first computing median attribution results at each grid cell (estimated from 15,000 ensemble members for each) and then computing statistics across grid cells within each region. Attributing changes in extreme events to country-specific wealthy emitter groups becomes increasingly challenging as emitter groups decrease in per capita size, meaning that their cumulative emissions decrease even if their relative emissions contributions increase. Projected temperature distributions are characterized by strong changes in the trend and by consistency across models, allowing robust attribution results even for seemingly small emission amounts, particularly in highly affected regions. The magnitude depends on the definition of extremes, with tail risks seeing the strongest increase (Supplementary Figs. For droughts, the situation is more complex. The SPI-3 signal is dominated by variability and considerable intermodel disagreement, which prevented us from deriving robust attribution results when emissions are small. This study introduces a framework to link wealth-based emissions to shifts in GMT and to changes in regional monthly heat and meteorological drought extremes. We found that the wealthiest 10% contributed 6.5 times more to global warming than the average, with the top 1% and 0.1% contributing 20 and 76 times more, respectively. For heat, this imbalance is more pronounced at the grid-cell-level: the wealthiest 10% and 1% contributed over 7 and 25 times more than the global average to the increase in frequency of pre-industrial 1-in-100-year heat extremes during months with peak summer temperatures. The warming contributions of the wealthy are associated with considerable transboundary effects—for example, the contributions of the wealthiest 10% within the United States and China led to a two- to threefold increase in heat extremes across vulnerable regions such as the Amazon, Southeast Asia and southeast Africa. The robust attribution of drought signals is more complex. Our analysis also underscores the critical role of CH4 emissions in near-term warming (Supplementary Fig. 2) and calls for new research to disentangle income-based emissions at the level of individual gases. Reducing CH4 emissions in line with Paris Agreement-compatible pathways could yield immediate reductions in global temperatures and climate extremes27. We note that our study focuses on monthly extreme heat (that is, extremely hot months), meaning that heatwaves—defined as prolonged periods of abnormally high temperatures lasting from two days to months28—do not directly relate to our metric. However, the probability of extreme daily temperatures is amplified during extremely hot months29, implying that attributable results for short-duration heatwaves might be even more pronounced. Further research is needed to explicitly explore this relationship. Wealth-based emissions comprise private consumption and investment in capital formation across production sectors that supply goods and services consumed by society. Recognizing the associated unequal warming contributions can inform policy interventions. For example, deliberation over a coordinated global wealth tax can draw on this work, illustrating the climate co-benefits of attenuating stark wealth-based disparities in climate impact responsibilities30,31. The transboundary impacts we identify highlight how high-emitting individuals contribute to intensified extremes, even in distant regions. This is particularly relevant for the wealthiest 1% and 0.1%, whose transboundary contributions to worsening local extremes arise primarily through investments, rather than consumption. Efforts to redirect these financial flows should also consider the shared responsibilities of governments to expedite systemic changes in financial and regulatory structures33. Our granular impact analysis shows that low-income regions incur the brunt of the harm caused by emissions concentrated among wealthier populations worldwide. From an adaptation and loss-and-damage perspective, this provides a basis for policy discussions around contributions to compensatory and preventative measures. The amounts of adaptation and loss-and-damage finance provided currently are minuscule compared with the assessed needs34. Our work motivates innovative policy instruments targeted at wealthy individuals to bridge these glaring finance gaps35. Such policies can also improve perceptions of climate justice, a vital factor in fostering social acceptance of climate action36,37. Moving towards evidence-based and targeted policies that reflect polluter-pays principles, including on the domestic level in terms of individual contributions, may therefore be an important cornerstone to enhance policy support for climate action in general. Considerations of our results in policy debates must, however, recognize the conceptual challenges and value judgements in our approach and implementation. First, our attribution relies on consumption-based emissions accounting, allocating emissions between consumers and shareholders through shared ownership (see ref. Our approach contrasts with production-based approaches used to quantify the responsibilities of producers15. Exploring diverse accounting frameworks is key to developing policy mechanisms that address multiple dimensions, but requires care in determining responsibility. Second, we employ the ‘but for' attribution method. This approach directly links emissions to observed climate change while accounting for the timing of emissions and impacts. Third, our approach is limited to changes in climate hazards only, and does not account for on-the-ground vulnerabilities and exposure, which are often key to driving the eventual impact of extreme climate events38. Integrating other causal drivers into extreme event attribution analysis increasingly allows us to address those limitations38,39 in a step towards using extreme event attribution to inform the discourse on loss and damage40,41. At the same time, wealth levels are a key determinant for adaptive capacity and vulnerability in the face of climate change, particularly at the levels of households and individuals42,43. We would therefore expect wealth-related drivers of vulnerability and exposure to further exacerbate the inequalities in responsibilities and experience of the impacts of the climate hazards we studied here. All quantitative estimates are tied to these three assumptions, and we must recognize the choices and value judgements involved in the analysis when evaluating the ethical and legal implications of our findings. Our analysis is further limited by the lack of data on how GHG emission compositions vary with income and wealth. This limits the accuracy of our results, given the role of non-CO2 GHGs in recent warming. In addition, our drought indicator only considers precipitation, which may lead to underestimations in drought risks24. Finally, our analysis is based on modelled data, which may deviate from observations44,45. Accordingly, our analysis does not explicitly assign full responsibility for resulting climate impacts, nor does it determine fair emission levels for any income group. Such determinations require an integrated view of fairness, justice and socio-economic factors46,47, with different reference points for societies at varying levels of development. In conclusion, our findings demonstrate how individual emitter groups have contributed to increases in regional extremes globally. In times of growing economic and climate inequalities, advancing frameworks for attributing emissions to individual emitters can inform global climate action and enhance climate justice. We quantified intensity and frequency changes in extremely hot (dry) months attributable to specific emitter groups. The methodological framework relies on three steps (Fig. 1): first, we constructed counterfactual emissions pathways (that is, emissions pathways with and without the emissions of selected population groups); second, we translated emissions into gridded temperature, precipitation and potential drought data via a chain of computationally efficient emulators; and third, we built on the framework of extreme event attribution to quantify changes in the grid-cell-level distributions of the climatic variables. We relied on the SPI-3 to identify meteorological droughts20. The SPI-3 is computed from precipitation data only, meaning that it does not account for changes in soil- and plant-based water demands. As climate-driven precipitation signals are dominated by natural variability and intermodel disagreement24, climate change-induced trends in our drought indicator are probably a conservative estimate of actual changes20. Therefore, we also computed the SPEI-348. The SPEI-3 takes changes in water demands via potential evapotranspiration (PET) into account. Ideally, PET is estimated from temperatures, radiation, wind speed and humidity via the Penman–Monteith equation20,49. Given that our emulation framework only depicts temperature and precipitation, we relied on the Thornthwaite method to compute PET from temperature data only50. However, PET estimates via the Thornthwaite method are prone to overestimations in terms of magnitude and temporal trends51. This left us with an indicator for meteorological droughts (SPI-3) that probably underestimates drought risks and an additional indicator for potential droughts (SPEI-3 via the Thornthwaite method) that probably provides an overestimation. We used the conservative estimates in the main part of our analysis and show potential drought risks in Supplementary Section 5. We assessed what our climate today would look like if the wealthiest 10%, 1% and 0.1% globally, as well as in the United States, EU27, India and China, had not contributed to global emissions between 1990 and 2019. 52 to construct a time series of historic baseline emissions from 1850–2019 resolved by gas. Next, we removed emitter-specific contributions from these baseline emissions (Fig. To do so, we relied on a dataset of consumption-based CO2e emissions categorized by country and income decile between 1990 and 20193. The estimates relate to all emissions except those from agriculture, forestry and other land use. Our analysis required us to make assumptions about how to disaggregate the reported basket emissions into individual gases. We focused on decomposing emissions into CO2, nitrogen oxide (N2O) and CH4. These three gases make up 98.7% of the total global GHG emissions (excluding agriculture, forestry and other land use)53. The composition of production-side GHG emissions varies strongly by country, ranging from primarily CO2-based emissions (for example, Singapore) to almost equal shares of CO2 and CH4/N2O (for example, Qatar) and, in low-income countries in particular, primarily CH4/N2O (for example, Chad)54. The carbon inequality dataset from ref. 3 employs input–output tables that redistribute production-side emissions to consumers across countries. About one-half of global CH4 emissions are embodied in global trade, with household consumption dominating the final demand category55. Given these considerations, and a lack of alternative data, we chose to apply the same decomposition assumptions across countries and emitter groups. For our central estimate, we assumed that emissions for each GHG scale proportionally with the globally aggregated emissions. We tested the sensitivity to this assumption by providing two extreme cases in which the wealthy emitters (1) solely emit carbon (CO2case) or (2) solely emit CH4 and N2O (non-CO2case). We converted between individual GHGs and CO2e using the Global Warming Potential 100. We transformed counterfactual emissions into grid-cell-level distributions of temperature and precipitation using emulators and subsequently computed drought measures from the emulated data. The emulation consisted of two steps: first, converting emissions into GMT; and second, translating GMT into grid-cell-level monthly mean temperature and precipitation distributions (Fig. The first translation step was carried out with MAGICC10,56. MAGICC is a simple, computationally efficient climate model for global climate indicators. Our temperature outcomes were calculated with MAGICC v7.5 in a probabilistic setting that reflects the assessed uncertainty ranges from the IPCC's Sixth Assessment Report24. We generated 600 GMT trajectories for each scenario. The second translation step was carried out using MESMER-M-TP11. MESMER-M-TP combines parametric approaches and stochastic sampling to approximate the behaviour of individual climate models. For any climate model, the emulator can be calibrated with a small set of actual climate model data and then used to generate gridded temperature and precipitation data that statistically resemble the climate model data. Here we calibrated MESMER-M-TP with 24 different models from the Phase Six of the Coupled Model Intercomparison Project (Supplementary Table 4). Subsequently, we converted each GMT trajectory into a single gridded time series of temperature and precipitation. We computed the SPI-3/SPEI-3 indicator following ref. 48 and used the gamma distribution for normalization. This provided us with a dataset containing 4 variables × 600 realizations × 2,652 grid points × 170 years × 12 months for each scenario. Traditional attribution studies typically aim to understand how climate change altered the statistics of a specified observed extreme. Our study deviates from this approach. We were interested in understanding the extent to which changes in a broad class of historic extremes can be related to emissions from specific emitter groups. We therefore used the framework for event attribution as a guideline14 but modified it according to our research questions. Most importantly, our analysis fully relied on modelled data, meaning that we were not taking observational data into account. Hence, the event attribution framework was reduced to three essential steps: first, we defined extreme events; second, we performed an analysis using emulated (climate model) data; and third, we synthesized the hazards into an attribution statement. We defined extreme events relative to the reference period 1850–1900 and focused on 1-in-100-year (main text) and 1-in-50/1-in-10,000-year (Supplementary Information) events. In a first step, we tested whether changes in the grid-cell-level distribution of a climatic variable under a given counterfactual scenario were significantly different from its present-day distribution. To this end, we computed the differences between the present-day and counterfactual present-day distributions and employed a Student's t-test57 to verify that the distribution was significantly different from zero. If this was the case, we proceeded with the actual attribution. We used the modelled distribution of climatic variables over the reference period to derive grid-cell-specific intensity thresholds for our defined events. To assess frequency changes, we counted how many times the reference intensity threshold was exceeded in a present-day (2020) climate and in a counterfactual 2020 climate, and attributed the difference to a specific emitter group. Similarly, we quantify intensity changes by assessing how hot (dry) a specific extreme would be in a present-day climate as compared to a counterfactual climate, and attribute the difference in values. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The data generated for this study are available via Zenodo at https://doi.org/10.5281/zenodo.14860538 (ref. 58). The results can be reproduced using public data records. The starting point of our analysis was time series of per capita CO2e emissions from ref. 3. We also used historic emissions data available in ref. 52. 10,56) to translate our input data into GMT levels and MESMER-M-TPv0.1.0 (refs. 11,59) to generate a large ensemble of temperature and precipitation data. Our code is publicly available via GitHub at https://github.com/sarasita/attribution.git. The exact version used to produce this study is available via Zenodo at https://doi.org/10.5281/zenodo.15011461 (ref. 60). Note, parts of our code rely on processing data according to ref. The global costs of extreme weather that are attributable to climate change. Warner, K. & Weisberg, M. A funding mosaic for loss and damage. Chancel, L. Global carbon inequality over 1990–2019. Wallemacq, P., Below, R. & McClean, D. Economic Losses, Poverty and Disasters: 1998-2017 (United Nations Office for Disaster Risk Reduction, 2018); https://www.preventionweb.net/files/61119_credeconomiclosses.pdf Climate change through a poverty lens. Dhakal, S. et al. in Climate Change 2022: Mitigation of Climate Change (eds Shukla, P. R. et al.) 215–294 (IPCC, Cambridge Univ. Mar, K. A., Unger, C., Walderdorff, L. & Butler, T. Beyond CO2 equivalence: the impacts of methane on climate, ecosystems, and health. Beusch, L., Gudmundsson, L. & Seneviratne, S. I. Emulating Earth system model temperatures with MESMER: from global mean temperature trajectories to grid-point-level realizations on land. Meinshausen, M., Raper, S. C. & Wigley, T. M. Emulating coupled atmosphere-ocean and carbon cycle models with a simpler model, MAGICC6 – part 1: model description and calibration. Schöngart, S. Introducing the MESMER-M-TPv0.1.0 module: spatially explicit Earth system model emulation for monthly precipitation and temperature. Otto, F. E. Attribution of extreme events to climate change. & Allen, M. R. Human contribution to the European heatwave of 2003. Pathways and pitfalls in extreme event attribution. Beusch, L. et al. Responsibility of major emitters for country-level warming and extreme hot years. Callahan, C. W. & Mankin, J. S. National attribution of historical climate damages. Comparison of formalisms for attributing responsibility for climate change: non-linearities in the brazilian proposal approach. Otto, F. E., Skeie, R. B., Fuglestvedt, J. S., Berntsen, T. & Allen, M. R. Assigning historic responsibility for extreme weather events. De Polt, K. et al. Quantifying impact-relevant heatwave durations. Seneviratne, S. et al. in (eds) Ch. Allen, M. R. et al. Indicate separate contributions of long-lived and short-lived greenhouse gases in emission targets. Cook, B. I. et al. Twenty-first century drought projections in the CMIP6 forcing scenarios. Wu, Y. et al. Hydrological projections under CMIP5 and CMIP6: sources and magnitudes of uncertainty. Chen, D. et al. in Climate Change 2021: The Physical Science Basis (eds Masson-Delmotte, V. et al.) 147–286 (IPCC, Cambridge Univ. Yao, Y., Ciais, P., Viovy, N., Joetzjer, E. & Chave, J. How drought events during the last century have impacted biomass carbon in amazonian rainforests. Kumar, N. et al. Joint behaviour of climate extremes across India: past and future. McKenna, C. M., Maycock, A. C., Forster, P. M., Smith, C. J. Stringent mitigation substantially reduces risk of unprecedented near-term warming rates. IPCC Climate Change 2022: Impacts, Adaptation and Vulnerability (eds Pörtner, H.-O. The physics of heat waves: what causes extremely high summertime temperatures? Chancel, L., Bothe, P. & Voituriez, T. The potential of wealth taxation to address the triple climate inequality crisis. Zucman, G. A Blueprint for a Coordinated Minimum Effective Taxation Standard for Utra-High-Net-Worth Individuals (Tax Observatory, 2024); https://www.taxobservatory.eu/www-site/uploads/2024/06/report-g20-24_06_24.pdf Fairness considerations in global mitigation investments. Bhattacharya, A., Songwe, V., Soubeyran, E. & Stern, N. Raising Ambition and Accelerating Delivery of Climate Finance (Grantham Research Institute on Climate Change and the Environment, 2024); https://www.lse.ac.uk/granthaminstitute/wp-content/uploads/2024/11/Raising-ambition-and-accelerating-delivery-of-climate-finance_Third-IHLEG-report.pdf Dibley, A. et al. Biases in ‘sustainable finance' metrics could hinder lending to those that need it most. & Lissner, T. Research agenda for the loss and damage fund. Climate justice beliefs related to climate action and policy support around the world. Effective climate action must address both social inequality and inequality aversion. Broadening the scope of anthropogenic influence in extreme event attribution. Perkins-Kirkpatrick, S. E. et al. Frontiers in attributing climate extremes and associated impacts. Event attribution is ready to inform loss and damage negotiations. King, A. D., Grose, M. R., Kimutai, J., Pinto, I. Event attribution is not ready for a major role in loss and damage. Towards scenario representation of adaptive capacity for global climate change assessments. Smiley, K. T. et al. Social inequalities in climate change-attributed impacts of hurricane Harvey. Jensen, L., Gerdener, H., Eicker, A., Kusche, J. & Fiedler, S. Observations indicate regionally misleading wetting and drying trends in CMIP6. Kim, Y.-H., Min, S.-K., Zhang, X., Sillmann, J. & Sandstad, M. Evaluation of the CMIP6 multi-model ensemble for climate extreme indices. Zimm, C. et al. Justice considerations in climate research. Kikstra, J. S., Mastrucci, A., Min, J., Riahi, K. & Rao, N. D. Decent living gaps and energy needs around the world. Tirivarombo, S., Osupile, D. & Eliasson, P. Drought monitoring and analysis: standardised precipitation evapotranspiration index (SPEI) and standardised precipitation index (SPI). Vicente-Serrano, S. M., Van der Schrier, G., Beguería, S., Azorin-Molina, C. & Lopez-Moreno, J.-I. Contribution of precipitation and reference evapotranspiration to drought indices under different climates. Monthly potential evapotranspiration estimated using the Thornthwaite method with gridded climate datasets in southeastern brazil. Sheffield, J., Wood, E. F. & Roderick, M. L. Little change in global drought over the past 60 years. Reduced complexity model intercomparison project phase 1: introduction and evaluation of global-mean temperature response. In Climate Change 2022: Mitigation of Climate Change (eds Shukla, P. R. et al.) (Cambridge Univ. The PRIMAP-hist national historical emissions time series. Zhang, B. et al. Consumption-based accounting of global anthropogenic CH4 emissions. Meinshausen, M. et al. Greenhouse-gas emission targets for limiting global warming to 2 °C. Büning, H. & Trenkler, G. Nichtparametrische Statistische Methoden (Walter de Gruyter, 2013). Schoengart, S. Data accompanying publication "High-Income Groups Disproportionately Contribute to Climate Extremes Worldwide.". Schöngart, S. sarasita/attribution: code version accompanying publication "High-Income Groups Disproportionately Contribute to Climate Extremes Worldwide.". S.S. acknowledges support from the German Federal Environmental Foundation (DBU). acknowledge funds from the European Union's Horizon 2020 Research and Innovation Programme under grant number 101003687 (PROVIDE). acknowledges funds from the European Union's Horizon Europe Programme under grant number 101081369 (SPARCCLE). acknowledges support from the European Union's Horizon 2020 Research and Innovation Funding Programme (grant number 101003536, Earth System Models for the Future, ESM2025). R.H. acknowledges funding from the European Union's Horizon Europe Programme under grant agreement number 101094551 (SPES) and grant agreement number 101162653 (2C-RISK) supported by the European Research Council. acknowledges funding from the European Union's Horizon Europe Research and Innovation Programme under grant number 101056873 (ELEVATE). We also gratefully acknowledge funding from IIASA and the National Member Organizations that support the institute. Views and opinions expressed are those of the author(s) only and do not necessarily reflect those of the European Union or the European Research Council Executive Agency. Neither the European Union nor the granting authority can be held responsible for them. Open access funding provided by Swiss Federal Institute of Technology Zurich International Institute for Applied Systems Analysis (IIASA), Laxenburg, Austria Sarah Schöngart, Zebedee Nicholls, Roman Hoffmann, Setu Pelz & Carl-Friedrich Schleussner Institute for Atmospheric and Climate Science, ETH Zürich, Zurich, Switzerland Climate Resource, Melbourne, Victoria, Australia School of Geography, Earth and Atmospheric Sciences, The University of Melbourne, Melbourne, Victoria, Australia You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar S.S. performed the data analysis with contributions from Z.N. S.S. wrote the manuscript with contributions from Z.N., R.H., S.P. have read and agreed to the published version of the manuscript. The authors declare no competing interests. Nature Climate Change thanks Christopher Callahan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Schöngart, S., Nicholls, Z., Hoffmann, R. et al. High-income groups disproportionately contribute to climate extremes worldwide. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            