
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-21</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/try-these-logic-puzzles-from-the-international-logic-olympiad/'>How Logical Are You? Test Your Skills With These Problems from the New International Logic Olympiad</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 17:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Test Your Skills With These Problems from the New International Logic Olympiad In only its second year, the International Logic Olympiad is already booming as logic becomes more and more crucial in our ever changing world Earlier this month, 36 sharp-witted high school students from around the globe stepped foot on the warm and breezy campus of Stanford University. Out of more than 4,000 students from more than 2,000 schools in more than 90 different countries, these 36 people were selected to compete as finalists in the second-ever International Logic Olympiad (ILO). The winners this year were 11th-grade students Luke Song, Zixuan Yin, Kingston Zhang and Max Yang, who, unhappy with their official moniker of “Team I,” informally dubbed themselves “Team Goblin Tribe” after a video skit they watched during a review session. The key to their success, they contend, was lots of practice and teamwork. “I think part of the reason why we were able to do so well in this was because I know my teammates really well and we've been friends for many years,” Song says. Doctors employ logic to diagnose patients, lawyers use logical arguments in the courtroom, and logic is more essential than ever to evaluate the rapidly changing world around us, he notes. “It doesn't all have to do with mathematics and formulas and algebra,” Genesereth adds. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Most of this year's winners have an interest in computer science, alongside environmental science, applied math and electrical engineering, but Zhang says that neither math nor computer science are his “strong suit” and that he plans to go into political science. “A lot of the problems require very expansive thinking and creative solutions, and I think that'll definitely help me if I go into policy in the future,” he explains. A special aspect of the olympiad is its collaborative nature, in which teams work together to solve problems, Yin says. He was particularly proud of how he and his teammates collaborated on a puzzle called Nations (below). After he reasoned out that a solution offered by one of the members of his group must be wrong and came up with an alternate one that proved correct, that teammate “just kept on telling me how fortunate he was to have me on his team,” Yin says. “Having these amazing people to work on logic—it's something that unites our friend group together.” Here are a few curated puzzles from the competition that you can try your hand at with a friend group of your own. Some of the puzzles have been edited to better fit the format of this article. Four students, numbered 1, 2, 3 and 4, vote among themselves to determine who should lead their review session. Each student is required to vote “yes” or “no” for each person in the group, including themselves. The following are true statements about their ballots: Our translations of those statements are at left. Determine, to the extent possible from these statements, who did and did not vote for whom. Note: If we have a statement A that is not true, we consider any sentence of the form “if A, then B” to be true. You will need this fact to solve the puzzle. What is the answer to question 2? A combination safe is opened with a series of four switches that can be flipped on (1) or off (0). The safe is broken, so in order to open it, you only need to get the position of two particular switches correct—but there's no way of knowing which switches are the ones that matter. Find the smallest set of combinations you can try to guarantee that one of them will open the safe. There are two types of nations: strong and weak. If a strong nation invades a weak nation, it will annex the weak nation, but it will become weak, and thus invadable, for some period of time. Each nation wants to be as big as possible but not at the expense of being annexed itself. Assume all nations are completely rational. There are five strong nations and one weak nation. Will the weak nation be invaded? Hint: Try starting with one strong nation and one weak nation first and then build up from there. This puzzle was also included in the ILO. Competitors had to solve it under a time limit using only a pencil and paper. To guarantee you can crack the safe, you'll need to have every possible set of positions for each pair of switches represented. That way, no matter which two switches are the ones that matter or which positions open the safe, one of the codes will crack it. The minimum number of combinations needed is five. Here's one way to think this through. The strong nation will naturally want to invade the weak one because there is no other nation to invade the strong one after it does so. What about two strong nations and one weak nation? Well, if one of the strong nations were to annex the weak nation, it would temporarily become weak and be invaded by the other, so neither of them would want to invade in the first place, knowing they'd be invaded right back. Now consider three strong nations and one weak nation: again, if a strong nation invades, we are left with two strong nations and one weak nation, a situation in which we just determined no one would invade—so all three strong nations would want to invade in this case. Finally, this leads us to five strong nations and one weak nation, a scenario in which all of the strong nations would want to invade because, once there are four strong nations and one that has become weak from annexing, nobody will invade. Emma R. Hasson is a Ph.D. candidate in mathematics at the City University of New York Graduate Center with expertise in math education and communication. Hasson is also a 2025 AAAS Mass Media Fellow at Scientific American. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American is part of Springer Nature, which owns or has commercial relations with thousands of scientific publications (many of them can be found at www.springernature.com/us). Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a65440487/largest-black-hole-merger/'>Scientists Found a Black Hole That Shouldn't Exist. Now Physics Has a Problem.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At 225 solar masses, this gargantuan merger of two black holes challenges our thinking on these famously elusive objects. The Laser Interferometer Gravitational-wave Observatory, or LIGO, made major headlines in 2015 when scientists confirmed the first ever detection of gravitational waves—ripples in spacetime caused by highly energetic deep space phenomena (think: black hole mergers, supernovae, and neutron star collisions). “This is the most massive black hole binary we've observed through gravitational waves, and it presents a real challenge to our understanding of black hole formation,” Mark Hannam, a member of the LVK Collaboration from Cardiff University, said in a press statement. “Black holes this massive are forbidden through standard stellar evolution models.” This means they potentially lie in what's known as the “upper-mass gap”—a range of masses in which black holes aren't thought to form from stars directly (the resulting supernovae of these hugely massive stars should leave behind no stellar remnant at all). However, these black holes' masses aren't the only mystery, as both were spinning between 80 and 90 percent of their top speed limit. This makes them the highest spinning black holes ever recorded by LVK. “That makes the signal difficult to model and interpret. It's an excellent case study for pushing forward the development of our theoretical tools.” Because the detectors are sensitive to black holes of around 100 solar masses, detecting one more than double that size certainly pushes LIGO to its limits. LIGO's decades-long mission to detect gravitational waves has given scientists a whole new understanding of the universe, and nearly a decade after its first detection, it shows no signs of stopping. Scientists Just Saw the Birth of a Solar System Vera Rubin Images Herald a New Era of Astronomy What If Aliens Are Talking, We Just Can't Hear?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250720034027.htm'>AI uncovers 86,000 hidden earthquakes beneath Yellowstone's surface</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 13:28:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Yellowstone, a popular tourist destination and namesake of an equally popular TV show, was the first-ever national park in the United States. In a new study, published July 18 in the high impact journal Science Advances, Western engineering professor Bing Li and his collaborators at Universidad Industrial de Santander (Industrial University of Santander) in Colombia and the United States Geological Survey used machine learning to re-examine historical earthquake data from the Yellowstone caldera over a 15-year period. "While Yellowstone and other volcanoes each have unique features, the hope is that these insights can be applied elsewhere," said Li, an expert in fluid-induced earthquakes and rock mechanics. "By understanding patterns of seismicity, like earthquake swarms, we can improve safety measures, better inform the public about potential risks, and even guide geothermal energy development away from danger in areas with promising heat flow." Prior to the application of machine learning, earthquakes were generally detected through manual inspection by trained experts. This process takes time, is cost-intensive and often detects fewer events than possible now with machine learning. Machine learning has sparked a data-mining gold rush in recent years as seismologists revisit the wealth of historical waveform data stored in datacenters across the world and learn more about current and previously unknown seismic regions around the world. "If we had to do it old school with someone manually clicking through all this data looking for earthquakes, you couldn't do it. The study also shows that earthquake swarms beneath the Yellowstone caldera have occurred along relatively immature, rougher fault structures, compared to more typical mature fault structures seen in regions such as southern California and even immediately outside the caldera. The roughness was measured by characterizing earthquakes as fractals, which are geometric shapes that exhibit self-similarity, meaning they appear similar at different scales. We can only indirectly measure space and time between events," said Li. Note: Content may be edited for style and length. “Remarkable” Results – Scientists Say This Supplement Reversed Heart Damage MIT's Tiny New Device Could Save Diabetics From Deadly Blood Sugar Crashes Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250720034024.htm'>A tiny chemistry hack just made mRNA vaccines safer, stronger, and smarter</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 12:13:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But what if mRNA vaccines could be redesigned to sidestep that response altogether? In a new paper in Nature Biomedical Engineering, researchers at the University of Pennsylvania show that tweaking the structure of the ionizable lipid, a key component of the lipid nanoparticles (LNPs) that deliver mRNA, not only reduces inflammation but also boosts vaccine effectiveness for preventing or treating a range of diseases, from COVID-19 to cancer. Adding phenol groups, chemical compounds with anti-inflammatory properties famously found in foods like olive oil. "By essentially changing the recipe for these lipids, we were able to make them work better with fewer side effects," says Michael J. Mitchell, Associate Professor in Bioengineering (BE) and the paper's senior author. Until now, the ionizable lipids in LNPs -- one of four types of lipids in LNPs, and arguably the most important -- have largely been synthesized using chemical reactions that combine two components into a new molecule, much like two halves of a sandwich coming together. "Because these processes have been so successful, there hasn't been much effort to look for alternatives," says Ninqiang Gong, a former postdoctoral fellow in the Mitchell Lab and co-first author of the paper. Rather than two components, the Mannich reaction combines three precursors, allowing for a greater variety of molecular outcomes. "We were able to create hundreds of new lipids," says Gong. "It's kind of like the secret sauce," says Gong. Previous studies have found that phenol-containing compounds reduce inflammation by negating the harmful effects of free radicals, molecules with unpaired electrons that can disrupt the body's chemistry. Too many free radicals and too few antioxidants result in "oxidative stress," which degrades proteins, damages genetic material and can even kill cells. "The best-performing LNP, which we built using a phenol-containing ionizable lipid produced by the Mannich reaction, actually caused less inflammation," says Emily Han, a doctoral student in BE and co-author of the paper. "Lowering oxidative stress makes it easier for LNPs to do their job," says Dongyoon Kim, a postdoctoral fellow in the Mitchell Lab and co-first author of the paper. C-a16 LNPs not only produced longer-lasting effects, but also improved the efficacy of gene-editing tools like CRISPR and the potency of vaccines for treating cancer. The glow in mice was about 15 times brighter compared to the LNPs used in Onpattro, an FDA-approved treatment for hereditary transthyretin amyloidosis (hATTR), a rare genetic liver disease. In cancer treatments, the results were just as striking. The new lipids also gave cancer-fighting T cells a boost, helping them recognize and destroy tumor cells more efficiently -- and with less oxidative stress. "By causing less disruption to cellular machinery, the new, phenol-containing lipids can enhance a wide range of LNP applications," says Kim. "We tried applying one reaction discovered a century ago, and found it could drastically improve cutting-edge medical treatments," says Mitchell. This study was conducted at the University of Pennsylvania School of Engineering and Applied Science (Penn Engineering) and the Perelman School of Medicine (Penn Medicine), and was supported by a U.S. National Institutes of Health (NIH) Director's New Innovator Award (DP2 TR002776), a Burroughs Wellcome Fund Career Award at the Scientific Interface (CASI), a U.S. National Science Foundation CAREER Award (CBET-2145491), the American Cancer Society (RSG-22-122-01-ET), two US National Science Foundation Graduate Research Fellowships (DGE 1845298, DGE 1845298), a GEM Fellowship, and the NIH/National Cancer Institute Pre-doc to Post-doc Transition Award (F99 CA284294). Note: Content may be edited for style and length. Scientists Warn: Popular Sugar Substitute Linked to Brain Cell Damage Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/landmark-langlands-proof-advances-grand-unified-theory-of-math/'>Mathematicians Just Cracked a Legendary Problem—Here's Why It Matters</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Breakthrough Proof Brings Mathematics Closer to a Grand Unified Theory after More Than 50 Years of Work The Langlands program has inspired and befuddled mathematicians for more than 50 years. A major advance has now opened up new worlds for them to explore Yet the work's true impact might lie not in what it settles, but in the new avenues of inquiry it reveals. But rather than closing a door, this proof throws open a dozen others,” says David Ben-Zvi at the University of Texas at Austin, who was not involved with the work. Ultimately, it took a team of nine mathematicians to crack the problem, in a series of five papers spanning almost 1,000 pages. The group was led by Dennis Gaitsgory at the Max Planck Institute for Mathematics in Bonn, Germany, and Sam Raskin at Yale University in New Haven, Connecticut, who completed his PhD with Gaitsgory in 2014. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The magnitude of their accomplishment was quickly recognized by the mathematical community: in April, Gaitsgory received the US$3-million Breakthrough Prize in Mathematics, and Raskin was awarded a New Horizons prize for promising early-career mathematicians. Like many landmark results in mathematics, the proof promises to forge bridges between different areas, allowing the tools of one domain to tackle intractable problems in another. All told, it's a heady time for researchers in these fields. “It gives us the strongest evidence yet that something we've believed in for decades is true,” says Ben-Zvi. “Now we can finally ask: what does it really mean?” The Langlands programme traces its origins back 60 years, to the work of a young Canadian mathematician named Robert Langlands, who set out his vision in a handwritten letter to the leading mathematician André Weil. Langlands' aim was to connect two very separate major branches of mathematics — number theory (the study of integers) and harmonic analysis (the study of how complicated signals or functions break down into simple waves). A special case of the Langlands programme is the epic proof that Andrew Wiles published, in 1995, of Fermat's last theorem — that no three positive integers a, b and c satisfy the equation an + bn = cn if n is an integer greater than 2. On the cover page to the letter (left), Langlands says, “If you are willing to read it as pure speculation I would appreciate that; if not — I am sure you have a waste basket handy. Institute for Advanced Study (Princeton, N.J.) Shelby White and Leon Levy Archives Center Both concern properties of Riemann surfaces, which are ‘complex manifolds' — structures with coordinates that are complex numbers (with real and imaginary parts). “To truly understand the Langlands correspondence, we have to realize that the ‘two worlds' in it are not that different — rather, they are two facets of one and the same world,” says Frenkel. We are still far from it in the original formulation. In basic terms, the fundamental group of a Riemann surface describes all the distinct ways in which loops can be tied around it. These tools of algebraic geometry are rules that allot ‘vector spaces' (where vectors — arrows — can be added and multiplied) to points on a manifold in much the same way as a function describing a gravitational field, say, can assign numbers for the strength of the field to points in standard 3D space. Work on bridging this divide began back in the 1990s. Using earlier work on Kac–Moody algebras, which ‘translate' between representations and sheaves, Drinfeld and Alexander Beilinson, both now at the University of Chicago, Illinois, described how to build the right kind of sheaves to make the connection. Their paper (see go.nature.com/4ndp5ev), nearly 400 pages long, has never been formally published. Gaitsgory, together with Dima Arinkin at the University of Wisconsin–Madison, made this relationship more precise in 2012; then, working alone, Gaitsgory followed up with a step-by-step outline of how the geometric Langlands might be proved. “The conjecture as such sounds pretty baroque — and not just to outsiders,” says Ben-Zvi. “I think people are much more excited about the proof of geometric Langlands now than they would have been a decade ago, because we understand better why it's the right kind of question to ask, and why it might be useful for things in number theory.” Peter Scholze, at the Max Planck Institute for Mathematics, has been instrumental in forging connections between the local and global Langlands programmes. But initially, even he was daunted by the geometric side. Working together, Scholze and Fargues spent seven years showing that this strategy could help to make progress on proving a version of the local arithmetic Langlands conjecture concerning the p-adic numbers, which involve the primes and their powers. “So I'm really happy about the proof,” Scholze says. According to some researchers, one of the most surprising bridges that the geometric Langlands programme has built is to theoretical physics. This elegant symmetry underpins a broader idea in quantum field theory, known as S-duality. In 2007, Edward Witten at the Institute for Advanced Study (IAS) in Princeton, New Jersey, and Anton Kapustin at the California Institute of Technology in Pasadena were able to show that S-duality in certain four-dimensional gauge theories — a class of theories that includes the standard model of particle physics — possesses the same symmetry that appears in the geometric Langlands correspondence. Although their theories include hypothetical particles, called superpartners, that have never been observed, their insight suggests that geometric Langlands is not just a rarefied idea in pure mathematics; instead, it can be seen as a shadow of a deep symmetry in quantum physics. “I do think it is fascinating that the Langlands programme has this counterpart in quantum field theory,” says Witten. Among the first to take that possibility seriously was Minhyong Kim, director of the International Centre for Mathematical Sciences in Edinburgh, UK. “Even simple-sounding problems in number theory — like Fermat's last theorem — are hard,” he says. One way to make headway is by using ideas from physics, like those in Witten and Kapustin's work, as a sort of metaphor for number-theoretic problems, such as the arithmetic Langlands conjecture. Kim is working on making these metaphors more rigorous. “I take various constructions in quantum field theory and try to cook up precise number-theoretic analogues,” he says. Ben-Zvi, together with Yiannis Sakellaridis at Johns Hopkins University in Baltimore, Maryland, and Akshay Venkatesh at the IAS, is similarly seeking inspiration from theoretical physics, with a sweeping project that seeks to reimagine the whole Langlands programme from the perspective of gauge theory. Witten and Kapustin studied two gauge theories connected by S-duality, meaning that, although they look very different mathematically, the theories are equivalent descriptions of reality. Building on this, Ben-Zvi and his colleagues are investigating how charged materials behave in each theory, translating their dual descriptions into a network of interlinked mathematical conjectures. “Their work really stimulated a lot of research, especially in the number-theory world,” says Raskin. One of their most striking results concerns a two-way relationship between quite different mathematical objects called periods and L-functions. Many mathematicians are confident that the proof of the geometric conjecture will stand, but it will take years to peer review the papers setting it out, which have all been submitted to journals. Gaitsgory, however, is already pushing forward on several fronts. Gaitsgory and his collaborators are now hoping to extend their results to the more intricate, ramified case by accounting for more-complex behaviour around points as well as for singularities or ‘punctures' in the surface. To that end, they are extending their work to the local geometric Langlands conjecture to understand in more detail what happens around a single point — and collaborating with, among others, Jessica Fintzen at the University of Bonn. “This result opens the door to a whole new range of investigations — and that's where our interests start to converge, even though we come from very different worlds,” she says. She constructs the matrices explicitly — essentially, deriving a recipe for writing them down — and this seems to be the kind of local information that must be incorporated into the global geometric case to ramify it, Gaitsgory says. This article is reproduced with permission and was first published on July 16, 2025. First published in 1869, Nature is the world's leading multidisciplinary science journal.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250720034023.htm'>New Duke study finds obesity rises with caloric intake, not couch time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 09:52:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A newly released study from Duke University's Pontzer Lab, housed in the Department of Evolutionary Anthropology in Trinity College of Arts & Sciences, looks at the correlations between economic development, daily energy expenditure and the rise in a country's obesity level. While many experts have offered that rising obesity rates are due to declining physical activity as societies become more industrialized, the findings show that people in wealthier countries expend just as much -- or even more -- energy daily. This large, international, collaborative effort allows us to test these competing ideas. The researchers analyzed thousands of measurements of daily energy expenditure, body fat percentage and body mass index (BMI) from adults aged 18 to 60 across 34 populations spanning six continents. The more than 4,200 adults included in the study came from a wide range of lifestyles and economies, including hunter-gatherer, pastoralist, farming and industrialized populations. To further categorize the level of industrialization, they also integrated data from the United Nations Human Development Index (HDI) to incorporate measures of lifespan, prosperity and education. This suggests that other factors, such as dietary changes, are driving the increases in body fat that we see with increasing economic development," says Amanda McGrosky, a Duke postdoctoral alumna and lead investigator for the study who is now an assistant professor of biology at Elon University. Instead, the data support an emerging consensus that both diet and exercise should be prioritized. Original written by Margo Lakin, Duke University Trinity College of Arts & Sciences. Scientists Warn: Popular Sugar Substitute Linked to Brain Cell Damage Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03827-z'>Polygenic prediction of body mass index and obesity through the life course and across ancestries</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 09:10:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Polygenic scores (PGSs) for body mass index (BMI) may guide early prevention and targeted treatment of obesity. In the ALSPAC study, children with higher PGSs showed accelerated BMI gain from age 2.5 years to adolescence, with earlier adiposity rebound. Adding the PGS to predictors available at birth nearly doubled explained variance for BMI from age 5 onward (for example, from 11% to 21% at age 8). Higher PGSs were associated with greater adult weight gain. In intensive lifestyle intervention trials, individuals with higher PGSs lost modestly more weight in the first year (0.55 kg per s.d.) but were more likely to regain it. Overall, these data show that PGSs have the potential to improve obesity prediction, particularly when implemented early in life. Obesity is a major public health concern that causes or exacerbates many chronic diseases and leads to reduced life expectancy1,2,3. By 2035, more than half of the global population is projected to be living with overweight or obesity4. Although intensive lifestyle interventions (ILIs), bariatric surgery and weight loss medications are effective treatment options5,6, they are not without risk and likely to remain inaccessible to most people. In contrast to many other chronic conditions, obesity often manifests itself during childhood and tends to persist into adulthood7,8,9. Therefore, predictors available in early life, such as genetic variants, which are fixed at conception, could be of particular value. In recent years, PGSs that capture an individual's inherited polygenic susceptibility to a trait or disease have shown great promise in enhancing disease risk prediction and population screening10,11. However, it remains unclear how, when and under what circumstances PGSs for obesity might demonstrate utility for risk prediction. The widely used PGS for obesity by Khera et al.12, based on a genome-wide association study (GWAS) of BMI in over 339,000 people of predominantly European ancestry, explains approximately 8.5% of the variation in BMI in adults. However, as a PGS based on one ancestry population may have weak transferability to other ancestry populations13,14, there is a growing recognition that PGSs that represent a broad range of populations are needed to ensure quality healthcare for all15. By leveraging the results of the largest GWAS meta-analyses for BMI from the Genetic Investigation of ANthropometric Traits (GIANT) consortium and 23andMe, we derived ancestry-specific and multi-ancestry PGSs for BMI and obesity to examine their performance (1) across diverse adult populations, (2) across childhood and adolescence and (3) in the context of ILIs aimed at weight loss (Fig. PGSs were constructed using ancestry-specific GWAS summary statistics, using ancestry-specific (PRS-CS) and ancestry-combined (PRS-CSx) approaches. The best-performing score across multiple ancestries was taken forward to independent validation studies (linear combination version of PRS-CSx with ϕ 1 × 10−2). Population descriptors shown in the figure reflect a combination of self-identified ethnicity and genetic similarity. To develop a PGS for BMI, we used GWAS meta-analysis summary statistics for BMI from over 200 studies from the GIANT consortium and 23andMe (Supplementary Tables 1 and 2), excluding eight studies used for tuning parameters and for testing performance (Methods). We refer to these groups, and to the most closely aligning, genetically inferred population groups from our PGS tuning and testing studies, as being of European-like ancestry (EUR), East Asian-like ancestry (EAS), American-like ancestry (AMR), African-like ancestry (AFR) and South Asian-like ancestry (SAS), respectively, while acknowledging that these groupings oversimplify the actual genetic diversity among participants (see Supplementary Tables 2 and 3 for study-specific population descriptors). Using PRS-CS(x)16,17, we created ancestry-specific and multi-ancestry PGSs leveraging up to 1.3 million common variants. We first identified the optimal genome-wide shrinkage parameter and linear combination weights for PRS-CS(x) that achieved the highest explained variance for BMI in six ancestry subpopulations of the UK Biobank (UKBB)18, including individuals of Middle Eastern-like ancestry (MID). For the EUR-tuning population, we selected a random subset of 20,000 unrelated individuals (Methods and Supplementary Tables 3 and 4). Overall, a multi-ancestry PGS consisting of a linear combination of five ancestry-specific PGSs (PGSLC) was the best-performing score (Methods, Fig. In absolute terms, the explained variance of this multi-ancestry PGS for BMI ranged between 7.2% (AFR) and 17.5% (EUR), with a median of 14.0% (Fig. The multi-ancestry PGS resulted in a higher explained variance than the PGSs trained only with GWAS summary statistics most closely corresponding to the target population (‘ancestry-matched') (Supplementary Table 5). This was particularly evident for the populations of African-like and Central/South Asian-like ancestry (2.6-fold and 2.8-fold increase, respectively), consistent with the smaller GWAS sample sizes available for these populations. The performance of a PGS consisting of near-independent, genome-wide significant variants from the overall multi-ancestry GWAS meta-analysis was generally intermediate to that of the ancestry-matched and multi-ancestry PGSs, with the exception of populations of East Asian-like and European-like ancestry, in whom it performed worse than either (Fig. a, Explained variance for BMI in UKBB tuning populations, defined as adjusted R2 of the rank-based inverse-normal transformed BMI (by sex) predicted by the PGS, incremental to age, genotyping array and ancestry principal components. Error bars represent 95% confidence intervals (CIs) from 1,000 bootstrap resamples. The genome-wide significant score reflects a weighted sum of near-independent SNPs obtained from approximate COJO multi-SNP analyses of a fixed-effect meta-analysis of all contributing GWASs. The multi-ancestral PGSLC reflects the best-performing PRS-CSx score consisting of a linear combination of five ancestry-specific scores, with weights being specific to the validation population (for example, AFR). Population labels follow PAN-UKBB assignment of genetically determined ancestry. Sample sizes (distinct individuals): African 6,154; Admixed American 971; Middle Eastern 1,553; East Asian 2,660; Central/South Asian 8,005; European 20,000. b, Explained variance for BMI within validation populations, comparing the multi-ancestry PGSLC to a previously published score (PGSKhera) based on a smaller BMI GWAS meta-analysis. Same R2 definition and CI estimation as in a. Population descriptors reflect a combination of self-identified ethnicity and genetic similarity. Sample sizes (distinct individuals), from left to right: AFR 12,263, 2,332, 18,701; AMR 10,281, 8,096; AS 4,201; EAS 1,359; SAS 1,177; EUR 13,673, 69,828, 340,224. c, Separation in BMI, body fat percentage (BF%) and waist-to-hip ratio (WHR) across deciles of the PGSLC within the validation subset of the UKBB participants of European-like ancestry (n ~ 340,000). All traits were rank-based inverse-normal transformed by sex. The prevalence of obesity varied substantially across populations and cohorts, with those having obesity class I or higher (that is, BMI ≥ 30 kg m−2 in AFR, AMR and EUR populations and BMI ≥ 27.5 kg m−2 for Asian-like ancestry populations23) ranging from 4.3% for the GPC-UGR to more than 45% for all ancestry subgroups in the MVP and mean BMI ranging from 22.2 kg m−2 to 30.6 kg m2 (Extended Data Fig. A pronounced lower performance was seen for populations with greater proportions of African-like ancestry, with the explained variance being 6.3% and 5.1% in African American populations (from BioMe and MVP, respectively) and 2.2% in the GPC-UGR population from rural southwestern Uganda (Fig. Overall, we observed a median explained variance of 10.3% across our testing populations. Compared to the previously reported PGS by Khera et al. (PGSKhera)12, which was based on a smaller BMI GWAS of up to 339,224 individuals of primarily European ancestry24, we observed a 1.9–2.6-fold increase in explained variance for BMI (Fig. Within the UKBB participants of European-like ancestry, the explained variance was marginally higher in males than in females (males: 17.9%; females: 17.3%) and higher in younger compared to older participants (≤50 years: 18.8%; >50 years: 17.0%) (Supplementary Table 8), both directionally consistent with sex-specific and age-specific genetic differences previously observed for BMI25,26. increase of PGSLC was associated with a 1.98 kg m−2 higher BMI (equivalent to 5.7 kg of body weight for a 1.7-m tall person) (Supplementary Table 9). The corresponding mean separation across deciles of PGSLC was less pronounced for body fat percentage and waist-to-hip ratio compared to BMI (Fig. 2c), revealing that the score does not equally capture differences in body composition. Among European-like ancestry population groups, the PGSLC showed an improved capacity to differentiate between participants with and without obesity (BMI ≥ 30 kg m−2) compared to PGSKhera. increase in PGSLC was associated with a median 1.9–2.6-fold increase in odds of obesity class I or higher (Fig. We observed larger effects for more severe obesity (Extended Data Fig. The area under the receiver operating characteristic curve (AUC) similarly increased with the severity of obesity and neared 0.80 for severe obesity in multiple populations (Fig. The AUC for PGSLC on its own was consistently larger than those for age and sex, PGSKhera and within the UKBB self-reported comparative body size at age 10 (Fig. a, Separation in prevalence of obesity (BMI ≥ 30 kg m−2) across 1% groups of PGSKhera and PGSLC within the validation subset of the UKBB participants of European-like ancestry (n ~ 340,000), with reference lines for the bottom and top 1% groups. b, Odds ratios with 95% CIs for prevalent obesity class I or higher, per s.d. of PGSKhera and PGSLC, adjusted for age, sex, principal components of ancestry and genotyping array. All PGSs were standardized using the mean and s.d. of the PGS within individuals who did not have obesity class I or higher, to account for differences in prevalence across validation populations. Sample sizes (distinct individuals), from left to right: AFR 12,263, 2,332, 18,701; AMR 10,281, 8,096; AS 4,201; EAS 1,359; SAS 1,177; EUR 13,673, 69,828, 340,224. c, AUC classification of prevalent obesity outcomes in the BioMe Biobank, the MVP and the UKBB. Models including PGSs (included as a continuous predictor) additionally include principal components of ancestry. CBS10, self-reported comparative body size at age 10 years. After an ancestry adjustment (Methods and Extended Data Fig. 4), the median odds ratio for obesity class I or higher across BioMe's population groups was 3.6 for those in the top 10% of PGSLC (compared to bottom 90%) and 4.1 for the top 5% (versus those in the bottom 95%) (Supplementary Table 12), with greater separation between these two tail estimates for European-like and Asian-like ancestry groups. We investigated whether PGSLC has predictive value at an early age within the Avon Longitudinal Study of Parents and Children (ALSPAC), a geographically homogeneous prospective birth cohort from the southwest of England with follow-up until early adulthood (Methods and Supplementary Table 3)27,28,29. Repeated cross-sectional associations with BMI showed small effects for PGSLC soon after birth with much stronger effects emerging in early childhood, from 0.12 s.d. of PGSLC at 12 months to nearly quadruple that size (0.45 s.d. of PGSLC) by age 12 years, after which effects plateaued (Fig. The PGS was also associated, albeit less strongly, with height in early childhood with effects increasing until age 12 years, after which they returned to zero by mid-adolescence, suggesting that genetic predisposition for higher BMI early in life promotes increased body size in general, including postnatal linear growth, but that this early growth does not translate into differences in height after puberty (Extended Data Fig. Over time, BMI in children with a higher genetic predisposition (PGS ≥10th percentile) increased at a faster rate than those with a lower genetic predisposition, most evident after age 2.5 years (Methods, Fig. Both boys and girls with a very early age of adiposity rebound (≤43 months), a well-established predictor of future obesity risk31, had a higher mean PGS than those with later ages of adiposity rebound (Supplementary Table 14). a, Repeated cross-sectional linear regression associations of standardized PGSLC with BMI and height, with both standardized within sample by sex and timepoint. Data are presented as regression coefficient with 95% confidence interval (CI). Ponderal index was used instead of BMI at birth. Associations were adjusted for age and principal components of ancestry. Sample sizes, based on repeated measurements, from left to right, for BMI: 4,740, 638, 847, 814, 769, 41, 732, 729, 725, 720, 699, 5,816, 4,863, 5,570, 5,368, 5,187, 4,910, 4,556, 4,024, 3,603, 2,780; and height: 4,802, 638, 847, 814, 771, 741, 734, 729, 726, 723, 701, 5,820, 5,160, 5,572, 5,379, 5,188, 4,956, 4,561, 4,032, 3,606, 2,782. b, Sequentially plotted mean BMI trajectories from the age of 4 months to 24 years with knot points from linear spline multilevel models, accounting for sex and principal components of ancestry, of PGSLC (bottom 10%, middle 80%, top 10%). c, Contribution of PGSLC to explained variance (adjusted R2) for BMI, rank-based inverse-normal transformed by sex and timepoint. Ponderal index was used instead of BMI at birth. Data are presented as R2 values computed from the original dataset, with error bars representing 95% confidence intervals (2.5th–97.5th percentiles) estimated from 1,000 bootstrap resamples. In contrast, the right panel shows explained variance for BMI measured at 18 years, with early-life BMI measurement shown on the x axis used as predictors. Sample sizes, based on repeated measurements, from left to right: 3,800, 839, 714, 1,192, 4,062, 3,044, 940, 594, 737, 725, 890, 3,310. We then examined whether the PGS adds predictive value over and above clinically available predictors of obesity. When added to predictors that are measurable at birth (birthweight, maternal education, pre-pregnancy maternal BMI, maternal age at date of birth and household social status), the PGS showed no clear added value for predicting ponderal index at birth or BMI at ages 3 years and 5 years (Methods and Fig. However, for predicting BMI at later ages (8, 11 and 15 years), the contribution of the predictors at birth plateaued, whereas inclusion of the PGS roughly doubled the total explained variance from 11% to 21% at age 8 and from 13% to 26% at age 15 (Fig. BMI measured at age 8 explained 44% of the variation in BMI at age 18, and adding the PGS to the prediction model only raised this to 49%. The addition of the PGS to a model with birth year, sex and BMI at age 20 was associated with modest increases in discriminative ability for predicting obesity at age 50 (incremental AUC ranged from 0.01 to 0.03; Fig. There was evidence of statistical interaction between the PGS and BMI at age 20 for obesity at age 50 (Methods and Supplementary Table 17), which translated into a larger incremental AUC of the PGS for those who did not have overweight or obesity at age 20 (range, 0.02–0.05; Fig. 5), without strong evidence of sex specificity (Extended Data Fig. Analyses performed within the PLCO Cancer Screening Trial. Population descriptors were provided by the PLCO investigators and reflect genetically determined ancestry using Genetic Relationship and Fingerprinting (GRAF). Lifestyle modification remains a cornerstone of weight management, with multicomponent ILIs having shown success in achieving clinically relevant weight loss33. However, ILIs require substantial, sustained investment from both healthcare providers and patients, with large variation in weight loss and unintentional weight regain being common34,35,36. To examine whether polygenic predisposition to obesity might modify the effectiveness of such interventions, we examined whether the PGS was associated with weight loss during the first year of an ILI, and with weight regain thereafter, in two randomized controlled trials with similar intervention arms aimed at achieving 7% weight loss: the Diabetes Prevention Program (DPP33) and the Look AHEAD (Action for Health in Diabetes37) studies (Methods). Among 3,909 participants (Supplementary Table 4), individuals with a higher PGSLC lost more weight during the first year in response to the ILI compared to the control group (−0.55 kg per s.d. In addition, among those who lost at least 3% of their baseline weight during the first year, a higher PGS was associated with more weight regain in the following years (up to 3 years) (0.48 kg per PGS s.d., 95% confidence interval: 0.00–0.95) (Fig. These findings were directionally consistent across (self-)reported population groups. Interaction effect between PGSLC and trial arm (ILI versus comparison arm) for weight change within the first year of follow-up (left) and weight change after the first year in the subset of individuals who had lost ≥3% of baseline weight at year 1 (right). Data are presented as study-specific and study-combined interaction regression coefficients (with 95% confidence interval (CI)), pooled through inverse-variance weighted fixed-effect meta-analysis. Weight change was assessed by using weight at follow-up timepoint(s) as outcome in the linear mixed models, adjusting for initial weight (baseline or year 1, respectively). Population descriptors reflect race and ethnicity, as reported by dbGAP variable phv00201855.v2.p1 (DPP) or self-report by participants (LA). Study-specific sample sizes (distinct individuals, with pooled sample sizes for ‘All'), from top to bottom, for forest plot on the left: 295, 374, 251, 428, 839, 1,722, 1,385, 2,524; and on the right: 118, 186, 128, 233, 428, 951, 674, 1,370. We used GWAS summary statistics from the GIANT consortium and 23andMe, encompassing over 5.1 million people, to create ancestry-specific and multi-ancestry PGSs capturing genetic predisposition to weight gain and obesity. Our multi-ancestry PGS more than doubled the explained variance for BMI compared to the widely used PGS by Khera et al.12. Additionally, we demonstrate the potential added value of the PGS in two distinct clinical applications: predicting adult BMI at an early age and weight change in response to ILIs. Across diverse populations, the increases in explained variance for BMI were accompanied by substantial improvements in effect sizes and discrimination metrics for obesity, with the PGSs' standalone AUC nearing 0.80 for severe obesity. Consistent with other polygenic traits, our results underscore the importance of training sample size in driving PGS performance improvements. Nonetheless, our results also show diminishing returns of sample size increases, as has been observed for other polygenic traits38, with a 15-fold increase in total sample size (compared to PGSKhera, ~10-fold in EUR alone) leading to a 2–3-fold increase in prediction accuracy. However, recent Bayesian methods have shown promising results with smaller sample sizes by including additional variants and functional annotation, meriting consideration for future comparisons39,40. Although our results highlight the value of multi-ancestry PGS to increase PGS performance, a considerable performance gap persists for populations with substantial African-like genetic ancestry compared to other populations. This discrepancy is likely due to the underrepresentation of individuals with African-like ancestry in the training GWAS, particularly from continental Africa41, and differences in minor allele frequency (MAF) and linkage disequilibrium (LD) patterns42,43. The lower performance of the PGS in the GPC-UGR is in line with those seen therein for PGSs for other traits and the limited transferability of PGS performance between African ancestry populations, reflecting genetic and environmental differences as well as their interactions22,44,45,46,47, emphasising that much remains to be done to improve the performance of genetic risk prediction across the genetically diverse populations within Africa. For example, it remains unclear whether a finer classification of broad ancestry groups in GWAS meta-analyses and PGS development could improve downstream PGS generalizability, especially given the dominant role of GWAS sample size. Given the observed performance gap, careful implementation of the score is essential to avoid potentially introducing public health disparities. For example, establishing a standardized ancestry threshold14,48 may improve the accuracy of population-specific risk stratification and ensure that the score is returned only in populations where it meets an acceptable performance threshold. Within ALSPAC, the PGS—which was developed for adult BMI—already shows an effect on BMI early in life. Effect sizes rapidly increased with age and were accompanied by clear divergence of BMI trajectories throughout childhood and adolescence. As a result, the PGS showed added value beyond clinically available predictors assessed at birth for predicting BMI measured after age 5. This is in line with previous research on the dynamic relationship between genetic variation and adiposity in early life12,49,50. Particularly elucidating are the observations from the Norwegian Mother, Father and Child Cohort Study (MoBa) with repeated measurements across narrow age windows51. They show that BMI has a rapidly changing genetic architecture during the first 8 years of life with a ‘late rise' cluster of variants, emerging in late childhood, which show limited association before the adiposity rebound but have persistent effects on BMI into adult life51. This also explains why our PGS shows its value for predicting adult BMI during the earliest years of life, up to age 5, when measured BMI has limited value as a predictor. With a growing number of obesity prevention trials commencing in early life52,53, our results provide important context at what age(s) PGSs for obesity, potentially in tandem with PGSs of obesity-related complications, could be considered promising candidate predictors to help guide risk stratification and the implementation of such interventions. In the PLCO study, a higher PGS was associated with weight gain over a 30-year period from early adulthood to midlife. Additionally, for individuals aged 20, knowing their PGS modestly improved the prediction of obesity at age 50, more so for those without overweight or obesity at age 20. This suggests that particularly individuals with an early adulthood BMI below their genetically predicted BMI are at risk of gaining weight to match their innate predisposition. Our findings are consistent with those from the CARDIA study54, showing that PGSs for obesity only marginally improve the prediction of midlife BMI when combined with early adulthood BMI measurements, contrasting with its predictive value in early life. Analyzing clinical trial data of ILIs, we observed that individuals with a higher PGS lost modestly more weight during the first year. However, this group was also at higher risk of weight regain after this most intensive portion of the intervention had concluded. This may seem counterintuitive to the expectation that those with a higher genetic risk will benefit less from weight loss interventions. However, this observation is supported by a strong body of literature reporting that those most genetically predisposed to obesity are also those most responsive to changes in an obesogenic environment55,56,57,58,59,60,61,62,63. However, the literature on the role of genetics modifying response to weight loss interventions remains sparse and ambiguous, reflecting differences in variants considered, methods and study design, highlighting the need for well-powered PGSs64,65,66,67,68,69,70,71,72,73. It will be of interest to investigate whether PGSs for obesity have predictive utility for the fast-growing arsenal of pharmacotherapies aimed at weight loss. Separately, the current results offer crucial context when conveying genetic results of obesity risk to participants, which has generally shown either no or even short-lived adverse effects on risk-reducing behavior74,75,76,77. Rather, our findings emphasize that individuals with a high genetic predisposition to obesity may respond more to lifestyle changes and, thus, contrast with the determinist view that genetic predisposition is unmodifiable78. This is further reinforced by evidence that polygenic susceptibility can mitigate or exacerbate the impact of pathogenic variants in MC4R, underscoring the complexity of genetic influences on obesity79. We look forward to the findings of the Electronic MEdical Records and GEnomics (eMERGE) network's PGS-based genome-informed risk assessment being returned to 25,000 diverse adults and children for 11 conditions, including obesity/BMI14. Taken together, we show that BMI PGSs can be used for prediction of adult obesity throughout the life course, particularly in early life, and for severe obesity. This PGS represents a substantial improvement compared to previous scores and may help to identify individuals at high risk to allow for timely prevention or treatment of obesity, such as through integration in a broader predictive framework jointly modeling genetic and environmental risk. The study populations described below are those that contributed as a PGS tuning or validation population. The UKBB is a prospective cohort study that enrolled approximately 500,000 people from across the UK, aged 40–69 years at recruitment, between 2006 and 2010 (ref. At recruitment, participants completed detailed questionnaires, underwent a range of physical measures and provided blood, urine and saliva samples. BMI was calculated using height (measured in whole centimeters) and weight (to the nearest 0.1 kg). As a measure of comparative childhood body size, participants were asked: ‘When you were 10 years old, compared to average, would you describe yourself as: (i) thinner, (ii) plumper, (iii) about average? The MVP has recruited over 1 million people from Veteran Affairs (VA) Medical Centers across the United States since 2011 (ref. Veterans who volunteer provide a blood sample for biobanking, complete baseline and lifestyle questionnaires and consent to allow access to clinical data from VA electronic health records (EHRs). For the current study, PGS performance was assessed in the MVP's third release of genomic data (R3) using additional participants without overlap, with earlier data releases contributing to the GIANT GWAS meta-analyses. Phenotypic and genomic data related to the association analysis of BMI were previously described20. The Institute for Personalized Medicine BioMe Biobank, founded in 2007, is an ancestrally and culturally highly diverse EHR-linked biorepository enrolling participants non-selectively from across the Mount Sinai Health System in New York City21. The clinical and EHR information is complemented by a baseline questionnaire that gathers demographic and lifestyle information. BMI was calculated using weight and height from baseline and outpatient EHR measurements, using median height and weight measured at or within 60 days before/after the enrollment visit, after several cleaning steps (Supplementary Table 3). The GPC-UGR is a population-based open cohort study established in 1989 by the Medical Research Council (UK) in collaboration with the Uganda Virus Research Institute (UVRI) to monitor the HIV epidemic and its determinants in neighboring villages in rural southwestern Uganda, in the Kyamulibwa subcounty of the Kalungu district, approximately 120 km from Entebbe town. Since 2010, its mandate has expanded to incorporate the epidemiology and genetics of both communicable and non-communicable diseases22. The GPC-UGR was initially recruited and assessed through annual house-to-house census and survey rounds until 2012, when biannual surveys commenced. For the current study, we included individuals aged 18 years and older who have been either whole-genome genotyped or whole-genome sequenced80,81. The ALSPAC is a prospective birth cohort from the southwest of England established to investigate environmental and genetic characteristics that influence health, development and growth of children and their parents27,28,29. Full details of the cohort and study design are available at http://www.alspac.bris.ac.uk. In brief, pregnant females residing in Avon, UK, with expected dates of delivery between 1 April 1991 and 31 December 1992 were invited to take part in the study. The initial number of pregnancies enrolled was 14,541, with 13,988 children who were alive at 1 year of age. The children resulting from these pregnancies have been followed-up to date with measures obtained through regular questionnaires and clinical visits, providing information on a range of behavioral, lifestyle and biological data. More specifically, a 10% sample of the ALSPAC cohort, known as the Children in Focus (CiF) group, attended clinics at the University of Bristol at various time intervals between 4 months and 61 months of age, whereas the entire ALSPAC cohort was invited to attend regular research clinics from age 7. When the oldest children were approximately age 7, an attempt was made to bolster the initial sample with eligible individuals who had failed to join the study originally. The total sample size available for analyses using any data collected after the age of 7 is, therefore, 15,447 pregnancies, with 14,901 children who were alive at 1 year of age. Study data were collected and managed using Research Electronic Data Capture (REDCap) tools hosted at the University of Bristol82. REDCap is a secure, web-based software platform designed to support data capture for research studies. Details on sample selection for the current analyses are presented in Supplementary Table 3. Please note that the study website contains details of all data that are available through a fully searchable data dictionary and variable search tool (http://www.bristol.ac.uk/alspac/researchers/our-data/). The PLCO Cancer Screening Trial was a multicenter randomized controlled trial in the United States that enrolled males and females aged 55–74 years from 1993 to 2001 to evaluate the effectiveness of different screening programs on cancer mortality32. In the baseline questionnaire, participants self-reported their weight at age 20 years, age 50 years and baseline as well as their height. For the current study, we restricted the analyses to genotyped participants who had not contributed to the underlying GWAS meta-analyses and had information available on their weight and BMI at both age 20 and age 50. The DPP was a 27-site parallel-arm randomized controlled trial designed to determine whether either the oral diabetes drug metformin or an ILI (primarily fat gram, calorie and physical activity goals) aimed at approximately 7% weight loss, compared to inactive tablets and standard lifestyle recommendations, could prevent or delay type 2 diabetes onset in ethnically diverse high-risk individuals with prediabetes and overweight or obesity33. For the current study, we excluded individuals from the metformin arm as well as those from a fourth intervention arm, troglitazone, which was discontinued at an early stage. Moreover, we focused on the individuals comprising the three largest race and ethnicity categories (White, Black and Hispanic, as reported by database of Genotypes and Phenotypes (dbGAP) variable phv00201855.v2.p1) who had consented to being genotyped. Look AHEAD was a 16-site parallel-arm randomized controlled trial that assessed the long-term effects of an ILI in ethnically diverse patients with overweight or obesity and type 2 diabetes on cardiovascular morbidity and mortality37. The lifestyle intervention was modeled after the DPP lifestyle intervention and similarly aimed at 7% weight loss but with more ambitious individual goals for several intervention components83. Although those in both trial arms were provided one session of education on diabetes and cardiovascular risk at baseline, the comparison arm thereafter received the option of attending only three sessions per year on nutrition, physical activity and social support, with no explicit weight loss goals. As for the DPP, we restricted our analyses to genotyped participants who self-reported as being of African American/Black, Hispanic and White race and ethnicity on the baseline questionnaire. No new data (that is, measurements) were collected for this study, and no participant compensation was provided for the analyses conducted. Our research complies with all relevant ethical regulations. 11/NW/0382), and all participants provided written informed consent to participate in the UKBB study. Informed consent was obtained from all participants. 07-0529) operates under a Mount Sinai IRB-approved research protocol. All study participants provided written informed consent. HS 1978), the Uganda National Council for Science and Technology (UNCST no. SS 4283) and the East of England-Cambridge South (formerly Cambridgeshire 4) NHS Research Ethics Committee UK. Consent for biological samples has been collected in accordance with the Human Tissue Act (2004). Written informed consent was given by all participants. The DPP and Look AHEAD trials were approved by the IRB at each center, and all participants gave written informed consent. To derive our scores, we used summary statistics from the ongoing GWAS meta-analyses for BMI conducted by the GIANT consortium (https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium) in collaboration with 23andMe. Prior to each GWAS, BMI (kg m−2) underwent rank-based inverse-normal transformation by sex, similarity to major ancestry group (AFR, AMR, EAS, EUR, SAS) and case–control status as appropriate, with age, age2, principal components of population structure and study-specific covariates regressed out. Ancestry group assignment was defined independently by each contributing study, as previously described84. Additive GWAS analyses were performed primarily using RVTESTS for 1000 Genomes Project (1000G) phase 3 or Haplotype Reference Consortium (HRC)-imputed data85. For 23andMe, which performed its GWAS on BMI using self-reported height and weight, see Supplementary Table 2 for details on phenotyping, ancestry group assignment and GWAS methodology. Quality control of the study-specific files was conducted using EasyQC86, followed by fixed-effect meta-analysis by ancestry group using RAREMETAL87. When performing the GWAS meta-analyses to train the PGSs with, we purposely excluded data from several populations (ALSPAC, BioMe and UKBB). No additional genomic control was applied to the resulting summary statistics. For PGS derivation, we employed PRS-CS and its multi-ancestral extension PRS-CSx (versions 1.0.0)16,17. In short, PRS-CS(x) uses a Bayesian regression framework and assumes a continuous shrinkage prior on single-nucleotide polymorphism (SNP) effects. Whereas PRS-CS focuses on creating ancestry-specific scores, informed only by the GWAS summary statistics and LD information of a given population, PRS-CSx uses a shared prior to couple SNP effects across populations and explicitly models population-specific allele frequencies and LD patterns to create multi-ancestry scores. Both methods were implemented using the software's default settings (that is, gamma–gamma priors set to a = 1 and b = 0.5, Markov chain Monte Carlo (MCMC) total and burn-in iterations set at 1,000 times and 500 times the number of discovery populations and MC thinning factor set at 5), with input being ancestry-matched summary statistics and PRS-CS(x) developer-provided 1000G LD reference panels of common HapMap3 variants (AFR, AMR, EAS, EUR, SAS). PRS-CS(x) uses a global scaling parameter (Φ), which requires tuning. In addition to small-scale grid testing of Φ (1 × 10−6, 1 × 10−4, 1 × 10−2, 1), we ran PRS-CS(x) using its ‘auto' option, which learns Φ automatically from the GWAS summary statistics. Finally, PRS-CSx was performed separately using its default ‘linear combination' as well as its ‘meta' version. In the first, a set of population-specific PGSs is outputted (that is, five in our case), for which the optimal linear combination needs to be derived for each target population (for example, AFR) in a population-matched tuning dataset. For ‘meta', the population-specific posterior SNP effects are integrated using an inverse-variance weighted meta-analysis in the Gibbs sampler, producing a single score that can theoretically be applied regardless of the target population. The variants considered for PGS derivation were those present in the 1000G reference panels mentioned above (HapMap3 variants (n = 1–1.2 million across panels), which were common in 1000G populations, excluding ambiguous A/T or G/C variants), which additionally had an INFO > 0.3 (that is, imputation quality) in the overall UKBB. Ancestry-specific summary statistics were further restricted to the variants with a variant-specific sample size of at least a third of the maximum sample size for that ancestry. More specifically, we used genetically determined relatedness and ancestry assignments provided by the Pan-UKBB Team (https://pan.ukbb.broadinstitute.org, UKBB return 2442) to subset the UKBB into groups of unrelated individuals of African-like, Admixed American-like, Central/South Asian-like, East Asian-like, European-like (random subset of 20,000 individuals) and Middle Eastern-like ancestry. The target-population-specific (for example, AFR) linear combination weights for PRS-CSx (for each Φ) were derived from a joint linear regression model of rank-based inverse-normal transformed BMI (by sex and population) on the five population-specific scores (each standardized to mean zero and unit variance in the corresponding tuning dataset—for example, UKBBAFR), age, 10 principal components and genotyping array. Explained variance for BMI was defined as the incremental adjusted R2 from linear regression for rank-based inverse-normal transformed BMI (by sex and population), when adding the PGS predictor to a model containing age, principal components and genotyping array. Confidence intervals (95%) were determined using bootstrapping with 1,000 repetitions. As comparator, we created a score of quasi-independent associations after performing approximate conditional and joint (COJO) multiple-SNP analysis88, as implemented in GCTA89, using the results from the multi-ancestral GWAS summary statistics when leaving out only the UKBB. We used genotypes from a previously described set of 50,000 unrelated UKBBEUR participants84 as our LD reference panel for GCTA-COJO, with parameters set to: –diff-freq 0.1; –maf 0.01; –cojo-collinear 0.9; –cojo-p 5 × 10−9. The scores with the highest explained variance across multiple tuning population datasets (PRS-CSx linear combination and meta versions, with Φ 1 × 10−2) were then taken forward to the validation populations, which performed study-level variant filtering on imputation quality when constructing each PGS (Supplementary Table 3). Of note, in August 2023, a new version of PRS-CSx was released (version 1.1.0) to address potential reductions in performance due to truncating of GWAS summary statistics P values below 1 × 10−323. Given our inclusion of several variants with P values below this threshold (among the PGS variants: n = 154 for EUR, n = 12 for AMR), we reran PRS-CSx with Φ 1 × 10−2 but did not observe markedly different results in PGS performance within the UKBB tuning populations (Supplementary Table 18). In addition to explained variance (%) for BMI, several population-stratified metrics for overall performance and discrimination were calculated in the validation studies with adult participants: Nagelkerke R2, with a null model of age, sex, principal components and genotyping array, for prevalent obesity categories based on World Health Organization (WHO) BMI cutoff points (membership of category or above versus rest), using Asian-specific cutoffs where applicable (Supplementary Table 3)23. Confidence intervals (95%) were calculated using bootstrapping with 1,000 repetitions. Mean difference in BMI, both untransformed and after rank-based inverse-normal transformation by sex, per s.d. Covariates included age, sex (where applicable), principal components and genotyping array. of PGS for prevalent WHO obesity outcome. Here, the PGSs were standardized to mean zero and unit variance using the mean and s.d. of the PGS observed within participants without the outcome (for example, individuals who do not fall within obesity class I (or higher), where this was the outcome), to account for prevalence differences across studies. AUC statistic for prevalent WHO obesity outcomes. In the UKBB, we additionally ran models including and excluding self-reported comparative body size at age 10 (UKBB field ID 1687). Confidence intervals (95%) were calculated using DeLong's method. In addition, the explained variance for BMI was calculated separately in age strata (≤50 years or >50 years, with adjustment for residual age differences within age strata) and sex strata of the European UKBB participants. The main comparator for all analyses was a previously published LDpred-derived PGS for BMI (PGSKhera)12, which was based on the results from a GIANT consortium GWAS meta-analysis including up to 339,224 individuals of primarily European ancestry24. As BioMe had contributed to this meta-analysis, metrics for PGSKhera were determined in individuals who had not contributed nor were related to those who did (second or stronger degree of relatedness, based on KING-derived kinship coefficients). We chose PGSKhera as our comparator as it is the most widely recognized and extensively used PGS for BMI within the obesity research community, thereby serving as a well-established benchmark. Due to on-average differences in LD structure and allele frequencies, the distribution of a PGS (for example, mean and variance) can differ across ancestral populations. This was also observed for the five population-specific PGSs that were linearly combined to create PGSLC (Extended Data Fig. As performance metrics that rely on thresholding the PGS distribution (for example, top 10%) may be impacted by such on-average differences90, particularly in admixed populations, we modeled the mean and variance of the population-specific PGSs through principal component analysis of the 1000G reference panel. For this, we applied a correction described by the eMERGE network14,91, which represents a modified version of the approach by Khera et al.92. Starting with a publicly available, curated version of the 1000G reference panel (https://broadinstitute.github.io/warp/docs/Pipelines/Imputation_Pipeline/references_overview; gs://broad-gotc-test-storage/imputation/1000 G_reference_panel), we first excluded variants with MAF < 1% across 1000G superpopulations, before determining the overlap with genotyped variants available for both the UKBB and BioMe. After excluding long-range LD regions, we pruned the list of variants using plink (–indep-pairwise 1000 50 0.05) and ran principal component analysis with flashpca93, whereafter we projected UKBB and BioMe participants to the same 1000G principal component space. For each of the five population-specific PGSs underlying PGSLC (that is, AFR, AMR, EAS, EUR and SAS), we regressed the PGS of 1000G participants against the first 10 principal components of ancestry: In addition, we modeled its residual variance (\({\delta }^{2}\)) as a function of the same principal components: Then, using the projected principal component, predicted mean and residual s.d. were calculated and used to create ancestry-adjusted z-scores of each PGS for UKBB and BioMe participants: We observed that mean and variance differences of scores across ancestry populations were largely, but not fully, resolved through this correction in both study populations (Extended Data Fig. In keeping with PRS-CSx's linear combination approach, weights to linearly combine scores were derived within each UKBB tuning population and applied to BioMe validation populations. Given the relatively lower number of East Asian and South Asian individuals available for these tail comparisons, we present pooled estimates. For this, we used PRS-CS, combining GIANT's combined multi-ancestral GWAS summary statistics (including BioMe but excluding the UKBB and the ALSPAC study) with a previously described multi-ancestry 1000G-based LD reference panel created following the same protocol as described by the PRS-CS authors94. We restricted variants to those present in the LD reference panel, which additionally had MAF ≥ 0.1% and INFO ≥ 0.8 in the overall UKBB, resulting in a total of 1,217,710 variants included. We did not specify a genome-wide shrinkage parameter for PRS-CS, which it therefore learned from the training data using a fully Bayesian approach (its ‘auto' option). In collaboration with the eMERGE network, this GIANT-only PGS was included among a select group of trait-specific scores currently being returned as part of a genome-informed risk assessment (GIRA) to 25,000 diverse adults and children and their healthcare providers in an ongoing prospective cohort study across 10 clinical sites, as recently described14,48. In line with the aim of returning a high-risk versus a not-high-risk GIRA status to participants, we estimated the odds ratio for obesity class I (or higher) for being in the top 3% of the PGS distribution (versus bottom 97%) within the subset of BioMe participants genotyped with the Global Screening Array (with 1000G imputation) who had not contributed to the underlying GIANT GWAS meta-analysis nor were related to those who did. Accounting for age, sex and four principal components, we observed that values within the top 3% of this PGS were associated with 4.08 (95% confidence interval: 3.02–5.52) times as high odds for prevalent obesity in individuals of self-reported European descent, with 2.54 (95% confidence interval: 1.55–3.98) for individuals of self-reported African descent, with 2.33 (95% confidence interval: 1.64–3.31) for individuals of self-reported Hispanic/Latino descent and with 5.73 (95% confidence interval: 2.28–14.57) for individuals of self-reported Asian descent. Within the ALSPAC study, repeated anthropometric measurements are available from birth to age 24. We employed both cross-sectional and multilevel analyses to provide complementary perspectives: cross-sectional analyses facilitate granular comparisons at each timepoint, whereas multilevel models capture longitudinal trajectories and population-level divergence over time. First, we assessed repeated cross-sectional associations between the standardized PGS (PGSLC using linear combination weights from UKBBEUR) and all available measures of BMI (or ponderal index at birth) and height. Anthropometric measurements were rank-based inverse-normal transformed separately by sex, at each timepoint, and analyses were adjusted for age at timepoint and principal components. Longitudinal analyses using linear spline multilevel models were conducted to examine the association between the PGS (bottom 10%, middle 80%, top 10%) and change in untransformed BMI and height between 4 months and 24 years. Multilevel models estimate the mean trajectories of each anthropometric trait while accounting for non-independence of repeated measures within individuals, change in scale and variance of measures over time and differences in the number and timing of measurements between individuals (using all available data from all eligible participants under a missing-at-random assumption). Linear splines allow knot points to be fitted at different ages to derive periods of change that are approximately linear. All participants with at least one measure of the anthropometric traits were included under a missing-at-random assumption to minimize selection bias in trajectories estimated using linear spline multilevel models (with two levels of random effects: measurement occasion (that is, age to the nearest integer in years) and individual), allowing individuals to have different intercepts and slopes and, thus, their own trajectories. Additionally, interaction terms between sex and the first 10 genetic principal components with each spline were included to estimate the difference in intercepts and slopes between males and females; therefore, models were adjusted for sex and principal components. These children represent a 10% sample of the cohort, randomly selected for more detailed investigations, and are representative of the entire cohort (https://www.bristol.ac.uk/alspac/researchers/cohort-profile/). One-way ANOVA was used to test for differences in mean PGS across categories, in boys and girls separately and combined. Added value of the PGS for prediction of BMI (rank-based inverse-normal transformed by sex) at given timepoints was quantified as incremental explained variance, when adding the PGS to different sets of predictors. First, we examined R2 for BMI at birth and at 3, 5, 8, 11 and 15 years, when adding the PGS to a model containing multiple relatively easily obtainable clinical variables that would be available at birth (birthweight, maternal education, pre-pregnancy maternal BMI, maternal age at date of birth and household social status). Details regarding phenotyping of these variables can be found in Supplementary Table 3. Separately, for R2 for BMI at age 18, we added the PGS to separate models containing BMI from a given early-life timepoint (1, 2, 3, 4, 5 and 8 years). Confidence intervals (95%) were calculated using bootstrapping with 1,000 repetitions. Within PLCO, we assessed the population-statified and sex-stratified association between the standardized PGSLC (using linear combination weights from the UKBB most closely corresponding to each testing population) and per-5-year change between ages 20 and 50 in weight and BMI, with and without adjustment for the initial measurement at age 20. These analyses were additionally adjusted for birth year, sex (where appropriate) and principal components. For this, we ran models including (i) birth year and sex, (ii) birth year, sex, principal components and PGSLC, (iii) birth year, sex, principal components and BMI at age 20 and (iv) birth year, sex, principal components, BMI at age 20 and PGSLC. We separately ran model (iv) when including an interaction term between the PGS and BMI at age 20. Based on evidence for statistical interaction between the PGS and BMI at age 20 for predicting obesity at age 50 across multiple populations (Supplementary Table 16), we stratified our analyses on the presence of overweight or obesity at age 20 and, additionally, on sex. Confidence intervals (95%) for AUC were calculated using DeLong's method. Although neither intervention nor comparison arms were identical across DPP and Look AHEAD, both intervention arms were aimed at 7% weight loss while focused on the same process features83, and both represent a large shift in lifestyle behavior relative to the comparison arms. As such, we decided to pool study-specific effect estimates via inverse-variance weighted meta-analysis. In each study, we ran analyses after pooling the three largest population groups present in both trials: Black and/or African American, Hispanic and White (Supplementary Table 3). Population-stratified analyses were separately run as well. Within each study-specific population, we applied the linear combination weights for PGSLC most closely aligned with the target population (AFR, AMR and EUR, respectively). These scores were then standardized within the matching population, ignoring trial arm, whereafter we standardized again across all individuals for the population-pooled analyses. Due to restricting analyses to genotyped individuals from the three largest population groups, trial randomization was, by definition, broken. To investigate potential shifts in the PGS, we plotted its distribution by trial arm, which showed minimal shifts in distribution (Extended Data Fig. The weight loss nadir in both trials is known to occur at the 1-year mark, after which many individuals start regaining weight. As such, we separately examined the association of the PGS with (i) weight at year 1 (adjusting for baseline weight) and (ii) repeated weight measurements beyond year 1 up to year 4 (adjusting for weight at year 1) in the subset of individuals who had lost ≥3% of their initial body weight in the first year. Analyses were run by trial arm using linear mixed models with random intercepts, with additional covariates being age, sex, principal components, the initial weight measurement (baseline or year 1) and random slopes for time (years). In addition, we ran trial–arm combined models in which we examined interaction effects between trial arm and PGS. These interaction terms are our main estimates of interest, as they reflect how genetic predisposition to obesity modifies the intervention's effect on weight change, relative to the comparison arm. Study-specific main and interaction effect estimates were separately pooled by means of inverse-variance weighted fixed-effect meta-analysis. Participant characteristics are presented in Supplementary Table 4. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Pan-UKBB relatedness and ancestry determinations can be obtained via UKBB return 2442. Due to US Department of Veterans Affairs (VA) regulations and our ethics agreements, the analytic datasets used for this study are not permitted to leave the Million Veteran Program (MVP) research environment and the VA firewall. This limitation is consistent with other MVP studies based on VA data. However, the MVP data are made available to researchers with an approved VA and MVP study protocol. Information regarding access to the BioMe Biobank is available at https://icahn.mssm.edu/research/ipm/programs/biome-biobank. For access to the Uganda GPC-UGR data, contact should be made to the director of the MRC/UVRI & LSHTM Uganda Research Unit, Entebbe, by email to mrc@mrcuganda.org. The data underlying this article are available upon reasonable request to the PLCO Cancer Data Access System: https://cdas.cancer.gov/plco/. Requests from researchers to access ALSPAC data and samples are welcomed; relevant procedures can be found at https://www.bristol.ac.uk/alspac/researchers/access/. Data from the Diabetes Prevention Program (V9) (https://doi.org/10.58020/3hw5-cf91) reported here are available for request at the NIDDK Central Repository (NIDDK-CR) website, Resources for Research (R4R) (https://repository.niddk.nih.gov/). Researchers can access data from the primary Look AHEAD data trial on the NIDDK repository. Due to limitations specified in informed consent, the Look AHEAD genetic data are not publicly available. We used publicly available software tools for all analyses. Dale, C. E. et al. Causal associations of adiposity and body fat distribution with coronary heart disease, stroke subtypes, and type 2 diabetes mellitus: a Mendelian randomization analysis. Nyberg, S. T. et al. Obesity and loss of disease-free years owing to major non-communicable diseases: a multicohort study. Treating chronic diseases without tackling excess adiposity promotes multimorbidity. Elmaleh-Sachs, A. et al. Obesity management in adults: a review. Muller, T. D., Bluher, M., Tschop, M. H. & DiMarchi, R. D. Anti-obesity drug discovery: advances and challenges. The epidemiological burden of obesity in childhood: a worldwide epidemic requiring urgent action. Wray, N. R., Kemper, K. E., Hayes, B. J., Goddard, M. E. & Visscher, P. M. Complex trait prediction from genome data: contrasting EBV in livestock to PRS in humans: genomic prediction. Kullo, I. J. et al. Polygenic scores in biomedical research. Khera, A. V. et al. Polygenic prediction of weight and obesity trajectories from birth to adulthood. Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Selection, optimization and validation of ten chronic disease polygenic risk scores for clinical implementation in diverse US populations. Promoting equity in polygenic risk assessment through global collaboration. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Improving polygenic prediction in ancestrally diverse populations. The UK Biobank resource with deep phenotyping and genomic data. Gaziano, J. M. et al. Million Veteran Program: a mega-biobank to study genetic influences on health and disease. Huang, J. et al. Genomics and phenomics of body mass index reveals a complex disease network. Belbin, G. M. et al. Toward a fine-scale population health monitoring system. The general population cohort in rural south-western Uganda: a platform for communicable and non-communicable disease studies. Locke, A. E. et al. Genetic studies of body mass index yield new insights for obesity biology. The influence of age and sex on genetic associations with adult body size and shape: a large-scale genome-wide interaction study. Ge, T., Chen, C. Y., Neale, B. M., Sabuncu, M. R. & Smoller, J. W. Phenome-wide heritability analysis of the UK Biobank. Fraser, A. et al. Cohort profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort. The Avon Longitudinal Study of Parents and Children (ALSPAC): an update on the enrolled sample of index children in 2019. Rolland-Cachera, M. F., Deheeger, M., Maillot, M. & Bellisle, F. Early adiposity rebound: causes and consequences for obesity in children and adults. Prorok, P. C. et al. Design of the Prostate, Lung, Colorectal and Ovarian (PLCO) Cancer Screening Trial. Reduction in the incidence of type 2 diabetes with lifestyle intervention or metformin. Maintenance of weight loss after lifestyle interventions for overweight and obesity, a systematic review. Machado, A. M. et al. Understanding weight regain after a nutritional weight loss intervention: systematic review and meta-analysis. & Mariman, E. C. M. Obesity-induced and weight-loss-induced physiological factors affecting weight regain. Look AHEAD Research Group et al. Cardiovascular effects of intensive lifestyle intervention in type 2 diabetes. Zhang, Y. D. et al. Assessment of polygenic architecture and risk prediction based on common variants across fourteen cancers. Improving GWAS discovery and genomic prediction accuracy in biobank data. Zheng, Z. et al. Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. Fatumo, S. & Choudhury, A. African American genomes don't capture Africa's genetic diversity. Kachuri, L. et al. Principles and methods for transferring polygenic risk scores across global populations. Majara, L. et al. Low and differential polygenic score generalizability among African populations due largely to genetic diversity. B. et al. Transferability of genetic risk scores in African populations. Nalwanga, D. et al. Mortality among children under five years admitted for routine care of severe acute malnutrition: a prospective cohort study from Kampala, Uganda. The transferability of lipid loci across African, Asian and European cohorts. Returning integrated genomic risk and clinical recommendations: the eMERGE study. Sovio, U. et al. Association between common variation at the FTO locus and changes in body mass index from infancy to late childhood: the complex nature of genetic association through growth and development. Associations between genetic obesity susceptibility and early postnatal fat and lean mass: an individual participant meta-analysis. The Baby's First Bites RCT: evaluating a vegetable-exposure and a sensitive-feeding intervention in terms of child health outcomes and maternal feeding behavior during toddlerhood. Murthy, V. L. et al. Polygenic risk, fitness, and obesity in the Coronary Artery Risk Development in Young Adults (CARDIA) study. Physical activity attenuates the genetic predisposition to obesity in 20,000 men and women from EPIC-Norfolk prospective population study. Physical activity attenuates the influence of FTO variants on obesity risk: a meta-analysis of 218,166 adults and 19,268 children. Yang, J. et al. FTO genotype is associated with phenotypic variability of body mass index. Qi, Q. et al. Sugar-sweetened beverages and genetic risk of obesity. Qi, Q. et al. Fried food consumption, genetic risk, and body mass index: gene-diet interaction analysis in three US cohort studies. Tyrrell, J. et al. Gene–obesogenic environment interactions in the UK Biobank study. Brandkvist, M. et al. Genetic associations with temporal shifts in obesity and severe obesity during the obesity epidemic in Norway: a longitudinal population-based cohort (the HUNT Study). Variation in the heritability of child body mass index by obesogenic home environment. Variation at the melanocortin 4 receptor gene and response to weight-loss interventions in the diabetes prevention program. Papandonatos, G. D. et al. Genetic predisposition to weight loss and regain with lifestyle intervention: analyses from the Diabetes Prevention Program and the Look AHEAD randomized controlled trials. Valsesia, A. et al. Genome-wide gene-based analyses of weight loss interventions identify a potential role for NKX6.3 in metabolism. Lamiquiz-Moneo, I. et al. Genetic predictors of weight loss in overweight and obese subjects. McCaffery, J. M. et al. Genetic predictors of change in waist circumference and waist-to-hip ratio with lifestyle intervention: the Trans-NIH consortium for genetics of weight loss response to lifestyle intervention. The influence of single nucleotide polymorphisms on body weight trajectory after bariatric surgery: a systematic review. & McCaffery, J. M. Abdominal obesity genetic variants predict waist circumference regain after weight loss. Dashti, H. S., Scheer, F., Saxena, R. & Garaulet, M. Impact of polygenic score for BMI on weight loss effectiveness and genome-wide association analysis. The impact of communicating genetic risks of disease on risk-reducing health behaviour: systematic review with meta-analysis. Wang, C. et al. A randomized trial examining the impact of communicating genetic and lifestyle risks for obesity. Effect of direct-to-consumer genomewide profiling to assess disease risk. Impact of direct-to-consumer genomic testing at long term follow-up. & Loos, R. J. F. The role of polygenic susceptibility to obesity among carriers of pathogenic mutations in MC4R in the UK Biobank population. Gurdasani, D. et al. Uganda Genome Resource enables insights into population history and genomic discovery in Africa. Fatumo, S. et al. Uganda Genome Resource: a rich research database for genomic studies of communicable and non-communicable diseases in Africa. Harris, P. A. et al. Research electronic data capture (REDCap)—a metadata-driven methodology and workflow process for providing translational research informatics support. Delahanty, L. M. & Nathan, D. M. Implications of the diabetes prevention program and Look AHEAD clinical trials for lifestyle interventions. Yengo, L. et al. A saturated map of common genetic variants associated with human height. Zhan, X., Hu, Y., Li, B., Abecasis, G. R. & Liu, D. J. RVTESTS: an efficient and comprehensive tool for rare variant association analysis using sequence data. Winkler, T. W. et al. Quality control and conduct of genome-wide association meta-analyses. Feng, S., Liu, D., Zhan, X., Wing, M. K. & Abecasis, G. R. RAREMETAL: fast and powerful meta-analysis for rare variants. Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Yiangou, K. et al. Polygenic score distribution differences across European ancestry populations: implications for breast cancer risk prediction. Khan, A. et al. Genome-wide polygenic score to predict chronic kidney disease across ancestries. Khera, A. V. et al. Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction. & Inouye, M. FlashPCA2: principal component analysis of Biobank-scale genotype datasets. Leckie, G. & Charlton, C. runmlwin: a program to run the MLwiN multilevel modeling software from within Stata. We acknowledge the participants in each cohort contributing to this study. Additional acknowledgements are provided in the Supplementary Note. Support for title page creation and format was provided by AuthorArranger (https://authorarranger.nci.nih.gov/), a tool developed at the National Cancer Institute. This research was supported by the following funding bodies, which had no role in study design, data collection and analysis, preparation of the manuscript or decision to publish. ), N.1-AG-1-1, N.1-AG-1-2111, N01-AG-1-2100, N01-AG-1-2109, N01-AG-6-2101, N01-AG-6-2103, N01-AG-6-2106, N01-HC-25195, N01-HC-95159, N01-HC-95160, N01-HC-95161, N01-HC-95162, N01-HC-95163, N01-HC-95164, N01-HC-95165, N01-HC-95166, N01-HC-95167, N01-HC-95168, N01-HC-95169, N01-HD-1-3228, N01-HD-1-3329, N01-HD-1-3330, N01-HD-1-3331, N01-HD-1-3332, N01-HD-1-3333, N01-HG-65403, N01HC55222, N01HC85079, N01HC85080, N01HC85081, N01HC85082, N01HC85083, N01HC85086, N02-HL-64278, National Institute on Aging Intramural Research Program (E.M.S. ), 018996, EUROSPAN (LSHG-CT-2006-018947), 223004, HEALTH-F4-2007-201413 by the European Commission (QLG2-CT-2002-01254), EUROHEADPAIN (602633), HEALTH-F4-2007-9602768, HEALTH-F2-2009-223175, EpiMigrant, 279143, RiskyCAD (305739), HEALTH-F2-2013-601456 (CVGenes@Target), research and innovation programme (G.E.D. ), LSHM-CT-2007-037273, the Innovative Medicines Initiative Joint Undertaking under grant agreement no. ), the NIHR Official Development Assistance (ODA, 16/136/68), the NIHR Oxford Biomedical Research Centre, the NIHR to the Biomedical Research Centre at the Institute of Cancer Research and Royal Marsden Foundation NHS Trust, the NIHR-funded BioResource, Clinical Research Facility and Biomedical Research Centre based at Guy's and St. Thomas' NHS Foundation Trust in partnership with King's College London (TwinsUK and M.M. ), the UK National Institute for Health Research Academic Clinical Fellowship (S.J.H. ), University of Bristol NIHR Biomedical Research Centre (BRC-1215-2001) (N.J.T.). ), the Netherlands Genomics Initiative (NGI)/NWO Netherlands Consortium for Healthy Aging (NCHA) 050-060-810, NGI/NWO - the Consortium for Systems Biology (NCSB), the Center for Medical Systems Biology (CSMB, NWO Genomics), The Netherlands Organization for Health Research and Development (ZonMw) 6130.0031, NWO/ZonMw Rubicon grant 452182321 (R.A.J.S. ), National Health and Medical Research Council Leadership Investigator GNT 2025349 (E.Hy. ), the Medical Research Council Doctoral Training Programme in Precision Medicine (MR/N013166/1) (P.R.H.J.T. ), 352792 (J.K.), 356405, 286284, 134309, 77299, Center of Excellence in Complex Disease Genetics (312062) (Sa.R.). ), grant for the Swedish Infrastructure for Medical Population-based Life-course Environmental Research – SIMPLER, Linnaeus grant Dnr 349-2006-237, Strategic Research Area Exodiab Dnr 2009-1039, the Swedish Heart-Lung Foundation (2022-0344 and 2023-03332) (J.G.S.). ), RACE V (CVON 2014–9), RED-CVD (CVON2017-11), from Top Sector Life Sciences & Health to the Dutch Heart Foundation (PPP Allowance; CVON-AI (2018B017)), Dutch Heart Foundation and ZonMW 01-002-2022-0118 EmbRACE. ), RG/17/1/32663, SP/04/002, SP/16/4/32697 (C.P.N), the British Heart Foundation Centre for Research Excellence (H.W.). German Research Council (DFG): EI 969/2-3, ER 155/6-1;6-2, HO 3314/2-1;2-2;2-3;4-3, INST 58219/32-1, JO 170/8-1, KN 885/3-1, PE 2309/2-1, SI 236/8-1;9-1;10-1. The American Diabetes Association (ADA): Jr Faculty Development 11-22-JDFPM-06 (C.N.S. The US Department of Veterans Affairs: Baltimore Geriatrics Research, Education, and Clinical Center, Merit Grant BX004821, BX003340, BX003362 and BX005831, I01-BX004821 (MVP 001, P.W.F.W. Additional funding came from the following sources. ‘Indremedisinsk Forskningsfond' at Bærum Hospital; 23andMe; a Royal Society URF (J.F.W. ); Abbott; Accare Centre for Child and Adolescent Psychiatry; ADAS software (S.N. ); Amsterdam Public Health (APH); Arthritis Research UK; Astellas Pharma, Inc.; AstraZeneca; Avera Institute for Human Genetics; Bayer Pharma AG; Beijing Municipal Center for Disease Prevention and Control; Beijing Municipal of Health Reform and Development Project no. 2019-4 (Beijing Eye Study); Biobanking and Biomolecular Resources Research Infrastructure (BBMRI-NL), CP 32; Biosense Webster (S.N. ); Boston University Chobanian & Avedisian School of Medicine; Bristol Myers Squibb; Budgetary funds from the National Institute for Health and Welfare (THL) of Finland; Business Finland through the Personalized Diagnostics and Care program coordinated by SalWe Ltd. (3986/31/2013); California Cancer Research Fund 99-00527V-10182; Cancer Australia; Cancer Australia PdCCRS (J.B); Cancer Centre at Barts and QMUL (H.R.W. ); Cancer Council Queensland (J.B); Cancer Council Tasmania; Cancer Councils of Victoria and South Australia; Chief Scientist Office of the Scottish Government Health Directorates (CZD/16/6); Chinese National Human Genome Center at Shanghai; Competitive State Research Financing of the Expert Responsibility area of Kuopio, Tampere and Turku University Hospitals (X51001); Concerted Research Action of the KU Leuven (GOA/15/017) (F.C. ); Daiichi Sankyo Co., Ltd.; Danish Ministry of Internal Affairs and Health; Dell Loy Hansen Heart Foundation (M.J.Cu. ); Department of Science and Technology, Government of India through PURSE II CDST/SR/PURSE PHASE II/11 provided to Jawaharlal Nehru University; Departments, Division and the Board of Directors of the Leiden University Medical Centre; Dutch Ministry of Economic Affairs, Agriculture and Innovation (project KB-15-004-003); Edith Cowan University; EFPIA companies; EHRA; Erasmus MC, Rotterdam; Estonian Research Council PUT 1371 (Kr.L. 2015-1651); Fondo Sectorial de Investigación en Salud y Seguridad Social (SSA/IMSS/ISSSTE CONACYT, project 150352) in Mexico; Foundation for the National Institutes of Health through the Accelerating Medicines Partnership (no. ); GlaxoSmithKline; governmental funding of clinical research within the Swedish National Health Service (J.G.S. ); Greek Secretariat for Research and Technology (Cooperation 890/2009); Health Care Centers in Vasa, Närpes and Korsholm; Health~Holland PPP Allowance ‘Getting the Perfect Image' (S.W.v.d.L. ); Heinz Nixdorf Foundation; Helmholtz Zentrum München–German Research Center for Environmental Health, which is funded by the German Federal Ministry of Education and Research (BMBF) and by the State of Bavaria; Hjartavernd (the Icelandic Heart Association); Hong Kong Kadoorie Charitable Foundation; ImriCor (S.N. ); Institut National du Cancer (INCa); Instituto de Salud Carlos III: CPII22/00007 (M.S.-L.), FEDER (PI08/1770, PI09/00773-Cantabria, PI11/01889-FEDER, PI12/00265, PI12/01270, PI12/00715, PI15/00069, RD09/0076/00036); Italian Ministry of Health and the US National Institute on Aging (263 MD 9164 and 263 MD 821336); Italian Ministry of Health through RC 01/21 (M.P.C.). ; Juvenile Diabetes Research Foundation International (JDRF) (BEAT-DKD); Komen Foundation for the Cure; Korea Healthcare Technology R&D Project, Ministry of Health and Welfare, Republic of Korea (A102065); Korea National Institute of Health intramural grant 2019-NG-053-02; Kuwait Foundation for the Advancements of Sciences (KFAS) (T.A.T. and F.A.-M.); Leiden University Medical Center; Leo Laboratories; Ligue départementale du Val de Marne; Ligue Nationale Contre le Cancer; Medical University of Graz (ASPS); Merck Sharp & Dohme Corp.; MEXT (16H05250, 15K19242, 16H06277, 19K19434, 20K10514, 21H03206, 22H04923, 22H03350, 23K24608, 24K02690); Mitsubishi Tanabe Pharma Corporation; Murdoch University; National Institute of Health, Republic of Korea (2022-NI-067-02); National Institute of Health, Republic of Korea (4845–301, 4851–302, 4851–307); National Key Research and Development Program of China (2016YFC 0900500, 0900501, 0900504, 1303904); National Research Foundation of Korea (NRF): RS-2023-00245960 (Y.S.C. ); NCHA (Netherlands Consortium Healthy Ageing) Leiden/Rotterdam; Netherlands Twin Registry Repository and the Biobanking and Biomolecular Resources Research Infrastructure (BBMRI–NL, 184.021.007 and 184.033.111); Neuroscience Campus Amsterdam (NCA); NextGenerationEU Project code CN_00000033, CUP B83C22002930006; Nordic Center of Excellence in Health-Related e-Sciences (Xueping.L. ); Novartis Institute for Biomedical Research; Novo Nordisk, Inc.; NZO (Dutch Dairy Association); Oak Foundation fellowship (B.F.); Otsuka Pharmaceutical Co., Ltd.; Pfizer; philanthropic donation to Northshore University Health System; Pla Director d'Oncologia de Catalunya (XBTC); Prostate Cancer Canada Movember Discovery Grant (D2013-17) (RJH); Prostate Research Campaign UK (now PCUK/Prostate Action); Queen's University Belfast; Region of Southern Denmark; Research Foundation of Copenhagen County; Research Project for Excellence IKY/SIEMENS; Rosetrees Trust; Royal Marsden Foundation NHS Trust; Safe Work Australia, WA Health, Government of Western Australia (WADOH) (Future Health WA G06302); Sanofi; Sardinian Autonomous Region (L.R. 7/2009) cRP3-154; Servier Research Group; Shanghai Municipal Science and Technology Major Project (2017SHZDZX01); Siemens Healthineers; Social Sciences Council medium-sized investment grants GB-MaGW 480-01-006 and GB-MaGW 480-07-001; Social Sciences Council project grants GB-MaGW 452-04-314 and GB-MaGW 452-06-004; Special Governmental Grants for Health Sciences Research, Turku University Hospital; Stanley Medical Research Institute; Taisho Pharmaceutical Co., Ltd.; Takeda Medical Research Foundation; Takeda Pharmaceutical Co., Ltd.; Temas Prioritarios de Salud Instituto Mexicano del Seguro Social (2014-FIS/IMSS/PROT/PRIO/14/34); the Aarhus and Copenhagen universities and university hospitals; the Accelerating Clinical Trials funding stream in Canada (P.Ki. Sundell Foundation; the Canada Research Chair Program (G.L. ); the Wellcome Trust/Wolfson Foundation; the Yrjö Jahnsson Foundation; TrygFonden (7-11-0213); University of Helsinki HiLIFE Fellow and Grand Challenge grant (Sa.R. ); the University of Michigan; the University of Newcastle; Vestre Viken Hospital Trust; VIAgenomics (SP/19/2/344612); VicHealth and Cancer Council Victoria; Vincent Fairfax Family Foundation; VU University Medical Center, Amsterdam; VU University's Institutes for Health and Care Research (EMGO+); Wageningen University, Wageningen; Wake Forest School of Medicine grant M01 RR07122 and Venture Fund; Wellcome Trust Sanger Institute; and the Women and Infants Research Foundation. Present address: Department of Mathematics and Statistics, St. Full lists of members and their affiliations appear in the Supplementary Information. The Charles Bronfman Institute for Personalized Medicine, Icahn School of Medicine at Mount Sinai, New York, NY, USA Roelof A. J. Smit, Michael H. Preuss, Ghislain Rocheleau, Marta Guindo-Martínez, Daiane Hemerich, Zhe Wang & Ruth J. F. Loos Roelof A. J. Smit, Raymond Noordam, Dennis O. Mook-Kanamori & Frits R. Rosendaal Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark Roelof A. J. Smit, Malene R. Christiansen, Marta Guindo-Martínez, Mette K. Andersen, Jette Bork-Jensen, Kristoffer S. Burgdorf, Anette P. Gjesing, Anna Jonsson, Niels Grarup, Torben Hansen, Oluf Pedersen, Tuomas O. Kilpeläinen & Ruth J. F. Loos Department of Environmental Medicine, Icahn School of Medicine at Mount Sinai, New York, NY, USA Kaitlin H. Wade, Simon J. Haworth, Ruth E. Mitchell & Nicholas J. Timpson Atlanta VA Health Care System, Decatur, GA, USA Joshua D. Arias, Moara Machado, Shengchao A. Li, Stephen J. Chanock, Wen-Yi Huang & Sonja I. Berndt Xianyong Yin, Lars G. Fritsche, Anne U. Jackson, Anita Pandit, Laura J. Scott, Goncalo R. Abecasis & Michael Boehnke Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Center for Genomic Data Analytics, Icahn School of Medicine at Mount Sinai, New York, NY, USA Sarah E. Graham, Whitney E. Hornsby, Tori L. Melendez & Cristen J. Willer Department of Epidemiology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Department of Population Health Sciences, Geisinger, Danville, PA, USA Marielisa Graff, Heather M. Highland, Daeeun Kim, Rebecca Rohde, Anne E. Justice, Kari E. North & Kristin L. Young Eirini Marouli, Olga Giannakopoulou, Stavroula Kanoni, Ioanna Ntalla, Julia Ramirez, Helen R. Warren, Mark J. Caulfield, Patricia B. Munroe & Panos Deloukas Saori Sakaue, Masato Akiyama, Masahiro Kanai & Yoichiro Kamatani Saori Sakaue, Sailaja Vedantam, Eric Bartell, Anders U. Eliasen, George Hindy, Masahiro Kanai, Richa Saxena, Wei Zhou, Samuli Ripatti & Cecilia M. Lindgren Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Cassandra N. Spracklen, Emma P. Wilson, Daeeun Kim, Laura M. Raffield & Karen L. Mohlke Teresa Ferreira, Deborah E. Malden, Zammy Fairhurst-Hunter, Kuang Lin, Jun Liu, Iona Y. Millwood, Zhengming Chen, Cornelia M. van Duijn, Cecilia M. Lindgren, Peter M. Visscher & Robin G. Walters Tugce Karaderi, Anuj Goel, Anubha Mahajan, Mark I. McCarthy, Hugh Watkins & Cecilia M. Lindgren Carolina Medina-Gomez, Fernando Rivadeneira, Nathalie Van der Velde & M. Carola Zillikens Division of Biostatistics and Epidemiology, RTI International, Durham, NC, USA Thurston Arthritis Research Center, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Eric Bartell, Brian E. Cade, Saiju Pyarajan, Julie E. Buring, Paul L. Huang, Amit V. Khera, JoAnn E. Manson, Susan Redline, Paul M. Ridker, Daniel I. Chasman & Christopher J. O'Donnell Genomic Research on Complex Diseases (GRC-Group), CSIR-Centre for Cellular and Molecular Biology, Hyderabad, India Lawrence F. Bielak, Jennifer A. Smith, Wei Zhao, Sharon L. R. Kardia & Patricia A. Peyser Joshua C. Bis, Jennifer A. Brody, Rozenn N. Lemaitre & Bruce M. Psaty Sailalitha Bollepalli, Aki S. Havulinna, Sanni E. Ruotsalainen, Jaakko Kaprio, Samuli Ripatti, Tiinamaija Tuomi & Elisabeth Widen Yuki Bradford, Xinyuan Zhang, Scott M. Damrauer, Struan F. A. Grant, Daniel J. Rader, Marylyn D. Ritchie & Babette S. Zemel Caroline Brandl, Alexander Dietl, Mathias Gorski, Martina E. Zimmermann, Iris M. Heid, Klaus J. Stark & Thomas W. Winkler Department of Congenital Disorders and Danish National Biobank, Statens Serum Institut, Copenhagen, Denmark Department of Medicine, Brigham and Women's Hospital, Boston, MA, USA Brian E. Cade, Saiju Pyarajan, JoAnn E. Manson, Susan Redline & Christopher J. O'Donnell Qiuyin Cai, Jirong Long, Wanqing Wen, Xiao-Ou Shu & Wei Zheng Silvia Camarda, Giuseppe Giovanni Nardone, Francesco Piluso, Aurora Santin, Paola Tesolin, Giorgia Girotto & Beatrice Spedicati Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK Centre for Public Health, Queen's University of Belfast, Northern Ireland, UK Marisa Cañadas-Garre, Laura J. Smyth, Frank Kee & Amy Jayne McKnight Jin-Fang Chai, Xueling Sim, E. Shyong Tai & Rob M. van Dam Center for Spatial and Functional Genomics, Division of Human Genetics, Children's Hospital of Philadelphia, Philadelphia, PA, USA Cardiovascular Disease Initiative, Broad Institute of MIT and Harvard, Cambridge, MA, USA Seung Hoan Choi, Patrick T. Ellinor & Steven A. Lubitz Gabriel Cuellar-Partida, Yunxuan Jiang, Jingchunzi Shi & Adam Auton Division of Preventive Medicine, Brigham and Women's Hospital, Boston, MA, USA Rebecca Danning, Franco Giulianini, Julie E. Buring, JoAnn E. Manson, Paul M. Ridker & Daniel I. Chasman Ayşe Demirkan, Maria J. Knol, Jun Liu, Natalie Terzikhan, M. Arfan Ikram, Maryam Kavousi & Cornelia M. van Duijn Department of Nutrition and Dietetics, School of Health Science and Education, Harokopio University, Athens, Greece Maria Dimitriou, Aliki-Eleni Farmaki, Ioanna P. Kalafati & George V. Dedoussis Rajkumar Dorajoo, Chiea Chuen Khor & Jianjun Liu German Center for Diabetes Research (DZD), Neuherberg, Germany Fabian Eichelmann, Christian Gieger, Harald Grallert, Annette Peters & Matthias B. Schulze Molecular Genetics Section, Center for Precision Health Research, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA Jessica D. Faul, Jennifer A. Smith, Wei Zhao & David R. Weir Rogel Cancer Center, University of Michigan Medicine, Ann Arbor, MI, USA Eurac Research Institute for Biomedicine, Bolzano, Italy Department for Health Evidence, Radboud University Medical Center, Nijmegen, The Netherlands Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark Genetic Epidemiology, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark The Institute for Translational Genomics and Population Sciences, Department of Pediatrics, Lundquist Institute for Biomedical Innovation at Harbor-UCLA Medical Center, Torrance, CA, USA Xiuqing Guo, Marie Lauzon, Xiaohui Li, Kent D. Taylor, Jie Yao, Yii-Der Ida Chen & Jerome I. Rotter Division of Public Health Sciences, Fred Hutchinson Cancer Center, Seattle, WA, USA Aki S. Havulinna, Pekka Jousilahti, Heikki A. Koistinen, Veikko Salomaa & Jaakko Tuomilehto Elizabeth Holliday, Carol A. Wang, John Attia & Craig E. Pennell Jouke-Jan Hottenga, Eco J. C. de Geus & Gonneke Willemsen Chan School of Public Health, Boston, MA, USA Hongyan Huang, Constance Turman, A. Heather Eliassen, Peter Kraft, Lorelei A. Mucci, Meir J. Stampfer & Walter C. Willett School of Public Health and Emergency Management, Southern University of Science and Technology, Shenzhen, Guangdong, China Institute for Global Health and Development, Peking University, Beijing, China Center for Genomic Medicine and Diabetes Unit, Endocrine Division, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA Shaofeng Huo, Liang Sun, Huaixing Li & Xu Lin Research Institute, National Center for Global Health and Medicine, Tokyo, Japan Department of Child and Adolescent Psychiatry and Pediatric Psychology, Section Complex Trait Genetics, Amsterdam Neuroscience, Vrije Universiteit Medical Center, Amsterdam, The Netherlands Anuradhani Kasturiratne, Ananda R. Wickremasinghe & H. Janaka de Silva Corporal Michael J. Crescenz VA Medical Center, Philadelphia, PA, USA Katherine A. Kentistou, Paul R. H. J. Timmers, Peter K. Joshi, Igor Rudan & James F. Wilson Department of Clinical Research Design and Evaluation, Samsung Advanced Institute for Health Sciences and Technology, Sungkyunkwan University, Seoul, Republic of Korea Center for Clinical Epidemiology, Research Institute for Future Medicine, Samsung Medical Center, Seoul, Republic of Korea Department of Clinical Sciences, Genetic and Molecular Epidemiology Unit, Lund University, Malmö, Sweden Genomics Research Centre, Centre for Genomics and Personalised Health, School of Biomedical Sciences, Qld University of Technology, Kelvin Grove, Queensland, Australia Oneomics, Soonchunhyang Mirai Medical Center, Gyeonggi-do, Republic of Korea Center for Alzheimer's and Related Dementias, National Institutes of Health, Bethesda, MD, USA Data Tecnica International, Glen Echo, MD, USA Department of Population and Quantitative Health Sciences, Case Western Reserve University, Cleveland, OH, USA Ken Sin Lo, Jean-Claude Tardif & Guillaume Lettre Laura Lores-Motta, Marc Pauper, Anneke I. den Hollander & Carel B. Hoyng Department of Clinical Sciences, Lund University Diabetes Centre, Malmö, Sweden Department of Clinical Chemistry, Fimlab Laboratories, Tampere, Finland Leo-Pekka Lyytikäinen, Pashupati P. Mishra, Nina Mononen & Terho Lehtimäki Md Zubbair Malik, Fahd Al-Mulla & Thangavel A. Thanaraj NIHR Biomedical Research Centre at Guy's and St. Thomas' Foundation Trust, London, UK MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Western General Hospital, Edinburgh, Scotland Jonathan Marten, Paul R. H. J. Timmers, Caroline Hayward, Veronique Vitart, James F. Wilson & Alan F. Wright Biomedical and Translational Informatics Institute, Geisinger, Danville, PA, USA Juha Mykkänen, Harri Niinikoski, Katja Pahkala & Olli T. Raitakari Juha Mykkänen, Harri Niinikoski, Katja Pahkala & Olli T. Raitakari Public Health Informatics Unit, Department of Integrated Health Sciences, Nagoya University Graduate School of Medicine, Nagoya, Japan Department of Clinical Biochemistry, Lillebaelt Hospital, Kolding, Denmark Ilja M. Nolte, Peter J. van der Most & Harold Snieder Department of Human Genetics, Wellcome Sanger Institute, Hinxton, UK Institute of Genetics and Biophysics A. Buzzati-Traverso, CNR, Naples, Italy Department of Clinical Biochemistry, Lillebaelt Hospital, Vejle, Denmark Institute of Translational Genomics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany Program for Personalized and Genomic Medicine, University of Maryland School of Medicine, Baltimore, MD, USA Unit of Genomics of Complex Diseases, Institut de Recerca Sant Pau (IR Sant Pau), Barcelona, Spain Centre for Biomedical Network Research on Rare Diseases (CIBERER), Instituto de Salud Carlos III, Madrid, Spain Institute for Maternal and Child Health, IRCCS Burlo Garofolo, Trieste, Italy Aurora Santin, Paola Tesolin, Maria Pina Concas, Giorgia Girotto & Beatrice Spedicati Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA Laboratory of Epidemiology and Population Sciences, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA Botong Shen, Michele K. Evans, Lenore J. Launer & Alan B. Zonderman Istituto di Ricerca Genetica e Biomedica, Consiglio Nazionale delle Ricerche (CNR), Cagliari, Italy Carlo Sidore, Simona Vaccargiu, Magdalena Zoledziewska & Francesco Cucca Roderick C. Slieker, Leen M. ‘t Hart & Femke Rutters Valgerdur Steinthorsdottir, Gudmar Thorleifsson, Kari Stefansson & Unnur Thorsteinsdottir Department of Internal Medicine, Section of Gerontology and Geriatrics, Leiden University Medical Center, Leiden, The Netherlands Niek Verweij, Michiel Rienstra & Pim van der Harst Division of Endocrinology and Metabolism, Department of Internal Medicine, Taichung Veterans General Hospital, Taichung, Taiwan Department of Epidemiology and Biostatistics, MRC-PHE Centre for Environment and Health, School of Public Health, Imperial College London, London, UK Unit for Lifelong Health and Ageing at UCL, Institute of Cardiovascular Science, University College London, London, UK Institute of Molecular Medicine, McGovern Medical School, The University of Texas Health Science Center at Houston, Houston, TX, USA Department of Neurological Sciences, Rush University Medical Center, Chicago, IL, USA Hannah Young, William G. Iacono, Matt McGue & Scott Vrieze Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Carolina Population Center, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Department of Human Genetics, Radboud University Medical Center, Nijmegen, The Netherlands Latin American Brain Health (BrainLat), Universidad Adolfo Ibáñez, Santiago, Chile Bjørn Olav Åsvold, Ben Brumpton & Kristian Hveem HUNT Research Center, Department of Public Health and Nursing, Norwegian University of Science and Technology, Levanger, Norway Geriatric Unit, Azienda Toscana Centro, Florence, Italy Systems Genomics Laboratory, School of Biotechnology, Jawaharlal Nehru University (JNU), New Delhi, India Human Genetics Center and Department of Epidemiology, The University of Texas Health Science Center at Houston, Houston, TX, USA Department of Nephrology and Rheumatology, Kliniken Südostbayern, Regensburg, Germany Human Genomics Laboratory, Pennington Biomedical Research Center, Baton Rouge, LA, USA Department of Cardiology, Ealing Hospital, London North West University Healthcare NHS Trust, Middlesex, UK Unidad de Investigación Médica en Bioquímica, Hospital de Especialidades, Centro Médico Nacional Siglo XXI, Instituto Mexicano del Seguro Social, Mexico City, Mexico Center for Translational and Computational Neuroimmunology, Department of Neurology, Columbia University Medical Center, New York, NY, USA Channing Division of Network Medicine, Department of Medicine, Brigham and Women's Hospital, Boston, MA, USA Chan School of Public Health, Boston, MA, USA A. Heather Eliassen, Paul W. Franks, Meir J. Stampfer & Walter C. Willett Cardiac Arrhythmia Service, Massachusetts General Hospital, Boston, MA, USA Cardiovascular Research Center, Massachusetts General Hospital, Boston, MA, USA Patrick T. Ellinor, Paul L. Huang, Steven A. Lubitz & Christopher Newton-Cheh Molecular Cardiology Division, Victor Chang Cardiac Research Institute, Darlinghurst, New South Wales, Australia Department of Cardiology, St. Vincent's Hospital, Darlinghurst, New South Wales, Australia Faculty of Medicine, UNSW Sydney, Kensington, New South Wales, Australia Department of Medicine, Harvard Medical School, Boston, MA, USA Diabetes Unit, Massachusetts General Hospital, Boston, MA, USA Human Genetics Center, School of Public Health, The University of Texas Health Science Center at Houston, Houston, TX, USA Department of Clinical Sciences, Genetic and Molecular Epidemiology Unit, Lund University, Helsingborg, Sweden Section on Nephrology, Department of Internal Medicine, Wake Forest School of Medicine, Winston-Salem, NC, USA Research Unit of Molecular Epidemiology, Institute of Epidemiology, Helmholtz Zentrum München Research Center for Environmental Health, Neuherberg, Germany Khoo Teck Puat - National University Children's Medical Institute, National University Health System, Singapore, Singapore Centre for Eye Research Australia, Royal Victorian Eye and Ear Hospital, University of Melbourne, Melbourne, Victoria, Australia Division of Cardiology, Massachusetts General Hospital, Boston, MA, USA Department of Genetics, Shanghai-MOST Key Laboratory of Heath and Disease Genomics, Chinese National Human Genome Center and Shanghai Industrial Technology Institute, Shanghai, China Department of Life Sciences, College of Health and Life Sciences, Brunel University London, Uxbridge, UK Institut Français de Myopie, Hôpital Fondation Rothschild, Paris, France L V Prasad Eye Institute, Hyderabad, India Einthoven Laboratory for Experimental Vascular Medicine, LUMC, Leiden, The Netherlands Brigham and Women's Hospital, Boston, MA, USA Department of Urology, Radboud University Medical Center, Nijmegen, The Netherlands Saevit Eye Hospital, Goyang-si, Republic of Korea German Center for Cardiovascular Research, Partner Site, Hamburg/Kiel/Lübeck, Germany Minerva Foundation Institute for Medical Research, Helsinki, Finland First Department of Propaedeutic Internal Medicine, Medical School, National and Kapodistrian University of Athens, Laiko General Hospital, Athens, Greece Institute for Social and Economic Research, University of Essex, Wivenhoe Park, UK Institute of Clinical Medicine, Internal Medicine, University of Eastern Finland and Kuopio University Hospital, Kuopio, Finland Department of Epidemiology and Biostatistics, Peking University Health Science Center, Beijing, China Peking University Center for Public Health and Epidemic Preparedness & Response, Beijing, China Division of Cardiovascular Medicine and Abboud Cardiovascular Research Center, University of Iowa Hospitals and Clinics, Iowa City, IA, USA Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden Clinical Institute of Medical and Chemical Laboratory Diagnostics, Medical University of Graz, Graz, Austria Psychiatric Genetics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Department of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Department of Drug Treatment, Sahlgrenska University Hospital, Gothenburg, Sweden Laboratorio de Inmunogenómica y Enfermedades Metabólicas, Instituto Nacional de Medicina Genómica, CDMX, Mexico City, Mexico Paavo Nurmi Centre and Unit for Health and Physical Activity, University of Turku, Turku, Finland Institute for Precision Health, David Geffen School of Medicine at UCLA, Los Angeles, CA, USA Pat MacPherson Centre for Pharmacogenetics and Pharmacogenomics, Division of Population Health and Genomics, School of Medicine, University of Dundee, Ninewells Hospital and Medical School, Dundee, UK Centre Nutrition, santé et société (NUTRISS), Institute of Nutrition and Functional Foods, Université Laval, Québec, QC, Canada Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany Institute for Medical Information Processing, Biometry and Epidemiology, Medical Faculty, Ludwig Maximilians Universität München, Munich, Germany Population, Policy and Practice, UCL Great Ormond Street Hospital Institute of Child Health, London, UK Second Department of Cardiology, Medical School, National and Kapodistrian University of Athens, University General Hospital Attikon, Athens, Greece Center for Biostatistics and Data Science, Washington University School of Medicine, St. Louis, MO, USA Genetics, Merck Sharp & Dohme Corp., Kenilworth, NJ, USA Departments of Medicine, Pharmacology, and Biomedical Informatics, Vanderbilt University Medical Center, Nashville, TN, USA Center for Cohort Studies, Total Healthcare Center, Kangbuk Samsung Hospital, Sungkyunkwan University School of Medicine, Seoul, Republic of Korea Department of Pharmaceutical Sciences, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA Gottfried Schatz Research Center (for Cell Signaling, Metabolism and Aging), Medical University of Graz, Graz, Austria Institute of Molecular and Genomic Medicine, National Health Research Institutes, Zhunan, Taiwan Department of Medicine, Vanderbilt University Medical Center, Nashville, TN, USA Diabetes Research Group, King Abdulaziz University, Jeddah, Saudi Arabia Unidad de Biología Molecular y Medicina Genómica, Instituto de Investigaciones Biomédicas, UNAM, Mexico City, Mexico Unidad de Investigación Médica en Epidemiología Clínica, Hospital de Especialidades, Centro Médico Nacional Siglo XXI, Instituto Mexicano del Seguro Social, Mexico City, Mexico Hero DMC Heart Institute, Dayanand Medical College, Ludhiana, India USC Diabetes and Obesity Research Institute, Keck School of Medicine of USC, Los Angeles, CA, USA Diabetes Unit, KEM Hospital and Research Centre, Pune, India School of Life Sciences, Westlake University, Hangzhou, China Westlake Laboratory of Life Sciences and Biomedicine, Hangzhou, China Cancer Epidemiology and Prevention Program, UPMC Hillman Cancer Center, University of Pittsburgh, Pittsburgh, PA, USA Technical University of Munich and Klinikum Rechts der Isar, TUM School of Medicine and Health, Munich, Germany Institute for Research and Medical Consultations, Imam Abdulrahman Bin Faisal University, Dammam, Saudi Arabia McDonnell Genome Institute and Department of Medicine, Washington University School of Medicine, St. Louis, MO, USA Departments of Pediatrics and Genetics, Harvard Medical School, Boston, MA, USA The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA All other authors were involved in the design, management, coordination or analysis of contributing studies. Consortium representatives: publication-review@23andme.com (23andMe Research Team); aejustice1@geisinger.edu (DiscovEHR); marylyn@pennmedicine.upenn.edu (eMERGE); segun.fatumo@lshtm.ac.uk (GPC-UGR); practical@icr.ac.uk (PRACTICAL); mkumari@essex.ac.uk (Understanding Society Scientific Group); yan.v.sun@emory.edu (VA Million Veteran Program). (since June 2019) and A.Mah (since January 2020) are employees of Genentech and holders of Roche stock. has received Roche funding for unrelated work. is Chief Scientist for Genomics England, a UK Government company. has received research awards from Pfizer, Inc. D.H. is currently employed at Bristol Myers Squibb. Since completing the work contributed to this paper, D.J.T. has left the University of Cambridge and is now employed by Genomics plc. is an employee and stock owner of Gilead Sciences; this work was conducted before employment by Gilead Sciences. is listed as inventor on two issued patents held by the University of Hamburg (Atrial Fibrillation Therapy WO 2015140571 and Markers for Atrial Fibrillation WO 2016012783). received consultancy fees from Bayer (OCEANIC-AF national PI) and InCarda Therapeutics (RESTORE-SR national PI) to the institution, unrelated to this research/dataset. has served on advisory boards or consulted for Boston Scientific, Biosense Webster, Janssen Scientific Affairs and Johnson & Johnson. has acted as a member of advisory boards and as a speaker for Eli Lilly and the Novo Nordisk Foundation, for which she has received fees. is a scientific advisor to Circle software, ADAS software, CardioSolv and ImriCor and receives grant support from Biosense Webster, ADAS software and ImriCor. received sponsored research support from Bristol Myers Squibb, Pfizer, Boehringer Ingelheim, Fitbit, Medtronic, Premier and IBM and has consulted for Bristol Myers Squibb, Pfizer, Blackstone Life Sciences and Invitae. receives sponsored research support from Bayer AG, Bristol Myers Squibb, Pfizer and Novo Nordisk; he has also served on advisory boards or consulted for Bayer AG. has received honoraria for consulting from AstraZeneca, MSD/Merck, Daiichi, Servier, Amgen and Takeda Pharma. He has additionally received honoraria for lectures and/or chairs from AstraZeneca, BayerVital, BRAHMS, Daiichi, Medtronic, Novartis, Sanofi and Servier. serves on the Steering Committee of the Yale Open Data Access Project funded by Johnson & Johnson. declares having received research grants through his affiliation from Novo Nordisk and Pharmaserve Lilly and consulting honoraria from Pharmaserve Lilly, Sanofi-Aventis, Novo Nordisk, MSD, AstraZeneca, ELPEN Pharma, Boehringer Ingelheim, Galenica Pharma, Epsilon Health and Winmedica. is currently employed by and holds stock in Regeneron Pharmaceuticals, Inc. G.R.A. is an employee of Regeneron Pharmaceuticals and owns stock and stock options in Regeneron Pharmaceuticals. is currently employed by and holds stock in Regeneron Pharmaceuticals. is an employee and stock owner of UCB Pharma; this work was conducted before employment by UCB. is employed by SYNLAB Holding Deutschland GmbH. reports grants and personal fees from AMGEN, BASF, Sanofi, Siemens Diagnostics, Aegerion Pharmaceuticals, AstraZeneca, Danone Research, Numares, Pfizer and Hoffmann-La Roche; personal fees from MSD and Alexion; and grants from Abbott Diagnostics, all outside the submitted work. is employed by Synlab Holding Deutschland GmbH. is a current employee of Novartis Biomedical Research. is currently employed by and holds stock in Regeneron Pharmaceuticals. received speaking honoraria from AstraZeneca, Merck and Novo Nordisk for talks over which he had full control of content and a consulting honorarium from AstraZeneca. receives support from a consulting contract between Data Tecnica International and the National Institute on Aging (NIA), National Institutes of Health (NIH). receives support from a consulting contract between Data Tecnica and the NIA, NIH. is a co-founder and shareholder of ZOE, Ltd. P.R.H.J.T. is a salaried employee of BioAge Labs, Inc. A.I.d.H. The remaining authors declare no competing interests. Nature Medicine thanks Adam Gordon, Iftikar Kullo and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Anna Maria Ranzoni and Sonia Muliyil, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Asian-specific BMI cutoffs (see Methods) were applied to East Asian (EAS), South Asian (SAS), and non-Hispanic Asian (AS) populations, in contrast to African (AFR), American (AMR), European (EUR), and Middle Eastern (MID) populations. Sample sizes (distinct individuals), from left to right, for Tuning: 6154, 971, 1553, 2660, 8005, 20000; Validation: 12263, 2332, 18701, 10281, 8096, 4201, 1359, 1177, 13673, 69828, 340224; and BMI: 12263, 10281, 1359, 1177, 13673, 2332, 18701, 8096, 4201, 69828, 6154, 971, 1553, 2660, 8005, 340224. Prevalence of obesity category, with 95% confidence intervals based on the normal approximation to the binomial distribution, across 1%-groups of PGSKhera and PGSLC within the validation subset of the UKBB participants of European ancestry (N ~ 340k). All PGS were standardised using the mean and SD of the PGS of individuals who did not have the outcome of interest (that is, those with a BMI value below the threshold for a given obesity category), to account for differences in prevalence across validation populations. MVP, Million Veteran Program; GPC-UGR, Uganda General Population Cohort; UKB, UK Biobank. Sample sizes (distinct individuals) per grouped set of estimates, left to right: AFR 12263, 2332, 18701; AMR 10281, 8096; AS 4201; EAS 1359; SAS 1177; EUR 13673, 69828, 340224. Distributional shifts of ancestry-specific scores which are linearly combined to create PGSLC, across UK Biobank-tuning and BioMe-validation populations, before (top) and after (bottom) ancestry–calibration using 1000 Genomes reference data (Methods). Each plot shows a single ancestry-specific score, with separate lines showing the distribution across ancestries. For these plots, each score was standardised to mean zero and variance one once across all participants of a given study. Estimated using linear spline multilevel models with repeated measures from the ALSPAC cohort. Estimated in the Prostate, Lung, Colorectal and Ovarian (PLCO) Cancer Screening Trial. Scores were standardised to mean zero and variance one by population group, ignoring assigned trial arm. Vertical lines indicate the mean PGS per arm. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Smit, R.A.J., Wade, K.H., Hui, Q. et al. Polygenic prediction of body mass index and obesity through the life course and across ancestries. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250720034018.htm'>The heatwave that shattered ecosystems, starved whales, and drove fish north</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 06:46:51
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research from the University of Victoria (UVic) highlights how marine heatwaves can dramatically impact marine ecosystems and offers a stark preview of how future ocean warming will reshape ocean life. From 2014 to 2016, the Pacific coast of North America experienced the longest marine heatwave ever recorded, with temperatures reaching two to six degrees above historical averages over a prolonged period. Researchers from UVic's Baum Lab have compiled a comprehensive overview of the heatwave's ecological impacts, reviewing the findings from 331 primary studies and governmental reports. "The marine heatwave resulted in unprecedented ecological disturbance across thousands of kilometers of North America's west coast," says Samuel Starko, lead author and former UVic postdoctoral fellow. According to the research, 240 different species were found outside of their typical geographic range during the heatwave, with many of them found further north than ever before. Several species, such as the northern right whale dolphin and the sea slug Placida cremoniana, were found over 1,000 kilometers north of their typical habitat. A key rocky shore predator, Pycnopodia helianthoides, came close to extinction. Many of the impacts of the heatwave were cascading, with direct impacts on some species driving complex dynamics that affected everything from plankton to whales. Temperature-linked diseases, such as sea star wasting disease, contributed to ecosystem collapse. The reduced abundance and nutritional quality of forage fish caused problems for predators. Plankton communities reorganized and offshore oceanographic productivity was altered. "As heatwaves become more frequent and intense under climate change, the 2014-16 Northeast Pacific marine heatwave provides a critical example of how climate change is impacting ocean life, and how our future oceans may look," says Julia Baum, UVic marine ecologist and special advisor, climate. "This study underscores the urgent need for proactive, ecosystem-based marine conservation strategies and climate change mitigation measures." Note: Content may be edited for style and length. Scientists Warn: Popular Sugar Substitute Linked to Brain Cell Damage Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250720034017.htm'>This genetic breakthrough could help thousands of species cheat extinction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-21 06:25:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Gene editing technologies -- such as those used in agriculture and de-extinction projects -- can be repurposed to offer what an international team of scientists is calling a transformative solution for restoring genetic diversity and saving endangered species. In a new Nature Reviews Biodiversity Perspective article published on July 18, the authors explore the promises, challenges and ethical considerations of genome engineering, and propose an approach for its implementation into biodiversity conservation. They argue that gene editing could recover lost genetic diversity in species at risk of extinction using historical samples, such as DNA from museum collections, biobanks and related species. The multidisciplinary team of conservation geneticists and biotechnologists is co-led by Prof Cock van Oosterhout at the University of East Anglia (UEA) and Dr Stephen Turner from Colossal Biosciences, in collaboration with the Colossal Foundation, the Durrell Institute of Conservation and Ecology (University of Kent), Globe Institute (University of Copenhagen), Mauritius Wildlife Foundation (MWF), the Mauritius National Parks and Conservation Service (NPCS), and Durrell Wildlife Conservation Trust. "We're facing the fastest environmental change in Earth's history, and many species have lost the genetic variation needed to adapt and survive," said Prof van Oosterhout. "Gene engineering provides a way to restore that variation, whether it's reintroducing DNA variation that has been lost from immune-system genes that we can retrieve from museum specimens or borrowing climate-tolerance genes from closely related species. Conservation successes such as captive breeding and habitat protection often focus on boosting population numbers but do little to replenish the gene variants lost when a species' numbers crash. Without intervention, species that recovered from a population crash may remain genetically compromised, with reduced resilience to future threats like new diseases or shifting climates. The technology is already common in agriculture: crops resistant to pests and drought cover millions of hectares worldwide. "The same technological advances that allow us to introduce genes of mammoths into the genome of an elephant can be harnessed to rescue species teetering on the brink of extinction," said Dr Beth Shapiro, Chief Science Officer at Colossal Biosciences. "It is our responsibility to reduce the extinction risk faced today by thousands of species." The scientists outline three key applications for gene editing in conservation: They also address the risks, such as off-target genetic modifications and unintentional further reductions in genetic diversity, cautioning that the approaches remain experimental. The authors stress that genetic interventions must complement, not replace, habitat restoration and traditional conservation actions. Scientists Warn: Popular Sugar Substitute Linked to Brain Cell Damage Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            