
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-25</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09202-9'>Rescuing dendritic cell interstitial motility sustains antitumour immunity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 16:12:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The dendritic cell (DC)-initiated and sustained cancer immunity cycle is indispensable for effective endogenous and therapeutically mobilized antitumour T cell responses1,2,3,4,5,6,7,8. This necessitates the continuous migration of antigen-carrying DCs from the tumour microenvironment (TME) to the tumour draining lymph nodes (tdLNs)7,8,9,10,11,12,13. Here, through longitudinal analysis of human and mouse tumours, we observed a progressive decrease in migratory conventional DCs (mig-cDCs) in the tdLNs during tumour progression. This decline compromised tumour-specific T cell priming and subsequent T cell supply to the TME. Using a genome-wide in vivo CRISPR screen, we identified phosphodiesterase 5 (PDE5) and its substrate cyclic guanosine monophosphate (cGMP) as key modulators of DC migration. Advanced tumours disrupted cGMP synthesis in DCs to decrease their motility, while PDE5 perturbation preserved the cGMP pool to restore DC migration. Mechanistically, cGMP enhanced myosin-II activity through Rho-associated factors, extending the paradigm of cGMP-regulated amoeboid migration from Dictyostelium to mammalian immune cells. Pharmacological inhibition of PDE5 using sildenafil restored mig-cDC homing to late-stage tdLNs and sustained antitumour immunity in a DC-dependent manner. Our findings bridge fundamental DC interstitial motility to antitumour immunity, revealing that its disruption in chaotic TME promotes immune evasion, and its enhancement offers a promising direction for DC-centric immunotherapy. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data supporting the findings of this study are available in the Article and its Supplementary Information. RNA-seq data files are available at the Gene Expression Omnibus (GEO) under accession number GSE246849. All other data are available in the Article and its Supplementary Information. Source data are provided with this paper. & Steinman, R. M. Dendritic cells and the control of immunity. 3rd, Dutertre, C. A., Ginhoux, F. & Murphy, K. M. Genetic models of human and mouse dendritic cell development and function. Cancer immunotherapy via dendritic cells. Merad, M., Sathe, P., Helft, J., Miller, J. The dendritic cell lineage: ontogeny and function of dendritic cells and their subsets in the steady state and the inflamed setting. Cabeza-Cabrerizo, M., Cardoso, A., Minutti, C. M., Pereira da Costa, M. & Reis e Sousa, C. Dendritic cells revisited. Wculek, S. K. et al. Dendritic cells in cancer immunology and immunotherapy. Chen, D. S. & Mellman, I. Oncology meets immunology: the cancer-immunity cycle. Pittet, M. J., Di Pilato, M., Garris, C. & Mempel, T. R. Dendritic cells as shepherds of T cell immunity in cancer. Heuze, M. L. et al. Migration of dendritic cells: physical principles, molecular mechanisms, and functional implications. & Forster, R. Dendritic cell migration in health and disease. Alvarez, D., Vollmann, E. H. & von Andrian, U. H. Mechanisms and consequences of dendritic cell migration. de Winde, C. M., Munday, C. & Acton, S. E. Molecular mechanisms of dendritic cell migration in immunity and cancer. Liu, J., Zhang, X., Cheng, Y. & Cao, X. Dendritic cell migration in inflammation and immunity. Forster, R., Davalos-Misslitz, A. C. & Rot, A. CCR7 and its ligands: balancing immunity and tolerance. Weber, M. et al. Interstitial dendritic cell guidance by haptotactic chemokine gradients. Identification of a novel population of Langerin+ dendritic cells. Broz, M. L. et al. Dissecting the tumor myeloid compartment reveals rare activating antigen-presenting cells critical for T cell immunity. Roberts, E. W. et al. Critical role for CD103+/CD141+ dendritic cells bearing CCR7 for tumor antigen trafficking and priming of T cell immunity in melanoma. Expansion and activation of CD103+ dendritic cell progenitors at the tumor site enhances tumor responses to therapeutic PD-L1 and BRAF inhibition. Allan, R. S. et al. Migratory dendritic cells transfer antigen to a lymph node-resident dendritic cell population for efficient CTL priming. Lammermann, T. & Sixt, M. Mechanical modes of ‘amoeboid' cell migration. Paluch, E. K., Aspalter, I. M. & Sixt, M. Focal adhesion-independent cell migration. Moreau, H. D., Piel, M., Voituriez, R. & Lennon-Dumenil, A. M. Integrating physical and molecular insights on immune cell migration. Rapid leukocyte migration by integrin-independent flowing and squeezing. Vicente-Manzanares, M., Ma, X., Adelstein, R. S. & Horwitz, A. R. Non-muscle myosin II takes centre stage in cell adhesion and migration. Renkawitz, J. et al. Adaptive force transmission in amoeboid cell migration. Bergert, M. et al. Force transmission during adhesion-independent migration. Faure-Andre, G. et al. Regulation of dendritic cell migration by CD74, the MHC class II-associated invariant chain. Vargas, P. et al. Innate control of actin nucleation determines two distinct migration behaviours in dendritic cells. de Saint-Vis, B. et al. A novel lysosome-associated membrane glycoprotein, DC-LAMP, induced upon DC maturation, is transiently expressed in MHC class II compartment. A. Cyclic nucleotide phosphodiesterases: molecular regulation to clinical use. Lucas, K. A. et al. Guanylyl cyclases and signaling by cyclic GMP. Francis, S. H., Busch, J. L., Corbin, J. D. & Sibley, D. cGMP-dependent protein kinases and cGMP phosphodiesterases in nitric oxide and cGMP action. van Haastert, P. J. M. et al. Forty-five years of cGMP research in Dictyostelium: understanding the regulation and function of the cGMP pathway for cell movement and chemotaxis. Bosgraaf, L. & van Haastert, P. J. The regulation of myosin II in Dictyostelium. Cell migration and antigen capture are antagonistic processes coupled by myosin II in dendritic cells. Goldberg, J. M., Bosgraaf, L., Van Haastert, P. J. & Smith, J. L. Identification of four candidate cGMP targets in Dictyostelium. & Brunton, L. L. Cyclic nucleotide research–still expanding after half a century. Cruz-Burgos, M. et al. New approaches in oncology for repositioning drugs: the case of PDE5 inhibitor sildenafil. Use of phosphodiesterase 5 inhibitors is associated with lower risk of colorectal cancer in men with benign colorectal neoplasms. Huang, W., Sundquist, J., Sundquist, K. & Ji, J. Phosphodiesterase-5 inhibitors use and risk for mortality and metastases among male patients with colorectal cancer. Sutton, S. S., Magagnoli, J., Cummings, T. H. & Hardin, J. W. The association between phosphodiesterase-5 inhibitors and colorectal cancer in a national cohort of patients. Weed, D. T. et al. Tadalafil reduces myeloid-derived suppressor cells and regulatory T cells and promotes tumor immunity in patients with head and neck squamous cell carcinoma. A. et al. Tadalafil augments tumor specific immunity in patients with head and neck squamous cell carcinoma. Luginbuhl, A. J. et al. Tadalafil enhances immune signatures in response to neoadjuvant nivolumab in resectable head and neck squamous cell carcinoma. Hassel, J. C. et al. Tadalafil has biologic activity in human melanoma. Results of a pilot trial with tadalafil in patients with metastatic melanoma (TaMe). Das, A., Durrant, D., Salloum, F. N., Xi, L. & Kukreja, R. C. PDE5 inhibitors as therapeutics for heart disease, diabetes and cancer. Ohno, M., Gibbons, G. H., Dzau, V. J. & Cooke, J. P. Shear stress elevates endothelial cGMP. Role of a potassium channel and G protein coupling. Rising behind NO: cGMP-dependent protein kinases. Weed, D. T. et al. The reversal of immune exclusion mediated by tadalafil and an anti-tumor vaccine also induces PDL1 upregulation in recurrent head and neck squamous cell carcinoma: interim analysis of a phase i clinical trial. Sporadic adenomatous polyp regression with exisulind is effective but toxic: a randomised, double blind, placebo controlled, dose-response study. Philip, M. et al. Chromatin states define tumour-specific T cell dysfunction and reprogramming. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Hänzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. & Butte, A. J. xCell: digitally portraying the tissue cellular heterogeneity landscape. Li, T. et al. TIMER2.0 for analysis of tumor-infiltrating immune cells. Nucleic Acids Res. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Yu, G., Wang, L. G., Han, Y. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Zilionis, R. et al. Single-cell transcriptomics of human and mouse lung cancers reveals conserved myeloid populations across individuals and species. Cheng, S. et al. A pan-cancer single-cell transcriptional atlas of tumor infiltrating myeloid cells. Pelka, K. et al. Spatially organized multicellular immune hubs in human colorectal cancer. & Marioni, J. C. A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Pertz, O., Hodgson, L., Klemke, R. L. & Hahn, K. M. Spatiotemporal dynamics of RhoA activity in migrating cells. Surks, H. K. et al. Regulation of myosin phosphatase by a specific interaction with cGMP-dependent protein kinase Iα. Chung, K. K. H. et al. Fluorogenic DNA-PAINT for faster, low-background super-resolution imaging. Gunn, M. D. et al. Mice lacking expression of secondary lymphoid organ chemokine have defects in lymphocyte homing and dendritic cell localization. We thank all of the members of the Ting Zhou and Yongdeng Zhang laboratories for discussions and support; the staff at the Laboratory Animal Resources Center, High-Performance Computing Center, Flow Cytometry Core, Genomic Core and Microscopy Core at Westlake University for support with techniques; and K. Guan, Q. Ma, D. Li, J. Wang, X. Zhou, D. He, H. Xu and C. Rosen for discussions and suggestions. This study was supported by National Natural Science Foundation of China grant 32270968, the Noncommunicable Chronic Diseases-National Science and Technology Major Program (2023ZD0500402 to T.Z. ), the ‘High Risk High Impact' (HRHI) program of Westlake Laboratory of Life Sciences and Biomedicine W101110566022301, ‘Pioneer' and ‘Leading Goose' R&D Program of the Department of Science and Technology of Zhejiang Province 2023SDXHDX0001, Westlake Multidisciplinary Research Initiative Center (MRIC) Seed Fund MRIC20210202 (T.Z.) and the Education Foundation of Westlake University. These authors contributed equally: Haichao Tang, Zongfang Wei School of Medicine, Zhejiang University, Hangzhou, China Westlake Laboratory of Life Sciences and Biomedicine, Hangzhou, China Haichao Tang, Zongfang Wei, Bei Zheng, Peihan Wu, Lulu Wu, Xiaohe Ma, Yanqin Chen, Jinmin Xu, Yu Qiao, Ying Zhang, Juju Miao, Zijing Yu, Yaodong Zhao, Zhen Xia, Qi Xie, Luming Zhao, Xu Li, Yongdeng Zhang & Ting Zhou School of Medicine, Westlake University, Hangzhou, China Haichao Tang, Zongfang Wei, Lulu Wu, Xiaohe Ma, Jinmin Xu, Ying Zhang, Juju Miao, Yaodong Zhao, Luming Zhao & Ting Zhou School of Life Sciences, Westlake University, Hangzhou, China Haichao Tang, Zongfang Wei, Bei Zheng, Peihan Wu, Lulu Wu, Xiaohe Ma, Yanqin Chen, Jinmin Xu, Yu Qiao, Ying Zhang, Juju Miao, Zijing Yu, Yaodong Zhao, Zhen Xia, Qi Xie, Luming Zhao, Xu Li, Yongdeng Zhang & Ting Zhou Department of Pathology, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China Yumeng Cai, Si Su & Huanwen Wu Hangzhou Cancer Hospital, Affiliated Hangzhou First People's Hospital, School of Medicine, Westlake University, Hangzhou, China Rongjing Zhou & Bing Xia Department of Breast Surgery, Affiliated Hangzhou First People's Hospital, School of Medicine, Westlake University, Hangzhou, China Jian Liu & Jufeng Guo Shanghai Institute of Immunology, Department of Immunology and Microbiology, Shanghai Jiao Tong University School of Medicine, Shanghai, China Zhaoyuan Liu & Florent Ginhoux Gustave Roussy Cancer Campus, Villejuif, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed all of the experiments. performed imaging experiments. and Yongdeng Zhang performed super-resolution imaging. collected and prepared human patient samples. performed western blot experiments. performed biotin–cGMP pull-down assays and MS analysis. and Y. Zhao helped to maintain the mice. and F.G. provided DC-related genetic mice. helped to provide agents and assays. edited and discussed the paper. Correspondence to Ting Zhou. The authors declare no competing interests. Nature thanks Ana-Maria Lennon-Duménil, Caetano Reis e Sousa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b, Immunofluorescent staining of cDCs (CD11C and LAMP3), T cells (CD3), or B cells (CD19) in early- or late-stage SLNs from breast cancer (n = 5 vs 6 patients) (a) and colon cancer (n = 5 vs 5 patients) (b) patients. Scale bars, overview, 1 mm; zone a and b, 100 μm. Quantifications on the right panel were CD11C/LAMP3 double positive area out of the total SLN area (%) (upper), or area value of 2 randomly selected areas (500 × 500 μm2) within each sample (lower) (n = 10 vs 12 areas in a; n = 10 areas each in b from corresponding patients). Lower quantification is displayed as area per 10 μm2. c,d, Quantifications of tumour volume and tdLN mig-cDCs (CD11c+MHC-IIhi) at day 7, 10, and 14 (c, B16F10, n = 4, 4, 6 mice; d, MC38, n = 4 mice per group). e, Quantification of IFNγ and TOX frequencies in CD44+PD1+CD8+ T cells in B16F10 tumour (n = 3 vs 6 mice). f, Quantifications of mig-cDCs, mig-CD103+ cDCs, or mig-CD11b+ cDCs in tdLNs of MC38 tumour-bearing mice (n = 5 vs 6 mice). g, Flow cytometric gating strategy for DC populations in tdLNs (sample derived from B16F10-OVA-ZsGreen tumour bearing mice). ZsGreen negative control was indicated as grey. h, Quantifications of migratory XCR1+ and SIRPα+ cDCs in tdLNs of B16F10-OVA-ZsGreen -bearing mice (n = 6 mice per group). i, Quantifications of migratory CD103+ and CD11b+ cDCs in non-tdLNs of B16F10-OVA-ZsGreen-bearing mice (n = 6 mice per group). j, Quantifications of total CD8+ T cells, CD44+CD8+ T cells and PD1+CD8+ T cells in tdLNs from mice bearing early- or late-stage tumours (n = 6 mice per group). Data are merged (a, b) or representative (c-j) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Student's unpaired t-test. a, Flow cytometric gating strategy for DC populations in TME. b,c, Quantifications of total cDCs, CD103+ cDCs, and CD11b+ cDCs in TME from early or late stage (b, B16F10, n = 5 mice; c, MC38, n = 6 mice). d, LAMP3+ DC signatures in early- or late-stage cancer patients based on TCGA database. e,i, All the mapped sgRNA hits for the pre-round (e) or major-round (i). Data are merged from 5 (e) or 3 (i) independent experiments. The x-axis indicates sgRNA numbers and y-axis indicates the calculated log2 fold change. f,j, Coverage calculation in each step from pre-round (f) or major-round (j) screening. g,k, Gini-Index analysis for pre-round (g) or major-round (k) screening. h, Minipool gRNAs selection strategy after the pre-round screening. l, Distribution of the enriched sgRNAs for top-hit genes from the major-round screening. m, Western blot analysis of Pde5 expression in BMDCs derived from Pde5+/+ or Pde5−/− mice. β-actin was used as control. n, Frequencies of CTV-labelled Pde5+/+ and CFSE-labelled Pde5−/− DCs in the TME two days post peritumour transfer (n = 6 mice). Quantifications on right panel were normalized to the mixture ratio before cell transfer. o, Whole-mount tissue 3D reconstitution (left) and cross-section image (middle) of tdLNs from mice co-transferred with Pde5+/+ and Pde5−/− DCs. Cells were surface rendered and DC signals were processed as dots. Data quantified from full tissue sections from each mouse (n = 2 mice). p, Representative staining and quantification of CTV-labelled Pde5+/+ and CFSE-labelled Pde5−/− mature BMDCs in mouse ear explants (n = 7 lymphatic vessels). Data are representative (b, c, m, n) or merged (e, g, i, k, l, o, p) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Student's unpaired (b, c) or paired (n, p) t-test or two tailed Mann–Whitney U-tests (d). a, Gene set enrichment analysis of down-regulated genes associated with cytoskeletal protein binding in late-stage tumoural DCs. NES, normalized enrichment score. b, Gene ontology enrichment analysis of downregulated pathways in late-stage tumoural DCs. c, Differentially expressed genes associated with GTPase in early- or late-stage tumoural DCs. d, Representative images and quantifications of tumoural DCs within early- or late-stage tumour margin lymphatic vessels (white arrowheads; n = 57 lymphatic vessels). e, Transcriptional analysis of Ccr7 (left, n = 2 biological samples) and flow cytometric quantification of CCR7 on tumoural DCs (right, n = 6 vs 5 mice). f,g, Representative trajectories of BMDC migration (corresponding to experiments and velocity statistics in Fig. h, ELISA examining intracellular cGMP levels in Pde5+/+ and Pde5−/− BMDCs (n = 3 biological replicates). i, Mean velocities of BMDCs treated with vehicle (n = 169 cells), 10 μM (n = 157 cells), 100 μM (n = 182 cells), or 500 μM 8-Br-cGMP (n = 142 cells) in 3D migration assay. j,k, Trajectories and mean velocities of mature BMDCs in 3D collagen gel assay migration assay treated with vehicle (n = 181 cells) or 500 μM 8-Br-cGMP (n = 175 cells) (j), or using Pde5−/− BMDCs (Pde5+/+, n = 193 cells; Pde5−/−, n = 116 cells) (k). l, Schematic of cGMP pathway components in regulating DC interstitial motility. The diagrams were created in BioRender. m, t-SNE plot visualization of flow cytometric quantified cell populations (left) and NOS2 fluorescence intensity (right) in B16F10 TME. n, Representative trajectories of BMDC migration (corresponding to Fig. o, Trajectories and mean velocities of mature BMDCs either treated with vehicle (n = 188 cells) or 100 μM DETA NONOate (n = 154 cells). Data are representative (e, h, m) or merged (d, f, g, i-k, n, o) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Mann–Whitney U-tests (d, j, k, o), two-tailed Student's unpaired t-test (e, h) or Kruskal–Wallis test with multiple comparisons (i). Trajectories were randomly selected from cells after quantifying their velocities under the same conditions. a, pMLC signal visualization using DNA-PAINT imaging at nanoscale resolution in whole cell (overlay) or y-z slice view (x = 200 nm) of BMDCs treated with vehicle or 8-Br-cGMP. pMLC signal quantifications are presented on the right (n = 75 steps per group, 1 step = 50 nm). White arrows indicate enhanced pMLC signals in peri-nuclear regions. Colour bar denotes depth of z axis. b, pMLC signal visualization in mature BMDCs treated with vehicle or 500 μM 8-Br-cGMP. Colour bar denotes depth. c, Quantifications of the pMLC signal ratio between the back and front of DCs treated with vehicle or 500 μM 8-Br-cGMP (n = 20 cells per group) (see Methods). d, Western blot analysis of pMLC expression in mature BMDCs treated with 500 μM 8-Br-cGMP for varying durations (0, 5, and 15 min). GAPDH was used as control. e-f, Western blot analysis of pMLC and pROCK levels in mature BMDCs treated with 500 μM 8-Br-cGMP plus 10 μM Y27632 (e) or 50 μM Rhosin hydrochloride (f) for 5 min. g, Quantifications of RhoA activity in BMDCs treated with vehicle (n = 26 cells) or 500 μM 8-Br-cGMP (n = 50 cells) using FRET sensor, with FRET ratios plotted. h, Mean velocities of BMDCs treated with vehicle (n = 125 cells), 500 μM 8-Br-cGMP (n = 158 cells), 8-Br-cGMP plus 50 μM Rhosin (n = 100 cells) or 10 μM Y27632 (n = 132 cells). i,j, Transcript analysis of PRKG1 and PRKG2 in different cell populations of Pan-cancer (i) or colorectal cancer (CRC) (j) patients (see Methods). k, Mean velocities of mature BMDCs treated with vehicle (n = 132 cells), 500 μM 8-Br-cGMP (n = 115 cells), 30 μM sildenafil (n = 137 cells), or each plus 86 μM PKG inhibitor (n = 121, 130, 123 cells correspondingly) in 3D collagen gel assay. Data are representative (a-f) or merged (g, h, k) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Mann–Whitney U-tests (g) or Kruskal–Wallis test with multiple comparisons (h, k). a,b, Trajectories and mean velocities of immature BMDCs (n = 147 vs 136 cells) (a) or mature BMDCs (n = 150 vs 174 cells) (b) treated with 30 μM sildenafil in 3D collagen gel assay. c, Mean velocities of BMDCs treated with vehicle (n = 151 cells), 10 μM (n = 137 cells), 30 μM (n = 173 cells), or 50 μM sildenafil (n = 137 cells) in 3D migration assay. d-i, Comparative characteristics between B16F10/B16F10Pde5−/− (d-f) and MC38/MC38Pde5−/− tumour cells (g-i) (n = 3 biological replicates for d, g; n = 6 mice per group for e, f, h, i). j,k, Tumour growth and Kaplan–Meier survival curves of MC38Pde5−/− bearing mice treated with 7 doses of vehicle or sildenafil every other day via intraperitoneal (i.p.) injection (n = 10 vs 13 mice) (j), or oral gavage (gava.) (n = 11 vs 10 mice) (k). l, Tumour growth curves of MC38Pde5−/− bearing mice treated with vehicle or sildenafil at specified doses (vehicle, n = 7; 200 μg sildenafil, n = 6; 400 μg sildenafil, n = 7). m, Tumour growth curves of Batf3+/− and Batf3−/− mice bearing MC38Pde5−/− tumour and treated with vehicle or sildenafil (Batf3+/− + vehicle, n = 5; Batf3+/− + sildenafil, n = 6; Batf3−/− + vehicle, n = 5; Batf3−/− + sildenafil, n = 6). n, Tumour growth curves of MC38Pde5−/− bearing mice treated with vehicle plus isotype, sildenafil plus isotype, vehicle plus anti-CD8, or sildenafil plus anti-CD8 antibody (n = 3, 3, 6, 4 mice). o, Quantifications of mig-CD103+ or CD11b+ cDCs in tdLNs from each group (n = 5, 3, 4 mice). p, Tumour growth curve of MC38Pde5−/− bearing mice treated with vehicle, sildenafil, sildenafil plus 50 μM blebbistatin, or 50 μM Y27632 (n = 3, 3, 6, 6 mice). q, IF staining of DCs in tumoural lymphatic vessels (n = 22, 31, 30 vessels). r, Quantifications of H-2Kb/SIINFEKL, CD80 and CD86 levels on mig-cDCs in tdLNs from each group (n = 5, 5, 3, 4 mice). Data are merged (a-c, j-l, q) or representative (d-i, m-p, r) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Mann–Whitney U-tests (a, b), two-tailed Student's unpaired t-test (d, f, g, i, o) or Kruskal–Wallis test with multiple comparisons (c, q), two-way ANOVA with multiple comparisons corrected using Šidák's multiple comparison test (d, e, g, h and j–n) or one-way ANOVA with Tukey's multiple-comparison test (r). a, Quantifications of TCF1+TIM3− CD8+ T cells in the TME in each group (n = 6 vs 8 mice). b, Quantifications of CD44+PD-1+TIM3−CD8+ T cells, IFNγ+CD44+PD-1+CD8+ T cells, and CD44+PD-1+IFNγ+CD8+ T cells in the TME in each group (n = 6 vs 8 mice). c, Tumour growth and survival curves of B16F10Pde5−/− bearing mice treated with 7 doses of vehicle or sildenafil every other day via subcutaneous (s.c.) injection (n = 10 vs 12 mice). d, Kaplan–Meier survival curve of previously cured mice rechallenged with a double dose of MC38Pde5−/− cells (WT, n = 4; cured, n = 5). Age- and sex-matched mice served as controls. e, Quantifications of CD44+CD8+ and Ki67+CD8+ T cells in tdLNs from each group (n = 6). f, Representative flow cytometric histograms of IFNγ or GZMB in CD44+PD-1+CD8+ and quantifications of CD44+PD-1+CD8+ T cells and GZMB+CD44+PD-1+CD8+ T cells in the TME from each treatment group (n = 6 per group). GZMB, Granzyme B. g,h, TCGA pan-cancer analysis using DC gene signature combined with PDE5 expression (n = 1066 patients). Kaplan–Meier survival curve of patients stratified by median PDE5 expression into high or low (n = 537 vs 529 patients) DC gene signature groups (g). Correlation between PDE5 and CD8 T cell infiltration levels (h). i, Quantifications of mig-CD103+ and CD11b+ cDCs in Zbtb46crePde5fl/fl mice bearing MC38Pde5−/− tumour and treated with vehicle or sildenafil (vehicle, n = 5; sildenafil, n = 4). j, Quantifications of tdLNs CD44+CD8+ or TME IFNγ+CD44+PD-1+CD8+ T cells in Zbtb46crePde5fl/fl mice bearing tumour and treated with vehicle or sildenafil (n = 5 vs 4 mice). k, Schematic of the design strategy for LSL-Pde5 mice. l, Survival curves corresponding to results in Fig. m, Quantifications of tdLN mig-CD103+ and CD11b+ cDCs in Zbtb46crePde5LSL mice bearing MC38Pde5−/− tumour and treated with vehicle or sildenafil (n = 6 per group). n, Quantifications of tdLN CD44+CD8+ or IFNγ+CD44+PD-1+CD8+ T cells in Zbtb46crePde5LSL mice bearing MC38Pde5−/− tumour and treated with vehicle or sildenafil (n = 6 per group). o, Tumour growth curves of Zbtb46cre or Zbtb46crePde5LSL mice inoculated with MC38Pde5−/− tumour. Data were re-analysed from Fig. p,q, Quantifications of tdLN migratory cDCs (p), or migratory CD103+ and CD11b+ cDCs (q) in Zbtb46cre and Zbtb46crePde5LSL mice (n = 4 per group). r,s, Quantifications of CD44+CD8+ T cells in tdLNs (r) or IFNγ+CD44+PD-1+CD8+ T cells in the TME (s) in Zbtb46cre and Zbtb46crePde5LSL mice (n = 4 per group). Data are representative (a, b, d, e-j, m, n, p-s) or merged (c, l, o) of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Student's unpaired t-test (a, b, i, j, m, n, p-s), two-way ANOVA with multiple comparisons corrected using Šidák's multiple comparison test (c, o), or one-way ANOVA with Tukey's multiple-comparison test (e, f). a, Representative flow cytometric plots and quantifications of FITC+ migratory and resident cDCs in tdLNs from Pde5fl/fl and Zbtb46crePde5fl/fl mice (Pde5fl/fl, n = 6; Zbtb46crePde5fl/fl, n = 7). Mice were painted with FITC-LPS mixture and tdLNs were analysed 48 h post painting to assess FITC uptake. b, Quantifications of cDC percentages, CCR7 expression, or surface CD80/CD86 expression from Pde5+/+ and Pde5−/− BMDCs differentiated with FLT3L for 9 days (n = 3 biological replicates per group). c,d, Quantifications of migratory cDCs (c), and migratory XCR1+ and SIRPα+ cDCs (d) in mediastinal LNs from Pde5fl/fl and Zbtb46crePde5fl/fl mice at homeostasis (n = 6 per group). e, Quantification of total cDCs in lung tissue from Pde5fl/fl and Zbtb46crePde5fl/fl mice (n = 6 per group). Cells were gated on live CD45+CD3−B220−Gr1−F4/80−. Data are representative of at least two to three independent experiments and present as mean ± s.e.m. Statistical analysis was performed using two-tailed Student's unpaired t-test. 1: the gating strategies for flow cytometry analysis. b, The gating strategy for analysis of immune cells in the TME. 2: uncropped immunoblot images with size marker indications. The loading control of β-actin or GAPDH was run on the same gels. Results from the pre-round and major-round in vivo CRISPR screens. This table includes gene-level and sgRNA-level results from both the pre-round and major-round in vivo CRISPR library screens, as well as the composition of the minipool used in the major round. LFC, log2 fold change; RRA, robust rank aggregation [RRA] lo value, reflecting positive selection. Identification of cGMP-binding proteins by MS. This table lists proteins enriched in the cGMP-pull-down assay, including protein identities and their corresponding MS intensities. NC, negative control group treated with vehicle. Patient information for SLN staining and DC motility analysis. This table presents the clinical and pathological characteristics of the patient cohort evaluated for SLN staining and DC motility. Flow cytometry antibodies used in this study and the associated product information. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Tang, H., Wei, Z., Zheng, B. et al. Rescuing dendritic cell interstitial motility sustains antitumour immunity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01745-1'>Wake up call for AI: computer-vision research increasingly used for surveillance</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 16:12:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Imaging research in the popular field of computer vision almost always involves analysing humans and their environments, and most of the subsequent patents can be used in surveillance technologies, a study has found. Computer-vision research involves developing algorithms to extract information from images and videos. It can be used to spot cancerous cells, classify animal species or in robot vision. Police forces and governments say that AI-powered surveillance allows them to better protect the public. The analysis assessed 19,000 computer-vision papers published between 1990 and 2020 at the leading Conference on Computer Vision and Pattern Recognition, as well as 23,000 patents that cited them. In a wider analysis the researchers searched all the patents for a list of keywords linked to surveillance, such as ‘iris', ‘criminal' and ‘facial recognition'. They found that in the 2010s, 78% of computer-vision papers that led to patents produced ones related to surveillance, compared with 53% in the 1990s (see ‘Surveillance-enabling research'). Almost “the entire field is working on faces and gaits, on detecting people in images, and nobody seems to be saying, ‘wait what are we doing here? Although many have assumed that there was a pipeline from computer-vision research to surveillance, that is “very different than having actual empirical evidence”, says Sandra Wachter, who studies technology and regulation at the University of Oxford, UK. Read the related News & Views: Computer-vision research is hiding its role in creating ‘Big Brother' technologies Computer-vision research is hiding its role in creating ‘Big Brother' technologies Is facial recognition too biased to be let loose? Can industry fill the gap left by US research funding cuts? These are the top companies and countries for industry research Nations urged to ‘pick up the ball' after DEI research cuts A Postdoctoral position is available at an NIH-funded lab to study the role of inflammasome, nucleic acid sensing and interferon signaling in neutr... Computer-vision research is hiding its role in creating ‘Big Brother' technologies An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09172-y'>Spatiotemporal orchestration of mitosis by cyclin-dependent kinase</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 15:41:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mitotic onset is a critical transition for eukaryotic cell proliferation. The commonly held view of mitotic control is that the master regulator, cyclin-dependent kinase (CDK), is first activated in the cytoplasm, at the centrosome, initiating mitosis1,2,3. Bistability in CDK activation ensures that the transition is irreversible, but how this unfolds in a spatially compartmentalized cell is unknown4,5,6,7,8. Here, using fission yeast, we show that CDK is first activated in the nucleus, and that the bistable responses differ markedly between the nucleus and the cytoplasm, with a stronger response in the nucleus driving mitotic signal propagation from there to the cytoplasm. Abolishing cyclin–CDK localization to the centrosome led to activation occurring only in the nucleus, spatially uncoupling the nucleus and cytoplasm mitotically, suggesting that centrosomal cyclin–CDK acts as a ‘signal relayer'. We propose that the key mitotic regulatory system operates in the nucleus in proximity to DNA, which enables incomplete DNA replication and DNA damage to be effectively monitored to preserve genome integrity and to integrate ploidy within the CDK control network. This spatiotemporal regulatory framework establishes core principles for control of the onset of mitosis and highlights that the CDK control system operates within distinct regulatory domains in the nucleus and cytoplasm. At mitotic onset, the cell undergoes many rapid changes across the cell, including modifications to the nuclear membrane, chromosome condensation, disassembly of cytoskeletal microtubules and formation of the mitotic spindle9. These changes are driven mainly by cyclin–CDK complexes10 through phosphorylation of hundreds of different substrates10,11,12. However, a full understanding of the spatial and temporal dynamics of the CDK control network has been difficult to establish. This is in part owing to the existence of multiple different cyclin–CDK complexes and regulators in most eukaryotes, which have potential overlapping and redundant functions, making interpretation of experiments complex12,13,14,15. In addition, in vitro systems designed to investigate the dynamics of CDK activation that take account of cellular spatial compartmentation have been limited5,6,14. Finally, there is a lack of precise and simultaneous single-cell measurements of in vivo CDK activity and endogenous levels of CDK and its regulators16,17. The Cdk1 kinase in complex with its regulatory subunit cyclin is the main driver triggering cellular changes at mitotic entry18. Its activity increases gradually during the cell cycle until late G2, when it increases more rapidly to bring about the onset of mitosis through a combination of cyclin accumulation and regulation of Cdk1 tyrosine residue phosphorylation (Y15 in most eukaryotes) near its ATP binding site4,5,6,19,20. The inhibitor Wee1 kinase family phosphorylates Y15 to inactivate Cdk1, while the activator Cdc25 family dephosphorylates this site21,22,23,24. Theoretical modelling and biochemical experiments of these positive (Cdk1 and Cdc25) and double-negative (Cdk1 and Wee1) feedback loops have shown they are able to abruptly increase Cdk1 activity at mitotic entry4,5,6. Furthermore, these feedback loops can lead to bistability and hysteresis, where for a given concentration of cyclin, CDK activity can either be high if the state of the system is mitotic or low if it is in interphase4,6,8. This ensures that the mitotic transition is irreversible and that the interphase and mitotic states are clearly distinguished, thus preventing cells from slipping back and forth between them4,5,6,29. Such spatial complexity poses a challenge to understanding how a cell coordinates mitotic entry across the cell7,13,16,30,31. Previous studies have suggested that the initial mitotic CDK activation is triggered in the cytoplasm at the centrosome, the spindle pole body (SPB) in yeasts, which acts as a mitotic signalling hub that controls the timing of mitosis, from yeast to humans1,2,3,16,32. This view has been challenged by experiments using Xenopus egg extracts, which suggest that the nucleus might control the timing of mitosis33,34. Differential localization of mitotic regulators such as Wee1 (enriched in the nucleus)35 also raise the possibility of differences in the bistable responses between the nucleus and cytoplasm7. Schizosaccharomcyes pombe (fission yeast) has been an excellent model system for understanding core principles in eukaryotic mitotic control36. This is partly owing to the relative simplicity of its cell cycle control architecture, with only one mitotic cyclin–CDK complex, the B-type cyclin Cdc13, complexed with the Cdk1 homologue Cdc237,38. Cell cycle control can be further simplified by engineering a Cdc13–wild-type Cdc2 (Cdc2(WT)) fusion, which can drive the entire cell cycle36. Here we use fission yeast to reveal the underlying spatiotemporal regulatory principles of how CDK orchestrates the onset of mitosis in vivo. We explored whether there were differences in CDK substrate phosphorylation in different subcellular locations by reanalysing data from an earlier fission yeast phosphoproteomics experiment, to determine when CDK substrates are phosphorylated during the cell cycle11. We focused on sites that were phosphorylated in late G2 and mitosis using a localization annotation method39 to categorize phosphorylation sites on the basis of their cellular compartment11. 1a,b and Extended Data Fig. There was no significant correlation between the sensitivity of the sites to CDK activity, as judged by half-maximal inhibition concentrations (IC50 values)11 and the timing of increases in phosphorylation at mitotic onset (Extended Data Fig. These results suggest that mitotic CDK activation occurs first in the nucleus before activation in other compartments, including the cytoplasm and SPB. a, Mean phosphorylation of CDK late sites categorized on the basis of annotated localization data. b, Mean times of the change points in substrate phosphorylation. An unequal two-sample, left-tailed t-test was performed to test whether nuclear sites had an earlier rate change compared with sites in other compartments. There was no variability with the estimated rate change points for cytoplasmic periphery and SPB sites. Red circles represent individual data points. c, Diagram of the minimal phosphorylation module used for construction of NucCDK. NucCDK normally resides in the nucleus (blue) but upon phosphorylation by CDK, translocates into the cytoplasm (grey). d, Representative profile of NucCDK–mScarletI showing the cytoplasm to nuclear ratio of mean intensity (C/N). Cells were imaged in Edinburgh minimal medium (EMM) with 5-min time intervals. e,f, Representative single-cell trace of NucCDK readout in wild-type (e) and cdc25-22 mutant (f) strains. Images were acquired every 10 min in YE4S medium at 36 °C. To investigate this, new single-cell sensors were required to assay CDK activity in the nucleus and cytoplasm in unperturbed cells. To assay nuclear CDK activity, we developed the CDK sensor NucCDK, which resides in the nucleus but translocates to the cytoplasm upon phosphorylation by nuclear CDK (Fig. NucCDK sensor readout increased gradually throughout G2, switching to a rapid increase before stoppage of cell elongation, a feature of mitotic cells40 (Fig. We tested whether this switch occurred at mitotic onset using a temperature sensitive cdc25-22 mutant that blocks cells in late G2 (ref. Unlike wild-type cells, there was no rapid switch at 36 °C, confirming that the switch was associated with mitotic onset (Fig. Thus, this switch provides a readout of mitotic CDK activation. It is localized in the cytoplasm in interphase but translocates into the nucleus upon CDK phosphorylation42. We used this sensor (hereafter referred to as CytCDK) with NucCDK, and performed dual-colour imaging to determine the timing of CDK activation simultaneously in both the nucleus and cytoplasm (Fig. a, Diagram of the dual sensor system. b, Montage with NucCDK–mScarletI (magenta) and CytCDK–mNG (green), taken every 5 min, in YE4S medium. Representative from three biological repeats. c, Representative trace of results from the dual sensor system. Black circles and dotted lines indicate where the rate change algorithm detected significant changes in sensor readouts. Sensor readouts were normalized to minimum and maximum values. Cell cycle time (normalized) was normalized to the division time for the individual cell. d, Probability density function (PDF) showing time differences from nuclear activation to cytoplasmic activation. n = 93 cells; 1 of 3 biological repeats. DMSO was added after eight time points of image acquisition as a control. A 3-min delay (shown with dashed lines) of when sensor readouts begin to decrease owing to normal cell cycle CDK inactivation. Mean delay time (Tdelay) is shown below the graph, with s.d. n = 18 cells; 1 of 2 biological repeats. f, Representative trace of NucCDK and CytCDK in Cdc13–Cdc2As background after release from G2, followed by inhibition with 200 nM 1-NmPP1, followed by release. g, Representative trace of dual sensor strain with NucCDK–mScarletI and CytCDK V2–mNG. Experiments were done in YE4S medium, with images taken every 5 min. h, PDF of time differences from nuclear activation to cytoplasmic activation using NucCDK–mScarletI and CytCDK V2–mNG. n = 123 cells; 1 of 3 biological repeats. i, Representative trace in the Cdc13–analogue-sensitive Cdc2AF (Cdc13–Cdc2AFAs) background from block and release from G2. DMSO was added after 11 time points of image acquisition as a control. A 3-min delay (shown with dashed lines) of when sensor readouts begin to decrease owing to normal cell cycle CDK inactivation. Mean Tdelay is shown below the graph, with s.d. n = 30 cells; 1 of 2 biological repeats. The readouts of NucCDK and CytCDK increased rapidly at the onset of mitosis, with the increase in NucCDK readout preceding that of CytCDK (Fig. The time delays were about 12 min and 7 min near the beginning and near the peak, respectively, suggesting that nuclear CDK activation precedes cytoplasmic CDK activation (Extended Data Fig. To further investigate this delay, we systematically identified the initial point where the rate change occurs for each sensor readout, marking the change in CDK activity from a gradual to a rapid increase owing to mitotic CDK activation (Fig. We observed a time delay between nuclear and cytoplasmic activation of 5–10 min, similar to the previous estimate of approximately 12 min near the beginning of the increase in NucCDK readout (Fig. These experiments indicate that nuclear CDK activation occurs before cytoplasmic CDK activation, raising the possibility that the increase in nuclear CDK activity might bring about the subsequent increase in cytoplasmic CDK activity. However, the time delay at mitotic onset between the NucCDK and CytCDK readouts could be due to a delay either in CDK phosphorylation of the sensors or to differences in sensor translocation kinetics. To distinguish these possibilities, we first acutely inhibited CDK activity after release from a G2 block during the rapid increase in CDK activity at mitotic onset, in a strain carrying an analogue-sensitive (As) version of the Cdc13–Cdc2(WT) fusion protein36 and both NucCDK and CytCDK (Fig. Upon release from the G2 block, there was a delay of around 6 min in the increase of NucCDK and CytCDK, similar to our estimate of 5–10 min (Fig. However, when cells were blocked and released during mitosis, there was no detectable delay between the increases of NucCDK and CytCDK, suggesting that both sensors translocate rapidly after CDK phosphorylation, and that the approximately 6-min delay is due to differences in the timing of CDK activation between the nucleus and cytoplasm (Fig. We also examined the response of the sensors after CDK inhibition. There was no delay between CDK inhibition and a change in NucCDK readout, but there was a delay of around 3 min in the change in CytCDK readout (Extended Data Fig. We also characterized the in vivo sensitivities of NucCDK and CytCDK to CDK activity by determining the CDK inhibition dose–response curves for both sensors (Methods). This revealed that NucCDK is more sensitive to CDK activity than CytCDK (Extended Data Fig. To test whether this might affect our observation on the order of CDK activation, we developed a version of CytCDK that was more sensitive to CDK activity (CytCDK V2; Methods). CytCDK V2 had similar sensitivity to CDK activity as NucCDK (Extended Data Fig. 2g), so we repeated the dual sensor experiments. CytCDK V2 showed an earlier gradual increase in readout during G2, but the timing of the transition to a rapid increase representing CDK activation was the same as when using CytCDK (Fig. 2g,h and Extended Data Fig. 2h), confirming our intepretation of the sequential order of nuclear and cytoplasmic CDK activation. Overall, these experiments indicate that NucCDK readout increases before that of CytCDK at mitotic onset owing to differences in the timing of CDK activation between the nucleus and cytoplasm. We performed a further experiment to determine whether nuclear CDK activation occurs sequentially before cytoplasmic CDK activation by genetically bringing about a more gradual increase in CDK activity at mitotic onset, with the assumption that at mitotic onset a critical nuclear CDK activity threshold has to be reached to trigger the subsequent increase in cytoplasmic CDK activity17 (discussed further below). The more gradual increase in nuclear CDK activity would mean that it takes longer to reach this critical threshold, extending the delay in the increase of cytoplasmic CDK activity. We used a mutant strain carrying an analogue-sensitive version of the Cdc13–Cdc2(T14A/Y15F) fusion protein36 (Cdc13–Cdc2AF) that abolishes the CDK Y15 feedback loops thus making the increase in CDK activity more gradual. These results further support that there is a sequential order of nuclear and cytoplasmic CDK activation, and that nuclear CDK activity has a role in controlling the timing of the increase of mitotic cytoplasmic CDK activity. The sensors are CDK substrates that are also phosphatase substrates, which may influence the timing of the switches in sensor readout43. Greatwall kinase links CDK activity to PP2A–B55 phosphatase activity, so that when CDK is activated at mitotic onset, PP2A–B55 is inactivated, resulting in a rapid increase in substrate phosphorylation43,44,45. Therefore, Greatwall might influence the delay between switches in sensor readout of NucCDK and CytCDK or CytCDK V2 through phosphatase regulation. However, deletion of ppk18, which encodes the main Greatwall kinase in fission yeast44, had no significant effect on the timing of the switches (Extended Data Fig. We conclude from the sensor data and the phosphoproteomics analysis that there is a sequential temporal order in CDK activation at mitotic onset, with activation occurring first in the nucleus. The translocation of cyclin B1 in human cells at the onset of mitosis is important for mitosis16,31, so we determined whether translocation of cyclin–CDK also occurs in fission yeast, coupling cytoplasmic activation to nuclear activation. We performed timelapse microscopy of Cdc13 levels using Cdc13 tagged with superfolder GFP (sfGFP) combined with a nuclear mask to segment the nucleus (Methods). We observed a decrease in nuclear intensity of Cdc13 just before SPB separation, with a corresponding increase in the cytoplasm, indicative of nuclear export at mitotic onset (Fig. This was followed by a decrease throughout the cell after spindle assembly due to Cdc13 degradation at mitotic exit (Fig. For subsequent experiments, we used the mean of the top 15% of pixels as an estimate of nuclear intensity, as nuclear area comprises approximately 15% of cell area46. This gave similar results to using a nuclear mask (Extended Data Fig. 3c), but avoided excessive light exposure. Timelapse imaging of Cdc2–mNeonGreen (mNG) indicated a similar nuclear export happening before spindle assembly, followed by further export after Cdc13 degradation (Fig. We performed dual-colour imaging of Cdc13–sfGFP with NucCDK and CytCDK to identify when Cdc13 export occurs in relation to nuclear and cytoplasmic activation. We found that nuclear activation occurred about 5 min before export of Cdc13, whereas cytoplasmic activation occurred concurrently with export (Fig. A similar 5-min difference was observed for Cdc2–mNG with respect to nuclear activation, suggesting that Cdc13 and Cdc2 translocate as a complex (Extended Data Fig. When we calculated the nuclear and cytoplasmic mean Cdc13–sfGFP intensities at nuclear and cytoplasmic activation, we found the nucleus had a higher value, indicating differences in the cyclin threshold for CDK activation between the nucleus and cytoplasm (Fig. These differences are unlikely to be due to differences in sensor sensitivities, because the rate change point in sensor readout does not change with sensors that have different sensitivities (Fig. This indicates that CDK is first activated in the nucleus, where there is a higher cyclin threshold, and this is followed by a fraction of the nuclear cyclin–CDK translocating to the cytoplasm, which overcomes the cyclin threshold in the cytoplasm to activate cytoplasmic CDK. a, Left, representative traces showing mean intensities of whole cell, nucleus and cytoplasm, using MCM2-3-Ala–mScarletI as nuclear mask. Dashed lines mark beginning and end of nuclear export. Cells were imaged in EMM, at 5-min intervals. b, Representative time trace of Cdc2–mNG mean intensity in nucleus and cytoplasm, calculated using the top 15% method. Dashed lines mark beginning and end of nuclear export. Imaged at 5-min intervals in EMM. c, Representative trace of NucCDK–mScarletI and Cdc13–sfGFP showing nuclear activation with respect to nuclear export (dashed line). Cells were imaged in EMM, with 5-min intervals. d, Representative trace of CytCDK–mScarletI and Cdc13–sfGFP showing cytoplasmic activation with respect to nuclear export, marked with dashed lines. Imaged at 5-min intervals in EMM. In c,d, orange circles on the nuclear mean intensity traces represent the point at which SPB separation is observed and yellow circles represent when nuclear division is observed. e, Histogram of nuclear (n = 76 cells; 1 of 3 biological repeats) and cytoplasmic (n = 57 cells; 1 of 3 biological repeats) activation with respect Cdc13–sfGFP nuclear export. f, Violin plots of nuclear mean intensity and cytoplasmic mean intensity after background subtraction, with nuclear (n = 76 cells) and cytoplasmic (n = 57 cells) CDK activation, respectively. Mean background values were calculated as the mean of the median values shown in Extended Data Fig. 3f and subtracted from each value. Bistability (multiple stable steady states of a system) and hysteresis (state of system dependent on its history) in Cdk1 activity has been shown in Xenopus egg extracts using steady-state measurements of Cdk1 activity on non-degradable cyclin6,8. However, to our knowledge, hysteresis in CDK activity has not been directly tested in vivo, and potential spatial differences have not been investigated in any system. Furthermore, hysteresis and positive feedback loops can be dispensable for stable oscillations in certain conditions5. Therefore, we determined the dynamic single-cell phase plots in the nucleus and cytoplasm in vivo, which represent the naturally occurring phase orbits in the dynamical system5, and investigated whether hysteresis would allow translocation of Cdc13–Cdc2(WT) to the cytoplasm without collapse of the mitotic state in the nucleus. The phase plots enable us to visualize the different states of CDK activity for different concentrations of cyclin4,5,6. The phase plots using data from Fig. 3c–f indicated a significant difference in the phase orbits between the nucleus and cytoplasm (Extended Data Fig. This was confirmed using the Cdc13–Cdc2(WT) fusion protein tagged with mNG, which confines analysis to a single cyclin–CDK complex36, together with NucCDK and CytCDK V2. The dynamics of Cdc13–Cdc2(WT)–mNG mimicked uncomplexed Cdc13 for both nuclear export and sensor readout profiles (Extended Data Fig. The nuclear and cytoplasmic phase orbits were plotted on the same phase space, after normalization to minimum and maximum of sensor readouts (Fig. Notably, the nuclear phase orbit cycled around a much larger region of phase space than the cytoplasm. A high amount of Cdc13–Cdc2(WT) accumulated in the nucleus before CDK activation to the high-activity mitotic state, with little increase in activity before that point. There was an approximately vertical line on the far right of the orbit at the point of CDK activation, consistent with an underlying bistable trigger. In the nucleus, once the system had reached the mitotic high CDK activity state, it was able to tolerate decreases in Cdc13–Cdc2(WT) without collapsing back to the low-activity state. By contrast, only a small amount of Cdc13–Cdc2(WT) was needed in the cytoplasm for CDK activation, and a small decrease in Cdc13–Cdc2(WT) resulted in collapse out of the cytoplasmic mitotic state (Fig. This suggests that CDK activity in the nucleus is much more stable with respect to cyclin–CDK fluctuations compared with CDK activity in the cytoplasm. The individual nuclear phase plots extended and shrunk significantly horizontally compared with the cytoplasm, also suggesting less sensitivity to cyclin–CDK concentrations (Fig. a, Representative phase plots of the nuclear phase orbit (Cdc13–Cdc2(WT)–mNG nuclear mean intensity and NucCDK–mScarletI) and cytoplasmic phase orbit (Cdc13–Cdc2(WT)–mNG cytoplasmic mean intensity and CytCDK V2–mScarletI). Sensor readouts were normalized to minimum and maximum values for comparison. b, Ten representative traces for nucleus and cytoplasm. c, Phase plots for the nucleus in Cdc13–Cdc2(WT)–mNG background. Normalized to minimum and maximum values. n = 89 cells; 1 of 3 biological repeats. d, Phase plots for the cytoplasm in Cdc13–Cdc2(WT)–mNG background. Normalized to minimum and maximum values. n = 37 cells; 1 of 3 biological repeats. Normalized to minimum and maximum values. Normalized to minimum and maximum values. Green dots represent individual time points, with distances separated by a time interval of 5 min. Single-cell phase plots of the nucleus after normalization to minimum and maximum of Cdc13–Cdc2(WT)–mNG mean intensities indicated a considerable gap between the low-activity and high-activity states, and after nuclear export the system was not prone to collapse back to the low-activity state (Fig. Furthermore, the orbits were very similar in different single cells, suggesting the CDK regulatory system oscillates with robustness to biological noise. By contrast, the normalized cytoplasm phase plots showed that the gaps between the high-activity and low-activity states for the same Cdc13–Cdc2(WT)–mNG intensity were less pronounced, and the cytoplasm was more sensitive to Cdc13–Cdc2(WT) accumulation (Fig. We repeated the experiments using Cdc13–Cdc2AF–mNG36. This tested whether CDK Y15 feedback loops are responsible for the stability in oscillations and for the distinction of the low and high CDK activity states in the nucleus and cytoplasm5. Nuclear export appeared less switch-like, suggesting that it is influenced by the altered CDK activity regulation (Extended Data Fig. The sensors displayed a linear increase in sensor readout (Extended Data Fig. 4g,h), and the phase diagrams of both the nucleus and cytoplasm exhibited almost a complete collapse of their orbits, with approximately linear profiles (Fig. 4e,f and Extended Data Fig. Thus, the CDK Y15 feedback loops promote oscillations between different states in both the nucleus and cytoplasm. We conclude that the CDK Y15 feedback loops provide stable oscillations in the nucleus, facilitating translocation of cyclin–CDK from the nucleus to the cytoplasm, and produce a weaker response in the cytoplasm. This is likely to reflect the underlying steady-state hysteretic responses that differ between the nucleus and cytoplasm5. Furthermore, our data suggest that at mitotic onset, the CDK Y15 feedback loops are major drivers in hysteresis and the increase in substrate phosphorylation, rather than phosphatase inactivation43. Our results do not support the view that mitotic CDK activation is triggered first in the cytoplasm at the centrosome or SPB thereby dictating mitotic onset timing1,3,32. Therefore, we investigated the possible role of cyclin–CDK at the SPB at mitotic onset. In fission yeast, CDK activation at the SPB has been shown to act through a polo kinase (Plo1)-mediated feedback loop, dependent on Plo1 association with the SPB scaffold protein Cut12 (refs. The mutant cut12.s11 promotes earlier recruitment of Plo1 to the SPB and global Plo1 activation48,49, so we determined whether cut12.s11 advanced the timing of nuclear CDK activation (Fig. Cell length is a proxy for cell cycle position in fission yeast51. Length at nuclear CDK activation assayed using NucCDK (Fig. 5c, right), were found to be the same in both cut12.s11 and wild-type cells. This indicates that promoting premature CDK activity at the SPB does not affect either nuclear CDK activation or the timing of cell division. This is in line with previous reports that promoting SPB feedback under a normal cell cycle context does not influence mitotic timing47. c, Left, wild-type (n = 96 cells; 1 of 3 biological repeats) and cut12.s11 (n = 195 cells; 1 of 3 biological repeats) cells, showing length at nuclear activation using NucCDK–mScarletI. e, Representative trace of NucCDK–mScarletI in cells expressing Cdc13(HPM)–sfGFP. f, Traces of mean nuclear Cdc13(HPM)–sfGFP intensity. n = 36 cells; 1 of 3 biological repeats. g, Representative trace of CytCDK–mScarletI in cells expressing Cdc13(HPM)–sfGFP. The y axis of CytCDK sensor readout was scaled on the basis of minimum and maximum values for CytCDK in Cdc13 wild-type cells. h, Model of CDK activation. During interphase, cyclin B–CDK accumulates in the nucleus, and CDK activity increases gradually, with a high threshold for activation. In late G2, CDK activation occurs and is amplified by CDK-Y15-dependent feedback loops. Subsequently, cyclin B–CDK nuclear export (dependent on centrosomal cyclin B–CDK) and cyclin B degradation occur, and high CDK activity is maintained by CDK-Y15-dependent feedback loops. Thus, the nuclear CDK system undergoes stable oscillations with respect to cyclin B–CDK concentration. By contrast, in the cytoplasm, a lower threshold for activation enables cytoplasmic CDK to activate upon import of cyclin B–CDK from the nucleus. Degradation of cytoplasmic cyclin B results in an immediate drop in CDK activity. Therefore, the cytoplasmic CDK system oscillates in an unstable manner with respect to cyclin–CDK concentration. We next investigated the role of cyclin–CDK localization at the SPB for mitotic control using the hydrophobic patch mutant of Cdc13 (Cdc13(HPM)), which does not localize to the SPB during G239 (Fig. This property is conserved with mammalian cyclin B1(HPM), which also does not localize to the centrosome39. Cells containing only Cdc13(HPM) do not display signs of mitotic entry, a phenotype partly rescued by tethering Cdc13(HPM) back to the SPB52. We performed timelapse microscopy of Cdc13(HPM)–sfGFP and wild-type Cdc13 (Cdc13(WT))–sfGFP using the CDK sensors (Methods) in a strain with other cyclins deleted. Cdc13(WT)–sfGFP displayed normal sensor readouts and Cdc13 dynamics (Extended Data Fig. NucCDK in the Cdc13(HPM)–sfGFP mutant showed that activation in the nucleus still occurred, even though Cdc13(HPM)–sfGFP did not localize to the SPB (Fig. However, this nuclear CDK activation was not followed by a decrease in nuclear levels, indicating that export from the nucleus did not take place (Fig. We repeated the timelapse experiments with Cdc13(HPM)–sfGFP and CytCDK and found that cytoplasmic CDK activation was also absent (Fig. Therefore, although nuclear activation still occurs, cytoplasmic CDK activation is impaired, a probable consequence of Cdc13(HPM)–Cdc2(WT) not being exported from the nucleus. We propose that Cdc13 localization to the SPB is not required for CDK activation in the nucleus, but is required for the translocation of Cdc13–Cdc2(WT) to the cytoplasm and subsequent activation of cytoplasmic CDK. These results are consistent with previous phosphoproteomics data39, which showed phosphorylation of nuclear substrates with Cdc13(HPM) was similar to that with Cdc13(WT), whereas phosphorylation of SPB and cytoplasmic substrates was impaired. They also support the sequential ordering of nuclear activation and cytoplasmic activation, with Cdc13 SPB localization having an important role in bringing about this ordering of events. Here we have revealed the underlying spatiotemporal regulatory framework in fission yeast that determines how CDK orchestrates the onset of mitosis in vivo (Fig. 5h), based on precise measurements of cyclin–CDK levels, the sensing of in vivo CDK activity and the generation of in vivo phase plots, to investigate the effects of cellular compartmentation in single unperturbed cells. Our work has shown that CDK is first activated in the nucleus, thereby controlling the timing of mitotic onset. Promoting activity at the SPB had no effect on the timing of nuclear CDK activation or timing of cell division. However, in the absence of cyclin B–CDK localization to the SPB, CDK activation occurs only in the nucleus. We propose that this is owing to the absence of signal propagation brought about by nuclear export of cyclin B–CDK to the cytoplasm. This stresses the importance of signal propagation for spatially coherent mitotic onset, and supports the view that the SPB acts as a signalling hub, but as a signal-relaying hub rather than one that controls the timing of mitotic onset. We also generated novel in vivo single-cell phase plots of CDK activity versus concentrations of cyclin B and of cyclin B–CDK complexes simultaneously in both nucleus and cytoplasm. These showed that the CDK oscillator operates in very different regulatory domains in the nucleus and cytoplasm. In the nucleus, there is a strong bistable or hysteretic response, which can buffer against fluctuations in cyclin B–CDK concentrations in both interphase and mitosis, resulting in stable oscillations of CDK activity. The strong bistable response and higher cyclin B threshold for CDK activation in the nucleus reduces susceptibility to ‘noise' and to the risk of slipping in and out of mitosis, helping to maintain genomic stability5,29. This regulation could be owing to the higher concentrations of CDK regulators in the nucleus such as Wee1 and Cdc25 (refs. By contrast, the less well stabilized CDK circuit and lower cyclin B threshold in the cytoplasm allows cytoplasmic CDK to respond rapidly to the translocation of cyclin B–CDK from the nucleus. Therefore, the cytoplasm may exist in a more fluid state, where the distinction between interphase and mitosis is reduced. Theoretical modelling of the metazoan mitotic network7 has proposed differences between the nucleus and cytoplasm in bistable responses and cyclin B thresholds similar to those that have been shown here experimentally. Bistability is a common characteristic of biological signalling networks and cell transitions6,53,54, and we have shown that the responses can be very different between cellular compartments. Modelling of the mammalian mitotic network has suggested that nuclear envelope breakdown (NEBD) creates a potential ‘stress' at mitotic entry55, owing to dispersion of cyclin B1–Cdk1 and Greatwall kinase, resulting in mitotic collapse—a collapse that is prevented by Cdk1 -Y15 feedback loops, which retain high CDK activity55. The nuclear export of cyclin B–CDK observed here in fission yeast can potentially lead to a similar stress in the nucleus. This is also prevented by CDK Y15 feedback loops which help to retain high CDK activity, ensuring that nuclear mitotic events such as chromosome condensation and mitotic spindle formation continue. By contrast, the cytoplasm is ill-suited for propagating CDK activity because of its weaker bistable response. We propose that the strong bistable response in the nucleus allows propagation of CDK activity from the nucleus to the cytoplasm without collapse of nuclear CDK activity. Although the fission yeast cyclin B–CDK(T14A/Y15F) (CDKAF) is viable, it exhibits sluggish mitotic progression and aberrant timing of mitotic entry, and generates a higher frequency of inviable cells36,42. The fission yeast CDK regulatory framework can be expected to be relevant to other eukaryotes. The Xenopus extract studies proposing that the nucleus is the ‘pacemaker' for the cell cycle33,34 suggest that our observations are not simply due to differences between a closed versus open mitosis. In human cells, the mechanism behind cyclin B1 nuclear translocation before NEBD is thought to be due to either Cdk1 phosphorylation of transport machinery or through a spatial positive feedback mechanism based on cyclin B1 autophosphorylation whereby the nucleus promotes the translocation of cyclin B116,31. This translocation is thought to be important for triggering NEBD17. In fission yeast, the translocation of Cdc13–Cdc2(WT) from the nucleus to the cytoplasm could be functionally anagolous to translocation of cyclin B1–Cdk1 triggering NEBD. Therefore, although the precise mechanisms may lead to specific differences, the general concepts of the nucleus acting as the mitotic pacemaker and translocation being an important aspect for the onset of mitosis may be conserved16,33,34. Why does the initial mitotic CDK activation and strong bistable or hysterisis response occur in the nucleus? A major vulnerability of the mitotic cell cycle is the risk of premature CDK activation and mitotic onset when DNA is incompletely replicated or has unrepaired damage56. Having the CDK regulatory system within the nucleus, positions the bistable circuit close to DNA, so incomplete DNA replication and DNA damage can be readily integrated with control of CDK activity57,58. Such a tight association between DNA and CDK activation would help to ensure genome integrity. It also enables genome content (ploidy) to be more easily monitored, which is necessary given that cell size at mitosis increases with ploidy42. Thus, DNA may serve as a platform for control of CDK activity and the initiation of mitotic entry42,57. The principles we that have developed here will be a useful guide for future experiments in the more complex metazoan cells for investigating how CDK brings about cell cycle progression. Furthermore, the signalling network in the CDK oscillator system has been shown to have similar features to other biological oscillators, such as bistability and positive feedback loops59. Therefore, our work here should also be useful for understanding how other biological oscillators operate across space and time in vivo. Sensors were expressed under the low expression constitutive calmodulin (cam1) promoter60. The sensors were tagged with either mNeonGreen or mScarletI61,62. NucCDK was derived from a minimal budding yeast CDK-dependent transport module, which was shown to be exported out of the nucleus after phosphorylation by Cdc28 (budding yeast Cdk1)63 (Fig. This sensor functions similarly to a CDK2 translocation-based sensor, based on a fragment of human DNA helicase B fragment (DHB), which translocates to the cytoplasm upon CDK2 phosphorylation64. The MCM2-3 sensor module displayed a readout patten that increased at S phase, similar to that of early CDK substrates, consistent with observations in budding yeast11,63 (Extended Data Fig. Although this version of the sensor is useful for the G1/S transition given its high sensitivity to CDK activity65, we aimed to reduce its sensitivity so that it can be used to read out activity at G2/M. The module contains a full CDK consensus site beginning with a serine, next to the Mcm3 NLS. Mutation of this residue to alanine (MCM2-3 Ala) reduced the readout of the sensor and rendered it prominently nuclear, albeit with a modest increase after stoppage of cell elongation (Extended Data Fig. Therefore, we hypothesized that mutating this serine to a threonine could alter the sensitivity of the sensor as it has been shown that CDK has a greater specificity towards serine residues than threonine residues, and, furthermore, that threonines are preferentially dephosphorylated by PP2A66,67. As shown in Extended Data Fig. 1e, this mutation markedly altered the profile of the sensor, and gave a readout of CDK activity throughout the cell cycle. We refer to this sensor as NucCDK. The profile was very similar to a recent CDK Forster resonance energy transfer sensor tagged with a NLS, developed in fission yeast, indicating that NucCDK is biased towards sensing nuclear CDK activity68. We did not notice a significant difference in the pattern when using a nuclear mask for determining the cytoplasmic to nuclear ratio in mean intensities, as opposed to using the mean of top 15% of pixels and bottom 85% of pixels as estimates for nuclear and cytoplasmic mean intensities, respectively, as has been previously used in fission yeast46 (Extended Data Fig. Unless stated that a nuclear mask was used, the top 15% and bottom 85% method was used for sensor readout measurements. Syncut3 (CytCDK) translocation into the nucleus is dependent on phosphorylation of T19 by CDK42. CytCDK V2 was developed in a similar manner to NucCDK except T19 was mutated to a serine, for increased sensitivity to CDK activity. The timecourse analysed for Fig. 1a was from the Cdc13–Cdc2(WT) fusion experiment in which Cig1, Cig2 and Puc1 had been deleted (∆CCP), and cells grown in EMM4S + SILAC (stable isotope labelling with amino acids in cell culture) medium at 32 °C (ref. Localization annotation for CDK sites was implemented as previously described39. In brief, the order of determining the localization of a protein by priority based on existing literature was as follows: (1) direct visualization of fluorescently tagged proteins labelled at the endogenous locus; (2) fluorescently tagged proteins from exogenous nmt41 or nmt81 promoters; (3) indirect methods suchs as chromatin immunoprecipitation assays, fractionated western blotting and immunoprecipation experiments. Only sites annotated as being late sites as well as belonging to solely one spatial compartment were used for analysis. Strains were grown, and cell cultures and media were prepared as described69. Strains were constructed either by transformation or genetic crossing, as described69. All experiments were done in exponentially growing cells at 2.5–4.0 × 106 cells per ml. Insertions were checked with colony PCR and sequencing in the case of deletions. All strains and plasmids used are listed in Supplementary Table 1. EMM powder (MP Biomedicals) with ammonium chloride and dextrose was used, supplemented with adenine at final concentrations of 0.15 g l−1 where required. Where yeast extract with adenine, leucine, histidine and uridine (YE4S) was used, supplements were added at 0.15 g l−1 final concentrations. The Cdc13 strain internally tagged with sfGFP is described in ref. Except for some data in Figs. 2 and 4, microscopy was done on 1% agarose pads, using GeneFrames, to prevent drying of pad during acquisition71 (125 μl, Thermofisher AB-0578). N-propyl gallate (Sigma, 02370) was added to the agarose pads at 0.1 mM final concentration to prevent photobleaching72. A 1-ml culture of cells was spun down once at 2,000 rpm (Eppendorf Centrifuge 5424) for 30 s, to minimize stress. Approximately 50 μl of supernatant was left and mixed. Two microlitres of cell culture was spread across an agarose pad and left to dry for a few minutes. A coverslip (Epredia, 22 × 32 mm, no. 1.5) was placed onto the pad, but only pressure along the gene frame was applied to seal up the frame. Nail polish was added along the frame to ensure that the coverslip and frame were sealed. Imaging was done on a Nikon Ti2 inverted microscope equipped with a perfect focus system (PFS), an Okolab environmental chamber, and a Prime BSI sCMOS camera (Photometrics). A Nikon 100x Plan Apo oil immersion lens with a NA of 1.45 was used for imaging. Image acquisition was controlled using µManager V2.0 software73. 2 × 2 pixel binning was done to improve fluorescence signal detection with a resulting pixel size of 130 nm × 130 nm. Unless stated otherwise, imaging was done at 25°. Fluorescence excitation was done using a SpectraX LED light engine (Lumencor). Imaging for mNG-tagged proteins was done with the YFP channel (both in excitation and emission) to minimize the influence of cell autofluorescence74. We did not notice any issues with growth or cell length at division under the imaging conditions used. Cells contained a copy of Cdc13WT or Cdc13HPM expressed under the native Cdc13 promoter and a second copy of Cdc13WT, expressed under the thiamine-repressible nmt41 promoter39. Cells were initially grown in EMM without thiamine but for experiments, cells were washed once in EMM + thiamine and placed on a 1% EMM + thiamine agarose pad. Thiamine was added to EMM at a final concentration of 30 μM. Only cells that were born around the beginning of acquisition and blocked at the end of acquisition were considered for analysis. The total acquisition was 80 time points with time intervals of 10 min, for around 13 h in total in acquisition. Unless otherwise stated, all analysis was done using custom scripts written in Matlab R2022a. For processing and analysis of fluorescence z-stacks, maximum intensity projections were used using Fiji75. Cells were segmented using a custom algorithm as described52. Nuclear masks were obtained with Fiji, using MCM2-3 Ala to identify the nucleus. With the exception of cases where mutants that block cells in G2 were used, only cells that were born and that divided within the timelapse acquisition period were used for analysis. For histograms, the left edges of bins are inclusive, while the right edges are not inclusive except for the last bin. Bin widths were set to the time interval used for image acquisition, with the exception of Fig. Violin plots display distributions as kernel density estimates. The median value is plotted as a white circle. For all other figures, points representing CDK activation were identified using an automated rate change analysis algorithm. Sensor traces were lightly smoothed using the Matlab function, smoothdata and the algorithm sgolay to assist in identifying change points. However, plots in figures represent the raw data. To confine analysis to the period in G2 where the rate change in sensor readout occurs, the trace was cropped to 10–15 time points before the maximum peak in sensor readout, up to the maximum peak. The abrupt changes in sensor readout were identified using a change point algorithm77,78 in Matlab called findchangepts, where abrupt changes in the slope were identified using the specifier ‘linear'. A similar process was followed to identify points of stoppage of cell elongation. For the phosphoproteomics analysis to determine statistical significance of nuclear substrate phosphorylation compared to the other compartments, the rate change analysis was similarly performed on individual sites. We used an analogue-sensitive sensitive version of Cdc2, Cdc2as, that renders it susceptible to inhibition by the ATP analogue 1-NmPP179, to determine how quickly the sensors respond to changes in CDK activity. Experiments were performed on soybean lectin-coated MatTec dishes in YE4S medium at 32 °C, with 3-min intervals42. Cells were blocked in G2 for 3 h using 1 μM 1-NmPP1 and released following 3× washes using YE4S medium. Subsequently, either 10 μM 1-NmPP1 was added to the dish to inhibit CDK activity or DMSO (control) was added. The mean decay curves were fitted using either a two-exponential model (NucCDK) or a one-exponential model (CytCDK) to determine the half-lives of decay. Dose–response curves of NucCDK and CytCDK/CytCDK V2 to CDK inhibition were done similarly as described11,42. We used the Cdc13–Cdc2AFAs strain to remove potential non-linear responses in phosphorylation of the sensors due to CDKY 15 feedback loop activation. Cells were blocked at 32 °C in YE4S for 1.5 generations. They were washed (3×) and released into release medium containing varying concentrations of 1-NmPP1. Image acquisition was done after 12 min from the first wash on microscope slides. The mNG tag was used at the C terminus of the Cdc13–Cdc2(WT) fusion, as it has better photophysical properties compared with sfGFP and allowed use of the YFP channel, which is less affected by cellular autofluorescence61,74. 4 were done using the CellASIC ONIX V1 microfluidics platform with Y04C microfluidics plates (Merck) at 32 °C in YE4S, as they are the optimal conditions used for Cdc13–Cdc2AF80. All 4 experimental conditions—Cdc13–Cdc2(WT)–mNG + (NucCDK–mScarletI or CytCDK V2–mScarletI) and Cdc13–Cdc2AF–mNG + (NucCDK–mScarletI or CytCDK V2–mScarletI)—were done on the same plate using a flow rate of 2.5 pound per square inch (psi). Fifty microlitres of culture at an optical density of 0.2–0.3 was added to plate. Images were acquired every 5 min. Phase plots shown in figures were plotted after lightly smoothing the raw data using the matlab function smoothdata with the algorithm sgolay to reduce noise and improve the quality of the phase plots. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All strains and plasmids available upon request. Owing to the size of the timelapse imaging datasets, we have stored these at the Francis Crick Institute and have made them available upon request by contacting the authors. Matlab scripts used for image and data analysis can be found at: https://github.com/nkapadia27/Spatiotemporal-Orchestration-of-Mitosis and https://doi.org/10.5281/zenodo.11072087 (ref. Grallert, A. et al. Centrosomal MPF triggers the mitotic and morphogenetic switches of fission yeast. Hachet, V., Canard, C. & Gonczy, P. Centrosomes promote timely mitotic entry in C. elegans embryos. Active cyclin B1–Cdk1 first appears on centrosomes in prophase. Numerical analysis of a comprehensive model of M-phase control in Xenopus oocyte extracts and intact embryos. & Ferrell, J. E. Jr. Systems-level dissection of the cell-cycle oscillator: bypassing positive feedback produces damped oscillations. Pomerening, J. R., Sontag, E. D. & Ferrell, J. E. Jr. Building a cell cycle oscillator: hysteresis and bistability in the activation of Cdc2. & Gelens, L. Dynamic bistable switches enhance robustness and accuracy of cell cycle transitions. Sha, W. et al. Hysteresis drives cell-cycle transitions in Xenopus laevis egg extracts. Dantas, M., Lima, J. T. & Ferreira, J. G. Nucleus–cytoskeleton crosstalk during mitotic entry. Nurse, P. Universal control mechanism regulating onset of M-phase. Swaffer, M. P., Jones, A. W., Flynn, H. R., Snijders, A. P. & Nurse, P. CDK substrate phosphorylation and ordering the cell cycle. Hegarat, N. et al. Cyclin A triggers mitosis either via the greatwall kinase pathway or cyclin B. EMBO J. Lindqvist, A., Rodriguez-Bravo, V. & Medema, R. H. The decision to enter mitosis: feedback and redundancy in the mitotic entry network. Deibler, R. W. & Kirschner, M. W. Quantitative reconstitution of mitotic CDK1 activation in somatic cell extracts. The roles of cyclin A2, B1, and B2 in early and late mitotic events. & Pines, J. Activation of cyclin B1–Cdk1 synchronizes events in the nucleus and the cytoplasm at mitosis. Progressive activation of cyclinB1–Cdk1 coordinates entry to mitosis. Cellular substrates of p34(cdc2) and its companion cyclin-dependent kinases. Welburn, J. P. et al. How tyrosine 15 phosphorylation inhibits the activity of cyclin-dependent kinase 2-cyclin A. J. Biol. Murray, A. W. & Kirschner, M. W. Cyclin synthesis drives the early embryonic cell cycle. Strausfeld, U. et al. Dephosphorylation and activation of a p34cdc2/cyclin B complex in vitro by human CDC25 protein. Russell, P. & Nurse, P. Negative regulation of mitosis by wee1+, a gene encoding a protein kinase homolog. B., McGowan, C. H., Lenaers, G., Jones, R. & Russell, P. p80cdc25 mitotic inducer is the tyrosine phosphatase that activates p34cdc2 kinase in fission yeast. Parker, L. L. & Piwnica-Worms, H. Inactivation of the p34cdc2–cyclin B complex by the human WEE1 tyrosine kinase. & Ferrell, J. E. Jr. Substrate competition as a source of ultrasensitivity in the inactivation of Wee1. & Ferrell, J. E. Jr. Ultrasensitivity in the regulation of Cdc25C by Cdk1. Hoffmann, I., Clarke, P. R., Marcote, M. J., Karsenti, E. & Draetta, G. Phosphorylation and activation of human cdc25-C by cdc2–cyclin B and its involvement in the self-amplification of MPF at mitosis. McGowan, C. H. & Russell, P. Cell cycle regulation of human WEE1. Rapid cycling and precocious termination of G1 phase in cells expressing CDK1AF. & Hunter, T. Human cyclins A and B1 are differentially located in the cell and undergo cell cycle-dependent nuclear transport. Santos, S. D., Wollman, R., Meyer, T. & Ferrell, J. E. Jr. Spatial positive feedback at the onset of mitosis. Arquint, C., Gabryjonczyk, A. M. & Nigg, E. A. Centrosomes as signalling centres. Nuclei determine the spatial origin of mitotic waves. Afanzar, O., Buss, G. K., Stearns, T. & Ferrell, J. E. Jr. Heald, R., McLoughlin, M. & McKeon, F. Human wee1 maintains mitotic timing by protecting the nucleus from cytoplasmically activated Cdc2 kinase. Coudreuse, D. & Nurse, P. Driving the cell cycle with a minimal CDK control network. Booher, R. N., Alfa, C. E., Hyams, J. S. & Beach, D. H. The fission yeast cdc2/cdc13/suc1 protein kinase: regulation of catalytic activity and nuclear localization. Lee, M. G. & Nurse, P. Complementation used to clone a human homologue of the fission yeast cell cycle control gene cdc2. The hydrophobic patch directs cyclin B to centrosomes to promote global CDK phosphorylation at mitosis. Nurse, P., Thuriaux, P. & Nasmyth, K. Genetic control of the cell division cycle in the fission yeast Schizosaccharomyces pombe. Patterson, J. O., Basu, S., Rees, P. & Nurse, P. CDK control pathways integrate cell size and ploidy information to control cell division. Kamenz, J., Gelens, L. & Ferrell, J. E. Jr. Bistable, biphasic regulation of PP2A-B55 accounts for the dynamics of mitotic substrate phosphorylation. Chica, N. et al. Nutritional control of cell size by the greatwall–endosulfine–PP2A·B55 pathway. Mochida, S., Maslen, S. L., Skehel, M. & Hunt, T. Greatwall phosphorylates an inhibitor of protein phosphatase 2A that is essential for mitosis. Curran, S., Dey, G., Rees, P. & Nurse, P. A quantitative and spatial analysis of cell cycle regulators during the fission yeast cycle. Removal of centrosomal PP1 by NIMA kinase unlocks the MPF feedback loop to promote mitotic commitment in S. pombe. MacIver, F. H., Tanaka, K., Robertson, A. M. & Hagan, I. M. Physical and functional interactions between polo kinase and the spindle pole component Cut12 regulate mitotic commitment in S. pombe. Mulvihill, D. P., Petersen, J., Ohkura, H., Glover, D. M. & Hagan, I. M. Plo1 kinase recruitment to the spindle pole body and its role in cell division in Schizosaccharomyces pombe. The role of Plo1 kinase in mitotic commitment and septation in Schizosaccharomyces pombe. Nurse, P. Genetic control of cell size at cell division in yeast. Roberts, E. L. et al. CDK activity at the centrosome regulates the cell cycle. Ferrell, J. E. & Xiong, W. Bistability in cell signaling: how to make continuous processes discontinuous, and reversible processes irreversible. & Novak, B. Molecular mechanisms creating bistable switches at cell cycle transitions. Tuck, C., Zhang, T., Potapova, T., Malumbres, M. & Novak, B. Robust mitotic entry is ensured by a latching switch. Cell death by mitotic catastrophe: a molecular definition. Lemmens, B. et al. DNA replication determines timing of mitosis by restricting CDK1 and PLK1 activation. Matsuyama, M. et al. Nuclear Chk1 prevents premature mitotic entry. Robust, tunable biological oscillations from interlinked positive and negative feedback loops. & Yoshida, M. A series of promoters for constitutive expression of heterologous genes in fission yeast. Shaner, N. C. et al. A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum. Bindels, D. S. et al. mScarlet: a bright monomeric red fluorescent protein for cellular imaging. Liku, M. E., Nguyen, V. Q., Rosales, A. W., Irie, K. & Li, J. J. CDK phosphorylation of a novel NLS–NES module distributed between two subunits of the Mcm2-7 complex prevents chromosomal rereplication. The proliferation–quiescence decision is controlled by a bifurcation in CDK2 activity at mitotic exit. Ord, M. et al. Proline-rich motifs control G2–CDK target phosphorylation and priming an anchoring protein for polo kinase localization. Godfrey, M. et al. PP2A(Cdc55) phosphatase imposes ordered cell-cycle phosphorylation by opposing threonine phosphorylation. Ord, M. et al. Multisite phosphorylation code of CDK. Sugiyama, H., Goto, Y., Kondo, Y., Coudreuse, D. & Aoki, K. Live-cell imaging defines a threshold in CDK activity at the G2/M transition. & Nurse, P. Molecular genetic analysis of fission yeast Schizosaccharomyces pombe. Kamenz, J., Mihaljev, T., Kubis, A., Legewie, S. & Hauf, S. Robust ordering of anaphase events by adaptive thresholds and competing degradation pathways. Kapadia, N. et al. Processive activity of replicative DNA polymerases in the replisome of live eukaryotic cells. & Pollard, T. D. High-speed super-resolution imaging of live fission yeast cells. Edelstein, A. D. et al. Advanced methods of microscope control using muManager software. Davidson, R., Liu, Y., Gerien, K. S. & Wu, J. Q. Real-time visualization and quantification of contractile ring proteins in single living cells. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Chalfoun, J. et al. Lineage mapper: a versatile cell and particle tracker. A. Optimal detection of changepoints with a linear computational cost. Lavielle, M. Using penalized contrasts for the change-point problem. Bishop, A. C. et al. A chemical switch for inhibitor-sensitive alleles of any protein kinase. & Nurse, P. Identification of mutants with increased variation in cell size at onset of mitosis in fission yeast. The authors thank T. Zeisner, J. Curran, and J. Greenwood for comments on the manuscript. This work was supported by the Francis Crick Institute, which receives core funding from Cancer Research UK (CC2003), the UK Medical Research Council (CC2003), and the Wellcome Trust (CC2003). For the purpose of Open Access, the author has applied a CC•BY copyright licence to any author accepted manuscript version arising from this submission. In addition, this work was supported by a Wellcome Trust grant to P.N. (grant number 214183) and the Breast Cancer Research Foundation (BCRF-23-117). was supported by an EMBO postdoctoral fellowship (EMBO ALTF 705-202) and a HFSP postdoctoral fellowship (HFSP LT000587/2021-L). Open Access funding provided by The Francis Crick Institute. Cell Cycle Laboratory, The Francis Crick Institute, London, UK Laboratory of Yeast Genetics and Cell Biology, Rockefeller University, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar N.K designed and performed all the experiments. generated the strains and plasmids. N.K performed the image and data analysis including writing custom scripts. The authors declare no competing interests. Nature thanks James Ferrell Jr and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. b) left) Plot of IC50 value vs rate change point after G2 release (minutes). Spatial compartments are colour coded as legend shows. Right) Plot as left but scaled to show IC50 values in the range of 20 nM to 400 nM. c) Top: Traces of the original MCM2-3 sensor tagged with mScarletI, using the cytoplasmic to nuclear ratio as a measure of sensor readout, taken every 10 min, in EMM media. The mean top 15% of pixels were used as a measure for mean nuclear intensity, while the mean of bottom 85% of pixels was used as a measure for mean cytoplasmic intensity. n = 35 cells; 1 of 3 biological repeats. d) Top: MCM2-3-Ala-mScarletI traces, taken every 5 min, in EMM media. Nuclear masks were obtained using MCM2-3-Ala-mScarletI. Y-axis scaled to MCM2-3 readout, shown in a). Bottom: Representative time trace of MCM2-3-Ala. e) NucCDK traces, where MCM2-3-Ala-mNG was used as a nuclear mask, taken every 5 min in EMM media. n = 42 cells; 1 of 3 biological repeats. f) NucCDK time traces where the top 15% method was used with representative time trace (same cell as Fig. 2c showing time delays at different points. b) Histogram of the delay from nuclear activation to cytoplasmic activation using NucCDK-mScarletI and CytCDK-mNG with 3-min intervals in YE4S. n = 40 cells; 1 of 3 biological repeats. c) Schematic of block and release experiments. d) Representative trace in Cdc13-Cdc2As background. 10 µM 1-NmPP1 was added after 8 time points. 3-min delay refers to the delay when readouts begin to drop (shown with dotted lines). e) Decay curves of meaned data after CDK inhibition (n = 24 cells; 1 of 2 biological repeats). NucCDK was fit with a two-exponential model to correct for slight rise in readout. Estimated half life with 95% confidence intervals are shown in brackets. f) Mean readouts of sensors in Cdc13-Cdc2AFAs cells released into different concentrations of 1-NmPP1. g) same as f except using CytCDK V2. h) Representative traces of CytCDK V2. n = 30 cells; 1 of 3 biological repeats. i) Representative trace in Cdc13-Cdc2AFAs background. 10 µM 1-NmPP1 was added after 11 time points. j) Decay curves of meaned data after CDK inhibition (n = 42 cells; 1 of 2 biological repeats). Estimated half life with 95% confidence intervals are shown in brackets. k) Histogram of the delay from nuclear activation to cytoplasmic activation using the sensors, NucCDK-mScarletI and CytCDK-mNG, in a ppk18∆ background, using 5 min intervals in YE4S. n = 66 cells; 1 of 3 biological repeats. a) top) Montage of Cdc13-sfGFP showing nuclear export. Images taken every 5 min in EMM media, and each image represents 5 min. Contrast was adjusted for better visualization of export. Red arrows mark cells where export was visually seen. Bottom) same as top but with no contrast adjustment. b) Single cell traces showing nuclear (top) and cytoplasmic (bottom) mean intensities. MCM2-3-Ala-mScarletI was used as a nuclear mask. n = 45 cells; 1 of 4 biological repeats. c) Representative trace of nuclear and cytoplasmic mean intensities using top 15% and bottom 85% of pixels, respectively. Same cell as in Fig. d) top) Montage of Cdc2-mNG taken every 5 min, in EMM media, showing two cells going through nuclear export. Contrast was adjusted for better visualization of export. Bottom) Same as top but with no contrast adjustment. e) Histogram of nuclear activation using NucCDK-mScarletI, with respect to Cdc2-mNG nuclear export. Data was collected every 5 min. n = 45 cells; 1 of 3 biological repeats. f) Background control of imaging in GFP channel with MCM2-3-Ala-mScarletI under same conditions used in Fig. White dot represents median value, black solid rectangle represents the IQR (bounded by the 25th and 75th percentiles), and black solid line represents the whiskers bounded by the minimum and maximum defined as data points within 1.5x IQR values. n = 33 cells; 1 of 2 biological repeats. g) Phase plots in nucleus and cytoplasm, plotted on the same phase space. Sensor readouts were min max normalized for comparison. 5 cell traces are shown for each cellular compartment. a) Single-cell traces of Cdc13-Cdc2-mNG mean intensities in nucleus (top) and cytoplasm (bottom). n = 89 cells; 1 of 3 biological repeats. b) Representative trace of Cdc13-Cdc2-mNG mean intensities in the nucleus and cytoplasm. c) Representative trace of NucCDK-mScarletI in Cdc13-Cdc2-mNG background. d) Representative trace of CytCDK V2-mScarletI in Cdc13-Cdc2-mNG background. e) Single-cell traces of Cdc13-Cdc2AF-mNG mean intensities in nucleus (top) and cytoplasm (bottom). n = 24 cells; 1 of 3 biological repeats. f) Representative trace of Cdc13-Cdc2AF-mNG mean intensities in the nucleus and cytoplasm. g) Single-cell traces of NucCDK sensor readout in Cdc13-Cdc2AF-mNG background. n = 24 cells; 1 of 3 biological repeats. h) Single cell traces of CytCDK V2 sensor readout in Cdc13-Cdc2AF-mNG background. n = 49 cells; 1 of 3 biological repeats. i) Min max normalized phase plots of the nucleus in the Cdc13-Cdc2AF-mNG background. j) Min max normalized phase plots of the cytoplasm in the Cdc13-Cdc2AF-mNG background. a) Single-cell traces of NucCDK readout in Cdc13WT-sfGFP background, in EMM + thiamine to repress the second copy of Cdc13WT. Images were acquired every 10 min. n = 29 cells; 1 of 3 biological repeats. b) Single-cell traces of CytCDK readout in Cdc13WT-sfGFP in EMM + thiamine to repress the second copy of Cdc13WT. Images were acquired every 10 min. n = 58 cells; 1 of 3 biological repeats. c) Single-cell traces of Cdc13-sfGFP mean nuclear intensity in Cdc13WT-sfGFP background, where the second copy of Cdc13WT was repressed. e) Single-cell traces of NucCDK in Cdc13HPM-sfGFP background. n = 36 cells; 1 of 3 biological repeats. f) Traces of CytCDK readout in Cdc13HPM-sfGFP background. Y-axis scaled based on minimum and maximum sensor readout in Cdc13WT -sfGFP background shown in b). n = 23 cells; 1 of 3 biological repeats. Sheet 1: S. pombe strains used in the study and their genotype. Sheet 2: Plasmids used to construct strains in the study. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Kapadia, N., Nurse, P. Spatiotemporal orchestration of mitosis by cyclin-dependent kinase. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65161270/gwada-negative-48-blood-group/'>Scientists Have Discovered a New Blood Type. Only One Person on Earth Has It.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 14:36:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Named “Gwada negative,” it originated from a blood sample taken 15 years ago. Most people are familiar with the four broad human blood types—A, B, AB, and O—and it's likely they may even know that they come in positive or negative varieties (an indicator of what's called an Rh factor) as well. In fact, humans theoretically contain millions of different blood types, as there are more than 600 antigens that have been identified on human blood cells. Any combination of those antigens could be considered a new blood type. However, the International Society of Blood Transfusion (ISBT) only recognizes 47 distinct blood type groups, with the latest being added just last year after a 50-year-long mystery. Now, according to a press statement published on LinkedIn by the French Blood Establishment (EFS), a 48th member has officially joined the party. Some 15 years ago, researchers received blood from a French woman originating from the Caribbean island of Guadeloupe, colloquially referred to as “Gwada.” The patient drew blood while undergoing routine tests before surgery. Its immensely important to know the antigen make-up of your blood before undergoing blood transfusions, as mismatch can trigger a life-threatening immune response. Although there are potentially millions of blood types, transfusions made with blood that matches your type according to the well-established ABO-Rh type system have a 99.8 percent chance of success. This is because your body's immune response doesn't attack all antigens equally, so even if there are some differences, the transfusion will likely be a success. However, it's still important to identify these ultra-rare blood types so that life-threatening immune responses can be avoided. “Discovering new blood groups means offering patients with rare blood a better level of care,” the EFS team said in a press statement. Advancements in gene sequencing have supercharged the discovery of these blood groups. For example, the 44th member of this group, known simply as “Er,” was discovered in 2022, meaning that scientists are currently discovering more than one new blood group a year. 2025 might hold one or two more blood-based surprises of its own. A New Study Says You Can Delay Death—but Not Aging</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/trump-administration-ousts-national-science-foundation-from-headquarters/'>Trump Administration Ousts National Science Foundation from Headquarters Building</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 14:25:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Trump Administration Ousts National Science Foundation from Headquarters Building Employees at the National Science Foundation say they've been blindsided by a plan for the Department of Housing and Urban Development to take over their offices But as of Tuesday evening, staff at the science foundation hadn't been informed by management about their building's incoming occupants, leaving them feeling blindsided and unsure about where they're expected to work. One NSF employee said that they had “literally zero idea” the move was coming until reports began circulating among staffers Tuesday evening. That person was granted anonymity because they fear retaliation. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Jesus Soriano, president of the union that represents NSF employees, said he was expecting a press conference Wednesday morning in the NSF lobby including HUD Secretary Scott Turner and Virginia Republican Gov. Soriano said he was informed about the plans by NSF employees. Soriano sent an alert to union members Tuesday evening informing them that NSF's management “learned this afternoon [from HUD]” that the Wednesday news conference would include an announcement that “HUD will take over the NSF building” and that the science agency was not involved in the decision. “There is no planning for NSF, no identified future location, appropriation for a new building or a move,” he wrote. HUD and the General Services Administration announced in April that HUD wanted to move out of its current location in the Robert C. Weaver Federal Building near L'Enfant Plaza in Washington. That building, which opened in 1968, faces over $500 million in deferred maintenance and modernization needs, GSA said. The building would be at half of its capacity with every HUD staffer at headquarters, according to GSA. An NSF spokesperson directed a request for comment to GSA. GSA, the White House and a Youngkin spokesperson did not immediately respond to requests for comment. “Is the government more interested in the technological and scientific leadership of this country or on using taxpayer dollars to offer luxury accommodations to government officials?” Soriano said. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250625075018.htm'>Mammals didn't walk upright until late—here's what fossils reveal</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 14:17:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For over a century, scientists have puzzled over a fundamental mystery in our evolutionary history: how did mammals go from sprawling like lizards to striding like cats and dogs? Now, a groundbreaking study in PLOS Biology led by Dr. Robert Brocklehurst, a former postdoctoral fellow in the Department of Organismic and Evolutionary Biology (OEB) at Harvard University, offers a surprising answer: the path to upright posture wasn't linear, but full of unexpected detours, evolutionary experimentation, and dramatic anatomical upheaval. "However, what we discovered was a more nonlinear evolutionary progression throughout mammalian history." All mammals -- from bats and whales to moles and humans -- share a distinctive way of moving: they hold their limbs underneath their bodies, unlike the sprawling posture with limbs out to the side. This dramatic transition was also accompanied by big changes in limb bone shape and mechanics. To assess these changes, the team analyzed the humerus (upper arm bone) of over 60 non-mammalian synapsid fossils and 140 living animals, including mammals, reptiles and amphibians. Using a novel analytical technique, pioneered in senior author Professor Stephanie Pierce's lab (also in OEB), researchers mapped each bone's surface to measure traits like length, mass distribution, muscle leverage, and torsion (the degree to which the bone twists along its length). "By correlating bone shape and limb biomechanics with posture, we could test how well the fossil bones were optimized for specific functional tasks, like upright walking versus sprawled walking," Brocklehurst said. The researchers achieved this by mapping the fossil non-mammalian synapsids onto a functional adaptive landscape, similar to a topographic map, with peaks and valleys that relate to high and low performance of different locomotor postures. "We expected to see a neat progression -- from sprawling pelycosaurs to a bit more upright therapsids, then cynodonts, then fully upright mammals," said Brocklehurst. One fossil, a close relative of today's marsupials and placentals, showed bone features consistent with a modern upright gait, suggesting that fully parasagittal postures evolved relatively late in mammalian history as opposed to previously held hypotheses. "Our work challenges the idea that posture changed gradually and early on," said Pierce, "instead, it shows that upright posture and locomotion were a late evolutionary innovation, not an early defining trait of the mammalian lineage." "Our study showed that most synapsid limbs functioned differently than those of modern reptiles. They're not just copies of reptiles, but distinctive animals in their own right that are a little different from anything that's alive today," says co-author Kenneth Angielczyk of Chicago's Field Museum. Traditional methods that describe shape in similar structures didn't work. So, the team re-engineered an existing R software package designed for a different task, transforming it into a novel "slice-based" landmarking tool tailored for this study. The study builds on a rich scientific legacy -- both at Harvard and in paleontology. Pierce, who is also Curator of Vertebrate Paleontology in the Museum of Comparative Zoology (MCZ), noted that, "Researchers and former MCZ curators, like Alfred Sherwood Romer and Farish Jenkins, Jr., were grappling with these same questions a century ago. This study marks the first large-scale evolutionary analysis of mammalian posture using quantitative biomechanics. The team is now building detailed models of forelimbs in select fossil species to understand how joints and muscles functioned in ancient animals, offering even deeper insights into the evolution of mammalian motion. As Brocklehurst put it: "Understanding how mammals came to walk upright isn't just about bones, it's about uncovering the dynamic history of life on Earth." Materials provided by Harvard University, Department of Organismic and Evolutionary Biology. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a65162811/methane-sea-spiders/'>3,000 Feet Beneath the Ocean, Scientists Found Spiders With a Hidden Superpower</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This previously unknown symbiotic relationship helps keep methane—a major greenhouse gas—trapped in the ocean. When you live in the ocean's aphotic zone, where absolute darkness reigns, you need some creative solutions to some pretty existential questions. Chief among them: what are you going to eat? For a variety of ocean floor-dwelling species, including mussels, worms, and sponges, that answer lies with chemosynthetic bacteria—organisms that flourish around methane seeps where the gas escapes the Earth's crust. A new study shows that three species of the sea spider genus  Sericosura also demonstrate a similar ability, which helps explain why these spindly-legged organisms are found in such abundance near methane seeps. Although they look remarkably similar, sea spiders have a thinner cephalothorax than their eight-legged land-based lookalikes, and their legs tend to have more joints. This hard-to-reach ecosystem is why many aspects of their lives remain a mystery. In 2023, Goffredi—along with two students, including Bianca Dal Bó—embarked on a two-week research expedition on the R/V Atlantis to study these creatures in their deep-sea habitats. “It made me feel all the more connected to my project and these fascinating ecosystems.” While performing experiments on board, Goffredi and Dal Bó incubated the sea spiders in seawater with methanol and a heavy isotope of methane. “Evidence from tissue isotopic analysis, microbiome sequencing, and live-animal incubations followed by 13C-methane isotope probing confirms active incorporation of methane-derived carbon into spider tissues,” the authors wrote. “This research highlights a previously unknown interaction between an animal lineage and chemically fueled microbes, introducing another symbiotic pathway for direct microbial transfer of methane carbon into animal biomass in the deep sea.” This transfer of methane carbon into animal biomass is particularly important for scientists to understand, as methane is one of the most potent greenhouse gases. “While the deep sea feels really far away, all organisms are interconnected,” Goffredi told CNN. “Even though they're small, these animals have a big impact in that environment. Dinos Had Cancer, Too, According to a New Study A Man Let Deadly Snakes Bite Him For Science</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250625075013.htm'>Vitamin C flips your skin's “youth genes,” reversing age-related thinning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 12:24:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>To combat aging's impact on skin, numerous studies have emphasized the benefits of vitamin C (VC), a vitamin well known for its role in skin health and antioxidant properties. Now, researchers in Japan have discovered that VC helps thicken the skin by directly activating genes that control skin cell growth and development. Their findings, published online in the Journal of Investigative Dermatology on April 20, 2025, suggest that VC may restore skin function by reactivating genes essential for epidermal renewal. Associate Professor Ayami Sato from TMIG (currently at the Toyo University); Associate Professor Yasunori Sato, Professor Toshiyuki Kimura, and Mr. Hideki Tanaka (currently at the University of Fukui Hospital) from Hokuriku University; and Ms. Florence, Ms. Akari Kuwano, Mr. Yasunari Sato, and Mr. Tsuyoshi Ishii from ROHTO Pharmaceutical Co., Ltd also co-authored the study. To investigate how VC affects skin regeneration, the team used human epidermal equivalents, which are laboratory-grown models that closely mimic real human skin. In this model, skin cells are exposed to air on the surface while being nourished from underneath by a liquid nutrient medium, replicating the way human skin receives nutrients from underlying blood vessels while remaining exposed to the external environment. On assessing its effect, they found that VC-treated skin showed a thicker epidermal cell layer without significantly affecting the stratum corneum (the outer layer composed of dead cells) on day seven. Samples treated with VC showed increased cell proliferation, demonstrated by a higher number of Ki-67-positive cells -- a protein marker present in the nucleus of actively dividing cells. Importantly, the study revealed that VC helps skin cells grow by reactivating genes associated with cell proliferation. When DNA is methylated, methyl groups attach to cytosine bases, which can prevent the DNA from being transcribed or read, thereby suppressing gene activity. These enzymes convert 5-methylcytosine (5-mC) into 5-hydroxymethylcytosine (5-hmC), a process in which Fe2+ is oxidized to Fe3+. VC helps maintain TET enzyme activity by donating electrons to regenerate Fe2+ from Fe3+, enabling continued DNA demethylation. The researchers further identified over 10,138 hypomethylated differentially methylated regions in VC-treated skin and observed a 1.6- to 75.2-fold increase in the expression of 12 key proliferation-related genes. When a TET enzyme inhibitor was applied, these effects were reversed, confirming that VC functions through TET-mediated DNA demethylation. These findings reveal how VC promotes skin renewal by triggering genetic pathways involved in growth and repair. This suggests that VC may be particularly helpful for older adults or those with damaged or thinning skin, boosting the skin's natural capacity to regenerate and strengthen itself. "We found that VC helps thicken the skin by encouraging keratinocyte proliferation through DNA demethylation, making it a promising treatment for thinning skin, especially in older adults," concludes Dr. Ishigami. Materials provided by Tokyo Metropolitan Institute for Geriatrics and Gerontology. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/a-sodom-and-gomorrah-story-shows-scientific-facts-arent-settled-by-public/'>A Sodom and Gomorrah Story Shows Scientific Facts Aren't Settled by Public Opinion</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 11:48:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A Sodom and Gomorrah Story Shows Scientific Facts Aren't Settled by Public Opinion Its retraction shows that scientific conclusions aren't decided by majority rule in the public square The Destruction Of Sodom And Gomorrah, painting by John Martin, 1852. This alleged destruction of Tall el-Hammam around 1650 BCE, with reports of melted pottery and mudbricks, pointed to the Bible, the team concluded in Scientific Reports, noting “what could be construed as the destruction of a city by an airburst/impact event.” One study co-author responded to the retraction in an online post with claims that the editor had caved to harassment by skeptics, concluding that the “court of public opinion is much more powerful than a shadowy hatchetman spamming a corrupt editor's inbox.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Public opinion does influence policy decisions and funding priorities in science. People are interested in new medical cures and new starry discoveries, which helps explain why we have a NASA and an NIH. That's why it is important for the public to be scientifically literate and well informed. But scientific facts are determined by the scientific method, logic and evidence, all presented in peer-reviewed publications that require reproducible results. Press releases, rife with references to Sodom and biblical implications, appeared to be focused as much on titillation as on science. A meme, in its original definition, is a self-propagating unit of cultural information that is highly fit in the evolutionary sense. Science by press release can be an effective first step in the creation and laundering of such memes into the public's collective consciousness. The Sodom airburst meme was so successful that it achieved pop culture status and public acceptance within a year of the paper's publication, in this “Final Jeopardy!” question: “A 2021 study suggested that an asteroid that struck the Jordan Valley c. 1650 B.C, gave rise to the story of this city in Genesis 19.” (Winning answer: “What is Sodom?”) I think it is far more likely that it will join the large and growing pantheon of persistent false beliefs, folk facts and urban legends. It won't hurt you to avoid kissing toads, for example. Belief in other scientifically incorrect claims can be extremely dangerous. Avoid vaccinating your children, and you subject them to the risk of serious illness or death. What would it hurt if most people thought that God sent an asteroid to wipe out the people of Sodom, because of their wicked ways? The Old Testament, in Ezekial 16:49-50, says that they were punished because they were “arrogant, overfed and unconcerned; they did not help the poor and needy.” Would it be a bad thing if fear of an asteroid makes us better people? But it could also generate opposition to planetary defense programs to plan for and prevent the impact of an asteroid if we discover one on a collision course. If the majority of people think it's God's will and that we've got it coming, then why shouldn't we just accept our fate? Ultimately, science-informed choices are always the best ones, whether they involve personal decisions about vaccination or public policies for climate change mitigation. When faith inspires people to better themselves, I'm all for that, too. It shouldn't take irrational and unscientific fear of fire and brimstone from an asteroid airburst to make us want to be more humble, kind and generous than the people of Sodom supposedly were. This is an opinion and analysis article, and the views expressed by the author or authors are not necessarily those of Scientific American. Mark Boslough is a physicist who assesses the risk of impacts and airbursts for planetary defense.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41929-025-01359-w'>Operando probing dynamic migration of copper carbonyl during electrocatalytic CO</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 11:37:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Catalysis volume 8, pages 579–594 (2025)Cite this article Single crystals and shape-controlled nanocrystals are well known to exhibit facet-dependent catalytic properties. However, few studies have investigated how those nanocrystals evolve and (de)activate during reactions, calling for the development of nanoscale time-resolved operando methods. In this context, we have designed Cu nanocubes as a model system to elucidate the underlying driving force of dynamic nanocatalyst reconstruction during the CO2 reduction reaction (CO2RR). Operando electrochemical liquid-cell scanning transmission electron microscopy (EC-STEM) and synchrotron-based X-ray spectroscopy reveal the size- and potential-dependent complete transformation from (100)-oriented Cu@Cu2O nanocubes to polycrystalline metallic Cu nanograins under CO2RR conditions. In addition, machine learning-assisted operando four-dimensional STEM reveals that large Cu nanograins derived from nanocubes form mainly crystalline domains, while their smaller counterparts are more amorphous due to faster evolution kinetics. In situ Raman spectroscopy and density functional theory calculations suggest that CO drives the ejection of single Cu atoms, resulting in few-nanometre Cu clusters and the surface migration of highly mobile copper carbonyl (Cu–CO) species. Combined, these multimodal operando methods and theoretical approaches pave the way for understanding the complex structural evolution of energy-related nanocatalysts under electrochemical conditions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All electronic structure files used in the DFT simulations are available in Supplementary Data 1. Raw EC-STEM video datasets are available in Supplementary Videos 1–5. Owing to the terabyte size of the EC-STEM images and 4D-STEM datasets, all relevant TEM data are available from the corresponding authors upon request. Mariano, R. G., McKelvey, K., White, H. S. & Kanan, M. W. Selective increase in CO2 electroreduction activity at grain-boundary surface terminations. Google Scholar Mefford, J. T. et al. Correlative operando microscopy of oxygen evolution electrocatalysts. Google Scholar Yang, Y. et al. Electrocatalysis in alkaline media and alkaline membrane-based energy technologies. Google Scholar Ross, M. B. et al. Designing materials for electrochemical carbon dioxide recycling. Google Scholar Birdja, Y. Y. et al. Advances and challenges in understanding the electrocatalytic conversion of carbon dioxide to fuels. Google Scholar Wu, Y., Jiang, Z., Lu, X., Liang, Y. & Wang, H. Domino electroreduction of CO2 to methanol on a molecular catalyst. Google Scholar Zhu, P. et al. Continuous carbon capture in an electrochemical solid-electrolyte reactor. Google Scholar Yang, Y. et al. Operando studies reveal active Cu nanograins for CO2 electroreduction. Google Scholar Feijóo, J. et al. Operando high-energy-resolution X-ray spectroscopy of evolving Cu nanoparticle electrocatalysts for CO2 reduction. Article PubMed Google Scholar Yang, Y. et al. Operando electrochemical liquid-cell scanning transmission electron microscopy (EC-STEM) studies of evolving Cu nanocatalysts for CO2 electroreduction. Google Scholar Chen, P.-C. et al. Chemical and structural evolution of AgCu catalysts in electrochemical CO2 reduction. Google Scholar Lian, Z., Dattila, F. & López, N. Stability and lifetime of diffusion-trapped oxygen in oxide-derived copper CO2 reduction electrocatalysts. Google Scholar Huang, J. et al. Potential-induced nanoclustering of metallic catalysts during electrochemical CO2 reduction. Google Scholar Grosse, P. et al. Dynamic transformation of cubic copper catalysts during CO2 electroreduction and its impact on catalytic selectivity. Google Scholar Lee, S. H. et al. Oxidation state and surface reconstruction of Cu under CO2 reduction conditions from in situ X-ray characterization. Google Scholar Takahashi, I., Koga, O., Hoshi, N. & Hori, Y. Electrochemical reduction of CO2 at copper single crystal Cu(S)-[n(111)×(111)] and Cu(S)-[n(110)×(100)] electrodes. Google Scholar Hori, Y., Takahashi, I., Koga, O. & Hoshi, N. Selective formation of C2 compounds from electrochemical reduction of CO2 at a series of copper single crystal electrodes. Google Scholar Eren, B., Weatherup, R. S., Liakakos, N., Somorjai, G. A. & Salmeron, M. Dissociative carbon dioxide adsorption and morphological changes on Cu(100) and Cu(111) at ambient pressures. Google Scholar Eren, B. et al. Activation of Cu(111) surface by decomposition into nanoclusters driven by CO adsorption. Google Scholar Xu, L. et al. Formation of active sites on transition metals through reaction-driven migration of surface atoms. Google Scholar Arán-Ais, R. M., Scholten, F., Kunze, S., Rizo, R. & Roldan Cuenya, B. The role of in situ generated morphological motifs and Cu(I) species in C2+ product selectivity during CO2 pulsed electroreduction. Google Scholar Kim, J. et al. Revealing CO2 dissociation pathways at vicinal copper (997) interfaces. Google Scholar Amirbeigiarab, R. et al. Atomic-scale surface restructuring of copper electrodes under CO2 electroreduction conditions. Google Scholar Kim, Y.-G., Baricuatro, J. H., Javier, A., Gregoire, J. M. & Soriaga, M. P. The evolution of the polycrystalline copper surface, first to Cu(111) and then to Cu(100), at a fixed CO2RR potential: a study by operando EC-STM. Google Scholar Kim, Y.-G. et al. Surface reconstruction of pure-Cu single-crystal electrodes under CO-reduction potentials in alkaline solutions: a study by seriatim ECSTM-DEMS. Google Scholar Simon, G. H., Kley, C. S. & Roldan Cuenya, B. Potential‐dependent morphology of copper catalysts during CO2 electroreduction revealed by in situ atomic force microscopy. Google Scholar Zelinski, N. D. De la métallisation des organisms. Google Scholar Boomer, E. H., Morris, H. E. & Argue, G. H. Apparent formation of copper carbonyl. Google Scholar Robinson, P. L. & Stainthorpe, K. R. Two new carbonyls: copper and tellurium. Google Scholar Kösy, F. A volatile compound of copper. Google Scholar Bloom, H. Evidence for the formation of a copper carbonyl. Google Scholar Huber, H., Kuendig, E. P., Moskovits, M. & Ozin, G. A. Binary copper carbonyls. Synthesis and characterization of tricarbonylcopper, dicarbonylcopper, monocarbonylcopper, and hexacarbonyldicopper. Google Scholar Alikhani, M. E. & Manceron, L. The copper carbonyl complexes revisited: why are the infrared spectra and structures of copper mono and dicarbonyl so different? Google Scholar Li, M. et al. Thermal stability of size-selected copper nanoparticles: effect of size, support and CO2 hydrogenation atmosphere. Google Scholar Vavra, J. et al. Solution-based Cu+ transient species mediate the reconstruction of copper electrocatalysts for CO2 reduction. Chen, C. et al. Highly crystalline multimetallic nanoframes with three-dimensional electrocatalytic surfaces. Google Scholar Xia, Y. et al. One‐dimensional nanostructures: synthesis, characterization, and applications. & Mathias, M. F. The priority and challenge of high-power performance of low-platinum proton-exchange membrane fuel cells. Arán-Ais, R. M. et al. Identical location transmission electron microscopy imaging of site-selective Pt nanocatalysts: electrochemical activation and surface disordering. Beermann, V. et al. Real-time imaging of activation and degradation of carbon supported octahedral Pt–Ni alloy fuel cell catalysts at the nanoscale using in situ electrochemical liquid cell STEM. Williamson, M. J., Tromp, R. M., Vereecken, P. M., Hull, R. & Ross, F. M. Dynamic microscopy of nanoscale cluster growth at the solid–liquid interface. Yang, Y. et al. Operando methods in electrocatalysis. Yang, Y. et al. Operando methods: a new era of electrochemistry. Yang, Y., Shao, Y.-T., DiSalvo, F. J., Muller, D. A. & Abruña, H. D. Metal monolayers on command: underpotential deposition at nanocrystal surfaces: a quantitative operando electrochemical transmission electron microscopy study. ACS Energy Lett. Pankhurst, J. R., Iyengar, P., Loiudice, A., Mensi, M. & Buonstanti, R. Metal–ligand bond strength determines the fate of organic ligands on the catalyst surface during the electrochemical CO2 reduction reaction. Shan, Y. Nanometre-resolved observation of electrochemical microenvironment formation at the nanoparticle–ligand interface. Yang, Y. et al. Elucidating cathodic corrosion mechanisms with operando electrochemical transmission electron microscopy. Yang, Y. et al. Dynamic evolution of copper nanowires during CO2 reduction probed by operando electrochemical 4D-STEM and X-ray spectroscopy. Shi, C. et al. Uncovering material deformations via machine learning combined with four-dimensional scanning transmission electron microscopy. npj Comput. Shi, C. et al. Domain-dependent strain and stacking in two-dimensional van der Waals ferroelectrics. Voyles, P. M. & Muller, D. A. Fluctuation microscopy in the STEM. Yang, Y. et al. Operando resonant soft X-ray scattering studies of chemical environment and interparticle dynamics of Cu nanocatalysts for CO2 electroreduction. De Ruiter, J. et al. Probing the dynamics of low-overpotential CO2 -to-CO activation on copper electrodes with time-resolved Raman spectroscopy. Weng, Z. et al. Active sites of copper-complex catalytic materials for electrochemical carbon dioxide reduction. Timoshenko, J. et al. Reversible metal cluster formation on nitrogen-doped carbon controlling electrocatalyst particle size with subnanometer accuracy. Zhang, L. et al. Elucidating the structure–stability relationship of Cu single-atom catalysts using operando surface-enhanced infrared absorption spectroscopy. Spiro, T. G., Soldatov, A. V. & Balakrishnan, G. CO, NO and O2 as vibrational probes of heme protein interactions. Li, J., Li, X., Gunathunge, C. M. & Waegele, M. M. Hydrogen bonding steers the product selectivity of electrocatalytic CO reduction. Hollins, P. The influence of surface defects on the infrared spectra of adsorbed species. Gunathunge, C. M. et al. Spectroscopic observation of reversible surface reconstruction of copper electrodes under CO2 reduction. An, H. et al. Sub-second time-resolved surface-enhanced Raman spectroscopy reveals dynamic CO intermediates during electrochemical CO2 reduction on copper. Liu, S. Alkali cation-induced cathodic corrosion in Cu electrocatalysts. Hersbach, T. J. P. & Koper, M. T. M. Cathodic corrosion: 21st century insights into a 19th century phenomenon. Louisia, S. et al. The presence and role of the intermediary CO reservoir in heterogeneous electroreduction of CO2. Xu, L. & Mavrikakis, M. Structure sensitivity in adsorbate-induced adatom formation on FCC transition-metal surfaces. Förster, S., Apostol, L. & Bras, W. Scatter: software for the analysis of nano- and mesoscale small-angle scattering. McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. Open Source Softw. A K-means clustering algorithm. Kresse, G. & Furthmüller, J. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Kresse, G. & Furthmüller, J. Efficiency of ab initio total-energy calculations for metals and semiconductors using a plane-wave basis set. Perdew, P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Blöchl, P. E. Projector augmented-wave method. Monkhorst, H. & Pack, J. Special points for Brillouin-zone integrations. & Scheffler, M. Adsorbate-substrate and adsorbate–adsorbate interactions of Na and K adlayers on Al(111). Bengtsson, L. Dipole correction for surface supercell calculations. Haynes, W. M. CRC Handbook of Chemistry and Physics 94th edn (CRC Press, 2014). Henkelman, G., Uberuaga, B. P. & Jonsson, H. A climbing image nudged elastic band method for finding saddle points and minimum energy paths. Download references This work was supported by the Director, Office of Science, Office of Basic Energy Sciences, Chemical Sciences, Geosciences, and Biosciences Division of the US Department of Energy under contract DE-AC02-05CH11231, FWP CH030201 (Catalysis Research Program). Operando EC-STEM work was supported by the Center for Alkaline-Based Energy Solutions (CABES), an Energy Frontier Research Center (EFRC) programme supported by the US Department of Energy under grant DE-SC0019445. This work used the TEM facilities at the Molecular Foundry, supported by the Office of Science, Office of Basic Energy Sciences of the US Department of Energy under contract DE-AC02-05CH11231. This research used beamline 11.0.1.2 of the Advanced Light Source, which is a DOE Office of Science User Facility, under contract DE-AC02-05CH11231. This work made use of the TEM facilities at the Cornell Center for Materials Research (CCMR), which is supported by the National Science Foundation Materials Research Science and Engineering Center (NSF MRSEC, DMR-1719875). This work is based on research conducted at the Center for High-Energy X-ray Sciences (CHEXS), which is supported by the National Science Foundation (BIO, ENG and MPS Directorates) under award DMR-2342336. are supported by the NSF CAREER (CMMI–2239545) and Welch Foundation (C-2065). We thank H. Celik and UC Berkeley's NMR facility at the College of Chemistry (CoC-NMR), which is supported in part by NIH S10OD024998. Work at UW-Madison was supported by the US Department of Energy, Basic Energy Sciences (DOE-BES), Division of Chemical Sciences, Catalysis Science Program under grant DE-FG02-05ER15731. We used the resources at the National Energy Research Scientific Computing Center, a DOE Office of Science User Facility, supported by the Office of Science of the US Department of Energy under contract DE-AC02-05CH11231 using NERSC award BES-ERCAP0032205. and J.J. gratefully acknowledge support through Suzhou Industrial Park Scholarships. acknowledges generous support through a Miller Research Fellowship, a Kavli Institute at Cornell (KIC) Instrumentation grant, and a fast grant from Cornell Atkinson Center for Sustainability and the 2030 Project: A Cornell Climate Initiative. These authors contributed equally: Yao Yang, Julian Feijóo, Marc Figueras-Valls, Chubai Chen. Department of Chemistry, University of California, Berkeley, CA, USA Yao Yang, Julian Feijóo, Chubai Chen, Maria V. Fonseca Guzman, Geonhui Lee, Jianbo Jin, Nathan E. Soland & Peidong Yang Miller Institute for Basic Research in Science, University of California, Berkeley, CA, USA Yao Yang Chemical Sciences Division, Lawrence Berkeley National Laboratory, Berkeley, CA, USA Yao Yang, Julian Feijóo, Chubai Chen, Maria V. Fonseca Guzman, Yu Shan & Peidong Yang Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY, USA Yao Yang, Shikai Liu & Valentín Briega-Martos Department of Chemical and Biological Engineering, University of Wisconsin-Madison, Madison, WI, USA Marc Figueras-Valls, Michael Rebarchik, Lang Xu & Manos Mavrikakis Department of Materials Science and NanoEngineering, Rice University, Houston, TX, USA Chuqiao Shi & Yimo Han Department of Chemical Engineering, University of Massachusetts Amherst, Amherst, MA, USA Yves Murhabazi Maombi, Pulkit Jain & Zhu Chen Advanced Light Source, Lawrence Berkeley National Laboratory, Berkeley, CA, USA Zhengxing Peng & Cheng Wang Department of Materials Science and Engineering, University of California, Berkeley, CA, USA Yu Shan, Miquel B. Salmeron & Peidong Yang Cornell High Energy Synchrotron Source, Wilson Laboratory, Cornell University, Ithaca, NY, USA Materials Sciences Division, Lawrence Berkeley National Laboratory, Berkeley, CA, USA Miquel B. Salmeron & Peidong Yang Kavli Energy NanoScience Institute, Berkeley, CA, USA Peidong Yang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar initiated the project under the guidance of P.Y. performed atomic-scale STEM-EELS and operando EC- and 4D-STEM measurements. performed HERFD XAS measurements of Cu nanocatalysts with the help of C.J.P. performed all electrochemical measurements and performance tests in H-cells. performed operando XAS analysis of CuPc catalysts with the help of M.V.F.G., Y.S. synthesized Cu nanocubes. M.F.-V. performed DFT simulations with the help of M.R. under the guidance of M.M. performed machine learning-based 4D-STEM clustering under the guidance of Y.H. performed in situ Raman measurements of Cu nanocatalysts under the guidance of P.Y. performed in situ SERS studies of single-atom CuPc catalysts and the corresponding analysis under the guidance of Z.C. performed atomic-scale STEM imaging analysis under the guidance of Y.Y. performed RSoXS analysis under the guidance of C.W. performed MEA measurements under the guidance of P.Y. J.J. prepared the schematic in Fig. performed SEM measurements. M.B.S and V.B.-M. contributed to discussions on Cu migration mechanisms. All authors wrote, revised and approved the paper. Correspondence to Manos Mavrikakis or Peidong Yang. The authors declare no competing interests. Nature Catalysis thanks Debangshu Mukherjee, Bert Weckhuysen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–65, Tables 1–7 and references. Operando EC-STEM images of 55-nm Cu nanocubes at 0 V versus RHE after the initial LSV scan showing the transformation of the Cu spongy shell to spherical polycrystalline Cu clusters. Note that STEM videos were acquired without potential applied from −40 to 0 s to show the absence of beam-induced damage at a dose rate of ~20 e nm−2 s−1. Corresponding selected EC-STEM images can be found in Fig. 2a–d and Supplementary Fig. Operando EC-STEM images of 55-nm Cu nanocubes at −1.0 V versus RHE after the initial LSV scan showing the transformation of the cubic core to Cu nanograins following the evolution of the spongy Cu shell at 0 V as shown in Supplementary Video 1. Note that STEM videos were acquired without potential applied from −40 to 0 s to show no beam-induced damage at a dose rate of ~20 e nm−2 s−1. Corresponding selected EC-STEM images can be found in Fig. 2e–h and Supplementary Fig. Operando EC-STEM images of 27-nm Cu nanocubes at −1.0 V versus RHE after the initial LSV scan showing the stepwise evolution of the cubic core followed by the spongy shell into Cu nanograins. Corresponding selected EC-STEM images can be found in Fig. Operando EC-STEM images of 55-nm nanocubes surrounded by spongy shell after LSV to 0 V, under a mild potential at 0 V near the tip of the carbon WE with a much stronger electric field, showing the structural transformation of the Cu spongy shell followed by the cubic Cu core to polycrystalline Cu nanograins. Note that STEM videos were acquired without potential applied from −16 to 0 s to show no beam-induced damage at a dose rate of ~20 e nm−2 s−1. The image fluctuation at 8 s is likely due to the liquid flow commonly observed at the beginning of applied potentials. Corresponding selected EC-STEM images can be found in Supplementary Fig. Operando EC-STEM images of 27-nm Cu nanocubes at −1.0 V versus RHE after the initial LSV scan showing the stepwise evolution of the cubic core followed by the spongy shell into Cu nanograins. Corresponding selected EC-STEM images can be found in Supplementary Fig. DFT-optimized structures of CO overlayer structure for CO coverages of 0 to 6/9 ML on Cu(511). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Yang, Y., Feijóo, J., Figueras-Valls, M. et al. Operando probing dynamic migration of copper carbonyl during electrocatalytic CO2 reduction. Download citation Published: 25 June 2025 Issue Date: June 2025 DOI: https://doi.org/10.1038/s41929-025-01359-w Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Catalysis ISSN 2520-1158 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/see-vaccine-recommendations-backed-by-science-in-these-handy-charts/'>Here's Your Cheat Sheet for Vaccine Recommendations Backed by Science</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>See Vaccine Recommendations Backed by Science in These Handy Charts These graphics will guide you through science-based vaccine guidelines for children and adults Vaccines are a marvel of modern medicine: the carefully tested and regulated technologies teach people's immune systems how to fight off potentially fatal infections, saving both lives and health care costs. But for as long as vaccines have existed, people have opposed them, and in recent years the antivaccine movement has gained visibility and power. Now the Department of Health and Human Services is led by Robert F. Kennedy, Jr.—an environmental lawyer with no medical training and a history of antivaccine activism. And these lifesaving medical interventions are coming under threat. And in June Kennedy dismissed all 17 sitting members of a crucial vaccine oversight group, the Centers for Disease Control and Prevention's Advisory Committee on Immunization Practices (ACIP), which, in the past, has made independent, science-based recommendations on vaccine access for people in the U.S. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. As a public resource, Scientific American has created graphics outlining the vaccines recommended by ACIP as of its final meeting in 2024. Vaccine recommendations have always been in flux as new products have been developed and continuing research has suggested better practices: The COVID pandemic required brand-new vaccines for a novel virus, for example. But traditionally, these decisions have been made by scientists based on solid research done within the confines of accepted ethical practices. These principles mean, for example, that a vaccine's side effects are carefully monitored and evaluated against its immune benefit and that potential replacement vaccines are tested against their predecessors, not—as Kennedy has proposed—an inert placebo that would leave people vulnerable to an infection that doctors already have the tools to combat. Kennedy's decision to replace ACIP wholesale and the comments he has made about deviating from standard vaccine policymaking practice suggest that new recommendations won't be backed by established vaccine science—hence our reproduction of the vaccine recommendations as of the end of 2024. Note that these are generalized recommendations; people should talk with their health care providers about individual risks and needs, as well as how to proceed after missing a dose. Pregnant people can consult additional resources from the American College of Obstetricians and Gynecologists for vaccines recommended during pregnancy. People planning to travel internationally should also check what vaccines are recommended for their destination and consult with a health care professional more than a month before departure. Hepatitis A and B: Both of these viruses cause liver infections. Rotavirus: This common gastrointestinal virus causes diarrhea that is sometimes severe enough to require hospitalization. Diphtheria: This bacterial infection has become rare in the U.S. through vaccination; before the vaccine was available, case rates could be as high as 200,000 annually. Half of untreated people die; children under age five and adults more than 40 years old are most vulnerable. Tetanus: Sometimes called lockjaw because an early symptom is muscle pain and spasms in the jaw, tetanus is caused by toxins from a bacterium. U.S. infection levels have generally run between 10,000 and 20,000 diagnosed cases per year; the disease hospitalizes more than one in five infected children under six months old. Haemophilus influenzae type b infection: This bacterium—unrelated to the influenza virus—causes a host of infections, including mild cases in the ears and lungs but also severe cases in systems such as the bloodstream and central nervous system. Before the vaccine was developed, the U.S. saw 20,000 severe infections annually in children under five years of age, and one in 20 of these cases was fatal. Pneumococcal disease: The bacterium Streptococcus pneumoniae can cause a range of infections, including so-called invasive infections that tend to be more serious. Pneumococcal disease can include pneumonia—pneumococcal pneumonia hospitalizes more than 150,000 people in the U.S. each year. But other types of pathogens also cause pneumonia, and pneumococcal disease can manifest anywhere in the body. Polio: This virus most frequently causes asymptomatic infections, but symptomatic infections can have quite severe symptoms, including paralysis of one or more limbs or even of the muscles involved in breathing. Polio can also trigger new symptoms many years after the initial infection in what's called postpolio syndrome. The virus also causes lingering and sometimes debilitating systemic issues known as long COVID, including in children. Influenza: This respiratory virus is most prevalent in North America between October and May. Although many cases can be treated at home, flu infections can be very serious, particularly in young children and adults aged 65 or older, as well as people with immune issues and other chronic conditions. Chickenpox: The varicella-zoster virus causes a characteristic itchy rash of small blisters that appear in conjunction with a fever, headache and other mild symptoms. Severe cases can cause more systemic infections, pneumonia, brain swelling and toxic shock syndrome. Adults who did not have chickenpox as a child are more vulnerable to serious infection. Measles: Measles is one of the most contagious viruses known to experts, and historically most children contracted it before the age of 15. Doctors have no cure for measles; they can only treat its symptoms. About one in 1,000 cases causes brain inflammation; even rarer complications can occur years after the initial infection. Mumps: Mumps is a viral infection characterized by the swelling of certain salivary glands, but other organs can also be affected, including the testicles, ovaries, brain, spinal cord and pancreas. Rubella: Sometimes called German measles, rubella is a viral infection that is unrelated to measles but also causes a rash. For most people, rubella is a mild illness, but it triggers serious birth defects in as many as 90 percent of cases in which the virus infects someone during the first 12 weeks of pregnancy. The infection is characterized by a painful rash and flulike symptoms. Vaccination is not available in the contiguous U.S., but it is available in U.S. territories and freely associated states for children aged nine to 16 who have had the disease before and live in a region where the infection is common. Shingles: This infection is caused by the same virus as chickenpox, which remains in the body after a chickenpox infection. When the previously dormant virus reactivates, it can cause shingles, a painful localized rash that is most common in people aged 50 or older and can lead to ongoing pain, vision issues and neurological problems. Subsequently she moved to Washington, D.C., to join the staff of National Geographic (first as an assistant art director–researcher hybrid and then as a designer), spent four years as a freelance science communicator and returned to Scientific American in 2007. Christiansen presents and writes on topics ranging from reconciling her love for art and science to her quest to learn more about the pulsar chart on the cover of Joy Division's album Unknown Pleasures. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250625011635.htm'>The pleasure prescription: Why more sex means less menopause pain</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 09:35:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It's no secret that women often become less interested in sex with age. However, orgasm and satisfaction have been shown to not decline significantly with age. A new study suggests regular sexual activity may limit vulvar pain, irritation, and dryness, which are all common reasons women have less sex as they get older. GSM includes genital, sexual and urinary symptoms -- all of which can affect the frequency of sexual activity for women aged in their 40s to 70s. In this new study involving more than 900 women aged 40 to 79 years, researchers sought to examine the association between sexual regularity and vulvovaginal-related problematic menopause symptoms. Common problems experienced with menopause include itching, burning, pain, decreased lubrication, and changes in skin appearance. Not surprisingly, the researchers confirmed that the proportion of women having regular sexual activity decreased significantly with age, which aligns with the fact that Female Sexual Function Index scores for sexual desire, arousal, and lubrication also significantly decreased with age. Noteworthy, however, was that the scores for orgasm and satisfaction did not change with age. This study also revealed that women with regular sexual activity showed a low prevalence of GSM-related symptoms. Study results are published in the article "Cross-sectional study of the association between regular sexual activity and sexual function and genitourinary syndrome of menopause-related symptoms." And although optimal sexual health is integral to overall well-being, it is also imperative to recognize the effect these symptoms can have on women who aren't sexually active. Treatment should be offered to anyone with symptoms, whether engaging in sexual activity or not. Normalizing use of local low-dose estrogen therapy should be a thing," says Dr. Monica Christmas, associate medical director for The Menopause Society. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250624224813.htm'>Mojave lichen defies death rays—could life thrive on distant exoplanets?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-25 03:00:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The question of whether Earth is alone in harboring life has captivated humanity for millennia. In recent years, scientists have turned to Earth-like planets in other solar systems that may show the most promise, but many revolve around stars that emit much stronger solar radiation than our own. Now, a new study offers evidence that life as we know it may be able to thrive on those Earth-like exoplanets. Published June 12th in Astrobiology, the new research demonstrated that lichen found in the Mojave Desert survived for 3 months under levels of solar radiation previously considered lethal. The common lichen, Clavascidium lacinulatum, was injured, but able to recover and replicate. The results show that photosynthetic life may be possible on planets exposed to intense solar radiation. "I was just walking in the desert and I noticed that the lichens growing there aren't green, they're black. And that pigment turned out to be the world's best sunscreen." Life on Earth has evolved to withstand solar radiation known as UVA and UVB rays, which encompass longer wavelengths of ultraviolet radiation. UVC rays represent the shorter, more damaging wavelengths; these are filtered by Earth's atmosphere and therefore do not reach the surface. UVC radiation is so lethal to life, in fact, that it is used to sterilize air, water, and surfaces of microorganisms like viruses and bacteria. Even a brief exposure to UVC radiation will damage an organism's DNA and prevent it from reproducing. Scientists have wondered, then, whether many of the Earth-like planets discovered in recent years could possibly harbor life. Many of these planets revolve around stars known as M and F stars that emit intense UVC radiation, especially during solar flares. "We're talking about planets that have liquid water and an atmosphere." We also wanted to go beyond just activity and demonstrate viability." To explore how this was chemically possible, Sun teamed up with chemists from the University of Nevada, Reno. They conducted two experiments that demonstrated how lichen acids are the natural world's equivalent of the additives used to make plastics UV-resistant. They investigated the lichen's protective layer by cutting a cross-section of it and found that the top layer was darker, like a human's suntan. The discovery that lichen has evolved this protective layer to UVC radiation was surprising, Sun said, because it isn't necessary for their survival. Earth's atmosphere was already filtering out UVC rays by the time lichen appeared, so the protection is a mere bonus thanks to their remarkable UVA and UVB protection. This layer acts as a photo stabilizer and even protects the cells from harmful chemical reactions caused by the radiation, including reactive oxygen." "This work reveals the extraordinary tenacity of life even under the harshest conditions, a reminder that life, once sparked, strives to endure," said Tejinder Singh, who led the study with Sun and is now at NASA Goddard Space Flight Center. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            