
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-21</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251021083644.htm'>This simple innovation could change blood pressure testing forever</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 15:39:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A newly developed method that improves the accuracy of ankle blood pressure measurements could transform care for people who are unable to have their blood pressure taken from the arm. Researchers from the University of Exeter Medical School, in a study published in BMJ Open and funded by the National Institute for Health and Care Research (NIHR), analyzed data from more than 33,000 individuals to create a personalized predictive model. This model enables healthcare professionals to estimate arm blood pressure more precisely using ankle readings (when compared with earlier approaches). More than one billion people worldwide live with high blood pressure, a major risk factor for heart disease, stroke, and kidney problems. Reliable measurements are crucial for diagnosing and managing the condition effectively. Although blood pressure is typically measured on the upper arm, some people cannot have it taken there due to disability, limb loss, or conditions such as stroke. However, ankle measurements are usually higher than arm readings, and because treatment guidelines are based on arm data, this difference can lead to inaccurate diagnoses and inappropriate treatment decisions. This doesn't sound a big number but remember, around a third of adults have high blood pressure and once you get into your 60s it's more than half of the adults. The NHS Health Check Programme diagnoses 38,000 new cases annually in England alone, so two percent equates to 750 fewer potential misdiagnoses per year in England, and tens of thousands globally." To achieve this, the researchers used statistical modeling on data from 33,710 people (mean age 58 years, 45 per cent female) across multiple countries. This advancement could help close a significant health gap by providing accurate and personalized blood pressure results for people who have been excluded from standard monitoring. It is estimated that up to 10,000 adults in the UK live with upper limb loss, while 75 per cent of the country's 1.3 million stroke survivors experience upper limb difficulties that can make arm-based measurements challenging or impossible. Juliet Bouverie OBE, CEO of the Stroke Association, said: "Someone in the UK has a stroke every five minutes, with high blood pressure accounting for around half of those. Around two-thirds of stroke survivors will leave hospital with some form of disability, including paralysis in an arm, which can prevent getting accurate blood pressure readings from the affected limb. Professor Kevin Munro, Director of NIHR's Research for Patient Benefit Programme, said: "This research has identified an ingenious solution to an important problem -- finding a way to measure blood pressure for people who cannot have it monitored via the upper arm. Keeping track of blood pressure is a vital tool to help keep people healthy and this NIHR-funded research will help to spot high blood pressure and treat it even more widely." An individual participant data meta-analysis from the INTERPRESS-IPD Collaboration" is published in BMJ Open. "Why should I not be able to have my blood pressure taken?" TV presenter Sue Kent, 62 from Swansea, has an upper limb disability caused by the drug Thalidomide, which was prescribed to her mother during pregnancy. She has eight-inch arms which aren't big enough for blood pressure to be taken. Whether it was accurate or not nobody worried, but I didn't seem to have blood pressure problems. She said: "I was worried I was going to have a stroke because Meniere's can be an indicator the blood flow isn't going to the brain. Prodding about in the dark and guessing isn't really a safe thing to do. Sue hopes this new method could potentially help her and thousands of others like her have something most of us take for granted -- an accurate blood pressure reading. This could put us on a level playing field with everybody else when it comes to blood pressure. It means reassurance and maybe an early diagnosis if something is wrong." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a69074272/black-hole-star/'>This Faraway Star Looks Weird. It May Be Because There's a Black Hole Inside It.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Almost from the moment NASA's James Webb Space Telescope saw first light, it spotted mysterious little red dots, or LRDs, glowing in the distance. They were thought to be extremely ancient and unusually mature galaxies that raised questions about galaxy formation. Seeking answers that were less vague, the international team of researchers who analyzed the first dataset that had been released by Webb in 2022 recently analyzed them again and came up with an alternative hypothesis for whatever these objects are. They also seem to act like stars, except not quite, because stars run on nuclear fusion while LRDs appear to be powered by something else. Spectral analysis suggests that it is one enormous and cold (at least for a constantly burning ball of gas) star. Now named the Cliff, this particular red dot about 11.9 billion light-years away has an atmosphere too thick for just any star. Because its supposed black hole within is so ravenous, it pulls matter in from all directions and at super fast speeds, digesting it so it is converted into energy and blazes with red light. “Determining whether the [spectral energy distributions] of LRDs are dominated by stars or [active galactic nuclei] is a major challenge, even with high-quality rest-optical spectroscopy and photometry ranging from X-ray to radio wavelengths,” the researchers said in a study recently published in Astronomy & Astrophysics. While the Cliff and other LRDs like it are exceptionally huge stars that only appear tiny because they are so distant, and most stars in the universe are actually cold low-mass stars, their light often drowned out by scorching massive stars with a much higher luminosity. Spectra from the red dots revealed none of them had nearly that much heat. What is unusual about these red dots possibly being supermassive black holes in disguise is that gases swirling around in supermassive black hole accretion disks are much hotter and brighter. Event Horizon Telescope images of the M83 black hole and our own Sagittarius A* are proof. Their light ranges from shades of orange and yellow on the edges that brighten to white hot on the inside. They suggest that a void might not be visible because of dense and turbulent gas clouds that shroud the accretion disk and mirror an actual stellar atmosphere. It could even be that, even though accretion disks are typically thought to feed black holes, something else could be giving the monster its strength, because, as they also said in the study, “any powerful ionizing source inside the dense gaseous envelope could produce [the spectra observed].” “Crucially, this is the first LRD for which such a strong conclusion can be drawn solely by analyzing the [spectral energy distribution].” Maybe the song “Black Hole Sun” was onto something. ➡ Inside the Secret Island Where Death Is Optional ➡ This Tech Rebel Threw Away $900 Million in a Municipal Dump. ➡ Sex Workers, LSD, and Mind Control: What Happened in the CIA's Lab of Nightmares at 225 Chestnut Street ➡ NASA Has a Plan to Save Earth from Planet-Destroying Asteroids. ➡ A Naval Officer Says Underwater UFOs Are Legitimate Threats. ➡ This Undersea Explorer Found America's Greatest Sunken Treasure. ➡ Is Bigfoot Hiding in the Swamps of Florida? ➡ Scientists May Have Gotten the Global-Warming Timeline Seriously Wrong. ➡ Scientists Successfully Revive a Dead Brain, Redefining the Boundary Between Life and Death ➡ For 80 Years, the North Sea Held a Deadly Killer. Now Scientists Are Racing to Defuse the Threat. Why Is Everyone So Afraid to Step Inside It? ➡ This Guy Says He Knows the Truth About UFOs. ➡ Scientists Are Now One Step Away From Solving Nuclear Fusion—And Unlocking Unlimited Energy. ➡ A New Era of Missile Warfare Has Begun—and the U.S. ➡ Inside the Deranged Plot to Smuggle Cocaine With an Armed Soviet-Era Submarine ➡ This Brilliant Engineer Helped Build the B2 Bomber—Then He Sold America's Stealth Secrets to China ➡ Your Consciousness Can Connect With the Whole Universe ➡Untold Secrets Reveal How the Castle Bravo Test Became America's Worst Nuclear Disaster ➡ The Army's Machine Gun Is No Match for Cheap Chinese Body Armor. ➡ A New Era of Missile Warfare Has Begun—and the U.S. It's also available on Apple News+.Featuring:➡The Man Who Knows Too Much About Area 51 ➡ The Secret to a Perfect Lawn Lies in One of These 10 Electric Lawnmowers ➡ Scientists Believe They've Unlocked Consciousness—and It Connects to the Entire Universe ➡ Why This Unstoppable Stealth Bomber Will Rule the Skies ➡ America Is Developing a New Nuclear Bomb—But Can't Test Whether It Works It's also available on Apple News+.Featuring:➡ How Deadly Nuclear Waste Is Menacing This St. Louis Neighborhood ➡ The Terrifying History of Russia's Nuclear Submarine Graveyard ➡  Strange Mutations in Stray Dogs Near Chernobyl Suggest They Are Rapidly Evolving ➡  America Dumped 56 Million Gallons of Radioactive Material Along the Columbia River—Then It Started to Leak It's also available on Apple News+.Featuring:➡ A New Clue In Amelia Earhart's Disappearance Emerges From the Ocean ➡ China Just Built a Terrifying New Aircraft Carrier and May Soon Dominate the Seas ➡ How Rats Took Over Our Cities—And Why We Can't Stop Them ➡ This Language Is on the Verge of Extinction. ➡ America's Deadliest Warplane Returns in a New Doomsday Role ➡ The Scientific Breakthrough That Could Put an End to Gray Hair. ➡ Your Next iPhone (and Nuclear Subs) Will Be Powered By Space Metal ➡Scientists Now Think We Can Build a Warp Drive ➡ China and Russia Have Cracked the Stealth Code. ➡ These Florida Homes Aren't Just Hurricane-Proof—They're Blueprints for the Future ➡ There's No Weapon Russia Fears More Than the HIMARS Rocket Launcher ➡The Nuclear-Submarine Arms Race Is Getting Intense, and the U.S. Just Took a Massive Leap Forward ➡ Iran Is Becoming a Drone Superpower—By Stealing American Technology ➡ The Greatest Treasure Hunt in American History Ended—and Then Things Got Weird ➡ These Are the High-Powered Weapons Ukraine Needs to Send Russia Running ➡ The Secret War to Take Out Iran's Fleet of F-14 Jets ➡ Russia Is Trying to Intimidate the U.S. with Hypersonic Missiles and Big, Scary Nukes—And It's More Than a Threat ➡ China and Russia Are Dominating the Hypersonic Arms Race—And It's Not Even Close ➡ When the South Fork Dam Broke, a Pennsylvania City Washed Away. ➡ The Navy's New $13 Billion Aircraft Carrier Is Already Obsolete. ➡ Can America's M1 Abrams Still Compete With China's and Russia's Latest Battle Tanks? ➡ How the Massive Cargo Ship Felicity Ace Sank, Taking $400 Million Worth of Exotic Supercars With It➡ I Turned My Old Gas-Guzzler Into a Zippy EV for $15,000 ➡ Cosmic Secrets of the 17 Most Powerful Mega-Telescopes on Earth—and Beyond ➡ America's Most Fearsome Howitzer Has Entered the War in Ukraine ➡ Every Single Drone Fighting in the Skies Over Ukraine➡ How to Buy a New Car in 2022 Without Getting Fleeced➡ This Megastructure Could Keep Us Alive Forever➡ The Race to Revolutionize EV Batteries Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. This Signal May Have Come Through a Wormhole Einstein and Hawking Were Right About Black Holes Scientists Witnessed the Birth of a Space Monster Life on Earth May Have Started In Space It's Possible to Safely Jump Into a Black Hole</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03340-w'>Universities are embracing AI: will students get smarter or stop thinking?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 10:20:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. When students entered Tsinghua University in Beijing this year, one of the first representatives they met wasn't a person. The bot is designed to answer students' questions about courses, clubs and life on campus. At Ohio State University in Columbus, students this year will take compulsory AI classes as part of an initiative to ensure that all of them are ‘AI fluent' by the time they graduate. And at the University of Sydney, Australia, students will take traditional, in-person tests to prove that they have learnt the required skills and not outsourced them to AI. All this is part of a sea change sweeping through campuses as universities and students scramble to adapt to the use of generative AI tools such as ChatGPT. These tools can, within seconds, analyse complex information, answer questions and generate polished essays — some of the exact skills that universities have conventionally taught. Eighty-six per cent of university students were regularly using AI in their studies in 2024, according to one global survey1 — and some polls show even more. “We are seeing students become power users of these tools,” says Marc Watkins, a researcher who specializes in AI and education at the University of Mississippi in Oxford. Universities such as Tsinghua and Ohio State are already weaving AI into their teaching — and some studies hint that AI-powered tools can help students to learn. A randomized controlled trial2 involving physics undergraduates at Harvard University in Cambridge, Massachusetts, suggested that those who used a custom-built AI tutor learnt more in less time than did students who were taught by humans alone. Faculty members also worry that students are using AI to short-cut their way through assignments and tests, and some research3 hints that offloading mental work in this way can stifle independent, critical thought. Universities must move with the times: how six scholars tackle AI, mental health and more Universities must move with the times: how six scholars tackle AI, mental health and more An open letter released in June by scholars objecting to the uncritical adoption of AI technologies by universities quickly racked up more than 1,000 signatures from academics worldwide (see go.nature.com/4qpkygs). There is one point of agreement among both enthusiasts and sceptics: the future has already arrived. “The rate of adoption of various generative AI tools by students and faculty across the world has been accelerating too fast for institutional policies, pedagogies and ethics to keep up,” says Shafika Isaacs, chief of section for technology and AI in education at the United Nations cultural organization UNESCO, in Paris. But generative AI tools present a fundamental challenge, researchers say. “The thing that is different about AI to every other technology is the speed at which it moves,” says Sue Attewell, head of AI at Jisc, a non-profit digital and data body for UK higher education, headquartered in Bristol. Surveys around the world show that students are quickly incorporating AI into their lives. “The number of users who are essentially young teens is just astronomically high,” says Watkins. It found that students in science, technology, engineering and mathematics were using it disproportionately more than were those in areas such as business and humanities4. In a survey of more than 1,000 UK students by the Higher Education Policy Institute, a think tank in Oxford, UK, most were, unsurprisingly, using AI tools to write, improve or summarize text (see ‘How university students use AI'). Nearly 90% of students said they had used generative AI for assessments such as exams or coursework. Attewell, who has conducted student focus groups on AI, says many students are using the tools to streamline their studies and become more efficient, not to cheat. And many just want guidance on how to boost their learning with AI, she says. Institutions around the world have been scrambling to introduce policies for staff and students on the best ways to use AI — but with mixed results. On US campuses, “it's been chaotic for the past two and a half years, and I don't think it's going to change anytime soon,” Watkins says. That's because many institutions allow individual faculty members to decide whether and how AI can be used in their courses, he says, which leaves some students confused. In Australia, by contrast, there has been a strong, nationally coordinated response, educators say, in part because the Tertiary Education Quality and Standards Agency has worked to develop guidelines with universities since 2023. Some staff are using AI tools to automate their own work, such as by helping to draft feedback on assignments, say researchers, whereas others are rejecting it outright. A survey this year of more than 1,600 faculty members from 28 countries by the Digital Education Council, an organization focused on technology in higher education, found that around 60% were using AI tools in teaching — less than the proportion of students using them6. “We're inundated by companies wanting to partner with us,” says Ravi Bellamkonda, Ohio State University's executive vice-president and provost. Last year, OpenAI introduced ChatGPT Edu, a version of the popular chatbot tailored for universities. This February, when the California State University system announced it was giving its 520,000 students and faculty members access to that tool, it became “the largest implementation of ChatGPT by any single organization or company anywhere in the world”, according to OpenAI. Education specialists say that tech firms are motivated by commercial interests and the opportunity to embed their AI systems into the lives of millions of young users. Danny Liu, a biologist who specializes in educational technology at the University of Sydney, was one of the first to develop a generative AI platform tailored to higher education. Called Cogniti and developed in 2023, it is now embedded in the university's digital learning platform and allows teachers to design custom AI agents for their courses. This might be an AI tutor for a science module, or an AI tool that expands brief marking comments into more specific student feedback. The great university shake-up: four charts show how global higher education is changing The great university shake-up: four charts show how global higher education is changing At Tsinghua University, academics also quickly started experimenting with ChatGPT after its release in November 2022, says Shuaiguo Wang, who directs the Center for Online Education there. It also didn't want to be reliant on any one AI model — such as the large language models (LLMs) that underlie ChatGPT and related tools — because “a new model emerges almost every day”, Wang says. Another key goal was to correct hallucinations, the inaccurate content that LLMs sometimes spout. The bottom layer is where AI models are plugged in: the system draws on about 30 models from companies including DeepSeek, Alibaba Cloud, OpenAI and Google, Wang says. The middle layer contains ‘knowledge engines' that hold accurate, up-to-date information for different academic fields. The top layer consists of various student-facing platforms, including an educational one that hosts teaching assistants, as well as the AI bot to help students starting at Tsinghua. This allows them to ask a pop-up AI agent questions. Wang says the system has been adopted by hundreds of universities across China. This year, Wang is changing tack and launching studies to ask a crucial and outstanding question: do students using AI tools actually learn more? Early results — as yet unpublished — have raised some red flags. Wang and his colleagues found that students who use an AI tutor often score higher in tests immediately after a class than do those who don't use the agent. But two to three weeks later, the AI users scored lower than their peers. “Students might have a false sense of understanding,” he says. Wang is not the only one worried about the impact of AI on students' learning — as Nataliya Kosmyna discovered when, in June, she and her colleagues released a preprint7 suggesting that students who use ChatGPT to help write an essay use less of their brains than do students who write it on their own. She has since received some 4,000 e-mails from school and university teachers worldwide who are concerned that the result reflects what is happening to students in their classrooms who are using AI. In a lot of cases, “they actually sound very desperate”, she says. Kosmyna, a research scientist at the Massachusetts Institute of Technology in Cambridge, and her team used electro-encephalography (EEG) to study the patterns of electrical activity in 54 students' brains as they wrote 20-minute essays on questions such as “Do works of art have the power to change people's lives?” The students were split into three groups: one with access to an LLM; another that could use websites and search engines but not LLMs; and a third that relied only on their own knowledge. The LLM-using group showed the weakest connectivity — suggesting that there was less cross-brain communication — and could sometimes barely remember a word of their prose. Kosmyna acknowledges that the study involved only a small number of students and did not look at the long-term impacts of using AI, among other limitations. (She also found it ironic that some commentators misinterpreted the study because they used AI to generate incorrect summaries of her 200-page preprint.) Anthropic's analysis of one million student–AI conversations in Claude showed that nearly half of them involved individuals seeking direct answers or content with little engagement on their part. “Their skills have dramatically changed,” she says: some struggle to reference a source or write an essay. The great university shake-up: four charts show how global higher education is changing Universities must move with the times: how six scholars tackle AI, mental health and more Cake to the rescue: how these PhD students are cooking up a sense of community Brazil must beef up its COP30 scheme to preserve tropical forests Digital flocks, predators allow ecologists to simulate real-world animal behaviour Established or emerging researchers with a strong biomedical background focused on nutrition and health. The great university shake-up: four charts show how global higher education is changing Universities must move with the times: how six scholars tackle AI, mental health and more An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42256-025-01116-5'>Towards deployment-centric multimodal AI beyond vision and language</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 10:06:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Machine Intelligence (2025)Cite this article Multimodal artificial intelligence (AI) integrates diverse types of data via machine learning to improve understanding, prediction and decision-making across disciplines such as healthcare, science and engineering. However, most multimodal AI advances focus on models for vision and language data, and their deployability remains a key challenge. We advocate a deployment-centric workflow that incorporates deployment constraints early on to reduce the likelihood of undeployable solutions, complementing data-centric and model-centric approaches. We also emphasize deeper integration across multiple levels of multimodality through stakeholder engagement and interdisciplinary collaboration to broaden the research scope beyond vision and language. To facilitate this approach, we identify common multimodal-AI-specific challenges shared across disciplines and examine three real-world use cases: pandemic response, self-driving car design and climate change adaptation, drawing expertise from healthcare, social science, engineering, science, sustainability and finance. By fostering interdisciplinary dialogue and open research practices, our community can accelerate deployment-centric development for broad societal impact. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Source data are provided with this paper. They are also available at https://github.com/multimodalAI/multimodal-ai-landscape, where they will be updated annually. Wang, H. et al. Scientific discovery in the age of artificial intelligence. Kabir, M. M., Jim, J. R. & Istenes, Z. Terrain detection and segmentation for autonomous vehicle navigation: a state-of-the-art systematic review. Woloshin, S., Patel, N. & Kesselheim, A. S. False negative tests for SARS-CoV-2 infection—challenges and implications. Zhao, F., Zhang, C. & Geng, B. Deep multimodal data fusion. ACM Comput. Baltrušaitis, T., Ahuja, C. & Morency, L.-P. Multimodal machine learning: a survey and taxonomy. IEEE Trans. Ektefaie, Y., Dasoulas, G., Noori, A., Farhat, M. & Zitnik, M. Multimodal learning with graphs. Xu, P., Zhu, X. & Clifton, D. A. Multimodal learning with transformers: a survey. IEEE Trans. Liang, P. P., Zadeh, A. Foundations & trends in multimodal machine learning: principles, challenges, and open questions. ACM Comput. Acosta, J. N., Falcone, G. J., Rajpurkar, P. & Topol, E. J. Multimodal biomedical AI. Kline, A. et al. Multimodal machine learning in precision health: a scoping review. npj Digit. Krones, F., Marikkar, U., Parsons, G., Szmul, A. Review of multimodal machine learning approaches in healthcare. Notin, P., Rollins, N., Gal, Y., Sander, C. & Marks, D. Machine learning for functional protein design. Song, B., Zhou, R. & Ahmed, F. Multi-modal machine learning in engineering design: a review and future directions. Ofodile, O. C. et al. Predictive analytics in climate finance: assessing risks and opportunities for investors. GSC Adv. Quatrini, S. Challenges and opportunities to scale up sustainable finance after the COVID-19 crisis: lessons and promising innovations from science and practice. Gupta, V. et al. An emotion care model using multimodal textual analysis on COVID-19. Chaos Solitons Fractals 144, 110708 (2021). Anshul, A., Pranav, G. S., Zia Ur Rehman, M. & Kumar, N. A multimodal framework for depression detection during COVID-19 via harvesting social media. IEEE Trans. Bordes, F. et al. An introduction to vision–language modeling. Preprint at https://doi.org/10.48550/arXiv.2405.17247 (2024). Zhang, J., Huang, J., Jin, S. & Lu, S. Vision–language models for vision tasks: a survey. IEEE Trans. van Breugel, B. & van der Schaar, M. Position: why tabular foundation models should be a research priority. 41st International Conference on Machine Learning (eds Salakhutdinov, R. et al.) 48976–48993 (PMLR, 2024). Ma, M. et al. SMIL: multimodal learning with severely missing modality. AAAI Conference on Artificial Intelligence Vol. Wu, R., Wang, H., Chen, H.-T. & Carneiro, G. Deep multimodal learning with missing modality: a survey. Preprint at https://doi.org/10.48550/arXiv.2409.07825 (2024). Wang, F., Zhou, Y., Wang, S., Vardhanabhuti, V. & Yu, L. Multi-granularity cross-modal alignment for generalized medical visual representation learning. Zhao, T., Zhang, L., Ma, Y. & Cheng, L. A survey on safe multi-modal learning systems. 30th ACM SIGKDD Conference on Knowledge Discovery and Data Mining 6655–6665 (Association for Computing Machinery, 2024). Pranjal, R. et al. Toward privacy-enhancing ambulatory-based well-being monitoring: investigating user re-identification risk in multimodal data. In IEEE International Conference on Acoustics, Speech and Signal Processing 1–5 (IEEE, 2023). Paleyes, A., Urma, R.-G. & Lawrence, N. D. Challenges in deploying machine learning: a survey of case studies. ACM Comput. Jo, A. The promise and peril of generative AI. Brown, T. et al. Language models are few-shot learners. 34th International Conference on Neural Information Processing Systems (eds Larochelle, H. et al.) 1877–1901 (Curran Associates Inc., 2020). Achiam, J. et al. GPT-4 technical report. Preprint at https://doi.org/10.48550/arXiv.2303.08774 (2023). Seedat, N., Imrie, F. & van der Schaar, M. Navigating data-centric artificial intelligence with DC-Check: advances, challenges, and opportunities. IEEE Trans. Liu, H., Li, C., Wu, Q. & Lee, Y. J. Visual instruction tuning. 37th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) 34892–34916 (Curran Associates Inc., 2023). Li, C. et al. LLaVA-med: training a large language-and-vision assistant for biomedicine in one day. 37th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) 28541–28564 (Curran Associates Inc., 2023). Roberts, M. et al. Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans. Kreuzberger, D., Kühl, N. & Hirschl, S. Machine learning operations (MLOps): overview, definition, and architecture. IEEE Access 11, 31866–31879 (2023). Lavin, A. et al. Technology readiness levels for machine learning systems. Nielsen, M. W. et al. Intersectional analysis for science and technology. Lekadir, K. et al. FUTURE-AI: international consensus guideline for trustworthy and deployable artificial intelligence in healthcare. Huang, Y. et al. What makes multi-modal learning better than single (provably). 35th International Conference on Neural Information Processing Systems (eds Ranzato, M. et al.) 10944–10956 (Curran Associates Inc., 2021). Meng, X., Babaee, H. & Karniadakis, G. E. Multi-fidelity Bayesian neural networks: algorithms and applications. Article MathSciNet Penwarden, M., Zhe, S., Narayan, A. & Kirby, R. M. Multifidelity modeling for physics-informed neural networks (PINNs). Article MathSciNet Zitnik, M. et al. Machine learning for integrating data in biology and medicine: principles, practice, and opportunities. Lunke, S. et al. Integrated multi-omics for rapid rare disease diagnosis on a national scale. Wu, E. et al. How medical AI devices are evaluated: limitations and recommendations from an analysis of FDA approvals. van Breugel, B., Liu, T., Oglic, D. & van der Schaar, M. Synthetic data in biomedicine via generative artificial intelligence. Al-Rubaie, M. & Chang, J. M. Privacy-preserving machine learning: threats and solutions. Wendland, P. et al. Generation of realistic synthetic data using multimodal neural ordinary differential equations. npj Digit. Che, L., Wang, J., Zhou, Y. & Ma, F. Multimodal federated learning: a survey. Wang, Q., Zhan, L., Thompson, P. M. & Zhou, J. Multimodal learning with incomplete modalities by knowledge distillation. 26th ACM SIGKDD International Conference on Knowledge Discovery & Data Mining 1828–1838 (Association for Computing Machinery, 2020). & Tao, D. Multi-modal factorized bilinear pooling with co-attention learning for visual question answering. IEEE International Conference on Computer Vision 1839–1848 (IEEE, 2017). Weidinger, L. et al. Holistic safety and responsibility evaluations of advanced AI models. Preprint at https://doi.org/10.48550/arXiv.2404.14068 (2024). Luccioni, S., Akiki, C., Mitchell, M. & Jernite, Y. Stable bias: evaluating societal representations in diffusion models. 37th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) 56338–56351 (Curran Associates Inc., 2023). Murdoch, W. J., Singh, C., Kumbier, K., Abbasi-Asl, R. & Yu, B. Definitions, methods, and applications in interpretable machine learning. Article MathSciNet Rudin, C. Stop explaining black box machine learning models for high stakes decisions and use interpretable models instead. Gawlikowski, J. et al. A survey of uncertainty in deep neural networks. Khojasteh, D., Davani, E., Shamsipour, A., Haghani, M. & Glamore, W. Climate change and COVID-19: interdisciplinary perspectives from two global crises. Total Environ. Elmer, T., Mepham, K. & Stadtfeld, C. Students under lockdown: comparisons of students' social networks and mental health before and during the COVID-19 crisis in Switzerland. Valdez, D., Ten Thij, M., Bathina, K., Rutter, L. A. & Bollen, J. Social media insights into US mental health during the COVID-19 pandemic: longitudinal analysis of Twitter data. Liu, Z. et al. Near-real-time monitoring of global CO2 emissions reveals the effects of the COVID-19 pandemic. Zheng, B. et al. Satellite-based estimates of decline and rebound in China's CO2 emissions during COVID-19 pandemic. Mosser, P. C. Central bank responses to COVID-19. Nicola, M. et al. The socio-economic implications of the coronavirus pandemic (COVID-19): a review. Ding, X. et al. Wearable sensing and telehealth technology with potential applications in the coronavirus pandemic. IEEE Rev. Charlton, P. H. et al. Wearable photoplethysmography for cardiovascular monitoring. Zong, Y., Mac Aodha, O. & Hospedales, T. Self-supervised multimodal learning: a survey. Yang, X., Zhang, T. & Xu, C. Cross-domain feature learning in multimedia. Han, R. et al. Randomised controlled trials evaluating artificial intelligence in clinical practice: a scoping review. Plana, D. et al. Randomized clinical trials of machine learning interventions in health care: a systematic review. Imrie, F., Davis, R. & van der Schaar, M. Multiple stakeholders drive diverse interpretability requirements for machine learning in healthcare. Mincu, D. & Roy, S. Developing robust benchmarks for driving forward AI innovation in healthcare. Soni, A. et al. in Machine Learning for Robotics Applications (eds Bianchini, M. et al.) 139–151 (Springer, 2021). Badue, C. et al. Self-driving cars: a survey. Yeong, D. J., Velasco-Hernandez, G., Barry, J. & Walsh, J. Sensor and sensor fusion technology in autonomous vehicles: a review. Yang, J. et al. Generalized predictive model for autonomous driving. IEEE/CVF Conference on Computer Vision and Pattern Recognition 14662–14672 (IEEE, 2024). Zhang, W. & Xu, J. Advanced lightweight materials for automobiles: a review. & Lundgren, B. Self-driving vehicles—an ethical overview. Chowdhury, A., Karmakar, G., Kamruzzaman, J., Jolfaei, A. & Das, R. Attacks on self-driving cars and their countermeasures: a survey. IEEE Access 8, 207308–207342 (2020). Dey, K. C., Mishra, A. & Chowdhury, M. Potential of intelligent transportation systems in mitigating adverse weather impacts on road mobility: a review. Kamran, S. S. et al. Artificial intelligence and advanced materials in automotive industry: potential applications and perspectives. Today Proc. Raissi, M., Perdikaris, P. & Karniadakis, G. E. Physics-informed neural networks: a deep learning framework for solving forward and inverse problems involving nonlinear partial differential equations. Article MathSciNet Karniadakis, G. E. et al. Physics-informed machine learning. Liu, P., Yang, R. & Xu, Z. Public acceptance of fully automated driving: effects of social trust and risk/benefit perceptions. Bi, K. et al. Accurate medium-range global weather forecasting with 3D neural networks. Lam, R. et al. Learning skillful medium-range global weather forecasting. Article MathSciNet Mathiesen, K. Rating climate risks to credit worthiness. Breitenstein, M., Ciummo, S. & Walch, F. Disclosure of Climate Change Risk in Credit Ratings. Occasional Paper Series 303 (European Central Bank, 2022). Imran, M., Ofli, F., Caragea, D. & Torralba, A. Using AI and social media multimodal content for disaster response and management: opportunities, challenges, and future directions. Tanir, T., Yildirim, E., Ferreira, C. M. & Demir, I. Social vulnerability and climate risk assessment for agricultural communities in the United States. Total Environ. Kumar, P. et al. An overview of monitoring methods for assessing the performance of nature-based solutions against natural hazards. Tkachenko, N., Jarvis, S. & Procter, R. Predicting floods with Flickr tags. Thulke, D. et al. ClimateGPT: towards AI synthesizing interdisciplinary research on climate change. Preprint at https://doi.org/10.48550/arXiv.2401.09646 (2024). Bodnar, C. et al. Aurora: a Foundation Model of the Atmosphere. Technical Report MSR-TR-2024-16 (Microsoft Research AI for Science, 2024). Rasp, S. et al. WeatherBench 2: a benchmark for the next generation of data-driven global weather models. Morshed, S. R. et al. Decoding seasonal variability of air pollutants with climate factors: a geostatistical approach using multimodal regression models for informed climate change mitigation. Reichstein, M. et al. Early warning of complex climate risk with integrated artificial intelligence. Liang, P. P. et al. MultiBench: multiscale benchmarks for multimodal representation learning. 35th Conference on Neural Information Processing Systems Track on Datasets and Benchmarks (eds Vanschoren, J. de la Fuente, J. et al. Towards a more inductive world for drug repurposing approaches. Torabi, F. et al. The common governance model: a way to avoid data segregation between existing trusted research environment. Crosswell, L. C. & Thornton, J. M. ELIXIR: a distributed infrastructure for European biological data. Trends Biotechnol. Liang, K. et al. A survey of knowledge graph reasoning on graph types: static, dynamic, and multi-modal. Archit, A. et al. Segment anything for microscopy. Li, C. et al. Multimodal foundation models: from specialists to general-purpose assistants. Fei, N. et al. Towards artificial general intelligence via a multimodal foundation model. Narayanswamy, G. et al. Scaling wearable foundation models. In The Thirteenth International Conference on Learning Representations (ICLR, 2025). Cui, H. et al. Towards multimodal foundation models in molecular cell biology. Introducing the Model Context Protocol. Anthropic https://www.anthropic.com/index/model-context-protocol (2024). Weisz, J. D. et al. Design principles for generative AI applications. 2024 CHI Conference on Human Factors in Computing Systems (eds Mueller, F. F. et al.) 378 (Association for Computing Machinery, 2024). Li, J., Li, D., Xiong, C. & Hoi, S. BLIP: Bootstrapping language–image pre-training for unified vision–language understanding and generation. 39th International Conference on Machine Learning (eds Chaudhuri, K. et al.) 12888–12900 (PMLR, 2022). Li, J., Li, D., Savarese, S. & Hoi, S. BLIP-2: Bootstrapping language–image pre-training with frozen image encoders and large language models. 40th International Conference on Machine Learning (eds Krause, A. et al.) 19730–19742 (PMLR, 2023). Driess, D. et al. PaLM-E: an embodied multimodal language model. 40th International Conference on Machine Learning (eds Krause, A. et al.) 8469–8488 (PMLR, 2023). Tan, Z. et al. Large language models for data annotation and synthesis: a survey. 2024 Conference on Empirical Methods in Natural Language Processing (eds Al-Onaizan, Y. et al.) 930–957 (Association for Computational Linguistics, 2024). Ding, B. et al. Data augmentation using LLMs: data perspectives, learning paradigms and challenges. In Findings of the Association for Computational Linguistics ACL 2024 (eds Ku, L.-W. et al.) 1679–1705 (Association for Computational Linguistics, 2024). Chan, A.-W. et al. Reporting guidelines for clinical trial reports for interventions involving artificial intelligence: the CONSORT-AI extension. de Hond, A. Guidelines and quality criteria for artificial intelligence-based prediction models in healthcare: a scoping review. npj Digit. Tomczak, K., Czerwińska, P. & Wiznerowicz, M. The Cancer Genome Atlas (TCGA): an immeasurable source of knowledge. Johnson, A. E. W. et al. MIMIC-IV, a freely accessible electronic health record dataset. Tong, E., Zadeh, A., Jones, C. & Morency, L.-P. Combating human trafficking with multimodal deep models. 55th Annual Meeting of the Association for Computational Linguistics (eds Barzilay, R. et al.) 1547–1556 (Association for Computational Linguistics, 2017). Razi, A. et al. Instagram data donation: a case study on collecting ecologically valid social media data for the purpose of adolescent online risk detection. In CHI Conference on Human Factors in Computing Systems Extended Abstracts (eds Barbosa, S. et al.) 1–9 (Association for Computing Machinery, 2022). Caesar, H. et al. nuScenes: a multimodal dataset for autonomous driving. IEEE/CVF Conference on Computer Vision and Pattern Recognition 11621–11631 (IEEE, 2020). Yu, H. et al. DAIR-V2X: a large-scale dataset for vehicle–infrastructure cooperative 3D object detection. IEEE/CVF Conference on Computer Vision and Pattern Recognition 21361–21370 (IEEE, 2022). Jain, A. et al. Commentary: The Materials Project: a materials genome approach to accelerating materials innovation. Lu, P. et al. MathVista: Evaluating mathematical reasoning of foundation models in visual contexts. In International Conference on Learning Representations (ICLR, 2024). iNaturalist research-grade observations. Hersbach, H. et al. The ERA5 global reanalysis. Mathur, P. et al. Monopoly: financial prediction from monetary policy conference videos using multimodal cues. 30th ACM International Conference on Multimedia 2276–2285 (Association for Computing Machinery, 2022). Cheng, D., Yang, F., Xiang, S. & Liu, J. Financial time series forecasting with multi-modality graph neural network. Chen, K. et al. The operational medium-range deterministic weather forecasting can be extended beyond a 10-day lead time. Jin, Q. et al. Spatiotemporal inference network for precipitation nowcasting with multimodal fusion. Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Johnson, A. E. W. et al. MIMIC-III, a freely accessible critical care database. This work was enabled and supported by the Alan Turing Institute. We thank T. Chakraborty and C. Li for inspiring this work, D. A. Clifton for his support and T. Dunstan for contributing to the climate change adaptation section. is supported by donations from D. Naik and S. Naik. is supported by EPSRC (grant EP/Y017544/1). was supported by EPSRC (grant EP/Y028880/1). is supported by UKRI CDT in AI for Healthcare (grant EP/S023283/1). A.G. is supported by the Research Council of Norway (secureIT project 288787). is supported by EPSRC (grant EP/X031276/1). is supported by UKRI CDT in AI-enabled Healthcare (grant EP/S021612/1). is supported by the Royal Society (grant IEC\NSFC\233558). is supported by NERC (grant NE/W004747/1). is supported by UKRI CDT in Application of Artificial Intelligence to the study of Environmental Risks (grant EP/S022961/1). is supported by Google DeepMind. is supported by NIHR Barts BRC (grant NIHR203330). is supported by UKRI CDT in Accountable, Responsible and Transparent AI (grant EP/S023437/1). L.F. is supported by MRC (grant MR/W006804/1). N.J. is supported by the EU's co-funded HE project MuseIT (grant 101061441). is supported by St George's Hospital Charity. A.C.-C. is supported by EPSRC (grant EP/Y028880/1). is supported by MRC (grant MR/X030075/1). is supported by the Royal Society (grant GS\R2\242355). was supported by the Royal Academy of Engineering (grant RF\201819\18\109). is supported by EPSRC (grant EP/Y009800/1). is supported by BHF (grant FS/20/20/34626). is supported by EPSRC (grant UKRI396). The views expressed in this material are those of the authors and do not necessarily represent the views of their affiliated institutions or funders. These authors contributed equally: Xianyuan Liu, Jiayang Zhang, Shuo Zhou, Haiping Lu. Centre for Machine Intelligence, University of Sheffield, Sheffield, UK Xianyuan Liu, Jiayang Zhang & Mohammod N. I. Suvon School of Computer Science, University of Sheffield, Sheffield, UK Shuo Zhou, Sina Tabakhi, Peizhen Bai, Chen Chen, Roger K. Moore & Haiping Lu The Alan Turing Institute, London, UK Thijs L. van der Plas, Louisa van Zeeland & Alejandro Coca-Castro Department of Metabolism, Digestion and Reproduction, Imperial College London, London, UK Department of Applied AI, Simula Research Laboratory, Oslo, Norway Information School, University of Sheffield, Sheffield, UK NHS England, Leeds, UK Institute of Health Informatics, University College London, London, UK Christopher Tomlinson & Jinge Wu Department of Engineering, King's College London, London, UK Yuhan Wang, Hak-Keung Lam & Oya Celiktutan Department of Computing Science, University of Aberdeen, Aberdeen, UK School of Informatics, University of Edinburgh, Edinburgh, UK School of Engineering Mathematics and Technology, University of Bristol, Bristol, UK Xiang Li, Yan Ge & Zahraa S. Abdallah Department of Informatics, King's College London, London, UK Arunav Das & Nitisha Jain Department of Earth Sciences, University of Cambridge, Cambridge, UK Orlando Timmerman Department of Computer Science, University of Manchester, Manchester, UK Department of Computer Science, Queen Mary University of London, London, UK Zahraa Al Sahili Digital Environment Research Institute, Queen Mary University of London, London, UK Omnia Alwazzan & Gregory G. Slabaugh Department of Computer Science, University of Bath, Bath, UK Department of Classics, King's College London, London, UK School of Electrical, Electronic and Mechanical Engineering, University of Bristol, Bristol, UK School of Engineering, University of the West of England, Bristol, UK Cancer Research UK Scotland Institute, Glasgow, UK School of Business and Management, Queen Mary University of London, London, UK Natalia Efremova & Valentin Danchev City St George's University of London, London, UK British Antarctic Survey, Cambridge, UK School of Health and Wellbeing, University of Glasgow, Glasgow, UK Chief Data & AI Office, Lloyds Banking Group, London, UK Nataliya Tkachenko School of Life Course & Population Sciences, King's College London, London, UK Lei Lu Institute of Biomedical Engineering, University of Oxford, Oxford, UK Department of Computer Science, Hong Kong Baptist University, Hong Kong, China Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK Peter H. Charlton Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the manuscript. designed the figures. coordinated the entire project. All authors contributed to writing, resources or editing. Correspondence to Haiping Lu. is a scientific advisory board member at BioAI Health. provides consulting services to Cambridge University Technical Services for wearable manufacturers. The remaining authors declare no competing interests. Nature Machine Intelligence thanks Girmaw Abebe Tadesse and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Statistical data for Supplementary Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Liu, X., Zhang, J., Zhou, S. et al. Towards deployment-centric multimodal AI beyond vision and language. Nat Mach Intell  (2025). DOI: https://doi.org/10.1038/s42256-025-01116-5 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Machine Intelligence (Nat Mach Intell) © 2025 Springer Nature Limited Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42256-025-01119-2'>Tailored structured peptide design with a key-cutting machine approach</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 09:31:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. A preprint version of the article is available at bioRxiv. Computational protein and peptide design is emerging as a transformative framework for engineering macromolecules with precise structures and functions, offering innovative solutions in medicine, biotechnology and materials science. However, current methods predominantly rely on generative models, which are expensive to train and modify. Here, we introduce the Key-Cutting Machine (KCM), an optimization-based platform that iteratively leverages structure prediction to match desired backbone geometries. KCM requires only a single graphics processing unit and enables seamless incorporation of user-defined requirements into the objective function, circumventing the high retraining costs typical of generative models while allowing straightforward assessment of measurable properties. By employing an estimation of distribution algorithm, KCM optimizes sequences on the basis of geometric, physicochemical and energetic criteria. We benchmarked its performance on α-helices, β-sheets, a combination of both and unstructured regions, demonstrating precise backbone geometry design. As a proof of concept, we applied KCM to antimicrobial peptide design by using a template antimicrobial peptide as the ‘key', yielding a candidate with potent in vitro activity against multiple bacterial strains and efficacy in a murine infection model. KCM thus emerges as a robust tool for de novo protein and peptide design, offering a flexible paradigm for replicating and extending the structure–function relationships of existing templates. Protein and peptide design aims to construct synthetic proteins and peptides or modify existing ones to achieve new functionalities1,2. Successful designs include proteins that inhibit viral infections such as influenza3,4, SARS-CoV-25 and HIV6, as well as enzymes that catalyse reactions for which no natural counterparts were previously known7. Additional applications include biosensors to detect various molecules8, including fentanyl9, self-assembling protein nanoparticle vaccines for SARS-CoV-210, self-assembling nanomaterials11 and biological logic gates12. In the case of peptides, a simple generative model was proposed to design antimicrobial peptides against Cutibacterium acnes13. ProT-Diff14, another generative model, produced peptides; of these, 34 showed antibacterial activity against both Gram-positive and Gram-negative bacteria, as well as in vivo activity against a clinically relevant drug-resistant Escherichia coli strain. These advances highlight the broad therapeutic and biotechnological potential of computational protein and peptide design15. Computational protein and peptide design is often framed as a combinatorial optimization problem, where the goal is to identify an amino-acid sequence that folds into a structure with minimal free energy1. This challenge is partially mitigated by discretizing side-chain conformations (rotamers)16. Thus, the task reduces to finding the optimal combination of rotamers that minimizes free energy for a given backbone. Alternative formulations include mixed-integer linear programming17 and cost function networks18. However, the protein design problem remains extraordinarily complex19,20. Deep learning methods have emerged as powerful tools for protein design. Given a structure in Protein Data Bank (PDB) format, these models predict the amino-acid sequences most likely to fold into the provided structure. Later methods improved upon ProteinMPNN and ESM-IF1 by enhancing sequence recovery and reducing perplexity through the incorporation of additional structural features. For example, SPDesign24 leverages structurally similar templates and builds a sequence profile to guide design. SPIN-CGNN25 computes distance-based contact maps and encodes them into a graph representation. PiFold26 extracts spatially invariant geometric features from atomic coordinates using a residue-level featurizer. These strategies support more accurate, flexible and context-aware sequence prediction. These models, which include RFDiffusion27 and GRADE-IF28, can generate, from random noise, backbone structures that can then be paired with protein design networks (for example, ProteinMPNN) to produce corresponding sequences. Additionally, the integration of language model innovations has shown promise in diverse protein design tasks. For example, StructureGPT29 has been used to suggest sequence modifications that would improve solubility and structural stability. Similarly, TPGen30 has demonstrated the ability to generate short, diverse sequences that reliably fold into predefined secondary structures, as confirmed by molecular dynamics simulations. The main purpose of generative models is to explore new sequences and structures associated with new functions. However, a major limitation of existing generative models is their substantial computational cost, primarily due to the need for retraining from scratch whenever a new measurable property is introduced into the loss function. For instance, training ESM-IF1 incurs a substantial computational cost. The model was trained on over 19,000 protein structures from the CATH 4.2 database (40% non-redundant) and supplemented with 12 million AlphaFold 2-predicted structures. Each epoch required 88 h on 32 graphics processing units (GPUs), totalling approximately 653 GPU-days. This level of resource consumption highlights the impracticality of retraining such models for iterative design tasks. To overcome this limitation, we propose an optimization-based model for protein and peptide design—referred to as the Key-Cutting Machine (KCM) model (Fig. KCM requires only a single GPU and enables the seamless incorporation of user-defined requirements into the objective function, thus avoiding the repeated retraining overhead typical of generative models and simplifying the assessment of measurable properties. b, Distribution for GDT_TS and RMSD_S from the best 100 designs according to their fitness, for each protein. Continuous colour range for both GDT_TS and RMSD_S. Correspondence between characters and proteins for b–d: A, 1EMN; B, 1LQL; C, 1NP6; D, 1O6W; E, 2E45; F, 2KXQ; G, 1S7M; H, 1P9N; I, 5U1Y; J, 3CLQ; K, 3SB1; L, 2QQ8; M, 3M9Q; N, 3H25; O, 3EWK; P, 3C8V; Q, 2QIW; R, 2OAR; S, 2LKM; T, 1MSL; U, 3W68; V, 1R5L; W, 1N7D. In this framework, the ‘key' corresponds to the target structure and its sequence, and a structure-prediction method generates multiple copies of this key. KCM then refines these sequences by comparing the original key with its predicted copies. We implement an estimation of distribution algorithm (EDA)31 using an island model and employ ESMFold as the structure predictor to guide the optimization process. Importantly, the KCM algorithm does not require training, which allows it to be applied directly without the need for large training datasets or computationally expensive pretraining procedures. Other approaches that use evolutionary algorithms with a structure predictor in the loop address different design goals32,33. We computationally designed derivatives of this small peptide, synthesized them chemically and tested their antimicrobial activity both in vitro and in vivo. Our methodology consists of three stages (Fig. First, we define an optimization model for the KCM approach, characterized by an objective function to be maximized. Next, we propose an EDA to solve this model. Finally, we apply the algorithm to a dataset of proteins with known sequences and secondary structures, including α-helices, β-sheets and unstructured proteins. We also tested the KCM on proteins that are a combination of α-helices and β-sheets. The protein design problem is daunting because of the immense sequence space and the unpredictable mapping from amino-acid sequences to structures35. Even a single amino-acid mutation can markedly alter the structure of a given protein or peptide. We hypothesize that our EDA can accurately estimate the distribution of structures within sequence space36,37. In our experiments, protein designs dominated by α-helices required fewer generations to converge than their β-sheet counterparts, in part because α-helices are typically shorter. A termination condition of 100 generations was applied to α-helical proteins (for example, PDB identifiers (IDs) 5U1Y, 3CLQ, 3SB1, 2QQ8, 3M9Q, 3H25, 3EWK, 3C8V, 2QIW, 2OAR, 2LKM and 1MSL) and proteins lacking defined secondary structure (PDB IDs 3W68, 1R5L and 1N7D). For β-sheet proteins (PDB IDs 1EMN, 1LQL, 1NP6, 1O6W, 2E45, 2KXQ, 1S7M and 1P9N), we used a termination condition of 1,000 generations or until the global distance test total score (GDT_TS)38 reached at least 0.9. When the threshold was met earlier (for example, PDB IDs 1O6W, 2E45, 2KXQ, 1S7M and 1LQL), the algorithm was halted before 1,000 generations. For proteins with PDB IDs 1EMN, 1NP6 and 1P9N, the algorithm completed the full 1,000 generations (parameters are listed in Supplementary Table 1). For proteins with a combination of α-helices and β-sheets the termination criterion was a GDT_TS of 0.9 or higher or 350 generations, whichever came first. In each generation, the number of objective function evaluations was calculated by multiplying the population size (five individuals) by the samples generated per individual (five) and the total number of islands (20), yielding 500 evaluations. In the 21st island, the individuals are no longer stochastic generators but amino-acid sequences, so we have 25 sequences in this island, requiring 25 evaluations, resulting in 525 objective function calls per generation. For proteins primarily composed of α-helices, GDT_TS distributions trended toward higher values, approaching 1, whereas standard root mean square deviation (RMSD_S) distributions approached 0 (Fig. This pattern indicates high structural similarity and stability among the designs, with minimal variance. This variance may stem from disordered regions at one terminus, which lack well-defined secondary structure and thus need more generations for convergence. Proteins dominated by α-helices showed narrow GDT_TS s.d. values (0.00–0.07) yet larger RMSD_S variation (0.04–0.42) (Supplementary Table 2). Although higher RMSD_S variability helps prevent premature convergence, β-sheet proteins typically required longer runs and showed more structural diversity, evident in broader RMSD_S and GDT_TS distributions. Because β-sheet proteins had an average length of 32 residues—nearly double that of α-helical proteins (18 residues)—their search space was inherently more complex. Unstructured proteins posed even greater challenges for the algorithm. Data points are colour-coded by secondary structure type: yellow (β-sheet), red (α-helix) or blue (unstructured). Red points cluster toward higher GDT_TS and lower RMSD_S values, whereas blue points show lower accuracy. Protein 2QQ8 (L), which contains an unstructured region, stands out with lower GDT_TS and RMSD_S scores compared with other α-helical proteins. Superimposing the highest-fitness designs onto their reference structures (Fig. 1d) reveals notable structural similarity, particularly in α-helical and β-sheet regions. Proteins with unstructured segments showed lower similarity, although extended runs did improve alignment. Notably, the highest sequence identity between the best designs and the reference sequences was merely 24%, with an average of 11% (Supplementary Table 2), suggesting that the algorithm can converge on structurally similar solutions despite low sequence identity. On this same dataset of 23 proteins, using a Bayesian ranking39, we compared KCM with three well-known generative models: ProteinMPNN, ESM-IF1 and ProteinSolver. For a given target backbone, ProteinMPNN generated 256 sequences in four runs, while ESM-IF1 and ProteinSolver each produced 250 sequences. For KCM, we likewise chose the best sequences according to each criterion. For a fair comparison, all solutions from each algorithm were sorted on the basis of the fitness function used in our algorithm. When examining only 50 solutions (Supplementary Fig. 1a–f and Supplementary Table 3), KCM surpassed the other approaches in RMSD but lagged behind ESM-IF1 and ProteinMPNN in GDT_TS. When 250 solutions were considered (Supplementary Fig. 1g–l and Supplementary Table 4), KCM again outperformed all other methods in RMSD but fell short of ESM-IF1 in GDT_TS. Note that these proteins might have been used in the generative models' training process. In this case, of the 23 proteins tested, only two (PDB IDs 1P9N and 5U1Y) were not included in CATH 4.2, from which the training sets of ProteinMPNN and ESM-IF1 were taken. To analyse the performance of KCM when proteins with a combination of α-helix and β-sheet are included, eight new designs were undertaken (Supplementary Table 5); the results are reported in Supplementary Table 6. Here six out of eight designs achieved a GDT_TS larger than 0.9; this is better than the case for β-sheet-only designs, where only four out of eight achieved a GDT_TS larger than 0.9, even when more computation effort was allowed. The superpositions of the targets and their corresponding designs are shown in Supplementary Fig. To estimate the cost of designing longer sequences, we evaluated the structure prediction time of ESMFold on randomly generated sequences ranging from 10 to 400 residues. Predictions for sequences up to 100 residues required less than 2 s each, while sequences of 400 residues required approximately 10 times more computation time (Supplementary Fig. The algorithm was also applied to design two proteins of 100 residues each, PDB IDs 2F77 and 2HLQ. After 2,000 generations the GDT_TS reached 0.38 for both targets, showing a limitation in designing proteins of this size without parameter tuning. As a proof of concept, we selected IDR-2009, a 12-residue peptide (sequence KWRLLIRWRIQK) known for its potent antimicrobial activity34. This peptide was chosen for practical reasons: its antimicrobial activity can be readily validated in vitro and short sequences are amenable to synthesis. Additionally, KCM performed well on α-helices (Supplementary Table 2). We tested multiple objective function configurations to evaluate whether KCM could generate variants with favourable solubility and synthetic feasibility. We did not assume that the peptide's function directly depends on its geometry. Instead, we investigated whether mirroring its predicted backbone and its amino-acid properties and minimizing its energy would yield sequences with comparable structure and properties. To design a set of peptides using the IDR-2009 backbone as a template, we proceeded as follows. Within AlphaFold 2, we used the default parameters but increased to 48 recycles and three relaxation iterations. Even though this implementation of AlphaFold 2 does not use structural templates and uses a limited-size database to construct the multiple sequence alignment, the resulting average predicted local distance difference test was higher than 0.8, which is considered a confident predicted local distance difference test score42,43. This resulting backbone structure served as the input for our KCM. (2) We conducted the design under four different schemes, primarily varying the protein similarity function. All terms in equation (1) were applied. where KL is the mean Kullback–Leibler divergence (Supplementary equation (14), see details in Supplementary Section 1.7) of the descriptors computed from the sequence. Notice that the difference between equations (3) and (1) is that in equation (3) the geometric term is neglected and therefore so is the energy term, the descriptor difference remaining as the main component in the objective function, while in equation (1) all terms are taken into account. For each scheme, we generated a set of sequences and ranked them in descending order according to their fitness (Supplementary Table 7 lists the evolutionary algorithm parameters). We then analysed these ranked sequences with AMPDiscover44 to predict their potential antimicrobial activity. We further examined the top 10 sequences from each scheme (Supplementary Tables 8–10). We synthesized 12 sequences selected for their ease of synthesis and low aggregation potential: five from AT, four from NE and three from NG. We excluded the ND group because its peptides were challenging to synthesize and solubilize. We determined their minimum inhibitory concentrations (MICs) against 11 clinically relevant strains, including the ESKAPEE pathogens. Nine of the 12 peptides (75%) exhibited MIC values of ≤64 μmol l−1 against at least one strain (Fig. 2a,b), surpassing hit rates from many existing machine learning/deep learning methods. The three inactive peptides (AT1, AT2, AT3) belonged to the AT group, had low net charge (<2) and displayed lower normalized hydrophobicity. Further investigation is needed to confirm this hypothesis. a, Structure prediction of the synthesized peptides. b, Heatmap of the antimicrobial activities of the peptides against 11 clinically relevant pathogens, including four strains resistant to conventional antibiotics. Briefly, 106 bacterial cells and serially diluted peptides (1-64 μmol l−1) were incubated at 37 °C. One day after treatment, the optical density at 600 nm was measured in a microplate reader to evaluate bacterial growth in the presence of the peptides. c–f, Ternary plots showing the percentage of secondary structure for each peptide (at 50 μmol l−1) in four different solvents: water (c), 60% TFE in water (d), 10 mmol l−1 SDS in water (e) and 50% methanol (MeOH) in water (f). Secondary structure fractions were calculated using the BeStSel server65. Because the peptides were designed using different criteria, we sought to assess their secondary-structure tendencies. To evaluate secondary structure, we subjected the peptides to four different media: water (Fig. 4b, e), a membrane-mimicking environment (SDS at 10 mmol l−1; Fig. When geometric similarity criteria were omitted and energy terms were minimally weighted (NG group), the designed peptides displayed greater structural plasticity. They were initially unstructured forms in water and the methanol/water mixture. However, they adopted highly helical conformations in the TFE/water mixture and predominantly β-like conformations in the membrane-mimicking medium. Peptides from the AT and NE groups remained unstructured in water yet adopted mostly β-like conformations in SDS and the methanol/water mixture. Notably, in the presence of SDS micelles, NE1 showed an equal proportion of β-like and helical structures. Conversely, AT3 and NE3, which were among the more amphiphilic peptides, predominantly retained a β-like structure even in the helix-inducing medium. No clear trend emerged linking secondary structure directly to antimicrobial activity. We next explored the membrane-related mechanism of action of a subset of peptides (AT4, NE1, NE2, NG1, NG3) against Acinetobacter baumannii and vancomycin-resistant Enterococcus faecalis (Fig. Using N-phenyl-1-naphthylamine (NPN) uptake assays (Fig. 5a) and membrane depolarization assays with 3,3′-dipropylthiadicarbocyanine iodide (DiSC3-5) (Fig. 5b, c), we found that all peptides disrupted the bacterial membrane potential to varying degrees. NE1 and NG3 were particularly effective at depolarizing the cytoplasmic membrane of A. baumannii. Although none outperformed polymyxin B in outer-membrane permeabilization, several peptides matched levofloxacin's effects, underscoring their potential as membrane-targeting agents. a, To assess whether the peptides act on bacterial membranes, all active peptides against A. baumannii American Type Culture Collection (ATCC) 19606 were subjected to outer-membrane permeabilization, and peptides active against A. baumannii ATCC 19606 and vancomycin-resistant E. faecalis ATCC 700802 were tested in cytoplasmic-membrane depolarization assays. b, The fluorescent probe NPN was used to assess membrane permeabilization induced by the tested peptides. c,d, The fluorescent probe DiSC3-5 was used to evaluate membrane depolarization caused by the designed peptides in A. baumannii ATCC 19606 (c) and vancomycin-resistant E. faecalis ATCC 700802 (d). e, Cytotoxic concentrations leading to 50% cell lysis (CC50) were determined by interpolating the dose–response data using a nonlinear regression curve. All experiments were performed in three independent replicates. Cytotoxicity was evaluated using human embryonic kidney (HEK293T) cells (Fig. None of the 12 peptides caused substantial cytotoxicity at the tested concentrations (4–64 μmol l−1), contrasting with the parental IDR-2009 peptide, which did show cytotoxicity34. The lowest cytotoxic concentration leading to 50% cell lysis among inactive peptides (AT3) was 269 μmol l−1, indicating a favourable therapeutic window. We tested NE2 and NG1 in two murine models: skin abscess46,47,48,49 (Fig. 4) and a deep thigh infection50,51,52 (Supplementary Fig. In both models, the peptides reduced A. baumannii bacterial loads by up to two orders of magnitude, comparable to the last-resort antibiotic polymyxin B and to our other antibiotic control levofloxacin. No weight loss or skin damage was observed, indicating good tolerability. These results confirm that KCM-designed peptides can achieve clinically relevant anti-infective efficacy. a, Schematic representation of the skin abscess mouse model used to assess the anti-infective activity of the designed peptides (n = 6) against A. baumannii ATCC 19606. b, NE2 and NG1, administered at their MIC (32 and 4 μmol l−1, respectively) in a single dose post-infection, inhibited the proliferation of the infection for up to 4 d after treatment compared with the untreated control group. Notably, NE2 reduced the infection in some mice, demonstrating activity comparable to the control antibiotics, polymyxin B and levofloxacin. Statistical significance in panel b was determined using one-way ANOVA followed by Dunnett's test; P values are shown in the graphs. This study introduces and validates the KCM approach for computational structured peptide design. In contrast to purely generative frameworks, which use trained models to rapidly generate large numbers of candidate sequences, KCM iteratively refines candidate sequences through structure prediction and optimization. The results observed with mixed secondary structures such as α-helices and β-sheets support the idea that the α-helices are easier to design than β-sheets; when we have only β-sheets we have a lower success percentage (50%, Supplementary Table 2) than when we design α-helices and β-sheets (success percentage of 75%, Supplementary Table 5). When we have only α-helices, KCM achieves 91.6% success (Supplementary Table 2). Here, designs that achieved GDT_TS of 90% or more are considered successful. Overall, our results demonstrate that KCM can converge on sequences whose backbone geometries strongly resemble those of target proteins, spanning α-helices, β-sheets, combinations of α-helices and β-sheets, and unstructured regions, even when sequence identity is low. This suggests that KCM effectively explores the structural landscape by continuously updating the distribution of candidate sequences on the basis of both geometric and physicochemical criteria. However, without a proper parameter tuning, KCM was limited when designing proteins composed of 100 amino-acid residues or larger. A key advantage of KCM is its ability to seamlessly integrate a wide range of objective functions beyond backbone geometry. In particular, it can incorporate Kullback–Leibler divergence or Jeffrey's distance for amino-acid descriptors, as well as energy terms and other chemically relevant properties. This flexibility allows researchers to add or remove objectives—such as solubility, stability or functional motifs—within a single GPU environment, eliminating the need to retrain a deep learning model when new properties are introduced into the loss function. In contrast to generative models that require substantial computational resources for retraining, KCM offers an adaptable, resource-efficient solution for computational protein design. Another key advantage of the proposed KCM framework lies in its interpretability and accessibility. Unlike deep learning-based generative models, KCM does not require a training phase; instead, it uses an explicitly defined objective function to guide the design process, enabling users to control and tailor design preferences directly. In addition, KCM employs an island-based evolutionary paradigm that is inherently parallelizable. While the current implementation operates on a single GPU, each island can, in principle, be executed independently across multiple processors or GPUs. This conceptual scalability allows KCM to benefit from parallel environments when available, while remaining efficient and accessible on modest hardware. Beyond benchmarking KCM on proteins of diverse secondary structure, we applied it to the design of antimicrobial peptides. Despite not explicitly encoding direct activity constraints into our fitness function, the KCM-derived peptides retained antimicrobial activity that was comparable to or exceeded many machine learning/deep learning-based designs reported in the literature13,14. Several designs also exhibited lower cytotoxicity than the parental antimicrobial peptide (IDR-2009), demonstrating that KCM can discover sequences with improved therapeutic windows. Structural analyses revealed that relaxing energy and geometric similarity constraints leads to higher structural plasticity, with designed sequences frequently transitioning between helical and β-like conformations depending on environmental factors. Notably, this plasticity did not necessarily reduce antimicrobial activity, suggesting that multiple structural pathways can yield effective membrane-disrupting peptides. These outcomes highlight the need to balance structural and energetic constraints with functional sequence-space exploration. In summary, KCM provides an alternative paradigm for peptide and protein design, combining the adaptability of an iterative, optimization-based framework with the power of modern structure-prediction tools. Its demonstrated success in designing peptides with potent antimicrobial activity and reduced toxicity highlights the method's potential in therapeutic development. Future work may explore more sophisticated descriptors—such as immunogenicity, proteolytic stability and post-translational modifications—as well as advanced multi-objective strategies and protein–water interactions, especially in the case of intrinsically disordered proteins, to enhance both throughput and exploration of sequence space. Ultimately, the KCM model represents a promising step toward more versatile and customizable computational design methodologies for proteins and peptides. Despite these advantages, several challenges remain with the KCM model. Its reliance on iterative structure prediction makes it more computationally intensive than standard pretrained generative approaches, such as ProteinMPNN or ESM-IF1, which can complete designs in seconds. For example, designing an α-helical sequence in our benchmark set requires about 12 h on a single GPU (RTX 3090 Ti) coupled with an Intel i5 CPU with 32 GB of random-access memory. This task is particularly challenging for short, dynamic peptides, which may adopt multiple conformations in solution. Integrating ensemble-based structure prediction or incorporating biophysical data (for example, nuclear magnetic resonance) could improve accuracy for more flexible targets. KCM is currently limited for larger proteins (100 amino-acid residues or larger), because population size and number of generations will need to increase considerably. This limits KCM's current design capability to structured peptides. Nonetheless, a key benefit of the KCM framework is that it does not require retraining. For instance, incorporating new design constraints or targeting a different functionality in a traditional generative model typically necessitates costly model retraining. In contrast, KCM enables these changes to be implemented directly by modifying the objective function, eliminating the need for reoptimization of the generative backbone. This affords greater flexibility in exploring diverse design criteria. While repeated structure predictions introduce computational overhead, this cost may be mitigated in future iterations through parallelization strategies or the integration of faster structure predictors. Additionally, a single key (design target) was used to generate copies (peptide variants) for experimental testing. Further validation is needed to fully establish the generalizability of the proposed method for functional design. The objective function that we propose in this work for the key-cutting machine approach consists of a combination of different similarity measures between the structure, energy and chemical/evolutionary descriptors of a designed protein and those of the target protein. The Rosetta energy function REF1556 is used to measure the energy of a protein structure. Given the descriptor vectors for a model's sequence and the reference's sequence, the similarity between them can be computed by using Jeffrey's distance58 (described in detail in Supplementary Section 1). Notice that all these criteria measure the similarity or difference between a particular aspect of a designed protein and the reference protein. For example, RMSD_S, TM_S, GDT_TS and RMSD_DM all measure structural similarity. In this case, the structure of a designed sequence is predicted using ESMFold, and the resulting structure is then superimposed with the reference structure. On the other hand, Jeffrey's distance between ESM-2 and iLearn descriptor vectors measures how similar the two proteins are in terms of both iLearn and ESM-2 descriptors. The objective function to be maximized by our protein design algorithm is given as and Ed and Er are, respectively, the energy values of the model and reference structures, both calculated using the Rosetta energy function. Note that the number of such vectors for each protein is ten, where nine are computed with iLearn and one with ESM-2. Finally, ω ∈ R is a parameter for contracting or dilating the exponential function. The objective function is unaffected by rotations and translations of the structure, since both the model and reference structures are superimposed before calculating all criteria. Also, each of the terms f1, f2 and f3 contributes with a different magnitude to the total value of f. Since the maximum value given by the said function is 7, f1 represents four-sevenths of the total value, while f2 and f3 represent one-seventh and two-sevenths of the total, respectively. The weights for f1, f2 and f3 were empirically determined; identifying the optimal weighting for each component of the objective function remains an important direction for future work. Function f will also have values close to zero when the model and reference proteins are different in all the criteria considered. An important advantage of KCM is its ability to seamlessly incorporate user-defined constraints. For example, if a user wishes to enforce the presence of hydrophobic amino-acid residues at specific positions (for example, positions i and j), the sequence generators (Supplementary Section 2) can be configured to assign zero probability to amino-acid residues that do not meet this criterion at those positions. Additionally, if the user aims to favour properties such as solubility or reduced aggregation, machine learning-based predictors can be integrated to define additional terms in the objective function, as demonstrated in a related approach59. The islands evolve synchronously and within each island we have a general procedure (Fig. That is, each individual in the population encodes a probability distribution over amino acids at each position in the sequence, on the basis of one of six stochastic generator models. Sequences are sampled from these distributions and evaluated using the objective function, with top performers retained for the next generation. Details of the six generator models are provided in Supplementary Table 12. The second component is a three-dimensional structure predictor; in this case, ESMFold was used. It was chosen over the more accurate and confident AlphaFold 2 due to its notably faster prediction. Finally, the fourth module is a repository where the top t sequences generated within the islands are stored according to their fitness. The island model is equipped with a communication strategy. At the end of each generation, with probability P, two islands can collaborate, and the information shared is through the sequences. With probability proportional to their fitness, m1 sequences are selected from the t stored in the first island. The same procedure is performed in reverse order. Each island from 1 to 20 is equipped with a mechanism that operates as an EDA-type evolutionary algorithm (Fig. Each island from 1 to 20 will have pop individuals (stochastic generators) with the same structure. In turn, these generators depend on how the parameters that define them are updated. Furthermore, an additional island is introduced, where a traditional genetic algorithm is implemented (island 21). All islands evolve synchronously, but structure prediction presents a computational bottleneck. Although each island independently generates candidate sequences, all sequences rely on the same instance of ESMFold for structure prediction, which is executed sequentially on a single GPU. The individuals on the islands are constructed from six stochastic models (Supplementary Table 12). In model 1, for each position in the sequence, the probability of occurrence of each amino acid is determined. The individuals on islands 1, 2, 3 and 4 are constructed on the basis of this model. Models 2–4 include knowledge of amino acids—for example, how they are grouped on the basis of properties such as polarity, propensity to form part of a specific secondary structure or evolutionary similarity. In this way, three hierarchical models are constructed, where for each position in the sequence the probability of belonging to one of the groups is determined first, and then, given the current group, the probability of belonging to the next group is determined, and so on until the amino-acid level is reached. The individuals on islands 5–16 are constructed on the basis of these models. In this network, an arc is drawn from each position to the next, reflecting the dependence between successive positions in the sequence. The individuals on islands 17 and 18 are constructed on the basis of model 5. Finally, model 6 represents a Markov chain between each pair of consecutive positions in the amino-acid sequence. Additionally, the update of the individuals' parameters is carried out under four different schemes. In the second scheme (B, Supplementary Table 12), the parameters (probabilities) are calculated from the absolute frequencies of appearance but resetting these every ngr generations (islands 3, 7, 11, 15, 18 and 19). To update the parameters of the stochastic generator in generation i, two sets of sequences are selected. The first set contains m* sequences, while the second set contains t* sequences. To construct the second set, t* sequences are selected from the repository that contains the t sequences with the highest fitness generated in each island (t* ≤ t). The implemented island model, where each island employs a distinct stochastic generator within an EDA framework, seeks to maximize both exploration and exploitation of the vast protein sequence space. The six stochastic models capture complementary biological and statistical properties, from simple position-wise amino-acid probabilities to hierarchical and context-aware models such as Bayesian networks and Markov chains. Combined with four diverse parameter update strategies, this design introduces a range of learning dynamics across islands. Such diversity improves the algorithm's ability to escape local optima, enhances robustness across protein classes and enables the transfer of information through migration. A traditional genetic algorithm implemented on island 21 further complements this set-up, supporting recombination-driven exploration. This heterogeneous, multi-island architecture provides a flexible and scalable foundation for general protein sequence design. To test the quality of the solutions produced by the KCM, we selected a small subset of proteins from the CATH Protein Structure Classification database v.4.3.0, which is a free and publicly available online resource that provides information on the evolutionary relationships of protein domains61 and classifies them according to the secondary structures that compose them. This database is available at http://download.cathdb.info/cath/releases/all-releases/v4_3_0/cath-classification-data/cath-domain-description-file-v4_3_0.txt. In particular, we selected protein segments containing mainly α-helices and β-sheets. We also selected some proteins (Supplementary Table 14) that did not have well-defined secondary structures. In total, 23 proteins were selected, each between 15 and 33 residues long. We selected these proteins because they represent the three most frequent secondary structure types found in experimentally determined protein structures: α-helices, β-sheets and disordered regions. Additionally, we included test proteins that contain a combination of both α-helices and β-sheets. The KCM algorithm can be installed locally as a standalone Python program using the Anaconda package manager. Detailed installation instructions are available in the repository's README file62. For improved performance, we recommend installing a local copy of the ESMFold model. To do this, first install ESMFold using the environment.yml file provided in its official GitHub repository63, which creates the esmfold environment. Once this is done, activate this environment using the conda activate esmfold command and follow the instructions described in listing 1. The Anaconda commands to install KCM's required packages. All peptides for the experiments were obtained from Pepmic and synthesized using solid-phase peptide synthesis with the fluorenylmethyloxycarbonyl protecting group strategy. Peptides were purified using reverse-phase liquid chromatography (purity >95%). In this study, we used the following pathogenic bacterial strains: A. baumannii ATCC 19606, E. coli AIC221 (E. coli MG1655 phnE_2::FRT (control strain for AIC222)) and E. coli AIC222 (E. coli MG1655 pmrA53 phnE_2::FRT (polymyxin resistant; colistin-resistant strain)), Klebsiella pneumoniae ATCC 13883, Pseudomonas aeruginosa PAO1, P. aeruginosa PA14 and Staphylococcus aureus ATCC 12600. Pseudomonas isolation (P. aeruginosa strains) agar plates were exclusively used in the case of Pseudomonas species. In all the experiments, bacteria were inoculated from one isolated colony and grown overnight (16 h) in liquid medium at 37 °C. On the following day, inocula were diluted 1:100 in fresh media and incubated at 37 °C to mid-logarithmic phase. Broth microdilution assays64 were conducted to establish the MIC for each peptide. Peptides were added to untreated polystyrene 96-well microtiter plates and serially diluted twofold in sterile water, ranging from 1 to 64 μmol l−1. A bacterial inoculum at a concentration of 106 CFU ml−1 in LB medium was then mixed in a 1:1 ratio with the peptide solution. The MIC was determined as the lowest peptide concentration that completely inhibited bacterial growth after 24 h of incubation at 37 °C. Each assay was performed in three independent replicates. The circular dichroism experiments were conducted using a J1500 circular dichroism spectropolarimeter (JASCO) in the Biological Chemistry Resource Center at the University of Pennsylvania. Experiments were performed at 25 °C; the spectra graphed are an average of three accumulations obtained with a quartz cuvette with an optical path length of 1.0 mm, ranging from 260 to 190 nm at a rate of 50 nm min−1 and a bandwidth of 0.5 nm. The concentration of all peptides tested was 50 μmol l−1, and the measurements were performed in water, in a mixture of TFE and water with a 3:2 ratio, in a mixture of methanol and water with a 1:1 ratio and in SDS in water at 10 mmol l−1, with respective baselines recorded before measurement. A Fourier transform filter was applied to minimize background effects. Secondary-structure fraction values were calculated using the single-spectrum analysis tool on the server BeStSel65. Ternary plots were created in https://www.ternaryplot.com/ and subsequently edited. Inocula of A. baumannii ATCC 19606 were grown to an optical density at 600 nm of 0.4 ml−1, centrifuged (9,391 g at 4 °C for 10 min), washed and resuspended in 5 mmol l−1 HEPES buffer (pH 7.4) containing 5 mmol l−1 glucose. The bacterial solution was added to a white 96-well plate (100 μl per well) together with 4 μl of NPN at 0.5 mmol l−1. Subsequently, peptides diluted in water were added to each well, and the fluorescence was measured at excitation wavelength (λex) = 350 nm and emission wavelength (λem) = 420 nm over time for 45 min. A. baumannii ATCC 19606 in the mid-logarithmic phase were washed and resuspended at 0.05 optical density ml−1 (optical value at 600 nm) in HEPES buffer (pH 7.2) containing 20 mmol l−1 glucose and 0.1 mol l−1 KCl. Membrane depolarization was then followed by reading changes in the fluorescence (λex = 622 nm, λem = 670 nm) over time for 60 min. HEK293T cells were obtained from the ATCC (CRL-3216). The cells were cultured in high-glucose DMEM supplemented with 1% penicillin and streptomycin (antibiotics) and 10% fetal bovine serum and grown at 37 °C in a humidified atmosphere containing 5% CO2. One day before the experiment, an aliquot of 100 μl of the cells at 50,000 cells ml−1 was seeded into each well of the cell-treated 96-well plates used in the experiment (that is, 5,000 cells per well)45. The MTT reagent was dissolved at 0.5 mg ml−1 in medium without phenol red and was used to replace cell culture supernatants containing the peptides (100 μl per well), and the samples were incubated for 4 h at 37 °C in a humidified atmosphere containing 5% CO2, yielding the insoluble formazan salt. The resulting salts were then resuspended in hydrochloric acid (0.04 mol l−1) in anhydrous isopropanol and quantified by spectrophotometric measurements of absorbance at 570 nm. All assays were carried out as three biological replicates. The backs of six-week-old female CD-1 mice under anaesthesia were shaved and injured with a superficial linear skin abrasion made with a needle. An aliquot of A. baumannii ATCC 19606 (2 × 106 CFU ml−1; 20 μl) previously grown in LB medium until 0.5 optical density ml−1 (optical value at 600 nm) and then washed twice with sterile PBS (pH 7.4, 9,391 g for 3 min) was added to the scratched area. Peptides diluted in sterile water at MIC value were administered to the wound area 1 h after the infection. Two and four days after infection animals were euthanized, and the scarified skin was excised, homogenized using a bead beater (25 Hz for 20 min), tenfold serially diluted, and plated on McConkey agar plates for CFU quantification. The experiments were performed using six mice per group. Mice were housed in groups of three and maintained under a 12-h light/dark cycle at 22 °C with humidity controlled at 50%. The skin abscess infection mouse model was revised and approved by University Laboratory Animal Resources from the University of Pennsylvania (protocol 806763). Six-week-old female CD-1 mice were rendered neutropenic by two doses of cyclophosphamide (150 mg kg−1 and 100 mg kg−1) applied intraperitoneally 3 and 1 d before the infection. On day 4 of the experiment, the mice were infected in their right thighs with a 100-μl intramuscular injection of the A. baumannii ATCC 19606 in PBS at concentration of 5 × 106 CFU ml−1. The bacterial cells were grown in LB broth, washed twice with PBS solution and diluted at the desired concentration. Two days after infection mice were euthanized, and the tissue from the right thigh was excised, homogenized using a bead beater (25 Hz for 20 min), tenfold serially diluted, and plated on McConkey agar plates for bacterial colony counting. The experiments were performed using six mice per group. Mice were housed in groups of three and maintained under a 12-h light/dark cycle at 22 °C with humidity controlled at 50%. The deep thigh infection mouse model was revised and approved by University Laboratory Animal Resources from the University of Pennsylvania (protocol 807055). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The test data for in silico experiments are available at GitHub (https://github.com/LabBiocomp/KCM; ref. All data pertaining to the experimental validation of generated peptides are available in the Supplementary Data or from the corresponding authors upon reasonable request. Source data are provided with this paper. The KCM algorithm is available at GitHub (https://github.com/LabBiocomp/KCM; ref. & de la Fuente-Nunez, C. Machine learning for antimicrobial peptide identification and design. Fleishman, S. J. et al. Computational design of proteins targeting the conserved stem region of influenza hemagglutinin. Koday, M. T. et al. A computationally designed hemagglutinin stem-binding protein provides in vivo protection from influenza independent of a host immune response. Cao, L. et al. De novo design of picomolar SARS-CoV-2 miniprotein inhibitors. Discovery of entry inhibitors for HIV-1 via a new de novo protein design framework. Kiss, G., Çelebi‐Ölçüm, N., Moretti, R., Baker, D. & Houk, K. N. Computational enzyme design. Quijano-Rubio, A. et al. De novo design of modular and tunable protein biosensors. Bick, M. J. et al. Computational design of environmental sensors for the potent opioid fentanyl. Walls, A. C. et al. Elicitation of potent neutralizing antibody responses by designed protein nanoparticle vaccines for SARS-CoV-2. Chen, Z. et al. Self-assembling 2D arrays with de novo protein building blocks. Chen, Z. et al. De novo design of protein logic gates. Novel antimicrobial peptides against Cutibacterium acnes designed by deep learning. Wang, X.-F. et al. ProT-Diff: a modularized and efficient strategy for de novo generation of antimicrobial peptide sequences by integrating protein language and diffusion models. Ponder, J. W. & Richards, F. M. Tertiary templates for proteins. Zhu, Y. Mixed-integer linear programming algorithm for a computational protein design problem. Traoré, S. et al. A new framework for computational protein design through cost function network optimization. Chazelle, B., Kingsford, C. & Singh, M. A semidefinite programming approach to side chain positioning with new rounding strategies. Robust deep learning–based protein sequence design using ProteinMPNN. & Kim, P. M. Fast and flexible protein design using deep graph neural networks. Hsu, C. et al. Learning inverse folding from millions of predicted structures. 39th International Conference on Machine Learning Vol. 162 (eds Chaudhuri, K. et al.) 8946–8970 (PMLR, 2022). & Zhang, G. SPDesign: protein sequence designer based on structural sequence profile using ultrafast shape recognition. Zhang, X., Yin, H., Ling, F., Zhan, J. & Zhou, Y. SPIN-CGNN: improved fixed backbone protein design with contact map-based graph construction and contact graph neural network. Gao, Z., Tan, C., Chacón, P. & Li, S. Z. PiFold: toward effective and efficient protein inverse folding. Watson, J. L. et al. De novo design of protein structure and function with RFdiffusion. Yi, K., Zhou, B., Shen, Y., Lió, P. & Wang, Y. Graph denoising diffusion for inverse protein folding. In Advances in Neural Information Processing Systems 36: Annual Conference on Neural Information Processing Systems 2023, NeurIPS 2023 (eds Oh, A. et al.) 10238–10257 (Curran Associates, 2023). Zalba, N., Ursua-Medrano, P. & Bustince, H. Protein design with StructureGPT: a deep learning model for protein structure-to-sequence translation. Min, X. et al. TPGen: a language model for stable protein design with a specific topology structure. In silico evolution of autoinhibitory domains for a PD-L1 antagonist using deep learning models. Hong, L. & Kortemme, T. An integrative approach to protein sequence design through multiobjective optimization. Haney, E. F., Mansour, S. C., Hilchie, A. L., de la Fuente-Núñez, C. & Hancock, R. E. W. W. High throughput screening methods for assessing antibiofilm and immunomodulatory activities of synthetic peptides. Principles that govern the folding of protein chains. Evolutionary algorithms: a critical review and its future prospects. In 2016 International Conference on Global Trends in Signal Processing, Information Computing and Communication (ICGTSPICC) 261–265 (IEEE, 2016). & Fidelis, K. Processing and analysis of CASP3 protein structure predictions. & Zaffalon, M. Time for a change: a tutorial for comparing multiple classifiers through Bayesian analysis. Highly accurate protein structure prediction with AlphaFold. Mirdita, M. et al. ColabFold: making protein folding accessible to all. & Gulsevin, A. Benchmarking AlphaFold2 on peptide structure prediction. Agarwal, V. & McShan, A. C. The power and pitfalls of AlphaFold2 for structure prediction beyond rigid globular proteins. Pinacho-Castellanos, S. A., García-Jacas, C. R., Gilson, M. K. & Brizuela, C. A. Alignment-free antimicrobial peptide predictors: improving performance by a thorough analysis of the largest available data set. Silveira, G. G. O. S. et al. Antibiofilm peptides: relevant preclinical animal infection models and translational potential. Torres, M. D. T. et al. Structure–function-guided exploration of the antimicrobial peptide polybia-CP identifies activity determinants and generates synthetic therapeutic candidates. Torres, M. D. T. et al. Mining human microbiomes reveals an untapped source of peptide antibiotics. & de la Fuente-Nunez, C. Peptides from non-immune proteins target infections through antimicrobial and immunomodulatory properties. Ling, L. L. et al. A new antibiotic kills pathogens without detectable resistance. Torres, M. D. T. et al. Mining for encrypted peptide antibiotics in the human proteome. & de la Fuente-Nunez, C. Deep-learning-enabled antibiotic discovery through molecular de-extinction. Scoring function for automated assessment of protein structure template quality. The Rosetta all-atom energy function for macromolecular modeling and design. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Njirjak, M. et al. Reshaping the discovery of self-assembling peptides with generative AI guided by hybrid deep learning. Sloss, A. N. & Gustafson, S. in Genetic Programming Theory and Practice XVII (eds Banzhaf, W. et al.) 307–344 (Springer, 2020). Dawson, N. L. et al. CATH: an expanded resource to predict protein function through structure and sequence. Meta AI ESM Metagenomic Atlas https://esmatlas.com/about#api (2025). Wiegand, I., Hilpert, K. & Hancock, R. E. W. Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances. Micsonai, A. et al. BeStSel: webserver for secondary structure and fold prediction for protein CD spectroscopy. C.d.l.F.-N. holds a Presidential Professorship at the University of Pennsylvania. Research reported in this publication was supported by NIH R35GM138201 and DTRA HDTRA1-21-1-0014. We thank de la Fuente Lab members for insightful discussions. Molecules were rendered using the PyMOL Molecular Graphics System v.3.1.1, Schrödinger. These authors contributed equally: Yan C. Leyva, Marcelo D. T. Torres. Computer Science Department, Center for Scientific Research and Higher Education at Ensenada (CICESE), Ensenada, Mexico Machine Biology Group, Departments of Psychiatry and Microbiology, Institute for Biomedical Informatics, Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Marcelo D. T. Torres & Cesar de la Fuente-Nunez Departments of Bioengineering and Chemical and Biomolecular Engineering, School of Engineering and Applied Science, University of Pennsylvania, Philadelphia, PA, USA Marcelo D. T. Torres & Cesar de la Fuente-Nunez Marcelo D. T. Torres & Cesar de la Fuente-Nunez Penn Institute for Computational Science, University of Pennsylvania, Philadelphia, PA, USA Marcelo D. T. Torres & Cesar de la Fuente-Nunez Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed experiments and interpreted the data. All authors wrote and revised the manuscript. Correspondence to Cesar de la Fuente-Nunez or Carlos A. Brizuela. C.d.l.F.-N. is a co-founder of, and scientific advisor to, Peptaris, Inc., provides consulting services to Invaio Sciences, and is a member of the Scientific Advisory Boards of Nowture S.L., Peptidus, European Biotech Venture Builder, the Peptide Drug Hunting Consortium (PDHC), ePhective Therapeutics, Inc. and Phare Bio. is a co-founder and scientific advisor to Peptaris, Inc. All other authors declare no competing interests. Nature Machine Intelligence thanks Divya Nayar and the other, anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Statistical data for the ternary plots of the comparison between our method and ESM-IF1, ProteinSolver and ProteinMPNN. Statistical data for the runtime of ESMFold as a function of sequence length. Statistical data for the circular dichroism spectra and heatmap with the percentage of secondary structure obtained on the BeStSel server. Statistical data for the outer-membrane permeabilization and cytoplasmic-membrane depolarization fluorescence measurements. Statistical data for the cytotoxicity experiments: cell viability versus peptide concentration, anti-infective activity in deep thigh infection and mouse weight changes over time in both skin scarification and deep thigh infection mouse models. Zip containing the pdb files for each target and the corresponding best design. Source data for the RMSD_S and GDT_TS for the 100 best designs for each target peptide, GDT_TS as a function of RMSD_S of the highest fitness design for each protein and superposition of target and designed peptides. Source data for the minimal inhibitory concentration heatmap and circular dichroism ternary plots. Source data for the outer membrane permeabilization and cytoplasmic membrane depolarization relative fluorescence measurements and cytotoxic concentration needed to kill 50% of mammalian cells (CC50) heatmap. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tailored structured peptide design with a key-cutting machine approach. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01788-6'>Enhancer activation from transposable elements in extrachromosomal DNA</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 09:31:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Extrachromosomal DNA (ecDNA) drives oncogene amplification and intratumoural heterogeneity in aggressive cancers. While transposable element reactivation is common in cancer, its role on ecDNA remains unexplored. Here we map the 3D architecture of MYC-amplified ecDNA in colorectal cancer cells and identify 68 ecDNA-interacting elements—genomic loci enriched for transposable elements that are frequently integrated onto ecDNA. We focus on an L1M4a1#LINE/L1 fragment co-amplified with MYC, which functions only in the ecDNA-amplified context. Using CRISPR-CATCH, CRISPR interference and reporter assays, we confirm its presence on ecDNA, enhancer activity and essentiality for cancer cell fitness. These findings reveal that repetitive elements can be reactivated and co-opted as functional rather than inactive sequences on ecDNA, potentially driving oncogene expression and tumour evolution. Our study uncovers a mechanism by which ecDNA harnesses repetitive elements to shape cancer phenotypes, with implications for diagnosis and therapy. Extrachromosomal DNA (ecDNA) is a prevalent form of oncogene amplification present in approximately 15% of cancers at diagnosis1,2,3,4,5. ecDNAs are megabase-scale, circular DNA elements lacking centromeric and telomeric sequences and found as distinct foci apart from chromosomal DNA6. Recent work has underscored the importance of ecDNA in tumour initiation and various aspects of tumour progression, such as accelerating intratumoural heterogeneity, genomic dysregulation and therapeutic resistance7,8,9,10,11. The biogenesis of ecDNA is complex and tied to mechanisms that induce genomic instability, such as chromothripsis and breakage–fusion–bridge cycles, which are prevalent in tumour cells6,12,13,14,15,16,17. A key aspect of ecDNA function is their ability to hijack cis-regulatory elements that increase oncogene expression beyond the constraints imposed by endogenous chromosomal architecture18,19,20,21,22,23. Consequently, their nuclear organization is tightly tied to their ability to amplify gene expression18,20. Likewise, repetitive genomic elements provide a vast network of cryptic promoters or enhancers capable of rewiring gene regulatory networks for proto-oncogene expression—including long-range gene regulation24,25,26. By investigating the three-dimensional (3D) organization of ecDNA, we identified an enrichment of repetitive elements associated with ecDNA structural variation, which we classify as ecDNA-interacting elements (EIEs). We found that insertion of a particular EIE containing a fragment of an ancient L1M4a1 LINE within ecDNA leads to expression of said element that is critical for cancer cell fitness. Our data reveal a relationship between the presence of specific repetitive elements and aberrant expression of oncogenes on ecDNA. To interrogate the conformational state of ecDNA, we performed Hi-C on COLO320DM colorectal cancer cells (Fig. Previous investigation of COLO320DM utilizing DNA fluorescent in situ hybridization (FISH) and whole-genome sequencing identified a highly rearranged (up to 4.3 MB) ecDNA amplification containing several genes, including the oncogene MYC and the long non-coding RNA PVT118,20. As a large fraction of the ecDNA in COLO320DM is derived from chromosome 8, with smaller contributions from chromosomes 6, 16 and 13, we elected to focus on the chromosome-8-amplified locus containing MYC and PVT120. a, Schematic of the Hi-C method performed in the ecDNA-containing COLO320DM cell line. Sixty-eight individual EIEs were manually annotated across all chromosomes based on the interaction across the entirety of the MYC-amplified region of chromosome 8. The visualization represents the ecDNA from chromosome 8, with three examples of EIEs localized on other chromosomes. c, An example of a specific interaction, EIE 14 on chromosome 3, is enlarged, and associated genes are shown for both loci. The arrow and purple hexagon indicate EIE. d, Overlap fraction between EIE sequences and annotated LINE, SINE and LTR elements as reported in RepBase. EIEs are clustered according to similarity in overlap fraction across these three classes of repetitive elements. e, Pipeline for using Oxford Nanopore ultralong-read sequencing to identify the overlap of ecDNA genomic intervals and EIE-containing reads. f, The number of reads that contain a particular EIE and overlap with an ecDNA interval in the COLO320DM cell line. Counts are reported as log10(1 + x). Average genome coverage (approximately 12.1) is represented as a dashed red line. g, Reconstruction of the ecDNA breakpoint graph for COLO320DM from Oxford Nanopore ultralong-read data using the CoRAL algorithm. The EIE 14 region is highlighted in red, and the breakpoint indicating its translocation to the amplified chromosome 8 locus is annotated. Analysis of the Hi-C maps identified 68 interactions between the chromosome-8-amplified ecDNA locus and other chromosomes that displayed a striking pattern (Fig. By binning the data at 1 kb resolution, we found that linear elements in the genome contacted the entirety of the megabase-scale ecDNA amplification in a distinctive stripe (Fig. These contacts were spread across all chromosomes in the genome (Supplementary Table 1). This atypical interaction pattern suggested a complex structural relationship between the chromosome-8-amplified ecDNA and the endogenous chromosome regions (Fig. Further inspection revealed these genomic interactions were enriched for transposable elements (TEs) annotated as LINEs (long interspersed nuclear elements), SINEs (short interspersed nuclear elements) and LTRs (long terminal repeats (Fig. As these retrotransposons can acquire the ability to regulate transcription when active, we reasoned that the spatial relationship with oncogenes such as MYC may be important for enhanced expression in COLO320DM cells27,28. We hereafter refer to these 1-kb interactions, often containing retrotransposons, as EIEs. While Hi-C is widely used to map genome-wide chromatin interactions, it can also be repurposed to identify structural variants, including rearrangements that are hallmarks of cancer genomes29,30. We considered that the atypical striping pattern observed in our Hi-C data was probably the result of structural variation either within the COLO320 genome or due to the insertion of repetitive elements into ecDNA. To discern between these two possibilities, we performed long-read nanopore sequencing (Methods). We chose long-read sequencing to also capture potential heterogeneity in insertion sites in the case of single or multiple integrations (Fig. We generated median read lengths of 67,000 bp, with the longest read spanning 684,457 bases. Across the 68 EIEs identified, we determined that each participated in a broad spectrum of structural variation—some involved with hundreds or thousands of different rearrangement events (Extended Data Fig. After confirming that the identified EIEs were associated with structural rearrangements, we next investigated the overlap between ecDNA and EIE rearrangements. We first reconstructed ecDNA utilizing the CoRAL algorithm31, a pipeline that leverages long-read data to accurately infer a set of ecDNAs from the breakpoints (that is, structural variation) associated with amplified regions of the genome (Methods). We found that reads containing EIEs often overlapped ecDNA intervals with greater coverage than expected based on the average genome coverage of our dataset (approximately 12.1), suggesting that these EIEs are present in at least a subset of ecDNA amplifications (Fig. We further investigated CoRAL's reconstruction of COLO320DM's complex and heterogeneous MYC-containing amplicon and identified a high-confidence breakpoint connecting a chromosome-3-amplified EIE (EIE 14) to an intergenic region between CASC8 and MYC on the chromosome 8 amplification (Fig. We selected this EIE (EIE 14) for further characterization of EIE biology owing to its proximity to MYC on the ecDNA and because it contains a segment with homology to L1M4a1, an ancient element distantly related to LINE-1. The percentage of nucleotide conservation of this segment to the L1M4a1 consensus sequence is consistent with L1M4a1's Kimura divergence value of 34%. We reasoned that this degree of sequence divergence would allow us to specifically target and interrogate its function without unintentionally targeting other repetitive elements in the genome. We also identified a fragment of LINE-1 PA2 and an ORF2-like protein on EIE 1432,33 (Extended Data Fig. Although the mechanism generating the adjacency of the fragments remains uncertain, the L1M4a1-like segment harbours a polyA-signal-like motif (AAAAAG), supporting a model in which an L1PA2 transcript reads through its own 3′ end and terminates at this neighbouring signal, producing a 3′-transduced RNA that could be mobilized in trans by the LINE-1 enzyme32,33 (Extended Data Fig. To confirm the computational reconstruction of the ecDNA and the heterogeneity of different ecDNA molecules, we turned to CRISPR-CATCH (Cas9-assisted targeting of chromosome segments)—a method for isolating and sequencing ecDNA—to elucidate the size and variations of ecDNAs containing EIE 1422 (Fig. Targeting EIE 14 with two independent gRNAs, we successfully isolated ecDNA fragments from the COLO320DM cell line for sequencing (Fig. Sequence analysis of these bands confirmed the presence of EIE 14, originally annotated on chromosome 3, to be inserted onto chromosome 8 between the CASC8 and CASC11 genes approximately 200 kilobases away from MYC, in agreement with the long-read nanopore sequencing (Figs. Multiple bands of different sizes on the pulsed-field gel electrophoresis (PFGE) gel indicated the presence of varying sizes of ecDNAs, all sharing the EIE 14 insertion within the chromosome 8 amplicon (Fig. Beyond EIE 14, the CRISPR-CATCH approach allowed us to capture and sequence a subset of EIEs initially identified through Hi-C analysis (Fig. The identification of the additional EIEs observed in the Hi-C data suggest that the ‘striping' between the ecDNA and endogenous chromosomes is an artefact of these sequences' presence on ecDNAs, rather than true trans contacts, at least for this identified subset. Although the recent T2T genome build34 annotates EIE 14 to chromosome 3 (Extended Data Fig. 2c), we found evidence that the structural variant described here between EIE 14 and the MYC-containing amplicon region is identified as a translocation event between Chr8:128,533,830 and Chr3:111,274,086 in approximately 46% (minor allele frequency of 0.467646) of individuals without disease35 (Supplementary Table 4, row 7). This suggests that this structural variant was preexisting before cancer formation in the COLO320-originating patient and was subsequently amplified as a passenger on ecDNA. a, A schematic diagram illustrating the CRISPR-CATCH experiment designed to isolate and characterize ecDNA components. DNA is embedded in agarose, followed by PFGE, allowing band extraction and subsequent next-generation sequencing (NGS) of ecDNA fragments. Negative control (NC) is a guide RNA not targeting ecDNA. b, The PFGE gel image displays the separation of DNA fragments, including lanes for the left ladder, ladder, empty lane, negative control, sgRNA #1, sgRNA #2, and band numbers corresponding to those analysed by NGS in in c and d. Targeting EIE 14 with guide RNAs leads to cleavage of the ecDNA chromosome 8 sequences, resulting in multiple discrete bands and confirming the insertion of EIE 14 onto ecDNA. sgRNA #1 ATATAGGACAGTATCAAGTA; sgRNA #2 TATATTATTAGTCTGCTGAA; full EIE 14 sequences from long-read sequencing are presented in Supplementary Table 6. c, Whole-genome sequencing results confirm the presence of EIE 14, originally annotated on chromosome 3, within the ecDNA, between the CASC8 and CASC11 genes, approximately 200 kilobases upstream from MYC. Each band is an ecDNA molecule of a different size that contains the EIE 14 insertion. d, Additional EIEs identified in the initial Hi-C screen, captured and sequenced in the CRISPR-CATCH gel bands from b. Each EIE is represented by a vertical shaded box with genomic coordinates, indicating insertion events within the ecDNA. e, ORCA visualization of the COLO320DM cell nucleus. The leftmost panel is an overlay of all images registered to nanometre precision (Methods). We then utilized Optical Reconstruction of Chromatin Architecture (ORCA) to quantify the spatial relationship of EIE 14 with MYC36,37 (Fig. Barcoded probes were designed targeting the unique portion of EIE 14 (1 kb), MYC exon 2 (3.1 kb), PVT1 exon 1 (2.5 kb) and the endogenous chromosome 3 region flanking EIE 14 (3 kb) (Supplementary Table 7) to determine the spatial organization of EIE 14 relative to the ecDNA. These specific exons were chosen to account for the fact that amplicon reconstruction of ecDNA in the COLO320DM cell line demonstrated an occasional rearrangement of MYC exon 2 replacement by PVT1 exon 1 (ref. Because EIE 14 is classified as a repetitive element, we confirmed probe specificity by staining the EIE 14 locus in K562 cells that do not contain ecDNA. Indeed, we detect only one to three labelled regions in the non-amplified context (Extended Data Fig. By contrast, when labelling COLO320DM cells, EIE 14 colocalized with the ecDNA and amplified to a similar copy number per cell (Fig. The extensive structural variation detected in the long-read sequencing and the amplification of EIE 14 visualized by ORCA (Extended Data Fig. 3b) suggest a model in which the element resides in the sequence amplified on ecDNA and participates in cis and/or trans contacts with other ecDNA molecules. It has been proposed that amplified loci within ecDNA are able to regulate oncogene expression through cis interactions on the same ecDNA molecule as well as trans interactions between ecDNAs via a clustering mechanism20. As such, it is important to understand not only the structural variations of ecDNA, but also their spatial organization within the nucleus to gain a comprehensive understanding of their potential regulatory functions. We quantified the spatial distributions of MYC exon 2, PVT1 exon 1 and EIE 14; the imaged loci were fitted in three dimensions with a Gaussian fitting algorithm to extract x,y,z coordinates (Fig. The copy number of identified loci varied from zero detected points to 150 per cell. Similar distributions of points per cell, as well as strong correlation (r > 0.7) between the number of points per loci per cell (Extended Data Fig. 3c), suggest that this EIE is not inserted into multiple sites on a single ecDNA. a, x,y,z projections of MYC exon (purple), PVT1 (blue) and EIE 14 (pink). b, Endogenous coordinates of all three measured genomic regions. c, Single-cell projection of the 3D fitted points from a. d, Pairwise distances between MYC (purple), PVT1 (blue) and EIE 14 (pink) of a single cell. e, Histogram showing the distribution of observed shortest pairwise distances between EIE 14 signals, compared with the expected shortest pairwise distances from randomly simulated points within a sphere (two-tailed Wilcoxon ranksum P < 1 × 10−10) of n = 1,329 analysed cells across two biological replicates. f, As in e but for MYC-to-MYC shortest pairwise distances (two-tailed Wilcoxon ranksum P < 1× 10−10). g, Schematic of Ripley's K function to describe clustering behaviours over different nucleus volumes. Top: the nucleus divided into different shell intervals and how the K value is plotted for increasing radius (r). Bottom: an example of what clustered K(r) > 1 versus random K(r) ≈ 1 points could look like. K values greater than one indicate clustering behaviour relative to a random distribution over that given distance interval (r), K values of ~1 denote random distribution, while K values less than 1 indicate dispersion behaviour. h, The average K(r) value across distance intervals of 0.01–0.5 μm in 0.02-μm step sizes to describe the clustering relationship of PVT1 and EIE 14 relative to MYC across different distance intervals (μm). Once the centroids of each point per cell were identified (Fig. 3c), we calculated the all-to-all pairwise distance relationship (Fig. The off-diagonal pattern of distances between EIE 14, MYC and PVT1 suggested a tendency for these loci to cluster at genomic distances <1,000 nm. We further quantified the spatial relationships across all 1,329 imaged cells by calculating the shortest pairwise distances between the three loci. To determine if these ecDNA molecules were spatially clustering in cells, we leveraged our observation that each ecDNA molecule carries a single copy of MYC and EIE 14. Thus, distances between MYC and other MYC loci should be closer than random if the ecDNA were spatially clustered. Random distances were simulated in a sphere with the identical number of points per a given cell. The distribution of shortest pairwise distances between MYC and MYC and between EIE 14 and EIE 14 were left-shifted compared with the randomly simulated points, suggesting a non-random organization (Fig. The median observed versus expected distances between each EIE 14 loci were 748 nm and 927 nm, respectively, and the median observed versus expected distances between each MYC loci were 707 nm and 814 nm, respectively. Previous work has proposed that enhancers can exert transcriptional regulation on promoters at a distance of up to 300 nm via accumulation of activating factors38,39,40,41. To determine whether EIE 14 and MYC are within this regulatory distance range on ecDNA molecules, we calculated the pairwise distances between loci. Although the median distances between MYC and EIE 14 (797 nm) and PVT1 (585 nm) were greater than 300 nm, 12% and 20% of these loci, respectively, were within the regulatory range of MYC (Extended Data Fig. To investigate the spatial relationship between EIE 14 and MYC while controlling for locus density, we calculated the degree of spatial clustering across distance intervals using Ripley's K spatial point pattern analysis (Methods; Fig. MYC exhibited the strongest clustering with EIE 14 at distances less than 200 nm (K value >1), and this behaviour approached a random distribution at greater distances (K value ~1; Fig. On average, the distances between MYC and EIE 14 were greater than those between MYC and PVT1. However, at distances below 300 nm, EIE 14 and PVT1 displayed similar clustering behaviour with MYC (Fig. This clustering suggests that EIE 14 is acting as a proximity-dependent regulator of MYC reminiscent of enhancer–promoter interactions42. Altogether, the spatial clustering behaviour of this ecDNA species measured here and previously20, the propensity for MYC to engage in ‘enhancer hijacking'43 and the ability of reactivated repetitive elements to engage in long-range gene activation27 suggest that any genomically linear separation of MYC and EIE 14 is overcome in both cis (interaction with MYC on the same ecDNA molecule) and trans (ecDNA–ecDNA interactions). To test whether the identified TEs are important for the cancer cell proliferation, we performed a CRISPR interference (CRISPRi) growth screen targeting a subset of EIEs in COLO320DM cells engineered to stably express dCas9-KRAB44 (Fig. We were able to target 36 out of the 68 EIEs with single guide RNAs (sgRNAs) that met the following criteria: (1) must meet stringent specificity criteria to reduce potential off targets intrinsic to repetitive sequences (Methods) and (2) have at least two sgRNAs per EIE. We also included 125 non-targeting controls (NTC) that were introduced into cells with the EIE sgRNAs via lentiviral transduction (Supplementary Table 10). After transduction, we monitored cell proliferation at multiple timepoints: 4 days (baseline), 3 days after baseline, 14 days and 1 month (30 days), followed by deep sequencing to quantify sgRNA frequencies (Fig. We obtained highly reproducible guide counts across replicates and timepoints (Extended Data Fig. a, Schematic of the CRISPRi screening strategy used to evaluate the regulatory potential of the 68 EIEs by designing 4–6 gRNAs per element for a total of 257 genomic regions tested and 125 non-targeting control sgRNAs. The screen involved the transduction of cells with a lentivirus expressing dCas9-KRAB and the sgRNAs such that each cell received one sgRNA, followed by calculation of cell growth phenotype over a series of timepoints (baseline (4 days), baseline + 3 days, baseline + 14 days and baseline + 1 month). The screen was further filtered on guide specificity (Methods), and 36/68 targeted EIEs met the qualifying threshold. b, The growth phenotype of COLO320DM cells 2 weeks post-transduction, relative to NTC. Each point represents the average guide effect (Z score) for sgRNAs targeting the 36 qualifying EIEs, ranked by their impact on cell growth. EIE 14 is indicated by dashed rectangle with negative Z score <−1 (significant negative impact on cell viability). Positive hits are labelled in pink with their corresponding EIE. Each EIE is indicated by a vertical bar. The browser displays the annotations for genes and repetitive elements such as Alu, LINE and LTR elements (RepeatMasker); the ATAC-seq dataset20 is normalized for library size (Methods). d, Zoom-in of EIE 14's histone marks: enrichment of H3K27 acetylation18, BRD4 binding20 and ATAC-seq peaks. ChIP-seq data were normalized to input to control for copy number. ATAC-seq data were normalized to library size (Methods). e, H3K9me3 histone modification of EIE 14 across ENCODE cell lines50,51. Our data showed that the growth phenotype curve for 3 out of 36 of our targeted EIEs at various timepoints indicated a Z score of less than −1, which suggested a significant negative impact on cell viability, with an acute growth defect after only 3 days (Fig. These elements were categorized as evolutionarily older based on their retrotransposition activity in the human genome and spanned classes (LINEs, SINEs and LTRs) (Supplementary Table 11). The enrichment of old TEs may be confounded by the relative ease of targeting sequences with increased sequence divergence. They are generally found in gene-poor regions, making it unlikely that silencing would lead to secondary effects from heterochromatin spreading. Collectively, these results suggest that a subset of our targeted EIEs, including EIE 14, can contribute to cancer cell growth and fitness. We speculate that this is related to EIE interaction with MYC, as knockdown of this oncogene has been shown to have similar effects on COLO320DM growth and survival45,46. In addition, 3 out of 36 of the measured EIEs also had a Z score greater than 1, indicating a significant increase of cell growth or fitness. The identity of these elements also spanned multiple element classes, with two (EIE 68 and EIE 45) located within uncharacterized non-coding RNAs, and one (EIE 57) within the first exon of the ANKRD30B protein-coding gene, which has been implicated in cell proliferation47. Further investigation of these hits is warranted in future studies to explain their positive effects on cell growth, especially those within the uncharacterized non-coding RNA regions. The strongest growth defect was observed for perturbation of EIE 14 (Fig. 4b), which when combined with our finding of its co-localization with ecDNA-amplified MYC (Fig. 3h), suggests a potential enhancer-like regulatory role for this EIE. To examine the epigenetic landscape of this element we leveraged copy-number-normalized chromatin immunoprecipitation sequencing (ChIP-seq) measuring H3 lysine 27 acetylation (H3K27ac), BRD4 occupancy and assay for transposase-accessible chromatin using sequencing (ATAC-seq) accessibility data. These epigenetic features are all commonly associated with enhancer activity18,48,49. Notably, many EIEs, including EIE 14, were accessible in COLO320DM cells (Fig. 4c,d and Extended Data Fig. The measured accessibility of EIE 14 contrasts the normally silenced H3 lysine 9 trimethylation (H3k9me3) state across annotated human cell lines50,51 (Fig. Cross-referencing our identified EIEs with accessibility data from other ecDNA-containing cell lines demonstrated that accessibility of EIEs is a more generalizable phenomenon beyond COLO320DM cells (Extended Data Fig. Altogether, the accessibility and proximal clustering of EIE 14 points towards active regulatory potential of this element in COLO320DM cells, while identification of accessible EIEs across cell lines suggests a broader functional relevance of EIE regulatory potential on ecDNA48,49 (Extended Data Fig. To determine whether EIE 14 activity is a consequence of ecDNA formation, we performed RNA-FISH on the sequence-specific 1-kb segment of EIE 14 in COLO320DM and isogenic COLO320HSR cells. The homogeneously staining region (HSR) cell line contains a similar copy number amplification of the MYC-amplified portion of chromosome 8, but the majority of these copies have integrated into chromosomes18 (Fig. We reasoned that, if the unique extrachromosomal context of ecDNA facilitates activation of EIE 14, we should not see evidence of its activity in the COLO320HSR genome-integrated context. Indeed, we observed distinct transcription events in the COLO320DM line (median n = 8 transcripts per cell) but not in the HSR line(median n = 0 transcripts per cell; Fig. a, Schematic outlining the COLO320DM cell line as having high copy number and high ecDNA levels, versus the HSR cell line, which has high copy number but low ecDNA. b, RNA-FISH labelling for EIE 14 and MYC exon 2 transcription in COLO320 DM and HSR. Median transcripts for EIE 14 are 4 and 0 for the DM and HSR cells (two-tailed Wilcoxon ranksum P = 8.22 × 10−94), respectively. DM cells have a median of 14 MYC transcripts, and HSR cells have a median of 8 transcripts per cell (two-tailed Wilcoxon ranksum P = 2.18 × 10−66). c, Luciferase enhancer assay schematics and fold change in luciferase signal driven by either MYC or TK promoter normalized to promoter-only construct. P values obtained from two-tailed unpaired t-test. Error bars are standard deviations from the mean. d, Schematic outlining EIE 14 as a translocation event in healthy patients where EIE 14 is normally inactive across annotated cell lines (see a). EIE 14 gains regulatory potential when it is amplified within ecDNA as a consequence of translocation near MYC. EIE 14 can then act as a regulator of MYC in both cis and trans contacts within and between ecDNAs. Finally, to directly test the ability for the EIE 14 sequence to act as an enhancer of MYC expression, we performed a luciferase reporter assay measuring its ability to activate transcription of TK and MYC promoters20,52 (Fig. EIE 14 significantly increased MYC promoter-mediated reporter gene expression relative to the promoter-only control, signifying bona fide enhancer activity (Fig. Separating EIE 14 into L1M4a1 and L1PA2 fragments further demonstrated that both sequences can individually act as enhancers, with an additive effect when combined (Extended Data Fig. In sum, the enhancer-associated features and regulatory activity of the luciferase assay suggested that EIE 14, and possibly other EIEs, have been co-opted as regulatory sequences when found on ecDNA, influencing the expression of ecDNA-borne oncogenes (Fig. This study uncovers a mechanism by which TEs, typically silenced by heterochromatin, may acquire regulatory potential when amplified on ecDNA53,54,55. Somatically active retrotransposition events56, as induced by LINEs and SINEs, are abundant in the human genome and represent a major source of genetic variation57. Across cancer types, retrotransposon insertions contribute significantly to structural variation, genomic rearrangements, copy number alterations and mutations—including in colorectal cancer58,59,60,61,62,63,64,65. The activity of these elements in cancer can induce genomic instability and drive the acquisition of malignant traits. For instance, when reactivated LINE-1 elements are inserted into the APC tumour suppressor gene in colorectal cancer, they disrupt gene function and confer a selective advantage66. In other contexts, TEs act as bona fide transcriptional enhancers, amplifying oncogenic gene expression and promoting tumorigenesis26. Here, we describe the enhancer-like activity of a specific identified element, EIE 14, which becomes active through its association with ecDNA (Fig. ecDNAs, which are randomly segregated during cell division, are subject to strong selective pressure10. The recurrent co-amplification of TEs on ecDNA-containing cell lines suggests they may contribute to ecDNA fitness and oncogenic function. We show that retrotransposons such as L1M4a1/EIE 14 can escape the inactive chromatin environment of their native genomic loci when inserted within the transcriptionally permissive landscape of ecDNA18. In fact, we demonstrate that EIE 14 is transcriptionally active only in the context of ecDNA and not in the endogenous chromosomal context of the copy-number-matched, isogenic COLO320HSR cells. The context-specific transcription suggests a purely epigenetic regulation imbued by the local environment of ecDNA. This environment enables EIE 14 to potentially influence nearby oncogenes such as MYC. Given that LINEs have been shown to exhibit enhancer-like behaviour when reactivated27,28,67, the clustering of ecDNA molecules observed through ORCA may further enhance spatial feedback68 of both cis- and trans-regulatory interactions of EIE 14 with oncogenic targets. Although EIE 14 is incapable of autonomous transposition and lacks a complete L1M4a1 sequence, its activity following integration into ecDNA suggests that degenerate ancient sequences can become functionally active under appropriate conditions. Previous work has shown that single-nucleotide polymorphisms associated with familial cancer risk often affect the biochemical activity of noncoding enhancer elements linked to oncogenes activated in cancer69,70. Our results extend this model by proposing that inherited variation in ancient TE insertions, such as EIE 14 near MYC, can create latent enhancers that become activated when the oncogene locus is excised into ecDNA. Perturbation of EIE 14 through CRISPRi resulted in impaired cell growth in COLO320DM cells, indicating that its reactivation contributes to the colorectal cancer phenotype. Quantifying the precise downregulation of MYC is constrained by ecDNA heterogeneity, a narrow temporal window in MYC-addicted cells, rapid growth arrest and subsequent loss of successfully targeted cells. While this functional evidence supports a potential oncogenic role, further studies focusing on in vivo analyses are necessary to determine whether TEs on ecDNA are sufficient to confer a survival advantage or correlate with poor patient prognosis. Notably, recurrent LINE-1 amplification on ecDNA has been observed in primary oesophageal cancer, providing in vivo support for the clinical relevance of this phenomenon71. Finally, the amplification of retrotransposable elements onto ecDNA introduces a mechanism that increases ecDNA structural variation by leveraging the approximately 40% of the genome composed of typically silenced repetitive elements. Retrotranspositions are, in fact, the second-most frequent type of structural variant in colorectal adenocarcinomas72. Just as transposons have played a major role in bacterial plasmid evolution through cycles of insertion and recombination73, our findings suggest a parallel evolutionary trajectory in human oncogenic ecDNAs. The transcriptionally permissive state of ecDNA enables these elements to potentiate oncogene activation and selection—making them both prognostic biomarkers and potential therapeutic targets. Cell lines were obtained from ATCC. COLO320DM (CCL-220) and COLO320-HSR (CCL-220.1) cells were maintained in RPMI; Life Technologies, cat. 11875-119 supplemented with 10% foetal bovine serum (Hyclone, cat. All cell lines were routinely tested for mycoplasma contamination. The presence of ecDNA in cell lines was confirmed via metaphase spreads. Ten million cells were fixed in 1% formaldehyde in aliquots of one million cells each for 10 min at room temperature and combined after fixation. We performed the Hi-C assay following a standard protocol to investigate chromatin interactions within colorectal cancer cells74. HiC libraries were sequenced on an Illumina HiSeq 4000 with paired-end 75-bp read lengths. Paired-end HiC reads were aligned to hg19 genome with the HiC-Pro pipeline75. The pipeline was run with default settings, configured to assign reads to DpnII restriction fragments and to filter for valid pairs. The data were then binned to generate raw contact maps that then underwent iterative correction and eigenvector decomposition normalization to remove biases. The HiCCUPS function in Juicer76 was then used to call high-confidence loops. Visualization was done using Juicebox (https://aidenlab.org/juicebox/). To assess the overlap of classes of repetitive elements with our identified EIEs, we obtained the ‘RepeatMasker' and ‘Interrupted Repeats' tracks from UCSC Genome Browser for hg19. For each EIE, we computed the fraction of the sequence that overlapped with the merged BED file containing the RepeatMasker and Interreputed Repeats annotations. We report the overlap separately for LINE, SINE and LTR repetitive element classes. Importantly, each EIE is exactly 1 kb long, so no length normalization is performed. To compute an expected proportion, we computed the fraction of hg19 covered by each repetitive element class. The results are reported in Fig. High-molecular-weight (HMW) genomic DNA was extracted from approximately 6 million COLO320DM cells using the Monarch HMW DNA Extraction Kit for Tissue (NEB #T3060L) following the Oxford Nanopore Ultra-Long DNA Sequencing Kit V14 protocol. After extracting HMW genomic DNA, we constructed Nanopore libraries using the Oxford Nanopore Ultra-Long DNA Sequencing Kit V14 (SQK-ULK114) kit according to the manufacturer's instructions. We sequenced libraries on an Oxford Nanopore PromethION using a 10.4.1. Flow Cell (FLO-PRO114M) according to the manufacturer's instructions. Basecalls from raw POD5 files were computed using Dorado (v.0.2.4). We first identified Nanopore reads containing a single element by aligning reads with minimap277 and filtered out reads that were not mapped by the algorithm (denoted by an asterisk in the RNAME column of the BAM entry). Then, taking these reads, we performed genomic alignment once again using minimap2 against hg19. From these new alignments of only the reads found to contain the element under consideration, we performed two analyses for each element. First, we detected structural variant detection using Sniffles278. Second, we identified overlap of reads with ecDNA-containing intervals that were reconstructed with long reads (see ‘Reconstruction of ecDNA amplicons with long-read data” section). 1f), we counted the number of reads covering regions contained with cycles reconstructed with CoRAL algorithm31. While this analysis does not explicitly distinguish reads originating from chromosomal versus extrachromosomal regions, we reasoned that elements carried on ecDNA would be amplified and therefore exhibit higher coverage; conversely, regions primarily chromosomal would show read counts similar to the overall genome coverage. We reconstructed ecDNA amplicons from ultralong Oxford Nanopore reads using the CoRAL algorithm31. In brief, this algorithm determines focally amplified regions of the genome using CNVkit79 and then finds reads that support this focally amplified region. In doing so, CoRAL identifies genomic breakpoints between the focally amplified seed region and disparate parts of the genome to create a ‘breakpoint graph'. From this breakpoint graph, putative ecDNA cycles are identified. We report the breakpoint graph in Fig. 1g, which includes a breakpoint between EIE 14 (annotated on chr3) and an intergenic region between CASC8 and MYC on chr8. In addition to detecting EIE 14 on the MYC-amplifying ecDNA in COLO320DM, we additionally quantified the number of reads that span a given EIE and any part of the COLO320DM genome amplified as ecDNA. We report the number reads that support an EIE as amplified on ecDNA in Fig. 2b, we visualized reads connecting EIE 14 on chr3 with the chr8 ecDNA-amplified region using Ribbon (v 2.0.0)80. Previously, ATAC-seq data were mapped to hg19. While ChIP-seq data were normalized to input, as input is not sequenced with ATAC-seq, these data were further normalized by library size. Specifically, ATAC-seq data were converted to a bedGraph format reporting the number of reads supporting each base position; these read densities were then normalized to parts per 10 million by dividing each position's count by a normalization factor based on the total library size. These library-size-normalized data were used for downstream plotting To classify TEs as old or young, we conducted a classification of EIE sequences listed in Supplementary Table 2. Elements were categorized based on their known evolutionary activity in humans. Young elements were defined as those from recently active subfamilies, including L1HS, L1PA2, SVA and AluY, which are known to have current or recent retrotransposition activity in the human genome. Classifications can be found in Supplementary Table 11. 46911) plasmid was modified to dCas9-BFP-KRAB-2A-Blast as previously described81. Lentiviral particles were produced by co-transfecting HEK293T cells with the plasmid along with packaging plasmids psPAX2 and pMD2.G using a standard transfection method. Viral supernatants were collected at 48 and 72 h post-transfection, filtered through a 0.45-μm filter and concentrated by ultracentrifugation at 25,000 rpm for 2 h at 4 °C. Cells were transduced with lentivirus, incubated for 2 days selected with 1 μg ml−1 blasticidin for 10–14 days, and BFP expression was analysed by flow cytometry. We took sgRNA specificity into account from the design phase of the CRISPRi screen. Our guide selection criteria included off-target scoring from ref. We designed the library in benchling (https://benchling.com) with multiple independent sgRNAs per EIE element. This redundancy helps to distinguish on-target biological effects from off-target noise. To increase our stringency and ensure that the effects of low-efficiency or low-specificity guides do not interfere with the interpretation of the screen, we used FlashFry82 to score our gRNAs with multiple tools (Supplementary Table 12) and specifically selected the CRISPRi specificity score developed by ref. We report effects only for elements with at least two guides achieving a specificity score greater than 0.2, which is the standard cut-off for this scoring parameter (similar to the Doench et al.84 cumulative distribution function score). The oligo pool encoding guides (Supplementary Table 10) were synthesized by Twist Bio and inserted into addgene Plasmid #52963 lentiGuide-Puro digested with Esp3I enzyme (NEB). The oligo pool was sequence validated. To investigate the effects of CRISPRi, we utilized a lentiviral delivery system to introduce sgRNAs into cells stably expressing the dCas9-KRAB repressor. Lentiviral particles were produced as described above. The viral titre was determined by transducing HEK293T cells with serial dilutions of virus and assessing transduction efficiency via flow cytometry for GFP expression. For transduction, cells were seeded at a density of 1 × 106 cells per well in six-well plates and transduced overnight with lentivirus at a low multiplicity of infection of 0.3, ensuring single sgRNA integration per cell. The following day, the medium was replaced with fresh growth medium. Two days post-transduction, cells were selected with 0.5 μg ml−1 puromycin for 4 days to enrich successfully transduced cells. GFP expression was monitored by flow cytometry to assess transduction efficiency. After selection, cells were collected at multiple timepoints: baseline (day 4 after transduction), day 3, week 1 and month 1 (30 days). Genomic DNA was extracted using the DNeasy Blood & Tissue Kit (Qiagen) following the manufacturer's instructions. Integrated sgRNA sequences were amplified from genomic DNA using a multistep PCR process. First, sgRNA cassettes were amplified using Primer set 1: hU6_pcr_out_fw (tggactatcatatgcttaccgtaacttgaaagt) and efs_pcr_rev (ctaggcaccggatcaattgccga). PCR reactions contained 0.8 μl each of 25 μM primers, 1–2 μg genomic DNA, water and 25 μL NEB 2x master mix in a total volume of 50 μl. PCR products (~400 bp) were verified by gel electrophoresis and purified. The second PCR step added Illumina sequencing adapters using primers (P5 stagger -hu6 and p7adpt_spRNAl105nt_rev). Reactions contained 10–50 ng of purified PCR1 product, 0.8 μl of each primer, water and 25 μl of NEB 2× master mix in a total volume of 50 μl. PCR products (200–300 bp) were gel-verified and purified using AMPure XP beads. A final indexing PCR step was performed using Truseq-based P5 and P7 indexing primers. Reactions contained 10–50 ng DNA from PCR2, 0.8 μl of each primer, water and 25 μl NEB 2× master mix in 50 μl total volume. Products were purified with AMPure XP beads and sequenced on an Illumina NextSeq platform using single-end 50-bp reads. Sequencing data were processed to quantify sgRNA representation at each timepoint, allowing analysis of sgRNA abundance dynamics over the experiment duration. To compute the effect of each guide on cell fitness, we first quantified guide counts from sequencing libraries. To normalize counts across libraries, we converted raw guide counts to counts per million (CPM) and retained guides that had CPM values of at least 20 across all days tested. We also filtered out guides with high off-target scores (Supplementary Table 12, 0.2 cut-off from optimized CRISPRi design parameters83) and excluded EIEs with fewer than two guides remaining after filtering. After confirming that normalized guide abundances were robust across replicates, we proceeded with our analysis using the average of guide replicates at each timepoint. Finally, we transformed this distribution to Z scores and reported this as the relative fitness effect of each guide. In our study, we used the CRISPR-CATCH technique to isolate and analyse ecDNA structures. Following the standard protocol22, we designed two sgRNAs targeting specific enhancer regions: sgRNA #1 (ATATAGGACAGTATCAAGTA) and sgRNA #2 (TATATTATTAGTCTGCTGAA). These sgRNAs directed the Cas9 nuclease to introduce double-strand breaks at the targeted sites, linearizing the circular ecDNA molecules. The linearized DNA was then subjected to PFGE using Saccharomyces cerevisiae and Hansenula wingei DNA ladders as molecular weight markers to facilitate size-based separation. Distinct DNA bands corresponding to the targeted ecDNA were excised from the gel for downstream analyses, including sequencing. Probes were designed against human genome assembly hg19, tiling the regions in Supplementary Table 7 using the probe designing software described previously36,37. We restricted the selection of the 40-mer probe targeting regions to a GC content between 20% and 80% and a melting temperature of 65–90 °C, and excluded sequences with non-unique homology—defined as sharing a 17-mer or longer sequence with other genomic regions—or homology to common repetitive elements in the human genome listed in RepBase, using a 14-mer cut-off. Targeting probes were then appended with a 20-mer barcode per target region. Probe design software is available via GitHub at https://github.com/BoettigerLab/ORCA-public. Finalized probe libraries were ordered as an oligo-pool from GenScript. ORCA hybridization was performed as previously described36,37. In brief, 40-mm Bioptechs coverslips were prepared with EMD Millipore poly-D-lysine solution (1 mg ml−1, 20 ml, dilute 1:10) (Sigma, cat. Coverslips were then rinsed three times in 1× PBS. Cells were passaged onto the coverslips and allowed to adhere overnight. The next day, the coverslip with cells were rinsed three times in 1× PBS and then fixed for 10 min in 4% paraformaldehyde. For DNA imaging, cells were then permeabilized in 0.5% Triton-X 1× PBS for 10 min followed by 5 min of denaturing in 0.1 M HCl. A 35-min incubation in hybridization buffer prepared samples for the primary probe. Primary probes were added (1 μg) directly to the sample in hybridization solution, and then the sample was heated to 90 °C for 3 min. An overnight 42 °C incubation (or at least 8 h incubation) was performed, followed by post-fixation in 8% paraformaldehyde + 2% glutaraldehyde in 1× PBS, before being stored in 2× SSC or used immediately for imaging. For RNA imaging, the HCl, heat and post-fixation steps were omitted. DNA samples were imaged on one of two different homebuilt set-ups designed for ORCA, ‘scope-1' or ‘scope-3', depending on instrument availability. Microscope design parameters were deposited in the Micro-Meta App85. The design and assembly of the scope-1 system is described in detail in our prior protocol paper37. Both systems use a similar auto-focus system, fluidics system and scientific complementary metal-oxide-semiconductor camera (Hamamatsu FLASH 4.0), although scope-3 had a larger field of view (2,048 × 2,048 108-nm pixels) compared with scope-1 (1,024 × 1,024 154-nm pixels). RNA samples were imaged on a different homebuilt set-up designed for ORCA designated as the ‘Yale lumencor system'. This system uses a similar auto-focus system and fluidics system, with a scientific complementary metal-oxide-semiconductor camera (Hamamatsu ORCA BT fusion) with a field of view of 2,304 × 2,304 at 108 nm per pixels and an Olympus PlanApo 60× objective. Automated fluidics handling is described in detail in our prior protocol paper37. In brief, fluid exchange between each imaging step was performed by a homebuilt robotic set-up. The system used a three-axis computer numerical control router engraver, buffer reservoirs and hybridization wells (96-well plate) on a three-axis stage, ethylene tetrafluoroethylene tubing, imaging chamber (FCS2, Bioptechs), a needle and peristaltic pump (Gilson F155006). The needle was moved between buffers or hybridization wells and transported across the samples through tubing using a peristaltic pump. Sequential imaging of ORCA probes was conducted alternating between hybridization of fluorescent adapter probes, readout probes complementary to the barcodes on the primary probe sequences, imaging and stripping of probes, as described previously36,37. In brief, a z-stack spanning 10 μm was acquired with 250 nm step sizes, alternating laser excitation between the data channel and fiducial marker at each step. Readout probes were labelled with Alexa-750 fluorophores. The fiducial probe was labelled in cy3 and added only in the initial round. Image processing was performed with custom MATLAB functions available via GitHub at https://github.com/BoettigerLab/ORCA-public. In brief, cells were maximum projected, and pixel-scale alignment across all fields of view was computed using the fiducial signal. This alignment was then applied in three dimensions across all 250-nm z steps. Cellpose86 was then used to segment individual cells. A cell-by-cell fine scale (subpixel) alignment was then computed, and aligned individual cells were then ready for 3D-spot calling. The individual ecDNA spots and their 3D positions computed to subpixel accuracy using the corresponding raw 3D image stacks and the 3D DaoSTORM function in storm-analysis toolbox (https://doi.org/10.5281/zenodo.3528330), an open-source software for single-molecule localization, adapted for dense and overlapping emitters following the DaoSTORM algorithm87. DaoSTORM was run in the 2D-fixed mode, as the 3D fitting modes are for estimating axial position from astigmatism in the xy plane, rather than computing it directly from a z-stack. The fixed-width point spread function of the microscope is precomputed using 100-nm (subdiffraction) fluorescent beads. A minimum detection threshold of 30 sigma was used for the fit. The z-position of the localizations was computed using Gaussian fit to the vertically stacked localizations, with an axial Gaussian width also precomputed from z-stack images with 100-nm fluorescent beads. Additional information can be found in the read-the-docs for storm-analysis at https://storm-analysis.readthedocs.io/en/latest/. All pairwise distances between genomic regions were calculated on a per-cell basis. The shortest distances were saved for each MYC centroid and EIE 14 and PVT1 such that each MYC centroid has one corresponding shortest distance per EIE 14 and PVT1. For each cell, a sphere radius r = 4um (the average radius of cells calculated with Cellpose mask) with randomly simulated points corresponding to the number of MYC, EIE 14 and PVT1 centroids. The same minimum pairwise distance quantification was calculated on the randomly simulated points. To calculate the density-corrected distance ratios, a distance cut-off of 2 μm and an interval density of 0.01:0.01:2 was used. On a per-cell basis, the distance density function was calculated, truncated at the specified cut-off. A uniform distribution was then computed over the same interval, and a ratio of these values was taken. This ratio was then corrected by the volume of the interval shell. All plasmids are made with Gibson assembly (NEB HIFI DNA assembly kit) according to the manufacturer's protocol. We used a plasmid from this publication20 containing the MYC promoter (chr8:128,745,990–128,748,526, hg19) driving NanoLuc luciferase (PVT1p-nLuc) and a constitutive thymidine kinase (TK) promoter driving Firefly luciferase. This plasmid served as the negative control. pGL4-tk-luc2 (Promega) plasmids with an enhancer (chr8:128347148–128348310) were used as the positive control20. In the test plasmid, the cis-enhancer was replaced by 1.7 kb sequence of EIE 14 or by Part #1: L1PA2 or by Part #2: L1M4a1 (Supplementary Table 13). To assess luciferase reporter expression, COLO320DM cells were seeded into a 24-well plate with 100,000 cells per well. Reporter plasmids were transfected into cells the next day with Lipofectamine 3000 following the manufacturer's protocol, using 0.25 μg DNA per well. Luciferase levels were quantified using Nano-Glo Dual reporter luciferase assay (Promega). All statistical tests used, replicate information and sample size information are reported in the figure legends. No statistical method was used to predetermine sample size. No samples or data points were excluded. The experiments were not randomized. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All sequencing data generated in this study are available via Gene Expression Omnibus (GEO) accession number GSE277492 at https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE277492 and via BioProject NCBI ID: 1162466 at https://www.ncbi.nlm.nih.gov/bioproject/1162466. Raw RNA imaging data related to Fig. 5 are available via Zenodo at https://doi.org/10.5281/zenodo.16921322 (ref. All raw imaging data related to the DNA are available upon request as they are large. The processed data tables from image analysis recording x,y,z positions of RNA and DNA are available via GitHub at https://github.com/sedona-Eve/Kraft_Murphy_Jones_ecDNA/. Source data are provided with this paper. The image analysis code is publicly available via GitHub at https://github.com/BoettigerLab/ORCA-public/ and at https://storm-analysis.readthedocs.io/en/latest/analysis.html. Code for reconstructing amplicons from long-read data with the CoRAL algorithm is available via GitHub at https://github.com/AmpliconSuite/CoRAL. Wahl, G. M. The importance of circular DNA in mammalian gene amplification. Benner, S. E. Double minute chromosomes and homogeneously staining regions in tumors taken directly from patients versus in human tumor cell lines. Kim, H. Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers. Yan, X. Extrachromosomal DNA in cancer. Parallel sequencing of extrachromosomal circular DNAs and transcriptomes in single cancer cells. Rosswog, C. Chromothripsis followed by circular recombination drives oncogene amplification in human cancer. Turner, K. M. Extrachromosomal oncogene amplification drives tumour evolution and genetic heterogeneity. Abeysinghe, H. R. Amplification of C-MYC as the origin of the homogeneous staining region in ovarian carcinoma detected by micro-FISH. deCarvalho, A. C. Discordant inheritance of chromosomal and extrachromosomal DNA elements contributes to dynamic disease evolution in glioblastoma. Lange, J. T. The evolutionary dynamics of extrachromosomal DNA in human cancers. Luebeck, J. Extrachromosomal DNA in the cancerous transformation of Barrett's oesophagus. Gisselsson, D. Chromosomal breakage–fusion–bridge events cause genetic intratumor heterogeneity. Roy, N. Translocation–excision–deletion–amplification mechanism leading to nonsyntenic coamplification of MYC and ATBF1. Rausch, T. Genome sequencing of pediatric medulloblastoma links catastrophic DNA rearrangements with TP53 mutations. Nones, K. Genomic catastrophes frequently arise in esophageal adenocarcinoma and drive tumorigenesis. Ly, P. Chromosome segregation errors generate a diverse spectrum of simple and complex genomic rearrangements. Shoshani, O. Chromothripsis drives the evolution of gene amplification in cancer. Wu, S. Circular ecDNA promotes accessible chromatin and high oncogene expression. Helmsauer, K. Enhancer hijacking determines extrachromosomal circular MYCN amplicon architecture in neuroblastoma. Hung, K. L. ecDNA hubs drive cooperative intermolecular oncogene expression. Zhu, Y. Oncogenic extrachromosomal DNA functions as mobile enhancers to globally amplify chromosomal transcription. Hung, K. L. Targeted profiling of human extrachromosomal DNA by CRISPR-CATCH. Hung, K. L. Coordinated inheritance of extrachromosomal DNA species in human cancer cells. Babaian, A. et al. Onco-exaptation of an endogenous retroviral LTR drives IRF5 expression in Hodgkin lymphoma. & Mager, D. L. Endogenous retroviral promoter exaptation in human cancer. Deniz, Ö et al. Endogenous retroviruses are a source of enhancers with oncogenic potential in acute myeloid leukaemia. Li, X. et al. LINE-1 transcription activates long-range gene expression. Sundaram, V. & Wysocka, J. Transposable elements as a potent source of diverse cis-regulatory sequences in mammalian genomes. & Yue, F. Analysis of Hi-C data for discovery of structural variations in cancer. Integration of Hi-C with short and long-read genome sequencing reveals the structure of germline rearranged genomes. Zhu, K. et al. CoRAL accurately resolves extrachromosomal DNA genome structures with long-read sequencing. Baldwin, E. T. Structures, functions and adaptations of the human LINE-1 ORF2 protein. Adney, E. M. Comprehensive scanning mutagenesis of human retrotransposon LINE-1 identifies motifs essential for function. Altemose, N. Complete genomic and epigenetic maps of human centromeres. Abel, H. J. Mapping and characterization of structural variation in 17,795 human genomes. Mateo, L. J. et al. Visualizing DNA folding and RNA in embryos at single-cell resolution. Mateo, L. J. Tracing DNA paths and RNA profiles in cultured cells and tissues with ORCA. Alexander, J. M. Live-cell imaging reveals enhancer-dependent Sox2 transcription in the absence of enhancer proximity. Benabdallah, N. S. Decreased enhancer–promoter proximity accompanying enhancer activation. & Levine, M. S. Enhancer–promoter communication: hubs or loops?. New insights into promoter–enhancer communication mechanisms revealed by dynamic single-molecule imaging. & Herranz, D. The MYC Enhancer-ome: long-range transcriptional regulation of MYC in cancer. Zimmerman, M. W. MYC drives a subset of high-risk pediatric neuroblastomas and is activated through mechanisms including enhancer hijacking and focal enhancer amplification. Larson, M. H. CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Penttinen, R. P. Biosynthesis, secretion and crosslinking of collagen with reference to aging. Hongxing, Z. Depletion of c-Myc inhibits human colon cancer colo 320 cells' growth. Stover, E. H. et al. Pooled genomic screens identify anti-apoptotic genes as targetable mediators of chemotherapy resistance in ovarian cancer. Buenrostro, J. D. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Fulco, C. P. Activity-by-contact model of enhancer-promoter regulation from thousands of CRISPR perturbations. Consortium, E. P. An integrated encyclopedia of DNA elements in the human genome. New developments on the Encyclopedia of DNA Elements (ENCODE) data portal. Long, H. K. Loss of Extreme Long-Range Enhancers in Human Neural Crest Drives a Craniofacial Disorder. Castro-Diaz, N. Evolutionally dynamic L1 regulation in embryonic stem cells. Liu, N. Selective silencing of euchromatic L1s revealed by genome-wide screens for L1 regulators. Robbez-Masson, L. The HUSH complex cooperates with TRIM28 to repress young retrotransposons and new genes. Widespread somatic L1 retrotransposition in normal colorectal epithelium. Lander, E. S. Initial sequencing and analysis of the human genome. Iskow, R. C. Natural mutagenesis of human genomes by endogenous retrotransposons. Beck, C. R. LINE-1 elements in structural variation and disease. Helman, E. Somatic retrotransposition in human cancer revealed by whole-genome and exome sequencing. Scott, E. C. A hot L1 retrotransposon evades somatic repression and initiates human colorectal cancer. Cajuso, T. Retrotransposon insertions can initiate colorectal cancer and are associated with poor survival. Payer, L. M. & Burns, K. H. Transposable elements in human genetic disease. Ardeljan, D. Cell fitness screens reveal a conflict between LINE-1 retrotransposition and DNA replication. McKerrow, W. LINE-1 expression in cancer correlates with p53 mutation, copy number alteration, and S phase checkpoint. Miki, Y. Disruption of the APC gene by a retrotransposal insertion of L1 sequence in a colon cancer. Fuentes, D. R. Systematic perturbation of retroviral LTRs reveals widespread long-range effects on human gene regulation. Murphy, S. E. & Boettiger, A. N. Polycomb repression of Hox genes involves spatial feedback but not domain compaction or phase transition. Corces, M. R. The chromatin accessibility landscape of primary human cancers. Ng, A. W. T. Disentangling oncogenic amplicons in esophageal adenocarcinoma. Rodriguez-Martin, B. Pan-cancer analysis of whole genomes identifies driver rearrangements promoted by LINE-1 retrotransposition. Cohen, S. N. Transposable genetic elements and plasmid evolution. Rao, S. S. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Servant, N. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Durand, N. C. Juicer provides a one-click system for analyzing loop-resolution Hi-C experiments. Li, H. Minimap2: pairwise alignment for nucleotide sequences. Detection of mosaic and population-level structural variants with Sniffles2. Talevich, E., Shain, A. H., Botton, T. & Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. Nattestad, M., Aboukhalil, R., Chin, C.-S. & Schatz, M. C. Ribbon: intuitive visualization for complex genomic variation. Hsu, P. D. et al. DNA targeting specificity of RNA-guided Cas9 nucleases. & Shendure, J. FlashFry: a fast and flexible tool for large-scale CRISPR target design. Jost, M. et al. Titrating gene expression using libraries of systematically attenuated CRISPR guide RNAs. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. Rigano, A. Micro-Meta App: an interactive tool for collecting microscopy metadata based on community specifications. Stringer, C. Cellpose: a generalist algorithm for cellular segmentation. Holden, S. J. DAOSTORM: an algorithm for high- density super-resolution microscopy. Murphy, S. Enhancer activation from transposable elements on extrachromosomal DNA. This project was supported by Cancer Grand Challenges CGCSDF-2021\100007 with support from Cancer Research UK and the National Cancer Institute (H.Y.C. was supported by a Stanford Bio-X SIGF Fellowship and the NIH award DP5OD037361 through the OD and the NIDCR. A.B.-S. was supported by the Stanford Medical Scholars Research Program and an Alpha Omega Alpha Carolyn L. Kuckein Student Research Fellowship. was supported by a Stanford Graduate Fellowship and an NCI Predoctoral to Postdoctoral Fellow Transition Award (NIH F99CA274692 and K00CA274692). was supported by an NSF Graduate Research Fellowship (DGE-1656518). was supported by the Schmidt Science Fellows program. was an Investigator of the Howard Hughes Medical Institute. was supported in part by the Cancer Grand Challenges partnership funded by Cancer Research UK (CGCATF-2021/100025) and the National Cancer Institute (OT2CA278635), U24CA264379, and by R01GM114362. We thank M. Koska for help with luciferase measurement. Katerina Kraft, Quanming Shi & Howard Y. Chang Present address: Amgen Research, South San Francisco, CA, USA Present address: Max Planck Institute For Molecular Genetics, Berlin, Germany Present address: Department of Biology, Massachusetts Institute of Technology, Cambridge, MA, USA Present address: Department of Neuroscience, Scripps Research, La Jolla, CA, USA These authors contributed equally: Katerina Kraft, Sedona E. Murphy, Matthew G. Jones. Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA Katerina Kraft, Matthew G. Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, Britney J. He, Rui Li & Howard Y. Chang Sedona E. Murphy, Michael T. Montgomery & Howard Y. Chang Department of Cell Biology, Yale School of Medicine, New Haven, CT, USA Sarafan ChEM-H Institute and Department of Pathology, Stanford University, Stanford, CA, USA Aarohi Bhargava-Shah, Natasha E. Weiser, Yanbo Wang & Paul S. Mischel Department of Chemical and Systems Biology, Stanford University, Stanford, CA, USA Basic Science and Engineering Initiative, Stanford Children's Health, Betty Irene Moore Children's Heart Center, Stanford, CA, USA Department of Computer Science and Engineering, University of California at San Diego, La Jolla, CA, USA Institute for Systems Genetics, NYU Langone Health, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the paper with input from all authors. provided guidance on TE analysis. provided guidance on paper content. contributed instrument time and software and advised on image data analysis. is a cofounder of Accent Therapeutics, Boundless Bio, Cartography Biosciences and Orbital Therapeutics; he was an advisor of 10x Genomics, Arsenal Biosciences, Chroma Medicine and Spring Discovery until 15 December 2024. is a consultant and holds equity in Tahoe Therapeutics. is a co-founder and advisor of Boundless Bio. is a founder and director of CDI Labs, Inc.; a founder of and consultant to Opentrons LabWorks/Neochromosome, Inc.; and serves or served on the scientific advisory boards of the following: CZ Biohub New York, LLC; Logomix, Inc.; Modern Meadow, Inc.; Rome Therapeutics, Inc.; Sangamo, Inc.; Tessera Therapeutics, Inc.; and the Wyss Institute. is a cofounder, serves on the scientific advisory board of Boundless Bio and Abterra and holds equity in both companies. The other authors declare no competing interests. Nature Cell Biology thanks Miguel Branco and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Overlap of each EIE with the annotated genomic coordinates of LINE, SINE, or LTR elements. The background genome average of each class of repetitive element is reported as a solid black line. The graph (top) demonstrates the number of structural variations called in stripe alignments. Relationship between structural variations and read count for each element (bottom). Pearson correlation is 0.61. c. Schematics of ecDNA harboring 1.7 kb sequence obtained from long-read analysis of EIE 14. The region spanning 6-710 bp shows alignments with 3' end of the LINE-1 element (L1PA2), whereas the region from 711-1690 bp is notably unique to intron 2 of the CD96 locus on chromosome 3 (L1M4a1). The L1M4a1-like segment harbors a polyA-signal–like motif (AAAAAG). d. Top panel, alignment of predicted protein from 6-710 bp with LINE-1 ORF2 (L1PA2). Bottom panel, amino acids alignment of LINE-1 ORF2 (L1PA2) and 6-710 bp coding protein by clustalW. a. Screenshot of the IGV viewer with selected long reads depicting insertion sizes in purple. b. Long-read pileup across chr3 and chr8 demonstrating EIE 14 translocation to chr8 ecDNA locus. c. Sequence alignment of the T2T genome. Arrow points to the FISH signal. The experiment was performed twice with similar results. b. Quantification of copy number of MYC, PVT1 and EIE 14 across all measured cells (n = 1329, 2 biological replicates). Mean copy number of MYC is 29 copies per cell, PVT1 is 31 copies per cell and EIE 14 is 22 copies per cell. c. Correlation plots between the loci per cell. Pearson's correlation coefficient calculated for PVT1-MYC r = 0.82, EIE 14-MYC r = 0.71, EIE 14-PVT1 r = 0.74. d. Violin plots of shortest distances of MYC to PVT1 and EIE 14 (median distance denoted by red line). Red line denotes median distance. e. Histogram of shortest distances of MYC to PVT1 (blue) and MYC to EIE 14 (orange) (Wilcoxon two-sided ranksum p = 1.23e^-05). a. Schematic of the CRISPRi screening strategy used to evaluate the regulatory potential of the 68 EIEs by designing 4-6 gRNAs per element for a total of 257 genomic regions tested and 125 non-targeting control sgRNAs. The screen involved the transduction of cells with a lentivirus expressing dCas9-KRAB and the sgRNAs such that each cell received 1 sgRNA, followed by calculation of cell growth phenotype over a series of time points (Baseline(4 days), Baseline + 3 days, Baseline + 14 days, and Baseline + 1 month). The screen was further filtered on guide specificity (methods) and 36/68 targeted EIEs met the qualifying threshold. The growth phenotype of COLO320DM cells and reproducibility of counts between two biological replicates at different timepoints for the 36 qualifying EIEs. Each point represents the average guide effect (Z-score). Pearson correlations (r) are reported for reproducibility plots. c. Growth phenotypes of the qualifying EIE-targeting guides in COLO320DM cells across multiple time points. Each point represents the average guide effect (Z-score) for sgRNAs targeting a specific EIE. Guides with an average abs(Z-score) > 1 are annotated. Previously, ATAC-seq data was mapped to hg19. ATAC-seq data was further normalized by library size. a. Quantification of RNA-FISH signal on a per cell basis from COLO320-DM cells labeling MYC exon 2 and EIE 14 (see Methods for quantification method). n = 712 cells across 2 biological replicates. As in (a) but for COLO320-HSR cells. n = 681 cells across 2 biological replicates. c. Fold change in luciferase signal driven by the L1PA2 (part 1), L1M4a1 (part2) and combined (EIE14) regions of EIE 14 for n = 3 biological replicates. P-value MYC promoter (ctrl) vs part 1 p = 0.0006; p-value MYC promoter (ctrl) vs part 2 p = 0.0015; value MYC promoter (ctrl) vs part 1+part2 (EIE14) p = 0.0001 (p-values obtained from two-tailed unpaired t-test). Error bars are stanford deviations from the mean. This table includes genomic sequences identified from Hi-C analyses aligned to the hg19 genome reference. EIE from T7 in RepBase, describing genomic regions matching known elements from RepBase database. Columns: ‘Class': classification of the element (for example, NonLTR/SINE). ‘Dir': direction of element integration. ‘Sim': similarity percentage to reference sequences. Reference annotations from RepBase database used in T4. Structural variant (SV) details identified by Nanopore sequencing aligned to the hg19 genome. ‘ALT': alternative allele indicating the variant or sequence. ‘SVLEN': length of the structural variant. ORCA primary probes for RNA and DNA FISH imaging. Each primary probe has two sequences corresponding to forward and reverse primer index, a common readout sequence, a unique readout sequences and the 40-bp sequence with homology to target. CRISPRi results showing Z scores for individual guide RNAs. Columns: ‘Baseline_zscore', ‘3days_zscore', ‘2weeks_zscore', ‘1month_zscore': Z scores measured at different timepoints. Details of guide RNAs used in CRISPRi experiments. Classification of TEs as old or young based on evolutionary activity. Columns: ‘TE_Age': Indicates if TE is considered ‘old' (inactive) or ‘young' (recently active). Analysis for off-target effects for sgRNAs used based on different publications indicated in columns. Sequences of enhancer elements tested in luciferase reporter assays, separated into two parts (L1PA2 and L1M4a1). 1a: statistical source data related to Fig. 1f: statistical source data related to Fig. 5c: statistical source data for Fig. 3: statistical source data related to Fig. 1f: statistical source data related to Fig. 1a: statistical source data related to Fig. Unprocessed gel related to Fig. Unprocessed images related to Fig. Unprocessed image related to Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Enhancer activation from transposable elements in extrachromosomal DNA. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03969-0'>Safety and pharmacokinetics of SARS-CoV-2 DNA-encoded monoclonal antibodies in healthy adults: a phase 1 trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 09:23:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Local intramuscular administration of synthetic plasmid DNA (pDNA) encoding monoclonal antibodies (mAb) offers an alternative to recombinant protein-based mAb delivery. In this phase 1 dose-escalation study, we evaluated the safety, tolerability and pharmacokinetics of a pDNA cocktail encoding AZD5396 and AZD8076, modified versions of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) neutralizing mAb cocktail tixagevimab/cilgavimab in healthy adults. Participants received up to four intramuscular doses of pDNA encoding both DNA-based mAbs (DMAbs), administered using CELLECTRA electroporation. The primary endpoints were safety and pharmacokinetics. All 44 participants received at least one dose; DMAbs were detected in 100% of evaluable participants (n = 39), with serum concentrations reaching a peak of 1.61 µg ml−1. Sustained expression was observed in all participants during the 72 weeks of follow-up. The study product was well tolerated, with no product-related serious adverse events reported. Exploratory analyses demonstrated binding to multiple SARS-CoV-2 Spike protein variants and neutralizing activity in a standard pseudovirus assay. No antidrug antibodies were detected across approximately 1,000 serum samples using validated tiered assays. To our knowledge, these data represent the first-in-human proof-of-concept that synthetic pDNA DMAb technology permits the durable in vivo production of a functional mAb cocktail. This study further underscores the collective importance of synthetic design, formulation and delivery to achieve biologically relevant expression of gene-encoded biologics. DMAb delivery may represent a long-acting, scalable, cold-chain-independent platform against a wide range of diseases that can be targeted with mAbs and their derivatives. The coronavirus disease 2019 (COVID-19) pandemic, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has driven the global scientific community to advance development of new medical countermeasures. Nucleic acid technologies, including notably mRNA vaccines, had an instrumental role in mitigating the spread of SARS-CoV-2 and reducing COVID-19 disease morbidity and mortality1,2. Despite the notable achievements in vaccine development and distribution, a substantial portion of the population remains inadequately protected because of their inability to raise sufficient levels of virus-neutralizing antibodies in response to vaccination3. This gap in immunity underscores the need for additional prophylactic and therapeutic approaches for pandemics, which may also be useful as more generalized biological tools. Infectious disease-targeting monoclonal antibodies (mAbs) have emerged as promising medical countermeasures. Recombinant mAbs have demonstrated the ability to neutralize several strains of SARS-CoV-2, thereby offering a potential pathway to both treat4,5,6,7,8, and prevent COVID-199. Two mAb products received Emergency Use Authorization for the prevention of severe COVID-19 disease and hospitalization in persons who do not respond to vaccination indication: tixagevimab/cilgavimab (authorized from 8 December 2021 to 26 January 2023) and pemivibart (authorized 22 March 2024). Despite their immense clinical value, the delivery of mAbs as recombinant proteins poses notable delivery hurdles, in addition to manufacturing, logistical and economic challenges. In some cases, intravenous administration can restrict the feasibility of large-scale implementation across broad patient populations. Recombinant mAb delivery also requires a continuous cold chain for stability, presenting further challenges for distribution to resource-limited settings, including low-income and middle-income countries. Optimizing additional strategies to extend the in vivo biological half-life of antibodies and further improve the duration of protection remains a key objective. Innovative solutions are being pursued to improve developmental timelines, delivery and durability of antibody-based interventions. Among these, the use of nucleic-acid-based platforms, such as mRNA, DNA and vectored immune prophylaxis, present new methods for the in vivo production of mAbs. If successful, these techniques could allow a single dose to maintain continuous production of the intended products, making subsequent doses unnecessary. While in vivo gene-based production of mAbs has generated excitement, challenges have been exposed. These include a short duration of expression for an intravenously delivered mRNA-delivered chikungunya virus mAb10 and, for studies that achieved longer expression such as adeno-associated virus (AAV)-based vectored immunoprophylaxis, a high incidence (38%, 3 of 8 participants) of antidrug antibody (ADA) induction that prevented continued expression of the mAb transgene11. In this study, we evaluated the use of synthetic DNA-based monoclonal antibody (DMAb) technology for the in vivo production of functional antibodies. Preclinical studies in multiple animal models demonstrated robust expression and protective efficacy of this DMAb approach12,13. To translate these findings to humans, we conducted a phase 1, dose-escalation trial assessing the safety, tolerability and pharmacokinetics of this DNA-delivered antibody platform in healthy adult participants. In this article, we present full data from the 72 weeks of follow-up, including assessments of in vivo DMAb expression, binding and neutralizing activity, and immunogenicity. These findings support the potential of DNA-based delivery as a long-acting, cold-chain-independent strategy for antibody-based interventions. This phase 1, open-label, single-center, dose-escalation trial evaluated the safety and pharmacokinetic (PK) profile of DNA-encoded mAbs (primary endpoints), with exploratory analyses of antigen binding and virus neutralization, and a post hoc assessment of ADAs; further details on trial design are provided in Methods. Synthetic DNA plasmids encoding the light and heavy chains (HCs) of AZD5396 and AZD8076 were co-delivered intramuscularly as two separate constructs (pAZD5396 and pAZD8076) for in vivo antibody production. Delivery was facilitated using the CELLECTRA 2000 device with side-port needle to apply localized electroporation (EP) after injection, enhancing muscle uptake and expression efficiency (Fig. The clinical study included eight dosing cohorts with varying injection frequencies, volumes and EP parameters. A schematic overview of the study design and participant flow is shown in Fig. 2; a detailed description is provided in Methods. a, Four synthetic DNA constructs were designed for optimal in vivo expression of the HCs and light chains (LCs) of AZD5396 and AZD8076. pAZD5396 and pAZD8076 LC and HC synthetic DNA construct cocktails were each formulated with the human recombinant hyaluronidase enzyme. c, CELLECTRA EP delivery system for enhanced local transgene expression. The EP electrical field parameters and injected fluid distribution have been optimized to align to permit enhanced transgene expression in the muscle (shown as green fluorescent protein reporter gene expression in New Zealand rabbit muscle). Illustrations in a and b created using BioRender.com. a, CONSORT-style diagram showing the number of participants screened, enrolled, dosed and included in the safety and PK analyses across all cohorts. aOne participant in cohort B received only one dose and was analyzed as part of cohort A1. b, Diagram of the study design and cohort progression. The trial followed a sequential dose-escalation strategy beginning with two single-dose cohorts: cohort A1 (0.5 mg, single dose) and cohort A2 (1.0 mg, single dose). Enrollment into each cohort was sequential and contingent on safety review of the preceding group. Eligible participants were healthy adults aged 18–60 years with a body mass index (BMI) between 20 kg m−2 and 30 kg m−2, normal laboratory and electrocardiogram (ECG) findings, and no notable comorbidities or immunosuppressive conditions. EP was delivered using the CELLECTRA 2000 with OpBlock 0078 for all cohorts except cohort F, which used a modified pulse setting (OpBlock 0070) (Fig. Baseline demographic characteristics are summarized in Fig. a, Demographic characteristics of the 44 participants enrolled in the study, including age, sex (self-reported), race and ethnicity. b, Frequency and severity of elicited AEs considered related to the study product or administration procedure, reported within 7 days after administration. Events are grouped according to category (for example, local, systemic) and graded according to protocol-defined criteria (mild, moderate, severe). Five participants withdrew early from the study and were replaced per protocol. Four participants discontinued after receiving the first EP dose on day 0; one participant withdrew at week 4 for personal reasons. These withdrawals occurred in cohorts D (one on day 0 and one at week 4), E (one on day 0) and F (two on day 0). All participants who received at least one dose were included in the safety analysis. However, the five early withdrawals were excluded from the PK analyses because of insufficient follow-up data. The early discontinuations were attributed to discomfort associated with the EP procedure or personal choice and were not related to adverse events (AEs) caused by the study product (Fig. AEs were categorized into two groups: elicited events, which were systematically collected during the 7 days after each administration, and unsolicited (non-elicited) AEs, which were monitored throughout the study duration. The most frequently reported elicited events were injection site pain (evaluated using a visual analog scale) and erythema, both of which were transient and typically resolved within minutes of administration (Extended Data Fig. These local reactions are consistent with expected responses to intramuscular injection and EP. Elicited AEs were further categorized according to severity and body system (Fig. 3b), with comprehensive listings in Supplementary Tables 2, 3, 5 and 8. Detailed characterization is provided in Supplementary Tables 2 and 4, while Supplementary Tables 6 and 7 summarize the number of participants who experienced at least one local or systemic elicited reaction within 7 or 10 days after dosing. Across all dosing levels, no dose-dependent trend in treatment-related AEs was observed. Most local reactions, including occasional scabbing, were mild and resolved without intervention. Importantly, systemic AEs typically associated with high serum antibody levels were not observed, aligning with the gradual in vivo expression and modest peak DMAb concentrations characteristic of this DNA-based delivery platform. Three serious AEs (SAEs) were reported: two episodes of recurrent pneumothorax in a single participant, one spontaneous miscarriage and a new diagnosis of melanoma in situ that was cured with surgery. All SAEs were assessed as unrelated to study product administration (Supplementary Table 9). Serum concentrations of DMAbs AZD5396 and AZD8076 were measured using validated electrochemiluminescence (ECL) immunoassays (Supplementary Fig. Longitudinal PK data were available through week 72 for participants in all cohorts. Notably, AZD5396 and AZD8076 were coexpressed at similar levels in individuals, supporting the feasibility of simultaneous delivery of multiple plasmids using this platform (Extended Data Figs. a–c, Sera collected at the indicated time points after DMAb administration were analyzed using a qualified quantitative binding assay to determine serum concentrations (ng ml−1) in individual participants who received a single dose of 0.5 mg or 1 mg on day 0 (indicated by the arrow) (a), two doses of 0.25 mg, 0.5 mg, 1 mg or 2 mg on days 0 and 3 (indicated by the arrows) (b) and four doses of 0.5 mg on days 0, 3, 28 and 31 (c) (indicated by the arrows). d, The mean serum concentrations of dose groups/cohorts are shown. e, Maximum DMAb serum concentration according to group. f, AUC through week 72 according to group. The study evaluated how pDNA dose, injection frequency and EP parameters influenced systemic DMAb levels. Escalating the pDNA dose from 0.25 mg (cohort D) to 0.5 mg (cohort B) per injection resulted in an 80% increase in average peak serum concentrations (Cmax), from 229 to 413 ng ml−1 (Extended Data Table 1). This trend was supported by the area under the curve (AUC) analysis, which showed comparable serum exposure across cohorts B, C and E, despite increasing doses (Fig. To evaluate injection frequency, an identical pDNA formulation (0.5 mg per injection) was administered across one-dose (cohort A1), two-dose (cohort B) and four-dose (cohort G) regimens (Fig. 4d,e), respectively, reflecting a more than 3.25-fold enhancement in peak DMAb levels (Extended Data Table 2). AUC analyses mirrored these results (Extended Data Fig. 3), supporting a dose-sparing strategy through multisite administration. Finally, the impact of the EP pulse pattern was assessed by comparing cohort F, which received a truncated delay (0.2 s between pulses; OpBlock 0070), with cohort B, which used standard 1-s spacing (OpBlock 0078). Despite using the same plasmid dose and formulation, cohort F achieved similar average peak concentrations (287.9 ng ml−1; range 138–452), although with greater interparticipant variability (Fig. 4) and worse tolerability, with two participants withdrawing from the study after day 0, which required replacement. Across all evaluable cohorts, serum DMAb levels increased steadily, peaking around weeks 6–8 after administration. Expression remained stable and durable in all participants for 72 weeks of follow-up. The functional activity of DMAbs expressed in vivo was assessed through binding to the SARS-CoV-2 Spike receptor binding domains (RBDs) and pseudovirus neutralization assays. All participants had preexisting anti-Spike antibodies because of prior infection or vaccination (Extended Data Table 3), necessitating use of an anti-YTE-specific antibody to detect AZD5396 and AZD8076 in the serum (Supplementary Fig. Using this anti-YTE capture, DMAbs were evaluated for RBD binding against several SARS-CoV-2 variants, including ancestral (Wuhan), Delta, Omicron BA.2 and Omicron BA.4/BA.5. Binding was detected in all 39 participants tested and was sustained through the final available time points, including up to week 72 for participants in cohorts A1–D and up to week 52 for those in cohorts E and F, and up to week 42 for cohort G (Fig. Binding levels correlated well with measured serum concentrations from Meso Scale Discovery (MSD) assays (Supplementary Fig. a–c, Anti-AZD5396 (left) and anti-AZD8076 (right) ADA measurements are shown for individual participants in the single-dose cohorts (A1/A2) (a), the two-dose cohorts (B–E) (b) and the four-dose cohort (G) (c). aADA levels were only tested up to week 24, not 72 like in other cohorts. Neutralization activity was tested using a pseudovirus assay targeting ancestral SARS-CoV-2 Spike. To isolate DMAb-specific activity, AZD5396 and AZD8076 were purified from serum using anti-YTE Dynabeads and quantified using both anti-YTE and anti-idiotype enzyme-linked immunosorbent assays (ELISAs) (Supplementary Fig. Neutralizing activity was detected in all 37 participants tested in cohorts A–G, with half maximal inhibitory concentration (IC50) values comparable to those of the parental recombinant antibodies (Fig. Participants 6302-0014 and 6302-0039 could not be evaluated because of insufficient antibody levels for purification. a–c, A DMAb-specific RBD binding assay (left) was developed, in which an anti-YTE antibody was used to capture DMAbs from participant sera followed by probing with biotinylated ancestral Spike RBDs. Graphs are plotted as the AUC × 1,000 for individual participants in each cohort; group means are shown in dark blue. To evaluate antiviral activity, DMAbs were purified from individual participant sera collected either before (day 0) or after (pooled from weeks 12–52) administration using anti-YTE-coated Dynabeads. The graphs depict the percentage (%) neutralization of individual samples at day 0 (black lines) and after DMAb delivery (colors) when tested at the indicated dilution. Group means are shown in dark blue. RBD binding activity (left) and pseudovirus neutralization (right) are shown for the single-dose cohorts (A1/A2) (a), the two-dose cohorts (B–E) (b) and the four-dose cohort (G) (c). d, Average IC50 (ng ml−1) of purified DMAbs from the indicated cohorts, calculated as the group geometric mean + 95% confidence interval (CI). Together, these results demonstrate that DMAbs expressed in vivo retained high-affinity binding and neutralization activity against SARS-CoV-2 variants over time. To evaluate potential host immune responses against DMAbs expressed in vivo, we used a validated ECL three-tiered ADA assay designed to detect antibodies targeting the Fab region of AZD1061 and AZD8895, and the parental mAbs of AZD5396 and AZD8076, respectively (Supplementary Fig. Approximately 1,000 serum samples were analyzed from participants who completed dosing. No confirmed ADAs were detected in any participant. 34, 36, 43 and 58) exhibited positive signals in the tier 1 screening assay for anti-AZD8895 or anti-AZD5396, including one (participant no. 34) with additional screening positivity for both targets. However, all samples tested negative in the tier 2 confirmatory assay and were considered false positives (Fig. These findings support the immunological tolerability of synthetic DNA-encoded DMAbs, even with multidose and multisite administration. Notably, previous studies of antibody gene delivery using viral vectors demonstrated host immune responses and ADA development that compromised antibody expression14,15,16. In contrast, the absence of ADA detection in this study suggests that the DNA platform may offer a more favorable immunological profile. This study demonstrates the in vivo production of functional mAbs in humans using synthetic DNA delivered via EP. The optimized DNA constructs were formulated to support efficient coexpression of HCs and LCs, with adaptive delivery techniques enabling sustained in vivo expression. Formulation with human recombinant hyaluronidase facilitated diffusion through the extracellular matrix, while side-port needles and the CELLECTRA adaptive EP platform enabled precise, localized delivery. In this phase 1 trial, we evaluated the safety, efficiency and durability of DMAb expression across eight dose-escalating cohorts. Across 324 plasmid deliveries, successful antibody expression was confirmed in 322 cases, reflecting highly efficient in vivo protein assembly. DMAb levels were durable and biologically relevant, ranging from hundreds of nanograms per milliliter to over 1 µg ml−1 in some individuals. Dose escalation revealed a saturation threshold; increasing pDNA from 0.25 mg to 0.5 mg notably raised expression, but increasing to 2.0 mg did not yield proportional gains. Interestingly, delivering the same total dose across four injection sites resulted in substantially greater expression than delivering it at one site, which is consistent with findings from AAV studies showing spatial limitations to delivery19. The DMAbs maintained high-affinity binding to SARS-CoV-2 RBD variants, including the Delta and Omicron sublineages, and demonstrated neutralizing activity in all 37 evaluable participants. This confirms that the antibodies were not only present but also functionally active. Importantly, no ADAs were detected in any of the 39 participants, including those who received multiple doses. This distinguishes the DMAb platform from traditional protein-based mAbs, which are associated with ADA development in 1–10% of cases for fully human products and up to 60% for chimeric or murine antibodies20. Recent clinical studies demonstrated the potential of alternative gene delivery platforms for mAb expression. A phase 1 trial of mRNA-1944, which encodes a chikungunya virus-neutralizing antibody, showed dose-dependent expression after intravenous infusion at doses of 0.1, 0.3 or 0.6 mg kg−1. The antibody exhibited a half-life of approximately 69 days, with sustained expression lasting up to 16 weeks at the higher dose levels; however, transient increases in inflammatory markers, such as C-reactive protein, interleukin-6, complement and IP-10, were observed shortly after dosing and resolved within 48 h. Participants in the 0.6 mg kg−1 group received corticosteroids to mitigate these inflammatory responses10. In parallel, two clinical trials evaluated AAV vectors for in vivo delivery of HIV broadly neutralizing antibodies (bNAbs). In a 2019 study, an AAV1 vector encoding the anti-HIV bNAb PG9 was administered intramuscularly in dose-escalation cohorts. PG9 activity was detected in serum and muscle biopsies in some participants; however, immunogenicity limited the response, with ADA to PG9 and anti-AAV1 immune responses noted, particularly at higher doses21. A more recent trial assessed an AAV8 vector encoding the anti-HIV bNAb VRC07. Participants with elevated baseline anti-AAV8 antibody titers were excluded from enrollment. Expression levels of VRC07 exceeded 0.1 µg ml−1 in most participants and persisted for up to 150 weeks in one individual. Nonetheless, ADA against VRC07 was observed in three of eight participants, alongside the induction of anti-AAV8 antibodies in most individuals and rare but detectable anti-vector T cell responses11,14,15,16. Larger, more diverse studies are needed to evaluate the generalizability of these findings and to assess platform performance across varied genetic backgrounds, delivery parameters and clinical settings. While the levels of mAbs achieved with DMAb administration were below those typically attained immediately after recombinant protein infusion, they remained within the therapeutic ranges reported for multiple biological indications. The observed concentrations persisted for 72 weeks after a single or repeated DNA administration. These findings are promising for long-acting prophylaxis and merit further optimization. Additional formulation development, dose scheduling and EP refinements may increase expression levels and support clinical efficacy. Importantly, the DMAb levels observed in this study are comparable to those reported for other effective biologics. For example, human anti-chikungunya virus, respiratory syncytial virus and Zika-virus-neutralizing mAbs22,23 demonstrate functional potency in the low ng ml−1 range. Similarly, approved bispecific antibodies such as Columvi (Ctrough ≈ 590 ng ml−1)24 and Blincyto (Css ≈ 228–537 pg ml−1)25, which are approved treatments for B cell lymphomas, and Kimmtrak (Css ≈ 13 ng ml−1) for treatment of uveal melanoma26, are active at similar levels. Moreover, protein therapeutics like incretins (for example, GLP-1 analogs) exert biological effects at single-digit ng ml−1 concentrations27. Despite its limitations, this study marks a notable advance in antibody gene delivery. We demonstrate that DNA-encoded mAbs can be reliably expressed in vivo with high fidelity, sustained levels and minimal immunogenicity. The consistency of in vivo expression from multivalent plasmid formulations, delivered intramuscularly using optimized EP, suggests robust uptake and transcription within postmitotic muscle fibers, which are long lived and capable of maintaining episomal DNA without dilution through cell division. This biological context, combined with the method's high take-up rate and codelivery efficiency, may explain the extended antibody expression observed. This study demonstrates a transformative approach for delivering epigenetic elements without the intention of genetic modification, resulting in long-lived therapeutic potential and circumventing multiple major limitations that have constrained previous strategies. The platform's simplicity, scalability, cold chain independence and potential for cost-effective deployment offer major advantages for global access and equitable distribution. Future studies to expand delivery formats, optimize immune potency and validate protective efficacy in broader populations are warranted. These findings lay the groundwork for a versatile, synthetic, nonviral gene therapy strategy with applications in infectious disease, chronic conditions and beyond. Synthetic DMAb technology was used to enable in vivo production of mAbs targeting SARS-CoV-2. Optimized DMAb constructs were developed in response to the COVID-19 pandemic, encoding AZD5396 and AZD8076, which are based on the parental mAb clones COV2-2130 and COV2-2196, respectively—the precursors of AZD7442. Preclinical studies in mice, hamsters and nonhuman primates demonstrated robust in vivo expression and protective efficacy of this DMAb cocktail12,13. Each DMAb was encoded by two synthetic plasmids (pAZD5396 and pAZD8076), encoding the HCs and LCs, respectively. Plasmids were administered intramuscularly and delivered using the CELLECTRA 2000 with side-port needle in vivo EP technology17,18. The EP system was configured to optimize electric field distribution at the injection site, thereby enhancing plasmid uptake and transgene expression in muscle tissue. This phase 1, open-label, dose-escalation trial was conducted at a single clinical site and approved by the institutional review board (IRB) of the University of Pennsylvania. All participants provided written informed consent before enrollment. Able and willing to comply with all study procedures. Screening laboratory values within normal limits or with only grade 0–1 findings. Women of child-bearing potential who agreed to use medically effective contraception (oral contraception, barrier methods, spermicide) or with a partner who was sterile from enrollment to 6 months after the last injection, or a partner who was medically unable to induce pregnancy. Abstinence was acceptable per investigator discretion if documented and if medically effective contraception was used when engaging in sexual activities, with the study team notified. Sexually active men considered sexually fertile agreed to use either a barrier method of contraception during the study and for at least 6 months after the last injection, or with a partner permanently sterile or medically unable to become pregnant. No history of clinically notable immunosuppressive or autoimmune disease. Individuals with HIV infection virologically suppressed for more than 1 year and with current CD4 count greater than 500 cells μl−1 were allowed into the study. Administration of a SARS-CoV-2 vaccine in the last 90 days or planning to have any standard-of-care vaccines within 14 days from the last administration of the study products. Comorbid conditions, including diabetes, hypertension, asthma and any cardiovascular disease. Positive serological test for hepatitis B surface antigen or any potentially communicable infectious disease as determined by the Principal Investigator or Medical Director. Positive serological test for hepatitis C (exception: successful treatment with confirmation of sustained virological response). Immunosuppressive illness, including hematological malignancy, history of solid organ or bone marrow transplantation. Current or anticipated treatment with tumor necrosis factor inhibitors (for example, infliximab, adalimumab, etanercept). Prior major surgery or any radiation therapy within 6 months of the first dose. Presence of a cardiac pacemaker or automatic implantable cardioverter defibrillator. Fewer than two acceptable sites available for intramuscular injection and EP (deltoid and anterolateral quadriceps), including sites with tattoos, keloids, scars within 2 cm of the injection site; implantable cardioverter defibrillator/pacemaker ipsilateral to the deltoid site (unless deemed acceptable by a cardiologist); or metal implants/medical devices at the EP site. Prisoner or participants who are compulsorily detained for treatment of a physical or psychiatric illness. Not willing to allow storage and future use of samples for SARS-CoV-2 virus-related research. Any illness or condition that, in the opinion of the investigator, may affect participant safety or endpoint evaluation. Known bleeding diathesis or use of blood thinners within 30 days before enrollment (low-dose aspirin (81 mg daily) acceptable). Known previous intolerance or contraindication to methylprednisolone (for participants in cohort D). Participants were recruited through the University of Pennsylvania Clinical Trials Unit using IRB-approved advertisements, outreach to prior research volunteers and investigator referrals. As enrollment was voluntary, potential self-selection bias is acknowledged; however, this is unlikely to affect the internal validity of the study given its primary objectives of assessing safety and pharmacokinetics in healthy adults. Initially designed as a five-cohort dose-escalation study, the protocol underwent multiple amendments in response to U.S. Food and Drug Administration (FDA) and IRB feedback, as well as emerging clinical and procedural data. The protocol was also temporarily amended to allow use of the ProFusion therapeutic needle after concerns about the original needle, which were subsequently resolved. Cohort D was revised to receive a reduced dose without methylprednisolone. As the study progressed, three additional cohorts (E, F and G) were added, expanding the total number to eight. Additional follow-up time points and flexibility in PK/ADA sampling were introduced to improve PK modeling. Based on the tolerability data, EP parameters were changed from OpBlock 0070 to OpBlock 0078 for all cohorts except cohort F. Participants received intramuscular injections of synthetic DNA plasmids formulated with HYLENEX Recombinant to enhance tissue permeability and improve the efficiency of in vivo transfection, in line with earlier reports17,18. The study followed a single ascending dose design using a modified 3 + 3 schema across eight cohorts (A1–G), with plasmid doses ranging from 0.25 mg to 2 mg and injection volumes from 0.25 to 1 ml. HYLENEX doses ranged from 34 to 135 units, adjusted in proportion to the plasmid dose and volume. Some cohorts received a single injection on day 0, while others followed a more intensive schedule with additional doses on days 3, 28 and 31 (see Fig. 2 and Extended Data Table 1 for cohort-specific details). Sixty-one healthy adults were screened; of these, 44 participants received at least one dose of the investigational DNA plasmids pAZD5396 or pAZD8076. One participant withdrew consent before completing screening (Fig. Study oversight was provided by an independent Data and Safety Monitoring Board (DSMB). Sequential enrollment began with cohort A1, where the first participant in each new cohort was dosed and monitored for 14 days before dosing the remaining participants. This waiting period was waived for cohorts D, F and G, where doses were comparable to or lower than those previously tested. Because of the higher dose administered in cohort E, despite lower expression levels observed in earlier cohorts compared to protein-based mAbs, the 14-day observation period was reinstated. In all cohorts, subsequent participants were dosed 3 days apart. If dose-limiting toxicity (DLT) occurred in the first three participants of a cohort, the DSMB conducted an ad hoc review to determine whether to proceed. If deemed unsafe, the remaining participants would have not been enrolled, although the next cohort would not open until 28 days of safety monitoring had been completed. Any dose level with more than one DLT among six participants would have been considered not tolerated. No DLT was observed in the trial. Each participant underwent a comprehensive baseline evaluation upon consent, including demographic and medical history, concomitant medication review and assessment of eligibility. EP was performed using the CELLECTRA 2000 device. All cohorts were treated using the OpBlock 0078 parameter set, except for cohort F, which used OpBlock 0070. Dosing strategy, plasmid formulation, HYLENEX concentration and the EP settings for each cohort are summarized in Fig. Safety was assessed in all participants who received at least one dose of the investigational product (n = 44). Participants were monitored for AEs using the Toxicity Grading Scale for Healthy Adult and Adolescent Volunteers Enrolled in Preventive Vaccine Clinical Trials28, with laboratory abnormalities evaluated based on site-specific reference ranges. Until the reporting cutoff date, AEs have been documented, including the severity of all related and unrelated AEs (systemic and local) by body system, descriptions of those AEs, elicited local reactions for the first 7 and 10 days after dosing, and the number of individuals who experienced at least one elicited local or systemic reaction in the first 7 days after dosing. Clinical safety evaluations included vital signs, physical examinations, 12-lead ECG, complete blood counts with differential, serum chemistries, HbA1c, coagulation parameters, serologies, creatine phosphokinase, urinalysis, pregnancy testing (when applicable), PK sampling, ADA assessments and a nasopharyngeal swab for SARS-CoV-2 (Fig. The primary safety evaluation focused on the occurrence, nature, timing, duration, intensity and relatedness of both injection site and systemic AEs elicited within 7 days of each administration. Events were categorized using the Medical Dictionary for Regulatory Activities by System Organ Class and Preferred Term. SAEs were collected throughout the study and characterized according to onset, outcome and relatedness to the investigational product. Injection site pain was assessed using a visual analog scale immediately, and at 5 and 10 min after injection/EP, after each dose. Injection sites (deltoid or quadriceps) varied according to participant preference. Local and systemic elicited AEs were actively monitored for 7 days after each dose. If a predefined safety stopping criterion was met, the study would be paused for review by the DSMB and study team. All safety data were summarized descriptively, with particular attention to potential dose-related trends. Horseradish peroxidase (HRP)-conjugated goat anti-human IgG (polyclonal) (1:8,000 dilution, cat. Anti-YTE monoclonal antibody (clone 23F7.1) used at 1–2 µg ml−1. All primary reagents were confirmed using ELISAs. The anti-YTE mAb demonstrated specific binding to YTE-containing AZD5396 and AZD8076 but not to control human IgG serum. For all ELISAs, recombinantly purified anti-AZD5396 and anti-AZD8076 antibodies were used as standards to confirm reagent specificity. All antibodies were validated either by the manufacturer, in prior peer-reviewed publications, or in-house during assay development. Full validation details and supporting references are provided in Reporting Summary. AZD5396 and AZD8076 were measured using separate ECL bridging immunoassays. For quantitation of AZD5396, 50 µl each of biotinylated anti-idiotypic antibody and SULFO-TAG (ruthenium)-labeled anti-YTE antibody (each at 1 µg ml−1) were mixed in a 96-well polypropylene plate with 50 µl of study serum samples pre-diluted 1:5 in MSD Diluent 100 (Meso Scale Discovery). Plates were incubated on a shaker at 600 rpm for 1 h. In parallel, MSD Streptavidin Gold plates were blocked with 200 µl per well of StartingBlock blocking buffer (Thermo Fisher Scientific) for at least 30 min, then washed three times with PBS-Tween 20. Then, 100 µl of the reaction mix was transferred to the streptavidin plate and incubated for 30 min at 600 rpm. After three washes with PBS-Tween 20, 150 µl of 2× MSD Read Buffer T was added, and the plate was read on an MSD Sector S600 Imager. AZD5396 concentrations were interpolated from a standard curve using recombinant AZD1061 protein (range 1.23–300 ng ml−1 in 20% pooled normal human serum). The assay for AZD8076 followed the same procedure, except that biotinylated anti-YTE and SULFO-TAG-labeled anti-idiotypic antibodies for AZD8076 were used at 0.25 µg ml−1 each. The standard curve used recombinant AZD8895, which is identical in amino acid sequence to AZD8076 except for the YTE substitutions also present in AZD1061. Study serum samples were assessed for the presence of ADAs using a validated three-tiered ECL assay (screening, confirmation and titer), previously validated by PPD for tixagevimab (AZD8895) and cilgavimab (AZD1061), as described in ref. The Fc regions also include YTE mutations to extend the half-life, which have not been associated with notable ADA induction in prior clinical studies. For samples with signals below the cutoff points in screening or confirmatory assays, ADA titers were reported as less than 40 for AZD5396 and less than 80 for AZD8076, reflecting the minimum required serum dilution for the respective assays. Preexisting antibodies against SARS-CoV-2 at study entry (day 0) were evaluated using an ELISA; 96-well plates were coated overnight at 4 °C with 1 µg ml−1 SARS-CoV-2 RBD proteins from ancestral (cat. The following day, plates were washed with PBS + 0.05% Tween 20, blocked with Blocker Casein in PBS (cat. 37528, Thermo Fisher Scientific) for 90 min at room temperature, then incubated for 1 h with participant sera serially diluted fourfold in blocking buffer. After washing, HRP-conjugated goat anti-human IgG (1:8,000 dilution) was added for 30 min at room temperature. Plates were developed with 1-step Turbo TMB (cat. 34022, Thermo Fisher Scientific), stopped with 2N H2SO4 and read at 450/570 nm using a BioTek Synergy Neo2 plate reader (Agilent Technologies). Endpoint titers were determined using the method by Frey et al.30 and analyzed using Excel and Prism v.10.2.1 (GraphPad Software). Because all participants had preexisting anti-Spike RBD antibodies, a modified capture ELISA was performed to detect DMAbs produced in vivo; 96-well flat-bottom half-area plates (Corning) were coated overnight at 4 °C with 2 µg ml−1 anti-YTE mAb (clone 23F7.1). Plates were washed with PBS-Tween 20 and blocked with 5% nonfat dry milk + 0.2% Tween 20 in PBS for 1 h at room temperature. Sera were diluted 1:4 and serially in twofold steps in 1% newborn calf serum with 0.2% Tween 20 in PBS, and incubated on the coated plates for 2 h at room temperature. SPD-C82EW, ACRO Biosystems) variants were used for detection. After incubation, plates were washed and incubated with HRP-conjugated streptavidin (1:10,000, cat. N100, Thermo Fisher Scientific) developed with Ultra TMB (cat. 34028, Thermo Fisher Scientific), quenched with 2N H2SO4 and read at 450/570 nm on a BioTek Synergy 2 plate reader. Data were exported to Excel, analyzed using Prism v.10 and reported as AUC over time. HEK 293T cells transformed with SV40 large T antigen (CRL-3216, ATCC) were used for producing the SARS-CoV-2 Spike pseudovirus. Cells were maintained according to the supplier's recommendations in DMEM supplemented with 10% FCS and 1% penicillin-streptomycin. All experiments involving cell lines were conducted under approved institutional biosafety protocols. Pseudoviruses bearing the SARS-CoV-2 Spike protein (USA-WA1/2020 isolate) were produced by cotransfecting HEK 293T cells with a 1:1 ratio of pNL4-3.Luc.R-E- plasmid (NIH AIDS Reagent Program) and various Spike-expressing plasmids (GenScript) using GeneJammer (Agilent Technologies). After 48 h, the viral supernatants were collected and supplemented with FCS to a final concentration of 12%. VCeL-Wyb019, Creative Biolabs) were plated at 10,000 cells per well in 96-well plates in D10 medium (DMEM + 10% FCS + penicillin-streptomycin), then incubated overnight. The next day, heat-inactivated serum or purified DMAb samples were serially diluted and incubated with pseudovirus for 90 min at room temperature before transfer to huCHOAce2 cells. Plates were incubated for 72 h, then lysed using britelite plus (cat. 6066769, Revvity); luminescence was read using a BioTek Synergy 2 plate reader. Neutralization curves were fitted using Prism v.10 (nonlinear regression with Hill slope <0). Minimal infective dose (ID50) values were calculated as the reciprocal dilution yielding 50% neutralization. DMAbs were purified from serum using anti-YTE-coated Dynabeads (MyOne Tosylactivated, cat. Day 0 or pooled week 24–52 sera were diluted 1:4 in PBS and incubated with anti-YTE Dynabeads overnight at 37 °C. Bead complexes were washed and eluted using Pierce IgG Elution Buffer (cat. For cohorts A1, A2 and D, samples were concentrated using Amicon 30 kDa MWCO (Merck) filters. DMAb concentration was confirmed using ELISA with anti-YTE capture and secondary anti-idiotype mAbs (M54-2E9 for AZD5396, M16H5.1 for AZD8076). Plates were coated overnight with 1 µg ml−1 anti-YTE or 5 µg ml−1 anti-idiotype mAbs, washed, developed using the same protocol as RBD ELISAs and read on a BioTek Synergy Neo2 reader. Data were analyzed in Gen5 and graphed using Prism v.10.2.1. This phase 1 study was not powered for hypothesis testing. Cohort sizes were determined based on prespecified DLT monitoring parameters, consistent with FDA guidance for first-in-human clinical trials. Participants were enrolled in cohorts of n = 3, 5 or 6 depending on dose level, with a target DLT threshold of 30% used to guide cohort expansion or modification. Statistical analyses were performed using descriptive methods. AEs were tabulated according to dose level, system organ class and preferred term, with frequencies, percentages and exact 90% Clopper–Pearson CIs calculated. Laboratory values, vital signs and other continuous measures were summarized using the mean, s.d., median and range, while PK endpoints, including detection rates and time to a 50% decline from peak concentration, were estimated with corresponding CIs. Disposition, including enrollment, dose administration, study completion and discontinuations, was also described. Interim analyses were conducted in response to emerging safety events; missing data were managed using participant replacement per investigator discretion. No formal power analysis was performed because the study focused on estimation rather than hypothesis testing. All statistical analyses were performed using STATA v.16 (StataCorp). Prism v.10.0 was used for data analysis and graph generation. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All requests for raw and analyzed clinical trial data and materials will be promptly reviewed by the corresponding authors (P.T. to determine if they are subject to intellectual property or confidentiality obligations, particularly regarding planned PK modeling analyses to be reported in a subsequent publication. Any data and materials that can be shared will be released via a material transfer agreement (requested from D.B.W. Sequences for the plasmids pAZD5396 and pAZD8076 have been submitted under patent no. Plasmids can be shared on completion of a material transfer agreement with the corresponding authors. Polack, F. P. et al. Safety and efficacy of the BNT162b2 mRNA Covid-19 vaccine. Baden, L. R. et al. Efficacy and safety of the mRNA-1273 SARS-CoV-2 vaccine. Lee, A. et al. Efficacy of covid-19 vaccines in immunocompromised patients: systematic review and meta-analysis. Dougan, M. et al. A randomized, placebo-controlled clinical trial of bamlanivimab and etesevimab together in high-risk ambulatory patients with COVID-19 and validation of the prognostic value of persistently high viral load. Gupta, A. et al. Effect of sotrovimab on hospitalization or death among high-risk patients with mild to moderate COVID-19: a randomized clinical trial. Weinreich, D. M. et al. REGEN-COV antibody combination and outcomes in outpatients with Covid-19. Levin, M. J. et al. Intramuscular AZD7442 (tixagevimab–cilgavimab) for prevention of COVID-19. August, A. et al. A phase 1 trial of lipid-encapsulated mRNA encoding a monoclonal antibody with neutralizing activity against Chikungunya virus. Casazza, J. P. et al. Safety and tolerability of AAV8 delivery of a broadly neutralizing antibody in adults living with HIV: a phase 1, dose-escalation trial. Parzych, E. M. et al. DNA-delivered antibody cocktail exhibits improved pharmacokinetics and confers prophylactic protection against SARS-CoV-2. In vivo delivery of engineered synthetic DNA-encoded SARS-CoV-2 monoclonal antibodies for pre-exposure prophylaxis in non-human primates. Wang, J. H., Gessler, D. J., Zhan, W., Gallagher, T. L. & Gao, G. Adeno-associated virus as a delivery vector for gene therapy of human diseases. Lek, A. et al. Death after high-dose rAAV9 gene therapy in a patient with Duchenne's muscular dystrophy. Duan, D. Lethal immunotoxicity in high-dose systemic AAV therapy. Hollevoet, K. et al. Clinically relevant dosing and pharmacokinetics of DNA-encoded antibody therapeutics in a sheep model. Welles, H. C. et al. Vectored delivery of anti-SIV envelope targeting mAb via AAV8 protects rhesus macaques from repeated limiting dose intrarectal swarm SIVsmE660 challenge. Song, S., Yang, L., Trepicchio, W. L. & Wyant, T. Understanding the supersensitive anti-drug antibody assay: unexpected high anti-drug antibody incidence and its clinical relevance. Priddy, F. H. et al. Adeno-associated virus vectored immunoprophylaxis to prevent HIV in healthy adults: a phase 1 randomised controlled trial. Characterization of potent RSV neutralizing antibodies isolated from human memory B cells and identification of diverse RSV/hMPV cross-neutralizing epitopes. Robbiani, D. F. et al. Recurrent potent human neutralizing antibodies to Zika virus in Brazil and Mexico. Guidance for Industry: Toxicity Grading Scale for Healthy Adult and Adolescent Volunteers Enrolled in Preventive Vaccine Clinical Trials (US Department of Health and Human Services, US Food and Drug Administration, Center for Biologics Evaluation and Research, 2007). Forte-Soto, P. et al. Safety, tolerability and pharmacokinetics of half-life extended severe acute respiratory syndrome coronavirus 2 neutralizing monoclonal antibodies AZD7442 (tixagevimab-cilgavimab) in healthy adults. & Zurakowski, D. A statistically defined endpoint titer determination method for immunoassays. We thank The Wistar Institute Core facilities. This work was funded by the Defense Advanced Research Projects Agency and the Joint Program Executive Office (JPEO) for Chemical, Biological, Radiological and Nuclear Defense (CBRND) (award no. This project was supported by the JPEO-CBRND Joint Project Lead for Enabling Biotechnologies in collaboration with the Defense Health Agency COVID funding initiative for the phase 1, dose-escalation trial of the safety and pharmacokinetics of SARS-CoV-2 DMAb in healthy adults, The Wistar Institute, Inovio Pharmaceuticals, AstraZeneca and the University of Pennsylvania (HR0011-21-9-0001) for this effort. References to nonfederal entities do not constitute or imply Department of Defense or Department of the Army endorsement of any company or organization. Pablo Tebas, Amanda Baer, Maria Caturla, Chungdhak Tsang & Knashawn Morales Vaccine & Immunotherapy Center, The Wistar Institute, Philadelphia, PA, USA Ami Patel, Elizabeth M. Parzych, Sukanya Ghosh, Mansi Purwar, Nicole Bedanova, Daniel W. Kulp, Jesper Pallesen & David B. Weiner Joseph T. Agnes, Dinah Amante, Paul D. Fisher, Laurent Humeau & Trevor R. F. Smith Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The other authors developed and validated several assays. also contributed critical technical insight to assay design and interpretation. All authors contributed to data interpretation, reviewed the manuscript and approved the final version for submission. Correspondence to Pablo Tebas or David B. Weiner. has received grant funding, participates in industry collaborations, has received speaking honoraria and has received fees for consulting, including serving on scientific review committees and board series. also discloses the following paid associations with commercial partners: Geneos (advisory board), AstraZeneca (advisory board, speaker), Inovio (BOD, SRA, stock) and Sanofi (advisory board). are employees of Inovio Pharmaceuticals; as such, they receive salary and benefits, including ownership of stock and stock options. and D.B.W are listed as inventors on patent application no. WO2023064841A1 ‘Antibodies for use against SARS-COV2', published 20 April 2023, filed by The Wistar Institute. Plasmids are available upon request and completion of a material transfer agreement. The other authors declare no competing interests. Nature Medicine thanks Benjamin Goldman-Israelow, James Kublin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Saheli Sadanand, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. d, Mean serum concentrations (±SEM) for each dose group. d, Mean serum concentrations (±SEM) for each dose group. AUC data are shown for AZD5396 (left), AZD8076 (middle), and total DMAb (right). Bars represent mean concentrations (±SEM) for each dose group. Points indicate AUC values for individual participants. Serum DMAb concentrations (ng/mL) measured using qualified quantitative binding assays are shown for study participants receiving two doses of 0.5 mg of SARS-CoV-2 DMAb delivered using electroporation with a truncated pulse pattern (OpBlock 0070). Serum concentrations are shown for AZD5396 (left), AZD8076 (middle), and total DMAb (right). An anti-YTE antibody was used to capture DMAb from participant sera. Graphs are plotted as area under the curve (AUC x 1000) for individual participants and the group mean is shown in navy blue. An anti-YTE antibody was used to capture DMAb from participant sera. Binding was detected with biotinylated ancestral, Delta, BA.2, or BA.4/BA.5 Spike RBDs. Graphs are plotted as area under the curve (AUC x 1000) for individual participants and the group mean is shown in navy blue. A visual analog scale (0 to 10 cm) was used to evaluate pain following the electroporation procedure. Pain typically resolved within minutes of administration (panel a). No significant differences in pain intensity were observed with repeated administrations (panel b). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Safety and pharmacokinetics of SARS-CoV-2 DNA-encoded monoclonal antibodies in healthy adults: a phase 1 trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02856-w'>Integrated epigenetic and genetic programming of primary human T cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 09:11:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Targeted epigenetic engineering of gene expression in cell therapies would allow programming of desirable phenotypes without many of the challenges and safety risks associated with double-strand break-based genetic editing approaches. Here, we develop an all-RNA platform for efficient, durable and multiplexed epigenetic programming in primary human T cells, stably turning endogenous genes off or on using CRISPRoff and CRISPRon epigenetic editors. We achieve epigenetic programming of diverse targeted genomic elements without the need for sustained expression of CRISPR systems. CRISPRoff-mediated gene silencing is maintained through numerous cell divisions, T cell stimulations and in vivo adoptive transfer, avoiding cytotoxicity or chromosomal abnormalities inherent to multiplexed Cas9-mediated genome editing. Lastly, we successfully combined genetic and epigenetic engineering using orthogonal CRISPR Cas12a–dCas9 systems for targeted chimeric antigen receptor (CAR) knock-in and CRISPRoff silencing of therapeutically relevant genes to improve preclinical CAR-T cell-mediated in vivo tumor control and survival. Engineered T cells, containing transgenic T cell receptors (TCRs), chimeric antigen receptors (CARs) or other synthetic antigen receptors, are an emerging modality to treat cancer, autoimmunity and infectious diseases1,2,3,4. Although autologous CAR-T cells have been transformative for treating aggressive hematological malignancies, substantial advances are needed to achieve similar success in treating solid tumors and generating allogeneic cell therapies. Solid tumors demonstrate a number of challenges including immunosuppressive tumor microenvironments, physical barriers and T cell exhaustion that limit the responses of current therapies5,6,7. Allogeneic CAR-T cells must additionally overcome T cell rejection by the host immune system and toxicities such as graft-versus-host disease8. A number of studies have nominated genes that, in principle, can be manipulated to overcome these challenges; however, enacting these strategies in cell products remains a major challenge and will require clinically relevant, robust, nontoxic and multiplexed approaches9,10,11. CRISPR-based genome editing has become a predominant approach for engineering therapeutic T cell products. CRISPR–Cas9 can facilitate gene inactivation by introducing DNA double-strand breaks (DSBs) or stimulate precise genome editing through homology-directed repair (HDR)12. Base editing and prime editing can generate efficient point mutations or small insertions and deletions without introducing DSBs13,14. However, each of these approaches results in permanent changes to the genome and potential unintended chromosomal abnormalities induced by on-target or off-target genome editing15,16,17. An alternative strategy for modulating gene function is through programmable control of endogenous gene expression. We and others have shown that repurposed CRISPR proteins can perturb gene expression in T cells without the formation of DSBs or any permanent change to the genetic code18,19,20. Unfortunately, transcriptional editing approaches such as CRISPRi, CRISPRa and Cas13d require sustained expression of the CRISPR proteins to maintain control of gene expression, which largely precludes their use in therapeutic applications in T cell therapies because of immune recognition of the bacterial Cas proteins and rejection of the transplanted cells21,22,23. Recent work from our group and others has shown that heritable gene silencing can be achieved through transient expression of epigenetic effectors targeted to specific genomic loci24,25,26,27. In particular, we showed that CRISPRoff, an epigenetic editor protein composed of dCas9 fused to DNMT3A, DNMT3L and ZNF10 KRAB protein domains, can write an epigenetic silencing program that is persistent for over 450 cell divisions in HEK293T cells27 when delivered as plasmid DNA. We also showed that CRISPRoff gene silencing could be reversed by CRISPRon, an epigenetic editor consisting of dCas9 fused to a TET1 catalytic domain that enables targeted erasure of DNA methylation. Importantly, epigenetic editors such as CRISPRoff and CRISPRon do not require DNA damage for their mechanisms of action, thereby eliminating the reliance on specific DNA repair outcomes and the genotoxicity, cytotoxicity and chromosomal abnormalities associated with these pathways. CRISPRoff and CRISPRon epigenome editing (epi-editing) is, thus, in principle, highly multiplexable, making complex therapeutic gene programming efforts possible for a vast array of therapeutic applications. However, this requires efficient and durable epigenetic editing in therapeutically relevant cell types such as primary human T cells. Here, we develop an optimized, clinically compatible RNA-based epigenetic engineering platform for turning genes on and off in primary human T cells. We show that our T cell epi-editing platform is potent and durable and can be multiplexed, suggesting that this is a versatile approach for broad use in therapeutic applications. CRISPRoff gene silencing is highly specific to the intended targets, is effective in a wide range of therapeutically relevant genes and eliminates the cytotoxicity and chromosomal translocations observed with multiplexed Cas9 gene editing. Additionally, CRISPRon achieves targeted DNA demethylation of an endogenous enhancer, resulting in stable induction of FOXP3, a therapeutically relevant gene in human primary T cells. Lastly, we successfully couple targeted epigenome engineering with targeted CAR knock-in (KI) at the TRAC locus to generate epi-edited TRAC CAR-T cells with enhanced functionality ex vivo and in vivo in a preclinical model of adoptive cell therapy. To determine whether CRISPRoff can stably silence gene expression in human primary T cells, we designed a panel of seven mRNAs encoding the previously reported Streptococcus pyogenes-based CRISPRoff-V2.3 effector27. We compared the effects of three mRNA cap modifications (Cap1, ARCA and m7g), base modifications (1-Me ps-UTP) and two codon optimization algorithms27,28 (Supplementary Fig. The CD151 cell surface protein was selected for initial optimizations given the presence of a known CpG island (CGI) at its promoter and prior validation with CRISPRoff in HEK293T cells27. Each mRNA that incorporated base modifications had more CD151 knockdown (KD) than our unoptimized mRNA, which had no 1-Me ps-UTP substitution (Supplementary Fig. We also compared codon optimization algorithms (Supplementary Fig. All CRISPRoff mRNA variants demonstrated efficient CD151 KD at high mRNA concentrations, with complete silencing in 85–99% of cells and no observed cellular toxicity (Fig. However, dose titration showed significant differences across designs in mRNA potency for gene silencing. We, therefore, proceeded with CRISPRoff 7 mRNA (referred to as CRISPRoff hereafter) for all subsequent experiments. We then compared CRISPRoff, CRISPRi and Cas9 mRNA activity at four different Lonza 4D nucleofector pulse codes and at four different time points (0, 2, 5 and 12 days after activation) (Supplementary Fig. Multiple pulse codes performed well for CRISPRoff KD across time points, highlighting the flexibility of this mRNA electroporation approach. We decided to use DS137 for subsequent experiments in part because it has been used previously for mRNA electroporation into T cells29. CRISPRoff was sufficiently efficient that we did not need to use any drug selection or cell sorting to select for CRISPRoff-positive cells. CD151 expression was assessed using flow cytometry 5 days after electroporation. We modeled CD151 positivity as a function of dose and mRNA variant and then computed a P value for the difference between CRISPRoff 7 and the rest of the mRNA variants using the standard error (Methods). P values were then adjusted using the Benjamini–Hochberg procedure. b, Comparison of Cas9 (black), CRISPRi (red) or CRISPRoff (blue) mRNA KO or KD activity on CD151, CD55 and CD81 loci over a time course of 28 days after electroporation. c, Representative flow cytometry histogram plots of CD151 KD (or KO) by CRISPRoff, CRISPRi or Cas9 on day 5 and day 28 after electroporation. d,e, Transcriptomic assessment by RNA-seq of CRISPRoff activity and specificity upon silencing of CD55 (d) or CD81 (e) relative to NTC. Cells were harvested 28 days after electroporation for RNA extraction. Yellow dots indicate significantly downregulated DEGs and gray dots have no significance (empirical Bayes moderated statistics with Benjamini–Hochberg FDR control, adjusted P < 0.05; CD55 adjusted P = 2.01 × 10−5 and CD81 adjusted P = 1.65 × 10−4; n = 2 donors). f, Comparison of CpG methylation analyzed by WGBS within a 20-kb window centered on the CD55 TSS. CGIs are depicted in green and the sgRNA targeting site is annotated. Tracks represent samples electroporated with CRISPRoff mRNA and an sgRNA targeting the CD55 TSS or NTC for two independent donor replicates (D1 and D2). Cells were collected at 30 days after electroporation. g, The Manhattan plot displays DMRs between cells treated with CRISPRoff and an sgRNA targeting CD55 or NTC and analyzed by WGBS (cells were collected at 30 days after electroporation). Red dots represent DMRs that gained DNA methylation in the targeting sgRNA samples. Blue dots represent DMRs that gained DNA methylation in NTC samples. h, Day 27 transcript levels of FAS, PTPN2, RC3H1 (Roquin 1) and SUV39H1 relative to NTC as measured by RT–qPCR (n = 3 donors). i, Transcriptomic assessment by RNA-seq of CRISPRoff activity upon silencing of FAS relative to NTC. Cells were electroporated with CRISPRoff mRNA and an sgRNA targeting FAS or an NTC and then harvested at 7 days after electroporation for RNA extraction. The yellow dot indicates the target gene, which is significantly downregulated, and gray dots have no significance (empirical Bayes moderated statistics with Benjamini–Hochberg FDR control, adjusted P < 0.05; FAS adjusted P = 3.35 × 10−20; n = 4 donors). We next examined whether CRISPRoff could initiate and maintain programmable gene silencing in primary human T cells at multiple endogenous gene targets over many cell divisions. We designed experiments to compare CRISPRoff, Cas9 and CRISPRi activity across time when delivered as mRNA to primary human T cells. We selected CD151, CD55 and CD81 for targeting as all three of these target genes contain known CGIs and are not essential for cell proliferation or survival of T cells in vitro30. We previously showed that CRISPRoff and CRISPRi have equivalent design and targeting rules for optimal single guide RNA (sgRNA) activity27. Hereafter, throughout this study, for all CRISPRoff and CRISPRi experiments, we used the top predicted 1–6 sgRNAs31 without prior validation in T cells. For both CRISPRoff and CRISPRi experiments, we coelectroporated a pool of three sgRNAs targeting within a 250-bp region immediately downstream of the transcription start site (TSS) of each gene or a nontargeting control sgRNA (NTC) along with CRISPRoff or CRISPRi mRNA. For Cas9 experiments, we electroporated one sgRNA predicted for optimal knockout (KO) activity or an NTC along with Cas9 mRNA32,33. Cell surface levels of each targeted gene's protein product were monitored by flow cytometry over a time course of 28 days. As expected, CRISPRi targeting led to transient gene silencing that was progressively lost over time, notably upon T cell restimulation using anti-CD2/CD3/CD28 soluble antibodies on day 9 (Fig. In contrast, CRISPRoff programmed durable gene silencing that was comparable to Cas9 KO for at least 28 days after electroporation with absence of cell surface expression in over 93% of cells for each gene target (Fig. Notably, CRISPRoff silencing persisted through three anti-CD2/CD3/CD28 soluble antibody restimulations over a 28-day time course, demonstrating that CRISPRoff gene silencing memory is stably propagated across approximately 30–80 cell divisions in vitro (Methods). RNA sequencing (RNA-seq) confirmed that CRISPRoff gene silencing was highly specific, with robust repression of the CD55 or CD81 target gene and no other differentially expressed genes (DEGs) at 28 days after electroporation (Fig. Whole-genome bisulfite sequencing (WGBS) further confirmed the specificity of DNA methylation deposited at the target locus by CRISPRoff; the highest differentially methylated region (DMR) between targeting samples and NTC samples occurred at the CD55 TSS (Fig. We then tested the ability to silence therapeutically relevant genes with CGIs that are known to modulate T cell signaling or adoptive T cell function including FAS (ref. For FAS, PTPN2, RC3H1 and SUV39H1, we electroporated CRISPRoff mRNA along with the top six predicted sgRNAs for each gene in an arrayed format alongside an NTC and then maintained cells in vitro for up to 27 days after electroporation, with restimulation using anti-CD2/CD3/CD28 soluble antibodies every 9–10 days. We also targeted each gene for KO using Cas9 as described above. Cell pellets were collected for RNA or DNA extraction and bulk RNA-seq (on day 7 after electroporation), indel analysis (on day 7 after electroporation) or qPCR (on day 27 after electroporation) was performed to measure target gene silencing or KO. We found that, for each gene, at least one and generally multiple sgRNAs could potently and durably mediate CRISPRoff silencing of target gene expression (Fig. Cas9 gene KO was also efficient (Supplementary Fig. RNA-seq analysis enabled us to profile the biological consequences of silencing or KO of FAS, MED12, PTPN2, RC3H1, SUV39H1 and RASA2 and examine the specificity of CRISPRoff gene silencing (Fig. The only genes that decreased upon SUV39H1 targeting were SUV39H1 and LINC02446; LINC02446 reduction is almost certainly a biological secondary effect of SUV39H1 ablation because it was also decreased by SUV39H1 KO with Cas9 and an independent sgRNA. PTPN2 targeting also had relatively specific effects on the transcriptome with a modest number of additional downregulated genes, which could be either off-target effects or secondary effects of target KD. RASA2 KD or MED12 KD (targeted with a pool of three sgRNAs) had broader effects on the transcriptome. For MED12, to further examine CRISPRoff specificity and to determine whether observed DEGs besides the target gene were secondary transcriptional effects or potential off-targets of CRISPRoff, we compared CRISPRoff RNA-seq results to Cas9 KO RNA-seq results (Supplementary Fig. For example, top downregulated DEGs include KLF2, CCR7 and IL7R, which are all expected biological secondary effects of MED12 loss on the basis of past observations from our group and others38,40 (Supplementary Fig. To further investigate CRISPRoff specificity, we analyzed gene expression changes for neighboring genes within a 100-kb window around each target gene (to look for on-target, off-gene effects). We also examined gene expression changes for predicted off-target sgRNA-binding sites (https://www.idtdna.com/site/order/designtool/index/CRISPR_SEQUENCE) within ±1 kb of a gene's TSS, according to previously established rules for CRISPRoff activity and specificity27. Only one gene across 151 putative off-target sites for all genes targeted showed evidence of potential CRISPRoff off-target activity (Supplementary Fig. Specifically, in RASA2-KD samples, we observed decreased expression of LARP1B. However, further work would be necessary to determine whether LARP1B is a true off-target or an indirect effect associated with RASA2 KD. In summary, CRISPRoff gene silencing proved programmable, efficient, specific and durable at many endogenous genes in primary human T cells. In addition to its activity at CGIs, CRISPRoff was previously shown in HEK293T cells to allow durable silencing (in a DNA methylation-dependent manner) of genes lacking a CGI27. To explore whether this biology extends to primary T cells, we attempted to use CRISPRoff for durable silencing of five genes lacking CGIs: CD5, LAG3, PDCD1, ENTPD1 (CD39) and PTPRC (CD45). Each of these genes encodes a cell surface protein that is important for sensing external stimuli41, signaling42,43,44 and function45,46 in T cells. We first electroporated CRISPRoff mRNA and an individual sgRNA or a pool of three sgRNAs targeting the TSS of CD5, LAG3, PDCD1 or CD39 or a single NTC sgRNA. As a control, we also electroporated Cas9 mRNA and one KO sgRNA targeting each of these genes or an NTC sgRNA. We then measured cell surface levels of each target protein at time points up to 35 days after electroporation. For each time point, cells were restimulated using anti-CD2/CD3/CD28 soluble antibodies and cell surface expression was measured 24 h later using flow cytometry. For the non-CGI genes targeted, we observed stable, partially stable or unstable silencing over the course of ~30 days. For CD5 and LAG3, CRISPRoff silencing was comparable to or even more efficient than Cas9 KO. At 30 days after electroporation, the CRISPRoff pooled sgRNA conditions for CD5 and LAG3 remained up to 99.5% and 99.1% silenced, respectively (Fig. While we observed some differences between the efficiency of PD1 silencing using CRISPRoff between CD4+ and CD8+ T cells, PD1 remained stably silenced out to 30 days after electroporation in most cells across of a bulk population (78.15% PD1-KD cells versus ~90% PD1-KO cells) (Fig. CD39 exhibited partially stable silencing, with a fraction of cells regaining CD39 expression with time, although most cells (53%) remained CD39− negative compared to the NTC on day 35 (Fig. We then evaluated CRISPRoff or CRISPRi mRNA with a single sgRNA targeting the CD45 TSS or an NTC sgRNA in comparison to Cas9 mRNA with a single sgRNA targeting protein coding exon 2 or an NTC sgRNA. Initially, CRISPRoff, CRISPRi and Cas9 all showed efficient ablation of CD45 (~99% for KD and ~85% for KO); however, by 7 days after electroporation, both CRISPRoff and CRISPRi effects diminished until reaching the levels of the NTC by 24 days after electroporation (Fig. Taken together, we observed a range of how effectively and durably non-CGI genes can be silenced and found that some genes with particularly low levels of CpG dinucleotides around the TSS remain challenging to stably silence (Fig. Further work is needed to elucidate rules for governing stable silencing at non-CGI genes in primary human T cells. Nonetheless, we demonstrate that expression from CGI and non-CGI genes can be stably silenced in primary human T cells through transient delivery of CRISPRoff. a, Comparison of Cas9 KO (black) or CRISPRoff KD activity with either a single sgRNA (light blue) or a pool of three sgRNAs (dark blue) at CD5 over a time course of 30 days after electroporation (n = 2 donors). b, Representative flow cytometry histogram plots of CD5 KD (blue) or KO (black) compared to NTC (gray) at 30 days after electroporation. c, Comparison of Cas9 KO (black) or CRISPRoff KD activity with either a single sgRNA (light purple) or a pool of three sgRNAs (dark purple) at LAG3 over a time course of 31 days after electroporation (n = 2 donors). d, Representative flow cytometry histogram plots of LAG3 KD (purple) or KO (black) compared to NTC (gray) at 31 days after electroporation. e, Comparison of Cas9 KO (black) or CRISPRoff KD activity with either a single sgRNA (light green) or a pool of three sgRNAs (dark green) at PDCD1 over a time course of 30 days after electroporation (n = 2 donors). f, Representative flow cytometry histogram plots of PD1 KD (green) or KO (black) compared to NTC (gray) on day 31 after electroporation. g, Comparison of Cas9 KO (black) or CRISPRoff KD activity with either a single sgRNA (light pink) or a pool of three sgRNAs (dark pink) at CD39 over a time course of 35 days after electroporation (n = 2 donors). h, Representative flow cytometry histogram plots of CD39 KD (pink) or KO (black) compared to NTC (gray) at 31 days after electroporation. i, Comparison of Cas9 KO (black), CRISPRi (gold) or CRISPRoff activity with a single sgRNA (orange) targeting CD45 over a time course of 24 days after electroporation (n = 2 donors). j, Representative flow cytometry histogram plots of KO (black), CRISPRi KD (gold) or CRISPRoff KD (orange) compared to NTC (gray) at 24 days after electroporation. In a,c,e,g,i, black arrows along the x axis indicate restimulations with anti-CD2/CD3/CD28 soluble antibodies. k, Fold change in surface protein expression (% positive cells) from the first time point to the last time point for cells treated with CRISPRoff mRNA and the most potent guide targeting the TSS of each respective gene. Red dots indicate genes with CGI annotations in the UCSC genome browser. Black dots indicate genes that do not have a CGI annotation. Epi-editing can modulate gene expression without inducing DSBs, in contrast to Cas9 nuclease targeting. This feature could offer important advantages in the context of multiplexed gene targeting approaches, as genome editing with nuclease-active Cas9 can result in translocations or chromosomal loss, which both have potential to be detrimental to cell proliferation, cell survival and perhaps the safety of therapeutic cell products9,47,48,49. To explore this approach, we simultaneously targeted sets of three, four or five nonessential genes with CRISPRoff or nuclease-active Cas9. Targeting multiple genes with nuclease-active Cas9 resulted in substantial cellular toxicity in human T cells, which may be attributed to the multiple DSBs generated by this approach (Fig. In contrast, targeting three, four or five genes for silencing with CRISPRoff resulted in minimal to no observable cellular toxicity compared to electroporation alone at either a high or low dose of mRNA (Fig. CRISPRoff multiplexed epi-editing averaged across four donors was efficient and durable out to 30 days after electroporation with combined silencing of three, four and five target genes at 93.5%, 82.4% and 65.8%, respectively (Fig. For some of these multiplexed gene combinations, silencing was marginally improved by increasing the dose CRISPRoff mRNA (Supplementary Fig. For these multiplexing experiments, we confirmed efficient CRISPRoff gene silencing for each gene individually with greater than 95% KD when targeting one sgRNA to the TSS (Supplementary Fig. We also achieved efficient, multiplexed gene silencing of three potentially therapeutically relevant genes (FAS, RC3H1 and SUV39H1), knocking down each target transcript by ~80% (Fig. 3d).We anticipate that durable multiplexing silencing could be further improved by empirically testing individual sgRNAs targeting genes of interest at lower doses to identify sgRNAs with optimized potency for use in multiplexed combinations. a, Graph showing the number of live T cells following cell editing by CRISPRoff and Cas9 when targeting either three, four or five genes simultaneously as compared to an empty electroporation (EE) control. Live-cell counts were measured 5 days after electroporation (n = 4 donors; mean ± s.e.m. The percentage of cells with all genes silenced was calculated from flow cytometry analysis (n = 4 donors; mean ± s.e.m.). c, A representative flow plot of cells targeted for triple-gene (CD151, CD81 and CD44) silencing (top) or NTC (bottom). Cells were analyzed at 30 days after electroporation. d, An RNA-seq log2 CPM (normalized counts per million) plot showing triple-target-gene KD in cells electroporated with CRISPRoff mRNA and sgRNAs targeting FAS, RC3H1 and SUV39H1 or an NTC sgRNA. Cells were collected for RNA-seq 7 days after electroporation (n = 4 donors; mean ± s.e.m.). e, Pie charts depicting the outcomes of three-gene, four-gene or five-gene silencing or KO shown in b. Epi-editing would be further enabled by technology to stably activate target chromatin sites, in addition to the silencing capability demonstrated above. We previously developed CRISPRon, a deactivated Cas9 enzyme fused to the TET1 DNA demethylase catalytic domain, to remove DNA methylation from a targeted locus and induce gene expression27. However, this prior work with CRISPRon was performed in immortalized cell lines and only demonstrated reactivation of genes through demethylation of TSSs that were previously silenced by CRISPRoff. Here, we set out to extend the capabilities in three important ways: (1) enable use in primary cells; (2) demonstrate activation of a genomic element that is naturally DNA methylated and epigenetically silenced; and (3) extend the use of epi-editing from TSSs to an enhancer element. We generated three variants of CRISPRon mRNA (TETv3, TETv4 and TETv5), which differed in the linker length used between TET1 and dCas9, with the cap structure and base modifications we optimized for CRISPRoff (Fig. To test whether CRISPRon could modulate clinically relevant gene expression in primary human T cells, we turned toward the FOXP3 (forkhead box P3) locus. FOXP3 is a transcription factor necessary for establishing immune-suppressive regulatory T cells (Tregs). Tregs are essential for immune homeostasis through establishing tolerance against self-antigens, limiting inflammation and aiding in tissue repair. Tregs are usually defined by high and sustained FOXP3 expression, whereas conventional CD4+ T cells (Tconvs) only transiently express FOXP3 upon activation. The difference in FOXP3 expression between cell types can be partly attributed to an intronic enhancer that harbors the Treg-specific demethylated region (TSDR), which is completely demethylated in Tregs but remains highly methylated in Tconvs50 (Fig. We hypothesized that targeting the TSDR in Tconvs with CRISPRon could remove endogenous repressive DNA methylation at this regulatory element, resulting in constitutive FOXP3 expression51. We isolated CD4 + CD25low Tconvs from two to four human donors and stimulated cells with anti-CD2/CD3/CD28 soluble antibodies. At 2 days after stimulation, we electroporated CRISPRon-TETv3 and five individual sgRNAs targeting the TSDR, FOXP3 TSS or AAVS1 safe harbor locus as a control (Supplementary Fig. We then measured FOXP3 expression at 9 days after initial stimulation, at which point Tconvs have entered a resting state and, thus, should express low amounts of FOXP3. Only sgRNAs targeting the TSDR increased FOXP3 expression, while targeting the TSS did not increase FOXP3 relative to control sgRNAs (Supplementary Fig. At 48 h after restimulation, when control Tconvs transiently express FOXP3, cells treated with CRISPRon-TETv3 targeting the TSDR expressed higher levels of FOXP3 than AAVS1-targeted control cells (Supplementary Fig. We then took the top three performing guides targeting the TSDR from this initial experiment (guide 1, guide 3 and guide 4) and tested them either as individual sgRNAs or as pools of two or three sgRNAs with each CRISPRon mRNA variant. We observed that targeting CRISPRon to the FOXP3 TSDR resulted in an increased fraction of FOXP3+ cells across multiple donors, CRISPRon designs and sgRNA number relative to the CRISPRon AAVS1 controls (Fig. Each bar in the heat map represents an individual CpG. c, Percentage of FOXP3+ Tconv cells after epi-editing with CRISPRon-TETv3 mRNA and 1–3 sgRNAs targeting the TSDR (blue) or AAVS1 control region (gray) as measured by flow cytometry at 9 days after electroporation (n = 4 donors per condition; mean ± s.e.m. d, Representative flow plots from day 9 after electroporation depicting FOXP3 expression after epi-editing with CRISPRon-TETv3 targeting the TSDR with a pool of three sgRNAs or an AAVS1 control. e, Percentage of FOXP3+ Tconv cells after epi-editing with CRISPRon-TETv3 mRNA targeting the TSDR with 1–3 sgRNAs or an AAVS1 control as measured by flow cytometry at 28 days after initial activation (n = 4 donors per condition; mean ± s.e.m. f, Representative flow cytometry histograms of FOXP3 median fluorescence intensity for CRISPRon-TETv3 and a pool of three sgRNAs targeting the TSDR or an AAVS1 control at 28 days after electroporation. g, Mean percentage of methylation across all CpGs assayed per targeted region (TSDR or TSS). We maintained the TSDR targeting conditions and AAVS1 control cells in culture to assess the stability and persistence of FOXP3 expression over time, restimulating cells with anti-CD2/CD3/CD28 soluble antibodies every 9–11 days. On day 28 after initial stimulation, Tconvs were collected for flow cytometry and we observed that FOXP3 expression was stably upregulated in a population of cells over weeks in vitro (Fig. Targeted bisulfite sequencing confirmed reduced methylation at the TSDR in TSDR-targeted cells even though the bulk bisulfite sequencing was performed on a heterogeneous population of cells, as we did not sort for FOXP3+ expression before bisulfite sequencing (Fig. 4g) nor did it change at other Treg associated genes (IL2RA and IKZF2) (Supplementary Fig. Our optimized CRISPRon results contrast with a previous effort to demethylate the TSDR locus in a targeted manner in Tconvs, which showed rapid remethylation of the locus at late time points in culture, even when FOXP3− negative clones were isolated51. Here, without sorting or clonal isolation, we were able to achieve a significantly increased fraction of FOXP3+ cells across donors after 28 days in culture with CRISPRon targeting the TSDR, as compared to the AAVS1 sgRNA control. These results establish CRISPRon as a powerful tool to control expression of an important endogenous gene expression through enhancer targeting with potential to be applied toward next-generation cell therapies. Having established a robust toolbox for epigenome engineering in primary T cells, we applied it to enhance immune cell therapy function in a preclinical model of cancer. We aimed to use CRISPRoff to enhance CAR-T cell function by simultaneous targeted genomic integration of a CAR (or other antigen receptor) transgene along with targeted epigenetic silencing using CRISPRoff. Targeted insertion of a CAR to the endogenous TCRα constant (TRAC) locus using CRISPR–Cas9 genome editing can enhance T cell potency by placing CAR expression under the regulated and dynamic control of the endogenous TCRα promoter, limiting exhaustion and dysfunction52. This approach offers potential functional, safety and cost benefits over current lentiviral and gammaretroviral transduction methods. Our rationale for combining this approach with CRISPRoff is based on the clinical observation that introduction of a CAR alone is insufficient to achieve durable responses or cures for most cancers. Our group and many others have identified additional genes that can be disrupted to further enhance CAR-T cell function in challenging tumor microenvironments, which we reasoned would be appropriate targets for epi-editing38,53,54,55. In particular, we discovered that RASA2 ablation promotes T cell function across a variety of immunosuppressive conditions, improving antigen sensitivity and durable effector function39. We reasoned that combining TRAC CAR KI with CRISPRoff-mediated silencing of additional targets could boost CAR-T cell function while avoiding translocations and other genotoxic events seen with prior multiplexed KO approaches56. We first explored an orthogonal Cas approach using Acidaminococcus sp. Cas12a (AsCas12a) ribonucleoproteins (RNPs) for targeted CAR KI in combination with stable epigenetic silencing of RASA2 using the S. pyogenes dCas9-based CRISPRoff system57,58. AsCas12a was precomplexed with a TRAC CRISPR RNA (crRNA) and coelectroporated with CRISPRoff mRNA and 1–3 sgRNAs targeting RASA2. Following electroporation, cells were transduced with an adeno-associated virus (AAV) HDR template (HDRT) containing a CD19-specific 28z CAR transgene flanked by TRAC locus homology arms, which serves as the HDR donor for KI (Fig. The addition of CRISPRoff mRNA and sgRNAs targeting RASA2 did not reduce CAR KI efficiency or yield (Fig. Likewise, CRISPRoff exhibited robust RASA2 silencing activity, similar in cells with or without integration of a CAR (Fig. In addition, we tested a fully nonviral approach for CAR KI using Cas9-target-site-modified single-stranded DNA (ssDNA) templates that were previously adapted for good manufacturing practice (GMP)59. We observed that using the same species of Cas9 and dCas9 for KI and KD resulted in translocations between TRAC and RASA2 and less efficient CAR KI, presumably because of sgRNA swapping that led to Cas9-mediated DSBs at both loci (Supplementary Fig. To address guide swapping, we tested truncated sgRNAs (16-bp protospacer) for CRISPRoff targeting RASA2 with the goal of retaining dCas9 binding and transcriptional control while eliminating Cas9 nuclease activity61. Truncated sgRNAs ameliorated RASA2:TRAC translocations (Supplementary Fig. 7a), retained efficient CAR KI (Supplementary Fig. 7b) and maintained silencing activity, albeit to a lesser extent than did full-length sgRNAs (Supplementary Fig. Taken together, we developed multiple approaches that could be made GMP-compatible that combine targeted CRISPR KI and programmable epigenome engineering. a, Schematic of a method for simultaneously generating epigenetically and genetically engineered CAR-T cells using Cas12a RNP for CAR KI. b, Graph comparing KI efficiency of CD19-specific CAR with no mRNA present or CRISPRoff mRNA used in combination with an NTC or sgRNA targeting RASA2. CRISPRoff was electroporated with either one sgRNA or a pool of three sgRNAs targeting the RASA2 TSS. Conditions noted as 0 sgRNA indicate an NTC (mean ± s.e.m. d, Cells with a TRAC CAR KI and RASA2 KD or NTC were collected at 7 days after electroporation for RT–qPCR. e,f, T cell immunophenotypes on day 7 based on CD45RA and CD62L expression, measured by flow cytometry and shown as either raw flow plots (e) or bar charts (f). g, Schematic of the repetitive stimulation assay to examine the functional efficacy of RASA2 epi-silenced CAR-T cells. h, Graphs show CAR-T cell cytotoxicity according to Incucyte analysis after five repetitive stimulations with target cancer cells. Each row represents an E:T ratio (top, 1:2; middle, 1:1; bottom, 2:1). i, Western blot for RASA2 expression in CAR-T cells that were treated with either one sgRNA or a pool of three sgRNAs targeting RASA2, which were isolated after the fifth repetitive stimulation. Tumor burden was monitored by BLI. The line is the mean and shaded areas depict the 95% confidence interval across replicates (n = 4–5 mice per group across four independent experiments and four donors, for a total of 23 mice per group; two-sided Mann–Whitney U-test: P = 0.0004). Individual experiments are shown in Supplementary Fig. 9. k, Survival of RASA2 epi-silenced CD19-CAR-T cells shown in i. Survival curves were compared using a log-rank test (P = 2.2 × 10−16). l, Representative images for the mice shown in k. We then tested the durability and functional effect of RASA2 silencing by CRISPRoff in CAR-T cells through an in vitro repetitive stimulation assay with RASA2 KD using a pool of three full-length sgRNAs or NTC. Experiments were performed with AsCas12a-based KI using an AAV HDRT given the more stable epigenome engineering observed with the full-length CRISPRoff sgRNAs. Most control CAR+ cells displayed an immunophenotype consistent with a T memory stem cell population at 7 days after electroporation on the basis of CD45RA and CD62L expression, although RASA2-KD cells shifted slightly to a more T effector-like population, consistent with previous reports39 (Fig. RASA2-silenced CAR-T cells were cocultured with CD19-expressing tumor cells at multiple effector-to-target (E:T) ratios repeatedly every 48 h (Fig. Consistent with previous reports, this repetitive stimulation assay caused control-edited CAR-T cells (treated with CRISPRoff and an NTC sgRNA) to decline progressively in their ability to control cancer cells by the last stimulation (Fig. 5h), consistent with the reported behavior of RASA2-KO CAR-T cells39. RASA2 remained stably silenced in RASA2-targeted CAR-T cells isolated after the last stimulation, confirmed by western blot (Fig. Nonviral Cas9-based KI cells that had RASA2 silenced with a pool of three truncated sgRNAs also performed better than control-edited cells in a repetitive stimulation assay (Supplementary Fig. We next examined the stability of CRISPRoff-induced silencing in CAR-T cells when transferred in vivo. As RASA2 silencing confers CAR-T cells with an in vivo fitness advantage over control-edited CAR-T cells, we instead chose to target CD151, which has no known role in T cell fitness in vivo. First, A375 melanoma cells engineered to express CD19 were engrafted in the flanks of NSG mice. Epi-edited T cells were engineered as previously described with an AsCas12a RNP precomplexed with a TRAC crRNA for CD19-CAR KI and CRISPRoff mRNA coelectroporated with a pool of three sgRNAs targeting the CD151 TSS or an NTC (Supplementary Fig. Epi-edited or control-edited CAR-T cells were cultured in vitro for 1 week after electroporation and then transferred in vivo through tail-vein injection 1 week after A375 engraftment. At 14 days after CAR-T cell transfer, tumors and spleens were isolated from mice and CAR-T cell CD151 expression was assessed by flow cytometry. Compared to NTC CAR-T cells, CD151 targeted CAR-T cells obtained from the tumor and spleen retained highly efficient CD151 KD, suggesting that CRISPRoff silencing is stable upon transfer in vivo and tumor-antigen recognition (Supplementary Fig. Lastly, we tested the functional efficacy of RASA2-silenced CAR-T cells in vivo. As described above, we generated RASA2-silenced TRAC CAR-T using AsCas12a KI with an AAV template and CRISPRoff with a pool of three full-length sgRNAs targeting RASA2 or a single NTC sgRNA (Supplementary Fig. NSG mice were injected intravenously with Nalm6 leukemia cells and, 4 days later, injected with RASA2-silenced TRAC CD19-28z CAR-T cells, control TRAC CD19-28z CAR-T cells (treated with CRISPRoff and a single NTC sgRNA) or TRAC-KO T cells through the tail vein (Supplementary Fig. Tumor burden was monitored by bioluminescence imaging (BLI) for ~40 days. We found that RASA2-silenced CAR-T cells had a significant advantage over NTC CAR-T cells at controlling tumor burden in vivo in cohorts of mice treated with cells from multiple human donors (Fig. Mice treated with RASA2-silenced TRAC CAR-T cells also had significantly extended survival relative NTC TRAC CAR-T cells (Fig. Taken together, these data suggest that epi-edited CAR-T cells can maintain stable target gene silencing even through multiple rounds of successful antigen-positive cancer cell killing, enabling functional enhancement of CAR-T cells through silencing of ‘checkpoint' genes without the need for multiplexed gene cleavage. We established an all-RNA CRISPR-based epigenetic editing platform for multiplexed primary human T cell programming. Previous work with CRISPRoff and related systems have demonstrated robust and stable epigenetic silencing in cell lines such as HEK293T cells24,26,27,62. While important for initial optimization, these cell lines demonstrate a variety of abnormalities such as endogenously low levels of the TET enzymes that reverse DNA methylation from CpG dinucleotides63. We now show that, in primary human T cells, which express high levels of TET2 and TET3 enzymes64, silencing of endogenous genes with and without a CGI is stable following only transient expression of CRISPRoff62. This approach is highly specific to the target loci and durable through multiple T cell activations, numerous cell divisions and transfer in vivo. It is also compatible with massive multiplexing, eliminating the cytotoxicity and genotoxicities observed with genome engineering using nuclease-active Cas9 or base-editing approaches65. Critically, because CRISPRon and CRISPRoff need to be delivered only transiently to exert stable effects, they circumvent the immunogenicity of constitutive Cas protein required for altering expression at the RNA level through CRISPRa, CRISPRi or RNA-targeting Cas species21,22,23. Transient delivery of CRISPRoff is critical to prevent host rejection of the ultimate cellular products. Here, we optimized mRNA delivery for CRISPRoff by combining cap structure, codon optimization and base modifications to substantially increase mRNA potency. Our approach is compatible with current electroporation-based manufacturing processes and the required GMP reagents and equipment. We expect that, for any CRISPR technology, there exists the risk of off-targets and this risk should be carefully evaluated for each unique sgRNA and gene target in any clinical program. Additionally, we developed an all-RNA platform for CRISPRon that can remove endogenous methylation from the TSDR of FOXP3. While prior studies with CRISPRon in HEK293T cells have reactivated genes that were previously silenced by CRISPRoff, here, we targeted a critical endogenously methylated noncoding region to establish stable de novo expression of FOXP3 in a CD4+ Tconv cell population over time. These data contrast with a previous attempt to deliver plasmid encoding dCas9 fused to TET1 and an sgRNA targeting the TSDR to primary human T cells, which resulted in rapid remethylation of the TSDR over time, even when FOXP3-expressing clones were isolated51. In this study, we only demonstrated CRISPRon activity at one enhancer; however, future efforts may focus on establishing the generalizability of this tool across diverse genomic elements. We anticipate that multiplexing with both CRISPRon and CRISPRoff will provide a foundation for systematic reprogramming of chromatin architecture in primary human cells. Lastly, we demonstrated durable silencing for a variety of clinically relevant T cell genes and developed two strategies compatible with clinical translation that combine CRISPRoff silencing of RASA2 with targeted TRAC locus CAR KI, using either truncated sgRNAs or orthogonal Cas species to circumvent the issue of guide swapping. There are many alternative therapeutic targets and a remaining question is how generalizable CRISPRoff-mediated gene silencing will be across different genomic loci with varying amounts of CpG dinucleotides. We show robust and durable silencing at promoter regions both with and without well-defined CGIs. However, silencing at two of the five non-CGI genes exhibited reduced stability. In addition, regulation of gene expression can be complex and driven by multiple regulatory elements in a cell-state-specific manner. Additional studies are needed to establish rules for stable silencing or activation across diverse genomic loci and cell states, as well as the requirements for CpG content and genomic context. We anticipate that large-scale functional genomics screens across promoters, enhancers and other regulatory regions will be enabled by this platform and could shed light on the rules governing stable versus metastable gene silencing. These studies can also provide important information for mapping and dissecting the functions of noncoding elements in the genome, which can lead to novel therapeutic strategies as with the context-specific enhancer targeted in therapies for sickle cell disease and β-thalassemia recently approved by the US Food and Drug Administration66. We expect that leveraging CRISPRoff and CRISPRon will offer insights into gene regulation, epigenetic landscapes and the intricacies of cellular differentiation. Moreover, leveraging these technologies in primary human cells paves the way for the next wave of advanced cellular products with finely tuned control of the epigenetic state to improve the potency, durability and safety of engineered cellular therapies. Human peripheral blood Leukopaks enriched for peripheral blood mononuclear cells (PBMCs) from deidentified healthy donors were purchased from StemCell Technologies (200-0092). CD3+ T cells were isolated using the EasySep Human T cell isolation kit (100-069) per the manufacturer-provided instructions. After isolation, T cells were seeded at 1 × 106 cells per ml and activated with anti-CD3/CD28 Dynabeads (Life Technologies, 40203D). T cells were maintained in culture at a density of 1 × 106 cells per ml throughout and cultured in complete X-VIVO-15 (cX-VIVO), consisting of X-VIVO 15 (Lonza Bioscience, 04-418Q) supplemented with 5% fetal calf serum (R&D systems, lot M19187), 5 ng mL−1 IL-7 and 5 ng mL−1 IL-15, unless otherwise indicated. CD4+CD25low Tconvs were isolated from washed PBMCs using the EasySep Human CD4+CD127lowCD25+ Treg isolation kit (StemCell Technologies, 18063) according to manufacturer's protocol. Tconvs were activated using Immunocult CD2/CD3/CD28 T cell activation reagent (StemCell Technologies, 10990) at 12.5 μl ml−1. Tconvs were maintained in culture in cX-VIVO supplemented with 300 IU per ml of IL-2 and passaged every 2–3 days. The CRISPRoff 143533B (CRISPRoff v7) mRNA design was used for in vitro transcription (IVT) to make mRNA in house using Cap1 (TriLink Biotechnologies #N-7113-5). For IVT reactions, plasmids containing the CRISPRoff-V2.3 codon-optimized sequence, CRISPRon-TETv3, CRISPRon-TETv4 or CRISPRon-TETv5 were cloned into a mutated T7 promoter plasmid as previously described67,68. IVT templates were produced by PCR amplification of CRISPRoff-V2.3 or CRISPRon variants with the forward primer correcting the T7 mutation and reverse primer appending a poly(A) tail, such that the final template contained the wild-type T7 promoter, 5′ untranslated region (UTR) including Kozak sequence, codon-optimized CRISPRoff-V2.3 coding sequence or CRISPRon variants, 3′ UTR and 145-bp poly(A) tail. The PCR product was purified using solid-phase reversible immobilization bead selection and stored at −20 °C until use. IVT reactions were performed with the HiScribe T7 high-yield RNA synthesis kit (New England Biolabs, E2040S) under full substitution of pseudo-UTP and in presence of 4 mM CleanCap AG (which encodes Cap1) (TriLink Biotechnologies, N-7113-5) with the addition of RNAse Inhibitor (New England Biolabs, M0314L) and yeast inorganic pyrophosphatase (New England Biolabs, M2403L). Transcribed mRNA was purified with lithium chloride and eluted in water. After quantification by NanoDrop spectrophotometer and normalization to 1 μg μl−1, mRNA product was assessed on an Agilent 4200 TapeStation system and subsequently stored at −80 °C. CleanCap Cas9 mRNA was purchased from TriLink (L-7606). For experiments using Cas9, CRISPRi or CRISPRoff mRNA, fresh CD3+ T cells were activated with a 1:1 bead-to-cell ratio with anti-CD3/CD28 Dynabeads (Life Technologies, 40203D) in the presence of 5 ng μl−1 IL-7 and 5 ng μl−1 IL-15 at 1 × 106 cells per ml. After 2 days of stimulation, T cells were magnetically debeaded, washed with PBS and resuspended in TheraPEAK P3 buffer with supplement (Lonza, G4LP3-126000) at 0.75 × 106 cells in 20 μl. Cas9, CRISPRi and CRISPRoff mRNA were added to 20 μl of cells at an equimolar ratio (1, 1.07 or 1.6 μg, respectively) with 2 μg of chemically modified sgRNA (Synthego) and cells were electroporated on a Lonza 4D Nucleofector using pulse code DS137. Immediately after electroporation, 80 μl of prewarmed cX-VIVO was added to each electroporation well and cells were incubated for 30 min in a CO2 incubator at 37 °C followed by the distribution of each electroporation reaction into three wells of a 96-well round-bottom plate. Each well was brought to 200 μl with cX-VIVO. Cells were maintained and expanded by the addition of cX-VIVO every 2 or 3 days and restimulated with ImmunoCult Human CD2/CD3/CD28 T cell activation reagent (StemCell Technologies, 10990) every 9–10 days at 6.25 μl ml−1. All sgRNA sequences used are listed in Supplementary Table 1. To evaluate CRISPRoff mRNA designs, we electroporated seven CRISPRoff mRNA designs across a range of doses along with an sgRNA targeting CD151. We then compared the CRISPRoff activity data across constructs using ordinary least square regression. We modeled CD151 expression as a function of dose and mRNA variant and then computed a P value for the difference between mRNA variants across all doses using the standard error. For experiments using CRISPRon mRNA, CD4+CD25low (Tconv) cells were isolated from PBMCs and activated using Immunocult CD2/CD3/CD28 T cell activation reagent (StemCell Technologies, 10990) at 12.5 μl ml−1. Then, 2 days after activation, Tconvs were electroporated with 1.6 μg of CRISPRon mRNA and 2 μg of chemically modified sgRNA (Synthego) with pulse code DS137 as described above. After electroporation, Tconv cells were maintained and expanded in cX-VIVO supplemented with 300 U per ml For all experiments with flow cytometry as a readout looking at cell surface markers, 0.5 × 105–1 × 105 cells per condition were transferred to a round-bottom 96-well plate, centrifuged, washed once with 200 μl of cell staining buffer and stained with antibodies (1:50 dilution) for 20 min at 4 °C in the dark (antibodies are listed in Supplementary Table 1). Samples were measured using an Attune NXT cytometer with a 96-well autosampler (Invitrogen) and analyzed using FlowJo version 10.9.0 unless otherwise stated. For experiments measuring PD1, LAG3 and CD39 surface expression over time, cells were stimulated with ImmunoCult Human CD2/CD3/CD28 T cell activation reagent (StemCell Technologies, 10990) at 6.25 μl ml−1 24 h before flow cytometry readout. To obtain comparable live-cell counts between conditions, events were recorded using a fixed volume for all samples. To determine the number of cell divisions in in vitro experiments over time, we plated 0.16 × 106 cells into 96-well round-bottom wells following electroporation. We then counted cells on an Attune NXT Cytometer every 48 h or at each passage time. For intracellular flow cytometry staining, 0.5 × 105–1 × 105 cells per condition were transferred to a 96-well V-bottom plate, centrifuged and washed once with 200 μl of staining buffer. Cells were resuspended in 30 μl of staining buffer containing Ghost Dye red 780 (Tonbo, 13-0865-T500) and antibodies targeting surface proteins of interest and stained for 20 min at 4 °C in the dark. After staining, cells were washed once with 170 μl of staining buffer and then resuspended in 50 µl of 1× FOXP3 Fix/Perm buffer (BioLegend, 421403) and incubated at room temperature for 30 min in the dark. After fixation, cells were permeabilized in 200 µl of 1× FOXP3 Perm buffer for 15 min at room temperature in the dark. After permeabilization, cells were spun down and washed once with 1× FOXP3 Perm buffer and then resuspended in 30 µl of 1× FOXP3 Perm buffer containing antibodies targeting intracellular proteins and stained in the dark at room temperature for 30 min. Following intracellular staining, cells were washed once with the addition of 170 µl of staining buffer and centrifuged at 300g for 5 min; the supernatant was removed. Cells were resuspended in 200 μl of staining buffer and then measured using the Attune NXT Cytometer with a 96-well autosampler. Human primary T cells were harvested 27 days after electroporation (CD55 and CD81) or 7 days after electroporation (FAS, PTPN2, RC3H1, SUV39H1 and MED12). A total of 1 × 106 cells were harvested per condition and RNA was isolated using a Quick-RNA MicroPrep Kit (Zymo, R1050). Isolated RNA was treated with TURBO DNase (Invitrogen, AM2239) and concentrated using an RNA clean and concentrate kit (Zymo, NC0622892). Library preparation was carried out using the QuantSeq FWD V2 with UDI Set A1 kit and UMI module (Lexogen, 191.96). Final libraries were assessed using a 4200 TapeStation (Agilent), quantified using the Qubit ds HS assay kit (Invitrogen) and sequenced as single-end 50-bp reads on a HiSeq 4000 (Illumina) or NextSeq 500 (Illumina). RNA-seq data were aligned and counts were generated using the RNA-seq pipeline of nf-core (version 3.18)69. Differential gene expression analysis was conducted using the limma-voom framework, with donor variation included as a covariate in the statistical model. Gene expression was normalized using the trimmed mean of M values method and lowly expressed genes were filtered before analysis. DEGs were identified by comparing treated samples to NTCs, with significance criteria set at an adjusted P value < 0.05 (empirical Bayes moderated statistics with Benjamini–Hochberg correction) and absolute log2 fold change > 1. Results were visualized using volcano plots displaying log2 fold change versus −log10(adjusted P value), with genes colored on the basis of significance thresholds or target gene. The MED12-KO RNA-seq data shown in Supplementary Fig. 2n were from previously generated data in the A.M. lab40 and are representative of CD4+ cells collected 8 days after activation with ImmunoCult Human CD2/CD3/CD28 T cell activation reagent (StemCell Technologies, 10990). Genotyping measured by NGS showed ~80% editing at MED12. The RNA-seq reads were analyzed as previously described and genes with a false discovery rate (FDR)-adjusted P value < 0.05 were considered significant. We correlated our MED12 CRISPRoff KD (76.5% KD) with this dataset to better match the degree of gene disruption as our MED12 Cas9 KO data only had ~55% indel-editing efficiency (Supplementary Fig. 2b) and we observed fewer DEGs than expected. Off-target predictions were generated through the Integrated DNA Technology (IDT) CRISPR–Cas9 guide RNA checker (https://www.idtdna.com/site/order/designtool/index/CRISPR_SEQUENCE) for both Cas9 KO and CRISPRoff KD sgRNAs. For CRISPRoff sgRNAs, predicted off-target loci were filtered for sites that fell within ±1 kb of a gene promoter. We also performed ‘on-target, off-gene' analyses by assessing effects on proximal genes that fell within a 100-kb window around the intended target. Only predicted off-target or proximal genes that had an absolute KD log fold change > 1 and adjusted P value < 0.05 were considered potential true off-target genes. We generated WGBS libraries for 12 samples, corresponding to EE (empty electroporation), NTC and targeting for CD55 across two donors, each done in technical replicate. Genomic DNA was extracted using the QIAamp DNA Mini Kit (Qiagen) and 250 ng of DNA was diluted to 2.27 ng μl−1 in 110 μl with 2 μl of 0.5% lambda DNA spike-in and sheared using a Covaris E220 evolution with intensifier for 50 s to an average length of ~500 bp. Sonicated DNA was recovered using the MinElute reaction cleanup kit (Qiagen), bisulfite conversion was performed using the EZ DNA methylation-Gold kit (Zymogen) and the resulting ssDNA was quantified on the Qubit ssDNA assay kit (Invitrogen). Library preparation was performed using the xGen methylation-sequencing DNA library preparation kit (IDT, 10009860) and xGen Normalase UDI primers plate 1 (IDT, 10009796). The prepared libraries were quantified on a 4200 TapeStation system (Agilent) and Qubit double-stranded DNA HS assay kit (Invitrogen). Libraries were sequenced using paired-end 150-bp reads on a NovaSeqX with a 10% PhiX spike-in to diversify the sample pools. Differential CpG DNA methylation analysis was performed using the methylKit R package70. CpG methylation data from Bismark coverage files was imported. To search for differentially methylated tiles, the ‘tileMethylCounts' function was used with options ‘win.size = 1000' and ‘step.size = 100'. DMRs were scored by the percentage methylation difference and q values were calculated using the ‘calculateDiffMeth' function with ‘overdispersion = MN' and ‘adjust = BH' options using the replicates as a covariate in fitting the model. Results were visualized as Manhattan plots to display −log10-transformed P values associated with individual methylation tiling windows. Statistically significant DMRs with FDR < 0.05 (Benjamini–Hochberg) were colored on the basis of their methylation status. To visualize the methylation status at individual loci in Integrative Genomics Viewer (IGV), the base-level methylation status was extracted from BedGraph files from the nf-methylseq pipeline. Then, results were converted into an IGV-friendly format and data were displayed as bar charts, in which methylated regions were considered as a methylation percentage of 50–100% shown in the range of 0.5 to 1 in red and unmethylated regions were considered as a methylation percentage of 0–50% shown in the range of −1 to −0.5 in blue. PBAT-seq analysis was conducted as previously described50. PBAT-seq tracks were visualized using a sliding binning strategy with a bin size of 1,500 and step size of 300 in ggplot2 (version 3.5.1). A total of 200,000 cells were collected for conditions coelectroporated with CRISPRon and Guide 3 targeting the TSDR or two AAVS1 control targets, spun down and frozen at −80 °C. Targeted bisulfite sequencing was conducted by EpigenDX at two sites across the FOXP3 locus (TSDR or TSS) and off-target sites including IL2RA and IKZF2. Cells were lysed and reverse-transcribed as described below. Briefly, 0.1 × 105–0.2 × 105 T cells were spun down in 96-well U-bottom plates and washed once with DPBS (without Ca2+ and Mg2+) (StemCell Technologies, 37350). Cells pellets were either frozen on dry ice and then stored at −80 °C until further use or lysed in 50 μl of complete RNA lysis buffer (9.6 mM Tris-HCl (pH 7.8), 3 U per ml proteinase K, 300 U per ml DNAse 1, 0.5 mM MgCl2, 0.44 mM CaCl2, 10 μM DTT and 0.1% (w/v) Triton X-114). Cells were incubated in RNA lysis buffer for 8 min at room temperature and then 30 μl of lysed cells were added to 3 μl of RNA stop solution in a new 96-well PCR plate (1 mM proteinase K inhibitor, 90 mM EGTA and 113 μM DTT in UltraPure water) and incubated for 3 min at room temperature to stop the lysis reaction. Then, 32 μl of RevertAid RT kit (Thermo Fisher, K1691) was aliquoted in a separate 96-well PCR plate and 8 μl of the lysis samples were added and mixed. Samples were either immediately used for qPCR or frozen and stored at −80 °C. A master mix was made using TaqMan Fast advanced master mix for qPCR (Thermo Fisher, 4444557) and primer probes (IDT) that targeted either the housekeeping gene, GAPDH, or a gene of interest (FAS, MED12, PTPN2, RASA2, RC3H1 or SUV39H1). Next, 15 μl of master mix was added to 9.6 μl of complementary DNA from the RT reaction above and qPCR was performed in 5-μl reactions with technical quadruplicates in a 384-well plate format using the QuantStudio real-time PCR system (Thermo Fisher). The fold change in gene expression was then calculated (2−∆∆Ct). T cells were electroporated as described above. In multiplexed conditions with either CRISPRoff or Cas9 mRNA, each gene targeted received 1.6 μg of sgRNA. Cells were prepared for flow cytometry to collect live-cell counts and cell surface protein expression as described above. FCS files were analyzed using FlowJo (version 10.9.0) to create a gating scheme; cells were gated on lymphocytes, then single cells and then live cells and a positive or negative gate was drawn for each target gene. To calculate the proportion of cells with a given number of knocked down genes, the FlowJo workspace was read into R using the ‘flowCore', ‘CytoML' and ‘openCyto' packages. First, each individual cell was recorded as either positive or negative for each target gene, with negative indicating KD of a target, on the basis of thresholds set in FlowJo. These thresholds were verified through visualization with the ‘ggcyto' R package. Each cell was then annotated with the total number of genes successfully knocked down, from zero to five target genes. Lastly, the proportion of all cells for each number of knocked down genes was calculated. For nonviral integration of a BCMA-specific CAR transgene at the TRAC locus, T cells were isolated and stimulated as described above. After 48 h of stimulation, cells were magnetically debeaded and prepared for electroporation. Briefly, to prepare the guide RNA targeting the TRAC locus, aliquots of crRNA and trans-activating crRNA (Edit-R, Dharmacon Horizon) were thawed and mixed 1:1 (v/v) and annealed by incubation at 37 °C for 30 min to form an 80 μM solution. ssDNAenh was mixed into the gRNA solution at a 0.8:1 volume ratio before adding 40 μM Cas9-NLS (Berkely QB3 MacroLab) at a 1:1 (v/v) to attain a molar ratio of sgRNA-Cas9 of 2:1. Final RNP mixtures were incubated at 37 °C for 15–30 min, after which 50 pmol of RNP was used for each electroporation. The TRAC-targeting Cas9-RNP was mixed with a 2,923-nt ssDNA HDRt encoding the BCMA-specific CAR transgene, incubated for 10–15 min and then mixed with cells that were resuspended in 20 μl of TheraPEAK P3 buffer with supplement (Lonza, G4LP3-126000). The CRISPRoff mRNA and sgRNA targeting RASA2 or NTC (Synthego) were added on top of the cells last and then cells were electroporated using the Lonza 4D Nucleofector with pulse code EH115. For any experiments incorporating an RNP in the electroporation and mRNA, we used pulse code EH115, as this code is the most effective for RNPs (demonstrated elsewhere) while still maintaining efficiency for mRNAs. For RASA2 silencing with CRISPRoff mRNA, we codelivered either a chemically modified full-length (20 nt) sgRNA or a chemically modified truncated sgRNA (17 nt with the last base pair mismatched to effectively make a 16-nt truncated guide) (Synthego). For integration of a CD19-specific CAR transgene at the TRAC locus using AAV6, Alt-R A.s. Cas12a (Cpf1) Ultra (IDT, 10001272) was mixed with a crRNA targeting the TRAC locus (IDT) at room temperature for 10–15 min. Cells that were resuspended in 20 μl of TheraPEAK P3 buffer with supplement (Lonza, G4LP3-126000) were mixed with TRAC-Cas12a-RNP and CRISPRoff mRNA and sgRNA targeting either RASA2 or an NTC were added on top of the cells and electroporated using pulse code EH115. At 30 min after electroporation, cells were transduced with AAV encoding the CD19-CAR as previously described52. The AAV-ITR plasmids containing the 1928z CAR transgene and TRAC-targeting homology arms for HDR was packaged into AAV6 by transfection of HEK293T cells together with pHelper and pAAV Rep-Cap plasmids using polyethylenimine. AAVs were further purified using iodixanol gradient ultracentrifugation. AAVs were tittered using qPCR on DNase I (New England Biolabs)-treated, proteinase K (Qiagen)-digested samples. qPCR was performed with SsoFast EvaGreen Supermix (BioRad, 1725201) on a StepOnePlus real-time PCR System (Applied Biosystems). AAV was added to the cells at a multiplicity of infection of 1 × 105 and cells were incubated overnight in serum-free medium. Then, 1 day after electroporation, the AAV-containing medium was removed and the edited T cells were resuspended in fresh cX-VIVO and expanded using standard culturing conditions. The KI efficiency for both nonviral-mediated HDRT and AAV HDRT KI was evaluated by flow cytometry several days later. Genomic DNA from 1 × 106–2 × 106 cells was purified using the QIAamp DNA mini kit (Qiagen) following the manufacturer's protocol. DNA was quantified using the NanoDrop One (Thermo Fisher Scientific). All DNA samples were digested with HindIII in 10× rCutSmart buffer (New England Biolabs) before the ddPCR. A ddPCR assay was designed to measure the occurrence of balanced translocations between TRAC and RASA2. The assays used a pair of primers targeting a balanced translocation with TRAC on the 5′ end and RASA2 on the 3′ end and a fluorescent FAM probe. A 20-µl PCR reaction was used to generate lipid droplets with an automated droplet generator (BioRad). Data analysis was conducted with the QX manager software (BioRad) and thresholds were set manually to obtain the number of positive droplets for each channel. For immunoblotting experiments, 2 × 106–3 × 106 cells were harvested, resuspended in 70 μl of Pierce radioimmunoprecipitation assay buffer (Thermo Fisher, 89901) supplemented with protease and phosphatase inhibitor cocktail (Fisher Scientific, 78440) and incubated at 4 °C for 40 min. Then, 15 μg of protein per sample was loaded onto 4–15% Tris–glycine SDS gels (BioRad) followed by transfer to PVDF membrane (BioRad) using the Biorad Trans-Blot transfer system. After transfer, membranes were blocked with 5% (w/v) nonfat milk in PBS containing 0.1% Tween-20 for 30 min. Primary antibody incubations were performed for either 2 h at room temperature or overnight at 4 °C (antibodies provided in Supplementary Table 1). For in vitro cytotoxicity assays, we generated epi-silenced CAR-T cells with either TRAC BCMA-specific CAR KI using our nonviral approach or TRAC CD19-specific CAR-T cells using AAV as described above. For coculture assays, we generated CD19+ or BCMA+ nuclear-localized RFP+ A375 melanoma target cells. At 6 days after electroporation, 300 of these target cells were seeded in 50 μl of complete RPMI per well in a 384-well plate. Complete RMPI includes RPMI (Gibco, 21870076), 10% fetal calf serum (R&D systems, lot M19187), 1% L-glutamine, 1% penicillin–streptomycin, 10 mM HEPES solution (Sigma, H0887) and 1 mM sodium pyruvate (Gibco, 11-360-070). The next morning, epi-silenced TRAC CD19-specific CAR-T cells or BCMA-CAR-T cells were counted and CAR expression was assessed by flow cytometry. CAR-T cell numbers were normalized and added to the target cells according to the indicated E:T cell ratios. Target cell counts were measured using the Incucyte live-cell imaging system (Sartorius) with imaging at 6-h intervals based on RFP expression. For repetitive stimulation assays, CD19-A375 or BCMA-A375 target cells were seeded in complete RPMI medium 1 day before coculture. The next day, half of the medium was replaced with cX-VIVO and CD19-CAR-T cells or BCMA-CAR-T cells were seeded on top of the target cells at a 1:1 E:T ratio. This was repeated every 48 h for up to 5–7 stimulations. For each coculture, CAR-T cells were collected, strained through a 70-μm filter and counted using an Attune NXT Cytometer (Invitrogen). CAR expression was assessed using flow cytometry before each repetitive stimulation to normalize CAR-T cell counts between conditions. Before using the CAR-T cells for any downstream assay, the T cells were collected, counted and purified using EasySep Release human CD45 positive selection kit (StemCell, 100-0105). All mice for animal experiments were housed and used in accordance with ethical guidelines approved by the University of California, San Francisco (UCSF) Institutional Animal Care and Use Committee (IACUC). To assess whether CRISPRoff-mediated silencing persists in CAR-T cells in vivo, we generated epi-edited Cas12a-compatible TRAC CD19-CAR-T cells in combination with CRISPRoff mRNA and a pool of three sgRNAs targeting CD151 or an NTC. Mice were injected with 1 × 106 A375 melanoma cells (engineered to express CD19) through subcutaneous injection to the right flank. Then, 1 week later, mice were randomized on the basis of width and length of the tumors and 7.5 × 105 epi-edited or control-edited CAR-T cells were injected into the tail vein. Mouse health and tumor growth were monitored over time. We generated Cas12a-compatible TRAC CD19-CAR-T cells treated with CRISPRoff mRNA and three guides targeting RASA2 or an NTC as described previously. CRISPRoff silencing activity of RASA2 in CAR-T cells was validated using western blot or RT–qPCR before injection. If RASA2 silencing was not observed in CD19-CAR-T cells before injection (because of electroporation error), we excluded those conditions from analysis. Tumor burden was monitored using BLI over time and weight was assessed as were any signs of morbidity per our UCSF IACUC protocol guidelines. For all experiments, mice were randomized on the basis of the BLI signal from day 3 after Nalm6 injection to ensure equal tumor distribution in each group before T cells were transferred. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. Code used in this paper is available through Github (https://github.com/GilbertLabUCSF/T_Cell_CRISPRoff/). June, C. H., O'Connor, R. S., Kawalekar, O. U., Ghassemi, S. & Milone, M. C. CAR T cell immunotherapy for human cancer. Rapoport, A. P. et al. NY-ESO-1-specific TCR-engineered T cells mediate sustained antigen-specific antitumor effects in myeloma. Roybal, K. T. et al. Engineering T cells with customized therapeutic response programs using synthetic Notch receptors. & Riley, J. L. CAR T cells for infection, autoimmunity and allotransplantation. Labanieh, L. & Mackall, C. L. CAR immune cells: design principles, resistance and the next generation. CAR T cell therapy in solid tumors: a review of current clinical trials. Albelda, S. M. CAR T cell therapy for patients with solid tumours: key lessons to learn and unlearn. Depil, S., Duchateau, P., Grupp, S. A., Mufti, G. & Poirot, L. ‘Off-the-shelf' allogeneic CAR T cells: development and challenges. & Lum, J. J. Metabolic engineering for optimized CAR-T cell therapy. Liu, X. et al. CRISPR–Cas9-mediated multiplex gene editing in CAR-T cells. Engineering the next-generation of CAR T-cells with CRISPR–Cas9 gene editing. Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Leibowitz, M. L. et al. Chromothripsis as an on-target consequence of CRISPR–Cas9 genome editing. Whole genomic analysis reveals atypical non-homologous off-target large structural variants induced by CRISPR–Cas9-mediated genome editing. Whole chromosome loss and genomic instability in mouse embryos after CRISPR–Cas9 genome editing. Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. & Qi, L. S. Beyond editing: repurposing CRISPR–Cas9 for precision genome regulation and interrogation. Tieu, V. et al. A versatile CRISPR–Cas13d platform for multiplexed transcriptomic regulation and metabolic engineering in primary human T cells. Identification of preexisting adaptive immunity to Cas9 proteins in humans. Li, A. et al. AAV-CRISPR gene editing is negated by pre-existing immunity to Cas9. E., Tan, S. X., Hoon, S. & Yeo, G. W. Pre-existing adaptive immunity to the RNA-editing enzyme Cas13d in humans. Amabile, A. et al. Inheritable silencing of endogenous genes by hit-and-run targeted epigenetic editing. O'Geen, H., et al. Ezh2-dCas9 and KRAB-dCas9 enable engineering of epigenetic memory in a context-dependent manner. & Bernstein, B. E. Epigenome editing strategies for the functional annotation of CTCF insulators. Nuñez, J. K. et al. Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing. Chu, S. H. et al. Rationally designed base editors for precise editing of the sickle cell disease mutation. Schmidt, R. et al. Base-editing mutagenesis maps alleles to tune human T cell functions. Shifrut, E. et al. Genome-wide CRISPR screens in primary human T Cells reveal key regulators of immune function. Replogle, J. M. et al. Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Yamamoto, T. N., et al. T cells genetically engineered to overcome death signaling enhance adoptive cancer immunotherapy. Wiede, F. et al. PTPN2 phosphatase deletion in T cells promotes anti-tumour immunity and CAR T-cell efficacy in solid tumours. Zhao, H., et al. Genome-wide fitness gene identification reveals Roquin as a potent suppressor of CD8 T cell expansion and anti-tumor immunity. Jain, N. et al. Disruption of SUV39H1-mediated H3K9 methylation sustains CAR T-cell function. Enhanced T cell effector activity by targeting the Mediator kinase module. Carnevale, J. et al. RASA2 ablation in T cells boosts antigen sensitivity and long-term function. Arce, M. M. et al. Central control of dynamic gene circuits governs T cell rest and activation. Courtney, A. H. et al. CD45 functions as a signaling gatekeeper in T cells. & Weiss, A. CD45: a critical regulator of signaling thresholds in immune cells. Matson, C. A. et al. CD5 dynamically calibrates basal NF-κB signaling in T cells during thymic development and peripheral activation. Timperi, E. & Barnaba, V. CD39 regulation and functions in T cells. Simon, S. & Labarriere, N. PD-1 expression on tumor-specific T cells: friend or foe for immunotherapy? Andrews, L. P. et al. LAG-3 and PD-1 synergize on CD8+ T cells to drive T cell exhaustion and hinder autocrine IFN-γ-dependent anti-tumor immunity. Frequent aneuploidy in primary human T cells after CRISPR–Cas9 cleavage. Detection and modulation of DNA translocations during multi-gene genome editing in T cells. Tsuchida, C. A. et al. Mitigation of chromosome loss in clinical CRISPR–Cas9-engineered T cells. Ohkura, N. et al. Regulatory T cell-specific epigenomic region variants are a key determinant of susceptibility to common autoimmune diseases. Targeted de-methylation of the FOXP3-TSDR is sufficient to induce physiological FOXP3 expression but not a functional Treg phenotype. Targeting a CAR to the TRAC locus with CRISPR/Cas9 enhances tumour rejection. Deleting DNMT3A in CAR T cells prevents exhaustion and enhances antitumor activity. Zhang, X. et al. Depletion of BATF in CAR-T cells enhances antitumor activity by inducing resistance against exhaustion and formation of central memory cells. Rupp, L. J. et al. CRISPR/Cas9-mediated PD-1 disruption enhances anti-tumor efficacy of human chimeric antigen receptor T cells. Stadtmauer, E. A. et al. CRISPR-engineered T cells in patients with refractory cancer. Foss, D. V. et al. Peptide-mediated delivery of CRISPR enzymes for the efficient editing of primary human lymphocytes. Dai, X. et al. One-step generation of modular CAR-T cells with AAV–Cpf1. High-yield genome engineering in primary cells using a hybrid ssDNA repair template and small-molecule cocktails. Ting, P. Y. et al. Guide Swap enables genome-scale pooled CRISPR–Cas9 screening in human primary cells. Kiani, S. et al. Cas9 gRNA engineering for genome editing, activation and repression. Altering TET dioxygenase levels within physiological range affects DNA methylation dynamics of HEK293 cells. Fiumara, M. et al. Genotoxic effects of base and prime editing in human hematopoietic stem cells. Frangoul, H. et al. CRISPR–Cas9 gene editing for sickle cell disease and β-thalassemia. Directed evolution of adenine base editors with increased activity and therapeutic application. The nf-core framework for community-curated bioinformatics pipelines. Akalin, A. et al. methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles. We thank many members of the J.E., B.R.S., A.M. and L.A.G. is supported by the National Science Foundation Graduate Research Fellowship Program (2021316455). was funded by the Arc Institute, National Institutes of Health (NIH; DP2CA239597 and UM1HG012660) and a Pew-Stewart Scholars for Cancer Research award. B.R.S is supported by NIH grants K08CA273529 and L30TR002983, the UCSF CRISPR Cures for Cancer Initiative and the UCSF Living Therapeutics Initiative. is supported by a Cancer Research Institute Irvington postdoctoral fellowship. J.C. is supported by the NIH National Cancer Institute (K08, 1K08CA252605-01), a Burroughs Wellcome Fund Career Award for Medical Scientists, the Lydia Preisler Shorenstein Donor Advised Fund and PICI. Gladstone-UCSF Institute of Genomic Immunology, San Francisco, CA, USA Laine Goudy, Alvin Ha, Jennifer M. Umhoefer, Lauren Chow, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Sanjana Subramanya, Jae Hyun J. Lee, Joseph J. Muldoon, Chang Liu, Ralf Schmidt, Julia Carnevale, Justin Eyquem, Brian R. Shy & Alex Marson Arc Institute, Palo Alto, CA, USA Laine Goudy, Ashir A. Borah, Aidan Winters, Abolfazl Arab & Luke A. Gilbert Alvin Ha, Lauren Chow, Jae Hyun J. Lee & Brian R. Shy Alexis Talbot, Zhongmei Li, Nupura Kale, Joseph J. Muldoon, Chang Liu, Julia Carnevale, Justin Eyquem & Alex Marson Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA Julia Carnevale, Justin Eyquem & Alex Marson UCSF Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA, USA Justin Eyquem, Brian R. Shy, Alex Marson & Luke A. Gilbert Department of Microbiology and Immunology, University of California, San Francisco, San Francisco, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Justin Eyquem, Brian R. Shy, Alex Marson or Luke A. Gilbert. L.A.G has filed patents on CRISPR tools and CRISPR functional genomics and is a cofounder of Chroma Medicine. A.M. is a cofounder of Site Tx, Arsenal Biosciences, Spotlight Therapeutics and Survey Genomics, serves on the boards of directors at Site Tx, Spotlight Therapeutics and Survey Genomics, is a member of the scientific advisory boards of network.bio, Site Tx, Arsenal Biosciences, Cellanome, Spotlight Therapeutics, Survey Genomics, NewLimit, Amgen and Tenaya, owns stock in network.bio, Arsenal Biosciences, Site Tx, Cellanome, Spotlight Therapeutics, NewLimit, Survey Genomics, Tenaya and Lightcast and has received fees from network.bio, Site Tx, Arsenal Biosciences, Cellanome, Spotlight Therapeutics, NewLimit, Abbvie, Gilead, Pfizer, 23andMe, PACT Pharma, Juno Therapeutics, Tenaya, Lightcast, Trizell, Vertex, Merck, Amgen, Genentech, GLG, ClearView Healthcare, AlphaSights, Rupert Case Management, Bernstein and ALDA. A.M. is an investor in and informal advisor to Offline Ventures and a client of EPIQ. The A.M. laboratory has received research support from PICI, the Emerson Collective, Arc Institute, Juno Therapeutics, Epinomics, Sanofi, GlaxoSmithKline, Gilead and Anthem and reagents from Genscript, Illumina, 10X Genomics, Ultima, and Cellanome. B.R.S is a compensated member of the scientific advisory board for Kano Therapeutics. is an inventor on patent applications pertaining to and based on the findings described in this paper, a subset of which have been licensed by the University of California. J.C. is a consultant for Waypoint Bio. is a compensated cofounder at Mnemo Therapeutics. is a current employee of Site Tx. The remaining authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–9 and uncropped western blot for Supplementary Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Integrated epigenetic and genetic programming of primary human T cells. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251020092826.htm'>Scientists just found hidden life thriving beneath the Arctic ice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 06:38:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Yet researchers have found that the same melting process could help sustain life in unexpected ways. Now, an international team led by the University of Copenhagen has discovered that more nitrogen may become available than scientists once believed. The study is the first to confirm that nitrogen fixation -- a process in which certain bacteria transform nitrogen gas (N2) dissolved in seawater into ammonium -- occurs beneath Arctic sea ice, even in its most remote and central areas. Ammonium not only helps these bacteria thrive but also nourishes algae and, by extension, the creatures that depend on them. "Until now, it was believed that nitrogen fixation could not take place under the sea ice because it was assumed that the living conditions for the organisms that perform nitrogen fixation were too poor. Unlike most other oceans where cyanobacteria dominate nitrogen fixation, the Arctic Ocean relies on an entirely different group of bacteria known as non-cyanobacteria. As climate change accelerates ice retreat, this expanding melt zone could allow more nitrogen to enter the ecosystem. "In other words, the amount of available nitrogen in the Arctic Ocean has likely been underestimated, both today and for future projections. This could mean that the potential for algae production has also been underestimated as climate change continues to reduce the sea ice cover," says von Friesen. "Because algae are the primary food source for small animals such as planktonic crustaceans, which in turn are eaten by small fish, more algae can end up affecting the entire food chain," she adds. This new nitrogen source could also influence how much carbon dioxide the Arctic Ocean takes in. But biological systems are very complex, so it is hard to make firm predictions, because other mechanisms may pull in the opposite direction," explains Lasse Riemann, professor at the Department of Biology and senior author of the study. But it is clear that we should include an important process such as nitrogen fixation in the equation when we try to predict what will happen to the Arctic Ocean in the coming decades as sea ice declines," adds Riemann. These microorganisms consume dissolved organic matter -- often released by algae -- and in turn, produce fixed nitrogen that promotes further algal growth. This exchange creates a small but vital nutrient loop beneath the ice. As they grow, they pull carbon dioxide from the air, which can later sink to the ocean floor as part of their biomass. Their findings are based on two major research expeditions aboard the icebreakers IB Oden and RV Polarstern. Note: Content may be edited for style and length. Scientists Just Found a Tiny Genetic Switch That Could Feed Billions These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds New Study Challenges Common Beliefs About “Unhealthy” Processed Fats Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251020092833.htm'>Feeling stressed? Science finds a simple way to take back control</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 02:54:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When several of these small frustrations build up, stress can quickly escalate. But according to new research that includes scientists from Penn State, simply feeling more in control might help make those everyday challenges easier to handle. The study found that on days when people felt they had greater control over their stressors, they were 62% more likely to take action -- such as calling a plumber or addressing a difficult conversation -- to solve the problem. The results suggest that a person's sense of control can serve as an important psychological resource that can be strengthened over time to help manage stress and improve health outcomes. "This research shows that even small boosts in how much control people feel they have over everyday hassles make it more likely that those hassles actually get resolved," said David Almeida, professor of human development and family studies at Penn State and senior author on the paper. "Learning to find and act on these pockets of control in daily life may not only reduce stress but also support long-term health and well-being." Previous work led by Almeida has shown that stress responses fluctuate over time and that even minor daily inconveniences can affect health when they accumulate. Actively resolving stressors -- such as clearing up an argument -- is a key part of processing stress and allows emotions to settle more quickly. In this new study, the researchers wanted to know whether a sense of personal control -- how much people feel they can influence their daily challenges -- affects whether a stressor gets resolved. "If perceived stressor control promotes stress resolution, can we leverage that as a modifiable resource to influence stressor resolution and therefore our emotional health and well-being?" The same survey was repeated with these participants 10 years later to see whether the connection between perceived control and stress resolution changed with time. Results showed that people's sense of control can shift greatly from one day to the next. Across all ages, on days when people felt more in control than usual -- for instance, feeling "some" control instead of "a little" -- they were significantly more likely to resolve the stressor. This pattern held true regardless of the stressor's type or intensity. Early in the study, participants who felt a higher-than-usual level of control were 61% more likely to resolve their stressors that day. A decade later, the same boost in perceived control raised that likelihood to 65%. "This work also begins to show that as we get older, not only do we have more control but that control helps us get better at handling stress," Witzel said. The findings suggest that perceived control can serve as a key tool for reducing daily stress. It can be strengthened through practical strategies such as setting priorities or reframing what's within reach," Almeida said. Practical ways to build control include focusing on what's within reach, breaking large challenges into smaller steps, and using time blocking or lists to track progress. Asking for help or delegating tasks can also boost the sense of support and control. The team plans to keep studying how perceived control interacts with long-term or chronic stress. "In this study, we're talking about daily stressors, the minor inconveniences that occur throughout the day, but there's also chronic stress where people are continually impacted by stressors again and again," Witzel said. "Exploring the idea of whether resolution can be a mechanism that decreases the effect of chronic stress is an interesting area to explore." Eric Cerino, associate professor at Northern Arizona University, is co-lead author of the paper. Scientists Just Found a Tiny Genetic Switch That Could Feed Billions These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251020092824.htm'>Eating ultra-processed foods may rewire the brain and drive overeating</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-21 02:24:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An international team of scientists has analyzed brain scans from around 30,000 participants and found striking connections between the frequent consumption of ultra-processed foods (UPFs) and measurable differences in brain structure. These differences could be part of a feedback loop that promotes overeating and food addiction. "Our findings suggest that higher consumption of ultra-processed foods is associated with differences in the brain. The observed associations are not solely explained by inflammation or obesity; ingredients and additives typical to UPFs, such as emulsifiers may also play a role, although this requires further longitudinal or experimental evidence," explains the study's shared first author, Arsène Kanyamibwa from the University of Helsinki. Many processed items, especially those based on plants, are beneficial parts of a balanced diet. However, ultra-processed foods -- those containing industrial additives and chemically altered ingredients -- raise significant health concerns. "In particular, processed foods of plant origin, such as frozen vegetables, can be recommended. Kanyamibwa highlights how emerging research like this can influence both personal food choices and public health policy. "Given the growing body of evidence, reducing ultra-processed food intake and strengthening regulatory standards in food manufacturing may be crucial steps toward ensuring better public health outcomes," Kanyamibwa asserts. Scientists Just Found a Tiny Genetic Switch That Could Feed Billions These Popular Drugs Could Reduce Your Risk of Dementia, Study Finds Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            