
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250715043355.htm'>Chirality shock: Geneva chemists forge millennia-stable ‘mirror-proof' drugs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 20:41:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Understanding and controlling this phenomenon is crucial to drug design. This work opens up new prospects for the design of geometry-controlled drugs. A molecule, or any object, is said to be chiral if it cannot be superimposed on its mirror image by any combination of rotations, translations and geometric changes. This is similar to our two hands, which appear identical but cannot be superimposed, whether viewed from the back or the palm. This universal molecular asymmetry requires chemists to design chiral molecules capable of interacting precisely with living systems. Within a molecule, chirality often arises from the presence of one or more asymmetry centers, known as stereogenic centers. These are often made up of a central carbon atom, itself linked to four different groups or chains of atoms, usually carbon as well. This time, the central carbon atom is not surrounded by carbon chains, but only by oxygen and nitrogen atoms. ''Molecules with this new type of stereogenic center had never before been isolated in a stable form. The stability of chiral molecules is a particularly important parameter. Mirror molecule pairs are structurally very close, and in many cases spontaneous switching from one to the other is possible, for example under the effect of temperature. In this way, we could switch from a drug to an inactive or even toxic molecule! Olivier Viudes, PhD student and first author of the study explains: "Using dynamic chromatography techniques and quantum chemistry calculations, we have shown that, for the first molecule developed, it would take 84,000 years at room temperature for half a sample to transform into its mirror molecule." ''These novel stereogenic centers offer a new way of organizing molecular space. Note: Content may be edited for style and length. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/pneumonic-plague-infections-in-modern-times-show-the-black-death-isnt-dead/'>Pneumonic Plague Infections in Modern Times Show the Black Death Isn't Dead</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 16:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Pneumonic Plague Infections in Modern Times Show the Black Death Isn't Dead Human infections and fatalities from plague are relatively rare in the U.S.; according to the Centers for Disease Control and Prevention, seven human cases are reported annually on average. Y. pestis arrived in port cities in the U.S. around 1900 and has since become endemic to rats and other rodents in western U.S. states including New Mexico, Arizona, Colorado, California, Oregon and Nevada. “From a public health standpoint in the U.S., it's a scary thing that it's plague, and it's tragic that that this was a fatal case, but people need to remember that it's extremely rare,” says David Wagner, executive director of the Pathogen and Microbiome Institute at Northern Arizona University, who has studied plague for more than 25 years. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. You have an infected rodent; a flea feeds on the blood of that rodent, and it picks up some Y. pestis. It's constantly cycling back and forth between rodents and fleas in nature; that's how it's been maintained for thousands of years in the environment around the world. Shuyao Xiao; Source: Centers for Disease Control and Prevention (data) What's the difference between bubonic and pneumonic plague? What stands apart [with the recent case is] that it's pneumonic plague. If there isn't a rodent host for that flea to feed on, it will look for other mammals to feed on. So let's say I was bit on my wrist; then the bacteria would go to that lymph node in my underarm and start to reproduce there. These days, it's a dead end because there's not flea-borne transmission from one human to another. It just stops there with the treatment or death of that individual. Left untreated, though, bubonic plague can get down into your lungs via the bloodstream. What people might not know is that plague has been endemic throughout the western U.S. in rodent populations for more than 100 years. Someone could also get pneumonic plague from an animal—for example, if they were handling an infected animal and that animal coughed. Sometimes hunters in Central Asia will kill [infected] ground squirrels, and when they're skinning them can inhale particles. People also talk about septicemic plague, and that means it's gotten into your bloodstream, and that typically also arises from bubonic plague. You could also get [septicemic plague] directly if you had cuts on your hands and were handling rodents without gloves. Can pets get infected or transmit plague to humans? Pets, especially free-roaming ones, may come into contact with dead rodents that have died of plague. Fleas can hop onto pets, which then bring them into the home. This is pretty rare because there are so few [human] cases in the U.S., but that is something we think about. Flea and tick collars are a good idea. If animals do get sick, most of the evidence shows that dogs fight off the infection and can create antibodies against Y. pestis. With bubonic plague, typically people develop a fever, headache, chills and fatigue, and then they'll get those swollen lymph nodes called buboes. It typically takes a few days to manifest because it sort of starts off in stealthy mode inside the body to try and avoid the immune system. If untreated, bubonic plague mortality rates may be somewhere between 30 to 60 percent, depending on the situation. Pneumonic plague, left untreated, is almost always fatal. Shuyao Xiao; Source: Centers for Disease Control and Prevention (data) What people might not know is that plague has been endemic throughout the western U.S. in rodent populations for more than 100 years. It just so happened that a lot of the rodent host species in Central Asia, where it evolved, were quite similar to some of the ground squirrels that we have here in the U.S. I'm also currently working on plague with colleagues in Madagascar, which has more human plague cases than any other country—around 75 percent of the global [case] total. It has hundreds to thousands of cases every year, with quite a few deaths. We don't have a good handle on which rodents are actually maintaining it in the U.S. So we've been going out and collecting plague-infected fleas from prairie dogs for more than 25 years because it's our window into plague activity. Prairie dogs are ground squirrels that live in dense colonies, so they're very conspicuous. If the prairie dogs die off, we go out and collect fleas from their burrows. Then we can study the Y. pestis DNA directly from those fleas. Every year we review the symptoms and they give us a prescription for antibiotics. The prairie dogs can't be maintaining it because they're just so susceptible—it'll just wipe out a whole colony when it gets in. And so there's other rodents out there maintaining plague, but it's still a bit of a mystery in the western U.S. There were three great historical plague pandemics, and it's estimated that maybe 250 million people died [total]. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03834-0'>The Global Neurodegeneration Proteomics Consortium: biomarker and drug target discovery for common neurodegenerative diseases and aging</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 15:49:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). More than 57 million people globally suffer from neurodegenerative diseases, a figure expected to double every 20 years. Despite this growing burden, there are currently no cures, and treatment options remain limited due to disease heterogeneity, prolonged preclinical and prodromal phases, poor understanding of disease mechanisms, and diagnostic challenges. Identifying novel biomarkers is crucial for improving early detection, prognosis, staging and subtyping of these conditions. High-dimensional molecular studies in biofluids (‘omics') offer promise for scalable biomarker discovery, but challenges in assembling large, diverse datasets hinder progress. It includes approximately 250 million unique protein measurements from multiple platforms from more than 35,000 biofluid samples (plasma, serum and cerebrospinal fluid) contributed by 23 partners, alongside associated clinical data spanning Alzheimer's disease (AD), Parkinson's disease (PD), frontotemporal dementia (FTD) and amyotrophic lateral sclerosis (ALS). This dataset is accessible to GNPC members via the Alzheimer's Disease Data Initiative's AD Workbench, a secure cloud-based environment, and will be available to the wider research community on 15 July 2025. Here we present summary analyses of the plasma proteome revealing disease-specific differential protein abundance and transdiagnostic proteomic signatures of clinical severity. Furthermore, we describe a robust plasma proteomic signature of APOE ε4 carriership, reproducible across AD, PD, FTD and ALS, as well as distinct patterns of organ aging across these conditions. This work demonstrates the power of international collaboration, data sharing and open science to accelerate discovery in neurodegeneration research. Neurodegenerative diseases, including Alzheimer's disease (AD), Parkinson's disease (PD), amyotrophic lateral sclerosis (ALS), frontotemporal dementia (FTD) and other related conditions, affect more than 57 million people worldwide1. Until recently, treatment options were limited to managing symptoms, but approvals for disease-modifying drugs for AD and genetic forms of ALS point to considerable progress2,3. Eventually, it may be possible to provide patients with targeted treatments, possibly in combination, that can prevent, slow, stop or reverse the progression of their disease4,5,6. However, several major obstacles have delayed the realization of this vision. First, many neurodegenerative conditions have an extended preclinical or prodromal period where diagnosis using available symptom-based assessments is either not possible or extremely difficult due to subtle manifestations that are not detectable by current clinical tools. Second, heterogeneity in the concordance between molecular pathology and clinical syndrome as well as common co-occurrence of multiple pathologies (‘co-pathology') contribute to misdiagnosis in clinical settings. Third, additional variability exists in the rate and pattern of symptom progression within conditions, impeding efforts to accurately prognosticate disease course. These diagnostic and prognostic challenges ultimately hinder the efficacy of clinical trials and make successful treatment of patients with any approved disease-modifying therapy challenging. Biomarkers have the potential to resolve some of these obstacles by enabling earlier diagnosis linked to pathological processes, providing methods to subtype diseases, predicting outcomes and ultimately guiding effective intervention7,8. They may also improve clinical trial design through precision recruitment and serve as pharmacodynamic or surrogate endpoints in experimental medicine. Recognizing this potential, the field has seen rapid advances in imaging and fluid biomarker research, leading to their growing incorporation into clinical trials and regulatory frameworks. Fluid biomarkers, in particular, offer a real-time window into brain pathology and may help bridge the longstanding disconnect between neuropathology and clinical symptoms in living patients. Reflecting this progress, clinical guidelines have begun to integrate fluid biomarkers into routine diagnostic workflows9. To date, fluid biomarker development has been most successful for AD, where markers of amyloid and tau pathology are now widely used. However, there remains an urgent need for reliable biomarkers of other neurodegenerative pathologies, including α-synuclein, TDP-43 and non-AD tauopathies. In addition, biomarkers that reflect non-specific but disease-relevant biological processes, such as neuroinflammation, metabolic dysregulation and vascular dysfunction, are essential to fully characterize the pathophysiological and molecular landscape of neurodegeneration. The accelerating development of high-throughput molecular profiling technologies, combined with increasingly powerful computational tools applied to large, deeply phenotyped cohorts, is transforming the landscape of biomarker discovery10. Although multi-omics approaches, such as integration of genomics, transcriptomics and metabolomics, contribute to rich data-driven biomarker discovery, proteomics is uniquely positioned to impact both diagnosis and treatment of neurodegenerative disease. This is due to several key factors: (1) many clinically established biomarkers are protein based; (2) high-dimensional proteomic platforms such as SomaScan, Olink and mass spectrometry now offer sufficient depth to capture a sizable portion of the circulating proteome; and (3) protein-level changes often capture biological processes proximal to neurodegeneration, providing functional insights that are directly relevant to disease pathogenesis. Proteomic profiles derived from peripheral biofluids such as plasma and cerebrospinal fluid (CSF) not only hold promise for identifying biomarkers of disease presence and progression but also offer new avenues for therapeutic target discovery. Robust high-dimensionality omics research in heterogeneous clinical groups necessitates the use of large datasets due to poor reproducibility of findings from single-site or smaller cohorts11, but the siloing of data among a fragmented research community has been a barrier to such biomarker discovery12. Although many research institutions and initiatives have embarked on a variety of open data efforts, there is no standard model for providing researchers with easy access to data from multiple cohorts. Moreover, the use of such multicohort sources requires data aggregation and harmonization. The genetics research community has enabled huge consortia with joint data access and collaborative analysis13. In the field of neurodegenerative disease, large data-sharing efforts such as the ADNI14, AMP-AD15, PPMI16, ALS TDI's ARC17 and Answer ALS18 are examples of open datasets that facilitate cross-study collaborations. In addition, most data have either never been shared or have been kept behind restrictive barriers to access12. The GNPC was created to systematically address these challenges to large data analysis, accelerate biomarker discovery and advance the research and development of more precise treatments for neurodegenerative disease. Our goal was to generate a large proteomics resource using available samples from established cohort studies, accompanied by a harmonized clinical dataset, and to make these data available to the scientific community in a rapid and easily accessible manner. Having reached the planned point of public data release in July 2025, the GNPC has established what it thinks is the world's largest neurodegenerative disease-focused proteomics dataset for biomarker research, with 23 partners contributing more than 35,000 analyzed biosamples and approximately 250 million unique protein measurements with matched and harmonized clinical data. Here we summarize the GNPC version 1 (V1) dataset, together with key analysis vignettes. Traditional ‘on premises' data science analyses have become more challenging as datasets have increased in size, with a corresponding increase in resources required for moving data from one location to another. Moreover, local analysis presents a challenge in ensuring data integrity, safety and confidentiality. For GNPC V1, we opted to use SOMAmer technology (provided by SomaLogic) as the primary proteomics platform as it was one of the broadest discovery platforms available. However, we also analyzed a subset of samples with Olink and mass spectrometry methods to allow for cross-platform comparison. In general, the GNPC's approach is agnostic to platforms and is guided primarily by coverage, reproducibility and affordability. We also think that different platforms bring complementary information as they may measure different isoforms and/or posttranslational modifications of a given protein. To bring together data from several countries required navigating legal regimes with different requirements, including the GDPR (Europe), the Data Protection Act (United Kingdom) and HIPAA (United States). The GNPC's legal team worked with institutions in each jurisdiction to address specific concerns and agree on a framework for data sharing that worked across the board, including collecting the data on servers physically located in Western Europe. Analysis of the harmonized dataset is organized in four workstreams to allow members of the consortium to collaborate on areas of related interest: longitudinal profiling, cross-sectional profiling, proteogenomics and prediction modeling. The GNPC V1 harmonized dataset is focused on neurodegenerative diseases and, more specifically, on AD, PD, ALS, FTD and aging among 18,645 participants, drawn from 23 individual cohorts across a total of 31,083 unique peripheral plasma, serum and CSF samples, culminating in 35,056 unique proteomic assays and approximately 250 million individual protein measurements (Table 1 and Supplementary Table 1). Most of the proteomics characterization comes from the SOMAmer-based capture array (SomaScan version 4.1, version 4 and version 3 platforms), measuring approximately 7,000 (n = 26,458), 5,000 (n = 4,528) or 1,300 (n = 95) unique aptamers per biosample, respectively. Additionally, 1,975 of the plasma samples characterized on the version 4.1 SomaScan platform had tandem mass tag mass spectrometry performed. The harmonized dataset additionally includes 40 clinical features, including demographic data, vital signs data and clinical features collected with each blood or CSF draw (Supplementary Table 3). These aggregated and harmonized data demonstrated their value to the consortium immediately, as they served to rapidly confirm signals originally identified in smaller datasets across the entirety of the GNPC V1 dataset, thereby serving as an ‘instant validation' resource24,25. To evaluate the structure and comparability of the blood-based proteomics data, we conducted a principal component analysis on plasma and serum samples (Supplementary Fig. Serum samples clustered distinctly from plasma, reflecting a clear matrix effect. Among plasma samples, a modest offset was observed between the 5K and 7K SomaScan platforms, whereas EDTA and citrate plasma samples appeared largely similar. All three vignettes described below focused exclusively on plasma proteomic data, and, where relevant, platform-related differences between 5K and 7K assays were addressed using scaling or predictive modeling approaches. To highlight the breadth and utility of the GNPC V1 resource, we present three illustrative vignettes that showcase how this harmonized dataset can be applied to address key questions in neurodegenerative disease and aging research: (1) disease-specific differential abundance profiling, (2) biological aging across organ systems and (3) protein markers of genetic risk as exemplified by the apolipoprotein E (APOE) genotype. As summarized below and described in more detail in the accompanying papers in this issue19,20,21, these vignettes reflect the analytical depth enabled by the GNPC and are intended to catalyze further exploration by the broader research community upon public data release. We examined the plasma proteome as measured using the SomaLogic 7K platform, among people with AD, PD, FTD and ALS (referred to hereafter as ‘Patients', n = 3,002), and separately among people with no neurodegenerative disease diagnosis and cognitively normal test screenings (referred to hereafter as ‘Controls', n = 5,879) (see Vignette 1 methods for sample selection criteria). First, we sought to identify proteins differentially abundant in the plasma of patients with different neurodegenerative diseases with cognitive effects—namely, AD, PD, FTD and ALS. Leveraging the breadth of cohorts included in the GNPC, we first performed cohort-stratified analyses to internally validate the most robust protein changes, focusing on those consistently altered across multiple study cohorts. Cohort-stratified results were subsequently combined using a meta-analysis for AD (Fig. a–d, Meta-analytic differential abundance analysis showing changes in relative protein expression of AD (a), PD (b), FTD (c) and ALS (d) compared to Controls. The parallel line at the bottom of each plot shows which proteins are significant after FDR correction for multiple comparisons. The line above shows proteins further surviving Bonferroni correction. Dots are colored based on the number of cohorts where the protein was found to be independently significant after (within-cohort) FDR correction and changed in the same direction relative to Controls (that is, increased or decreased compared to Controls). e–h, Significant proteins from the differential abundance analyses were fed into Reactome enrichment analysis for AD (e), PD (f), FTD (g) and ALS (h), using unique SomaScan 7K proteins as background. Enriched Reactome pathway terms for each condition are visualized as dot plots, with dot size corresponding to the number of differentially abundant proteins assigned to a given pathway (one-sided Fisherʼs test with FDR adjustments). Full Reactome enrichment summary statistics are reported in Supplementary Table 6. HIV, human immunodeficiency virus; FGF, fibroblast growth factor; TBC/RABGAPs, Tre2–Bub2–Cdc16 (TBC) domain-containing RAB-specific GTPase-activating proteins. i, Violin plots displaying LASSO-derived clinical severity protein signatures across CDR global level in training and test sets. In AD (n = 1,966), 27 proteins from AD Patients robustly emerged as being significantly elevated compared to Controls across at least six of the 10 different cohorts, including ACHE, SPC25, LRRN1 and CTF1. Furthermore, we observed significant decreases in RPS12, NPTXR and NT5C across five cohorts. Follow-up analysis (see Vignette 3) also indicated that elevation of some targets, such as SPC25, LRRN1 and CTF1, reflected underlying APOE ε4 genotype effects rather than AD diagnosis per se. Reactome pathway analyses (n = 2,640, Bonferroni-adjusted P < 0.05) revealed enrichment for terms related to sugar metabolism (‘glucose metabolism' and ‘glycolysis') and protein prenylation (‘RAB geranylgeranyltransferase'), reinforcing links to bioenergetics and vesicle trafficking (Fig. In PD (n = 607), 40 proteins were significantly elevated across at least three of seven different cohorts, including SUMF1, PRR15, AARDC3 and RDH16, which were elevated in at least four cohorts (Fig. Meta-analytic effect sizes were very high for PRSS8, BAGE3, NPS, PRL and HEXB, all of which decreased in PD but were less reproducible across contributing cohorts, suggesting cohort-specific factors driving depleted abundance of these targets. Similar to in AD, Reactome pathway analyses (n = 2,251, Bonferroni-adjusted P < 0.05) revealed enrichment for terms related to Ras superfamily/small GTPases and vesicle trafficking (‘vesicle-mediated transport' and ‘signaling by RHO GTPases'), highlighting a plasma proteome pathway overlap between AD and PD (Fig. Although FTD clinical syndromes are less common than AD or PD and have greater clinical and neuropathological diversity, nine targets exhibited decreased abundance in FTD plasma (n = 175) across multiple cohorts after Bonferroni correction (Fig. Eleven proteins were significantly elevated in FTD plasma with a conventional false discovery rate (FDR) correction but did not survive Bonferroni correction. Despite limited power, Reactome pathway analyses (n = 71, FDR < 0.05) revealed two significantly enriched terms, ‘posttranslational protein phosphorylation' and ‘regulation of insulin-like growth factor transport and uptake by IGFBPs', highlighting conserved peripheral signatures of neurodegeneration even amid the clinical and pathological heterogeneity of FTD (Fig. In ALS (n = 254), we analyzed plasma proteomic profiles from a single contributing cohort with Patients and Controls (Fig. After FDR correction, 44 targets exhibited significantly increased abundance in ALS, including a host of proteins related to skeletal muscle structure and function (PDLIM3, MYOM2, MYLPF and TNNI2). Thirty-eight targets exhibited significantly decreased abundance in ALS, including two aptamers targeting ART3, an ADP-ribosyltransferase enriched in skeletal muscle, as well as additional proteins linked to growth factor signaling and/or extracellular matrix composition (ANTXR2, CRTAC1 and RGMA). Reactome pathway analyses (n = 82, FDR < 0.05) confirmed this strong biological enrichment for skeletal muscle-related processes (‘muscle contraction' and ‘collagen chain trimerization') (Fig. 1h), underscoring a clear peripheral proteomic footprint of ALS consistent with its primary motor system pathology. After identifying disease-related differential abundance patterns, we combined data across AD, PD and FTD to identify a global signature of dementia severity. Specifically, a 256-protein clinical impairment signature was derived using least absolute shrinkage and selection operator (LASSO)-based prediction of Clinical Dementia Rating (CDR) global scores, which was subsequently evaluated in a held-out test set (2,047 records; 30% of the dataset). Signature values correlated with CDR global scores (train: Pearsonʼs r = 0.68; test: r = 0.58) in a stepwise fashion, increasing at each level of clinical severity (Fig. To demonstrate concordance with an orthogonal clinical outcome, we observed that higher signature values were also associated with lower cognitive test scores (standardized Montreal Cognitive Assessment/Mini-Mental State Examination (MoCA/MMSE); r = −0.47), consistent with expected inverse correlations between cognition and CDR. In disease-stratified analysis combined across training and test sets, the multivariate protein signature was reliably elevated with greater clinical severity in AD (r = 0.55), FTD (r = 0.85) and PD (r = 0.70), supporting its relevance as a transdiagnostic marker of clinical severity (Fig. Top proteins with high feature importance in the clinical impairment signature (Fig. One advantage of plasma proteomics is the ability to simultaneously query the health of distinct organ systems. We applied previously validated plasma proteomic organ aging models26 to assess accelerated organ-specific aging across multiple Patient and Control data in the GNPC. Predicted organ ages showed moderate to strong correlations with chronological age (r range, 0.36–0.92; Fig. Elevated cognition-enriched brain age gaps, reflecting the subset of brain-specific proteins that previously enhanced model age gap prediction of cognitive impairment, were associated with higher odds of AD (odds ratio = 1.33 per 1-s.d. Non-cognition-enriched brain age gap was weakly associated with AD risk (odds ratio = 1.08 (95% confidence interval: 1.02–1.14)) but not other conditions. Beyond brain, we observed contributions of artery (odds ratio = 1.18 (95% confidence interval: 1.11–1.25)), liver (odds ratio = 1.11 (95% confidence interval: 1.05–1.17)) and intestine aging to AD (odds ratio range, 1.12–1.18) as well as a unique link between muscle aging and PD (odds ratio = 1.12 (95% confidence interval: 1.05–1.19)). These findings extend previous work by demonstrating shared and distinct patterns of blood-detectable accelerated organ aging across AD, FTD and PD, underscoring connections between systemic health and neurodegenerative disease that may be related as a cause, correlate or consequence. Pearsonʼs correlation coefficient r is reported for each clock. b, Body plots showing associations of standardized organ age gaps with neurodegenerative disease based on binary logistic regression models. Red dots indicate positive associations (higher age gap with disease); blue dots indicate negative associations (lower age gap with disease). The body plots were created in BioRender: Oh, H. (2025): https://BioRender.com/afoqtwz. Using a combination of machine learning and biological enrichment approaches, we sought to isolate the molecular signatures of APOE ε4, the main genetic risk factor for sporadic AD, independent of AD and other conditions. Several proteins, including SPC25, LRRN1, S100A13 and NEFL, were strongly associated with APOE ε4 versus other alleles (Fig. 3a), paralleling prior observations for some of these targets in serum27. Some proteins, such as SPC25 and LRRN1, showed no difference between AD and Controls but exhibited dose-dependent associations with APOE ε4 (Fig. 3b), suggesting that previously identified links to AD from Vignette 1's differential abundance analysis were driven by APOE ε4 enrichment in AD cases. Conversely, well-known AD-associated neuromodulatory proteins such as NPTXR and GDF2 were robustly associated with AD diagnosis, irrespective of APOE ε4 allelic dose (Fig. a, Volcano plot shows the protein association profile of APOE ε4 after adjusting for AD dementia diagnosis, with red representing significant associations (after FDR correction). b,c, Box plots show plasma protein level changes of the proteins with the strongest APOE ε4 associations (b) and for APOE ε4-associated proteins strongly tied to AD dementia diagnosis (c). For b and c, the y axis represents residual protein levels after adjusting for age, sex, mean protein level and contribution site. Whiskers extend to the most extreme values within 1.5 times the interquartile range; data points beyond this range were excluded as outliers. Welch's t-test was used to compare residual protein levels between groups. P values were not adjusted for multiple comparisons, as only prespecified group contrasts are shown. d, Receiver operating characteristic area under the curve (ROC-AUC) showing the performance of a machine learning model using only five proteins to predict APOE ε4 status across different diagnostic groups, in a held-out sample. e, Protein interaction network including four of those five proteins (red). f, Neural cell type expression of RNA transcripts encoding the five APOE ε4-predictive proteins. Plot shows mix-max scaling of protein-coding transcripts per million for each identified APOE ε4 protein. g, Correlation of effect sizes for proteins associated with APOE ε4 in cognitively unimpaired samples (x axis) and AD associated with AD diagnosis in APOE ε3/ε3 homozygotes. Limma t-statistic is shown for both contrasts; only proteins associated with both APOE ε4 and AD (adjusted P < 0.05 for both analyses) with the same direction of effect are visualized. For visibility purposes, t-statistic values higher than 10 were capped. PDD, Parkinsonʼs disease dementia; pFDR, FDR-corrected P value. Notably, the effects of APOE ε4 genotype on the plasma proteome were so robust that a machine learning model with only five proteins (SPC25, NEFL, S100A13, TBCA and LRRN1) predicted APOE ε4 carrier status in unseen patients with high accuracy, both within AD and within non-AD Patients (area under the curve (AUC) range, 0.90–0.96; Fig. Leveraging protein–protein interaction libraries and single brain cell RNA sequencing data from the Human Protein Atlas28, we observed that three of these proteins (SPC25, TBCA and S100A13) were central nodes in the protein–protein interaction network (Fig. Ubiquitin-C, however, was found to be a key connection point across the central node proteins, suggesting potential convergence on proteostatic pathways. Lastly, to identify potential genotype–phenotype links, we compared proteins associated with APOE ε4 in cognitively unimpaired individuals (n = 2,817; 215 proteins) to those associated with AD in APOE ε3 homozygotes (n = 1,843; 2,150 proteins). Forty-four overlapping proteins showed consistent directionality (Fig. Patients involved in immunovascular signaling (MMP8), synaptic vesicle fusion (SNAP23) and lipid trafficking (APOB) pathways relevant to APOE biology and cognitive decline. This overlap highlights potential early molecular footprints of AD pathophysiology present even in asymptomatic APOE ε4 carriers and underscores the utility of contextualizing by APOE genotype when seeking AD-relevant proteomic signals. These shared proteins may reflect core features of APOE-related biology that are also prominent in AD, highlighting potential mechanisms through which APOE ε4 contributes to disease vulnerability. The availability of high-dimensional molecular datasets has led to an increasing number of large-scale collaborative programs to share and use these data—a trend forged by the genetics community. Following these data-sharing collaborations, numerous initiatives have led the way in data sharing with the wider scientific community. We would like to highlight in particular two programs focused on neurodegeneration or proteomics. The first large-scale open data-sharing program in neurodegeneration was the Alzheimer's Disease Neuroimaging Initiative (ADNI): an immensely productive public–private partnership that continues today and has spawned many followers. These include AddNeuroMed/InnoMed, an ADNI-like program in Europe that served as a pilot for the European Union Innovative Medicines Initiative (IMI) funding scheme that itself has generated many data-sharing and sample-sharing programs, including, for example, IMI-EMIF and IMI-EPND in neurodegeneration. Notably, a key point of emphasis of the GNPC's mission is the strong intent to share this dataset with the global research community early in its life cycle. We do not present these analyses as definitive but, rather, as the pilot experiments by a subset of researchers whose datasets contributed to the construction of GNPC V1. We hope this summary paper and more in-depth papers serve as an invitation of collaboration and/or independent analysis from the global community. Only then will we be able to maximize disease insights from GNPC V1 and its combination with other datasets to accelerate translation of insights into the next generation of diagnostics and therapeutics for neurodegenerative diseases. These results not only validate previously reported protein markers but also nominate new candidates for mechanistic follow-up and blood biomarker development—an urgent clinical need across diseases. Second, organ aging clocks applied to the GNPC dataset uncovered disease-specific patterns of accelerated aging across brain and peripheral organs, offering a systems-level view of proteomic aging that bridges central and systemic health. These findings extend previous work on biological aging by demonstrating that distinct conditions are associated with unique organ-specific age gaps, supporting their relevance to age-related neurodegenerative diseases. Finally, proteome-wide analysis of the APOE genotype revealed a robust and disease-independent APOE ε4 signature, with potential mechanistic relevance to proteostasis and lipid transport. These vignettes, each explored further in companion publications19,20,21, illustrate the range of insights made possible by GNPC V1 and set the stage for future hypothesis-driven and exploratory research by the broader scientific community, with the potential to better support trial design, monitoring and subtyping of clinical patients. These data, as well as these vignette analyses and those in the accompanying papers, suggest that very large protein datasets have potential to add value to drug discovery. Hitherto, the value of genetic data has been increasingly recognized in supporting effective drug discovery. Notably, targets with genetic support are more likely to progress through the drug discovery pipeline29 with probability of success recently being calculated to be 2.6 times greater than in targets lacking genetic support, even for genes with small effect sizes30. In contrast to the inherited traits represented by genetic variants, proteomics represents biological states. Although genetic factors are intrinsically causal in their relationship with disease, a proteomic association with disease might be consequential of the disease, a factor associated with disease (including response to a therapeutic) or reflect a causal process. Although these observations might suggest a role for proteomics more in supporting drug discovery through biomarker discovery rather than for target identification, the vignettes reported here and in the accompanying papers illustrate the potential for more direct drug target identification/validation. For example, in Vignette 1, the proteomic profiles identified include strong support for synaptic dysfunction with proteins identified that are already clearly part of a mechanism targeted for neurodegeneration drug discovery, such as NPTXR31, whereas, in Vignette 3, proteins are identified that are very strongly associated with APOE status and with disease state. These proteins will surely now be considered as possible targets for drug discovery, especially considering that the APOE genotype is a striking example of a very strong genetic risk factor that has not been the source of equally strong drug discovery programs. Preclinical models might be used to generate signatures of targets or interventions, and, using GNPC data, these signatures now can be used to predict outcomes. Whether in conjunction with genetic data or with preclinical models, the proteomics data now being made available are likely to become a strong additional component of effective target identification and validation. As additional datasets similar to the GNPC and UKBB-PPP become available, it is possible that proteomics will become as important an element of drug discovery as genetics is today. These include the relative lack of diversity, reflecting much of past observational bioresources where most research participants have been individuals of European ancestry living in the Global North. The dataset would also be enhanced with other proteomics platforms, including complementary molecular biomarker data including genomics, transcriptomics and metabolomics as well as imaging and clinical data and with other disease types related to neurodegeneration. Data harmonization challenges in a post hoc meta-analysis such as the GNPC include site differences in sample processing, clinical methodologies for diagnoses and patient demographics. For example, incomplete medication information limited the ability to identify and separate definitive drug-induced changes to the proteome from disease-relevant changes and/or to correlate proteomic signatures comprehensively with amyloid and synuclein biomarkers. The GNPC has been built to accommodate growth and aims to increase diversity in terms of patients and measurements over time. As a next step, the GNPC is poised to incorporate additional cohorts, samples and data platforms in a V2 dataset and will seek to rectify some of these limitations as we build out the collaboration. The GNPC's aspiration is to facilitate neurodegenerative disease research and development, driving advances toward better outcomes for people with neurodegenerative diseases through precision, combination therapy optimized for a patient's disease subtypes. Biofluid specimens were independently shipped by each contributing cohort to SomaLogic, with coordination support provided by Gates Ventures. The SomaScan platform uses slow off-rate modified aptamers (SOMAmers) to quantify thousands of proteins in human biofluids, including plasma, serum and CSF. All participating cohorts confirmed that all contributed clinical and generated biosample data were in compliance with the individual patient consents prior to contributing data to the GNPC. Proteomic data from each contributing cohort were processed separately, following SomaLogic's standardized adaptive normalization by maximum likelihood (ANML) pipeline for hybridization normalization, signal calibration and quality control. These procedures adjust for systematic variation using internal reference standards and buffer controls included on each assay plate. Samples with signal intensities that substantially deviated from expected ranges are flagged by SomaLogic for quality concerns but were not removed from the dataset. After cohort-level processing and normalization, datasets were combined to form the harmonized GNPC V1 dataset. At the time of release, we have included 53 clinical variables across all datasets and will continue to increase this number toward a target of over 150 harmonizable variables in the next version of the harmonized dataset. To harmonize the clinical metadata across the 23 cohorts, we started with a minimum set of 13 required features that were requested from each cohort: age, gender, years of education, date of visit, diagnosis, date of diagnosis, medication use, comorbidities, vital signs, at least one psychiatric measure, at least one cognitive dementia or functional rating score, at least one cognitive test score and disease-specific genotype data. After initial data contribution, five key vital signs (height, weight, body mass index, resting heart rate and blood pressure) and 14 comorbidities (alcohol use, smoking/tobacco use, stroke, transient ischemic attack, traumatic brain injury, cancer, congestive heart failure, chronic obstructive pulmonary disease, myocardial infarction, atrial fibrillation, angina, hyperlipidemia and hypertension) were identified as common features across at least five cohorts for harmonization. Demographic and biometric information was normalized to a common scale. Biologically impossible values were cleaned from the dataset. To handle outlier values in years of education, height, weight, body mass index, resting heart rate, blood pressure and total years smoked, an additional variable was created for each feature to indicate if a value was within 2 or 3 s.d. Using the provided diagnosis data from each site and CDR test scores, a harmonized Clinical Diagnosis variable was created categorizing each participant into one of four categories: Cognitively Normal (CN) (CDR = 0 or confirmed recruited control participant), MCI (CDR = 0.5), Dementia (CDR ≥ 1 or a confirmed diagnosis of AD or FTD) or Other Neurodegenerative Disease (a confirmed case of PD or ALS). For participants with no formal diagnosis information or CDR reported, cognitive test scores (MMSE or MoCA) were used to categorize an individual's cognitive impairment as Not Impaired (MMSE ≥ 24 and MoCA ≥ 17) or Impaired (MMSE < 24 and MoCA < 17) (Supplementary Tables 2 and 3). Datasets were merged using unique participant identifiers, followed by quality control checks to rectify inconsistencies. Finally, validation was performed by comparing results from the integrated dataset with original study findings. Each of the three vignettes presented here was conducted by separate workstreams within the GNPC, each using distinct criteria and methodological frameworks tailored to their specific research questions. These analyses were intentionally designed to highlight the breadth of analytic approaches enabled by the GNPC dataset—from disease-specific differential abundance and transdiagnostic clinical severity modeling to biological aging clocks and genotype-based signatures. As such, they serve as illustrative examples rather than a unified analytic pipeline, showcasing the flexibility and depth of the GNPC V1 resource for diverse scientific inquiries. For AD, PD, FTD and ALS, we used protein data from each participant's first available plasma sample, based on clinical data from the cohorts. To reduce diagnostic ambiguity, we excluded participants with conflicting or overlapping clinical labels. This included individuals labeled with both AD and mild cognitive impairment (MCI)/subjective cognitive impairment (SCI) (n = 271) and those assigned two distinct neurodegenerative disease diagnoses (n = 90). We opted to include only individuals diagnosed with AD at the dementia stage, thus excluding those in prodromal stages. CN Control participants were included if they were explicitly labeled as Controls in their respective cohorts and/or had a CDR score of 0. We also removed participants missing age or sex data (n = 54). Protein aptamer abundance levels (relative fluorescence units) were log2 transformed prior to analysis. Extreme outliers, defined as values more than 5 s.d. To ensure biological relevance, only aptamers targeting human proteins were retained for analysis, resulting in a final analytic set of 7,289 unique aptamers (Supplementary Table 5). Given the inherent heterogeneity across contributing GNPC cohorts, all differential abundance analyses were first conducted within each cohort. For each neurodegenerative disease, Patient samples were compared to Control samples using linear regression with the protein used as the dependent variable and disease diagnosis as the independent variable, with age and sex included as covariates. Only cohorts with at least five patients in each diagnostic category (AD, PD, FTD or ALS) were analyzed. In the few cohorts that lacked internal control samples (for example, Cohorts R and G for AD, Cohorts R and T for PD and Cohort S for ALS), patient data were compared to pooled Controls from the remaining cohorts. See Supplementary Table 3 for a breakdown of Controls and Patients across cohorts. After estimating effect sizes within each cohort, we conducted fixed-effects meta-analyses to identify proteins with reproducible disease associations across cohorts, thereby providing internal replication of top signals. For each aptamer, a weighted average of effect sizes was calculated, taking into account the sample size of patients at each cohort, and meta-analytic Pvalues were computed using the weighted z-score method via the ‘metapro R' package. These Pvalues were then adjusted for multiple comparisons using both FDR and Bonferroni correction (separately). To further assess reproducibility, we calculated the number of cohorts in which each aptamer was significantly differentially abundant (FDR < 0.05) as well as the consistency of directionality (that is, the number of cohorts showing concordant upregulation or downregulation for a given target.) Proteins identified in the meta-analytic differential abundance analysis were filtered into gene set enrichment analysis using the R libraries ‘ReactomePA' and ‘clusterProfiler'. Proteins were selected based on Bonferroni cutoffs where possible (AD and PD comparisons). For FTD and ALS, where statistical power was more limited, we applied a less stringent threshold of FDR < 0.05. Enrichment analysis was performed with the Reactome library against a background universe consisting of 6,404 unique human proteins measured on the SomaScan 7K platform. Independent of diagnostic category, we also sought to identify a plasma proteomic signature of clinical severity across the GNPC disease continuum by leveraging the subset of Patients with global CDR scales (N = 6,187) and cognitive test scores (N = 5,969). A proteome-wide association analysis for CDR global scores was conducted by age (linear and quadratic), sex, their interactions and smoking/alcohol status. Significant proteins (FDR < 0.05) were used to construct a multivariate protein signature via LASSO regression, with data split into 70% training and 30% test sets. Diagnosis-stratified models also tested the multivariate protein signature's ability to track cognitive severity across each clinical condition. To demonstrate robustness of prediction to orthogonal clinical measures, the CDR-derived protein signature was modeled as a predictor of cognitive performance using cross-walked, harmonized MoCA/MMSE scores (0–30 scale). A series of recent studies has generated compelling evidence for the use of molecular ‘clocks' to estimate biological aging at the organismal and organ-specific level using plasma proteomics. Building upon this foundation, we applied validated organ age models to the GNPC dataset to evaluate disease-specific patterns of accelerated organ aging32. This analysis was restricted to nine GNPC cohorts that included both Patients and CN Control participants, as control data were required to compute normative organ aging estimates. Protein levels from the SomaScan version 4.1 platform were normalized to match the version 4.0 reference using internal reference-based scaling methods, enabling consistent application of the organ aging models across assays. Organ age estimates were computed for brain, liver, kidney, muscle, adipose, immune, lung, intestine, artery, pancreas and heart tissues, using established organ aging models that use proteins specific to each organ. Additionally, whole-organism (‘organismal') age estimates, using only proteins common across organs, and ‘conventional' age estimates, using all measured proteins regardless of tissue enrichment, were also calculated. The organ age gap was derived as the difference between predicted age and the cohort-specific locally weighted scatterplot smoothing (LOWESS) regression estimate for individuals with a normal cognitive clinical diagnosis (Supplementary Table 9). The APOE ε4 allele is the leading genetic risk factor of late-onset AD, yet its high frequency in patients with AD complicates efforts to disentangle APOE from AD-related proteomic changes. Participants were selected from GNPC cohorts based on strict diagnostic and cognitive performance criteria to ensure well-defined AD and Control groups. We first identified individuals labeled with an AD dementia diagnosis or recruited as Controls. Participants with multiple neurodegenerative disease diagnoses were excluded. Patients with AD dementia were required to have either MMSE < 24 or MoCA <17 and CDR ≥ 1, to ensure alignment with established diagnostic thresholds for dementia. CN participants did not meet criteria for any clinical diagnosis (for example, AD, MCI, FTD, ALS and PD) and had CDR = 0 and, at a minimum, cognitive test scores above a dementia threshold (MMSE ≥ 24 and/or MoCA ≥ 17) (Supplementary Table 10). After applying the above criteria, a total of 3,934 participants were identified for further analysis. Consistent with differential abundance analyses (see Vignette 1 methods), log2-transformed aptamer values that deviated more than 5 s.d. from the mean were defined as outliers and excluded. Proteins associated with APOE ε4 carriers (carrier versus non-carrier) were identified using linear regression models, adjusting for AD dementia diagnosis (CN versus AD dementia diagnosis), age, sex, contributing cohort code and mean overall protein level (Supplementary Table 11). Multiple testing correction was performed using FDR adjustment (Benjamini–Hochberg method, α = 0.05). Residualized protein values were used for visualization. Proteomic data were split into training (70%) and test (30%) sets, with standardization performed independently within each set. Model performance was stratified by CN and AD as well as FTD, PD and Parkinsonʼs disease dementia. First-order functional networks were built from proteins of interest identified by the CART model. In addition, brain cell subtype-specific enrichment analysis was performed using single-nuclei brain RNA sequencing data from the Human Protein Atlas. To explore proteins influenced independently by APOE ε4 carriership and AD diagnosis, two additional comparisons were performed. First, proteins associated with APOE ε4 in cognitively unimpaired individuals were identified. Multiple testing correction was performed using FDR adjustment (Benjamini–Hochberg method, α = 0.05). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The harmonized GNPC data used to generate these findings were provided to consortium members in June 2024 and will be made available for public request by the AD Data Initiative in July 2025. All custom code and data that were employed for this paper are saved within the GNPC AD Workbench. All custom code central to the findings in this paper will be made available outside of private workspaces upon publication. Huang, L., Kuan, Y., Lin, H.-W. & Hu, C. Clinical trials of new drugs for Alzheimer disease: a 2020–2023 update. Miller, R. G., Mitchell, J. D., Lyon, M. & Moore, D. H. Riluzole for amyotrophic lateral sclerosis (ALS)/motor neuron disease (MND). Boxer, A. L. & Sperling, R. Accelerating Alzheimer's therapeutic development: the past and future of clinical trials. Cummings, J. et al. Alzheimer's disease drug development pipeline: 2023. van der Flier, W. M., de Vugt, M. E., Smets, E. M. A., Blom, M. & Teunissen, C. E. Towards a future where Alzheimer's disease pathology is stopped before the onset of dementia. Alzheimer's Disease Sequencing Project (ADSP) Data Discovery Portal (National Institute on Aging Genetics of Alzheimer's Disease Data Storage Site (NIAGADS), accessed 2 July 2025); https://adsp.niagads.org/ Accelerating Medicines Partnership® Program for Alzheimer's Disease (AMP® AD) (National Institute on Aging, US Department of Health and Human Services, National Institutes of Health, accessed 2 July 2025); https://www.nia.nih.gov/research/amp-ad PPMI: A Landmark Study to Identify Biomarkers of Parkinson's Disease Progression (Parkinson's Progression Markers Initiative, 2 July 2025); https://www.ppmi-info.org/ ARC Data Commons (ALS Therapy Development Institute, 2025); https://www.als.net/arc/data-commons/ Farinas, A. et al. Disruption of the cerebrospinal fluid–plasma protein balance in cognitive impairment and aging. Shvetcov, A. et al. APOE ε4 carriers share immune-related proteomic changes across neurodegenerative diseases. The pursuit of approaches to federate data to accelerate Alzheimer's disease and related dementia research: GAAIN, DPUK, and ADDI. Oh, H. S.-H. et al. A cerebrospinal fluid synaptic protein biomarker for prediction of cognitive resilience versus decline in Alzheimer's disease. Large-scale plasma proteomic profiling unveils novel diagnostic biomarkers and pathways for Alzheimer's disease. Frick, E. A. et al. Serum proteomics reveal APOE-ε4-dependent and APOE-ε4-independent protein signatures in Alzheimer's disease. Uhlen, M. et al. A genome-wide transcriptomic analysis of protein-coding genes in human blood cells. Plenge, R. M., Scolnick, E. M. & Altshuler, D. Validating therapeutic targets through human genetics. Minikel, E. V. & Nelson, M. R. Human genetic evidence enriched for side effects of approved drugs. The neuronal pentraxin Nptx2 regulates complement activity and restrains microglia-mediated synapse loss in neurodegeneration. Oh, H. Organ aging signatures in the plasma proteome track health and disease. Shvetcov, A. et al. Proteome profiling of cerebrospinal fluid using machine learning shows a unique protein signature associated with APOE4 genotype. We thank the participants, their families as well as the cohorts, institutions and their dedicated staff who helped create this consortium. Additional funding support was provided by the National Institutes of Health (NIH)-NIA for Frontotemporal Dementia: Genes, Images, and Emotions: P01 AG019724 and the UCSF Alzheimer's Disease Research Center: P30 AG062422. Samples from the National Centralized Repository for Alzheimer's Disease and Related Dementias (NCRAD), which receives government support under a cooperative agreement grant (U24 AG21886) awarded by the NIA, were used in this study. The ALLFTD proteomic data included in this paper were funded by a grant from Gates Ventures. The ALFA proteomic data included in this paper were funded by a grant from Gates Ventures. Additionally, ZonMw Memorabel program project ‘PRODIA', as part of the Deltaplan Dementie, and 2bike4alzheimer (Alzheimer Nederland) are acknowledged for funding this study for the generation of proteomics data for Amsterdam UMC. The baseline ABC-PD study (study period 2014–2017) was funded by a pre-competitive grant from Janssen Research and Development, a division of Janssen Pharmaceutica N.V. Fox Foundation for Parkinson's Research (grant no. Neither study sponsor was involved in collection, analysis and interpretation of data, writing of the report or decision to submit the manuscript for publication. Janssen Research and Development was only involved in study design of the baseline visit. The proteomic data included in this paper were funded by a grant from Gates Ventures. Additionally, this work was supported by the SciLifeLab & Wallenberg Data Driven Life Science Program (grant no. The Emory ADRC participation was supported by the Emory Goizueta Alzheimer's Research Disease Center (P50AG025688) and grants from Gates Ventures for the generation of proteomics data, and data collection was supported through funding by NIA grants R01AG053960, R01AG057911, R01AG061800, RF1AG057471, RF1AG057470, R01AG061800, R01AG057911 and R01AG057339. The proteomic data included in this paper were funded by a grant from Gates Ventures. Additionally, this work was supported by access to equipment made possible by the Hope Center for Neurological Disorders, the Neurogenomics and Informatics Center (https://neurogenomics.wustl.edu/) and the Departments of Neurology and Psychiatry at Washington University School of Medicine. A full list of members and their affiliations appears in the Supplementary Information. NeuroGenomics and Informatics Center, Washington University School of Medicine, St. Louis, MO, USA Barcelonaβeta Brain Research Center (BBRC), Pasqual Maragall Foundation, Barcelona, Spain Hospital del Mar Research Institute, Barcelona, Spain Department of Neurological Sciences, Rush Alzheimer's Disease Center, Chicago, IL, USA Alexa Pichet Binette, Oskar Hansson, Lina Lu & Niklas Mattsson-Carlgren Department of Physiology and Pharmacology, Université de Montréal, Montreal, Quebec, Canada Indiana Alzheimer's Disease Research Center, Indianapolis, IN, USA Amelia Farinas, Hamilton Se-Hwee Oh & Tony Wyss-Coray Amelia Farinas, Hamilton Se-Hwee Oh & Tony Wyss-Coray Translational Gerontology Branch, National Institute on Aging, Bethesda, MD, USA Emory University School of Medicine, Atlanta, GA, USA Biomedical Research Networking Centre in Neurodegenerative Diseases (CIBERNED), National Institute of Health Carlos III, Madrid, Spain Glenn Biggs Institute for Alzheimer's & Neurodegenerative Diseases and Department of Microbiology, Immunology and Molecular Genetics, Long School of Medicine, The University of Texas Health Science Center, San Antonio, TX, USA Neurodegenerative Diseases, Hertie Institute for Clinical Brain Research, Tübingen, Germany IB Hochschule für Gesundheit und Soziales, Standort Stuttgart, Germany Memory Clinic, Skåne University Hospital, Malmö, Sweden Department of Neurology, Mayo Clinic, Rochester, MN, USA Robert Packard Center for ALS Research, Johns Hopkins University, Baltimore, MD, USA Department of Neurology, Hospital del Mar, Barcelona, Spain Department of Human Genetics, Radboud University Medical Center, Nijmegen, The Netherlands Laboratory of Behavioral Neuroscience, National Institute on Aging, Bethesda, MD, USA advised on data harmonization across the multiple disease types. All authors contributed to review and refinement of the manuscript. He has been a consultant for Alchemab, Alector, Alexion, Amylyx, Arrowhead, Arvinas, Eli Lilly, Muna, Neurocrine, Ono, Oscotec, Pfizer, Switch, Transposon and UnlearnAI. has received research support from GSK and Eisai. has a patent pending for compounds and methods targeting human tau. L.F. has given unpaid seminars and/or webinars sponsored or co-sponsored by SomaLogic. has received grants from National Institute on Aging and the state of Arizona; receives philanthropic funding from the Banner Alzheimer's Foundation, Sun Health Foundation and Roche/Roche Diagnostics; receives personal fees from Alkahest, Alzheon, Aural Analytics, Denali, Green Valley, MagQ, Takeda/Zinfandel and United Neuroscience; has since submission of manuscript become a cofounder of ALZpath, which aims to further develop P-tau217 and fluid biomarkers and advance their use in research, drug development and clinical settings; holds a patent owned by Banner Health for a strategy to use biomarkers to accelerate evaluation of Alzheimer prevention therapies; and is a principal investigator of prevention trials that include research agreements with Genentech/Roche and Novartis/Amgen, PET studies that include research agreements with Avid/Lilly and several NIH and Foundation-supported research studies. Nature Medicine thanks Boris Decourt, Jin-Tai Yu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Jerome Staal, Saheli Sadanand and Liam Messin in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03835-z'>APOE ε4 carriers share immune-related proteomic changes across neurodegenerative diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 15:48:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The APOE ε4 genetic variant is the strongest genetic risk factor for late-onset Alzheimer's disease (AD) and is increasingly being implicated in other neurodegenerative diseases. Using the Global Neurodegeneration Proteomics Consortium SomaScan dataset covering 1,346 cerebrospinal fluid (CSF) and 9,924 plasma samples, we used machine learning-based proteome profiling to identify an APOE ε4 proteomic signature shared across individuals with AD, frontotemporal dementia (FTD), Parkinson's disease dementia (PDD), Parkinson's disease (PD), amyotrophic lateral sclerosis (ALS) and nonimpaired controls. This signature was enriched in pro-inflammatory immune and infection pathways as well as immune cells, including monocytes, T cells and natural killer cells. Analysis of the dorsolateral prefrontal cortex proteome for 262 donors from the Accelerating Medicines Partnership for AD UPenn Proteomics Study revealed a consistent APOE ε4 phenotype, independent of neurodegenerative pathology, including amyloid-β tau and gliosis for all diseases, as well as TDP-43 in ALS and FTD cases, and α-synuclein in PD and PDD cases. While systemic proteomic changes were consistent across APOE ε4 carriers, their relationship with clinical and lifestyle factors, such as hypertension and smoking, varied by disease. These findings suggest APOE ε4 confers a systemic biological vulnerability that is necessary but not sufficient for neurodegeneration, emphasizing the need to consider gene–environment interactions. Overall, our study reveals a conserved APOE ε4-associated pro-inflammatory immune signature persistent across the brain, CSF and plasma irrespective of neurodegenerative disease, highlighting a fundamental, disease-independent biological vulnerability to neurodegeneration. This work reframes APOE ε4 as a pleiotropic immune modulator rather than an AD-specific risk gene, providing a foundation for precision biomarker development and early intervention strategies across neurodegenerative diseases. The ε4 variant of the apolipoprotein E (APOE ε4) gene is well-established as the largest genetic risk factor of late-onset Alzheimer's disease (AD)1. Growing evidence, however, indicates that APOE ε4 carriage may also have a role in other age-associated neurodegenerative diseases. Studies have linked APOE ε4 to increased risk and lower age of onset of frontotemporal dementia (FTD)2,3,4,5,6, Parkinson's disease (PD)5,7,8 and amyotrophic lateral sclerosis (ALS)9,10. APOE ε4 is also linked to a faster rate of cognitive decline and poor cognition in PD, increasing the risk of PD dementia (PDD)11,12,13,14,15,16. Despite the deleterious impact of APOE ε4, little is known about the biological mechanisms underlying this effect and if, or how, it changes across the different neurodegenerative diseases. We recently showed that APOE ε4 carriers, irrespective of cognitive status in AD and mild cognitive impairment, had the same proteomic signature in the cerebrospinal fluid (CSF) associated with a pro-inflammatory immune molecular phenotype17. However, whether this extends to other neurodegenerative diseases is unknown. To identify systemic proteomic changes associated with APOE ε4 carriers who develop neurodegenerative diseases, we used the Global Neurodegeneration Proteomics Consortium (GNPC) dataset. Here the plasma and CSF proteome were assessed using the SomaScan assay for 11,270 APOE ε4 carriers and noncarriers with AD, FTD, PD, PDD, ALS and nonimpaired controls. Using supervised machine learning, we identified and characterized an APOE ε4 proteome signature across the CSF and plasma. We then confirmed whether the same APOE ε4-enriched pathways in the periphery were mirrored in the brains of carriers and noncarriers. The dorsolateral prefrontal cortex (dlPFC) proteome of 262 AD, FTD, PDD, PD, ALS and nonimpaired control donors from the Accelerating Medicines Partnership for AD (AMP-AD) UPenn Proteomics Study was measured using label-free mass spectrometry (MS). In these samples, we also examined postmortem histopathological markers, including the presence of amyloid-β plaques, tau neurofibrillary tangles, gliosis and angiopathy. In FTD and ALS cases, we also examined TDP-43, while in PD and PDD, we examined α-synuclein. Lastly, to assess potential interactions between proteins in the APOE ε4 signature and environmental variables across the diseases, we performed a correlation network analysis between proteins and 18 clinical and lifestyle variables collected in the GNPC dataset, such as hypertension, smoking, and diabetes (Supplementary Table 1 and Fig. a, Study design using the GNPC and AMP-AD UPenn Proteomics Study cohorts for identifying and characterizing systemic proteome changes in APOE ε4 carriers. Panel a is created with BioRender.com. b, PCA of all 6,340 measured CSF proteins showing no clear clustering. c, PCA of 229 APOE ε4 CSF proteins identified using mutual information that shows clustering based on the presence or absence of APOE ε4 allele rather than specific neurodegenerative disease. d, PCA of 229 APOE ε4 CSF proteins showing that clustering is based on the specific APOE genotype and number of APOE ε4 alleles (Supplementary Table 1 lists the distribution of APOE ε4 cases). e, Heatmap visualizing the upregulation (red) and downregulation (blue) of proteins within the APOE ε4 CSF proteome signature of 229 proteins, which shows distinctions based on the presence or absence of an APOE ε4 allele rather than disease. f, Supervised machine learning modeling using CART showing mean AUC ± s.d. across fivefold repeated five times. Models were trained and validated on a 70% training dataset and tested using a 30% withheld testing dataset. g, Functional enrichment analysis of PANTHER biological processes enriched for APOE ε44 CSF proteins showing the most significant (FDR = 9.34 × 10−13) enrichment for viral processes. h, Given the most significant enriched biological process was viral processes, we performed a functional enrichment analysis of KEGG immune-related pathways enriched for APOE ε4 CSF proteins, showing significant (FDR < 0.05) enrichment for immune, infection and pro-inflammatory pathways. i, Immune cell-type-specific enrichment analysis of APOE ε4 CSF proteins showing involvement across the innate, adaptive and innate-like T cells and lymphoid cells (mixed). j, Liver cell-type-specific enrichment analysis of APOE ε4 CSF proteins showing involvement across parenchymal and immune cells. Cell-type-specific enrichments are based on single-cell RNA-sequencing data from the Human Protein Atlas19. Plot shows min–max scaling of protein-coding transcripts per million for each identified protein in the APOE ε4 CSF signature. AUC, area under the curve; NI, nonimpaired controls; FcγR, Fc gamma R; FcεRI, Fc epsilon RI; NOD, nucleotide oligomerization domain; RIG, retinoic acid-inducible gene; HTLV-1, human T cell leukemia virus type 1. CSF proteome profiling reveals a distinct signature in APOE ε4 carriers We used SomaScan (6,340 proteins measured per sample) proteomic data from the GNPC dataset to profile the CSF proteome of 526 AD, 247 PD and 573 nonimpaired control individuals (Supplementary Table 1). An initial principal component analysis (PCA) revealed that there was no clustering across all 6,340 proteins (Fig. We used mutual information (>0.01) to identify APOE ε4 proteins. Unlike traditional correlation coefficients, which capture only linear relationships, mutual information can detect both linear and nonlinear associations. This makes it well suited for complex high-dimensional biological data, where relationships between variables may not follow simple patterns. Using this, we identified 229 CSF proteins that were APOE ε4 associated in nonimpaired controls (Methods; Supplementary Table 2 and Extended Data Fig. A subsequent PCA showed that these proteins led to distinct clustering of groups based on APOE genotype (Fig. 1c) and the number of APOE ε4 alleles (Fig. This effect was further visualized using a heatmap showing that APOE ε4 proteins were upregulated or downregulated based on genotype but not disease (Fig. Many of these APOE ε4 proteins were also identified in our earlier work profiling the CSF proteome of APOE ε4 carriers with mild cognitive impairment and AD from the AD Neuroimaging Initiative cohort17 further highlighting the robustness and generalizability of our finding. Using classification and regression trees (CART) modeling, we showed that these 229 CSF proteins were able to reliably (performance metrics > 0.95) predict APOE ε4 carriers from noncarriers across AD and PD (Extended Data Table 1 and Fig. We also found that there were no sex differences in the ability of our identified proteins to distinguish between APOE ε4 carriers and noncarriers (Extended Data Table 1 and Fig. To determine if the APOE ε4 signature might be due to differences in amyloid-β levels between carriers and noncarriers, we examined CSF amyloid-β A4 protein levels measured in the SomaScan assay. A Wilcoxon test indicated that there were no significant differences (P > 0.05) in CSF amyloid-β A4 protein levels between APOE ε4 carriers and noncarriers with AD, PD or nonimpaired controls (Extended Data Fig. A functional enrichment analysis in the protein analysis through evolutionary relationships (PANTHER) database18 of the CSF APOE ε4 proteins revealed significant (false discovery rate (FDR) < 0.05) enrichment for viral processes, apoptosis, rhythmicity, cellular processes, protein phosphorylation and folding and RNA/DNA processes (Fig. Given that the most significant enrichment was observed for viral processes, we also performed enrichment analysis for immune pathway-specific processes. Immune-specific Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways revealed that APOE ε4 proteins were enriched in numerous infection-related pathways, including hepatitis, herpes, measles, Epstein–Barr virus (EBV), and influenza A. There was also significant enrichment for T cell, B cell and inflammatory signaling cascades, including Toll‑like receptor (TLR), tumor necrosis factor (TNF), interleukin 17 (IL-17), JAK/STAT and nuclear factor-κB (NF-κB; Fig. Using single-cell RNA-sequencing data from the Human Protein Atlas19, we performed an immune cell subtype enrichment analysis on APOE ε4 proteins. Across innate immune cells, APOE ε4 proteins were the most enriched in nonclassical and intermediate monocytes. In adaptive immune cells, memory CD8 T cells were the most enriched for APOE ε4 proteins, followed by Tregs and memory CD4 T cells. Given that we found enrichment for hepatitis KEGG pathways, we also performed a cell-type-specific enrichment analysis in the liver. Across parenchymal cells, APOE ε4 proteins were the most enriched in hepatocytes and Kupffer cells, in line with both cell types being the primary producers of the APOE4 proteoform in the liver20 (Fig. These results show that APOE ε4 carriers have a distinct CSF proteomic signature characterized by enriched viral processes and pro-inflammatory immune pathways and cells. We find enrichment across hepatocytes and Kupffer cells in the liver, further implicating APOE4 proteoform synthesis sites in the periphery. This finding may also be reflective of brain-liver signaling and liver responses to neuroinflammation21. Notably, these changes were independent of neurodegenerative disease and sex, suggesting that APOE ε4 carriers share a common molecular phenotype. We next sought to determine whether the APOE ε4 CSF proteome changes observed were also reflected in the plasma. Further leveraging the GNPC dataset, we performed plasma proteome profiling of 2,929 AD, 75 FTD, 169 PDD, 422 PD, 230 ALS and 6,099 nonimpaired control individuals with and without an APOE ε4 allele (Supplementary Table 1). A PCA of all 6,340 proteins revealed no group clustering (Fig. Using mutual information, we identified 58 plasma proteins in nonimpaired controls (Supplementary Table 4 and Extended Data Fig. 2a) that were associated with APOE genotype (Fig. 2b) and led to clustering based on the number of APOE ε4 alleles (Fig. 2c) rather than by neurodegenerative disease. A heatmap also revealed that the 58 APOE ε4 plasma changes were upregulated or downregulated based on genotype (Fig. Two of these proteins, TBCA and LRRN1, were also identified in the serum of healthy APOE ε4 centenarians22, providing further external validation of our finding and the importance of these proteins in APOE ε4 carriers. a, PCA of all 6,340 measured plasma proteins showing no clear clustering. b, PCA of 58 APOE ε4 plasma proteins identified using mutual information that shows clustering based on the presence or absence of APOE ε4 allele rather than specific neurodegenerative disease. c, PCA of 58 APOE ε4 plasma proteins showing that clustering is based on the specific APOE genotype and number of APOE ε4 alleles (Supplementary Table 1 lists the distribution of APOE ε4 cases). d, Heatmap visualizing the upregulation (red) and downregulation (blue) of proteins within the APOE ε4 plasma proteome signature of 58 proteins that shows distinctions based on the presence or absence of an APOE ε4 allele rather than disease. e, Supervised machine learning modeling using CART showing mean AUC ± s.d. across fivefold repeated five times. Models were trained and validated on a 70% training dataset and tested using a 30% withheld testing dataset. f, Functional enrichment analysis of PANTHER biological processes enriched for APOE ε4 plasma proteins showing the most significant (FDR = 1.31 × 10−26) enrichment for viral processes. g, Given the most significant enriched biological process was viral processes, we performed a functional enrichment analysis of KEGG immune-related pathways enriched for APOE ε4 plasma proteins. This showed significant (FDR < 0.05) enrichment for immune, infection and pro-inflammatory pathways. h, Immune cell-type-specific enrichment analysis of APOE ε4 plasma proteins showing involvement across the innate, adaptive and innate-like T cells and lymphoid cells (mixed). i, Liver cell-type-specific enrichment analysis of APOE ε4 CSF proteins showing involvement across parenchymal and immune cells. Cell-type-specific enrichments are based on single-cell RNA-sequencing data from the Human Protein Atlas19. Plot shows min–max scaling of protein-coding transcripts per million for each identified protein in the APOE ε4 plasma signature. CART models using these 58 proteins as predictors showed a strong ability to differentiate between APOE ε4 carriers and noncarriers across each of the neurodegenerative disease groups (performance metrics > 0.85; Extended Data Table 2). CART modeling revealed that there were no sex differences in the APOE ε4 plasma signature (Extended Data Table 2 and Fig. We extended this to show that there were no effects of race. Here we trained our models on White individuals and tested them using proteomic data from either Black/African American or American Indian/Alaskan Native individuals. In both cases, our models were reliably able to predict APOE ε4 carriers from noncarriers (Extended Data Table 2 and Fig. We also compared amyloid-β A4 protein levels between APOE ε4 carriers and noncarriers across neurodegenerative diseases to determine if the plasma APOE ε4 signature was related to amyloid-β pathology. A Wilcoxon test revealed that there were no significant differences in plasma amyloid-β A4 protein between APOE ε4 carriers and noncarriers (Extended Data Fig. As in the CSF, viral processes were the most significantly enriched biological process (Fig. This was further supported by similar KEGG immune and infection pathway enrichments, including EBV and hepatitis (Fig. There was also significant enrichment for inflammatory and cytokine signaling pathways, including TNF, IL-17, TLR and NF-κB (Fig. Among immune cell subtypes, nonclassical and intermediate monocytes were implicated in APOE ε4 proteins, as seen in the CSF. Unlike the CSF, however, basophils were enriched for APOE ε4 proteins. Of adaptive immune cell subtypes, plasma APOE ε4 proteins were enriched for memory CD8 T cells, Tregs and naive CD8 T cells. NK cells and γδ T cells were also associated with APOE ε4 (Fig. In the liver, we found cell-type-specific enrichment primarily for Kupffer cells and T cells and, unlike the CSF, very little enrichment for hepatocytes. Our results demonstrate that genotype-specific proteomic changes observed in the CSF are also reflected in the plasma of APOE ε4 carriers and noncarriers, consistently indicating pro-inflammatory immune dysregulation across multiple pathways and immune cell populations. The APOE ε4 proteomic signature remained independent of neurodegenerative disease status and sex, with CART models showing strong predictive power for carrier status. We further showed that this signature generalizes across racial groups, underscoring its robustness and broad applicability. Unlike in the CSF, plasma APOE ε4 proteins were not enriched in hepatocytes but were significantly enriched in Kupffer and T cells, suggesting that peripheral immune activation is distinct from central nervous system (CNS) immune-metabolic signaling along the liver-brain axis in APOE ε4 carriers. We then sought to determine whether the proteomic changes observed in the periphery were reflective of central changes in APOE ε4 carriers and to further validate our findings from the GNPC cohort. To do this, we leveraged label-free MS proteomic and postmortem histopathological data from the AMP-AD UPenn Proteomics study for the dlPFC of 49 AD, 31 FTD, 47 PDD, 33 PD, 55 ALS and 47 nonimpaired individual donors (Supplementary Table 6). SomaScan and label-free MS proteomic assays had different coverage of specific proteins; therefore, we focused on confirming enrichment for biological processes and pathways across the CSF, plasma, and dlPFC of carriers and noncarriers. We again used mutual information to identify APOE ε4 proteins from the dlPFC within each group independently. Across all neurodegenerative disease groups, we identified 248 APOE ε4 proteins (Supplementary Table 7). This was confirmed with PCAs showing clustering based on APOE genotype (Extended Data Fig. Functional enrichment analyses revealed that three of the main biological processes (viral processes, negative regulation of apoptosis, and protein folding) identified in the CSF and plasma were also significantly enriched in the dlPFC of APOE ε4 carriers across all neurodegenerative diseases (Fig. Further, four of the most significantly enriched KEGG immune pathways in the CSF and plasma were also identified in APOE ε4 carriers in a disease-independent manner, including EBV, hepatitis B, viral carcinogenesis and pathogenic Escherichia coli infection (Fig. a, Bar plot comparing functional enrichment analysis of overlapping significant (FDR < 0.05) PANTHER biological processes enriched for APOE ε4 proteins in the CSF, plasma and brain (dlPFC). b, Heatmap comparing functional enrichment analysis of overlapping significant (FDR < 0.05) PANTHER biological processes enriched for APOE ε4 proteins in the CSF, plasma and brain (dlPFC). c, Comparative functional enrichment analysis of significant (FDR < 0.05) KEGG immune-related pathways enriched for APOE ε4 proteins in the CSF, plasma and brain (dlPFC). d, Comparative functional enrichment analysis of overlapping significant (FDR < 0.05) KEGG immune-related pathways enriched for APOE ε4 proteins in the CSF, plasma and brain (dlPFC). Previous research suggests that APOE ε4 carriers may develop neurodegenerative pathology, including amyloid-β plaques and tau neurofibrillary tangles, even in the absence of symptomatic disease8,23,24,25,26. We therefore sought to determine if the changes seen in APOE ε4 carriers may be due to the presence of APOE ε4-driven brain pathology. There was no relationship between APOE ε4 and tau or thio-S-positive plaques across nonimpaired controls and disease groups (Supplementary Table 9). Although none of the neurodegenerative disease groups showed a relationship between APOE ε4 and angiopathy, there was a small statistically significant association found in nonimpaired controls (Supplementary Table 9). Taken together, these findings suggest a relationship between peripheral and central proteomic changes. Both are characterized by pro-inflammatory immune responses, further highlighting that APOE ε4 carriers have systemic immune dysregulation. This demonstrates that the major proteome changes seen in the periphery of APOE ε4 carriers are indeed reflective of ongoing processes in the brain and that peripheral processes may actively contribute to or even drive changes in the brain proteome. Notably, APOE ε4-associated changes are independent of neurodegenerative disease, whether measured simply by diagnosis or by specific pathological changes in the brain. Given that APOE ε4 carriers across neurodegenerative diseases share major underlying systemic proteomic changes, we sought to better understand additional drivers that may be disease specific. To do this, we leveraged the GNPC dataset and a correlation network analysis to identify the relationship between APOE ε4 proteins and demographic, clinical, and lifestyle variables, such as age, blood pressure, and smoking (Supplementary Table 1 and see Extended Data Fig. Of the 40 proteins that overlapped across the CSF and plasma of APOE ε4 carriers, we chose 16 of these that had more than 20 functional connections (Fig. These proteins represent central ‘nodes' in the protein–protein network, suggesting that any changes are more likely to disrupt diverse pathways and functions27. Of note, two of the overlapping proteins, APOE and PHGDH, were differentially expressed in opposite directions across the plasma and CSF. Both proteins were upregulated in the CSF but downregulated in the plasma. As we had a higher number of individuals with plasma samples, thereby improving statistical power, the correlation analysis reflects the decreased APOE and PHGDH levels observed in plasma. a, Venn diagram showing the overlapping APOE ε4 proteins identified in the CSF and plasma. Of the 40 overlapping proteins, the 16 named proteins represent central protein nodes in the protein–protein interaction network, with more than 20 functional connections. Panel a is created with BioRender.com. b, Hierarchical tree showing the unique, neurodegenerative disease-specific relationships between APOE and demographic and clinical variables. c, Categorical heatmap showing the unique, neurodegenerative-disease-specific relationships between the remaining 15 central node APOE ε4 proteins and demographic, lifestyle and clinical (cardiovascular, neurological/psychological, metabolic and other) variables. We first calculated Spearman's rank correlation (continuous variables) or correlation ratio (categorical variables) for APOE and demographic, clinical, and lifestyle variables in nonimpaired, AD, PDD and PD APOE ε4 carriers. Due to low numbers, we were unable to calculate similar correlations for FTD and ALS. Our analyses revealed unique, neurodegenerative disease-specific significant relationships with APOE (Fig. APOE was associated with sex and race in AD, with diabetes in PD, with resting heart rate and chronic obstructive pulmonary disease (COPD) in PDD, and with age, body mass index, and hypertension in nonimpaired controls. A key limitation is the lack of longitudinal data for nonimpaired APOE ε4 carriers, who may later develop neurodegenerative disease. Thus, observed associations with clinical variables may reflect preclinical disease. We extended these correlation analyses to the remaining 15 central node APOE ε4 proteins. This analysis further identified that APOE ε4 central node proteins are significantly correlated (P < 0.05) with demographic, clinical, and lifestyle variables in a disease-specific way in APOE ε4 carriers (Fig. Together, these findings indicate that although all APOE ε4 carriers have the same underlying proteomic signature, specific proteins within this signature are differentially correlated with demographic, lifestyle, and clinical variables. Critically, these correlations were neurodegenerative disease specific. The GNPC dataset represents a substantial advancement in neurodegenerative disease research, by providing a real-world clinical proteomic dataset that comprises over 35,000 (11,270 with APOE genotype) individuals across AD, FTD, PDD, PD, ALS and normal aging from across more than 20 clinical sites in the US, UK and Europe28. This enabled us to ask whether the APOE ε4-associated proteomic signature is shared across multiple neurodegenerative diseases. Our results demonstrate that all APOE ε4 carriers, irrespective of neurodegenerative disease, have a unique proteome signature that extends across the plasma and CSF. Unlike prospectively designed cohorts, the GNPC dataset reflects real-world clinical heterogeneity, highlighting the robustness and generalizability of our findings. This signature is associated with pro-inflammatory immune dysregulation and an enrichment for circulating immune cells, including monocytes, memory CD8 and γδ T cells, Tregs and NK cells. This molecular phenotype extends to the brains of APOE ε4 carriers in a similar disease-independent manner and is not associated with the presence of any disease-specific brain pathology (Fig. Although all APOE ε4 carriers have a systemic immune-related proteome signature, we find that the relationships between proteins within this signature are uniquely associated with demographic, lifestyle, and clinical variables in a neurodegenerative disease-specific manner. Notably, this suggests that although the biological changes associated with APOE ε4 carriage are essential for neurodegeneration, broadly, interactions with underlying biological vulnerability and the environment may be key for driving the pathogenesis of the specific neurodegenerative disease. APOE ε4 carriers across different neurodegenerative diseases share a common systemic proteomic change reflective of pro-inflammatory immune dysregulation. The figure is created with BioRender.com. There is evidence that the APOE ε4 genotype is a modern-day example of antagonistic pleiotropy. In younger adults, APOE ε4 is associated with increased survival and fertility in environments with high levels of infectious disease29,30,31,32. For example, healthy individuals who are APOE ε4 heterozygotes exhibit heightened cytokine release, increased plasma TNF levels, a more pronounced hyperthermic response and an earlier onset of IL-6 production following immune challenge with TLR2/TLR4/TLR5 ligands or lipopolysaccharide33. While this immune response protects younger APOE ε4 carriers from infectious diseases, prolonged states of inflammation and cytokine release are likely deleterious with age34,35. Although this study identified a proteomic signature indicative of a pro-inflammatory phenotype, a limitation is the absence of direct measures of routine inflammatory markers such as C-reactive protein or cytokines. Future studies should incorporate these markers in prospectively designed cohorts to clarify their association with APOE ε4. To date, the biological effects of APOE ε4 carriage have largely only been studied in the context of AD. A notable finding of our study is that the pro-inflammatory molecular phenotype associated with APOE ε4 extends to individuals with other neurodegenerative diseases, including FTD, PDD, PD and ALS. This raises two key considerations. First, our findings underscore the need to shift focus from the continued identification of genetic risk loci via genome-wide association studies toward functional characterization of established variants36. Notably, the absence of a statistically significant association between a variant and a specific disease phenotype does not preclude biological relevance. Despite APOE ε4 being overrepresented in the AD cohort, we observe a consistent molecular signature associated with APOE ε4 across multiple neurodegenerative diseases, highlighting an underrecognized role for this variant beyond AD that may have been overlooked due to its historically strong link with AD risk. Second, our data support reconceptualizing APOE ε4 not only as a disease-specific risk factor but also as a broader susceptibility allele contributing to shared pathogenic mechanisms across neurodegenerative diseases. It remains unclear, however, why APOE ε4 is more strongly associated with AD in terms of prevalence despite exhibiting systemic biological effects across neurodegenerative diseases. One possibility is that interactions between APOE ε4 and additional age-related, environmental, or comorbid factors may selectively amplify neurodegenerative pathways characteristic of AD. Another possibility is that CNS-specific vulnerabilities or cellular contexts (for example, in hippocampal circuits or dopaminergic neurons) modulate how the APOE ε4 inflammatory phenotype manifests clinically. Thus, while APOE ε4 confers a shared biological susceptibility, disease expression likely depends on a combination of genetic background, cellular context(s), and lifetime exposures. Large-scale integrative efforts, such as the GNPC, which harmonize data across distinct disease cohorts into unified datasets, provide a powerful framework for advancing this line of inquiry. By enabling cross-disease comparisons, such efforts may help delineate modifiers that influence why some APOE ε4 carriers develop AD while others remain healthy or develop different neurodegenerative diseases. This has important implications for prognosis and risk stratification in midlife individuals who carry APOE ε4. A limitation of our study is the absence of validated biomarkers (for example, CSF p-tau217) to confirm clinical diagnoses, which reflects both the nature of the GNPC dataset and global heterogeneity in clinical practice28. However, several features mitigate concerns regarding potential misdiagnosis. First, the APOE ε4 signature was derived from nonimpaired individuals. While it is possible that a minority of these nonimpaired individuals harbored asymptomatic pathology (for example, asymptomatic AD), the majority would be free of overt disease, reducing the likelihood of confounding results. Second, the consistent presence of the signature across all APOE ε4 carriers, irrespective of clinical diagnosis, supports its generalizability and suggests it reflects a broader APOE ε4-related biological phenotype rather than disease-specific changes. Finally, we validate our findings using postmortem brain proteomics and histopathology, where diagnostic certainty is highest. Here APOE ε4 status was not associated with hallmark pathologies, including amyloid-β, tau, TDP-43 or α-synuclein, across respective disease groups. This postmortem validation reinforces the robustness of our findings. Future studies would benefit from prospective cohorts incorporating validated CSF or plasma biomarkers to confirm and extend these observations. An unexpected finding of our study was that plasma neurofilament light (NEFL) levels were lower in APOE ε4 carriers across neurodegenerative diseases, despite NEFL's growing recognition as a biomarker of neurodegeneration37. Prior studies have reported conflicting results—some found no association38,39, another reported increased levels in APOE ε4 carriers40, while others using the SomaScan41 and Simoa42 assays observed decreased levels, consistent with our findings. These discrepancies may reflect differences in sample size, with studies reporting decreased NEFL levels generally including larger cohorts (~600 to 5,000 participants, and 9,924 in our study). Alternatively, APOE ε4-related blood–brain barrier (BBB) or metabolic dysfunction may alter peripheral clearance of NEFL and affect its plasma or CSF concentration. These results raise important questions about the reliability of NEFL as a stand-alone biomarker and underscore the need for mechanistic studies and a shift toward precision biomarkers that integrate genetic and environmental context. Notably, brain proteomics performed using label-free MS orthogonally validated findings from SomaScan-based CSF and plasma analyses. While SomaScan aptamer technology offers high-throughput protein quantification, it is relatively insensitive to proteoform diversity, including post-translational modifications43, a limitation when examining signaling pathways and protein networks where such modifications are functionally significant. In contrast, MS enables the detection of broader proteoforms, offering an independent assessment of protein abundance and pathway enrichment. The concordance of APOE ε4-associated protein signatures and enrichment patterns across plasma, CSF and brain proteomic datasets suggests that our findings are not artifacts of platform-specific biases but likely reflect underlying biology. Supporting this interpretation, we observed enrichment of APOE ε4-associated proteins in both hepatocytes and Kupffer cells in the CSF, but only in Kupffer cells and T cells in plasma. This pattern implies that plasma reflects chronic peripheral immune activation, whereas the CSF may capture liver-brain axis-mediated inflammatory signaling in APOE ε4 carriers21. Further experimental studies are warranted to validate these mechanisms. Prospective cohort studies incorporating cross-platform, multi-tissue analyses from the same individuals will be essential to confirm and expand upon these observations. While the exact mechanisms underlying the mirroring of peripheral and central pro-inflammatory states remain unclear, they may involve interactions between pro-inflammatory peripheral immune cells and the BBB. A recent study of individuals with long COVID-19 demonstrated that hyperactive peripheral blood mononuclear cells adhere to the endothelial cells of the BBB, driving inflammation, degradation, and symptoms of brain fog44. In support of this, cognitively healthy APOE ε4 carriers exhibit early markers of BBB dysfunction in the hippocampus and medial temporal lobe45. As the BBB becomes increasingly compromised, it allows the infiltration of blood-derived proteins and pro-inflammatory immune cells into the brain, contributing to exacerbated neuroinflammation and neurodegeneration46. In our study, we identify enriched pro-inflammatory immune cell subpopulations, including nonclassical and intermediate monocytes47,48, memory CD8+ T cells49, γδ T cells50 and NK cells51,52. These immune cell types may represent peripheral blood mononuclear cells that interact with the BBB to promote neurodegeneration in APOE ε4 carriers. Given that APOE ε4 is linked to a shared molecular phenotype across neurodegenerative diseases, this hypothesis is consistent with current understanding. Healthy, nonimpaired APOE ε4 carriers have a systemic pro-inflammatory phenotype17 (and as we have shown here), early signs of BBB disruption45, alterations in brain activity and connectivity53,54, and sleep disturbances55, all of which are known to be risk factors for neurodegenerative diseases, broadly56,57,58. Further supporting this is the recent evidence that vaccinations protect against dementia59,60, including the shingles vaccines Zostavax61 and Shingrix62. Although none of these studies explicitly examined the interactive effects between vaccination status and APOE ε4 genotype, the fact that more than 60% of individuals with dementia are APOE ε4 carriers63 suggests that APOE ε4 may represent a critical modifier of the observed associations, warranting further investigation. Future studies in human disease-relevant models, such as patient stem cell-derived organoids, are needed to further elucidate these mechanisms. If all APOE ε4 carriers share a common underlying biological vulnerability to neurodegeneration, an important question is what determines which specific neurodegenerative disease they develop. This likely reflects complex interactions between genetic risk and environmental exposures64. In our study, proteins within the APOE ε4 signature were differentially correlated with demographic, lifestyle, and clinical variables, in a neurodegenerative disease-specific manner. This suggests that genotype–environment interactions may modulate disease trajectories in APOE ε4 carriers. While limited by the variables consistently collected across the GNPC's global real-world cohorts, our findings highlight meaningful associations that warrant further investigation. Future prospective studies should incorporate more detailed environmental and behavioral measures, such as physical activity, diet, sleep, substance use and immunization history, to better characterize modifiers of neurodegenerative disease risk in APOE ε4 carriers. Given the cross-sectional nature of our study, we cannot infer causality or perform mediation analyses, as clinical variables like hypertension may have arisen after disease onset. One plausible mechanism is that APOE ε4 carriers exhibit heightened inflammatory responses to environmental or pathological stressors, including comorbidities such as hypercholesterolemia and ischemic heart disease, to which they are predisposed65,66,67. This suggests that the pro-inflammatory phenotype of APOE ε4 carriers may be, at least in part, independent of cardiovascular comorbidities. Our findings further support this interpretation. Our PANTHER biological processes enrichment analyses of the APOE ε4 CSF proteome showed no significant enrichment of cardiovascular-related processes. In the plasma, we found only a single enrichment for heart development; however, this was modest relative to the top enrichment of viral processes. These results suggest that additional environmental or pathological insults may act synergistically with a genetically primed pro-inflammatory state to drive neurodegeneration in APOE ε4 carriers. Another interpretation of our data, however, is that individuals with pre-existing neurodegenerative diseases are more likely to develop comorbidities68,69, such as hypertension, which may be a consequence of neurodegenerative processes rather than a precursory event. Longitudinal studies tracking APOE ε4 carriers over time will be essential to disentangle causal pathways and advance personalized prevention strategies. Neurodegenerative disease risk is reflective of a complex polygenic architecture and co-occurring variants in genes, such as TREM2, MAPT, GRN, GBA and LRRK2, may modulate or interact with APOE ε4-associated molecular pathways. For example, joint carriage of APOE ε4 and MAPT mutations resulted in a significantly lower age of FTD onset6. Genome-wide genetic data beyond APOE genotype, however, are not currently available within the GNPC cohort and therefore could not be included in the present study. Despite this, a key strength of our study is the demonstration that a well-established risk variant for one neurodegenerative disease (APOE ε4 in AD) also exerts a conserved molecularsignature across multiple neurodegenerative conditions. These findings provide proteomic evidence that a single risk variant is implicated across the neurodegenerative disease spectrum. This indicates the need for a conceptual shift in the field, moving away from viewing genetic risk variants as disease specific and toward recognizing their potential as shared, pleiotropic modifiers of neurodegenerative vulnerability. Future work, therefore, would greatly benefit from a focus on the role of other genetic variants across the neurodegenerative disease spectrum, even if they are enriched in only one disease. Overall, our study provides a conceptual and translational advance in understanding APOE ε4-mediated risk for neurodegeneration. We identify a conserved, systemic pro-inflammatory immune proteomic signature associated with APOE ε4 across plasma, CSF and brain, irrespective of neurodegenerative disease or pathology. Notably, this signature is also present in asymptomatic individuals, suggesting it precedes clinical symptom onset and reflects an intrinsic biological vulnerability. The ability to detect this signature in plasma, a minimally invasive and clinically accessible biofluid, also offers promising opportunities for blood-based precision biomarkers to identify individuals at risk before symptom onset. Together, these findings demonstrate that APOE ε4 confers an intrinsic biological vulnerability characterized by a chronic pro-inflammatory immune phenotype that is independent of downstream disease processes. While this chronic immune activation may predispose APOE ε4 carriers to neurodegeneration, it is unlikely to be sufficient alone to drive the development of neurodegenerative disease. Instead, the transition to clinical disease likely involves complex interactions between genetic susceptibility and modifiable environmental, lifestyle, and clinical factors. By linking the APOE ε4 signature to such factors, our findings provide actionable insights for early intervention and prevention. They also establish a roadmap for future exploration of gene–gene and gene–environment interactions, including polygenic and epistatic effects. More broadly, our findings reframe APOE ε4 as a pleiotropic immune modulator rather than an AD-specific gene, underscoring the need for cross-disease, genetically informed models of risk stratification and therapy. Finally, this work lays the foundation for future wet laboratory-based mechanistic studies to elucidate how APOE ε4 shapes immune function and neuroinflammatory signaling, a critical step toward developing targeted and individualized immunotherapies for early and preclinical disease. The GNPC cohort represents the largest collection of SomaScan proteomic data for individuals with neurodegenerative diseases and nonimpaired controls sourced from study sites across the USA, UK and Europe. In the present study, we included 11,270 individuals from the GNPC cohort, with 6,672 nonimpaired controls, 3,455 AD, 75 FTD, 169 PDD, 669 PD and 230 ALS individuals. Of these, 4,325 individuals were APOE ε4 carriers (either heterozygous or homozygous) and 6,945 were non-APOE ε4 carriers. All included individuals in the study provided either CSF or plasma samples, but none provided both. Individuals were diagnosed based on diagnostic criteria from each research group, as described in the GNPC cohort summary paper1. Cognitive impairment for AD and PDD patients was further assessed using a clinical dementia rating score of ≥1, Mini-Mental State Exam score of ≤24 and/or Montreal Cognitive Assessment score of ≤23. Most individuals with AD had cognitive impairment scores indicative of mild or moderate AD. We excluded individuals with mild cognitive impairment in the current study due to its heterogeneity as a clinical diagnosis, and because it does not necessarily reflect neurodegenerative pathology. CSF or plasma samples were collected from all participants at a single timepoint along with demographic and clinical variables (Supplementary Table 1). Participants from each of the included study sites in the GNPC cohort provided written informed consent and studies were approved by the relevant institution's ethics committee28. The AMP-AD UPenn Proteomics Study is a cohort of autopsy-collected samples from the dlPFC of 49 AD, 31 FTD (with TDP-43 inclusions), 47 PDD, 33 PD, 55 ALS and 47 nonimpaired individual donors from the University of Pennsylvania School of Medicine Brain Bank (https://www.synapse.org/Synapse:syn21438414). Individual diagnoses were confirmed through postmortem neuropathological evaluation for neuritic plaque distribution according to the Consortium to Establish a Registry for AD criteria70 and neurofibrillary tangle pathology according to the Braak staging system71. Other neuropathologic assessments, including for α-synuclein, TDP-43, gliosis and angiopathy, were made in line with established criteria and guidelines72,73. Demographic variables for the cohort are listed in Supplementary Table 6. Human postmortem tissues were acquired under proper institutional review board protocols. CSF and plasma proteomics in the GNPC cohort were performed using the SomaScan v4.1 assay, which measures approximately 7,000 proteins74,75. This technology uses aptamer-based approaches, specifically slow off-rate modified aptamers, which incorporate chemically modified nucleotides that bind with high specificity and affinity to target proteins74. SomaLogic provides ‘raw' data that has been standardized, normalized, and calibrated, including using adaptive normalization by maximum likelihood. Protein measurements are provided in relative fluorescent units. Aptamers were mapped to Uniprot before being included in the GNPC cohort dataset. Details on the creation and harmonization of this dataset are described elsewhere28. Before our own analyses, we log2 transformed and standardized training and testing datasets separately. Proteins from postmortem brain homogenates from the dlPFC were measured using label-free MS on a Q-Exactive Plus mass spectrometer as previously described76. Proteins included in this study were identified using MaxQuant' MaxLFQ (label-free quantification) algorithm. Normalized label-free quantitation protein intensities were extracted from the proteinGroups.txt MaxQuant output file using a custom R script (LoaderAndBatchCorrection-UPenn354cases.R)76. To account for missing protein quantitation values in the dataset, we imputed values using the median for proteins with ≤20% missing values and excluded those proteins with ≥20% missing values. We identified APOE ε4-associated proteins using mutual information, a statistical measure that quantifies the amount of information one variable provides about another77. In the context of our study, it measures how informative each protein is for distinguishing APOE ε4 carriers from noncarriers. A mutual information score of zero indicates no association, while higher values indicate greater dependency between a protein and APOE ε4 status. We used a cutoff of ≥0.01 to indicate proteins that were likely to be associated with APOE ε4 carrier status. For CSF and plasma, we used nonimpaired controls to identify APOE ε4-associated proteins. This ensured that we were identifying proteins in only those individuals who are not symptomatic for any one neurodegenerative disease. Appropriate feature selection (identification of proteins specifically associated with APOE ε4) was further confirmed using PCA and heatmap. In the brain, we used mutual information to identify APOE ε4 proteins specific to each group of individuals. PCA was then used to confirm that within each group, APOE ε4-identified proteins clustered based on APOE ε4 carrier status. For visualization of differential protein abundance, volcano plots were constructed using log2 (fold change(FC)) on the x axis and −log10-transformed adjusted P values on the y axis. Plots show fold change and significance thresholds that were applied at log2(FC) = ± 0.585 (corresponding to a 1.5-fold change) and FDR-adjusted P < 0.05, respectively. Mutual information was calculated in R (v4.4.1) using the package ‘FSelectorRcpp'78 and PCA plots and heatmaps were made using ‘ggplot2'79 and ‘pheatmap', respectively. The generalizability of our identified APOE ε4 proteins across different neurodegenerative diseases was tested. We performed this analysis using supervised machine learning with CART via the ‘caret' package in R (v4.4.1). For all CART models, we first split the dataset into a 70% training and validation dataset and a 30% withheld (unseen) testing dataset. Model training and evaluation were conducted using ‘caret' with a fivefold cross-validation procedure repeated five times. Where classes (groups) were imbalanced, we addressed this by using upsampling. Model performance was assessed using the 30% unseen dataset and several metrics, including sensitivity, specificity, positive predictive value, negative predictive value and area under the curve. To identify potential sex-specific differences in the generalizability of APOE ε4 proteins, we trained and tested the CART models on a mixed sample of male and female APOE ε4 carriers and noncarriers. To assess any race-specific effects in plasma, we trained the CART models on White APOE ε4 carriers and noncarriers and then tested them using only individuals of another race (Black/African American or American Indian/Alaskan Native). To identify a potential relationship between APOE ε4 carrier status and postmortem histopathology in the dlPFC, we used a linear regression. Postmortem histopathology data for all donors included tau, thio-S-positive plaques, and angiopathy. Additional measures for AD and nonimpaired donors included antibody-positive plaques, Thal amyloid score, and gliosis. FTD, PDD and PD donors also had postmortem histopathological assessments for α-synuclein. FTD and ALS donors included TDP-43 assessments. To determine a potential relationship between overlapping CSF and plasma proteins identified as being APOE ε4-specific and demographic, clinical and lifestyle variables, we performed a correlation network analysis. Protein levels were based on those measured in the plasma, although only overlapping APOE ε4 identified across both CSF and plasma were included. Before computing any correlations, rows with missing values or encoded as an indeterminate response (for example, ‘unsure', coded as −1 or NA) were excluded on a pairwise basis. This ensured that each test was conducted only on complete and valid data. For associations between two continuous variables, we used Spearman's rank correlation coefficient (ρ). For associations between categorical variables and continuous protein expression values, we employed a correlation ratio (η2), which measures the proportion of variance in the continuous variable explained by the grouping variable. Categorical variables included binary indicators (for example, presence/absence of comorbidities) as well as variables with more than two levels (for example, ‘yes', ‘no', ‘unsure'). All categorical variables were explicitly converted to factors, and associations were computed only where at least two factor levels were present in the subset of complete cases. The correlation network analysis figure was created using GraphPad Prism (v10.0.0 for Windows). Protein–protein enrichment analyses to assess the functions enriched for APOE ε4 proteins were done using NetworkAnalyst (v3.0)80,81,82. Here generic protein–protein interactions were identified using a first-order network from the International Molecular Exchange Consortium interactome database83 and InnateDB84. Network enrichment for biological processes was performed using the PANTHER classification system18 and KEGG pathways85. Significance was determined by an FDR of >0.05. Immune cell-type-specific enrichment analyses for immune and liver cells were done using single-cell RNA-sequencing data from the Human Protein Atlas (https://www.proteinatlas.org/, v23; Ensembl v109)19. For each APOE ε4 protein, we identified the corresponding protein-coding transcripts per million counts. We then normalized the expression for each cell type using min–max scaling and used heatmaps to visually represent these enrichments. Enrichment bar graphs and heatmaps were created using GraphPad Prism (v10.0.0 for Windows). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The harmonized GNPC data used to generate these findings was provided to Consortium Members in June 2024 and will be made available for public request by the AD Data Initiative by 15 July 2025. Members of the global research community will be able to access the metadata and place a data use request via the AD Discovery Portal (https://discover.alzheimersdata.org/). Access is contingent on adherence to the GNPC Data Use Agreement and the Publication Policies. The AMP-AD UPenn Proteomics Study data is available through the AD Knowledge Portal (https://adknowledgeportal.synapse.org/). Researchers who wish to access this controlled dataset are required to submit a Data Use Agreement. More information can be found here: https://adknowledgeportal.synapse.org/Data%20Access. All code used for analyzing the proteomic and clinical data are available at https://github.com/Art83/gnpc_apoe. Belloy, M. E., Napolioni, V. & Greicius, M. D. A quarter century of APOE and Alzheimer's disease: progress to date and the path forward. Fabre, S. F. et al. Clinic-based cases with frontotemporal dementia show increased cerebrospinal fluid tau and high apolipoprotein E ε4 frequency, but no tau gene mutations. The effects of APOE and tau gene variability on risk of frontotemporal dementia. Rubino, E. et al. Apolipoprotein E polymorphisms in frontotemporal lobar degeneration: a meta-analysis. Ferrari, R. et al. Genetic architecture of sporadic frontotemporal dementia and overlap with Alzheimer's and Parkinson's diseases. Koriath, C. et al. ApoE4 lowers age at onset in patients with frontotemporal dementia and tauopathy independent of amyloid-β copathology. Li, Y. J. et al. Apolipoprotein E controls the risk and age at onset of Parkinson's disease. Ibanez, L. et al. Functional genomic analyses uncover APOE-mediated regulation of brain and cerebrospinal fluid β-amyloid levels in Parkinson disease. Drory, V. E., Birnbaum, M., Korczyn, A. D. & Chapman, J. Association of APOE ε4 allele with survival in amyotrophic lateral sclerosis. Zetterberg, H., Jacobsson, J., Rosengren, L., Blennow, K. & Andersen, P. A. Association of APOE with age at onset of sporadic amyotrophic lateral sclerosis. Real, R. et al. Association between the LRP1B and APOE loci and the development of Parkinson's disease dementia. Mata, I. F. et al. APOE, MAPT, and SNCA genes and cognitive performance in Parkinson disease. A. et al. GBA and APOE impact cognitive decline in Parkinson's disease: a 10-year population-based study. Tan, M. M. X. et al. Genome-wide association studies of cognitive and motor progression in Parkinson's disease. Tsuang, D. et al. APOE ε4 increases risk for dementia in pure synucleinopathies. The role of APOE in cognitive trajectories and motor decline in Parkinson's disease. Shvetcov, A. et al. Proteome profiling of cerebrospinal fluid using machine learning shows a unique protein signature associated with APOE4 genotype. Mi, H., Muruganujan, A., Casagrande, J. T. & Thomas, P. D. Large-scale gene function analysis with PANTHER classification system. Uhlen, M. et al. A genome-wide transcriptomic analysis of protein-coding genes in human blood cells. Butterworth, R. F. The liver-brain axis in liver failure: neuroinflammation and encephalopathy. Sebastiani, P. et al. A serum protein signature of APOE genotypes in centenarians. Fouquet, M., Besson, F. L., Gonneaud, J., La Joie, R. & Chetelat, G. Imaging brain effects of APOE4 in cognitively normal individuals across the lifespan. Morris, J. C. et al. APOE predicts amyloid-beta but not tau Alzheimer pathology in cognitively normal aging. Mishra, S. et al. Longitudinal brain imaging in preclinical Alzheimer disease: impact of APOE ε4 genotype. Robinson, J. L. et al. Neurodegenerative disease concomitant proteinopathies are prevalent, age-related, and APOE4-associated. Evidence for dynamically organized modularity in the yeast protein–protein interaction network. The Global Neurodegeneration Proteomics Consortium: biomarker and drug target discovery for common neurodegenerative diseases and aging. Van Exel, E. et al. Effect of APOE ε4 allele on survival and fertility in an adverse environment. Martin, G. M. APOE alleles and lipophylic pathogens. Finch, C. E. & Sapolsky, R. M. The evolution of Alzheimer disease, the reproductive schedule, and apoE isoforms. Tuminello, E. R. & Han, D. S. The apolipoprotein E antagonistic pleiotropy hypothesis: review and recommendations. Gale, S. C. et al. APOε4 is associated with enhanced in vivo innate immune responses in humans. Fernandez-Calle, R. et al. APOE in the bullseye of neurodegenerative diseases: impact of the APOE genotype in Alzhiemer's disease pathology and brain diseases. Tao, Q. et al. Association of chronic low-grade inflammation with risk of Alzheimer disease in ApoE4 carriers. Gallagher, M. D. & Chen-Plotkin, A. S. The post-GWAS era: from association to function. Ashton, N. J. et al. A multicentre validation study of the diagnostic value of plasma neurofilament light. Plasma neurofilament light chain as prognostic marker of cognitive decline in neurodegenerative diseases, a clinical setting study. & Fuh, J.-L. Levels of plasma neurofilament light chain and cognitive function in patients with Alzheimer or Parkinson disease. Plasma NfL is associated with the APOE ε4 allele, brain imaging measurements of neurodegeneration, and lower recall memory scores in cognitively unimpaired late-middle-aged and older adults. Frick, E. A. et al. Serum proteomics reveal APOE-ε4-dependent and APOE-ε4-independent protein signatures in Alzheimer's disease. A., Zetterberg, H., Blennow, K. & Alzheimer's Disease Neuroimaging Initiative. Association of plasma neurofilament light with neurodegeneration in patients with Alzheimer disease. Rutledge, J. et al. Comprehensive proteomics of CSF, plasma, and urine identify DDC and other biomarkers of early Parkinson's disease. Greene, C. et al. Blood–brain barrier disruption and sustained systemic inflammation in individuals with long COVID-associated cognitive impairment. Montagne, A. et al. APOE4 leads to blood–brain barrier dysfunction predictive cognitive decline. Sweeney, M. D., Montagne, A., Sagare, A. P., Nation, D. A. & Zlokovic, B. V. Broken blood–brain barrier in Alzheimer's disease and other neurodegenerative disorders. The three human monocyte subsets: implications for health and disease. Akbar, A. N. & Henson, S. M. Are senescence and exhaustion intertwined or unrelated processes that compromise immunity. Vivier, E., Tomasello, E., Baratin, M., Walzer, T. & Ugolini, S. Functions of natural killer cells. Dennis, N. A. et al. Temporal lobe functional activity and connectivity in young adult APOE ε4 carriers. Reduced rapid eye movement sleep in late middle-aged and older apolipoprotein E ε4 allele carriers. Sweeney, M. D., Zhao, Z., Montagne, A., Nelson, A. R. & Zlokovic, B. V. Blood–brain barrier dysfunction across neurodegeneration. Seeley, W. W., Crawford, R. K., Zhou, J., Miller, B. L. & Greicius, M. D. Neurodegenerative diseases target large-scale human brain networks. Pathways to neurodegeneration: mechanistic insights from GWAS in Alzheimer's disease, Parkinson's disease, and related disorders. The impact of routine vaccinations on Alzheimer's disease risk in persons 65 years and older: a claims-based cohort study using propensity score matching. Adult vaccination as a protective factor for dementia: a meta-analysis and systematic review of population-based observational studies. Eyting, M. et al. A natural experiment on the effect of herpes zoster vaccination on dementia. The recombinant shingles vaccine is associated with lower risk of dementia. Prevalence of the apolipoprotein E ε4 allele in amyloid β positive subjects across the spectrum of Alzheimer's disease. & Hypponen, E. Apolipoprotein E (APOE) genotype-associated disease risks: a phenome-wide, registry-based, case-control study utilising the UK Biobank. Relationships between APOE, type 2 diabetes, and cardiovascular disease in postmenopausal women. Lahoz, C. et al. Apolipoprotein E genotype and cardiovascular disease in Framingham Heart Study. Bunn, F. et al. Comorbidity and dementia: a scoping review of the literature. Addressing comorbidities in people with Parkinson's disease: considerations from an expert panel. The Consortium to Establish a Registry for Alzheimer's disease (CERAD). Gelb, D. J., Oliver, E. & Gilman, S. Diagnostic criteria for Parkinson disease. Gold, L. et al. Aptamer-based multiplexed proteomic technology for biomarker discovery. Plasma protein patterns as comprehensive indicators of health. Large-scale proteomic analysis of Alzheimer's disease brain and cerebrospinal fluid reveals early changes in energy metabolism associated with microglia and astrocyte activation. Ross, B. C. Mutual information between discrete and continuous data sets. Fayyad, U. M. & Irani, K. B. Multi-interval discretization of continuous-valued attributes for classification learning. 13th International Joint Conference on Uncertainty in Artificial Intelligence 1022–1027 (1993). Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag, 2016). & Hancock, R. E. W. NetworkAnalyst—integrative approaches for protein–protein interaction network analysis and visual exploration. Xia, J., Gill, E. & Hancock, R. E. W. NetworkAnalyst for statistical, visual and network-based approaches for meta-analysis of expression data. Zhou, G. et al. NetworkAnalyst 3.0: a visual analytics platform for comprehensive gene expression profiling and meta-analysis. Protein interaction data curation—the International Molecular Exchange Consortium (IMEx). Breuer, K. et al. InnateDB: systems biology of innate immunity and beyond—recent updates and continuing curation. Kanehisa, M. & Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. We are grateful to the cohort contributors, patients, donors and families who helped to make this research possible. We are also grateful to the members of the Global Neurodegeneration Proteomics Consortium, who provided ongoing feedback. This work was supported by the Australian Government's Medical Research Future Fund (MRF2040081 to C.A.F. and A.S.); philanthropic funding from Neil & Norma Hill Foundation, Annemarie & Arturo Gandioli-Fumagali Foundation, Perpetual Foundation—John Williams Endowment and Hillcrest Foundation (to C.A.F. and Robert Packard Center for ALS Research at Johns Hopkins University (to J.D.R.). The results published here are in whole or in part based on data obtained from the AD Knowledge Portal (https://adknowledgeportal.org). Study data were provided through the Accelerating Medicine Partnership for AD (U01AG046161 and U01AG061357), based on specimens provided by the Emory Goizueta Alzheimer's Disease Research Center (P50AG025688) where data collection was supported through funding by NIA (grants R01AG053960, R01AG057911, R01AG061800, RF1AG057471, RF1AG057470, R01AG061800, R01AG057911 and R01AG057339). A list of authors and their affiliations appears at the end of the paper. A full list of members and their affiliations appear in the Supplementary Information. Neurodegeneration and Disease Modelling Research Group, Westmead Institute for Medical Research, The University of Sydney, Westmead, New South Wales, Australia Department of Neurology, Emory University, Atlanta, GA, USA Goizueta Alzheimer's Disease Research Centre, Emory University, Atlanta, GA, USA Laboratory of Behavioral Neuroscience, National Institute on Aging, Intramural Research Program, Baltimore, MD, USA Heather M. Wilkins, Jeffrey M. Burns, Russell H. Swerdlow & Chad Slawson Heather M. Wilkins, Jeffrey M. Burns & Russell H. Swerdlow Department of Biochemistry and Molecular Biology, University of Kansas Medical Center, Kansas City, KS, USA Heather M. Wilkins, Russell H. Swerdlow & Chad Slawson OnPoint Scientific, Inc., San Diego, CA, USA Brain Science Institute, Johns Hopkins University School of Medicine, Baltimore, MD, USA Department of Neurology, Johns Hopkins University School of Medicine, Baltimore, MD, USA Gates Ventures, Seattle, WA, USA Department of Cell Biology and Physiology, University of Kansas Medical Center, Kansas City, KS, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar provided additional genotypes and postmortem histopathological data for the AMP-AD UPenn Proteomics Study cohort. provided key insights into the analytical strategy and PD. provided key insights into the effects of APOE genotypes. key insights for amyotrophic lateral sclerosis. provided key insights into AD, FTD and PDD. Correspondence to Artur Shvetcov or Caitlin A. Finney. The authors declare no competing interests. Nature Medicine thanks Boris Decourt, Omar El-Agnaf and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Liam Messin and Anna Maria Ranzoni, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Volcano plots showing fold change and adjusted p values for the APOE ε4 CSF proteins. Red dots indicate the proteins with the highest mutual information value (>0.1) for reference to the feature selection method used to identify APOE ε4 proteins. Differential protein abundance was assessed using linear modeling and empirical Bayes moderation. All tests were two-sided, and p-values were corrected for multiple comparisons using the Benjamini-Hochberg FDR method. Volcano plots display log2 fold change on the x-axis and -log₁₀-transformed adjusted p-values on the y-axis, with thresholds set at log2(FC) = ± 0.585 and FDR-adjusted p < 0.05. (b) Box plots showing amyloid beta A4 protein CSF levels in relative fluorescent units (RFU) across APOE ε4 carriers and non-carriers with AD, PD, and non-impaired controls. Data points outside this range are plotted individually as outliers. N = 526 AD, 247 PD, and 573 non-impaired control individuals. (a) Volcano plots showing fold change and adjusted p values for the APOE ε4 plasma proteins. Red dots indicate the proteins with the highest mutual information value (>0.1) for reference to the feature selection method used to identify APOE ε4 proteins. Differential protein abundance was assessed using linear modeling and empirical Bayes moderation. All tests were two-sided, and p-values were corrected for multiple comparisons using the Benjamini-Hochberg FDR method. Volcano plots display log2 fold change on the x-axis and −log10-transformed adjusted p-values on the y-axis, with thresholds set at log₂(FC) = ± 0.585 and FDR-adjusted p < 0.05. (b) Box plots showing amyloid beta A4 protein plasma levels in relative fluorescent units (RFU) across APOE ε4 carriers and non-carriers with AD, ALS, FTD, PDD, PD, and non-impaired controls. Data points outside this range are plotted individually as outliers. N = 2,929 AD, 75 FTD, 169 PDD, 422 PD, 230 ALS, and 6,099 non-impaired control individuals. Abbreviations: AD: Alzheimer's disease; ALS: amyotrophic lateral sclerosis; FTD: frontotemporal dementia; NI: non-impaired controls; PD: Parkinson's disease; PDD: Parkinson's disease dementia. Bar graph representation of the proportion of APOE ε4 and non-carriers with comorbidity across the clinical variables included in our correlation network analysis between APOE ε4 proteins and demographic, lifestyle, and clinical variables. Only those individuals who said ‘yes' to having comorbidity are included in the graph (Supplementary Table 1). APOE4+: APOE ε4 carrier; APOE4−: APOE ε4 non-carrier; afib: atrial fibrillation; chf: congestive heart failure; copd: chronic obstructive pulmonary disease; tbi: traumatic brain injury; tia: transient ischemic attack; mi: myocardial infarction. APOE ε4 proteins identified using label-free MS from the AMP-AD UPenn Proteomics Study cohort and mutual information. (a,c,e,g,i,k) PCA showing all label-free MS measured proteins in each donor group and no clear clustering across APOE ε4 carriers and non-carriers. (b,d,f,h,j,l) PCA showing only APOE ε4-identified proteins in each donor group, which leads to clustering in each group based on the presence or absence of an APOE ε4 allele. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. APOE ε4 carriers share immune-related proteomic changes across neurodegenerative diseases. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03807-3'>Large-scale proteomics project seeks cures for neurodegenerative diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 15:48:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. A new Global Neurodegeneration Proteomics Consortium provides a wealth of data to boost mechanistic understanding, biomarker research and drug development during aging and in neurodegenerative diseases — and dementia in particular. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Prices may be subject to local taxes which are calculated during checkout Tay, L. X., Ong, S. C., Tay, L. J., Ng, T. & Parumasivam, T. Value Health Reg. Farinas, A. et al. Nat. Shvetcov, A. et al. Nat. Imam, F. et al. Nat. Ali, M. et al. Nat. Beach, T. G., Monsell, S. E., Phillips, L. E. & Kukull, W. J. Neuropathol. Ossenkoppele, R. et al. Nat. Wilkins, C. H. et al. JAMA Neurol. Funding was provided by the National Institute on Health/National Institute on Aging (U24 AG082930 (M.M.M. )), Alzheimer's Drug Discovery Foundation (GC-2013717 (M.N.S., B.D. )), and Lewy Body Dementia Association (LBDA1811MS (M.N.S., B.D. Texas Tech University Health Sciences Center, School of Medicine, Department of Pharmacology and Neuroscience, Lubbock, TX, USA Alzheimer's and Memory Disorders Division, Barrow Neurological Institute, Phoenix, AZ, USA Department of Epidemiology and Prevention, Wake Forest University School of Medicine, Winston-Salem, NC, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Boris Decourt. has served on scientific advisory boards and/or has consulted for Acadia, Athira, Biogen, Cognito Therapeutics, Eisai, Lilly, Merck, PeerView Institute, Neurogen Biomarking, Novo Nordisk, Roche, Siemens Healthineers and Sunbird Bio. is an editor-in-chief of Neurology and Therapy, and has served as the board of director for EIP Pharma/CervoMed and consulted for Roche-Genentech, Eisai, Lilly, NeuroTherapia, Signant Health, Novo Nordisk, Anavex, Cognito Therapeutics, GSK, Abbvie, Lighthouse Pharmaceuticals, Alzheon and Athira. consults for Lighthouse Pharmaceuticals. Large-scale proteomics project seeks cures for neurodegenerative diseases. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02237-y'>One potent gene raises risk of Alzheimer's, Parkinson's and other brain diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 15:47:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Almost 60 million people globally have neurodegenerative diseases such as Alzheimer's disease.Credit: Mark & Mary Stevens Neuroimaging and Informatics Institute/Science Photo Library The gene variant, a version of apolipoprotein E called APOE ε4, produces a distinct set of proteins that contribute to chronic inflammation, finds an analysis1 using the largest proteomics database for neurodegenerative disease. Researchers know that people who carry the APOE ε4 variant have an increased risk of developing late-onset Alzheimer's disease, but studies are beginning to implicate this version, or allele, of APOE in other neurodegenerative diseases. Caitlin Finney and Artur Shvetcov, who study neurodegenerative diseases at the Westmead Institute for Medical Research in Sydney, Australia, and their colleagues wanted to better understand how this genetic risk factor contributes to disease. They took advantage of a newly established proteomics database that allowed them to look beyond individual diseases, says Finney. The Global Neurodegeneration Proteomics Consortium (GNPC) data set2 includes samples from more than 18,600 individuals, mainly of European ancestry, including many with Alzheimer's, Parkinson's, a form of motor neuron disease called amyotrophic lateral sclerosis (ALS) and types of dementia, as well as individuals without neurological disorders. “It's one of the most powerful databases that we have available for proteomics right now,” says Maryam Shoai, a bioinformatician at University College London. “It's a huge study,” says Tomas Deierborg, who studies neuroinflammation at Lund University in Sweden. The results are published today in Nature Medicine, along with a suite of other papers using the GNPC database, which has also been made publically accessible today. A similar protein signature was observed when the team analysed data from a separate collection of brain tissue samples taken postmortem from individuals with neurodegenerative diseases. A man was destined for early Alzheimer's — these genes might explain his escape China pours money into brain chips that give paralysed people more control This AI ‘thinks' like a human — after training on 160 psychology studies Will Gates and other funders save massive public health database at risk from Trump cuts? Join the International Institute for Intelligent Nanorobots and Nanosystems (IIINN) at Fudan University. IMRAM at Tohoku University invites applications for open tenure-track assistant and associate professor positions. A man was destined for early Alzheimer's — these genes might explain his escape An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250715043353.htm'>Weight-loss wonder drug Mounjaro/Zepbound shrinks breast cancer tumors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 13:26:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The anti-obesity medication tirzepatide, marketed as Mounjaro for diabetes and Zepbound for obesity, reduced obesity-associated breast cancer growth in a mouse model, according to a study presented at ENDO 2025, the Endocrine Society's annual meeting in San Francisco, California. "Obesity is a significant risk factor for breast cancer, and while it is very preliminary data, our studies in mice suggest that these new anti-obesity drugs may be a way to reduce obesity-associated breast cancer risk or improve outcomes," said study author Amanda Kucinskas, B.S., a Ph.D. candidate in the labs of Drs. Erin Giles and Kanakadurga Singer at the University of Michigan in Ann Arbor, Mich. Existing research has shown that having obesity can lead to worse breast cancer outcomes compared to those who do not have obesity, and weight loss can improve outcomes. However, there are many challenges with traditional weight loss methods. Kucinskas and colleagues leveraged tirzepatide, one of a new class of effective anti-obesity medications that target GLP-1 (glucagon-like peptide 1) and GIP (glucose-dependent insulinotropic polypeptide) receptors. The researchers sought to learn whether or not tirzepatide would reduce obesity-associated breast cancer growth. "While these are very preliminary results, they suggest that this new anti-obesity drug may also have a beneficial impact on breast cancer outcomes," Kucinskas said. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65383604/poland-trapezoid-pyramids/'>Archaeologists Found Trapezoidal Tombs Older Than the Egyptian Pyramids</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The megalithic tombs, located in Poland, are likely around 5,500 years old. The new discovery of uniquely shaped “Polish pyramids” from 5,500 years ago highlights the distinct way early European agricultural societies buried their dead. While the tombs rarely reach taller than 13 feet, the sheer size and design make them a distinct sight in Poland. Each tomb was typically covered with cobblestones and massive boulders were placed in front, some weighing as much as 10 tons, a show of the culture's intricate teamwork and tool know-how to make happen. Each oversized tomb often only contained a single skeleton of a key leader, with the body placed in an upright position and surrounded by grave gifts. “Each generation built its own megalith, honoring the deceased who played a vital role in their community,” said Golis. While no such skeletons were found in the most recent tomb discovery, archaeologists remain hopeful they can locate grave goods still buried nearby, most likely axes and pottery. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Archaeologists May Have Found a Maya King's Tomb Someone Just Found a ‘Holy Grail' Scroll From 1847 Experts Found a Sarcophagus in an Ancient Tomb Archaeologists Are Saving Ruins of a World Wonder</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a65383962/dragon-pseudoscorpion/'>4 Eyeless Creatures Have Emerged From One Cave After Hiding in the Dark for Centuries</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Despite their ominous name and oversized jaws, these animals only measure about one to three millimeters in length. In Aristotle's De historia animalium (one of the founding texts of zoology), the great Greek thinker pens the very first description of a strange creature that is “found in books” and “look[s] like scorpions; they have no tails, and are very small.” This is a humble pseudoscorpion—a pint-sized false scorpion (as its name suggests) that nonetheless looks a lot like it's more-dangerous, bigger cousin. While a member of the class Arachnida, they shouldn't inspire as much fear as other poisonous members of this group. These creatures use poison to subdue prey, but pose no danger to humans, and can actually be immensely beneficial for pest control—they'll munch on moth larvae, mites, and small flies. And true to Aristotle's original observation, they can often be found within the pages of books, as one of their favorite prey species (booklice) feasts on the starch typically found in bookbindings. But not all pseudoscorpions call literary habitats their home. A new study—published in the journal PLOS One and led by scientists at Jeonbuk National University—details four new species of “dragon pseudoscorpions,” named as such because of their dragon-like jaws. “In Korea, pseudoscorpions are a ‘neglected' invertebrate taxon with presently 28 species in 13 genera that belong to eight families,” the authors wrote. “However, a recent preliminary barcoding study has indicated that the species biodiversity in surface habitats must be significantly higher. The subterranean fauna is probably also diverse, but to date, only five pseudoscorpion species have been recorded from caves in South Korea.” Because these creatures live in caves, they have numerous adaptations that are considered “troglomorphic,” due to living in constant darkness. For example, these four new species lack eyes and possess very little body pigmentation, causing them to range from light orange to brown. The researchers theorize that these species are likely endemic to one single cave or karst system, but South Korea is chock-full of caves, with an estimated 2,000 spread across the Taebaek and Sobaek mountain ranges. “Many people have believed there is only one species (Spelaeochthonius dentifier) distributed in Korea,” Kyung-Hoon Jeong, a master student and lead author of the study from Jeonbuk National University, told IFLScience. Like all ecosystems on the planet, caves are threatened by climate change, and understanding the wide variety of animal life in these caves can help scientists conserve them. A Colossal Squid Might Have Shown Its Secret Face The Rodents of Chicago Are Evolving in Real Time</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250715043347.htm'>Big-Bang echoes unmask a billion-light-year hole around Earth—and it's stretching space faster</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 10:06:04
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Earth and our entire Milky Way galaxy may sit inside a mysterious giant hole which makes the cosmos expand faster here than in neighboring regions of the universe, astronomers say. Their theory is a potential solution to the 'Hubble tension' and could help confirm the true age of our universe, which is estimated to be around 13.8 billion years old. It can be measured by observing the distance of celestial objects and how fast they are moving away from us. This therefore gives the appearance of a faster local expansion rate." Directly counting galaxies does support the theory, because the number density in our local universe is lower than in neighbouring regions. However, the existence of such a large and deep void is controversial because it doesn't mesh particularly well with the standard model of cosmology, which suggests matter today should be more uniformly spread out on such large scales. "These sound waves travelled for only a short while before becoming frozen in place once the universe cooled enough for neutral atoms to form," he explained. "They act as a standard ruler, whose angular size we can use to chart the cosmic expansion history. "By considering all available BAO measurements over the last 20 years, we showed that a void model is about one hundred million times more likely than a void-free model with parameters designed to fit the CMB observations taken by the Planck satellite, the so-called homogeneous Planck cosmology." This involves looking at galaxies that are no longer forming stars. By observing their spectra, or light, it is possible to find what kinds of stars they have and in what proportion. Since more massive stars have shorter lives, they are absent in older galaxies, providing a way to establish a galaxy's age. This sheds light on the universe's expansion history. It can be measured by observing the distance of celestial objects and how fast they are moving away from us. They provide an independent way to measure the expansion rate of the universe and how that rate has changed throughout cosmic history. Note: Content may be edited for style and length. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250714052235.htm'>Obesity is driving a hidden cancer epidemic—13 types and rising deaths nationwide</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 07:38:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The study, which examined more than 33,000 deaths from obesity-associated cancers, revealed sharp increases in cancer deaths, especially among women, older adults, Native Americans and Black Americans. "Obesity is a significant risk factor for multiple cancers, contributing to significant mortality," said lead researcher Faizan Ahmed, M.D., of Hackensack Meridian Jersey Shore University Medical Center in Neptune City, N.J. "This research underscores the need for targeted public health strategies such as early screening and improved access to care, especially in high-risk rural and underserved areas." According to the U.S. Centers for Disease Control and Prevention (CDC), 40.3% of adults have obesity. Obesity is a complex disease resulting from multiple genetic, physiological, hormonal, environmental and developmental factors. In addition to certain types of cancer, obesity raises the risk of developing serious chronic conditions such as high blood pressure, high cholesterol, prediabetes, type 2 diabetes, heart disease, and chronic and end-stage kidney disease. He found age-adjusted mortality rates increased from 3.73 to 13.52 per million over two decades, with steep increases among women, older adults, Black people, Native Americans and rural populations. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250714052231.htm'>Sweet but risky: Common sweeteners may be accelerating puberty in kids</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-15 06:18:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The researchers found that consuming aspartame, sucralose, glycyrrhizin and added sugars was significantly associated with a higher risk of early puberty, especially in children with certain genetic traits. "This study is one of the first to connect modern dietary habits -- specifically sweetener intake -- with both genetic factors and early puberty development in a large, real-world cohort," said Yang-Ching Chen, M.D., Ph.D., of Taipei Municipal Wan Fang Hospital and Taipei Medical University in Taipei, Taiwan. "It also highlights gender differences in how sweeteners affect boys and girls, adding an important layer to our understanding of individualized health risks." Chen's previous research found that certain sweeteners can directly influence hormones and gut bacteria linked to early puberty. For example, one artificial sweetener called acesulfame potassium or AceK was shown to trigger the release of puberty-related hormones by activating "sweet taste" pathways in brain cells and increasing stress-related molecules. Another sweetener, glycyrrhizin -- found in licorice -- was found to change the balance of gut bacteria and reduce the activity of genes involved in triggering puberty. The researchers assessed teens' sweetener intake through validated questionnaires and testing of urine samples. Genetic predisposition was quantified using polygenic risk scores derived from 19 genes related to central precocious puberty. Early puberty was diagnosed based on medical exams, hormone levels and scans. "The findings are directly relevant to families, pediatricians and public health authorities," Chen said. This could lead to new dietary guidelines or risk assessment tools for children, supporting healthier development." NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            