
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-25</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03802-1'>What happened at COP30? 4 science take-homes from the climate summit</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 16:34:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. COP30 left many countries disappointed because no new road maps were created to help nations transition away from fossil fuels.Credit: Wagner Meier/Getty But, there were some signs of hope that multilateralism can tackle climate change. A glaring omission was language calling for the creation of road maps to phase out fossil fuels and halt deforestation, but Brazil has announced it will push those ideas forward independently of the COP process. Here, Nature takes a look at the results from COP30 and what comes next. As a result, scientists with the Climate Action Tracker consortium still project that the world is on track for upwards of 2.6 ° C of warming by 2100. “It's really a sign that countries have limited appetite to support something more ambitious on climate.” More than 80 countries joined Brazil's President Luiz Inácio Lula da Silva in the call for the creation of a road map to phase out fossil fuels, but the proposal ultimately foundered after reported opposition from oil-producing nations including Saudi Arabia and others. Protesters carrying signs that read “our forests are not for sale” broke through security lines of the COP30 climate talks on 12 November.Credit: Pablo Porciuncula/AFP via Getty China pledges to cut emissions by 2035: what does that mean for the climate? China pledges to cut emissions by 2035: what does that mean for the climate? It's a step towards ensuring that wealthy countries meet their responsibilities to provide climate finance to those in need, says Jodi-Ann Wang, who researches equity and climate finance at the London School of Economics and Political Science. Read the paper: Achieving net zero greenhouse gas emissions critical to limit climate tipping risks How nitrogen compounds in fertilizers and fossil-fuel emissions affect global warming China pledges to cut emissions by 2035: what does that mean for the climate? Join ITM Strasbourg as a tenure-track PI advancing AI-powered translational research in fibrosis, cancer, and therapeutic target discovery. Sanhang Institute for Brain Science and Technology (SiBST) at Northwestern Polytechnical University Read the paper: Achieving net zero greenhouse gas emissions critical to limit climate tipping risks How nitrogen compounds in fertilizers and fossil-fuel emissions affect global warming China pledges to cut emissions by 2035: what does that mean for the climate? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251125081909.htm'>Tiny Yellowstone quakes ignite a surge of hidden life underground</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 14:14:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Eric Boyd and his research team investigated how a burst of small earthquakes in 2021 affected the communities of microbes living deep beneath the Yellowstone Plateau Volcanic Field. These microbes inhabit rock and water systems far below the surface, where sunlight never reaches. This chemistry provides the energy that supports many forms of underground life. Earthquakes have the potential to reshape this environment in several ways. Shaking can break open new rock surfaces, force out fluids that were previously sealed away, and redirect the flow of water through the subsurface. To understand how seismic activity influenced this hidden ecosystem, the researchers collected water samples from a nearly 100-meter deep borehole located along the western edge of Yellowstone Lake. They sampled the site five times throughout 2021, giving them a rare look at how conditions changed both immediately and over the following months. Analyses of these samples showed notable increases in hydrogen, sulfide, and dissolved organic carbon after the earthquakes. These compounds serve as important energy sources for many subsurface organisms. Beyond detecting an increase in cell numbers, Boyd and colleagues observed that the types of microbes present changed over time. This result stands out because subsurface microbial communities in continental bedrock aquifers are often considered relatively stable. Their findings imply that even small seismic events can drive meaningful ecological shifts underground. Many regions around the world experience regular seismic activity that could similarly reshape subsurface energy supplies. If this mechanism is widespread, it could help explain how microbial life persists in deep and isolated environments. If earthquakes or similar geological motions can refresh chemical resources below the surface, this could expand the possible habitats for microbes on worlds such as Mars. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/green-tech/a69489569/newberry-volcano/'>Scientists Want to Use One of the Most Dangerous Volcanoes in the U.S. to Power Our Future</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The planet is a roiling ball of geothermal energy, but so far, the U.S. hasn't been great at tapping into this near limitless energy source. According to the U.S. Energy Information Administration, the country has geothermal plants located in only seven states—all out west—producing roughly 0.4 percent (17 billion kilowatt-hours) of utility-scale electricity generation as of 2023. However, it doesn't take much imagination to see the untapped potential of geothermal energy. Iceland, for example, generates more than one-fourth of its electricity needs from geothermal energy directly. In an effort to tap into that potential, a start-up called Mazama Energy, is focusing its sights on Newberry Volcano, a large, active, shield volcano located roughly 20 miles south of Bend, Oregon. While not as well known as other Cascade volcanoes, such a Mount St. Helens, Mount Rainier, or even Mount Hood, Newberry is the largest volcano in the Cascade Volcanic Range, and the United States Geological Survey considers it a “very high threat” volcano, similar to Mauna Loa and Kīlauea in Hawaii. However, mixed within that danger is also an unprecedented geothermal opportunity. With superhot rock, on the other hand, energy companies inject water into areas with dry superheated rock at tempratures that are preferably over the threshold of 705 degrees Fahrenheit. According to The Washington Post, this supercritical water can hold more heat like a liquid but flow like a gas, providing five to ten times more energy than typical geothermal operations. In fact, according to a model produced by the Clean Air Task Force, 20 percent of the U.S.'s landmass, or around 750 thousand square miles, has superhot rock energy potential. But this energy doesn't come without risks, the biggest of which being man-made earthquakes. Fracking for natural gas is known to cause tremors by changing pressure conditions along faults, and in 2018, South Korea experienced its second most-powerful earthquake in modern history due to a geothermal plant. Mazama Energy still needs to face the daunting task of keeping drilling and energy-producing infrastructure running under such punishing conditions, but the company tells The Washington Post they hope to produce 15 megawatts of electricity on Newberry Volcano's western flank by next year, and eventually pump out as much as 200 megawatts, roughly the energy budget of a small city. However, those numbers are small in comparison to the site's theoretical potential, which could be as high as five gigawatts, according to Vasantharajan—roughly two-thirds of Oregon's average energy output. Those geothermal aspirations, however, currently remain out of reach, but the U.S. hopes for geothermal to provide 16 percent of the country's energy mix by 2050. If it has any hope of achieving that dream, superhot rocks could be the key reason why.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/which-thanksgiving-pie-causes-the-biggest-blood-sugar-spike-pecan-apple-or/'>Which Pie Gives You the Biggest Sugar Rush: Pecan, Apple or Pumpkin?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Which Pie Gives You the Biggest Sugar Rush: Pecan, Apple or Pumpkin? Scientific American asked experts which type of Thanksgiving pie spikes blood sugar the most—and how to eat healthier while still enjoying the holidays But it contains an ungodly amount of corn syrup and sugar, so it got me thinking: Which type of pie spikes your blood sugar the fastest? Pie varieties abound, so I focused on some of the classics: pecan, pumpkin and apple. Regardless of filling, when you take a bite of pie, the carbohydrates get delivered to your bloodstream as glucose that ultimately gets absorbed by cells. To get to the bottom of this gooey puzzle, I talked to Sarah Berry, a professor of nutritional sciences at Kings College London, and Sandra Arévalo, director of community health and wellness at Montefiore Nyack Hospital. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Not all pies are created equally: you have more control over the ingredients in homemade pies, so they tend to be healthier than store-bought ones. In general, though, many pecan pies are on the sweeter side. “But what we also know is the pecan tends to be higher in protein as well and tends to be a little bit higher in fiber.” Protein also stimulates the release of the hormone insulin, which moves sugar into your cells and out of your bloodstream, Berry says. But because pecans aren't that sweet, people often add a lot of sugar, Arévalo says. Pecan pie recipes often have corn syrup in them not only for the sweetness but for the texture, she says. So you could reduce the amount of corn syrup and replace it with chia seeds, for example. Another option for slowing the sugar rush in pecan pie is to serve it with a scoop of plain yogurt or some unsweetened cream, Berry and Arévalo suggest. Pumpkin pie generally contains less sugar and more dietary fiber than pecan. “Pumpkin is pretty sweet already,” Arévalo says, “so you might not need to add as much sugar. Here again, you could serve the pie with some plain yogurt or even some nuts on top for a little extra protein. Nuts also give the pie some crunch, she says. For the filling, Arévalo recommends using apples with their fiber-rich peels on. Overall there's not a huge difference in the fiber or protein content of all three pies, Berry says. As mentioned already, pecan pies tend to pack the most powerful punch, sugar-wise, and pumpkin pie isn't necessarily any better. But apple pie isn't always the healthier option here. “If the pecan pie doesn't have a lot of sugar, then I will put pecan pie first. You can make any homemade pie healthier by adjusting the amount of sugar, protein and fiber. And there are other ways to offset the effects of your pie consumption by taking a brisk walk after dessert. Ultimately, you should just enjoy your Thanksgiving feast. Tanya Lewis is senior desk editor for health and medicine at Scientific American. She writes and edits stories for the website and print magazine on topics ranging from COVID to organ transplants. Previously, she has written for outlets that include Insider, Wired, Science News and others. She has a degree in biomedical engineering from Brown University and one in science communication from the University of California, Santa Cruz. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/this-fossil-is-rewriting-the-story-of-how-plants-spread-across-the-planet/'>This Fossil Is Rewriting the Story of How Plants Spread across the Planet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An enigmatic group of fossil organisms has finally been identified—and is changing the story of how plants took root on land Around 410 million years ago, terrestrial life was relatively simple. There were no forests or prairies—land was largely dominated by slimy microbial mats. A new discovery is rewriting the story of how these vascular plants, as they are called, spread onto land. Researchers may have finally resolved a debate about an enigmatic but widespread fossil called Spongiophyton: it seems to have been a strange life-form called a lichen that may have helped pave the way for plants to thrive on land. Fragment isolated from the stem of Spongiophyton nanum showing its upper surface with pores. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “Until now, most people thought lichens appeared only after vascular plants, but this study shows they were already there at the beginning, literally helping prepare the ground for plant life.” Lichens are the symbiotic result of fungi and photosynthetic algae or cyanobacteria working together. Today that amalgam helps churn lifeless rocks and sediments into nutrient-rich soil everywhere from polar deserts to tropical forests, says study lead author Bruno Becker-Kerber, a paleontologist at Harvard University. Vascular plants have tissues that funnel those soil nutrients from the ground to their stems and leaves. Because lichens' soft body tissues are rarely preserved in the fossil record, their origins have remained mysterious. A 2019 genetic analysis suggested they evolved well after the emergence of vascular plants, indicating they likely played little to no role in early land colonization. Scientists have long debated whether Spongiophyton, which flourished around 410 million years ago, was lichen or algae. Unlike algae, which are lined with cell walls, lichen contain fungi lined with chitin, the same material that makes up the exoskeletons of insects. There were other fungal traits present, too, such as a distinct branching pattern exhibited by growing fungal cells called hyphae. The results suggest lichen evolved around 410 million years ago, shortly after the initial spread of vascular plants 420 million years ago and just before the earliest known forests around 390 million years ago. “It's a major shift in how we view the complexity of life's first steps onto land,” Becker-Kerber says. Spongiophyton “likely weathered rocks, stabilized sediments, cycled nutrients and contributed to the formation of protosoils just before forests developed.” “If Spongiophyton was a lichen, it may have enabled the expansion of land plants into areas previously uncolonized,” says Matthew Nelsen, an evolutionary biologist at the Field Museum of Natural History, who was not involved with the research. “What our study shows is that fungi and lichens were also part of it.” Taylor Mitchell Brown is a San Diego–based journalist covering anthropology and paleontology, with occasional forays into other disciplines. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41929-025-01447-x'>Charting the influence of deuterium sources in hydrogen isotope exchange using a cobalt(III) catalytic platform</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 10:59:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Catalysis (2025)Cite this article A preprint version of the article is available at ChemRxiv. Directed hydrogen isotope exchange strategies have become an essential tool for accessing isotopically labelled organic scaffolds and interrogating the mechanisms of transition metal-catalysed C–H activation. However, the rationale behind deuterium source selection remains largely absent from the literature; an oversight that directly affects mechanistic interrogation strategies and hinders the development of new deuteration methodologies. Here we explore the influence of the deuterium source in base-assisted site-selective C–H deuteration reactions across a broad range of substrates using cobalt catalysis. We employ a synergistic combination of experimental studies and multivariable linear regression models based on proposed catalytic intermediates. Our findings demonstrate that the deuterium source can directly alter the operative mechanism, leading to distinct reaction pathways under different conditions. These results highlight previously overlooked complexity in hydrogen isotope exchange reactions and provide an example of how data-driven mechanistic analysis can expose subtle, reagent-dependent mechanistic shifts in catalytic behaviour. Transition metal-catalysed hydrogen isotope exchange (HIE) bridges fundamental research and practical synthetic utility1,2,3,4,5. These reactions serve both as a mechanistic probe for studying the reversibility of C–H activation, and as an effective methodology for introducing isotopic labels into molecules relevant to medicinal chemistry6,7,8. Nevertheless, an elusive aspect of HIE is the suitability of a given deuterium source for a specific transformation. Even a close examination of the literature within the vibrant field of directed, base-assisted C–H activation, reveals an assortment of isotope sources across transition metal catalysis (Fig. 1a and Supplementary Table 1). This ambiguity led us to question whether hidden behind this deceptively simple transformation was a common yet overlooked pitfall—different combinations of conditions and substrates can trigger distinct mechanistic pathways. a, Challenges and opportunities. The literature search for papers involving directed HIE was performed using SciFinder in March 2025 (Supplementary Table 1). b, Synergistic combination of experimental studies and MLR models based on proposed catalytic intermediates to uncover the influence of deuterium sources in HIE (this work). DG, directing group. Most experimental or computational mechanistic approaches are predominantly geared to explore the mechanism of a reaction using a model substrate9,10,11,12,13. However, we sought to develop an approach that could shed light on complex mechanistic landscapes forged by changes to substrates and reaction conditions. To this end, we turned our attention to linear free energy relationship (LFER) studies, which follow the influence of substituent effects on a reaction outcome14. The underlying assumption behind LFER is that as long as substrates follow a similar mechanistic route, a linear correlation can be identified between a molecular descriptor (for example, Hammett electronic values or Tolman cone angles) and the reaction outcome (for example, the yield or selectivity)15,16. In recent years, this idea has been extended to multivariable linear regression (MLR), wherein several molecular descriptors are required to describe the factors that influence a reaction outcome17,18,19,20,21,22,23,24. Whereas descriptors are most often drawn from the structural properties of starting materials, in some cases meaningful correlations only emerge when features of key reaction intermediates are included25,26,27,28,29,30,31. Motivated by the idea that molecular descriptors from different reaction intermediates can more faithfully capture substrate behaviour in distinct reaction manifolds, we envisioned a strategy to explicitly evaluate divergent mechanistic pathways. On the basis of this premise, in this work, we uncover mechanistically relevant pathways by analysing correlations between descriptors from different plausible intermediates and the reaction outcome (Fig. Namely, we seek to ask whether multiple mechanisms could be operative in HIE using the most common inexpensive and convenient deuterium sources, such as D2O, CD3OD or AcOD-d4. Our approach, rooted in the comparison of reaction intermediates, reveals a complex yet strong relationship between the substrate structure and the required deuterium source, revealing patterns inaccessible by intuition alone. These findings suggest that the isotope source can act as a mechanistic trigger, eliciting distinct pathways depending on the substrate structure. We initiated our investigation by establishing a catalytic platform tailored to uncover the factors that govern hydrogen isotope exchange. Cp*CoIII complexes were chosen as an ideal model because they operate through a reversible C–H activation step32,33,34, enabling efficient H/D exchange across a wide array of directing groups and multiple deuterium sources35,36,37,38,39. We then evaluated its suitability for isotopic labelling using different deuterium sources on diverse scaffolds (Fig. To this end, we targeted the following widely used directed C–H functionalization substrates, 2-phenylpyridine (I), benzaldehyde (II) and acetanilide (III), and identified three optimized reaction conditions (Supplementary Tables 2–4). Both 2-phenylpyridine (I) and benzaldehyde (II) underwent efficient H/D exchange using D2O as the deuterium source (Conditions A and B, respectively). By contrast, CD3OD provided the highest levels of deuterium incorporation for acetanilide (III, Condition C). Deuterium incorporations were determined by 1H NMR spectroscopy. aFor substrate II, we added aniline (20 mol%). To construct a diverse dataset for statistical analysis, we selected substrates that would have practical relevance but would also provide a broad distribution of structural diversity and deuteration levels (Fig. Under Condition A, we examined a series of 2-phenylpyridine derivatives, observing a broad functional group tolerance for both electron-donating and electron-withdrawing substituents at the phenyl para position (2–4). Compounds 5 and 6, which contain additional coordinating functionalities, are deuterated only at the position ortho to the pyridine-directing group, which is consistent with the strong coordinating effect of N-containing heterocycles. Steric hindrance at the ortho position (7) did not strongly affect the outcome, and modifications to the pyridine ring itself (8–10) preserved reactivity. We also evaluated the late-stage deuteration of biologically relevant molecules. The pyridine-containing antiretroviral drug atazanavir (11) afforded high deuterium incorporation in excellent yield. Condition A: substrate (0.25 mmol), Cp*Co(CO)I2 (10 mol%), KOAc (20 mol%), AgSbF6 (20 mol%), D2O (25 equiv. ), DCE (0.5 ml), 50 °C, 2 h. Condition B: substrate (0.25 mmol), Cp*Co(CO)I2 (10 mol%), Cu(OAc)2 (10 mol%), AgSbF6 (20 mol%), D2O (10 equiv. ), DCE (2.5 ml), 100 °C, 2 h. Condition C: substrate (0.25 mmol), Cp*Co(CO)I2 (10 mol%), Cu(OAc)2 (10 mol%), AgSbF6 (20 mol%), CD3OD (10 equiv. ), DCE (5 ml), 100 °C, 2 h. aReaction carried out at 0.1 mmol scale. bAniline (20 mol%) was added. cYield determined by 1H NMR of the reaction crude with 1,3,5-trimethoxybenzene as internal standard. For Condition B, we targeted aromatic aldehydes, using aniline as a transient directing group for in situ imine formation40,41. The transient role of aniline was confirmed by submitting the preformed imine of benzaldehyde (13), N-benzylideneaniline (12), to Condition B in the absence of aniline, which afforded the same deuterium incorporation as benzaldehyde. A diverse range of aromatic aldehydes were efficiently deuterated (13–26) under these conditions. Compound 20, which contains an aldehyde and pyridine group, was deuterated across all ortho positions. The same reactant under Condition A was uniquely deuterated at the positions ortho to pyridine (see compound 5). We note that the acetamido group of compound 19 did not facilitate isotope labelling and only the positions ortho to the aldehyde were deuterated. When 5-phenylthiophene-2-carbaldehyde (26) was subjected to Condition B, we only observed H/D exchange at the ortho positions of the carbonyl group, indicating that thiophene is not a suitable directing group under these conditions. Interestingly, in the absence of aniline, Condition B remained highly efficient for arenes containing strong N-based directing groups (substrates 27–46), which had shown limited reactivity under Condition A. These results include several pharmaceutically relevant heterocycles, such as pyrazoles and oxazoles, and the late-stage functionalization of the anti-inflammatory drug etoricoxib (45), as well as the arthritis medication oxaprozin (46). Finally, we examined the compatibility of Condition C with a variety of amide derivatives, which are prevalent in medicinal chemistry42. Acetanilides bearing electron-donating or electron-withdrawing moieties with different substitution patterns reached moderate to high degrees of deuterium incorporation (47–53). Sterically demanding groups connected to the carbonyl group proved to be beneficial for this transformation (54 and 55). A urea derivative (56) and monosubstituted amides (57 and 58) were compatible with our protocol. A clear decrease in deuterium incorporation was observed for disubstituted amides (59–62), but lactam 63, a cyclic disubstituted amide, afforded improved deuterium incorporation. Lastly, functional group compatibility was tested in selected biologically active molecules containing challenging moieties, such as hydroxy or carboxylic acid motifs (64 and 65). In agreement with our results, the necessity of using different deuterium sources is also widespread in the HIE literature1,2,3,4,5. However, there are no studies focused on uncovering the mechanistic implications of the deuterium source on a given organic scaffold in base-assisted C–H deuteration. Thus, we sought to apply MLR models to correlate the structural features of scaffolds with their corresponding deuterium incorporation values43,44,45. Initially, we attempted to produce models using molecular descriptors derived directly from substrate structures. However, these features led to poor correlations (Supplementary Fig. As a result, we hypothesized that molecular descriptors produced from reaction intermediates could provide improved models and might better capture mechanistic distinctions between the tested condition sets (Fig. The main setback towards extracting such molecular descriptors is the identification of catalytically relevant intermediates that could represent the mechanism of the studied reaction. a, Illustration of the most predictive MLR models identified for each condition. The linear graphs represent the correlation between the predicted and measured deuterium levels taken as \(\triangle \triangle G=\,-{RT}\log \left(\frac{ \% {\text{H}}}{ \% {\text{D}}}\right)\) to reflect the energy difference for obtaining each product with a 90% prediction interval in grey. Below each plot are the linear equations and the features identified to provide the best fit with the experimental outcomes. Each data point was sampled from a crude reaction performed in duplicates at a scale of 0.1 mmol. Proposed intermediates (Int) that were tested by MLR. Below each of the intermediates is a bar graph showing the goodness-of-fit R2 values (blue) and the leave-one-out (LOO) cross-validation Q2 values (orange) for all the intermediates (values given between 0.6 and 0.9, see Supplementary Figs. 31–33) in each one of the conditions. b, Comparison of the deuterium incorporation obtained with different deuterium sources for Conditions A, B and C. We viewed this challenge as an opportunity to develop a workflow based on sifting through correlations of features extracted from different plausible intermediates. This approach was based on the assumption at the crux of LFER studies that the best correlation with experimental results can enclose information regarding the relevant mechanistic pathway. By extending this assumption to reaction intermediates, we aimed to address a gap in the application of computational chemistry to tease out experimental mechanistic information. Whereas density functional theory (DFT) studies play a crucial role in revealing complete reaction pathways for a single model substrate, our approach provides a view of the reaction mechanism landscape through the prism of a broad set of substrates. In this work, to probe the potential involvement of different D shuttles under our reaction conditions, we selected several plausible intermediates to test on the basis of previous mechanistic proposals for directed base-assisted C–H activation46,47,48. Setting all the tested substrates in each of these intermediates, we performed ground-state DFT computations and extracted relevant molecular descriptors (Supplementary Figs. A linear regression model search across the features of each of the seven proposed intermediates was conducted to identify the best-fitted correlation to the deuteration level in the three reaction condition sets (Fig. We note that for intermediates where ROD molecules participate, we computed both D2O and CD3OD options. For Condition A, a predictive model was identified based on intermediate Int5water, where acetic acid is directly coordinated to the metal and the deuterium source (D2O) bridges the D transfer through hydrogen bonding. The model reveals a tight correlation between deuterium incorporation and three molecular features: the partial charge on an acetic acid oxygen atom connected to the metal centre (q1), the overall dipole of the intermediate (μtot) and the minimal width of the substrate (Sterimol B1). According to the normalized regression coefficients the most impactful features are the charge and the total dipole of the intermediate, indicating that electronic effects probably play an important role in determining deuterium incorporation. In addition, all the features are localized on either the acetate or substrates, with the purely steric feature, B1, reflecting that the planarity of the substrates contributes to increasing deuterium incorporation. This model suggests that both acetic acid and water may be involved in the deuteration event in a cooperative deuterium transfer mechanism. When comparing the two models stemming from Int5 for the results of Condition A, the structure containing water provided a more robust model compared to the one with methanol (Int5water: R2 = 0.88, Q2LOO = 0.81; Int5methanol: R2 = 0.81, Q2LOO = 0.64). These outcomes are consistent with the experimental observation that D2O is a more effective deuterium source under Condition A (Supplementary Table 2). For Condition B, the model identified intermediate 3 (Int3water) as having the best fit with the experimental results. Similarly to the substrates under Condition A, we identified an improved fit for the intermediate containing water as a deuterium source, which, here too, agrees with the experimental results (Supplementary Table 3). The model reveals a correlation between deuterium incorporation and four molecular features: the partial charge on the hydrogen atom that bridges between water and acetate (q2), the difference in partial charge between the cobalt and the carbon undergoing deuteration (∆q3), the difference in partial charge between the water hydrogen atom and the oxygen of the acetate (∆q4) and the distance between the cobalt and directing group (d1). Here, similarly to Condition A, the features lean towards stronger electronic control of reactivity, with charge-related features being particularly dominant. The coefficients of the normalized model are all similar in magnitude and the features are localized to either the substrate or acetate, yet, unlike Condition A, here the substrate features seem to be more electronic in nature. Lastly, for Condition C, the model with the best fit to the experimental results was Int2, in which acetic acid is directly coordinated to the metal with no additional deuterium source included in the intermediate. The molecular features in this model include the partial charge on one of the ring carbons (q5), the overall dipole of the intermediate (μtot), the component of the dipole moment perpendicular to the benzene ring (μz) and the maximal width of the directing group (Sterimol B5). The normalized equation shows that μtot and Sterimol B5 are the most significant features, which indicates that this model leans toward steric control of deuterium incorporation compared with other conditions. Furthermore, the features are localized only on the substrates and not the acetate, unlike in Conditions A and B. In this model, the substrate is influenced by both steric and electronics features, and deuterium incorporation is increased when the width of the substrates' directing group is minimized and the dipole moment and charge on the substrate increase. In all three models, acetate or acetic acid is connected to the cobalt centre. However, for Conditions A and B, the best-fitting intermediates involve a well-orchestrated organization of the deuterium source, acetate/acetic acid and substrate around the metal centre. By contrast, the intermediate identified for Condition C, relies solely on acetic acid. Interestingly, Conditions A and B use water as their deuterium source, whereas Condition C requires methanol. Thus, the intermediates that correlate with the experimental results might imply that the primary role of water as a deuterium source is deuterium shuttling at the product-determining transition state, whereas methanol might have a distinct role. The superior performance of the model stemming from Int2 under Condition C, which only contains acetic acid, led us to speculate that methanol does not participate in the deuterodemetallation step, but rather facilitates a prior deuterium exchange. This proposal was supported by the fact that methanol's increased miscibility in dichloroethane (DCE) compared to water could endow it with the propensity to more easily transfer deuterium to acetate49. Indeed, under Condition C D2O leads to poor deuterium incorporation. To test the hypothesis that under Condition C acetic acid might be the effective deuterating agent, we conducted experiments using AcOD-d4 with model substrates under each of the reaction conditions (Fig. 4b and Supplementary Tables 7–9). Gratifyingly, for acetanilide under Condition C, AcOD-d4 led to the same deuterium incorporation as CD3OD, supporting our hypothesis that acetic acid is the effective deuterating agent. By contrast, for 2-phenylpyridine under Condition A, AcOD-d4 led to a significant decrease in deuterium incorporation compared to D2O. This result fits well with the suggested intermediate Int5water, which requires the presence of both acetic acid and water to react in concert. Further supporting this hypothesis is the decrease in deuterium incorporation levels when switching from D2O to AcOD-d4 for 1-phenyl pyrazole under Condition B, for which Int3water that contains both an acetate and water provided the best fit. Interestingly, this decrease under Condition B is much less pronounced compared to the decrease under Condition A. We turned to the correlated models to seek an explanation for this difference in outcomes and specifically to the correlations under each of these conditions with Int2, which only contained acetic acid. For Condition B the second best-fitted model corresponded to Int2 (R2 = 0.75, Q2LOO = 0.63), whereas for Condition A this intermediate provided a poor fit (R2 = 0.68, Q2LOO = 0.56; Supplementary Fig. This result might suggest the existence of two competing pathways, which under Condition B are closer in energy and, therefore, could more easily be triggered by the presence of a different deuterium source. The work discussed here seeks to unlock the untapped potential of reaction intermediate descriptors to reveal hidden mechanistic structure–function relationships. This approach is complementary to insight garnered by DFT computations, which often provide a detailed account of the full reaction trajectory for a model substrate, but are seldom extended to multiple substrates and conditions. By integrating this approach with detailed experimental data from base-assisted directed HIE reactions, we demonstrate that beyond simple differences in deuterium incorporation levels, the choice of deuterium source might lead to a mechanistic shift, highlighting the need to treat deuterium sources with caution in mechanistic studies. Our analysis suggests that under Conditions A and B the deuterium source (D2O) participates directly in the deuteration step, whereas under Condition C isotope exchange with the deuterium source (CD3OD) probably occurs off-cycle. These mechanistic hypotheses were supported experimentally and illustrate the value of statistical models based on proposed reactive intermediates for uncovering hidden mechanistic complexity in catalytic reactions. Our findings reveal that the deuterium source can actively influence the reaction pathway, a factor that is critical for mechanistic interpretations, and which impacts reaction efficiency. Our work argues against assuming a single mechanistic pathway across substrates or metals and underscores the importance of evaluating multiple deuterium sources for each directing group. Furthermore, when employed in their non-deuterated form, these sources may mediate C–H bond cleavage when acting as solvents or additives. Therefore, the implications of this work extend beyond the specific catalytic system studied or HIE literature, as it reveals overlooked mechanistic distinctions that might be prevalent in organometallic catalysis. Deuterium incorporations were determined by 1H NMR using D1 = 1 s when taking aromatic signals as reference. For the two examples (59, 62) in which aliphatic signals need to be taken as reference, D1 = 50 s was used. Isolated yields were calculated taking into consideration the deuterium incorporation in the final product. Cross-validation of the deuterium incorporation with HRMS was performed in 13 selected examples across the three reaction conditions. Cp*Co(CO)I2 (11.9 mg, 0.025 mmol, 10 mol%) was charged into a 5 ml crimped vial and transferred to a glovebox, where KOAc (4.9 mg, 0.05 mmol, 20 mol%), AgSbF6 (17.2 mg, 0.05 mmol, 20 mol%), substrate (0.25 mmol, 1 equiv. (note: if the substrate is a liquid, it was added after the solvent)) and DCE (0.5 ml) were added. The vial was crimped, taken out of the glovebox and D2O (112 µl, 6.25 mmol, 25 equiv.) was added through the septum of the cap. The mixture was stirred for 2 hours at 50 °C. The crude was purified by column chromatography employing a AcOEt:hexane (1:1) mixture as eluent. Cp*Co(CO)I2 (11.9 mg, 0.025 mmol, 10 mol%) was charged into a 5 ml crimped vial and transferred to a glovebox, where Cu(OAc)2 (4.5 mg, 0.025 mmol, 10 mol%), AgSbF6 (17.2 mg, 0.05 mmol, 20 mol%), substrate (0.25 mmol, 1 equiv. (note: if the substrate is a liquid, it was added after the solvent)) and DCE (2.5 ml) were added. The vial was crimped, taken out of the glovebox and aniline (4.6 μl, 0.05 mmol, 20 mol% (note: only added for substrates that are aldehydes)) and D2O (45 μl, 2.5 mmol, 10 equiv.) were added through the septum of the cap. The mixture was stirred for 2 hours at 100 °C. The crude was purified by column chromatography employing a AcOEt:hexane (1:5) mixture as eluent. Cp*Co(CO)I2 (11.9 mg, 0.025 mmol, 10 mol%) was charged into a 5 ml crimped vial and transferred to a glovebox, where Cu(OAc)2 (4.5 mg, 0.025 mmol, 10 mol%), AgSbF6 (17.2 mg, 0.05 mmol, 20 mol%), substrate (0.25 mmol, 1 equiv. (note: if the substrate is a liquid, it was added after the solvent)) and DCE (5 ml) were added. The vial was crimped, taken out of the glovebox and CD3OD (102 μl, 2.5 mmol, 10 equiv.) was added through the septum of the cap. The mixture was stirred for 2 hours at 100 °C. The crude was filtered through a celite and silica pad. Deuterium incorporations were determined by 1H NMR spectroscopy of the crude. All substrates were modelled in a conformation directing the hydrogen and directing group in the same orientation. Substrates were computed using the M06-2X functional with the def2-TZVP basis set. Reaction intermediates were computed using the M06-2X functional with the 6-31g(d) basis set. For each intermediate, a set of computations was created, each time with a different substrate according to the experimental scope. After geometry optimization, features were extracted for each intermediate–substrate pair under each condition, and the resulting dataset was used to generate linear regression models. Feature extraction and modelling were performed using the moleculaR50 package in R. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data are available in the main text or Supplementary Information, or are available from the authors upon request. The datasets (in .csv format), .xyz files and the corresponding results are openly available via Zenodo at https://doi.org/10.5281/zenodo.17086123 (ref. 51) and via GitHub at https://github.com/Milo-group/Deciphering_HIE_Multiple_Pathways. Atzrodt, J., Derdau, V., Fey, T. & Zimmermann, J. The renaissance of H/D exchange. Atzrodt, J., Derdau, V., Kerr, W. J. & Reid, M. C−H functionalisation for hydrogen isotope exchange. Kopf, S. et al. Recent developments for the deuterium and tritium labeling of organic molecules. Prakash, G., Paul, N., Oliver, G. A., Werz, D. B. & Maiti, D. C–H deuteration of organic compounds and potential drug candidates. Barranco, S. & Pérez-Temprano, M. H. in Advances in Organometallic Chemistry Vol. 79 (ed Pérez, P. Gant, T. G. Using deuterium in drug discovery: leaving the label in the drug. Atzrodt, J., Derdau, V., Kerr, W. J. & Reid, M. Deuterium- and tritium-labelled compounds: applications in the life sciences. Pirali, T., Serafini, M., Cargnin, S. & Genazzani, A. Applications of deuterium in medicinal chemistry. Gensch, T. & Glorius, F. The straight dope on the scope of chemical reactions. Kariofillis, S. K. et al. Using data science to guide aryl bromide substrate scope analysis in a Ni/photoredox-catalyzed cross-coupling with acetals as alcohol-derived radical sources. Kozlowski, M. C. On the topic of substrate scope. Rana, D., Pflüger, P. M., Hölter, N. P., Tan, G. & Glorius, F. Standardizing substrate selection: a strategy toward unbiased evaluation of reaction generality. Schleinitz, J. et al. Designing target-specific data sets for regioselectivity predictions on complex substrates. Wells, P. R. Linear free energy relationships. Jencks, W. P. When is an intermediate not an intermediate? Enforced mechanisms of general acid-base, catalyzed, carbocation, carbanion, and ligand exchange reaction. Lustosa, D. M. & Milo, A. Mechanistic inference from statistical models at different data-size regimes. & Sigman, M. S. Predictive and mechanistic multivariate linear regression models for reaction development. Strieth-Kalthoff, F., Sandfort, F., Segler, M. H. S. & Glorius, F. Machine learning the ropes: principles, applications and directions in synthetic chemistry. Zahrt, A. F., Athavale, S. V. & Denmark, S. E. Quantitative structure-selectivity relationships in enantioselective catalysis: past, present, and future. Muratov, E. N. et al. QSAR without borders. Jorner, K., Tomberg, A., Bauer, C., Sköld, C. & Norrby, P. O. Organic reactivity from mechanism to machine learning. Williams, W. L. et al. The evolution of data-driven modeling in organic chemistry. Shalit Peleg, H. & Milo, A. Small data can play a big role in chemical discovery. Raghavan, P. et al. Dataset design for building models of chemical reactivity. Oslob, J. D., Åkermark, B., Helquist, P. & Norrby, P.-O. Steric influences on the selectivity in palladium-catalyzed allylation. Park, Y., Niemeyer, Z. L., Yu, Y.-Q. & Sigman, M. S. Quantifying structural effects of amino acid ligands in Pd(II)-catalyzed enantioselective C–H functionalization reactions. Anand, M. & Nørskov, J. K. Scaling relations in homogeneous catalysis: analyzing the Buchwald–Hartwig amination reaction. Lau, S. H. et al. Ni/Photoredox-catalyzed enantioselective cross-electrophile coupling of styrene oxides with aryl iodides. Boni, Y. T., Cammarota, R. C., Liao, K., Sigman, M. S. & Davies, H. M. L. Leveraging regio- and stereoselective C(sp3)–H functionalization of silyl ethers to train a logistic regression classification model for predicting site-selectivity bias. Deep learning for enantioselectivity predictions in catalytic asymmetric β-C–H bond activation reactions. et al. Nickel/biimidazole-catalyzed electrochemical enantioselective reductive cross-coupling of aryl aziridines with aryl iodides. Ikemoto, H., Yoshino, T., Sakata, K., Matsunaga, S. & Kanai, M. Pyrroloindolone synthesis via a Cp*CoIII-catalyzed redox-neutral directed C–H alkenylation/annulation sequence. Sanjosé-Orduna, J. et al. Capturing elusive cobaltacycle intermediates: a real-time snapshot of the Cp*CoIII-catalyzed oxidative alkyne annulation. Sanjosé-Orduna, J., Sarria Toro, J. M. & Perez-Temprano, M. H. HFIP-assisted C−H functionalization by Cp*CoIII: access to key reactive cobaltacycles and implication in catalysis. Moselage, M., Li, J. & Ackermann, L. Cobalt-catalyzed C–H activation. C–H functionalization by high-valent Cp*Co(III) catalysis. Chirila & Whiteoak, P. G. C. J. Recent advances using [Cp*Co(CO)I2] catalysts as a powerful tool for C–H functionalization. Yoshino, T. & Matsunaga, S. Pentamethylcyclopentadienyl)cobalt(III)-catalyzed C–H bond functionalization: from discovery tounique reactivity and selectivity. Baccalini, A., Vergura, S., Dolui, P., Zanoni, G. & Maiti, G. Recent advances in cobalt-catalysed C–H functionalizations Org. John-Campbell, S. & Bull, J. Transient imines as ‘next generation' directing groups for the catalytic functionalisation of C–H bonds in a single operation. Transient imine directing groups for the C–H functionalisation of aldehydes, ketones and amines: an update 2018–2020. Ertl, P., Altmann, E. & McKenna, J. M. The most common functional groups in bioactive molecules and how their popularity has evolved over time. Dhayalan, V., Gadekar, S. C., Alassad, Z. & Milo, A. Unravelling mechanistic features of organocatalysis with in situ modifications at the secondary sphere. Gadekar, S. C. et al. Rerouting the organocatalytic benzoin reaction toward aldehyde deuteration. Lustosa, D. M., Barkai, S., Domb, I. Effect of solvents on proline modified at the secondary sphere: a multivariate exploration. Lapointe, D. & Fagnou, K. Overview of the mechanistic work on the concerted metallation-deprotonation pathway. O´Duill, M. & Engle, K. M. Protodepalladation as a strategic elementary step in catalysis. Altus, K. M. & Love, J. The continuum of carbon–hydrogen (C–H) activation mechanisms and terminology. Rumble, J. R. CRC Handbook of Chemistry and Physics–Physical Constants of Organic Compounds (CRC, 2022). Barkai, S. MoleculaR: molecular feature extraction from Gaussian log files. R package version 1.0.0. Barranco, S. et al. Charting the influence of deuterium sources in hydrogen isotope exchange using a cobalt(III) catalytic platform. M.H.P.-T. thanks ICIQ, CERCA Program/Generalitat de Catalunya, the Ministerio de Ciencia, Innovación y Universidades/Agencia Estatal de Investigación MICIU/AEI/10.13039/501100011033 (PID-2020-112733GB-I00, PID2023-146130NB-I00 co-funded by FEDER,EU and Severo Ochoa Excellence Accreditation CEX2024-001469-S) and AGAUR (2021 SGR 01154). A.M. thanks the Israel Science Foundation (grant no. acknowledges the Ministerio de Universidades for a FPU predoctoral fellowship (FPU20/00610). thanks the China Scholarship Council for the predoctoral fellowship (CSC201906280437). acknowledges the PBC Scholarships for Outstanding Doctoral Female Students in High-Tech scholarship and the Kreitman Graduate School for her PhD funding. These authors contributed equally: Sergio Barranco, Inbal L. Eshel. Institute of Chemical Research of Catalonia (ICIQ), Tarragona, Spain Sergio Barranco, Jiayu Zhang, Marco Di Matteo & Mónica H. Pérez-Temprano Departament de Química Analítica i Química Orgánica, Universitat Rovira i Virgili, Tarragona, Spain Sergio Barranco & Jiayu Zhang Department of Chemistry, Ben-Gurion University of the Negev, Beer Sheva, Israel Inbal L. Eshel & Anat Milo Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Experimental mechanistic investigation: S.B. Computations and MLR analysis: I.L.E. All authors discussed the results and co-wrote the manuscript. Correspondence to Anat Milo or Mónica H. Pérez-Temprano. The authors declare no competing interests. Nature Catalysis thanks Volker Derdau and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–34, Discussion, Tables 1–9 and Scheme 1. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Barranco, S., Eshel, I.L., Zhang, J. et al. Charting the influence of deuterium sources in hydrogen isotope exchange using a cobalt(III) catalytic platform. Version of record: 25 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01706-2'>Heterochromatin boundaries maintain centromere position, size and number</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 10:14:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Centromeres are defined by a unique single chromatin domain featuring the histone H3 variant, centromere protein A (CENP-A), and ensure proper chromosome segregation. Centromeric chromatin typically occupies a small subregion of low DNA methylation within multimegabase arrays of hypermethylated alpha-satellite repeats and constitutive pericentric heterochromatin. Here, we define the molecular basis of how heterochromatin serves as a primary driver of centromere and neocentromere position, size and number. Using single-molecule epigenomics, we uncover roles for H3K9me3 methyltransferases SUV39H1/H2 and SETDB1, in addition to noncanonical roles for SUZ12, in maintaining H3K9me3 boundaries at centromeres. Loss of these heterochromatin boundaries leads to the progressive expansion and/or repositioning of the primary CENP-A domain, erosion of surrounding DNA methylation and nucleation of additional functional CENP-A domains across the same alpha-satellite sequences. Our study identifies the functional importance and specialization of different H3K9 methyltransferases across centromeric and pericentric domains, crucial for maintaining centromere domain size and suppressing ectopic centromere nucleation events. Centromeres are the chromosomal loci that drive chromosome segregation during mitosis and meiosis1,2. Human centromeres are characterized by multimegabase arrays of A+T-rich alpha-satellite DNA3,4. Intriguingly, these sequences, while abundant, are not strictly required for centromere function. Instead, centromere position in humans is, to a large extent, defined epigenetically. The key centromere-defining element is the histone H3 variant centromere protein A (CENP-A), which is sufficient to induce de novo centromere formation and appears as a central node in a self-propagating feedback loop that maintains centromere position5,6,7. Once formed, these neocentromeres are functional and propagated indefinitely in somatic cells and, in certain cases, are even transgenerationally inherited8,12,13. Moreover, by deleting the endogenous centromere locus in model systems, neocentromeres can be induced experimentally, which rescues chromosome stability14,15,16,17,18. In human cells, this approach allows us to explore the genetic and epigenetic contributions to centromere seeding and inheritance, as well as monitor the evolution of spontaneously formed centromeres. Centromeres are propagated by the stable transmission of CENP-A chromatin that serves as a template for the self-directed assembly of new CENP-A nucleosomes each cell cycle19,20,21. This self-propagating cycle is predicted to generate variation in the precise positioning of the centromere because of the dynamic nature of histone recycling during DNA replication and transcription22. Indeed, centromere drift has been observed in evolutionary new centromeres of donkeys and chicken Z chromosomes23,24. However, despite this local drift, vertebrate centromeres remain restricted to their broader chromosomal locus even if CENP-A is temporarily removed25 and their size remains restricted to ~100–500 kb at both canonical and neocentromeres3,18,26,27. The tightly constrained CENP-A domain is somewhat surprising as such ‘islands' of CENP-A typically reside in megabase-sized domains of homogenous alphoid repeats, indicating the existence of non-sequence-defined centromere boundaries. One potential boundary is 5mCG DNA methylation. Endogenous human centromeres occur within arrays of alpha-satellite higher-order repeat (HORs) that generally have hypermethylated DNA, except for a distinct site of hypomethylation coinciding with strong enrichment of CENP-A chromatin, termed centromere dip regions (CDRs)3,28. In addition, constitutive heterochromatin, marked by histone H3 lysine 9 trimethylation (H3K9me3), spans human satellite (HSat) DNA families in flanking pericentromeric regions, such as the classical HSats 1–3 (refs. 29,30), and has been hypothesized to maintain centromere position31. H3K9me3 has also been observed to at least partially overlap with centromeric repeats32,33,34,35,36 and was recently mapped within active alpha-satellite HOR arrays37, indicating that the chromatin environment at and around the CENP-A locus may be of functional importance. Whether heterochromatin forms a true functional boundary and how this is regulated remain open questions. We previously isolated a de novo human neocentromere in cultured human RPE1 cells (Fig. This neocentromere, on the p arm of Chr4, Neo4p13, is 90 kb in size and formed in a gene-poor region, within high levels of H3K9me3 constitutive heterochromatin18, analogous to canonical centromeres. This system offers a unique time-stamped centromere to understand how centromere position and size are maintained, following its formation, independent of satellite sequences (Fig. We took an early Neo4p13 isolate and proliferated three independent populations for 100 days. We then randomly isolated three clonal subpopulations from each independently evolved pool and assessed centromere position using CENP-A CUT&RUN (Fig. In early-passage cells (red), CENP-A mapped to a 90-kb window from 42.86 Mb to 42.96 Mb (Fig. Upon prolonged cell culture (day 100, gray), we observed a gradual centromere drift over time (Fig. 1d–f), with shifts in either direction and an average shift of peak center of 8.74 kb (Fig. This gradual movement of the CENP-A domain is consistent with previous studies in evolutionary new centromeres23,24,38. Strikingly, however, the overall size of the Neo4p13 locus remained relatively tight at approximately 90–100 kb even as this centromere moved (Fig. As the CENP-A domain drifts in one direction, CENP-A nucleosomes are depleted from the trailing end of the domain (for example, Fig. These observations suggest that mechanisms exist to tightly maintain centromere boundaries and size of this domain. b, Schematic predictions of centromere inheritance behavior over time. Centromeres may stay stationary, move left or right with tight size and boundary control or gradually expand over time. c, Schematic of long-term culture assay to monitor inheritance of Neo4p13, culturing early-passage Neo4p13 for 100 days, followed by single-cell sorting and CENP-A CUT&RUN. d, CENP-A CUT&RUN of early-passage Neo4p13 (red) versus 100-day clones (gray). Black dotted lines represent early-passage centromere position. The light-gray dotted line represents the position of clone 3.2 as an example of centromere shift. The degree of movement is indicated for each clone (arrows) e, Quantitation of CENP-A enrichment across the genomic region 42.8–43.0 Mb (Chr4 Hap2) represented in 1-kb bins. The gray trace represents the combined mean enrichment for three groups of day 100 clones, BR1, BR2 and BR3, with day 100 ± 1 s.d. (light gray) f, Relative CENP-A enrichment in three genomic regions: the initial Neo4p13 locus (defined as a 90-kb window from 42.868 to 42.958 Mb) and immediate flanking regions, 90 kb upstream (‘left (−90 kb)') and downstream (‘Right (+90 kb)'). Enrichment values are normalized to the early-passage control sample, represented by a baseline of 1 (dashed line). Data are presented as the mean values ± s.e.m. Bar colors highlight the initial Neo4p13 locus (red, opaque) and the outside regions (grays). To understand how neocentromere position is maintained, we mapped the local chromatin environment surrounding the CENP-A domain. The RPE1-Neo4p13 cells are diploid, effectively carrying two distinct centromere epialleles, one at the canonical centromeric alpha-satellites of chromosome 4 and one at 4p13 (Fig. To accurately map chromatin features in a haplotype-specific manner, selectively at the neocentromere allele, we used Oxford Nanopore Technologies (ONT)-based ultralong-read DiMelo-seq37, in combination with the cell-line-specific RPE1 reference genome39 (Fig. Neo4p13 maps to Chr4 haplotype 2 (Hap2), indicated by the presence of a 6mA peak at p13 on Hap2 specifically directed toward CENP-A and CENP-C (Fig. Consistent with our previous observations using ChIP-seq18, this locus is enriched with H3K9me3 on both haplotypes. We previously estimated around 100 CENP-A nucleosomes per haplotype in RPE1 cells40. Within a ~100-kb centromere domain, this translates to one in five nucleosomes containing CENP-A. Thus, while the loss of H3K9me3 is explained in part by the displacement of histone H3 by CENP-A, our findings suggest that CENP-A seeding also results in the active removal of H3K9 trimethylation. a, Schematic of the workflow of DiMelo-Seq and mapping to the diploid RPE1 genome. b,c, Enrichment of CENP-A (red), CENP-C (orange), H3K9me3 (green) and 5mCG (blue) at the genomic locus of Neo4p13, mapped to RPE1 Chr4 Hap1 (b) and Chr4 Hap2 (c). d–g, CENP-A enrichment (d; % 6mA/A, red), CENP-C enrichment (e; % 6mA/A, orange), % 5mCG/C (f; blue) and H3K9me3 (g; % 6mA/A, green) at the Neo4p13 locus for two haplotypes: Chr4 (Hap1) and Chr4 (Hap2). Data are aggregated across a 500-kb region centered at the Neo4p13 coordinates, with flanking regions extending 250 kb on either side. Mean enrichment values are calculated in 100 evenly spaced bins per region. Canonical alpha-satellite-based centromeres are characterized by distinct dips in 5mCG methylation at regions with high CENP-A occupancy3,28,41. Using DiMeLo-seq, we can directly measure endogenous 5mCG on the same single molecules on which we measure exogenous 6mA deposited around H3K9me3 or CENP-A. We found that the neocentromere arose in a broad domain of generally low DNA methylation (Fig. While we cannot directly establish causality, our data on a newly formed neocentromere suggest that CENP-A seeding can specifically drive local CpG demethylation (Fig. Previously, we determined that the overall A+T content of the 4p13 neocentromere region is roughly equal to that of the chromosome average18. Here, we analyzed the distribution of transposable elements (TEs) that have previously been implicated in centromere function, although the roles of TEs have been elusive42. Using the diploid RPE1 genome39, we characterized the genome-wide TE distribution using RepeatMasker (http://www.repeatmasker.org). We found that the 4p13 neocentromere locus features a higher number of long terminal repeats (LTRs) versus the chromosomal averages for Chr4 Hap2 (Extended Data Fig. 1a,b), most of which are present near the telomere-proximal side of the CENP-A peak (Extended Data Fig. In contrast, long (LINEs) and short (SINEs) interspersed nuclear elements are closer to the genome average (Extended Data Fig. We also analyzed the loci of previously characterized patient-derived neocentromeres on chromosomes 4, 8 and 13 (PDNC4, MS4221 and IMS13q, respectively)26,27. Unlike Neo4p13, the PDNC4 neocentromere at 4q21 is enriched in SINEs, with only a minor increase in LTRs (Extended Data Fig. IMS13q on chromosome 13 showed a slight enrichment for LINEs (Extended Data Fig. Interestingly, at MS4221, although overall depleted in LINEs, CENP-A maps precisely onto LINE1 elements at this neocentromere (Extended Data Fig. Thus, while interspersed repeats are generally associated with neocentromere activity, there appears to be no unique repeat linked to centromere function. In all cases analyzed, there were numerous other regions across the chromosome that feature higher numbers of all three TE classes (Extended Data Fig. 1), making TEs unlikely to be defining features of neocentromeres. Furthermore, not all neocentromere-forming regions appear to be strongly enriched in broad domains of H3K9me3. Known patient-derived neocentromeres on chromosomes 4q and 13q have appeared in regions that, at least in our RPE genome analysis, are depleted of H3K9me3 or contain more refined H3K9me3 peaks rather than a broad domain of H3K9me3 (Extended Data Fig. These lower levels of H3K9me3 were also observed at different neocentromeres across the 13q arm27, as well as at the 4q21 neocentromere, where variation in H3K9m3 levels correlates with changes in CENP-A position44. Given the maintenance of size control of the 4p13 neocentromere that we observed in long-term culture, we hypothesized that adjacent H3K9me3-marked heterochromatin forms an effective boundary (Fig. We predicted that the H3K9me3 methyltransferases SUV39H1 and SUV39H2 may be responsible for maintaining the heterochromatin domain at 4p13, as these are known to mark pericentric heterochromatin at canonical centromeres45,46,47. We also considered the H3K9me3 methyltransferase SETDB1 as a putative candidate, as this enzyme is known to have roles distinct from SUV39 enzymes, particularly in TE silencing47,48,49. a, Model showing that heterochromatin acts as a boundary that limits neocentromere expansion and maintains locus size. Histone H4 and α-tubulin were used as loading controls (≥2 independent replicates). c, CUT&RUN profiles across the Neo4p13 region for CENP-A, H3K9me3 and H3K27me3 in indicated genotypes; dashed lines mark Neo4p13 centromere boundaries. d, CENP-A CUT&RUN comparing early-passage parental Neo4p13 to SUV39H1Δ/H2Δ clones at early passage and after ~100 days; black dotted lines show the initial centromere position and arrows indicate the shift magnitude. e, CENP-A enrichment (1-kb bins) across 42.8–43.0 Mb (Chr4 Hap2) for parental (red) versus SUV39H1Δ/H2Δ clones. Early-passage traces are shown in orange and the day 100 mean (BR1–BR3) ± s.d. f, CENP-A enrichment within initial Neo4p13 interval (42.868–42.958 Mb) and flanks 90 kb upstream (‘left') and downstream (‘right'), for SUV39H1Δ/H2Δ clones normalized to parental early passage (baseline = 1). Data are shown as the mean ± s.e.m. g, CENP-A CUT&RUN comparing early-passage parental Neo4p13 to SETDB1Δ clones, analogous to d. h, CENP-A enrichment analogous to e but for parental versus SETDB1Δ clones. i, CENP-A enrichment within initial Neo4p13 interval and flanks, analogous to f but for SETDB1Δ clones. Data are shown as the mean ± s.e.m. j, CENP-A CUT&RUN comparing early-passage parental Neo4p13 to SUV39H1Δ/H2Δ/SUZ12Δ clones, analogous to d. k, CENP-A enrichment analogous to e but for parental versus SUV39H1Δ/H2Δ/SUZ12Δ clones. l, CENP-A enrichment within initial Neo4p13 interval and flanks, analogous to f but for SUV39H1Δ/H2Δ/SUZ12Δ clones. Data are shown as the mean ± s.e.m. We constructed SETDB1-null (SETDB1Δ) and SUV39H1/H2-double-null mutant lines (SUV39H1Δ/H2Δ). Loss of SUV39H1/H2 resulted in a modest reduction in global H3K9me3 levels, likely because of other H3K9 methyltransferases (Fig. In contrast, SETDB1 loss did not affect overall H3K9me3 levels detectable by immunoblot (Fig. Using H3K9me3-directed CUT&RUN, we found that deletion of SUV39H1/H2 but not SETDB1 resulted in the near-complete loss of H3K9me3 from the neocentromere domain, except for some discrete H3K9me3 peaks (Fig. While the CENP-A domain maintained its position in SETDB1 mutants (Fig. 3g–i, red tracks), loss of H3K9me3 in SUV39H1Δ/H2Δ cells resulted in a substantial drift of the CENP-A domain in a telomere-proximal direction, following the generation and expansion of mutant cells (~50 days in culture) with a telomere-proximal peak center shift of 14.2 kb (Fig. To assess long-term adaptation, we used the clonal culture strategy, outlined in Fig. Following an additional 100 days of continuous proliferation, all SUV39H1Δ/H2Δ clones moved to a more telomere-proximal position compared to parental Neo4p13 (Fig. Moreover, CENP-A enrichment is generally higher across all clones versus parental early-passage cells. Importantly, the overall size of the CENP-A domain remained largely consistent at 90–110 kb in evolved clones, suggesting another layer of size control that maintains neocentromere size, preventing spreading beyond a confined locus. While deletion of SETDB1 did not result in notable changes in H3K9me3 and CENP-A positioning initially, analysis of 100-day long-term passaged clones revealed extensive heterogeneity in CENP-A position, in either direction, with a mean shift of peak center of 30.2 kb (Fig. To determine whether SETDB1 possibly has a redundant role in regulating H3K9me3 at the 4p13 neocentromere, we compared H3K9me3 CUT&RUN data in SUV39H1Δ/H2Δ, SETDB1Δ and combined SUV39H1Δ/H2Δ/SETDB1Δ mutant backgrounds (Extended Data Fig. Upon removal of SUV39H1 and SUV39H12, there remained some more refined H3K9me3 peaks near LTRs within the Neo4p13 region that were removed upon additional SETDB1 deletion. These findings suggest that SETDB1 has a more subtle role in silencing TE elements in this region and a contributing role in maintaining CENP-A boundaries. We also analyzed the role of H3K9me3 modifiers in heterochromatin distribution at other RPE1 loci known to be associated with patient-derived neocentromeres (PDNC4, IMSq13 and MS4221)26. We found that the discrete H3K9me3 peaks surrounding the PDNC4 and IMSq13 neocentromere loci were largely dependent on SETDB1 (Extended Data Fig. In contrast, H3K9me3 at the MS4221 neocentromere locus appeared more broadly SUV39-regulated (Extended Data Fig. Thus, neocentromere loci appear to be at least in part influenced by SETDB1 and/or SUV39H1/H2 activity. Indeed, we observed an increase in Polycomb repressive complex 2 (PRC2)-mediated H3K27me3 facultative heterochromatin upon loss of SUV39-mediated H3K9me3 at 4p13 (Fig. 5a), consistent with previous studies29, a phenomenon also observed on H3K9me3-depleted human artificial chromosomes50. To remove H3K27me3, we disrupted the gene encoding suppressor of zeste 12 (SUZ12), an essential component of the PRC2 complex (Fig. Deletion of SUZ12 resulted in a complete loss of H3K27me3 in SUV39H1Δ/H2Δ/SUZ12Δ triple-mutant Neo4p13 cells (Fig. Strikingly, size control of the CENP-A domain is largely lost in this context, resulting in a doubling in size compared to the original Neo4p13 locus (Fig. CENP-A moves much further than in SUV39H1Δ/H2Δ to a size up to 150 kb and breaching into the nearest gene boundary (GRXCR1), with a mean peak center shift of 41.3 kb. Combined, these results indicate that heterochromatin marked by H3K9me3 and H3K27me3 can form clear functional boundaries at this neocentromere and maintain tight size control of the CENP-A domain, independent of 5mCG methylation and alpha-satellites. Next, we investigated whether similar boundary mechanisms exist at canonical centromeres. Characteristic features of canonical centromeres, such as repetitive alpha-satellite DNA and high levels of DNA methylation, may also contribute to maintaining CENP-A positioning. To assess chromatin organization at highly repetitive satellite DNA, we used ONT ultralong-read DiMelo-seq in combination with the isogenomic RPE reference genome39 to accurately map CENP-A, CENP-C and H3K9me3 occupancy at canonical centromeres (Fig. We performed this analysis in the same RPE1-Neo4p13-bearing cell line for consistency across our experiments. As ONT enables direct genomic DNA sequencing lacking PCR amplification, we can determine the single-molecule occupancy of our target of interest across centromeres or pericentromeres. a, Diagram of DiMeLo-Seq workflow with averaged 6mA/A probabilities and single-molecule alignments. Vertical lines of each read represent exogenous 6mA directed toward CENP-A, CENP-C or H3K9me3. b, View of the entire HOR of CENX (Hap1), with DiMelo-Seq enrichments for CENP-A (red), CENP-C (orange) and H3K9me3 (green), as well as 5mCG enrichment (blue). Single molecules of each long read for H3K9me3 (black). Specifically where CENP-A and CENP-C reside, there is the characteristic dip in DNA methylation (5mCG) and H3K9me3 heterochromatin. c, A closer view of CENX, showing the dip in H3K9me3 and boundaries surrounding the active CENP-A/CENP-C-containing region. Single molecules at this site contain largely unmethylated adenine-called reads. d, Heat maps illustrating the distribution of 5mCG (top, blue) and H3K9me3-directed 6mA (bottom, green) modifications across centromeric active HOR regions. Each line on the heat map spans the active HOR of a single centromere and flanking regions ±1 Mb upstream and downstream. All active HOR regions were scaled to the same length to allow direct comparison of methylation patterns across all centromeres. Signal intensity is represented as the mean methylated fraction across fixed genomic bins, with darker shades indicating higher modification levels. The start and end of the HOR regions are indicated, highlighting methylation patterns in and around these active centromeric domains. e, Heat maps as in d but specifically across CDRs, as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region. All CDRs were scaled to the same length. f, Individual traces for each centromere (gray) and aggregated (colored) modification traces of CENP-A, CENP-C, H3K9me3 and 5mCG across all CDRs, centered on CDR midpoints, with ±250-kb flanking regions. To determine the specificity of the CENP-A DiMelo-seq 6mA signal, we used an endogenous homozygous knock-in AID-eYFP-CENP-A hTERT-RPE1 cell line to auxin-deplete all endogenous CENP-A52, followed by CENP-A DiMeLo-seq (Extended Data Fig. CENP-A was effectively depleted from cells (Extended Data Fig. 2b) and the corresponding CENP-A 6mA signal was specifically eliminated from all CDRs, indicating that 6mA is CENP-A specific and providing us with a true background CENP-A 6mA/A signal (Extended Data Fig. Consistent with previous observations3,28,53, active alpha-satellite HOR arrays generally contained high levels of DNA methylation (Fig. Moreover, active centromeric regions containing CENP-A featured the previously characterized CDRs, depleted of 5mCG, which tightly inversely correlate with all CENP-A (red) and CENP-C (orange) containing regions (Fig. Interestingly, 5mCG levels were unaffected by acute loss of CENP-A (Extended Data Fig. 2d–f), indicating that the local dip in DNA methylation is relatively stable, at least within a few hours of CENP-A depletion. To determine the resolution of our H3K9me3 DiMeLo-seq, we first assessed H3K9me3 in pericentric regions, specifically using HSat3 as an example (Extended Data Fig. We could achieve near-single-nucleosome resolution for H3K9me3, giving us an estimate of methylated nucleosome densities in these regions that is consistent with the reported average chromatin repeat unit length for HSat3 (Extended Data Fig. In the classic centric–pericentric dichotomy, heterochromatin, marked by H3K9me3, is considered to be largely pericentric, whereas the kinetochore-forming CENP-A domain is largely depleted of heterochromatin29,30,35,58. Moreover, centromeres on human artificial chromosomes resist silencing mediated by H3K3me3 and H3K27me3 (ref. 50) and H3K9me3 is not required for CENP-A-driven epigenetic memory59, suggesting an inverse relationship and functional boundary potential. However, using DiMeLo-seq, it was unambiguously shown that H3K9me3 is also present within HORs that form a template for CENP-A37. Consistent with these data, we found high levels of H3K9me3 across all pericentromeres and active HORs in RPE1 (Fig. Importantly, we identified a distinct depletion in H3K9me3 within the CENP-A domain at all centromeres, forming a tight boundary around the CENP-A/CENP-C domain in a manner analogous to 5mCG (Fig. 4c,e,f and Extended Data Fig. Thus, the CDR is defined not only by depletion of 5mCG but also by a dip in the heterochromatin mark H3K9me3. To uncover the importance of heterochromatin boundaries at canonical alpha-satellite-based centromeres, we assessed H3K9me3 levels in the SETDB1Δ, SUV39H1Δ/H2Δ and SUV39H1Δ/H2Δ/SUZ12Δ double and triple mutants described above. We found that, in SUV39H1Δ/H2Δ double mutants, H3K9me3 levels were decreased both across the flanking pericentromere and within centromeric active HORs, compared to parental cells (Fig. Surprisingly, we found that SETDB1 contributes to H3K9me3 deposition specifically within the centromeric HORs and not in the flanking pericentric regions (Fig. 5a,c), indicating it has centromere-specific roles distinct from SUV39 enzymes. a, Heat maps for WT, SUV39H1Δ/H2Δ, SETDB1Δ and SUV39H1Δ/H2Δ/SUZ12Δ cell lines, illustrating the distribution of 5mCG (blue) and H3K9me3-directed 6mA (green) modifications across centromeric active HORs. Each line on the heat map spans the active HORs of a single centromere and flanking regions ± 1 Mb upstream and downstream. All active HOR regions were scaled to the same length to allow direct comparison of methylation patterns across all centromeres. b, Heat maps as in a but specifically across CDRs, as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region. All CDRs were scaled to the same length. c, Average DNA methylation signal left; (in % 5mCG/C) and H3K9me3 signal (right; in % 6mA/A), across all active HOR regions for indicated mutants (scaled equally), flanked by ±1 Mb. d, Average DNA methylation signal (left; in % 5mCG/C) and H3K9me3 signal (right; in % 6mA/A) for indicated mutants, across all CDRs (scaled equally) regions flanked by ±250 kb. SUV39H1Δ/H2Δ/SUZ12Δ triple mutants offered the most prominent effect on H3K9me3. Whilst levels were reduced in the pericentromeric regions to similar levels as in SUV39H1Δ/H2Δ double-mutant cells, there was a striking and substantial loss of H3K9me3 that specifically affected active HORs, even beyond that observed in SETDB1Δ cells (Fig. Hence, whilst loss of SUV39H1 and SUV39H2 generally reduced H3K9me3 in pericentric and alpha-satellite-containing chromatin, SETDB1 and SUZ12 deletion appeared to have selective effects on alpha-satellite H3K9me3. Next, we determined whether loss of H3K9me3 in SUV39H1Δ/H2Δ and SETDB1Δ mutants resulted in compensation by H3K27me3, assessed by DiMeLo-seq. We found no strong enrichment for the PRC2-mediated mark within the HOR in parental active HORs (Extended Data Fig. Loss of H3K9me3 in SUV39H1Δ/H2Δ and SETDB1Δ mutants did not result in any compensatory H3K27me3 deposition, except for pericentric, divergent satellites in SUV39H1Δ/H2Δ (Extended Data Fig. These observations indicate that the contributing role of SUZ12 to maintaining H3K9m3 in SUV39H1Δ/H2Δ mutants is not mediated by H3K27me3 but possibly through a direct noncanonical contribution of this PRC2 subunit to heterochromatin maintenance. We then analyzed the effect of heterochromatin mutants on H3K9me3 boundaries at the CENP-A-containing DNA methylation-depleted regions (CDRs). We found local H3K9me3 boundaries to be effectively eliminated in SUV39H1Δ/H2Δ/SUZ12Δ backgrounds, with visible reductions in boundary strength also apparent in SUV39H1Δ/H2Δ and SETDB1Δ (Fig. Importantly, the 5mCG-defined CDRs were largely maintained (Fig. Therefore, in this context, we primarily assessed the contributions of H3K9me3 to centromere position, largely independent of the role of DNA methylation or H3K27me3 chromatin. Next, having established means to selectively deplete heterochromatin from active HORs and pericentric HSats, we assessed its contribution to CENP-A positioning. In parental cells, we observed a tight positioning of CENP-A and CENP-C within the CDR, with levels rapidly dropping off outside of the 5mCG CDR boundaries (Figs. In SETDB1Δ, SUV39H1Δ/H2Δ and SUV39H1Δ/H2Δ/SUZ12Δ mutants, we observed a notable overall expansion of CENP-A following ~50 days in culture (time to generate these knockouts) (Fig. The majority of centromeres moved beyond their original CDR boundaries of parental cells, most prominently in the SUV39H1Δ/H2Δ/SUZ12Δ triple-mutant cells, where H3K9me3 levels were the most reduced. The CENP-A enrichment ratio within versus outside the CDR in control samples dropped in both mutants (Fig. 6d), even below a ratio of 1 in many centromeres, indicating that CENP-A became more enriched in the surrounding 100-kb region than within the originally defined centromere boundaries. a, Whole-HOR view of CEN8 (Hap1) in parental (RPE-Neo4p13), SUV39H1Δ/H2Δ, SETDB1Δ and SUV39H1Δ/H2Δ/SUZ12Δ lines showing aggregated DiMeLo-seq for CENP-A (red), CENP-C (orange), 5mCG (blue) and H3K9me3 (green) plus single-molecule alignments of CENP-A–directed 6mA (black). b, Heat maps of CENP-A and CENP-C across CDRs, defined by 5mCG dips flanking each parental CENP-A region. Each row is one CDR or centromere; CDRs are numerically sorted and scaled by length. Left, total CDR/sub-CDR length relative to the corresponding parental centromere. Right, CENP-A domain length relative to parental. Box plots summarize parent and mutants with individual centromeres as points. Ratio of CENP-A signal within CDRs versus flanks (±100 kb). All 5mCG CDR boundary profiles are centered on boundaries (±100 kb); the mean aggregated 5mCG is shown for each sample. CENP-A enrichment inside versus outside the parental 5mCG-defined CDR boundary (±100 kb); the mean aggregated CENP-A (6mA/A) trace is shown for each sample. Box plots of 6mA enrichment density per 10 kb for reads within CENP-A peaks, normalized to matched IgG controls (one BR). Quantitative analysis of 6mA read density normalized to IgG (Extended Data Fig. 6) across the HORs revealed that CENP-A and CENP-C density was largely maintained within CENP-A peaks in SETDB1Δ and increased in SUV39H1Δ/H2Δ conditions (Fig. Notably, CENP-A and CENP-C densities did not fall below parental density, indicating that the total copy number of CENP-A nucleosomes (and, to a lesser extent, CENP-C) across HORs was increased rather than diluted across the expanded CDRs (Fig. Interestingly, the DNA methylation dips were largely maintained, even if H3K9me3 levels were uniformly low across the CDR and adjacent HORs (Figs. Nevertheless, we could observe erosion of 5mCG at the centromere boundaries, with a mean increase in CDR size of 18% (SETDB1Δ), 19% (SUV39H1Δ/H2Δ) and 56% (SUV39H1Δ/H2Δ/SUZ12Δ), indicating that the H3K9me3-depositing machinery had an indirect effect on DNA methylation (Fig. In summary, the expansion of CENP-A correlates with the loss of H3K9me3 and, to a lesser extent, with a reduction in DNA methylation at the boundary. These observations indicate that H3K9me3 heterochromatin is the primary functional boundary maintaining CENP-A position. We find that, in heterochromatin mutants, the dip in DNA methylation (CDR) is largely maintained (Fig. Nevertheless, the tight correlation between the CENP-A domain and reduced DNA methylation3,28,41 suggest a functional link. Indeed, recent experiments directly implicated CpG methylation in maintaining CENP-A levels and boundaries60. The de novo DNA methyltransferase DNMT3B is associated with centromeric and pericentromeric components in various contexts, including CENP-C61, Mis18α (ref. We deleted DNMT3B in parental and SETDB1Δ backgrounds to assess any individual or combinatorial contributions toward CENP-A position (Extended Data Fig. Whilst DNMT3BΔ generally reduced 5mCG methylation in the active HOR by 26% (Extended Data Fig. 7b), CENP-A position remained relatively constant at the vast majority of centromeres (Extended Data Fig. The removal of DNMT3B in the context of a SETDB1 mutant in which heterochromatin boundaries are eroded resulted in a more notable loss of CENP-A domain definition (Extended Data Fig. For instance, in the absence of SETDB1, loss of DNMT3B led to CENP-A-invaded monomeric satellite regions flanking the active HOR at CEN16 Hap1 (characterized by non-5mCG-methylated adjacent satellites) (Extended Data Fig. These results indicate a role for DNMT3B in maintaining DNA methylation, particularly when heterochromatin boundaries are defective. Given our observations that Neo4p13 drifts gradually over time (Fig. 1d), we sought to determine whether canonical centromeres display a similar behavior. 3j) in which the CENP-A domain at the Neo4p13 locus shifted substantially following 100 days of proliferation. We then performed CENP-A DiMelo-seq to determine any differences in CENP-A position and size over time. Similar to the neocentromere, canonical centromeres move subtly with time over the 100-day period (Extended Data Fig. Parental centromere length (Extended Data Fig. 8c) and CDR boundary definition (Extended Data Fig. 8e) were largely maintained or subtly eroded after 100 days. CENP-A position began to gradually shift from the initial centromere boundary (Extended Data Fig. In contrast, already in early-passage SUV39H1Δ/H2Δ/SUZ12Δ cells and in day 100 clones, the CENP-A domain shifted substantially beyond any drift observed in parental cells, even following 100 days of proliferation (Extended Data Fig. In addition, centromere length expanded (Extended Data Fig. 8c) and CDR definition was greatly eroded (Extended Data Fig. We also observed cases where CENP-A position almost entirely vacated the parental position in the SUV39H1Δ/H2Δ/SUZ12Δ 100-day background (Extended Data Fig. Interestingly, here, 5mCG methylation began to reoccupy the initial CDR in the absence of CENP-A (Extended Data Fig. This suggests that, in addition to our observation at the neocentromere, where seeding CENP-A coincides with a loss of DNA methylation, the inverse can also happen, where the gradual loss of CENP-A results in the reestablishment of DNA methylation. Moreover, our assessment of long-term CENP-A domain dynamics indicates that the dramatic movement of CENP-A in methyltransferase mutants is the result of the loss of these enzymes and not simply a consequence of clonal selection and long-term culture. In addition to the expansion of existing CENP-A boundaries, we observed many cases where the loss of heterochromatin resulted in entirely new CENP-A peaks. These formed distal to the primary CENP-A domain but within the same alpha-satellite HORs and correlated with the loss of 5mCG and H3K9me3, thus forming characteristically similar chromatin features to the primary active centromere or CDR. For example, CEN1 and CEN9 of SUV39H1Δ/H2Δ/SUZ12Δ both contained new CDRs ~3 Mb away from the original primary CDR (Fig. SETDB1Δ cells similarly showed specific examples of new CDRs at CEN19 and CEN20 (Fig. Furthermore, the presence of the key CCAN component CENP-C indicates that these ectopic CENP-A domains are likely functionally active as centromeres (Fig. Thorough analysis using CDR-Finder63 followed by manual curation (Methods) identified all new CENP-A-containing CDRs that formed distally on the same alpha-satellite HOR in SUV39H1Δ/H2Δ/SUZ12Δ and SETDB1Δ mutants (Supplementary Table 1) and two new CDRs in DNMT3BΔ. A new CENP-A peak was also present on CEN21 Hap1 of parental cells following 100 days of culture, suggesting that new CENP-A domain seeding could be a general phenomenon that occurs at low frequency even in wild-type cells over time. Noticeably, some HOR regions appear to be preferred sites for new CENP-A domain formation across our cell lines (for example, Chr9, Chr20 and Chr21). No new CDRs appeared in SUV39H1Δ/H2Δ cells. a, Representative genome tracks for the entire alpha-satellites of CEN1 (Hap1), CEN9 (Hap2), CEN20 (Hap2) and CEN19 (Hap2), showing CENP-A (red), CENP-C (orange), 5mCG (blue) and H3K9me3 (green) for parental, SUV39H1Δ/H2Δ, SUV39H1Δ/H2Δ/SUZ12Δ and SETDBΔ cell lines. b, Representative immunofluorescence and mFISH images of metaphase spreads, showing karyotype and dicentric chromosomes for SETDB1Δ and SUV39H1Δ/H2Δ/SUZ12Δ cells as per a, identified by CENP-C (orange) and chromosome-specific mFISH paint. Scale bar, 10 μm (n = 2 independent experiments, with a minimum of 20 nuclei per replicate). c, Model summarizing the differential role of H3K9me3 methyltransferases at neocentromeres and canonical centromeres and their role in defining CENP-A domain size, position and number. Left, loss of SUV39H1/H2-mediated heterochromatin (H3K9me3) and compensatory H3K27me3 (PRC2) at Neo4p13 results in expansion and drift of the neocentromere. SETDB1 contributes to localized H3K9me3, possibly at TEs. Right, loss of H3K9me3 at canonical alpha-satellite centromeres leads to centromere expansion, drift and nucleation of new centromeres at distal sites on the alpha-satellite, creating new CDRs. SUV39H1/H2 broadly contributes to H3K9me3 deposition, while SETDB1 and SUZ12 action is largely restricted to HORs. We found all mutant lines to generally maintain a diploid karyotype as measured by DNA content (Extended Data Fig. 9a), although we observed a small polyploid population in the SUV39H1Δ/H2Δ double-mutant and SUV39H1Δ/H2Δ/SUZ12Δ triple-mutant cells (Extended Data Fig. We did not observe new CENP-A domains and CDRs in the SUV39H1Δ/H2Δ, indicating that polyploidy of these cells is not directly linked to ectopic CENP-A domains but rather to other defects, induced by heterochromatin loss. Furthermore, using spectral karyotyping, we identified translocations and aneuploidies of specific chromosomes across each cell line (Extended Data Fig. However, we did not find that the translocations resulted from centromeric breakages and appeared to occur independently of new CENP-A peaks. We observed some instances of HOR destabilization not observed in parental cells (for example, CEN9 Hap2 in SETDB1Δ; Fig. However, the majority of HORs appeared stable and translocation events appeared not to be directly linked to centromere defects. Lastly, to determine whether the ectopic CENP-A peaks form distinct centromere domains, we directly visualized these new centromeres on mitotic chromosomes. Using a combination of multicolor FISH (mFISH) and immunofluorescence for CENP-C, we could resolve dicentric chromosomes in metaphase spreads for chromosomes 1 and 9 of SUV39H1Δ/H2Δ/SUZ12Δ mutants and chromosomes 19 and 20 in SETDB1Δ cells. Here, CENP-C appeared as distinct double loci at these centromeres (Fig. 7b, orange), corresponding to the new CENP-A and CENP-C DiMeLo-seq peaks (Fig. 7a), indicating that these loci form functional dicentrics with two spatially adjacent CENP-A domains within the same alpha-satellite array. In this study, we discovered that the human neocentromere Neo4p13 (ref. 18) is enriched in SUV39H1-dependent and SUV39H2-dependent H3K9me3 but is formed in generally low levels of DNA methylation and lacks alpha-satellites, indicating that the latter are not the primary contributors to centromere positioning, at least at this neocentromere. We found that both H3K9me3 and redundant H3K27me3 within this domain form effective boundaries, preventing CENP-A domain drift and expansion (Fig. We hypothesize that these heterochromatin marks are maintained through self-templated propagation, forming a competitive, incompatible barrier for CENP-A deposition. In this context, it is interesting to note that treatment with an HDAC inhibitor and resulting increase in histone acetylation were also shown to cause CENP-A domain movements at a human neocentromere, an early indicator that changes to heterochromatin affect centromere boundaries64. We found that, while loss of SETDB1 had little impact on H3K9me3, it results in movement of the CENP-A domain, indicating that a subtle impact on H3K9me3 levels (particularly at TEs) may have notable consequences for the centromere boundaries (Fig. Alternatively, SETDB1 may impact other aspects of chromatin, such as H3K9me2, as recently identified at the centromere65, or possibly a nonenzymatic role, which requires further investigation. As at the 4p13 neocentromere, we found H3K9me3 chromatin to be depleted within the CENP-A domains at satellites, correlating with a dip in CpG methylation. Perturbation of these H3K9me3 boundaries resulted in substantial movement and expansion of CENP-A domains at all centromeres. Importantly, while loss of H3K9me resulted in some erosion of CpG methylation at the CDR boundary, the DNA methylation dips were largely maintained. Recent work showed that strong and rapid depletion of DNA methylation within centromeric HORs results in an increase in CENP-A assembly within the CDR and alterations to CENP-A boundary definition60. Our work now demonstrates that DNA methylation may not act as the sole boundary to maintain the CENP-A domain and heterochromatin has a critical role in restricting the functional CENP-A domain, particularly at the long timescales we analyzed. Centromeric and pericentromeric repeats are largely repressed by H3K9me3-associated chromatin37,54,55. How H3K9me3 is regulated within the centromere has remained unclear. Our data support a general role for SUV39 methyltransferases at both pericentric HSat sequences and centromeric alpha-satellites. A role for SETDB1 was also recently observed at centromeres in K562 leukemia cells65. We discovered that SETDB1 acts as a methyltransferase primarily at active alpha-satellite HORs (Fig. SETDB1 is known to drive heterochromatinization and silence TEs47,48,49. While the reason for the HOR-specific effect is unknown, active HORs, like TEs, are highly methylated compared to adjacent inactive arrays3,28. Possibly, the high degree of methylation contributes to SETDB1 recruitment. While pure alpha-satellites enriched in CENP-A are usually devoid of TEs3, they may nevertheless share similar features. For instance, members of the diverse family of zinc finger DNA-binding factors that normally recruit SETDB1 complexes to TEs were recently implicated in heterochromatin initiation and directly bind pericentric DNA in a sequence-specific manner66. Understanding whether cryptic sequences within active HORs contribute toward SETDB1 targeting is an important future goal. Indeed, it was previously shown that SUZ12 regulates H3K9 methylation and HP1α distribution in flies67. Furthermore, proteomic analyses found PRC2 components enriched on H3K9me3 chromatin, indicating extensive overlap in H3K9me3 and H3K27me3 binding profiles68,69. Both EZH2 methyltransferase and SUZ12 appear to be required for HP1 protein stability, and HP1 binding to H3K9me3 peptides is greatly increased in the presence of H3K27me3 (ref. On the basis of this functional overlap, loss of SUZ12 may destabilize and erode H3K9me3-marked heterochromatin, consistent with our observations. While we cannot exclude the secondary effects of long-term gene deletions in this study, these observations suggest a possible direct functional interaction among SETDB1, SUV39 and SUZ12 at centromeric repeats, which requires further careful biochemical and genetic dissection. Prior evidence suggests that centromere nucleation is a concentration-dependent mechanism5,6, requiring a critical concentration of CENPs to trigger an active centromere state. Indeed, CENP-A is loaded onto chromatin at low concentrations outside the centromere40,71,72 but is removed through DNA replication except at the centromere71. A key function of repressive chromatin surrounding the active centromeric locus may be to maintain a high local concentration of CENP-A. Our discovery of new CENP-A domains distal to the primary CDR, which appear to form functional centromeres in heterochromatin-deficient mutants, is consistent with a role for heterochromatin and DNA methylation60, not only in acting as a boundary at existing CENP-A domains but also in suppressing aberrant centromere formation at regions outside of the primary active centromere locus. As centromeric transcription appears to be implicated in CENP-A function73 and is indeed activated upon centromere repositioning74, one possible role of H3K9me3 and DNA methylation may be to constrain centromeric transcription and associated chromatin remodeling specifically within CDRs, where CENP-A assembles. The new CENP-A domains we observe in heterochromatin mutants occur within the HOR and likely act as functional dicentrics that are mitotically stable and generally permissive to chromosome segregation, consistent with previous work75. Nevertheless, multiple CENP-A domains within the HOR may affect mitotic fidelity, particularly inducing merotelically because of a loss of a tight bioriented centromere architecture76. Moreover, the recent description of the bipartite organization of the centromere and kinetochore may be impacted by ectopic CENP-A domains, even if proximal to the canonical site, which may impact chromosome segregation fidelity77. In sum, we identified heterochromatin and its associated H3K9 methyltransferases as an important contributor to maintaining CENP-A boundaries. How these boundaries are set at the remarkably constant size of about 100 kb, independent of underlying sequence context, remains an open question. Understanding the dynamic feedback involving CENP-A chromatin, its boundaries and the centromere–kinetochore complex is an important future goal. All human cell lines were grown at 37 °C, 5% CO2. The RPE-Neo4p13 cell line was generated as previously described18. For SUV39H1/H2, SETDB1 and DNMT3B deletion in the RPE-Neo4p13 background, guide RNAs (gRNAs) were designed (Supplementary Table 2; synthesized by Merk Life Sciences) and cloned into LentiCRISPRv2-Blast (Addgene, 83480) according to the standard lentiCRISPRv2 cloning protocol79. Lentivirus was produced in HEK293T cells in a T25 flask using a plasmid mixture (0.62 µg of pMD2 (Addgene, 12259), 1.87 µg of psPAX2 (Addgene, 12260) and 2.5 µg of pLentiCRSPRv2), transfected with Lipofectamine LTX. The virus supernatant was then harvested, filtered through a 0.45-µm filter and either frozen at −80 °C or used immediately. For transduction, RPE-Neo4p13 cells were seeded at 5 × 104 cells per well in a six-well plate the day before infection. Filtered viral supernatant was mixed with antibiotic-free medium, supplemented with 3.2 µl of 10 mg ml−1 polybrene and added to the cells for 48 h. After infection, cells were washed four times with PBS, replaced with regular medium and, if required, treated with the 5 µg ml−1 blasticidin for 5 days. Cells were then clonally sorted into 96-well plates, grown out and screened by immunoblot for positive clonal knockouts for each target. For generation of the SUV39H1Δ/H2Δ/SUZ12Δ cell line, SUZ12 gRNAs (Supplementary Table 2) were designed and cloned into pSpCas9(BB)-2A-GFP (PX458) plasmid using the standard Zhang lab gRNA insertion protocol80. Clonal SUV39H1Δ/H2Δ cells were transiently reverse cotransfected with PX458 using the Mirus TransIT-LT1 protocol at a 2:1 ratio of reagent to DNA in a six-well plate at a density of 6 × 104 cells per well. After 48 h, cells were sorted for high GFP expression into a polyclonal population and allowed to recover for 1 week. When cells were proliferating, cells were clonally sorted (GFP-negative) into 96-well plates and allowed to expand, followed by screening by immunoblot for clones in which both SUZ12 and resulting H3K27me3 were lost. For FACS and cytometry, cells were harvested by centrifuging at 500g for 5 min and resuspended in ice-cold sorting medium, which consisted of 1% FBS in PBS, supplemented with 0.25 mg ml−1 Fungizone (Thermo Fisher Scientific) and 0.25 μg ml−1 amphotericin B and 10 μg ml−1 gentamicin (Gibco). The cell suspension was filtered through 5-ml polystyrene round-bottom tubes equipped with cell-strainer caps (Falcon) before sorting and analysis using the FACSAria III or FACSAria Fusion cell sorter (BD Biosciences). Sorted cells were collected into 96-well plates containing conditional medium, prepared as a 1:1 mix of fresh complete medium and 0.45-μm-filtered medium from proliferating cell cultures, supplemented with 20% FBS, 0.25 mg ml−1 Fungizone (Thermo Fisher Scientific), 0.25 μg ml−1 amphotericin B and 10 μg ml−1 gentamicin (Gibco). For cell-cycle analysis, cells were harvested and washed with PBS. Fixation was performed by adding 1 ml of cold 70% ethanol dropwise to the cell pellet while vortexing to ensure thorough fixation and minimize clumping. Cells were fixed for at least 30 min on ice. Fixed cells were washed twice with PBS. To selectively stain DNA, cells were treated with 50 µl of RNase A, followed by the addition of 400 µl of propidium iodide solution (50 µg ml−1) per million cells directly to the RNase-A-treated pellet. The samples were mixed thoroughly and incubated at room temperature for 10 min. The stained samples were analyzed by flow cytometry and data were collected from at least 20,000 single cells per condition. Data were analyzed using FlowJo version 10.8 (BD Life Sciences). Cell pellets were resuspended in RIPA buffer, incubated for 20 min on ice and then centrifuged at 15,000g for 20 min at 4 °C. The supernatant was transferred to a fresh Eppendorf tube; protein concentration was measured with a Bradford assay and adjusted to the same concentration. Samples were resuspended to 1× sample buffer (125 mM Tris–HCl pH 6.8, 10% glycerol, 1% SDS, 0.2% (w/v) orange G and 10% β-mercaptoethanol) and boiled at 95 °C for 5 min. Then, 10 µg of protein was loaded per lane in a 4–20% SDS–PAGE gel (BioRad), followed by transfer to a nitrocellulose membrane (BioRad Transblot Turbo). Membranes were blocked in 5% (w/v) milk or BSA in TBS-T (20 mM Tris–HCl pH 7.5, 150 mM NaCl and 0.1% Tween-20) for 1 h and incubated with primary antibodies at 4 °C overnight with gentle rocking. Membranes were washed three times with TBS-T for 10 min each, followed by incubation with secondary antibody diluted in 5% (w/v) milk or BSA in TBS-T for 1 h at room temperature with gentle rocking, followed by washing three times with TBS-T for 10 min each. For fluorescent antibodies, membranes were visualized with an Odyssey Fc gel imager (LI-COR). For horseradish-peroxidase-conjugated secondaries (Extended Data Fig. 3b), enhanced chemiluminescence reagent (BioRad Clarity ECL reagent, 1705061) was used for detection and membranes were visualized on X-ray films (Cytiva, GE28-9068-37), developed on an OPTIMAX 2010 X-ray film processor (PROTAC-Med). Primary and secondary antibodies are listed in Supplementary Table 3. CENP-A, H3K9me3 and H3K27me3 CUT&RUN were performed natively with 5 × 105 cells using the CUTANA ChIC/CUT&RUN kit (EpiCypher, 14-1048) according to the manufacturer's protocol (version 3, 4 or 5). The following antibodies and concentrations were used: CENP-A (Enzo, ADI-KAM-CC006, 1:50), H3K9me3 (Abcam, ab8898, 1:50) and H3K27me3 (C36B11, 1:50) (Supplementary Table 3). DNA sequencing libraries were prepared using the NEBNext Ultra II DNA library prep kit for Illumina (New England Biolabs) following the published protocol. For multiplexing, we used NEBNext multiplex oligos for Illumina (index primer sets 1 and 2). Size selection was performed using Ampure XP beads (Beckman Coulter) to isolate mononucleosomal DNA fragments of approximately 150–180 bp (excluding adaptors). Library yield and quality were evaluated using the Qubit HS double-stranded DNA quantification assay kit (Thermo Fisher Scientific) and the TapeStation 4150 System (Agilent). Multiplexed libraries were diluted to final concentrations of 1, 2 or 4 nM and sequenced at 1.8 pM on a NextSeq 500 system (Illumina) with the NextSeq 500/550 high output version 2.5 (75 cycles) kit (Illumina). For the analysis of CUT&RUN data, raw FASTQ files were downloaded and concatenated per sample using basic Unix commands. Read quality was assessed using Basespace (Illumina) and FastQC software81. Reads were then aligned to the RPE1 genome using Bowtie2 (ref. 82), with trimming parameters adjusted on the basis of read quality. SAM-to-BAM file conversion, BAM sorting and indexing were carried out using SAMtools (version 1.16)83. Duplicate reads were removed using the MarkDuplicates command in Picard (https://broadinstitute.github.io/picard/). The resulting sorted and duplicate-removed BAM files were used to calculate read counts, normalize data (counts per million) and convert to BigWig format using the bamCoverage command in deepTools (version 2)84. BigWig files were visualized using Integrative Genomics Viewer and with pyGenomeTracks85. Read count matrices were generated using the multiBigWigSummary command in deepTools (version 2) and the output was visualized using matplotlib. MBP–nanobody–Hia5 proteins were overexpressed by rhamnose induction in BL21 DE3 cells for 16–20 h at 18 °C before harvesting by centrifugation. Cells were pelleted and resuspended in lysis buffer (50 mM HEPES pH 7.5, 300 mM NaCl, 10% glycerol and 0.5% Triton X-100) with one ETDA-free protease inhibitor tablet per liter of culture. The cells were then lysed by probe sonication (Qsonica Q125; six pulses, 30 s on and 1 min off at 200 W). After a clearance centrifugation at 40,000g for 1 h, the supernatant was added to washed Ni-NTA beads and mixed for 1 h at 4 °C. After dialysis, the protein was concentrated, filtered and applied to a Recourse Q anion-exchange column. After confirmation of protein purity by SDS–PAGE, the protein was concentrated to 1 mg ml−1, snap-frozen and stored at −80 °C until use. DiMeLo-seq was performed according to the standard protocol in Maslan et al.87 with some minor changes; all buffer compositions followed the published protocol. Briefly, 6 × 106 RPE cells were pelleted and resuspended in 1× PBS. For acute IAA experiments and CENP-C DiMeLo-seq, cells were fixed with 0.1% paraformaldehyde (PFA) by adding 6.2 µl of 16% PFA to 1 ml of suspension, followed by gentle vortexing for 2 min. Fixation was quenched with 1.25 M glycine (60 µl per 1 ml of suspension), followed by centrifugation at 500g for 3 min at 4 °C, supernatant removal and resuspension in Dig-Wash buffer (0.02% digitonin). All other samples were performed natively as per the published protocol (antibodies and concentrations in Supplementary Table 3). HMW genomic DNA was extracted using the NEB Monarch HMW DNA extraction kit for tissue, resuspending in a final volume of 750 µl of EEB buffer (ONT). This 750-µl volume was then taken forward to the tagmentation step as per the standard SQK-ULK114 protocol. CENP-A and H3K9me3 DiMeLo-seq DNA was sequenced on a PromethION 2 Solo device (ONT) with R10.4.1 flow cells with adaptive sampling for all centromeres and all of Chr4 to depths of 25 Gb (CENP-A and Mouse IgG1κ) and 20 Gb (H3K9me3, H3K27me3, CENP-C and rabbit IgG). Pod5 files were basecalled using dorado version 0.7.3 using the dorado basecaller sup@v4.3.0 for 6mA and 5mCG/5hmCG (https://github.com/nanoporetech/dorado). Basecalled BAM files were then aligned to the RPE1 genome39 using dorado. Aligned BAM files were then sorted using SAMtools (version 1.16.1), filtered with SAMtools view -F 2308 to remove unmapped alignments (0 × 4), nonprimary alignments (0 × 100) and supplementary alignments (0 × 800), following by indexing. Indexed BAM files were then processed through a refModMatch script (https://github.com/altemoselab/miscTools/blob/main/refModMatch.py) to ensure we only kept bases in the MM tag if they were a match to the reference genome. Next, 6mA and 5mCG modification filtering was performed using modkit (https://github.com/nanoporetech/modkit) using modkit pileup with the following thresholds: --filter-threshold A:0.8 --mod-threshold a:0.98 --filter-threshold C:0.8 --mod-threshold m:0.8. The 6mA thresholds were tuned for specificity and signal-to-noise ratio using endogenous CENP-A auxin depletion (Extended Data Fig. 3) where endogenous CENP-A was fully depleted. BigWig files for visualization were generated from columns 1, 2, 3 and 11 (fraction modified) of the output bedmethyl file from modkit. To visualize single molecules, we used fibertools-rs (version 0.5.4; https://github.com/fiberseq/fibertools-rs)88 and used the ft extract tool to extract --m6A and --cpg motifs into a BED file with --ml 250 and --ml 204 thresholds, respectively, for each modification (in line with modkit filtering above). BigWig files and single-molecule tracks were visualized with pyGenomeTracks85. Alpha-satellite annotations were previously identified using HumAS-HMMER For AnVIL (https://github.com/fedorrik/HumAS-HMMER_for_AnVIL)39. The resulting annotations were compared with CHM13 CenSat annotations3 to identify active, inactive and divergent HORs, as well as monomeric satellites, in the RPE1 genome. For analysis of active HORs in Figs. 4 and 5, active HOR annotations were lifted to include only the CDR/CENP-A-containing active HOR. Cases such as Cen3 (Hap1 and Hap2), Cen4 (Hap1) and Cen13 (Hap2), which contain multiple split active HORs, were contained to just the CENP-A-containing HOR. These CENP-A-containing active HORs were normalized to the same length, as described below, and plotted as active HOR ± 1 Mb, with flanking regions broadly defined as ‘pericentromere'. To standardize regions of varying lengths for heat map analysis, each region was normalized to a fixed number of bins. Regions were extended symmetrically with upstream and downstream flanking areas. For active HORs, regions were normalized to 1,000 bins, with flanking pericentromeric regions (±1 Mb) divided into 500 bins each. CDRs were scaled to 200 bins, while each flank (±125 kb) was divided into 125 bins. Mean signal intensities for each bin were extracted from BigWig files using the pyBigWig.stats() function, with missing values replaced by NaN. The normalized signal profiles for all regions were combined into matrices for visualization, where each row represents a single active HOR or CDR, sorted numerically (for example, chr1_hap1, chr1_hap2, chr2_hap1, etc. CDRs were initially identified with CDR-Finder (https://github.com/EichlerLab/CDR-Finder)63 using the following parameters: height_perc_valley_threshold, 0.39; prom_perc_valley_threshold, 0.45. After initial CDR detection, each CDR BED file was manually inspected for accuracy and compared to CENP-A DiMelo-seq data. Any non-CENP-A-containing CDRs were manually removed. To ensure comparison of only primary CDRs, all nonprimary and newly formed CDRs were removed. The total length of each CDR was then summed and directly compared on a per-centromere basis to the corresponding parent centromere. To detect newly formed CDRs, the CDR-Finder63 output was manually inspected as above for accuracy, without removing nonprimary CDRs. All CDRs within 500 kb were merged and considered as one single centromere locus. Any CENP-A-containing CDRs not present in the parent CDR database were considered newly formed (that is, any new centromere peak more than 500 kb from the primary CDR site). The following chromosomes were not considered in all CDR analyses: Cen4 Hap2 (neocentromere-containing haplotype), CENX Hap2 (not present in genome), Cen18 Hap1 and Cen18 Hap2 (poorly defined CDRs). CENP-A and CENP-C peaks were identified using a seeded expansion approach from preannotated CDRs (above). For each BigWig file containing 6mA/A-modified fraction data, the mean signal within each CDR was calculated and a threshold was set at 1% of this mean. A genomic window extending 30 kb beyond the CDR boundaries was loaded, with the signal smoothed using a 2-kb rolling average to mitigate noise. Resulting peak annotations for each centromere were sorted and output as BED files. To quantify 6mA density in detected CENP-A peaks, regions enriched for CENP-A relative to IgG controls, DiMeLo-seq reads in BED12 format from the fibertools extract were processed for each sample (parent, SUV39H1/H2Δ, SUV39H1/H2Δ/SUZ12Δ and SETDB1Δ). The BED12 file output from the fibertools extract provided the read's chromosome, start and end positions (columns 1–3), read name (column 4), block count (column 10, representing the number of 6mA sites), block sizes (column 11, typically 1 for point modifications) and relative block starts (column 12, indicating 6mA positions relative to the read start). Reads overlapping sample-specific CDR intervals by at least 10 kb (after excluding chromosomes chr4_2, chr18, chrX_2, chr13_2 and chr21_2) were identified and 6mA counts within these overlaps were tallied per molecule by checking whether each block's absolute position fell within the CDR boundaries. Density was computed as the number of 6mA sites per 10 kb of overlapping CDR sequence. Enrichment was calculated by dividing CENP-A densities by corresponding IgG control densities and plotted with matplotlib and seaborn. Cells were treated with KaryoMAX Colcemid (100 ng ml−1, Gibco) for 3 h and mitotic cells were collected by mitotic shakeoff. To prepare cells for spreading, a 75 mM KCl solution was prewarmed in a 37 °C water bath and a fresh Carnoy fixative solution (methanol and acetic acid, 3:1) was prepared and placed on ice. The cell suspension was centrifuged at 500g for 5 min and the supernatant was discarded, leaving a small residual volume in which the pellet was gently resuspended. For hypotonic treatment, 1 ml of 75 mM KCl (prewarmed to 37 °C) was slowly added to the cell suspension, followed by an additional 4 ml of the same solution. The tubes were incubated in a 37 °C water bath for 15 min, with gentle shaking once or twice during incubation to homogenize the solution. While the tube was vortexed at the lowest speed, 1,000 μl of ice-cold fixative was added dropwise, followed by an additional 10 ml of fixative. The suspension was centrifuged at 230g for 10 min at 4 °C and the cell pellet was resuspended in 10 ml of fresh fixative. The prepared cells were stored at −20 °C until ready for chromosomal spreading. For dropping onto slides, Epredia Superfrost Plus slides were cleaned and kept in a coplin jar with methanol and kept at 4 °C until required. When ready for dropping, a cleaned slide was removed and placed tilted on an Eppendorf tube rack. Then, 100 μl of fixed cells were dropped from a ~30-cm height onto the slide and allowed to flow down the glass. Excess fixative was removed on a paper towel and the slide was allowed to dry fully for ~5 min. Immunofluorescence on cells fixed in methanol and acetic acid was performed as described in Beh et al.89, with some modifications. The primary antibody used to identify the dicentric chromosome was a guinea pig anti-CENP-C (1:1,000, MBL, PD030) and the secondary antibody was used at 1:500 (anti-guinea pig Alexa Fluor 488, Jackson ImmunoResearch Laboratories). The Metafer imaging platform (MetaSystems, Metafer4 version 3.13.5) and Isis software (version 5.8.14) were used for automated acquisition of the chromosome spreads. For mFISH following immunocytochemistry, slides were washed in PBS for 3 min as described in Beh et al.89 and mFISH was performed with a 24 XCyte probe (MetaSystems) following the manufacturer's instructions. The Metafer imaging platform (MetaSystems, Metafer4 version 3.13.5) and Isis software (version 5.8.14) were used for the second acquisition of chromosome spreads and mFISH image analysis. Requests for cell lines, plasmids or other resources generated in this study should be directed to L.E.T.J. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All sequencing data were deposited to the European Nucleotide Archive under accession number PRJEB85119. Processed data files and annotations are available through BioStudies under accession number S-BSST2137. This version supersedes a previous submission (S-BSST1935) associated with the bioRxiv version of this paper. Source data are provided with this paper. The scripts for the bioinformatic analyses with their parameters were deposited to a public GitHub repository (https://github.com/bencarty2/Cartyetal_heterochromatin-boundaries). McKinley, K. L. & Cheeseman, I. M. The molecular basis for centromere identity and function. Complete genomic and epigenetic maps of human centromeres. Willard, H. F. Centromeres of mammalian chromosomes. Mendiburo, M. J., Padeken, J., Fülöp, S., Schepers, A. & Heun, P. Drosophila CENH3 is sufficient for centromere formation. Barnhart, M. C. et al. HJURP is a CENP-A chromatin assembly factor sufficient to form a functional de novo kinetochore. Amor, D. J. et al. Human centromere repositioning ‘in progress'. Voullaire, L. E., Slater, H. R., Petrovic, V. & Choo, K. H. A. A functional marker centromere with no detectable alpha-satellite, satellite III, or CENP-B protein: activation of a latent centromere?. Voullaire, L. et al. Mosaic inv dup(8p) marker chromosome with stable neocentromere suggests neocentromerization is a post-zygotic event. Characterization of neo-centromeres in marker chromosomes lacking detectable alpha-satellite DNA. Marshall, O. J., Chueh, A. C., Wong, L. H. & Choo, K. H. A. Neocentromeres: new insights into centromere structure, disease development, and karyotype evolution. & Choo, K. H. A. Neocentromeres: role in human disease, evolution, and centromere study. Ishii, K. et al. Heterochromatin integrity affects chromosome reorganization after centromere dysfunction. Schotanus, K. & Heitman, J. Centromere deletion in Cryptococcus deuterogattii leads to neocentromere formation and chromosome fusions. Shang, W. H. et al. Chromosome engineering allows the efficient isolation of vertebrate neocentromeres. Ketel, C. et al. Neocentromeres form efficiently at multiple possible loci in Candida albicans. Induction of spontaneous human neocentromere formation and long-term maturation. Jansen, L. E. T., Black, B. E., Foltz, D. R. & Cleveland, D. W. Propagation of centromeric chromatin requires exit from mitosis. Schuh, M., Lehner, C. F. & Heidmann, S. Incorporation of Drosophila CID/CENP-A and CENP-C into centromeres during early embryonic anaphase. Bodor, D. L., Valente, L. P., Mata, J. F., Black, B. E. & Jansen, L. E. T. Assembly in G1 phase and long-term stability are unique intrinsic features of CENP-A nucleosomes. Patterns and mechanisms of ancestral histone protein inheritance in budding yeast. Hori, T. et al. Constitutive centromere-associated network controls centromere drift in vertebrate cells. Birth, evolution, and transmission of satellite-free mammalian centromeric domains. Hoffmann, S. et al. A genetic memory initiates the epigenetic loop necessary to preserve centromere position. Alonso, A., Hasson, D., Cheung, F. & Warburton, P. E. A paucity of heterochromatin at functional human neocentromeres. Gershman, A. et al. Epigenetic patterns in a complete human genome. Peters, A. H. F. M. et al. Partitioning and plasticity of repressive histone methylation states in mammalian chromatin. Rice, J. C. et al. Histone methyltransferases direct different degrees of methylation to define distinct chromatin domains. Sidhwani, P. & Straight, A. F. Epigenetic inheritance and boundary maintenance at human centromeres. van de Werken, C. et al. Paternal heterochromatin formation in human embryos is H3K9/HP1 directed and primed by sperm-derived histone modifications. Decombe, S. et al. Epigenetic rewriting at centromeric DNA repeats leads to increased chromatin accessibility and chromosomal instability. Okada, T. et al. CENP-B controls centromere formation depending on the chromatin context. & Karpen, G. H. Centromeric chromatin exhibits a histone modification pattern that is distinct from both euchromatin and heterochromatin. Dudka, D. et al. Satellite DNA shapes dictate pericentromere packaging in female meiosis. Altemose, N. et al. DiMeLo-seq: a long-read, single-molecule method for mapping protein–DNA interactions genome wide. Purgato, S. et al. Centromere sliding on a mammalian chromosome. The quantitative architecture of centromeric chromatin. The structure, function and evolution of a complete human chromosome 8. & O'Neill, R. J. Transposable elements: genome innovation, chromosome diversity, and centromere conflict. Alonso, A. et al. Co-localization of CENP-C and CENP-H to discontinuous domains of CENP-A chromatin at human neocentromeres. Mahlke, M. A. et al. Evolution and instability of human centromeres are accelerated by heterochromatin boundary loss and CENP-A overexpression. Lehnertz, B. et al. Suv39h-mediated histone H3 lysine 9 methylation directs DNA methylation to major satellite repeats at pericentric heterochromatin. Complete loss of H3K9 methylation dissolves mouse heterochromatin organization. Matsui, T. et al. Proviral silencing in embryonic stem cells requires the histone methyltransferase ESET. Fukuda, K. & Shinkai, Y. SETDB1-mediated silencing of retroelements. Liu, S. et al. Setdb1 is required for germline development and silencing of H3K9me3-marked endogenous retroviruses in primordial germ cells. Martins, N. M. C. et al. Epigenetic engineering shows that a human centromere resists silencing mediated by H3K27me3/K9me3. Pasini, D., Bracken, A. P., Jensen, M. R., Denchi, E. L. & Helin, K. Suz12 is essential for mouse development and for EZH2 histone methyltransferase activity. Hoffmann, S. et al. CENP-A is dispensable for mitotic centromere function after initial centromere/kinetochore assembly. Lachner, M., O'Carroll, D., Rea, S., Mechtler, K. & Jenuwein, T. Methylation of histone H3 lysine 9 creates a binding site for HP1 proteins. Nakayama, J., Rice, J. C., Strahl, B. D., Allis, C. D. & Grewal, S. I. S. Role of histone H3 lysine 9 methylation in epigenetic control of heterochromatin assembly. Rea, S. et al. Regulation of chromatin structure by site-specific histone H3 methyltransferases. Dubocanin, D. et al. Conservation of dichromatin organization along regional centromeres. Blower, M. D. & Karpen, G. H. The role of Drosophila CID in kinetochore formation, cell-cycle progression and heterochromatin interactions. Martins, N. M. C. et al. H3K9me3 maintenance on a human artificial chromosome is required for segregation but not centromere epigenetic memory. Salinas-Luypaert, C. et al. DNA methylation influences human centromere positioning and function. A., Trazzi, S., Della Valle, G. & Robertson, K. D. DNMT3B interacts with constitutive centromere protein CENP-C to modulate DNA methylation and the histone code at centromeric regions. Identification and annotation of centromeric hypomethylated regions with CDR-Finder. Craig, J. M., Wong, L. H., Lo, A. W. I., Earle, E. & Choo, K. H. A. Centromeric chromatin pliability and memory at a human neocentromere. & Straight, A. F. Histone H3 lysine methyltransferase activities control compartmentalization of human centromeres. Targeting pericentric non-consecutive motifs for heterochromatin initiation. De La Cruz, C. C. et al. The Polycomb group protein SUZ12 regulates histone H3 lysine 9 methylation and HP1α distribution. Vermeulen, M. et al. Quantitative interaction proteomics and genome-wide profiling of epigenetic histone marks and their readers. Eberl, H. C., Spruijt, C. G., Kelstrup, C. D., Vermeulen, M. & Mann, M. A map of general and specialized chromatin readers in mouse tissues generated by label-free interaction proteomics. Boros, J., Arnoult, N., Stroobant, V., Collet, J.-F. & Decottignies, A. Polycomb repressive complex 2 and H3K27me3 cooperate with H3K9 methylation to maintain heterochromatin protein 1α at chromatin. Nechemia-Arbely, Y. et al. DNA replication acts as an error correction mechanism to maintain centromere identity by restricting CENP-A to centromeres. Falk, S. J. et al. CENP-C reshapes and stabilizes CENP-A nucleosomes at the centromere. Perea-Resa, C. & Blower, M. D. Centromere biology: transcription goes on stage. Naughton, C. et al. Human centromere repositioning activates transcription and opens chromatin fibre structure. Stimpson, K. M. et al. Telomere disruption results in non-random formation of de novo dicentric chromosomes involving acrocentric human chromosomes. The centromere geometry essential for keeping mitosis error free is controlled by spindle forces. Sacristan, C. et al. Vertebrate centromeres in mitosis are functionally bipartite structures stabilized by cohesin. Rowley, G. & Jansen, L. E. T. A brief historical perspective on cell cycle control of CENP-A assembly and inheritance. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Ran, F. A. et al. Genome engineering using the CRISPR–Cas9 system. Andrews, S. FastQC: a quality control tool for high throughput sequence data. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. The Sequence Alignment/Map format and SAMtools. Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Lopez-Delisle, L. et al. pyGenomeTracks: reproducible plots for multivariate genomic datasets. Gamarra, N. et al. DiMeLo-cito: a one-tube protocol for mapping protein–DNA interactions reveals CTCF bookmarking in mitosis. Maslan, A. et al. Mapping protein–DNA interactions with DiMeLo-seq. Jha, A. et al. DNA-m6A calling and integrated long-read epigenetic and genetic analysis with fibertools. Beh, T. T., MacKinnon, R. N. & Kalitsis, P. Active centromere and chromosome identification in fixed cell lines. We thank members of the L.E.T.J. lab for discussion and critical analysis of this paper, L. Fulcher (Oxford) for technical advice regarding lentiviral targeting, A. lab for advice on Hia5 purification, K. Miga and B. McNulty (University of California, Santa Cruz) for advice on HMW DNA extraction and D. Cleveland (University of California, San Diego) for the CENP-C antibody. We further thank R. Hedley and V. Tsioligka for providing technical assistance in cell-cycle analysis and single-cell sorting at The Don Mason Facility of Flow Cytometry, Sir William Dunn School of Pathology, University of Oxford. We also acknowledge the Cell and Tissue Imaging Platform (PICT-IBiSA, a member of the French national research infrastructure France BioImaging; ANR10-INBS-04). This work was funded by a UK Research Institute Marie Sklodowska-Curie Actions European postdoctoral fellowship (101066059), a Wellcome early career award (303812/Z/23/Z) and John Fell award (0013250) (Oxford University Press) to B.L.C. receives salary support from the CNRS. is supported by Institut Curie. S.G. receives salary support from the Rita Levi Montalcini Award. is supported by a National Recovery and Resilience Plan PhD fellowship from the Italian Ministry of University and Research. Work in the S.G. lab is supported by an AIRC startup grant (202-25189), Sapienza Excellence grant and European Research Council CENTROFUN grant (101078838) to S.G. Work in the N.A. lab was supported by a Howard Hughes Medical Institute Hanna H. Gray Faculty Fellowship and by the Chan Zuckerberg Biohub. and his lab are generously funded by a Wellcome Senior Research Fellowship 210645/Z/18/Z and a Wellcome Discovery Award 304643/Z/23/Z. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the paper. Present address: Infectious Disease and Microbiology Unit. Present address: Nencki Institute of Experimental Biology, Polish Academy of Sciences, Warsaw, Poland Ben L. Carty, Marina Murillo-Pineda, Pawel Mikulski, Julia Humes, Oliver Whittingham & Lars E. T. Jansen Department of Genetics, Stanford University, Palo Alto, CA, USA UMR144 and UMR3664, CNRS, Institut Curie, PSL Research University, Sorbonne Université, Paris, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar is an inventor on a patent application related to the DiMeLo-seq technology used in this work. The other authors declare no competing interests. Nature Structural & Molecular Biology thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Dimitris Typas and Melina Cassadio, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, c, e, g, Retrotransposon density and H3K9me3 CUT&RUN enrichment in RPE1 cells across loci that either feature a neocentromere (Neo4p13) or at loci where CENP-A has been detected in patient-derived neocentromere cells (PDNC4, IMSq13 and MS4221). b, d, f, h, Violin plot of total LTR, LINE and SINE density across each neocentromere, highlighting bins directly overlapping with or immediately adjacent to each neocentromere. The bin size for each violin plot is set to the overall CENP-A domain size for the corresponding neocentromere. a, Schematic of 3 hr auxin depletion using hTERT-RPE1 eYFP-AID-CENP-A+/+ cells52. b, Immunoblot for CENP-A and Histone H4 on nuclear extracts +/- 3hr IAA treatment (Representative of n = 2 independent experiments). e, View of the entire HOR of CEN6 (Hap1) +/- IAA conditions with CENP-A 6mA/A and 5mCG/C averaged genomic tracks, and single molecule alignments with CENP-A-directed 6mA modifications. f, Zoom into the CDR of CEN6 (Hap1) +/- IAA conditions with CENP-A 6mA/A and 5mCG/C averaged genomic tracks, and single molecule alignments with CENP-A-directed 6mA modifications. a, Visualisation of a 40 kb region on human satellite 3 (HSat3) of chromosome 9, showing H3K9me3 averaged m6A/A track (green) and single molecule alignments (black, bottom). Black vertical lines represent H3K9me3-directed 6mA modification. b, Zoom in to a 5 kb region of HSat3 showing H3K9me3 averaged m6A/A track (green) and single molecule alignments (black, bottom). 6mA modifications appear as regularly spaced nucleosome-like intervals. On this single molecule, we can estimate nucleosome positioning of H3K9me3-containing nucleosomes to be approximately 17 nucleosomes within this 3kb H3K9me3-dense region (176 bp/nucleosome, which is consistent with previous estimates of 183 bp/nucleosome for HSat3 heterochromatic regions based on Fiber-Seq measurements57. Black vertical lines represent 6mA modification. b, Centromere X (Haplotype 1) visualised with CENP-A (red, 6mA/A) and H3K9me3 (green, 6mA/A). c, 10kb zoom in to regions inside (left) versus outside (right) of the CENX CDR for single DNA molecules, showing the specificity for CENP-A and H3K9me3 detection at our chosen m6A probability filtering. d, Centromere X (Haplotype 1) visualised with CENP-C (orange, 6mA/A) and H3K9me3 (green, 6mA/A). e, 10kb zoom in to regions inside (left) versus outside (right) of the CENX CDR for single DNA molecules, showing the specificity for CENP-C detection at our chosen m6A probability filtering. a, Visualisation of the broader Neo4p13 region (40.75 Mb to 45.0 Mb) showing CENP-A (red) and H3K27me3 averaged m6A/A track (grey) for each of parental (RPE-Neo4p13) and SUV39H1Δ/H2Δ, SUV39H1Δ/H2Δ/SUZ12Δ and SETDB1Δ. b, Heatmaps for Parental, SUV39H1Δ/H2Δ, SUV39H1Δ/H2Δ/SUZ12Δ and SETDBΔ cell lines, illustrating the distribution of H3K27me3-directed 6mA (left) and Rabbit IgG-directed 6mA (right) modifications across centromeric Active HORs. Each line on the heatmap spans the Active HORs of a single centromere and flanking regions ±1 Mb upstream and downstream. All Active HOR regions have been scaled to the same length to allow direct comparison of methylation patterns across all centromeres. c, Heatmaps for H3K27me3 and Rabbit IgG as in (b), but specifically across Centromere Dip Regions (CDRs), as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region. All CDRs have been scaled to the same length. d, Average 6mA 6mA modification signal for H3K27me3 (left) and Rabbit IgG (right) across all Active HOR regions (scaled equally), flanked by ±1 Mb. e, Average indicated 6mA modification signal of H3K27me3 (left) and Rabbit IgG (right) across all CDRs regions flanked by ±250 kb. 6, for Parental (RPE-Neo4p13), SUV39H1Δ/H2Δ, SUV39H1Δ/H2Δ/SUZ12Δ, SETDB1Δ, DNMT3BΔ and SETDB1Δ/DNMT3BΔ cell lines with aggregate DiMelo-Seq enrichments for Mouse IgG1 (red) and 5mCG enrichment (blue). Single molecules alignments for CENP-A-directed 6mA (black). b, Heatmaps for each mutant illustrating the distribution of Mouse IgG1 (red) across Centromere Dip Regions (CDRs), as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region in parental RPE-Neo4p13. Each line on the heatmap represents a single CDR/Centromere. All CDRs have been sorted numerically and scaled to the same length for comparison. a, Extended immunoblots as shown in Fig. 3b, now including DNMT3BΔ and SETDB1Δ/DNMT3BΔ characterisation (Representative of n≥2 independent replicates). b, Average 5mCG/C modification signal across all Active HOR regions (scaled equally), flanked by ±1000 Kb. c, Overview of the entire HOR of CEN 16 (Hap 1) for Parental (RPE-Neo4p13), SETDB1Δ, DNMT3BΔ and SETDB1Δ/DNMT3BΔ cell lines with aggregate DiMelo-Seq enrichments for CENP-A (red) and 5mCG enrichment (blue). Single-molecule alignments for CENP-A-directed 6mA (black). d, Heatmaps for each mutant illustrating the distribution of CENP-A (red) across Centromere Dip Regions (CDRs), as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region in parental cells. Each line on the heatmap represents a single CDR/Centromere. All CDRs have been sorted numerically and scaled to the same length for comparison. e, Centromere length as defined by the difference in sum total length of CDR/sub-CDRs relative to the corresponding parent centromere. f, CENP-A Enrichment: Box and strip plot depicting the ratio of CENP-A enrichment within CDRs relative to their flanking regions for parent and mutant lines. A dashed reference line at 1 indicates equal enrichment within and outside the HORs. For boxplots in e and f above: boxplot centre=median; box bounds=25th/75th percentiles; whiskers=min/max within 1.5×IQR from box edges. g, 5mCG boundary definition: plot of all 5mCG CDR boundary profiles centred on CDR boundaries (±100 Kb). Mean of combined aggregated traces is shown for all samples. h, CENP-A boundary definition: Plot of CENP-A enrichment inside versus outside the CDR boundary (defined by Parental 5mCG) for each cell line (±100 Kb). Mean of combined CENP-A (6mA/A) aggregated trace is shown for all samples. a, View of the entire HOR of CEN 20 (Hap 2) for Parental (RPE-Neo4p13), Parental (Day 100 Cone 1.2), SUV39H1Δ/H2Δ/SUZ12Δ (early Passage, EP) and SUV39H1Δ/H2Δ/SUZ12Δ (Day 100 Clone 1.2) with aggregate DiMelo-Seq enrichments for CENP-A (red) and 5mCG enrichment (blue). Single molecules alignments for CENP-A-directed 6mA (black). b, Heatmaps for each mutant illustrating the distribution of CENP-A (red) across Centromere Dip Regions (CDRs), as defined by the boundary of 5mCG dips on either side of each CENP-A-containing region in parental RPE-Neo4p13. Each line on the heatmap represents a single CDR/Centromere. All CDRs have been sorted numerically and scaled to the same length for comparison. c, Centromere length as defined by the difference in sum total length of CDR/sub-CDRs relative to the corresponding parent centromere. A dashed reference line at 1 indicates early passage parental centromere length. d, CENP-A Enrichment: Box plot depicting the ratio of CENP-A enrichment within CDRs relative to their flanking regions (±100 Kb) for parent and mutant lines. For boxplots in c and d above: boxplot centre=median; box bounds=25th/75th percentiles; whiskers=min/max within 1.5×IQR from box edges. e, 5mCG boundary definition: plot of all 5mCG CDR boundary profiles centred on CDR boundaries (±100 Kb). Mean of combined aggregated traces is shown for all samples. f, CENP-A boundary definition: Plot of CENP-A enrichment inside versus outside the CDR boundary (defined by Parental 5mCG) for each cell line (±100 Kb). Mean of combined CENP-A (6mA/A) aggregated trace is shown for all samples. a, Measure of cell cycle distribution and ploidy by Propidium Iodide DNA-staining and flow cytometry for each cell line used in this study. b, Representative mFISH karyotypes for each cell line used in this study, alongside heatmap (below) indicating the frequency of translocations and chromosome breakages for each cell line based on 20 mFISH images per cell line. c, Table listing most frequent translocations identified in each cell line. d, Table listing most frequent chromosome breakages identified in each cell line. Tables with coordinates of all new CENP-A-containing CDRs, gRNA primers and antibodies used. MultiBigWig summary output and data for bar plots. MultiBigWig summary output and data for bar plots. Nanopore sequencing source data, centromere length source data and chromosome enrichment summaries. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Carty, B.L., Dubocanin, D., Murillo-Pineda, M. et al. Heterochromatin boundaries maintain centromere position, size and number. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251124094325.htm'>Cocoa and tea may protect your heart from the hidden damage of sitting</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-25 06:53:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New findings from the University of Birmingham suggest that regularly eating foods rich in flavanols, including tea, berries, apples, and cocoa, may help protect men's blood vessels from the negative effects that occur during long periods of sitting. Young adults spend an estimated six hours a day seated, and extended sitting is known to reduce how well blood vessels function. Earlier research has shown that even a small 1% drop in vascular function, measured through brachial Flow-mediated dilatation (FMD), corresponds to a 13% rise in the risk of cardiovascular conditions such as heart disease, strokes, and heart attacks. Flavanols are naturally occurring polyphenols found in various fruits, tea, nuts, and cocoa beans. They have previously been linked to cardiovascular benefits, including support for the vascular system during psychological stress. Dr. Catarina Rendeiro, Assistant Professor in Nutritional Sciences at the University of Birmingham and lead author, explained: "Whether we are sitting at desks, behind the wheel of a car, on a train, or on the sofa reading a book or watching TV, we all spend a lot of time seated. Finding ways to mitigate the impact that sitting for uninterrupted periods has on our vascular system could help us cut the risk of developing cardiovascular diseases." Recent analyses also estimate that cardiovascular diseases cost the UK approximately £29 billion. Each participant consumed either a high-flavanol cocoa drink (695 mg of total flavanols per beverage) or a low-flavanol cocoa drink (5.6 mg of total flavanols per beverage) before beginning the sitting session. Women were not included in the study because fluctuations in estrogen during the menstrual cycle may influence how flavanols affect vascular function. The researchers note that this should be explored in future trials. These participants also experienced increases in diastolic blood pressure, reductions in shear rate and blood flow, and lower leg muscle oxygenation. This is the first study to demonstrate that flavanols can prevent sitting-induced vascular dysfunction in healthy young men. Dr. Sam Lucas, Professor of Cerebrovascular, Exercise & Environmental Physiology at the University of Birmingham and co-author, said: "Our experiment indicates that higher fitness levels do not prevent the temporary impairment of vascular function induced by sitting when only drinking low-flavanol cocoa. Importantly, after the high-flavanol drink, both fitter and less-fit participants kept their FMD the same as it was before sitting for two hours." This suggests that flavanols may help support vascular health regardless of someone's fitness level. Alessio Daniele, PhD student at the University of Birmingham, noted: "It is actually quite easy to add high flavanol foods to your diet. Dr. Catarina Rendeiro added: "Our research shows that consuming high-flavanol foods and drinks during periods spent sitting down is a good way to reduce some of the impact of inactivity on the vascular system. "Given how common sedentary lifestyles have become and the increased risk this can have to vascular health, using flavanol-rich food and drink, especially in combination with breaking up periods of inactivity by going for a short walk or standing up, could be a good way to enhance long-term health, no matter the individual's fitness level." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            