
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-12</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09833-y'>Olympiad-level formal mathematical reasoning with reinforcement learning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 16:32:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Please note there may be errors present which affect the content, and all legal disclaimers apply. A long-standing goal of artificial intelligence is to build systems capable of complex reasoning in vast domains, a task epitomized by mathematics with its boundless concepts and demand for rigorous proof. Recent AI systems, often reliant on human data, typically lack the formal verification necessary to guarantee correctness. We present AlphaProof, an AlphaZero-inspired2 agent that learns to find formal proofs through RL by training on millions of auto-formalized problems. For the most difficult problems, it uses Test-Time RL, a method of generating and learning from millions of related problem variants at inference time to enable deep, problem-specific adaptation. Combined with AlphaGeometry 23, this performance, achieved with multi-day computation, resulted in reaching a score equivalent to that of a silver medallist, marking the first time an AI system achieved any medal-level performance. Our work demonstrates that learning at scale from grounded experience produces agents with complex mathematical reasoning strategies, paving the way for a reliable AI tool in complex mathematical problem-solving. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Thomas Hubert, Rishi Mehta, Laurent Sartran, Miklós Z. Horváth, Goran Žužić, Eric Wieser, Aja Huang, Julian Schrittwieser, Yannick Schroecker, Hussain Masoom, Ottavia Bertolli, Tom Zahavy, Amol Mandhane, Jessica Yung, Iuliya Beloshapka, Borja Ibarz, Vivek Veeriah, Lei Yu, Oliver Nash, Paul Lezeau, Salvatore Mercuri, Calle Sönne, Bhavik Mehta, Alex Davies, Daniel Zheng, Fabian Pedregosa, Yin Li, Ingrid von Glehn, Mark Rowland, Samuel Albanie, Ameya Velingker, Simon Schmitt, Edward Lockhart, Edward Hughes, Henryk Michalewski, Nicolas Sonnerat, Demis Hassabis, Pushmeet Kohli & David Silver This PDF contains numerical solve rates supplementing Extended Figure 4, Hyper-parameter values, details on problems used to benchmark autoformalization, and sample proofs from the AlphaProof agent. Pseudocode written in Python elaborating the high level structure of AlphaProof: RL, autoformalization, and variant generation. Hubert, T., Mehta, R., Sartran, L. et al. Olympiad-level formal mathematical reasoning with reinforcement learning. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09722-4'>Convergent genome evolution shaped the emergence of terrestrial animals</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 16:06:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The challenges associated with the transition of life from water to land are profound1, yet they have been met in many distinct animal lineages2,3,4,5. These constitute a series of independent evolutionary experiments from which we can decipher the role of contingency versus convergence in the adaptation of animal genomes. Here we compare 154 genomes from 21 animal phyla and their outgroups to reconstruct the protein-coding content of the ancestral genomes linked to 11 animal terrestrialization events, and to produce a timescale of terrestrialization. We uncover distinct patterns of gene gain and loss underlying each transition to land, but similar biological functions emerged recurrently pointing to specific adaptations as key to life on land. We show that semi-terrestrial species evolved convergent functional patterns, in contrast with fully terrestrial lineages that followed different paths to land. Our timeline supports three temporal windows of land colonization by animals during the last 487 million years, each associated with specific ecological contexts. Although each lineage exhibits distinct adaptations, there is strong evidence of convergent genome evolution across the animal kingdom suggesting that, in large part, adaptation to life on land is predictable, linking genes to ecosystems. The colonization of land by life is one of the most notable transitions in the history of Earth, substantially shaping modern ecosystems, life forms and the planet itself1. Terrestrialization has occurred multiple times independently within the animal kingdom, including among arthropods5, vertebrates4, rotifers2, molluscs6, annelids2, nematodes2, tardigrades7 and onychophorans3. Nevertheless, each of these natural experiments in terrestrialization had to overcome similar physiological and environmental challenges2. Phenotypic adaptations—such as water-retentive skin or cuticle2, adapted immune systems8, changes in skeletal design and locomotion2, elevated metabolic rates9, developmental adaptations (such as encapsulated larvae and brooding) and adaptation of vision in aerial environments10—show widespread convergence across terrestrial lineages, suggesting largely predictable responses to similar environmental pressures. At the genotype level, recent studies have shown that genomic changes, including gene innovation11, duplication12 and loss13, were crucial to major metazoan evolutionary transitions. Furthermore, specific genes (for example, aquaporin-coding genes) have been linked to terrestrialization in several clades14, and genomic changes have been associated with individual lineages15,16,17,18; these support the roles of genes related to metabolism, stress response, osmoregulation and immunity in terrestrialization. However, in comparison to land plants19, the genomic basis of terrestrialization across animal lineages remains largely uncharacterized. These parallel natural experiments provide a unique opportunity to determine whether terrestrialization has led to lineage-specific contingent genomic adaptations or whether these are predictable changes in response to the same environmental challenges. Here we apply a comparative genomics pipeline to a dataset of 154 genomes to explore the role of convergence and contingency in the evolutionary response of animal genomes to the process of terrestrialization, and to establish the timeline of acclimatization of animals to land. Our results reveal that independent terrestrial events were driven by the emergence of similar biological functions, although semi-terrestrial and fully terrestrial lineages exhibit different patterns of genomic adaptation to terrestrialization. We identify three temporal windows in which animals colonized land, offering a new chronological framework for these transitions. We designed an approach that we named intersection framework for convergent evolution (InterEvo), which identifies the intersection of biological functions between different sets of genes that were independently gained or reduced in different nodes along the phylogeny (Extended Data Fig. In brief, we mined 154 genomes—151 from 21 animal phyla and 3 from non-animal holozoans (Fig. 2 and Supplementary Table 1)—and filtered them by completeness. These represent the diversity of animals and our sampling focuses on species flanking nodes that represent terrestrialization events. In this dataset, we identified 11 such events20 (Fig. For Onychophora, Tardigrada and Malacostraca (Armadillidium), only one or two genomes were available at the time of starting these analyses. The phylogeny is based on 154 sampled taxa, with taxon sampling numbers shown after clade names in parentheses. Terrestrial events are highlighted in green text. HG content of terrestrial nodes is displayed in each corresponding box (from left to right, top to bottom: novel HGs, novel core HGs, expanded HGs, contracted HGs and lost HGs). Branch lengths are scaled to evolutionary time, with divergence times and 95% highest posterior density intervals indicated for terrestrial nodes. Organism silhouettes (sourced from phylopic.org and the authors, released under public domain dedication (CC0 1.0 Universal)) represent the 11 terrestrial events. Geological time abbreviations are as follows: Neop., Neoproterozoic (part); Camb., Cambrian; Ord., Ordovician; Sil., Silurian; Dev., Devonian; Carb., Carboniferous; Perm., Permian; Tri., Triassic; Palg., Paleogene; Ng., Neogene; Q., Quaternary. The 3,934,362 protein sequences from these genomes were clustered into 483,458 homology groups (HGs), groups of proteins that have distinctly diverged from other groups, comprising orthologues and/or paralogues21. Using a previously described approach11,13,19, we reconstructed the HG content for the key nodes in the tree and classified the HGs based on their mode of evolution: gene gains (novel, novel core and expanded) and gene reductions (contracted and lost). Novel HGs are those that are present in the ingroup and absent in all the outgroups, with core HGs being present in all the species of the ingroup (permitting one absence). Lost HGs are those that are absent in the ingroup but present in the sister groups and other species in outgroup. Finally, we used CAFE522 to infer expanded or contracted HGs, which are those presenting an increase or reduction, respectively, in the number of gene copies (more detailed definitions are provided in Methods and Supplementary Table 2). Terrestrialization nodes are characterized by a large turnover of gene gains and reductions (Fig. All terrestrial lineages display large gene gains (novel genes and expansions) compared to their immediate ancestors, except arachnids, hexapods and the novel genes in myriapods. Novelty is much higher in bdelloid rotifers, nematodes, tetrapods and land gastropods, the latter only in gene expansions, a finding supported by a previous study15. Similarly, terrestrial gene reduction (losses and contractions) is pervasive except in arachnids and hexapods; there is no increase in gene loss in land gastropods, tetrapods and bdelloid rotifers, the latter probably due to the massive losses in the last common ancestor (LCA) of the sampled rotiferans. Nematoda, Tardigrada and Onychophora show the largest gene losses, in agreement with previous studies12,13, together with Rotifera. Although sparse taxon sampling and the fast-evolving nature of some lineages can inflate gene turnover estimates13,23, the pattern persists after normalization by divergence time, measured as the accumulation of novel and novel core HGs per million years (Supplementary Table 3). Expansion and contraction need no correction as the birth–death model is intrinsically scaled by branch length. For the significance of gene turnover, a permutation test confirmed that the observed novel gene rates found in terrestrial lineages are significantly higher than in aquatic nodes (P = 0.0015; Extended Data Fig. 3a and Supplementary Information section 1.1.1). In summary, most terrestrialization events seem to display high levels of gene turnover, reflecting genome plasticity24 during the animal transition from water to land associated with new environmental challenges. Arachnids and hexapods show lower levels of plasticity, which may indicate that their evolution was dominated by gene co-option instead. A total of 154 genomes were analysed to infer HGs and reconstruct ancestral states. Each bar chart represents one terrestrial event node and its three immediate ancestors. For each node, five categories of HGs are quantified: novel, novel core, expanded, contracted and lost. The y axis indicates the number of HGs in each category of each clade. Organism silhouettes (sourced from phylopic.org and the authors, released under public domain dedication (CC0 1.0 Universal)) represent 11 terrestrial events. To infer functional convergence across the 11 terrestrial events, we annotated the functions of their novel and novel core HGs using both gene ontology (GO)25,26 and Pfam protein domains27. The number of shared GO terms among at least ten nodes are shown in Fig. 3a and shared Pfams among at least five nodes are shown in Fig. For novel HGs, there are 118 GO terms shared by different combinations of at least 10 nodes (green bars in Fig. 1), and 26 for novel core HGs (blue bars in Fig. Our analyses show that novel gene families that emerged independently in different terrestrialization events are involved in osmosis (regulation of water transport in cells), metabolism (namely fatty acids, probably related to changes in diet), reproduction, detoxification, sensory reception and reaction to stimuli. All gene families reported here stem from the genome-wide comparative analysis of every HG, and none was chosen a priori. a, Distribution of shared GO terms of gene novelty across terrestrialization. Bars indicate the number of GO terms from novel HGs and novel core HGs shared by at least ten terrestrial nodes. b, Tree map visualization of GO terms from novel HGs shared across terrestrialization. The 55 most specific GO terms from the 118 shared novel HGs are grouped into 3 major GO categories: biological processes, cellular components, and molecular functions. Tree maps were generated with REVIGO61. The hierarchical layout illustrates the relationships among GO terms within each category. c, Distribution of shared Pfam domains of gene novelty across terrestrialization. Bars indicate the number of Pfam domains from novel HGs (green) and novel core HGs (blue) shared by at least five terrestrial nodes, the orange bar indicates only one Pfam domain shared among fully terrestrial lineages. These Pfams are labelled above the bars using short names. The 55 ‘most specific' (‘bottom' in the GO hierarchy) GO functions (Fig. 3b) in novel HGs include locomotion, membrane ion transport and transporter activity (osmosis), response to stimulus and neuronal functions (detection and reaction to stimulus), as well as metabolic, reproductive and developmental processes (lifecycle and diet adaptations). Additionally, cellular components include plasma membrane (related to better nutrient uptake, cell barriers and detoxification in adaptation to life on land28) and protein-containing complex (a crucial factor of membrane protein insertion29). Pfam domains echo these functions, recovering osmoregulation by neurotransmitter-gated ion channel domains, stimulus and neuronal functions by transmembrane receptor, and detoxification by cytochrome P450 (Fig. Moreover, the total number of HGs performing these functions also increased in terrestrial nodes (Supplementary Information section 1.2.1 and Extended Data Fig. 4), especially in Bdelloidea, Clitellata, Tardigrada, Onychophora, Armadillidium and Tetrapoda. Genes encompassed by these GO terms in humans (tetrapods) and the fruit fly (hexapods) also highlight the importance of biological functions linked to terrestrialization (Extended Data Table 1 and Supplementary Information section 1.2.2). Biological functions specific to terrestrial nodes further support these key adaptations to survival in terrestrial environments. Unique GO or unique Pfams are GO terms or Pfams associated with novel genes that are present in terrestrial nodes that are absent in the GO terms or Pfams of their ancestor nodes (Supplementary Figs. All nodes except bdelloid rotifers and arachnids contain unique GOs, and all except stylommatophorans and arachnids contain unique Pfams. Shared unique GOs and Pfams are related to metabolism (fatty acid metabolism and kinase activity) and ion transport, which would have helped terrestrial animals maintain water and osmotic balance30 and allow them to interact with new terrestrial environments, diets and adapt their life cycles. Although some functions of terrestrial novel genes represent exaptations from freshwater ancestors (Supplementary Fig. 4 and Supplementary Table 5), these novel genes remain functionally distinct from genes in aquatic lineages (Extended Data Fig. Also, some functions appear to be contingent, being gained early and later lost in terrestrial events (Supplementary Fig. For gene families predating these transitions, we investigated the presence of convergent gene expansions and contractions across the terrestrialization events using CAFE522. These gene families are shared with aquatic ancestors and relatives, so their convergent change in gene repertoire is an indication of parallel exaptations. Our analysis revealed ten HGs that significantly expanded their gene copy numbers in different combinations of four terrestrial nodes (Fig. 4a and Supplementary Table 7); no shared expanded HGs were found in more than four events. These convergently expanded gene families are involved in detoxification, oxidative stress, metabolism and reception of stimuli. Notable examples include the gene families cytochrome P450, which has crucial roles in xenobiotic metabolism particularly in the digestive tract31; flavin-containing monooxygenases, essential for processing toxic plant metabolites32; and glutathione S-transferase, which reduces reactive oxygen species and shows contraction in cetaceans33. We also find expansions in the G-protein-coupled receptor family, which is crucial for sensing environmental stimuli such as odours and light34. GO term enrichment analysis of expanded HGs, using the bilaterian ancestral genes as background, further supported the importance of stimulus response and ion transport functions in terrestrialization (Supplementary Information section 1.2.3, Supplementary Fig. a, Shared expanded HGs across terrestrial nodes. b, Shared contracted HGs across terrestrial nodes. The UpSet diagrams show the intersections of HGs across different combinations of 11 terrestrial nodes. Bars indicate shared expanded and contracted HGs among at least four terrestrial events (green), semi-terrestrial events (blue), and fully terrestrial events (orange). Semi-terrestrial and fully terrestrial groups are differentiated by blue and orange in set sizes (bottom left). HGs with no more than three members are labelled above the bars, other HGs are listed in Supplementary Table 7 (expanded) and Supplementary Table 10 (contracted). Approaches for HGs expansion and contraction inference and functional annotations are described in the Methods. In summary, our results suggest that gene gains (novel, novel core and expanded gene families) are a key driver across all the transitions from water to land, indicating that functions such as response to stimuli, oxidative stress, lipid metabolism and ion transporter activity had an important role in these adaptive processes. Gene reduction is another important genetic change in terrestrial events, including lost and contracted genes. Lost HGs are relatively higher in number than gene gains in most nodes (Fig. We identified lost HGs shared in terrestrial events (Supplementary Fig. 7 and Supplementary Table 9) and found the Dbl-homology domain gene family lost in 8 out of 11 terrestrial events, and the pleckstrin-homology domain gene family lost in 7 out of 11 terrestrial events; these are retained mainly in bdelloids, stylommatophorans, myriapods and tetrapods. For reference, there are no gene families convergently expanded or contracted in more than four nodes (Fig. 4), thus finding lost HGs in seven or eight terrestrial nodes is remarkable. Both domains are components of guanine nucleotide exchange factors of Rho GTPases (RhoGEF), implicated in regeneration (neurons35 and muscles36) and wound healing37. Other lost HGs among terrestrial events include chlorophyllase protein family (chlorophyll degradation38), potentially indicating dietary shifts during land colonization, and the Shugoshin C-terminal domain-containing protein, which regulates chromosome cohesion and segregation during meiosis and is involved in reproduction39. Gene families showing convergent reduction in copy number also point to key adaptations to life on land. There are four HGs that are convergently contracted in at least four terrestrial lineages (green bars in Fig. 4b, and Supplementary Table 10): chloride channel protein members (osmoregulation40), two different carbohydrate sulfotransferases (extracellular communication and adhesion41), and melatonin-related receptors (circadian rhythms42). We categorized the terrestrial lineages as semi-terrestrial or fully terrestrial according to their dependence on water, as no consensus definition of terrestriality is universally accepted. Semi-terrestrial animals rely on humid environments to avoid drying out, and include bdelloid rotifers, nematodes, tardigrades and some microscopic annelids, which require a film of water or pore spaces to live in43, as well as onychophorans and other annelids (such as clitellates). Fully terrestrial animals are less water-dependent, and encompass lineages such as land gastropods44, Arachnida, Myriapoda, Armadillidium (woodlice), Hexapoda5 and Tetrapoda. We compared GO and Pfam compositions associated with novel genes of terrestrial animal clades to capture function variation, performing both principal component analysis (PCA; Supplementary Fig. 8) and principal coordinates analysis (PCoA; Fig. In the PCoA based on Jaccard dissimilarity, semi-terrestrial and fully terrestrial groups showed partial separation, and permutational multivariate analysis of variance (PERMANOVA) confirmed significant differences between the two groups with both GO terms (Fig. The enriched novel gene functions reveal that semi-terrestrial species carry an expansive and versatile toolkit for environmental flexibility, emphasizing cuticle remodelling, visual development and stress response, while fully terrestrial species display a small and streamlined set centred on neuronal development and ion membrane homeostasis vital for permanent colonization (Supplementary Table 11). a, PCoA of Jaccard dissimilarities based on GO terms presence/absence profiles. b, PCoA of Jaccard dissimilarities based on Pfams presence/absence profiles. Each dot represents 1 of the 61 sampled terrestrial species, coloured by taxonomic group as indicated in the legend. Distances between dots correspond to Jaccard dissimilarities. Statistical ellipses highlight the semi-terrestrial group (orange; including Bdelloidea, Clitellata, Nematoda, Tardigrada and Onychophora) and the fully terrestrial group (green; including Stylommatophora, Arachnida, Myriapoda, Armadillidium, Hexapoda and Tetrapoda). Ellipses were generated using normal distribution parameters to visualize clustering patterns of taxonomic groups (95% confidence). The two axes represent the first two principal coordinates (PCoA1 and PCoA2) with their respective explained percentages of variation. Group separation was tested with PERMANOVA, which showed the significant differences between semi- and fully terrestrial groups for GO terms (R2 = 0.0995, P < 0.01) and Pfam domains (R2 = 0.0992, P < 0.01). Approaches for functional annotations are described in the Methods. Notably, consistent with this expansive-versus-streamlined enrichment pattern, semi-terrestrial groups share broad biological functions whereas fully terrestrial lineages show little overlap. 9 for novel genes, Fig. 4a and Supplementary Table 7 for expanded genes) across semi-terrestrial lineages converged on crucial functions for land adaptation, including circulatory system development, osmoregulation, nutrient processing, muscle function, energy metabolism, detoxification and sensory response mechanisms. These adaptations enabled essential physiological processes required for semi-terrestrial animals to cope with soil-dependent environments, from basic survival needs such as gas exchange, locomotion and nutrient uptake to environmental challenges such as osmotic stress and exposure to pollutants. By contrast, fully terrestrial lineages show limited convergence in the functions associated with gene novelty, with no shared GO terms and only one Pfam domain among novel genes and few shared expanded HGs (Figs. Most shared adaptations among fully terrestrial lineages are found in arthropods, where each terrestrial lineage emerged independently from aquatic ancestors that likely started from a similar genetic toolkit and evolved parallel streamlining later. Only glucose transport and stimulus sensing mechanisms are shared between woodlice and land snails, suggesting that fully terrestrial lineages probably evolved through diverse rather than common adaptive patterns. In addition, both semi-terrestrial and fully terrestrial lineages display diverse gene reduction patterns, with few shared reductions (Fig. 4b for contraction, Supplementary Fig. 7 and Supplementary Table 14 for lost genes) except within arthropods, indicating low convergence of gene reduction across both habitat categories. By uncovering novel core and exclusively expanded HGs, we inspected the gene functions associated with each of the 11 terrestrial nodes (Supplementary Information section 1.4 and Supplementary Tables 15–18). These include stress-response genes in bdelloid rotifers, nervous system and muscle adaptations of clitellates, shell formation, mucus secretion and estivation genes in land snails, and cuticle-related genes in nematodes. Tardigrades exhibit unique stress-resistance genes, whereas onychophorans share traits such as oxygen adaptation and nutrient uptake with woodlice. Arthropods and tetrapods, as well-studied terrestrial lineages, were further explored for their distinct adaptations. Arthropods, the most diverse animal phylum, originated in the sea and colonized the land multiple times independently. In this study, we focus on the fully terrestrial clades Hexapoda, Myriapoda and Arachnida and the crustacean Armadillidium5 (Supplementary Information section 1.4.7). These lineages exhibit convergent evolution of traits for terrestrial adaptation, such as exoskeleton structure, water conservation and sensory development (Fig. For instance, myriapods and hexapods expand gene families linked to the synthesis of the exoskeleton wax layer responsible for waterproofing45. Similarly, retinol-binding protein genes required in the retinal pigment cells expanded in arthropods to adapt their vision to light conditions on land46. Hexapods5 show enriched GO annotations in expanded genes (Supplementary Table 15) related to moulting (for example, terpenoid metabolic process, juvenile hormone metabolic process, sesquiterpenoid metabolic process and steroid metabolic process) and vision (for example, rhodopsin biosynthetic process). The other major lineage of terrestrial animals is land vertebrates (Supplementary Information section 1.4.8), which show both novel and expanded genes with enriched GO annotations related to immunity functions (Supplementary Table 18 for novel HGs, Supplementary Tables 15 and 16 for expanded HGs): T cell co-stimulation, positive regulation of activated T cell proliferation, and innate immunity-related processes (for example, neutrophil degranulation and specific granule lumen). Similar innate immunity functions are also found in expanded gene families, such as the Ly-6/uPAR family, siglecs, mucins and resistin. Previous studies have also supported innate immunity as crucial to evolving a specialized and reinforced epidermis with an active keratinization process and a resistant outer stratum corneum47. These defend against pathogens that spread in the terrestrial environment, forming both physical and chemical barriers48, as evidenced by our study. The invasion of land by life influenced global biogeochemical cycles through effects on carbon storage and weathering, representing a notable milestone in the evolutionary history of the planet1. Land plant colonization paved the way to new habitats for animals and fungi and produced the emergence of new ecosystems49. Molecular timescales estimate the age of lineages, including soft-bodied animals that may not be well-represented in the fossil record. Here we focus on the terrestrialization events for which we have more than one taxon or genome. Our molecular evolutionary timescale (Fig. 1) is congruent with other recent studies50 and shows that the animal conquest of land occurred in three major temporal windows. These windows might not overlap and may be separated by millions of years, each contributing to the complexity of terrestrial ecosystems. The first temporal window of terrestrialization occurred between the Middle Cambrian and Middle Ordovician epochs. Early land plants emerged49 approximately 515.0–473.6 million years ago (Ma), quickly followed by nematodes and arthropods. Our study suggests that nematodes (533.9–421.9 Ma), myriapods (521.9–402.8 Ma), hexapods (487.6–436.9 Ma) and arachnids (489.7–435.2 Ma) were among the first animals to transition to land, overlapping the rise of early land plants; an arthropod-focused study similarly reports temporal concordance5. A study estimated the origin of nematodes between Ediacaran and Silurian periods51 (620–455 Ma), overlapping, or even preceding, our interval. These early terrestrial species developed traits helping mitigate desiccation and providing structural support, including the arthropod exoskeletons and the nematode cuticles. In our analyses, gene gains in these lineages shared functions associated with cuticle formation, exoskeleton maintenance and lipid metabolism, as well as involvement in responses to drought, excessive light and oxidative stress (Supplementary Table 19), consistent with selection for water conservation and stress tolerance in patchy, intermittently wet terrestrial settings shaped by cryptogamic and bryophyte covers52. The second temporal window of terrestrialization spans the Late Devonian to Early Carboniferous subperiod, a time of episodic flooding, deepening soils and strongly seasonal wetlands53. In this ecological setting, clitellate annelids (464.5–262.8 Ma) and the first tetrapods (351.2–337.7 Ma) independently adapted to land. The first land vertebrates evolved limbs for locomotion, lungs for aerial respiration and skin barriers to minimize water loss4. Clitellates adapted their nervous and muscular systems to cope with terrestrial challenges, enhancing locomotion and desiccation resistance. These species contributed to the establishment of modern terrestrial niches by enhancing nutrient cycling, improving soil structure and influencing ecosystem communities, thereby laying the conditions for further evolutionary innovations in terrestrial life. The floodplains of this period likely provided the ecological opportunities and selection pressures that drove these terrestrial transitions. The third temporal window of terrestrialization, between 130–86 Ma during the Cretaceous period, saw bdelloid rotifers (180.9–78.4 Ma) and land gastropods (127.1–39.3 Ma) making their way onto land and sharing it with dinosaurs, as well as early mammals and birds. Bdelloid rotifers evolved exceptional stress tolerance mechanisms, including resistance to desiccation, extreme temperatures, and radiation, enabling them to thrive in harsh environments. Meanwhile, terrestrial snails developed adaptations such as shell formation, mucus secretion, and estivation to withstand diverse climatic conditions. At the molecular level, both clades exhibit gene expansions (Supplementary Table 19) in HGs, including ammonium transporters for water and ion homeostasis, NADP-dependent oxidoreductases54 and G-protein-coupled receptors for stress resistance. These shared adaptations are likely to reflect Cretaceous greenhouse landscapes, characterized by high sea levels, angiosperm expansion, coastal wetlands and seasonally dry microhabitats55 that favoured water and ion conservation and broad stress tolerance. We applied comprehensive comparative genomic analysis to uncover the convergent evolutionary processes underlying 11 independent terrestrialization events. Our results reveal that these are marked by extensive gene turnover across all 11 lineages, adaptations to terrestrial environments. We found that terrestrial events generally display a high level of genomic novelties, mainly related to osmoregulation, stress response, immunity, sensory reception, metabolism and reproduction. We found shared ion transport functions in gene novelty, supporting their critical role in the adaptation from water to land by maintaining water and ionic balance. This is especially crucial for animals adapting to low-salinity and dry environments, as it involves changes in osmoregulation to maintain ion and water homeostasis and prevent water loss. Gene reductions show notably large numbers in terrestrial lineages, with some convergent losses of genes related to regeneration56. Although all terrestrial lineages exhibit a certain degree of convergent functional evolution, semi-terrestrial lineages share functional and molecular features, whereas fully terrestrial animals do not. However, it should be noted that habitat classifications are diverse and not universal. For example, another classification, including cryptic forms, poikilohydric organisms and homoiohydric organisms, categorizes Myriapoda and woodlice as cryptic forms2, whereas here they are classified as fully terrestrial5. More comparisons using various classifications are needed in the future. In addition to the convergent emergence of biological functions, each terrestrial lineage exhibits unique adaptations to land. The distinct features indicate the various genomic pathways to thrive in terrestrial environments. Additionally, three major temporal windows of terrestrialization identified by our molecular timescale occurred during the Ordovician, Devonian–Carboniferous and Cretaceous periods. These windows were potentially driven by major ecological and geological changes, forming new terrestrial niches. These results largely support the temporal congruence between the rise of land plants and the first window of terrestrial animals, providing new insights into the tempo of terrestrialization. There are interesting convergences between the adaptation to life on land by plants and terrestrial animals19. Plants also evolved genes linked to adaptations to life outside of the water similarly to animals, such as lignin to avoid desiccation and environmental responses (abscisic acid, salicylic acid and jasmonic acid). However, they also present new genes related to UV light protection, a signal that is not observed in animals. The study faces certain limitations, such as the classification of terrestrialization highlighted above. Moreover, annotating lost and contracted genes poses challenges as these are lost in the most common model organisms, which are the reference for many functional annotations. For example, a significantly contracted HG in nematodes contains no gene copies in Caenorhabditis elegans and only a few poorly annotated copies in other nematodes. Functional analysis for such HGs often relies on distant homologues in humans or fruit flies, which may not accurately reflect functions owing to substantial sequence divergence across lineages. In more extreme cases, where the HGs are lost in traditional model organisms, annotation becomes virtually impossible. Similarly, many lost genes are classified as uncharacterized proteins, reflecting their absence in well-studied terrestrial models. Another methodological limitation is that we did not determine whether gene duplications occurred at the terrestrial nodes or independently within lineages, as CAFE5 infers gene expansions based on copy number, not gene trees. However, the observed expansions remain robust and meaningful as they consistently occur in terrestrial lineages, regardless of when they arose. Also, although our study relies on a robust phylogenetic framework, phylogenetic position incongruence complicates interpretations of terrestrial transitions, such as the debated relationships with Chelicerata. We followed recent studies that place Xiphosura as the sister group to Arachnida57, implying a single origin of terrestrialization in arachnids, whereas some placements propose that Xiphosura may be nested within arachnids58, which would suggest an alternative scenario. Additionally, there is limited taxon sampling for certain lineages, such as tardigrades, onychophorans and woodlice, which may lead to HG numbers that are not representative of the gene content of the clade. In the future, with more and more genomes being sequenced, the inclusion of more taxa in datasets will be possible. Future efforts should also focus on developing advanced annotation tools for lost and contracted genes, such as machine learning approaches (for example, language models59) to overcome challenges caused by sequence divergence and limited homologues. Moreover, improving gene family expansion inference by integrating gene tree-based approaches will be crucial to pinpoint duplication events more precisely. This study uses an integrative approach, seamlessly combining comparative genomic analysis, functional annotation and evolutionary timescale reconstruction. By leveraging the InterEvo framework, which assesses the overlap of biological functions in genes repeatedly gained or reduced in these transitions, we systematically reveal the convergent genomic patterns across diverse metazoan taxa, capturing the breadth of animal diversity and providing a robust methodology for studying convergent genome evolution. Furthermore, the analysis incorporates diverse comparisons across terrestrial lineages, offering a comprehensive perspective on both global trends and category-specific patterns of terrestrial adaptations. Many genomic adaptations to terrestrial animal life are convergent, suggesting broadly predictable molecular responses. Yet, convergence is part of the story. Each terrestrial lineage also displays its own contingent adaptations, shaped by its unique evolutionary history, genomic background and ecological context. Even when facing similar challenges, different lineages often arrive at distinct molecular solutions, reflecting their ancestral constraints and trajectories. Terrestrialization, therefore, illustrates the interplay between convergence and contingency, highlighting both the repeatability and the uniqueness of evolutionary innovation. We compiled 154 genome samplings from published projects uploaded in UniProt62, NCBI63, Ensembl64 and other resources (Supplementary Table 1). 154 genomes were downloaded, containing 3,934,362 predicted proteins, including 151 metazoan and 3 unicellular organism genomes. A side script provided by OrthoFinder (primary_transcript.py) and Cd-hit v.4.8.1 (ref. 65) (using a similarity threshold of 1.00) were used to extract the canonical proteins from the original data. The quality of canonical proteins from 154 genomes were assessed by BUSCO v.5.4.7 (ref. The completeness greater than 85% and fragmentation less than 15% were preferable. It should be noted that we considered not only genome completeness but also the habitat and phylogeny of the species, so we selected some genomes that did not perfectly meet the above standard. We then inferred HGs by Orthofinder v.2.5.5 (ref. 67), using dependencies of MAFFT v.7.505 (ref. The HGs were then used for analysing gene content. We drew a guide tree that was used for gene expansions/contractions analyses and the later time tree. This guide tree, based on the species positions inferred from previous literature70,71,72,73, was used to build the phylogeny of metazoans through the following steps. While some nodes are controversial (namely the position of sponges, ctenophors, or acoels), they are far removed from the terrestrial nodes of interest. First, we started from the conserved single-copy genes in the Metazoa_odb10 from BUSCO v.5.4.7 (ref. Homo sapiens contains 943 of these genes, which were extracted to serve as reference orthologous. These identified conserved protein sequences were aligned using MAFFT v.7.505 (ref. 68) and trimmed using trimAl v.1.4.rev.15 (ref. 74) to remove poorly aligned regions. Next, we concatenated the trimmed alignments into a single supermatrix using FASconCAT-G v.1.05.1 (ref. Finally, the concatenated supermatrix was used to build the phylogeny with IQ-TREE v.2.2.2.6 (ref. 76), using C60 + G + I model, using the guide tree as a constraint, and performing 1,000 bootstrap replicates. The resulting phylogeny, with branch lengths representing genetic changes, was subsequently used in CAFE5 for further analysis (see following methods). Novel HGs: HGs that are present in at least one species within the LCA of a lineage (following we called a node), while being absent in all species of outgroup. Novel core HGs: HGs that are present in all species within a node (or absent only once for node containing more than three species), while being absent in all species of outgroup. For the node with two species, novel HGs are equal to novel core HGs. Lost HGs: HGs that are lost in all species within a node, while being present in the sister groups and other species in outgroup. Expanded HGs: the increase in the number of gene copies occurred within HGs, often due to gene duplication events. Contracted HGs: the reduction in the number of gene copies occurred within HGs. Novel, novel core and lost HGs were inferred by our host pipeline Phylogenetically Aware Parsing Script described by Paps and Holland11 (GitHub: https://github.com/PapsLab) with Perl v.5.30.0. Expanded and contracted HGs were inferred by CAFE522. First, we generated an ultrametric phylogenetic tree with ape, TreeTools and phytools packages in R, based on phylogenetic tree built using IQ-TREE. CAFE5 was launched with the ultrametric tree. Owing to the large dataset, we were unable to run the entire phylogeny at once; therefore, we split the phylogeny into three smaller trees: Lophotrochozoa, Ecdysozoa and Deuterostomia. For each smaller tree, CAFE5 was run with Poisson distribution and error model, applying two- and three-lambda models ten times each to test convergence of Model Base Final Likelihood (-lnL). We selected the highest lnL from two- and three-lambda models to compare their fit using a likelihood ratio test with chi-squared distribution (via lmtest package in R), which indicated that three-lambda models are a better fit for all three phylogenies (P < 0.001). However, further tests using simulation function of CAFE5 revealed that values (including lambda and -lnL) of three-lambda model of Deuterostomia fluctuated, while that of two-lambda model were stable, thus the two-lambda model was judged to be the better fit for this phylogeny. For Lophotrochozoa and Ecdysozoa, the simulation tests showed stable values for the three-lambda models, which were therefore chosen as the better fit (Supplementary Table 20). To test the robustness, novel core HGs were tested by BLASTp v.2.14.0 +77 using NCBI RefSeq database78 (downloaded on 23 August 2023), which contains a broad range of high-quality molecular sequences. We launched BLASTp locally and searched novel core HGs against RefSeq records, excluding protein sequences from the in-groups (terrestrial nodes) with the option “-negative_taxidlist”. The results shown that BLASTp returned very weak hits; the vast majority of sequences had e-value > 10−10 and identity <50% (Supplementary Table 21). We evaluated if the number of novel genes emerging per million years in terrestrial nodes differ from aquatic nodes. We collected the rate of emergence of novel gene in 11 terrestrial nodes and randomly selected 11 aquatic nodes (Actinopterygii, Ambulacraria, Bivalvia, Branchiopoda, Chondrichthyes, Cnidaria, Decapoda, Platyhelminthes, Priapulida, Sabellida and Vetigastropoda). We calculated the observed total evolutionary rate in the 11 terrestrial nodes as the total number of novel HGs divided by total divergence time (Rterr = 4.900). We then performed 10,000 bootstrap draws: in each permutation we sampled (with replacement) 11 aquatic nodes from this pool, recalculated the evolutionary rate (Rboot = total novel HG counts divided by total divergence time) and recorded the value, producing a null distribution of novel gene rates in aquatic nodes. The empirical one-tailed P value was the proportion of bootstraps with Rboot ≥ Rterr. We assessed if the GO term composition of terrestrial lineages differs from that of aquatic lineages. We included lineages with the biggest taxon sampling from random aquatic lineages, including Actinopterygii, Ambulacraria, Bivalvia, Branchiopoda, Cnidaria, Decapoda and Platyhelminthes. We converted the GO matrix derived from the novel genes for each lineage into a binary presence/absence matrix, then quantified the dissimilarity between terrestrial and aquatic GO term profiles by measuring the proportion of non-shared terms (Jaccard distance). For the permutation test, we randomly reshuffled the ‘aquatic/terrestrial' labels across lineages 10,000 times; in each permutation we rebuilt the two group profiles and recalculated Jaccard distance between them. Both analyses were conducted in R using the packages vegan, car and ggplot2. For each terrestrial event, we selected one species as representative. For annotating Pfam domains and GO terms of the HGs of interest, egg-NOG-mapper v.2 (ref. 79) was applied online with default parameters. Further analysis of names of genes of interest was performed in UniProt62 using mapping IDs in sequence headers. We also used PANTHER 19.0 (ref. 80) to classify genes with ‘protein class'. We conducted GO enrichment analysis to find overrepresented GO terms in novel and expanded HGs of terrestrial events, using background of GO terms hitting all HGs present in the LCA of Bilateria. Using Fisher's exact test, we compared the number of HGs hitting each GO term between the terrestrial events and bilaterian background. The P values for multiple comparisons were corrected with the Benjamini–Hochberg method. GO terms with adjusted P values < 0.05 were considered significantly enriched. In another way, we also used all HGs present in the LCA of terrestrial events as background to perform enrichment analysis, such as GO terms hitting expanded HGs of hexapods comparing with GO terms hitting all HGs present in LCA hexapods. However, to ensure normalization across all terrestrial events, we chose bilaterian background for the following analysis. To identify biological functions driving the separation of semi-terrestrial and fully terrestrial groups (following the PCoA), we tested differential presence of GO terms or Pfams between semi-terrestrial and fully terrestrial groups using binary matrices (present/absent). Functional terms that lacked variability (present in all species or in none) were discarded. For every remaining feature we compiled a 2 × 2 contingency table (presence/absence number of species in two habitat categories) and subjected it to a two-tailed Fisher's Exact Test in R, using the marginal totals across entire pool of species as the background. P values were corrected for multiple comparisons using the Benjamini–Hochberg method. The functional terms with adjusted P < 0.05 were considered significantly enriched and those of P < 0.01 were reported. To retain biological relevance, we excluded the functional terms present in ≤10% proportion in both groups. To compare the distribution in GO terms linked to novel and ancestral HGs among semi-terrestrial and fully terrestrial lineages, we performed a PCA. PCA was conducted using the prcomp function in R. The GO terms of species were plotted using the first two principal components, PC1 and PC2. Statistical analyses were applied to assess differences between semi-terrestrial and fully terrestrial groups. Analysis of variance (ANOVA) and Tukey's honest significant difference (HSD) test was performed on the principal components scores to evaluate significant differences among these two groups and pairwise comparisons. Then, a multivariate analysis of variance (MANOVA) was conducted to examine the combined effect of these two groups on PC1 and PC2, respectively. Two ellipses, using normal distribution-based ellipse fitting, were merged based on their habitats, representing semi-terrestrial and fully terrestrial groups. The semi-terrestrial group includes bdelloid rotifers, clitellates, nematodes, tardigrades and onychophorans, while fully terrestrial group includes stylommatophorans, arachnids, myriapods, Armadillidium, hexapods and tetrapods. The analyses were performed in R using the ggplot2, ggforce and car packages. However, because shared absences might bias Euclidean-based PCA on binary presence/absence data, inflating similarity between groups that simply lack many of the same features, we further performed PCoA. We quantified compositional differences in GO term and Pfam presence/absence profiles between semi-terrestrial and fully terrestrial species based on Jaccard dissimilarity. Pairwise dissimilarities among species were computed using the Jaccard distance in vegan R package. We then performed PCoA on the Jaccard distance matrix. The two axis labels explain the percentages of Jaccard distance variation. To test for overall group difference between semi-terrestrial and fully terrestrial groups, we performed a PERMANOVA (adonis2 function) on the Jaccard distances with 10,000 permutations, with reassigning species to the two groups. To confirm the PERMANOVA results are not driven by unequal within-group spread, we tested for homogeneity of multivariate dispersion using betadisper function with permutations of sample–group labels (centroids recomputed each 999 permutation). Plots were generated using ggplot2 package and group ellipses represent 95% concentration regions. The comparison between freshwater and terrestrial species used the same methods described above. Molecular clock analysis was performed using a two-step approach in MCMCTree81 (PAML package82). Using the previously described concatenated alignment of 943 conserved orthologous genes generated using BUSCO v.5.4.7 (ref. The analysis was conducted in two steps. In the first step, branch lengths were estimated by maximum likelihood using CODEML83, which calculated the gradient and Hessian of the likelihood function at the maximum likelihood estimates. Subsequently, MCMCTree was executed to estimate divergence times, using the same independent rates clock model and discrete gamma distribution with 4 categories and a shape parameter alpha = 0.5. Using an R script provided by MCMCTree tutorial (GitHub: https://github.com/sabifo4/Tutorial_MCMCtree), the prior for the substitution rate was determined based on the approximate root age (591.255 Ma), resulting in a gamma distribution with shape α = 2 and scale β = 5.1. For each analysis, we ran the MCMC for about 20 million generations, with the first 100,000 generations discarded as burn-in, sampling every 1,000 generations to obtain 20,000 samples. To ensure convergence and reliability of the results, we performed six independent Markov chain Monte Carlo runs. We assessed convergence using Tracer v.1.7.2 (ref. 84), which showed effective sample sizes exceeding 200 for all parameters across all runs. Based on the consistency of results across runs and comparative summary statistics, we selected the fourth run for our final divergence time estimates (Supplementary Table 22). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All genome data analysed in this study are available from public databases, including UniProt, NCBI, Ensembl and other resources. The specific publications and download links for the 154 genomes are provided in Supplementary Table 1. The computer code used in the analyses has been deposited in GitHub (https://github.com/JLWei7/animal_terrestrialisation). Kenrick, P., Wellman, C. H., Schneider, H. & Edgecombe, G. D. A timeline for terrestrialization: consequences for the carbon cycle in the Palaeozoic. Selden, P. A. Encyclopedia of Life Sciences: Terrestrialization (Precambrian–Devonian) (John Wiley & Sons, 2005). Brusca, R. C., Giribet, G. & Moore, W. Invertebrates, 4th edn (Oxford Univ. Ashley-Ross, M. A., Hsieh, S. T., Gibb, A. C. & Blob, R. W. Vertebrate land invasions-past, present, and future: an introduction to the symposium. Lozano-Fernandez, J. et al. A molecular palaeobiological exploration of arthropod terrestrialization. Barker, G. M. Naturalised terrestrial Stylommatophora (Mollusca: Gastropoda). Survival in extreme environments—on the current knowledge of adaptations in tardigrades. Menter, D. G. et al. Of vascular defense, hemostasis, cancer, and platelet biology: an evolutionary perspective. & Rezende, E. L. Temperature variability and metabolic adaptation in terrestrial and aquatic ectotherms. Nilsson, D. E. Evolution: an irresistibly clear view of land. & Holland, P. W. H. Reconstruction of the ancestral metazoan genome reveals an increase in genomic novelty. Fernandez, R. & Gabaldon, T. Gene gain and loss across the metazoan tree of life. Guijarro-Clarke, C., Holland, P. W. H. & Paps, J. Widespread patterns of gene loss in the evolution of the animal kingdom. Parallel duplication and loss of aquaporin-coding genes during the “out of the sea” transition as potential key drivers of animal terrestrialization. Aristide, L. & Fernández, R. Genomic insights into mollusk terrestrialization: parallel and convergent gene family expansions as key facilitators in out-of-the-sea transitions. Thomas, G. W. C. et al. Gene content evolution in the arthropods. Parallel and convergent genomic changes underlie independent subterranean colonization across beetles. Vargas-Chavez, C. et al. An episodic burst of massive genomic rearrangements and the origin of non-marine annelids. The origin of land plants is rooted in two bursts of genomic novelty. World Register of Marine Species (WoRMS) (Flanders Marine Institute, 2024); https://www.marinespecies.org. Fernández, R., Gabaldon, T. & Dessimoz, C. Phylogenetics in the Genomic Era: Orthology: Definitions, Prediction, and Impact on Species Phylogeny Inference (2020). Mendes, F. K., Vanderpool, D., Fulton, B. & Hahn, M. W. CAFE 5 models variation in evolutionary rates among gene families. Natsidis, P., Kapli, P., Schiffer, P. H. & Telford, M. J. Systematic errors in orthology inference and their effects on evolutionary analyses. Ashburner, M. et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium The Gene Ontology knowledgebase in 2023. Finn, R. D. et al. Pfam: the protein families database. The evolution of plant proton pump regulation via the R domain may have facilitated plant terrestrialization. Guna, A., Volkmar, N., Christianson, J. C. & Hegde, R. S. The ER membrane protein complex is a transmembrane domain insertase. Thiel, M. & Watling, L. Lifestyles and Feeding Biology: The Natural History of the Crustacea (Oxford Univ. Snyder, M. J. Cytochrome P450 enzymes in aquatic invertebrates: recent advances and future directions. Naumann, C., Hartmann, T. & Ober, D. Evolutionary recruitment of a flavin-dependent monooxygenase for the detoxification of host plant-acquired pyrrolizidine alkaloids in the alkaloid-defended arctiid moth Tyria jacobaeae. Tian, R., Seim, I., Ren, W., Xu, S. & Yang, G. Contraction of the ROS scavenging enzyme glutathione S-transferase gene family in cetaceans. & Kobilka, B. K. The molecular basis of G protein-coupled receptor activation. The integrin signaling network promotes axon regeneration via the Src–Ephexin–RhoA GTPase signaling axis. You, J. S. et al. ARHGEF3 regulates skeletal muscle regeneration and strength through autophagy. Nakamura, M., Verboon, J. M. & Parkhurst, S. M. Prepatterning by RhoGEFs governs Rho GTPase spatiotemporal dynamics during wound repair. Tsuchiya, T. et al. Cloning of chlorophyllase, the key enzyme in chlorophyll degradation: finding of a lipase motif and the induction by methyl jasmonate. Orth, M. et al. Shugoshin is a Mad1/Cdc20-like interactor of Mad2. Bradley, T. J. Terrestrial Animals: Animal Osmoregulation (Oxford Univ. Bowman, K. G. & Bertozzi, C. R. Carbohydrate sulfotransferases: mediators of extracellular communication. & Kato, M. Terrestrial invasion of pomatiopsid gastropods in the heavy-snow region of the Japanese Archipelago. Wang, T. & Montell, C. Rhodopsin formation in Drosophila is dependent on the PINTA retinoid-binding protein. Water relations of tetrapod integument. Riera Romo, M., Perez-Martinez, D. & Castillo Ferrer, C. Innate immunity in vertebrates: an overview. The timescale of early land plant evolution. Carlisle, E., Yin, Z., Pisani, D. & Donoghue, P. C. J. Ediacaran origin and Ediacaran–Cambrian diversification of Metazoa. Qing, X. et al. Phylogenomic insights into the evolution and origin of nematoda. Mitchell, R. L. et al. Cryptogamic ground covers as analogues for early terrestrial biospheres: initiation and evolution of biologically mediated proto-soils. The terrestrial landscapes of tetrapod evolution in earliest Carboniferous seasonal wetlands of SE Scotland. Selles Vidal, L., Kelly, C. L., Mordaka, P. M. & Heap, J. T. Review of NAD(P)H-dependent oxidoreductases: Properties, engineering and application. Benton, M. J., Wilf, P. & Sauquet, H. The angiosperm terrestrial revolution and the origins of modern biodiversity. Alibardi, L. Regeneration among animals: an evolutionary hypothesis related to aquatic versus terrestrial environment. Lozano-Fernandez, J. et al. Increasing species sampling in chelicerate genomic-scale datasets provides support for monophyly of Acari and Arachnida. Martínez-Redondo, G. I. et al. FANTASIA leverages language models to decode the functional dark proteome across the animal tree of life. & Gehlenborg, N. UpSetR: an R package for the visualization of intersecting sets and their properties. Supek, F., Bošnjak, M., Škunca, N. & Šmuc, T. REVIGO summarizes and visualizes long lists of gene ontology terms. Database resources of the National Center for Biotechnology Information in 2023. Martin, F. J. et al. Ensembl 2023. Li, W. & Godzik, A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. Manni, M., Berkeley, M. R., Seppey, M., Simao, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Buchfink, B., Reuter, K. & Drost, H. G. Sensitive protein alignments at tree-of-life scale using DIAMOND. Laumer, C. E. et al. Revisiting metazoan phylogeny with genomic sampling of all phyla. The role of homology and orthology in the phylogenomic analysis of metazoan gene content. Evidence for sponges as sister to all other animals from partitioned phylogenomics with mixture models and recoding. Simion, P. et al. A large and consistent phylogenomic dataset supports sponges as the sister group to all other animals. Capella-Gutierrez, S., Silla-Martinez, J. M. & Gabaldon, T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Kuck, P. & Longo, G. C. FASconCAT-G: extensive functions for multiple sequence alignment preparations concerning phylogenetic studies. Minh, B. Q. et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. Bealer, K. et al. BLAST+: architecture and applications. O'Leary, N. A. et al. Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Cantalapiedra, C. P., Hernandez-Plaza, A., Letunic, I., Bork, P. & Huerta-Cepas, J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Thomas, P. D. et al. PANTHER: Making genome-scale phylogenetics accessible to all. Alvarez-Carretero, S. et al. A species-level timeline of mammal evolution integrating phylogenomic data. Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. Álvarez-Carretero, S., Kapli, P. & Yang, Z. Beginner's guide on the use of PAML to detect positive selection. Rambaut, A., Drummond, A. J., Xie, D., Baele, G. & Suchard, M. A. Posterior summarization in Bayesian phylogenetics Using Tracer 1.7. This work used ACRC HPC University of Bristol. We thank P. Holland, T. Williams and J. Vinther for their comments and suggestions on the analyses. is supported by China Scholarship Council-University of Bristol joint-funded Scholarship (202206350023). M.Á.-P. and J.P. are supported by the Wellcome Trust (210101/Z/18/Z) and the School of Biological Sciences (University of Bristol). M.Á.-P. was also supported by a fellowship from the Fundación General CSIC´s ComFuturo, which received funding from the European Union's Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement 101034263. M.Á.-P. is recipient of Ramon y Cajal grant RYC2023‐043807‐I of Spanish Ministry of Science, Innovation and Universities (MCIN/AEI/ and El FSE). is supported by Gordon and Betty Moore Foundation grant GBMF9741, Leverhulme Trust Research Fellowship grant RF-2022-167, Biotechnology and Biological Sciences Research Council grants BB/T012773/1 and BB/Y003624/1. is supported by Gordon and Betty Moore Foundation grant GBMF9741 and Leverhulme Research Project Grant RPG-2024-030. These authors contributed equally: Marta Álvarez-Presas, Jordi Paps Jialin Wei, Davide Pisani & Jordi Paps Davide Pisani & Philip C. J. Donoghue Institut de Biologia Evolutiva (CSIC-Universitat Pompeu Fabra), Barcelona, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Methodology: J.W., M.Á.-P. and J.P. Data analysis: J.W. Writing, review and editing: J.W., D.P., P.C.J.D., M.Á.P. Correspondence to Jordi Paps. The authors declare no competing interests. Nature thanks José Martín-Durán, Sebastian Shimeld and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The pipeline used in this study comprises three main analyses following homology group (HG) inference and ancestral genome reconstruction. Analyses of novel and expanded HGs involve identifying shared biological functions (including Gene Ontology (GO) terms and Pfam domains) across transition nodes, followed by enrichment analysis. The workflow integrates comparative genomics, homology inference, and functional annotation to detect convergent evolution patterns across independent terrestrialisation events. Species tree of the 154 taxa sampled in this study. The habitat types are indicated as follows: marine (dark blue nodes), brackish (grey nodes), freshwater (light blue nodes), semi-terrestrial (yellow circles with green outlines), and fully terrestrial (red circles with green outlines). a, Permutation test assesing whether the rate of novel genes emergence per million years (Myr) in terrestrial nodes is significantly higher than in aquatic nodes. The observed terrestrial rate is indicated by the red bar. b, Permutation test assesing whether the biological functions in terrestrial nodes are significantly different from those in other nodes The observed GO distance between terrestrial and aquatic groups is indicated by the red bar. See Supplementary Text 1.1 for details. The heatmap compares the number of novel HGs associated with 10 most specific GO terms in terrestrial nodes and their three immediate ancestors. These 10 GO terms were selected from the bottom-level hierarchy of the 27 GO terms of novel genes shared across all terrestrial nodes. Terrestrial nodes are highlighted in green text, and their ancestor nodes are shown in black. Columns represent GO terms, and cells show the number of novel HGs associated with each term. The colour gradient from red (low) through white to blue (high) represents the log-transformed values of HG numbers to improve visualisation of differences in scale. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Convergent genome evolution shaped the emergence of terrestrial animals. Version of record: 12 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69278578/reverse-anxiety-amygdala/'>Scientists Have Pinpointed Where Our Anxiety Comes From—and Might Be Able to Reverse It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists identified a gene in the amygdala that can cause anxiety by over-expressing itself. They treated that gene in mice and saw an instant change. Dystopian headlines in the news, not to mention the doomscrolling sessions they breed, already seem like enough to trigger near-apocalyptic levels of anxiety, but (dumpster fires and social media aside) its origins were uncertain—until now. Overactivation of the amygdala, which processes emotional stimuli, switches on an inflammatory response when faced with stress. Inflammation is at least partly behind conditions that involve anxiety and depression (common comorbidities). Stress is also often a contributing factor, which can lead to the dysfunction of neurotransmitters like serotonin and epinephrine (antidepressants can help treat this dysfunction by raising neurotransmitter levels). He and his research team found that over-expression of one gene can stress the amygdala and put the brain on edge. Butut this can cross the threshold to neural overexcitement. Lerma wanted to see what would happen if he targeted specific neurons in the amygdalas of mice that had been genetically modified with extra copies of Grik4. “Mice overexpressing the Grik4 gene display anxiety, depression, social deficits, and disrupted amygdala excitability.” Previous studies have suggested that this region helps with overcoming stress by regulating how the brain learns after a stressful experience. The researchers focused on these neurons because of their potential to become overexcited. Grik4 is expressed throughout the amygdala, and if expression spikes in basolateral amygdala neurons, it can affect communication with neurons in the central amygdala—especially a group in the centrolateral amygdala, which are responsible for firing nerve impulses. Lowering Grik4 levels normalized communication between neural groups in the basolateral and centrolateral amygdala. “The identification of such a critical role for a small population of neurons in the [centrolateral] amygdala circuitry with specific firing properties pinpoints how subtle changes in input strength may be enough to reversibly distort circuit activity and provides a new target for the therapy of affective disorders,” Lerma said. Further research will look at how other parts of the brain, such as the hippocampus, may be involved in conditions that produce or contribute to high anxiety. Opportunities for human research might arise in the future, but until then, some deep breathing exercises can help. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Ozempic-Style Drugs May Soon Not Need a Needle Flipping A Cellular Death Switch Could Slow Aging Scientists Are Closer Than Ever To a Cure for HIV</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03025-4'>How geopolitics is reshaping research: Data hint at shifting collaborations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 11:50:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Geopolitical tensions are affecting international collaborations in key areas of research, including the semiconductor sector.Credit: Rebecca Bailey/AFP via Getty Data indicate that geopolitical tensions are affecting how scientists around the world collaborate on research projects. “We will have a much clearer picture once the complete data for 2025 has been compiled and reviewed early next year,” Paz adds. However, academic boycotts are dangerous and may very well cause more harm than good,” they wrote in the press release. Unlike individual scientists boycotting Israeli research, the decline in partnerships with Russia involves top-down policy statements from governments, most notably in the US, “to reduce or eliminate scientific work with Russia,” says Caroline Wagner, a science policy researcher at Ohio State University in Columbus. An analysis by Nature in 2023 found research collaborations between Russia and Western institutions had dropped a year into its war on Ukraine. This decline was not seen for China and India, whose collaborations with Russia increased the same year. A separate analysis published in August this year by VoxUkraine, an independent analytical platform based in Kyiv, found that global publications with Russian co-authors dropped by 26% between 2021 and 2024. “Sanctions, funding cuts, brain drain, and tighter state control have deepened the decline since 2022”, says Zhang. China–US research collaborations are in decline — this is bad news for everyone China–US research collaborations are in decline — this is bad news for everyone ‘Science is happening': Israeli and Palestinian scientists continue collaborations amid c Data hint at Russia's shifting science collaborations after year of war ‘Godfather of AI' becomes first person to hit one million citations Influential list of highly cited researchers now shuts out more scientists: here's why The future of ageing: science aims to deliver another leap in lifespan Influential list of highly cited researchers now shuts out more scientists: here's why We are looking for a motivated and skilled Senior Scientist to join Antibody-Drug Conjugates (ADC) Target Discovery Team. We are looking for a Research Scientist within Orion Pharma's Immuno-Oncology target discovery team. We are seeking an accomplished Target Discovery Lead to join the Cancer Genomics Research Team. Principal investigator recruitment in infection and immunity at Institute of Human Immunology, SMART ‘Science is happening': Israeli and Palestinian scientists continue collaborations amid c An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-01994-7'>Stereodivergent construction of non-adjacent stereocentres via migratory functionalization of alkenes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 11:25:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Chemistry (2025)Cite this article Stereodivergent construction of multiple stereocentres is one of the most essential tasks in asymmetric synthesis. However, strategies for assembling all possible stereoisomers of optically active compounds bearing non-adjacent stereocentres remain scarce and suffer from certain limitations in terms of reaction types and chemical space. Here we succeed in utilizing the chain walking strategy to realize the simultaneous construction of acyclic 1,n-non-adjacent (n = 3 or 4) stereocentres via Ni-catalysed migratory hydroalkylation of trisubstituted alkenes in an enantioselective and diastereodivergent manner. A series of alkyl units can be site-selectively installed at α-C(sp3)–H sites adjacent to nitrogen, affording chiral amines bearing an α-stereogenic centre and a remote (γ- or δ-) all-alkyl-substituted stereocentre. All four stereoisomers can be accessed using a single catalyst via an appropriate selection of the olefin geometry and ligand configuration, with exceptional control of stereochemistry. This simple and mild platform offers opportunities for the streamlined synthesis of complex bioactive molecules and medicinally relevant scaffolds. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Crystallographic data for the structures reported in this article have been deposited in the CCDC with numbers 2342411 (3g), 2429194 ((R,S)-4ae), 2431389 ((S,S)-4ae) and 2431460 ((R,R)-4ae). Copies of the data are available free of charge at https://www.ccdc.cam.ac.uk/structures/. All other data in support of the findings of this study are available within the article and its Supplementary Information. Jozwiak, K., Lough, W. J. & Wainer, I. W. Drug Stereochemistry: Analytical Methods and Pharmacology 3rd edn (Informa, 2012). Rentsch, K. M. The importance of stereoselective determination of drugs in the clinical laboratory. Google Scholar Wermuth, C. G. The Practice of Medicinal Chemistry 3rd edn (Elsevier, 2008). Prosser, K. E., Stokes, R. W. & Cohen, S. M. Evaluation of 3-dimensionality in approved and experimental drug space. ACS Med. Lett. PubMed PubMed Central Google Scholar Hoveyda, A. H., Evans, D. A. & Fu, G. C. Substrate-directable chemical reactions. Google Scholar Yoon, T. P. & Jacobsen, E. N. Privileged chiral catalysts. Google Scholar Zhou, Q.-L. Privileged Chiral Ligands and Catalysts (Wiley, 2011). Wang, Y., Liu, X. & Deng, L. Dual-function cinchona alkaloid catalysis: catalytic asymmetric tandem conjugate addition-protonation for the direct creation of nonadjacent stereocenters. J. Google Scholar Wang, B., Wu, F., Wang, Y., Liu, X. & Deng, L. Control of diastereoselectivity in tandem asymmetric reactions generating nonadjacent stereocenters with bifunctional catalysis by cinchona alkaloids. J. Google Scholar Tsubogo, T., Saito, S., Seki, K., Yamashita, Y. & Kobayashi, S. Development of catalytic asymmetric 1,4-addition and [3+2] cycloaddition reactions using chiral calcium complexes. Google Scholar Zhu, B. et al. Chemoselective switch in the asymmetric organocatalysis of 5H-oxazol-4-ones and nItaconimides: addition-protonation or [4+2] cycloaddition. Google Scholar Li, Z., Hu, B., Wu, Y., Fei, C. & Deng, L. Control of chemoselectivity in asymmetric tandem reactions: direct synthesis of chiral amines bearing nonadjacent stereocenters. PubMed PubMed Central Google Scholar Trost, B. M., Zell, D., Hohn, C., Mata, G. & Maruniak, A. Enantioand diastereoselective synthesis of chiral allenes by palladium-catalyzed asymmetric [3+2] cycloaddition reactions. Google Scholar Trost, B. M., Schultz, J. E., Chang, T. & Maduabum, M. R. Chemo-, regio-, diastereo-, and enantioselective palladium allylic alkylation of 1,3-dioxaboroles as synthetic equivalents of α-hydroxyketones. Google Scholar Davis, C. R., Luvaga, I. K. & Ready, J. M. Enantioselective allylation of alkenyl boronates promotes a 1,2-metalate rearrangement with 1,3-diastereocontrol. PubMed PubMed Central Google Scholar Jiang, X. et al. Asymmetric synthesis of chiral 1,2-bis(boronic) esters featuring acyclic, non-adjacent 1,3-stereocenters. Google Scholar Krautwald, S., Sarlah, D., Schafroth, M. A. & Carreira, E. M. Enantio- and diastereodivergent dual catalysis: α-allylation of branched aldehydes. Google Scholar Huo, X., He, R., Zhang, X. & Zhang, W. An Ir/Zn dual catalysis for enantio- and diastereodivergent α-allylation of α-hydroxyketones. Google Scholar Masson-Makdissi, J., Prieto, L., Abel-Snape, X. & Lautens, M. Enantio- and diastereodivergent sequential catalysis featuring two transition-metal-catalyzed asymmetric reactions. Google Scholar Chang, X. et al. Stereodivergent construction of 1,4-nonadjacent stereocenters via hydroalkylation of racemic allylic alcohols enabled by copper/ruthenium relay catalysis. Google Scholar Xie, J., Hou, Y., Feng, Z. & You, S. Stereodivergent construction of 1,3-chiral centers via tandem asymmetric conjugate addition and allylic substitution reaction. Google Scholar Wang, H., Zhang, R. & Zi, W. Synergistic palladium/copper-catalyzed 1,4-difunctionalization of 1,3-dienes for stereodivergent construction of 1,5-nonadjacent stereocenters. Google Scholar Yang, S., Wang, Y., Zhao, W., Lin, G. & He, Z. Stereodivergent synthesis of tertiary fluoride-tethered allenes via copper and palladium dual catalysis. Google Scholar Zhang, J. et al. Enantio- and diastereodivergent construction of 1,3-nonadjacent stereocenters bearing axial and central chirality through synergistic Pd/Cu catalysis. Google Scholar You, C., Shi, M., Mi, X. & Luo, S. Asymmetric α-allylic allenylation of β-ketocarbonyls and aldehydes by synergistic Pd/chiral primary amine catalysis. PubMed PubMed Central Google Scholar Dai, J. et al. Construction of acyclic all-carbon quaternary stereocenters and 1,3-nonadjacent stereoelements via organo/metal dual catalyzed asymmetric allenylic substitution of aldehydes. Google Scholar Zhang, J. et al. Synergistic Pd/Cu-catalyzed 1,5-double chiral inductions. Google Scholar Gayen, P., Sar, S. & Ghorai, P. Stereodivergent synthesis of spiroaminals via chiral bifunctional hydrogen bonding organocatalysis. Google Scholar Wang, Y., Li, E. Q. & Duan, Z. Ligand-dependent, palladium-catalyzed stereodivergent synthesis of chiral tetrahydroquinolines. PubMed PubMed Central Google Scholar Pan, Q. et al. Ligand-controlled, nickel-catalyzed stereodivergent construction of 1,3-nonadjacent stereocenters. Google Scholar Vasseur, A., Bruffaerts, J. & Marek, I. Remote functionalization through alkene isomerization. Google Scholar Sommer, H., Juliá-Hernández, F., Martin, R. & Marek, I. Walking metals for remote functionalization. ACS Cent. PubMed PubMed Central Google Scholar Han, J., He, R. & Wang, C.Transition metal-catalyzed asymmetric three-component dicarbofunctionalization of unactivated alkenes. Chem Catal. Google Scholar Rodrigalvarez, J., Haut, F. L. & Martin, R. Regiodivergent sp3 C–H functionalization via Ni-catalyzed chain-walking reactions. JACS Au 3, 3270–3282 (2023). PubMed PubMed Central Google Scholar Wang, Y., He, Y. & Zhu, S. Nickel-catalyzed migratory cross-coupling reactions: new opportunities for selective C–H functionalization. Google Scholar Fu, C. et al. Copper/ruthenium relay catalysis for stereodivergent access to δ-hydroxy α-amino acids and small peptides. Google Scholar Li, P. et al. Stereodivergent access to non-natural α-amino acids via enantio-and Z/E-selective catalysis. Google Scholar Shi, S. L., Wong, Z. L. & Buchwald, S. L. Copper-catalysed enantioselective stereodivergent synthesis of amino alcohols. PubMed PubMed Central Google Scholar Zhang, W. W. & Li, B. J. Enantioselective hydrosilylation of β,β-disubstituted enamides to construct α-aminosilanes with vicinal stereocenters. Google Scholar Lu, H. X. et al. Rhodium-catalyzed highly enantioselective hydroboration of acyclic tetrasubstituted alkenes directed by an amide. Google Scholar Wang, M. et al. Asymmetric hydrogenation of ketimines with minimally different alkyl groups. Google Scholar Wang, Z. C. et al. Enantioselective C–C cross-coupling of unactivated alkenes. Google Scholar Xie, L. et al. Directed nickel-catalyzed regio- and diastereoselective arylamination of unactivated alkenes. PubMed PubMed Central Google Scholar Luo, X. et al. Enantioselective synthesis of multifunctional alkylboronates via N-heterocyclic carbene–nickel-catalysed carboboration of alkenes. Google Scholar Zhao, L. et al. Ligand-controlled NiH-catalyzed regiodivergent chain-walking hydroalkylation of alkenes. Google Scholar Xie, L. et al. Ligand-controlled NiH-catalyzed regiodivergent and enantioselective hydroamination of alkenyl amides. ACS Catal. Google Scholar Liu, M. et al. Asymmetric NiH-catalyzed regioselective hydroalkynylation of unactivated alkenes. CCS Chem. Google Scholar Huang, J. et al. Enantioselective Ni-catalyzed 1,2-borylalkynylation of unactivated alkenes. Google Scholar Wang, J.-W. et al. Nickel-catalyzed switchable site-selective alkene hydroalkylation by temperature regulation. Google Scholar Yang, P.-F. & Shu, W. Orthogonal access to α-/β-branched/linear aliphatic amines by catalyst-tuned regiodivergent hydroalkylations. Google Scholar Chen, C., Guo, W., Qiao, D. & Zhu, S. Synthesis of enantioenriched 1,2-cis disubstituted cycloalkanes by convergent NiH catalysis. Google Scholar Wang, X., Xue, J. & Rong, Z. Q. Divergent access to chiral C2- and C3-alkylated pyrrolidines by catalyst-tuned regio- and enantioselective C(sp3)–C(sp3) coupling. Google Scholar Wang, J. W. et al. Nickel-catalyzed remote asymmetric hydroalkylation of alkenyl ethers to access ethers of chiral dialkyl carbinols. Google Scholar Rodrigalvarez, J., Wang, H. & Martin, R. Native amides as enabling vehicles for forging sp3–sp3 architectures via interrupted deaminative Ni-catalyzed chain-walking. PubMed PubMed Central Google Scholar Li, P., Kou, G., Feng, T., Wang, M. & Qiu, Y. Electrochemical NiH-catalyzed C(sp3)−C(sp3) coupling of alkyl halides and alkyl alkenes. Google Scholar Chen, C. et al. Rapid access to chiral α-alkylated azacycles via NiH-catalyzed asymmetric migratory hydroalkylation of N-heterocyclic alkenes. CCS Chem. Google Scholar He, H. D. et al. Diphosphine ligand-enabled nickel-catalyzed chelate-assisted inner-selective migratory hydroarylation of alkenes. Google Scholar Zhu, S., Niljianskul, N. & Buchwald, S. A direct approach to amines with remote stereocentres by enantioselective CuH-catalysed reductive relay hydroamination. PubMed PubMed Central Google Scholar Chen, J. & Zhu, S. Nickel-catalyzed multicomponent coupling: synthesis of α-chiral ketones by reductive hydrocarbonylation of alkenes. Google Scholar Jiang, X. et al. Nickel-catalysed migratory hydroalkynylation and enantioselective hydroalkynylation of olefins with bromoalkynes. PubMed PubMed Central Google Scholar Liu, D., Wang, J. W., Lu, X. & Fu, Y. Entropy effects in temperature-regulated nickel-catalyzed regiodivergent alkene hydroalkylation. Google Scholar Al-Qawasmeh, R. A. et al. 11-Phenyl-[b,e]-dibenzazepine compounds: novel antitumor agents. Google Scholar Nchinda, A. T., Chibale, K., Redelinghuys, P. & Sturrock, E. D. Synthesis of novel keto-ACE analogues as domain-selective angiotensin I-converting enzyme inhibitors. Google Scholar Zhou, W. et al. A practical approach to asymmetric synthesis of dolastatin 10. Google Scholar Birch, M. et al. A simplified process for the manufacture of imagabalin hydrochloride (PD-0332334), an α2δ-ligand for the treatment of generalised anxiety disorder. Process Res. Google Scholar Bera, S., Fan, C. & Hu, X. Enantio- and diastereoselective construction of vicinal C(sp3) centres via nickel-catalysed hydroalkylation of alkenes. Google Scholar Schley, N. D. & Fu, G. C. Nickel-catalyzed Negishi arylations of propargylic bromides: a mechanistic investigation. PubMed PubMed Central Google Scholar Wang, Z., Yin, H. & Fu, G. C. Catalytic enantioconvergent coupling of secondary and tertiary electrophiles with olefins. PubMed PubMed Central Google Scholar Download references This work was supported by the National Natural Science Foundation of China (grant no. and grant nos. 22073066 and 22471191 to G.H. ), the Natural Science Foundation of Tianjin (grant no. and funding from Tianjin Normal University to C.W. These authors contributed equally: Guodong Ju, Xueyuan Yan. Tianjin Key Laboratory of Structure and Performance for Functional Molecules, College of Chemistry, Tianjin Normal University, Tianjin, China Guodong Ju, Haohao Bai, Mengyuan Liu, Xinping Xi, Yi Liu & Chao Wang Department of Chemistry, School of Science, Tianjin University, Tianjin, China Xueyuan Yan & Genping Huang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out most of the experiments. performed the DFT calculations. helped to separate and purify some target products. wrote the paper. designed the project and directed the work. All authors participated in the discussion and preparation of the paper. Correspondence to Genping Huang or Chao Wang. The authors declare no competing interests. Nature Chemistry thanks Zhihui Shao and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The reactions were performed with 1a (0.2 mmol) and 2a (0.4 mmol) in the presence of NiBr2•DME (10 mol%), ligand (13 mol%), LiOH (2.5 equiv. ), Me(EtO)2SiH (2.0 equiv.) in tBuOH (1.0 mL) at 40 °C for 24 h. The yield was determined by GC-MS. value was determined by HPLC. a, Calculated energy profile of the reaction. Spin states are indicated in superscript (CSS refers to the closed-shell singlet spin state). b, Optimized geometries and distortion/interaction analysis of selectivity-determining transition states. Energies and bond distances are given in kcal/mol and Å, respectively. c, Proposed mechanism. Supplementary protocols, discussions, figures, NMR spectra, HPLC traces and X-ray crystallographic data. The xyz coordinates of DFT calculations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Ju, G., Yan, X., Bai, H. et al. Stereodivergent construction of non-adjacent stereocentres via migratory functionalization of alkenes. Download citation Received: 09 September 2024 Accepted: 07 October 2025 Published: 12 November 2025 Version of record: 12 November 2025 DOI: https://doi.org/10.1038/s41557-025-01994-7 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemistry ISSN 1755-4349 (online) ISSN 1755-4330 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03679-0'>Chinese researchers reveal unexplored section of mysterious Arctic Ocean ridge</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 10:44:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Alexandra Witze writes for Nature from Boulder, Colorado. Chinese scientists have become the first to visit one of Earth's most remote and geologically intriguing realms: an underwater volcanic ridge in the Arctic Ocean. A team of scientists took a submersible vessel beneath the Arctic sea ice and completed more than 40 dives, going as deep as 5,277 metres. They provide scientists with some of the best opportunities to understand how life might arise and evolve in icy oceans on worlds beyond Earth, such as on Jupiter's ice-encrusted moon Europa. Yet, somehow that sluggish geology manages to produce enough heat and chemical energy to sustain hydrothermal vents on the sea floor. Researchers have returned multiple times to that area to study the rich chemistry and biology of the deep, dark vent system2,3. “It's so hard to get there that anything anybody does is almost guaranteed to be exciting and different and new,” says Christopher German, a marine geochemist at the Woods Hole Oceanographic Institution in Massachusetts. “The Chinese expedition is poised to yield significant new insights,” adds Elmar Albers, a marine geoscientist at the Alfred Wegener Institute in Bremerhaven, Germany. The Chinese expedition, organized by the Ministry of Natural Resources and the Chinese Academy of Sciences, paired a deep-sea crewed submersible, named Fendouzhe, with an ice-breaker. Does US science have a future in Antarctica? Trump cuts threaten to cancel fieldwork and more Controlling pyramidal nitrogen chirality by asymmetric organocatalysis Synthesis of enantioenriched atropisomers by biocatalytic deracemization The contribution of rock strength to soil production Single video camera tells the story of deadly Myanmar quake We are looking for a motivated and skilled Senior Scientist to join Antibody-Drug Conjugates (ADC) Target Discovery Team. We are looking for a Research Scientist within Orion Pharma's Immuno-Oncology target discovery team. We are seeking an accomplished Target Discovery Lead to join the Cancer Genomics Research Team. Principal investigator recruitment in infection and immunity at Institute of Human Immunology, SMART Shenzhen Medical Academy of Research and Translation Does US science have a future in Antarctica? Trump cuts threaten to cancel fieldwork and more An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41566-025-01788-w'>On-site enhancement and control of spin-forbidden dark excitons in a plasmonic heterostructure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 10:40:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Photonics (2025)Cite this article Dark excitons in atomically thin van der Waals materials provide an exciting platform for information transport and nanophotonic applications. Although dark excitons are difficult to access through free-space radiation, hybrid heterostructures incorporating plasmonic nanocavities provide a powerful platform to tailor their interactions with photons. Here we design a heterostructure consisting of optimized plasmonic nanocubes coupled to a WSe2 monolayer encapsulated between thin hexagonal boron nitride layers to unveil a new family of dark excitons. The emission from these dark excitons is 2,700 times stronger than bright excitons, yielding a striking enhancement factor of 3 × 105. We demonstrate the spin-forbidden nature of these dark states by studying their magneto-optical response. Furthermore, we selectively activate them by controlling the Fermi level via electric doping. Prominent features of these dark excitons include narrow linewidths, long lifetime, efficient electrical and magnetic modulation. Our findings unlock the potential for exploring exciton physics in two-dimensional materials using photonic heterostructures that preserve the intrinsic optical properties of two-dimensional materials in the coupling process. The demonstrated on-site control and ease of integration with passive photonic components make this platform particularly compelling for nanophotonic and sensing applications. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The main data supporting the findings of this study are available within the Article and its Supplementary Information. Additional data are available from the corresponding authors on request. Mak, K. F., Lee, C., Hone, J., Shan, J. & Heinz, T. F. Atomically thin MoS2: a new direct-gap semiconductor. Malic, E. et al. Dark excitons in transition metal dichalcogenides. Mai, C. et al. Many-body effects in valleytronics: direct measurement of valley lifetimes in single-layer MoS2. Nano Lett. Chand, S. B. et al. Interaction-driven transport of dark excitons in 2D semiconductors with phonon-mediated optical readout. Park, K. D., Jiang, T., Clark, G., Xu, X. & Raschke, M. B. Radiative control of dark excitons at room temperature by nano-optical antenna-tip Purcell effect. Lo, T. W. et al. Plasmonic nanocavity induced coupling and boost of dark excitons in monolayer WSe2 at room temperature. Nano Lett. Zhou, J. et al. Near-field coupling with a nanoimprinted probe for dark exciton nanoimaging in monolayer WSe2. Nano Lett. Hasz, K., Hu, Z., Park, K. D. & Raschke, M. B. Tip-enhanced dark exciton nanoimaging and local strain control in monolayer WSe2. Nano Lett. Rahaman, M. et al. Observation of room-temperature dark exciton emission in nanopatch-decorated monolayer WSe2 on metal substrate. Wen, X. et al. Pathways of exciton triggered hot-carrier injection at plasmonic metal–transition metal dichalcogenide interface. Shan, H. et al. Direct observation of ultrafast plasmonic hot electron transfer in the strong coupling regime. Light: Sci. Deng, M. et al. Light-controlled near-field energy transfer in plasmonic metasurface coupled MoS2 monolayer. Brongersma, M. L. Plasmon-induced hot carrier science and technology. Kang, Y. et al. Plasmonic hot electron induced structural phase transition in a MoS2 monolayer. Liu, H. et al. Spontaneous chemical functionalization via coordination of Au single atoms on monolayer MoS2. He, Z. et al. Quantum plasmonic control of trions in a picocavity with monolayer WS2. Shi, J. et al. Enhanced trion emission and carrier dynamics in monolayer WS2 coupled with plasmonic nanocavity. Li, Z. et al. Emerging photoluminescence from the dark-exciton phonon replica in monolayer WSe2. Robert, C. et al. Measurement of the spin-forbidden dark excitons in MoS2 and MoSe2 monolayers. Barbone, M. et al. Charge-tuneable biexciton complexes in monolayer WSe2. He, M. et al. Valley phonons and exciton complexes in a monolayer semiconductor. Koperski, M. et al. Single photon emitters in exfoliated WSe2 structures. Parto, K., Azzam, S. I., Banerjee, K. & Moody, G. Defect and strain engineering of monolayer WSe2 enables site-controlled single-photon emission up to 150 K. Nat. Akselrod, G. M. et al. Probing the mechanisms of large Purcell enhancement in plasmonic nanoantennas. Hoang, T. B. et al. Ultrafast spontaneous emission source using plasmonic nanoantennas. Huang, J., Akselrod, G. M., Ming, T., Kong, J. & Mikkelsen, M. H. Tailored emission spectrum of 2D semiconductors using plasmonic nanocavities. ACS Photonics 5, 552–558 (2018). Robert, C. et al. Fine structure and lifetime of dark excitons in transition metal dichalcogenide monolayers. Dang, J. et al. Identifying defect-related quantum emitters in monolayer WSe2. npj 2D Mater. Aivazian, G. et al. Magnetic control of valley pseudospin in monolayer WSe2. Srivastava, A. et al. Valley Zeeman effect in elementary optical excitations of monolayer WSe2. Li, Z. et al. Momentum-dark intervalley exciton in monolayer tungsten diselenide brightened via chiral phonon. ACS Nano 13, 14107–14113 (2019). Molas, M. R. et al. Probing and manipulating valley coherence of dark excitons in monolayer WSe2. Zhang, X. X. et al. Magnetic brightening and control of dark excitons in monolayer WSe2. Liu, E. et al. Gate tunable dark trions in monolayer WSe2. Li, Z. et al. Direct observation of gate-tunable dark trions in monolayer WSe2. Nano Lett. Wang, L. One-dimensional electrical contact to a two-dimensional material. Purdie, D. G. et al. Cleaning interfaces in layered materials heterostructures. This work was primarily supported by the Air Force Office of Scientific Research, the Office of Naval Research, the National Science Foundation and the Simons Foundation. were partially funded by the National Science Foundation via grant numbers NSF ECCS-2130552 and MRSEC DMR-2308817. gratefully acknowledges support from Welch Foundation chair F-0014. acknowledges partial support from the National Science Foundation (DMR-2044281) and from the Graduate Center of CUNY. acknowledge support from the JSPS KAKENHI (grant numbers 21H05233 and 23H02052) and World Premier International Research Center Initiative (WPI), MEXT, Japan. We gratefully acknowledge helpful discussions with E. Malic from Philipps-Universität Marburg and C. H. Lui from the University of California, Riverside. Present address: School of Physical Science and Engineering, Tongji University, Shanghai, China Photonics Initiative, Advanced Science Research Center, City University of New York, New York, NY, USA Jiamin Quan, Michele Cotrufo, Saroj Chand, Xuefeng Jiang, Enrique Mejia, Wei Wang, Gabriele Grosso & Andrea Alù Department of Physics, Complex Quantum Systems, and Texas Materials Institutes, The University of Texas at Austin, Austin, TX, USA Zhida Liu & Xiaoqin Li Research Center for Electronic and Optical Materials, National Institute for Materials Science, Tsukuba, Japan Takashi Taniguchi Research Center for Materials Nanoarchitectonics, National Institute for Materials Science, Tsukuba, Japan Kenji Watanabe Physics Program, The Graduate Center, The City University of New York, New York, NY, USA Gabriele Grosso & Andrea Alù Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the idea. led the optical experiments, with assistance from M.C., S.C., X.J., Z.L. led the numerical simulations. prepared the sample with contributions from M.C. provided the hBN sample. wrote the manuscript. supervised the project. All authors discussed the results. Correspondence to Xiaoqin Li or Andrea Alù. The authors declare no competing interests. Nature Photonics thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Quan, J., Cotrufo, M., Chand, S. et al. On-site enhancement and control of spin-forbidden dark excitons in a plasmonic heterostructure. Accepted: 26 September 2025 Published: 12 November 2025 Version of record: 12 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Photonics © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251112011841.htm'>Astronomers stunned by three Earth-sized planets orbiting two suns</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-12 09:24:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An international group of scientists has confirmed the discovery of three Earth-sized planets within the binary stellar system known as TOI-2267, located roughly 190 light-years from Earth. The finding, published in Astronomy & Astrophysics, offers new insight into how planets can form and remain stable in double-star systems, which were once thought too chaotic for complex planetary development. "This makes TOI-2267 the first binary system known to host transiting planets around both of its stars." Despite this, researchers have detected three Earth-sized planets in tight orbits, a surprising outcome that challenges long-held theories about where rocky worlds can exist. NASA's TESS space telescope first provided the data leading to this discovery. Two of the planets were initially identified by astronomers at ULiège and IAA-CSIC using their custom software tool, SHERLOCK. Designed to detect small exoplanets around cool, dim stars, these robotic instruments were vital for verifying the planets and studying their characteristics in detail. "It allows us to test the limits of planet formation models in complex environments and to better understand the diversity of possible planetary architectures in our galaxy." Pozuelos adds, "This system is a true natural laboratory for understanding how rocky planets can emerge and survive under extreme dynamical conditions, where we previously thought their stability would be compromised." The discovery opens up new questions about how planets form and persist in binary systems. Upcoming observations with the James Webb Space Telescope (JWST) and next-generation ground-based observatories could reveal more details, such as the planets' masses, densities, and even their atmospheric compositions. Beyond its scientific importance, the finding underscores the value of combining data from space-based observatories like TESS with precise ground-based instruments such as SPECULOOS and TRAPPIST. Together, they continue to expand our understanding of how planetary systems form and evolve across the galaxy. New Vitamin D Strategy Cuts Second Heart Attack Risk in Half Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            