
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-20</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250619090857.htm'>Invisible quantum waves forge shape-shifting super-materials in real time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 13:39:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This advance will enable researchers to incorporate desired mechanical properties into reconfigurable, solution-processible metamaterials, which have wide-ranging applications -- from shock absorption to devices that guide acoustic and optical energy in high-powered computer applications. Phonons are natural phenomena that can be thought of as discrete packets of energy waves that move through the building blocks of materials, whether they are atoms, particles or 3D-printed hinges, causing them to vibrate and transfer energy. "Using the liquid-phase electron microscopy technique developed in our lab at Illinois, the new study marks the first time we've been able to observe phonon dynamics in nanoparticle self-assemblies, acting as a new type of mechanical metamaterials," said Qian Chen, a professor of material science and engineering at the University of Illinois Urbana-Champaign. "This opens a new research area where nanoscale building blocks -- along with their intrinsic optical, electromagnetic, and chemical properties -- can be incorporated into mechanical metamaterials," Mao said, "Enabling emerging technologies in multiple fields from robotics and mechanical engineering to information technology." "This work also demonstrates the potential of machine learning to advance the study of complex particle systems, making it possible to observe their self-assembly pathways governed by complex dynamics," Pan said. "It opens new avenues for data-driven inverse design of reconfigurable colloidal metamaterials using machine learning and artificial intelligence." Materials provided by University of Illinois at Urbana-Champaign, News Bureau. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a65090477/magnetism-oxygen/'>Earth's Magnetic Field Might Weirdly Be Controlling the Air We Breathe, Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Magnetism and oxygen may intrinsically influence each other, but experts aren't sure why. We may earn commission if you buy from a link. If you list all of the things that had to go right to make life on Earth possible, our minuscule existence (cosmically speaking) seems all the more amazing. The Earth is situated in our Sun's habitable zone. Although scientists have known about these life-giving aspects of Earth for centuries, they're still discovering surprising connections between them. In a new study published last week in the journal Science Advances, scientists from NASA, the University of Washington, and the University of Leeds in the U.K. discovered a surprising connection between two of Earth's most important life-sustaining features—its oxygenated atmosphere and its magnetic field. After all, Mars (which is also in the Sun's habitable zone) used to have an atmosphere, but without a robust magnetic field, the Red Planet eventually lost that atmosphere to the unrelenting lashing of solar winds. However, the authors of this new paper found that, on Earth, the correlation between these two systems runs much deeper than we previously imagined. “We find that both exhibit strong linearly increasing trends, coupled with a large surge in magnitude between 330 and 220 million years ago,” the authors wrote. “Our findings suggest unexpected strong connections between the geophysical processes in Earth's deep interior, the surface redox budget, and biogeochemical cycling.” Analyzing data stretching back to the Cambrian some 541 million years ago, the researchers found that the rise in Earth's magnetic field and the rise in its oxygen levels were very closely aligned—slowly increasing overtime, except for one bout of increased activity lasting from 330 million to 220 million years ago. This would provide a clue, since a stronger, longer-lasting fire means that there was more oxygen in the atmosphere to fuel said fire. To compare this record with Earth's magnetic field history, the team analyzed certain magnetic crystals that formed in ancient volcanoes and—due to their composition—essentially act like a “compass frozen in time,” according to Nature. Well, the researchers aren't exactly sure, but they have a few guesses. The most likely one is that Earth's magnetic field directly impacts oxygen levels, as it protects Earth (and oxygen-producing plants) from solar radiation. However, it's also possible that increased oxygenation coupled with plate tectonics—which drive oxygen toward the liquid outer core that produces the magnetic field—could also play a role. The authors also aren't ruling out the idea that a third, currently unknown mechanism could provide an explanation for this steadily upward trend. We're trying to fit all of it together and see what's the big picture.” While 540 million years is an unfathomably long time compared to our human lifespan (or even our species' existence), it's only around 12 percent of Earth's entire existence, so these trends could simply be coincidental. All we can do is continue searching for answers among the clues that we do have, and try to grasp just how wondrous our home planet really is. All Life on Earth Follows This One Simple Rule This Clock Will Skirt the Laws of Thermodynamics</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65102187/axolotl-limb-regeneration-breakthrough/'>Humans Already Have the Ingredients to Regrow Limbs, Scientists Find</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Axolotls can do that without even trying—in fact, it is possible for an axolotl to lose virtually any body part (even its brain and internal organs) and fully regenerate it. Mysteries surrounding these virally adorable amphibians' regenerative powers have fueled the sci-fi dreams (and frustrations) of scientists for almost two centuries, but not until recently did anyone understand the mechanism behind this ability. Positional memory means that an axolotl somehow knows if it needs to grow back a lost finger, hand, or entire arm. This was already known to be the basic mechanism behind vertebrate regeneration, but what Monaghan found was that it begins with retinoic acid and the enzyme CYP26B1. It is just a matter of axolotls being able to use them differently. Larger limbs at proximal sites closer to the body, such as arms, contain more retinoic acid and less CYP26B1 (which breaks the retinoic acid down). “Regenerating limbs retain their proximodistal (PD) positional identity following amputation. This positional identity is genetically encoded by PD patterning genes that instruct blastema cells to regenerate the appropriate PD limb segment,” Monaghan and his team said in a study recently published in the journal Nature Communications. When a predator bites an arm (or anything else) off of an axolotl, retinoic acid is synthesized in the middle layer of skin and spreads to the limb bud. Blastema mirror the behaviors of limb buds that grow as an embryo develops, and in both embryos and adult axolotls that have been injured, positional information is exchanged between stem cells in the blastema and other cells in this budding limb to ensure that the appropriate tissues regenerate where they are supposed to grow. The gene Hoxa13 activates CYP26B1, which breaks down retinoic acid where it is not needed and uses it to create a pattern for the limb being regenerated. This breakdown determines how much retinoic acid is at an amputation site and, consequently, the position and structure of the limb which is regrown. Higher levels of retinoic acid activate the Shox gene—a transcription factor that both gives directions for producing a protein that regulates what other genes do and is involved with forming the skeleton. Some other animals that regenerate, like lizards, may grow back the end of a missing tail, but in a simpler form than the original. Healing wounds without scarring might even be possible in the near future. What we need to do next to make that future a reality is find out what happens inside blastema cells during regeneration, and which parts of these cells are targeted by retinoic acid. “If we can find ways of making our fibroblasts listen to these regenerative cues, then they'll do the rest,” Monaghan said in a recent press release. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. One Single Ancestor Gave Rise to All Life on Earth Researchers Have a New Way to Detect Prime Numbers</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250620030326.htm'>Self-esteem skyrockets 131% after weight-loss surgery, study reveals</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 12:58:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Self-esteem scores more than doubled within one year of weight-loss surgery, according to a new study* presented today at the American Society for Metabolic and Bariatric Surgery (ASMBS) 2025 Annual Scientific Meeting. "Understanding weight stigma and psychosocial factors associated with obesity is essential to offering holistic care. While these factors should not dictate the decision to have bariatric surgery, they should be an important part of the conversation," said study co-author Justin Dhyani, MD, Geisinger Medical Center in Danville, PA. Weight stigma is associated with adverse health outcomes including depression, anxiety, disordered eating, and low self-esteem. "Weight stigma is a serious issue that places an extra psychological burden on patients struggling with obesity and there is no excuse for it," said Ann M. Rogers, MD, MD, FACS, FASMBS, President, ASMBS, who was not involved in the study. Materials provided by American Society for Metabolic and Bariatric Surgery. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250620030423.htm'>Fitness trackers are failing millions — this fix could change everything</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 12:33:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But for those living with obesity, who are known to exhibit differences in walking gait, speed, energy burned and more, these devices often inaccurately measure activity -- until now. But for those living with obesity, who are known to exhibit differences in walking gait, speed, energy burned and more, these devices often inaccurately measure activity -- until now. Scientists at Northwestern University have developed a new algorithm that enables smartwatches to more accurately monitor the calories burned by people with obesity during various physical activities. It is transparent, rigorously testable and ready for other researchers to build upon. Their next step is to deploy an activity-monitoring app later this year that will be available for both iOS and Android use. "People with obesity could gain major health insights from activity trackers, but most current devices miss the mark," said Alshurafa, associate professor of behavioral medicine at Northwestern University Feinberg School of Medicine. Current activity-monitoring algorithms that fitness trackers use were built for people without obesity. Hip-worn trackers often misread energy burn because of gait changes and device tilt in people with higher body weight, Alshurafa said. "Without a validated algorithm for wrist devices, we're still in the dark about exactly how much activity and energy people with obesity really get each day -- slowing our ability to tailor interventions and improve health outcomes," said Alshurafa, whose team tested his lab's algorithm against 11 state-of-the-art algorithms designed by researchers using research-grade devices and used wearable cameras to catch every moment when wrist sensors missed the mark on calorie burn. Alshurafa was motivated to create the algorithm after attending an exercise class with his mother-in-law who has obesity. "She worked harder than anyone else, yet when we glanced at the leaderboard, her numbers barely registered," Alshurafa said. "That moment hit me: fitness shouldn't feel like a trap for the people who need it most." By using data from commercial fitness trackers, the new model rivals gold-standard methods of measuring energy burn and can estimate how much energy someone with obesity is using every minute, achieving over 95% accuracy in real-world situations. This advancement makes it easier for more people with obesity to track their daily activities and energy use, Alshurafa said. The study participants went through a set of physical activities to measure their energy burn during each task. In another group, 25 study participants wore a fitness tracker and body camera while just living their lives. "Many couldn't drop to the floor, but each one crushed wall-pushups, their arms shaking with effort," he said, "We celebrate 'standard' workouts as the ultimate test, but those standards leave out so many people. These experiences showed me we must rethink how gyms, trackers and exercise programs measure success -- so no one's hard work goes unseen." The study is titled, "Developing and comparing a new BMI inclusive energy burn algorithm on wrist-worn wearables." This work also was done in collaboration with Mahdi Pedram and Whitney A. Morelli, formerly of Northwestern. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65092695/swedish-lords-prayer-wawa-runestone/'>Archaeologists Found a Slab in the Middle of Nowhere—With the Lord's Prayer Carved in It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“Well, it's certainly among the least expected finds that I think I've encountered during my career,” Primrose told the CBC. Initial research, according to an OCARE statement, showed that the carving was likely written in Futhark characters—a runic script once used in northern Europe and Scandinavia. This led some experts to think that the carving must have been completed as far back as the Viking era, especially considering the second carving of a boat (which resembles a Viking longboat, contains about 16 occupants, and is flanked by several crosses or stars) found adjacent to the script. Williams said that the runic writing was a version of the Christian Lord's Prayer, which had been carved in Futhark. “They are really deeply carved into the rock. Someone must have spent a couple of weeks carving this thing.” Whether this was a popular religious site—the inscription was found under soil after the tree fell, and no other artifacts have been found nearby—or the work of a single person toiling alone is still a question. But with this announcement, many other questions have been answered. “Canada now has a total of 11 objects claimed to bear runes but only five in fact do so, and three of those constitute modern commemorative inscriptions,” Williams wrote in an OCARE report. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Found a Coffin Depicting an Epic Battle The ‘Holy Grail of Shipwrecks' Is Still Underwater An Ancient Coin May Mark the Rise of Christianity Researchers Found an Odd Remnant of Farm History</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-60204-7'>Iron deficiency causes aspartate-sensitive dysfunction in CD8+ T cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 11:11:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Iron is an irreplaceable co-factor for metabolism. Nevertheless, how iron deprivation impacts immune cell function remains poorly characterised. We interrogate how physiologically low iron availability affects CD8+ T cell metabolism and function, using multi-omic and metabolic labelling approaches. Iron limitation does not substantially alter initial post-activation increases in cell size and CD25 upregulation. However, low iron profoundly stalls proliferation (without influencing cell viability), alters histone methylation status, gene expression, and disrupts mitochondrial membrane potential. Glucose and glutamine metabolism in the TCA cycle is limited and partially reverses to a reductive trajectory. Previous studies identified mitochondria-derived aspartate as crucial for proliferation of transformed cells. Despite aberrant TCA cycling, aspartate is increased in stalled iron deficient CD8+ T cells but is not utilised for nucleotide synthesis, likely due to trapping within depolarised mitochondria. Exogenous aspartate markedly rescues expansion and some functions of severely iron-deficient CD8+ T cells. Overall, iron scarcity creates a mitochondrial-located metabolic bottleneck, which is bypassed by supplying inhibited biochemical processes with aspartate. These findings reveal molecular consequences of iron deficiency for CD8+ T cell function, providing mechanistic insight into the basis for immune impairment during iron deficiency. Studies of metabolism in cancer and in different cell lineages have changed understanding of how metabolic processes regulate cellular transformation and cell fate1,2. In contrast, how common nutritional deficiencies influence metabolic pathways and alter cell biology is relatively poorly studied. Iron deficiency is the most common micronutrient deficiency worldwide3, inhibiting erythropoiesis, impairing cognitive development and disabling immunity, and is a comorbidity for several disorders, including heart failure4,5,6. Key conserved biochemical processes are iron-dependent, such as the electron transport chain (ETC), tricarboxylic acid (TCA) cycle, nucleotide synthesis, DNA repair, histone and DNA demethylation and oxygen sensing6,7. Cells typically obtain iron by receptor-mediated internalisation of transferrin, a dedicated iron-chaperone protein in plasma and extracellular fluid8. However, the amount of iron bound to transferrin shows marked physiological variation. For example, in severe iron deficiency or in the presence of inflammation, plasma iron concentrations can drop by ~90%9,10, drastically decreasing iron supply to cells. Notably, human genetic studies and pre-clinical work link decreased transferrin-iron acquisition by proliferating lymphocytes to impaired adaptive immunity5,11. Patients homozygous for a mutation in Tfrc (the gene encoding the transferrin receptor, TFR1, also known as CD71) that inhibits cellular iron uptake, have a severe combined immunodeficiency characterised by reduced lymphocyte function and suppressed antibody titres11. Similarly, low serum iron availability suppresses B and T cell responses in mice following vaccination and influenza infection5. How iron deficiency mechanistically impairs adaptive immune cells remains unclear, but understanding this issue could inform how iron regulates immunity and provide further rationale for correcting iron deficiency within human populations. Recent information summarising how iron influenced the immune system has been reviewed here12. T cells dramatically remodel their metabolism post-activation, including upregulation of glycolysis and oxidative phosphorylation (OXPHOS), in order to meet the increased energetic and biosynthetic requirements of proliferation and effector function13. Transition from naïve to activated T cell is accompanied by greatly increased expression of the iron uptake protein, TFR1 and CD8+ T cell iron content is calculated to triple within the first 24 h post-activation14. In this study, we investigate how low iron supply influences activated CD8+ T cells via ‘omics approaches and isotope tracing. We find mitochondrial defects, dysregulated metabolic processes and stalled cell division, but show that a single amino acid, aspartate, rescues multiple aspects of cell dysfunction, including proliferation. These results offer insights into the molecular effects of iron deficiency within the immune system. Iron is utilised by ~400 proteins, of which 204 are described to be expressed in T cells7,14,15. Iron-interacting proteins operate in a wide diversity of pathways ranging from mitochondrial metabolism to DNA synthesis6,7, suggesting that iron deficiency likely has important and wide-acting effects on T cell biochemistry. We therefore took an unbiased approach to initially define the impact of iron limitation on global CD8+ T cell transcription and translation. We employed RNA-sequencing (RNA-seq) and quantitative protein-mass spectrometry (protein-MS) of mouse OT-I CD8+ T cells activated in vitro across physiologically-titrated transferrin-iron concentrations (0.001–0.625 mg/mL holotransferrin, with total transferrin kept constant at 1.2 mg/mL) for 48 h (Fig. Meanwhile, 0.001 mg/mL holotransferrin is much lower than levels found in plasma but may be reflective of the almost negligible levels of iron detected in lymphatic fluid17. a CD8+ OT-I T cells isolated from mice were activated with 5 µg/mL plate-bound α-CD3, 1 µg/mL α-CD28 and 50 U/mL IL-2 for 48 h in a titration of holotransferrin conditions. Naïve T cells were collected on day 0. Where comparisons between high and low iron conditions are made, the holotransferrin concentrations used are 0.625 (high) and 0.001 (low) mg/mL. b, c Division assessed using cell trace violet (CTV), n = 5. d CD25, e CD44, f TFR1/CD71, g CD98, h IL-2, i TNF and j IFN-γ mean fluorescence intensity (MFI), n = 4 except for (g) where n = 5. k Volcano plot with the significance thresholds of log2|fold change (FC)| > 0.585, p value < 0.05, n = 4. No correction for multiple testing was applied. l Correlation plot of the log2|FC| between high and low iron conditions by RNA-seq and protein-mass spectrometry (protein-MS), n = 4. m Hallmark gene set enrichment analysis (GSEA) for the protein-MS, n = 4. A Benjamini-Hochberg adjustment was used to correct for multiple testing. n p53 induces CDKN1A expression which inhibits the G1-S phase transition. Data are mean ± standard error of the mean (SEM), where each datapoint per condition denotes cells from independent donor mice. Statistics are: (b, d–j, o–q) sampled matched one-way ANOVAs with the Geisser-Greenhouse correction; l two-sided Pearson correlation R2 value. In line with previous work showing that in vivo iron deprivation significantly reduces T cell expansion5, in vitro iron limitation profoundly suppressed cellular proliferation, measured using cell trace violet (CTV; Fig. Iron deficiency also impaired expression of the activation marker, CD25, while having no effect on CD44 (Fig. 1f) consistent with iron limitation18 while the amino acid transporter, CD98 (SLC3A2:SLC7A5 heterodimer; LAT1), which is induced post-T cell activation19, was significantly decreased in low iron conditions (Fig. In line with previous reports20,21, the cytokine, IL-2, was induced during iron deprivation (Fig. TNF was similarly induced with iron scarcity (Fig. Therefore, in low iron conditions, stimulated OT-I T cells acquire some though not all characteristics of activation on a per-cell basis, and because of a failure to proliferate, overall, there are fewer competent effector T cells when iron is restricted. Notably, the impaired proliferation and altered effector function observed under low iron conditions are not due to a failure to activate. Cells cultured in both low iron and iron-replete conditions show robust upregulation of CD25 expression and increased cell size (measured as forward scatter) relative to naïve cells maintained in IL-7 (Supplementary Fig. Further, cells in low iron display a transcriptional profile distinct from naïve cells, which is substantially more similar to cells in iron-replete conditions (Supplementary Fig. Despite observing dramatic changes in cellular division during iron restriction, only 193 genes were identified as significantly differentially regulated between iron-replete and deficient conditions by RNA-seq (Supplementary Fig. However, clear segregation was still observed between the two iron concentrations by principal component analysis (PCA; Supplementary Fig. Genes identified by RNA-seq as induced during iron deficiency also displayed an enrichment of the activating histone marks, H3K4me3 and H3K27ac at transcription start sites (TSSs) in the low iron condition (e.g. Cdkn1a; Supplementary Fig. Meanwhile, genes suppressed by iron deficiency at the RNA level showed depletion of these activating marks around their TSSs (e.g. Asns). The positive relationship between the transcriptionally associated histone modifications, H3K4me3 and H3K27ac, with transcription suggests that RNA expression changes observed during iron deficiency are likely due to transcriptional differences rather than alterations in RNA stability. These data are consistent with the canonical post-transcriptional upregulation of Tfrc via increased mRNA stability under iron limitation by the iron-response element (IRE)/iron-response protein (IRP) system18. Protein-MS analysis identified 116 differentially expressed proteins (Fig. Despite the differential expression of 116 proteins, no differences were observed in overall protein mass or protein molecules per CD8+ T cell activated across a titration of holotransferrin conditions (Supplementary Fig. 1i, j), indicating that despite decreased cell division, iron-deficient CD8+ T cells do not accumulate extra protein. 5091 mRNAs/proteins were mutually detected by the RNA-seq and protein-MS with a significant positive correlation in the log2|FC| values (Fig. Of these, only 15 were significantly differentially regulated at both the mRNA and protein levels (Supplementary Fig. These modest expression changes indicate changes in protein function rather than expression may be the predominant driver of the profound suppressive effect of iron deficiency on CD8+ T cell proliferation. Of note, 7 of the 15 mutually differentially expressed genes encoded proteins related to metabolism, including amino acid transporters (Slc1a4, Slc7a3) or enzymes (Dglucy, Ephx1, Asns, Cth, Pck2), indicating changes in cellular metabolism may underpin this functional alteration. In addition, gene set enrichment analysis (GSEA), conducted on both RNA-seq and protein-MS datasets, also indicated altered expression of genes attributed to the p53 pathway (Fig. The p53 pathway coordinates responses to cellular stress such as DNA damage, hypoxia and nutrient deprivation, with downstream effects including apoptosis, DNA repair and cell cycle arrest22. p53 has also been shown to regulate cellular metabolism with effects including glycolytic suppression and mitochondrial maintenance23. Altered activity of genes involved in the p53 pathway may therefore also contribute to the titratable impairment of cellular proliferation in low iron conditions (Fig. Consistent with this, expression of Cdkn1a (also known as p21, Cip1 and Waf1), a p53 target and suppressor of the G1-S phase cell cycle transition was induced by iron deprivation at 24 h, prior to the first cell division and continued to be upregulated as CD8+ T cells entered their proliferative phase at 48 h (Fig. However, while cell division halted, cells remained viable at 48 h (Fig. Notably, Cdkn1a is induced by iron chelation in cell lines24. This data suggests that induction of Cdkn1a expression, potentially via the p53 signalling pathway, may contribute to drive cell cycle suppression, but iron deprivation does not cause cell death at this time point. p53 induction during iron chelation has been described in cell lines, macrophages and CD4+ T cells25,26,27,28 and is understood to occur via HIF1α stabilisation due to impaired activity of their regulators, the iron and oxygen-dependent PHD proteins28. We therefore tested whether HIF1α was similarly stabilised in cells deprived of iron more physiologically, via decreased availability of transferrin-bound iron. While the iron chelator deferiprone (DFP) clearly induced HIF1α stabilisation in CD8+ T cells, low iron conditions did not alter HIF1α levels relative to iron-replete controls (Supplementary Fig. Moreover, the proteomic profile of iron-deficient CD8+ T cells bears little resemblance to the proteomes of 5 day activated cytotoxic CD8+ T cells exposed to hypoxia for 24 h as reported by Ross et al.29 (Supplementary Fig. Both hypoxia and iron chelation have also been shown to induce glycolytic genes25,29, but this was not observed in our cells in low iron media. Given the discrepancy between how HIF1α responded in CD8+ T cells under iron chelation and iron deficiency, we were interested to understand whether the difference in cellular response extended beyond HIF1α. We processed raw RNA-seq data from Wang et al.30 (not analysed in their manuscript) of Th1 polarised CD4+ T cells treated with CPX iron chelator or dimethyl sulfoxide control for 4 h. CPX-treated cells showed differential expression of 3523 genes, which is ~18-fold higher than the 193 differentially expressed genes identified in our model of low iron media (Supplementary Fig. CPX also failed to upregulate Tfrc (Supplementary Fig. 2e), suggesting that the IRP-IRE iron signalling system is no longer intact. While CPX iron chelation suppressed pathways associated with cell cycle (G2M checkpoint) and metabolic reprogramming (MYC targets; Supplementary Fig. 2f), similar to iron deficiency, only 45 genes were found to be mutually differentially expressed (Supplementary Fig. The extreme changes in transcriptional profile observed with only 4 h of CPX treatment relative to our 48-h culture model in low iron media and the failure to upregulate Tfrc suggest that the biochemical impacts of chelators can be non-physiological, especially if given at high doses. We observed decreased activity of the key upstream metabolic regulator, mTOR, measured as phosphorylation of its target S6 (pS6; Fig. 1g), and may also relate to lack of total protein accumulation in non-proliferating iron-deficient cells (Supplementary Fig. However, unexpectedly, we observed upregulation of proteins involved in mitochondrial processes, including OXPHOS and fatty acid metabolism (Fig. The enrichment of the fatty acid signature was driven largely by increases in β-oxidation proteins involved in fatty acid and branched chain amino acid breakdown rather than fatty acid synthesis (Fig. ETC proteins involved in the highly iron-dependent respiratory complexes I (CI), CIII and CIV were also upregulated. Consistent with increased abundance of these mitochondrial proteins, mitochondrial mass, measured with Mitotracker green (MTG), was also elevated in low iron conditions (Fig. Of note, by selecting for mitochondrially localised proteins using the MitoCarta3.0 gene set31, we observed improved segregation within the first two principal components of CD8+ T cells cultured in different iron concentrations compared to when all proteins were analysed (Fig. 2f, g), indicating mitochondrial proteins are disproportionately influenced by iron availability relative to all proteins detected. CD8+ T cells were activated as described in Fig. a mTORC1 and MYC are metabolic regulators that enable biosynthesis downstream of TCR stimulation. b, c mTORC1 activity measured via its downstream target, phospho-S6 (pS6), n = 4. Controls were treated overnight with rapamycin (rapa; 1 µM). d Heatmap of proteins in selected metabolic pathways defined using GO terms where the p value < 0.05, n = 4. e Mitochondrial membrane mass measured using Mitotracker green (MTG) MFI, n = 4. PCA of the protein-MS f given prior selection for proteins in the MitoCarta3.0 gene set and g of all proteins, n = 4. h, i mROS MFI measured using MitoSOX red, n = 4. j Mitochondrial membrane potential calculated as the ratio of Mitotracker deep red (MTDR) to Mitotracker green (MTG), n = 4. k SOD2 protein expression by protein-MS, n = 4. Data are mean ± SEM, where each datapoint per condition denotes cells from independent donor mice. Statistics are: b, h, k matched one-way ANOVAs with the Geisser-Greenhouse correction; e, j matched two-tailed t-test. Mitochondria are enriched for iron-interacting proteins, with 7% of mitochondrial proteins classified as iron-interacting compared to 2% cell-wide7. Aconitase 2 (ACO2) and SDH of the TCA cycle and CI–CIV of the ETC require iron cofactors for function32. Mitochondria are also home to haem and iron–sulphur (Fe–-S) cluster synthesis pathways32. Analysis of proteomics data from Howden et al.15 revealed upregulation of haem and Fe–S cluster synthesis proteins in T cells 24 h and 6 days post-activation (Supplementary Fig. The heavy reliance of mitochondrial function on iron in combination with the upregulation of mitochondrial proteins suggests that mitochondrial function may be disrupted during iron scarcity. To directly interrogate CD8+ T cell mitochondrial function under iron deprivation, we first assessed a metric of mitochondrial health, specifically levels of superoxide (O2•−)—a key exemplar of mROS species. As available iron declined, mROS levels increased in CD8+ T cells (Fig. CI, CII and CIII of the ETC, which all require iron for electron transfer32 are key mROS producers33. Given the observed increase in ETC proteins (Fig. 2d), but lower availability of iron during iron restriction, we propose an imbalance in ETC proteins to iron cofactors may impair efficient electron transfer, resulting in increased mROS generation. Consistent with this, iron deprivation decreased the mitochondrial membrane potential (Fig. 4a) and was previously shown to suppress mitochondrial ATP generation5. Upregulation of the mitochondrial superoxide detoxifying protein, SOD2, was also observed under iron scarcity (Fig. 2k), indicating compensatory mechanisms to suppress mROS, which at the lowest iron concentrations still fail to control them. Taken together with the alterations in the mitochondrial proteome, these data indicate that iron-deprived CD8+ T cells may have defective mitochondrial metabolism. To directly assess this, we next performed stable isotope-based tracing coupled to metabolite-mass spectrometry (metabolite-MS). CD8+ T cells were activated for 24 h and then incubated with 13C6-glucose or 13C5-glutamine for a further 24 h under iron-replete or -deficient conditions. Here, minimal changes in overall abundance of the glycolytic metabolites, pyruvate and lactate, or the amino acids which may be derived from glycolytic intermediates or imported (alanine, serine and glycine) were observed (Fig. Furthermore, there were no significant changes in the abundance of total lactate or the fraction of lactate labelled from 13C6-glucose in the supernatants of iron-deprived CD8+ T cells (Supplementary Fig. 4b, c) together suggesting glycolytic activity was preserved under low iron conditions. This is consistent with minimal changes in HIF1α activity (Supplementary Fig. 2a–c) and in agreement with glycolysis being an iron-independent pathway (none of the enzymes involved in glycolysis require iron). CD8+ T cells were activated as described in Fig. For tracing experiments, T cells were activated in standard media for 24 h and then incubated in media containing 13C6-glucose or 13C5-glutamine for a further 24 h. a Relative metabolite abundance from T cells cultured in low iron (0.001 mg/mL holotransferrin) versus high iron (0.625 mg/mL holotransferrin), normalised to spiked in glutaric acid. Pooled relative total abundances from the 13C6-glucose and 13C5-glutamine experiments, n = 8. P values are shown in red. Relative abundance of labelled and unlabelled metabolites calculated as the fraction labelled multiplied by the raw glutaric acid normalised abundance. Blue-filled circles indicate carbon atoms labelled from 13C5-glutamine. Empty circles indicate unlabelled carbon atoms. Relative abundance of c α-KG, d succinate, e fumarate and f malate mass isotopomers from 13C-glutamine tracing calculated as the fraction labelled multiplied by the raw glutaric acid normalised abundance, n = 4. g NAD+/NADH ratio. Data from independent experiments denoted by different symbols, n = 13. Data are mean ± SEM, where each datapoint per condition denotes cells from independent donor mice. Statistics are: a matched two-way ANOVA with the Geisser-Greenhouse correction and the Fisher's least significant difference (LSD) test for multiple comparisons; b–f matched two-way ANOVAs with the Geisser-Greenhouse correction and the Sidak correction for multiple comparisons; g matched two-tailed t-test. In these experiments we also noted that while ~20% labelling of 13C6-glucose into the TCA cycle was observed, 13C5-glutamine was much more readily incorporated into the TCA cycle, labelling 60-80% of TCA cycle metabolites (Supplementary Fig. The larger incorporation of glutamine relative to glucose into the TCA cycle is consistent with reports indicating substantial reduction of glucose-derived pyruvate to lactate in activated T cells, alongside increased glutamine anaplerosis into the TCA cycle34. ACO2 is an iron-dependent enzyme in the TCA cycle, which converts citrate to isocitrate. In agreement with reduced TCA cycle activity, decreased abundance of α-KG was observed in low iron cells (Fig. Decreased α-KG M + 3 isotopomers from 13C5-glutamine and a trend towards reduced α-KG M + 2 isotopomers from 13C6-glucose support reduced ACO2 and/or IDH2 activity (Fig. However, since α-KG is also produced from glutamate by the activity of glutamate dehydrogenase (GDH), decreased α-KG may also reflect decreased activity of this enzyme. Indeed, decreased relative abundance of α-KG M + 5 mass isotopomers from 13C5-glutamine indicates a trend towards decreased GDH activity under low iron conditions (Fig. Consistent with this, 13C-labelled glutamate accumulated in low iron conditions following 13C5-glutamine tracing (Fig. The TCA cycle metabolite succinate lies upstream of the second iron-dependent TCA cycle enzyme (SDH). Consistently, we observed that while succinate abundance was depressed in iron limiting conditions (−23%; Fig. 3a, b), it was relatively accumulated compared to the downstream metabolites, fumarate (−47%) and malate (−41%), suggesting suppressed progression from succinate to fumarate (Fig. Decreased M + 4 13C-labelling from 13C5-glutamine into fumarate and malate also supports this (Fig. Decreased oxidative TCA cycle progression, due to limited glutamine anaplerosis and ACO2 and SDH activity, agrees with diminished NAD+ reduction and consequent increases in the NAD+/NADH ratio (Fig. Since NADH is a positive allosteric regulator of GDH, this may also partly explain the indicated decrease in GDH activity35. α-KG levels were decreased by ~40% in iron-deficient CD8+ T cells (Figs. α-KG is a substrate for many dioxygenases, including the histone lysine demethylases (KDMs)36 (Fig. 4b), and alterations in T cell metabolism have been shown to alter KDM activity and T cell fate37. Crucially, most KDMs require an iron catalytic core to mediate hydroxylation of histone methyl groups using α-KG and oxygen as substrates36. The unstable methyl-hydroxy intermediate spontaneously dissociates, leaving a demethylated product38,39. This dual dependency of KDMs for α-KG and iron means that a decrease of α-KG availability under iron limitation may exert a double hit on KDM activity, with potential impacts on the appropriate chromatin restructuring necessary for CD8+ T cell differentiation. The abundance of KDMs substantially increases upon T cell activation, with KDM6B being the second most upregulated KDM in CD8+ T cells14,15. KDM6B is responsible for the removal of the repressive histone mark, H3K27me3, a process critical for T cell differentiation and effector function acquisition40,41. Consistent with decreased activity under iron limitation, CD8+ T cells demonstrated a titratable failure to remove H3K27me3 relative to iron-replete controls, with H3K27me3 levels remaining almost as high as IL-7-treated, naïve-like cells (Fig. Using ChIPmentation, we confirmed that H3K27me3 levels were significantly elevated at TSSs in iron-deficient CD8+ T cells relative to iron-replete (Fig. The genome-wide increase in H3K27me3 during iron deficiency agrees with the concept that iron deprivation generally impairs the activity of KDM6 enzymes during T cell activation. It is also possible that increased methylation by methylases could contribute to the H3K27me3 accumulation phenotype. These findings therefore indicate that altered CD8+ T cell metabolic activity under low iron may have direct implications for their epigenetic status and differentiation capacity. While we hypothesise that KDM6 impairment occurs due to iron deficiency coupled to suppressed α-KG production, KDM6 activity may be impaired by other mechanisms. For instance, ROS, which are induced by iron deficiency (Fig. 2h), have been shown to inhibit other iron-dependent, oxygen-dependent and α-KG-dependent dioxygenases such as the PHD proteins, which regulate HIF1α42. Of note, however, direct supplementation with cell-permeable dimethyl-α-KG failed to rescue H3K27me3 levels and cellular proliferation (Supplementary Fig. This is consistent with KDM enzymes requiring all of iron, oxygen and α-KG. Thus, even if α-KG is no longer limiting, KDM enzymes will remain dysfunctional in the absence of sufficient iron cofactors. CD8+ T cells were activated as described in Fig. Data from independent experiments denoted by different symbols, n = 8. b Lysine demethylase (KDM) enzymes use iron cofactors and α-KG and oxygen substrates to mediate the hydroxylation of methyl groups, which spontaneously dissociate to leave a demethylated histone. KDM6 enzymes remove the repressive histone mark, H3K27me3, from effector gene loci upon T cell activation. Naïve-like control cells were cultured in IL-7 (5 ng/mL), n = 4. e, f H3K27me3 enrichment at transcription start sites (TSSs) of all genes assessed by ChIPmentation, n = 3. Data are mean ± SEM, where each datapoint per condition denotes cells from independent donor mice. Histograms are normalised to the mode. Statistics are: a paired two-tailed t-test; c matched one-way ANOVA with the Geisser-Greenhouse correction. Aspartate is produced downstream of the TCA cycle metabolite oxaloacetate and critically supports cellular proliferation through de novo purine and pyrimidine synthesis43,44,45. In cancer cells, ETC inhibition or iron chelation impairs aspartate synthesis, resulting in suppressed proliferation of these immortalised cells46,47,48. However, despite TCA cycle inhibition, aspartate was unexpectedly higher in iron-deficient cells, with the majority derived from glutamine (M + 4 labelled; Figs. CD8+ T cells were activated as described in Fig. For the 13C6-glucose tracing experiments, T cells were activated for 24 h and then incubated in media containing 13C6-glucose for a further 24 h. a Relative abundance of aspartate normalised to a glutaric acid spike in. Data from independent experiments denoted by different symbols, n = 8. b Aspartate is incorporated into purine and pyrimidine nucleotides. Relative abundance of c AICAR, n = 4. AICAR was normalised to spike in 15N-dT. d PPAT protein via protein-MS, n = 4. Relative abundance of e carbamoyl-aspartate and f orotate, n = 4. Carbamoyl-aspartate and orotate were normalised to be spiked in glutaric acid. Orange and green circles indicate 13C-labelled atoms. Orange circles show labelling expected from oxidative TCA cycling via PDH. Green circles indicate labelling from reductive TCA cycling via PC. PC activity measured via the h fractional labelling into heavy labelled metabolites expected from reductive TCA cycling and via i the ratio of (Malate M3−Succinate M3)/Pyruvate M3, n = 4. j PC protein expression via protein-MS, n = 4. k PCK2 protein expression via protein-MS, n = 4. l Division measured using cell trace violet (CTV) with or without pyruvate (10 mM), n = 4. Data are mean ± SEM, where each datapoint per condition denotes cells from independent donor mice. Statistics are: a, c, e, f, i paired two-tailed t-tests; d, j, k one-way ANOVAs with the Geisser-Greenhouse correction; h matched two-way ANOVA with the Sidak correction for multiple comparisons; l matched two-way ANOVA with the Geisser-Greenhouse correction. To understand whether nucleotide synthesis was altered downstream of decreased TCA activity in iron-depleted CD8+ T cells, we measured the abundance of nucleotides and their upstream precursors by LC-MS, observing that AICAR, a metabolite which lies two steps downstream of aspartate incorporation into purine synthesis49 was substantially decreased during iron limitation (Fig. PPAT, the initiating enzyme of purine synthesis, which has been predicted to be iron-dependent7 was also reduced in iron scarcity (Fig. Carbamoyl-aspartate and orotate, which lie downstream of aspartate incorporation into pyrimidine synthesis, were similarly depleted (Fig. Consistent with these observations, certain nucleotides and dNTPs were decreased during iron deficiency (Supplementary Fig. 6b–d), albeit to a lesser extent, which may be explained by their decreased usage under suppressed CD8 + T cell proliferation and associated DNA synthesis (Fig. Together, these data indicate that nucleotide synthesis from aspartate is limited in iron-deficient CD8+ T cells, despite overall cellular abundance of aspartate being increased rather than decreased (Figs. This aspartate accumulation could be somewhat explained by increased production by an alternative pathway, for instance, pyruvate anaplerosis to oxaloacetate via pyruvate carboxylase (PC) and a partial reversal of TCA cycling (Fig. Carbon labelling from 13C6-glucose through the oxidative (canonical forward) TCA cycle via pyruvate dehydrogenase (PDH) results in entry of two 13C atoms into TCA cycle metabolites (M + 2 mass isotopomers)50. Meanwhile, entry of carbon from 13C6-glucose to the reductive (reverse) TCA cycle via PC produces M + 3 mass isotopomers. During iron deficiency, significant increases in 13C6-glucose labelling into M + 3 TCA cycle metabolites, including fumarate and malate, were observed, indicative of increased PC contribution to these metabolites and reductive TCA cycling from oxaloacetate to fumarate (Fig. M + 3 and M + 5 citrate isotopomers are also likely derived from PC routed 13C6-glucose atoms (Fig. PC protein also trended towards increased expression in low iron conditions (Fig. 5j) while PCK2, which mediates the reverse reaction, converting oxaloacetate to the glycolytic intermediate phosphoenolpyruvate (PEP) was suppressed (Fig. 5k) suggesting oxaloacetate production is favoured in low iron conditions. In line with increased PC usage, pyruvate supplementation provided a proliferative advantage to iron-depleted cells (Fig. A partial reversal of TCA cycle activity is also in agreement with the observed increase in the NAD+/NADH ratio (Fig. This suggests that under iron limiting conditions, CD8+ T cells may utilise PC and a partial reversal of the TCA cycle to replenish the depleted metabolites fumarate and malate, circumventing the use of the iron-dependent enzymes, ACO2 and SDH. Alternatively, increased PC activity may be a consequence of a futile attempt to increase nucleotide production via aspartate. Aspartate accumulation under iron depletion could also be explained by decreased usage, as shown for nucleotide synthesis above. Aspartate is also used for the synthesis of asparagine, however at 48 h post-activation, CD8+ T cells in both iron-deficient and -replete conditions had not synthesised asparagine, indicated by the absence of 13C-labelling from 13C5-glutamine (Fig. This agrees with previous reports that CD8+ T cells lack asparagine producing capacity (asparagine synthetase expression) at 48 h post-activation, but gain this later into their activation and differentiation45. Accordingly, asparagine supplementation provided no proliferative benefit (Supplementary Fig. Taken together, the increased abundance of aspartate alongside decreased nucleotide synthesis indicates that aspartate usage by nucleotide synthesis is suppressed under iron restriction. To interrogate whether nucleotide abundance limits CD8+ T cell proliferation under iron deficiency, we first tested whether genetically increasing nucleotide abundance could rescue proliferation. Cellular nucleotide balance is maintained via the activity of iron-dependent ribonucleotide reductase (RNR) and SAMHD152 (Supplementary Fig. While RNR enables production of dNTPs, SAMHD1 degrades dNTPs52. We isolated CD8+ T cells from Samhd1-KO or wild-type littermates and activated them as described above. Samhd1-KO CD8+ T cells were less sensitive to iron scarcity in terms of a block on proliferation relative to wild-type cells (Supplementary Fig. This effect was modest, likely explained by the fact that SAMHD1-KO prevents dNTP breakdown but cannot rescue impaired nucleotide production or iron-dependent RNR activity. SAMHD1-KO CD8+ T cells showed appropriate upregulation of Tfrc expression in low iron concentrations (Supplementary Fig. 7c, d) but failed to rescue CD25 or perforin expression (Supplementary Fig. Notably, SAMHD1-KO cells showed comparable expression of Cdkn1a (Supplementary Fig. 7g), indicating that increased nucleotide pools provide a proliferative advantage despite elevated Cdkn1a expression. For aspartate to be utilised for nucleotide synthesis, it must be in the cytosol. Thus, an explanation for decreased aspartate usage could be inappropriate retention of aspartate in the mitochondria. 6a), but requires a proton gradient to mediate transport55, which is decreased in iron-depleted CD8+ T cells. Therefore, it is possible that a reduction in ETC chain activity due to iron deficiency may impair the mitochondrial proton gradient, in turn inhibiting SLC25A12/13 and sequestering aspartate in the mitochondria where it cannot be used for nucleotide synthesis. CD8+ T cells were activated as described in Fig. b Viable cell counts, n = 5. c Percentage of divided cells at 48 h and d percentage of cells undergoing 3+ divisions at 72 h assessed using cell trace violet (CTV), n = 5. e TFR1/CD71, f CD25 and g perforin MFI, n = 5. h IFN-γ MFI, n = 4. i mTORC1 activity measured via phospho-S6 (pS6), n = 4. j SAICAR and k N-carbamoyl-aspartate relative abundance normalised to spiked-in glutaric acid, n = 4. l H3K27me3 MFI, n = 4. Naïve-like control cells were cultured in IL-7 (5 ng/mL). Data are mean ± SEM, where each datapoint per condition denotes cells from independent donor mice. Statistics are: b non-linear regressions using exponential growth equations with an extra sum-of-squares F test applied for either high or low holotransferrin concentrations between aspartate-treated and untreated; c–i, l two-way ANOVAs with the Geisser-Greenhouse correction; j, k one-way ANOVAs with the Geisser-Greenhouse correction. We directly assessed whether supplementation of aspartate into cell culture media, such that it can be taken up directly into the cytosol, can rescue CD8+ T cell proliferation. 1a in the presence or absence of aspartate (40 mM). While CD8+ T cells in low iron conditions with no added aspartate showed almost no population expansion over 72 h of culture, the addition of aspartate increased the carrying capacity of the low iron culture by ~104 additional cells (Fig. When assessing division at 48 h using CTV, aspartate increased the percentage of divided cells from 25% in the lowest iron condition (0.0002 mg/mL holotransferrin) to 61% (Fig. At 72 h, the fraction of cells which could undergo three or more divisions in low iron conditions was profoundly increased with aspartate supplementation (increased from 7% to 56% in the lowest iron condition; Fig. Together, these data indicate that aspartate alone can substantially overcome the cell cycle impairment induced by iron scarcity and promotes a more profound recovery of cell division compared to exogenous pyruvate or Samhd1-KO. While clonal expansion is critical for effective CD8+ T cell effector responses, other aspects of CD8+ T cell biology provide important indicators of function. Tfrc mRNA expression was increased in low iron concentrations, indicative of cellular iron scarcity, irrespective of whether aspartate was supplemented or not (Supplementary Fig. However, aspartate supplementation did permit CD8+ T cells to maintain high TFR1/CD71 expression in the lowest iron concentrations when expression typically begins to drop (Fig. TFR1/CD71 levels may drop in the lowest iron conditions due to induction of tristetraprolin (TTP), a protein previously shown to suppress TFR1 expression under prolonged iron chelation56 or may be due to decreased TFR1 protein synthesis under mTORC1 suppression (Fig. Aspartate may therefore provide benefit via promoting increased iron uptake capacity during iron starvation, although the mechanism is unclear. Independent of iron, aspartate supplementation also promoted CD8+ T cell expression of the activation marker, CD25, the cytolytic molecule, perforin and the cytokine, IFN-γ, indicating its availability within the cytosol supports multiple cellular processes beyond proliferation (Fig. To understand if aspartate, in addition to altering cell proliferation and function, induces a shift in gene expression, we conducted an RNA-seq of CD8+ T cells in iron-replete and iron-deficient conditions, with or without aspartate (Supplementary Data 3). Samples were segregated largely based on iron concentrations rather than aspartate treatment (Supplementary Fig. 8b), suggesting that aspartate supplementation does not drive broad alterations in transcriptional profile. Aspartate supplementation induced relatively few transcriptional changes, with only 76 and 48 differentially expressed genes in high and low iron conditions, respectively. Notably, the transcriptional changes induced by aspartate in low iron conditions correlate with the changes mediated by aspartate in high iron conditions (Supplementary Fig. 8c), indicating aspartate has similar, albeit small, effects regardless of iron availability. Aspartate supplementation only marginally reduced Cdkn1a expression in low iron conditions (Supplementary Fig. 8d), indicating that the proliferative advantage conferred by aspartate is not entirely due to Cdkn1a suppression. GSEA revealed that the dominant aspartate effect is suppression of interferon (IFN) response pathways (Supplementary Fig. 8e) and was particularly driven by reduced IFN-stimulated gene expression. Similar effects were observed for aspartate treatment at high iron concentrations (Supplementary Fig. Notably, type-I IFN can have anti-proliferative effects that may be related to upregulation of Cdkn1a57. Thus, the rescue effect of aspartate on CD8+ T cell division may be partly facilitated if anti-proliferative type-I IFN signalling is counteracted. Aspartate did not alter mROS generation (Supplementary Fig. Aspartate-treated CD8+ T cells also displayed increased glycolytic and total ATP production (Supplementary Fig. 8i–k) and increased the abundance of the purine and pyrimidine precursors, SAICAR and N-carbamoyl-aspartate, albeit not significantly, suggesting that aspartate partially rescues nucleotide production, whilst consumption likely also increases in these more highly proliferating cells (Fig. Finally, aspartate counteracted the accumulation of the repressive histone mark H3K27me3 by iron-deficient CD8+ T cells (Fig. Therefore, even in cells cultured in very low iron conditions, aspartate availability enables TCR-triggered CD8+ T cells to acquire an activated phenotype, reconfigure important aspects of their metabolism and proliferate. Given the breadth and conserved nature of pathways involving iron-interacting proteins, it follows that iron deprivation would likely have broad implications on cell biochemistry, health and function, with potential knock-on effects at the tissue or systemic levels. Our work explains why iron depletion potently suppresses T cell responses in models of immunisation, infection and autoimmunity5,26,30,58,59. We also propose that the iron deficiency-associated perturbations described here may in part be responsible for cellular dysfunction in other cell types and tissues during iron deficiency. Cells that undergo rapid proliferation are more likely to be impacted by iron deficiency than quiescent populations. This is because in iron-scarce environments, dividing cells will dilute out iron stores between daughter cells without exogenous iron pools to replenish the difference, and dividing cells have high biosynthetic demands. Meanwhile, quiescent cells can furnish iron-interacting proteins via internal recycling of iron cofactors. This principle is demonstrated by T cells homozygous for the TFR1Y20H mutation, which reduces iron uptake efficiency11. Quiescent naïve TFR1Y20H T cells are not noticeably impacted by the mutation but become extremely sensitive upon activation11. Moreover, some tissues, such as muscle, may be able to resist iron deficiency to some degree by upregulation of glycolysis60. However, the capacity to switch to a glycolytic programme may depend on having reduced demands for nucleotides and other macromolecular synthesis processes necessary for proliferation. We demonstrate that iron deficiency in CD8+ T cells promotes an altered metabolic state. Iron deficiency caused relatively modest perturbation of the cellular transcriptome and proteome, at odds with the significant inhibition of proliferation. Iron deficiency acts most immediately to derail metabolism, with the most drastic effects on mitochondrial function. We observed decreased TCA cycle activity at the iron-dependent enzymes ACO2 and SDH, resulting in depletion of downstream metabolites. Despite suppressed TCA cycling, aspartate levels were augmented, but metabolites immediately downstream of aspartate metabolism, used for cytosolic nucleotide synthesis, were suppressed. We propose that aspartate accumulates within mitochondria due to iron-deficiency-associated suppressed activity of the proton-dependent mitochondrial aspartate carrier, SLC25A12/13. Remarkably, aspartate supplementation of culture media, assumed to access the cytosol directly, profoundly rescued iron-deficiency impaired proliferation and enhanced other functional aspects of activated T cells in both iron-dependent and -independent ways. Our results place disrupted aspartate handling front and centre in the metabolic derangements of cellular iron deficiency. Iron deficiency induced profound alterations to the transcriptional and proteomic landscape with significant perturbations in metabolic gene expression, including suppressed mTORC1 pathway-associated genes and elevated OXPHOS and β-oxidation proteins. Iron deficiency partially recapitulates the phenotype of mTORC1-inhibited CD8+ T cells, which feature reduced expression of ribosomal proteins15. However, in contrast to the decline in mitochondrial proteins observed with rapamycin15, we instead saw increased mitochondrial proteins, indicating that iron deficiency's impact on metabolism is not simply due to suppression of mTORC1. We hypothesise that the accumulation of mitochondrial proteins in low iron conditions could be due to augmented generation of mitochondrial proteins to compensate for suppressed mitochondrial activity or impaired clearance by proteolysis or mitophagy. Increased H3K27me3 in iron-deficient CD8+ T cells is suggestive of KDM6 dysfunction. The inability of CD8+ T cells to remodel their chromatin environment likely has implications for cellular differentiation. Moreover, suppressed levels of α-KG and/or iron may be capable of inhibiting iron and α-KG-dependent enzymes more generally, including other KDM enzymes and the ten-eleven translocation DNA demethylases, as has been seen for KDM3B, which can act as an iron sensor influencing H3K9 methylation and mTORC1 activation62, and the PHD proteins, which regulate HIF1α28,63. However, iron scarcity did not stabilise HIF1α or induce a proteomic response resembling that induced under hypoxic conditions. Different iron-dependent enzymes have been shown to have differential responses to iron inaccessibility, likely due to different iron binding affinities and expression levels62. For instance, KDM3B has a 40-fold lower affinity for iron than KDM3A (KM[Fe] = 100 ± 30 μM and KM[Fe] = 2.6 ± 0.8 μM, respectively), providing rationale for why iron chelation results in accumulation of repressive H3K9me2 in a KDM3B, but not KDM3A, dependent manner62. Similarly, the PHD proteins have very high affinities for iron64, potentially explaining our result that iron chelators, which potently suppress iron availability, induce a HIF1α response while iron deficiency caused by low extracellular transferrin-iron supply cannot. In 1925, Otto Warburg proposed that ‘every living cell contains iron and that life without iron is impossible'65, referring to the fact that cell culture media requires iron in order to support cellular growth65. Activated iron-deficient T cells do not progress through the proliferative cycle, but also do not apoptose, as has been observed by others62. Cellular iron deficiency mediated by depriving physiological sources of extracellular iron imparts a distinct and dysregulated metabolic signature. Our findings demonstrate that a lack of iron alters mitochondrial function, with knock-on effects influencing utilisation of cytosolic and nuclear metabolic pathways, ultimately affecting cell identity. Iron-deficient activated CD8+ T cells are ‘stunned' but not moribund, and can be substantially restored by aspartate, partially overcoming Warburg's limitation. Aspartate is generally poorly taken up by cells, but our findings suggest that engineering cells, such as tumour-targeted chimeric antigen receptor T cells, to overexpress the plasma membrane aspartate transporter (SLC1A1/2/3) could provide resistance to iron depletion, for example, in niches such as the tumour microenvironment. This research complies with all relevant ethics regulations at the University of Oxford. Animal procedures were completed under the authority of the UK Home Office project and personal licences granted under the Animals (Scientific Procedures) Act (ASPA) of 1998. Mice were housed in individually ventilated cages and fed ad libitum A03 standard maintenance diet (SAFE diets, SAFE A03). OT-I CD45.1 mice on a C57BL6 background were acquired from Vincenzo Cerundolo, University of Oxford and Audrey Gerard, University of Oxford. Samhd1-KO mice on a C57BL6 background were acquired from Jan Rehwinkel54. Sex was not considered in the experimental design. Mice were euthanised using a rising concentration of CO2 followed by cervical dislocation. To manipulate iron availability, iron-free medium was prepared with RPMI 1640 media (Gibco, 21875034), 1% penicillin/streptomycin (Sigma Aldrich, P0781-100ML), 1% glutamine (Sigma Aldrich, G7513-100ML) and 10% iron-free serum substitute (Pan Biotech, custom ordered as iron-free based on P04-95080). Iron-free media was supplemented with set concentrations of human holotransferrin (R&D Systems, 2914-HT-001G/Sigma Aldrich, T0665) ranging from 0.0002-0.625 mg/mL. Total transferrin was kept constant at 1.2 mg/mL by adjusting human apotransferrin (unbound transferrin) concentrations (R&D Systems, 3188-AT-001G/Sigma Aldrich, T1147) accordingly. Meanwhile, the low holotransferrin concentrations of 0.001 mg/mL are much lower than those found in plasma, but are hypothesised to be representative of highly iron-depleted environments such as those in the tumour microenvironment48. Aspartate (Scientific Laboratory supplies, CHE2306) was dissolved in iron-free media by slowly titrating in 1 M NaOH (Sigma Aldrich, 30620-1KG-M) until aspartate was completely solubilised and the pH was ~7.5. Asparagine and pyruvate were dissolved directly into iron-free media at the described concentrations. Plates for CD8+ T cell activation were coated with 5 μg/mL α-CD3 (Biolegend, 100239) in phosphate-buffered saline (PBS) for 2–3 h at 37 °C. Lymph nodes and spleens (inguinal, axillary, brachial, cervical and mesenteric) from mice were sterilely collected in iron-free media and homogenised through 40 μm filters using EasySep buffer (Stem Cell Technologies, 20144) or an in-house alternative (PBS + 2% FBS + 1 mM EDTA (Invitrogen, AM9260G)) to produce single cell suspensions. CD8+ T cells were isolated from total homogenate using the EasySep Mouse CD8+ T cell (Stem Cell Technologies, 19853) isolation kit and EasyPlate EasySep magnets (Stem Cell Technologies, 18102) or EasyEights EasySep magnets (Stem Cell Technologies, 18103), according to the manufacturer's protocols. Cells were optionally stained with CTV (Invitrogen, C34557) for 8 min at 37 °C prior to culture. Cells were plated at 0.5 × 106 cells/mL in iron-free media supplemented with defined holotransferrin concentrations (see section “Cell culture media”). CD8+ T cells were cultured at 37 °C, 5% CO2 for 24–72 h. CD8+ T cells were collected from cell culture plates and washed twice with PBS; cell pellets were immediately snap frozen on dry ice. T cell pellets were resuspended in 350 µL of RLT+ buffer from the Qiagen RNeasy plus mini kit (Qiagen, 74136) and RNA was extracted according to the manufacturer's instructions. For RNA extracted for RNA-sequencing, samples were treated on-column with the Qiagen RNase-free DNase I set (Qiagen, 79254) according to the method described in Appendix D of the Qiagen RNeasy mini kit manual (Qiagen, 74104). RNA concentrations were determined using a Nanodrop one instrument (Thermofisher Scientific). cDNA was synthesised using the High-Capacity RNA-to-cDNA kit (Applied Biosystems, 4388950), and qPCR experiments were completed on a QuantStudio 7 flex real-time PCR system (Applied Biosystems, 4485701) using the Taqman gene expression master mix (Applied Biosystems, 4369016) and appropriate Taqman Gene Expression Assay (B2m, Mm00437762_m1; Cdkn1a, Mm04205640_g1; Tfrc, Mm00441941_m1). B2m was used as the endogenous control gene. Library preparation and bulk mRNA-sequencing to a depth of 30 million paired-end reads using the Illumina NovaSeq 6000 platform was conducted by Novogene. Read alignment was conducted using RNA-star to the Mus Musculus genome (mm39), and features were annotated using FeatureCounts from the subread package. Differential gene expression analysis was conducted using EdgeR (R package) with the thresholds of log2|fold change| > 1.5 and an FDR < 0.05 applied. GSEA was completed using FGSEA (R package) and Hallmark pathways with a significance threshold set as FDR < 0.05. CD8+ T cells were activated for 48 h, after which they were collected, washed twice in PBS and fixed in 1% paraformaldehyde (PFA, Thermo Scientific, 28906) for 10 min at room temperature with rotation. Cells were washed twice at 16,000 × g for 30 s, and dry pellets were snap frozen on dry ice. Antibody binding buffer was prepared: 0.5% BSA and 1/200 protease inhibitor cocktail (PIC) in PBS. 10 µL of protein A dynabeads (ThermoFisher, 1001D) per sample were washed twice with antibody binding buffer on a magnet. Beads were blocked with 1 µL of antibody (α-H3K27ac, Diagenode, C15410196; α-H3K4me3, Diagenode, C15410003; α-H3K27me3, Sigma Aldrich, 07-449) in antibody binding buffer for 3–4 h at 4 °C with rotation. Just prior to adding the chromatin, the beads were washed with antibody binding buffer. Cell pellets were resuspended in 120 µL of lysis buffer (50 mM Tris pH 8, 10 mM EDTA, 0.5% SDS) with 1/200 PIC and transferred into Covaris tubes (Covaris, 520045). Samples were diluted with lysis buffer with 1/200 PIC to a 900 µL total volume, and 100 µL of 10% Triton X-100 was added to neutralise. Samples were incubated for 10 min with rotation. 10 µL of prewashed protein A dynabeads were added and incubated for 30 min at 4 °C. Samples were placed on a magnet, and 250 µL of sample supernatant was added to antibody-bound beads. Chromatin/antibody/bead mixtures were incubated overnight at 4 °C with rotation. Samples were washed three times with 150 µL RIPA wash buffer (50 mM HEPES pH 7.6, 500 mM LiCl, 1 mM EDTA, 1% NP-40, 0.7% NaDeoxycholate), once with 150 µL TE buffer and once with 150 µL 10 mM Tris pH 8, each time for 5 min. Samples were incubated for 5 min with vigorous mixing at 1100 rpm at 37 °C in a thermomixer. 150 µL of RIPA buffer was added immediately to degrade the Tn5. Beads were washed once with 150 µL of RIPA buffer and then resuspended with 22.5 µL of ddH2O. 25 µL of 2X NEBNext Ultra II Q5 MasterMix (New England Biosciences, M0544S), 1.25 µL 5 µM universal adaptor primer and 1.25 μL 5 µM index adaptor primer was added to each sample. Supernatants were removed and 50 µL of Ampure XP beads (Beckman Coulter, A63880) were added and pipetted 10 times and then incubated at room temperature for 2 min. Beads were washed once with 150 µL 80% ethanol, incubating for 2 min at room temperature. The ethanol was removed, and the beads were allowed to air dry for 3–5 min. DNA sequencing to a depth of 10 million paired-end reads using the Illumina NovaSeq 6000 platform was conducted by Novogene. Data was analysed using the SeqNado analysis pipeline (https://github.com/alsmith151/SeqNado). Peak enrichment for H3K27ac and H3K4me3 was conducted for the promoters of genes previously identified as being differentially regulated at the RNA level by the RNA-seq. CD8+ T cells were activated for 48 h, harvested and washed twice in PBS. Cells were fixed in 2% PFA (Thermo Scientific, 28906) for 30 min, washed and resuspended in PBS. Cells that were alive at the time of fixation were sorted by flow cytometry using forward and side scatter into PBS. Cells were pelleted and snap frozen on dry ice. Pelleted cells were lysed using the 2-step trypsin lysis protocol described by Kelly et al.66. Cell pellets were resuspended in 200 µL TEAB digest buffer (0.1 M TEAB, 1 mM MgCl2, 1:80 benzonase, pH 8) and incubated for 20 min at 37 °C in a thermomixer. Samples were incubated overnight at 37 °C in a thermomixer. The remaining trypsin was added, and samples were incubated for 60 min at 37 °C in a thermomixer. The samples were acidified to a final concentration of 1% trifluoroacetic acid (TFA) and subjected to a C18 stage-tip desalting with the following buffers: condition (100% acetonitrile), wash (0.1% TFA), elute (66.6% acetonitrile, 0.1% TFA), ion exchange (2:1 ratio of 100% acetonitrile to 0.1% TFA). Samples were dried using a SpeedVac at 65 °C. Protein-MS analysis was performed as described previously67. Briefly, peptides were analysed on a Q-Exactive-HF-X (Thermo Scientific) mass spectrometer coupled with a Dionex Ultimate 3000 RS (Thermo Scientific). LC buffers were the following: buffer A (0.1% formic acid in Milli-Q water (v/v)) and buffer B (80% acetonitrile and 0.1% formic acid in Milli-Q water (v/v)). 2 μg of each sample were loaded at 15 μL/min onto a trap column (100 μm × 2 cm, PepMap nanoViper C18 column, 5 μm, 100 Å, Thermo Scientific) equilibrated in 0.1% TFA. The trap column was washed for 3 min at the same flow rate with 0.1% TFA, then switched in line with a Thermo Scientific, resolving C18 column (75 μm × 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å). The peptides were eluted from the column at a constant flow rate of 300 nL/min with a linear gradient from 3% buffer B to 6% buffer B in 5 min, then from 6% buffer B to 35% buffer B in 115 min, and finally to 80% buffer B within 7 min. The column was then washed with 80% buffer B for 4 min and re-equilibrated in 3% buffer B for 15 min. Two blanks were run between each sample to reduce carry-over. The column was always kept at a constant temperature of 50 °C. The data were acquired using an easy spray source operated in positive mode with spray voltage at 1.9 kV, the capillary temperature at 250 °C, and the funnel RF at 60 °C. The MS was operated in data-independent acquisition (DIA) mode as reported earlier with some modifications68. MS survey scan was followed by MS/MS DIA scan events using the following parameters: default charge state of 3, resolution 30,000, maximum ion injection time 55 ms, AGC 3 × 106, stepped normalised collision energy 25.5, 27 and 30, fixed first mass 200 m/z. Data for both MS and MS/MS scans were acquired in profile mode. Mass accuracy was checked before the start of sample analysis. Quantification of reporter ions was completed using Spectronaut (Biognosys; Spectronaut 14.10.201222.47784) in library-free (directDIA) mode. Minimum peptide length was set to 7, and maximum peptide length was set to 52, with a maximum of 2 missed cleavages. Trypsin was specified as the digestive enzyme used. The FDR at the precursor ion level and protein level was set at 1% (protein and precursor Q value cutoff). The max number of variable modifications was set to 5, with protein N-terminal acetylation and glutamine and asparagine deamidation, and methionine oxidation set as variable modifications. Carbamidomethylation of cysteine residues was selected as a fixed modification. Data filtering and protein copy number quantification were performed in the Perseus software package, version 1.6.6.0. Copy numbers were calculated using the proteomic ruler69. The ratio between the histone peptide intensity and summed peptide intensities of all other identified proteins is then used to estimate the protein copy number per cell for all the identified proteins. We also applied the typical p value threshold of <0.05 when comparing the high (0.625 mg/mL holotransferrin) and low (0.001 mg/mL holotransferrin) iron concentrations via a t-test as well as an additional threshold of p values < 0.05 when a one-way ANOVA was conducted across all conditions. GSEA was conducted using FGSEA (R package) and the Hallmark pathways. For heavy isotope tracing experiments, CD8+ T cells were isolated and activated as described above. Iron-free media with varying holotransferrin concentrations were prepared using phenol red-free RPMI 1640 (Gibco, 11835030) as phenol red can interfere with metabolite-ms. At 24 h prior to cell collection (24 h post-activation), cells were collected, washed and replated in media containing the heavy isotope of interest (13C6-glucose (Cambridge Isotope Laboratories, Inc., CLM-1396-1) and 13C5-glutamine (CK Isotopes, CNLM-1275)) on plates coated with α-CD3. Iron-free tracing media was prepared using SILAC RPMI 1640 flex media (Gibco, A24945201), which lacks glucose, phenol red, glutamine, arginine and lysine. Arginine (MP Biomedicals, 194626) and lysine hydrochloride (MP Biomedicals, 194697) were supplemented at standard RPMI 1640 concentrations of 1.2 mM and 0.2 mM, respectively. Holotransferrin and apotransferrin, as well as activation reagents (BME, α-CD28 and IL-2) were added to the iron-free tracing media as described under ‘CD8+ T cell activation' at standard concentrations. Metabolites were measured by Gas chromatography-mass spectrometry as described below. Forty-eight-hour activated CD8+ T cells were collected, counted and 2−4 × 106 cells (experiment dependent, however, the same number of cells were used for all samples within an experiment) were washed twice in ice-cold 0.9% saline made with NaCl (Sigma Aldrich, 31434-500G-M) and ultrapure HPLC grade water (Alfa Aesar, 22934). Cells were pelleted and snap frozen on dry ice. Cells were extracted in 1:1:1 pre-chilled methanol, HPLC grade water (containing 1.75 μg/mL D6-glutaric acid) and chloroform. The upper aqueous phase was collected and evaporated under vacuum. Metabolite derivatization was performed using an Agilent autosampler. Dried polar metabolites were dissolved in 15 μL of 2% methoxyamine hydrochloride in pyridine (Thermo Fisher Scientific, 25104) at 55 °C, followed by an equal volume of N-tert-Butyldimethylsilyl-N-methyltrifluoroacetamide with 1% tertbutyldimethylchlorosilane after 60 min, and incubation for a further 90 min at 55 °C. GC-MS analysis was performed using an Agilent 6890 GC equipped with a 30 m DB-35 MS capillary column. The GC was connected to an Agilent 5975C MS operating under electron impact ionisation at 70 eV. The detector was operated in scan mode, and 1 μL of derivatised sample was injected in splitless mode. The GC oven temperature was held at 80 °C for 6 min and increased to 325 °C at a rate of 10 °C/min for 4 min. The run time for each sample was 59 min. For the determination of the mass isotopomer distributions, spectra were corrected for natural isotope abundance. Data processing was performed using MATLAB. CD8+ T cells were collected after 48 h of culture, and 3 × 106 cells were washed twice with PBS. Cell pellets were resuspended in 0.01% formic acid (Sigma Aldrich, 1003445799) containing 0.8 µM 15N2-dT (Cambridge Isotope Labs, NLM-3901-PK) and were incubated at 37 °C for 30 min. Samples were centrifuged at maximum speed for 10 min, and the lysates were collected and added to 3 kDa Amicon Ultra 0.5 centrifugal filter units (Millipore, UFC500324) and centrifuged for 20 min at 16,000 × g. The filtrates were transferred to a mass spectrometry vial for analysis of compounds in nucleotide synthesis. Samples were analysed on a TSQ Altis Triple Quadrupole Mass Spectrometer in selected reaction monitoring mode, interfaced to an UltiMate 3000 uHPLC. The uHPLC was fitted with a nanoEase M/Z Symmetry C18 Trap Column, 100 Å, 5 µm, 180 µm × 20 mm (Waters) at RT and a Luna Omega 3 µm PS C18 column (150 × 0.3 mm) connected to an EASY-Spray™ source with an EASY-Spray Cap flow emitter (15 µM). Buffers used were from Romil and of Ultra LC standard. This was followed by 2 wash pulses (1–99% B) and equilibration to 1% B (45 min total run time). The trap column was held at a constant 1% B, and switching from the trap to the main column occurred at 1 min and back at 40 min. Mass spectrometry conditions were as follows: source voltage of 2400 V in positive ionisation mode; ion transfer tube temperature 275 °C, CID gas pressure 1.5 mTorr, scan widths for Q1 and Q3 at 0.7 m/z, a chromatographic filter was used with a peak width of 6 s, dwell time 20 ms. Collision energy voltage, and RF voltage were optimised using the Thermo Tune software for the compounds assayed using authentic compounds dissolved in Buffer A. Peaks retention times were confirmed in samples by co-injection with the relevant standard. Data was analysed using the FreeStyle 1.6 software and the Genesis peak detection algorithm with its default settings. Data was analysed by ratioing the target peak area with the 15N2-dT peak area. CD8+ T cells were collected at 48 h post-activation, and 3 × 106 cells were washed twice with ice-cold 0.9% saline made with NaCl (Sigma Aldrich, 31434-500G-M) and ultrapure HPLC grade water (Alfa Aesar, 22934). Cells were pelleted and snap frozen on dry ice. 8.8 µL of 15% ammonium bicarbonate (Supelco, 5.33005) solution was added to neutralise, and samples were left on dry ice for a further 15 min. Supernatants were collected for LC-MS analysis. LC-MS analysis was performed on an Agilent LC-MS QToF 6546 using a Waters Premier BEH Z-HILIC VanGuard Fit column (1.7 µm, 2.1 × 150 mm) with the following mobile phases: Mobile phase A: 20 mM Ammonium hydrogen carbonate (LiChropur, Supelco) in LC-MS grade water (Ultra CHROMASOLV, Honeywell Riedel-de Haën) with 0.1% ammonium hydroxide (Alfa Aesar, Thermo Scientific) and 5 µM InfinityLab deactivator additive (Agilent Technologies). Mobile phase B: 90% UHPLC grade acetonitrile (BioSolv, Greyhound Chemicals), 10% LC-MS grade water (Ultra CHROMASOLV, Honeywell Riedel-de Haën) with 5 µM InfinityLab deactivator additive (Agilent Technologies). Full scan data was acquired between m/z 50–1050 at 1 Hz whilst using online mass correction. Data extraction was performed using Agilent Profinder 10.0. Data were normalised to the internal standard (D6-Glutaric acid) peak area. NAD+/NADH ratio measurements were made using the NAD/NADH-Glo assay (Promega, G9071) according to the manufacturer's instructions with modifications for measuring NAD+ and NADH individually (rather than pooled). Cultured CD8+ T cells were collected, washed in PBS and counted. 1 × 105 cells in 50 µL of PBS were lysed in 50 µL of 0.2 M NaOH (Sigma Aldrich, 30620-1KG-M) with 1% dodecyltrimethylammonium bromide (DTAB, Alfa Aesar, A10761) per condition in technical duplicates. Each sample was split (50 µL) into a second tube for matched NAD+ and NADH measurements. For measurement of NAD+, NADH was first degraded by adding 25 µL of 0.4 M HCl (Fisher Scientific, H/1111/PB17) to the first tube, heated to 60 °C for 15 min followed by incubation at room temperature for 10 min. Twenty-five microlitres 0.5 M TRIZMA base (Sigma Aldrich, T1503-1KG) were added to quench the HCl. To measure NADH, NAD+ was degraded by heating the second tube to 60 °C for 15 min followed by incubation at room temperature for 10 min. Fifty microlitres of 0.2 M HCl (Fisher Scientific, H/1111/PB17) and 0.25 M TRIZMA base (Sigma Aldrich, T1503-1KG) solution were added to quench the NaOH. The NAD/NADH-Glo detection reagent was prepared as instructed. Fifty microlitres of each sample (either purified for NAD+ or NADH) and 50 µL of NAD/NADH-Glo detection reagent were added to a white 96-well plate and mixed gently. The plate was incubated at room temperature and luminescence was measured using a Promega GloMax multi-detection luminometer at 30, 45 and 60 min. Extracellular flux analysis (Seahorse) was conducted using the Seahorse XF Real-Time ATP rate assay kit (Agilent, 103592-100). The day prior to the assay, the Agilent Seahorse XF96 analyser was turned on, and a sensor cartridge (Agilent, 103792-100) was hydrated in sterile dH2O at 37 °C in a non-CO2 incubator overnight. Cells were collected, washed and 1 × 105 cells in 50 µL of ATP assay media were plated per well on a Poly-D-lysine (Gibco, A3890401) coated XF96 microplate (Agilent, 101085-004). Plates were spun down, supernatant removed and cells incubated at 37 °C for 30 min in a non-CO2 incubator, followed by the addition of 130 µL of ATP assay media and further incubation for 25–30 min. Oligomycin and rotenone/antimycin A were used at a final concentration of 1.5 µM and 0.5 µM, respectively. Twenty microlitres of oligomycin (15 µM) and 22 µL of rotenone/antimycin A (5 µM) were loaded into the sensor cartridge in ports A and B, respectively. The ATP rate assay was run on a Seahorse XF96 analyser with 3 measurements taken for each step (baseline, oligomycin, rotenone/antimycin A) using the following injection settings: 3 min mix, 0 min wait, 3 min measure. Data were analysed using the Agilent Seahorse Wave desktop software (version 2.6) and the Seahorse XF real-time ATP rate assay report generator. Cells were transferred from cell culture plates to 96-well round-bottom plates and washed with PBS. In intracellular cytokine staining experiments, cells were first stimulated with cell activation cocktail (1:2500) (Biolegend, 423301), BFA (5 mg/mL) (Biolegend, 420601) and monensin (2 mM) (Biolegend, 420701) in iron-free media at 37 °C/5% CO2 for 5 h prior to staining. Cells were stained with 20 µL of surface antibody cocktail with Zombie NIR fixable viability kit (1:400-1000, Biolegend, 423105) in PBS for 20 min on ice. Cells were fixed with 2% paraformaldehyde (Pierce, 28906) diluted in PBS or commercial fixation buffer (Biolegend, 420801) for 20 min on ice. For nuclear staining of markers such as H3K27me3, cells were fixed in Foxp3 transcription factor fixation buffer (eBioscience, 00-5523-00) for 1 h on ice. Intracellular targets were stained with 20 µL of intracellular stain prepared in perm/wash buffer for 20 min to overnight. Cells were resuspended in PBS, and data were acquired using an Attune NxT flow cytometer (Thermofisher Scientific). Data was analysed using FlowJo (BD Biosciences). A representative gating scheme can be found in Supplementary Fig. Detection of mROS was conducted using MitoSOX dye (Invitrogen, M36008). Cells were resuspended in 200 µL of MitoSOX dye diluted to 5 µM in phenol red iron-free media and incubated at 37 °C/5% CO2 for 15 min before surface staining. The acquisition was conducted on live cells. Cells were stained with MTG (M7514, Thermofisher Scientific) and Mitotracker Deep Red (MTDR; M22426, Thermofisher Scientific) at 100 nM in iron-free media for 30 min at 37 °C/5% CO2 prior to staining. The ratio of MTDR to MTG was calculated as a metric of mitochondrial membrane potential relative to mitochondrial mass. For flow staining for the metabolic regulators, pS6 and HIF1α, cells were washed once with a solution of 50% RPMI 1640, 50% HBSS (Gibco, 14025-092) and 0.5% BSA (PAN biotech, PO6-139310) before fixation in 1% PFA (Pierce, 28906) for 20 min. Cells were washed with PBS and optionally surface-stained for fixation of stable epitopes. Cells were permeabilised using 90% ice-cold methanol (Sigma Aldrich, 34860-1L-R) for 20 min at −20 °C prior to permeabilization and acquisition. Cells were transferred into 96-well round-bottom TC plates and washed once with 2% FBS in PBS (FACS buffer). Cells were resuspended in FACS buffer containing 0.5 µg/mL 7-AAD (Biolegend, 420403) and incubated on ice for 10 min and then acquired directly on the Attune NxT flow cytometer (Thermofisher Scientific). Fifty microlitres of cells were acquired per well to enable accurate calculation of cell concentration, and wells were resuspended using a multichannel pipette between each column on the 96-well plate to account for cell settling effects. Unless explicitly specified, data analysis was completed using Excel (Microsoft), Prism (GraphPad), FlowJo (BD Biosciences), or published R packages. Statistics are specified in the figure legends. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The metabolomics dataset for measuring AICAR has been deposited in the MetaboLights repository here: https://metabolights-labs.org/u/christopher_lee_millington/h/megan-teh-raw-lc-ms-data. All other data are available in the article and its Supplementary files or from the corresponding author upon request. Source data are provided with this paper. Ryan, D. G. et al. Coupling Krebs cycle metabolites to signalling in immunity and cancer. Prevalence, years lived with disability, and trends in anaemia burden by severity and cause, 1990-2021: findings from the Global Burden of Disease Study 2021. Frost, J. N. et al. Hepcidin-mediated hypoferremia disrupts immune responses to vaccination and infection. Dautry-Varsat, A. Receptor-mediated endocytosis: the intracellular journey of transferrin and its receptor. Rapid growth is a dominant predictor of hepcidin suppression and declining ferritin in Gambian infants. Shah, A., Frost, J. N., Aaron, L., Donovan, K. & Drakesmith, H. Systemic hypoferremia and severity of hypoxemic respiratory failure in COVID-19. Jabara, H. H. et al. A missense mutation in TFRC, encoding transferrin receptor 1, causes combined immunodeficiency. Drakesmith, H. & Frost, J. N. Iron and the immune system. Reina-Campos, M., Scharping, N. E. & Goldrath, A. W. CD8(+) T cell metabolism in infection and cancer. Teh, M. R., Frost, J. N., Armitage, A. E. & Drakesmith, H. Analysis of iron and iron-interacting protein dynamics during T-cell activation. Howden, A. J. M. et al. Quantitative analysis of T cell proteomes and environmental sensors during T cell differentiation. Interpretation of biochemical tests for iron deficiency: diagnostic difficulties related to limitations of individual tests. Lymph protects metastasizing melanoma cells from ferroptosis. Wilkinson, N. & Pantopoulos, K. The IRP/IRE system in vivo: insights from mouse models. Sinclair, L. V. et al. Control of amino-acid transport by antigen receptors coordinates the metabolic reprogramming essential for T cell differentiation. Cutting edge: activation-induced iron flux controls CD4 T cell proliferation by promoting proper IL-2R signaling and mitochondrial function. Berg, V. et al. Iron deprivation in human T cells induces nonproliferating accessory helper cells. The P53 pathway: What questions remain to be explored?. Fu, D. & Richardson, D. R. Iron chelation and regulation of the cell cycle: 2 mechanisms of posttranscriptional regulation of the universal cyclin-dependent kinase inhibitor p21CIP1/WAF1 by iron depletion. Acute iron deprivation reprograms human macrophage metabolism and reduces inflammation in vivo. Lai, Y. et al. Iron controls T helper cell pathogenicity by promoting glucose metabolism in autoimmune myopathy. Kim, B. M., Choi, J. Y., Kim, Y. J., Woo, H. D. & Chung, H. W. Desferrioxamine (DFX) has genotoxic effects on cultured human lymphocytes and induces the p53-mediated damage response. An, W. G. et al. Stabilization of wild-type p53 by hypoxia-inducible factor 1alpha. Ross, S. H., Rollings, C. M. & Cantrell, D. A. Quantitative analyses reveal how hypoxia reconfigures the proteome of primary cytotoxic T lymphocytes. Wang, Z. et al. Iron drives T helper cell pathogenicity by promotiNG RNA-binding protein PCBP1-mediated proinflammatory cytokine production. Rath, S. et al. MitoCarta3.0: an updated mitochondrial proteome now with sub-organelle localization and pathway annotations. Turrens, J. F. Mitochondrial formation of reactive oxygen species. Pearce, E. L., Poffenberger, M. C., Chang, C. H. & Jones, R. G. Fueling immunity: insights into metabolism and lymphocyte function. Bera, S. et al. Allosteric regulation of glutamate dehydrogenase deamination activity. Jaccard, A. et al. Reductive carboxylation epigenetically instructs T cell differentiation. Loenarz, C. & Schofield, C. J. Physiological and biochemical aspects of hydroxylations and demethylations catalyzed by human 2-oxoglutarate oxygenases. Cribbs, A. P. et al. Histone H3K27me3 demethylases regulate human Th17 cell development and effector functions by impacting on metabolism. Distinct epigenetic signatures delineate transcriptional programs during virus-specific CD8(+) T cell differentiation. Kaelin, W. G. Jr ROS: really involved in oxygen sensing. Cancer cells tune the signaling pathways to empower de novo synthesis of nucleotides. Birsoy, K. et al. An essential role of the mitochondrial electron transport chain in cell proliferation is to enable aspartate synthesis. Supporting aspartate biosynthesis is an essential function of respiration in proliferating cells. Schwartz, A. J. et al. Hepcidin sequesters iron to sustain nucleotide metabolism and mitochondrial function in colorectal cancer epithelial cells. Leone, R. D. et al. Glutamine blockade induces divergent metabolic programs to overcome tumor immune evasion. Elia, I. et al. Tumor cells dictate anti-tumor immune responses by altering pyruvate utilization and succinate signaling in CD8(+) T cells. Coggins, S. A., Mahboubi, B., Schinazi, R. F. & Kim, B. SAMHD1 functions and human diseases. The deoxynucleotide triphosphohydrolase SAMHD1 is a major regulator of DNA precursor pools in mammalian cells. Rehwinkel, J. et al. SAMHD1-dependent retroviral control and escape in mice. & Kunji, E. R. S. The SLC25 mitochondrial carrier family: structure and mechanism. Bayeva, M. et al. mTOR regulates cellular iron homeostasis through tristetraprolin. & Hehlmann, R. The interferon-alpha revival in CML. Li, L. et al. Iron deprivation restrains the differentiation and pathogenicity of T helper 17 cell. Voss, K. et al. Elevated transferrin receptor impairs T cell metabolism and function in systemic lupus erythematosus. Henderson, S. A., Dallman, P. R. & Brooks, G. A. Glucose turnover and oxidation are increased in the iron-deficient anemic rat. & Bailis, W. Metabolic signaling in T cells. Shapiro, J. S. et al. Iron drives anabolic metabolism through active histone demethylation and mTORC1. Henning, A. N., Roychoudhuri, R. & Restifo, N. P. Epigenetic control of CD8(+) T cell differentiation. McNeill, L. A. et al. Hypoxia-inducible factor prolyl hydroxylase 2 has a high affinity for ferrous iron and 2-oxoglutarate. Kelly, V., Al-Rawi, A., Lewis, D., Kustatscher, G. & Ly, T. Low cell number proteomic analysis using in-cell protease digests reveals a robust signature for cell cycle state classification. Reyes, L. et al. —A type I IFN, prothrombotic hyperinflammatory neutrophil signature is distinct for COVID-19 ARDS. Surpassing 10 000 identified and quantified proteins in a single run by optimizing current LC-MS instrumentation and data analysis strategy. & Mann, M. A “proteomic ruler” for protein copy number and concentration estimation without spike-in standards. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. Thank you to the staff of the Department of Biomedical Services, University of Oxford for animal husbandry and welfare, the members of the Drakesmith lab, Michael Murphy (MRC Mitochondrial Biology Unit, University of Cambridge), Sumana Sharma (MRC Translational Immune Discovery Unit, University of Oxford), Katja Simon (The Kennedy Institute of Rheumatology, University of Oxford) and Ana-Victoria Lechuga-Vieco (The Kennedy Institute of Rheumatology, University of Oxford) for helpful discussions, Natasha White and Giulia Pironaci (both MRC Translational Immune Discovery Unit, University of Oxford) for assistance with OT-I colony management, Dana Costigan (MRC Translational Immune Discovery Unit, University of Oxford) for practical help with pilot experiments, Andrew Howden and the FingerPrints Proteomic Facility for protein-MS analysis (University of Dundee) and the support and resources of the Birmingham Metabolic Tracer Analysis Core (University of Birmingham). were supported by a U.K. Medical Research Council, MRC Human Immunology Unit grant awarded to H.D. is supported by a Cancer Research Institute fellowship (award CRI4186) and the Center for Experimental Immuno-Oncology of Memorial Sloan Kettering Cancer Center. was funded by the Clarendon Fund and the Corpus Christi College A. E. Haigh graduate scholarship. and A.S. were supported by a Medical Research Council (MRC, UK) Molecular Haematology Unit grant awarded to T.A.M. was supported by a Wellcome Trust Principal Research Fellowship, awarded to Doreen Cantrell (205023/Z/16/Z). are supported by a Blood Cancer UK Project grant (21007) and an MRC New Investigator Research Grant (MR/V011588/1). are supported by a Cancer Research UK Programme grant (C42109/A24757). Present address: CeMM—Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria These authors contributed equally: Sarah Dimeloe, Hal Drakesmith. Megan R. Teh, Joe N. Frost, Hannah Murray, Alexandra E. Preston, Jan Rehwinkel, Andrew E. Armitage & Hal Drakesmith Nancy Gudgeon, Victoria Stavrou & Sarah Dimeloe Alastair L. Smith, Christopher L. Millington & Thomas A. Milne Nuffield Department of Clinical Medicine, NDM Centre for Global Health Research, University of Oxford, Oxford, UK Jennie Roberts, Bryan P. Marzullo & Daniel A. Tennant Mahidol-Oxford Tropical Medicine Research Unit, Mahidol University, Bangkok, Thailand Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Megan R. Teh or Hal Drakesmith. is a paid consultant for and shareholder in Dark Blue Therapeutics Ltd. D.A.T. undertakes paid consultancy work for Sitryx Ltd. The remaining authors declare no competing interests. Nature Communications thanks Kostas Pantopoulos and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Iron deficiency causes aspartate-sensitive dysfunction in CD8+ T cells. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01420-w'>Strategies for the vascular patterning of engineered tissues for organ repair</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 11:10:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Biomedical Engineering (2025)Cite this article Metrics details The loss of organ function following traumatic injury is often irreversible and the demand for organ replacements continues to exceed supply. This discrepancy has driven the development of therapies and engineered tissues for the repair or replacement of damaged tissues. However, the survival of engineered tissues is constrained by the challenge of establishing a functional vasculature. Efforts have therefore focused on strategies that induce vascularization in tissue implants or stimulate vascular growth in recipients of the therapies. Here we discuss recent advances in vascular biology, biomaterials chemistry and 3D printing techniques for vascular patterning in engineered tissues. For tissue regeneration to be clinically viable, vascular formation must be guided across scales ranging from micrometres to millimetres through biological, chemical and physical approaches. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Brittberg, M., Recker, D., Ilgenfritz, J. & Saris, D. B. F. Matrix-applied characterized autologous cultured chondrocytes versus microfracture: five-year follow-up of a prospective randomized trial. Sports Med. Article PubMed Google Scholar Schmidt, C. Gintuit cell therapy approval signals shift at US regulator. Google Scholar Swan, M. Steady advance of stem cell therapies: report from the 2011 World Stem Cell Summit, Pasadena, California, October 3–5. Rejuvenation Res. Article PubMed Google Scholar Carmeliet, P. & Jain, R. K. Angiogenesis in cancer and other diseases. Google Scholar Miller, J. S. et al. Rapid casting of patterned vascular networks for perfusable engineered three-dimensional tissues. Google Scholar Phelps, E. A. & Garcia, A. J. Update on therapeutic vascularization strategies. Article PubMed Google Scholar Gandhi, J. K., Opara, E. C. & Brey, E. M. Alginate-based strategies for therapeutic vascularization. Google Scholar Bowers, D. T., Song, W., Wang, L.-H. & Ma, M. Engineering the vasculature for islet transplantation. Acta Biomater. Google Scholar Gianni‐Barrera, R. et al. Therapeutic vascularization in regenerative medicine. Stem Cells Transl. Google Scholar Wilson, Z. E. et al. Inter-individual variability in levels of human microsomal protein and hepatocellularity per gram of liver. Google Scholar Risau, W. & Flamme, I. Vasculogenesis. Cell Dev. Google Scholar Risau, W. Mechanisms of angiogenesis. Google Scholar Zhang, B. et al. Microfabrication of AngioChip, a biodegradable polymer scaffold with microfluidic vasculature. Google Scholar Zhang, B. et al. Biodegradable scaffold with built-in vasculature for organ-on-a-chip engineering and direct surgical anastomosis. Google Scholar Szklanny, A. A. et al. 3D bioprinting of engineered tissue flaps with hierarchical vessel networks (VesselNet) for direct host-to-implant perfusion. Google Scholar Wang, X. et al. Improving islet engraftment by gene therapy. Google Scholar Ajioka, I. et al. Establishment of heterotropic liver tissue mass with direct link to the host liver following implantation of hepatocytes transfected with vascular endothelial growth factor gene in mice. Tissue Eng. Google Scholar Vlahos, A. E. et al. Endothelialized collagen based pseudo-islets enables tuneable subcutaneous diabetes therapy. Google Scholar Weaver, J. D. et al. Vasculogenic hydrogel enhances islet survival, engraftment, and function in leading extrahepatic sites. Google Scholar Phelps, E. A., Templeman, K. L., Thulé, P. M. & García, A. J. Engineered VEGF-releasing PEG-MAL hydrogel for pancreatic islet vascularization. Drug Deliv. Google Scholar Mirabella, T. et al. 3D-printed vascular networks direct therapeutic angiogenesis in ischaemia. Google Scholar Baranski, J. D. et al. Geometric control of vascular networks to enhance engineered tissue integration and function. Natl Acad. Google Scholar Song, W. et al. Engineering transferrable microvascular meshes for subcutaneous islet transplantation. Google Scholar Mastrullo, V., Cathery, W., Velliou, E., Madeddu, P. & Campagnolo, P. Angiogenesis in tissue engineering: as nature intended? Google Scholar Goldie, L. C., Nix, M. K. & Hirschi, K. K. Embryonic vasculogenesis and hematopoietic specification. Google Scholar Asahara, T. et al. Isolation of putative progenitor endothelial cells for angiogenesis. Google Scholar Asahara, T. et al. VEGF contributes to postnatal neovascularization by mobilizing bone marrow-derived endothelial progenitor cells. EMBO J. Google Scholar Takahashi, T. et al. Ischemia- and cytokine-induced mobilization of bone marrow-derived endothelial progenitor cells for neovascularization. Google Scholar De Spiegelaere, W. et al. Intussusceptive angiogenesis: a biologically relevant form of angiogenesis. Article PubMed Google Scholar Rouwkema, J. & Khademhosseini, A. Vascularization and angiogenesis in tissue engineering: beyond creating static networks. Trends Biotechnol. Google Scholar Burri, P. H., Hlushchuk, R. & Djonov, V. Intussusceptive angiogenesis: its emergence, its characteristics, and its significance. Article PubMed Google Scholar Hellström, M. et al. Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis. Article PubMed Google Scholar Roesel, J. F. & Nanney, L. B. Assessment of differential cytokine effects on angiogenesis using an in vivo model of cutaneous wound repair. Google Scholar Wei, L.-H. et al. Interleukin-6 promotes cervical tumor growth by VEGF-dependent angiogenesis via a STAT3 pathway. Google Scholar Li, A., Dubey, S., Varney, M. L., Dave, B. J. & Singh, R. K. IL-8 directly enhanced endothelial cell survival, proliferation, and matrix metalloproteinases production and regulated angiogenesis. Google Scholar Voronov, E. et al. IL-1 is required for tumor invasiveness and angiogenesis. Natl Acad. Google Scholar Miller, J. S. et al. Bioactive hydrogels made from step-growth derived PEG–peptide macromers. Google Scholar Djonov, V., Baum, O. & Burri, P. H. Vascular remodeling by intussusceptive angiogenesis. Cell Tissue Res. Article PubMed Google Scholar Caduff, J. H., Fischer, L. C. & Burri, P. H. Scanning electron microscope study of the developing microvasculature in the postnatal rat lung. Google Scholar Grundmann, S., Piek, J. J., Pasterkamp, G. & Hoefer, I. E. Arteriogenesis: basic mechanisms and therapeutic stimulation. Google Scholar Carmeliet, P. Mechanisms of angiogenesis and arteriogenesis. Google Scholar Giacca, M. & Zacchigna, S. VEGF gene therapy: therapeutic angiogenesis in the clinic and beyond. Gene Ther. Google Scholar Henry, T. D. et al. The VIVA trial. Google Scholar Simón-Yarza, T. et al. Vascular endothelial growth factor-delivery systems for cardiac repair: an overview. Google Scholar Darden, J., Payne, L. B., Zhao, H. & Chappell, J. C. Excess vascular endothelial growth factor-A disrupts pericyte recruitment during blood vessel formation. Google Scholar Blatchley, M. R., Hall, F., Wang, S., Pruitt, H. C. & Gerecht, S. Hypoxia and matrix viscoelasticity sequentially regulate endothelial progenitor cluster-based vasculogenesis. This study describes the mechanism through which hypoxia and the material properties of the matrix impact vasculogenesis. Google Scholar Longchamp, A. et al. Amino acid restriction triggers angiogenesis via GCN2/ATF4 regulation of VEGF and H2S production. Google Scholar Fiedler, J. et al. Functional microRNA library screening identifies the hypoxaMiR MiR-24 as a potent regulator of smooth muscle cell proliferation and vascularization. Redox Signal. Google Scholar Dou, C. et al. Graphene-based microRNA transfection blocks preosteoclast fusion to increase bone formation and vascularization. Google Scholar Li, Y. et al. Injectable hydrogel with MSNs/microRNA-21-5p delivery enables both immunomodification and enhanced angiogenesis for myocardial infarction therapy in pigs. Google Scholar Lamichhane, T. N., Leung, C. A., Douti, L. Y. & Jay, S. M. Ethanol induces enhanced vascularization bioactivity of endothelial cell-derived extracellular vesicles via regulation of microRNAs and long non-coding RNAs. Google Scholar Welten, S. M. J. et al. Inhibition of 14q32 microRNAs miR-329, miR-487b, miR-494, and miR-495 increases neovascularization and blood flow recovery after ischemia. Google Scholar Liu, B. et al. Cardiac recovery via extended cell-free delivery of extracellular vesicles secreted by cardiomyocytes derived from induced pluripotent stem cells. Google Scholar Kawamoto, A. et al. Intramyocardial transplantation of autologous endothelial progenitor cells for therapeutic neovascularization of myocardial ischemia. Article PubMed Google Scholar Losordo, D. W. et al. Intramyocardial transplantation of autologous CD34+ stem cells for intractable angina. Article PubMed Google Scholar Calderon, G. A. et al. Tubulogenesis of co-cultured human iPS-derived endothelial cells and human mesenchymal stem cells in fibrin and gelatin methacrylate gels. Google Scholar Moon, J. J. et al. Biomimetic hydrogels with pro-angiogenic properties. Google Scholar Pedowitz, N. J., Batt, A. R., Darabedian, N. & Pratt, M. R. MYPT1 O-GlcNAc modification regulates sphingosine-1-phosphate mediated contraction. This study details how a particular post-translational modification affects diabetic wound healing. Google Scholar Nadkarni, S. et al. Neutrophils induce proangiogenic T cells with a regulatory phenotype in pregnancy. Natl Acad. Google Scholar Mor, F., Quintana, F. J. & Cohen, I. R. Angiogenesis–inflammation cross-talk: vascular endothelial growth factor is secreted by activated T cells and induces Th1 polarization. Google Scholar Marek, N. et al. Increased spontaneous production of VEGF by CD4+ T cells in type 1 diabetes. Google Scholar Ribatti, D. & Crivellato, E. Immune cells and angiogenesis. Google Scholar Graney, P. L. et al. Macrophages of diverse phenotypes drive vascularization of engineered tissues. Google Scholar Fleischer, S., Tavakol, D. N. & Vunjak-Novakovic, G. From arteries to capillaries: approaches to engineering human vasculature. Google Scholar Zhao, S. et al. Application of stem cells in engineered vascular graft and vascularized organs. Cell Dev. Google Scholar Kumar, A. H. S. & Caplice, N. M. Clinical potential of adult vascular progenitor cells. Google Scholar Bayraktutan, U. Endothelial progenitor cells: potential novel therapeutics for ischaemic stroke. Google Scholar Nowbar, A. N. et al. Discrepancies in autologous bone marrow stem cell trials and enhancement of ejection fraction (DAMASCENE): weighted regression and meta-analysis. Google Scholar Krawiec, J. T. & Vorp, D. A. Adult stem cell-based tissue engineered blood vessels: a review. Google Scholar Galat, V. et al. Transgene reactivation in induced pluripotent stem cell derivatives and reversion to pluripotency of induced pluripotent stem cell-derived mesenchymal stem cells. Stem Cells Dev. Google Scholar Hentze, H. et al. Teratoma formation by human embryonic stem cells: evaluation of essential parameters for future safety studies. Stem Cell Res. Article PubMed Google Scholar Quint, C. et al. Decellularized tissue-engineered blood vessel as an arterial conduit. Natl Acad. Google Scholar Neff, L. P. et al. Vascular smooth muscle enhances functionality of tissue-engineered blood vessels in vivo. Article PubMed Google Scholar Kaushal, S. et al. Functional small-diameter neovessels created using endothelial progenitor cells expanded ex vivo. Google Scholar He, H., Shirota, T., Yasui, H. & Matsuda, T. Canine endothelial progenitor cell-lined hybrid vascular graft with nonthrombogenic potential. Article PubMed Google Scholar Zhu, C. et al. Development of anti-atherosclerotic tissue-engineered blood vessel by A20-regulated endothelial progenitor cells seeding decellularized vascular matrix. Google Scholar Yang, L. et al. Human cardiovascular progenitor cells develop from a KDR+ embryonic-stem-cell-derived population. Google Scholar Bu, L. et al. Human ISL1 heart progenitors generate diverse multipotent cardiovascular cell lineages. Google Scholar Ferreira, L. S. et al. Vascular progenitor cells isolated from human embryonic stem cells give rise to endothelial and smooth muscle like cells and form vascular networks in vivo. Google Scholar Levenberg, S., Ferreira, L. S., Chen-Konak, L., Kraehenbuehl, T. P. & Langer, R. Isolation, differentiation and characterization of vascular cells derived from human embryonic stem cells. Google Scholar Hill, K. L. et al. Human embryonic stem cell-derived vascular progenitor cells capable of endothelial and smooth muscle cell function. Google Scholar Levenberg, S., Golub, J. S., Amit, M., Itskovitz-Eldor, J. & Langer, R. Endothelial cells derived from human embryonic stem cells. Natl Acad. Google Scholar Cheung, C. & Sinha, S. Human embryonic stem cell-derived vascular smooth muscle cells in therapeutic neovascularisation. Google Scholar Sone, M. et al. Pathway for differentiation of human embryonic stem cells to vascular cell components and their potential for vascular regeneration. Google Scholar Taura, D. et al. Induction and isolation of vascular cells from human induced pluripotent stem cells—brief report. Google Scholar Patsch, C. et al. Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells. Cell Biol. Google Scholar Sundaram, S., Echter, A., Sivarapatna, A., Qiu, C. & Niklason, L. Small-diameter vascular graft engineered using human embryonic stem cell-derived mesenchymal cells. Tissue Eng. Google Scholar Wang, L. et al. Fabrication of tissue-engineered vascular grafts with stem cells and stem cell-derived vascular cells. Expert Opin. Article PubMed Google Scholar Jang, S., Collin de l'Hortet, A. & Soto-Gutierrez, A. Induced pluripotent stem cell-derived endothelial cells: overview, current advances, applications, and future directions. Google Scholar Margariti, A. et al. Direct reprogramming of fibroblasts into endothelial cells capable of angiogenesis and reendothelialization in tissue-engineered vessels. Natl Acad. Google Scholar Shen, M., Quertermous, T., Fischbein, M. P. & Wu, J. C. Generation of vascular smooth muscle cells from induced pluripotent stem cells: methods, applications, and considerations. Google Scholar Generali, M. et al. Autologous endothelialized small-caliber vascular grafts engineered from blood-derived induced pluripotent stem cells. Acta Biomater. Google Scholar Abaci, H. E. et al. Human skin constructs with spatially controlled vasculature using primary and iPSC-derived endothelial cells. Google Scholar Ciampi, O. et al. Engineering the vasculature of decellularized rat kidney scaffolds using human induced pluripotent stem cell-derived endothelial cells. Google Scholar Wang, Y. Y. et al. Engineering vascular tissue with functional smooth muscle cells derived from human iPS cells and nanofibrous scaffolds. Google Scholar Luo, J. et al. Tissue-engineered vascular grafts with advanced mechanical strength from human iPSCs. Google Scholar Gui, L. et al. Implantable tissue-engineered blood vessels from human induced pluripotent stem cells. Google Scholar Atchison, L. et al. iPSC-derived endothelial cells affect vascular function in a tissue-engineered blood vessel model of Hutchinson–Gilford progeria syndrome. Stem Cell Rep. 14, 325–337 (2020). Google Scholar Owens, G. K., Kumar, M. S. & Wamhoff, B. R. Molecular regulation of vascular smooth muscle cell differentiation in development and disease. Google Scholar Majesky, M. W. Developmental basis of vascular smooth muscle diversity. Google Scholar Turner, M. et al. Toward the development of a global induced pluripotent stem cell library. Google Scholar Wilmut, I. et al. Development of a global network of induced pluripotent stem cell haplobanks. Google Scholar Lee, S. et al. Repurposing the cord blood bank for haplobanking of HLA-homozygous iPSCs and their usefulness to multiple populations. Stem Cells 36, 1552–1566 (2018). Google Scholar Hu, X. et al. Hypoimmune induced pluripotent stem cells survive long term in fully immunocompetent, allogeneic rhesus macaques. Google Scholar Gornalusse, G. G. et al. HLA-E-expressing pluripotent stem cells escape allogeneic responses and lysis by NK cells. Google Scholar Deuse, T. et al. Hypoimmunogenic derivatives of induced pluripotent stem cells evade immune rejection in fully immunocompetent allogeneic recipients. Google Scholar Neve, A., Cantatore, F. P., Maruotti, N., Corrado, A. & Ribatti, D. Extracellular matrix modulates angiogenesis in physiological and pathological conditions. Google Scholar Senger, D. R. & Davis, G. E. Angiogenesis. Cold Spring Harb. Google Scholar Stupack, D. G. & Cheresh, D. A. ECM remodeling regulates angiogenesis: endothelial integrins look for new ligands. Article PubMed Google Scholar Ouyang, L. et al. MMP‑sensitive PEG hydrogel modified with RGD promotes bFGF, VEGF and EPC‑mediated angiogenesis. Google Scholar Nemati, S. et al. Alginate–gelatin encapsulation of human endothelial cells promoted angiogenesis in in vivo and in vitro milieu. Google Scholar Li, Z. et al. Injectable gelatin derivative hydrogels with sustained vascular endothelial growth factor release for induced angiogenesis. Acta Biomater. Google Scholar Seo, Y., Jung, Y. & Kim, S. H. Decellularized heart ECM hydrogel using supercritical carbon dioxide for improved angiogenesis. Acta Biomater. Google Scholar Griffin, M. E., Sorum, A. W., Miller, G. M., Goddard, W. A. & Hsieh-Wilson, L. C. Sulfated glycans engage the Ang–Tie pathway to regulate vascular development. Google Scholar Jia, J. et al. Evolutionarily conserved sequence motif analysis guides development of chemically defined hydrogels for therapeutic vascularization. Google Scholar Ruehle, M. A. et al. Extracellular matrix compression temporally regulates microvascular angiogenesis. Google Scholar Mandrycky, C., Hadland, B. & Zheng, Y. 3D curvature-instructed endothelial flow response and tissue vascularization. This study quantifies the effect of vessel geometry and perfusion-related forces on endothelial cells. Google Scholar Lee, J. et al. Mechanobiological conditioning of mesenchymal stem cells for enhanced vascular regeneration. Google Scholar Wolf, K. J., Weiss, J. D., Uzel, S. G. M., Skylar-Scott, M. A. & Lewis, J. A. Biomanufacturing human tissues via organ building blocks. Google Scholar Munarin, F., Kant, R. J., Rupert, C. E., Khoo, A. & Coulombe, K. L. K. Engineered human myocardium with local release of angiogenic proteins improves vascularization and cardiac function in injured rat hearts. Google Scholar Nih, L. R., Gojgini, S., Carmichael, S. T. & Segura, T. Dual-function injectable angiogenic biomaterial for the repair of brain tissue following stroke. Google Scholar Lee, A. S. et al. Prolonged survival of transplanted stem cells after ischaemic injury via the slow release of pro-survival peptides from a collagen matrix. Google Scholar Aday, S. et al. Synthetic microparticles conjugated with VEGF165 improve the survival of endothelial progenitor cells via microRNA-17 inhibition. Google Scholar Coindre, V. F., Carleton, M. M. & Sefton, M. V. Methacrylic acid copolymer coating enhances constructive remodeling of polypropylene mesh by increasing the vascular response. Google Scholar Coindre, V. F., Kinney, S. M. & Sefton, M. V. Methacrylic acid copolymer coating of polypropylene mesh chamber improves subcutaneous islet engraftment. Google Scholar Yu, Y. et al. Sulfated polysaccharide directs therapeutic angiogenesis via endogenous VEGF secretion of macrophages. Google Scholar Li, X. et al. Nanofiber-hydrogel composite-mediated angiogenesis for soft tissue reconstruction. Google Scholar Mazio, C. et al. Pre-vascularized dermis model for fast and functional anastomosis with host vasculature. Google Scholar Roberts, S. et al. Injectable tissue integrating networks from recombinant polypeptides with tunable order. This study describes the creation of partially ordered peptides for vascularizing materials with tunable properties on the basis of ordered/disordered domains. Google Scholar Wang, L. et al. Development of a centrally vascularized tissue engineering bone graft with the unique core-shell composite structure for large femoral bone defect treatment. Google Scholar Dor, Y. Conditional switching of VEGF provides new insights into adult neovascularization and pro-angiogenic therapy. EMBO J. Google Scholar Tafuro, S. et al. Inducible adeno-associated virus vectors promote functional angiogenesis in adult organisms via regulated vascular endothelial growth factor expression. Google Scholar Ozawa, C. R. et al. Microenvironmental VEGF concentration, not total dose, determines a threshold between normal and aberrant angiogenesis. Google Scholar Ozaki, H. et al. Intravitreal sustained release of VEGF causes retinal neovascularization in rabbits and breakdown of the blood–retinal barrier in rabbits and primates. Eye Res. Google Scholar Murphy, W. L., Peters, M. C., Kohn, D. H. & Mooney, D. J. Sustained release of vascular endothelial growth factor from mineralized poly(lactide-co-glycolide) scaffolds for tissue engineering. Google Scholar Lee, K. W. et al. Sustained release of vascular endothelial growth factor from calcium-induced alginate hydrogels reinforced by heparin and chitosan. Google Scholar Zisch, A. H. et al. Cell-demanded release of VEGF from synthetic, biointeractive cell-ingrowth matrices for vascularized tissue growth. FASEB J. Google Scholar Seliktar, D., Zisch, A. H., Lutolf, M. P., Wrana, J. L. & Hubbell, J. A. MMP-2 sensitive, VEGF-bearing bioactive hydrogels for promotion of vascular healing. Google Scholar Zisch, A. H., Schenk, U., Schense, J. C., Sakiyama-Elbert, S. E. & Hubbell, J. A. Covalently conjugated VEGF–fibrin matrices for endothelialization. Google Scholar Moulisová, V. et al. Engineered microenvironments for synergistic VEGF–integrin signalling during vascularization. Google Scholar Lee, T. T. et al. Light-triggered in vivo activation of adhesive peptides regulates cell adhesion, inflammation and vascularization of biomaterials. Google Scholar Brady, A. C. et al. Proangiogenic hydrogels within macroporous scaffolds enhance islet engraftment in an extrahepatic site. Tissue Eng. Google Scholar Liang, J. P., Accolla, R. P., Jiang, K., Li, Y. & Stabler, C. L. Controlled release of anti-inflammatory and proangiogenic factors from macroporous scaffolds. Tissue Eng. Google Scholar Chen, T. T. et al. Anchorage of VEGF to the extracellular matrix conveys differential signaling responses to endothelial cells. Cell Biol. Google Scholar Yin, N. et al. VEGF-conjugated alginate hydrogel prompt angiogenesis and improve pancreatic islet engraftment and function in type 1 diabetes. Google Scholar Lee, J., Yang, C., Ahn, S., Choi, Y. & Lee, K. Enhanced NO-induced angiogenesis via NO/H2S co-delivery from self-assembled nanoparticles. Google Scholar Santulli, G. et al. In vivo properties of the proangiogenic peptide QK. Google Scholar D'Andrea, L. D. et al. Targeting angiogenesis: structural characterization and biological properties of a de novo engineered VEGF mimicking peptide. Natl Acad. This study describes the creation of a VEGF mimic with vascularizing properties similar to those of the parent protein. Google Scholar Kumar, V. A. et al. Highly angiogenic peptide nanofibers. ACS Nano 9, 860–868 (2015). This study describes the creation of an angiogenic self-assembling peptide–hydrogel with a VEGF mimic in the primary sequence. Google Scholar Kumar, V. A. et al. Treatment of hind limb ischemia using angiogenic peptide nanofibers. Google Scholar Moore, A. N. et al. Nanofibrous peptide hydrogel elicits angiogenesis and neurogenesis without drugs, proteins, or cells. Google Scholar Lopez-Silva, T. L. et al. Chemical functionality of multidomain peptide hydrogels governs early host immune response. Google Scholar Carrejo, N. C. et al. Multidomain peptide hydrogel accelerates healing of full-thickness wounds in diabetic mice. ACS Biomater. Google Scholar Lopez-Silva, T. L. et al. Self-assembling multidomain peptide hydrogels accelerate peripheral nerve regeneration after crush injury. Google Scholar Lai, C. S. E. et al. A combined conduit-bioactive hydrogel approach for regeneration of transected sciatic nerves. ACS Appl. Bio Mater. Google Scholar Howard, D., Buttery, L. D., Shakesheff, K. M. & Roberts, S. J.Tissue engineering: strategies, stem cells and scaffolds. Google Scholar Kinstlinger, I. S. & Miller, J. S. 3D-printed fluidic networks as vasculature for engineered tissue. Lab Chip 16, 2025–2043 (2016). Google Scholar Sarker, M. D., Naghieh, S., Sharma, N. K. & Chen, X. 3D biofabrication of vascular networks for tissue regeneration: a report on recent advances. Google Scholar Devillard, C. D. & Marquette, C. A. Vascular tissue engineering: challenges and requirements for an ideal large scale blood vessel. Google Scholar Hynes, W. F. et al. Examining metastatic behavior within 3D bioprinted vasculature for the validation of a 3D computational flow model. Google Scholar Kinstlinger, I. S. et al. Generation of model tissues with dendritic vascular networks via sacrificial laser-sintered carbohydrate templates. Google Scholar Papaioannou, T. G. & Stefanadis, C. Vascular wall shear stress: basic principles and methods. Google Scholar Yilmaz, B., Al Rashid, A., Mou, Y. A., Evis, Z. & Koç, M. Bioprinting: a review of processes, materials and applications. Google Scholar Emerson, A. E., McCall, A. B., Brady, S. R., Slaby, E. M. & Weaver, J. D. Hydrogel injection molding to generate complex cell encapsulation geometries. ACS Biomater. Google Scholar Skylar-Scott, M. A. et al. Biomanufacturing of organ-specific tissues with high cellular density and embedded vascular channels. This study describes a method for rapidly creating vascular networks between densely packed cellular aggregates, for tissue engineering applications. Google Scholar Lee, A. et al. 3D bioprinting of collagen to rebuild components of the human heart. Fabricates large-scale tissue engineering scaffolds using extrusion 3D printing. Google Scholar Grigoryan, B. et al. Multivascular networks and functional intravascular topologies within biocompatible hydrogels. This study describes the use of 3D printing to fabricate high-resolution structures using biocompatible materials. Google Scholar Kelly, B. E. et al. Volumetric additive manufacturing via tomographic reconstruction. Google Scholar Regehly, M. et al. Xolography for linear volumetric 3D printing. Google Scholar Anandakrishnan, N. et al. Fast stereolithography printing of large-scale biocompatible hydrogel models. Google Scholar Galarraga, J. H., Kwon, M. Y. & Burdick, J. 3D bioprinting via an in situ crosslinking technique towards engineering cartilage tissue. Google Scholar Ouyang, L., Highley, C. B., Sun, W. & Burdick, J. A generalizable strategy for the 3D bioprinting of hydrogels from nonviscous photo-crosslinkable inks. Google Scholar Datta, P., Dey, M., Ataie, Z., Unutmaz, D. & Ozbolat, I. T. 3D bioprinting for reconstituting the cancer microenvironment. npj Precis. Google Scholar Ma, X. et al. Deterministically patterned biomimetic human iPSC-derived hepatic model via rapid 3D bioprinting. Natl Acad. Google Scholar Skylar-Scott, M. A., Mueller, J., Visser, C. W. & Lewis, J. A. Voxelated soft matter via multimaterial multinozzle 3D printing. Google Scholar Shiwarski, D. J., Hudson, A. R., Tashman, J. W. & Feinberg, A. W. Emergence of FRESH 3D printing as a platform for advanced tissue biofabrication. APL Bioeng. Google Scholar Gao, G. et al. Construction of a novel in vitro atherosclerotic model from geometry-tunable artery equivalents engineered via in-bath coaxial cell printing. Google Scholar Cho, W. W., Ahn, M., Kim, B. S. & Cho, D. W. Blood-lymphatic integrated system with heterogeneous melanoma spheroids via in-bath three-dimensional bioprinting for modelling of combinational targeted therapy. Google Scholar Kim, B. S. et al. Construction of tissue-level cancer-vascular model with high-precision position control via in situ 3D cell printing. Small Methods 5, 2100072 (2021). Google Scholar Singh, N. K. et al. Coaxial cell printing of a human glomerular model: anin vitroglomerular filtration barrier and its pathophysiology. Google Scholar Boularaoui, S., Al Hussein, G., Khan, K. A., Christoforou, N. & Stefanini, C. An overview of extrusion-based bioprinting with a focus on induced shear stress and its effect on cell viability. Google Scholar Wang, Z. et al. A simple and high-resolution stereolithography-based 3D bioprinting system using visible light crosslinkable bioinks. Article PubMed Google Scholar Derakhshanfar, S. et al. 3D bioprinting for biomedical devices and tissue engineering: a review of recent trends and advances. Google Scholar Hwang, H. H., Zhu, W., Victorine, G., Lawrence, N. & Chen, S. 3D-printing of functional biomedical microdevices via light- and extrusion-based approaches. Small Methods 2, 1700277 (2018). Article PubMed Google Scholar Ikuta, K. & Hirowatari, K. Real three dimensional micro fabrication using stereo lithography and metal molding. In Proc. IEEE Micro Electro Mechanical Systems 42–47 (IEEE, 2002). Fonseca, A. C. et al. Emulating human tissues and organs: a bioprinting perspective toward personalized medicine. PubMed Central Google Scholar Warner, J., Soman, P., Zhu, W., Tom, M. & Chen, S. Design and 3D printing of hydrogel scaffolds with fractal geometries. ACS Biomater. Google Scholar Zhu, W. et al. Direct 3D bioprinting of prevascularized tissue constructs with complex microarchitecture. Google Scholar Tomov, M. L. et al. A 3D bioprinted in vitro model of pulmonary artery atresia to evaluate endothelial cell response to microenvironment. Google Scholar Bagheri Saed, A. et al. An in vitro study on the key features of poly l-lactic acid/biphasic calcium phosphate scaffolds fabricated via DLP 3D printing for bone grafting. Google Scholar Jiang, P. et al. Grayscale stereolithography of gradient hydrogel with site-selective shape deformation. Google Scholar You, S. et al. Mitigating scattering effects in light-based three-dimensional printing using machine learning. Google Scholar Bernal, P. N. et al. Volumetric bioprinting of organoids and optically tuned hydrogels to build liver-like metabolic biofactories. Google Scholar You, S. et al. High cell density and high-resolution 3D bioprinting for fabricating vascularized tissues. Google Scholar Seymour, A. J., Westerfield, A. D., Cornelius, V. C., Skylar-Scott, M. A. & Heilshorn, S. C. Bioprinted microvasculature: progressing from structure to function. This review provides a comprehensive overview of the challenges and strategies surrounding the 3D printing of vasculature. Google Scholar Urciuolo, A. et al. Hydrogel-in-hydrogel live bioprinting for guidance and control of organoids and organotypic cultures. Betz, C., Lenard, A., Belting, H. G. & Affolter, M. Cell behaviors and dynamics during angiogenesis. Labarrere, C. A., Dabiri, A. E. & Kassab, G. S. Thrombogenic and inflammatory reactions to biomaterials in medical devices. Zheng, Y. et al. In vitro microvessels for the study of angiogenesis and thrombosis. Natl Acad. Kelly, B. E. et al. Computed axial lithography (CAL): toward single step 3D printing of arbitrary geometries. Preprint at https://arxiv.org/abs/1705.05893 (2017). Hutson, C. B. et al. Synthesis and characterization of tunable poly(ethylene glycol): gelatin methacrylate composite hydrogels. Tissue Eng. Ye, W. et al. 3D printing of gelatin methacrylate-based nerve guidance conduits with multiple channels. Zhang, B. et al. Strengths, weaknesses, and applications of computational axial lithography in tissue engineering. Hiob, M. A., She, S., Muiznieks, L. D. & Weiss, A. S. Biomaterials and modifications in the development of small-diameter vascular grafts. ACS Biomater. Ravi, S. & Chaikof, E. L. Biomaterials for vascular tissue engineering. Jang, T. S. et al. 3D printing of hydrogel composite systems: recent advances in technology for tissue engineering. Akentjew, T. L. et al. Rapid fabrication of reinforced and cell-laden vascular grafts structurally inspired by human coronary arteries. Pei, B. et al. Fiber-reinforced scaffolds in soft tissue engineering. Wang, Z. et al. 3D-printable self-healing and mechanically reinforced hydrogels with host–guest non-covalent interactions integrated into covalently linked networks. Zhang, J. et al. 3D printing of silk particle-reinforced chitosan hydrogel structures and their properties. ACS Biomater. Cellular & gene therapy guidances, FDA https://www.fda.gov/vaccines-blood-biologics/biologics-guidances/cellular-gene-therapy-guidances (2024). FDA https://www.fda.gov/vaccines-blood-biologics/cellular-gene-therapy-products/laviv (2018). Schurr, M. J. et al. Phase I/II clinical evaluation of StrataGraft: a consistent, pathogen-free human skin substitute. Grigoryan, B. et al. Development, characterization, and applications of multi-material stereolithography bioprinting. Camasão, D. B. & Mantovani, D. The mechanical characterization of blood vessels and their substitutes in the continuous quest for physiological-relevant performances. A critical review. Today Bio 10, 100106 (2021). Erbel, R. & Eggebrecht, H. Aortic dimensions and the risk of dissection. Keelan, J. & Hague, J. P. The role of vascular complexity on optimal junction exponents. Sherman, T. F. On connecting large vessels to small. The meaning of Murray's law. Painter, P. R., Edén, P. & Bengtsson, H. U. Pulsatile blood flow, shear force, energy dissipation and Murray's law. Morawietz, H. et al. Regulation of the endothelin system by shear stress in human endothelial cells. Taylor, D. J. et al. Refining our understanding of the flow through coronary artery branches; revisiting Murray's law in human epicardial coronary arteries. Adam, J. Blood vessel branching: beyond the standard calculus problem. Xu, J. & Shi, G. P. Vascular wall extracellular matrix proteins and vascular diseases. Tucker, W. D., Arora, Y. & Mahajan, K. Anatomy, blood vessels. in StatPearls [Internet] (StatPearls Publishing, 2023); https://pubmed.ncbi.nlm.nih.gov/29262226/ Russo, T. A., Banuth, A. M. M., Nader, H. B. & Dreyfuss, J. L. Altered shear stress on endothelial cells leads to remodeling of extracellular matrix and induction of angiogenesis. PLoS ONE 15, e0241040 (2020). Kubota, Y., Kleinman, H. K., Martin, G. R. & Lawley, T. J. Role of laminin and basement membrane in the morphological differentiation of human endothelial cells into capillary-like structures. Cell Biol. Kalebic, T., Garbisa, S., Glaser, B. & Liotta, L. A. Basement membrane collagen: degradation by migrating endothelial cells. Marieb, E. N. Essentials of Human Anatomy & Physiology 11th edn (Pearson, 2015). Courtney, J. M. & Sutherland, B. Harnessing the stem cell properties of pericytes to repair the brain. Neural Regen. Article PubMed Daneman, R. & Prat, A. The blood–brain barrier. Cold Spring Harb. Hanrahan, V. et al. The angiogenic switch for vascular endothelial growth factor (VEGF)-A, VEGF-B, VEGF-C, and VEGF-D in the adenoma–carcinoma sequence during colorectal cancer progression. Holmes, D. I. R. & Zachary, I. The vascular endothelial growth factor (VEGF) family: angiogenic factors in health and disease. Genome Biol. Chau, K., Hennessy, A. & Makris, A. Placental growth factor and pre-eclampsia. Baldwin, M. E. et al. Vascular endothelial growth factor D is dispensable for development of the lymphatic system. Zhang, F. et al. VEGF-B is dispensable for blood vessel growth but critical for their survival, and VEGF-B targeting inhibits pathological angiogenesis. Natl Acad. Achen, M. G. et al. Vascular endothelial growth factor D (VEGF-D) is a ligand for the tyrosine kinases VEGF receptor 2 (Flk1) and VEGF receptor 3 (Flt4). Natl Acad. Duffy, A. M., Bouchier-Hayes, D. J. & Harmey, J. H. Vascular endothelial growth factor (VEGF) and its role in non-endothelial cells: autocrine signalling by VEGF. in Madame Curie Bioscience Database [Internet] (Landes Bioscience, 2013); https://www.ncbi.nlm.nih.gov/books/NBK6482/ Bates, D. O. Vascular endothelial growth factors and vascular permeability. Thurston, G. Complementary actions of VEGF and angiopoietin-1 on blood vessel growth and leakage. Krilleke, D., Ng, Y.-S. E. & Shima, D. T. The heparin-binding domain confers diverse functions of VEGF-A in development and disease: a structure–function study. Park, J. E., Keller, G. A. & Ferrara, N. The vascular endothelial growth factor (VEGF) isoforms: differential deposition into the subepithelial extracellular matrix and bioactivity of extracellular matrix-bound VEGF. Ruhrberg, C. et al. Spatially restricted patterning cues provided by heparin-binding VEGF-A control blood vessel branching morphogenesis. Genes Dev. Catena, R. et al. Increased expression of VEGF121/VEGF165–189 ratio results in a significant enhancement of human prostate tumor angiogenesis. Roskoski, R. VEGF receptor protein–tyrosine kinases: structure and regulation. Takahashi, T., Ueno, H. & Shibuya, M. VEGF activates protein kinase C-dependent, but Ras-independent Raf–MEK–MAP kinase pathway for DNA synthesis in primary endothelial cells. Eliceiri, B. P. et al. Selective requirement for Src kinases during VEGF-induced angiogenesis and vascular permeability. Chen, D. & Simons, M. Emerging roles of PLCγ1 in endothelial biology. Perrin, R. M. et al. Diabetic retinopathy is associated with a switch in splicing from anti- to pro-angiogenic isoforms of vascular endothelial growth factor. Download references We thank J. S. Miller for guidance and helpful discussions. We acknowledge funding from the Defense Advanced Research Projects Agency under agreement numbers FA8650-21-1-7119 and AWD00001596, the Advanced Research Projects Agency for Health under award number AY1AX000003, and Breakthrough T1D awards 3-SRA-2025-1640-S-B, 3-SRA-2022-1255-S-B, 3-SRA-2023-1398-S-B, 3-SRA-2021-1023-S-B and 3-SRA-2024-1564-S-B. These authors contributed equally: Kevin D. Janson, Siavash Parkhideh. Department of Bioengineering, Rice University, Houston, TX, USA Kevin D. Janson, Siavash Parkhideh, Jeffrey D. Hartgerink & Omid Veiseh Department of Chemistry, Rice University, Houston, TX, USA Joseph W. R. Swain & Jeffrey D. Hartgerink School of Biological and Health Systems Engineering, Arizona State University, Tempe, AZ, USA Jessica D. Weaver Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed equally to researching, drafting and editing the majority of the paper and figures. researched and drafted numerous sections that discuss chemistry-based approaches for encouraging vascularization in vivo. provided valuable additions to the text and performed the final editing. conceptualized the paper and supervised the writing. Correspondence to Omid Veiseh. is a co-founder of and holds equity in ImmunoShield Therapeutics. The other authors declare no competing interests. Nature Biomedical Engineering thanks Shaochen Chen, Milica Radisic and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Janson, K.D., Parkhideh, S., Swain, J.W.R. et al. Strategies for the vascular patterning of engineered tissues for organ repair. Download citation Received: 31 October 2022 Accepted: 07 May 2025 Published: 20 June 2025 DOI: https://doi.org/10.1038/s41551-025-01420-w Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biomedical Engineering ISSN 2157-846X (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-does-a-gravitational-slingshot-work/'>Here's How Interplanetary Missions Slingshot around the Solar System</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Spacecraft can get a significant boost by stealing energy from planets This artist's concept shows an example of a gravitational assist maneuver, used by spacecraft to travel through the solar system. In this example, the spacecraft flies close by Mars, then by Jupiter, each time receiving a boost that allows it to ultimately reach Saturn. Stay connected to The Universe: Get email alerts for this weekly column by Phil Plait You've probably watched this sort of science-fiction scene more than once: some stalwart starship captain and their crew are fleeing from aliens/escaping a supernova /running out of fuel and are seemingly out of options, about to get eaten/vaporized/stuck. But then, just ahead, they spot a planet! So they head right for it, rockets blazing, then dive down and use its gravity to slingshot to safety. So it goes on the silver screen, at least. But does this maneuver work in real life? It's widely known as a gravitational slingshot, though most scientists refer to it as a gravitational assist, and it's an essential tool for most interplanetary missions. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. But there's more to it than that: the spacecraft can actually use the planet's gravity to speed up or slow down after this maneuver, allowing easier voyages to the outer or inner planets, respectively. It's related to the symmetry of gravity. If you hold a rubber ball some distance from the ground and drop it, the ball will accelerate as it falls, speeding up until impact. But either way, it can't bounce any higher than the height from which you dropped it. This action is symmetric, so at best (if you had a perfectly elastic ball and did this experiment in a vacuum), it would bounce to the same height from which you dropped it. So if all the bonus speed is lost on the way out, how can this maneuver be used to accelerate a spacecraft? But that's your speed measured against the planet. If you approach the planet from behind (that is, in the direction of its motion), then, as the planet's gravity gives you a boost, it also, in a heliocentric sense, pulls you along, adding some of its orbital velocity to yours. That gives you a kick relative to the sun, speeding you up on your way to your destination. In essence, the spacecraft gets a net gain in speed by stealing a little bit of the planet's orbital kinetic energy. In turn, this means the planet actually slows a bit in its orbit around the sun—which sounds dangerous! But fear not: the planet slows in proportion to how much more massive it is than the spacecraft. Given a typical one-ton probe compared with a multisextillion-ton world, the planet doesn't noticeably slow at all. You could launch a million probes at it and never be able to tell the difference in its orbital speed. For our current rocketry, these speeds are so low and the interplanetary distances so great that even the fastest, most direct voyages take years (or even decades for destinations in the outer solar system). This catch-22 is described by what is called the rocket equation, and it means the amount of fuel you must add to move even slightly faster reaches prohibitive scales very quickly. So shaving time off your voyage requires some other method—such as siphoning speed from a big, juicy planet along the way! For example, the Cassini probe to Saturn, which launched in 1997, was a huge spacecraft, the size of a school bus, and had a mass of 2.5 metric tons without fuel. In fact, just to get out to Jupiter in the first place, Cassini also performed two fuel-saving flybys of Venus and one of Earth, stealing planetary orbital energy every time. A gravitational assist works the other way, too. Earth orbits the sun at more than 30 km/s, so firing a probe at the sun or the inner planets is extremely hard because of all that sideways velocity. Instead mission planners prefer a more circuitous route. BepiColombo, a joint European Space Agency and Japan Aerospace Exploration Agency mission to Mercury, did exactly this, passing Earth once and Venus twice to get in Mercury's vicinity. Even then, it had to do a total of six gravity assists past Mercury to match the planet's orbital speed around the sun. Gravitational assists are an emblematic example of why space travel is hard—it is exactly rocket science, after all. It's ironic, then, that gravity can make reaching most of the rest of the solar system so much easier. Phil Plait is a professional astronomer and science communicator in Virginia. His column for Scientific American, The Universe, covers all things space. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250619090850.htm'>How life endured the Snowball Earth: Evidence from Antarctic meltwater ponds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-20 05:27:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When the Earth froze over, where did life shelter? MIT scientists say one refuge may have been pools of melted ice that dotted the planet's icy surface. These small, watery oases may have persisted atop relatively shallow ice sheets present in equatorial regions. There, the ice surface could accumulate dark-colored dust and debris from below, which enhanced its ability to melt into pools. At temperatures hovering around 0 degrees Celsius, the resulting meltwater ponds could have served as habitable environments for certain forms of early complex life. The team drew its conclusions based on an analysis of modern-day meltwater ponds. The conditions along these polar ice sheets are similar to what likely existed along ice sheets near the equator during Snowball Earth. The researchers analyzed samples from a variety of meltwater ponds located on the McMurdo Ice Shelf in an area that was first described by members of Robert Falcon Scott's 1903 expedition as "dirty ice." The MIT researchers discovered clear signatures of eukaryotic life in every pond. The team also found that salinity plays a key role in the kind of life a pond can host: Ponds that were more brackish or salty had more similar eukaryotic communities, which differed from those in ponds with fresher waters. "We've shown that meltwater ponds are valid candidates for where early eukaryotes could have sheltered during these planet-wide glaciation events," says lead author Fatima Husain, a graduate student in MIT's Department of Earth, Atmospheric and Planetary Sciences (EAPS). "This shows us that diversity is present and possible in these sorts of settings. The question has been: How and where did life survive? "We're interested in understanding the foundations of complex life on Earth. We see evidence for eukaryotes before and after the Cryogenian in the fossil record, but we largely lack direct evidence of where they may have lived during," Husain says. "The great part of this mystery is, we know life survived. There are a number of ideas for where organisms could have sheltered during Snowball Earth, including in certain patches of the open ocean (if such environments existed), in and around deep-sea hydrothermal vents, and under ice sheets. In considering meltwater ponds, Husain and her colleagues pursued the hypothesis that surface ice meltwaters may also have been capable of supporting early eukaryotic life at the time. "There are many hypotheses for where life could have survived and sheltered during the Cryogenian, but we don't have excellent analogs for all of them," Husain notes. "Above-ice meltwater ponds occur on Earth today and are accessible, giving us the opportunity to really focus in on the eukaryotes which live in these environments." For their new study, the researchers analyzed samples taken from meltwater ponds in Antarctica. In 2018, Summons and colleagues from New Zealand traveled to a region of the McMurdo Ice Shelf in East Antarctica, known to host small ponds of melted ice, each just a few feet deep and a few meters wide. Wind-driven loss of ice from the surface creates a sort of conveyer belt that brings this trapped debris to the surface over time, where it absorbs the sun's warmth, causing ice to melt, while surrounding debris-free ice reflects incoming sunlight, resulting in the formation of shallow meltwater ponds. These microbial mats are made up of cyanobacteria, prokaryotic, single-celled photosynthetic organisms that lack a cell nucleus or other organelles. While these ancient microbes are known to survive within some of the the harshest environments on Earth including meltwater ponds, the researchers wanted to know whether eukaryotes -- complex organisms that evolved a cell nucleus and other membrane bound organelles -- could also weather similarly challenging circumstances. To characterize the eukaryotes, the team analyzed the mats for specific lipids they make called sterols, as well as genetic components called ribosomal ribonucleic acid (rRNA), both of which can be used to identify organisms with varying degrees of specificity. These two independent sets of analyses provided complementary fingerprints for certain eukaryotic groups. As part of the team's lipid research, they found many sterols and rRNA genes closely associated with specific types of algae, protists, and microscopic animals among the microbial mats. "There are repeating casts of characters, but they're present in different abundances. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            