
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-13</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69290706/stone-of-destiny-fragments/'>A Researcher Tracked Down the Missing Pieces of the Infamous ‘Stone of Destiny'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Sally Foster tracked down some of the missing fragments. When four Scottish Nationalist students swiped the Stone of Destiny from Westminster Abbey on Christmas morning in 1950, the clandestine removal didn't exactly go off without a hitch. A repair at the time, overseen by stonemason and politician Robert Gray, created a collection of fragments that Gray then gifted out to family, friends, and influential figures. A new study published in The Antiquaries Journal has tracked some of those fragments, telling their stories for the first time. Only one fragment was officially recognized before Foster started her research, but she was able to trail other pieces to politicians and others living on different continents. “Since my findings started to emerge, many members of the public have contacted me with their family's knowledge of credible stone fragments, often accompanied by supporting evidence,” she said in a statement, “but there are many gaps yet to fill.” In 1296, Edward I of England took the stone from Scotland and brought it back to England, where it was given to Westminster Abbey and eventually incorporated into the famous Coronation Chair. With such a storied history, it should be no surprise that its Christmas Day theft in 1950 caused an uproar. “This is not just any stone,” Foster said. But the story of the stone, which has been long and politically charged, has a relatively happy ending. And that new home has opened up the opportunity for new scientific studies including, according to Foster, “the existence and significance of a diverse, dispersed body of small fragments of the stone [which] has been overlooked.” The only previously known fragment and its accompanying letter—a piece that was at one time gifted to Alex Salmond—is owned (as of 2018) by Historic Environment Scotland. The list of original recipients, Foster said, includes Winnie Ewing (who wore a locket containing a chip when interviewed on television in 1967) and Margo MacDonald (another leading politician). Gray also gave one of the pieces to a visiting Australian tourist, and upon her death in 1967, her family donated the fragment and letter to Queensland Museum. “With the likely perception of the fragments as being stolen property,” Foster said, “few people opted to brazenly flaunt and taunt with their possession, except for some politicians. Families cared for them, emotionally and physically, and we can also trace the progression of fragments to valued heirlooms.” Foster sees the stone's fragments as an opportunity. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A Carnivorous ‘Death Ball' Emerged from the Deep Are There More Humans on Earth Than We Thought? Did Scientists Just Bend the Laws of Physics?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a69384091/cybertruck-lightbar-recall/'>Tesla Is Recalling Cybertrucks Again. Yep, More Pieces Are Falling Off.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>That said, even armed with electronic locking differentials and a suite of off-road modes, it's not going to outclass anything like a Ford F-150 Raptor or Ram TRX. But Tesla did offer a dealer-installed lightbar to improve your visibility when the Sun goes down. For most applications outside of the automotive industry, primer is generally used to give you a better-than-surface-level adhesion between two objects. For instance, primer is often used to join PVC joints together. It's a bit like soldering metals together instead of welding them. But according to the National Highway Traffic Safety Administration (NHTSA), it appears that Tesla may have used an incorrect primer, which could be the catalyst for these lightbar detaching results. “Tesla Service will inspect the light bar and install an additional mechanical attachment or replace the light bar using tape to adhere the light bar to the windshield as well as an additional mechanical attachment as necessary, free of charge.” NHTSA also states that all 6,197 owner letters are expected to be mailed on December 26, 2025. However, it's unclear when owners will be able to get their vehicles into a service center for the fix they need. If the Cybertruck's previous glue recall is anything to go by, it could be a matter of months. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. How to Clean Your Headlights and Make Them Shine Las Vegas Police Have a New Fleet of Cybertrucks How Often Should You Really Change Your Car's Oil? How to Get Keys Out of a Locked Car TuxMat Is the Floor Mat Your Vehicle Deserves</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a69367718/boring-billion-complex-life/'>Experts Say a Crumbling Supercontinent May Have Started Life on Earth</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For the past three decades, scientists have been bad-mouthing a sizable chunk of Earth's history (roughly 1.8 billion years ago to 800 million years ago) by giving it nicknames like the “Barren Billion,” the “Boring Billion,” or the Earth's “Middle Ages.” At first glance, the monikers may be warranted—compared to more dynamic eons in Earth's history, this “boring” stretch (which includes the Statherian, Mesoproterozoic, and early Tonian periods) is characterized by relatively stable tectonics, an invariable climate, and generally uneventful progress in biological evolution. But within the past decade, studies have been challenging this dull perception. “For a long time, the boring billion was commonly thought to be remarkably unremarkable,” Timothy Lyons, a geochemist at the University of California Riverside, told Science News in 2015. “But it's a critical chapter in the history of life on Earth, and there are basic questions we don't understand.” Now, a new study, published in the journal Earth and Planetary Science Letters, adds even more evidence to that assertion. To explore the interwoven dynamics of plate tectonics, the global carbon cycle, ocean chemistry, and how all of those factors relate to the rise of eukaryotic life, the team developed a model that simulated 1.8 billion years of moving plate boundaries and continental margins. “Our work reveals that deep Earth processes, specifically the breakup of the ancient supercontinent Nuna, set off a chain of events that reduced volcanic carbon dioxide emissions and expanded the shallow marine habitats where early eukaryotes evolved,” Müller said in a press statement. Around 1.46 billion years ago, Nuna began fragmenting, which resulted in an explosion of shallow continental shelves. Although there wasn't a lot of oxygen to go around, these shallow, temperate seas likely provided the ideal conditions for complex life to take hold. Decreased volcanic outgassing and increased carbon storage in the ocean crust only improved these conditions. “We think these vast continental shelves and shallow seas were crucial ecological incubators,” Juraj Farkaš, a co-author of the study from the University of Adelaide, said in a press statement. Another study from the University of Arizona last year showed how complex life required significantly more energy and oxygen than single-celled protists. Are There More Humans on Earth Than We Thought? The Ocean Is Holding In a Devastating ‘Burp'</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251112111027.htm'>New Neanderthal footprints in Portugal reveal a life we never expected</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 12:04:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An international team of researchers has identified a previously unknown Neanderthal site on the southern edge of the Iberian Peninsula, located along Portugal's Algarve coast. Researchers from institutions in Portugal, Spain, Gibraltar, Italy, Denmark and China also took part. The earliest Neanderthal footprints known from Portugal were found in two Algarve locations: Praia do Monte Clérigo, where the geological layers date to about 78,000 years ago, and Praia do Telheiro, dated to roughly 82,000 years. At Monte Clérigo, researchers identified 5 trackways and 26 individual footprints created by adults and by children slightly over a year old on the slope of what was once a coastal dune. Praia do Telheiro revealed a single footprint attributed to either a teenager or an adult woman, found near fossilized bird tracks that are typical of rocky and coastal habitats. Neanderthal footprints preserve information that other archaeological remains, such as bones or tools, often cannot provide. Imprinted in sediments or sedimentary rock, they record a specific moment of activity and confirm the exact location where an individual stood or moved. Artifacts, in contrast, may be displaced or left behind long after their original use. The footprints show how Neanderthals used space, how they explored coastal environments, forests, dunes or riverbanks, something that is difficult to infer solely from artifacts," explain Neto de Carvalho and Muñiz. Footprints left by infants and very young children, which are typically rare in the archaeological record, add valuable information about family groups and social organization. The trackways also show how Neanderthals moved through different types of terrain, revealing decisions about route selection, proximity to their base camp and possible hunting strategies. These results reveal that Neanderthals living along the Atlantic coast were more flexible and environmentally skilled than previously understood. The study offers a detailed look at their mobility, behavior and social interactions, highlighting a level of ecological and cognitive adaptability that adds new depth to the story of Neanderthal life. Scientists Discover a Diet That Burns Fat Like Cold Exposure, Leading to Significant Weight Loss From Plastic to Pure Water: Scientists Turn Trash Into a Super Catalyst Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01419-2'>Uridine-sensitized screening identifies demethoxy-coenzyme Q and NUDT5 as regulators of nucleotide synthesis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 11:03:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Rapidly proliferating cells require large amounts of nucleotides, making nucleotide metabolism a widely exploited therapeutic target against cancer, autoinflammatory disorders and viral infections. However, regulation of nucleotide metabolism remains incompletely understood. Here, we reveal regulators of de novo pyrimidine synthesis. Using uridine-sensitized CRISPR-Cas9 screening, we show that coenzyme Q (CoQ) is dispensable for pyrimidine synthesis, in the presence of the demethoxy-CoQ intermediate as alternative electron acceptor. We further report that the ADP-ribose pyrophosphatase NUDT5 directly binds PPAT, the rate-limiting enzyme in purine synthesis, which inhibits its activity and preserves the phosphoribosyl pyrophosphate (PRPP) pool. In the absence of NUDT5, hyperactive purine synthesis exhausts the PRPP pool at the expense of pyrimidine synthesis, which promotes resistance to purine and pyrimidine nucleobase analogues. Of note, the interaction between NUDT5 and PPAT is disrupted by PRPP, highlighting an intricate allosteric regulation. Overall, our findings reveal a fundamental mechanism of nucleotide balance and position NUDT5 as a regulator of nucleobase analogue metabolism. The latter pathway is especially crucial in rapidly proliferating cells, which must meet increased demands for nucleotides to sustain DNA replication and cell growth1,2,3. Pyrimidine de novo synthesis involves the sequential action of three key enzymes (Fig. 1a) starting with the multifunctional protein CAD (carbamoyl-phosphate synthetase II, aspartate transcarbamylase and dihydroorotase). It is followed by dihydroorotate dehydrogenase (DHODH), localized in the mitochondria, whose activity canonically relies on electron transfer to CoQ (ubiquinone), participating in the mitochondrial electron transport chain4,5,6. The final enzyme, uridine monophosphate synthase (UMPS), acts in the cytosol to link the pyrimidine ring with phosphoribosyl pyrophosphate (PRPP) to form uridine monophosphate (UMP), the precursor of all pyrimidine nucleotides. In contrast, de novo biosynthesis of purines is initiated directly on the PRPP backbone through a ten-step pathway beginning with the amidophosphoribosyltransferase PPAT (Fig. Mutations in the genes required for nucleotide biosynthesis lead to rare genetic disorders that, in the case of pyrimidine deficiency, can be treated by oral supplementation with uridine, the main substrate for pyrimidine salvage7,8,9. a, Simplified representation of pyrimidine de novo synthesis and salvage pathways. b, Proliferation assay of K562 cells with indicated knockouts over 4 days supplemented, as indicated, with 200 µM uridine or 500 µM orotate (four replicates, two-sided Student's t-test with Bonferroni–Dunn correction; P = 2 × 10−9, 1 × 10−7, 4 × 10−7, 2 × 10−8, 2 × 10−6, respectively). c, Proliferation assay of K562 cells over 5 days supplemented, as indicated, with 200 µM uridine, 200 µM CADi (sparfosic acid), 10 µM DHODHi (brequinar) or 10 µM UMPSi (pyrazofurin) (four replicates, two-sided Student's t-test with Bonferroni–Dunn correction; P = 3 × 10−8, 6 × 10−11, 6 × 10−9, respectively). d, Gene representation indicated by z-score in medium with uridine (x axis) or without uridine (y axis) from uridine-sensitized knockout screen in K562 cells. Each point is one gene. e, Ranked GSEA using gene ∆Z = Z−uridine – Z+uridine from uridine-sensitized screen and GO Biological Processes database with a Kolmogorov–Smirnov test. Each point represents one gene set. f, Relative metabolite abundances in K562 cells with indicated knockouts (four replicates, nonparametric analysis of variance (ANOVA) (Kruskall–Wallis test)). All bar graph data are mean ± s.e.m. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, the medium was refreshed 4–6 h before collection. Carb-Asp, carbamoyl-aspartate; CoQox, oxidized coenzyme Q; CoQred, reduced coenzyme Q; sgCtrl, control sgRNA. Precise regulation of nucleotide biosynthesis is vital for maintaining nucleotide balance and cellular homeostasis. CAD and PPAT, as the first commitment steps of their respective pathways, are rate-limiting enzymes and are strictly regulated to control de novo nucleotide synthesis4,10,11,12,13,14. Cancer cells, characterized by rapid proliferation, rely on these pathways to maintain their enhanced metabolic needs, and both cancer and immune cells can catabolize nucleosides when glucose is scarce15,16,17,18,19,20, making nucleotide metabolism a prime target for therapeutic intervention. Nucleotide analogues mimic endogenous nucleotides, thereby disrupting DNA replication and RNA stability, and are widely used in therapies against cancer, autoimmunity and viral infections1,21. However, despite the clinical success of over 20 US Food and Drug Administration-approved analogues for treating malignancies such as leukaemia and pancreatic cancer, autoinflammatory conditions such as rheumatoid arthritis, or for blocking viral replication, resistance frequently arises due to genetic instability and competition with endogenous substrates22,23,24. These limitations underscore the need to identify new regulatory mechanisms and therapeutic targets within nucleotide metabolism to overcome resistance and improve treatment outcomes. Here, we leverage the convergence of pyrimidine de novo synthesis and salvage pathways to design a uridine-sensitized CRISPR-Cas9 screening method to identify regulators of pyrimidine de novo synthesis. We reveal that a CoQ precursor can functionally replace CoQ for pyrimidine de novo synthesis and identify a non-catalytic role for the ADP-pyrophosphatase NUDT5 in nucleotide metabolism. We demonstrate that NUDT5 maintains PRPP levels essential for pyrimidine synthesis and mediates nucleobase analogue toxicity by inhibiting PPAT through PRPP-sensitive protein–protein interaction. Our results highlight NUDT5 as a critical and physiological node in ensuring balanced nucleotide production, thereby sensitizing cells to nucleotide analogue therapies. To discover genes involved in pyrimidine metabolism, we sought to exploit the dependency on nucleoside salvage exhibited when de novo synthesis is impaired25,26,27. We used CRISPR-Cas9 to deplete the three key enzymes required for de novo pyrimidine synthesis (CAD, DHODH and UMPS) in K562 myelogenous leukaemia cells, which resulted in a strong reduction in proliferation (Fig. 1a,b and Extended Data Fig. We next supplemented uridine to the cell culture medium and found that it was sufficient to restore proliferation in cells genetically depleted for, or treated with inhibitors of, any of the three key enzymes (Fig. This contrasted with supplementation of cytidine or thymidine, two downstream pyrimidine nucleosides and with the intermediate orotate, which rescued only CAD and DHODH depletion, but not UMPS (Fig. Having confirmed dependency on uridine salvage, we next conducted a genome-wide CRISPR-Cas9 depletion screen comparing cell proliferation in the presence or absence of supplemental uridine (Extended Data Fig. We applied two analytical methods, a z-score-based approach28 and the Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) algorithm29, both of which highlighted the three key enzymes of de novo pyrimidine biosynthesis as the top differentially essential genes in the absence of uridine, while salvage enzymes (UCK1 and UCK2) were dispensable (Fig. We confirmed these findings by gene set enrichment analysis (GSEA)30,31 using both Gene Ontology (GO; Biological Processes)32,33 or a curated set of mitochondrial pathways (MitoPathways)34 gene sets (Fig. These analyses further identified requirement for CoQ biosynthesis and its downstream electron acceptors, complexes III and IV (CIII and CIV) of the respiratory chain, an expected result as CoQ is the canonical electron acceptor for DHODH4,5,6 and is linked to the mitochondrial respiratory chain (Fig. CoQ synthesis within the mitochondria relies on eight known catalytic enzymes, as well as accessory factors that may facilitate substrate access35. Among the known enzymes catalysing steps in CoQ synthesis tested in our screen, only COQ7 did not score significantly (Fig. Of note, our screen also highlighted other factors not previously linked to pyrimidine biosynthesis, and by prioritizing genes with high scores using both analytical methods we selected for further investigation genes encoding the ADP-sugar pyrophosphatase (NUDT5), the serine/threonine-protein phosphatase PP1-b catalytic subunit (PPP1CB) and one of its regulatory subunits (PPP1R18), the catalytic subunit of N-terminal acetyltransferase C (NatC) complex (NAA30), an importin (IPO5) and two genes encoding poorly characterized proteins (SPICE1 and C19orf53) (Supplementary Table 1). Using CRISPR-Cas9, we individually depleted each of these genes, all three de novo pyrimidine synthesis enzymes, and five CoQ biosynthetic enzymes involved in CoQ head group maturation, including COQ7, in K562 cells. We used targeted metabolomics to analyse the levels of carbamoyl-aspartate, dihydroorotate and orotate, the three intermediates of de novo pyrimidine biosynthesis (Fig. 1a), as we reasoned that changes in their abundance would indicate the biosynthetic steps affected in these cells. In validation of this approach, we found altered levels of pyrimidine precursors in cells depleted for each of the three enzymes of de novo pyrimidine synthesis, with metabolomes characterized by (1) a profound decrease in all intermediates following CAD depletion; (2) accumulation of carbamoyl-aspartate and dihydroorotate, but decreased orotate following DHODH depletion; or (3) accumulation of all three intermediates following UMPS depletion; in all conditions with a significant decrease in pyrimidine nucleotides, yet with a milder effect after UMPS depletion, possibly illustrating lower sgRNA efficiency, compensatory pathways or traces of nucleotide precursors in the dialysed FBS (Fig. We found that the amplitude of changes in these intermediates were several orders of magnitude larger than changes observed in proliferation rates (Fig. 1b), indicating that measurement of pyrimidine intermediates by metabolomics represents a more sensitive method for assessing defects in pyrimidine synthesis. We next analysed the metabolic profiles of our genes of interest and found that most fall into one of three major categories: PPP1CB, its binding partner PPP1R18 and, to a lesser degree, IPO5 resembled depletion of CAD; COQ2, COQ3, COQ5 and COQ6, as well as NAA30 resembled depletion of DHODH; and NUDT5 resembled depletion of UMPS, illustrated by the accumulation of all three intermediates. COQ7, SPICE1 and C19orf53 showed no significant effects (Fig. Therefore, our targeted metabolomics approach validated most of the genes highlighted in our screen and assigned genes to discrete steps in de novo pyrimidine synthesis. CoQ is the canonical electron acceptor for several enzymes on the inner mitochondrial membrane, including DHODH and the respiratory chain (Fig. Thus, depletion of CoQ biosynthetic enzymes is expected to block both pyrimidine de novo synthesis and respiration30,35,36. Our screen effectively revealed and validated central genes in the CoQ biosynthesis pathway, with the notable exception of COQ7 (Fig. 1f and Supplementary Table 1), which is required for conversion of the CoQ precursor demethoxy-coenzyme Q (DMQ) into demethyl-coenzyme Q (DMeQ), the final intermediate of CoQ biosynthesis37,38 (Fig. Given the role of CoQ in DHODH function, the observation that COQ7 is dispensable for pyrimidine synthesis was unexpected. To investigate this anomaly, we generated K562 single-cell knockout (KO) clones (COQ7KO) and measured the levels of human DMQ (DMQ10) using lipidomics (Fig. As expected, we found that while DMQ10 levels were very low in control cells and in cells depleted of four other COQ enzymes, it accumulated strongly in the absence of COQ7 (Fig. 2b), showing that the conversion of DMQ10 could not proceed without COQ7. Similarly, we confirmed lower CoQ10 levels across COQ KOs, including COQ7 depletion (Fig. 2b), resulting in impaired respiration and failure to thrive in galactose medium, indicating an inability to perform oxidative phosphorylation (OXPHOS) (Fig. 2c,d and Extended Data Fig. However, while uridine supplementation could partially rescue the growth defects of COQ2- and COQ3-depleted cells, as expected from the additional roles of CoQ in cellular metabolism, COQ7 depletion had no visible effect on uridine-dependent growth and pyrimidine synthesis (Figs. Thus, while mature CoQ seems strictly necessary for OXPHOS, we found through our advanced validation of the screen that pyrimidine synthesis is still possible in the absence of its electron acceptor CoQ in COQ7-depleted human cells, which accumulate high levels of the DMQ precursor. a, Simplified representation of CoQ maturation and of CoQ as the electron acceptor for DHODH or CI–II of the respiratory chain. Enzymes that were present in the Brunello library65 are coloured and further underlined if significant in the screen. b, Relative abundances of DMQ10 and CoQ10 in K562 cells with indicated KOs (three replicates, 12 for sgCtrl, P = 4 × 10−9, 4 × 10−12, 2 × 10−4, 8 × 10−22, 4 × 10−21, 1 × 10−15, 5 × 10−15, 2 × 10−12, 2 × 10−16, respectively) or in single-cell clones (four replicates, two clones each, P = 4 × 10−21, 2 × 10−20, 8 × 10−26, 2 × 10−25, respectively). Superscript numbers refer to clone identification. c, Oxygen consumption rate (OCR) of K562 cells with indicated KOs (30 replicates, 60 for sgCtrl, P = 3 × 10−30, 1 × 10−13, 6 × 10−12, respectively). (o), oligomycin; (c), CCCP; (a), antimycin A. e, Proliferation assay of K562 cells with indicated knockouts over 5 days supplemented, as indicated, with 200 µM uridine (four replicates, eight for sgCtrl, P = 8 × 10−8, 9 × 10−6, respectively). f, Proliferation assay of COQ7 clones over 5 days supplemented, as indicated, with 200 µM uridine (four replicates, two clones each). g, Chemical structures of DMQ10 and CoQ10. h, AlphaFold 3 model of CoQ10 or DMQ10 binding to DHODH showing hydrogen bonds (black dashed lines). Purple oval highlights head group difference between CoQ and DMQ. Proposed model of electron transfer depending on COQ7 expression. Without COQ7, DMQ accumulates and can sustain electron transfer for DHODH but not CI–II (grey dotted line). Statistical tests were one-way ANOVA with Bonferroni correction for metabolomics, two-sided Student's t-test with Bonferroni correction for respirometry and growth assays. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, the medium was refreshed 4–6 h before collecting. Similar to experimentally obtained structures of DHODH with the inhibitory quinone analogue brequinar39, our model predicted multiple interactions between CoQ and the quinone-binding pocket of DHODH, but none formed between the methoxy group in position six of CoQ, which is absent in DMQ (Fig. 2g,h and Extended Data Fig. Accordingly, DMQ docking in DHODH led to a similar model, suggesting that DHODH may accommodate both CoQ and DMQ (Fig. Following reduction by enzymes such as DHODH, CoQ is normally oxidized by CIII and electrons are transferred through CIV to the final acceptor, oxygen. Notably, we observed residual CIII-dependent (antimycin A-sensitive) respiration in COQ7-depleted cells (Fig. In addition, blocking CIII activity with antimycin A induced uridine auxotrophy in COQ7KO cells (Extended Data Fig. 2f), showing requirement of CIII for pyrimidine synthesis even in the absence of CoQ. Together, our results indicate that CoQ is dispensible for pyrimidine synthesis and suggest that DMQ accumulation following COQ7 depletion, while unable to support CI–II-mediated respiration, can functionally substitute CoQ to efficiently support electron transfer from DHODH to CIII, thus driving pyrimidine synthesis (Fig. The top hit from our screen, apart from the three de novo pyrimidine biosynthetic enzymes, was NUDT5 (NUDIX5) (Fig. 1e), encoding for a member of the NUDIX (nucleoside diphosphate linked to moiety-X) hydrolase family. NUDT5 cleaves ADP-ribose to form ribose-5-phosphate (R5P) and AMP or ATP, depending on phosphate availability, and was also reported to cleave oxidized guanylate nucleotides at high pH40,41,42 (Extended Data Fig. We validated that uridine supplementation promotes the growth of NUDT5-depleted cells, including at physiological levels found in human plasma-like medium (HPLM)43 (Extended Data Fig. 3b,c), and, to gain a broader understanding of the influence of NUDT5 on cell metabolism, we performed an expanded targeted metabolomics analysis on NUDT5-depleted K562 cells in which we observed significant accumulation of the intermediates of de novo pyrimidine synthesis and decrease of mature pyrimidines (Fig. In contrast, we found an accumulation of both mature purines and their intermediates, validated both in cell pellets and secreted into the medium, indicating nucleotide imbalance (Fig. 3c,d and Extended Data Fig. We confirmed depleted pyrimidines alongside elevated purines in three additional cell lines depleted of NUDT5 (Extended Data Fig. 3g,h), including the breast cancer line MCF7, in which earlier work on NUDT5 was performed41,42,44. a, Relative metabolite abundances comparing K562 NUDT5 KO against control (four replicates). Each point is one metabolite. b, Ranked Metabolite Set Enrichment Analysis in NUDT5 KO against control. Each point is one metabolic pathway. KEGG, Kyoto Encyclopedia of Genes and Genomes. c, Representation of purine biosynthesis including PRPP synthesis. PPAT is regulated by nucleotides (inhibitory feedback, blue dotted lines) and by PRPP (positive feedback, green dashed arrow). d, Relative metabolite abundances in K562 cells with indicated KOs (four replicates, eight for sgCtrl, P = 3 × 10−3, 4 × 10−2, 3 × 10−3, respectively). e, Relative metabolite abundances in NUDT5 clones as detected using luminescence assays for ADP or ATP (three replicates, two clones each, P = 9 × 10−5, 9 × 10−4, respectively), fluorescence assay for (hypo)xanthine (three replicates, two clones each, P = 1 × 10−2) or targeted metabolomics for other metabolites (four replicates, two clones each, P = 9 × 10−8, 2 × 10−4, 4 × 10−6, respectively). f, Relative metabolite abundances in NUDT5 clones complemented with indicated cDNAs (four replicates, P = 5 × 10−9, 8 × 10−7, 4 × 10−9, 5 × 10−3, respectively). NUDT5E112Q is a catalytic inactive mutant of NUDT5. g, Representation of U-13C-glucose tracer with labelled carbon as orange circles (left). Relative m + 5 metabolite abundances in NUDT5 clones complemented with indicated cDNAs and cultured for 5 h with U-13C-glucose tracer (four replicates) (right). Bar graph data are mean ± s.e.m. Statistical tests were a two-sided Student's t-test. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, the medium was refreshed 4–6 h before collecting. To extend our investigation, we generated NUDT5KO K562 single-cell clones (Extended Data Fig. 3i), in which we confirmed altered pyrimidine and purine metabolism using both targeted metabolomics and orthogonal biochemical assays (Fig. In addition, we measured ADP-ribose and R5P, two main metabolites of the NUDT5 reaction, but found no differences in their abundance, indicating that ADP-ribose catabolism from NUDT5 does not participate significantly to these pools at steady state (Fig. Having observed a similar phenotype upon either UMPS or NUDT5 depletion (Fig. 1f), we next analysed the ribose donor PRPP and found that its levels were significantly decreased in NUDT5-depleted cells (Fig. This observation explains the decreased pyrimidine synthesis seen upon NUDT5 depletion, but was in part unexpected, as PRPP is also the precursor to purine synthesis, which we found elevated (Fig. We next tested whether the catalytic activity of NUDT5 was required for its function in nucleotide metabolism and expressed a wild-type (WT) or a catalytically inactive mutant (E112Q)42,45 of NUDT5 in NUDT5KO cells (Extended Data Fig. We found that both the WT or mutant NUDT5 could rescue the levels of PRPP and pyrimidine synthesis intermediates to baseline (Fig. 3f), suggesting that NUDT5 catalytic activity is not essential for its effect on pyrimidine synthesis. Similarly, a nanomolar NUDT5 catalytic inhibitor (TH5427 (ref. 46)) had no effect on pyrimidine intermediates nor PRPP (Extended Data Fig. To further validate our observations, we used uniformly labelled 13C-glucose, as glucose is the main precursor to PRPP (Fig. We found that the incorporation of ribose (m + 5) was attenuated in pyrimidines and elevated in adenylate purines in NUDT5-depleted cells (Fig. Reintroduction of NUDT5 WT and E112Q cDNAs in these cells restored nucleotide synthesis to baseline levels (Fig. Unlabelled (m + 0) levels of nucleotides were also affected by NUDT5 depletion, as expected as pyrimidine synthesis is a linear pathway in which baseline and pulse-labelled nucleotide abundance depend both on the same biosynthesis and degradation rates (Supplementary Table 3). Together, our results indicate that whereas PRPP and mature pyrimidine pools are low in NUDT5-depleted cells, purine synthesis seems to function at a higher rate, suggesting preferential mobilization of the PRPP pool towards purines, with a detrimental effect on pyrimidine synthesis. Having excluded an enzymatic mechanism through which NUDT5 affects pyrimidine synthesis, we investigated changes in the proteome that occur upon NUDT5 depletion. However, no substantial changes were observed in the abundance of enzymes involved in purine, pyrimidine or PRPP synthesis (Extended Data Fig. We then immunoprecipitated Flag-tagged NUDT5 and investigated its binding partners by mass spectrometry (Fig. We found a number of NUDT5-interacting proteins, including cytoskeletal elements, and notably PPAT, the rate-limiting and first committed enzyme required for de novo purine biosynthesis12 (Fig. An interaction between NUDT5 and PPAT has previously been reported in proteome-scale protein–protein interaction studies47,48,49. Because both NUDT5 and PPAT migrate at the same sizes as the immunoglobulins used for immunoprecipitation, we confirmed the interaction using either of NUDT5–Flag WT, its catalytic inactive E112Q mutant or PPAT–Flag as bait (Fig. a, Endogenous protein binding partners enriched from immunoprecipitation (IP) of Flag-tagged NUDT5 in 293T cells followed by mass spectrometry-based proteomics. Each point represents one protein. b, Coimmunoprecipitation from 293T cells using a Flag tag as bait. c, Schematic representation of PRPP use for purine or pyrimidine de novo synthesis. d, Immunoblot of NUDT5 clones with indicated KOs. e, Relative metabolite abundances in NUDT5 clones with indicated KOs (four replicates, two clones each, P = 4 × 10−11, 7 × 10−6, 3 × 10−4, respectively). f, Parallel native PAGE on NUDT5 clones. g, AlphaFold 3 prediction of a complex consisting of PPAT tetramer (pink) and two NUDT5 dimers (blue), showing a zoom to the interaction interface of one NUDT5 and two PPAT molecules with amino acid residues and hydrogen bonds (dashed white lines) indicated. h, Parallel native PAGE on NUDT5 clones with indicated cDNA complementation. i, Relative metabolite abundances in NUDT5 clones with indicated cDNA complementation (four replicates, P = 1 × 10−4, 2 × 10−4, 4 × 10−3, respectively). j. Coimmunoprecipitation from 293T cells using Flag-tagged PPAT as bait with 10 mM PRPP in the IP buffer or left untreated. k, Parallel native PAGE on K562 cells with 10 mM PRPP in the lysis buffer or left untreated. l, Parallel native PAGE on indicated mouse organs with 10 mM PRPP in the lysis buffer or left untreated. m, Proposed model of PPAT regulation. NUDT5 binding promotes formation of the large low-activity PPAT form. When interaction with NUDT5 is disrupted, PPAT constitutively forms the small high-activity form, depleting PRPP and hindering pyrimidine synthesis. In physiological conditions, elevated PRPP similarly promotes complex dissociation and PPAT activity for purine synthesis. n, Proliferation assay of NUDT5 clones over 5 days in response to a dose curve of purine and pyrimidine nucleotide analogues (two replicates, two clones each, P = 6 × 10−9, 1 × 10−10, respectively). A molecular representation is shown with the nitrogenous base (dark blue polygon), ribose sugar (light blue pentagon) and phosphate (green circle). and are fitted by four-parameter logistic regression. Bar graph data are mean ± s.e.m. Statistical tests were a two-sided Student's t-test. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, medium was refreshed 4–6 h before collection. PPAT and UMPS both consume and potentially compete for PRPP, a metabolite we found to be critically low in NUDT5-depleted cells (Fig. To determine whether PPAT may be at the origin of the pyrimidine deficiency seen in these cells, we sought to simultaneously deplete PPAT and NUDT5 in an epistasis genetic experiment. However, similar to uridine dependency when pyrimidine de novo synthesis is compromised (Fig. 1b), PPAT-depleted cells become dependent on purine salvage, and we therefore supplemented these with inosine (Extended Data Fig. Of note, we found that PPAT depletion was sufficient to restore carbamoyl-aspartate, dihydroorotate and orotate to WT levels in NUDT5KO cells (Fig. Our observations indicate a contribution for PPAT to the pyrimidine phenotype and position PPAT downstream of NUDT5. Seminal work from the 1970s identified the presence of PPAT in two interconvertible forms, consisting of a larger and partially inactive ~300-kDa form, and a smaller active form. We thus further investigated the NUDT5–PPAT interaction using native gel electrophoresis in K562 cells, where we predominantly observed the large ~300-kDa form with low enzymatic activity50,51,52 (Fig. Notably, this large PPAT oligomer co-migrated with NUDT5, which was also present independently as a dimer42, thus allowing us to confirm the interaction between the two endogenous proteins (Fig. 4f); however, when NUDT5 was depleted, the PPAT complex dissociated (Fig. 4f), forming smaller oligomers consistent with the active form50,51,52, and suggesting the interaction is inhibitory for PPAT, as hinted to by our genetic experiment (Fig. We next aimed to model the NUDT5–PPAT complex, but no experimental structure of human PPAT currently exists, likely due to its labile iron-sulfur cluster and sensitivity to oxygen that have hindered its recombinant production53. We thus used AlphaFold 3 (ref. 54) to predict the structure of a human PPAT monomer (57 kDa) and tetramer (228 kDa), both of which exhibited strong similarities to experimentally determined bacterial PPAT homologue purF55,56 (Extended Data Fig. We then predicted the structure of an NUDT5 dimer, also highly consistent with experimentally determined structures57 (Extended Data Fig. When we added one or two NUDT5 dimers to the PPAT tetramer, we found that these can cap either side of the PPAT barrel (Fig. The predicted interaction interfaces consisted of NUDT5 residues RTLHY70–74, unique to NUDT5 and located outside the conserved NUDIX motif (97–118), which interacts with two PPAT molecules, associated with residues TQLDVPH25–31 on one and PNMRL360–364 on the other (Fig. 4g), all of which are conserved across vertebrates (Extended Data Fig. To determine the validity of the AlphaFold 3 model, we next mutated NUDT5 residues 70–74 to alanines and reintroduced the corresponding cDNA into NUDT5KO cells (Extended Data Fig. Of note, we found that NUDT5-(A)70–74 was unable to rescue the formation of the NUDT5–PPAT complex, consistent with the AlphaFold 3 predictions (Fig. 5c), nor levels of pyrimidine precursors (Fig. 4i), and showing that NUDT5 must physically interact with PPAT to support pyrimidine synthesis. We next investigated whether the NUDT5–PPAT interaction could itself be regulated. Besides its role as a substrate, PRPP is an established allosteric activator of PPAT that influences its oligomerization, as addition of PRPP to cell lysates is sufficient to convert PPAT's large form in its active smaller form50,51,52,58. Notably, and consistent with these reports, we found that PRPP was sufficient to disrupt the interaction between NUDT5 and PPAT across immunoprecipitations in 293T, K562 or HeLa cells (Fig. Furthermore, we observed that PRPP could induce formation of smaller PPAT oligomers in native gel electrophoresis that were similar to those seen in NUDT5KO cells (Fig. Quantifying this interaction, we found that PRPP could disrupt the PPAT oligomer in K562 cells at a half-maximal concentration (IC50) of 0.1 mM, falling within the reported range of intracellular PRPP in pathophysiological conditions59,60 (Extended Data Fig. We further confirmed the existence of the NUDT5–PPAT complex across ten cell lines and six post-mitotic mouse organs, where we found on average ~33% of NUDT5 associated to PPAT, with cell type- and organ-specific variability (Fig. Notably, in all cases we found that PRPP was able to disrupt the interaction of the two proteins. Together, our genetic and biochemical observations demonstrate that NUDT5 interacts with PPAT to promote the formation of a high-molecular weight, low-activity PPAT form that can be disrupted by the allosteric activator PRPP, and that in NUDT5-depleted cells, constitutively active PPAT in its low-molecular weight form consumes the endogenous PRPP pool to synthesize purines, at the expense of pyrimidine synthesis (Fig. Having seen nucleotide imbalance and a strong effect on the PRPP pool in NUDT5KO cells, we next tested whether NUDT5 could affect the sensitivity of cells to nucleobase analogues, which require PRPP for their conversion into toxic nucleotide analogues23,61,62,63,64. This hypothesis is consistent with the results of two high-throughput screens that identified a role for NUDT5 in 6-thioguanine (6-TG) resistance65,66, although in both cases the mechanism of resistance remained unclear. We first determined the IC50 of 5-fluorouracil (5-FU), a pyrimidine nucleobase analogue primarily metabolized by TYMP, but that can also be metabolized by UMPS61,62,63,67, and observed that NUDT5KO cells were an order of magnitude more resistant to 5-FU than their corresponding WT counterparts (Fig. In contrast, sensitivity to 5-fluorouridine, the nucleosidic form of 5-FU that does not require PRPP for processing, was unchanged. We extended these observations to four other clinically used purine and pyrimidine analogues and again observed that while NUDT5KO cells remained equally sensitive to all tested nucleotide and nucleoside analogues, they were consistently more resistant to nucleobase analogues that rely on PRPP, irrespective of whether the molecules were purine- or pyrimidine-based (Fig. We observed a similar trend in cells lines from the Cancer Cell Line Encyclopedia (CCLE)68 (Extended Data Fig. Together, our findings indicate that the role of NUDT5 in nucleotide regulation has strong implications for nucleobase analogue therapy. By combining uridine sensitization and genome-wide CRISPR-Cas9 screening, we discovered factors required for de novo pyrimidine synthesis. Our approach successfully recovered CAD, DHODH and UMPS as the most important, and also highlighted a previously unidentified role for several other genes. We extensively validated such factors using metabolomics and lipidomics, assigning genes to specific steps in pyrimidine synthesis. This included all three canonical enzymes of pyrimidine synthesis, the phosphatase-related genes PPP1CB and PPP1R18 whose depletion resembles that of CAD; genes of CoQ synthesis (COQ2-3-5-6) and N-terminal acetylation (NAA30), all of which resemble DHODH depletion; and the ADP-ribose pyrophosphatase NUDT5 whose depletion resembles that of UMPS. We investigated NUDT5 in detail and, based on metabolic profiles, speculate that the other genes not previously linked to pyrimidine synthesis might act by regulating the activity, or the stability, of the enzymes involved in nucleotide metabolism and CoQ synthesis. Our findings challenge the presumed indispensability of CoQ for pyrimidine synthesis. Given the relationship between DHODH and CoQ, recovery of catalytic factors of CoQ biosynthesis was an expected result, but, notably, we found that pyrimidine synthesis persists in COQ7-depleted cells, despite marked CoQ deficiency. Supporting this observation, the COQ7 accessory factor, COQ9 (refs. 69,70), did not score in our screen. While COQ4 was not included in the Brunello CRISPR library used in this study65, accessory factors (COQ8A, COQ8B, COQ10A and COQ10B), showed no uridine dependency phenotype, confirming limited or redundant functions. The dispensability of COQ7 was unexpected and, based on the accumulation of DMQ and its molecular resemblance to CoQ (Fig. 2), we propose that DMQ may functionally substitute as an electron acceptor for DHODH. Our model is supported by the residual CIII-dependent respiration we observed upon COQ7 depletion (Fig. 2), as reported previously71, which indicates continued electron transfer in the absence of CoQ. Recent research has highlighted rhodoquinone as an alternative electron carrier able to maintain pyrimidine synthesis in the absence of CoQ72. Therefore, DMQ may play a similar role as it is interesting to note that CoQ, DMQ and rhodoquinone differ only by a single chemical group on the same position of the polar head group. In contrast, however, while rhodoquinone can also maintain OXPHOS72, we found that COQ7 loss severely impairs respiration (Fig. 2), indicating limited ability of DMQ to function as an electron acceptor from CI–II. We propose that while DMQ can maintain pyrimidine synthesis through electron transfer to CIII and ultimately oxygen, it is insufficient to maintain electron transfers for mitochondrial CI–CII. While additional work, notably using chemically synthesized DMQ, will be required to validate electron transfer from DHODH to DMQ, our results hint at structural differences in CoQ binding sites and may pave the way for increasingly specific DHODH inhibitors. Through genetic screening and metabolomics, we identified NUDT5 as a mediator of nucleotide balance. We validated the effect of NUDT5 depletion on uridine-dependent growth, but found that the effect size was mild compared with the metabolic phenotype, although in both cases it reached significance (Fig. Several technical reasons may explain this observation, including cellular adaptation, differences between pooled screen assays and individual cell lines, or traces of nucleotide precursors in dialysed FBS. It is also possible that the depletion of nucleotides might need to reach a certain threshold before presenting a strong effect on growth. Furthermore, and similar to the partial rescue of COQ2- and COQ3-depleted cells with uridine (Fig. 2e), uridine supplementation can only rescue the pyrimidine synthesis aspect of the phenotype, and in the case of NUDT5, it will not affect the potentially deleterious accumulation of purines and nucleotide imbalance, nor the other aspects of NUDT5 function that are not related to pyrimidine synthesis, such as DNA repair42,73, highlighting the limitations of growth assays. The difference in amplitude with the canonical enzymes of pyrimidine synthesis is coherent with our model of NUDT5 being a regulator, rather than a catalytic step, in nucleotide metabolism. NUDT5 belongs to the NUDIX family of proteins that comprises 22 enzymes in human which are poorly conserved but share a NUDIX domain, 13 of which we detected through proteomics of K562 cells (Supplementary Table 4). This family was initially described as sanitation proteins, able to clear oxidized or otherwise modified nucleotides74, but more detailed characterization has revealed a broader role as pyrophosphatases affecting a range of substrates41,75. We now show that NUDT5 is an inhibitory binding partner to the rate-limiting purine enzyme PPAT, binds PPAT outside its conserved NUDIX domain, acts independently of its catalytic activity and that its loss promotes purine biosynthesis, depleting the endogenous PRPP pool, and thus impairing pyrimidine de novo synthesis. Of note, while PRPP also acts as an allosteric activator for PPAT50,51,52,58, our findings expand the view of PPAT regulation by identifying NUDT5 and showing that its binding is sensitive to PRPP treatment. Further research, possibly including detailed structural analysis or in vitro binding assays using purified proteins, will be needed to clarify the interplay between NUDT5- and PRPP-mediated PPAT regulation and investigate their full impact on nucleotide synthesis and macromolecular structures such as the purinosome76. Furthermore, nutrient availability, already known to affect PRPP levels and nucleotide synthesis77,78, may represent a driving factor behind this fundamental regulatory mechanism and will merit further investigation. Our findings also position NUDT5 in the context of nucleotide analogue toxicity, including in cancer, autoinflammatory disorder, and antiviral therapies, a notion supported by other recent reports, notably those identifying germline NUDT5 genetic variants associated with 6-TG resistance in acute lymphoblastic leukaemia79,80,81,82. Based on our results, two models could explain the resistance of NUDT5-depleted cells to nucleobase analogues: (1) the limited PRPP pool that would hinder conversion of nucleobase analogues into toxic nucleotides, and (2) competition with endogenous purines, in the case of purine analogues only. While both models are not mutually exclusive, we favour the central role of PRPP, given our experimental and CCLE data on the resistance of NUDT5-depleted cells not only to purine, but also to pyrimidine nucleobase analogues. Our work highlights the power of nucleoside-sensitized genetic screens to identify genes involved in nucleotide metabolism and human disease. We expect that our approach can be extended to investigate other pathways of nucleotide metabolism, for example by screening in inosine-containing conditions (Extended Data Fig. Together, our findings are highly relevant to understanding the limitations, and improving the effectiveness, of nucleotide analogue-based therapies. All animal experiments were approved by the Swiss Cantonal authorities (VD3788) and all relevant ethical regulations were followed. Animals were male C57BL/6J mice aged 12–13 weeks provided with standard laboratory chow (Safe 150) and water ad libitum, and with a standard 12-h light–dark cycle. Sex was not considered in the study design. UACC-257 (a gift from D. Fisher), Jurkat (ATCC, TIB-152), THP1 (ATCC, TIB-202), U937 (ATCC, CRL-1593.2) and LCL (Coriell, GM12878) cells were maintained in RPMI with 10% FBS (Gibco A5256701) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). UACC-257 cells were reauthenticated by STR profiling at ATCC in 2020. K562, 293T, HeLa and U2OS were acquired from ATCC less than 4 years before submission of the manuscript and were not reauthenticated. Other cell lines were not reauthenticated. All cells were cultured under 5% CO2 at 37 °C. Cells were periodically tested to ensure the absence of Mycoplasma. Gene-specific single-guide RNAs (sgRNAs) were selected from the two best scoring sequences from the CRISPR-Cas9 screen and cloned into lentiCRISPR v2 vector (Addgene, 52961). The negative control (sgCtrl) was generated using guides targeting OR2M4 and OR11A1 that are not expressed in K562. For gene-specific complementary DNA rescue, sgRNA-resistant gene sequences were cloned into pLV-EF1a-IRES-Puro vector (Addgene, 85132). Flag-tagged green fluorescent protein (GFP) in the same vector was used as a control (Addgene, 201636). The list of sgRNA and cDNA sequences used for cloning can be found in Supplementary Table 5. Reagent requests should be addressed to the corresponding author. Individual cloned sgRNA plasmids were electroporated into K562 cells alongside GFP using the Cell Line Nucleofector kit V (Lonza, VCA-1003) according to the manufacturer's protocol with the T-016 program. Cells were grown for 48 h then stained with Zombie Violet (BioLegend, 423114) and fluorescence-activated single-cell sorting (FACS) was used to sort GFP+ Zombie− cells into flat-bottom 96-well plates at one cell per well. Cells were grown for 12 days and wells with single colonies were selected for based on brightfield microscopy. Single-cell clones were expanded over 5 weeks and KOs were verified by immunoblot and by sequencing genomic DNA (gDNA), extracted using the QIAamp DNA kit (QIAGEN, 51304) according to the manufacturer's protocol. Lentiviruses were produced from 293T cells after transfection with a cDNA expressing plasmid, or with two plasmids containing sgRNAs to the control or target genes, as previously described83. Supernatant was collected 72 h following transfection, filtered through 0.45 µm and stored at −72 °C. For infections, cells at 0.5 × 106 cells per ml with 10 µg ml−1 Polybrene (Sigma-Aldrich, TR-1003) were grown in a 1:1 ratio medium to virus supernatant for 24 h. Selection was performed with 2 µg ml−1 puromycin (InvivoGen, ant-pr-1) over 48 h. Cells were maintained in standard cell culture medium for 5 additional days before analysis or further experiments. Gene KO or rescue were confirmed by immunoblot and cDNA rescue with NUDT5 WT or E112Q mutant were further verified by sequencing gDNA extracted using the QIAamp DNA kit (QIAGEN, 51304), according to the manufacturer's protocol. K562 cells were seeded at 0.05 × 106 cells per ml in black flat-bottom 96-well plates (Thermo Scientific, 137101) for analysis by Prestoblue, or in flat-bottom 12-well plates for analysis by cell count. To compare glucose and galactose conditions, the test medium consisted of DMEM (Gibco, 11966-025) with 1 mM sodium pyruvate (Gibco, 11360070), 2 mM glutamine (BioConcept, 5-10K00-H), 0.2 mM uridine, 10% dialysed FBS (dFBS) (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H) supplemented with 25 mM glucose or galactose. The base test medium for other assays consisted of DMEM (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). This was supplemented as indicated in figure legends with metabolites, small molecule inhibitors or volumetric equivalents of water or dimethylsulfoxide (DMSO). Drugs purchased from MedChemExpress were sparfosic acid trisodium (HY-112732B), pyrazofurin (HY-122502), 5-fluorouridine (HY-107856), clofarabine (HY-A0005), fludarabine-phosphate (HY-B0028), and gemcitabine (HY-17026), and from Sigma-Aldrich were antimycin A (A8674), 5-FU (F6627) and 6-thioguanine (A4882). Cells were grown for 5–7 days under test conditions and proliferation was determined either using the Prestoblue dye (Invitrogen, A13262) and measuring fluorescence (ex/em 560/590 nm) with the BioTek Synergy Plate Reader (Agilent Technologies) after 1.5 h incubation at 37 °C. Background values were subtracted from Prestoblue data before analysis. For cell counts, cells were counted after 5–7 days using trypan blue-based cell counting (Vi-cell Blu counter, Beckman Coulter) and only live cells were considered. Cells were collected, washed in PBS and lysed by 10 min incubation on ice in RIPA buffer (25 mM Tris, pH 7.5, 150 mM NaCl, 0.1% SDS, 0.1% sodium deoxycholate and 1% NP-40) with 1:100 protease inhibitor (Thermo Scientific, 87786) and 1:500 nuclease (Thermo Scientific, 88702). Proteins were separated by SDS–PAGE on Novex Tris-Glycine Mini Protein Gels (Invitrogen, XP08160BOX and XP10200BOX) and were transferred to nitrocellulose membranes using a wet transfer chamber with buffer consisting of 0.302% (w/v) Tris base, 1.44% (w/v) glycine and 20% ethanol in water. Transfer was verified by Ponceau S Staining Solution (Thermo Scientific, A40000278). Immunoblotting was performed with 5% milk or 5% bovine serum albumin (Sigma-Aldrich, A9647) in TBS (20 mM Tris-HCl, pH 7.4 and 150 mM NaCl) with 0.1% Tween-20 (TBS–Tween) and 1:500 to 1:1,000 primary antibodies or 1:5,000 secondary antibodies (LI-COR, 102673-330, 102673-328 or 102673-408). Membranes were imaged by fluorescence detection at 700 and 800 nm with an Odyssey CLx Imager (LI-COR). For incubation with additional antibodies, membranes were incubated 15 min in mild stripping buffer (15% (w/v) glycine, 1% Tween-20 and 0.1% SDS, in water at pH 2.2), washed in water and reblocked. The protein signal was quantified using Image Studio Software v.4.0 (LI-COR). Genome-wide CRISPR-Cas9 screening was performed in K562 cells using the Brunello lentiviral library65 as previously described84. In brief, K562 cells were infected in duplicate at 500 cells per sgRNA with a multiplicity of infection of 0.3 in the presence of 10 µg ml−1 Polybrene (Sigma-Aldrich, TR-1003). After 24 h, cells were selected with 2 µg ml−1 puromycin (InvivoGen, ant-pr-1) for 48 h. On day 7 post-infection an aliquot was frozen for comparative analysis. At this time, cells were plated at 105 cells per ml (equivalent to 1,000 cells per sgRNA) in DMEM–GlutaMAX (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392), 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H) and either 200 µM uridine or volumetric equivalent in sterile water. Cells were passaged every 3 days for 3 weeks and 1,000 cells per sgRNA were collected on day 28 post-infection (21 days following medium switch). Genomic DNA was extracted using NucleoSpin Blood XL kit (Machery-Nagel, 740954.20), according to the manufacturer's protocol. Barcode sequencing, mapping and read counts were performed by the Genome Perturbation Platform (Broad Institute). For steady-state experiments, cells were maintained in exponential phase and grown for at least 4 days in DMEM–GlutaMAX (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). PPAT KO cells were further supplemented with 200 µM inosine. Treatment with 10 µM TH5427 (Tocris Bioscience, 6534) or DMSO was carried out over the last 36 h. Before collection, 107 cells per replicate were transferred to fresh medium for 4 h at 37 °C. Cells were collected, washed with PBS and centrifuged for 1 min at 2,000g at 4 °C. Supernatant was discarded and nitrogen vapour was used to displace ambient air. Pellets were flash-frozen in liquid nitrogen. For tracer analysis, cells were first maintained in exponential phase for 7 days in DMEM–GlutaMAX (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). Before collection, cells were washed with PBS and seeded at 2 × 106 per ml in no-glucose DMEM (Gibco, 11966-025) with 10% dFBS (Sigma-Aldrich, F0392), 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H), 1 mM pyruvate (Gibco, 11360039) and 25 mM U-13C-glucose (Omicron Biochemicals, GLC-082) for 5 h at 37 °C. Cells were collected, washed with PBS and centrifuged for 1 min at 2,000g at 4 °C. The supernatant was discarded and nitrogen vapour was used to displace ambient air. Pellets were flash-frozen in liquid nitrogen. Cells were grown for 4 days in DMEM–GlutaMAX (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). Before collection, cells were seeded at 1 × 107 per ml in fresh medium and were incubated alongside a flask of medium alone for 6 h at 37 °C. Cells were pelleted and supernatant was collected, centrifuged for 5 min at 2,000g at 4 °C and flash-frozen in liquid nitrogen. Metabolites were extracted using successive freeze–thaw in methanol. Cell pellets were first resuspended in 200 µl of pre-cooled 100% (v/v) methanol containing 1 µM of 13C-labelled 4-hydroxybenzoic acid as an internal control and then immediately frozen in liquid nitrogen. After brief thawing, samples were centrifuged 5 min at 8,000g and supernatant was collected. This process was repeated twice on the remaining pellet first with 200 µl 100% (v/v) methanol and then with 100 µl 100% (v/v) water, pooling the supernatants for a final concentration of 80% (v/v) methanol. Pooled supernatants were dried under vacuum, reconstituted in 50 µl of 50% (v/v) acetonitrile:water and moved into amber glass vials for analysis. Liquid chromatography–mass spectrometry (LC–MS) analysis was performed using a Thermo Vanquish Horizon UHPLC system coupled to a Thermo Exploris 240 Orbitrap mass spectrometer. For LC separation, a Vanquish binary pump system (Thermo Scientific) was used with a Waters Atlantis Premier BEH Z-HILIC column (100 × 2.1 mm, 1.7-µm particle size) held at 35 °C under 300 μl min−1 flow rate. Mobile phase A consisted of 5:95 (v/v) acetonitrile:water with 5 mM ammonium acetate (Sigma Millipore) and 200 µl l−1 25% ammonium solution (Sigma Millipore). Mobile phase B consisted of 95:5 (v/v) acetonitrile:water. Descriptions of all targeted metabolomics LC methods containing times and percentages of mobile phases are given in Supplementary Table 6. Then, 1 µl of sample was injected by a Vanquish Split Sampler HT autosampler (Thermo Scientific) while the autosampler temperature was kept at 4 °C. The samples were ionized by a heated electrospray ionization (ESI) source kept at a vaporizer temperature of either 350 °C or 200 °C depending on the LC method used. The inlet ion transfer tube temperature was kept at 325 °C with a 70% RF lens. The identity and retention time of targeted metabolites was first validated using commercial standards when available or unique MS2 fragments from a metabolite library (mzcloud.org) when not. Quantification of experimental samples was performed using either parallel reaction monitoring with a higher-energy collisional dissociation (30%, 40% and 50%), targeting selected ion fragments generated from the fragmentation of the hydrogen loss (H−) ion or targeted single-ion monitoring. A full list of the reported precursor and fragment ions is given in Supplementary Table 6. Peak integration was performed using TraceFinder v.5.1 (Thermo Scientific). This method was used to analyse metabolites in Figs. Frozen cell pellets were thawed on ice, then 150 mM KCl (50 µl) was added to each sample, followed by ice-cold methanol (600 µl) with 1 µM CoQ8 as internal standard (Avanti Polar Lipids). Ice-cold petroleum ether (400 µl) was added to extract the lipids, and the samples were vortexed again for 3 min at 4 °C. Samples were centrifuged 3 min at 1,000g at 21 °C and the top petroleum ether layer was collected in a new tube. The extracted lipids were dried under argon before being resuspended in isopropanol (40 µl) and transferred to an amber glass vial (Supelco, QSertVial, 12 × 32 mm, 0.3 ml). LC–MS analysis was performed using a Thermo Vanquish Horizon UHPLC system coupled to a Thermo Exploris 240 Orbitrap mass spectrometer. For LC separation, a Vanquish binary pump system (Thermo Scientific) was used with a Waters Acquity CSH C18 column (100 × 2.1 mm, 1.7-µm particle size) held at 35 °C under 300 μl min−1 flow rate. Mobile phase A consisted of 5 mM ammonium acetate in 70:30 (v/v) acetonitrile:water with 125 μl l−1 acetic acid. Mobile phase B consisted of 5 mM ammonium acetate in 90:10 (v/v) isopropanol:acetonitrile with the same additive. For each sample run, mobile phase B was initially held at 2% for 2 min and then increased to 30% over 3 min. The column was re-equilibrated for 5 min at 2% B before the next injection. Five microliters of the sample were injected by a Vanquish Split Sampler HT autosampler (Thermo Scientific), while the autosampler temperature was kept at 4 °C. The samples were ionized by a heated ESI source kept at a vaporizer temperature of 350 °C. Sheath gas was set to 50 units, auxiliary gas to 8 units, sweep gas to 1 unit, and the spray voltage was set to 3,500 V for positive mode and 2,500 V for negative mode. The inlet ion transfer tube temperature was kept at 325 °C with a 70% RF lens. MS acquisition parameters included a resolution of 45,000, HCD collision energy (45% for positive mode and 60% for negative mode) and 3-s dynamic exclusion. Automatic gain control targets were set to standard mode. The resulting CoQ intermediate data were processed using TraceFinder v.5.1 (Thermo Scientific). Raw intensity values were normalized to the CoQ8 internal standard. Cell pellets were extracted with 80% (v/v) methanol, sonicated and homogenized with ceramic beads (Precellys Cryolys). Lysates were centrifuged for 15 min at 15,000g at 4 °C and the supernatant was evaporated to dryness. Dried extracts were reconstituted in methanol according to total protein content as measured by a BCA assay. Samples were analysed by ultra-high performance liquid chromatography–tandem mass spectrometry (UHPLC–MS/MS), using the Triple Quadrupole mass spectrometer (6495 iFunnel, Agilent Technologies) and dynamic multiple-reaction monitoring (dMRM) acquisition mode, following previously described methods85,86. Two complementary liquid chromatography modes coupled to positive and negative electrospray ionization MS, respectively, were used to maximize the metabolome coverage87. Raw UHPLC–MS/MS data were processed using the MassHunter Quantitative Analysis software v.B.07.00 (Agilent Technologies). Extracted ion chromatogram areas for MRM transitions were used for relative quantification. The data, consisting of peak areas of detected metabolites across all samples, were processed and filtered depending on the coefficient of variation (CV) evaluated across quality control samples that were analysed periodically throughout the batch. Peaks with analytical variability above CV of 30% were discarded. This method was used to analyse metabolites in Fig. Cells were collected, washed with PBS and lysed by 10 min incubation on ice in 1× Native Buffer (NativePAGE Sample Prep kit, Invitrogen, BN2008), 1% digitonin (Invitrogen, BN2006), 1:100 protease inhibitor (Thermo Scientific, 87786), 1:500 nuclease (Thermo Scientific, 88702) and 10 mM individual metabolites as indicated. Frozen mouse tissue was homogenized (gentleMACS Octo Dissociator, Miltenyi Biotec) in 50 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1 mM MgCl2, 1% NP-40, 1:100 protease inhibitor (Thermo Scientific, 87786) and 1:500 nuclease (Thermo Scientific, 88702) then centrifuged for 10 min at 3,000g. Supernatant was diluted in the same buffer and 10 mM metabolites were added as indicated. Lysates were centrifuged for 10 min at 20,000 g at 4 °C and the supernatant was saved. Proteins were separated on a 4–16% native gel (Invitrogen, BN1004BOX) with 1× anode buffer (NativePAGE Running Buffer kit, Invitrogen, BN2007) and transferred to nitrocellulose membranes using a wet transfer chamber and buffer consisting of 0.302% (w/v) Tris base, 1.44% (w/v) glycine, 20% ethanol and 0.05% (w/v) SDS in water. Transfer was verified by Ponceau S Staining Solution (Thermo Scientific, A40000278). Membranes were incubated 5 min in 8% acetic acid and washed with water before proceeding to immunoblotting. For standard respirometry, cells were seeded at 125,000 cells per well in Seahorse XF DMEM (Agilent Technologies, 103575-100) supplemented with 25 mM glucose and 2 mM glutamine, centrifuged for 1 min at 100g and incubated 1 h at 37 °C. The oxygen consumption rate was measured by the Agilent Seahorse XFe96 Extracellular Flux Analyzer (Agilent Technologies) using the program XF Cell MitoStress Test and successive treatment with 2 µM oligomycin A (Tocris, 4110), 1.5 µM CCCP (Sigma-Aldrich, C2759) and 1.6 µM antimycin A (Sigma-Aldrich, A8674). To determine antimycin A-dependent respiration, COQ7 clones were instead seeded at 250,000 cells per well and tested with a single injection of antimycin at final concentration 1.6 µM. Data were analysed using Seahorse Wave Desktop Software (Agilent Technologies). The 293T cells were transfected with Flag-tagged NUDT5, NUDT5 mutants, PPAT or GFP as control using lipofectamine 2000 (Invitrogen, 11668019) according to the manufacturer's protocol. HeLa or K562 cells were infected as described above to stably overexpress Flag-tagged PPAT or GFP. Cells were grown to 80% confluency, collected by scraping, washed with PBS and lysed by a 15–30-min incubation on ice in IP buffer (50 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1 mM MgCl2 and 1% NP-40) with 1:100 protease inhibitor (Thermo Scientific, 87786) and 10 mM metabolites as indicated. Lysates were spun for 10 min at 20,000g at 4 °C and supernatants were transferred to new tubes. A 1% volume was kept aside for inputs. Anti-Flag M2 magnetic beads (Millipore, M8823) were washed in IP buffer and incubated 1–16 h with samples at 4 °C with gentle agitation. Beads were washed five times in IP buffer and changed twice to new tubes. Then, 0.1 mg ml−1 3×Flag peptide (Sigma-Aldrich, F4799) in TBS was added to beads in two steps and each time incubated for 30 min at 4 °C with gentle agitation; the supernatant was saved on ice. Then, 100 µl trichloroacetic acid (Sigma-Aldrich, T9159) was added to the supernatant and incubated for 30 min at 4 °C. Samples were centrifuged for 20 min at 20,000g at 4 °C, the supernatant was discarded and pellets were washed in −20 °C acetone. Samples were centrifuged for 10 min at 20,000g, the supernatant was discarded and pellets were heated at 55 °C until dry. The pellets were resuspended in 2× SDS sample buffer for analysis by SDS–PAGE or MS. Cells were cultured 5 days in DMEM–GlutaMAX (Gibco, 31966021) with 10% dFBS (Sigma-Aldrich, F0392) and 100 U ml−1 penicillin–streptomycin (BioConcept, 4-01F00-H). A total of 5 × 106 cells were collected and washed with PBS, and pellets were flash-frozen in liquid nitrogen. Protein extraction, library construction and analyses, and sample analyses were all carried out as described previously88, with minor changes. In brief, proteins were extracted using a modified iST method and dried by centrifugal evaporation. Then, a fraction of each of the eight samples were pooled for library construction and were fractionated by off-line basic reversed-phase fractionation (bRP). Dried bRP fractions were redissolved in 30 µl 2% acetonitrile with 0.5% TFA and 6 µl was injected for LC–MS/MS analysis. LC–MS/MS analyses were carried out on a TIMS-TOF Pro mass spectrometer (Bruker) interfaced through a nanospray ion source to an Ultimate 3000 RSLCnano HPLC system (Dionex) using data-dependent acquisition for library construction and data-independent acquisition for sample analysis, and analysed using Spectronaut v.17.5 (Biognosys), as previously reported88. Samples were immunoprecipitated as described above. Digestion was carried out using the SP3 method89 with magnetic Sera-Mag Speedbeads (Cytiva, 45152105050250). Proteins were alkylated with 32 mM idoacetamine for 45 min at 21 °C in the dark. Beads were added at a ratio 10:1 (w/w) to samples and proteins were precipitated on beads with ethanol at a final concentration of 60%. After three washes with 80% ethanol, beads were digested in 50 µl of 100 mM ammonium bicarbonate with 1 µg trypsin (Promega V5073) and incubated 1 h at 37 °C. The same amount of trypsin was added to samples for an additional 1 h incubation. Supernatant was then recovered, transferred to new tubes, acidified with formic acid at 0.5% final concentration and dried by centrifugal evaporation. To remove traces of SDS, two sample volumes of isopropanol containing 1% TFA were added to the digests and samples were desalted on a strong cation exchange plate (Oasis MCX, Waters) by centrifugation. Digests were washed with 1% TFA in isopropanol then 0.1% formic acid with 2% acetonitrile. Peptides were eluted in 200 µl of 80% acetonitrile, 19% water and 1% (v/v) ammonia, and dried by centrifugal evaporation. Samples were then analysed using LC–MS/MS analyses were carried out on a TIMS-TOF Pro mass spectrometer (Bruker) interfaced through a nanospray ion source to an EvoSep One liquid chromatography system (EvoSep), as described previously90. Cells were collected, washed in PBS and seeded at 106 cells per ml in white flat-bottom 96-well plates (Thermo Scientific, 136101). Cellular ADP and ATP levels were measured using the ADP/ATP Ratio Assay kit (Sigma-Aldrich, MAK135) according to the manufacturer's protocol with a BioTek Plate Reader (Agilent Technologies). Cells were collected, washed with PBS and lysed by a 10-min incubation on ice in Assay Buffer (Abcam, ab155900) at 2 × 107 cells per ml. (Hypo)xanthine concentration was measured using the fluorometric protocol of the Xanthine/Hypoxanthine Assay kit (Abcam, ab155900) in black flat-bottom 96-well plates (Thermo Scientific, 137101). Fluorescence was measured at ex/em 535/587 nm with the BioTek Synergy Plate Reader (Agilent Technologies). For analysis by z-score, sgRNA read count data were processed using an approach previously described28. In brief, data were normalized to reads per million and transformed to log2 space. A log2 fold change for each sgRNA in each medium condition was determined relative to the mean day 7 control read count (before medium switch). The mean log2 fold change was calculated across sgRNAs for each gene and results were averaged across two infection replicates. Low-expression genes (transcripts per million (TPM) < 1 in DepMap 24Q2 dataset68) were used to define the mean and s.d. of a null distribution for each medium condition and z-scores for each gene were defined based on this distribution. These low-expression genes and their corresponding sgRNA were excluded from further analyses. sgRNA read count data from day 28 (day 21 post-infection) were used as input for MAGeCK (v.0.5.9.2)29 with default parameters and using the medium condition supplemented with uridine as the reference. GSEA was performed using GSEA30,31 v.4.2.2 on genes ranked according to ∆Z = Z−uridine – Z+uridine or by gene log2 fold change as calculated by MAGeCK. Enrichment was performed against the GO Biological Processes database c5.go.bp.v2023.2.Hs.symbols or against the MitoPathways database34 without collapse. For analysis using the GO database, maximum size exclusion was set to 50 and minimum to 2. For analysis using the MitoPathways database, minimum size exclusion was set to 15. All other parameters were kept as default. The top 50 negatively enriched pathways were manually annotated for their relationship to either pyrimidine biosynthesis or CoQ metabolism. Metabolites were mapped to their KEGG91 IDs using MetaboAnalyst v.6.0 ‘Compound ID Conversion' tool (https://www.metaboanalyst.ca/). KEGG pathway identifiers for metabolism pathways were taken from their website (https://www.genome.jp/kegg/pathway.html on 24 October 2024), excluding the subsections ‘Xenobiotics degradation and metabolism' and ‘Chemical structure transformation maps'. The list of metabolites for each pathway was obtained using the KEGGREST v.1.44.1 package (https://bioconductor.org/packages/KEGGREST) in R v.4.4.1. These data were used to generate a metabolite database and four additional pathways were added manually, delimiting mature purines or pyrimidines from the intermediate metabolites of their respective de novo synthesis pathways. Metabolite set enrichment analysis was performed using GSEA30,31 v.4.2.2 on metabolites ranked according to their log2 fold change (sgNUDT5/sgCtrl) in abundance. Enrichment was performed against the custom database described above without collapse. Maximum size exclusion was set to 500 and minimum to 3, all other parameters were kept as default. Analyses were conducted with the Perseus software package (v.1.6.15.0)92. Standard protein contaminants were removed according to an established analytical pipeline (available at the Protein Analysis Facility and using the software package above), data were log2-transformed and only proteins quantified in at least four samples of one group were kept. Missing values were imputed based on normal distribution using Perseus default parameters. Student's t-tests were carried out and the log2 fold change over control samples (sgCtrl) was calculated. Analyses were carried out using an in-house developed software tool (available on https://github.com/UNIL-PAF/taram-backend). Standard protein contaminants were removed according to an established analytical pipeline (available at the Protein Analysis Facility and on GitHub, detailed above), data were log2-transformed and only proteins quantified in at least two samples of one group were kept, resulting in 2,949 protein groups. Missing values were imputed based on a normal distribution with a width of 0.3 s.d., downshifted by 1.8 s.d. Student's t-tests were carried out and the log2 fold change over control samples (GFP–3×Flag) was calculated. Data were filtered to exclude proteins that had not been reliably detected in both NUDT5–Flag replicate samples. Remaining data were queried against the Contaminant Repository for Affinity Purification (CRAPome) database H. sapiens Single Step Epitope tag AP-MS93, and proteins with an average spectral score >2.5 were excluded as common IP–MS contaminants. Protein structures and interactions were modelled using AlphaFold Server54, and the highest confidence prediction was selected. ChimeraX v.1.10 was used to view predicted models and to calculate hydrogen bonds. Gene expression (OmicsExpressionProteinCodingGenesTPMLogp1.csv, v.24Q4) and drug treatment (Repurposing_Public_24Q2_Extended_Primary_Data_Matrix.csv) datasets were obtained from the CCLE Dependency Map portal94. For each of the analysed drugs, drug perturbation scores were correlated with gene expression scores across cell lines. Pearson's correlation coefficients were calculated using R v.4.5.0. For thioguanine, drugs named ‘thioguanine' and ‘tioguanine' were combined by taking the mean of their drug perturbation scores for each cell line. All 559 cell lines with gene expression scores and drug perturbation scores for the analysed drugs were included. Statistical analyses as described in the figure legends were performed using Prism v.10 (GraphPad Software) and exact P values are shown where P < 0.05. All data are expressed as mean ± s.e.m. and all Student's t-tests were two-sided. No statistical methods were used to predetermine sample sizes but our sample sizes are similar to those reported in previous publications15,88,90. Data distribution was assumed to be normal but this was not formally tested. Data collection and analysis were not performed blind to the conditions of the experiments, as data blinding is generally not considered relevant to cell culture experiments. Representative immunoblots and native gels are representative of a minimum of three independent replicates. Validation of coimmunoprecipitation in other cell lines was performed once. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All raw MS proteomics data together with raw output tables are available via the Proteomexchange data repository (www.proteomexchange.org) with accession IDs PXD060320 and PXD060353. Source data are provided with this paper, and include statistical data and unmodified images of all immunoblots. Ariav, Y., Ch'ng, J. H., Christofk, H. R., Ron-Harel, N. & Erez, A. Targeting nucleotide metabolism as the nexus of viral infections, cancer, and the immune response. Fairbanks, L. D., Bofill, M., Ruckemann, K. & Simmonds, H. A. Importance of ribonucleotide availability to proliferating T-lymphocytes from healthy humans: disproportionate expansion of pyrimidine pools and contrasting effects of de novo synthesis inhibitors. Tran, D. H. et al. De novo and salvage purine synthesis pathways across tissues and tumors. Jones, M. E. Pyrimidine nucleotide biosynthesis in animals: genes, enzymes, and regulation of UMP biosynthesis. Matsuura, T. & Jones, M. E. in Lipmann Symposium. Energy Transformation in Biological Systems (eds Richter, D. & Lipmann-Symposium) 422–434 (De Gruyter, 1974). Löffler, M., Jöckel, J., Schuster, G. & Becker, C. Dihydroorotat-ubiquinone oxidoreductase links mitochondria in the biosynthesis of pyrimidine nucleotides. del Caño-Ochoa, F. et al. Cell-based analysis of CAD variants identifies individuals likely to benefit from uridine therapy. Frederick, A. et al. Triacetyluridine treats epileptic encephalopathy from CAD mutations: a case report and review. Hereditary orotic aciduria: long-term therapy with uridine and a trial of uracil. Coleman, P. F., Suttle, D. P. & Stark, G. R. Purification from hamster cells of the multifunctional protein that initiates de novo synthesis of pyrimidine nucleotides. Aoki, T., Morris, H. P. & Weber, G. Regulatory properties and behavior of activity of carbamoyl phosphate synthetase II (glutamine-hydrolyzing) in normal and proliferating tissues. Holmes, E. W. Kinetic, physical, and regulatory properties of amidophosphoribosyltransferase. Ben-Sahra, I., Howell, J. J., Asara, J. M. & Manning, B. D. Stimulation of de novo pyrimidine synthesis by growth signaling through mTOR and S6K1. Robitaille, A. M. et al. Quantitative phosphoproteomics reveal mTORC1 activates de novo pyrimidine synthesis. Skinner, O. S. et al. Salvage of ribose from uridine or RNA supports glycolysis in nutrient-limited conditions. Nwosu, Z. C. et al. Uridine-derived ribose fuels glucose-restricted pancreatic cancer. Whyte, D. et al. Uridine phosphorylase-1 supports metastasis by altering immune and extracellular matrix landscapes. Wang, T. et al. Inosine is an alternative carbon source for CD8+ T-cell function under glucose restriction. A. Nucleosides are overlooked fuels in central carbon metabolism. Li, Y. et al. UPP1 promotes lung adenocarcinoma progression through the induction of an immunosuppressive microenvironment. & Singh, P. K. Nucleotide metabolism: a pan-cancer metabolic dependency. Halbrook, C. J. et al. Macrophage-released pyrimidines inhibit gemcitabine therapy in pancreatic cancer. Negative feedback–defective PRPS1 mutants drive thiopurine resistance in relapsed childhood ALL. Induction of thymidylate synthase as a 5-fluorouracil resistance mechanism. King, M. P. & Attardi, G. Human cells lacking mtDNA: repopulation with exogenous mitochondria by complementation. & Gravel, D. On auxotrophy for pyrimidines of respiration-deficient chick embryo cells. Abt, E. R. et al. Metabolic modifier screen reveals secondary targets of protein kinase inhibitors within nucleotide metabolism. To, T.-L. et al. A compendium of genetic modifiers of mitochondrial dysfunction reveals intra-organelle buffering. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Ashburner, M. et al. Gene Ontology: tool for the unification of biology. The Gene Ontology Consortium et al. The Gene Ontology knowledgebase in 2023. Pagliarini, D. J. et al. A mitochondrial protein compendium elucidates complex I disease biology. Guerra, R. M. & Pagliarini, D. J. Coenzyme Q biochemistry and biosynthesis. Fabra, M. A. et al. New variants expand the neurological phenotype of COQ7 deficiency. Asaumi, S., Kuroyanagi, H., Seki, N. & Shirasawa, T. Orthologues of the Caenorhabditis elegans longevity gene clk-1 in mouse and human. Liu, S., Neidhardt, E. A., Grossman, T. H., Ocain, T. & Clardy, J. Structures of human dihydroorotate dehydrogenase in complex with antiproliferative agents. Ito, R. et al. Cleavage of oxidized guanine nucleotide and ADP sugar by human NUDT5 protein. Carreras-Puigvert, J. et al. A comprehensive structural, biochemical and biological profiling of the human NUDIX hydrolase family. Wright, R. H. G. et al. ADP-ribose-derived nuclear ATP synthesis by NUDIX5 is required for chromatin remodeling. Cantor, J. R. et al. Physiologic medium rewires cellular metabolism and reveals uric acid as an endogenous inhibitor of UMP synthase. The novel phosphatase NUDT5 is a critical regulator of triple-negative breast cancer growth. Zha, M. et al. Molecular mechanism of ADP-ribose hydrolysis by human NUDT5 from structural and kinetic studies. Targeted NUDT5 inhibitors block hormone signaling in breast cancer cells. Huttlin, E. L. et al. Dual proteome-scale networks reveal cell-specific remodeling of the human interactome. Architecture of the human interactome defines protein communities and disease networks. Jiao, F. et al. Two-dimensional fractionation method for proteome-wide cross-linking mass spectrometry analysis. & Kelley, W. N. Human glutamine phosphoribosylpyrophosphate amidotransferase. Two molecular forms interconvertible by purine ribonucleotides and phosphoribosylpyrophosphate. King, G., Meade, J. C., Bounous, C. G. & Holmes, E. W. Demonstration of ammonia utilization for purine biosynthesis by the intact cell and characterization of the enzymatic activity catalyzing this reaction. Itakura, M., Sabina, R. L., Heald, P. W. & Holmes, E. W. Basis for the control of purine biosynthesis by purine ribonucleotides. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Muchmore, C. R. A., Krahn, J. M., Smith, J. L., Kim, J. H. & Zalkin, H. Crystal structure of glutamine phosphoribosylpyrophosphate amidotransferase from Escherichia coli. Zha, M., Zhong, C., Peng, Y., Hu, H. & Ding, J. Crystal structures of human NUDT5 reveal insights into the structural basis of the substrate specificity. & Christopherson, R. I. Accumulation of 5-phosphoribosyl-1-pyrophosphate in human CCRF-CEM leukaemia cells treated with antifolates. & Pinedo, H. M. The concentration of 5-phosphoribosyl 1-pyrophosphate in monolayer tumor cells and the effect of various pyrimidine antimetabolites. Peters, G. J., Laurensse, E., Leyva, A., Lankelma, J. & Pinedo, H. M. Sensitivity of human, murine, and rat cells to 5-fluorouracil and 5′-deoxy-5-fluorouridine in relation to drug-metabolizing enzymes. Nakano, J. et al. Evaluations of biomarkers associated with 5-FU sensitivity for non-small-cell lung cancer patients postoperatively treated with UFT. Sawyer, R. C., Stolfi, R. L., Martin, D. S. & Balis, M. E. Inhibition by methotrexate of the stable incorporation of 5-fluorouracil into murine bone marrow DNA. Yang, F. et al. HPRT1 activity loss is associated with resistance to thiopurine in ALL. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Sagi, I. et al. Derivation and differentiation of haploid human embryonic stem cells. Longley, D. B., Harkin, D. P. & Johnston, P. G. 5-Fluorouracil: mechanisms of action and clinical strategies. The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. Lohman, D. C. et al. An isoprene lipid-binding protein promotes eukaryotic coenzyme Q biosynthesis. Structure and functionality of a multimeric human COQ7:COQ9 complex. & Hekimi, S. Minimal mitochondrial respiration is required to prevent cell death by inhibition of mTOR signaling in CoQ-deficient cells. Valeros, J. et al. Rhodoquinone carries electrons in the mammalian electron transport chain. Qi, H., Wright, R., Beato, M. & Price, B. D. The ADP-ribose hydrolase NUDT5 is important for DNA repair. Bessman, M. J., Frick, D. N. & O'Handley, S. F. The MutT proteins or ‘Nudix' hydrolases, a family of versatile, widely distributed, ‘housecleaning' enzymes. McLennan, A. G. The Nudix hydrolase superfamily. An, S., Kumar, R., Sheets, E. D. & Benkovic, S. J. Reversible compartmentalization of de novo purine biosynthetic complexes in living cells. Conversion of PRPS hexamer to monomer by AMPK-mediated phosphorylation inhibits nucleotide synthesis in response to energy stress. Cunningham, J. T., Moreno, M. V., Lodi, A., Ronen, S. M. & Ruggero, D. Protein and nucleotide biosynthesis are coupled by a single rate-limiting enzyme, PRPS2, to drive cancer. The NUDIX hydrolase NUDT5 regulates thiopurine metabolism and cytotoxicity. Lin, J.-M. G. et al. Functional profiling reveals a non-enzymatic role of NUDT5 in repressing purine de novo synthesis. Targeted protein degradation of NUDT5 reveals an unexpected non-enzymatic role in 6-thioguanine-mediated toxicity. Wu, Z. et al. NUDT5 regulates purine metabolism and thiopurine sensitivity by interacting with PPAT. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Lisci, M., Vericel, F., Gallant-Ayala, H., Ivanisevic, J. A. FBXW7 alleviates c-MYC repression of pyruvate carboxylase to support metabolic flexibility. van der Velpen, V. et al. Systemic and central nervous system metabolic alterations in Alzheimer's disease. Medina, J. et al. Single-step extraction coupled with targeted HILIC-MS/MS approach for comprehensive analysis of human plasma lipidome and polar metabolome. Gallart-Ayala, H. et al. A global HILIC-MS approach to measure polar human cerebrospinal fluid metabolome: Exploring gender-associated variation in a cohort of elderly cognitively healthy subjects. Foged, M. M. et al. Cytosolic N6AMT1-dependent translation supports mitochondrial RNA processing. Hughes, C. S. et al. Single-pot, solid-phase-enhanced sample preparation for proteomics experiments. Harhai, M. et al. An updated inventory of genes essential for oxidative phosphorylation identifies a mitochondrial origin in familial Ménière's disease. Kanehisa, M. & Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. The Perseus computational platform for comprehensive analysis of (prote)omics data. The CRAPome: a contaminant repository for affinity purification mass spectrometry data. Defining a cancer dependency map. The authors thank the past and present members of the Jourdain laboratory, M. Quadroni, R. Wright, the Metabolomics Platform (University of Lausanne), the Protein Analysis Facility (University of Lausanne) and L. Abriata from Ecole Polytechnique Fédérale de Lausanne's Protein Structure Core Facility for the molecular docking models made with AlphaFold 3. This work was supported by a grant from the Swiss National Science Foundation (310030_200796 to A.A.J. ), a grant from the Emma Muschamp Foundation (to A.A.J. ), a National Institutes of Health award (R35GM131795 to D.J.P.) and funds from the BJC Investigator Program (to D.J.P.). is an Investigator of the Howard Hughes Medical Institute. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Abigail Strefeler, Sylvain Chollet, Mads M. Foged & Alexis A. Jourdain Department of Cell Biology & Physiology, Washington University School of Medicine, St. Louis, MO, USA Zakery N. Baker, Rachel M. Guerra & David J. Pagliarini Department of Biochemistry and Molecular Biophysics, Washington University School of Medicine, St. Louis, MO, USA Howard Hughes Medical Institute, Washington University School of Medicine, St. Louis, MO, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the manuscript with input from all authors. The authors declare no competing interests. Nature Metabolism thanks Pankaj Singh and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Alfredo Gimenez-Cassina and Jean Nakhle, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Proliferation assay of K562 cells with indicated knockouts over 5 days supplemented, as indicated, with 200 µM cytidine or 25 µM thymidine (6 replicates, two-sided Student t-test with Bonferroni-Dunn correction). The lower dose of thymidine was selected due to toxicity. C. Representation of uridine-sensitized genome-wide knockout screen. K562 cells were infected with the Brunello genome-wide sgRNA knockout library, split into medium with or without 200 µM uridine supplementation, and cultured for 21 days. Data are RPM + 1 or as a fold-change with day 7 post-infection (day of medium switch). Each point is one sgRNA of an expressed gene. E. Gene representation in medium without uridine supplementation relative to medium with 200 µM uridine, as calculated by Z-score (left) or MAGeCK (right) analysis of uridine-sensitized screen using Kolmogorov–Smirnov test. Each point is one gene. F. Ranked gene set enrichment analysis using gene fold-changes from MAGeCK analysis of uridine-sensitized screen and gene ontology biological processes database. Each point is one gene set. G. Ranked gene set enrichment analysis using gene ∆Z = Z-uridine – Z+uridine from uridine-sensitized screen and MitoPathways database. Each point is one gene set. Bar graph data are mean ± SEM. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, media were refreshed 4–6 h prior to harvesting. Abbreviations; CoQ: coenzyme Q, NES: normalized enrichment score, RPM: reads per million, sgCtrl: control sgRNA. A. Immunoblot of COQ7 single cell K562 clones. Superscript numbers refer to clone identification. Proliferation assay of COQ7 clones over 6 days supplemented with 25 mM glucose or galactose (4 replicates, two clones each, p = 2 x 10−15). D. Structures of CoQ10, DMQ10, or brequinar docking in DHODH, as determined experimentally for brequinar39 or using AlphaFold 3 predictive modelling for CoQ10 and DMQ10. E. Oxygen consumption rate (OCR) of COQ7 clones seeded at high density (24 replicates, two clones each). (a): antimycin A. F. Proliferation assay of COQ7 clones over 5 days supplemented, as indicated, with 100 nM antimycin A and 200 µM uridine (4 replicates, two clones each, p = 4 x 10−12, 7 x 10−7, respectively). Statistical tests: two-sided Student t-test. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. Representation of reported NUDT5 enzymatic activities. Proliferation assay of K562 cells with indicated knockouts over 7 days in medium supplemented, as indicated, with 200 µM uridine and presented as a representative example (left) or fold-change (right, 8 replicates). C. Proliferation assay of K562 cells with indicated knockouts over 5 days in Human plasma-like medium (HPLM, 3 replicates). D.Relative metabolite abundances in K562 cells with indicated knockouts from multiple-pathways analysis (4 replicates). E. Relative metabolite abundances secreted into the medium of indicated knockouts over 6 h, normalized to medium alone (not shown). G. Immunoblot of NUDT5 depletion in MCF7, 293 T, or HeLa cells. H. Relative metabolite abundances in MCF7, 293 T, or HeLa cells with indicated knockouts (3 replicates) and using two distinct analytical approaches for metabolite detection (Method 1: multiple-pathways targeted, Method 2: targeted). Superscript numbers refer to clone identification. J. Immunoblot of NUDT5 clones with indicated cDNA complementation. K. Relative metabolite abundances in K562 cells treated for 36 h with 10 µM TH5427 or DMSO (4 replicates). L. Immunoblot of cells in (J). M. Relative m + 5 metabolite abundances in NUDT5 clones complemented with indicated cDNAs and cultured for 5 h with U-13C-glucose tracer (4 replicates). Bar graph data are mean ± SEM. Statistical tests: two-sided Student t-test. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. For metabolomics analysis, media were refreshed 4–6 h prior to harvesting. Abbreviations; 8-oxo-dGDP: oxidized deoxyguanosine diphosphate, ADPR: adenosine diphosphate-ribose, AICAR: 5-phosphoribosyl-4-carbamoyl-5-aminoimidazole, Carb-Asp: carbamoyl-aspartate, PRPP: phosphoribosyl pyrophosphate, R5P: ribose-5-phosphate, SAICAR: 5'-phosphoribosyl-4-(N-succinylcarboxamide)-5-aminoimidazole, sgCtrl: control sgRNA. Global proteomics comparing K562 cells with NUDT5 knockout against control. Each point is one protein. B. Coimmunoprecipitation from 293 T using FLAG-tag as bait. NUDT5E112Q is a catalytic inactive mutant. C. Proliferation assay of NUDT5WT K562 clones with indicated knockouts over 5 days in medium supplemented, as indicated, with 200 µM inosine (3 replicates, two clones each, p = 4 x 10−6). Bar graph data are mean ± SEM. Statistical tests: two-sided Student t-test. Abbreviations; IP: immunoprecipitation, PRPP: phosphoribosyl pyrophosphate, sgCtrl: control sgRNA. Structures of bacterial purF determined experimentally in complex with AMP55,56 and AlphaFold 3 predicted structure of human PPAT. Monomers were isolated from the tetramer structures. Structure of human NUDT5 dimer as determined experimentally57 or through AlphaFold 3 predictive modelling. Monomers were isolated from the dimer structures. C. AlphaFold 3 predicted alignment error of indicated NUDT5–PPAT complexes. D. Sequence alignment of NUDT5 and PPAT proteins across vertebrate species in regions predicted by AlphaFold 3 to mediate the NUDT5–PPAT interaction (black boxes). E. Immunoblot of NUDT5 clones with indicated cDNA complementation. F. Coimmunoprecipitation from K562 or HeLa cells using FLAG-tag as bait with 10 mM PRPP in the lysis buffer or left untreated. G. Quantification of NUDT5 in complex with PPAT from K562 cells in response to a dose curve of PRPP in the lysis buffer, calculated from native PAGE (2 replicates). H. Parallel native PAGE across seven cell lines with 10 mM PRPP in the lysis buffer or left untreated. Percentage NUDT5 in complex with PPAT or as a homodimer as quantified from native gels in H (left) and Fig. All sgRNA-treated cell lines were analysed or further treated 8 days after sgRNA transduction. Abbreviations; IC50: half-maximal inhibitory concentration, IP: immunoprecipitation, PRPP: phosphoribosyl pyrophosphate, sgCtrl: control sgRNA. A. Pearson correlations of gene expression of 19,187 genes with resistance to indicated nucleobase analogues (drug perturbation scores) based on 559 cell lines. a, sgRNA read count data from expressed genes. For each sgRNA, read counts and reads per million are indicated for two replicate infections on day 7 (day of medium switch) or day 28 post-infection in medium supplemented with 200 µM uridine (plus-u condition) or water (minus-u condition). b, z-score analysis of screen. c, MAGeCK analysis of screen using robust rank aggregation algorithm. f, Required gene sets from GSEA ranked by ∆Z = Z-uridine – Z+uridine and using MitoPathways gene sets. GSEA data are analysed using a Kolmogorov–Smirnov test. a, Abundance of metabolites detected through multiple-pathways targeted metabolomics on K562 cells depleted of NUDT5 or control and associated statistics (two-sided Student's t-test with false discovery rate correction for multiple testing). b, Metabolite set enrichment analysis of above. Pathway names follow KEGG nomenclature, except where added manually to distinguish nucleotide de novo synthesis. For metabolomics analysis, the medium was refreshed 4–6 h before collection. a, Abundance of (m + 0) and (m + 5) labelled metabolites in NUDT5 clones complemented with indicated cDNAs and cultured for 5 h with U-13C-glucose tracer. a, Label-free quantification of proteins detected through global proteomics on K562 cells depleted of NUDT5 or control, including statistics calculated using imputed values. b, Label-free quantification of proteins detected through IP–MS proteomics of Flag-tagged NUDT5 or GFP control, including statistics calculated using imputed values and average spectral counts (SCs) from the CRAPome database. Missing values are shown as ‘NaN'. sgRNA and cDNA sequences used in this study for individual gene depletion or expression. a, Mass table for targeted metabolomics analysis. b, Timing details of liquid chromatography methods for targeted metabolomics analysis. AlphaFold 3 molecular modelling of PPAT tetrameric complex and association with two dimers of NUDT5. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Strefeler, A., Baker, Z.N., Chollet, S. et al. Uridine-sensitized screening identifies demethoxy-coenzyme Q and NUDT5 as regulators of nucleotide synthesis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-65690-3'>A large-scale human toxicogenomics resource for drug-induced liver injury prediction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 10:27:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Drug-Induced Liver Injury (DILI) remains one of the most critical challenges in drug development, causing patient safety concerns, clinical trial failures and drug withdrawals. We introduce ToxPredictor, a toxicogenomics framework combining RNA-seq data from primary human hepatocytes with pharmacokinetic data to predict dose-resolved DILI risks and safety margins. At its core is DILImap, an RNA-seq library tailored for DILI research, comprising 300 compounds at multiple concentrations. ToxPredictor achieves 88% sensitivity at 100% specificity in blind validation, outperforming state-of-the-art methods. It flagged recent phase III clinical failures, including Evobrutinib, TAK-875, and BMS-986142, overlooked by animal studies. Beyond prediction, ToxPredictor provides mechanistic insights into hepatotoxic pathways, enabling early de-risking and actionable safety decisions. Unlike single-endpoint readouts—even from 3D models—transcriptomics offers a multi-dimensional system-level view of hepatocyte responses, capable of detecting diverse DILI mechanisms not captured by conventional assays. Scalable, actionable, and integrated into a broader AI/ML drug discovery platform, this work establishes toxicogenomics as a promising tool for developing safer therapeutics and addressing one of the most pressing challenges in toxicology. Drug-Induced Liver Injury (DILI) presents a poorly understood late-stage challenge in drug development, costing an estimated $350 million annually per pharmaceutical company1. Its rarity and unpredictability in clinical populations, often less than 1 in 10,000 persons, hinder detection in clinical studies, masking its severity until post-market exposure. Animal models fail to identify about half of the pharmaceuticals that exhibit clinical DILI2. This makes DILI a leading cause of drug candidate failure and market withdrawal, impeding the development of new therapies3. DILI arises from complex, multifactorial mechanisms, involving dose-dependent intrinsic mechanisms and with current methods unpredictable idiosyncratic reactions4,5 influenced by genetic predisposition, environment, and individual health status6. DILI involves various cellular disruptions including mitochondrial dysfunction, oxidative stress, bile acid imbalance, inhibition of specific enzymes or transporters, and reactive metabolites formation7,8. However, the precise mechanisms and contributing factors are not fully delineated4 and the lack of biomarkers hampers early detection9. Pre-clinical methods, such as quantitative structure-activity relationship (QSAR) models, offer low specificity and binary predictions lacking mechanistic insights10,11. In vitro models use diverse cell sources (e.g., HepG2, THLE, HepaRG cells, primary human hepatocytes) with endpoints ranging from cytotoxicity markers (e.g., LDH, ATP) to mechanistic assessments using high-content imaging (HCI) and multi-parametric strategies. 3D liver models aim to better mimic human tissue characteristics and in vivo responses12. However, despite improved physiological relevance, these models remain constrained by low-dimensional readouts—typically a limited panel of markers such as ATP levels, LDH release, or imaging-based features—which fail to capture the full spectrum of molecular responses and often miss the mechanistic cause. This gap continues to result in late-stage drug withdrawals and clinical failures, highlighting the urgent need for more comprehensive and predictive DILI models9. Inspired by the idea of viewing cells as complex systems, we adopt a machine learning-driven toxicogenomics approach to analyze how DILI-associated compounds alter gene expression, identifying early gene signatures indicative of liver injury. Encompassing the interplay of pathways in response to toxic compounds, we aim to decipher the molecular mechanisms underlying DILI. Utilizing resources like DILIrank13 and LiverTox14 for drug categorization into DILI positives and negatives, we employed the TG-GATES15 microarray database for an initial proof-of-concept, enabling us to accurately predict DILI with 62% sensitivity and 92% specificity in blind validation. Building on this initial success, we created DILImap, a purpose-built and significantly expanded transcriptomic library designed to capture a broader spectrum of DILI mechanisms. DILImap features full-transcriptome RNA-seq data from 300 compounds profiled at multiple concentrations in primary human hepatocytes (PHHs), making it, to the best of our knowledge, the largest toxicogenomics dataset available for DILI modeling. ToxPredictor, our random forest-based machine learning model trained on DILImap, achieves 88% sensitivity (29/33 DILI positives) at 100% specificity (14/14 DILI negatives) in blind validation. It outperforms 20+ pre-clinical models in a head-to-head comparisons, including mechanistic assays16,17,18,19,20,21, cytotoxicity markers22,23,24,25,26,27, physicochemical properties28, bioactivation29, BSEP30 approaches, and the latest in-silico models31,32,33, effectively identifying DILI risks in drugs previously overlooked by traditional models. To the best of our knowledge, it is the first pre-clinical model to flag DILI risks in high-profile clinical failures, such as Evobrutinib, TAK-875 and BMS-986142, all recently withdrawn in phase III trials due to liver injury despite clean preclinical profiles. The model provides dose-resolved predictions and mechanistic insights, demonstrating its utility for prioritizing safer drug candidates. Beyond generalization to unseen compounds, the model has a distinct edge in its mechanistic breadth. The model leverages the full transcriptomic landscape to detect DILI-related mechanisms such as mitochondrial dysfunction, oxidative stress, immune activation, and metabolic perturbation—often well before cell death. These advantages are particularly evident when compared to high-content 3D liver assays26,34,35,36, which, while physiologically relevant, are typically constrained to low-dimensional viability or imaging endpoints. In head-to-head comparisons, our model uniquely identifies non-cytotoxic risks missed by 3D assays. This systems-level resolution enables more comprehensive and unbiased detection of toxic liabilities across diverse compound classes. This work, integral to a broader AI/ML drug discovery platform, aims at enhancing predictive power and operational efficiency in drug development. It showcases that the shift from a single-target to a systems-level perspective holds great promise and positions machine learning in toxicogenomics as significant enhancement to existing methods. To advance the field and foster collaborative innovation, we have made our open-source model and validation data publicly available at dilimap.org, providing a powerful tool for de-risking drug candidates and setting the stage for a paradigm shift in safety evaluations. We have created DILImap, a comprehensive RNA-seq library tailored for drug-induced liver injury (DILI) modeling, encompassing 300 compounds tested at four concentrations. As the most extensive toxicogenomics resource to date, DILImap includes a curated selection of DILI-positive and DILI-negative compounds that span a wide range of known DILI mechanisms, including well-documented liver-injuring drugs and idiosyncratic compounds with no characteristic signature (Fig. A We created DILImap, the largest toxicogenomics dataset to date, encompassing RNA-seq profiles from 300 compounds tested at multiple concentrations in primary human hepatocytes. Compounds span a range of DILI mechanisms, idiosyncratic effects, and non-DILI controls, ensuring broad mechanistic coverage. This resource forms the foundation for training our DILI model. B ToxPredictor, our machine learning model trained on the DILImap, learns pathway signatures indicative of DILI risk. C ToxPredictor de-risks new drug candidates by estimating DILI risk and safety margins. In blind validation, it achieved 88% sensitivity (29/33 DILIs detected) and 100% specificity (all 14 non-DILI compounds accurately classified). This framework sets the stage for a paradigm shift in safety evaluations. All compounds were screened in sandwich-cultured primary human hepatocytes (PHHs), the gold standard and most physiologically relevant in vitro model for liver toxicity, which preserve key hepatic functions such as metabolic activity and bile canaliculi formation37,38. Each compound was tested in triplicate across six concentrations using lactate dehydrogenase (LDH) and Adenosine Triphosphate (ATP) cell viability assays. RNA-seq profiling was performed at four selected doses, spanning the pharmacologically relevant range from therapeutic plasma Cmax to the highest tolerated non-cytotoxic dose just below the IC₁₀ threshold (Supplementary Fig. We selected a 24-hour post-exposure time point, based on the trade-off between signal strength and cellular viability: earlier time points (e.g., 2 h or 8 h) yield weaker transcriptional responses39, while longer incubations risk hepatocyte de-differentiation and RNA degradation40. This strategy allowed us to capture early transcriptional responses without compromising RNA integrity. Marker analysis confirmed retention of hepatocyte identity at 24 hours. We further ensured data quality by including only wells with sufficient total RNA counts and low mitochondrial RNA content, indicating viable, transcriptionally active cells. This streamlined workflow—including automated solubility testing, viability screening, and IC₁₀-based dose selection—enabled us to profile 300 compounds in four months, including 110 drugs tested preclinically for the first time as part of a systematic benchmark (Supplementary Fig. To support comprehensive benchmarking, we provide detailed annotations for each compound, including clinical DILI labels13, DILI mechanisms14, molecular information41, consensus plasma Cmax from various studies10,19,20,23,24,25,28,42,43, and DILI classification results from over 20 pre-clinical studies (Supplementary Data S1–S4). Compounds were categorized based on DILIrank13 and LiverTox14 as follows (Supplementary Fig. Likely DILI: drugs with documented liver injury cases (Most-DILI-Concern or LiverTox score A/B). Idiosyncratic DILI: rare cases without clear dose–response (LiverTox score C/D, <12 case reports). Unlikely DILI: discordant or weak evidence across databases (Less-DILI-concern, but LiverTox score E). Withdrawn, Known, and Likely DILI serve as positive controls; No DILI as negative controls; while Idiosyncratic and Unlikely DILI, due to their label ambiguity, are excluded from training and reserved for downstream testing. For blind validation, a separate experiment was conducted using an independent set of 51 compounds (33 DILI + , 14 DILI-, and 4 with unknown labels, including real-world clinical failures). This carefully curated dataset provides a robust foundation for predictive modeling and mechanistic insights into DILI. ToxPredictor, a machine learning model trained on our DILImap library, predicts DILI risk from pathway-level transcriptional signatures. These signatures are derived through enrichment analysis (WikiPathways44, FDR-adjusted p-values) of genes differentially expressed between compound- vs. DMSO-treated samples using DESeq245, computed for each dose of every compound in DILImap (Fig. For model training, we used only compounds with unambiguous DILI labels, resulting in a high-confidence training set of 111 DILI+ (Withdrawn, Known, Likely) and 52 DILI− (No DILI), while the remaining training data were held out to assess model robustness. To ensure high-confidence DILI labels, we further restricted training to drug concentrations tested at more than 20x of their clinical Cmax to reduce the risk of false-negative labeling for DILI+ compounds that may appear safe at lower doses. For 5-fold cross-validation, we applied stratified, compound-level splitting to ensure that all doses and replicates of a given compound were held out together in each fold, mimicking real-world generalization to unseen compounds. From 193 tested configurations across eight model classes, we selected a Random Forest classifier for its strong validation AUC, minimal overfitting, and highest consistency across folds. These properties, combined with its interpretability, motivated its choice over more complex boosting and deep learning models (Supplementary Fig. The final model is an ensemble of 30 Random Forest models (ensemble members) trained on different cross-validation splits, which together enhance generalization and prediction stability. The ensemble size was chosen based on empirical benchmarking that showed stable test AUC and consistency between models (Supplementary Fig. By estimating DILI probabilities across dose levels, ToxPredictor enables calculation of drug safety margins, defined as the ratio between the first predicted DILI dose (i.e., the lowest dose with predicted probability >0.7) and the maximum plasma concentration (Cmax) at therapeutic levels. This provides a transcriptomics-based surrogate of the clinical therapeutic window. A safety margin threshold of 80 provides an actionable classification into high- and low-risk compounds. The probability threshold of 0.7 and margin of safety (MOS) cutoff of 80 were both optimized on held-out training data to reach performance plateaus while minimizing false positives. Our selected MOS threshold of 80, while on the higher end of literature-reported ranges (10–100)17,23,28,34,36, reflects the greater sensitivity of transcriptomic assays compared to cytotoxicity or mechanistic readouts. Since transcriptional changes often occur at lower doses—before overt toxicity—a higher cutoff is needed to avoid false positives and maintain high specificity in a transcriptome-based model (Supplementary Fig. Among available exposure measures, we used total Cmax instead of free Cmax due to its broader availability across compounds. Both measures showed comparable predictive performance, with total Cmax performing slightly better, possibly due to more robust consensus values derived from a greater number of studies (Supplementary Fig. All model selection, hyperparameter tuning, and threshold optimization were performed exclusively on the training data. For final evaluation, we used a fully independent blind-validation set of 51 compounds (33 DILI + , 14 DILI − , and 4 unknowns), profiled in a separate experiment using separate plates and sequencing runs. This set was withheld from all stages of model development. Compound selection for the validation study was finalized prior to training and intentionally enriched for withdrawals and recent clinical failures. The four unknowns represent compounds currently in clinical use or trials without confirmed DILI liability (Supplementary Table S1). In blind validation, the model achieved 88% sensitivity, correctly identifying 29 of the 33 DILI+ compounds, and 100% specificity, with all 14 DILI− compounds correctly classified as safe (Fig. These results represent a substantial improvement over our initial proof-of-concept model trained on TG-GATES microarray data, which achieved 62% sensitivity and 92% specificity. Leveraging our DILImap library, ToxPredictor substantially improved both sensitivity of 88% and specificity of 100% on the same validation set (Fig. This enhancement is attributed to DILImap's larger dataset with broader mechanistic coverage and the higher resolution of RNA-seq over microarrays, enabling better gene detection and a wider quantitative range for expression level changes compared to microarrays46. Notably, post-market withdrawals, missed in both pre-clinical models and clinical trials, were most confidently flagged by our model as high DILI risk. Each data point represents the safety margin of an individual compound. Boxplots show the median (center line), interquartile range (box), and data within 1.5 × interquartile range (whiskers). The safety margin—calculated as the ratio of the first toxic dose to the therapeutic maximum plasma concentration (Cmax)—effectively separates DILI-positive from non-DILI compounds in blind validation using a safety margin threshold of 80 (dashed line). These results highlight the ability of transcriptomics-based modeling to improve DILI detection beyond traditional methods, offering enhanced sensitivity for withdrawn and known high-risk compounds while maintaining high specificity for non-DILI controls. Our DILI safety margin and classification is derived from three parameters: Cmax (baseline concentration), cell viability assays (indicating cell death), and transcriptomics (based on differential pathways). To assess each parameter's contribution, we assessed their ability to classify DILI cases independently. Out of 144 DILIs, 29 were detected solely based on plasma Cmax (>25 μM) and 42 through the LDH cytotoxicity assay (safety margin <80), both at ≥90% specificity. Combining our transcriptomics-based model with Cmax and LDH data was most effective, identifying 110 out of 142 DILIs (safety margin <80), underscoring the added value of toxicogenomics in DILI detection beyond mere cell death (Supplementary Fig. ToxPredictor achieved a ROCAUC of 0.82 in cross-validation, compared to 0.66 for viability alone. In blind validation, it achieved a ROCAUC of 0.96, compared to 0.65 for using viability alone (Supplementary Fig. Our model highlights distinct DILI profiles among closely related COX-2 inhibitor non-steroidal anti-inflammatory drugs (NSAIDs) and imparts unique mechanistic insights linking predictions to mechanisms such as hepatocellular injury, oxidative stress, and mitochondrial dysfunction. 3A), while Sulindac, an arthritis treatment with rare but established idiosyncratic DILI cases, and Lumiracoxib, withdrawn due to severe liver failures, are flagged as DILI risks with safety margins below the classification threshold of 80 (Fig. A Valdecoxib, deemed safe, is predicted no DILI risk across all dose levels, with a margin of safety (MOS) greater than 300. B Sulindac, associated with idiosyncratic DILI, and Lumiracoxib, withdrawn from the market due to severe liver failure, are both flagged as high-risk by the model with MOS values of 31 and 1, respectively. C The predictions are driven by pathways implicated in DILI. Differential pathway signatures highlight key mechanisms underlying DILI risk, include oxidative stress, drug metabolism, receptor-mediated signaling, fat and cholesterol synthesis, and bile acid regulation. Pathway enrichment was performed using Enrichr (gseapy), based on a Fisher's exact test with Benjamini–Hochberg correction. D Dose-resolved DILI risk probabilities illustrate how increasing dosages impact the likelihood of liver injury. Dots indicate mean predictions and blue shaded confidence intervals represent the standard deviations across ensemble models. These predictions align closely with clinical outcomes, demonstrating its utility in DILI risk stratification, pathway-based mechanistic understanding, and guiding safer drug development. Crucially, it highlights the pathways implicated in DILI, encompassing direct contributors like oxidative stress leading to cell injury, as well as indirect factors such as disturbances in fatty acid metabolism, which can be particularly relevant to explain idiosyncratic effects (Suppl. Sulindac, for example, is linked to disruptions in fatty acid synthesis and cholesterol biosynthesis, aligning with recent studies connecting it to hepatic steatosis47. By pinpointing these pathways, the model provides mechanistic insights into idiosyncratic DILI, offering an understanding previously thought unpredictable (Fig. The model assesses DILI risks in a dose-resolved manner, revealing how dosage impacts liver injury likelihood. A key demonstration is its accurate prediction of Sulindac's DILI risk, despite it being believed to be unpredictable in a dose-resolved manner48. These results highlight the model's ability to deliver actionable predictions and enable targeted optimization of drug safety profiles by focusing on critical pathways (Fig. DILI arises from disruptions in diverse pathways. Our model highlights pathways with high predictive value (AUC ≈ 0.8) strongly associated with DILI risk, including amino acid metabolism (toxic metabolite buildup causing oxidative stress and liver injury), fatty acid biosynthesis (disruptions leading to lipid accumulation and hepatocyte damage), tryptophan metabolism (toxic intermediates driving oxidative stress and inflammation) and ferroptosis (iron-dependent oxidative stress leading to lipid peroxide accumulation)49,50,51,52,53. Additionally, pathway activations highly correlated with predicted DILI risk include nuclear receptor signaling (e.g., PXR/CAR/FXR), one-carbon metabolism, and bile acid regulation—highlighting transcriptional reprogramming and metabolic stress as key contributors to hepatotoxicity54,55,56 (Fig. When compounds are ranked by predicted DILI probabilities, a clear gradient of pathway activation emerges, revealing distinct enrichment patterns for these biological processes. This correlation reinforces the direct mechanistic relevance and interpretability of the model's predictions and highlights these pathways as potential drivers of DILI (Fig. A Pathways are ranked by their ability to distinguish DILI vs. non-DILI compounds, measured by AUC. To quantify the strength of association between pathway dysregulation and predicted DILI risk, we show the Spearman correlation between pathway activation scores and DILI probability across compounds. This highlights transcriptional and metabolic stress pathways—such as nuclear receptor signaling, one-carbon metabolism, and bile acid regulation—as key contributors to DILI risk. B Compounds are ordered by their DILI risk probability, showing a correlation between DILI risk and the activation of pathways associated with liver injury. Pathway enrichment was performed using Enrichr (gseapy), based on a Fisher's exact test with Benjamini–Hochberg correction. The most affected pathways include those critical to drug metabolism, oxidative stress and lipid homeostasis. Key pathways include Cytochrome P450 Oxidation, NRF2-ARE regulation for oxidative stress response, and Fatty Acid Beta-Oxidation, highlighting the interplay between detoxification, mitochondrial function, and energy metabolism. Additionally, nuclear receptor pathways such as Pregnane X Receptor (PXR) and Sterol Regulatory Element-Binding Protein (SREBP) signaling reveal disruptions in lipid and bile acid regulation. Top 100 genes most frequently upregulated in DILI cases include those related to drug metabolism (e.g., CYP1A2, CYP51A1, UGT1A8, AKR1C1, AKR1C2, AKR1C3), drug transport (e.g., SLC3A2), stress response (e.g., TXNRD1, SRXN1, GLRX, GLRX3, GCLM, HSP90AA1, HSP90AB1, HSPE1), and lipid metabolism (e.g., PLIN2, INSIG1, SREBF1, SCD, LPIN2, FADS2). Less commonly studied but significant genes include those involved in inflammation (e.g., GDF15, TNFRSF12A, S100A10, LITAF), autophagosome formation (e.g., MAP1LC3B, SQSTM1), and mitochondrial function (e.g., VDAC2, RAN, CHCHD10). D Top 100 genes most frequently downregulated in DILI cases include those involved in drug metabolism and detoxification (e.g., CYP4A11, CYP2E1, AKR1C4, GSTA1, GSTA2, UGT2B10, UGT2B15), drug transport (e.g., SLC10A1, SLC22A1, SLC22A7, SLC38A3, SLC27A5), and protein processing (e.g., ALB, AHSG, TTR, APOA1, APOE). Genes associated with amino acid metabolism and mitochondrial function (e.g., MAT1A, ARG1, CPS1, HMGCS2, ACAA1, ACAA2, ECHS1) also show significant downregulation. Key oxidative stress regulators and redox enzyme (e.g., CAT, ABAT, GNMT, and DHTKD1) are also reduced. Pathways related to lipid metabolism (e.g., CIDEB, ACSM2A, ACSM2B, ACSM5) and coagulation or inflammatory responses (e.g., SERPINF2, SERPINC1, SERPINA6, SERPINA10, FGB, FGG, FGA) are prominently affected. Notable genes with less established links to DILI but showing significant downregulation include ITIH4, PGLYRP2, BHMT, and GUCA2B, which could represent novel mechanisms or pathways contributing to the progression of liver injury. The changes in gene expression reflect early cellular alterations that may lead to DILI and highlight the complexity of DILI mechanisms, which encompass multiple aspects of liver function. To identify genes most significant for DILI, we determined the frequency at which each gene was differentially up- or downregulated across DILI drugs in our library, using an adjusted p-value threshold of 0.05. This analysis focused on the concentrations at which toxic effects were first predicted, aiming to uncover early upstream regulators potentially driving DILI. Most frequently up-regulated genes were associated with drug metabolism, transport, stress response, and lipid metabolism. Novel genes linked to inflammation, autophagy, and mitochondrial dysfunction were also implicated (Fig. Frequently down-regulated genes include those critical for liver functions such as drug metabolism, transport, lipid metabolism, amino acid metabolism, mitochondrial function, coagulation and inflammatory responses. Altered expression of these genes may serve as early indicators of liver injury and reflect DILI's multifaceted mechanisms (Fig. Establishing that the DILI pathways and genes identified by our model are specific to liver toxicity rather than general toxicity is inherently challenging. However, the model's precision is evident in its accurate classification of non-DILI compounds with known toxicities in other systems, such as Valdecoxib (cardiovascular toxicity), Bupropion (neurologic and cardiovascular toxicity), and Warfarin (hematologic toxicity)57,58,59, indicating the model's ability to distinguish liver-specific toxicity from other forms of organ damage. Bruton tyrosine kinase (BTK) inhibitors, despite their promise in oncology and autoimmune diseases, have faced clinical holds due to liver injury. Recent examples include Evobrutinib, BMS-986142, and Orelabrutinib, all of which were withdrawn or put on hold in phase III in 2023 due to DILI cases. We validated ToxPredictor on four clinical failures: Evobrutinib, BMS-986142, Orelabrutinib, and TAK-875 (type 2 diabetes drug), along with two investigational BTK inhibitors (Rilzabrutinib, Remibrutinib) and two FDA-approved JAK inhibitors (Tofacitinib, Upadacitinib) as negative controls. All clinical failures were flagged as high or medium-high risk with low MOS values, particularly TAK-875, Evobrutinib, and BMS-986142, consistent with their phase III withdrawals. The investigational drugs were classified as medium risk (MOS = 14) and low risk (MOS = 101), which have not yet been linked to DILI in clinical studies yet, while the DILI-negative JAK inhibitors were classified as low risk. These results align closely with their clinical safety profiles (Fig. A. ToxPredictor provides outputs including the first DILI concentration, safety margin, and corresponding risk classification. Clinical validation confirms that drugs labeled as high risk by the model, such as TAK-875, Evobrutinib, and BMS-986142, were recently withdrawn and halted in Phase III trials due to DILI. Conversely, low-risk drugs such as Tofacitinib and Upadacitinib, as classified by the model, exhibit no DILI association. B Model-derived DILI risk likelihood curves for representative compounds are plotted against hypothetical Cmax values. These curves delineate dose regimes associated with low, medium, mid-high and high DILI risk, enabling dose recommendations within the low-risk range. Risk classification is determined based on the actual efficacious Cmax of each compound: Evobrutinib is classified as high risk, Rilzabrutinib as medium risk (recommended dose <100 mg q.d. ), and Remibrutinib as low risk (recommended dose <155 mg q.d.). This demonstrates the model's utility in guiding dose selection to minimize DILI risk. ToxPredictor provides dose-dependent DILI risk curves derived from empirical DILI likelihoods across various hypothetical Cmax values, enabling safe dosing recommendations (see Methods). For instance, Rilzabrutinib is categorized as low risk at doses below 100 mg q.d., which is lower than its efficacious dose of 400 mg. In contrast, Remibrutinib's efficacious dose of 100 mg falls within the recommended low-risk range of <155 mg q.d. These findings highlight ToxPredictor's value in informing safe dosing strategies, making it a valuable tool for de-risking new drug candidates (Fig. We benchmark our model along two key axes: predictive performance and scalability. Predictive performance, measured by balanced accuracy, reflects the model's ability to distinguish DILI-positive from DILI-negative compounds. Scalability captures both technical throughput and biological breadth—the capacity to generalize across diverse chemistries and mechanisms, including previously uncharacterized ones (Fig. A Balanced accuracy versus scale (number of DILI + /− compounds) across published preclinical models. ToxPredictor achieves the highest performance among both in vitro and in silico methods, outperforming high-content imaging, cytotoxicity, bioactivation-based and structure-based models, demonstrating robust performance even when scaled to hundreds of compounds. B Head-to-head comparison on overlapping compounds shows consistent performance gains over other studies. Results highlight transcriptomics-based toxicogenomics as a leading strategy for mechanistically informed, scalable DILI prediction. Our model outperforms a wide range of pre-clinical DILI models, including mechanistic assays16,17,18,19,20,21, cytotoxicity markers22,23,24,25,26,27, physicochemical properties28, bioactivation29 and BSEP30 approaches. In a head-to-head comparison across matched compound sets, it identified 46 out of 66 DILI cases versus the 27 out of 66 identified by Xu et al.17 HCI assay (49/66 vs 27/66); it shows superior performance over Garside et al.20 HCI assay (37/46 vs 29/46 DILIs), Vorrink et al.26 cytotoxicity assay using CD spheroids (37/43 vs 30/43 DILIs), Sakatis et al.29 bioactivation endpoint GSH adduct (47/65 vs 25/65) as well as their combined assay integrating covalent binding and dose (47/65 vs 32/65). When compared to Kohonen et al.'s transcriptomics-based cytotoxicity model60, our approach showed improved sensitivity (26/36 vs. 16/36 DILIs). These comparisons, all at 100% specificity evaluated on the same compounds, underscore the added value of our systems-level, mechanism-agnostic readout (Fig. These findings are slightly below the balanced accuracy of 0.59 reported by Seal et al. (2024)33 for DILIPredictor. On a benchmark subset of unseen compounds overlapping with DILImap (n = 97), TxGemma reached 63% sensitivity (39/62) and 57% specificity (20/35), while our model achieved 76% sensitivity (47/62) and 86% specificity (30/35). Similarly, DILIGeNN showed perfect sensitivity (5/5) at moderate specificity (2/3), while our model reached 100% on both (5/5 and 3/3). DILIPredictor reached complete sensitivity (23/23) but at the expense of poor specificity (1/7), while ToxPredictor maintained high sensitivity (20/23) at markedly higher specificity (5/7). Low specificity is a key limitation of structure-based models, which lack biological context and tend to over-call toxicity. This results in false positives for commonly prescribed drugs with no risk of hepatotoxicity, such as biotin (flagged DILI+ by DILIPredictor), vitamin D (flagged DILI+ by DILIGeNN), and pemetrexed (flagged DILI+ by all three models). Moreover, they provide only binary outputs, without dose or mechanistic insight. In contrast, transcriptomics enables dose-resolved predictions, mechanistic interpretability, and safety margin estimation—critical for evaluating toxic liabilities and guiding follow-up experiments (Supplementary Fig. 3D liver systems offer important physiological context. High-content imaging in 3D models, such as those by Walker et al.34 and Ewart et al.35, achieves similar performance on small, curated panels (Walker: 23/27 vs. 23/27; Ewart: 11/14 vs. 13/14). They may perform well on narrow, curated panels, but struggle with unknown mechanisms or mechanisms not captured by the low-dimensional endpoint, as shown in the following comparison. To explore the unique capabilities of 2D transcriptomics vs 3D cytotoxicity assays, we conducted direct compound-level comparisons with larger 3D screening studies: Vorrink et al.26 and Fäs et al.36 In Vorrink et al. 3D cytotoxicity uniquely detected 3 compounds (Fialuridine, Methotrexate, Trazodone), all linked to cytotoxic effects that result in acute cell death. Conversely, our model uniquely flagged 10 compounds—including fluconazole, phenytoin, and zileuton—associated with immune activation, metabolic stress, or enzyme modulation, which are not readily captured by viability endpoints. A similar pattern emerged in the Fäs et al. study: 3D cytotoxicity exclusively identified 4 compounds (e.g., Haloperidol, Fialuridine) whose toxicities depend on structural or metabolic context. Our model uniquely identified 5 compounds (e.g., Cimetidine, Fluconazole, Ximelagatran) marked by subtle transcriptional responses rather than overt cell death. These comparisons highlight a key limitation of fixed single-endpoint models: while effective in narrow contexts, they struggle with broader chemical and mechanistic diversity. Our transcriptomic approach, by contrast, offers systems-level resolution that generalizes across DILI pathways—not only detecting known cytotoxic responses but also uncovering less immediate, non-lethal mechanisms often missed by traditional assays (Supplementary Data S4). As a result of its unbiased modeling, our approach shows improved detection of idiosyncratic compounds—a class of toxicities that often escape detection in targeted or phenotypically narrow assays. These compounds, many of which are associated with extremely rare clinical incidence (<12 case reports), present a significant challenge for preclinical screening. Our model identified 29 out of 65 of those cases (44%) the highest detection rate among all evaluated models, while maintaining a specificity of 88% (Supplementary Fig. Next, we analyzed how combining toxicogenomics with orthogonal assays further enhances detection. The three most effective combinations include pairing our model with Walker et al.'s 3D-based HCI assay34 to improve DILI detection from 23/27 to 26/27 cases, pairing with Persson et al.'s 2D-based HCI assay19 to improve DILI detection from 28/37 to 30/37 cases, and pairing with Sakatis et al. GSH depletion assay29 to increase detection from 47/65 to 53/65. Based on these insights, we propose a tiered de-risking funnel strategy that begins with straightforward endpoints, such as PK data (e.g., Cmax <25 μM) and cytotoxicity assays, to flag overt hepatotoxicity. For candidates showing no early toxicity signals, toxicogenomics provides the most comprehensive and unbiased assessment of DILI risk—capturing both known and novel mechanisms. For a select few advanced candidates, with sufficient resources, applying toxicogenomics in advanced 3D liver models may offer the most accurate prediction of in vivo responses. This strategy ensures a resource-efficient and mechanistically broad DILI risk assessment in drug development. Our toxicogenomics approach offers a comprehensive and unbiased perspective on cellular responses, providing rich information for a nuanced understanding of liver toxicity, including idiosyncratic reactions with unknown mechanisms. By shifting from single-target analyses to a systems-level viewpoint, we demonstrate that applying machine learning to toxicogenomics holds great promise as a significant enhancement over existing toxicology methods. It enables dose-specific predictions and safety margins, moving beyond binary DILI/No-DILI assessments. Its extensive mechanistic coverage presents a substantial advantage over current pre-clinical models. By creating a comprehensive toxicogenomics library specifically designed for DILI research, we achieved notable improvements in predicting DILI risks compared to state-of-the-art methods. The high sensitivity (88%) and specificity (100%) obtained in blind validation, along with the identification of DILI in withdrawn drugs and clinical failures overlooked in animal and clinical studies, underscore its practical utility in early drug development phases. We consider this an important step forward in predictive toxicology, positioning our approach as a significant advancement in the field. The scalability and adaptability of our method, a central component of a larger AI/ML platform, are designed to enhance the predictive power and efficiency of drug development pipelines. While our approach represents a meaningful step forward, it is important to acknowledge the inherent limitations of our method. First, the multifactorial nature of DILI, involving genetic, environmental, and lifestyle factors, means even the most advanced models cannot capture the full spectrum of potential mechanisms. Our approach, though comprehensive, does not account for all interindividual variability or rare genetic predispositions contributing to DILI risk. Gaps in the database, particularly in relation to poorly documented idiosyncratic reactions, can limit the model's accuracy. Third, our reliance on a 2D hepatocyte culture system may fail to replicate the complex interactions between hepatocytes and other cell types or tissues that can drive certain DILI mechanisms. Fourth, the exposure timepoint of 24 hours limit the detection of delayed or immune-mediated toxicities. Fifth, as with any preclinical model, the ultimate test of its utility lies in its ability to predict clinical outcomes, a domain where uncertainties and unpredicted variables can significantly impact performance. Looking ahead, toxicogenomics holds tremendous promise for advancing our understanding and prediction of DILI. Integrating RNA-seq with advanced 3D culture systems—such as spheroids and liver-on-chip platforms—may enable longer drug exposures and the inclusion of non-parenchymal cells like Kupffer cells and hepatic stellate cells. These co-culture models are essential for capturing immune activation, inflammation, and fibrosis—hallmarks of idiosyncratic and chronic DILI that hepatocyte monocultures cannot recapitulate. As long 3D models rely on high-content imaging or cytotoxicity endpoints that capture specific phenotypic responses they remain limited in mechanistic scope and scalability. Their performance is often demonstrated on small, curated compound sets with known mechanisms, which may not translate to broader chemical space. In contrast, transcriptomics provides a scalable, unbiased readout capable of detecting diverse DILI mechanisms, including those not captured by existing assays. As RNA-seq becomes more feasible in physiologically relevant 3D systems, we anticipate a powerful synergy—combining the mechanistic breadth of transcriptomics with the physiological relevance of 3D models. Our work lays the foundation for such integration, demonstrating that transcriptomics alone can robustly capture DILI risk across a wide range of mechanisms. Combining RNA-seq data with structural information of molecules could enable a deeper understanding of the interactions between drugs and cellular components, facilitating more accurate predictions of toxicity. Using early estimates as surrogate for plasma Cmax, such as target activity, could help derive safety margins earlier in the drug discovery process. The development of multivariate models that include DILI regulatory networks represents another exciting frontier. Such models can incorporate the complex interplay of genes, proteins, and metabolic pathways involved in DILI. Furthermore, exploring additional data types, such as chromatin accessibility, proteomics, or metabolomics, could yield further insights into DILI mechanisms. As these advanced models and datasets become integrated into the early stages of drug development, we anticipate a decrease in liver-related adverse events, improved efficiency in drug development, significant cost savings, and, most importantly, enhanced patient safety. The ongoing evolution of toxicogenomics approaches, bolstered by advancements in computational machine learning methods and multi-omics technologies, marks an important step toward more predictive drug safety evaluation—one that has the potential to support the development of safer therapeutics and improved patient outcomes. We employed a systematic, high-throughput approach using cryopreserved primary human hepatocytes (PHHs) to study transcriptional changes underlying drug-induced liver injury (DILI). Cryopreserved PHHs were obtained from LifeNet Health (Virginia Beach, VA, USA) under standard provider agreements. LifeNet Health procured tissues under informed donor consent and Institutional Review Board (IRB) approval in accordance with U.S. regulations. Cellarity did not create any new cell lines for this publication. Cryopreserved PHHs were selected for high viability (>95%), long-term plateability (10–15 days), compatibility with 96-well formats, and Grade A quality. Donor 1917277-01, a 37-year-old Caucasian female (BMI 26), was used. Hepatocytes were cultured in collagen I-coated 96-well plates and maintained in a sandwich configuration (collagen base with a 100 μg/mL Matrigel overlay) to preserve hepatocyte function. Cells were matured for three days with daily media changes before compound treatment. After 24 h of treatment, cells were either assayed for viability (LDH/ATP) or lysed for RNA extraction and sequencing (Supplementary Fig. Human hepatocyte thawing media, human hepatocyte culture media, HHCM supplement, human hepatocyte plating media, HHPM supplement (LifeNet Health) and Pen/Strep (Gibco) were thawed and filtered before plating. PHHs were counted using a Luna-FL Cell Counter and Acridine Orange/Propidium Iodide Stain (Logo Bio). Cells were plated at a cell density of 0.5 million cells / mL in collagen I-coated 96-well plates (Gibco). Cells were left to attach in the incubator for 6 hours and then replaced with maintenance medium (LifeNet Health). The next day, cells were overlayed with a thin coat of Matrigel and left to incubate for an additional day with a daily media change to allow for full maturation of cells. On Day 3, cells were treated with compounds at multiple concentrations for 24 h and then were either taken down for either viability testing (LDH and ATP readouts) or lysed for RNA sequencing. The objective of this experiment was to screen 300 compounds to create a hepatotoxicity intervention library for building a predictive DILI model. PHHs were first screened for maximum tolerated dose (MTD) in six-point log curves (0.01 μM to 1 mM, Figure. MTD was defined as the highest concentration before observing >10% cell death in the LDH assay. RNA sequencing was conducted on cells treated with compound concentrations ranging from Cmax to MTD to evaluate the safety margin between therapeutic and toxic doses. All compounds were purchased from MedChemExpress (MCE). Compound dissolution was prepared manually fresh on the day of compound treatment to avoid freeze/thaw cycles. In a previous DMSO tolerance test, 0.5% of DMSO in compound was dictated as the ideal concentration. Compound dissolution with DMSO started at a highest concentration of 1000 µM. If compounds were not solubilized at the 200x stock, dissolution was attempted at 100x and 50x concentrations. The compounds were added to a barcoded plate and transferred to the Hamilton MicroLab Star liquid handler for titration (Fig. The plate layout for titration varied between toxicity screens and RNA sequencing runs (Fig. As primary viability assay, we employed the non-radioactive cytotoxicity assay from Promega, which utilizes lactate dehydrogenase (LDH) as a marker for cell death. For reagent preparation, we thawed LDH buffer at 4 °C overnight and used it to reconstitute the LDH substrate bottles by adding 12 mL, which were then stored at –20 °C. We utilized designated untreated cells to establish the 100% lysate positive control, against which we normalized all subsequently treated cells for % viability calculations. To these designated cell wells, we added 10x Lysis buffer in a 1:10 volume ratio. We then mixed 50 μL of the collected sample with 50 μL of LDH substrate in a flat-bottom tissue culture plate. Plates were covered by and incubated at room temperature for 30 min. Following the incubation, we added 50 μL of thawed Stop solution and used a SpectraMax i3x plate reader with absorbance settings at 490 nm to read the plate. Another orthogonal viability assay that was utilized for toxicity screening was CellTiter-Glo (CTG) Luminescent Cell Viability Assay (Promega) which determined the ATP content within the wells. CTG aliquots were thawed the day of the takedown, and DPBS (Gibco) was added to CTG substrate at a 1:1 ratio. Following the lysis, plates were covered with aluminum foil and set on an orbital shaker for 2 min at 400 rpm. Following the 2 min, the plates remained covered and were left at room temperature for 10 minutes. RLT buffer (Qiagen) and 2-mercaptoethanol (Thermofisher Scientific) were prepared to create a RLT + 1% BME reagent. 140 uL of lysis buffer was added across all wells of the plate using a multichannel pipette. After ensuring the cells were lysed under a microscope, the 140 uL of lysate was transferred to an Eppendorf twin.tec 96-well PCR plate (Fisher Scientific) and placed into a −80 °C freezer. Total RNA was prepared from cell lysates in 96-well plates using a QIAcube HT robotic workstation (Qiagen) in conjunction with RNeasy 96 QIAcube HT kits (Qiagen) according to the manufacturer's recommended protocol. SMART-Seq DE3 libraries were prepared from total RNA according to the manufacturer's protocol. Briefly, for each row of the plate, polyadenylated mRNA was selected using uniquely barcoded oligo(dT) primers. First strand cDNA was generated via reverse transcription; double-stranded cDNA was created via template switching with limited cycles of PCR amplification. Each row of samples was then pooled and subjected to transposon-based fragmentation using the Nextera XT DNA Library Preparation Kit (Illumina). Libraries were then PCR amplified using unique combinations of Illumina P5 and P7 barcodes and mixed in equimolar pools prior to sequencing. Sequencing was performed on an Illumina NovaSeq6000 using custom read lengths of 89 bp (Read1) and 26 bp (Read2). Cmax values were compiled from multiple resources, and the median of these values was used to derive a consensus total Cmax. For compounds where no studies provided reliable Cmax estimates or where estimates varied significantly across studies, additional manual annotations were performed by searching PubChem and relevant clinical studies. DILIrank and LiverTox serve as key resources for categorizing drugs as DILI positive or negative. DILIrank, developed by the FDA, classifies over 1000 drugs into four levels of DILI concern — most, less, no concern, and undetermined — based on historical liver injury data. LiverTox, created by the NIDDK, is an exhaustive online database with detailed information on these drugs, including their clinical manifestations, mechanisms of action and likelihood of causing DILI. It also offers a system to assess the likelihood of DILI, from well-known cases to idiosyncratic drugs without a characteristic signature to drugs considered safe. These categorizations serve as DILI endpoints for the development and refinement of our model. Compounds were systematically categorized based on DILIrank and LiverTox into the following categories: Known DILI: Compounds with well-established DILI risk (Most-DILI-Concern in DILIrank or LiverTox score A; e.g., Isoniazid at therapeutic doses, Acetaminophen at overdose levels). Likely DILI: Compounds with documented cases of DILI in specific contexts (Most-DILI-Concern or LiverTox score A/B; e.g., Progesterone). No DILI: Compounds with no clinical or preclinical documented evidence of DILI (No-DILI-Concern in DILIrank; LiverTox score E). Our models were trained to differentiate between positive and negative control compounds. The categories Unlikely DILI and Idiosyncratic DILI, due to their label ambiguity, were excluded from model training and reserved for downstream testing. In the training data, DILI positives include the highest concentrations and those above 20x Cmax, while DILI negatives include the lowest concentrations and those below 80x Cmax. This approach ensures unambiguous labeling while minimizing bias from overdose signatures in negatives and inactive signatures in positives. Using these categorizations, our model was trained to distinguish between 177 positive and 93 negative control data points. An additional set of 70 compounds known for their idiosyncratic effects, each linked to fewer than 12 case reports, were exclusively used for testing. Idiosyncratic hepatotoxicity, characterized by its unpredictability and lack of dose-response or temporal patterns, often leads to drug withdrawals despite thorough clinical testing. An independent in-house experiment was conducted for blind validation, comprising 46 compounds: 32 DILI positives and 14 negatives. This included well-established DILI positives/negatives and a real-world set with 4 recent clinical failures. Human toxicogenomics benefits from resources such as TG-GATES, CMap and L1000. CMap and L1000 catalogue genomic responses in in vitro cell lines, primarily cancer lines, and can be used to study cytotoxicity, however, their utility for DILI is limited as our initial efforts showed poor predictive performance likely due to the lack of physiologically relevant cell types and relevant range of concentrations. In contrast, TG-GATES provides a rich microarray database tailored for DILI research, including data from primary human hepatocytes (PHH) exposed to 158 compounds. Our initial proof-of-concept, utilizing TG-GATES data, yielded meaningful predictions. In a rigorously designed in-house pilot experiment involving 46 compounds for blind validation, our model demonstrated a 62% sensitivity in accurately identifying DILI positives and an 92% specificity for negative control compounds, demonstrating the potential of toxicogenomics in identifying DILI risks during drug development. We have created DILImap, our RNA-seq library encompassing 300 compounds, to improve the predictive power and mechanistic coverage of our ML model. We applied stringent quality control metrics and filtering criteria to retain only high-quality RNA-seq samples: In addition to these technical filters, we performed a hepatocyte fidelity check: each sample was assessed for expression of liver-specific marker genes and the preservation of a hepatocyte-like transcriptional profile. Samples passing all QC filters were used to compute compound-specific differential expression signatures. We used DESeq2, a standard RNA-seq analysis tool, which models read counts using a negative binomial distribution. DESeq2 adjusts for library size differences and estimates gene-specific dispersion, enabling accurate detection of differentially expressed genes (DEGs). For each compound-dose combination, we compared treated samples against matched DMSO controls from the same plate. This plate-specific normalization controls for potential batch effects and ensures that the derived signatures reflect treatment-specific transcriptional responses. Pathway enrichment analysis is a statistical approach used to identify whether specific biological pathways are significantly enriched with differentially expressed genes (DEGs) in RNA-seq data. In our analysis, we utilized the widely adopted hypergeometric test to compute p-values. To derive pathway signatures from p-values, we applied a -log10 transformation of the adjusted p-value, or False Discovery Rate (FDR). The resulting -log10(FDR) scores serve as the input to our model, providing a robust representation of pathway enrichment. Each model was evaluated using a 5-fold stratified compound-level cross-validation strategy, ensuring that all samples from a single compound were held out together in each fold to prevent data leakage. This compound-level splitting reflects a realistic generalization scenario for novel compound prediction. We optimized hyperparameters using grid search, assessing performance across the following metrics: Area Under the ROC Curve (AUC): Measures the overall ability to distinguish DILI from No-DILI compounds. Precision: Fraction of predicted DILI compounds that are truly DILI-positive, relevant for minimizing false positives. Recall: Fraction of true DILI compounds that are correctly identified, reflecting the model's ability to avoid false negatives. Inter-fold correlation: Quantifies agreement in predicted probabilities between models trained on different splits. Monotonic dose response: Assesses whether increasing doses of a compound correspond to increasing predicted DILI risk. Generalizability: prioritize high validation AUC with low train–validation gap to avoid overfitting → Best models had AUC ≥ 0.74 (RF, HistGB, LGBM, XGB); RF showed the lowest gap (~0.09 vs. 0.16–0.20 for GBMs). Stability: require high inter-fold correlation for consistent predictions across folds → RF achieved 0.98–1.0, higher than GBMs (0.8–0.9). Clinical utility: balance precision (reduce false positives) and recall (reduce false negatives) → RF performance was comparable to other top models. Interpretability: prefer simpler, easier-to-interpret models at similar performance → RF is more interpretable than boosting methods. Final choice: Random Forest was selected as the most balanced and robust model, combining strong validation AUC, minimal overfitting, high stability, and interpretability. Although XGBoost and LightGBM reached similar AUCs, they showed greater overfitting (train-validation AUC gap) and lower stability (inter-fold correlation). These characteristics along with its ability to handle non-linear relationships, its robustness to noise, and its suitability for datasets of moderate size, make Random Forest the most robust choice for generalization to unseen compounds. This combination of performance and robustness established Random Forest as the base model for ToxPredictor. To ensure predictive robustness, we employed a 30-model Random Forest ensemble using bootstrap aggregation (bagging). Each model was trained on a unique bootstrap sample from the training data, ensuring diversity in training instances and mitigating overfitting. We selected 30 folds to strike a balance between accuracy and ensemble stability. In our benchmarks, ensembles with 30 independent learners consistently achieved smooth, reproducible predictions and stable dose-responses. All fold models used the hyperparameters optimized during cross-validation. At prediction time on unseen compounds, model outputs are averaged with equal weighting, producing a consensus decision that mitigated individual model variance, ensuring robust performance by combining the strengths of each base learner, leading to stable and reliable classification of DILI vs. No-DILI. In blind validation, where predictions were made on compounds not seen during training or model selection, the ensemble demonstrated high AUC, dose consistency, and biological plausibility, confirming its utility for real-world DILI risk assessment. This probability reflects the model's confidence that a compound induces liver injury, based on its pathway perturbation profile. To enable binary classification for downstream evaluation and decision-making, we defined a DILI risk threshold of 0.7. This threshold was empirically selected to maximize the trade-off between sensitivity and specificity in our benchmark datasets, ensuring robust identification of true DILI compounds while minimizing false positives. ToxPredictor computes a Margin of Safety (MOS) for each compound, defined as the ratio between the first DILI dose and its maximum plasma concentration (Cmax) at therapeutic levels. We use total Cmax values rather than free (unbound) Cmax, as both approaches yield comparable predictive performance, but total Cmax offers broader availability across clinical data sources and aligns with common reporting in literature. For validation and benchmarking, to classify a compound as DILI, we selected a Margin of Safety (MOS) threshold of 80, determined empirically from the training set. This value reflects an optimized trade-off between sensitivity and specificity—minimizing false positives while reaching a performance plateau that captures the majority of true DILI liabilities. Importantly, this threshold is consistent with values commonly used in the literature, where MOS cutoffs typically range from 10 to 100 depending on context, placing our choice within accepted toxicological standards. To identify and prioritize features and pathways relevant to DILI risk prediction, feature importance within the Random Forest ensemble was assessed using statistical significance, impurity reduction, permutation importance, and direct discriminative power, ensuring a comprehensive and biologically meaningful selection of key predictors of DILI risk: Statistical Significance: Features were prioritized if they demonstrated at least one significant differential occurrence across compounds, with a p-value threshold of 0.001. Mean Decrease in Impurity (MDI): A standard metric for tree-based models that quantifies the reduction in impurity (e.g., Gini impurity) achieved by splits on specific features. The MDI values were averaged across all trees in the ensemble, providing a robust measure of a feature's overall predictive contribution. Features with higher MDI scores were deemed more important for the model's decision-making. A greater reduction in AUC indicated a higher importance of the feature, offering additional insight into the features that most strongly influence the model's predictions. Discriminative Power (AUC): Each feature's ability to distinguish between DILI and no-DILI categories was evaluated directly by calculating its individual AUC score, ensuring that features with strong discriminative capabilities were prioritized. Empirical cumulative DILI likelihoods are calculated to inform dose recommendations that minimize the risk of DILI. For each compound, safety margins are computed at varying hypothetical Cmax values as the ratio of the model-predicted first DILI dose to the corresponding Cmax. For each safety margin, two cumulative percentages are derived: (1) the percentage of DILI compounds with safety margins above the given value, which increases monotonically from 0 to 1 as Cmax increases, and (2) the percentage of non-DILI compounds with safety margins below the same value, which decreases monotonically from 1 to 0. The empirical cumulative DILI likelihood at each safety margin is calculated as the difference between these two percentages, effectively representing the relative enrichment of DILI compounds compared to non-DILI compounds at or above a given margin. This approach captures the overall relationship between safety margins and DILI likelihood across the dataset, rather than focusing on isolated points, thereby enabling robust differentiation between high-risk and low-risk compounds. The resulting cumulative risk profiles provide a quantitative framework to guide dose selection. We benchmarked ToxPredictor against three state-of-the-art structure-based models: DILIGeNN, a graph neural network trained on molecular graphs; DILIPredictor, a random forest ensemble model that integrates chemical structure, physicochemical properties, pharmacokinetic parameters and predicted proxy-DILI data; and TxGemma, a generalist large language model (LLM) fine-tuned on biomedical tasks from the Therapeutic Data Commons (TDC). The evaluation aimed to assess the predictive performance of each approach on overlapping and unseen compounds using balanced accuracy, sensitivity, and specificity. A Random Forest model trained on nine proxy-DILI labels (e.g., mitochondrial toxicity, BSEP inhibition) in conjunction with chemical structural features derived from SMILES strings of 1111 DILI compounds. We reproduced the model from its public repository, converting the Poetry environment to Conda (via poetry2conda) and using scikit-learn v1.2.0 to match the pretrained model version. As compound names were not provided, we standardized SMILES and generated InChIKeys (non-isotopic, non-stereochemical layer) to identify 483 unseen compounds; 6 failed at inference, 6 more were deduplicated, resulting in 471 (98 DILI + , 373 DILI − ). This approach may still introduce limited data leakage if SMILES differ from those used during training. For benchmarking against other in-silico chemistry models, we focused on 314 compounds identified as unseen by both DILIGeNN and DILIPredictor, to provide a more conservative evaluation set that helps reduce the risk of data leakage; particularly important for DILIPredictor, which lacked compound identifiers and used a final model re-trained on the test set. For benchmarking against ToxPredicor, we took the overlap of the 471 compounds with DILImap yielding 30 compounds (23 DILI + , 7 DILI − ). A graph neural network (GNN) trained on molecular graphs derived from SMILES strings of 1167 DILI compounds. We benchmarked the best-performing GraphSAGE models obtained after sequential warm starts, and used the recommended custom molecular graph representations of SMILES for each compound. To identify unseen compounds, we cross-referenced compound names, SMILES and InChIKeys (non-isotopic, non-stereochemical layer), and identified 349 unseen compounds; 18 failed inference, yielding a final evaluation set of 331 compounds (47 DILI + , 284 DILI-). Final predictions were based on the mean probability and majority vote across four GraphSAGE models. In addition, we benchmarked DILIGeNN against DILIPredictor on 314 shared unseen compounds (45 DILI + , 269 DILI-) to evaluate real-world performance of in silico models across a broader evaluation set. A suite of generalist large language models (LLMs), fine-tuned from Gemma-2 (2B, 9B, 27B parameters) on biomedical tasks from the Therapeutic Data Commons (TDC). For DILI classification, models were trained on 475 compounds (325 train, 54 validation, 96 test). To avoid data leakage, we used SMILES and InChIKeys to select 715 unseen compounds (178 DILI + , 537 DILI-); however, note that data leakage could not be fully ruled out. Of these, 97 compounds (69 DILI + , 35 DILI-) overlapped with DILImap and were used for benchmarking. TxGemma models were deployed on Google Cloud Vertex AI using the official cookbook (https://github.com/google-gemini/gemma-cookbook/tree/main/TxGemma) with recommended prompts. The 27B achieved the best performance and was used for comparison with ToxPredictor; AUROC was not computed due to the model only providing DILI label outputs rather than prediction probability outputs. Performance improvements may be possible with additional fine-tuning and prompt engineering. Dask enabled scalable computing and quilt3 managed dataset access. DESeq2 supported differential expression and gseapy pathway enrichment. XGBoost and LightGBM were used for model comparison to select the optimal base model for ToxPredictor. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All datasets and trained models are accessible via an S3 bucket for seamless integration with Jupyter notebooks, with detailed descriptions and access instructions provided at https://www.dilimap.org. Processed training data (pathway-level signatures used as model input) and both raw and processed validation data are provided to enable full reproducibility of all model training and validation steps. The raw training data (gene expression count matrices) contain proprietary information and are not publicly available. Academic researchers may request access for internal, non-commercial use via DILImap@cellarity.com, with requests reviewed within 4–8 weeks. Approved data are available for 2 weeks and must be deleted within 6 months. Commercial access requires a data-sharing agreement. The data from the Open TG-GATEs database (http://dbarchive.biosciencedbc.jp/en/open-tggates/download.html)15 were used in an initial proof-of-concept to establish a toxicogenomics baseline performance, whereas all results reported in this manuscript are based on the internally generated data described above. The use of all datasets in this study complies with the terms and conditions of their respective repositories and data providers. All code, reproducibility notebooks, and results are available at https://www.dilimap.org, which serves as the central access point for this work. The full Python implementation is hosted at https://www.github.com/Cellarity/DILImap, with reproducibility notebooks and results at https://www.github.com/Cellarity/DILImap_reproducibility. Both repositories are also archived at https://zenodo.org/records/1729052061. Olson, H. et al. Concordance of the toxicity of pharmaceuticals in humans and in animals. Current limitations and future opportunities for prediction of DILI from in vitro. & Lucena, M. I. Biomarkers in DILI: One more step forward. Chalasani, N. & Björnsson, E. Risk factors for idiosyncratic drug-induced liver injury. Mosedale, M. & Watkins, P. B. Drug-induced liver injury: Advances in mechanistic understanding that will inform risk management. Allison, R. et al. Drug induced liver injury - a 2023 update. Managing the challenge of drug-induced liver injury: a roadmap for the development and deployment of preclinical predictive models. Chen, M. et al. Quantitative structure-activity relationship models for predicting drug-induced liver injury based on FDA-approved drug labeling annotation and using a large collection of drugs. Kim, E. & Nam, H. Prediction models for drug-induced hepatotoxicity by using weighted molecular fingerprints. & Xia, M. Liver three-dimensional cellular models for high-throughput chemical testing. Chen, M. et al. DILIrank: the largest reference drug list ranked by the risk for developing drug-induced liver injury in humans. Serrano, J. LiverTox: An online information resource and a site for case report submission on drug-induced liver injury. Igarashi, Y. et al. Open TG-GATEs: a large-scale toxicogenomics database. O'Brien, P. J. et al. High concordance of drug-induced human hepatotoxicity with in vitro cytotoxicity measured in a novel cell-based model using high content screening. Xu, J. J. et al. Cellular imaging predictions of clinical drug-induced liver injury. Tolosa, L. et al. Development of a multiparametric cell-based protocol to screen and classify the hepatotoxicity potential of drugs. A high content screening assay to predict human drug-induced liver injury during drug discovery. Proctor, W. R. et al. Utility of spherical human liver microtissues for prediction of clinical drug-induced liver injury. Khetani, S. R. et al. Use of micropatterned cocultures to detect compounds that cause drug-induced liver injury in humans. Prediction of liver injury induced by chemicals in human with a multiparametric assay on isolated mouse liver mitochondria. Gustafsson, F., Foster, A. J., Sarda, S., Bridgland-Taylor, M. H. & Kenna, J. G. A correlation between the in vitro drug toxicity of drugs to cell lines that express human P450s and their propensity to cause liver injury in humans. Vorrink, S. U., Zhou, Y., Ingelman-Sundberg, M. & Lauschke, V. M. Prediction of drug-induced hepatotoxicity using long-term stable primary hepatic 3D spheroid cultures in chemically defined conditions. Prediction of human drug-induced liver injury (DILI) in relation to oral doses and blood concentrations. Aleo, M. D. et al. Moving beyond binary predictions of human drug-induced liver injury (DILI) toward contrasting relative risk potential. Sakatis, M. Z. et al. Preclinical strategy to reduce clinical hepatotoxicity using in vitro bioactivation data for >200 compounds. & Kenna, J. G. In vitro inhibition of the bile salt export pump correlates with risk of cholestatic drug-induced liver injury in humans. Wang, E. et al. TxGemma: Efficient and Agentic LLMs for Therapeutics. Improving drug-induced liver injury prediction using graph neural networks with augmented graph features from molecular optimisation. Improved detection of drug-induced liver injury by integrating predicted in vivo and in vitro data. Walker, P. A., Ryder, S., Lavado, A., Dilworth, C. & Riley, R. J. The evolution of strategies to minimise the risk of human drug-induced liver injury (DILI) in drug discovery and development. Performance assessment and economic analysis of a human Liver-Chip for predictive toxicology. Fäs, L. et al. Physiological liver microtissue 384-well microplate system for preclinical hepatotoxicity assessment of therapeutic small molecule drugs. Comparison of primary human hepatocytes and hepatoma cell line Hepg2 with regard to their biotransformation properties. Olsavsky, K. M. et al. Gene expression profiling and differentiation assessment in primary human hepatocyte cultures, established hepatoma cell lines, and human liver tissues. Grinberg, M. et al. Toxicogenomics directory of chemically exposed human hepatocytes. Kiamehr, M. et al. Dedifferentiation of primary hepatocytes is accompanied with reorganization of lipid metabolism indicated by altered molecular lipid and miRNA profiles. The ChEMBL Database in 2023: a drug discovery platform spanning multiple bioactivity data types and time periods. Setting clinical exposure levels of concern for drug-induced liver injury (DILI) using mechanistic in vitro assays. Williams, D. P., Lazic, S. E., Foster, A. J., Semenova, E. & Morgan, P. Predicting drug-induced liver injury with Bayesian machine learning. Kelder, T. et al. WikiPathways: building research communities on biological pathways. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Comparison of RNA-Seq and microarray gene expression platforms for the toxicogenomic evaluation of liver from short-term rat toxicity studies. Allard, J. et al. Drug-induced hepatic steatosis in absence of severe mitochondrial dysfunction in HepaRG cells: proof of multiple mechanism-based toxicity. in LiverTox: Clinical and Research Information on Drug-Induced Liver Injury (2012). & Sledzinski, T. The role of fatty acids in non-alcoholic fatty liver disease progression: An update. Osborne, T. F. & Espenshade, P. J. Lipid balance must be just right to prevent development of severe liver damage. Gut microbiota-derived tryptophan metabolites alleviate liver injury via AhR/Nrf2 activation in pyrrolizidine alkaloids-induced sinusoidal obstruction syndrome. The emerging role of ferroptosis in various chronic liver diseases: Opportunity or challenge. Chen, S.-S. et al. Serum metabolomic analysis of chronic drug-induced liver injury with or without cirrhosis. Wagner, M., Zollner, G. & Trauner, M. Nuclear receptors in liver disease. & Deminice, R. One-carbon metabolism in fatty liver disease and fibrosis: One-carbon to rule them all. & Boyer, J. L. The role of bile acids in cholestatic liver injury. Ray, W. A., Griffin, M. R. & Stein, C. M. Cardiovascular toxicity of valdecoxib. Shrier, M., Díaz, J. E. & Tsarouhas, N. Cardiotoxicity associated with bupropion overdose. Kohonen, P. et al. A transcriptomics data-driven gene space accurately predicts liver cytopathology and drug-induced liver injury. We sincerely thank Bill Pennie and Atli Thorarensen for bringing in pivotal ideas that shaped the work. Our gratitude extends to Robb Nicewonger for his work on the chemistry SOP, Cameron Reilly for his support with RNA extraction and automation, Laura Isacco for coordinating data generation timelines, Thao Tran and Wynter Guess for their assistance with compound screening, Winnie Lee and Brian Yi for their work on RNA extraction, all of which contributed to data generation foundational to this work. Finally, we thank the reviewers for their thoughtful and constructive requests, which helped refine and strengthen the work. Volker Bergen, Konstantia Kodella, Sreenath Srikrishnan, Ornella Barrandon, Sara Anderson, Max Rogers-Grazado, Casey Fowler, Hirit Beyene, Nicole Robichaud, Timothy Fulton, Nina Lapchyk, Mauricio Cortes, Nick Plugis, Matthew Goddeeris & Mahdi Zamanighomi Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar S.S. implemented and ran benchmarking against state-of-the-art in silico models. contributed to the study's conceptual framework. established the initial experiments and contributed to RNA-seq method selection. implemented automation for data generation workflows. led compound management, screening and contributed to the chemistry SOP. contributed to compound management and screening. was involved in the original conception of using transcriptomics to model off-target effects. provided oversight for the machine learning aspects and strategic direction. Correspondence to Volker Bergen or Mahdi Zamanighomi. The research was funded by Cellarity. are inventors on patent application WO2025/024525, related to methods for DILI prediction, assigned to Cellarity. Nature Communications thanks Volker Lauschke, James Dear and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Bergen, V., Kodella, K., Srikrishnan, S. et al. A large-scale human toxicogenomics resource for drug-induced liver injury prediction. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02895-w'>: differentiable simulation enables large-scale training of detailed biophysical models of neural dynamics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 10:26:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Biophysical neuron models provide insights into cellular mechanisms underlying neural computations. A central challenge has been to identify parameters of detailed biophysical models such that they match physiological measurements or perform computational tasks. Here we describe a framework for simulating biophysical models in neuroscience—Jaxley—which addresses this challenge. By making use of automatic differentiation and GPU acceleration, Jaxley enables optimizing large-scale biophysical models with gradient descent. Jaxley can learn biophysical neuron models to match voltage or two-photon calcium recordings, sometimes orders of magnitude more efficiently than previous methods. Jaxley also makes it possible to train biophysical neuron models to perform computational tasks. We train a recurrent neural network to perform working memory tasks, and a network of morphologically detailed neurons with 100,000 parameters to solve a computer vision task. Jaxley improves the ability to build large-scale data- or task-constrained biophysical models, creating opportunities for investigating the mechanisms underlying neural computations across multiple scales. The latter provide fine-grained mechanistic explanations of cellular processes underlying neural activity, typically described as systems of ordinary differential equations1,2,3. However, it has been highly challenging for neuroscientists to create biophysical models that can explain physiological measurements3,4,5 or that can perform computational tasks6,7. It is hardly ever possible to directly measure all relevant properties of the system with sufficient precision to constrain all parameters directly, necessitating the use of inference or fitting approaches to optimize free model parameters. However, finding the right parameters for even a single-neuron model with only a few parameters can be difficult5,8, and large-scale morphologically detailed biophysical network models may have thousands of free parameters governing the behavior of ion channels (for example, maximal conductance), synapses (for example, synaptic conductance or time constant) or neural morphologies (for example, radius or branch length). Recently, in many domains of science such as particle physics, geoscience and quantum chemistry, differentiable, GPU-accelerated simulators have enabled parameter inference for even complicated models using automatic differentiation techniques9,10,11. Such differentiable simulators make it possible to train simulators with gradient descent methods from deep learning: Backpropagation of error (‘backprop') makes the computational cost of computing the gradient of the model with respect to the parameters independent of the number of parameters, making it possible to efficiently fit large models. In addition, GPU acceleration allows computing the gradient for many inputs (or model configurations) in parallel, which allows fitting simulations to large datasets. Numerical solvers for biophysical models in neuroscience are used extensively, and several software packages exist, in particular the commonly used Neuron simulation environment12. Yet, none of these simulators allows performing backprop, and currently used simulation engines are primarily CPU-based, with GPU functionality only added post hoc13,14,15. As a consequence, state-of-the-art methods for parameter estimation in biophysical neuron models are based on gradient-free approaches such as genetic algorithms8,16 or simulation-based inference17, which do not scale to models with many parameters. Inspired by the capabilities of deep learning to adjust millions (or even billions) of parameters given large datasets, we here propose to optimize biophysical parameters with gradient descent. To this end, we developed Jaxley, a toolbox for biophysical simulation which, unlike previous simulation toolboxes for biophysical models, can compute the gradient with backprop. In addition, Jaxley leverages GPU acceleration to speed up training. We apply Jaxley to a series of tasks, ranging from fitting physiological data (that is, match experimental recordings such as voltage or calcium measurements8,18) to solving computational tasks7,19 (Fig. We show that gradient descent can be orders of magnitude more efficient than gradient-free methods, and that it enables training biophysical networks with 100,000 parameters. This unlocks possibilities for data-driven and large-scale biophysical simulations in neuroscience. a, Schematic of goal: training biophysically detailed neural systems. b, Schematic of method: our simulator, Jaxley, can simulate biophysically detailed neural systems, and it can also perform backprop. d, Reconstruction of a CA1 neuron24 and responses to a step current obtained with the Neuron simulator and with Jaxley. e, Left: time to run 10,000 simulations with Neuron on a CPU and with Jaxley on a GPU. Right: Simulation time (top) for the CA1 neuron shown in d and for a point neuron, as a function of the number of simulations. Bottom: same as top, for computing the gradient with backprop. f, Biophysically detailed network built from reconstructions of CA1 neurons (left) and its neural activity in response to step currents to the first layer (right). Runtimes were evaluated on an A100 GPU. M, million; ML, machine learning; Sim., simulation. Jaxley is a Python toolbox for simulation and training of biophysical neuron models. Jaxley implements numerical routines required for efficiently simulating biophysically detailed neural systems, so-called implicit Euler solvers, in the deep learning Python framework JAX20. The automatic differentiation capabilities of JAX enable Jaxley to use backprop to efficiently compute the gradient with respect to any biophysical parameter, including ion channel, synaptic or morphological parameters (Fig. For computational speed, Jaxley implements differential equations such that networks, parameter sets or input stimuli can be processed in parallel on GPUs, providing speedups for datasets (via stimulus parallelization) or for parameter sweeps (via parameter parallelization)13,15. Jaxley further speeds up simulation and training with just-in-time compilation (Fig. Training biophysical models with gradient descent leads to instabilities resulting from parameters having different scales, networks having a large computation graph21 and loss surfaces being non-convex. Jaxley implements methods that have been developed to overcome these specific issues in deep neural networks (Extended Data Fig. Furthermore, we designed Jaxley with a user-friendly interface, allowing neuroscientists to build biophysical models (for example, for inserting recordings, stimuli and channels into various branches or cells, or implementing different connectivity structures such as sparse or dense connectivity) and to use automatic differentiation and GPU parallelization. In a dedicated library open to the community, it also implements a growing set of ion channel and synapse models. Jaxley is openly available at https://github.com/jaxleyverse/jaxley/. We benchmarked the accuracy, speed and scalability of Jaxley for simulation of biophysical models. First, we evaluated the accuracy of Jaxley by creating biophysically detailed multicompartment models of a CA1 pyramidal cell in the rat hippocampus24,25 and of four layer 5 neurons in the mouse visual area from the Allen Cell Types Database26. Every model contained sodium, potassium and leak channels in all branches. Jaxley matched the voltages of the Neuron simulator at sub-millisecond and sub-millivolt resolution (Fig. Next, we evaluated the simulation speed of Jaxley on CPUs and GPUs. We simulated the above-described CA1 cell and a single-compartment model for 20 ms. On a GPU, Jaxley was much faster for large systems or many parallel simulations, with a speedup of around two orders of magnitude (Fig. For single-compartment neurons, Jaxley could parallelize the simulation of up to 1 million neurons, thereby allowing fast parameter sweeps. For backpropagation, the forward pass must be stored in-memory, which can easily correspond to terabytes of data for large neural systems. To overcome this, Jaxley implements multilevel checkpointing22, which reduces memory usage by strategically saving and recomputing intermediate states of the system of differential equations. Finally, we show that in addition to parallelizing across parameters (or across stimuli), Jaxley can parallelize across branches or compartments in a network. We built a network consisting of 2,000 morphologically detailed neurons with Hodgkin–Huxley dynamics, connected by 1 million biophysical synapses (3.92 million differential equation states in total; Fig. On a single A100 GPU, Jaxley computed 200 ms (that is, 8,000 steps at Δt = 0.025 ms) of simulated time in 21 s. We then used backprop to compute the gradient with respect to all membrane and synaptic conductances in this network (3.2 million parameters in total), which took 144 s. Estimating the gradient with finite differences—as would be required for packages that do not support backprop—would take more than 2 years (3.2 million forward passes, 21 s each). With simplified morphologies, Jaxley can scale backprop to networks with many millions of synapses on a single GPU (Extended Data Fig. Having demonstrated the accuracy and speed of Jaxley, we applied it to a series of tasks that demonstrate how Jaxley opens up opportunities for building task-driven or data-driven biophysically detailed neuroscience models in a range of scenarios. As a first proof-of-principle, we applied Jaxley to fit single-neuron models with few parameters. We learned these parameters from a synthetic somatic voltage recording given a somatic step-current stimulus with a known set of ground-truth parameters (Fig. b, Top: synthetic somatic voltage recording (black) and windows that are used to compute summary statistics (top). Bottom: fits obtained with gradient descent. c, Loss value of individual gradient descent runs (light blue), their minimum (dark blue), in comparison to the minimum loss across ten genetic algorithm runs (black). d, Neuron morphologies and patch-clamp recordings (black) in response to step currents from the Allen Cell Types Database. Additional models in Extended Data Fig. e, (Task 2) Synthetic conductance profile of an L5PC morphology. f, Simulated voltages given the synthetic conductance profile after 1.5 ms and 2.5 ms. g, Predicted voltages of gradient descent fit. h, Ground-truth (gt) conductance profile as a function of distance from the soma (black) and 90% confidence interval obtained with multi-chain gradient-based Hamiltonian Monte Carlo. i, Loss for gradient descent and genetic algorithm. j, (Task 3) Nonlinearly separable input stimulus amplitudes (left), and a simplified morphology with 12 compartments (right). k, Voltage traces of model found with gradient descent. l, Decision surface of the model reveals nonlinear single-neuron computation. m, Minimum loss across ten independent runs for gradient descent and genetic algorithm. We used gradient descent to identify parameter sets that minimize the mean absolute error to summary statistics of the voltage trace. Because gradient descent requires differentiable summary statistics, but commonly used summary statistics of intracellular recordings—such as spike count—can be discrete or non-differentiable, we used the mean and standard deviation of the voltage in two time windows17. Starting from randomly initialized parameters, gradient descent required only nine steps (median across ten runs) to find models whose voltage traces are visually similar to the observation (Fig. A state-of-the-art indicator-based genetic algorithm (IBEA)16 required similarly many iterations, although each iteration of the genetic algorithm used ten simulations. As a consequence, gradient descent required almost ten times fewer simulations than the genetic algorithm, and, despite the additional cost of backpropagation, found good parameter sets in less runtime than the genetic algorithm on a CPU (Fig. We then used gradient descent to identify parameters that match patch-clamp recordings of four cells from the Allen Cell Types Database26. We inserted the same set of ion channels, but, to account for the diversity and complexity of the experimental recordings, we made six additional parameters trainable and used a loss function based on dynamic time warping (DTW). We first used gradient descent with a low computational budget (ten runs with ten iterations each; loss in Extended Data Fig. 4b) and found that the resulting traces roughly matched the firing rate of experimental recordings, but did not yet match other features such as spike frequency adaptation (Extended Data Fig. To improve the fits, we used the ability of Jaxley to parallelize several fitting runs. We parallelized 1,000 gradient descent runs on a GPU and found parameter sets whose voltage traces closely resembled experimental recordings (Fig. Using Jaxley, we also parallelized a genetic algorithm on a GPU and found that the resulting fits were of similar quality (Extended Data Fig. Overall, these results demonstrate the ability of gradient descent to fit biophysical models to intracellular recordings, being competitive with state-of-the-art genetic algorithms even on tasks for which those have been extensively optimized. How does gradient descent scale to models with large numbers of parameters? We demonstrate here that, in contrast to genetic algorithms, gradient descent can optimize a single-neuron model with 1,390 parameters. We used the above-described model of an L5PC. Unlike in the above experiments, we fit the maximal conductance of ion channels in every branch in the morphology, thereby allowing us to model effects of nonuniform conductance profiles27. This increased the number of free parameters to 1,390. To generate a synthetic recording, we assigned a different maximal conductance to each branch (sampled from a Gaussian process), depending on the distance from the soma (Fig. We recorded the voltage at every branch of the model in response to a 5-ms step-current input (Fig. Experimentally, such data could be obtained, for example, through voltage imaging. We used gradient descent to identify parameters that match this recording, with a regularizer that penalizes the difference between parameter values in neighboring branches27. Despite the large number of parameters, gradient descent found a parameter set whose voltage response closely matched the observed voltage throughout the dendritic tree (Fig. To understand how much the whole-cell voltage recording constrains the parameters, we used Bayesian inference (implemented with gradient-based Hamiltonian Monte Carlo) to infer an ensemble of parameter sets all of which match the observed voltage (Extended Data Fig. 2h, around 400 μm), but it also revealed conductance profiles that were only weakly constrained by the data (Fig. Finally, we compared our method with an indicator-based genetic algorithm8 and, as expected, we found that access to gradients leads to better convergence: While gradient descent converged to values of low loss within 100 iterations, the genetic algorithm had two orders of magnitude higher loss even after 500 iterations (Fig. Next, we trained a single-neuron model to solve a nonlinear pattern separation task on its dendritic tree. While it has been demonstrated extensively that single-neuron models respond nonlinearly to inputs28, it has so far been difficult to train biophysically detailed neurons on a particular task. Here, we show that stochastic gradient descent enables training single-neuron models with dendritic nonlinearities to perform nonlinear computations. We defined a simple morphology consisting of a soma and two dendrites, and inserted sodium, potassium and leak channels into all neurites of the cell. We then learned ion channel densities as well as length, radius and axial resistivity of every compartment (72 parameters in total) for the neuron to have low somatic voltage (−70 mV), when both dendrites were stimulated with step currents of intermediate strength, and high somatic voltage (35 mV) when one of the dendrites was stimulated strongly and the other one weakly (Fig. Therefore, the two classes were not linearly separable, requiring the neuron to perform a nonlinear computation. After training the parameters with gradient descent, we found that the cell indeed learned to perform this task and spiked only when one dendrite was stimulated strongly (Fig. 2k), effectively having a nonlinear decision surface (Fig. We again compared gradient descent to an indicator-based genetic algorithm and found that gradient descent finds regions of lower loss more quickly than genetic algorithms (Fig. Overall, these results show that gradient descent performs better than gradient-free methods in models with many parameters, opening up possibilities for studying at scale biophysical mechanisms throughout the full neuronal morphology. So far, we have learned parameters of single-neuron models using small datasets consisting of few stimulus–response pairs. Using a network model of the mouse retina, we demonstrate that Jaxley can simultaneously infer cell-level and network-level parameters, such that model simulations match large-scale datasets. We consider transient Off alpha retinal ganglion cells (RGCs) in the mouse retina, which show compartmentalized calcium signals in their dendrites in response to visual stimulation18. To understand the mechanistic underpinning of this behavior, we built a hybrid model with statistical and mechanistic components: We modeled photoreceptors as a convolution with a Gaussian filter, bipolar cells as point neurons with a nonlinearity and an RGC as a morphologically detailed biophysical neuron, with six different ion channels distributed across its soma and dendritic tree (Fig. a, Schematic of experimental setup and hybrid model. b, Schematic of training procedure and loss function. c, Measured and model-predicted calcium response across 50 noise images (200 ms each). d, Left: calcium response (color map) of the trained hybrid model to a step current to a single branch indicated by step-current sketch. Right: intracellular calcium concentration in the same two recording sites. Scale bars, 50 ms, 30 mV and 0.025 mM. e, Receptive fields of the hybrid model. f, Pearson correlation coefficient between experimental data and model for train (top) and test (bottom) data, for a linear network, a multilayer perceptron, and the hybrid model. Asterisks denote a statistically significant difference between mean correlations of hybrid model and multilayer perceptron (MLP; one-sided t-test at P < 0.05, P = {0.0030, 0.0044, 0.017, 0.066, 0.80}). a.u., arbitrary units; BCs, bipolar cells; intra., intracellular; MAE, mean absolute error; PRs, photoreceptors. Using Jaxley, we trained the hybrid model to predict dendritic calcium on 15,000 pairs of checkerboard noise stimuli and calcium recordings. After training, we evaluated the trained model on a held-out test dataset. The model had a positive Pearson correlation coefficient with the experimental recording on 146 of 147 recording sites, with an average correlation of 0.25, and a maximum of 0.51 (Fig. Next, we tested whether the trained model also reproduced the compartmentalized structure of calcium responses, which has been experimentally measured18. We found that the calcium signal in response to local stimulation did not propagate through the entire cell (Fig. This demonstrated a compartmentalized response of the model and qualitatively matched the receptive fields obtained from experimental measurements18. Therefore, we investigated whether this inductive bias of the hybrid model could lead to better generalization to new data, especially when training data are scarce. We trained a linear model, a two-layer perceptron and the hybrid model on reduced datasets of recordings and compared their performance on a held-out test set (Fig. While the linear model and the perceptron performed better than the hybrid model on training data, the hybrid model performed better on held-out test data, when little training data were available. These results indicate that the inductive bias brought by the hybrid model effectively can limit the amount of overfitting in the model, suggesting that hybrid components could be used as regularizers for deep neural network models of neural systems31. Our results demonstrate that gradient descent enables fitting networks of biophysical neurons to large calcium datasets and allows simultaneous learning of cell-level and network-level parameters. In particular, recurrent neural networks (RNNs) have been used to form hypotheses about population dynamics underlying cognition. Typically, such RNNs consist of point neurons with rate-based or simplified spiking dynamics, which prevents studying the contribution of channel dynamics or cellular processes. We here show how Jaxley makes it possible to train biophysical models of neuronal networks to perform such tasks. We implemented in Jaxley an RNN consisting of Hodgkin–Huxley-type neurons with a simplified apical and basal dendrite, with each neuron equipped with a variety of voltage-gated ion channels4. We sparsely connected the recurrent network with conductance-based synapses and obtained the outputs from passive readout units (Fig. b, Autonomous dynamics of the recurrently connected neurons before learning the parameters for different values of synaptic gain. c, Maximal Lyapunov exponent indicates transition to chaos with increasing synaptic gain. Gaussian noise stimulus (top) and voltage traces of the two readout neurons. e, Histogram of initial and trained input, recurrent and output weights of the network. Two stimuli separated by a delay (for two of four input patterns), raster plots of the trained network activity, and readout neuron prediction. We first investigated the dynamics in this biophysical RNN before training. As with rate-based RNNs, these dynamics were strongly dependent on a global scaling factor (called ‘gain') of all recurrent synaptic maximal conductances32. Our RNN transitioned from a stable to a chaotic regime when the gain was increased, with an intermediate region, where networks displayed regular firing (Fig. The ability of Jaxley to perform automatic differentiation allowed us to quantify the stability of networks by numerically computing Lyapunov exponents33 (Fig. We then trained the biophysical RNN to perform two working memory tasks, starting with a perceptual decision-making task requiring evidence integration over time34. We built a network of 20 recurrent neurons and stimulated each recurrent neuron with a noisy time series with either positive or negative mean value (Fig. We trained input weights, recurrent weights and readout weights (109 parameters) such that the network learned to differentiate between positive and negative stimuli during a response period after 500 ms. Despite the long time horizon of this task (500 ms, corresponding to 20,000 time steps of the simulation), gradient descent found parameters such that the RNN was able to perform the task (Fig. To solve this evidence integration task, some input, recurrent and output weights were pushed toward zero and the remaining weights were pushed toward their positive and negative constraints during training (Fig. In addition, the trained RNN generalized to evidence integration tasks of longer durations (Extended Data Fig. We next used the RNN to solve a more challenging working memory task, a delayed-match-to-sample task, where the RNN had to maintain information over an extended period of time19,35. We trained the RNN to classify patterns, consisting of two step-current inputs with a delay between them, into matching (same identity of the inputs) or non-matching (different identity of the inputs; Fig. We used curriculum learning to solve this task. By training input, recurrent and readout weights, as well as synaptic time constants of a network with 50 recurrent neurons (542 parameters), we found parameter sets that solved the task and correctly classified all four patterns (Fig. Overall, these results demonstrate that gradient descent allows training RNNs with biophysical detail to solve working memory tasks. This will allow a more quantitative investigation of the role of cellular mechanisms contributing to behavioral and cognitive computations. Finally, we show that gradient descent enables training of large biophysical models with thousands of cellular-level and network-level parameters on machine learning-scale datasets to solve classical computer vision tasks such as image recognition. We implemented a feedforward biophysical network model in Jaxley and trained it to solve the classical MNIST task, without artificial nonlinearities such as ReLU activations. The network had three layers: The input and output layers consisted of neurons with ball-and-stick morphologies and the hidden layer consisted of 64 morphologically detailed models obtained from reconstructions of CA1 cells (Fig. The network was interconnected by biophysical synapses. We trained sodium, potassium and leak conductances of every branch in the circuit (55,000 parameters), as well as all synaptic weights (51,000 parameters). b, Voltages (left and middle) and softmax probabilities of the output neurons (right) measured at the somata of neurons in the trained network in response to an image labeled as ‘0'. c, Histograms of test-set accuracy of 50 linear networks (gray), 50 multilayer perceptrons with 64 hidden neurons and ReLU activations (black) and the biophysical network (blue). d, Histogram of parameters before (black) and after (green) training. gsyn1, gsyn2, synaptic conductances in the first and second layer, respectively. e, Test-set accuracy for trained network when subsets of parameters are reset to their initial value. Blue line indicates the fully trained network. f, Clean image (top, left) and adversarial image perturbation (top, right), as well as corresponding voltage traces of the output neurons (bottom). g, Accuracy across 128 test-set examples, as a function of the norm of the adversarial image perturbation. The network achieved an accuracy of 94.2% on a held-out test dataset, which is higher than a linear classifier, demonstrating that the biophysical network used its nonlinearities to improve classification performance. The biophysical network, however, performed slightly worse than a multilayer perceptron with ReLU nonlinearities, suggesting either that the spike/no-spike nonlinearities are more difficult to train than ReLU nonlinearities, or that the (binary) spike/no-spike representations lead to lower bandwidth than graded ReLU activations (Fig. After training, the biophysical network also developed interpretable hidden-layer tuning (Extended Data Fig. How do the learned parameters of the biophysical network contribute to its ability to classify MNIST digits? This does not mean that the learned values of these parameters do not contribute to the learned network dynamics: While the ranges of parameters did not change substantially, individual parameters could vary largely (Extended Data Fig. Furthermore, resetting subsets of parameters to their initial value reduced classification performance, sometimes to chance-level accuracy (10%; Fig. This indicates that biophysical simulations built purely from the aggregate statistics of measured parameter values could not be sufficient for the models to develop the ability to perform tasks4. Finally, efficient access to gradients not only allows training biophysical models to match physiological data or perform a task, but also opens up possibilities to perform new kinds of analyses, such as evaluating adversarial robustness. We stimulated the biophysical network with an image showing a ‘6', and then altered minimally the image with gradient descent such that the network would classify it as a ‘4' (Fig. We found that the two networks are similarly vulnerable to adversarial attacks (Fig. 5g), in contrast to previous studies suggesting that biophysical networks could largely improve adversarial robustness15. Overall, these results demonstrate that the ability of Jaxley to perform backprop in biophysical models can be used to train large-scale networks at full biophysical detail. In particular, backprop overcomes virtually any computational limit on the number of parameters of biophysical networks that can be included in the optimization. Backprop and computational frameworks that provide efficient, scalable and easy-to-use implementations have been key to advances in deep learning. They have made it possible to efficiently optimize even very big systems with gradient descent. However, for biophysical neuron models, currently used tools12,36,37,38 cannot perform automatic differentiation (comparison to other toolboxes in Supplementary Note 1). We here presented Jaxley, a computational framework for differentiable simulation of neuroscience models with biophysical detail. Jaxley can perform automatic differentiation through its differential equation solver, thereby enabling backprop to compute the gradient with respect to virtually any biophysical parameter. We expect that Jaxley will enable a range of new investigations in neuroscience: First, it will make it possible to efficiently optimize detailed single-cell models. This will allow insights into cellular properties across cell types and their relationship with, for example, transcriptomic measurements39,40,41, as well as into the contribution of dendritic processing to neural computations28. Second, it will facilitate creating large-scale biophysical network models. Such network models4 have so far primarily been built in a bottom-up fashion. Jaxley opens up possibilities to train thousands of parameters with gradient descent, allowing the tuning of large biophysical networks as a whole. Third, in addition to training such models, Jaxley will also enable numerous other applications, such as gradient-based Bayesian inference, the analysis of stability via Lyapunov exponents, adversarial attacks, the computation of maximally excitable stimuli or the design of optimally discriminative experiments. A central challenge in training biophysical models to perform computational tasks will be to bridge the timescales between biophysical mechanisms (milliseconds) and behavior (seconds). We showed that it is possible to fit data of up to 1 s in length, but optimizing models on tasks that require backpropagating gradients along even longer simulations is challenging. In addition, the loss landscape of biophysical models can be highly non-convex, which could trap optimization in local minima42. Gradient descent also requires the definition of a single loss function to be optimized, which can cause challenges when different tasks or features of experimental data are captured best by different parameter sets43. To investigate how gradient-based optimization could accommodate such scenarios, we have designed Jaxley such that it can be combined with any scheme for updating parameters. For example, for some tasks and training paradigms, forward mode automatic differentiation44 or evolutionary algorithms16,21 have been reported to perform similarly to (or sometimes even better than) backprop. Jaxley directly supports GPU-accelerated implementations of either of these algorithms, opening up possibilities to develop new methods for training of biophysical neural systems. By building upon the framework JAX20, Jaxley will further benefit from advances in the deep learning community at scaling and training large simulations. Finally, as the parameter space grows, there will typically be a wide range of solutions that match experimental recordings, and a single ‘best' fit might not be representative of the full solution space in the presence of parameter degeneracy2. To address such situations, Jaxley could be combined with methods that aim to identify the full space of data-compatible models, such as Bayesian inference17, or its parallelization capabilities could be exploited for parameter scans2. New experimental tools allow scientists to measure connectivity, morphology, genetic identity and activity of neural circuits at increasing levels of detail and scale. Jaxley is a powerful tool that can integrate measurements of connectivity and morphology into biophysical simulations, while allowing the resulting networks to be fitted to data or computational tasks—just like deep neural networks. This will enable investigations of the biophysical basis of neural computation at unprecedented scales. We used an implicit Euler solver for the voltage equations, and an exponential Euler solver for the gates (Supplementary Note 2). We simulated voltage at internal nodes and, similar to Neuron, also simulated terminal nodes at branch points. To solve the tridiagonal system of equations of every branch, we used the algorithm presented by Stone45. We used the following tools to improve training accuracy and robustness. First, we used parameter transformations that ensure that optimization of bounded parameters can be performed in unconstrained space and that biophysical parameters are on the same scale. In particular, we used an inverse sigmoid transformation \(T(\theta )=-\log (\frac{1}{(\theta -l)/(u-l)}-1)\), where l is the lower bound and u is the upper bound. Second, we used a variant of Polyak stochastic gradient descent23, which computes the step as \({\rm{step}}=\gamma \frac{\nabla {\mathcal{L}}}{| | \nabla {\mathcal{L}}| {| }^{\beta }}\). This optimizer overcomes large variations of the gradient between steps. This affects the Polyak gradient descent optimizer because it normalizes by the gradient norm. Fourth, we performed multilevel checkpointing to reduce the memory requirements of backpropagation through time. When performed, we interrupted gradient computation every 50 ms. All of these tools are implemented in Jaxley. We provide an overview of training schemes for all examples in Supplementary Note 3. Jaxley can be combined with any parameter optimization or learning method. The ability to evaluate the gradient with backprop opens possibilities for training biophysical systems with gradient descent or with gradient-based reinforcement learning methods, but Jaxley can also be efficiently combined with gradient-free parameter optimization methods (for example, genetic algorithms, Bayesian optimization, gradient-free reinforcement learning approaches or biologically plausible learning rules such as Hebbian learning). In these cases, Jaxley can parallelize forward simulations whose output can inform the gradient-free update rule. We used two types of sodium, five types of potassium, two types of calcium and a hyperpolarization-activated channel and inserted them in soma, basal and apical dendrites, and axon, following Van Geit et al.16. These were the same parameters as used by Van Geit et al.16, but our model did not contain a persistent sodium channel in the axon. We used the same parameter search bounds as Van Geit et al.16 for fitting the L5PC to synthetic data (Fig. 2b), but we enforced that somatic potassium existed (lower bound 0.25 S cm−2). For the experimental recordings from the Allen Cell Types Database, we used slightly larger bounds for three of the parameters to increase the flexibility of the model: We used bounds of [0, 6] S cm−2 (instead of [0, 4] S cm−2) for somatic and axonal sodium channels and a lower bound of 1 ms for the delay of the calcium buffer (instead of 20 ms). In addition, we made the following six parameters trainable: To account for the fact that channel models were built upon measurements from other cells, we allowed a fivefold variation of the time constant of the high-voltage-activated and low-voltage-activated calcium channels and the M channel, and we introduced a parameter that can shift the activation curve of the somatic sodium channel by up to 10 mV to the left. In addition, to account for ion channels typically being selective to more than a single ion, we allowed the potassium reversal potential to vary within [−100, −70] mV and the sodium reversal potential to vary within [40, 60] mV. To fit recordings from the Allen Cell Types Database (IDs 485574832, 488683425, 480353286 and 473601979), we modified the leak conductance to 10−4 for all cells, the leak reversal potential to −88, −88, −95 and −95 mV and capacitance to 6, 2, 6 and 2.5 μF cm−2, respectively. Optimizing parameters of biophysical models with gradient descent requires that the loss function, and therefore also the summary statistics, are differentiable. This is not the case for typically used features such as spike count, which led us to defining different loss functions. 2b), we split the voltage trace into two windows and used the mean and standard deviation of these two windows as summary statistics. This led to a total of four summary statistics. To standardize the data, we divided the mean voltages by 8.0 and standard deviations by 4.0. We used a mean absolute error loss on the standardized summary statistics. To fit data from the Allen Cell Types Database, we used AllenSDK to download morphologies and electrophysiology. We used a loss function based on DTW. Unlike the standard Euclidean (pointwise) distance, which can overly penalize minor temporal misalignment of fast events such as action potentials, DTW actively aligns the two signals in time before measuring their pointwise differences. This alignment emphasizes the geometric similarity between time series rather than strict temporal correspondence, which we found to work well for fitting spiking voltage traces. Because the standard DTW discrepancy is not fully differentiable, we use the soft DTW discrepancy46. For efficient computation on GPUs, we use a custom implementation of soft DTW discrepancy in JAX. We preprocessed the raw voltage traces by applying a sliding window maximum reduction with a window size of 50 time steps and a stride of 30 time steps (each time step is 0.025 ms). For the synthetic data, we used our variant of Polyak gradient descent with γ = 1/3 and β = 0.8. We trained for ten optimization steps. We also ran the genetic algorithm (IBEA-DEAP as implemented in the BluePyOpt package, version 1.14.11)16 for ten iterations, with ten simulations per iteration. For models based on the Allen Cell Types Database, we fitted the first 200 ms of the trace with a loss based on dynamic time warping (see above) and Polyak gradient descent with γ = 5 × 10−4 and β = 0.99. For the fits with large computational budget (Fig. 2d), we parallelized 1,000 gradient descent runs on a GPU and trained for 50 steps. We ran the genetic algorithm with 1,000 offspring (which we also parallelized on GPU) for 50 generations. For the fits with reduced computational budget (Extended Data Fig. 5), we followed the same procedure and used the same hyperparameters, but we used only ten gradient descent runs and trained only for ten steps. We did this for the maximal conductances of all active ion channels in the apical dendrite (three active mechanisms) and the axon (seven active mechanisms). We used a single parameter for each mechanism in the soma and basal dendrite. This resulted in a total of 1,390 parameters. We used a mean absolute error loss function between the observed voltages and the simulated voltages, evaluated at every fifth time step between 1 ms and 5 ms of simulation time. We trained the system for 500 iterations. We regularized the optimization such that neighboring branches had similar conductance values. To perform Bayesian inference over the parameters, we used a uniform prior over the free parameters with equivalent bounds as with the original training procedure. We sampled from the posterior distribution with Hamiltonian Monte Carlo, which leverages the available gradient to efficiently sample from high-dimensional distributions. We initialized 200 chains with samples from the prior distribution. Each chain was run for 500 iterations in parallel on a single NVIDIA GeForce A100 GPU. For each iteration, we performed five integration steps for the Hamiltonian dynamics with a step size of 0.01, leading to an average acceptance rate of ≈65%. We used the BlackJAX47 implementation of Hamiltonian Monte Carlo. To visualize the learned conductance profile, we discretized distance from soma into 11 bins and grouped all parameters within each bin. We then calculated histograms for each of the bins and generated a spline interpolation of all quantile lines (Fig. We trained sodium, potassium, and leak maximal conductances, as well as radius, length, and axial resistivity of every compartment in the model. Parameters were initialized randomly within uniform bounds. We used a mean absolute error loss function between the simulated voltages after 3 ms of simulated time and the class label (35 mV or −70 mV). We used the Adam optimizer with a learning rate of 0.01, and a batch size of 1. We trained the system for 200 epochs. One iteration of gradient descent took 1.7 ms on an Apple MacBook Pro CPU. Ran et al.18 presented images of 15 × 20 pixels to a mouse retina. Each pixel had a size of 30 × 30 μm. In total, 15 recording fields were made for the Off alpha cell used in our work, leading to a total of 1,024 × 15 training data points. Within each recording field, Ran et al.18 defined a variable number of regions of interest, within which the calcium activity was recorded. In total, the data contained 232 regions of interest. To denoise the calcium data, we low-pass filtered the raw calcium data with a Butterworth filter and a cutoff frequency of 7 Hz. We z-scored the resulting signal, with a different mean and standard deviation for each region of interest. Next, we generated a single label for each image. We did this by performing a linear regression from the image onto delayed calcium signals, and then used the calcium value at the delay, which was most predictive (that is, had the highest Pearson correlation between the linear regression prediction and data). This led us to a delay of 1.8 s. As the label, we used the low-pass filtered calcium value after this delay (starting from image onset). We followed Ran et al.18 to compute receptive fields. We used automatic smoothness detection48 with 20 iterations of evidence optimization. We standardized all receptive fields to range from 0 to 1 and, for contours, thresholded the receptive fields at a value of 0.6. We modeled photoreceptors as a spatial linear Gaussian filter. The filter had a standard deviation of 50 μm. We modeled bipolar cells as point neurons with a nonlinearity. The point neurons were spaced on a hexagonal grid with a distance between neurons of 40 μm. The nonlinearity was taken from values measured by Schwartz et al.29. We connected every bipolar cell onto every branch of the RGC that was within 20 μm of the bipolar cell. We convolved this value with a double-exponential kernel to model dynamics of the calcium indicator. Our model of the RGC contained six ion channels, which were developed based on measurements from the RGCs in cats49. The model had all of these channels in every compartment of the model. We initialized all membrane conductances at previously published values49, except for sodium conductances, which we initialized at 0.15 S cm−2 in the soma and 0.05 S cm−2 in the dendrite. We sampled the initial synaptic strengths randomly within 0 and 0.1 nS, and then divided the synaptic conductance by the number of postsynaptic connections that a bipolar cell makes (such that, in expectation, every bipolar cell has the same impact on the RGC). We initialized the axial resistivity of every compartment at 5,000 Ωcm. Finally, we initialized the radius of every dendritic compartment at 0.2 μm. We trained the following set of parameters: One value for each maximal conductance in the soma (six parameters), one value for each maximal conductance in the dendrites, shared across all dendrites (six parameters), one value for each branch radius (147 parameters), one value for the axial resistivity of each branch (147 parameters) and one value for each synaptic conductance from the bipolar cells onto the RGCs (250 parameters). In total, the model had 556 parameters. All membrane conductance units are S cm−2. For dendritic conductances, we used the same bounds apart from the lower bound of 0 for sodium. For the branch radii, we used bounds of [0.1, 1.0] μm (ref. For the axial resistivities, we used [100, 10,000] Ωcm. For the synaptic conductances, we used [0.0, 0.2] nS. We considered every kind of parameter (somatic conductance, dendritic conductance, radii, axial resistivities and synaptic conductances) as separate parameters (which influences the computation of the gradient norm in Polyak stochastic gradient descent). We trained the model for ten epochs with a learning rate of 0.01 and a momentum of 0.5. We used β = 0.99 to compute the norm in Polyak stochastic gradient descent. We used a batch size of 256 and used two-level checkpointing to reduce the memory of backpropagation. To avoid vanishing or exploding gradients, we truncated the gradient in time. Specifically, we reset the gradient to 0 after 50, 100 and 150 ms of simulations. We trained the model with mean absolute error loss between the experimentally measured (low-pass filtered) calcium value and the model-predicted calcium value after 200 ms. Recording sites for which no recording was available in the data were masked out in the loss computation. To evaluate the inductive bias of the hybrid model, we trained several such models, each on a reduced dataset. We reduced the dataset by using only a subset of stimulus–recording pairs from one scan field. We repeated this procedure over seven scan fields and five sizes of stimulus–recording pairs, namely 32, 64, 128, 256 and 512. We trained each model with stochastic gradient descent as described above. To ensure that the gradient is stochastic, we used batch sizes of 4, 4, 8, 16 and 32 for the five dataset sizes, respectively. We performed early stopping based on a validation set that contained 25% of the training dataset and evaluated performance on 512 test data points. To avoid exceedingly long training times, we trained for at most 100 steps. For the artificial neural network, we used a multilayer perceptron with three hidden layers of [100, 100, 50] units and with ReLU activation functions. The two readout neurons were passive. The synapses were conductance based, as described by Abbott and Marder50. All recurrent units had synapses onto the two readout neurons. Around 50% of recurrent units had inhibitory outgoing synapses, created by setting their synaptic reversal potential to −75 mV, and the other half were excitatory with a synaptic reversal potential of 0 mV. 4b had 50 units with an 80% excitatory/20% inhibitory split, and a connection probability of 0.2. We used bounds of [−0.2, 0.2] for the input weights and [0, \(3\max (g)\)] for the maximal synaptic conductances. In total, the model had 109 trainable parameters. Initial values for the maximum synaptic conductances were drawn from a standard normal distribution scaled by an initial gain g such that the bulk of the eigenspectrum of the synaptic weights lay in a circle on the complex plane with radius g (after multiplying inhibitory synapse weights by −1). We presented inputs by stimulating neurons at their basal dendrites. We set g = 5.0π/5 × 103, which is close to the transition point between stable and chaotic dynamics. For recurrent connections, we set the rate constant for transmitter–receptor dissociation rate (the k_ parameter2), that influences the synaptic time constants, to 1/1.0 ms. For connections to the rate-based readout neurons, we used slower synapses, with k_ set to 1/40 ms. All recurrent units received stimulus input scaled by random initial weights drawn from \({\mathcal{N}}(0,0.1)\). Stimuli were generated by sampling values from normal distributions and then low-pass filtering them with a maximum frequency cutoff of 2,500 Hz. We used a batch size of four, three levels of checkpointing, the Adam optimizer with a learning rate of 0.01, 2,000 gradient steps and gradient normalization with β = 0.8. We sampled new training data at every gradient step. Sweeps were used to inform the choice of hyperparameters. We used a cross-entropy loss function with logits calculated as the mean readout activities in the last 20 ms of stimulus presentation. We used the same channel mechanisms, compartments and synapses as in the evidence integration task. We used a network of 50 units, with a recurrent connectivity probability of 0.05, and all units connecting to the readout neurons. The connection probability from input to (the basal dendrite of) recurrent units was 0.1, with initial weights drawn from \({{\mathcal{U}}}_{[0,1]}\). Stimuli consisted of square pulses with additive Gaussian noise sampled from \({\mathcal{N}}(0,0.001)\). The onset period was 20 ms, and the stimulus and response durations were 50 ms. Initial delay durations were drawn uniformly from \({{\mathcal{U}}}_{[50,150]}\). The average delay duration was increased in steps of 100 ms, whenever the network got at least 95% accuracy on a single batch, until \({{\mathcal{U}}}_{[450,550]}\) was reached. We used a batch size of 64, two levels of checkpointing and the Adam optimizer with a learning rate of 0.001, which was exponentially decayed to 0.0001 over 1,000 epochs. We used a cross-entropy loss function for each time step, computed during the response period of the task. The training time was around 10 h on an NVIDIA GeForce RTX 3090. To obtain the maximal Lyapunov exponent, we discretized our model to obtain xt+1 = f(xt), where xt contains voltages and gating variables at time t, and f is one step of the solver. We used the following numerical algorithm to approximate \({\lambda }_{\max }({{\bf{x}}}_{0})\)33: First, we generated an initial state x0 and initial unit norm vector q0. After discarding initial transients for 4 s of simulation, we let the system xt+1 = f(xt) and \({{\bf{q}}}_{t+1}=D{\bf{f}}{| }_{{{\bf{x}}}_{t}}{{\bf{q}}}_{t}\) (where D denotes the Jacobian) evolve for T = 2.4 × 105 time steps (a further 6 s). Note that qt+1 can be efficiently computed using Jacobian vector products in JAX20. The first layer consisted of 28 × 28 neurons, each stimulated by a step current whose amplitude was proportional to a pixel value. Each neuron in the first layer had a ball-and-stick morphology: Each cell consisted of four compartments, where one compartment (the soma) had a radius of 10 μm and all other compartments had a radius of 1 μm and a length of 10 μm. The input and hidden layer contained sodium, potassium and leak channels in all branches, with dynamics following the default implementation of the Neuron package12. The output layer consisted of ten neurons with ball-and-stick morphologies (such as the input layer neurons) and with leak dynamics. We used conductance-based synapses as described by Abbott and Marder50. We optimized sodium, potassium and leak maximal conductances of every branch in the network (55,000 parameters) and all synaptic conductances (51,000 parameters). We initialized synaptic conductances as samples from a Gaussian distribution with a mean of 0 and standard deviation of 1 / 282 / 25 for the first layer and a standard deviation of 1 / 64 / 25 / 2 for the second layer. We set the synaptic rate constant for transmitter–receptor dissociation to k− = 1/4. We used a batch size of 16 and cross-entropy loss based on the values (v + 65)/3, where v is the somatic voltage of the output neurons after 10 ms of simulation. We used a cosine learning rate schedule and trained the network for seven epochs. Each gradient step took 25 s on a V100 GPU. To perform the adversarial attacks, we performed optimization of the input with gradient descent. We normalized every gradient step and used a learning rate of 5.0. We used bounds of [0, 1] for all pixel values during optimization. We used a cross-entropy loss function. We computed accuracy based on 128 adversarial attacks. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All code to generate results and figures is available at https://github.com/mackelab/jaxley_experiments/. We used AllenSDK (v2.16.2) to obtain data from the Allen Cell Types Database (IDs 485574832, 488683425, 480353286 and 473601979). 1d–f and 5) from https://neuromorpho.org/ (the cell shown in Fig. 1d has ID NMO_00120, other cells are from the same archive and of the same cell type). Our toolbox, Jaxley, is openly available at https://github.com/jaxleyverse/jaxley/ under the Apache 2.0 license. Tutorials and examples of how to use the toolbox are available at https://jaxley.readthedocs.io/en/latest/. A collection of channels and synapses for use with Jaxley is available at https://github.com/jaxleyverse/jaxley-mech/ under the MIT license. Hodgkin, A. L. & Huxley, A. F. A quantitative description of membrane current and its application to conduction and excitation in nerve. A., Bucher, D. & Marder, E. Similar network activity from disparate circuit parameters. Hay, E., Hill, S., Schürmann, F., Markram, H. & Segev, I. Models of neocortical layer 5b pyramidal cells capturing a wide range of dendritic and perisomatic active properties. Markram, H. et al. Reconstruction and simulation of neocortical microcircuitry. Systematic generation of biophysically detailed models for diverse cortical neuron types. Mante, V., Sussillo, D., Shenoy, K. V. & Newsome, W. T. Context-dependent computation by recurrent dynamics in prefrontal cortex. Lappalainen, J. K. et al. Connectome-constrained networks predict neural activity across the fly visual system. Van Geit, W., De Schutter, E. & Achard, P. Automated neuron model optimization techniques: a review. AlQuraishi, M. & Sorger, P. K. Differentiable biology: using deep learning for biophysics-based and data-driven modeling of molecular mechanisms. Dorigo, T. et al. Toward the end-to-end optimization of particle physics instruments with differentiable programming. Shen, C. et al. Differentiable modelling to unify machine learning and physical models for geosciences. Carnevale, N. T. & Hines, M. L. The Neuron Book (Cambridge University Press, 2006). Kumbhar, P. CoreNEURON: an optimized compute engine for the NEURON simulator. Ben-Shalom, R. et al. NeuroGPU: accelerating multi-compartment, biophysically detailed neuron simulations on GPUs. Zhang, Y. et al. A GPU-based computational framework that bridges neuron simulation and artificial intelligence. Van Geit, W. et al. BluePyOpt: leveraging open source software and cloud infrastructure to optimise model parameters in neuroscience. Gonçalves, P. J. et al. Training deep neural density estimators to identify mechanistic models of neural dynamics. Ran, Y. et al. Type-specific dendritic integration in mouse retinal ganglion cells. Task representations in neural networks trained to perform many cognitive tasks. Bradbury, J. et al. JAX: composable transformations of Python+NumPy programs. Hazelden, J., Liu, Y. H., Shlizerman, E. & Shea-Brown, E. Evolutionary algorithms as an alternative to backpropagation for supervised training of biophysical neural networks and neural ODEs. Achieving logarithmic growth of temporal and spatial complexity in reverse automatic differentiation. Polyak, B. T. Gradient methods for minimizing functionals. Zhurnal Vychislitel'noi Matematiki i Matematicheskoi Fiziki 3, 643–653 (1963). Ascoli, G. A., Donohue, D. E. & Halavi, M. NeuroMorpho.org: a central resource for neuronal morphologies. Migliore, M. & Shepherd, G. M. Emerging rules for the distributions of active dendritic conductances. Poirazi, P., Brannon, T. & Mel, B. W. Pyramidal neuron as two-layer neural network. The spatial structure of a nonlinear receptive field. Schröder, C. et al. System identification with biophysical constraints: a circuit model of the inner retina. McIntosh, L., Maheswaranathan, N., Nayebi, A., Ganguli, S. and Baccus, S. Deep learning models of the retinal response to natural scenes. A. Neuronal correlates of a perceptual decision. Fuster, J. M. & Alexander, G. E. Neuron activity related to short-term memory. Dura-Bernal, S. et al. NetPyNE, a tool for data-driven multiscale modeling of brain circuits. Akar, N. A. et al. Arbor—a morphologically-detailed neural network simulation library for contemporary high-performance computing architectures. In 2019 27th Euromicro International Conference on Parallel, Distributed and Network-based Processing (PDP) 274–282 (IEEE, 2019). Nandi, A. et al. Single-neuron models linking electrophysiology, morphology, and transcriptomics across cortical cell types. Combined statistical-biophysical modeling links ion channel genes to physiology of cortical neuron types. Scala, F. et al. Phenotypic variation of transcriptomic cell types in mouse motor cortex. Jedlicka, P., Bird, A. D. & Cuntz, H. Pareto optimality, economy–effectiveness trade-offs and ion channel degeneracy: improving population modelling for single neurons. Yu, Y., Xia, R., Ma, Q., Lengyel, M. & Hennequin, G. Second-order forward-mode optimization of recurrent neural networks for neuroscience. Stone, H. S. An efficient parallel algorithm for the solution of a tridiagonal linear system of equations. Cuturi, M. & Blondel, M. Soft-DTW: a differentiable loss function for time-series. & Louf, R. BlackJAX: composable Bayesian inference in JAX. Evidence optimization techniques for estimating stimulus-response functions. Fohlmeister, J. F. & Miller, R. F. Impulse encoding mechanisms of ganglion cells in the tiger salamander retina. We thank N. Bosch, P. Hennig, S. Müller, S. Axen, T. Euler, M. Bethge, T. Zenkel, F. D'Agostino, J.-M. Lueckmann and all members of our research groups for discussions. This work was supported by the German Research Foundation (DFG) through Germany's Excellence Strategy (EXC 2064 - project number 390727645; to P.B. and the Collaborative Research Center 1233 ‘Robust Vision' (to P.B. ), the ‘Certification and Foundations of Safe Machine Learning Systems in Healthcare' project funded by the Carl Zeiss Foundation (to J.H.M.) and the European Union (ERC, ‘DeepCoMechTome', ref. Views and opinions expressed are those of the authors only and do not necessarily reflect those of the European Union or the European Research Council Executive Agency. are members of the International Max Planck Research School for Intelligent Systems (IMPRS-IS). Open access funding provided by Eberhard Karls Universität Tübingen. Michael Deistler, Matthijs Pals, Manuel Gloeckler, Janne K. Lappalainen, Cornelius Schröder, Pedro J. Gonçalves & Jakob H. Macke Michael Deistler, Kyra L. Kadhim, Matthijs Pals, Jonas Beck, Ziwei Huang, Manuel Gloeckler, Janne K. Lappalainen, Cornelius Schröder, Philipp Berens, Pedro J. Gonçalves & Jakob H. Macke Kyra L. Kadhim, Jonas Beck, Ziwei Huang, Philipp Berens & Jakob H. Macke VIB-Neuroelectronics Research Flanders (NERF), Leuven, Belgium Department Empirical Inference, Max Planck Institute for Intelligent Systems, Tübingen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Michael Deistler or Jakob H. Macke. The authors declare no competing interests. Nature Methods thanks Andrew Davison and Samuel Neymotin for their contribution to the peer review of this work. Primary Handling Editor: Nina Vogt, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Histogram of two biophysical parameters (rows, random samples within optimization bounds) before (left) and after (right) parameter transformations. (b) Illustrative loss surface and update step made by our variant of Polyak gradient descent. Multi-level checkpointing requires multiple forward passes per gradient computation, but typically reduces memory requirements. (e) Illustration of truncated backpropagation through time. Truncated backpropagation through time allows to overcome vanishing or exploding gradients at the cost of providing only an approximate gradient. Top: Spike time at three recording sites (somatic, proximal, distal) for input stimuli of 10 different amplitudes, ranging from 0.2 nA to 1.1 nA (individual dots). Columns are different morphologies from the Allen Cell Types Database. Bottom: Same as top, for spike amplitude. The networks consist of 12k neurons, each with three branches and Hodgkin–Huxley dynamics. Networks were connected densely within clusters and sparsely between clusters via tanh-synapses, for a total of 25.4 million synapses. Jaxley can simulate networks of this complexity on a single A100 GPU (activity on the right) and also compute the gradient with respect to more than 25 million parameters (all sodium, potassium, leak, and synaptic conductances) with backpropagation. Number of simulations and runtime required to achieve a loss smaller than 0.55 when fitting the L5PC to synthetic recordings, with genetic algorithm (GA) and gradient descent (GD). Averaged across ten runs with 10 iterations each. Loss over the number of iterations for the median of ten genetic algorithms, each run with ten simulations per iterations (black) and the minimal loss across ten gradient descent runs (blue). Ten gradient descent runs require the same number of simulations as a single run of the genetic algorithm. Across all four morphologies, gradient descent identifies parameter sets with lower loss than the genetic algorithm, but we note that computing the gradient incurs additional computational cost. Gradient descent and the genetic algorithm were run for fifty iterations and with 1,000 simulations per step (in parallel on GPU). The gradient descent fit with reduced budget (light blue) was run for ten iterations and ten simulations per step. We used Jaxley with gradient-based Hamiltonian Monte–Carlo to infer the posterior distribution over membrane conductances of a layer 5 pyramidal cell. Blue lines are 90 % confidence intervals, black line is the ground truth that was used to generate the synthetic observation. Evidence integration task performance on three seconds of stimulus with positive integral (left) and negative integral (right). We used the same network parameters as in Fig. Long-term dynamics of the network from Fig. 4 trained to perform the delayed-match-to-sample task. We first smoothed each dynamic variable with a Hann window of 40 ms, and normalised, before computing the principal components. Right: Hidden layer tuning of all neurons after training. We evaluated the fraction of images in the dataset for which each hidden neuron spiked. Purple dots highlight the five most changed synaptic conductances. Especially for the first layer, synaptic conductances changed strongly during training. In both layers, parameters were significantly correlated to their initial value (p < 10−8, two-sided t-test). Supplementary Discussion, Results, Table 1 and Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Deistler, M., Kadhim, K.L., Pals, M. et al. Jaxley: differentiable simulation enables large-scale training of detailed biophysical models of neural dynamics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02396-8'>Genome-wide association analyses identify distinct genetic architectures for early-onset and late-onset depression</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 10:06:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Major depressive disorder (MDD) is a common and heterogeneous disorder of complex etiology. Studying more homogeneous groups stratified according to clinical characteristics, such as age of onset, can improve the identification of the underlying genetic causes and lead to more targeted treatment strategies. We leveraged Nordic biobanks with longitudinal health registries to investigate differences in the genetic architectures of early-onset (eoMDD; n = 46,708 cases) and late-onset (loMDD; n = 37,168 cases) MDD. Overall, the two MDD subtypes correlated moderately (genetic correlation, rg = 0.58) and differed in their genetic correlations with related traits. These findings suggest that eoMDD and loMDD have partially distinct genetic signatures, with a specific developmental brain signature for eoMDD. Importantly, we demonstrate that polygenic risk scores (PRS) for eoMDD predict suicide attempts within the first 10 years after the initial diagnosis: the absolute risk for suicide attempt was 26% in the top PRS decile, compared to 12% and 20% in the bottom decile and the intermediate group, respectively. Taken together, our findings can inform precision psychiatry approaches for MDD. Like other complex disorders such as type 2 diabetes1 and epilepsy2, the clinical heterogeneity observed in major depressive disorder (MDD) probably stems from the underlying etiological heterogeneity3,4,5. Recent advances in genome-wide association studies (GWAS) of MDD, facilitated by large samples from the Psychiatric Genomics Consortium6,7, 23andMe8, Million Veteran Program9 and global biobanks10,11,12,13, have yielded substantial progress in identifying MDD-associated genetic variants, and evidence of genetic differences between various clinical subtypes has emerged3,7,13. Despite these efforts, the quest for subtype-specific genetic loci in MDD has been slow, limiting our understanding of its complex etiology. In this study, we investigate an important source of MDD heterogeneity, that is, age at onset (AAO), by stratifying individuals into early-onset MDD (eoMDD) and late-onset MDD (loMDD) and conducting a large GWAS meta-analysis of the two subtypes in the ongoing Nordic TRYGGVE collaboration14,15 (Fig. eoMDD is associated with severe outcomes, including psychotic symptoms, suicidal behavior and comorbidities with other mental disorders and somatic diseases16,17, whereas loMDD tends to manifest with cognitive decline and increased cardiovascular disease risk18. Previous attempts to stratify MDD according to AAO have been hindered by methodological challenges, including large variations in AAO across samples, recall bias and relatively small sample sizes17. To address these challenges, we leveraged the Nordic biobanks and harmonized longitudinal health registries to stratify MDD cases based on age at first MDD diagnosis15. Previous research suggested that age at first diagnosis could be a useful proxy for AAO, given the high genetic correlation (rg = 0.95) between the two phenotypes16. Following harmonization of phenotypic definitions of MDD and age at first diagnosis across nine cohorts from five Nordic countries (Denmark, Estonia, Finland, Norway and Sweden; Methods), we identified 151,582 MDD cases, including 46,708 eoMDD cases with age at first diagnosis of less than 25 years (approximating the 25th percentile of an AAO distribution ≤ 20–21 (ref. 19; Methods)) and 37,168 loMDD cases with age at first diagnosis of 50 years or older (approximating the 75th percentile of an AAO distribution ≥ 44–45 years; Methods) (Table 1)19. We conducted harmonized GWAS analyses on eoMDD and loMDD using singularity containers20 in each cohort and then performed meta-analyses. After phenotype harmonization, we observed high genetic correlations (rg = 0.7–0.9) among the largest Nordic cohorts (the Integrative Psychiatric Research Consortium (iPSYCH), the Estonian Biobank (EstBB), FinnGen) (Supplementary Fig. To assess generalizability outside the Nordic cohorts, we also analyzed UK Biobank (UKB) data, which relies on self-reported age at first diagnosis, and conducted GWAS of eoMDD and loMDD with the same age cutoffs (Supplementary Fig. However, considering the major differences in samples and phenotypes, we conducted primary analyses based on Nordic cohorts of individuals with European ancestry, with the UKB as a comparison cohort for the identified loci. We identified 12 genome-wide significant loci for eoMDD and two other loci for loMDD (P < 5 × 10−8) (Fig. These loci were also captured in the GWAS of all cases with MDD, where we identified 80 significant loci (Supplementary Fig. Nearly all of these MDD loci have been reported in previous MDD GWAS13 (Supplementary Table 3), demonstrating the validity of our harmonized phenotypes and GWAS. For the 17 significant genes identified in the eoMDD analysis, BPTF21, PAX522, SDK123 and SORCS324 are involved in neurodevelopment or synaptic signaling (Supplementary Table 2). For loMDD, we identified four significant genes, with BSN implicated in synaptic neurotransmitter activity25 (Supplementary Table 2). Top loci were only partly replicated in the UKB, with only one locus on chromosome 9 showing nominal significance (P < 0.05); however, for the genome-wide significant loci in either sample, the correlation in their effect sizes was substantial (r = 0.84 for eoMDD; Supplementary Fig. a,b, Mirrored Manhattan plots of the GWAS results from the combined Nordic countries, using inverse-variance-weighted meta-analysis with a genome-wide significance threshold of P < 5 × 10−8 (denoted by the dashed horizontal line), for eoMDD (a) and loMDD (b). c, Enrichment of open chromatin marks in the eoMDD and loMDD GWAS. The dashed line indicates the one-sided Bonferroni-corrected P threshold, set at P = 0.05/102 = 0.0005. d, Single-nucleotide polymorphism (SNP)-based heritability \(\left({h}_{{\rm{SNP}}}^{2}\right)\) across a range of population prevalence estimates, with the labeled dashed lines indicating the point estimate of the population prevalence. e, Genetic overlap between broad MDD, eoMDD and loMDD, with the upper triangle containing the standard errors and the diagonal containing estimates of polygenicity from SBayesS. * For loMDD, the algorithm did not converge, making the polygenicity estimate unreliable, although it was consistently higher than for eoMDD across successful runs. By integrating GWAS findings on psychiatric disorders with tissue-specific open chromatin marks from the RoadMap Epigenomics Project26, previous research implicated gene regulation during fetal neurodevelopment27. We tested the specific hypothesis that the GWAS of eoMDD would be enriched at regulatory chromatin marks active in fetal brains. Indeed, we found that eoMDD genetic signals were significantly enriched in fetal brain tissues, whereas no enrichment was detected in adult brains (Fig. 2c and Supplementary Table 6a), suggesting a role of early brain development in the risk of eoMDD. For loMDD, we only detected enrichment of one epigenetic marker in male fetal tissues. We did not identify significant enrichment in Genotype-Tissue Expression (GTEx) brain tissues28 or human adult brain cell types29, potentially because of the relatively low statistical power in these subtype-specific GWAS (Supplementary Fig. The single-nucleotide polymorphism (SNP)-based heritability \(\left({h}_{{\rm{SNP}}}^{2}\right)\) estimates for eoMDD were higher than for loMDD at a range of population prevalences (Fig. Assuming a population prevalence of 6.2% (Methods), the \({h}_{{\rm{SNP}}}^{2}\) for eoMDD was estimated at 11.2% (95% confidence interval (CI) = 9.9–12.5%; liability scale), which was almost twice as high as that of loMDD at 6% (95% CI = 4.3–6.3%) at a prevalence of 3.2% (Supplementary Table 7). We also conducted a case–case GWAS directly comparing eoMDD with loMDD; however, this comparison showed a small \({h}_{{\rm{SNP}}}^{2}\) (2%, s.e. = 0.74%). Interestingly, analyses of the genetic architecture using SBayesS30 estimated that the polygenicity (that is, the proportion of SNPs with nonzero effects) in eoMDD (4%, posterior s.e. = 0.7%) was much lower than our GWAS of all MDD cases (7%, posterior s.e. 2e) and the estimate reported in the latest Psychiatric Genomics Consortium MDD GWAS (6%)6, suggesting that fewer causal variants underpin eoMDD. The two MDD subtypes correlated moderately (rg = 0.58, s.e. 2e and Supplementary Table 8) and showed differences in their genetic correlations with other traits (Fig. = 0.05), which was more than twice the rg between loMDD and suicide attempt (rg = 0.42, s.e. Similarly, substantial differences were found in their rg with posttraumatic stress disorder, childhood maltreatment, attention-deficit/hyperactivity disorder, autism spectrum disorder and schizophrenia. For somatic and lifestyle phenotypes, the overlap with eoMDD was significantly stronger for heart failure and body mass index (Supplementary Table 8). Given the rg between the two MDD subtypes, we further investigated the independent genetic effects of each subtype while conditioning on the other using genomic structural equation modeling (SEM) (Fig. In this way, the unique overlap between each subtype and the other traits could be evaluated, giving insight into how much of the overlap with the other trait was driven by overlap with the other subtype. After controlling for loMDD, the genetic associations of eoMDD with other traits remained similar, except that the negative rg between eoMDD and educational attainment was attenuated (Supplementary Table 9). On the other hand, after accounting for the genetic overlap with eoMDD, the genetic associations of loMDD were substantially reduced for many traits (for example, suicide attempt and suicide death), suggesting that the observed rg between loMDD and these traits were driven by the shared genetics with eoMDD. Overall, eoMDD had stronger genetic overlaps with psychiatric and general health traits than loMDD, with the most notable difference in their genetic correlations with suicide attempt. a, Genetic correlations from linkage disequilibrium (LD) score regression (LDSC) comparing eoMDD (based on GWAS n = 153,532) and loMDD (GWAS n = 158,588) with key health outcomes and related psychiatric disorders (GWAS n in Supplementary Table 15). P values for the associations are reported in Supplementary Table 8. b, Results from genomic SEM linear regression, where the outcomes are regressed on eoMDD (GWAS n = 153,532) while controlling for overlap with loMDD (GWAS n = 158,588), and vice versa. The single asterisks denotes a significant difference between eoMDD and loMDD at P < 0.05. We further investigated these relationships using two-sample Mendelian randomization (MR) (Fig. Considering the clear timing difference in the MDD subtypes, we were primarily interested in the role of eoMDD as a risk factor for general health outcomes, while providing the estimates of loMDD as a comparison, and the role of general health traits with loMDD as an outcome. Of note, eoMDD showed a significantly larger effect on suicide attempt than loMDD (eoMDD β = 0.61, s.e. = 0.052), while the magnitude of effects on suicide death was more comparable between the two subtypes (eoMDD β = 0.64, s.e. For educational attainment, our results were consistent with a previous report that suggested lower educational attainment as a putatively causal risk factor for MDD7, in particular for loMDD (β = −0.23, s.e. = 0.049; the effect on eoMDD was pleiotropic); furthermore, we showed that eoMDD had a putatively causal effect on lower educational attainment (β = −0.09, s.e. = 0.026). Similarly, our results that both subtypes had a small putatively causal effect on cardiovascular disease (CVD) were in line with previous findings considering MDD as a whole31; there was evidence suggesting that heart failure was putatively causal for loMDD. Results from sensitivity analyses were consistent with these main findings (Supplementary Table 10). b, Effects with MDD as the outcome (ordered according to the effect size in a). We only tested plausible relationships (in line with time ordering), with improbable relationships, for example, suicide death as a risk factor for MDD, excluded from the figure. If the MR-Egger sensitivity analysis had a significant intercept (triangles), this indicates that the MR estimate was pleiotropic and should not be interpreted. Estimates with an asterisk were significant after correction for multiple testing (exact P values are reported in Supplementary Table 10). To investigate whether polygenic risk scores (PRS) for eoMDD and loMDD were associated with clinical indicators or severe outcomes, we generated PRS using leave-one-out (LOO) GWAS summary statistics and tested PRS associations with these outcomes extracted from longitudinal health registry data in each cohort. In the meta-analysis and across individual cohorts, the eoMDD PRS explained a greater proportion of the phenotypic variance compared to the loMDD PRS for all outcomes of interest (Fig. increase in eoMDD PRS was associated with an increased risk of early-onset (odds ratio (OR) = 1.26, 95% CI = 1.13–1.4, P = 1.8 × 10−5) and lifetime risk of MDD (OR = 1.24, 95% CI = 1.17–1.31, P = 1.16 × 10−13) compared to the loMDD PRS for early (OR = 1.13, 95% CI = 1.08–1.17, P = 3.9 × 10−9) and lifetime risk of MDD (OR = 1.13, 95% CI = 1.07–1.19, P = 1.4 × 10−5). Similarly, compared to the loMDD PRS, the eoMDD PRS was more strongly associated with other outcomes of interest; differences in estimates for eoMDD and loMDD PRS were particularly pronounced for hospitalization and diagnostic conversion to bipolar disorder and schizophrenia. a, Meta-analyzed associations between PRS and MDD outcomes in the Nordic cohorts, using LOO GWAS summary statistics of eoMDD and loMDD. b, Selected mental-health-related and suicide-related ICD-10 group associated with eoMDD and loMDD from the PRS PheWAS. c, Number of unique and shared associations with eoMDD and loMDD from the PheWAS. d, Mean symptom scores of self-reported suicidality (Paykel Suicide Scale) stratified according to the top 10%, middle 80% and bottom 10% eoMDD PRS. e, Cumulative incidence of suicide attempt over a 10-year period since the first eoMDD diagnosis, stratified according to the top, middle and bottom eoMDD PRS. To understand how genetic predispositions underlying the two subtypes are associated with comorbidities, we conducted a PRS-based phenome-wide association study (PheWAS) using the EstBB medical records from both primary and specialist care (1,428 International Classification of Diseases, Tenth Revision (ICD-10) code-based diagnosis). After correction for multiple testing (Bonferroni correction, P threshold of 0.05/1,428), we identified both shared and unique associations with either subtype (Fig. Notably, the eoMDD PRS showed unique associations with psychiatric conditions such as conduct disorder (ICD-10 code: F91) and schizotypal disorder (F21), suicide attempt/intentional self-harm (X78) and problems related to negative life events in childhood (Z61), or other problems related to upbringing (Z62). In contrast, the loMDD PRS was more notably associated with mental and behavioral disorders because of the use of sedatives or hypnotics (F13) and obsessive–compulsive disorder (F42) (Fig. Prompted by our findings of the strong genetic link between eoMDD and suicide attempt, we investigated whether the eoMDD PRS predicts the risk of suicide attempt using self-report surveys and medical records from EstBB. We stratified individuals with eoMDD into three subgroups based on the eoMDD PRS (top and bottom deciles, and middle 80%). First, we observed a dose–response relationship between eoMDD PRS strata and the mean scores of the self-reported Paykel Suicide Scale32, that is, the higher eoMDD PRS, and the higher symptoms of suicide thoughts and attempts (mean symptoms scores = 0.85, 95% CI = 0.79–0.91; 1.05 (1.03–1.07), 1.22 (1.15–1.29) for the bottom, middle and top PRS strata, respectively) (Fig. Next, we estimated the hazard risk ratio (HRR) and the absolute risk for suicide attempt treated in primary or specialist care within a 10-year period since the first recorded MDD diagnosis. Compared with the middle PRS group, individuals in the lowest PRS decile had a significantly lower HRR for suicide attempt of 0.57 (95% CI = 0.49–0.68, P = 1.34 × 10−10), while those in the highest PRS decile had an increased risk (HRR = 1.13, 95% CI = 1.00–1.28, P = 0.058). Furthermore, individuals with the lowest PRS showed a consistently low absolute risk of suicide attempt over time after the first MDD diagnosis (12% cumulative incidence over a 10-year period), while there were no major differences in absolute risk between the middle and highest PRS group until 5.5 years, with the 10-year cumulative incidence at 20% and 26%, respectively (Fig. Previous research showed that the incidence of suicide attempts is particularly pronounced during adolescence and young adulthood33. Early identification of individuals at heightened risk for suicide is of major clinical importance. This period of high incidence of suicide attempts coincides with the risk for eoMDD, making it particularly relevant to perform risk prediction of suicide attempts in this subgroup. Together with our findings suggesting that eoMDD is putatively causal for suicide attempts, these results underscore the potential use of eoMDD PRS in stratifying the risk of suicide attempt among individuals with eoMDD (Supplementary Table 13b) and warrant further investigation of its relevance for suicide prevention33. MDD genomics has achieved tremendous progress, from zero significant associations in 2013 to 697 associations in 2025 for the MDD case–control phenotype6,7,34. In this study, we explored the strategy of going beyond case–control to target specific phenotypic subgroups, aiming to reduce genetic heterogeneity in MDD. Our harmonized healthcare data and analyses across countries improved analytical power and identified differential genetic signals in AAO-based subtypes. A similar approach can be extended to other clinical characteristics, such as vegetative symptoms, psychotic features and disability, to further target clinically relevant subtypes of MDD3,5,35,36. These findings may have important implications for guiding targeted treatment and prevention in psychiatry. To conduct a large GWAS of AAO-based MDD subtypes with consistent phenotypes, we identified nine cohorts across five Nordic countries (Denmark, Sweden, Norway, Finland and Estonia), and a comparison cohort from a non-Nordic country (the UK: UKB). Many of these cohorts are large biobanks linked with lifetime medical records in national patient registers, including iPSYCH, FinnGen, the Norwegian Mother, Father and Child Cohort Study (MoBa), the UKB and the EstBB (cohort details in Supplementary Methods). This study was approved by the Swedish ethical review authority (case no. 2023-03073), and each cohort was approved by the relevant institutional review boards. We expanded our previous effort of harmonizing the register-based phenotypes of MDD, age at first diagnosis and its outcomes in the three Scandinavian countries to other Nordic countries with similar registers15. Using national patient registers, we first extracted information on patient diagnoses of MDD using ICD-10 code F32 (depressive episode) or F33 (recurrent depressive episodes), with the exclusion criteria of a lifetime diagnosis of bipolar disorder or schizophrenia (ICD-10 codes: F30/F31/F32/F33, F20/F23.1/F23.2/F25) (Supplementary Table 14), resulting in a total combined number of MDD cases in the Nordic countries of n = 151,582. Our previous research revealed a high genetic correlation (rg ≈ 0.95) between AAO and age at first diagnosis for MDD16, and that onset on average predates first diagnosis by 5 years16. Therefore, we derived the age at first diagnosis for the MDD patient population and used that as a proxy for AAO for the definition of eoMDD and loMDD. The cutoffs were chosen based on careful review of the literature and the empirical registry data from Sweden. Previous meta-analysis reported that the median AAO for depressive disorder is around age 30, with the 25th and 75th percentile at age 21 and 44, respectively15,19; our Swedish registry data showed that for age at first MDD specialist diagnosis, the 25th and 75th percentile were at age 25 and 45, respectively15. Thus, we considered individuals with their first psychiatric specialist treatment contact for MDD at or before age 25 (approximate to the 25th percentile of the AAO distribution at age 20–21) as cases with eoMDD (n = 46,708); cases with loMDD were those with their first specialist treatment contact for MDD at or after age 50 (approximate to the 25th percentile of the AAO distribution at age 44–45; n = 37,168). Controls were individuals without a registered diagnosis of MDD, bipolar disorder or schizophrenia. Because of the differences in cohort design, controls were matched to cases for the population-based cohorts of EstBB, FinnGen, MoBa and UKB. The Danish and Swedish cohorts, which used case–cohort and case–control designs, did not require control matching. To ensure transparent and reproducible analyses of data across sites, we developed software containers and accompanying codes for data processing and analyses (for example, GWAS, PRS, LDSC) and distributed the containers across the study sites. Briefly, the primary software tools and dependencies were installed in virtual machines based on the Ubuntu 20.04 LTS Linux operating system via Docker (https://docker.com) and containerized using the Singularity Image File format (https://sylabs.io) for distribution20. All source codes and files are publicly available via GitHub and released under the GNU General Public License (GPL v.3.0). GWAS of individuals with European ancestries was conducted in each cohort using the REGENIE software package37 available in the software containers20. Analyses were adjusted for the first ten principal components (PCs), age and sex. Summary statistics from each cohort were included in a fixed-effects meta-analysis in METAL38 with the ‘SCHEME STDERR' command using the inverse variance weight of the corresponding standard errors for all phenotypes studied, resulting in 151,582 cases and 362,873 controls for all MDD, 46,708 cases and 106,824 controls for eoMDD, and 37,168 cases and 121,420 controls for loMDD. Additional variant filtering was applied: INFO score > 0.8, minor allele frequency ≥ 1% and n > 10,000. Genome-wide significance was set at P < 5 × 10−8. Genome-wide loci were identified based on genome-wide significant SNPs using PLINK (--clump command). A gene list was then created based on which genes physically overlapped the loci from the previous step (Supplementary Table 2). An additional case–case GWAS was performed comparing eoMDD cases to loMDD cases using the EstBB and FinnGen cohorts, as they include both eoMDD and loMDD cases in their biobanks. A total of 32,060 samples from FinnGen and 23,447 samples from EstBB were meta-analyzed using METAL; heritability \(\left({h}_{{\rm{SNP}}}^{2}\right)\) was estimated using LDSC39,40. We used MAGMA41 v.1.08 to test the aggregate association of variants by assigning SNPs to genes. Gene boundaries were expanded by 35 kb upstream and 10 kb downstream to allow for the inclusions of promoters and cis-enhancers. We used the updated deep whole-genome sequencing of the 1000 Genomes Project (European subset) as reference data42, downloaded from http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20220422_3202_phased_SNV_INDEL_SV/. We used stratified LDSC39 to perform tissue-type and cell-type analysis, estimating the enrichment of SNP heritability in a set of genome annotations. We downloaded precomputed LDSC scores from https://console.cloud.google.com/storage/browser/_details/broad-alkesgroup-public-requester-pays/LDSCORE/LDSC_SEG_ldscores/biorxiv/Roadmap_1000Gv3.tgz, containing data on 396 annotations of epigenetic marks from the ROADMAP project, restricting to the 102 annotations relating to the brain. For the gene expression data, we followed our previous protocol43, first aggregating gene expression across genes to get the average expression per gene in each cell type. We then defined cell-type specificity by dividing the expression of a gene in a cell type by its total expression across all cell types. For each cell type, we considered the top decile of specifically expressed genes as typifying that cell type, and computed LDSC scores for that annotation. In addition to the ROADMAP annotations, we also tested the enrichment of SNP-based heritability (one-sided) in human tissues (37 human tissues from GTEx28) and brain cell types using the latest human single-nucleus transcriptomic data (31 major human brain cell types from ref. We applied a Bonferroni correction for multiple testing in each annotation and trait (for example, in the analysis of ROADMAP data, significance was set at P < 0.05/102 for each trait). We used LDSC to estimate SNP-based heritability and genetic correlation40. To convert the SNP-based heritability to the liability scale, we calculated the sample prevalence of cases in the meta-analysis; for population prevalence, we used a lifetime estimate of diagnosed MDD of 16% (ref. To obtain corresponding estimates for eoMDD and loMDD, we estimated the proportion of cases with MDD who had an early-onset or a late-onset diagnosis in the largest population samples (EstBB and FinnGen) and took that as a percentage of the whole MDD prevalence estimate, resulting in a population estimate of 6.2% for eoMDD and 3.2% for loMDD. To estimate genetic correlations with other traits, we used phenotypes from publicly available summary statistics, selecting important psychiatric disorders and key health determinants and outcomes, including suicide, body mass index, educational attainment, substance use, mortality and cardiovascular disease (GWAS sources in Supplementary Table 15). Using SBayesS30, we confirmed the LDSC estimates of SNP-based heritability and further estimated the parameter of polygenicity, that is, the proportion of SNPs with nonzero effects, and negative selection. We used the precomputed LD Matrix from UKB participants (‘ukbEURu_imp_v3_HM3_n50k.chisq10'), which is available from the software authors. We ran four chains in parallel to estimate the convergence across starting parameters (--num-chains 4), with each run consisting of 25,000 iterations (--chain-length 25,000), with 5,000 iterations considered as burn-in iterations (--burn-in 5,000). Genomic SEM is an extension of SEM whereby GWAS summary statistics are used to capture the ‘observed' traits in the model so that the genetic architecture underlying the multivariate relationships between traits can be modeled. Genomic SEM relies on LDSC to estimate genetic correlations and is robust to any amount of sample overlap. We modeled the association of the MDD subtypes with other related phenotypes that have been well studied in the literature, including a range of psychiatric disorders, important health determinants, CVD and mortality (Supplementary Table 15). Next, we assessed the subtype-specific contribution to the other phenotypes by controlling for the other subtype (for example, we investigated the relationship between eoMDD and suicide attempt after controlling for loMDD). Thus, the unique contribution of each can be gauged while controlling for their shared variance. The genetic overlap between eoMDD and loMDD was moderate; therefore, the concern for strong bias through multicollinearity was limited. Following the analyses of genetic overlap, we used MR to assess whether the MDD subtypes were causally associated with selected health outcomes (risk factors, mortality and CVD; Supplementary Table 15). We did not include psychiatric outcomes because of the high likelihood of pleiotropy. Assuming that alleles are randomly distributed in the population (in a manner akin to experimental randomization), MR uses genetic variants as instrumental variables to capture a trait (‘exposure'). If the genetic variants are robust, non-pleiotropic instruments for the exposure and there is no direct association with the outcome or an unmeasured confounder, this provides support for a potential causal effect of the exposure. We conducted main and sensitivity analyses as implemented in the TwoSampleMR R package46 and computed the IVW statistic to estimate these associations. For interpretation, we focused on effect consistency across tests, rather than P values. Still, to give an indication of the extent of the multiple testing burden, we report significance according to false discovery rate-corrected P values (Benjamini–Hochberg method). We present several sensitivity analyses that correct for potential violations of the MR assumptions. Weighted median and mode analyses are more robust to invalid instruments and correct for outliers47. The MR-Egger analysis was used to assess any remaining pleiotropy (intercept) and adjust for such effects. The MR-Egger estimate is often underpowered, but it is still informative to check whether the effect is in the same direction as the other sensitivity analyses48. From the same TwoSampleMR package, it is possible to estimate the amount of heterogeneity (Q) in the SNP effects, and estimate the instrument strength F. We added a sensitivity analysis from the MR-PRESSO R package49, which has a different and more powerful way to adjust for the effects of outliers and is aimed at correcting for pleiotropy. Although we were not able to remove potential sample overlap from the analyses (LOO analyses would leave the exposure GWAS underpowered), this should not be a large threat to the validity of our results. As shown in Supplementary Table 15, the amount of sample overlap was limited in most cases and present only for a subset of traits. The aim of the MR analyses was to contrast causal associations between eoMDD and loMDD. However, there was overlap in the instrument sets for both traits. Although only one SNP is present in both sets, many had LD partners in common. Therefore, we performed another sensitivity analysis using a single instrument SNP for each trait. We selected the genome-wide significant SNP that had the largest P value for the other trait (filtering on a minor allele frequency > 0.10). For eoMDD, we used rs7622302 residing in the DAG1 gene on chromosome 3, with P = 0.98 for loMDD. This instrument explained 2.8% of the variance (R2) in eoMDD. For loMDD, we used rs3777421 in the IGF2R gene on chromosome 6, with P = 0.87 for eoMDD. This SNP explained 2.7% variance in loMDD. Although power was expected to be lower in these single-SNP analyses, pleiotropy was also reduced; instrument strength was still sufficient (F = 30.7 for eoMDD and F = 30.1 for loMDD) and the contrast between eoMDD and loMDD was maximized. PRS were calculated using LDpred252, which addresses some limitations of LDpred53. Briefly, LDpred2 is a Bayesian method that uses the LD structure in the genome combined with priors regarding the genetic architecture of a trait (SNP heritability and fraction of causal SNPs (hyperparameter p)) to compute PRS. We used the ‘auto' method of LDpred2 that maximizes the predictive performance of the PRS by testing a range of the fraction of causal SNPs. For our analyses, we used LD matrices provided by the authors of LDpred252, which were generated from the UKB samples (European ancestry) with 1.1 million HapMap3 SNPs. The LDpred2 calculations relied on the bigsnpr R library54 and were implemented in a set of custom R scripts55. Individual-level PRS in each cohort were computed using the LOO GWAS summary statistics on MDD subtypes and tested for associations with clinical characteristics or MDD-related outcomes (Supplementary Table 16) in logistic regression models adjusted for at least ten PCs (cohort-dependent), sex and birth year. For the outcomes that were available in all target samples, we performed fixed-effects meta-analysis using the R package metafor56. To investigate the genetic comorbidity of eoMDD and loMDD, we conducted a PRS PheWAS using logistic regression models, using the EstBB electronic health record (EHR) data. Analyses were performed in R (v.4.3). For each ICD-10 code with at least 50 cases (n = 1,151) present in the EstBB EHR, two models were constructed—eoMDD PRS and loMDD PRS as the predictor. These models used the normalized PRS (either eoMDD or loMDD) and a binary outcome variable indicating the presence or absence of the ICD-10 code. The EHR data encompassed both primary and secondary/inpatient care data through linkage to the Estonian central e-health database, covering the time from 2004 to 2022. The models included sex, year of birth and the first ten genetic PCs as covariates. To account for multiple testing, we applied Bonferroni correction, resulting in a conservative P threshold of 0.05/1,151. ORs (with 95% CIs), the number of cases per ICD-10 code and the P values for both eoMDD and loMDD models are provided in Supplementary Table 12 and Supplementary Figure 8a,b. To evaluate the effect of the eoMDD PRS on self-reported suicidality, we used data from the EstBB Mental Health Online Survey57. Self-reported suicidality was assessed using the Paykel Suicide Scale, which includes five items assessing self-reported suicidal thoughts and attempts32. A total of 58,732 individuals who had an MDD diagnosis were present in the EHRs (F32 or F33 codes) and responded to the Paykel Suicide Scale. For this analysis, the first four questions were scored on a scale from 0 to 4 (0 indicating the participant had never felt or experienced this, and 4 indicating the participant felt or experienced it often). The fifth question had binary responses (no/yes) corresponding to values of 0 or 2. The mean values and standard deviations of these sums were then computed for three groups based on eoMDD PRS deciles (1st, 2nd to 9th, 10th). Analyses were conducted using R (v.4.3). To estimate the absolute risk of EHR-based suicide attempt over 10 years after the first diagnosis of MDD, stratified according to eoMDD PRS, we used a Cox proportional-hazards model58, using data from the EstBB (results in Supplementary Table 13). Sex and year of birth were included as covariates. The analysis included a cohort of 10,539 individuals diagnosed with MDD (first diagnosis date in 2000 or later, identified by the ICD-10 codes F32 or F33, before or at the age of 25). Participants' birth years ranged from 1974 to 2004, with a mean birth year of 1990. We calculated the absolute risk (with 95% CIs) of suicide attempt after the first episode of depression for three groups based on eoMDD polygenic risk score deciles: the 1st decile, the 2nd to 9th deciles and the 10th decile. The risk of suicide attempt was assessed over a 10-year period (3,650 days) after the initial MDD diagnosis. ICD-10 codes X60-X84 and Y87 from the Estonian EHRs were used to identify suicide attempts. All analyses were conducted using R v.4.3, with the ggsurvfit (v.1.0.0)58 and survival (v.3.5.7) packages59. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The GWAS summary statistics reported in this article are available via figshare at https://doi.org/10.6084/m9.figshare.27830340 (ref. The central code repository for the software containers for the GWAS and post-GWAS analyses is available via GitHub at https://github.com/comorment/containers and Zenodo at https://doi.org/10.5281/zenodo.15096746 (ref. 55); the other tools and public reference data are available via GitHub at https://github.com/comorment/containers. Suzuki, K. et al. Genetic drivers of heterogeneity in type 2 diabetes pathophysiology. Stevelink, R. et al. GWAS meta-analysis of over 29,000 people with epilepsy identifies 26 risk loci and subtype-specific genetic architecture. Nguyen, T.-D. et al. Genetic heterogeneity and subtypes of major depression. The genetic basis of major depressive disorder. Nguyen, T.-D. et al. Genetic contribution to the heterogeneity of major depressive disorder: evidence from a sibling-based design using Swedish national registers. Adams, M. J. et al. Trans-ancestry genome-wide study of depression identifies 697 associations implicating cell types and pharmacotherapies. Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. Levey, D. F. et al. Bi-ancestral depression GWAS in the Million Veteran Program and meta-analysis in >1.2 million individuals highlight new therapeutic directions. Howard, D. M. et al. Genome-wide association study of depression phenotypes in UK Biobank identifies variants in excitatory synaptic pathways. Yao, X. et al. Integrative analysis of genome-wide association studies identifies novel loci associated with neuropsychiatric disorders. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Als, T. D. et al. Depression pathophysiology, risk prediction of recurrence and comorbid psychiatric disorders using genome-wide analyses. Lu, Y. et al. Genetic risk scores and family history as predictors of schizophrenia in Nordic registers. A. et al. Epidemiological overview of major depressive disorder in Scandinavia using nationwide registers. Harder, A. et al. Genetics of age-at-onset in major depression. Power, R. A. et al. Genome-wide association for major depression through age at onset stratification: Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium. Late versus early onset depression in elderly patients: vascular risk and cognitive impairment. Solmi, M. et al. Age at onset of mental disorders worldwide: large-scale meta-analysis of 192 epidemiological studies. Akdeniz, B. C. et al. COSGAP: Containerized statistical genetics analysis pipelines. Stankiewicz, P. et al. Haploinsufficiency of the chromatin remodeler BPTF causes syndromic developmental and speech delay, postnatal microcephaly, and dysmorphic features. Gofin, Y. et al. Delineation of a novel neurodevelopmental syndrome associated with PAX5 haploinsufficiency. Yamagata, M. Structure and functions of sidekicks. Breiderhoff, T. et al. Sortilin-related receptor SORCS3 is a postsynaptic modulator of synaptic depression and fear extinction. The presynaptic cytomatrix protein Bassoon: sequence and chromosomal localization of the human BSN gene. Kundaje, A. et al. Integrative analysis of 111 reference human epigenomes. Kim, S. S. et al. Leveraging single-cell ATAC-seq and RNA-seq to identify disease-critical fetal and adult brain cell types. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Siletti, K. et al. Transcriptomic diversity of cell types across the adult human brain. Widespread signatures of natural selection across human complex traits and functional genomic categories. Distinct biological signature and modifiable risk factors underlie the comorbidity between major depressive disorder and cardiovascular disease. Suicidal feelings in the general population: a prevalence study. Hu, K. et al. Risk factors, impact, and healthcare use related to initial suicide attempts: comprehensive analysis of Swedish population. Ripke, S. et al. A mega-analysis of genome-wide association studies for major depressive disorder. Nguyen, T.-D. et al. Genetic insights into psychotic major depressive disorder: bridging the mood-psychotic disorder spectrum. Kendler, K. S., Ohlsson, H., Lichtenstein, P., Sundquist, J. & Sundquist, K. The genetic epidemiology of treated major depression in Sweden. Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. de Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. Byrska-Bishop, M. et al. High-coverage whole-genome sequencing of the expanded 1000 Genomes Project cohort including 602 trios. Skene, N. G. et al. Genetic identification of brain cell types underlying schizophrenia. Kessler, R. C., Chiu, W. T., Demler, O., Merikangas, K. R. & Walters, E. E. Prevalence, severity, and comorbidity of 12-month DSM-IV disorders in the National Comorbidity Survey Replication. Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. The MR-Base platform supports systematic causal inference across the human phenome. Hartwig, F. P., Davey Smith, G. & Bowden, J. Robust inference in summary data Mendelian randomization via the zero modal pleiotropy assumption. Bowden, J., Davey Smith, G. & Burgess, S. Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. & Do, R. Detection of widespread horizontal pleiotropy in causal relationships inferred from Mendelian randomization between complex traits and diseases. Bowden, J. et al. Assessing the suitability of summary data for two-sample Mendelian randomization analyses using MR-Egger regression: the role of the I2 statistic. Bias correction for inverse variance weighting Mendelian randomization. Modeling linkage disequilibrium increases accuracy of polygenic risk scores. Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr. Viechtbauer, W. Conducting meta-analyses in R with the metafor package. Ojalo, T. et al. Cohort profile update: Mental Health Online Survey in the Estonian Biobank (EstBB MHoS). Sjöberg, D. D., Baillie, M., Fruechtenicht, C., Haesendonckx, S. & Treis, T. ggsurvfit: Flexible time-to-event figures. GWAS sumstats of early-onset, late-onset, and all MDD. This work was supported by the US National Institutes of Mental Health (R01s MH124871, MH121545 and MH123724 to P.F.S. ); the European Union's Horizon 2020 Research and Innovation Programme (grant nos. ); the European Research Council (grant no. ); the Research Council of Norway (grant nos. ); the Swedish Research Council (award no. ); the Swedish Brain foundation (no. ); the Swedish Research Council for Health, Working Life and Welfare (agreement no. ); the Estonian Research Council (no. ); and the Estonian Centre of Excellence for Well-Being Sciences, funded by grant no. TK218 from the Estonian Ministry of Education and Research (to K.L.). We acknowledge the Estonian Biobank (EstBB) research team: A. Metspalu, L. Milani, T. Esko, R. Mägi, M. Nelis and G. Hudjashov. Research in the EstBB was supported by the European Union through the European Regional Development Fund project no. Data analysis in the EstBB was carried out in part at the High-Performance Computing Center of the University of Tartu. The computations for the STAGE cohort and the UKB were enabled by resources provided by the National Academic Infrastructure for Supercomputing in Sweden and the Swedish National Infrastructure for Computing at the Uppsala Multidisciplinary Center for Advanced Computational Science, which is partially funded by the Swedish Research Council through grant agreement nos. This study used computing resources provided by UNINETT Sigma2 at the University of Oslo (UIO/USIT/TSD p697, grant no. The PREFECT study was funded by the Swedish foundation for Strategic Research (no. The BASIC study was supported by the Swedish Research Council (case no. Analyses in the UKB have been conducted under application no. The MoBa Study is supported by the Norwegian Ministry of Health and Care Services and the Ministry of Education and Research. We thank all the participating families in Norway who take part in this ongoing cohort study. The FinnGen project is funded by two grants from Business Finland (HUS 4685/31/2016 and UH 4386/31/2016) and the following industry partners: AbbVie, AstraZeneca UK, Biogen MA, Bristol Myers Squibb (and the Celgene Corporation & Celgene International II Sàrl), Genentech, Merck Sharp & Dohme, Pfizer, GlaxoSmithKline Intellectual Property Development Ltd., Sanofi US Services, Maze Therapeutics, Johnson & Johnson Innovative Medicine, Novartis, Boehringer Ingelheim and Bayer. All Finnish Biobanks are members of the BBMRI.FI infrastructure (www.bbmri-eric.eu/national-nodes/finland/). The Finnish Biobank Cooperative (FINBB) (https://finbb.fi/) is the coordinator of BBMRI-ERIC operations in Finland. Supplementary Table 17 lists the members of FinnGen. Open access funding provided by Karolinska Institute. These authors contributed equally: John R. Shorter, Joëlle A. Pasman, Siim Kurvits, Andreas Jangmo, Joonas Naamanka. These authors jointly supervised this work: Martin Tesli, Kelli Lehto, Alfonso Buil, Yi Lu. A full list of members and their affiliations appears in the Supplementary Information. Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden Joëlle A. Pasman, Arvid Harder, Kaarina Kowalec, Ying Xiong, Sara Hägg, Mikael Landén, Patrick F. Sullivan & Yi Lu Estonian Genome Centre, Institute of Genomics, University of Tartu, Tartu, Estonia Department of Chronic Diseases, Norwegian Institute of Public Health, Oslo, Norway Joonas Naamanka, Nelli Frilander & Iiris Hovatta Central Institute of Mental Health, Hector Institute for Artificial Intelligence in Psychiatry, Mannheim, Germany Espen Hagen, Elise Koch, Bayram C. Akdeniz, Oleksandr Frei & Ole A. Andreassen Department of Biochemistry and Medical Genetics, University of Manitoba, Winnipeg, Manitoba, Canada Institute of Biological Psychiatry, Mental Health Center Sct. Richard Zetterberg, Joeri J. Meijsen, Jesper R. Gådin, Thomas M. Werge & Alfonso Buil Center for Eating and feeding Disorders Research, Mental Health Center Ballerup, Copenhagen University Hospital, Mental Health Services Copenhagen, Copenhagen, Denmark Unit of Integrative Epidemiology, Institute of Environmental Medicine, Karolinska Institutet, Stockholm, Sweden Centre for Psychiatry Research, Department of Clinical Neuroscience, Karolinska Institutet & Stockholm Health Care Services, Region Stockholm, Karolinska Institutet, Stockholm, Sweden Department of Mental Health and Suicide, Norwegian Institute of Public Health, Oslo, Norway Departments of Genetics and Psychiatry, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA KG Jebsen Centre for Neurodevelopmental Disorders, University of Oslo, Oslo, Norway Division of Mental Health and Substance Abuse, Diakonhjemmet Hospital, Oslo, Norway Department of Mental Health, Norwegian Institute of Public Health, Oslo, Norway Department of Psychiatry, Østfold Hospital, Grålum, Norway Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The core analytical team consisted of J.R.S., J.A.P., S.K., A.J. proposed and designed the study. The core writing group consisted of J.R.S., J.A.P., S.K., A.J., J.N., M.T., K.L., A.B. contributed to the data or obtained funding. Correspondence to John R. Shorter or Yi Lu. was a paid adviser and shareholder for Neumora Therapeutics. has received speaker fees from Lundbeck, Janssen, Otsuka and Sunovion, and is a consultant to Cortechs.ai and Precision Health. has received speaker honoraria from Lundbeck and Otsuka. The other authors declare no competing interests. Nature Genetics thanks Nick Bass, Na Cai and Christel Middeldorp for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Shorter, J.R., Pasman, J.A., Kurvits, S. et al. Genome-wide association analyses identify distinct genetic architectures for early-onset and late-onset depression. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251112220239.htm'>A fierce crocodile ancestor that hunted before dinosaurs has been found</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-13 04:11:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A newly identified carnivorous reptile may look like a dinosaur at first glance, but researchers have confirmed that it was actually an early relative of modern crocodiles. This armored animal lived 240 million years ago, during a time just before dinosaurs appeared. Its long neck and agile body allowed it to strike quickly at prey. "This animal was an active predator, but despite its relatively large size, it was far from the largest hunter of its time with the same ecosystem home to giants as big as seven meters long," explains lead author Dr. Rodrigo Temp Müller, who led the paleontology team at the Universidade Federal de Santa Maria in Brazil. "Pseudosuchia were a diverse group of animals capable of tackling robust prey, as well as small hunters specialized in catching swift animals. Analysis of the bones revealed details about the animal's behavior and confirmed that Tainrakuasuchus bellator was protected by bony plates called osteoderms, which are also found in today's crocodiles. Although its limbs were not preserved, the researchers believe it moved on all fours, similar to related species. "Despite the diversity of pseudosuchians, they remain poorly understood, as fossils of some their lineages are extremely rare in the fossil record," Dr. Müller says. "Once the anatomical details were revealed, we were delighted and really excited to reveal that the specimen represented a species previously unknown to science. "This connection between animals from South America and Africa can be understood in light of the Triassic Period's paleogeography. "At that time, the continents were still united, which allowed the free dispersal of organisms across regions that are now separated by oceans. "Tainrakuasuchus bellator would have lived in a region bordering a vast, arid desert -- the same setting as where the first dinosaurs emerged. "It shows that, in what is now southern Brazil, reptiles had already formed diverse communities adapted to various survival strategies. Moreover, this discovery reveals that such diversity was not an isolated phenomenon." Note: Content may be edited for style and length. After Decades, Scientists Have Finally Discovered Tylenol's Secret Mechanism Scientists Discover a Diet That Burns Fat Like Cold Exposure, Leading to Significant Weight Loss From Plastic to Pure Water: Scientists Turn Trash Into a Super Catalyst Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            