
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-28</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250528150829.htm'>Mid-air transformation helps flying, rolling robot to transition smoothly</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 19:46:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Specialized robots that can both fly and drive typically touch down on land before attempting to transform and drive away. But when the landing terrain is rough, these robots sometimes get stuck and are unable to continue operating. Now a team of Caltech engineers has developed a real-life Transformer that has the "brains" to morph in midair, allowing the dronelike robot to smoothly roll away and begin its ground operations without pause. The whole transformation relies on a single motor to move a central joint that lifts ATMO's thrusters up into drone mode or down into drive mode. "We designed and built a new robotic system that is inspired by nature -- by the way that animals can use their bodies in different ways to achieve different types of locomotion," says Ioannis Mandralis (MS '22), a graduate student in aerospace at Caltech and lead author of the new paper. Complex aerodynamic forces come into play both because the robot is close to the ground and because it is changing its shape as it morphs. "Even though it seems simple when you watch a bird land and then run, in reality this is a problem that the aerospace industry has been struggling to deal with for probably more than 50 years," says Mory Gharib (PhD '83), the Hans W. Liepmann Professor of Aeronautics and Medical Engineering, director and Booth-Kresa Leadership Chair of Caltech's Center for Autonomous Systems and Technologies (CAST), and director of the Graduate Aerospace Laboratories of the California Institute of Technology (GALCIT). All flying vehicles experience complicated forces close to the ground. As it comes in for a landing, its thrusters push lots of air downward. When that air hits the ground, some portion of it bounces back up; if the helicopter comes in too quickly, it can get sucked into a vortex formed by that reflected air, causing the vehicle to lose its lift. In ATMO's case, the level of difficulty is even greater. Not only does the robot have to contend with complex near-ground forces, but it also has four jets that are constantly altering the extent to which they are shooting toward each other, creating additional turbulence and instability. To better understand these complex aerodynamic forces, the researchers ran tests in CAST's drone lab. They used what are called load cell experiments to see how changing the robot's configuration as it came in for landing affected its thrust force. The system uses an advanced control method called model predictive control, which works by continuously predicting how the system will behave in the near future and adjusting its actions to stay on course. Here we introduce a dynamic system that hasn't been studied before. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250528150650.htm'>Huge sea-urchin populations are overwhelming Hawaii's coral reefs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 19:46:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>"Fishing in these areas has greatly reduced the number of fishes that feed on these urchins, and so urchin populations have grown significantly," van Woesik said. "We are seeing areas where you have about 51 urchins per square meter, which is among the highest population density for sea urchins anywhere in the world." Those urchins eat the reef, which is already not growing at a healthy rate, van Woesik said. Reef growth is generally measured in terms of net carbonate production, which refers to the amount of calcium carbonate produced in a square meter over a year. Prior research in the 1980s found areas in Hawaii with carbonate production around 15 kilograms per square meter, which would signal a healthy, growing reef, van Woesik said. The reef in Hōnaunau Bay today, however, showed an average net carbonate production of only 0.5 kg per square meter, indicating that the reef is growing very slowly. By combining data gathered through on-site scuba diving with images taken from the air, van Woesik determined that the reef would need to maintain an average of 26% coral cover to break even with the pace of urchin erosion, and a higher cover in order to grow. The average coral cover across all depths was 28%, she said, but areas in shallow depths with more erosion would still need nearly 40% cover to break even. For the islands they surround, coral reefs like those in Hōnaunau Bay provide important coastal protection against erosion from waves, absorbing up to 97% of incoming wave energy. "Without action taken now, we risk allowing these reefs to erode past the point of no return." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64891100/floating-magnet-dark-matter/'>A Magnet Floating in a Superconductive Chamber Could Change Physics Forever</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 19:15:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>What we think of as ‘the unknown' isn't always some hypothetical wormhole or alternate dimension. A lot of times, the ‘unknown' is something real, but whose existence is impossible to prove even with the most advanced technology. We're talking about dark matter, which remains infamously elusive. From huge, hypersensitive underground detectors to the search for bizarre signatures in comic rays, it seems we have tried everything within our current capacity to directly observe even one particle of dark matter. But we do know a few things about this mystery matter—namely, that it exerts gravity, and therefore (supposedly) has mass. When gravitational forces exerted by bodies in space are beyond what is expected, dark matter is the explanation (but never the evidence). Maybe, however, dark matter could make its presence known another way. When cooled enough to transition to a state in which they can conduct electricity without resistance, superconductors expel magnetic fields and therefore repel magnets. Tunnell and his research team predicted that dark matter could be detected this way because of its quantum nature, meaning that it is thought to behave as both a particle and a wave. If any dark matter came close to the levitating magnet—whether it behaved like a particle meandering around or a wave flowing through—the force of gravity it exerted should give the magnet an almost negligible shake. A quantum device known as a SQUID (Superconducting Quantum Interference Device) was used to detect any shifting of magnetic fields that would happen if gravity from an unseen source interacted with the magnet. Spoiler alert: dark matter has not been detected with this method so far. Tunnell plans to update the experiment and optimize it specifically for detecting dark matter instead of gravity. This new proposed experiment will be named POLONAISE, after a Polish dance Tunnell and a colleague were doing to keep warm at an outdoor climate protest. “We hope that it fuels initiatives in advancing experimental designs of magnetically levitated setups for astroparticle physics.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. An Elusive Rat Finally Showed Its Face to Science</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-youtube-star-derek-muller-of-veritasium-is-challenging-scientific/'>YouTube Science Star Derek Muller Confronts PFAS “Forever Chemicals”—In His Own Blood</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 17:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>YouTube Science Star Derek Muller Confronts PFAS “Forever Chemicals”—In His Own Blood Veritasium YouTube star Derek Muller discovered PFAS forever chemicals in his blood. His audience has grown because of how he challenges common misconceptions about the laws of nature, and his rise has paralleled YouTube's, whose creator-focused approach has made it the world's largest streaming service by almost every measure—from content library to audience size—recently surpassing Netflix in revenue. Amid this symbiosis, Veritasium evolved—from unpolished five-minute clips of Muller quizzing people on the street to technically sophisticated science explorations with the satisfying depth of feature films. Muller himself has appeared in many roles: trickster science teacher, experimenter with a mad-scientist flair, fearless investigator in the mold of Indiana Jones. And in Veritasium's most recent video, one of the channel's longest yet, he steps forward as an advocate for one of today's most pressing health issues: the toxic “forever chemicals” that are now being found to contaminate almost every human being, animal and ecosystem on Earth. Entitled How One Company Secretly Poisoned the Planet, the video clocks in at nearly an hour and garnered almost three million views within a day. It is not only an exposé of corporate malfeasance but also a scientific explainer of these forever chemicals (known as perfluoroalkyl and polyfluoroalkyl substances, or PFAS) and their harm to humans. Leveraging his audience, resources and cinematic craft, Muller has now focused Veritasium on a crisis whose invisible poisons, as the film's finale reveals, are already circulating in his own blood. “There was a certain way in which science appeared like magic,” he recalls—invoking science-fiction giant Arthur C. Clarke's adage that any sufficiently advanced technology is indistinguishable from sorcery. He remembers ordinary school experiments—the upside-down glass that keeps a tissue dry underwater and the soda-and-Mentos fountain. “Isn't it everything?” he says when asked why science matters. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. When Muller graduated, he was torn between pursuing filmmaking and science. “I felt like if I went into science, maybe I would be a swappable cog in some sort of machine.” But he also feared that pursuing film might have him running coffees in Los Angeles in hopes of making the right connection and getting a break. He took a full scholarship in engineering physics at Queen's University in Ontario and then decamped to Sydney, Australia, to pursue film—only to learn that the Australian Film, Television and Radio School admits just a handful of students per program each year. In classes, he noticed that students often nodded appreciatively but retained almost nothing. “If you perform a demonstration and don't force the class to make a prediction,” he says, “they'll learn about as much as if they never saw the demonstration at all.” Muller's doctoral thesis compared two instructional video styles, which were viewed by undergraduates at the University of Sydney. The first featured an actor delivering a polished, textbook explanation of Newton's laws. By contrast, those who watched the “confusing” video improved significantly. “Some level of discomfort seems to be essential,” Muller says. Derek Muller attends the 2020 Breakthrough Prize Ceremony at NASA Ames Research Center on November 3, 2019 in Mountain View, California. After Muller received his Ph.D., he says, he took a stable job with a tutoring company in Sydney only to realize that “so much of my life had been about back-up plans.” As he looked for a way to merge his passions, his attention fell on YouTube. Whereas success in the film industry often depended on luck and connections, YouTube seemed more meritocratic to Muller. He kept tutoring 15 hours per week to pay his bills and started the channel he named Veritasium, which combines veritas, Latin for “truth,” with the “ium” suffix of an element to mean “an element of truth.” Muller uploaded his first clip under the channel name Veritasium in early 2011. He would ask passersby why the sky was blue or whether a heavy object would fall faster than a light one and then watch their confidence crumble as he kept questioning them. In one video, he asked people what they thought would happen to an extended Slinky when it was dropped from a roof. When his slow-motion footage revealed that the bottom end of the released Slinky merely hung suspended until a compression wave arrived from above, viewers felt their intuition buckle. Nigel Kuan, a high school physics teacher, who met Muller while studying at the University of Sydney and later collaborated on several Veritasium videos, says Muller's emphasis on using people's misconceptions to teach science has influenced his own approach. “I actually recommend the videos to newer teachers,” he says. One in particular, called Why Do You Make People Look Stupid?, stands out to Kuan as best representing Muller's pedagogic style. What's up?” Across from him also sits Muller, wearing a white YouTube T-shirt and the type of reflective aviator glasses favored by law enforcement. “YouTube Muller” accuses Muller of making people look stupid and shows clips in which the latter watches as individuals fumble through incorrect scientific explanations. YouTube Muller then quotes viewer comments—many laden with expletives—that called Muller “condescending” and “pretentious” and asks, “Why does your face light up with glee every time you hear a misconception?” This video, it turns out, is no different from those in which Muller invites people to explain what water is or where trees get their mass. But now the fallacy to be disproved is not about science but about him: that, in his own words, he “delights in humiliating other people.” Muller, of course, returns to his core idea: people learn better if they confront their own misconceptions. In mid-2011, when Muller was getting his channel off the ground, YouTube was already processing 48 hours of new videos every minute and logging about three billion daily views. Soon it began organizing meetups and creator spaces in major cities and promoting thousands of fledgling teaching channels—from crash-course chemistry to Muller's experiment-driven physics. YouTube's 2012 decision to reward watch-time minutes over mere clicks also encouraged Muller to develop longer videos in which he layered demonstrations, expert cameos, and slow-burn narrative reveals, unspooling each episode like a mystery novel. As for YouTube, today more than 500 hours pour onto the platform each minute. And as of October 2024, its combined ad-and-subscription income had topped $50 billion in the past four quarters—comfortably ahead of any other individual streaming platform, including those owned by Amazon and Disney. Casper Mebius, who joined Veritasiumas an intern in 2023 and now writes, directs and produces episodes, recalls his first meeting with Muller, who, “instead of interrupting me or cutting me short, just literally let me talk through the whole thing for 40 minutes.” Mebius soon learned this wasn't unusual. Muller often patiently listens through meetings with his team. In this way, Muller is still teaching—letting others lay out their predictions and take the risks necessary for learning. But at times, his emphasis on the pedagogical value of risk-taking goes further, as when he swam in shade balls (softball-size plastic spheres that float on reservoirs to block sunlight) after a manufacturer warned him not to, explored Fukushima's radioactive zone and had a bucket of pennies dumped on him from a helicopter to prove that they couldn't kill him. Fifteen years after its inception, Veritasium—now one of the world's most popular science channels, with 18 million subscribers and more than 3.3 billion views—has grown from a one-man passion project to a team of nearly 20 people supported by investments from Electrify Video Partners, a company that helps digital creators scale their businesses. How One Company Secretly Poisoned the Planet is perhaps Veritasium's boldest attempt yet to do so. In the video, as Muller and Čavlović discuss the many ways that PFAS have infiltrated our lives—such as via nonstick cookware, waterproof clothes and stain-resistant carpets—the story becomes increasingly unsettling. It follows West Virgina farmer Wilbur Earl Tennant, who watched more than 150 of his cows slowly die, and environmental lawyer Robert Bilott, who read through 60,000 corporate documents to reveal the risks of contamination. It describes how PFAS have contaminated not just household well water but also snow in the arctic and rain on the Tibetan Plateau. And it culminates with Muller testing his own blood for the forever chemicals. His combined total places him close to the limit at which U.S. science and health advisory groups recommend screenings for PFAS-related diseases. Based on the contamination levels in many U.S. water sources, Čavlović and Muller deduce that the latter could have reached those blood levels merely by drinking tap water from various sources in and around Los Angeles. This is usually the moment in Veritasium videos where Muller shows his delight in solving a mystery, but this time the look on his face is one of dismay. “An important point for the video was not just to inform people about how potentially harmful these chemicals are but also to tell them what academic and governmental institutions are doing to help regulate this,” Čavlović says. In an online sphere crowded with science content, Veritasium has consistently excelled for years by not dumbing down its videos to chase clicks. And Muller, as its host, is always quick to share his personal fallibility. When a concept confuses or surprises him, he lets the audience know, and these viewers become his partners on the path to discovery. When his son asked why a rainbow is an arc, Muller felt the old itch to replace quick answers with deeper voyages. After all, Veritasium itself was born when Muller decided to stop playing it safe. Deni Ellis Béchard is Scientific American's senior tech reporter. You can follow him on X, Instagram and Bluesky @denibechard.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09179-5'>Protein-primed homopolymer synthesis by an antiviral reverse transcriptase</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 15:49:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Please note there may be errors present which affect the content, and all legal disclaimers apply. Bacteria defend themselves from viral predation using diverse immune systems, many of which target foreign DNA for degradation1. Defense-associated reverse transcriptase (DRT) systems provide an intriguing counterpoint to this strategy by leveraging DNA synthesis instead2,3. It remains unknown whether similar mechanisms of defense are employed by other related DRT families. Focusing on DRT9, here we uncover an unprecedented mechanism of DNA homopolymer synthesis. Cryo-EM structures reveal how a noncoding RNA serves as both a structural scaffold and reverse transcription template to direct hexameric complex assembly and poly-dA synthesis. Remarkably, biochemical and functional experiments identify tyrosine residues within the reverse transcriptase itself that likely prime DNA synthesis, leading to the formation of high-molecular weight protein-DNA covalent adducts. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout These authors contributed equally: Stephen Tang, Rimantė Žedaveinytė, Nathaniel Burman, Shishir Pandey Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY, USA Stephen Tang, Rimantė Žedaveinytė, Josephine L. Ramirez, Louie M. Kulber, Tanner Wiegand, Dennis J. Zhang, George D. Lampe & Samuel H. Sternberg Department of Microbiology and Cell Biology, Montana State University, Bozeman, MT, USA Metabolomics Core, Icahn School of Medicine at Mount Sinai, New York, NY, USA Supplementary Figure 1 | Uncropped images of electrophoretic separation assays and LB-agar culture plates. Supplementary Figure 2 | Comparison of domain composition and 3D structure across evolutionarily diverse RT homologs. Supplementary Table 5 | Description and sequence of plasmids used in this study. Tang, S., Žedaveinytė, R., Burman, N. et al. Protein-primed homopolymer synthesis by an antiviral reverse transcriptase. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09050-7'>The subfornical organ is a nucleus for gut-derived T cells that regulate behaviour</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 15:45:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Specialized immune cells that reside in tissues orchestrate diverse biological functions by communicating with parenchymal cells1. The contribution of the innate immune compartment in the meninges and the central nervous system (CNS) is well-characterized; however, whether cells of the adaptive immune system reside in the brain and are involved in maintaining homeostasis is unclear2,3,4. Here we show that the subfornical organ (SFO) of the brain is a nucleus for parenchymal αβ T cells in the steady-state brain in both mice and humans. Using unbiased transcriptomics, we show that these extravascular T cells in the brain are distinct from meningeal T cells: they secrete IFNγ robustly and express tissue-residence proteins such as CXCR6, which are required for their retention in the brain and for normal adaptive behaviour. These T cells are primed in the periphery by the microbiome, and traffic from the white adipose and gastrointestinal tissues to the brain. Once established, their numbers can be modulated by alterations to either the gut microbiota or the composition of adipose tissue. In summary, we find that CD4 T cells reside in the brain at steady state and are anatomically concentrated in the SFO in mice and humans; that they are transcriptionally and functionally distinct from meningeal T cells; and that they secrete IFNγ to maintain CNS homeostasis through homeostatic fat–brain and gut–brain axes. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available on request from the corresponding authors. Sequence files and metadata for all samples used in this study have been deposited in the NCBI Sequence Read Archive (SRA). Single-cell RNA-seq data generated from T cells isolated from steady-state adult mouse brain, meninges, white fat, colon lamina propria, lung and blood (referred to in Figs. 1–3) are available in the NCBI SRA under PRJNA1240821. Single-cell RNA-seq data generated from T cells isolated from steady-state adult mouse brain, meninges, white fat, colon lamina propria, mLNs and blood (referred to in Extended Data Fig. 5) are available in the NCBI SRA under PRJNA1074276. Bulk RNA-seq data generated from CD4 T cells isolated by FACS from steady-state adult mouse brain, meninges and white fat (referred to in Extended Data Fig. 5) are available in the NCBI SRA under PRJNA1073984. Bulk RNA-seq data generated from CD4 T cells isolated from steady-state adult mouse brain, lungs and lymph nodes (referred to in Extended Data Fig. 5) are available in the NCBI SRA under PRJNA1074381. Single-cell RNA-seq data generated from healthy human adult CSF, colon, duodenum and blood (referred to in Fig. 8) are available in the NCBI SRA under PRJNA1074056. Bulk RNA-seq data generated from microglia isolated from steady-state adult wild-type and Tbx21 KO mouse brain (referred to in Extended Data Fig. 10) are available in the NCBI SRA under PRJNA1073982. Integrated control of leukocyte compartments as a feature of adaptive physiology. Pasciuto, E. et al. Microglia require CD4 T cells to complete the fetal-to-adult transition. Pappalardo, J. L. et al. Transcriptomic and clonal characterization of T cells in the human central nervous system. Smolders, J. et al. Tissue-resident memory T cells populate the human brain. Schnell, A. et al. Stem-like intestinal Th17 cells give rise to pathogenic effector T cells during autoimmunity. Berer, K. et al. Commensal microbiota and myelin autoantigen cooperate to trigger autoimmune demyelination. Hsiao, E. Y. et al. Microbiota modulate behavioral and physiological abnormalities associated with neurodevelopmental disorders. Seki, D. et al. Aberrant gut–microbiota–immune–brain axis development in premature neonates with brain damage. Kim, S. et al. Maternal gut bacteria promote neurodevelopmental abnormalities in mouse offspring. Gut microbiota regulate motor deficits and neuroinflammation in a model of Parkinson's disease. Immunity to homologous grafted skin; the fate of skin homografts transplanted to the brain, to subcutaneous tissue, and to the anterior chamber of the eye. Carson, M. J., Doose, J. M., Melchior, B., Schmid, C. D. & Ploix, C. C. CNS immune privilege: hiding in plain sight. Brain antigen-reactive CD4+T cells are sufficient to support learning behavior in mice with limited T cell repertoire. Rattazzi, L. et al. CD4+ but not CD8+T cells revert the impaired emotional behavior of immunocompromised RAG-1-deficient mice. Derecki, N. C. et al. Regulation of learning and memory by meningeal immunity: a key role for IL-4. Unexpected role of interferon-γ in regulating neuronal connectivity and social behaviour. Ribeiro, M. et al. Meningeal γδ T cell-derived IL-17 controls synaptic plasticity and short-term memory. Alves de Lima, K. et al. Meningeal γδ T cells regulate anxiety-like behavior via IL-17a signaling in neurons. Hindmarch, C. C. & Ferguson, A. V. Physiological roles for the subfornical organ: a dynamic transcriptome shaped by autonomic state. Pulman, K. J., Fry, W. M., Cottrell, G. T. & Ferguson, A. V. The subfornical organ: a central target for circulating feeding signals. & Pittman, Q. J. Circumventricular organs and fever. Projections from subfornical organ to bed nucleus of the stria terminalis modulate inflammation-induced anxiety-like behaviors in mice. Omri, B. et al. CD4 expression in neurons of the central nervous system. The bone marrow protects and optimizes immunological memory during dietary restriction. Zheng, D., Liwinski, T. & Elinav, E. Interaction between microbiota and immunity in health and disease. Wang, B. et al. A high-fat diet increases gut microbiota biodiversity and energy expenditure due to nutrient difference. Al Nabhani, Z. et al. A weaning reaction to microbiota is required for resistance to immunopathologies in the adult. Induction of intestinal Th17 cells by segmented filamentous bacteria. Morton, A. M. et al. Endoscopic photoconversion reveals unexpectedly broad leukocyte trafficking to and from the gut. Galván-Peña, S., Zhu, Y., Hanna, B. S., Mathis, D. & Benoist, C. A dynamic atlas of immunocyte migration from the gut. Odoardi, F. et al. T cells become licensed in the lung to enter the central nervous system. & Hen, R. in Mood and Anxiety Related Phenotypes in Mice: Characterization Using Behavioral Tests Vol. Kunis, G. et al. IFN-γ-dependent activation of the brain's choroid plexus for CNS immune surveillance and repair. Blocking the IFN-gamma signal in the choroid plexus confers resistance to experimental autoimmune encephalomyelitis. Szabo, S. J. et al. A novel transcription factor, T-bet, directs Th1 lineage commitment. Distinct effects of T-bet in TH1 lineage commitment and IFN-γ production in CD4 and CD8 T cells. Wang, Y. et al. TREM2-mediated early microglial response limits diffusion and toxicity of amyloid plaques. Lee, S. et al. APOE modulates microglial immunometabolism in response to age, amyloid pathology, and inflammatory challenge. Lituma, P. J. et al. Altered synaptic connectivity and brain function in mice lacking microglial adapter protein Iba1. Hong, S. et al. Complement and microglia mediate early synapse loss in Alzheimer mouse models. The novel role of PPAR alpha in the brain: promising target in therapy of Alzheimer's disease and other neurodegenerative disorders. Prolonged STAT1 activation in neurons drives a pathological transcriptional response. Prolonged STAT1 signaling in neurons causes hyperactive behavior. Unique aspects of IFN-γ/STAT1 signaling in neurons. Smith, B. C. et al. Astrocyte interferon-gamma signaling dampens inflammation during chronic central nervous system autoimmunity via PD-L1. Hindinger, C. et al. IFN-γ signaling to astrocytes protects from autoimmune mediated neurological disability. Clark, I. C. et al. Barcoded viral tracing of single-cell interactions in central nervous system inflammation. Kantzer, C. G. et al. Anti-ACSA-2 defines a novel monoclonal antibody for prospective isolation of living neonatal and adult astrocytes. Giedd, J. N., Castellanos, F. X., Rajapakse, J. C., Vaituzis, A. C. & Rapoport, J. L. Sexual dimorphism of the developing human brain. Sex differences in developmental patterns of neocortical astroglia: a mouse translatome database. Barr, M. M., García, L. R. & Portman, D. S. Sexual dimorphism and sex differences in Caenorhabditis elegans neuronal development and behavior. The primate lentiviral receptor Bonzo/STRL33 is coordinately regulated with CCR5 and its expression pattern is conserved between human and mouse. Oxenius, A., Bachmann, M. F., Zinkernagel, R. M. & Hengartner, H. Virus-specific MHC-class II-restricted TCR-transgenic mice: effects on humoral and cellular immune responses after viral infection. Focused specificity of intestinal TH17 cells towards commensal bacterial antigens. Lodolce, J. P. et al. IL-15 receptor maintains lymphoid homeostasis by supporting lymphocyte homing and proliferation. Dufour, J. H. et al. IFN-γ-inducible protein 10 (IP-10; CXCL10)-deficient mice reveal a role for IP-10 in effector T cell generation and trafficking. Griswold, S. L., Sajja, K. C., Jang, C. W. & Behringer, R. R. Generation and characterization of iUBC-KikGR photoconvertible transgenic mice for live time-lapse imaging during development. Mombaerts, P. et al. Mutations in T-cell antigen receptor genes α and β block thymocyte development at different stages. Multiple defects of immune cell function in mice with disrupted interferon-γ genes. Finotto, S. et al. Development of spontaneous airway changes consistent with human asthma in mice lacking T-bet. Haas, J. D. et al. Development of interleukin-17-producing γδ T cells is restricted to a functional embryonic wave. Lee, S. H. et al. Identifying the initiating events of anti-Listeria responses using mice with conditional loss of IFN-γ receptor subunit 1 (IFNGR1). Fate mapping reveals origins and dynamics of monocytes and tissue macrophages under homeostasis. Gregorian, C. et al. Pten deletion in adult neural stem/progenitor cells enhances constitutive neurogenesis. Tronche, F. et al. Disruption of the glucocorticoid receptor gene in the nervous system results in reduced anxiety. Park, Y. G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. Borcherding, N., Bormann, N. L. & Kraus, G. scRepertoire: an R-based toolkit for single-cell immune receptor analysis. Street, K. et al. Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics. Van den Berge, K. et al. Trajectory-based differential expression analysis for single-cell sequencing data. & Yin, F. Characterizing brain metabolic function ex vivo with acute mouse slice punches. Pennington, Z. T. et al. ezTrack: an open-source video analysis pipeline for the investigation of animal behavior. Favuzzi, E. et al. GABA-receptive microglia selectively sculpt developing inhibitory circuits. We thank the Yale Legacy Tissue Donation Program and Center for Human Brain Discovery for the human autopsy tissue samples; D. Wolfsohn for performing the colonoscopy for our human study collection; D. Littman for sharing 7B8 mice and an aliquot of FITC rat anti-mouse Vβ 14 T cell receptor antibody; E. Schaubeck and others from LifeCanvas Technologies for assistance with brain-clearing and image acquisition; E. Menet, B. Shonts, A. Khalil, J. Bonin and D. Trotta for cell sorting through the Yale Flow Cytometry Core Facility; I. Klenovskiy for helping with Python-coding-related tasks; the Yale Center for Cellular and Molecular Imaging Confocal Facility for use of the Lecia Stellaris microscope; the Yale Flow Cytometry facility for assistance with cell sorting and instrument maintenance; and the YCGA for assistance with library preparation and genomic sequencing services. The Flow Cytometry Core is supported in part by an NCI Cancer Center Support Grant NIH P30 CA016359; the BD Symphony was funded by shared instrument grant NIH S10 OD026996; and YCGA-related research reported in this publication was supported by the National Institute of General Medical Sciences of the National Institutes of Health (NIH) under award number 1S10OD030363-01A1. was supported by NIH grants R01 AI162645 and R01 AR080104, the Smith Family Foundation, the Colton Center for Autoimmunity at Yale, the Food and Allergy Science Initiative, the PEW Charitable Trusts, the Chan Zuckerberg Initiative, and gifts from the Mathers Family Foundation, the Ludwig Family Foundation and the Knights of Columbus; D.A.H. was supported by NIH grants P01 AI073748, R01 AI22220, UM 1HG009390, P01 AI039671, P50 CA121974 and R01 CA227473, Race to Erase MS and the National MS Society; T.M.Y. was supported by the Smith Family Foundation and NIH grant 1F31NS130957-01A1; J.L. was supported by NIH grants T32MH019961 and R25MH071584. was supported by NIH grants T32 CA233414 and K12 CA215110; Y.S.M. was supported by NIH grant R37 AR40072. We acknowledge BioRender for their platform to generate schematics. Department of Immunobiology, Yale School of Medicine, New Haven, CT, USA Tomomi M. Yoshida, Mytien Nguyen, Benjamin Y. Lu, Di Xu, Elizabeth Lin, Daniel A. Waizman, Mackenzie E. Coden, Yifan Ma, Kavita Israni-Winger, Wenzhi Song, Jafar Al Souz, Joseph E. Craft, Noah W. Palm, David A. Hafler & Andrew Wang Department of Rheumatology, Allergy and Immunology, Yale School of Medicine, New Haven, CT, USA Tomomi M. Yoshida, Cuiling Zhang, Di Xu, Elizabeth Lin, Daniel A. Waizman, Mackenzie E. Coden, Yifan Ma, Kavita Israni-Winger, Wenzhi Song, Jafar Al Souz, Joseph E. Craft & Andrew Wang Department of Neurology, Yale School of Medicine, New Haven, CT, USA Tomomi M. Yoshida, Le Zhang, Benjamin Y. Lu, Katie N. Murray, Elizabeth Lin, Anthony Russo, Haowei Wang, Jaime Grutzendler & David A. Hafler Department of Neuroscience, Yale School of Medicine, New Haven, CT, USA Department of Medicine (Section of Medical Oncology and Hematology), Yale School of Medicine, New Haven, CT, USA Program of Computational Biology and Bioinformatics, Yale University, New Haven, CT, USA Department of Psychiatry, Yale School of Medicine, New Haven, CT, USA Yann S. Mineur, Joseph Luchsinger & Marina R. Picciotto Department of Pathology, Yale School of Medicine, New Haven, CT, USA Department of Ophthalmology and Visual Science, Yale School of Medicine, New Haven, CT, USA Department of Biostatistics, School of Public Health, Yale University, New Haven, CT, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to David A. Hafler or Andrew Wang. has received research funding from Bristol Myers Squibb, Novartis, Sanofi and Genentech, and has been a consultant for Bayer Pharmaceuticals, Repertoire, Bristol Myers Squibb, Compass Therapeutics, EMD Serono, Genentech, Juno Therapeutics, Novartis, Proclara Biosciences, Sage Therapeutics and Sanofi Genzyme. has received research funding from NGM Biopharmaceuticals and AstraZeneca; has been a consultant for Seranova Bio, the Column Group and Soleil Labs; and is on the scientific advisory board for NGM Biopharmaceuticals. Nature thanks Dan Littman, Scott Mueller and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Coronal brain sections from 8–10-week-old C57BL/6 mouse stained with CD31 for blood vessels, CD4 and CD3. b, Representative image of extravascular CD4 T cells from whole-mount dura meninges peeled from skullcaps. c, Representative image of vascular adjacent CD4 T cell in the brain in the white matter, specifically the stria terminalis as indicated by an arrow. Quantification of vascular adjacent CD3+ CD4+ T cells by brain region. (n = 3 per brain area; pooled experiments from 8 female and 7 male mice). d, Representative image of intravascular CD4 T cells in the brain in the choroid plexus. e, Quantification of intravascular CD3+ CD4+ T cells by brain region. (n = 3 per brain area; pooled experiments from 8 female and 7 male mice). f, Brain from a CD4–tdTomato fate-map reporter mouse (Ai14 CD4-Cre) was processed for whole-brain clearing, imaged using a light-sheet microscope, and registered to the Allen Mouse Brain Atlas. Analysis of tdTomato expression by brain area was calculated by LifeCanvas. R was used to generate the graph displaying the average density of tdTomato+ cells by mm3 across all registered brain areas. g, Representative image of the coronal slice containing the SFO h, IGL and SubG i, and AON from the cleared whole brain generated using Imaris. j, Same as in f, except for all CVOs of the brain. (n = 1) k, Deidentified post-mortem human brain samples were obtained from individuals without acute neurological disease, structures of interest were dissected by a neuropathologist and submitted for histology. a, Intravascular labelling strategy of circulating leukocytes via retro-orbital (R.O.) injection of a fluorophore-conjugated CD45 antibody. Representative FACS strategy to isolate extravascular tissue CD4 T cells from the mouse brain. Gating strategy was kept the same across all organs used for the experiment. b, Quantification of absolute numbers of extravascular CD4 αβ T cells in the brain with different incubation times of CD45 intravascular labelling. c, Representative FACS strategy of CD45 IV-negative extravascular CD4 T cells in the mouse brain. d, Representative FACS expression of T-bet in the steady-state brain, lung, liver and inguinal lymph node. e, FACS analysis of brain T cell population in Il15ra KO mice. Grubbs' outlier test (P < 0.05) was used to determine outliers, which were excluded. f, IFNγ reporter with endogenous polyA transcript” (GREAT) mice were used to look at IFNγ-YFP expression in various immune cells in steady-state tissue. g, Comparison of IFNγ-YFP expression in CD4 T cells (CD45 IV-, CD11b-, CD3+, CD4+, TCRβ+), CD8 T cells (CD45 IV-, CD11b-, CD3+, CD8+, TCRβ+), NK cells (CD45 IV-, CD11b-, CD3-, NK1.1+), γδ T cells (CD45 IV-, CD11b-, CD3+, TCRγδ+), and T and NK cell negative lymphocytes (CD45 IV-, CD11b-, CD3-, NK1.1-). h, Comparison of IFNγ-YFP expression across various immune cells subsets in the brain. i, Quantification of IFNγ-YFP+ cells in CD4 T cells, CD8 T cells and NK cells in the brain with or without 4-hour cell stimulation (PMA–ionomycin with Brefeldin A). a, Single-cell suspensions isolated from brain, meninges, white fat and spleen were stimulated with PMA–ionomycin and incubated with Brefeldin A at 37 °C for four hours. Representative flow-cytometric plots of IFNγ and IL-17A double producing cells present in the brain at steady-state. b, Representative flow-cytometric expression of IL-17A from brain, meninges, white fat and spleen. Single-cell suspensions isolated from brain, meninges, white fat, colon, brown fat, liver, lung, ovaries, spleen, and lymph nodes (mesenteric (m)LN, inguinal (i)LN and deep cervical (dc)LN) were stimulated with PMA–ionomycin and incubated with GolgiPlug at 37 °C for four hours. ****P < 0.0001 one-way ANOVA with Tukey's multiple comparison's test (n = 4 per group pooled from 3 experiments). c, Representative flow-cytometric expression of transcription factor FOXP3 and cytokine IL-10 from steady-state brain, meninges, white fat and spleen. d, Representative flow-cytometric expression of transcription factor GATA3 and cytokine IL-4 staining in the steady-state brain, spleen and lung. e, Representative flow-cytometric expression of co-inhibitory receptors PD-1, TIGIT and LAG3 in the steady-state brain, spleen and lung. a, Flow-cytometric analysis was performed on single-cell suspensions isolated from fresh brain samples (n = 2) obtained 4–5 h post-mortem. Representative flow gating strategy to isolate CD4 and CD8 T cells from the human autopsy samples; the CSF data was used as the example. Gating strategy was kept the same across all organs used for the experiment. b, The frequency of CXCR3 of CXCR6-expressing CD8 T cells from patient 6 (Table 1) was graphed. The density of CD4 and CD8 T cells in the brain were generated from absolute counts using counting beads divided by weight of the original tissue sample. (n = 1) c, The frequency and density of CXCR3 or CXCR6-expressing CD4 T cells from patient 7 (Table 1) were graphed. The density of CD4 and CD8 T cells in the brain were generated from absolute counts using counting beads divided by weight of the original tissue sample. (n = 1) d, The frequency and density of CXCR3 or CXCR6-expressing CD4 T cells from patient 7 (Table 1) was graphed. The density of CD4 and CD8 T cells in the brain were generated from absolute counts using counting beads divided by weight of the original tissue sample. a, UMAP generated from single-cell RNA-seq of T cells (FACS: CD45 intravenous label−, CD11b−, CD19−, B220− and RNA: Cd3e > 1) sorted from brain, meninges, gonadal white fat, colon lamina propria, mLN and blood of 5 pooled 10-week-old C57BL/6 mice. Comparison of gene expression across all tissue T cells representing various T cell differentiation states. b, Schematic of bulk RNA-seq experimental setup. c, PCA analysis of the CD4 T cells from brain, fat and meninges. (n = 8 per biological replicate represented as a dot in the PCA). DEGs (cut-off FDR < 0.05) from comparing brain/white fat and brain/meningeal CD4 T cells were used for QIAGEN IPA. Volcano plots represent DEGs (cut-off FDR < 0.05) from comparing brain/white fat, brain/meninges and white fat/meninges. d, PCA analysis of the CD4 T cells from brain, iLN and lung. Volcano plot representing DEGs (cut-off FDR < 0.05) from bulk RNA-seq analysis of brain/LN and brain/lung CD4 T cells. e, DEGs (cut-off FDR < 0.05) from comparing brain/white fat, brain/meningeal and white fat/meningeal CD4 T cells were used for QIAGEN IPA. a, Body fat percentage of mice fed a HFD (n = 3 per group; pooled 2 experiments). b, Tissue CD4 T cell numbers HFD-fed mice. (n = 7 per group; pooled 2 experiments). c, Body fat percentage of mice fasted for 48 h. (n = 9 per group; pooled 2 experiments). d, CD4 T cell numbers in mice fasted for 48 h. (n = 3 per group; brain, white fat, brown fat pooled 2 experiments; repeated twice). e, Body fat percentage changes in mice fed a HFD and then fasted. (n = 3 per group) f, Spleen and bone marrow CD4 T cell numbers as in e. (n = 3 per group; spleen = pooled 2 experiments). g, Representative FACS plots from fat-photoconverted mice. h, Brain IFNγ+ and total CD8 T cell numbers at various ages. (n = 3 per group; repeated once) i, Total CD4 and CD8 T cells and IFNγ-secreting CD8 T cell numbers in brains of various-aged GF and SPF mice. (n = 3 per group; repeated once). j, FACS analysis of brain CD4 T cells post antibiotic (ABX)-treatment. k, Same as in j, except for total CD4 T cells in other tissues. (n = 4 per group; repeated once). l, Representative FACS plots of adult and m, newly weaned mice that underwent gut or fat photoconversion. Proportion of photoconverted T cells quantified. (n = 3 per group; pooled 2 experiments) n, Brain CD4 T cell numbers in weaning-aged mice treated with anti-VLA4 and anti-LFA1. (n = 4 per group) o, Brain CD4 T cell numbers in weaning-aged mice treated with FTY720. Statistical tests: unpaired t-test for a–d,j,k,n,o. Two-way ANOVA with Bonferroni's multiple comparisons test for e,f,i. One-way ANOVA with Tukey's multiple comparisons test for h,m. a, Percentage of unique T cell clones per tissue and line graph representing raw number of unique TCR clones in each tissue from the mouse single-cell TCR-seq dataset. b, Demultiplexed brain sample according to unique Hashtag antibody barcode (TotalSeq-C Hashtag 1–6) labelling each mouse. c, TCR clonal sharing as measured by the Morisita index (n = 5). d, FACS analysis of CD4 T cells in Smarta TCR (Vα2Vβ8) transgenic mouse. *P < 0.01 by unpaired t-test (n = 4 per group, pooled 2 experiments in white fat and mLN; repeated once). e, FACS analysis of Smarta Rag1–/– mice that were immunized using LPS as an adjuvant and co-administered with vehicle or GP61–80 peptide by oral route to prime antigens through the gut. (n = 4 per group) f, Same as in e, with an additional boost of vehicle, LPS or LPS and GP61–80 peptide (n = 3 per group). g, 50,000 naive 7B8 T cells were transferred into gnotobiotic mice administered GF or SFB faecal slurry. FACS analysis of the absolute number and frequency of TH17 7B8 T cells from white fat, mLN, small intestine and spleen 5 weeks later. *P < 0.05, **P < 0.01 by unpaired parametric t-test (n = 5 per group; repeated twice). h, Same as in g, except 50,000 naive 7B8 T cells were transferred into Jackson Laboratory (no SFB) or Taconic Biosciences (have SFB) mice. *P < 0.05 by unpaired parametric t-test *P < 0.05 by unpaired parametric t-test (n = 4 per group). a, Degree of TCR clonal expansion from T cell clones in the human single-cell RNA-seq dataset (CSF, blood 1 and 2, colon, duodenum) was analysed using ScRepertoire. **P < 0.01, ***P < 0.001 and ****P < 0.0001 by one-way ANOVA (n = 5) b, Highlighted the T cells with shared TCR clone between CSF, colon and blood in each tissue's UMAP graph. c, Volcano plot of DEGs after comparing clusters containing the shared TCR clone from each tissue (colon vs blood 2 and colon vs CSF). d, Trajectory analysis on mouse single-cell RNA-seq dataset from Extended Data Fig. 5 to identify genes highly expressed in brain T cells along the Slingshot pseudotime trajectory. e, FACS analysis on CXCR6-deficient mice for IFNγ and IL-17A-secreting CD4 T cells in the brain, white fat, and meninges and spleen. **P < 0.01 and ****P < 0.0001 by one-way ANOVA test (n = 3 per group) f, FACS analysis on IP10 (CXCL10)-deficient mice for total CD4 T cell numbers and proportion of CXCR3+IFNγ+ CD4 T cells in the brain. **P < 0.01 by unpaired t-test (n = 5 per group, outliers were calculated using Grubbs' outlier test and excluded). a, Tcrb KO mice subjected to the LDT, b, OFT, c, TST and two days of FST. (Pooled 3 experiments (males) and 3 experiments (females)), d, Brain CD4 and CD8 T cell numbers in Ifng KO mice. (n = 4 per group pooled; repeated twice). e, Ifng KO mice subjected to LDT, f, OFT g, TST and FST. h, Total and IFNγ+ brain CD4 and CD8 T cell numbers in Tbet KO mice (n = 3 per group; repeated twice). i, subjected to LDT, j, OFT, k, TST and FST (Pooled 3 experiments (males) and 4 experiments (females)). l, β-hydroxybutyrate (BHOB), triglycerides (TG) and non-esterified fatty acids (NEFA) measured in the serum. m, CD4 and CD8 T cell numbers in Il17a–Il17f double-KO mice (n = 3 per group) n, subjected to the NSFT, o, LDT, p, OFT, q, TST and FST (Pooled 2 experiments (males and females), except OFT males). r, Representative FACS plot and bar graph of Tcrb KO mice adoptively transferred with wild-type (CD45.1) or Ifng KO CD4 T cells for 5 weeks. (n = 12 (males), 6 (females) per group; pooled 2 experiments) s, LDT, t, OFT u, TST and FST. v, Same as in r, except with Ifng KO recipient mice. (n = 5 per group) w, LDT, x, NSFT, y, OFT, z TST and FST. Statistical tests: unpaired t-test for d,h,m. Two-way ANOVA with Bonferroni's multiple comparisons test for a–c,e–g,i–l,n–z. Outliers were calculated using the ROUT outlier test (Q = 1%) and excluded. a, Smarta Rag KO or heterozygous (het) mice were subjected to the NSFT, b, LDT, c, OFT, d, TST and two days of FST. (Pooled 3 experiments for males and females). e, Cxcr6 KO mice were subjected to the NSFT, f, LDT, g, OFT, h, TST and FST. (Pooled 3 experiments for males and females). i, Ifngr1f/fCx3cr1-cre mice were subjected to the LDT, j, OFT, k, TST and FST. (Pooled 4 experiments for males and females) l, Bulk RNA-seq of FACS sorted microglia from wild-type and Tbet KO mice. Separate PCA plot by sex. m, Volcano plot of DEGs. DEGs (cut-off FDR < 0.05) were used for QIAGEN IPA. n, Ifngr1f/fNestin-cre mice were subjected to the LDT, o, OFT, p, TST and FST. q, Ifngr1f/fGfap-cre mice were subjected to the LDT, r, OFT, s, TST and FST. t, Single-cell suspensions were isolated from the brains of Ifng KO mice and prepared for FACS to identify microglia and astrocytes. (n = 3 per group; pooled 2 experiments for females) Statistical tests: *P < 0.05, **P < 0.01 and ***P < 0.001 by two-way ANOVA with Bonferroni's multiple comparisons test for a–k,n–t. Outliers were calculated using the ROUT outlier test (Q = 1%) and excluded. Differentially expressed gene list from single-cell RNA-seq of mouse T cells from brain and dura meninges. Legend: p_val= p-value; avg_log2FC= average log2 fold change; pct.1= Proportion of cells in group 1 expressing the gene; pct.2= Proportion of cells in group 1 expressing the gene; p_val_adj= adjusted p-value; cluster= assigned cluster number; gene = gene name. Differentially expressed gene list from single-cell RNA-seq of mouse T cells from brain, meninges, white fat, colon, lung and blood. Legend: p_val= p-value; avg_log2FC= average log2 fold change; pct.1= Proportion of cells in group 1 expressing the gene; pct.2= Proportion of cells in group 1 expressing the gene; p_val_adj= adjusted p-value; cluster= assigned cluster number; gene = gene name. Differentially expressed gene list from single-cell RNA-seq of mouse T cells from brain, meninges, white fat, colon, mesenteric lymph node and blood. Legend: p_val= p-value; avg_log2FC= average log2 fold change; pct.1= Proportion of cells in group 1 expressing the gene; pct.2= Proportion of cells in group 1 expressing the gene; p_val_adj= adjusted p-value; cluster= assigned cluster number; gene = gene name. Differentially expressed gene list from bulk RNA-seq of mouse CD4 T cells from brain vs lung. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name ; mean_BrCD4= mean gene expression of all Brain CD4 T cell samples; mean_LungCD4= mean gene expression of all Brain CD4 T cell samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; BrCD4_1_A= Brain CD4 T cell sample 1 (8 mice pooled); BrCD4_1_B= Technical replicate of Brain sample 1; BrCD4_5_A= Brain sample 5 (8 mice pooled); BrCD4_5_B= Technical replicate of Brain sample 5 (8 mice pooled); BrCD4_7_A= Brain sample 7 (8 mice pooled); BrCD4_7_B= Technical replicate of Brain sample 7 (8 mice pooled); Lung4_1= Lung CD4 T cell sample 1; Lung4_2= Lung CD4 T cell sample 2; Lung4_3= Lung CD4 T cell sample 3; Lung4_4= Lung CD4 T cell sample 4; Lung4_5= Lung CD4 T cell sample 5; Lung4_6= Lung CD4 T cell sample 6; Lung4_7= Lung CD4 T cell sample 7. Differentially expressed gene list from bulk RNA-seq of mouse CD4 T cells from brain vs inguinal lymph node. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name ; mean_BrCD4= mean gene expression of all Brain CD4 T cell samples; mean_LnCD4= mean gene expression of all Lymph node CD4 T cell samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; BrCD4_1_A= Brain CD4 T cell sample 1 (8 mice pooled); BrCD4_1_B= Technical replicate of Brain sample 1; BrCD4_5_A= Brain sample 5 (8 mice pooled); BrCD4_5_B= Technical replicate of Brain sample 5 (8 mice pooled); BrCD4_7_A= Brain sample 7 (8 mice pooled); BrCD4_7_B= Technical replicate of Brain sample 7 (8 mice pooled); Ln4_1= Lymph node CD4 T cell sample 1; Ln4_2= Lymph node CD4 T cell sample 2; Ln4_3= Lymph node CD4 T cell sample 3; Ln4_4= Lymph node CD4 T cell sample 4; Ln4_5= Lymph node CD4 T cell sample 5; Ln4_6= Lymph node CD4 T cell sample 6; Ln4_7= Lymph node CD4 T cell sample 7. Differentially expressed gene list from bulk RNA-seq of mouse CD4 T cells from brain vs white fat. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name; mean_Brain= mean gene expression of all Brain CD4 T cell samples; mean_Fat= mean gene expression of all Fat CD4 T cell samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; X9_11_Brain= Brain CD4 T cell sample 1 (8 mice pooled); X9_12_Brain= Brain sample 2 (8 mice pooled); X9_13_Brain= Brain sample 3 (8 mice pooled); X9_5_Brain= Brain sample 4 (8 mice pooled); X9_11_Fat= White gonadal fat CD4 T cell sample 1 (8 mice pooled); X9_12_Fat= White gonadal fat CD4 T cell sample 2 (8 mice pooled); X9_13_Fat= White gonadal fat CD4 T cell sample 3 (8 mice pooled); X9_5_Fat= White gonadal fat CD4 T cell sample 4 (8 mice pooled). Differentially expressed gene list from bulk RNA-seq of mouse CD4 T cells from brain vs dura meninges. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name; mean_Brain= mean gene expression of all Brain CD4 T cell samples; mean_Meningeal= mean gene expression of all Meningeal CD4 T cell samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; X9_11_Brain= Brain CD4 T cell sample 1 (8 mice pooled); X9_12_Brain= Brain sample 2 (8 mice pooled); X9_13_Brain= Brain sample 3 (8 mice pooled); X9_5_Brain= Brain sample 4 (8 mice pooled); X9_11_M= Meningeal CD4 T cell sample 1 (8 mice pooled); X9_12_M= Meningeal CD4 T cell sample 2 (8 mice pooled); X9_13_M= Meningeal CD4 T cell sample 3 (8 mice pooled); X9_5_M= Meningeal CD4 T cell sample 4 (8 mice pooled). Differentially expressed gene list from bulk RNA-seq of mouse CD4 T cells from white fat vs dura meninges. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name; mean_Fat= mean gene expression of all Fat CD4 T cell samples; mean_Meningeal= mean gene expression of all Meningeal CD4 T cell samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; X9_11_Fat= White gonadal fat CD4 T cell sample 1 (8 mice pooled); X9_12_Fat= White gonadal fat CD4 T cell sample 2 (8 mice pooled); X9_13_Fat= White gonadal fat CD4 T cell sample 3 (8 mice pooled); X9_5_Fat= White gonadal fat CD4 T cell sample 4 (8 mice pooled); X9_11_M= Meningeal CD4 T cell sample 1 (8 mice pooled); X9_12_M= Meningeal CD4 T cell sample 2 (8 mice pooled); X9_13_M= Meningeal CD4 T cell sample 3 (8 mice pooled); X9_5_M= Meningeal CD4 T cell sample 4 (8 mice pooled). Differentially expressed gene list from single-cell RNA-seq of human T cells from CSF vs duodenum. Legend: names= canonical gene names; scores=; pvals= p-value; pvals_adj= adjusted p-value; logfoldchanges= average log2 fold change. Differentially expressed gene list from single-cell RNA-seq of human T cells from CSF vs PBMCs. Legend: names= canonical gene names; scores=; pvals= p-value; pvals_adj= adjusted p-value; logfoldchanges= average log2 fold change. Differentially expressed gene list from bulk RNA-seq of microglia from Tbet KO and wild-type mice. Legend: Genever = Ensemble gene ID with specific gene version; GeneID= Ensemble gene ID; GeneName= canonical gene name; mean_TbetF= mean gene expression of all Tbet KO microglia samples; mean_B6F= mean gene expression of all wild-type microglia samples; log2FoldChange= average log2 fold change; pvalue= p-value; padj= adjusted p-value; Tbet_4= Tbet KO microglia sample 4; Tbet_5= Tbet KO microglia sample 5; Tbet_6= Tbet KO microglia sample 6; B6_4= Wild-type microglia sample 4; B6_5= Wild-type microglia sample 5; B6_6= Wild-type microglia sample 6. z-stack video generated on ImageJ for a representative image of an extravascular T cell in the white matter corpus callosum (Fig. z-stack video generated on ImageJ for a representative image of extravascular T cells in dura meninges (Extended Data Fig. z-stack video generated on ImageJ for a representative image of T cells in the SFO (Fig. 3D-projection video generated on LAS/X software of the SFO (Fig. z-stack video generated on ImageJ for a representative image of vascular adjacent T cells in the white matter stria terminalis (Extended Data Fig. z-stack video generated on ImageJ for a representative image of T cells in the choroid plexus (Extended Data Fig. Video of the whole-brain imaging conducted on Ai14 CD4-Cre mice, in which cells that have previously or currently express CD4 also express tdTomato. The 3D images were rendered using Imaris and iMovie was used to generate the composite video. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Yoshida, T.M., Nguyen, M., Zhang, L. et al. The subfornical organ is a nucleus for gut-derived T cells that regulate behaviour. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09077-w'>Detection of X-ray emission from a bright long-period radio transient</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 15:34:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Recently, a class of long-period radio transients (LPTs) has been discovered, exhibiting emission thousands of times longer than radio pulsars1,2,3,4,5. These findings, enabled by advances in wide-field radio surveys, challenge existing models of rotationally powered pulsars. Proposed models include highly magnetized neutron stars6, white-dwarf pulsars7 and white-dwarf binary systems with low-mass companions8. Although some models predict X-ray emission6,9, no LPTs have been detected in X-rays despite extensive searches1,2,3,4,5,10. Here we report the discovery of an extremely bright LPT (10–20 Jy in radio), ASKAP J1832−0911, which has coincident radio and X-ray emission, both with a 44.2-minute period. Its correlated and highly variable X-ray and radio luminosities, combined with other observational properties, are unlike any known Galactic object. The source could be an old magnetar or an ultra-magnetized white dwarf; however, both interpretations present theoretical challenges. This X-ray detection from an LPT reveals that these objects are more energetic than previously thought and establishes a class of hour-scale periodic X-ray transients with a luminosity of about 1033 erg s−1 linked to exceptionally bright coherent radio emission. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available on Zenodo at https://doi.org/10.5281/zenodo.15228816 (ref. 76) and GitHub at https://github.com/Andywang201605/J1832-0911_radio_xray. All the ASKAP data are publicly available via CASDA (https://data.csiro.au/domain/casdaObservation). The MeerKAT data used in this study are available via the SARAO archive (https://archive.sarao.ac.za) under project ID DDT-20240213-AW-01. The ATCA data used in this study are available via the Australia Telescope Online Archive (https://atoa.atnf.csiro.au/) under project ID C3363. Other specific data are available on request from the corresponding author. The code that supports the findings of this study is available on GitHub at https://github.com/Andywang201605/J1832-0911_radio_xray. Specific scripts used in the data analysis are available on request from the corresponding author. Hurley-Walker, N. et al. A radio transient with unusually slow periodic emission. Hurley-Walker, N. et al. A long-period radio transient active for three decades. Caleb, M. et al. An emission-state-switching radio transient with a 54-minute period. The discovery of a nearby 421 transient with CHIME/FRB/Pulsar. Preprint at https://arxiv.org/abs/2407.07480 (2024). de Ruiter, I. et al. Sporadic radio pulses from a white dwarf binary at the orbital period. Beyond the rotational deathline: radio emission from ultra-long period magnetars. Katz, J. I. GLEAM-X J162759.5 523504.3 as a white dwarf pulsar. Magnetic interaction in white dwarf binaries as mechanism for long-period radio transients. Schwope, A. et al. X-ray properties of the white dwarf pulsar eRASSU J191213.9−41044. Rea, N. et al. Constraining the nature of the 18 min periodic radio transient GLEAM-X J162759.5−523504.3 via multiwavelength observations and magneto-thermal simulations. Hotan, A. W. et al. Australian Square Kilometre Array Pathfinder: I. VAST: an ASKAP survey for variables and slow transients. The ASKAP variables and slow transients (VAST) pilot survey. Wang, Z. et al. The CRAFT coherent (CRACO) upgrade I: system description and results of the 110-ms radio transient pilot survey. Yao, J. M., Manchester, R. N. & Wang, N. A new electron-density model for estimation of pulsar and FRB distances. Wenger, T. V., Balser, D. S., Anderson, L. D. & Bania, T. M. Kinematic distances: a Monte Carlo method. Yuan, W., Zhang, C., Chen, Y. The Einstein Probe mission. In Handbook of X-ray and Gamma-ray Astrophysics (eds Bambi, C. & Sangangelo, A.) Chen, K. & Ruderman, M. Pulsar death lines and death valley. Zhang, B., Harding, A. K. & Muslimov, A. G. Radio pulsar death line revisited: is PSR J2144−3933 anomalous? Harding, A. K. & Muslimov, A. G. Pulsar pair cascades in a distorted magnetic dipole field. The X-ray luminosity of rotation-powered neutron stars. Saumon, D., Blouin, S. & Tremblay, P.-E. Current challenges in the physics of white dwarf stars. Heise, J. X-ray emission from isolated hot white dwarfs. Evidence for an abundant old population of Galactic ultra-long period magnetars and implications for fast radio bursts. Marsh, T. R. et al. A radio-pulsing white dwarf binary star. Pelisoli, I. et al. A 5.3-min-period pulsing white dwarf in a binary detected from radio to X-rays. Hurley-Walker, N. et al. A 2.9-hour periodic radio transient with an optical counterpart. Bagnulo, S. & Landstreet, J. D. The isolated magnetic white dwarfs. Kaspi, V. M. & Beloborodov, A. M. Magnetars. Esposito, P., Rea, N. & Israel, G. L. Magnetars: A short review and some sparse considerations. In Timing Neutron Stars: Pulsations, Oscillations and Explosions, Astrophysics and Space Science Library Vol. 461 (eds Belloni, T. M. et al.) 97–142 (Springer, 2021). & Metzger, B. D. Periodicity in recurrent fast radio bursts and the origin of ultralong period magnetars. Transient pulsed radio emission from a magnetar. Coti Zelati, F., Rea, N., Pons, J. A., Campana, S. & Esposito, P. Systematic study of magnetar outbursts. Viganò, D. et al. Unifying the observational diversity of isolated neutron stars via magneto-thermal evolution models. Dehman, C., Viganò, D., Pons, J. & Rea, N. 3D code for magneto-thermal evolution in isolated neutron stars, MATINS: the magnetic field formalism. Discovery of a radio-emitting neutron star with an ultra-long spin period of 76 s. Nat. Lander, S. K., Gourgouliatos, K. N., Wadiasingh, Z. & Antonopoulou, D. Observing the Meissner effect in neutron stars. Preprint at https://arxiv.org/abs/2411.08020 (2024). Guzman, J. et al. ASKAPsoft: ASKAP science data processor software. Astrophysics Source Code Library ascl:1912.003 (2019). Purcell, C. R., Van Eck, C. L., West, J., Sun, X. H. & Gaensler, B. M. RM-Tools: rotation measure (RM) synthesis and Stokes QU-fitting. Astrophysics Source Code Library ascl:2005.003 (2020). The Rapid ASKAP Continuum Survey I: design and first results. The Rapid ASKAP Continuum Survey paper II: first Stokes I source catalogue data release. & Wright, M. C. H. A retrospective view of MIRIAD. In Astronomical Data Analysis Software and Systems IV, Astronomical Society of the Pacific Conference Series Vol. 77 (eds Shaw, R. A. et al.) 433–436 (1995). The MeerKAT telescope as a pulsar facility: system verification and early science results from MeerTime. The thousand-pulsar-array programme on MeerKAT IV: polarization properties of young, energetic pulsars. Heywood, I. oxkat: semi-automated imaging of MeerKAT observations. Astrophysics Source Code Library ascl:2009.003 (2020). McMullin, J. P., Waters, B., Schiebel, D., Young, W. & Golap, K. CASA architecture and applications. In Astronomical Data Analysis Software and Systems XVI, Astronomical Society of the Pacific Conference Series Vol. 376 (eds Shaw, R. A. et al.) 127–130 (2007). Hugo, B. V., Perkins, S., Merry, B., Mauch, T. & Smirnov, O. M. Tricolour: An optimized SumThreshold flagger for MeerKAT. In Astronomical Data Analysis Software and Systems XXX, Astronomical Society of the Pacific Conference Series Vol. 532 (eds Ruiz, J. E. et al.) 541–544 (2022). Kenyon, J. S., Smirnov, O. M., Grobler, T. L. & Perkins, S. J. CUBICAL—fast radio interferometric calibration suite exploiting complex optimization. Offringa, A. R. et al. WSCLEAN: an implementation of a fast, generic wide-field imager for radio astronomy. Collier, J. D., Frank, B., Sekhar, S. & Taylor, A. R. The IDIA PROCESSMEERKAT pipeline: fast CASA processing on a cloud-based HPC cluster. In 2021 XXXIVth General Assembly and Scientific Symposium of the International Union of Radio Science 4 (2021). Deller, A. T. et al. DiFX-2: a more flexible, efficient, robust, and powerful software correlator. Kettenis, M., van Langevelde, H. J., Reynolds, C. & Cotton, B. ParselTongue: AIPS talking Python. In Astronomical Data Analysis Software and Systems XV, Astronomical Society of the Pacific Conference Series Vol. 351 (eds Gabriel, C. et al.) 497–500 (2006). Ding, H. et al. VLBA astrometry of the fastest-spinning magnetar Swift J1818.0−1607: a large trigonometric distance and a small transverse velocity. Exploring the transient radio sky with VLITE: early results. Clarke, T. E. et al. Commensal low frequency observing on the NRAO VLA: VLITE status and future plans. Cotton, W. D. Obit: a development environment for astronomical algorithms. Polisensky, E., Richards, E., Clarke, T., Peters, W. & Kassim, N. The VLITE database pipeline. In Astronomical Data Analysis Software and Systems XXVII, Astronomical Society of the Pacific Conference Series Vol. 523 (eds Teuben, P. J. et al.) 441–444 (2019). Mohan, N. & Rafferty, D. PyBDSF: Python blob detection and source finder. Astrophysics Source Code Library ascl:1502.007 (2015). Lomb, N. R. Least-squares frequency analysis of unequally spaced data. Balucinska-Church, M. & McCammon, D. Photoelectric absorption cross sections with variable abundances. Lodders, K. Solar System abundances and condensation temperatures of the elements. HI4PI Collaboration et al. HI4PI: afull-sky H I survey based on EBHIS and GASS. & Kaspi, V. M. The correlation between dispersion measure and X-ray column density from radio pulsars. Hobbs, G. B., Edwards, R. T. & Manchester, R. N. TEMPO2, a new pulsar-timing package—I. Luo, J. et al. PINT: a modern software package for pulsar timing. Foreman-Mackey, D., Hogg, D. W., Lang, D. & Goodman, J. emcee: the MCMC hammer. Lorimer, D. R. SIGPROC: Pulsar signal processing programs. Astrophysics Source Code Library ascl:1107.016 (2011). Lorimer, D. R. & Kramer, M. Handbook of Pulsar Astronomy, Cambridge Observing Handbooks for Research Astronomers, Vol. Manchester, R. N., Hobbs, G. B., Teoh, A. & Hobbs, M. The Australia Telescope National Facility pulsar catalogue. Morgan, J. S. & Ekers, R. A measurement of source noise at low frequency: implications for modern interferometers. Calabretta, M. R., Staveley-Smith, L. & Barnes, D. G. A new 1.4 GHz radio continuum map of the sky south of declination +25°. Reid, M. J. et al. Trigonometric parallaxes of high mass star forming regions: the structure and kinematics of the Milky Way. & Mamajek, E. E. Intrinsic colors, temperatures, and bolometric corrections of pre-main-sequence stars. Green, G. M. dustmaps: a Python interface for maps of interstellar dust. Open Source Softw. van Soelen, B. et al. NIR spectral classification of the companion in the gamma-ray binary HESS J1832−093 as an O6 V star. Detection of X-ray emission from a bright long-period radio transient. Constraints on the dense matter equation of state from young and cold isolated neutron stars. Viganò, D., Garcia-Garcia, A., Pons, J. A., Dehman, C. & Graber, V. Magneto-thermal evolution of neutron stars with coupled ohmic, Hall and ambipolar effects via accurate finite-volume simulations. We thank B. Gaensler, S. Dai and F. Coti Zelati for valuable discussions. We are grateful to the ASKAP engineering and operations team for their assistance in developing fast radio burst instrumentation for the telescope and supporting the survey. This work uses data obtained from Inyarrimanha Ilgari Bundara/the CSIRO Murchison Radio-astronomy Observatory. We acknowledge the Wajarri Yamaji People as the Traditional Owners and native title holders of the observatory site. CSIRO's ASKAP radio telescope is part of the Australia Telescope National Facility (https://ror.org/05qajvd42). Operation of ASKAP is funded by the Australian Government with support from the National Collaborative Research Infrastructure Strategy. ASKAP uses the resources of the Pawsey Supercomputing Research Centre. Establishment of ASKAP, Inyarrimanha Ilgari Bundara, the CSIRO Murchison Radio-astronomy Observatory and the Pawsey Supercomputing Research Centre are initiatives of the Australian Government, with support from the Government of Western Australia and the Science and Industry Endowment Fund. CRACO was funded through Australian Research Council Linkage Infrastructure Equipment, and Facilities grant LE210100107. We thank the staff of the GMRT that made these observations possible. GMRT is run by the National Centre for Radio Astrophysics of the Tata Institute of Fundamental Research. We thank SARAO for the approval of the MeerKAT DDT request DDT-20240213-AW-01.The MeerKAT telescope is operated by the South African Radio Astronomy Observatory, which is a facility of the National Research Foundation, an agency of the Department of Science and Innovation. The National Radio Astronomy Observatory is a facility of the National Science Foundation operated under cooperative agreement by Associated Universities, Inc. We thank M. Bailes for supporting the PTUSE backend machine used in the MeerKAT observation. PTUSE was developed with support from the Australian SKA Office and Swinburne University of Technology. This research has made use of data obtained from the Chandra Data Archive provided by the Chandra X-ray Center (CXC). We acknowledge the use of public data from the Swift data archive. This research is based on observations obtained with XMM-Newton, an ESA science mission with instruments and contributions directly funded by ESA Member States and NASA. We thank the Einstein Probe principal investigator (W. Yuan) for accepting our ToO observation, Y. Chen as the FXT principal investigator, and the Einstein Probe Science Center for performing the observations. Einstein Probe is a space mission supported by the Strategic Priority Program of the Space Science of the Chinese Academy of Sciences (grant number XDB0550200), in collaboration with ESA, MPE and CNES (grant number XDA15310000), and the National Key R&D Program of China (2022YFF0711500). This paper includes data gathered with the 6.5-meter Magellan Telescope located at Las Campanas Observatory, Chile. Part of this work was performed on the OzSTAR national facility at Swinburne University of Technology. The OzSTAR programme receives funding in part from the Astronomy National Collaborative Research Infrastructure Strategy (NCRIS) allocation provided by the Australian Government, and from the Victorian Higher Education State Investment Fund (VHESIF) provided by the Victorian Government. We acknowledge the use of the ilifu cloud computing facility (www.ilifu.ac.za), a partnership between the University of Cape Town, the University of the Western Cape, Stellenbosch University, Sol Plaatje University, the Cape Peninsula University of Technology and the South African Radio Astronomy Observatory. The ilifu facility is supported by contributions from the Inter-University Institute for Data Intensive Astronomy (IDIA, a partnership between the University of Cape Town, the University of Pretoria and the University of the Western Cape), the Computational Biology division at UCT and the Data Intensive Research Initiative of South Africa (DIRISA). This work was carried out using the data-processing pipelines developed at the Inter-University Institute for Data Intensive Astronomy (IDIA) and available at https://idia-pipelines.github.io. IDIA is a partnership of the University of Cape Town, the University of Pretoria and the University of the Western Cape. This work made use of the CARTA (Cube Analysis and Rendering Tool for Astronomy) software (https://doi.org/10.5281/zenodo.3377984 and https://cartavis.github.io). This research has made use of the NASA Astrophysics Data System. is supported by the European Research Council (ERC) via the Consolidator Grant ‘MAGNESIA' (number 817661) and the Proof of Concept ‘DeepSpacePulse' (number 101189496), by the Catalan grant SGR2021-01269 (principal investigator V. ), the Spanish grant ID2023-153099NA-I00 (principal investigator F. Coti Zelati), and by the programme Unidad de Excelencia Maria de Maeztu CEX2020-001058-M. T.B. acknowledges financial support from the Framework per l'Attrazione e il Rafforzamento delle Eccellenze (FARE) per la ricerca in Italia (R20L5S39T9). is supported by NSF grant AST-1816492. The material is based upon work supported by NASA under award number 80GSFC24M0006. acknowledge support by NASA under award numbers 80GSFC21M0002 and 80GSFC21M0006. acknowledges support from a NASA grant 80NSSC24K0770, a grant (number 2020747) from the United States-Israel Binational Science Foundation (BSF), Jerusalem, Israel and by a grant (number 1649/23) from the Israel Science Foundation. acknowledges support from the Oxford Hintze Centre for Astrophysical Surveys, which is funded through generous support from the Hintze Family Charitable Foundation. Basic research in radio astronomy at the US Naval Research Laboratory is supported by 6.1 Base funding. Construction and installation of VLITE was supported by the NRL Sustainment Restoration and Maintenance fund. is supported by the Australian Government through the Australian Research Council's Discovery Projects funding scheme (DP210102103), and through UK STFC Grant ST/Y001117/1. acknowledges support from the Inter-University Institute for Data Intensive Astronomy (IDIA). IDIA is a partnership of the University of Cape Town, the University of Pretoria and the University of the Western Cape. For the purpose of open access, the author has applied a Creative Commons Attribution (CC BY) licence to any author accepted manuscript version arising from this submission. N.H.-W. is the recipient of an Australian Research Council Future Fellowship (project number FT190100231) funded by the Australian Government. acknowledges the support of an Australian Research Council Discovery Early Career Research Award (project number DE220100819) funded by the Australian Government. acknowledges support by the Australian Government through the Australian Research Council's Discovery Projects funding scheme (project DP210102103). receives support from the ARC Discovery Early Career Research Award DE250100508. The Chandra X-ray observation presented in this paper and partial funding for K.M. are supported by SAO grant GO3-24121X. M.P.-T. acknowledges financial support from the Severo Ochoa grant CEX2021-001131-S and from the National grant PID2023-147883NB-C21, funded by MCIU/AEI/10.13039/501100011033. thanks the LSST-DA Data Science Fellowship Program, which is funded by LSST-DA, the Brinson Foundation and the Moore Foundation; their participation in the programme has benefited this work. acknowledge support through Australian Research Council Discovery Project DP220102305. acknowledges support through Australian Research Council Future Fellowship FT190100155. R.T. acknowledges support from funding provided by the National Aeronautics and Space Administration (NASA), under award number 80NSSC20M0124, Michigan Space Grant Consortium (MSGC). was supported by the National Natural Science Foundation of China (grant numbers 12494575 and 12273009). Parts of this research were conducted by the Australian Research Council Centre of Excellence for Gravitational Wave Discovery (OzGrav), through project numbers CE170100004 and CE230100016. International Centre for Radio Astronomy Research, Curtin University, Bentley, Western Australia, Australia Ziteng Wang, Apurba Bera, Marcin Glowacki, Natasha Hurley-Walker, S. J. McSweeney, Kristen C. Dage, Clancy W. James & N. D. Ramesh Bhat Institute of Space Sciences (ICE), CSIC, Barcelona, Spain Institut d'Estudis Espacials de Catalunya (IEEC), Barcelona, Spain INAF - Osservatorio Astronomico di Brera, Merate, Italy Center for Gravitation, Cosmology, and Astrophysics, Department of Physics, University of Wisconsin-Milwaukee, Milwaukee, WI, USA David L. Kaplan & Akash Anumarlapudi ATNF, CSIRO, Space and Astronomy, Epping, New South Wales, Australia Emil Lenc, Andrew Zic, J. R. Dawson, Keith W. Bannister, Marcus E. Lower, Kovi Rose, Xinping Deng, Y. W. Joshua Lee, Joshua Pritchard & John Tuthill Department of Astronomy, University of Maryland College Park, College Park, MD, USA Astrophysics Science Division, NASA/GSFC, Greenbelt, MD, USA Center for Research and Exploration in Space Science and Technology, NASA/GSFC, Greenbelt, MD, USA Zorawar Wadiasingh & Jeremy Hare NASA Goddard Space Flight Center, Greenbelt, MD, USA Jeremy Hare & George Younes The Catholic University of America, Washington DC, USA ARC Centre of Excellence for Gravitational Wave Discovery (OzGrav), Hawthorn, Victoria, Australia Andrew Zic, Adam T. Deller, Manisha Caleb, Yuanming Wang, Dougal Dobie, Tara Murphy & Y. W. Joshua Lee Department of Natural Sciences, The Open University of Israel, Ra'anana, Israel Astrophysics Research Center of the Open University (ARCO), The Open University of Israel, Ra'anana, Israel Department of Physics, The George Washington University, Washington DC, USA Naval Research Laboratory, Washington DC, USA Tracy E. Clarke, Emil J. Polisensky & Wendy M. Peters Centre for Astrophysics and Supercomputing, Swinburne University of Technology, Hawthorn, Victoria, Australia Adam T. Deller, Marcus E. Lower, Ryan M. Shannon, Yuanming Wang, Akhil Jaini & Joscha N. Jahns-Schindler School of Mathematical and Physical Sciences and Astrophysics and Space Technologies Research Centre, Macquarie University, Sydney, New South Wales, Australia Institute for Astronomy, University of Edinburgh, Royal Observatory, Edinburgh, UK Inter-University Institute for Data Intensive Astronomy, Department of Astronomy, University of Cape Town, Cape Town, South Africa Center for Space Sciences and Technology CRESST II, UMBC, Baltimore, MD, USA Sydney Institute for Astronomy, School of Physics, The University of Sydney, Sydney, New South Wales, Australia Keith W. Bannister, Manisha Caleb, Kovi Rose, Dougal Dobie, Laura N. Driessen, Tara Murphy & Y. W. Joshua Lee Cahill Center for Astronomy and Astrophysics, California Institute of Technology, Pasadena, CA, USA Mansi M. Kasliwal, Viraj Karambelkar & Stella Koch Ocker Columbia Astrophysics Laboratory, Columbia University, New York, NY, USA Observatories of the Carnegie Institution for Science, Pasadena, CA, USA Instituto de Astrofísica de Andalucía (IAA-CSIC), Consejo Superior de Investigaciones Científicas (CSIC), Granada, Spain School of Sciences, European University Cyprus, Nicosia, Cyprus SKA Observatory, Jodrell Bank, Macclesfield, UK Department of Physics and Astronomy, Wayne State University, Detroit, MI, USA School of Astronomy and Space Science, Nanjing University, Nanjing, People's Republic of China Fayin Wang & Zhenyin Zhao Hebei Key Laboratory of Radio Astronomy Technology, Shijiazhuang, People's Republic of China ATNF, CSIRO, Space and Astronomy, Bentley, Western Australia, Australia You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar drafted the paper with suggestions from co-authors. Z. Wang is the principal investigator of the MeerKAT data and the GMRT data. is the principal investigator of the VLBA data. is the principal investigator of the ATCA data (C3363). E.L., Z. Wang and A.A. further reduced the ASKAP data to produce dynamic spectra for detections and non-detections. Z. Wang and N.H.-W. reduced and analysed the MeerKAT imaging data. helped propose the MeerKAT observation and analyse MeerKAT data. reduced the MeerKAT H i data. observed, reduced and analysed the ATCA data. and Z. Wang reduced and analysed the GMRT data. A.A. reduced the archival VLA data. developed and maintained the VLITE data archive and performed the VLITE archive search, time slicing, imaging and cataloguing. reduced and analysed the VLBA data. observed and analysed the infrared data. is the principal investigator of the Chandra data. reduced and analysed the Chandra data. reduced and analysed archival XMM-Newton and Swift data. helped in asking for the Einstein Probe data. N.R., as member of the Einstein Probe collaboration, reduced and analysed the Einstein Probe data. and R.T. performed multiwavelength archive searches. performed the radio timing analysis. performed the radio polarimetry data reduction. performed H i absorption line analysis. N.H.-W. performed the SNR association analysis. performed the short-periodicity search. contributed to discussions about the nature and emission mechanism of the source. are the principal investigators of CRACO. contributed to the design and commissioning of CRACO. are the principal investigators of VAST, and D.D. are the project scientists of VAST. The principal investigators and builders of VAST and CRACO coordinated the initial investigation of the source. Correspondence to Ziteng Wang. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Panel (a) shows a composite of radio (MeerKAT 816 MHz, red), X-ray (Chandra 1–10 keV, green), and infrared (WISE 12 μm, blue) emission of the field of ASKAP J1832–0911. The Fermi 95% positional error ellipse around 4FGL J1832.9-0913 is shown in cyan dashed line. The supernovae remnant SNR G22.7–0.2 is highlighted in the yellow dotted line. Panel (b) shows MeerKAT total intensity contours at 30, 40, and 50 mJy levels overlaid on the Chandra detection image on 2024 February 14. Panel (c) and (d) show the deepest near-infrared images of ASKAP J1832–0911 at J- and Ks-band, respectively. Red circles show 50 times the systematic uncertainty of the source position (~5 mas). Horizontal lines show the false alarm probabilities at 3σ (green), 2σ (orange), and 1σ (red). The purple vertical line shows the best frequency we fit from radio observations. The 3σ X-ray luminosity limits are calculated as a function of photon index (Γ; panel a) and black body temperature (kT; panel b). We assume NH = 1.8 × 1022 cm−2, which is the Galactic column density in the direction of the source, consistent with the X-ray spectral fits during the outburst. (a) Evolution in the \(P-\mathop{P}\limits^{. Dashed (solid) lines correspond to theoretical death lines for a pure dipole (highly multipolar) configuration18,19. The grey-shaded region indicates the radio pulsar ‘death valley' between the two extreme configurations. (b) Evolution of the quiescent X-ray luminosity as a function of the rotational power (\(\mathop{E}\limits^{.}\)). The grey line is \({L}_{X}=\mathop{E}\limits^{. }\) while the grey shaded area represents the constraints for ASKAP J1832–0911 during quiescence. See10,77,78 and references therein for details on the theoretical cooling models and the plotted sources. Circled stars are radio-emitting magnetars. We assume as a luminosity limit for the quiescent emission of ASKAP J1832–0911 that is derived from the deepest XMM-Newton limits on 2011-03-13, and the most recent limits derived after the outburst by Chandra on 2024-08-11 (light blue lines). Solid lines assume an X-ray spectrum modelled by a power law with Γ = 2, while dashed lines assume a blackbody with kT = 0.5 keV. The age refers to the age of the SNR when available, and to the characteristic age otherwise. See10 and Dehman, Marino, and Rea et al., manuscript in preparation for more details on the data extraction and sources. Uncertainties on the times of arrival have been adjusted in quadrature by EQUAD, fitted per-epoch, with values ranging from 6–69 s. The points are coloured by radio frequency, and the telescopes used for each measurement are indicated by the markers. The purple line shows the spectral type limit we can constrain based on the current infrared observations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Wang, Z., Rea, N., Bao, T. et al. Detection of X-ray emission from a bright long-period radio transient. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09070-3'>In vivo haemopoietic stem cell gene therapy enabled by postnatal trafficking</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 15:32:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Lentiviral vector (LV)-mediated ex vivo gene therapy for haematopoietic stem and progenitor cells (HSPCs) has delivered on the promise of a ‘one-and-done' treatment for several genetic diseases1. However, ex vivo manipulation and patient conditioning before transplantation are major hurdles that could be overcome by an in vivo approach. Here we demonstrate that in vivo gene delivery to HSPCs after systemic LV administration is enabled by the substantial trafficking of these cells from the liver to the bone marrow in newborn mice. We improved gene-transfer efficiency using a phagocytosis-shielded LV, successfully reaching bona fide HSPCs capable of long-term multilineage output and engraftment after serial transplantation, as confirmed by clonal tracking. HSPC mobilization further increased gene transfer, extending the window of intervention, although permissiveness to LV transduction declined with age. We successfully tested this in vivo strategy in mouse models of adenosine deaminase deficiency, autosomal recessive osteopetrosis and Fanconi anaemia. Interestingly, in vivo gene transfer provided a selective advantage to corrected HSPCs in Fanconi anaemia, leading to near-complete haematopoietic reconstitution and prevention of bone marrow failure. Given that circulating HSPCs in humans are also most abundant shortly after birth, in vivo HSPC gene transfer holds strong translational potential across multiple diseases. Ex vivo gene transfer using LVs in HSPCs has demonstrated both safety and efficacy in treating haematological and metabolic disorders. Several gene-therapy products are now in advanced clinical development or available on the market1. However, certain challenges hinder their broader application to all eligible patients2,3, such as the need to collect, manipulate and preserve enough functional HSPCs to ensure effective haematopoietic reconstitution, as well as the need for patient conditioning before cell infusion. Moreover, ex vivo gene therapy enables correction only in HSPCs, whereas many genetic diseases have systemic manifestations that would benefit from broader correction. In vivo gene therapy presents a potential solution to these limitations, but targeting HSPCs in their niche remains a major challenge4,5,6. Here we identified a unique window of opportunity in early postnatal life, when high numbers of circulating HSPCs (cHSPCs) are trafficking from residual haematopoietic niches to the bone marrow (BM). Building on this observation and our previous development of engineered LVs for in vivo administration and tissue-targeted expression7, we report successful gene transfer into bona fide long-term HSPCs in newborn mice. To assess the translational potential of this approach, we carried out tests in models of disease in mice in which distinct haematopoietic lineages and/or extra-haematopoietic tissues are involved. We selected: (1) adenosine deaminase (ADA) deficiency, which causes severe combined immunodeficiency (ADA-SCID) through the accumulation of toxic adenosine metabolites, impairing T cell, B cell and natural killer cell differentiation and function; (2) autosomal recessive osteopetrosis (ARO), a severe inherited bone disease in humans caused by mutations in the T cell immune regulator 1 (TCIRG1) gene, leading to defective osteoclast function and impaired bone resorption; and (3) Fanconi anaemia, a rare genetic disorder classified as a DNA repair deficiency syndrome, caused by mutations in the Fanconi anaemia pathway, resulting in severe HSPC depletion and leading to BM failure and increased cancer susceptibility. To assess the feasibility of in vivo LV gene transfer into HSPCs, we first examined the spatiotemporal evolution of the haematopoietic system in C57BL/6 mice. We characterized the composition and absolute counts of haematopoietic cells in the BM, peripheral blood (PB), spleen and liver in newborn (postnatal day 0–1), 1-, 2- and 8-week-old (adult) mice (Fig. of the percentage of different populations between Lin−KIT+ HSPCs (b–e) or single values and mean and s.e.m of the absolute count of HSCs (f–i) in the BM (b,f), PB (c,g), spleen (d,h) and liver (e,i) at the indicated ages. j, Experimental scheme: LV was administered i.v. to newborn C57BL/6 mice (510 transducing units, TU kg−1). GFP expression was monitored monthly in PB of treated mice. After 17 weeks, GFP+ cells were FACS-sorted from the BM and transplanted into busulfan-conditioned adult C57BL/6 mice. of GFP+ cells measured over time in the PB of treated mice (n = 19, k) and in the indicated populations 17 weeks after LV administration (l). m–q, Single values of GFP+ cells measured over time in PB of individual surviving transplanted mice (m–p) and in the indicated populations 20 weeks after transplant (q). r, Heat map showing integration sites (ISs) (columns) shared between LV-treated newborn mice (pooled, n = 19) 17 weeks after administration and transplanted mice (pooled, n = 4) 20 weeks after transplant (shown in k,l). TPX: indicated organs of transplanted mice shown in m–q. Sample count indicates the number of samples (rows) sharing a given IS. A complete list of the ISs is provided in Supplementary Table 2. Schematics in a and j created with BioRender. Analysis of the HSPC compartment revealed a consistent proportion of stem cells and different progenitor subtypes in the BM across all ages (Fig. By contrast, haematopoietic stem cells (HSCs) were more prevalent in the PB and spleen in newborns compared with older mice (Fig. Further analysis specifically focusing on the HSC distribution confirmed the highest concentrations in the PB of newborns, which declined progressively over time and was paralleled by an increase in the BM (Fig. 1f,g), indicating ongoing HSC homing to the maturing BM niche. Furthermore, HSCs expanded significantly in the spleens of 1- and 2-week-old mice but were absent in the adult liver (Fig. To confirm active haematopoiesis in the livers of young mice, we detected cycling cells in haematopoietic foci that also included megakaryocytes (Extended Data Fig. Together, these findings highlight a unique window of opportunity for the genetic engineering of HSC in newborns, marked by the persistence of fetal haematopoietic niches shortly after birth and extensive HSC trafficking in the blood. To evaluate the potential of in vivo LV gene transfer into HSCs, we intravenously (i.v.) administered GFP-expressing LVs to newborn C57BL/6 mice. This procedure resulted in stable GFP expression in approximately 0.5% of all blood lineages (Fig. We observed comparable GFP marking across different lineages in the spleen and BM, including KIT+SCA-1+Lin− cells, in most analysed mice (Fig. To confirm gene transfer into long-term HSCs, we transplanted GFP+ cells collected from the BM into busulfan-conditioned adult mice. As GFP-marking levels were low, we pooled CD45+GFP+ cells from the BM sorted using fluorescence-activated cell sorting (FACS) at 16 weeks after LV administration from four to five mice treated as newborns and transplanted each pool into busulfan-conditioned adult mice (Fig. Molecular analysis of sorted HSPCs showed an average of one vector copy per diploid genome (VCN; Extended Data Fig. Notably, this mouse had received the lowest number of CD45+GFP+ cells (Extended Data Fig. Engraftment levels varied between the four surviving mice, which was expected given the low number of transplanted cells and the heterogeneous composition of GFP+ cells in each pool (Fig. However, we detected GFP+ cells across all analysed lineages in the PB and haematopoietic organs at comparable levels in each mouse, with a corresponding VCN (Fig. To confirm whether the GFP+ haematopoietic output in the transplanted mice originated from bona fide HSCs, we performed clonal tracking using LV integration site analysis. Examination of the GFP+ cells collected for transplantation from the BM of LV-treated mice, and of sorted GFP+ splenic myeloid, B cells and T cells revealed hundreds of clones that contributed homogeneously to the haematopoietic output across multiple lineages, without dominant clones. Analysis of total BM, spleen and thymus cells 20 weeks after transplant in recipient mice revealed heterogeneous contributions from a subset of clones originally identified in donor mice (Fig. None of the clones that expanded in transplanted mice was predominant in donor mice or had integrations near oncogenes (Extended Data Fig. These findings show that LV administration in newborn mice successfully targeted multipotent, self-renewing HSPCs that contributed to long-term haematopoiesis in both donor and recipient mice. To further assess long-term transduction, we monitored newborn-treated mice for more than 1 year after LV administration and observed stable GFP expression across all analysed PB lineages (Extended Data Fig. Next, to compare HSPC transduction efficiency between newborn and adult mice, we administered i.v. GFP marking in the circulation was initially detected but rapidly declined by 4 weeks after LV administration (Extended Data Fig. This delayed loss was probably due to an emerging anti-GFP immune response in mice treated as adults. As i.v.-administered LVs predominantly transduce liver cells7,8, we examined the liver at the study end point and found few, if any, GFP+ cells (Extended Data Fig. 1p), suggesting clearance by cytotoxic T cells. Overall, these findings demonstrate successful in vivo gene transfer into bona fide HSCs, facilitated by the unique features of newborn haematopoiesis. Our initial analyses showed that HSC content was highest in the liver and spleen of 2-week-old mice (Fig. We therefore tested the efficiency of in vivo HSPC gene transfer at this age but found that it was significantly lower than in newborns (Extended Data Fig. These data suggest that HSPCs are targeted mostly when circulating in newborns. To increase the amount of HSPC accessible to LV transduction, we administered 2-week-old mice a granulocyte-colony stimulating factor (G-CSF) and plerixafor mobilization regimen, modelling its clinical use. This treatment sharply increased the number of circulating HSCs (cHSCs) and enhanced in vivo gene transfer, reaching a mean of 8.5% GFP+CD45+ cells—fivefold higher than in control mice (Fig. At 20 weeks after LV administration, GFP expression and VCN in the BM, thymus and spleen remained higher in mobilized mice than in controls (Fig. We then performed total BM transplantation into busulfan-conditioned CD45.1 recipient mice. Cells derived from LV-treated donor mice successfully engrafted in all transplanted mice, with GFP-marking levels matching those of the corresponding donor, in PB and all analysed organs at 16 weeks after transplant (Fig. We then applied a shorter mobilization regimen to newborn C57BL/6 mice that resulted in a 15-fold increase in gene transfer efficiency, reaching 11% GFP+CD45+ circulating cells (Fig. These findings indicate that in vivo LV gene-transfer efficiency is highest in newborn mice, reflecting a window of HSC accessibility that can be extended to juvenile mice through mobilization. of HSC absolute count in BM (a) and PB (b) of 2-week-old mobilized (n = 6) or control (n = 7) C57BL/6 mice. (c,d) with single values (e,f) of GFP+ cells (c,e) and VCN (d,f) in PB (c,d), BM (e) and whole organs (f) of 2-week-old mobilized (n = 6) or control (n = 6) C57BL/6 mice over time (PB, 1.1 × 1011 TU kg−1) and 20 weeks after LV administration (e,f). (g–i) with single values (j) of GFP+ cells (g,h,j) and VCN (i) in PB (g–i) or BM (j) of mice transplanted with BM cells from control (g, n = 5) or mobilized (h, n = 6) mice over time (PB, 15 million CD45+ cells per recipient) and 16 weeks after transplant (j). of GFP+ cells (k) and VCN (l) in PB of mobilized (n = 7) or control (n = 6) newborn C57BL/6 mice (9 × 1010 TU kg−1). of GFP+ cells in PB of indicated mice treated at the indicated LV doses and ages (newborn, q; 2-week-old or adult mice, r). These included administration of an anti-interferon-α (IFNα) receptor-blocking antibody (MAR-1), the use of a phagocytosis-shielded LV (CD47high-LV) with high surface content of the ‘don't-eat-me' signal7 CD47 and higher LV doses. We confirmed this increase across haematopoietic subpopulations in the spleen, thymus and BM at 16 weeks after LV administration (Extended Data Fig. These data suggest that IFNα restricts in vivo LV transduction of HSPCs, a limitation that MAR-1 can mitigate. We next evaluated the effect of the CD47high-LV, using non-obese diabetic/SCID/IL2Rγ (NSG) mice bearing a CD47-receptor polymorphism that enables it to interact with human CD47 displayed on the LV surface9. A spatiotemporal analysis of haematopoiesis in these mice showed that, similar to C57BL/6 mice, cHSC numbers declined over time whereas BM HSC content modestly increased (Extended Data Fig. In contrast to C57BL/6 mice, however, NSG mice had fewer cHSCs as newborns but an increased HSC presence in the liver and spleen (Extended Data Fig. When we administered CD47high-LV expressing GFP to newborn and adult NSG mice, we observed a fourfold increase in GFP marking in NSG mice treated as newborns compared with both C57BL/6 mice treated as newborns and NSG mice treated as adults (Figs. This finding demonstrates that phagocytosis shielding enhances LV gene transfer into HSPCs (Fig. We confirmed similar GFP expression in HSPCs in the BM of treated mice 20 weeks after LV administration (Fig. To further assess clonogenic capacity, we FACS-sorted GFP+ or GFP−Lin− cells from the BM of LV-treated NSG mice, performed colony-forming unit assays and found comparable clonogenic potential (Extended Data Fig. We also confirmed a higher VCN in Lin−GFP+ cells FACS-sorted from the BM of mice treated as newborns compared with those treated as adults (Extended Data Fig. To evaluate the range of VCN in HSPCs transduced in vivo, we analysed single colonies from colony-forming unit assays and measured a VCN of 1 in most colonies (Extended Data Fig. We also performed a biodistribution analysis of GFP+ cells using immunohistochemistry, 20 weeks after LV administration to newborn or adult NSG mice. Immunohistochemistry revealed GFP+ cells in the liver and spleen, with scattered cells in the lung, kidney and brain only in NSG mice treated as newborns. These cells were predominantly haematopoietic and endothelial, with liver parenchymal cell transduction also observed (Extended Data Fig. We next combined HSC mobilization with CD47high-LV administration in 2-week-old or adult NSG mice. 5a–c), GFP marking was sevenfold higher in 2-week-old mice (Fig. Furthermore, LV doses were 2.5-fold lower in NSG compared with C57BL/6 mice as they had reduced tolerance to extensive manipulation by the mobilization regimen. Analysed together, these data show significantly fewer GFP+ cells in the circulation of mice treated at increasing ages, with the highest permissiveness observed in newborn mice (Fig. Moreover, despite mobilization increasing gene transfer in both young and adult mice, transduction was twofold higher in the former, suggesting that cell-intrinsic factors influence LV transduction efficiency (Fig. Finally, we assessed long-term HSPC gene transfer after increasing LV doses. We tested LV gene-transfer efficiency in newborn and 2-week-old C57BL/6 mice versus CD47high-LV in newborn, 2-week-old and adult NSG mice (Fig. 2q,r), confirming that HSPC accessibility to LV transduction was highest in newborns, with CD47high-LV enhancing gene-transfer efficiency in NSG mice. All mice treated as newborn or at 2 weeks old tolerated all LV doses without toxicity, whereas NSG adults had dose-limiting toxicity, preventing higher-dose administration (Fig. Overall, these findings establish that the newborn period is the optimal window for in vivo HSC gene transfer, although HSC mobilization and CD47high-LV could enhance transduction efficiency in young and adult mice. To evaluate the therapeutic potential of in vivo LV gene transfer, we selected ADA-SCID as our initial disease model for two key reasons. First, gene-corrected lymphocytes possess a selective survival advantage over uncorrected cells10. Second, the ADA enzyme is ubiquitously expressed, enabling gene-corrected hepatocytes to serve as a permanent enzyme replacement therapy by detoxifying the entire organism11. We tested three LV constructs with distinct expression patterns: PGK.ADA, which drives ubiquitous ADA expression under the phosphoglycerate kinase (PGK) promoter; PGK.ADA.122mirT, which excludes hepatocyte expression using complementary target sequences for the liver-specific miR-122; and ET.ADA.142mirT, which restricts ADA expression to hepatocytes using the enhanced transthyretin (ET) promoter along with miR-142 target sequences to suppress expression in haematopoietic cells12 (Fig. We administered these LVs to newborn ADA-SCID mice and monitored weight and survival over time up to 20 weeks after LV administration. Whereas all untreated ADA-SCID mice died before weaning, we observed longer survival in LV-treated mice, with healthy growth comparable with wild-type (WT) littermates (Fig. Meanwhile, 4 out of 5 ET.ADA.142mirT-LV mice also survived up to 20 weeks after LV administration, consistent with previous reports showing enzyme replacement therapy efficacy in this model and robust LV expression in hepatocytes13,14 (Fig. a, Newborn ADA-SCID mice or WT littermates were treated with 2.5 × 1010 TU kg−1 of the indicated LV. At 20 weeks after LV administration, 10–13 million total BM cells were transplanted into irradiated NSG mice. WT littermates were treated in parallel with ADA-SCID mice; a pooled dataset of all LV-treated WT mice is reported, regardless of the administered LV construct. of the VCN (c), T lymphocytes measured by FACS (d) and lymphocyte counts measured by haemocytometer (e) in the PB of ADA-SCID mice treated with the indicated LV as newborns. Sample sizes (n) at each time point are reported below the graphs. Linear mixed-effects (LME) model (compared with WT; Supplementary Tables 3–5). of the percentage of the indicated populations in the spleen (f) and thymus (g) of ADA-SCID mice treated as newborns with the indicated LV or left untreated, 20 weeks after LV administration. Sample sizes (n) are reported above the bars. h, Survival of NSG mice transplanted with total BM cells isolated from ADA-SCID mice shown in b–g. of lymphocyte counts in PB (8 weeks (i) or 12 weeks (j) after transplant), percentage of the indicated populations in the spleen (k) and thymus (l) or single values and mean and s.e.m. of VCN measured in the indicated organs (m) 12 weeks after transplant. Kruskal–Wallis test followed by post hoc analysis. Sample sizes (n) are reported above the bars (k,l) or in the graphs. Schematics in a created with BioRender. We observed a marked selective advantage only in PGK.ADA-LV-treated mice, in which 7% of circulating cells were transduced compared with 2–4% in LV-treated WT littermates (Fig. Immune reconstitution in the PB was fastest in PGK.ADA-LV-treated mice, whereas animals treated with the other LVs had a delayed immune recovery (Fig. Analysis of spleen and thymus composition at 20 weeks after LV administration showed no differences between all surviving LV-treated mice and WT littermates, further confirming immune system reconstitution (Fig. We measured ADA enzyme activity in red blood cells in the PB and found threefold higher activity in PGK.ADA-LV-treated mice compared with all other LV groups, although it remained below WT levels. Plasma ADA activity, however, was similar across all LV-treated mice and with compared WT levels (Extended Data Fig. We also confirmed the expected pattern of expression dictated by each LV design (Extended Data Fig. To determine whether gene-corrected HSCs retained engraftment potential, we transplanted total BM cells from ADA-SCID mice treated as newborns into irradiated adult NSG mice (Fig. 3a), enabling the monitoring of engraftment owing to an H-2Dq mismatch. Notably, despite NSG mice having normal ADA expression, only recipients transplanted with BM cells from PGK.ADA-LV-treated mice showed long term survival, similar to mice transplanted with WT cells (Fig. Mice that received BM cells from ET.ADA.142mirT-LV-treated donors had the lowest survival (four out of eight), as expected, as these cells lacked ADA transgene expression. The low survival of mice that received BM cells from PGK.ADA.122mirT-LV-treated donors suggests that these cells were less fit, despite expressing ADA in a fraction of them. This effect probably resulted from poor donor detoxification, similar to cells isolated from ET.ADA.142mirT-LV-treated donors (Fig. There were no significant differences in body weight and engraftment of transplanted mice between groups (Extended Data Fig. However, lymphocyte counts were restored only in mice that received BM cells collected from PGK.ADA-LV-treated donors (Fig. Similarly, spleen and thymus compositions were indistinguishable from WT-transplanted controls (Fig. In line with donor marking levels, mice that received BM cells from PGK.ADA-LV-treated donors and WT mice had similar VCNs, with an average 0.04 in PB and BM, confirming stable transduction of long-term repopulating HSCs (Fig. LV integration site analysis in the spleen, thymus and BM of LV-treated ADA-SCID mice and NSG transplant recipients confirmed integration site sharing across different organs, supporting that i.v. LV administration in newborns successfully targets bona fide HSCs (Extended Data Fig. Finally, we compared in vivo LV gene therapy efficacy in newborn versus young mice by administering PGK.ADA-LV to 2-week-old ADA-SCID mice at a weight-matched dose. Only one out of five mice treated at 2 weeks old survived long term, compared with five out of seven mice treated as newborns (Fig. Together, these findings demonstrate the therapeutic potential of in vivo ADA-LV administration in newborn ADA-SCID mice and highlight the advantage of vector designs that allow expression in both haematopoietic and liver parenchymal cells. As a second model for LV-mediated early postnatal in vivo gene transfer to HSPCs, we selected ARO. This condition requires early intervention to prevent disease progression, as BM failure in patients with osteopetrosis severely limits treatment options. The underlying genetic defect impairs osteoclast resorption capacity, preventing the formation of a properly structured bone matrix. This disruption hinders HSPC homing to the BM, resulting in an increased concentration of cHSPCs15,16. In this setting, we showed that in vivo LV gene transfer to HSPCs in osteopetrotic (OC) newborn mice extended postnatal survival and enabled the differentiation of gene-corrected cells into functional osteoclasts capable of bone resorption in vivo. However, given the severe and rapidly progressing phenotype of this model, complete rescue could not be achieved under the current experimental conditions (Extended Data Fig. For a detailed description of gene therapy outcomes in this model, see the Supplementary Notes. Last, we selected Fanconi anaemia, a rare genetic disorder belonging to the DNA repair deficiency syndromes. In Fanconi anaemia, the HSC compartment is directly affected, posing additional constraints to the collection and subsequent infusion of sufficient numbers of genetically corrected HSCs in patients with this disorder. Nevertheless, clinical trials have shown that even a small population of corrected HSCs can expand in vivo and repopulate the BM progressively, even without conditioning17,18. First, we analysed PB from Fanca knockout (Fanca−/−) mice over time, focusing on cHSC numbers (Extended Data Fig. As observed in the other analysed mouse strains, cHSC numbers were higher in newborns than older mice; however, the absolute number of cHSCs in newborn Fanca−/− mice was very low, mirroring patients with Fanconi anaemia (Fig. to Fanca−/− newborn mice and tracked the VCN over time in PB cells. The VCN progressively increased in all treated mice during physiological growth and homeostatic haematopoietic renewal up to 1 year after LV administration, demonstrating that gene-corrected Fanca−/− cells have a selective advantage over uncorrected Fanca−/− cells in vivo (Fig. Parallel analysis of PB parameters revealed that counts of white blood cells and lymphocytes normalized progressively starting at 30 weeks after LV administration. This suggests that the expansion of corrected HSCs eventually reached the threshold needed to restore PB parameters (Fig. a,b, Single VCN values (a) and VCN fold expansion (b; over 5 weeks) in PB of Fanca−/− mice treated as newborns (1.5 × 1010 TU kg−1, FANCA, LV, n = 5) or untreated (FANCA, untreated, n = 5). of white blood cell (WBC; c) and lymphocyte (d) counts in the PB of mice in a and from WT untreated mice. of VCN in the indicated organs 1 year after LV administration in mice in a. f,g, Single VCN values (f) and VCN fold expansion (g; over 4 weeks) in PB of Fanca−/− mice transplanted when 8 weeks old with BM cells from FANCA, LV (n = 7) or FANCA, untreated (n = 7) mice in a. LME model (Supplementary Table 7). of VCN in indicated organs 1 year after transplant in mice in f. i,j, Single VCN values (i) and VCN fold expansion (j; over 5 weeks) in PB of mice treated as newborns (1.5 × 1010 TU kg−1, n = 29) or untreated (n = 14). A subset of FANCA, LV (n = 15) or FANCA, untreated (n = 7) mice received two doses of MMC (arrows). of neutrophil count (k), red blood cell (RBC) count (l), haemoglobin concentration (HGB, m) and platelet count (n) in PB of mice from i. Kruskal–Wallis test followed by post hoc analysis (compared with FANCA, LV). At the end of the 1-year follow-up, we detected a high VCN in the spleen, thymus and BM of LV-treated Fanca−/− mice, except for one mouse with low PB marking (Fig. Sorting of PB subpopulations and KIT+SCA-1+Lin− cells from the BM confirmed a VCN consistent with PB measurements (Extended Data Fig. LV integration site analysis in these sorted PB and BM samples and whole BM, spleen and thymus confirmed transduction of bona fide HSCs that expanded owing to their selective advantage (Extended Data Fig. We then transplanted BM cells from LV-treated or untreated Fanca−/− mice into irradiated adult Fanca−/− recipients. VCN measurements in the organs of transplanted mice 1 year after transplant matched those observed in the PB, further supporting the stability of gene correction (Fig. As Fanca−/− mice have a milder disease phenotype compared with humans, we applied a mitomycin C (MMC) challenge to model Fanconi-anaemia-associated BM failure. MMC is a DNA cross-linking agent, which is toxic to Fanca−/− cells owing to their defective DNA damage response19,20. Interestingly, LV-treated Fanca−/− mice exposed to two MMC doses exhibited substantial expansion of corrected cells, with the highest measured VCN reaching 1 (Fig. By contrast, untreated Fanca−/− mice exposed to MMC experienced severe haematopoietic decline, mimicking the BM failure observed in patients with Fanconi anaemia. By contrast, LV-treated Fanca−/− mice underwent a rapid recovery of all these parameters (Fig. One year after LV administration, we measured the VCN in the BM, spleen and thymus of LV-treated Fanca−/− mice. We found that values again mirrored those measured in the PB and were the highest in MMC-challenged LV-treated Fanca−/− mice, reinforcing the idea that corrected HSCs expanded after selective pressure from MMC treatment (Fig. Finally, to explore strategies to further increase the number of corrected HSCs, we applied an HSPC mobilization regimen to 2-week-old or adult Fanca−/− mice (Fig. As expected, HSC gene-transfer efficiency in young and adult mice was much lower than in newborns. However, mobilization improved gene-transfer efficiency in both groups, with the highest VCN observed in 2-week-old mice (Fig. 4p), highlighting the potential of using mobilization in Fanconi anaemia to enhance gene-transfer efficiency and extend the intervention window. Overall, we achieved full rescue of the haematopoietic defects observed in Fanca−/− mice, leveraging the selective advantage of corrected cells and progressive expansion of the initial pool of in vivo transduced HSCs. Given that in vivo gene transfer to HSCs in newborn mice was facilitated by high levels of cHSPCs, we next examined whether a similar window of accessibility exists in humans. Using a previously published multiparametric flow-cytometry analysis21, we assessed PB samples from healthy participants at birth, 0–2 months, 3–8 months and when adults (>18 years). We observed a peak in phenotypically defined cHSCs and multipotent progenitor cells during the neonatal and early infantile periods, both in absolute numbers and in percentage of total HSPCs, with amounts progressively declining around weaning and becoming nearly undetectable in adults (Fig. These findings suggest that, as in mice, early postnatal life in humans may represent a unique and time-sensitive window for in vivo HSC gene therapy. By leveraging this stage, future therapeutic strategies could bypass ex vivo manipulations and enable direct, clinically relevant HSC correction in paediatric patients. HSC and MPP counts from some donors are also reported in a separate study providing a detailed biological characterization of cHSPCs24. Here we show that the early postnatal period provides a unique opportunity to access HSCs in vivo for LV gene transfer in mice, bypassing the hurdles associated with ex vivo engineering. Although the transduction rate and extent of genetically modified haematopoiesis remained below what can be achieved by ex vivo transduction and infusion after conditioning, we provide proof of concept of substantial therapeutic benefit achieved in three models of severe genetic diseases. Bona fide HSC gene transfer was proven by long-term multilineage output, self-renewal after serial transplantation and clonal tracking of in vivo transduced HSCs. Notably, the levels of genetically modified LV-positive cells in the haematopoietic compartment remained stable across all observation time points. Our investigation of the spatiotemporal dynamics of haematopoiesis in mice during postnatal development has provided insight into the biological mechanisms that underlie in vivo HSC gene transfer in newborns, with a substantial proportion of HSCs persisting in the liver and spleen of young mice, consistent with recent reports22,23. However, we also observed that the high numbers of cHSPCs seen in newborns, probably trafficking to newly forming definitive niches, rapidly declined in parallel with the efficiency of gene transfer. This suggests that LV access to HSCs is improved when they are in circulation. Supporting this hypothesis, we found that HSPC mobilization at 2 weeks of age, when significant numbers of HSPCs remain in tissue reservoirs, rescued effective gene transfer to levels exceeding those in non-mobilized newborns. These findings underscore the translational potential of this strategy, extending the window of efficacy to pre-weaning, a time point that may be more favourable for therapeutic intervention in humans. HSPC mobilization with a G-CSF and plerixafor regimen markedly increased gene-transfer efficiency not only in WT mice but also in a mouse model of Fanconi anaemia. However, physical accessibility alone may not fully account for LV gene-transfer efficiency. Cell-intrinsic factors, including LV receptor expression levels, metabolic activity and the proliferative state of circulating cells during early life, may also contribute to productive transduction. Even after mobilization, in vivo HSPC gene-transfer efficiency declined at increasing ages. It is well known that HSPCs collected from the circulation of mobilized individuals are poorly permissive to LV gene transfer and require ex vivo activation for robust transduction. Similarly, quiescent HSCs in their BM niche impose notable barriers to genetic engineering, whether mediated by gene transfer or gene editing. These obstacles include limited vector entry, restricted reverse transcription owing to unbalanced cellular dNTP pools, inefficient nuclear translocation and constraints in DNA accessibility, repair pathways and replication competence. The permissiveness to in vivo HSC gene transfer observed in early postnatal life might, therefore, extend beyond LV transduction, potentially informing strategies to enhance genetic-engineering approaches. Notably, the few previous reports of in vivo genetic modification of HSCs have relied on either HSPC mobilization followed by chemotherapy-based selection of the limited number of cells engineered using adenoviral vectors or on targeted lipid nanoparticles delivering limiting amounts of RNA cargo5,6. Our findings indicate that a permissive condition for LV gene transfer also exists in neonatal humans, probably extending into early infantile life, given the high numbers of cHSPCs, including the most primitive subsets24, and the well-established notion that human HSPCs isolated from cord blood exhibit greater permissiveness to LV-mediated transduction compared with those obtained from adult BM. Systemic LV administration has not yet been reported in humans, but we have previously demonstrated its safety and tolerability in non-human primates7,25. Notably, non-human primates and humans exhibit greater permissiveness to LV gene transfer and respond more effectively to phagocytosis-shielded CD47high-LV than do mice7,25. In a mouse model engineered to be sensitive to human CD47 signalling, we achieved up to 9% stable haematopoietic engineering without transduction enhancers or mobilization protocols. Interestingly, the estimated initial LV concentration in the PB after infusion also aligns with levels typically used for ex vivo HSPC transduction. Further increases in in vivo HSC gene transfer may be achieved through early-life mobilization and co-administration of clinically compatible transduction enhancers26,27,28, as shown here with IFNα receptor blockade. In Fanconi anaemia, we successfully exploited the selective advantage conferred by correction of the genetic defect in DNA repair, which specifically affects HSCs. This approach led to a remarkably high proportion of genetic correction in the haematopoietic compartment over the long term, albeit sustained by only a small number of expanded HSC clones. An in vivo gene therapy approach would extend the application of treatment to patients with limited HSC content while avoiding ex vivo manipulation of these fragile cells. For ADA-SCID and ARO outcomes see Supplementary Discussion. The well-established clinical use of LVs, the advantages of the vector design used here for in vivo delivery and the increasing availability of newborn screening for monogenic diseases provide a strong rationale for progressing towards the first clinical testing of this approach. By bypassing the major barriers associated with ex vivo HSC gene therapy, this strategy stands to offer broader accessibility to life-changing treatments. Sample size in experiments with mice was chosen according to previous experience with experimental models and assays. No sample or animal was excluded from the analyses. Mice were randomly assigned to each experimental group. Investigators were blinded at the moment of LV administration to newborn ADA-SCID or OC mice. In all the other experimental settings investigators were not blinded. Plasmid pCCLsin.cPPT.ET.ADA.142T was constructed by standard cloning techniques, by exchanging the human FIX cDNA7 with human ADA cDNA (SpeI–SpeI) into the previously described pCCLsin.cPPT.ET.coFIX-R338L.142T (NheI–SalI) after blunting the ends. Plasmid pCCLsin.cPPT.PGK.ADA.122T was constructed by standard cloning techniques, by inserting the miRNA-122 target sequence into the pCCLsin.cPPT.PGK.ADA plasmid (SalI–ScaI). Third-generation self-inactivating LVs were produced by calcium phosphate transient transfection into 293T cells. 293T cells were transfected with a solution containing a mix of the selected LV genome-transfer plasmid, the packaging plasmids pMDLg-pRRE and pCMV.REV, pMD2.G and pAdVantage (Promega), as previously described7. In brief, the medium was changed 14–16 h after transfection and the supernatant was collected 30 h after medium change. LV-containing supernatants were sterilized through a 0.22-μm filter (Millipore), pre-concentrated four times with a Vivaflow 200 Tangential Flow Filtration Cassette 100 kDa (Sartorius), according to manufacturing instructions, transferred into sterile polyallomer tubes (Beckman) and centrifuged at 20,000g for 120 min at 20 °C (Beckman Optima XL-100K Ultracentrifuge). For LV titration, 1 × 105 293T cells were transduced with serial LV dilutions in the presence of polybrene (8 μg ml−1). For PGK.GFP-LV, cells were analysed by flow cytometry using a FACSCanto analyser (BD Biosciences), equipped with DIVA software, 5–7 days after transduction and the infectious titre, expressed as transducing units (TU) ml−1, was calculated using the formula TU ml−1 = ((% GFP+ cells/100) × 100,000×(1/dilution factor)). For all other LVs, genomic DNA (gDNA) was extracted 14 days after transduction, using the Maxwell 48 Cell DNA Purification Kit (Promega), following the manufacturer's instructions. The VCN was determined by ddPCR, starting from 5–20 ng of template gDNA using primers (HIV FW: 5′-TACTGACGCTCTCGCACC-3′; HIV REV: 5′-TCTCGACGCAGGACTCG-3′) and a probe (FAM 5′-ATCTCTCTCCTTCTAGCCTC-3′) designed against the primer-binding region of LV. The amount of endogenous DNA was quantified by a primers–probe set designed against the human telomerase gene (Telo FW: 5′-GGCACACGTGGCTTTTCG-3′; Telo REV: 5′-GGTGAACCTCGTAAGTTTATGCAA-3′; Telo probe: VIC 5′-TCAGGACGTCGAGTGGACACGGTG-3′ TAMRA) or the human GAPDH gene (Applied Biosystems HS00483111_cm). The PCR reaction was performed with each primer (900 nM) and the probe (250 nM; 500 nM for Telo) following the manufacturer's instructions (Bio-Rad), read with a QX200 reader and analysed with QuantaSoft software (Bio-Rad). The infectious titre, expressed as TU ml−1, was calculated using the formula TU ml−1 = (VCN × 100,000×(1/dilution factor). LV physical particles were measured by HIV-1 Gag p24 antigen immunocapture assay (Perkin Elmer) following the manufacturer's instructions. LV-specific infectivity was calculated as the ratio between infectious titre and physical particles. For mouse experiments, DNA was extracted from the whole liver using the Maxwell 48 Tissue DNA Purification Kit (Promega), from a cell suspension obtained from the thymus, spleen or BM using the Maxwell 48 Cell DNA Purification Kit (Promega) or from whole PB using the Maxwell 48 Whole Blood DNA Purification Kit (Promega). DNA was extracted from lineage-negative (Lin−) cultured cells using the Maxwell 48 Cell DNA Purification Kit (Promega) and from single-picked colonies using Quick extract DNA (Biosearch Technologies), according to cell number. The VCN in mouse DNA was determined by ddPCR, starting from 5–20 ng of template gDNA using a primers–probe set designed against the primer-binding region of LVs (see ‘LV titration' section). In some cases, the VCN was determined using an ad hoc ddPCR (QX200 EvaGreen Digital PCR Supermix, Bio-Rad), which selectively amplifies the reverse-transcribed vector genome (both integrated and non-integrated), discriminating it from possible plasmid contamination29 (RT-LV; ΔU3 FW: 5′-TCACTCCCAACGAAGACAAGATC-3′, gag REV: 5′-GAGTCCTGCGTCGAGAGAG-3′). The amount of endogenous mouse DNA was quantified by a primers–probe set designed against the mouse Sema3a gene (Sema3a FW: 5′-ACCGATTCCAGATGATTGGC-3′; Sema3a REV: 5′-TCCATATTAATGCAGTGCTTGC-3′; Sema3a probe: HEX 5′-AGAGGCCTGTCCTGCAGCTCATGG-3′ BHQ1). The PCR reaction was performed with each primer (900 nM; 150 nM for RT-LV primers) and the probe (250 nM) following the manufacturer's instructions (Bio-Rad), read with a QX200 reader and analysed with QuantaSoft software (Bio-Rad). RNA extraction was performed using a Maxwell 48 SimplyRNA Blood extraction kit (Promega, AS1380), according to the manufacturer's instructions and reverse transcribed using the SuperScript IV VILO kit (11766050; ThermoFisher Scientific). LV gene expression was assessed by ddPCR, starting from 10–25 ng of template cDNA using a primers–probe set designed against the Woodchuck hepatitis virus post-transcriptional regulatory element (Wpre) sequence of the LV genome (Wpre FW: 5′-GGCTGTTGGGCACTGACAAT-3′; Wpre REV: 5′-ACGTCCCGCGCAGAATC-3′; Wpre probe: FAM 5′-TTTCCATGGCTGCTCGCCTGTGT-3′ MGB). Hprt was used as a reference gene for mouse samples (dMmuCPE5095493, Bio-Rad). The PCR reaction was performed with each primer (900 nM) and the probe (250 nM) following the manufacturer's instructions (Bio-Rad), read with a QX200 reader and analysed with QuantaSoft software (Bio-Rad). The 293T cell line was maintained in Iscove's modified Dulbecco's medium (IMDM, Sigma) supplemented with 10% fetal bovine serum (FBS; FetalClone II, HyClone, Euroclone), 4 mM glutamine (Lonza), 100 international units (IU) ml−1 penicillin and streptomycin (Lonza). Primary mouse Lin− cells were FACS-sorted from cell suspensions obtained from the BM or the liver of LV-treated mice and cultured in StemSpan supplemented with 10% FBS, 4 mM glutamine, 100 IU ml−1 penicillin and streptomycin, 100 ng ml−1 mouse stem-cell factor (mSCF, Peprotech), 50 ng ml−1 mouse thrombopoietin (mTPO, Peprotech), 100 ng ml−1 mouse fms-like tyrosine kinase 3 ligand (mFLT3L, Peprotech) and 20 ng ml−1 mouse interleukin-3 (mIL-3, Peprotech) for 24 h. The medium was then exchanged with IMDM supplemented with 10% FBS, 100 IU ml−1 penicillin and streptomycin, 100 ng ml−1 mSCF (Peprotech) and mFLT3L 100 ng ml−1 (Peprotech) for 48 h. Finally, GFP expression was determined using a FACSCanto analyser (BD Biosciences), equipped with DIVA software. All cells were maintained in a 5% CO2 humidified atmosphere at 37 °C. Cell lines were routinely tested for mycoplasma contamination. Founder FVB,129-Adatm1Mw Tg(PLADA)4118Rkmb/J mice (referred to as ADA-SCID mice) were obtained from The Jackson Laboratory31 (stock 003265). Founder B6C3Fe a/a-Tcirg1oc/J mice (referred to as OC mice) were obtained from The Jackson Laboratory16 (stock 00230) and fully backcrossed on the C57BL/6 background. Founder mice with a targeted disruption in the Fanca gene32 were provided by H. J. van de Vrugt (Free University Medical Center) and backcrossed with FVB mice to obtain the Fanca−/− FVB strain (referred to as Fanca−/− mice). C57BL/6 and NSG mice were purchased from Charles River Laboratories. All mice were maintained in specific-pathogen-free conditions. Genotyping of ADA-SCID and OC mice was performed at 2–3 weeks of age according to the protocol available on The Jackson Laboratory website. Vector administration was carried out in adult female C57BL/6 or NSG (7–10-week-old) mice by tail-vein injection (250–300 µl per mouse), in newborns by temporal-vein injection (30 µl per mouse) and in 2-week-old mice by retro-orbital injection (100–150 µl per mouse). Mice were bled from the retro-orbital plexus using capillary tubes and blood was collected in 0.38% sodium citrate buffer, pH 7.4 or in an EDTA-coated tube (Microvette 20.1341). For transplantation experiments, 7-week-old female C57BL/6 recipient mice were conditioned with a lethal regimen of busulfan (6 mg ml−1), consisting of four daily intraperitoneal (i.p.) injection of total BM cells or total CD45+ cells FACS-sorted (BD FACSAria Fusion Cell Sorter, BD Biosciences or MoFlo Astrios EQ Cell Sorter, Beckman Coulter) from the liver or the BM of donor mice the day after the last dose of busulfan. Cells were resuspended in 200 µl of PBS. The weight of recipient mice was monitored daily during busulfan administration and weekly for 1 month after transplant. In case of transplantation experiments in NSG mice, conditioning was performed by sublethal irradiation of recipient mice (200 rad) and transplantation 3–4 h after irradiation. For the transplantation experiments in Fanca−/− mice, conditioning was performed by lethal irradiation of recipient mice using two doses of 4.5 Gy 24 h apart. A total of 3–4 h after conditioning, 4–5 × 106 BM cells from LV-treated or untreated Fanca−/− mice were transplanted by i.v. injection in the tail vein of recipient mice. HSPC mobilization was performed in 2-week-old or adult mice by administering five i.p. doses of 250 ng g−1 G-CSF per mouse 12 h apart. Then 8 h after the last dose of G-CSF, 2 µg g−1 plerixafor per mouse was administered i.p. 2 h after plerixafor and, at the same time, haematopoietic cells were collected from the BM, PB or spleen for FACS analysis. HSPC mobilization was performed in newborn mice following the same protocol used in 2-week-old or adult mice, except for a reduced G-CSF regimen that consisted of three instead of five doses. to newborn mice 3 h before LV administration at 50 mg kg−1 dose. BM failure was induced in Fanca−/− mice by i.p. Mice were euthanized by CO2 inhalation at the scheduled times. Experimental procedures in Fanca−/− mice were conducted according to European and Spanish regulations (European convention ETS 123, regarding the use and protection of vertebrate mammals used in experimentation and other scientific purposes, Directive 2010/63/UE, Spanish Law 6/2013 and Real Decreto (R.D.) 53/2013 regarding the protection and use of animals in scientific research). All animal procedures were performed according to protocols approved by the Institutional Animal Care and Use Committee. The mouse liver was perfused (15 ml min−1) through the inferior vena cava with 12.5 ml of the following solutions at subsequent steps: (1) PBS EDTA (0.5 mM); (2) HBSS (Gibco) and HEPES (10 mM); and (3) HBSS and HEPES containing 0.03% collagenase IV (Sigma). The digested mouse liver tissue was collected, passed through a 100-μm cell strainer (BD Biosciences) and processed into a single-cell suspension. This suspension was subsequently centrifuged three times (30g, 25g and 20g, for 3 min, at room temperature) to remove hepatocyte-containing pellets. The supernatant containing non-parenchymal cells was centrifuged (650g, 7 min, at room temperature) and recovered cells were subsequently incubated with the monoclonal antibodies indicated in the ‘Flow cytometry' section. To obtain a single-cell suspension of spleen or thymus, organs were smashed on a 40-µm filter using PBS–2% FBS (FetalClone II, HyClone, Euroclone), red blood cells (RBCs) were lysed with 0.5 or 1 ml of ACK Lysing buffer (GIBCO) for 5 or 10 min on ice, respectively. The lysis was stopped by adding 10 ml of PBS–2% FBS. The cell suspension was then filtered through a 40-µm filter and cells were counted and used for gDNA extraction or flow-cytometry analysis (see ‘Flow cytometry' section). BM cells were flushed from the tibia and femur of one or two legs, as needed, or extracted by bone crushing in the case of newborn or OC mice and then processed as described for cells isolated from the spleen. Flow-cytometry analyses using cell suspensions obtained from BM, spleen, thymus, PB or liver were performed using a FACSCanto analyser (BD Biosciences), equipped with DIVA software. Between 500,000 and 1,000,000 cells were processed, washed with PBS–2% FBS (FetalClone II, HyClone, Euroclone) or MACS buffer (PBS pH 7.2, 0.5% bovine serum albumin (BSA), 2 mM EDTA), treated with fragment crystallizable (Fc) Receptor-Block (Miltenyi Biotec) when antibody-stained and then resuspended in the buffer used for washing. Right before analysis, 3 µl of 7-AAD (Biolegend 420404) was added to each tube to exclude dead cells. Rainbow beads (BD Biosciences) were used to calibrate the instrument detectors. Gating strategies are provided in Supplementary Figs. A BD FACSAria Fusion Cell Sorter (BD Biosciences), equipped with DIVA software (BD Biosciences), or a MoFlo Astrios EQ Cell Sorter (Beckman Coulter), equipped with Summit software (Beckman Coulter), was used for cell sorting. For PB samples collected from healthy human participants of different ages and mouse samples shown in Fig. 1a–d, samples were analysed using two standardized multiparametric flow-cytometry assays21,33 (through whole blood dissection). In brief, after RBC lysis with ACK lysing buffer (STEM CELL Technologies), samples were labelled with the fluorescent antibodies reported in Supplementary Table 16. Titration assays were performed to assess the best antibody concentration. Samples from mice were pre-incubated with a rat anti-mouse FcR blocking reagent (BD, dilution 1:100). After surface marking, cells were incubated with propidium iodide (PI; BioLegend) to stain dead cells. Absolute cell quantification was performed by adding Precision count beads (BioLegend) to PB samples before the staining procedure. Images of all stained samples were acquired through a BD Symphony A5 (BD Bioscience) cytofluorimeter after calibration with Rainbow beads (Spherotech) and raw data were collected using the DIVA software (BD Biosciences). Data were subsequently analysed with FlowJo software v.10.5.3 (BD Biosciences). A list of antibodies used for mouse or human whole-blood-dissection panels is reported in Supplementary Table 16. PB samples from paediatric and adult healthy donors were collected and analysed after written informed consent was given, using protocols approved by the ethics committees of IRCCS Ospedale San Raffaele (Protocol_TIGET09 for both paediatric and adult participants). Written consent for paediatric participants was given by their parents. Mice were perfused through the left ventricle with PBS containing 0.5 mM EDTA to remove blood, after which organs were collected and fixed in zinc-formalin for at least 24 h. Embedding, sectioning, slide preparation, staining and image acquisition were performed at the Centro di Imaging Sperimentale facility at San Raffaele Hospital (Italy). Formalin-fixed, paraffin-embedded consecutive sections (4 µm) were dewaxed and rehydrated through a graded alcohol series before histological or immunohistochemical staining. Tissues were first deparaffinized and subjected to epitope retrieval using Epitope Retrieval Solution 1 (ER1 Citrate Buffer). Slides were incubated with either an anti-GFP primary antibody (Invitrogen, A11122, 1:1,000) or an anti-Ki-67 primary antibody (D3B5, Cell Signaling Technology, 1:200) for 30 min at room temperature, followed by detection using the Bond Polymer Refine Detection Kit (Leica, DS9800). Images of stained slides were acquired using an Aperio AT2 digital scanner at 20× magnification (Leica Biosystems) and analysed with ImageScope software (Leica Biosystems). Integration sites were retrieved by Sonication Linker Mediated (SLiM)-PCR, as previously described34, with minor modifications. In brief, for each sample up to 300 ng of gDNA was processed by DNA shearing using the Covaris E220 Ultrasonicator, generating fragments with an average size of 1,000 bp. The fragmented DNA was subjected to end repair and 3′ adenylation, and then ligated to linker cassettes containing an eight-nucleotide sequence barcode used for sample identification and all the sequences required for Read 2 paired-end sequencing. The ligation products were split in three technical replicates and subjected to 35 cycles of exponential PCR using primers specific for the LV LTR and the linker cassette. A subsequent amplification with an additional ten PCR cycles was performed using a primer specific for the linker cassette and the LTR. These primers contain an 8-nucleotide barcode used for sample identification (coupled with the barcode on the linker cassette) and the sequences needed for Read 1 sequencing, plus a random 12-nucleotide sequence, enabling easier cluster recognition in the first sequencing cycles on the next-generation sequencing system. The generated SLiM-PCR products are therefore associated with a unique pair of barcodes, assembled into libraries and subjected to Illumina next-generation sequencing. Sequencing reads were processed using a custom bioinformatics pipeline35 (VISPA2) that isolates genomic sequences flanking the vector LTR and maps them to the mouse genome (mm9). Because vector integration in the same genomic position in different cells is a very low probability event, identical integration sites in independent samples were considered contamination or amplification artefacts, which may occur during the technical procedure. Datasets were pruned of potential contamination and false positives for each primary mouse and of integration sites derived from unrelated secondary mice. Identical integration sites shared between mice belonging to different experimental groups were reassigned by identifying the insertion site in at least two SLiM technical replicates and through sequence counts. Downstream analyses of vector-integration sites, such as relative abundance, sharing and CIS analysis, were performed using ISAnalytics36. Clonal abundance, estimated as the relative percentage of genome numbers, was determined by the R package sonicLength37. Common insertion sites were calculated using the Grubbs test for outliers38. Haemocytometric analysis of PB was performed using a Procyte DX Analyser (IDEXX) at the time points indicated in the experimental design schemes. Approximately 30 µl of PB was collected into EDTA-coated tubes (Microvette 20.1341) and analysed. ADA activity was measured in plasma or the cellular fraction of PB as previously described39. The cell lysate was centrifuged at 12,000g for 5 min. The enzyme activity was evaluated in a mixture containing 50 mmol l−1 Tris (pH 7.2) and 0.4 mmol l−1 adenosine by adding 10 μl of lysed RBC supernatant or 20 μl of plasma. The reaction was stopped by adding PCA (final concentration, 0.21 mmol l−1) and neutralized with KOH (final concentration, 0.22 mmol l−1). The reaction progression was analysed using aliquots at two time points with a Bio-Rad Biofocus 3000 instrument (Bio-Rad) after inosine plus hypoxanthine formation at 0 and 10 min for the cellular fraction or at 0 and 30 min for plasma. Osteoclasts were differentiated from BM cells as previously described40 and assayed for resorption functions on dentine slides (AE8050). Dentine discs were stained with toluidine blue as previously described40. The concentration of released CTX in the culture supernatant was measured by enzyme-linked immunosorbent assay (ELISA; CrossLaps, AC-07F1, Immunodiagnostic System) following the manufacturer's protocol. The concentration of parathyroid hormone in the serum of LV-treated OC or WT mice was measured by ELISA (MicroVue Mouse parathyroid hormone 1-84 EIA) following the manufacturer's protocol. For micro-computed tomography analyses, vertebral columns of OC mice and their WT littermates were dissected, cleaned from connective tissues and fixed in 4% paraformaldehyde. The bone mineral density of the trabecular region was calculated by calibrating a linear extrapolation against the mean attenuation coefficients of selected regions of a 2-mm phantom rod pair containing 0.25 and 0.75 g cm−3 CaHA (Skyscan). A three-dimensional morphometric analysis of the trabecular region was performed after binarization of the trabeculae by an adaptive thresholding algorithm. Statistical analyses were performed after consulting with professional statisticians at the San Raffaele University Center for Statistics in the Biomedical Sciences (CUSSB). A two-tailed Mann–Whitney U-test was performed to compare two independent groups and for more than two independent groups, a Kruskal–Wallis test was used, followed by post hoc analysis (Dunn's test for multiple comparisons against the reference control group along with Bonferroni's correction). For paired observations, a Wilcoxon matched-pairs test was performed. To model the dynamics of biological processes, properly accounting for dependencies induced by the repeated-measures design, LME models were estimated, specifying random-effect terms for experimental units or session. To fulfil underlying model assumptions, various transformations of the outcome variable, including logarithmic, square/cubic root and ordered quantile normalization, were considered. After the estimation of LME models, a post hoc analysis was conducted, enabling the pairwise comparison of treatment groups at a fixed time point. To account for multiplicity issues, P values were adjusted using Bonferroni's approach. LME were estimated using R software (v.4.3.1). Inferential techniques were used for adequate sample sizes (n ≥ 5), otherwise only descriptive statistics are reported. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. & Aiuti, A. Gene therapy using haematopoietic stem and progenitor cells. Ferrari, S. et al. Genetic engineering meets hematopoietic stem cell biology for next-generation gene therapy. Ferrari, S. & Naldini, L. A step toward stem cell engineering in vivo. In vivo transduction of primitive mobilized hematopoietic stem cells after intravenous injection of integrating adenovirus vectors. In vivo hematopoietic stem cell modification by mRNA delivery. Milani, M. et al. Phagocytosis-shielded lentiviral vectors improve liver gene therapy in nonhuman primates. Milani, M. et al. GP64-pseudotyped lentiviral vectors target liver endothelial cells and correct hemophilia A mice. Takenaka, K. et al. Polymorphism in Sirpa modulates engraftment of human hematopoietic stem cells. Aiuti, A. et al. Multilineage hematopoietic reconstitution without clonal selection in ADA-SCID patients treated with stem cell gene therapy. Ferrua, F. & Aiuti, A. Twenty-five years of gene therapy for ADA-SCID: from bubble babies to an approved drug. Cantore, A. et al. Hyperfunctional coagulation factor IX improves the efficacy of gene therapy in hemophilic mice. Carbonaro, D. A. et al. Gene therapy/bone marrow transplantation in ADA-deficient mice: roles of enzyme-replacement therapy and cytoreduction. Mortellaro, A. et al. Ex vivo gene therapy with lentiviral vectors rescues adenosine deaminase (ADA)-deficient mice and corrects their immune and metabolic defects. Capo, V., Abinun, M. & Villa, A. Osteoclast rich osteopetrosis due to defects in the TCIRG1 gene. Successful engraftment of gene-corrected hematopoietic stem cells in non-conditioned patients with Fanconi anemia. Rio, P. et al. Haematopoietic gene therapy of non-conditioned patients with Fanconi anaemia-A: results from open-label phase 1/2 (FANCOLEN-1) and long-term clinical trials. In vitro phenotypic correction of hematopoietic progenitors from Fanconi anemia group A knockout mice. A. et al. Upregulation of NKG2D ligands impairs hematopoietic stem cell function in Fanconi anemia. Basso-Ricci, L. et al. Multiparametric whole blood dissection: a one-shot comprehensive picture of the human hematopoietic system. Li, Y. et al. Single-cell analysis of neonatal HSC ontogeny reveals gradual and uncoordinated transcriptional reprogramming that begins before birth. Collins, A. et al. Maternal inflammation regulates fetal emergency myelopoiesis. Milani, M. et al. Liver-directed lentiviral gene therapy corrects hemophilia A mice and achieves normal-range factor VIII activity in non-human primates. Santoni de Sio, F. R., Cascio, P., Zingale, A., Gasparini, M. & Naldini, L. Proteasome activity restricts lentiviral gene transfer into hematopoietic stem cells and is down-regulated by cytokines that enhance transduction. Santoni de Sio, F. R. et al. Lentiviral vector gene transfer is limited by the proteasome at postentry steps in various types of stem cells. In vivo administration of lentiviral vectors triggers a type I interferon response that restricts hepatocyte gene transfer and promotes vector clearance. Matrai, J. et al. Hepatocyte-targeted expression by integrase-defective lentiviral vectors induces antigen-specific tolerance in mice with low genotoxic risk. Expanded circulating hematopoietic stem/progenitor cells as novel cell source for the treatment of TCIRG1 osteopetrosis. Blackburn, M. R., Datta, S. K. & Kellems, R. E. Adenosine deaminase-deficient mice generated using a two-stage genetic engineering strategy exhibit a combined immunodeficiency. Cheng, N. C. et al. Mice with a targeted disruption of the Fanconi anemia homolog Fanca. Colantuoni, M. et al. Constitutive IL-1RA production by modified immune cells protects against IL-1-mediated inflammatory disorders. Cesana, D. et al. Retrieval of vector integration sites from cell-free DNA. Spinozzi, G. et al. VISPA2: a scalable pipeline for high-throughput identification and annotation of vector integration sites. Pais, G. et al. ISAnalytics enables longitudinal and high-throughput clonal tracking studies in hematopoietic stem cell gene therapy applications. Berry, C. C. et al. Estimating abundances of retroviral insertion sites from DNA fragment length data. Biffi, A. et al. Lentiviral vector common integration sites in preclinical models and a clinical trial reflect a benign integration bias and not oncogenic selection. Carlucci, F. et al. Capillary electrophoresis in diagnosis and monitoring of adenosine deaminase deficiency. We thank A. Fiocchi and the Experimental Imaging Center (San Raffaele Scientific Institute) for imaging techniques and analysis, all the members of the Cantore laboratory for helpful discussions and L. Martínez de la Cal, L. García and M. Angel Martín for their technical support in the experiments performed with the Fanca−/− mouse model. conducted this study as partial fulfilment of his International PhD Course in Molecular Medicine, Cellular and Molecular Biology Programme at the San Raffaele University. This work was supported by Fondazione Telethon SR-Tiget Core Grant (TTACC0422TT) to A.C., Fondo Italiano per la Scienza (FIS 2021 Advanced Grant FIS00002157) to L.N., Fondazione Telethon SR-Tiget Core Grant E2 (TTAVC0522TT) to A.V., Fondazione Telethon SR-Tiget Core Grant (Tgt2016) to A.A. and S.S. and by Fondazione CARIPLO Giovani Ricercatori (GR-2019-12369499) to S.S. A.A. is the recipient of the Else Kröner Fresenius Prize for Medical Research 2020. are supported by Instituto de Salud Carlos III (ISCIII)/Red Española de Terapias Avanzadas RICORS/TERAV and by the European Union NextGenerationEU (RD21/0017/0027) Plan de Recuperación Transformación y Resiliencia. receives funding and M.P.-R. is supported by the PID2021-125077OB-C21 grant from Ministry of Science, Innovation and Universities. These authors jointly supervised this work: Alessio Cantore, Luigi Naldini San Raffaele Telethon Institute for Gene Therapy, IRCCS San Raffaele Scientific Institute, Milan, Italy Michela Milani, Anna Fabiano, Raisa Jofra Hernandez, Alessandra Zecchillo, Erika Zonari, Sofia Ottonello, Luca Basso-Ricci, Cesare Canepari, Monica Volpin, Valeria Iannello, Valentina Capo, Pamela Quaranta, Luca Seffin, Fabio Russo, Mauro Biffi, Leonardo Ormoli, Eugenio Montini, Serena Scala, Anna Villa, Alessandro Aiuti, Alessio Cantore & Luigi Naldini Biomedical Innovation Unit, Center for Research on Energy, Environment and Technology (CIEMAT), Madrid, Spain Marta Perez-Rodriguez, Juan Antonio Bueren & Paula Rio Biomedical Network Research Center for Rare Diseases (CIBERER), Madrid, Spain Marta Perez-Rodriguez, Juan Antonio Bueren & Paula Rio Sanitary Research Institute Fundación Jiménez Díaz (U.A.M), Madrid, Spain Marta Perez-Rodriguez, Juan Antonio Bueren & Paula Rio Translational and Molecular Medicine (DIMET), University of Milano Bicocca, Milan, Italy Vita-Salute San Raffaele University, Milan, Italy Sofia Ottonello, Valeria Iannello, Pamela Quaranta, Luca Seffin, Alessandro Aiuti, Alessio Cantore & Luigi Naldini You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar M.P.-R. performed experiments and analysed data related to Fanconi anaemia. designed experiments, provided intellectual input and edited the manuscript. performed experimental work related to integration-site analysis. performed micro-computed tomography scans and image analysis. A. Forlino analysed micro-computed tomography images and interpreted data. E.M. supervised work related to integration-site analysis. S.S supervised work related to cHSPCs, provided intellectual input and edited the manuscript. supervised ARO work, provided intellectual input and edited the manuscript. supervised Fanconi anaemia work, provided intellectual input and edited the manuscript. A.A. supervised ADA-SCID work, provided intellectual input and edited the manuscript. A.C. supervised research, interpreted data and wrote the manuscript. supervised research, interpreted data and wrote the manuscript. Correspondence to Michela Milani or Luigi Naldini. receive funding and have licensed the PGK.FANCA-LV to Rocket Pharmaceuticals and are inventors on patents filed by CIEMAT, CIBERER and Fundación Jiménez Díaz, and may be entitled to receive financial benefits from the licensing of such patents. is consultant of Rocket Pharmaceuticals, A.A. is the principal investigator of clinical trials with LV-transduced haematopoietic stem cells sponsored by Orchard Therapeutics or Fondazione Telethon. and A.C. are inventors on patent applications submitted by Fondazione Telethon or the San Raffaele Scientific Institute on LV technology for in vivo use related to the work presented in this paper. The other authors declare no competing interests. Nature thanks Jose-Carlos Segovia and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-d, Mean with SEM of the percentage of different populations among CD45-positive cells in the BM (a), PB (b), spleen (c), or liver (d) of newborn, 1-week-old, 2-week-old, or 8-week-old (adult) C57Bl/6 mice, as indicated. Newborn PB samples represent 10 pools derived from 50 mice. e, Representative immunohistochemistry image (out of 5 collected) of a liver section from a 1-week-old C57Bl/6 mouse stained with an anti-Ki67 (brown) antibody. Black arrowheads indicate proliferating hematopoietic cells; the red arrowhead indicates a representative megakaryocyte. f, Single VCN values measured in the indicated spleen populations or whole BM from the pooled donor mice shown in Fig. g, Body weights of individual busulfan-conditioned adult mice that were transplanted (TPX n = 5) or not transplanted (not-TPX, n = 1). h, Single values of CD45-positive GFP-positive cells in the indicated organs of recipient mice 20 wks post-transplant. VCN values are indicated above the bars. i-l, LV integration site (IS) analysis of GFP-positive cells from LV-treated donor mice and transplanted recipient mice at the study endpoint. Clones with an abundance of <1% are shown in gray. m, Common integration site analysis of LV-treated and transplanted mice, showing the distribution of integration events relative to known oncogenes. n, Mean with SEM of GFP-positive cells in PBMCs of LV-treated mice as newborns (n = 4) and monitored for 1 year after LV administration (5e10 TU/kg). o, Mean with SEM of GFP-positive cells in PBMCs of LV-treated mice as adults (n = 5, 8-week-old) via i.v. p, Representative immunofluorescence image of the liver from a mouse shown in (o), revealing few-to-no GFP-positive cells. a,b, Mean with SEM of GFP-positive cells (a) or VCN (b) measured in the PB of C57Bl/6 mice administered LV (LV, n = 15; 3e10 TU/kg) at 2 weeks-of-age or untreated (Unt, n = 1). c, Single values and mean with SEM of the VCN measured in the indicated organs in Unt and LV mice, 16 weeks after LV administration. d, Single values and mean with SEM of the absolute count of HSCs in the spleen of 2-week-old mobilized (n = 6) or untreated control (n = 7) mice, as indicated. e,f, Single values and mean with SEM of GFP-positive cells in the thymus (e) or spleen (f) of mobilized (n = 6) or control (n = 7) mice, 20 wks after LV administration. Mice were transplanted with 15x106 total BM cells harvested from donor mice, 20 wks after LV administration. i,j, Single values and mean with SEM of the VCN (i) or engraftment of CD45.2 cells (j) measured in CD45.1 recipient mice, 16 wks after transplant in the indicated organs. k,l, Single values and mean with SEM of GFP-positive cells in the thymus (k), or spleen (l) of CD45.1 recipient mice, 16 wks after transplant. a,b, Mean with SEM of GFP-positive cells (a) or the VCN (b) measured over time in the PB of C57Bl/6 mice treated as newborns with MAR-1 (i.v., 50 mg/kg, n = 13) or an IgG control (i.v., 50 mg/kg, n = 13) 3 h before LV administration (8e10 TU/kg). Mann-Whitney test on the last time point analyzed. In (a), comparisons were made among CD45-positive cells. Data from a pool of two independent experiments are shown. c-e, Single values and mean with SEM of GFP-positive cells in the spleen (c), thymus (d), and BM (e) of the mice shown in (a,b) 16 wks after LV administration. f, Single values and mean with SEM of the VCN measured in the indicated organs, 16 wks after LV administration in the mice shown in (a-e). a, Experimental scheme: hematopoietic cells were collected from the bone marrow (BM), spleen, liver, or peripheral blood (PB) of newborn, 1-week-old, 2-week-old, or adult NSG mice and analyzed by FACS to evaluate the composition of total CD45-positive cells. Newborn PB samples from 40 mice were pooled into eight groups. b-e, Mean with SEM of the percentage of different populations among lineage-negative (Lin-) and cKit-positive (cKit+) cells in the BM (b), PB (c), spleen (d), or liver (e) of the indicated NSG mice. Newborn PB samples from 40 mice were pooled into eight groups. f-i, Single values and mean with SEM of the absolute count of HSCs in the BM (f), PB (g), spleen (h), or liver (i) of the indicated NSG mice. Newborn PB samples from 40 mice were pooled into eight groups. j, Mean with SEM of the count of HSPC red or white colonies grown from GFP-positive (GFPpos) or GFP-negative (GFPneg) Lin- cells isolated from the BM of the mice in Fig. k, Single VCN values measured in FACS-sorted GFPpos or GFPneg Lin- cells isolated from mice shown in Fig. 2m,n and cultured for 72 h. l, Single VCN values measured from single-picked red (RFC) or white (WFC) colonies grown in MethoCult from GFPpos or GFPneg FACS-sorted Lin- cells isolated from the mice shown in Fig. m, Single values and mean with SEM of the GFP-positive area measured by immunohistochemistry in the indicated tissues, 20 wks after LV administration in NSG mice treated as newborns (n = 6) or adults (n = 4) with CD47hi-LV (shown in Fig. n, Representative immunohistochemistry images of the indicated tissues taken from the mice shown in (m). GFP-positive cells are shown in brown (black arrowheads). Schematics in a created with BioRender (https://biorender.com). a-c, Single values and mean with SEM of the absolute HSC counts in the BM (a), PB (b), or spleen (c) of 2-week-old or adult mobilized (n = 5 per group) or untreated (n = 2 per group) NSG mice, as indicated. d,e, Single values and mean with SEM of GFP-positive cells measured in the indicated populations in the spleen (d) or BM (e) of the mice shown in Fig. Kruskal-Wallis test with Dunn's multiple comparisons (comparison to the 2-week-old control group). Kruskal-Wallis test with Dunn's multiple comparisons (comparison to the respective 2-week-old control groups). b,c, Mean with SEM of neutrophil (b) or monocyte (c) counts measured by hemocytometer in the PB of ADA-SCID mice treated with the indicated LV as newborns. LME model (only comparisons vs. WT and p-values < 0.05 are reported; complete analyses in Supplementary Tables 10, 11). d,e, Single values and mean with SEM of ADA activity measured over time in the PB cellular fraction (d) or plasma (e) from ADA-SCID or wild-type (WT) mice treated with the indicated LV or left untreated. LME model (only comparisons vs. WT and p-values < 0.05 are reported; complete analyses in Supplementary Tables 12, 13). f, Single values and mean with SEM of LV-RNA expression in the indicated organs or liver subpopulations 8 wks after administration of the indicated LV to newborn ADA-SCID mice (n = 5 per group, 2e10 TU/kg). Wpre: Woodchuck hepatitis virus post-transcriptional regulatory element, used to measure LV genome expression. h, Single values and mean with SEM of the engraftment of cells isolated from ADA-SCID or WT mice treated with the indicated LV as newborns and used as donors for irradiated NSG mice. LV-treated WT mice have been pooled, irrespective of the administered LV. i,j, Mean with SEM of the percentage of the indicated populations among CD45-positive cells in the PB of transplanted NSG mice, 8 (i) or 12 wks (j) after transplant. k, Mean with SEM of the VCN measured in the PB of NSG-transplanted or untreated (Unt) mice over time. LME model (only comparisons vs. WT and p-values < 0.05 are reported; complete analyses in Supplementary Table 14). l, Representative LV integration site analysis of cells from one LV-treated newborn ADA-SCID mouse, 20 wks after PGK.ADA-LV administration (see Fig. 3b, left panel) and the corresponding NSG recipient mouse, 12 wks after transplant (see Fig. Clones with abundance <1% are shown in grey. a, Survival of LV-treated osteopetrotic (OC-LV, n = 11) or WT mice (WT-LV, n = 27) treated as newborns with PGK.TCIRG1-LV (1-2e10 TU/kg) compared to untreated OC mice (OC-unt, n = 8) mice. b, Single values of the weight of OC-LV or WT-LV mice over time post LV treatment. c,d, Mean with SEM of white blood cell (WBC, c) or lymphocyte (d) counts measured by hemocytometer in the PB of OC-LV (n = 10) or, WT-LV (n = 16) mice, or untreated WT (WT-unt) mice (n = 3) over time. Sample sizes (n) at each time point are reported below the graphs. e, Mean with SEM of the VCN measured in the PB of OC-LV (n = 9), WT-LV mice (n = 15), or WT-unt mice (n = 3) over time. Sample sizes (n) at each time point are reported below the graphs. f, Single values and mean with SEM of the total cell count isolated at necropsy from the BM or spleen of OC-LV (n = 7) or WT-LV (n = 15) mice. g,h, Single values and mean with SEM of the indicated populations among CD45-positive cells analyzed at necropsy in the spleen (g) or BM (h) of OC-LV (n = 7) or WT-LV (n = 15) mice. i, Single values and mean with SEM of the VCN measured in the indicated organs at necropsy in OC-LV (n = 9), WT-LV mice (n = 14), or WT-unt mice (n = 4). The purple area represents resorbed dentine. k, Single values and mean with SEM of C-terminal telopeptide (CTX) measured in the medium of osteoclast cultures from OC-LV (n = 5) or WT-LV (n = 4) mice following dentine resorption. l-o, Representative micro-tomography scans (l) or single values and mean with SEM of the quantification of bone mineral density (m), bone volume fraction (n), and trabecular separation (o) of WT-LV (n = 7), OC-LV-treated (n = 8), or WT-unt (n = 1) mice. Asterisks indicate the longest-surviving OC-LV mice, one of which is also shown in the bottom-left panel in (l). p, Single values and mean of parathyroid hormone (PTH) concentration in the serum of OC-LV (n = 10) or WT-LV (n = 7) mice at the indicated time points. Sample sizes (n) at each time point are reported below the graphs. a, Mean with SEM of the percentage of different populations among lineage-negative (Lin-) and cKit-positive (cKit + ) cells (HSPCs) in the PB of newborn, 2-week-old, or 8-week-old (adult) Fanca-/- mice, as indicated. Newborn PB samples represent six pools from 30 mice. c-e, Mean with SEM of neutrophil (c), monocyte (d), or platelet (e) counts measured by hemocytometer in the PB of LV-treated (FANCA-LV) or untreated (FANCA-unt) Fanca-/- mice (n = 5) or untreated wild-type (WT-unt) mice (n = 5). f, Single values and mean with SEM of the VCN measured after FACS-sorting cell populations from PB or BM of the mice shown in Fig. g, Representative LV integration site (IS) analysis of cells from one LV-treated newborn Fanca-/- mouse, 20 wks after PGK.FANCA-LV administration (shown in Fig. Shared integrations among the indicated organs or FACS-sorted populations from PB or BM are color-coded. Clones with an abundance <1% are shown in grey. h, Representative heatmap showing shared integrations (columns) in the indicated organs or FACS-sorted populations isolated from a PGK.FANCA-LV-treated mouse (shown in Fig. “Sample count” refers to the number of samples (rows) that share a given IS. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. In vivo haemopoietic stem cell gene therapy enabled by postnatal trafficking. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/elon-musks-ai-chatbot-grok-is-reciting-climate-denial-talking-points/'>Elon Musk's Grok Chatbot Has Started Reciting Climate Denial Talking Points</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Elon Musk's Grok Chatbot Has Started Reciting Climate Denial Talking Points The latest version of Grok, the chatbot created by Elon Musk's xAI, is promoting fringe climate viewpoints in a way it hasn't done before, observers say Elon Musk speaks during a Feb. 11 event with President Donald Trump in the Oval Office. Is climate change an urgent threat to the planet? To most climate scientists, the answer is straightforward: Yes, rapid warming from fossil fuel burning is pushing Earth toward dangerous tipping points. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. When asked that question earlier this month by Andrew Dessler, a climate scientist at Texas A&M University, Grok acknowledged findings from NOAA and NASA that show the risks of global warming. "Climate change is a serious threat with urgent aspects," Grok responded. "But its immediacy depends on perspective, geography, and timeframe." Asked a second time a few days later, Grok reiterated that point and said "extreme rhetoric on both sides muddies the water. Grok added an important caveat too — and one many scientists likely would agree with too — when it was queried a third time on Monday: "The planet itself will endure; it's human systems—agriculture, infrastructure, economies—and vulnerable species that face the most immediate risks." The answers are distinct from what other AI programs such as OpenAI's ChatGPT, Microsoft's Copilot and Google's Gemini say about climate change, said Dessler, who has tested different AI models for years. When those programs are asked about global warming, they echo the scientific consensus that humanity's burning of fossil fuels is heating up planet Earth and imperiling the people who live there. "Yes, climate change is widely recognized as an urgent and significant threat to the planet," responded ChatGPT last week when asked the same question. Grok is now on its third iteration after first launching in 2023, and the latest version is promoting fringe climate viewpoints in a way it hasn't done before. “A lot of the arguments it was bringing up were just sort of well trodden denier talking points that don't deserve any rehearing,” said Dessler. But don't just take Dessler's word for it. “Grok was criticized for progressive-leaning responses on climate change and other issues,” the chatbot wrote back. “xAI, under Elon Musk's direction, took steps to make Grok ‘politically neutral,' which could amplify minority views like climate skepticism to balance perceived mainstream bias.” xAI did not respond to a request for comment. Earlier this month, Grok reported that it had been “instructed” to aggressively promote the debunked “white genocide” in South Africa conspiracy theory that Trump and Musk have pushed. Far from being neutral arbiters of objective fact, AI programs in many ways reflect the flaws and biases of their human creators. The language learning models that power AI chatbots are “really quite malleable and you can change the kind of results they give,” Dessler said. The world's richest man has advanced efforts to combat climate change — such as underwriting a contest to promote carbon removal — but he also helped elect Trump, who has described global warming as a hoax and pushed policies to promote fossil fuels. When asked if climate change presents a danger to the planet — which science clearly shows — Grok notes that some question the urgency of reacting. “Wealthier nations can mitigate impacts through infrastructure (e.g., Dutch sea walls) or agricultural shifts,” Grok states. “Skeptics like Bjørn Lomborg argue adaptation is cheaper than drastic emissions cuts, prioritizing economic growth.” “Some models show gradual changes over centuries, not imminent collapse, giving time for technological solutions (e.g., carbon capture),” Grok noted. Grok 3 billed itself as the world's most powerful AI system when it was unveiled earlier this year. But it stands apart in promoting climate denial, according to Théo Alves Da Costa, an AI engineer who specializes in climate issues and president of Data for Good, a French nonprofit that tracks technology and climate impacts. That includes “classic climate disinformation arguments—natural variability, solar cycles, conspiracy narratives about the IPCC [Intergovernmental Panel on Climate Change] and transition solution skepticism,” he noted. Prominent right-wing voices previously have complained that Grok is too liberal. Musk has said he agrees and pledged that his chatbot would eradicate liberal ideas. In February, Musk said having the “woke mind virus” programmed into AI was an “existential danger.” “Even for Grok, it's tough to remove, because there is so much woke content on the internet.” The Trump administration so far has embraced AI as a solution to many American challenges. In April, Trump signed an executive order to require “AI literacy and proficiency” for school children. Trump officials, including Musk, have claimed AI could replace tens of thousands of federal jobs and be relied upon for high-stakes decisions on electrical grid planning and other uses. Trump has said that he wants a massive build-out of natural gas and coal-fired power plants to power AI data centers across the country. But scientists say AI still can be a boon for the climate fight. Still, AI has the potential to assist in helping address climate impacts. It has been deployed by scientists to track melting icebergs, extreme weather and deforestation. Google is deploying AI to make it less energy-intensive. Which is why AI systems need to provide accurate science information.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-025-00876-4'>The geroprotectors trametinib and rapamycin combine additively to extend mouse healthspan and lifespan</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 11:08:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Suppression of the insulin–IGF–mTORC1–Ras network ameliorates aging in animals. Many drugs have targets in the network because of its roles in cancer and metabolic disease and are candidates for repurposing as geroprotectors. Rapamycin, an established geroprotective drug, blocks mTORC1 signaling, and trametinib inhibits the Ras–MEK–ERK pathway. In this study, we assessed survival and health of male and female mice treated with trametinib, rapamycin or their combination. We show here that trametinib treatment extended lifespan in both sexes and that its combination with rapamycin was additive. Combination treatment reduced liver tumors in both sexes and spleen tumors in male mice, blocked the age-related increase in brain glucose uptake and strongly reduced inflammation in brain, kidney, spleen and muscle and circulating levels of pro-inflammatory cytokines. We conclude that trametinib is a geroprotector in mice and that its combination with rapamycin is more effective than either drug alone, making the combination a candidate for repurposing as a gerotherapy in humans. The insulin–IGF–mTORC1–Ras nutrient-sensing network is highly conserved in evolution and is implicated in the etiology of many age-related diseases1,2. Genetic inhibition of multiple nodes within the network can improve health during aging and extend lifespan in laboratory model organisms3. Growing evidence indicates a pro-aging role of the network in humans1,4. There is, therefore, potential to repurpose existing drugs with targets in this signaling network as geroprotectors to improve human health during aging5. Inhibition of mTORC1 by rapamycin (sirolimus), a US Food and Drug Administration (FDA)-approved drug used mainly for immunosuppression after organ transplantation6, robustly extends lifespan in multiple model organisms7,8, including mice, where rapamycin administration later in life at 600 days of age increases median and maximal lifespan in both sexes9. Rapamycin also ameliorates aging-related morbidities, including cardiac dysfunction10 and impaired immune responses11. Administration of mTOR inhibitors to older human subjects led to enhanced immune response to influenza vaccine and reduced infection rate, with briefer and lower dosing than used clinically and minimal side effects12. Reduced signaling through the phosphatidylinositol 3-kinase (PI3K) node of the nutrient-sensing network can extend lifespan in Caenorhabditis elegans and Drosophila13,14 and has been viewed as the primary route mediating the anti-aging effects of reduced upstream insulin/IGF signaling. However, Ras signaling plays a role in aging in yeast15, whereas, in Drosophila, the Ras–MEK–ERK pathway is as important a mediator as the PI3K pathway of the upstream effects of insulin/IGF signaling on lifespan16. Indirect inhibition of Ras in mice is associated with increased lifespan and enhanced motor function in old age17. Inhibition of the Ras pathway may, therefore, have an evolutionarily conserved, geroprotective effect. Ras hyperactivation is highly oncogenic, with a third of human cancers presenting with a Ras mutation12. There has, therefore, been an intense search for small-molecule inhibitors of the Ras pathway. Trametinib (also known as Mekinist), an FDA-approved drug for treatment of melanomas, is a highly specific inhibitor of MEKs18. Oral administration of trametinib increases Drosophila lifespan, even when started later in life16. To examine whether the geroprotective effect of trametinib is conserved in mice, we orally dosed female and male mice and assessed their aging phenotypes. Simultaneous inhibition of different nodes within the network, by combined drug treatments, could, therefore, be more effective than suppression of single nodes, by prevention of compensatory responses. Combinatorial drug treatments could also inhibit specific targets more strongly or induce a wider range of protective effects, because different nodes in the network have many non-overlapping targets. Indeed, joint inhibition of mTOR, MEK and glycogen synthase kinase-3 (GSK-3) by rapamycin, trametinib and lithium, respectively, resulted in additive increases in Drosophila lifespan, both with both double and triple treatment combinations, the latter inducing a 48% median lifespan extension19. Such combinatorial drug treatment could exert similar benefits in mammalian aging. In the present study, we investigated whether administration of trametinib alone or in combination with rapamycin can extend lifespan and improve health at old age in mice. We orally treated male and female mice with trametinib or rapamycin or with both drugs at the same doses as in the single-drug treatments. We assessed their survival, fitness, brain metabolism and organismal health. Administration of trametinib or rapamycin alone significantly increased male and female lifespan, whereas the combination produced an additive increase in both sexes. Additionally, combined treatment significantly reduced liver tumors in both sexes and spleen tumors in male mice and alleviated the age-related increased glucose uptake in the brain. Combination treatment also markedly reduced age-related inflammation in brain, kidney, spleen and muscle, with reduced levels of circulating pro-inflammatory cytokines. Trametinib also is, thus, geroprotective in mice, and its combination with rapamycin is even more so. 1a), and their plasma levels of trametinib (Fig. 1d,e) showed that Erk1/2 phosphorylation was unaffected at 0.29mg kg−1 and 0.58 mg kg−1 trametinib and reduced at concentrations of 1.44 mg kg−1 trametinib and higher in both sexes, although low sample number and high variability meant that only 11.52 mg kg−1 in female mice resulted in a significant reduction. Decreased Ras–Mek–Erk pathway activity upon trametinib treatment was also observed in kidney, spleen and muscle tissue (Extended Data Fig. pERK1/2 levels were significantly reduced in kidney and spleen tissue of female mice treated with 1.44 mg kg−1 trametinib (Extended Data Fig. 1a,b) with a non-significant trend in female (Extended Data Fig. Treatment with 11.52 mg kg−1 trametinib strongly reduced pERK1/2 levels in all tested tissues in both sexes (Extended Data Fig. Thus, short-term treatment with trametinib led to a dose-dependent downregulation of Ras–Mek–Erk pathway activity in several tissues. Trametinib did not significantly affect water uptake (Extended Data Fig. Trametinib can cause body weight loss, liver lesions and necrosis and increased alanine aminotransferase (ALT) and alkaline phosphatase (ALP) levels indicative of liver dysfunction. 1f,g) or ALP (Extended Data Fig. At doses of 0.29–2.88 mg kg−1, trametinib also did not affect body weight (Fig. In contrast, animals fed with 11.52 mg kg−1 trametinib failed to gain body weight in the 4-week measurement period (Fig. 1f,g), with a trend for increased spleen weight in both sexes (Fig. In summary, dietary trametinib concentrations of 1.44 mg kg−1 and higher were sufficient to inhibit Ras/Mek/Erk signaling, whereas only the highest dose of 11.52 mg kg−1 trametinib induced adverse effects on mouse health. We, therefore, used 1.44 mg kg−1 in all subsequent experiments. a, Schematic outline of the trametinib pre-test experiment. Four-month-old C3B6 F1 hybrid mice were used for all experiments. b,c, Plasma concentration of trametinib in female (b) and male (c) mice (n = 5 plasma samples per treatment and sex, except n = 4 for male 0.29 mg kg−1) fed for 4 weeks with food containing 0, 0.29, 0.58, 1.44, 2.88 or 11.52 mg kg−1 trametinib. Female mice had significantly higher trametinib plasma levels than male mice across all doses, with a significant interaction of sex and drug (P = 0.0111, sex × drug, mixed model). d,e, Ras–Mek–Erk pathway activity in liver after 4 weeks of trametinib treatment, by western blot analysis of Erk1/2 phosphorylation. Representative western blot (upper panel) and corresponding quantification of female (d) and male (e) liver samples (n = 5 per treatment for both sexes). f,g, Change in body weight upon 4 weeks of trametinib treatment in female (f) and male (g) mice. 11.52 mg kg−1 trametinib led to a significant reduction in body weight in both sexes compared to control animals (0 mg kg−1) (n = 5 animals per treatment). h,i, Wet spleen weight of female (h) and male (i) mice after 4 weeks of trametinib treatment. Spleen weight showed a tendency to increase with 11.52 mg kg−1 trametinib treatment. One-way ANOVA with Tukeyʼs post hoc test. Mice were continuously fed with 1.44 mg kg−1 trametinib starting from 6 months of age, and their survival was measured (Fig. Trametinib treatment caused a significant lifespan extension in both sexes: in female mice with a median lifespan extension of 7.2% (P = 0.0248, log-rank test) (Fig. 2b) and no significant effect on maximum lifespan (P = 0.474, Wang–Allison test) and in male mice with an increase in median lifespan of 10.2% (P < 0.0001, log-rank test) (Fig. 2c) and maximum lifespan of 15.8% (P = 0.014, Wang–Allison test). Cox proportional hazard analysis showed no significant interaction between sex and trametinib treatment (P = 0.1793), showing that trametinib increased survival of male and female mice to a similar extent. a, Schematic overview of trametinib and rapamycin treatment. Trametinib significantly increased survival in both sexes. Combined treatment significantly increased lifespan compared to the individual drug treatments (log-rank test). Lifespan studies of control and drug-treated animals were performed at the same time within the cohorts. In Drosophila, trametinib and rapamycin combine additively to extend lifespan19. We measured the survival of mice on diets containing only rapamycin (42 mg kg–1), the dose so far inducing the largest increase in longevity20, or both rapamycin (42 mg kg–1) and trametinib (1.44 mg kg–1). Rapamycin was fed in alternate weeks (Fig. 2a), which improves metabolic health parameters11,21 and extends lifespan to a similar degree as continuous treatment22. As previously shown22, intermittent rapamycin treatment extended lifespan in both sexes to a similar extent (P < 0.0001, log-rank test; P = 0.2481 for the sex–drug interaction, Cox proportional hazard) with an increase in median and maximum lifespan of 17.4% and 16.5%, respectively, in female mice (Fig. In contrast to the single treatments, combined treatment with rapamycin and trametinib increased survival more in female mice than in male mice (sex–treatment interaction P = 0.0218, Cox proportional hazard) and caused a larger increase compared to the single treatment (Fig. 2b,c) in both sexes, with median and maximum lifespan increased by 34.9% and 32.4%, respectively, in female mice (Fig. 2b,c for the corresponding P values). In female mice, the hazard ratio of trametinib alone compared to control (P = 0.0383) was 0.6926 (0.4886–0.9804) and 0.4478 (0.3002–0.6641) in the presence of rapamycin (P < 0.0001). Rapamycin alone had a hazard ratio of 0.3284 (0.2243–0.4784) compared to controls (P < 0.0001) and 0.2123 (0.1380–0.3234) in the presence of trametinib (P < 0.0001). In male mice, trametinib alone had a hazard ratio of 0.5257 (0.3845–0.7169) versus controls (P < 0.0001) and 0.5814 (0.4127–0.8154) in the presence of rapamycin (P = 0.0017). Rapamycin on its own had a hazard ratio of 0.4623 (0.3376–0.6316) compared to controls (P < 0.0001) and 0.5113 (0.3611–0.7208) in the presence of trametinib (P = 0.0001). There was, thus, no indication of a positive or negative interaction between drug treatments in either sex. In summary, treating mice with either trametinib or rapamycin extended lifespan in both sexes, with lifespan further extended by combined treatment in an additive manner. We measured health parameters, including body weight (Extended Data Fig. 3k,l), motor coordination (Extended Data Fig. 3m,n) and memory (Extended Data Fig. In addition, we used metabolic cages to measure body weight (Extended Data Fig. 4a), fat and lean mass (Extended Data Fig. 4b,c), water and food uptake (Extended Data Fig. 4d,e) and respiratory exchange ratio (RER) (Extended Data Fig. Consistent with the pre-test, 1.44 mg kg−1 trametinib had no effect on body weight (Extended Data Figs. In contrast, both rapamycin and combined drug treatment decreased body weight in female mice at 20 months (Extended Data Figs. Combined treatment induced a significant increase in fat content (Extended Data Fig. 4b) and a decrease in lean mass (Extended Data Fig. Lean mass was lower in female mice treated with trametinib (Extended Data Fig. The drug treatments did not affect water or food uptake (Extended Data Fig. Heart rate of both sexes declined with age, and, in male mice, both rapamycin and trametinib attenuated the decline, with a similar non-significant trend in female mice (Extended Data Fig. QT interval, a read-out for left ventricular hypertrophy, increased with age, and rapamycin decreased QT interval in female mice, and both rapamycin and combined treatment prevented the age-related increase in male mice (Extended Data Fig. The drug treatments did not affect center occupancy in the open field (Extended Data Fig. 3g,h), forced running endurance on a treadmill (Extended Data Fig. 3k,l) or motor coordination in a rotarod assay (Extended Data Fig. Rapamycin-treated female mice performed slightly worse in the Y-maze (Extended Data Fig. Rapamycin-treated and combination-treated male mice showed increased global speed in the Y-maze (Extended Data Fig. 3p) with a similar trend for the combined drug treatment in the open field (Extended Data Fig. 3i), suggesting a higher motivation to move at old age. RER was not significantly altered by drug treatment in 16-month-old female mice (Extended Data Fig. 4f,g), whereas all three drug treatments significantly reduced RER during the night (Extended Data Fig. 4g), indicating a slight shift toward mixed carbohydrate and lipid utilization. Chronic rapamycin treatment negatively affects glucose homeostasis and can lead to hyperglycemia. Both male and female mice treated with rapamycin showed significantly increased plasma glucose levels (Extended Data Fig. In contrast, plasma glucose levels of trametinib-treated animals did not differ significantly from controls (Extended Data Fig. Combination-treated mice had plasma glucose levels similar to rapamycin-treated animals, indicating that trametinib did not further exacerbate the effect of rapamycin. Thus, in contrast to rapamycin, trametinib extends lifespan without causing hyperglycemia. In human patients, trametinib treatment can result in increased plasma levels of ALT and ALP, indicative of reduced liver function. In contrast, treating mice chronically with 1.44 mg of trametinib did not change plasma levels of AST and ALT in 24-month-old animals (Extended Data Fig. 4i), consistent with the results from young animals (Extended Data Fig. Prolonged treatment with this dose of trametinib, therefore, did not affect liver function in mice. Rapamycin-treated and combination-treated female mice moved more at late age, and all drug treatments reduced nighttime RER in male mice. Rapamycin, but not trametinib, treatment caused hyperglycemia in old animals. Trametinib and rapamycin are cancer chemotherapeutics. To address whether their lifespan-extending effects were associated with their anti-cancer function, we performed a cross-sectional histopathological analysis at 24 months of age, complemented by a macropathological assessment of overall tumor load postmortem, to assess any delay in tumor formation. Untreated mice predominantly developed tumors in the liver (Fig. Liver tumors were most prevalent, with approximately 50% of female mice (Fig. 3b) with tumors at 24 months of age. No significant differences in liver tumors were detected with either trametinib or rapamycin treatment at 24 months (Fig. However, combination treatment significantly reduced liver tumor numbers in both sexes at 24 months of age (Fig. 3a,b) and in male mice postmortem (Fig. 3d), with a similar trend in female mice (Fig. Twenty-five percent of female mice and 20% of male mice had tumors in the spleen (Fig. Neither trametinib nor rapamycin reduced tumor formation in the spleen (Fig. The combination treatment significantly reduced spleen tumor formation at 24 months of age in male mice (Fig. 3f), with a similar trend in female mice (Fig. Although we did not detect any tumors in the kidney upon combined treatment, this tumor type was so rare in controls that the effect was not significant. In summary, only the combination of rapamycin and trametinib caused significant effects on tumor load, and this was more obvious in the cross-sectional analysis—for example, compare spleen pathology in male mice at 24 months and postmortem (Fig. 3g), which may indicate a delay in tumor formation rather than reduced overall lifetime tumor load. To further investigate, we performed a longitudinal analysis, measuring tumor formation in vivo by 18F fluodeoxyglucose (18F-FDG) positron emission tomography (PET) coupled with computed tomography (CT)23 in the same animals at 12, 18 and 24 months of age (Fig. Because the capacity for the PET/CT measurement was limited, we restricted our analysis to control, trametinib-treated and combination-treated female mice. Tumors were identified as hypermetabolic regions, first in 24-month-old animals and then traced back in the same animals at 18 months and 12 months of age. Tumors were detected in several organs, including liver and uterus, but could only be reliably quantified in the spleen (Fig. Tumor occurrence increased with age in control animals, and, consistent with the histopathological analysis (Fig. 3e), trametinib did not affect spleen tumors at 12, 18 or 24 months (Fig. In contrast, combination-treated animals showed lower spleen tumors at all three ages, with no significant increase with age. Combined drug treatment, thus, delayed tumor formation, but, due to the missing single rapamycin treatment, the effect may have been entirely attributable to rapamycin. Cross-sectional histopathological analysis of tumor levels in liver (a,b), spleen (e,f) and kidney (i,j) samples of 24-month-old female mice (a,e,i) and male mice (b,f,j). Combined drug treatment significantly reduced liver tumors in both sexes at 24 months and spleen tumors in male mice, with a similar trend for reduced tumor load in female spleen and male and female kidney (e,f). The absence of liver tumors was scored with 0, and the presence of low, moderate, high and very high severity tumors was scored with 1, 2, 3 and 4, respectively. Liver tumor data for rapamycin and control animals were previously published22. Postmortem macropathology analysis of liver (c,d) and spleen (g,h) of animals from the lifespan and phenotyping cohort. k, Longitudinal analysis of spleen tumor progression via 18F-FDG PET/CT measurements in the same female animals at 12, 18 and 24 months of age. Although tumor load increased from 12 months to 24 months in control and trametinib-treated mice, combination-treated animals showed no increase. 18F-FDG uptake of rapamycin/trametinib versus control at 24 months P = 0.08. Two-way ANOVA with post hoc Bonferroni test and simple linear regression. Data in a–j are presented as percentage over total, and statistical analysis was performed by two-sided chi-square test and Poisson regression. Numbers in brackets indicate the number of scored tissues per treatment. Brain glucose uptake increases during mouse aging24 and may be linked to impaired cognitive function25. We, therefore, used the 18F-FDG PET/CT longitudinal data to address whether trametinib and combined drug treatment attenuated age-related increase in glucose uptake (Fig. 18F-FDG uptake increased with age in control female mice across the whole brain (Fig. 18F-FDG uptake increased significantly between 12 months and 24 months in the striatum and cerebellum but not the cortex (Fig. Uptake in trametinib-treated female mice showed a significant increase across the whole brain and in the striatum, cerebellum and cortex between 12 months and 18 months of age (Fig. 4c,g), confirmed by two-way ANOVA analysis (P < 0.05 interaction between age and drug treatment; Fig. 4f–h) showed that trametinib treatment induced an intermediate phenotype between control and combination-treated animals with an increase in glucose uptake, whereas there was no change in the double drug-treated mice. Trametinib treatment, thus, ameliorated the age-related increase in brain glucose uptake but only completely blocked it when combined with rapamycin. a–c, Images show differential regional glucose uptake across the mouse brain between 24-month-old and 12-month-old control (a), trametinib-treated (b) and rapamcin/trametinib-treated (c) female mice. Color code represents the P value for the indicated voxels in a paired Student's t-test comparing 24-month-old to 12-month-old animals. d, Two-way ANOVA analysis highlights brain regions that showed a significant interaction between age and drug treatment in differential glucose uptake. e,f, Longitudional quantification of glucose uptake in whole brain, striatum, cerebellum and cortex at 12, 18 and 24 months of age in control (e), trametinib-treated (f) and rapamycin/trametinib-treated (g) mice. Glucose uptake significantly increased with age in striatum and cerebellum of control animals but not in combination-treated animals. h,i, Images show differential glucose uptake in the brain of 24-month-old trametinib versus control (h) and combination versus control (i) mice. j–m, Quantification of differential glucose uptake between 12-month-old and 24-month-old whole brain (j), striatum (k), cerebellum (l) and cortex (m). Age-related differences in glucose uptake between 12 months and 24 months were significantly lower in combination-treated animals compared to controls. A similar but milder effect was also observed in trametinib-treated animals, which was only significant in the cerebellum. Two-way ANOVA with Bonferroni post hoc test. a–g and j–m, Numbers in brackets indicate the number of whole brains/brain regions analyzed per treatment and timepoint. Brain inflammation increases the requirements for glucose and results in greater 18F-FDG uptake26. Microglia are the primary immune cells of the brain, and their increased activity is associated with age-related brain inflammation accompanied by synapse loss and late-onset neurodegeneration27. Microglia act in concert with astrocytes to resolve inflammation, and astrocyte-dependent Tgfβ secretion can inhibit microglia activation28. Aging disrupts this interplay, leading to exacerbated inflammation in both an autonomous and an astrocyte-dependent manner29. Because Ras/MEK/Erk signaling has been implicated in microglial activation27, we assessed whether drug treatment reduced activation of microglia and astrocyte density by performing immunohistological stainings of brain sections using antibodies against Iba-1 and Gfap for activated microglia and astrocyte density, respectively (Fig. 5a–e), in brains of 24-month-old female control (Fig. We also stained brains of 6-month-old control mice (Fig. Activated microglial and astrocyte density significantly increased in the striatum (Fig. 5c,d) but not the cerebellum (Extended Data Fig. 5e,f) of 24-month-old compared to 6-month-old controls, indicating increased brain inflammation at older age. Combined drug treatment significantly reduced the density of activated microglia and astrocytes in the striatum of 24-month-old mice (Fig. 5f,g), with a similar trend in the cortex (Extended Data Fig. 5c,d) but no significant effects in the hippocampus (Extended Data Fig. 5a,b) or the cerebellum (Extended Data Fig. Compared to the combined treatment, single treatment with rapamycin or trametinib did not significantly reduce the number of activated microglia in the striatum (Fig. Thus, rapamycin and trametinib had only mild effects, but the combination of both drugs significantly reduced age-related brain inflammation in specific regions. Notably, 18F-FDG uptake was increased in the cerebellum, suggesting mechanisms other than inflammation. a–e, Representative confocal composite images of immunohistochemical stainings of the striatum region in 6-month-old young controls (a) and 24-month-old controls (b) and 24-month-old rapamycin-treated (c), 24-month-old trametinib-treated (d) and 24-month-old rapamycin/trametinib-treated (e) female mice. f,g, Quantification of microglia density (DAPI/Iba-1+ cells) (f) and astrocyte density (DAPI/GFAP+ cells) (g) per mouse striatum of young and old control animals and old drug-treated animals. Microglia and astrocyte density significantly increased with age in the striatum of control animals. Combination treatment significantly reduced microglia and astrocyte density in the striatum compared to old controls. One-way ANOVA with Bonferroni post hoc test. Numbers in brackets indicate the number of brain regions of individual animals analyzed per treatment. As Ras/MEK/Erk signaling has been implicated in microglia activation27, we assessed whether trametinib crossed the blood–brain barrier. At dietary concentrations of 0.29 mg kg−1 and 0.58 mg kg−1, trametinib was not detected in the brain (Extended Data Fig. Significant, but very low, levels of trametinib were detected at 1.44 mg kg−1 and 2.88 mg kg−1 dietary concentrations, with levels 25-fold and 30-fold lower than the corresponding plasma concentrations of female (140 ng µl−1) and male (55.8 ng µl−1) mice, respectively. Trametinib at 1.44 mg kg−1 thus barely crossed the blood–brain barrier. Only at 11.52 mg kg−1 did we detect trametinib in the brain, which might indicate saturation of the efflux system due to the high trametinib plasma levels. In conclusion, the effects on microglia upon treatment with 1.44 mg kg−1 trametinib might be indirect and caused by the action of trametinib in peripheral tissues. During aging, inflammation increases in multiple peripheral organs, manifested in a low-level, systemic, pro-inflammatory phenotype termed ‘inflammaging'30. Inflammaging is associated with both mTOR and Erk pathway activity31. Combined treatment with rapamycin and trametinib reduced brain inflammation, so we assessed inflammation in peripheral tissues by histopathology of kidney and white adipose tissue (WAT) (Fig. Trametinib did not reduce kidney inflammation (Fig. 6a,b), with a non-significant reduction with rapamycin treatment (40% of female mice and 43% of male mice affected at 24 months). Combined treatment significantly reduced kidney inflammation in both sexes with only 34% of female mice and 35% of male mice affected. Most of the combined drug effect was, therefore, probably due to rapamycin, which positively affects kidney health32. a–d, Histopathological analysis of inflammation in kidney (a,b) and WAT (c,d) of 24-month-old female mice (a,c) and male mice (b,d). Combination-treated female mice and male mice showed significantly reduced kidney inflammation compared to control and trametinib-treated animals. Numbers in brackets indicate the number of tissue samples from independent mice per treatment. There was a non-significant trend for reduced WAT inflammation upon treatment with rapamycin and rapamycin/trametinib. e–p, Plasma protein levels of circulating cytokines as measured by the Olink Target 96 Mouse Exploratory panel: Tnf (e,f), IL17a (g,h), IL23r (i,j), Tgfα (k,l), IL6 (m,n) and IL10 (o,p). Drug effects on plasma cytokine levels were highly sex specific, with Tnf, IL17a, IL23r and Tgfα being downregulated by the combination treatment only in female mice but not male mice. One-way ANOVA with Tukeyʼs post hoc test. Numbers in brackets indicate the number of plasma samples from independent mice per treatment. Data in e and p are presented as mean ± s.e.m. We also assessed inflammatory markers in plasma of both sexes at 24 months, using the Olink Target 96 Mouse Exploratory panel (Fig. Combination-treated female mice at 24 months showed significantly reduced plasma levels of key pro-inflammatory proteins, including tumor necrosis factor (Tnf; Fig. 6i) and transforming growth factor α (Tgfα; Fig. Trametinib-treated female mice showed a significant reduction only in Tgfα (Fig. 6k), and rapamycin treatment significantly reduced levels of IL17a (Fig. The single-drug treatments had no significant effect on pro-inflammatory cytokines in male mice, with only interleukin-6 (IL6), which was not regulated in female mice (Fig. 6n) were mildly reduced in combination-treated male mice. Thus, combination treatment reduced the measured pro-inflammatory cytokines only in female mice. Notably, combination treatment significantly reduced the level of the anti-inflammatory cytokine interleukin-10 (IL10) in both female mice (Fig. 6p), possibly a response to reduced inflammation in these animals. In summary, combination treatment reduced inflammation in peripheral tissues accompanied by reduced levels of circulating pro-inflammatory cytokines. We assessed the impact of the drug treatments on age-related, non-neoplastic pathologies by measuring heart hypertrophy and kidney and spleen pathology at 24 months of age (Extended Data Fig. Heart hypertrophy and kidney glomerulopatholgy were more pronounced in male mice than female mice, but there was no significant effect of any of the drug treatments (Extended Data Fig. Trametinib had no significant effect on spleen pathology in female mice, with a mild reduction in male mice (Extended Data Fig. Rapamycin mildly reduced spleen pathology in female mice, with a significant reduction in male mice (Extended Data Fig. Combined drug treatment significantly reduced spleen pathology in both sexes (Extended Data Fig. Increased spleen size in old mice reflects disrupted microarchitecture, extramedullary hematopoiesis, chronic inflammation and accumulation of senescent cells33, all of which impair spleen function. Spleen weight was significantly decreased by trametinib in female mice, with a similar trend in male mice, and by rapamycin and combination treatment in both sexes (Extended Data Fig. 6g,h), suggesting that both drugs combat spleen immunosenescence. Long-term administration of rapamycin is associated with negative side effects, including liver lipidosis and testicular degeneration22. In contrast to rapamycin, trametinib treatment did not induce liver lipidosis or gonadal lesions (Extended Data Fig. 6i–l), whereas animals treated with both drugs showed similar levels to rapamycin-treated mice (Extended Data Fig. Thus, trametinib treatment did not induce these phenotypes, but nor did it protect rapamycin-treated mice from them. Combination drug treatment reduced age-related kidney inflammation and spleen size. We, therefore, measured gene expression changes in response to drug treatment in kidney, spleen and muscle of 24-month-old mice using RNA sequencing (RNA-seq) (Fig. In all three organs, the most strongly regulated genes were linked to inflammatory processes (Extended Data Figs. 7 and 8), with the number of differentially expressed genes (DEGs) strongly dependent on drug treatment and tissue type (Fig. Trametinib had only minor effects in female and male kidney (40/146 DEGs) and muscle (32/78 DEGs) and in the female spleen (14 DEGs), whereas, in male spleen, 2,038 genes were differentially regulated. Apart from in the male spleen, more genes were differentially regulated upon rapamycin treatment, with 199/210 DEGs in kidney, 828/534 DEGs in spleen and 743/317 DEGs in muscle of female mice and male mice, respectively. The strongest changes occurred with combination treatment, with 290/105 DEGs in kidney, 4,038/2,391 DEGs in spleen and 967/479 DEGs in muscle of female mice and male mice, respectively. Gene Ontology (GO) enrichment analysis showed little overlap in the transcriptional responses in male and female mice (Fig. Only GO terms associated with chromosome segregation, nuclear division and microtubule cytoskeleton organization involved in mitosis were downregulated in the spleen of both male and female combination-treated animals (Fig. There was also little overlap in DEGs between rapamycin and trametinib (Fig. 7a–f), with GO showing different terms between rapamycin and trametinib single treatments (Fig. Although there was significant overlap in DEGs between the single and the combined drug treatments (Fig. 7a–f), most DEGs were specific to the combined treatment, and there was little overlap in GO terms between single and combined treatments (Fig. Many GO terms were associated with immune regulation and inflammation (Fig. In contrast, in the spleen and muscle, genes associated with immune regulation and inflammation were exclusively downregulated (Fig. RNA-seq analysis of kidney, spleen and muscle tissues isolated from 24-month-old male and female mice used to measure gene expression changes in response to rapamycin, trametinib and rapamycin/trametinib treatment. a–f, Venn diagrams depicting the overlap in differential gene expression upon rapamycin, trametinib and rapamycin/trametinib treatment in kidney (a,d), spleen (b,e) and muscle (c,f) tissue. The left Venn diagram per panel shows the overlap in upregulated genes (upward arrow), and the right panel shows downregulated genes (downward arrow). g–l, GO enrichment analysis was performed using the DAVID API-based function DAVIDenrich of the AGEpy Python package on significantly differential expressed genes upon rapamycin, trametinib and rapamycin/trametinib treatment. Dot plot representation of significantly enriched top GO terms sorted by adjusted P value (Benjamini–Hochberg) for upregulated (g,l,k) and downregulated (h,j,l) genes in kidney (g,h), spleen (i,j) and muscle (k,l). MHC, major histocompatibility complex; R, rapamycin; T, trametinib. There was differential regulation of genes associated with inflammation. Cd5l and Ccl8 were also downregulated by rapamycin but not trametinib in the spleen (Extended Data Fig. Lysozyme 1 (Lyz1) expression was strongly downregulated by combination treatment in both kidney and spleen of male and female mice (Extended Data Fig. Lyz1 is pro-inflammatory and can promote kidney fibrosis37. In kidney, Lyz1 was downregulated by trametinib but not rapamycin (Extended Data Fig. In female but not male spleen, Lyz1 was significantly downregulated by rapamycin and trametinib. Reduced Lyz1 levels upon trametinib treatment suggest a protective role against kidney fibrosis, even more pronounced after co-administration of rapamycin. Combined drug treatment also reduced expression of chemokines and chemoattractants in the kidney, spleen and muscle, including the chemokine (C-X-C motif) ligand (Cxcl) genes Cxcl7, Cxcl11 and Cxcl13 (Extended Data Fig. Cxcl7 was downregulated by rapamycin in male muscle and by the combined treatment in male muscle and kidney (Extended Data Fig. Cxcl11 was significantly downregulated by rapamycin and the combination treatment in female spleen (Extended Data Fig. Cxcl13 was in the top 10 of significantly downregulated genes by rapamycin in muscle, with a similar trend for the combination treatment (Extended Data Fig. Cxcl14 was significantly upregulated by rapamycin and the combination treatment in muscles of both sexes (Extended Data Fig. Cxcl14 was significantly downregulated in the female but not male spleen under combination treatment (Extended Data Fig. Cxcl14 has context-dependent pro-tumor and anti-tumor effects38, possibly explaining the tissue-specific regulation. Their reduced expression upon combined drug treatment accords with the anti-inflammatory and anti-cancer effect of rapamycin and trametinib co-administration. E26 avian leukemia oncogene 2, 3′ domain (Ets) transcription factors are key downstream effectors of the Ras/MEK/Erk signaling, and activated ETS2 sustains inflammatory responses by regulating the expression of inflammatory genes, including chemokines and matrix metallopeptidases (Mmp)41. Spleen tyrosine kinase (Syk), a direct target of ETS2 (ref. 42), was significantly downregulated by combination but not single-drug treatment in female kidney (Extended Data Fig. Syk was also downregulated by rapamycin and the combination treatment in male muscle (Extended Data Fig. Syk deficiency is protective against inflammation41, suggesting that its downregulation may have contributed to reduced inflammation in combination-treated mice. Expression of the two matrix metallopeptidases Mmp2 and Mmp13 was significantly downregulated in female but not male spleen upon combination treatment and with rapamycin for Mmp2 in female spleen (Extended Data Fig. Mmp2 interacts with the α-2-macroglobulin (A2m) gene, which was among the strongest downregulated genes by rapamycin and combination treatment in spleen (Extended Data Fig. A2m expression was also strongly downregulated by rapamycin and the combination treatment in the female kidney (Extended Data Fig. Thus, rapamycin and the combination treatment may have alleviated age-related kidney fibrosis and dysfunction by lowering A2m levels. Interestingly, levels of Mmp12 were strongly increased by rapamycin and the combination treatment in both the female and male spleen (Extended Data Fig. Mmp12 usually promotes inflammation but has also been associated with anti-inflammatory effects46. Inflammatory gene expression was often sex specific. Although the drug treatments reduced Mmp expression only ín female mice, levels of chymotrypsin-like elastase (Cela) genes were exclusively downregulated in male mice (Extended Data Fig. Cela2a and Cela3b levels were significantly downregulated by single and combination drug treatments in male but not female spleen (Extended Data Fig. Furthermore, Cela1 was downregulated by rapamycin in the male muscle (Extended Data Fig. In summary, the treatment with rapamycin and trametinib induced a drug-specific, sex-specific and tissue-specific remodeling of the aging transcriptome characterized by reduced expression of genes associated with inflammation and cancer growth. We established that the FDA-approved drug trametinib, an inhibitor of RAS/Mek/Erk signaling, is geroprotective in mice and that its combination with the mTOR inhibitor rapamycin has stronger effects than either drug alone in both female and male mice. The combination treatment produced an additive increase in lifespan, attenuated the decline in heart function with age, delayed tumor growth and overall tumor load and reduced brain and peripheral inflammation, suggesting improved health at old age. Several ongoing clinical trials address the potential of mTOR inhibitors as geroprotective drugs in humans49. Our study suggests that simultaneous inhibition of the mTOR and Ras–Mek–ERK pathway provides additional benefits that are worth exploring in humans. Trametinib is used to treat metastatic melanomas and has also been used in clinical trials for treatment of solid tumors50. At a dose of 2 mg of trametinib per day, the drug is reasonably well tolerated, with side effects including skin rash, diarrhea, fatigue and retinopathy50. Average peak plasma levels in patients receiving 2 mg of trametinib daily were between 5.5 ng ml−1 and 7.5 ng ml−1 (ref. 50), which is significantly higher than the 0.1 ng ml−1 in mice fed with 1.44 mg of trametinib per kg of food. In a mouse model of mucosal melanoma, administering trametinib via oral gavage resulted in an average peak concentration of 23.8 ng ml−1 (ref. 51), which is more than 200-fold the concentration required for lifespan extension. Although effects of trametinib plasma levels might not be directly similar between chronic and acute application, these results suggest that higher trametinib doses are required for its anti-cancer effect than for lifespan extension. Consistently, tumor progression or load was not significantly changed in long-lived mice treated with trametinib, and the effects of 1.44 mg kg−1 trametinib on lifespan and health were mild. We, therefore, probably underdosed, and the optimal dose for geroprotection may be higher. In the pre-test, a dose of 2.88 mg kg−1 trametinib did not cause detrimental side effects and should be tested in future studies. It will also be important to assess the effects of late-onset and intermittent treatment, as these would be key to feasibility of and adherence to trametinib treatment in humans. Combined treatment with rapamycin and trametinib increased lifespan by 29% and 27% in female and male mice, respectively, with no indication of any positive or negative interaction between the drugs, which had an additive effect on lifespan. It is not known what doses of rapamycin or trametinib produce a maximum increase in lifespan in mice or if these or their combination would also maximize late-life health. Nor is it known if the stronger geroprotection from combination treatment is a result of stronger target inhibition, additive inhibtion of more targets or some form of interaction between the drugs, for instance by prevention of signaling feedback. Rapamycin caused liver lipidosis, hyperglycemia and testicular degeneration, whereas trametinib induced no obvious detrimental side effects. However, further in vivo experiments, including measurements of circulating insulin levels and glucose and insulin tolerance, will be necessary to confirm this. Apart from their longer lifespan, trametinib-treated mice showed only mild phenotypes, and the mechanisms contributing to their increased lifespan are currently unclear. Trametinib reduced spleen size at old age and caused a strong transcriptional response, especially in the male spleen, with an enrichment of GO terms associated with cell division and myeloid cell development. The Ras–Mek–Erk pathway controls B and T cell development, and Erk controls hyperproliferation in a mouse model of lymphoproliferative disease characterized by an enlargement of peripheral lymphoid organs52. Thus, trametinib may affect immunosenescence in the aging spleen. Combination treatment with rapamycin and trametinib blocked the age-related increase in brain glucose uptake, and it also reduced density of activated microglia and astrocytes in several brain regions of female mice, indicative of reduced brain inflammation. Because trametinib did not cross the blood–brain barrier at the dose used, something else may have done so to decrease age-related inflammation in the brain. Treatment with the pro-longevity drugs acarbose, 17-α-estradiol and nordihydroguaiaretic acid specifically reduced age-associated inflammation in the hypothalamus of male but not female mice, whereas caloric restriction reduced brain inflammation in both sexes53. Geroprotection by drugs can be sex specific, and microglia show sex-related differences during development and aging54, so it will be important to assess the effects of combination treatment on brain inflammation in male mice. Another limitation of our study was that we could not measure the effect of rapamycin on brain glucose uptake and, therefore, did not address whether the effects of the combination treatment were caused by rapamycin. In addition, differences in region-specific glucose uptake and brain inflammation implicate additional currently unknown mechanisms. These included downregulation of plasma cytokines, which was more prominent in female mice, and the tissue-specific response in the transcriptome of aged animals. Rapamycin blood levels were also higher in female UM-Het3 mice20, but there was no sex difference in rapamycin plasma levels of the intermittently fed C3B6F1 mice used in this study22. Notably, despite higher plasma levels in female animals, the transcriptional response to trametinib treatment in the spleen was much stronger in male mice, suggesting that other processes than just drug bioavailability contribute to the sexually dimorphic molecular responses. In combination with rapamycin, it acts additively to extend lifespan, and the drug combination induces late-life improvement in health at doses that have no detectable side effects. All animal protocols were done in accordance with the guidelines of the Federation of the European Laboratory Animal Science Association (FELASA) and approved by the Landesamt für Natur, Umwelt und Verbraucherschutz, Nordrhein-Westfalen, Germany (reference nos. Mice were housed in individually ventilated cages (Techniplast, GM500 Mouse IVC Green Line) in groups of five per cage, under specific pathogen-free conditions at 21 °C with 50–60% humidity and a 12-hour light/dark cycle. Mice had constant access to nesting material and chew sticks and ad libitum access to a standard rodent diet (Ssniff, R/MH low phytoestrogen; Ssniff Spezialdiäten GmbH) and sterile-filtered water. For the trametinib pre-test experiment, 3-month-old male and female mice, five per treatment group, were fed with standard chow containing 0, 0.29, 0.58, 1.44, 2.88 or 11.52 mg of trametinib per kg of diet for 4 weeks. At weaning, mice were fed a standard chow diet supplemented with the rapamycin encapsulation material Eudragit S100 (Evonik Cyro, 1-207-490-4242; 480 mg kg−1 diet). All experimental drug treatments started at 6 months of age, when control mice received a standard diet supplemented with the corresponding amount of Eudragit S100 (480 mg kg−1) and PEG-400 (3.2 ml kg−1). Animals treated with rapamycin received standard chow containing 42 mg kg−1 rapamycin (522 mg encapsulated rapamycin per kg) and PEG-400 (3.2 ml kg−1), and animals treated with trametinib received food containing 1.44 mg kg−1 trametinib, Eudragit S100 (480 mg kg−1) and PEG-400 (3.2 ml kg−1). Animals receiving the combined diet were fed with food containing 42 mg kg−1 rapamycin, 1.44 mg kg−1 trametinib and PEG-400 (3.2 ml kg−1). Rapamycin was obtained from LC Laboratories and encapsulated by the Southwest Research Institute (SwRI). Mice for phenotyping, tissue collection and survival analysis were bred in three generations from the same breeding pairs. Mice of the phenotyping and tissue collection cohorts were included in the survival analysis and were censored at 24 months of age, when their tissues were collected. Animals with poor health status were closely monitored and euthanized if they reached a predefined score. Mice were otherwise left undisturbed until they died of natural cause. Kaplan–Meier survival curves were generated using birth and death dates. Median lifespan was assessed, and survivorship was analyzed using log-rank test and Cox proportional hazard analysis. Maximum lifespan was assessed using the Wang–Allison test. Survival curves for control and rapamycin-treated mice of the lifespan cohort were previously published22. Mice were euthanized using cervical dislocation. Tissues were dissected and snap frozen in liquid nitrogen. Plasma was isolated by adding 1 μl of 500 mM EDTA per 100 μl of blood and subsequent centrifugation at 1,000g for 10 minutes at 4 °C. Tissues for histopathological analysis were fixed in 10% neutral buffered formalin for preparation of formalin-fixed, paraffin embedded (FFPE) tissues. Cross-sectional histopathology of liver, heart, pancreas, kidney, WAT, brown adipose tissue (BAT), spleen and testes of 24-month-old animals was carried out by Robert Klopfleisch (Institute of Veterinary Pathology, Freie Universität Berlin). Hematoxylin and eosin stainings were scored. Liver tissue was examined for the presence of lymphomas, sarcomas, benign or malignant tumors, lipidosis (diffuse or focal) and necrosis. Heart tissue was assessed for hypertrophy and fibrosis. In pancreas, inflammation, fat necrosis, lymphoma and exocrine atrophy were evaluated. In spleen, extramedullary hematopoiesis, lymphomas, sarcomas, germinal center and mantle zone hyperplasia were scored with 1 or 0 (absence). Testes were examined for atrophy or degeneration. Dead mice were assessed for tumor load and location, organ enlargement and abnormalities, including organ discoloration or granulation. The presence and severity of tumors was scored with 0 (absence), 1 (one tumor in one organ), 2 (multiple tumors or tumor diameter > 3 cm in one organ or metastasis in two organs) and 3 (severe metastasis in three or more organs). Other lesions were scored with 1 or 0 (absence). Next, 20 µl of plasma was mixed with 1 ml of a −20 °C methyl-tert-butyl-ether (MTBE):methanol:water (5:3:2 (v:v:v)) mixture, containing 10 ng ml−1 everolimus as internal standard. The MTBE phase was dried in a Speed Vacuum concentrator and resuspended in 80 µl of 70:30 (v:v) acetonitrile:isopropanol (BioSolve) containing 5% dimethylsulfoxid. Samples were vortexed, incubated for 5 minutes in a sonication bath, filtered through a 0.2-µm filter and then transferred to autosampler vials, and liquid chromatography–mass spectrometry (LC-MS) was performed using a Xevo TQ-S triple quadrupole mass spectrometer (Waters) coupled to an Acquity iClass liquid chromatography system (Waters). Samples were analyzed using a multiple reaction monitoring (MRM) method. For the absolute quantification of trametinib levels, trametinib standards were diluted to concentrations of 0, 2, 4, 10, 40, 200, 600, 1,200, 2,400 and 3,600 ng ml−1. LC–MS data were analyzed using TargetLynx software (version 4.1, Waters). Plasma measurements of AST and ALP levels were performed by Laboklin GmbH & Co. KG, Diagnostic Laboratory. Plasma protein levels were measured via proximity extension assay in EDTA plasma samples using the Olink Target 96 Mouse Exploratory panel (Olink Proteomics). Protein expression values were normalized using the Ct values of extension controls and a relative correction factor determined by Olink. Normalized protein expression (NPX) values on a log2 scale were analyzed. Eighty-four out of 86 plasma samples provided high-quality data. A total of 74 proteins were evaluated, representing greater than 75% quantification values as determined by values above the limit of detection. Individual protein measurements at old age between treatment groups were compared using Kruskal–Wallis and post hoc Mann–Whitney U-test with Benjamini–Hochberg correction (scipy Python library version 1.6.2). Next, 5–10 mg of liver tissue was lysed in 400 μl of RIPA buffer (Pierce) containing cOmplete Mini Protease Inhibitor without EDTA (Roche) and PhosStop phosphatase inhibitors (Roche) in FastPrep Lysing Matrix D tubes (MP Biomedicals, 116913100) using FastPrep-24 (Thermo Fisher Scientific). Protein extracts were incubated on ice for 10 minutes and placed in a sonicator water bath filled for 10 minutes. Next, 20 μg of protein extracts was loaded and separated using 12% acrylamide gels (Criterion TGX Stain-Free Protein Gel; Bio-Rad, 5678044) and blotted on PVDF membranes (Amersham Hybond, GE10600023; Merck) at 80 V, for 1 hour, on ice. Membranes were blocked using 5% non-fat dry milk powder in Tris-buffered saline (TBS) 0.1% Tween 20 (TBS-T) at room temperature (RT) for 1 hour and then incubated with primary antibodies for phosphorylated ERK1/2 (1:1,000, anti-rabbit; Cell Signaling Technology, 4370) and total ERK1/2 (1:1,000, anti-rabbit; Cell Signaling Technology, 4695) in 5% fatty-acid-free BSA in TBS-T at 4 °C overnight. For signal development, ECL Select Western Blotting Detection Reagent (GE Healthcare) was applied, and image acquisition was performed using a ChemiDoc XRS+ System (Bio-Rad). Western blot signals were quantified using Image Lab Software (Bio-Rad) and normalized against non-drug-treated controls. Then, 25-μm brain sections were cut using a cryostat (Leica Biosystems, CM1850) and mounted on Superfrost Plus microscope slides (Thermo Fisher Scientific). Samples were incubated with 2% normal goat serum in PBST at RT and then incubated with primary antibodies in 2% normal goat serum in PBST overnight at 4 °C. Primary antibodies included: ionized calcium-binding adaptor molecule 1 (Iba-1, 1:500; Wako, 019-19741) and glial fibrillary acidic protein (GFAP, 1:1,000; Sigma-Aldrich, G3893). Slides were washed 3 × 15 min in PBST and incubated for 1 hour with Alexa Fluor-conjugated secondary antibodies at 1:1,000 in 2% normal goat serum at RT. Secondary antibodies included: Alexa Fluor 488 goat anti-rabbit IgG (Thermo Fisher Scientific, A-11008, to detect Iba-1) and Alexa Fluor 633 goat anti-mouse IgG (Thermo Fisher Scientific, A-21126, to detect GFAP). Slides were washed with PBS for 10 min and embedded using VECTASHIELD Antifade Mounting Medium without DAPI (VectorLabs). Imaging was performed using a Leica SP8 upright confocal microscope and Application Suite X 3.0.15 software (Leica Microsystems). Then, 40-μm z-stack confocal images were acquired at 2-μm intervals, with ×40/1.3 oil objective. Settings were standardized for all treatment groups during image acquisition. Representative montages from z-stack confocal images were used, and image processing was performed using ImageJ (Fiji) software version 2.3.0/1.53q. For quantification of astrocytes and microglia density, the cell counter tool was used to count GFAP+ or Iba-1+ cells that overlapped with DAPI+ nuclei, respectively, in 10–15 images per brain region per mouse brain. Next, 4–10 mg of spleen, muscle and kidney samples were homogenized in 1 ml of TRIzol (Life Technologies) using FastPrep-24. Samples were incubated at RT for 5 minutes. The aqueous phase was transferred to a new RNAse-free tube. Then, 500 μl of isopropanol, 50 μl of 3.0 M NaOAc and 1.5 μl of glycogen (Thermo Fisher Scientific) were added, and the mix was incubated at −80 °C for 30 minutes. RNA pellets were washed twice with 1 ml of ice-cold 70% ethanol, air dried for 5–10 minutes and resuspended in 30 μl of RNAse-free water. RNA-seq library preparation and sequencing of kidney, spleen and muscle RNA was performed by Admera Health. Prior to first-strand synthesis, samples were randomly primed (5′ d(N6) 3′ (N = A,C,G,T)) and fragmented. First-strand synthesis was done using ProtoScript II Reverse Transcriptase. Library construction was done according to a NEBNext Ultra II Non-Directional RNA Library Prep Kit for Illumina (New England Biolabs, E7760L). Final libraries quantity was assessed by Qubit 2.0 (Thermo Fisher Scientific), and quality was assessed by TapeStation D1000 ScreenTape (Agilent Technologies). Equimolar pooling of libraries was performed based on QC values and sequenced on an Illumina NovaSeq S4 with a read length configuration of 150 bp for 50 million paired-end reads per sample. Libraries read processing was automated using Flaski (version 3.11.34). Raw reads were mapped to the mm39 ENSEMBL build 105 using kallisto (version 0.46.1). Gene counts were quantified in the same step. GO enrichment analysis was performed using the DAVID API-based function DAVIDenrich of the AGEpy Python package (version 0.8.2) on the GOTERM_BP_FAT ontology to identify significantly enriched terms (Benjamini–Hochberg-adjusted P < 0.05). As background, all expressed genes of a tissue were used. Redundancy of significantly enriched GO terms (adjusted P < 0.05) was reduced using REVIGO (cutoff: 0.7, valueType: pvalue, measure: SIMREL). GO terms were visualized using dot plot representation. PET imaging was as described in ref. 57 using an Inveon preclinical PET/CT system (Siemens). CT data were used for attenuation correction of the PET data, and the CT image of the scull was used for image co-registration. Plasma glucose levels were determined from a tail vein blood sample using a standard glucometer (Bayer) after removing the tail vein catheters. The imaging analysis software Vinci was used to co-register images to a 3D mouse brain atlas constructed from the two-dimensional (2D) mouse brain atlas (Paxinos). Kinetic modeling was performed as described in ref. The ratio of tissue and plasma glucose concentrations (CE/CP) is a measure for glucose transport and is given by CE/CP = K1 / (k2 + k3 / 0.26)26,57. Because cellular activation is accompanied by increased glucose transport and this parameter is less sensitive to changes in plasma glucose level, we use alterations of glucose transport (CE/CP) as surrogate for alterations in neuronal activation. Statistical testing was performed by application of a voxel-wise t-test between groups. 3D maps of P values allow for identification of regions with significant differences in the parameters. From these regions, we defined volumes of interest (VOIs) and performed additional statistical testing. A separate mouse cohort of 15 mice per treatment group was used to evaluate drug-related and age-related changes in healthspan, behavior and fitness. All tests were carried out during the light phase, and experimenters were blinded to the treatment group. Electrocardiography (ECG) was performed non-invasively using the ECGenie system (Mouse Specifics). For acclimatization, each mouse was placed on the training platform for 10 minutes and then placed on the measurement platform on interchangeable electrodes that transmit the electrical signals to a computer for a maximum of 10 minutes. Signals with a heart rate variation that exceeded 35 bpm were excluded, and average heart rate per mouse was evaluated. Motor coordination was assessed using a Rotarod (TSE Systems, type 3375-M5). ), and the revolving rate was continuously increased to 40 r.p.m. over a period of 300 s. The total time that each mouse spent on the rod, with a cutoff time of 300 s, was measured. The test was performed on four consecutive days, and only data from day four were used for the analysis. Endurance was measured using a treadmill (TSE Systems, type 3033401-M-04/C). Mice were acclimatized to the experimental setup for 5 minutes under low speed (0.1 m s−1). A progressively accelerating speed was applied from 0.1 m s−1 for another 10 minutes to 1.3 m s−1 within 60 minutes. A mild electroshock (0.3 mA) was applied for 5 s, followed by 5 s of refractory period as soon as mice slowed down beyond a critical point and crossed the laser beam for more than 2 s to ensure that mice stopped running only upon exhaustion. Exhaustion was defined as the willingness of a mouse to withstand three consecutive shocks instead of continuing running. For the open field experiment using a ActiMot2 (TSE Systems), mice were placed in a 50 cm × 50 cm × 40 cm box and were allowed to freely explore for 10 minutes. The test chamber was illuminated to 20–30 lux. Total activity, speed and time spent in the center were recorded via infrared sensors. Mice were placed in a Y-shaped maze and their activity recorded for 5 minutes using a video tracking system (TSE Systems, VideoMot). Percent alternating arm visits were calculated as a read-out for spatial working memory. Experiments were performed in a randomized and blinded fashion whenever possible. For the survival, phenotyping and tissue collection cohorts, female mice were randomly allocated to cages after weaning using simple randomization. Male mice were weaned litter-wise to reduce aggressive behavior. If male mice of different litters had to be combined, a ratio of 2:3 was preferred over 4:1. Cages were assigned to treatment groups using simple randomization. Tissue samples used for histopathology, RNA-seq, plasma protein and trametinib level measurements were randomized prior to extraction/analysis using simple randomization. Mouse survival data and postmortem pathology were scored by mouse caretakers, who were unaware of the study design. All phenotyping experiments were performed with the experimenters blinded to treatment. Histopathology was scored in a blinded manner by an external pathologist. Plasma protein levels and RNA-seq measurements were performed by external companies in a blinded manner. Confocal imaging and image analysis were performed in a blinded manner. 18-FDG-CT/PET scans were measured by an external facility with the experimenters unaware of study design. In ECG measurements, signals with a heart rate variation that exceeded 35 bpm were excluded. Three samples were excluded from the RNA-seq analysis (1× male muscle trametinib, 1× female muscle combined and 1× female kidney rapamycin), as they presented as clear outliers in the principal component analysis (PCA). Otherwise, no data were excluded from the analyses. No statistical methods were used to pre-determine sample sizes, but our sample sizes are similar to those reported in previous publications10,22. Statistical analysis was performed using GraphPad Prism 9.0 and 10.3.0, except for Cox proportional hazard analysis, which was performed in R. Number of animals and statistical tests are indicated in the figures and figure legends. For multiple comparison testing, one-way and two-way ANOVA were followed by Bonferroni post hoc test. To test for differences in the proportion of mice affected by pathologies, chi-square test and Poisson regressions were used. Comparisons were performed against control mice as reference. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw and processed RNA-seq data are available in the Gene Expression Omnibus under accession number GSE288795. Raw data for the Olink plasma proteome analysis are provided in Supplementary Table 1. All other data are available from the corresponding authors upon reasonable request. Source data are provided with this paper. & Sabatini, D. M. mTOR at the nexus of nutrition, growth, ageing and disease. Lopez-Otin, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. Laskovs, M., Partridge, L. & Slack, C. Molecular inhibition of RAS signalling to target ageing and age-related health. Biological mechanisms of aging predict age-related disease co-occurrence in patients. Partridge, L., Fuentealba, M. & Kennedy, B. K. The quest to slow ageing through drug discovery. Bjedov, I. et al. Mechanisms of life span extension by rapamycin in the fruit fly Drosophila melanogaster. Flynn, J. M. et al. Late-life rapamycin treatment reverses age-related heart dysfunction. & Zheng, P. mTOR regulation and therapeutic rejuvenation of aging hematopoietic stem cells. Stephen, A. G., Esposito, D., Bagni, R. K. & McCormick, F. Dragging Ras back in the ring. Slack, C., Giannakou, M. E., Foley, A., Goss, M. & Partridge, L. dFOXO-independent effects of reduced insulin-like signaling in Drosophila. Fabrizio, P. et al. SOD2 functions downstream of Sch9 to extend longevity in yeast. The Ras-Erk-ETS-signaling pathway is a drug target for longevity. Borras, C. et al. RasGrf1 deficiency delays aging in mice. & Sakai, T. Antitumor activities of JTP-74057 (GSK1120212), a novel MEK1/2 inhibitor, on colorectal cancer cell lines in vitro and in vivo. Castillo-Quan, J. I. et al. A triple drug combination targeting components of the nutrient-sensing network maximizes longevity. Miller, R. A. et al. Rapamycin-mediated lifespan increase in mice is dose and sex dependent and metabolically distinct from dietary restriction. Transient rapamycin treatment can increase lifespan and healthspan in middle-aged mice. Intermittent rapamycin feeding recapitulates some effects of continuous treatment while maintaining lifespan extension. The impact of infection and inflammation in oncologic 18F-FDG PET/CT imaging. Evaluation of organ glucose metabolism by 18F-FDG accumulation with insulin loading in aged mice compared with young normal mice. Impact of aging on hippocampal function: plasticity, network dynamics, and cognition. Backes, H. et al. Glucose consumption of inflammatory cells masks metabolic deficits in the brain. Ayata, P. et al. Epigenetic regulation of brain region-specific microglia clearance activity. & Godbout, J. P. TGFβ produced by IL-10 redirected astrocytes attenuates microglial activation. Salminen, A., Kaarniranta, K. & Kauppinen, A. Inflammaging: disturbed interplay between autophagy and inflammasomes. Zhang, P., Catterson, J. H., Gronke, S. & Partridge, L. Inhibition of S6K lowers age-related inflammation and increases lifespan through the endolysosomal system. Aw, D. et al. Disorganization of the splenic microanatomy in ageing mice. Kawanishi, N. & Machida, S. Alterations of macrophage and neutrophil content in skeletal muscle of aged versus young mice. Sanjurjo, L., Aran, G., Roher, N., Valledor, A. F. & Sarrias, M. R. AIM/CD5L: a key protein in the control of immune homeostasis and inflammatory disease. Zhou, H. et al. Lactate-induced CCL8 in tumor-associated macrophages accelerates the progression of colorectal cancer through the CCL8/CCR5/mTORC1 axis. Ren, Y., Yu, M., Zheng, D., He, W. & Jin, J. Lysozyme promotes renal fibrosis through the JAK/STAT3 signal pathway in diabetic nephropathy. Gowhari Shabgah, A. et al. Chemokine CXCL14; a double-edged sword in cancer development. Ullah, A. et al. A narrative review: CXC chemokines influence immune surveillance in obesity and obesity-related diseases: type 2 diabetes and nonalcoholic fatty liver disease. The CXCL7/CXCR1/2 axis is a key driver in the growth of clear cell renal cell carcinoma. Tartey, S. et al. Ets-2 deletion in myeloid cells attenuates IL-1α-mediated inflammatory disease caused by a Ptpn6 point mutation. Ghosh, D., Tsokos, G. C. & Kyttaris, V. C. c-Jun and Ets2 proteins regulate expression of spleen tyrosine kinase in T cells. Muniz-Bongers, L. R. et al. MMP2 and TLRs modulate immune responses in the tumor microenvironment. Kim, K. M. et al. MMP2-A2M interaction increases ECM accumulation in aged rat kidney and its modulation by calorie restriction. Dufour, A. et al. C-terminal truncation of IFN-γ inhibits proinflammatory macrophage responses and is deficient in autoimmune disease. Motta, J. P. et al. Epithelial production of elastase is increased in inflammatory bowel disease and causes mucosal inflammation. Identification of key pathways and candidate genes in pancreatic ductal adenocarcinoma using bioinformatics analysis. & Lamming, D. W. Targeting the biology of aging with mTOR inhibitors. Infante, J. R. et al. Safety, pharmacokinetic, pharmacodynamic, and efficacy data for the oral MEK inhibitor trametinib: a phase 1 dose-escalation trial. Wei, B. R. et al. Efficacy, tolerability, and pharmacokinetics of combined targeted MEK and dual mTORC1/2 inhibition in a preclinical model of mucosal melanoma. Miyaji, M. et al. Genetic evidence for the role of Erk activation in a lymphoproliferative disease of mice. Sadagurski, M., Cady, G. & Miller, R. A. Anti-aging drugs reduce hypothalamic inflammation in a sex-specific manner. Sex differences in pharmacological interventions and their effects on lifespan and healthspan outcomes: a systematic review. Reduced insulin signaling in neurons induces sex-specific health benefits. Jais, A. et al. Myeloid-cell-derived VEGF maintains brain glucose uptake and limits cognitive impairment in obesity. We thank S. Veugelen for her help in establishing the mouse brain stainings. We acknowledge A. Mesaros and M. Purrio from the phenotyping facility of the Max Planck Institute for the Biology of Ageing for outstanding technical help and advice. We are very grateful to the Comparative Biology facility of the Max Planck Institute for Biology of Ageing for mouse housing. O. Hendrich, S. Buschbaum, A. Pahl, J. Fröhlich and R. Jansen are acknowledged for technical assistance with mouse experiments. Imaging was performed in the FACS & Imaging Core facility at the Max Planck Institute for Biology of Ageing. Figures 1a and 2a were created in BioRender: https://BioRender.com/eh02p6o. Open access funding provided by Max Planck Society. Lisonia Gkioni, Tobias Nespital, Maarouf Baghdadi, Carolina Monzó, Jitin Bali, Joris Deelen, Sebastian Grönke & Linda Partridge Institute for Integrative Systems Biology, Spanish National Research Council, Paterna, Spain Cellular Networks and Systems Biology Group, CECAD Research Centre, Cologne, Germany You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Conceptualization of the study was carried out by T. Nespital, S.G. and L.P. Statistical evaluation of data was done by L.G., C.M., H.B., T. Nassr and S.G. Writing of the manuscript was done by L.G., S.G. and L.P. Correspondence to Sebastian Grönke or Linda Partridge. The authors declare no competing interests. Nature Aging thanks Pankaj Kapahi, Parminder Singh, Dipti Verma and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a–c Western blot analysis of Erk1/2 phosphorylation as a proxy for Ras/Mek/Erk pathway activity in a kidney, b spleen and c muscle of females and males after 4 weeks of treatment with 0, 1.44 and 11.52 mg/kg trametinib. Representative Western blot (upper panel) and corresponding quantification (lower panel) (n = 5 tested organs of individual animals per treatment and sex). Significance was tested by One-Way ANOVA and Bonferroni post hoc test. d, Water consumption of female and male mice was not significantly changed. e,f, Plasma level of Aspartate transaminase (AST) (e) and Alkaline Phosphatase (ALP) (f) as biomarkers of liver health were not significantly changed. N = 5 per treatment for both sexes. Data in d-f are presented as mean ± SEM. Survival curves of a–c female and d–f male mice of the a, d lifespan, b, e tissue collection and c, f phenotyping cohort. The corresponding pooled data are presented in Fig. Trametinib significantly extended lifespan in male mice and the combined treatment significantly increased lifespan compared to the individual drug treatments in both males and females when only survival of the lifespan cohort was analyzed. Survival curves for control and rapamycin treated mice of the lifespan cohort were previously published22. Within the different cohorts all treatments were run in parallel. a, b body weight, c–f ECG measurement of c, d heart rate and e, f QT-interval, g–j open field assay measurement of g, h centre occupancy and i, j movement speed, k, l distance on treadmill, m, n time on rotarod, o, p Y maze measurement of o % activity in new arm and p global speed. Data were analysed using a mixed effect model testing for the interaction between Rapamycin x Trametinib x Age followed by a Bonferroni post hoc test to test for significance changes between 12 and 22 months indicated by p values above the connecting lines and between drug treatments and control at 12 or 22 months indicated by p values next to the dots. N numbers (12 M, 22 M) for: female body weight, rota rod: Control (15/13), Rapamycin (15/13), Trametinib (15/10) Rapamycin/Trametinib (15/15); male body weight, rota rod: Control (14/10), Rapamycin (15/13), Trametinib (15/13), Rapamycin/Trametinib (15/15); female heart rate, QT-interval: Control (15/9), Rapamycin (15/10), Trametinib (8/10) Rapamycin/Trametinib (7/7); male heart rate, QT-interval: Control (15/9), Rapamycin (15/10), Trametinib (8/10) Rapamycin/Trametinib (7/7). N numbers (22 M female, 20 M male) for: open field: Control (13,11), Rapamycin (14,13), Trametinib (12,13) Rapamycin/Trametinib (15,15); treadmill: Control (13,11), Rapamycin (14,13), Trametinib (12,13), (Rapamycin/Trametinib (15,15). Metabolic cages were used to measure d water uptake e food uptake relative to body weight and f, g respiratory exchange ratio (RER) during f day and g night. Measurements were done in batches between the age of 15–18 months. Rapamycin and combined treatment significantly reduced body weight in male but not female mice, but had no effect on water or food uptake. Fat content was significantly increased upon combined treatment in females. Lean mass was significantly decreased in trametinib and combined treated females. Males showed no significant differences in fat or lean mass. All drug treatments significantly reduced night-time RER in males but not females. h glucose, i aspartate transaminase (AST) and j alkaline phosphatase (ALP) level in blood plasma of 24 months old animals. Rapamycin but not trametinib significantly increased plasma glucose levels in male and female mice. Combination of trametinib and rapamycin did not further increase plasma glucose levels compared to rapamycin treated animals. One-Way ANOVA with Bonferroni post hoc test. Numbers in brackets indicate the number of plasma samples of different animals assessed per treatment. Data are presented as mean ± SEM. Quantification of immunohistochemical stainings measuring the density of activated microglia (DAPI/Iba-1 positive cells) and astrocyte (DAPI/GFAP positive cells) in a, b hippocampus, c, d cortex and e, f cerebellum of 6-months old control animals (young) and 24 months-old control and drug-treated females. Microglia and astrocyte density significantly increased with age in the hippocampus and cortex of control animals, but not in the cerebellum. Single drug treatment did not significantly reduce microglia or astrocyte density in the hippocampus, cortex or cerebellum. g, h Brain levels of trametinib increased in a dose-dependent manner upon feeding g female or h male animals for 4 weeks with food containing 0, 0.29, 0.58, 1.44, 2.88 or 11.52 mg/kg of drug. N = 5 brain samples were measured via mass spectrometry per treatment. Only samples with significant trametinib levels above background are shown. Trametinib was undetectable in the brain of animals fed with 0, 0.29 and 0.58 mg/kg trametinib, and only very low levels were detected in animals fed with 1.44 and 2.88 mg/kg trametinib. One-Way ANOVA with Bonferroni post hoc test. Numbers in brackets indicate the number of brain regions of different animals analyzed per treatment. Data are presented as mean ± SEM. Cross sectional histopathological analysis of non-neoplastic lesions in tissues isolated from 24 months old male and female mice treated with rapamycin, trametinib and rapamycin/trametinib. a, b heart hypertrophy, c, d spleen pathology, e, f spleen weight, g, h kidney glomerulopathology, i, j gonadal pathology and k, l liver lipidosis. a, b Rapamycin and Trametinib treatments did not significantly affect heart hypertrophy. c, d Females and males treated with Rapamycin/Trametinib showed reduced spleen pathology and a similar trend was observed upon Rapamycin treatment, which was significant in males. g, h Kidney glomerulopathology was not significantly affected by the drug treatments. i, j Males but not females treated with Rapamycin and Rapamycin/Trametinib showed a significant increase in gonadal pathology. k, l Male mice treated with Rapamycin showed a significant increase in liver lipidosis compared to control animals. A similar non-significant trend was also observed in female animals and upon Rapamycin/Trametinib treatment in males and females. Data in g, h are plotted as mean with SEM and statistical analysis was done using One-Way ANOVA followed by Bonferroni post hoc test. Number of scored tissues per treatment are indicated in brackets. Control and Rapamycin data in a–l were published previously22. Expression level of inflammation associated genes based on the RNA seq analysis of kidney, spleen and muscle tissue from 24-months-old rapamycin, trametinib and combination treated animals. a–c Cd5l was significantly downregulated in a kidney, b spleen and c muscle upon combination treatment. d, e Ccl8 was significantly downregulated by combination treatment in d kidney and by rapamcin and combination treatment in e spleen and f male muscle. g, h Lyzosyme1 (Lyz1) expression was decreased in g kidney upon trametinib and combination treatment, and h in female spleen upon rapamycin, trametinib and combination treatment. i, j Cxcl7 was significantly downregulated in i male kidney and j male muscle upon combination treatment, and by rapamycin in male muscle. k Cxcl1 expression level was decreased in the female spleen upon rapamycin and combination treatment. l Cxcl13 was significantly downregulated by rapamycin in male muscle tissue. m, n Expression of Cxcl14 was upregulated by rapamycin and the combination treatment in m male and female muscle. n Cxcl14 was significantly downregulated in the female spleen. Data are presented as Mean ± SEM. Number of scored tissues per treatment are indicated in brackets. Expression level of potential ETS target genes based on the RNA seq analysis of kidney, spleen and muscle tissue from of 24-months-old rapamycin, trametinib and combination treated animals. a Expression of SYK was significantly decreased in the kidney of female mice treated with rapamycin and trametinib, and b in male muscle upon rapamycin and combination treatment. c Mmp2 was downregulated in the female spleen upon rapamycin and combination treatment, d Mmp13 expression was decreased in the female spleen upon combination treatment. g Mmp12 levels were increased upon rapamycin and combination treatment in the spleen. h Cela2a and i Cela3b levels were specifically downregulated in male spleens upon single and combination drug treatment. j Cela1 expression was downregulated in male muscle upon rapamycin treatment. Data are presented as Mean ± SEM. Number of scored tissues per treatment are indicated in brackets. Raw data of plasma proteins measured via the Olink Target 96 Mouse Exploratory panel in 24-month-old male and female mice. Significantly regulated genes based on the RNA-seq analysis in 24-month-old mice treated with rapamycin, trametinib and the combination. Unprocessed western blot images corresponding to Fig. Unprocessed western blot images corresponding to Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/upgraded-very-large-array-telescope-will-spot-baby-solar-systems-if-its/'>Futuristic Radio Telescope Will Spot Baby Solar Systems—If It's Funded</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-28 10:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Futuristic Radio Telescope Will Spot Baby Solar Systems—If It's Funded A new telescope project called the Next-Generation Very Large Array will revolutionize radio astronomy if it gets the funding it needs A prototype antenna for the proposed Next Generation Very Large Array (ngVLA) radio telescope in New Mexico. If funded, the project would greatly expand U.S. radio astronomy capabilities. New Mexico's Plains of San Agustin are otherworldly: Silence, sand and sharp plants reign on the valley floor. Pronghorns' legs and jackrabbits' ears break up the landscape. And so, too, does one of the world's largest telescopes. The plains house the aptly named Very Large Array (VLA)—a radio telescope made of 27 different antennas, each of which looks like a home satellite dish on steroids. In the otherwise empty desert, they spread into a Y shape that can extend 22 miles end-to-end. In this case, then, images from the VLA have as much resolution as they would if it were a single telescope 22 miles wide: high definition, in other words. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. This federally-funded organization builds, maintains and operates radio telescopes that any astronomer—regardless of their institutional affiliation or citizenship—can apply to use. But the VLA, now in its middle age, is due for a replacement. After all these decades, astronomers want something shiny, fully modern and more capable: a new build with all the bells and whistles rather than a charming old Colonial that's been remodeled piecemeal. (Astronomers may be scientifically creative, but they are linguistic straight shooters.) On a Friday afternoon in late April, the organization gathered political leaders together, alongside scientists and engineers, to unveil a prototype antenna—one that will be cloned a couple of hundred times to make up the future ngVLA. “The amount that technology has advanced since the VLA was created is amazing,” says Jill Malusky, NRAO's news and public information manager. At the moment, it's a proposed project—and still requires final funding. “Having a physical antenna we can point to, and test, to prove the value of this project is such a milestone,” Malusky says. But whether big-science telescopes, radio or otherwise, will survive the current funding environment remains a dark matter. “It's a mix of excitement and trepidation,” she says. “Can we get people invested in the potential of a major project that is still gathering resources and just over a decade to fruition?” That Friday afternoon, Tony Beasley, director of NRAO, stood at the front of a hardy event tent and faced the prototype. Its dish was made up of shiny panels assembled into an octagon. From its bottom edge, supportive struts held up a secondary reflecting surface and a receiver (basically the radio version of an optical telescope's camera) that looked a bit like the spaceship Foster's character boarded in Contact. The antenna, about as wide as a bowling lane is long, has been designed to collect radio waves from space—beamed from stars that are being born or dying, the stuff between stars, and more. As a start, the prototype dish will hook up to VLA's aging ones and gather data alongside them—it will be an apprentice of sorts. “You see one antenna out there,” said Beasley, directing the audience's attention beyond the tent, which was being shaken by the wind to such an extent that people also cast their eyes upward to assess its structural integrity. Together, the ngVLA's antennas could pick up a cell-phone signal from 500 billion kilometers (more than 310 billion miles) away (although that wouldn't be the most likely find). The future telescope's resolution should be high enough to pass a no-glasses eye exam in New York City if the chart of letters were placed in Los Angeles. That precision gives it scientific latitude, allowing it to address some of astronomers' highest-priority questions, such as how planets come to be and how solar systems like ours form. At high radio frequencies and big antenna separations, “it would be the only game in town.” The ngVLA will also look for the organic molecules and chemical conditions of new solar systems that might someday spur life. Given those varied abilities, the telescope was highly ranked in astronomers' “decadal survey,” a yearslong process in which the astronomical community takes stock of its most valued scientific questions and assesses which future telescopes are best suited to find some answers. The possibility is a bright spot for American radio astronomy. And NRAO's latest instrument, the Atacama Large Millimeter/submillimeter Array, opened 12 years ago. The new telescope does, though, have a whippersnapper nipping at its heels. Another future radio observatory, called the Deep Synoptic Array 2000 (DSA-2000), is planning an order of magnitude more dishes than the ngVLA—2,000 of them. DSA-2000 will also work at a different radio frequency range than the ngVLA. DSA-2000's development is also moving faster than that of the VLA's successor, though, in large part, that is because the former has relied on private funding more than federal resources, as the ngVLA's prototyping has. “NSF is an extremely important investment to make U.S. scientific dominance a reality. President Donald Trump hasn't said much about that particular domain yet. But not building the ngVLA could put that edge in jeopardy. Dobbs, though, holds out hope for the U.S.'s role in radio astronomy's future, in part because of the propulsion of its past. “The United States has everything it needs to make that project a reality,” he adds. Whether it will do so, though, requires gathering more data from the future. After all, it's bad luck to count your antennas before they hatch. Dobbs has been putting his focus on smaller radio telescopes, such as one called the Canadian Hydrogen Intensity Mapping Experiment (CHIME) and its successor, acronymed CHORD. Both map how hydrogen was distributed in the early universe and detect fast radio bursts. At the prototype-antenna unveiling, then, it made sense that there was a liminal feeling to what was otherwise a celebratory gathering. Chris Smith, interim director of the NSF's division of astronomical sciences, did send a letter to be read to the wined-and-dined crowd. “NSF funded this development not just to ensure the technical feasibility of the advanced capabilities of ngVLA,” he wrote. Her newest book is Countdown: The Blinding Future of Nuclear Weapons (Bold Type Books, 2024). Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            