
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69797512/xenobots-conscious-cells/'>A ‘Third State' Exists Between Life and Death—And That Suggests Your Cells Are Conscious, Some Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 17:07:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A growing number of new studies have found that, at least for some cells, death isn't the end, but the beginning of something wholly unexpected. But when you examine existence at the cellular level, things get a bit more interesting. You, me, and all of the 108 billion or so Homo sapiens who've ever walked the Earth have all been our own constellation of some 30 trillion cells. Each of our bodies is a collective organism of living human cells and microbes working in cooperation to create what our minds view as “life.” However, a growing number of new studies have found that, at least for some cells, death isn't the end. Instead, it's possibly the beginning of something new and wholly unexpected. A growing snowball of research concerning a new class of AI-designed multicellular organisms known as “xenobots” is gaining scientific attention for their apparent autonomy. Xenobots are cells that form new roles beyond their original biological function—for example, using hairlike cilia for locomotion rather than transporting mucus. Because they appear to reassemble into this new form and function, the authors argue that xenobots form a kind of “third state” of life, wherein cells can reorganize after the death of an organism to form something new. Experiments with human cells, or “anthrobots,” exhibit this behavior, too. “The third state suggests that [an organism's] death may play a significant role in how life transforms over time.” The implications for these cellular robots, or biobots, are pretty big—imagine tailor-made medicines crafted from your own tissues to avoid a dangerous immune response. At least, that's what evolutionary biologist and physician William Miller thinks. Miller believes that xenobots are just another example of how we don't give credit to the inherent cognitive—or even conscious—abilities of the cells that make up our bodies. “We, as humans, have very limited capacity and finely honed ability to see intelligence in medium-sized objects moving at medium speeds through three-dimensional space,” says Tufts University developmental and synthetic biologist Michael Levin, PhD, in a video exploring cellular intelligence. For Miller, the concept of a sentient cell is a fundamental sea change in biology that challenges some Neo-Darwinian ideas like “survival of the fittest.” Because cells must work in concert to be successful, a more accurate microbial catchphrase might be “I serve myself best by serving others,” Miller says.Putting the intelligent cell at the center of biology “spills out an entirely new biological narrative where genes are not controlling, genes are tools. In which we understand why organisms choose to stick together in their trillions, to solve problems, [for] decision-making, mutual support, partnerships, synergies, co-dependencies, collaboration—it's not survival of the fittest,” Miller says. Many scientists aren't sold on this brave new future for biology. A 2024 letter published in the journal EMBO Reports] describes CBC theory as “merely an intellectual exercise without empirical evidence” and the authors remain equally skeptical of consciousness claims regarding xenobots or other “third state” organisms. “It's been known for maybe 75 years or more that cells can be induced to develop abnormally when taken out of context and cultured in vitro. “When an insect herbivore secretes hormones into plant leaves, causing the leaves to form galls [abnormal growths] that serve as houses for the insect, is that a ‘third state' of life?” Taiz has also tackled what he describes as “myths” surrounding plant consciousness and co-authored a review in 2019 titled “Plants Neither Possess nor Require Consciousness.” “And the key for your hypothesis is that it has to be falsifiable.” When cells are taken out of context and are no longer exchanging information or signals from nearby cells, different genes can be expressed than what's normal, Peer says. Simply put, the xenobots are an advanced version of “animal caps,” a well-known technique in developmental biology in which cells retain the ability to differentiate into other cells. However, despite this disagreement, both groups agree on at least one important point—understanding cells and exploring their many capabilities is a huge opportunity. Taiz compares the potential use of anthrobots in medicine to humans behaving as their own “gall-forming insects in plants,” via altering the development of stem cells to create particular cell behaviors. Conscious or not, it looks like cells will undoubtedly play a starring role in the unfolding future of human health. You Might Be Able to Time Travel in Your Mind Anesthesia May Be a Portal to the Universe Inside You</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251217082505.htm'>Scientists rewired Down syndrome brain circuits by restoring a missing molecule</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 17:00:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research suggests that disrupted brain circuits in Down syndrome may be linked to a shortage of a specific molecule that the nervous system relies on to develop and work properly. The team says that bringing this molecule back, known as pleiotrophin, might help support brain function in Down syndrome and potentially other neurological conditions, possibly even later in life. This raises the possibility of an advantage over earlier strategies aimed at strengthening Down syndrome related brain circuits, which would have required action during very narrow windows in pregnancy. "This study is really exciting because it serves as proof-of-concept that we can target astrocytes, a cell type in the brain specialized for secreting synapse-modulating molecules, to rewire the brain circuitry at adult ages," said researcher Ashley N. Brandebura, PhD, who was part of the research team while at the Salk Institute for Biological Studies and is now part of the University of Virginia School of Medicine. "This is still far off from use in humans, but it gives us hope that secreted molecules can be delivered with effective gene therapies or potentially protein infusions to improve quality of life in Down syndrome." Down syndrome affects about 1 in 640 babies born each year in the United States, according to the federal Centers for Disease Control and Prevention. It results from an error in cell division during development and can be associated with developmental delays, hyperactivity, a shorter lifespan, and a higher risk of health issues that can include heart defects, thyroid problems, and hearing or vision difficulties. Salk scientists led by Nicola J. Allen, PhD, set out to learn more about what drives Down syndrome by examining proteins inside brain cells in mouse models of the condition. The researchers also noted that pleiotrophin levels are reduced in Down syndrome. Viruses are often associated with illnesses like the flu, but researchers can modify them so they do not cause disease and instead carry helpful material. In this case, the virus was stripped of harmful components and loaded with beneficial cargo -- pleiotrophin -- so it could deliver the molecule directly into cells. The scientists reported that supplying pleiotrophin to astrocytes, a major type of brain cell, produced substantial effects. "This could one day allow us to rewire faulty connections and improve brain performance." They say more work is needed to understand the many contributors involved. Still, they argue that the results show the approach itself can work, and that it might eventually help beyond Down syndrome, including in other neurological diseases. "This idea that astrocytes can deliver molecules to induce brain plasticity has implications for many neurological disorders, including other neurodevelopmental disorders like fragile X syndrome but also maybe even to neurodegenerative disorders like Alzheimer's disease," Brandebura said. "If we can figure out how to 'reprogram' disordered astrocytes to deliver synaptogenic molecules, we can have some greatly beneficial impact on many different disease states." Support came from the Chan Zuckerberg Initiative and the National Institutes of Health's National Institute of Neurological Disorders and Stroke, grant F32NS117776. Titan's Hidden Ocean May Not Exist and That Changes Everything This New AI Is Cracking the Hidden Laws of Nature Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69798349/viking-silver-farm-treasure-found/'>Archaeologists Thought They Found Wires Buried on a Farm. It Was Actually Viking Treasure.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 16:29:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Their affinity for the metal gives discoveries of silver artifacts significant historical value—much like a treasure recently found in the mountains of Norway, one that had sat undisturbed since the 9th century AD. Before farmer Tårn Sigve Schmidt was able to carve a new road for his tractor on his mountainous farm near Årdal, northwest of Oslo, he had to call in archaeologists to make sure there were no unique areas he was about to disturb. Seven inches beneath the floorboards of what was likely once a house for Viking slaves, a team of archaeologists found four heavy silver bracelets, all with different decorations, likely from over 1,100 years ago. “At first, I thought it was a question of some twisted copper wires that you can often find in agricultural land,” field archaeologist Ola Tengesdal Lygre said in a translated statement from the University of Stavanger, “but when I saw that there were several next to each other and that they were not copper at all, but silver, I realized that we had found something exciting.” Further investigation showed that, at one point, a “large and powerful Viking farm” was located on the property, with multiple houses and shelter for animals. The excavation crew also found soapstone pots, rivets, knife blades, and whetstones for sharpening tools. As a rule, such valuable objects are discovered on fields that have been plowed, where an object has been completely taken out of its original context. Since the silver hoard has not been moved, it can give us completely new insights into life and society in the Viking Age.” X-rays were taken, and the team will test soil samples to help give a broader picture of the setting, including if the silver was wrapped in a cloth when it was buried. Experts believe that the farm was the victim of arson at one point, likely coinciding with a period of unrest during the Viking Age in Norway, which stretched from from 800 AD to 1050 AD. “If people who lived on this farm had to flee from an attack,” Demuth said, “it would be natural to hide away the valuables you had before escaping to the mountains. With no silver mines in Norway at the time of the stashing, any silver owned by the Vikings had to come from abroad, either through trade, as a gift, or even as loot from a raid. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. China's Regreening Has Messed Up Its Water Cycle Animals Keep Evolving Into Crabs, Which Is Weird A Volcano May Have Started the Black Plague</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09873-4'>Transient hepatic reconstitution of trophic factors enhances aged immunity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 16:07:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Ageing erodes human immunity, in part by reshaping the T cell repertoire, leading to increased vulnerability to infection, malignancy and vaccine failure1,2,3. Attempts to rejuvenate immune function have yielded only modest results and are limited by toxicity or lack of clinical feasibility1,3,4,5. Here we show that the liver can be transiently repurposed to restore age-diminished immune cues and improve T cell function in aged mice. These immune cues were found by performing multi-omic mapping across central and peripheral niches in young and aged animals, leading to the identification of Notch and Fms-like tyrosine kinase 3 ligand (FLT3L) pathways, together with interleukin-7 (IL-7) signalling, as declining with age. Delivery of mRNAs encoding Delta-like ligand 1 (DLL1), FLT3L and IL-7 to hepatocytes expanded common lymphoid progenitors, boosted de novo thymopoiesis without affecting haematopoietic stem cell (HSC) composition, and replenished T cells while enhancing dendritic cell abundance and function. Treatment with these mRNAs improved peptide vaccine responses and restored antitumour immunity in aged mice by increasing tumour-specific CD8+ infiltration and clonal diversity and synergizing with immune checkpoint blockade. These effects were reversible after dosing ceased and did not breach self-tolerance, in contrast to the inflammatory and autoimmune liabilities of recombinant cytokine treatments6,7. These findings underscore the promise of mRNA-based strategies for systemic immune modulation and highlight the potential of interventions aimed at preserving immune resilience in ageing populations. Ageing has a profound effect on the immune system, including the T cell repertoire, leading to reduced immune resilience1,2,3. Central to this decline in humans and most other mammals is the involution of the thymus. Thymic involution curtails naive T cell output, contracts T cell receptor (TCR) repertoire diversity and blunts primary responses, whereas peripheral T cells accrue dysfunctional states that heighten susceptibility to infection, vaccine failure and cancer. Efforts to counter immune ageing have primarily focused on reversing thymic involution through hormones8, cytokines9, small molecules10 and heterochronic parabiosis11, or by directly modulating haematopoiesis12. Although these strategies have provided valuable insights into immune ageing, they have been limited by effect size, toxicity or clinical feasibility4,5. Here we describe an approach for reconstituting thymus-derived factors in the liver to address age-related immune decline (Fig. We first identified signalling pathways in the thymus and peripheral blood T cells that decline with age. We then delivered mRNAs encoding these factors (DLL1, FLT3-L and IL-7) to the liver using lipid nanoparticles (LNPs). We found that this approach significantly improved immune response in ageing mice in both vaccination and cancer immunotherapy models with no adverse side effects or evidence of increased autoimmunity. These results highlight the potential of this approach to improve immune function and, more broadly, to use the liver as a transient ‘factory' for replenishing factors that decline with age. a, Overview of the approach to restore age-declining immune trophic cues by hepatic expression of Dll1, Flt3l and Il7 mRNAs. b, Spatial ligand–receptor interactions between thymic cortical epithelial cells (cTECs) and thymocytes decline with age (left), and ssGSEA shows reduced Notch pathway activity in circulating T cells (right). n = 47 spatial arrays and 96,683 blood T cell transcriptomes across 21 ages. Data are represented as violin plots with median + interquartile range. Statistical significance was determined by Mann–Whitney tests. c, cTEC–T cell IL-7 interaction (left) and downstream pathway in circulating T cell (right) activities are likewise diminished with age. n = 47 spatial arrays and 96,683 blood T cell transcriptomes across 21 ages. Data are represented as violin plots with median + interquartile range. Statistical significance was determined by Mann–Whitney tests. d, Thymus weight decreases with age (n = 18; 3 per timepoint). Interstitial FLT3-L levels are reduced in aged thymus by ELISA (n = 3 per group). ; statistical significance was determined by a two-tailed unpaired Student's t-test. e, mRNA (DFI; Dll1, Flt3l and Il7) constructs formulated in SM-102 LNPs. f, Representative RIBOmap images 6 h post-DFI show robust ribosome-bound transcripts in the liver. A representative image from three imaged DFI-treated animals is shown. h, Immunofluorescence of DLL1 protein over 0–48 h after 5 µg DFI reveals transient induction in total liver and hepatocyte surface (phalloidin co-stain). Fold induction from baseline (0 h) is shown. i, ELISA for FLT3-L levels in serum and the liver after 10 µg recombinant FLT3-L or 5 µg DFI at 3–48 h. Liver concentrations were normalized to liver weight; fold change from 0 h is shown. ; area under the cover (AUC) over 48 h compared by a two-tailed unpaired Student's t-test. j, ELISA for IL-7 in serum and the liver after 10 µg recombinant IL-7 or 5 µg DFI at 3–48 h with liver normalization and fold change from 0 h. n = 3 animals per time point per compartment per condition. ; 48-h AUC compared by a two-tailed unpaired Student's t-test. A functional T cell repertoire depends on pro-survival and trophic cues: cytokines, hormones and self-peptide MHC, which are abundant in healthy individuals but diminish with age, contributing to immune decline13,14,15. To nominate potentially immune restorative cues, we profiled trophic signalling across the mouse lifespan using single-cell RNA sequencing (scRNA-seq) or TCR-seq of peripheral blood T cells and spatially resolved Slide-TCR-seq on the thymus, the principal organ of T cell production16. After quality control, the dataset comprised approximately 97,000 circulating T cell transcriptomes (CD4+ and CD8+) and 47 spatial arrays covering approximately 1.26 million thymic positions spanning the day of birth to 90 weeks of age (Extended Data Fig. Using spatial proximity to infer bona fide cell–cell interactions (rather than co-regulation alone), we used permutation-based null models to identify age-dependent receptor–ligand pairs between thymocytes and thymic epithelial cells (TECs). Cortical TEC–T cell signalling declined markedly with age, whereas medullary TEC–T cell interactions were relatively preserved (Extended Data Fig. Integrating spatially informed interaction analysis with single-sample Gene Set Enrichment Analysis (ssGSEA) of peripheral blood-circulating T cells revealed age-linked attenuation of Notch1/3 and IL-7 signalling (Fig. Although thymic stromal cells primarily act locally to support T cell maturation and selection, several of their soluble products accumulate in the thymus, enter the circulation and contribute to systemic immune homeostasis13,19. Consistently, intrathymic as well as circulating T cells from aged mice showed reduced expression of Notch, IL-7 and FLT3-L target genes and diminished downstream activity (Fig. Soluble FLT3-L, produced in part by intrathymic fibroblasts and required to sustain T cell function in aged and post-transplant settings, also declined in thymic homogenates20,21 (Fig. These findings motivated us to transiently reconstitute age-diminished immune signalling using three immune trophic factors — DLL1 (to activate Notch), FLT3-L and IL-7 — in aged hosts. We selected DLL1 over DLL4 based on a more favourable safety and immunological profile: DLL4 is linked to angiogenesis and vascular remodelling and can overly restrict lymphoid differentiation, whereas we hypothesized that DLL1 supports T cell development without suppressing B cell output22,23. The unique haemodynamics and anatomical features of the liver enable priming and maintenance of adaptive immunity, and its protein-synthesis capacity is preserved even at advanced ages, making it a suitable site to modulate circulating T cells24,25,26. We chose mRNA delivery over recombinant proteins because recombinant cytokines clear rapidly, necessitating frequent high-dose administrations that often result in significant toxicity6,7. By contrast, mRNA allows for more controlled, transient protein production, and recent advances have further optimized mRNA stability and functionality while minimizing immune-related side effects for in vivo applications6,27,28,29. In addition, canonical Notch ligands are transmembrane proteins and require cell–cell contact, precluding soluble delivery. We therefore packaged mRNAs encoding DLL1 (Dll1), FLT3-L (Flt3l) and IL-7 (Il7) (collectively, DFI), or firefly luciferase (Luc) or GFP controls, in an SM-102 LNP formulation (Fig. All mRNAs were m1Ψ modified and 5′-m7GpppNm capped. Primary hepatocytes expressed DLL1 on the surface and secreted biologically active IL-7 and FLT3-L into the supernatant after transfection (Extended Data Fig. Systemic LNP administration predominantly targeted the liver with minimal translation in other tissues (Fig. In situ profiling of ribosome-bound (rather than endosome-trapped) transcripts (RIBOmap) confirmed robust translation of all three DFI mRNAs in hepatocytes in vivo (Fig. 4d,e), indicating hepatic expression and release with negligible off-target translation30. We next assessed pharmacokinetics and safety. Dll1 mRNA induced sustained DLL1 on the surface of hepatocytes for approximately 48 h (ref. mRNA-encoded FLT3-L achieved comparable serum half-life but higher single-dose systemic levels than recombinant FLT3-L (Fig. Intravenous recombinant IL-7 produced a sharp serum spike, whereas mRNA-encoded IL-7 yielded lower-amplitude, sustained levels with an approximately tenfold lower peak over 24 h (Fig. Because secreted IL-7 binds to heparan-sulfate proteoglycans, a substantial fraction of mRNA-delivered IL-7 remained in the hepatic extracellular matrix up to 24 h after serum levels normalized (Fig. 1j), potentially providing a compartmentalized source of IL-7 for cells traversing the sinusoids, where they also encounter hepatocyte-bound DLL1 (refs. Despite prolonged bioavailability of the three factors, we observed no changes in body weight, transaminases or liver function, and only minimal hepatic inflammation on histopathology after 4 weeks of DFI mRNA–LNPs in aged mice (Extended Data Fig. By contrast, recombinant FLT3-L and IL-7 administered on the same schedule (two doses per week for 28 days) induced marked elevations of GM-CSF, IL-10, IL-12 and IL-1β, underscoring the greater risk of systemic inflammation with recombinant cytokines (Supplementary Tables 2 and 3 and Supplementary Fig. Ageing is accompanied by a loss of naive T cells and accumulation of (virtual) memory and exhaustion-like states, most pronounced in CD8+ T cells and driven by thymic involution, chronic antigen exposure and inflammation1,17,18. We observed these shifts in our longitudinally profiled cohorts (Fig. This bias towards the memory phenotype in ageing compromises responses to new antigens. We therefore asked whether DFI rebalances naive T cell representation. a, Age-related decline in circulating naive T cells (Tcf7, Sell and Ccr7) and accumulation of exhausted-like T cells (Pdcd1, Ctla4, Lag3, Tigit, Entpd1 and Tnfrsf9) in mouse peripheral blood across age. Linear regression (red lines) shows Pearson's R and significance versus age. b–d, Flow cytometry of spleens from adult (6 weeks) and aged (72 weeks) mice treated twice weekly with DFI or control (Luc) mRNA–LNPs for 28 days. ; one-way ANOVA with Dunnett's post-hoc test. e, TCR repertoire diversity from bulk V(D)J sequencing (n = 5 adult, 12 aged + Luc and 9 aged + DFI). f, Rag2–eGFP tracing of thymocyte maturation (DN, double-positive (DP), single-positive (SP) CD4+ and SP CD8+) demonstrating increased thymopoiesis after DFI treatment. (error bands); repeated-measures two-way ANOVA with Dunnett's post-hoc test was used. g, Quantification of TCR excision circles (TRECs) in peripheral blood indicating recent thymic emigrants (n = 12 adult, 12 aged + Luc and 9 aged + DFI); data are mean ± s.e.m. ; one-way ANOVA with Tukey post-hoc test. h, Representative spatial STARmap projections of splenic dendritic cell (DC) subtypes from adult, Luc-treated and DFI-treated mice. i, Quantification of cDC1, cDC2 and other DC subsets per field of view (FOV; adult = 404, Luc = 386 and DFI = 408 FOVs from one animal per condition); data are represented as violin plots with median + interquartile range; one-way ANOVA with Tukey post-hoc test. j, Differential expression of activation markers (Cd40, Cd83, Cd86 and H2-K1) in the indicated DC subsets. k, Spatial STARmap projections of splenic B cell subtypes. l, Relative abundance of mature (Cd19+, Ms4a1+, Cd22+ and Cd40+) and age-associated (Cd19+, Ms4a1+, Itgax+ and Tbx21+) B cells per FOV (adult = 404, Luc = 386 and DFI = 408 FOVs from one animal per condition); data are represented as violin plots with median + interquartile range; one-way ANOVA with Tukey post-hoc test. Aged mice (72 weeks old) received DFI mRNA–LNPs or Luc mRNA–LNPs twice weekly for 28 days. DFI (but not individual factors) increased both frequency and absolute number of circulating naive (CD44−CD62L+) CD4+ and CD8+ T cells (Fig. Repeated administration of recombinant IL-7 has been shown to promote the proliferation of mature T cell subsets35,36,37. In line with this, Il7 mRNA alone expanded effector-memory (CD44+CD62L−) cells; however, full DFI did not increase memory subsets (Fig. Consequently, the naive-to-memory ratio rose in the blood and spleen of DFI-treated animals, indicating complementary or moderating activity of the three signals when delivered in combination (Fig. To delineate the origin of these DFI-induced shifts in the T cell compartment of aged mice, we next analysed the spleen, peripheral blood, bone marrow and thymus of adult and aged mice. Deep bulk V(D)J sequencing showed no increase in clonality following DFI conditioning (Fig. 2e), arguing against homeostatic proliferation and favouring enhanced thymic output as a driver of the observed increase in naive T cell counts. Although aged thymuses displayed reduced mass and cellularity relative to young controls, 28 days of DFI partially restored both (Extended Data Fig. Although overall distributions of double-negative (DN), double-positive or single-positive thymocytes were preserved, DFI selectively expanded early DN1–DN3 thymocytes — the stages of early T lineage commitment — and induced Rag2 in thymocytes of Rag2–eGFP mice within 12 h, followed by increased mature single-positive CD4+ and CD8+ thymocytes towards youthful levels after 28 days of treatment (Fig. Consistent with these data, TCR excision circles in peripheral blood were elevated (Fig. 2g), and Nur77 expression was elevated in circulating T cells (Extended Data Fig. 7d–f), indicative of an increase in recent thymic emigrants. Because thymic output depends on progenitor supply, we examined haematopoiesis. With ageing, HSCs expand in number but increasingly adopt a myeloid-biased differentiation program12,38,39; we recapitulated HSC expansion and CD150high myeloid-biased HSC enrichment with age, which DFI did not reverse (Extended Data Fig. Multipotent progenitors were largely unchanged aside from an increase in lymphoid-primed multipotent progenitors after DFI (Extended Data Fig. By contrast, DFI robustly expanded common lymphoid progenitors (CLPs), which are markedly depleted with age, in bone marrow and increased the number of circulating CCR9+ (but not CCR7+) CLPs, which are associated with preferential thymus homing38,40 (Extended Data Figs. CCR9 surface levels on CLPs rose from the bone marrow to the thymus in DFI-treated mice, mirroring the dynamics found in young animals and absent in aged controls (Extended Data Fig. 9c–e), consistent with a peripheral priming of circulating CLPs that potentially augments thymic entry. Together, these data support a model in which liver-expressed DLL1, together with systemic FLT3-L and IL-7, boosts CLP production, survival and recruitment to the thymus, thereby increasing thymopoiesis. Thus, DFI enhances naive T cell output by amplifying committed lymphoid progenitors and facilitating intrathymic maturation, without reprogramming HSC composition. Beyond T cells, DFI mitigated ageing-associated phenotypes in antigen-presenting and B cell compartments. DFI preferentially expanded splenic conventional type 1 dendritic cells (cDC1s), which are critical for antigen cross-presentation and decline in number and co-stimulatory function with age, probably by sustained FLT3-L-dependent differentiation from FLT3+ progenitors20,41,42 (Fig. In situ-seq of spleens by STARmap revealed cDC1 enrichment within periarteriolar lymphoid sheaths and a concomitant upregulation of H2-K1 and co-stimulatory molecules (CD40, CD83 and CD86), consistent with improved priming capacity43 (Fig. In addition, we observed a reduced abundance of splenic age-associated B cells (CD19+, Ms4a1+, Itgax+ and Tbx21+) and an expansion of mature follicular-like B cells (CD19+, Ms4a1+, Cd22+ and CD40+) localized to B cell follicles following DFI treatment44,45 (Fig. On the basis of the above findings, we next tested whether DFI could restore adaptive immune function in aged animals. As a test case, we looked at vaccine response, a T cell-mediated process known to be blunted by ageing. In an adjuvanted ovalbumin (OVA) prime–boost model in adult (6 weeks) and aged (72 weeks) mice, aged cohorts showed fewer OVA-specific CD8+ T cells in the spleen and blood upon vaccination, impaired antigen-driven proliferation and reduced IL-2 and IFNγ production on recall, approaching levels of sham-vaccinated mice (Supplementary Fig. These deficiencies align with the impaired T cell-mediated vaccine responses commonly observed in older animals and humans1,46. Pre-conditioning aged mice with individual factors or the full DFI combination before vaccination showed that Il7 mRNA alone increased total T cell counts, whereas only DFI increased both total T cells and the frequency of OVA-specific CD8+ cells, yielding approximately twofold more antigen-specific CD8+ T cells in the spleen and a similar increase in blood35,36,37 (Fig. Aged mice received DFI or control (Luc) mRNA–LNPs twice weekly for 28 days, then were immunized with adjuvanted OVA. ; one-way ANOVA with Tukey post-hoc test. c, Frequency of SIINFEKL–H-2K(b) tetramer+CD8+ T cells in the spleen (n = 4 per group). ; one-way ANOVA with Tukey post-hoc test. ; one-way ANOVA with Tukey post-hoc test. e, Frequency of naive T cells (CD44−CD62L+) in the spleen (n = 4 per group). ; one-way ANOVA with Tukey post-hoc test. f, Frequency of exhausted-phenotype T cells (CD62L−PD1high) in the spleen (n = 4 per group). ; one-way ANOVA with Tukey's post-hoc test. g, Proportions of naive (TN) and exhausted (TEX) T cells and the TN:TEX ratio after repetitive OVA or sham (vehicle) vaccination (n = 4 per treatment). h, Functional antigen recall: fold change in intracellular cytokines following 6-h SIINFEKL restimulation versus DMSO vehicle. IL-2 (left) and IFNγ (right) mean fluorescence intensity (n = 4 per group). In a longitudinal cohort covering increasing ages, SIINFEKL–H-2K(b) tetramer frequencies were fit with a quadratic (second-order) polynomial regression with 95% confidence bands (d.f. This standard curve was used to estimate an ‘immunological vaccination response age' for numerically 52-week-old mice preconditioned with either DFI or Luc mRNA–LNPs before immunization (N = 40 total; n = 4 per age and treatment). In addition to the reduced generation of new immune cells, ageing is characterized by the progressive shift to dysfunctional states by existing cells2,17. Aged mice were shown to accumulate PD1highCD62L− T cells, which we confirmed using scRNA-seq and flow cytometry in our longitudinal cohorts12,47 (Fig. This phenotype expanded further with repeated vaccination, unlike in adults (Fig. DFI conditioning preserved a higher naive fraction post-vaccination and reduced PD1highCD62L− cells (with a reduced effect from IL-7 alone), together producing a more balanced T cell composition (Fig. Functionally, CD8+ T cells from DFI-treated mice generated higher levels of IL-2 and IFNγ upon antigen-specific restimulation, whereas OVA-specific CD4+ frequencies and cytokines were largely unchanged (Fig. 4a–e), suggesting that DFI preferentially supports CD8+ T cell responses. In a longitudinal vaccination study across the lifespan of C57BL6/J mice, we found that DFI treatment increased vaccine-induced T cell counts in aged mice to levels comparable with those seen in much younger mice, effectively rejuvenating their response by approximately 24 weeks (Fig. Together, the full DFI combination, but not its single components, counteracts age-related defects in CD8+ vaccine responses, consistent with the broader principle that aged immunity retains the capacity to mount robust immune responses when sufficiently stimulated48. We next tested whether DFI improved responsiveness to tumour immunotherapy, which also diminishes with ageing. In B16-OVA melanoma and MC38-OVA colon carcinoma, we found that aged mice showed faster tumour progression and poorer survival than adults; immune checkpoint inhibition (ICI) with PDL1 blockade that controlled tumours in adults conferred little benefit in aged cohorts (Supplementary Fig. Pre-conditioning aged hosts with DFI for 28 days followed by a 72-h washout to control for direct antitumour effects improved endogenous control of MC38-OVA, increasing spontaneous rejection rates and prolonging survival (Supplementary Fig. In the more aggressive B16-OVA model, DFI drove complete rejection in 40% of aged mice, whereas all controls succumbed within 3 weeks despite anti-PDL1 treatment (Fig. Co-administration at treatment onset similarly delayed progression and improved survival over PDL1 blockade alone in aged animals with established tumours (Fig. Aged mice received DFI or control (Luc) mRNA–LNPs for 28 days, followed by subcutaneous B16-OVA tumour challenge and two doses of anti-PDL1 checkpoint blockade. b, Tumour growth over time in the B16-OVA model (n = 8 per group). Three out of eight animals did not establish a measurable tumour (black arrow). c, Tumour size on day 12 (mean ± s.e.m. d, Kaplan–Meier survival analysis for the B16-OVA cohort (n = 8 per group; log-rank test). e, Design of adjuvant DFI or Luc LNP therapy combined with anti-PDL1 checkpoint blockade in established B16-OVA tumours. f, Tumour growth trajectories during combination therapy (n = 10 per group). g, Tumour size on day 14 (mean ± s.e.m. h, Kaplan–Meier survival curves for combination treatment (n = 10 per group; log-rank test). i, Schematic of a parallel adjuvant DFI + anti-PDL1 experiment terminated at day 12 for immune profiling with matched tumour sizes. j, Tumour size at day 12 (n = 5 per group). k, Absolute counts of live TILs in explanted tumours (n = 5 per group). l, CD8+ T cell frequency (left) and CD4:CD8 ratio (right) among TILs (n = 5 per group). m, Relative (left) and absolute (right) numbers of tumour-specific SIINFEKL–H-2K(b)+ TILs (n = 5 per group). ; statistical significance was determined by a two-tailed unpaired Student's t-test. TFH, follicular helper T; TH, helper T; Treg, regulatory T. o, CITE-seq feature plot showing CD8+SIINFEKL–H-2K(b)+ (Tet+) TILs. p, Expression of canonical markers of tumour recognition and exhaustion in Luc-treated versus DFI-treated TILs. q, Relative abundance of transcriptional clusters among CD8+Tet+ clonotypes. All comparisons were by two-tailed unpaired t-tests (o–q). r, Shannon diversity of bystander (Tet−) and tumour-specific (Tet+) TCR clonotypes across treatments. In young mice, ICI confers antitumour immunity primarily through expansion and activation of tumour-specific CD8+ T cells. To probe the underlying mechanism of the observed antitumour effects, we analysed tumour-infiltrating lymphocytes (TILs) on day 12 post-implantation, before significant tumour size divergence (Fig. Total TIL numbers were unchanged (Fig. 4k), but DFI increased the frequency of intratumoural CD8+ T cells and consequently lowered the CD4:CD8 ratio (Fig. 4l); SIINFEKL-loaded tetramers confirmed intratumoural enrichment of antigen-specific CD8+ cells (Fig. In parallel, naive CD8+ cells expanded systemically (Supplementary Fig. 5j,k), suggesting that rejuvenated peripheral pools contribute to the TIL compartment. scRNA–V(D)J profiling and CITE-seq using oligo-conjugated SIINFEKL–H-2Kb tetramers resolved eight canonical CD8+–CD4+ TIL states and delineated tumour antigen-specific T cells (Fig. With DFI, both tumour-specific and bystander clonotypes showed lower expression of exhaustion-associated genes (Havcr2, Gzmk, Lag3, Pdcd1 and Tigit) and moderately higher Entpd1 (CD39) expression, consistent with increased tumour-antigen engagement or tissue residency rather than terminal dysfunction51 (Fig. The naive-like fraction among tetramer+CD8+ TILs rose approximately 1.8-fold (Fig. 5o), and clonal diversity (as assayed by Shannon index) increased in both tetramer+ and bystander repertoires (Fig. 5p), indicating broader recruitment and intratumoural TCR repertoire breadth. Together, DFI conditioning alone boosted endogenous antitumour control and synergized with PDL1 blockade to enhance therapeutic efficacy in aged hosts. Alongside recent work with mRNA-encoded cytokines6, these data highlight liver-encoded delivery of immune modulators as a strategy to mitigate age-related immune dysfunction and overcome components of immunotherapy resistance. To define durability and safety of DFI, aged mice received DFI or control mRNA–LNPs for 28 days and were then observed off-treatment for 28 days (Supplementary Fig. Thymic output rose during dosing, as evidenced by elevated TCR excision circles, and returned to baseline thereafter (Supplementary Fig. 6b,c), consistent with the short half-life of mRNA and the rapid decline of thymocyte Rag2 expression after a single dose (72 h or less; Fig. STARmap of whole spleens showed that, despite waning thymic export, splenic T cell and dendritic cell numbers remained modestly elevated at day 28 post-cessation (Supplementary Fig. 6d–g), indicating partial persistence of peripheral remodelling. However, when vaccination occurred 4 weeks after stopping DFI, previously observed benefits such as higher vaccine-specific T cells, fewer exhaustion-like cells and favourable naive–effector-memory balance were no longer evident (Supplementary Fig. Thus, immune enhancement by our approach is largely confined to the dosing window and reverses upon withdrawal. We next assessed whether DFI would exacerbate autoimmune disease. In NOD mice, which are prone to type 1 diabetes due to frequent generation of autoreactive TCRs, 4 weeks of DFI or control mRNA was followed by 6-month monitoring of glycaemia, glycosuria and disease onset (Fig. In addition, we quantified frequencies of CD8+ T cells specific to the NRP-V7 mimotope, a known target in NOD mice52 (Extended Data Fig. DFI did not alter blood glucose levels, diabetes onset or autoreactive T cell frequencies versus control littermates (Fig. 5b,c), whereas NOD.Cg-Tg(TcraTcrbNY8.3)1Pesa/DvsJ (NY8.3) NOD mice with a genetically encoded NRP-V7-restricted TCR uniformly developed diabetes within 8 weeks, providing a positive-control threshold. We did not observe any treatment-related adverse effects over the course of this 6-month study. a, Experimental design to evaluate autoimmune risk in NOD mice receiving DFI or control (Luc) mRNA–LNPs. NY8.3 mice, carrying an autoreactive TCR specific for NRP-V7 (KYNKANAFL), served as positive controls. Type 1 diabetes (T1D) onset was defined by blood glucose > 200 mg dl−1 in two consecutive measurements or by glucosuria. b, Longitudinal monitoring of blood glucose (top) and frequency of NRP-V7-specific TCRs in peripheral blood (bottom) of NOD mice treated as in panel a. NY8.3 (n = 5), NOD + Luc (n = 9) and NOD + DFI (n = 9). Statistical significance was tested using log-rank (Mantel–Cox) tests. d, Experimental set-up to test antigen-specific T cell tolerance in Act-mOVA mice. OT-I (CD8+, OVA257–264) and OT-II (CD4+, OVA329–337) transgenic mice served as positive controls for OVA-targeted responses. e, Frequencies of SIINFEKL–H-2K(b) tetramer+CD8+ T cells in peripheral blood of Act-mOVA mice treated with Luc or DFI mRNA for 28 days. ; one-way ANOVA with Tukey's post-hoc test; P values are shown for Act-mOVA comparisons. f, Frequencies of AAHAEINEA–I-A(b) tetramer+CD4+ T cells in the peripheral blood of Act-mOVA mice treated as in panel e. n = 5 per group. ; one-way ANOVA with Tukey's post-hoc test; two-tailed P values are indicated for Act-mOVA comparisons. g, Serum anti-OVA IgG ELISA (OD450–570) in Act-mOVA mice after Luc or DFI mRNA treatment followed by OVA–complete Freund's adjuvant prime and OVA–incomplete Freund's adjuvant boost (O) or sham vaccination (S). Wild-type (n = 5) vaccinated mice served as positive controls. To test safety in a model with intact central tolerance, we used Act-mOVA mice, which constitutively express germline-encoded OVA53 (Fig. This allowed us to use the same potent model antigen as in our previous vaccination experiments; however, in this case, OVA was subject to central tolerance. Despite enhancing responses to exogenous OVA in aged wild-type mice, DFI neither induced OVA-specific CD4+ or CD8+ T cells after 4 weeks (Fig. 5e,f) nor broke humoral tolerance after adjuvanted OVA challenge in Act-mOVA mice, in contrast to wild-type controls (Fig. Finally, in experimental autoimmune encephalitis, a polyclonal, antigen-driven central nervous system autoimmunity model, DFI was administered every 3 days following experimental autoimmune encephalitis induction by MOG35–55 peptide immunization and thereafter until symptom onset. Aged mice showed delayed disease onset, consistent with impaired T cell priming and previous reports, but worse subsequent clinical deterioration54,55 (Extended Data Fig. In line with our earlier vaccination experiments, DFI increased peripheral, but not central nervous system-infiltrating MHC class II I-Ab–MOG-specific CD4+ T cells after induction (Extended Data Fig. 10e,f) and did not worsen clinical scores or spinal cord inflammation and demyelination (Extended Data Fig. In adults, DFI neither increased MOG-reactive T cells in the periphery nor aggravated disease (Extended Data Fig. 10c,e–h), indicating correction of age-related deficits rather than indiscriminate immune amplification. Together with the vaccination data, these results show that DFI transiently augments antigen-specific immunity in aged hosts while preserving self-tolerance across several models. The effects are reversible and temporally restricted, supporting liver-encoded systemic immune modulation as a novel strategy to improve immunity in older individuals. Here we have showed that immune function can be improved by repurposing the liver as a platform for ectopic production of immune factors. Our results show that delivery to the liver of mRNA encoding DLL1, IL-7 and FLT3-L in combination successfully enhanced immune function in aged mice. This approach provides a scalable alternative to more invasive methods of immune rejuvenation, such as thymic transplantation56 or HSC manipulation12, which are clinically challenging. In addition, our disease-agnostic strategy holds potential for synergizing with other immunostimulatory treatments, as we observed in the context of ICI treatment. Although the biological roles of IL-7 and FLT3-L are well established, with both factors known to support thymopoiesis, dendritic cell expansion and peripheral T cell homeostasis20,21,35,36,37,41,42, combining them with DLL1 and delivering them to hepatocytes enables the creation of a transient, rejuvenated immune milieu in aged mice, distinct from conventional approaches based on recombinant protein infusion, which lack spatial control and tend to require chronic dosing or come with high toxicity. By using mRNA, we mitigated the systemic inflammation and autoimmune sequelae typically associated with recombinant cytokine-based therapies. However, the transient nature of mRNA delivery necessitates repeated administrations to sustain therapeutic effects. The long-term consequences of continuous exposure to these factors, especially in aged individuals should be analysed through extensive long-term safety studies. Future studies may therefore uncover additional strategies and factors to holistically target the hallmarks of immune ageing1. In summary, this study demonstrates that transiently repurposing the liver to express and secrete various therapeutic proteins could be a generalizable approach to engineering physiological processes. By mimicking specific signalling niches within the liver or secreting proteins for systemic circulation to restore homeostatic signals throughout the body, this strategy has the potential to improve health outcomes and address a wide range of human diseases and conditions. All experiments were performed in compliance with all relevant ethical regulations as approved by the Institutional Biosafety Committee (IBC) of the Broad Institute (protocol #IBC-2017-00146). All animal experiments were approved by the Institutional Animal Care and Use Committee of the Broad Institute (protocol ID 0017-09-14-2). Animal maintenance complied with all relevant ethical regulations and were consistent with local, state and federal regulations as applicable, including the National Institutes of Health (NIH) Guide for the Care and Use of Laboratory Animals. G-blocks encoding the human α-globin 5′ untranslated region (UTR) and 3′ FI element were synthesized de novo by IDT. In vitro transcription vectors were cloned by inserting UTRs into a pET45 vector via Gibson assembly using Gibson Assembly Master Mix (E2611L, NEB) and transformation into chemically competent Stbl3 cells. A hard-coded A30LA70 polyA tail was added by PCR and ligation using the KLD enzyme mix (New England Biolabs). Plasmid sequences were verified via next-generation sequencing, long-read sequencing (Primordium Labs) and PCR to verify the length of polyA tails. Plasmids were linearized, and a T7-driven in vitro transcription reaction (Life Technologies) was performed to generate mRNA with 101 nucleotide long polyA tails. Capping of mRNA was performed in concert with transcription through addition of a trinucleotide cap1 analogue CleanCap, and m1Ψ-5′-triphosphate (TriLink) was incorporated into the reaction instead of uridine-5′-triphosphate (UTP; Supplementary Fig. LiCl-based purification of mRNA was performed, mRNAs were then checked on an agarose gel and by a TapeStation RNA ScreenTape Analysis (Agilent; Supplementary Fig. We engineered a formulation of mRNA-encoded DFI within biodegradable lipopolyplexes, along with control formulations (Fig. To do this, we formulated LNPs by combining SM-102 as ionizable lipid, 1,2-distearoyl-sn-glycero-3-phosphocholine (DSPC), cholesterol and 1,2-dimyristoyl-rac-glycero-3-methoxypolyethylene glycol-2000 (DMG-PEG-2000) in a molar ratio of 50:10:38.5:1.5. These were formulated into LNPs along with mRNA using microfluidic mixing using a NanoAssemblr Ignite nanoparticle formulation system (Cytiva). In brief, an ethanol phase containing the above formulated lipidoid, phospholipid, cholesterol and DMG-PEG master mix was mixed with an aqueous phase (10 mM citrate buffer, pH 3) containing mRNA at a flow rate ratio of 1:3 and at a lipidoid:RNA weight ratio of 10:1. Upon formulation, mRNA–LNPs were diluted in sterile NaCl and the buffer was exchanged by concentrating with a 30-kDa spin filter (UFC9030, MilliporeSigma) to replace residual ethanol. NaCl-diluted mRNA–LNPs were stored at 4 °C until use. For all subsequent experiments, we utilized these SM-102 mRNA–LNPs encapsulating m1Ψ-5′-triphosphate-modified and m7GpppNm-capped DFI, Luc or GFP mRNA, respectively. The hydrodynamic size, polydispersity index (PDI) and zeta potential (ZP) of LNPs were measured using a DynaPro NanoStar II (Wyatt). The mRNA encapsulation efficiency of LNPs were determined using a modified Quant-iT RiboGreen RNA assay (Invitrogen) and found to be more than 85% on average (Supplementary Fig. LNP endotoxin levels were consistently found to be less than 1 endotoxin unit per ml. The average hydrodynamic diameter was approximately 75 nm with a polydispersity index of 0.02–0.03 (Supplementary Fig. Unless otherwise stated, mammalian cells were maintained in T75 flasks (156499, Thermo Fisher) at 37 °C with 5% CO2 in either DMEM-GlutaMAX (10569044, Thermo Fisher) or RPMI-GlutaMAX (61870127, Thermo Fisher). All media were supplemented with 10% FBS (97068-085, VWR) and 1× penicillin–streptomycin (15140122, Thermo Fisher). For growth of primary T cells, media were also supplemented with 50 µM 2-mercaptoethanol (21985023, Thermo Fisher). Images of transfected cells were acquired on a Leica DMI8 Confocal Microscope running Leica Application Suite X (1.4.3), equipped with a Lecia Stellaris 5 camera using an HC PL APO CS2 ×20/0.75 DRY objective and a pinhole setting of 1 Airy Unit. Images were processed using Fiji (https://imagej.net/software/fiji/downloads). Cells were prepared and stained according to the staining protocol of each experiment outlined below, pelleted at 500g for 5 min and resuspended in 200 µl of flow cytometry buffer (PBS supplemented with 2% EDTA (15575020, Life Technologies) and 5% FBS (97068-085, VWR)). Samples were run on Beckman Coulter Cytoflex LX flow cytometers and analysis was performed using the FlowJo v10 software. Representative schemes for gating and threshold setting of each experiment are shown in the Extended Data and Supplementary figures. All peptide–MHC tetramers were obtained from the NIH Tetramer Core Facility. Before viability dye and surface antibody staining, cells were incubated with appropriately titrated tetramer combinations. Unless stated otherwise, tetramer staining was performed in PBS for 20 min on ice, followed by washing and addition of antibody cocktails and/or fixation. For central tolerance experiments in Act-mOVA mice, the same SIINFEKL–H-2Kb tetramers (1:100 dilution) were used in combination with AAHAEINEA–I-Ab–PE and AAHAEINEA–I-Ab–APC tetramers (1:20 dilution). Non-vaccinated wild-type T cells (negative) and OT-I and OT-II T cells (positive) were included as staining controls for MHC class I and class II tetramers, respectively. For autoimmunity experiments in NOD mice, KYNKANAFL–H-2Kb–PE and KYNKANAFL–H-2Kb–APC tetramers were used at 1:50 dilution, with NY8.3 T cells serving as positive controls for staining. For experimental autoimmune encephalitis (EAE) experiments in C57BL/6J mice, GWYRSPFSRVVH–I-Ab–PE and GWYRSPFSRVVH–I-Ab–APC tetramers were used at 1:25 dilution. Control tetramers consisted of I-Ab-restricted human CLIP87–101 (PVSKMRMATPLLMQA) conjugated to PE and APC, also at 1:25 dilution. All animal experiments were approved by the Institutional Animal Care and Use Committee of the Broad Institute (protocol ID 0017-09-14-2). Animal maintenance complied with all relevant ethical regulations and were consistent with local, state and federal regulations as applicable, including the NIH Guide for the Care and Use of Laboratory Animals. Mice were acclimated at the animal facility for at least 7 days before performing any experiments. The sample size for vaccination experiments was decided based on a previous publication with similar experiments48. The sample size for tumour experiments was decided based on previous publications with similar experiments49,50. The sample size for NOD experiments was decided based on a previous publication with similar experiments52. The sample size for EAE experiments was decided based on a previous publication with similar experiments61. For all other exploratory experiments, no sample size calculations were performed. For all experiments, allocation of mice into experimental groups was randomized after stratifying for age and sex. Separate investigators performed treatment and data collection. Data-analysing investigators were not blinded to the treatment groups, as they involved internal controls, with the exception of pathologists for toxicity studies, who were blinded for the analyses. mRNA–LNPs (5 µg of each mRNA, equal molar, normalized to the control RNA) in a total volume of 100 μl sterile NaCl were injected through slow retro-orbital injection into each mouse. For Luc imaging, sterile NaCl injection was used as negative control. For all other experiments investigating DFI mRNA–LNPs, Luc mRNA–LNPs were used as negative control. In vivo activity of Luc following delivery of Luc mRNA–LNPs was measured using a Competent IVIS-Perkin Elmer IVIS Spectrum CT System (Perkin Elmer). Mice were injected through retro-orbital injection with Luc mRNA–LNPs at a dose of 5 μg mRNA per mouse, and bioluminescence imaging was performed on an IVIS imaging system (PerkinElmer). At 6 h post-injection, mice were anaesthetized with isoflurane and intraperitoneally injected with D-luciferin potassium salt (150 mg kg−1 (body weight)). Mice or dissected organs were imaged 10 min post-injection using auto-exposure settings. The luminescent activity was quantified using Aura 4.0 imaging software. To assess DLL1 protein induction following DFI, mice were intravenously injected with either 5 µg DFI mRNA–LNPs in 100 µl NaCl. The mice were then anaesthetized with isoflurane and rapidly decapitated. Liver tissue samples were harvested from the mediolateral lobe and placed in Tissue-Tek O.C.T. was frozen in liquid nitrogen and stored at −80 °C. Liver DLL1 levels were measured using immunofluorescence. In brief, liver tissue was cryo-sectioned at 15 µm thickness, fixed with 4% paraformaldehyde for 15 min and permeabilized with 0.5% Triton X-100 and 100 mmol l−1 glycine diluted in PBS for 10 min. Samples were blocked with blocking buffer (10% normal donkey serum (017-000-121, Jackson ImmunoResearch) in PBS–0.1% Tween-20) and stained with anti-DLL1 antibody (ab10554, Abcam; 1:800 dilution in blocking buffer) at 4 °C overnight. Samples were then washed with PBS–0.1% Triton X-100 for 3 × 10 min, stained with AF546-labelled secondary antibody (A10040, Invitrogen; 1:500 dilution in blocking buffer) at room temperature for 1 h and washed with PBS–0.1% Triton X-100 for 3 × 10 min. Samples were then stained with DAPI and AF488-phalloidin (A12379, Invitrogen) according to the manufacturer's protocol to visualize the plasma membrane of hepatocytes. Liver DLL1 levels were quantified by taking the immunofluorescence signal intensity normalized to DAPI intensity at 6, 12, 24 and 48 h after injection (n = 3 mice per condition and time point). Uninjected littermates served as baseline controls (0 h). Tissue concentrations were expressed as fold change relative to baseline. To assess cytokine induction following DFI or recombinant protein administration, mice were intravenously injected with either 5 µg DFI mRNA–LNPs in 100 µl NaCl or 10 µg recombinant mouse IL-7 (217-17, PeproTech) or 10 µg recombinant mouse FLT3-L (250-31L, PeproTech) in NaCl containing 0.1% BSA. Blood samples were collected by terminal cardiac puncture using EDTA-coated syringes and transferred to BD Microtainer tubes (365974, BD). Plasma was separated by centrifugation at 2,000g for 10 min at room temperature and snap frozen for later analysis. Liver tissue samples were harvested from the mediolateral lobe, weighed for normalization and snap frozen. Frozen tissue fragments were homogenized on ice in 200 µl PBS containing protease inhibitor cocktail (P8340-1ML, Sigma) using a pre-chilled glass douncer with 15 strokes. Homogenates were clarified by centrifugation (10,000 rpm for 10 min at 4 °C), and supernatants were stored at −80 °C. IL-7 and FLT3-L levels in both the serum and liver were quantified using ELISA kits (mouse IL-7, EMIL7, Invitrogen; mouse FLT3-L, EMFLT3L, Invitrogen) according to the manufacturer's protocol. Absorbance was measured at 450 nm with 570-nm background subtraction, and cytokine concentrations were calculated using logistic regression fitted to the standard curve. Samples were processed in technical triplicates, and the mean value was used for biological replicates. Uninjected littermates served as baseline controls (0 h). Tissue concentrations were normalized to organ weight and expressed as fold change relative to baseline. To assess age-dependent changes in thymic cytokine levels, thymus lobes were isolated from untreated C57BL/6J mice across a range of ages, including 3, 6, 12, 18 and 24 months (n = 3 per age group). Thymic tissue was weighed, snap frozen and processed identically to liver samples. FLT3-L concentrations were determined by ELISA, normalized to thymus weight and used to calculate total cytokine abundance across the lifespan. Adult (6 weeks) and aged (72 weeks) mice were immunized with 1 mg ml−1 full-length OVA protein emulsified in complete Freund's adjuvant (EK-0301, Hooke Laboratories), followed by one booster dose of 1 mg ml−1 protein emulsified in incomplete Freund's adjuvant (IFA; EK-0311, Hooke Laboratories). Mice were injected with antigen emulsified in complete Freund's adjuvant (CFA) subcutaneously at two sites on the chest, injecting 0.1 ml at each site (total of 0.2 per mouse). A booster injection of antigen emulsified in IFA was administered 14 days after immunization with antigen–CFA emulsion. The booster was given as a single subcutaneous injection with 0.1 ml of IFA emulsion, at one site on the sternum. Serum, peripheral blood or spleen samples were obtained 21 days after the initial immunization, unless otherwise specified. OVA-expressing melanoma B16 (B16-OVA) and MC38 (MC38-OVA) cell lines were provided by M. Kilian. Mice were inoculated subcutaneously with 1 × 105 B16-OVA or 5 × 105 MC38-OVA in Matrigel Matrix (Corning). Tumour growth was monitored daily by measuring with digital calipers using the two largest perpendicular axes until the area (0.5 × larger diameter × smaller diameter2). The size of the tumours was assessed in a blinded, coded manner every day following treatment start and recorded as tumour volume. Mice were euthanized when tumours reached 2,000 mm3 or upon ulceration. Of anti-PDL1 antibody (10 F.9G2, BioXCell), and/or control hamster and/or control rat IgG antibody (BioXCell), 100 μg was injected intraperitoneally every 3 days, as previously described, unless otherwise specified49. For all vaccination experiments, at the indicated time points post-vaccination, approximately 100 μl of blood was collected with EDTA-coated capillary tubes from each mouse and then transferred to an EDTA-coated tube. The collected blood samples were centrifuged at 2,000g for 10 min, followed by transferring the resulting plasma into another tube, and antibody staining was performed using the eBioscience one-step Fix/Lyse Solution (10X; 00-5333-54, Thermo Fisher). In brief, 50 or 100 µl of blood samples was incubated with 50 or 100 µl twofold concentrated antibody cocktails (1:100 final dilution) as well as TruStain FCX (anti-mouse CD16/32, clone 93; BioLegend; 1:50 final dilution) for 20 min at 4 °C in the dark, followed by the addition of 4 ml of one-step Fix/Lyse Solution and 15-min incubation at room temperature in the dark. Samples were then washed twice with 10 ml of PBS followed by centrifugation at 500g for 5 min and finally resuspended in 200 µl flow cytometry buffer. In the case of tetramer staining, samples were pre-incubated with the respective twofold concentrated tetramers (1:20 to 1:100 final dilution) for 20 min on ice in the dark, followed by incubation with twofold concentrated antibody cocktails (1:100 final dilution) as well as TruStain FCX (anti-mouse CD16/32, clone 93; BioLegend; 1:50 final dilution) for another 20 min at 4 °C in the dark. On day 21 following the initial subcutaneous immunization with OVA–CFA, mouse spleen single-cell suspensions were prepared in RPMI 1640 medium by mashing tissue against the surface of a 70-μm cell strainer (64752-00, BD Falcon). Then, the single-cell suspension was centrifuged at 500g for 5 min and the supernatant was removed. Red blood cells were lysed by adding 1 ml of ACK lysis buffer (Thermo Fisher) at 4 °C for 1.5 min, followed by centrifugation and removal of the supernatant. The cells were washed once with RPMI 1640 medium and then resuspended with RPMI 1640 medium (10% FBS and 1% penicillin-streptomycin antibiotic). Of splenocytes from each mouse, 4 × 106 were cultured in RPMI medium and stimulated with SIINFEKL peptide (synthesized at 99% purity by GenScript) at a final concentration of 1 μg ml−1 for each peptide for CD8+ T cell recall or 1× Cell Stimulation Cocktail containing phorbol 12-myristate 13-acetate and ionomycin (00-4970-03, eBioscience) for CD4+ T cell recall. The GolgiStop transport inhibitor cocktail (554724, BD) was added according to the manufacturer's instruction 2 h later. Then, 6 h later, the cells were collected and washed with flow cytometry buffer (PBS with 2% FBS) before Fc block with TruStain FCX (anti-mouse CD16/32, clone 93; BioLegend; 1:50 final dilution) and surface antibody staining for 20 min at 4 °C. Cells were washed with a flow cytometry buffer and then fixed and permeabilized using a BD Cytoperm fixation/permeabilization solution kit (554714, BD) according to the manufacturer's instructions. Cells were washed in perm/wash solution, followed by intracellular staining (for 45 min at 4 °C) using a cocktail of the respective cytokine or transcription factor antibodies. Samples were washed, resuspended in 200 µl flow cytometry buffer and acquired on a Beckman CytoFLEX LX Flow Cytometer. Analysis was performed using FlowJo v10 software. Snap-frozen spleens were submitted to Adaptive Biotechnologies for deep TCRβ repertoire profiling. Genomic DNA was extracted using the DNeasy Blood & Tissue Kit (69504, Qiagen), followed by deep sequencing of rearranged Tcrb gene segments using the immunoSEQ mmTCRB Deep Sequencing platform (Adaptive Biotechnologies). Library preparation, high-throughput sequencing and initial data processing, including demultiplexing, quality filtering and V(D)J gene annotation, were performed by Adaptive Biotechnologies using their proprietary pipeline. Output files included productive clonotype frequencies and diversity metrics used for downstream analysis. To quantify thymic output, signal joint T cell receptor excision circles (sjTRECs) were measured from peripheral blood. A total of 100 µl of cardiac blood was collected from mice following terminal anaesthesia and centrifuged to pellet cellular material. Genomic DNA was extracted using the DNeasy Blood & Tissue Kit (69504, Qiagen) according to the manufacturer's instructions. Analysis was performed using an Bio-Rad CFX Opus Real-Time PCR system, following previously published protocols for sjTREC detection in C57BL/6 mice62,63. The sjTREC-specific primers used were: forward, 5′-CCAAGCTGACGGCAGGTTT-3′; reverse, 5′-AGCATGGCAAGCAGCACC-3′. To control for input DNA variability, amplification of the constant region of the Tcra gene was used as an endogenous reference. The Tcra primer sequences were: forward, 5′-TGACTCCCAAATCAATGTG-3′; reverse, 5′-GCAGGTGAAGCTTGTCTG-3′. Cycle threshold (Ct) values were determined in technical duplicates for both targets. Relative sjTREC content was calculated as ΔCt = Ct_TREC − Ct_TCRA. For clarity, values are reported as −ΔCt, such that higher values reflect greater relative sjTREC abundance. NOD/ShiLtJ (NOD) mice were treated with DFI mRNA or Luc mRNA encapsulated in SM-102 LNPs twice per week for 4 weeks followed by clinical and molecular monitoring for the development of T1D. NY8.3 mice transgenic for the autoreactive TCR recognizing NRP-V7 were included as a positive control group. Experimental end point (onset of T1D) was reached in animals with blood glucose levels of more than 200 mg dl−1 in two independent measurements or in animals developing glucosuria52,64. All animals were of a C57BL/6J background. Mice were immunized subcutaneously with 100 μg of MOG35–55 peptide (110582, Genemed Synthesis) emulsified in CFA, which was freshly prepared by combining 20 ml of IFA (BD263910, BD Biosciences) with 100 mg of Mycobacterium tuberculosis H37Ra (231141, BD Biosciences) at a 1:1 ratio (v/v; 5 mg ml−1 final concentration). Pertussis toxin (180, List Biological Laboratories) was administered intraperitoneally at 320 ng per mouse (1.6 ng μl−1 in 200 μl PBS) on the day of immunization and again 48 h later. To assess the effects of DFI on disease progression, mice were randomly assigned to receive DFI or Luc control mRNA–LNPs after immunization until symptom onset. mRNA–LNPs were administered every 3 days starting on the day of immunization and continued until the onset of clinical symptoms in the first animals (day +15). All clinical scoring was performed in a blinded manner. To isolate central nervous system (CNS)-infiltrating lymphocytes, mice were terminally anaesthetized and perfused transcardially with ice-cold PBS. Spinal cords were flushed by hydrostatic pressure. CNS tissues were minced and enzymatically digested in RPMI supplemented with Liberase (5401119001, Roche) for 30 min at 37 °C with gentle agitation. Following digestion, tissues were dissociated by 20 trituration cycles using a 10-ml serological pipette and filtered through a 70-μm cell strainer. Mononuclear cells were enriched by 30% isotonic Percoll density gradient centrifugation at 800g for 30 min at room temperature without brake. The cell pellet was washed in PBS and resuspended in 200 μl PBS for downstream staining. For tetramer staining, 100 μl of the cell suspension was incubated for 1 h at room temperature with PE-conjugated or APC-conjugated I-Ab tetramers at a 1:25 dilution. After tetramer incubation, cells were washed and stained for extracellular markers using standard flow cytometry antibodies. Following final washes, cells were resuspended in 200 μl flow cytometry buffer (PBS + 2% FBS + 2 mM EDTA) and immediately analysed. Spinal columns were harvested and post-fixed overnight in 4% PFA at 4 °C. For histopathological analysis of both meningeal and parenchymal compartments, spinal columns were subsequently transferred to 0.5 M EDTA (pH 7.4; S28291GAL, Thermo Fisher Scientific) and decalcified at 4 °C with continuous inversion for 7 days. Following decalcification, tissues were processed in toto, paraffin-embedded, sectioned and mounted on glass slides. All histological analyses were performed by a board-certified anatomical pathologist (IDEXX BioAnalytics). Microscopic lesions were evaluated using established semiquantitative grading systems. Inflammatory changes were scored on a four-point scale as previously described65,66, ranging from no detectable inflammatory cells (score 0), to scattered infiltrates (score 1), perivascular clustering (score 2), and extensive perivascular cuffing with parenchymal extension or diffuse infiltration (score 3). Demyelination was graded based on Luxol Fast Blue staining from minimal subpial demyelination (score 1), through marked subpial and perivascular involvement (score 2), confluent subpial or perivascular demyelination (score 3), to extensive demyelination affecting one-half (score 4) or the entirety (score 5) of the spinal cord section, often accompanied by parenchymal immune infiltration. Numerical scores were used to quantify total lesion burden and to compare the severity and prevalence of pathological changes across treatment groups. STARmap padlock and primer probes were designed as previously described67. For the 64-gene STARmap data collection, 2–6 pairs of primer and padlock probes (Supplementary Table 4) were designed for each gene. The mice used in this study were anaesthetized with isoflurane and rapidly decapitated. The mouse spleen tissue was collected and placed in Tissue-Tek O.C.T. Two biological replicates were collected for each condition (Luc, DFI and wild type). For the post-DFI-treatment analysis, three biological replicates were collected for each condition (NaCl, Luc, DFI and 4-weeks post-DFI). was frozen in liquid nitrogen and stored at −80 °C. For mouse spleen tissue sectioning, the spleen tissue was transferred to cryostat (CM1950, Leica) and cut into 20-μm cross -ections at −20 °C. The slices were transferred and attached to glass-bottom 24-well plates pretreated with 3-(trimethoxysilyl)propyl methacrylate and poly-D-lysine. The STARmap procedure was conducted as previously described67. In brief, the spleen slices were fixed with 400 μl 4% PFA in 1× PBS at room temperature for 15 min, then permeabilized with 600 μl pre-chilled methanol at −20 °C for 1 h. The samples were then taken from −20 °C freezer to room temperature for 5 min, then quenched with 400 μl quenching solution (0.1 mg ml−1 yeast tRNA, 0.1 U μl−1 SUPERase·In RNase inhibitor, 100 mM glycine and 0.1% Tween-20 in PBS) at room temperature for 10 min. After quenching, the samples were rinsed with 600 μl PBSTR (1× PBS supplemented with 0.1% Tween-20 and 0.02 U μl−1 SUPERase·In RNase Inhibitor) twice. Then, the samples were incubated with 200 μl of 1× hybridization buffer (2× SSC, 10% formamide, 20 mM ribonucleoside vanadyl complex, 0.1 mg ml−1 yeast tRNA, 0.1 U μl−1 SUPERase·In, 1% Triton X-100, pooled padlock and primer probes at a concentration of 10 nM per oligo) at 40 °C in a humidified oven with parafilm wrapping and shaking for 20 h. The samples were washed with 300 μl PBSTR twice and 300 μl high-salt washing buffer (4× SSC dissolved in PBSTR) once, for 20 min at 37 °C for each wash. Finally, the samples were rinsed once with 300 μl PBSTR at room temperature. The samples were then incubated with a 200 μl ligation mixture (0.1 Weiss U μl−1 T4 DNA ligase, 0.5 mg ml−1 BSA and 0.2 U μl−1 of SUPERase·In RNase inhibitor in 1× T4 DNA ligase buffer) at room temperature for 3 h with gentle shaking. After the ligation reaction, the samples were washed twice with 300 μl PBSTR and then incubated with 200 μl rolling circle amplification mixture (0.2 Weiss U μl−1 Phi29 DNA polymerase, 250 μM dNTP, 20 μM 5-(3-aminoallyl)-dUTP, 0.5 mg ml−1 BSA and 0.2 U μl−1 of SUPERase·In RNase inhibitor in 1× Phi29 buffer) at 4 °C for 30 min then 30 °C for 3 h with gentle shaking. The samples were washed twice with 0.1% Tween-20 in PBS (PBST) before being temporarily stored at 4 °C overnight. The next day, the samples were treated with 300 μl freshly prepared modification mixture (20 mM methacrylic acid NHS ester in 100 mM sodium bicarbonate buffer) at room temperature for 1 h and then washed once by PBST for 5 min. The samples were rinsed once and then incubated with 200 μl monomer buffer (4% acrylamide, 0.2% bis-acrylamide and 0.2% tetramethylethylenediamine in 2× SSC) at 4 °C for 15 min. Then, the buffer was aspirated, and 50 μl polymerization mixture (0.2% ammonium persulfate dissolved in pre-cooled monomer buffer) was added to the centre of the sample and immediately covered by Gel Slick-coated glass coverslip (72226-01, Electron Microscopy Sciences). The polymerization reaction was performed for 1 h at room temperature in an N2 box, then washed by PBST twice for 5 min each. The samples embedded in hydrogel were digested with 300 μl proteinase K mixture (0.5 mg ml−1 proteinase K and 1% SDS in 2× SSC) at 37 °C for 3 h, then washed by PBST three times for 5 min each. Subsequently, the samples were treated with 200 μl dephosphorylation mixture (0.25 U μl−1 Antarctic phosphatase, 0.5 mg ml−1 BSA in 1× Antarctic phosphatase buffer) at 37 °C overnight and washed by PBST three times for 5 min each. For SEDAL sequencing, each sequencing cycle began with treating the sample with 800 μl stripping buffer (60% formamide and 0.1% Triton X-100 in H2O) at room temperature twice for 10 min each, followed by washing with 1 ml PBST three times for 5 min each. Then, the samples were incubated with a 250 μl sequencing mixture (0.1875 U μl−1 T4 DNA ligase, 0.5 mg ml−1 BSA, 10 μM reading probe and 5 μM decoding probes in 1× T4 DNA ligase buffer) at room temperature for at least 5 h. The samples were washed with 900 μl washing and imaging buffer (10% formamide in 2× SSC buffer) three times for 10 min each, then immersed in washing and imaging buffer for imaging. Images were acquired using Leica TCS SP8 confocal microscopy with ×40 oil immersion objective (NA 1.3) and a voxel size of 194 nm × 194 nm × 350 nm (x × y × z). DAPI staining was performed before the first cycle using 5× DAPI in PBST for 3 h at room temperature. Four cycles of imaging were performed to decode 64 genes. After SEDAL sequencing, the samples were treated with 800 μl of stripping buffer three times at room temperature for 10 min each. The samples were then washed with 1 ml PBST three times for 5 min each. To visualize the plasma membrane and aid cell segmentation, the samples were treated with 300 μl Flamingo staining mixture (1× Flamingo fluorescent gel stain and 5× DAPI in PBS) at room temperature overnight, then washed with 300 μl PBST three times for 5 min each. The Flamingo signal was collected and the DAPI signal was reimaged while the samples were immersed in PBST with 0.1× Flamingo fluorescent gel stain. STARmap data analysis was performed as previously described43,67. In brief, image deconvolution was achieved with Huygens Essential (v21.04; Scientific Volume Imaging (http://svi.nl)), using the CMLE algorithm, with SNR:10 and 10 iterations. Image registration, spot calling and barcode filtering were performed as previously described. For 3D cell segmentation, a synthetic image with improved contrast between cell nuclei was generated by multiplying the inverted Flamingo staining image and DAPI staining image after enhancing contrast using Fiji for each field of view (FOV). A StarDist 3D segmentation model was then trained using a manually labelled training dataset created from the synthetic data68. Subsequently, the model was applied to predict segmentation for each FOV. For tissue region identification, to isolate low-frequency (that is, large length scale) transcriptional patterns over the tissue, Laplacian smoothing over a spatial Delaunay triangulation (that is, a nearest neighbour mesh of cells) was performed. The specific low-pass filter used was a heat kernel with time t = 10. Principal component analysis was then performed on the resulting low-pass-filtered cell-by-gene matrix to identify region features, followed by clustering in the principal component space to produce categorical region labels. K-means was used for clustering to avoid smoothing-induced spatial autocorrelation artefacts69. For quality control and cell-type classification, cells with less than two reads and expressed fewer than two genes were excluded. Gene expression profiles were normalized and scaled using standard Scanpy procedures70. A hierarchical clustering approach was then utilized to create a three-level cell-type annotation (Supplementary Fig. Initially, 24 clusters were identified through k-means clustering of the preprocessed gene expression profile containing 19 genes, which were further categorized into four level 1 cell types (T cells, B cells, macrophages and dendritic cells; Supplementary Fig. Cells lacking expression of any of the selected 19 gene markers were removed from subsequent analysis. Level 2 CD4+ or CD8+ T cells then underwent a level 3 k-means clustering using a panel of 15 markers for further refinement in classification (Supplementary Fig. Sixteen genes were profiled to validate the expression pattern of the three therapeutic mRNAs on liver and spleen tissues. The three therapeutic mRNAs, Dll1, Il7 and Flt3l, were profiled using RIBOmap, whereas the 13 cell-type marker genes were profiled using STARmap. RIBOmap also excludes endosomal and extracellular mRNA signals, which are commonly found in mRNA-injected tissue samples. The mice used in this experiment were anaesthetized with isoflurane and rapidly decapitated. Three biological replicates with therapeutic mRNA injections were collected, and one control sample with Luc mRNA injection was collected. Then, the liver and spleen tissues in O.C.T. were frozen in liquid nitrogen and stored at −80 °C. The translation levels of Dll1, Il7 and Flt3l were quantified at a single-cell level. Heart blood was aspirated by cardiac puncture using EDTA-precoated syringes and collected in BD Microtainer Tubes with potassium EDTA additive (365974, BD). Serum was isolated by centrifugation at 2,000g for 10 min at room temperature and snap frozen on dry ice. All samples were received by IDEXX BioAnalytics and stored securely at −80 °C before analysis. Serum AST, ALT, CK, albumin, triglycerides and GGT were measured by an Olympus AU5400 (IDEXX BioAnalytics). Samples demonstrating haemolysis were excluded from the analysis. Heart blood was aspirated by cardiac puncture using EDTA-precoated syringes and collected in BD Microtainer Tubes with potassium EDTA additive (365974, BD). Serum was isolated by centrifugation at 2,000g for 10 min at room temperature and snap frozen on dry ice. All samples were received by IDEXX BioAnalytics and stored securely at −80 °C before analysis. Samples were tested on the Milliplex MAP mouse cytokine/chemokine magnetic bead panel (MCYTOMAG-70K-PMX, Millipore) according to the kit protocol as qualified. Analyte standards, quality controls and sample MFI values were adjusted for background. Calibrator data were fit to either a five-parameter logistic or four parameter logistic model depending on best fit to produce accurate standard curves for each analyte. Quality control and sample data were interpolated from the standard curves and then adjusted according to the dilution factor to provide calculated final concentrations of each analyte present in the sample. Samples demonstrating haemolysis were excluded from the analysis. To evaluate the hepatic safety of LNP administration, liver tissue was harvested from female C57BL/6J mice. Twelve mice (n = 4 per group) comprising adult untreated controls, aged mice treated for 28 days with Luc–LNPs and aged mice treated for 28 days with DFI–LNPs were intracardially perfused under terminal anaesthesia with ice-cold 1× PBS, followed by 4% PFA. Livers were dissected, post-fixed in 4% PFA overnight at 4 °C and submitted to IDEXX BioAnalytics for blinded histopathological assessment. Upon receipt, liver tissues were trimmed, paraffin embedded, sectioned and mounted onto glass slides. Sections were routinely stained with haematoxylin and eosin to assess general histology and hepatocellular integrity. Serial sections were stained with Masson's Trichrome to visualize connective tissue and assess the presence of fibrosis. Microscopic evaluation was performed by board-certified veterinary pathologists blinded to treatment groups. Numerical lesion scores were used to quantify both prevalence and severity of histological changes within and across experimental groups. Heart blood was obtained through cardiac puncture in mice following cervical dislocation and stored in 0.5% EDTA for subsequent processing. Red blood cells were lysed using 1 ml ACK lysis buffer (Thermo). Freshly isolated immune cells post-lysis of red blood cells were blocked with rat anti-mouse CD16/32 (0.5 μg per well, eBioscience). Subsequently, respective antibodies in PBS were added in a total volume of 50 μl and stained for 30 min. eFluor 780 fixable viability dye (eBioscience) was used per the manufacturer's protocol to exclude dead cells. For scRNA-seq, cells were divided into eight aliquots per animal sample and pre-incubated for 10 min with titrated amounts of TotalSeq hashtag antibodies (C0301-C0308, BioLegend). Cells were sorted on a BD Aria Fusion cell sorter using a 100-μM nozzle and four-way purity mode. From peripheral blood, viable T cells (live, CD45+ and CD3+) were sorted in 20 μl 0.04% BSA in PBS and kept on ice until processing. Single-cell capture, reverse transcription and library preparation were conducted on the Chromium platform (10X Genomics) with the single-cell 5′ reagent v2 kit (10X Genomics) following the manufacturer's protocol, using 40,000 cells as input per channel. The final library for each pool was subjected to paired-end sequencing (26 bp and 92 bp) on one Illumina NovaSeq 6000 S2 lane. Raw sequencing data were processed and aligned to the mouse genome (GRCm39 — mm39) using the CellRanger pipeline (10X Genomics, v7.1.0). Seurat datasets were generated for blood T cells at each time point. Singlets were identified per the published Seurat vignette (https://satijalab.org/seurat/articles/hashing_vignette.html) and used for downstream analyses. In addition, only cells with more than 500 and less than 4,000 unique features were detected, and less than 5% of mitochondrial counts were considered for further analysis. The integration process involved the application of the following arguments: NormalizeData(), FindVariableFeatures(selection.method = “vst”, nfeatures = 2000),ScaleData(), RunPCA(), RunHarmony(“orig.ident”, plot_convergence = TRUE, dims.use = 1:20), RunUMAP(reduction = “harmony”, dims = 1:20), FindNeighbors(reduction = “harmony”, dims = 1:20), FindClusters(resolution = 0.5). After removing cells failing quality control, integration was reiterated with the same settings, and final transcriptional clusters for downstream analyses were identified using the FindClusters function (resolution = 0.7). Annotation of mouse peripheral blood datasets was carried out manually, based on the final transcriptional clustering post-quality control and integration, using canonical marker genes and differential gene expression analyses through the FindAllMarkers function in Seurat. For subsequent visualization subsets for CD8 T cells (clusters: CD8_T activated, CD8_T cytotox, CD8_T effector-like, CD8_T exhausted, CD8_T IFN-responsive, CD8_T memory-like and CD8_T naive-like; 56,967 cells) and CD4 T cells (clusters: CD4_T activated, CD4_T IFN-responsive, CD4_T memory-like/naive, CD4_T naive and Treg; 39,716 cells) were built and a new UMAP embedding based on the harmony components was calculated. The statistical significance of changes in cell-type abundance over time in the single-cell data was assessed using Pearson correlation through the cor.test(…, method= “pearson”) function. For visualization, log2 fold changes of cell-type proportions at each time point relative to the 4-week time point were computed and graphically represented. Cell types exhibiting a significant correlation with increasing age (P < 0.05) were colour annotated. Density UMAP visualization was performed using the ggplot2 stat_density_2d() function. B16-OVA were explanted at day +12 post-injection so that tumour sizes remained comparable between treatment groups. Tumour tissues were minced and enzymatically digested in RPMI supplemented with Liberase (5401119001, Roche) for 30 min at 37 °C with gentle agitation. Following digestion, tissues were filtered through a 70-μm cell strainer and centrifuged. The resulting cell pellet was resuspended in a 40% Percoll–PBS solution and layered on top of a 80% Percoll–PBS solution in 50 ml followed by centrifugation at 1,260g for 20 min at room temperature with the acceleration at the lowest setting and no break. The middle interface layer containing TILs was isolated and stained with a fixable viability dye (eFluor780) respective fluorophore-labelled antibodies, as well as an oligo-conjugated and PE-labelled H-2Kb SIINFEKL dextramer (JD02163DXG PE 25, Immudex). Viable T cells (live, CD45+ and CD3+) were sorted in 20 μl 0.04% BSA in PBS and kept on ice until processing. Single-cell capture, reverse transcription and library preparation were conducted on the Chromium platform (10X Genomics) with the single-cell 5′ reagent v2 kit (10X Genomics) following the manufacturer's protocol, using 20,000 cells as input per channel. Raw sequencing data were processed and aligned to the mouse genome (GRCm39 — mm39) using the CellRanger pipeline (10X Genomics, v7.1.0). Single-cell transcriptomic and V(D)J data were generated from four parallel runs, each consisting of sorted TILs isolated and pooled from two animals. Raw gene expression and antibody-derived tag (ADT) matrices using the conjugated oligo sequence (CCCATATAAGAAA) of the H-2Kb SIINFEKL dextramer were processed in R (v4.3.2) using the Seurat package (v5.3.0)71. Cells were retained if they expressed at least 200 but no more than 7,000 genes and exhibited less than 5% mitochondrial gene content. Genes detected in fewer than three cells (CreateSeuratObject(…,min.cells = 3)) were excluded. ADT assays were normalized using centred log-ratio transformation. TCR contig annotations were added to the scRNA dataset per sample using scRepertoire (v2.0.7)72. Clonal proportion was calculated per run. For downstream analyses, only cells with annotated TCR were used. T cells were projected onto the default TIL reference atlas using ProjecTILs (v3.5.2) to enable canonical cell-type annotation73. For visualization, the reference atlas UMAP embedding was used. H-2Kb-SIINFEKL dextramer staining was analysed from ADT assays. Cells with a centred log-ratio-normalized tetramer signal of more than 1.7 were classified as SIINFEKL specific (Tet+). Clonal diversity was calculated using scRepertoire function clonalDiversity(…,cloneCall = “aa”). To this end, cells per treatment condition were downsampled to the same number of cells and used for visualization. Upstream data generation and analysis have been published elsewhere16. For each pair of cell types at each time point, a Z-score was computed by comparing the observed frequency of cell interactions for the pair within a 50-μm radius to the null hypothesis of the frequency of cell interactions for the same pair within a 150-μm radius. Subsequently, these Z-scores were plotted over time, and Pearson's r was used to assess the strength of the correlation between changing Z-scores and age. Upstream data generation and analysis have been published elsewhere16. We used the Squidpy74 integration of CellPhoneDB75 and Omnipath76 to identify shifts in receptor–ligand interactions at each time point. Subsequently, the mean values of each receptor–ligand cell–cell pair across time were used to calculate Spearman's R correlation and a P value. After correcting the P values through Bonferroni correction, unique interactions were categorized into those decreasing and increasing with age for visualization. Chemicals and enzymes are listed as name (catalog number, vendor): Tissue-Tek O.C.T. Compound (4583, SAKURA); glass bottom 24-well plates (P24-1.5H-N, Cellvis); 3-(trimethoxysilyl) propyl methacrylate (M6514, Sigma-Aldrich); poly-D-lysine (A-003-M, Sigma-Aldrich); 16% PFA (15710-S, Electron Microscopy Sciences); UltraPure DNase/RNase-free distilled water (10977023, Invitrogen); methanol (34860-1L-R, Sigma-Aldrich); PBS (10010-023, Gibco); Tween-20, 10% solution (655206, Calbiochem); yeast tRNA (AM7119, Thermo Fisher Scientific); SUPERase·In RNase inhibitor (AM2696, Thermo Fisher Scientific); UltraPure SSC, 20× (15557044, Invitrogen); formamide (655206, Calbiochem); ribonucleoside vanadyl complex (S1402S, New England Biolabs); T4 DNA ligase, 5 Weiss U μl−1 (EL0012, Thermo Fisher Scientific); Phi29 DNA polymerase (EP0094, Thermo Fisher Scientific); deoxynucleotide (dNTP) solution mix (N0447L, New England Biolabs); UltraPure BSA (AM2618, Thermo Fisher Scientific); 5-(3-aminoallyl)-dUTP (AM8439, Thermo Fisher Scientific); methacrylic acid NHS ester, 98% (730300, Sigma-Aldrich); DMSO, anhydrous (D12345, Invitrogen); acrylamide solution, 40% (161-0140, Bio-Rad); Bis solution, 2% (161-0142, Bio-Rad); ammonium persulfate (A3678, Sigma-Aldrich); N,N,N′,N′-tetramethylethylenediamine (T9281, Sigma-Aldrich); Gel Slick Solution (50640, Lonza Bioscience); OminiPur SDS, 20% (7991, Calbiochem); proteinase K solution, RNA grade (25530049, Thermo Fisher Scientific); Antarctic phosphatase (M0289L, New England Biolabs); DAPI (D1306, Molecular Probes); 10% Triton X-100 (93443, Sigma-Aldrich); and Flamingo fluorescent protein gel stain (1610490, Bio-Rad). For immunohistochemistry, immunofluorescence and STARmap analyses of DLL1: anti-DLL1 antibody 1:800 (ab10554, Abcam) and AF546-labelled secondary donkey anti-rabbit IgG antibody 1:500 (A10040, Invitrogen) were used. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All single-cell profiling data of circulating T cells and B16-OVA TILs in ageing mice are available on Zenodo77 (accession number 17385897). All spatial profiling (RIBOmap and STARmap PLUS) data of liver and spleen samples are available on Zenodo78 (accession number 13858686). All spatial profiling (Slide-TCR-seq) data of thymuses across age are available on Zenodo79 (accession number 17525729). All codes necessary to reproduce all spatial thymus RNA-seq analyses (https://github.com/immunoliugy/aging_mouse_thymus) and all RIBOmap and STARmap PLUS analyses (https://github.com/wanglab-broad/hepatic_recon) are available on GitHub. Mittelbrunn, M. & Kroemer, G. Hallmarks of T cell aging. López-Otín, C., Pietrocola, F., Roiz-Valle, D., Galluzzi, L. & Kroemer, G. Meta-hallmarks of aging and cancer. Ventevogel, M. S. & Sempowski, G. D. Thymic rejuvenation and aging. & van den Brink, M. R. M. Thymus: the next (re)generation. Beck, J. D. et al. Long-lasting mRNA-encoded interleukin-2 restores CD8+ T cell neoantigen immunity in MHC class I-deficient cancers. Quijano-Rubio, A. et al. A split, conditionally active mimetic of IL-2 reduces the toxicity of systemic cytokine therapy. The alarmin IL33 orchestrates type 2 immune-mediated control of thymus regeneration. Sharma, H. & Moroni, L. Recent advancements in regenerative approaches for thymus rejuvenation. Three month heterochronic parabiosis has a deleterious effect on the lifespan of young animals, without a positive effect for old animals. Depleting myeloid-biased haematopoietic stem cells rejuvenates aged immunity. & Dardenne, M. Hormonal control of T-cell development in health and disease. & Farber, D. L. Human T cell development, localization, and function throughout life. & Restifo, N. P. Human memory T cells: generation, compartmentalization and homeostasis. Liu, S. et al. Tissue architecture dynamics underlying immune development and decline in the thymus. & Artyomov, M. N. Immune ageing at single-cell resolution. Comprehensive profiling of an aging immune system reveals clonal GZMK+CD8+ T cells as conserved hallmark of inflammaging. New insights into the regulation of T cells by γc family cytokines. & Wodnar-Filipowicz, A. Intrathymic expression of Flt3 ligand enhances thymic recovery after irradiation. Williams, K. M. et al. FLT3 ligand regulates thymic precursor cells and hematopoietic stem cells through interactions with CXCR4 and the marrow niche. Hellström, M. et al. Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis. Morell, C. M. & Strazzabosco, M. Notch signaling and new therapeutic options in liver disease. Warren, A. et al. T lymphocytes interact with hepatocytes through fenestrations in murine liver sinusoidal endothelial cells. Kawashima, K., Andreata, F., Beccaria, C. G. & Iannacone, M. Priming and maintenance of adaptive immunity in the liver. Effect of age on liver protein synthesis and degradation. Sahin, U., Karikó, K. & Türeci, Ö mRNA-based therapeutics-developing a new class of drugs. Karikó, K. et al. Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. Chen, H. et al. Branched chemically modified poly(A) tails enhance the translation capacity of mRNA. Zeng, H. et al. Spatially resolved single-cell translatomics at molecular resolution. Scavuzzo, M. A. et al. Pancreatic cell fate determination relies on Notch ligand trafficking by NFIA. Roberts, R. et al. Heparan sulphate bound growth factors: a mechanism for stromal cell mediated haemopoiesis. Induction of T cell adhesion to extracellular matrix or endothelial cell ligands by soluble or matrix-bound interleukin-7. Elimination of large tumors in mice by mRNA-encoded bispecific antibodies. Schluns, K. S., Kieper, W. C., Jameson, S. C. & Lefrançois, L. Interleukin-7 mediates the homeostasis of naïve and memory CD8 T cells in vivo. & Mackall, C. L. The many faces of IL-7: from lymphopoiesis to peripheral T cell maintenance. & Gress, R. E. Harnessing the biology of IL-7 for therapeutic application. Pang, W. W. et al. Human bone marrow hematopoietic stem cells are increased in frequency and myeloid-biased with age. Zlotoff, D. A. et al. CCR7 and CCR9 together recruit hematopoietic progenitors to the adult thymus. Karsunky, H., Merad, M., Cozzio, A., Weissman, I. L. & Manz, M. G. Flt3 ligand regulates dendritic cell development from Flt3+ lymphoid and myeloid-committed progenitors to Flt3+ dendritic cells in vivo. The receptor tyrosine kinase Flt3 is required for dendritic cell development in peripheral lymphoid tissues. Shi, H. et al. Spatial atlas of the mouse central nervous system at molecular resolution. Hao, Y., O'Neill, P., Naradikian, M. S., Scholz, J. L. & Cancro, M. P. A B-cell subset uniquely responsive to innate stimuli accumulates in aged mice. Rubtsova, K., Rubtsov, A. V., Cancro, M. P. & Marrack, P. Age-associated B cells: a T-bet-dependent effector with roles in protective and pathogenic immunity. Collier, D. A. et al. Age-related immune response heterogeneity to SARS-CoV-2 vaccine BNT162b2. Aging promotes reorganization of the CD4 T cell landscape toward extreme regulatory and effector phenotypes. Tsukamoto, H. et al. Aging-associated and CD4 T-cell-dependent ectopic CXCL13 activation predisposes to anti-PD-1 therapy-induced adverse events. Georgiev, P. et al. Age-associated contraction of tumor-specific T cells impairs antitumor immunity. Duhen, T. et al. Co-expression of CD39 and CD103 identifies tumor-reactive CD8 T cells in human solid tumors. Prediction of spontaneous autoimmune diabetes in NOD mice by quantification of autoreactive T cells in peripheral blood. Ehst, B. D., Ingulli, E. & Jenkins, M. K. Development of a novel transgenic mouse for the study of interactions between CD4 and CD8 T cells during graft rejection. Aharoni, R. et al. Age dependent course of EAE in Aire−/− mice. Biological aging of CNS-resident cells alters the clinical course and immunopathology of autoimmune demyelinating disease. Desdín-Micó, G. et al. T cells with dysfunctional mitochondria induce multimorbidity and premature senescence. Han, S., Georgiev, P., Ringel, A. E., Sharpe, A. H. & Haigis, M. C. Age-associated remodeling of T cell immunity and metabolism. Park, M. D. et al. Hematopoietic aging promotes cancer by fueling IL-1α-driven emergency myelopoiesis. Sladitschek-Martens, H. L. et al. YAP/TAZ activity in stromal cells prevents ageing by controlling cGAS-STING. Sanmarco, L. M. et al. Lactate limits CNS autoimmunity by stabilizing HIF-1α in dendritic cells. Blood levels of T-cell receptor excision circles (TRECs) provide an index of exposure to traumatic stress in mice and humans. Broers, A. E. C. et al. Quantification of newly developed T cells in mice by real-time quantitative PCR of T-cell receptor rearrangement excision circles. Ma, X. et al. Berberine attenuates experimental autoimmune encephalomyelitis in C57 BL/6 mice. Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Weigert, M., Schmidt, U., Haase, R., Sugawara, K. & Myers, E. Star-convex polyhedra for 3D object detection and segmentation in microscopy. Maher, K. et al. Mitigating autocorrelation during spatially resolved transcriptomics data analysis. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Integrated analysis of multimodal single-cell data. Borcherding, N., Bormann, N. L. & Kraus, G. scRepertoire: an R-based toolkit for single-cell immune receptor analysis. Interpretation of T cell states from single-cell transcriptomics data using reference atlases. Palla, G. et al. Squidpy: a scalable framework for spatial omics analysis. & Vento-Tormo, R. CellPhoneDB: inferring cell-cell communication from combined expression of multi-subunit ligand-receptor complexes. Türei, D., Korcsmáros, T. & Saez-Rodriguez, J. OmniPath: guidelines and gateway for literature-curated signaling pathway resources. Friedrich, M. Sequencing data of Friedrich et al., Transient hepatic reconstitution of trophic factors enhances aged immunity [data set]. Friedrich, M. J. et al. Spleen STARmap PLUS dataset of Transient hepatic reconstitution of thymic factors enhances aged immunity [data set]. Liu, S. et al. Tissue architecture dynamics underlying immune development and decline in the thymus [data set]. is supported by the Howard Hughes Medical Institute; the K. Lisa Yang Brain-Body Center at MIT; the Broad Institute Programmable Therapeutics Gift Donors; the Pershing Square Foundation, William Ackman and Neri Oxman; and by the Phillips family and J. and P. Poitras. acknowledges support from Impetus Grants, the Mark & Lisa Schwartz AI Initiative, the Margaret Q. Landenberger Research Foundation and BroadIgnite. Howard Hughes Medical Institute, Cambridge, MA, USA Mirco J. Friedrich, Julie Pham, Blake Lash, Rhiannon K. Macrae & Feng Zhang Broad Institute of MIT and Harvard, Cambridge, MA, USA Mirco J. Friedrich, Julie Pham, Jiakun Tian, Hongyu Chen, Jiahao Huang, Niklas Kehl, Sophia Liu, Blake Lash, Fei Chen, Xiao Wang, Rhiannon K. Macrae & Feng Zhang McGovern Institute for Brain Research at MIT, Cambridge, MA, USA Mirco J. Friedrich, Julie Pham, Blake Lash, Rhiannon K. Macrae & Feng Zhang Mirco J. Friedrich, Julie Pham, Blake Lash, Rhiannon K. Macrae & Feng Zhang Mirco J. Friedrich, Julie Pham, Blake Lash, Rhiannon K. Macrae & Feng Zhang Jiakun Tian, Hongyu Chen, Jiahao Huang & Xiao Wang Ragon Institute of MGH, MIT and Harvard, Cambridge, MA, USA Harvard Stem Cell Institute, Cambridge, MA, USA Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA, USA Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed all the experiments with input from F.Z., R.K.M. performed the longitudinal scRNA–TCR-seq and Slide-TCR-seq experiments. performed the longitudinal scRNA–TCR-seq and Slide-TCR-seq analyses. provided critical mentorship and guidance in technical procedures. supervised this research and experimental design with support from R.K.M. wrote the manuscript with input from all authors. are co-inventors on US provisional patent application no. 63/530,465 filed by the Broad Institute and MIT relating to this work. is a scientific advisor and cofounder of Beam Therapeutics, Pairwise Plants, Arbor Biotechnologies, Aera Therapeutics and Moonwalk Biosciences; and is a scientific advisor for Octant. All other authors declare no competing interests. Nature thanks Maria Mittelbrunn, Marcel van den Brink and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (A) Experimental overview for longitudinal profiling of peripheral and thymic T cells. Circulating T cells were isolated from peripheral blood of adult (6 weeks), middle-aged (36 weeks), and aged (72 weeks) mice and subjected to 10x Genomics 5′ single-cell RNA-sequencing. Thymuses were explanted for spatial transcriptomic profiling using Slide-seq v2. (B) Representative flow cytometry gating strategy for sorting peripheral blood CD3+ T cells for single-cell RNA-sequencing. (C) UMAP of CD8+ T cells (CD45+ CD3+ CD8+) across all age groups with cell-type annotations. (D) Expression feature plots of canonical gene markers overlaid on UMAP from (C). (E) Proportional changes in CD8+ T cell subsets with age. Data shown as fold change relative to 4-week-old mice. Statistically significant changes highlighted in color (methods). (F) UMAP of CD4+ T cells (CD45+ CD3+ CD4+) across all age groups with cell-type annotations. (G) Expression feature plots of canonical gene markers overlaid on UMAP from (F). (H) Proportional changes in CD4+ T cell subsets with age. Data shown as fold change relative to 4-week-old mice. Statistically significant changes highlighted in color (methods). (I) Heatmap showing expression of exhaustion-associated genes across circulating T cells stratified by age. Arrowhead denotes the average time point of overt thymic involution in C57BL/6 J mice. (A) Dot plot of gene expression across annotated T cell subsets identified in single-cell RNA-seq data from peripheral CD45+ CD3+ T cells. (B) Left: UMAP of CD8+ T cells across all age groups annotated by subset. Right: same UMAP colored by mouse age. (C) Left: UMAP of CD4+ T cells across all age groups annotated by subset. Right: same UMAP colored by mouse age. (A-B) Age-associated decline in spatial interaction strength between thymic cortical epithelial cells (cTEC) and thymocytes (A), and medullary epithelial cells (mTEC) and thymocytes (B). Interaction scores derived from ligand-receptor inference models. Pearson's correlation (R) and P values shown. Arrowheads indicate time of thymic involution. Violin plots display mean interaction scores before and after involution; significance assessed by two-tailed unpaired t-test. (C-D) Receptor-ligand interaction heatmaps showing cell-cell communication pathways that decrease (C) or increase (D) with age across thymic cell populations. Interactions grouped by signaling axis and stratified by age category: neonatal (n = 7 spatial arrays from 3 mice), adult (n = 15 from 7 mice), middle-aged (n = 13 from 6 mice), and aged (n = 10 from 4 mice). (E) Pathway-level enrichment of Notch, FLT3, and IL-7 signaling in peripheral T cells. Violin plots of average enrichment scores per mouse. Data from n = 96,683 peripheral blood T cells. (F) Differential expression of Notch, FLT3, and IL-7 target genes in circulating T cells of adult vs. aged mice. Data from n = 96,683 peripheral blood T cells. (G) Absolute FLT3-L levels in explanted thymuses from mice across age (n = 3 animals per age group). (A-B) Verification of DFI factor production by primary hepatocytes. (A) Confocal immunofluorescence of primary murine hepatocytes stained for DLL1 surface expression and β-catenin (membrane marker). (B) Extracellular concentrations of biologically active IL-7 and FLT3-L secreted by mRNA-transfected immortalized hepatocytes, as measured by IL-7 and FLT3-L ELISA. Statistical significance was tested using two-tailed unpaired t-tests. (C) Supernatant cytokine levels of FLT3L and IL-7 from primary hepatocytes following DFI mRNA-LNP treatment, quantified by ELISA. (D) Representative RIBOmap deconvolved images of ribosome-bound transcripts in liver and spleen 6 h post-DFI injection. Note liver image (bottom) is repeated from Fig. Representative image from three imaged DFI-treated animals. (E) Venn diagram showing overlap of hepatocytes expressing ribosome-bound transcripts for each of the three DFI mRNAs. (F) Representative immunofluorescence staining of liver tissue for DLL1 protein (yellow), F-actin (Phalloidin, magenta), and nuclei (DAPI, cyan) following DFI administration. (G) Body weights of 72-week-old C57BL/6 J mice over 28 days of treatment with either Luc control or DFI mRNA–LNPs. (H) Serum liver function parameters including ALT, AST, ALP, and bilirubin measured in aged animals treated with vehicle (NaCl), DFI mRNA–LNPs, or recombinant IL-7/FLT3L protein. Statistical analysis was performed using one-way ANOVA followed by Tukey's multiple comparison test. (I-J) Representative hematoxylin and eosin (H&E) and Masson's trichrome stainings of liver sections from aged mice treated with NaCl (C) or DFI mRNA–LNPs (D). Representative images from one out of 12 imaged animals. (K) Histopathological scoring of liver lesions based on inflammation, hepatocellular degeneration, and fibrosis. (L) Cumulative pathology scores across all assessed features in aged NaCl-, Luc-, and DFI-treated mice. (A) Representative flow cytometry gating strategy for the identification and quantification of CD45+ CD3+ T cells in spleens from adult (6 weeks) and aged (72 weeks) mice. (B) Quantification of total CD45+ CD3+ T cell frequency following 28 days of biweekly DFI or Luc mRNA-LNP treatment. Statistical significance was assessed using one-way ANOVA followed by Tukey's post hoc test. (C-E) Frequencies of naïve (CD44− CD62L+), central memory (CM; CD44+ CD62L+), and effector memory (EM; CD44+ CD62L−) T cells (C), CD8+ T cells (D), and CD4+ T cells (E). Statistical comparisons were performed by one-way ANOVA with Tukey's post hoc test. (A) Weights of explanted thymuses from adult (6 weeks), aged + Luc mRNA, and aged + DFI mRNA-treated (72 weeks) mice. (B) Representative H&E-stained thymic tissue sections from each group, highlighting cortical and medullary organization. (C) Quantification of thymic cellularity (×10³ cells/mm²) from explanted thymuses. One-way ANOVA with Tukey's post hoc test was used to assess statistical significance. (D) Representative flow cytometry gating strategy for quantification of thymocyte subsets from explanted thymuses. (E) Quantification of major thymocyte subsets: double-negative (DN), double-positive (DP), and single-positive (SP) CD4+ and CD8+ thymocytes. Absolute counts were normalized to thymus weight (mg). Data are shown as mean ± s.e.m., one-way ANOVA with Tukey's post hoc test. Data are shown as mean ± s.e.m., one-way ANOVA with Tukey's post hoc test. DFI was injected at 0 h. (B) Representative flow cytometry gating strategy to quantify Rag2 expression levels in double- negative (DN), double-positive (DP), single-positive CD4+ (SP CD4+) and single-positive CD8+ (SP CD8+) thymocytes. (C) Mean fluorescence intensity (MFI) of Rag2-EGFP at indicated timepoints. n = 3 Rag2-EGFP−/− mice were used as negative control. Data are shown as mean ± s.e.m., repeated measures two-way ANOVA with Dunnett's post hoc test. DFI was injected at 0 h and 72 h. (E) Representative flow cytometry gating strategy to quantify Nur77 expression levels in CD4+ and CD8+ T cells in peripheral blood. (F) Mean fluorescence intensity (MFI) of Nur77-GFP at indicated timepoints in CD4+ (left) and CD8+ (right) T cells in peripheral blood. n = 3 Nur77-GFP−/− mice were used as negative control. Data are shown as mean ± s.e.m., one-way ANOVA with Dunnett's post hoc test. (A) Representative flow cytometry gating (left) and quantification (right) of total hematopoietic stem cells (HSCs; Lin− c-Kit+ Sca-1+ FLT3− CD34− CD150+), multipotent progenitor a (MPPa; Lin− c-Kit+ Sca-1+ FLT3− CD34+ CD150+), and multipotent progenitor b (MPPb; Lin− c-Kit+ Sca-1+ FLT3− CD34+ CD150−), as described previously17. (B) Gating (left) and quantification (right) of multipotent progenitor c (MPPc; Lin− c-Kit+ Sca-1+ FLT3+ CD34+ CD150−), as described previously17. (C) Gating (left) and quantification (right) of myeloid-biased HSCs (my-HSCs; Lin− c-Kit+ Sca-1+ FLT3− CD34− CD150high) and balanced HSCs (bal-HSCs; Lin− c-Kit+ Sca-1+ FLT3− CD34− CD150low). Gate to define my-HSC vs. bal-HSC was set as described previously17. (D) Ratio of my-HSCs to bal-HSCs across experimental groups. (E) Gating (left) and quantification (right) of common myeloid progenitors and granulocyte–macrophage progenitors (CMP/GMP; Lin− c-Kit+ Sca-1− CD34+ CD41−) (F) Gating (left) and quantification (right) of common lymphoid progenitors (CLPs; Lin− c-Kitint Sca-1int IL-7Rα+ FLT3+). (G) Schematic illustration of the hierarchical hematopoietic stem and progenitor cell (HSPC) lineage tree used for comparative analysis. (H) Fold changes in average frequencies of each HSPC subset in aged relative to adult bone marrow. (I) Fold changes in average frequencies of each HSPC subset in aged DFI-treated relative to aged Luc-treated bone marrow. Statistical significance was assessed by one-way ANOVA followed by Tukey's multiple-comparison post hoc test. (A) Representative flow cytometry gating strategy to identify circulating CLPs (Lin− IL-7Rα+ Sca-1+ c-Kit+) in peripheral blood. (B) Left: Weights of explanted thymuses from each group. (C) Left: Representative histograms showing CCR9 surface expression on intrathymic CLPs. Right: Frequencies of CCR9+ CLPs in thymus. (D) Top: Mean fluorescence intensity (MFI) of CCR9 on CLPs from bone marrow and thymus. Bottom: Fold change in CCR9 expression (thymus:bone marrow). Data analyzed by repeated measures two-way ANOVA and Tukey's post hoc test. (E) Top: MFI of CCR7 on CLPs from bone marrow and thymus. Bottom: Fold change in CCR7 expression (thymus:bone marrow). (A) Gating strategy for tetramer staining of KYNKANAFL-H-2d-specific CD8+ T cells in the peripheral blood of NOD mice. (B) Schematic of the experimental design for experimental autoimmune encephalomyelitis (EAE) induction in adult and aged mice treated with DFI or Luc mRNA. Mice were immunized with MOG35–55 peptide and pertussis toxin, treated with mRNA–LNPs every 3 days until symptom onset, and monitored over 28 days. (C-D) Longitudinal EAE clinical scores in (C) adult and (D) aged mice. Mice were scored twice daily using a 0–5 scale: 0, no signs; 1, limp tail; 2, hindlimb weakness; 3, hindlimb paralysis; 4, forelimb paralysis; 5, moribund. (E-F) Frequencies of MOG38–49-I-Ab tetramer-binding CD4+ T cells in spleens. (G) Representative spinal cord sections stained with H&E (top) and Luxol Fast Blue (bottom) to evaluate inflammatory infiltrates and demyelination. Arrows indicate areas of demyelination; arrowheads mark inflammatory foci. (H) Quantification of demyelination severity in spinal cords. P values calculated using one-way ANOVA with Tukey's multiple-comparison post hoc test. Raw values of serum cytokine measurements performed from peripheral blood of DFI mRNALNP, recombinant protein, or NaCl-treated animals. Raw values of serum cytokine measurements performed from peripheral blood of DFI mRNA-LNP, Luc mRNA-LNP, single DFI factor mRNA LNP, or NaCl-treated animals. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Transient hepatic reconstitution of trophic factors enhances aged immunity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/scientists-decry-trumps-plan-to-kill-crucial-atmospheric-science-center/'>Scientists Denounce Trump's Plan to Kill Crucial Atmospheric Science Center</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 15:22:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists Denounce Trump's Plan to Kill Crucial Atmospheric Science Center Mesa Laboratory at The National Center for Atmospheric Research in Boulder. Scientists who study climate, weather and other earth sciences have decried the move, first reported by USA Today, as “taking a sledgehammer” to an “iconic” center dedicated to studying our planet. “NCAR is quite literally our global mothership,” wrote Katharine Hayhoe, a climate scientist at Texas Tech University and chief scientist of the Nature Conservancy, on Bluesky. The center, based in Boulder, Colo., has become a target because of what Russ Vought, director of the White House Office of Management and Budget, described as “climate alarmism” in a statement to USA Today. Weather modeling and supercomputing work also done at the center will be moved to other research hubs, Vought said. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The work to dismantle NCAR, founded in 1960, will begin immediately, USA Today reported, and will involve fully closing the center's iconic Mesa Laboratory. The NCAR dismantling is just the latest target of a series of swings the Trump administration has taken to curtail climate change research and action, including massive cuts to the National Oceanic and Atmospheric Administration and efforts to roll back Environmental Protection Agency rules to limit pollution. NCAR employs about 830 people under the University Corporation for Atmospheric Research (UCAR), a nonprofit consortium of universities and colleges. The National Science Foundation contracts with UCAR to manage NCAR and provided about half of the center's operating budget for 2025, USA Today reported. Andrea Thompson is senior desk editor for life science at Scientific American, covering the environment, energy and earth sciences. She has been covering these issues for nearly two decades. Prior to joining Scientific American, she was a senior writer covering climate science at Climate Central and a reporter and editor at Live Science, where she primarily covered earth science and the environment. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. in atmospheric chemistry from the Georgia Institute of Technology. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251217082457.htm'>He ate a hamburger and died hours later. Doctors found a shocking cause</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 14:32:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The case involved a 47 year old man from New Jersey who was previously healthy and died suddenly about four hours after eating beef. For months, the cause of his death remained unclear. Platts Mills originally identified the condition years ago and continues to lead research into how it affects patients. How Lone Star Tick Bites Trigger Alpha Gal Allergy Once sensitized, people may experience allergic symptoms after eating foods such as beef, pork, or lamb. Scientists have long suspected that severe cases could escalate into life threatening anaphylaxis. Until now, however, no fatal cases had been officially confirmed. "The important information for the public is: First, that severe abdominal pain occurring 3 to 5 hours after eating beef, pork or lamb should be investigated as a possible episode of anaphylaxis; and, second, that tick bites that itch for more than a week or larvae of ticks often called 'chiggers' can induce or increase sensitization to mammalian-derived meat," said Platts Mills, former chief of UVA Health's Division of Asthma, Allergy and Clinical Immunology. "On the other hand, most individuals who have mild to moderate episodes of hives can control symptoms with an appropriate diet." He began feeling unwell shortly after 7 p.m. At 7:37 p.m., his son found him collapsed in the bathroom. An autopsy did not reveal a clear explanation for his death. His wife was not satisfied with that conclusion and asked another physician to review the findings. That doctor contacted Platts Mills and his team to explore whether alpha gal sensitivity could be involved. Platts Mills obtained blood samples that had been collected after the man's death. Testing showed that he had indeed been sensitized to alpha gal. The results also pointed to an extreme immune response consistent with fatal anaphylaxis. However, she recalled that he had suffered 12 or 13 itchy bites around his ankles earlier in the summer, which they believed were caused by chiggers. Platts Mills recognized that many bites thought to be from chiggers in the eastern United States are actually from Lone Star tick larvae. Platts Mills and his colleagues believe several conditions may have intensified the man's allergic response. These include drinking a beer with the hamburger, exposure to ragweed pollen, and physical exercise earlier that day. Family members also noted that he rarely ate red meat, which may have influenced how his immune system reacted. Following the case, Platts Mills is urging doctors to stay alert for patients who may have developed this allergy or who face higher exposure risk. He pointed out that deer populations are rapidly increasing in many states, creating ideal conditions for the Lone Star tick to spread. "More specifically, if they have unexpected episodes of severe abdominal pain occurring several hours after eating mammalian meat, they should be investigated for possible sensitization to the oligosaccharide alpha-gal." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69700405/incomplete-roman-city-switzerland/'>Archaeologists Found Ancient Roman Artifacts From a City That Never Existed</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A site in Switzerland contained evidence of an administrative center that was part of a bigger plan to build a urban center that never came to fruition. Northern Switzerland, it turns out, was very close to having a major Roman urban center in the late first century C.E. Archaeologists discovered artifacts that show a site near the Limmat River in Gebenstorf was not only a hotbed of Roman trade, but was about to burst onto the scene as a major political administrative urban center. One structure featured a double-aisled portico with columned walkways, another had a hall that dipped underground (as was commonly found in Roman marketplaces, known as forums), and the easternmost structure included courtyards, corridors, entrances, and rooms (a stark contrast from the other two buildings nearby). (They also created a documentary on the site.) This find marks the first discovery of sardines in Roman-era Switzerland. The archaeologist said that Romans were among the first societies to use fish resources extensively and establish large salt and preservation facilities for fish sauce production. In these sites, small fish were fermented, and the sauce was packed into amphorae for trade throughout the empire. They now believe it was likely produced on the coast of the Roman province of Baetica, present-day Andalusia (an area within Spain), but present-day Lyon, France, also remains a possible origin point. The Limmat River site was well on its way. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. China's Regreening Has Messed Up Its Water Cycle Archaeologists Found Artifacts In the Great Wall Life's Common Ancestor Lived 4.2 Billion Years Ago Animals Keep Evolving Into Crabs, Which Is Weird A Volcano May Have Started the Black Plague</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02157-0'>Phosphorylated tau exhibits antimicrobial activity capable of neutralizing herpes simplex virus 1 infectivity in human neurons</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 10:09:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Tau is a microtubule-associated cytoskeletal protein, which, when hyperphosphorylated and aggregated, can result in a myriad of different tauopathies, including Alzheimer's disease (AD). We previously showed that the principal component of senile plaques, amyloid beta (Aβ), is an antimicrobial peptide capable of binding and entrapping microbial pathogens. Here we show that tau is hyperphosphorylated in neurons in response to viral infection and can neutralize herpes simplex virus 1 (HSV-1) infectivity by directly binding to viral capsids. Our data suggest that the ‘pathogenic' characteristics of tau hyperphosphorylation, microtubule destabilization and aggregation are part of an antiviral response, in which tau serves as a host defense protein in the innate immune system of the brain. The combined antimicrobial activities of Aβ and phosphorylated tau resulting in Aβ plaques and neurofibrillary tangles, along with neuroinflammation, suggest that AD neuropathology may have evolved as an orchestrated innate immune host defense response to microbial infection in the brain. Amyloid beta (Aβ) deposited in senile plaques and hyperphosphorylated tau in neurofibrillary tangles (NFTs) have long been considered to play pathological roles in the etiology and pathogenesis of AD1. Tau is a microtubule-associated protein that promotes and coordinates tubulin polymerization into microtubules in neurons2. Microtubules are essential for axonal transport between the neuronal synapse and soma. Six different isoforms of tau exist in the human brain, generated by alternative splicing of the tau gene MAPT (microtubule-associated protein tau). These isoforms vary by containing three or four microtubule-binding domain repeats (3R or 4R, respectively) and zero to two N-terminal inserts (0N, 1N and 2N), with approximately equal representation of 3R and 4R tau isoforms in adult human brain3. Detachment of tau from microtubules, hyperphosphorylation and phosphorylated tau (p-tau) aggregation into NFTs are the defining pathological elements of tauopathies, including AD. Various types of tauopathy are differentiated by the isoform of tau that is prone to pathological aggregation and by the individual driving factors that cause tau to dissociate from microtubules—for example, genetic mutations, increased tau expression, decreased tau clearance and increased kinase activity4. Although these characteristics are primarily associated with the pathogenic nature of tau, they also share commonalities with antimicrobial peptides (AMPs)5,6,7. The innate immune system is an evolutionarily conserved, broad-spectrum host defense system comprising epitope-recognizing receptors, small microbial-binding peptides called AMPs and phagocytic cells—for example, myeloid cells. AMPs are crucial in the innate immune system's initial response to an infection, in which AMPs directly bind to, entrap and neutralize invading pathogens and occasionally function as a modulator of inflammatory response8,9. We and others previously demonstrated that Aβ is an AMP capable of entrapping, killing or neutralizing bacterial, fungal and viral infections in both in vitro and in vivo model systems10,11,12,13. The antimicrobial protection hypothesis of AD is based on these results, stipulating that Aβ is not an errant catalytic by-product of the Aβ precursor protein (APP), but evolved as a physiologically relevant antimicrobial peptide as part of the innate immune system, particularly in the brain14. Tau possesses many well-established characteristics and properties that mirror those of Aβ, raising the prospect that tau may also have a role as an AMP. Tau phosphorylation in AD is induced by Aβ amyloidosis, suggesting that it may play a tertiary role in microbial host defense, especially for pathogens that escape the AMP activity of Aβ15,16,17,18. This is most evident in the ability of Herpes simplex virus 1 (HSV-1) to bypass extracellular immune networks by retrograde axonal transport along the microtubules, which are stabilized by tau19,20. Aggregates of hyperphosphorylated tau derived from destabilized microtubules can form NFTs, which share aggregative properties with Aβ in AD pathogenesis. Previous studies identified multiple pathogenic infections that perpetuate total or p-tau increases in vitro21,22,23,24,25. Other recent studies demonstrated that the active binding sites of tau possess antibacterial capabilities26. Similar to our argument for a physiological role of Aβ, p-tau may, likewise, not simply be an errant form of tau but also play a physiological role akin to that of an AMP. Here we report that tau phosphorylation increases in response to HSV-1 infection and that p-tau localizes to cellular regions of HSV-1 infection. Moreover, we demonstrate that tau binds to HSV-1 and neutralizes infectivity. These data support a role for tau as an antiviral AMP and suggest a physiological role for aggregation of p-tau. The AMP activities of Aβ and p-tau leading to Aβ plaques and NFTs, together with neuroinflammation, suggest the possibility that AD neuropathology evolved as a multilayered and orchestrated host defense response to microbial infection in the brain. To test for potential antiviral properties of tau, we first pretreated human ReNcell VM neuronal cell cultures (neuronal cultures) with p-tau followed by infection with HSV-1 and assessed the response of the infection to tau. Mature differentiated two-dimensional (2D) neuronal cultures were preincubated for 24 hours with synthetic 2N4R GSK-3β p-tau (synthetic p-tau) prior to infection with an mRFP-tagged HSV-1. We then employed confocal microscopy to assay single-cell infection counts as well as viral plaque counts and growth (Fig. 2D ReNcell VM cultures were preincubated with dilutions of synthetic 2N3R tau, 2N4R tau or 2N4R GSK-3β p-tau followed by HSV-1 infection to assay the antimicrobial protective properties of tau. After HSV-1 infection, whole wells were imaged by confocal microscopy and analyzed for red fluorescence. a, Images comparing HSV-1 plaque formation in the absence or presence of 2N4R GSK-3β p-tau (1.25 µg ml−1). b−f, Whole-well images from the 2N4R GSK-3β p-tau (scale of 2.5 µg ml−1 to 0.625 µg ml−1) pretreated conditions were analyzed using Nikon Elements GA3 to compare the number of HSV-1 plaques (F3, 54 = 7.069, **P = 0.0015) (b), the number of HSV-1 single-cell infections (F3, 54 = 2.754, *P = 0.0291) (c), the ratio of plaques to single-cell infections (F3, 54 = 6.428, *P = 0.0169) (d), the size of individual HSV-1 plaques by area (*P = 0.0260) (e), and the average size of HSV-1 plaques by area distributed into quadrants (F3, 1,376 = 1.859, **P = 0.0022) (f). In addition, whole well images from the 2N3R and 2N4R tau conditions (1.25 µg ml−1) were analyzed using Nikon Elements General Analysis 3 (GA3) to compare the number of HSV-1 plaques (g) and the number of HSV-1 single-cell infections (h). (n = 11) depicting median and interquartile range, with whiskers denoting variability according to Tukey's method. Statistical mean comparisons were calculated by one-way ANOVA using Dunnett's multiple comparisons test (b−d), one-way ANOVA using Tukey's multiple comparisons test (g,h), two-tailed Kolmogorov−Smirnov test (e) for plaque size distribution and two-way ANOVA using Sidak's multiple comparisons test (f). Preincubation with p-tau significantly attenuated both HSV-1 single-cell infection (P < 0.05) and HSV-1 plaque counts (P < 0.01) compared to untreated controls (Fig. P-tau protection against HSV-1 infection was concentration dependent, with statistical significance achieved at 1.25 μg ml−1 (Fig. Pretreatment with p-tau also significantly reduced HSV-1 plaque counts based on the numbers of plaques per single-cell infection (P < 0.05; Fig. Specifically, at 1.25 μg ml−1 p-tau, the highest quartile of viral plaque sizes was significantly smaller than untreated controls (P < 0.01; Fig. Non-phosphorylated tau isoforms had no effect on HSV-1 single-cell infections or plaque counts (Fig. HSV-1 is an enveloped virus that merges its lipid bilayer with the host cell during infection, allowing the capsid to gain entry. For this purpose, we assayed different isoforms of tau for binding efficacy to isolated HSV-1 capsid using a modified binding ELISA as described in the Methods. Although human tau exists in six isoforms, we focused on the two most prominent forms in the central nervous system: 2N4R and 2N3R. 2N4R is the wild-type or full-length form of tau and aggregates efficiently at near physiological conditions. Additionally, full-length tau (441 amino acids) preserves the most phosphorylation sites. In a healthy adult brain, equal quantities of 3R and 4R tau isoforms are present. We chose 2N3R to represent our 3R isoform for comparison to 4R, thereby preserving the same N-terminal repeat number to isolate the comparison between the microtubule-binding domain repeats. These microtubule-binding domain repeats were previously identified as containing potential antimicrobial sequences26,27. We observed significantly increased binding affinity of p-tau with HSV-1 capsids compared to whole enveloped HSV-1 virions when normalized for available capsid binding targets using anticapsid antibodies VP21/VP22a (P < 0.0001; Fig. Similarly, we found increased tau capsid binding affinity over whole virion for all tested isoforms of tau once normalized for binding target availability, based on anticapsid antibody binding (P < 0.0001; Extended Data Fig. These findings suggest the possibility of intracellular binding of p-tau to HSV-1 capsid after entry of the virus into the cell. Dilutions of synthetic tau isoforms were incubated with heat-immobilized HSV-1 capsid or whole virion in indirect and competitive ELISAs to measure tau−virus binding affinity. a, 2N4R GSK-3β p-tau binding between HSV-1 isolated capsid or whole virion was compared after normalization of available binding sites assessed using VP21/VP22a antibody (****P < 0.0001). b, Dilutions of 2N4R GSK-3β p-tau (scale of 5 µg ml−1 to 0.3125 µg ml−1) were incubated with immobilized HSV-1 capsids (F4, 55 = 43.18, ****P < 0.0001). c, The binding affinity of 2N4R GSK-3β p-tau to HSV-1 capsids was compared to 2N3R, 50/50 mix of 2N3R/2N4R and 2N4R tau (F3, 44 = 8.996, *P = 0.267 ((2N4R GSK-3β p-tau versus 2N3R tau, ***P = 0.0001), (2N4R GSK-3β p-tau versus 2N3R/2N4R tau, ***P = 0.0007)). d,e, Tau adhesion inhibition was assessed by preincubation of immobilized HSV-1 capsids with an anti-VP21/VP22a antibody (F8, 32 = 7.071, ****P < 0.0001) (d) or an anti-pUL48-VP16 antibody (F8, 32 = 3.978, ***P = 0.004, ****P < 0.0001) (e) before exposure to 2N4R GSK-3β p-tau. f, Tau adhesion inhibition was repeated with the anti-VP21/VP22a antibody using a mannose-incubated 2N4R GSK-3β p-tau. (n = 12) depicting median and interquartile range, with whiskers denoting variability according to Tukey's method. Statistical significance was calculated by two-tailed Mann−Whitney test (a) and one-way ANOVA using Tukey's multiple comparisons test (b−f). Next, we tested various isoforms of tau for concentration-dependent binding to HSV-1 capsid (P < 0.001, Fig. Previous studies showed that increased binding repeats and phosphorylation enhance tau aggregation28,29. 2N4R p-tau, which contains the most microtubule-binding domain repeats, exhibited the highest binding efficiency to HSV-1 compared to both unphosphorylated tau isoforms tested, 2N3R (P < 0.001), 2N3R/2N4R 50/50 mix (P < 0.001) and 2N4R (P < 0.05) (Fig. As a control, the scrambled version of the AMP LL-37 was tested versus p-tau and failed to significantly bind viral capsid (P < 0.05; Extended Data Fig. We next investigated potential binding sites for tau on HSV-1 capsids. Immobilized HSV-1 capsid was pretreated with antibodies targeting the viral capsid scaffolding protein (VP21/VP22a), tegument proteins (VP16 and ICP4) or capsid proteins (ICP5 and UL25) prior to treatment with tau. Anti-VP21/VP22a and VP16 antibodies significantly decreased tau binding (P < 0.0001; Fig. 2d,e), whereas ICP4 antibodies had reduced inhibition (P = 0.0056; Extended Data Fig. To assess the importance of glycosylation of the target binding site, we preincubated p-tau with the monosaccharide mannose; p-tau failed to bind effectively to HSV-1, suggesting a glycoprotein binding mechanism similar to that of Aβ (Fig. Collectively, these data support antiherpetic activities of tau, in which tau preferentially targets intracellular HSV-1 by binding to the viral capsid. AMPs can form net-like structures trapping microbes and destabilizing membranes by targeting surface proteins and self-aggregation. We previously identified the ability of Aβ to bind surface glycoproteins on bacteria and viruses, subsequently stimulating the formation of amyloid fibrils30,31,32,33. Using synthetic 2N4R p-tau, we examined whether HSV-1 capsids are able to seed tau fibril formation. The interaction of HSV-1 capsid with synthetic tau was imaged by transmission electron microscopy (TEM) 24 hours after co-incubation. In a mixture consisting only of synthetic tau and purified HSV-1 capsids, we observed examples of both amorphous aggregated tau and fibril tau, serving to agglutinate multiple HSV-1 capsids together (Fig. Expanding on these results, we examined HSV-1-infected neuronal cultures by TEM analysis of ultramicrotomy slices. Tau was previously observed in the nucleus, nucleolus and pericentromeric heterochromatin34,35,36,37. Using the anti-p-tau antibody (PHF1) labeled with immunogold nanoparticles to identify phosphorylated tau, we observed the association of tau with the surface of HSV-1 capsids in cellular nuclei (Fig. Co-incubated isolated HSV-1 capsids and 2N4R GSK-3β p-tau were analyzed by TEM. a,b, Co-incubated HSV-1 capsid (arrows) and 2N4R GSK-3β p-tau displayed amorphous tau aggregates (triangles) and fibrillar tau structures (*) emanating from viral capsid surfaces. c, 3D ReNcell VM cultures infected with HSV-1 were sliced and probed with anti-p-tau-Au nanoparticles. d, Inside the cell nucleus, HSV-1 capsids were identified with anti-p-tau-Au binding on their surface (arrows). Micrographs are representative of data from multiple discrete imaging fields (n = 3). HSV-1 infection of murine neurons or human neuroblastoma cell cultures was previously reported to increase hyperphosphorylated tau, interpreted as an abnormal by-product of viral infection21,22. To examine the response of tau to HSV-1 infection in a neuronal environment, we infected 3D neuronal cultures and used PHF1 to assay for intraneuronal aggregation of p-tau in dystrophic neurites and cell bodies. We examined p-tau-stained cells 24 hours after infection using confocal microscopy. Consistent with HSV-1-driven aggregation of p-tau, we observed significant increases in p-tau-positive dystrophic neurites (P < 0.01) and neuronal soma (P < 0.001) compared to uninfected neuronal cultures (Fig. Co-localization of p-tau and the β-sheet staining dye, thioflavin S, confirmed the presence of aggregated p-tau during infection (Fig. In addition to p-tau aggregation, we observed increased anti-total tau fluorescent signal intensity and punctate distribution inside infected cells (Extended Data Fig. 3D ReNcell VM cultures were infected with HSV-1 to characterize tau's change in cellular distribution and levels in response to viral infection. As shown in a, 3.5−4-week-old 3D ReNcell VM cultures were infected with a replication-deficient HSV-1 (deletion of UL28 protein) for 24 hours, immunoprobed with anti-p-tau (PHF1) and anti-HSV1 labeled with a fluorescent secondary antibody and analyzed for neurons (GFP), HSV-1 (568) and p-tau (647) fluorescence by confocal microscopy. b,c, Fluorescent image captures from 384 wells over four experiments were compared by GA3 software in Nikon Elements for p-tau fluorescent neurites (triangles) (***P = 0.0002) (b) and cell bodies (arrows) (*P = 0.0022) (c) between uninfected and infected wells. d, Confocal images showing 3D ReNcell VM cultures that were infected with HSV-1 for 24 hours, immunoprobed with anti-p-tau (PHF1) labeled with a fluorescent secondary antibody and immunostained with thioflavin S. e, Pearson's correlation coefficient (error bars represent 95% confidence interval) was derived from analyzing uninfected and infected wells for co-localization of thioflavin S (green), HSV-1 (RFP) and p-tau (647) fluorescence. f−k, 3.5−4-week-old 3D ReNcell VM cultures were infected with serial dilutions of HSV-1 for 24 hours, and cell lysates were separated into soluble and insoluble tau fractions for analysis by MSD Multi-Spot Phospho (Thr 231)/Total Tau Assay. g, Comparison of cellular insoluble p-tau at different viral loads. h, Comparison of the ratio between cellular insoluble and soluble p-tau at different viral loads (F3,96 = 26.54, ****P < 0.0001). j, Comparison of cellular insoluble total tau at different viral loads (F3, 89 = 6.810, **P = 0.0049, ***P = 0.0002). k, Comparison of the ratio between cellular insoluble and soluble total tau at different viral loads (F3, 89 = 7.469, **P = 0.0057, ****P < 0.0001). ((f−h, n = 25), (i−k, n = 19)) depicting median and interquartile range, with whiskers denoting variability according to Tukey's method. Statistical mean comparisons were calculated by two-tailed unpaired t-tests (b,c) and one-way ANOVA using Dunnett's multiple comparisons test (f−k). Next, we used ELISA using Meso Scale Discovery (MSD) to validate the aggregation of p-tau and total tau in neuronal cultures infected with HSV-1. We observed a significant dose-dependent increase in the insoluble-to-soluble p-tau ratio and total tau after HSV-1 infection (P < 0.0001; Fig. This increase in p-tau was consistent with our immunofluorescence analysis, suggesting that the aggregated p-tau observed in dystrophic neurites and cell bodies is insoluble p-tau. Variability in p-tau distribution was observed via immunofluorescence analysis between PHF1 and Thr231 (antibody used in MSD) (Extended Data Fig. Compared to uninfected cells, we did not observe any cytotoxicity for cell death as assayed by lactate dehydrogenase (LDH) assay at any HSV-1 infection concentrations (Extended Data Fig. After HSV-1 infection, we observed a significant decrease in levels of intracellular soluble p-tau (−48.8%, P < 0.0001; Fig. In the absence of significant cell death (Extended Data Fig. 7), this decrease suggested a release or expulsion of p-tau after HSV-1 infection. To examine the movement of p-tau after HSV-1 infection, we measured and compared p-tau and total tau levels between culture media and cell lysates from infected and uninfected neuronal cultures. We observed significant increases in the p-tau to total tau ratios in the media (141.8%, P < 0.0001) (Fig. 5a–c), whereas there was a simultaneous, significant decrease in the soluble p-tau to total tau ratio in the cell lysates (−54.6%, P < 0.0001) (Fig. To examine whether increased extracellular p-tau influences surrounding cells as an AMP, we assessed neuronal cultures in a microfluidic device. These devices were constructed with microchannels connecting enclosed compartments on either side. Immunofluorescent analysis of HSV-1 (mRFP) and p-tau (anti-p-tau tagged with fluorescent secondary) revealed a significant increase in p-tau fluorescence intensity inside uninfected cells in proximity to HSV-1-infected cells when compared to uninfected cells not in proximity to the infection (26.9%, P < 0.001; Fig. We also noted a strong correlation between p-tau intensity and the viral load in proximity to the uninfected cells (Fig. Interestingly, the uninfected cells in close proximity to infection revealed no changes in total tau levels (Extended Data Fig. When comparing viral load to total tau intensity in uninfected cells residing proximally to infected cells, unlike p-tau, there was no correlation with total tau levels. This finding further reinforces the observed response of p-tau specific to viral infection (Extended Data Fig. These results are consistent with the notion that preincubation with p-tau provided protection from HSV-1 infection and reduced viral plaque growth (Fig. These data suggest that extracellular p-tau may stimulate the generation of p-tau in adjacent cells or promote endocytosis of p-tau released into the media, thereby providing increased protection from HSV-1 infection in close proximity38. ReNcell VM cultures were infected with HSV-1 for 24 hours to characterize p-tau's extracellular release and changes in proximity to viral infection. a−d, 3.5−4-week-old 3D ReNcell VM cultures were infected with serial dilutions of HSV-1 for 24 hours, and cell media and cell lysates were analyzed by MSD Multi-Spot Phospho (Thr 231)/Total Tau Assay for soluble p-tau and total tau. a, Comparison of cell media soluble p-tau at different viral loads (F3, 54 = 7.521 ((uninfected versus 2.55, ***P = 0.0005), (uninfected versus 5.10, ***P = 0.0004)). b, Comparison of cell media soluble total tau at different viral loads (F3, 55 = 10.49, **P = 0.0011). c, Comparison of the ratio between cell media p-tau and total tau at different viral loads (F3, 54 = 19.94, ****P < 0.0001). d, Comparison of the ratio between cell lysate soluble p-tau and soluble total tau at different viral loads (F3, 96 = 202.5, ****P < 0.0001). e, 2D ReNcell VM cultures in microfluidic devices were infected in the left chamber with HSV-1 for 48 hours, immunoprobed with anti-p-tau (PHF1) labeled with a fluorescent secondary antibody and analyzed for neurons (GFP), HSV-1 (RFP) and p-tau (647) fluorescence by confocal microscopy. Fluorescence signals for ReNcell VM, HSV-1 and p-tau were imaged for infected and uninfected conditions. g, Individual neuronal p-tau fluorescence intensity was compared to total proximal HSV-1−RFP fluorescence intensity (R2 = 0.7912). ((a−c, n = 13), (d, n = 25)) depicting median and interquartile range, with whiskers denoting variability according to Tukey's method. Statistical mean comparisons were calculated by one-way ANOVA using Dunnett's multiple comparisons test (a−d), one-way ANOVA using Tukey's multiple comparisons test (f) and simple linear regression (g). The increased presence of p-tau in uninfected cells proximal to infection sites (Fig. 5a–d) suggest that p-tau may be involved in cell-to-cell transfer as part of its antimicrobial action. To test this mechanism, we utilized a novel two-culture system wherein we seeded a ReNcell VM culture in a well insert with a 0.1-μm filter at the bottom and incubated it with azide-labeled amino acids. Once mature, we infected the culture with HSV-1 and subsequently transferred the insert to a well with mature uninfected 3D ReNcell VM cultures. The 0.1-μm pore size inhibited mature virus from transferring between cultures but allowed azide-labeled protein transfer. In this manner, we were able to interrogate protein migration in infected cell cultures. We observed internalization of azide-positive peptides in the uninfected cultures after the addition of the well insert, but no transfer of HSV-1 or viral infection (Fig. These azide-positive peptides were confirmed to be p-tau based on significant co-localization with the p-tau antibody PHF1 (Fig. Consistent with our findings of increased p-tau in uninfected cells proximal to infection (Fig. 5), we also observed a significant increase in p-tau intensity in cells that were positive for exogenous p-tau (P < 0.0001; Fig. ReNcell VM cultures were infected with HSV-1 to determine the degree of immune response. b, Distribution plot of individual AHA and p-tau pixel intensities after infection. c, Pearson's correlation coefficient (error bars represent 95% confidence interval) of images of internalized AHA and PHF1 in ReNcell VM cells. d, Fluorescent image captures from nine wells over three experiments were compared by GA3 software in Nikon Elements for p-tau fluorescence (****P < 0.0001) between azide-positive and azide-negative cells. e, One-week-old 2D ReNcell VM cultures were preincubated with dilutions of a GSK-3β inhibitor (scale of 50 nM to 25 nM) for 24 hours followed by a 24-hour HSV-1 infection and imaged by confocal microscopy for red fluorescence. Whole-well images were analyzed using Nikon Elements GA3 to compare the number of HSV-1 plaques between conditions (F3, 53 = 2.806, *P = 0.0268). f,h, Conditioned media collected from 1-week-old ReNcell VM uninfected (f) and 1-hour HSV-1 infected (h) (F2, 2 = 23.85, P** = 0.0025) cultures pretreated with 2N4R GSK-3β p-tau (1.25 µg ml−1) were run on a proinflammatory cytokine MSD. g, One-week-old 2D ReNcell VM cultures were preincubated with 2N4R GSK-3β p-tau (1.25 µg ml−1) and anti-IFNγ antibody (10 µg ml−1) for 24 hours followed by a 24-hour HSV-1 infection and imaged by confocal microscopy for red fluorescence. Whole-well images were analyzed using Nikon Elements GA3 to compare the number of HSV-1 plaques between conditions (F3, 56 = 4.863, *P = 0.0386). Statistical mean comparisons were calculated by two-tailed unpaired Welch's t-test (d), one-way ANOVA using Dunnett's multiple comparisons test (e), two-way ANOVA using Tukey's multiple comparisons test (f), one-way ANOVA using Tukey's multiple comparisons test (g) and two-tailed unpaired t-test (h). Next, we examined potential pathways responsible for promoting the increase in p-tau levels. To assess whether tau phosphorylation is necessary for the observed antimicrobial action, we infected neuronal cultures in the presence of GSK-3β kinase inhibitors. As a result, we observed a significant dose-dependent increase in viral plaque counts, cellular infections and plaque sizes with increased inhibitor (Fig. Next, we tested whether inflammation via interferon-γ (IFNγ) signaling pathway drives increases in p-tau and associated antiviral protection against HSV-1 infection versus the addition of exogenous p-tau. Under non-infectious conditions, the addition of exogenous p-tau did not elicit an inflammatory reaction as assessed by the measurement of 10 different proinflammatory cytokine levels (Fig. Similarly, attenuation of IFNγ by anti-IFNγ antibodies after HSV-1 infection did not exacerbate HSV-1 infection (Fig. Interestingly, anti-IFNγ antibody treatment neutralized the protective effects of synthetic p-tau pretreatment, indicating that an IFNγ-mediated pathway is important for the antimicrobial action of p-tau (Fig. In conjunction with our findings of p-tau aggregation and spread in the presence of infection (Figs. 4 and 5), these data provide compelling evidence for the antiviral activity of p-tau by both protecting against and modulating an innate immune response to HSV-1. Microglia are essential to the central nervous system (CNS) response to HSV-1 infection by identifying viral pathogen-associated molecular patterns (PAMPs), releasing inflammatory cytokines and clearing virus and damaged neurons. Interestingly, microglia were recently suggested to contribute to pathogenic tau spread due to endocytosis of tau aggregates and subsequent failure to fully digest the aggregates, followed by secretion of exosomes containing ‘tau seeds'39. To investigate the interaction among tau, HSV-1 and microglia, we seeded a neuronal/astrocytic 3D ReNcell VM culture with microglia derived from induced pluripotent stem cells and then infected the neural−glial tricultures with HSV-1. Twenty-four hours after infection, the cultures were fixed and prepared for immunocytochemistry (ICC) with anti-IBA1 and anti-p-tau (PHF1) antibodies. Co-localization of p-tau and HSV-1 was observed inside IBA1-positive microglial cells, which was not present in uninfected controls (Fig. Although more experimentation is necessary to determine whether the p-tau/HSV-1 is being phagocytosed as an independent complex or as part of infected neurons, these results warrant more detailed studies addressing microglia internalizing p-tau/HSV-1 complexes. Mature microglia were added to 3D ReNcell VM cultures prior to infection with HSV-1 to analyze the interaction among p-tau, HSV-1 and microglia. Uninfected and infected microglia−ReNcell VM cultures were immunoprobed with anti-p-tau (PHF1) and anti-IBA1 labeled with fluorescent secondary antibodies and analyzed for microglia (405), neurons (GFP), HSV-1 (RFP) and p-tau (647) fluorescence by confocal microscopy. Co-localization of HSV-1 (red) and p-tau (purple) inside of microglia (blue) was observed (arrows). Panels are representative of multiple image fields (n = 5). Evidence for a microbial component to the etiology and pathogenesis of AD has continued to grow over the last decade. Although a consensus on particular pathogens has still not been reached, animal studies, retrospective cohort studies for infection and AD risk and microbiome screens have all identified the epidemiological prevalence of microbes in dementia and pathogen-driven shifts in locations such as the gut microbiota that may promote AD-related neuropathology40,41,42,43. Previously, we reported antimicrobial properties of Aβ based on its ability to neutralize Salmonella, HSV-1 and human herpesvirus 6 (HHV-6) (refs. These studies led to the Antimicrobial Protection Hypothesis of AD, postulating that Aβ has evolved to protect the brain against infection, with amyloid plaques serving as extracellular traps for microbes. As a corollary to this idea, the gene mutations that predispose to AD pathology, today, may have been conserved in evolution owing to their ability to enhance innate immunity in the brain14. In the present study, we reveal a notable novel role for tau as an antiviral protein against HSV-1, based on affording increased protection from infection (Fig. We also observed an infection-based response with the phosphorylation of tau and the resulting p-tau shifting to an insoluble form (Fig. 4) and increased p-tau presence in uninfected cells in proximity to the location of infection (Fig. The sequestration of p-tau into insoluble forms enhances innate immunity by binding HSV-1 (Fig. 3), destabilizing microtubules, breaking down fast axonal transport and generating NFTs, all of which would serve to hamper neurotropic spread of virus across synapses from one neuron to another. These findings are consistent with the oligomerization of AMPs as an essential characteristic for their innate immune functions49,50. Previous research on antimicrobial properties of tau by Kobayashi et al.26 reported that tau peptide fragments of the microtubule-binding site possessed strong antimicrobial activities against Staphylococcus aureus and Escherichia coli. Although it was hypothesized that this binding activity was driven by negatively charged surfaces of Gram-negative bacteria, we observed preferential binding to viral capsid over viral envelope (Fig. 2b), despite the strong negative charge of the envelope from post-translational additions of sialic acid to the glycoproteins on the envelope surface, suggesting a more specific targeting beyond peptide charge51. A subsequent study by Powell-Doherty et al.52 examined Aβ and p-tau as components of an anti-threat response to HSV-1, demonstrating viral-induced phosphorylation of tau and accumulation of Aβ oligomers. They propose that p-tau does not inhibit viral replication and may alternatively be a danger-associated molecular pattern (DAMP). By contrast, our data support the direct role of p-tau in viral inhibition and binding (Figs. Utilizing antibodies against different viral and capsid peptides for steric interference of p-tau binding, we observed targeted binding of p-tau to a region occupied by inner tegument proteins. Blocking major capsid protein ICP5 or capsid stabilizing protein UL25 failed to inhibit p-tau binding entirely (Extended Data Fig. 4a,c), whereas blocking tegument-associated protein ICP4 led to modest inhibition (Extended Data Fig. Conversely, antibodies against tegument proteins VP21/22a and VP16 led to significant dose-dependent inhibition of p-tau binding to viral capsid (Fig. Notably, inner tegument protein VP1/VP2 binds VP16, VP16 binds VP22 and all three of these viral proteins are essential for the bidirectional transport of HSV-1 capsids along microtubules53,54. These data suggest a dual-action antiviral effect in which hyperphosphorylation dissociates tau from microtubules, destabilizing them and interrupting viral capsid transportation, and p-tau simultaneously binds to capsid/tegument proteins essential for viral microtubule transport. Although our data supporting antimicrobial properties of tau and p-tau are novel, evidence of tau modifications in response to infections was previously reported. In adult hippocampal neurons, p-tau levels increased after active and latent HSV-1 infection. Additionally, gingipains, exotoxins from Porphyromonas gingivalis, were reported to cleave tau, and the resulting fragments displayed antimicrobial properties27. Microtubules are essential for fast axonal transport in neurons but are also utilized by intracellular pathogens. Both viruses and bacteria utilize microtubule networks, and stabilization of the microtubules results in increased infection57,58. The protective effect of tau against HSV-1 infection (Fig. 1) and the increased ratio of insoluble versus soluble p-tau during infection (Fig. 4h,k) suggest that destabilization of microtubules by p-tau protects against microbial utilization of the cytoskeleton to spread. This is further supported by neuritic dystrophy observed where HSV-1 was isolated with p-tau (Extended Data Fig. 10a) and p-tau aggregation in neuronal somas around HSV-1 replication in the nucleus (Extended Data Fig. 10b), in effect preventing HSV-1 access to microtubule transport required for propagation. We also observed increased viral binding with both increased R domain repeats and phosphorylation of tau, with the synthetic 2N4R p-tau being the most effective (Fig. Once again, these findings suggest that the phosphorylation and increased aggregation of tau, classically observed as a pathological feature, can be alternatively viewed as a functional feature of tau designed to bind and neutralize microbial targets more effectively. Indeed, our data support the notion that the phosphorylation of tau is not just sufficient but necessary for antimicrobial activity. Non-phosphorylated tau isoforms failed to elicit any protection against HSV-1 infection (Fig. 1), and the presence of GSK-3β inhibitor during viral infection significantly increased the viral spread (Fig. Beyond AMPs, the innate immune system utilizes cytokines for essential communication and regulation of the immune response. One such key cytokine in response to HSV-1 is IFNγ, capable of inhibiting replication, impeding viral release and disrupting microtubule organization59,60,61. To differentiate the viral-driven p-tau antimicrobial response from a generalized viral immune response, we examined the role of cytokines in our model. Utilizing an anti-INFγ antibody to block the HSV-1-induced cytokine response, we observed that the removal of IFNγ during p-tau treatment negated the p-tau antiviral protection (Fig. By contrast, anti-IFNγ antibody treatment without the addition of p-tau had no effect on viral spread (Fig. MSD analysis of cytokines revealed that exogenous p-tau did not elicit any immune response (Fig. 6f), and the presence of synthetic p-tau lowered IFNγ levels released into the media (Fig. Collectively, these data suggest that IFNγ is critical for the antiviral activity of p-tau but not via inflammatory pathways. Future studies should consider the potential relationship between the induction of tight junction protein endocytosis by IFNγ combined with the effects of IFNγ on the antiviral properties of p-tau62. Cell-to-cell transfer and propagation of p-tau is yet another property considered to be pathological. However, our findings suggest that this p-tau propagation may also be considered a physiological antimicrobial feature of tau. The departure of tau from the somatic space is well documented, with evidence of its traversal from cell to cell via excretion and exosomes and through synaptic clefts63,64,65,66,67. The discovery that ‘pathogenic' forms of tau can self-propagate and promote the aggregation of soluble tau led to the suggestion that seeding and propagation of neuropathological tau resembles prion diseases68. Interestingly, a recent publication by Liu et al.69 demonstrated that active human endogenous retrovirus infections accelerated the cell-to-cell spread of tau aggregates. Our data revealing elevated p-tau in uninfected neurons in close proximity to infected cells support and may, therefore, help explain this finding (Fig. Additionally, we show that the ability of tau to protect against viral infection is enhanced by co-incubation of p-tau with neurons prior to HSV-1 infection (Fig. During HSV-1 infection, we also observed a significant increase in soluble p-tau being released into the extracellular space (Fig. These data suggest that ‘pathological' spreading of tau may actually have evolved as an antimicrobial feature, allowing tau to protect adjacent cells prior to the establishment of a spreading viral infection. Microglia, the resident macrophage-like cells in the brain, are an essential component of brain health, playing roles in both development and protection against trauma and pathogenic invasion. Microglia can phagocytose extracellular pathogens such as HSV-1 and bacteria that make it past the blood−brain barrier as well as damaged or necrotic cells70,71. The ability of microglia to take up Aβ and tau, in addition to their release of proinflammatory cytokines and chemokines, have made them a prime therapeutic target in AD. We observed co-localization of p-tau and HSV-1 inside of microglia in viral-infected 3D human neuronal/astrocytic cultures co-incubated with microglia (neural−glial tricultures), suggesting that tau may also play a role in promoting microglial uptake of HSV-1 (Fig. It is important to acknowledge that tau does not fully comply with the attributes of typical AMPs, which are typically cationic, are below 100 amino acids, display broad extracellular antimicrobial efficacy, and often harbor α-helical or β-sheet secondary structures. Although most known AMPs fall within these guidelines, there are exceptions. Although more than four times the typical size of AMP, tau shares many of these commonalities, such as cationic charge and assembly into β-sheets. LL-37 was found to interact with glyceraldehyde 3-phosphate dehydrogenase to immunomodulate macrophage activity72, demonstrating an AMP capable of acting intracellularly on a host cell for immunomodulation. NONO is a nuclear protein that binds to and detects HIV capsid in the nucleus to promote cGAS-mediated innate immune activation and is 471 amino acids long73. Combined with our data, these unique exceptions support the notion that, although tau may not be a prototypical AMP, its physical characteristics and antimicrobial properties warrant further research into its classification as an AMP. The data presented here strongly suggest that tau serves as a second layer of defense against pathogens in the brain. Although Aβ serves as an extracellular AMP to trap viruses in Aβ deposits, the viruses that escape and infect neurons can then be stopped by p-tau and NFTs, induced by either the virus itself or by Aβ oligomers10. In this manner, Aβ and p-tau combine to provide what can be considered a dual AMP host defense mechanism against viral infection. Like Aβ, tau also presents as a highly phylogenetically conserved protein, sharing the microtubule-binding domain with mammals, reptiles, birds and fish74,75. Our findings continue to reinforce the notion that the three major hallmarks of AD pathology—Aβ, NFTs and neuroinflammation—evolved as a multilayered and highly orchestrated host defense system as part of the innate immune system of the brain. GFP-expressing ReNcell VM neuronal cell line G10, previously described76, was seeded as a 3D culture in a 96-well plate (Greiner Bio-One, 655866) in ReN Differentiation Media (ReN Diff). ReN Diff changes were completed twice a week until the plate was 3.5−4 weeks old, at which the neuronal culture is considered mature, fully differentiated and ready for experimentation. The 3D human triculture system including neurons, astrocytes and microglia was previously described77. In brief, induced pluripotent stem cells are differentiated into mature microglia over 38 days. Once mature, the microglia were resuspended and added at a 6:1 ratio to the 3D ReNcell VM culture for 24 hours followed by a 24-hour HSV-1 infection. After infection, the wells were washed once with PBS and then fixed with 4% paraformaldehyde (PFA) for 2 hours at 4 °C. Then, the plate was washed three times with PBS and stored in PBS at 4 °C until ready for immunostaining. Plate was first blocked with Doo Block shaking at 4 °C overnight. The next day, primary antibodies anti-IBA1 (Abcam, ab283346) at 1:1,000 and anti-p-tau (PHF1, Albert Einstein College of Medicine) at 1:500 were added and incubated at 4 °C overnight shaking. The next day, the plate was washed five times at 5-minute intervals with TBS-Tween. Fluorescent secondaries anti-rat 405 (Invitrogen, A48268) and anti-mouse 647 (Invitrogen, A32787) were added at 1:500 for a 3-hour incubation at 4 °C shaking. Afterwards, the wells were washed five times for 5 minutes each with TBS-Tween and then stored in PBS. Plates were imaged by fluorescence confocal microscopy (Nikon, A1R-HD25) for microglia (405), neurons (GFP), HSV-1 (RFP) and p-tau (647). Two-dimensional (2D) ReNcell VM cultures were seeded in a 24-well plate (Corning, 3526) precoated with 1:100 Matrigel (Corning, 354234) diluted in cold DMEM F-12 at 300,000 cells per well in ReN Expansion Media (ReN Exp) overnight. At 1 week old, the media were removed and replaced with ReN Diff containing 2N4R GSK-3β p-tau (Abcam, ab269024), 2N3R tau (Abcam, ab72462) or 2N4R tau (Abcam, ab84700) overnight at 37 °C at the following concentrations: 2.5 μg ml−1, 1.25 μg ml−1 and 0.625 μg ml−1. ReN Diff alone was used for the control wells. The next day, the media were removed, and ReN Diff containing HSV-1 (diluted to 50−200 plaque-forming units (PFU)) were added to each of the wells and centrifuged (Thermo Fisher Scientific, ST8R centrifuge) at 500g for 1 hour at room temperature. The HSV-1 media were removed, and ReN Diff containing 0.36% agarose were added to the wells for an overnight incubation at 37 °C. The next day, the plate was imaged by confocal microscopy (Nikon, A1R-HD25) for viral RFP fluorescent signal. Images were analyzed using Nikon Elements General Analysis 3 (GA3) for viral plaque counts, HSV-1-positive single cells and viral plaque size. ‘Tau HSV-1 plaque assay' methodology was adapted to be performed with combinations of a GSK-3β inhibitor (Tocris Bioscience, 4423) and an anti-INFγ antibody (Invivogen, HIFNG-MAB7-02) in place of the synthetic taus from start to finish including image analysis. One-half percent Triton X-100 was added to 900 μl of TNE buffer and 100 μl of HSV-1 to create a virus mix stored at 4 °C for 5 minutes. TNE buffer with 35% sucrose was added to ultracentrifuge tubes (Beckman Coulter, 344059) before adding virus mix and then ultracentrifuged (Beckman Coulter, Optima XPN-100 Ultracentrifuge) at 37,000g for 45 minutes at 4 °C. After ultracentrifugation, supernatant was aspirated, and the capsid pellet was resuspended in 200 μl of TNE buffer. Capsid mix was added to TNE buffer with 35% sucrose and ultracentrifuged again at 37,000g for 45 minutes at 4 °C. The supernatant was aspirated, and capsid-isolated HSV-1 was resuspended in 200 μl of TNE buffer followed by storage at 4 °C for experimentation. Capsid-isolated HSV-1 or whole virion HSV-1 (5 × 106 PFU per well) in distilled water (50 μl per well) was added to a black polypropylene plate (Greiner Bio-One, 655209) and heat-fixed at 95 °C for 2 hours. After heat-fixing, the plate underwent three TBS washes and then was blocked at room temperature for 1-hour shaking using 4% non-fat dry milk. After blocking, the plate was washed three times with TBS-Tween. After tau incubation, the plate was washed three times with TBS-Tween and then blocked at room temperature for 1-hour shaking using 10% BSA. After blocking, the plate was washed three times with TBS-Tween followed by the addition of an anti-tau antibody (Agilent Technologies, A0024) at 1:10,000 in 10% BSA (100 μl per well) and incubated at 4 °C overnight shaking. The next day, the plate was washed three times with TBS-Tween followed by addition of an anti-rabbit HRP antibody (Cytiva, NA934-100UL) at 1:1,000 in 10% BSA (100 μl per well) and incubated at room temperature for 1-hour shaking. Afterwards, the plate was washed three times with TBS-Tween before adding SuperSignal ELISA Femto Substrate (Thermo Fisher Scientific, 37075). The plate was immediately read on a BioTek Synergy Neo2 Multi-Mode Reader for chemiluminescent signal. ‘Virus binding assay' methodology was adapted to be performed with anticapsid interference antibodies prior to synthetic tau addition. The initial steps up to 4% non-fat milk block and subsequent TBS-Tween washes remain the same. After the washes, anticapsid interference antibodies anti-VP21/VP22a (Thermo Fisher Scientific, MA5-16798), anti-ICP5 (Virusys Corporation, HA018-100), anti-pUL25 (University of Pittsburgh), anti-pUL48-VP16 (Novus Biologicals, NB206593) or anti-RS1-ICP4 (Abcam, ab6514) were added at stepwise dilutions (0.04, 0.2, 1.0 and 5.0 μg ml−1) to appropriate wells and incubated at 37 °C for 1 hour. The plate was then washed three times with TBS-Tween. 2N4R GSK-3β p-tau (Abcam, ab269024), control scrambled LL-37 or 2N4R GSK-3β p-tau incubated with mannose for 1 hour at room temperature was added to wells and incubated at 37 °C for 15 minutes. Then, the plate was washed three times with TBS-Tween followed by a 10% BSA block at room temperature for 1 hour. A subsequent three-times TBS-Tween wash was performed, and a 10% BSA of anti-tau antibody (Agilent Technologies, A0024) at 1:10,000 was added to each well (100 μl per well) for an overnight incubation at 4 °C shaking. The next day, the plate was washed three times with TBS-Tween, and an anti-rabbit HRP antibody (Cytiva, NA934-100UL) at 1:1,000 in 10% BSA (100 μl per well) was added to each well for a 1-hour room temperature incubation shaking. The plate was washed three times with TBS-Tween before adding SuperSignal ELISA Femto Substrate (Thermo Fisher Scientific, 37075). The plate was immediately read on a BioTek Synergy Neo2 Multi-Mode Reader for chemiluminescent signal. Sample containing capsid-isolated HSV-1 (10 μl) and 2N4R GSK-3β p-tau (10 μl) was incubated for 2 hours at room temperature before being added (10 μl) onto Formvar/Carbon Support Square Grids (Electron Microscopy Sciences, FCF100-Cu) for 2 minutes. Sample was removed by gently touching the perimeter of the grid with filter paper. The grid was transferred to water droplets for 30 seconds three times. After washes, the grid was dyed by adding 10 μl of 1% uranyl acetate for 1 minute before removing the dye with filter paper and letting the grid sit at room temperature for 10 minutes to dry. 3D ReNcell VM cultures were seeded in a six-well plate at 8 million cells per well in ReN Diff and were aged for 3.5 weeks. The cells were infected with HSV-1 (4.06 × 107 PFU per well) and centrifuged with minimal acceleration and deacceleration (Thermo Fisher Scientific, ST8R centrifuge) at 500g for 1 hour at room temperature and then incubated at 37 °C for 24 hours. The next day, the media were removed, and the cells were washed once with PBS followed by initial fixation in 4% PFA on a rotator for 30 minutes. Then, fresh 4% PFA was added, and the cells were incubated at 4 °C overnight. The next day, the cells were washed three times using PBS, scraped and spun at 2,200g for 15 minutes. The supernatant was removed, and the pellets were resuspended in PBS to be stored at 4 °C. Pellets were loaded onto grids and stained with anti-p-tau (PHF1) antibody incubated with immunogold particles in accordance with ‘Viral agglutination and TEM analysis' methodology. Electron microscopy was performed in the Microscopy Core of the Center for Systems Biology/Program in Membrane Biology, which is partially supported by an Inflammatory Bowel Disease grant (DK043351) and a Boston Area Diabetes and Endocrinology Research Center (BADERC) award (DK057521). 3D ReNcell VM cultures were seeded in a 96-well plate (Greiner Bio-One, 655866) and aged for 3.5−4 weeks. At maturation, half the media was removed from the wells and replaced with ReN Diff containing HSV-1 (6.67 × 105 PFU per μl) at 0.5 µl, 0.25 µl or 0.1 µl per well for 24 hours at 37 °C. The next day, the plate was washed once with cold PBS, and cold Sarkosyl lysis buffer was then added to the wells for a 1-hour incubation at 4 °C rocking. The cells were then scraped and collected in a 96-well polymerase chain reaction (PCR) plate to be stored at −20 °C. The cell lysate was thawed and spun (Thermo Fisher Scientific, ST8R centrifuge) at 2,129g for 30 minutes to pellet cellular debris. The supernatant was collected and added to an ultracentrifuge tube (Beckman Coulter, 343778) and spun at 279,000g (Beckman Coulter, Optima MAX-XP Ultracentrifuge) for 1 hour to separate insoluble and soluble tau. After ultracentrifugation, the supernatant was collected as soluble tau. The remaining pellet was dissolved using guanine hydrochloride and collected as insoluble tau. Both soluble and insoluble tau were added to a 96-well MSD Multi-Spot Phospho (Thr 231)/Total Tau Assay plate (MSD, K15121D-2) and run according to the manufacturer's instructions. Plates were read on an MSD MESO QuickPlex SQ 120 mm, and the resulting raw data were analyzed using MSD Discovery Workbench. Media for proinflammatory cytokine concentration determination were collected from 24-well plates preincubated with 1.25 µg ml−1 2N4R GSK-3β p-tau (Abcam, ab269024) for 24 hours with or without 1-hour HSV-1 infection the next day. Conditioned media were added to the 96-well V-PLEX Proinflammatory Panel 1 Human Kit (MSD, K15049D) and run according to the manufacturer's instructions. Plates were read on an MSD MESO QuickPlex SQ 120 mm, and the resulting raw data were analyzed using MSD Discovery Workbench. 3D ReNcell VM cultures were seeded in a 96-well plate (Greiner Bio-One, 655866) and aged for 3.5−4 weeks. At maturation, the plate was infected with 30 μl of replication-deficient HSV-1 virus (3.3 × 106 PFU per μl; Heming et al. 2013) mixed with 6 ml of ReN Diff or HSV-1−RFP fusion virus and incubated at 37 °C for 24 hours. The wells were then washed once with PBS and then fixed with 4% PFA at 4 °C for 2 hours. Afterwards, the plate was PBS washed three times followed by blocking with Doo Block at 4 °C shaking overnight. The next day, the following primary antibodies were added for an overnight 4 °C incubation shaking: anti-total tau (Agilent Technologies, A0024) at 1:1,000, anti-p-taus PHF1 (Albert Einstein College of Medicine, gift) or AT180 (Invitrogen, MN1040) at 1:500 and anti-HSV-1 (Invitrogen, PA5-115473) at 1:1,000 only if the replication-deficient HSV-1 was used. The next day, the plate was washed five times at 5-minute intervals using TBS-Tween, and the following secondary antibodies (1:500) were added for a 3-hour 4 °C incubation shaking: anti-rabbit 405 (Jackson ImmunoResearch, 711-475-152), anti-mouse 647 (Invitrogen, A32787) and anti-rabbit 568 (Invitrogen, A10042) if necessary. Afterwards, another five 5-minute TBS-Tween washes were repeated, and the plate was stored in PBS. Plates were imaged by fluorescence confocal microscopy (Nikon, A1R-HD25) and analyzed using Nikon Elements GA378. Polydimethylsiloxane microfluidic devices were designed and produced at the Center for Engineering in Medicine at Massachusetts General Hospital (MGH), as described previously77. The devices were precoated for 3 hours with 1:100 Matrigel (Corning, 354234) diluted in cold DMEM F-12. 2D ReNcell VM cultures were seeded at 300,000 cells for the outside chambers and 150,000 cells for the middle chamber. The devices were aged for 3.5 weeks followed by a 48-hour 0.5 μl per well HSV-1 (6.67 × 105 PFU per μl) infection of the left chamber. After a PBS wash, the devices were fixed with 4% PFA for 2 hours at 4 °C. The devices followed the immunostaining ‘p-tau and total tau immunofluorescence labeling' methodology with a distinction in antibody use. Instead, the antibodies anti-p-tau at 1:500 (PHF1, Albert Einstein College of Medicine) and anti-total tau at 1:750 (GeneTex, GTX49353) fluorescently labeled with secondaries anti-chicken 405 at 1:2,000 (Jackson ImmunoResearch, 703-475-155) and anti-mouse 647 at 1:1,000 (Invitrogen, A32787) were used. Plates were imaged by fluorescence confocal microscopy (Nikon, A1R-HD25) and analyzed using Nikon Elements GA3. 2D ReNcell VM cultures were seeded in a 24-well plate containing 0.1-µm inserts (CELLTREAT, 230634) at approximately 75,000 cells per well. The culture was aged for 1 week in methionine-free media (Sigma-Aldrich, D9785-10L). The inserts were infected with HSV-1 (6.67 × 105 PFU per μl) at 1:250 for 24 hours. Six hours after infection, 50 µM L-azidohomoalanine was added. After infection, the inserts were removed, and the wells were fixed in 4% PFA for 1 hour at room temperature. After three washes of PBS, a permeabilizing solution of 0.5% Triton X-100 PBS was added for 15 minutes at room temperature. After three washes of PBS, a click reaction mix of 50 µM Alexa Fluor 647 Alkyne (Life Technologies, A32787), 100 mM copper sulfate and 1 M dithiothreitol in PBS were added to the wells for 1 hour at room temperature while being protected from light. The wells were then washed three times with PBS, and 5% BSA was added overnight. The next morning, three PBS washes were followed with anti-p-tau (PHF1, Albert Einstein College of Medicine) antibody at 1:500 shaking overnight at 4 °C. After three PBS washes, the plate was stained with secondary anti-mouse 594 (Invitrogen, A11032) at 1:1,000 in 5% BSA. Three more PBS washes were completed, and the wells were stored in PBS. Wells were imaged by confocal microscopy (Nikon, A1R-HD25) and analyzed by Nikon Elements GA3. 3D ReNcell VM cultures seeded in a 96-well plate (Greiner Bio-One, 655866) were aged to 3.5 weeks. At 3.5 weeks, the media were completely removed and replaced with media containing HSV-1 (2.465 × 107 PFU per well) and incubated for 24 hours at 37 °C. The wells were then washed once with PBS and then fixed with 4% PFA PBS at 4 °C for 2 hours. Afterwards, the plate was PBS washed three times followed by blocking with Doo Block at 4 °C shaking overnight. Then, anti-p-tau (PHF1, Albert Einstein College of Medicine) at 1:500 in Doo Block was added, and the plate was incubated at 4 °C shaking overnight. After overnight incubation, the plates were washed five times at 5- minute intervals using TBS-Tween. Fluorescent secondary anti-mouse 647 (Invitrogen, A32787) was added at 1:1,000 in Doo Block, and the plate was incubated on a shaker for 3 hours at 4 °C. Five 5-minute washes were repeated, and then 0.05% thioflavin S dissolved in 50% ethanol/water was added to the wells for 10 minutes. Plates were imaged by fluorescence confocal microscopy (Nikon, A1R-HD25), and images were analyzed using Nikon Elements GA3. Three and one-half to four-week-old 3D ReNcell VM cultures were infected with serial dilutions of HSV-1 for 24 hours. Conditioned media were collected and stored at −20 °C in LDH storage buffer. The samples were later analyzed using an LDH-Glo Cytotoxicity Assay (Promega, J2381), and luminescence values were recorded from a plate reader (BioTek Synergy Neo2). Statistical parameters, tests used, sample sizes and number of repeats are noted in the figure legends. No statistical methods were used to predetermine sample sizes; sample sizes were chosen based on previous similar studies10,11. Data collection and analysis were performed blinded to the conditions of the experiments whenever possible—however, not for all experiments. Exclusion of data points was performed based on standard GraphPad Prism outlier analysis (ROUT method, Q = 1%). Images are representative of all experimental repeats. Statistical significance between two groups was determined using two-tailed Student's t-tests, or, in cases where assumptions of normal distribution or variance are violated, Mann−Whitney test or Welch's t-test was used, respectively. Statistical significance for more than two groups was determined with multiple comparison ANOVAs. In cases where normal distribution assumptions were violated, a normality test was performed, and the subsequent significance test was determined accordingly. Comparisons between treatment groups only to a single control group were corrected using Dunnett's correction. Comparisons between all groups were corrected using Tukey's correction. In several instances, multiple independent comparisons were corrected using Sidak's correction. Linear regression analyses were used to determine a rate of constancy between two quantitative variables. In several instances, Kolmogorov−Smirnov test was used to determine cumulative distribution function between two independent samples. All images from in vitro samples are representative of multiple image fields described in each figure legend from a minimum of three independent experiments. The threshold for determining statistical significance was P < 0.05. All statistical analyses were performed with GraphPad Prism 10. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. Nikon NIS Elements General Analysis 3 workflows or ‘recipes' for image analysis are available from the corresponding authors upon reasonable request. Bertram, L. & Tanzi, R. E. The genetic epidemiology of neurodegenerative disease. Drubin, D. G. & Kirshner, M. W. Tau protein function in living cells. Goedert, M., Spillantini, M. G., Jakes, R., Rutherford, D. & Crowther, R. A. Multiple isoforms of human microtubule-associated protein tau: sequences and localization in neurofibrillary tangles of Alzheimer's disease. Kumar, N., Sood, D., Tomar, R. & Chandra, R. Antimicrobial peptide designing and optimization employing large-scale flexibility analysis of protein-peptide fragments. Ba, Z. et al. Phosphorylation as an effective tool to improve stability and reduce toxicity of antimicrobial peptides. Chen, P. et al. Aggregation-prone antimicrobial peptides target gram-negative bacterial nucleic acids and protein synthesis. The roles of antimicrobial peptides in innate host defense. Territo, M. C., Ganz, T., Selsted, M. E. & Lehrer, R. Monocyte-chemotactic activity of defensins from human neutrophils. Eimer, W. A. et al. Alzheimer's disease-associated β-amyloid is rapidly seeded by herpesviridae to protect against brain infection. Kumar, D. K. V. et al. Amyloid-β peptide protects against microbial infection in mouse and worm models of Alzheimer's disease. White, M. R. et al. Alzheimer's associated β-amyloid protein inhibits influenza A virus and modulates viral interactions with phagocytes. Bourgade, K. et al. β-Amyloid peptides display protective activity against the human Alzheimer's disease-associated herpes simplex virus-1. Moir, R. D., Lathe, R. & Tanzi, R. E. The antimicrobial protection hypothesis of Alzheimer's disease. Choi, S. H. et al. A three-dimensional human neural cell culture model of Alzheimer's disease. Zhang, Y. et al. Amyloid-β toxicity modulates tau phosphorylation through the PAX6 signalling pathway. A. β-Amyloid fibrils induce tau phosphorylation and loss of microtubule binding. Kwak, S. S. et al. Amyloid-β42/40 ratio drives tau pathology in 3D human neural cell culture models of Alzheimer's disease. Smith, G. A., Pomeranz, L., Gross, S. P. & Enquist, L. W. Local modulation of plus-end transport targets herpesvirus entry and egress in sensory axons. Lindwall, G. & Cole, R. D. Phosphorylation affects the ability of tau protein to promote microtubule assembly. Alvarez, G., Aldudo, J., Alonso, M., Santana, S. & Valdivieso, F. Herpes simplex virus type 1 induces nuclear accumulation of hyperphosphorylated tau in neuronal cells. Wozniak, M. A., Frost, A. L. & Itzhaki, R. F. Alzheimer's disease-specific tau phosphorylation is induced by herpes simplex virus type 1. Torres, L. et al. Toxoplasma gondii alters NMDAR signaling and induces signs of Alzheimer's disease in wild-type, C57BL/6 mice. Wang, X.-L. et al. Helicobacter pylori filtrate induces Alzheimer-like tau hyperphosphorylation by activating glycogen synthase kinase-3β. Dominy, S. S. et al. Porphyromonas gingivalis in Alzheimer's disease brains: evidence for disease causation and treatment with small-molecule inhibitors. Kobayashi, N., Masuda, J., Kudoh, J., Shimizu, N. & Yoshida, T. Binding sites on tau proteins as components for antimicrobial peptides. Kanagasingam, S., von Ruhland, C., Welbury, R. & Singhrao, S. K. Antimicrobial, polarizing light, and paired helical filament properties of fragmented tau peptides of selected putative gingipains. Chang, E., Kim, S., Schafer, K. N. & Kuret, J. Pseudophosphorylation of tau protein directly modulates its aggregation kinetics. Zhong, Q., Congdon, E. E., Nagaraja, H. N. & Kuret, J. Tau isoform composition influences rate and extent of filament formation. Galzitskaya, O. V. et al. Amyloidogenic peptides: new class of antimicrobial peptides with the novel mechanism of activity. Torrent, M., Pulido, D., Nogués, M. V. & Boix, E. Exploring new biological functions of amyloids: bacteria cell agglutination mediated by host protein aggregation. Protein aggregation as an antibiotic design strategy. Loomis, P. A., Howard, T. H., Castleberry, R. P. & Binder, L. I. Identification of nuclear tau isoforms in human neuroblastoma cells. Presence of tau in isolated nuclei from human brain. Diez, L. & Wegmann, S. Nuclear transport deficits in tau-related neurodegenerative diseases. Mansuroglu, Z. et al. Loss of Tau protein affects the structure, transcription and repair of neuronal pericentromeric heterochromatin. Li, A. et al. IFN-γ promotes τ phosphorylation without affecting mature tangles. Brelstaff, J. H. et al. Microglia become hypofunctional and release metalloproteases and tau seeds when phagocytosing live neurons with P301S tau aggregates. De Chiara, G. et al. Recurrent herpes simplex virus-1 infection induces hallmarks of neurodegeneration and cognitive deficits in mice. Sipilä, P. N. et al. Hospital-treated infectious diseases and the risk of dementia: a large, multicohort, observational study with a replication cohort. Sun, J. et al. Hospital-treated infections in early- and mid-life and risk of Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis: a nationwide nested case-control study in Sweden. Gut microbiome alterations in Alzheimer's disease. Identification and localization of Chlamydia pneumoniae in the Alzheimer's brain. Wu, Y. et al. Toll-like receptor 4 and CD11b expressed on microglia coordinate eradication of Candida albicanscerebral mycosis. Evidence supportive of a bacterial component in the etiology for Alzheimer's disease and for a temporal-spatial development of a pathogenic microbiome in the brain. Intermittent cytomegalovirus infection alters neurobiological metabolism and induces cognitive deficits in mice. Herpes simplex virus type 1 in brain and risk of Alzheimer's disease. Saravanan, R. & Bhattacharjya, S. Oligomeric structure of a cathelicidin antimicrobial peptide in dodecylphosphocholine micelle determined by NMR spectroscopy. Boman, H. G. Peptide antibiotics and their role in innate immunity. Teuton, J. R. & Brandt, C. R. Sialic acid on herpes simplex virus type 1 envelope glycoproteins is required for efficient infection of cells. & Bertke, A. S. Amyloid-β and p-tau anti-threat response to herpes simplex virus 1 infection in primary adult murine hippocampal neurons. Targeting of herpesvirus capsid transport in axons is coupled to association with specific sets of tegument proteins. Zambrano, Á et al. Neuronal cytoskeletal dynamic modification and neurodegeneration induced by infection with herpes simplex virus type 1. Johnson, A. M. & Lukens, J. R. The innate immune response in tauopathies. A. Microtubule reorganization during herpes simplex virus type 1 infection facilitates the nuclear localization of VP22, a major virion tegument protein. Kespichayawattana, W., Rattanachetkul, S., Wanun, T., Utaisincharoen, P. & Sirisinha, S. Burkholderia pseudomallei induces cell fusion and actin-associated membrane protrusion: a possible mechanism for cell-to-cell spreading. & Halford, W. P. Alpha/beta interferon and gamma interferon synergize to inhibit the replication of herpes simplex virus type 1. Danastas, K. et al. Interferon inhibits the release of herpes simplex virus-1 from the axons of sensory neurons. The release and trans-synaptic transmission of Tau via exosomes. Dujardin, S. et al. Neuron-to-neuron wild-type Tau protein transfer through a trans-synaptic mechanism: relevance to sporadic tauopathies. Karch, C. M., Jeng, A. T. & Goate, A. M. Extracellular Tau levels are influenced by variability in Tau that is associated with tauopathies. Brunello, C. A., Merezhko, M., Uronen, R.-L. & Huttunen, H. J. Mechanisms of secretion and spreading of pathological tau protein. Guo, J. L. & Lee, V. M. Y. Cell-to-cell transmission of pathogenic proteins in neurodegenerative diseases. Reactivated endogenous retroviruses promote protein aggregate spreading. Ribes, S. et al. Toll-like receptor prestimulation increases phagocytosis of Escherichia coliDH5α and Escherichia coliK1 strains by murine microglial cells. Fekete, R. et al. Microglia control the spread of neurotropic virus infection via P2Y12 signalling and recruit monocytes through P2Y12-independent mechanisms. Mookherjee, N. et al. Intracellular receptor for human host defense peptide LL-37 in monocytes. Lahaye, X. et al. NONO detects the nuclear HIV capsid to promote cGAS-mediated innate immune activation. An evolutionary roadmap to the microtubule-associated protein MAP Tau. Moir, R. D. & Tanzi, R. E. low evolutionary selection pressure in senescence does not explain the persistence of Aβ in the vertebrate genome. Kim, Y. H. et al. A 3D human neural cell culture system for modeling Alzheimer's disease. Park, J. et al. A 3D human tri-culture system modeling neurodegeneration and neuroinflammation in Alzheimer's disease. B., Jones, L. M. & Homa, F. L. Isolation and characterization of the herpes simplex virus 1 terminase complex. We would like to acknowledge the support given by the National Institute of Aging (RF1AG061035-01 to R.D.M.) The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. The authors thank D. Irimia and the BioMEMS Core at MGH for assistance with microfluidic device design and production. We also thank D. Capen and the MGH PMB Microscopy Core Facility for technical assistance with electron microscopy. Genetics and Aging Research Unit, Henry and Allison McCance Center for Brain Health, MassGeneral Institute for Neurodegenerative Disease, Charlestown, MA, USA William A. Eimer, Alex S. Rodriguez, Michael T. DeFao, Joseph Park, Deepak K. Vijaya Kumar, Nanda K. Navalpur Shanmugam, Robert D. Moir & Rudolph E. Tanzi Department of Neurology, Massachusetts General Hospital and Harvard Medical School, Charlestown, MA, USA William A. Eimer, Alex S. Rodriguez, Michael T. DeFao, Joseph Park, Deepak K. Vijaya Kumar, Nanda K. Navalpur Shanmugam, Robert D. Moir & Rudolph E. Tanzi Harvard University, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the hypothesis and strategies for this project. carried out experiments and data analysis. wrote the manuscript, with help from A.S.R. All authors contributed to manuscript editing. Correspondence to William A. Eimer or Rudolph E. Tanzi. The authors declare no competing interests. Nature Neuroscience thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. After HSV1 infection, whole wells were imaged by confocal microscopy and analyzed for red fluorescence. (a) Whole well images comparing HSV1 plaque formation in the absence or presence of 2N4R GSK-3β p-tau (1.25 µg/mL). (b-g) Whole well images from the 2N3R or 2N4R tau pre-treated conditions (scale of 2.5 to 0.625 µg/mL) were analyzed using Nikon Elements GA3 Analysis to compare (b, e) the number of HSV1 plaques, (c, f) the number of HSV1 single cell infections, and (d, g) the ratio of plaques to single cell infections. Box plots are representative of ±SEM ([B-D, n=11], [E-G, n=12]) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical mean comparisons were calculated by one-way ANOVA using Dunnett's multiple comparison test (b-f). Anti-capsid antibodies and synthetic tau isoforms were incubated with heat-immobilized HSV1 capsid or whole virion in an indirect ELISA to measure tau-virus binding affinity differences between isolated viral capsids and whole virions with intact envelopes. (a) Anti-capsid antibodies VP21/22a and ICP5 viral binding were compared between isolated HSV1 capsids and whole virion (F(1, 20)=14.29, *P=0.0136, ****P<0.0001). (b) Synthetic tau isoforms 2N3R, 50/50 mix of 2N3R/2N4R, 2N4R, and synthetic 2N4R GSK-3β p-tau binding affinities were compared between isolated HSV1 capsids and whole virion normalized against VP21/22a antibody binding (F(3, 88)=4.957, ****P<0.0001). Box plots are representative of ±SEM ([A, n=6], [B, n=12]) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical significance was calculated by two-way ANOVA using Šídák's multiple comparisons test (a, b). Dilutions of synthetic tau isoforms were incubated with heat-immobilized HSV1 capsid or whole virion in an indirect ELISA to measure tau-virus binding affinity. (a) Dilutions of 2N3R tau (scale of 5 to 0.3125 μg/mL) are incubated with immobilized HSV1 capsids (F(4, 55)=33.42, *P=0.0232, ****P<0.0001). (c) Dilutions of 2N4R tau (scale of 5 to 0.3125 μg/mL) are incubated with immobilized HSV1 capsids (F(4, 55)=11.50, **P=0.0011, ****P<0.0001). (d) 2N4R GSK-3β p-tau binding to HSV1 was compared to 2N3R, 50/50 mix of 2N3R/2N4R, and 2N4R tau (F(3, 44)=6.913, *P=0.0211, [2N4R tau vs. 2N3R tau, **P=0.0025], [2N4R GSK-3β p-tau vs. 2N3R tau, **P=0.0012]) (e) 2N4R GSK-3β p-tau and scrambled-sequence LL-37 are incubated with immobilized HSV1 capsids (F(3, 16)=2.299, *P=0.0422, ***P=0.0002, ****P<0.0001). Box plots are representative of ±SEM (n=12) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical significance was calculated by one-way ANOVA using Tukey's multiple comparisons test (a-d) and two-way ANOVA using Šídák's multiple comparisons test (e). Synthetic 2N3R tau and 2N4R GSK-3β p-tau isoforms were incubated with heat-immobilized HSV1 capsid after pre-incubation with anti-capsid protein antibodies in an indirect ELISA to examine inhibition of tau-capsid binding. (a) 2N3R tau binding to HSV1 capsid was pretreated with anti-ICP5 antibody. (b) 2N4R GSK-3β p-tau binding to HSV1 capsid was pretreated with anti-RS1-ICP4 antibody (F(8, 32)=13.59, [0 vs. 1.0 μg/mL, **P=0.0064], [0 vs. 5.0 μg/mL, **P=0.0056]). (c) 2N4R GSK-3β p-tau binding to HSV1 capsid was pretreated with anti-pUL25 antibody. Box plots are representative of ±SEM (n=12) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical significance was calculated by two-tailed Mann-Whitney test (a) and one-way ANOVA using Dunnett's multiple comparison test (b, c). ReNcell VM cultures were infected with HSV1 for 24 hours to characterize total tau's distribution changes from infection and in proximity to viral infection. (a) 2D ReNcell VM cultures in microfluidic devices were infected in the left chamber with HSV1 for 48 hours, immunoprobed with anti-total tau labeled with a fluorescent secondary antibody, and analyzed for total tau (405), neurons (GFP), and HSV1 (RFP) fluorescence by confocal microscopy. Fluorescence signals were compared by General Analysis 3 (GA3). (c) Individual neuronal total tau fluorescent intensity was compared to total proximal HSV1-RFP fluorescent intensity (R²=1.350e-006). (d) Fluorescent image captures from 384 wells over 4 experiments were compared for total tau fluorescence in cell bodies (***P=0.0003) between uninfected and infected wells. Statistical mean comparisons were calculated by one-way ANOVA (b), simple linear regression (c), and two-tailed paired t-test (d). Images are representative of experiments repeated in triplicate. 3D ReNcell VM cultures were infected with HSV1 to characterize tau's change in cellular distribution and levels to viral infection. 96 well plates were immunoprobed with anti-p-tau antibodies AT180 and PHF-1 labeled with a fluorescent secondary antibody. Images were collected via confocal microscopy and immunofluorescent representative images show neurons (GFP), HSV1 (RFP), and p-tau (647). Images are representative of experiments repeated in triplicate. Conditioned media from 24 hour HSV1 infected 3D ReNcell VM cultures were analyzed via a cytotoxicity assay to characterize culture cell death. Box plots are representative of ±SEM (n=30) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical significance was calculated by one-way ANOVA using Dunnett's multiple comparison test. 1 week old 2D ReNcell VM cultures were pre-incubated with a GSK-3β inhibitor for 24 hours followed by a 24 hour HSV1 infection to assess reduced kinase activity on viral replication. Post-infection whole wells were imaged by confocal microscopy and analyzed for red fluorescence. Images were analyzed using Nikon Elements GA3 Analysis to compare (a) the number of HSV1 single cell infections (F(3, 53)=5.204, **P=0.0012), (b) the ratio of plaques to single cell infections (F(3, 53)=4.173, **P=0.0046), and (c) the size of individual HSV1 plaques by area (****P<0.0001). Bars represent the distribution of replicate wells (n=11), with whiskers denoting variability according to Tukey's method. Statistical mean comparisons were calculated by one-way ANOVA using Dunnett's multiple comparisons test (a, b) and two-tailed Kolmogorov-Smirnov test (c). 2D ReNcell VM cultures were pre-incubated with 2N4R GSK-3β p-tau or an IFNγ antibody followed by a short-term HSV1 infection to observe effects on extracellular IFNγ levels. (a) IFNγ concentrations in conditioned media were measured following a 24 hour pre-incubation of IFNγ antibody and again following HSV1 infection. (b) IFNγ concentrations in conditioned media were measured following a 24 h pre-incubation of 2N4R GSK-3β p-tau and again following HSV1 infection. Box plots are representative of ±SEM ([A, n=3], [B, n=4]) depicting median and IQR with whiskers denoting variability according to Tukey's method. Statistical mean comparison was calculated by two-tailed unpaired t-test (a) and two-tailed Welch's t-test (b). (a, b) 3D ReNcell VM cultures were infected with HSV1 for 24 h, immunoprobed with anti-p-tau (PHF-1) labeled with a fluorescent secondary antibody, and analyzed for neurons (GFP), HSV1 (RFP), and p-tau (647) fluorescence by confocal microscopy. (a) Dystrophic neurites, identified by punctate linear formations originating from a cell soma, contain p-tau and HSV1 fluorescence. (b) Increased punctate p-tau fluorescence observed in neuron soma adjacent to HSV1-positive cells and in cells with nuclear replication compartments (arrows). Panels are representative of data from multiple discrete imaging fields across experiments repeated in triplicate. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Phosphorylated tau exhibits antimicrobial activity capable of neutralizing herpes simplex virus 1 infectivity in human neurons. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02905-x'>Orthogonal RNA-regulated destabilization domains for three-color RNA imaging with minimal RNA perturbation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-17 10:09:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article RNA is one of the key molecules responsible for controlling gene expression regulation, and visualizing individual RNA molecules in living cells offers unique insights into the dynamics of this process. Recently, the RNA-regulated destabilization domain was developed for live-cell imaging of single RNA. However, this method is limited to single-color RNA imaging and its long RNA tag induces destabilization of the tagged RNA. Here we describe two orthogonal RNA-regulated destabilization domains (mDeg and pDeg) that enable three-color messenger RNA (mRNA) imaging in living cells. We show that these destabilization domains can image mRNA tethered to the endoplasmic reticulum membrane, the inner surface of the plasma membrane and in the cytosol. In addition, we show that mDeg can detect mRNA more effectively than the previously reported tDeg system. Moreover, mDeg can be combined with a short RNA tag (9XMS2) for single-molecule RNA imaging without perturbation of RNA stability. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The sequences of the plasmids used in this study have been deposited on GenBank with the following accession numbers: miniCMV-EYFP-mDeg (PX401316), U6 + 27-Tornado-F30-MS2 (PX401332), miniCMV-EYFP-pDeg (PX401317), U6 + 27-Tornado-F30-PP7 (PX401333), CMV-CyTERM-mTagBFP2-24XMS2 (PX401318), miniCMV-(AausFP1)2-mDeg (PX401319), CMV-CyTERM-mTagBFP2-24XPP7 (PX401320), miniCMV-(AausFP1)2-pDeg (PX401321), CMV-NanoLuc-24XMS2 (PX401322), miniCMV-(HaloTag)4-mDeg (PX401323), CMV-CDK6-24XPP7 (PX401324), miniCMV-tdTomato-pDeg (PX401325), CMV-mCherry-9XMS2 (PX401326), miniCMV-(mStayGold)8-mDeg (PX401327), CMV-NLS-mTagBFP2-24XMS2 (PX401328), miniCMV-Stargazin-tdTomato-mDeg (PX401329), CMV-NLS-mTagBFP2-24XPP7 (PX401330) and miniCMV-Stargazin-tdTomato-pDeg (PX401331). These plasmids are also available on Addgene according to the terms of the Uniform Biological Material Transfer Agreement. Data generated from this study are available through the corresponding author upon reasonable request. Source data are provided with this paper. Matera, A. G. & Wang, Z. A day in the life of the spliceosome. Kobayashi, H. & Tomari, Y. RISC assembly: coordination between small RNAs and Argonaute proteins. Bertrand, E. et al. Localization of ASH1 mRNA particles in living yeast. A., Patskovsky, Y., Almo, S. C. & Singer, R. H. Structural basis for the coevolution of a viral RNA-protein complex. & Jaffrey, S. R. RNA mimics of green fluorescent protein. Strack, R. L., Disney, M. D. & Jaffrey, S. R. A superfolding Spinach2 reveals the dynamic nature of trinucleotide repeat-containing RNA. Filonov, G. S., Moon, J. D., Svensen, N. & Jaffrey, S. R. Broccoli: rapid selection of an RNA mimic of green fluorescent protein by fluorescence-based selection and directed evolution. Chen, X. et al. Visualizing RNA dynamics in live cells with bright and stable fluorescent RNAs. Braselmann, E. et al. A multicolor riboswitch-based platform for imaging of RNA in live mammalian cells. Autour, A. et al. Fluorogenic RNA Mango aptamers for imaging small non-coding RNAs in mammalian cells. Sunbul, M. et al. Super-resolution RNA imaging using a rhodamine-binding aptamer with fast exchange kinetics. Han, K. Y., Leslie, B. J., Fei, J., Zhang, J. & Ha, T. Understanding the photophysics of the spinach-DFHBI RNA aptamer-fluorogen complex to improve live-cell RNA imaging. Wu, J. et al. Live imaging of mRNA using RNA-stabilized fluorogenic proteins. Li, W., Maekiniemi, A., Sato, H., Osman, C. & Singer, R. H. An improved imaging system that corrects MS2-induced RNA destabilization. Rink, M. R. et al. Concatemeric Broccoli reduces mRNA stability and induces aggregates. Bonger, K. M., Chen, L.-C., Liu, C. W. & Wandless, T. J. Small-molecule displacement of a cryptic degron causes conditional protein degradation. The eukaryotic proteome is shaped by E3 ubiquitin ligases targeting C-terminal degrons. Structural basis of pyrimidine specificity in the MS2 RNA hairpin-coat-protein complex. Litke, J. L. & Jaffrey, S. R. Highly efficient expression of circular RNA aptamers in cells using autocatalytic transcripts. Tars, K., Fridborg, K., Bundule, M. & Liljas, L. The three-dimensional structure of bacteriophage PP7 from Pseudomonas aeruginosa at 3.7-Å resolution. Hirano, M. et al. A highly photostable and bright green fluorescent protein. Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. Los, G. V. et al. HaloTag: a novel protein labeling technology for cell imaging and protein analysis. Zacharias, D. A., Violin, J. D., Newton, A. C. & Tsien, R. Y. Partitioning of lipid-modified monomeric GFPs into membrane microdomains of live cells. Gadella, T. W. J. et al. mScarlet3: a brilliant and fast-maturing red fluorescent protein. Shcherbakova, D. M. & Verkhusha, V. V. Near-infrared fluorescent proteins for multicolor in vivo imaging. B. et al. A general method to fine-tune fluorophores for live-cell and in vivo imaging. Costantini, L. M., Fossati, M., Francolini, M. & Snapp, E. L. Assessing the tendency of fluorescent proteins to oligomerize under physiologic conditions. Wu, B. et al. Synonymous modification results in high-fidelity gene expression of repetitive protein and nucleotide sequences. Wu, B., Chao, J. & Singer, R. H. Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells. Lambert, G. G. et al. Aequorea's secrets revealed: new fluorescent proteins with unique properties for bioimaging and biosensing. Single mRNA molecules demonstrate probabilistic movement in living mammalian cells. B. et al. Mapping translation ‘hot-spots' in live cells by tracking single molecules of mRNA and ribosomes. Bühler, B. et al. Avidity-based bright and photostable light-up aptamers for single-molecule mRNA imaging. Wang, C., Han, B., Zhou, R. & Zhuang, X. Real-time imaging of translation on single mRNA transcripts in live cells. Yan, X., Hoek, T. A., Vale, R. D. & Tanenbaum, M. E. Dynamics of translation of single mRNA molecules in vivo. The mouse stargazer gene encodes a neuronal Ca2+-channel gamma subunit. Natwick, D. E. & Collins, S. R. Optimized iLID membrane anchors for local optogenetic protein recruitment. Ando, R. et al. StayGold variants for molecular fusion and membrane-targeting applications. Tanenbaum, M. E., Gilbert, L. A., Qi, L. S., Weissman, J. S. & Vale, R. D. A protein-tagging system for signal amplification in gene expression and fluorescence imaging. Bos, T. J., Nussbacher, J. K., Aigner, S. & Yeo, G. W. Tethered function assays as tools to elucidate the molecular roles of RNA-binding proteins. Fliegel, L., Burns, K., MacLennan, D. H., Reithmeier, R. A. & Michalak, M. Molecular cloning of the high affinity calcium-binding protein (calreticulin) of skeletal muscle sarcoplasmic reticulum. Kendall, J. M., Dormer, R. L. & Campbell, A. K. Targeting aequorin to the endoplasmic reticulum of living cells. Filonov, G. S., Kam, C. W., Song, W. & Jaffrey, S. R. In-gel imaging of RNA processing using broccoli reveals optimal aptamer expression strategies. This work was supported by the UMass startup funds and grant no. was supported by grant no. We gratefully acknowledge the services and support of the University of Massachusetts Amherst Cell Culture Core Facility (grant no. RRID:SRC_023477), Flow Cytometry Core Facility, and Biophysical Characterization Core Facility (grant no. The microscopy analysis was performed using computation supplied by the Light Microscopy Facility (grant no. RRID:SCR_021148) and Nikon Center of Excellence at the Institute for Applied Life Sciences, UMass Amherst, with support from the Massachusetts Life Sciences Center. We thank S. Eyles, C. Bobst, J. Chambers, A. Burnside and M. Daley at the University of Massachusetts Amherst for technical support. We thank G. Srimathveeravalli at the University of Massachusetts Amherst for providing access to the qPCR system. We also thank other members of the Wu laboratory, the You laboratory and the Martin laboratory for useful comments and suggestions. Department of Chemistry, University of Massachusetts, Amherst, MA, USA Tien G. Pham, Omoyemi Ajayi, Jiaze He, Irina Sagarbarria, Jeanne A. Hardy & Jiahui Wu Center for Bioactive Delivery in the Institute for Applied Life Sciences, University of Massachusetts, Amherst, MA, USA Jeanne A. Hardy & Jiahui Wu Molecular and Cellular Biology Graduate Program, University of Massachusetts Amherst, Amherst, MA, USA Jeanne A. Hardy & Jiahui Wu Models to Medicine Center in the Institute for Applied Life Sciences, University of Massachusetts, Amherst, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and directed all aspects of the project. All authors designed experiments. performed RT–qPCR experiments and generated stable cell lines. performed electrophoretic mobility shift assay. performed protein expression and purification experiments. All authors wrote the paper. Correspondence to Jiahui Wu. The authors declare no competing interests. Nature Methods thanks Gal Haimovich and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editors: Rita Strack and Lei Tang, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Shown is the crystal structure of MS2-stdMCP (PDB: 2BU1) (a) and our initial designs for converting stdMCP to an MS2-regulated destabilization domain (b). (a) The MS2 RNA is depicted in orange and the stdMCP protein is depicted in grey with side chains of Arg49 and Arg83 highlighted in each MCP monomer. To engineer an MS2-regulated destabilization domain, we circularly permuted stdMCP at Arg49 and Arg83, respectively, and inserted “Arg-Arg-Gly” at the new C-terminus to generate the full “Arg-Arg-Arg-Gly” degron. We chose Arg49 and Arg83 for circular permutation because they are close to the MS2 binding site, and thus, binding of the MS2 RNA may block the Arg-Arg-Arg-Gly degron from recruitment of the proteasomal machinery needed for proteolysis. (b) Partial sequence of our designs for an MS2-regulated destabilization domain. Amino acid sequence of circularly permutated stdMCP is highlighted in grey, the “Arg-Arg-Arg-Gly” degron sequence is in red. Shown is the crystal structure of PP7-stdPCP (PDB: 2QUX) (a) and our initial design for converting stdPCP to a PP7-regulated destabilization domain (b). (a) The PP7 RNA is depicted in tiffany green and the stdPCP protein is depicted in grey with side chains of Arg45 (equivalent to Arg49 in stdMCP) highlighted in each PCP monomer. Similar to the initial design of mDeg, we circularly permuted stdPCP at Arg45 and inserted “Arg-Arg-Gly” at the new C-terminus to generate the full “Arg-Arg-Arg-Gly” degron. We termed this stdPCP-CP45-RRRG, also known as pDeg variant0 in Extended Data Fig. The idea is that the binding of the PP7 RNA to stdPCP-CP45-RRRG may block the Arg-Arg-Arg-Gly degron from recruitment of the proteasomal machinery needed for proteolysis. (b) Partial sequence of our design for a PP7-regulated destabilization domain. Amino acid sequence of circularly permutated stdPCP is highlighted in grey, the “Arg-Arg-Arg-Gly” degron sequence is in red. To construct a PP7-regulated destabilization domain, we used a similar strategy used for developing mDeg. Specifically, we titrated linker length from one to six amino acids between stdPCP-CP45 and the Arg-Arg-Arg-Gly degron, resulting in seven pDeg variant (0-6) as shown in (a). To test whether PP7 can regulate the stability of these different pDeg variants (0-6), we coexpressed each EYFP-pDeg variant with and without circular 1XPP7 in HEK293T cells, respectively. We found that stdPCP-CP45-RRRG (pDeg variant0) showed limited degradation efficiency as it exhibited 76% of yellow fluorescence compared to EYFP. The addition of linkers between stdPCP-CP45 and the Arg-Arg-Arg-Gly degron led to decreases of yellow fluorescence in the absence of circular PP7. When PP7 was coexpressed, we found that pDeg variant2 containing a “Gly-Arg” linker showed the highest fold change in yellow fluorescence of 22-fold, as shown in (b) and (c). Shown are representative images from 3 independent cell cultures. All cells were stained with Hoechst. Scale bar, 100 μm. Normalized average cellular yellow fluorescence of individual cells is plotted in (c). (For each bar from left to right, n = 348, 377, 330, 400, 369, 376, 370, 386, 394, 339, 355, 395, 347, 376, 332, and 295 cells, respectively). ****Pvariant0 = 2.6 ×10−26; ****Pvariant1 = 2.4 ×10−41; ****Pvariant2 = 1.2 ×10−110; ****Pvariant3 = 3.6 ×10−110; ****Pvariant4 = 9.5 ×10−75; ****Pvariant5 = 8.0 ×10−66; ****Pvariant6 = 1.1 ×10−71 using unpaired, two-tailed, Student's t-test. (a) Schematic of plasmids that encode protein of interest fused to pDeg, and circular 1XPP7 RNA, respectively. Proteins of interest include mEGFP, mScarlet-I3, and iRFP670. (b-d) PP7-pDeg can regulate the stability of various fluorescent proteins. To test whether the PP7 RNA stabilizes different proteins fused to pDeg, we imaged HEK293T cells expressing mEGFP (b), mScarlet-I3 (c), and iRFP670 (d) fused to pDeg with and without circular 1XPP7, respectively. In each case, there was a considerable increase in fluorescence. Shown are representative images from 3 independent cell cultures. All cells were stained with Hoechst. Scale bar, 100 μm. (e-g) Summary data of normalized fluorescence of pDeg-fused fluorescent proteins with and without circular PP7 as in (b-d). Normalized average cellular fluorescence of individual cells is plotted. ****PmEGFP-pDeg = 3.4 ×10−95; ****PmScarletI3-pDeg = 1.3 ×10−84; ****PiRFP670-pDeg = 1.2 ×10−79 using an unpaired, two-tailed, Student's t-test. To test whether mDeg and pDeg can be used for detecting 24XMS2- and 24XPP7-tagged mRNAs that are stably expressed in mammalian cell lines, we generated U2OS cell lines stably expressing NLS-mTagBFP2-24XMS2 (a) and CyTERM-mCherry-24XPP7 (b), respectively. We then transiently transfected (AausFP1)2-mDeg and (AausFP1)2-pDeg to the cell line stably expressing the reporter mRNA with the cognate tag, respectively. In the U2OS cells stably expressing NLS-mTagBFP2-24XMS2, we observed mobile green-fluorescent puncta in the cytosol, reflecting the NLS-mTagBFP2-24XMS2 mRNA (a). Furthermore, in the U2OS cells stably expressing CyTERM-mCherry-24XPP7, we observed green-fluorescent puncta colocalized with the ER (b). These results suggest that fluorogenic proteins containing mDeg and pDeg can be used for imaging 24XMS2- and 24XPP7-tagged mRNA that are stably expressed in U2OS cells. Shown are representative images from 3 independent cell cultures. Scale bar, 20 μm. To test whether fluorogenic proteins containing mDeg and pDeg can be incorporated into stable cell lines for RNA imaging, we generated U2OS cell lines stably expressing (AausFP1)2-mDeg (c) and (AausFP1)2-pDeg (d), respectively. We then transiently transfected plasmids encoding CyTERM-mTagBFP2-24XMS2 (c) and CyTERM-mTagBFP2-24XPP7 (d) to the stable cell line expressing the cognate fluorogenic proteins, respectively. In both cases, we observed ER-localized green-fluorescent puncta colocalized to the ER. These results suggest that (AausFP1)2-mDeg and (AausFP1)2-pDeg can be incorporated into stable cell lines for RNA imaging. Shown are representative images from 3 independent cell cultures. Scale bar, 20 μm. To test whether Stargazin can be combined with mDeg and pDeg for tethering RNA to the plasma membrane, we constructed plasmids expressing Stargazin-tdTomato-mDeg and Stargazin-tdTomato-pDeg, respectively (a). When each of these proteins was coexpressed with an NLS-mTagBFP2 mRNA reporter fused the cognate RNA tag, we observed red-fluorescent puncta (b) with limited mobility (D for MS2-mDeg and PP7-pDeg are 0.0064 ± 0.0001 µm2/s and 0.0042 ± 0.0001 µm2/s, respectively) (c), which is in agreement with previous measurements on mRNA tethered to the plasma membrane by the CAAX sequence36. These results suggest that mDeg and pDeg can be combined with Stargazin for tethering mRNA on the inner surface of the plasma membrane. Shown are representative images from 3 independent cell cultures. Scale bar, 20 μm. To design short tags with minimal perturbation to RNA stability, we performed RT-qPCR to quantify the levels of mCherry mRNA transcripts tagged with different repeat numbers (n = 3, 6, 9, 12, and 24, respectively) of MS2 (a) and PP7 (b), respectively, and compared these to the same mCherry mRNA transcript without any tag (n = 0) in HEK293T cells. Our results showed that 9XMS2 (a) and 6XPP7 (b) have minimal perturbation on the tagged mCherry mRNA stability in living cells. (n = 5 independent cell cultures for the MS2 tags; n = 3 independent cell cultures for the PP7 tags) Values are means ± s.d. To demonstrate the generality of the short tags, we constructed plasmids expressing reporter mRNAs containing different 3'UTRs (GAPDH, RPL13A, and RPL27) fused to different repeats of MS2 (a) and PP7 (b), respectively. We performed RT-qPCR to quantify the levels of reporter mRNA transcripts tagged with different repeat numbers of MS2 (a) and PP7 (b), respectively. Our results showed that 9XMS2 (a) and 6XPP7 (b) have minimal perturbation on the stability of the tagged reporter mRNAs, which is consistent with what we observed in Extended Data Fig. However, reporter mRNAs tagged with 24XMS2 (a) or 24XPP7 (b) showed a considerable decrease in their RNA levels. Thus, these results suggest that 9XMS2 and 6XPP7 do not significantly destabilize the tagged mRNA containing different 3'UTRs. (n = 3 independent cell cultures) Values are means ± s.d. 1–17, Table 1 and descriptions of Videos 1–12. MS2-mDeg can track the dynamics of RNA in living U2OS cells. To image the dynamics of mRNA using MS2-mDeg, we coexpressed a CyTERM-mTagBFP2-24XMS2 reporter mRNA and (AausFP1)2-mDeg in U2OS cells. We observed green-fluorescent puncta with low mobility before HT treatment (−HT). Then 30 min after HT treatment (+HT), we observed a considerable increase in mobility of the green-fluorescent puncta. Scale bar, 20 μm. PP7-pDeg can track the dynamics of RNA in living U2OS cells. To image the dynamics of mRNA using PP7-pDeg, we coexpressed a CyTERM-mTagBFP2-24XPP7 reporter mRNA and (AausFP1)2-pDeg in U2OS cells. We observed green-fluorescent puncta with low mobility before HT treatment (−HT). Then 30 min after HT treatment (+HT), we observed a considerable increase in mobility of the green-fluorescent puncta. Scale bar, 20 μm. Pepper-tDeg can track the dynamics of RNA in living U2OS cells. To image the dynamics of mRNA using Pepper-tDeg, we coexpressed a CyTERM-mTagBFP2-(F30-2xPepper)10 reporter mRNA and (AausFP1)2-tDeg in U2OS cells. We observed green-fluorescent puncta with low mobility before HT treatment (−HT). Then 30 min after HT treatment HT (+HT), we observed a considerable increase in mobility of the green-fluorescent puncta. (AausFP1)2-mDeg can be transiently transfected to U2OS cells stably expressing a 24XMS2-tagged mRNA for RNA imaging. To test whether mDeg can be used for detecting 24XMS2-tagged mRNA that are stably expressed in U2OS cells, we generated a U2OS cell line stably expressing NLS-mTagBFP2-24XMS2, then transiently transfected (AausFP1)2-mDeg as the fluorogenic protein. We observed mobile green-fluorescent puncta in the cytosol, reflecting the NLS-mTagBFP2-24XMS2 mRNA. Scale bar, 20 μm. (AausFP1)2-pDeg can be transiently transfected to U2OS cells stably expressing a 24XPP7-tagged mRNA for RNA imaging. To test whether pDeg can be used for detecting 24XPP7-tagged mRNA that are stably expressed in U2OS cells, we generated a U2OS cell line stably expressing CyTERM-mCherry-24XPP7, then transiently transfected (AausFP1)2-pDeg as the fluorogenic protein. We observed green-fluorescent puncta with limited mobility, reflecting the ER-tethered CyTERM-mCherry-24XPP7 reporter mRNA. Scale bar, 20 μm. (AausFP1)2-mDeg can be incorporated into stable cell lines for RNA imaging. To test whether (AausFP1)2-mDeg can be incorporated into stable cell lines for RNA imaging, we generated a U2OS cell line stably expressing (AausFP1)2-mDeg. We then transiently transfected plasmids encoding CyTERM-mTagBFP2-24XMS2, and observed green-fluorescent puncta with limited mobility, reflecting the ER-tethered CyTERM-mTagBFP2-24XMS2 reporter mRNA. Scale bar, 20 μm. (AausFP1)2-pDeg can be incorporated into stable cell lines for RNA imaging. To test whether (AausFP1)2-pDeg can be incorporated into stable cell lines for RNA imaging, we generated a U2OS cell line stably expressing (AausFP1)2-pDeg. We then transiently transfected plasmids encoding CyTERM-mTagBFP2-24XPP7, and observed green-fluorescent puncta with limited mobility, reflecting the ER-tethered CyTERM-mTagBFP2-24XPP7 reporter mRNA. Scale bar, 20 μm. MS2-mDeg can be used for tethering RNA to the plasma membrane. To tether 24xMS2-tagged mRNAs to the inner surface of the plasma membrane, we coexpressed Stargazin-tdTomato-mDeg and an NLS-mTagBFP2-24XMS2 mRNA reporter in U2OS cells. We observed red-fluorescent puncta with limited mobility, suggesting that mDeg can be combined with Stargazin for tethering mRNA on the inner surface of the plasma membrane. Scale bar, 20 μm. PP7-pDeg can be used for tethering RNA to the plasma membrane. To tether 24xPP7-tagged mRNAs to the inner surface of the plasma membrane, we coexpressed Stargazin-tdTomato-pDeg and an NLS-mTagBFP2-24XPP7 mRNA reporter in U2OS cells. We observed red-fluorescent puncta with limited mobility, suggesting that pDeg can be combined with Stargazin for tethering mRNA on the inner surface of the plasma membrane. Scale bar, 20 μm. MS2-mDeg, PP7-pDeg and Pepper-tDeg enable three-color RNA imaging in living cells. To simultaneously image three different RNA species in U2OS cells, we coexpressed three mRNA reporters: NanoLuc-24XMS2, CDK6-24XPP7 and CyTERM-mTagBFP2-(F30-2xPepper)10 with three fluorogenic proteins: (HaloTag)4-mDeg (with the JF646 ligand) (cyan), tdTomato-pDeg (magenta) and (mNeonGreen)4-tDeg (green). We observed fluorescent puncta in the cytosol reflecting cellular mRNAs. Scale bar, 20 μm. MS2-mDeg can be combined with photostable fluorogenic proteins for continuous RNA imaging for an extended period. To test whether mDeg can be combined with photostable fluorophores for continuous RNA imaging for an extended period of time. We coexpressed (HaloTag)4-mDeg (with the JF646 ligand) and a NanoLuc-24XMS2 mRNA in U2OS cells. We observed mobile far puncta with continuous image acquisition for 120 s with limited photobleaching. Scale bar, 20 μm. 9XMS2, a short RNA imaging tag with minimal perturbation of RNA stability, for single mRNA imaging. To image mRNA tagged with 9XMS2, we coexpressed an mCherry-9XMS2 mRNA reporter with (mStayGold)8-mDeg as the fluorogenic protein, in U2OS cells. We observed mobile green-fluorescent puncta in the cytosol of the U2OS cells, reflecting single mRNA molecules. Scale bar, 20 μm. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Pham, T.G., Ajayi, O., He, J. et al. Orthogonal RNA-regulated destabilization domains for three-color RNA imaging with minimal RNA perturbation. Version of record: 17 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            