
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-02</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/under-trump-national-science-foundation-cuts-off-all-funding-to-scientists/'>Under Trump, National Science Foundation Cuts Off All Funding to Scientists</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 17:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>National Science Foundation staff were told to freeze outgoing funding days after NSF leadership introduced a new policy that requires that grants be screened for “alignment with agency priorities” Staff members at the US National Science Foundation (NSF) were told on 30 April to “stop awarding all funding actions until further notice,” according to an email seen by Nature. The policy prevents the NSF, one of the world's biggest supporters of basic research, from awarding new research grants and from supplying allotted funds for existing grants, such as those that receive yearly increments of money. Earlier this week, NSF leadership also introduced a new policy directing staff members to screen grant proposals for “topics or activities that may not be in alignment with agency priorities”. Proposals judged not “in alignment” must be returned to the applicants by NSF employees. The policy has not been made public but was described in documents seen by Nature. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. An NSF staff member says that although good science can still be funded, the policy has the potential to be “Orwellian overreach”. Another staff member says, “They are butchering the gold standard merit review process that was established at NSF over decades”. One program officer says they are resigning because of the policy. Nature spoke with five NSF staffers for this story, all on the condition of anonymity because they are not authorized to speak to the media. The changes are hitting an agency already in crisis. In the past two weeks, the NSF has terminated roughly 1,040 grants that would have awarded US$739 million to researchers and their institutions. Uncertainty is also being felt by scientists outside the agency. Colin Carlson, an expert in disease emergence at Yale University in New Haven, Connecticut, leads an initiative to predict viruses that pose pandemic threats. The project, which involves roughly 50 researchers across multiple universities, is funded by a $US12.5 million NSF grant. But Matthew Lawrence, a specialist in administrative law at Emory University in Atlanta, Georgia, says that under a 1974 law called the Impoundment Control Act, the NSF must give Congress special notice of the grant halt, which would otherwise be unlawful. Science previously reported that US President Donald Trump will request a $4 billion budget for the agency in fiscal year 2026, a 55% reduction from what Congress appropriated for 2025. During Trump's first term, Republicans in Congress rejected many of the president's requested cuts to science funding, but it is not clear that they will do so again. A report by economists at American University in Washington DC estimates that a 50% reduction in federal science funding would reduce the US gross domestic product by approximately 7.6%. “This country's status as the global leader in science and innovation is seemingly hanging by a thread at this point,” one NSF staffer says. NSF staff expect hundreds more grants to be terminated Friday. This article is reproduced with permission and was first published on May 1, 2025. First published in 1869, Nature is the world's leading multidisciplinary science journal.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01325-3'>Blood of man who's had 200 snake bites helps make a potent antivenom</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 15:22:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Katherine Bourzac is a freelance journalist in San Francisco, California. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. He has also been bitten roughly 200 times by venomous snakes. Scientists say that the research could lead to direly needed treatments, but that its reliance on material from a person who performed dangerous experiments on himself makes it ethically murky. The paper's authors say they played no part in Friede's self-exposure to venom. “We did not advise Friede to do this and no one else needs to do this again — we have all the molecules we need,” says co-author Jacob Glanville, chief executive of biomedical firm Centivax in South San Francisco, California. Each antivenom protects against the venom of at most a few species. “Considering the advanced technologies available in immunology today, it is unacceptable to continue relying on these outdated methods for treating snake bites,” says Kartik Sunagar, a biologist who develops antivenoms at the Indian Institute of Science in Bengaluru. Both types of peptide bind to the same receptors on nerve cells, impairing communication between neurons and potentially causing muscle paralysis and respiratory failure. Glanville and his co-author Peter Kwong, a biochemist at Columbia University in New York City, had read news coverage about Friede, who took careful notes about his venom exposure. After receiving approval from an ethics review board, getting informed consent from Friede, and supplying him with documents about the dangers of snake venom, the team collected two vials of Friede's blood. Seeking to add protection against even more species, the researchers tested a third element: varespladib, which inhibits a snake-venom enzyme that breaks down muscle and nerve tissue. Why snakes probably aren't spreading the new China virus The end of AIDS is in sight: don't abandon PEPFAR now COVID vaccine works faster with both doses in the same arm Storm of seizures in a baby's brain calms after trial therapy The cell-division cycle is faster in cell types prone to forming cancer My fight to unlock cannabis and psychedelic drugs for use in medical research Shape strategy, drive impact, and advance cancer research in a leading translational institute. •	Studying the development and metabolism of lymph nodes through melanoma metastasis Researching oxidative stress within lymph nodes caused by metastatic melanoma Why snakes probably aren't spreading the new China virus An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64654378/south-africa-rock-art-uncovered/'>Archaeologists Uncovered a Painting That May Prove the Existence of a Mysterious Creature</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 14:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But the recent discovery of 200-year-old rock paintings found in South Africa now has scientists hypothesizing that this ancient creature may have been far more than just a legend. If we're lucky, it could provide further clues as to just how intertwined this horned serpent was with South Africa's indigenous San culture. “Altogether,” the Benoit wrote, “they suggest a case of indigenous paleontology.” “Of course, at this point it is speculative,” said Benoit, according to IFL Science, “but the tusked animal on the Horned Serpent panel was likely painted as a rain-animal, which means it was probably involved [in] rain-making ceremonies.” These ceremonies often evoked known extinct animals to help the people encourage the gods to send rain. The San were known to have a robust mix of animals belonging to their ‘spirit world,' but Benoit said that these animals were generally inspired by reality—even if extinct. There's plenty of leaps from legend to scientific grounding that may be too large for a fabled horned serpent to make. But after further studies, the 200-year-old paintings could spin a different tale. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. This Culture's Language Is Global, But Not Its DNA Excavation Uncovers Skull Which Could Be a King A Building Crew Found an Iron Age Chariot Wheel Experts Found an Ancient Tomb with a Fake Door</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01395-3'>Trump gutted two landmark environmental reports — can researchers save them?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 13:33:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Climate change is driving more frequent bouts of extreme weather that lead to floods and other damage.Credit: Scott Olson/Getty ‘Totally broken': how Trump 2.0 has paralysed work at US science agencies ‘Totally broken': how Trump 2.0 has paralysed work at US science agencies One of their inspirations is the National Nature Assessment, a report that Trump cancelled on his first day back in office. Last month, researchers involved with that report announced a campaign to publish it outside the governmental framework, using private funding. “I volunteered for this effort, and I'm ready to continue it,” says Costa Samaras, a systems engineer at Carnegie Mellon University in Pittsburgh, Pennsylvania, and lead author of a chapter of the climate assessment. “This is important work for the country,” adds Samaras, who worked on US energy policy under Trump's predecessor, Joe Biden. Both projects were coordinated by the US Global Change Research Program (USGCRP), a network of 15 federal agencies that deal with environmental issues across the US government. This week, administration officials dismissed nearly 400 university and other external scientists who were working on it. ‘Totally broken': how Trump 2.0 has paralysed work at US science agencies Plant diversity dynamics over space and time in a warming Arctic Diet outperforms microbial transplant to drive microbiome recovery in mice Exclusive: NSF stops awarding new grants and funding existing ones Exclusive: NIH to end billions of dollars in foreign research grants Shape strategy, drive impact, and advance cancer research in a leading translational institute. •	Studying the development and metabolism of lymph nodes through melanoma metastasis Researching oxidative stress within lymph nodes caused by metastatic melanoma ‘Totally broken': how Trump 2.0 has paralysed work at US science agencies An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64612958/computation-limit-biology/'>The Computational Limit of Life May Be So Much Higher Than We Thought, Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>According to some technologists, the human brain is capable of 1016 computations per second, and if a super-advanced AI were to ever that threshold (and gain a whole host of other abilities), we'd enter hit what is known in tech circles as the singularity. However, a new article written by theoretical physicist Philip Kurian argues that this limit—and all other neuron-based estimations of life's computational abilities—have woefully underestimated the true abilities of biological brains. Kurian includes a controversial (but increasingly influential) idea  in his calculations: that quantum processes in a biological system, when taken together, far exceed the computing power of even the most advanced quantum computer. Published in the journal Science Advances, this article expands on QBL's recent discovery of cytoskeleton filaments exhibiting quantum optical features and recalculates the computational capacity of carbon-based life on Earth. Artificial quantum systems generally require ultracold, approaching-absolute-zero temperatures to run, as qubits are incredibly sensitive to disturbances (this is why quantum computers also contain robust error correction measures). So, the warm and chaotic environment of, say, a human brain, is far from ideal for quantum processes. However, for decades, some theories (that have slowly become less out there with age) have suggested that quantum processes could in fact be occurring in the brain. In some hypotheses, they could even be responsible for consciousness itself. Kurian's paper focuses on the amino acid tryptophan, which is found in many proteins and can form large networks within structures like microtubules, amyloid fibrils, cilia, and neurons. Combined with QBL's discovery last year, an idea has taken shape that aneural organisms may be able to use these quantum signals to process information. It would be able to perform operations in mere picoseconds, which would allow the cells to operate a billion times faster than chemical processing alone. This revised limit may sound humbling, but if these aneural cells are using quantum signals to process information, that's good news for both the quantum computing world and the world of artificial intelligence. The quantum computing world should take serious notice,” Kurian said in a press statement. “In the era of artificial intelligences and quantum computers, it is important to remember that physical laws restrict all their behaviors.” Is the Key to Human Immortality This Sea Creature? This Culture's Language Is Global, But Not Its DNA</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01363-x'>Star ecologist ‘blurred boundaries' in lab — but colleagues criticize investigation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 13:12:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Thomas Crowther made a name for himself after publishing a paper that suggested Earth had eight times more trees than expected.Credit: Bryan Bedder/Getty Images for Bloomberg Philanthropies A university investigation into Thomas Crowther, a prominent tree researcher who ran a large laboratory at the Swiss Federal Institute of Technology in Zurich (ETH Zurich), reveals that he blurred personal and professional relationships in his lab and breached internal rules on financial and hiring compliance. However, some members of Crowther's laboratory have accused ETH Zurich of a flawed and biased investigation, in a letter sent to the university on 28 April and seen by Nature, and they call for an independent review into the university's handling of the case. The investigation led ETH Zurich to decide, in January, that it would not renew Crowther's contract — a tenure-track professorship — and his lab, which previously employed 40–60 people, is being dismantled. ETH Zurich published a retracted version of the report on 25 April. Crowther, who is on administrative leave until his contract expires in September, denies many of the allegations investigated in the report, but admits he made mistakes. “I am deeply sorry if this caused any discomfort for anyone,” Crowther said in a statement to Nature. “The President of ETH Zurich had already concluded in December 2024 that the basis of trust required for a permanent position was not given in this case.” His work garnered interest from influential people, including US President Donald Trump, and inspired a high-profile tree-conservation initiative by the World Economic Forum. Crowther has been at ETH Zurich since 2017, but in late 2024, Swiss media reported that several people had made allegations against him to the university. The report into these allegations, investigated in 2024 and 2025 by a law firm appointed by ETH Zurich, reveals that among the documents examined were five individuals' written statements and a whistle-blowing report on possible compliance issues. Although most employees appreciated his behaviour, the report says, others thought it was inappropriate and said he crossed personal boundaries. Examples of this reported behaviour included late-night WhatsApp messages inviting employees to parties. Crowther added: “I would never ever initiate any form of intimate interaction with a student. I am grateful that the ETH investigation uncovered clear evidence to contradict these accusations.” On financial and legal matters, the probe investigated several allegations, including that lab members worked for other organizations during their contracted hours and that the lab misused consultancy contracts. Crowther acknowledged a “few minor oversights” relating to spending on crisis communication and legal advice, but denies that consultancy contracts were misused. Crowther says that his lab was huge, with “countless contracts and collaborations”. But I can guarantee that no lab members worked for other organizations during their employment at ETH,” he adds. ‘Totally broken': how Trump 2.0 has paralysed work at US science agencies Shape strategy, drive impact, and advance cancer research in a leading translational institute. •	Studying the development and metabolism of lymph nodes through melanoma metastasis Researching oxidative stress within lymph nodes caused by metastatic melanoma An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64641931/scientists-turned-rain-into-electricity-it-could-one-day-overhaul-our-power-grid/'>Scientists Turned Rain Into Electricity. It Could One Day Overhaul Our Power Grid.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Have you ever absentmindedly run your hand over the fabric of your shirt, then touched a light switch and gotten zapped back into consciousness? That same effect could soon generate massive amounts of electric power with rain. Hydroelectric power may produce clean energy, but in order for it to work, the movement of water needs to activate some sort of mechanical system (like a turbine). This complex equipment cannot be used everywhere, and may even harm the environment in certain situations. It also generates energy magnitudes greater than continuously flowing water, because the charge created by drop after drop of rain takes much longer to decay. This is what engineer Siowling Soh—from the National University of Singapore—and his research team realized when they tried to generate electricity by catching rainwater in tubes. “Hence, it is inexpensive and environmentally friendly to install, operate, and maintain […] scaling up can be achieved readily in three dimensions for large-scale harvesting of energy from nature. It can be used anywhere, including in urbanized areas.” However, not only does the charge from flowing water decay more rapidly, but when tubes that small are used, that charge becomes just about negligible. It is also unnatural for water to flow through nano-tubes, which is why Soh stresses that macroscale tubes on the scale of at least millimeters wide should be used to harvest electricity from natural sources of water. Soh thinks it can be easily scaled up for longer stretches of time and higher voltages. “Importantly,” he said in the same study, “the combination of high efficiency and use of macroscale tubes enables plug flow to harvest renewable and clean energy from nature freely, including directly from rain.” In the future, maybe that electric bill will trigger less of a shock. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. The Computational Limit of Life May Be Way Higher Is the Key to Human Immortality This Sea Creature? This Culture's Language Is Global, But Not Its DNA</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a64636544/avian-tree-of-life/'>Scientists Mapped the Evolution of 11,000 Bird Species to Build the Avian Tree of Life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Those many millions of years have given birds time to evolve into some 11,000 species, and keeping track of all those species—not to mention their evolutionary history–can be quite the challenge. After combining additional data on the 1,800 or so species not included in these studies, the team formed a complete map of avian evolutionary history. “People love birds, and a lot of people work on birds,” Emily Jane McTavish, the lead author of the study from UC Merced, said in a press statement. “People publish scientific papers about birds' evolutionary relationships all the time. For years, McTavish worked on software known as the Open Tree of Life (OpenTree) project, a collaboration between evolutionary biologists and taxonomists to build a comprehensive evolutionary tree of all species on the planet—not just birds. “Many dozens of bird phylogenies (studies of evolutionary histories using genetics) get published every year, yet their findings—with implications for everything from taxonomy to our understanding of ancestral characters—aren't necessarily being used for downstream research,” Eliot Miller, a visiting scientist to Cornell Lab and senior author of the study, said in a press statement. “Our project should help to close this research loop so that these studies and their findings are better incorporated into follow-up research.” Because Cornell Lab is also in charge of popular citizen science bird tools like Merlin and eBird, this updated tree can also link with those datasets, providing even better models for scientists and birders alike. Crucially, similar techniques used to construct this evolutionary tree can be applied to other groups of species as well, slowly revealing the incredible and inter-related animal diversity that spans the globe. A Trap Set for the Loch Ness Monster Just Surfaced Experts Used a Mammoth Genome to Make Woolly Mice</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44260-025-00039-x'>A first-principles mathematical model integrates the disparate timescales of human learning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 11:11:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Lifelong learning occurs on timescales ranging from moments to decades. People can lose themselves in a new skill, practice for hours until exhausted, and pursue mastery intermittently over decades. A full understanding of learning requires an account that integrates these timescales. Here, in response to calls for more formal theory in the psychological sciences, we present a parsimonious mathematical model that unifies the nested timescales of learning. Our model recovers well-established patterns of skill acquisition, and explains how these patterns can emerge from short-timescale dynamics of motivation, fatigue, and effort. Conversely, the model explains how patterns in these short-timescale dynamics are shaped by longer-term dynamics of skill selection, mastery, and abandonment. We use this model to explore the theoretical benefits and pitfalls of a variety of training regimes. Our model connects disparate timescales—and the subdisciplines that typically study each timescale in isolation—to offer a unified, multiscale account of skill acquisition. Learning is a through-line of human life, from children at play to Picasso at his peak. The process of learning occurs on multiple timescales1,2,3. Over the long timecourse of mastering a skill, learning may appear continuous and gradual as one progresses from novice to expert. But this slow process consists of countless shorter, discrete periods of engagement, lasting from mere moments to entire days. An individual may start working but get bored. They may eventually master the skill, or give up, or switch focus to another skill entirely. In short, learning occurs on multiple nested timescales, from the moment-to-moment dynamics of motivation and fatigue, to briefer episodes of sustained effort, to the slow crawl toward mastery. A full understanding of learning requires an account that integrates these timescales3. Existing accounts, however, have typically focused on single timescales. Here, we present a minimal mathematical model that unifies the nested timescales of lifelong learning, offering a theoretical account of well-established patterns at both short- and long-timescales. The model captures the relations between the dynamics of motivation, fatigue, and sustained engagement on the shorter timescale of individual periods of work (e.g., minutes or hours), as well as well-established, classic patterns of learning and performance on the longer timescale of mastery (e.g., days or years). Our goal is to offer a formal explanation of why similar qualitative patterns in skill learning have been found to occur in a wide range of domains, from juggling to chess. We are thus responding to the call for more formal theory building in the psychological sciences4,5,6. Unlike statistical models, which aim to describe empirical data, or cognitive process models, which aim to capture in detail the internal mental processes that give rise to particular psychological outcomes, our model offers a more foundational account of common patterns found across superficially dissimilar domains of learning. This approach to theory building has been influential in fields such as ecology7 and statistical physics8. Formal models such as the one introduced here are opportunities to specify the minimal components that suffice to give rise to qualitative patterns, thus advancing our theoretical understanding of the origin of those patterns. Here, we offer such an account of multiscale lifelong learning. Below, we briefly review the longer and shorter timescales of learning. We then introduce our formal model, which unifies these timescales. An often-repeated cliché holds that mastery requires “10,000 h”—that is, two years of constant, daily effort, breaking only to sleep at night9,10. While overly simplifying, this saying captures the fact that real mastery requires sustained engagement over long periods9,10,11,12,13. The dynamics of learning on this timescale are typically described using so-called “learning curves” or “progress curves”12,14,15,16. These represent the gradual increase in skill over time, where skill is typically measured at the level of an entire session of work or practice: the final score in a game, the best performance in some athletic skill within a training session, mean typing speed within a testing period, etc. In domains as varied as music, chess, memory, typing, sport, and juggling, progress curves typically show diminishing returns, with the greatest rate of learning when skill is far from mastery, and a gradually diminishing rate of learning as mastery is approached (Fig. Empirical progress curves can be described using power or exponential functions, although there is debate about the best-fitting functional form18,19,20,21,22. Our mathematical model aims to recover these qualitative patterns. A Skill development (solid red) typically exhibits diminishing returns on practice as one approaches mastery of a task. Over time, one can switch to more challenging tasks (dashed blue line), thus creating opportunities for continued skill development. Zoomed inset shows a typical learning curve with diminishing returns over time. B Performance typically increases with practice but can reach plateaus. When one switches to a more challenging task, there can be a momentary dip in performance before performance outstrips the previous plateau. C For people who are motivated by skill development, motivation typically peaks when the task is sufficiently difficult to maximize learning. As the gap shrinks between skill and task difficulty, motivation typically decreases. Switching to a more challenging task can increase motivation. D Average work and rest period lengths. When individuals motivated by skill development encounter a task that is sufficiently but not overly challenging, they typically work for long periods (orange), and require comparatively little rest (grey). Over time, as the gap shrinks between skill and task difficulty, they may work for shorter and shorter periods as they get increasingly bored. Note that these dynamics reflect endogenous behavior of the model without external forcings. Indeed, more recent work has found that, at the individual level, learning often improves in distinct stages (Fig. These plateaus are periods where the learner has mastered a particular task and has not yet switched their focus to a more challenging one. The learner may eventually pursue a more challenging task, which might result in worse performance at first but ultimately better performance as the new, more challenging task is mastered (Fig. In some instances of this general scenario, the simpler “task” is a sub-optimal approach to a problem to which the more difficult “tasks” are better but more challenging solutions. An example of this is when athletes in the sport of high jump start with easier but less effective ways of jumping over the bar (e.g., jumping forwards) but eventually transition to the more challenging but ultimately more effective “Fosbury Flop” (i.e., jumping backwards). The Fosbury Flop was originally derided as an overly difficult approach to the problem of jumping over the bar, and it is indeed more challenging at first than jumping face first, but it is ultimately a more effective solution; indeed, it is now the standard approach at the highest levels of the sport23. Thus, on the slow timescale of skill mastery, individuals typically exhibit learning with diminishing returns, and may exhibit repeated plateaus, followed by brief dips in performance and increased learning. This general pattern has been documented in a striking range of domains, from juggling to mental arithmetic, from sports to video games23. People choose to start working, and then persist in that effort until they give up or require rest—and then, having rested, they may choose to start once more. When people start and stop is often a function of their motivation27 and fatigue28 (Exceptions include cases where people are forced to work by some external entity, such as a teacher, coach, or employer). While the development of skill and performance are typically measured at the level of entire work-sessions, people are nevertheless learning within each session. On this fast timescale of moment-to-moment engagement, motivation and fatigue may fluctuate, skills may creep ever upward, and accomplishments may accumulate slowly. Long-term mastery, therefore, emerges from the the dynamics of short-timescale engagement and learning. A full understanding of life-long learning and sustained engagement requires an understanding of both long and short timescales3. The precise timescale on which “long timescale” learning occurs will depend on the particular task and domain. Mastering a new video game, for instance, may require hours or days, while becoming a chess master can take decades. From our perspective, these are both “long timescale” processes, since they involve changes that are orders of magnitude slower than moment-to-moment fluctuations in motivation and fatigue. At present, however, we lack a unifying framework, one that situates known patterns on the short timescale of single work sessions within the context of well-established patterns of long-timescale skill development. This is a problem because changes on one timescale shape changes on the other29. Lifelong learning emerges from the moment-to-moment dynamics of motivation, fatigue, skill development, and performance30. The decision to work or rest, meanwhile, is a function of fatigue and motivation, but motivation is shaped by long-timescale changes in skill and in the task being learned31. The dynamics of each timescale only make sense in light of the other. Encouraged by successful models of nested timescales in other biological and social systems32,33,34,35, here we develop a first-principles mathematical model that unifies the nested timescales of learning. By bridging between levels—the long timescale of mastery, and the short timescale of effort and engagement—the model connects phenomena that have traditionally been investigated by distinct research traditions. To be clear, we are not offering a statistical model of empirical data from any particular study of skill acquisition. Moreover, any minimal model of a phenomenon as complex as human lifelong learning will only be able to account for major qualitative patterns. And outcomes that reflect factors that lie outside the model—such as exogenous pressures to work or rest, whether driven by diurnal cycles or workplace supervisors—will necessarily lie outside the model's scope. Rather, our goal is to offer a task-agnostic account that explains, from first-principles, why similar patterns have been found to recur across domains, from chess to juggling. Our contribution is thus theoretical, not empirical. In what follows, we introduce the formal model. We show that the model reproduces many classic, well-established patterns in skill acquisition, on multiple timescales. This first-principles model thus offers a theoretical account of why similar patterns recur across domains. We then explore the model's behavior in the context of different training regimes (e.g., continuously increasing task difficulty vs. discrete jumps in difficulty) and for individuals who differ in their motivation. We present a first-principles model that integrates the multiple timescales of learning, from moment-to-moment engagement to life-long growth and skill acquisition. We take a dynamical systems approach, where our model consists of a system of ordinary differential equations. The dynamical systems approach has been highly effective at providing elegant explanations of complex phenomena in human systems such as motor coordination36,37,38, cognitive development3,24, left-handedness in a minority of the population39, and the polarization of political parties40,41. We divide the description of the dynamical system into two components: three state variables that change on a long timescale, and another three that change on a short timescale. The mathematical model thus contains variables that change on both short and long timescales, with the different timescales connected by the individual's decision to work or rest. Figure 2A illustrates the mathematical model's conceptual framework and the relationships between key variables. See Table 1 for a summary of key model parameters on various time scales. A Short timescale variables (motivation, M, and fatigue, F) determine whether an individual decides to work, p. As an individual works, skill (S) improves as does task execution (E). Task difficulty (T) can change over time or remain constant. B Illustration of the distinction between long- and short-timescale variables. Long timescale variables (e.g., skill) tend to change slowly (dashed green line). Short timescale variables (e.g., motivation) change more rapidly, sometimes oscillating (solid blue curve). C The decision to work (p = 1) or rest (p = 0) is a function of the difference between motivation and fatigue (M − F). Starting to work requires sufficient motivation relative to fatigue (i.e., M − F > θ). The additional effort required to start working, θ, induces path-dependence in the decision to work or rest (i.e., hysteresis). D Skill S and execution E change more slowly, and their dynamics depend on the relative task difficulty (i.e., T − S). The three key long-timescale variables are S(t), the skill level of an individual at time t, T(t), the task difficulty, and E(t) (derived from S and T), the rate of execution, or how much work an individual is getting done at time t. Within psychology, rate of execution is sometimes referred to as efficiency42, though here we use ‘rate of execution' to avoid confusion with other uses of the term ‘efficiency' in economics and engineering. The variables S, E, and T all take continuous, non-negative values. The distinction between skill and execution captures the fact that people often learn best and perform best at different times43. One performs best when executing tasks that are easy, relative to one's skill level, while one develops skills when working on tasks that are more difficult (but not too much more difficult) than one's current skill level44,45. We calculate an individual's performance as the product of their current skill and current execution, S(t)E(t), since performance is jointly affected by skill level and rate of execution. In the model, skill improvement is a function of relative task difficulty, which is the difference between task difficulty and one's skill level (T(t) − S(t)). In other work, this is also referred to as functional task difficulty44. Specifically, skill improves when the task is harder than one's current skill level (T(t) − S(t) > 0), but only to an extent. If a task is too easy or too difficult, the individual working on it does not develop skill44,45. Put mathematically, the dynamics of skill are given by: where p(t) is a boolean variable denoting whether the individual is working at time t. If they are working, p(t) = 1, and if resting, p(t) = 0. Note that all variables and parameters in this and the following equations are dimensionless. In the Supplemental Information, we show how these dimensionless equations are derived from dimensional ones. For simplicity, we consider this function to take the shape of a triangle, peaking at value xm (where xm > 0), which we visualize in Fig. This piece-wise function peaks at height ym, denoting the maximum learning rate. Note that this approach, unlike power-law or exponential learning curves, allows us to model the period of skill acquisition when the initial task is much too difficult for the learner, given their current skill. During this period, learning is laborious and slow, with little skill acquisition occurring even with extended effort. Think of the young child who first encounters chess before they are sufficiently mature to even remember all the rules: repeated encounters with the game will produce little improvement in their ability to play chess. This period is often ignored in empirical work on skill acquisition, because participants and tasks are intentionally matched so the task is approachable. However, in many real-world scenarios, an individual encounters a task that is too difficult to even start—or if starting is possible, then it is initially quite laborious. In our modeling framework, this period corresponds to the right-hand side of the learning function where relative task difficulty is very high. Second, our approach is agnostic to the exact functional form of the learning curve. We show in the Supplementary Information that the function fS, which governs the rate of learning as a function of the relative task difficulty, can be modified to generate learning curves that follow, exactly, either an exponential or a power law curve. Here, we choose the simplest functional form that gives diminishing returns on effort: a piecewise linear relationship between relative task difficulty and rate of learning. This allows us to capture the qualitative features of empirical learning curves with fixed tasks, while also allowing us to model more complex scenarios involving variable task difficulty (e.g., when task difficulty is simultaneously changing with time or adapting to skill). Similarly, the rate of execution, a dimensionless quantity that indicates the proportion of skill level brought to conduct the task, is given by, Intuitively, this says that execution is at a maximum when the demands of the task are fully met by one's skill, but execution decreases linearly as the deficit between task difficulty and skill gets larger44. An individual's performance on a task reflects both their skill and execution. Performance is highest when they are both highly skilled and executing the task with maximum execution; it is lower for individuals with lower skill, individuals who are struggling to execute the task, or both. We thus measure an individual's performance as the product of their skill and their rate of execution, S(t)E(t). Unlike skill (S(t)) and execution (E(t)), which are solely governed by the internal dynamics of the dynamical system, our third long-timescale variable, task difficulty (T(t)), can be treated as an external input variable. As a result, our modeling framework allows us to explore how different ways of increasing task difficulty affect the dynamics of long-timescale learning and short-timescale engagement. Specifically, we evaluated several different training regimes: task difficulty remains unchanged (constant scenario), increases in a discrete fashion once skill catches up with task difficulty (stepwise scenario), and increases in a continuous fashion (continuous scenario). On the short timescale, three key variables that govern the individual's working behavior are motivation (M(t)), fatigue (F(t)), and work (p(t))46,47,48. For simple tasks that are primarily physical, this may be driven by levels of the neurotransmitter dopamine50; for more complex or conceptual tasks, motivation intensity is likely a composite of different psychological and physiological cues. Here, we remain agnostic about the specific physiological underpinning of motivation. We model motivation, M(t), as a continuous variable taking non-negative values. Fatigue is a sense of cumulative exertion or tiredness51. For some simple tasks that are primarily physical, the sense of fatigue may be driven by simple physiological processes, such as the accumulation of cerebral adenosine50; for more complex or conceptual tasks, much like for motivation, the sense of fatigue is likely a composite of different psychological and physiological cues. Here we also remain agnostic about the specific physiological underpinning of fatigue. We model fatigue, F(t), as a continuous variable taking non-negative values. We assume the level of motivation is affected by two processes. The first is that, in the absence of other influences, motivation will restore to a baseline52,53. In dimensionless equations, this baseline is represented as 1 (see Supplementary Information for details on the nondimensionalization process). The second is that motivation increases when an individual is learning (Eq. (1)) or executing a task (Eq. Mathematically, these dynamics are expressed as The parameters c1 and c2 are constants weighting the contribution of these two processes. As introduced above, S(t) denotes skill and E(t) denotes execution of an individual at time t. The relative influence of learning and performance on motivation is determined by parameter w, which takes values between 0 (only performance matters) and 1 (only learning matters). The level of fatigue is also affected by two processes. The first is that when one is working, the level of fatigue increases56. Here, we use the simplest form of this process, a linear increase. The second is that when one is resting, the level of fatigue recovers to baseline, zero. Here, we draw on the literature on recovery (e.g.,ref.28), which has found that recovery tends to involve an exponential decay in fatigue. These dynamics of accumulating and decaying fatigue are mathematically integrated into the following conditional function, This differential equation formulation is equivalent to a linear increase in time when working, and exponential decay when resting. An individual's decision to work or rest depends on their motivation and fatigue57,58. An individual will switch to working from resting when they have sufficient motivation relative to their fatigue—that is, when M(t) − F(t) increases across a critical threshold, θ(t), which increases with the difference between task difficulty and skill level, while limited to a minimum value, \(\theta (t)=\max (T(t)-S(t),{\theta }_{min})\). The minimum θmin reflects the minimum motivational start-up cost of starting to work. Individual stops working when their fatigue is equal to or greater than their motivation—that is, when M(t) − F(t) is smaller than or equal to zero. A sketch of this dynamic is shown in Fig. In discrete time, the function p(t) can be expressed as where Δt is a small time interval. The activation threshold should be lower for individuals at higher skill levels relative to task difficulty. In other words, it is easier for an individual to start working on a task when they are good at it. We first analyze a baseline scenario where an individual experiences constant task difficulty over time, illustrated in Fig. Real-world examples of this constant scenario are ubiquitous, at least within delimited time periods (e.g., lifting a certain weight, playing a card game against a fixed computer opponent). Note that this scenario is analogous to the regime illustrated in the inset of Fig. We assume, moreover, that the individual is motivated equally by learning and performance; in the last section of the Results, we weaken this assumption and explore the effects of individual differences in motivation. Constant task difficulty leads to plateauing skill and performance on the long timescale (A, B), and reduced motivation and working periods on the short timescale (C–E). A Skill (red solid line) approaches the task difficulty (blue dashed line), and eventually plateau. B Performance of an individual (S(t)E(t)), increases over time but eventually plateau. C, D Motivation and fatigue oscillate across recurring working sessions, where individual enter a work session when motivation is sufficiently greater than fatigue and exit the work session when motivation drops below fatigue. Peak motivation value of each work session diminishes as one's skill and performance plateau. E The durations of an individual's working period and rest period both decline over time, consistent with the decreasing motivation. Note that in the absence of some additional mechanism for sustaining engagement, the individual initiates work but immediately abandons the task, as it is no longer sufficiently motivating to sustain work for more than a moment. In the real world, an individual at this point would likely abandon the task entirely or would require some other mechanism to sustain their engagement, whether endogenous to the individual (e.g., an explicit dedication to persisting despite the lack of reward) or exogenous (e.g., a supervisor or coach who encourages sustained work). In this constant scenario, our model recovers the classic result that long-timescale skill development reaches a plateau of both skill and performance, as shown in Fig. The model thus generates the classic empirical pattern of diminishing returns in skill learning when task difficulty is held constant12,14,15,16. In our model, this pattern emerges naturally from short-timescale dynamics of motivation, fatigue, and engagement. The model also predicts that long-timescale skill acquisition should be accompanied by changes in short-timescale dynamics. As skill improves, both peak motivation and peak fatigue within working sessions decline monotonically (Fig. Consequently, one tends to work for shorter and shorter periods, suggesting an increasing degree of boredom. The apparently static “plateau” in skill development at the macro-scale is actually the product of an intricate dynamic balance of micro-scale processes. Indeed, in the real-world, the decision to begin working reflects not just the endogenous influences of motivation and fatigue captured by our model, but also exogenous influences such as diurnal cycles (i.e., people are not going to start working while sleeping). Moreover, individuals who are insufficiently motivated by a task will likely require exogenous influences to persist in working—a workplace supervisor or a coach, for instance. Such exogenous influences are beyond the scope of our minimal model. In the absence of such external pressures, individuals faced repeatedly with a task that is so unrewarding that it can only sustain their engagement for a moment would likely abandon the task for something more challenging and thus rewarding. The baseline scenario explored in Fig. 3 does not allow for this kind of adjustment; indeed, we assume for simplicity that individuals are forced to repeatedly attempt a task that is no longer rewarding. In subsequent sections, we relax this assumption to explore scenarios where the task is dynamically adjusted or even abandoned entirely for a new, more challenging task. In real-world settings, individuals tend to engage in increasingly difficult tasks throughout the long-term learning process. To capture this, we evaluated a scenario where task difficulty increases in discrete jumps whenever an individual's skill catches up with task difficulty, as shown in Fig. Real-world situations corresponding to this situation include strength training with incrementally heavier weights, attempting a new level in a video game, or skipping a grade in school. We set task difficulty to increase by a fixed increment of 0.8 whenever the gap between an individual's skill and the demands of the task (i.e., T − S) is smaller than a threshold, ϵ. A Model predictions with stepwise adaptive task difficulty. The model recovers well-established patterns of skill acquisition, performance, and engagement. Note that the qualitative patterns shown here remain robust to variations in the main parameters (see SI for details). B Model predictions with linearly increasing task difficulty at a slow pace. The learner adapts to increasing task difficulty by improving skill. C Model predictions with linearly increasing task difficulty at a faster pace. Eventually the learner cannot keep up with the fast pace (C1), no longer recovers motivation (C3), and thus stops working entirely (C5). In all panels, parameter w = 0.5. Within each period of constant task difficulty, behavior was similar to that found in the constant scenario, with plateaus in skill and performance as skill approached task difficulty. With repeated, discrete increases in task difficulty, however, both skill and performance surpassed these plateaus (Fig. Our model predicts that when the task difficulty is raised adaptively, one's performance exhibits plateaus, dips, and leaps, as shown in Fig. Whenever task difficulty increases after reaching a plateau, performance first dips, then increases to exceed the previous plateau. This qualitative pattern is in line with empirical studies of performance plateaus22,23. Similar to the model's predictions in the constant scenario, each episode of task difficulty level starts with a spike in peak motivation, a spike in peak fatigue, and longer duration of working periods. This suggests intense engagement as one first attempts to master a higher task level. Peak motivation, peak fatigue, and working period duration all decline as skill improves and approaches the target task difficulty, essentially recapitulating what we find in the constant scenario (Fig. Note that while we assume specific values for the timescale parameters c1 through c4 in our simulations, the qualitative findings remain robust to variations in these parameters (see Supplementary Information for details). To incorporate activities that are not characterized by stepwise change of difficulty, we also examine a scenario where the task becomes increasingly more challenging in a continuous manner (e.g., a gradually increasing speed of running and cycling (Fig. We observe a qualitatively similar pattern in skill learning and performance compared to that in the stepwise scenario—both scenarios induce sustained increases in skill and performance over the long timescale, with the continuous scenario producing smoother changes. When the rate of increase in task difficulty is not overwhelming, there is a gradual increase in peak motivation, peak fatigue, and the duration of working periods (Fig. This indicates increased levels of engagement. This contrasts with what was found in the constant scenario, where peak motivation, peak fatigue, and work period duration all decreased as skill increased (Fig. Both the stepwise and continuous scenarios can lead to sustained increase in skill learning and performance, but how far can one push the pace of task difficulty increase? We evaluated a special case in the continuous scenario where we increase the task difficulty at a very high rate (Fig. In this scenario, individuals give up after failing to keep up with the ever more challenging task (Fig. Notably, on short timescales, this special case is characterized by a substantial increase in both peak fatigue (Fig. 4C4) and the duration of resting periods (Fig. These short-timescale dynamics suggest the task became too demanding, with the individual taking longer recover, without sufficient motivation to compensate for the escalating exertion. We next analyze the impact of individual differences in motivation on the long-term development of skill and performance. Research in education and social psychology has found that individuals differ in their relationship to challenging learning opportunities: While some individuals are intrinsically motivated to seek out new challenges, others prefer situations where they can demonstrate their existing mastery59,60,61. One study, for instance, found that students who were more intrinsically motivated were more likely to choose a more challenging task, compared to students who were more competent but less intrinsically motivated60. We thus sought to simulate individual differences in learners' source of motivation and choice of task. We thus manipulated whether individuals were primarily motivated by learning (i.e., rate of increase of skill) or performance. (5), learning-driven individuals (i.e., high w value) derive proportionally more motivation from their skill development (dS(t)/dt), while performance-driven individuals (i.e., low w value) derive proportionally more motivation from their rate of execution (E(t)). These two scenarios, therefore, capture two behavioral profiles that have been established in the literature on intrinsic motivation and challenge-seeking. To capture the finding that intrinsically-motivated individuals are also more likely to seek out more challenging tasks60, we allowed ϵ—the threshold for increasing task difficulty in our stepwise scenario—to depend on whether individuals are motivated more by learning or performance: \(\epsilon ={\epsilon }_{\min }+kw\), where \({\epsilon }_{\min }=0.005\), and the slope k = 0.01. For simplicity, in exploring these individual differences, we chose a linear relation between source of motivation (w) and the threshold to seek out a more challenging task (ϵ), although more complicated nonlinear relationships are possible. 5, since performance-driven individuals are slower to attempt more difficult tasks, they suffer fewer dips in performance than learning-driven individuals. This reflects the learning-driven individual's drive to seek out more challenging tasks, even at the cost of short-term dips in performance. The performance of learning-driven individuals, therefore, may repeatedly drop below the performance of performance-driven individuals. This short term deficit, however, is overcome by the learning-driven individuals' improved skill, so that eventually learning-driven individuals perform better than performance-driven individuals, even when the learning-driven individual is experiencing a dip in performance while the performance-driven individual is at their peak. A Skill learning in individuals motivated by skill (w = 1; solid line) or performance (w = 0; dashed line). B Performance in individuals motivated by skill (w = 1; solid line) or performance (w = 0; dashed line). When individuals are learning-driven, they are quicker to attempt more difficult tasks, compared to performance-driven individuals. They thus achieve a higher skill and better performance over the long run. Life-long learning involves non-linear change on multiple timescales. Past research has established a number of classic patterns in these dynamics, patterns that strikingly recur across varied domains9,22,23,62. Motivation waxes and wanes from moment to moment27, skill can develop gradually over days or years1,2, and performance can rise and dip as new challenges are attempted23. Our formal model offers a framework for explaining these known patterns, including how dynamics on one timescale can shape or generate dynamics on another. Our model offers a parsimonious account of several important phenomena in learning. Learning and performance, for instance, are known to exhibit gradual increases, static plateaus, or short-term dips22,23,63. In our model, these long-timescale behaviors emerge from short-timescale fluctuations in motivation, fatigue, and effort. Seemingly static plateaus in long-timescale learning—a recurring pattern in studies of skill acquisition11,13,14,23—may thus reflect ongoing dynamic interactions between micro-level processes at short-timescales. Moreover, the peaks and troughs of motivation and fatigue—and thus the amount of time individuals are able to sustain their effort—reflect individual differences in whether one is motivated by intrinsic growth (i.e., skill development) or extrinsic achievement (i.e., performance)62,64. The model thus offers a formal account of how individual differences in motivation may relate to long-term patterns of learning. By situating these dynamics within different training regimes—that is, different plans for increasing the task difficulty over time—we investigated how people can achieve life-long learning or fail to maintain their personal growth. Our approach thus offers a general, task-agnostic framework for understanding the multiscale dynamics of learning across varied domains. Our model does not distinguish between the kind of task (such as between mental and physical skills) and captures the broad-stroke phenomena in motivation, fatigue, and skill learning that transcend specific activities. The model thus offers a formal account of why similar patterns recur across domains. This framework, however, can be modified to account for other aspects of the learning process, including task-specific factors that may play a role only within particular contexts (e.g., extrinsic encouragement from a coach or comrade), or distinct modes or types of learning that may occur during different stages of the learning process (e.g., declarative vs. procedural learning65). By formalizing key aspects of lifelong learning, we offer a theoretical foundation on which other more elaborated accounts can build. In our model, individuals are motivated in part by the gap between their current skill and the difficulty of the task (T − S). As a result, individuals do not work at all if the task is too difficult (Fig. 4C1), they experience peak motivation when the task is appropriately difficult relative to their current skill, and motivation decreases as task becomes relatively boring. In other words, when the task's challenge is aligned with the individual's skill—not too difficult (Fig. 4C1) but not too easy (Fig. 3)—then individuals experience high motivation, the capacity to work for extended periods of time, and the capacity to sustain their efforts in the face of high fatigue (Fig. These emergent attributes of our simulated work are hallmarks of deep, sustained engagement. They resemble features of the phenomenon of “flow,” a state of mind that typically occurs when a task is sufficiently difficult—not too difficult to make progress but also not too easy to offer a challenge62. Flow experiences consist of a cluster of features, including heightened motivation and an ability to work for long periods despite fatigue62,64. This pattern shares striking similarities with the emergent dynamics of work, rest, and motivation in our model, particularly when the task difficulty was neither too challenging nor too easy (e.g., Fig. This raises the possibility—admittedly speculative—that the minimal mechanisms that are formalized in our model may partially explain why flow states consist of the particular cluster of features with which they are associated62. While episodes of deep engagement, such as flow states, have traditionally been discussed as a phenomenon that emerges and disappears on short timescales—emerging during a single session of work, or reflecting the fit between current activities and goals66—our model offers a framework for situating the short-timescale dynamics of deep, sustained engagement within the long-timescale dynamics of skill development, performance, and task selection. When task difficulty increases in discrete jumps, for instance, individuals experience brief periods of deep engagement, which disappear as their skill catches up to the demands of the task, at which point the task is no longer sufficiently challenging (Fig. When the difficulty of the task is increased continuously, however, deep engagement can be sustained and even increased over time (Fig. While our minimal model does not speak to the subjective, phenomenological features of deep, sustained engagement, it offers a parsimonious account of the coupled dynamics of motivation, work, and fatigue on both short- and long-timescales. The emergence of deep, sustained engagement in our model depends on the source of an individual's motivation. When individuals are intrinsically motivated by learning (i.e., w close to 1), they sacrifice short-term gains in performance for the opportunity to improve their skill (Fig. This leads to frequent dips in performance. Over time, however, the benefits of long-term cumulative learning outweigh the costs of these temporary performance dips induced at the onset of new skill acquisition. This is a phenomenon that has been described empirically in skill acquisition ranging from the discovery of the Fosbury Flop technique in the sport of high jump (initial adoption of the Fosbury Flop can lead to a drop in performance before the technique is mastered), to the development of prodigious memory for a span of random digits23. Since performance dips are followed by periods of increased challenge and deep engagement, our model captures the notion of the so-called autotelic personality type, characterized by a disposition to actively seek challenges and deep engagement67. The long-timescale dynamics of learning thus depend on the moment-to-moment dynamics of motivation and task selection. Maximizing long-timescale learning, however, requires a careful balance. For individuals to experience sustained, long-term learning, they must engage in challenges above their current skill level, but only to a degree. If individuals choose to maximize their current performance rather than seek out new challenges (such as the performance-driven individuals in Fig. 5), then their long-term learning will progress more slowly. But if task difficulty is increased too quickly, then learning cannot keep up and eventually an individual will cease to learn, perform, and work (Fig. When the increase in learning is carefully calibrated to an individual's ability (Fig. 4A, B), then learning and peak motivation can both grow over long periods68,69,70. The decision to attack entirely new tasks, or the discovery of a more difficult but potentially more effective approach to an existing task, can produce learning curves that consist of piece-wise power laws22. Here, our model reproduces this long-timescale dynamic when the task difficulty is increased in discrete jumps (Fig. 4A), with each period of decreasing returns followed by a period of renewed rapid growth. In our model, however, each period of decreasing returns actually follows an exponential rather than power-law learning curve, in line with work showing that individual learning curves follow an exponential form14,20,71. As we show analytically in the Supplementary Information, this exponential decay in the rate of learning emerges from our assumption that the rate of learning depends only on relative task difficulty (i.e., the difference between task difficulty and skill, T − S). In order for learning to follow a power law, the rate of learning must depend not only on the amount of current challenge but also on the passage of time itself (e.g., decaying as a function of the number of practice trials). The difference between exponential and power laws of practice, therefore, reduces to whether the classic phenomenon of diminishing returns depends only on changes in skill (i.e., diminishing as skill approaches saturation), or on both skill and time (e.g., also diminishing with each attempt20). We speculate, therefore, that the functional form of an individual's learning curve may depend on the individual and the task. In cases where the rate of learning decays with time (e.g., with an individual who just tries less over time), then learning should follow a power law; in cases where the rate of learning depends only on the current challenge, then learning should follow an exponential curve. In the following paragraphs, we briefly summarize our work and highlight our contributions. Overall, skill mastery is the outcome of processes that operate on widely varying timescales. Motivation and fatigue can change rapidly in response to context and activity, decisions to work or rest are based on fluctuations in motivation and fatigue, while long-term improvements in skill and performance reflect days or years of work. Our work offers two contributions. First, our model offers a formal mathematical framework to study the interaction of these timescales, and to understand the multiscale dynamics of learning as an integrated dynamical system. The stability of long-term dynamics (e.g., skill plateaus), for instance, emerges from dynamic and intricate interactions on shorter timescales, and the functional form of long-timescale learning curve depends on the instantaneous rate of learning. The influence goes in the other direction, too—different long-timescale regimes for increasing task difficulty, for instance, produce different short-timescale dynamics of motivation and fatigue. While common sense suggests that there must be interactions between processes on these different timescales, our model spells out some of those potential interactions. Second, our work brings into conversation different research areas that are typically investigated in isolation. Research on life-long mastery is seldom connected to the small improvements possible during a session of practice. This paper is an attempt to bridge these timescales. The model has a number of limitations. First, since our goal was to capture domain-general patterns in skill acquisition, we have deliberately ignored task-specific constraints on skill acquisition. While learning chess and learning to juggle may share high-level, qualitative patterns, they differ of course in countless ways. Future work should incorporate task-specific considerations—such as limitations of the body, or task-specific strategies—to develop formal accounts of learning within specific domains. Second, our model does not currently account for interactions between different skills and activities. Many contemporary workplaces, for instance, require individuals to adapt to job requirements that change over time, such that workers must replace old skills with new ones, or even develop multiple new skills in parallel. Future work should explore the ways in which acquiring one skill may help, or even hinder, the acquisition of another, and how the pursuit of multiple skills in parallel can be synergistic or detrimental. As stated in the introduction, our mathematical and computational model answers the call for theory building in psychological science4 by offering a unified framework for theorizing about the multiple timescales on which human learning unfolds. In so doing, we hope our work encourages scientists working in separate disciplines to start a collective conversation about the complex, multiscale dynamics of skill learning. Additional details on mathematical derivations are included in Supplementary Information. The code used in the simulation, as well as data generated from the simulations, are available as supplementary files. Newell, K. M., Liu, Y.-T. & Mayer-Kress, G. Time scales in motor learning and development. Multiple time scales and multiform dynamics in learning to juggle. & Thelen, E. Development as a dynamic system. & Martin, A. E. How computational modeling can force theory building in psychological science. Borsboom, D., van der Maas, H. L., Dalege, J., Kievit, R. A. & Haig, B. D. Theory construction methodology: a practical framework for building theories in psychology. Smaldino, P. E. How to build a strong theoretical foundation. Marquet, P. A. et al. On theory in ecology. Kadanoff, L. P. Scaling and universality in statistical physics. Ericsson, K. A., Krampe, R. T. & Tesch-Römer, C. The role of deliberate practice in the acquisition of expert performance. Gladwell, M. Outliers: The Story of Success (Little, Brown, 2008). Simon, H. & Chase, W. Skill in chess. & Lehmann, A. C. Expert and exceptional performanceevidence of maximal adaptation to task constraints. Fitts, P. M. Human Performance (Belmont, California: Brooks/Cole Pub. Estes, W. K. The problem of inference from curves based on group data. & Rosenbloom, P. Mechanisms of skill acquisition. Anderson, J. R. & Fincham, J. M. Acquisition of procedural skills from examples. Stafford, T. & Dewar, M. Tracing the trajectory of skill learning with a very large sample of online game players. Gallistel, C. R., Fairhurst, S. & Balsam, P. The learning curve: implications of a quantitative analysis. Why aggregated learning follows the power law of practice when individual learning does not: Comment on Rickard (1997, 1999), Delaney et al. (1998), and Palmeri (1999). The power law repealed: the case for an exponential law of practice. Toward an explanation of the power law artifact: insights from response surface analysis. & Hardy, J. L. Piecewise power laws in individual learning curves. Gray, W. D. & Lindstedt, J. K. Plateaus, dips, and leaps: where to look for inventions and discoveries during skilled performance. van der Maas, H. L. & Molenaar, P. C. Stagewise cognitive development: an application of catastrophe theory. & van Ginkel, S. The added benefit of an extra practice session in virtual reality on the development of presentation skills: a randomized control trial. Mirzaei, M., Ghadiri, F., Arsham, S. & Rajabi, H. The effect of seven sessions of training program with massed and distributed practice frequency on the acquisition of cycling skills in autistic and healthy boy in tehran city. & Thompson, D. E. Work motivation: theory and practice. Konz, S. Work/rest: Part II—the scientific basis (knowledge base) for the guide. van der Steen, S., Steenbeek, H. W., Den Hartigh, R. J. & van Geert, P. L. The link between microdevelopment and long-term learning trajectories in science learning. & Cnossen, F. The role of motivation as a factor in mental fatigue. Hopstaken, J. F., Van Der Linden, D., Bakker, A. A multifaceted investigation of the link between mental fatigue and task disengagement. Hastings, A. Timescales, dynamics, and ecological understanding. Hastings, A. Timescales and the management of ecological systems. Flack, J. C. Multiple time-scales and the developmental dynamics of social systems. Wagner, L. M. & Clifton, S. M. Modeling the public health impact of e-cigarettes on adolescents and adults. Haken, H., Kelso, J. S. & Bunz, H. A theoretical model of phase transitions in human hand movements. Richardson, M. J., Marsh, K. L., Isenhower, R. W., Goodman, J. R. & Schmidt, R. C. Rocking together: dynamics of intentional and unintentional interpersonal coordination. A model balancing cooperation and competition can explain our right-handed world and the dominance of left-handed athletes. Yang, V. C., Abrams, D. M., Kernell, G. & Motter, A. E. Why are us parties so polarized? Leonard, N. E., Lipsitz, K., Bizyaeva, A., Franci, A. The nonlinear feedback dynamics of asymmetric political polarization. McChesney, G. The psychology of efficiency. Learning versus performance: an integrative review. Measurement of functional task difficulty during motor learning: what level of difficulty corresponds to the optimal challenge point? & Lee, T. D. Challenge point: a framework for conceptualizing the effects of various practice conditions in motor learning. Motivation on a Per-Second Timescale: Examining Approach- Avoidance Motivation During L2 Task Performance. In Dörnyei Z, MacIntyre P. & Henry A. (eds) Motivational Dynamics in Language Learning. Bristol, 109–138 Blue Ridge Summit: Multilingual Matters. Aragonés, D., Eekhoff, A., Horst, F. & Schöllhorn, W. I. Fatigue-related changes in technique emerge at different timescales during repetitive training. Burdack, J., Horst, F., Aragonés, D., Eekhoff, A. & Schöllhorn, W. I. Fatigue-related and timescale-dependent changes in individual movement patterns identified using support vector machine. Martin, K., Meeusen, R., Thompson, K. G., Keegan, R. & Rattray, B. Mental fatigue impairs endurance performance: a physiological explanation. Palmer, L. The relationship between stress, fatigue, and cognitive functioning. Coa, K. & Patrick, H. Baseline motivation type as a predictor of dropout in a healthy eating text messaging program. Ondersma, S. J., Winhusen, T., Erickson, S. J., Stine, S. M. & Wang, Y. Atkinson, J. W. Motivational determinants of risk-taking behavior. Anderman, E. M. Achievement motivation theory: balancing precision and utility. Massar, S. A., Wester, A. E., Volkerts, E. R. & Kenemans, J. L. Manipulation specific effects of mental fatigue: evidence from novelty processing and simulated driving. Nijp, H. H., Beckers, D. G., Kompier, M. A., van den Bossche, S. N. & Geurts, S. A. Worktime control access, need and use in relation to work-home interference, fatigue, and job motivation. Kumalasari, L. D. & Elgeka, H. W. S. The mediating role of intrinsic motivation in the relationship between organizational support and quantitative workload and work-related fatigue. Dweck, C. S. & Leggett, E. L. A social-cognitive approach to motivation and personality. Inoue, N. Why face a challenge? : the reason behind intrinsically motivated students' spontaneous choice of challenging tasks. Abuhamdeh, S. & Csikszentmihalyi, M. The importance of challenge for the enjoyment of intrinsically motivated, goal-directed activities. Csikszentmihalyi, M. Flow: The Psychology of Optimal Experience (Harper & Row, New York, 1990). Csikszentmihalyi, M. Beyond Boredom and Anxiety (Jossey-Bass Publishers, 1975) https://books.google.com/books?id=afdGAAAAMAAJ. An integrated theory for improved skill acquisition and retention in the three stages of learning. Chow, J.-Y., Renshaw, I., Button, C. & Davids, K. Effective learning design for the individual: a nonlinear pedagogical approach in physical education. In (eds Ovens, A., Hopper, T. & Butler, J.) Complexity Thinking in Physical Education (Routledge, 2012). Son, L. K. & Sethi, R. Metacognitive control and optimal learning. Headrick, J., Renshaw, I., Davids, K., Pinder, R. A. & Araújo, D. The dynamics of expertise acquisition in sport: the role of affective learning design. & Tweney, R. D. Artifactual power curves in forgetting. We thank the Santa Fe Institute (SFI) and the James S. McDonnell Foundation (JSMF) for jointly organizing the conference that sparked many of the ideas explored in this paper. We thank Hilary Skolnik (SFI) and Susan Fitzpatrick (JSMF) for organizing these conferences, as well as all our colleagues and employees at the Santa Fe Institute for their support. Additionally, we thank the Omidyar Fellowship for its support in making this research possible. These authors contributed equally: Mingzhen Lu, Tyler Marghetis, Vicky Chuqiao Yang. Department of Environmental Studies, New York University, 285 Mercer Street, New York, NY, 10003, USA Institute for Data, Systems, and Society, Massachusetts Institute of Technology, Cambridge, MA, 02142, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Mingzhen Lu, Tyler Marghetis or Vicky Chuqiao Yang. The authors declare no competing interests. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. A first-principles mathematical model integrates the disparate timescales of human learning. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02659-z'>A tissue-specific atlas of protein–protein associations enables prioritization of candidate disease genes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 10:16:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite progress in mapping protein–protein interactions, their tissue specificity is understudied. Here, given that protein coabundance is predictive of functional association, we compiled and analyzed protein abundance data of 7,811 proteomic samples from 11 human tissues to produce an atlas of tissue-specific protein associations. We find that this method recapitulates known protein complexes and the larger structural organization of the cell. Interactions of stable protein complexes are well preserved across tissues, while cell-type-specific cellular structures, such as synaptic components, are found to represent a substantial driver of differences between tissues. We validate protein associations for the brain through cofractionation experiments in synaptosomes, curation of brain-derived pulldown data and AlphaFold2 modeling. We also construct a network of brain interactions for schizophrenia-related genes, indicating that our approach can functionally prioritize candidate disease genes in loci linked to brain disorders. Protein–protein interactions mediate the biophysical structure and functioning of the cell, with disruptions potentially leading to disease. Unraveling the interactions of human proteins has been a long-standing challenge1, with recent advances in mass spectrometry (MS) coupled with high-throughput screens such as protein-fragment complementation2, yeast two-hybrid3, affinity purification (AP)4,5 or cofractionation6,7,8 enabling the discovery of interactions on a proteome-wide scale. These experimental datasets are complemented by modern computational approaches that compile collections of interactions using machine-learning-based methods often trained with experimental evidence1. Together, these experimental and computational efforts are the basis for numerous publicly available, high-quality protein interaction databases1,9,10,11. However, these databases typically aggregate interactions without specifying context, whereas the interactome is highly tissue or cell state specific, with less than half of the proteome being detected in all tissues12. Elucidating the tissue specificity of protein interactions is important for understanding cell-type-specific function, finding drug targets and developing a system-level understanding of the human cell. Indeed, specifying tissue specificity of the interactome has proven difficult. Initial attempts to generate tissue-specific interaction networks relied on gene expression data to identify coexpressed genes or to exclude proteins on the basis of a lack of expression in a given tissue13,14. However, the accuracy of predicting protein associations by mRNA coexpression is limited and it is unclear to what extent gene expression is the major driver of changes in protein interactions15,16,17,18. While there are some recent experimental efforts for establishing tissue-specific interactions on a proteome-wide scale4,6, such approaches require extensive resources even for a single tissue and often use immortalized cell lines or other models whose interactome may not appropriately represent the human tissue. One approach used changes in ratios for complex subunits from proteomic measurements to establish changes in protein interactions in a disease state19. An alternative approach used the coabundance of proteins for the purpose of establishing ‘protein associations'5,17,20,21. Here, associations are used instead of interactions to reserve the latter for physical interactions. Coabundance has been shown to be an accurate measure for predicting protein–protein associations likely because of the fact that protein complexes consist of subunits assembled in defined stoichiometries that are often coexpressed, with orphaned subunits often being degraded15,22,23,24,25. The accuracy of coabundance for predicting protein association and the rising number of large-scale proteomics studies of human cancers, with genetic heterogeneity underlying diverse cellular changes, offer a timely opportunity to systematically establish the tissue specificity of protein associations. Here, we present a protein association atlas derived from the coabundance of proteins in 7,811 human biopsies, scoring the likelihood of 116 million protein associations across 11 tissues. This atlas recovers well-known protein complexes, maps relationships between and across traits and cellular components and proposes context-relevant associations for disease genes. Focusing on the brain and using interactions derived from orthogonal approaches, we illustrate the use of our association atlas for prioritizing genes and interactions of disease genes in a tissue-specific manner. We started by collecting protein abundance data from proteomics studies of cohorts of participants with cancer. In total, we compiled a dataset of 50 studies across 14 human tissues, encompassing 5,726 samples of tumors and 2,085 samples of adjacent healthy tissue26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75 (Fig. Following previous studies, we used the fact that protein complex members are strongly transcriptionally and post-transcriptionally coregulated to compute probabilities of protein–protein associations from the abundance data5,17,20 (Fig. 1b, Methods and Supplementary Fig. In short, we preprocessed the abundance data to obtain a log-transformed and median-normalized abundance across participants. For each study, we then computed a coabundance estimate of a protein pair as the Pearson correlation when both proteins were quantified in at least 30 samples (Supplementary Fig. Lastly, with pairs of subunits for curated stable protein complexes as ground-truth positives (CORUM76), we used a logistic model for each study to convert the coabundance estimates to probabilities of protein–protein associations (Supplementary Figs. a, Number of tumor and healthy samples per tissue. Bar sections indicate individual studies, using multiplexed proteomics with isobaric labeling (dark blue) or other methods (light blue). b, Schematic representation of workflow. Subunits of protein complexes occur in fixed stoichiometries. Protein coabundance is estimated through correlation of protein abundance profiles and converted to probabilities through a logistic model using interactions between subunits of protein complexes (CORUM) as positives. c, ROC curves for association probabilities in lung tissue derived from protein coabundance (coabund. ; blue), mRNA coexpression (coexpr. ; orange) and protein cofractionation (cofrac. The gray dashed line shows the performance of a random classifier. d, AUC values for association probabilities as illustrated in c. Shown are studies that quantified both protein coabundance (blue; n = 29) and mRNA coexpression (orange; n = 29) or protein cofractionation (green; n = 10). e, AUC values for association probabilities derived from protein coabundance combined with mRNA coexpression through a linear model (purple) and protein coabundance after regressing gene expression out of the protein abundance (pink). Shown are the same studies as in d. In d,e, each dot represents one study with paired transcriptomics and proteomics data. Protein pairs were filtered for having association probabilities from both modalities. Error bars show the mean and s.e.m. In c–e, negatives are all quantified protein pairs not reported as complex members. f, Clustering of the n = 48 cohorts using association probabilities of protein pairs with the most variable associations (CV above the median). The radial dendrogram shows complete-linkage clustering with the Pearson correlation distance. Cohorts are labeled according to the type of cancer; colors represent the different human tissues. Leaf-joint distances were shortened. g, Heat map of AUCs for recovering tissue-specific associations with cohorts that were withheld when predicting these associations. Each square represents the average AUC for all cohorts of a given tissue. Tissues were clustered through complete-linkage clustering with the Manhattan distance. To test the ability of the association probabilities to recover known complex members, we computed receiver operating characteristic (ROC) curves for probabilities derived from protein coabundance, mRNA coexpression and protein cofractionation6,7,77 (Fig. We found that protein coabundance (area under the curve (AUC) = 0.80 ± 0.01 (mean ± s.e.m.)) outperformed protein cofractionation (AUC = 0.69 ± 0.01) and mRNA coexpression (AUC = 0.70 ± 0.01) data for recovering known interactions (Fig. In addition, the combination of mRNA and protein abundance data did not significantly improve the recovery of known complex members (Fig. 1e; AUC = 0.82 ± 0.01, P = 0.15, according to a one-sided Welch's t-test). Therefore, with roughly half of all cohorts having paired mRNA expression data available, we chose to only use protein coabundance for computing association probabilities. Additionally, we found similar AUCs when regressing gene expression out of the protein abundance before computing protein coabundance estimates (AUC = 0.78 ± 0.01, P = 0.18), suggesting that post-transcriptional processes but not regulation of gene expression drive most of the predictive power for protein associations. Having established that the association probabilities derived from protein coabundance data recover known interactions of protein complex members, we sought to test whether replicate studies of the same tissue yielded association probabilities that were representative for each tissue. As a starting point, we used the gene expression data to establish that the association probabilities were not driven by cell-type composition78 (Supplementary Fig. Next, using the 1,115,405 association probabilities that were quantified for all studies, we found that the replicate cohorts from the same tissue generally clustered together (Fig. 1f; for example, blood, brain, liver and lung). Next, we selected the associations that were tissue specific, that is associations whose average probability exceeded the 95th percentile for a given tissue (0.68 ± 0.01 across tissues) and whose average probability remained below 0.5 across all other tissues. Through a hold-one-out methodology, we found that the tissue-specific associations were primarily recovered by cohorts of the same tissue of origin (AUC = 0.71 ± 0.01) compared to cohorts from different tissues (AUC = 0.56 ± 0.00, P < 0.05 for all tissues, according to a one-sided Welch's t-test) (Fig. 1g, Methods and Supplementary Fig. Together, these observations suggest that the tissue of origin is a major driver of differences between cohorts. With the replicate cohorts representing the tissue of origin, we aggregated the association probabilities from cohorts of the same tissue into single association scores for 11 human tissues (Fig. Aggregating the replicate cohorts was advantageous, as all but one of the individual cohorts were outperformed by the tissue-level scores for recovering known protein interactions (P = 1.3 × 10−9, according to a one-sided Welch's t-test). Moreover, the tumor-derived scores outperformed the healthy-tissue-derived scores for all tissues (Fig. 2b; AUC = 0.87 ± 0.01 and 0.82 ± 0.01, respectively, P = 8.3 × 10−5, according to a one-sided Welch's t-test). In addition to the biopsy, where the genetic heterogeneity of tumors increased variation between samples (Supplementary Fig. 8), we found several other factors affecting the recovery of known interactions, such as the available number of cohorts per tissue, the number of samples per cohort, the tissue of origin and the MS methodology (Supplementary Figs. The healthy-tissue-derived and tumor-derived scores originated from separate dissections of the same tissues and participants and could, thus, serve as independent replicates. Analogous to the cohorts, we computed tissue-specific associations from the healthy-tissue-derived scores, which we then recovered with the tumor-derived scores (Fig. For all tissues, we found that the tumor-derived scores primarily recovered the tissue-specific associations of the same healthy tissue (AUC = 0.74 ± 0.02) compared to the other healthy tissues (AUC = 0.53 ± 0.01, P = 5.9 × 10−5, according to a one-sided Welch's t-test). These analyses show that the coabundance-derived tissue-level association scores recover known protein interactions and are reproducible and representative of the tissue of origin (Supplementary Fig. b, AUC values for the association scores derived from healthy samples (green; n = 6) and tumor samples (blue; n = 11), using interactions between subunits of protein complexes (CORUM) as positives. Association scores were filtered for protein pairs having probabilities in all cohorts of a tissue. c, Heat map of AUCs for using tumor-derived association scores to recover tissue-specific associations defined by the association scores from healthy tissues. Association scores only include cohorts that had both healthy and tumor samples. Tissues were clustered through complete-linkage clustering with the Manhattan distance. d, Atlas of protein associations in n = 11 human tissues. The radial diagram shows, for each tissue, the numbers of protein pairs that were quantified (gray), are likely to interact (light green; association score > 0.5) or were confidently quantified (dark green; association score > 0.8). The bar graph shows the number of associations that were quantified in the given number of tissues. e, Probability of associations of a tissue to likely be in a healthy-tissue-derived replicate (orange; n = 12) or between pairs of tissues (green; n = 110) as a function of threshold association score. Scores only include protein pairs quantified for both tissues or replicates. Shown is the median probability across pairs of replicates or tissues. f, Likely associations shared between pairs of tissues as quantified by the Jaccard index (gray dots), compared to shared associations restricted to complex members (CORUM), physical associations (STRING scores > 400), biological pathways (Reactome) and signaling (SIGNOR) (purple dots) or associations detected through yeast two-hybrid (HuRI) or AP (BioPlex) experiments (blue dots). Each dot represents a pair of tissues. Error bars show the mean and s.e.m. We defined a protein association atlas with association scores for all quantified protein pairs by averaging the association probabilities over the cohorts of each tissue. The resulting association atlas scores the association likelihood for 116 million protein pairs across 11 human tissues (Fig. On average, each tissue contains association scores for 56 ± 6.2 million protein pairs, of which 10 ± 1.0 million are likely to be associated (score > 0.5, average accuracy = 0.81 over all tissues, recall = 0.73 and diagnostic odds ratio = 13.0) and 0.49 ± 0.08 million are ‘confident' associations (score > 0.8, average accuracy = 0.99 across tissues, recall = 0.21 and diagnostic odds ratio = 31.9) (Supplementary Fig. These protein associations tended to be likely and confident in only a few tissues, with 99,103 protein pairs having likely associations in all tissues (Fig. One of the well-known drivers of differences in protein interactions between tissues is gene expression; proteins can interact only if their gene is expressed in a tissue. Indeed, the proteins that were quantified in a given tissue were generally enriched for genes with elevated expression for that same tissue but not the other tissues (Supplementary Fig. 13; P = 1.3 × 10−6, according to a one-sided Mann–Whitney U-test). However, only up to 7% of differences in (likely) associations between tissues can be explained by differences in gene expression and only through the lack of detection (Supplementary Fig. These observations demonstrate that the likely associations for each tissue reflect but are not defined by differences in gene expression, further supporting our previous observation that protein coabundance is primarily driven by post-transcriptional processes. Having established that association scores generally reproduce well and that differences between tissues are not driven by gene expression, we sought to measure the share of tissue-specific associations. To do so, we used a threshold association score to quantify the percentage of a tissue's associations that were likely (score > 0.5) for the replicate (Fig. 2e, orange curve, comparing healthy-tissue-derived and tumor-derived replicates). As expected, we found that the percentage of likely associations increases with the threshold score, with 46.3% of likely associations and 90.2% of confident associations (score > 0.8) also being likely for the replicate tissue. Similarly, we found that these percentages decreased to 32.9% and 54.6%, respectively, when comparing associations between pairs of tissues from the association atlas (Fig. Lastly, depending on threshold scores, we found between 18.8% and 34.0% (interquartile range) of likely associations to be tissue specific between pairs of tissues, given the difference in probabilities between the curves for replicates and tissues. Therefore, with up to 7% of likely associations not quantified in other tissues because of gene expression (Supplementary Fig. 14), we estimated over 25.8% (18.8% + 7%) of likely associations to be tissue specific. We sought to characterize the likely associations that were shared between tissues. Compared to all likely associations (average Jaccard index = 0.19), we found that the similarity between pairs of tissues increased as we restricted the likely associations to interactions identified through high-throughput screens such as yeast two-hybrid (Jaccard index = 0.30; HuRI)3 or AP (Jaccard index = 0.41; BioPlex)4 experiments (Fig. Likewise, the similarity between pairs of tissues increased when restricting likely associations to known interactions reported for signaling (Jaccard index = 0.32; SIGNOR)79, biological pathways (Jaccard index = 0.48; Reactome)80, physical associations (Jaccard index = 0.56; STRING)81 or human protein complexes (Jaccard index = 0.74; CORUM)76. Lastly, we found that the quantified differences and similarities between tissues were not sensitive to the choice of score cutoffs (Supplementary Fig. Thus, known protein interactions are typically shared by the tissues in our association atlas, with signaling interactions being less commonly recovered between tissues than stable protein complexes. These observations reflect the divergence between tissues for different types of interactions and may also reflect differences in accuracy for recovering associations for stable protein complexes compared to spatiotemporal interactions that are dynamic. Well-characterized protein complexes were generally preserved across tissues, becoming more variable as the complex-averaged association scores decreased (ρ = −0.77, P = 6.2 × 10−125) (Supplementary Fig. As seen in other proteomics datasets82, more variable complexes are typically involved signaling and regulation (for example, tumor necrosis factor and emerin), while more stable complexes are involved central cellular structures (for example, ribosomes and the respiratory chain) (Supplementary Fig. While protein complexes varied little between tissues, we found that associations varied strongly for tissue-specific cellular components, for example, for the brain (synapse-related components), throat (structural components of muscle fiber), lung (motile cilia) and liver (peroxisomes) (Supplementary Fig. This suggests that tissue-specific and cell-type-specific cellular components are an important driver of tissue-specific protein associations that are independent of simple expression differences. To explore cell-type-specific associations in our association atlas, we took the AP2 adaptor complex as a well-known example. The AP2 complex has neuron-specific functions in addition to functions that are general to all cells83. Indeed, the subunits of the AP2 complex were coabundant in all tissues (average association score between subunits = 0.80). We found 91 proteins that had association scores with all AP2 subunits in all tissues and were known to associate with AP2 (STRING score > 400). Among these, the 51 synaptic proteins (SynGO84) had higher association scores with the AP2 complex in the brain (average score = 0.54) compared to the other tissues (average score = 0.48 ± 0.00, P = 6.7 × 10−6, according to a one-sided Mann–Whitney U-test). Conversely, the nonsynaptic interactors had lower association scores with the AP2 complex in the brain (average score = 0.33) compared to the other tissues (average score = 0.43 ± 0.00, P = 1.1 × 10−21, according to a one-sided Mann–Whitney U-test) (Fig. We explored further examples by focusing on cell-type-specific associations in the context of disease. We found that proteins of hemoglobin are related to anemia and have likely associations with anemia proteins but only in the blood (Fig. Likewise, we found that subunits of chylomicron, which transports dairy lipids from the intestines, contain and have likely associations with proteins related to Crohn's disease but only in the colon85,86. Lastly, we found that subunits of fibrinogen, synthesized in the liver, contain and have liver-only likely associations with proteins related to liver disease87,88. For the other tissues, we could find many examples of tissue-specific and cell-type-specific associations for protein complexes, cellular components and disorders such as diabetes and asthma (Supplementary Fig. These examples demonstrate that our association atlas can be used to study tissue-specific functions of protein complexes and context-dependent associations for disease genes. a, Association scores between AP2 subunits and known AP2 interactors (STRING scores > 400) that are synaptic proteins (NECAP1 and BIN1; SynGO) or not (DAB2 and NECAP2). Heat maps show association scores in the brain and averaged association scores for the other tissues. Proteins (nodes) are complex members (gray) and disease genes (black edge). Associations are likely in all tissues (thin gray lines) or likely in a single tissue and not likely in all others (thick colored lines). Thick gray lines are associations with prior evidence (STRING scores > 400). Disease genes defined through OTAR (Methods). Relationships are scored by aggregating the association scores between all pairs of proteins from disjoint sets. d, Relationship scores of cellular components (light gray; GO), GWAS traits (dark blue; OTAR L2G ≥ 0.5) and between traits and components (light blue). Each dot represents the relationship between two sets, indicating the average and CV of relationship scores relative to the tissue median. Green dots show relations of the ribosome and spliceosome (score > 1.75; green box); purple dots show relations of synaptic components (CV > 0.4; purple box). e, Dendrograms of the 15 most brain-specific GWAS traits (left) and the 15 GO cellular components having the most brain-specific relationship with OCD (L2G ≥ 0.5; right). Dendrograms were constructed with complete-linkage clustering using the Manhattan distance on the relationship scores between traits (left) or between cellular components (right). Heat maps show genes overlapping between cellular components and OCD (orange; Jaccard index) or the enrichment of nonoverlapping genes from cellular components with drug targets, genes associated with OCD in mice or genes less confidently linked to OCD through GWAS (purple–green; conditional log2 odds ratios of one-sided Fisher exact test; dots show BH-adjusted P values < 0.05; Methods). SV, synaptic vesicle; CCV, clathrin-coated vesicle; m., membrane; SC, Schaffer collateral; ASD, autism spectrum disorder. We sought to generalize these examples of context-specific associations by systematically mapping the relationships amongst traits and multiprotein structures such as complexes or cellular components. As sets of proteins, we defined cellular components by Gene Ontology (GO) and defined human traits on the basis of the genome-wide association studies (GWAS) Open Targets (OTAR) locus-to-gene (L2G) score (≥0.5)89,90,91. We then scored the relationship between sets of proteins with the median association score of all possible protein pairs between the sets (Fig. 3c, schematic, omitting the intersection between gene sets). In total, we scored the relationships between traits (107,306 pairs), between traits and cellular components (240,967 pairs) and between components (134,002 pairs) across all tissues (Fig. The relationship scores that were high in all tissues were primarily of core cellular components such as the ribosome and spliceosome (72% of relationships with relative average score > 1.75), while the relationship scores that varied most across tissues often involved tissue-specific structures such as synaptic components (61% of relationships with a coefficient of variation (CV) > 0.4). These observations suggest that the relationship scores recapitulate the relatedness of protein sets in a tissue-specific manner, particularly for the brain. Additionally, we unbiasedly scored the tissue specificity of each protein set as the median association score between all pairs of its proteins (Methods and Supplementary Tables 5 and 6). Using these scores, we then selected the 15 traits most specific to the brain, 13 of which were indeed related to the brain (Supplementary Table 7). Clustering these traits using the trait–trait relationship scores from the brain revealed a hierarchical organization of traits with co-occurring conditions such as anorexia nervosa, obsessive–compulsive disorder (OCD) and Tourette syndrome closely clustering together92,93 (Fig. As an example, we further determined the 15 cellular components that had the strongest brain-specific relationships with OCD, all but one of which were related or specific to neurons (Fig. 3e, right dendrogram, and Methods). The majority of these cellular components had few genes in common with the genes confidently associated with OCD (Fig. 3e, orange heat map; Jaccard indices < 0.04). However, after removing the few genes confidently associated with OCD through GWAS, we found that almost all components were still enriched with or contained OCD-related genes (Fig. 3e, purple–green heat maps), that is, drug targets for OCD (odds ratio = 8.4 ± 1.8; ChEMBL clinical stage 2 or higher)94, genes related to OCD from mouse deletion phenotypes (odds ratio = 4.8 ± 0.8; International Mouse Phenotyping Consortium (IMPC) score ≥ 0.5)95 or genes less confidently linked to OCD through GWAS (odds ratio = 1.6 ± 0.2, OTAR L2G score < 0.5) (Methods). Moreover, these 15 components with the strongest OCD relationships in the brain were more strongly enriched with OCD-related genes than other cellular components that contained OCD-linked genes (P = 4.3 × 10−11 (drug targets), 3.4 × 10−15 (genes related to OCD in mice) and 6.6 × 10−7 (genes less confidently linked to OCD through GWAS), according to one-sided Mann–Whitney U-tests). Together, the results above demonstrate that the proposed relationship scores can prioritize cellular components that are enriched with trait-relevant genes. Analogously, we found that we could use the relationship scores for reconstructing the hierarchical organization of the cell, including maps of subcellular structures and modules of tissue-specific relations between cellular components (Supplementary Fig. These observations demonstrate the potential for our association atlas to facilitate the systematic mapping of relations among traits, cellular compartments and likely other ontology terms. The results above indicate that the tissue-specific associations could facilitate the prioritization of disease-linked genes by functional association. Indeed, direct interactors of disease-linked genes have been used to prioritize causal genes in genetically linked loci and shown to be enriched in successful drug candidates96,97,98. To explore this in more detail, we constructed a network of brain interactions for schizophrenia (SCZ)-related genes. Specifically, we sought to prioritize highly ranked associations for the brain that involve SCZ-related genes and that have additional evidence from orthogonal methodologies. We started by taking n = 369 genes associated with SCZ through GWAS studies (‘starting genes', L2G scores ≥ 0.5) and computed the top 25 traits and cellular components that had the strongest tissue-specific relation to SCZ in each tissue (Fig. This gave us a collection of genes related to SCZ in a tissue-specific manner. For each tissue, we then filtered for protein pairs that had one SCZ starting gene and one SCZ-related gene and required these protein pairs to have association scores exceeding the 97th percentile of the tissue scores (association score = 0.70 ± 0.01, 0.73 in the brain), leading to tissue-specific networks of associations for SCZ-related genes (Supplementary Fig. After removing the SCZ starting genes from the brain network, the remaining genes were still enriched for genes associated with SCZ in mice (Benjamini–Hochberg (BH)-adjusted P value = 1.5 × 10−5, IMPC score ≥ 0.5), drug targets for SCZ (BH-adjusted P value = 9.8 × 10−5, ChEMBL clinical stage 2 or higher) and other variants associated with SCZ (BH-adjusted P value = 1.0 × 10−7, OTAR L2G scores < 0.5), according to one-sided Fisher exact tests. This enrichment was specific to the brain compared to any of the other tissues, suggesting that the proposed methodology presents a systematic approach for prioritizing disease genes of tissue-specific traits (Fig. Genetic variants associated with SCZ are used together with relationships between traits and cellular components and the tissue scores to prioritize associations for SCZ-related genes. b, Enrichment of predicted associations with genes related to SCZ through mouse phenotypes (IPMC; crosses), drug targets (clinical stage 2 or higher; circles) or GWAS variants (L2G scores < 0.5; squares). c, Enrichment of pulldown interactions in the predicted associations for SCZ-related genes. In b,c, purple symbols represent the brain and gray symbols represent other tissues. Scatter plots show the conditional odds ratios and BH-adjusted P values, according to one-sided Fisher exact tests. d, Simplified network of validated brain interactions for SCZ-related genes (Methods). Circular and hexagonal nodes were prey and bait proteins in the pulldown studies, respectively. Nodes are colored as GWAS variants (green), drug targets (red), associated with SCZ in mice (blue) or other (gray). Gray edges were predicted from association scores in the brain and validated through pulldown experiments. Yellow edges are known interactors (physical associations in STRING; scores > 750). Purple edges have ipTM scores > 0.5. Purple labels annotate subgraphs of known interactors with the most enriched GO cellular components (exponent of BH-adjusted P value between brackets, according to a one-sided Fisher exact test). e, AlphaFold2 model of the interface between HCN1 and 14-3-3 proteins. Shown are the HCN complex (Protein Data Bank 6UQF; light green) aligned with the AlphaFold2 model of HCN and YWHAZ. The sequence shows the 14-3-3-binding site of HCN1 according to the AlphaFold2 models (green text, 10-Å cutoff; inset), overlaid with the predicted 14-3-3-binding site (green box, 14-3-3-Pred score = 0.457) and phosphorylation site S789 (black box). Interface residues are colored by predicted local-distance difference test. Indeed, for brain-related disorders, including autism, attention deficit hyperactivity disorder (ADHD), Tourette syndrome and others, we found that the proposed methodology of prioritizing protein associations for disease-linked genes enriched, specifically in the brain, for other genes associated with the respective disorder through mouse phenotypes or drug targets, with many also being enriched for GWAS variants that are less confidently linked to the disorder (Supplementary Fig. Having established that the presented methodology prioritizes protein associations that enrich for disease-related genes in a tissue-specific manner, we sought to validate the prioritized protein associations through additional evidence. Specifically, to further validate the networks of predicted associations for SCZ-related genes, we assembled a curated dataset of brain interactions established experimentally through pulldowns using human brain cells (that is, AP–MS or coimmunoprecipitation–MS in human microdissected brain tissue or human induced pluripotent stem cell-derived neurons99,100,101,102,103). This dataset contained 7,887 human brain interactions for 30 bait proteins and has been incorporated into the IntAct database9 (Supplementary Table 9). We filtered these brain interactions for the bait proteins that were associated with SCZ (OTAR L2G score ≥ 0.5) and further filtered the tissue-specific networks of SCZ-related genes for associations involving at least one bait protein. The remaining associations of SCZ-related genes were strongly enriched with the interactions from pulldowns with SCZ-related baits, especially for the brain (log BH-adjusted P value = 84.3, according to a one-sided Fisher exact test) compared to the other tissues (log BH-adjusted P value = 1.8 on average) (Fig. Thus, the brain associations of SCZ-related genes but not the other tissues were experimentally validated by pulldown studies and enriched with SCZ-related association partners. Next, we filtered the prioritized brain associations for SCZ-related genes for interactions that were also found through the pulldown studies to obtain a network of 205 validated brain interactions for SCZ-related genes (average association scores = 0.86 in the brain), which we simplified to only show synaptic (SynGO) genes having prior evidence (Fig. 4d, Methods and Supplementary Table 10). The visualized network contained 56 proteins connected to three bait proteins through a collection of 66 validated brain interactions. These connected proteins included SCZ drug targets (3 proteins; clinical stage 2 or higher), proteins associated with SCZ in mice (12 proteins; IMPC score ≥ 0.5) or proteins linked with SCZ with weaker prior evidence (15 proteins; OTAR L2G scores < 0.5). Surprisingly, only four of the visualized brain interactions were confidently reported in any of the major protein interaction databases before our curation effort (CACNA1C with CACNB1, CACNB2 and CACNB4 (STRING scores > 750) and SRC (Signor)) and only the interactions of SHANK3 with AP2B1 and CLTB were quantified in more than three other tissues (association scores = 0.34 ± 0.08 and 0.29 ± 0.04 in the other tissues). These observations support our earlier analysis that the network of validated interactions of SCZ-related genes is specific to the brain. The visualized network contained several groups of highly interconnected proteins (STRING scores > 750). These groups were enriched with genes of cellular components typical for neuronal functioning and SCZ, such as a group of proteins for the postsynaptic cytoskeleton104 (BH-adjusted P value = 2.3 × 10−8) or for clathrin-coated vesicles (9.4 × 10−14), according to one-sided Fisher exact tests. For the clathrin vesicle coat, the network connected all subunits of the AP2 complex and clathrin proteins to HCN1. Interestingly, previous pulldown studies showed that HCN channels directly interact with TRIP8b (refs. TRIP8b regulates the trafficking of HCN channels106 and mainly associates with the AP2 complex107. Moreover, while AP2 and clathrin are not cell type specific, both HCN1 and TRIP8b were found to be enriched at parvalbumin (PV)-positive synapses108, with HCN channels being specific to PV neurons and important for their high firing frequencies109,110. Given the link of PV neurons with SCZ111,112,113,114,115, these observations suggest that AP2 and clathrin may be involved in a PV neuron-specific disruption of HCN channel trafficking with SCZ. To suggest putative interface models for the validated brain interactions for SCZ-related genes, we used AlphaFold2 to predict the structures for 205 protein interactions, including the entire visualized network (Fig. The predicted models had more confident interactions compared to models for known complex members from CORUM116 (average pDockQ scores = 0.20 and 0.13, respectively, P = 1.6 × 10−19, according to a one-sided Mann–Whitney U-test). In total, we identified 15 moderate-confidence interactions (interface predicted template modeling (ipTM) scores > 0.5). These included the brain-specific binding of all three 14-3-3 proteins (YWHAG, YWHAH and YWHAZ) with HCN1 (Fig. The interfaces of these three models overlap and are located in the C-terminal disordered region of HCN1 (residues 775–802). This consensus interface includes a predicted 14-4-3-binding site (centered around S789; average ipTM score = 0.75 at the putative binding site, 14-3-3-Pred score = 0.457)117 that has been verified through pulldown experiments118. Indeed, the binding of 14-3-3 proteins with HCN1 was found to be dependent on the phosphorylation of S789, with the interaction between 14-3-3 and HCN1 likely inhibiting HCN1 degradation118. Lastly, the network contained 15 genes within loci genetically associated with SCZ that had weaker evidence supporting them as the causal genes at each locus. Given their interaction with other SCZ-related genes, these could be prioritized as more likely causal because of their functional roles. Some of these genes (AP2B1, ATP2B2 and SYNGAP1) had the highest L2G score for their respective locus with single-nucleotide polymorphisms (SNPs) linked to SCZ but had scores below the 0.5 cutoff used (0.457, 0.264 and 0.251, respectively)90,91. In addition to the AP2 complex, we found a member of the AP3 complex (AP3B2). AP3B2 had the second highest score for the locus with SNPs (rs783540), which had splicing and expression quantitative trait locus (QTL) associations with AP3B2 but was ranked higher for disruption of CPEB1 given that the variant lies within a CPEB1 intron. Similarly, MARK2 was ranked second for two SNPs (rs7121067 and rs11231640), both having splicing and promoter capture Hi-C associations with MARK2 but being ranked higher for disrupting RCOR2 because of proximity to its transcription start site. CDC42 and NTRK2 had the second highest L2G scores for their locus but the top associated genes (WNT4 and AGTPBP1, respectively) had lower and less specific expression in the brain12. As a final application of our protein association atlas, we focused on the interactome of synapses. To do so, we prepared and purified synaptosomes from rat brains as an orthogonal approach for validating the brain associations. We fractionated the synaptosomes with size-exclusion chromatography (SEC) into 75 fractions that were then subjected to liquid chromatography (LC)–MS/MS. A total of 3,409 unique proteins were detected, including well-known protein complexes such as the CCT complex subunits, whose profiles correlate across the fractions (Fig. In vivo synaptosomes (light blue) from rat neurons were purified and fractionated into 75 fractions through SEC and subjected to LC–MS/MS. Elution profiles show the protein intensities from MS for the CCT complex members (right). b, AUC values for the cofractionation studies (in vivo mouse brain, gray; subcellular glioblastoma, gray; in vivo rat synaptosome, light blue) and for the merged synaptic interactome (dark blue). Positives were defined by complex members in CORUM. c, Comparison of association scores of interactions between synaptic proteins or interactions of other proteins in the brain (purple) and the other tissues (gray). Interactions are from the synaptic interactome (score > 0.8). Data were derived using a one-sided Mann–Whitney U-test, BH-adjusted P values and median likelihood ratios. Synaptic proteins included those that were enriched in the synapse (crosses), reported in SynGO (circles), associated with GO synaptic components (squares) or had brain-elevated expression according to The Protein Atlas (pluses). d, Networks of validated interactions between synaptic proteins (SynGO or enriched in mouse tissues) related to ADHD, bipolar disorder, Parkinson disease, unipolar depression or Tourette syndrome. Nodes are colored by association to the trait through GWAS (green), drug targets (red), mouse phenotypes (blue) or other (gray) (Methods). Gray edges were predicted from association scores in the brain and validated through the cofractionation studies. Yellow edges are known interactors. Purple edges have ipTM scores > 0.5. We preprocessed the fractionation profiles of the synaptosome and computed the coabundance of proteins across fractions to score the cofractionation of 4,276,350 protein pairs in rat synapses (Methods). To increase the confidence of the interaction scores, we then combined the rat synaptosome with other cofractionation profiles from in vivo mouse brains6 and subcellular fractionation profiles from human glioblastoma cells7. We merged the cofractionation studies by orthologs that were quantified in all three datasets and computed interaction probabilities with a logistic model using the CORUM database as positives (Methods and Supplementary Table 12). The resulting synaptic interactome quantified 1,309,771 interaction probabilities for 1,619 proteins and improved the recovery of known interactions compared to the cofractionation studies individually (Fig. Of the 1,619 proteins in the interactome, 24% are annotated as synaptic proteins in the SynGO database, 49% have been reported as synapse-enriched in mouse brains and 56% have previously been identified through crosslinking MS (XL-MS) of the mouse synaptosome84,108,119. All XL-MS interactions for these proteins were quantified in our synaptic interactome, having association scores of 0.59 compared to 0.37 for other associations (P = 2.0 × 10−44, according to a one-sided Mann–Whitney U-test). Moreover, the synaptic interactions between synapse-enriched proteins were more likely compared to other interactions (average association scores = 0.40 and 0.36, P < 1 × 10−300, according to a one-sided Mann–Whitney U-test). Together, these observations suggest that our synaptic interactome largely aligns with current state-of-the-art synaptic interaction resources and scores the likelihood of interactions for a wide range of synaptic proteins, with interactions being more likely for synaptic proteins compared to other proteins. The interactions between synapse-enriched proteins formed more likely associations compared to associations of nonsynaptic proteins, especially for the brain compared to the other tissues of our association atlas (Fig. Given this brain-specific elevated likelihood of synaptic interactions, we constructed a network of interactions between synaptic proteins (in SynGO or synapse-enriched in mouse brains) that were both likely coabundant in the brain (109,913 associations) and likely cofractionated for the synaptosome (121,732 interactions). The resulting network consisted of a collection of 37,318 validated protein interactions between synaptic proteins (Supplementary Table 13). These synaptic interactions were primarily specific to the brain because few of the interactions were likely for the majority of other tissues in our association atlas (20%) and only a small fraction have been reported in any protein interaction database (5.8% in STRING, 1.3% in HuMAP, 3.6% in IntAct and 1.6% in BioPlex). We were particularly interested in the validated interactions of synaptic proteins that are associated with brain disorders. As before, we filtered for GWAS traits that had associated genes through mouse phenotyping (IMPC) or known drug targets (ChEMBL) and whose trait-level association scores were elevated in the brain compared to the other tissues (z score > 1; Methods). We found that 10 of the resulting 13 traits were disorders clearly related to the brain and selected the 727 confident synaptic interactions between genes associated with these brain-specific traits (association scores = 0.7 and 0.81 on average in the synaptosome and brain, respectively). Additionally, to suggest putative interface models, we used AlphaFold2 to predict the structures for these 727 interactions. The predicted models had more confident interactions compared to models for known interactors in CORUM (average pDockQ scores = 0.28 and 0.13, P = 3.7 × 10−147, according to a one-sided Mann–Whitney U-test) or HuMAP (pDockQ scores = 0.28 and 0.25, P = 2.4 × 10−17) and high-confidence models (ipTM scores > 0.7) were enriched for additional evidence from XL-MS experiments in mouse synaptosomes119 (Supplementary Fig. In total, we identified 105 moderate-confidence interactions (ipTM scores > 0.5; Supplementary Table 14). Lastly, we visualized (simplified) the networks of validated synaptic interactions between trait-related genes of the brain disorders (Fig. Several genes in the networks had weaker prior evidence supporting them as the causal genes at loci genetically associated with the brain disorders. These genes could be prioritized as more likely to be causal for the disorders because of their validated synaptic interactions with other genes that were confidently associated with the same disorders. As before, we looked for genes with the highest (below cutoff) L2G scores for their respective loci with SNPs linked to the brain disorders. Of these genes, all but ATP2A2, MDH1, MYO18A and PIK3C3 had the highest or second highest expression in brain tissue12. Additionally, we found genes with synaptic interactions that did not have the highest L2G score for the respective SNP linked to the brain disorders but still had additional evidence. For example, we found that PAFAH1B1 associated with RHOA in our synaptic interactome, with both having weak prior evidence for being causal to depression. PAFAH1B1 has a role in neural mobility and is required for activation of Rho guanosine triphosphatases such as RhoA120. PAFAH1B1 was the second highest scoring gene for the locus with an SNP (rs12938775) linked to unipolar depression. This variant has expression QTL associations with PAFAH1B1 and lies within a PAFAH1B1 intron, despite being more distant to the transcription start site of PAFAH1B1 compared to the highest scoring gene (CLUH). However, PAFAH1B1 encodes a synaptic protein, whereas CLUH does not, with the expression of PAFAH1B1 being higher and more specific to the brain compared to CLUH12. Overall, this example demonstrates how orthogonal approaches targeting subcellular structures and individual tissues can provide tissue-specific protein interaction networks and aid in the prioritization of genes likely to be causal for tissue-related disorders. Despite progress in mapping protein interactions on a large scale, we lack a comprehensive understanding of how these interactions differ across tissues. It has been previously shown that the protein–protein associations derived from correlated protein abundance measurements can be more accurate than those derived from mRNA coexpression15,16,17,18. Here, we showed additionally that these associations can be more accurate than those found by correlation analysis of cofractionation. However, in line with other reports, we observed that the combination of different cofractionation datasets can improve the predictions121 and it is likely that more advanced analyses of cofractionation data may yield more accurate results122. Interestingly, we noted that we can exclude the variation in protein levels that is explained by mRNA changes without significantly affecting the performance of the method. This suggests that protein interaction partners are often post-transcriptionally coregulated. This is in line with the observations that protein abundances of subunits within a complex can be limited by the abundance of the complex itself, with active degradation of excess unbound subunits15,16,22,24,123. Several lines of evidence attest to the quality of the derived association scores. Cohorts of the same tissue had similar association scores, association scores derived from tumor and healthy dissections of the same tissue tended to replicate in a tissue-specific manner and essential protein complexes were generally very well predicted across all tissues. Around 46% of likely associations and 90% of confident associations were replicated between healthy-tissue-derived and tumor-derived scores, suggesting that these associations, complemented with other data, can be useful in generating testable hypotheses regarding tissue-specific biology. In particular, our human tissue-derived associations can provide supporting evidence for interactions in human tissues obtained from less physiologically relevant material, given that human tissue is often not available for interaction studies. Taking the brain as example, we complemented the coabundance associations with experimental data from pulldowns, cofractionation and AlphaFold2 predictions to provide a more confident subset of interactions for disease–gene mapping. We estimated that over 25% of associations are tissue specific. While we observed a large fraction of associations that were not quantified in different tissues, likely because of technical issues of detectability, we estimated that only a small fraction of the differences across tissues are because of changes in expression levels (up to 7%). We found that cell-type-specific structures such as synaptic components may contribute greatly to differences in protein associations across tissues, with differences in post-translational modifications being other mechanisms that could explain some variation in tissue-specific protein associations. Lastly, we showed that our atlas can be used to derive relations between traits and with cellular components. This approach is particularly effective for disorders of the brain and synaptic components, demonstrating that the network of brain associations is more specific and distinct from the other tissues. We presented validated brain interactions enriched for disease-associated genes and drug targets and showed how this can be useful to prioritize novel disease-linked genes and presumably drug candidates. The combination of genetics, tissue-specific networks and AlphaFold2 models provides an integrated network for enhanced understanding of disease mechanisms and target prioritization. In addition, our approach may improve the safety of targeting disease genes as the predicted disease–gene associations will tend to be tissue specific and, therefore, the proposed genes may be safer to target. Publicly available protein abundance data were filtered by removing nonabundance samples and measurements (internal controls, reference samples, samples that were withdrawn or that did not have further annotation, etc.). Protein measurements without gene identifiers were removed and the abundance data of cohorts with replicate samples for some but not all participants were averaged at the participant level (for 3 of 50 cohorts, for example, for cohorts having a sample with five technical replicates as the internal control). Zeros were replaced with NaN values for all entries of cohorts that were not log-transformed to ensure that undetected proteins did not have abundance data. Cohorts were then split into tumor and healthy samples if not reported separately. The sample names were homogenized to obtain paired RNA expression and protein abundance data and paired tumor and healthy biopsies. The gene identifiers were homogenized and converted to HUGO Gene Nomenclature Committee (HGNC) gene symbols across cohorts to allow for comparison between studies (Supplementary Table 15 and Supplementary Methods). Then, sequentially, duplicate measurements were averaged (that is, averaging the reported abundance of proteins whose identifier occurred more than once), the protein abundances were median-normalized for each participant, the data were log2-transformed if not already on a log scale (13 of 50 cohorts) and proteins with measurements in fewer than ten participants were removed. Note that abundances were not further normalized across participants because correlation coefficients are invariant under linear transformations. RNA expression data were processed in identical fashion. As deviations from the above, samples were removed from cohorts that shared participants by filtering the follow-up studies for participants whose protein abundance was not previously reported47,48. Coabundance estimates were computed by calculating the correlation matrix consisting of Pearson correlations between all possible pairs of proteins for a cohort. Each pair of proteins was required to have at least n = 30 paired abundances across samples for computing a correlation value (that is, protein pairs had to be quantified in at least 30 of the same samples, available for 48 of 50 studies). Correlations were then reported for each unique protein pair (alphabetically and sorted by gene symbols), ignoring selfcorrelations and missing values. For each study, a logistic model was fitted to convert the correlations to association probabilities. The set of ground-truth positives was defined as all protein interactions reported in the CORUM database76. Specifically, all possible protein pairs from the subunits for each protein complex in CORUM were defined as ground-truth positives and all protein pairs that had correlation coefficients but were not ground-truth positives were labeled as negatives, resulting in a collection of labels \({\left\{\,{y}_{i}\right\}}_{i\in [1,n]}\) for the \(n\) protein pairs (where \(y=1\) is a positive and \(y=0\) is a negative). Using the set of defined positives and negatives, a logistic regression with two free parameters was fitted for each study to the correlation values \({\left\{{x}_{i}\right\}}_{i\in [1,n]}\), Here, each study's model had an intercept \({\beta }_{0}\), parameter \({\beta }_{1}\) and no penalty. The logit function was defined as \({{\rm{logit}}\; p}=\mathrm{ln}(\frac{p}{1-p})\) and the model's parameters were optimized by minimizing the cost function, where \(W={\sum }_{i=1}^{n}{w}_{i}\) is the total weight and \({\left\{{w}_{i}\right\}}_{i\in [1,n]}\) are the sample weights for the protein pairs. These sample weights were determined by balancing the positive and negative classes when fitting the logistic model. In detail, each sample was weighted inversely proportional to the number of samples in its class, such that the total weight of the positive samples was equal to the total weight of the negative samples. Specifically, with \(Y={\sum }_{i=1}^{n}{y}_{i}\) being the total number of positives, the positive samples were weighted as \({w}_{i}=\frac{n}{2 Y}\) while the negative samples were weighted as \({w}_{i}=\frac{n}{2 (n-Y)}\), such that the total weight of the positives (\(Y\times \frac{n}{2 Y}=\frac{n}{2}\)) was equal to the total weight of the negatives (\((n-Y)\times \frac{n}{2 (n-Y)}=\frac{n}{2}\)) and the weights in the cost function summed to 1, Finally, the fitted logistic model was used to transform the correlation values to probabilities for each study. Note that the logistic (log-linear) model yields a rank-preserving transformation, preserving the ranking of protein pairs when sorting correlations and probabilities from large to small. The association probabilities per tissue are available through the BioStudies repository for this paper (S-BSST1423). Cofractionation data were collected from studies of in vivo mouse tissues (SEC with stable isotope labeling in mammals; 2 × 55 fractions for seven tissues)6, human breast tissue-derived cell lines (ion exchange with high-performance (HP)LC–MS; 3 × 2 × 192 fractions of three cell lines with two technical replicates each)77 and subcellular fractionation of human cancer cell lines (high-resolution isoelectric focusing with LC–MS; 5 × 10 fractions for five different tissues and an additional 4 × 10 fractions from three replicates and one condition in the lung)7. The protein identifiers of the mouse data were converted to human orthologs (Ensembl BioMart, September 28, 2023). Replicates and conditions were concatenated and treated as different fractions. The resulting ten cofractionation datasets (seven mouse tissues, one human breast cell line, one human lung cell line and one human mixed cell line) were further processed as follows. Data were filtered for fractions and proteins that had nonzero measurements and abundances of duplicate protein identifiers were averaged. Fractions were then normalized by total intensity, missing values were imputed as 90% of the minimum nonzero value for each protein and the data were log2-transformed (mouse tissues and breast cell line). Setting empty fractions to 90% of the minimum nonzero value was advantageous as it increased the number of protein pairs having sufficient observations across fractions for computing association probabilities (3.6-fold more protein pairs on average) and improved the cofractionation estimates by including the fractions where proteins were coabsent (increased AUCs by 14% on average). Here, the cofractionation estimates were not sensitive to the value used for empty fractions (that is, 0.7% difference in AUC values on average between using 90% or 10% of the minimum nonzero values). Association probabilities were then computed as described earlier. We defined the set of ground-truth positives as all protein interactions reported in CORUM. Specifically, for each protein complex, the positives were defined as all possible pairs of subunits. The space of negatives was then defined as all protein pairs for which association scores were quantified for a given dataset (cohort, tissue, etc) but that were not ground-truth positives (in CORUM). ROC curves were computed as the true positive rates as function of the false positives rates across all possible score thresholds, using the defined positives and negatives as labels and the association probabilities as scores. Association scores for a tissue were computed by averaging, for each protein pair, the association probabilities of all cohorts of the tissue. Missing values were ignored. Two cohorts were excluded on the basis of their performance for recovering tissue-specific associations of the other cohorts of their respective tissues (Supplementary Fig. Tissues were only included if association probabilities from at least two studies were available (not for the bladder, prostate and uterus (tumor) and only for the lung, throat, liver, kidney, colon and stomach (healthy)). Likely associations were defined as the protein pairs whose association score exceeded 0.5 for a given tissue, whereas confident associations were defined as the protein pairs whose association score exceeded 0.8 for a given tissue. Evidence was downloaded from the OTAR repository (version 24.06)90,91. Three types of evidence collected from OTAR were used for associating genes to traits. Specifically, genes were considered associated with traits through GWAS, associated with traits through mouse phenotypes (IMPC)95 or as drug targets (ChEMBL)94. For genes associated with traits through GWAS, the evidence was loaded from the OTAR ‘ot_genetics_portal' source (scores ≥ 0.5 (confident) or scores < 0.5 (weak)). For genes associated with traits through mouse phenotypes and drug targets, the evidence was loaded from the OTAR sources ‘impc' (scores ≥ 0.5) and ‘chembl' (clinical phase of at least 1 or at least 2), respectively. Gene identifiers were converted to HGNC gene symbols following the symbols used in the association atlas (Supplementary Table 15 and Supplementary Methods). Trait identifiers (experimental factor ontology) were converted to trait names using annotations from the OTAR platform (version 24.06). Evidence was then summarized per source (Supplementary Tables 16–19). The descriptions of GO terms were taken from their GO annotations and filtered for not being obsolete, having a description and being cellular components but not protein complexes (namespace = ‘cellular_component', name does not contain ‘complex'; 2,068 components). The cellular components were annotated with associated genes using the QuickGO annotations from the European Bioinformatics Institute (EBI; August 27, 2024). Additionally, the cellular compartments were annotated with associated genes reported by UniProtKB (August 27, 2024) through their Rest application programming interface (API). Both sets of associated genes were merged and gene identifiers were converted to HGNC gene symbols following the symbols used in the association atlas (Supplementary Table 15 and Supplementary Methods). Lastly, 889 GO cellular components had associated genes (Supplementary Table 20). Protein sets were defined by the genes associated with GO cellular components (541 components having at least five associated proteins) or genes associated with traits through GWAS (465 traits having at least 20 associated proteins). Set-level association scores and relationship scores were computed as follows. Let \({S}_{A}\) be some protein set \(A\). The set-level association score \({P}_{T}(A)\) for tissue \(T\) and \(A\) was defined as the median association score of all pairs of proteins in \({S}_{A}\) that were quantified for tissue \(T\), Analogously, the relationship score \({P}_{T}(A,B)\) for tissue \(T\) and protein sets \(A\) and \(B\) was defined as the median of all association scores between proteins from \(A\) (that were not in \(B\)) and proteins from \(B\) (that were not in \(A\)), Lastly, set-level association scores were computed for all protein sets in all tissues (Supplementary Tables 5 and 6), and relationship scores were computed between all pairs of protein sets in all tissues (Supplementary Tables 2–4). Relationship scores that could not be quantified were set as zero and ignored throughout. Structures were predicted using AlphaFold-multimer version 2.3.1. Input protein sequences were downloaded from UniProt (release 2023_03 to release 2024_02). The use of templates was turned on. Five structures were predicted for each protein pair and the predicted structure with the highest model confidence score was considered the ‘best' structure for each pair. The pDockQ scores were calculated for the top-ranked structure using a previously published implementation116. Interaction interfaces were defined by considering that any residue with at least one atom within 10 Å (or 5 Å; Fig. 4e) of the other protein chain is part of the interface. Cortices of adult rats (Sprague–Dawley) were dissected and a synaptosome preparation was generated using Syn-PER (Thermo Fisher Scientific) according to the manufacturer's instructions. All animal experiments were carried out under institutional guidelines (ZH172/18 Kanton Zürich Gesundheitsdirektion Veterinäramt). The procedures used for cell lysis and SEC-based fractionation followed experimental protocols previously published124,125, with slight adaptations regarding the lysis, detergent and SEC column. In short, synaptosomes were lysed in HNN buffer (50 mM HEPES, 100 mM NaCl and 50 mM NaF) supplemented with protease inhibitor (1× of a 500× stock solution), 20 mM sodium vanadate (of a 200 mM stock solution), 1 mM PMSF (from a 100 mM stock solution) and 1% n-dodecyl β-d-maltoside (DDM), from a freshly prepared 10% DDM stock solution. Samples were lysed by gentle pipetting up and down. The lysate was preclarified by centrifugation at 16,000g at 4 °C for 20 min. The supernatant was split to multiple ultracentrifuge tubes to remove insoluble cell debris by ultracentrifugation in a TLA 100.1 rotor (Beckman-Coulter) at 35,000g at 4 °C for 15 min. Next, supernatants were transferred to Eppendorf tubes and protein concentration was assessed using the BCA assay (Thermo Fisher Scientific). A total protein amount of 1.2 mg (or 319 μl of protein lysate) was then diluted up to 4 ml with HNN buffer (without detergent). To concentrate the sample and remove surplus detergent, samples were concentrated over a filter with a 30-kDa molecular weight cutoff (MWCO; Amicon). Samples were centrifuged at 3,600g at 4 °C for 5 to 10 min with 5-min intervals to monitor lysate volume. First, samples were concentrated to approximately 500 μl, before diluting them again up to 1 ml. This dilution step was repeated a total of three times before samples were concentrated to approximately 100 μl. For the separation of complexes, a 1260 Infinity II HPLC system (Agilent) connected to an SRT-C SEC-1000 (Sepax) 7.8 × 300 mm column with an SRT-C SEC-1000 (Sepax) 7.8 × 50 mm precolumn was used. The system was operated at a flow rate of 500 μl min−1. The column was equilibrated with SEC buffer (50 mM HEPES and 100 mM NaCl, pH 7.5) for at least four column volumes. Before collection of the separated sample, a replicate of the rat synaptosome extract was injected to prime the column. The input for the prime and SEC experiment was kept at 950 μg of total protein amount (according to BCA). The sample was separated into 75 100-μl fractionations. For separation, the HPLC was cooled to 4 °C and the column was placed on ice. Column performance was monitored by injecting a standard protein mix (Phenome protein standard mixture, diluted 1:3 in SEC buffer) before and after the experiment. SEC fractions were further processed using a high-recovery and high-throughput filter-aided sample preparation protocol124. In short, the 96-well Acroprep advance filter plate (10,000-kDa MWCO) was equilibrated by flushing the plate twice with 100 μl of H2O (HPLC-grade) followed by centrifugation at 1,800g. Each fraction was diluted 1:1 (v/v) with SEC buffer and added to the filter plate. Additionally an aliquot of SEC input was added to the filter plate. The sample was loaded by centrifugation at 1,800g to completely remove the SEC buffer. Next, 50 μl of TUA buffer (5 mM TCEP, 8 M urea and 20 mM ammonium bicarbonate) was added to each fraction. After cooling the plate to room temperature, 20 μl of 35 mM IAA was added to each well and the plate was incubated for 1 h at 400 rpm at room temperature. Urea, IAA and TCEP were removed by centrifugation at 1,800g and the plate was washed three times with 100 μl of 20 mM ammonium bicarbonate with centrifugation steps in between. After the last centrifugation, there was less than 10 μl of 20 mM ammonium bicarbonate left in each well. Before adding the digestion mix, the filter plate was moved to a new receiver plate. Proteins were digested overnight at 300 rpm at 37 °C by adding 1 μg of trypsin and 0.3 μg of Lys-C diluted in 50 μl of 20 mM ammonium bicarbonate to each well. Peptides were collected by centrifugation at 2,400g for 30 min. The filter plate was subsequently washed with 100 μl of of HPLC-grade H2O. To increase the recovery of hydrophobic peptides, 50 μl of 50% acetonitrile was added to the receiver plate. Each fraction was then transferred to LoBind tubes and peptides were dried under reduced pressure at 42 °C. For MS analysis, peptides were reconstituted in 5% acetonitrile and 0.1% formic acid containing iRT peptides (Biognosys). The peptides were analyzed on a Fusion Lumos MS instrument (Thermo Fisher Scientific) using a 2-cm Acclaim PepMap 100 C18 HPLC trap column (Thermo Fisher Scientific) and 25-cm EASY-Spray HPLC analytical column (Thermo Fisher Scientific) setup connected to an EASY-nLC 1200 instrument (Thermo Fisher Scientific)126,127. Peptides were loaded in 100% buffer A (98% H2O, 2% acetonitrile and 0.15% formic acid) and eluted at a flow rate of 250 nl min−1 with a segmented 1-h gradient from 1% to 58% buffer B (80% acetonitrile and 0.15% formic acid). The data were acquired in data-independent acquisition mode. The Orbitrap-based method used contained 40 dynamic windows over a scan range of 350–1,650 m/z with a resolution of 30,000 and higher-energy collision-induced dissociation energy of 27% and a survey scan with a resolution of 120,000, maximum injection time of 50 ms and default charge state of 2. Raw files were converted to HTRMS using HTRMSConverter (Biognosys) and analyzed in Spectronaut 13 (Biognosys) applying the ‘only protein group-specific' proteotypicity filter; otherwise, the standard manufacturer's settings for directDIA were applied using the UniProt reference proteome for Rattus norvegicus (retrieved October 2019), quality control standards and common laboratory contaminants. The Spectronaut output (BGS report) was then further preprocessed (Supplementary Table 21). The total sum of intensity of the fractions was scaled to account for different dilutions applied at the MS injection stage (fractions 44–54 were multiplied by 2 and fractions 56–75 were multiplied by 4). Proteins without measurements were removed and fractions were removed when the total sum of intensity was less than 50% of the average total sum of intensity for the adjacent ten fractions (fractions 44, 45, 50, 51, 63 and 69). Rat protein identifiers were converted to human orthologs (Ensembl BioMart, September 26, 2023). The data were then further preprocessed as described earlier. The processed protein information and protein–protein associations are available through the BioStudies repository for this paper (S-BSST1423). Cofractionation data from in vivo mouse brains (110 fractions)6 and a subcellular fractionation of a human glioblastoma cell line (U251, 10 fractions)7 were preprocessed as described earlier. For each study and the in vivo rat synaptosomes (75 fractions), the correlations of the preprocessed intensities were computed per protein pair across fractions as described earlier (Supplementary Table 22). The correlation estimates of the studies were then filtered for protein pairs that were quantified for all three studies (1,309,771 pairs) and converted to association probabilities using a logistic model and all three studies as variates (model weights = 1.00 (glioblastoma), 2.05 (mouse brain) and 0.61 (rat synaptosome)), with complex members as positives, as described earlier. AUC values for the recovery of known interactions were computed using the association probabilities for each study individually and for the combined studies, as described earlier. Methodology for the analyses of figure panels is described in the Supplementary Methods. The source code and all data are available through the BioStudies repository for this paper (S-BSST1423). Cell lysis, SEC and sample preparation Column performance check standard aqueous SEC 1 Guard column, SRT-C1000, 5 μm, 1,000 Å, 7.8 × 50 mm LC operating software Agilent AcroPrep Advance 96-well filter plates, 350 μl, Omega 10-kDa MWCO Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All processed, analyzed and generated data in this study are publicly available (BioStudies S-BSST1423). The MS proteomics data were deposited to the PRIDE repository (identifier PXD049084). Other proteomics data were collected from public studies (Supplementary Table 1). Approved HGNC gene symbols obtained online (https://www.genenames.org) and gene symbols were harmonized or converted to human orthologs with Ensembl (https://www.ensembl.org). Evidence for disease-related genes was obtained from the OTAR genetics platform (https://platform.opentargets.org/). GO annotations were collected for SynGO (https://www.syngoportal.org) and other ontology terms (https://geneontology.org) and associations of genes with GO terms were collected using the UniProt rest API (https://www.uniprot.org) and EBI (https://www.ebi.ac.uk/QuickGO). Mutation frequencies in cancer were collected from cBioPortal (https://www.cbioportal.org). Consensus RNA expression data (https://www.proteinatlas.org/humanproteome/tissue/data#consensus_tissues_rna) and protein location data (https://www.proteinatlas.org/humanproteome/subcellular/data#locations) were collected from The Protein Atlas (https://www.proteinatlas.org). Source data are provided with this paper. All code for data processing, analysis and the figures in this study is publicly available as Jupyter notebooks (BioStudies S-BSST1423). & Marcotte, E. M. hu.MAP 2.0: integration of over 15,000 proteomic experiments builds a global compendium of human multiprotein assemblies. Tarassov, K. et al. An in vivo map of the yeast protein interactome. Luck, K. et al. A reference map of the human binary protein interactome. Huttlin, E. L. et al. Dual proteome-scale networks reveal cell-specific remodeling of the human interactome. Hein, M. Y. et al. A human interactome in three quantitative dimensions organized by stoichiometries and abundances. Skinnider, M. A. et al. An atlas of protein–protein interactions across mouse tissues. Orre, L. M. et al. SubCellBarCode: proteome-wide mapping of protein localization and relocalization. Havugimana, P. C. et al. A census of human soluble protein complexes. The MIntAct project—IntAct as a common curation platform for 11 molecular interaction databases. The BioGRID database: a comprehensive biomedical resource of curated protein, genetic, and chemical interactions. The STRING database in 2021: customizable protein–protein networks, and functional characterization of user-uploaded gene/measurement sets. Uhlén, M. et al. Tissue-based map of the human proteome. Sharing and specificity of coexpression networks across 35 human tissues. Greene, C. S. et al. Understanding multicellular function and disease with human tissue-specific networks. Sousa, A. et al. Multi-omics characterization of interaction-mediated control of human protein abundance levels. Ryan, C. J., Kennedy, S., Bajrami, I., Matallanas, D. & Lord, C. J. A compendium of co-regulated protein complexes in breast cancer reveals collateral loss events. Detection of dysregulated protein-association networks by high-throughput proteomics predicts cancer vulnerabilities. Wang, J. et al. Proteome profiling outperforms transcriptome profiling for coexpression based gene function prediction. Buljan, M. et al. A computational framework for the inference of protein complex remodeling from whole-proteome measurements. Roumeliotis, T. I. et al. Genomic determinants of protein abundance variation in colorectal cancer cells. Kustatscher, G. et al. Co-regulation map of the human proteome enables identification of protein functions. Taggart, J. C., Zauber, H., Selbach, M., Li, G.-W. & McShane, E. Keeping the proportions of protein complex components in check. Juszkiewicz, S. & Hegde, R. S. Quality control of orphaned proteins. Widespread post-transcriptional attenuation of genomic copy-number variation in cancer. Kinetic analysis of protein stability reveals age-dependent degradation. Herbst, S. A. et al. Proteogenomics refines the molecular classification of chronic lymphocytic leukemia. The proteogenomic subtypes of acute myeloid leukemia. Kramer, M. H. et al. Proteomic and phosphoproteomic landscapes of acute myeloid leukemia. Stratmann, S. et al. Proteogenomic analysis of acute myeloid leukemia associates relapsed disease with reprogrammed energy metabolism both in adults and children. Yang, M. et al. Proteogenomics and Hi-C reveal transcriptional dysregulation in high hyperdiploid childhood acute lymphoblastic leukemia. Archer, T. C. et al. Proteomics, post-translational modifications, and integrative analyses reveal molecular heterogeneity within medulloblastoma subgroups. Integrated pharmaco-proteogenomics defines two subgroups in isocitrate dehydrogenase wild-type glioblastoma with prognostic and therapeutic opportunities. Integrated proteogenomic characterization across major histological types of pediatric brain cancer. et al. Proteogenomic and metabolomic characterization of human glioblastoma. Yanovich-Arad, G. et al. Proteogenomics of glioblastoma associates molecular patterns with survival. Integrated proteogenomic characterization across major histological types of pituitary neuroendocrine tumors. Anurag, M. et al. Proteogenomic markers of chemotherapy resistance and response in triple-negative breast cancer. Asleh, K. et al. Proteomic analysis of archival breast cancer clinical specimens identifies biological subtypes with distinct survival outcomes. et al. Proteome-centric cross-omics characterization and integrated network analyses of triple-negative breast cancer. Breast cancer quantitative proteome and proteogenomic landscape. Krug, K. et al. Proteogenomic landscape of breast cancer tumorigenesis and targeted therapy. Mertins, P. et al. Proteogenomics connects somatic mutations to signalling in breast cancer. Integrated omics of metastatic colorectal cancer. Vasaikar, S. et al. Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities. Wang, J. et al. Colorectal cancer cell line proteomes are representative of primary tumors and predict drug sensitivity. Li, Y. et al. Histopathologic and proteogenomic heterogeneity reveals features of clear cell renal cell carcinoma aggressiveness. Qu, Y. et al. A proteogenomic analysis of clear cell renal cell carcinoma in a Chinese population. Qu, Y. et al. Proteogenomic characterization of MiT family translocation renal cell carcinoma. Dong, L. et al. Proteogenomic characterization identifies clinically relevant subgroups of intrahepatic cholangiocarcinoma. Integrated proteogenomic characterization of HBV-related hepatocellular carcinoma. The proteomic characterization of the peritumor microenvironment in human hepatocellular carcinoma. Jiang, Y. et al. Proteomics identifies new therapeutic targets of early-stage hepatocellular carcinoma. Ng, C. K. Y. et al. Integrative proteogenomic characterization of hepatocellular carcinoma across etiologies and stages. et al. Proteogenomics of non-smoking lung cancer in East Asia delineates molecular signatures of pathogenesis and progression. Gillette, M. A. et al. Proteogenomic characterization reveals therapeutic vulnerabilities in lung adenocarcinoma. Lehtiö, J. et al. Proteogenomics of non-small cell lung cancer reveals molecular subtypes associated with specific therapeutic targets and immune evasion mechanisms. Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. Soltis, A. R. et al. Proteogenomic analysis of lung adenocarcinoma reveals tumor heterogeneity, survival determinants, and therapeutically relevant pathways. Stewart, P. A. et al. Proteogenomic landscape of squamous cell lung cancer. et al. Integrative proteomic characterization of human lung adenocarcinoma. McDermott, J. E. et al. Proteogenomic characterization of ovarian HGSC implicates mitotic kinases, replication stress in observed chromosomal instability. Integrated proteogenomic characterization of human high-grade serous ovarian cancer. Cao, L. et al. Proteogenomic characterization of pancreatic ductal adenocarcinoma. Hyeon, D. Y. et al. Proteogenomic landscape of human pancreatic ductal adenocarcinoma in an Asian population reveals tumor cell-enriched and immune-rich subtypes. The proteogenomic landscape of curable prostate cancer. Fan, Y. et al. Proteomic profiling of gastric signet ring cell carcinoma tissues reveals characteristic changes of the complement cascade pathway. Ge, S. et al. A proteomic landscape of diffuse-type gastric cancer. Li, Y. et al. Proteomic characterization of gastric cancer response to chemotherapy and targeted therapy reveals new therapeutic strategies. Huang, C. et al. Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. Li, S. et al. Integrative proteomic characterization of adenocarcinoma of esophagogastric junction. Large-scale and high-resolution mass spectrometry-based proteomics profiling defines molecular subtypes of esophageal cancer for therapeutic targeting. Bateman, N. W. et al. Proteogenomic landscape of uterine leiomyomas from hereditary leiomyomatosis and renal cell cancer patients. Dou, Y. et al. Proteogenomic characterization of endometrial carcinoma. Ruepp, A. et al. CORUM: the comprehensive resource of mammalian protein complexes—2009. Havugimana, P. C. et al. Scalable multiplex co-fractionation/mass spectrometry platform for accelerated protein interactome discovery. A. et al. Atlas of clinically distinct cell states and ecosystems across human solid tumors. Lo Surdo, P. et al. SIGNOR 3.0, the signaling network open resource 3.0: 2022 update. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Romanov, N. et al. Disentangling genetic and environmental effects on the proteotypes of individuals. Guardia, C. M., De Pace, R., Mattera, R. & Bonifacino, J. S. Neuronal functions of adaptor complexes involved in protein sorting. Koopmans, F. et al. SynGO: an evidence-based, expert-curated knowledge base for the synapse. Huang, L.-H. et al. Postprandial chylomicron output and transport through intestinal lymphatics are not impaired in active Crohn's disease. Ghoshal, S., Witta, J., Zhong, J., de Villiers, W. & Eckhardt, E. Chylomicrons promote intestinal absorption of lipopolysaccharides. Kopec, A. K. & Luyendyk, J. P. Role of fibrin(ogen) in progression of liver disease: guilt by association? Northup, P. G. & Caldwell, S. H. Coagulation in liver disease: a guide for the clinician. Ashburner, M. et al. Gene Ontology: tool for the unification of biology. Ghoussaini, M. et al. Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. The prevalence and comorbidity of tic disorders and obsessive–compulsive disorder in Chinese school students aged 6–16: a national survey. Kumar, A., Trescher, W. & Byler, D. Tourette syndrome and comorbid neuropsychiatric conditions. Davies, M. et al. ChEMBL web services: streamlining access to drug discovery data and utilities. Dickinson, M. E. et al. High-throughput discovery of novel developmental phenotypes. Sadler, M. C., Auwerx, C., Deelen, P. & Kutalik, Z. Multi-layered genetic approaches to identify approved drug targets. Barrio-Hernandez, I. et al. Network expansion of genetic associations defines a pleiotropy map of human cell biology. Lee, I., Blom, U. M., Wang, P. I., Shim, J. E. & Marcotte, E. M. Prioritizing candidate disease genes by network-based boosting of genome-wide association data. Using brain cell-type-specific protein interactomes to interpret neurodevelopmental genetic signals in schizophrenia. Protein interaction studies in human induced neurons indicate convergent biology underlying autism spectrum disorders. O'Neill, A. C. et al. Spatial centrosome proteome of human neural cells uncovers disease-relevant heterogeneity. Drummond, E. et al. Phosphorylated tau interactome in the human Alzheimer's disease brain. Tracy, T. E. et al. Tau interactome maps synaptic and mitochondrial processes associated with neurodegeneration. Runge, K. et al. Neurodegeneration markers in the cerebrospinal fluid of 100 patients with schizophrenia spectrum disorder. Lewis, A. S. et al. Alternatively spliced isoforms of TRIP8b differentially control h channel trafficking and function. Santoro, B. et al. TRIP8b regulates HCN1 channel trafficking and gating through two distinct C-terminal interaction sites. Popova, N. V., Plotnikov, A. N., Ziganshin, R. K., Deyev, I. E. & Petrenko, A. G. Analysis of proteins interacting with TRIP8b adapter. van Oostrum, M. et al. The proteomic landscape of synaptic diversity across brain regions and cell types. Roth, F. C. & Hu, H. An axon-specific expression of HCN channels catalyzes fast action potential signaling in GABAergic interneurons. Cai, W., Liu, S.-S., Li, B.-M. & Zhang, X.-H. Presynaptic HCN channels constrain GABAergic synaptic transmission in pyramidal cells of the medial prefrontal cortex. Kaar, S. J., Angelescu, I., Marques, T. R. & Howes, O. D. Pre-frontal parvalbumin interneurons in schizophrenia: a meta-analysis of post-mortem studies. Arime, Y. et al. Activation of prefrontal parvalbumin interneurons ameliorates working memory deficit even under clinically comparable antipsychotic treatment in a mouse model of schizophrenia. Hashimoto, T. et al. Gene expression deficits in a subclass of GABA neurons in the prefrontal cortex of subjects with schizophrenia. Lewis, D. A., Hashimoto, T. & Volk, D. W. Cortical inhibitory neurons and schizophrenia. Bitanihirwe, B. K. Y., Lim, M. P., Kelley, J. F., Kaneko, T. & Woo, T. U. W. Glutamatergic deficits and parvalbumin-containing inhibitory neurons in the prefrontal cortex in schizophrenia. Towards a structurally resolved human protein interaction network. Madeira, F. et al. 14-3-3-Pred: improved methods to predict 14-3-3-binding phosphopeptides. Identification of HCN1 as a 14-3-3 client. Gonzalez-Lozano, M. A. et al. Stitching the synapse: cross-linking mass spectrometry into resolving synaptic protein interactions. Moon, H. M., Hippenmeyer, S., Luo, L. & Wynshaw-Boris, A. LIS1 determines cleavage plane positioning by regulating actomyosin-mediated cell membrane contractility. & Foster, L. J. Meta-analysis defines principles for the design and analysis of co-fractionation mass spectrometry experiments. Frommelt, F. et al. DIP-MS: ultra-deep interaction proteomics for the deconvolution of protein complexes. Liu, Y. et al. Multi-omic measurements of heterogeneity in HeLa cells across laboratories. Fossati, A. et al. System-wide profiling of protein complexes via size exclusion chromatography–mass spectrometry (SEC–MS). In Shotgun Proteomics: Methods and Protocols (eds Carrera, M. & Mateos, J.) Bludau, I. et al. Complex-centric proteome profiling by SEC–SWATH–MS for the parallel detection of hundreds of protein complexes. Wildschut, M. H. E. et al. Proteogenetic drug response profiling elucidates targetable vulnerabilities of myelofibrosis. Comparison of protein quantification in a complex background by DIA and TMT workflows with fixed instrument time. The ProteomeXchange consortium at 10 years: 2023 update. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. This project was funded by OTAR, under the OTAR2066 Context Networks project. were supported by the Swiss Federal Institute of Technology (ETH) grant application ETH-25 15-2 and the strategic focus area of ETH Zurich ‘Personalized Health and Related Technologies'. were supported by the Innovative Medicines Initiative project ULTRA-DD (FP07/2007-2013, grant no. were supported by European Molecular Biology Laboratory (EMBL) core funding. was supported by the National Human Genome Research Institute, Office of Director (OD/DPCPSI/ODSS), National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Institute of General Medical Sciences, National Institute of Diabetes and Digestive and Kidney Diseases, National Eye Institute, National Cancer Institute and National Heart, Lung and Blood Institute of the National Institutes of Health (award no. U24HG007822; the content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health) and by EMBL core funding. is supported by a Wellcome Grant (reference 220540/Z/20/A, ‘Wellcome Sanger Institute Quinquennial Review 2021–2026'). is supported by the SciLifeLab and Wallenberg Data-Driven Life Science Program (grant KAW 2020.0239). was supported by the Helmut Horten Stiftung and the ETH Zurich Foundation. Open access funding provided by Swiss Federal Institute of Technology Zurich. Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich, Switzerland Diederik S. Laman Trip, Marc van Oostrum, Fabian Frommelt, Ruedi Aebersold, Matthias Gstaiger, Bernd Wollscheid & Pedro Beltrao Swiss Institute of Bioinformatics, Lausanne, Switzerland Diederik S. Laman Trip & Pedro Beltrao Department of Health Sciences and Technology, Institute of Translational Medicine, ETH Zurich, Zurich, Switzerland Marc van Oostrum & Bernd Wollscheid European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK Danish Memon, Kalpana Panneerselvam, Henning Hermjakob, Sandra Orchard, Ellen M. McDonagh & Pedro Beltrao Open Targets, Wellcome Genome Campus, Cambridge, UK Danish Memon, Kalpana Panneerselvam, Henning Hermjakob, Sandra Orchard, Gosia Trynka, Ellen M. McDonagh & Pedro Beltrao Gulbenkian Institute for Molecular Medicine, Oeiras, Portugal Delora Baptista & Pedro Beltrao Computational Biology, Functional Genomics, GSK, Stevenage, UK Department of Biology, University of Rome Tor Vergata, Rome, Italy Gosia Trynka & Ellen M. McDonagh Science for Life Laboratory, Department of Microbiology, Tumor and Cell Biology, Karolinska Institute, Solna, Sweden You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceptualized and designed the project. performed most of the computational analysis, D.M. performed the preliminary computational analysis and D.B. performed the structural modeling of protein–protein interactions. performed and analyzed the cofractionation experiments. supervised parts of the work. assisted with access and discussions of the integration of genetic data from the OTAR genetics portal. wrote the paper with input from all authors. Correspondence to Diederik S. Laman Trip or Pedro Beltrao. is an employee of GSK. The other authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Methods and Figs. All supplementary tables and file descriptions. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Laman Trip, D.S., van Oostrum, M., Memon, D. et al. A tissue-specific atlas of protein–protein associations enables prioritization of candidate disease genes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41562-025-02171-z'>Towards a personalized happiness approach to capturing change in satisfaction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-02 09:12:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Human Behaviour (2025)Cite this article Metrics details Contemporary approaches examining the determinants of happiness have posited that happiness is determined bidirectionally by both top-down, global life satisfaction and bottom-up, domain satisfaction processes. We propose a personalized happiness perspective, suggesting that the determinants and consequences of happiness are idiographic (that is, specific) to each individual rather than assumed to be the same for all. We showed the utility of a personalized happiness approach by testing associations between life and domain satisfaction at both the population and personalized levels using nationally representative data of 40,074 German, British, Swiss, Dutch and Australian participants tracked for up to 33 years. The majority of participants (41.4–50.8%) showed primarily unidirectional associations between domain satisfactions and life satisfaction, and only 19.3–25.9% of participants showed primarily bidirectional associations. Moreover, the population models differed from personalized models, suggesting that aggregated, population-level research fails to capture individual differences in personalized happiness, showing the importance of a personalized happiness approach. Patterns of individual differences are robust, yet distinguishing between individual-level patterns and random error is challenging, highlighting the need for future work and innovative approaches to study personalized happiness. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Each data source was drawn from publicly accessible longitudinal studies. However, data cannot be shared directly due to contractual obligations and agreements necessary to use the data. In the text and online materials, we provide detailed information on how to gain access to raw data sources. All data analysis was conducted on the raw data files provided by the data maintainers, and all the utilized variables are documented in detailed in the study codebook. For GSOEP data, see https://www.diw.de/en/diw_01.c.601584.en/data_access.html; for BHPS, https://www.iser.essex.ac.uk/bhps/about/latest-release-of-bhps-data; for SHP, https://forsbase.unil.ch/project/study-public-overview/15632/0/ for HILDA, https://melbourneinstitute.unimelb.edu.au/hilda/for-data-users; and for LISS, https://liss.statements.centerdata.nl/. All code, results, tables and figures are available on the Open Science Framework (https://osf.io/r7vgq/) and GitHub (https://github.com/emoriebeck/personalised-happiness) and depicted in an interactive R Shiny web app (https://emoriebeck.shinyapps.io/personalised-happiness/). All scripts proceed from raw data that can be attained by downloading data files from the maintainers for each sample. Beck, E. D. & Jackson, J. J. A mega-analysis of personality prediction: robustness and boundary conditions. Google Scholar Lyubomirsky, S., King, L. & Diener, E. The benefits of frequent positive affect: does happiness lead to success? Article PubMed Google Scholar Diener, E., Oishi, S. & Lucas, R. E. National accounts of subjective well-being. Article PubMed Google Scholar Beck, E. D. & Jackson, J. J. Consistency and change in idiographic personality: a longitudinal ESM network study. Article PubMed Google Scholar Beck, E. D. & Jackson, J. J. Idiographic traits: a return to Allportian approaches to personality. Google Scholar Diener, E. Subjective well-being. Feist, G. J., Bodner, T. E., Jacobs, J. F., Miles, M. & Tan, V. Integrating top-down and bottom-up structural models of subjective well-being: a longitudinal investigation. Heller, D., Watson, D. & Ilies, R. The role of person versus situation in life satisfaction: a critical examination. Article PubMed Kuykendall, L., Tay, L. & Ng, V. Leisure engagement and subjective well-being: a meta-analysis. Article PubMed Jebb, A. T., Tay, L., Diener, E. & Oishi, S. Happiness, income satiation and turning points around the world. Article PubMed Kahneman, D. & Deaton, A. High income improves evaluation of life but not emotional well-being. Natl Acad. Helliwell, J. F., Huang, H., Norton, M., Goff, L. & Wang, S. in World Happiness Report 2023 11th edn (eds Helliwell, J. F. et al.) Ch. 2 (Sustainable Development Solutions Network, 2023). Andrews, F. M. & Withey, S. B. Social Indicators of Well-Being: Americans' Perceptions of Life Quality (Springer, 1976). Campbell, A., Converse, P. E. & Rodgers, W. L. The Quality of American Life: Perceptions, Evaluations, and Satisfactions (Russell Sage Foundation, 1976). Deaton, A. The financial crisis and the well-being of Americans: 2011 OEP Hicks Lecture. Nakazato, N., Schimmack, U. & Oishi, S. Effect of changes in living conditions on well-being: a prospective top-down bottom-up model. Diener, E. & Lucas, R. E. in Well-Being: The Foundations of Hedonic Psychology (eds Kahneman, D. et al.) 213–229 (Russell Sage Foundation, 1999). Brickman, P., Coates, D. & Janoff-Bulman, R. Lottery winners and accident victims: is happiness relative? Brickman, P. & Campbell, D. T. in Adaption-Level Theory (ed. Appley, M. H.) 287–301 (Academic, 1971). Diener, E., Lucas, R. E. & Scollon, C. N. in The Science of Well-Being: The Collected Works of Ed Diener (ed. Diener, E.) 103–118 (Springer, 2009); https://doi.org/10.1007/978-90-481-2350-6_5 Arvey, R. D., Bouchard, T. J., Segal, N. L. & Abraham, L. M. Job satisfaction: environmental and genetic components. Bartels, M. Genetics of wellbeing and its components satisfaction with life, happiness, and quality of life: a review and meta-analysis of heritability studies. Ilies, R. & Judge, T. A. On the heritability of job satisfaction: the mediating role of personality. Article PubMed Lykken, D. & Tellegen, A. Happiness is a stochastic phenomenon. Stubbe, J. H., Posthumn, D., Boomsma, D. I. & De Geus, E. J. C. Heritability of life satisfaction in adults: a twin-family study. Ludwig, J. et al. Neighborhood effects on the long-term well-being of low-income adults. Luechinger, S. Valuing air quality using the life satisfaction approach. Bolier, L. et al. Positive psychology interventions: a meta-analysis of randomized controlled studies. BMC Public Health 13, 119 (2013). White, C. A., Uttl, B. & Holder, M. D. Meta-analyses of positive psychology interventions: the effects are much smaller than previously reported. PLoS ONE 14, e0216588 (2019). Bialowolski, P. & Weziak-Bialowolska, D. Longitudinal evidence for reciprocal effects between life satisfaction and job satisfaction. J. Happiness Stud. Judge, T. A. & Watanabe, S. Another look at the job satisfaction-life satisfaction relationship. Gana, K. et al. Relationship between life satisfaction and physical health in older adults: a longitudinal test of cross-lagged and simultaneous effects. Health Psychol. Article PubMed Lucas, R. E. Top-down and bottom-up models of life satisfaction judgments. In 6th International German Socio-Economic Panel Study User Conference (2004). Veenhoven, R. Is happiness relative? Erdogan, B., Bauer, T. N., Truxillo, D. M. & Mansfield, L. R. Whistle while you work: a review of the life satisfaction literature. Fisher, A. J., Medaglia, J. D. & Jeronimus, B. F. Lack of group-to-individual generalizability is a threat to human subjects research. Natl Acad. Molenaar, P. C. M. A manifesto on psychology as idiographic science: bringing the person back into scientific psychology, this time forever. Borsboom, D., Mellenbergh, G. J. & van Heerden, J. The theoretical status of latent variables. Article PubMed Wright, A. G. C. & Woods, W. C. Personalized models of psychopathology. Article PubMed Beck, E. D. & Jackson, J. J. Personalized prediction of behaviors and experiences: an idiographic person-situation test. Beckmann, J. S. & Lew, D. Reconciling evidence-based medicine and precision medicine in the era of big data: challenges and opportunities. Genome Med. Khoury, M. J. & Galea, S. Will precision medicine improve population health? Epskamp, S., Waldorp, L. J., Mõttus, R. & Borsboom, D. The Gaussian graphical model in cross-sectional and time-series data. Wild, B. et al. A graphical vector autoregressive modelling approach to the analysis of electronic diary data. BMC Med. Freeman, L. C. Centrality in social networks conceptual clarification. Barrat, A., Barthélemy, M., Pastor-Satorras, R. & Vespignani, A. The architecture of complex weighted networks. Natl Acad. Bringmann, L. F. et al. Assessing temporal emotion dynamics using networks. Article PubMed Headey, B. & Wearing, A. in Subjective Well-Being: An Interdisciplinary Perspective (eds Strack, F. et al.) 49–73 (Pergamon, 1991). Funder, D. C. & Ozer, D. J. Evaluating effect size in psychological research: sense and nonsense. Methods Pract. Wang, L. P. & Maxwell, S. E. On disaggregating between-person and within-person effects with longitudinal data using multilevel models. Article PubMed Snijders, T. A. & Bosker, R. J. Multilevel Analysis: An Introduction to Basic and Advanced Multilevel Modeling (SAGE, 2011). van Agteren, J. et al. A systematic review and meta-analysis of psychological interventions to improve mental wellbeing. Article PubMed Easterlin, R. A. in Economics and Happiness: Framing the Analysis (eds Bruni, L. & Porta, P. L.) 29–64 (Oxford University Press, 2005); https://doi.org/10.1093/0199286280.003.0002 Haar, J. M., Russo, M., Suñe, A. & Ollier-Malaterre, A. Outcomes of work–life balance on job satisfaction, life satisfaction and mental health: a study across seven cultures. Hanley, A., Warner, A. & Garland, E. L. Associations between mindfulness, psychological well-being, and subjective well-being with respect to contemplative practice. J. Happiness Stud. Bache, I. How does evidence matter? understanding ‘what works' for wellbeing. Rohrer, J., Seifert, I. S., Arslan, R. C., Sun, J. & Schmukle, S. C. The effects of satisfaction with different domains of life on general life satisfaction vary between individuals (but we cannot tell you why). Collabra Psychol. Nissen, A. T. & Beck, E. D. The joy is mine: re-centering the person in the study of well-being. Preprint at OSF https://doi.org/10.31219/osf.io/3xvfw_v2 (2024). Beck, E. D. & Jackson, J. J. in The Handbook of Personality Dynamics and Processes (ed. Rauthmann, J. F.) 75–100 (Academic, 2021); https://doi.org/10.1016/B978-0-12-813995-0.00004-2 Beck, E. D. & Jackson, J. J. in Measuring and Modeling Persons and Situations (eds Wood, D. et al.) 465–497 (Academic, 2021); https://doi.org/10.1016/B978-0-12-819200-9.00003-X Emmons, R. A. Personal strivings: an approach to personality and subjective well-being. Sonnentag, S. Dynamics of well-being. Beck, E. et al. A taxonomy of data synthesis: a tutorial. Preprint at PsyArXiv https://doi.org/10.31234/osf.io/m3ywe (2024). Ram, N., Brinberg, M., Pincus, A. L. & Conroy, D. E. The questionable ecological validity of ecological momentary assessment: considerations for design and analysis. Park, J. J., Chow, S.-M., Fisher, Z. F. & Molenaar, P. C. M. Affect and personality: ramifications of modeling (non-)directionality in dynamic network models. Batra, R., Johal, S. K., Chen, M. & Ferrer, E. Consequences of sampling frequency on the estimated dynamics of AR processes using continuous-time models. Methods https://doi.org/10.1037/met0000595 (2023). Saef, R. M., Beck, E. D. & Jackson, J. J. in Examining and Exploring the Shifting Nature of Occupational Stress and Well-Being Vol. 19 (eds Harms, P. D. et al.) 179–200 (Emerald, 2021). Schaefer, B., Haehner, P. & Luhmann, M. Network dynamics in subjective well-being and their differences across age groups. Article PubMed Goebel, J. et al. The German Socio-Economic Panel (SOEP). Für Natl Stat. Fumagalli, L., Knies, G. & Buck, N. Understanding Society, The UK Household Longitudinal Study, Harmonised British Household Panel Survey (BHPS) User Guide (Institute for Social and Economic Research, University of Essex and ESRC, 2017); https://repository.essex.ac.uk/21094/1/bhps-harmonised-user-guide.pdf Platt, L., Knies, G., Luthra, R., Nandi, A. & Benzeval, M. Understanding Society at 10 years. Tillmann, R. et al. The Swiss household panel study: observing social change since 1999. Life Course Stud. Wilkins, R., Laß, I., Butterworth, P. & Vera-Toscano, E. The Household, Income and Labour Dynamics in Australia Survey: Selected Findings from Waves 1 to 16 (Melbourne Institute: Applied Economic & Social Research, University of Melbourne, 2018); https://melbourneinstitute.unimelb.edu.au/__data/assets/pdf_file/0009/2874177/HILDA-report_Low-Res_10.10.18.pdf Scherpenzeel, A. & Das, M. in Social and Behavioral Research and the Internet: Advances in Applied Methods and Research Strategies (eds Das, M. et al.) 77–103 (Taylor & Francis, 2010). Cohen, P., Cohen, J., Aiken, L. S. & West, S. G. The problem of units and the circumstance for POMP. Epskamp, S., Cramer, A. O. J., Waldorp, L. J., Schmittmann, V. D. & Borsboom, D. qgraph: network visualizations of relationships in psychometric data. Fruchterman, T. M. J. & Reingold, E. M. Graph drawing by force-directed placement. Borkenau, P. & Ostendorf, F. The big five as states: how useful is the five-factor model to describe intraindividual variations over time? Rohrer, J. M. & Murayama, K. These are not the effects you are looking for: causality and the within-/between-persons distinction in longitudinal data analysis. Methods Pract. Haslbeck, J. M. B. & Waldorp, L. J. mgm: estimating time-varying mixed graphical models in high-dimensional data. R Core Team. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2013); http://www.R-project.org/ Hamaker, E. L. & Grasman, R. P. To center or not to center? Investigating inertia with a multilevel autoregressive model. Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. Skinner, H. A. Differentiating the contribution of elevation, scatter and shape in profile similarity. Rohrer, J. M. Thinking clearly about correlations and causation: graphical causal models for observational data. Methods Pract. Download references 's time was supported by National Institute on Aging grants T32 AG00030-3, R01-AG067622 and R01-AG018436. Department of Psychology, University of California, Davis, Davis, CA, USA Emorie D. Beck Department of Psychology, University of Toronto, Toronto, Ontario, Canada Felix Cheung Department of Psychology, University of Tulsa, Tulsa, OK, USA Stuti Thapa Department of Psychological and Brain Sciences, Washington University in St. Louis, St. Louis, MO, USA Joshua J. Jackson You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar : conceptualization, data curation, formal analysis, investigation, methodology, visualization, writing—original draft, writing—review and editing. : conceptualization, data curation, investigation, writing—original draft. : conceptualization, writing—original draft, writing—review and editing. : conceptualization, investigation, visualization, writing—original draft, writing—review and editing. Correspondence to Emorie D. Beck. The authors declare no competing interests. Nature Human Behaviour thanks Todd Kashdan, Julia Rohrer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Tables 1–3 and Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Beck, E.D., Cheung, F., Thapa, S. et al. Towards a personalized happiness approach to capturing change in satisfaction. Nat Hum Behav  (2025). Download citation Received: 16 June 2023 Accepted: 12 March 2025 Published: 02 May 2025 DOI: https://doi.org/10.1038/s41562-025-02171-z Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Human Behaviour (Nat Hum Behav) ISSN 2397-3374 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            