
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-21</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09987-9'>Ageing promotes microglial accumulation of slow-degrading synaptic proteins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 16:53:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Neurodegenerative diseases affect 1 in 12 people globally and remain incurable. Central to their pathogenesis is a loss of neuronal protein maintenance and the accumulation of protein aggregates with ageing1,2. Here we engineered bioorthogonal tools3 that enabled us to tag the nascent neuronal proteome and study its turnover with ageing, its propensity to aggregate and its interaction with microglia. We show that neuronal protein half-life approximately doubles on average between 4-month-old and 24-month-old mice, with the stability of individual proteins differing among brain regions. Furthermore, we describe the aged neuronal ‘aggregome', which encompasses 1,726 proteins, nearly half of which show reduced degradation with age. The aggregome includes well-known proteins linked to diseases and numerous proteins previously not associated with neurodegeneration. Notably, we demonstrate that neuronal proteins accumulate in aged microglia, with 54% also displaying reduced degradation and/or aggregation with age. Among these proteins, synaptic proteins are highly enriched, which suggests that there is a cascade of events that emerge from impaired synaptic protein turnover and aggregation to the disposal of these proteins, possibly through microglial engulfment of synapses. These findings reveal the substantial loss of neuronal proteome maintenance with ageing, which could be causal for age-related synapse loss and cognitive decline. Ageing is accompanied by a loss of proteostasis, which involves the maintenance of a balanced and functional proteome1,4. The loss of proteostasis in the brain contributes to age-associated vulnerability to reduced cognitive and motor abilities and neurodegenerative diseases2. Indeed, experimentally compromising broad proteostasis pathways can induce dementia-like phenotypes2,5,6. Understanding the dynamics of proteostasis in a neuron-specific manner may define mechanisms or individual proteins that could be exploited for therapeutic purposes. However, despite the emergence and application of several tools to study cellular proteomes3,7,8,9,10,11,12,13,14,15,16, such research has been hindered by a lack of robust models to examine protein dynamics in a cell-specific manner in mammals. Here we develop in vivo models that enable robust tagging of nascent proteomes with non-canonical biorthogonal amino acids in a cell-specific manner through the expression of mutant aminoacyl-tRNA synthetases (aaRSs). We leverage these models to study key features of neuronal proteostasis dynamics with ageing to provide detailed insights into the decline of neuronal proteostasis with age. We also describe a microglia-mediated mechanism that maintains neuronal proteostasis. Expanding on our previous in vitro studies3, we generated two new bioorthogonal non-canonical amino acid tagging (BONCAT) knock-in mouse lines with cassettes that express the mutant aaRSs flox-stop-flox-eGFP-p2a-PheRS(T413G) and flox-stop-flox-eGFP-p2a-TyrRS(Y43G) (hereafter, termed PheRS* and TyrRS*, respectively) (Fig. In vitro studies using these constructs confirmed that proteome tagging depended on the incorporation of the non-canonical amino acids during protein synthesis (Extended Data Fig. We first compared the protein tagging efficacy of our models to each other and to that of the current standard BONCAT knock-in mouse line based on the expression of a mutant methionine aaRS (hereafter, termed MetRS*)7,8. Each of the BONCAT lines was crossed to a Camk2a-cre driver, and the resulting offspring (Camk2a-cre+/−;BONCAT+/–) were uniformly treated with their respective azido-modified amino acid (AzAA) to evaluate nascent protein labelling in CAMK2A+ neurons (Fig. Examination by in-gel fluorescence revealed that the Camk2a-cre;PheRS* model showed a high fluorescence signal over its respective background control, whereas the Camk2a-cre;TyrRS* and Camk2a-cre;MetRS* models did not show an appreciable difference relative to their respective background controls (Extended Data Fig. These results were supported by in situ tissue staining for the azide-modified proteins (Fig. Labelled proteins in brain sections from Camk2a-cre;PheRS* mice colocalized to GFP+ neurons with the expected spatial distribution of CAMK2A+ neurons (Fig. Last, we evaluated protein labelling by performing liquid chromatography with mass spectrometry (LC–MS) on BONCAT-labelled proteins enriched by bead-based pull-down (Extended Data Fig. Principal component analysis (PCA) clearly separated the different models (Fig. 1f) and a high fold change in labelled signals relative to the background (Fig. 1i,j) were observed in the Camk2a-cre;PheRS* model. 1k,l) did not induce HSP90 expression (Extended Data Fig. This result suggests that azide-modified residues do not induce proteostatic stress or a local immune response. a, Schematics of BONCAT knock-in mice and methodology. The Camk2a-cre;PheRS* (Cam;PheRS*) and Camk2a-cre;TyrRS* (Cam;TyrRS*) lines were developed in this study. c, Images of BONCAT-labelled proteins (Click-555) in brain sections from the indicated Camk2a-cre+/−;BONCAT+/− mice. Cb, cerebellum; Ctr, cortex; Ob, olfactory bulb. e, Venn diagram comparing the number of proteins identified in each Camk2a-cre+/−;BONCAT+/− mouse line. f, Heatmap comparing P values of proteins identified in the indicated Camk2a-cre+/−;BONCAT+/− mouse line. g, Top, volcano plots showing the enrichment of proteins identified in the indicated Camk2a-cre+/−;BONCAT+/− mouse lines relative to background controls (WT). h, Volcano plot for Camk2a-cre;PheRS* mice as in g, with dots colour-coded by cell-type enrichment. CellMarker and Panglao DB databases were used for analyses. i, GO cellular component analysis of BONCAT-labelled proteins in Camk2a-cre;PheRS* mice. j, Images of BONCAT-labelled proteins in the motor cortex, striatum and hippocampus of a Camk2a-cre;PheRS* mouse. k, Venn diagram comparing the number of different proteins in the motor cortex, hippocampus and striatum of Camk2a-cre;PheRS* mice. l, PCA based on the abundance of BONCAT-labelled proteins from the motor cortex, striatum and hippocampus of Camk2a-cre;PheRS* mice. m, Heatmap comparing the z scored abundance of BONCAT-labelled proteins from the motor cortex, striatum and hippocampus of Camk2a-cre;PheRS* mice. Protein clusters enclosed by a white dotted line are regional marker proteins. n, Heatmap comparing pathway fold enrichment of the top ten GO biological processes for the motor cortex, striatum and hippocampus based on the regional marker proteins in m. n = 4–5 BONCAT mice and 4 respective background control mice in d–h. n = 4 mice per experimental group in k and l. P values in e,g,h and k were derived from two-tailed, two-sample Student's t-tests. To test whether the efficacy of protein labelling by each BONCAT line is related to the tissue examined, we performed similar experiments as described above in CMV-cre;BONCAT mice to induce ubiquitous cellular labelling in all tissues17. Depending on the tissue examined, different BONCAT lines had varying strengths in labelling tissue proteins (Extended Data Fig. Several factors potentially contribute to the differences in labelling among tissues, including varying cell-type-specific expression of cognate tRNA molecules and cell states (Supplementary Text). These data demonstrate the strengths and potential utility of all three BONCAT lines in different tissue contexts. Given the robust labelling of neuronal proteins in Camk2a-cre;PheRS* mice, we further characterized this model. Many proteins (606) identified were annotated as neuronal (LY6H, SACS and SCNA, among others) with few (65) annotated as specific to other cell types (Fig. As expected, many proteins labelled were marker genes of glutamatergic neurons (Extended Data Fig. 2b–e), a finding that was supported by in situ staining (Extended Data Fig. All major neuronal anatomical features were represented by hundreds of proteins (Fig. To assess regional proteomes in Camk2a-cre;PheRS* mice, we dissected the motor cortex, striatum and hippocampus, regions that exhibited robust labelling (Fig. 1j), and performed LC–MS on the enriched labelled proteins. A total of 3,054 proteins were commonly identified among all regions, but each region had 276–338 uniquely identified proteins (Fig. PCA of the regional CAMK2A+ neuronal-labelled proteomes separated all three regions (Fig. 1l), which was reflected by hierarchical clustering and heatmap analyses (Fig. We validated three regionally enriched proteins by immunostaining in situ (Extended Data Fig. Gene ontology (GO) biological process enrichment showed that each cluster or region was relatively unique in their pathway representation, which highlighted their regional specialization (Fig. Given the fundamental role of protein turnover in neurodegenerative diseases, particularly in long-lived, nonmitotic neurons18, we sought to determine how neuronal protein degradation changes with age. To rapidly deliver the BONCAT machinery to aged mice, we developed an adeno-associated virus (AAV) expression vector encoding a Camk2a-driven PheRS* (Extended Data Fig. Mice transduced with this construct showed significantly higher labelling than background controls (Extended Data Fig. The AAV construct also enabled protein labelling in aged, 21-month-old mice, which facilitated comparisons of the ‘aged' and ‘young' neuronal-labelled proteomes (Extended Data Fig. To study how protein degradation changes with age, young (4-month-old), middle-aged (12-month-old) and aged (24-month-old) mice were transduced with AAV:Camk2a-PheRS* by retroorbital injection with a pulse-chase AzF administration scheme (Fig. Mice were euthanized at 4 time points within the 2-week chase period, and brain regions were dissected immediately after brain extraction (Fig. 2c) showed a dilution of tagged-protein fluorescence signals that progressed through the chase period, a result indicative of protein degradation. To quantify degradation rates for individual neuronal proteins across brain regions and ages, we enriched for tagged neuronal proteins, multiplexed enriched peptide fractions by tandem mass tag (TMT) labelling and analysed the plexes by LC–MS (Fig. We obtained degradation trajectories of the per cent protein remaining over time for every protein identified for each region and each age (Fig. The average degradation trajectories for all proteins among regions differed (Fig. Moreover, the average degradation trajectories of regions in aged mice relative to their respective regions in young and middle-aged mice were broader or had lower slopes (Fig. This finding indicates that protein degradation slows with ageing and emerges after middle age, a result that was quantitatively supported (Extended Data Fig. a, Schematic of the approach used to study protein degradation by BONCAT. n = 4 BONCAT mice per time point (TP1–TP4) for each age and n = 2 mice per age for background controls. b, In-gel fluorescence images (top and middle) and quantification (bottom) of BONCAT-labelled proteins in whole brain lysates derived from Camk2a-cre;PheRS* mice at the indicated time points in the chase period. c, Images of BONCAT-labelled proteins in the cortex of brain tissue sections from Camk2a-cre;PheRS* mice at the indicated time points in the chase period. Each thin line represents one protein derived from averaging four biological replicates. Proteins were filtered to only exclude proteins with a 5% increase between any two time points. e, Plot of the estimated protein half-life in days for the indicated brain regions and ages (A, aged; MA, middle-aged; Y, young). For each individual brain region, only proteins commonly identified between all ages of that region are plotted. f, Plot of log2[FC] of estimated protein half-life values between the indicated brain regions and ages. Each dot represents one protein and is the same as those displayed in f. g, Bar plot of the number of proteins with an age-increased half-life that are also risk genes for the indicated brain disorders. Proteins with an absolute value difference of >1 were considered regionally vulnerable. P values were determined by two-sided Person's correlation tests. i, Bar plot of the number of regionally vulnerable proteins among the indicated brain regions. P values in e and f were determined by paired two-tailed t-tests between young and aged proteins. ***P < 0.0001 risk genes quantified in g and j were derived from the H-MAGMA study28 and considered only if the originally reported P value was <0.05. AD, Alzheimer's disease; ALS, amyotrophic lateral sclerosis; CAD, coronary artery disease; DDA, data-dependent mode; MDD, major depressive disorder; MS, multiple sclerosis; PD, Parkinson's disease; SCZ, schizophrenia. We estimated protein half-life by using established modelling techniques19,20, which correlated well with direct interpolation of the half-life values from the trajectories (Extended Data Fig. The estimated half-life values further showed relatively stable average half-lives from young to middle age (except for the hypothalamus), with an average increase of 2.27–5.04 days from young to aged mice (Fig. The average fold change in half-life among all regions was 1.2 (around 20% increase, mostly attributed to the hypothalamus) from young to middle age, but approximately 2.03 (100% increase or doubling) from young to aged (Fig. The observation of reduced protein degradation with age is consistent with previous reports21,22,23, as was the lack of correlation between protein abundance and half-life24,25 (Extended Data Fig. 4f). Of the cortical proteins, those with the greatest fold change increase (top 10%) from young to aged were enriched for proteins of the synapse (PPP2R1A, DTNA and DNM2), cell junctions (GRM3, RTN3 and GJA1) and mitochondria (Extended Data Fig. These neuronal features have been observed to be compromised in ageing and dementias26,27. Notably, several hundred of the proteins that exhibited an aged-increased half-life are encoded by neurodevelopmental or neurodegenerative risk genes identified in a study that used Hi-C-coupled multimarker analysis of genomic annotation (H-MAGMA)28 (Fig. Some of the proteins with the most age-increased half-life are encoded by neurodegenerative risk genes such as CPLX1, DCLK1, FERMT2 and YWHAQ (Fig. These proteins localize to cell junctions and the actin cytoskeleton and are involved in cell–cell junction organization and signalling, which implies that reduced protein degradation has repercussions for both the host cells and their signalling partners. Age-reduced degradation only slightly correlated with certain protein features (Extended Data Fig. Next, we compared half-life fold changes of proteins shared across regions. We speculated that proteins with differing half-life changes with age could contribute to regional vulnerability or resilience to ageing and diseases. No two regions were perfectly correlated (Fig. The visual cortex and hypothalamus displayed the fewest changes in half-life fold change (n = 3 proteins, |log2[FCAged/Young Visual cortex] – log2[FCAged/Young Hypothalamus]| > 1), with more changes present when comparing the sensory cortex to either the hippocampus (n = 39 proteins) or hypothalamus (n = 12 proteins) (Fig. In support of the hypothesis that these proteins could confer vulnerability or resilience, several of these proteins are encoded by known neurodegenerative risk genes (Fig. Some of the proteins identified, such as LRPPRC or RAPGEF2, have been experimentally demonstrated to contribute to Alzheimer's disease progression, and others have strong disease correlations. However, the contribution of many other proteins, including DLG2, NLGN3, STXBP1 and TMED10, to ageing and neurodegeneration remains to be elucidated. As half-life values can oversimplify nuances of complex kinetic degradation trajectories such as those shown in Fig. 2d, we performed analyses on the degradation trajectories to extract additional information (Fig. First, we clustered the degradation trajectories on a per region basis (Fig. As an example, the ‘young' sensory cortex had five clusters with visually distinguishable average profiles (Fig. The top five most enriched GO biological process terms of each cluster in the sensory cortex were mostly unique to each cluster and were largely represented by one or two broad biological processes (Fig. This result supported the biological meaningfulness of the clustering, which was generally recapitulated among other brain regions (Extended Data Fig. These data suggest that proteins in similar pathways have coordinated degradation rates, an observation complementary to previous findings that individual proteins in multiprotein complexes share similar half-lives24,29. Red lines represent proteins closer to the cluster average. b, Plot of the average degradation trajectory for each cluster visualized in a from the sensory cortex of young mice. c, Bar plot comparing the slopes of the kinetic degradation trajectories of each protein in each cluster (C1–C5) for the sensory cortex of young mice. d, Heatmap of the top five most significant GO biological processes identified for each cluster in the sensory cortex of young mice. Heatmap colours represent the fold enrichment for each pathway. e, Overlap of degradation trajectories of the six clusters identified in the sensory cortex of young (4-month-old) and aged (24-month-old) mice, with lines colour-coded by age. f, Bar plot comparing the integral values of proteins from young and aged mice in each cluster of the sensory cortex. g, Bar plot comparing the integral values of proteins from young and aged mice on a per-region basis. h, Heatmap of the top five most enriched GO biological processes identified for each cluster in each brain region examined. Heatmap colours represent the fold enrichment for each pathway. The data in c and e–g were determined by an ordinary one-way analysis of variance (ANOVA) with significant comparisons identified by Tukey tests. We next compared how clusters change with age in the sensory cortex (Fig. For the sensory cortex and all other brain regions examined, profiles from aged mice had a larger integral value (area under the curve) than profiles from young mice for each cluster (Fig. 5a and Supplementary Table 2d), which signified reduced degradation rates in aged mice. Only the visual cortex and hypothalamus showed increased average integral values from young to middle-aged mice (Extended Data Fig. 5c), which signified earlier degradation deficits in these regions. We calculated the average difference of the integral values for proteins from aged and young mice of each cluster to obtain a delta integral (ΔIntegral) score for each cluster (Fig. In the sensory cortex, cluster 1, which was enriched for synapse transmission functions (Fig. 3d), had one of the largest ΔIntegral scores (1.55) (Fig. By contrast, cluster 5, which was enriched for metabolic processes (Fig. The ΔIntegral scores suggest that certain biological processes, such as synaptic transmission, are more vulnerable to the consequences of age-related degradation than other processes, such as metabolism, in the sensory cortex. We extended this integral score and pathway analysis to all clusters of all regions (Fig. Although most clusters, regardless of region, had similar integral scores, a few clusters had substantially higher or lower ΔIntegral scores (Fig. 3h, ΔIntegral annotation on the heatmap), which suggests that there is more prominent vulnerability or resilience, respectively, with ageing. There are many potential causes for the age-related reduction in protein degradation, including the formation of protein aggregates4. Neuronal protein aggregation increases with age in mice, and aggregates have been detected (using Proteostat30) in human brains from old individuals (Fig. Moreover, protein aggregates are commonly associated in age-related brain diseases18. Therefore, we followed up on the connection of aggregation and age-reduced protein degradation. Combining protein aggregate isolation techniques31 with neuronal BONCAT labelling enabled us to define the neuronal aggregome, a catalogue of neuronal proteins that contribute to protein aggregates in the aged brain (Fig. LC–MS analyses enabled us to identify 1,726 neuronal proteins present in aggregates in the aged brain (Fig. 6a–c and Supplementary Table 3a), 392 of which have been identified in aggregates of human brains from old individuals32 (Extended Data Fig. We selected RTN3 and SRSF3 for orthogonal validation, as these have been previously identified in aggregate omics data32,33 and been implicated to have a role in dementias34,35,36. Through immunofluorescence staining of brain tissue, we confirmed that RTN3 and SRSF3 formed ubiquitin-tagged and p62-tagged aggregate-like puncta in aged mice but not in young mice, particularly in the hippocampus (Fig. Some proteins identified are well known to aggregate in neurodegenerative diseases, including TDP43, FUS and NSF, whereas most were previously not reported to aggregate (Supplementary Table 3a). In further support of the likely relevance of these aggregating proteins in contributing to ageing and diseases, 1,195 (69%) of the aggregating neuronal proteins are encoded by risk genes as defined by the H-MAGMA study28 (Fig. Several protein features implicated in aggregation propensity were altered between aggregating and non-aggregating proteins in the sensory cortex (Extended Data Fig. GO cellular component analysis revealed that aggregating neuronal proteins could be ascribed to several neuronal compartments. However, synapse-related terms were recurrently represented (Fig. GO biological process analysis showed that several cellular functions were enriched, with protein localization recurrently represented (Extended Data Fig. a, Images of young (4-month-old) and aged (24-month-old) mouse cortex sections stained for neurons (NeuN, red) and protein aggregates (Proteostat, green). b, Quantification comparing aggregate number (left) and area (right) between mouse cortices from young and aged mice. c, Image of a human brain tissue section from an old individual stained for protein aggregates (Proteostat, green). d, Experimental approach used to determine the identity of neuronal proteins in aggregates from brains of aged (22-month-old) mice. e, Volcano plot showing the enrichment of BONCAT-labelled neuronal proteins in protein aggregates from aged mice relative to the background. f, Images of hippocampus sections from young (4-month-old) and aged (24-month-old) mice (top) and other indicated brain regions from aged mice (bottom) for RTN3 aggregates (red). g, Images of hippocampus sections from young (4-month-old) and aged (24-month-old) mice for SRSF3 aggregates (red). h, Images of hippocampus sections from aged (24-month-old) mice visualizing the colocalization of RTN3 aggregates (red, left) and SRSF3 aggregates (red, right) with the protein aggregate tag ubiquitin (green, top) or p62 (green, bottom). i, Volcano plot showing the enrichment of BONCAT-labelled neuronal protein aggregates from aged mice relative to the background. Proteins are colour-coded based on the disease or disorder for which they have been identified as risk genes according to H-MAGMA. j, Bar plot of the number of aggregating neuronal proteins in aged brains that are also H-MAGMA risk genes of the indicated brain diseases and disorders. k, Donut plots showing the percentage of all aggregating neuronal proteins in aged brains that are risk genes of the indicated diseases. l, GO cellular component analysis on all aggregating neuronal proteins in aged brains. False discovery rate (FDR) values were derived from one-sided Benjamini–Hochberg tests. m, Donut plots showing the percentage of all proteins with an age-increased half-life in the indicated brain regions that were also identified in protein aggregates in aged mice. n, Upset plot showing the overlap of aggregating neuronal proteins with age-increased half-lives among the brain regions (colours are as for m). o, Density plots comparing protein half-life values of proteins identified in aggregates from aged mice compared with proteins not identified as aggregated in the indicated brain regions (colours are as for m). P values were determined by two-tailed Wilcoxon tests. P values in b,e and i were determined by two-tailed, two-sample Student's t-tests. Risk genes used i–k were derived from the H-MAGMA study28 and considered only if the reported P value was <0.05 as determined by a two-tailed, two-sample Students t-test. Scale bars, 5 µm (h, top right three images), 10 µm (a (bottom row), f (top right), g,h (left and bottom right three images)), 20 µm (a (top row), c,f (bottom row)), or 250 µm (f, top left). Most proteins (1,352, or around 78%) identified were present in 2 other label-free datasets of aggregates in the aged brain (Extended Data Fig. Notably, no change in the total mass of insoluble proteins (Extended Data Fig. These findings indicate that BONCAT labelling does not artificially induce aggregation. We next queried whether the aggregation of proteins could explain their slower degradation that accompanies ageing. Overall, 46.8–54.6% of proteins with age-reduced degradation were also found in neuronal aggregates (Fig. In detail, 54 proteins, including 17 synaptic proteins (VCP, HSPA8 and EEF2, among others) displayed both reduced degradation with age and aggregation with age among all brain regions examined (Fig. This finding indicates that many proteins are prone to both reduced degradation and aggregation in a non-regional dependent manner. The distribution of protein half-lives of aggregating proteins in most brain regions in aged mice was slightly less than that of non-aggregating proteins in the same respective region (Fig. 4o), a result consistent with the observation that shorter-lived proteins are more prone to aggregate37. Collectively, these data suggest that protein aggregation could be a contributor to the reduced protein degradation observed with age. Microglia maintain neuronal homeostasis by detecting, engulfing and processing neuron-derived proteins38,39. We next aimed to identify neuron-derived proteins that accumulate in microglia, with the aim of inferring potential neuronal–microglia interactions that contribute to neuronal proteostasis. We labelled neuronal proteins in young and aged Camk2a-cre;PheRS* mice for 1 week, after which we freshly isolated viable CD11b+ cells (Extended Data Fig. 7a,b) from the whole brain by fluorescence-activated cell sorting (FACS) (Fig. We used an engulfment inhibitor cocktail40 throughout the cell-isolation process to prevent artificial ex vivo engulfment of neuronal proteins. We performed multiple experiments to confirm the purity of the sorted cells and the inability of protein tagging to occur in microglia (Extended Data Fig. We lysed the microglia and performed bead-based pull-down of any tagged neuronal proteins in them and analysed the digested peptides by LC–MS (Fig. Analyses of microglia from both ages revealed a range of neuron-derived proteins that were identified in microglia above the background (Fig. Through in situ antibody staining, we validated that two of these neuronal proteins, SV2A and SV2B, which are not expressed at the RNA level in microglia, were found as protein inside microglia (Fig. a, Approach used to identify BONCAT-labelled neuronal proteins in microglia (MG) from young (3–5-month-old) and aged (18–23-month-old) Camk2a-cre;PheRS* mice. n = 10 replicates in the young group, n = 8 replicates in the aged group and n = 6 replicates in the background group, with each replicate being the brains pooled from 3–4 mice, totalling 900,000 cells. b, Volcano plot showing the enrichment of BONCAT-labelled neuronal proteins in microglia of all ages combined relative to the background control. c, Images of mouse cortical microglia (IBA1, green) and neuronal protein SV2A (red) in optical sections (left), a three-dimensional rendering (top right) and a three-dimensional reconstruction of microglia and any SV2A protein that colocalized with the IBA1 signal (bottom right). d, Volcano plots as shown in b colour-coded by the presence of a signal peptide (top), annotation as mammalian exosome cargo (middle) and annotation as an ‘eat-me' signal (bottom). e, SynGO biological process analysis of neuronal proteins transferred to microglia. Proteins used in the analysis were those identified as hits in b. f, Sunburst plot showing synaptic functional representation of neuronal proteins transferred to microglia. Proteins used in the analysis were those identified as hits in b. g, SynGO cellular component analysis of neuronal proteins transferred to microglia. Proteins used in the analysis were those identified as hits in b. h, Sunburst plot showing synaptic anatomical representation of neuronal proteins transferred to microglia. Proteins used in the analysis were those identified as hits in b. DCV, dense core vesicle; SV, synaptic vesicle. i, Bar chart showing the number of neuronal proteins transferred to microglia common between microglia from young and aged mice and unique to each age. Proteins used for the analysis were those identified in each age separately compared with background controls, defined as having a log2[FC] > 1 above the background with a P < 0.05. j, Density plots of the log2[average intensity] for all neuronal proteins transferred to microglia for young (yellow) and aged (orange) mice. Proteins used for the analysis are as described in i. Statistical values were derived from a two-sided Kolmogorov–Smirnov test. k, Volcano plot showing the differential abundance of BONCAT-labelled neuron-derived proteins transferred to microglia in aged versus young mice. Proteins used for the analysis were the combination of those identified in microglia above background for young and aged mice, as described in i. Missing values were imputed as described in the Methods. l, Venn diagram showing the overlap of neuronal proteins identified to have a reduction in degradation with age (green), identified in aged aggregates (blue) and transferred to microglia (yellow). Overenrichment of overlapping proteins and associated P values were derived from hypergeometric tests. P values in b,d,i and k were derived from two-tailed, two-sample Students t-tests. We analysed whether these proteins are secreted factors, exosome cargo or targets of microglial phagocytosis. Overall, 15.6% of the proteins contained signal peptides and 68.4% are known exosome cargo or markers (Fig. 8e,f and Supplementary Table 4c,d), which suggests that secretory exchange is a potential transfer route. CALR, an ‘eat-me' signal, was also detected (Fig. 5d), which indicated possible phagocytosis of neuronal parts. Analysis of microglial LysoTag data41 revealed that 54.3% of these neuronal proteins localized to microglial lysosomes (Extended Data Fig. 8g–i), which suggests that degradation is one likely fate. Next, we sought to better understand the types of neuronal proteins acquired by microglia. Neuronal compartments, including the cell body, axons and dendrites, were represented by the proteins identified. However, synaptic, membrane and mitochondrial proteins were particularly overrepresented by GO analyses (Extended Data Fig. Focused analysis of synaptic components using SynGo42 revealed that both presynaptic and postsynaptic proteins were represented (Fig. As an independent validation strategy, we analysed an existing dataset43 of freshly isolated mouse and human microglia for synaptic proteins (Supplementary Table 4f). In this dataset, nearly 1,000 mouse and 600 human proteins were annotated as synaptic by the GO synaptic gene set (Extended Data Fig. More than half of the neuron-derived proteins identified in microglia in our dataset were also identified in those of the mouse (87.12% overlap) and human (62.71% overlap) microglia proteomes (Extended Data Fig. To address how the profile of neuronal proteins in microglia changes with age, we compared the BONCAT-labelled neuronal protein content of microglia from young and aged mice. After separately filtering each age against the background (Extended Data Fig. 8o–q), microglia from both young and aged mice commonly accumulated 119 neuronal proteins, but microglia from aged mice accumulated these proteins in higher abundances and had many unique proteins (1,212) (Fig. Differential expression analysis of the proteins in microglia between aged and young mice revealed 1,027 proteins enriched in microglia from aged mice and only 1 protein enriched in young mice (Fig. Despite the accumulation of more protein species in microglia from aged mice, the ratio of presynaptic to postsynaptic proteins did not substantially change relative to those in young mice (Extended Data Fig. To unify all our age-related observations, we questioned whether any of the neuronal proteins that accumulated in microglia in aged mice were those observed to have age-related proteostasis aberrations (reduced protein degradation or present in aggregates in the brains of aged mice). Overlap of the lists of proteins identified to have slower degradation kinetics with ageing (Fig. 2), proteins present in neuronal aggregates of aged mice (Fig. 4) and neuronal proteins enriched in microglia of aged mice (Fig. A total of 160 proteins overlapped uniquely between slower degradation kinetics with ageing and neuronal proteins enriched in microglia of aged mice, and 224 proteins overlapped uniquely between neuronal aggregates of aged mice and neuronal proteins found in microglia (Fig. Cumulatively, 550 out of the 1,027 (53.5%) neuronal proteins enriched in microglia in aged mice showed age-related proteostasis deficits. The intersection of protein lists had an overenrichment of overlapping proteins beyond which would be expected by random chance for the overlap of all three groups (1.86× overenrichment) and the overlap between proteins with age-increased half-life and neuronal proteins enriched in microglia in aged mice (1.28× overenrichment) (Fig. From these results, we posit that the accumulation of neuronal proteins with age-related alterations (slower degradation and/or aggregation) in microglia is not random but a specific mechanism to remove aberrant protein species from neurons to maintain neuronal proteostasis (Extended Data Fig. Loss of proteostasis is a hallmark of ageing, with crucial implications for neurons and therefore cognitive function and dementia risk1,2,4. Despite efforts to understand brain proteostasis with ageing21,22, to our knowledge, no studies have been performed at scale in vivo with a neuron-specific perspective. Here we generated new nascent protein labelling models that enabled cell-specific at-scale analyses of proteostasis dynamics in mice across their lifespan. Using these models, we were able to study key aspects of neuronal proteostasis with ageing, namely, protein degradation, protein aggregation and intercellular protein transfer of neuronal proteins to microglia. We first showed that neuronal protein degradation rates decline on average by approximately twofold with ageing, a deficit that emerged mostly after middle age and showed region-dependent variations in relation to the extent of turnover reduction. The reduction in neuronal protein degradation was consistent with two studies that examined age-related protein turnover changes at the level of the whole brain and synaptosome using SILAC pulse-chase methodologies21,22. However, we observed on average a doubling in protein half-life with age, whereas a study that examined whole-brain protein degradation reported a 20% average increase22, a difference that is partially explained by the neuron-specific perspective achieved in our study or the difference in timepoint sampling. By examining neuronal proteins that aggregate in aged brains, we found that approximately 50% of neuronal proteins that had age-reduced degradation also aggregated with age. This result suggests that aggregation is one likely contributor to reduced degradation, albeit not the singular cause, and it cannot be excluded as a consequence of reduced degradation. A common theme emerged among proteins displaying age-reduced degradation and a propensity to aggregate: there was an enrichment for synaptic proteins that represented a diverse range of synaptic compartments and functions. This result is intriguing considering evidence of synaptic dysfunction and loss with ageing and age-related diseases, which suggests that loss of synaptic proteostasis could be at the centre of age-related synapse dysfunction. Given that many of these proteins are encoded by neurodegenerative risk genes, it is possible that some or all risk gene variants of the proteins we identified confer a propensity for altered degradation or aggregation, as has been observed for mutant proteins including APP, TDP43, HTT and SOD1. Proteostasis is traditionally described as being intrinsically regulated in cells through chaperones, the ubiquitin–proteasome system and autophagy4. Although these mechanisms are critical, emerging evidence suggests that cell-extrinsic partners also have a role. For example, trans-synaptic transfer of protein aggregates between neurons can help to balance proteostasis across cells44. We speculate that neuronal proteostasis is partially maintained by the transfer of neuronal proteins to microglia, cells that are known to contribute on a broader level to brain homeostasis38,39,45. Enabled by our protein-labelling models, we identified hundreds of BONCAT-labelled neuronal proteins that accumulated in microglia in aged mice, and these proteins were enriched for synaptic proteins. Notably, just over half (53.5%) of these proteins also displayed age-related proteostasis aberrations, with slower degradation with age and/or found in neuronal aggregates of aged mice. The number of age-impaired proteins that accumulate in microglia of aged mice is significantly more than expected by random chance. Therefore, we propose that the transfer of these protein species from neurons to microglia is a mechanism to maintain neuronal proteostasis. Indeed, there is mounting evidence for an intercellular spread of protein aggregates from neurons to glia via their release through exosomes or transfer through tunnelling nanotubules46,47,48. As synaptic proteins are enriched among the neuronal proteins transferred to microglia and in the proteins found in brain aggregates, we propose that an additional mechanism may be the selective engulfment of proteostatically stressed synapses by microglia. This mechanism may partially explain synaptic dysfunction and loss with age26,49,50. Besides gaining a better understanding of the mechanisms of protein transfer, the effects of such transfer on neurons, microglia and the brain as a whole will be imperative to explore. The transfer of old and aggregated proteins from neurons to microglia may represent a short-term gain to neurons but a long-term loss when considering the combined detrimental effects these proteins could have on recipient microglia and the collective loss of synapses. Cumulatively, our findings revealed several age-related neuronal proteostasis aberrations that have links to synaptic dysregulation and proteinopathies, raising new hypotheses related to the causes of age-related synaptic dysfunction. Mice were housed in standard conditions on a 12-h light–dark cycle and provided with water and standard chow ad libitum. In some experiments, as documented in the main text, mice were provided with water infused with an azido-amino acid or 0.1% methionine, 0.35% cysteine (low methionine) chow (Envigo, TD.160659). All animal procedures were approved by the Administrative Panel on Laboratory Animal Care at Stanford University. No power analysis was performed to determine sample sizes. Randomization and blinding were not performed and were generally not relevant to the experiments reported herein. All transgenic mouse lines were obtained from The Jackson Laboratory. Homozygous cre lines were bred to homozygous BONCAT lines or the Ai14 reporter line to generate offspring heterozygous for the cre driver and heterozygous for the BONCAT transgene or Ai14 reporter transgene. Wild-type C57BL/6 mice used for ageing-related AAV transduction experiments were obtained from the National Institute of Aging colony. Wild-type C57BL/6 used for non-ageing-related AAV transduction experiments or used as background controls were obtained from The Jackson Laboratory. The new BONCAT models, PheRS(T413G) and TyrRS(Y43G), introduced in this paper were generated in collaboration with The Jackson Laboratory. sgRNAs (ACTGGAGTTGCAGATCACGA and GCAGATCACGAGGGAAGAGG) were designed to insert a cassette encoding a CMV-IE enhancer/chicken β-actin/rabbit β-globin hybrid promoter (CAG) followed by a floxed stop cassette containing 3×SV40 polyadenylation signals, an eGFP sequence, a viral 2A oligopeptide (P2A) self-cleaving peptide, which mediates ribosomal skipping, and either the gene encoding PheRS(T413G) or TyrRS(Y43G) into the Gt(ROSA)26Sor locus. gRNA, the cas9 mRNA and a donor plasmid were introduced into the cytoplasm of C57BL/6J-derived fertilized eggs with well-recognized pronuclei. Injected embryos were transferred to pseudopregnant females. Surviving embryos were transferred to pseudopregnant females. Resulting progeny were screened by DNA sequencing to identify correctly targeted pups, which were then bred to C57BL/6J mice for germline transmission. This colony was backcrossed to C57BL/6J mice for at least three generations. Sperm was cryopreserved at The Jackson Laboratory. To establish our live colony, an aliquot of frozen sperm was used to fertilize C57BL/6J oocytes. All mice were maintained as individual biological replicates except for the neuron-to-microglia protein-transfer experiments. In the neuron-to-microglia protein-transfer experiment, enriched labelled neuronal protein from microglia was expected to be relatively minimal. Therefore, to ensure detection by LC–MS, the brains of 3 mice, for a total of 900,000 microglia, were pooled to generate a single biological replicate. Background controls are necessary for all BONCAT studies to account for proteins that are nonspecifically enriched by the DBCO pull-down method or nonspecifically labelled with fluorescent alkynes. Background control mice, wild-type mice genetically incapable of incorporating the NCAA, were treated identically with the NCAA relative to their BONCAT counterparts. In all of our experiments, background controls represent the same biological material (brain region, mouse age, protein fraction and/or cell type) as the labelled sample under analysis. In the protein degradation study, background samples for each labelled age and region were derived from the same respective age and region of background control mice. In the protein aggregation study, background samples for each labelled aged aggregate sample were derived from aggregates isolated from aged background control mice. Mice were anaesthetized with isoflurane and 3 × 1011–5 × 1011 AAV genome copies were injected in 100 μl sterile 1× PBS via the retroorbital sinus. Equal genome copies were injected in mice between which comparisons would be made. For maximal transgene expression, mice were used no sooner than 3 weeks following initial transduction. All AzAAs, including 4-azido-l-phenylalanine (Vector Laboratories; 1406-5G), N-epsilon-azido-l-lysine hydrochloride (Iris Biotech, HAA1625.0005) and 3-azido-l-tyrosine (Watanabe Chemical Industries, J00560) were prepared as a 12.35 mg ml–1 solution for intraperitoneal injections. To hasten the dissolution of AzF, it was first dissolved in 1 M NaOH at 111 mg ml–1, after which it was brought to 12.35 mg ml–1 with sterile 1× PBS. Immediately before intraperitoneal injection at 185 mg kg–1, aliquots were brought to a neutral pH by the addition of 1 M HCl. Unless otherwise noted, NCAAs were injected once daily for 7 consecutive days for both BONCAT lines and wild-type background control mice. 1 were derived from experiments in which the NCAAs were administered by intraperitoneal injections as described above. For all experiments, including the protein degradation experiments, mice were killed approximately 16 h after the final amino acid administration to allow for sufficient incorporation into nascent proteins. Note that labelling occurs during protein synthesis, which is occurring at different periods and rates for different proteins, not all at one instance after amino acid administration. Thus a sufficient amount of time between amino acid administration and sample collection is necessary to achieve sufficient labelling. This could affect half-life determinations, but this is an unavoidable challenge of any protein degradation experiment. For the experiments in which 4-azido-l-phenylalanine-infused water was given to mice, 4-azido-l-phenylalanine was dissolved in sterile water at 1 mg ml–1. The only experiment to use an AzF water treatment protocol was the neuron-to-microglia protein-transfer experiments. In such experiments, mice also received AzF via intraperitoneal injections as described above. This AzF treatment protocol aimed to enhance neuronal labelling and to increase the likelihood of detecting neuronal proteins in microglia, which we reasoned would be a rare event. After perfusion, brains were immediately extracted and fixed in 4% paraformaldehyde, snap-frozen in tubes on dry ice or immediately enzymatically dissociated. For experiments requiring brain region dissection, after brain extraction, regions were immediately dissected on ice using a ‘rodent brain matrix' 1-mm coronal slicer (Tedpella, 15067) according to coordinates obtained from the Mouse Brain Library (http://www.mbl.org/) using the C57BL/6J atlas as a reference. After dissection, regions were snap-frozen in tubes. In cases in which tissue was fixed, tissue was fixed for 24 h and then prepared for either paraffin embedding or sucrose cryoprotection. To isolate microglia, whole brains were first enzymatically dissociated with the addition of an engulfment inhibitor cocktail to prevent ex vivo engulfment of neuronal debris40. At all steps during microglia isolation, staining and sorting, liquids were supplemented with the engulfment inhibitor cocktail containing the final concentrations of the following reagents: 25 µM Pitstop2 (Abcam, ab120687), 2 µM cytochalasin D (Tocris, 1233), 2 µM wortmannin (Tocris, 1232), 40 µM Dynasore (Tocris, 2897) and 40 µM bafilomycin A1 (Tocris, 1334), with each reagent being prepared as a 1,000× stock. Immediately after extraction of perfused brains, they were placed in 800 µl 1× D-PBS+/+ (Thermo Fisher Scientific, 14040117) on ice. Next, brains were minced on ice using fine scissors for approximately 2 min until brain pieces were small enough to triturate with a p1000 pipette with little resistance during pipetting. Brains were triturated until there was no resistance during pipetting. The supernatant was removed by pipetting, and an enzymatic cocktail prepared from a Multi-tissue Dissociation kit 1 (Miltenyi Biotec, 130-110-201), consisting of 100 µl enzyme D, 50 µl enzyme R, 12.5 µl enzyme A and 2.4 ml D-PBS+/+, was added. The pellet was resuspended by pipetting, after which the suspensions were transferred to a tube rotator at 37 °C for a 20 min incubation. Halfway through and at the end of this incubation, brain suspensions were triturated with a p1000 approximately 20 times to help to break up brain pieces until the suspension was largely devoid of any visible clumps. After the incubation step, 10 ml ice-cold DPBS+/+ was added to each brain suspension, after which the entire suspension was run through a 70 µm cell strainer into a new tube. Next, myelin was removed from the preparations using Debris Removal solution (Miltenyi Biotec, 130-109-398). Each brain pellet was resuspended up to 3.1 ml with cold D-PBS+/+ and 0.9 ml Debris Removal solution was added to each pellet and mixed by gentle inversion of the tube. Next, 4 ml cold D-PBS+/+ was overlaid on top of the brain–Debris Removal solution mixture. Following centrifugation, the myelin interface and liquid above it were removed by pipetting, and 11 ml cold DPBS+/+ was mixed with the remaining cell suspension. The cell suspension was centrifuged at 1,000g for 13 min at 4 C with 0 break, and the resulting supernatant removed by pipetting. The largely myelin-depleted cell pellets were resuspended in 80 µl AstroMACS separation buffer (Miltenyi Biotec, 130-117-336) containing 10 µl FcR blocking reagent, mouse (Miltenyi Biotec, 130-092-575) and incubated on ice for 10 min. Next, 10 µl anti-ACSA-2 MicroBeads (Miltenyi, 130-097-679) was mixed into the cell suspension and incubated for 15 min on ice. After incubation, cells were placed in 1 ml AstroMACS separation buffer and centrifuged at 300g for 10 min at 4 °C. The supernatant was removed and pellets were resuspended in 500 µl AstroMACS separation buffer and loaded onto a pre-washed LS column (Miltenyi Biotec, 130-042-401). The LS columns were washed 3 times, each time with 3 ml AstroMACS separation buffer. The flow through was retained, as this contained microglia, whereas the cells retained in the column were eluted with 5 ml of AstroMACS separation buffer and retained as an astrocyte fraction used for other experiments. The supernatant was removed and cells were stained 1:10 with APC/cyanine7 anti-mouse/human CD11b antibody (BioLegend, 101225) and calcein-AM (BioLegend, 425201) at a final concentration of 1× in cell staining buffer (BioLegend, 420201) for 30 min on ice. Supernatant was removed and cells were resuspended in an appropriate volume of cell staining buffer for FACS. CD11b+ brain macrophages were sorted by gating on CD11b+, calcein-AM+ singlets on a Sony MA900 cell sorter (Sony Biotechnology). Three biological replicates-worth totalling 900,000 of brain macrophages were pooled into a single replicate and frozen at −80 °C before lysing the cells and enriching for BONCAT-labelled proteins, which was performed on microglia from both BONCAT-labelled samples and wild-type background controls. Microglia from wild-type and Camk2a-cre;Ai14 hosts were isolated as described above and underwent flow cytometry analysis to determine microglia purity or the expression of tdTomato. To analyse tdTomato expression, microglia were stained with CD11b-BV785 (BioLegend, 101243). For AAV vectors (used in vivo), the sequences for PheRS(T413G) and the Camk2a promoter were ordered as gBlocks HiFi Gene Fragments from Integrated DNA Technologies (IDT). For AAV vectors, the PheRS(T413G) gene fragment was first cloned into a pAAV-CAG-GFP (Addgene, 37825) backbone through the removal of GFP by BamHI/EcoRV double-restriction digest followed by Gibson assembly (NEB, E2611S). From this Gibson assembly product, we then removed the CAG promoter by XbaI/NdeI double-restriction digest and cloned in the Camk2a promoter by Gibson assembly. Propagation of AAV plasmids was performed in NEB Stable Competent Escherichia coli (NEB, C3040H) at 30 °C to avoid mutations in the AAV ITR sequences. For lentivirus vectors (used in vitro), the sequence for PheRS(T413G) was ordered as a gBlock hiFi Gene Fragment from IDT. The sequence was modified to add a 3× Flag tag and p2a sequence to the C terminus of PheRS(T413G) and flanks to enable Gibson Assembly cloning. All preparations were ultrapurified and used the PHP.eB serotype. Cell culture models were used to verify the tagging of proteins during protein synthesis. Py8119 cells were transduced with LV-CMV-PheRS*-p2a-mCherry to stably express PheRS*. B16-F10 cells were stably transfected with a Piggybac vector expressing TyrRS(Y43G) as previously reported3. One hour later, NCAA was added to both the non-treated and cycloheximide-treated cell cultures to a final concentration of 1 mM. The cultures were incubated for 24 h before lysates were collected for analysis of BONCAT-tagged proteins by in-gel fluorescence. Original cell lines were obtained from American Type Culture Collection. Cell lines were not independently authenticated by the authors and they were not tested for mycoplasma. Brain tissue was first homogenized by sonication in a strong lysis buffer (8 M urea, 1% SDS, 100 mM choloracetamide (CAA), 20 mM iodoacetamide (IAA), 1 M NaCl and 1× protease inhibitor in 1× PBS). Sonication was performed for at least 3 cycles of 10 s of sonication with at least 5-s breaks between sonication cycles at an amplitude of 90% using a probe sonicator. The resultant supernatant was retained, and aliquots were immediately measured by BCA to obtain the protein concentration. The remaining supernatant was frozen at −80 °C until it was further processed to enrich for BONCAT-labelled proteins. A click reaction was performed on the lysates to ‘click' a fluorophore onto azide side chains of the labelled proteins. The following chemical cocktail was added to the normalized lysates for 1 h with constant shaking to perform the click reaction: 0.83 µl Alexa Fluor 647 alkyne, triethylammonium salt (Thermo Fisher Scientific, A10278) at 5 mM, 1.04 µl copper(II) sulfate (Millipore Sigma, 451657-10G) at 6.68 mM, 2.087 µl THPTA (Click Chemistry Tools, 1010-500) at 33.3 mM, 4.17 µl aminoguanidine hydrochloride (Millipore Sigma, 396494-25G) at 100 mM, 8.33 µl sodium l-ascorbate (Fisher Scientific, A0539500G) at 100 mM and 33.5 µl PBS. Importantly, 20 mM CuSO4 and 50 mM THPTA were mixed at a 1:2 ratio for 15 min before combining the rest of the click reaction. After 1 h of incubation for the click reaction, the reactions were filtered through Zeb Spin Desalting columns, 7 K MWCO, 0.5 ml format (Thermo Fisher Scientific, 89882) following the manufacturer's protocol to remove unbound fluorophore. Samples were heated at 95 °C for 10 min to denature the proteins. Clicked and denatured lysates were loaded onto a NuPAGE 12%, Bis-Tris gel (Thermo Fisher Scientific, NP0341BOX) and run at 200 V for 45 min. The gel was imaged to detect the Alexa 647-clicked proteins using a LI-COR Odyssey XF imaging system. To detect total loaded protein, gels were stained with GelCode Blue Stain reagent (Thermo Fisher Scientific, 24590), destained in water for at least 1 h and then again imaged using a LI-COR Odyssey XF imaging system. Important in-gel fluorescence results are displayed as uncropped images, including molecular weight ladder and controls, in Supplementary Fig. Tissue sections were prepared for click staining of azide-modified proteins as described in the immunofluorescence staining of tissue sections until the blocking step. After blocking, tissue sections were stained for 1 h in the following click reaction cocktail: 2 µl Alexa Fluor 647/594/555/488 alkyne, triethylammonium salt (Thermo Fisher Scientific, A10278) at 5 mM, 5 µl copper(II) sulfate (Millipore Sigma, 451657-10 G) at 20 mM, 10 µl THPTA (Click Chemistry Tools, 1010-500) at 50 mM, 100 µl aminoguanidine hydrochloride (Millipore Sigma, 396494-25G) at 50 mM, 100 µl sodium l-ascorbate (Fisher Scientific, A0539500G) at 50 mM and 783 µl PBS. Importantly, 20 mM CuSO4 and 50 mM THPTA were mixed at a 1:2 ratio for 15 min before combining the rest of the click reaction. After staining, tissue sections were washed 3 times in Tris-buffered saline–Tween-20 (TBS-T) and either stained further with antibodies as described below in the immunofluorescence staining of tissue sections or mounted and coverslipped. For sucrose-cryopreserved tissues, 40-µm-thick sections were sectioned on a Lecia sliding microtome equipped with a cooling unit and cooling stage. After cooling to room temperature, tissues were blocked and permeabilized in 5% normal donkey serum (Jackson Immuno Research, 017-000-121) and 0.3% Triton X-100 (Millipore Sigma, 93443-100ML) in 1× PBS for 1 h. After blocking, tissues were stained with primary antibody diluted in 1% (w/v) bovine serum albumin (Fisher Scientific, BP9703100) and 0.3% Triton X-100 in 1× PBS overnight at 4 °C with gentle rocking agitation. After primary antibody staining, tissue sections were washed 3 times in TBS-T. After washing, tissue sections were stained with secondary antibodies diluted 1:500 in 1% (w/v) bovine serum albumin and 0.3% Triton X-100 in 1× PBS for 3 h at room temperature with gentle rocking agitation. All secondary antibodies recognized the IgG domain of the primary antibodies, were conjugated to Alexa fluorophores and purchased from Jackson Immuno Research. After secondary antibody staining, tissues were washed as described above, briefly stained with 4′,6-diamidino-2-phenylindole, dihydrochloride (Thermo Fisher Scientific, D1306) and mounted and coverslipped on Superfrost Plus microscope slides (Fisher Scientific, 12-550-15) with Fluoromount-G Slide mounting medium (Fisher Scientific, 50-259-73). Staining of formalin-fixed paraffin-embedded tissue sections was similar to the methodology used for free-floating sucrose-cryopreserved tissue sections, except that tissue sections on slides were deparaffinized and rehydrated by incubation through a gradient of xylene, 100% ethanol, 90% ethanol, 80% ethanol, 70% ethanol and water, and heat-induced antigen retrieval was always performed by heating slides in 1× SignalStain Citrate Unmasking solution for 10 min in a microwave. Proteostat aggresome staining was only performed on formalin-fixed paraffin-embedded tissue sections following deparaffinization, rehydration, antigen retrieval and blocking of tissue sections. Proteostat (Fisher Scientific, NC0098538) was diluted 1:1,000 in 1× PBS and incubated on tissue sections for 5 min at room temperature and subsequently washed several times with TBS-T. After washing, slides were either coverslipped or subjected to antibody staining. When capturing images to be compared, all imaging parameters and post-acquisition processing parameters were kept identical between images to be compared. The Zeiss Axioimager was accessed in the Stanford University Cell Sciences Imaging Facility (CSIF, RRID: SCR_017787). Heatmaps of raw fluorescence images were created using ImageJ. Images were converted to 8-bit and a colour look-up table was applied. Quantification of protein aggregate number and area was performed using Fiji. Brightness and contrast were adjusted equally among all images. Images were converted to 8-bit and binary, after which masked particles, which represented aggregates, were analysed and summary statistics related to aggregate number and average area were recorded. Brain tissue was first homogenized by sonication in a strong lysis buffer (8 M urea, 1% SDS, 100 mM CAA, 20 mM IAA, 1 M NaCl and 1× protease inhibitor in 1× PBS). Sonication was performed for at least 3 cycles of 10 s of sonication with at least 5-s breaks between sonication cycles at an amplitude of 90% using a probe sonicator. The resultant supernatant was retained, and aliquots were immediately measured by BCA to obtain protein concentration. Denatured and reduced samples were run on a NuPAGE 12%, Bis-Tris gel for approximately 2 h at 110 V. Proteins were transferred to 0.45 µM methanol-activated PVDF membrane by standard wet-transfer methodology at 400 mA for approximately 90 min at 4 °C. Membranes were blocked in Intercept TBS blocking buffer (Fisher Scientific, NC1660550) for 1 h with gentle shaking. After blocking, membranes were incubated in primary antibody diluted 1:1,000 in 5% bovine serum albumin overnight at 4 °C with gentle shaking. Primary antibodies used were rabbit anti-β-actin (Cell Signaling Technology, 4970S) and rabbit anti-HSP90 (Cell Signaling Technology, 4877T). Following washes, membranes were stained with IRDye 800CW goat anti-rabbit IgG secondary antibody (Li-Core, 926-32211) diluted 1:5,000 in 5% bovine serum albumin by light-protected incubation with gentle shaking for 1 h. Following secondary antibody staining, membranes were washed three times with TBS-T with gentle shaking. Last, membranes were imaged using a LI-COR Odyssey XF imaging system. An image of the uncropped membrane image, including molecular weight ladder, is presented in Supplementary Fig. The protocol for insoluble and aggregated protein isolation was slightly modified from a previous publication31. Each hemisphere was pulverized to powder by ten hammer strokes to the sample on a liquid-nitrogen-cooled pestle. The resultant homogenate powder was rapidly transferred to tubes on dry ice, pooling three pulverized brains to generate one biological replicate. This methodology of cell lysis is used to preserve aggregates, which could be compromised using other lytic techniques (for example, sonication or detergent-based solutions). The homogenate powder was quickly weighed to avoid thawing. Next, for every 0.8 ml homogenate solution, 100 μl 5 M NaCl and 100 μl 10% Sarkosyl was added to the homogenate solution on wet ice. The homogenate was gently sonicated for 3 separate intervals for 5 s at an amplitude of 30% using a probe sonicator (Qsonica, Q125-110) at 4 °C. The homogenate solution was titrated with a p1000 pipette and filtered through a 70 μm cell strainer and centrifuged at 500g for 5 min at 4 °C to obtain a smooth, clump-free homogenate. Equal protein amounts of bulk protein homogenate were aliquoted from each sample and diluted to 10 mg ml–1 in 1% Sarkosyl buffer (1% Sarkosyl, 0.5 M NaCl and 1× protease inhibitor in low-salt buffer). Samples were ultracentrifuged at 180,000g for 30 min at 4 °C, during which soluble and non-aggregated proteins remained in the supernatant and insoluble and aggregated proteins were pelleted. The supernatant was gently removed and frozen, whereas pellets were washed in 1% Sarkosyl and ultracentrifuged again at 180,000g for 30 min at 4 °C. Pellets were retained and frozen at −80 °C until further processing for various analyses, including enriching for BONCAT-labelled proteins, which was performed on aggregate pellets from both BONCAT-labelled mice and wild-type background control mice. It is important to note that downstream enrichment of BONCAT-labelled proteins from total aggregates should only result in obtaining neuronal proteins that were part of aggregates, but not non-neuronal co-aggregating proteins. The reason for this is due to the buffer required for pull-down (see section below on enrichment of BONCAT-labelled proteins), which both solubilizes and denatures proteins. Introduction of total aggregates to this buffer should result in the release of non-neuronal co-aggregating proteins from neuronal aggregates. The pull-down then selectively enriches for BONCAT-labelled proteins, excluding non-BONCAT-labelled (non-neuronal coaggregating proteins) from further analysis. A total of 400,000 sorted Cd11b+ microglia or 1 hemisphere of a whole brain was used as input for RNA extraction. Hemibrains were mechanically lysed using a Qiagen TissueLyser II (Qiagen, 85300) for 2 cycles, each cycle being 2.5 min at a frequency of 30. Following lysis, RNA was extracted per the instructions from a miRNeasy Micro kit (Qiagen, 217084). Eluted RNA quality was evaluated on an Agilent Bioanalyzer and frozen until used for RNA sequencing library preparation. mRNA was purified from total RNA using poly-T oligonucleotide-attached magnetic beads. After fragmentation, the first-strand cDNA was synthesized using random hexamer primers followed by second-strand cDNA synthesis. The library was ready after end repair, A-tailing, adapter ligation, size selection, amplification and purification. The library was checked with Qubit and real-time PCR for quantification and a bioanalyzer for size distribution detection. After library quality control, different libraries were pooled based on the effective concentration and targeted data amount, then subjected to Illumina sequencing (NovaSeq X Plus Series–PE150). HISAT2 (v.2.2.1) was used to build the index of the reference genome, and HISAT2 was used to align paired-end clean reads to the reference genome. FPKM, the expected number of fragments per kilobase of transcript sequence per millions base pairs sequenced, considers the effect of sequencing depth and gene length for the read counts at the same time and is currently the most commonly used method for estimating gene expression levels. DESeq2 was used to analyse the RNA sequencing data, with the raw gene counts used as input. Brain tissue was first homogenized by sonication in a strong lysis buffer (8 M urea, 1% SDS, 100 mM CAA, 20 mM IAA, 1 M NaCl and 1× protease inhibitor in 1× PBS). Sonication was performed for at least 3 cycles of 10 s of sonication with at least 5-s breaks between sonication cycles at an amplitude of 90% using a probe sonicator. The resultant supernatant was retained, and aliquots were immediately measured by BCA to obtain protein concentration. The the remaining supernatant was frozen at −80 °C until it was further processed to enrich for BONCAT-labelled proteins. Samples to be compared were normalized to equal protein amounts (1–2 mg total) and equal volumes in lysis buffer. Lipids were removed by the addition of 10 µl Cleanascite (Fisher Scientific, NC0542680) per 40 µl homogenate and incubation with constant agitation on a thermomixer set to 1,500–2,000 rpm for 10 min. Samples were then centrifuged at >16,000g for 3 min to pellet lipids. The resultant supernatant was retained and 7.5 units of benzonase (Millipore Sigma, 70664-3) was added per 40 µl sample to digest nucleotides over 30 min with constant agitation on a thermomixer set to 1,500–2,000 rpm. Following benzonase treatment, samples were diluted to 1 ml total with lysis buffer and added to 200 µl dry control agarose beads (Thermo Fisher Scientific, 26150) pre-washed 3 times before sample addition (washed one time with water and two times with 0.8% SDS). Samples were pre-cleared with the control agarose beads to remove nonspecific bead binders by 1 h of light-protected end-over-end rotation. After pre-clearing, samples were centrifuged at 1,000g for 5 min to pellet the plain agarose beads. The resultant supernatant was added to 20 µl dry DBCO beads (Vector, 1034-25) pre-washed 4 times before sample addition (washed once with water and 3 times with 0.8% SDS). Quenching was performed for 30 min with light-protected end-over-end rotation. After quenching of the DBCO beads, samples were centrifuged for 5 min at 1,000g and the supernatant was discarded and the DBCO beads were retained. DBCO beads were washed by the addition of 1 ml water and again centrifuged for 5 min at 1,000g. The supernatant was discarded and 0.5 ml 1 mM dithiothreitol (DTT; Thermo Fisher Scientific, R0861) was added to each sample. After incubation, samples were centrifuged for 5 min at 1,000g and the resultant supernatant discarded. The DBCO agarose beads were resuspended in 0.5 ml 40 mM IAA (Millipore Sigma, I1149-25G) and incubated light-protected for 30 min to alkylate proteins. After incubation, samples were centrifuged for 5 min at 1000g and the resultant supernatant discarded and the DBCO agarose beads were resuspended in 500 µl 0.8% SDS. The DBCO agarose beads were subjected to extensive washing to further remove nonspecifically bound proteins. This was accomplished by washing each sample with 50 ml of 0.8% SDS, 8 M urea and 20% acetonitrile (Fisher Scientific, PI51101). The speed of washes was enhanced by performing them in Poly-Prep chromatography columns (Bio-Rad, 7311550) connected to a vacuum manifold (Fisher Scientific, NC0994627); approximately 7 ml of a wash was added to the column to resuspend the DBCO agarose beads, and then the vacuum was applied to draw through the wash buffer, leaving DBCO agarose beads in the column. Following all washes, DBCO agarose beads were resuspended in 700 µl of 50 mM HEPES (pH 8.0) and immediately transferred to a 1.5 ml tube. DBCO beads were centrifuged for 5 min at 1,000g. After centrifugation, the supernatant was completely removed and 200 µl 50 mM HEPES (pH 8.0) (Fisher Scientific, AAJ63002-AE) was added to the DBCO agarose beads. Next, 10 µl of a 0.1 µg µl−1 trypsin–Lys-C mix (Promega, V5073) was added to each sample. Proteins bound to the DBCO agarose beads were on-bead digested overnight at 37 °C on a thermomixer set to 1,500–2,000 rpm. The next morning, approximately 16 h after initiating on-bead digestion, samples were centrifuged for 10 min at 1000g. Supernatant containing digested peptides was transferred to a new tube and frozen at −80 °C until further processing. Peptide amounts were quantified using a Pierce Quantitative Peptide Assays & Standards kit (Thermo Fisher Scientific, 23290). Peptides destined for single-shot LC–MS experiments were desalted using Nest Group Inc BioPureSPN Mini, PROTO 300 C18 columns (Fisher Scientific, NC1678001). The desalting process involved conditioning the column with 200 µl methanol for 5 min followed by centrifugation at 25g until dry, washing the column twice with 200 µl 50% acetonitrile, 5% formic acid (Thermo Fisher Scientific, 28905) by centrifugation at 25g until dry, washing the column 4 times with 5% formic acid by centrifugation at 25g until dry, passing peptides through the column in 40 µl increments by centrifugation at 25g until dry, washing the column 4 times with 200 µl 5% formic acid by centrifugation at 25g until dry, and finally eluting the peptides 2 times with 100 µl 80% acetonitrile, 0.1% formic acid by centrifugation at 25g. Following desalting, peptides were dried in a speed vac and then maintained at −80 °C before being run by LC–MS. Peptides destined for TMT and pooling were dried in a speed vac and subsequently resuspended in 25 µl 100 mM TEAB (pH 8.5) (Millipore-Sigma, T7408-100ML). TMTpro 18-plex reagents (Thermo Fisher Scientific, A52047) were reconstituted to 4 µg µl–1 in anhydrous acetonitrile (Millipore-Sigma, 271004-1L). Labelling reactions were quenched by adding 2 µl 50% hydroxylamine (Thermo Fisher Scientific, B22202.AE) for 15 min with occasional vortexing. Equal volumes of TMT-labelled peptides were pooled and dried in a speed vac, after which peptides were desalted as described above for single-shot LC–MS preparations. In-solution digest was used for experiments examining bulk brain proteome and aggregates from wild-type aged mice. The upper phase was discarded and 650 µl methanol was added to the sample, vortexed and centrifuged for 17,000g for 5 min. The dried protein pellet was resuspended in 10 µl of 8 M urea and 0.1 M Tris-HCL (pH 8.5). Next, 2.25 µl 10 mM DTT was added to the sample and vortexed, followed by incubation on a thermomixer at 30 °C with shaking at 650 rpm for 90 min. Next, 2.83 µl 50 mM IAA was added to the sample and vortexed, followed by light-protected incubation for 40 min. After incubation, 90 µl 50 mM Tris (pH 8) was added to dilute the urea concentration. Last, trypsin–Lys-C mix was added to a mass to mass ratio of 1:50 and the samples were digested overnight at 30 °C with shaking at 650 rpm on a thermomixer. Following digestion, peptides were desalted as described for preparation of BONCAT-labelled proteins. Bruker timsTOF Pro was generally used for small-scale comparisons (eight or fewer samples to be directly compared) and/or when the peptide amount was limited and high sensitivity was still desired. timsTOF was used to acquire the following data in this paper: brain region comparison data in Fig. 1; CMV-cre;BONCAT data from various tissues in the Supplementary Information. Samples were analysed using a TimsTOF Pro mass spectrometer (Bruker Daltonics) coupled to a NanoElute system (Bruker Daltonics) with solvent A (0.1% formic acid in water) and solvent B (0.1% formic acid in acetonitrile). Dried peptides were reconstituted with solvent A and injected onto the analytical column, Aurora Ultimate CSI 25 × 75 C18 UHPLC column, using a NanoElute system at 50 °C. Eluted peptides were measured in DDA-PASEF mode using timsControl 3.0. The MS1 and MS2 spectra were captured from 100 to 1,700 m/z in data-dependent parallel accumulation-serial fragmentation (PASEF) mode with 4 PASEF MS/MS frames in 1 complete frame. Data captured were processed using Peaks Studio (v.10.6 built on 21 December 2020, Bioinformatics Solution) for sequence database search with the Swiss-Prot Mouse database. Carbamidomethylation of cysteine was set as a fixed modification. Protein N-terminal acetylation and methionine oxidation were set as variable modifications, with a maximum of three variable post-translational modifications allowed per peptide. Estimate FDR with decoy fusion was activated. timsTOF Ultra was used to acquire the following data in this paper: aged neuronal aggregates and neuron-to-microglia protein transfer. Eluted peptides were measured in diaPASEF mode with a base method m/z range of 100–1,700 and 1/K0 range of 0.64–1.45 V s−1 cm−2. The PASET m/z window range was 400 to 1,000 and the mobility range was 0.64–1.37 V·s·cm−2 with a 96 ms cycle time at 100% duty cycle. Data captured were processed using Spectronaut (v.19.3 build on 23 October 2024, Biognosys) for directDIA search with Swiss-Prot Mouse database downloaded on 3 March 2023. The default setting was used, but with a slight modification of minimum peptide length of six and cross-run normalization deactivated. A Thermo Eclipse was used for any experiments using TMT, as this instrument is capable of running TMT samples. TMT, and by extension the Thermo Eclipse, were used for larger-scale comparisons to avoid any time-associated ‘drifts' that would make comparisons between samples run on an instrument far apart in time less accurate. TMT was also used when quantitative precision and consistent peptide identification across replicates or samples to be compared were critical, such as for the protein degradation experiments in which the quantification of all time points of a single region and single age was instrumental to the overall success of the experiment. TMT labelling and Thermo Eclipse were used for the following experiments: transgenic line comparisons in Fig. 2 and 3; AAV and transgenic mouse comparisons in the Supplementary Data; aged versus young in Supplementary Data. Samples were analysed using an Easy-nLC 1200 coupled to a Thermo Scientific Orbitrap Eclipse Tribrid mass spectrometer with EasySpray Ion source and FAIMS Pro interface. Digested samples were reconstituted in 0.1% formic acid in water and were loaded to a trap column (Thermo Scientific Acclaim PepMap C18 column, 2 cm × 75 µm i.d., 3 µm) and separated on a Thermo Scientific Acclaim PepMap RSLC C18 column, 25 cm × 75 µm i.d., 2 µm. Solvent A was 0.1% formic acid in water and solvent B was 80% acetonitrile in water with 0.1% formic acid. The gradient was ramped from 2% B to 40% B in 179 min at a flow rate of 300 nl min–1. TMT-labelled peptides were analysed by data-dependent acquisition mode using the synchronous precursor selection (SPS) MS3 real time search (RTS) approach. For full MS scan, the resolution was set at 60,000 and the mass range was set to 350–1,500 m/z. The most abundant multiply charged (charge 2–7) parent ions were selected for CID MS2 in the ion trap. The CID collision energy was set at 35%. RTS using the UniProt Mus musculus database was performed. Carbamidomethyl on cysteine (C) and TMTpro 16plex on lysine (K) and peptide terminal were set as static modifications. Oxidation on methionine (M) was set as variable modifications. Up to ten parent ions from MS2 will be selected by SPS for HCD MS3. MS3 spectra were acquired at 30,000 resolution (at m/z 200) in an Orbitrap MS with 55% normalized HCD collision energy. TMT data were processed using Thermo Scientific Proteome Discoverer software (v.2.4). Spectra were searched against the UniProt Mus musculus database using the SEQUEST HT search engine. A maximum of two missed cleavage sites was set for protein identification. Variable modifications included oxidation (M) and acetylation (protein N terminus). Resulting peptide hits were filtered for maximum 1% FDR using the Percolator algorithm. Precursor mass tolerance was set as 10 ppm and fragment mass tolerance for CID MS 2 Spectra obtained by Ion Trap was set as 0.6 Da. The peak integration tolerance of reporter ions generated from SPS MS3 was set to 20 ppm. For the MS2 methods, reporter ion quantification was performed on FTMS MS2 spectra and for identification, where they were searched with precursor mass tolerance of 10 ppm and fragment mass tolerance of 0.02 Da. For the reporter ion quantification in all methods, no normalization and scaling were applied. The average reporter singal-to-noise ratio threshold was set to ten. Correction for the isotopic impurity of reporter Quan values was applied. For experiments in which the goal was to simply quantify the number of different proteins labelled in BONCAT-labelled samples and/or identify proteins labelled with confidence over the background, non-normalized data were used as input. The reason for using non-normalized data is because the background samples were expected to have few proteins and low abundance relative to labelled samples, and this inherent difference should be preserved for analyses of labelled samples relative to background samples; normalizing would remove this inherent difference. The following steps were performed on the data: data were log2 transformed, proteins were filtered based on possessing valid values in a certain number of replicates in at least one group, missing values were replaced by imputation (width of 0.3 and downshift of 1.8), fold change was calculated for each protein between BONCAT-labelled replicates versus the respective background control replicates, and P values for these comparisons were derived from a two-tailed t-test. The number of replicates that were required to possess a valid value for a protein depended on the number of replicates used in the experiment, but in all cases required over 50% of the replicates in at least one group to possess a valid value for any given protein. The Camk2a-cre;BONCAT benchmarking experiment required 3 out of 4 replicates to have valid values in at least one group. The AAV Camk2a-cre;PheRS* experiment required 3 out of 4 replicates to have valid values in at least one group. The CMV-cre;BONCAT experiments required 2 out of 2 replicates to have valid values in at least one group. The neuronal aggregate experiment required 6 out of 11 labelled replicates to have valid values in at least one group. The neuronal protein transfer experiment required at least 6 labelled replicates to have valid values when comparing all ages to the background, at least 6 out of 10 labelled replicates to have valid values when compared with the background in the context of labelled proteins in young mice, and at least 5 out of 8 labelled replicates to have valid values when compared with the background in the context of labelled proteins in aged mice. P values less than 0.05 were considered significant in all analyses, following precedent set by other protein enrichment MS publications10,13. The fold change cut-off between two groups being compared was log2[FC] > 1. For PCA, individual data frames from each group being compared underwent filtering as described above in the basic fold change analysis with any proteins remaining after filtering in each group being retained as true hits. Subsequently, data frames for each group containing the raw abundance values were merged with all proteins being retained regardless of whether they were shared or not among the three groups. The raw abundance values were log2 transformed, and missing values, which were mostly proteins that were identified in one BONCAT line or region but not others, were replaced by imputation (width of 0.3 and downshift of 1.8). These values were then used for PCA in Perseus software. In a few experiments, labelled protein fold changes between two or more experimental conditions that were also labelled were compared. The experiments include regional comparisons in Fig. 1, aged versus young comparison in Extended Data Fig. 2, and aged versus young comparison of microglia derived from labelled mice in Fig. In these experiments, data frames from each group being compared underwent filtering as described above in the basic fold change analysis with any proteins remaining after filtering in each group being retained as true hits for further analysis. Data frames of true hits for each group were merged to keep all proteins identified among all groups. The raw abundance values for the retained proteins from all groups were log2 transformed, and missing values, which were mostly proteins that were identified in one condition but not the other, were replaced by imputation (width of 0.3 and downshift of 1.8). The fold change for these values were calculated for each protein between conditions, and P values for these comparisons were derived from a two-tailed t-test. 1, the resultant values were z scored before visualizing in a heatmap. As described above, non-normalized data were used as input for protein turnover analyses with a similar rationale to preserve inherent differences in protein abundance between time points, with less protein being expected at each successive time point progressing into the chase period; normalization would risk losing these inherent differences. First, the log2[FC] in protein abundance between time point 1 replicates and wild-type background control replicates (n = 2 per age per region combination) was calculated. Any proteins enriched >1.5 in time point 1 over the background control were considered for further analysis, and the other proteins were discarded as background proteins. This fold change over background filtering was performed for each age and region combination separately. Time point 1 was used in this filtering approach rather than other time points because time point 1 represents the time point of maximal labelling and would give the fairest evaluation of background compared with later time points at which enriched proteins successively approach levels closer to that of wild-type controls and would lead to discarding more proteins, probably unjustly. Background was further mediated by subtracting the average of the background samples from the average of each individual time point for each protein, similar to how background is subtracted for colorimetric assays. When making comparisons between different ages of a single region, proteins were further filtered to those that were commonly detected between ages and detected in all replicates, with the only exception being the degradation kinetic trajectories shown in Fig. With this relatively stringent filtering approach, we ensured that equally reliable half-life predictions and trajectory analysis were performed without a need to assign uncertainty values due to varying drop-out rates. We did not impute values for this experiment as few missing values were in a plex (consisting of all time points for one region and one age), but most missing values were between plexes (meaning protein X could be present in one plex but missing from all others), and there was the risk that imputing values of proteins missing entirely between different ages and regions would not produce meaningful or reliable results. Time point 1 was considered 100% protein remaining for each protein in each region and age and the other time point percentages were calculated by dividing the average abundance of the successive time point by the average abundance of time point 1. Differences between all subsequent time points were calculated, and only decreasing trajectories or trajectories with an up to 5% increase between two time points were retained. Because labelled protein should either decrease or remain stable during a pulse-chase experiment, we considered an increase above 5% between time points as caused by measurement noise, we elected to exclude proteins that exceeded this 5% threshold between any two consecutive time points. Of note, in a study that used SILAC labelling in vitro to measure protein degradation20, a threshold of 130% protein remaining was used (see figure 1d in that paper), which is much less stringent than that which we apply. For this experiment, a few technical notes should be made. First, all replicates of all time points from one region and one age were labelled with TMTs and combined into one plex to enable the most accurate quantitative analysis of protein degradation. Second, from the resulting data in each plex, for any given protein in one region and for one age, the amount of protein present or remaining at time point 1 was the maximum and considered 100% and the per cent remaining in subsequent time points was the fraction of the average abundance of biological replicates at that time point divided by the average abundance of biological replicates of time point 1. To compare protein turnover between different regions and/or ages, the percentages of protein remaining at specific times were compared. Notably, by comparing the per cent protein remaining between regions and ages rather than directly comparing raw abundance values, natural differences in protein synthesis and/or variability in protein labelling would not skew analyses and data interpretation. Trajectories were clustered using fuzzy c-means clustering. The optimal number of clusters was determined using minimal centroid distance. For comparison with the aged groups, matching proteins in that group were separated in the same cluster distribution. Integrals for each protein in each cluster were calculated based on the trajectories used for clustering. The integral for each protein was calculated by trapezoidal numerical integration using MatLab trapz. The significance of the average ΔIntegral values was determined by a one-way ANOVA with significant comparisons determined by a Tukey tests. For half-life estimations, we normalized the mean trajectory for each protein such that the first measurement at time point 1 corresponds to the value of one. After calculating the AIC, where AIC = n log(RSS/n) + 2k) for both models, we chose the model with the lower AIC and its corresponding half-life. These modelling approaches were derived from previous publications that studied protein degradation by pulse-chase methodology and subsequently estimated half-life values19,20. It is important to note that as shown in the Extended Data, the modelling approach exhibited good correlation with direct interpolation of half-lives from degradation trajectories. Because of the good correlation and the benefit of being able to estimate the half-life of proteins that did not reach or go below 50% remaining in our data, we used the modelling approach. Likewise, as also shown in the Extended Data, we alternatively calculated half-lives based on individual replicates, which also strongly correlated with the results based on the averaging of all replicates of one time point. To elaborate on the approach based on individual replicates, the data were normalized by computing the mean value at time point 1 and dividing all values by this. A base-10 logarithm was then applied to the normalized data to reduce variance in each time point. The logarithmically transformed normalized trajectories, consisting of four values per time point, were fitted using scipy.optimize.minimize. Other methods for analysing protein degradation data have been implemented and thoroughly described. We tested different approaches and settled on those we reported above because they performed best with our data, achieving a balance of confidence in the reported results without being overly stringent and discarding too much data or introducing artificial data. It is important to note there are many experimental and computational factors that affect ultimate half-life calculations. These factors include, but are not limited to, rate of protein synthesis, use of pulse-chase labelling versus continuous labelling, the length of time between the final pulse of the labelling reagent (such as the NCAA) and the collection of the sample, and choice of interpolation or modelling methods to estimate half-life. These factors are inescapable caveats of current protein turnover measurements and have been thoroughly reviewed52. Last, protein IDs were extracted by the command “which(lbl=x)”, where x represents the cluster number. Protein features were extracted from a comprehensive table of proteins and protein features from a previous publication21 and matched to the proteins of interest in this study. Comparisons of protein features were made between the groups of interest as reported in the main text with statistical analyses being performed either by t-test or one-way ANOVA with a Tukey test. GO analyses were performed using the ShinyGO web application (http://bioinformatics.sdstate.edu/go/)53. Default parameters were used to run the analysis unless otherwise specified. The background brain proteome used was derived from a previous brain proteome publication54 and that used by SynGO42. The output, visuals and tables, including enriched terms, enrichment FDR, number of genes in the pathway and fold enrichment, were filtered by significance (FDR < 0.05) and reported in this paper. To annotate cell types, we used the ClusterMole R package (v.1.1). This package leverages a curated database of cell-type marker genes from Panglao DB and Cell Marker databases to assign cell-type probabilities to differentially expressed genes (DEGs). Specifically, ClusterMole compares the set of upregulated and downregulated genes identified to known cell-type marker signatures. A hypergeometric test was used to calculate P values for overrepresentation of cell-type signatures in the DEG sets. When no cell type was annotated to a particular protein, it was considered non-cell-type specific. Neurodegenerative and neurodevelopmental risk genes were derived from the H-MAGMA study28. By analysing gene regulatory relationships in the disease-relevant tissue, this study identified neurobiologically relevant target genes, improving on existing MAGMA studies. Lists of adult brain risk genes and summary statistics were downloaded from the GitHub repository of the studies (https://github.com/thewonlab/H-MAGMA). Signal peptide prediction was performed by querying protein sequences in SignalP, a server that predicts the presence of signal peptides and the location of their cleavage sites in proteins from Archaea, Gram-positive bacteria, Gram-negative bacteria and Eukarya55. Individual protein sequences were retrieved from UniProt and entered one by one into the SignalP browser search (https://services.healthtech.dtu.dk/services/SignalP-6.0/). We considered protein sequences with signal peptide scores of >0.1 to contain a bona fide signal peptide sequence and to be considered secreted. Proteins with signal peptide scores of <0.1 but >0.02 were considered to contain a likely signal peptide sequence. Proteins with signal peptide scores <0.02 were considered to unlikely contain a bona fide signal peptide sequence and therefore considered unlikely to be secreted. Details of SignalP analysis can be found in the original publication55. Individual gene symbols or protein names were entered one by one into the ExoCarta browser query search (http://exocarta.org/query.html). If the search resulted in any mammalian hit, it was considered a potential exosome cargo. Details of ExoCarta analysis can be found in the original publication56. An in-depth analysis of synaptic ontologies of a protein list was performed by using SynGO, an evidence-based, expert-curated resource for synapse function and gene enrichment studies42. Gene lists were input to the SynGO browser (https://www.syngoportal.org) and default analysis parameters were applied. Visualizations of enrichment analysis on SynGO cellular components and biological processes were exported from the SynGo browser. Details of SynGO analysis can be found in the original publication42. Enrichr57 permits the analysis of enriched pathways in a list of genes and was used to analyse the RNA sequencing data. UniProt IDs were converted to gene symbols using the Retrieve/ID mapping web tool in UniProt (https://www.uniprot.org/id-mapping). In cases when multiple gene symbols were returned for a single UniProt ID, the entry name—the unique gene symbol identifier associated with the UniProt ID—was used for most in-text references and visualizations. All gene symbols associated with a single UniProt ID are listed in the Supplementary Tables with the entry name listed first in the list. Analysis of mouse and human microglia proteomes was performed on processed LC–MS data from a previous publication16. The reported copy number of the four replicates of freshly isolated 3.5-month-old male mouse microglia were averaged, and any protein with an average copy number of >0 was considered as detected. The Allen Brain Cell Atlas58 ‘Whole mouse brain 10xv2 single cell' dataset was mined to examine neuronal and microglia marker genes as specified in the paper. The dataset was down sampled to 10,000 cells and marker genes were determined using the ‘FindAllMarkers' command in Seurat59. For neurotransmitter class and neuron class marker genes, the minimum fraction of cells expressing the gene was set to 25% and the log2[FC] threshold was set to 0.25 and only DEGs with adjusted P < 0.05 were further considered. Analyses comparing microglia versus neuronal marker genes were conducted similarly, except to be considered differentially expressed, the gene had to be expressed in at least 50% of one population (for example, neurons) and no more than 10% of the other population (for example, microglia) and the adjusted P was set to <0.05. Microglia LysoTag data were derived from a pre-print41. Two different microglia LysoTag models were used in the analyses: one model driven by Cx3cr1-cre and the other model driven by Fcrl2-cre. For each model, we considered a protein a hit if the q value was <0.05 and the protein was enriched by immunoprecipitation in the LysoTag model compared with the mock immunoprecipitation from wild-type mice. Hypergeometric tests were performed by using the equation All other numeric inputs were derived from the Venn diagram displayed in Fig. Details of statistical methods are described in relevant subsections above and/or indicated in the figure legends. q values were calculated using the qvalue command from Bioconductor in R Studio and Benjamini–Hochberg-corrected P values were calculated using the p.adjust command with the method being ‘BH' in R Studio. The data in the following figure panels were repeated in a total of three independent experiments: Figs. Unless stated otherwise above, data visualizations were performed in R studio (Posit Software), GraphPad Prism (GraphPad Software), FlowJo (Becton, Dickinson and Company) or Adobe Illustrator (AdobeA) with aesthetic enhancements performed in Adobe Illustrator. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. For datasets related to comparing BONCAT models in the context of a Camk2a-cre driver, the project accession is PXD057020. File name annotations indicate the BONCAT line (MetRS*, PheRS* or TyrRS*) examined in the dataset, which consists of TMT-plexed samples containing BONCAT-labelled samples and respective wild-type background controls. For CMV-cre;BONCAT datasets, the project accession is PXD056569. File name annotations are as follows: WT in name indicates that the sample is a background control (no BONCAT-labelling); a 4-digit code to start file name indicates that the sample is a BONCAT-labelled sample. An abbreviation after WT or the 4-digit code indicates tissue type (B, brain; Liv, liver; H, heart; I, intestine; Lug, lung; M, muscle). For datasets related to BONCAT-labelled neuronal protein differential expression by region, the project accession is PXD057261. File name annotations are as follows: the number indicates the mouse ID; FC, Hipp or ST annotation refers to the brain region of sample (FC, motor cortex; ST, striatum; Hipp, hippocampus); TG annotation indicates that the mouse was a transgenic BONCAT mouse in which protein labelling occurred; WT annotation indicates that the mouse was a wild-type mouse in which BONCAT labelling could not occur and is thus a background control. For datasets related to comparing BONCAT labelling in Camk2a-cre;PheRS* knock-in mice to mice transduced with AAV-Camk2a;PheRS*, the project accession is PXD057456. For datasets related to comparisons between aged and young proteomes from mice transduced with AAV-Camk2a;PheRS*, the project accession is PXD057488. File name annotations are as follows: Y or A annotation in the file name indicates whether sample was from a young (Y) or aged (A) mouse; TP1 in file name indicates that the sample was BONCAT-labelled; BG annotation in file name indicates that the sample was a non-BONCAT-labelled background control. For datasets related to protein degradation among brain regions and tissues, the project accession is PXD056701. File name annotations are as follows: Y, M or A annotation in file name indicates whether the plex consists of young, middle-aged or aged samples, respectively. A number preceding Y, M or A represents the brain region in the plex (3, sensory cortex; 4, visual cortex; 6, hippocampus; 8, hypothalamus). For datasets related to BONCAT-labelled neuronal proteins in aggregates in aged mice, the project accession is PXD066053. File name annotations are as follows: IDs 2864–2875 (n = 11) were a background controls; IDs 2829-39 and 2898–2900 (n = 11) were derived from a BONCAT-labelled model. For datasets related to label-free aged brain aggregates, the project accession is PXD057455. All files are replicates of label-free aggregates from the aged brain. For datasets related to BONCAT-labelled neuronal proteins in microglia, the project accession is PXD066030. File name annotations are as follows: MG22–27 (n = 6) were background controls; MG1–10 (n = 10) were derived from a young BONCAT-labelled model; MG11–17 and MG19 (n = 8) were derived from aged BONCAT-labelled model. For the RNA sequencing datasets comparing sorted microglia to bulk brain homogenate, the Gene Expression Omnibus accession is GSE303078. Unique codes generated and/or modified and used in this study to analyse the data are available from GitHub (https://github.com/Bamees/ProteinAnalysis). López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. The hallmarks of aging. Hetz, C. Adapting the proteostasis capacity to sustain brain healthspan. Multiple click-selective tRNA synthetases expand mammalian cell-specific proteomics. The proteostasis network and its decline in ageing. Ribeiro, F. C. et al. Synaptic proteasome is inhibited in Alzheimer's disease models and associates with memory impairment in mice. Bhukel, A. et al. Autophagy within the mushroom body protects from synapse aging in a non-cell autonomous manner. Alvarez-Castelao, B. et al. Cell-type-specific metabolic labeling of nascent proteomes in vivo. Alvarez-Castelao, B., Schanzenbächer, C. T., Langer, J. D. & Schuman, E. M. Cell-type-specific metabolic labeling, detection and identification of nascent proteomes in vivo. Yuet, K. P. et al. Cell-specific proteomic analysis in Caenorhabditis elegans. Evans, H. T., Bodea, L.-G. & Götz, J. Cell-specific non-canonical amino acid labelling identifies changes in the de novo proteome during memory formation. Qin, W., Cho, K. F., Cavanagh, P. E. & Ting, A. Y. Deciphering molecular interactions by proximity labeling. Zanivan, S., Krueger, M. & Mann, M. In vivo quantitative proteomics: the SILAC mouse. Application of bio-orthogonal proteome labeling to cell transplantation and heterochronic parabiosis. Ernst, R. J. et al. Genetic code expansion in the mouse brain. & Bodea, L.-G. Three methods for examining the de novo proteome of microglia using BONCAT bioorthogonal labeling and FUNCAT click chemistry. Wang, X., Zhou, X., Lee, J., Furdui, C. M. & Ma, T. In-depth proteomic analysis of de novo proteome in a mouse model of Alzheimer's disease. & Rajewsky, K. A cre-transgenic mouse strain for the ubiquitous deletion of loxP-flanked gene segments including deletion in germ cells. Wilson, D. M. et al. Hallmarks of neurodegenerative diseases. Sin, C., Chiarugi, D. & Valleriani, A. Degradation parameters from pulse-chase experiments. Kinetic analysis of protein stability reveals age-dependent degradation. Protein lifetimes in aged brains reveal a proteostatic adaptation linking physiological aging to neurodegeneration. & Savas, J. N. Derailed protein turnover in the aging mammalian brain. Precisely measured protein lifetimes in the mouse brain reveal differences across tissues and subcellular fractions. Morrison, J. H. & Baxter, M. G. The aging cortical synapse: hallmarks and implications for cognitive decline. Berth, S. H. et al. Disruption of axonal transport in neurodegeneration. A. et al. A computational tool (H-MAGMA) for improved prediction of brain-disorder risk genes by incorporating brain chromatin interaction profiles. Taggart, J. C., Zauber, H., Selbach, M., Li, G.-W. & McShane, E. Keeping the proportions of protein complex components in check. Diner, I., Nguyen, T. & Seyfried, N. T. Enrichment of detergent-insoluble protein aggregates from human postmortem brain. Diverse proteins aggregate in mild cognitive impairment and Alzheimer's disease brain. & Yan, R. RTN1 and RTN3 protein are differentially associated with senile plaques in Alzheimer's brains. Hu, X. et al. Transgenic mice overexpressing reticulon 3 develop neuritic abnormalities. Baets, G. D. et al. An evolutionary trade-off between protein turnover rate and protein aggregation favors a higher aggregation propensity in fast degrading proteins. & Dénes, Á Shaping neuronal fate: functional heterogeneity of direct microglia–neuron interactions. & Stevens, B. Microglia: dynamic mediators of synapse development and plasticity. Dissing-Olesen, L. et al. FEAST: A flow cytometry-based toolkit for interrogating microglial engulfment of synaptic and myelin proteins. Ghoochani, A. et al. Cell-type resolved protein atlas of brain lysosomes identifies slc45a1-associated disease as a lysosomal disorder. Koopmans, F. et al. SynGO: an evidence-based, expert-curated knowledge base for the synapse. Deep proteomic analysis of microglia reveals fundamental biological differences between model systems. A., Leyns, C. E. G. & Holtzman, D. M. Intercellular spread of protein aggregates in neurodegenerative disease. Chakraborty, R., Nonaka, T., Hasegawa, M. & Zurzolo, C. Tunnelling nanotubes between neuronal and microglial cells allow bi-directional transfer of α-synuclein and mitochondria. Scheiblich, H. et al. Microglia rescue neurons from aggregate-induced neuronal dysfunction and death through tunneling nanotubes. Huttenlocher, P. R. Synaptic density in human frontal cortex—developmental changes and effects of aging. Petralia, R. S., Mattson, M. P. & Yao, P. J. Communication breakdown: the impact of ageing on synapse structure. & Hirai, H. Comparative study of neuron-specific promoters in mouse brain transduced by intravenously administered AAV-PHP.eB. Fornasiero, E. F. & Savas, J. N. Determining and interpreting protein lifetimes in mammalian tissues. Ge, S. X., Jung, D. & Yao, R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. Teufel, F. et al. SignalP 6.0 predicts all five types of signal peptides using protein language models. Keerthikumar, S. et al. ExoCarta: a web-based compendium of exosomal cargo. Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. Dictionary learning for integrative, multimodal and scalable single-cell analysis. We thank all members of the Wyss-Coray Laboratory for their technical, conceptual and emotional support, with special thanks to past members K. Brewer and S. Shuken for insightful technical comments provided through the genesis of this work, and D. Channappa, K. Dickey, D. Berdnik and H. Zhang for laboratory management; members of the Bertozzi Laboratory, namely N. Riley and B. Floyd for consultation regarding LC–MS experimental design and analyses, and A. Isakova and J. Harberberger of the Knight Initiative for Brain Resilience for granting access to core facility equipment and Allen Brain Cell Atlas data; K. Martens and staff at the Genetic Engineering Technologies Service at the Jackson Laboratory for their contribution. This study was supported by The Phil and Penny Knight Initiative for Brain Resilience (T.W.-C.), Simons Foundation 811253 (T.W.-C.), the International Neuroimmune Consortium with a grant from the Alzheimer's Association ADSF-24-1345203-C (T.W.-C. and M.A.-R.), the NIH Pathway to Independence Award 1K99AG088304-01 (I.H.G), MAC3 Dementia and Ageing Fellowship (I.H.G. ), Innovation and Technology Commission (InnoHK Initiative) of Hong Kong S.A.R. ), NIH Director's Early Independence Award 1DP5OD033381 (A.C.Y), Burroughs Welcome Fund Career Awards at the Scientific Interface (A.C.Y. ), Howard Hughes Medical Institute Fellowship of the Life Sciences Research Foundation (J.F.H. The new knock-in mouse models were developed with funding from an anonymous organization. These authors contributed equally: Andreas Keller, Andrew C. Yang, Tom H. Cheung Ian H. Guldner, Viktoria P. Wagner, Patricia Moran-Losada, Sophia M. Shi, Sophia W. Golub, Kelly Chen, Yann Le Guen, Nannan Lu, Zimin Guo, Jian Luo & Tony Wyss-Coray Ian H. Guldner, Viktoria P. Wagner, Patricia Moran-Losada, Sophia M. Shi, Sophia W. Golub, Kelly Chen, Hamilton Se-Hwee Oh, Nannan Lu, Zimin Guo & Tony Wyss-Coray Chair for Clinical Bioinformatics, Saarland University, Saarbruecken, Germany The Institute for Chemistry, Engineering and Medicine for Human Health (Sarafan ChEM-H), Stanford University, Stanford, CA, USA Sophia M. Shi, Ali Ghoochani, Carolyn R. Bertozzi & Monther Abu-Remaileh Hamilton Se-Hwee Oh, Monther Abu-Remaileh & Tony Wyss-Coray Quantitative Sciences Unit, Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA Biosciences Central Research Facility, The Hong Kong University of Science and Technology, Hong Kong, China The Jackson Laboratory, Bar Harbor, ME, USA Science for Life Laboratory, School of Engineering Sciences in Chemistry, Biotechnology and Health, KTH Royal Institute of Technology, Stockholm, Sweden Chan Zuckerberg Biohub, San Francisco, CA, USA Division of Life Science, Center for Stem Cell Research, HKUST–Nan Fung Life Sciences Joint Laboratory, State Key Laboratory of Molecular Neuroscience, Molecular Neuroscience Center, The Hong Kong University of Science and Technology, Hong Kong, China Guangdong Provincial Key Laboratory of Brain Science, Disease and Drug Development, Shenzhen–Hong Kong Institute of Brain Science, HKUST Shenzhen Research Institute, Shenzhen, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar assisted in running samples on LC–MS under the supervision of T.H.C. assisted in some of the mouse procedures and/or collections; S.W.G., K.C. assisted with in-gel fluorescence and/or tissue staining experiments and some steps of LC–MS preparations. performed formal analyses, with analyses related to protein degradation kinetics, half-life and clustering being under the supervision of A.K. developed wet-bench methodology related to click chemistry. LysoTag data were collected and processes by A.G. under the supervision of M.A.-R. I.H.G. Nature thanks Chris Bennett and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. In-gel fluorescence images comparing the presence of Alexa 647-clicked and thus BONCAT labeled proteins from cell lysates derived from the indicated cell lines expressing either PheRS* (left) or TyrRS* (right) that either did or did not undergo inhibition of protein synthesis prior to non-canonical amino acid treatment. As shown in the experimental schematic (top), BONCAT-expressing cell lines were treated with protein synthesis inhibitor before being treated with the non-canonical amino acid, which abrogated protein tagging relative to cells not treated with protein synthesis inhibitor. b. In-gel fluorescence image of Alexa 647-clicked and thus BONCAT labeled proteins from total brain lysates derived from young Camk2aCre;MetRS*, Camk2aCre;PheRS*, and Camk2aCre;TyrRS* BONCAT transgenic mice and their respective wildtype background controls. c. Fluorescence images of Alexa 488-clicked and thus BONCAT labeled proteins in cortical tissue sections from young Camk2aCre;PheRS* model. Tissues are co-stained with anti-GFP, which should be co-expressed in all cells expressing PheRS*, and DAPI. d. Fluorescence images of Alexa 594-clicked and thus BONCAT labeled proteins in cortical tissue sections from young Camk2aCre;PheRS* model. Tissues are co-stained with anti-NeuN, a neuronal marker, and DAPI to demonstrate localization of BONCAT labeled proteins to neurons. e. In situ heatmap of Camk2a mRNA expression from the Allen Brain Atlas. f. Schematic of methodology used to enrich BONCAT-labeled proteins from total lysates for LC-MS. g. Overlaid histograms comparing the number of proteins pulled down by the methodology illustrated in (f) in unlabeled (wildtype mice treated with AzF) and labeled samples (Camk2aCre;PheRS* mice treated with AzF). Effect size determined by Cohen's D. P values determined by two-tailed Kolmogorov-Smirnov Test and two-tailed Wilcoxon Rank Sum Test. h. Histogram showing the number of labeled proteins (from Camk2aCre;PheRS* mice treated with AzF) and their respective percent increase in enrichment over unlabeled proteins (from wildtype mice treated with AzF). Box and whisker plots comparing the log2-transformed protein intensities for all proteins identified by LC-MS in BONCAT-labeled mice and their respective background controls (wildtype mice treated with the same non-canonical amino acid). Bar chart showing the number of proteins identified without any filtering (black bars), after filtering based on valid values as described in the methods (light grey bars), and after filtering based on valid values, fold change (log2(fold change) > 1 over the background) and p value (<0.05, determined from a two-tailed, two-sample students t-test) (light grey bars) for each Camk2aCre;BONCAT line. m. Western blot image of HSP90 and loading control beta-actin on whole brain lysates derived from various BONCAT-labeled models and ages and respective non-labeled controls to show whether BONCAT-labeling induces an HSP90-mediated heat shock response. n. Fluorescence images for microglia (Iba1, green) staining in cortical tissue sections of an AzF-treated Camk2aCre;PheRS*mouse (left), wildtype mouse not administered AzF (middle), and wildtype mouse administered AzF (right) to determine whether BONCAT-labeling induces microgliosis as evaluated by cellular morphology. All sections were additionally clicked with Alkyne 594 and imaged with the appropriate and equal laser settings for Alkyne 594 detection in order to verify the presence of labeled proteins or lack of labeled proteins in each section. a. Venn Diagrams showing the overlap and exclusivity of proteins labeled by the CMVCre;MetRS*, CMVCre;PheRS*, and CMVCre;TyrRS* models in the indicated tissues. Only proteins identified in both labeled biological replicates with a log2 fold change (BONCAT/background control) > 1 and p value < 0.05 determined from a two-tailed, two-sample students t-test were considered in this analysis. Bar chart of the number of proteins identified in Camk2aCre;PheRS* model that are marker genes of the indicated neurotransmitter types as identified by the Allen Brain Cell Atlas. d. Bar chart of the number of proteins identified in Camk2aCre;PheRS* model that are marker genes of the indicated neuronal classes as identified by the Allen Brain Cell Atlas. f. Fluorescence images of Alkyne 594-clicked and thus BONCAT labeled proteins in the whole brain (top) and cortex (bottom) of a Camk2aCre;PheRS* mouse treated with AzF. The tissue is co-stained for excitatory neuron marker Satb2 (green) to demonstrate the extent of labeled protein localization to excitatory neurons. g. Fluorescence images of Alkyne 594-clicked and thus BONCAT labeled proteins in the whole brain (top) and cortex (bottom) of a Camk2aCre;PheRS* mouse treated with AzF. The tissue is co-stained for inhibitory neuron marker Parvalbumin (green) to demonstrate the extent of labeled protein localization to inhibitory neurons. h. Bar charts of the z-scored expression of the indicated proteins differentially expressed between the motor cortex, striatum, and hippocampus (left) and immunofluorescence images of these proteins and their respective fluorescence intensity heatmaps to validate the regionally-enriched expression (right). b. In-gel fluorescence image of Alexa 647-clicked and thus BONCAT labeled proteins of total brain lysates derived from young (3 months) AAV-Camk2a;PheRS* transduced-mice and the respective background controls. c. Fluorescence images of Alexa 594-clicked and thus BONCAT labeled proteins in brain tissue sections from young AAV-Camk2a;PheRS* transduced-mice. The image on the right shows co-staining for neurons (NeuN, green) to show overlap between click signal and neurons as would be expected from this model. d. Fluorescence images of Alexa 594-clicked and thus BONCAT labeled proteins in brain tissue sections from young AAV-Camk2a;PheRS* transduced-mice. The image on the left shows co-staining for BONCAT labeled proteins and excitatory neurons (Satb2, green) and the image on the right shows co-staining for BONCAT labeled proteins and inhibitory neurons (Parvalbumin, green) in selected regions to illustrate which neuronal populations are BONCAT-labeled in the viral model e. Scatter plots showing the correlation of log2 fold change (BONCAT/background control) (top) and -log10 p values (bottom) of proteins identified in young Camk2aCre;PheRS* transgenic mouse model compared to that of the young AAV-Camk2a;PheRS* model. Only proteins commonly detected with a log2 fold change over respective wildtype background controls and p value < 0.05 were plotted. Statistics derived from a two-sided Pearson's correlation test. f. Bar chart showing the number of proteins identified by LC-MS commonly and exclusively in the young Camk2aCre;PheRS* transgenic mouse model and young AAV-Camk2a;PheRS* model. Only proteins with a log2 fold change over respective wildtype background controls and p value < 0.05 determined by a two-tailed, two-sample students t-test were used. Proteins are color-coded by cell type enrichment as defined by the PanglaoDB and CellMarker databases. P values are derived from a two-tailed, two-sample students t-test. h. Gene Ontology (GO) Cellular Component analysis on BONCAT labeled proteins in the young AAV-Camk2a;PheRS* BONCAT model. Proteins used in the analysis had a log2 fold change > 1 over the respective background control with a p value < 0.05. i. Schematic of AAV-Camk2a;PheRS* transduction and labeling in an experiment to compare nascent neuronal proteins of young (3 m) and aged mice (21 m). j. Volcano plot of neuronal proteins differentially expressed between young and aged mice. P values are derived from a two-tailed, two-sample students t-test. k. GO Biological Process analysis on neuronal proteins downregulated in aged mice relative to young mice. Downregulated proteins were those with a log2 fold change <0 and p value < 0.05, color-coded in blue in (i). a. In-gel fluorescence image of Alexa 647-clicked and thus BONCAT labeled proteins of brain lysates derived from the indicated brain regions at the indicated time points in the chase period following labeling of proteins with azido-phenylalanine. Box and whisker plots of the log2-transformed protein intensities identified by LC-MS for each replicate of BONCAT-labeled mice and their respective background controls (wildtype mice treated with the same non-canonical amino acid) for the protein degradation experiment. Data represents proteins after filtering based on steps 1–3, described in panel c. n = 748, 269, 421, and 391 proteins per replicate of the sensory cortex, visual cortex, hippocampus, and hypothalamus, respectively. c. Schematic of data filtering strategy for the protein degradation experiment. d. Bar charts showing the average percent reduction of neuronal protein abundance between consecutive time points for each age in each region analyzed. e. Scatter plots showing the correlation of neuronal protein half-life in days estimated by modeling versus directly interpolated from the kinetic degradation plots. Only proteins that reached or surpassed 50% remaining are plotted because direct interpolation can only measure such proteins. f. Scatter plots showing the correlation between protein abundance and protein half-life for the sensory cortex (top) and hippocampus (bottom). g. GO Cellular Component analysis of neuronal proteins from the sensory cortex within the top 10% greatest fold change (reduced degradation) from young to aged. h. Box and whisker plots comparing properties of neuronal protein from the sensory cortex within different quartiles of half-life fold change with aging. P values derived from a one-way ANOVA with significant comparisons identified by a Tukey test. i. Scatter plots comparing the log2 fold change of estimated protein half-lives (young to aged) between proteins commonly detected between the indicated regions. Proteins with an absolute value difference >1 were considered regionally vulnerable. P value determined by a two-sided Person's correlation test. a. Elbow plots of cluster number by minimum centroid distance used to determine cluster number for subsequent clustering analyses of young kinetic degradation trajectories with cluster cutoff indicated by a red dotted line (left) and associated clustering and overlap of young and aged kinetic degradation trajectories of the indicated brain regions. Protein membership in the aged clusters was determined by the clustering of young samples to serve as a baseline. The p value was determined by a one-way ANOVA with significant comparisons identified by a Tukey test (right). b. Heatmap of the top 5 most enriched GO Biological Processes identified for each cluster in the young visual cortex (left), hippocampus (middle), and hypothalamus (right). Heatmap colors represent fold enrichment for each pathway. c. Bar plot comparing the integral values of young, middle-aged, and aged proteins on a per-region basis. P value determined by a one-way ANOVA with significant comparisons identified by a Tukey test. Box and whisker plots of the log2-transformed protein intensities identified by LC-MS for each replicate in the neuronal protein aggregate experiment. Bar chart showing the number of proteins identified without any filtering (black bars), after filtering based on valid values as described in the methods (light grey bars), and after filtering based on valid values, fold change (log2(fold change) > 1 over the background) and p value determined by a two-tailed, two-sample students t-test. (< 0.05) (light grey bars) for each BONCAT-labeled neuro-derived aggregates. c. Volcano plot showing the enrichment of BONCAT-labeled neuronal proteins in aged protein aggregates identified by LC-MS relative to the wildtype background control. Proteins with a log2 fold change > 1 over wildtype background controls with a q value < 0.05 are considered hits. d. Venn Diagram showing the overlap of aggregating neuronal proteins in aged mice identified by this study with aggregating proteins identified in the brains of cognitively normal aged humans from Kepchia et al. (left) and GO Cellular Component analysis on the overlapping proteins (right). e. Box and whisker plots comparing properties of neuronal protein from the sensory cortex identified in aged neuronal aggregates compared to those not identified in aged neuronal aggregates. P values derived from a two-tailed unpaired t test. f. Sunburst plots showing synaptic functional representation (left) and synaptic anatomical representation (right) of neuronal proteins identified in aged protein aggregates. h. Venn Diagram showing the overlap of neuronal proteins identified in aged protein aggregates by BONCAT methodology with proteins identified in aged protein aggregates without labeling methodology by us and an independent publication by Molzahn et al. i. Bar charts comparing mass of insoluble protein/protein aggregates between young Camk2aCre;PheRS* mice provided azido-phenylalanine (AzF) and young wildtype mice not provided AzF (left) and aged AAV-Camk2a;PheRS* transduced mice provided azido-phenylalanine (AzF) and aged wildtype mice not provided AzF (right). P values are derived from a two-tailed, two-sample students t-test. j. Fluorescence images comparing protein aggregate (Proteostat, orange) between young Camk2aCre;PheRS* mice provided azido-phenylalanine (AzF) and young wildtype mice not provided AzF (left) and aged AAV-Camk2a;PheRS* transduced mice provided azido-phenylalanine (AzF) and aged wildtype mice not provided AzF (right). Representative flow cytometry plots using canonical microglia, border-associated macrophage (BAM), and peripheral macrophage markers to evaluate the populations represented by the CD11b+ population examined in the protein transfer experiments. d. Volcano plot of genes differentially expressed between sorted microglia and the whole brain. Genes are color-coded according to neuronal marker gene status (purpose) or microglia marker gene status (orange) as determined from performing differential expression analysis on neurons and microglia from the Allen Brain Cell Atlas. e. Bar charts showing normalized counts of canonical microglia markers (left) and canonical neuronal markers (right) between sorted microglia and the whole brain. P values are derived from a two-tailed, two-sample students t-test. f. Cell type enrichment analysis based on Cell Marker 2024 database on genes enriched in sorted microglia relative to the whole brain (left) and genes enriched in the whole brain relative to sorted microglia (right). P values derived from Fisher's exact test and adjusted by Benjamini-Hochberg correction. h. Representative flow cytometry plot to evaluate the potential expression of tdTomato in microglia (CD11b+) from Camk2aCre+/−;Ai14(tdTomato)+/+ mouse. There is strong tdTomato expression in synaptosomes derived from these mice, but the microglia population does not express tdTomato. P values are derived from a two-tailed, two-sample students t-test. Box and whisker plots of the log2-transformed protein intensities identified by LC-MS for each replicate in the neuronal to microglia protein transfer experiment. b. Volcano plot showing the enrichment of BONCAT-labeled neuronal proteins in microglia of all ages combined identified by LC-MS relative to the wildtype background control. Proteins with a log2 fold change > 1 over wildtype background controls with a q value < 0.05 are considered hits. c. Violin plot of Sv2a RNA expression in neurons and microglia from the Allen Brain Cell Atlas. P values are derived from two-tailed Wilcoxn ranked sum test and adjusted by Benjamini-Hochberg correction. d. Violin plot of Sv2b RNA expression in neurons and microglia from the Allen Brain Cell Atlas (left) and fluorescence images of mouse cortical microglia (Iba1, green) and neuronal protein SV2b (red) demonstrating the localization of neuronal SV2b inside microglia in optical sections (left), a three-dimensional rendering of the original image (top right), and a three-dimensional reconstruction of microglia and any SV2b protein that co-localized within the Iba1 signal (bottom right). P values are derived from two-tailed Wilcoxon ranked sum test and adjusted by Benjamini-Hochberg correction. e. Plot of signal peptide score for each neuronal protein identified as transferred to microglia as described in Fig. f. Bar chart showing the percentage of neuronal proteins identified in microglia that are putative secreted proteins based on the presence of a signal peptide or mammalian exosome cargo based on being annotated as such by Exocarta. i. Stacked bar chart showing the proportion of neuronal proteins transferred to microglia that are also found in the microglia lysosome by microglia LysoTag models. Go Cellular Component analysis on neuronal proteins transferred to microglia as described in Fig. m. Venn Diagram showing the overlap of the mouse microglia proteome from Lloyd et al. and the neuronal proteins transferred to microglia as described in Fig. n. Venn Diagram showing the overlap of the human microglia proteome from Lloyd et al. and the neuronal proteins transferred to microglia as described in Fig. Bar chart showing the number of proteins identified without any filtering (black bars), after filtering based on valid values as described in the methods (light grey bars), and after filtering based on valid values, fold change (log2(fold change) > 1 over the background) and p value (<0.05) (light grey bars) for microglia in young BONCAT mice relative to the background (left) and microglia in aged BONCAT mice relative to background (right). p. Volcano plot of neuronal proteins transferred to microglia identified over the background for young microglia. P values are derived from a two-tailed, two-sample students t-test. q. Volcano plot of neuronal proteins transferred to microglia identified over the background for aged microglia. P values are derived from a two-tailed, two-sample students t-test. r. Volcano plot showing the differential abundance of neuronal proteins transferredto aged versus young microglia. Proteins used for the analysis were as described in Fig. Proteins with a log2 fold change > 1 over wildtype background controls with a q value < 0.05 are considered hits. s. Stacked bar charts showing the ratio of presynaptic proteins to postsynaptic proteins as defined by SynGO for young microglia and aged microglia. t. Schematic of study summary and working model. Assessing proteome labelling by different BONCAT lines. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Guldner, I.H., Wagner, V.P., Moran-Losada, P. et al. Ageing promotes microglial accumulation of slow-degrading synaptic proteins. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09917-9'>Probing quantum mechanics with nanoparticle matter-wave interferometry</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 16:13:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 649, pages 866–870 (2026)Cite this article The quantum superposition principle is a fundamental concept of physics1 and the basis of numerous quantum technologies2,3. Yet, it is still often regarded counterintuitive because we do not observe its key features on the macroscopic scales of our daily lives. It is, therefore, interesting to ask how quantum properties persist or change as we increase the size and complexity of objects4. A model test for this question can be realized by matter-wave interferometry, in which the motion of individual massive particles becomes delocalized and needs to be described by a wave function that spans regions far larger than the particle itself5. Over the years, this has been explored with a series of objects of increasing mass and complexity6,7,8,9 and a growing community aims at pushing this to ever larger limits. Here we present an experimental platform that extends matter-wave interference to large metal clusters, a qualitatively new material class for quantum experiments. We specifically demonstrate quantum interference of sodium nanoparticles, which can each contain more than 7,000 atoms at masses greater than 170,000 Da. They propagate in a Schrödinger cat state with a macroscopicity10 of μ = 15.5, surpassing previous experiments5,9,11 by an order of magnitude. When Louis de Broglie postulated that we need to ‘associate a periodic phenomenon with any isolated portion of matter or energy', he predicted that these new ideas would ‘solve almost all the problems brought up by quanta'12. The quantum wave function has become a core concept in modern physics13 and has withstood all tests to date. However, it is still a matter of debate whether quantum physics is already the ultimate theory or if it needs to be extended to explain its transition into classical phenomena. This debate has sparked general interest in the scientific community, shown by a series of recent experiments that have pushed the limits of quantum mechanics. Single atoms were delocalized on the half-metre scale5 or for times longer than a minute14. Matter-wave interference was seen in complex molecules, from fullerene diffraction6, to interference with biomolecules15, van der Waals clusters of organic molecules8 and families of fluorinated oligoporphyrins9. Mechanical cantilevers were cooled to their quantum ground state, both cryogenically16 and optically17. Crystal oscillators18 and levitated nanoparticles were cooled to the lowest level of their harmonic motion in one or two degrees of freedom19,20,21,22,23. Recently, the vibration mode of a bulk acoustic resonator was prepared in a quantum superposition state, with an effective mass of 16 μg (1019 Da) (ref. Here, we present our work on nanoparticle interferometry in a complementary regime. In our case, the centre-of-mass position of clusters containing more than 7,000 atoms becomes delocalized over a distance exceeding the diameter of the particle by more than an order of magnitude. This quantum state is analogous to Schrödinger's cat: here, a macroscopic object that defies intuition because it involves a superposition of classically distinct trajectories25. The unique combination of mass and delocalization is particularly well suited for probing theories that modify the Schrödinger equation through nonlinear and stochastic terms to suppress macroscopic superpositions4. These macrorealistic models have been proposed as a solution to the quantum measurement problem26 as they would explain why very massive objects are always found in a well-defined position27. They assume that the wave function collapses to a localized state, spontaneously28 or induced by gravity29,30, such that a nanoparticle in an interferometer would lose its quantum coherence and the interference fringes would fade. In our present experiment, we observe interference of widely delocalized massive particles, demonstrating that standard quantum mechanics holds at this scale with no need to modify the Schrödinger equation. We quantify the size of our superposition in terms of quantum macroscopicity, a measure that provides a unified framework for constraining a wide range of macrorealist modifications. The observed macroscopicity exceeds that of all previous quantum experiments by an order of magnitude31. The de Broglie wavelength λdB = h/mv of a matter-wave beam is determined by Planck's constant h, the particle mass m and its velocity v. Matter-wave interference at high masses requires both the preparation of low particle velocities and the ability to handle short de Broglie wavelengths. In our multiscale cluster interference experiment (MUSCLE), we achieve this by combining a cryogenic metal cluster source with three ultraviolet (UV) diffraction gratings in a Talbot–Lau configuration, shown in Fig. a, Photo-ionizing gratings as beam splitters. Clusters passing through the antinodes of the optical grating are ionized and removed, whereas those passing through the nodes remain neutral. This confines particles to a spatial region within the grating nodes, resulting in a momentum uncertainty. The light field also induces a dipole moment, imprinting a position-dependent phase onto the clusters. b, Schematic of an optical Talbot–Lau interferometer. Starting with incoherent matter waves, the first grating (G1) prepares coherence by spatially confining the particles, as described in a. Transverse coherence grows towards G2, behind which a Talbot–Lau carpet emerges in the near field. Finally, a third grating acts as a position-resolving detection mask scanned across the interference pattern. c, Schematic of the multiscale cluster interference setup. An effusive sodium source in an aggregation chamber generates the cluster beam. The beam is transmitted through several differential pumping stages into the interferometer chamber kept at ultrahigh vacuum conditions (about 9 × 10−9 mbar). The cluster beam overlaps with three perpendicular standing light waves equally spaced at a distance of L = 0.983 m, forming optical gratings with a period of d = 133 nm. The intensities of the first and third gratings are chosen such that they act as absorptive gratings, whereas the second grating is operated at lower laser intensity, realizing an optical phase grating. After passing through the interferometer, the remaining neutral clusters are photo-ionized using a 425 nm laser diode and mass-filtered. The third grating is scanned transversely across the molecular beam. The integrated signal is then recorded as a function of the displacement of the grating. The nanoparticle and optical components in a and c were rendered in Blender using assets by Ryo Mizuta Graphics. Cluster aggregation sources enable scalable synthesis of particles across a wide mass range, and they are versatile in handling a variety of materials32,33. Here, we prepare sodium clusters consisting of 5,000–10,000 atoms, in a helium–argon mixture at 77 K. They travel at velocities around 160 m s−1 with de Broglie wavelengths between 10 fm and 22 fm. The short de Broglie wavelength makes far-field diffraction challenging even for grating periods on the 100 nm scale: it would require beam collimation to below 200 nrad. However, in 1997, John Clauser proposed using near-field interferometry for grating-based coherent self-imaging of ‘small rocks and live viruses'34, noting that this approach is compact, tolerates initially incoherent beams and offers high spatial resolution. This has been demonstrated with atoms35,36, X-rays37, positrons38, as well as organic and tailored macromolecules7,9. Here, we use it to open a window to matter-wave research with a whole new class of quantum objects, namely, massive metal nanoparticles. A Talbot–Lau interferometer is built from three gratings with period d and spacing close to the Talbot distance LT = d2/λdB (ref. 39). The first and third gratings act as periodic spatial filters to prepare matter-wave coherence in G1 and to resolve the interference fringes that emerge at G3. Standing light waves are favoured over nanomechanical diffraction gratings because their period is precisely defined, and their transmission amplitude can be modified in situ. In contrast to atom interferometry, where optical beam splitters are commonly tailored to specific electronic transitions40,41, ionization and phase gratings are compatible with a large variety of materials and particle sizes. Ultraviolet light serves well as an amplitude or photodepletion grating when the clusters in the antinodes are ionized and discarded. The standing light field additionally induces an oscillating dipole moment in the transmitted clusters, in proportion to their optical polarizability. Thus, it also imprints a spatially periodic phase shift onto the de Broglie wave associated with each nanoparticle. The light for the three gratings is derived from a single-line green laser beam, which is frequency doubled in an external cavity to produce up to 1 W of power at 266 nm. It is split into three partial beams, which are retro-reflected to form three standing light waves, separated by 0.983 m. Neutral clusters transmitted by the interferometer are photo-ionized and counted by a quadrupole mass spectrometer using a conversion dynode and electron multiplier. We sample the interference patterns by scanning G3 across the cluster beam while counting the number of transmitted clusters as a function of the G3 position. The resulting fringes are phase stable to within 3–5 nm over several hours and can be fitted with a sinusoid to determine the visibility \(V=({S}_{\max }-{S}_{\min })/({S}_{\max }+{S}_{\min })\), where Smax and Smin are the maximum and minimum of the fit, respectively. 2a, we show two representative interference fringes of sodium clusters with a diameter around 8 nm and masses ranging from 143 kDa to 197 kDa. We have measured a fringe visibility of up to V = 0.10 ± 0.01, which is limited by the finite photodepletion efficiency in the first and third gratings. a, Interference fringes of sodium clusters with a mean mass of 172 kDa. The experimental data of two independent measurement runs (purple and green dots) are fitted by a sine function (purple and green line) with a visibility of V = 0.10 ± 0.01 and V = 0.08 ± 0.01, for grating laser powers P1 = (62 ± 2) mW, P2 = (15.2 ± 0.3) mW and P3 = (68 ± 2) mW. b, Fringe visibility versus grating laser power of G2. Each data point shows the weighted mean visibility per power bin from multiple independent interference scans of sodium clusters with masses centred around 172 kDa. Visibilities and error bars are derived from per-measurement 1σ confidence intervals of nonlinear least square sine fits (Methods). In this plot, both theory curves were scaled by the same global factor of 0.78. The observation of fringes in the cluster density distribution alone does not provide sufficient evidence for wave-like quantum propagation. They could also be explained by models in which the particles follow classical trajectories. In the presence of three nanomechanical gratings, classical flight paths would produce moiré-like shadow patterns. A similar classical picture is conceivable for sinusoidal transmission gratings in G1 and G3 and a phase grating in G2, in which the latter acts as an array of microlenses because of the optical dipole force. To obtain clear evidence for the wave nature of the observed fringes, their visibility is analysed as a function of the laser power P2 of the second grating, shown by the solid circles in Fig. We compare this to the contrast predicted by both the classical (blue dotted line) and the quantum model (solid red line). It accounts for all coherent and incoherent grating interactions and enables a direct comparison with the prediction of classical mechanics (Methods). We account for the experimental constraints on velocity, ionization cross-section, mass distribution and polarizability by the shaded areas along the theory curves. Interferometer misalignment, gravitational and rotational phase averaging, mechanical vibrations and the scattering of gas particles and thermal radiation can reduce the predicted contrast (Supplementary Information). We take this into account by a global scale factor of 0.78, which is equally applied to the quantum and the classical prediction in this figure. With this single experimental factor included, our experiments are well described by the quantum model and clearly distinct from the classical prediction. Our assumptions regarding the mass, size and velocity distributions of the clusters, as well as the mass dependence of their ionization cross-section, are independently supported by the measured transmission probability as a function of the laser power in G2 (dashed black curve). The model reproduces the experimental data (black crosses) very well, without any additional scaling factor. For substantially more massive clusters, with masses between 400 kDa and 1 MDa, we observe even higher fringe visibilities of V = 0.66 ± 0.09 (Supplementary Information). Although this may seem counterintuitive, it becomes plausible when we consider that the ionization cross-section increases and the transmissive regions in each grating become narrower with increasing size of the cluster. However, the de Broglie wavelength in this mass range (λdB ≱ 3 fm) is too short to distinguish quantum from classical predictions, for our interferometer configuration (Fig. For L ≤ LT, near-field matter-wave dynamics gradually transitions to geometrical optics, in agreement with Bohr's correspondence principle43. a,b, Results are shown for the quantum model (a) and the classical model (b), which both include the effects of ionization and of the dipole force in the grating interaction. Both calculations assume a mean velocity of 160 m s−1, a Gaussian velocity spread of 10 m s−1 and grating powers of P1 = P3 = 100 mW. The colour scale indicates fringe visibility V. For masses beyond the Talbot condition, the quantum and classical models converge. c, Slowing the particles to approximately 25 m s−1 will enable our setup to reliably distinguish quantum from classical dynamics for masses exceeding 1 MDa. Figure 3 shows how the predicted visibilities from quantum (Fig. 3a) and classical theory (Fig. 3b) converge at high cluster masses. At the same time, it highlights a clear discrepancy between quantum and classical predictions in the mass range below 200 kDa (Fig. 3c, we show that it will become possible to unambiguously demonstrate the quantum wave nature of clusters in the MDa range if their velocities can be reduced to about 25 m s−1. While Schrödinger speculated about the possibility of a cat being ‘dead and alive' in the same quantum state—something clearly impossible to observe in our macroscopic world—early experiments with trapped ions44 and cavity fields45 already showed that such superpositions can exist in microscopic systems. Here, we took this idea to a much more massive scale: a nanometer-sized piece of metal being ‘here and there' in the same quantum state with a 133 nm separation between the two locations, more than an order of magnitude greater than the particle itself. What would seem impossible in a classical worldview becomes here an experimental fact of quantum physics. Observing matter-wave interference of the most massive objects to date reveals no breakdown of the quantum superposition principle related to mass or size alone. Moreover, this work establishes a new platform for metal nanoparticles, a material class previously inaccessible to such tests, and it suggests the feasibility of quantum-interference experiments with complex nanobiological objects which cover a similar mass range. To put our experiment into context with other demonstrations of quantum superposition states, we evaluate the macroscopicity measure μ as defined in refs. 10,31. This value quantifies to what extent a given quantum experiment probes the validity of quantum mechanics and how well it can exclude minimal modifications of the Schrödinger equation, which would break the quantum superposition principle at some macroscopic scale. Every successful demonstration of quantum interference falsifies a generic class of minimally invasive, macrorealistic modifications of quantum theory. To obtain the macroscopicity μ, all raw experimental data are used to narrow down the parameter space of these models by Bayesian updating, as explained in ref. 31. This requires a quantitative model for the outcome probabilities in the presence of macrorealistic modifications46. Any experimental imperfection and all decoherence processes are attributed to the macrorealistic modification and will therefore only decrease the macroscopicity (Methods). From our data, we obtain the value μ = 15.5, which surpasses the previous record11 by an order of magnitude, as shown in Fig. a, Macroscopicity values of selected quantum experiments. Blue circles represent atom interferometry; red diamonds represent molecule interferometry; orange crosses represent Bose–Einstein condensates (BECs); green squares represent mechanical resonators; and red star represents sodium nanoclusters in this study, with μ = 15.5. Reference data are taken from refs. 10,11,31 and explained in the Supplementary Information. b, Visualization of size and complexity. The sodium clusters studied here behave as quantum particles at about 0.2 MDa and show high contrast up to the MDa regime. The number of atoms and their mass are compatible with those of large proteins and small viruses (from protein database53). The main motivation for this line of research is to explore the quantum-classical interface bottom-up, systematically, and with all parameters under control. Our interferometer is unique in that it can accept various metals and also dielectric nanoparticles with different mass densities in the same machine. This additional factor would boost the attainable macroscopicity by six orders of magnitude in a ground-based experiment, which may open new opportunities to test the weak equivalence principle with vastly different types of matter. On the applied side, coherent self-imaging creates a cluster density pattern in free flight, which can be shifted by external forces or directed momentum kicks. Particle-like properties, such as electric or magnetic susceptibility, can then be measured on clusters while they are propagating as delocalized waves. These measurements are complementary to explorations in physical chemistry48,49,50 and promise high force resolution. The mass of our sodium clusters (170 kDa) already surpasses that of a coconut cadang-cadang viroid (CCCVd, 81 kDa; refs. 51,52), or a protein such as immunoglobulin G (IgG, 150 kDa; ref. 53). In the next generation of experiments, it is anticipated to approach the MDa mass range of small viruses, such as the satellite tobacco necrosis virus, shown in Fig. Although realizing quantum superpositions with these massive bio-nanomaterials still demands marked advancements in beam preparation, coherent manipulation and detection technologies, recent progress in the generation54,55, in tools for coherent photodepletion56 and in detection of beams of massive biomolecules57 suggests that these challenges will also be solved. The theory of Talbot–Lau interference is best formulated in phase space using the Wigner–Weyl representation of quantum mechanics42. This framework can account for incoherent particle sources, phase and absorption gratings, and all laser-induced photophysical effects, as well as any relevant decoherence process. It also allows for a direct comparison between the predictions of quantum and classical mechanics within the same formalism and set of assumptions. For a cluster with mass m and longitudinal velocity vz, the probability of being detected behind the interferometer can be written as a Fourier series in the transverse position x3 of G3: In a symmetric setup with equal grating separations L and periods d, the Fourier coefficients are where the Talbot–Lau coefficients \({B}_{{\ell }}^{(j)}\) of order ℓ for the jth grating still need to be determined as a function of the Talbot length LT = mvzd2/h. We assume that every absorbed grating photon results in the ionization of the sodium cluster. The transmission of the particle beam through a standing wave of incident laser power P, wavelength λL and Gaussian beam waist wy is then characterized by the mean number of ionizing photons absorbed in each grating antinode as well as by the phase shift induced by the optical dipole potential The values of the UV polarizability α266 and ionization cross-section σion,266 are mass-dependent and determined further below. We can then express the Talbot–Lau coefficients as58 where the coherent phase shift and the ionization depletion are described by For short de Broglie wavelengths, as ξ ≡ L/LT → 0, the latter turn asymptotically into the expressions which appear in the classical description. It yields the same expression (equations (2)–(5)) for the signal, except that equations (6) and (7) are replaced by equations (8) and (9). In our setup, both the quantum and the classical signal are well approximated by a sinusoidal with fringe visibility V = 2|S1|/S0. To assess the macroscopicity of the demonstrated quantum superposition, it is necessary to calculate how the predicted interference signal is affected by the class of minimal macrorealist modifications (MMM) of quantum mechanics10. These are parameterized by the classicalization time scale τe, and by the momentum spread σq and spatial spread σs of a phase space distribution. For our symmetric Talbot–Lau setup, the impact of an MMM is accounted for by multiplying the Fourier coefficients (equation (2)) by with Rcl the radius of the spherical clusters, me the electron mass, j1 a spherical Bessel function and f(x) = 1 − Si(x)/x involving the sine integral10. The dependence on σs can be neglected for this setup. The mean count rate is unaffected by MMM since R0 = 1. The macroscopicity is obtained by using the raw experimental data \({\mathcal{C}}\) (cluster counts at given grating shift x3 and grating powers) for a Bayesian test of the hypothesis that MMM holds with a classicalization time no greater than τe (ref. 31). Bayesian updating yields the posterior probability distribution \(p({\tau }_{{\rm{e}}}| {\mathcal{C}},{{\sigma }}_{{\rm{q}}})\) of the classicalization time τe, starting from Jeffreys' prior, by using the likelihoods obtained by incorporating equation (10) in the detection probability S(x3) (ref. 46). The lowest 5% quantile τm(σq) of the posterior distribution then determines the macroscopicity as \(\mu =\mathop{\text{max}}\limits_{{{\sigma }}_{{\rm{q}}}}({\log }_{10}({\tau }_{{\rm{m}}}({{\sigma }}_{{\rm{q}}})/1{\rm{s}}))\). In our case, a total number of 3,895 data points yield a distribution very well approximated by a Gaussian (Kullback–Leibler divergence 1.27 × 10−3) whose 5% quantile τm = 2.84 × 1015 s (maximized at ħ/σq = 10  nm) remains constant to three decimal places after 3,280 data points. This indicates that sufficient data were recorded and that the distribution is independent of the prior. The resulting macroscopicity is μ = 15.45. Large sodium clusters are generated in a custom-built aggregation chamber, inspired by earlier work32,59. The sodium is evaporated at 650–700 K into a cold mixture of argon and helium at a liquid nitrogen temperature of 77 K and pressure of less than 1 mbar. The resulting distribution covers masses beyond 1 MDa and velocities between 120 m s−1 and 170 m s−1. The clusters exit through a 5-mm aperture and pass three differential pumping stages before they reach the interferometer (Supplementary Information). Two horizontal collimation slits dH1,H2 = 0.5 mm spaced by 1.8 m facilitate the alignment of the grating yaw angles perpendicular to the molecular beam with a precision of about 200 μrad. Two vertical collimation slits dV1 = 0.5 mm and dV2 = 1 mm, spaced by 2.2 m, confine the beam height and ensure good overlap with the standing light wave. This also reduces the influence of gravitationally induced phase averaging. The optical polarizability α266, absorption cross-section σabs,266 and ionization potential Ei depend on the size, mass and purity of the cluster. They determine transmission, the maximal matter-wave phase shift ϕ0 and the mean number of absorbed photons n0 in the antinodes of the grating. Photophysics60 and thermodynamics61 of small sodium clusters have been extensively studied, and the preparation of particles up to 1 MDa has been demonstrated before59. However, the mass-selected UV polarizability has not been known. Here, we use the high-contrast fringe patterns of clusters between 0.4 MDa and 1 MDa to determine it in a mass range for which the classical and quantum models predict the same visibilities. We derive a value of α266/atom = −4πε0 × (4.5 ± 0.5) Å3 (Supplementary Information), which is consistent with the experiments and the quantum model for m = 100–200 kDa. By measuring the mass-selected transmission of the interferometer for different grating powers, we determine an effective cross-section of σion,266 = (0.537 × m [kDa] − 1.5) × 10−20 m2 for our clusters. After passing all gratings, the cluster beam is photo-ionized using 425 nm light and the cations are filtered by their m/z ratio using a quadrupole mass spectrometer. The mass filter includes guiding ion optics (Extrel) and 300 mm long quadrupole rods (Oxford Applied Research) with a diameter of 25.4 mm. The mass filter is operated at a resolution of Δm/m = 0.32. Interference scans centred on mass m, therefore, involve clusters within a mass range of  ±Δm/2, where the transmission function is close to rectangular shape and taken into account in our models. The mass filter was centred at 170 kDa. The underlying mass distribution, convoluted with the trapezoidal transmission, shifts the effective mass centre towards 172 kDa. The selected cluster ions are counted by a channel electron multiplier with a conversion dynode at 10 kV. We must also account for the mixing of multiply charged ions with identical m/z ratios. Based on the measured work function of W = (2.4 ± 0.1) eV (Supplementary Information), neutral clusters with a diameter of dCl ~ 8  nm exhibit an ionization threshold of Ei = 2.53 eV, followed by Ei,+1 = 2.88 eV and Ei,+2 = 3.23 eV for subsequent ionization processes. The detection laser has a photon energy of Eph = 2.92 eV and can generate doubly charged ions, whereas triply charged ions remain energetically out of reach. We have selected doubly charged clusters in the detector and verified the correct cluster mass by analysing mass spectra at both low and high detection laser powers (Supplementary Information). In the antinodes of the gratings, the 266 nm light can also lead to multiply charged ions. However, this does not affect the interference pattern, because every ion is removed from the cluster beam by electrostatic deflection, independent of its charge state. Only clusters that remain neutral while passing through all gratings contribute to the final interference pattern. The cluster velocity distribution is determined from a time-of-flight measurement, in which we imprint a start time signal onto the cluster beam by UV photodepletion close to G1, and we measure the cluster arrival time behind the ionizing mass spectrometer. The time-of-flight data are corrected for the drift time inside the quadrupole, where it is slightly accelerated by the entrance voltage U to \(v{\prime} =v+\sqrt{2eU\,/\,m}\). A convolution of a Gaussian drift time distribution and a rectangular chopper opening function is then fitted to the corrected unsmoothed data. The results are converted to a velocity distribution. Time-of-flight and velocity spectra for m/z = 100 kTh clusters are shown in the Supplementary Information. Up to 4 W of 532 nm light (Coherent Verdi V18) is converted to up to 1 W of 266 nm UV light by intracavity second harmonic generation (Sirah Wavetrain 2). The UV output is vertically expanded and split into three grating beams, using polarizing beam splitters and half-wave plates to regulate the power for each grating. Cylindrical lenses (f = 140 mm) focus the laser horizontally onto high-reflectivity (R = 99.5%) mirrors in vacuum to generate the standing light waves. The beam waists before the lenses are W1 × H1 = 1,130 × 620 μm2, W2 × H2 = 1,020 × 575 μm2 and W3 × H3 = 1,020 × 575 μm2, with ΔHi = ΔWi = ±50 μm. Here, Wi represents the 1/e2 waist radii along the molecular beam direction and Hi is the vertical waist. This small waist alleviates the alignment requirements with regard to the cluster beam tilt angle. The waist is still sufficiently large to ensure that the Rayleigh length, zR = 4.7 mm, is an order of magnitude larger than the cluster beam width of 500 μm. The surfaces of all three grating mirrors are aligned parallel to the particle beam axis, with the standing light wave along the mirror normal. The gratings exhibit three angular degrees of freedom: pitch, yaw and roll. The yaw angle, between the mirror surface and the particle beam, is adjusted to better than 200 μrad. The relative roll of the three mirrors, that is, their rotation around the axis parallel to the cluster beam is aligned to a difference less than 20 μrad. They are all stabilized with respect to the gravitational field of Earth to better than 50 μrad. We obtain the interference scans by measuring the number of transmitted clusters as a function of the transverse displacement of the third grating G3, which is moved in steps of Δx = 15 nm. At each position, the mass-filtered ion signal is integrated for a time interval of up to four seconds. A sinusoidal fit to the data then provides the periodicity, phase and amplitude of the fringes. By design of first-order Talbot–Lau interferometry, the periodicity is equal to the grating period. Each visibility \({{\mathcal{V}}}_{i}\) results from a nonlinear least-squares sine fit to the raw counts and is accompanied by 1σ confidence bounds \(({{\mathcal{V}}}_{i,{\rm{lb}}},{{\mathcal{V}}}_{i,{\rm{ub}}})\). Measurements are grouped by optical power into bins. As a consistency check, we compute the reduced chi-square \({\chi }_{{\rm{red}}}^{2}\) using the same per-point uncertainties as the weights. For overdispersed bins (\({\chi }_{{\rm{red}}}^{2} > 1.5\)), we scale the upper and lower error bars of the mean by \(\sqrt{{\chi }_{{\rm{red}}}^{2}}\). For visualization, plotted lower bounds are truncated at 0; all weighting and dispersion checks use the untruncated values. Data and code supporting the findings of this research are available at Zenodo (https://doi.org/10.5281/zenodo.17502163). Additional data or materials used in the study can be provided upon request. & Sands, M. The Feynman Lectures on Physics, Vol. III: Quantum Mechanics (Addison-Wesley, 1965). Tino, G. & Kasevich, M. Atom interferometry. International School of Physics 'Enrico Fermi' (eds Tino, G. M. & Kasevich, M. Bassi, A., Lochan, K., Satin, S., Singh, T. P. & Ulbricht, H. Models of wave-function collapse, underlying theories, and experimental tests. Kovachy, T. et al. Quantum superposition at the half-metre scale. Arndt, M. et al. Wave-particle duality of C60 molecules. Gerlich, S. et al. A Kapitza–Dirac–Talbot–Lau interferometer for highly polarizable molecules. Haslinger, P. et al. A universal matter-wave interferometer with optical ionization gratings in the time domain. Fein, Y. Y. et al. Quantum superposition of molecules beyond 25 kDa. Nimmrichter, S. & Hornberger, K. Macroscopicity of mechanical quantum superposition states. Schrinski, B. et al. Macroscopic quantum test with bulk acoustic wave resonators. De Broglie, L. Waves and quanta. Schrödinger, E. An undulatory theory of the mechanics of atoms and molecules. Coherence limits in lattice atom interferometry at the one-minute scale. Mairhofer, L. et al. Quantum-assisted metrology of neutral vitamins in the gas phase. O'Connell, A. D. et al. Quantum ground state and single-phonon control of a mechanical resonator. Chan, J. et al. Laser cooling of a nanomechanical oscillator into its quantum ground state. Doeleman, H. M. et al. Brillouin optomechanics in the quantum ground state. Delić, U. et al. Cooling of a levitated nanoparticle to the motional quantum ground state. Tebbenjohanns, F., Mattana, M. L., Rossi, M., Frimmer, M. & Novotny, L. Quantum control of a nanoparticle optically levitated in cryogenic free space. Simultaneous ground-state cooling of two mechanical modes of a levitated nanoparticle. Dania, L. et al. High-purity quantum optomechanics at room temperature. Troyer, S. et al. Quantum ground-state cooling of two librational modes of a nanorotor. Bild, M. et al. Schrödinger cat states of a 16-microgram mechanical oscillator. Schrödinger, E. Die gegenwärtige Situation in der Quantenmechanik. Quantum Theory and Measurement (Princeton Univ. & Ghirardi, G. Dynamical reduction models. & Weber, T. Unified dynamics for microscopic and macroscopic systems. Diosi, L. Models for universal reduction of macroscopic quantum fluctuations. Penrose, R. On gravity's role in quantum state reduction. Schrinski, B., Nimmrichter, S., Stickler, B. & Hornberger, K. Macroscopicity of quantum mechanical superposition tests via hypothesis falsification. Haberland, H., Karrais, M. & Mall, M. A new type of cluster and cluster ion source. Pedalino, S., Sousa, T., Fein, Y. Y., Gerlich, S. & Arndt, M. Exploring metal nanoparticles for matter-wave interferometry. Clauser, J. F. in De Broglie-Wave Interference of Small Rocks and Live Viruses (eds Cohen, R. S. et al.), 1–11 (Kluwer Academic, 1997). Clauser, J. F. & Li, S. Talbot–von Lau atom interferometry with cold slow potassium. Fray, S., Diez, C. A., Hänsch, T. W. & Weitz, M. Atomic interferometer with amplitude gratings of light and its applications to atom based tests of the equivalence principle. Pfeiffer, F., Weitkamp, T., Bunk, O. & David, C. Phase retrieval and differential phase-contrast imaging with low-brilliance x-ray sources. Ariga, A. et al. First observation of antimatter wave interference. Lau, E. Beugungserscheinungen an Doppelrastern. Kasevich, M. & Chu, S. Atomic interferometry using stimulated Raman transitions. Riehle, F., Kisters, T., Witte, A., Helmcke, J. & Borde, C. J. Optical Ramsey spectroscopy in a rotating frame: Sagnac effect in a matter-wave interferometer. Nimmrichter, S. & Hornberger, K. Theory of near-field matter-wave interference beyond the eikonal approximation. Bohr, N. The quantum postulate and the recent development of atomic theory. Monroe, C., Meekhof, D. M., King, B. E. & Wineland, D. J. A ‘Schrödinger cat' superposition state of an atom. Observing the progressive decoherence of the ‘meter' in a quantum measurement. Schrinski, B., Nimmrichter, S. & Hornberger, K. Quantum-classical hypothesis tests in macroscopic matter-wave interferometry. Kiałka, F., Fein, Y. Y., Pedalino, S., Gerlich, S. & Arndt, M. A roadmap for universal high-mass matter-wave interferometry. Xu, X. S., Yin, S. Y., Moro, R. & Heer, W. A.de Magnetic moments and adiabatic magnetization of free cobalt clusters. Rivic, F., Fuchs, T. M. & Schäfer, R. Discriminating the influence of thermal excitation and the presence of structural isomers on the Stark and Zeeman effect of AlSn12 clusters by combined electric and magnetic beam deflection experiments. Antoine, R. et al. Electric susceptibility of unsolvated glycine-based peptides. Saucedo Carabez, J. R., Teliz Ortiz, D., Vallejo Perez, M. R. & Beltran Pena, H. The avocado sunblotch viroid: an invisible foe of avocado. The RCSB protein data bank: integrative view of protein, gene and 3D structural information. Controlled beams of shock-frozen, isolated, biological and artificial nanoparticles. Mucha, E. et al. in Spectroscopy of Small and Large Biomolecular Ions in Helium-Nanodroplets (eds Slenczka, A. Schätti, J. et al. Neutralization of insulin by photocleavage under high vacuum. Highly sensitive single-molecule detection of macromolecule ion beams. Nimmrichter, S., Haslinger, P., Hornberger, K. & Arndt, M. Concept of an ionizing time-domain matter-wave interferometer. Göhlich, H., Lange, T., Bergmann, T., Näher, U. & Martin, T. P. Ionization energies of sodium clusters containing up to 22000 atoms. & Issendorff, B. V. Photoelectron spectroscopy of sodium clusters: direct observation of the electronic shell structure. Negative heat capacity for a cluster of 147 sodium atoms. Funding for this project was provided by a Gordon and Betty Moore Foundation grant (GBMF10771, https://doi.org/10.37807/GBMF10771) and by the Austrian Science Fund (FWF) grant 32542-N. Open access funding provided by University of Vienna. We thank P. Geyer and Y. Y. Fein for their contributions at the early stage of the experiment and S. Sindelar for his support with metal cluster beams. is indebted to B. v. Issendorff for many discussions on cluster science throughout the years leading to this experiment. We thank V. Kresin for comments on alkali cluster sources and A. Shekhoon for discussions on the cluster work function. Open access funding provided by University of Vienna. Sebastian Pedalino, Bruno E. Ramírez-Galindo, Richard Ferstl, Markus Arndt & Stefan Gerlich Sebastian Pedalino, Bruno E. Ramírez-Galindo & Richard Ferstl Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar was responsible for experimental design, measurement, data analysis, theory and writing; B.E.R.-G. and R.F. were involved in experimental design, measurement and theory; K.H. provided the theory and the Bayesian estimate of quantum macroscopicity and contributed to writing; M.A. contributed to experimental design, theory and writing; S.G. was responsible for experimental design, analysis and theory. Correspondence to Markus Arndt. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. This file contains the following sections: (1) Experiment; (2) Phase averaging and decoherence; and (3) Quantitative analysis. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Pedalino, S., Ramírez-Galindo, B.E., Ferstl, R. et al. Probing quantum mechanics with nanoparticle matter-wave interferometry. Version of record: 21 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09983-z'>Baby-to-baby strain transmission shapes the developing gut microbiome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 16:08:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The early infant microbiome is largely primed by microbial transmission from the mother between birth and the first few weeks of life1,2,3, but how interpersonal transmission further shapes the developing microbiome in the first year remains unexplored. Here we report a metagenomic survey to model microbiome transmission in the nursery setting among babies attending the first year, their educators and their families (n = 134 individuals). We performed dense longitudinal microbiome sampling (n = 1,013 faecal samples) during the first year of nursery and tracked microbial strain transmission within and between nursery groups across 3 different facilities. We detected extensive baby-to-baby microbiome transmission within nursery groups even after only 1 month of nursery attendance, with nursery-acquired strains accounting for a proportion of the infant gut microbiome comparable to that from family by the end of the first term. Baby-to-baby transmission continued to grow over the nursery year, in an increasingly intricate transmission network with single strains spreading in some classes, and with multiple baby-acquisition and species-transmissibility patterns. Having siblings was associated with higher microbiome diversity and reduced strain acquisition from nursery peers, while antibiotic treatment was the condition that most accounted for the increased influx of strains. This study shows that microbiome transmission between babies is extensive during the first year of nursery, and points to social interactions in infancy as crucial drivers of infant microbiome development. The early infant microbiome assembles via intricate and partially stochastic microbial acquisitions that have the mother as the primary source and other family members as additional ones1,2,3,4,5. The infant microbiome then evolves during the following few years with complex dynamics that later result in a more stable adult-like microbiome6. Although early family-to-baby microbiome strain transmission has been quite extensively investigated1,2,3,4,7, later infant developmental stages, including those involving interaction with other peers in social contexts, have received very little attention. As the person-to-person intra-generational microbiome transmission has been recently revealed to be extensive and impact the personal microbiome make-up8, we predicted that early social contexts such as nurseries might exert a large impact on infant microbiomes via baby-to-baby transmission. Beyond work on pathogen spreading9,10 and linked immune competence development11,12,13, microbiome investigations in nurseries are limited in observing increased microbial diversity among attendants14. This leaves a major gap in the understanding of the dynamics of human microbiome maturation during the key first 1,000 days of life15. Here we present microTOUCH-baby, a strain-resolved longitudinally dense metagenomic study modelling interpersonal gut microbiome transmission between babies attending the nursery for the first time and their close contacts, including family members and nursery educators. We set up the microTOUCH-baby cohort to study the dynamics of microbiome development and transmission among babies of about 1 year of age and their close social interactions network (Methods). Participants included 43 babies attending the first year of nursery (median age at nursery admission 10 months), 7 co-living siblings, 39 mothers and 30 fathers of the babies, and 5 pets from the participants' houses, as well as 10 nursery educators (134 volunteers in total; Fig. Baby participants were enroled from three public nurseries in Trento (Italy). Babies spent on average 8 hours per weekday (after the ‘settling-in period'; Methods) in the nursery, with limited activities and spaces shared between the two classes in the same nursery, which are followed by different educational staff. a, microTOUCH-baby study design and overview. b, Species-level microbiome composition overview of the microTOUCH-baby cohort during first term (principal coordinate analysis on Jaccard dissimilarity, n  = 646). Samples are coloured by host categories and shapes indicate the nursery. Baby samples' colour intensity is according to time point (from initial T01 to final T15). c,d, Average SGB richness across all timepoints and for all individuals in each family member category having versus not having a sibling (c) and having versus not having a pet (d). e,f, Change in alpha-diversity (SGB richness; e) and beta-diversity (Jaccard dissimilarity; f) across participant types between the beginning and the end of the first term, with n indicating the number of individual–individual pairs. Beta-diversity refers to the all-versus-all within-nursery dissimilarities. P values are reported where statistically significant (two-sided Mann–Whitney U-test in c and d and two-sided Wilcoxon signed-rank tests for e and f); all other comparisons are non significant. Sampling started before the beginning of the first term (T01), hence before participants from different families had any nursery-related contact among them, and ended after the Christmas nursery closure (Fig. During nursery attendance, we collected stool samples of the babies on a weekly basis, whereas educators and parents were less densely sampled (Methods). For all participants in group 1 of nursery A, sample collection continued through the second term. Two additional follow-up samples were collected for all participants at nursery year's conclusion (TA) and at the end of the summer break (TB) (Fig. Overall, we collected and metagenomically sequenced 1,013 microbiome samples (average sequencing depth 15.61 Gbp; Methods). Host metadata information included exact age, past and current host-health data, antibiotic exposures, maternal delivery information (Methods, Fig. Metagenomes were processed via the MetaPhlAn 4 computational tool16 to generate taxonomic profiles at species-level genome bin (SGB)17 resolution (Supplementary Table 4 and Extended Data Fig. 1a), including yet-to-characterize species (that is, unknown SGBs accounting for 46.37% of total SGBs). We then used the StrainPhlAn 4 computational tool16,18 to generate strain-level phylogenies for 311 known SGBs and 201 unknown SGBs that were used to infer microbiome strain transmission8 (Methods). We first observed expected microbiome structures1,19,20, with large compositional divergence between adults and babies (Fig. 1b, 2 and 3, and Supplementary Table 5), age-dependent differences in babies (Extended Data Fig. 4a–e), and diet-dependent microbial stratification in adults (Extended Data Fig. 4f), but not in babies after accounting for age (Supplementary Table 6). Interestingly, at T01 (median age 10 months), the impact of maternal intrapartum antibiotic prophylaxis against group B Streptococcus and of mode of delivery on alpha-diversity was already not detected as statistically significant (Mann–Whitney U-test, n = 37, U = 137, P = 0.68 and n = 37, U = 109, P = 0.89, respectively; Extended Data Fig. Some compositional patterns were suggestive of a role of microbiome transmission. Babies having a sibling had, for example, an overall higher SGB richness compared with babies without brothers and sisters (n = 40, U = 271, P = 0.012; Fig. 1c and Supplementary Table 6), further supporting previous observations21,22 and suggesting that siblings may provide important sources for infant microbiome enrichment. In contrast, babies with pets showed lower overall SGB richness (n = 40, U = 61, P = 0.012; Fig. 1d), but significance was lost after adjusting for age (Supplementary Table 6). Babies' alpha-diversity increased during the 3 months of nursery attendance (Fig. 1e), and although the total pool of microbial species detected among babies in the nursery did not change noticeably throughout the study (Extended Data Fig. 5e), the inter-baby beta-diversity decreased significantly (7% average decrease; Wilcoxon signed-rank test n = 116 baby pairs, W = 1,026, P = 7.0 × 10−11; Fig. As overall this might be indicative of baby microbiome convergence influenced by inter-individual transmission, we performed strain-level transmission analysis to investigate this hypothesis. Extending our StrainPhlAn-based validated pipeline8 (Methods), we defined a strain-sharing event as the identification of the same strain (that is, differing by a genetic distance lower than the pre-computed optimal species-specific threshold distinguishing between inter- and intra-individual genetic distance distributions) in different microbiome samples. Strain-sharing rates (SSRs) are computed as the number of strains shared between a pair of microbiome samples over the number of species with profiled strains present in both samples (Methods). Applied on the task of inferring mother–baby transmission, the pipeline estimated a 50% median SSR for babies at the beginning of the study, which is highly consistent with previous results irrespective of population (Extended Data Fig. Overall, we captured over 9.47 million instances of the same SGB typed at the strain level in different samples (including those from the same participant and from different participants), with a total of 5.97% of cases in which the same strain of the SGB was present, resulting in 565,258 detected strain-sharing events (Supplementary Tables 7 and 8). Within-individual strain-sharing accounted for 27.9% of the total (157,599 events, with 99% likelihood of samples from the same individual sharing at least 1 strain, and 87% at least 5) but also strain-sharing between different individuals in the same family was very high (51,483 events, 9.1% of the total, with 86% likelihood of sharing at least 1 strain, 47% at least 5), with rarer between-family strain-sharing instances at T01 (46% likelihood of sharing at least 1 strain and 3% at least 5; Extended Data Fig. Although most strain-sharing over the first term was observed among individuals from different families (356,176 events, 63%), this reflected the >75-times greater number of between-family comparison pairs; after normalizing for the number of comparisons, one order of magnitude fewer strains were shared between families versus within family (0.7 versus 7.9 strains shared per sample pair; Supplementary Table 7). The 0.7 average strains shared by unrelated individuals represent the cohort's microbiome sharing background, including untraced social interaction before T01, clonal strains spreading into the nursery-associated local community and possible false-positive instances, among other factors. As a representative example of the combined capabilities of our study design and metagenomic pipeline to trace complex strain transmission chains, we illustrate the interpersonal transfer of a nursery-acquired strain of Akkermansia muciniphila (SGB9226) in group 1 of nursery B. A strain from this species was first introduced in the nursery group by a baby (B05) who probably obtained it from their mother, passed to another baby (B06), to then be found in their mother (M06) and father (F06), in the latter replacing another A. muciniphila strain (Fig. A. muciniphila strains contain CRISPR arrays that can be used as unique genetic tags for strains23,24 that further confirmed A. muciniphila strain identity across volunteers (Methods). Metagenomic assembly also validated such transmission patterns for the limited number of strains (8 out of 19 StrainPhlAn-positive samples) that could be reconstructed into draft genomes of sufficient quality, with high genomic similarity between assemblies from samples with the same strain according to StrainPhlAn (pairwise average nucleotide identity (ANI) 99.97%, which aligned with same-strain boundaries independently estimated elsewhere25,26). We note that the missing detection of A. muciniphila strains (grey circles in Fig. 2a) was overall consistent with the absence of the species as shown in a high-sensitivity, SGB-specific polymerase chain reaction (PCR) (Methods and Extended Data Fig. Within this example, we found only one sample in which we missed the metagenomic strain profiling to be PCR-positive at the SGB-level, concordantly with a non-zero relative abundance (0.04%) in its MetaPhlAn profile (B05_T08; Extended Data Fig. 6b), being thus the single case in Fig. Another transmission chain example involved Alistipes finegoldii (SGB2301) and included an educator (Extended Data Fig. 6c), further contributing to show the potential of our approach to recapitulate microbial transmission in nurseries. a, Strain-level profiling for A. muciniphila SGB9226 (left) uncovers the chain of transmission events of one strain of this species in group 1 of nursery B (right). Participant types are identified by shape (mother, diamond; baby, circle; father, square) containing participant identifiers composed of the first letter indicating participant type (M, mother; B, baby; F, father) and the family number; familiar relations are also highlighted by same-colour filling. On the right, each circle represents a sample collected from the participants depicted, with colour filling indicating the identity of the strain of A. muciniphila detected in the sample (except grey, used to indicate that the SGB was not detected/typable at the strain level) and arrows indicating the most likely transmission event. The light orange and grey circle identifies SGB9226-positive sample (B05_T08) in which a strain could not be profiled by StrainPhlAn. The identification of shared CRISPR spacers of the target strain of A. muciniphila (orange circles) across different samples is indicated by an asterisk. b, Strains present at most in one baby before nursery admission (T01) and spreading to other participants in the same nursery, reaching ≥50% prevalence in the following time points, until T15. Left and right y axes show the proportion and number of babies in which the outbreaker strain was detected, respectively. The left y axis also refers to the proportion of babies in which the SGB was detected (that is, their prevalence in the nursery). We also explored potential gut microbiome transmission between household pets and their families. Anecdotally (given the only five pets considered), we overall identified a low total number of pet–human strain-sharing events, with intra-family pet–baby strain-sharing significantly higher than inter-family (Fisher's exact test, n = 211, P = 0.005; Extended Data Fig. Strains found to be transmitted between babies and pets belonged to human-associated species that had also been previously detected in pet gut microbiomes (Faecalimonas umbilicata, Ruminococcus gnavus, Clostridium sp. AT4 and Phocaeicola vulgatus27,28,29,30), indicating they may be ecologically fit to overcome host-species boundaries. First, we found the overall pool of distinct strains to decrease over time (that is, average nursery strain heterogeneity decreasing from 0.91 at T01 to 0.77 at T15, Mann–Whitney U-test, n = 454, U = 34,312, P = 1.3 × 10−11). Considering that the total reservoir of microbial species did not increase (Extended Data Fig. 5e), this indicates that some strains within the same species may have spread among babies and prevailed over other strains initially present (Extended Data Fig. We then focused on strains that showed efficient spreading within a nursery. We found 8 cases of strains initially detected in no more than one baby before nursery start (T01) reaching ≥50% prevalence afterwards (Fig. Among these, a Streptococcus gallolyticus (nursery A) and a Bifidobacterium pseudocatenulatum (nursery B) strain were introduced in the nursery after approximately the first month of attendance and progressively spread to seven and eight babies, respectively (Fig. Although S. gallolyticus spread appeared to dwindle after reaching the maximum diffusion, B. pseudocatenulatum presence was steadily detected, consistent with the high prevalence of the Bifidobacterium genus in the infant population2. Other cases of bacterial strain diffusion involved Escherichia coli and Veillonella dispar in nursery B, and Clostridium innocuum in nursery C, which was possibly limited in its spread by other conspecific strains and niche preemption dynamics31. Quantification of strains shared between babies attending the same nursery over time revealed they had, on average, more shared strains at the end of the first term than before nursery admission (the average number of strains shared with any other baby was 2.5 at T01 and 7.2 at T15, or 8.8 at T15 when disregarding strains already present at T01 and only for babies with samples available at both time points; Fig. Accordingly, whereas at T01 baby strain-sharing relations were not recapitulating nursery attendance, at T15 they clustered consistently with it (Fig. We thus found strong evidence of quantitatively relevant acquisition of nursery-specific microbial profiles by babies, occurring via inter-individual strain transmission even in the relatively short time frame of the first nursery term. a, Average number of strains shared between each baby and other participants at T01 and T15. The triangles under the boxes report the average number of strains shared between the baby and any participant in ‘family' (mother, father, sibling) or ‘nursery' (other babies, educator). b, Average number of shared strains between baby pairs in the same versus different nursery; P values (two-sided permutation test for means; Methods) for intra- versus inter-nursery comparisons are shown in italic in the circle. The statistics for a and e are in Supplementary Tables 9 and 10. Networks are built on strain-sharing matrices among all babies at T01 and T15. c, Strain replacement rate (one minus the SSR) between initial and final time points. P values are reported where statistically significant (two-sided Mann–Whitney U-tests); all other pairwise comparisons are non significant. d, Baby–baby SSR and average number of strains shared throughout the first term. In d and e, statistical significance asterisks refer to the highest significant P value adjusted for multiple comparisons (two-sided permutation test for medians; Methods) for the set of comparisons indicated in the legend, with **P < 0.01 and ***P < 0.001. Left and right y axes indicate average strains shared and average common SGBs. e, Baby–baby SSR and average number of strains shared at T01, at the beginning and the end of the second term (T15 and TA), and after the summer break (TB), across all babies in all nurseries. At the top, P values are reported where statistically significant (two-sided Wilcoxon signed-rank test evaluating longitudinal SSR for paired baby–baby pairs attending the same nursery). Investigating longitudinal gut microbiome changes, babies showed the lowest rate of SGB retention (defined as the Jaccard similarity between samples from initial and final time points of the same individual; Extended Data Fig. 7b) and the highest rate of strain replacement (defined as 1 − SSR) among the retained SGBs (Fig. A median 44.4% of the retained SGBs in babies showed baseline strain replacement during the 5 months of the study. In contrast, all other participants replaced a much lower fraction of strains in their gut (medians below 11.1%), with strain replacement rates correlated although non-significantly with age among non-baby participants (Spearman's test, n = 68, ρ = 0.22, P = 0.071; Extended Data Fig. This reflects the expected high plasticity of the infant gut microbiome with its rapidly evolving ecosystem and limited colonization resistance6,32. To assess the extent to which nursery attendance affects microbiome assembly in babies via microbiome transmission, we quantified and compared the SSR between pairs of babies within the same group or nursery, and across different nurseries at each time point (Fig. Strain sharing among babies in the same nursery group was significantly higher after approximately only 1 month of nursery attendance compared with babies from different nurseries (median SSR 8.3% versus 0% at T04; permutation test for medians, n = 249, P = 0.001). This is all the more noteworthy in view of the first 2 weeks of the nursery's ‘settling-in period' during which babies attend discontinuously and for shorter periods. In addition, at the end of the first term (T15), the SSR in the same nursery group reached an average of 20.2%, significantly higher than the SSR between babies attending different nurseries (4.6%; permutation test for medians, n = 312, P < 0.001) and higher than the SSR among babies attending the same nursery but in different groups (16.1%; permutation test for medians, n = 122, P = 0.079, significant at T08 P = 0.026, T10 P < 0.001 and T13 P = 0.001). By extending the investigation to the second term of nursery, we found the baby–baby SSR within the same nursery (regardless of group) to reach a median 33.3% at the end of school year (TA; versus median 17.9% at T15; Wilcoxon signed-rank test, n = 58, W = 86, P = 6.2 × 10−9; Fig. 3e), with a progressive increase occurring during the whole second term, as observed for the class that was densely sampled over such a period (group 1 of nursery A; Extended Data Fig. Although the baby–baby SSR decreased during the summer break (TB), it remained significantly higher compared with post-Christmas-break levels (T15; median 23.7% at TB versus 17.9% at T15; Wilcoxon signed-rank test, n = 31, W = 68, P = 2.0 × 10−4). These results highlight that social relations outside of the household and continued spatial proximity are key determinants of infant microbiome transmission and development at levels that are substantially higher than what was recently observed for adults8. The parent–baby SSR at T01 averaged 37.3% for mothers and 19.6% for fathers, consistent with available reports1,4,8,33,34,35. Such patterns persisted throughout the first term (Fig. The contributions of sibling strains to the baby was even higher (average SSR 56.2%; Fig. As expected, strain transmission between babies and individuals from different families remained negligible throughout the first term (Fig. 4a,b), a testament of the reliability of the strain-transmission-inference approach. a, SSR and average number of strains shared between pairs of babies and parents (at T01) from the same versus different families at each time point. In a and b, statistical significance asterisks refer to two-sided permutation tests for medians (Methods) adjusted for multiple comparisons for same family versus different family across each family member type, with ***P < 0.001. Exact P values for a–e are provided in Supplementary Table 11. In all box plots, box edges indicate the lower and upper quartiles, the centre line represents the median, and whiskers extend to the most extreme data point within 1.5× the IQR. b, Strain-sharing between pairs of babies and siblings (T01) from the same versus different families at each baby time point. c, Proportion of strains acquired from group versus family, and corresponding cumulative relative abundance (bottom). For each baby time point, comparisons were performed against past or contemporaneous samples of the family and the nursery group (Methods). Statistical significance asterisks refer only to the proportion of strains acquired from the same group versus the family. In c–e, the two-sided Mann–Whitney U-test was used, with *P < 0.05, **P < 0.01 and ***P < 0.001. d, Association between having a sibling and the number of strains acquired from the nursery group. e, Breakdown between acquisition of new SGBs typed at the strain level and strain replacement for the strains acquired from the nursery (top) and association between either means of strain acquisition and having a sibling (bottom). Statistical significance asterisks refer to the comparison between SGB acquisition from nursery for babies with versus without siblings. f, Number of strains either donated (dark green) or acquired (light green) by each baby over the first term (left y axis), and ratio of donated strains to acquired strains (dashed line; right y axis). To establish the relative contribution of strain transmission from the nursery with respect to strain transmission from the family, we computed, for each baby, the proportion of strains in the baby microbiome that were exclusively shared with, and hence putatively acquired from, either family members or other babies in the nursery group (Methods) and we refer to it as ‘proportion of strains acquired'. We found that the proportion of strains acquired from the nursery group—but not of strains acquired from the family—changed significantly over time. The proportion of strains acquired from family members fluctuated from an average of 24.0% per baby at T01 to 20.0% at the end of the first term of nursery (Wilcoxon signed-rank test, n = 25, W = 112, P = 0.18; Extended Data Fig. 7e), whereas those putatively acquired from the nursery group increased from an average of 6.5% to 28.4% at the end of the first term (Wilcoxon signed-rank test, n = 25, W = 0, P = 6.0 × 10−8; Extended Data Fig. 7e), significantly surpassing the proportion of strains acquired from the family (Mann–Whitney U-test, n = 52, U = 463, P = 0.023; Fig. This indicates that after only 3 months of nursery attendance, babies had proportionally more strains acquired from nursery peers than from their family. A similar trend was observed when quantifying the relative abundance of strains acquired from either the family or the nursery group (Fig. Family contribution slightly diminished over time (from an average 33.2% at T01 to 20.6% at T15; Wilcoxon signed-rank test, n = 25, W = 72, P = 0.014; Extended Data Fig. 7f) whereas the contribution from the nursery group greatly expanded (reaching an average of 39.6% at T15 from a starting 10.2%; Wilcoxon signed-rank test, n = 25, W = 18, P = 1.5 × 10−5; Extended Data Fig. Strains shared with both family and group also increased significantly (from average 0.9% to 8.5%; Wilcoxon signed-rank test, n = 25, W = 0, P = 4.4 × 10−4; Extended Data Fig. 7f), probably reflecting reciprocal transmission between family and nursery (Fig. Overall, this suggests that the nursery collectively contributes to a larger extent to the strain composition of the gut microbiome of babies than to that of the family by the end of the first term (39.6% versus 20.6% at T15; Mann–Whitney U-test, n = 52, U = 479, P = 0.01; Extended Data Fig. The extended longitudinal analysis of group 1 of nursery A revealed that the proportion of strains acquired from nursery peers continued to gradually increase during the second term (Extended Data Fig. Samples from all babies across nurseries at year-end (TA) confirmed comparable contributions of family and nursery to the baby (17.6% median proportion of strains acquired from nursery versus 15% from family; Mann–Whitney U-test, n = 19, U = 218, P = 0.29; Extended Data Fig. 8b) that non-significantly tended toward a greater family contribution after summer nursery closure, (8.7% median proportion of strains acquired from nursery versus 16.7% from family; Mann–Whitney U-test, n = 17, U = 122, P = 0.43; Extended Data Fig. Babies showed lower strain retention and higher strain replacement across the summer break (that is, between TA and TB) compared with adults, despite no differences in the carriage of SGBs typed at the strain level (Extended Data Fig. Interestingly, family-acquired strains were significantly more retained and less replaced in babies over the summer break than nursery-acquired strains (Wilcoxon signed-rank test, n = 11, W = 5, P = 0.019 and P = 0.022 respectively; Extended Data Fig. 8g,h), suggesting that continuous seeding linked to continued contact is a factor behind long-term colonization. Predicting a potential role of siblings in the transmission patterns, we found that at T01, babies showed a higher SSR with their siblings (average 52.3%) than with their fathers (24.9%; Mann–Whitney U-test, n = 36, U = 147, P = 0.026) as well as with their mothers, although non-significantly (46.1%; Mann–Whitney U-test, n = 36, U = 120, P = 0.47; Extended Data Fig. Of note, an average of 10.4 strains were shared exclusively with siblings at T01, whereas only 2.0 and 2.4 were shared exclusively with the mother or the father (Extended Data Fig. 8j), possibly reflecting closer intestinal ecology, physical interaction and development stage, which are probably some of the same factors leading to the higher nursery strain acquisition observed in our cohort. We further observed that having a sibling was associated with babies acquiring significantly fewer strains from their nursery group compared with babies without a sibling at T15 (Mann–Whitney U-test, n = 28, U = 117, P = 0.004; Fig. Although causality cannot be inferred, this might be linked to early acquisition from siblings ‘saturating' the overall strain acquisition potential, which would be in line with babies with a sibling having higher alpha-diversity (Fig. 1c) and acquiring fewer new SGBs than only-children (Fig. Notably though, although all babies both spread and acquired strains in the nursery, the ratio between acquired and donated strains varied widely between babies (Fig. We next assessed species-level transmissibility by counting the number of strain-sharing events for each SGB in our cohort over the total potential number of strain-sharing events (Methods). Microeukaryotic taxa were not found to be abundant enough in babies to try to infer transmission, with Blastocystis, the most common human gut microeukaryote36, identified in 9.18% of the samples but never in babies (Supplementary Table 12). Focusing thus on prokaryotic taxa, out of the 64 SGBs with highest transmissibility (henceforward ‘T') over all participant categories (Extended Data Fig. 9a and Supplementary Table 13), many known SGBs encompassed aerotolerant (S. gallolyticus, Rothia mucilaginosa and B. pseudocatenulatum) and spore-forming species (for example, Tyzzerella nexilis and Clostridium fessum). We also identified the spore-forming Clostridioides difficile among the most-transmissible SGBs between baby–baby pairs only (T = 0.38, prevalence in babies 24% and in adults 0%), in line with widespread carriage in asymptomatic babies37,38. Exceptions to this trend were prevalent non-sporulating human gut anaerobes (such as Blautia wexlerae and Faecalibacterium prausnitzii). SGB transmissibility correlated with SGB prevalence in both adults (Spearman's test, n = 461, ρ = 0.35, Padj = 9.8 × 10−14) and babies (Spearman's test, n = 461, ρ = 0.40, Padj = 1.2 × 10−17; Extended Data Fig. The highest transmissibility scores were highlighted for SGBs shared in baby–siblings pairs, namely, A. finegoldii, Bacteroides ovatus and Bacteroides caccae, the butyrate-producing Roseburia intestinalis and Agathobaculum butyriciproducens39,40, Bifidobacterium bifidum, and Bifidobacterium breve (all with T = 1; Extended Data Fig. B. caccae strains were also commonly transmitted between mothers and babies, alongside strains of two undescribed Clostridium spp., Phocaeicola vulgatus and the typically maternally derived B. bifidum and B. pseudocatenulatum32,41. Highly transmitted SGBs between fathers and babies included Clostridium sp. Finally, with the exception of the microaerophilic Streptococcus salivarius and S. wadsworthensis (T = 0.83 and T = 0.82, respectively), highly transmitted SGBs between mother–father pairs included multiple bifidobacteria and Blautia spp. Interestingly, many of the species are fibre-degrading specialists in the gut42,43,44, with known beneficial effects on the host45, indicating that within-family microbial transmission may hold a favourable potential for health-associated microbiome development. We looked further into our dataset to identify species differentially more transmitted baby-to-baby in the nursery setting compared with baby–mother and baby–father pairs (Supplementary Table 14). B. breve, a highly prevalent and health-promoting species in (breast-fed) babies6,46, was differentially more transmissible among baby pairs, compared with mother–baby pairs, as was the case also for Dorea formicigenerans, an age progression biomarker in babies47 (Extended Data Fig. infantis, a specialized gut colonizer of breast-fed babies46 with anti-inflammatory effects48, was detected exclusively in babies in our cohort (Methods and Extended Data Fig. 10c,d), with prevalence peaking at approximately 50% mid-term (T08), before declining (Extended Data Fig. 10e); its transmission was significantly higher than B. longum subsp. longum among baby–baby pairs (T = 85.3% versus T = 19.4%, respectively; Fisher's exact test, n = 142, P = 5 × 10−12), showing that the acquisition of B. longum subsp. infantis may specifically occur via interpersonal transmission among babies. In addition to the effect of having a sibling, age also significantly affected strain donation (increasing frequency in older babies, Spearman's test, n = 39, ρ = 0.43, P = 0.007), but not strain acquisition (n = 39, ρ = 0.24, P = 0.14; Extended Data Fig. Interestingly, potentially delayed microbial colonization at birth (owing to caesarean delivery or intrapartum antimicrobial prophylaxis) did not influence microbial strain acquisition for babies in the nursery (Extended Data Fig. 11d,e), in line with no T01 alpha-diversity differences (Extended Data Fig. Analysis of the influence of diet of babies on strain-sharing revealed that infants consuming milk at T01, particularly maternal milk, exhibited elevated albeit not statistically significant SSRs with their mothers at T01 (Extended Data Fig. Further exploration of dietary impacts on strain acquisition and donation patterns failed to identify significant associations (Extended Data Fig. 12c–k), suggesting an overall negligible impact of diet on interpersonal microbiome transmission; however, putative dietary effects on the establishment of specific strains in a recipient microbiome cannot be definitely excluded, given the limited granularity of our dietary data. Finally, we assessed the impact of antibiotic interventions on adult and babies' interpersonal transmission, exploiting the recorded antibiotic administration events that included amoxicillin alone (n = 7 events) and in combination with clavulanic acid (n = 13), betamethasone dipropionate (n = 6) and the macrolide azithromycin (n = 4), routine treatments for bronchitis, inflammatory skin conditions, and upper respiratory, ear and intestinal infections49,50. Antibiotic treatment (ATB) significantly reduced the absolute number of retained strains between consecutive time points in both adults (average 86.4 control pre–post versus 60.1 ATB pre–post; Mann–Whitney U-test, n = 74, U = 729, P = 5.1 × 10−4) and babies (average 24.3 control pre–post versus 14.1 ATB pre-post; Mann–Whitney U-test, n = 77, U = 1,169, P = 1.3 × 10−5; Extended Data Fig. Even for SGBs typed at the strain level that were present at both time points, the strain retention rate was also significantly diminished after treatments in adults (average 93.8% control pre–post versus 88.4% ATB pre–post; Mann–Whitney U-test, n = 74, U = 631, P = 0.028; Fig. 5a) and in babies (average 90.6% control pre–post versus 70.2% ATB pre–post; Mann–Whitney U-test, n = 76, U = 1,215, P = 2.9 × 10−7; Fig. 5a), but to a greater extent in the latter (ATB pre–post average strain retention rate adults versus babies: 88.4% versus 70.2%; Mann–Whitney U-test, n = 53, U = 466, P = 0.001; Fig. a, Strain retention rate (that is, the within-individual SSR) in adult and baby participants (n = 69 and n = 41, respectively, in a–c) who underwent antibiotic treatment (ATB pre–post) versus untreated controls (control pre–post). In all box plots, box edges indicate the lower and upper quartiles, the centre line represents the median, and whiskers extend to the most extreme data point within 1.5× the IQR. Statistical significant P values in all panels refer to two-sided Mann–Whitney U-tests. All other comparisons are non significant. b, Acquisition rate of SGBs typed at the strain level in babies and adults who underwent antibiotic treatment (ATB pre–post) versus untreated controls (control pre–post). c, Fraction of the strains present in pre samples replaced in post samples for babies and adults who underwent antibiotic treatment (ATB pre–post) versus untreated controls (control pre–post). Siblings and pets are excluded. Comparisons were performed between consecutive control pre–post and ATB pre–post time points (one per volunteer). After antibiotic use, the gut microbiomes of babies were replenished with new strains (Fig. This was driven by both the acquisition of new SGBs (average SGB acquisition rate 30.4% control pre–post versus 49.2% ATB pre–post; Mann–Whitney U-test, n = 59, U = 164, P = 2.9 × 10−4; Fig. 12m), and by the strain replacement within SGBs (average 2.1 replaced strains and 7.1% fraction of pre-ATB strains replaced control pre–post versus 3.9 and 13.6% ATB pre–post; Mann–Whitney U-test, n = 59, U = 209, P = 0.003 and P = 0.004; Fig. In contrast, adult microbiomes appeared to be less prone to new colonizations after antibiotic treatment via either means of strain acquisition (ATB pre–post average SGB acquisition rate adults versus babies: 34.2% versus 49.2%; Mann–Whitney U-test, n = 33, U = 74, P = 0.041, Fig. 5b; ATB pre–post average fraction of pre-ATB strains replaced adults versus babies: 7.5% versus 13.6%; Mann–Whitney U-test, n = 33, U = 69, P = 0.026; Fig. 5c), suggesting that although infant microbiomes tend to be more impacted by antibiotic therapy, their richness is also more easily recovered. Our longitudinal strain-resolved metagenomic framework revealed that the infant gut microbiome largely assembles, expands and modifies in the nursery via extensive baby–baby strain transmission, extending earlier work on family-to-baby transmission1,2,3,4,34,35 and overviews of the infant microbiome in nurseries14,51,52. After a few months of nursery attendance, the microbial strains acquired from peers in the same nursery group accounted for a larger proportion of the infant microbiome than those from the mother and—more generally—family members (Fig. 4c), who are known to exert the greatest influence on babies' microbiome in the first months of life. Contributions to the infant microbiome by family and nursery were not influenced by birth practices or feeding regimes (Extended Data Figs. 11d,e and 12a–k), and became comparable by the end of the second term, possibly indicative of strains being shared with both family and nursery. In addition to their already established effects in emotional and cognitive development53,54, social relations among peers in the nursery are thus a hub for microbial enrichment during infancy, particularly of key early-life gut colonizers such as B. longum subsp. infantis and B. breve6,46 (Extended Data Fig. Horizontal infant microbiome transmission does not occur only in nursery settings, as we found that baby–sibling strain-sharing surpasses transmission between parent–baby pairs (Extended Data Fig. 5g) and correlates with a later decrease in infant microbial acquisition in the nursery (Fig. Even pets might contribute strains to the babies but not to the adults (Extended Data Fig. 6d,e), and although limited and somewhat conflicting evidence has been produced on the effect of having a pet on human microbiomes55,56,57, larger studies specifically focused on strain transmission and medium-term retention should be promoted. Overall, our data further reinforce the role of horizontal intra-generational (and possibly inter-host species) over vertical inter-generational transmission not only in adults8 and nurseries (the main point of the present work) but also within a family context. In several cases, we observed very effective spread of a single strain within nurseries (Fig. Such diffusion patterns are akin to typical pathogenic outbreaks within closed communities58,59. However, although pathogenic spread typically elicits an acute immunological reaction and/or requires treatment, leading to somewhat rapid clearance after transmission, for gut microbiome members, colonization may be long-lasting as we reported in several cases in our cohort (Fig. 2b), even though it remains unsettled whether colonization persisted for many years after the end of nursery school. Moreover, further elucidation of the phenotypes linked to the propagation of fast-spreading strains may be highly relevant towards a better comprehension of the factors favouring the development of a healthy host–microbiome mutualism. Among the factors that may influence microbial transmission in babies, we found antibiotic usage to be the strongest one. Despite the infant microbiome being highly perturbed by antibiotic treatment during the first year of life, as previously reported60,61,62, it is also fast-recovering via extensive strain acquisition (Fig. 12m,n), consistent with antibiotic treatment before faecal microbiota transplantation increasing donor strain engraftment in adults63. However, we found strong evidence that the extent and the rate of post-antibiotic strain acquisition was substantially higher in babies compared with adults (Fig. 5), and this clearly reinforces the risks—but potentially also the opportunities—of infant antibiotic intake connected with a deep reprogramming of the structure of the infant microbiome induced by post-antibiotic strain acquisition. Whether the rapid acquisition of microbial diversity after antibiotic courses in babies is driven specifically by the high level of peer-to-peer interaction in the nursery environment should be investigated further, but it is reasonable to consider that prolonged isolation within the family of antibiotic-treated babies would result in a slower microbiome recovery and acquisition of fewer baby-specific microbial species. Methodologically, our strain-sharing pipeline models the genetics of the dominant strains of each species (SGBs) present in any given microbiome sample64 to enable identification of strain transmission events. Although recent surveys have pointed out the usual presence in the gut of a single strain of each species25, further advances in metagenomic strain-profiling tools could reveal the complexity of multiple coexisting conspecific strains and shed light on their role in influencing strain(s) transmission dynamics and long-term colonization in the gut microbiome. Overall, our results reveal the centrality of social factors in shaping the infant microbiome via inter-individual microbial transmission, thus rebalancing social interactions as key to building a healthy microbiome, beyond their epidemiological role in the spread of (opportunistic) pathogens. Continued efforts on this topic should be focused on investigating the transmission of further microbiome components such as phages, plasmids and operons, as well as on applying experimental tools to profile the microbial features favouring diverse modes of transmission. A total of 134 volunteers comprising babies (4–15 months old at nursery start, median 10 months, 18 male, 25 female) about to attend the first year of nursery school, their parents (29–50 years old, median 36 years old, 30 male, 39 female), siblings (2–21 years old, median 2 years old, 3 male, 4 female) and house pets (n = 5, 2 cats and 3 dogs), and educators (34–56 years old, median 38.5 years old, 10 female) were recruited and enroled across 3 nursery schools (here identified as A, B and C), each with 2 distinct classes, in the municipality of Trento (Italy) in June 2022. The classes within the same nursery shared few activities (that is, baby drop-off and pick-up) and spaces throughout the day, and were followed by different educators. The protocol of this study was approved by the Ethics Committee of the University of Trento (protocol number 2022-040) and by the Ethics Panel of the European Research Council Executive Agency after evaluation of the project (microTOUCH Grant agreement ID 101045015). Upon enrolment, volunteers were asked to provide informed consent and complete metadata questionnaires. Consent for participation of babies was obtained directly from parents. Date of birth, sex, anthropometric data (weight, height), and antibiotic treatment in the 3 months preceding the start of the study or supplemented during its course, in addition to information regarding putative contacts with other volunteers preceding the beginning of nursery, were collected for volunteers of all ages. Metadata specifically collected for babies included gestation length, mode of delivery and general diet at nursery admission (breast or formula milk feeding and weaning date of start). Adult participants were also required to provide information regarding past or ongoing chronic conditions and relative treatments, and putative maternal anti-Streptococcus B prophylaxis during birth. Diet metadata for babies and adults are detailed in the next section. In brief, most babies had begun weaning at T01 (weaned n = 38, not weaned n = 2, not available = 3) and received identical solid meals while in the nursery. The majority followed a mixed feeding approach during weaning, combining solid foods with any type of milk (mixed diet n = 24, exclusively solid food n = 14, not available = 5). Among those babies receiving milk supplementation, feeding types were relatively balanced (breast-fed n = 9, formula-fed n = 10, receiving both n = 5). Finally, adults detailed their long-term dietary habits via the compilation of the EPIC Food Frequency Questionnaire (FFQ). FFQs were used to calculate the healthy Plant-based Diet Index65. Quality and quantity of plant-based foods were derived from FFQs for a total of 18 food groups, and divided into quintiles and assigned positive or negative scores. Participants whose intake exceeded the highest quintile received a score of 5, whereas those below the lowest quintile received a score of 1. Healthy plant-based foods received positive scores, whereas less healthy or unhealthy plant-based and animal-based foods received a negative score. A final score was derived by summarizing the scores of each participant. Metadata were collected and utilized after pseudonymization of volunteers IDs. Sample collection began a week before the start of the first term of nursery (August 2022) and ended after the Christmas holidays (January 2023) for all volunteers. During the first 2 weeks the nursery organized a ‘settling-in phase', in which babies were gradually introduced to the nursery and attended it for about 3 hours per weekday. In the following weeks, babies attended the nursery for about 8 hours per weekday. Throughout the term length (about 14 weeks), stool samples of infant participants were collected weekly (from before nursery admission T01 to at the end of Christmas holidays T15) by the nursery staff or the researcher in the nursery from nappies stored at room temperature on the same day of use, using collection tubes for specimen collection containing 9 ml of DNA/RNA Shield buffer (Zymo). Sample collection was extended until the end of the second term of the year (about 30 weeks, ending July 2023) for all donors in group 1 of nursery A, including babies, parents, educators and pets, maintaining sampling time-point frequencies and modalities. Two follow-up time points were collected for all participants enroled, at the end of the year of nursery (July 2023, ‘TA') and at the end of the summer break (August/September 2023, ‘TB'). The samples collected were moved to the lab and DNA-extracted within 2 weeks of delivery. Samples collection of babies during summer or winter breaks time points together with those of siblings and pets were performed directly at home by the parents and stored at room temperature until the beginning of nursery (maximum 2 weeks later). All adult participants' samples were self-collected following detailed instructions, delivered to the lab and processed as previously. Educators donated monthly, whereas parents collected one additional sample halfway the study period, in addition to initial and final sample time points. After vortex homogenization, DNA was extracted using the DNeasy PowerSoil Pro Kit (Qiagen), following the directions of the Human Microbiome Project protocol66. Additional homogenized aliquots were stored at −20 °C. DNA was quantified using Qubit 2.0 fluorometer (Thermo Fisher Scientific). Sequencing libraries were prepared using the Nextera DNA Library Preparation Kit (Illumina), as described by the manufacturer's guidelines. The sequencing was performed on the Illumina NovaSeq 6000 platform following manufacturer's protocols. The sequencing depth was set at 15 Gbp. Stool samples sequences were pre-processed using the pipeline described at https://github.com/SegataLab/preprocessing. In brief, metagenomic reads were quality-controlled and reads of low quality (quality score <Q20), short reads (<75 bp) and reads with >2 ambiguous nucleotides were removed with Trim Galore (v0.6.6). Contaminant and host DNA was identified with Bowtie2 (v2.3.4.3)67 using the -sensitive-local parameter, allowing confident removal of the phiX 174 Illumina spike-in and human-associated reads (hg19/GRCh37 human genome release). Remaining high-quality reads were sorted and split to create standard forward, reverse and unpaired reads output files for each metagenome. Metagenomes with at least 1 Gbp were included in the analysis (n  = 1,021), whereas metagenomes with insufficient sequencing depth were excluded (n  =  5). Profiling at the resolution of SGBs was performed with MetaPhlAn (v4.1)16,68 using the vJun23_202307 markers database and using the –unclassified_estimation parameter (Supplementary Table 4). SGBs with <0.1% relative abundance in all stool samples were removed from taxonomic profiles for calculation of diversity indices. To reliably detect strain-sharing events, we augmented our dataset with oral samples from the same cohort not analysed in this study (n = 342) and additional samples from 16 public longitudinal cohorts. To do so, we queried the curatedMetagenomicData (v3.18)69 for stool samples sequenced at least at 1-Gbp depth from healthy westernized human individuals with at least 2 time points per individual and 3 such individuals per dataset. We went through the corresponding papers and excluded studies involving an intervention between the sampled time points. Thus we have included samples satisfying the above criteria from the following datasets: ShaoY_2019 (ref. In total, phylogenetic trees were built using data from 1,405 samples from this cohort and 4,322 samples from additional cohorts. For all the SGBs detected in our cohort, we queried MetaRefSGB, a microbial genomic database containing >156,000 isolate genomes and >952,000 metagenome-assembled genomes (MAGs) as of vJun23 (ref. 16), for isolate genomes or MAGs from food sources, and included them in the trees as references84. To build each tree, we included all the samples for which the SGB was detected by MetaPhlAn. SGBs detected in less than 20 samples from our cohort were discarded, leaving 1,363 SGBs. The strain-level phylogenetic trees were built for each SGB with StrainPhlAn (v4.1)16,68. For 1,107 SGBs, we were able to build the phylogenetic tree; the remaining 256 SGBs did not have a sufficient number of samples with enough marker genes with minimal coverage as reported by StrainPhlAn. We discarded trees built from alignments shorter than 1,000 nt (n = 111 SGBs discarded). In the phylogenetic trees with genomes from a food source, we considered the ANI on the marker genes (mutation rates in StrainPhlAn) and discarded samples closer than 99.85% ANI as probably coming from food. When more than 20% of the samples would be discarded, we dropped the SGB altogether (n = 7 SGBs discarded). We normalized distance distributions within each tree by dividing by the median distance. To define strain-sharing events, we calculated a threshold best separating the within-individual distribution from the across-individuals distribution of phylogenetic distances. For the within-individual distribution, pairs of samples from the same individual sampled maximum 6 months apart were considered using maximum one pair per individual. In case of ties, we maximized the higher estimated coverage of the two. For the across-individuals distribution, we pick pairs of samples coming from different datasets, one sample per individual maximizing the coverage. When there were less than 50 pairs in the across-individual distribution we discarded the SGB (n = 365 SGBs discarded). When there were less than 20 pairs in the within-individual distribution (n = 276 SGBs), we calculated the threshold as the 3rd percentile of the across-individual distribution, that is, setting the expected false discovery rate to 3%. When the within-individual distribution had at least 20 pairs (n = 309 SGBs), the threshold separating the distributions was calculated as maximizing Youden's index, unless the expected false discovery rate exceeded 5%, in which case we set the threshold as the 5th percentile of the across-individual distribution (n = 61 SGBs). After manual curation of the trees, including outlier branches removal, a further 112 SGBs were discarded. In total, we assessed strain-sharing for 512 SGBs. For each pair of samples, we called a strain-sharing event of an SGB when their phylogenetic distance in the corresponding tree was lower than the corresponding calculated threshold. For each sample, we considered the SGBs in which corresponding trees it was placed, that is, profiled by StrainPhlAn. Finally, we define the SSR as the number of strain-sharing events divided by the number of SGBs profiled at the strain level in both of them by StrainPhlAn. For pairs with less than five SGBs profiled in common, we set the SSR as undefined and such pairs were excluded from SSR analysis. Strain retention rate is defined as the within-individual SSR. The within-individual strain replacement rate is defined as the number of strains not retained longitudinally among retained SGBs over the number of retained SGBs. Samples found to be contaminated or mislabelled according to strain-sharing analysis (and validated with CrocoDeEL (v1.0.6)85) were removed post hoc (n = 8), reducing the dataset to 1,013 metagenomes. Samples collected during antibiotic treatment (n = 26) were excluded from all following analyses. Within-SGB strain heterogeneity (Extended Data Fig. 7a) was computed on a per-nursery basis as the number of strains of a given SGB present among babies at a given time point over the number of babies having the SGB at the same time point. A within-SGB strain heterogeneity of 1 indicates there is no strain-sharing among babies having the SGB (that is, they all have a different strain of the SGB). 3b) with the R packages ggraph (v2.2.1) and tidygraph (v1.3.1), where only nodes with degree >0 are shown. SGB transmissibility (Extended Data Fig. 9a) was computed as the number of individual pairs sharing the strain over the total number of potentially callable strain-sharing events involving the SGB (that is, the number of pairs of individuals in which the SGB was present and typable according to StrainPhlAn)8. When the SGB was not present among at least three pairs within a category, the transmissibility of the SGB within the category was set as undefined. Differential transmissibility between baby–baby and baby–mother or baby–father pairs was assessed for all SGBs having at least 10 baby–baby pairs sharing it, with application of a Fisher's exact test (including false discovery rate control) in case there were at least 10 baby–mother pairs or 10 baby–father pairs sharing the SGB; otherwise the number of SGB- and strain-sharing events were reported for each group without application of the test. To compare the contribution of familial and nursery strains to babies' microbiome composition, for each baby, we computed at each baby time point (T01 to T15) the number of strains shared either with any member of the family or with any other infant of the nursery group, disregarding strains shared with both (unless otherwise stated). This allowed us to compute the proportion of strains for a given baby microbiome that was putatively acquired from either the nursery or family (referred to in the text and figures as the ‘proportion of strains acquired'), as the strains exclusively shared with either one of the two groups of individuals. Considering that family members were less densely sampled than infants, we considered a maximum of three samples for each of the other babies in the nursery group (baseline T01, halftime T08 and final T15, when available, emulating the sampling timeline of parents), adding other babies and family samples to the longitudinal analysis considering the time in which they were sampled (that is, looking for strain-sharing only with samples collected in the past or contemporaneous to the target time point of the target baby). This probably explains the nonlinear increases of proportion of strains acquired from the nursery group at baby T08 and T15 observed (for example, Fig. Moreover, as a negative control for the considerably larger number of individuals in the nursery group (average n = 7) compared with the family (average n = 2), we also analysed the strain-sharing dynamics with a random nursery group from another nursery. MAGs were generated through a previously validated metagenomic assembly pipeline17, including assembly of contigs with MEGAHIT (v1.1.1)86, calculation of contigs coverage with Bowtie2 (v2.2.9)67, binning of contigs with MetaBAT2 (v2.12.1)87 and quality-checking of bins with CheckM (v1.1.3)88. Medium- and high-quality MAGs were identified following the criteria previously proposed89 and low-quality MAGs were discarded. The ANI between MAGs was computed using skani (v.0.2.1)90. 2a, CRISPR arrays were identified from MAGs using MinCED (v0.4.2, default parameters)91. CRISPR spacers and repeats were extracted from raw sequencing reads using Crass (v1.0.1, parameters -d 20 -D 55 -s 20 -S 55 --longDescription)92. Following the identification of 5 CRISPR arrays and 39 CRISPR spacers in this set of MAGs, we looked for the CRISPR spacers of this strain in the metagenomic reads of the whole dataset. Although single CRISPR spacers were found in the metagenomic reads of up to 16/19 samples containing the strain (Fig. To evaluate the transmissibility of distinct subspecies of B. longum SGB17248 in our dataset, we constructed a StrainPhlAn 4 phylogenetic tree of all 591 B. longum strains identified in our cohort, which revealed 2 distinct clusters (cluster_1 and cluster_2; Extended Data Fig. Cluster_1 consisted exclusively of strains present in babies and hence was hypothesized to belong to B. longum subsp. infantis, whereas cluster_2 contained strains from both babies and adults, possibly representing other subspecies of B. longum. To definitively assign these strains to subspecies, we succeeded in generating MAGs (as described above) for 262 of the 591 strains and then calculated the ANI between these MAGs and 15 reference genomes representing the 3 well-known B. longum subspecies93 (5 reference genomes each for subsp. The 24 MAGs from cluster_1 showed highest similarity to B. longum subsp. infantis reference genomes (median ANI 98.04%), compared with lower ANI values with subsp. Conversely, the 238 MAGs from cluster_2 showed the highest genomic similarity to B. longum subsp. longum reference genomes (median ANI 98.87%), with substantially lower similarity to subsp. The 591 B. longum strains as profiled with StrainPhlAn 4 were then assigned according to their genome-level assignment. The presence of Blastocystis in metagenomic samples was assessed using a previously validated computational workflow94. In brief, nine reference genomes for eight distinct Blastocystis subtypes (that is, subtype 1 (ST1) (ST1_LXWW01), ST2 (ST2_JZRJ01), ST3 (ST3_JZRK01), ST4 (GCF_000743755 and ST4_BT1_JZRL01), ST6 (ST6_JZRM01), ST8 (ST8_JZRN01) and ST9 (ST9_JZRO01)) were mapped against metagenomic reads with Bowtie2 (v2.5). Then, SAMtools (v1.19) and bedtools (v2.30) were used to compute the breadth of coverage of each genome. We reported a sample to be positive for a Blastocystis ST if the respective genome had a breadth of coverage of at least 10%. To further test how much potential problems with limit of detection in the metagenomic approach could influence strain transmission inference, we implemented an SGB-specific PCR assay for A. muciniphila (SGB9226) and applied it to the representative example depicted in Fig. We designed the primers (F-5′-TGACTGGACTCTATTGCCTGAAG-3′ and R-5′-GCCTTTCAATATGCCCTTCGTAC-3′; amplicon length 101 bp) to recognize the SGB9226-specific core gene (UniRef90_A0A2N8IRV1; identified by MetaPhlAn 4 (ref. 16)), using ConsensusPrime (v.1.0, with set consensus similarity 0.8 and consensus threshold 0.95)95 and Primer3 (v. 2.6.1; with set primer size 18–28 bp, optimal melting temperature 57–63 °C, GC content 40–60%, PRIMER_MAX_HAIRPIN_TH 24.0, PRIMER_INTERNAL_MAX_HAIRPIN_TH 24.0, PRIMER_MAX_END_STABILITA 9.0)96. Assay sensitivity was independently evaluated using a spike-in approach with DNA from the A. muciniphila type strain ATCC BAA-835 (from 1 M down to 1 single genome copy) into an A. muciniphila-negative faecal test sample. The assay achieved a limit of detection equivalent to a single genome copy of A. muciniphila (Extended Data Fig. PCRs were performed using GoTaq G2 Hot Start Green Master Mix (Promega) with 500 nM of each primer. The thermal cycling programme included an initial denaturation at 95 °C for 10 min, followed by 40 cycles of annealing at 62 °C. Positive bands were observed by electrophoresis on a 2% agarose gel. Statistical analyses were performed in Python (v3.10.12) using libraries scikit-bio (v0.5.9), scipy (v1.10.1) and statsmodels (v0.14.0). Cross-sectional comparisons between groups were performed using the Mann–Whitney U-test (two groups) or the Kruskal–Wallis test with post hoc Dunn tests (multiple groups) for independent observations. Cross-sectional dependent observations were compared with a permutation test for medians (or means, when comparing the number of shared strains), with P values calculated as the proportion of times (out of a 1,000 permutations) that the observed difference in the median between groups with shuffled labels is equal or more extreme than the observed in the correctly labelled data. Longitudinal comparisons between time points were performed using the Wilcoxon signed-rank test. Jaccard dissimilarity matrices were computed from taxonomic profiles and compositional differences between groups were evaluated using permutational multivariate analysis of variance. When appropriate, correction for multiple testing was applied using the Benjamini–Hochberg procedure (Padj), with significance defined as Padj < 0.05. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Shotgun metagenomic data for microTOUCH-baby are available at the NCBI SRA under accession number PRJNA1140720, with respective sample-wise metadata information available in Supplementary Table 2 and at https://doi.org/10.5281/zenodo.17663257 (ref. Accession numbers for sequencing data from additional longitudinal datasets can be found in the original publications. The human genome release used for host decontamination is available at NCBI RefSeq (GCF_000001405.13). All the software used in this study are available in the MetaPhlAn4 package (which includes StrainPhlAn 4 and the script for strain transmission inference), available at http://segatalab.cibio.unitn.it/tools/metaphlan with the open-source code at https://github.com/biobakery/MetaPhlAn. Ferretti, P. et al. Mother-to-infant microbial transmission from different body sites shapes the developing infant gut microbiome. Yang, B. et al. Development of gut microbiota and bifidobacterial communities of neonates in the first 6 weeks and their inheritance from mother. Yassour, M. et al. Strain-level analysis of mother-to-child bacterial transmission during the first few months of life. Dubois, L. et al. Paternal and induced gut microbiota seeding complement mother-to-infant transmission. Heidrich, V., Valles-Colomer, M. & Segata, N. Human microbiome acquisition and transmission. Stewart, C. J. et al. Temporal development of the gut microbiome in early childhood from the TEDDY study. Transmission of human-associated microbiota along family and social networks. Song, Z., Chen, L., Sun, S., Yang, G. & Yu, G. Unveiling the airborne microbial menace: novel insights into pathogenic bacteria and fungi in bioaerosols from nursery schools to universities. Reduction of acute respiratory infections in day-care by non-pharmaceutical interventions: a narrative review. Ball, T. M. et al. Siblings, day-care attendance, and the risk of asthma and wheezing during childhood. Côté, S. M. et al. Short- and long-term risk of infections as a function of group child care attendance: an 8-year population-based study. Kamper-Jørgensen, M., Wohlfahrt, J., Simonsen, J., Grønbaek, M. & Benn, C. S. Population-based study of the impact of childcare attendance on hospitalizations for acute respiratory infections. Gut microbiome development in early childhood is affected by day care attendance. Enav, H., Bäckhed, F. & Ley, R. E. The developing infant gut microbiome: a strain-level view. Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Truong, D. T., Tett, A., Pasolli, E., Huttenhower, C. & Segata, N. Microbial strain-level population structure and genetic diversity from metagenomes. Yatsunenko, T. et al. Human gut microbiome viewed across age and geography. Odamaki, T. et al. Age-related changes in gut microbiota composition from newborn to centenarian: a cross-sectional study. The developing airway and gut microbiota in early life is influenced by age of older siblings. Laursen, M. F. et al. Having older siblings is associated with gut microbiota development during early childhood. Karcher, N. et al. Genomic diversity and ecology of human-associated Akkermansia species in the gut microbiome revealed by extensive metagenomic assembly. Complete genome sequence of the bile-resistant pigment-producing anaerobe Alistipes finegoldii type strain (AHN2437T). Gut microbiota strain richness is species specific and affects engraftment. Rodriguez-R, L. M. et al. An ANI gap within bacterial species that advances the definitions of intra-species units. Comparative genomics reveals extensive intra-species genetic divergence of the prevalent gut commensal Ruminococcus gnavus. The Kitty Microbiome Project: defining the healthy fecal ‘core microbiome' in pet domestic cats. Gut microbiome of captive wolves is more similar to domestic dogs than wild wolves indicated by metagenomics study. Huang, Z., Pan, Z., Yang, R., Bi, Y. The canine gastrointestinal microbiota: early studies and research frontiers. Woelfel, S., Silva, M. S. & Stecher, B. Intestinal colonization resistance in the context of environmental, host, and microbial determinants. Shao, Y. et al. Stunted microbiota and opportunistic pathogen colonization in caesarean-section birth. Hildebrand, F. et al. Dispersal strategies shape persistence and evolution of human gut bacteria. Bogaert, D. et al. Mother-to-infant microbiota transmission and infant microbiota development across multiple body sites. Infant gut strain persistence is associated with maternal origin, phylogeny, and traits including surface adhesion and iron acquisition. Piperni, E. et al. Intestinal Blastocystis is linked to healthier diets and more favorable cardiometabolic outcomes in 56,989 individuals from 32 countries. Kubota, H. et al. Longitudinal investigation of carriage rates, counts, and genotypes of toxigenic Clostridium difficile in early infancy. Li, Z. et al. Clostridioides difficile infection in infants: a case report and literature review. Duncan, S. H., Louis, P. & Flint, H. J. Lactate-utilizing bacteria, isolated from human feces, that produce butyrate as a major fermentation product. Ahn, S. et al. Agathobaculum butyriciproducens gen. nov. sp. nov., a strict anaerobic, butyrate-producing gut bacterium isolated from human faeces and reclassification of Eubacterium desmolans as Agathobaculum desmolans comb. Detailed mapping of Bifidobacterium strain transmission from mother to infant via a dual culture-based and metagenomic approach. Beyond purified dietary fibre supplements: compositional variation between cell wall fibre from different plants influences human faecal microbiota activity and growth in vitro. Chung, W. S. F. et al. Modulation of the human gut microbiota by dietary fibres occurs at the species level. Ze, X., Duncan, S. H., Louis, P. & Flint, H. J. Ruminococcus bromii is a keystone species for the degradation of resistant starch in the human colon. Flint, H. J., Duncan, S. H., Scott, K. P. & Louis, P. Links between diet, gut microbiota composition and gut metabolism. The genome sequence of Bifidobacterium longum subsp. infantis reveals adaptations for milk utilization within the infant microbiome. Early life microbial succession in the gut follows common patterns in humans across the globe. Henrick, B. M. et al. Colonization by B. infantis EVC001 modulates enteric inflammation in exclusively breastfed infants. Tsalik, E. L. et al. Efficacy and safety of azithromycin versus placebo to treat lower respiratory tract infections associated with low procalcitonin: a randomised, placebo-controlled, double-blind, non-inferiority trial. Schouwenburg, S. et al. A pooled population pharmacokinetic study of oral and intravenous administration of clavulanic acid in neonates and infants: targeting effective beta-lactamase inhibition. Hermes, G. D. A., Eckermann, H. A., de Vos, W. M. & de Weerth, C. Does entry to center-based childcare affect gut microbial colonization in young infants? Thompson, A. L., Monteagudo-Mera, A., Cadenas, M. B., Lampl, M. L. & Azcarate-Peril, M. A. Milk- and solid-feeding practices and daycare attendance are associated with differences in bacterial diversity, predominant communities, and metabolic and immune function of the infant gut microbiome. The Ecology of Human Development: Experiments by Nature and Design (Harvard Univ. Ilyka, D., Johnson, M. H. & Lloyd-Fox, S. Infant social interactions and brain development: a systematic review. Kates, A. E. et al. Household pet ownership and the microbial diversity of the human gut microbiota. & Ying, L. Effects of cat ownership on the gut microbiota of owners. Exposure to household furry pets influences the gut microbiota of infants at 3–4 months following various birth scenarios. Badovinac, V. P. & Harty, J. T. Adaptive immunity and enhanced CD8+ T cell response to Listeria monocytogenes in the absence of perforin and IFN-γ. Wu, Y. et al. Epidemiological study of post-pandemic pediatric common respiratory pathogens using multiplex detection. Antibiotic exposure prevents acquisition of beneficial metabolic functions in the preterm infant gut microbiome. Effects of early-life antibiotics on the developing infant gut microbiome and resistome: a randomized trial. Uzan-Yulzari, A. et al. Neonatal antibiotic exposure impairs child growth during the first six years of life by perturbing intestinal microbial colonization. Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases. Global genetic diversity of human gut microbiome species is related to geographic location and host health. Satija, A. et al. Healthful and unhealthful plant-based diets and the risk of coronary heart disease in U.S. adults. The Human Microbiome Project Consortium. A framework for human microbiome research. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with bioBakery 3. eLife 10, e65088 (2021). Accessible, curated metagenomic data through ExperimentHub. Stability of the human faecal microbiome in a cohort of adult men. Schirmer, M. et al. Dynamics of metatranscription in the inflammatory bowel disease gut microbiome. Lloyd-Price, J. et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. Bäckhed, F. et al. Dynamics and stabilization of the human gut microbiome during the first year of life. Costea, P. I. et al. Subspecies in the global human gut microbiome. The dynamics of the human infant gut microbiome in development and in progression toward type 1 diabetes. Louis, S., Tappu, R.-M., Damms-Machado, A., Huson, D. H. & Bischoff, S. C. Characterization of the gut microbial community of obese patients following a weight-loss intervention using whole metagenome shotgun sequencing. B. et al. A novel Ruminococcus gnavus clade enriched in inflammatory bowel disease patients. Birth mode is associated with earliest strain-conferred gut microbiome functions and immunostimulatory potential. Identification and assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes. Integrated multi-omics of the human gut microbiome in a case study of familial type 1 diabetes. Chu, D. M. et al. Maturation of the infant microbiome community structure and function across multiple body sites and in relation to mode of delivery. Studying vertical microbiome transmission from mothers to infants by strain-level metagenomic profiling. Carlino, N. et al. Unexplored microbial diversity from 2,500 food metagenomes and links with the human microbiome. Goulet, L. et al. CroCoDeEL: accurate control-free detection of cross-sample contamination in metagenomic data. Li, D., Liu, C.-M., Luo, R., Sadakane, K. & Lam, T.-W. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. & Yu, Y. W. Fast and robust metagenomic sequence comparison through sparse chaining with skani. Bland, C. et al. CRISPR recognition tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats. Skennerton, C. T., Imelfort, M. & Tyson, G. W. Crass: identification and reconstruction of CRISPR from unassembled metagenomic data. Proposal to reclassify the three biotypes of Bifidobacterium longum as three subspecies: Bifidobacterium longum subsp. Large-scale comparative metagenomics of Blastocystis, a common member of the human gut microbiome. Collatz, M., Braun, S. D., Monecke, S. & Ehricht, R. ConsensusPrime — a bioinformatic pipeline for ideal consensus primer design. Untergasser, A. et al. Primer3—new capabilities and interfaces. Ricci, L., Heidrich, V., Punčochář, M. & Segata, N. Metadata files associated with the microTOUCH baby cohort. We thank all study participants for their commitment, and A. Zanetti, L. Zambaldi and R. Vit of the Childcare Service Office of the Trento Council (Trento, Italy) and all nursery coordinators and educators for their collaboration; the CIBIO Next Generation Sequencing Facility of the University of Trento (V. de Sanctis, R. Bertorelli, P. Cavallerio and C. Valentini) for sequencing the metagenomic libraries; and B. Servaes for support in designing figure icons in Figs. This work was supported by the European Research Council (ERC-CoG microTOUCH-101045015) to N.S. These authors contributed equally: Liviana Ricci, Vitor Heidrich, Michal Punčochář Department of Cellular, Computational and Integrative Biology, University of Trento, Trento, Italy Liviana Ricci, Vitor Heidrich, Michal Punčochář, Federica Armanini, Matteo Ciciani, Elisa Piperni, Charlotte Servais, Federica Pinto, Francesco Asnicar & Nicola Segata European Institute of Oncology, Scientific Institute for Research, Hospitalization and Healthcare, Milan, Italy Amir Nabinejad, Farnaz Fazaeli, Elisa Piperni & Nicola Segata Department of Medicine and Life Sciences, Universitat Pompeu Fabra, Barcelona, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar managed cohort recruitment, data and sample acquisition. performed DNA extraction and shotgun sequencing libraries preparation. All authors revised the paper and approved the final version for submission. The authors declare no competing interests. Nature thanks Martin Laursen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Distribution of unclassified reads percentage across all samples analyzed in the study (n = 1,013), as estimated using MetaPhlAn v4.1. (b) T01 alpha diversity represented as species-level genome bin (SGB) richness divided by participant type. In box plots, box edges indicate the lower and upper quartiles, the center line represents the median, and whiskers extend to the most extreme data point within 1.5× the interquartile range (IQR). Statistically significant P-values refer to two-sided Mann-Whitney U tests. All other pairwise comparisons are non significant. Heatmap shows SGBs with prevalence ≥30% at 0.1% relative abundance in either babies or adults. The SGBs are ordered based on the difference between the median relative abundance among babies and adults. (a) Ordination overview of the species-level microbiome composition of the microTOUCH-baby cohort (principal coordinate analysis on Jaccard dissimilarity, n  = 646), with samples coloured by SGB richness and shapes indicating participant types. (b) Association between SGB richness and first four PCoA ordination components. Statistical significance refers to the two-sided Spearman's test (n = 646). (a–b) a, Association between baby microbiome diversity at T01 (n = 36) and b, T15 (n = 27) and baby age before nursery start (T01). (c–d) c, Association between Jaccard dissimilarity at T01 (n = 491 baby-baby pairs) and d, T15 (n = 312 baby-baby pairs) and age difference (months), in babies. (e) Association between Jaccard dissimilarity during the first term of nursery attendance (T01-T15) and baby age (months) at T01 (n = 26). (f) Association between species-level genome bin (SGB) richness before nursery start (T01) and healthy Plant-based Diet Index (hPDI), as an indicator of diet quality, in adult volunteers (n = 69). In (a-f) statistical significance refers to the two-sided Spearman's test. (a-b) a, Association between alpha diversity in babies at T01 and maternal intrapartum antibiotic prophylaxis against Streptococcus B and b, mode of delivery. In box plots (a-b, f-g), box edges represent the lower and upper quartiles, the center line indicates the median, and whiskers extend to the most extreme data point within 1.5× the IQR. Statistical significance refers to two-sided Mann-Whitney U tests, with P ≥ 0.05, ns for comparisons in a-b. (c-d) Ordination overview of the species-level microbiome composition of babies at T01 (principal coordinate analysis on Jaccard dissimilarity, n  = 37) in relation with birth practices. Statistical significance refers to the two-sided PERMANOVA test. Samples are coloured according to c, administration of maternal intrapartum antibiotic prophylaxis and d, mode of delivery. (e) Change in total species-level genome bin (SGB) richness of all babies in the three nurseries throughout the first term of nursery. Average SGB richness is shown by the black dotted line. (f) Mother-baby strain-sharing rates in Valles-Colomer et al. (V-C), 2023, stratified in the [1–15] and [4–15] months age ranges (median SSR = 48% and 45%), and in this study (median SSR = 50%). Only pairs of samples with at least 10 SGBs in common were included. Statistical significance refers to two-sided Mann-Whitney U tests, V-C 2023 [1–15] months vs this study (U = 474.0), and V-C 2023 [4–15] months vs this study (U = 198.0), with P ≥ 0.05, ns, for both comparisons. (g) Strain-sharing rate between family members pairs within the same vs different families before nursery start (T01). Statistically significant P-values refer to two-sided permutation test for medians (Methods) adjusted for multiple comparisons, with n indicating the number of subject-subject pairs. 2a) using a SGB-specific PCR assay; a, Sensitivity of the SGB9226-specific PCR assay, assessed by testing a SGB9226-negative human fecal DNA spiked with seven ten-fold dilutions of genomic DNA from A. muciniphila ATCC BAA-835, corresponding to an estimated 106 to 1 genome copies (“D1” to “D7”). Controls included the human fecal matrix alone (“Spike-in Matrix”) and a no-template control (NTC). b, Application of the SGB9226-specific PCR to fecal samples in Fig. Sample IDs include the volunteer type and family number, while per volunteer longitudinal samples are identified by timepoint. Relative abundances according to MetaPhlAn 4 (“Rel. Abun., %”) and sample IDs are colored dark blue (positive) or grey (negative) based on strain identification according to StrainPhlAn (Fig. Genomic DNA from A. muciniphila ATCC BAA-835 is included as a positive control, alongside a NTC. Participant types are identified by shape (educator, cross; baby, circle; mother, diamond) containing participant identifiers (with the initial identifying the participant type followed by specific family number). Familial relations are highlighted by same-color filling. Each circle represents a sample collected from participants depicted, with color filling indicating the identity of the strain of Alistipes finegoldii SGB2301 detected in the sample (except gray, used to indicate the SGB was not detected/typable) and arrows indicating the most likely transmission event. (d-e) Strain-sharing between pets (n = 5) and human hosts within the same family (turquoise) and across different ones (gray). d, The average total number of strains shared between participant pairs across contemporaneous samples are reported above the connecting lines; in brackets are indicated the number of pet-human pairs with at least one strain shared over the total number of pet-human pairs. Statistical significance according to a two-sided Fisher's exact test is depicted for the same family vs different family pet-baby comparison. All other comparisons are non significant. e, Average number shared strains between pet and human across contemporaneous samples, in different vs same family. Statistical significance P-values refer to two-sided Mann-Whitney U tests, with n indicating the number of pet-human pairs. All other comparisons are non significant. (a) Change in average number of strains per SGB typed at the strain level present in babies separated by nursery at each timepoint. Species-level genome bin (SGB) retention rate during the first term in different participant types. (b) The metric is defined as the Jaccard similarity between samples from initial and final timepoints of the same subject. Statistically significant P-values refer to two-sided Mann-Whitney U tests. All other pairwise comparisons are non significant. In box plots (b, e-f), box edges indicate the lower and upper quartiles, the center line represents the median, and whiskers extend to the most extreme data point within 1.5× the IQR. (c) Association between overall strain replacement rates and group age among non-baby participants. Statistical significance refers to the two-sided Spearman's test (n = 68). (d) Longitudinal average baby-baby strain-sharing rate (with each baby-baby observation in the background) and average number of strains shared during the entire first year of nursery, for group 1 of nursery A. Only pairs with at least 5 species-level genome bins (SGBs) in common were included. common SGBs (right y-axis) subplot, the vertical lines report the standard deviation, with n indicating the number of baby-baby pairs. (e) Percentage of strains acquired from either their family, nursery group and a random different group from another nursery (as a control) at T01 and T15. (f) Relative abundance (ra) of strains acquired from either family, nursery group and both, at T01 and T15. Statistically significant P-values refer to two-sided Mann-Whitney U tests, except for the ra family vs ra group comparison, for which the two-sided Wilcoxon signed-rank test was used. All other comparisons are non significant. (a-b) a, Proportion of strains acquired from group vs family, corresponding average number of strains acquired and total relative abundance, during the entire first year of nursery, for group 1 of nursery a only and b, at the end of the second trimester (TA) and at the end of summer break (TB), for all participants in the study. In (a-b), for each baby timepoint, comparisons were performed against past/contemporaneous samples of the family and the nursery group (Methods). Statistically significant P-values refer to a two-sided Mann-Whitney U test for the proportion of strains acquired from the same group vs the family. Comparisons at other timepoints are non significant. In all box plots (a-f, i-j), box edges indicate the lower and upper quartiles, the center line represents the median, and whiskers expand the IQR. (c) Proportion of SGBs typed at strain-level that were lost after the summer break (TB) over those present at year-end (TA), in adult and baby participants. In c–f, significant P-values refer to a two-sided Mann-Whitney U test. All other comparisons are non significant. (d) Proportion of SGBs typed at strain-level that were newly acquired during the summer break (TB) over the total present at TB, in adult and baby participants. (e) Proportion of strains retained after summer break (TB) over the total found at year-end (TA), in adult and baby participants. (f) Proportion of strains replaced at TB over the total strains present at TA, in adult and baby participants. (g–h) g, Proportion of baby strains acquired from either family or nursery that were retained or h, replaced by different conspecific strains after summer break (TB) over the total strains found at year-end (TA), in baby participants. In g–h, significant P-values refer to two-sided Wilcoxon signed-rank tests. (i) Strain-sharing rate between babies and other family members at T01. (j) Average number of strains shared between pairs of babies and siblings or one parent or more than one family member at the time. In i-j, statistically significant P-values refer to two-sided Mann-Whitney U tests. All other pairwise comparisons are non significant. (a) The 64 species-level genome bins (SGBs) typed at the strain level with highest transmissibility scores (Methods) in the nursery and among family members, divided per participant type pairs and ordered by their overall transmissibility score (avg. across all participant type pairs). The prevalence among adults (parents and educators) and babies at T01 are shown. SGBs included had a transmissibility score of ≥ 0.3 (computed over at least 10 pairs containing the SGB) for at least one participant type pair. The transmissibility score was not computed for participant type pairs having less than 3 pairs containing the SGB. Symbols in squares highlight significantly higher (Methods) baby-baby transmissibility over baby-adult pairs (with exact P-values provided in the legend and in Supplementary Table 14). “Untested higher transmissibility” indicates a transmissibility difference (transmissibility baby-baby - transmissibility baby-adult) ≥ 0.3 for cases in which not enough SGB-sharing pairs were identified among baby-adult pairs for reliable statistical comparison (Methods), and was also highlighted using symbols (legend). (b) Association between SGB transmissibility between pairs of different participant types and SGB prevalence (in adults, babies, or the ratio of prevalence baby/adults). “Population” includes all unrelated adults in the microTOUCH-baby study cohort, while “Overall” reports the average of all category pairs in which the SGB transmissibility could be computed, including unrelated volunteers. For example, higher transmissibility of prevalent SGBs was captured among unrelated volunteers, such as adults (Spearman's test, n = 427 SGBs, ρ = 0.24, Padj = 4.6e-6) and babies in the nursery (Spearman's test, n = 129, ρ = 0.21, Padj = 0.031). (c) Association between SGB transmissibility between pairs of different participant types (Baby-Baby, n = 129 SGBs; Baby-Father, n = 80; Baby-Mother, n = 97) and SGB prevalence ratio (Baby/Adult). P-values and correlation coefficients (ρ) according to a two-sided Spearman's test are shown at the bottom of the figure. (a–b) a, Phylogenetic tree of Bifidobacterium breve SGB17247 and b, Dorea formicigenerans SGB4575 that were found to be differentially more transmitted baby-to-baby in the nursery setting, compared to parent-baby pairs. In phylogenetic trees (a-c), participants are identified by shape and colour in the inner ring, while nursery group affiliation by colour in the outer ring, as indicated in the legend. (c) Phylogenetic tree of Bifidobacterium longum SGB17248. Colour coding and symbols as in a-b. (d) Intra-subspecies average nucleotide identity (%) used to define subspecies-level clusters within B. longum. (e) Prevalence of B. longum subsp. infantis in babies during the first term of nursery. (a–c) a, Association between number of strain donations b, acquisitions and c, ratio of donations over acquisitions and baby age (months) at T01. Statistical significance refers to the two-sided Spearman's test (n = 39). (d-e) d, Association between the number of strains shared with the nursery group and mode of delivery (C-section or vaginal) and e, maternal intrapartum anti-Streptococcus B prophylaxis. In both box plots, box edges show the lower and upper quartiles, the center line indicates the median, and whiskers extend to the most extreme data point within 1.5× the IQR. Statistical significance asterisks refer to two-sided Mann-Whitney U tests, with P < 0.05, *. All other comparisons are non significant. (a–b) a, Mother-baby strain-sharing rate in babies drinking vs not drinking any type of milk and b, in babies receiving maternal milk vs formula milk at T01. In box plots (a-n), box edges represent the lower and upper quartiles, the center line indicates the median, and whiskers extend to the most extreme data point within 1.5× the IQR. In (a-n), statistically significant P-values refer to two-sided Mann-Whitney U tests. All other comparisons are non significant. (c-e) Association between number of strain donations and c, weaning status d, supplementation of milk and e, milk type at T01. (f-h) Association between number of strain acquisitions and f, weaning status g, supplementation of milk and h, milk type at T01. (i–k) i, Association between ratio of strain donations over acquisitions and weaning status, j, supplementation of milk and k, milk type at T01. (l-n) l, Absolute number of retained strains, m, absolute number of newly acquired SGBs (i.e. acquisition of strains assigned to previously undetected SGBs) and n, absolute number of strains replaced in adult and baby participants (n = 69 and 41) that underwent antibiotic treatment (ATB pre-post) vs untreated controls (Ctrl pre-post). Siblings and pets are excluded. Comparisons were performed between consecutive Ctrl pre-post and ATB pre-post timepoints (one per volunteer). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Ricci, L., Heidrich, V., Punčochář, M. et al. Baby-to-baby strain transmission shapes the developing gut microbiome. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09971-3'>Quantum spin resonance in engineered proteins for multimodal sensing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 16:08:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Sensing technologies that exploit quantum phenomena for measurement are finding increasing applications across materials, physical and biological sciences1,2,3,4,5,6,7. Until recently, biological candidates for quantum sensors were limited to in vitro systems, had poor sensitivity and were prone to light-induced degradation. These limitations impeded practical biotechnological applications, and high-throughput study that would facilitate their engineering and optimization. We recently developed a class of magneto-sensitive fluorescent proteins including MagLOV, which overcomes many of these challenges8. Here we show that through directed evolution, it is possible to engineer these proteins to alter the properties of their response to magnetic fields and radio frequencies. We find that MagLOV exhibits optically detected magnetic resonance in living bacterial cells at room temperature, at sufficiently high signal-to-noise for single-cell detection. These effects are explained through the radical-pair mechanism, which involves the protein backbone and a bound flavin cofactor. Using optically detected magnetic resonance and fluorescence magnetic-field effects, we explore a range of applications, including spatial localization of fluorescence signals using gradient fields (that is, magnetic resonance imaging using a genetically encoded probe), sensing of the molecular microenvironment, multiplexing of bio-imaging and lock-in detection, mitigating typical biological imaging challenges such as light scattering and autofluorescence. Taken together, our results represent a suite of sensing modalities for engineered biological systems, based on and designed around understanding the quantum-mechanical properties of magneto-sensitive fluorescent proteins. Coupling electromagnetic fields with biological processes through fluorescence has revolutionized quantitative biology9. Meanwhile, quantum-sensing tools (that is, those for which function arises from electron-spin- or nuclear-spin-dependent processes) have been developed for their unique advantages in biological applications, but have until recently been limited to realizations using non-biological probes1,2,3 or measurements under ex vivo conditions4,5,6,10. We previously reported the development of a library of magneto-responsive fluorescent protein (MFP) variants derived from the LOV2 domain (broadly termed MagLOV), which exhibit fluorescence signals with large magnetic-field effects (MFEs)8 (Fig. With this work, we demonstrate that at room temperature, we can detect optically detected magnetic resonance (ODMR)11,12 in living cells expressing these fluorescent proteins, including at the single-cell level. The ODMR signature implies a quantum system with properties and dynamics influenced by the local environment, opening up a broad range of possibilities for cellular biosensing1. The ODMR arises from electron spin resonance (ESR) that we propose originates from a spin-correlated radical pair (SCRP)11,13 (Fig. 1b) involving the LOV2 domain non-covalently bound flavin cofactor chromophore. This theory is based on previous evidence14 and supported by the MFE, ODMR and spectroscopic studies we present. a, Structure of AsLOV2 PDB 2V1A58 with MagLOV 2 mutations highlighted. Spin transitions driven by radio-frequency (RF) fields in the presence of a static magnetic field are optically detected via fluorescence measurements. b, Simplified photocycle diagram in the case of a large external magnetic field. The radical pair is born in a triplet state \(| {\rm{T}}\rangle \) with spin projections \(| {{\rm{T}}}_{0}\rangle \), \(| {{\rm{T}}}_{+}\rangle \) and \(| {{\rm{T}}}_{-}\rangle \) (refs. 20,21) and undergoes field-dependent singlet–triplet interconversion to the singlet state, \(| {\rm{S}}\rangle \), driven by nuclear-electron spin–spin interactions. c, Microscope measurement of a single cell expressing MagLOV 2 showing an MFE of about 50%. For MFE measurements, the magnetic field was switched between 0 mT and 10 mT, here with a period of 20 seconds. The intensity over time is integrated over pixels covering the single cell, with a background photobleaching trendline removed. d, Data from a single cell expressing MagLOV 2 shows an ODMR signal with about 10% contrast. The static field B0 is about 21.6 mT. The blue line (shading) is the mean (standard deviation) of all single-cell data in a field of view (about 1,000 cells). Inset: microscope image cropped to a single cell expressing MagLOV 2. e, The static magnetic field B0 was varied by adjusting the magnet's position, and the ODMR spectra recorded. The blue line shows the theoretical prediction for the resonance frequency of an electron spin with gyromagnetic ratio \({\overline{\gamma }}_{e}=28\,{\rm{MHz}}\,{{\rm{mT}}}^{-1}\) or ge = 2.00, with the shaded region representing uncertainty (±0.25 mT) in the magnetic-field strength (as determined by a Hall probe) at the sample position. Both MFE and ODMR signals are straightforward to detect in cells on a standard wide-field fluorescence microscope, supporting further development and application of this discovery. Beyond the ease of detecting MFPs' magnetic resonance via emission, MFPs are advantageous over other candidate spin sensors for biological uses because they can be expressed directly in the host organism, allowing direct coupling and regulation by biological processes, and because their performance can be engineered genetically, such as through rational design or directed evolution. This engineerability is demonstrated through a selection approach previously reported8 and here used to generate protein variants specialized for sensing applications. We demonstrate applications of MFPs as reporters that can be used for lock-in signal amplification in noisy measurement environments (as often encountered in biological applications15), and to enable signal multiplexing by engineering variants with differing dynamic responses. We also show that the MagLOV MFE is attenuated by interaction with magnetic resonance imaging (MRI) contrast agents, with a dose-dependent effect consistent with spin relaxation, implying MagLOV's ability to sense the surrounding environment. Finally, we realize applications to spatial imaging; because the ODMR resonance condition depends on the static magnetic field at the location of the protein, it is possible to use gradient fields (as in MRI) to determine the spatial distribution of MFPs with scattering-independent measurements of a sample's fluorescence. We demonstrate this by building a fluorescence MRI instrument based on a small-animal MRI coil with a one-dimensional magnetic gradient, which we apply to simultaneously localize the depth position of multiple bands of bacterial cells embedded in a three-dimensional volume. Ultimately, this work represents a proof of principle for MFPs and their applications, which may develop into a paradigm of quantum-based tools for biological sensing, measurement and actuation. Reaction yield detected magnetic resonance (RYDMR; a form of ODMR) is both a diagnostic test for the existence of a proposed SCRP16 and, owing to its relative simplicity, an effective measurement modality for performing readout from quantum-sensing devices in biological and materials applications1,2,17,18,19. ODMR studies the spin dynamics of such systems through the application of oscillating radio-frequency magnetic fields, B1, resonant with spin transition energies, and facilitates optical readouts through light emission or absorption. SCRPs are transient reaction intermediates, often generated by (photoinitiated) rapid electron transfer from a donor (in biological systems, often an aromatic amino acid) to an acceptor—here flavin mononucleotide (FMN), which serves as the field-sensitive fluorophore in our system. As electron transfer occurs under conservation of total spin angular momentum, the total spin of the radical pair is defined by that of its molecular precursor (either a singlet \(| {\rm{S}}\rangle \) (S = 0) or a triplet state \(| {\rm{T}}\rangle \) (S = 1), where S is the total spin quantum number). If the radicals in the pair are weakly coupled, the spin system is created in a superposition of the uncoupled states. Consequently coherent interconversion occurs between the \(| {\rm{S}}\rangle \) and \(| {\rm{T}}\rangle \) states, driven by the interactions between electron and nuclear spins (such as 1H and 14N; Fig. At zero and low static magnetic fields, this interconversion occurs rapidly involving all four states, but at higher fields, singlet–triplet mixing is restricted to \(| {\rm{S}}\rangle \) and \(| {{\rm{T}}}_{0}\rangle \) with \(| {{\rm{T}}}_{\pm }\rangle \) energetically isolated from \(| {\rm{S}}\rangle \) and \(| {{\rm{T}}}_{0}\rangle \) by Zeeman splitting20,21. Importantly, singlet and triplet radical pairs have different fates: whereas the singlet pair is able to recombine to yield the ground-state donor and acceptor, the triplet cannot do so and instead forms other forward photoproducts (for example, by protonation or deprotonation), a route also open to the singlet pair. Under continuous illumination, the impact of the field on the singlet–triplet interconversion can be detected conveniently if either donor or acceptor or both form fluorescent excited states. In this case, a drop in fluorescence with an applied strong (about 10 mT) magnetic field (B0) is expected for a triplet-born pair as mixing into the recombining singlet state is impeded; hence, the system exhibits an MFE. Meanwhile, application of an additional resonant radio-frequency field (B1) would reconnect \(| {{\rm{T}}}_{0}\rangle \) with \(| {{\rm{T}}}_{\pm }\rangle \), leading to an increase in fluorescence intensity, measured as ODMR. This hypothesis motivated consideration of MFE and ODMR measurements as approaches to confirm that the radical pair mechanism of MagLOV is triplet-born20. Previous studies have similarly explained MFEs arising from interactions between a flavin cofactor and protein in terms of the SCRP mechanism22,23,24,25,26,27,28, with the most prominent example being cryptochromes implicated in avian magnetoreception29,30,31,32. Furthermore, radical-pair intermediates are known to form in the LOV2 domain Avena sativa phototropin 1 (AsLOV2) variant C450A (the precursor protein to MagLOV)33, as well as in related LOV domains34,35,36. Parallel with our results, ODMR has recently been reported in purified enhanced yellow fluorescent protein (EYFP) at room temperature, and in mammalian cells at cryogenic temperature10, and subsequent preprints have reported ODMR in purified protein solutions of Drosophila melanogaster cryptochrome (DmCry), mScarlet-flavin and MagLOV at room temperature, as well as mScarlet-flavin in Caenorhabditis elegans at room temperature32,37. First, we measured the MagLOV MFE (see example trace in Fig. 1c) using a custom-built fluorescence microscopy platform, confirming that the variant MagLOV 2 exhibits a large MFE of \(\Delta {\mathcal{I}}/{{\mathcal{I}}}_{\mathrm{off}}=-50 \% \), where \({{\mathcal{I}}}_{{\rm{on}},{\rm{off}}}\) is the fluorescence intensity when the electromagnet is on or off, and \(\Delta {\mathcal{I}}={{\mathcal{I}}}_{{\rm{off}}}-{{\mathcal{I}}}_{{\rm{on}}}\). 1d, we show the ODMR resonance of a single cell and the ODMR resonance averaged over many (about 1,000) cells in a field of view. As expected, the signal-to-noise is significantly improved by averaging over many cells; however, it is also possible to extract an ODMR signal from a single cell, with an ODMR contrast of 10% (Fig. The remarkable per-cell magnetic sensitivity of η0 = 26 μT Hz−1/2 (Supplementary Note 1) is afforded by a combination of optical detection and the high spin polarization of the radical-pair system1. Next, we recorded ODMR spectra at various static (B0) fields by adjusting the z position of the static magnet (Fig. The central ODMR resonance follows the expected ESR relationship \({f}_{\mathrm{RF}}={\bar{\gamma }}_{e}{B}_{0}\) (with \({\bar{\gamma }}_{e}\) the electron gyromagnetic ratio), confirming that the radio-frequency field is driving spin transitions of a spin-1/2 electron. Finally, we performed additional control experiments confirming the ODMR signal's source, verifying against negative controls that an ODMR signal is present only when the MagLOV protein is present (Supplementary Note 2). The AsLOV2 domain has been widely used as a starting point for engineering optogenetic and other light-dependent protein functionalities38. For MFPs, depending on a target application (such as lock-in signal detection, multiplexing or sensing), one could choose to optimize for metrics including MFE size, rate of MFE saturation, ODMR contrast, ODMR saturation rate and others. Here we demonstrate this potential by performing selection to improve MFE contrast and saturation rate, generating variants specialized for applications demonstrated later in our work. Starting from ancestor variant AsLOV2 C450A (refs. 8,39), we used directed evolution to create variants of MFPs (summarized in Supplementary Note 3). This engineering process involved successive rounds of mutagenesis (introducing all single amino acid changes to a given variant), followed by screening of samples from this variant library to select for increased MFE magnitude, eventually yielding MagLOV 2. To demonstrate the possibility of selecting on another metric using the same methodology, we further engineered MagLOV 2 by selecting for maximization of saturation rate (rather than magnitude) of MFE, producing ‘MagLOV 2 fast'. We chose four variants to characterize in detail, which was done both using single-cell microscopy (Fig. 2a) and measurement of bulk cell suspensions (Supplementary Note 4). a, MFP variants were engineered by mutagenesis and directed evolution; from AsLOV2 R2 to MagLOV 2 selection was performed to increase the MFE magnitude at saturation, whereas MagLOV 2 fast was selected for increased rate (that is, reduced time constant τ when the MFE with time t is modelled as e−t/τ (ref. 25), shown in red fit). In these measurements, a magnetic field of B0 = 10 mT was switched on and off with a period of 20 seconds. The traces shown are the average (and standard deviation, shaded) over multiple periods. As a result, part of the variability results from the background photobleaching curve fit which attempts to match the true background curve (see Supplementary Note 9 for details). b, A similar experiment was performed using ODMR on-resonance, with a constantly applied static field of B0 = 21.6 mT and corresponding resonant field B1 frequency of ωRF = 604 MHz switched on and off with a 20-second period, averaged as above. Both sets of experiments were performed in identical imaging set-ups with an illumination intensity of about 800 mW cm−2 at 450 nm. Note that B0 in a is half that of B0 in b, explaining why the ODMR contrast is in some instances larger than the MFE. The observed differences in MFE can be interpreted based on past work investigating flavin magnetic-field-sensitive photochemistry: MFE enhancement kinetics (that is, time to MFE saturation) are determined by the ratio between the rates that donor and acceptor free radicals return to the ground state25. With the mutations introduced in MagLOV 2 fast, the time constant of the response is decreased, which may indicate that the acceptor return rate increases relatively to the donor. Interestingly, we observe the ODMR contrast and rates of each variant differ significantly (Fig. 2b), but not necessarily in simple correlation with MFE magnitude. This raises the possibility of engineering orthogonal fluorescence signatures, expanding again the number of tags available for multiplexing. For instance, with further engineering the total might be (number of emission colours) × (number of resolvable MFE signatures) × (number of resolvable ODMR signatures). Adopting the SCRP model prompts consideration of the electron donor and acceptor identities. Both previous studies33,35 and our spectral data (Fig. For all variants of MFPs expressed in cells, we find that the wavelength-resolved fluorescence intensity modulated by the applied magnetic field (\(\Delta {\mathcal{I}}\)) (Fig. 3a,b) matches the FMN emission spectrum39,40, supporting that both MFE and ODMR are detected on the flavin emission. The excitation spectrum shows vibrational fine structure (Fig. 3c) and is in excellent agreement with the dark-state absorption spectrum of AsLOV2 C450A (ref. 41), confirming that the emission originates from bound FMN. Furthermore, this corroborates control experiments (Supplementary Note 2) that show that observed ODMR signatures are not a result of cellular autofluorescence. The absorption spectrum of purified MagLOV 2 fast, before and after continuous irradiation with blue light, is shown in Fig. 3d (the full temporal evolution is provided in Supplementary Note 4). The slow formation of the stable radical FMNH• after several minutes of illumination is characterized by the appearance of a broad band featuring 2 peaks centred around 575 nm and 615 nm, as observed for AsLOV C450A, and accompanied by the expected decrease in ground-state absorption, centred around 450 nm (ref. At later times, most of the flavin is converted to the fully reduced form, FMNH−, resulting in a rise in absorption around 325 nm, although some remains present as FMN and FMNH•. The blueshift in MagLOV-bound FMN absorption relative to AsLOV C450A, evident both in the excitation and absorption spectra, may indicate a change in polarity of the flavin binding pocket. The observed photostability of MagLOV correlates with the slow formation of FMNH• on a timescale of minutes in these conditions. In comparison, related flavoproteins, such as cryptochrome, are reduced to the semiquinone state within seconds, even under much weaker irradiation intensities42. Unlike the recently reported MFE in mScarlet3 and FMN mixtures, which rely on a bimolecular reaction between the excited-state mScarlet3 and a fully reduced flavin in solution28, MagLOV requires no pre-illumination or additives for an MFE to develop as the FMN is non-covalently bound. a, Normalized emission spectra with 450-nm excitation acquired for cell suspensions in PBS buffer. The spectra have been smoothed with a moving average filter of 1-nm bandwidth. b, Wavelength dependence of magnetic-field-induced change of emission intensity where \(\Delta {\mathcal{I}}={\mathcal{I}}({B}_{0}=10\,{\rm{mT}})-{\mathcal{I}}({B}_{0}=0\,{\rm{mT}})\). It is noted that here we display the absolute intensity change as division by \({\mathcal{I}}({B}_{0}=0\,{\rm{mT}})\) would obscure the wavelength dependence. A moving average filter of 1-nm bandwidth was applied to the spectra. c, Normalized excitation spectra for 510-nm emission for bulk MagLOV cell suspensions in phosphate-buffered saline (PBS) buffer. Vibrational fine structure (multiple peaks) in the S0 → S1 and S0 → S2 bands (where Sn is the nth excited singlet state) centred at about 450 nm and about 350 nm, respectively, indicates that the emitting flavin is bound. d, Ultraviolet–visible absorption spectrum of purified MagLOV 2 fast at different times after the onset of blue LED illumination. Inset: a reduced wavelength range with the characteristic absorption of FMNH•. Literature reference spectra are shown as dashed lines41,59,60. Following photoexcitation (kExc), the excited singlet flavin (1FMN*) can either emit a photon (kF) or undergo intersystem crossing (ISC) to the excited triplet state (3FMN*). The primary radical pair (RP1) is formed by electron transfer (ET) from a nearby donor, and can undergo singlet–triplet interconversion, which is altered in the presence of an applied magnetic field (B). Only the overall singlet RP can undergo back-electron transfer (BET) to reform the ground state, whereas either RP can form secondary (perhaps spin uncorrelated) radicals (RP2) through protonation and/or deprotonation reactions (kH/Dep). These long-lived secondary radicals return to the ground through slow redox reactions (kFMNH, kD). The formation of FMNH• from the excited triplet state, 3FMN*, probably proceeds via initial formation of FMN•− in an SCRP on a nanosecond timescale as in AsLOV C450A (ref. 33), followed by slow protonation. Alternatively, FMNH• could be formed by proton-coupled electron transfer. A proposed photoscheme is given in Fig. 3e, and a model based on this photoscheme is simulated in Supplementary Note 5, which successfully fits both the experimental MFE and ODMR data. In cryptochrome, the SCRP is formed by a cascade of electron transfers along a tryptophan tetrad30; however, a single donor aromatic amino acid can be sufficient, as demonstrated by the magnetosensitivity of cryptochrome-mimicking flavomaquettes43,44 and FMN bound inside the bovine serum albumin protein45. Regarding the donor species, single-point mutations in AsLOV2 C450A leading to quenching of the emissive NMR signal suggest W491 as the electron donor46, which was corroborated by isotopic labelling of Trp residues47. However, given the extent of mutations in the variants studied here, we cannot confirm that W491 is still the counter-radical. For example, in the structurally related iLOV-Q489, derived from the Arabidopsis thaliana phototropin-2 (AtPhot2) LOV2 domain, transient absorption spectra revealed that a neutral tryptophan radical, Trp•, is formed in conjunction with FMNH•, and photoinduced flavin reduction in single-point mutations of selected tyrosine and tryptophan residues suggested that several amino acids might be involved in SCRP formation48. Our library of MFP variants shows differences in the rate and magnitude of response across both MFE and ODMR characterization (Fig. Where such differences can be engineered orthogonally between variants, they open the possibility of using libraries of MFP reporters as a multiplexing tool to extract several signals from a single measurement modality (Fig. As a demonstration, we characterized two cell populations expressing different MFP variants, for which distributions of MFE saturation timescales fit to single cells show strong separation (Fig. This enabled population decomposition when we applied a classification algorithm to a mixed population of about 2,000 cells (Fig. The classifier was trained on MFE traces normalized by amplitude (on a per-cell basis), meaning that it utilizes only the relative shape of the curves and not the magnitude of the MFE. Therefore classification is robust to scaling or offsets in the absolute brightness of the signal, as might be caused by scattering or autofluorescence (which pose practical limits on many sensing applications as described in Supplementary Note 6). Future application of this subpopulation labelling technique would benefit from engineering of variants with greater variation in dynamics such that the separation of histograms (as in Fig. 4b,c) is significantly greater than each population's intracellular variability. a, MagLOV variants can be used to label cell populations, which can be identified when mixed based on differing MFE responses. b, Exponential curves with timescale parameter τ were fit to the MFE of each cell in each field of view. Here populations are measured separately, illustrating that MagLOV 2 has a greater MFE saturation timescale than AsLOV R5. c, Populations were mixed in an equal ratio and a classifier (Methods) trained on the time-series data in b was used to classify the mixture into two subpopulations. d, Schematic of the microfluidic chip, composed of single-cell-wide trenches (vertical channels). Cells were engineered to weakly express MagLOV and co-express mCherry for ground-truth identification, and mixed with another cell population expressing only EGFP. Cells are classified based on MFE response, demonstrating the possibility of identifying MagLOV reporters mixed with other, non-magnetic responsive fluorescent reporters in the same spectral range, and under conditions where MagLOV produces only a small signal. e, Cropped view of single trenches, with cells circled in red as identified by a cell-segmentation algorithm. f, Time series of the 450-nm illumination fluorescence for the individual cells depicted in e over time, as a magnetic field B0 = 10 mT is switched on and off (red line). Traces for each cell in the trench (grey) and the average over all the cells in the trench (black) are shown. g, Confusion scatter plots for classifying whether cells express MagLOV or EGFP based on magnetic response, taking presence of mCherry fluorescence (red highlights) as ground-truth control and using the MFE lock-in value as the predictor. The balanced accuracy by cells and by trenches is 0.99. h, The standard deviation σ of the lock-in values in g is calculated between cells in each of the trenches (blue histogram), and between all cells in all trenches. Using a microfluidic set-up (in which populations of five to eight clonal cells are confined in individual trenches; Supplementary Note 7), we further investigated the sources of intracellular variability of MFP variants, and demonstrate the possibility of lock-in detection in weak signal environments. With MFE-based lock-in detection45, cells with minimal MagLOV expression could be identified distinctly from cells expressing enhanced green fluorescent protein (EGFP)49 with a balanced accuracy of about 0.99 (using a second fluorescence reporter, mCherry, as ground truth), with accuracy improving when averaging over a trench compared with distinguishing single cells (Fig. This approach allows variability to be attributed to inter-clonal or intra-clonal sources; we observe that mean intra-trench variability (quantified by standard deviation over cells in one trench) is approximately half that of the variability over all MagLOV-positive cells (Fig. This suggests that approximately one-quarter of the total variance arises from intra-clonal noise sources (for example, phenotypic variability over two or three generations, camera and accompanying measurement noise), with inter-clonal sources (longer-term phenotypic variability, variation in local environment) contributing the remaining three-quarters. Methods for the spatial localization of fluorescence signals in biological samples such as cell cultures and tissue samples are of significant interest for both diagnostics and treatment development50,51. However, techniques based on localization using fluorescence, for example, fluorescence-modulated tomography, are challenging owing to the inherent scattering and absorbing nature of tissue, and the requirement to localize the fluorescence via detailed modelling and inversion of the optical signal52. As such, we sought to explore whether MagLOV could be used as a fluorescent marker localized by optically detected MRI. First, we tested localizing MagLOV in the wide-field microscope set-up, using a permanent magnet to vary the resonance condition across the field of view (Fig. The sequence of images acquired during a radio-frequency B1 frequency sweep was integrated over cross-sections in which the B0 field is approximately constant, yielding an image (Fig. 5b) where the frequency of peak response (y axis) denotes the position in space along the z axis, which varies across a 0.5-mm field of view as anticipated for a field gradient of 1 mT mm−1. a, Schematic of the wide-field microscopy set-up for demonstrating localization of cells in a two-dimensional plane. The permanent magnet creates an approximately linear gradient field B0 along the z axis, perpendicular to the radio-frequency B1 field (which rotates at the Larmor frequency around z) generated by the radio-frequency coil. The frequency of B1 is scanned while the entire field of view is imaged. b, Subsequently, the images are divided into regions (red highlight in a) and integrated over that region. The integrated brightness versus frequency forms a one-dimensional spatial map—in this case, the sample is present in the entire field of view, thus we see a diagonal line, shifting by about 14 MHz over the 0.5-mm field of view as anticipated for the 1 mT mm−1 field gradient. c,d, Schematic of the custom-built MRI set-up illustrating the optical illumination and collection paths (c) and the magnetic-field gradient inside the resonant MRI coil (d). e,f, We embedded cells expressing MagLOV 2 fast (chosen for its ODMR contrast, fast saturation and low overall brightness to make detection challenging) into a polydimethylsiloxane (PDMS) sample at two different positions along the coil axis separately (e) and simultaneously (f). Lock-in ODMR detection was used to locate the samples along the coil axis. The red and blue curves are raw measurements, and the grey shaded regions are after processing with a deconvolution algorithm (which uses the known ESR linewidth from Fig. 1 but makes no assumption about the number or location of peaks). The location of the samples is identifiable on their own, and resolvable via deconvolution together. Against a ground-truth separation of 7.5 mm, deconvolved individual samples had a calculated peak separation of 6.6 mm and the combined sample 6.1 mm. Using the individual sample data to calibrate the measurement yields a combined-sample distance estimate of approximately 6.9 mm. Next, we converted a preclinical MRI 28-mm-diameter ‘birdcage' radio-frequency coil, used for creating a spatially highly homogeneous B1 radio-frequency field at 500 MHz, into an optically detected fluorescence MRI instrument via integration of a fibre-coupled illumination system and imaging using a photodiode (Fig. 5c,d and Supplementary Note 8). It is noted that the photodiode is in effect a ‘single-pixel' detector, meaning that it collects no spatial information and directional scattering of light would cause no reduction in information of final signal (apart from a possible decrease in absolute brightness). As the static B0 field is swept, the radio-frequency field B1 is switched on and off such that the ODMR contrast can be measured via lock-in detection. We found that good localization could be achieved when isolating single samples at different positions (Fig. 5e), and despite an increase in noise, deconvolution of the signal enabled two samples at different depths to be simultaneously localized (Fig. 5f) to within about 0.6 mm of a calibrated ground truth. We note that the generation of homogeneous radio-frequency fields at about 500 MHz within living systems (including humans) is an active area of research with known working designs for different species and anatomies53. In this way, our approach forms an alternative method for spatially resolved and scattering-insensitive sensing of genetically encodable fluorescent proteins. We next sought to determine whether MagLOV could function as a quantum sensor of its local environment. Although FMN is bound within the protein scaffold, we hypothesized that it might exhibit magneto-optical properties that depend on nearby paramagnetic species, analogous to nitrogen-vacancy centres in nanodiamond-based sensing approaches2,54,55. To test this, we diluted purified MagLOV 2 fast samples to 3.7 μM in solutions containing the paramagnetic contrast agent gadolinium (total spin S = 7/2; Gd3+ chelated as the MRI contrast agent gadobutrol). We anticipated that these freely diffusing paramagnetic species would act like diffusing point dipoles, modulate the dipolar interactions of the radical pair, and lead to enhanced relaxation and characteristic attenuation of the MFE contrast. This hypothesis was confirmed by our measurements (Fig. 6), which show that, despite the constrained cofactor geometry, MagLOV exhibits a clear dose-dependent reduction in MFE signal with increasing paramagnetic ion impurity concentration. We note that there was no visual change in the microscopy images, nor any gadobutrol concentration-dependent difference in absolute brightness of the sample, indicating that the chelated gadolinium did not cause protein denaturation or aggregation, or FMN dissociation. This result implies that MFPs could be used as an in situ, in vivo quantum sensor, which, analogous to other quantum-sensing modalities, opens up possible applications as a sensor of the cellular microenvironment. a, The MFE contrast of purified MagLOV 2 fast is measured at varying concentrations of the paramagnetic contrast agent gadobutrol, demonstrating a reciprocal dependence on concentration, consistent with spin relaxation and indicating that MagLOV is sensitive to its surrounding spin environment. For each gadobutrol concentration, the contrast was measured at three spatially separated fields of view. Insets are cartoon depictions of the solution with Gd (left) and without (right). Red arrows illustrate the local magnetic field generated by Gd; black arrows indicate rotational motion. b, The MFE time trace is shown for three points on the dose dependence curve of a. In general, we observed significantly lower MFEs for purified protein compared with in-cell measurements (see Supplementary Note 1 for further data). Directed evolution, enabled by straightforward fluorescence screening, has proved to be a powerful technique to engineer proteins exhibiting magneto-sensitive responses. The advent of stable, highly responsive magneto-sensitive proteins represents a paradigm shift; transitioning from quantum biological systems studied primarily for scientific interest towards engineerable tools with potential for widespread application. Previously, existing natural and engineered proteins (typically designed as model representatives of the cryptochrome) showed comparatively small responses to magnetic fields, required sophisticated experimental apparatus for study, did not exhibit measurable MFEs in living cells, were prone to rapid light-induced degradation, and were therefore unsuitable for biotechnological applications or high-throughput set-ups required for directed evolution22,27,43. These challenges are simultaneously overcome by MagLOV. Furthermore, compared with other candidates for quantum biological sensing, two unique advantages of a protein-based system are: (1) that it is configurable, that is, significant engineering improvements can be made (relatively simply) by changing the DNA encoding the protein; and (2) that it can be endogenously expressed, enabling coupling of its expression to diverse genetic or chemical signals. MFPs therefore are the best of both worlds—enabling sensitive quantum measurements while also being highly amenable to engineering and cellular integration. As we demonstrate, this system unlocks a host of measurement applications. First, modulating MagLOV fluorescence by applying a time-varying magnetic field enables multiplexing to expand the number of fluorescent reporters that can be distinguished in a single experiment. Meanwhile, lock-in could enable fluorescent protein measurements to be performed where previously not possible owing to poor signal quality, for example, if only small quantities of a fluorescent marker can be produced, or in tissue measurements where autofluorescence and scattering are limiting factors15,45,56,57. Second, we demonstrate that using ODMR it is possible to spatially localize fluorescence signals in a three-dimensional volume, utilizing the fact that resonance occurs only when the required conditions are met by (orthogonally controllable and tissue-penetrating) radio-frequency and magnetic fields. Finally, magnetic resonance sensing can be used to determine the presence of molecular species creating local magnetic noise in our protein's environment. This could be used to measure the presence of free radicals or paramagnetic metalloproteins, both critical to a number of physiological processes55. Although the properties of the MFPs we generated are superior to previously studied proteins that exhibit MFEs, their optimization is by no means complete. Much like fluorescent proteins, we expect that MFPs may be engineered to make general improvements, such as to solubility, photostability, spectral response and quantum yield, as well as further enhancing their MFE and ODMR properties. There also remains significant opportunity for mechanistic investigation utilizing the wide array of techniques previously applied to biological systems exhibiting MFEs and ESR22,31,34. Crucially, mechanistic understanding and high-throughput bioengineering (similar to and more advanced than we demonstrate here) can go hand in hand—for instance, enabling the creation of rational design tools that can optimize MFE and ODMR properties for a specific application. Finally, we hope that the development of MFPs can serve as the starting point for a class of magnetically controlled biological actuators, whereby application of a local magnetic field can be be coupled to downstream cellular effects—such a technology would be of significant biomedical and biotechnological utility. T7 Express Escherichia coli (New England Biolabs, C2566) were used for MFE and ODMR experiments in Figs. E. coli MG1655 cells were used for multiplexing and mother machine lock-in experiments in Fig. 4, owing to its effectiveness as a heterologous gene expression strain, and past optimization of mother machine chip loading protocols for this host. MG1655 was also used for Fig. NEB Dh5α E. coli was used for plasmid production and genetic engineering steps, but not experimental data collection. TB auto-induction medium (Formedium, AIMTB0260) was used for growth of liquid cultures of T7 Express strains. LB medium (Formedium, LBX0102) was used for growth of MG1655 strains in liquid cultures and agar plates (Formedium, AGR10). Antibiotic stocks were prepared as follows: 100 mg ml−1 carbenicillin (Formedium, CAR0025) dissolved in 50% (v/v) ethanol and 20 mg ml−1 chloramphenicol (Fisher, 10368030) dissolved in 100% (v/v) ethanol; stocks were diluted 1,000× for experiments. Whole plasmid sequencing was performed by Plasmidsaurus using Oxford Nanopore Technology. Derivative plasmids of those generated by directed evolution (for example, MagLOV + mCherry) were constructed using the EcoFlex kit (Addgene kit 1000000080) following standard EcoFlex protocols61. Level 1 assemblies were performed using NEB BsaI-HFv2, NEB T4 DNA Ligase Reaction Buffer (B0202S) and Thermo Scientific T4 DNA Ligase (EL0011). Level 2 assemblies were performed using NEBridge Golden Gate Assembly Kit BsmBI-v2 and NEB T4 DNA Ligase Reaction Buffer (B0202S). pTU1 was used as a negative control plasmid (Supplementary Note 2); it is part of the EcoFlex kit. pRSET-AsLOV_R2, pRSET-MagLOV, pRSET-MagLOV-2 and pRSET-MagLOV-2_R11-f plasmids (used for Figs. 1–3) contain MFP variants created via directed evolution (described below) under control of T7 promoters. It is noted that despite the T7 promoter being chemically inducible in its host strain, experiments were performed without induction (that is, transcription was owing to leaky expression). pVS-01-03_EGFP expressing EGFP was used for microfluidic experiments. It was assembled solely from EcoFlex parts (pTU1-B-RFP, pBP-J23108, pBP-pET-RBS, pBP-ORF-eGFP and pBP-BBa-B0012). pVS-02-04_mCherry+MagLOV was used for microfluidic experiments. EcoFlex Golden Gate assembly was used to create a plasmid that expresses both MagLOV and mCherry. In a level 1 reaction, pVS-01-01_mCherry and pVS-01-02_MagLOV were assembled using the following bioparts from the EcoFlex kit. The BP-01_MagLOV sequence with overlaps was gained via PCR using NEB Q5 polymerase with pGA-01-01 as template using primers MagLOV_EcoFlex_FWD and MagLOV_EcoFlex_REV. Both plasmids were combined in a subsequent Level 2 EcoFlex reaction with the pTU2-a backbone, leading to pVS-02-04_mCherry+MagLOV. The predicted translation rate is 250. pRH-01-17, pGA-01-47 and pGA-01-49 were used for multiplexing experiments and spatial localization. The coding sequences (CDSs) for AsLOV2 R5, MagLOV 2 and MagLOV 2 fast, respectively, were codon optimized and the ribosome binding site (RBS) designed to maximize constitutive expression using the CDS Calculator and RBS Calculator tools of ref. The sequence was synthesized as a gene fragment (Twist Bioscience) and assembled into a level 1 plasmid as above, again using the J23119 promoter. MFP expression levels were estimated for constructs using ref. For pVS-02-04, an unoptimized codon sequence and RBS were used, resulting in a predicted translation rate in its expression strain (E. coli MG1655) of 261. For pRSET constructs, the predicted translation rate in expression strains (E. coli NEB T7) is 50 × 103. For pRH-01-17, pGA-01-47 and pGA-01-49, the optimized codon sequence and RBS yielded a predicted translation rate in E. coli MG1655 of 106; as such this represents a 20× increase in predicted rate over the pRSET strains and a 4,000× increase in predicted translation over pVS-02-04. It is noted that these predictions also do not account for the dual expression in pVS-02-04 of mCherry. We previously reported the directed evolution of MFPs resulting in MagLOV8. Directed evolution of MFP variants was initiated with AsLOV2 C450A39, a protein originally isolated from the common oat Avena sativa. The mutagenesis protocol was based on methods developed previously63. In particular, we used 142 PCR reactions with NNK semi-random primer pairs (supplied by Eton Bioscience) to make all single amino acid mutants of AsLOV2 (404–546) C450A at all locations, which produces a library containing 2,982 protein variants, although (owing to the pooled nature of the screen) it is likely that not all variants are present for each round. This library was transformed into E. coli strain BL21(DE3) and spread across about ten LB-agar plates for screening. After transformation, plates were left for 2 days at room temperature (25 °C) as this was observed to lead to stronger magnetoresponses. Screening used the same fluorescence photography system described in ref. 63, with addition of an electromagnet (KK-P80/10, Kaka Electric) below the sample, which was turned on/off every 15 seconds using an Arduino Uno while acquiring successive images. After each screen, images were processed to identify the single colony (across all plates) with the largest fractional change in fluorescence between the magnet on/off conditions. The selected colony was picked (manually) from the corresponding plate, and used as the basis for the next round of mutagenesis and selection. Subsequent mutagenesis rounds used the same primers to avoid re-making the primer library with each mutation (which we hypothesize selects against multiple consecutive nearby mutations), except before rounds leading to AsLOV R5, MagLOV R7 and MagLOV 2, at which point primers in the library that overlapped mutated sites were updated to the current variant's sequence. In total, 11 rounds of mutagenesis were undertaken, all selecting for amplitude of MFE with the exception of round 11 (MagLOV 2 fast) where mutants from round 10 were selected based on fast response time (quantified as the mutant with the largest contrast measured in the first 100 ms following magnetic-field change). After each round of mutagenesis, selected variants were sequenced using the Sanger method (supplied by Quintara Biosciences). The variants measured in this paper and some intermediaries are provided in Supplementary Note 3. Plasmids expressing EGFP, MFP variants or negative control plasmids expressing only antibiotic selection markers were used to transform E. coli NEB T7 via heatshock at 42 °C for 45 seconds. Colonies were grown overnight at 37 °C on Agar plates, then a single colony was picked, resuspended in 1 ml LB media, spread onto a plate, and grown at 37 °C for 24 hours followed by 24 hours at room temperature to form a lawn of cells. Cells from this lawn were then resuspended in PBS buffer. Both MFE and ODMR imaging were performed using a custom-built wide-field epifluorescence microscope (Supplementary Notes 10 and 11). From cells suspended in PBS buffer, an approximately 1-μl droplet was confined between two glass coverslips (SLS MIC2162) atop a stripline antenna printed circuit board (Supplementary Note 12). The antenna printed circuit board was placed inverted on the microscope stage and either an electromagnet (for MFE) or a permanent magnet (for ODMR) was positioned above. The antenna printed circuit board was used to mount the MFE experiments (despite not delivering any B1 field) to that ensure lighting conditions were consistent with those in ODMR experiments. For MFE experiments, the electromagnet supplied a static field of 0 mT or 10 mT at the sample. For ODMR experiments, the permanent magnet supplied a static field (B0 ≈ 20 mT) in the z direction (Supplementary Note 13), perpendicular to the radio-frequency field (B1 ≈ 0.2 mT) supplied by the stripline antenna (Supplementary Note 14). It is noted that the imaging step and exposure times (Supplementary Note 11) were maintained across experiments unless otherwise stated, whereas light-emitting-diode power levels (Supplementary Notes 9 and 15) were varied between samples to account for differing expression levels of MFPs. Data processing was performed using Python (v3.11.11), SciPy (v1.15.1)64, NumPy (v.126.4)65, scikit-learn (v1.6.1)66 and scikit-image (v0.20.0)66. We define the MFE by \(\Delta {\mathcal{I}}/{{\mathcal{I}}}_{\mathrm{off}}\), where \({\mathcal{I}}\) is the fluorescence intensity, \({{\mathcal{I}}}_{\mathrm{off}}\) is the fluorescence intensity immediately before switching the magnet on, and \(\Delta {\mathcal{I}}={\mathcal{I}}-{{\mathcal{I}}}_{\mathrm{off}}\). Similarly, for ODMR measurements, the detrended signal is defined to be \(\Delta {\mathcal{I}}/{{\mathcal{I}}}_{\mathrm{bg}}\), where \({{\mathcal{I}}}_{\mathrm{bg}}\) is a background curve fit to the data as described in Supplementary Note 9, and \(\Delta {\mathcal{I}}={\mathcal{I}}-{{\mathcal{I}}}_{\mathrm{bg}}\). Wavelength-resolved MFE measurements (Fig. 3 and Supplementary Note 4) were performed in bulk using a home-built cuvette-based fluorescence spectrometer described in ref. In brief, a 450-nm laser diode was used to excite the sample, which was housed inside custom-built Helmholtz coils. The emission was collected through a lens pair and dispersed by a spectrograph (Andor Holospec) onto a charge-coupled-device array (Andor iDus420). Cells were resuspended in PBS buffer at an optical density of about 0.3 and placed in a quartz fluorescence cuvette (Hellma). Similarly to the wide-field measurements, a magnetic field was switched on and off with field strength 10 mT and a period of 20 seconds (10 seconds on, 10 seconds off). The sample was illuminated at 450 nm (Oxxius LBX-450), at roughly 1 kW m−2, and the emission was filtered using a 458-nm longpass filter (RazorEdge ultrasteep). Excitation spectra in Fig. 3c and Supplementary Note 4 were recorded using an Edinburgh Instruments FS5 Spectrofluorometer, using a xenon lamp as an excitation source. 3a) and time-resolved emission spectra (Supplementary Note 4) were recorded on the same instrument, but using an HPL450 (450 nm) for excitation. It is noted that this instrument set-up suffers from laser excitation artefacts, which create the small peaks near 550 nm and 650 nm. To demonstrate the potential of using MFE for multiplexing in fluorescence microscopy, we first cloned strains of AsLOV R5 (one of the variants evolved between AsLOV R2 and MagLOV; Supplementary Note 3) and MagLOV 2 into E. coli MG1655. Cells were grown in liquid culture from −80 °C freezer stocks, then resuspended in PBS and made up to equal concentration as measured by optical density (OD600). Individually, and as a 1:1 mixture, monolayers of cells were sandwiched between a glass slide and cover for MFE imaging. We initially characterized each variant in isolation, measuring fluorescence traces for approximately 2,000 cells in a field of view (Fig. The same processing was performed as in Fig. 2 but to individual cells, yielding a value of τ (the MFE saturation timescale) for each cell, which allows measurement of each sample's intrapopulation variability (Fig. To perform the population decomposition, we trained a machine-learning classifier (XGBoost67) on the dynamic data (that is, fluorescence versus time) used to generate Fig. Before training, and for classification, the MFE response (\(\Delta {\mathcal{I}}/{{\mathcal{I}}}_{\mathrm{off}}\)) curves of each cell were normalized to range from 0 to 1. Without further training, this classifier was used to classify a combined population of cells (Fig. 4b) mixed with 1:1 ratio (determined by OD600). The ratio of the two variants by classification was R5:ML2 = 0.9:1, which probably differs from the anticipated 1:1 ratio owing to both classifier accuracy and typical measurement errors expected from the use of OD600 to quantify cell counts68. Cells expressing EGFP49 and cells co-expressing mCherry69 with very weak MagLOV expression (expression level predicted62 at about 0.025% of that in multiplexing experiments) were grown in liquid culture, then mixed and loaded into a microfluidic ‘mother machine' chip consisting of two rows of evenly spaced trenches fed by a central channel bringing fresh media into the system and carrying excess cells out. After a few hours of growth, each trench was filled only with cells whose ancestor is the cell at the closed end of the trench (that is, they may be considered clonal; Fig. The cells were imaged using the same wide-field fluorescence microscopy set-up as described previously. Microfluidic manufacturing and set-up techniques are described in Supplementary Note 7. Algorithms for cell segmentation and image processing and quantification are described in Supplementary Note 16. Quantitative lock-in classification results are provided Supplementary Note 17. 4g, the classifications are shown including the decision boundaries based on the control (565-nm fluorescence) and based on the lock-in value. The graphs can be interpreted as confusion matrices: top-left quadrant are false negatives, top-right quadrant are true positives, bottom-left quadrant are true negatives, and bottom-right quadrant are false positives. The fluorescence MRI instrument (described in detail in Supplementary Note 8) used a permanent neodymium rare-earth magnet to impose a static magnetic field, B0, which varied linearly along the z axis with a gradient strength of approximately 0.95 T m−1 at the radio-frequency isocentre of the coil used. Our methodology assumes that (1) this gradient is uniform within the coil and (2) there is no variation in x−y planes; violation of these assumptions would degrade the localization accuracy. The imaging isocentre is both spatially the centre of the radio-frequency coil's sensitive area and was chosen to be at 17.8 mT corresponding to an ESR Larmor frequency of f ≈ 500 MHz, the coil's resonant frequency. The magnetic field was modulated axially by a Helmholtz coil such that \({B}_{z}={B}_{z}^{{\rm{magnet}}}(z)+{B}_{z}^{{\rm{Helmholtz}}}(t)\) where t is time. By varying the Helmholtz coil current between ±1 A, we were able to shift the effective magnetic field by ±5.87 mT around the resonant condition, providing a total field of view of approximately 12 mT along the gradient direction (see Supplementary Note 8 for a detailed calculation). The device is therefore able to scan spatially in one dimension as the Larmor frequency remains fixed at \(f={\bar{\gamma }}_{e}{B}_{z}(z,t)\), where different spatial positions are brought into resonance through current modulation while maintaining fixed radio-frequency excitation. To simulate a three-dimensional volumetric sample, we cast a 40-cm3 volume of PDMS (matching the region of uniformity of the radio-frequency coil) with empty cylinders of 0.4-mm-diameter embedded in it perpendicular to the B0 gradient and separated in z by 7.5 mm. We filled these cylinders with cells expressing MagLOV 2 fast (centrifuge-concentrated liquid culture), measuring at different depths both individually and together. Intensity data were processed (grey solid area in Fig. 5e,f) using a Lucy–Richardson deconvolution64 with a Gaussian impulse response function of fixed width 3 mm, which is estimated based on the ODMR linewidth of 80 MHz (see Supplementary Note 8 for further details). It is noted that although the deconvolution algorithm uses the known ODMR linewidth (and hence point-spread function), it does not make any assumption about the number or location of peaks. This fluorescence MRI instrument was designed and built to achieve proof of concept and is limited in terms of signal-to-noise performance of the optical path and photodiode sensor, as well as B0 field uniformity in the target volume. Future development could significantly improve its capabilities via implementing full three-dimensional control of B0 field gradients, an endoscope-type imaging system to collect light with x–y spatial information from the target volume without sensor perturbation by radio frequency or magnetic field, addition of fast optical lock-in to the fluorescence excitation or measurement, optical path re-design to improve collection and filtering efficiency and reduce stray light, and indeed directed evolution of variants with faster ODMR response dynamics to allow averaging over more on/off cycles. Purified MagLOV 2 fast solutions for the data in Fig. 6 were prepared as follows. It is noted that all MagLOV proteins expressed include a His-tag at the N-terminus. Protein was purified following using HisPur Cobalt Resin (Thermo Scientific) following the manufacturer's protocol. Purified protein was suspended in PBS buffer with 30% glycerol and stored at −80 °C. We estimated the concentration the purified protein samples via the method used in ref. 70 to determine the extinction coefficient of LOV-based fluorescent proteins. First, purified protein was heated to 95 °C for 5 minutes to dissociate the FMN from the protein (we assume that one FMN corresponds to one protein, as free FMN should be removed by the protein purification process). The concentration of flavin (and hence protein) was then determined by performing a serial dilution and measuring absorbance at 450 nanometres (A450nm), using the free FMN extinction coefficient ϵFMN = 12,200 M−1 cm−1 (ref. Contrast agent experiments were performed on the wide-field microscope configured for MFE detection, using a six-chamber microfluidic chip (ibdi μ-Slide VI 0.5 Glass Bottom) to hold samples of gadobutrol (CRS Y0001803) diluted in PBS buffer in serial dilution (MagLOV concentration the same for all conditions). Measurements were acquired in a randomized sequence (to compensate for any stray light photobleaching or time and sequence correlated effects) by programming 6 × 3 fields of view (6 chambers of differing concentrations, 3 fields of view in each chamber), then automatically performing an MFE measurement acquisition at each field of view. For each MFE acquisition, 10 periods of duration 40 seconds (20 seconds magnet on, 20 seconds off) were acquired at 450-nm light-emitting-diode intensity 280 mW cm−2, 100-ms image exposure time and 10-mT magnetic-field strength. Optimization of experimental protocols for spin-relaxometry-based sensing using MagLOV or other radical-pair fluorescent proteins will be required for broader application; here we demonstrate simply that the spin-radical pair of MagLOV is indeed sensitive to its surroundings despite the flavin being bound. To consider the expected effect of paramagnetic species upon the MagLOV MFE more quantitatively, we note that paramagnetic impurities can be modelled as stochastic point dipoles that modify the T1 or T2 relaxation rates in radical-pair kinetics72, thereby changing the contrast. We therefore anticipated that a characteristic timescale for this process, τ, would scale as \(\tau \approx \frac{1}{{k}_{{\rm{STD}}}}+{\left(\frac{1}{T}+R[{{\rm{Gd}}}^{3+}]\right)}^{-1}\) where kSTD is a stochastic decoherence rate, T is a semiempirical T1 or T2 relaxation time, and R the effective relaxivity of the paramagnetic impurity. We therefore expect contrast to fit a functional form of approximately (a[x] + b)−1 + c, and indeed this is consistent with Fig. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The experimental data that support the findings of this study are available at https://doi.org/10.25446/oxford.30344995. Plasmid sequencing data have been made available on the European Nucleotide Archive (ENA) with the identifier PRJEB83586. Code required to generate the figures are available at https://doi.org/10.25446/oxford.30344995. Simpson, D. A. Quantum probes for biology: unlocking single molecule dynamics. Segawa, T. F. & Igarashi, R. Nanoscale quantum sensing with nitrogen-vacancy centers in nanodiamonds—a magnetic resonance perspective. Ishiwata, H., Song, J., Shigeno, Y., Nishimura, K. & Yanai, N. Molecular quantum nanosensors functioning in living cells. Preprint at ChemRxiv https://doi.org/10.26434/chemrxiv-2025-k7db1-v2 (2025). Braun, T., Drescher, M. & Summerer, D. Expanding the genetic code for site-directed spin-labeling. Kerpal, C. et al. Chemical compass behaviour at microtesla magnetic fields strengthens the radical pair hypothesis of avian magnetoreception. Chauviré, T., Chandrasekaran, S., Dunleavy, R., Freed, J. H. & Crane, B. R. Flavoproteins as native and genetically encoded spin probes for in cell ESR spectroscopy. Denton, M. C. J. et al. Magnetosensitivity of tightly bound radical pairs in cryptochrome is enabled by the quantum Zeno effect. Magnetic control of the brightness of fluorescent proteins. Sukumar, U. K., Natarajan, A., Massoud, T. F. & Paulmurugan, R. in Fluorescent Imaging in Medicinal Chemistry (ed. Feder, J. S. et al. A fluorescent-protein spin qubit. & Kwiram, A. L. Investigation of the triplet state of flavines and flavoproteins by optical detection of magnetic resonance. Stass, D. V., Woodward, J. R., Timmel, C. R., Hore, P. J. & McLauchlan, K. A. Radiofrequency magnetic field effects on chemical reaction yields. Buckley, C. D., Hunter, D. A., Hore, P. J. & McLauchlan, K. A. Electron spin resonance of spin-correlated radical pairs. Spin dynamics of flavoproteins. Marriott, G. et al. Optical lock-in detection imaging microscopy for contrast-enhanced imaging in living cells. Henbest, K. B., Kukura, P., Rodgers, C. T., Hore, P. J. & Timmel, C. R. Radio frequency magnetic field effects on a radical recombination reaction: a diagnostic test for the radical pair mechanism. Imaging current paths in silicon photovoltaic devices with a quantum diamond microscope. Levine, E. V. et al. Principles and techniques of the quantum diamond microscope. Aslam, N. et al. Quantum sensors for biomedical applications. A study of spin chemistry in weak magnetic fields. & Dasari, D. Quantum sensing and control of spin-state dynamics in the radical-pair mechanism. Magnetic field effects in flavoproteins and related systems. Dodson, C. A. et al. Fluorescence-detected magnetic field effects on radical pair reactions from femtolitre volumes. Sensitive fluorescence-based detection of magnetic field effects in photoreactions of flavins. Kattnig, D. R. et al. Chemical amplification of magnetic field effects relevant to avian magnetoreception. Antill, L. M. & Woodward, J. R. Flavin adenine dinucleotide photochemistry is magnetic field sensitive at physiological pH. Bialas, C. et al. Ultrafast flavin/tryptophan radical pair kinetics in a magnetically sensitive artificial protein. Mechanism of giant magnetic field effect in a red fluorescent protein. Gravell, J. et al. Spectroscopic characterization of radical pair photochemistry in nonmigratory avian cryptochromes: magnetic field effects in GgCry4a. Magnetic sensitivity of cryptochrome 4 from a migratory songbird. Detection of magnetic field effects by confocal microscopy. Meng, K. et al. Optically detected and radio wave-controlled spin chemistry in cryptochrome. Preprint at bioRxiv https://doi.org/10.1101/2025.04.16.649006 (2025). Richter, G. et al. Photochemically induced dynamic nuclear polarization in a C450A mutant of the LOV2 domain of the avena sativa blue-light receptor phototropin. Schleicher, E. et al. On the reaction mechanism of adduct formation in LOV domains of the plant blue-light receptor phototropin. Kothe, G. et al. Detecting a new source for photochemically induced dynamic nuclear polarization in the LOV2 domain of phototropin by magnetic-field dependent 13C NMR spectroscopy. Coupled methyl group rotation in FMN radicals revealed by selective deuterium labeling. Magnetic resonance control of reaction yields through genetically-encoded protein:flavin spin-correlated radicals in a live animal. & Niopek, D. Enlightening allostery: designing switchable proteins by photoreceptor fusion. Petrenčáková, M. et al. Conformational properties of LOV2 domain and its C450A variant within broad pH region. Song, S.-H. et al. Absorption and emission spectroscopic characterisation of combined wildtype LOV1–LOV2 domain of phot from Chlamydomonas reinhardtii. Kay, C. W. M. et al. Blue light perception in plants: detection and characterization of a light-induced neutral flavin radical in a C450A mutant of phototropin. Zoltowski, B. D. et al. Chemical and structural analysis of a photoactive vertebrate cryptochrome from pigeon. Bialas, C. et al. Engineering an artificial flavoprotein magnetosensor. Zollitsch, T. M. et al. Magnetically sensitive radical photochemistry of non-natural flavoproteins. Antill, L. M., Kohmura, M., Jimbo, C. & Maeda, K. Introduction of magneto-fluorescence fluctuation microspectroscopy for investigating quantum effects in biology. Eisenreich, W., Joshi, M., Weber, S., Bacher, A. & Fischer, M. Natural abundance solution 13C NMR studies of a phototropin with photoinduced polarization. Eisenreich, W. et al. Tryptophan 13C nuclear-spin polarization generated by intraprotein electron transfer in a LOV2 domain of the blue-light receptor phototropin. Kopka, B. et al. Electron transfer pathways in a light, oxygen, voltage (LOV) protein devoid of the photoactive cysteine. Cormack, B. P., Valdivia, R. H. & Falkow, S. FACS-optimized mutants of the green fluorescent protein (GFP). Dang, X. et al. Deep-tissue optical imaging of near cellular-sized features. & Rudin, M. Fluorescence molecular tomography: principles and potential for pharmaceutical research. Hernandez, D. & Kim, K.-N. A review on the RF coil designs and trends for ultra high field magnetic resonance imaging. Norouzi, N. et al. Relaxometry for detecting free radical generation during Bacteria's response to antibiotics. Dynamic quantum sensing of paramagnetic species using nitrogen-vacancy centers in diamond. Yang, N. & Cohen, A. E. Optical imaging through scattering media via magnetically modulated fluorescence. Jablonski, A. E. et al. Signal discrimination between fluorescent proteins in live cells by long-wavelength optical modulation. Halavaty, A. S. & Moffat, K. N- and C-terminal flanking regions modulate light-induced signal transduction in the LOV2 domain of the blue light sensor phototropin 1 from Avena sativa. Sakai, M. & Takahashi, H. One-electron photoreduction of flavin mononucleotide: time-resolved resonance Raman and absorption study. & Penzkofer, A. Photo-induced reduction of flavin mononucleotide in aqueous solutions. Moore, S. J. et al. EcoFlex: a multifunctional MoClo Kit for E. coli synthetic biology. Automated design of synthetic ribosome binding sites to control protein expression. Seidel, Z. P., Wang, J. C. K., Riegler, J., York, A. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Array programming with NumPy. Pedregosa, F. et al. Scikit-Learn: machine learning in Python. Chen, T. & Guestrin, C. XGBoost: a scalable tree boosting system. 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining 785–794 (ACM, 2016); https://doi.org/10.1145/2939672.2939785. Robust estimation of bacterial cell count from optical density. Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. The photophysics of LOV-based fluorescent proteins—new tools for cell biology. Whitby, L. G. A new method for preparing flavin-adenine dinucleotide. Antill, L. M. & Vatai, E. RadicalPy: a tool for spin dynamics simulations. Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. We thank K. Henbest, E. Vatai and L. Gerhards for discussions; D. Cubbin for help with the ultraviolet–visible characterization; I. Robertson and P. Reineck for assistance with proof-of-concept experiments; C. Carr and G. Mazur for lending of radio-frequency equipment; P. Freemont for gift of an EcoFlex kit; and the ChimeraX73 team for protein-structure rendering tools used in Fig. and S.S. were supported by funding from the Biotechnology and Biological Sciences Research Council (UKRI-BBSRC; grant number BB/T008784/1). J.J. and V.T.-F. were supported by funding from the Engineering and Physical Sciences Research Council (UKRI-EPSRC; grant number EP/W524311/1). are supported in part by the UKRI-EPSRC under the EEBio Programme Grant, EP/Y014073/1, and EP/X017982/1 and UKRI-BBSRC (grant number BB/W012642/1). R.H. is supported by funding from the UKRI-EPSRC (grant number EP/Y034791/1). are supported by the European Research Council under the European Union's Horizon 2020 research and innovation programme, grant agreement number 810002, Synergy Grant: ‘QuantumBirds', C.R.T. thanks the US Army. acknowledges support from the Novo Fonden (NNF21OC0068683). recognizes support from the Philip Leverhulme Prize. These authors contributed equally: Ana Štuhec, Vincent Spreng Department of Engineering Science, University of Oxford, Oxford, UK Gabriel Abrahams, Vincent Spreng, Robin Henry, Idris Kempf, Jessica James, Kirill Sechkar, Scott Stacey, Vicente Trelles-Fernandez & Harrison Steel Department of Chemistry, University of Oxford, Oxford, UK Ana Štuhec, Lewis M. Antill & Christiane R. Timmel Institute of Pharmacy and Molecular Biotechnology (IPMB), Faculty of Engineering Sciences, Heidelberg University, Heidelberg, Germany Institute of Quantum Biophysics, Department of Biophysics, Sungkyunkwan University, Suwon, Republic of Korea The MR Research Centre, Aarhus University, Aarhus, Denmark Calico Life Sciences, South San Francisco, CA, USA Maria Ingaramo & Andrew G. York Department of Physics, School of Science, RMIT University, Melbourne, Victoria, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived of the study. performed the directed-evolution experiments. performed the spectroscopy experiments. designed and built the MRI set-up. developed the simulations and theory for the SCRP mechanism. coordinated work and wrote the paper with input from all authors. Correspondence to Gabriel Abrahams or Harrison Steel. is a cofounder and shareholder of Nonfiction Laboratories, a start-up company developing magnetogenetic control for therapeutic proteins. Nature thanks Seok-Hyun Yun and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Abrahams, G., Štuhec, A., Spreng, V. et al. Quantum spin resonance in engineered proteins for multimodal sensing. Version of record: 21 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-02055-9'>Hyperpyramidalized alkenes with bond orders near 1.5 as synthetic building blocks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 10:51:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Alkenes typically have trigonal planar geometries at each terminus, with favourable σ- and π-bonding leading to a bond order of ~2. Here we consider unusual alkenes that possess an extreme form of geometric distortion, termed hyperpyramidalization. In a hyperpyramidalized alkene, geometries deviate remarkably from the typical trigonal planar alkene geometry, leading to weak π-bonding and abnormal alkene bond orders approaching 1.5. Cubene and 1,7-quadricyclene, first validated in 1988 and 1979, respectively, but overlooked for decades since, are the focus of the present study. We leverage their unusually weak π-bonds in cycloadditions, enabling the construction of complex scaffolds and access to previously unrealized chemical space. The origins of the unusually low bond orders were investigated using computational methods. These efforts are expected to prompt future studies of molecules that display hyperpyramidalization or atypical bond orders. Carbon–carbon double bonds (C=C), known as alkenes, are essential functional groups in organic chemistry. It is well understood that the geometry at each alkene carbon is typically trigonal planar, as seen in ethylene (1), which leads to maximal p-orbital overlap in the alkene (Fig. However, it is also possible to deviate from this conventional trigonal planar geometry when an alkene is generated in a confined ring system1. The present study focuses on a specific type of geometric change known as pyramidalization2,3, which is exemplified in the pyramidalized depiction of ethylene 1-p, where the alkene carbon substituents no longer reside in the typical alkene plane. Pyramidalization ultimately leads to a weaking of the π-bond and lowering of bond order to below the typical bond order near 2 (ref. 4), as will be discussed more thoroughly later in this Article. a, Geometry and bond order of typical ethylene (1), and reduction of alkene bond order by pyramidalization of carbon termini (1-p). b, Pyramidalized alkenes (2–7) with non-integer bond orders, wherein there is little or no resonance stabilization. c, Non-integer alkene bond orders as a result of resonance effects, as exemplified by 8 and 9. d, Present study of cubene (10) and 1,7-quadricyclene (11), which have discrete hyperpyramidalized alkenes possessing non-integer bond orders near 1.5. The trapping and elaboration of 10 and 11 permit access to rigid, three-dimensional scaffolds. The depicted bond orders for 2–11 are calculated MBO values (⍵B97X-D/def2-TZVP). Me, methyl; Bn, benzyl group. Several examples of alkenes that display pyramidalization are shown in Fig. 1b, along with their calculated Mayer bond orders (MBOs)5,6. Compounds 2–5 are historical examples of so-called pyramidalized alkenes that have been studied as early as the 1950s2,3, but have escaped the attention of synthetic chemists in recent years. trans-Cyclooctene (6) also displays pyramidalization and has been valuable in bioorthogonal chemistry7,8. Recently, anti-Bredt olefin 7 and derivatives9,10,11, which display multiple forms of geometric distortion including pyramidalization, have emerged for use in synthesis. Compounds 2–7 have weakened π-bonds, as seen by the decreased alkene bond orders below 2 that we have calculated (MBOs ranging from 1.68 to 1.91; see the Supplementary Information, part II, section J). Thus, this indicates a relationship between geometric distortion in the form of pyramidalization and non-integer bond orders for discrete alkenes. This is notable as non-integer alkene bond orders are most commonly associated with resonance effects based on historical studies12. For example, the non-integer bond order values computed for stable compounds such as s-trans-butadiene (8) and benzene (9) (Fig. 1c) are well known to arise from resonance (MBOs of 1.87 and 1.41, respectively; see the Supplementary Information, part II, section J). Can discrete alkenes with even greater pyramidalization and smaller bond order values, particularly those near 1.5, be leveraged in chemical synthesis? This curiosity is notable as single, double and triple bonds are most commonly associated with integer bond orders of 1, 2 and 3, respectively, whereas such extreme weakening of a discrete bond is not yet a common consideration in synthetic chemistry. Moreover, the general use of pyramidalized alkenes in chemical synthesis has remained underdeveloped. The present study focuses on two alkenes that display severe pyramidalization, which we describe here as hyperpyramidalization. This is exemplified by structure 1-hp (Fig. 1d) and the two targets of interest: cubene (10) and 1,7-quadricyclene (11). While intriguing due to their unconventional geometries and unusually weak π-bonds (that is, bond orders of ~1.5), 10 and 11 were considered especially attractive targets owing to their possible use as building blocks in synthesis. Notably, rigid aliphatic frameworks have attracted considerable attention in medicinal chemistry in recent years due to their potential to improve drug-like properties and provide well-defined exit vectors13,14,15. Attractive features include increased metabolic stability16 due to higher C–H bond dissociation energies resulting from strain17, as well as improved lipophilicity and solubility13. Several recent studies of cubane functionalization are available, thus highlighting the interest in this burgeoning area18,19,20,21. As quadricyclanes have seen use in drug discovery22, bioorthogonal chemistry23 and energy storage applications24, new methods to synthesize substituted versions could be valuable in several applications. Cubene (10) and 1,7-quadricyclene (11) were first studied decades ago in seminal experimental25,26 and computational studies27,28,29,30, but have been largely overlooked in modern synthesis. Key prior experimental studies of cubene include: (1) Eaton's 1988 report of cubene generation from 1,2-diiodocubane using tert-butyllithium25, wherein a promising Diels–Alder trapping using a highly reactive diene (that is, 11,12-dimethylene-9,10-dihydro-9,10-ethanoanthracene) was reported, as well as the formation of cubane dimers that inferred the possibility of cubene intermediacy; and (2) Eaton's 1995 synthesis of Kobayashi-type cubene precursor, which bears a trialkylsilyl group and adjacent iodide group on the cubane scaffold31. Using the same diene trap as in their prior studies, cubene generation was validated, providing evidence that Kobayashi-type precursors32 to cubene are promising. Since these seminal contributions, no further cubene experimental studies have been reported, presumably due to harsh reaction conditions used in the case of the 1988 study, the limited scope of trapping to one cycloaddition partner25,31 and some challenging or ‘capricious' steps33 in the synthesis of Kobayashi-type cubene precursors. With regard to quadricyclene, several constitutional isomers have been considered in early reports34. We focus here on the 1,7-isomer, first generated by strong base-mediated dehydrohalogenation in a seminal 1979 study by Szeimies26. Formation of a Diels–Alder adduct with anthracene in modest yield confirmed the intermediacy of this highly distorted alkene, with subsequent reports35,36 demonstrating additional trapping reactions. However, the necessity of strong base and competitive formation of another isomer in some cases (that is, the 1,5-quadricyclene)34 have limited the scope of the early studies and the practical use of 11 in synthesis. A general, mild approach to 1,7-quadricyclene (11) has not been reported. Here, we describe studies of cubene (10) and 1,7-quadricyclene (11) (Fig. 1d), including the development of their practical synthetic methods and computational investigations of these unusual strained intermediates. We report access to suitable Kobayashi-type precursors, as well as the generation and in situ trapping of the desired hyperpyramidalized alkenes in cycloadditions. Reactions proceed under mild conditions and provide access to unusual structures typically bearing four highly substituted contiguous carbon centres within a rigid, three-dimensional framework (that is, 12 or 13). Moreover, we demonstrate the further elaboration of cycloadducts to give compounds with considerable structural complexity, including heterodimer 14 containing both the cubane and quadricyclane motifs. The geometric distortion parameters that ultimately lead to the low bond orders for cubene (10) and 1,7-quadricyclene (11), which are largely attributed to hyperpyramidalization of the alkene termini, are discussed. Overall, our discoveries provide a simple means to access exceedingly intricate structures of potential value for medicinal chemistry and other applications13,14,15,18,19,20,21,22,23,24, while also inspiring the future design and strategic manipulation of new intermediates that display hyperpyramidalization or non-integer bond orders in chemical synthesis. The caged structures of cubene (10) and 1,7-quadricyclene (11) are severely distorted from conventional alkene geometries, which, in turn, leads to unusual properties. As prior ab initio calculations of cubene (10)27,28 and 1,7-quadricyclene (11)29,30 structure and properties were reported decades ago using the methods available at the time, we performed calculations on these compounds using modern levels of theory. More specifically, geometry-optimized structures, MBOs, olefin strain energies and diradical characters were computed using appropriate levels of theory (Supplementary Information, part II, sections B–D, J and K). Several properties are notable (Fig. Both cubene (10) and 1,7-quadricyclene (11) are highly strained, possessing calculated olefinic strain energies of approximately 50 and 66 kcal mol−1, respectively. Confinement of the π-bond within the caged structures leads to severe geometric distortions associated with the alkenes in the form of bending and pyramidalization. More specifically, in cubene, the angles around the alkene termini are 93°, 93° and 99°, deviating substantially from typical alkene geometry (that is, 120° bond angles). These angles are a result of the rigid cubane framework. Correspondingly, rather than being trigonal planar, the alkene termini are pyramidalized to a severe extent. The pyramidalization angle as defined by Borden (Φp(Borden))37, which is applicable for alkenes with C2V symmetry, is calculated for cubene (10) to be 85.2°. This parameter shows the remarkable extent by which the positioning of the two alkene substituents deviate from what is typically observed in a planar alkene. We also calculated the pyramidalization angle (Φp) of 10 to be 31.6° as defined by Haddon's π-orthogonal average vector theory38, which is generally useful for any given pyramidalized alkene carbon. This is an unusually high Haddon pyramidalization angle, and we propose that alkenes with severely pyramidalized termini—where the average Φp of the alkene termini exceeds 20°—can be classified as hyperpyramidalized alkenes. Similar hyperpyramidalization is present in the geometry-optimized structure of 1,7-quadricyclene (11), which has bond angles of 63°, 93° and 112°, reminiscent of the parent quadricyclane. The Haddon pyramidalization angles (Φp) at the alkene termini are 33.2°, reflective of even greater pyramidalization in 11 compared with 10. The overall geometric distortion leads to both cubene and quadricyclene having substantial diradical character (y0) of 14% and 13%, respectively, rendering them diradicaloids39,40. These values are purely theoretical, based on configuration interaction calculations on the ground states (rather than transition states). These diradical characters are notably larger than the diradical characters of non-distorted alkenes (7% calculated for 1) and are related to the higher reactivity for 10 and 11 compared with typical alkenes. Lastly, we highlight the C=C bond lengths and the MBOs5 of these unusual tetra-substituted alkenes. The alkene bond lengths in 10 and 11 are 1.38 Å and 1.35 Å, respectively, slightly longer than the standard 1.33–1.34 Å alkene bond length. However, the MBOs are calculated to be approximately 1.59 and 1.55 for 10 and 11, respectively, opposite to what would be expected based on the pioneering empirical model of bond order described by Pauling41,42. This reflects the unusual nature of these weak, non-conjugated π-bonds, which will be further discussed later in this Article. a, Structural properties of 10 and 11. Geometry optimizations and MBOs computed at the ωB97X-D/def2-TZVP level of theory. Olefin strain energy calculated at the CCSD(T)/cc-pVTZ level of theory. Diradical character (y0) calculated at CASPT2/CASSCF(6,6)/aug-cc-pVDZ level of theory. b, Synthesis of cubene precursor 17 from amide 15 based on Eaton's seminal route. c, Synthesis of 1,7-quadricyclene precursor 20 from ketal 18. Φp, pyramidalization angle; TMP, 2,2,6,6-tetramethylpiperidide; t-Bu, tert-butyl; i-Pr, iso-propyl; DMF, N,N-dimethylformamide; DMAP, 4-dimethylaminopyridine; Bu, butyl; HMDS, hexamethyldisilazane; Ph, phenyl; Tf, trifyl; Et, ethyl. With the aim of generating cubene (10) and quadricyclene (11) under mild reaction conditions, which, in turn, could enable broad use in synthesis, we sought to prepare Kobayashi-type precursors to these strained intermediates. Kobayashi precursors, typically characterized by the presence of a silyl substituent adjacent to a good leaving group (that is, sulfonates or halides) have become the go-to substrates for accessing strained intermediates32. The exact choice of silyl substituent and leaving group for a given Kobayashi precursor typically stems from a combination of factors including synthetic accessibility, stability and efficiency of strained intermediate generation as observed empirically. Although Kobayashi precursors typically require multistep synthesis, they allow efficient strained intermediate generation and trapping under mild, robust and user-friendly reaction conditions. The synthesis of silyl iodide 17, the precursor to cubene (10), was accomplished using Eaton's strategy31 with modified protocols (Fig. Beginning from readily available amide 15, which was purchased or can be made from the corresponding carboxylic acid33, β-silylation was achieved using a two-step sequence involving: (1) metallation and iodination with N-iodosuccinimide (NIS); and (2) halogen–metal exchange and quenching with trimethylsilyl chloride. Next, the amide was elaborated into the required leaving group through a three-step sequence involving reduction of the amide, oxidation to the carboxylic acid, and Barton decarboxylation, ultimately affording silyl iodide 17. Although efforts to improve the route efficiency (for example, direct silylation of amide 15) were unsuccessful, the current optimized protocols allow access to cubene precursor 17 in a reproducible and scalable fashion. As a Kobayashi precursor to 1,7-quadricyclene (11) was not known, we developed the scalable route to silyl triflate 20 shown in Fig. It begins from readily available known bromo-ketal 18, which is easily synthesized from commercially available bicyclo[2.2.1]hetp-5-en-2-one43. Halogen–metal exchange, followed by quenching with triethylsilyl chloride, enabled the introduction of the required silicon substituent. Treatment of the silylated intermediate with HCl gave rise to ketone 19 in 92% yield. Subsequent triflation provided a norbornadiene intermediate, which then underwent photochemical (2 + 2) cycloaddition44 to construct the quadricyclane core and furnish Kobayashi precursor 20. With access to the Kobayashi precursors, we commenced experimental studies on these strained intermediates, beginning with efforts to generate and trap cubene (10) in Diels–Alder reactions (Fig. As noted earlier, Eaton had shown one example of a presumed Diels–Alder trapping31 using 11,12-dimethylene-9,10-dihydro-9,10-ethanoanthracene, yet the corresponding use of Kobayashi precursors to cubene (10) in Diels–Alder chemistry with other dienes has not been reported. Thus, we examined the reaction of silyl iodide 17 with a fluoride source in the presence of trapping agents to give cycloadducts (12). The fluoride source, stoichiometry, solvent, additives and temperature were varied, ultimately leading to the identification of optimal reaction conditions that involved the use of tetrabutylammonium fluoride (Bu4NF) in tetrahydrofuran (THF) at 40 °C. Under these mild conditions, precursor 17 was typically fully consumed. a, Conditions: 17 (1 equiv. ), trapping agent (1.5–10 equiv. ), THF (0.05 M), 40 °C, 2–71 h, sealed vessel. For entry 8, the observed regioselectivity ratio (r.r.) indicates distribution of constitutional isomers, with the major isomer formed being depicted. b, Access to homocubane 39 via Diels–Alder cycloaddition, followed by dyotropic rearrangement. c, (5 + 2) cycloaddition with oxidopyridinium 40 to furnish azabicycle 41. d, (3 + 2) cycloaddition to introduce a five-membered ring fused to the cubane scaffold. e, Alternative approach to five-membered ring fused to cubane scaffold (that is, 44) involving oxidative cleavage of cycloadduct 32. Bu, butyl; Me, methyl; Ph, phenyl; Et, ethyl; Boc, butyloxycarbonyl; Bn, benzyl. As shown in Fig. 3a, our optimal conditions were amenable to the use of a variety of electron-rich dienes as trapping agents. The use of anthracenes 21 and 23 gave their respective cycloadducts 22 and 24, in excellent yields (entries 1 and 2, respectively). As will be discussed later in more detail, it should be noted that these and other products made via this methodology have unusually complex structures, bearing four consecutive highly substituted carbon atoms (3° or 4°). We also examined the use of diphenylisobenzofuran (25) and furan 27, which furnished oxygenated cycloadducts 26 and 28, respectively (entries 3 and 4, respectively). Moreover, when pyrroles 29 and 31 were used, the respective azabicycles 30 (entry 5) and 32 (entry 6) were obtained. When electron-poor cyclopentadiene 33 was utilized (entry 7), the formation of adduct 34 was achieved, probably via an inverse electron-demand Diels–Alder cycloaddition45,46. Use of other Diels–Alder trapping agents, such as tropolone 35 and exocyclic diene 37, gave the respective adducts 36 and 38 (entries 8 and 9, respectively). The methodology permits access to several other exotic structures, as shown in Fig. Using 9-bromoanthracene as a diene trap, followed by treatment of the resulting Diels–Alder cycloadduct with silica gel, facilitated a dyotropic rearrangement47, ultimately yielding homocubane 39 in 57% yield (Fig. X-ray crystallography was used to corroborate this atypical structure. Moreover, this example highlights the privileged ability of cubanes to undergo unique rearrangements48 to access related caged moieties. We were also delighted to find that other types of cycloaddition were possible. Use of oxopyridinium 40 led to [3.2.1] azabicycle 41, which boasts an unusual scaffold bearing three consecutive fully substituted stereocentres (Fig. 3c), presumably via (5 + 2) cycloaddition49 of the cubene (10) intermediate. (2 + 2) and (3 + 2) cycloadditions were also explored, several of which were unsuccessful, presumably because of unfavourable kinetics in forming a cubane-fused [2.2.2]-propellane or five-membered ring. However, the trapping of cubene (10) with ene carbamate 42 (ref. 50) delivered pyrrolidine 43, which provides a rare example of a heterocycle-fused cubane derivative51,52,53,54 (Fig. A related heterocycle, bearing an isomeric pyrrolidine core in comparison with 43, was accessed using an ozonolysis strategy with reductive work-up (32→44; Fig. As shown in Fig. 4a, 1,7-quadricyclene (11) generation and Diels–Alder trapping experiments were also performed. Similar to our studies of cubene (Fig. 3a), silyl triflate 20 was treated with fluoride in the presence of a diene trapping agent to give cycloadducts 13. Through experimentation, we found that the use of Bu4NF in THF was generally optimal, although the preferred temperature in our studies of 11 was deemed to be 23 °C (versus 40 °C in our studies of cubene). Stoichiometry and reaction times were empirically optimized for each entry. a, Conditions: 20 (1 equiv. ), trapping agent (3–20 equiv. ), THF (0.05 M), 23 °C, 3–24 h, sealed vessel. Where applicable, observed diastereoselectivities and regioselectivities are reported as a ratio of isomers (diastereomeric ratio (d.r.) or regioisomeric ratio (r.r. ), respectively), with the major isomer formed being depicted. bYield determined by 1H NMR analysis with mesitylene as an external standard. cOnly the depicted diastereomer was observed likely due to instability of the other diastereomer. b, Ene reaction to give 56. c, Elaboration of 45 to give cyclobutenyl-fused norbornene 59. d, Pd/C mediated isomerization of adduct 55 to bridged norbornadiene 60 and hydrogenolysis of 55 to furnish 61 and 62. Bu, butyl; Me, methyl; Ph, phenyl; Et, ethyl; PMB, p-methoxybenzyl; Bn, benzyl; Tf, triflyl. We were delighted to find that the trapping using 9-methoxyanthracene (23) furnished the desired cycloadduct 45, bearing a bridged bicycle appended to the quadricyclane core (Fig. Of note, this transformation proceeds more efficiently compared with the corresponding report by Szeimies36 (67% versus 38% yield) that utilized a dehydrohalogenation approach with a strong base, rather than the mild Kobayashi-type strategy shown herein. Trapping of furan derivatives 25 and 27 gave rise to oxabicycles 46 and 47, respectively, with only endo products being observed (entries 2 and 3). Moreover, the use of tropolone 35 or tropone 49 afforded bicyclic enones 48 (entry 6) or 50 (entry 7), respectively, in good yields and moderate diastereoselectivities. Lastly, exocyclic dienes 51, 53 and 37 were examined, each possessing a different heterocyclic framework (that is, tetrahydrooxepine, pyrrolidine or oxazolidinone, respectively) as shown in entries 6–8. The use of these dienes in 1,7-quadricyclene (11) trapping experiments provided fused cycloadducts 52, 54 and 55, respectively. Other experiments involving quadricyclene or cycloadducts were performed, with key results shown in Fig. The Alder-ene reaction55 proceeded readily using 56 to furnish quadricyclane derivative 57, which is adorned with a styrene-containing sidechain (Fig. This result demonstrates the viability of a group-transfer pericyclic reaction. Attempts to achieve other types of cycloaddition, such as (3 + 2) or (2 + 2) reactions, proved challenging, presumably due to disfavourable reaction kinetics, the instability of potential adducts due to their strain, or a combination thereof. Nonetheless, we pursued alternative approaches to access unique compounds by modifying the highly strained quadricyclane scaffold56. As shown in Fig. 4c, reaction of cycloadduct 45 and dimethyl acetylenedicarboxylate (58)36 gave rise to norbornene 59, bearing a cyclobutene ring, presumably via a regioselective ring-opening cycloaddition process. Other modified scaffolds were accessible using palladium catalysis, as shown in Fig. Exposure of adduct 55 to catalytic palladium on carbon (Pd/C) led to isomerization57 of the quadricyclane core, giving bridged norbornadiene 60. This could also be achieved thermally, albeit using prolonged reaction times, whereas the reverse reaction occurs using ultraviolet light (Supplementary Information, part I, section G). In addition, treatment of 55 with H2 and Pd/C furnished notricyclane 61 and bridged norbornane 62 (2:1 ratio of separable isomers). Several features of the synthetic compounds depicted in Figs. 3 and 4 should be emphasized. The products formed from cycloaddition reactions, all arising from just two common Kobayashi precursors (that is, 17 and 20), bear vicinal substitution stemming from the rigid cubane or quadricyclane cores. This is notable given that: (1) methods to prepare cubanes and 1,7-quadricyclanes with vicinal substitution patterns are underdeveloped; (2) having the di-substitution in the form of a ring or bicycle fused to the cubane or quadricyclane framework is rare; and (3) rigid, strained three-dimensional scaffolds are of current interest in medicinal chemistry13,14,15, as discussed earlier, but can often be difficult to access. Thus, the methodology we describe provides a practical means to access new and coveted scaffolds. With regard to the specific ring systems accessible, we show the possibility of accessing both five- and six-membered rings fused to the cubane or quadricyclane frameworks (for example, 38, 43 and 44 in Fig. In the case of products containing bridged bicyclic ring systems, [3.2.2]-, [3.2.1]-, [2.2.2]- and [2.2.1]-ring systems can be made (see entries 1–8 in Fig. 3c and entries 1–5 in Fig. Some limitations should also be noted. For example, (2 + 2) cycloadditions to give cyclobutyl-fused products have thus far proven challenging. Similarly, additions of nucleophiles to give mono-substituted cubanes or quadricyclanes have been met with mixed results and will be reported separately in due course. Nonetheless, the structural complexity that can be generated by this methodology and derivatization is underscored by the successful modifications of cycloadducts as shown in Figs. 3b,e and 4c,d. Finally, most products obtained possess four newly formed consecutive, highly substituted carbon centres (3° or 4°), appended to the cubane or quadricyclane cores, which themselves are also composed primarily of 3° carbons. Thus, many of the compounds contain eight to ten consecutive highly substituted carbons (3° or 4°). The low bond order and consequent high reactivity of cubene (10) and 1,7-quadricyclene (11) are essential for enabling access to the exquisite architectures shown herein. As previously discussed, cubene (10) and 1,7-quadricyclene (11) are unusual alkenes that are geometrically distorted in rather remarkable ways. The two major modes of distortion associated with 10 and 11 are bending and pyramidalization, which result from their rigid caged scaffolds. The impact of geometric distortion can be assessed by various parameters, such as: (1) strain energy, or Schleyer's olefin strain energy58, which provides a useful means to compare similar structures, and their reactivities based on the relationship defined by the Evans–Polanyi principle59, (2) delocalization, as discussed by Sterling, Anderson and Duarte60, and (3) diradical character, which has been used recently to explain an unusual reaction of strained cyclic allenes that plausibly proceeds through a one-electron process40,61. The complex interplay between these parameters is the subject of ongoing investigations and will be described in due course. Nonetheless, in the present study, we focus on the impact of geometric distortion on alkene bonding, given the unusually low bond orders of ~1.5 seen in both cubene (10) and 1,7-quadricyclene (11) and the potential to use bond order as a guiding parameter in synthetic design. We provide insight into the relationship between geometric distortions and bond order, which are ultimately related to the high reactivity of 10 and 11. Computational studies were performed using ethylene (1) as a model system, with geometric distortions being introduced systematically. For each distorted structure, the alkene bond order5,6 was calculated, with results for bending and pyramidalization summarized in Fig. 5a,b (see the Supplementary Information, part II, section I, for the MBO dependence on bond length). Figure 5a shows the bond order of systematically bent ethylene without pyramidalization. Beginning with ethylene at its equilibrium geometry (1–eq), the expected bond order value of roughly 2 is observed. In the distorted structure 1a, the H–C=C bond angles are increased to ~150°, which gives a slight increase in the bond order (MBO 1.97 → 2.04). Conversely, a slight drop in the bond order is observed when the H–C=C bond angles are contracted to ~94°, as seen in 1b (MBO 1.97 → 1.86). Overall, we conclude that bending of the H–C=C bonds, without pyramidalization, leads to only a minor change in bond order. a, MBO of systematically distorted ethylene at varying H–C=C bond angles to assess the effect of bending. b, MBO of systematically distorted ethylene at varying pyramidalization angles (Φp). c, Comparison of π molecular orbital (π MO) contour plots and approximate AO composition of π MO of planar ethylene (1c), pyramidalized ethylene (1d) and hyperpyramidalized ethylene (1e). MO structures were obtained at the HF/6-31G(d,p) level of theory. d, Localized depiction of π MOs and calculated HOMOs of ethylene (1), cubene (10) and 1,7-quadricyclene (11). Geometry optimizations were performed at the ωB97X-D/def2-TZVP level of theory. MO structures were obtained at the HF/6-31G(d,p) level of theory. MO surfaces are plotted at an isovalue of ±0.023 a.u. e, Computations of the Diels–Alder reaction of anthracene 21 with either olefin 1, 10 or 11, performed at the ⍵B97XD/def2-TZVP/SMD(THF) level of theory. See the Supplementary Information for details. f, Unification of cubene and quadricyclene cycloadducts provides heterodimer 14. Me, methyl; Bn, benzyl; microED, microcrystal electron diffraction. A much more pronounced impact on alkene bond order is observed when considering pyramidalization. As shown in Fig. 5b, both Borden (Φp(Borden)) and Haddon (Φp) pyramidalization angles were considered as ethylene was systematically pyramidalized and then hyperpyramidalized. Structure 1c (Φp(Borden) and Φp = 0°) with all bond angles at 120° reflects ethylene near its equilibrium planar geometry, showing the expected alkene bond order value of ~2. Upon syn-pyramidalization of the two ethylene termini, the bond order drops remarkably. For example, in structure 1d (Φp(Borden) ≈ 58° and Φp ≈ 21°), the bond order approaches 1.8. An even more drastic decrease of the alkene bond order was observed upon further increase of the pyramidalization angles. In the extreme case of hyperpyramidalized structure 1e (Φp(Borden) ≈ 98° and Φp ≈ 39°), an alkene bond order approaching 1.4 is calculated. We have also plotted Φp versus bond order for other strained intermediates and observe similar correlations (Supplementary Information, part II, section I). These results generally align with the bond orders and pyramidalization angles of cubene (10) and 1,7-quadricyclene (11), suggesting that hyperpyramidalization is probably the major contributing factor to the low bond order seen in these species. We also highlight a point of contrast between resonance-stabilized alkenes and hyperpyramidalized alkenes. Although both have atypical non-integer bond orders below 2, resonance stabilization leads to increased stability, whereas hyperpyramidalization leads to decreased stability and higher reactivity. Calculations were performed to gauge how increasing pyramidalization leads to a lowering of the alkene bond order. Orbital occupancy-perturbed MBO analysis62 indicates that the lowering of total alkene bond orders in pyramidalized and hyperpyramidalized alkenes is attributed to the weakening of π-bonding rather than σ-bonding (Supplementary Information, part II, section E). This is consistent with earlier studies by Borden, which suggest weakening of the π-bond in pyramidalized ethylene37. Indeed Borden et al. have shown that the p orbitals that typically constitute the π-bond in planar ethylene instead constitute the σ-bond in highly pyramidalized ethylene, resulting in a drop in CC overlap population37. We also studied the π-bonding in 1c, 1d and 1e and carried out atomic orbital (AO) component analysis63,64,65 (Fig. The contour plots of the π molecular orbitals in alkenes 1c–1e are shown. As one would expect for non-distorted ethylene with trigonal planar geometry at the alkene termini, orbital 1c–π is composed of nearly 100% p orbitals, arising entirely from the Pz orbital contribution as defined by the xyz axes depiction. Upon pyramidalization as seen in 1d–π, orbital mixing occurs66,67, with the resulting π-bond being composed of roughly 8% s character and 91% p character. With regard to the latter, the Pz contribution drops from 100% to 68%, while the Py contribution increases from 0% to 23%. The increase in the Py contribution is reflected in the contour map of 1d–π and suggested by the change in directionality of the blue lobe, now tilting towards the y axis (see red arrows)68. In the case of hyperpyramidalization (see 1e–π), the s character in the π-bond further increases to 22%, with a corresponding decrease in p orbital contribution to 74%, leading to sp3-like character. The Pz contribution lowers to only 16%, while the Py contribution increases to 58%. The directionality of the lobes (see red arrows) is further tilted towards the y axis, away from the z axis, leading to a substantial reduction in effective hybrid orbital overlap. Overall, the studies shown in Fig. 5c demonstrate that, upon hyperpyramidalization, the substantial decrease in Pz character, with increases in s and Py character, leads to an extreme lowering of the alkene bond order. An analysis of the π molecular orbitals of cubene (10) and 1,7-quadricyclene (11) is shown in Fig. In comparison with ethylene (see 1–π), the corresponding orbitals in cubene (10–π) and 1,7-quadricyclene (11–π) have increased s character due to hyperpyramidalization, as seen in pyramidalized variants of ethylene (see discussion of Fig. 5c), and are extended towards the exterior of the caged scaffold66. Moreover, the orbitals shown in 10 and 11 (see 10–π and 11–π, respectively) are no longer parallel68. Instead, the larger lobes are oriented away from one another outside of the caged scaffold, with the smaller lobes pointed towards each other within the cages (see 10–π and 11–π; also see the Supplementary Information, part II, section F & G), ultimately resulting in decreased overlap. Such orbital reorientation is most pronounced in the case of 1,7-quadricyclene (11), as the alkene termini are more pyramidalized in 11 compared with the alkene termini in cubene (10). The decrease in effective π-bonding in 10 and 11, in comparison with ethylene (see highest occupied molecular orbital (HOMO) 1–π), is also seen in the calculated molecular orbitals HOMO 10–π and HOMO 11–π. This consideration of the π molecular orbitals of 10 and 11 provides an explanation for the weak π-bonding and the consequential low alkene bond orders of the hyperpyramidalized alkene intermediates cubene (10) and 1,7-quadricyclene (11). The weakening of the alkene π-bond upon hyperpyramidalization correlates to an increase of the π HOMO energy, with a corresponding substantial lowering of the π lowest unoccupied molecular orbital (LUMO) energy, contributing to high reactivity and reaction exothermicity27. Calculations were performed to assess the effect of hyperpyramidalization on transition state barriers and reaction exothermicity (Fig. 5e; also see the Supplementary Information, part II, section H, for transition state analysis using density functional theory). The reaction between ethylene (1) and anthracene (21) is calculated to proceed with a high activation barrier (ΔG‡ = 37.1 kcal mol−1) and high enthalpy of activation (ΔH‡ = 23.8 kcal mol−1). The reaction free energy (ΔGr) and enthalpy (ΔHr) are –12.6 and –27.0 kcal mol−1, respectively. By contrast, the corresponding cycloadditions involving the hyperpyramidalized alkenes in cubene (10) and quadricyclene (11) are calculated to be much more exothermic and exergonic, and the computed ΔG‡ for the Diels–Alder reactions of 10 and 11 with anthracene (21) were found to be only 18.7 and 13.8 kcal mol−1, respectively. The exothermicities (ΔHr) for the reactions of 10 and 11 were calculated to be –74.6 and –83.3 kcal mol−1, respectively. Thus, hyperpyramidalization allows cycloadditions to occur readily due to large strain release upon reaction. Consistent with the Bell–Evans–Polanyi relationship, the increase in exothermicity is accompanied by a lowering of the activation barrier by approximately half that amount. The ultimate consequence of hyperpyramidalization is that cubene (10) and 1,7-quadricyclene (11) react rapidly69 as dienophiles in normal electron-demand Diels–Alder cycloadditions, despite being tetra-substituted without electron-withdrawing groups. Products bearing four newly formed contiguous 3° or 4° carbon centres are readily accessible, thus enabling the introduction of great structural complexity into caged scaffolds. Figure 5f provides a final demonstration of the high level of structural complexity attainable via this methodology. 63-Li was obtained by lithiation of the corresponding alkyne, which, in turn, was prepared in one step from cubene precursor 17 (Supplementary Information, part I, section H). Treatment of this intermediate with 1,7-quadricyclene cycloadduct 48 led to union of the two fragments, ultimately affording heterodimer 14. Of note, the tolerance of both coupling partners to such conditions involving organolithium chemistry bodes well for the future applicability of our methodology in multistep synthesis. Heterodimer 14, the structure of which was confirmed by microcrystal electron diffraction (microED)70, possesses nine 3° or 4° carbon centres that were formed through the cycloaddition methodology and fragment coupling, thus pushing the limits of structural and stereochemical complexity attainable using strained intermediate chemistry. These studies harness the chemistry of cubene (10) and 1,7-quadricyclene (11) to provide a simple means to access exceedingly intricate chemical structures of value to medicinal chemists. These intermediates are unusual, as they possess hyperpyramidalized alkenes with weak π-bonding and, consequently, bond orders approaching 1.5. As such, we expect these studies will enable the future design and strategic manipulation of other unconventional intermediates that display hyperpyramidalization or non-integer bond orders for use in chemical synthesis. The general procedure for the generation and trapping of cubene (10) is described as follows. To a 2-dram vial equipped with a stir bar was added cubene precursor 17 (30 mg, 0.10 mmol, 1.0 equiv), followed by the trapping agent (0.15-1.0 mmol, 1.5–10 equiv.). The headspace of the reaction was purged with N2 for 5 min, then THF (1.5 ml) and Bu4NF (1.0 M in THF, 0.5 ml, 0.50 mmol, 5.0 equiv.) were added sequentially via syringe in a single portion. The vial was sealed with a Teflon-lined screw cap and stirred at 1,000 rpm at 40 °C. After the specified reaction time, the reaction vessel was allowed to cool to 23 °C. The crude mixture was filtered through a 0.5 × 2 cm silica gel plug, eluting with EtOAc (10 ml). The eluate was subsequently concentrated under reduced pressure to dryness, and the crude material was analysed by 1H nuclear magnetic resonance (NMR) spectroscopy. The sample for NMR analysis was then recombined with the crude residue and concentrated. Subsequent purification by flash column chromatography on silica gel or preparative thin-layer chromatography using appropriate eluents, followed by drying of the products under high vacuum, yields the corresponding cycloadducts. The general procedure for the generation and trapping of 1,7-quadricyclene (11) is described as follows. To a 2-dram vial containing a stirred solution of 1,7-quadricyclene precursor 20 (35.5 mg, 0.10 mmol, 1.0 equiv.) and trapping agent (0.18–2.0 mmol, 1.8–20 equiv) in THF (1.5 ml) was added Bu4NF (1.0 M in THF, 0.5 ml, 0.50 mmol, 5.0 equiv.) in one portion. The pierced septum cap was sealed with melted paraffin wax and the reaction mixture was stirred at 800–1,200 rpm at 23 °C. After the specified reaction time, saturated aq. NH4Cl (2 ml) was added to quench the reaction. The crude mixture was extracted with diethyl ether/pentane (1:1, 3× 2 ml), and the organic extracts were dried over Na2SO4. The crude mixture was concentrated under reduced pressure to dryness, and the crude material was analysed by 1H NMR spectroscopy. The sample for NMR analysis was then recombined with the crude residue, concentrated, and purified by chromatography using appropriate eluents, followed by drying under high vacuum to yield the corresponding cycloadducts. Experimental procedures, characterization data, computational methods and computational data are provided in the Supplementary Information. Crystallographic data for the structures reported in this Article have been deposited at the Cambridge Crystallographic Data Centre, under registry numbers CCDC 2446782 (39) and 2456006 (14). Copies of the data can be obtained free of charge at https://www.ccdc.cam.ac.uk/structures. Reprints and permissions information is available at www.nature.com/reprints. McDermott, L., Walters, Z. G., Clark, A. M. & Garg, N. K. Geometric distortion as an enabling tool for organic synthesis. Borden, W. T. Pyramidalized alkenes. Vázquez, S. & Camps, P. Chemistry of pyramidalized alkenes. Kaiser, J. & Chalk, S. J. (eds) bond order. in IUPAC Compendium of Chemical Terminology 5th edn, Online version 5.0.0 https://doi.org/10.1351/goldbook.BT07005 (International Union of Pure and Applied Chemistry, 2025). Mayer, I. Charge, bond order and valence in the ab initio SCF theory. Mayer, I. Bond order and valence indices: a personal account. Selvaraj, R. & Fox, J. M. trans-Cyclooctene-a stable, voracious dienophile for bioorthogonal labeling. Levandowski, B. J., Svatunek, D., Sohr, B., Mikula, H. & Houk, K. N. Secondary orbital interactions enhance the reactivity of alkynes in Diels–Alder cycloadditions. Bredt, J. Über sterische Hinderung in Brückenringen (Bredtsche Regel) und über die meso-trans-Stellung in kondensierten Ringsystemen des Hexamethylens. Justus Liebigs Ann. Concerning the existence of 1-norbornene. McDermott, L. et al. A solution to the anti-Bredt olefin synthesis problem. Anslyn, E. V. & Dougherty, D. A. Modern Physical Organic Chemistry (Univ. Tsien, J., Hu, C., Merchant, R. R. & Qin, T. Three-dimensional saturated C(sp3)-rich bioisosteres for benzene. Subbaiah, M. A. M. & Meanwell, N. A. Bioisosteres of the phenyl ring: recent strategic applications in lead optimization and drug design. Lazzara, P. R. & Moore, T. W. Scaffold-hopping as a strategy to address metabolic liabilities of aromatic compounds. Lovering, F., Bikker, J. & Humblet, C. Escape from flatland: increasing saturation as an approach to improving clinical success. Feng, Y., Liu, L., Wang, J.-T., Zhao, S.-W. & Guo, Q.-X. Homolytic C–H and N-H bond dissociation energies of strained organic compounds. A. et al. Validating Eaton's hypothesis: cubane as a benzene bioisostere. Reekie, T. A., Williams, C. M., Rendina, L. M. & Kassiou, M. Cubanes in medicinal chemistry. Wiesenfeldt, M. P. et al. General access to cubanes as benzene bioisosteres. Yasukawa, T., Håheim, K. S. & Cossy, J. Functionalization of cubane formation of C–C and C–heteroatom bonds. Dzhemilev, U. M. et al. A new original approach to the design of anticancer drugs based on energy-rich quadricyclanes. Sletten, E. M. & Bertozzi, C. R. A bioorthogonal quadricyclane ligation. Orrego-Hernández, J., Dreos, A. & Moth-Poulsen, K. Engineering of norbornadiene/quadricyclane photoswitches for molecular solar thermal energy storage applications. Eaton, P. E. & Maggini, M. Cubene (1,2-dehydrocubane). Harnisch, J. et al. Intermediacy of tetracyclo[3.2.0.02.7.04.6]hept-1(7)-ene, a highly strained bridgehead olefin. & Borden, W. T. Ab initio calculations of the olefin strain energies of some pyramidalized alkenes. & Borden, W. T. Ab initio calculations of the relative energies of 1,2-, 1,3-, and 1,4-dehydrocubane: prediction of dominant through-bond interaction in 1,4-dehydrocubane. Podlech, J., Polborn, K. & Szeimies, G. J. Tetracyclo[3.2.0.02,7.04,6] hept-1-ene: formation and trapping of 1, 2-dehydroquadricyclane. Ab initio calculations on dehydroquadricyclanes. Hoenigman, R. L., Kato, S., Bierbaum, V. M. & Borden, W. T. Heat of hydrogenation of 1,5-dehydroquadricyclane. A computational and experimental study of a highly pyramidalized alkene. Lukin, K. & Eaton, P. E. Dimerization of cubene. 1-Iodoadamantane as a probe for radical intermediates. Shi, J., Li, L. & Li, Y. o-Silylaryl triflates: a journey of Kobayashi aryne precursors. Houston, S. D. et al. The cubane paradigm in bioactive molecule discovery: further scope, limitations and the cyclooctatetraene complement. & Szeimies, G. Nucleophile austauschreaktionen am 1-chlorquadricyclan: 1,5 und 1,7-dehydroquadricyclanals reaktive zwischenstufen. Baumgärtel, O. et al. Diels–Alder-addukte des 1,7-dehydroquadricyclans. Volland, W. V., Davidson, E. R. & Borden, W. T. Effect of carbon atom pyramidalization on the bonding in ethylene. Haddon, R. C. Comment on the relationship of the pyramidalization angle at a conjugated carbon atom to the σ bond angles. Stuyver, T. et al. Do diradicals behave like radicals?. Tena Meza, A. et al. σ-Bond insertion reactions of two strained diradicaloids. Pauling, L. The Nature of the Chemical Bond (Cornell Univ. Pauling, L. Atomic radii and interatomic distances in metals. Lai, K. & Salomon, R. G. 6-Substituted bicyclo[2.2.1]hept-5-en-2-one ketals. Cristol, S. J. & Snell, R. L. Bridged polycyclic compounds. The photoisomerization of bicyclo [2,2,1]hepta-2,5-diene-2,3-dicarboxylic acid to quadricyclo[2,2,l,02,6,03,5]heptane-2,3-dicarboxylic acid. Kelleghan, A. V., Tena Meza, A. & Garg, N. K. Generation and reactivity of unsymmetrical strained heterocyclic allenes. & Raines, R. T. Click chemistry with cyclopentadiene. Eaton, P. E. & Yip, Y. C. The preparation and fate of cubylcarbinyl radicals. Biegasiewicz, K. F., Griffiths, J. R., Savage, G. P., Tsanaktsidis, J. & Priefer, R. Cubane: 50 years later. Yamamoto, Y. Update for oxidopyridinium cycloadditions and their synthetic applications: advances after Katritzky's pioneering studies. Gilmore, C. D., Allan, K. M. & Stoltz, B. M. Orthogonal synthesis of indolines and isoquinolines via aryne annulation. Eaton, P. E. & Pramod, K. Cubanourea: a cubane–propellane. Gilardi, R., George, C., Karle, J., Eaton, P. E. & Rao, M. Azidocarbonyl-substituted cubanoureas. Nagasawa, S., Hosaka, M. & Iwabuchi, Y. ortho-C-H acetoxylation of cubane enabling access to cubane analogues of pharmaceutically relevant scaffolds. Hosaka, M., Nagasawa, S. & Iwabuchi, Y. C–H alkylation of cubanes via catalytic generation of cubyl radicals. Alder, K., Pascher, F. & Schmitz, A. Über die Anlagerung von Maleinsäure-anhydrid und Azodicarbonsäure-ester an einfach ungesättigte Koh an einfach ungesättigte Kohlenwasserstoffe. Zur Kenntnis von Substitutionsvorgängen in der Allyl-Stellung. Petrov, V. A. & Vasil'ev, N. V. Synthetic chemistry of quadricyclane. Hogeveen, H. & Volger, H. C. Valence isomerization of quadricyclene to norbornadiene catalyzed by transition metal complexes. Maier, W. F. & Schleyer, P. V. R. Evaluation and prediction of the stability of bridgehead olefins. Evans, M. G. & Polanyi, M. Further considerations on the thermodynamics of chemical equilibria and reaction rates. Sterling, A. J., Smith, R. C., Anderson, E. A. & Duarte, F. Beyond strain release: delocalization enabled organic reactivity. Fernando, S. H., Migliaro, I. & Sterling, A. J. Diradical character in strained molecules accelerates bioisostere synthesis. Trends Chem. Gorelsky, S. I. Complexes with a single metal–metal bond as a sensitive probe of quality of exchange–correlation functionals. Lu, T. & Chen, F. Calculation of molecular orbital composition. Lu, T. & Chen, F. Multiwfn: a multifunctional wavefunction analyzer. Lu, T. A comprehensive electron wavefunction analysis toolbox for chemists, Multiwfn. Inagaki, S., Fujimoto, H. & Fukui, K. Orbital mixing rule. Flexible olefins with nonplanar puckered structures. Houk, K. N. et al. Electronic origins and consequences of pyramidalization of asymmetric alkenes in ground and triplet excited states. Bickelhaupt, F. M. & Houk, K. N. Analyzing reaction rates with the distortion/interaction–activation strain model. Jones, C. G. et al. The cryoEM method microED as a powerful tool for small molecule structure determination. We are grateful to the NIH-NIGMS and NSF (grant nos. ), the UCLA Cota Robles Fellowship programme (for C.A.R. ), the UCLA Graduate Division Dissertation Year Fellowship (for D.C.W. ), the Foote family (for A.T.M. ), the Stone Family (for D.C.W.) and the Trueblood family (for N.K.G.). We thank T. Kerr (UCLA) for X-ray analysis and microED structural refinement. We thank N. Vlahakis (UCLA) and D. Cascio at the UCLA-DOE X-ray and EM Structure Determination Core Facilities, which are supported by DOE Grant DE-FC02-02ER63421. These studies were supported by shared instrumentation grants from the NSF (grant no. CHE-1048804), the NIH NCRR (grant no. S10RR025631), and the NIH ORIP (grant no. Calculations were performed on the Hoffman2 cluster and the UCLA Institute of Digital Research and Education (IDRE) at UCLA and the Extreme Science and Engineering Discovery Environment (XSEDE), which is supported by the National Science Foundation (OCI-1053575). These authors contributed equally: Jiaming Ding, Sarah A. French. Department of Chemistry and Biochemistry, University of California, Los Angeles, Los Angeles, CA, USA Jiaming Ding, Sarah A. French, Christina A. Rivera, Arismel Tena Meza, Dominick C. Witkowski, K. N. Houk & Neil K. Garg Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and performed experiments and analysed experimental data. designed, performed and analysed computational studies. advised on computational findings and manuscript preparation. directed the experimental and computational investigations and prepared the manuscript with contributions from all authors; all authors contributed to discussions. These authors contributed equally: C.A.R., A.T.M. Correspondence to Neil K. Garg. The authors declare no competing interests. Nature Chemistry thanks Ignacio Funes-Ardoiz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 6–108, Tables 1–14, Experimental procedures and Computational details. Output files for reported computations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Ding, J., French, S.A., Rivera, C.A. et al. Hyperpyramidalized alkenes with bond orders near 1.5 as synthetic building blocks. Version of record: 21 January 2026 DOI: https://doi.org/10.1038/s41557-025-02055-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00089-8'>Girls are starting puberty younger — why, and what are the risks?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 10:19:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cassandra Willyard is a science journalist in Madison, Wisconsin. Search author on: PubMed Google Scholar The average age at which girls start puberty has been falling. When Lola was eight years old, she went through a massive growth spurt and started developing acne. Her mother, Elise, thought Lola was just growing fast because of genes inherited from her father. A visit to an endocrinologist in 2023 confirmed that Lola's brain was already producing hormones that had kick-started puberty. “She would have panic attacks every day at school,” says Elise, who lives in Minneapolis, Minnesota, and asked that her surname and Lola's real name be omitted. Although eight might seem young to start puberty, it's not as rare as it once was. Data show that girls around the world are entering puberty younger than before. In the 1840s, the average age of first menstruation, or menarche, was about 16 or 17; today, it's around 12. (Although some data suggest that puberty is happening earlier for boys too, the shift seems to be less pronounced.) Scientists have found a range of possible drivers for this change, with increasing body weight and obesity almost certainly playing a part. Some researchers suspect that exposure to hormone-disrupting chemicals or stress during childhood could be pushing puberty earlier, but studies have produced conflicting results. The guidelines will reconsider how to treat girls on the border between typical and ‘precocious' puberty, which has commonly been defined as before the age of eight in girls, but that some specialists argue should be younger. Research over the past few years is also making the health risks of early puberty increasingly clear. Studies have linked it to greater risk of conditions including obesity, heart disease, breast cancer, depression and anxiety. Other research suggests that children who go through puberty earlier are more likely to experience discrimination because of their race or ethnicity, or otherwise be treated differently from their peers. This might involve medications to pause the process, but also better support and puberty education for children to protect them from some of the psychological and social risks. “We want to intervene right in that moment before people start internalizing some of those feelings of being othered,” says Michael Curtis, a family social scientist at the University of Minnesota in Minneapolis. What triggers this process isn't fully understood — it's probably a complex interaction between genes and environmental factors. But the result is a hormonal cascade that leads to the release of the sex hormones oestrogen (in girls) and testosterone (in boys), which drive physical changes, including menarche. (The binary terms ‘girls' and ‘boys' are used in this article to reflect language used in studies and by interviewees.) The drop in average age of menarche from the mid-nineteenth to the mid-twentieth century is often attributed to improvements in health, such as reductions in infectious disease and malnutrition (see ‘Younger puberty'). This probably sped up growth and sexual maturation. Most researchers assumed that the timing of puberty had remained relatively stable since then. “Studies from the 1960s showed that it was kind of levelling off at 12 and a half years,” says Paul Kaplowitz, a retired paediatric endocrinologist who was at Children's National Hospital in Arlington, Virginia. In 1969, British paediatrician James Tanner and biologist William Marshall reported one of the most comprehensive studies1 of puberty's timing as part of a two-decade study at a children's home in Harpenden, UK. They observed that breast development is the first outward sign of puberty in girls and that it begins around 11 years old, on average. (For boys, the onset of puberty2 was closer to 12.) The ‘Tanner stages', which demarcate five stages of progress towards sexual maturity, became widely used in medicine and research. By the late 1980s, however, Marcia Herman-Giddens was questioning Tanner's timings. As part of her work as a physician's associate at Duke University in Durham, North Carolina, Herman-Giddens had examined thousands of girls in the United States and observed that some were developing breasts and pubic hair “way younger than the Tanner standards”, she says. Herman-Giddens and her team set out to develop benchmarks for US children. With the help of physicians from across the country, they collected data on pubertal timing from around 17,000 girls who had undergone physical examinations in physician offices between 1992 and 1993. This showed that the mean age at which breast development started was just under ten years old for white girls and nine years for Black girls. It was the first large study to suggest that puberty was beginning much earlier than Tanner had suggested, at least in the United States. Do smartphones and social media really harm teens' mental health? Do smartphones and social media really harm teens' mental health? Anders Juul, a paediatric endocrinologist at the University of Copenhagen, didn't see similar figures in Denmark and, with obesity on the rise, he suspected that US physicians had mistaken fat tissue for growing breasts. The change couldn't be attributed to increased weight, because the girls' body mass index (BMI) had not changed. “To our surprise, there were no differences in obesity between the old cohort and the more contemporary cohort,” he says. A 2020 meta-analysis of 30 studies6 — and the most recent comprehensive review of global trends — revealed that the median age of breast development fell by almost three months each decade between 1977 and 2013. The United States had the earliest onset (a median of 8.8–10.3 years), Africa had the latest (10.1–13.2 years) and Europe and Asia fell in between. An update7 to this study, presented at a 2025 European endocrinology meeting, shows that the trend has continued. Researchers don't know whether puberty will continue occurring even earlier or at what point it might hit a biological floor. Globally, physicians now typically consider puberty onset between the ages of 8 and 13 in girls as in the normal range. For years, researchers have been trying to work out why puberty is starting earlier. Globally, obesity rates have risen from around 2% of children and adolescents in 1990 to around 8% in 2022, and from around 11% to more than 20% in the United States, according to the World Health Organization. A 2022 study8 of nearly 130,000 US children found a clear association between obesity and earlier puberty in children. “It is beyond any doubt that obesity is a major driver,” says geneticist John Perry, who studies growth and reproduction at the University of Cambridge, UK. One way in which body weight influences puberty is through leptin, a hormone produced by fat cells. This can interact with the brain circuits that control development and reproduction. “We don't think that leptin initiates puberty,” Kaplowitz says. “But it's important for puberty to progress.” Other researchers, including Juul, suspect that hormone-disrupting chemicals in the environment could be at least partly responsible for advancing puberty. They point in particular to chemicals found in plastics, such as phthalates, forever chemicals called PFAS and synthetic fragrances, all of which gained widespread use in the twentieth century. These compounds can interfere with hormones by mimicking them or disrupting their activity. But results are inconsistent, and proving a link to any single substance has been incredibly difficult. “There haven't been really any good studies that have shown this in a way that everybody says, ‘yep, that's the answer',” Kaplowitz says. A third possible piece of the puzzle is psychological stress. Some research suggests that girls who encounter stressors such as domestic violence, abuse, poverty and discrimination are more likely to start puberty at a younger age than those who do not. One 2022 longitudinal study9 found that physical or emotional abuse in early life was linked to earlier menarche in US girls. But it might interact with excess body weight, says Lauren Houghton, an epidemiologist at Columbia University in New York City. Her unpublished research suggests that girls who experience high levels of stress, have elevated stress hormones and a high BMI start developing breasts, on average, seven months earlier than do girls who experience low levels of stress and have a low BMI. Stress might also be a reason why more girls entered puberty early during the COVID-19 pandemic than in the years preceding it. Soon after the pandemic began in 2020, paediatric endocrinologists in Italy noticed that the number of referrals for precocious puberty soared. They later reported10 that 41% of those referred in 2020 met the criteria for the condition, compared with 26% in 2019. Studies from other countries have revealed a similar phenomenon and some suggest that puberty progressed faster too11. We saw “truncated and shorter puberty”, says Louise Greenspan, a paediatric endocrinologist at Kaiser Permanente San Francisco in California. Do smartphones and social media really harm teens' mental health? Why kids need to take more risks: science reveals the benefits of wild, free play Untangling the connection between dopamine and ADHD Illuminating how the bird inner retina works without oxygen solves a 350-year-old structural mystery ADHD is on the rise, but why? Trump one year on: How six US researchers plan to protect science amid chaos and cuts ADHD is on the rise, but why? Untangling the connection between dopamine and ADHD Job Title: Locum Associate or Senior Editor (Chemical Biology / Biological Chemistry), Nature Chemistry Company: Nature Portfolio Contract Type: Fu... Postdoc project integrates biochemical, cell biology and novel chemical methods to study molecular mechanisms of canonical and noncanonical autophagy The Health + Life Science Alliance is looking for outstanding early career postdocs for bridging projects between partner institutions. The School of Science and Engineering (SSE) at theCUHK-Shenzhen sincerely invites applications for Tenure/Teaching/Research Stream faculty positions The School of Science and Engineering (SSE) at theCUHK-Shenzhen sincerely invites applications for Tenure/Teaching/Research Stream faculty positions Do smartphones and social media really harm teens' mental health? Why kids need to take more risks: science reveals the benefits of wild, free play An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/california-wildfire-smoke-linked-to-increased-autism-diagnoses-new-study/'>California wildfire smoke linked to increased autism diagnoses, new study finds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>California wildfire smoke linked to increased autism diagnoses, new study finds Children born to mothers who were exposed to smoke in southern California showed increased rates of autism, although the reason why is unclear The causes of autism are not fully known and likely multifaceted, but the new research builds on existing evidence that air pollution may be tied to autism. Pregnant women who were in their third trimester and exposed to as few as one to five smoke days were about 11 percent more likely to have a child who was diagnosed as autistic by age five than those who saw no smoke days. The more smoke days that mothers were exposed to, the higher the likelihood that their children would be diagnosed as autistic: women who were exposed to between six and 10 smoke days were 12 percent more likely to have a child who received such a diagnosis by age five, while this was 23 percent more likely among those who were exposed to more than 10 smoke days. “This is one of the first large population-based studies to specifically examine prenatal wildfire smoke exposure and autism risk,” says Mostafijur Rahman, an assistant professor of environmental health sciences at Tulane University and an author of the new study. “Our findings suggest that wildfire smoke exposure during sensitive periods of pregnancy—particularly late pregnancy—may be associated with an increased risk of autism diagnosis in children.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Importantly, the study doesn't identify a direct causal link between autism and wildfire smoke, Rahman says. Rather the study “highlights wildfire smoke as a potentially modifiable environmental risk factor that may contribute to risk in combination with other factors,” he says. Mandell notes, however, that some children of mothers who experienced higher concentrations of wildfire smoke in her third trimester and didn't move house during the study period didn't show higher rates of autism—which is not the “dose-response gradient that one might expect,” he says. Autism and its causes have been a central focus of the Trump administration and Secretary of Health and Human Services Robert F. Kennedy, Jr., who has, in the past, claimed that autism was most likely caused by a variety of environmental exposures, not all of which have been backed by solid science. According to the U.S. Centers for Disease Control and Prevention, around one in 31 children are diagnosed as autistic before age nine. The new study jibes with past research showing children whose mothers were exposed to high rates of fine particulate pollution, as well as diesel exhaust and mercury, were more likely to be diagnosed as autistic than the children of those who breathed cleaner air. Wildfires have become an ever-present risk in the U.S., especially for Californians. “As wildfires become more frequent and intense due to climate change, understanding their potential long-term health impacts is increasingly important,” Rahman says. Jackie Flynn Mogensen is a breaking news reporter at Scientific American. Before joining SciAm, she was a science reporter at Mother Jones, where she received a National Academies Eric and Wendy Schmidt Award for Excellence in Science Communications in 2024. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260120000333.htm'>Stanford scientists found a way to regrow cartilage and stop arthritis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 04:56:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A study led by Stanford Medicine researchers has found that an injection blocking a protein linked to aging can reverse the natural loss of knee cartilage in older mice. The same treatment also stopped arthritis from developing after knee injuries that resemble ACL tears, which are common among athletes and recreational exercisers. Researchers note that an oral version of the treatment is already being tested in clinical trials aimed at treating age-related muscle weakness. Human cartilage samples taken from knee replacement surgeries also responded positively. When treated, the tissue began forming new, functional cartilage. Together, the findings suggest that cartilage lost due to aging or arthritis may one day be restored using either a pill or a targeted injection. If successful in people, such treatments could reduce or even eliminate the need for knee and hip replacement surgery. Current treatments focus on managing pain or replacing damaged joints surgically. There are no approved drugs that can slow or reverse the underlying cartilage damage. In mice, higher levels of 15-PGDH are linked to declining muscle strength with age. Blocking the enzyme using a small molecule boosted muscle mass and endurance in older animals. The protein has also been connected to regeneration in bone, nerve, and blood cells. In this case, chondrocytes change how their genes behave, shifting into a more youthful state without relying on stem cells. "This is a new way of regenerating adult tissue, and it has significant clinical promise for treating arthritis due to aging or injury," said Helen Blau, PhD, professor of microbiology and immunology. "We were looking for stem cells, but they are clearly not involved. Wang is now an assistant professor at the Sanford Burnham Institute in San Diego. "Millions of people suffer from joint pain and swelling as they age," Bhutani said. Until now, there has been no drug that directly treats the cause of cartilage loss. But this gerozyme inhibitor causes a dramatic regeneration of cartilage beyond that reported in response to any other drug or intervention." The human body contains three main types of cartilage. Fibrocartilage is dense and tough, helping absorb shock in places like the spaces between spinal vertebrae. This type, also called articular cartilage, is the form most commonly damaged in osteoarthritis. Osteoarthritis develops when joints are stressed by aging, injury, or obesity. Chondrocytes begin releasing inflammatory molecules and breaking down collagen, the main structural protein in cartilage. Inflammation then leads to swelling and pain, which are hallmarks of the disease. Under normal conditions, articular cartilage has very limited ability to regenerate. While some stem or progenitor cells capable of forming cartilage have been identified in bone, similar cells have not been successfully found within articular cartilage itself. Earlier research from Blau's lab showed that prostaglandin E2 is essential for muscle stem cell function. By blocking 15-PGDH or increasing prostaglandin E2 levels, researchers previously supported the repair of damaged muscle, nerve, bone, colon, liver, and blood cells in young mice. Researchers then injected older mice with a small molecule that inhibits 15-PGDH. In both cases, cartilage that had become thin and dysfunctional with age thickened across the joint surface. "Cartilage regeneration to such an extent in aged mice took us by surprise," Bhutani said. The team observed similar benefits in mice with knee injuries resembling ACL tears, which often occur during sports involving sudden stopping, pivoting, or jumping. Although such injuries can be surgically repaired, about half of affected people develop osteoarthritis in the injured joint within 15 years. Mice that received twice-weekly injections of the gerozyme inhibitor for four weeks after injury were far less likely to develop osteoarthritis. In contrast, animals given a control treatment had double the levels of 15-PGDH compared with uninjured mice and developed osteoarthritis within four weeks. "Interestingly, prostaglandin E2 has been implicated in inflammation and pain," Blau said. "But this research shows that, at normal biological levels, small increases in prostaglandin E2 can promote regeneration." Closer analysis showed that chondrocytes in older mice expressed more genes linked to inflammation and the conversion of cartilage into bone, along with fewer genes involved in cartilage formation. One group of chondrocytes that produced 15-PGDH and cartilage-degrading genes dropped from 8% to 3%. Another group associated with fibrocartilage formation declined from 16% to 8%. A third population, which did not produce 15-PGDH and instead expressed genes tied to hyaline cartilage formation and maintenance of the extracellular matrix, rose from 22% to 42%. The researchers also tested cartilage taken from patients undergoing total knee replacement for osteoarthritis. "The mechanism is quite striking and really shifted our perspective about how tissue regeneration can occur," Bhutani said. "It's clear that a large pool of already existing cells in cartilage are changing their gene expression patterns. Blau added, "Phase 1 clinical trials of a 15-PGDH inhibitor for muscle weakness have shown that it is safe and active in healthy volunteers. Our hope is that a similar trial will be launched soon to test its effect in cartilage regeneration. Blau, Bhutani, and other co-authors are inventors on patent applications held by Stanford University related to 15-PGDH inhibition in cartilage and tissue rejuvenation, which are licensed to Epirium Bio. New Drug Slashes Dangerous Blood Fats by Nearly 40% in First Human Trial Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260120000328.htm'>Patients tried everything for depression then this implant changed their lives</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-21 04:46:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Many people improve after trying a few treatments, but for as many as one-third of patients, standard antidepressants or psychotherapy do not provide enough relief. This condition, known as treatment-resistant depression, can persist for years or even decades. New research now suggests that a small implanted device may offer meaningful and long-lasting improvement for people with the most severe forms of the illness. Scientists at Washington University School of Medicine in St. Louis led a large, multicenter clinical trial to evaluate this approach. The researchers found that a device designed to stimulate the vagus nerve was linked to sustained improvements in depressive symptoms, daily functioning, and overall quality of life. The participants in the study had lived with depression for an average of 29 years and had already tried about 13 treatments without success. These included intensive options such as electroconvulsive therapy and transcranial magnetic stimulation, highlighting just how difficult their condition had been to treat. "We believe the sample in this trial represents the sickest treatment-resistant depressed patient sample ever studied in a clinical trial," said lead author Charles Conway, MD, a professor of psychiatry and director of the WashU Medicine Treatment Resistant Mood Disorders Center. "There is a dire need to find effective treatments for these patients, who often have no other options. With this kind of chronic, disabling illness, even a partial response to treatment is life-altering, and with vagus nerve stimulation we're seeing that benefit is lasting." The RECOVER study was designed to test whether adding vagus nerve stimulation (VNS) to ongoing care could improve outcomes for people with treatment-resistant depression. The study is collecting long-term data on mood, daily function, and quality of life in people with severe treatment-resistant depression. Because many private insurers follow CMS decisions, approval could make the treatment accessible to far more patients, as cost has been a major barrier. Nearly 500 patients were enrolled across 84 locations in the United States. Researchers tracked changes in depression severity, quality of life, and everyday functioning. Conway emphasized that even modest improvements can dramatically change a person's life. Severe depression can leave people feeling "paralyzed by life," unable to manage basic daily activities and at higher risk of hospitalization or early death. Earlier findings from the blinded first year of the trial showed that patients with activated devices spent more time with improved mood, better functioning, and higher quality of life than those whose devices were not active. They also examined whether some patients who did not improve in the first year might respond later with continued treatment. Out of 214 patients who received active treatment from the beginning, about 69%, or 147 people, showed a meaningful response at one year in at least one outcome measure. Among those who benefited at 12 months, more than 80% maintained or improved their results by the two-year mark across measures of depression, quality of life, and daily functioning. The researchers also found that more than 20% of treated patients, or 39 people, were in remission after 24 months. This means their symptoms had eased enough for them to function normally in daily life, a result Conway described as especially notable. "We were shocked that one in five patients was effectively without depressive symptoms at the end of two years," he said. These results are highly atypical, as most studies of markedly treatment-resistant depression have very poor sustainability of benefit, certainly not at two years. Conway has received research support from the American Foundation for Suicide Prevention, Assurex Health, August Busch IV Foundation, Barnes-Jewish Hospital Foundation, LivaNova, National Institute of Mental Health, and the Taylor Family Institute for Innovative Psychiatric Research. He has also served as a consultant for LivaNova. Note: Content may be edited for style and length. Africa's Forests Are No Longer Absorbing Carbon, Scientists Warn New Drug Slashes Dangerous Blood Fats by Nearly 40% in First Human Trial Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            