
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-13</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64968838/laser-powered-lightsail-interstellar-travel/'>To Unlock Interstellar Travel, Scientists Are Building the World's Most Powerful Laser Beam</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 15:29:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A lightsail built in one day could travel at 20 percent the speed of light. We may earn commission if you buy from a link. To travel beyond our solar system to destinations like Alpha Centauri, our nearest neighboring star system, it would take many human lifetimes with current technology. It began in 2016 and aims to design small, lightweight spacecraft powered by light that can reach Alpha Centauri in just over 20 years. If this type of machine comes to fruition, it will vastly outpace today's fastest spacecraft, like the Helios probes that NASA and West Germany collaborated on in the 1970s. Those older probes would take nearly 20,000 years to travel 4.4 light years (roughly 25 trillion miles) to Alpha Centauri, scientists estimate. But a new featherweight probe would give scientists unprecedented access to the farthest reaches of space, allowing them to capture close-up images of stellar objects like exoplanets for the first time in history, says Harry Atwater, Ph.D., a professor of applied physics and materials science at the California Institute of Technology. Formerly, Atwater was the lightsail research director for Breakthrough Starshot; he is continuing this work independently at Caltech. “There's a lot of really transformative work going on in astronomy right now using large telescopes [. to do remote observations of distant stars and exoplanets,” Atwater says. “But there are certainly new things that we [can] learn as a result of actually going to them.” One way to lighten your spacecraft is to do away with rocket fuel altogether and rely on a type of weightless fuel instead: light. For this project, scientists supporting Breakthrough Starshot are working to develop what they call a “photon engine.” If successful, the device will create a powerful light beam that can propel a swarm of future Starshot craft into deep space. This may seem counterintuitive at first, because the light from an everyday laser pointer or flashlight could never propel a feather let alone a spacecraft. But when scaled to high enough intensities, a laser beam can act similarly to wind propelling a sailboat across water. Instead of absorbing solar power, the panels would be composed of many densely packed lasers that can emit 45 watts each. He stresses that this photon engine would not be hazardous to unsuspecting passersby or create a threat to national security in the form of a weapon. “If you were flying in an airplane above [the laser field] within the Earth's atmosphere, you would experience something that would be roughly a few times higher than theintensity of sunlight, but not enough to burn yourself,” Atwater says. To achieve the kind of acceleration that these craft will need to reach an interstellar target like Alpha Centauri in just about 20 years, they not only need a strong power source like a souped-up laser beam, but also must weigh no more than 1 gram per component—that includes the lightsail itself and a microchip payload. In this case, the sail must be about 200 nanometers thick, which is a thousand times thinner than a human hair. Building these types of sails is a daunting task that can be expensive and time intensive. To do so, the team used AI optimization techniques to design a scalable lightsail that could be manufactured in just one day and at a fraction of the standard cost. “The two main things that we were going for, performance and cost, don't necessarily go hand in hand,” Norte says. “This is where the machine learning algorithm had to make choices about what to do, and what it came up with was something that looked like a bunch of potatoes.” This shape was confusing to him at first, but Norte and the research team later realized that it imitates a 200-year-old tiling technique called a pentagonal lattice. After nearly a decade of work, a full-fledged photon engine is still far from reality, but Atwater says scientists have worked out most of the technical kinks. Still, actually launching these craft into interstellar space remains a distant goal. Sarah is a science and technology journalist based in Boston interested in how innovation and research intersect with our daily lives. She has written for a number of national publications and covers innovation news at Inverse. This AI Model Could One Day Read Your Mind Scientists Are Learning How to Turn Off Pain Our Universe May Be a Simulation, Scientist Says</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41577-025-01192-z'>Finding and filling the knowledge gaps in mechanisms of T cell-mediated TB immunity to inform vaccine design</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 14:28:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Immunology (2025)Cite this article Metrics details Mycobacterium tuberculosis, the bacterium that causes tuberculosis (TB), results in more human mortality than any other single pathogen, in part because of the lack of an effective vaccine. Although T cells are essential for immunity to TB, the mechanisms that provide protective immunity are poorly understood. In this Review, we describe current gaps in our knowledge about T cell-mediated immune responses to M. tuberculosis and discuss how recent technologies, including multiphoton intravital microscopy, spatial multiomics and high-resolution in vivo analyses of cell–cell interactions, may be used to gain insights that can inform the design of T cell-targeted TB vaccines. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Darrah, P. A. et al. Prevention of tuberculosis in macaques after intravenous BCG immunization. Google Scholar Dijkman, K. et al. Prevention of tuberculosis infection and disease by local BCG in repeatedly exposed rhesus macaques. Google Scholar Hansen, S. G. et al. Prevention of tuberculosis in rhesus macaques by a cytomegalovirus-based vaccine. Google Scholar Dijkman, K. et al. A protective, single-visit TB vaccination regimen by co-administration of a subunit vaccine with BCG. NPJ Vaccines 8, 66 (2023). Google Scholar Woodworth, J. S. et al. A Mycobacterium tuberculosis-specific subunit vaccine that provides synergistic immunity upon co-administration with Bacillus Calmette-Guérin. Google Scholar Nemeth, J. et al. Contained Mycobacterium tuberculosis infection induces concomitant and heterologous protection. PLoS Pathog. Google Scholar Coussens, A. K. et al. Classification of early tuberculosis states to guide research for improved care and prevention: an international Delphi consensus exercise. Lancet Respir. Google Scholar Garcia-Basteiro, A. L. et al. A global tuberculosis dictionary: unified terms and definitions for the field of tuberculosis. Lancet Glob. Google Scholar Cohen, S. B. et al. Alveolar macrophages provide an early Mycobacterium tuberculosis niche and initiate dissemination. Cell Host Microbe 24, 439–446.e434 (2018). Google Scholar Pisu, D. et al. Single cell analysis of M. tuberculosis phenotype and macrophage lineages in the infected lung. Google Scholar Wolf, A. J. et al. Mycobacterium tuberculosis infects dendritic cells with high frequency and impairs their function in vivo. Myeloid DCs are key reservoirs for M. tuberculosis in the lungs and lymph nodes, while strategically impairing antigen presentation to CD4+ T cells, thereby promoting persistent infection. Google Scholar Lee, J. et al. CD11cHi monocyte-derived macrophages are a major cellular compartment infected by Mycobacterium tuberculosis. PLoS Pathog. Google Scholar Zheng, W. et al. Mycobacterium tuberculosis resides in lysosome-poor monocyte-derived lung cells during chronic infection. PLoS Pathog. Google Scholar Lai, R., Williams, T., Rakib, T., Lee, J. & Behar, S. M. Heterogeneity in lung macrophage control of Mycobacterium tuberculosis is modulated by T cells. Google Scholar Kwan, C. K. & Ernst, J. D. HIV and tuberculosis: a deadly human syndemic. Google Scholar Jasenosky, L. D., Scriba, T. J., Hanekom, W. A. & Goldfeld, A. E. T cells and adaptive immunity to Mycobacterium tuberculosis in humans. Google Scholar Flynn, J. L. & Chan, J. Immune cell interactions in tuberculosis. Google Scholar Nathan, A. et al. Multimodally profiling memory T cells from a tuberculosis cohort identifies cell state associations with demographics, environment and disease. Google Scholar Gideon, H. P. et al. Multimodal profiling of lung granulomas in macaques reveals cellular correlates of tuberculosis control. Advanced single-cell analyses, coupled with detailed in vivo measurements of Mtb granulomas, identified the cellular and transcriptional characteristics associated with an effective host immune response to TB in NHPs. Google Scholar Bromley, J. D. et al. CD4+ T cells re-wire granuloma cellularity and regulatory networks to promote immunomodulation following Mtb reinfection. Google Scholar Chackerian, A. A., Alt, J. M., Perera, T. V., Dascher, C. C. & Behar, S. M. Dissemination of Mycobacterium tuberculosis is influenced by host factors and precedes the initiation of T-cell immunity. Google Scholar Wolf, A. J. et al. Initiation of the adaptive immune response to Mycobacterium tuberculosis depends on antigen production in the local lymph node, not the lungs. Google Scholar Miller, E. A. & Ernst, J. D. Anti-TNF immunotherapy and tuberculosis reactivation: another mechanism revealed. Google Scholar Mathe, G. et al. Intravenous B.C.G. in monkeys and man. Google Scholar Crispen, R. G. Immunotherapy with intravenous B.C.G. Google Scholar Davis, J. M. & Ramakrishnan, L. The role of the granuloma in expansion and dissemination of early tuberculous infection. Prompted reconsideration of the dogma that granulomas simply contain mycobacterial infection by demonstrating the role of ongoing cell recruitment in enabling cell-to-cell spread and progression of infection. Google Scholar Cronan, M. R. et al. A non-canonical type 2 immune response coordinates tuberculous granuloma formation and epithelialization. Google Scholar Cohen, S. B., Gern, B. H. & Urdahl, K. B. The tuberculous granuloma and preexisting immunity. Google Scholar Sawyer, A. J. et al. Spatial mapping reveals granuloma diversity and histopathological superstructure in human tuberculosis. Google Scholar Kauffman, K. D. et al. Defective positioning in granulomas but not lung-homing limits CD4 T-cell interactions with Mycobacterium tuberculosis-infected macrophages in rhesus macaques. Mucosal Immunol. Google Scholar Carow, B. et al. Spatial and temporal localization of immune transcripts defines hallmarks and diversity in the tuberculosis granuloma. Google Scholar McCaffrey, E. F. et al. The immunoregulatory landscape of human tuberculosis granulomas. TB granulomas of individuals with active TB were mapped using multiplexed ion beam imaging by time of flight, identifying three spatial modules including the myeloid core enriched with IDO1 and PD-L1, lymphocytic cuff and stromal compartment revealing novel functional–spatial relationships. Google Scholar Esaulova, E. et al. The immune landscape in tuberculosis reveals populations linked to disease and latency. Cell Host Microbe 29, 165–178.e168 (2021). Google Scholar Magoulopoulou, A. et al. Spatial resolution of Mycobacterium tuberculosis bacteria and their surrounding immune environments based on selected key transcripts in mouse lungs. Google Scholar Ernst, J. D., Cornelius, A., Desvignes, L., Tavs, J. & Norris, B. Limited antimycobacterial efficacy of epitope peptide administration despite enhanced antigen-specific CD4 T cell activation. Szabo, P. A., Miron, M. & Farber, D. L. Location, location, location: tissue resident memory T cells in mice and humans. Google Scholar Srivastava, S. & Ernst, J. D. Cutting edge: direct recognition of infected cells by CD4 T cells is required for control of intracellular Mycobacterium tuberculosis in vivo. Direct recognition of infected cells by CD4+ T cells is essential for controlling intracellular Mtb, as long-range cytokine diffusion alone is insufficient to reduce bacterial burden, highlighting a critical mechanism of host defence against TB. Google Scholar Muller, A. J. et al. CD4+ T cells rely on a cytokine gradient to control intracellular pathogens beyond sites of antigen presentation. Google Scholar Lin, K. Y. et al. Ectopic expression of vascular cell adhesion molecule-1 as a new mechanism for tumor immune evasion. Cancer Res. Google Scholar Tian, C. et al. Motility and tumor infiltration are key aspects of invariant natural killer T cell anti-tumor function. Google Scholar Nunes-Alves, C. et al. In search of a new paradigm for protective immunity to TB. Google Scholar Winchell, C. G. et al. CD8+ lymphocytes are critical for early control of tuberculosis in macaques. The study investigates the role of innate and conventional CD8 immunity in TB, revealing that CD8αα+ lymphocytes have a key role in the early suppression of Mtb growth in macaque lungs. Google Scholar Cai, Y. et al. Single-cell immune profiling reveals functional diversity of T cells in tuberculous pleural effusion. Google Scholar Schurr, E. et al. Mycobacterium tuberculosis resisters despite HIV exhibit activated T cells and macrophages in their pulmonary alveoli. Poly-cytotoxic T cells and alveolar macrophages contribute to natural resistance to Mtb infection in individuals with HIV. Google Scholar Simonson, A. W. et al. CD4 T cells and CD8α+ lymphocytes are necessary for intravenous BCG-induced protection against tuberculosis in macaques. This article presents evidence that CD8+ and CD4+ T cells have a crucial role in mediating protection in NHPs vaccinated with intravenous BCG. Google Scholar Patankar, Y. R. et al. Limited recognition of Mycobacterium tuberculosis-infected macrophages by polyclonal CD4 and CD8 T cells from the lungs of infected mice. Mucosal Immunol. Google Scholar Carpenter, S. M., Nunes-Alves, C., Booty, M. G., Way, S. S. & Behar, S. M. A higher activation threshold of memory CD8+ T cells has a fitness cost that is modified by TCR affinity during tuberculosis. PLoS Pathog. Google Scholar Billeskov, R., Vingsbo-Lundberg, C., Andersen, P. & Dietrich, J. Induction of CD8 T cells against a novel epitope in TB10.4: correlation with mycobacterial virulence and the presence of a functional region of difference-1. Google Scholar Yang, J. D. et al. Mycobacterium tuberculosis-specific CD4+ and CD8+ T cells differ in their capacity to recognize infected macrophages. PLoS Pathog. Google Scholar Sutiwisesak, R. et al. A natural polymorphism of Mycobacterium tuberculosis in the esxH gene disrupts immunodomination by the TB10.4-specific CD8 T cell response. PLoS Pathog. Google Scholar Nyendak, M. et al. Adenovirally-induced polyfunctional T cells do not necessarily recognize the infected target: lessons from a phase I trial of the AERAS-402 vaccine. Google Scholar Behar, S. M., Martin, C. J., Nunes-Alves, C., Divangahi, M. & Remold, H. G. Lipids, apoptosis, and cross-presentation: links in the chain of host defense against Mycobacterium tuberculosis. Microbes Infect. Google Scholar Athman, J. J. et al. Mycobacterium tuberculosis membrane vesicles inhibit T cell activation. Google Scholar Caouaille, M., Hudrisier, D. & Neyrolles, O. Mycobacterial d-serine impairs TB control. Google Scholar Cheng, H. et al. Mycobacterium tuberculosis produces d-serine under hypoxia to limit CD8+ T cell-dependent immunity in mice. Google Scholar Mahon, R. N. et al. Mycobacterium tuberculosis cell wall glycolipids directly inhibit CD4+ T-cell activation by interfering with proximal T-cell-receptor signaling. Google Scholar Mahon, R. N. et al. Mycobacterium tuberculosis ManLAM inhibits T-cell-receptor signaling by interference with ZAP-70, Lck and LAT phosphorylation. Cell Immunol. Google Scholar Mwebaza, I. et al. Impact of Mycobacterium tuberculosis glycolipids on the CD4+ T cell-macrophage immunological synapse. Google Scholar Sande, O. J. et al. Mannose-capped lipoarabinomannan from Mycobacterium tuberculosis induces CD4+ T cell anergy via GRAIL. Google Scholar Collins, J. M. et al. Tryptophan catabolism reflects disease activity in human tuberculosis. JCI Insight 5, e137131 (2020). Google Scholar Gautam, U. S. et al. In vivo inhibition of tryptophan catabolism reorganizes the tuberculoma and augments immune-mediated control of Mycobacterium tuberculosis. Natl Acad. Inhibiting IDO activity in tuberculomas in macaques induces the reorganization of the granulomata, with T cells otherwise present in the peripheral region of lesions being able to gain greater access to the core region. This can potentially improve TB control by enhancing immune-mediated control and reducing bacterial burden and pathology. Google Scholar Desvignes, L. & Ernst, J. D. Interferon-gamma-responsive nonhematopoietic cells regulate the immune response to Mycobacterium tuberculosis. Google Scholar Ahmed, A. & Vyakarnam, A. Emerging patterns of regulatory T cell function in tuberculosis. Google Scholar Kotov, D. I. et al. Early cellular mechanisms of type I interferon-driven susceptibility to tuberculosis. Plasmacytoid DCs drive Mtb pathogenesis by producing type I interferons that impair interstitial macrophage responses to IFNγ, promoting bacterial replication, neutrophil recruitment and active tuberculosis disease. Google Scholar Ji, D. X. et al. Type I interferon-driven susceptibility to Mycobacterium tuberculosis is mediated by IL-1Ra. Google Scholar Berry, M. P. et al. An interferon-inducible neutrophil-driven blood transcriptional signature in human tuberculosis. Google Scholar Moreira-Teixeira, L., Mayer-Barber, K., Sher, A. & O'Garra, A. Type I interferons in tuberculosis: foe and occasionally friend. Google Scholar Lazar-Molnar, E. et al. Programmed death-1 (PD-1)-deficient mice are extraordinarily sensitive to tuberculosis. Natl Acad. Google Scholar Kauffman, K. D. et al. PD-1 blockade exacerbates Mycobacterium tuberculosis infection in rhesus macaques. Google Scholar Jayaraman, P. et al. TIM3 mediates T cell exhaustion during Mycobacterium tuberculosis infection. PLoS Pathog. Google Scholar Zhu, C. et al. An IL-27/NFIL3 signalling axis drives Tim-3 and IL-10 expression and T-cell dysfunction. Google Scholar Torrado, E. et al. Interleukin 27R regulates CD4+ T cell phenotype and impacts protective immunity during Mycobacterium tuberculosis infection. Google Scholar Panda, S. et al. Identification of differentially recognized T cell epitopes in the spectrum of tuberculosis infection. Google Scholar Lindestam Arlehamn, C. S. et al. A quantitative analysis of complexity of human pathogen-specific CD4 T cell responses in healthy M. tuberculosis infected South Africans. PLoS Pathog. Google Scholar Lewinsohn, D. M. et al. Human Mycobacterium tuberculosis CD8 T cell antigens/epitopes identified by a proteomic peptide library. PLoS ONE 8, e67016 (2013). Google Scholar Samstein, M. et al. Essential yet limited role for CCR2+ inflammatory monocytes during Mycobacterium tuberculosis-specific T cell priming. Google Scholar Srivastava, S. & Ernst, J. D. Cell-to-cell transfer of M. tuberculosis antigens optimizes CD4 T cell priming. Cell Host Microbe 15, 741–752 (2014). During Mtb infection, inefficient antigen presentation by infected migratory DCs is bypassed through the release and transfer of bacterial antigens to uninfected resident lymph node DCs, enabling optimal CD4+ T cell priming and enhancing immune control of TB. Google Scholar Delamarre, L., Pack, M., Chang, H., Mellman, I. & Trombetta, E. S. Differential lysosomal proteolysis in antigen-presenting cells determines antigen fate. Google Scholar Commandeur, S. et al. An unbiased genome-wide Mycobacterium tuberculosis gene expression approach to discover antigens targeted by human T cells expressed during pulmonary infection. Google Scholar Bold, T. D., Banaei, N., Wolf, A. J. & Ernst, J. D. Suboptimal activation of antigen-specific CD4+ effector cells enables persistence of M. tuberculosis in vivo. PLoS Pathog. Google Scholar Rogerson, B. J. et al. Expression levels of Mycobacterium tuberculosis antigen-encoding genes versus production levels of antigen-specific T cells during stationary level lung infection in mice. Google Scholar Banchereau, J. & Steinman, R. M. Dendritic cells and the control of immunity. Google Scholar Rothchild, A. C. et al. Alveolar macrophages generate a noncanonical NRF2-driven transcriptional response to Mycobacterium tuberculosis in vivo. Google Scholar Eisenbarth, S. C. Dendritic cell subsets in T cell programming: location dictates function. Google Scholar Shah, J. A., Lindestam Arlehamn, C. S., Horne, D. J., Sette, A. & Hawn, T. R. Nontuberculous mycobacteria and heterologous immunity to tuberculosis. Google Scholar Simeone, R. et al. Cytosolic access of Mycobacterium tuberculosis: critical impact of phagosomal acidification control and demonstration of occurrence in vivo. PLoS Pathog. Google Scholar Divangahi, M. et al. Mycobacterium tuberculosis evades macrophage defenses by inhibiting plasma membrane repair. Google Scholar Woodworth, J. S., Fortune, S. M. & Behar, S. M. Bacterial protein secretion is required for priming of CD8+ T cells specific for the Mycobacterium tuberculosis antigen CFP10. Google Scholar Woodworth, J. S. & Behar, S. M. Mycobacterium tuberculosis-specific CD8+ T cells and their role in immunity. Google Scholar Lewinsohn, D. A. et al. Immunodominant tuberculosis CD8 antigens preferentially restricted by HLA-B. PLoS Pathog. Google Scholar Lewinsohn, D. M. et al. Secreted proteins from Mycobacterium tuberculosis gain access to the cytosolic MHC class-I antigen-processing pathway. Google Scholar Tobian, A. A., Canaday, D. H., Boom, W. H. & Harding, C. V. Bacterial heat shock proteins promote CD91-dependent class I MHC cross-presentation of chaperoned peptide to CD8+ T cells by cytosolic mechanisms in dendritic cells versus vacuolar mechanisms in macrophages. Google Scholar Grotzke, J. E., Siler, A. C., Lewinsohn, D. A. & Lewinsohn, D. M. Secreted immunodominant Mycobacterium tuberculosis antigens are processed by the cytosolic pathway. Google Scholar Grotzke, J. E. et al. The Mycobacterium tuberculosis phagosome is a HLA-I processing competent organelle. PLoS Pathog. Google Scholar Mazzaccaro, R. J. et al. Major histocompatibility class I presentation of soluble antigen facilitated by Mycobacterium tuberculosis infection. Natl Acad. Google Scholar Mott, D. et al. High bacillary burden and the ESX-1 type VII secretion system promote MHC class I presentation by Mycobacterium tuberculosis-infected macrophages to CD8 T cells. Google Scholar Schaible, U. E. et al. Apoptosis facilitates antigen presentation to T lymphocytes through MHC-I and CD1 in tuberculosis. Google Scholar Tzelepis, F. et al. Annexin1 regulates DC efferocytosis and cross-presentation during Mycobacterium tuberculosis infection. Article PubMed Google Scholar Smith, V. L., Cheng, Y., Bryant, B. R. & Schorey, J. S. Exosomes function in antigen presentation during an in vivo Mycobacterium tuberculosis infection. Google Scholar Baptista, A. P. et al. The chemoattractant receptor Ebi2 drives intranodal naive CD4+ T cell peripheralization to promote effective adaptive immunity. Google Scholar Kapoor, V. N. et al. Gremlin 1+ fibroblastic niche maintains dendritic cell homeostasis in lymphoid tissues. Google Scholar Leal, J. M. et al. Innate cell microenvironments in lymph nodes shape the generation of T cell responses during type I inflammation. Google Scholar Daniel, L. et al. Stromal structure remodeling by B lymphocytes limits T cell activation in lymph nodes of Mycobacterium tuberculosis-infected mice. Google Scholar Musvosvi, M. et al. T cell receptor repertoires associated with control and disease progression following Mycobacterium tuberculosis infection. Using both deep TCR sequencing and genome-wide antigen screen, the authors identified peptides targeted by T cell similarity groups enriched either in controllers or in progressors. Google Scholar Chugh, S., Bahal, R. K., Dhiman, R. & Singh, R. Antigen identification strategies and preclinical evaluation models for advancing tuberculosis vaccine development. NPJ Vaccines 9, 57 (2024). Google Scholar Comas, I. et al. Human T cell epitopes of Mycobacterium tuberculosis are evolutionarily hyperconserved. The hyperconservation of human T cell epitopes in Mtb suggests strong purifying selection, indicating that immune recognition may have a role in the persistence and evolutionary success of the pathogen. Google Scholar Ogongo, P. et al. Rare variable M. tuberculosis antigens induce predominant Th17 responses in human infection. Preprint at bioRxiv https://doi.org/10.1101/2024.03.05.583634 (2024). Rare variable Mtb antigens preferentially induce IL-17-producing TH17 cells, suggesting their potential as vaccine targets to enhance protective immunity in individuals already exposed to TB. Lindestam Arlehamn, C. S., Lewinsohn, D., Sette, A. & Lewinsohn, D. Antigens for CD4 and CD8 T cells in tuberculosis. Cold Spring Harb. Google Scholar Sakai, S. et al. CD4 T cell-derived IFN-gamma plays a minimal role in control of pulmonary Mycobacterium tuberculosis infection and must be actively repressed by PD-1 to prevent lethal disease. PLoS Pathog. Google Scholar Coppola, M. et al. New genome-wide algorithm identifies novel in-vivo expressed Mycobacterium Tuberculosis antigens inducing human T-cell responses with classical and unconventional cytokine profiles. Google Scholar Khader, S. A. et al. IL-23 and IL-17 in the establishment of protective pulmonary CD4+ T cell responses after vaccination and during Mycobacterium tuberculosis challenge. Vaccination induces IL-17-producing CD4+ T cells that, upon Mtb infection, drive chemokine-mediated recruitment of IFNγ-producing CD4+ T cells to the lung, accelerating bacterial control and highlighting IL-23 as a key regulator of protective immunity. Google Scholar Scriba, T. J. et al. Sequential inflammatory processes define human progression from M. tuberculosis infection to tuberculosis disease. PLoS Pathog. Google Scholar Moguche, A. O. et al. Antigen availability shapes T cell differentiation and function during tuberculosis. Cell Host Microbe 21, 695–706.e695 (2017). Effective Mtb vaccine design requires tailored strategies, as Ag85B-specific CD4+ T cells are limited by reduced antigen expression during persistent infection, and ESAT-6-specific T cells become functionally exhausted due to chronic stimulation, restricting their ability to control TB. Google Scholar Coscolla, M. et al. M. tuberculosis T cell epitope analysis reveals paucity of antigenic variation and identifies rare variable TB antigens. Cell Host Microbe https://doi.org/10.1016/j.chom.2015.10.008 (2015). Coppola, M. et al. In-vivo expressed Mycobacterium tuberculosis antigens recognised in three mouse strains after infection and BCG vaccination. NPJ Vaccines 6, 81 (2021). Google Scholar Nunes-Alves, C. et al. Human and murine clonal CD8+ T cell expansions arise during tuberculosis because of TCR selection. PLoS Pathog. Google Scholar Carpenter, S. M., Yang, J. D., Lee, J., Barreira-Silva, P. & Behar, S. M. Vaccine-elicited memory CD4+ T cell expansion is impaired in the lungs during tuberculosis. PLoS Pathog. The transient protection of TB vaccines is driven by the inability of memory CD4+ T cells to sustain expansion in the lung, as their initial proliferation and recruitment curb early Mtb growth but fail to maintain long-term immunity. Google Scholar Bhattacharyya, N. D. et al. TCR affinity controls the dynamics but not the functional specification of the antimycobacterial CD4+ T cell response. Google Scholar Ogongo, P. & Ernst, J. D. Finding antigens for TB vaccines: the good, the bad and the useless. Google Scholar Alon, R. et al. Leukocyte trafficking to the lungs and beyond: lessons from influenza for COVID-19. Google Scholar Hoft, S. G. et al. The rate of CD4 T cell entry into the lungs during Mycobacterium tuberculosis infection is determined by partial and opposing effects of multiple chemokine receptors. Google Scholar Santiago-Carvalho, I. et al. T cell-specific P2RX7 favors lung parenchymal CD4+ T cell accumulation in response to severe lung infections. Extracellular ATP released by damaged cells is sensed by CD4+ T cells expressing P2RX7 receptor and critical to induce tissue CD4+ T cell accumulation through upregulation of the chemokine receptor CXCR3. Google Scholar Egen, J. G. et al. Intravital imaging reveals limited antigen presentation and T cell effector function in mycobacterial granulomas. Intravital imaging in BCG-infected liver show that few T cells are arrested within the granuloma and stably interact with APC, indicating that limited antigen presentation in mycobacterial granulomas leads to a muted T cell response, using only a fraction of the host's potential effector capacity during chronic infection. Google Scholar Stark, R. et al. TRM maintenance is regulated by tissue damage via P2RX7. Extracellular ATP and NAD+ enhanced TRM cell death via P2RX7, whereas TCR activation downregulated P2RX7, making TRM cells resistant to NAD-induced death. Tissue damage regulates TRM cells through P2RX7, favouring antigen-specific TRM cell persistence. Article PubMed Google Scholar Gern, B. H. et al. TGFβ restricts expansion, survival, and function of T cells within the tuberculous granuloma. Cell Host Microbe 29, 594–606.e596 (2021). Google Scholar Kang, T. G. et al. Viral coinfection promotes tuberculosis immunopathogenesis by type I IFN signaling-dependent impediment of Th1 cell pulmonary influx. Google Scholar Potter, E. L. et al. Measurement of leukocyte trafficking kinetics in macaques by serial intravascular staining. Google Scholar Takamura, S. et al. Specific niches for lung-resident memory CD8+ T cells at the site of tissue regeneration enable CD69-independent maintenance. Google Scholar Asrir, A. et al. Tumor-associated high endothelial venules mediate lymphocyte entry into tumors and predict response to PD-1 plus CTLA-4 combination immunotherapy. Cancer Cell 40, 318–334.e319 (2022). Tumour-associated HEVs serve as key entry points for lymphocytes into tumours, enhancing immune checkpoint blockade efficacy by promoting infiltration of stem-like CD8+ T cells, with their presence correlating with better responses and survival in patients with metastatic melanoma. Google Scholar Blanchard, L. & Girard, J. P. High endothelial venules (HEVs) in immunity, inflammation and cancer. Google Scholar Carow, B. et al. Immune mapping of human tuberculosis and sarcoidosis lung granulomas. Google Scholar Kahnert, A. et al. Mycobacterium tuberculosis triggers formation of lymphoid structure in murine lungs. Google Scholar Pally, D. & Naba, A. Extracellular matrix dynamics: a key regulator of cell migration across length-scales and systems. Cell Biol. Google Scholar Overstreet, M. G. et al. Inflammation-induced interstitial migration of effector CD4+ T cells is dependent on integrin alphaV. Google Scholar Wilson, E. H. et al. Behavior of parasite-specific effector CD8+ T cells in the brain and visualization of a kinesis-associated system of reticular fibers. Google Scholar Salmon, H. et al. Matrix architecture defines the preferential localization and migration of T cells into the stroma of human lung tumors. Google Scholar The extracellular matrix regulates granuloma necrosis in tuberculosis. Google Scholar Bromley, S. K., Peterson, D. A., Gunn, M. D. & Dustin, M. L. Cutting edge: hierarchy of chemokine receptor and TCR signals regulating T cell migration and proliferation. This study provided insight into the relationships between chemokine receptor and TCR signalling in determining trafficking and positioning of T cells in immunity. Google Scholar Li, J. et al. Mycobacterium tuberculosis-specific memory T cells in bronchoalveolar lavage of patients with pulmonary tuberculosis. Google Scholar Yang, Q. et al. Cutting edge: characterization of human tissue-resident memory T cells at different infection sites in patients with tuberculosis. Google Scholar Morrison, H. & McShane, H. Local pulmonary immunological biomarkers in tuberculosis. Google Scholar Hunter, R. L. Tuberculosis as a three-act play: a new paradigm for the pathogenesis of pulmonary tuberculosis. Article PubMed Google Scholar Jameson, S. C. The naming of memory T-cell subsets. Cold Spring Harb. Google Scholar Christo, S. N., Park, S. L., Mueller, S. N. & Mackay, L. K. The multifaceted role of tissue-resident memory T cells. Google Scholar Mueller, S. N., Gebhardt, T., Carbone, F. R. & Heath, W. R. Memory T cell subsets, migration patterns, and tissue residence. Google Scholar Amezcua Vesely, M. C. et al. Effector TH17 cells give rise to long-lived TRM cells that are essential for an immediate response against bacterial infection. Google Scholar Haddadi, S. et al. Expression and role of VLA-1 in resident memory CD8 T cell responses to respiratory mucosal viral-vectored immunization against tuberculosis. Google Scholar Perdomo, C. et al. Mucosal BCG vaccination induces protective lung-resident memory T cell populations against tuberculosis. Google Scholar Hu, Z. et al. Sendai virus mucosal vaccination establishes lung-resident memory CD8 T cell immunity and boosts BCG-primed protection against TB in mice. Google Scholar Dijkman, K. et al. Pulmonary MTBVAC vaccination induces immune signatures previously correlated with prevention of tuberculosis infection. Cell Rep. Med. Google Scholar Palendira, U., Bean, A. G., Feng, C. G. & Britton, W. J. Lymphocyte recruitment and protective efficacy against pulmonary mycobacterial infection are independent of the route of prior Mycobacterium bovis BCG immunization. Google Scholar Ganchua, S. K. et al. Antibiotic treatment modestly reduces protection against Mycobacterium tuberculosis reinfection in macaques. Article PubMed Google Scholar Stolley, J. M. et al. Depleting CD103+ resident memory T cells in vivo reveals immunostimulatory functions in oral mucosa. Google Scholar Behr, F. M. et al. Tissue-resident memory CD8+ T cells shape local and systemic secondary T cell responses. Google Scholar Florido, M. et al. Pulmonary immunization with a recombinant influenza A virus vaccine induces lung-resident CD4+ memory T cells that are associated with protection against tuberculosis. Mucosal Immunol. Google Scholar Turner, D. L. et al. Lung niches for the generation and maintenance of tissue-resident memory T cells. Mucosal Immunol. Google Scholar Ogongo, P. et al. Tissue-resident-like CD4+ T cells secreting IL-17 control Mycobacterium tuberculosis in the human lung. Google Scholar Alexandrov, T., Saez-Rodriguez, J. & Saka, S. K. Enablers and challenges of spatial omics, a melting pot of technologies. Google Scholar Breart, B., Lemaitre, F., Celli, S. & Bousso, P. Two-photon imaging of intratumoral CD8+ T cell cytotoxic activity during adoptive T cell therapy in mice. Google Scholar Filipe-Santos, O. et al. A dynamic map of antigen recognition by CD4 T cells at the site of Leishmania major infection. Cell Host Microbe 6, 23–33 (2009). Google Scholar Ashouri, J. F. & Weiss, A. Endogenous Nur77 is a specific indicator of antigen receptor signaling in human T and B cells. Google Scholar Moran, A. E. et al. T cell receptor signal strength in Treg and iNKT cell development demonstrated by a novel fluorescent reporter mouse. Formaglio, P. et al. Nitric oxide controls proliferation of Leishmania major by inhibiting the recruitment of permissive host cells. Halle, S. et al. In vivo killing capacity of cytotoxic T cells is limited and involves dynamic interactions and T cell cooperativity. Looney, M. R. & Headley, M. B. Live imaging of the pulmonary immune environment. Cell Immunol. The review summarizes how advances in intravital microscopy have enhanced our understanding of lung function and immunity in steady state and diseases by visualizing immune dynamics in real-time. Ueki, H., Wang, I. H., Zhao, D., Gunzer, M. & Kawaoka, Y. Multicolor two-photon imaging of in vivo cellular pathophysiology upon influenza virus infection using the two-photon IMPRESS. Barlerin, D. et al. Biosafety level 3 setup for multiphoton microscopy in vivo. Gazit, R. et al. Lethal influenza infection in the absence of the natural killer cell receptor gene Ncr1. Ovchinnikov, D. A. et al. Expression of Gal4-dependent transgenes in cells of the mononuclear phagocyte system labeled with enhanced cyan fluorescent protein using Csf1r-Gal4VP16/UAS-ECFP double-transgenic mice. Lindquist, R. L. et al. Visualizing dendritic cell networks in vivo. Boulch, M. et al. A cross-talk between CAR T cell subsets and the tumor microenvironment is essential for sustained cytotoxic activity. Boulch, M., Grandjean, C. L., Cazaux, M. & Bousso, P. Tumor immunosurveillance and immunotherapies: a fresh look from intravital imaging. Trends Immunol. Grandjean, C. L., Garcia, Z., Lemaitre, F., Breart, B. & Bousso, P. Imaging the mechanisms of anti-CD20 therapy in vivo uncovers spatiotemporal bottlenecks in antibody-dependent phagocytosis. Wu, J. et al. A highly polarized TH2 bladder response to infection promotes epithelial repair at the expense of preventing new infections. Hoekstra, M. E., Dijkgraaf, F. E., Schumacher, T. N. & Rohr, J. C. Assessing T lymphocyte function and differentiation by genetically encoded reporter systems. Trends Immunol. Chitirala, P. et al. Studying the biology of cytotoxic T lymphocytes in vivo with a fluorescent granzyme B-mTFP knock-in mouse. Vorobjev, I. & Barteneva, N. S. Multi-parametric imaging of cell heterogeneity in apoptosis analysis. Fukumura, D., Duda, D. G., Munn, L. L. & Jain, R. K. Tumor microvasculature and microenvironment: novel insights through intravital imaging in pre-clinical models. McCarthy, C. E., White, J. M., Viola, N. T. & Gibson, H. M. In vivo imaging technologies to monitor the immune system. MacGilvary, N. J. & Tan, S. Fluorescent Mycobacterium tuberculosis reporters: illuminating host-pathogen interactions. Huang, L., Nazarova, E. V., Tan, S., Liu, Y. & Russell, D. G. Growth of Mycobacterium tuberculosis in vivo segregates with host macrophage metabolism and ontogeny. Tan, S., Sukumar, N., Abramovitch, R. B., Parish, T. & Russell, D. G. Mycobacterium tuberculosis responds to chloride and pH as synergistic cues to the immune status of its host cell. PLoS Pathog. Liu, Z. et al. Detecting tumor antigen-specific T cells via interaction-dependent fucosyl-biotinylation. Qiu, S. et al. Use of intercellular proximity labeling to quantify and decipher cell-cell interactions directed by diversified molecular pairs. Pasqual, G. et al. Monitoring T cell-dendritic cell interactions in vivo by intercellular enzymatic labelling. Nakandakari-Higa, S. et al. Universal recording of immune cell interactions in vivo. A new genetically encoded method for proximity labelling to identify cell–cell interactions in vivo, such as T cell–APC interactions. Kiepiela, P. et al. CD8+ T-cell responses to different HIV proteins have discordant associations with viral load. Ranasinghe, S. et al. HIV-specific CD4 T cell responses to different viral proteins have discordant associations with viral load and clinical outcome. Coppola, M. & Ottenhoff, T. H. Genome wide approaches discover novel Mycobacterium tuberculosis antigens as correlates of infection, disease, immunity and targets for vaccination. Nair, S. K. et al. High-throughput identification and dendritic cell-based functional validation of MHC class I-restricted Mycobacterium tuberculosis epitopes. Lichti, C. F., Vigneron, N., Clauser, K. R., Van den Eynde, B. J. & Bassani-Sternberg, M. Navigating critical challenges associated with immunopeptidomics-based detection of proteasomal spliced peptide candidates. Cancer Immunol. Gulden, P. H. et al. A listeria monocytogenes pentapeptide is presented to cytolytic T lymphocytes by the H2-M3 MHC class Ib molecule. Purcell, A. W., Ramarathinam, S. H. & Ternette, N. Mass spectrometry-based identification of MHC-bound peptides for immunopeptidomics. Shao, W. et al. The SysteMHC atlas project. Nucleic Acids Res. Bettencourt, P. et al. Identification of antigens presented by MHC for vaccines against tuberculosis. NPJ Vaccines 5, 2 (2020). Leddy, O., White, F. M. & Bryson, B. D. Immunopeptidomics reveals determinants of Mycobacterium tuberculosis antigen presentation on MHC class I. eLife 12, e84070 (2023). Cayabyab, M. J., Qin, L., Kashino, S. S., Izzo, A. & Campos-Neto, A. An unbiased peptide-wide discovery approach to select Mycobacterium tuberculosis antigens that target CD8+ T cell response during infection. Baert, L. et al. Induced pluripotent stem cell-derived human macrophages as an infection model for Leishmania donovani. PLoS Negl. Higaki, K. et al. Generation of HIV-resistant macrophages from IPSCs by using transcriptional gene silencing and promoter-targeted RNA. Nucleic Acids 12, 793–804 (2018). Hong, D. et al. Human-induced pluripotent stem cell-derived macrophages and their immunological function in response to tuberculosis infection. Stem Cell Res. Nielsen, M., Ternette, N. & Barra, C. The interdependence of machine learning and LC–MS approaches for an unbiased understanding of the cellular immunopeptidome. Huang, S. et al. CD1 lipidomes reveal lipid-binding motifs and size-based antigen-display mechanisms. Beckman, E. M. et al. Recognition of a lipid antigen by CD1-restricted αβ+ T cells. Moody, D. B. et al. CD1c-mediated T-cell recognition of isoprenoid glycolipids in Mycobacterium tuberculosis infection. Huang, H., Wang, C., Rubelt, F., Scriba, T. J. & Davis, M. M. Analyzing the Mycobacterium tuberculosis immune response by T-cell receptor clustering with GLIPH2 and genome-wide antigen screening. Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Mead, H. L. et al. Integrated, high-dimensional analysis of CD4 T cell epitope specificities and phenotypes reveals unexpected diversity in the response to Mycobacterium tuberculosis. Preprint at bioRxiv https://doi.org/10.1101/2024.11.05.622086 (2024). Ogongo, P. et al. Differential skewing of donor-unrestricted and γδ T cell repertoires in tuberculosis-infected human lungs. Kramnik, I., Dietrich, W. F., Demant, P. & Bloom, B. R. Genetic control of resistance to experimental infection with virulent Mycobacterium tuberculosis. Natl Acad. Churchill, G. A. et al. The collaborative cross, a community resource for the genetic analysis of complex traits. Churchill, G. A., Gatti, D. M., Munger, S. C. & Svenson, K. L. The diversity outbred mouse population. Niazi, M. K. et al. Lung necrosis and neutrophils reflect common pathways of susceptibility to Mycobacterium tuberculosis in genetically diverse, immune-competent mice. Smith, C. M. et al. Host–pathogen genetic interactions underlie tuberculosis susceptibility in genetically diverse mice. Lai, R. et al. Host genetic background is a barrier to broadly effective vaccine-mediated protection against tuberculosis. Wu, R., Velickovic, M. & Burnum-Johnson, K. E. From single cell to spatial multi-omics: unveiling molecular mechanisms in dynamic and heterogeneous systems. Larenas-Munoz, F. et al. Proteomic analysis of granulomas from cattle and pigs naturally infected with Mycobacterium tuberculosis complex by MALDI imaging. Greco, F. et al. Lipids associated with atherosclerotic plaque instability revealed by mass spectrometry imaging of human carotid arteries. Shariatgorji, R. et al. Spatial visualization of comprehensive brain neurotransmitter systems and neuroactive substances by selective in situ chemical derivatization mass spectrometry imaging. Rajbhandari, P., Neelakantan, T. V., Hosny, N. & Stockwell, B. R. Spatial pharmacology using mass spectrometry imaging. Trends Pharmacol. Blanc, L. et al. High-resolution mapping of fluoroquinolones in TB rabbit lesions reveals specific distribution in immune cell types. Moore, M. J. et al. Humanization of T cell-mediated immunity in mice. Patel, S., Liu, W., K, R., McCormick, C. & Fan, Y. Engineering immune organoids to regenerate host immune system. Wagar, L. E. et al. Modeling human adaptive immune responses with tonsil organoids. Ramachandra, L., Noss, E., Boom, W. H. & Harding, C. V. Processing of Mycobacterium tuberculosis antigen 85B involves intraphagosomal formation of peptide-major histocompatibility complex II complexes and is inhibited by live bacilli that decrease phagosome maturation. Mehra, A. et al. Mycobacterium tuberculosis type VII secreted effector EsxH targets host ESCRT to impair trafficking. PLoS Pathog. Portal-Celhay, C. et al. Mycobacterium tuberculosis EsxH inhibits ESCRT-dependent CD4+ T-cell activation. Madan-Lala, R. et al. Mycobacterium tuberculosis impairs dendritic cell functions through the serine hydrolase Hip1. Dolasia, K., Nazar, F. & Mukhopadhyay, S. Mycobacterium tuberculosis PPE18 protein inhibits MHC class II antigen presentation and B cell response in mice. Saini, N. K. et al. Suppression of autophagy and antigen presentation by Mycobacterium tuberculosis PE_PGRS47. Fulton, S. A. et al. Inhibition of major histocompatibility complex II expression and antigen processing in murine alveolar macrophages by Mycobacterium bovis BCG and the 19-kilodalton mycobacterial lipoprotein. Gehring, A. J. et al. The Mycobacterium tuberculosis 19-kilodalton lipoprotein inhibits gamma interferon-regulated HLA-DR and Fc gamma R1 on human macrophages through toll-like receptor 2. Pai, R. K., Convery, M., Hamilton, T. A., Boom, W. H. & Harding, C. V. Inhibition of IFN-gamma-induced class II transactivator expression by a 19-kDa lipoprotein from Mycobacterium tuberculosis: a potential mechanism for immune evasion. Su, H. et al. Recombinant lipoprotein Rv1016c derived from Mycobacterium tuberculosis is a TLR-2 ligand that induces macrophages apoptosis and inhibits MHC II antigen processing. Cell Infect. Pecora, N. D., Gehring, A. J., Canaday, D. H., Boom, W. H. & Harding, C. V. Mycobacterium tuberculosis LprA is a lipoprotein agonist of TLR2 that regulates innate immunity and APC function. Gehring, A. J., Dobos, K. M., Belisle, J. T., Harding, C. V. & Boom, W. H. Mycobacterium tuberculosis LprG (Rv1411c): a novel TLR-2 ligand that inhibits human macrophage class II MHC antigen processing. Huber, A. et al. Mycobacterial cord factor reprograms the macrophage response to IFN-gamma towards enhanced inflammation yet impaired antigen presentation and expression of GBP1. Srivastava, S., Grace, P. S. & Ernst, J. D. Antigen export reduces antigen presentation and limits T cell control of M. tuberculosis. Cell Host Microbe 19, 44–54 (2016).Mtb evades CD4+ T cell immunity by exporting antigens from infected DCs, diverting them from the antigen presentation pathway, a process dependent on kinesin-2, whose depletion enhances T cell activation and intracellular bacterial control. Grace, P. S. & Ernst, J. D. Suboptimal antigen presentation contributes to virulence of Mycobacterium tuberculosis in vivo. Sreejit, G. et al. The ESAT-6 protein of Mycobacterium tuberculosis interacts with beta-2-microglobulin (β2M) affecting antigen presentation function of macrophage. PLoS Pathog. Velmurugan, K. et al. Mycobacterium tuberculosis nuoG is a virulence gene that inhibits apoptosis of infected host cells. PLoS Pathog. Winau, F. et al. Apoptotic vesicles crossprime CD8 T cells and protect against tuberculosis. Singh, B. et al. Inhibition of indoleamine dioxygenase leads to better control of tuberculosis adjunctive to chemotherapy. JCI Insight 8, e163101 (2023). Barber, D. L., Mayer-Barber, K. D., Feng, C. G., Sharpe, A. H. & Sher, A. CD4 T cells promote rather than control tuberculosis in the absence of PD-1-mediated inhibition. Download references This study was supported by the French National Research Agency: JCJC (grant ANR-20-CE14-0045-01) and TBVAC Horizon Europe programme (E.L.); Agence Nationale de le Recherche sur le SIDA — Maladies Infectieuses Emergentes KILL-TB ECTZ206385 (D.H.); TBVAC-HORIZON (EU), Immunotherapies for Tuberculosis and Other Mycobacterial Diseases (EU) and EXPLORE-TB (Fondation Bettencourt Schueller) (O.N. ); National Institutes of Health, National Institute of Allergy and Infectious Diseases (grants R01AI172905 to S.M.B., U01AI166309, R21AI156407 and R21AI176234 to J.D.E. and contracts 75N93019C00071 to S.M.B. and 75N93024C0054 to J.D.E. These authors contributed equally: Emma Lefrançais, Denis Hudrisier. These authors jointly supervised this work: Olivier Neyrolles, Samuel M. Behar, Joel D. Ernst. Institut de Pharmacologie et de Biologie Structurale, Université de Toulouse, CNRS, UPS, Toulouse, France Emma Lefrançais, Denis Hudrisier & Olivier Neyrolles Department of Microbiology, University of Massachusetts Chan Medical School, Worcester, MA, USA Samuel M. Behar Division of Experimental Medicine, Department of Medicine, University of California San Francisco, San Francisco, CA, USA Joel D. Ernst You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar All authors researched data for the article. All authors contributed substantially to discussion of the content. All authors wrote the article. All authors reviewed and/or edited the manuscript before submission. Correspondence to Joel D. Ernst. The authors declare no competing interests. Nature Reviews Immunology thanks Stefan Kaufmann, Thomas Scriba, Cecilia Lindestam Arlehamn and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Lefrançais, E., Hudrisier, D., Neyrolles, O. et al. Finding and filling the knowledge gaps in mechanisms of T cell-mediated TB immunity to inform vaccine design. Nat Rev Immunol  (2025). Download citation Accepted: 24 April 2025 Published: 13 June 2025 DOI: https://doi.org/10.1038/s41577-025-01192-z Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Immunology (Nat Rev Immunol) ISSN 1474-1741 (online) ISSN 1474-1733 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/quantum-computers-simulate-particle-string-breaking-in-a-physics/'>Quantum Computers Simulate Particle 'String Breaking' in a Physics Breakthrough</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 12:40:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Quantum Computers Simulate Particle 'String Breaking' in a Physics Breakthrough Physicists are a step closer to using quantum computers for simulations that are beyond the ability of any ordinary computers Subatomic particles such as quarks can pair up when linked by ‘strings' of force fields — and release energy when the strings are pulled to the point of breaking. Two teams of physicists have now used quantum computers to mimic this phenomenon and watch it unfold in real time. The results, described in two Nature papers on June 4, are the latest in a series of breakthroughs towards using quantum computers for simulations that are beyond the ability of any ordinary computers. “String breaking is a very important process that is not yet fully understood from first principles,” says Christian Bauer, a physicist at the Lawrence Berkeley National Laboratory (LBNL) in Berkeley, California. Physicists can calculate the final results of particle collisions that form or break strings using classical computers, but cannot fully simulate what happens in between. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Each experiment was conducted by an international collaboration involving academic and industry researchers — one team at QuEra Computing, a start-up company in Cambridge, Massachusetts, and another at the Google Quantum AI Lab in Santa Barbara, California. The quantum state of each atom — a qubit that could be excited or relaxed — represented the electric field at a point in space, explains co-author Daniel González-Cuadra, a theoretical physicist now at the Institute for Theoretical Physics in Madrid. In both cases, the teams set up strings in the field that effectively acted like rubber bands connecting two particles. Although simulating strings in a 2D electric field could have applications for studying the physics of materials, it is still a long way from fully simulating high-energy interactions, such as those that occur in particle colliders, which are in 3D and involve the much more complex strong nuclear force. “We do not have a clear path at this point how to get there,” says Monika Aidelsburger, a physicist at the Max Planck Institute of Quantum Optics in Munich, Germany. Still, the latest results are exciting, and progress in quantum simulation in general has been “really amazing and very fast,” Aidelsburger says. Last year, Bauer and his LBNL colleague Anthony Ciavarella were among the first teams to simulate the strong nuclear force on a quantum computer. This article is reproduced with permission and was first published on June 5, 2025. Davide Castelvecchi is a staff reporter at Nature who has been obsessed with quantum spin for essentially his entire life. First published in 1869, Nature is the world's leading multidisciplinary science journal. Nature publishes the finest peer-reviewed research that drives ground-breaking discovery, and is read by thought-leaders and decision-makers around the world.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/tectonic-plates-can-infect-one-another-with-earth-shaking-subduction-zones/'>Tectonic Plates Can ‘Infect' One Another with Earth-Shaking Subduction Zones</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Tectonic Plates Can ‘Infect' One Another with Earth-Shaking Subduction Zones Evidence from Earth's deep past suggests dramatic subduction zones can spread like a contagion Subduction zones, where one tectonic plate dives underneath another, drive the world's most devastating earthquakes and tsunamis. A study in Geology presents evidence that subduction can spread like a contagion, jumping from one oceanic plate to another—a hypothesis previously difficult to prove. “This study builds an argument based on the geological record.” Because subduction drags crust deep into the earth, its beginnings are hard to examine. The new study provides a rare ancient example of potential subduction “infection.” Its authors say they've discovered evidence that neighboring collisions triggered East Asia's “Ring of Fire,” a colossal subduction system currently fueling earthquakes and volcanoes from Alaska to the southern Indian Ocean. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. Nearly 300 million years ago China was a scattering of islands separated by the ancient Tethys and Asian oceans. Established subduction zones consumed these oceans, welding the landmasses into a new continent and raising mountains from Turkey to China. By 260 million years ago this subduction seems to have spread and begun pulling down the neighboring Pacific plate. “In one form or another, it's been diving down ever since.” “It's like seeing someone's fingerprint at a crime scene,” Allen says. The researchers suspect that transform faults—boundaries where plates slide past one another, like the San Andreas Fault—may act as weak spots where slight changes in collision angle or speed can destabilize dense oceanic crust, causing it to sink. If subduction spreads this way, could the Atlantic Ocean's relatively quiet plate margins be next? The massive 1755 Lisbon earthquake hints at early subduction invasion there. Duarte suggests parts of Iberia and the Caribbean are undergoing this process's initial stages: “In another 100 million years a new Atlantic ‘Ring of Fire' may form—just as it once did in the Pacific.”</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250613013831.htm'>Running rewires your brain cells—igniting memory-saving genes against alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 11:41:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Using advanced single-nuclei RNA sequencing (snRNA-seq) and a widely used preclinical model for Alzheimer's disease, researchers from Mass General Brigham and collaborators at SUNY Upstate Medical University have identified specific brain cell types that responded most to exercise. "While we've long known that exercise helps protect the brain, we didn't fully understand which cells were responsible or how it worked at a molecular level," said senior author Christiane D. Wrann, DVM, PhD, a neuroscientist and leader of the Program in Neuroprotection in Exercise at the Mass General Brigham Heart and Vascular Institute and the McCance Center for Brain Health at Massachusetts General Hospital. The research team leveraged single-nuclei RNA sequencing, a relatively new technologies that allow researchers to look at activity at the molecular level in single cells for an in-depth understanding of diseases like Alzheimer's. They then analyzed gene activity across thousands of individual brain cells, finding that exercise changed activity both in microglia, a disease-associated population of brain cells, and in a specific type of neurovascular-associated astrocyte (NVA), newly discovered by the team, which are cells associated with blood vessels in the brain. "That we were able to modulate newborn neurons using our new target genes set underscores the promise our study," said lead author Joana Da Rocha, PhD, a postdoctoral fellow working in Dr. Wrann's lab. To ensure the findings were relevant to humans, the team validated their discoveries in a large dataset of human Alzheimer's brain tissue, finding striking similarities. "This work not only sheds light on how exercise benefits the brain but also uncovers potential cell-specific targets for future Alzheimer's therapies," said Nathan Tucker, a biostatistician at SUNY Upstate Medical University and co-senior of the study. "Our study offers a valuable resource for the scientific community investigating Alzheimer's prevention and treatment." Authorship: In addition to da Rocha and Wrann, Mass General Brigham authors include Renhao Luo, Pius Schlachter, Luis Moreira, Mohamed Ariff Iqbal, Paula Kuhn, Sophia Valaris, Mohammad R. Islam, Gabriele M. Gassner, Sofia Mazuera, Kaela Healy, Sanjana Shastri, Nathaniel B. Hibbert, Kristen V. Moran-Figueroa, Erin B. Haley, Sema Aygar, and Ksenia V. Kastanenka. Additional authors include Michelle L. Lance, Robert S. Gardner, Ryan D. Pfeiffer, Logan Brase, Oscar Harari, Bruno A. Benitez, and Nathan R. Tucker. Disclosures: Wrann is an academic co-founder and consultant for Aevum Therapeutics. Wrann has a financial interest in Aevum Therapeutics, a company developing drugs that harness the protective molecular mechanisms of exercise to treat neurodegenerative and neuromuscular disorders. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/mapping-nearby-stars-could-solve-a-deep-cosmic-mystery/'>Cosmic Secrets Are Hiding in This Census of Our Interstellar Neighborhood</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This New Map of Nearby Stars May Solve a Cosmic Mystery A near-complete census of our interstellar neighborhood hopes to answer how stars, brown dwarfs and rogue planets form throughout the universe Newborn stars set gas clouds aglow in this mosaic of images from the James Webb Space Telescope. This is the star-forming region NGC 1333, located some 960 light-years from Earth; a census of stars much closer to our solar system is revealing new details about how this and other stellar nurseries across the universe manufacture stars. ESA/Webb, NASA & CSA, A. Scholz, K. Muzic, A. Langeveld, R. Jayawardhana (CC BY 4.0) Stay connected to The Universe: Get email alerts for this weekly column by Phil Plait I don't think it's overly poetic to say that stars are how we know the universe. When we look out at the night sky, stars are overwhelmingly what we see—thousands of them, ranging from Sirius, the brightest in the night sky, to ones so dim they are known by mere catalog designations instead of names. The heavier elements making up our planet and even our body were forged in stars long ago, and our own sun is a star, of course—so in a very real sense, to study stars is to study ourselves. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Yet for all that, some basic questions about stars remain unanswered. While we have a pretty decent grasp of how individual stars are born, there are still gaps in our knowledge of their statistics en masse. Say you have a giant gas cloud that is busily manufacturing stars. As a percentage, how many of them will be like the sun? The mass of a star—how much matter is in it—determines most of its properties, including its temperature, color, brightness, evolution and even its destiny, so this is something astronomers are very keen to understand. And in theory, it's not too hard to answer: just observe a bunch of stars, determine their masses and then figuratively put them into the proper box. Massive stars are bright and easy to spot. We can't, however, see small, dim ones if they are very far away—which means our cosmic census of low-mass stars is largely limited to whatever happens to be relatively near our solar system. Still, given time, these stars mix pretty well throughout the galaxy, so we can assume such fainter stellar next-door neighbors are representative of the galaxy at large. For lower-mass stars, which shine brightest in infrared, Gaia's observations were augmented by data from NASA's Wide-field Infrared Survey Explorer and the Spitzer Space Telescope. Brown dwarfs, which are objects of intermediate mass between planets and stars, give off essentially no visible light and can only be detected in the infrared. And in general, these objects are so dim that even nearby ones can elude detection. Luhman 16 is a binary brown dwarf system just 6.5 light-years from the sun—the third closest system to us—yet was only discovered in 2013. I'm used to thinking on much larger spatial scales, so finding 3,600 objects within a mere 65 light-years of the sun is more than I'd have guessed. Mind you, our Milky Way galaxy is a flat disk 120,000 light-across, which is roomy enough for hundreds of billions of stars and other celestial citizenry. Brown dwarfs cooler than about 325 degrees Celsius are so faint that our current technology can't directly see them beyond about 50 light-years. There could also be smaller binary companions to some stars that have gone undiscovered. What that means is that some brown dwarfs have likely been undercounted, which is actually pretty problematic for trying to determine the full range of object masses spawned in galactic gas clouds. Think of it this way: if you smash a rock with a hammer, you're likely to get one or two big pieces, a dozen or two midsize ones, hundreds of small chunks and thousands or tens of thousands of tiny grains. Before, it wasn't clear if objects had a mass cutoff at the lower end. Possibly, but until brown dwarfs were discovered and counted, we weren't sure they could even form like stars do. But the census does reveal some quirks: the tally of objects flattens out a bit as mass descends from the stellar regime to brown dwarfs but then starts rising again at lower, more planetary-scale masses. Does it flatten out again at some lower mass, such as around a few times that of Jupiter? Still, this survey is a big step forward. Extrapolating it to the Milky Way and other galaxies will help us understand how galaxies behave—and how they change their behavior over time, churning out different mixes of stars as they age. Phil Plait is a professional astronomer and science communicator in Virginia. His column for Scientific American, The Universe, covers all things space. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/invasive-asian-needle-ants-sting-is-painful-can-cause-anaphylaxis/'>This Invasive Ant's Sting Is like Fiberglass under the Skin. Here's How to Avoid It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The Asian needle ant looks ordinary, but its sting can be worse than a fire ant's and can be dangerous for anyone who has experienced anaphylaxis Warnings of a potentially dangerous invasive ant have popped up on news sites and social media this week. You'd be forgiven for thinking a new threat had arrived. But this insect, the Asian needle ant (Brachyponera chinensis), is no newcomer—just a master of remaining inconspicuous. The ant is getting attention after University of Georgia (UGA) entomologist Dan Suiter issued a warning that its sting can be remarkably painful and sometimes cause a serious allergic reaction called anaphylaxis. The ant is not new or spreading any faster than it used to, Suiter says. Rather, as a UGA Cooperative Extension entomologist, he's been getting calls about people being stung by the ant and needing hospital care. “By this time last year, I had fielded three calls of people who had been stung by an Asian needle ant—they knew it was an Asian needle ant—and suffered anaphylaxis,” Suiter says. “You're not thinking that ‘this tiny little creature right here could have consequences if it stung me. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The Asian needle ant is native to Japan, Korea and China. It arrived in the U.S. around or before the 1930s, probably in ships carrying plants and livestock “before we even realized that is a great way to spread things around,” says Theresa Dellinger, a diagnostician at the Insect Identification Lab at Virginia Tech. These ants don't lay down pheromone trails to lead their colony to food, so you won't see them marching in a line like the little black ant (Monomorium minimum). Instead, when an Asian needle ant finds food, it will return to its colony, pick up another worker and physically carry it to the jackpot, says Christopher Hayes, an entomologist at North Carolina State University. Instead they love nesting in damp wood, such as fallen logs or wet mulch. The little boy was fine after a few minutes, Hayes says, but then woke up after midnight that night crying about the pain again. In their native range, Asian needle ant bites are associated with a 2 percent risk of anaphylaxis compared with a 0.5 to 1.5 percent risk from the bites of fire ants, Hayes says. Gently flick or brush it off, Hayes and Suiter say. If it does sting you, try to capture the ant for identification purposes if you can (though it will probably scurry away too quickly). If you start to experience hives, swelling of the lips or tongue or difficulty breathing, get to an emergency room because these symptoms are signs of anaphylaxis. There are no centralized entities mapping out the ants' spread, but Suiter says he hopes more public attention will spur more scientific research. Identification is particularly important for those who have had allergic reactions to bee and ant stings in the past. “If people have a family member that suffers from anaphylaxis and knows it, and they carry around an Epipen,” he says, “somebody like that needs to know what this ant looks like.” Stephanie Pappas is a freelance science journalist based in Denver, Colo.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01684-z'>A checklist for designing and improving the visualization of scientific data</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 10:29:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Creating clear and engaging scientific figures is crucial to communicate complex data. In this Comment, I condense principles from design, visual perception and data visualization research in a checklist that can help researchers to improve their data visualization, by focusing on clarity, accessibility and design best practices. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Friendly, M. in Handbook of Data Visualization (eds Chen, C. et al.) 15–56 (Springer, 2008). Tufte, E. R. The Visual Display of Quantitative Information (Graphics, 2011). Better Posters: Plan, Design and Present an Academic Poster (Pelagic, 2021). Senft, R. A. et al. PLoS Biol. The Digital Cell: Cell Biology as a Data Science (Cold Spring Harbor Laboratory Press, 2019). Wilke, C. Fundamentals of Data Visualization: a Primer on Making Informative and Compelling Figures (O'Reilly Media, 2019). Franconeri, S. L., Padilla, L. M., Shah, P., Zacks, J. M. & Hullman, J. Psychol. I thank my colleagues T. Weissgerber, A. Gucek, S. Royle and J. P. Saenz for feedback on the draft version of this Comment, as well as the many students of my data-viz courses, whose examples continually expand my expertise and whose requests for an easy-to-follow workflow inspired this checklist. Institute for Data Analysis, Artificial Intelligence, Visualization and Simulation, University of Applied Sciences of the Grisons, Chur, Switzerland National Center for Tumor Diseases, University Cancer Center, NCT-UCC, Universitätsklinikum Carl Gustav Carus an der Technischen Universität Dresden, Dresden, Germany You can also search for this author inPubMed Google Scholar Correspondence to Helena Klara Jambor. received a salary from a habilitation award of the Medical Faculty of the Technische Universität Dresden. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Nature Cell Biology thanks Beth Cimini and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A checklist for designing and improving the visualization of scientific data. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250613013823.htm'>New MIT study reveals how biofilms help stop microplastic build-up</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 09:54:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But predicting where these ubiquitous particles will accumulate, and therefore where remediation efforts should be focused, has been difficult because of the many factors that contribute to their dispersal and deposition. New research from MIT shows that one key factor in determining where microparticles are likely to build up has to do with the presence of biofilms. These thin, sticky biopolymer layers are shed by microorganisms and can accumulate on surfaces, including along sandy riverbeds or seashores. The study found that, all other conditions being equal, microparticles are less likely to accumulate in sediment infused with biofilms, because if they land there, they are more likely to be resuspended by flowing water and carried away. The open-access findings appear in the journal Geophysical Research Letters, in a paper by MIT postdoc Hyoungchul Park and professor of civil and environmental engineering Heidi Nepf. "Microplastics are definitely in the news a lot," Nepf says, "and we don't fully understand where the hotspots of accumulation are likely to be. This work gives a little bit of guidance" on some of the factors that can cause these particles, and small particles in general, to accumulate in certain locations. Most experiments looking at the ways microparticles are transported and deposited have been conducted over bare sand, Park says. "But in nature, there are a lot of microorganisms, such as bacteria, fungi, and algae, and when they adhere to the stream bed they generate some sticky things." These substances are known as extracellular polymeric substances, or EPS, and they "can significantly affect the channel bed characteristics," he says. In some experiments the bed consisted of pure sand, and in others the sand was mixed with a biological material to simulate the natural biofilms found in many riverbed and seashore environments. Water mixed with tiny plastic particles was pumped through the tank for three hours, and then the bed surface was photographed under ultraviolet light that caused the plastic particles to fluoresce, allowing a quantitative measurement of their concentration. Immediately around the rods that stood in for above-ground roots, turbulence prevented particle deposition. So, if I spilled a large amount of microplastic in two rivers, and one had a sandy or gravel bottom, and one was muddier with more biofilm, I would expect more of the microplastics to be retained in the sandy or gravelly river." But it provides a "nice lens" to provide some suggestions for people who are trying to study the impacts of microplastics in the field. "They're trying to determine what kinds of habitats these plastics are in, and this gives a framework for how you might categorize those habitats," she says. As an example, Park suggests, in mangrove ecosystems, microplastics may preferentially accumulate in the outer edges, which tend to be sandy, while the interior zones have sediment with more biofilm. Thus, this work suggests "the sandy outer regions may be potential hotspots for microplastic accumulation," he says, and can make this a priority zone for monitoring and protection. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02713-8'>Fossil evidence of proboscidean frugivory and its lasting impact on South American ecosystems</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 09:41:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Ecology & Evolution (2025)Cite this article Metrics details Most megaherbivores in the Americas went extinct around 10,000 years ago, presumably disrupting the long-distance seed dispersal of large, fleshy-fruited plant species. The neotropical anachronism hypothesis, proposed by Janzen and Martin, suggests that large fruits evolved in response to past selective pressures from now-extinct megafauna. While this explains many key adaptations of ‘megafaunal fruit' plants, it lacks robust palaeoecological evidence. Here we use a multiproxy approach that provides evidence of frugivory by the extinct South American proboscidean Notiomastodon platensis. Using a machine learning model that reveals the extinction effects of these megafaunal dispersers in modern ecosystems, we show how losing seed dispersal interactions increased the extinction risk of megafaunal fruit plants in regions across South America lacking smaller surrogate dispersers. Our results imply that current ecosystems may undergo further functional changes under continuing anthropogenic pressures. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Datasets are provided at https://github.com/ricardosegovia/Gonzalez-Guarda_etal_2025. Source data are provided with this paper. Code is provided at https://github.com/ricardosegovia/Gonzalez-Guarda_etal_2025. Barnosky, A. D. et al. Variable impact of Late-Quaternary megafaunal extinction in causing ecological state shifts in North and South America. Natl Acad. Google Scholar Tonni, E. P., Cione, A. L. & Soibelzon, L. H. The broken zig-zag: Late Cenozoic large mammal and tortoise extintion in South America. Google Scholar Chapin III, F. S. et al. Consequences of changing biodiversity. Google Scholar Galetti, M. et al. Ecological and evolutionary legacy of megafauna extinctions. Google Scholar Ripple, W. J. et al. What is a trophic cascade? Trends Ecol. Google Scholar Guimaraes, P. R. Jr, Galetti, M. & Jordano, P. Seed dispersal anachronisms: rethinking the fruits extinct megafauna ate. PLoS ONE 3, e1745 (2008). Google Scholar Janzen, D. H. & Martin, P. S. Neotropical anachronisms: the fruits the gomphotheres ate. Google Scholar Pires, M. M., Guimarães, P. R., Galetti, M. & Jordano, P. Pleistocene megafaunal extinctions and the functional loss of long‐distance seed‐dispersal services. Google Scholar Johnson, C. N. Ecological consequences of Late Quaternary extinctions of megafauna. B: Biol. Google Scholar Corlett, R. T. The shifted baseline: prehistoric defaunation in the tropics and its consequences for biodiversity conservation. Google Scholar Doughty, C. E. et al. Megafauna extinction, tree species range reduction, and carbon storage in Amazonian forests. Google Scholar Brodie, J. F. Evolutionary cascades induced by large frugivores. Natl Acad. Google Scholar Gill, J. L. Ecological impacts of the Late Quaternary megaherbivore extinctions. Google Scholar Cordeiro, N. J. & Howe, H. F. Low recruitment of trees dispersed by animals in African forest fragments. Google Scholar Moran, C., Catterall, C. P. & Kanowski, J. Reduced dispersal of native plant species as a consequence of the reduced abundance of frugivore species in fragmented rainforest. Google Scholar Poulsen, J. R. et al. Long distance seed dispersal by forest elephants. Google Scholar Cares, R. A., Sáez-Cordovez, C., Valiente-Banuet, A., Medel, R. & Botto-Mahan, C. Frugivory and seed dispersal in the endemic cactus Eulychnia acida: extending the anachronism hypothesis to the Chilean Mediterranean ecosystem. Google Scholar Carvalho, C. S., Galetti, M., Colevatti, R. G. & Jordano, P. Defaunation leads to microevolutionary changes in a tropical palm. Google Scholar Young, A., Boyle, T. & Brown, T. The population genetic consequences of habitat fragmentation for plants. Trends Ecol. Google Scholar Collevatti, R. G., Lima, J. S. & Ballesteros-Mejia, L. Megafauna seed dispersal in the Neotropics: a meta-analysis shows no genetic signal of loss of long-distance seed dispersal. Google Scholar Fricke, E. C. et al. Collapse of terrestrial mammal food webs since the Late Pleistocene. Google Scholar Cocker, S. L. et al. Dung analysis of the East Milford mastodons: dietary and environmental reconstructions from central Nova Scotia at ∼75 ka years BP. Earth Sci. Google Scholar Laudermilk, J. D. & Munz, P. A. Plants in the Dung of Nothrotherium from Gypsum Cave, Nevada (Carnegie Institution of Washington, 1934). Newsom, L. A. & Mihlbachler, M. C. in First Floridians and Last Mastodons: The Page–Ladson Site in the Aucilla River (ed. Birks, H. H. et al. Evidence for the diet and habitat of two Late Pleistocene mastodons from the Midwest, USA. Google Scholar Asevedo, L., Winck, G. R., Mothé, D. & Avilla, L. S. Ancient diet of the Pleistocene gomphothere Notiomastodon platensis (Mammalia, Proboscidea, Gomphotheriidae) from lowland mid-latitudes of South America: stereomicrowear and tooth calculus analyses combined. Google Scholar Cammidge, T. S., Kooyman, B. & Theodor, J. M. Diet reconstructions for end-Pleistocene Mammut americanum and Mammuthus based on comparative analysis of mesowear, microwear, and dental calculus in modern Loxodonta africana. Google Scholar Rivals, F., Semprebon, G. M. & Lister, A. M. Feeding traits and dietary variation in Pleistocene proboscideans: a tooth microwear review. Mothé, D. & Avilla, L. Mythbusting evolutionary issues on South American Gomphotheriidae (Mammalia: Proboscidea). Domingo, L., Prado, J. L. & Alberdi, M. T. The effect of paleoecology and paleobiogeography on stable isotopes of Quaternary mammals from South America. González-Guarda, E. et al. Late Pleistocene ecological, environmental and climatic reconstruction based on megafauna stable isotopes from northwestern Chilean Patagonia. González-Guarda, E. et al. Multiproxy evidence for leaf-browsing and closed habitats in extinct proboscideans (Mammalia, Proboscidea) from Central Chile. González-Guarda, E. et al. Dietary ecological traits of extinct mammalian herbivores from the Last Glacial Termination at the Pilauco site, Chile. Denniston, R. H. The growth and organization of the starch grain. Arts Lett. Kraemer, H. The Differentiation and Specificity of Starches in Relation to Genera, Species, etc. Stereochemistry Applied to Protoplasmic Processes and Products, and as a Strictly Scientific Basis for the Classification of Plants and Animals. By Edward Tyson Reichert, M.D., Professor of Physiology in the University of Pennsylvania, Research Associate of the Carnegie Institution of Washington. In two parts. Published by the Carnegie Institution of Washington, Washington, D. C. 1913. Starch: Metabolism and Structure (Springer, 2015). Gott, B., Barton, H., Samuel, D. & Torrence, R. in Ancient Starch Research (eds Torrence, R. & Barton, H.) 35–45 (Routledge, 2016). Li, M. et al. Starch grains from dental calculus reveal ancient plant foodstuffs at Chenqimogou site, Gansu Province. China Earth Sci. Zona, S. Raphides in palm embryos and their systematic distribution. Jara-Arancio, P. et al. Genetic diversity and population structure of Jubaea chilensis, an endemic and monotype gender from Chile, based on SNP markers. Delhon, C. & Orliac, C. in The Gotland Papers: Selected Papers from the VII International Conference on Easter Island and the Pacific: Migration, Identity, and Cultural Heritage (eds Wallin, P. & Martinsson-Wallin, H.) 97–110 (Gotland Univ. Patterer, N. I., Passeggi, E. & Zucol, A. F. Análisis fitolíticos de suelos del sudoeste de la Provincia de Entre Ríos (Argentina) como una herramienta para comprender sus procesos pedológicos. Semprebon, G. M., Rivals, F., Solounias, N. & Hulbert, R. C. Jr Paleodietary reconstruction of fossil horses from the Eocene through Pleistocene of North America. Solounias, N. & Semprebon, G. Advances in the reconstruction of ungulate ecomorphology with application to early fossil equids. Brummitt, R. K., Pando, F., Hollis, S. & Brummitt, N. A. World Geographical Scheme for Recording Plant Distributions Vol. 951 (International Working Group on Taxonomic Databases for Plant Sciences (TDWG), 2001). Dantas, M. A. T., Liparini, A., Asevedo, L., de Melo França, L. & Cherkinsky, A. Annual isotopic diet (δ13C, δ18O) of Notiomastodon platensis (Ameghino, 1888) from Brazilian Intertropical Region. Jordano, P. in Seeds: The Ecology of Regeneration in Plant Communities 2nd edn (ed. Fenner, M.) 125–165 (CABI Publishing, 2022). Beirne, C. et al. Climatic and resource determinants of forest elephant movements. Nevo, O., Schmitt, M. H., Ayasse, M. & Valenta, K. Sweet tooth: elephants detect fruit sugar levels based on scent alone. González-Guarda, E. et al. The extinct Notiomastodon platensis (Proboscidea, Gomphoteriidae) inhabited Mediterranean ecosystems during the Late Pleistocene in north-central Chile (31° S–36° S). Blake, S. & Inkamba‐Nkulu, C. Fruit, minerals, and forest elephant trails: do all roads lead to Rome? Asevedo, L. et al. Palynological analysis of dental calculus from Pleistocene proboscideans of southern Brazil: a new approach for paleodiet and paleoenvironmental reconstructions. Asevedo, L. et al. Isotopic paleoecology (δ13C, δ18O) of Late Quaternary herbivorous mammal assemblages from southwestern Amazon. Onstein, R. E. et al. To adapt or go extinct? The fate of megafaunal palm fruits under past global change. Van Zonneveld, M. et al. Human diets drive range expansion of megafauna-dispersed fruit species. Muñoz-Concha, D., Muñoz, K. & Loayza, A. P. Anachronic fruit traits and natural history suggest extinct megafauna herbivores as the dispersers of an endangered tree. Cordero, S., Gálvez, F. & Fontúrbel, F. E. Multiple anthropogenic pressures lead to seed dispersal collapse of the southernmost palm Jubaea chilensis. Loayza, A. P., Luna, C. A. & Calviño-Cancela, M. Predators and dispersers: context-dependent outcomes of the interactions between rodents and a megafaunal fruit plant. Muñoz‐Concha, D. et al. Could foxes be surrogate seed dispersers of a megafaunal fruit vine in southern South America? Pires, M. M. et al. Reconstructing past ecological networks: the reconfiguration of seed-dispersal interactions after megafaunal extinction. Smith, F. A. et al. After the mammoths: the ecological legacy of Late Pleistocene megafauna extinctions. Prisms: Extinction 1, e9 (2023). Bunney, K., Bond, W. J. & Henley, M. Seed dispersal kernel of the largest surviving megaherbivore—the African savanna elephant. Sekar, N., Lee, C. L. & Sukumar, R. Functional nonredundancy of elephants in a disturbed tropical forest. Arroyo, M. T. et al. El hotspot chileno, prioridad mundial para la conservación. Diversidad de ecosistemas, ecosistemas terrestres (Comisión Nacional del Medio Ambiente (Chile), 2006). Malhi, Y. et al. Megafauna and ecosystem function from the Pleistocene to the Anthropocene. Lim, J. Y., Svenning, J.-C., Göldel, B., Faurby, S. & Kissling, W. D. Frugivore-fruit size relationships between palms and mammals reveal past and future defaunation impacts. Barnosky, A. D. et al. Merging paleobiology with conservation biology to guide the future of terrestrial ecosystems. Phillips, D. L. Converting isotope values to diet composition: the use of mixing models. D'Angela, D. & Longinelli, A. Oxygen isotopes in living mammal's bone phosphate: further results. Koch, P. L., Tuross, N. & Fogel, M. L. The effects of sample treatment and diagenesis on the isotopic integrity of carbonate in biogenic hydroxylapatite. Tornero, C., Bălăşescu, A., Ughetto-Monfrin, J., Voinea, V. & Balasse, M. Seasonality and season of birth in Early Eneolithic sheep from Cheia (Romania): methodological advances and implications for animal economy. Semprebon, G. M., Godfrey, L. R., Solounias, N., Sutherland, M. R. & Jungers, W. L. Can low-magnification stereomicroscopy reveal diet? Xafis, A., Nagel, D. & Bastl, K. Which tooth to sample? A methodological study of the utility of premolar/non-carnassial teeth in the microwear analysis of mammals. Zizka, A., Andermann, T. & Silvestro, D. IUCNN–deep learning approaches to approximate species' extinction risk. R Core Team. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2013). Rivals, F., Semprebon, G. & Lister, A. An examination of dietary diversity patterns in Pleistocene proboscideans (Mammuthus, Palaeoloxodon, and Mammut) from Europe and North America as revealed by dental microwear. Semprebon, G. M. et al. Dietary reconstruction of pygmy mammoths from Santa Rosa Island of California. Download references This study was funded by ANID FONDECYT postdoctoral project 3200806 and ANID FONDECYT initiation project 11241036 to E.G. The study was also partly supported by the Spanish Ministry of Science and Innovation through the María de Maeztu excellence accreditation (CEX2019-000945-M), grant 2021-SGR-01237 from the Generalitat de Catalunya (AGAUR), and the READ project, funded by the European Research Council (ERC) under the European Union's HORIZON ERC programme (grant agreement no. was supported by the Spanish Ministry of Science and Innovation (RYC 2020-029404-I). acknowledge funding from ANID grant FB210006 to the Instituto de Ecología y Biodiversidad (IEB). also acknowledges partial support from ANID FONDECYT grant 1231820. L.A. thanks the Conselho Nacional de Desenvolvimento Científico e Tecnológico – CNPq (process no. 173000/2023-1) for their postdoctoral scholarship. Instituto de Ciencias de la Ingeniería, Universidad de O'Higgins, Rancagua, Chile Institut Català de Paleoecologia Humana i Evolució Social, Centres de Recerca de Catalunya (IPHES-CERCA), Tarragona, Spain Erwin González-Guarda, Florent Rivals, Iván Ramírez-Pedraza & Carlos Tornero Instituto de Ecología y Biodiversidad (IEB), Santiago, Chile Andrea P. Loayza, Ricardo A. Segovia & Claudio Latorre Departamento de Botánica, Facultad de Ciencias Naturales y Oceanográficas, Universidad de Concepción, Concepción, Chile Ricardo A. Segovia Departament d'Història i Història de l'Art, Universitat Rovira i Virgili (URV),  Tarragona, Spain Florent Rivals, Alia Petermann-Pichincura & Iván Ramírez-Pedraza Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain Departamento de Estratigrafia e Paleontologia, Faculdade de Geologia, Universidade do Estado do Rio de Janeiro, Rio de Janeiro, Brazil Department of Prehistory, Autonomous University of Barcelona (UAB), Barcelona, Spain Carlos Tornero Departamento de Antropología, Facultad de Ciencias Sociales, Pontificia Universidad Católica de Chile, Santiago, Chile Rafael Labarca Centro UC Desierto de Atacama and Facultad de Ciencias Biológicas, Pontificia Universidad Católica de Chile, Santiago, Chile Centro de Regulación del Genoma (CRG), Pontificia Universidad Católica de Chile, Santiago, Chile You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar E.G.-G, project leader, led the manuscript writing, performed the analyses and supervised all aspects of the paper. co-authored a significant portion of the manuscript and categorized the species in the tree species database as megafaunal or non-megafaunal fruit plants. co-authored a significant portion of the manuscript and performed the tree risk extinction modelling analyses. A.P.-P. co-authored parts of the Methods and Results sections and conducted the analysis of the dental calculus samples. I.R.-P. co-authored parts of the Methods and Results sections and conducted the analysis of the dental microwear samples. L.A. co-authored parts of the Methods and Results sections and conducted the analysis of the ecological niche of N. platensis. contributed to the laboratory work, specifically regarding the stable isotope analyses. acquired the dental calculus samples from the N. platensis molars. assisted in writing and revising the manuscript and analysed the dental microwear samples. significantly contributed to the supervision, writing and editing of the manuscript, and provided additional funding and analyses. Correspondence to Erwin González-Guarda or Claudio Latorre. The authors declare no competing interests. Nature Ecology & Evolution thanks Pedro Jordano, Dimila Mothé, Haijun Song and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Graphical results of isotopic analyses derived from multiple sources showing the mean values of δ13Cbioapatite (‰, VPDB) and δ18Ometeoric water (‰, SMOW) for Notiomastodon platensis and Pudu puda. The geographical distribution of the main species of fleshy fruits that we analyzed in central Chile is observed. a, Geoffroea decorticans. b, Trichocereus chilensis. c, Jubaea chilensis. d, Beilschmiedia miersii. e, Gomortega keule. Supplementary Methods, Results, Discussion, Tables 1–7 and References. A palaeoenvironmental reconstruction of the landscape of the extinct proboscidean species N. platensis can be observed for the Late Pleistocene of central Chile (that is, La Campana National Park). Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions González-Guarda, E., Loayza, A.P., Segovia, R.A. et al. Fossil evidence of proboscidean frugivory and its lasting impact on South American ecosystems. Nat Ecol Evol  (2025). Download citation Received: 18 March 2024 Accepted: 15 April 2025 Published: 13 June 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Ecology & Evolution (Nat Ecol Evol) ISSN 2397-334X (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250613013808.htm'>Guest molecules ride perfect waves in dna droplets—A breakthrough for synthetic biology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 08:08:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at Johannes Gutenberg University Mainz (JGU), the Max Planck Institute for Polymer Research and the University of Texas at Austin have identified a form of molecular motion that has not previously been observed. When what are known as 'guest molecules' - molecules that are accommodated within a host molecule - penetrate droplets of DNA polymers, they do not simply diffuse in them in a haphazard fashion, but propagate through them in the form of a clearly-defined frontal wave. "This is an effect we did not expect at all," points out Weixiang Chen of the Department of Chemistry at JGU, who played a major role in the discovery. It is usually the case that molecules are distributed throughout liquids by means of simple diffusion. For instance, if you add a blue dye to a glass of water, the dye gradually disperses in the liquid, forming soft, blurry color gradients. However, the observed behavior of guest molecules in DNA droplets is quite different. Moreover, these droplets have their counterparts in biological cells, which are able to employ similar condensates to arrange complex biochemical processes without the need for membranes. "Our synthetic droplets thus represent an excellent model system with which we can simulate natural processes and come to better understand them," emphasizes Chen. Chen adds: "The well-defined, highly concentrated front continues to move forward in a linear fashion over time, driven by chemical binding, material conversion and programmable DNA interactions. Something that is completely new when it comes to soft matter." This would also be of interest when it comes to the treatment of neurodegenerative disorders in which proteins migrate from cell nuclei into the cytoplasm, forming condensates there. "It is quite conceivable that we may be able to find a way of influencing these aging processes with the aid of our new insights, so that, over the long term, an entirely new approach to the treatment of neurodegenerative diseases could emerge," concludes Walther. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250612081312.htm'>Scientists warn of bat virus just one mutation from infecting humans</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 05:06:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A group of bat viruses closely related to the deadly Middle East respiratory syndrome coronavirus (MERS-CoV) could be one small mutation away from being capable of spilling over into human populations and potentially causing the next pandemic. A recent study published in the journal Nature Communicationsexamined an understudied group of coronaviruses known as merbecoviruses -- the same viral subgenus that includes MERS-CoV -- to better understand how they infect host cells. The research team, which included scientists at Washington State University, the California Institute of Technology and the University of North Carolina, found that while most merbecoviruses appear unlikely to pose a direct threat to people, one subgroup known as HKU5 possesses concerning traits. "Merbecoviruses - and HKU5 viruses in particular - really hadn't been looked at much, but our study shows how these viruses infect cells," said Michael Letko, a virologist at WSU's College of Veterinary Medicine who helped to spearhead the study. "What we also found is HKU5 viruses may be only a small step away from being able to spill over into humans." During the past two decades, scientists have cataloged the genetic sequences of thousands of viruses in wild animals, but, in most cases, little is known about whether these viruses pose a threat to humans. Letko's lab in WSU's Paul G. Allen School for Global Health focuses on closing that gap and identifying potentially dangerous viruses. For their most recent study, Letko's team targeted merbecoviruses, which have received limited attention apart from MERS-CoV, a zoonotic coronavirus first noted in 2012 that is transmitted from dromedary camels to humans. It causes severe respiratory disease and has a mortality rate of approximately 34%. While most merbecoviruses appear unlikely to be able to infect humans, HKU5 viruses - which have been found across Asia, Europe, Africa and the Middle East - were shown to use a host receptor known as ACE2, the same used by the more well-known SARS-CoV-2 virus that causes COVID-19. One small difference: HKU5 viruses, for now, can only use the ACE2 gene in bats, but do not use the human version nearly as well. Researchers on another study that came out earlier this year analyzed one HKU5 virus in China that has already been documented to have jumped into minks, showing there is potential for these viruses to cross species-barriers. Up until this point, such structural analysis required months of lab work and specialized equipment. The results matched those recently documented by a research team that used traditional approaches. Jefferson's work was supported by an NIH T32 training grant. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250612081317.htm'>Brain-computer interface restores real-time speech in als patient</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-13 04:27:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at the University of California, Davis, have developed an investigational brain-computer interface that holds promise for restoring the voices of people who have lost the ability to speak due to neurological conditions. In a new study published in the scientific journal Nature, the researchers demonstrate how this new technology can instantaneously translate brain activity into voice as a person tries to speak -- effectively creating a digital vocal tract. The system allowed the study participant, who has amyotrophic lateral sclerosis (ALS), to "speak" through a computer with his family in real time, change his intonation and "sing" simple melodies. "Translating neural activity into text, which is how our previous speech brain-computer interface works, is akin to text messaging. By comparison, this new real-time voice synthesis is more like a voice call," said Sergey Stavisky, senior author of the paper and an assistant professor in the UC Davis Department of Neurological Surgery. Decoding brain signals at heart of new technology His ability to communicate through a computer has been made possible with an investigational brain-computer interface (BCI). "The main barrier to synthesizing voice in real-time was not knowing exactly when and how the person with speech loss is trying to speak," said Maitreyee Wairagkar, first author of the study and project scientist in the Neuroprosthetics Lab at UC Davis. "Our algorithms map neural activity to intended sounds at each moment of time. This makes it possible to synthesize nuances in speech and give the participant control over the cadence of his BCI-voice." The brain-computer interface was able to translate the study participant's neural signals into audible speech played through a speaker very quickly -- one-fortieth of a second. This short delay is similar to the delay a person experiences when they speak and hear the sound of their own voice. He was able to modulate the intonation of his generated computer voice to ask a question or emphasize specific words in a sentence. The participant also took steps toward varying pitch by singing simple, short melodies. His BCI-synthesized voice was often intelligible: Listeners could understand almost 60% of the synthesized words correctly (as opposed to 4% when he was not using the BCI). The process of instantaneously translating brain activity into synthesized speech is helped by advanced artificial intelligence algorithms. This gave the researchers information about what he was trying to say. "Our voice is part of what makes us who we are. "The results of this research provide hope for people who want to talk but can't. This kind of technology could be transformative for people living with paralysis." It will be crucial to replicate these results with more participants, including those who have speech loss from other causes, such as stroke. To learn more about the study, visit braingate.org or contact [email protected]. Materials provided by University of California - Davis Health. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            