
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-22</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/does-tylenol-use-during-pregnancy-cause-autism-what-the-research-shows/'>Does Tylenol Use during Pregnancy Cause Autism? What the Research Shows</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 17:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>President Donald Trump previewed the announcement on Sunday, saying at a memorial for the late Charlie Kirk, “I think we found an answer to autism.” A press conference will take place this afternoon. There is no simple answer to what causes autism, more than 50 years of scientific research has shown. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “The evidence [from a handful of studies] was really mixed, and the effects were really small,” said David Mandell, a professor of psychiatry studying autism at the University of Pennsylvania, in an interview with Scientific American earlier this month. Acetaminophen is one of the few pain relievers deemed safe to take during pregnancy, in consultation with doctors, according to the American College of Obstetricians and Gynecologists (ACOG). “Despite recent unfounded claims, there's no clear evidence linking prudent use to issues with fetal development. When pain relief is needed during pregnancy, acetaminophen should be used in moderation, and after consulting your doctor,” wrote the professional body in a statement on September 5. Here's what we know about Tylenol and the causes of autism. The majority of autism cases can be linked to genetic causes. But that DNA blueprint doesn't determine everything: as a fertilized human egg turns into an embryo and then into a fetus, environmental factors can influence how those genetic instructions are carried out. Some nongenetic factors have been repeatedly linked to an elevated risk of developing autism spectrum disorder. (But even this effect might tie back to genetics because, as people age, they accumulate more genetic mutations that they might pass on to their offspring.) Other nongenetic factors, such as premature birth or birth by cesarean section, maternal diabetes and exposure to air pollution or certain antiseizure drugs in utero have been shown to increase a person's risk of autism in some studies. That's because it is usually impossible, impractical or unethical to run randomized trials on pregnant populations. A handful of studies from the U.S. and Europe have found modest increases in autism risk among the offspring of people who took acetaminophen while pregnant. The 2024 analysis looked at nearly 2.5 million people born in Sweden between 1995 and 2019. This elevated risk disappeared when researchers just looked at sibling-pair cases in which a person took acetaminophen during one pregnancy and not the other, however. This suggests that what initially looked like an elevated risk of autism from acetaminophen during pregnancy may have been a result of other risk factors. This means that the fever or underlying infections Tylenol was used to treat could be part of what drove the increased risk of autism that was shown in some studies. If a pregnant person is hospitalized with an infection, the likelihood that their child will develop autism increases by approximately 30 percent, a study of more than two million people found. “We believe independent, sound science clearly shows that taking acetaminophen does not cause autism. We strongly disagree with any suggestion otherwise and are deeply concerned with the health risk this poses for expecting mothers,” a spokesperson of Kenvue, Tylenol's manufacturer, said in a statement sent to Scientific American. “We know that folate deficiency during pregnancy results in neural tube defects or increases risk for neural tube defects,” says Mandell, referring to the structure in an embryo that eventually develops into the brain and spinal cord. “There's some thought that it could also result in learning disability, developmental delay—including autism.” This idea has led many doctors and parents of people with autism to try treating the condition with B vitamins, including leucovorin, also known as folinic acid. A few small, randomized controlled trials of folinic acid as a treatment for autism spectrum disorder have been conducted, and they have shown small effects. “Should we run a large-scale randomized trial of folinic acid and autism? And the reason we should do it is because the science needs to catch up with practice here [because] it's already being prescribed,” Mandell continues. Autism rates have been rising, something that Robert F. Kennedy, Jr., frequently refers to as an “epidemic.” The causes of this rise are relatively well known, however. As our understanding of autism has improved, the diagnostic criteria have broadened, and a major change occurred in 2013 when autism and several other conditions were combined into one continuous spectrum disorder in the Diagnostic and Statistical Manual of Mental Disorders. Improved screening is alerting clinicians to more cases in children, and adults whose autism was missed when they were younger are seeking diagnoses. Additionally, improved understanding of autism has allowed for better diagnoses of people who, in prior decades, may have received other diagnoses such as “intellectual disability.” But science often does not happen on such short timelines, experts say. The administration's decision to highlight acetaminophen and folate does not come from new evidence but rather from previously published results that are mixed and point to the need for more research. “A press statement that talks about a potential association will cause lots of fear,” said Debra Houry, former chief medical officer of the Centers for Disease Control and Prevention, in a press call this morning hosted by the group Defend America Action. Houry participated alongside other former CDC leaders who had resigned this month. “If there is not the science to back it up, we will see practice changes, worried moms, all sorts of things, and that's not appropriate. You have to be grounded in gold standard science,” she said. As a multimedia journalist, she contributes to Scientific American's podcast Science Quickly. Parshall's work has also appeared in Quanta Magazine and Inverse. She graduated from New York University's Arthur L. Carter Journalism Institute with a master's degree in science, health and environmental reporting. She has a bachelor's degree in psychology from Georgetown University. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/culture/gaming/g66040414/prime-big-deal-days-lego-deals-2025/'>Legos Are Getting Major Discounts Ahead of Prime Big Deal Days—Save Up to 40% Best-Selling Sets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 15:22:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Before you ask, yes, there are a bunch of Star Wars sets on sale. We may earn commission if you buy from a link. There's no shortage of Lego deals during one of Amazon's big savings events, and October's Big Deal Days is no different. Amazon likes to get started early, and we can hardly blame them. Of course, it would be wise to bookmark this page going forward—we expect a lot of action this time around. Be prepared for more great sets to go on sale, and I'll update this page as they continue to pop up. He's iconic, and it's impossible to talk about Lego Star Wars without including the Sith Lord himself. Some newer live-action iterations have been universally panned, but the recent Lilo & Stitch remake was not one of them. We'll switch over to the good guys here, with this R2-D2 set. Well, at least one made out of plastic bricks. Recreate the magic with this 165‑piece set for ages 7 and up. It includes Harry Potter, Ron Weasley, plus Hedwig the owl and Scabbers the rat, along with wands and a suitcase. The car features opening doors and a removable roof for easy access, and there's storage space in the boot for accessories. You can't buy an R2-D2 set without also buying its inseparable (and objectively more annoying) droid counterpart. Another 25th anniversary centerpiece to add to your collection, this C-3PO figure scales perfectly with his little buddy and comes with a Lego C-3PO minifigure and commemorative plaque. This is a significant departure from the older, albeit much larger Lego set that was released a few years ago and retailed for well over $400. Arguably one of the best Star Wars Lego sets to ever exist, this replica also comes with a Lego Star Wars 25th Anniversary Brick for display to show off your fanhood. This set is a 598‑piece skyline‑builder that blends design, architecture, and display. It includes iconic landmarks such as the Flatiron Building, Chrysler Building, Empire State Building, One World Trade Center, and the Statue of Liberty—all built on a sleek 4×32 base topped with a “New York City” nameplate. For ages 12 and older, it offers enough challenge to engage adults, while its visual appeal and compact footprint make it a striking piece for a bedroom shelf or an office desk. The Lego Classic Creative Brick Box is a vibrant 790‑piece building set packed with 33 different colors, specially designed pieces like doors, windows, tires and rims, plus two green baseplates. Ideal for kids aged 4 and older, it's a perfect starter kit or addition to any Lego collection—great for home or classroom play. This miniature version of Ahsoka's legendary Jedi Ineptor comes with some key minifigures, including Ahsoka Tano herself, Anakin Skywalker, and an R7-A7 Lego droid figure. It's a pretty solid replica, too, and features insane functionality like an opening cockpit for the figures, a custom space for R7-A7, fully adjustable wings, dual spring-loaded shooters, and additional lightsaber storage. We can't not feature a car set at Popular Mechanics, right? This toy also features two Lego mini-figures, “drivers,” in addition to a pair of highly detailed collectible cars. He has a strong affinity for cats, Batman, and early 90's Nicolas Cage films, in that order. Sarah Connor is a Reviews Editor for Popular Mechanics. She also contributes to Runner's World, Bicycling, Best Products, and Biography.com. She holds a master's in Journalism from Syracuse University and a dual bachelor's in English and Communications from the University of Pittsburgh. When she isn't writing up a review of the newest gadgets, you can find her heading to the nearest rink for a figure skating session or enjoying the outdoors with her beloved Australian Shepherd, Rosie. These Are The Best Early Prime Day Deals Worth Buying DeWalt Tools Are On Sale For October Prime Day The Prime Day Vacuum Deals of 2025 Are Out This $42 Projector Turns Your Phone Into a Theater</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a68001332/tunnels-unknown-organism-rocks/'>Scientists Discovered the Tunnels of a Possibly Unknown Ancient Lifeform</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 14:51:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Looks like limestone's on the menu for some unknown creepy crawly. For most organisms on Earth, rocks are objects, not food. But for one oddball microbe, desert limestone seems to have been on the menu. Recently, geologist Cees Passchier from Johannes Gutenberg University Mainz came across what looked like tiny burrows in marble and limestone in the deserts of Namibia, Oman, and Saudi Arabia. “The investigated areas are currently arid, but experience occasional rain showers and regular dense coastal fog, while wet periods occurred in the past.” So, what type of microogranisms could have made the burrows? Bacteria, fungi and lichens have shown that they can survive extreme conditions, and some of them are endolithic, meaning they live inside rocks. Fungi secrete digestive agents that were not present in the rock, and they also create a complex network of hyphae, or filaments, known as a mycelium. The burrows were parallel and evenly spaced, which would be unusual for fungi, and there were no other patterns observed. Because the burrows were found to be too wide to have only been made by one organism at a time, and they showed growth rings, its was more likely that they were formed by colonies of microbes. Calcium carbonate dust found in the tunnels is also a common excretion from microbes that live in these types of rocks. However, no fossilized organisms have yet been found—just evidence of their existence. These setbacks, however, did not rule out the idea of life. On the contrary—while weathering or abiotic chemical processes can create structures mistaken for signs of life, a thorough microscopic examination showed this was not the case. “As no known chemical or physical weathering mechanism can explain this phenomenon with the microstructural and geochemical observations presented here, and the micro-burrows form inside the host rock,” Passchier and his colleagues said in the same study, “we suggest that they are of biological origin.” Whatever microbes carved out the tunnels have been long dead, though there are questions around whether the mystery species still exists. Maybe it's still creeping around somewhere, digging new tunnel systems for us to one day uncover. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a68001072/firefighters-ancient-stone-face-discovery-in-kazakhstan/'>Firefighters Went on a Routine Patrol—and Stumbled Upon a Breathtaking Ancient Face</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 14:28:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Because Kazakhstan sits at such a crossroad of history, it's also one of the most archaeologically rich places on Earth—and a new discovery by unsuspecting fire crews once again proves this fact. While patrolling the Sandyktau region, an area located some 200 miles northwest of Kazakhstan's capital, Astana, Nursultan Ashkenov and Akhmet Zaripov, both employees of the district fire service, stumbled upon an austere face carved into the side of a granite boulder. The face is around 10.5 inches long and roughly 8 inches wide, and some archeologists hypothesize that the carving was likely part of some ritual complex. “The face is clearly visible,” Sergey Yarygin, a scientist at the Alkey Margulan Institute of Archaeology, said to Archaeology Magazine (click link to see images), “with large eyes, a long straight nose, and protruding lips. However, figuring out when this face was used for such ritualistic purposes is a bit more complicated. According to Yarygin, similar carvings have been found at Bronze Age sites across Central Asia and Western Europe, but the carving also holds a striking resemblance to other types of iconography found in Iron Age sites in southern Siberia or even medieval Turkish cultures. The mysteries of Kazakhstan's historical melange strike again. Teams investigating the carving will likely need more time to determine its age and what culture might've created the stunningly somber carving. Archaeologists uncovered a number of artifacts from the northwestern Pavlodar region, including pottery fragments, kitchen utensils, and an arrowhead dated to around the mid-13th to 8th century BCE. Other discoveries in the central Karaganda region also point to a variety of artifacts likely belonging to the Alakul culture, a subculture of the Andronovo culture, that thrived in this region around 2000 to 1700 BCE. This Arm Bone Could Be the Oldest Mummy Ever Found Experts Find Missing Piece of Ramesses II Statue Fully Preserved Roman War Helmet Found in the Sea</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/almost-impossible-deep-earth-diamonds-confirm-how-these-gems-form/'>‘Almost Impossible' Deep-Earth Diamonds Confirm How These Gems Form</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 13:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A pair of diamonds that formed hundreds of kilometers deep in Earth's malleable mantle both contain specks of materials that form in completely opposing chemical environments—a combination so unusual that researchers thought their coexistence was “almost impossible.” The substances' presence provides a window into the chemical goings-on of the mantle and the reactions that form diamonds. The two diamond samples were found in a South African mine. As with plenty of other precious gemstones, they contain what are called inclusions—tiny bits of surrounding rocks captured as the diamonds form. That's especially true when diamonds form deep in the unreachable mantle, because they carry these inclusions basically undisturbed to the surface—the only way those minerals can rise hundreds of kilometers without being altered from their original deep-mantle state. The two new diamond samples each contain inclusions of carbonate minerals that are rich in oxygen atoms (a state known as oxidized) and oxygen-poor nickel alloys (a state known as reduced, in the parlance of chemistry). Typically, diamond inclusions show just one or the other, so the presence of both perplexed Yaakov Weiss, a senior lecturer in Earth sciences at the Hebrew University of Jerusalem, and his colleagues—so much so that they initially put the samples aside for a year in confusion, he says. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Theoretical calculations have given researchers a notion of how the planet shifts from oxidized to reduced with depth. “We knew about that reduction with some empirical data, with real samples down to maybe 200 kilometers,” says Maya Kopylova, a professor of Earth, ocean and atmospheric science at the University of British Columbia, who was not involved in the new study but who wrote an editorial accompanying the paper. “What happened below 200 km [was] just our idea, our models, because it's so difficult to get the materials.” There are only a few samples from below this depth, she said. These new samples, which come from between 280 and 470 km below Earth's surface, provide the first real-world fact-check on this theoretical mantle chemistry. One finding, Weiss says, is that oxidized melted material exists deeper than expected. Kimberlites, the erupted rocks that bring diamonds to the surface, are oxidized, so researchers had thought they couldn't originate much below 300 km of depth. Diamond-forming reactions likely happen when carbonate fluids are dragged down by subducting tectonic plates, which bring oxygen-heavy minerals in contact with the metal alloys of the mantle, Weiss says. (Another way chemists think diamonds may form is by precipitating out of carbon-rich fluids that cool as they rise upward in the mantle, like sugar crystalizing from syrup. The new paper doesn't rule out that process happening as well.) The nickel-rich inclusions might also help explain an odd occurrence in some diamonds: occasional atoms of nickel seem to replace the carbon of these diamonds' crystal lattice. That's been a mystery, Kopylova says, because nickel is so much heavier than carbon that it shouldn't be able to easily swap into the crystal structure. “Now, looking at these data, I see that it might be just a sign of diamond formation at certain depths,” she says. Stephanie Pappas is a freelance science journalist based in Denver, Colo. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03024-5'>Gestational diabetes linked to autism in study: what scientists say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 13:29:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Rachel Fieldhouse is a reporter for Nature in Sydney, Australia. Gestational diabetes can have long-term effects on women's health. Last week, a study involving more than nine million pregnancies reported that children whose mothers had gestational diabetes during pregnancy had a higher chance of developing attention deficit–hyperactivity disorder (ADHD) and autism than did children whose mothers didn't have the condition. It is not the first to link gestational diabetes to neurodevelopmental disorders in children, but it is one of the largest. Researchers pooled results from 48 studies across 20 countries, finding that children born to people with gestational diabetes had lower IQ scores, a 36% higher risk of ADHD and a 56% higher risk of autism spectrum disorders. Estimates suggest the prevalence of autism in the general population is one in 127 people1 and between 3-10%2 of children and teenagers have ADHD. The latest results mirror those of another meta-analysis3 published in The Lancet Diabetes & Endocrinology journal in June, which included 56 million mother–child pairs and found that all types of diabetes in pregnancy, including type 1, type 2 and gestational diabetes, increase the risk of the baby developing ADHD and autism. “There's no doubt that there is a signal here, but certainly further research is required,” says Alex Polyakov, an obstetrician and researcher at the University of Melbourne in Australia. On Sunday, while speaking at the memorial for conservative activist Charlie Kirk, Trump said: “I think we found an answer to autism. How about that?” The Washington Post reports that Trump's administration is poised to announce a link between autism and the use of acetaminophen, also called paracetamol, during pregnancy. Medical guidelines say acetominophen is safe to use during pregnancy. Environmental factors, such as folic-acid deficiency, air pollution and gestational diabetes, have a small-to-moderate effect on the risk of these conditions, says Ling-Jun Li, a clinician–scientist at the National University of Singapore and a co-author on the latest gestational diabetes study. Polyakov agrees: “In practice, both genetic predisposition and environmental factors likely interact.” Studies over the past two decades5 have shown that gestational diabetes has long-term effects on women's health, such as increasing their risk of type 2 diabetes, heart disease, stroke and chronic kidney disease. Pregnant mice illuminate risk factors that could lead to autism Your risk of dying from chronic disease has dropped — if you live in these countries Hope for diabetes: CRISPR-edited cells pump out insulin in a person — and evade immune detection Obesity drugs made in China could power next wave of treatments Peroxisomal metabolism of branched fatty acids regulates energy homeostasis ‘Brain dial' turns food consumption on or off in mice of Microbiology, Biochemistry, and Molecular Genetics at Rutgers New Jersey Medical School of Microbiology, Biochemistry, and Molecular Genetics at Rutgers New Jersey Medical School Biogen is seeking a Vice President of Clinical Development Pregnant mice illuminate risk factors that could lead to autism An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/09/250922074936.htm'>Scientists finally capture water's hidden state that's both solid and liquid</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 12:46:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Despite being one of the most familiar substances on Earth, water holds many secrets that scientists are still working to understand. When confined to extremely small spaces -- such as within certain proteins, minerals, or artificial nanomaterials -- water behaves in ways that are drastically different from its bulk liquid form. These confinement effects are critical for many natural and technological processes, including regulating the flow of ions through cell membranes and the properties of nanofluidic systems. In this unique phase, water behaves as if it were on the cusp of freezing and melting at the same time, thus defying simple liquid or solid classifications. However, it has proven difficult to study the premelting state and other confined water dynamics in detail. While techniques such as diffraction methods (example: X-ray analysis) are useful for pinpointing the positions of atoms other than hydrogen, they are not sensitive enough to capture the picosecond-scale rotational motion of hydrogen and the motion of individual water molecules. In a recent study, a research team led by Professor Makoto Tadokoro alongside Lecturer Fumiya Kobayashi and first-year PhD student Mr. Tomoya Namiki, from the Department of Chemistry, Tokyo University of Science, Japan, shed new light on the mysteries of confined water. To perform their experiments, the team produced hexagonal rod-like crystals, with quasi-one-dimensional channels containing a nanopore approximately 1.6 nm in diameter and filled them with heavy water (D2O). It essentially constitutes a novel phase of water in which frozen H2O layers and slowly moving H2O coexist," explains Prof. Tadokoro. Simply put, this means that while the water molecules' positions were relatively fixed as one would expect of a solid, their rotational motions were extremely fast and liquid-like. Taken together, these findings build toward a more comprehensive understanding of how water behaves in extreme confinement. Looking ahead, these insights could also lead to practical innovations. "By creating new ice network structures, it may be possible to store energetic gases such as hydrogen and methane and develop water-based materials such as artificial gas hydrates," says Prof. Tadokoro. This work was supported by JSPS KAKENHI Grant-in-Aid for Scientific Research (B) JP23K26672 and JSPS KAKENHI Grant-in-Aid for Early-Career Scientists JP23K13767 from the Ministry of Education, Culture, Sports, Science, and Technology, Japan. Deep-Sea Ghost Sharks Grow Teeth on Their Foreheads, and Scientists Finally Know Why New Eye Drops Sharpen Aging Eyes in Just One Hour Half Quit “Miracle” Weight-Loss Shots Ozempic and Wegovy Within One Year Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250921091002.htm'>A hormone that silences the immune system may unlock new cancer treatments</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 12:33:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at UT Southwestern Medical Center have discovered how a hormone interacts with a receptor on the surface of immune cells to shield cancer cells from the body's natural defenses. The findings, published in Nature Immunology, could lead to new immunotherapy approaches for treating cancer as well as potential treatments for inflammatory disorders and neurologic diseases. Our study suggests that receptors on these myeloid cells get stimulated by this hormone and end up suppressing the immune system," said Cheng Cheng "Alec" Zhang, Ph.D., Professor of Physiology and a member of the Harold C. Simmons Comprehensive Cancer Center at UT Southwestern. Current immunotherapies, such as immune checkpoint inhibitors, are effective for only about 20%-30% of cancer patients, Dr. Zhang said, suggesting that there are multiple ways that cancers evade attack from the immune system. Stimulating this receptor blocked the myeloid cells' ability to attack tumors. Dr. Zhang, Dr. Yang, and their colleagues then did a genome-wide screen of all proteins that might interact with LILRB4. Although researchers have suggested that SCG2 plays a role in immune response, its function and receptor were unknown. In mice genetically altered to express the human form of LILRB4, injected cancer cells that produced SCG2 grew rapidly as tumors. Treating these mice with an antibody that blocks LILRB4 significantly slowed cancer growth, as did artificially ridding the animals' bodies of SCG2. Dr. Zhang suggested that disrupting this interaction could someday offer a new immunotherapy option to treat cancer. Dr. Zhang and his colleagues plan to investigate both ideas in future studies. Dr. Xuewu Zhang and Dr. Xu are members of the Simmons Cancer Center. 111077), the National Institutes of Health (R35GM130289), and NCI Cancer Center Support Grant (P30CA142543). Dr. Alec Zhang holds equity in and had sponsored research agreements with Immune-Onc. Deep-Sea Ghost Sharks Grow Teeth on Their Foreheads, and Scientists Finally Know Why Half Quit “Miracle” Weight-Loss Shots Ozempic and Wegovy Within One Year Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/vaccine-panel-overhaul-head-trauma-in-sports-and-strange-reproduction-in/'>New Vaccine Panel, Hidden Brain Damage in Sports and Ants with Hybrid Offspring</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Vaccine Policy Shift, Brain Changes in Athletes and Ants That Harness Another Species' DNA A revamped CDC advisory committee faces vaccine debates, studies reveal brain changes in athletes, and climate change drives deadly heat waves across Europe. For Scientific American's Science Quickly, I'm Rachel Feltman. On Thursday and Friday of last week the Centers for Disease Control and Prevention's Advisory Committee on Immunization Practices, or ACIP, met to review and vote on recommendations for official U.S. vaccine guidelines. An agenda released ahead of last week's meeting stated that the ACIP would propose recommendations for the hepatitis B, COVID and measles, mumps, rubella and varicella vaccines. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Here's Lauren Young, associate editor for health and medicine at Scientific American, with a quick update as of Friday. Lauren Young: So far we've seen a few votes come through. This is the measles, mumps, rubella, and varicella vaccine. Varicella is commonly known as chickenpox, and they decided not to recommend the single combined shot for kids younger than age four. There wasn't really any changes to this, but members did vote in support that all pregnant people should be tested for hepatitis B infection, which is the current standard practice of care. ACIP also considered an additional recommendation to remove the birth dose of hepatitis B vaccine, with the first dose not given earlier than one month of age, [for infants born of people who test negative for the virus]. But they decided to table that vote because there was some confusion around some of the phrasing around that recommendation. That's been a really big theme around these meetings this past week. So that's just come to show how much these meetings have changed since RFK, Jr., has overhauled the ACIP panel. And now it's turned more into a lot of confusion and a lot of discourse around questioning the evidence in ways that just don't quite always make sense. So check out our website for the latest updates. Feltman: Speaking of vaccines, last Tuesday, AHIP—the national trade association representing the health insurance industry—put out a statement on vaccine coverage. The organization said its member insurance plans would continue to cover all immunizations recommended by ACIP as of September 1, 2025, through the end of 2026. Now for some public health news on a very different topic: head injuries in sports. You've probably heard of chronic traumatic encephalopathy, or CTE. This progressive neurodegenerative disease, which can only be diagnosed with certainty via autopsy, is believed to be caused by repeat brain trauma, including the kind of repetitive head injuries one might endure while playing contact sports. It can include symptoms such as headaches, confusion, memory loss, poor judgment, depression, dementia, sensory-processing disorders and other potentially debilitating issues. CTE symptoms generally appear some years after repetitive brain trauma occurs. Researchers studied frozen brain tissue from 28 adult men. According to the study authors, this suggests that all athletes who frequently hit their heads could be at risk of experiencing some amount of brain damage. A separate study also published last Wednesday, this one in the journal Neurology, found an association between headers, or a player using their head to control or pass a soccer ball, and changes in the brain. The study asked 352 amateur adult soccer players to estimate their number of head impacts over the course of a year. The study isn't causational, and it relies on players' memories to estimate their actual rates of head banging. In some combination health and climate news a study released last week found that climate change likely tripled the number of heat-related deaths in European cities this summer. The new study estimated excess heat-related deaths by combining climate models with temperature and epidemiological data. The team estimated that 68 percent of those would not have occurred without climate change. I know we've covered some real bummers today, so let's end on a couple of fun stories. Based on an analysis of fruit collected from two chimp study sites in Africa, the researchers say, the average adult is consuming about 14 grams of pure ethanol a day. When you adjust for their body size that's like an adult human having about two drinks, at least based on the American standard serving. And that's just a conservative estimate based on the fact that the animals consume some 5 to 10 percent of their body weight in ripe fruit every day. But the average chimp doesn't show any sign of intoxication, probably because they consume their alcohol gradually over the course of the day—not to mention with plenty of fiber to slow them down. Another study published earlier this month in Nature found something a little more majestic than boozy apes: a type of ant whose queen can produce eggs from two different species. The Iberian harvester ant, or Messor ibericus, has a remarkable method for maintaining its workforce: in addition to laying eggs of her own species the queen can also produce offspring from a genetically distinct group: the species Messor structor. The authors suspect this started with a slightly less bonkers reproductive strategy known as sperm parasitism, where a queen mates with males from different ant species to produce hybridized, sterile worker ants. That gives them a self-sustaining source of the outsider DNA, allowing them to make as many hybrid workers as they want without actually interacting with the other species. In a sense you could say they've domesticated the other species's genome. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01365-z'>The Neurolipid Atlas: a lipidomics resource for neurodegenerative diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 09:51:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. To facilitate comparative lipidomic research across brain diseases, we establish a data common named the Neurolipid Atlas that we prepopulated with isogenic induced pluripotent stem cell (iPS cell)-derived lipidomics data for different brain diseases. Additionally, the resource contains lipidomics data of human and mouse brain tissue. Leveraging multiple datasets, we demonstrate that iPS cell-derived neurons, microglia and astrocytes exhibit distinct lipid profiles that recapitulate in vivo lipotypes. Notably, the Alzheimer disease (AD) risk gene ApoE4 drives cholesterol ester (CE) accumulation specifically in human astrocytes and we also observe CE accumulation in whole-brain lipidomics from persons with AD. Multiomics interrogation of iPS cell-derived astrocytes revealed that altered cholesterol metabolism has a major role in astrocyte immune pathways such as the immunoproteasome and major histocompatibility complex class I antigen presentation. Our data commons, available online (https://neurolipidatlas.com/), allows for data deposition by the community and provides a user-friendly tool and knowledge base for a better understanding of lipid dyshomeostasis in neurodegenerative diseases. As one of the most lipid-rich organs in our body1, the brain heavily relies on proper lipid homeostasis. Mutations in lipid metabolic genes cause rare but severe juvenile neurodegenerative diseases such as neuronal ceroid lipofuscinoses2 and Niemann Pick type C3. More recently, changes in lipid metabolism have been implicated in common neurodegenerative diseases such as Alzheimer disease (AD)4,5,6,7,8,9,10, Parkinson disease (PD)11,12, Huntington disease13,14,15, spinocerebellar ataxia16, amyotrophic lateral sclerosis (ALS)17,18 and frontotemporal dementias (FTDs) including primary tauopathies19,20,21,22. Conditions associated with neurodegenerative disease pathogenesis such as aging23, microglial reactivity to demyelination or fibrillar amyloid-β24,25,26, astrocyte reactivity27 and altered sleep cycles28 also appear to disrupt brain lipid metabolism. Together, these findings strongly indicate that alterations in brain lipid metabolism can contribute to neurodegenerative diseases. More importantly, these data suggest that lipid-targeting interventions could be a promising therapeutic strategy to prevent or even treat neurodegenerative conditions. The exact number of endogenous mammalian lipids is unknown. However, it is likely that thousands of individual lipid species together shape cell specific lipidomes (lipotypes) that dictate cellular function and dysfunction in the brain29. As a result of lipid complexity and a lack of standardized tools, sufficient detail on the exact lipid species and downstream pathways that contribute to the various neurodegenerative diseases is lacking. Mapping primary disease-associated changes in the human brain lipidome is particularly challenging, as confounders such as aging, diet, postmortem interval and secondary neurodegenerative processes (for example, cell death) strongly affect lipid metabolism. While animal models have been instrumental for our progress in understanding neurodegeneration, they have limited use for the study of lipids, as the human brain lipidome is intrinsically more complex33. Furthermore, while specific brain cell types can be sorted from brain tissue23,25,34, studies of lipid metabolism in the human or rodent brain are typically performed in bulk brain tissue, not capturing cell-type-specific changes, for example, in neurons, astrocytes and microglia. The use of induced pluripotent stem cell (iPS cell) technology presents a promising solution to overcome these challenges and enables the study of cell-type-specific regulation of lipid metabolism. Especially in combination with CRISPR–Cas9 gene editing, this technology provides a powerful tool to study how disease-specific mutations and risk variants affect downstream disease phenotypes (for example, amyloid overproduction, pTau levels and α-synuclein levels)35,36,37,38. Furthermore, iPS cell models are scalable, allowing high-throughput drug discovery of lipid-targeting agents7. To understand genotype–lipid interactions in human brain cells, we developed a standardized pipeline that combines isogenic iPS cell technology and lipidomics analysis capable of quantifying more than 1,000 different lipid species. We also generated a lipidomics data commons, the Neurolipid Atlas (available at https://neurolipidatlas.com/) that allows for user-friendly exploration of (neuro)lipidomics data. For this study, we populated the Neurolipid Atlas with newly generated data from a variety of different human iPS cell-derived disease models and states (AD, PD, ALS and FTDs), as well as novel postmortem human and mouse brain tissue, as benchmarking datasets. Using this pipeline and data analysis tool, we show that human iPS cell-derived neurons (iNeurons), astrocytes (iAstrocytes) and microglia (iMicroglia) have distinct lipid profiles resembling in vivo lipotypes. Through comparative lipidomic profiling of APOE3/3, APOE4/4 and reactive APOE3/3 iAstrocytes, we show that cholesterol esters (CEs) and triacylglycerides (TGs) accumulate in ApoE4 iAstrocytes (as in AD brain) but decrease in reactive astrocytes. Through proteomic and functional characterization, we demonstrate that cholesterol metabolism directly controls astrocyte reactivity, specifically interferon-dependent pathways such as the immunoproteasome and major histocompatibility complex (MHC) class I antigen presentation. High levels of free cholesterol enhance immune reactivity, whereas cholesterol esterification (increased in ApoE4 astrocytes) buffers immune reactivity. Overall, the Neurolipid Atlas provides a neurolipidomics data repository and research tool for the neuroscience field, forming a cornerstone for future research into cell and (neurodegenerative) disease-specific alterations of lipid metabolism. Here, we exemplify the potential of our tool by combing multiple lipidomics datasets to show that altered cholesterol metabolism in ApoE4 astrocytes affects their immune reactivity. To allow easy exploration, analysis and sharing of brain lipidomics data, we generated a novel resource that we named Neurolipid Atlas (Fig. This resource consists of two modules, for both of which data were newly generated: one module containing datasets generated from iPS cell-derived brain cells and one module containing data from human and mouse whole-brain tissue (Fig. To understand lipid changes in human brain cells, we developed a standardized iPS cell lipidomics pipeline capable of quantifying >1,000 lipid species across 16 different classes in a cell-type-specific manner (Fig. iPS cell-derived brain cells have been robustly shown to resemble in vivo brain cell types at the transcript level (albeit being more immature)39,40,41,42. Whether iNeurons, iAstrocytes and iMicroglia also resemble the in vivo lipidome is not known. Consequently, we differentiated iPS cells from a control iPS cell line (BIONi037-A)43 into glutamatergic iNeurons as detailed by Zhang et al.44, iAstrocytes as described by Fong et al.45 and iMicroglia as described by Haenseler et al.39. We confirmed cell fate and purity with cell-type-specific markers MAP2 (neurons), aquaporin 4 (AQP4; astrocytes) and Iba1 (microglia) (Fig. Subsequently, we analyzed the cellular lipidomes using comprehensive, quantitative shotgun lipidomic analysis46,47 and found that iNeurons, iAstrocytes and iMicroglia present with distinct lipid profiles (Fig. 1); individual lipid species are listed in Extended Data Fig. Additionally, we compared the lipidomics data from our iPS cell-derived brain cells to previously published lipotypes of primary mouse cells isolated from brain tissue48. Phosphatidylcholine (PC) and phosphatidylethanolamine (PE) were the most abundant lipid class in all cell types. Consistent with mouse brain cells, we observed the highest relative PC and PE levels in iNeurons48 (Fig. Moreover, the PC-derived and PE-derived lysophospholipids (LPC and LPE) were most abundant in iNeurons. Sphingomyelins (SMs) were highly abundant in iMicroglia, with lower levels in iAstrocytes and very low levels in iNeurons, similar to freshly isolated murine cells48 (Fig. Consistent with mouse data, phosphatidylserine (PS) was most abundant in iMicroglia and iAstrocytes but low in iNeurons48 (Fig. 1d,f,g), while iNeurons had the highest relative levels of ceramides (CERs) (Fig. Not in keeping with the mouse data were the relatively high DG levels in our iMicroglia, while phosphatidylglycerol (PG) lipids were relatively low (Fig. Of the lipid classes that were not measured in the previous mouse study, we found that TGs and free fatty acids (FAs) were most abundant in iMicroglia. iAstrocytes had the highest CE stores, in line with the role of astrocytes as a cholesterol supplier for other brain cell types49,50. Overall, these data indicate that iNeurons, iAstrocytes and iMicroglia recapitulate brain cells at not only the transcriptomic and proteomic level but also the lipidomic level. As for the remainder of the manuscript, all lipidomics data are available for exploration, analysis and download through the Neurolipid Atlas (https://neurolipidatlas.com/). a, Schematic overview of the Neurolipid Atlas workflow and resource. b, Schematic overview of iPS cell differentiation protocols. c, Representative confocal microscopy images of iNeurons, iAstrocytes and iMicroglia in monoculture of at least three independent differentiations per cell type. d, Heat map of z-scored lipid class abundance in iNeurons, iAstrocytes and iMicroglia (BIONi037-A parental line). e, PCA analysis of iPS cell-derived brain cell lipotypes. f, Pie charts showing relative abundance of all detected lipid classes in the iPS cell-derived brain cell types. g, Bar graphs present individual lipid class levels in each cell type, normalized to total lipid level. Data are shown as the mean ± s.d. Images in a,b were created using BioRender.com. To compare lipid changes in iPS cell models to those seen in vivo, the second module of the Neurolipid Atlas (Fig. 1a) contains multiple new lipidomic datasets from whole human and mouse brain tissue under different disease conditions. While lipid (specifically cholesterol) changes have been frequently implicated in AD51, only few lipidomic studies have been performed on human AD brain tissue4,52,53. We determined the control (N = 13) and AD (N = 20) lipidome across three different brain areas (Fig. We selected a brain area where AD pathology is abundant (frontal cortex) and an area where pathology is generally lower (cerebellum)54. We found significant regional differences in brain lipid composition (Fig. White matter from the frontal cortex had high levels of CERs and SM, consistent with the enrichment of these lipids in myelin (Fig. On the contrary, phospholipid and storage lipid (for example, CE and TG) levels were relatively higher in gray matter from the cerebellum and frontal cortex (Fig. 2 and 3) AD versus control brains, principal component analysis (PCA) largely separated control and AD samples in gray and white matter from the frontal cortex but less so in the cerebellum (Fig. This separation was driven mostly by TG and CE species (Extended Data Fig. At the class level, we found that CEs were significantly upregulated in AD in gray and white matter from the frontal cortex and trended toward increased levels in the cerebellum (Fig. Analysis at the level of individual lipid species also showed an increase for most CE species (Supplementary Fig. 2a–c) but no single CE specie reached significance, likely reflecting high variation in FA tails of CEs in individuals. In addition, other neutral lipids such as TG (trend in all areas) and DG (significant only for white matter from the frontal cortex) were increased in persons with AD (Fig. Lactosyl-CERs (LacCERs) were also significantly increased in white matter from the AD frontal cortex (Fig. Because astrogliosis is known to be increased in late stages of AD and reactive astrocytes adopt a distinct lipid profile with increased phospholipid saturation27, we tested whether saturated phospholipid species were enriched in AD brain tissue. No consistent changes in phospholipid or TG saturation could be observed in the AD brain (Supplementary Fig. Our results in human AD brain, combined with previous findings in persons with AD4,52,53, strongly suggest that accumulation of neutral lipids (specifically CE) is a key lipidomic hallmark of AD. a, Schematic overview of human postmortem brain tissue samples and summary of participant characteristics. Metadata for individual participants can be found in the Methods. b, PCA analysis of unbiased lipidomic analysis from indicated brain areas (control group only). c, Heat map shows z-scored relative lipid class abundance (control group) per brain region. d, PCA plot of unbiased lipidomic analysis of AD (purple) and control (green) brain tissue samples from frontal cortex gray matter. e, Heat map depicting changes (log2 fold change for AD group versus average control group) at the lipid class level for each individual with AD and each brain area. Samples from participants with AD are ordered 1–20 from left to right in each brain area (metadata in Methods). f, Average log2 fold change of lipid classes in all AD brain samples compared to control samples per brain area. g, Changes in levels of CE, DG and TG (neutral) lipid species in control versus AD group. Data are shown as the median and interquartile range (IQR). All lipid values in this figure are plotted as a percentage of total lipids (raw concentration in Extended Data Fig. With CE accumulation being a key lipid feature of AD, it is important to understand how it might contribute to AD pathogenesis. We previously showed that CE accumulation drives pTau buildup in human neurons7 and others showed that CE buildup alters microglial function after microglia phagocytose cholesterol-rich myelin24,25. While CE levels are highest in human iAstrocytes (Fig. 1), the possible effect of CE accumulation in these cells is unknown. Importantly, astrocytes express high levels of the AD risk gene and cholesterol carrier APOE. A common variant in ApoE, ApoE4, is the major genetic risk factor for AD, which increases the risk for AD from 3–4-fold (heterozygosity) to 14-fold (homozygosity) depending on ethnicity56,57, with even higher reported odds ratios in neuropathologically confirmed cases58. To map how ApoE4 affects the astrocytic lipidome, we differentiated isogenic pairs of APOE3/3 and APOE4/4 iPS cells to iAstrocytes. We selected isogenic pairs from the iPS cell Neurodegenerative Disease Initiative (iNDI; parental line, Kolf2.1J, APOE3/3; edited lines, APOE4/4 Kolf2.1J C112R hom3 (set 1) and C112R hom2 (set 2))59,60 and a completely independent isogenic set from the European Bank for iPS cells (parental line, BIONi037-A, APOE3/3; edited line, BIONi037-A4, APOE4/4)43, all from Caucasian origin (Fig. Neither of these isogenic pairs has been characterized by lipidomic and/or proteomic profiling. The ApoE genotype of the iPS cells was confirmed by Sanger sequencing on receipt and after differentiation to astrocytes, as well as after each experiment to verify sample identity (Extended Data Fig. Successful differentiation to astrocytes was validated by astrocyte marker staining and evaluation of astrocyte marker expression by RNA sequencing (RNA-seq)61 (Extended Data Fig. Less secreted ApoE was present in ApoE4 iAstrocytes (Extended Data Fig. 3d), as has been reported in other iAstrocytes and cerebrospinal fluid (CSF)5,8,10,62,63. We performed lipidomics on three isogenic pairs using our standardized iPS cell lipidomic pipeline (individual replicates in Fig. Strikingly, consistently across experiments and lines, we observed an ApoE4-dependent increase in CE (Fig. This effect was mediated by an increase in all detected CE species (Extended Data Fig. Multiple TG species were also significantly increased, which resulted in a significant increase at the class level (Fig. TGs containing saturated or monounsaturated FAs and highly polyunsaturated FAs (>5 double bonds) were most upregulated in our ApoE4 iAstrocytes (Fig. Consistent with higher levels of the neutral storage lipids, lipid droplets were increased in ApoE4 iAstrocytes (Fig. SM levels were significantly downregulated in ApoE4 iAstrocytes at both the species and the class level (Fig. The BIONi037 ApoE4 iAstrocytes also showed a strong and consistent increase in LacCER, HexCER and CER species, which was not consistently observed in the Kolf2.1J ApoE4 iAstrocytes (Fig. We did not find evidence for increased saturation of phospholipids in our ApoE4 iAstrocytes (Extended Data Fig. Overall, our data indicate that ApoE4 strongly drives the accumulation of CEs and, to a lesser extent, TGs in human iAstrocytes while decreasing SM levels. Lipidomics data of all isogenic pairs and biological replicates from our ApoE4 and ApoE3 iAstrocytes are available on the Neurolipid Atlas. a, Schematic overview of isogenic iPS cell lines and experimental design. b–e, Volcano plot presents a typical example of log2 fold change of altered lipid species in Kolf2.1J set 1 (b) and BIONi037 (d) ApoE4 versus ApoE3 iAstrocytes. Also shown are heat maps of most differentiating lipid species between ApoE4 and ApoE3 iAstrocytes in Kolf2.1J set 1 (c) and BIONi037 (e). f, Summary data of changes in all detected lipid classes in ApoE4 versus ApoE3 iAstrocytes (N = 9; three independent experiments from three isogenic sets). *P < 0.05 (two-sided paired t-test or Mann–Whitney U-test with BH correction). g, Fold change in TGs with indicated number of double bonds (unsaturation) in ApoE4 versus ApoE3 iAstrocytes. All experiments presented in f are included here. Data are shown as the mean. h, Representative images and quantification of the average lipid droplet number per astrocyte based on Plin2 staining (N = 6; three independent experiments from two isogenic sets). Data are shown as the mean. Open circles and triangles indicate the mean per experiment, while solid dots represent all independent wells. Our data show that ApoE4 increases CE levels in human iAstrocytes. To understand the functional consequence, we performed proteomic and transcriptomic analysis on our ApoE4 versus ApoE3 iAstrocytes (BIONi037 and Kolf2.1J set 1) (Figs. Notably, the proteomics and lipidomics were performed on the same batch of iAstrocytes (Methods) to allow for multiomic integration. ApoE was among these DEPs, showing downregulation in the ApoE4 iAstrocytes (Fig. We focused our analysis on proteins that were either downregulated or upregulated in both ApoE4 lines (Fig. Through overrepresentation analysis (ORA), we found that cell adhesion (for example, NCAM1 interactions and integrin cell surface interactions) and extracellular matrix (ECM)-related pathways (for example, collagen chain trimerization and ECM proteoglycans) were upregulated by ApoE4 in both isogenic lines (Fig. On the contrary, immune pathways (for example, immunoregulatory interactions between a lymphoid and nonlymphoid cell, MHC I antigen presentation and interferon signaling) were downregulated (Fig. This includes the term ‘endosomal/vacuolar pathway', which contained mainly MHC terms (Fig. Strikingly, proteins in the MHC class I antigen presentation pathway were consistently downregulated, as were immunoproteasome subunits, two pathways directly downstream of interferon signaling (Fig. As a confirmatory readout, we stained against human leukocyte antigens (HLAs) class I and transporters associated with antigen processing 1 (TAP1) and 2 (TAP2) and confirmed downregulation by western blot (Fig. Decreases in HLA were also confirmed by flow cytometry (Fig. To be able to compare our results to previous transcriptomic studies with different ApoE4 iAstrocytes8,10, we also performed transcriptomic analysis (Extended Data Fig. Through unbiased gene set enrichment analysis, the MHC class I antigen presentation pathway was also found to be downregulated by ApoE4 at the transcriptome level in the BIONi037 isogenic set (Extended Data Fig. 6c), which additionally showed downregulation of interleukin and interferon immune signaling pathways, the complement cascade and endoplasmic reticulum (ER) phagosome transport, whereas translation-related terms were upregulated, as was the lipid droplet marker perilipin 2 (Plin2) (Extended Data Fig. When comparing to transcriptomics from other ApoE4 lines, downregulation of the interferon pathway was consistent in all ApoE4 isogenic and case–control sets from Lin et al.8 and Tcw et al.10 and trended similarly in our Kolf2.1J set (Fig. Changes in other pathways such as the complement cascade and translation initiation were observed but the direction of change was highly variable across lines (Fig. Overall, these data show that the ApoE4 genotype decreases ApoE levels, drives CE accumulation and inhibits interferon signaling-dependent pathways such as MHC class I antigen presentation and the immunoproteasome in human iAstrocytes. a,b, Volcano plots present log2 fold changes in protein levels in ApoE4 versus ApoE3 iAstrocytes from Kolf2.1J set 1 (a) and BIONi037 (b). The top ten proteins with the highest log2 fold changes and top ten proteins with the most significant P values are labeled (N = 4 wells per genotype). Statistical analysis was performed using a two-sided pairwise t-test. c, Number of DEPs (fold change > 1.5, FDR < 0.05) detected in ApoE4 versus ApoE3 iAstrocytes of Kolf2.1J (set 1) and BIONi037 isogenic sets. d, Relative ApoE protein levels in ApoE3 and ApoE4 iAstrocytes (from proteomic analysis) from BIONi037 and Kolf2.1J background. Data are shown as the mean and s.d. e,f, Venn diagrams depicting the number of DEPs significantly upregulated (e) or downregulated (f) (fold change > 1.25, log2 fold change > 0.3) in Kolf2.1J, BIONi037 or both ApoE4 iAstrocytes. A Reactome ORA was performed on the 105 common upregulated (e) or 109 common downregulated (f) proteins and the enrichment ratio was plotted for all significant pathways (FDR < 0.05). g, Schematic overview of interferon-dependent regulation of MHC class I antigen presentation (in blue) and immunoproteasome (in green) pathways. The heat map indicates the log2 fold change of indicated proteins in ApoE4 versus ApoE3 iAstrocytes (proteomics). h,i, Representative western blot (h) and quantification (i) of MHC I levels (stained for HLA class I heavy chain) in ApoE4 versus ApoE3 iAstrocytes (N = 10; five independent experiments from two isogenic sets). Data are shown as the mean. j,k, Representative histogram (BIONi037) (j) and quantification (k) of plasma membrane MHC I levels (stained for HLA-A, HLA-B and HLA-C) by flow cytometry (N = 9; five (Kolf2.1J) or four (Bi037) independent experiments from two isogenic sets). Data are shown as the mean. l,m, Example images (l) and quantification (m) of MHC I levels as measured by immunofluorescence microscopy (stained for HLA class I heavy chain) (N = 14; seven independent experiments from two isogenic sets). Data are shown as the mean. n, Comparison of significant Reactome pathways (by gene set enrichment analysis) from our transcriptomic analysis of ApoE4 versus ApoE3 astrocytes (BIONi037) with previously published datasets. Shown is the average log2 fold change of all measured genes in the indicated pathway in each isogenic set or case–control set. Tcw et al. ind1–ind4 (four different isogenic sets) and population (control versus ApoE4) represent iAstrocytes from a previous study10. Lin et al. represents one isogenic set of ApoE4 versus ApoE3 iAstrocytes from a previous study8. o, Heat map shows the log2 fold change in individual genes in the MHC I and immunoproteasome pathway across indicated studies, including our data here. Open triangles indicate the mean per experiment, while solid dots represent all independent wells. The reduction in interferon-dependent pathways is striking, as ApoE4 is thought to enhance, not decrease, immune signaling, as astrogliosis is a key hallmark of end-stage AD5,10,64,65,66,67. Yet, our data clearly demonstrate a consistent reduction in the expression of proteins in these pathways, including five class I leukocyte antigens (HLA-A, HLA-B, HLA-C, HLA-E and HLA-F) and all specific subunits of the immunoproteasome (PSMB8, PSMB9 and PSMB10) (Fig. To better understand our lipidomic and proteomic findings in the context of astrocyte immune function, we also performed multiomic analysis of reactive (treated with tumor necrosis factor (TNF), interleukin-1α (IL-1α) and complement component 1q (C1q)) iAstrocytes (Kolf2.1J and BIONi037-A) (Fig. By comparing the lipidomic dataset of ApoE4 versus ApoE3 and ApoE3 versus ApoE3 reactive iAstrocytes, we found that neutral lipids were oppositely regulated (Figs. Whereas CEs and TGs were increased in ApoE4 iAstrocytes, they were strongly downregulated in reactive astrocytes at the species (Fig. Cholesterol secretion was increased in reactive iAstrocytes (Extended Data Fig. Interestingly, LacCER was increased in both reactive iAstrocytes and ApoE4 iAstrocytes, as well as in AD brain (Figs. Saturated phospholipids were increased in reactive iAstrocytes (Fig. 5d) in contrast to our ApoE4 iAstrocytes (Extended Data Fig. To directly compare how the effect of astrocyte reactivity on the lipidome is conserved across species, we also tested how treatment with TNF, IL-1α and C1q affected the lipidome of mouse astrocytes (under different culture conditions) (Extended Data Fig. In keeping with the human iAstrocytes, reactivity strongly decreased CE and TG levels in mouse astrocytes across culturing paradigms while increasing saturated phospholipid levels (Extended Data Fig. However, the increase in LacCER and HexCER was not observed in mouse astrocytes and, thus, seemed specific to the human astrocytes (Fig. To confirm the effect of reactivity on CE and TG levels (decrease) and LacCER and HexCER levels (increase) in human iAstrocytes, lipidomics was performed on another completely independent parental line (WTC11, M.K. 7e–g), where the same effect was observed, indicating a robust human astrocyte lipidomic response. While ApoE4 iAstrocytes at baseline have higher CE and TG levels (Fig. 3), stimulation of ApoE4 iAstrocytes with TNF, IL-1α and C1q still reduced CE and TG levels and increased MHC I levels indicating that the ApoE4 effect can be overcome by a strong proinflammatory stimulus (Extended Data Fig. In these lines, an increase in LacCER and HexCER was also observed (Extended Data Fig. To test whether the decrease in CE and TG lipids in reactive astrocytes was coupled to increased HLA-reactivity, we performed proteomics on the reactive iAstrocytes (Fig. We focused our analysis on proteins that were upregulated or downregulated by reactivity in both lines (Fig. ORA did not identify any significant downregulated pathways (Extended Data Fig. However, the top upregulated pathways (Fig. 5h) were the endosome and vacuolar pathway, immunoregulatory interactions between a lymphoid and nonlymphoid cell, MHC I antigen presentation and interferon signaling, pathways that were all downregulated in ApoE4 iAstrocytes (Fig. Beyond these terms, analysis showed that virtually all immune upregulated proteins in reactive iAstrocytes were downregulated in the ApoE4 versus ApoE3 iAstrocytes at baseline (Fig. Using flow cytometry, we confirmed that MHC class I levels (HLA-A, HLA-B and HLA-C) were indeed increased in reactive iAstrocytes (Fig. Overall, our results indicate that ApoE4 and reactive iAstrocytes have opposing lipidomic and proteomic phenotypes. CEs and TGs are upregulated in ApoE4 astrocytes but downregulated in reactive iAstrocytes, whereas interferon signaling-dependent pathways, the immunoproteasome and MHC class I are downregulated in ApoE4 iAstrocytes but upregulated in reactive iAstrocytes (Fig. a, Schematic overview of experimental design. A cocktail of TNF, IL-1α and C1q was added for 24 h to make astrocytes reactive. b, The volcano plot presents the log2 fold change of altered individual lipid species in reactive versus control iAstrocytes (Kolf2.1J set 1, ApoE3). c, Summary data of changes in all detected lipid classes in reactive versus control iAstrocytes (N = 6; three independent experiments from two ApoE3 lines). d, Fold change of all phospholipid species with indicated number of double bonds (unsaturation) in reactive versus control iAstrocytes. All experiments presented in c are included here. Data are shown as the mean. e, Number of DEPs (fold change > 1.5, FDR < 0.05) in reactive versus control iAstrocytes for indicated lines. f,g, The log2 fold changes in protein levels of reactive versus control iAstrocytes for Kolf2.1J set 1 (f) and BIONi037 (g). The top ten proteins with the highest log2 fold change and top ten proteins with the highest P values are labeled (N = 4 wells per genotype). Statistical analysis was performed using a two-sided pairwise t-test. h,i, Venn diagram depicting the number of proteins that were significantly upregulated (h) or downregulated (i) (fold change > 1.25, log2 fold change > 0.3) in reactive Kolf2.1J, BIONi037 and both iAstrocytes. A Reactome ORA was performed on the 275 common upregulated or 129 common downregulated proteins. No significantly enriched downregulated pathways were observed; the enrichment ratios for all significantly (FDR < 0.05) upregulated pathways are plotted in k. j, Top, heat map depicting the log2 fold change of indicated lipid classes in ApoE4 or reactive iAstrocytes versus ApoE3 control iAstrocytes. Lipid classes that were changed in ApoE4 or reactive iAstrocytes with P < 0.05 are shown. Bottom, heat map depicting the log2 fold change of indicated proteins from the MHC class I and immunoproteasome pathway in ApoE4 or reactive iAstrocytes versus ApoE3 control iAstrocytes (based on proteomics data). k, Relative membrane MHC I levels (stained for anti-HLA-A, anti-HLA-B and anti-HLA-C) according to flow cytometry in reactive versus control iAstrocytes (N = 9; four (Kolf2.1J) or five (Bi037) independent experiments from two isogenic sets). Data are shown as the mean. l, Schematic representation of opposing lipidomic and proteomic phenotypes in ApoE4 and reactive iAstrocytes. Open circles or triangles indicate the mean per experiment, while solid dots represent all independent wells. The images in a,l were created using BioRender.com. Using proteomic analysis, we also found that a specific cluster of cholesterol synthesis genes were upregulated in reactive iAstrocytes (Extended Data Fig. To test whether cholesterol regulates immune reactivity in human iAstrocytes, we treated Kolf2.1J and BIONi037 control (ApoE3) iAstrocytes with cholesterol (Fig. Both MHC class I presentation and IL-6 secretion were significantly increased after cholesterol treatment (Fig. CEs are generated through conjugation of free cholesterol to an FA by acyl coenzyme A cholesterol acyltransferases (ACATs). The combined addition of cholesterol with the ACAT inhibitor avasimibe further increased IL-6 secretion, indicating that CE formation buffered the immune reactivity mediated by cholesterol treatment (Extended Data Fig. We also found that RNA levels of interferon regulatory factor 1 (IRF1), a master regulator of MHC class I pathway gene expression, and HLA-B were upregulated by exogenous cholesterol (Extended Data Fig. 10c), indicating that the effect of cholesterol on these pathways might be transcriptionally mediated. Moreover, the addition of cholesterol to astrocytes was sufficient to increase saturated phospholipid levels indicative of astrocyte reactivity (Fig. Exogenous cholesterol also potentiated immune reactivity (as measured by MHC class I levels and IL-6 secretion) of iAstrocytes treated with TNF, IL-1α and C1q (Fig. Conversely, pretreatment with atorvastatin (to reduce cholesterol levels) inhibited MHC class I upregulation and IL-6 secretion upon iAstrocyte reactivity (Fig. Lastly, we found that the addition of free cholesterol rescued MHC I expression and IL-6 secretion in the ApoE4 iAstrocytes (Fig. Overall, our data indicate that cholesterol is a major regulator of MHC class I antigen presentation and immune reactivity in human astrocytes and dysregulated cholesterol metabolism by ApoE4 impairs astrocyte immune reactivity (Fig. a, Schematic representation of the experimental design. b, Representative image of lipid droplet staining by Lipidspot in iAstrocytes (BIONi037 ApoE3) following 24-h treatment with 50 μM cholesterol. c, Normalized membrane MHC I levels (stained for anti-HLA-A, anti-HLA-B and anti-HLA-C) in control versus 50 μM cholesterol-treated ApoE3 iAstrocytes, as determined by flow cytometry (N = 7; three (Kolf2.1J) or four (Bi037) independent experiments from two isogenic sets). Data are shown as the mean. d, Normalized IL-6 secretion in control versus cholesterol-treated ApoE3 iAstrocytes (N = 6; two (Kolf2.1J) or four (Bi037) independent experiments from two isogenic sets). Data are shown as the mean. e, Fold change of phospholipid species with indicated number of double bonds (unsaturation) in cholesterol-treated versus control iAstrocytes (BIONi037 ApoE3) (N = 3 independent experiments from Bi037A iAstrocytes). Data are shown as the mean. f,g, Representative histogram (f) and quantification (g) of normalized MHC I membrane levels determined by flow cytometry (stained for anti-HLA-A, anti-HLA-B and anti-HLA-C) in response to indicated treatment conditions in iAstrocytes (N = 4–6; three (control and cholesterol) or two (atorvastatin) independent experiments from two isogenic sets. Data are shown as the mean. *P < 0.05 (two-sided one-sample t-test for cholesterol and atorvastatin versus 1). h, Secreted Il-6 levels in medium of ApoE3 iAstrocytes that were pretreated with or without exogenous cholesterol (10 μM) or atorvastatin (0.5 μM) for 1 h before 24-h cotreatment with increasing doses of TNF, IL-1α and C1q (N = 5; two (Kolf2.1J) or three (Bi037) independent experiments from two isogenic sets). Data are shown as the mean and s.e.m. **P < 0.01 (intercept difference by linear regression model). Relative IL-6 levels with vehicle 0.25× cocktail dose set at 1. i, Relative changes in membrane MHC I levels determined by flow cytometry (stained for anti-HLA-A, anti-HLA-B and anti-HLA-C) in ApoE3, ApoE4 or ApoE4 iAstrocytes treated with 50 μM cholesterol (N = 5; two (Kolf2.1J) or three (Bi037) independent experiments from two isogenic sets. Data are shown as the mean. *P < 0.05 (two-sided one-sample t-test for E4 and E4 + cholesterol versus 1). *P < 0.05 (paired t-test for E4 versus E4 + cholesterol). BH correction was applied to the three P values. j, Relative changes in IL-6 secretion in ApoE3, ApoE4 or ApoE4 iAstrocytes treated with 50 μM cholesterol (N = 6; one (Kolf2.1J) or five (Bi037) independent experiments from two isogenic sets). Data are shown as the mean. *P < 0.05 (two-sided one-sample t-test for E4 and E4 + cholesterol versus 1). *P < 0.05 (paired t-test for E4 versus E4 + cholesterol). BH correction was applied to the three P values. k, Schematic representation of ApoE4 decreasing MHC class I expression and immune function in human glia by increased cholesterol storage in CEs. Open circles or triangles indicate mean per experiment, while solid dots represent all independent wells. Images in a,k were created using BioRender.com. While repositories for RNA and proteomics datasets are common in the field of neurodegenerative diseases, similar resources are currently lacking for (neuro)lipidomics data. To standardize and allow exploration of data without prior bioinformatics knowledge, we generated an online lipidomics browser (Fig. This browser includes all data presented above, as well as (un)published lipidomics data generated together with a large group of collaborators, currently totaling more than 70 datasets over four neurodegenerative diseases and multiple treatment conditions (Supplementary Table 1). All current datasets in the Neurolipid Atlas (and in the figures of the manuscript) were newly generated for this manuscript, except for the ALS datasets that were recently published18. New datasets from our lab and our collaborators will be uploaded to the database in a continuous manner. Labs can contribute their neurolipidomics data to our resource (‘upload dataset') by providing annotated lipidomics data in line with LipidMaps shorthand notation68, a metadata sheet (methods provided on the website) and a short description of the dataset. All data will be quality-controlled (QCed) and curated by the Neurolipid Atlas team, including checking compliance with Lipidomics Standards Initiative69, before it is published online. All datasets require a digital object identifier (DOI) from the associated preprint or publication or (for unpublished data) a DOI generated through the lipidomics minimal reporting checklist69. The Neurolipid Atlas currently includes Lipidyzer data analyzed by Shotgun Lipidomics Assistant (SLA) software47 but other pipelines will also be accommodated. All neuro-related data are welcomed, including iPS cell models, as well as human and animal model brains and cerebrospinal fluid. Contact information, including for information on data uploads, can be found online (https://neurolipidatlas.com/). The Neurolipid Atlas allows for the download of all raw data and metadata, as well as in-browser analysis, which includes QC, blank filtering, normalization and the generation and customization of figures. A link to all datasets is included on the homepage. Datasets can be searched for by name, cell type, genotype, treatment type, parental line or contributing lab. Using the ‘explore dataset with SODA data browser' link, users can explore independent replicate experiments and visualize changes at the lipid class level (bar graphs) or at the species level (volcano plots, heat maps, PCA and FA analysis) and interact with the data by hovering over different lipid species. All figures and their source data can be downloaded. A comprehensive ‘quick start' guide is available on the Neurolipid Atlas homepage (https://neurolipidatlas.com/) to help users navigate the data, while a detailed manual outlining all analysis and visualization options is included with each dataset in the help section. Overview of the Neurolipid Atlas data commons (https://neurolipidatlas.com) to explore all lipidomics datasets from this study. A representative image of the homepage is shown, where one can proceed to a list of experiments by selecting one of the cell type, human or mouse icons or enter a search term as indicated in the top panel. Alternatively, a list of all datasets can be found by selecting; ‘go to all datasets. To explore data, experiments can be selected in the dataset browser window as indicated. Independent replicates of the experiment can be selected in the left column under ‘data'. Visualization options can be (de)selected in the top horizontal bar. Examples of bar graphs, volcano plots and heat maps for visualization of changes in lipid class (bar graphs) or lipid species (volcano plots and heat maps) levels between selected conditions are shown. The ‘QC' and ‘help' modules present in the left column offer extensive background information on the QC, data-processing steps, lipid class measurements and visualization options. Lastly, a summary list of currently available datasets is shown. Lipid metabolism is affected and implicated in various neurodegenerative diseases2,3,4,6,7,8,9,10,11,12,14,15,16,17,18,19,20,22,24,25. Here, we generated a new resource, the Neurolipid Atlas, to facilitate insights into lipid changes associated with different neurodegenerative diseases in a disease-specific and cell-type-specific manner. Leveraging multiple datasets, we demonstrate that iNeurons, iMicroglia and iAstrocytes exhibit distinct lipid profiles that recapitulate in vivo lipotypes. Notably, the AD risk gene ApoE4 drives CE accumulation in human iAstrocytes and CE accumulation is also observed in the human AD brain. Subsequently, through an integrated multiomics approach, we show that CE accumulation in ApoE4 iAstrocytes represses their immune function, specifically interferon pathways, MHC class I antigen presentation and immunoproteasome pathways. Our findings further consolidate the notion that CE accumulation is a major pathogenic hallmark of AD4,52,53. CEs in CSF were shown to correlate with progression from mild cognitive impairment to AD70. Accumulation of CEs has also been detected in microglia upon TREM2 or ApoE loss25 and inhibition of CE formation improved Aβ clearance74. In ApoE4 oligodendrocytes, CE accumulation led to perturbed neuronal myelination75. We show that ApoE4 iAstrocytes exhibit increased CE accumulation and this altered cholesterol metabolism impairs their immune function, including suppression of the MHC class I pathway. Our data show that CEs accumulate in both white and gray matter of the frontal cortex, which indicates that cells other than solely myelinating oligodendrocytes75 contribute to this phenotype. Secondary neurodegenerative processes (protein aggregation, neuronal death and demyelination) and CE accumulation in microglia might be a possible explanation for the observed CE accumulation in persons with AD24,25,26. Yet, our finding that the AD genetic risk factor ApoE4 strongly drives CE accumulation in astrocytes in the absence of pathology indicates that CE accumulation might not merely be a downstream effect of neurodegeneration but rather directly downstream of AD risk genes. How CEs accumulate in ApoE4 astrocytes needs further investigation but could involve altered import and export of cholesterol or changes in lysosomal lipid clearance76. We confirm previous reports that ApoE4 increases levels of polyunsaturated TGs9,77 but also find an increase in monounsaturated and saturated TG levels (Fig. Importantly, ApoE4 iAstrocytes do not show the increase in saturated phospholipids typical for reactive astrocytes27 (Extended Data Fig. Overall, our data show that ApoE4 and AD present a unique lipotype that is primarily characterized by strong CE accumulation and changes in TGs and SM. To better understand how CE contributes to AD pathogenesis in vivo (through astrocyte immune repression and/or in other cell types), it will be important to define which cell types accumulate CE and how this accumulation potentially alters their function. As the strongest genetic risk factor for AD (ApoE) is highly expressed in astrocytes, there is an urgent need to understand how the AD risk variant ApoE4 affects astrocytes. Here, we provide a characterization of isogenic APOE3/3 and APOE4/4 iAstrocytes from the iNDI line Kolf2.1J, which we hope will serve as a reference for the field. We also provide a full characterization of a second isogenic pair of iAstrocytes (BIONi037, EBISC). As astrogliosis is a major feature of end-stage AD78,79,80,81, we were very surprised to find major immune pathways such as interferon signaling, the immunoproteasome and MHC class I antigen presentation to be downregulated in ApoE4 iAstrocytes. To place this in context, we also generated the first lipidomic and proteomic analysis of reactive human iAstrocytes treated with TNF, IL-1α and C1q. We confirmed that reactive human iAstrocytes, as in mice27, also increase phospholipid saturation (Extended Data Fig. Strikingly, the top upregulated proteomic pathways in our reactive iAstrocytes were interferon pathways including MHC class I. Furthermore, CE and TG were downregulated in reactive astrocytes but upregulated in ApoE4 astrocytes (Fig. The results provide strong evidence that ApoE4 intrinsically inhibits rather than activates astrocyte immune function. These results fit with recent reports in AD mice showing decreased immune function of ApoE4 microglia, including reduced antigen presentation82,83. Interestingly, human stem-cell-derived microglia xenotransplanted into APPNL-G-F mice were shown to transition into a human-specific HLA-expressing state and ApoE4 selectively reduced the proportion of cells acquiring this HLA phenotype84. In line with this, an AD-protective variant in PLCg2 (PLCg2-P522R) was recently shown to reduce CE accumulation in iMicroglia24 while increasing microglial MHC I levels and providing benefit through increased recruitment of T cells85. On the basis of these data, the presence of a similar ApoE4–cholesterol–immune axis in microglia, as we identified here for astrocytes, is likely but needs to be confirmed. Overall, these findings (including ours in human brain cells) indicate that ApoE4 intrinsically limits immune reactivity rather than inducing immune activation. This could indicate that the immune reactivity (of astrocytes and microglia) is needed to stave off AD and restoration (or activation) of glial activity in ApoE4 carriers might prevent AD pathogenesis. With current technologies, the hypothesis that ApoE4 suppresses glial immune reactivity before AD onset is difficult to validate in humans as postmortem material normally reflects late disease states. It would, therefore, be highly relevant to evaluate, for example, immunoproteasome levels, MHC class I expression and lipid levels in healthy ApoE4 carriers early in life, such as trough tissue obtained from normal-pressure hydrocephalus biopsies86. The exact pathway connecting cholesterol levels to interferon pathways, the immunoproteasome and MHC class I antigen presentation also needs more study. Processes such as demyelination or phagocytosis of neuronal debris could contribute to increasing levels of cholesterol in astrocytes in the AD brain. Our data indicate that cholesterol can increase IRF1 levels, which is a master regulator of MHC class I expression. Cholesterol might intersect with interferon signaling at the plasma membrane in lipid rafts87,88 or through interactions with the cGAS–STING pathway89,90,91. However, our study here clearly shows that ApoE4 affects glial immune function and that this effect is mediated by ApoE4-induced changes in glial lipid metabolism and storage. In vitro growth and medium composition are major regulators of lipid metabolism; hence, it was striking to see that iPS cell-derived brain cells largely recapitulate in vivo lipotypes. Specifically, for our iAstrocytes, the astrocytes were differentiated in medium consisting of serum that contains protein and lipid factors that are not present in the brain parenchyma under healthy conditions but penetrate the brain when the blood–brain barrier is impaired (such as in aging or AD92,93,94). To limit possible confounding effects of serum in the medium, we removed serum from our iAstrocytes during terminal experiments. In addition, we were able to recapitulate major reactive phenotypes in mature astrocytes isolated from mouse brain that were isolated and cultured without any serum (Extended Data Fig. Our study also highlights significant variability across ApoE4 effects depending on the parental line in which the mutation is introduced (lipidomics effects in Fig. 3; proteomic and transcriptomic differences in Fig. The effect of ApoE4 genotype is likely affected by genetic context, such as haplotype, gender or ethnicity. The variation between parental lines might be biologically meaningful. However, more studies with a higher number of isogenic sets will be required to address these questions. For the interpretation of our data, here, we focused on lipid classes (CE and TG) and Reactome pathways that were changed consistently across all isogenic sets and, where possible (for example, transcriptomics data), we compared our findings to previously published isogenic lines. We found that interferon signaling, MHC class I antigen presentation and ER–phagosome pathways were downregulated in ApoE4 iAstrocytes from all these studies (Fig. 4), indicating a robust and likely genetic context-independent effect of ApoE4 on the suppression of these immune pathways. Overall, our data highlight the important role of lipid (particularly cholesterol) metabolism in AD. We created a new tool (the Neurolipid Atlas) as a resource of neurolipidomic datasets for different cell types, mutations, neurodegenerative diseases and model organisms. As a proof of concept, we show that iNeurons, iAstrocytes and iMicroglia have distinct lipidomes that recapitulate in vivo lipotypes. Our data solidify the link between AD and cholesterol, further establishing CE accumulation as a hallmark of AD. Lastly, we show that cholesterol regulates astrocytic immune function, which is impaired by the genetic AD risk variant ApoE4. We will continue to grow the Neurolipid Atlas database and invite other researchers to upload their data as well, working toward a detailed understanding of the role that lipids have in brain health and disease. All research complied with the relevant ethical regulations and institutional guidelines at the Vrije Universiteit. Isogenic Kolf2.1J (APOE3/3), Kolf2.1J C112R Hom3 (APOE4/4 set 1) and Kolf2.1J C112R Hom2 (APOE4/4 set 2) human iPS cells were a kind gift from iNDI (donor: 57-year-old man). Kolf2.1J C112R Hom2 (APOE4/4 set 2) corresponds to JIPSC001142 available through iNDI. Isogenic BIONi037-A (APOE3/3) and BIONi037-A4 (APOE4/4) human iPS cell lines were obtained through EBISC (donor: 77-year-old woman). iPS cells were cultured in six-well plates precoated with 120–180 µg ml−1 Geltrex (Fisher Scientific, A1413302) in Gibco Essential 8 medium (E8; Fisher Scientific, 15190617) + 0.1% penicillin and streptomycin (P/S; Fisher Scientific, 11548876), with daily full medium refreshments. iPS cell colonies grown until 90% confluency were dissociated using 1 mM EDTA (Invitrogen, 15575-038) in 1× PBS (VWR, 392-0434) and replated in E8 medium supplemented with 5 μM ROCK inhibitor (RI; Tebu-Bio, Y27632). The genomic integrity of iPS cell lines was periodically tested using single-nucleotide polymorphism (SNP) arrays, as described below. In addition, cell cultures were regularly tested for Mycoplasma contamination. DNA from cell cultures was isolated using a ReliaPrep genomic DNA tissue miniprep system (Promega, A2052). Samples were processed by the Global Screening Array (GSA) Consortium Project at Erasmus MC Rotterdam on the Illumina GSA beadchip GSA MD version 1. The iPsychCNV package was used for copy-number variant (CNV) calling, which integrates B allele frequency distribution and log R ratio to reduce false-positive detection (https://doi.org/10.5281/zenodo.46235)95. CNVs larger than 25 kB and containing more than 100 SNPs were flagged and compared to gene lists associated with brain development and synapse Gene Ontology terms. In addition, DNA from iAstrocytes in each experiment was isolated to confirm the APOE genotype. iPS cells were infected in suspension (in E8 + RI) with ultrahigh-titer lentiviral particles provided by ALSTEM, encoding pTet-O-Ngn2-puro (Addgene, 52047) and FUΔGW-rtTa (Addgene, 19780). To start neuronal induction, 100,000 infected iPS cells per cm2 were plated in N2-supplemented medium (DMEM/F12 + GlutaMAX (Thermo Fisher, 31331093), 3 g L−1 d-glucose (Thermo Fisher, A2494001), 1% N2 supplement B (StemCell Technologies, 07156) and 0.1% P/S) supplemented with 5 µM RI, 2 μg ml−1 doxycycline hyclate (Sigma-Aldrich, D9891) and dual SMAD inhibitors (100 nM LDN193189 (Stemgent, 04-0074), 10 μM SB431542 (Tebu-Bio, T1726) and 2 μM XAV939 (Sigma-Aldrich, X3004)). On day 2, 100% of the medium was refreshed (including all day 1 supplements except RI) and 3 µg ml−1 puromycin was added (Cayman Chemical, 13884-25). On day 3, 100% medium was exchanged for N2-supplemented medium with doxycycline hyclate, puromycin and 10 µM 5-fluoro-2′-deoxyuridine (Sigma-Aldrich, F0503). Six-well plates were coated with 20 µg ml−1 poly(l-ornithine) (PLO; Sigma-Aldrich, P3655) overnight at room temperature (RT), followed by three wash steps with PBS on day 4. PLO-coated wells were subsequently coated with 5 µg ml−1 laminin (lam; BioTechne, 3400-010-02) for 2–4 h at 37 °C. iNeurons were washed with 1× PBS before dissociation with Accutase (Merck, SCR005) for 5 min at 37 °C. iNeurons were collected in DMEM (VWR, 392-0415P) and pelleted by a 5-min spin at 180g. iNeurons were resuspended and plated at 600,000 cells per well in PLO–lam-coated six-well plates in Neurobasal medium (NBM; Fisher Scientific, 11570556), supplemented with 200 mM GlutaMAX (Thermo Fisher, 35050038), 3 g L−1 d-glucose (Thermo Fisher, A2494001), 0.5% nonessential amino acids (NEAA; Fisher Scientific, 11350912), 2% B27 (Fisher Scientific, 17504044), 0.1% P/S, 10 ng ml−1 brain-derived neurotrophic factor (StemCell Technologies, 17189321), 10 ng ml−1 ciliary neurotrophic factor (CNTF; Peprotech, 450-13) and 10 ng ml−1 glial-cell-derived neurotrophic factor (StemCell Technologies, 78058.3). iNeurons were cultured at 37 °C and 5% CO2 and medium was replaced with 50% fresh medium once a week. The purity of iNeuronal cultures was assessed by determining the percentage of MAP2-positive cells in culture. Columbus version 2.5.2 (PerkinElmer) was used to detect intact nuclei and quantify MAP2 signal intensity in a 6-μm ring around the nucleus. The MAP2-positive neuron population was determined as the percentage of MAP2-positive cells with a signal-to-noise ratio (SNR) > 3. iPS cells were differentiated to neuronal progenitor cells (NPCs) according to a previous study96. On day 1, iPS cells were plated at 100% density in six-well plates in NMM medium (50% DMEM/F12 + GlutaMAX (Thermo Fisher, 31331093), 50% NBM, 100 mM GlutaMAX, 0.5% N2 supplement B, 1% B27, 0.5% ITS-A (Thermo Scientific, 51300044), 0.5% NEAA, 0.08% 2-mercaptoethanol (Fisher Scientific, 11528926) and 1% P/S supplemented with 10 µM SB431542 and 0.5 µM LDN193189. Complete medium was replaced daily for 7 days. On day 8, cells were expanded to PLO–lam-coated 6-cm dishes. Then, 1 ml of EDTA per well was added after one PBS wash and cells were incubated at 37 °C for 3–4 min. Cells were collected in clumps using a cell scraper and plated in 5 ml of complete NMM medium supplemented with 5 μM RI. On day 9, medium was exchanged for plain NMM medium without inhibitors after one PBS wash. This medium was refreshed daily for two more days. On day 12, the medium was exchanged for NMM medium supplemented with 10 ng ml−1 fibroblast growth factor (FGF; Peprotech, 100-18B). This medium was refreshed daily for two more days. On day 15, cells were incubated in Accutase after one PBS wash for 5 min at 37 °C and collected in NMM + 5 µM RI. After a 5-min spin at 1,000 rpm, the pellet was resuspended in NMM supplemented with FGF and RI before plating the NPC cells (passage 1, P1) in two PLO–lam-coated 10-cm dishes. NPCs were maintained at high density and refreshed every 2–3 days. NPCs were plated for control stainings (Nestin/Pax6) at P4 to confirm NPC identity, after which astrocyte differentiation was started according to Fong et al.45. One confluent 10-cm dish of NPCs was washed with 1× PBS before adding 9 ml of NMM + FGF. Cells were collected in clumps by cell scraper and transferred at 3 ml per well to a noncoated six-well plate. Plates were placed on an orbital shaker (90 rpm) in a 37 °C incubator. After 24 h, when tiny neurospheres had formed, 5 µM RI was added per well. Then, 48 h later, the medium was changed back to NMM without FGF. Next, 1 week after cell scraping of the NPCs, the NMM medium was exchanged for astrocyte medium (AM; ScienCell, 1801); subsequently, the medium was refreshed three times a week for the next 2 weeks. Neurospheres from three wells were collected and plated in one PLO–lam-coated 10-cm dish. iAstrocytes differentiated from the neurospheres were passaged to uncoated 10-cm dishes using Accutase and maintained in AM + 2% FBS (ScienCell, 1801/0010) until P4. Columbus version 2.5.2 (PerkinElmer) was used to detect intact nuclei and quantify AQP4 and vimentin intensity in each cell. of the mean secondary antibody signal. iMic were generated according to a previous study39, with small modifications. In brief, iPS cells were detached with Accutase (Gibco) and collected as a single-cell suspension. After centrifugation (5 min, 300g, RT), 2.5 million cells were plated into 24-well AggreWell800 plates (StemCell Technologies; pretreated with anti-adherence rinsing solution) in 2 ml of embryoid body (EB) induction medium (mTeSR+ (StemCell Technologies) + 20 ng ml−1 stem cell factor (R&D Systems) + 50 ng ml−1 bone morphogenetic protein 4 (BMP4; Miltenyi) + 50 ng ml−1 vascular endothelial growth factor (Miltenyi), supplemented with 10 μM Y27632 (StemCell Technologies) for the first 24 h) per well to generate EBs. To allow the formation of EBs, cells remained in AggreWell plates with daily 75% medium changes for 5 days. After 5 days, EBs were isolated and equally distributed to two six-well plates (Corning) in 2 ml of EB differentiation medium (X-Vivo 15 (Lonza), 2 mM GlutaMAX (Gibco), 0.55 mM β-mercaptoethanol (Gibco), 100 U per ml and 100 μg ml−1 P/S (Thermo Fisher Scientific), 25 ng ml−1 IL-3 (Miltenyi Biotec) and 100 ng ml−1 macrophage colony-stimulating factor (Miltenyi Biotec)) per well. The EBs were kept in EB differentiation medium at 37 °C and 5 % CO2 with full medium changes every 7 days. After 2–3 weeks, nonadherent microglial precursor cells (pre-iMics) started to be released into the medium from EBs. Pre-iMics were isolated during regular medium changes by collecting the supernatant medium and straining through a 40-µm cell strainer (Greiner). Pre-iMics isolated in weeks 3–6 after emergence were pooled and sustained in EB differentiation medium in T75 flasks (Corning) with weekly medium changes. Once sufficient cell numbers were collected, pre-iMics were plated at 15,000 cells per cm2 in T175 flasks (Sarstedt) in iMic medium (50% advanced NBM (Gibco), 50% advanced DMEM/F12 (Gibco), 1× B27 supplement with vitamin A (Gibco), 2 mM GlutaMAX (Gibco), 0.1 mM β-mercaptoethanol (Gibco), 100 ng ml−1 IL-34 (Miltenyi Biotec) and 20 ng ml−1 macrophage colony-stimulating factor (Miltenyi Biotec)) and differentiated to iMics for 14 days. For each line, four replicates were plated and processed in parallel. iMics were cultivated at 37 °C and 5% CO2 with three full medium changes per week. On day 14 (or on days 0, 3 or 7 for qPCR), iMics were washed briefly with PBS and detached with Accutase for 6–7 min at 37 °C until cells detached upon tapping the flask. Cells were collected with wash buffer (advanced DMEM/F12 (Gibco) + 0.1% BSA fraction V (Gibco)) and centrifuged at 300g (5 min, RT) before they were resuspended in PBS and counted using a hemocytometer (Neubauer Zählkammer Improved, Bard). Appropriate volumes containing 1 million cells were transferred to 1.5-ml microcentrifugation tubes (Eppendorf) and centrifuged at 400g (4 °C, 5 min). The supernatant was aspirated and cell pellets were frozen to −80 °C before shipment for lipidomics analysis. The quality and purity of EB cultures were assessed by flow cytometry. Pre-iMicroglias were isolated as described above from EB cultures. Cells were centrifuged at 300g (5 min, RT) and resuspended in 100 µl of fluorescence-activated cell sorting (FACS) buffer (PBS, 0.1% BSA fraction V (Gibco) and transferred into low-protein-binding 1.5-ml Eppendorf tubes. Cells were incubated with human Fc block (BD Biosciences, 564219) for 15 min at 4 °C in the dark on a rotor before 2 µl of each FACS antibody was added: APC anti-human CD45 antibody, mouse IgG1, HI30 (BioLegend, 304011) and Brilliant violet 421 anti-mouse/human CD11b antibody, clone M1/70, rat IgG2b (Biolegend, 101235). Pre-iMics were incubated with antibodies for 30 min at 4 °C in the dark on a rotor. Next, 1 ml of FACS buffer was added for washing and cells were centrifuged for 5 min (300g, 4 °C). The supernatant was removed and the cell pellet was resuspended in 500 µl of FACS buffer. In total, 20,000 cells per sample were analyzed. Unstained cells served as a gating control. Data analysis was conducted in FlowJo (BD Biosciences). Mouse astrocytes were isolated and cultured according to a protocol adopted from Clayton et al.97. Timed pregnant C57bl6j mice were purchased from Charles River or bred in house. Brains were extracted from both male and female pups (E18–P3). Cortices were isolated after the removal of meninges; cortices from each brain were digested in Papain (LK003150, Worthington) at 37 °C for 30–45 min and triturated with supplemented DMEM + DNAse. The cell suspension was centrifuged at 300g for 5 min and cell pellet was resuspended in 50% DMEM/F12 (Gibco, 31331093) and 50% NBM (Gibco, 11570556) supplemented with 1% N2 supplement (StemCell Technologies, 07156), 2% B27 (Fisher Scientific, 17504044), 1% GlutaMAX (Life Technologies, 35050038), 1% NEAA (Fisher Emergo, 11350912), 1% P/S (Fisher Emergo, 11548876), 0.3% glucose (Life Technologies), 5 μg ml−1 N-acetyl cysteine (Sigma, A9165), 20 ng ml−1 FGF2 (Peprotech, 100-18B), 10 ng ml−1 CNTF (Peprotech, 450-13), 10 ng ml−1 BMP4 (Peprotech, 120-05ET) and 5 ng ml−1 heparin-binding epidermal growth factor (hbEGF; Peprotech, 100-47) and filtered through a 100-μm filter before plating cells in a 10-cm dish (one brain per dish) that was precoated for 24 h with PLO (Sigma, P3655) followed by lam (BioTechne, 3400-010-02) for 2 h at 37 °C. On day 2, plates were washed with 1× PBS and the medium was replaced by expansion medium, 50% DMEM/F12 (Gibco, 31331093) and 50% NBM (Gibco, 11570556) supplemented with 1% N2 supplement (StemCell Technologies, 07156), 1% GlutaMAX (Life Technologies, 35050038), 1% NEAA (Fisher Emergo, 11350912), 1% P/S (Fisher Emergo, 11548876), 0.3% glucose (Life Technologies), 5 μg ml−1 N-acetyl cysteine (Sigma, A9165), 20 ng ml−1 FGF2 (Peprotech, 100-18B), 10 ng ml−1 CNTF (Peprotech, 450-13), 10 ng ml−1 BMP4 (Peprotech, 120-05ET) and 5 ng ml−1 hbEGF (Peprotech, 100-47). Cells were lifted by TrypLE and collected in DMEM/F12 (Gibco, 31331093) before spinning at 300g for 5 min. Pellets were resuspended and cells were replated for an experiment in 50% DMEM/F12 (Gibco, 31331093) and 50% NBM (Gibco, 11570556) supplemented with 1% N2 supplement (StemCell Technologies, 07156), 1% GlutaMAX (Life Technologies, 35050038), 1% NEAA (Fisher Emergo, 11350912), 1% P/S (Fisher Emergo, 11548876), 0.3% glucose (Life Technologies), 5 μg ml−1 N-acetyl cysteine (Sigma, A9165), 20 ng ml−1 FGF2 (Peprotech, 100-18B), 10 ng ml−1 CNTF (Peprotech, 450-13), 10 ng ml−1 BMP4 (Peprotech, 120-05ET) and 5 ng ml−1 hbEGF (Peprotech, 100-47); alternatively, cells were frozen in CryoStor (StemCell Technologies, 07959). For reactive astrocyte analysis mouse astrocytes were plated. Then, 24 h after plating, the medium was replaced by experiment medium, 50% DMEM/F12 (Gibco, 31331093) and 50% NBM (Gibco, 11570556) supplemented with 1% N2 supplement (StemCell Technologies, 07156), 1% GlutaMAX (Life Technologies, 35050038), 1% NEAA (Fisher Emergo, 11350912), 1% P/S (Fisher Emergo, 11548876), 0.3% glucose (Life Technologies), 5 μg ml−1 N-acetyl cysteine (Sigma, A9165) and 5 ng ml−1 hbEGF (Peprotech, 100-47). In parallel, astrocytes were cultured identically but 2% FBS was added to the culture medium (+FBS condition, to mimic the human astrocyte growth condition). Cells were cultured in their respective −FBS or + FBS medium for 5 days. After 5 days, the culture medium (+FBS or −FBS) was replaced by experiment medium without FBS for 24 h. After these 24 h, the medium was again replaced with experiment medium (without FBS) with or without the addition of the reactive cocktail (30 ng ml−1 TNF (300-01A, Peprotech), 3 ng ml−1 IL-1α (AF-200-01A, Peprotech) and 400 ng ml−1 C1q (204876, Sigma-Aldrich)). After 24 h, cells were lifted by TrypLE and pellets were collected and snap-frozen for lipidomics analysis. Each lipidomics or qPCR datapoint represents astrocytes from one independent mouse brain. Lipidomic analysis was undertaken on human postmortem brain material including frontal cortex gray matter, frontal cortex white matter and cerebellum tissue from 13 control donors and 20 donors with AD. Brain tissue was obtained from the Queen Square Brain Bank, University College London Queen Square Institute of Neurology. All donor information, including postmortem delay, age, sex, APOE genotype and pathological information, is listed in Supplementary Table 2. Ethical approval for the study was obtained from the National Health Service research ethics committee in accordance with the human tissue authority's code of practice and standards (license number 12198). Processing of postmortem samples for lipidomics was carried out as follows. After adding stainless-steel beads and liquid chromatography–mass spectrometry (LC–MS)-grade water, brain samples were homogenized using a Next Advance bullet blender. From these homogenized samples, aliquots containing the equivalent of 5 mg of tissue were prepared as described below. Briefly, 25 µl of Lipidyzer internal standard mix containing 54 deuterated standards was added to the cell pellet or 5-mg homogenized sample and extraction followed a methyl tert-butyl ether-based protocol. After drying under a gentle stream of nitrogen, samples were dissolved in running buffer (methanol and dichloromethane 1:1, containing 10 mM ammonium acetate) and injected into the Lipidyzer platform, consisting of a Sciex QTrap 5500 MS instrument equipped with a SelexION DMS interface and Nexera X2 ultrahigh-performance LC system. The order of samples was randomized before each batch was run on the Lipidyzer platform. Lipidyzer data analysis was further accomplished using SODA-light as a built-in data browser for the Neurolipid Atlas repository. Lipid species concentration datasets were imported and filtered, with individual species required to have a minimal intensity of two times the blank in at least 80% of all samples measured. If lipid species were absent or below two times the blank in >20% of all samples, they were removed. An exception was made for lipid species that were uniquely present in one group; if a lipid species was present in at least 60% of the samples (with a minimal intensity of two times the blank) within one of the experimental groups, the lipid species was reintroduced for the analysis. Because a dataset can have several grouping variables (for example, genotype, treatment and sample type), a new group variable is created by concatenating all grouping variables. This new group variable was used as the group variable for the blank filtering. No missing value imputation was applied. The SLA control software, including all up-to-date dictionaries and isotope correction algorithms can be found on Github (https://github.com/syjgino/SLA). SODA-light is a development branch of iSODA99 (https://github.com/ndcn/soda-ndcn) and part of the Neurolipid Atlas. SODA-light was forked as a lipidomics-only instance of iSODA99, a multiomics data visualization and integration application developed on R 4.4.0. As such, SODA-light is designed for efficient data exploration, providing interactive plots with extensive flexibility in terms of input data, analytical processes and visual customization. The code for SODA-light is available on GitHub (https://github.com/CPM-Metabolomics-Lipidomics/soda-light). SODA-light version 0.2 was used for the generation of all figures in this manuscript. Lipidyzer data analyzed by SLA can be added to the Neurolipid Atlas website by contacting the corresponding authors or the email address at https://neurolipidatlas.com/. Upload requires concentrations of individual lipid species in a sample of interest, along with blank (empty) samples and QCs (for example, serum). Human data need to be anonymized. Metadata items in bold (left column) need to be completed by the lab that provides the data. For adding other types of quantitative (targeted) lipidomics data, please contact the corresponding authors. To investigate differences in saturation of lipid classes between groups, the sum of the concentration of the lipid species with identical numbers of double bonds within the TGs, CEs or within all phospholipid classes was calculated. These summed values were normalized over total lipid concentration. For lipidomics, fully differentiated iAstrocytes were plated on day 1 at 17,000 cells per cm2 in AM + 2% FBS in uncoated 10-cm dishes. On day 2, the medium was replaced by AM without FBS after one PBS wash. After 24 h, on day 3, iAstrocytes were collected by Accutase dissociation after one PBS wash and counted. Cells were pelleted at 500,000 per vial using a centrifuge with a swing-out rotor. For reactive iAstrocyte lipidomics, the setup was identical; however, on day 3, the medium was replaced by AM without FBS supplemented with a reactive cytokine cocktail (30 ng ml−1 TNF (300-01A, Peprotech), 3 ng ml−1 IL-1α (AF-200-01A, Peprotech) and 400 ng ml−1 C1q (204876, Sigma-Aldrich)) or AM without FBS supplemented with an equal amount of PBS + 0.1% BSA (Tebu-Bio, 1501) as a control. On day 4, iAstrocytes were collected by Accutase dissociation after one PBS wash and counted. Cells were pelleted at 500,000 per vial using a centrifuge with a swing-out rotor. For one experiment comparing ApoE4 to ApoE3 iAstrocytes and one experiment comparing reactive to control iAstrocytes, lipidomics was performed as part of a multiomics experiment, as discussed below. Each lipidomics experiment included two or three replicate wells of iAstrocytes per condition. For integrative analysis, we generated samples for lipidomics and proteomics from the same preparation of BIONi037 (lipidomics experiment BIONi037 replicate I) and Kolf2.1J (lipidomics experiment Kolf2.1J set 1, replicate II) isogenic iAstrocytes. Transcriptomics was performed on a separate preparation for both isogenic sets (described below). To generate lipidomics and proteomics from one preparation, on day 1, fully differentiated iAstrocytes were plated at 17,000 cells per cm2 in AM + 2% FBS in uncoated 10-cm dishes. On day 2, the medium was replaced by AM without FBS after one PBS wash. After 24 h, on day 3, iAstrocytes were collected by Accutase dissociation after one PBS wash and counted. iAstrocytes were pelleted at 500,000 per vial separately for lipidomics and proteomics. Three replicate samples were generated for lipidomics and four replicate samples were generated for proteomics. iAstrocyte pellets were stored at −80 °C until shipment for further analysis. Generation of transcriptomic samples was performed under identical conditions (1,000,000 cells per vial; three replicates) for the Kolf2.1J isogenic set (set 1), whereas, for the BIONi037 isogenic set, the cells were kept in AM medium + 2% FBS. Similarly to above, lipidomic and proteomic samples were generated from the same preparation of reactive versus control iAstocytes for integrative analysis. Day 1 (fully differentiated) BIONi037-A (ApoE3/3) or Kolf2.1J set 1 (ApoE3/3) astrocytes were plated at 17,000 cells per cm2 in AM + 2% FBS in uncoated 10-cm dishes. On day 2, the medium was replaced by AM without FBS after one PBS wash. On day 3, the medium was replaced by AM without FBS supplemented with a reactive cytokine cocktail (30 ng ml−1 TNF (300-01A, Peprotech), 3 ng ml−1 IL-1α (AF-200-01A, Peprotech) and 400 ng ml−1 C1q (204876, Sigma-Aldrich)) or AM without FBS supplemented with an equal amount of PBS + 0.1% BSA (Tebu-Bio, 1501) as a control. After 24 h, iAstrocytes were collected by Accutase dissociation after one PBS wash. iAstrocytes from each replicate dish were divided over two vials in which (1 million for Kolf2.1J or 500,000 for BIONi037-A) iAstrocytes were pelleted for proteomic and lipidomic analysis. Cell pellets were stored at −80 °C before further processing. For iAstrocytes on the WTC11 background, differentiation was performed as previously described100, with minor modifications. Briefly, WTC11 iPS cells were edited to introduce a doxycycline-induced cassette driving proastrocyte transcription factors NFIA and SOX9. These iPS cells were differentiated into NPCs using dual SMAD inhibition and EB formation. NPCs were purified by FACS for CD133+CD271− populations. Purified NPCs were further differentiated into iAstrocytes by doxycycline treatment (2 μg ml−1; Millipore Sigma, D9891) and exposure to AM (ScienCell, 1801) for 20 days. For experiments using serum-containing growth conditions, day 20 iAstrocytes were plated at 20,000 cells per cm2 in phenol-red-free AM (prfAM) (ScienCell, 1801-prf) overnight, with a full medium change to fresh prfAM on day 21. The medium was changed to fresh prfAM every 2 days. After 24 h, the medium was removed, cells were washed with 1× Dulbecco's (DPBS) and cell pellets were collected and stored at −80 °C before further processing for lipidomics. Studies with human iPS cells at the University of California, San Francisco were approved by the Human Gamete, Embryo and Stem Cell Research Committee. Informed consent was obtained from participants when the iPS cell lines were originally derived. TMEM106B-knockout iPS cells, genetically engineered from the parental KOLF2.1J iPS cell line59, were obtained from iNDI60 through the Jackson Laboratory. These iPS cells, along with wild-type parental KOLF2.1J iPS cells, were cultured in feeder-free conditions on Matrigel in E8 medium (Life Technologies) and passaged by Accutase dissociation followed by E8 plus Chroman-I RI. A piggybac-based tet-on NGN2 transgene cassette was stably integrated into the genome of iPS cells as described previously (https://doi.org/10.17504/protocols.io.q26g744b1gwz/v1)101, followed by puromycin selection to eliminate iPS cells that did not successfully integrate the transgene. iNeurons were differentiated on six-well PLO-coated dishes, with a plating density of 500,000 cells per well on day 4, by doxycycline-induced expression of the NGN2 transgene, as described previously (https://doi.org/10.17504/protocols.io.8epv5969ng1b/v1)102. Cells were harvested on day 21 after doxycycline addition and snap-frozen before lipidomic analysis. Cells from two wells were combined into one 1.5-ml tube as one sample pellet. Day 1 (fully differentiated) BIONi037-A (ApoE3/3) astrocytes were plated at 26,000 cells per cm2 in AM + 2% FBS in uncoated six-well plates. On day 2, the medium was replaced by AM without FBS after one PBS wash. On day 3, the medium was replaced by AM without FBS supplemented with 50 µM methyl-β-cyclodextrin (MBCD)-coupled (water-soluble) cholesterol (C4951, Sigma-Aldrich) or AM without FBS supplemented with an equal amount of sterile water (TKF7114, Baxter) as a control. After 24 h, iAstrocytes were collected by Accutase dissociation after one PBS wash. iAstrocytes from each replicate well were pelleted for lipidomic analysis. Each lipidomics experiment included one or two replicate wells of iAstrocytes per condition. Cell pellets were stored at −80 °C before further processing. Frozen pellets corresponding to ~500,000 cells were dissolved in 25 μl of PBS supplemented with one tablet of cOmplete Mini EDTA-free protease inhibitor per 50 ml. One volume equivalent of 2× lysis buffer (100 mM HEPES pH 8.0, 50 mM DTT and 4% (w/v) SDS) was added. Samples were sonicated in a Bioruptor Plus (Diagenode) for ten cycles with 1 min on and 30 s off with high intensity at 20 °C. Samples were heated for 5 min at 95 °C and a second sonication cycle was performed as described above. Samples were alkylated using freshly made 15 mM iodoacetamide (Sigma-Aldrich, I1149) for 30 min at RT in the dark. Subsequently, proteins were acetone precipitated and digested using LysC (PTMScan; Cell Signaling, 39003) and trypsin (Promega sequencing-grade, V5111), as described previously103. The digested proteins were then acidified with 10% (v/v) trifluoracetic acid and desalted using a Waters Oasis HLB µElution plate (30 µm; Waters, 186001828BA) following the manufacturer's instructions. The eluates were dried down using a vacuum concentrator and reconstituted in 5% (v/v) acetonitrile and 0.1% (v/v) formic acid. Samples were transferred to an MS vial, diluted to a concentration of 1 µg µl−1 and spiked with iRT kit peptides (Biognosys, Ki-3002-2) before analysis by LC–MS/MS. Peptides were separated in trap/elute mode using the nanoAcquity MClass ultrahigh-performance LC system (Waters) equipped with trapping (nanoAcquity Symmetry C18, 5 μm, 180 μm × 20 mm) and analytical (nanoAcquity BEH C18, 1.7 μm, 75 μm × 250 mm) columns. Solvent A was water and 0.1% formic acid while solvent B was acetonitrile and 0.1% formic acid. First, 1 μl of the samples (∼1 μg on column) were loaded with a constant flow of solvent A at 5 μl min−1 onto the trapping column. Peptides were eluted through the analytical column with a constant flow of 0.3 μl min−1. During the elution, the percentage of solvent B increased nonlinearly from 0% to 40% in 120 min. The total run time was 145 min, including equilibration and conditioning. The LC instrument was coupled to an Orbitrap Exploris 480 (Thermo Fisher Scientific) using the Proxeon nanospray source. The peptides were introduced into the MS instrument using a Pico-Tip Emitter (outer diameter, 360 μm; inner diameter, 20 μm inner diameter; 10-μm tip; New Objective) heated at 300 °C and a spray voltage of 2.2 kV was applied. The capillary temperature was set at 300 °C. The radiofrequency ion funnel was set to 30%. For data-independent acquisition (DIA), full-scan MS spectra with a mass range of 350–1,650 m/z were acquired in profile mode in the Orbitrap with the resolution of 120,000 full width at half-maximum (FWHM). The default charge state was set to 3+. The filling time was set at a maximum of 60 ms with a limitation of 3 × 106 ions. DIA scans were acquired with 40 mass window segments of differing widths across the MS1 mass range. Higher-energy collisional dissociation fragmentation (stepped normalized collision energy: 25%, 27.5% and 30%) was applied and MS/MS spectra were acquired with a resolution of 30,000 FWHM at a fixed first mass of 200 m/z after the accumulation of 3 × 106 ions or after a filling time of 35 ms (whichever occurred first). Data were acquired in profile mode. For data acquisition and processing of the raw data, Xcalibur 4.3 (Thermo Fisher Scientific) and Tune version 2.0 were used. DIA raw data were analyzed using the directDIA pipeline in Spectronaut version 18 (Biognosys) with default settings besides the following parameters: protein LFQ method, QUANT 2.0; proteotypicity filter, only protein group specific; major group quantity, median peptide quantity; minor group quantity, median precursor quantity; data filtering, Q value; normalizing strategy, local normalization. The data were searched against UniProt (Homo sapiens, 20,375 entries) and a contaminants database (247 entries). The identifications were filtered to satisfy a false discovery rate (FDR) of 1% on the peptide and protein level. Relative protein quantification was performed in Spectronaut using a pairwise t-test performed at the precursor level followed by multiple-testing correction according to a previous study104. RNA isolation, QC, preprocessing and data analysis were performed as previously described from frozen pellets105. Briefly, total RNA was isolated from each sample using the Qiagen RNeasy mini kit. RNA samples for each participant were entered into an electronic tracking system and processed at the University of California, Irvine Genomics Research and Technology Hub. RNA was QCed using an Agilent Bioanalyzer and quantified by Nanodrop. RNA quality was measured as the RNA integrity number (RIN) and 260/280 and 260/230 ratios to evaluate any potential contamination. Only samples with RIN > 8 were used for library prep and sequencing. Library prep processing was initiated with total RNA of 1 µg using a Ribo-Zero gold ribosomal RNA depletion and Truseq Stranded total RNA kit. RNA was chemically fragmented and subjected to reverse transcription, end repair, phosphorylation, poly(A) tailing, ligation of barcoded sequencing adaptors and enrichment of adaptor-ligated complementary DNA (cDNA). RNA-seq libraries were titrated by qPCR (Kapa), normalized according to size (Agilent Bioanalyzer 2100 high-sensitivity chip). Each cDNA library was then subjected to Illumina (Novaseq 6000) paired-end, 100-cycle sequencing to obtain approximately 50–65 million paired-end reads. FASTQ files were subjected to QC and reads with quality scores > Q15 were collected. Raw reads were mapped to the GRCh38 reference genome using Hisat2 (version 2.2.1), QC, normalization and transformation before further exploratory and differential expression analysis. Raw counts were normalized and transformed using the ‘regularized log' transformation pipeline from the R package DESeq2. Statistical analyses was performed in R and differentially expressed genes detected for each covariate using FDR or Bonferroni adjustment for multiple-testing correction. PCA was performed using the plotPCA function in R with default settings. Following regularized log transformation in DESeq2, the top 500 highly variable genes were used as input for PCA and clustering of samples. Subsequently, we used the differentially expressed genes for each comparison to perform gene set enrichment analyses using Webgestalt106. Expression data of iAstrocytes from Tcw et al.10 and Lin et al.8 were analyzed and downloaded using GEO2R32. Tcw et al. performed bulk RNA-seq on four isogenic sets of APOE3/3 and APOE4/4 iAstrocytes, as well as bulk RNA-seq on seven APOE3/3 and six APOE4/4 population iAstrocyte lines. Lin et al. performed bulk RNA-seq on one isogenic set of APOE3/3 and APOE4/4 iAstrocytes. From all five isogenic sets and the population model, we gained the differential gene expression data of APOE4/4 versus APOE3/3. We calculated the average log2 fold change in ApoE4 versus ApoE3 of the pathways that were the top ten upregulated and downregulated pathways in our APOE4/4 versus APOE3/3 (BIONi037) transcriptomics for each line or the population data. On day 1, iAstrocytes were plated at 30,000–40,000 cells per cm2 in 6-well, 12-well and 96-well plates depending on the specific experiment in AM + 2% FBS (ScienCell). Then, 24 h after plating (day 2), the medium was replaced by AM without FBS after one PBS wash. On day 3, if needed, medium was collected and stored at −20 °C until further analysis by mesoscale discovery (MSD) cytokine ELISA. Attached iAstrocytes were either fixed by 3.7% formaldehyde (FA; Electron Microscopy Sciences, 15681) for 10–15 min at RT and stored at 4 °C in 1× PBS for immunofluorescence staining, collected by Accutase dissociation for flow cytometry, lysed in Laemmli sample buffer with DTT (LSB; made in house) for western blot, lysed with RLY plus TCEP (Meridian, BIO-52073; Thermo Scientific, 20491) for qPCR or lysed in radioimmunoprecipitation assay (RIPA) buffer (made in house) for BCA analysis. On day 1, iAstrocytes were plated at 30,000–40,000 cells per cm2 in 6-well, 12-well and 96-well plates depending on the specific experiment in AM + 2% FBS (ScienCell). Then, 24 h after plating (day 2), the medium was replaced by AM without FBS after one PBS wash. On day 3, iAstrocytes were treated with 50 µM MBCD-coupled (water-soluble) cholesterol (C4951, Sigma-Aldrich), 0.5 µM avasimibe (PZ0190, Sigma-Aldrich), the reactive cytokine cocktail (30 ng ml−1 TNF (300-01A, Peprotech), 3 ng ml−1 IL-1α (AF-200-01A, Peprotech) and 400 ng ml−1 C1q (204876, Sigma-Aldrich)) (dose 1) or lower titrated doses of the cocktail indicated by 0.5 (15 ng ml−1 TNF, 1.5 ng ml−1 IL-1α and 200 ng ml−1 C1Q), 0.25, 0.125 etc. Where indicated, iAstrocytes were preincubated for 1 h with 0.5 µM avasimibe (PZ0190, Sigma-Aldrich), 10 µM MBCD-coupled (water-soluble) cholesterol (C4951, Sigma-Aldrich) or 0.5 µM atorvastatin (HY-17379, MedChemExpress) before combined incubation with one of the previously mentioned treatments. Then, 24 h later, on day 4, if needed, the medium was collected and stored at −20 °C until further analysis by MSD cytokine ELISA. Attached iAstrocytes were fixed by 3.7% FA for 10–15 min at RT and stored at 4 °C in 1× PBS for immunofluorescence staining, collected by Accutase dissociation for flow cytometry, lysed in LSB for western blot, lysed with RLY plus TCEP for qPCR or lysed in RIPA buffer for BCA analysis. The medium was thawed and cellular debris was removed by a 5-min spin at 2,000g. Cytokine levels were determined by MSD Il-6 V-plex (K151QXD-2, MSD) according to the manufacturer's protocol. Medium samples were analyzed either undiluted or diluted 1:5 or 1:10 in diluent 2 when iAstrocytes were treated with the reactive cocktail. Raw cytokine values per well were normalized over nuclei number per well on the basis of fluorescence staining of the fixed iAstrocytes in the plate or over protein content per well determined by a Pierce BCA protein assay kit (Thermo Scientific, 23225). The Pierce BCA protein assay was performed in a microplate as described in the user guide provided by Thermo Scientific. The medium was thawed and tenfold concentrated with centrifugal filters (3-kDa molecular weight cutoff; Millipore, UFC200324). Raw values were normalized over protein content per well determined by a Pierce BCA protein assay kit (Thermo Scientific, 23225). The Pierce BCA protein assay was performed in a microplate as described in the user guide provided by Thermo Scientific. On day 1, iAstrocytes were plated at 30,000–40,000 cells per cm2 in six-well plates in AM + 2% FBS (ScienCell). Then, 24 h after plating (day 2), the medium was replaced by AM without FBS after one PBS wash. On day 3, iAstrocytes were treated with or without the reactive cytokine cocktail (30 ng ml−1 TNF (300-01A, Peprotech), 3 ng ml−1 IL-1α (AF-200-01A, Peprotech) and 400 ng ml−1 C1q (204876, Sigma-Aldrich)). The medium was collected after 24 h on day 4 and stored at −20 °C before shipment for analysis. Cells in each well were collected for lipidomics analysis. Secreted cholesterol levels in the medium samples were normalized to total lipid levels in the corresponding iAstrocyte cell pellets. For total cholesterol analysis, 1.0 ml of astrocyte conditioned medium was freeze-dried overnight. Subsequently, lipids were hydrolyzed using 200 µl of a 0.1 M ethanolic sodium hydroxide solution (80% ethanol) under nitrogen atmosphere and total cholesterol quantification using gas chromatography–MS followed published protocols107. After fixation, cells were permeabilized with 0.5% Triton X-100 (Fisher Scientific, T/3751/08) for 5 min at RT and blocked in PBS with 0.1% Triton X-100 and 2% NGS (Fisher Scientific, 11540526) for 30 min at RT or normal donkey serum (Biozol Diagnostika, LIN-END9000) in the case of microglia staining. Next, the iAstrocytes were incubated with the primary antibodies in blocking solution for 2 h at RT or overnight at 4 °C. The following primary antibodies were used: anti-Plin2 (15294-1-AP, Proteintech; 1:250), anti-AQP4 (AQP-004, Alomone Labs; 1:500), anti-GFAP (173 004, Synaptic systems; 1:1000), anti-vimentin (sc-6260, Santa Cruz Biotechnology; 1:1,000), anti-MAP2 (ab5392, Abcam; 1:500), anti-Smi 312 (SMI-312P-050, Eurogentec; 1:500), anti-Iba1 (NB100-1028, Novus Biologicals; 1:250), anti-PU.1 (MA5-15064, Thermo Fisher Scientific; 1:200) and anti-HLA class I heavy chain (kind gift from S. Neefjes; 1:250). After three washes in 1× PBS, the iAstrocytes were incubated with Alexa Fluor secondary antibodies (Invitrogen; 1:1,000), combined with DAPI (Carl Roth, 6843.1) and optional Lipidspot 488 in Fig. 6b (70065, Bio Connect; 1:1,000) in blocking solution, for 1 h at RT. Secondary antibodies used for microglia stainings were donkey anti-Goat Alexa-488 (Jackson ImmunoResearch 705-545-147) and donkey anti-rabbit Alexa-568 (Invitrogen, A10042). The iAstrocytes were washed three times in 1× PBS and either left in PBS to be imaged on the CellInsight CX7 LED Pro HCS platform (Fisher Scientific) or mounted on coverslips with Mowiol 4-88 (Sigma-Aldrich, 475904) for confocal imaging on a Nikon Ti-Eclipse microscope (Amstelveen), equipped with a confocal scanner model A1R+, using a ×40 oil-immersion objective (numerical aperture, 1.3). Image analysis was conducted using Columbus version 2.5.2 (PerkinElmer) after imaging on the Cell Insight CX7 and using Fiji108 after confocal imaging on the Nikon Ti-Eclipse. Cells were first identified on the basis of segmentation of the nuclei and cytoplasm to select only those fully contained within the field of view. For quantification of lipid droplets, the ‘find spots' function was applied for automated detection and counting of lipid droplets within each segmented cell. The fluorescence intensity or lipid droplet number of at least 50 cells per well was quantified in each experiment. After Accutase dissocation, FACS buffer (DPBS + 2% FBS (Fisher Scientific, A5256701)) was added to a volume of 300 µl and the iAstrocytes were transferred to a round-bottom 96-well plate. After centrifuging for 1 min at 2,000 rpm, the iAstrocytes were stained with 1:50 phycoerythrin anti-human HLA-A, HLA-B and HLA-C antibody (BioLegend, 311406) for 30 min at 4 °C. Following another centrifugation step, the iAstrocytes were fixed with 2% FA (Sigma-Aldrich, P6148) for 15 min at RT. After a last centrifugation step, the iAstrocytes were transferred to FACS tubes and 10,000 events of each sample were analyzed using BD LSRFortessa X-20 (BD Biosciences). Data analysis was performed in FlowJo (BD Biosciences). First, a life gate was set using the forward and side scatter area, after which the single cells were gated using the forward scatter width and height (Supplementary Fig. RNA was isolated using the Isolate II RNA mini kit (BIO-52073, GC Biotech) according to the manufacturer's protocol. RNA quantity and quality were analyzed by nanodrop. cDNA was generated by a SensiFAST cDNA synthesis kit (BIO-65054, GC Biotech) according to the manufacturer's protocol. The SensiFast SYBR Lo-Rox kit was used for performing qPCR on a QuantStudio 3 real-time PCR system (Thermo Fisher Scientific) and analysis was performed according to the ΔΔCt method to determine fold changes in RNA expression. For qPCR on iMic, the RNA Nucleospin Plus XS kit by Macherey-Nagel was used according to the kit's instructions for cell lysis and RNA extraction. The Applied Biosystems cDNA reverse transcription kit (Thermo Fisher Scientific) was used according to the manufacturer's protocol for reverse transcription. The list of primers used is provided in Supplementary Table 4. After lysing the iAstrocytes with LSB, samples were denatured at 95 °C for 5 min. The samples were shortly vortexed and loaded onto 4–15% Criterion TGX stain-free gel (Bio-Rad, 5678085). After running the gel (90 V for 30 min followed by 150 V for 45 min), the gel was transferred to a low-fluorescence (LF) PVDF membrane using the Trans-Blot Turbo RTA Midi 0.45-µm LF PVDF transfer kit (Bio-Rad, 1704275). After blocking in 5% skim milk powder (Sigma-Aldrich, 115363) for 1 h at RT on a shaker, the membrane was incubated with antibodies to HLA class I heavy chain (kind gift from S. Neefjes; 1:250), TAP1 (kind gift from S. Neefjes; 1:500), TAP2 (kind gift from S. Neefjes; 1:3,000) and GAPDH (elabscience, E-AB-40337; 1:3,000) overnight at 4 °C on a rocking plate. The following day, the membrane was incubated with secondary antibodies IRDye 800CW or IRDye 680RD (LI-COR; 1:10,000) for 1 h at RT on a shaker. No statistical methods were used to predetermine sample sizes but our sample sizes are similar to those reported in previous publications. In all graphs, open triangles or open circles indicate the mean result per experiment. Solid dots indicate the mean result of each replicate well in the experiments. Data collection and analysis were not performed blind to the conditions of the experiments because automated analysis was performed. Statistical analyses were performed on the mean values per experiment. For the comparison of lipid class changes in Figs. 2g, 8i, 9e and 10e, statistical analyses were performed on the percentage of total lipid values for each group (not fold change). One ApoE4/4 sample from lipidomics iAstrocytes BIONi037 ApoE4 versus ApoE3 I was excluded as outlier. 6j, one outlier experiment was removed on the basis of robust regression and outlier removal (Q = 1). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Bulk RNA-seq data of ApoE3/3 and ApoE4/4 iAstrocytes from the BIONi037-A and KOLF2.1J lines were deposited to the Gene Expression Omnibus under accession code GSE302826. All proteomics data were deposited to MassIVE: ApoE3/3 and ApoE4/4 iAstrocytes from the BIONi037-A line under accession code MSV000098665, ApoE3/3 and ApoE4/4 iAstrocytes from the KOLF2.1J line under accession code MSV000098666 and data of control and reactive iAstrocytes from the BIONi037-A and KOLF2.1J lines under accession code MSV000098668. All lipidomics data are available through https://neurolipidatlas.com/. Source data are provided with this paper. The code for SODA-light is available from GitHub (https://github.com/CPM-Metabolomics-Lipidomics/soda-light). Brain lipidomics: from functional landscape to clinical significance. Cell biology and function of neuronal ceroid lipofuscinosis-related proteins. Vance, J. E. Lipid imbalance in the neurological disorder, Niemann–Pick C disease. Comparative lipidomic analysis of mouse and human brain with Alzheimer disease. de Leeuw, S. M. et al. APOE2, E3, and E4 differentially modulate cellular homeostasis, cholesterol metabolism, and inflammatory response in isogenic iPSC-derived astrocytes. Di Paulo, G. & Kim, T. W. Linking lipids to Alzheimer's disease: cholesterol and beyond. Kant, Rvander et al. Cholesterol metabolism is a druggable axis that independently regulates tau and amyloid-β in iPSC-derived Alzheimer's disease neurons. Lin, Y. T. et al. APOE4 causes widespread molecular and cellular alterations associated with Alzheimer's disease phenotypes in human iPSC-derived brain cell types. Sienski, G. et al. APOE4 disrupts intracellular lipid homeostasis in human iPSC-derived glia. Fanning, S. et al. Lipidomic analysis of α-synuclein neurotoxicity identifies stearoyl CoA desaturase as a target for Parkinson treatment. Gedalya, T. Ben et al. Synuclein and polyunsaturated fatty acids promote clathrin mediated endocytosis and synaptic vesicle recycling. Boussicault, L. et al. CYP46A1 protects against NMDA-mediated excitotoxicity in Huntington's disease: analysis of lipid raft content. Boussicault, L. et al. CYP46A1, the rate-limiting enzyme for cholesterol degradation, is neuroprotective in Huntington's disease. Leoni, V. & Caccia, C. The impairment of cholesterol metabolism in Huntington disease. Restoring brain cholesterol turnover improves autophagy and has therapeutic potential in mouse models of spinocerebellar ataxia. B. et al. Alterations in lipid metabolism of spinal cord linked to amyotrophic lateral sclerosis. Giblin, A. et al. Neuronal polyunsaturated fatty acids are protective in ALS/FTD. Li, Y. et al. Microglial lipid droplet accumulation in tauopathy brain is regulated by neuronal AMPK. Liu, Y. et al. A C9orf72–CARM1 axis regulates lipid metabolism under glucose starvation-induced nutrient stress. & Wang, J. C9orf72-dependent lysosomal functions regulate epigenetic control of autophagy and lipid metabolism. & Krainc, D. Progranulin mutations result in impaired processing of prosaposin and reduced glucocerebrosidase activity. Marschallinger, J. et al. Lipid-droplet-accumulating microglia represent a dysfunctional and proinflammatory state in the aging brain. Andreone, B. J. et al. Alzheimer's-associated PLCγ2 is a signaling node required for both TREM2 function and the inflammatory response in human microglia. A. et al. TREM2 regulates microglial cholesterol metabolism upon chronic phagocytic challenge. Haney, M. S. et al. APOE4/4 is linked to damaging lipid droplets in Alzheimer's disease microglia. Guttenplan, K. A. et al. Neurotoxic reactive astrocytes induce cell death via saturated lipids. Haynes, P. R. et al. A neuron–glia lipid metabolic cycle couples daily sleep to mitochondrial homeostasis. Capolupo, L. et al. Sphingolipids control dermal fibroblast heterogeneity. Edgar, R., Domrachev, M. & Lash, A. E. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. The PRIDE database and related tools and resources in 2019: improving support for quantification data. Barrett, T. et al. NCBI GEO: archive for functional genomics data sets—update. Bozek, K. et al. Organization and evolution of brain lipidome revealed by large-scale analysis of human, chimpanzee, macaque, and mouse tissues. Brain delivery and activity of a lysosomal enzyme using a blood–brain barrier transport vehicle in mice. Byers, B. et al. SNCA triplication Parkinson's patient's iPSC-derived DA neurons accumulate α-synuclein and are susceptible to oxidative stress. Israel, M. A. et al. Probing sporadic and familial Alzheimer's disease using induced pluripotent stem cells. Modeling Alzheimer's disease with iPSCs reveals stress phenotypes associated with intracellular Aβ and differential drug responsiveness. The familial Alzheimer's disease APPV717I mutation alters APP processing and tau expression in iPSC-derived neurons. Haenseler, W. et al. A highly efficient human pluripotent stem cell microglia model displays a neuronal-co-culture-specific expression profile and inflammatory response. Handel, A. E. et al. Assessing similarity to primary tissue and cortical layer identity in induced pluripotent stem cell-derived cortical neurons through single-cell transcriptomics. Combining NGN2 programming with developmental patterning generates human excitatory neurons with NMDAR-mediated synaptic transmission. TCW, J. et al. An efficient platform for astrocyte differentiation from human induced pluripotent stem cells. Nimsanor, N. et al. Generation of induced pluripotent stem cells derived from a 77-year-old healthy woman as control for age related diseases. Rapid single-step induction of functional neurons from human pluripotent stem cells. Fong, L. K. et al. Full-length amyloid precursor protein regulates lipoprotein metabolism and amyloid-β clearance in human astrocytes. Ghorasaini, M. et al. Cross-laboratory standardization of preclinical lipidomics using differential mobility spectrometry and multiple reaction monitoring. Su, B. et al. A DMS shotgun lipidomics workflow application to facilitate high-throughput, comprehensive lipidomics. Fitzner, D. et al. Cell-type- and brain-region-resolved mouse brain lipidome. Ferris, H. A. et al. Loss of astrocyte cholesterol synthesis disrupts neuronal function and alters whole-body metabolism. Pfrieger, F. W. & Ungerer, N. Cholesterol metabolism in neurons and astrocytes. Feringa, F. M. & van der Kant, R. Cholesterol and Alzheimer's disease; from risk genes to pathological effects. Akyol, S. et al. Lipid profiling of Alzheimer's disease brain highlights enrichment in glycerol(phospho)lipid, and sphingolipid metabolism. Bandaru, V. V. R. et al. ApoE4 disrupts sterol and sphingolipid metabolism in Alzheimer's but not normal brain. Raji, C. A., Lopez, O. L., Kuller, L. H., Carmichael, O. T. & Becker, J. T. Age, Alzheimer disease, and brain structure. Effects of dietary sphingomyelin on central nervous system myelination in developing rats. Corder, E. H. et al. Gene dose of apolipoprotein E type 4 allele and the risk of Alzheimer's disease in late onset families. Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease: a meta-analysis. Reiman, E. M. et al. Exceptionally low likelihood of Alzheimer's dementia in APOE2 homozygotes from a 5,000-person neuropathological study. Pantazis, C. B. et al. A reference human induced pluripotent stem cell line for large-scale collaborative studies. B., Cookson, M. R. & Ward, M. E. Tackling neurodegenerative diseases with genomic engineering: a new stem cell initiative from the NIH. Barbar, L. et al. CD49f is a novel marker of functional and reactive human iPSC-derived astrocytes. Plasma apolipoprotein E levels in longitudinally followed patients with mild cognitive impairment and Alzheimer's disease. Martínez-Morillo, E. et al. Total apolipoprotein E levels and specific isoform composition in cerebrospinal fluid and plasma from Alzheimer's disease patients and controls. Arnaud, L. et al. APOE4 drives inflammation in human astrocytes via TAGLN3 repression and NF-κB activation. Cell type-specific roles of APOE4 in Alzheimer disease. Lee, S. et al. APOE4 drives transcriptional heterogeneity and maladaptive immunometabolic responses of astrocytes. Mhatre-Winters, I., Eid, A., Han, Y., Tieu, K. & Richardson, J. R.Sex and APOE genotype alter the basal and induced inflammatory states of primary astrocytes from humanized targeted replacement mice. Liebisch, G. et al. Update on LIPID MAPS classification, nomenclature, and shorthand notation for MS-derived lipid structures. The lipidomics reporting checklist a framework for transparency of lipidomic experiments and repurposing resource data. Dakterzada, F. et al. Cerebrospinal fluid neutral lipids predict progression from mild cognitive impairment to Alzheimer's disease. The ACAT inhibitor CP-113818 markedly reduces amyloid pathology in a mouse model of Alzheimer's disease. Puglielli, L. et al. Acyl-coenzyme A: cholesterol acyltransferase modulates the generation of the amyloid β-peptide. Shibuya, Y. et al. Acyl-coenzyme A: cholesterol acyltransferase 1 blockage enhances autophagy in the neurons of triple transgenic Alzheimer's disease mouse and reduces human P301L-tau content at the presymptomatic stage. & Chang, T. Y. Inhibiting ACAT1/SOAT1 in microglia stimulates autophagy-mediated lysosomal proteolysis and increases Aβ1–42 clearance. Blanchard, J. W. et al. APOE4 impairs myelination via cholesterol dysregulation in oligodendrocytes. Decreased lipidated ApoE–receptor interactions confer protection against pathogenicity of ApoE and its lipid cargoes in lysosomes. A. et al. APOE traffics to astrocyte lipid droplets and modulates triglyceride saturation and droplet size. Labib, D. et al. Proteomic alterations and novel markers of neurotoxic reactive astrocytes in human induced pluripotent stem cell models. Akiyama, H. et al. Inflammation and Alzheimer's disease. Orre, M. et al. Reactive glia show increased immunoproteasome activity in Alzheimer's disease. Overmyer, M. et al. Astrogliosis and the ApoE genotype. An immunohistochemical study of postmortem human brain tissue. Liu, C. C. et al. Cell-autonomous effects of APOE4 in restricting microglial response in brain homeostasis and Alzheimer's disease. Yin, Z. et al. APOE4 impairs the microglial response in Alzheimer's disease by inducing TGFβ-mediated checkpoints. Mancuso, R. et al. Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-β pathology. The P522R protective variant of PLCG2 promotes the expression of antigen presentation genes by human microglia in an Alzheimer's disease mouse model. Early Alzheimer's disease pathology in human cortex involves transient cell states. Identification of a new cholesterol-binding site within the IFN-γ receptor that is required for signal transduction. O'Carroll, S. M., Henkel, F. D. R. & O'Neill, L. A. J. Metabolic regulation of type I interferon production. Zhang, B. C. et al. Cholesterol-binding motifs in STING that control endoplasmic reticulum retention mediate anti-tumoral activity of cholesterol-lowering compounds. Shin, H. & Chung, H. SMPDL3A links cholesterol metabolism to the cGAS–STING pathway. Limiting cholesterol biosynthetic flux spontaneously engages type I IFN signaling. Banks, W. A., Reed, M. J., Logsdon, A. F., Rhea, E. M. & Erickson, M. A. Knox, E. G., Aburto, M. R., Clarke, G., Cryan, J. F. & O'Driscoll, C. M. The blood–brain barrier in aging and neurodegeneration. Montagne, A. et al. Blood–brain barrier breakdown in the aging human hippocampus. Directed differentiation of human pluripotent stem cells to cerebral cortex neurons and neural networks. Clayton, B. L. L. et al. A phenotypic screening platform for identifying chemical modulators of astrocyte reactivity. Ghorasaini, M. et al. Congruence and complementarity of differential mobility spectrometry and NMR spectroscopy for plasma lipidomics. Olivier-Jimenez, D. et al. iSODA: a comprehensive tool for integrative omics data analysis in single- and multi-omics experiments. Leng, K. et al. CRISPRi screens in human iPSC-derived astrocytes elucidate regulators of distinct inflammatory reactive states. Cookson, M., Ward, M. & Flores, E. L. iNDI PiggyBac-TO-hNGN2 transfection protocol version 1. protocols.io https://doi.org/10.17504/protocols.io.q26g744b1gwz/v1 (2022). Flores, E. L. et al. iNDI transcription factor-NGN2 differentiation of human iPSC into cortical neurons version 1. protocols.io https://doi.org/10.17504/protocols.io.8epv5969ng1b/v1 (2022). Buczak, K. et al. Spatially resolved analysis of FFPE tissue proteomes by quantitative mass spectrometry. Storey, J. D. A direct approach to false discovery rates. Large-scale differentiation of iPSC-derived motor neurons from ALS and control subjects. Liao, Y., Wang, J., Jaehnig, E. J., Shi, Z., & Zhang, B.WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Pebriana, R. B., Simsek, S., Heijink, M. & Giera, M. Rapid analysis of sterols in blood-derived samples. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. This work was supported by the Chan Zuckerberg Initiative, collaborative pairs grant DAF2022-250616 to R.v.d.K. lab was supported by an Alzheimer's Association Grant (2019-AARG-643165), an Alzheimer's Association Grant through the AD strategic fund (ADSF-21-831212-C), an Alzheimer Nederland grant (WE-03-2017-09P), a Vidi 2021 (09150172110086) from the Dutch Research Council and a grant from an anonymous foundation. was supported by a ZonMW memorabel fellowship (733050515), Alzheimer Nederland pilot grant (WE.03-2019-13) and ZonMW Enabling Technologies Hotels (435005033). was supported by a Chan Zuckerberg Initiative donor-advised fund (2022-250474) and Hertie Foundation grant (P1230003). was supported by two grants from the National Institutes of Health (NIH; P01 NS084974-06A1 and R35 NS116872). was supported by a PhD Scholarship from the German Academic Scholarship Foundation (Studienstiftung des Deutschen Volkes) and the International Max-Planck Research School for the Mechanisms of Mental Functions and Dysfunctions. was supported by an Alzheimer Nederland fellowship (WE.15-2019-16). was funded by a European Molecular Biology Organization postdoctoral fellowship award and the Live Like Lou Foundation. A.G. was funded by an Alzheimer's Research UK PhD Scholarship. was funded by a California Institute for Regenerative Medicine grant (EDUC412812) and NIH grant (T32NS115706). lab was funded by an Alzheimer's Association Grant (ADSF-21-816750), a Ben Barres Early Career Acceleration Award by the Chan Zuckerberg Initiative Neurodegeneration Challenge Network and NIH National Institute of Aging grants (R01 AG062359 and U01 AG072464). was supported by the Intramural Research Program of the National Institute of Neurological Disorders and Stroke. This work was supported, in part, by the Chan Zuckerberg Initiative Neurodegeneration Challenge Network (award numbers 2020-221617, 2021-230967 and 2022-250618) to A.O. lab, K. de Kleijn and M. Verhage for helpful discussions of this work. We thank R. Wijdeven and the Advanced Cytometry Research Core Facility of the VU University Medical Center (O2Flow) for technical support. We thank J. Wu for assistance in processing RNA-seq data. We also thank the Genomics Research and Technology Hub Shared Resource of the Cancer Center Grant (P30CA62203) at the University of California, Irvine for facilities and carrying out RNA-seq. We thank F. Limone and the S. Liddelow lab for sharing their mouse culture protocol. We thank M. Heijink and I. Sidorov for technical assistance with total cholesterol quantification. Present address: Genentech, Inc., San Francisco, CA, USA These authors contributed equally: Femke M. Feringa, Sascha J. Koppes-den Hertog. These authors jointly supervised this work: Martin Giera, Rik van der Kant. Femke M. Feringa, Sascha J. Koppes-den Hertog, Iris Kruijff, Jorin Heijneman, Lill Eva Johansen & Rik van der Kant Sascha J. Koppes-den Hertog, Iris Kruijff, Jorin Heijneman & Rik van der Kant Lian Y. Wang, Rico J. E. Derks, Niek Blomberg, Damien Olivier-Jimenez, Yassene Mohammed & Martin Giera Department of Cellular Neurology, Hertie Institute for Clinical Brain Research, University of Tübingen, Tübingen, Germany Department of Psychiatry and Human Behavior, University of California, Irvine, Irvine, CA, USA Institute for Memory Impairments and Neurological Disorders, University of California, Irvine, Irvine, CA, USA Department of Neurodegenerative Disease, Queen Square Institute of Neurology, University College London, London, UK Alexander J. Cammack, Christina E. Toomey, Adrian M. Isaacs & Tammaryn Lashley Alexander J. Cammack, Ashling Giblin & Adrian M. Isaacs Institute for Neurodegenerative Diseases, University of California, San Francisco, San Francisco, CA, USA National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Department of Biochemistry and Biophysics, University of California, San Francisco, San Francisco, CA, USA Gerald Bronfman Department of Oncology, McGill University, Montreal, Quebec, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar codesigned and performed the proteomics experiments. provided the human brain samples for lipidomics sampling. A.G. generated the ALS neurons for lipidomics. generated the TMEM-line neurons for lipidomics. generated the lipidomics data on TMEM lines. designed and oversaw the generation of ALS lipidomics data. designed and oversaw the generation of microglia lipidomics data. provided the human brain samples for lipidomics sampling. Correspondence to Martin Giera or Rik van der Kant. R.K. has acted as a consultant for NewAmsterdam Pharma. is a coscientific founder of Montara Therapeutics, serves on the scientific advisory boards of Engine Biosciences, Casma Therapeutics, Cajal Neuroscience, Alector and Montara Therapeutics and is an advisor to Modulo Bio and Recursion Therapeutics. is a consultant to Boehringer-Ingelheim Pharma and an inventor of patent application WO2023172132A1. The other authors declare no competing interests. Nature Metabolism thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Alfredo Giménez-Cassina, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Representative image of iNeurons used for purity quantification. White circles in the right panel indicate ring region around automatically detected intact nuclei where MAP2 intensity was measured. Mean c-d) Representative images (c) and quantification (d) of the percentage of Vimentin or AQP4 positive ApoE3 and ApoE4 iAstrocytes. e) Representative flow cytometry plots showing that >97% of cells generated from the EB differentiation protocol are pre-macrophages that are then plated for final differentiation to iMicroglia. f) Representative images of differentiated iMicroglia (day 14). g) Relative mRNA expression levels determined by qPCR of indicated microglial genes at 0-14 days of differentiation from pre-macrophage stage. h) Heatmap of most differentiating lipid species different between iPSC-derived neurons, astrocytes and microglia. alpha = 0.8 is used for discriminant analysis. a) PCA analysis of unbiased lipidomic analysis from indicated brain areas (control group subjects only, showing PC1 and PC2). b) Heatmap with Z-scored lipid class concentrations (nmol/mg brain material) from indicated brain regions (control subjects only, main text Fig. 2c shows same data as percentage of total lipidome). c) PCA loadings plot shows how individual lipid species drive PC1 and PC2 in PCA plot of AD vs control brain tissue from Fig. d-e) PCA plot of unbiased lipidomic analysis of AD (purple) and control (green) brain tissue samples from (FC) white matter (d) and Cerebellum (e). f) Heatmap depicting changes in lipid classes (based on concentrations) for individual AD samples compared to the average of control samples. g) Average log2fold change in AD subject group compared to control samples per lipid class and brain area. *P<0.05, t-test or Mann-Whitney U test with Benjamini-Hochberg correction. h-j) Bar graphs present individual lipid class levels in FC gray matter (h), FC white matter (I) and cerebellum (j) in AD and control samples (group levels) as concentrations. k-m) Bar graphs present individual lipid class changes in FC gray matter (k), FC white matter (l) and Cerebellum (m) in AD versus control samples (group level) as percentage of total lipids. a) Representative sequencing result for confirming cell identity and expected ApoE genotype. b) Representative images of differentiated iAstrocytes, from both Kolf2.1J (set #1) and BIONi037 lines. c) Gene expression levels (as determined by RNAseq) of indicated mature and immature astrocyte markers in our iPSC-derived astrocytes. d) Relative ApoE levels secreted in the medium. Mean e) Percentage of LD positive (>5 LDs quantified) iAstrocytes. N=6, three independent experiments from two isogenic sets. In all panels open triangles indicate mean per experiment, while solid dots represent all independent wells. a-c) Volcano plots of lipid species changed in ApoE4 vs ApoE3 iAstrocytes in the independent replicate experiments in Kolf2.1J set #1 (a), BIONi037 (b) or Kolf2.1J set #2 (c). d) Bar graphs of lipid class levels in ApoE3 and ApoE4 iAstrocytes from all datasets presented in Fig. e-f) Fold change of cholesterol ester (e) or phospholipid (f) species with indicated number of double bonds in ApoE4 vs ApoE3 iAstrocytes. In all panels open circles indicate mean per experiment, while solid dots represent all independent wells. a-b) Gene-set enrichment analysis with redundancy reduction by a weighted set cover algorithm was performed on proteomics data from ApoE4 versus ApoE3 iAstrocytes for the Kolf2.1J set #1 and the BIONi037 lines. Plotted are the enrichment scores (for both lines) for the Reactome pathways significantly enriched in Kolf2.1J set #1 (a) and pathways significantly enriched in BIONi037 (b). c) Representative western blots and quantifications of TAP1 and TAP2 protein levels in ApoE4 van ApoE3 iAstrocytes. d) Plasma membrane MHC I levels (stained for HLA-A,B,C) by flow cytometry. 4k after outlier removal based on the ROUT method (Q=1). In all panels open triangles indicate mean per experiment, while solid dots represent all independent wells. a-b) Volcano plots present log2fold change of differentially expressed genes (DEGs) in ApoE4 vs ApoE3 iAstrocytes from BIONi037 (a) or Kolf2.1J set #1 (b) from transcriptomics experiments. Top ten genes with the highest log2fold change and top ten genes with the most significant P-value are labeled. c-d) Top 10 Reactome pathways upregulated or downregulated (with lowest FDR) in ApoE4 vs ApoE3 iAstrocytes by gene-set enrichment analysis of transcriptomics data. e) Heatmap shows transcriptomic changes of indicated genes (all genes from the Reactome interferon signaling pathway) in ApoE4 vs ApoE3 iAstrocytes from our study and previous studies (as indicated). f) Plin2 gene expression based on the transcriptomic analysis of ApoE4 vs ApoE3 iAstrocytes from BIONi037 or Kolf2.1J set #1. n=3 wells per genotype in each isogenic set. g) Relative mRNA expression levels of indicated genes in ApoE4 vs ApoE3 iAstrocytes quantified by qPCR. N=3 independent experiments from each isogenic set. *P<0.05 one sample t-test with Benjamini-Hochberg correction on the p-values. b) Bar graphs of lipid class levels in reactive and control iAstrocytes from all datasets presented in Fig. N=6, three independent experiments from two ApoE3 lines, Mean + sd. c) Relative levels of cholesterol in the medium in control vs reactive iAstrocytes (ApoE3). d) Fold change of cholesterol ester (left panel) or triacylglyceride (right panel) species with indicated number of double bonds in reactive vs control iAstrocytes (ApoE3). e) Summary data of changes in all detected lipid classes in reactive vs control iAstrocytes from the Kampmann lab (WTC11 iPSC line, ApoE3/3 cultured in 2% FBS). f-g) Fold change of phospholipid (f) or triacylglyceride (g) species with indicated number of double bonds in reactive vs control iAstrocytes (WTC11, ApoE3). h) Summary data of changes in all detected lipid classes in reactive vs control iAstrocytes (ApoE4). i) Fold change of phospholipid, triacylglyceride or cholesterol ester species with indicated number of double bonds in reactive vs control iAstrocytes (ApoE4). All experiments presented in H are included here. j) Relative membrane MHC I levels (stained for anti-HLA-A,B,C) by flow cytometry in reactive vs control iAstrocytes (ApoE4). N=7, three or four independent experiments from two ApoE4 lines, Mean ****p<0.0001 one sample t-test. In all panels open triangles or circles indicate mean per experiment, while solid dots represent all independent wells. a) Representative image of purified mouse astrocytes. b) Relative mRNA expression of indicated genes in mouse astrocytes after treatment with the reactive cocktail analyzed by qPCR. c) Scatterplot presents log2fold change of individual lipid species in Kolf2.1J set #1 iAstrocytes vs mouse astrocytes. Both run in the same lipidomics batch to allow for direct comparison. d) Volcano plots present log2fold change of individual lipid species in reactive vs control astrocytes in mouse astrocytes cultured in FBS with 24h FBS removal (as for human astrocytes), mouse astrocytes that never saw FBS and Kolf2.1J iAstrocytes (ApoE3 from set #1) treated with TNF/Il-1α/C1q. e) Summary data of changes in all detected lipid classes in reactive vs control astrocytes of indicated datasets, corresponding to the data shown in D. Median + IQR. *P<0.05 paired t-test or paired Mann Whitney U test with Benjamini Hochberg correction. f) Fold change of phospholipid species with indicated number of double bonds in reactive vs control astrocytes of indicated datasets, corresponding to data shown in D. Mean. In all panels open circles and triangles indicate mean per experiment, while solid dots represent all independent wells. a) Venn diagram depicting the number of proteins that were significantly downregulated >1.25 fold (>0.3 log2fold) in reactive Kolf2.1J set #1, BIONi037 and both iAstrocytes. Top 10 (non-significant) enriched Reactome pathways detected by overrepresentation analysis are plotted. b) Heatmap shows the log2fold change of all proteins significantly upregulated >1.5 fold in both Kolf2.1J and BIONi037 reactive astrocytes. Next to it are the log2fold change values of these proteins in ApoE4 vs ApoE3 iAstrocytes. c) Heatmap shows the log2fold change of all proteins significantly downregulated >1.5 fold in Kolf2.1J and BIONi037 reactive astrocytes and the log2fold change values of these proteins in ApoE4 vs ApoE3 iAstrocytes. a) STRING analysis of all proteins significantly upregulated >1.5 fold in Kolf2.1J and BIONi037 reactive astrocytes (dataset from proteomics in Fig. Zoom in shows cluster of proteins related to cholesterol metabolism. b) Normalized Il-6 secretion in iAstrocytes (ApoE3), pre-treated with vehicle or avasimibe (0.5μM) for one hour and then treated for 24 hours with vehicle, cholesterol (50μM) or cholesterol + avasimibe. N=6, four or two independent experiments from two ApoE3 lines, Mean *P<0.05, ***P<0.001 one sample t-test for Chol and Ava + Chol vs 1. *P<0.05 t-test for Chol vs Ava + Chol with Benjamini-Hochberg correction on the three p-values. c) Relative mRNA expression of indicated genes after cholesterol (10μM) + avasimibe (0.5μM) treatment vs control, quantified by qPCR. N=5, two or three independent experiments from two ApoE3 lines, Mean + sem *P<0.05 one sample t-test with Benjamini-Hochberg correction on the p-values. d) Volcano plots represent log2fold change of individual lipid species in cholesterol treated vs control iAstrocytes in three independent replicate lipidomics experiments (BIONi037-A). e) Summary data of changes in all detected lipid classes in cholesterol vs control treated iAstrocytes (BIONi037-A). f) Bar graphs of lipid classes in cholesterol vs control treated iAstrocytes from BIONi037-A background (% of total). All experiments presented in E are included here. g-h) Fold change of phospholipid (g) and triacylglyceride (h) species with indicated number of double bonds in cholesterol treated vs control iAstrocytes (BIONi037-A). All experiments presented in E are included here. In all panels open circles or triangles indicate mean per experiment, while solid dots represent all independent wells. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The Neurolipid Atlas: a lipidomics resource for neurodegenerative diseases. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02056-4'>A bottom-up septal inhibitory circuit mediates anticipatory control of drinking</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 09:45:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Drinking behavior is not only homeostatically regulated but also rapidly adjusted before any changes in blood osmolality occur, known as anticipatory thirst satiation. Homeostatic and anticipatory signals converge in the subfornical organ (SFO); however, the neural pathways conveying peripheral information to the SFO before changes in blood composition are incompletely understood. Here we reveal an inhibitory pathway from the medial septum (MS) to the SFO that is involved in the control of anticipatory drinking behavior in mice. MS γ-aminobutyric acid (GABA)ergic neurons encode water-satiation signals by integrating cues from the oral cavity and tracking gastrointestinal signals. These neurons receive inputs from the parabrachial nucleus and relay to SFOCaMKII neurons, forming a bottom-up pathway with activity that prevents overhydration. Disruption of this circuit leads to excessive water intake and hyponatremia. Our findings reveal a septal pathway that integrates multiple layers of presystemic signals to fine-tune drinking behavior. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout RNA sequencing data used in this study are available on the National Center for Biotechnology Information's Sequence Read Archive (accession no. Source data are provided with this paper. Additional data that support the findings of this study are available from the corresponding author upon request. The codes used in this study can be accessed at: https://github.com/Fred-Huangchenyuan/A-bottom-up-septal-inhibitory-circuit-mediates-anticipatory-control-of-drinking. Fitzsimons, J. T. Angiotensin, thirst, and sodium appetite. McKinley, M. J. et al. The sensory circumventricular organs of the mammalian brain. Oka, Y., Ye, M. & Zuker, C. S. Thirst driving and suppressing signals encoded by distinct neural populations in the brain. Zimmerman, C. A., Leib, D. E. & Knight, Z. A. Neural circuits underlying thirst and fluid homeostasis. Augustine, V. et al. Hierarchical neural architecture underlying thirst regulation. Augustine, V., Lee, S. & Oka, Y. Neural control and modulation of thirst, sodium appetite, and hunger. New neuroscience of homeostasis and drives for food, water, and salt. Wallace Lind, R., Thunhorst, R. L. & Johnson, A. K. The subfornical organ and the integration of multiple factors in thirst. & Johnson, A. K. The physiological regulation of thirst and fluid intake. Bourque, C. W. Central mechanisms of osmosensation and systemic osmoregulation. Stricker, E. M. & Hoffmann, M. L. Presystemic signals in the control of thirst, salt appetite, and vasopressin secretion. & Le Magnen, J. Eating as a regulatory control of drinking in the rat. Zimmerman, C. A. et al. Thirst neurons anticipate the homeostatic consequences of eating and drinking. Pool, A. H. et al. The cellular basis of distinct thirst modalities. Nakamura-Utsunomiya, A., Hiyama, T. Y., Okada, S., Noda, M. & Kobayashi, M. Characteristic clinical features of adipsic hypernatremia patients with subfornical organ-targeting antibody. Allen, W. E. et al. Thirst-associated preoptic neurons encode an aversive motivational drive. & Hunt, H. F. Effect of septal lesions on thirst in the rat as indicated by water consumption and operant responding for water reward. Blass, E. M. & Hanson, D. G. Primary hyperdipsia in the rat following septal lesions. Saad, W. A., de Arruda Camargo, L. A., Antunes-Rodrigues, J. & Simões, S. Effect of electrolytic and chemical lesion by ibotenic acid of the septal area on water and salt intake. Andersson, B., Leksell, L. G. & Lishajko, F. Perturbations in fluid balance induced by medially placed forebrain lesions. Vanderweele, D. A., Davison-Barb, S. & Abelson, R. M. Septal lesions and hyperdipsia in the Mongolian gerbil. Smardencas, A., Denton, D. A. & McKinley, M. J. Hyperdipsia in sheep bearing lesions in the medial septal nucleus. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. Saleem, M. et al. Sox6 as a new modulator of renin expression in the kidney. N, E. & Hull, C. L. Principles of behavior. An introduction to behavior theory. Tsai, N. Y. et al. Trans-Seq maps a selective mammalian retinotectal synapse instructed by nephronectin. Matsuda, T., Hiyama, T. Y., Kobayashi, K., Kobayashi, K. & Noda, M. Distinct CCK-positive SFO neurons are involved in persistent or transient suppression of water intake. el Ghissassi, M., Thornton, S. N. & Nicolaidis, S. Angiotensin II-induced thirst, but not sodium appetite, via AT1 receptors in organum cavum prelamina terminalis. Lenkei, Z., Corvol, P. & Llorens-Cortes, C. The angiotensin receptor subtype AT1A predominates in rat forebrain areas involved in blood pressure, body fluid homeostasis and neuroendocrine control. Matsuda, T. et al. Distinct neural mechanisms for the control of thirst and salt appetite in the subfornical organ. Klapoetke, N. C. et al. Independent optical excitation of distinct neural populations. Yamamoto, M. et al. Reversible suppression of glutamatergic neurotransmission of cerebellar granule cells in vivo by genetically manipulated expression of tetanus neurotoxin light chain. Ly, T. et al. Sequential appetite suppression by oral and visceral feedback to the brainstem. Shen, L. et al. A bottom-up reward pathway mediated by somatostatin neurons in the medial septum complex underlying appetitive learning. Menani, J. V., De Luca, L. A. & Johnson, A. K. Role of the lateral parabrachial nucleus in the control of sodium appetite. Kim, D. Y. et al. A neural circuit mechanism for mechanosensory feedback control of ingestion. Ryan, P. J., Ross, S. I., Campos, C. A., Derkach, V. A. & Palmiter, R. D. Oxytocin-receptor-expressing neurons in the parabrachial nucleus regulate fluid intake. & Madias, N. E. The challenge of hyponatremia. & Zietse, R. Diagnosis and treatment of hyponatremia: compilation of the guidelines. Augustine, V. et al. Temporally and spatially distinct thirst satiation signals. Palmiter, R. D. The parabrachial nucleus: CGRP neurons function as a general alarm. Karthik, S. et al. Molecular ontology of the parabrachial nucleus. Chiang, M. C. et al. Parabrachial complex: a hub for pain and aversion. Scott, T. R. & Small, D. M. The role of the parabrachial nucleus in taste processing and feeding. Wickersham, I. R., Finke, S., Conzelmann, K.-K. & Callaway, E. M. Retrograde neuronal tracing with a deletion-mutant rabies virus. Wang, H. et al. Parallel gut-to-brain pathways orchestrate feeding behaviors. Xu, L. et al. An H2R-dependent medial septum histaminergic circuit mediates feeding behavior. Lin, W. et al. Whole-brain mapping of histaminergic projections in mouse brain. Pachitariu, M. et al. Suite2p: beyond 10,000 neurons with standard two-photon microscopy. Preprint at bioRxiv https://doi.org/10.1101/061507 (2016). Nichterwitz, S. et al. Laser capture microscopy coupled with Smart-seq2 for precise spatial transcriptomic profiling. Liao, J. et al. De novo analysis of bulk RNA-seq data at spatially resolved single-cell resolution. Picelli, S. et al. Full-length RNA-seq from single cells using Smart-seq2. Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. We thank D. Wang for helpful discussions and X. Tong for assistance with customized code development. Cartoons in all figures were created using BioRender.com (agreement no. This study was supported by grants from National Key R&D Program of China (no. ), the National Natural Science Foundation of China (nos. ), the China Postdoctoral Science Foundation (no. and the Postdoctoral Fellowship Program of China Postdoctoral Science Foundation (no. These authors contributed equally: Lingyu Xu, Yuhao Sun, Chenyuan Huang, Yanrong Zheng. Zhejiang Key Laboratory of Neuropsychopharmacology, School of Pharmaceutical Sciences, Zhejiang Chinese Medical University, Hangzhou, China Lingyu Xu, Yanrong Zheng, Xiaolong Ma, Qiujie Shi, Mengting Liu, Xiaoyun Qiu, Chenshu Gao, Yi Wang & Zhong Chen Institute of Pharmacology and Toxicology, College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, China Yuhao Sun, Jialu Chen, Qikun Zhao, Jie Liao, Yi Wang & Zhong Chen Department of Surgery, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project and designed experiments. led and performed experiments. conducted RNA sequencing experiments and data analysis. performed patch-clamp electrophysiology experiments. analyzed and interpreted other experimental data. performed modeling and computational analysis. wrote the paper with input from all the authors. Correspondence to Zhong Chen. The authors declare no competing interests. Nature Neuroscience thanks Charles Bourque and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic of viral injection for identification the upstream inputs to the SFO (n = 3 mice). b, Representative images showing SFO local injection site and retrograde labelled neurons in the TS, triangular septal nucleus; MnPo, median preoptic nucleus; OVLT, MPA, MS/LS, Arc and MnR. Scale bars, 200 µm in the upper panel, 100 µm in the lower right panel. MnPO, OVLT, organum vasculosum laminae terminalis; MPA, medial preoptic area; MS, medial septum; LS, lateral septum, Arc, arcuate hypothalamic nucleus; MnR, median raphe nucleus. a, Timeline for dual labeling of dehydration-activated (FosTrap) and satiation-activated (c-Fos) neurons in Fos-CreERT2 mice. b, Representative MS histology showing dehydration-activated (red), satiation-activated (green), and overlapping neurons (n = 5 mice). Scale bar, Scale bars, 200 µm in the MS, 50 µm for the magnified images. c, Quantification of overlap across satiation-activated and dehydration-activated neurons. d-e, Dehydration-responded (d) and rehydration-responded (e) neuronal populations project to the SFO but not the MnPO (n = 5 mice). Scale bars, 200 μm in the MS and MnPO, 100 μm in the SFO. a, Schematic representation of the experimental design to determine whether MnPO and MS GABAergic neurons are the same GLP1r+ population. b, Representative images showing the GABAergic neurons (green) co-labelled with GLP1r (red) in the MS and MnPO. c, Quantification of the overlap between vGAT and GLP1r signals in the MS and MnPO (n = 6 mice). d, Schematic of viral injection to label the SFO-projecting GABAergic neurons in the MS and MnPO for floresence-guided microdissection. e, Schematics of Smart-seq2 coupled with LCM (Spatial-seq) for analysis of the MS and MnPO GABAergic neurons. f, Volcano plot showing the differentially expressed genes of MSvGAT relative to MnPOvGAT neuronal subpopulation (n = 4 mice). Statistical analysis was performed using two-sided tests. Genes with p-value < 0.05 and log2 fold change > 1 were considered significantly different. g, Heatmap showing differentially expressed genes between the SFO-projecting MSvGAT and MnPOvGAT neuron populations (n = 4 mice). h, Schematic of viral injection to label the SFO-projecting GABAergic neurons in the MS and MnPO for IHC validation. i, Representative images showing that the majority of mCherry+ neurons (red) in the MS, but not in the MnPO, are Sox6+. j, Quantification of the overlap between mCherry and Sox6 signals in the MS and MnPO (n = 3 mice). Data are mean ± SEM ****P < 0.0001. a, Schematic experimental setup. b, Peri-stimulus time histogram (PSTH) of MSvGAT-SFO circuit average responses to grooming behavior. c, Quantification of AUC ΔF/F0 during the ‘pre' (2 s prior to grooming) and ‘post' (5 s after the grooming onset) sessions (n = 5 mice). Data are mean ± SEM ns, non-significant (P > 0.9999) Pre versus Post using two-tailed paired t-test. a, Schematic of i.g. infusion of different solutions. b, Quantification of the total Fos positive numbers in the MS with different test solutions (n = 3 mice for the sham group, n = 4 mice for the saline group, n = 3 mice for the ddH2O group). Data are mean ± SEM. c, Quantification of the overlap between vGLUT2, vGAT and Fos signals with different test solutions (n = 3 mice for the sham group, n = 4 mice for the saline group, n = 3 mice for the ddH2O group). Data are mean ± SEM **P = 0.0090 for (Fos+ & vGAT+) % in ddH2O group versus sham group using ordinary two-way ANOVA with Turkey's multiple comparisons test. d, Representative in situ hybridization (FISH) images showing Fos (red) expression pattern co-labelled with vGAT (green) and vGLUT2 (magenta) in the MS. White arrows indicate Fos+ neurons co-labelled with vGLUT2 (magenta), grey arrows indicate Fos+ neurons co-labelled with vGAT (green), while open arrows indicate Fos+ neurons. Scale bars, 200 µm in the MS, 50 µm for the magnified images. Data are mean ± SEM. a, Schematic of the experimental setup during water intake in free-moving mice (lower) and representative field of view with circles indicating ROIs (upper). b, Heatmap of deconvolved activity rate from individual MSvGAT neurons aligned to licking onset. c, Classification of neuronal responses during drinking. Neurons were categorized as inhibited, activated, or unresponsive using a log-transformed response index. Pie chart shows the proportion of neurons across these categories. d, PSTH showing the average responses from individual MSvGAT neurons to lick onset, comparing drinking-inhibited group (blue traces) versus drinking-unresponsive group (grey traces). Data are mean ± SEM. e, Schematic illustration of intraoral infusion experimental setup. f, Heatmap showing deconvolved activity rate from individual MSvGAT neurons in response to different I.O. infusion rates (upper) and representative traces showing different single neuron responses to corresponding I.O. infusion rates (bottom). g. Classification of neuronal responses to different I.O. Neurons were categorized as inhibited, activated, or unresponsive using a log-transformed response index. Pie charts show the proportion of neurons across these categories for each infusion rate. h, Schematic illustration of intragastric infusion experimental setup. Representative photographs of the stomach before and after I.G. Scale bar, 1 cm. i, Heatmap showing normalized activity from individual MSvGAT neurons in response to I.G. j-k, Principal Component Analysis (PCA) with k-means clustering of individual MSvGAT neurons based on their normalized activity during I.G. infusion (j) and average activity traces for each identified cluster corresponding to I.G. a, Schematic of viral injection for loss-of-function of MnPOGLP1r neurons with representative histology images for oChIEF-mediated drinking suppression in water-restrained mice. Scale bar, 100 μm. b, Silencing MnPOGLP1r neurons does not affect MSvGAT-SFO-mediated drinking suppression during OFF-ON-OFF photostimulation. Black lines, licks. c, Quantification of total licks during OFF-ON-OFF cycles (n = 6 mice). Data are mean ± SEM **P = 0.0020 for the Pre versus Stim. and *P = 0.0191 for the Pre versus Post, using RM one-way ANOVA with Bonferroni's multiple comparisons test. a, Schematic of anterograde, monosynaptic tracing using Cre-dependent mWGA-mCherry virus targeting the MS. b, Representative images of the mWGA injection site in the MS and anterograde monosynaptic labeling in the SFO and the MnPO. Robust projections are observed in the SFO, but not in the MnPO (n = 3 mice). c, Schematic of Cre-dependent ArchT-GFP virus injection into the MS. d, Representative images of the ArchT-labeled fibers in the MS, SFO and MnPO. Dense fiber projections are apparent in the SFO, while no fiber projections are observed in the MnPO (n = 3 mice). Scale bars, 200 μm in the MS and MnPO, 100 μm in the SFO. a, Schematic of rabies retrograde tracing in vGAT-Cre mice. b, Representative images showing starter cells in the SFO and retrograde labelled cells in the MS, co-labelled with GAD1 (red) and vGLUT2 (magenta). Filled arrows indicate RV labelled neurons (green) in the MS. Scale bars, 100 µm in the SFO, 200 µm in the MS, 50 µm for the magnified images. c, Quantification of the overlap between vGLUT2, GAD1 and RV labelled neurons in the MS (n = 3 mice). d, Schematic of viral injection, fiber placement and representative histology images for ChrimsonR-mediated fiber photometry recording. Scale bars, 100 µm in the SFO, 200 µm in the MS. e, Heatmap of individual animal responses to the photostimulation (n = 3 mice). f, PSTH showing the average responses of SFOvGAT neurons to MSvGAT stimulation (n = 3 mice). g, Quantification of AUC ΔF/F0 for the 470 nm recording versus 410 nm recording (n = 3 mice). ns, non-significant (P = 0.5416) for the 470 nm recording versus 410 nm recording using two-tailed paired t-test. h, Schematic of anterograde trace for identification the downstream cell types of the MSvGAT neurons. i, Representative image (left) and quantifications (right) of the viral specificity in the MS (n = 3 mice). Scale bar, 200 µm in the MS, 100 µm for the magnified images. Data are mean ± SEM. j, Left, representative image showing the anterograde labelled neurons (red) co-labelled with CCK (green) and CaMKII (grey) in the SFO. Scale bars, 200 µm in the SFO, 100 µm for the magnified images; Right, quantification of the anterograde labelled neurons (red) co-labelled with CCK (green) and CaMKII (grey) in the SFO (n = 3 mice). Data are mean ± SEM. k, Schematic of anterograde viral tracing combined with immunohistochemistry (IHC) staining to identify the target cell types innervated by MSvGAT neurons in the SFO. l, Representative image showing anterogradely labeled neurons (red) co-labeled with AT1R (green) and CCK (cyan) in the SFO. Scale bars, 100 µm in the SFO, 20 µm for magnified images. m, Quantification of anterogradely labeled neurons (red) co-labeled with AT1R (green) and CCK (cyan) in the SFO (n = 6 mice). Data are mean ± SEM. n, Left, Schematic of viral injection, fiber placement and representative images histology in water-restrained mice. Scale bar, 100 μm. o, Example trace (upper) and heatmaps (lower) of calcium signals upon lick (n = 3 mice). p, Quantification of AUC Zscore during the pre and post sessions (n = 3 mice, 3 trials for each mouse). ***P = 0.0070 for Pre versus Post using two-tailed paired t-test. Data are mean ± SEM. a, Schematic of viral injection to identification monosynaptic inputs to MSvGAT neurons (left) and representative image showing starter cells in the MS (right). Scale bars, 200 µm. b, Schematic of strategy for retrograde tracing using rabies virus. c, Representative images showing monosynaptically connected neurons in the SFO, paraventricular hypothalamus (PVH), Hippocampus, the supramammillary nucleus (SuM), the hypothalamus, periaqueductal gray (PAG) and the parabrachial nucleus (n = 3 mice). Scale bars, 200 µm. d, Schematic of viral injection to identification monosynaptic inputs to SFO-projecting MSvGAT neurons using rabies virus strategy. e, Representative images showing no RV signals detected in the PVH, hippocampus, SuM, or PAG (n = 3 mice). Scale bars, 200 μm. The video shows calcium activity of a single GABAergic neuron in the MS during water drinking. The neuron was retrogradely labeled from the SFO in VGAT-Cre mice using AAV2/2-RETRO PLUS-DIO-GCAMP6S. The video shows mice's drinking behavior during optogenetic activation of the MS-SFO GABAergic pathway. Statistical methods and results. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Xu, L., Sun, Y., Huang, C. et al. A bottom-up septal inhibitory circuit mediates anticipatory control of drinking. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02355-y'>Structural constraint integration in a generative model for the discovery of quantum materials</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 09:06:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Billions of organic molecules have been computationally generated, yet functional inorganic materials remain scarce due to limited data and structural complexity. Here we introduce Structural Constraint Integration in a GENerative model (SCIGEN), a framework that enforces geometric constraints, such as honeycomb and kagome lattices, within diffusion-based generative models to discover stable quantum materials candidates. SCIGEN enables conditional sampling from the original distribution, preserving output validity while guiding structural motifs. This approach generates ten million inorganic compounds with Archimedean and Lieb lattices, over 10% of which pass multistage stability screening. High-throughput density functional theory calculations on 26,000 candidates shows over 95% convergence and 53% structural stability. A graph neural network classifier detects magnetic ordering in 41% of relaxed structures. Furthermore, we synthesize and characterize two predicted materials, TiPd0.22Bi0.88 and Ti0.5Pd1.5Sb, which display paramagnetic and diamagnetic behaviour, respectively. Our results indicate that SCIGEN provides a scalable path for generating quantum materials guided by lattice geometry. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout We have compiled a comprehensive database of AL materials generated by SCIGEN. The dataset provides the folders of all generated materials (10.06 million), the materials that survived after the four-stage prescreening process (1.01 million materials) and DFT-relaxed structures (24,743). The folder with DFT calculation contains materials structures before and after relaxation. The Supplementary Dataset is available from figshare via https://doi.org/10.6084/m9.figshare.c.7283062 (ref. Source data are provided with this paper. The source code is available from GitHub via https://github.com/RyotaroOKabe/SCIGEN. Fu, L. Topological crystalline insulators. Article PubMed Bihlmayer, G., Noël, P., Vyalikh, D. V., Chulkov, E. V. & Manchon, A. Rashba-like physics in condensed matter. Martin, L. & Rappe, A. Thin-film ferroelectric materials and their applications. Hung, N. T. et al. Symmetry breaking in 2D materials for optimizing second-harmonic generation. Armitage, N. P., Mele, E. J. & Vishwanath, A. Weyl and Dirac semimetals in three-dimensional solids. Hashimoto, M., Vishik, I. M., He, R.-H., Devereaux, T. P. & Shen, Z.-X. Energy gaps in high-transition-temperature cuprate superconductors. Savary, L. & Balents, L. Quantum spin liquids: a review. Article PubMed Broholm, C. et al. Quantum spin liquids. Article PubMed Kang, M. et al. Topological flat bands in frustrated kagome lattice CoSn. Slot, M. R. et al. Experimental realization and characterization of an electronic Lieb lattice. Checkelsky, J. G., Bernevig, B. A., Coleman, P., Si, Q. & Paschen, S. Flat bands, strange metals and the Kondo effect. Van Speybroeck, V. et al. Advances in theory and their application within the field of zeolite chemistry. Article PubMed Chamorro, J. R., McQueen, T. M. & Tran, T. T. Chemistry of quantum spin liquids. Article PubMed Xie, T., Fu, X., Ganea, O.-E., Barzilay, R. & Jaakkola, T. Crystal diffusion variational autoencoder for periodic material generation. International Conference on Learning Representations https://openreview.net/pdf?id=03RLpj-tc_ (ICLR, 2022). Yang, M. et al. Scalable diffusion for materials generation. International Conference on Learning Representations https://openreview.net/pdf?id=2vt5z5x9fS (ICLR, 2024). Jiao, R. et al. Crystal structure prediction by joint equivariant diffusion. Merchant, A. et al. Scaling deep learning for materials discovery. Jiao, R., Huang, W., Liu, Y., Zhao, D. & Liu, Y. Space group constrained crystal generation. International Conference on Learning Representations 2024 (ICLR, 2024) Zeni, C. et al. A generative model for inorganic materials design. Cao, Z., Luo, X., Lv, J. & Wang, L. Space group informed transformer for crystalline materials generation. Preprint at https://arxiv.org/abs/2403.15734 (2024). Martinez, J. Archimedean lattices. Eddi, A., Decelle, A., Fort, E. & Couder, Y. Archimedean lattices in the bound states of wave interacting particles. Zimmermann, N. E. & Jain, A. Local structure order parameters and site fingerprints for quantification of coordination environment and crystal structure similarity. RSC Adv. Yin, J.-X., Lian, B. & Hasan, M. Z. Topological kagome magnets and superconductors. Tsai, W.-F., Fang, C., Yao, H. & Hu, J. Interaction-driven topological and nematic phases on the Lieb lattice. Mukherjee, S. et al. Observation of a localized flat-band state in a photonic Lieb lattice. Vicencio, R. A. et al. Observation of localized states in Lieb photonic lattices. Zhou, X. et al. High-temperature superconductivity. Pickett, W. E. The dawn of the nickel age of superconductivity. Merker, H. A. et al. Machine learning magnetism classifiers from atomic coordinates. Chang, J. et al. Direct observation of competition between superconductivity and charge density wave order in YBa2Cu3O6.67. Tranquada, J. et al. Coexistence of, and competition between, superconductivity and charge-stripe order in La1.6−xNd0.4SrxCuO4. Xie, T. & Grossman, J. C. Crystal graph convolutional neural networks for an accurate and interpretable prediction of material properties. Jain, A. et al. Commentary: The Materials Project: a materials genome approach to accelerating materials innovation. Pan, H. et al. Benchmarking coordination number prediction algorithms on inorganic crystal structures. Zachariasen, W. Metallic radii and electron configurations of the 5f–6d metals. Lugmayr, A. et al. Repaint: inpainting using denoising diffusion probabilistic models. In 2022 IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR) (ed O'Conner, L.) 11461–11471 (IEEE, 2022). Song, Y. et al. Score-based generative modeling through stochastic differential equations. International Conference on Learning Representations 2021 https://openreview.net/pdf?id=PxTIG12RRHS (ICLR, 2021). Davies, D. W. et al. Smact: Semiconducting materials by analogy and chemical theory. Open Source Softw. Geiger, M. & Smidt, T. e3nn: Euclidean neural networks. Preprint at https://arxiv.org/abs/2207.09453 (2022). Chen, Z. et al. Direct prediction of phonon density of states with Euclidean neural networks. Riebesell, J. et al. A framework to evaluate machine learning crystal stability predictions. Kresse, G. & Furthmüller, J. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Blöchl, P. E. Projector augmented-wave method. Kresse, G. & Joubert, D. From ultrasoft pseudopotentials to the projector augmented-wave method. Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Wang, V., Xu, N., Liu, J.-C., Tang, G. & Geng, W.-T. VASPKIT: a user-friendly interface facilitating high-throughput computing and analysis using VASP code. Momma, K. & Izumi, F. VESTA 3 for three-dimensional visualization of crystal, volumetric and morphology data. Stukowski, A. Visualization and analysis of atomistic simulation data with OVITO—the Open Visualization Tool. Okabe, R. Structural constraint integration in a generative model for discovery of quantum material candidates. thank C. Batista, A. Christianson, F. Frenkel, A. May, R. Moore, B. Ortiz and F. Ronning for helpful discussions. acknowledges support from the US Department of Energy (DOE), Office of Science (SC), Basic Energy Sciences (BES), award number DE-SC0021940 and the Heiwa Nakajima Foundation. A.C. acknowledges support from National Science Foundation (NSF) Designing Materials to Revolutionize and Engineer Our Future (DMREF) Program with award number DMR-2118448. are partially supported by the Artificial Intelligence Initiative as part of the Laboratory Directed Research and Development (LDRD) program of Oak Ridge National Laboratory (ORNL), managed by UT-Battelle, LLC, for the US Department of Energy under contract DE-AC05-00OR22725. This research used resources of the National Energy Research Scientific Computing Center (NERSC), a Department of Energy Office of Science User Facility using NERSC award DDR-ERCAP0030758. Computing resources for a portion of the work were made available through the VirtuES project, funded by the LDRD Program and Compute and Data Environment for Science (CADES) at ORNL. Another portion of simulation results were obtained using the Frontera computing system at the Texas Advanced Computing Center. were supported by the Department of Energy, grant DE-FG02-98ER45706. thank G. J. Miller for offering clusters to perform LMTO calculations. acknowledges the support from NSF ITE-2345084, the Class of 1947 Career Development Chair and support from R. Wachnik. Quantum Measurement Group, Massachusetts Institute of Technology, Cambridge, MA, USA Ryotaro Okabe, Mouyang Cheng, Abhijatmedhi Chotrattanapituk, Manasi Mandal, Kiran Mak, Denisse Córdova Carrizales, Nguyen Tuan Hung & Mingda Li Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA, USA Department of Materials Science and Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA Mouyang Cheng & Kiran Mak Center for Computational Science & Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA Mouyang Cheng Department of Electrical Engineering and Computer Science, Massachusetts Institute of Technology, Cambridge, MA, USA Abhijatmedhi Chotrattanapituk, Xiang Fu & Tommi S. Jaakkola Department of Nuclear Science and Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA Manasi Mandal, Denisse Córdova Carrizales, Nguyen Tuan Hung & Mingda Li Department of Physics, Massachusetts Institute of Technology, Cambridge, MA, USA Frontier Research Institute for Interdisciplinary Sciences, Tohoku University, Sendai, Japan Nguyen Tuan Hung Chemical Spectroscopy Group, Spectroscopy Section, Neutron Scattering Division, Oak Ridge National Laboratory, Oak Ridge, TN, USA Bowen Han & Yongqiang Cheng Department of Chemistry, Emory University, Atlanta, GA, USA Department of Chemistry, Michigan State University, East Lansing, MI, USA Department of Chemistry, Princeton University, Princeton, NJ, USA Robert J. Cava Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar led the project, developed the framework, implemented the code, trained the generative model and constructed the database. performed DFT calculations for electronic band structures and prepared related figures. A.C. developed the theoretical proof of our proposed method. performed DFT structural relaxations. carried out an analysis of experimental data. analysed the generated materials dataset. performed experimental synthesis and characterization. contributed to the establishment of evaluation metrics and proposal writing for computational resources. contributed to method discussions. supervised the overall project. contributed to writing the manuscript. Correspondence to Ryotaro Okabe, Weiwei Xie, Yongqiang Cheng or Mingda Li. The authors declare no competing interests. Nature Materials thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Discussion, Figs. Supplementary PDF file (532 pages) for generated materials after structure relaxation (24k). a–c, CIF files of the presented materials. d,e, Raw data for bar charts. a–g, CIF files of the presented materials. c,d, CIF files of the presented materials. c, Raw data for subplots. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Okabe, R., Cheng, M., Chotrattanapituk, A. et al. Structural constraint integration in a generative model for the discovery of quantum materials. Received: 18 July 2024 DOI: https://doi.org/10.1038/s41563-025-02355-y Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250921090853.htm'>Tiny new lenses, smaller than a hair, could transform phone and drone cameras</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 06:04:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new approach to manufacturing multicolor lenses could inspire a new generation of tiny, cheap, and powerful optics for portable devices such as phones and drones. "Our design has a lot of nice features that make it applicable to practical devices." "It's easy to manufacture because it has a low aspect ratio and each layer can be fabricated individually and then packaged together, it's also polarisation insensitive, and is potentially scalable through mature semiconductor nanofabrication platforms," Mr Jordaan said. The paper reporting their design is published in Optics Express. Initially the team attempted to focus multiple wavelengths with a single layer, but they hit up against some fundamental constraints, Mr Jordaan said. They guided the software to search for metasurface shapes that, for a single wavelength, created simple resonances in both the electric and magnetic dipole, known as Huygens resonances. By employing resonances, the team were able to improve on previous designs by other groups, and develop metalens designs that were polarization independent, and had greater tolerances in manufacturing specifications - crucial in the quest to scale fabrication to industrial quantities. The optimization routine came up with a library of metamaterial elements in a surprising range of shapes, such as rounded squares, four-leaf clovers and propellers. These tiny shapes, around 300 nm tall and 1000 nm wide, spanned the full range of phase shifts, from zero to two pi, enabling the team to create a phase gradient map to achieve any arbitrary focusing pattern - although they were initially just aiming for a simple ring structure of a conventional lens. However, the multilayer approach is limited to a maximum of around five different wavelengths, Mr Jordaan said. Within these constraints, Mr Jordaan said the ability to make metalenses to collect a lot of light will be a boon for future portable imaging systems. Materials provided by ARC Centre of Excellence for Transformative Meta-Optical Systems. Note: Content may be edited for style and length. Long-Standing Planetary Mystery Solved: Scientists Discover Mars Has an Earth-Like Core Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250921090850.htm'>New crystal camera lets doctors see inside the body like never before</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-22 01:46:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Physicians rely on nuclear medicine scans, like SPECT scans, to watch the heart pump, track blood flow and detect diseases hidden deep inside the body. But today's scanners depend on expensive detectors that are difficult to make. Now, scientists led by Northwestern University and Soochow University in China have built the first perovskite-based detector that can capture individual gamma rays for SPECT imaging with record-breaking precision. The new tool could make common types of nuclear medicine imaging sharper, faster, cheaper and safer. For patients, that could mean shorter scan times, clearer results and lower doses of radiation. "Perovskites are a family of crystals best known for transforming the field of solar energy," said Northwestern's Mercouri Kanatzidis, the study's senior author. "Now, they are poised to do the same for nuclear medicine. "Our approach not only improves the performance of detectors but also could lower costs," said co-corresponding author Yihui He, a professor at Soochow University. "That means more hospitals and clinics eventually could have access to the best imaging technologies." Yihui He is a former postdoctoral fellow from Kanatzidis' laboratory. Each gamma ray is like a pixel of light. Today's detectors, which are either made from cadmium zinc telluride (CZT) or sodium iodide (NaI), have several disadvantages. In 2012, his group built the first solid-film solar cells made from perovskites. Then, in 2013, Kanatzidis discovered that single perovskite crystals were highly promising for detecting X-rays and gamma rays. This breakthrough, enabled by his group's growth of high-quality single crystals, sparked a worldwide surge of research and effectively launched a new field in hard radiation detection materials. "This work demonstrates how far we can push perovskite detectors beyond the laboratory," Kanatzidis said. Now, we're showing that perovskite-based detectors can deliver the resolution and sensitivity needed for demanding applications like nuclear medicine imaging. By carefully growing and shaping these crystals, the researchers created a pixelated sensor -- just like the pixels in a smartphone camera -- that delivers record-breaking clarity and stability. He, Kanatzidis and their team demonstrated that perovskite-based detectors can achieve record energy resolutions and unprecedented single-photon imaging performance, paving the way for practical integration into next-generation nuclear medicine imaging systems. "Designing this gamma-ray camera and demonstrating its performance has been incredibly rewarding," He said. "By combining high-quality perovskite crystals with a carefully optimized pixelated detector and multi-channel readout system, we were able to achieve record-breaking energy resolution and imaging capabilities. This work shows the real potential of perovskite-based detectors to transform nuclear medicine imaging." It also sensed extremely faint signals from a medical radiotracer (technetium-99m) commonly used in clinical practice and distinguished incredibly fine features, producing crisp images that could separate tiny radioactive sources spaced just a few millimeters apart. Because these new detectors are more sensitive, patients potentially could require shorter scan times or smaller doses of radiation. Northwestern spinout company Actinia Inc. is commercializing this technology -- working with partners in the medical device field to bring it out of the lab and into hospitals. Because they are easier to grow and use simpler components, perovskites offer a far less expensive alternative to CZT and NaI detectors without sacrificing quality. "Demonstrating that perovskites can deliver single-photon gamma-ray imaging is a milestone," He said. "It shows these materials are ready to move beyond the laboratory and into technologies that directly benefit human health. From here, we see opportunities to refine the detectors further, scale up production and explore entirely new directions in medical imaging." "High-quality nuclear medicine shouldn't be limited to hospitals that can afford the most expensive equipment," Kanatzidis said. Long-Standing Planetary Mystery Solved: Scientists Discover Mars Has an Earth-Like Core Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            