
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-05</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a69644938/climate-change-sea-sponge-timeline/'>Oops, Scientists May Have Miscalculated Our Global Warming Timeline</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 16:03:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Clues have emerged that reveal a much hotter history than we thought. We may earn commission if you buy from a link. Whatever your stance is on climate change (it's real, let's move on), it's impossible to have missed the near-ubiquitous call to action to “keep temperatures from exceeding 1.5 degrees Celsius compared to pre-industrial levels.” Over the past few years, the somewhat bureaucratic phrase has become a rallying cry for the climate conscious. This ambitious target first surfaced following the Paris Climate Agreement, and describes a sort of climate threshold—if we pass a long-term average increase in temperature of 1.5 degrees Celsius, and hold at those levels for several years, we're going to do some serious damage to ourselves and our environment. Well, a paper from the University Western Australia Oceans Institute has some bad news: the world might've blown past that threshold four years ago. These sponges are commonly studied by climate scientists and are referred to as “natural archives” because they grow so slowly. This essentially allows them to lock away climate data in their limestone skeletons, not entirely unlike tree rings or ice cores. The sponges watery home in the Caribbean is also a plus, as major ocean currents don't muck up or distort temperature readings. This data could be particularly useful ,as direct human measurement of sea temperature only dates back to roughly 1850, when sailors dipped buckets into the ocean. That's why the Intergovernmental Panel on Climate Change (IPCC) uses 1850 and 1900 as its preindustrial baseline, according to the website Grist. That's a big “woah, if true” moment, but some scientists are skeptical. One such scientist, speaking with LiveScience, said that “ it begs credulity to claim that the instrumental record is wrong based on paleosponges from one region of the world … It honestly doesn't make any sense to me.” Other experts expressed wanting to see more data before completely upending the IPCC's climate goalposts, which say the Earth is currently hovering at a long-term temperature change of around 1.2 degrees Celsius. Unfortunately, even if the sponges are wrong, there's mounting evidence that we are in the process of crossing that 1.5 degree threshold as we speak. This January was the hottest on record, clocking in at 1.7 degrees above pre-industrial temperatures. According to New Scientist, that means we've been above 1.5 degrees of change for at least a year. In order to save the planet for future habitability, humans need to curtail emissions immediately—after all, the sea sponges are telling us so. Gut Feelings Are Memories From the Future, Scientists Say Earth's Hidden Eighth Continent Is No Longer Lost Dentist Finds Code In da Vinci's Vitruvian Man</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/cdc-vaccine-panel-scraps-guidance-for-universal-hepatitis-b-shots-at-birth/'>CDC Vaccine Panel Scraps Guidance for Universal Hepatitis B Shots at Birth</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 15:40:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>CDC Vaccine Panel Scraps Guidance for Universal Hepatitis B Shots at Birth New guidance from the CDC's vaccine advisory panel would do away with a decades-old universal birth dose recommendation for hepatitis B that helped cut infections by 99 percent in the U.S. The Centers for Disease Control and Prevention's Advisory Committee on Immunization Practices at its December 4 meeting. The vote passed with eight in favor and three against. The new Advisory Committee on Immunization Practices (ACIP) guidance recommends that the three-dose vaccine regimen begin at no earlier than two months of age for those infants. ACIP members also passed a second vote to recommend that parents discuss subsequent doses with a health care provider based on blood tests of the newborn's immunity levels (protective antibody titers).“I will just say: we have heard that do no harm is a moral imperative. And I vote no,” said ACIP member Cody Meissner, a pediatric infectious disease epidemiologist, during the first of three votes presented at today's meeting. The CDC didn't immediately respond to requests for comment from Scientific American. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The decision comes after several meetings filled with heated debate and admitted confusion among members on what they were voting on. Many public health experts and medical societies are “pretty clear that they do not want to eliminate the birth dose," she says. Some members also raised concerns over adverse effects and limited data on vaccine safety, despite dozens of studies and CDC reviews that have demonstrated that the hepatitis B vaccine is safe and highly effective at preventing infections in infants. Today's vote moves away from a vaccine policy established in 1991 that has dramatically reduced hepatitis B rates in children, according to a review of 400 studies published earlier this week. “The vaccine has essentially virtually eliminated childhood hepatitis B from the United States, and so it is really devastating to think about losing that,” says Yvonne Maldonado, a pediatrician at the Stanford University School of Medicine and a former ACIP member. Private and public insurers, including VFC, fully cover vaccines ACIP recommends under shared clinical decision-making. But public health experts have expressed concern that ACIP's discussion and new shared decision-making recommendation will still ultimately weaken people's ability to receive the vaccine. “When you remove the strength of that recommendation, pregnant women may or may not be offered that vaccine,” said Rochelle Walensky, former director of the CDC and now a senior fellow at Harvard University, in a press call on Thursday. Hepatitis B is a very contagious viral disease that damages the liver. The virus spreads through bodily fluids, including blood and semen. The disease has been stigmatized because people associate it with unprotected sex and injection drug use, but the virus can also be transmitted during pregnancy and through childbirth and close contact of bodily fluids after birth. “In my career, I have seen children with hepatitis B. I've seen children die of liver failure, which is a horrible death. I would never wish this on anybody,” Maldonado says. And the regimen has been effective: cases from 1990 to 2019 dropped by 99 percent. Delaying the first dose to the second month of age for infants born to pregnant parents who test negative for the virus or whose infection status is unknown could cause 1,437 more childhood infections, 304 liver cancer cases and 482 hepatitis B-related deaths, a recent preprint paper suggests. Public health experts note that the updated recommendation's success is contingent on effective screening for hepatitis B in pregnant people. Only 84 to 88 percent of pregnant people get screened for the virus in the U.S.; more than 80 percent of people are unaware that they're infected. And other caregivers, such as family members and daycare workers, who might not be screened could also carry the virus, Maldonado says. “The simplest, safest, most effective way to protect all of the babies in this country is to give them that birth dose, along with the subsequent two doses, which is going to keep them protected for virtually their entire young adult life,” she adds. Editor's Note: This is a developing story and may be updated. Young is associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/math/a69644580/dog-years-human-years-formula/'>Scientists Found the Formula to Calculate Your Dog's Actual Age—And It's Not What You Expect</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 15:39:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This method is much more accurate than simply multiplying by 7. Scientists have gained insight into the epigenetic clock that controls how dogs age and have devised a new formula to pinpoint exactly how old that good boy or girl in your life is. The researchers studied DNA methylation, or chemical modifications to certain DNA segments, within the genomes of 104 labrador retrievers, all ranging between 4 weeks and 16 years of age. Using this information, scientists have been able to create an epigenetic clock, of sorts, to better understand how we age. Scientists have learned that other animals like mice and wolves experience DNA methylation. Now, they're using this research to understand the process of aging in man's best friend. A dog's life stages largely sync up with our own; puppies and babies start teething at roughly equivalent ages, for example. Whether you have a pitbull, pug, or Pembroke Welsh corgi, your pup will reach puberty at around 10 months and will likely die before turning 20. Scientists have long known that dogs are susceptible to many of the same age-related diseases that we are, such as cancer, arthritis, and heart disease. The researchers also devised a new way to calculate a dog's age, but it's arguably more complicated than simply multiplying by seven. To calculate the age, you'll have to multiply the natural logarithm of your pup's age by 16 and then add 31. If you're wondering why your 2-year-old dog clocks in at around 40 human years, it's because his epigenetic clock ticks a bit faster than yours, but slows down as he ages. The team hopes to expand its research to include additional dog species, and there are plenty of other research teams that are diving into a dog's genome to unlock even more secrets. A graduate of the Science Communication Program at UC Santa Cruz, her work has appeared in The Atlantic, Scientific American, Science News and Nature. Her favorite stories illuminate Earth's many wonders and hazards. Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a69644378/why-dog-brains-are-getting-bigger/'>The Dogs of Today Are Undergoing a Mysterious Brain Expansion, Study Suggests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 15:07:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Modern dog breeds haven't quite caught up to the wolf when it comes to the size of their brains, but that size discrepancy is shrinking. The modern dog's brain is growing larger, and it could be thanks to their interactions with humans. The research team found that dog brains are getting larger the further genetically that they have evolved away from a wolf. “The results show that the breeding of modern dog breeds has been accompanied by an increase in brain size compared to ancient breeds,” Enikó Kubinyi, a senior research fellow at the Department of Ethology at ELTE Institute of Biology, says in a news release. In the study, a team from Hungary and Sweden used CT images of over 850 dogs representing 159 breeds to reconstruct different individuals' brains and determine their brain volume. This confirms past research that domestication decreases brain size by about 20 percent, as animals no longer need to hunt or fend for themselves—nor do they face as much peril as they do in the wild. “There is no need to sustain the energetically costly large brain, and the freed-up energy can be directed toward other purposes, such as producing more offspring,” László Zsolt Garamszegi, an evolutionary biologist at the Ecological Research Centre in Hungary, says in the news release, “which is important for domesticated animals.” And it didn't matter if the dog had energy-sucking responsibilities, such as herding or guarding. “Perhaps the more complex social environment, urbanization, and adaptation to more rules and expectations have caused this change,” Kubinyi speculates, “affecting all modern breeds.” The study says that functional category, skull shape, longevity, and litter size all had no association with relative brain size. “Different dog breeds live in varying levels of social complexity and perform complex tasks, which likely require a larger brain capacity,” Niclas Kolm of Stockholm University, says in the news release. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69587645/recharging-cells/'>Scientists Think We Could ‘Recharge' Our Cells—And Rewind the Aging Clock</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Think of cells as the biological answer to battery-powered electronics. Mitochondria are the batteries that supply them with enough energy to keep going. Unfortunately, just like the two standard AAs in your remote control, they eventually run out of power and die—but (much like actual batteries) they can also be recharged and replaced. Wear and tear can happen with age, usually from years of exposure to free radicals that cause oxidative stress and inflammation, but can also be caused by injury from degenerative diseases or mitochondrial toxicity from certain drugs and other harmful substances. This is why biomedical engineer Akhilesh Gaharwar and his research team at Texas A&M University have come up with a way to regenerate them. There is a repair mechanism already built in. While healthy cells can transfer some mitochondria to replace what was lost in ailing cells, much like an emergency run to the hardware store, this process is far from efficient when it comes to replacing all the dead batteries. Dysfunctional mitochondria are not only the result of neurodegenerative, cardiovascular or metabolic disorders, but can also be part of the cause. Gaharwar was able to fast-forward intracellular mitochondrial transfer by creating nanostructures which were then implanted into human mesenchymal stem cells. “Mesenchymal stem cells (MSCs) display a particular propensity for initiating mitochondrial transfer to nearby cells; their mitochondria enhance cellular respiration, induce cell reprogramming, and repair metabolic function in recipient cells,” he said in a study recently published in PNAS. Because they have strong immune capabilities and do not burn as much energy as many other types of cells, mesenchymal stem cells were ideal for the team to experiment on in vitro. The few methods that exist to boost mitochondrial transfer require an immense amount of effort and still face obstacles. Gaharwar got around this by creating “nanoflowers” out of molybdenum disulfide, a biomaterial that has also been used to eliminate free radicals that cause cell degeneration. What sound like alien blossoms are actually structures with vacancies that encourage the formation of mitochondria in the stem cell. Mesenchymal stem cells are already known for their ability transfer mitochondria. The presence of nanoflowers activated a pathway that increases the production of mitochondria, and pushed stem cells to generate up to twice the number of mitochondria they normally would. Smooth muscle cells found in the walls of blood vessels, especially arteries, need high energy levels and respiration efficiency for vascular contraction, a process which narrows blood vessels and helps control blood pressure. “Our MoS2 nanoflower-based strategy offers a broadly applicable platform for pathologies requiring mitochondrial repair and replacement,” he said. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Gut Feelings Are Memories From the Future, Scientists Say Earth's Hidden Eighth Continent Is No Longer Lost Dentist Finds Code In da Vinci's Vitruvian Man</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251205054727.htm'>Scientists reveal a powerful heart boost hidden in everyday foods</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 14:05:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>People who frequently include foods and beverages rich in polyphenols, such as tea, coffee, berries, cocoa, nuts, whole grains and olive oil, may experience better heart health over time. A team from King's College London reported that individuals who followed dietary patterns high in polyphenols had lower predicted cardiovascular disease (CVD) risk. Polyphenols are naturally occurring plant compounds associated with a wide range of health benefits, including support for the heart, brain, and gut. These biomarkers showed that people with higher levels of polyphenol metabolites (especially those linked to flavonoids and phenolic acids) had lower cardiovascular risk scores. This score reflects intake of 20 common polyphenol-rich foods in the UK, including tea, coffee, berries, olive oil, nuts, and whole grains. The PPS demonstrated stronger links to cardiovascular health than estimates of total polyphenol intake. Professor Ana Rodriguez-Mateos, senior author and Professor of Human Nutrition at King's College London, said: "Our findings show that long-term adherence to polyphenol-rich diets can substantially slow the rise in cardiovascular risk as people age. Even small, sustained shifts towards foods like berries, tea, coffee, nuts, and whole grains may help protect the heart over time." Dr. Yong Li, first author of the study, added: "This research provides strong evidence that regularly including polyphenol-rich foods in your diet is a simple and effective way to support heart health. These plant compounds are widely available in everyday foods, making this a practical strategy for most people." The researchers noted that although cardiovascular risk naturally increases with age, participants with higher polyphenol intake experienced a slower rise in risk across the 11-year follow-up period. They also highlighted the importance of future dietary intervention trials to confirm and expand on these findings. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69550280/haunting-warning-shattered-skull/'>Archaeologists Uncovered a Haunting Warning from the Roman Empire</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Maybe Game of Thrones is not exactly the best reflection of history, but there is some truth in that ghoulish scene where (spoiler alert) Ned Stark's decapitated head was skewered on a pike and displayed outside the gates of King's Landing. This was not out of the norm for Roman legions. They were known to expose severed heads, hands, feet, and even entire corpses to the elements and onlookers as an intimidation tactic that possibly doubled as a public display of victory. With an eye on capturing the Iberian Peninsula, they pursued the Cantabri, who lived in what is now northern Spain, in a series of conflicts known the Cantabrian Wars. While there is no evidence of it being raised on a pike, it was displayed above the wall as a ghoulish reminder of what would happen if anyone else tried to take down the Romans. “His intention was to complete the conquest of the Iberian Peninsula by defeating the last two Celtic nations to remain outside the control of Rome in western continental Europe,”  archaeologist Santiago Domínguez-Solera, director of Heroica Archaeology and Cultural Heritage, said of Augustus in a study recently published in Journal of Roman Archaeology. Domínguez-Solera, who led a team of researchers in excavating the La Loma site, discovered the skull among ruins covered in ash from a Roman fire and littered with arrowheads, bladed weapons, jewelry, and fragments of armor. The skull was not found in one piece. What remained of it was scattered, possibly broken when it fell from collapsing walls of the fort and was then crushed by them. Without a name or anything close to identifying information, the only way the skull could speak was through physical and genetic analysis. Bone fragmentation made it difficult to figure out an age or sex from physical remnants alone. It also became more obvious that the skull had been left outside when the researchers examined damage to the bone caused by weathering, such as cracking, flaking, splitting, and whitening. Because the skull had held on to 53% of its DNA, it was possible to recover its entire mitochondrial genome. Genetic evidence said more than any physical characteristics could. Further genetic analysis placed him in the north of Spain near other ancient individuals from Iron Age populations. But genetic analysis alone could not answer one pressing question: why did he meet such a ghoulish end? Egyptians who murdered Pompey the Great on Pharaoh Ptolemy's orders presented his head and ring to Caesar. After Cicero was executed for being an enemy of the state for his rivalry with Mark Antony, his dismembered head and hands were displayed. The shattered skull found at La Loma may still keep some secrets, but what was once a trophy is now a reminder that if anyone tried to oppose one of the most powerful empires in the world, heads would quite literally roll. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Gut Feelings Are Memories From the Future, Scientists Say Earth's Hidden Eighth Continent Is No Longer Lost Dentist Finds Code In da Vinci's Vitruvian Man</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/cosmic-magnification-is-one-of-the-universes-weirdest-optical-illusions/'>Do Objects Always Appear Smaller with Distance? Not on Cosmic Scales!</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 11:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Cosmic Magnification Is One of the Universe's Weirdest Optical Illusions Stay connected to The Universe: Get email alerts for this weekly column by Phil Plait It's always amazing, and more than a little humbling, when the universe reminds us that our “common sense” is provincial, falling apart on cosmic scales. If you're on the surface of Earth—and I'm betting you are—there are many ways to reliably estimate the distance to some object. Of course, some humans are larger or smaller than average, but you can still account for that to get a decent distance estimate. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. This all works great for familiar objects up to a few kilometers away, but astronomers are notoriously unsatisfied with these relatively tiny scales. We want to know the distances to objects that are trillions of kilometers away—or even billions of times farther than that! In deep images of the sky from giant telescopes, galaxies abound. Just by looking at the image, how could you tell? You might assume the galaxies that appear smaller are farther away, in keeping with our earthly intuition, but this won't work; like humans, galaxies come in a range of sizes. When you examine an image, you might be looking at a massive galaxy nearly at the edge of the observable universe—or a tiny dwarf galaxy right in our cosmic backyard. Just judging by the image, it's impossible to tell. It's possible there could be some standard physical scale to galaxies, some way to gauge their distance by linking details of their structure to their overall size, but such scenarios are apparently too simple for the universe's true complexity. It turns out that the standard linear scaling we use on Earth would only apply at cosmic scales if our universe were static—unchanging in size over time—which it isn't! Instead the universe is expanding, growing larger every day. As so often happens, the cosmos really is a lot odder than you think. When we say a galaxy is, for example, 12 billion light-years away, what we usually mean is that the light from that galaxy took 12 billion years to reach us. When the light left that galaxy 12 billion years ago, the galaxy was closer to us, so it appears bigger than expected for that distance once its light arrives here! Their apparent size isn't affected enough to detect. But the effect ramps up with distance and begins to dominate for objects with a light travel time of about 9.5 billion years. Around that point, we're looking so far back in time—so far back in the universe's history of expansion—that the galaxies are effectively magnified, appearing larger than they otherwise would. The exact distance where this effect really kicks in depends on many complicated factors, including how rapidly the universe expands and how much matter it contains. In fact, if we could precisely measure this apparent growth in size, we could then use it to better determine these important cosmological parameters. Unfortunately, because galaxies don't come in standard sizes, that's quite a difficult task. Worse, this cosmic magnification effect confusingly makes some galaxies look dimmer: if they appear bigger, their light is more spread out, so they become fainter and even more difficult to observe! As surprising as this effect is, what may be even more astonishing is that we've been able to see far enough to detect it at all from our planetary perch in the backwaters of the Milky Way. Doing so can reveal information about such objects, of course, but also about the universe around them and the way that it behaved when it was very young. Provided, of course, that we take to heart this hard lesson: once we start talking about distances measured in billions of light-years, our own parochial evolution utterly fails us, and we have to be very careful not to extrapolate willy-nilly from our own experience on Earth. Phil Plait is a professional astronomer and science communicator in Virginia. His column for Scientific American, The Universe, covers all things space. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02915-2'>In vivo gene editing of human hematopoietic stem and progenitor cells using envelope-engineered virus-like particles</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 10:30:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Engineered virus-like particles (VLPs) are a promising technology for in vivo gene editing of human hematopoietic stem and progenitor cells (HSPCs). Here we design and test two different VLP envelopes for human HSPC editing in vitro and in vivo. The first is an optimized version of the baboon envelope BaEVTR, which efficiently transduces human HSPCs in vitro. We show that the optimized BaEVTR VLP enables in vivo editing of β2 microglobulin in long-term human HSPCs (31% at 8 weeks after dosing) and editing of two hemoglobinopathy-relevant loci, BCL11A and HBG1/2 (26% and 7.5%, respectively, at 5 days after dosing), inducing fetal hemoglobin. Our second VLP design uses a CD133-targeted envelope designed to reduce the transduction of mature blood cells and achieves higher in vivo specificity for HSPCs compared to the optimized BaEVTR VLP. As avoiding delivery in filter organs such as the liver would enhance efficiency and safety, we also demonstrate that both VLPs avoid human hepatocytes in a humanized liver model. Hematopoietic stem and progenitor cells (HSPCs) can sustain lifelong production of a diverse range of blood cell types through a hierarchically organized continuum of cell states1,2. Over the past two decades, ex vivo HSPC gene therapy (GT) has become an established treatment for various hematological and metabolic diseases3,4,5,6,7,8,9,10 but faces challenges related to the high cost and complexity of autologous gene-modified cell manufacturing, as well as safety concerns regarding conditioning regimens before transplantation9,11,12,13,14,15,16,17,18. A systemic injection of a vector delivering in vivo HSPC GT directly to the natural cellular niche would overcome many limitations by eliminating the need for complex cell manufacturing, prolonged hospitalization and bone marrow (BM) conditioning. An ideal in vivo delivery system should be capable of efficiently reaching long-term multipotent HSPCs in their natural BM while avoiding off-target cells in main filter organs such as the liver. A recent study described the delivery of genetic payloads in vivo into murine hematopoietic stem cells (HSCs) by a lipid nanoparticle (LNP) encapsulating an mRNA targeted to CD117, a stem cell factor receptor expressed on murine HSPCs19. However, the in vivo studies were limited to mRNA delivery to induce Cre-lox recombination in reporter mice and underscored that current retargeted LNP delivery systems20 retain substantial biodistribution in off-target filter organs such as the spleen and the liver. LNPs are also currently limited in their capacity to incorporate complex gene-editing protein cargos. In contrast, virus-like particles (VLPs), an envelope-based delivery system lacking a viral genome, can package and deliver complex gene-editing payloads through linkage to the viral capsid protein (gag)22,23,24. These particles offer specific advantages over other delivery systems. First, VLPs can accommodate large ribonucleoproteins (RNPs), which, because of their shorter half-life compared to mRNA-encoded editors, can potentially reduce the chances of off-target editing in vivo23. Additionally, delivering RNP instead of mRNA makes editing independent of the transcriptional activation state of target cells, which is especially important when targeting quiescent long-term multipotent HSPCs25. Retargeting of VLPs can be achieved through the engineering of envelope glycoproteins, as achieved for lentiviral vector (LV) constructs26,27,28. To our knowledge, VLPs envelopes have not been engineered for efficient in vivo gene editing of human HSPCs. In this study, we tested two VLP designs. We first chose the modified Baboon endogenous viral envelope glycoprotein (BaEVTR) because it has a 20-fold higher propensity than the most commonly used vesicular stomatitis virus G protein (VSV-G) to transduce human HSPCs in vitro under conditions of low cytokine activation24,29,30,31,32. BaEVTR is resistant to human complement inactivation and there is no prior exposure to the Baboon endogenous virus in the general human population, which reduces the chances of strong immunogenicity upon the first systemic administration31. We implemented several modifications to BaEVTR. Further engineering the protease site of the BaEVTR cytoplasmic domain could enable a more efficient removal of the R peptide sequence that normally inhibits virus-to-cell fusion31. Because RetroNectin enhances viral-mediated gene transfer to target cells expressing the integrin receptors VLA-4/5 such as human HSPCs33, designing moieties that bind the same integrins could further enhance BaEVTR affinity for human HSPCs. However, because BaEVTR binds to the widely expressed ASCT1/2 solute carriers, it also transduces mature peripheral blood (PB) cells. To increase specificity to HSPCs by redirecting VLP tropism to a receptor of interest26,34,35, we leveraged Nipah virus (NiV) envelopes because of the modular structure of their fusogen complex36. Engineering of the NiV receptor binding complex to recognize specific cell-surface molecules has been used to target LVs to HSPCs35 but comprehensive in vitro and in vivo validation of this retargeting system is still lacking. We, therefore, equipped VLPs with a newly designed NiV fusion complex, retargeted with custom-engineered binders to the CD133 molecule, and tested this envelope extensively in vitro and in vivo, demonstrating enhanced specificity for human HSPCs compared to BaEVTR. Lastly, we showed that both engineered envelopes avoid human hepatocytes upon systemic administration in vivo in a humanized liver mouse model, a feature that would be beneficial in clinical application. For our initial tests, we leveraged LV constructs carrying the green fluorescent protein (GFP) reporter gene under the control of a spleen focus-forming virus (SFFV) promoter and pseudotyped with two envelope proteins commonly used to transduce ex vivo human HSPCs: VSV-G and BaEVTR (Fig. To best simulate in vivo delivery scenarios, we used an in vitro protocol designed to test transduction of human HSPCs without cytokine stimulation (Supplementary Information and Extended Data Fig. We performed transductions with varying dilutions of VSV-G LV or BaEVTR LV co-delivered with the Vectofusin-1 (VF1) adjuvant37 for 24 h. In line with previous findings in low cytokine stimulation conditions37, the BaEVTR LV demonstrated significantly higher potency on transducing resting CD34+ cells as compared to the VSV-G LV (Fig. To show that BaEVTR can access all HSPC subtypes in resting conditions, we sorted five distinct progenitors using fluorescence-activated cell sorting (FACS) (Supplementary Information Fig. 1) and exposed each HSPC subpopulation to BaEVTR LV using the same transduction conditions used for the bulk CD34+ population. 2b, we achieved equivalent levels of transduction in all progenitor subtypes at multiplicities of infection (MOIs) 32 and 158, respectively, including HSCs and multipotent progenitors (MPPs) (average: 18% and 61%). This result demonstrates the potential of BaEVTR to access a population of cells endowed with long-term multilineage potential in their resting state. a, Schematic representation of the VSV-G and BaEVTR LV vectors. b, Percentage of GFP+ cells measured in CD34+ cells upon BaEVTR or VSV-G LV transduction without CC110 prestimulation (resting condition) at different vector MOIs. Each curve shows the results of a different vector batch. c, Average percentage of GFP+ cells measured upon transduction of FACS-sorted HSC and MPP versus the remaining committed progenitors in resting condition at MOI 32 or MOI 158 (Extended Data Fig. d, Schematic representation depicting the introduction of an HIV-derived protease cleavage site (MACA) between the cytoplasmic (CyT) and regulatory (R) domains of the BaEVTR. e, Schematic representation of the RetroNectin CoD. f, Representative titration curves in resting CD34+ cells using a BaEVTR LV (gray dots), BaEVTR LV with protease modification (MACA) (red dots) or BaEVTR LV MACA with additional CoD modification (yellow-filled dots). g, Schematic representation of the BaEVTR LV vector constructs with RetroNectin CoD. h, Overview of an in vivo study for testing the CoD construct. LV pseudotyped with BaEVTR-CoD or BaEVTR alone was infused in NBSGW mice at days 7–9 after humanization (3.04 × 108 TU per mouse per dose for both LVs). AMD3100 was infused 2 h before each vector dosing. i, Percentage of GFP+ cells on BM huCD45+ cells at week 16 after dosing of a BaEVTR LV without CoD. j, Percentage of GFP+ cells on BM HSPCs measured over time after dosing of a BaEVTR LV without CoD reported as the mean ± s.d. k, Percentage of GFP+ cells on P and M populations (left) or LP and L populations (right) measured over time in the BM after dosing of a BaEVTR LV without CoD (mean ± s.d.). m, Percentage of GFP+ cells on splenic HSPCs at week 16 after dosing of a BaEVTR LV without CoD. Numbers on curves or bars show the average values measured for given experimental conditions or time points. Data are presented as the mean values ± s.d. For bars, statistical significance was measured using a one-way ANOVA with Tukey's multiple-comparison test and the exact P value is reported; for curves, statistical significance was measured using a two-tailed unpaired t-test with Welch's correction and reported using asterisks (*P < 0.05). Symbols represent biological replicates unless otherwise stated. We next sought to develop BaEVTR envelope variants with increased in vivo potency and affinity for human HSPCs. Firstly, we swapped the cleavage site used by the murine leukemia virus (MLV) protease to cleave the cytoplasmic tail of the BaEVTR molecule with a MA/CA (MACA) cleavage site, which is recognized by a human immunodeficiency virus (HIV) protease38 (Fig. Furthermore, we added a custom-designed surface molecule encompassing the VLA-4-binding and VLA-5-binding domains from the C-domain and CS-1 site, respectively, of the RetroNectin recombinant fibronectin fragment39,40 (Fig. We tested this construct, called codisplay (CoD), in LV vectors. 2e, we could measure increasingly higher titers in vitro in resting CD34+ cells, adding the MACA cleavage site first and then the CoD construct to our BaEVTR LV. Overall, the resulting BaEVTR-CoD LV (defined from this point on as including both MACA and CoD modifications; Fig. 1g) showed higher on-target potency in vitro versus the standard BaEVTR LV construct. We next aimed to validate the potency these envelopes in vivo using NOD.Cg-Kit−41JTyr+PrkdcscidIl2rgtm1Wjl/ThomJ (NBSGW) mice41 humanized with mobilized PB CD34+ cells. 2) human HSPCs reside almost exclusively in the BM and are not yet heavily skewed toward lymphoid progenitors (LPs). In this scenario, we observed that administering AMD3100 before particle dosing did not induce HSPC egress to the PB but enhanced transduction of HSPCs within the BM niche (Supplementary Text, Extended Data Fig. Addressing the volumetric limitations of intravenous (IV) dosing in mice, we tested a dosing schedule over three consecutive days achieving 7.5% transduction of BM human CD45+ (huCD45+) cells with a BaEVTR LV 5 days after the last dose (Supplementary Text and Extended Data Fig. To compare the performance of our vector designs in vivo, we then administered three consecutive doses of a BaEVTR LV or BaEVTR-CoD LV in short-term humanized NBSGW mice (Fig. 1h) and followed up treated animals for 16 weeks. To monitor BM engraftment dynamics over the course of the study, we characterized the changes in relative contribution of four populations enriched in primitive (P, CD34+CD13/CD33+), myeloid (M, CD34−CD13/CD33+), LP (CD34−CD13/CD33−) and L (CD34−CD13/CD33−) cells (Supplementary Text and Supplementary Fig. Confirming the in vitro results, we detected a significantly higher percentage of GFP+ cells at week 16 in the BM huCD45+ cell compartment of mice treated with the BaEVTR-CoD LV versus the standard BaEVTR LV (Fig. Over time, we observed a higher percentage of transduced HSPCs in the BaEVTR-CoD LV group (Fig. Similar dynamics were detected measuring the percentage of GFP+ cells in the four BM fractions described above, particularly concerning the LP and L cell populations (Fig. 1l), reflecting the output of gene-modified long-term multipotent stem cells. These transduction levels were also confirmed molecularly by vector copy number (VCN) evaluation (Extended Data Fig. A similar percentage of GFP+ cells was observed in the spleen-resident huCD45+ cell progeny again showing superior long-term transduction using the BaEVTR-CoD LV over the BaEVTR LV (Fig. Histology performed on sterna collected from treated animals at different time points after infusion showed colocalization of human nuclei and GFP signals both in proximity of sinusoidal vessels and in the endosteum, confirming the stable transduction by both vectors not only of committed progenitors but also of long-term HSCs, which are more likely to reside in the endosteal niche (Supplementary Fig. We chose to target the B2M locus because it provides a surrogate flow cytometry (FC) readout for gene editing, as exemplified in Fig. 1a), the efficiency was 75% B2M− cells as measured by FC, corresponding to up to 93% of edited alleles, as measured by high-throughput sequencing (HTS) of the B2M locus (Fig. We next isolated five HSPC fractions and exposed them individually in resting conditions to the BaEVTR VLP. These results underscored that the FC assessment of B2M− cells tends to underestimate the editing efficiency of our VLP, particularly in the more quiescent HSPC subtypes such as HSCs and MPPs. When measuring the relative proportion of editing variants in each HSPC subtype, we could observe that the predominant fraction was constituted by deletions. Notably, the granulocyte/macrophage progenitor fraction hosted a substantially higher number of insertion events as compared to the other subpopulations, suggesting that more committed progenitors may be biased toward resolving edits by insertion (Extended Data Fig. Once dosed in vivo in a pilot short-term study, our BaEVTR VLP achieved 20% and 47% B2M− cells (as measured by FC) corresponding to 47% and 71% B2M editing, as measured by HTS, 5 days after one dose and three doses, respectively (Fig. 2e, Supplementary Text and Extended Data Fig. Repeating the single-dose arm of this study in an NBSGW mouse at 9 weeks after humanization, where BM is fully humanized (~85% huCD45+), yielded comparable levels of B2M− HSPCs (Supplementary Text and Extended Data Fig. b, Representative FACS plot showing the percentage of B2M− cells induced through disruption of the B2M locus in vitro in resting huCD34+ cells using a BaEVTR VLP at MOI 16 versus the untransduced control (UT). c, Titration curves of two BaEVTR VLP batches upon transduction of CD34+ cells at different vector dilutions as the percentage of B2M− cells measured by FC or percentage of B2M edited alleles measured by HTS. d, Editing efficiency measured by FC or HTS in the resting HSPC bulk population and in FACS-sorted resting HSPC subtypes using a BaEVTR VLP at MOI 16. e, Comparison of B2M− cells versus B2M edited alleles measured by FC versus HTS, respectively, in BM huCD45+ cells days after VLP dosing in vivo (Extended Data Fig. Statistical analyses were performed comparing the editing efficiency of VLP versus VLP-CoD using FC or HTS readouts. f, Overview of an in vivo delivery study where NBSGW mice were humanized by infusion of CD34+ cells and then treated on days 7–9 with three IV doses of BaEVTR VLP. AMD3100 was infused 2 h before each vector dosing. g, Percentage of B2M− cells on huCD45+ cells measured by FC at week 16 after BaEVTR VLP dosing. h, Percentage of B2M− cells on BM HSPCs measured by FC over time after BaEVTR VLP dosing. i, Percentage of B2M− cells on BM HSPC subtypes measured by FC at week 16 after BaEVTR VLP dosing. j, Percentage of B2M− cells on P, M, LP and L populations measured over time by FC in the BM after BaEVTR VLP dosing. l, Percentage of B2M− cells on splenic HSPCs measured by FC at week 16 after BaEVTR VLP dosing. m, Comparison of B2M− cells versus B2M edited alleles measured over time by FC versus HTS, respectively, in BM huCD45+ cells after BaEVTR VLP dosing. n, Comparison of B2M− cells versus B2M edited alleles measured over time by FC versus HTS, respectively, in BM fractions after BaEVTR VLP dosing. o, Comparison of B2M− cells versus B2M edited alleles in BM huCD45+ cells measured by FC versus HTS, respectively, at week 16 after BaEVTR VLP dosing. Statistical significance was measured using a two-tailed unpaired t-test with Welch's correction. p, Relative contribution of editing variants of the B2M locus measured in PB huCD45+ cells over time by HTS after BaEVTR VLP dosing. Numbers on curves or bars show the average values measured for given experimental conditions or time points. Data are presented as the mean values ± s.d. Symbols represent biological replicates unless otherwise stated. Next, we conducted a comprehensive long-term study to assess permanent editing of long-term human HSPCs, with a terminal readout at week 16 (Fig. Firstly, we could confirm that 6.4% of edited huCD45+ cells could still be detected by FC readout in the BM of the treated animals 16 weeks after dosing (Fig. Over time, we noticed a substantially higher initial fraction of B2M− cells (48%), dropping from week 4 to week 8, which may have resulted from the exhaustion of edited short-living progenitors. These data were reflected by similar editing levels at week 16 in several HSPC subtypes including the HSC and MPP population (Fig. Examining the editing levels in the four main human BM cell fractions over time, the largest drop in B2M− cells between week 4 and week 8 was indeed observed in the L and LP subpopulations (Fig. 2j), representing the most active progenitors during this phase of human cell engraftment in the mouse BM. We observed comparable dynamics in the mature human PB progeny over time with final readouts at week 16 showing an average of 3.5%, 2.5% and 3.9% B2M− cells in the huCD45+, huCD13+CD33+ and huCD19+ fractions, respectively (Fig. The spleen displayed a similar level of HSPC edited cells at week 16 as compared to the BM (Fig. Because we showed that the FC readout tends to underestimate the level of B2M editing, we next performed a comprehensive analysis of molecular editing at all time points by HTS. Comparable results were observed from HTS conducted on the four BM fractions reflecting the dynamics measured by FC but highlighting higher levels of editing (Fig. The discrepancy between FC and HTS data suggested that the most differentiated cells could carry monoallelic edits or editing variants whose effect on B2M expression is not easily identifiable by FC. Looking at the selection of editing variants in PB cells over time, we noticed that, while the percentage of deletions remained relatively stable over time (21% at week 4 versus 17% at week 16), the relative fraction of insertions dropped from 43% at week 4 to 7.4% at the end of the study (Fig. We observed a similar negative selection for insertions events over time across all BM fractions (Supplementary Fig. Overall, these data showed that the BaEVTR VLP achieved high editing in resting human HSPCs in vitro and in long-term BM HSPCs in vivo, which was maintained for at least 16 weeks after vector dosing. Next, we designed and produced a BaEVTR-CoD VLP (Fig. 3a) and tested it in vitro on resting huCD34+ cells. 6a,b), similarly to what was shown for the LV constructs. For an 8-week in vivo study in mice, we administered three consecutive IV doses of a BaEVTR VLP carrying the MACA cleavage site, with or without CoD, on days 7–9 after humanization to overcome single-bolus dosing volume limitations (Extended Data Fig. In line with the results obtained in vitro, we detected a significantly higher percentage of B2M− cells, measured by FC on BM huCD45+ cells at week 8, in the group treated with the BaEVTR-CoD VLP versus the BaEVTR VLP design (Fig. A similar trend was observed when looking at the B2M− cell contribution in the BM HSPC compartment (Fig. 3d), over time in all main PB populations (Fig. We could confirm the superior in vivo performance of our BaEVTR-CoD VLP construct when measuring the levels of molecular edits by HTS in the PB over time (Fig. 3g) and in the BM at takedown (Fig. b, Percentage of B2M− cells on huCD45+ cells measured by FC at week 8 after dosing of a BaEVTR VLP with or without CoD (1.1 × 108 EU per mouse per dose and 1.8 × 107 EU per mouse per dose, respectively) (in vivo experimental design in Extended Data Fig. c, Percentage of B2M− cells on BM HSPCs measured by FC at week 8 after dosing of a BaEVTR VLP without CoD. d, Percentage of B2M− cells on BM P, M, LP and L populations measured by FC at week 8 after dosing of a BaEVTR VLP without CoD. e, Percentage of B2M− cells on PB huCD45+ cells (left), M cells (middle) or B L cells (right) measured by FC over time after dosing of a BaEVTR VLP without CoD reported as the mean ± s.d. f, Percentage of B2M− cells on splenic HSPCs measured by FC at week 8 after dosing of a BaEVTR VLP without CoD. g, Comparison of B2M− cells versus B2M edited alleles measured over time by FC versus HTS, respectively, in BM huCD45+ cells after VLP (left) or VLP-CoD (right) dosing (mean ± s.d.). h, Comparison of B2M− cells versus B2M edited alleles in BM huCD45+ cells measured by FC versus HTS, respectively, at week 8 after VLP or VLP-CoD dosing (VLP versus VLP-CoD, P = 0.0029). i, B2M edited alleles measured by FC versus HTS in BM fractions at week 8 after VLP or VLP-CoD dosing. j, Schematic representation of BaEVTR-CoD VLPs designed to carry an ABE and BCL11A gRNA or a Cas9 editor and HBG1/2 gRNA. k, Schematic representation of the gRNA designs and the relative targeted human loci (more details in Supplementary Table 1). E-box, enhancer box; HPFH, hereditary persistence of HbF. l, Percentage of BCL11A or HBG1/2 edited alleles measured on day 12 by HTS in BM huCD45+ cells after VLP-CoD dosing. n, Representative histogram plot showing HbF expression in CFU generated from CD34+ cells isolated ex vivo from mice on day 12 after VLP-CoD dosing. FMO Ctrl, fluorescence minus one control. o, Bar plots showing the percentage of HbF+ cells and HbF MFI in each CFU plate (two technical replicates per sample) generated from seeding BM CD34+ cells isolated ex vivo on day 12 from each mouse. Numbers on curves or bars show the average values measured for given experimental conditions or time points. Data are presented as the mean values ± s.d. For bars, statistical significance was measured using a one-way ANOVA with Tukey's multiple-comparison test and the exact P value is reported; for curves, statistical significance was measured using a two-tailed unpaired t-test with Welch's correction and reported using asterisks (*P < 0.05 and **P < 0.01). Symbols represent biological replicates unless otherwise stated. Next, we studied editing at loci of therapeutic relevance for human hemoglobinopathies (Fig. One BaEVTR-CoD VLP was designed to carry a cargo composed by an ABE8e adenine base editor (ABE)42 fused to the MLV gag protein and gRNA for editing the GATA1-binding site of the human BCL11A gene. An overview of guide targets and designs is provided in Fig. Upon in vitro titration in resting CD34+ cells, we confirmed by HTS the ability of these two cargos to either induce an A>T to G>C conversion in the GATA-binding site of the BCL11A locus (average titer of 4.4 × 105 with 82% editing at top dose) or to disrupt the BCL11A-binding site of the HGB1/2 locus (average titer of 4.9 × 104 with 26% editing at top dose), although the efficiency of editing was lower than that achieved with an identical construct targeting the B2M locus. We next tested these two VLPs in short-term humanized NBSGW mice and measured by HTS the editing efficiencies in the human BM fraction at the two loci 5 days after vector dosing (Extended Data Fig. We obtained, in BM huCD45+ cells, 26% A>G base conversion in the BCL11A target site and 7.5% editing of the HBG1/2 locus (Fig. When looking at the editing variants obtained with the ABE cargo, we observed 59% conversion of A>G at both target nucleotides and 40% conversion of the A nucleotide immediately downstream of the GATA-binding site (Fig. Lastly, we assessed effects of the edits on inducing fetal hemoglobin (HbF) upon erythroid differentiation of the gene-edited HSPCs. We isolated CD34+ cells from the BM of the mice dosed in vivo with our VLPs and performed a colony assay in a semisolid medium containing cytokines promoting erythroid differentiation. We observed similar proportions of granulocyte/macrophage (GM) colony-forming units (CFU) and erythroid cell burst-forming units (BFU) in all treated animals as compared to the untreated group (Extended Data Fig. However, when looking at the HbF expression in our colonies by FC, we detected a higher percentage of HbF+ cells and a higher HbF mean fluorescence intensity (MFI) in the mice dosed with our VLPs, as compared to the untreated animals (used as controls for the expected background HbF induction resulting from this in vitro assay) (Fig. Overall, these data demonstrate that our BaEVTR-CoD VLP is effective with different editors and gRNAs and in loci of therapeutic relevance. To restrict tropism to HSPCs by targeting HSPC-specific receptors, we also explored the potential of engineering the binder component of the NiV fusogen (Fig. 4a) to recognize the CD133 molecule expressed on human HSPCs. We chose CD133 as target because this receptor is absent from the surface of mature blood cell lineages43 while it is expressed on HSPCs at higher density than CD117 (Extended Data Fig. a, Schematic representation of the NiV fusogenic complex (left) and NiV-CD133-retargeted LV with RetroNectin CoD (right). b, Percentage of GFP+ cells measured in CD34+ cells upon BaEVTR, VSV-G, NiV-CD133 or NiV-CD133-CoD LV transduction without CC110 prestimulation (resting condition) at different vector MOIs. Each curve represents a different vector batch. c, Titration curves of NiV-CD133 LV without CoD on CD34+ cells or CD3+ T cells in resting (R) or stimulated (S) conditions. Right, histograms and relative numeric labels show the expression of CD133 and percentage of CD133+ cells in each population at the time of transduction as measured by FC. d, Heat map showing, on a color gradient from white to red, the titers obtained by transducing different human primary cells (rows) using different LVs (columns) dose-matched on the basis of TU per resting CD34+ cells (Supplementary Figs. BLOD, titer below limit of detection. e, Overview of an in vivo delivery study where BaEVTR or NiV-CD133-CoD LVs were dosed immediately after infusion of either huCD34+ cells or a mixture of huCD34+ cells and PBMCs from the same donor. f, Percentage of M (CD13+CD33+), B (CD19+) or T (CD3+) L cells on PB huCD45+ on day 7 or day 12 after infusion of CD34+ cells alone or CD34+ PBMCs. g, Percentage of GFP+ cells on BM HSPCs 12 days after BaEVTR and NiV-CD133-CoD LV vector dosing upon infusion of CD34+ cells alone or CD34+ PBMCs. h, Percentage of GFP+ cells on BM CD133+ measured 12 days after BaEVTR and NiV-CD133-CoD LV vector dosing upon infusion of CD34+ cells alone or CD34+ PBMCs. i, Stacked bars showing the relative contribution of huCD133+ cells on huBM or GFP+ cells in all mice after BaEVTR or NiV-CD133-CoD LV dosing in the CD34+ PBMC condition (NiV-CD133-COD versus BaEVTR on GFP+, P = 0.0049) (results relative to the CD34+ only condition in Extended Data Fig. Numbers on curves or bars show the average values measured for given experimental conditions or time points. Data are presented as the mean values ± s.d. For bars, statistical significance was measured using a one-way ANOVA with Tukey's multiple-comparison test and the exact P value is reported. Symbols represent biological replicates unless otherwise stated. We tested the system first with LV vectors. We generated two CD133-targeted LV constructs (NiV-CD133) carrying the engineered fusogen alone or in combination with a CoD (Fig. In human HSPCs in vitro in the same conditions used above for the BaEVTR LV, we found that NiV-CD133 LV and BaEVTR LV showed similar titers on resting CD34+ cells (Fig. Comparing CD34+ cells (expected to express the CD133 target molecule) and CD3+ T cells (which are instead CD133−) to the broadly tropic VSV-G LV and BaEVTR LV as controls, we found that NiV-CD133 vectors efficiently transduced CD34+ HSPCs in both stimulated and resting conditions while exhibiting negligible levels of T cell transduction even upon cytokine activation. These results were obtained using NiV-CD133 at a higher physical particle concentration compared to the VSV-G LV or the BaEVTR LV (Extended Data Fig. Testing NiV-CD133 LV against HSPCs or T cells in vitro at different dilutions, we confirmed that the specificity for CD133+ HSPCs was maintained at all doses, irrespective of the presence of the CoD (Fig. In vitro tests in immortalized cell lines confirmed the tropism of our NiV-CD133 construct for CD133+ cells (Supplementary Text and Supplementary Fig. We next expanded our specificity analysis to a broader set of human primary cells, including endothelial cells, epithelial cells and primary human hepatocytes (PHHs). All vectors were equivalently tolerated at all doses by the primary cell lines (Supplementary Fig. Across the remaining test panel, varying degrees of transduction were observed. The RNA-seq analysis underscores the correlation of these results with the expression levels of genes encoding known target receptors for VSV-G (LDLR) and BaEVTR (ASCT1/2), as well as CD133 and the CoD target molecules VLA-4 and VLA-5 (Supplementary Text, Supplementary Fig. Taken together, these data show the specific tropism of our NiV-CD133-CoD construct for human HSPCs. Both the density of CD133 and the frequency of CD133+ cells change rapidly during early human engraftment (Supplementary Text and Extended Data Fig. 8a,b), making this experimental setting suboptimal for testing CD133-targeted particles. Therefore, we dosed our NiV-CD133-CoD LVs immediately after the infusion of CD34+ cells when the surface expression of the CD133 marker in huCD34+ cells would still be at physiological levels (Fig. To test specificity for HSPCs over mature blood cells, we infused a group of mice with a mix of 5.0 × 105 CD34+ cells diluted in 1.0 × 107 huCD133− human PB mononuclear cells (PBMCs) from the same donor. This was immediately followed by dosing of BaEVTR LV or NiV-CD133-CoD LV at 3.3 × 108 transduction units (TU) per mouse and 7.8 × 107 TU per mouse, respectively. We then measured transduction efficiency on day 12, comparing the results to the group of mice infused only with huCD34+ cells. As expected, on day 7 and day 12 after infusion, the mice dosed with CD34+ PBMCs showed a substantially higher fraction of mature PB blood cells, including CD3+ T cells, acting as a potential off-target cell population compared to the mice dosed with CD34+ cells only (Fig. The BaEVTR LV showed higher transduction of total huCD45+ cells in the CD34+ PBMC-infused group (24%) as compared to the NiV-CD133-CoD LV-treated mice (19%), although this difference was not significant (Extended Data Fig. Conversely, in the same group, we observed a significantly higher transduction of HSPCs in animals dosed with the NiV-CD133-CoD LV (20%) versus the BaEVTR LV (14%) (Fig. Concordantly, we detected more GFP+ cells in the P and LP cell fractions of animals receiving the NiV-CD133-CoD LV as compared to BaEVTR LV, which instead transduced the more mature M and L cell populations with higher efficiency (Fig. Moreover, in the CD34+ PBMC-infused group, the NiV-CD133-CoD LV displayed significantly lower transduction of mature PB cells on day 12 compared to the BaEVTR LV (Fig. We further discuss the results obtained in this experimental setting and in a related experiment using short-term humanized mice in the Supplementary Text (Extended Data Fig. Overall, these data show the higher in vivo specificity of the NiV-CD133-CoD construct for human HSPCs compared to a broadly tropic vector. We next explored whether this engineered envelope could be efficiently incorporated in a VLP. We tested multiple versions of an NiV-CD133 VLP (Fig. 5a), combining this envelope design with the CoD construct and an MLV or HIV capsid. All NiV-CD133 VLPs were able to efficiently edit resting CD34+ in vitro, although their titers were substantially lower than what was observed with the BaEVTR VLP design (Fig. Therefore, we assessed the on-target potency of this construct in vivo. We used the same experimental conditions described above, dosing a BaEVTR VLP (at 1.8 × 107 editing units (EU) per mouse) or the NiV-CD133-CoD VLP (at 5.7 × 106 EU per mouse) immediately after CD34+ cell infusion (Fig. b, Comparison of titration curves in resting CD34+ cells using BaEVTR VLP and different iterations of a NiV-CD133 VLP without CoD encompassing an MLV-derived or HIV-derived capsid gag. d, Percentage of B2M− cells measured by FC on PB huCD45+ cells over time upon dosing a BaEVTR or a NiV-CD133 VLP reported as the mean ± s.d. f, Stacked bars showing the relative contribution of huCD133+ cells on human BM or GFP+ cells in all mice after BaEVTR or NiV-CD133-CoD VLP dosing. g, Comparison of B2M− cells versus B2M edited alleles in BM huCD45+ cells measured by FC versus HTS, respectively, at week 8 after BaEVTR VLP (left) or NiV-CD133-CoD VLP (right) dosing (mean ± s.d.). h, Comparison of B2M− cells versus B2M edited alleles in BM huCD45+ cells measured by FC versus HTS, respectively, at week 8 after BaEVTR VLP or NiV-CD133-CoD VLP dosing. i, Schematic diagrams of the vectors used (top) and the design of an in vivo experiment (bottom) based on IV dosing different LVs in FRG mice humanized with human hepatocytes. j, Heat map showing, on a color gradient from white to red, the percentage of GFP+ cells measured 12 days after LV dosing in each mouse or experimental group (rows) on human hepatocytes or NPCs (columns). Numbers on curves or bars show the average values measured at given experimental conditions or time points. Data are presented as the mean values ± s.d. For bars, statistical significance was measured using a one-way ANOVA with Tukey's multiple-comparison test and the exact P value is reported. Symbols represent biological replicates unless otherwise stated. Despite having dosed the NiV-CD133-CoD VLP at a threefold lower functional dose as compared to the BaEVTR VLP, we achieved similar levels of editing in PB huCD45+ cells in both groups over time up to week 8 after dosing (Fig. 5e, left), in the BM, we detected almost twofold higher editing with the BaEVTR VLP formulation (Fig. 5f), underscoring that the CD133-retargeted VLP displays a higher specificity for CD133+ cells as compared to BaEVTR. Overall, these data demonstrate that our NiV-CD133-CoD VLP enables sustained in vivo gene editing in human hematopoietic cells, reflecting efficient delivery of gene-editing cargos to long-term HSPCs, even at low doses, because of its specificity for CD133+ cells. Both BaEVTR-CoD and NiV-CD133-CoD LVs showed a potential to transduce PHHs in vitro (Fig. To investigate this aspect, we exploited the Fah−/−Rag2−/−Il2rg−/− (FRG) mouse model, which can host a liver humanized with PHH44,45. As a positive control, we dosed a group of animals with one single dose of a VSV-G LV (2.7 × 109 TU per mouse per dose), which is known to transduce human hepatocytes. These data were confirmed by liver histology for measuring colocalization of human nuclei and GFP signal (Extended Data Fig. Thus, both BaEVTR-CoD and NiV-CD133-CoD envelopes avoid human hepatocytes upon systemic administration. We developed and tested envelope-engineered particles that enable efficient, specific gene editing in HSPCs both in vitro and in vivo while avoiding human hepatocytes. To assess the potency and tropism of our engineered envelopes, we used LV vectors, which allowed for easy measurement of GFP expression using FC and histology, with the latter providing detailed insights into biodistribution in the BM (Supplementary Fig. 1 indicate that the CoD construct may enhance access to long-term HSCs, as evidenced by higher transduction at late time points in all compartments in the BaEVTR-CoD LV-treated group. The observation that AMD3100 boosts transduction even without relying on HSPC egress into the PB (Extended Data Fig. Potential mechanisms include enhanced HSPC proliferation, alterations in the BM microenvironment facilitating vector penetration, increased receptor expression on HSPCs or temporary disruption of the adhesion of HSPCs to their niche46,47. After validating BaEVTR constructs in vitro and in vivo with LV vectors, we incorporated the envelope moieties into VLPs carrying gene-editing cargo. To our knowledge, no published study has assessed baboon-derived envelopes for in vivo gene editing in human HSPCs. 3e show an almost twofold higher editing efficiency in the PB CD13+CD33+ population 1 week after vector dosing using the BaEVTR-CoD VLP versus the BaEVTR VLP (47% versus 25%, respectively). This suggests that CoD might have facilitated gene editing of M progenitors or multipotent stem cells during the early phases of BM engraftment. To improve specificity for human HSPCs, we also engineered NiV fusogens to recognize the CD133 molecule. When paired with the CoD modification and tested in vivo in the presence of PB mature cell sinks, the retargeted NiV-CD133-CoD LV displayed significantly higher specificity for HSPCs as compared to BaEVTR vectors. The NiV-CD133-CoD VLPs exhibited lower in vitro potency than BaEVTR VLP, suggesting that further optimization is needed. Upon systemic delivery in FRG mice hosting human hepatocytes, we observed minimal transduction of this key population by both engineered envelopes compared to the results obtained using a standard VSV-G formulation. To our knowledge, no previous data on the uptake of BaEVTR or Nipah particles by human hepatocytes in vivo have been published. By studying two different envelope configurations, we demonstrated the modularity of VLPs for in vivo HSPC GT, showing that the CoD modification enhances particle potency across different envelope architectures and that hepatocyte avoidance can be achieved with multiple envelope designs. Accumulation in the liver is a limitation of current LNP systems. Moreover, using information available in Li et al.48, we estimate that LNP doses targeting HSPCs are typically much higher than the particle doses used in our study (~2.0 × 1016–2.0 × 1017 LNPs versus ~3.0 × 1012 VLPs). Further studies are needed to compare VLPs and LNPs, given their differences in biodistribution, cellular uptake and gene delivery efficiency. Another important topic for future research is the immunogenicity of our modified vectors. Pseudotypes and other viral components have been assessed for compatibility with in vivo delivery on the basis of the expected human immune recognition of these particles. For instance, while measles virus and NiV share similar glycoprotein subunits, NiV components have minimal neutralizing antibodies in the human population, in contrast to measles35,49. Additionally, LV vectors have been dosed systemically in immunocompetent nonhuman primate studies without adverse events, although immunosuppressants were sometimes used50. We anticipate that the human immune system could respond to BaEVTR-derived or NiV-derived envelopes, potentially limiting on-target bioavailability and redosing possibilities. Both BaEVTR-derived and NiV-derived envelopes appear resistant to human complement inactivation31,35,51. Lastly, from a translational standpoint, our finding of stable gene editing in 27% of long-term HSCs achieved with VLPs may provide a basis for developing a gene-editing system for treating sickle cell disease52 and other hemoglobinopathies. In this regard, we demonstrated that VLPs are versatile and capable of carrying base editors and/or delivering gRNA for editing clinically relevant loci in human HSPCs. Amino acid sequences for VSV-G, BaEVTR31 and BaEVTR_MACA variant were codon-optimized, synthesized (ATUM or Twist) and cloned into pcDNA3.1 (VSV-G) or pTwist-CMV-β-globin (BaEVTR) plasmid backbones. Nipah G and F amino acid sequences were similarly codon-optimized and cloned into pCAGGS backbone. Blinded Nipah G protein (four point mutations in head) was engineered with binders through a flexible 3×G4S linker at the C terminus. G and F proteins also incorporated cytoplasmic tail modifications to enable pseudotyping35. Single-domain antibodies targeting huCD133 were obtained from the literature. Anti-CD133 scFv sequences were obtained from the literature and cloned into the blinded Nipah G protein C terminus (ATUM). The RetroNectin CoD sequence was determined by screening a library of constructs with combinatorial optimizations of the retronectin CS-1 and C-domains, transmembrane anchor domains and linkers between the different segments. These amino acid sequences were codon-optimized and cloned into pTwist-CMV-β-globin plasmid backbone. When Nipah G and F proteins were used in the context of retargeting to the huCD133 receptor, additional G cytoplasmic tail and linker modifications and F modifications were screened for improved on-target titers with this composition. Briefly, for the Nipah G modifications, cytoplasmic tails from related paramyxoviruses or from unrelated viruses that were identified to pseudotype retroviral vectors with high titers were synthesized and cloned into the G protein along with diversification of the linker between the G head and the binder (ATUM). Linker length and flexibility were varied according to sequences in the literature. The Nipah G variant library was screened for improvements in on-target versus off-target cell line titers. The top 20 variants were then screened in combination with the top Nipah F variants that were separately screened in the same manner. Lead G and F combinations were then made at larger scale and concentrated, followed by testing on primary CD34+ cells in vitro. HEK-293T Lenti-X cells were obtained from Takara (632180) and maintained in DMEM (Thermo Fisher Scientific, 10569-044) with 10% FBS (Thermo Fisher Scientific, 10437028). Suspension HEK-293T clones were generated and screened for highest titer production of BaEVTR LV vectors or MLV Gag–Cas9 VLP vectors. After adaptation to suspension culture and growth in serum-free medium, the top clones were tested for the production of LV vectors. The top ten clones with the highest p24 concentration in pg ml−1 and functional titer were then furthered screened for the highest functional titer of BaEVTR LV vectors or BaEVTR MLV VLP vectors. The top clone (SCC4180) was then further used for production and maintained in LV-MAX medium (Thermo Fisher Scientific, A4124004) when culturing. HEK-293T Lenti-X cells were engineered to overexpress huCD133 by standard LV transduction. Briefly, transfer plasmids were constructed to express full-length codon-optimized huCD133 (NP_006008.1) coding sequences, followed by an IRES–puromycin cassette. VSV-G-pseudotyped LV vectors were produced with transfer plasmid and HEK-293T Lenti-X cells were transduced at multiple vector doses. Then, 3 days after transduction, cells were treated with 1 μg ml−1 puromycin and assessed for huCD133 expression by FC after ten more days. Conditions with the appropriate expression levels were expanded and clonally sorted into 96-well plates. Clones that maintained a high and tight expression level (HEK CD133 overexpression), with minimal silencing, were chosen for future experiments. For LV vectors produced from adherent 293T Lenti-X cells, fusogen plasmids were combined with psPAX2 and a transgene vector containing enhanced GFP under the control of an SFFV promoter in the pSF backbone at a 0.2:0.3:0.5 ratio of fusogen, packaging and transgene, respectively. Plasmids were complexed with TransIT-293 transfection reagent (Mirus Bio, MIR 2706) and added to producer cells that were seeded the day prior, reaching 60–80% confluency at the time of transfection. The next day, a full medium exchange was performed. On the second day after transfection, the cell supernatant was collected and centrifuged at 1,000g for 10 min to pellet any producer cells. After passing through a 0.45-µm filter, crude material was aliquoted and frozen at −80 °C. For 100× or 300× concentrations, vectors were concentrated by 90-min ultracentrifugation over a sucrose cushion, followed by resuspension in formulation buffer. Concentrated LV vectors were aliquoted and frozen at −80 °C for later use. For LV vectors produced from suspension 293T clones, fusogen plasmids were combined with pALD-GagPol and pALD-Rev packaging plasmids (Aldevron), along with the pSF-SFFV-GFP transgene plasmid, at a 0.16:0.21:0.11:0.52 ratio of fusogen, Rev, GagPol and transgene, respectively. Plasmids were complexed with PEIpro transfection reagent (Polyplus/Sartorius) and added to producer cells that were seeded at 2.5 × 106 cells per ml 1–2 h before transfection. On the second day after transfection, cell supernatant was harvested as described above and crude or concentrated LV vectors were obtained. For VLPs produced from suspension 293T clones, fusogen plasmids were combined with MLV-based or HIV-based GagPol and Gag–Cas9 plasmids, along with the gRNA plasmid at a 0.475:0.36:0.115:0.05 ratio of gRNA, MLV GagPol, MLV Gag–Cas9 and BaEVTR, respectively, or 0.364:0.238:0.238:0.128:0.032 ratio of gRNA, HIV GagPol, HIV Gag–Cas9, Nipah F and Nipah G, respectively. The Gag–Cas9 fusion strategy was adapted from a previously described protocol22,24. For 100× or 300× concentrations of LV or VLP vectors, freshly harvested crude supernatant was concentrated by 90-min ultracentrifugation or overnight (20–24 h) centrifugation at 4,100g over a sucrose cushion, followed by resuspension in formulation buffer. Concentrated LV vectors were aliquoted and frozen at −80 °C for later use. All absolute particle calculations reported in the manuscript for LVs and VLPs were performed on the basis of the results of ELISA assays for p24 (QuickTiter lentivirus titer kit, lentivirus-associated HIV p24) and p30 (QuickTiter MuLV core antigen ELISA kit), respectively. To determine functional titer (TU per ml for LVs or EU per ml for VLPs), cell lines were seeded in 96-well tissue culture-treated plates (30,000 cells per well for all HEK-293T lines) and then crude or concentrated vectors were serial diluted onto the cells 2–4 h after cell seeding. In the case of NiV-CD133 constructs, titers were determined on the HEK CD133 overexpression cell line (generated as described above) because of the low permissiveness for transduction by CD133-targeted vectors of the parental HEK-293T cell line, which expresses low endogenous levels of huCD133. GFP and B2M were both analyzed by FC. Cells were harvested from the 96-well plates, washed and incubated with Draq7 viability dye to determine live cells before assessing GFP expression. For B2M readout, cells were blocked in cell stain buffer (Biolegend, 420201) and stained with anti-B2M antibody (1:100; Biolegend, 316306). After washing, cells were incubated with Draq7 and analyzed by FC. When LVs or VLPs were dosed on the basis of MOI, specific calculations were used. Cryopreserved granulocyte colony-stimulating factor (G-CSF) and Plerixafor-mobilized PB-derived CD34+ cells (Allcells) were thawed in StemSpan SFEM II medium (StemCell Technologies, 09065) supplemented with Glutamax (Thermo Fisher, 35050061) and penicillin–streptomycin (PS; Thermo Fisher, 15140122). Then, 100 μl of cells were plated in a round-bottom 96-well plate at 2 × 105 viable cells per mL. After overnight rest in a 37 °C incubator, 100 μl of diluted vector + vectofusin (Miltenyi Biotec, 130-111-163) was added at a final concentration of 12 μg ml−1. After 24 h of transduction, StemSpan SFEM II medium containing 2× CC110 cytokine cocktail (StemCell Technologies, 02697) was added. After 6–7 days in culture with cytokines, we assessed the degree of transduction by GFP expression on a BD Fortessa instrument. The CC110 cytokine cocktail was excluded from media in the experimental groups described in Fig. To assess the off-target risk, cells were seeded at 100 µl according to the number from Supplementary Table 1 in flat-bottom 96-well plates before transduction. The vectors were normalized to receive 1 TU per cell on the basis of their titers on CD34+ primary cells. Vectors were prediluted on culture medium and vectofusin (Milteny Biotec, 130-111-163) was added to them to a final concentration of 12 µg ml−1. Human PB pan-T (StemCell Technologies, 70024) and CD34+ cells were tested for transduction efficiency on their activated versus unactivated (resting) phase. The activated pan-T cells received interleukin 2 (IL-2) cytokine (StemCell Technologies, 78036.2) and CD3/CD28 activator beads (StemCell Technologies, 10991), whereas the activated CD34+ cells received a CC110 cytokine cocktail (StemCell Technologies, 02697) before transduction. A medium change was performed on day 4. The analysis of the transduction efficiency was performed by FC and VCN assay on day 7 after transduction. HEK-293 Lenti-X cell lines were cultured in DMEM + 10% FBS. Caco-2 cell lines (ATCC, HTB-37) were cultured in EMEM + 10% FBS. Hulec-5a cell lines (ATCC, CRL-3244) were cultured in MCDB131 medium, without L-glutamine, supplemented with 10 ng ml−1 epidermal growth factor, 1 μg ml−1 hydrocortisone, 10 mM glutamine and 10% FBS. For primary cell culture, activated and resting pan-T cells were cultured in Immunocult supplemented with 10 ng ml−1 IL-2 and PS. Human pulmonary alveolar epithelial cells (ScienCell, 3200) were cultured on lysine-coated plates in alveolar (AEpiCM) medium supplemented with FBS, endothelial cell growth supplement (ECGS) and PS. Human renal epithelial cells (ScienCell, 4120) were cultured on lysine-coated plates in EpiCM medium supplemented with FBS, ECGS and PS. Human lymphatic endothelial cells were cultured on fibronectin-coated plates in endothelial cell medium supplemented with FBS, ECGS and PS. Human aortic endothelial cells (ATCC, PCS-100-011), human pulmonary artery endothelial cells (ATCC, PCS-100-022) and human coronary artery endothelial cells (ATCC, PCS-100-020) were cultured in Vasculife medium + supplement. PHHs (Lonza, HUCPI) were cultured in hepatocyte basal medium + supplement. For in vivo studies, we used two FC panels. Immediately before running the samples on FC, cells were stained with 7-AAD (Biolegend 420404). For the lineage-negative panel, cells were washed in PBS and stained with live–dead fixable near-infrared dead cell stain kit for 633-nm or 635-nm excitation (Invitrogen, L10119). Subsequently, cells were incubated with Human TruStain FcX receptor block (Biolegend, 422302; 1:20 in BSA stain buffer (BD Biosciences, 554657)). For the detection of B2M− cells upon in vitro titration experiments, we used a B2M PE antibody (Biolegend, 316306). For in vivo readouts, we added to both panels a B2M FITC antibody (Biolegend, 316304). For FACS sorting of CD34+ cell subtypes, approximately 106 total CD34+ cells (StemCell Technologies, 70008) were stained in an antibody cocktail containing CD34 APC (BD Biosciences, 340441), lineage cocktail Pacific Blue (Biolegend, 348805), CD15 Pacific Blue (Biolegend, 980504), CD38 PE/Cy5 (Biolegend, 303508), CD90 PE/CF594 (BD Biosciences, 562385), CD45RA BV711 (BD Biosciences, 740806), CD7 AF700 (BD Biosciences, 561603), CD10 BV605 (Biolegend, 312222), CD135 PE (Biolegend, 313306) and DAPI (Thermo Fisher, D1306). Cells were sorted into various HSPC subpopulations using a BD Symphony S6 cell sorter. All husbandry and experimental procedures were approved by the Institutional Animal Care and Use Committee of the Charles River Accelerator and Development Lab. NBSGW mice (Jackson Labs, strain 026622) were used. Mouse age at the time of study start was 6–9 weeks. On the day of cell dosing, huCD34+ cells (double-mobilized by 100 μg ml−1 G-CSF and 5 mg kg−1 AMD3100) were thawed from cryovials stored in liquid nitrogen, washed with sterile PBS and resuspended at 300,000–500,000 cells in 200 μl of PBS. For the PBMC spike-in study, 500,000 CD34+ cells and 107 PBMCs from the same donor were mixed in 200 μl of PBS and dosed IV. On the day of LV or VLP dosing, LV or VLP was thawed and kept on ice. LV or VLP was premixed with vectofusin (Miltenyi Biotec, 130-111-163) at a final concentration of 12 µg ml−1 exactly 10 min before dosing in vivo. For PB, 75–100 µl of whole mouse blood was added to a V-bottom 96-well plate. Then, 75–100 µl of PBS (Gibco, 10010-023; 1:1 ratio with the blood) was added to the wells containing blood samples. Next, 200 µl of ACK lysis buffer (Gibco, A10492-01) was added to each well that contained a blood pellet. The pellets were mixed well in the ACK buffer by pipetting and incubated at room temperature for 3 min. Cells were spun again at 1,000g for 2 min and the supernatant was discarded. Next, 200 µl of 1× PBS (Gibco, 10010-023) was added to each well. Cells were spun again at 1,000g for 2 min and the supernatant was discarded. Then, 200 µl of ACK lysis buffer (Gibco, A10492-01) was added to each well a second time. The pellets were mixed well in the ACK buffer by pipetting and incubated at room temperature for 3 min. Cells were spun again at 1,000g for 2 min and the supernatant was discarded. Next, 200 µl of 1× PBS (Gibco, 10010-023) was added to each well. Cells were spun again at 1,000g for 2 min, the supernatant was discarded and cells were resuspended in 1× PBS (Gibco, 10010-023) and transferred over to a 96-well U-bottom plate for staining for FC. Bones were transferred into a sterile mortar containing 5 ml of ice-cold 1× PBS (Gibco, 10010-023) and smashed with a pestle. The crushed bones were filtered through a 40-µm nylon cell strainer to remove solid fragments, the volume of the filtrate was brought up to 10 ml of PBS (Gibco, 10010-023) and the filtrate was centrifuged at 300g for 10 min. Following centrifugation, the supernatant was removed, the cell pellet was resuspended in 10 ml of 1× PBS (Gibco, 10010-023) and filtered through a 40-µm nylon cell strainer. The filtrate was centrifuged for 5 min at 350g (4 °C). Following centrifugation, the supernatant was removed and cells were resuspended in 1× PBS for staining for FC. Spleens were isolated from mice and stored on ice in 1× PBS (Gibco, 10010-023). To start tissue processing, spleens were placed into 2 ml of ACK lysis buffer (Gibco, A10492-01). Spleens were dissociated by repeatedly crushing between two frosted sides of a glass microscope slide. The dissociated splenic tissue was incubated at room temperature in 2 ml of ACK lysis buffer (Gibco, A10492-01) for 10 min. The dissociated splenic tissues were then filtered through a 40-µm nylon cell strainer to remove solid fragments, the volume of the filtrate was brought up to 10 ml in PBS (Gibco, 10010-023) and the filtrate was centrifuged at 350g for 5 min. Following centrifugation, the supernatant was removed and the cell pellet was resuspended in 10 ml of 1× PBS (Gibco, 10010-023) and filtered through a 40-µm nylon cell strainer. Following centrifugation, the supernatant was removed and cells were resuspended in 1× PBS for staining for FC. The FRG humanized liver mouse model was used to assess vector potential to transduce human hepatocytes in vivo. Mice were maintained and the study was conducted by Yecuris. FRG mice have an FAH−/− mutation, which enables ablation of the murine hepatocytes, creating a niche for engraftment of donor hepatocytes. They also have Rag2−/−Il2rg−/− immunodeficiency mutations, which allow persistence and expansion of engrafted human hepatocytes. The NPCs of the liver (Kupffer cells and endothelial cells) remained of murine origin. After confirmation of >80% humanization by human albumin levels (>3.5 mg ml−1), mice were dosed with ~300× concentrated vector at a 5 ml kg−1 dose. Unless otherwise noted, mice were killed 14 days after injection and livers were isolated for tissue dissociation and FC analysis of the human and murine cell populations. For FC analysis, hepatocytes were stained using hASGR1 AF647 (R&D Systems, FAB43941R, clone 950203), mCD81 PE (Biolegend, 104906, clone Eat-2) and eBioscience eFluor780 viability dye (Thermo Fisher). NPCs were stained with F40/80 PE (Biolegend, 123110, clone BM8), CD31 APC (Biolegend, 102410, clone 390) and eBioscience eFluor780 viability dye (Thermo Fisher). Cells were washed with PBS once to get rid of the media. Suspension cells were centrifuged at 500g for 5 min and the supernatant was discarded before washing. DNA was extracted using QuickExtract DNA extraction solution (BioSearch Technologies, QE0905T) by incubating the cells with solution at 65 °C for 6 min, followed by 98 °C for 2 min. They were then diluted at least threefold with ultrapure water to avoid the inhibition of PCR because of the abundant presence of the extraction solution. A PCR master mix was prepared at a final volume of 20 µl containing ddPCR Supermix for Probes, no UTP (Bio-Rad, 186-3024), delU3 primer/probe mix (DelU3 forward primer, 5′-GGAAGGGCTAATTCACTCCC-3′; DelU3 reverse primer, 5′-GGTTTCCCTTTCGCTTTCAGG-3′; DelU3 probe, 5′-/56-FAM/TGCCCGTCTGTTGTGTGACTCTG/3IABxFQ/-3′) and ArX primer/probe mix (ArX forward primer, 5′-TATGTTCAGATGCCCATTAGGG-3′; ArX reverse primer, 5′-CTTGCTCAAAGGACTGTGATTTC-3′; ArX probe, 5HEX/AGTGCCTTT/ZEN/CAGATGGAAACGGGT/3IABkFQ/). A Bio-Rad QX200 AutoDG droplet generator was used to generate droplets. Genomic DNA was purified using QIAamp DNA micro kit (Qiagen, 56304) according to the manufacturer's manual. Purified genomic DNA was first amplified with primer pairs spanning around 200–300 bp at the target edited regions using Q5 hot start high-fidelity 2× master mix (Qiagen, M0494L) for 35 cycles (98 °C for 60 s, followed by 35 cycles of 98 °C for 10 s, 66 °C for 20 s and 72 °C for 30 s, with a final step of 72 °C for 2 min, before holding at 4 °C). Pooled indexed libraries were cleaned up by AMPure XP solid-phase reversible immobilization reagent (Beckman Coulter, A63881) and measured with a 4150 Tapstation system (Agilent) using D5000 DNA ScreenTape assays (5067-5588/89, Agilent) for size range and a Qubit 4 fluorometer (Thermo Fisher Scientific) with high-sensitivity dsDNA assays (Thermo Fisher Scientific, Q33231) for quantification before clustering on Illumina sequencers. Forward and reverse primers complementary to sequences upstream and downstream of the region of interest were designed with 5′–3′ overhang adaptors. A subsequent limited-cycle amplification step was performed to add multiplexing indices and Illumina sequencing adaptors. An amplicon of ~200 bp was amplified by the primer pair. Libraries were normalized, pooled and sequenced on an Illumina MiSeq or NextSeq depending on throughput required and availability: forward overhang, 5′ CTCTTTCCCTACACGACGCTCTTCCGATCT-[locus-specific seq]; reverse overhang, 5′ CTGGAGTTCAGACGTGTGCTCTTCCGATCT-[locus-specific seq] (additional details in Supplementary Information Table 2). The paired-end FASTQ files were trimmed for readthrough of the Illumina TruSeq adaptor and amplicon primers using Cutadapt version 2.10. FASTQ files were aligned to the human genome reference (GRCh38) using Minimap2 version 2.24. Similarly, the percentage of editing variants was calculated as (total reads carrying a given editing variant/total reads) × 100. Read counts for each of the motif variants (TTTATCG, TTTGTCA and TTTGTCG) were obtained for the four BCL11A ABE-edited samples. 8), samples were sent to GENEWIZ (Azenta Life Sciences) where total RNA was extracted from each sample, followed by library prep according to Illumina specifications and sequencing with 150-bp paired-end reads on an Illumina HiSeq platform. Genes with ≤2 counts in <3 samples were filtered from subsequent analyses. Differential expression analyses for pairwise comparisons were processed using DESeq2 version 1.40.2. Differently expressed genes were defined with |log2FC| > 2 and adjusted P value < 0.01. Multiple groups were compared using a one-way analysis of variance (ANOVA) and Bonferroni correction with α = 0.05. For the HEK-293 Lenti-X, Hulec-5a, HUVEC, Raji, SupT1 and human primary cells (Supplementary Fig. 9b), sample processing and sequencing for RNA-seq were completed by GENEWIZ (Azenta Life Sciences) with 150-bp paired-end reads on an Illumina HiSeq platform. A single representative sample was chosen from publicly available data for each of the Caco-2 and KG1A cell lines and is indicated in the legend information in Supplementary Table 3 (refs. The log2-transformed CPM normalized counts (edgeR version 3.42.4) were visualized in R version 4.3.0 with pheatmap version 1.0.12 for the immortalized and primary cell lines with the six genes of interest (LDLR, ASCT1, ASCT2, CD133, VLA-4 and VLA-5). No statistical analyses were conducted in this context. The insertion site analysis shown in Extended Data Fig. Briefly, the protocol was based on the fragmentation by sonication of genomic DNA isolated from CD34+ cells, followed by ligation of a linker cassette (LC) containing a known sequence and a unique molecular identifier (UMI) for abundance estimation. Each ligation product underwent two rounds of exponential PCR amplification using long terminal repeat-specific and LC-specific primers. An additional PCR step was carried to introduce into each sample Illumina adaptor sequences and a known sample index sequence for multiplexing. A pooled sample library was then loaded on a MiSeq System (Illumina) and the sequence analysis was performed using the IS-Seq computational pipeline55. For the bubble plot generation and the Shannon diversity index calculation reported in Extended Data Fig. 1e,f, we used UMI counts as the insertion site abundance quantification method. All immunohistochemistry was performed on the Leica Bond RX fully automated staining instrument running Bond 6.0 software. Antibodies were all diluted in SignalStain antibody diluent (Cell Signaling Technology, 8112). Dual chromogenic staining of human nucleoli (Abcam, 190710, clone NM95) at 1:4,000 dilution was performed using the bond polymer refine detection system (Leica Biosystems, DS9800) with EDTA antigen retrieval using bond epitope retrieval 2 (Leica Biosystems, AR9640) for 20 min at 90 °C. Staining of GFP (Cell Signaling Technology, 2956, clone D5.1) at 1:50 dilution was performed using the bond polymer refine red detection system (Leica Biosytems, DS9390) with EDTA antigen retrieval using bond epitope retrieval 2 (Leica Biosystems, AR9640) for 20 min at 90 °C. Counterstaining was performed using hematoxylin from the bond polymer refine red detection system (Leica Biosytems, DS9390). Slides were air-dried, submerged in xylene and cover-slipped using CV mountant (Leica Biosystems, 14070936261). Whole-slide imaging was performed on the Leica Versa 200 microscope at ×40 magnification. TIF files were extracted from the Leica SVS scan files using Leica Aperio Imagescope version 12.4.6.5003. Sternum and liver samples were fixed in 10% neutral buffered formalin (Epredia, 534801) for 48 h at room temperature. Following fixation, sternum samples were decalcified overnight using Formical-2000 decalcifier (StatLab, 1314). Both sternum and liver samples were processed on the Leica Pearl tissue processor through a graded ethanol series (Fisher, A962P4) as follows: 70%, 80%, 90%, 100%, 100% and 100%. This was followed by three xylene solvent steps (Fisher, X3P) and three paraffin steps (Paraplast, Leica, 39601006) using the default 12-h protocol. Samples were embedded using the Leica HistoCore Arcadia embedding center and sectioned at 5 µm on the Leica HistoCore Multicut microtome. CD34+ sorted human HSPCs harvested from mouse BM were seeded at 5 × 102 cells per tube in Methocult classic (StemCell Technologies, 04444, lot 1000138489). Two plates were seeded per animal from the same Methocult tube. Cells were incubated in StemSpan SFEM II medium with PS (Gibco). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. RNA-seq data are available from the Gene Expression Omnibus under accession number GSE267423. Source data are provided with this paper. Pellin, D. et al. A comprehensive single cell transcriptional landscape of human hematopoietic progenitors. Anjos-Afonso, F. & Bonnet, D. Human CD34+ hematopoietic stem cell hierarchy: how far are we with its delineation at the most primitive level? A. et al. Gene therapy in patients with transfusion-dependent β-thalassemia. Kanter, J. et al. Biologic and clinical efficacy of LentiGlobin for sickle cell disease. Biffi, A. et al. Lentiviral hematopoietic stem cell gene therapy benefits metachromatic leukodystrophy. Frangoul, H. et al. CRISPR–Cas9 gene editing for sickle cell disease and β-thalassemia. & Aiuti, A. Gene therapy using haematopoietic stem and progenitor cells. Kohn, D. B. Gene therapy for blood diseases. Al-Saif, A. M. Gene therapy of hematological disorders: current challenges. Tucci, F., Galimberti, S., Naldini, L., Valsecchi, M. G. & Aiuti, A. A systematic review and meta-analysis of gene therapy with hematopoietic stem and progenitor cells for monogenic disorders. Acute myeloid leukemia case after gene therapy for sickle cell disease. Hsieh, M. M. et al. Myelodysplastic syndrome unrelated to lentiviral vector in a patient treated with gene therapy for sickle cell disease. & DeBaun, M. R. Leukemia after gene therapy for sickle cell disease: insertional mutagenesis, busulfan, both, or neither. Kanter, J. et al. Lovo-cel gene therapy for sickle cell disease: treatment process evolution and outcomes in the initial groups of the HGB-206 study. Uchida, N. et al. Fertility-preserving myeloablative conditioning using single-dose CD117 antibody–drug conjugate in a rhesus gene therapy model. The role of conditioning in hematopoietic stem-cell gene therapy. Bearman, S. I. et al. Regimen-related toxicity in patients undergoing bone marrow transplantation. In vivo hematopoietic stem cell modification by mRNA delivery. In vivo delivery of plasmid DNA by lipid nanoparticles: the influence of ionizable cationic lipids on organ-selective gene expression. Kim, H. et al. Lipid nanoparticle-mediated mRNA delivery to CD34+ cells in rhesus monkeys. Engineered virus-like particles for efficient in vivo delivery of therapeutic proteins. Raguram, A., Banskota, S. & Liu, D. R. Therapeutic in vivo delivery of gene editing agents. Mangeot, P. E. et al. Genome editing in primary cells and in vivo using viral-derived Nanoblades loaded with Cas9–sgRNA ribonucleoproteins. Functions and regulatory mechanisms of resting hematopoietic stem cells: a promising targeted therapeutic strategy. Lév, C., Verhoeyen, E. & Cosset, F. L. Surface engineering of lentiviral vectors for gene transfer into gene therapy target cells. Sinn, P. L. et al. Gene transfer to respiratory epithelia with lentivirus pseudotyped with Jaagsiekte sheep retrovirus envelope glycoprotein. Envelope protein-specific B cell receptors direct lentiviral vector tropism in vivo. Levy, C. et al. Baboon envelope pseudotyped lentiviral vectors efficiently transduce human B cells and allow active factor IX B cell secretion in vivo in NOD/SCIDγc−/− mice. Bernadin, O. et al. Baboon envelope LVs efficiently transduced human adult, fetal, and progenitor T cells and corrected SCID-X1 T-cell deficiency. Girard-Gagnepain, A. et al. Baboon envelope pseudotyped LVs outperform VSV-G-LVs for gene transfer into early-cytokine-stimulated and resting HSCs. Efficient and robust NK-cell transduction with baboon envelope pseudotyped lentivector. Tsukamoto, T. & Okada, S. The use of RetroNectin in studies requiring in vitro HIV-1 infection of human hematopoietic stem/progenitor cells. Frank, A. M. & Buchholz, C. J. Surface-engineered lentiviral vectors for selective gene transfer into subtypes of lymphocytes. Bender, R. R. et al. Receptor-targeted Nipah virus glycoproteins improve cell-type selective gene delivery and reveal a preference for membrane-proximal cell attachment. Singh, R. K. et al. Nipah virus: epidemiology, pathology, immunobiology and advances in diagnosis, vaccine designing and control strategies—a comprehensive review. Effective retroviral transduction of primary T cells and hematopoietic stem cells using the soluble transduction enhancer: vectofusin-1. Improved GaLV-TR glycoproteins to pseudotype lentiviral vectors: impact of viral protease activity in the production of LV pseudotypes. Hanenberg, H. et al. Optimization of fibronectin-assisted retroviral gene transfer into human CD34+ hematopoietic cells. Lee, H. J. et al. Retronectin enhances lentivirus-mediated gene delivery into hematopoietic progenitor cells. McIntosh, B. E. et al. Nonirradiated NOD,B6.SCID Il2rγ−/−KitW41/W41 (NBSGW) mice support multilineage engraftment of human hematopoietic cells. Richter, M. F. et al. Phage-assisted evolution of an adenine base editor with improved Cas domain compatibility and activity. Yin, A. H. et al. AC133, a novel marker for human hematopoietic stem and progenitor cells. Selection and evaluation of clinically relevant AAV variants in a xenograft liver model. Hatit, M. Z. C. et al. Species-dependent in vivo mRNA delivery and cellular responses to nanoparticles. Samuelson, C. et al. AMD3100 redosing fails to repeatedly mobilize hematopoietic stem cells in the nonhuman primate and humanized mouse. In vivo HSC transduction in rhesus macaques with an HDAd5/31 vector targeting desmoglein 2 and transiently overexpressing cxcr4. Payload distribution and capacity of mRNA lipid nanoparticles. Palomares, K. et al. Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations in vitro and bypass the liver sink when administered in vivo. Milani, M. et al. Phagocytosis-shielded lentiviral vectors improve liver gene therapy in nonhuman primates. & Parks, G. D. A novel factor I activity in Nipah virus inhibits human complement pathways through cleavage of C3b. Fitzhugh, C. D. et al. At least 20% donor myeloid chimerism is necessary to reverse the sickle phenotype after allogeneic HSCT. Therapeutic targeting of metabolic vulnerabilities in cancers with MLL3/4-COMPASS epigenetic regulator mutations. Wang, H. et al. Disruption of dNTP homeostasis by ribonucleotide reductase hyperactivation overcomes AML differentiation blockade. Yan, A. et al. IS-Seq: a bioinformatics pipeline for integration sites analysis with comprehensive abundance quantification methods. We would like to acknowledge E. Verhoeyen, C. Dunbar and J. Tisdale for their scientific input and consultancy. We are also grateful for the support of the leadership team at Sana Biotechnology including chief executive officer S. Harr and former chief strategy officer D. Williams. Lastly, we are indebted to the many scientists, people managers, technical support and administrators at our locations in Cambridge, South San Francisco and Seattle for their great assistance and collaboration. These authors contributed equally: Vladimir V. Botchkarev Jr., Sean Harrington. Vladimir V. Botchkarev Jr., Sean Harrington, Matteo Stoppato, Alexander Justen, Casey Kimber, Anjali Kapuria, Keylie M. Gibson, Chi-Shuen Chu, Yuanxin Xu, Kelsey Haugh, Ramya Ankala, Nathan Kipniss, Andre DeGroot, Emerson Moore, Rowena de Jesus, Funmi Adewale, Kathy Daniels, Samantha Crocker, Anna Liang, Shannon Joyce, Nicole Roberto, Pamela Angel, Derek Smith, Athena Wong, Nick Adler, Valeria Berlfein, Shirisha Amatya, Patricia Cruite, Shariq Usmani, Albert Ruzo, Benjamin Ferland, Sundeep Chandra, Edward J. Rebar, Jagesh V. Shah, Kyle Trudeau & Luca Biasco University College of London (UCL) Faculty of Population Health Science, Infection Immunity and Inflammation Department, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and tested the gRNAs used in this study. were responsible for the design of FC readouts and the sorting of human HSPCs. provided in vivo support for the animal studies. and S.A. provided analytical support for the in vitro assays on cell lines and the CFU assay. supervised the design of cargo and envelope constructs. supervised the gene-editing and GT teams, respectively. All authors are or have been employees of Sana Biotechnology. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Experimental scheme of the in vitro transduction protocol with or without (w/o) CC110 pre-stimulation (CC110 = StemSpan™ CC110 cytokine cocktail). (b) in vitro cell counts (left panel) and viability (right panel) of CD34+ cells from two donors (D1 and D2) w/o CC110 pre-stimulation over the 8 days of culture. Statistical analysis performed at end of culture on + CC110 vs – CC110 pre-stimulation conditions (3 technical replicates per donor). (c) Highest transduction achieved with a VSV-G or BaEVTR LV. Each dot represents a different vector batch. (d) Highest MOI employed for each LVs. Each dot represents a different vector batch. (e) Bubble plots showing insertion sites contributing >0.01% of the insertion site pool upon transduction of CD34+ cells in resting or stimulated conditions with a BaEVTR LV at MOI 25. (f) Shannon Diversity Index of the insertional repertoire (white bars) and percent of GFP+ cells (colored bars) from the same transduction experiment of panel f. (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; for curves statistical significance was measured using two-tailed unpaired t-test with Welch's correction and reported as **P < 0.01; symbols represent biological replicates unless otherwise stated). (b) Transduction of individual HSPC subtypes in resting conditions with a BaEVTR LV at two different MOIs. (HSC = Hematopoietic Stem Cells, MPP = Multipotent Progenitors, CMP = Common Myeloid Progenitors, GMP = Granulocytes/Macrophage Progenitors, MEP = Megakaryo/Erythroid Progenitors) (c) Transduction of CD34+ cells in resting vs stimulated conditions measured at day 8 (D8) using a BaEVTR LV at MOI 25. (d) Average Vector Copy Number (VCN) measured in CD34+ cells in resting vs stimulated conditions at day 8 (D8) using a BaEVTR LV at MOI 25. (e) Titers obtained in resting CD34+ cells with a BaEVTR LV (grey dots), BaEVTR LV with the protease modification (MACA) (red dots), or the BaEVTR LV MACA with the additional co-display modification (CoD) (yellow-filled dots). (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; symbols represent biological replicates unless otherwise stated). (a) Overview of a pilot in vivo delivery study where NBSGW mice were humanized via infusion of CD34+ cells and then treated on D7 with a saline solution or AMD3100, followed by IV dosing of BaEVTR or VSV-G LVs. (b) Percent of GFP+ cells on BM huCD45+ cells, HSPCs, or hematopoietic stem cells (HSC) measured at D12 after LV dosing following saline or AMD3100 treatment (UT = untransduced control group). (c) Percent of huCD45+ cells measured in the BM, PB or spleen (SPL) at day of LVs dosing (AMD = AMD3100). (d) Overview of an in vivo delivery study where NBSGW mice were humanized via infusion of CD34+ cells and then treated at D(day)7 or D7-9 with 1 or 3 IV doses of BaEVTR LV respectively. AMD3100 was infused 2 hours prior to each vector dosing. (e) Percent of GFP+ cells on huCD45+ cells (left panel) or HSPC subtypes (right panel) at day 12-15 of humanization (5 days after last LV dose) following 1 or 3 BaEVTR LV dose(s). (f) Average Vector Copy Number (VCN) measured in the PB of the NBSGW mice receiving a standard BaEVTR LV or a BaEVTR-CoD LV dosed at day 7 of humanization (experiment described in Fig. (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; for bars statistical significance was measured using one-way ANOVA with Tukey's multiple comparison test and the exact p-value is reported; symbols represent biological replicates unless otherwise stated). (a) Editing efficiency measured by FC or HTS in the resting HSPC bulk population and in FACS-sorted resting HSPC subtypes using a BaEVTR VLP at MOI 83. (b) Stacked bar showing the relative contribution of editing variants in the resting HSPC bulk population and in FACS-sorted resting HSPC subtypes using a BaEVTR VLP at MOI 16 (left panel) or at MOI 83 (right panel) (SNV = single nucleotide variants). (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; symbols represent biological replicates unless otherwise stated). (a) Overview of an in vivo delivery study where NBSGW mice were humanized via infusion of CD34+ cells and then treated at D7 or D7-9 with 1 or 3 IV doses of BaEVTR VLP respectively. AMD3100 was infused 2 hours prior to each vector dosing (b) Percent of B2M- cells on huCD45+ cells measured by FC in the BM 5 days after last VLP dose. (c) Percent of B2M- cells on HSPC subtypes measured by FC in the BM 5 days after last VLP dose. (d) Stacked bar showing the relative contribution of editing variants measured in vivo at D12 after 1 or 3 doses of a BaEVTR VLP. (e) Overview of an in vivo delivery study where NBSGW mice were humanized via infusion of CD34+ cells and then treated at Week 9 with 3 doses of AMD3100 followed by a single infusion of BaEVTR VLP. (f) Percent of huCD45+ cells observed in the BM and PB at time of vector dosing (Week 9). (g) Percent of B2M- cells on huCD45+ cells and HSPCs measured by FC in the BM or PB of NBSGW mice at Wk11 after humanization (Week 2 after vector dosing). (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; for bars statistical significance was measured using one-way ANOVA with Tukey's multiple comparison test and the exact p-value is reported; symbols represent biological replicates unless otherwise stated). (a) Titers obtained in resting CD34+ cells with a BaEVTR VLP (grey squares), BaEVTR VLP with the protease modification (MACA) (red squares), or the BaEVTR VLP MACA with the additional co-display modification (CoD) (yellow-filled squares). (b) Representative titration curves in resting CD34+ cells using a BaEVTR VLP with the protease modification (MACA) (red squares) or the BaEVTR VLP MACA with the additional co-display modification (CoD) (yellow-filled squares). (c) Schematic diagram of an in vivo experiment based on IV dosing of a BaEVTR VLP or a BaEVTR-CoD VLP (both vectors carrying the MACA modification) over three consecutive days at day 7-9 post-humanization in mice treated with AMD3100 prior to each vector infusion. (d) Stacked bar showing the relative contribution of editing variants measured in BM huCD45+ cells at Week 8 after BaEVTR VLP or a BaEVTR-CoD VLP dosing. (e) Overview of an in vivo delivery study where NBSGW mice were humanized via infusion of CD34+ cells and, at D7-9 post-humanization, dosed IV with a BaEVTR-CoD VLP carrying the payloads described in Fig. AMD3100 was infused 2 hours prior to each vector dosing. (f) Percent of CFU-GM (Colony Forming Unit -Granulocytes Macrophages) or BFU-E (Burst Forming Units- erythroid cells) over the total Colonies Forming Units (CFU) generated from CD34+ cells isolated from the BM of treated mice from the experiment described in panel (e) and in Fig. (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; symbols represent biological replicates unless otherwise stated). (a) Histogram plots showing the expression of CD117 (top panels) or CD133 (bottom panels) in CD34+ vs CD34- human mobilized peripheral blood cells from two separate donors. (b) Transduction achieved in CD34+ or CD3+ cells in Resting (R) or Stimulated (S) conditions using a VSV-G, BaEVTR, or NiV-CD133 LV, each vector used at two different doses. The percentage of GFP+ cells is reported inside each gate, while the physical particles per well (each well containing 2.0E + 04 cells in 200ul) are shown at the top of each column. (a) Representative histogram plot (top panel) and summary curves (bottom panel) showing the changes in percent CD133 + cells and CD133 MFI (calculated within the CD133+ population) over the first 12 days of NBSGW humanization after CD34+ cells infusion. (b) Percent of phenotypically defined HSPCs (Lin-CD34+ cells) within the CD133+ vs CD133- population over the first 12 days of NBSGW humanization after CD34+ cells infusion. (c) Percent of huCD45+ cells measured in the PB over the course of the study after infusion of CD34+ cells alone or CD34 + PBMC (see experiment described in Fig. (d) Percent of GFP+ cells on BM huCD45+ cells measured 12 days after BaEVTR and NiV-CD133-CoD LV vector dosing upon infusion of CD34+ cells alone or CD34 + PBMC (see experiment described in Fig. (e) Percent of GFP+ cells on the main human BM cell fractions measured 12 days after BaEVTR and NiV-CD133-CoD LV vector dosing upon of CD34 + PBMC infusion (see experiment described in Fig. (f) Heatmap showing the differentially expressed genes among three differently treated human primary hepatocytes (PHH): freshly thawed after cryopreservation (CRYO), thawed, and cultured for 24 h at time of vector dosing for in vitro titration experiments (24 h), and isolated from FRG mice at time of vector dosing for in vivo experiments (FRG). Expression values are displayed as log2 normalized counts plotted on a color gradient from blue to white to red as per legend on the right. Unsupervised clustering based on this set of genes was performed among groups and displayed through the dendrogram at the top of the heatmap. Genes are reported with their relative aliases. Normalized expression values are plotted as log2 normalized counts on a color gradient from blue to white to red as per legend on the right. (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; for bars statistical significance was measured using one-way ANOVA with Tukey's multiple comparison test and the exact p-value is reported, n.s.= not significant; symbols represent biological replicates unless otherwise stated). (a) Stacked bars showing the relative contribution of huCD133+ cells on huBM or GFP+ cells in all mice after BaEVTR or NiV-CD133-CoD LV dosing in the CD34only condition relative to the in vivo experiment described in Fig. (***BaEVTR vs NiV-CD133-CoD LV P = 0.0049) (b) Overview of an in vivo delivery study where BaEVTR or NiV-CD133-CoD LVs were dosed at D7-9 (each day 2 h after AMD3100 administration) after infusion of either CD34+ cells or a mixture of CD34+ cells and PBMC from the same donor. (c) Stacked bars showing the relative contribution of CD133+ cells on huBM or GFP+ cells in all mice after BaEVTR or NiV-CD133-CoD LV dosing in the CD34 + PBMC condition relative to the in vivo experiment described in panel (b). (Notes: numbers on curves or bars show average values measured at given experimental conditions or timepoints; Data are presented as mean values +/- SD; for bars statistical significance was measured using one-way ANOVA with Tukey's multiple comparison test and the exact p-value is reported; symbols represent biological replicates unless otherwise stated). Dual Chromogenic staining of Human Nucleoli (brown signal) and GFP (red signal) on representative liver sections isolated from VSV-G, BaEVTR, BaEVTR-CoD, NiV-CD133, and NiV-CD133-CoD LV dosed FRG mice at 14 days after vector dosing (see experimental scheme described in Fig. This experiment was repeated on liver sections from one additional mice per each treatment group and generated similar results. Supplementary Text, Material, Figures and Tables. List and abundance of LV insertion sites from Extended Data Fig. List of primary and immortalized cell lines. Absolute expression values for cell lines genes analyzed with RNA-seq. Normalized expression values for cell lines genes analyzed with RNA-seq. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. In vivo gene editing of human hematopoietic stem and progenitor cells using envelope-engineered virus-like particles. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02911-4'>Impacts of an industrial deep-sea mining trial on macrofaunal biodiversity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 08:37:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In 2022 a large-scale test of a commercial deep-sea mining machine was undertaken on the abyssal plain of the eastern Pacific Ocean at a depth of 4,280 m, recovering over 3,000 t of polymetallic nodules. Here, using a quantitative species-level sediment-dwelling macrofaunal dataset, we investigated spatio-temporal variation in faunal abundance and biodiversity for 2 years before and 2 months after test mining. This allowed for the separation of direct mining impacts from natural background variation, which we found to be significant over the 2-year sampling period. Macrofaunal density decreased by 37% directly within the mining tracks, alongside a 32% reduction in species richness, and significantly increased community multivariate dispersion. While species richness and diversity indices within the tracks were reduced compared with controls, diversity was not impacted when measured by sample-size independent measures of accumulation. We found no evidence for change in faunal abundance in an area affected by sediment plumes from the test mining; however, species dominance relationships were altered in these communities reducing their overall biodiversity. These results provide critical data on the effective design of abyssal baseline and impact surveys and highlight the value of integrated species-level taxonomic work in assessing the risks of biodiversity loss. Proposals for deep-sea mineral extraction, first discussed in the mid 1960s1, include the mining of seafloor massive sulfides found at hydrothermal vents2, mining of cobalt-rich crusts found on deep seamounts3 and the extraction of polymetallic nodules found on abyssal plains (3,000–6,000 m depth)4. Of these, nodule mining currently receives the most attention from the nascent deep-sea mining industry, with efforts focused on the Clarion–Clipperton Zone (CCZ)—a 6 million km2 area of the central Pacific Ocean, estimated to hold over 21 billion tonnes of nickel-, cobalt- and copper-rich polymetallic nodules4. Extensive baseline biodiversity surveys conducted over the last 50 years in the CCZ have predominantly focused on the metazoan invertebrate fauna living in or on the abyssal sediments5,6,7,8,9. The best-studied component of the benthic assemblage is the sediment macrofauna—animals typically ranging in size from 0.3 mm to 2 cm, dominated by annelid worms, peracarid crustaceans and molluscs. Despite low abundance and biomass these macrofaunal communities have relatively high biodiversity, with higher species diversity than found in comparable deep-sea sedimented communities10, and an estimated 90% of the collected taxa being undescribed11. While the abyssal plains may have once been assumed to host homogenous, static and depauperate communities12, studies have since shown these habitats to be home to biodiverse and heterogenous communities which can vary markedly across small spatial13,14,15 and temporal16,17,18 scales. Tests of nodule-mining vehicles have been carried out in both the Pacific and Indian Ocean since the 1970s, as well as seven published disturbance experiments designed to try and understand the potential environmental effects of seabed mining before commercial-scale operations19,20,21,22. These studies have produced mixed results: while some report no impact on infaunal (macrofauna and meiofauna) abundance between 2 years and 44 years following disturbance22,23,24,25, others have found significant reductions in macrofaunal abundance immediately following disturbance26, with residual community-level changes still visible 7 years later27. Only one of these studies followed a full-scale contemporary mining test23, and all have certain methodological limitations such as low statistical replication and lack of pre-impact baseline data. In October 2022, Nauru Ocean Resources (NORI) a wholly owned subsidiary of The Metals Company (TMC), together with the company AllSeas, conducted an extensive trial of a large-scale prototype polymetallic nodule collection vehicle within the NORI-D contract area of the CCZ (hereafter termed the collector test). This collector tracked over 80 km of abyssal seafloor, within a 2 × 4 km test field, and recovered over 3,000 t of nodules using a riser-pipe—the largest of its kind thus far. Here we report on the results of a large species-level quantitative abyssal sediment-dwelling macrofaunal dataset, collected over 2 years, including the combined analysis of both natural temporal variation and the effects 2 months following a large-scale nodule mining test (Fig. Following a modified Before-After-Control-Impact (BACI) design, we investigated the variability of macrofaunal density and diversity, to distinguish between natural spatiotemporal changes and those directly resulting from mining impacts, providing a critical evidence-base to inform conservation planning and effective management of deep-sea mining. a, Map of study region sampled in the NORI-D exploration contract area of the CCZ, East Pacific Ocean. Inset map shows location of the CCZ, with the star indicating the location of NORI-D. Points indicate boxcore sample locations taken across all four expeditions, with coloured boxes showing sampled sites. Plume samples were taken in close proximity to each other and so appear as one point on the map. The PRZ sampled in November 2020 is located 120 km to the northeast of the primary sample locations. Full boxcore collection details can be found in Supplementary Table 8. b, Overview of samples taken at each location at each time point, numbers show number of boxcore samples/total number of macrofaunal individuals. c–e, Example seafloor photographs of each impact level: unimpacted seafloor (c); mining tracks (d); and plume-impacted area (e). Credit: photos in c–e by Daniel Jones (National Oceanography Centre). A total of 4,350 sediment macrofauna (sensu stricto) were sorted from 80 boxcore samples across four sampling campaigns (from November 2020 to December 2022), from which 3,826 specimens (88%) could be identified to a total of 788 species. The dataset used for statistical analyses included all samples from the Collector Test Area (CTA) – which included two subcategories ‘plume' and ‘track' taken after impact—and two control sites, Far Field West (FFW) and Near Field East (NFE) (Fig. Samples from the Far Field East (FFE) and Preservation Reference Zone (PRZ) sites were excluded from statistical analyses because of low temporal replication, although they are presented in all figures for visual comparison. The analysed data included 67 boxcore samples, containing 3,519 specimens, from which 3,089 individuals (87.8%) were identified to 692 species. As is typical for abyssal sediment communities, polychaete annelids were the most abundant taxa across the dataset (44.5% of specimens), closely followed by peracarid crustaceans (isopods, tanaids and amphipods; 37.5%) and molluscs (13.7%). Other phyla including Echinodermata, Nemertea, Bryozoa and Cnidaria occurred at much lower abundances, comprising less than 4% of the total faunal abundance. Mean macrofaunal density across all samples was 217.5 ± 72 individuals per m2 (ind.m−2) (mean ± s.d.) A two-way analysis of variance (ANOVA) found evidence of a statistically significant interaction between site and time on macrofaunal density (ind.m−2) when including the track samples in the time series, but not when including the plume (track, F(6,50) = 8.81, P < 0.001; plume, F(6,45) = 2.52, P = 0.04) (Fig. 2 and Supplementary Table 1; for discussion of P values see Methods). We found evidence of significant natural temporal declines in macrofaunal density across all sites during the pre-impact period of November 2020 to September 2022 (Fig. 2; see Supplementary Table 2 for all one-way ANOVA results and Supplementary Table 3 for results of pairwise t-tests). Similar patterns of change were observed across all phyla (Extended Data Fig. The relationship between macrofaunal density and the multivariate El Niño/Southern Oscillation (ENSO) index (MEI), a potential driver of food availability, is visualized in Extended Data Fig. 2 and indicates a decrease in macrofaunal abundance with an increasingly negative MEI index. The bars and circles indicate medians and data points, respectively. Outliers are denoted by stars. Boxes indicate the interquartile range (25th–75th percentile) and whiskers indicate lower and upper quartiles. Shaded area highlights samples taken following the collector test. Sample sizes (n): C5A FFE = 4, C5A FFW = 2, C5A NFE = 4, C5A PRZ = 5, C5A CTA = 4, C5D FFE = 4, C5D FFW = 3, C5D NFE = 4, C5D CTA = 4, C7A FFW = 5, C7A NFE = 5, C7A CTA = 10, C7B FFW = 5, C7B NFE = 5, C7B plume = 5, C7B track = 10. Following the work of Underwood28,29, we consider an ‘impact' to be a difference in the change of mean abundance/diversity, or time course of mean abundance/diversity, in a defined ‘impacted' area (in this instance, the mining track or plume) from before to after the disturbance, compared with such changes that occur from before to after in the control locations. Immediately before the collector test (September 2022), there was no evidence for significant spatial differences in macrofaunal density between the control sites and the CTA (ANOVA, F(2,17) = 3.01, P = 0.076). However, following the collector test, significant differences in density between sites were observed (ANOVA, F(3,21) = 12.4, P < 0.001). Samples collected from within the mining track had significantly lower macrofaunal densities than NFE (t-test, P < 0.001) and FFW (t-test, P = 0.002) and non-statistically significant lower densities than those sampled from within the plume area (t-test, P = 0.087) (Table 1). Samples taken within the plume area were not statistically different in density than the control sites (Supplementary Table 3). Macrofauna within the mining tracks were 37% lower in density than 1 month before the collector test (t-test, P < 0.001). In comparison, there was no significant change in density observed at NFE (t-test, P = 0.489) or at FFW (t-test, P = 0.06) across the same period. Densities within the plume were not significantly different from those found in the CTA pre-impact (t-test, P = 0.788). A total of 788 species were identified across NORI-D. Despite this high number of species, rarefaction curves did not reach asymptotes when rarefied by either specimens or number of boxcores (Extended Data Fig. Extrapolation analyses estimate needing to sample over 15,000 individuals or over 400 boxcores (equivalent to 100 m2) before the complete diversity of the area is captured. Non-parametric extrapolation analyses predict the total number of macrofaunal species within NORI-D to be between 1,148 and 1,391 (Supplementary Table 4), of which an estimated 521–647 species are polychaete worms. Spatiotemporal changes in macrofaunal species richness and diversity were examined in the same manner as for macrofaunal density (Methods). There was a statistically significant interaction between site and time on macrofaunal species richness (species per 0.25 m2) when including the track samples, but not when including the plume site (two-way ANOVA, track, F(6,50) = 8.69, P < 0.001; plume, F(6,45) = 2.73, P = 0.02). Results of all two- and one-way ANOVAs can be found in Supplementary Tables 1 and 2. Concomitant with the observed reduction in macrofaunal density between sampling periods, we found significant natural temporal changes in diversity for both the CTA and NFE sites, but not for FFW. The CTA also showed significant natural temporal changes in evenness, which were not found in NFE or FFW (Supplementary Table 2). From September 2022 to December 2022, following the collector test, NFE significantly increased in species richness and Gini-Simpson diversity, while there were no significant changes observed in FFW (Fig. At 1 month before impact, species richness within the CTA was 32.5 ± 6.26 (mean ± s.d. ), which significantly decreased within the mining tracks by 32% to a mean of 22.2 ± 7.24 (t-test, P = 0.003) (Fig. Samples from within the plume site had a mean species richness of 32.6 ± 10.36, which was not significantly different from before impact (t-test, P = 0.985). The same pattern was seen for Gini-Simpson diversity in the CTA (Table 1 and Fig. 3b), with diversity significantly reduced within the tracks, but not significantly different within the plume (Fig. In comparison, samples taken in the tracks were significantly more even than before impact (t-test, P = 0.026), while evenness per 0.25 m2 within the plume was not significantly different from the pre-impact CTA (Fig. a–c, Mean (square) and standard deviation (bar) of different alpha diversity measures at each sampled site and time point, circles show individual data points. a, Species richness, measured as number of species per sample (boxcore). Solid lines are observed values, dashed lines represent extrapolated values and shading shows 95% confidence intervals. f, k-dominance curves for each sampled area. For clarity, only sites sampled after test-mining impact in December 2022 (C7B) and the pre-impact collector test area (C7A CTA) are shown. For full curves see Extended Data Fig. Sample sizes (n): C5A FFE = 4, C5A FFW = 2, C5A NFE = 4, C5A PRZ = 5, C5A CTA = 4, C5D FFE = 4, C5D FFW = 3, C5D NFE = 4, C5D CTA = 4, C7A FFW = 5, C7A NFE = 5, C7A CTA = 10, C7B FFW = 5, C7B NFE = 5, C7B plume = 5, C7B track = 10. Increased variability in diversity and composition is often an indicator of disturbance within benthic communities30 and so Levene's tests were used to examine changes in variance of diversity between sites over time. Gini-Simpson diversity within the mining tracks was significantly more variable than the control sites following mining impact (track-NFE: Levene's test, F(1,13) = 6.45, P = 0.025; track-FFW: Levene's test, F(1,13) = 7.96, P = 0.014) and significantly more variable than the CTA 1 month before impact (Levene's test, F(1,18) = 9.05, P = 0.007) (Supplementary Table 5). Individual-based rarefaction curves did not reach asymptotes at the local scale for any site at any time (Fig. For clarity, a reduced number of curves have been presented, including the track and plume, the CTA 1 month before impact (C7A CTA) and the two control sites following the impact (C7B FFW and C7B NFE) (Fig. 3d,e); however, full curves for each site separated by time can be found in Extended Data Fig. When considering individual-based rarefaction—a measure of species accumulation independent of sample size—the communities found within the mining tracks were not notably lower in diversity than other sites. The expected number of species from a random draw of 100 individuals (ES(100)) within the track was 80.5 ± 3.2 (estimated value ± s.e.) compared with a range of 75.1–81.6 from the other sites and time points (Table 1). In comparison, samples taken within the plume area were lower in diversity, with an ES(100) of 73.6 ± 3.2 (Fig. This pattern was also visible when using non-parametric diversity estimators, where the plume has an overall lower number of estimated species than nearly all other sites (Supplementary Table 4). Alternatively, when looking at sample-based rarefaction curves the track was notably lower in diversity than other sampled sites (Fig. 3e, Table 1 and Extended Data Fig. The expected number of species from a random draw of four boxcores within the track was 77.5 ± 16.1, compared with an estimated 105.19 ± 20.63 species within the CTA before the collector test (Table 1). Percentages of species dominance were largely similar between sites and times, with the top ten most-dominant species comprising between 15% and 25.2% of the total abundance at each site (Supplementary Table 6). When plotted on k-dominance curves (Fig. 4c), the plume is less diverse than other sites, with a greater percentage of common species, a similar pattern as seen in the specimen-based rarefaction (Fig. Using permutational multivariate analysis of variance (PERMANOVA) to examine differences in multivariate community composition, we found both site and time to have a significant effect, with no significant interaction between the two, when including both the track and the plume samples in analyses (site: track, pseudo-F = 1.487, P < 0.001 and plume, pseudo-F = 1.375, P < 0.001; time: track, pseudo-F = 1.273, P = 0.002 and plume, pseudo-F = 1.261, P = 0.002) (Supplementary Table 1e). Pairwise PERMANOVA comparisons found significant natural and impact-related temporal changes in community composition within the CTA, including from 1 month pre-impact to 2 months post-impact within the mining tracks (pseudo-F = 1.364, P = 0.021), while there was no such change observed within the plume (Supplementary Table 3e). No significant temporal changes in community composition were observed for either of the control sites. Following the test-mining impact, community composition within the mining tracks and plume was not significantly different to controls (Supplementary Table 3e). When visualized using a non-metric multidimensional scaling (nMDS) ordination, there is a large overlap between most sites at each time, with the PRZ as the most distinct from all other sites (Fig. Samples taken from within the tracks are widely dispersed across the entire plot, overlapping with other sites and times. The multivariate homogeneity of group dispersions was tested using the PERMDISP2 procedure and visualized using a principal coordinate analysis (PCoA) plot (Fig. We found significant variation in multivariate dispersion between sites (permutation test, permutations = 9,999, F(15,64) = 9.87, P < 0.001), of which, communities within the tracks were significantly more dispersed than all other sites at all time points (Fig. a, An nMDS plot of square-root transformed Bray–Curtis dissimilarity between boxcore community data. b, Multivariate homogeneity of group dispersions visualized on the first two axis of a PCoA plot. Larger points show group centroids, with smaller points representing individual samples. Individual points for a and b are coloured by site, with the shape representing sampling period. c, Median distance to group centroid of each sampled community within a site. The bars and circles indicate medians and data points, respectively. Outliers are denoted by stars. Boxes indicate the interquartile range (25th–75th percentile) and whiskers indicate lower and upper quartiles. Sample sizes (n): C5A FFE = 4, C5A FFW = 2, C5A NFE = 4, C5A PRZ = 5, C5A CTA = 4, C5D FFE = 4, C5D FFW = 3, C5D NFE = 4, C5D CTA = 4, C7A FFW = 5, C7A NFE = 5, C7A CTA = 10, C7B FFW = 5, C7B NFE = 5, C7B plume = 5, C7B track = 10. When examining the overall community composition, there is no clear difference in relative proportion of phyla or annelid families between sites and times (Extended Data Fig. Within the mining track, there were lower relative abundances of the families Maldanidae, Nereididae and Sabellidae than there had been in the CTA pre-impact (September 2022) and higher relative abundances of Paraonidae, Spionidae and Phyllodocidae. Indicator species analysis found no significant associations of any taxon to within the mining tracks, either for polychaete family groups or for class-level identifications (Fig. Echinoids (sea urchins), however, were significantly associated with the plume (indicator value (IndVal) = 0.571, P = 0.01). For polychaete families, the Orbiniidae were found to be significantly associated with the plume (IndVal = 0.542, P = 0.033). Cells represent square-root transformed relative abundance of each taxonomic group across all sampled sites. Symbols to the left show significant associations of taxa to specific sites or sampling periods, as identified by Indicator Species Analysis. Only two species were found at all sites at all sampling times: a tanaid crustacean, Stenotanais sp. [NHM_6880_TH_TAN_1] and a benthic serpulid polychaete Serpulidae sp. The PRZ, which was only sampled on one occasion and is located 120 km to the northeast of the other sites, had the highest percentage (27.3%) of species unique to the area (Table 1). Of the species found within the track, 13.4% were found nowhere else, while the percentage of unique species at each other sites and time points (excluding the PRZ) ranged from 10.7% to 21.2%. Venn diagrams depicting the number of species overlaps between the CTA pre- and post-impact and the CTA, FFW and NFE sites can be found in Extended Data Fig. NFE had more unique species (153) than the CTA (118) and FFW (103) when grouping samples from all time points. Deep-sea mining within the CCZ is at a critical juncture, as the industry looks to move beyond the exploration phase and into commercial exploitation31. Consequently, there is a clear need for direct assessment of the impacts of mining on faunal abundance and biodiversity at the seafloor. Our results show that against a background of strong natural variation we can determine significant impacts of a large-scale deep-sea mining test on several aspects of benthic abundance and diversity. While past studies of smaller disturbances have also been able to show significant effects21,22,27, this study attempts to disentangle direct mining impacts from the natural variation inherent in abyssal ecosystems via the creation of a temporal baseline. Immediately following impact, macrofaunal densities decreased significantly by 37% within the path of the nodule collector, while densities within the control sites either increased or remained unchanged. This is in line with patterns seen in previous smaller deep-sea disturbance experiments which have reported 38–63% decreases in macrofaunal density26,32,33. Most abyssal infauna occur in the top 2 cm of sediment8—within our dataset, 74% of individuals were found in this top layer. As such, the observed reduction in faunal density is unsurprising given that nodule mining machines will directly disturb the top layers of sediment during the nodule-removal procedure. Sediment plumes generated by nodule mining vehicles have been predicted to cause significant adverse effects on benthic fauna, including damaging gills and feeding structures and increasing mortality due to toxicity from elevated metal concentrations34. While some impacts have been demonstrated for megafauna35, there is limited empirical or in situ data evaluating these effects on macrofauna. Here we found no significant change in macrofaunal abundance or species richness 2 months following sediment plume deposition, as also reported for meiofauna23 and macrofauna24 by previous studies. Interestingly, while species richness within the plume area was unchanged, diversity as measured by evenness was reduced across the community as a result of a change in dominance relationships—a similar ecological response to that seen in benthic communities impacted by turbidity flows36. Indicator species analysis also found a significant association of the polychaete family Orbiniidae within the plume. Orbiniid species are known to increase in density in response to pollutants or organic enrichment in shallow waters37 and the Antarctic benthos38. While our sample numbers for individual families are low, it is an interesting observation that certain deposit-feeding taxa may respond to the deposition of mining plumes. This follows results from the DISCOL experiment which found a consistent association of a scavenging polychaete species in the family Sigalionidae with plume-impacted areas27. What cannot currently be established is whether this response is caused by the competitive advantage of more resilient species, the mortality of less robust species, or a combination of both. Longer-term sampling and analysis of the impacted site would provide a better assessment of the ecological succession in the community after disturbance. The impacts of the test mining on faunal density and diversity were observed against the backdrop of significant natural temporal changes. This highlights the importance of collecting temporal data across several control and impact sites to accurately measure the impacts of deep-sea mining. The most likely driver of change in benthic animal abundance is food availability, driven potentially by shifts in the ENSO (MEI)39. Macrofaunal densities in NORI-D decreased in parallel with the MEI over the 22-month study period. Significant seasonal variations in benthic communities have been recorded from across the abyssal Pacific16,40, with trends extending across interannual periods where food supplies are higher or lower41. Without repeated sampling at shorter intervals over longer timescales it will remain unclear as to whether the observed variations in NORI-D are part of stochastic or seasonal variation, or a progressive trend related to longer-term climate change42. These data will be crucial to the understanding of long-term recovery rates in disturbed sediments, as significant climate-related changes may have cumulative effects on already impacted communities43. Parallel with the observed reduction in faunal abundance, we found a significant decrease in species richness and diversity within the mining tracks. The observed number of species within a sample is intrinsically linked and sensitive to underlying differences in abundance44. Therefore, the significantly lower abundance of macrofauna within the mining tracks is the most likely driver of the reduced diversity per sample unit. While previous studies have recorded recovery of macrofaunal abundance after disturbance22,27, none has published species-level data for the full community. Therefore, there is currently very limited data on whether diversity can or will completely recover following nodule mining. Seven years after the DISCOL disturbance experiment (involving a plough-harrow which did not remove any nodules) the diversity of polychaetes remained reduced with lower evenness and less singleton species27. High diversity in abyssal sediments is thought to be maintained through a variety of processes including niche diversification over periods of long-term stability and patch-mosaic dynamics45. Given that physical disturbance within the CCZ has been shown to persist over decades22 and is predicted to last for centuries, it is reasonable to hypothesize that the removal of nodules and associated sediment disturbance will result in persistent long-term decreases in species diversity in the mined areas, as has been demonstrated to occur in benthic habitats impacted by bottom-fishing46. Contrary to this local reduction in species diversity per unit area, diversity measured using sample-size independent accumulators was not significantly reduced within the collector test tracks. All rarefaction curves, at both local and regional scales, indicate that NORI-D remains under-sampled for diversity—extrapolation analyses estimate needing to sample over 15,000 individuals or 400 boxcores (equivalent to 100 m2) before curves begin to reach asymptotes. This hinders the conclusions that can be drawn about differences in diversity. Rarefaction assumes that the area being sampled represents a single habitat44, whereas the abyssal seafloor is known to be both physically and biologically heterogenous12,14,47. As more microhabitats within an area are sampled, the estimated number of species found per number of individuals rises, further exacerbating the ‘under-sampling' effect. It is therefore difficult to ascertain if biodiversity within the track is truly in line with unimpacted sites. Reported changes in species diversity following disturbance have been shown to be scale-dependent, with declines more prominent when examined across larger areas48. Monitoring changes in diversity patterns from commercial-scale mining operations will require a specifically designed survey to account for this effect. We suggest that such a sampling design should aim to cover representative sites of differing environmental variables such as nodule type and density, both of which have been shown to influence the community composition of different faunal size classes8,49. This should occur at sufficient replication both temporally and spatially to encompass natural premining variation, an aspect that has been highlighted as often lacking in deep-sea mining environmental impact assessments50. Here we were able to statistically detect variation between sites and time points with two to ten boxcore samples per site/time; however, to improve the robustness of analyses we would recommend a minimum of five, and ideally ten, cores per site and time with samples spatially randomized within strata51. Following the collector test we found significantly higher variability in both diversity and multivariate community composition within the mining track. Increased ecological variability is common following disturbance and is caused by the destabilization of community dynamics, as loss in abundance across populations and the redistribution of resources leads to altered competitive interactions and spatial aggregation of species52. The observed even reduction in abundance across broad taxonomic groups, and similar dominance patterns and percentage of singleton species between the track and controls suggest that the collector test caused an equal removal of individuals across the top sediment layers, rather than disproportionately impacting specific taxonomic groups. Such species-specific effects may only be detectable at longer timescales following impact. While dependent on several ecological and environmental parameters, recolonization and recruitment is a primary determinant of the extent and persistence of disturbance impacts52. Knowledge of this is currently very poor in both the CCZ and the broader abyssal ecosystem53. Increased spatial variability is exacerbated when species are patchily distributed and when recolonization primarily occurs through lateral migration of adults and juveniles rather than larval settlement, both of which are thought to be the case for abyssal infauna14,27. We can therefore hypothesize that in a commercial-scale nodule mine covering 8,500 km2 of seafloor54, community heterogeneity may persist over long timescales. This has already been demonstrated in other deep-sea settings, with elevated variance-mean ratios still observed in the DISCOL area 7 years after impact27. Disturbed deep-sea sediments experience highly localized colonization patterns, creating a spatiotemporal mosaic of patches in various stages of succession55. Longer-term monitoring of the impacted sites would enable better evidence on rates of recovery. This is particularly relevant for the areas impacted by plume settlement, where ecological changes may occur more slowly than in directly impacted sites and so should be a focus of future environmental impact studies. In conclusion, through a replicated spatio-temporal sampling programme we detected significant changes in sediment-dwelling macrofaunal density, diversity and community composition in response to a large-scale test of a polymetallic nodule mining machine. This sets a standard for baseline ecological data in the CCZ and further highlights the importance of integrated species-level taxonomic work56 in assessing ecological responses to disturbance and the risks of biodiversity loss. The study was conducted in the exploration contract area NORI-D, licenced by the International Seabed Authority (ISA) to Nauru Ocean Resources Inc. (NORI). The NORI-D contract area is located in the southeastern CCZ, occupying an area of approximately 27,000 km2 and with water depths of 2,959–4,602 m (Fig. In October 2022, NORI, a wholly owned subsidiary of TMC together with the company AllSeas, conducted a trial of a prototype polymetallic nodule collection vehicle within the NORI-D contract area of the CCZ. This collector test tracked over approximately 80 km of abyssal seafloor within a 2 × 4 km test field and removed over 3,000 t of nodules using a riser-pipe. The full extent of environmental impacts following a disturbance event can only be accurately detected and assessed when sufficient data are available to disentangle the effect of interest from natural spatial and temporal variability. Considering this, we used an asymmetrical Before-After-Control-Impact (BACI) style experimental design28,29 with several control sites, following the principles outlined by Etter 51 that formed the basis of the first set of ISA exploration regulations57. A random stratified sampling design was used, with data collected from four control sites (FFE, FFW, NFE and PRZ) chosen to be spatially equivalent to the area being impacted by the mining disturbance and one impacted site, referred to as the CTA. Sites varied in depth from 4,121 m to 4312 m. Samples were collected at three time points before the collector test (November 2020, May 2021 and September 2022) and again 2 months following the collector test (December 2022). Following the impact, samples taken in the CTA were split into ‘track' and ‘plume' categories. The track samples were collected directly within the mining track. The plume samples were taken 400 m from the mining track. Currently, the sedimentation depth from the impact is not yet published either from modelled or empirical studies58. Examination of autonomous underwater vehicle footage taken in the region by the National Oceanography Centre UK team has provided an image of the seafloor in the region of our plume site. This shows a clear veneer of sediment covering the seafloor and the nodules (Fig. While we do not know the exact depth of the sedimentation, we can confidently state that the plume site 400 m from the track has been impacted by a plume and thus report the impacts on the macrofauna at that distance from the mining track. This will be useful information going forward as further studies are published. A summary of samples taken can be found in Fig. 1, with full details in Supplementary Table 8. Samples were collected across four expeditions to the NORI-D area spanning a total period of 2 years. Campaign 5A (C5A) took place from October to November 2020 aboard the Maersk Launcher and campaign 5D (C5D) took place from April to June 2021 also aboard the Maersk Launcher. Campaigns 7A (C7A) and 7B (C7B) both took place on the MV Island Pride, from August to September 2022 and November to December 2022, respectively. Macrofaunal samples were taken with a USNEL Spade Box Core (Ocean Instruments BX-650 and BX-750) with a sample dimension of 50 × 50 cm (BX-650) or 75 × 75 cm (BX-750). Samples taken with the BX-750 were subsampled using a stainless-steel insert frame of 0.25 m2 (the same equivalent area as the BX-650, the standard size used on previous CCZ surveys listed above). To ensure boxcores were taken inside the caterpillar tracks created by the collector test, the ‘track' cores were visually guided using a remote operated vehicle. The full deployment protocol and subsequent processing of samples followed ref. In short, following recovery and quality assessment, surface water above the sediment was removed with a hose and passed through a 300 µm sieve. Biota visible to the eye on the sediment surface were removed and fixed separately. All nodules were removed from the sediment surface and washed with cold-filtered seawater (CFSW) over buckets to retain the sediment, which was then included with the 0–2-cm live-sorted layer. The sessile fauna directly attached to the nodules were treated separately and will be the subject of a separate paper. A 15 × 15 cm subcore of sediment was taken and sliced in two layers, 0–2 cm and 2–5 cm, before being transferred to CFSW, sieved through a 300 µm sieve and then taken to the at-sea laboratory where individual animals were picked, identified and photographed from the sediment and preserved individually in 80% non-denatured ethanol for high-quality DNA and morphology. All remaining sediment was sliced in layers of 0–2 cm, 2–5 cm and 5–10 cm and sieved on 300 µm sieves in CFSW, before being bulk fixed in 100% non-denatured ethanol. Sediment residues from the 15 × 15 cm live-sorted fraction were returned to respective bulk-fixed depth layers following the removal and processing of animals. The specimens and data from the live-sorted sample are also subsequently merged into the entire boxcore for analysis. A total of 80 boxcores were fully quantitatively assessed. All samples were initially sent to the Natural History Museum, London (NHMUK), where the bulk-fixed sediment residues were sorted under stereomicroscopes. Arthropods and annelids were only counted if the head was present, and echinoderms only if the oral disc was present. Sessile animals such as bryozoans (excluding Ctenostomatida, which are free-living within the sediment), brachiopods, crinoids and sponges were separated and sorted, however were not included in subsequent analyses as they are considered nodule fauna (due to their dependence on the substratum) and not sediment fauna. Serpulid polychaetes were excluded from analyses for the same reason, except for certain species which were determined to be free-living. To minimize systematic error in the sorting process, a subsection (approximately 10%) of sample residues were re-sorted by different experts to ensure no fauna were missed. Once sorted, individual groups were split between institutions for subsequent identification by taxonomic experts, with annelids and molluscs sent to the University of Gothenburg (Sweden), arthropods to the National Oceanography Centre (Southampton, UK) and all other groups remaining at the NHMUK. There are no published field guides to the fauna of the CCZ and approximately 90% of the known species are considered undescribed11. As such, ‘identification' of the species by traditional methods is not possible (for example, comparison with published literature). Taxonomic assignments therefore considered information drawn from both molecular and morphological investigation of every specimen. Using a morphology-based approach, every specimen was assigned to the lowest operational taxonomic unit (OTU; hereby simply referred to as ‘species'), each representing a species. In cases where several specimens were identified as the same species, the voucher code of the best-preserved specimen was used as the identifier of that species, for example: Ophiohelus sp. This avoids confusion with the use of sp. C and so on, where informal and confusing synonyms can easily arise. Open nomenclature was used following ref. 60 and species authorities can be found on WoRMS61. At least one representative from each species, and specimens which could not be assigned by morphology to a species, were then DNA barcoded using standard genetic markers (for example: cytochrome c oxidase subunit 1, 16S ribosomal DNA) for different taxa (for example, refs. These data have been and are being published in a series of taxonomic publications separate to this study (for example, refs. The genetic sequences generated were compared against an internal reference database to further guide identifications. A macrofauna sensu stricto concept was used when defining the specimens to be included in analyses, this included any metazoan retained on a 300 µm sieve, but excluded traditionally considered meiofaunal taxa such as Ostracoda, Copepoda, Nematoda and Halicarida, as well as known sessile nodule fauna (the subject of a separate study) and pelagic or likely pelagic contaminants (such as Hyperiid amphipods). This approach allows for the greatest comparison with published data. Processing of boxcore samples with regards to sliced layers differs between studies7,8,9,16,69 and, as such, the data from all layers (0–10 cm) are summarized together for all analyses. All specimen data used for analyses can be found in Darwin Core (DwC) format in Supplementary Table 8. All analyses were conducted using R v.4.4.1., using the packages vegan70, SpadeR71, iNEXT72, BiodiversityR73, indicspecies74 and rstatix75. Following the work of Underwood28,29, we consider an ‘impact' to be defined as a difference in the change of mean abundance/diversity, or time course of mean abundance/diversity, in a defined ‘impacted' area (in this instance, the mining track or plume area) from before to after the disturbance, compared with such changes that occur from before to after in the control locations. While the same sites were sampled at several time points, we do not consider them to be part of a ‘repeated measures' sample design. Boxcores were randomly taken within the predefined areas and as deep-sea macrofaunal community composition can vary at the centimetre scale14 we consider samples to be independent. Samples from the PRZ and FFE sites are included in all figures for visual comparison, however, were excluded from statistical analyses because of a lack of temporal replication. To examine differences in total faunal abundance the full dataset was used, including specimens which could only be identified to higher taxonomic levels (for example, genus or family level, such as ‘Polychaeta indet.'). Faunal abundances were transformed into densities (individuals per m2) before all calculations. A two-way ANOVA with site and time as fixed factors was used to assess if there were significant interactions between site and sampling time on macrofaunal abundance. Where significant factor effects were found, one-way ANOVAs were conducted on site at each level of time and time at each level of site. These were followed by pairwise t-tests to test for significant differences between sites at each time point and between time points for each site. No adjustment of P values was performed, as the argument in favour of adjusting for type I errors applies to random distributions, which is seldom the case when studying living systems76,77. Before each analysis, normality and homogeneity of variances were checked with Shapiro–Wilkes and Levene's tests, respectively. Levene's tests were performed on data within each time point and within each site across all time points. Transformation could not stabilize significantly heterogeneous variances in all identified cases and so untransformed data were analysed. Where significant heterogeneity of variance was found, a non-parametric Welches ANOVA was performed in place of the specific one-way ANOVA. For the two-way ANOVA, results were considered robust if not significant (at P > 0.05), since the probability of type II error is not affected by heteroscedasticity, or significant at P < 0.01 (instead of P < 0.05), to compensate for increased probability of type I error78. For all other analyses a significance level of 0.05 was used. As increased variability can be an indicator of disturbance in marine communities30, pairwise Levene's tests were performed between samples to identify significant differences in variance. For biodiversity analyses, we examined both species richness (the total number of species, irrespective of abundance) and species diversity, measures including both the number of species and degree of evenness or dominance. A reduced dataset was used for these analyses, including only specimens identified to species level, either a formally described species (for example, Vesicomya galatheae) or an identified OTU (for example, Acrocirridae sp. On the basis of abundance data from an assemblage, calculation of species richness (both observed and estimated) is statistically difficult, especially for highly diverse assemblages with many rare species, as is typically the case for the abyssal benthos10,79. As such, we examined biodiversity and the way it varies both spatially and temporally in several ways at different scales, considering species richness and diversity, diversity accumulators and sample-size independent measures. To compare biodiversity between sites and time points in terms of the observed diversity per unit area, species richness (S), Gini-Simpson (1-D) and Simpson's evenness (E1/D) indices were calculated for each boxcore sample. To detect spatial and temporal changes in richness and diversity both before and after test mining, these measures of diversity were analysed using two-way and one-way ANOVAs, pairwise t-tests and Levene's tests, as described for abundance data. Indices were also calculated for the communities as a whole, grouping all boxcores into one sample. Both individual-based and sample-based rarefaction curves were computed for each sampled area and time point, to provide sample-size independent measures of diversity based on accumulation rates. On the basis of these data, the expected number of species was calculated for 100 individuals (ES(100)) from each area and the expected number of species to be sampled from four boxcores (ES(4bc)). In addition, non-parametric species richness estimators were computed for each area to represent local diversity. While rarefaction compares observed richness among samples, richness estimators evaluate the total richness of a community. Abundance-based estimators included Chao180 and Chao1-bc (Chao1 bias-corrected)81. Incidence-based estimators included first- and second-order Jackknife82. To visualize the dominance structure of species within communities, k-dominance curves were plotted. k-dominance, or cumulative curves, are the cumulative relative abundance of species plotted against the species rank, where the most elevated curve has the lowest diversity83. Community composition analyses used the same reduced ‘species-level' dataset as used in diversity analyses. Analyses were based on a resemblance matrix using the Bray–Curtis similarity index after square-root transformation of abundance data. This transformation procedure allows for all species, including singletons, to contribute to the similarity matrix while giving the most common species greater weight. Multivariate differences in community structure between sites and time points were determined graphically using non-metric multidimensional scaling (nMDS) and quantified using PERMANOVA84, with P values obtained by 9,999 permutations of the residuals. PERMANOVA tests were configured as described for univariate analyses, with site and time as fixed factors. Following PERMANOVA analysis, pairwise comparisons between communities were made using pairwise.adonis2, a wrapper function for multilevel pairwise comparison using adonis2 from the package vegan85. Significant differences in multivariate dispersion were tested for using the betadisper function in the package vegan, which implements the PERMDISP method of ref. 86, a multivariate analogue of a Levene's test, to analyse multivariate homogeneity of group dispersions. This was followed by a permutation test for significance and pairwise t-tests to identify significant differences in dispersion. Multivariate dispersion has been used previously to detect disturbance responses in deep-sea benthic communities33. Indicator species analysis87 was performed using multilevel pattern analysis88 to determine if any taxa were significantly associated with any site or time point. The IndVal index measures the strength of association between a taxonomic unit and a site group and is based on the product of the mean abundance and relative frequency of occurrence of each group within a given dataset. Statistical significance of the relationship between taxa and site is tested using Monte Carlo randomizations with 9,999 permutations87. Analysis was conducted using abundance data for polychaete families and for taxa identified to Class level (or higher) to improve statistical power, due to the low abundances of each species. The full dataset, including all specimens, was used for this analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated for this study and used in analyses are available in Supplementary Tables. Data handling and analyses were implemented using standard methods, software tools and code functions detailed in Methods. Mining of deep-sea seafloor massive sulfides: a review of the deposits, their benthic communities, impacts from mining, regulatory frameworks and management strategies. & & Dunham, R. E. Seamount characteristics and mine-site model applied to exploration- and mining-lease-block selection for cobalt-rich ferromanganese crusts. & Kuhn, T. Deep-ocean polymetallic nodules as a resource for critical materials. Z. Benthic Baseline Survey of the DOMES Area (NOAA, 1977). Wilson, G. D. F. Macrofauna abundance, species diversity and turnover at three sites in the Clipperton–Clarion fracture zone. Patterns of macrofaunal biodiversity across the Clarion-Clipperton zone: an area targeted for seabed mining. Chuar, C. H., Tong, S. J. W., Chim, C. K., Wong, H. P. S. & Tan, K. S. Abyssal macrofaunal community structure in the polymetallic nodule exploration area at the easternmost region of the Clarion-Clipperton fracture zone, Pacific Ocean. Characterization of deep-sea macrofauna in the Korean exploration claim area in the Clarion-Clipperton fracture zone, Northeastern Pacific Ocean. Stewart, E. C. D. et al. Biodiversity, biogeography, and connectivity of polychaetes in the world's largest marine minerals exploration frontier. Rabone, M. et al. How many metazoan species live in the world's largest mineral exploration region?. & Etter, R. J. Deep-Sea Biodiversity: Pattern and Scale (Harvard Univ. Glover, A. G. et al. Polychaete species diversity in the central Pacific abyss: local and regional patterns, and relationships with productivity. McClain, C. R., Nekola, J. C., Kuhnz, L. & Barry, J. P. Local-scale faunal turnover on the deep Pacific seafloor. The importance of different spatial scales in determining structural and functional characteristics of deep-sea infauna communities. Effects of environmental and climatic drivers on abyssal macrobenthic infaunal communities from the NE Pacific nodule province. Enduring science: three decades of observing the Northeast Atlantic from the Porcupine Abyssal Plain Sustained Observatory (PAP-SO). Horton, T. et al. Are abyssal scavenging amphipod assemblages linked to climate cycles?. The environmental impacts of deep-sea mining. The large-scale environmental impact experiment DISCOL—reflection and foresight. Biological responses to disturbance from simulated deep-sea polymetallic nodule mining. Long-term impact and biological recovery in a deep-sea mining track. Lefaible, N. et al. Industrial mining trial for polymetallic nodules in the Clarion–Clipperton zone indicates complex and variable disturbances of meiofaunal communities. International Symposium on Environmental Studies for Deep-Sea Mining 331–335 (MMAJ, 1997). Trueblood, D. D., Ozturgut, E., Pilipchuk, M. & Gloumov, I. F. The ecological impacts of the Joint U.S.–Russian benthic impact experiment. Second Ocean Mining Symposium 139–145 (ISOPE, 1997). A., Rathod, V. & Rodrigues, N. Response of deep-sea macrobenthos to a small-scale environmental disturbance. Borowski, C. Physically disturbed deep-sea macrofauna in the Peru Basin, southeast Pacific, revisited 7 years after the experimental impact. The mechanics of spatially replicated sampling programmes to detect environmental impacts in a variable world. On beyond BACI: sampling designs that might reliably detect environmental disturbances. Warwick, R. M. & Clarke, K. R. Increased variability as a symptom of stress in marine communities. McVeigh, K. Canadian company in negotiations with Trump to mine seabed. Borowski, C. & Thiel, H. Deep-sea macrofaunal impacts of a large-scale physical disturbance experiment in the Southeast Pacific. Effects of experimental in situ seabed disturbance on deep-sea macrofaunal communities of Chatham Rise, Southwest Pacific. & Drazen, J. C. Evaluating deep-sea communities' susceptibility to mining plumes using shallow-water data. Biological effects 26 years after simulated deep-sea mining. Review and syntheses: impacts of turbidity flows on deep-sea benthic communities. Bigot, L. et al. Assessment of the ecological quality status of soft-bottoms in Reunion Island (tropical Southwest Indian Ocean) using AZTI marine biotic indices. Stark, J. S., Kim, S. L. & Oliver, J. S. Anthropogenic disturbance and biodiversity of marine benthic communities in Antarctica: a regional comparison. & Smith, K. L. Shifts in deep-sea community structure linked to climate and food supply. & Smith, K. L. Sediment community response to a temporally varying food supply at an abyssal station in the NE Pacific. & Smith, K. L. Connections between climate, food limitation, and carbon cycling in abyssal sediment communities. Glover, A. G. et al. Temporal change in deep-sea benthic ecosystems: a review of the evidence from recent time-series studies. Defining “serious harm” to the marine environment in the context of deep-seabed mining. Magurran, A. E. Measuring Biological Diversity (Wiley-Blackwell, 2004). On some hypotheses of diversity of animal life at great depths on the sea floor. Thrush, S. F. & Dayton, P. K. Disturbance to marine benthic habitats by trawling and dredging: implications for marine biodiversity. Simon-Lledó, E. et al. Megafaunal variation in the abyssal landscape of the Clarion Clipperton zone. Dumbrell, A. J. et al. Changes in species diversity following habitat disturbance are dependent on spatial scale: theoretical and empirical evidence. Fleming, B. F. M., Simon-Lledó, E., Benoist, N., O'Malley, B. Influence of seabed heterogeneity on benthic megafaunal community patterns in abyssal nodule fields. Clark, M. R., Durden, J. M. & Christiansen, S. Environmental impact assessments for deep-sea mining: can we improve their future effectiveness? General sampling design for baseline studies. in Standardization of Environmental data and information—Development Guidelines 427–447 (International Seabed Authority, 2002). A. Disturbance-driven changes in the variability of ecological patterns and processes. Amon, D. J. et al. Assessment of scientific gaps related to the effective environmental management of deep-seabed mining. Madureira, P., Brekke, H., Cherkashov, G. & Rovere, M. Exploration of polymetallic nodules in the area: reporting practices, data management and transparency. & Fonseca, G. Active colonisation of disturbed sediments by deep-sea nematodes: evidence for the patch mosaic model. Glover, A. G., Wiklund, H., Chen, C. & Dahlgren, T. G. Managing a sustainable deep-sea ‘blue economy' requires knowledge of what actually lives there. Standardization of Environmental Data and Information—Development of Guidelines (International Seabed Authority, 2002). TMC Subsidiary NORI shares preliminary findings on environmental impacts of pilot nodule collection system test. Glover, A. G., Dahlgren, T. G., Wiklund, H., Mohrbeck, I. & Smith, C. R. An end-to-end DNA taxonomy methodology for benthic biodiversity survey in the Clarion–Clipperton zone, Central Pacific Abyss. Recommendations for the standardisation of open taxonomic nomenclature for image-based identifications. World Register of Marine Species (WoRMS, 2025). Wiklund, H. et al. Abyssal fauna of the UK-1 polymetallic nodule exploration area, Clarion–Clipperton zone, central Pacific Ocean: Mollusca. Wiklund, H. et al. Checklist of newly-vouchered annelid taxa from the Clarion–Clipperton zone, central Pacific Ocean, based on morphology and genetic delimitation. Stewart, E. C. D., Bribiesca-Contreras, G., Weston, J. N. J., Glover, A. G. & Horton, T. Biogeography and phylogeny of the scavenging amphipod genus Valettietta (Amphipoda: Alicelloidea), with descriptions of two new species from the abyssal Pacific Ocean. Dahlgren, T. G. et al. Abyssal fauna of the UK-1 polymetallic nodule exploration area, Clarion–Clipperton zone, central Pacific Ocean: Cnidaria. Bribiesca-Contreras, G. et al. Benthic megafauna of the western Clarion–Clipperton zone, Pacific Ocean. Neal, L. et al. Taxonomy, phylogeny, and biodiversity of Lumbrineridae (Annelida, Polychaeta) from the Central Pacific Clarion–Clipperton zone. Drennan, R. et al. On Anguillosyllis cf. hessleri Maciolek, 2020—a species complex from the Clarion–Clipperton zone, abyssal central Pacific. Bonifácio, P., Martínez Arbizu, P. & Menot, L. Alpha and beta diversity patterns of polychaete assemblages across the nodule province of the eastern Clarion–Clipperton fracture zone (equatorial Pacific). Oksanen, J. et al. vegan: Community ecology package. Chao, A., Ma, K. H., Hseih, T. C. & Chiu, C. H. SpadeR: psecies-richness prediction and diversity estimation in R. R version 0.1.1 (2015). Hsieh, T. C., Ma, K. H. & Chao, A. iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). Kindt, R. BiodiversityR: package for community ecology and suitability analysis. De Cáceres, M., Jansen, F., Endicott, S. & Dell, N. Package ‘indicspecies': Relationship between species and groups of sites. Kassambara, A. rstatix: Pipe-friendly framework for basic statistical tests. No adjustments are needed for multiple comparisons. & Katsanevakis, S. Strengthening statistical usage in marine ecology. Experiments in Ecology: Their Logical Design and Interpretation Using Analysis of Variance (Cambridge Univ. McClain, C. R. The commonness of rarity in a deep-sea taxon. Chao, A. Nonparametric estimation of the number of classes in a population. An improved nonparametric lower bound of species richness via a modified Good–Turing frequency formula. Burnham, K. P. & Overton, W. S. Robust estimation of population size when capture probabilities vary among animals. Clarke, K. R. Comparisons of dominance curves. Anderson, M. J. Permutational multivariate analysis of variance (PERMANOVA). Wiley StatsRef: Statistics Reference Online https://doi.org/10.1002/9781118445112.stat07841 (2017). Martinez Arbizu, P. pairwiseAdonis: Pairwise multilevel comparison using adonis. Anderson, M. J. Distance-based tests for homogeneity of multivariate dispersions. Dufrêne, M. & Legendre, P. Species assemblages and indicator species:the need for a flexible asymmetrical approach. De Cáceres, M., Legendre, P. & Moretti, M. Improving indicator species analysis by combining groups of sites. We would like to thank the masters, crew and technical staff aboard the Maersk Launcher and MV Island Pride for their outstanding support, C. Dalgleish and T. Adamson for managing at-sea operations and L. Marsh for managing the entire NORI-D environmental programme. We thank all researchers aboard the vessels who assisted with boxcore processing, sorting and sieving samples. We acknowledge the continued support from the NHMUK Consultancy team (H. Rousham, R. Fryer and J. Herrington). was supported through funding from NERC (grant no. Natural History Museum, London, UK Eva C. D. Stewart, Helena Wiklund, Lenka Neal, Guadalupe Bribiesca-Contreras, Regan Drennan, Corie M. B. Boolukos, Lucas D. King, Muriel Rabone, Maria B. Arias & Adrian G. Glover School of Ocean and Earth Sciences, University of Southampton, Southampton, UK National Oceanography Centre, Southampton, UK Guadalupe Bribiesca-Contreras, Georgina Valls Domedel, Amanda Serpell-Stevens & Tammy Horton Department of Geography, School of the Human Environment, University College Cork, Cork, Ireland Centre for Life's Origins and Evolution, University College London, London, UK NORCE Norwegian Research Centre, Bergen, Norway Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the study. collected and processed samples at sea, led by H.W. processed samples in the laboratory. performed all analyses and data visualizations. and all authors reviewed, edited and approved the final version. Correspondence to Eva C. D. Stewart or Adrian G. Glover. This work was funded by TMC through its subsidiary NORI. NORI holds exploration rights to the NORI-D contract area in the CCZ regulated by the ISA and sponsored by the government of Nauru. It is required by international law to carry out the environmental work to the best standard and to make data and analyses available to all. The funders had no influence on data analysis, interpretation or presentation. Nature Ecology & Evolution thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Squares indicate mean and bars show standard deviation. ‘Other' includes Echinodermata, Nemertea, Bryozoa, Cnidaria, and un-identifiable metazoans. Sample sizes (n): C5A FFE = 4, C5A FFW = 2, C5A NFE = 4, C5A PRZ = 5, C5A CTA = 4, C5D FFE = 4, C5D FFW = 3, C5D NFE = 4, C5D CTA = 4, C7A FFW = 5, C7A NFE = 5, C7A CTA = 10, C7B FFW = 5, C7B NFE = 5, C7B Plume = 5, C7B Track = 10. MEI values extracted from https://psl.noaa.gov/enso/mei/. Squares indicate mean and bars show standard deviation of grouped macrofaunal densities from all sampled sites within NORI-D, excluding samples taken from within sites impacted by the Oct. 2022 mining test (Track and Plume). Sample sizes (n) from left to right = 20, 15, 20, 10. Solid lines are observed values, dashed lines represent extrapolated values, and shading shows 95% confidence intervals. Solid lines are observed values, dashed lines represent extrapolated values, and shading shows 95% confidence intervals. Relative proportion of invertebrate macrofaunal (a) phyla, and (b) annelid families across all sampled sites and time points. Comparisons between (a) the CTA pre-impact (September 2022, C7A) and the post-impact track and plume sites; and (b) the CTA, FFW, and NFE sites, with all sampling times combined. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Stewart, E.C.D., Wiklund, H., Neal, L. et al. Impacts of an industrial deep-sea mining trial on macrofaunal biodiversity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67062-3'>Complexity of quantum tomography from genuine non-Gaussian entanglement</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-05 07:43:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications , Article number:  (2025) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Quantum state tomography typically requires exponentially many copies of a quantum state, due to the complex correlations present in large systems. We show that, for bosonic systems, the scaling is completely determined by the nature of these correlations. Motivated by the Hong-Ou-Mandel effect and boson sampling, we define Gaussian-entanglable (GE) states, produced by generalized interference between separable bosonic modes. GE states greatly extend the Gaussian family, encompassing separable states, multi-mode Gottesman-Kitaev-Preskill codes, entangled cat states, and boson-sampling outputs—resources for error correction and quantum advantage. We prove that any pure GE state of m modes can be learned efficiently, requiring only poly(m) copies, via a protocol based on Gaussian unitaries, local tomography, and classical post-processing; for boson-sampling states, no Gaussian unitaries are needed. For states outside GE, we define an operational monotone—the minimal number of ancillary modes needed to make them GE—which exactly characterizes the exponential tomography overhead. We also show that deterministic generation of NOON states with N ≥ 3 via two-mode interference is impossible. The data generated in this study have been deposited in GitHub72. The theoretical results of the manuscript are reproducible from the analytical formulas and derivations presented therein. Codes for numerical simulation are available in GitHub72. Smithey, D., Beck, M., Raymer, M. G. & Faridani, A. Measurement of the Wigner distribution and the density matrix of a light mode using optical homodyne tomography: application to squeezed states and the vacuum. Google Scholar Lvovsky, A. I. & Raymer, M. G. Continuous-variable optical quantum-state tomography. Google Scholar Anshu, A. & Arunachalam, S. A survey on the complexity of learning quantum states. Google Scholar Eisert, J. et al. Quantum certification and benchmarking. Google Scholar O'Donnell, R. & Wright, J. Efficient quantum tomography. In Proceedings of the Forty-Eighth Annual ACM Symposium on Theory of Computing (eds Wichs, D. & Mansourby, Y.) 899–912 (ACM, New York, NY, USA, 2016). Kueng, R., Rauhut, H. & Terstiege, U. Low rank matrix recovery from rank one measurements. Google Scholar Mele, F. A. et al. Learning quantum states of continuous variable systems. Fanizza, M., Rouzé, C. & França, D. S. Efficient Hamiltonian, structure and trace distance learning of Gaussian states. Preprint at https://arxiv.org/abs/2411.03163 (2025). A., Tirone, S. & Lami, L. Optimal estimates of trace distance between bosonic Gaussian states and applications to learning. Weedbrook, C. et al. Gaussian quantum information. Google Scholar Serafini, A. Quantum Continuous Variables: A Primer of Theoretical Methods (CRC Press, 2017). Hong, C. K., Ou, Z. Y. & Mandel, L. Measurement of subpicosecond time intervals between two photons by interference. Google Scholar Boto, A. N. et al. Quantum interferometric optical lithography: exploiting entanglement to beat the diffraction limit. Google Scholar Mitchell, M. W., Lundeen, J. S. & Steinberg, A. M. Super-resolving phase measurements with a multiphoton entangled state. Google Scholar Aaronson, S. & Arkhipov, A. The computational complexity of linear optics. In Proceedings of the Forty-Third Annual ACM Symposium on Theory of Computing (ed. Vadhan, S. P.) 333–342 (ACM, New York, NY, USA, 2011). Zhong, H.-S. et al. Quantum computational advantage using photons. Google Scholar Madsen, L. S. et al. Quantum computational advantage with a programmable photonic processor. Google Scholar VanMeter, N. et al. General linear-optical quantum state generation scheme: applications to maximally path-entangled states. Google Scholar Parellada, P. V., i Garcia, V. G., Moyano-Fernández, J. J. & Garcia-Escartin, J. C. No-go theorems for photon state transformations in quantum linear optics. Results Phys. Google Scholar Parellada, P. V. et al. Lie algebraic invariants in quantum linear optics. Preprint at https://arxiv.org/abs/2409.12223 (2024). Diringer, A. A., Blumenthal, E., Grinberg, A., Jiang, L. & Hacohen-Gourgy, S. Conditional-not displacement: fast multioscillator control with a single qubit. Google Scholar Brady, A. J., Eickbusch, A., Singh, S., Wu, J. & Zhuang, Q. Advances in bosonic quantum error correction with Gottesman–Kitaev–Preskill codes: theory, engineering and applications. Quantum Electron. Relative entropy is an exact measure of non-Gaussianity. Google Scholar Genoni, M. G., Paris, M. G. & Banaszek, K. Quantifying the non-Gaussian character of a quantum state by quantum relative entropy. Google Scholar Genoni, M. G. & Paris, M. G. Quantifying non-gaussianity for quantum information. Zhuang, Q., Shor, P. W. & Shapiro, J. H. Resource theory of non-Gaussian operations. Takagi, R. & Zhuang, Q. Convex resource theory of non-Gaussianity. Albarelli, F., Genoni, M. G., Paris, M. G. & Ferraro, A. Resource theory of quantum non-gaussianity and Wigner negativity. Royer, B., Singh, S. & Girvin, S. M. Encoding qubits in multimode grid states. PRX Quantum 3, 010335 (2022). Wu, J., Brady, A. J. & Zhuang, Q. Optimal encoding of oscillators into more oscillators. Conrad, J., Eisert, J. & Arzani, F. Gottesman-Kitaev-Preskill codes: a lattice perspective. Brady, A. J., Wu, J. & Zhuang, Q. Safeguarding oscillators and qudits with distributed two-mode squeezing. Gottesman, D., Kitaev, A. & Preskill, J. Encoding a qubit in an oscillator. Chitambar, E. & Gour, G. Quantum resource theories. Walls, D. F. & Milburn, G. J. Quantum Opt. (Springer Science & Business Media, 2007). Flühmann, C. et al. Encoding a qubit in a trapped-ion mechanical oscillator. Blais, A., Girvin, S. M. & Oliver, W. D. Quantum information processing and quantum optics with circuit quantum electrodynamics. Giovannetti, V., Garcia-Patron, R., Cerf, N. J. & Holevo, A. S. Ultimate classical communication rates of quantum optical channels. Lawrie, B. J., Lett, P. D., Marino, A. M. & Pooser, R. C. Quantum sensing with squeezed light. Ganapathy, D. et al. Broadband quantum enhancement of the LIGO detectors with frequency-dependent squeezing. Larsen, M. V., Chamberland, C., Noh, K., Neergaard-Nielsen, J. S. & Andersen, U. L. Fault-tolerant continuous-variable measurement-based quantum computation architecture. Prx Quantum 2, 030325 (2021). Holevo, A. Some statistical problems for quantum Gaussian states. Holevo, A. S. Probabilistic and Statistical Aspects of Quantum Theory, Vol. 1 (Springer Science & Business Media, 2011). Eisert, J., Scheel, S. & Plenio, M. Distilling Gaussian states with Gaussian operations is impossible. Giedke, G. & Cirac, J. I. Characterization of Gaussian operations and distillation of Gaussian states. Fiurášek, J. Gaussian transformations and distillation of entangled Gaussian states. Niset, J., Fiurášek, J. & Cerf, N. J. No-go theorem for Gaussian quantum error correction. Banaszek, K. & Wódkiewicz, K. Nonlocality of the Einstein-Podolsky-Rosen state in the Wigner representation. Banaszek, K. & Wódkiewicz, K. Testing quantum nonlocality in phase space. Lloyd, S. & Braunstein, S. L. Quantum computation over continuous variables. Bartlett, S. D. & Sanders, B. C. Universal continuous-variable quantum computation: requirement of optical nonlinearity for photon counting. Mari, A., Giovannetti, V. & Holevo, A. S. Quantum state majorization at the output of bosonic Gaussian channels. Nature Commun. De Palma, G., Trevisan, D., Giovannetti, V. & Ambrosio, L. Gaussian optimizers for entropic inequalities in quantum information. Chabaud, U. & Walschaers, M. Resources for bosonic quantum computational advantage. Nielsen, M. A. & Chuang, I. Quantum Computation and Quantum Information (Cambridge university press, 2002). Becker, S., Datta, N., Lami, L. & Rouzé, C. Classical shadow tomography for continuous variables quantum systems. IEEE Trans. Mauro D'Ariano, G., Paris, M. G., & Sacchi, M. F. Quantum tomography. In Advances in Imaging and Electron Physics Vol. Hawkes, P. W.) 205–308 https://doi.org/10.1016/S1076-5670(03)80065-4 (Elsevier, 2003). Chabaud, U., Douce, T., Grosshans, F., Kashefi, E., & Markham, D. Building trust for continuous variable quantum states. In 15th Conference on the Theory of Quantum Computation, Communication and Cryptography https://arxiv.org/abs/arXiv:1905.12700 (2020). Chabaud, U., Grosshans, F., Kashefi, E. & Markham, D. Efficient verification of boson sampling. Horodecki, M., Shor, P. W. & Ruskai, M. B. Entanglement breaking channels. Saner, S. et al. Generating arbitrary superpositions of nonclassical quantum harmonic oscillator states. Preprint at https://arxiv.org/abs/2409.03482 (2024). Walter, M., Gross, D. & Eisert, J. Multipartite entanglement. In Quantum Information: From Foundations to Quantum Technology Applications, (eds Bruß, D. & Leuchs, G.) 293 (Wiley-VCH (John Wiley & Sons), Weinheim, Germany, 2016). Bădescu, C. & O'Donnell, R. Improved quantum data analysis. In Proceedings of the 53rd Annual ACM SIGACT Symposium on Theory of Computing (eds Khuller, S. & Williams, V. V.) 1398–1411 (ACM, New York, NY, United States, 2021). Knill, E., Laflamme, R. & Milburn, G. J. A scheme for efficient quantum computation with linear optics. Park, J., Lee, J., Ji, S.-W. & Nha, H. Quantifying non-Gaussianity of quantum-state correlation. Sperling, J., Perez-Leija, A., Busch, K. & Silberhorn, C. Mode-independent quantum entanglement for light. Chabaud, U. & Mehraban, S. Holomorphic representation of quantum computations. Lopetegui, C. E., Isoard, M., Treps, N. & Walschaers, M. Detection of mode-intrinsic quantum entanglement. Optica Quantum 3, 312–328 (2025) Navarrete-Benlloch, C., García-Patrón, R., Shapiro, J. H. & Cerf, N. J. Enhancing quantum entanglement by photon addition and subtraction. Walschaers, M., Fabre, C., Parigi, V. & Treps, N. Entanglement and Wigner function negativity of multimode non-Gaussian states. Laha, P., Yasir, P. A. & van Loock, P. Genuine non-Gaussian entanglement of light and quantum coherence for an atom from noisy multiphoton spin-boson interactions. Zhao, X., Liao, P., Mele, F. A., Chabaud, U. & Zhuang, Q. Complexity of Quantum Tomography from Genuine non-Gaussian Entanglement (GitHub, accessed 16 November 2025,); https://github.com/peng-cheng-liao/Complexity-of-quantum-tomography-from-genuine-non-Gaussian-entanglement (2025). Download references We thank Xun Gao, Haocun Yu, and Changhun Oh for useful comments on an earlier version of this manuscript. This project is supported by NSF 2240641 (Q.Z. ), an unrestricted gift from Google (Q.Z. ), PRIN 2022 “Recovering Information in Sloppy QUantum modEls (RISQUE)”, code 2022T25TR3, CUP E53D23002400006 (F.A.M. ), European Union ERC StG ETQO, Grant Agreement no. and European Union's Horizon Europe Framework Programme (EIC Pathfinder Challenge project Veriqub) under Grant Agreement No. Ming Hsieh Department of Electrical and Computer Engineering, University of Southern California, Los Angeles, CA, USA Xiaobin Zhao, Pengcheng Liao & Quntao Zhuang Scuola Normale Superiore, Pisa, Italy Francesco Anna Mele DIENS, École Normale Supérieure, PSL University, CNRS, INRIA, Paris, France Department of Physics and Astronomy, University of Southern California, Los Angeles, CA, USA Quntao Zhuang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar proposed the study and supervised the project. derived all analytical results, with inputs from all authors. performed all numerical simulations, with inputs from X.Z. made important contributions to Theorem 2 in the degenerate case, Theorem 3 and Corollary 4. wrote the manuscript, with inputs from all authors. Correspondence to Xiaobin Zhao or Quntao Zhuang. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Zhao, X., Liao, P., Mele, F.A. et al. Complexity of quantum tomography from genuine non-Gaussian entanglement. Nat Commun  (2025). Download citation Received: 27 May 2025 Accepted: 21 November 2025 Published: 05 December 2025 DOI: https://doi.org/10.1038/s41467-025-67062-3 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            