
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/military/a68815223/ufo-missile-collision-yemen/'>A Missile Mysteriously Collided with a UFO. An Expert Finally Reveals the Truth About What Happened That Day.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 21:41:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Science reveals how we see the world differently through a moving camera. The blurry and short clip, appearing as if filmed from a point above the intersecting objects, was released at a House Oversight transparency hearing on Sept. 9. It attracted national attention, but what many people thought they saw was an illusion. So we asked a missile physics expert for some help. Fred Lamb, PhD, a theoretical astrophysicist at the University of Illinois, chaired a ballistic missile report earlier this year for the American Physical Society. He cautioned by e-mail that any analysis requires “many, many assumptions” for experts such as himself who have “no independent knowledge of the event,” but said he would do his best proceeding on assumptions. He emphasized that this type of missile strike is not what the public would expect to see from watching Hollywood movies or TV series of ballistic explosions. “The main reason for this is to make the movie or TV event loud, dramatic, and exciting—not because that is what the destruction of a target, such as a balloon, would look like in reality,” he said. But there are at least 10 variants of Hellfire, Lamb said. His video analysis assumes one of these variants was fired from an MQ-9 Reaper drone, based on media reports, and that the camera that took the footage was flying on a second drone far above, using a “very high magnification thermal imaging camera,” he said. This drone type has a reported operational altitude of 25,000 feet, a little lower than typical commercial airliner flight routes, he said. However, Lamb said one possibility is that the camera is zooming far down to the intersecting objects, creating an optical illusion known as “parallax motion.” In a restricted field of view such as a camera lens, the target may be appearing to move very quickly—and that would especially be the case when there are few background objects with which to judge the motion. Other UAPs that have appeared to be moving very rapidly have been found to be moving not under their own volition, but only because a wind was pushing them along, Lamb said. He therefore suggested it is plausible—but not a slam dunk—that the “target” might be a balloon high in the air reflecting sunlight, which “appears to be moving rapidly mostly because of ‘parallax motion,' but also because it is being carried along by a high-altitude wind.” When asked if it is a pilot, a commanding officer or somebody else, the U.S. Army said it was unable to release that information “due to operational considerations.” Lamb added that generally, “There are many possible reasons why the U.S. military might want to bring down a balloon that was floating in an area where a conflict is ongoing.” As for why the missile continued and did not appear to explode? Lamb said there are a couple of possibilities. Or more likely, it is a “type of Hellfire missile that … did not have a substantial explosive warhead—but could nevertheless cut or tear the balloon.” The most likely example would be the AGM-114R9X, he added, which pops out sharp blades with only a small amount of explosive. They were deployed during the 1991 Gulf War operations—also known as “Desert Storm” in the U.S., and continue to be used through the ongoing “War on Terror.” Hellfire missiles have been launched from both drones and helicopters. While the nature of the video means we cannot know if a Hellfire missile was ripping up a balloon—or draw any detailed conclusions about what was shown—it's an opportunity for all of us to realize that sometimes, what we think we're seeing in a visual recording may not be what's actually in front of us. Elizabeth Howell (Ph.D., she/her) is one of a few space journalists in Canada. She has written five books, and was Space.com's former staff reporter in spaceflight. Elizabeth holds university degrees in journalism, science and history and also teaches an astronomy course, with Indigenous content, at Canada's Algonquin College. Aside from watching several astronaut missions launching from Florida and Kazakhstan, Elizabeth once lived like an astronaut at the Mars Society's Mars Desert Research Station in Utah. The Army's New Trucks Are a Death Trap, Officer Warns</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68214510/visible-light-speed/'>Scientists Just Made Light Speed Visible. The Images Will Break Your Brain.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The method involves stitching together many thin “slices” of light reflecting off an object. We may earn commission if you buy from a link. Earlier this year, researchers in Austria were able to put on display an effect of special relativity that's been theorized for 100 years. To recreate this effect in an experiment, scientists from Vienna Center for Quantum Science and Technology (TU Wien) and the University of Vienna combined lasers and high-speed photography to turn light speed into something we can witness up close. In the paper, the team—led by corresponding author Dominik Hornof and senior author Peter Schattschneider—explains how physicist Anton Lampa first theorized about these photographic phenomena in 1924. 35 years later, physicists Roger Penrose and Nelson James Terrell separately had the same further insight: a photograph would capture the way the object would appear not condensed (as one might expect), but rotated. In other words, you would go from seeing one face of a cube to seeing two sides and the corner they formed, instead of one distorted single face. “If you wanted to take a picture of the rocket as it flew past, you would have to take into account that the light from different points took different lengths of time to reach the camera,” Schattschneider explained in a TU Wien statement. “This makes it look to us as if the cube had been rotated.” But the speed of light is 299,792,458 meters per second. Our best particle accelerators—like those at CERN or Fermilab—can get close enough, but for various reasons, we can't photograph those particles the right way or observe their shape. Instead, the TU Wien team seemed to take a cue from the iPhone's panoramic mode: What if they sliced an object into micro-thin segments to photograph, then assembled the slices into a composite? To “slice” the object, they used a setup similar to something we usually don't want: the blinding reflection of bright light. In our homes or cars, we shift to avoid the worst of these reflections, but they're exactly what you want when microphotographing light itself. That's like stitching together nearly 150 million photos of your trip to the Grand Canyon. In the meantime, make sure to point your good sides (plural) toward the camera when you're traveling at light speed. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. She's also an enthusiast of just about everything. Fire Patrol Finds Ancient Face Carving in Boulder</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68162812/human-migration-art/'>Etchings on a Wall in the Desert Could Rewrite the Story of Human Migration</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 17:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Nomads who created these monumental images had apparently found a way to survive the arid conditions of the desert, and they left behind images that showed the way to water for those who came after. We may earn commission if you buy from a link. But these animals will never reach the lake oasis they are so eager to drink from, as they are all etched in stone. This was a particularly dry period that offered little relief, save for the winter rains. Archaeologists Maria Guagnin (from the Max Planck Institute of Geoanthropology) and Ceri Shipton (from the Institute of Archaeology at University College London) had not been expecting to make this discovery when they were excavating the site of Jebel Misma in 2023. But when one of Shipton's assistants looked up and shouted jamal (“camel” in Arabic), she noticed something extraordinary. As the morning Sun rose above rock walls a hundred feet high, it brought to light etchings of life-sized camels and other animals on the move. “The primary aim of this research was to test if the rock art panels and archaeological deposits represent earlier occupations during the Holocene humid period, resulting in a condensed chronology of rock art production, or if they belong to earlier periods and represent a longer period of human presence in northern Arabia,” she and Guagnin said in a study recently published in the journal Nature Communications. Shipton and Guagnin were awestruck, realizing that the Pleistocene and early Holocene artists who depicted these creatures in stone would have had to find a foothold somewhere on the sheer sandstone wall and draw everything right in front of them without being able to see the entire picture until they climbed down. There are also several layers of etchings, with some earlier depictions that appeared to have been reinforced. Jebel Misma was not the only site with these monumental images. Later etchings are sometimes superimposed on earlier ones by artists who passed through this region. Some of the animals appear more natural, while others show exaggerated features. Similar art was found at the Jebel Mleiha site. Stone tools found at Jebel Arnaan were luminescence and radiocarbon dated to the Pleistocene-Holocene transition. The etchings may have been intended to mark these routes. Though winter is the rainy season in Saudi Arabia (which can also be inferred by the heavier coats of the camels), it is also mating season (evidenced by how males' necks were swollen from mating vocalizations). “The engravings, which may have been created over a time span of millennia, would have reminded people of ancient symbolisms and beliefs of their group, which likely structured their highly seasonal lives and thus enhanced their ability to thrive in these marginal landscapes,” the researchers said. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. This Ancient Furnace May Have Started the Iron Age</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/home/tools/a68812147/astounding-pop-mech-show-gut-feelings-precognition/'>Déjà Vu Isn't a Glitch, Scientists Say—It's Possibly Your Brain Slipping Back From the Future</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 15:37:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>John Gilpatrick and Andrew Daniels discuss how your consciousness may be jumping through time. Ever get that weird feeling where you just know something's going to happen? Turns out, science might actually have an explanation for it. And it's weirder than anything you've seen in a sci-fi movie. In one of Pop Mech's most-read stories of the year, “Your Consciousness Can Jump Through Time—Meaning ‘Gut Feelings' Are Memories From the Future, Scientists Say,” researchers explore the idea that what we've long dismissed as a gut feeling or coincidence might be called “precognition,” which is when your brain remembers an event before it's happened. Some scientists now believe consciousness doesn't necessarily follow a linear timeline, and that it may be possible to entangle with your own future memories. Watch Pop Mech's John Gilpatrick and Andrew Daniels dig into the wild world of time-slipping minds, including a landmark 1990s study that made scientists start to take precognition seriously. John and Andrew also discuss why your brain may be more like a walkie-talkie to your future self, and what that means for déjà vu, free will, and the fabric of reality itself. The Third State of Life That Begins After Death One Single Ancestor Gave Rise to All Life on Earth</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251003033913.htm'>The invisible chemical in the air that could be raising Parkinson's risk</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 15:05:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Trichloroethylene is a chemical used in metal degreasing, dry cleaning and other industrial applications. Although TCE has been banned for certain uses, it remains in use today as an industrial solvent and is a persistent environmental pollutant in air, water and soil across the United States. The study does not prove that TCE exposure causes Parkinson's disease, it only shows an association. Researchers used Medicare data to identify people over age 67 newly diagnosed with Parkinson's between 2016 and 2018. Each person was compared with five people who did not have the disease. They mapped exposure to outdoor TCE concentrations using U.S. Environmental Protection Agency data and participants' residential neighborhood based on their ZIP +4 location. Each participant's exposure was based on their neighborhood two years prior to diagnosis. Researchers divided participants into 10 groups based on their estimated TCE exposure. Those in the lowest exposure group experienced levels between 0.005 and 0.01 micrograms per cubic meter (μg/m³), while those in the highest group had exposures ranging from 0.14 to 8.66 μg/m³. "This underscores the need for stronger regulations and more monitoring of industrial pollutants." A limitation of the study is that it focused only on Medicare-aged individuals, so findings may not apply to younger people or those with early-onset Parkinson's disease. Note: Content may be edited for style and length. The Pregnancy Pill Millions Trust Faces Alarming New Questions About Child Brain Health Scientists Discover Cancer's “Power-Up” – and a New Way To Switch It Off Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251003033909.htm'>Scientists just cracked the mystery of why cancer immunotherapy fails</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 13:14:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In what experts are calling a paradigm-shifting landmark study, scientists from The Ohio State University Comprehensive Cancer Center - Arthur G. James Cancer Hospital and Richard J. Solove Research Institute (OSUCCC - James) report key findings about the underlying mechanisms of immune system stress response to protein misfolding, launching a new approach to cancer immunotherapy treatment targeting the protein production cycle. For this study, OSUCCC - James researchers sought to answer a long-held question: Why do T cells, which are crucial for fighting infections and recognizing cancer, sometimes become "exhausted" and lose their effectiveness? This overload poisons the T cells, crippling their ability to attack tumors. Strikingly, when researchers blocked key drivers of TexPSR in preclinical models, exhausted T cells recovered their function and cancer immunotherapy became markedly more effective. He emphasized: "Researchers worldwide are tackling T-cell exhaustion through studying genetics, epigenetics, metabolism and others, but the role of protein quality control has been largely overlooked -- until now." The Ohio State cancer research team also found that high levels of TexPSR in T cells from cancer patients were linked to poor clinical responses to immunotherapy. This suggests that targeting TexPSR could be a new way to enhance cancer treatment in the clinics. "When T cells become exhausted, they continue creating molecular weapons but then destroy the weapons before they can do their job," said Yi Wang, first author and doctoral student in Li's laboratory. Li and colleagues report their findings in the latest issue of Nature. The Pregnancy Pill Millions Trust Faces Alarming New Questions About Child Brain Health Scientists Discover Cancer's “Power-Up” – and a New Way To Switch It Off Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03248-5'>First human transplant of kidney modified to have ‘universal' blood type</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 13:08:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Rachel Fieldhouse is a reporter for Nature in Sydney, Australia. Researchers converted a kidney with type-A blood to type-O.Credit: Sherry Yates Young/Science Photo Library Currently, organs from deceased donors can be transplanted into people only if they have a compatible blood type. Organs that are blood type O do not have A or B antigens, so anyone can receive them. The enzyme converts type-A blood into type-O, says study author Stephen Withers, a chemist at the University of British Columbia in Vancouver, Canada. The type-O kidney was then transplanted into a 68-year-old brain-dead man in Chongqing, China. The organ remained healthy for two days before showing signs of rejection. The results are groundbreaking, says Natasha Rogers, a transplant clinician at Westmead Hospital in Sydney, Australia. They could improve access to donor organs and reduce transplant wait lists. If the blood type of the organ was no longer a barrier for transplantation, physicians could focus on things such as matching other antigens unrelated to blood type, which are important in terms of how long a transplant will last, she adds First pig kidney transplant in a person: what it means for the future Pig-organ transplants: what three human recipients have taught scientists Mirror of the unknown: should research on mirror-image molecular biology be stopped? ‘Amazing feat': US man still alive six months after pig kidney transplant Fixing the imbalance in cancer rates between Black and white women Efferocytic remodelling of pancreatic islet macrophages by limited β-cell death Scripps Research Institute is seeking outstanding applicants for a tenure track Assistant Professor position in La Jolla, CA. Seeking senior faculty candidates with experience in interdisciplinary understanding of cognition Robert Wood Johnson Medical School at Rutgers University invites applications The SiRIUS Institute of Medical Research at Tohoku University invites applications for 5 tenure-track Lecturer and Assistant Professor positions. First pig kidney transplant in a person: what it means for the future Pig-organ transplants: what three human recipients have taught scientists An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02355-3'>A genetic map of human metabolism across the allele frequency spectrum</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 12:48:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Genetic studies of human metabolism have been limited in scale and allelic breadth. Here we provide a data-driven map of the genetic regulation of circulating small molecules and lipoprotein characteristics (249 traits) measured using proton nuclear magnetic resonance spectroscopy across the allele frequency spectrum in ~450,000 individuals. Trans-ancestral meta-analyses identify 29,824 locus–metabolite associations mapping to 753 regions with effects largely consistent between men and women and large ancestral groups represented in UK Biobank. We observe and classify extreme genetic pleiotropy, identify regulators of lipid metabolism, and assign effector genes at >100 loci through rare-to-common allelic series. We propose roles for genes less established in metabolic control (for example, SIDT2), genes characterized by phenotypic heterogeneity (for example, APOA1) and genes with specific disease relevance (for example, VEGFA). Our study demonstrates the value of broad, large-scale metabolomic phenotyping to identify and characterize regulators of human metabolism. Our understanding of human metabolism is mostly based on dedicated hypothesis testing in experimental settings, informed by model organisms or observations in patients with rare diseases. Only recently has high-throughput profiling of small molecules in large-scale studies enabled systematic testing of genetic variation across the genome and provided an agnostic approach for discovering genes that encode key metabolic regulators1,2,3,4,5,6,7,8,9,10,11. These efforts have provided important new insights into how genetic variation shapes human chemical and metabolic individuality1 and have corroborated a large body of biochemical knowledge1,2,10,12. The importance of such genome–metabolome-wide association studies (mGWAS) extends beyond the mapping of biochemical pathways, sometimes demonstrating almost immediate clinical value. They provided examples of how readily available supplementation strategies may prevent disease or delay onset in high-risk individuals, such as serine for macular telangiectasia type 2, a rare eye disorder2. They further identified unknown variants that affect the absorption, distribution, metabolism and excretion of exogenous compounds, most importantly drugs1,13, thereby providing pathways to mitigate adverse drug effects. However, there are several challenges that currently limit the potential of mGWAS analyses, particularly for causal inference. These include (1) the still rather small number of, at most, a dozen genetic variants linked to single molecules, (2) the inability to distinguish whether pleiotropic variants act on different molecules or pathways independently (horizontal pleiotropy), or whether they serve as ‘root causes' of successive downstream changes (vertical pleiotropy), (3) the difficulty in distinguishing between locus-specific and metabolite abundance effects when colocalization at disease-risk loci is observed1 and (4) the challenge of confidently assigning effector genes at newly identified loci. Here, we integrated rare (based on whole exome sequencing) and common genetic variation with measures of 249 metabolic phenotypes, including small molecules and detailed lipoprotein characteristics, among >450,000 UK Biobank (UKB) participants representing three distinct ancestries. We demonstrate largely consistent genetic regulation across ancestries and sexes for almost 30,000 locus–metabolite associations and systematically categorize abundant genetic pleiotropy. By integrating machine-learning-derived effector gene assignments with rare exonic variation, we identify previously unknown regulators of metabolism and observe heterogeneity in association profiles for variants mapping to the same gene. Finally, we demonstrate how systematic integration of statistical colocalization and Mendelian randomization can identify pathways with the potential to mitigate cardiovascular disease (CVD) risk beyond current approaches focused primarily on lowering low-density lipoprotein (LDL) cholesterol. We integrated genome-wide association studies (GWAS; population-specific minor allele frequency (MAF) ≥0.5%) with rare exome-wide association studies (ExWAS; MAF ≤0.05%) on plasma concentrations of 249 metabolite phenotypes, quantified using 1H nuclear magnetic resonance (NMR) spectroscopy. We included up to 450,000 UKB participants across three major ancestries (British White European, EUR (n = 434,646); British African, BA (n = 6,573); British Central/South Asian, BSA (n = 8,796)) (Extended Data Fig. The NMR measures comprised 14 lipoprotein subclasses and associated characteristics (that is, extra-large very-low-density lipoprotein (VLDL) to small high-density lipoprotein (HDL) particles), along with small molecules such as amino acids and ketone bodies quantified in molar concentration units (Supplementary Table 1). We identified 29,824 regional sentinel–NMR measure associations in trans-ancestral meta-analyses, representing 753 nonoverlapping genomic regions (Fig. Nearly half of these regions (n = 359, 47%) associated with more than ten NMR measures, demonstrating considerable pleiotropy. Characteristics of large HDL particles, such as particle size and lipid composition, were associated with the largest number of regions (median 166, interquartile range 126–195), compared with all NMR measures (median 105, interquartile range 68–142), findings that considerably extended previous work3 and replicated parallel efforts using UKB9 (Extended Data Fig. Genes with well-characterized roles in human metabolism were significantly enriched across different significance bins (adjusted P values <4.24 × 10−9; Supplementary Fig. 1), suggesting that ever-larger studies of omnigenic traits, such as metabolites, still yield biological plausible findings. a, A top-down Manhattan plot showing trans-ancestral sentinel variants for 249 metabolic phenotypes at a metabolome-adjusted genome-wide significance threshold of P < 2.0 × 10−10. Each row represents an NMR measure, colored for biochemical class. Chromosomal positions are shown on the x axis. P values are raw −log10(P value) from a two-sided Z test across effect estimates derived within three ancestral groups. b, Weighted average allele frequency compared with estimated effect size for trans-ancestral sentinel variants. Points are colored for biochemical classification. c, A comparison of effect sizes between British White European samples (x axis) and British African samples (y axis). We considered variants that were significant in either population. d, Similar to c but comparing British Central/South Asian samples. Dots are colored according to their absolute Z score in British White European samples. We observed significant evidence of heterogeneity (P < 1 × 10−4) across ancestries for very few loci (n = 342; 1.14%), and ancestral-wise comparison of effect estimates demonstrated largely concordant effect estimates (Fig. 1c,d, Extended Data Fig. All sentinels seen in individuals of British African and British Central/South Asian ancestry were replicated in individuals of European ancestry, except for one locus that was specific to British Africans. The previously reported14 missense variant rs3211938 within CD36, which is common among individuals of African ancestry (MAFBA = 0.12) but absent among individuals of European ancestry (MAFEUR = 0.0), was significantly associated (P values <1.49 × 10−10) with lower plasma concentrations of omega 3 fatty acids and 15 other NMR measures, including lipoprotein particle characteristics. This is in line with the role of CD36 encoding for a fatty acid translocase, facilitating the recognition and uptake of long-chain fatty acids. We note that the sample sizes in the smaller ancestral groups did not permit comprehensive replication. While we observed highly correlated effect sizes across female and male participants (median r = 0.98, range 0.90–0.99), we also identified 360 putative sex-differential loci for 239 NMR measures, representing 1,800 heterogenous associations in sex-stratified meta-analyses (heterogeneity P value <5 × 10−8), most of which (65.3%; n = 1,175 loci) could not be explained by confounding factors (Supplementary Note, Supplementary Fig. Putative sex-differential loci were generally directionally concordant between the sexes (Fig. 2a), in line with previous proteomics analyses and suggesting that significant sex interactions do not reflect sex-discordant effects15. a, Comparison of effect sizes of putatively sex-differential loci (defined as loci with heterogeneity P < 5 × 10−8 in a two-sided Z-score meta-analysis across the sexes). b, Rank distributions for each of the five matching NMR traits compared with the Lipids Genetics traits across genetic loci. Per locus–trait combination, 205 lipid-related NMR traits were ranked based on their absolute effect size and compared with the NMR trait that corresponds the Lipids Genetics consortium trait. Pie charts show the percentage of loci where the corresponding NMR trait is ranked among the top 10% of associated traits. TC, total cholesterol; TG, triglycerides. We next used a two-stage strategy to refine regional associations to a smaller number of candidate causal variants. We first identified 3,007 statistically independent metabolite quantitative trait loci (mQTLs) associated with one or more NMR measure, representing a total of 43,322 credible set–NMR measurement pairs (Supplementary Table 5). Lead fine-mapped mQTLs per NMR trait explained on average 6.9% (range 0.57–13.42%) of variance in plasma metabolite concentrations (Extended Data Fig. Second, we leveraged the different linkage disequilibrium (LD) structure in British African and British Central/South Asian individuals to further refine 3,386 credible sets that contained >1 variant and with suggestive evidence in either ancestry, leading to an increase in the number of credible sets with high-confidence variants and decrease in mean credible set size from 9 to 4 variants (Supplementary Note and Supplementary Fig. Trans-ancestral fine-mapping improved resolution in loci that did not resolve in individuals of European ancestry alone, but we note that the overall improvement was marginal. Instead of refining already tight credible sets, future studies should therefore focus on scaling discovery in non-European ancestries to identify unknown causal variants. To assess the value of metabogenomic studies of 1H NMR-spectrometry-based lipoprotein profiling over standard clinical markers, we classified NMR metabolome association profiles for 1,657 genetic variants reported for commonly measured clinical markers (LDL cholesterol, HDL cholesterol, total cholesterol and triglycerides) obtained in 1.6 million people16. Around 25% of associated variants had the corresponding NMR measure among the top 10% of the most strongly associated NMR measures, with 22.5% of genetic variants showing significantly stronger association with refined lipoprotein measures compared with their matching measure on the NMR platform, an observation most pronounced for non-HDL and LDL cholesterol concentrations (Fig. Relevant loci for lipoprotein metabolism can thus be discovered using readily available clinical measurements; however, refined lipoprotein profiles are necessary for better understanding the relevant biological pathways, including any inference about druggability or use for genetic causal inference methods. One such example was the PNPLA3 locus (tagged by rs3747207, associated with LDL cholesterol by the Global Lipids Genetics Consortium; β = −0.014, P = 2.3 × 10−21), where we observed no association with LDL cholesterol (β = −0.001, P = 0.49) but with LDL particle size (β = 0.045, P = 1.04 × 10−73), and multiple characteristics of extra-large VLDL particles (Extended Data Fig. The intronic rs3747207 variant is in strong LD (r2 = 0.98) with the well-known missense variant rs738409 (p.Ile148Met) that has been demonstrated to confer hepatic lipid accumulation by altering ubiquitination of patatin-like phospholipase domain-containing protein 3 (PNPLA3)17. Our results provide human genetic support for a recently proposed role of PNPLA3 in the secretion of large VLDL particles18. We successfully assigned effector genes for almost three-quarters of European ancestry fine-mapped mQTLs (73.6%; n = 2,213) with at least moderate confidence (candidate gene score ≥1.5, range 0–3), including about 28.2% with high-confidence assignments (score ≥2; n = 848), by training a machine learning model that integrates functional genomic resources with pathway information inspired by the ProGeM framework19 (Supplementary Table 6). For example, we prioritized the fatty acid elongase gene ELOVL6 for 16 different VLDL/HDL characteristics (tagged by rs3813829). The gene product, ELOVL fatty acid elongase 6, catalyzes the rate-limiting step in long-chain fatty acid elongation, which are subsequently incorporated into lipoprotein particles. We also prioritized genes with upstream roles in metabolism, including a locus on 17q25.3 where we prioritized cytohesin-1 (CYTH1) as the putative effector gene for 5 independent genetic variants linked to 11 distinct NMR measures mostly comprising characteristics of VLDL particles. CYTH1, previously associated with type 2 diabetes20, promotes activation of ADP-ribosylation factors (ARF)1, ARF5 and ARF6, regulators of lipid vesicle transport, membrane lipid composition and modification21, demonstrating a relevant but indirect link to lipoprotein metabolism. We observed considerable overlap of machine-learning-guided effector gene predictions (top three genes) with those reported based on manually curated biological plausibility (191 out of 283 loci)3 or based on colocalization with protein quantitative trait loci (pQTLs) that have not been used to train the algorithm22 (81 out of 143; Supplementary Table 6). While missing overlap indicates room for improvement, 24 high-confidence assignments strongly disagreed with either external source (gene score > 2 but no match among pQTLs prioritized or manually curated ones). For example, we prioritized PEPD (score 2.42) as opposed to CEBPA3 for rs62102718. PEPD encodes peptidase D, which has been shown to promote adipose tissue fibrosis in mouse knock-out models promoting insulin resistance23. Insulin resistance, in turn, provides a very plausible explanation for the pleiotropic effect of the variant on diverse lipoprotein characteristics (n = 31). Assigned effector genes were significantly enriched in different tissues, reflecting known and lesser-established organ contributions (Extended Data Fig. Genes characteristic of the liver, adipose tissue, adrenal gland and female breast tissue (probably reflecting its high adipose tissue content) were significantly enriched among effector gene sets across the metabolic measures captured by NMR. This included significant enrichment of all amino acids in liver tissue (for example, phenylalanine: odds ratio (OR) 14.8, P < 1.3 × 10−8, histidine: OR 7.9, P < 2.9 × 10−11) but also for skeletal muscle in alanine metabolism (OR 3.82; P < 7.9 × 10−9). Similar enrichments were observed when using the closest gene instead of our annotated effector genes for mQTLs (Extended Data Fig. Pleiotropy is widespread but poorly understood. We developed a framework to characterize four different modes of metabolic pleiotropy (Fig. 3a–d, Extended Data Fig. 7, Supplementary Table 6 and Methods). About half of the pleiotropic mQTLs (n = 880; ≥2 NMR measures) showed evidence for two different modes of vertical pleiotropy. First, within confined pathways (n = 218; ‘pathway pleiotropy'; Fig. 3a) or, second, as a function of the correlation with the ‘lead' NMR measure (n = 662; ‘proportional pleiotropy'; Fig. A prototypical example for proportional pleiotropy was an mQTL tagged by rs624698 for which we prioritized ANGPTL3 as the likely effector gene (Fig. Angiopoietin-like 3, encoded by ANGPTL3, inhibits lipoprotein lipase activity but also endothelial lipase, resulting in increased triglycerides, HDL cholesterol and phospholipid concentrations, consistent with HDL-particle characteristics being the most strongly associated NMR measure (P < 1.0 × 10−546). Other associations reflected downstream effects on lipoprotein metabolism rather than acting on independent pathways (Fig. 3b), considerably expanding previous genetic observations24. a–d, Representative scatterplots opposing the squared trait correlation of the lead NMR measure for the listed variant against the absolute Z score from linear regression models for all associated NMR measures. The colors indicate different modes of pleiotropy and correspond to the legend in e. For each plot, a linear regression fit (lines) with 95% confidence interval (bands) is given. Scatterplots in a–d represent examples of mQTLs classified as pathway pleiotropy (a), proportional pleiotropy (b), disproportional pleiotropy (c) and nonspecific pleiotropy (d). e, The number of associated NMR measures for each of 3,007 mQTL groups opposed to associations reported in the GWAS Catalog after pruning the GWAS Catalog for metabolic phenotypes (Methods). Coloring is according to modes of pleiotropy. f, A scatterplot opposing the number of associated NMR measures (x axis) of each mQTL group with the number of reported EFO parent categories in the GWAS Catalog. g, ORs (rectangle) and 95% confidence intervals (CIs; lines) from logistic regression models testing whether EFO categories (x axis) are more frequently reported for pleiotropic mQTL groups compared with specific ones. Darker colors indicated estimates passing corrected statistical significance. n = 3,007 mQTL groups have been used for enrichment testing. The remaining half of pleiotropic mQTLs showed evidence for two modes of horizontal pleiotropy: those with evidence for ‘disproportional pleiotropy' (n = 68) and a larger group with evidence for ‘nonspecific pleiotropy' (n = 720). For example, a small deletion on chromosome 1 (chr1:92982441:CA>C) was associated with a highly correlated cluster of NMR measures, including characteristics of intermediate density lipoprotein (IDL), LDL and VLDL particles (Fig. 3c), but for which we detected no correlation of association strengths according to the lead NMR measure, the concentration of esterified cholesterol in medium-sized VLDL particles (P < 6.8 × 10−14). We prioritized EVI5 as the most likely effector gene, supported by previous studies on rare functional variants25. The gene product of EVI5, ecotropic viral integration site 5, has no apparent link to (lipoprotein) metabolism, in line with most of the gene assignments for mQTLs with a similar nonspecific pleiotropy pattern. An example of nonspecific pleiotropy was the APOB missense variant rs676210 (p.Pro2739Leu) associated with 126 NMR measures across the entire lipoprotein density range, but also creatinine and glycoprotein acetyl concentrations (Fig. The differential effects of the same genetic variation on distinct lipoprotein subgroups aligns with changes in lipid profiles seen with mipomersen, an antisense oligonucleotide against APOB, that demonstrated reductions in LDL cholesterol but also subsequent increases in the triglyceride content of VLDL particles as hepatic adaption occurs26. Modes of molecular pleiotropy only partially translated into phenotypic pleiotropy (Fig. We observed a twofold enrichment of ‘proportional pleiotropic' (OR 2.11; P < 2.0 × 10−14) and to a lesser extent an enrichment of ‘nonspecific pleiotropic' (OR 1.52; P < 1.1 × 10−5) variants among variants reported in the GWAS Catalog for ≥5 nonmetabolomic trait categories (Methods). By contrast, the set of pleiotropic GWAS Catalog variants was significantly depleted for ‘specific' mQTLs (OR 0.42; P < 1.6 × 10−21). Systemic mechanisms explaining effects of ‘proportional' and ‘nonspecific' pleiotropic mQTLs were further indicated by a more than 20-fold significant enrichment of associated trait categories such as ‘metabolic disease', ‘fatty liver disease' and ‘arterial disorders' (Fig. We next sought to understand convergence of rare and common genetic findings to systematically identify allelic series that increase confidence in causal gene assignment. We identified rare variation (MAF ≤0.05%) in 209 genes to be significantly (P < 1.1 × 10−8) linked to one or more of 249 NMR measures combining ultrarare gene burden analysis (3,709 significant associations; Supplementary Table 8) and rare exonic variant analysis (4,131 significant associations; Supplementary Table 9). Effect sizes were significantly larger compared with more frequent variant effects (Fig. For example, participants carrying rare predicted loss-of-function (LoF) variants in SLC13A5 had more than 1.4 s.d. units higher plasma citrate concentrations per copy of the possibly damaging allele (β = 1.41; P < 2.6 × 10−20). a, Effect estimates against MAF of significantly associated gene burden (diamonds; two-sided P < 1.2 × 10−8 and rare exonic variants (MAF <0.05%; circles; two-sided P < 2.0 × 10−10). b, Effect estimates and two-sided raw −log10(P values) for associations of the rare intronic variant chr11:117186662:C>T within SIDT2 across all 249 NMR measures. The dotted horizontal line indicates the multiple testing threshold (P < 2.0 × 10−10). c, Genomic distance between gene burden (blue) or rare exonic variants (orange) toward the next common credible set variant. d, Evidence for allelic series based on (i) gene burden analysis (bottom), (ii) rare exonic variants (middle) and (iii) common variants with prioritized effector gene matching to the evidence from exonic analysis. For each gene, only the NMR measure most significantly associated with the strongest common variant is shown in cases where multiple NMR measures were associated. Some bars for the number of associated rare exonic variants have been capped to fit into plotting margin, but the number is given in the plot. e, Effect estimates (dots) and 95% CIs (lines) from our European-based exWAS for 7 variants mapping to APOA1 as well as a cumulative burden of high-confidence pLOF variants within APOA1 and bespoke circulating measures of ApoA1 (clinical indicates measurements by immunoturbidimetric analysis on a Beckman Coulter AU5800) and HDL particles (color gradient). f, Top: a heatmap of standardized effect estimates (per variant) across 87 NMR measures for each associated variant and a cumulative burden within APOA1. Variants mapping into the region encoding the protein are surrounded by a rectangle. Variant effects have been aligned to the minor allele. Middle: the corresponding variants mapped to their respective transcripts encoding different forms of APOA1. Bottom: missense variants mapped onto the amino acid sequence of the protein. Variant names colored similarly had highly correlated association profiles. We also observed considerable pleiotropy, including 47 genes associated with 20 or more NMR measures. Many of these genes encode for well-known enzymes and transportes, with nearly half (n = 23/51 genes) being involved in (peripheral) cholesterol metabolism (Extended Data Fig. Some rare pleiotropic variants with large effect sizes (MAF <0.02% and β > 0.6 s.d. units) pointed toward less-established regulators of metabolism, including SIDT2 (chr11:117186662:C>T, n = 124 associated NMR traits), JAK2 (chr9:5073770:G>T (p.Val617Phe), n = 73 associated NMR traits) or CEP164 (chr11:117356670:C>G, n = 49 associated NMR traits). Experimental work already suggested a role for the gene product of SIDT2 (SID1 transmembrane family member 2) in hepatic lipid metabolism and apolipoprotein A1 (ApoA1) secretion, the main protein component of HDL particles, which constituted the majority of associated NMR measures27,28 (Fig. Variation in JAK2 predisposes to somatic mutations inducing hematopoiesis of indeterminate potential (CHIP)29, but other studies linked the gene product Janus kinase 2 (JAK2) to metabolism in liver30, adipocytes31 or macrophages32. The strong inverse association with parameters of HDL particles thereby best aligned with a role of JAK2 in promoting the interaction with ATP-binding cassette transporter A1 (ABCA1) and subsequent HDL-mediated lipid removal from cells, including atherogenic macrophages32. These findings considerably expanded an earlier hypothesis that attributed effects of the same JAK2 variant on LDL cholesterol primarily to myeloid cells in a mouse model33. This hypothesis only partially aligns with—and in some respects contrasts—our human genetic findings across the lipoprotein-density gradient. We observed strong overlap between gene burden and common variant findings, with 85.4% of rare variant (n = 3,528) and 75.5% of gene burden (n = 2,802) associations being <100 kb away from the nearest statistically independent lead credible set variant (Fig. By contrast, most common variant findings (92.3%) were not within 500 kb of matching rare variant/burden evidence. Notably, 12.1% of gene burden results were more than 1 Mb away from the next common credible set variant for the respective NMR measure, aligning with recent observations that both approaches prioritize partly different genes34. At 116 genes (55.5%), rare variant and/or burden evidence overlapped with effector gene predictions for close by common credible set variants (≤200 kb) for one or more associated NMR measure (Fig. 4d), providing independent support for allelic series (Fig. For example, we identified an allelic series composed of seven rare LoF, one gain-of-function and four common variants for serum citrate levels at SLC13A5 encoding a sodium-dependent citrate co-transporter. Another allelic series at ANKH comprised four common variants (rs185448606, MAF 1.3%; rs17250977, MAF 4.0%; rs826351, MAF 44.3%; rs2921604, MAF 45.9%) and a rare missense variant chr5:14745916:T>C (MAF 0.0069%) being also associated with lower serum concentrations of citrate (β = −2.18 s.d. units, P < 5.2 × 10−11) (Fig. ANKH encodes a multipass transporter, recently shown to transport citrate35, with an important role in bone health35. We observed evidence that genetic variants within 17 genes associated with >10 NMR measures had differential metabolic consequences within an allelic series (Supplementary Table 10). The most outstanding example included seven variants (five rare; two common) and a cumulative burden of rare predicted LoF variants at APOA1. They distinctively associated with one or more of 87 NMR measures, most strongly with diverse characteristics of HDL particles of which the gene product, Apolipoprotein A1 (ApoA1), is the major component (Fig. This included four rare missense variants (MAF ≤0.03%) encoded in exon 4 that partly differentially associated with the number, size and cholesterol content of HDL particles (Fig. 4e), only one of which (p.Leu158Pro) primarily associated with serum ApoA1 concentrations and HDL particle number, mimicking the cumulative burden of high-confidence predicted LoF variants in APOA1 and suggesting a potentially dysfunctional protein that lacks interaction with lecithin cholesterol acyl transferase to facilitate cholesterol uptake36. By contrast, p.Lys131del and p.Arg201Ser seemed to rather predispose to a shift in cholesterol content from large towards small HDL particles, a pattern opposed by p.Asp113Glu (Fig. Consistently, amyloid formation by ApoA1 has been observed in early case reports of p.Lys131del (ApoA-IHelsinki37) in which HDL-cholesterol or ApoA1 concentrations are only mildly changed but aggregation of misfolded ApoA1 protein can confer organ damage later in life38. Because p.Asp113Glu and p.Arg201Ser have not yet been identified to cause amyloidosis, we cannot rule out the possibility that each variant maps to distinctive parts of ApoA1 with subsequently different consequences on function and/or stability (Supplementary Fig. While results for serum ApoA1 concentrations were largely confirmed using an alternative assay, we observed some discrepancies that may imply that, in the presence of rare missense variants, the procedure to quantify ApoA1 concentrations from 1H NMR spectra may need recalibration. We observed a >3-fold enrichment of genes previously linked to Mendelian diseases39 (‘OMIM genes') among those associated with NMR measures in gene burden and rare exonic variant analyses (OR 3.30, P < 6.5 × 10−17; Supplementary Table 11), in line with previous mGWAS1,2,7,8. For 15 out of 106 genes, we found evidence of significantly associated disease risk (P < 7.5 × 10−7), largely replicating signs and symptoms of corresponding rare disorders (Supplementary Note and Supplementary Table 12). When we tested more generally whether a rare variant burden in metabolic genes was associated with disease susceptibility, we observed a significant enrichment among susceptibility genes for endocrine and metabolic disorders, such as type 2 diabetes and different lipidemias but not among other disease categories (Supplementary Fig. Genetic predisposition to high LDL cholesterol is strongly associated with increased atherosclerotic CVD (ACVD) risk (‘level effect'), and genetic variations that mimic potent drug targets, such as at PCSK9, show strong evidence of shared effects on both LDL cholesterol and ACVD (‘locus effect')40. To identify potential pathways to mitigate the residual risk not addressed by lowering of LDL cholesterol41, we systematically integrated outcome data across 25 CVD phenotypes42,43,44,45,46,47,48,49,50,51,52,53,54,55,56 with NMR phenotypes (Supplementary Table 13). We identified significant evidence (false discovery rate (FDR) <5%) for 1,146 ‘level effects' across 218 NMR measures with one or more of 22 CVD phenotypes using pleiotropy-curated genetic instruments in Mendelian randomization (Fig. Independently, we observed evidence for 5,527 ‘locus effects', suggesting a shared genetic architecture (posterior probability (PP) >80%) between 87 mQTLs associated with 247 NMR measures and 17 CVD phenotypes (Fig. For 46 NMR–CVD combinations, we found converging evidence for level and locus effects, including 23 not associated in our study with parameters of LDL metabolism (Fig. 5b), providing potential alternatives for addressing residual cardiovascular risk (Supplementary Table 16). a, Summary of two-sample Mendelian randomization analysis testing for putatively causal effects of NMR measures on the risk on diverse CVD. Shown are effect estimates for NMR–disease pairs passing multiple testing. Metabolites are colored according to the scheme from Fig. 1. b, Locus–disease network highlighting loci for which at least 1 NMR measure showed evidence of colocalization with 1 or more CVDs (PP ≥80%). Only loci without evidence for unspecific pleiotropy are depicted. Loci were annotated with the most likely causal gene. Loci colored in blue showed evidence for being associated with LDL cholesterol whereas red did not. c, Dose–response plot for SNPs associated with HDL particle size (after filtering for pleiotropic SNPs) against the risk for CAD. Effect estimates (dots) and 95% CIs (lines) are given and MR-regression lines added. Effect estimates derived from our European ancestry-based GWAS (x axis, n = 434,646) and Aragam et al.42 (y axis, n = 1,589,012). d, Effect of rs4711750 across the NMR metabolome. The y axis is a two-sided raw −log10(P value) derived from the European ancestry-based GWAS (n = 434,646). e, LocusZoom plot centered around VEGFA demonstrating colocalization for the genetic signal for HDL particle size and CAD. The y axis represents the raw −log10(P value) from the European ancestry-based GWAS. For example, we observed robust evidence that, among other measures related to HDL size and composition, genetic susceptibility to larger HDL particle size was associated with a 35% reduced risk of coronary artery disease (CAD; OR 0.65; 95% CI 0.50–0.83; Padj < 0.007; Fig. 5c) along with evidence of a shared and directionally concordant genetic signal at the VEGFA locus (rs4711750, PP 99%; Fig. The locus has been implicated in CAD risk42, and our results now suggest that one likely pathway to modulate CAD risk might be via HDL particle size or characteristics of large HDL particles not captured by HDL cholesterol. Vascular endothelial growth factor A (VEGFA), encoded by VEGFA, is primarily known for its role in angiogenesis57 but has been described as a regulatory factor of transendothelial transport of esterified cholesterol from HDL but not LDL particles via activation of scavenger receptor BI (SR-BI) during reverse cholesterol transport58. Inhibition of VEGFA is a major pharmaceutical target to suppress vascularization of malignant tumors57, and agents targeting VEGF signaling are well known for adverse cardiovascular effects59, suggesting that VEGFA activation, rather than inhibition, might be necessary to reduce CAD risk. Our observations contribute to a growing body of evidence that more tailored approaches, rather than increasing HDL cholesterol content, will probably be needed for potential cardiovascular benefits, given the discouraging trials for most agents increasing HDL cholesterol60. We note, however, that HDL-particle size might still only be a ‘measurable' surrogate, rather than being the true underlying mechanism. For example, inhibition of reverse cholesterol transport via dysfunctional SR-BI increased HDL particle size as well as CAD risk61. Having established pleiotropy categories, we finally aimed to demonstrate its application for nonlipid NMR measures in a disease-wide Mendelian randomization screen (Supplementary Note and Supplementary Table 17). We observed converging evidence for a risk-increasing effect of genetically predicted plasma glycoprotein acetyl concentrations on type 2 diabetes risk (OR per 1 s.d. increase 1.67; P < 3.9 × 10−7) that persisted after exluding variants with evidence for phenotypic pleiotropy (OR 1.69; P < 9.1 × 10−5). This is in line with the rare LoF variant chr20:44413714:C>T (MAF 0.02%) within HNF4A on plasma glycoprotein acetyl concentrations (β = 0.60; P < 8.3 × 10−15) and the cumulative effect of ultrarare LoF HNF4A variants on type 2 diabetes risk (OR 2.68; P = 6.5 × 10−10). However, we note that plasma glycoprotein acetyl concentrations proxy a complex chronic inflammatory state62 that warrants further follow-up analysis to establish mechanistic links to type 2 diabetes. The genetic basis of circulating metabolites provides insights into the complexity of human metabolic regulation and its subsequent influence on health and disease. By integrating common and rare genetic variation with circulating metabolite concentrations in 450,000 individuals from three different ancestries, we provide here a data-driven map of the circulating metabolome across the allele frequency spectrum. This map identifies previously unrecognized modulators of metabolism with potential health implications. By combining machine-learning-guided common variant-to-gene annotation with rare exonic variation, we provided high-confidence effector gene assignments at >100 loci, including some with less established roles in (lipoprotein) metabolism, such as SIDT2, presenting compelling candidates for functional follow-up studies in humans. Large-scale studies similar to ours, but with a broader coverage of the plasma metabolome, will probably uncover more genes with yet undefined roles in metabolism, complementing hypothesis-driven research in experimental models. After more than two decades of GWAS, it has become clear that pleiotropic effects of genetic variants are ubiquitous (see, for example, ref. Little distinction has been possible beyond the generic concepts of ‘vertical' and ‘horizontal' pleiotropy or measures of simple counting. We refine these concepts by observing variants associated with dozens of NMR measures but consistent with the concept of effects diluting or propagating along. Conversely, we observe variants associated with comparatively few NMR measures in an inconsistent pattern, suggesting distinct effects on otherwise highly correlated traits. Our data-driven approach augments previous concepts based on biochemical pathways reporting directionally discordant pleiotropy to discover metabolic bottlenecks64. Disturbance in metabolism or rearrangements thereof are a hallmark of many diseases, including those not classically considered as ‘metabolic', such as eye disorders2, but whether these are pathways for prevention or intervention, rather than a consequence of the disease, often remains elusive in humans. We demonstrated considerable overlap between mQTLs with disease risk loci, including rare-to-common allelic series that can reveal unknown effector genes. However, many such ‘locus effects' were characterized by nonspecific pleiotropy, implicating the plasma metabolite as a bystander rather than cause of the disease. This observation aligns with the relatively few notable exceptions, such as HDL particle characteristics and CAD, from two-sample Mendelian randomization (MR) analyses that contrasted the broad spectrum of observed disease associations described for the same NMR platform65. These observations might be best explained by the concept of metabolic flexibility, which includes built-in redundancy in key pathways to combat various intrinsic and extrinsic perturbations. An important distinction of our study compared with most previous efforts was the availability of highly standardized measurements in a well-designed single large cohort, mitigating influences of preanalytical variables and enabling analyses of even ultrarare variants. However, this also meant that we had little opportunity to investigate the influence of different states of metabolism on our genetic results (such as an overnight fast) or investigate robustness of findings in different environments or at scale in other ancestries. For example, UKB participants were not asked to fast overnight before their baseline visit, which has been shown to impact genetic findings3. Other limitations included the sensitivity and coverage of the 1H NMR platform, and future efforts are likely to reveal more diverse phenotypic consequences of genetically constrained flexibility of human metabolism. Another technical aspect to consider in the interpretation of our results is the indirect nature of 1H NMR derived measurements of certain analytes, including apolipoproteins, that may no longer be reliable in the presence of rare damaging variants that change the properties of apolipoproteins as observed for ApoA1. The UKB is a prospective cohort study from the UK that contains more than 500,000 volunteers between 40 and 69 years of age at inclusion. The study design, sample characteristics and genotype data have been described elsewhere66,67. The UKB was approved by the National Research Ethics Service Committee North West Multi-Centre Haydock and all study procedures were performed in accordance with the World Medical Association Declaration of Helsinki ethical principles for medical research. We included 460,036 individuals across the three major ancestries in UKB in our analyses for whom inclusion criteria (given consent to further usage of the data, availability of genetic data and passed quality control (QC) of genetic data) applied. Data from UKB were linked to death registries and hospital episode statistics (HES). We used the ancestry assignments as defined by the pan-UKB68 and further assigned unclassified individuals to their respective ancestries based on a k-nearest neighbor approach using genetic principal components. All analyses were conducted under UKB applications 44448 and 30418. Up to 249 targeted metabolomic measurements were quantified using the Nightingale NMR platform in human EDTA plasma samples. Detailed experimental procedures for the NMR platform are described elsewhere65,69. The NMR platform covers a wide range of metabolic biomarkers, including lipoprotein lipids, fatty acids and small molecules such as amino acids, ketone bodies and glycolysis metabolites, quantified in molar concentration units. We combine here three data releases that cover the full breadth of the UKB. Metabolomics data were available for 482,276 individuals, including 19,699 samples with data from baseline and repeat visit. Metabolites were reliably detected, with only one biomarker over 2.5% missingness in releases 1/2 (creatinine) and release 3 (3-hydroxybutyrate). Ninety-eight percent of the samples had <5% missingness over all biomarkers in releases 1/2 and release 3. We used the ukbnmr70 R package (v2.2, R v4.3.2) for QC and removal of technical variation in the NMR data. This includes technical confounders such as sample preparation time, shipping plate well, spectrometer effects, time drift within spectrometers and outlier plates. We removed samples that were flagged by Nightingale for poor quality and used the MICE (Multivariate Imputation by Chained Equations)71 R package to impute the remaining dataset. In total, we imputed 0.16% and 0.17% of data in releases 1/2 and release 3, respectively. We observed overall good consistency with the overlapping routine blood biomarkers previously measured in the same cohort (median r = 0.9, range 0.62–0.94) (Extended Data Fig. We sought to adjust the NMR data for medication use, especially cholesterol-lowering medication, to avoid false-positive results driven by medication use in downstream genetic analyses. For male and female participants separately, we fit linear models to quantify the impact of six drug categories on each NMR phenotype: cholesterol-lowering medicine, blood pressure medication, diabetic medication including Metformin usage, oral contraceptive pill or minipill (female only) and hormone replacement therapy (female only) (UKB fields 6177 and 6153) (Supplementary Fig. We used data from individuals with both baseline (NMRbaseline) and repeat (NMRfollow-up) assessment metabolic data available and estimated the effect of medication (med terms) in individuals that did not take any drugs at the time of the baseline visit (n = 6,312 male, n = 6,713 female participants) using the following model: We note that the sample sizes for diabetic medication (nmale = 45, nfemale = 29), oral contraceptive medication (n = 27) and hormone replacement therapy (n = 148) were too small to reliably estimate any effects. Effect estimates for diabetic medication were correlated to estimates for cholesterol-lowering medicine. The effect estimates for blood pressure medication were minimal across the phenotypes. We considered thus only the impact of cholesterol-lowering medicine and corrected the metabolic data in a sex-specific manner. GWAS was performed on 249 metabolic traits measured by the NMR platform on British European (n = 434,646), British Central/South Asian (n = 8,796) and British African participants (n = 6,573) that had complete phenotypic, covariate and genetic information available. We used the Haplotype Reference Consortium-imputed genetic data, including all autosomal chromosomes and the X chromosome. We performed GWAS under the additive model using REGENIE (v3.2.5)72 that uses a two-step procedure to account for population structure. We derived a set of high-quality genotyped variants per population by applying the following filters: (MAF >1%, minor allele count (MAC) >100, missingness rate <10%, PHWE > 1 × 10−15). Furthermore, linkage disequilibrium pruning was performed using a 1,000-kb window, shifting by 100 variants and removing variants with LD (r2) >0.8. We used these variants as input for the first step of REGENIE to generate individual trait predictions using the leave-one-chromosome-out scheme. These predictions are used in the second step where individual variants are tested. Models were adjusted for age, sex and the first ten genetic principal components. We tested variants with a MAF >0.5%, amounting to 11.5 million variants in British European individuals, 11.5 million variants in British Central/South Asian individuals and 19.3 million variants in British African individuals. For initial discovery, we performed a meta-analysis across the three ancestral groups using METAL73. We required variants to be present in at least two ancestral groups. To declare significance, we considered a stringent P-value threshold (2.0 × 10−10) by dividing the standard genome-wide threshold by the number of metabolic phenotypes (5.0 × 10−8/249). We tested our results for genomic inflation and calculated the single-nucleotide polymorphism (SNP)-based heritability using LD-score regression74 (Supplementary Table 19). We used regional clumping (±500 kb) around sentinel variants from the analyses including British European samples to select independent genomic regions associated with a metabolic phenotype and collapsed neighboring regions using BEDtools (v2.30.0). We treated the extended MHC region (chr6: 25.5–34.0 Mb) as one region. Within each region of interest, excluding the MHC region, we performed statistical fine-mapping for all phenotypes associated with that region using the ‘Sum of single effects' model (SuSiE) implemented in the susieR (v0.12.35) R package75. In brief, SuSiE uses a Bayesian framework for variable selection in a multiple regression problem with the aim to identify sets of independent variants each of which probably contains the true causally underlying genetic variant. We implemented the workflow using default prior and parameter settings, apart from the minimum absolute correlation, which we set to 0.1. Because SuSiE is implemented in a linear regression framework, we used the GWAS summary statistics with a matching correlation matrix of dosage genotypes instead of individual-level data to implement fine-mapping (susie_rss()) as recommended by the authors75. To determine the appropriate number of credible sets within each region, we iterated over the maximum credible sets parameter in susieR from two to ten, thus generating fine-mapped results constrained to a range of maximum number of credible sets. For each collection of credible sets, we pruned sets where the lead variant was correlated to the lead variant of other credible sets (r2 > 0.25). After pruning, we considered the fine-mapped results with the largest number of credible sets. We performed several sensitivity analyses by computing joint models per locus–phenotype combination, jointly modeling the effect of all distinct lead credible set variants in a single linear model. Subsequently, we retained only credible sets where the lead variant reached genome-wide significance (P = 5.0 × 10−8) in both marginal and joint statistics. Furthermore, we ensured the estimated coefficients were directionally concordant and of similar magnitude between joint and marginal models (±25%). Linear models were implemented in R using the glm() function and used only unrelated British European participants and the same set of covariates as described above. Finally, we used LD clumping (r2 > 0.6) to identify credible sets shared across metabolic phenotypes. We computed the correlation matrix with LDscore v2.0 using genetic data from 50,000 randomly selected, unrelated White European UKB participants. In situations where SuSiE did not deliver a credible set, we used the Wakefield approximation76 to compute 95%-credible sets. We replicated our trans-ancestral genetic signals using two independent studies: (1) the so-far largest published mGWAS3 and (2) a parallel effort using overlapping UKB data9, both using the same NMR platform. We considered a set of metabolic traits that were directly measured by the NMR platform and not inferred from other traits to avoid multiplicative errors in these more sensitive phenotypes. In total, we were able to match 144 (Karjalainen et al.3) and 169 (Tambets et al.9) metabolic traits, for which we compared sentinel variants that passed metabolome-adjusted, genome-wide significance in our trans-ancestral meta-analysis and that overlapped between the studies. To assign candidate genes for all metabolite QTLs residing outside the MHC region, we first collected annotations for each genetic variant or proxies thereof (r2 > 0.6), including distance to the gene body and putative functional consequences based on the Variant Effect Predictor (VEP) tool offered by Ensembl. We further collated up to ten closest genes within a 2-Mb window and subsequent gene features such as: (1) eQTL evidence for a given variant–gene pair for each tissue available in the eQTL Catalogue release 777; (2) evidence of being annotated as metabolic in the MGI or Orphanet databases as defined in ProGem19; (3) evidence of being listed in the Online Mendelian Inheritance in Man (OMIM) database39; (4) and evidence of being an already assigned drug target in Open Targets78 clinical stages III and IV. With no universally accepted standard for variant-to-gene assignments, we relied on prior biological and genomic information to create three sets of ‘putative true positive' (PTP) set: genes part of cholesterol pathway in the Kyoto Encyclopedia of Genes and Genomes (KEGG)79 or REACTOME80 database (n = 6,791, 722 unique SNPs), lipid pathway (n = 5,670, 603 unique SNPs) and amino acid-related pathway (n = 8,349, 895 unique SNPs). We used all fine-mapped SNPs associated with metabolites classified in the respective NMR metabolite class (Cholesterol: cholesterol, cholesteryl esters, free cholesterol; Lipid: total lipids, other lipids, relative lipid concentration, phospholipids; Amino Acid: amino acid) in the PTP set and used overlapping SNPs in only one PTP set. We trained (7:3 training:test ratio without overlapping variants) a random forest classifier using fivefold cross-validation with subsampling to account for the unbalanced datasets (scikit-learn v1.4.1). We used the balanced accuracy score to choose the best-performing forest from each training set. Subsequently, we used the best-performing classifier from each PTP set to assign candidate scores for all putative effector genes across the entire set of metabolite QTLs. We calculated the median score across classifiers and selected the highest-scoring gene per variant. Within each PTP set, we omitted features used to define true positive sets. Each of the three classifiers exhibited consistent performance (mean ROC-AUC: 0.80, mean balanced accuracy score 0.69) (Supplementary Fig. We used the sum across all three classifiers to assign effector gene scores but present only genes as potential effector genes that reached sufficient support as indicated by largest difference between consecutively prioritized genes. To provide another layer of evidence for assignment of causal genes at metabolic loci, we performed cis-colocalization with protein targets measured in the independent Fenland study22. Cis (for example, gene body ± 500 kb) summary statistics were preprocessed using MungeSumStats81. To relax the single causal variant assumption, we used a colocalization approach where we fine-mapped all traits with SuSiE and then performed colocalization among all credible sets using functionality of the coloc (v5.2.3)82,83 and susieR (v0.12.35)75 R packages. For this, we set the prior probability that a SNP is associated with both traits to 5 × 10−6 and restricted the maximum number of credible sets for the outcome data to five82. We tested whether genes proximal to metabolic loci and assigned effector genes were enriched in tissue compartments by leveraging data from the Human Protein Atlas84. Specifically, we used a two-sided Fisher's test whether metabolic genes were enriched among tissue-specific genes (tissue-enriched or tissue-enhanced as defined by the Protein Atlas) against all protein-coding genes as background. To assign modes of pleiotropy for each mQTL, we first clumped lead credible set variants across NMR measures by LD, collating variants with r2 ≥ 0.6 as a single signal, referred to hereafter as mQTL group. This was done based on dosage files of all unrelated British European UKB participants and implemented with the igraph (v.2.0.1.1) package in R. For each mQTL, we computed pairwise Pearson correlation coefficients among associated NMR measures. We classified each mQTL group on: (1) the 25th percentile of all pairwise correlations, and (2) the Pearson correlation coefficient between the association strengths for each measure (− log10(P value)) and its correlation coefficient with the most strongly associated measure within the mQTL. The latter is a measure to what extent the association between NMR measures at a given locus (‘pleiotropy') can be explained by being correlated with the most proximal associated measure. Based on opposing those two measures for all mQTLs we defined the following five groups: (1) ‘specific' mQTLs associated with only ≤3 highly correlated NMR measures (rho ≥0.6); (2) ‘pathway pleiotropic' mQTLs associated with highly correlated NMR measures (rho ≥0.6) that followed the described association pattern (rho ≥0.6); (3) ‘proportional pleiotropic' mQTL groups associated with, in part, uncorrelated NMR measures but highly correlated association statistics (rho ≥0.6); (4) ‘disproportional pleiotropic' mQTLs associated with highly correlated NMR measures (rho ≥0.6), but without evidence that this translated into a correlation of association statistics (rho <0.6), and; (5) all remaining mQTLs as ‘unspecific pleiotropic' groups. To quantify the extent to which our pleiotropy assignment extends beyond the NMR measures analyzed here, we intersected mQTLs and proxies thereof with results reported in the GWAS Catalog (downloaded 20 May 2024). We first pruned GWAS Catalog entries for those with mapped traits (to minimize double counting), results that met genome-wide significance (P < 5 × 10−8) and had location information available. We further dropped results similar to NMR measures based on broad Experimental Factor Ontology (EFO) terms (for example, EFO:0005105 and child terms indicating ‘lipid or lipoprotein measurement'). To further account for traits mapping to similar categories, we iteratively traced back-mapped EFO terms to broader parent terms. We finally classified mQTLs to be ‘specific' in the GWAS Catalog if they associated with fewer than five parent EFO terms and ‘unspecific' otherwise. We next aimed to investigate the shared genetic basis of the 249 NMR and 25 selected CVD traits. We utilized public databases (GWAS Catalog, openGWAS, CVD-KP) to collect CVD data comprising the largest currently publicly available GWAS datasets on CAD and myocardial infarction, angina pectoris, aortic aneurysm, heart failure and stroke, and peripheral arterial disease, including two to five subtypes for some phenotypes (Supplementary Table 13). Data were harmonized and, if necessary, lifted over to GRCh37 using the MungeSumstats (v1.13.2) R package81. We queried mQTL lead variants and proxies in strong LD (r2 > 0.8; LD backbone based on UKB, as described above) of each NMR trait in each region and corresponding summary statistics for each CVD trait. To investigate ‘locus' effects, we performed statistical colocalization for all combinations of the NMR traits–CVD traits as described before (see ‘Causal gene assignment' section). To estimate ‘level' effects of NMR metabolite concentrations on CVD outcomes, we performed Mendelian Randomization analysis using the TwoSampleMR package (v0.5.1), implementing the inverse-variance weighted and the MR-Egger methods. We used all 249 NMR metabolites as exposure variables, the 25 CVDs as outcome variables and assessed separately four sets of instruments: (1) sentinel variants, (2) lead credible set variants, (3) lead credible set variants restricted for molecular pleiotropy (for example, ‘pathway pleiotropy') and (4) lead credible set variants restricted for both molecular and phenotypic pleiotropy. We used the Wald ratio method to estimate the effect of NMR concentrations on CVD outcomes using only single genetic variants85. We used MR-Egger to test for evidence of a pleiotropic association, an intercept P value >0.0001 indicating evidence of no pleiotropy and checked for concordance between the effect estimates of inverse-variance weighted Mendelian randomisation (IVW-MR), MR-Egger and single genetic variant MR. We controlled the FDR at 5% (ref. To further limit the possible extent of pleiotropic associations, we only reported ‘level effects' passing these filters in the variant sets 2–4, prioritizing the association in the more stringent variant set. The overlap of ‘locus effects' showing no ‘disproportional pleiotropy' according to the section ‘Pleiotropy assignment and overlap with the GWAS Catalog' as well as a significant single variant MR (FDR 5%) and ‘level effects' calculated from metabolite-specific or metabolite- and phenome-specific variants was used to identify gene–metabolite pairs associated with CVD risk independent of LDL metabolism. We considered loci as independent from LDL metabolism if they did not associate with clinical LDL cholesterol at the locus with P < 2.0 × 10−10 and the effect estimate of any variant on clinical LDL-C ranked upward the 80th percentile of all effect estimates at the locus. An in-depth description of whole exome sequencing, including experimental details, variant calling and standard QC measures for the UKB has been extensively reported by Backman et al.87. We performed additional QC steps at the UKB Research Analysis Platform (RAP; https://ukbiobank.dnanexus.com/). We used bcftools (v1.15.1) to process population-level Variant Call Format (pVCF) files. Initially, we normalized the data using the reference sequence GRCh38 build, followed by splitting multiallelic variants. Subsequently, we conducted QC on these variants using a set of parameters outlined below to filter high-quality variants for downstream genetic analyses. Genotypes for SNPs were set to missing if the read depth was less than 7 (or less than 10 for INDELs) or if the genotype quality was below 20. Furthermore, we excluded variants if the allele balance was less than 0.25 or greater than 0.8 in heterozygous carriers. Finally, we excluded variants with missingness >50%. Variants were annotated using ENSEMBL VEP88 (v106.1) with the most severe consequence for each variant chosen across all protein-coding transcripts. We further utilized additional plugins REVEL89, CADD v1.690 and LOFTEE91 for variant annotation. Based on these scores, we defined six partially overlapping variant masks: (1) high-confidence predicted LoF (pLOF, based on LOFTEE and includes stop-gained, splice site disrupting, and frameshift variants); (2) any pLOF assigned high impact by VEP; (3) pLOF and high-impact missense variants (CADD score >20 or REVEL score >0.5); (4) pLOF and any missense variants; (5) only high-impact variants; and (6) any missense variants but not pLOF. We tested synonymous variants separately as a negative control. We tested each mask in different MAF bins, using 0.5% and 0.005% as thresholds. We performed rare variant association testing (RVAT) using whole exome sequencing (WES) data across 249 NMR phenotypes using REGENIE (v3.1.1) via the DNAnexus Swiss Army Knife tool (v4.9.1). Similar to common variant GWASs, we used a two-step approach by REGENIE. We additionally generated step 1 leave-one-chromosome-out (LOCO) files with and without adjusting for common signals via a polygenic score (PGS derived from all lead credible set variant per NMR trait) in the RVAT models per phenotype. All RVAT models were then adjusted for PGS in addition to age, biological sex, fasting duration and the first ten genetic PCs. We first performed aggregated gene burden testing across for 19,026 genes using a set of masks as defined above. For gene burden testing, we used the aggregated Cauchy association test to estimate P values for each gene across masks and allele frequency bins. The aggregated Cauchy association test first computes P values for all sets defined by various masks within a gene and then takes these P values as input to compute one P value for the respective gene via a well-approximated Cauchy distribution. We performed single variant association testing for exonic variants (ExWAS). For the ExWAS, we tested variants with MAC >5 and reported results for variants with MAF <0.0005. We have performed these analyses in individuals of British European, British African and British Central/South Asian ancestry. We considered findings as robust if they passed multiple-testing-corrected statistical significance (gene burden: P < 1.2 × 10−8 (corrected for the number of genes × number of traits); ExWAS: P < 2.0 × 10−10 (same as for common variant GWAS, conventional genome-wide significance corrected for the number of traits)) in both the model with and without adjusting for the common variant PGS and effect sizes did not differ by more than 20% between these models, as this might otherwise indicate that rare variant findings cannot clearly be distinguished from common variant effects. To systematically test for phenotypic consequences of genes identified through rare variant analysis, we collated 626 disease entities following previous work1 by aggregating information from self-report, HES, death certificates and primary care data (45% of the UKB population). Each disease entity had at least one significant common variant, and we used a similar analysis workflow using REGENIE as described for NMR measures but using logistic regression with saddle point approximation. We downloaded the OMIM gene–disease list (9 November 2023) and kept 7,327 unique entries after filtering for gene entries with high confidence (level 3). We computed the enrichment of genes associated with any NMR measure from rare variant or gene burden analysis against a background of 19,989 protein coding genes using Fisher's exact test. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All individual-level data are publicly available to bona fide researchers via the UKB at https://www.ukbiobank.ac.uk/. Full summary statistics for all analyses are publicly available through the NHGRI-EBI GWAS Catalogue (GWAS Catalog identifiers GCST90497044–GCST90501341; see GitHub repository). Code for the main analyses is freely available via GitHub at https://github.com/comp-med/ukb-mgwas and permanently archived via Zenodo at https://doi.org/10.5281/zenodo.14716599 (ref. Surendran, P. et al. Rare and common genetic determinants of metabolic individuality and their effects on human health. Lotta, L. A. et al. A cross-platform approach identifies genetic regulators of human metabolism and health. Karjalainen, M. K. et al. Genome-wide characterization of circulating metabolic biomarkers. Kettunen, J. et al. Genome-wide association study identifies multiple loci influencing human serum metabolite levels. Chen, Y. et al. Genomic atlas of the plasma metabolome prioritizes metabolites implicated in human diseases. Effects of protein-coding variants on blood metabolite measurements and clinical biomarkers in the UK Biobank. Long, T. et al. Whole-genome sequencing identifies common-to-rare variants associated with human blood metabolites. et al. An atlas of genetic influences on human blood metabolites. Tambets, R. et al. Genome-wide association study for circulating metabolites in 619,372 individuals. Preprint at medRxiv https://doi.org/10.1101/2024.10.15.24315557 (2024). Yin, X. et al. Genome-wide association studies of metabolites in Finnish men identify disease-relevant loci. van der Meer, D. et al. Pleiotropic and sex-specific genetic mechanisms of circulating metabolic markers. Khan, A. et al. Metabolic gene function discovery platform GeneMAP identifies SLC25A48 as necessary for mitochondrial choline import. Schlosser, P. et al. Genetic studies of paired metabolomes reveal enzymatic and transport processes at the interface of plasma and urine. Love-Gregory, L. et al. Variants in the CD36 gene associate with the metabolic syndrome and high-density lipoprotein cholesterol. Sex differences in the genetic regulation of the human plasma proteome. Graham, S. E. et al. The power of genetic diversity in genome-wide association studies of lipids. BasuRay, S., Wang, Y., Smagris, E., Cohen, J. C. & Hobbs, H. H. Accumulation of PNPLA3 on lipid droplets is the basis of associated hepatic steatosis. Johnson, S. M. et al. PNPLA3 is a triglyceride lipase that mobilizes polyunsaturated fatty acids to facilitate hepatic secretion of large-sized very low-density lipoprotein. Stacey, D. et al. ProGeM: a framework for the prioritization of candidate causal genes at molecular quantitative trait loci. Nucleic Acids Res. Vujkovic, M. et al. Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis. Donaldson, J. G. & Jackson, C. L. ARF family G proteins and their regulators: roles in membrane transport, development and disease. Pietzner, M. et al. Mapping the proteo-genomic convergence of human diseases. Pellegrinelli, V. et al. Dysregulation of macrophage PEPD in obesity determines adipose tissue fibro-inflammation and insulin resistance. Wang, Q. et al. Metabolic profiling of angiopoietin-like protein 3 and 4 inhibition: a drug-target Mendelian randomization analysis. Hindy, G. et al. Rare coding variants in 35 genes associate with circulating lipid levels—a multi-ancestry analysis of 170,000 exomes. Sjouke, B., Balak, D. M. W., Beuers, U., Ratziu, V. & Stroes, E. S. G. Is mipomersen ready for clinical implementation? A transatlantic dilemma. Chen, X., Gu, X. & Zhang, H. Sidt2 regulates hepatocellular lipid metabolism through autophagy. Sampieri, A., Asanov, A., Méndez-Acevedo, K. M. & Vaca, L. SIDT2 associates with apolipoprotein A1 (ApoA1) and facilitates ApoA1 secretion in hepatocytes. Jaiswal, S. et al. Clonal hematopoiesis and risk of atherosclerotic cardiovascular disease. Sivasubramaniyam, T. et al. Hepatic JAK2 protects against atherosclerosis through circulating IGF-1. Nordstrom, S. M., Tran, J. L., Sos, B. C., Wagner, K.-U. & Weiss, E. J. Disruption of JAK2 in adipocytes impairs lipolysis and improves fatty liver in mice with elevated GH. Dotan, I. et al. Macrophage Jak2 deficiency accelerates atherosclerosis through defects in cholesterol efflux. Liu, D. J. et al. Exome-wide association study of plasma lipids in >300,000 individuals. Weiner, D. J. et al. Polygenic architecture of rare coding variation across 394,783 exomes. The membrane protein ANKH is crucial for bone mechanical performance by mediating cellular export of citrate and ATP. & Kardassis, D. HDL dysfunction caused by mutations in apoA-I and other genes that are critical for HDL biogenesis and remodeling. Tilly-Kiesi, M. et al. ApoA-IHelsinki (Lys107 →0) associated with reduced HDL cholesterol and LpA-I:A-II deficiency. Zanoni, P. & Von Eckardstein, A. Inborn errors of apolipoprotein A-I metabolism: implications for disease, research and development. Amberger, J. S., Bocchini, C. A., Schiettecatte, F., Scott, A. F. & Hamosh, A. O. M. I. M. org: Online Mendelian Inheritance in Man (OMIM®), an online catalog of human genes and genetic disorders. Nucleic Acids Res. Plenge, R. M., Scolnick, E. M. & Altshuler, D. Validating therapeutic targets through human genetics. Hoogeveen, R. C. & Ballantyne, C. M. Residual cardiovascular risk at low LDL: remnants, lipoprotein(a), and inflammation. Aragam, K. G. et al. Discovery and systematic characterization of risk variants and genes for coronary artery disease in over a million participants. Sakaue, S. et al. A cross-population atlas of genetic associations for 220 human phenotypes. Roychowdhury, T. et al. Genome-wide association meta-analysis identifies risk loci for abdominal aortic aneurysm and highlights PCSK9 as a therapeutic target. Roychowdhury, T. et al. Regulatory variants in TCF7L2 are associated with thoracic aortic aneurysm. Miyazawa, K. et al. Cross-ancestry genome-wide analysis of atrial fibrillation unveils disease biology and enables cardioembolic risk prediction. Yu Chen, H. et al. Dyslipidemia, inflammation, calcification, and adiposity in aortic stenosis: a genome-wide study. Global Biobank Meta-analysis Initiative: powering genetic discovery across human disease. Kavousi, M. et al. Multi-ancestry genome-wide study identifies effector genes and druggable pathways for coronary artery calcification. Henry, A. et al. Genome-wide association study meta-analysis provides insights into the etiology of heart failure and its subtype. Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases. Mishra, A. et al. Stroke genetics informs drug discovery and risk prediction across ancestries. Roselli, C. et al. Genome-wide association study reveals novel genetic loci: a new polygenic risk score for mitral valve prolapse. A. et al. Genome-wide analysis identifies novel susceptibility loci for myocardial infarction. van Zuydam, N. R. et al. Genome-wide association study of peripheral artery disease. Adlam, D. et al. Genome-wide association meta-analysis of spontaneous coronary artery dissection identifies risk variants and genes related to artery integrity and tissue-mediated coagulation. Pérez-Gutiérrez, L. & Ferrara, N. Biology and therapeutic targeting of vascular endothelial growth factor A. Nat. Velagapudi, S. et al. VEGF-A regulates cellular localization of SR-BI as well as transendothelial transport of HDL but not LDL. & Cleck, J. N. Adverse effects of anticancer agents that target the VEGF pathway. Tall, A. R., Thomas, D. G., Gonzalez-Cabodevilla, A. G. & Goldberg, I. J. Addressing dyslipidemic risk beyond LDL-cholesterol. Zanoni, P. et al. Rare variant in scavenger receptor BI raises HDL cholesterol and increases risk of coronary heart disease. The biomarker GlycA is associated with chronic inflammation and predicts long-term risk of severe infection. Watanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits. Smith, C. J. et al. Integrative analysis of metabolite GWAS illuminates the molecular basis of pleiotropy and genetic correlation. Julkunen, H. et al. Atlas of plasma NMR biomarkers for health and disease in 118,461 individuals from the UK Biobank. The UK Biobank resource with deep phenotyping and genomic data. Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. Karczewski, K. J. et al. Pan-UK Biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects. Würtz, P. et al. Quantitative serum nuclear magnetic resonance metabolomics in large-scale epidemiology: a primer on -omic technologies. Ritchie, S. C. et al. Quality control and removal of technical variation of NMR metabolic biomarker data in ~120,000 UK Biobank participants. Buuren, S. V. & Groothuis-Oudshoorn, K. mice: multivariate imputation by chained equations in R. J. Stat. Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. Wakefield, J. Bayes factors for genome-wide association studies: comparison with P-values. Kerimov, N. et al. A compendium of uniformly processed human gene expression and splicing quantitative trait loci. The next-generation Open Targets Platform: reimagined, redesigned, rebuilt. Nucleic Acids Res. Kanehisa, M. KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. The Reactome Pathway Knowledgebase 2024. Nucleic Acids Res. Murphy, A. E., Schilder, B. M. & Skene, N. G. MungeSumstats: a Bioconductor package for the standardization and quality control of many GWAS summary statistics. Wallace, C. Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses. Wallace, C. A more accurate method for colocalisation analysis allowing for multiple causal variants. Uhlén, M. et al. Tissue-based map of the human proteome. Burgess, S., Small, D. S. & Thompson, S. G. A review of instrumental variable estimators for Mendelian randomization. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Backman, J. D. et al. Exome sequencing and analysis of 454,787 UK Biobank participants. The Ensembl Variant Effect Predictor. Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Kircher, M. et al. A general framework for estimating the relative pathogenicity of human genetic variants. Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Zoodsma, M., Beuchel, C. & Kohleick, L. comp-med/ukb-mgwas: release v0.1. We acknowledge the Scientific Computing of the IT Division at the Charité – Universitätsmedizin Berlin for providing computational resources that have contributed to the research results reported in this article (https://www.charite.de/en/research/research_support_services/research_infrastructure/science_it/#c30646061). We acknowledge Nightingale Health Plc for access to the UKB NMR biomarker data. We are deeply grateful to the participants, investigators and teams of the UKB and FinnGen studies. Wild for assistance in data processing and helpful discussions. This work was supported by DZHK (German Centre for Cardiovascular Research) and BMBF (German Ministry of Education and Research) grants to C.L. and cofunded by a European Union grant to M.P. (ERC, GenDrug, 101116072) and supported by the Friede Springer Cardiovascular Prevention Center at Charité – Universitätsmedizin Berlin, Germany to A.W. is the British Heart Foundation Chair for Cardiovascular Proteomics (BHF Chair CH/16/3/32406, BHF Programme Grant RG/F/21/110053) and supported by the Imperial BHF Research Excellence Award (4) (RE/24/130023) and the VASCage Research Centre on Clinical Stroke Research, Austria. VASCage is a COMET Centre within the Competence Centers for Excellent Technologies (COMET) programme and funded by the Federal Ministry for Climate Action, Environment, Energy, Mobility, Innovation and Technology, the Federal Ministry of Labour and Economy, and the federal states of Tyrol, Salzburg, and Vienna. COMET is managed by the Austrian Research Promotion Agency (Österreichische Forschungsförderungsgesellschaft) FFG (project number 898252). Views and opinions expressed are, however, those of the authors only and do not necessarily reflect those of the European Union or the European Research Council. Neither the European Union nor the granting authority can be held responsible for them. The funders had no role in the study design, data collection and analysis, decision to publish or preparation of the manuscript. These authors contributed equally: Maik Pietzner, Claudia Langenberg. Computational Medicine, Berlin Institute of Health at Charité – Universitätsmedizin Berlin, Berlin, Germany Martijn Zoodsma, Carl Beuchel, Summaira Yasmeen, Leonhard Kohleick, Aakash Nepal, Alice Williamson, Maik Pietzner & Claudia Langenberg DZHK (German Centre for Cardiovascular Research), partner site Berlin, Berlin, Germany Martijn Zoodsma, Carl Beuchel, Maik Pietzner & Claudia Langenberg Precision Healthcare Institute, Queen Mary University of London, London, UK Mine Koprulu, Alice Williamson, Maik Pietzner & Claudia Langenberg Institute of Genetic Epidemiology, Medical University of Innsbruck, Innsbruck, Austria National Heart and Lung Institute, Imperial College London, London, UK Friede Springer Cardiovascular Prevention Center at Charité, Charité University Medicine Berlin, Berlin, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Maik Pietzner or Claudia Langenberg. The authors declare no competing interests. Nature Genetics thanks Patrick Sulem and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Replication of estimated genetic effects on circulating metabolites in Karjalainen et al.3. Bar plots represent the correlation of effect sizes (top), correlation between the P-value (middle), and the fraction of our sentinel variants that reached genome-wide significance in the replication study (bottom). b, Identical to a but using data from Tambets et al.9. For both comparisons, we only considered directly measured traits. a, b, Cross-ancestry comparison of estimated genetic effects. Comparing estimates (points) obtained within UK Biobank participants of European ancestry (x-axis, n = 434,646) to those of British Central/South Asian ancestry (n = 8,796) (a) or British African ancestry (n = 6,573) (b). Bars denote standard errors of the estimates. Variance explained by fine-mapped lead variants on metabolomic concentrations. Each dot represents a metabolite, colored for biochemical class. Boxplot center refers to the median, bounds are the upper and lower quartiles, and whiskers indicate 1.5× interquartile range. Forest plot showing the strongest associated NMR traits for rs3747207, previously associated with LDL-cholesterol. Stars represent whether traits are significantly differently associated compared to LDL-cholesterol. Effect estimates (dots) and standard errors of the estimate (bars) are taken from the European ancestry-based GWAS (n = 434,646). a, b, Odds ratios for enrichment of assigned effector genes (a) and genes proximal to fine-mapped lead variants (b) across tissue compartments. Columns represent each of the 249 metabolic traits, annotated for biochemical class. Rows and columns were clustered based on Euclidean distance. Odds ratios are derived from a two-sided Fisher's test. a, Scatterplot opposing mQTL characteristics. The x-axis denotes for each mQTL the 25th percentile of all possible correlations among associated NMR measures. The y-axis depicts the correlation between the strongest trait of interest and the association strength for all other traits. A value of one would indicate that all other associated NMR measures can be directly explained as function of correlation, whereas a value of zero would indicate independent effects of the mQTL on different measures. b, Bar plot showing number of variants for each mode of pleiotropy. c-f, Same Pearson correlation networks of NMR measures, clustering highly correlated traits by spatial proximity. Each node is colored according to the strength of associations ( − log10(P-value)) with one of the four genetic variants indicated in the title of each plot. Variants were chosen to represent each of the four modes of pleiotropy. Convergence of gene burden (blue) and common variant (orange) burden results for genes involved in cholesterol metabolism. Comparison of eight metabolic traits measured on the NMR platform (x axis) overlapping with routine blood biomarkers previously measured in the same cohort (y axis). Supplementary Note and Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Zoodsma, M., Beuchel, C., Yasmeen, S. et al. A genetic map of human metabolism across the allele frequency spectrum. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02142-0'>Discovery and artificial intelligence-guided mechanistic elucidation of a narrow-spectrum antibiotic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 12:47:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Current clinical antibiotics are largely broad-spectrum agents that can alter the gut microbiome and promote colonization by Enterobacteriaceae, which are often drug resistant. This includes adherent-invasive Escherichia coli (AIEC), particularly in patients with inflammatory bowel disease, in which dysbiosis creates a niche for this pathogen to colonize. There is an urgent and unmet need for novel narrow-spectrum and microbiome-sparing antibiotics. Here we screened 10,747 bioactive small molecules for antibacterial activity against AIEC and discovered enterololin, an antibacterial compound with targeted activity against Enterobacteriaceae species. Enterololin could overcome intrinsic and acquired resistance mechanisms in clinical isolates when combined with a subinhibitory concentration of SPR741, a polymyxin B analogue used here to increase outer membrane permeability in Gram-negative bacteria. Molecular substructure- and deep learning-guided mechanism-of-action investigations revealed that enterololin perturbs lipoprotein trafficking through a mechanism involving the LolCDE complex, laboratory-evolved resistant mutants predominantly mapped to lolC and lolE, with an in vitro frequency of resistance of ~10−8 to 10−7. Enterololin showed low mammalian cytotoxicity (HEK293 half-maximal inhibitory concentration ~100 µg ml−1) and suppressed AIEC infection in mouse models when administered in combination with SPR741, while largely preserving the overall microbiome composition. This study highlights the utility of deep learning methods for predicting molecular interactions and identifies a promising Enterobacteriaceae-specific antibacterial candidate for further development. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The BIO2 library was assembled by, and is housed in, the CMCB at McMaster University. Researchers may contact the CMCB (https://cmcb-iidr.mcmaster.ca/) for any inquiries. The Sequence Read Archive accession number for sequencing of enterololin-resistant mutants and 16S rRNA sequencing of mouse faecal pellets is PRJNA1282726. The Gene Expression Omnibus accession number for RNA sequencing datasets is GSE300731. Source data are provided with this paper. The DiffDock-L code used in this study can be found via GitHub at https://github.com/gcorso/DiffDock. Maddamsetti, R. et al. Duplicated antibiotic resistance genes reveal ongoing selection and horizontal gene transfer in bacteria. Liu, G. & Stokes, J. M. A brief guide to machine learning for antibiotic discovery. Naghavi, M. et al. Global burden of bacterial antimicrobial resistance 1990–2021: a systematic analysis with forecasts to 2050. O'Neill, J. Antimicrobial Resistance: Tackling a Crisis for the Health and Wealth of Nations. Review on Antimicrobial Resistance (HM Government & Wellcome Trust, 2014). Hawkey, P. M. et al. Treatment of infections caused by multidrug-resistant Gram-negative bacteria: report of the British Society for Antimicrobial Chemotherapy/Healthcare Infection Society/British Infection Association Joint Working Party. Tzouvelekis, L. S., Markogiannakis, A., Psichogiou, M., Tassios, P. T. & Daikos, G. L. Carbapenemases in Klebsiella pneumoniae and other Enterobacteriaceae: an evolving crisis of global dimensions. Ordooei Javan, A., Shokouhi, S. & Sahraei, Z. A review on colistin nephrotoxicity. Li, J. et al. Colistin: the re-emerging antibiotic for multidrug-resistant Gram-negative bacterial infections. Lancet Infect. Nitzan, O., Elias, M., Peretz, A. & Saliba, W. Role of antibiotics for treatment of inflammatory bowel disease. Zangara, M. T., Darwish, L. & Coombes, B. K. Characterizing the pathogenic potential of Crohn's disease-associated adherent-invasive Escherichia coli. Levy, M., Kolodziejczyk, A. & Elinav, E. Dysbiosis and the immune system. Gevers, D. et al. The treatment-naive microbiome in new-onset Crohn's disease. Cell Host Microbe 15, 382–392 (2014). Baumgart, M. et al. Culture independent analysis of ileal mucosa reveals a selective increase in invasive Escherichia coli of novel phylogeny relative to depletion of Clostridiales in Crohn's disease involving the ileum. Morgan, X. C. et al. Dysfunction of the intestinal microbiome in inflammatory bowel disease and treatment. Tang, X. et al. Structural basis for bacterial lipoprotein relocation by the transporter LolCDE. Baron, A. P. et al. Probing the catalytic functions of Bub1 kinase using the small molecule inhibitors BAY-320 and BAY-524. King, A. M. et al. Aspergillomarasmine A overcomes metallo-β-lactamase antibiotic resistance. Ricci, D. P. & Silhavy, T. J. The Bam machine: a molecular cooper. Blair, J. M. & Piddock, L. J. Structure, function and inhibition of RND efflux pumps in Gram-negative bacteria: an update. Cox, G. et al. A common platform for antibiotic dereplication and adjuvant discovery. French, S. et al. Potentiation of antibiotics against Gram-negative bacteria by polymyxin B analogue SPR741 from unique perturbation of the outer membrane. Macnair, C. R. & Brown, E. D. Outer membrane disruption overcomes intrinsic, acquired, and spontaneous antibiotic resistance. Zurawski, D. V. et al. SPR741, an antibiotic adjuvant, potentiates the in vitro and in vivo activity of rifampin against clinically relevant extensively drug-resistant Acinetobacter baumannii. Eckburg, P. B. et al. Safety, tolerability, pharmacokinetics, and drug interaction potential of SPR741, an intravenous potentiator, after single and multiple ascending doses and when combined with β-lactam antibiotics in healthy subjects. Gill, S. R. et al. Metagenomic analysis of the human distal gut microbiome. Liu, G. et al. Deep learning-guided discovery of an antibiotic targeting Acinetobacter baumannii. Breidenstein, E. B. M. et al. SMT-738: a novel small-molecule inhibitor of bacterial lipoprotein transport targeting Enterobacteriaceae. Nayar, A. S. et al. Novel antibacterial targets and compounds revealed by a high-throughput cell wall reporter assay. Lorenz, C., Dougherty, T. J. & Lory, S. Correct sorting of lipoproteins into the inner and outer membranes of Pseudomonas aeruginosa by the Escherichia coli LolCDE transport system. Nickerson, N. N. et al. A novel inhibitor of the LolCDE ABC transporter essential for lipoprotein trafficking in Gram-negative bacteria. Muñoz, K. A. et al. A Gram-negative-selective antibiotic that spares the gut microbiome. Corso, G., Stärk, H., Jing, B., Barzilay, R. & Jaakkola, T. DiffDock: diffusion steps, twists, and turns for molecular docking. Preprint at https://arxiv.org/abs/2210.01776 (2022). Corso, G. et al. Deep confident steps to new pockets: strategies for docking generalization. Preprint at https://arxiv.org/abs/2402.18396 (2024). Fan, J., Fu, A. & Zhang, L. Progress in molecular docking. Bender, B. J. et al. A practical guide to large-scale docking. Bonev, B., Hooper, J. Principles of assessing bacterial susceptibility to antibiotics using the agar diffusion method. Tokuda, H. & Matsuyama, S. I. Sorting of lipoproteins to the outer membrane in E. coli. Nicholson, W. L. & Maughan, H. The spectrum of spontaneous rifampin resistance mutations in the rpoB gene of Bacillus subtilis 168 spores differs from that of vegetative cells and resembles that of Mycobacterium tuberculosis. Cho, S. H. et al. Detecting envelope stress by monitoring β-barrel assembly. Hussein, N. A. et al. Distinct domains of Escherichia coli IgaA connect envelope stress sensing and down-regulation of the Rcs phosphorelay across subcellular compartments. Peters, J. M. et al. A comprehensive, CRISPR-based functional analysis of essential genes in bacteria. Larson, M. H. et al. CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Lacour, S., Bechet, E., Cozzone, A. J., Mijakovic, I. & Grangeasse, C. Tyrosine phosphorylation of the UDP-glucose dehydrogenase of Escherichia coli is at the crossroads of colanic acid synthesis and polymyxin resistance. Jayol, A. et al. Resistance to colistin associated with a single amino acid change in protein PmrB among Klebsiella pneumoniae isolates of worldwide origin. Dößelmann, B. et al. Rapid and consistent evolution of colistin resistance in extensively drug-resistant Pseudomonas aeruginosa during morbidostat culture. Small, C. L. N., Reid-Yu, S. A., McPhee, J. & Coombes, B. K. Persistent infection with Crohn's disease-associated adherent-invasive Escherichia coli leads to chronic inflammation and intestinal fibrosis. Shaler, C. R., et al. Psychological stress impairs IL22-driven protective gut mucosal immunity against colonising pathobionts. Bernstein, C. N. Antibiotic use and the risk of Crohn's disease. Oberc, A. M., Fiebig-Comyn, A. A., Tsai, C. N., Elhenawy, W. & Coombes, B. K. Antibiotics potentiate adherent-invasive E. coli infection and expansion. Hu, L., Benson, M. L., Smith, R. D., Lerner, M. G. & Carlson, H. A. Binding MOAD (Mother Of All Databases). Polizzi, N. F. & DeGrado, W. F. A defined structural unit enables de novo design of small-molecule-binding proteins. Catacutan, D. B., Alexander, J., Arnold, A. & Stokes, J. M. Machine learning in preclinical drug discovery. Wong, F. et al. Discovery of a structural class of antibiotics with explainable deep learning. Gentile, F. et al. Deep Docking: a deep learning platform for augmentation of structure based drug discovery. Arnold, A., McLellan, S. & Stokes, J. M. How AI can help us beat AMR. Swanson, K. et al. Generative AI for designing and validating easily synthesizable and structurally novel antibiotics. Godinez, W. J. et al. Design of potent antimalarials with generative chemistry. Shi, C. et al. GraphAF: a flow-based autoregressive model for molecular graph generation. Preprint at https://arxiv.org/abs/2001.09382 (2020). Hoogeboom, E., Satorras, V. G., Vignac, C. & Welling, M. Equivariant diffusion for molecule generation in 3D. Grisoni, F. et al. Combining generative artificial intelligence and on-chip synthesis for de novo drug design. Swanson, K. et al. SyntheMol-RL: a flexible reinforcement learning framework for designing novel and synthesizable antibiotics. Preprint at bioRxiv https://doi.org/10.1101/2025.05.17.654017 (2025). Mangat, C. S., Bharat, A., Gehrke, S. S. & Brown, E. D. Rank ordering plate data facilitates data visualization and normalization in high-throughput screening. Maggiora, G., Vogt, M., Stumpfe, D. & Bajorath, J. Molecular similarity in medicinal chemistry. & Lal, S. P. Tanimoto based similarity measure for intrusion detection system. Zdrazil, B. et al. The ChEMBL Database in 2023: a drug discovery platform spanning multiple bioactivity data types and time periods. Nucleic Acids Res. O'Shea, R. & Moser, H. E. Physicochemical properties of antibacterial compounds: implications for drug discovery. Wang, R., Fang, X., Lu, Y., Yang, C.-Y. & Wang, S. The PDBbind Database: methodologies and updates. Cheng, H. et al. ECOD: an evolutionary classification of protein domains. Passaro, S. et al. Boltz-2: towards accurate and efficient binding affinity prediction. Preprint at bioRxiv https://doi.org/10.1101/2025.06.14.659707 (2025). Chai Discovery Team et al. Chai-1: decoding the molecular interactions of life. Preprint at bioRxiv https://doi.org/10.1101/2024.10.10.615955 (2024). McNutt, A. T., Li, Y., Meli, R., Aggarwal, R. & Koes, D. R. GNINA 1.3: the next increment in molecular docking with deep learning. Riniker, S. & Landrum, G. A. Better informed distance geometry: using what we know to improve conformation generation. Kolmogorov, M., Yuan, J., Lin, Y. Assembly of long, error-prone reads using repeat graphs. Deatherage, D. E. & Barrick, J. E. Identification of mutations in laboratory-evolved microbes from next-generation sequencing data using breseq. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Calvo-Villamañán, A. et al. On-target activity predictions enable improved CRISPR–dCas9 screens in bacteria. Nucleic Acids Res. Depardieu, F. & Bikard, D. Gene silencing with CRISPRi in bacteria and optimization of dCas9 expression levels. Beal, S. L. Ways to fit a PK model with some data below the quantification limit. Dhaliwal, J. et al. Outcomes following acute severe colitis at initial presentation: a multi-centre, prospective, paediatric cohort study. Quast, C. et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Res. Wickham, H. et al. Welcome to the Tidyverse. Open Source Softw. & Holmes, S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. Lahti, L. & Shetty, S. Microbiome Analytics (Bioconductor, 2017); https://doi.org/10.18129/B9.bioc.microbiome Wickham, H. Reshaping data with the reshape Package. Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. Kuznetsova, A., Brockhoff, P. B. & Christensen, R. H. B. lmerTest Package: tests in linear mixed effects models. Searle, S. R., Speed, F. M. & Milliken, G. A. Population marginal means in the linear model: an alternative to least squares means. Wright, E. S. Using DECIPHER v2.0 to analyze big biological sequence data in R. R J. Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Stephens, M. False discovery rates: a new deal. Keseler, I. M. et al. EcoCyc: fusing model organism databases with systems biology. Nucleic Acids Res. Karp, P. D. Pathway databases: a case study in computational symbolic theories. Karp, P. D. et al. Pathway Tools version 23.0 update: software for pathway/genome informatics and systems biology. We thank K. Koteva for technical assistance with mass spectrometry experiments. We thank E. Brown for providing the SPR741 used in this study. We thank A. Pao and J. Tindall for assistance with mouse model sample processing. We thank J. Szamosi for valuable discussions regarding the statistical modelling and analysis of the mouse faecal microbiome. We thank K. Hotz for valuable feedback on the paper. This study was supported by the Weston Family Foundation (J.M.S. ), the David Braley Centre for Antibiotic Discovery (J.M.S. ), the Canadian Institutes of Health Research (J.M.S. ), the Natural Sciences and Engineering Research Council of Canada (J.M.S.) and a gift from M. and M. Heersink (J.M.S.). was supported by Canada Graduate Scholarship—Master's (Canadian Institutes for Health Research (CIHR)) and an Ontario Graduate Scholarship Award. was supported by Canada Graduate Scholarship—Master's (CIHR) and a Heersink scholarship. A.A. was supported by Canada Graduate Scholarship—Master's (CIHR). was supported by Canada Graduate Scholarship—Master's (CIHR). acknowledge the support of the Abdul Latif Jameel Clinic for Machine Learning in Health and the DTRA Discovery of Medical Countermeasures Against New and Emerging (DOMANE) Threats programme. Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario, Canada Denise B. Catacutan, Vian Tran, Autumn Arnold, Jeremie Alexander, Yeganeh Yousefi, Megan M. Tu, Stewart McLellan, Dominique Tertigas, Jakob Magolan, Michael G. Surette, Eric D. Brown, Brian K. Coombes & Jonathan M. Stokes Michael G. DeGroote Institute for Infectious Disease Research, McMaster University, Hamilton, Ontario, Canada Denise B. Catacutan, Vian Tran, Autumn Arnold, Jeremie Alexander, Yeganeh Yousefi, Megan M. Tu, Stewart McLellan, Dominique Tertigas, Jakob Magolan, Eric D. Brown, Brian K. Coombes & Jonathan M. Stokes David Braley Centre for Antibiotic Discovery, McMaster University, Hamilton, Ontario, Canada Denise B. Catacutan, Vian Tran, Autumn Arnold, Jeremie Alexander, Megan M. Tu, Stewart McLellan, Jakob Magolan, Eric D. Brown & Jonathan M. Stokes Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA, USA Gabriele Corso & Regina Barzilay Farncombe Family Digestive Health Research Institute, McMaster University, Hamilton, Ontario, Canada Yeganeh Yousefi, Dominique Tertigas, Michael G. Surette & Brian K. Coombes Department of Chemistry and Chemical Biology, McMaster University, Hamilton, Ontario, Canada Department of Medicine, McMaster University, Hamilton, Ontario, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the study. conducted the chemical inhibition screen. completed computational and data analysis. performed mechanistic investigations. performed deep learning-based molecular docking training and predictions. performed spectrum-of-activity experiments. conducted cytotoxicity assays. performed intracellular macrophage infection experiments. conducted mouse model experiments. wrote the paper. supervised the study. Correspondence to Jonathan M. Stokes. are consultants for Stoked Bio. is a founder of Stoked Bio. The other authors declare no competing interests. Nature Microbiology thanks Joao Xavier, Lei Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Replicate plot showing screening data of 10,747 small molecules for those that inhibited the growth of AIEC NR5857c in LB medium at 50 µM. (b) Molecular weight distribution of 1,007 molecules from ChEMBL 3365 with annotated antibacterial activity. (c to f), Growth inhibition of AIEC NRG857c (c), K. pneumoniae ATCC 43816 (d), A. baumannii ATCC 17978 (e), and P. aeruginosa PAO1 (f) by SPR741. Experiments were conducted in biological duplicate. Individual replicates with means connected are plotted. Source data Predicted enterololin binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). Pose-10 has confidence −1000 and could not be visualized. Note how the Pose-1 binding site of enterololin is predicted to reside in the transmembrane domain of LolE, which is unlikely due to steric hinderance. Enlarged structure of enterololin highlighting conserved molecular substructures between enterololin and previously discovered LolCDE inhibitors. (a) Predicted LolCDE-IN-1 binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). Pose-9 and Pose-10 had confidence −1000 and could not be visualized. (b) Growth inhibition of AIEC NRG857c by LolCDE-IN-1, with (red square) and without (blue circle) SPR741, in LB medium incubated at 37˚C. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological duplicate with each data point representing optical density measurement at 600 nm. Individual replicates with means connected are plotted. (c) Enlarged structure of LolCDE-IN-1 highlighting conserved molecular substructures between enterololin and other previously discovered LolCDE inhibitors. TSE is shown. Source data (a) Predicted LolCDE-IN-2 binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). Pose-8 to Pose-10 had confidence -1000 and could not be visualized. (b) Growth inhibition of AIEC NRG857c by LolCDE-IN-2, with (red square) and without (blue circle) SPR741, in LB medium incubated at 37˚C. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological duplicate with each data point representing optical density measurement at 600 nm. Individual replicates with means connected are plotted. (c) Enlarged structure of LolCDE-IN-2 highlighting conserved molecular substructures between enterololin and other previously discovered LolCDE inhibitors. TSE is shown. Source data (a) Predicted G0507 binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). Pose-10 had confidence -1000 and could not be visualized. (b) Growth inhibition of AIEC NRG857c by G0507, with (red square) and without (blue circle) SPR741, in LB medium incubated at 37˚C. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological duplicate with each data point representing optical density measurement at 600 nm. Individual replicates with means connected are plotted. (c) Enlarged structure of G0507 highlighting conserved molecular substructures between enterololin and other previously discovered LolCDE inhibitors. TSE is shown. Source data (a) Predicted SMT-738 binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). (b) Growth inhibition of AIEC NRG857c by SMT-738, with (red square) and without (blue circle) SPR741, in LB medium incubated at 37˚C. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological duplicate with each data point representing optical density measurement at 600 nm. Individual replicates with means connected are plotted. (c) Enlarged structure of SMT-738 highlighting conserved molecular substructures between enterololin and other previously discovered LolCDE inhibitors. TSE is shown. Source data (a) Predicted lolamicin binding site in the LolCDE complex embedded in the inner membrane. The labelled region shows the positions of mutations in LolC (light blue) and LolE (pink) enterololin-resistant strains in Pose-1 (highest ranked). Pose-10 had confidence −1000 and could not be visualized. (b) Growth inhibition of AIEC NRG857c by lolamicin, with (red square) and without (blue circle) SPR741, in LB medium incubated at 37˚C. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological duplicate with each data point representing optical density measurement at 600 nm. Individual replicates with means connected are plotted. (c) Enlarged structure of lolamicin highlighting conserved molecular substructures between enterololin and other previously discovered LolCDE inhibitors. TSE is shown. Source data (a to g) Growth inhibition of E. coli BW25113 ∆bamB∆tolC and seven independent enterololin-resistant mutants by a collection of diverse antibiotics. From left to right: E. coli BW25113 ∆bamB∆tolC, Lpp variant (IS 1(+) +8bp), LolC variant 1 (K253Q; 1,171,639A → C), LolC variant 2 (Q258H; 1,171,656G → T), LolC variant 3 (G147D; 1,171,322G → A), LolC variant 4 (E195K; 1,171,465G → A), LolE variant 1 (L60Q; 1,172,954 T → A), and LolE variant 2 (L199P; 1,173,371 T → C). Experiments were conducted in biological duplicate. Note that the enterololin-resistant mutants do not display cross-resistance to other antibiotics. (h) RNA sequencing of E. coli BW25113 ∆bamB∆tolC treated with 5× MIC enterololin for 4 h. Data are the mean of biological triplicates. Transcript abundance in drug-treated samples is normalized to no-drug control cultures grown in identical conditions. Expression differences were quantified with the DeSeq2 pipeline (version 1.40.2) using a two-sided Wald test; P values were adjusted for multiple comparisons with the Benjamini-Hochberg method. We defined the cut-off of log2(FC) ≥ 1.5 and padj ≤ 0.01 to identify differentially expressed genes, represented by the vertical grey and horizontal lines. Note the significant upregulation of genes involved in colonic acid metabolic process as well as downregulation of genes involved in aerobic electron transport chain and transmembrane ion transport. (i) The cell density of E. coli BW25113 ∆bamB∆tolC harboring an empty CRISPRi vector (cyan and dark blue) or an sgRNA targeting LolC (orange and red) without (-aTC) and with induction (+aTc) to knock down lolC. Experiments were conducted in biological triplicate. Individual replicates are plotted with mean as the center line. (j) Growth inhibition of E. coli BW25113 ∆bamB∆tolC harboring an empty CRISPRi vector (cyan) or an sgRNA targeting LolC (orange) by enterololin. Both strains were grown in LB medium without induction. Cell density was normalized prior to growth inhibition assay. Experiments were conducted in biological duplicate. Individual replicates with means connected are plotted. (k) Growth inhibition of E. coli BW25113 ∆bamB∆tolC harboring an empty CRISPRi vector (cyan and dark blue) or an sgRNA targeting LolC (orange and red) with (+aTc) and without induction (-aTc) by LolCDE-IN-1. Induced strains were grown in LB medium with aTc to knock down lolC. Cell density was normalized prior to growth inhibition assay. Experiments were conducted in biological duplicate. Individual replicates with means connected are plotted. (l to r) Growth inhibition of E. coli BW25113 ∆bamB∆tolC harboring an empty CRISPRi vector (cyan and dark blue) or an sgRNA targeting LolC (orange and red) with (+aTc) and without induction (-aTc) by a collection of diverse antibiotics. Experiments were conducted in biological duplicate. Individual replicates with means connected are plotted. (s) Cytotoxicity of MG-132 against HEK-293 cells. HEK-293 cells were treated with MG-132 at concentrations ranging from 0.2 µg ml−1 to 200 µg ml−1. Experiments were performed using two-fold serial dilutions. Experiments were performed in biological triplicate. The mean and individual replicates are plotted. Source data Differentially abundant OTUs (99% clustering) in vancomycin-treated mice. Negative log2FoldChange values indicate OTUs depleted after vancomycin treatment compared to before treatment, and positive log2FoldChange values indicate OTUs enriched after vancomycin treatment compared to before treatment. OTUs that are significantly differentially abundant relative to vehicle (adjusted p-value < 0.05) were visualized. Note the greater number of differentially abundant OTUs, the majority of which are depleted, along with an enrichment of Enterobacteriaceae – patterns not observed in the enterololin-, SPR741-, or combination-treated groups shown in Extended Data Fig. (a to c) Differentially abundant OTUs (99% clustering) in enterololin- (a), SPR741- (b), and combination-treated mice (c). Negative log2FoldChange values indicate OTUs depleted after each monotreatment or combination treatment compared to before treatment, and positive log2FoldChange values indicate OTUs enriched after each monotreatment or combination treatment compared to before treatment. OTUs that are significantly differentially abundant relative to vehicle (adjusted p-value < 0.05) were visualized. Note the fewer number of differentially abundant OTUs, the majority of which are enriched, along with a depletion of Enterobacteriaceae in the combination-treated group – patterns not observed in the vancomycin-treated groups shown in Extended Data Fig. Growth inhibition data against AIEC and GO enrichment for differentially regulated transcripts in E. coli ∆bamB∆tolC treated with 5 × MIC enterololin. Source data for Supplementary Fig. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Fig. Source data for Extended Data Fig. Source data for Extended Data Figs. Source data for Extended Data Fig. Source data for Extended Data Fig. Source data for Extended Data Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Catacutan, D.B., Tran, V., Arnold, A. et al. Discovery and artificial intelligence-guided mechanistic elucidation of a narrow-spectrum antibiotic. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/10/251002074013.htm'>Scientists uncover a mysterious Jurassic lizard with snake-like jaws</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 11:03:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research has uncovered a species of hook-toothed lizard that lived about 167 million years ago and has a confusing set of features seen in snakes and geckos -- two very distant relatives. Breugnathair had snake-like jaws and hook-like, curved teeth similar to those of modern-day pythons, paired with the short body and fully-formed limbs of a lizard. "Snakes are remarkable animals that evolved long, limbless bodies from lizard-like ancestors," said the study's lead author Roger Benson, Macaulay Curator in the American Museum of Natural History's Division of Paleontology. "Breugnathair has snake-like features of the teeth and jaws, but in other ways, it is surprisingly primitive. This might be telling us that snake ancestors were very different to what we expected, or it could instead be evidence that snake-like predatory habits evolved separately in a primitive, extinct group." Breugnathair has been placed in a new group of extinct, predatory squamates called Parviraptoridae, which was previously known only from more fragmentary fossils. The new work on Breugnathair rejects those earlier findings, showing that both snake-like and gecko-like features exist together in a single animal. Breugnathair was discovered in 2016 by Stig Walsh from the National Museums Scotland while on an expedition with Benson and others on the Isle of Skye. "I first described parviraptorids some 30 years ago based on more fragmentary material, so it's a bit like finding the top of the jigsaw box many years after you puzzled out the original picture from a handful of pieces. The mosaic of primitive and specialized features we find in parviraptorids, as demonstrated by this new specimen, is an important reminder that evolutionary paths can be unpredictable." Nearly 16 inches long from head to tail, Breugnathair was one of the largest lizards in its ecosystem, where it likely preyed on smaller lizards, early mammals, and other vertebrates, like young dinosaurs. Because it has such an unusual mixture of features, and because other fossils that shed light on early squamate evolution are rare, the researchers did not arrive at a conclusive answer. Another possibility is that Breugnathair could be a stem-squamate, a predecessor of all lizards and snakes, that independently evolved snake-like teeth and jaws. "However, it makes us even more excited about the possibility of figuring out where snakes come from." The Pregnancy Pill Millions Trust Faces Alarming New Questions About Child Brain Health Scientists Discover Cancer's “Power-Up” – and a New Way To Switch It Off Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/an-anechoic-chamber-at-nokia-bell-labs-reveals-the-hidden-sounds-of-your/'>A Silent Chamber at Nokia Bell Labs Reveals How Sound Shapes Memory, Emotion and Human Experience</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Go Inside a Room That Lets You Hear Your Nervous System By Rachel Feltman, Jeffery DelViscio, Kelso Harper, Fonda Mwangi & Naeem Amarsy Science Quickly host Rachel Feltman stands inside the Nokia Bell Labs anechoic chamber in Murray Hill, N.J. Seth Cluett: This experiment just pops a balloon. And normally, when a balloon would pop, you'd hear [makes an exploding noise]—the whole room just kind of expand, right? If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Cluett: But in this room there's none of that. So you're gonna hear it as a very sharp sound that just disappears completely. Cluett: Welcome to the anechoic chamber. Wow, it is already super quiet in here [laughs]. Cluett: And it's gonna get even more quiet when we close the door. Feltman: It did get a lot more quiet, yeah [laughs]. Inside one of the quietest rooms in the world, host Rachel Feltman meets artist-in-residence Seth Cluett at the historic anechoic chamber at Bell Labs to explore the science of silence and sound perception. And you may notice if you're listening to this, or if you're watching it, that there's some interesting stuff going on with the sound. Seth, would you tell us more about why that is? Cluett: Yeah, so we are in the historic anechoic chamber at Bell Labs. It is a room that absorbs 99.999 percent of sound-wave propagation and eliminates sound from the outside almost entirely. So an anechoic chamber is intended to absorb as close to 100 percent of incidental reflection as you can possibly do. The building is a shell that has [roughly] two-foot-thick walls and an air gap and then another wall, and that air gap separates the sound waves from the outside. And then inside the room there are these [roughly] four-foot wedge panels in groups of three in an offset kind of orthogonal pattern—a, a grid pattern. So they, they come to a kind of inverted point in the inside of the wall, and when sound gets into that point it kind of bounces back and forth on the diagonal, and by the time it gets to the outside of the wedge there's no more sound energy left to reflect. Feltman: Mm, so when you're in this totally quiet space what kinds of things can you feel and perceive in your body? Cluett: Yeah, I think most people feel a kind of pressure against their ears first. Cluett: It sort of feels like the air is heavy or thick. Then, then you start to notice a kind of low pulse and realize that's your heart beating and you can hear it in the air, in, in addition to through your body. And for most people you can even hear a high pitch, and you think, “Oh, I have tinnitus,” or “There's something wrong with my hearing.” You're actually hearing your nervous system ... Cluett: Bone conduction through your, through your skull, and it's quiet enough that for the first time you can hear a part of your body that's been along with you for the whole time you've been alive. Harper: You're just hearing your nervous system ... DelViscio: Listening to your nervous system .... Cluett: Hypersensitive part of your stereocilia on your basilar membrane that are like, “Wake up! Could you show us some demonstrations to maybe help our listeners and viewers understand, like, how unique this space is? You've also let us borrow a very fancy mic so that we can present 360 stereo audio so that our, our listeners and viewers can, you know, experience being in the space as, as closely as we can approximate. It's subtle, and I hope that your viewers are wearing headphones. Feltman: So how does this demonstration work? Cluett: Okay, so in order to demonstrate how much of the sound the room absorbs, I'm gonna sing a sinusoid, a pure tone, straight ahead. You're gonna hear that as the loudest, most direct sound. I am not gonna change the volume of my sound at all, but I'm gonna turn around in a circle, and so as I turn you're gonna hear the room absorbing more and more of my sound. [Starts singing and then stops to clear his throat.] Cluett: [Clears his throat and then starts singing while slowly moving in a circle.] Feltman: So how does this one work? Cluett: Okay, so one of the kind of miraculous things about an anechoic chamber is it allows you to hear something that you can't hear outside of the, of the chamber. And that's that for every doubling of distance a sound source has to the listener there's a cutting in half of the volume. Cluett: And so, and so I'm gonna speak to you at this level, and then I'm gonna double the distance, and you're going to hear the same timbre, the same quality, the same emotional content as I'm talking now but just at half the volume. And then I'll double it again, and you'll hear even more of a falloff in the, in the volume. Cluett: Okay, so this is me talking at the normal volume. And I'm going to move back double. And this is me talking at that same volume; it's just half as loud for you. And this is me talking at the same volume, and it's half as loud again. Cluett: It's kind of overwhelming, honestly. The “bit” of digital binary was invented here, and as an extension Claude Shannon and John Pierce invented pulse-code modulation, which is the way we record sound digitally. Even The Jazz Singer, the first [feature-length] sound film [with synchronized sound for dialogue sequences] ... Cluett: The technology for that was developed by Western Electric—Bell Labs, how Bell Labs was founded, making it possible for us to synchronize sound and image for film. Feltman: And what kind of experiments have happened in this chamber since then? Like, we think about the Touch-Tone phone ... Cluett: And the acoustic research paired with psychology, or what we call psychoacoustics, was done in this room in order to, you know, help people figure out how to memorize the most phone numbers as the nation's telephone grid got larger and larger ... Cluett: And it wasn't just, you know, “Bronson 257,” right? So in addition to that, you know, there was a, a fundamental research into synthesizing the sound of the human voice ... We talk about microphone research and loudspeaker research, but that's not the entirety of it. Behind that is signal-processing research, that's about: How do you optimize a signal to go long-distance? And at first that was: How do you optimize a signal to go over a long run of copper wire, right? And now in wireless: How do you, how do you encode a signal to minimize the amount of data taken up and maximize the quality of the voice? You know, we think of science as being hard measurements and, and facts, and one of the things that's remarkable about Bell Labs in the, in the '60s and '70s was human-factors research, like: How does it feel to use technology? Cluett: And one of the convincing things in telephony was: How do you make sure, as you're reducing the signal, that you're not throwing away emotion ... Cluett: That you're not throwing away recognition, that you can still recognize the person that you love on the other end of the line? And I think one of the things that this room actually shows us, and the audio for this podcast will actually reveal a little, is that the digital silence in cell phones is, is actually really dehumanizing ... So I know that you are an artist-in-residence here. Feltman: How did you get involved with the lab? Cluett: Sure, so I've been here since 2017, so this is my eighth year in the artist-in-residence role. When I first arrived I came in as a re—as part of a reboot of the historical Experiments in Art and Technology program, which was founded in the 1960s. And then in 2014 there was a reboot of, of, like, the role of artistic research at Bell Labs in, in one form or another. And because so many media technologies intersect with scientific research, the idea was: Let's bring in artists, left-brain thinkers, to, to ask kind of divergent questions in a linear engineering space. And so when I first came I was very active, moving lab to lab, asking questions of engineers and, and technical staff of what their research was and getting them to, to enter a dialogue to try to see what sort of divergent left-brain thinking might contribute to a kind of linear engineering research and development space. So what kind of stuff have you been working on? Cluett: So I've been really interested in what I'm calling “the last 10 percent,” which is, like, in psychoacoustic research so much of the things that we know about the sound world were developed either for telephony or for the military ... Cluett: So: Can we get a signal efficiently across a line, or can we triangulate something in space—so sonar and radar and those sorts of things? In virtual reality and in artificial intelligence, now the question is: How do we pass the kind of uncanny valley of, like, this is almost real, but it's not, right? What I'm interested in is: What are the aspects of sound that are kind of soft factors, that are the human things that ... The auditory equivalent of depth perception ... Cluett: Was something that was never, you know, looked at seriously in the initial sort of foundations research. And so I've been doing experiments around loudspeaker-array design and microphone-array design to try to think about what is hiding in the signal that our brain is processing that might not appear in the physical acoustics that give us a real sense of belonging—like, a sense of space. Feltman: Mm, and I know that in addition to the kind of experiments that have been done here to contribute to technological advancements, there's also been just sort of a lot of interesting art projects that have happened here ... Feltman: What are some of the, like, stranger, more interesting things that have been done in this room? I mean, a lot of people don't realize that the first time a computer sang was at Bell Labs, right? Cluett: So at the end of 2001: A Space Odyssey, when HAL is singing—is dying ... Cluett: That was heard by Arthur C. Clarke when he did a tour of Bell Labs ... Cluett: And then was incorporated into the movie because it was the first time a computer sang, so the idea of giving the computer some human character came from trying to figure out speech-synthesis problems. Because if we could understand the nature of the way that speech was synthesized digitally, we could initiate the revolution of automated telephone answering, right ... Cluett: Like, or the synthesized voices that AI responds with. All of that foundational research was done in spaces like this, and it was done first by asking a computer to sing. And I know somebody brought a grand piano in here. We learned that as reassurance that the wire floor was not gonna give out underneath us [laughs]. Because, you know, I feel like acoustically, this is objectively not great ... Cluett: I wasn't around for the grand-piano experiments, but at the start of my residency I—we had a artist-in-residence partner, collaborator, the International Contemporary Ensemble, and we brought a 12-person chamber orchestra into this room ... So we could dial in Carnegie Hall ... And so that ability to, like, “act” on signals that haven't yet been touched by space ... Cluett: Helps us understand how to process the signals that give us a sense of, like, where sound is coming from. Feltman: I feel like a lot of the sort of mainstream conversation about anechoic chambers is that it feels really freaky to be in them ... And I'm also curious: How do you feel about being in the space? One of the remarkable things about this room, and the, you know, the people who are doing the recording around us right now are beginning to experience it, when you're in here for 20 minutes, your fight-or-flight response that's active all the time—like, if you step your foot off of a curb and you hear a bicycle messenger ding their bell and you pull your foot back automatically ... Otherwise, we'd be, you know, subject to the same kind of sensory-processing disorders that, that, you know, are so common, right? When you're in here for 20 minutes, the little hair cells on your basilar membrane inside of your ear relax, and it gets 10 percent quieter. Where's the stimulus?” And that feels panicking, in the same way as, like, “Why hasn't anybody responded to my text?” Cluett: And I think if you just—if you change your perception to a longer arc of, “Where is my body right now?” you start to realize, like, “My heart rate's lowered, lymphatic production slowed down. I hold two roles: I'm the artist-in-residence at Nokia Bell Labs, and I'm the director of the historic Computer Music Center at Columbia University. And in both of those places I'm interested in the same thing: what I'm calling sort of “sensory computing” or “speculative acoustics.” Like, this idea that there's—there—it's important for computers to not just process text and language but for computers to understand the world around them with the same level of resolution that we do, right? Cluett: If we want to expect intelligence out of machines, we don't just know things from memorizing things off of—out of books and off the Internet. We know things by walking around the world. I think the solution to so many of the, of the questions of, like, what the next 100 years is going to feel like in terms of the way we interact with computers are gonna come down to how embodied we feel and then, as an extension, how safe we feel around the computer. Feltman: So I understand that this facility is, is gonna be closing down in the next few years. Cluett: I'm, I'm tempted to say something sort of profound about my own practice and, and, and its effect on my practice, which has been sort of immeasurable, right? Like, having the ability to record an absolutely dry audio for someone who composes electronic music is, like, unmeasurably positive, right? But for me the real impact has been watching and leading dozens of school groups through, from elementary school through college, students from Harvard and students from Rutgers, students from Ramapo College and students from, from Carnegie Mellon, coming in and for the first time experiencing something absolutely new, right? Cluett: And I think for a lot of people it's a real revelation that maybe, maybe technology writes on us a little bit too much, and it's time to kind of shake the Etch A Sketch gray and start writing over again. Feltman: Well, thank you so much for taking the time to chat with us and for, you know, showing us around, telling us how this room works. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. He is former director of multimedia at STAT, where he oversaw all visual, audio and interactive journalism. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He holds dual master's degrees in journalism and in Earth and environmental sciences from Columbia University. He has worked onboard oceanographic research vessels and tracked money and politics in science from Washington, D.C. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. Kelso Harper is an award-winning senior multimedia editor at Scientific American. As a producer, editor and host, they work on short documentaries, social videos and Scientific American's podcast Science Quickly. They have a bachelor's degree in chemistry from Johns Hopkins University and a master's degree in science writing from the Massachusetts Institute of Technology. Previously, they worked with Wired, Science, Popular Mechanics, and MIT News. Follow them on LinkedIn and Instagram. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. She holds a master's degree in journalism and public affairs from American University in Washington, D.C. Naeem Amarsy is a documentary filmmaker and multimedia editor based in New York City. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. SciAm always educates and delights me, and inspires a sense of awe for our vast, beautiful universe. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today. Scientific American is part of Springer Nature, which owns or has commercial relations with thousands of scientific publications (many of them can be found at www.springernature.com/us). Scientific American maintains a strict policy of editorial independence in reporting developments in science to our readers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03201-6'>AI has designed thousands of potential antibiotics. Will any work?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 09:51:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Rachel Fieldhouse is a reporter for Nature in Sydney, Australia. In minutes, AI can design thousands of chemical compounds with potential antibacterial properties, although there are hurdles to overcome before the first of these medicines can be tested in people. Last week, the US Centers for Disease Control reported that rates of dangerous bacterial infections surged by 69% between 2019 and 2023. The standard method of antibiotic discovery involves going into nature and sifting through dirt to find antibacterial compounds, says César de la Fuente, a machine biologist at the University of Pennsylvania, Philadelphia. His team has been using AI to discover antibiotics for about a decade. The whole process of discovering a candidate, creating it in the laboratory and testing it in cells can be done “within a week or two”, he adds. Machine learning and generative AI (genAI) can speed up the process. The AI designs antibiotics by looking for fragments of proteins with antibacterial properties in data sets that it has not seen before. Earlier this month, de la Fuente and his team reported that their genAI model designed 50,000 peptides, or short chains of amino acids, that have antimicrobial properties and can destroy pathogens1. A deep-learning model then ranked these on the basis of how effective it thinks they might be at killing several bacteria types. Of the top 46 synthesized peptides, about 35 killed at least one bacterial strain in a dish, and most were not toxic to human embryonic kidney cells. The top two candidates were then tested and found to be effective against Acinetobacter baumannii in mouse models. But making AI-designed antibiotics in the lab can be challenging. For example, bioengineer Jim Collins, at the Massachusetts Institute of Technology in Cambridge, has found that some AI-designed antibiotics are chemically unstable and cannot be synthesized. New antibiotic that kills drug-resistant bacteria discovered in technician's garden ‘Smart' antibiotic can kill deadly bacteria while sparing the microbiome Antibiotic use in farming set to soar despite drug-resistance fears A scientist's guide to AI agents — how could they help your research? Scripps Research Institute is seeking outstanding applicants for a tenure track Assistant Professor position in La Jolla, CA. Seeking senior faculty candidates with experience in interdisciplinary understanding of cognition Robert Wood Johnson Medical School at Rutgers University invites applications The SiRIUS Institute of Medical Research at Tohoku University invites applications for 5 tenure-track Lecturer and Assistant Professor positions. New antibiotic that kills drug-resistant bacteria discovered in technician's garden Antibiotic use in farming set to soar despite drug-resistance fears An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251002074005.htm'>Scientists just found the shocking reason Chile's quake shook so hard</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-03 03:22:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In July 2024, a 7.4-magnitude earthquake struck Calama, Chile, damaging buildings and causing power outages. The country has endured violent earthquakes, including the most powerful recorded in history: a 9.5-magnitude "megathrust" event that struck central Chile in 1960, causing a tsunami and killing between 1,000 to 6,000 people. Earthquakes this deep usually produce much more subdued shaking on the surface. In a recent study in Nature Communications the researchers describe a newly-discovered chain of events that was responsible for increasing the earthquake's intensity. "These Chilean events are causing more shaking than is normally expected from intermediate-depth earthquakes, and can be quite destructive," said the study's lead author Zhe Jia, a research assistant professor at the UT Jackson School of Geosciences. Intermediate-depth earthquakes, such as the one in Calama, were long thought to occur due to pressure building up as the rock dried out - a phenomenon called "dehydration embrittlement." This dehydration process is typically thought to stop where temperatures exceed 650 degrees Celsius. But according to the researchers, the Calama quake was so powerful because it breached this limit - going 50 kilometers deeper into hotter zones through a second mechanism called "thermal runway." "It's the first time we saw an intermediate-depth earthquake break assumptions, rupturing from a cold zone into a really hot one, and traveling at much faster speeds," said Jia, who is part of the University of Texas Institute for Geophysics (UTIG), a research unit of the Jackson School. "That indicates the mechanism changed from dehydration embrittlement to thermal runaway." Becker and Jia said that learning more about how earthquakes occur at different depths could help with understanding what controls the size and nature of likely future events, which could help predict the degree of shaking and inform infrastructure planning, early warning systems, and rapid response systems. Note: Content may be edited for style and length. The Pregnancy Pill Millions Trust Faces Alarming New Questions About Child Brain Health Scientists Discover Cancer's “Power-Up” – and a New Way To Switch It Off Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            