
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-05</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09721-5'>Atomically accurate de novo design of antibodies with RFdiffusion</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 16:22:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite the central role of antibodies in modern medicine, no method currently exists to design novel, epitope-specific antibodies entirely in silico. Instead, antibody discovery currently relies on immunization, random library screening or the isolation of antibodies directly from patients1. Here we demonstrate that combining computational protein design using a fine-tuned RFdiffusion2 network with yeast display screening enables the de novo generation of antibody variable heavy chains (VHHs), single-chain variable fragments (scFvs) and full antibodies that bind to user-specified epitopes with atomic-level precision. We experimentally characterize VHH binders to four disease-relevant epitopes. Cryo-electron microscopy confirms the binding pose of designed VHHs targeting influenza haemagglutinin and Clostridium difficile toxin B (TcdB). A high-resolution structure of the influenza-targeting VHH confirms atomic accuracy of the designed complementarity-determining regions (CDRs). Although initial computational designs exhibit modest affinity (tens to hundreds of nanomolar Kd), affinity maturation using OrthoRep3 enables production of single-digit nanomolar binders that maintain the intended epitope selectivity. We further demonstrate the de novo design of scFvs to TcdB and a PHOX2B peptide–MHC complex by combining designed heavy-chain and light-chain CDRs. Cryo-electron microscopy confirms the binding pose for two distinct TcdB scFvs, with high-resolution data for one design verifying the atomically accurate design of the conformations of all six CDR loops. Our approach establishes a framework for the computational design, screening and characterization of fully de novo antibodies with atomic-level precision in both structure and epitope targeting. Antibodies are the dominant class of protein therapeutics, with over 160 antibody therapeutics currently licensed globally and a market value expected to reach US$445 billion in the next 5 years4. Antibody development generally proceeds in two stages: (1) the discovery of antibodies that bind to a specific epitope; and (2) the subsequent affinity maturation and clinical optimization of those antibodies. Currently, identifying epitope-specific antibodies relies on animal immunization or screening of antibody libraries to identify candidate molecules that bind to a desired target, followed by subsequent epitope mapping. These methods are laborious, time-consuming and can fail to identify antibodies that interact with the therapeutically relevant epitope1. Efforts at computational design of antibodies have generally focused on the second optimization step of antibody development, such as sampling alternative native CDR loops to improve affinities5,6 or using Rosetta7 sequence design to improve the interacting regions. More recently, structure-based and sequence-based deep learning networks have been trained to design novel antibody sequence variants8,9,10, but these methods require an initial binding antibody from which to optimize. There have also been recent advances in antibody optimization with deep learning methods trained on data generated by powerful new experimental methods11,12. By contrast, computational methods able to perform the first stage of antibody design (generating epitope-specific binding antibodies) do not exist, and de novo (no homology to an existing antibody targeting that epitope) design of antibodies therefore remains an unsolved problem. There has been rapid progress in designing binding proteins (not antibodies) using RFdiffusion2,13,14. However, as with other methods for de novo interface design15,16,17, these binders almost exclusively rely on regular secondary structure-based (helical or strand) interactions with the target epitope, and the original (‘vanilla') RFdiffusion network is therefore unable to design antibodies de novo (Supplementary Fig. An ideal method for designing de novo antibodies would enable: (1) targeting of any specified epitope on any target of interest; (2) focusing of sampling on the CDR loops, keeping the framework sequence and structure close to a user-specified highly optimized therapeutic antibody framework; and (3) sampling of alternative rigid-body placements of the designed antibody with respect to the epitope. We hypothesized that a specialized version of RFdiffusion fine-tuned on antibody structures should be capable of designing de novo CDR-mediated interfaces, given the diversity and quality of de novo interfaces that RFdiffusion can design and given that the underlying thermodynamics of interface formation are the same, and set out to develop such a method. 19) and RF2 frame representation of protein backbones comprising the Cα coordinate and N-Cα-C rigid orientation for each residue. During training, a noising schedule is used that, over a set number of ‘timesteps' (T), corrupts the protein frames towards random prior distributions (Cα coordinates are corrupted with three-dimensional Gaussian noise, and residue orientations with Brownian motion on SO3). During training, a Protein Data Bank (PDB) structure and a random timestep (t) are sampled, and t noising steps are applied to the structure. RFdiffusion predicts the de-noised (pX0) structure at each timestep, and a mean squared error loss is minimized between the true structure (X0) and the prediction (pX0). At inference time, a random residue distribution (XT) is sampled, and RFdiffusion iteratively de-noises this to generate novel protein structures. We fine-tuned RFdiffusion predominantly on antibody complex structures (Fig. At each step of training, the antibody structure is corrupted. To permit specification of the framework structure and sequence at inference time, the framework sequence and structure are provided as conditioning input to RFdiffusion during training (Fig. Because it is desirable for the rigid-body position (dock) between antibody and target to be designed by RFdiffusion along with the CDR loop conformations, the framework structure is provided in a global-frame-invariant manner during training (Fig. We utilize the ‘template track' of RF2/RFdiffusion to provide the framework structure as a two-dimensional matrix of pairwise distances and dihedral angles between each pair of residues (a representation from which three-dimensional structures can be accurately recapitulated)20 (Supplementary Fig. The framework and target templates do not encode their relative positions in the three-dimensional space. We trained RFdiffusion with an additional one-hot encoded ‘hotspot' feature, which provides some fraction of the residues that the antibody CDRs interact with, such that at inference, we can direct antibodies towards a specific site (Fig. For simplicity, we refer to this fine-tuned version of the network as RFdiffusion for the remainder of this paper. a, RFdiffusion is trained such that at time T, a sample is drawn from the prior distribution (three-dimensional Gaussian distribution for translations and uniform SO3 distribution for rotations), and de-noised between times T and 0 to generate an (in this case) scFv. b, The antibody framework is provided as a sequence and ‘template' to RFdiffusion; the latter specifying the pairwise distances and dihedral angles between framework residues. For example, one can specify the design of a VHH (top) or scFv (bottom). Diverse docking modes are sampled by RFdiffusion. d, The epitope is specified by provision of ‘hotspot' residues, which direct the designed antibody (compare orange, left, versus pink, right). e, Overview of the computational design pipeline described in this article. RFdiffusion performs the backbone design step, given a target, epitope hotspots and antibody framework. Fine-tuned RoseTTAFold2 predicts the structure of the designed antibody, given the target (sequence, structure and, optionally, some fraction of hotspot residues) and designed antibody sequence. Self-consistency (high similarity between predicted and designed structures) and high confidence (low predicted alignment error) define in silico success. Note that AlphaFold3, not available at the time of this work, is a better predictor of success than RoseTTAFold2. f, The contribution of this work is the epitope-specific antibody design pipeline depicted in panel e. Several methods can be used to experimentally validate designs and subsequently affinity-mature or optimize them. In this work, we used yeast surface display and/or E. coli expression with SPR for experimental validation (taking approximately 6 weeks and 2 weeks post-oligonucleotide order, respectively), and OrthoRep affinity maturation. With this training regime, RFdiffusion is able to design antibody structures that closely match the structure of the input framework structure and target the specified epitope with novel CDR loops (Supplementary Fig. The designed antibodies make diverse interactions with the target epitope and differ significantly from sequences in the training dataset (Extended Data Fig. There was no correlation between training dataset similarity and binding success (Extended Data Fig. Design pipelines typically produce a wide range of solutions to any given design challenge. An effective way to filter designed proteins and interfaces that are most likely to succeed experimentally is based on the similarity of the designed structure to the AlphaFold2-predicted structure for the designed sequence (this is often referred to as ‘self-consistency'), which has been shown to correlate well with experimental success21,22. In the case of antibodies, however, AlphaFold2 fails to accurately predict antibody–antigen structures23, preventing its use as a filter in an antibody design pipeline, and at the outset of this project, AlphaFold3 (ref. We sought to improve design filtering by fine-tuning RoseTTAFold2 on antibody structures. The rationale for providing this information is that the target structure and binding location are available during design (but are typically not available during general structure prediction). With this training regimen and additional information, RF2 is able to robustly distinguish true antibody–antigen pairs from decoy pairs and often accurately predicts antibody–antigen complex structures, but only when the bound (holo) conformation of the target structure and epitope information is provided (Extended Data Fig. When this fine-tuned RF2 network is used to re-predict the structure of RFdiffusion-designed VHHs, a significant fraction are confidently predicted to bind in an almost identical manner to their designed structure (Extended Data Fig. Furthermore, in silico cross-reactivity analyses demonstrated that RFdiffusion-designed VHHs are rarely predicted to bind to unrelated proteins (Extended Data Fig. VHHs that are confidently predicted to bind to their designed target are predicted to form high-quality interfaces, as measured by Rosetta ddG (Extended Data Fig. This indicates that RF2 filtering might enrich for experimentally successful binders. We initially focused on the design of single-domain antibodies (VHHs) produced by camelids25. To date, two VHH-based therapies have been approved by the FDA with many clinical trials ongoing25. Despite having fewer CDR loops (three) than conventional antibodies (six), the average interaction surface area of a VHH is very similar to that of an antibody26, suggesting that a method capable of VHH design could also be suitable for antibody design. Indeed, in silico metrics for scFvs and VHHs showed similar qualities of interfaces, as assessed by Rosetta7 and fine-tuned RF2 (Extended Data Fig. We chose a widely used humanized VHH framework (h-NbBcII10FGLA)27 as the basis of our VHH design campaigns, and designed VHHs to a range of disease-relevant targets: C. difficile TcdB, influenza H1 haemagglutinin, respiratory syncytial virus (RSV) sites I and III, SARS-CoV-2 receptor-binding domain (RBD) and IL-7Rα. Computationally filtered designs were screened either at high throughput by yeast surface display (9,000 designs per target; RSV sites I and III, RBD and influenza haemagglutinin) or at lower throughput with Escherichia coli expression and single-concentration surface plasmon resonance (SPR; 95 designs per target; TcdB, IL-7Rα and influenza haemagglutinin; the latter was screened using both methods). The highest affinity binders to RSV site III, influenza haemagglutinin, RBD and TcdB are shown in Fig. 2a–c,e, respectively (see also Supplementary Fig. 2 for all the SPR traces of confirmed VHH binders identified in this study and Supplementary Methods Table 6 for success rates against each target, which range from 0% to 2%). The CDR loops are distinct from VHHs observed in nature, indicating substantial generalization beyond the training dataset (Extended Data Fig. Of the haemagglutinin binders tested against the insect-cell-produced haemagglutinin monomer, the highest affinity binder had a dissociation constant (Kd) of 78 nM (Fig. For TcdB, the target epitope was the Frizzled interface, for which there are no antibodies or VHHs targeting this site in the PDB. For the best-designed VHH from both RBD (Kd = 5.5 μM; Fig. 2d), binding was confirmed to be to the desired epitope: binding was completely abolished upon addition of a previously designed, structurally characterized de novo binder to that epitope (AHB2 (PDB ID 7UHB28) for RBD and FZD48 (PDB ID 9CM5 (ref. This TcdB VHH also neutralized TcdB toxicity in CSPG4-knockout cells (an alternative TcdB receptor) with a half-maximal effective concentration (EC50) of 460 nM (Extended Data Fig. For TcdB, the interactions were specific, with no binding observed to the highly related (70% sequence homology) Paeniclostridium sordellii lethal toxin L (TcsL; Extended Data Fig. These data demonstrate the ability of RFdiffusion to design VHHs that make specific interactions with the target epitope. a,b, Nine thousand designed VHHs were screened against RSV site III (a; VHH_RSV_01) and influenza haemagglutinin (b; VHH_flu_01) with yeast surface display, before soluble expression of the top hits in E. coli. SPR demonstrated that the highest affinity VHHs to RSV site III and influenza haemagglutinin bound their respective targets with 1.4 μM and 78 nM, respectively. c, Nine thousand VHH designs were tested against the SARS-CoV-2 RBD, and after soluble expression, SPR confirmed an affinity of 5.5 μM to the target for design VHH_RBD_D4 (left). Binding was to the expected epitope, confirmed by competition with a structurally confirmed de novo binder (AHB2 (PDB ID 7UHB), right). d, Ninety-five VHH designs were tested against C. difficile TcdB. The highest affinity VHH, VHH_TcdB_H2, bound with 262 nM affinity (left), and also competed with a structurally confirmed de novo binder (FZD48, PDB ID 9CM5 (ref. 4a–c for quantification of the competition shown in panels c,d. For all panels, the measured binding response is indicated in a solid blue line, and the global fit using a 1:1 binding interaction model is indicated with a black dashed line. We sought to evaluate design accuracy by cryo-electron microscopy (cryo-EM) structure determination of the designed anti-haemagglutinin VHHs in complex with natively glycosylated, trimeric influenza haemagglutinin glycoprotein (strain A/USA:Iowa/1943 H1N1; Supplementary Fig. 4), which retains the conserved stem epitope used during computational VHH design and upstream biochemical screening. Cryo-EM data processing revealed that one VHH design effectively bound to the fully glycosylated haemagglutinin trimer (out of the four tested), denoted hereafter as VHH_flu_01 (Fig. Two-dimensional classification of all particles in the dataset (Fig. 3b and Supplementary Methods Table 10) identified approximately 66% of haemagglutinin particles bound to a maximum of two VHHs per trimer (Fig. This partial occupancy is probably attributable to the N296 glycan, which, in unbound subunits, partially occludes the target epitope but reorients when bound to VHH_flu_01 (see Fig. a, Labelled cryo-EM two-dimensional class averages of designed VHH_flu_01 bound to influenza haemagglutinin (HA) strain A/USA:Iowa/1943 H1N1. b, The 3.0 Å cryo-EM three-dimensional reconstruction shows VHH_flu_01 bound to H1 along the stem in two protomers. c, Cryo-EM structure of VHH_flu_01 bound to influenza haemagglutinin. e, Comparison of predicted CDR3 rotamers compared with the built 3.0 Å cryo-EM structure. h, Examination of apo haemagglutinin protomers juxtaposed with those bound to the designed VHH shows repositioning of glycan N296 to allow for binding of the designed VHH to the stem. i, Labelled cryo-EM two-dimensional class averages of the designed VHH, VHH_TcdB_H2, bound to full-length TcdB. CROPs, combined repetitive oligopeptides; GTD, glucosyltransferase domain. The predicted design closely matches the experimentally determined complex in structure, epitope targeting and overall conformation. l, Labelled cryo-EM two-dimensional class averages of the designed VHH, VHH_TcdB_H2_ortho, bound to full-length TcdB. The resulting structure shows no detectable change in binding orientation or docking angle compared with the original design, indicating that OrthoRep maturation preserved the predicted mode of epitope engagement. In all panels: yellow indicates haemagglutinin; grey denotes the computational design prediction; pink or navy shows VHH (cryo-EM); and teal indicates glycan. 5) reveals a VHH approach angle that closely matches the predicted model (Fig. 3f) and a VHH backbone that is very close to the RFdiffusion design, with a calculated root mean square deviation (RMSD) of 1.45 Å (Fig. 3d), with residues V100, V101, S103 and F108 in the de novo-designed CDR3 loop interacting with the influenza haemagglutinin stem epitope in the cryo-EM structure, as designed by RFdiffusion and re-predicted with RF2 (Fig. Together, these results demonstrate the VHH design with atomic-level precision. To improve the binding affinity of de novo-designed VHHs, we utilized the orthogonal error-prone DNA replication system, OrthoRep, for continuous hypermutation of target genes in vivo30,31. OrthoRep has been shown to drive the rapid affinity maturation of yeast surface-displayed antibodies. We used this capability to affinity-mature VHHs targeting TcdB, influenza H1 haemagglutinin and the SARS-CoV-2 RBD. Affinity-matured VHHs acquired several mutations relative to the parent designs and improved binding affinities by approximately two orders of magnitude (Supplementary Fig. 3), making them suitable candidates for downstream cryo-EM structural characterization. TcdB consists of four functional domains including a central delivery and RBD (DRBD) where the VHHs were designed to bind. Analysis via two-dimensional and three-dimensional classification revealed a mix of bound and unbound TcdB particles (Fig. Extensive three-dimensional classification and local refinement identified multiple structural states of TcdB within the dataset, including an extended bound state (Extended Data Fig. Three-dimensional refinement of the bound VHH in the extended TcdB state yielded a modest 4.6 Å map, into which the design model was confidently rigid-body docked, showing high agreement with the intended design structure (Fig. These experiments revealed a high proportion of TcdB particles now bound by the VHH, consistent with its enhanced affinity (Fig. Using a similar processing pipeline as for the parent VHH–TcdB complex, we resolved the affinity-matured VHH–TcdB complex to a modest 5.7 Å resolution, enabling us to confidently dock the designed VHH into the cryo-EM density with close agreement. This confirmed that the VHH maintained targeting to the correct epitope and retained its original binding pose after OrthoRep-mediated affinity maturation (Fig. These results underscore the capability of RFdiffusion to design accurate de novo VHHs that are capable of targeting previously unexplored epitopes and are amenable to downstream affinity maturation. We next used cryo-EM to characterize an affinity-matured VHH (VHH_RBD_D4_ortho19) targeting the SARS-CoV-2 spike RBD, where competition experiments indicated that the parental VHH bound the intended epitope (Fig. The RBD transitions between ‘up' and ‘down' conformations, with the ‘up' state enabling receptor binding and viral entry32. Cryo-EM two-dimensional class averages and three-dimensional classification reconstructions of the VHH-bound complex revealed a mixture of RBD conformations (1–2 ‘up'), with VHH density observed exclusively in the up state. This is consistent with its design, as the target epitope is occluded in the down conformation (Supplementary Fig. Global refinement with an average estimated resolution of 3.9 Å provided well-defined density for the lower portion of the spike protein (local resolution of approximately 2.5 Å), but the relative flexibility of the RBD resulted in substantial signal averaging, causing density loss at higher contour levels, which precluded assessment of VHH design accuracy (Supplementary Fig. 9f,g), in agreement with our biochemical competition data (Fig. However, although the VHH bound the correct RBD epitope, its binding mode deviated notably from the design model, instead adopting a predominantly framework-mediated interaction that more closely matched retrospective AlphaFold3 predictions (Supplementary Fig. Given the success of RFdiffusion at designing VHHs with three de novo CDRs, we next tested its ability to design both heavy and light chains in scFv format. RFdiffusion was used to generate scFvs targeting specific epitope sites, following a strategy similar to the VHH design approach. However, unlike VHHs, where only three CDRs were built de novo, scFv design involved constructing all six CDRs on both the heavy and the light chains in addition to the docking mode. The gene synthesis problem is more formidable for scFvs than for VHHs as they are too long to be simply assembled from pairs of conventional oligonucleotides synthesized on oligonucleotide arrays, and are challenging to uniquely pair due to high sequence homology between scFvs. We developed stepwise assembly protocols that enable the construction of libraries with heavy and light chains either specifically paired as in the design models (Supplementary Figs. 10 and 11) or combinatorially mixed within subsets of designs specifically with similar target-binding modes (Supplementary Fig. The latter approach helps to overcome the greater challenge of accurate design of six CDRs de novo, which increases the possibilities for error compared with the VHH problem as only one suboptimal CDR can compromise binding. We found that given sets of nearly superimposable designs targeting the same site with the same binding mode, new scFvs generated by combining pairs of heavy and light chains from different designs were confidently predicted to bind to the target site in the designed binding mode at similar frequencies as compared to the original designs (Extended Data Fig. By contrast, random, structure-agnostic pairing rarely led to predicted binders (Extended Data Fig. Hence, by mixing CDRs from different designs that bind in the same orientation, we can effectively overcome failures due to single imperfectly designed CDRs, thereby offering a combinatorial solution to a combinatorially more complex problem (two-chain scFv design versus one-chain VHH design). This strategy highlights a key advantage of structure-based design: ‘intelligent' pairing of heavy and light chains is possible with a structural model of every antibody, and allows de novo-designed antibody libraries to reach scales attainable by traditional library assembly methods, despite current limits in gene synthesis. We succeeded in identifying epitope-specific scFvs from the heavy–light combinatorial libraries (of a theoretical complexity of approximately 10 million; Extended Data Figs. 7b,c and 8a–c) but not the fixed pairing libraries (Supplementary Fig. Following expression and purification, SPR analysis of six distinct scFvs originating from two unique docks targeting the Frizzled epitope of TcdB revealed a range of affinities (Fig. 4d–h): the highest affinity binder, scFv6, had a Kd of 72 nM (Fig. Conversion of the scFv to a full length IgG1 generated antibodies that bind with comparable (68 nM) affinity, demonstrating that our design method can be used to generate full-length antibodies (Fig. There are no antibodies binding to this epitope in the PDB, hence, this success cannot be attributed to memorization. Subsequent prediction of the structure of the scFv with AlphaFold3 showed a binding mode identical to that of the two nearly superimposable parent designs that contributed the light and heavy chains (Supplementary Fig. Competition with a known receptor, Frizzled-7, to this epitope confirmed that binding of scFv5 was on target (Fig. By contrast, no competition was seen in the presence of CSPG4, an alternative receptor that interacts with an epitope at the toxin core. Thus, scFvs targeting user-specified epitopes can be identified from structure-aware designed combinatorial libraries. a, Multiple sequence alignment of six scFvs that bind to TcdB. scFvs 1–5 originate from the same structural cluster, whereas scFv6 originates from a distinct cluster. The predicted orientation of scFv6 relative to TcdB is rotated compared with scFv5. d, Affinity of scFv5 to TcdB was 460 nM by SPR. e, Computational prediction of the scFv5–TcdB interface for VH (variable heavy-chain fragment; left) and VL (variable light-chain fragment; right). f, scFv5, when expressed as a full-length IgG1, shows a binding affinity of 380 nM to TcdB by SPR. g, Affinity of scFv6 to TcdB was 72 nM by SPR. h, Computational prediction of the scFv6–TcdB interface for VH (left) and VL (right). i, scFv6, when expressed as a full-length IgG1, shows a binding affinity of 68 nM to TcdB by SPR. scFv5 was conjugated to a CM5 chip and TcdB RBD was flowed over at 50 nM either alone or mixed with 1 μM of Frizzled-7, CSPG4 or scFv5. scFv was immobilized and then on-target and off-target binding was measured across an eight-step, twofold titration with an upper concentration of 5 μM. Steady-state kinetic analysis (k) and raw SPR trace (l) of on-target and off-target binding indicate specific binding to the intended target. m, AlphaFold3 predictions of HLA-C*07:02 with peptide PHOX2B (left) and PHOX2B(R6A) (right). R6 of PHOX2B is predicted to be solvent exposed. n, AlphaFold3 prediction of scFv B1.2.1 in complex with C*07:02–PHOX2B (left). Predicted polar contacts with R6 of the PHOX2B peptide (right), mediated by CDRH3, CDRL1 and CDRL2, are also shown. Figure was created using BioRender (http://biorender.com). We next targeted a clinically relevant epitope: the QYNPIRTTF peptide derived from the PHOX2B neuroblastoma-dependency gene and master transcriptional regulator in complex with the major histocompatibility complex (MHC) allotype HLA-C*07:02 (we refer to this peptide below simply as PHOX2B). The PHOX2B peptide was originally discovered by immunopeptidomics of neuroblastoma patient-derived samples, and has been targeted with peptide-centric chimeric antigen receptors (PC-CARs) for treating high-risk neuroblastoma33. However, the PC-CARs identified previously are restricted to recognizing PHOX2B presented on HLAs of the A9 serological group, excluding the common allotype HLA-C*07:02 (ref. Targeting the PHOX2B–HLA-C*07:02 complex could meaningfully increase the addressable patient population for these immunotherapies, and has been the focus of ongoing therapeutics development. Recently, computationally designed (non-antibody) binders for PHOX2B–HLA-C*07:02 have been developed, using the TRACeR-I system35, whereas high-affinity TCRs have been identified for targeting peptides on the common HLA-C*08:02/HLA-C*05:01 allotypes36. A benefit of structure-based design is the ability to target specific peptide residues to achieve binding specificity (rather than binding only to the MHC), and we therefore used RFdiffusion to target the R6 residue, which is known to be important for binding in the PC-CAR34. Given the low stability of the PHOX2B–HLA-C*07:02 complex (Tm of 44.2 °C)34, we leveraged a disulfide-stabilized approach to prepare a stabilized form of the pHLA target37. Using the combinatorial assembly approach described above, we identified modest-affinity (400 nM as measured by SPR and 1 μM as measured by isothermal titration calorimetry (ITC); Fig. Binding was specific to the peptide, with no detectable binding to the R6A point mutant PHOX2B peptide (PHOX2B(R6A)–HLA-C*07:02; Fig. Attempts were made to incorporate scFv binders into a 4-1BB-CAR, but T cell cytotoxicity assays demonstrated no detectable killing of a range of neuroblastoma cell lines (Supplementary Fig. 14), probably because of the modest binding affinity and/or low levels of antigen density expressed on the tumour cells. Although there is still considerable room for improvement in affinity, this demonstrates the ability of structure-based antibody design, paired with appropriate library assembly methods, to design specific binders to challenging and clinically important target epitopes. To evaluate the accuracy of de novo scFv design, we determined the cryo-EM structures of two combinatorially assembled scFvs, scFv5 and scFv6, both targeting the Frizzled epitope of TcdB. Cryo-EM analysis confirmed that both scFvs bound the Frizzled epitope as designed (Fig. High-resolution two-dimensional class averages of scFv6 revealed clear density for both TcdB and the bound scFv, further supported by a 3.6 Å three-dimensional reconstruction (Fig. Superposition of the cryo-EM structure with the design model demonstrated remarkable agreement, with both heavy and light chains interacting with the epitope as intended (Fig. This agreement extended to the rotameric conformations of CDR side chains and their interactions with the Frizzled epitope, underscoring the accuracy of RFdiffusion in designing de novo scFv–target interactions (Fig. b, A 3.6 Å cryo-EM three-dimensional reconstruction of the complex shows scFv6 bound to TcdB along the Frizzled epitope. c, The cryo-EM structure of scFv6 in complex with TcdB closely matches the design model. d, Cryo-EM structure of scFv6 bound to TcdB. e, Cryo-EM reveals the accurate design of scFv6 using RFdiffusion. f, Superposition of each of the six designed scFv6 CDR loop predicted structures as compared with the built cryo-EM structure. g, Comparison of predicted CDRH3 rotamers compared with the built 3.6 Å cryo-EM structure. h, Labelled cryo-EM two-dimensional class averages of the designed scFv, scFv5, bound to full-length TcdB. This approach demonstrates that the predicted design closely matches the experimentally determined complex in structure, epitope targeting and overall conformation. In models, yellow denotes TcdB; navy indicates the variable heavy-chain fragment (cryo-EM); pink shows the variable light-chain fragment (cryo-EM); and grey denotes the computational design prediction. A 6.1 Å cryo-EM reconstruction confirmed scFv5 binding to the TcdB Frizzled epitope, with two-dimensional class averages showing clear density for the complex (Fig. Although our results demonstrate that the de novo design of antibodies is possible, the experimental success rates remain low. A key contributor to previous successes in de novo binder design was improved filters (primarily AlphaFold2 (ref. 19), which enriched for experimental success in the subset of designs that are tested experimentally2,22. At the outset of this study, we sought to build such a filter by fine-tuning RoseTTAFold2 (Extended Data Figs. 2 and 3), but the filtering power of this model is limited (at least with the settings used; providing 100% of interface ‘hotspots'). Subsequent to the design work in this study, AlphaFold3 (ref. Retrospectively, we can assess how filtering with AlphaFold3 would have improved experimental success rates. First, AlphaFold3 accurately predicts the experimentally validated structure of the inaccurately designed SARS-CoV-2 VHH (Supplementary Fig. Had AlphaFold3 been used as an initial filter, this design would have been rejected due to the discrepancy between the predicted and intended structures, thereby preventing its experimental testing. Second, we predicted the structures of the SARS-CoV-2, influenza haemagglutinin, TcdB and IL-7Rα VHH designs using AlphaFold3 with a multiple sequence alignment (MSA) and templates for the target and only a template for the VHH (as CDRs are de novo, we reasoned the MSA would be of limited utility). We analysed the predictions for libraries with at least one structurally validated VHH (TcdB, influenza haemagglutinin and SARS-CoV-2). These results are dominated by anti-haemagglutinin VHHs as the majority of successful binders came from this library. We found that the AlphaFold3 interface predicted template modelling (ipTM) score, a measure of model confidence over the interface, is predictive of binding success (area under the curve = 0.86; Extended Data Fig. Overall, only 9% of our ordered VHH designs have an ipTM > 0.6, suggesting that success rate will be improved by incorporation of an ipTM filter. We found that successful designs cluster to higher AlphaFold3 ipTM scores than the parental designs (Extended Data Fig. Only 4% of the initial design library has ipTM > 0.85, whereas 5 out of the 6 experimentally confirmed designs pass this threshold, again suggesting that filtering by AlphaFold3 ipTM should increase success rates (Extended Data Fig. Our results demonstrate that de novo design of antibody domains targeting specific epitopes on a target is possible. The cryo-EM structural data for the designed VHHs to influenza haemagglutinin and TcdB reveals very close agreement to the computational design models, showing that our approach can design VHH complexes with atomic accuracy—including the highly variable H3 loop and the overall binding orientation—that are highly dissimilar from any known structures in the PDB. Moreover, cryo-EM structural data of designed scFvs bound to TcdB demonstrate the ability of RFdiffusion to design two-chain scFvs accurately. To our knowledge, these are the first structurally validated cases of de novo-designed antibodies. Our computational method synergizes with experimental screening approaches developed for retrieving antibodies from large random libraries in several ways. First, yeast display selection methods widely used for antibody library screening enable the retrieval of the highest affinity binders among large sets of designs, which is currently necessary due to the quite low design success rate. Second, screening combinatorial libraries that mix heavy and light chains from designs with similar binding modes allows for the identification of scFvs composed of structurally compatible chains targeting specific epitopes, as demonstrated here for TcdB and PHOX2B–peptide MHC. Third, affinity maturation using OrthoRep3 improves the measured affinity of initial VHH designs down to the single-digit nanomolar or subnanomolar range, while preserving the original designed-binding mode. From a practical standpoint, the key advance of this work is not the ability to generate VHHs and scFvs against a target—something often achievable through purely experimental methods—but rather the ability to accurately target specific binding epitopes. The epitope specificity is critical for therapeutic applications such as antagonists that block receptor–ligand interactions, antibodies that avoid competing with endogenous molecules, modulators that induce conformational changes to trigger signalling, or antibodies targeting conserved or evolutionarily restricted viral epitopes. There remains considerable room for improvement. For the backbone design step, incorporating recent architectural improvements39 and new advances in generative modelling40,41,42 may yield design models with higher designability and diversity. RoseTTAFold2 and RFdiffusion have also recently been extended to model all biomolecules (rather than just proteins)43, and incorporating this capability into the antibody design RFdiffusion variant should permit the accurate design of antibodies to epitopes containing non-protein atoms, such as glycans. ProteinMPNN was not modified in this current work, but designing sequences that more closely match human CDR sequences would be expected to reduce the potential immunogenicity of designed antibodies44,45. Indeed, designed sequences are currently somewhat less human (as assessed by an OASis score46) than therapeutic antibody CDRs (Supplementary Fig. Further improvements in antibody structure prediction methods should allow faster optimization of upstream design methods and improve experimental success rates. Ultimately, computational de novo design of antibodies using our RFdiffusion and related approaches47 could revolutionize antibody discovery and development. As the method improves and success rates increase, it has the potential to be faster and more cost-effective than immunizing animals or screening random libraries. A structure-based approach to antibody design should also aid the optimization of key pharmaceutical properties, such as aggregation, solubility and expression levels (all major challenges in antibody development) in a structure-informed manner. Together, we expect that computational design of antibodies will increase the number of tractable clinical targets and diseases accessible to antibody therapeutics. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The antibody training dataset is available on Zenodo48 (https://zenodo.org/records/15741710). The codes for running the RFdiffusion antibody design, ProteinMPNN and fine-tuned RoseTTAFold have been released as a single repository on GitHub, free for academic, personal and commercial use (https://github.com/RosettaCommons/RFantibody). Wilson, P. C. & Andrews, S. F. Tools to therapeutically harness the human antibody response. Watson, J. L. et al. De novo design of protein structure and function with RFdiffusion. Paulk, A. M., Williams, R. L. & Liu, C. C. Rapidly inducible yeast surface display for antibody evolution with OrthoRep. The global landscape of approved antibody therapies. & Vendruscolo, M. Rational design of antibodies targeting specific epitopes within intrinsically disordered proteins. Liu, X. et al. Computational design of an epitope-specific Keap1 binding antibody using hotspot residues grafting and CDR loop swapping. Leaver-Fay, A. et al. ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules. & Kim, P. M. Antibody-SGM, a score-based generative model for antibody heavy-chain design. Deep generative design of epitope-specific binding proteins by latent conformation optimization. Shanehsazzadeh, A. et al. Unlocking de novo antibody design with generative artificial intelligence. Rapid discovery of high-affinity antibodies via massively parallel sequencing, ribosome display and affinity screening. A. et al. AlphaBind, a domain-specific model to predict and optimize antibody–antigen binding affinity. Vázquez Torres, S. et al. De novo design of high-affinity binders of bioactive helical peptides. Improved protein binder design using beta-pairing targeted RFdiffusion. Cao, L. et al. Design of protein-binding proteins from the target structure alone. Gainza, P. et al. De novo design of protein interactions with learned surface fingerprints. One-shot design of functional protein binders with BindCraft. Cutting, D., Dreyer, F. A., Errington, D., Schneider, C. & Deane, C. M. De novo antibody design with SE(3) diffusion. Highly accurate protein structure prediction with AlphaFold. Improved protein structure prediction using predicted interresidue orientations. Wang, J. et al. Scaffolding protein functional sites using deep learning. Improving de novo protein binder design with deep learning. Yin, R. & Pierce, B. G. Evaluation of AlphaFold antibody–antigen modeling with implications for improving predictive accuracy. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Jin, B., Odongo, S., Radwanska, M. & Magez, S. Nanobodies: a review of generation, diagnostics and therapeutics. Analysis of nanobody paratopes reveals greater diversity than classical antibodies. Vincke, C. et al. General strategy to humanize a camelid single-domain antibody and identification of a universal humanized nanobody scaffold. Hunt, A. C. et al. Multivalent designed proteins neutralize SARS-CoV-2 variants of concern and confer protection against infection in mice. Ragotte, R. J. et al. De novo design of potent inhibitors of clostridial family toxins. Continuous evolution of user-defined genes at 1 million times the genomic mutation rate. & Liu, C. C. Scalable, continuous evolution of genes at mutation rates above genomic error thresholds. Unexpected receptor functional mimicry elucidates activation of coronavirus fusion. Targeting of intracellular oncoproteins with peptide-centric CARs. Structural principles of peptide-centric chimeric antigen receptor recognition guide therapeutic expansion. Targeting peptide antigens using a multiallelic MHC I-binding system. Sim, M. J. W. et al. High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D. Sun, Y. et al. Universal open MHC-I molecules for rapid peptide loading and enhanced complex stability across HLA allotypes. Wang, C. et al. Proteus: pioneering protein structure generation for enhanced designability and efficiency. Yim, J. et al. Fast protein backbone generation with SE(3) flow matching. SE(3)-stochastic flow matching for protein backbone generation. 12th International Conference on Learning Representations (ICLR, 2024). Geffner, T. et al. Proteina: scaling flow-based protein structure generative models. 13th International Conference on Learning Representations (ICLR, 2025). Krishna, R. et al. Generalized biomolecular modeling and design with RoseTTAFold All-Atom. Gao, S. H., Huang, K., Tu, H. & Adler, A. S. Monoclonal antibody humanness score and its applications. Dreyer, F. A., Cutting, D., Schneider, C., Kenlay, H. & Deane, C. M. Inverse folding for antibody sequence design using deep learning. Prihoda, D. et al. BioPhi: a platform for antibody design, humanization, and humanness evaluation based on natural antibody repertoires and deep learning. Bio, N. & Biswas, S. De novo design of epitope-specific antibodies against soluble and multipass membrane proteins with high specificity, developability, and function. Watson, J. L. Antibody training dataset for “Atomically accurate de novo design of antibodies with RFdiffusion” [data set]. Altschul, S. F. et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Dunbar, J. et al. SAbDab: the structural antibody database. The scFv fragment of the antibody hu4D5-8: evidence for early premature domain interaction in refolding. Kawai, S., Hashimoto, W. & Murata, K. Transformation of Saccharomyces cerevisiae and other fungi. This work was supported by gifts from Microsoft (to D.L.S. ), The Donald and Jo Anne Petersen Endowment for Accelerating Advancements in Alzheimer's Disease Research (to N.R.B. ), the National Eye Institute of the National Institutes of Health under award number T32EY032448 (to Y.Y. ), grant 5U19AG065156-02 from the National Institute for Aging (to D.B. ), grant R01CA260415 from the National Cancer Institute (to C.C.L. ), grant R35GM136297 from the National Institute of General Medical Sciences (to C.C.L. ), the Institute for Rapid Antibody Engineering and Evolution as part of the Engineering+Health Initiative of the UCI Samueli School of Engineering (to C.C.L. ), the Open Philanthropy Project Improving Protein Design Fund (to R.J.R. ), a grant (INV-010680) from the Bill and Melinda Gates Foundation (to J.L.W., C.W., E.L.S., K.D.C. ), an EMBO Postdoctoral Fellowship (grant number ALTF 292-2022; to J.L.W. ), Howard Hughes Medical Institute COVID-19 Initiative (to C.W. ), Defense Threat Reduction Agency grant HDTRA1-21-1-0007 (to B.H. ), a National Science Foundation Training Grant (EF-2021552; to P.J.Y.L. ), a Grants for Resident Innovation and Projects award from the Children's Hospital of Philadelphia (to R.A.), as part of the NexTGen team supported by the Cancer Grand Challenges partnership funded by Cancer Research UK (CGCATF-2021/100002), the National Cancer Institute (CA278687-01) and The Mark Foundation for Cancer Research (to J.M.M. ), a grant (U19 AG065156) from the National Institute for Aging (to S.V.T. ), a Washington Research Foundation Postdoctoral Fellowship program (to R.J.R. ), the Defense Threat Reduction Agency Grant HDTRA1-21-1-0038 (to I.G. ), the Howard Hughes Medical Institute (to N.R.B., R.J.R. ), a grant from the Institute for Basic Science IBS-R030-C1 (to H.M.K. ), the Bill and Melinda Gates Foundation for Adjuvant Research (to C.C. and an EMBO long-term fellowship (to B.I.M.W.). Figure 4 was created using BioRender (http://biorender.com). These authors contributed equally: Nathaniel R. Bennett, Joseph L. Watson, Robert J. Ragotte, Andrew J. Borst, DéJenaé L. See, Connor Weidle, Riti Biswas, Yutong Yu Nathaniel R. Bennett, Joseph L. Watson, Robert J. Ragotte, Andrew J. Borst, DéJenaé L. See, Connor Weidle, Riti Biswas, Ellen L. Shrock, Philip J. Y. Leung, Buwei Huang, Inna Goreshnik, Kenneth D. Carr, Benedikt Singer, Cameron Criswell, Basile I. M. Wicky, Susana Vázquez Torres & David Baker Institute for Protein Design, University of Washington, Seattle, WA, USA Nathaniel R. Bennett, Joseph L. Watson, Robert J. Ragotte, Andrew J. Borst, DéJenaé L. See, Connor Weidle, Riti Biswas, Ellen L. Shrock, Philip J. Y. Leung, Buwei Huang, Inna Goreshnik, Kenneth D. Carr, Benedikt Singer, Cameron Criswell, Basile I. M. Wicky, Dionne Vafeados, Mariana Garcia Sanchez, Susana Vázquez Torres, Sidney Chan & David Baker Graduate Program in Molecular Engineering, University of Washington, Seattle, WA, USA Nathaniel R. Bennett, Riti Biswas & Philip J. Y. Leung Center for Synthetic Biology, University of California, Irvine, CA, USA Howard Hughes Medical Institute, University of Washington, Seattle, WA, USA Molecular Medicine Program, The Hospital for Sick Children, Toronto, Ontario, Canada Center for Biomolecular and Cellular Structure, Institute for Basic Science (IBS), Daejeon, Republic of Korea Graduate Program in Biological Physics, Structure and Design, University of Washington, Seattle, WA, USA Center for Computational and Genomic Medicine, Department of Pathology and Laboratory Medicine, Children's Hospital of Philadelphia, Philadelphia, PA, USA Division of Oncology and Center for Childhood Cancer Research, Children's Hospital of Philadelphia, Philadelphia, PA, USA Timothy T. Spear, Keelan O'Reilly & John M. Maris Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the study, and may change the order of their names for personal pursuits to best suit their own interests. led the experimental work, with help from E.L.S., P.J.Y.L., B.H., I.G., M.G.S., D.V., R.A., S.V.T., S.M.S., T.T.S. supervised the experimental work and provided reagents. led the negative stain electron microscopy and cryo-EM structural characterization work, with help from C.W. All authors read and contributed to the manuscript. Correspondence to Joseph L. Watson, Robert J. Ragotte or David Baker. are co-inventors on US provisional patent number 63/607,651, which covers the computational antibody design pipeline described here. are currently employed by Xaira Therapeutics. have received payments relating to the licensing of the inventions described here to Xaira Therapeutics. is a co-founder of K2 Therapeutics, which uses OrthoRep in antibody engineering and evolution. The other authors declare no competing interests. Nature thanks Haiyan Liu, Yufeng Liu, Carlos Outeiral, Amalio Telenti and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A) Designed VHH sequences are distinct from the training dataset. Note also that the 28 VHHs confirmed to bind their targets by SPR do not show enhanced similarity to the training set (red lines). B-I) Designed VHH are structurally distinct from immunoglobulins binding (when existent) to the epitopes used in this study. Designed VHHs (pink) are shown in complex with their designed target (teal and tan). The closest complex was identified visually (Methods). B) Designed TcdB VHH aligned against 3 VHHs from PDB ID: 6OQ5 (shades of blue). The designed TcdB VHH binds to a site for which no antibody or VHH structure exists in the PDB. C) Designed RSV Site III VHH aligned against VHH from PDB ID: 5TOJ (blue). D) Highest affinity designed influenza HA VHH aligned against Fv from PDB ID: 8DIU (shades of blue). E) Highest affinity designed influenza HA VHH aligned against VHH from PDB ID: 6FYT (blue). F) Structurally characterized (cryoEM) designed influenza HA VHH aligned against Fv from PDB ID: 8DIU (shades of blue). G) Structurally characterized (cryoEM) designed influenza HA VHH aligned against VHH from PDB ID: 6FYT (blue). I) Designed SARS-CoV-2 VHH aligned against FAb from PDB ID: 7FCP (shades of blue). The sequence of antibody 7Y1B was provided either with the correct (PDB ID: 7Y1B) or decoy (PDB ID: 8CAF) target. Both with 100% (A) or 10% (B) of “hotspots” provided, RF2 near-perfectly predicts binding (top row) or non-binding (bottom row). C) Fine-tuned RoseTTAFold2 reliably predicts its own accuracy. Correlation between RF2 predicted aligned error (pAE) and RMSD to the native structure with 100% (top) or 10% (bottom) of “hotspot” residues provided. With mean pAE <10, 80.3% of structures are within 2 Å when 100% of “hotspots” are provided (along with the holo target structure), with this falling to 52.6% when only 10% of hotspots are provided. D) Quantification of the fine-tuned RF2's ability to distinguish true targets from decoy targets with both pAE (top row) and pBind (bottom row). Note that this ability depends on the proportion of “hotspots” provided. Without any “hotspots” provided, RF2 is hardly predictive, because RF2 without privileged information is quite rarely confident or accurate in its antibody complex predictions. E-F) Fine-tuned RoseTTAFold is also performant at antibody monomer prediction. 86 antibodies released after the RF2, AlphaFold and IgFold training dataset date cutoff (January 13th, 2023) that share <30% target sequence similarity to any antibody complex released prior to this date were predicted as monomers with either fine-tuned RF2, IgFold, AF2 and AF3. E) The median Fv quality prediction (by overall RMSD) of fine-tuned RF2, of (PDB ID: 8GPG), with (right) and without (left) sidechains shown (gray: native; colors: prediction). While the backbone RMSD is close to the true structure, some sidechains are incorrectly positioned. F) Fine-tuned RF2 slightly outperforms IgFold at prediction accuracy. Overall prediction accuracy is slightly improved in fine-tuned RF2 vs IgFold (p = 0.026, Student's Paired T-test), with greater improvements in CDR H3 prediction accuracy (p = 0.0003, Student's Paired T-test). A) Comparison of RF2 pAE and RMSD of the prediction to the design model. A significant fraction of designs are re-predicted by RF2 (given 100% of “hotspots”), and pAE correlates well with accuracy to the design model. B) RF2 can be used to assess quality of designed VHHs. Providing the VHH sequence with the true target structure (used during design) leads to higher rates of high-confidence predictions than predicting the same sequence with a decoy structure (not used in design), as assessed by the fraction of predictions with pAE <10 (normalized to the fraction of predictions with pAE <10 for that target with its “correct” VHH partners). In these experiments, the true or decoy target was provided along with 100% of hotspot residues, with those hotspot residues derived from the target with its “true” designed VHH bound. C) Orthogonal assessment of designed VHHs with Rosetta demonstrates that the interfaces of RF2-approved (RMSD < 2 Å to design model, pAE <10) VHH designs have estimated binding energy (ddG) only slightly less favourable than native VHH (left) and slightly higher spatial aggregation propensity (SAP) score as compared to natives (right). D) RFdiffusion was used to generate scFv designs using the framework from Herceptin (hu4D5-8), which has been used to make scFvs previously51. Five targets were chosen (IL10 Receptor-α, TLR4, β-lactamase, TcdB and SARS-CoV-2 (omicron) RBD (PDB IDs: 6X93, 4G8A, 4ZAM, 7ML7, 7WPC). Shown are two examples with close agreement between the design model and the fine-tuned RF2 prediction (RMSD (Å): 0.60, 0.43; pAE: 4.73, 3.52). E) Top: Against the same four targets to which VHHs were successfully designed, fine-tuned RF2 also predicts good specificity of designed scFvs to the designed target vs decoy targets. Bottom: against the set of five aforementioned targets, fine-tuned RF2 similarly predicts high scFv specificity to the designed target vs decoy targets. F) Orthogonal assessment of designed scFvs with Rosetta demonstrates that the interfaces of RF2-approved (RMSD < 2 Å to design model, pAE <10) scFv designs have low ddG (top; only slightly worse than native Fabs) and lower spatial aggregation propensity (SAP) score as compared to natives (bottom). A) TcdB VHH binds to the correct epitope. The average response during VHH injection normalized to the response immediately preceding VHH injection for TcdB VHH competition with minibinder fzd48. No binding is observed to the closely related Clostridium sordellii TcsL toxin, indicating that it is binding through specific interactions. C) SARS-CoV-2 RBD VHH competition with the minibinder AHB2. As in (A), when minibinder is injected prior to the VHH injection (middle bar) no response is observed. When the VHH is injected without a preceding minibinder injection (right bar), the VHH binds as expected. D) Brightfield microscopy of Vero CSPG4 KO cells treated with vehicle, TcdB alone or TcdB + VHH after 24 h. E) Neutralization of TcdB by VHH_TcdB_H2 in CSPG4 KO Vero cells. Cell viability is measured after 48 h. Points indicate the mean and error bars are the standard deviation across two independent replicates. A) Representative raw micrograph showing ideal particle distribution and contrast. B) 2D Class averages of Influenza H1+designed VHH with clearly defined secondary structure elements and a full-sampling of particle view angles. C) Cryo-EM local resolution map calculated using an FSC value of 0.143 viewed along two different angles. E) Orientational distribution plot demonstrating complete angular sampling. Local refinement and masking used to reduce noise from His tag and improve resolution. A) Local Resolution map (Å), calculated using an FSC value of 0.143 viewed along two different angles. B) Global Fourier Shell Correlation plot, Local Refinement. Structure-based design permits the rational combinatorial assembly of heavy and light chains, assembling only heavy and light chains from structurally similar pairs. A) Fine-tuned RoseTTAFold (left), and AlphaFold3 (right) validate that pairing heavy and light chains from structurally similar (i.e. high pairwise TM score) designs yields scFvs that are more likely to be predicted to bind with high confidence (RF2 pBind, left; AF3 ipTM, right) than heavy and light chains from structurally-dissimilar (low pairwise TM score) designs. Note that the extremely high pBind distribution of the “designed pairings” (rightmost bar of left plot) is an artifact of those designs being specifically filtered for high pBind scores prior to the library being ordered. B-C) combinatorial assembly leads to dramatically larger library sizes. Plots show the number of clusters (pink) at different TM score similarity thresholds for TcdB (left) and Phox2b (right) scFvs. For the amplification strategy to work, each “cluster” becomes a PCR subpool, requiring independent PCR reactions (3 per subpool). Hence, we limit ourselves to large subpools (>= 100 designs), which maximizes the combinatorial amplification for the amount of additional library assembly work. We additionally plot the theoretical library size for each target (blue), calculated as number_of_clusters x cluster_size2. Gray lines indicate the TM threshold chosen for library assembly, where library sizes approximately match the transformation efficiency of yeast (107)52. Each sample was treated with a titration of soluble biotinylated TcdB fragment (1285-1804) (bn-TcdB) concentrations and visualized with anti-C-Myc FITC + SAPE. (right) Percentage of expressing cells which are within the gate increase with bn-TcdB concentration. B) scFvs were designed to bind to the Frizzled epitope and therefore should compete with Frizzled-7. Designed scFvs should not compete with CSPG4, which binds at a different epitope on full-length TcdB. Binding signal specifically decreases when Frizzled-7 is added, supporting that scFv4 binds at the Frizzled-7 epitope. Binding signal does not significantly decrease when CSPG4 is added. D) C*07:02/PHOX2B titration results with yeast surface display of anti-C*07:02/PHOX2B scFv B1.2.1, tested in the “HL” orientation with a (G4S)3 linker. For the tetramer condition, the biotinylated C:07:02/PHOX2B pHLA was tetramerized on streptavidin-PE (SAPE) prior to validation. E) AF3 prediction of construct B1.2.1 docked to C*07:02/PHOX2B. F) Left: Surface plasmon resonance (SPR) data characterizing binding of B1.2.1 in the HL and LH orientations in the 10LH-based framework (“phox” prefix, left column) and the trastuzumab framework (“her” prefix, right column). Right: SPR data characterizing on-target binding of C*07:02/PHOX2B (“phox2b”) versus the same HLA bound to the R6A mutant of PHOX2B (“phox2b_r6a”). G) Representative ITC titration of HLA-C*07:02/PHOX2B (30 μM) into a sample containing 2 μM herceptin_VLVH-His-Avi binder. Both samples contain 1 mM excess of PHOX2B peptide, to prevent the formation of empty HLA. Fitted values for KD, ΔH, and ΔS were determined using a 1-site binding model. Thin ice was targeted for imaging, and only extended TcdB were observed. Heterogeneous refinement and apo structure were used to sort scFv bound TcdB (41,837 particles) and unbound apo TcdB (14,384 particles). B) Representative micrograph C) Local Resolution map (Å), calculated using an FSC value of 0.143 viewed along two different angles D) Global Fourier Shell Correlation plot, Non Uniform Refinement E) Orientational distribution plots. A) ipTM distributions of design VHH libraries against 4 targets. B) ROC curve demonstrating strong retrospective predictive power of AF3 at discriminating designed VHH binders from non-binders (AUC = 0.86). Note though that this plot is dominated by influenza HA binders, which are more numerous than confirmed binders to SARS-CoV-2 RBD and TcdB. C-D) Similar retrospective analyses of TcdB scFv binders. These binders were assembled combinatorially from structurally similar “parent” designs. The successfully-binding combined designs have significantly higher AF3 ipTM scores than the parent designs from which they emanate (C, 6 binders from 12 parent designs; two-sided Students t-test; p = 0.0025), and from the parental library as a whole (D). These analyses indicate the utility of AF3 for antibody design filtering. This Supplementary Information file contains tables, figures, and three main text sections: 1) A description of the computational methods used in this study. We describe the fine tuning of RFdiffusion and RF2, and the computational evaluation of these models for antibody design; 2) A description of the experimental methods used to synthesize and characterize the designed antibodies; 3) A description of the electron microscopy methods used to structurally characterize several antibodies in this study. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Bennett, N.R., Watson, J.L., Ragotte, R.J. et al. Atomically accurate de novo design of antibodies with RFdiffusion. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09698-1'>Synthetic α-synuclein fibrils replicate in mice causing MSA-like pathology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 16:22:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Multiple-system atrophy (MSA) is a rapidly progressive neurodegenerative disease of unknown cause, typically affecting individuals aged 50–60 years and leading to death within a decade1,2,3. It is characterized by glial cytoplasmic inclusions (GCIs) composed of fibrillar α-synuclein (aSyn)4,5,6,7,8, the formation of which shows parallels with prion propagation9,10. While fibrils extracted from brains of individuals with MSA have been structurally characterized11, their ability to replicate in a protein-only manner has been questioned12, and their ability to induce GCIs in vivo remains unexplored. By contrast, the synthetic fibril strain 1B13,14, assembled from recombinant human aSyn, self-replicates in vitro and induces GCIs in mice15—suggesting direct relevance to MSA—but lacks scrutiny at the atomic scale. Here we report high-resolution structural analyses of 1B fibrils and of fibrils extracted from diseased mice injected with 1B that developed GCIs (1BP). We show in vivo that conformational templating enables fibril strain replication, resulting in MSA-like inclusion pathology. Notably, the structures of 1B and 1BP are highly similar and mimic the fold of aSyn observed in one protofilament of fibrils isolated from patients with MSA11. Moreover, reinjection of crude mouse brain homogenates containing 1BP into new mice reproduces the same MSA-like pathology induced by the parent synthetic seed 1B. Our findings identify 1B as a synthetic pathogen capable of self-replication in vivo and reveal structural features of 1B and 1BP that may underlie MSA pathology, offering insights for therapeutic strategies. These inclusions appear in neurons and in oligodendrocytes (OLs), particularly within the myelinated bundles of the white matter16,17,18. They are observed only rarely in other sporadic α-synucleinopathies1,2. The phenomenon of a step-by-step progression that characterizes brain invasion by GCIs, and the speed at which the GCIs induce the death of neurons in myelinated tracts1, has led to the notion that MSA may obey a molecular mechanism analogous to that governing the intracerebral diffusion of pathogenic prions9,10. According to this hypothesis, the specific aSyn fibril strain found in GCIs has a central role as both initiator and propagator of the pathology9,10. aSyn fibrils in GCIs were first observed in situ4,5,6,7,8, and were extracted and described using electron microscopy (EM) more than 25 years ago19,20. Such fibrils, extracted using the detergent sarkosyl, were characterized at high resolution using cryo-transmission EM (cryo-EM)11 but, to date, it has not been demonstrated that they can self-replicate like prions or induce the formation of GCIs in animal models. The hypothesis was put forward12 that unknown cofactors might be required for the replication of these extracted assemblies. In parallel to the extractive approach11, three synthetic aSyn fibril strains were obtained from recombinant aSyn, and these fibrils could self-replicate in vitro without the need for an external cofactor and induce GCIs in mice. Chronologically, the first strain was a non-helical double-filament fibrillar assembly (ribbons) composed of monomers of human aSyn, capable of inducing GCIs, but only in rats overexpressing human aSyn concomitantly21. The second strain corresponded to mouse synthetic aSyn fibrils, which could induce the delayed appearance of GCIs in non-transgenic mice22. Finally, the third case concerned a fibril strain that some of us isolated and characterized (strain 1B), formed of human recombinant aSyn and causing the rapid appearance of GCIs in interfascicular OLs of wild-type (WT) mice15. In contrast to the aSyn fibrils extracted from patients with MSA, a high-resolution structure of these types of synthetic aSyn fibrils was lacking. This prevented the comparison of these fibrils to the atomic models obtained from patients to understand the structural basis of their ability to induce GCIs. Here we report the structural characterization of aSyn fibrils of strain 1B at a resolution of 1.94 Å, and compare them to the aSyn filaments extracted from the brains of 1B-injected diseased mice in which GCIs formed (1BP, resolved at 3.8 Å). The marked structural similarity between 1B and 1BP demonstrates that a conformational templating and replication process is indeed taking place in vivo and that it is responsible for fibril growth and inclusion pathology buildup. Furthermore, the structural analogy of the 1B and 1BP fold with the human MSA fold found in patient 5 of the seminal study of Goedert's group11 suggests that a similar conformational replication process underlies MSA in humans. 1B fibrils present a typical rod-like compact appearance with a width compatible to the intertwining of two protofilaments13 and a left-handed helicoidal twist (Extended Data Fig. We observed in our founder experiments that 1B had the ability to seed a particularly strong aSyn pathology in neuronal cultures (Extended Data Fig. In this early work, we found that the pathology induced by 1B at early timepoints was made of Lewy neurites, neuronal cytoplasmic inclusions (NCIs) and, notably, neuronal intranuclear inclusions (NIIs)13,14. Together with GCIs and glial nuclear inclusions (GNIs), NIIs constitute a key feature of MSA pathology, that is, they are distinctive for MSA compared with Parkinson's disease or dementia with Lewy bodies (DLB), in which NIIs are absent23,24. This very cytopathological feature led us to investigate the possibility that the synthetic 1B fibrils could represent an MSA-like fibril strain. Using a directed anatomical approach and later timepoints, we observed that, 6 months after intracerebral inoculation, 1B was indeed capable of also causing the appearance of GCIs in the anterior commissure (AC) of WT mice15, that is, in a major myelinated tract containing densely packed rows of interfascicular OLs. 1a–e), while IBA1+ microglia did not particularly cluster around aSyn inclusion pathology (Fig. Confocal three-dimensional (3D) volume reconstructions of affected OLs in the BNST showed the typical flame-like cytoplasmic distribution of phosphorylated aSyn (Fig. The global distribution of GCIs in full horizontal brain sections from animals euthanized at 1.5, 12 or 24 months after 1B fibril injection in their right caudate putamen (CP) showed generalized spread of GCIs (Fig. A hovering section overview corresponding to Fig. b, Higher-magnification view of the boxed region in a, showing invasion of the posterior limb of the AC (magnified in c) and BNST (magnified in d) by aSyn inclusions. c, The posterior limb of the AC: most interfascicular OLs contain aSyn inclusions, that is, GCIs. Sparse IBA1+ microglia (blue) are also visible. d, BNST: SOX10+ OLs contain aSyn inclusions (GCIs); sparse IBA1+ microglia are detectable. e, Confocal 3D reconstruction centred on three OLs of the BNST bearing GCIs, from the section shown in a. See also Supplementary Video 1. f, The neuroanatomical distribution of GCIs in brains at 1.5 (left), 12 (centre) and 24 (right) months after 1B injection into the right CP. Whole-brain horizontal sections are shown with every GCI manually annotated based on pS129aSyn and SOX10. The 24-month section corresponds to the N + 1 section adjacent to a. g, CLEM analysis of pS129aSyn+ regions. From left to right, an EM overview with light-microscopy correlation inset, a closer view of the pathology and a high-magnification view used for tomography to confirm fibrillar material are shown. 1f coexisted with a variable load of surrounding neuronal somatic aSyn inclusions (NCIs and NIIs), which represent the forefront of the somatic aSyn pathology at earlier timepoints to progressively make way to a growing population of GCIs, as previously reported15,22 (Extended Data Fig. This is supportive of a neuronal-to-oligodendroglial inclusion transfer as in MSA24,25. Seeding of GCIs by 1B was dependent neither on the mouse strain nor on the intracerebral injection site. When C57BL/6 mice (instead of 129SV mice) were inoculated with 1B above their right SN, 17 months later, numerous aSyn inclusions could be observed on the right side (Extended Data Fig. Note the depletion of tyrosine hydroxylase-positive neurons also on the right side of the pars compacta of this mouse (compare with Extended Data Fig. Here also, the aSyn inclusions comprised numerous GCIs as demonstrated by co-staining the OL nuclei with SOX10 (Extended Data Fig. IBA1+ microglial cells were also observed but without evident signs of clustering around aSyn inclusions (Extended Data Fig. Correlative light and electron microscopy (CLEM)26,27 was used to study inclusions in the brains of C57BL/6 mice. Consecutive brain tissue slices were imaged using fluorescence microscopy to identify inclusions positive for aSyn phosphorylated at Ser129 (hereafter, pS129aSyn). Identified sections were then resin-embedded and heavy-metal stained, and serial ultrathin sections were collected consecutively on glass slides and on EM grids. Glass slides were used for immunohistochemistry (IHC) to relocalize the pS129aSyn+ cells and stained with Toluidine Blue. The recorded images were correlated with EM images taken on the consecutive grids. We used a mouse of the 1B-injected series corresponding to Extended Data Fig. We imaged 30 pS129aSyn+ inclusions in a region straddling the right GP and CP and followed them throughout the brain tissue by serial sectioning. We could identify fibrillar material and membranous cytoplasmic subcompartments in the pS129aSyn-positive areas. Room-temperature electron tomography analysis of various positions within the IHC-positive regions was used to verify fibrillar morphology, which showed fibrillar morphology in a dispersed, bundled and clustered arrangement (Fig. We additionally verified that the fibrillar material in the pS129aSyn+ IHC regions corresponded to aSyn protein assemblies using immunogold labelling (Extended Data Fig. Six cells with nuclei of more than 8.5 µm diameter were assumed to be neurons with NCIs, and 16 pathological cells with a nucleus size of below 8.5 µm were assumed to be GCIs in OLs. In many cases, the correlated area consisted of dispersed fibrillar material within a membranous local environment with various vesicular compartments. Fibril bundles could not be observed in all inclusions. If they were present, they varied strongly in length, thickness, curvature and localization within the cell. Measurement of the identified bundles in 23 inclusions throughout 5 brain slices showed an averaged bundle length of 0.81 ± 0.41 µm. In most cases, the bundle length was of a similar size of around 0.62 ± 0.07 µm. In a few exceptional cases, we measured bundles with individual lengths of 1.08 µm up to 3.64 µm (Fig. Fibrils appeared also in bigger clusters in irregular, chaotic arrangements, with fibrils radiating outward (Extended Data Fig. Moreover, we could observe the regular proximity or direct contact of different and morphologically distinct cells to pathological ones (Extended Data Fig. The morphology of these cells is reminiscent of cells described as dark microglia in mouse models of Alzheimer's disease28,29, as well as of cells recently described in human post-mortem MSA brain30. To further challenge the pathogenic impact of 1B using a humanized in vivo model, we inoculated 1B fibrils into hemizygous transgenic M83 (M83+/−) mice expressing human SNCAA53T (encoding A53T aSyn). M83+/− mice have a normal lifespan and do not spontaneously develop fibrillar aSyn inclusions, despite the presence of one copy of the human SNCAA53T transgene9,10. We found that the intracerebral injection of 1B fibrils in M83+/− mice systematically caused their rapid demise in a notably synchronized manner after only 16 weeks (Fig. Mice at death presented numerous intracerebral aSyn inclusions, in particular in the posterior limb of the AC, in which the presence of Lewy neurite bundles, GCIs and GNIs could be observed (Fig. Non-seeded M83+/− mice of similar age were healthy and showed no intracerebral aSyn inclusions (Fig. This indicated that, similar to in the two previous WT mouse strains, 1B fibrils were able to also cause the buildup of an inclusion pathology with cytopathological elements typical of MSA in transgenic mice expressing human A53T aSyn. a, Kaplan–Meier survival curves of 28 hemizygous transgenic mice expressing human A53T aSyn (M83+/−) without (14 mice, orange curve) or after (14 mice, purple curve) a unilateral injection (right CP) of 1B fibrils (the time of injection is indicated by the arrow). aSyn inclusions are revealed using the anti-phosphorylated aSyn antibody EP1536Y (green), nuclei are revealed with DRAQ7 (blue). c, 1B fibrils seed the buildup of aSyn GCIs and GNIs in OLs of M83+/− mice. At the time of death (4 months after injection), many commissural OLs have GCIs (bottom image row). OLs containing both a GCI and a glial intranuclear inclusion can also be observed (top image row). Using cryo-EM, we determined the 1B fibril structure at a resolution of 1.94 Å (Fig. Fibrils were composed of two identical, twisting amyloid protofilaments in 21 screw symmetry, each comprising a pseudo-Greek-key motif consisting of nine β-sheets. As that density could not be unambiguously assigned to any side chains, we did not build an amino acid sequence for this stretch. The main electrostatic clusters are a positive group formed by Lys43, Lys45 and His50, and a negative group formed by Glu57 and Thr59, flanking the β-sheet (Extended Data Fig. Moreover, His50 appears to make an inter-rung connection within the same protofilament to the following subunit along the z direction to Lys45 (Fig. a, Density maps of 1B and 1BP, respectively, visualized at a local resolution range of 1.9–4.8Å. Wrapping around the stretch starting with Val37, an additional density is present on both protofilaments, although at a lower resolution (black arrowheads), which can also be observed at different positions in 1BP. Moreover, the structure of 1BP indicates highly flexible areas that are poorly resolved at those densities and close to the N-terminal region (red arrowheads). c, Sequence overlay of 1B (blue) and 1BP (orange). One protofilament is shown as a stick representation, and one is shown with only the backbone. The magnified views of the protofilament interface highlight changes in interaction between side chains. To identify the conformation of aSyn fibrils in the seeded pathology, we used the fibril purification method previously developed for human MSA brain tissue11. Despite the low yield of material after the extraction procedure, pS129aSyn+ fibrils were observed in negative-stain imaging (Extended Data Fig. Subsequently, the sample was vitrified for structure determination using cryo-EM. The extracted material was heterogeneous, with a limited concentration of suitable aSyn fibrils for structure determination. 1BP fibrils are composed of two identical protofilaments of which the fold is highly similar to the 1B fibril seeding them (overall root mean squared deviation (r.m.s.d.) of one protofilament: 1.433 Å), confirming a prion-like templated misfolding mechanism that underlies inclusion formation in this mouse model. At the same time, an instance of strain adaptation can be observed at the interprotofilament interface, which spans from Val52 to Lys58 in 1B but is shifted by one amino acid in 1BP, where it spans from Gly51 to Glu57. This shift, which we suppose is driven by the A53T mutation expressed by M83+/− mice, brings the side chains in the 1BP steric zipper that point towards each other into adjacent positions—as in the case of Thr53 and Val55 (Fig. As a result, the intercalation of small hydrophobic side chains observed in 1B is lost in 1BP, but hydrophobic interactions between opposing protofilaments are still maintained. The fundamental organization of a single protofilament of 1B shows similarities not only to 1BP but also to already published structures as well, mainly the patient-purified MSA fibrils11 (Protein Data Bank (PDB): 6XYQ), the corresponding amplification12 (PDB: 7NCK), the H50Q variant31 (PDB: 6PEO) and the WT and 1–121 aSyn variants31,32 (PDB: 80QI and 6H6B) (Fig. The superimposition of these structures to 1B highlights differences in the positioning of Ile88 and Phe94. The region, which was previously reported as the C-pocket and acting as the binding site for thioflavin-T (ThT)33, is identical in arrangement for a subset of structures and particularly in MSA fibrils (PDB: 6XYQ and PDB: 7NCK) and the H50Q variant (PDB: 6PEO). This is notable because, like the 1B fibrils, the H50Q fibrils (PDB: 6PEO) are also ThT negative31. The structural comparisons of single protofilaments with a similar fold are summarized with the respective r.m.s.d. values in Supplementary Table 1. b, Magnified view of the arrangement of the C-pocket on the example of 1B (left; PDB: 9EUU) and 1–121 aSyn (right; PDB: 6H6B) and the arrangement of the N-pocket on the example of 1B (left; PDB: 9EUU), 1–121 aSyn (centre; PDB: 6H6B) and ThT-bound (grey) aSyn (right; PDB: 7YNM). c, The full modelled sequences with transparent surface volume, as well as a ribbon representation with stick representation of Tyr39, Lys45, Glu46, His50, Ile88 and Phe94 for all compared structures. Structural similarities are also given as r.m.s.d. In the 1B structure, Tyr39 is shifted so that it faces Thr44 and Glu46. As a consequence, Tyr39 is not available for interaction with ThT in the N-pocket. Note that a similar conformation of the N-pocket with a shifted Tyr39 presumably preventing ThT binding is also observed in the amplified MSA fibrils (PDB: 7NCK)12 and the H50Q mutation (PDB: 6PEO), again consistent with the parallel observation that seed amplification products templated on MSA brain extracts or on MSA cerebrospinal fluid are ThT negative34. Whether this orientation is important and confers specific MSA-like properties to 1B is unclear, especially as the >3 Å resolutions of PDB structures 6XYQ and 7NCK do not allow for conclusions on its existence in the MSA fibrils. The 1B fibril structure also shows an outer, non-identified density stretch interacting with Gly36 to Val40 (Fig. 3a), a feature that is also present in fibrils amplified from MSA (PDB: 7NCK), in H50Q fibrils (PDB: 6PEO) and in 1BP fibrils where poorly resolved densities flank the N terminus (Fig. This is important, because these outer stretches in 1B block access of the disaggregase complex effector-component HSC70 to its binding site at the N terminus of the fibrils, therefore most likely preventing their disassembly and clearance35. This structural feature might therefore represent a reason for the acute in vivo seeding ability of 1B14. Despite repeated attempts, we were not able to recover enough fibrils from the brains of WT mice seeded with 1B for cryo-EM analysis, most likely because the inclusions are much less numerous than in 1B-injected M83+/− mice. We therefore used the conformation-specific amyloid probe h-FTAA36 to characterize the GCIs seeded by 1B directly in WT mouse brain sections without extraction and to also compare them with the inclusions present in MSA (and DLB) post-mortem brain tissue sections. Indeed, in agreement with previous work37, we observed a clear-cut difference between the signature of GCIs in MSA and that of Lewy bodies in DLB using fluorescence lifetime imaging microscopy (FLIM) of h-FTAA (Extended Data Fig. Using this in situ approach, we observed that the signature of the MSA inclusions was similar to that of the 1B brain inoculum in our mouse experiments and even closer to the signature of the inclusions seeded by 1B (Extended Data Fig. This was also in good agreement with the fact that the inclusion pathology seeded by 1B appeared positive to the MSA-specific silver impregnation staining technique of Gallyas38, further suggestive of a structural analogy between 1B, 1B-seeded fibrils in WT animals and MSA fibrils in GCIs in human brain tissue sections (Extended Data Fig. This is the same conclusion that we drew from the cryo-EM structural comparisons of 1B with 1BP and MSA fibrils that were extracted from brain tissue (Fig. Finally, given the marked similarity between 1B, 1BP and MSA filaments, we sought to determine whether 1BP raised in transgenic animals could still trigger MSA-like neuropathology in WT animals, as synthetic 1B does. To address this question, we faced the challenge of the very low yield of the sarkosyl-based purification protocol for extracting 1BP from 1B-seeded diseased mice—a yield that is insufficient to obtain the amount of intact fibrils required for in vivo reinoculation experiments. To overcome this limitation, we instead used a tenfold dilution of a total brain homogenate from a 1B-seeded diseased mouse as the inoculum, without any purification step. We also focused on early post-injection effects (6 weeks) to maximize the likelihood of detecting a seeded synucleinopathy driven by trace amounts of 1BP fibrils delivered by the inoculum (that is, 2 μl of tenfold-diluted whole-brain homogenate). Our results indicate that the amount of 1BP contained in the inoculate reached a critical value sufficient to seed anew an inclusion pathology in WT mice with the formation of NCIs, NIIs, GCIs and GNIs (Fig. 5), similar to what is observed when using synthetic 1B fibrils as seeds13,14,15. Diluted crude brain homogenates were prepared from prematurely dying M83+/− mice injected 4 months earlier with 1B fibrils. These homogenates were injected into the right CP of WT 129SV mice. After 6 weeks, the mice were euthanized and the brains were processed for horizontal sectioning and immunofluorescence neuropathology. a–c, aSyn inclusions were revealed with anti-phosphorylated aSyn (EP1536Y, green) antibodies, the nigrostriatal tract with anti-tyrosine hydroxylase (TH, red) antibodies and nuclei with DAPI (white). a, One animal is shown with four sections spanning the nigrostriatal tract dorsoventrally. The green dots annotate aSyn inclusions (Lewy neurites, NCIs, NIIs, GCIs and GNIs). b,c, Close-up views of the different inclusion types annotated in a. d,e, Further examples of MSA-specific aSyn inclusions seeded by 1BP-containing homogenates. The sections were stained with TH (turquoise, shown only in d), SOX10 (red, OL marker) and pS129aSyn (#64, green) antibodies, and DAPI (white, shown only in e). Images are from the right SN. d, Four dopaminergic neurons (TH-positive cytoplasms) and four OLs (SOX10-positive nuclei) are visible (DAPI omitted for clarity). Three MSA-typical inclusions are present: two NIIs and one GNI. e, Among eight cells, four OLs aligned in a row and one solitary OL are visible; one OL harbours a nuclear inclusion (pS129aSyn, GNI). As control, diluted homogenates from spontaneously dying homozygous M83+/+ mice were used. Although these brains contain aSyn inclusions, they are not MSA-type, and their homogenates did not induce inclusion pathology in recipient WT mice (not shown). Scale bars, 3 mm (a) and 10 µm (b–e). Given the structural similarity of the synthetic 1B seed (which does not undergo any sarkosyl purification before cryo-EM analysis) and of 1BP (which does go through this detergent-based purification step before cryo-EM), we can deduce that the extraction procedure has no substantial impact on the structural arrangement of 1BP, which is essentially acquired by templating. Moreover, consistent with the notion that non-extracted 1BP and 1B have similar structures, the pathology seeded by brain homogenates containing 1BP is identical to the one seeded by synthetic 1B. Together, this indicates that the 1B strain characteristics are encoded in the pseudo-Greek-key motif of the 1B/1BP fold rather than in the unattributable densities specifically observed in either 1B or in 1BP after adaptation. Notably, the differential impact of the various aSyn mutations on the replication of 1B that we report here is similar to what was reported previously for MSA ‘prions'39,40. In other words, the fold kernel shared by 1B, 1BP and MSA filaments is probably necessary and sufficient to induce both MSA and MSA-like neuropathologies. As previously discussed, such closure accounts for the ThT invisibility of both 1B13,14 and the amplified MSA filaments34. This is particularly relevant given that ThT invisibility is also observed in two other synthetic fibril assemblies of unknown structure, both of which were reported to seed GCIs in vivo21,22,41. It is therefore tempting to speculate that closed N- and C-pockets are also present in these latter fibrils, further supporting the idea that, within the fold kernel shared by 1B, 1BP and MSA filaments, the closure of the N- and C-pockets represents the minimal common structural determinant responsible for the induction of both MSA and MSA-like pathology. Here we present the structures of aSyn fibrils 1B as well as of their in vivo seeding product 1BP. Indicative of a high-fidelity conformational replication process, the structures present a high similarity and can perpetuate MSA-like pathology through passage in humanized and WT mice. This study provides evidence of a protein-only replication process occurring in vivo and evidenced at a resolution of 3.8 Å. Although we cannot rule out the presence of additional fibril polymorphs different from 1BP in the 1B-inoculated mouse model that may have arisen through secondary nucleation and are undetectable by cryo-EM, the robust propagation of the 1B protofilament fold provides strong support for a prion-like templating mechanism. This notion is strengthened by our observation that brain homogenates containing 1BP can trigger in WT mice the same MSA inclusion pathology as the one seen after 1B injection, indicating that untemplated secondary nucleations, if any, do not have a substantial role in the MSA pathology seeded by 1B. This MSA kernel is shared in one of the two protofilaments of MSA fibrils extracted from patient 5 in the seminal study11, as well as the MSA fibrils resulting from the in vitro amplification of the same patient's sample12. To date, it was not possible to purify the fibrils naturally occurring in homozygous M83+/+ mice to compare them with the templated structure of 1BP using cryo-EM. However, we observed that brain homogenates from dying M83+/+ mice were not capable of eliciting any synucleinopathy when injected into the brain of WT mice like it is the case for 1BP-containing brain homogenates (see also Fig. This suggests that either the fibrils in the brains of spontaneously dying M83+/+ mice are a different polymorph, or the fibril titre is too low for stable transmission. Notably, the templating fidelity of the MSA kernel between 1B and 1BP is most pronounced in two particular regions—the N- and C-pockets. These two regions are of particular interest because they define the binding sites for the amyloid probe ThT33, but are closed and not accessible to the dye in the MSA kernel. Our data suggest that structural features of aSyn fibrils—specifically closed N- and C-pockets—are related to their ThT invisibility and their seeding properties, in particular their ability to seed GCIs. This is consistent with the fact that the three synthetic fibril assemblies that have been reported to induce GCIs in mice and rats are all weak ThT binders, and that the fibrils amplified from MSA patient brain samples and CSFs34 are ThT negative as well. This collectively points to a major pathophysiological role of C- and N-pocket closure with direct relevance to MSA. Compared with other aSyn fibril structures with an open C-pocket, in 1B/1BP, the inner hook of the pseudo-Greek-key motif along the stretches of Val74 to Phe94 is shifted by one amino acid position starting at Ala85. As a result, Ala85 points outwards at the most prominent position of the inner hook, which causes a consequential inversion of sidechain positioning for the remaining sequence. In 1B/1BP, this causes Ser87—a known modification site for fibril inhibition by O-Glc-N acylation42 or by phosphorylation43—to be inaccessible and Ile88 to be positioned outwards, undergoing a connection with Phe94 and therefore closing the C-pocket. Enforcing the post-translational modification of Ser87 or forcing open the C-pocket using small molecules might therefore represent promising means to derail the templating process during fibril growth, and force fibril elongation to take place on a modified template which has lost part of the MSA kernel. This would presumably not prevent fibril growth but could perhaps impinge on the acute replicative and toxic properties of MSA fibrils and slow down disease progression. The closed N-pocket is a shared characteristic of the 1B and 1BP structures, and this was also observed in the amplified MSA filaments from patient 5 (ref. However, fibrils extracted from patients with MSA11 did not share the closed N-pocket conformation (Fig. Whether this points to a dispensable pathogenic role of N-pocket closure remains to be determined. Finally, we note that the ability of crude brain homogenates from patients with MSA to seed pathological inclusions in WT mice has, to our knowledge, not been reported to date. This may be due to a lack of systematic investigation, or it may suggest that the fibrils present in human MSA brain samples are intrinsically less aggressive than 1B or 1BP fibrils. The latter would explain why human MSA brain extracts trigger little MSA-like pathology in M83+/− animals9. Notably, these include (1) a closed versus open N-pocket; (2) the pseudosymmetric arrangement of two identical protofilaments in 1B/1BP, as opposed to the asymmetric fold of two distinct protofilaments in MSA fibrils; and (3) additional, unattributed densities in 1B and 1BP, which—although distinct—may obscure chaperone clearance sites in both. The relevance of these structural differences remains to be determined. Regardless of the precise mechanism underlying the high seeding efficiency, it is important to emphasize that, in the absence of previous structural knowledge of synthetic or patient-derived α-synuclein fibrils, such materials should be handled with extreme caution. Taken together, this study demonstrates the existence of a synthetic aSyn fibril strain that is capable of inducing MSA-like pathology in WT mice and replicates in M83+/− mice through a protein-only templating process. The structures of the fibril seeds and of fibrils formed in vivo highlight the similarity of 1B and MSA fibrils, and postulate a link between fibrillar structure, assembly, mechanism of prion-like seeding and spreading, as well as animal-to-animal transmission. Escherichia coli strain BL21(DE3) was transformed with pT7-7 aSyn WT vector43 from Addgene, (plasmid 36046) and plated onto a Luria broth (LB) agar plate containing ampicillin (100 μg ml−1) and 1 g l−1 glucose. A preculture in 5 ml of LB medium was inoculated with one clone and incubated at 37 °C under 200 rpm shaking for 4 h. Cells from the LB preculture were recovered by centrifugation (1,000g, 10 min) and used for inoculating 200 ml of LB medium. Cells were grown overnight at 37 °C under 200 rpm shaking and then diluted in 2 l of culture. Protein expression was induced by adding 0.5 mM isopropyl-β-d-thiogalactopyranoside during the exponential phase, evaluated at an optical density at 600 nm of 0.6. Cells were collected after 5 h of culture at 30 °C by centrifugation at 7,000g and 4 °C for 20 min (JLA 8.1000, Beckman Coulter), and the pellet was kept at −20 °C until purification. The pellets were resuspended in lysis buffer (10 mM Tris and 1 mM EDTA (pH 7.2) with protease inhibitor tablets (cOmplete, EDTA-free protease inhibitor cocktail, Roche)) and sonicated at 50% maximum energy, 30 s on and 30 s off for three rounds with a probe sonicator (Q-Sonica). The pH of the supernatant was then reduced to pH 3.5 using HCl, and the mixture was stirred at room temperature for 20 min and then centrifuged at 60,000g for 30 min. The pH of the supernatant was then increased to pH 7.4 with NaOH and then dialysed against 20 mM Tris-HCl (pH 7.4) and 100 mM NaCl buffer before loading onto a 75 pg HiLoad 26/600 Superdex column equilibrated with the same buffer on the ÄKTA pure system. Monomeric fractions were collected and concentrated if needed by using the Vivaspin 15R 2 kDa cutoff concentrator (Sartorius Stedim). Purification fractions were analysed by polyacrylamide gel electrophoresis (PAGE) Tris-tricine 13% dying with ProBlue Safe Stain. Quantification of the preparations using the Pierce chromogenic LAL kit indicated a low endotoxin (lipopolysaccharide, LPS) residual value of 0.03–0.06 EU μg−1 of recombinant protein. Solutions of monomeric aSyn at 4 to 5 mg ml−1 in saline (H2O, 100 mM NaCl and 20 mM Tris-HCl pH 7.40) were sterilized by filtration through 0.22-μm Millipore single-use filters and stored in sterile 15 ml conical Falcon tubes at 4 °C. Sterilized stock was then distributed into safe-lock Biopur individually sterile-packaged 1.5 ml Eppendorf tubes as 500 μl aliquots and were seeded with 1% of 1B strain fibrils13,14,15. The tubes were cap-locked and additionally sealed with parafilm. All of the previous steps were performed aseptically in a particle-free environment under a microbiological safety laminar flow hood. The temperature was set to 37 °C with continuous shaking at 2,000 rpm for 4 days. Before intracerebral injections, 1B aSyn fibril stocks (4 mg ml−1) were distributed in cap-locked, sterile 0.5 ml PCR tubes (Thermo Fisher Scientific). Sonication was performed at 25 °C in the Bioruptor Plus water bath sonicator (Diagenode) equipped with thermostatic control and an automated tube carousel rotator. The sonication power was set to high, and 10 cycles of 30 s on followed by 10 s off were applied. In agreement with our previous quantifications14, over 80% of the fibrils were 50-nm long or less after application of this protocol (not shown). Adult 129SV (intrastriatal injections), C57BL/6 (nigral injections) and transgenic M83 (hemizygous, intrastriatal injections) mice were housed in a temperature-controlled (22 °C) and light-controlled environment under a 12 h–12 h light–dark cycle with forced ventilation (humidity below 50%) and with access to food and water ad libitum. All of the experimental procedures were conducted in accordance with the European Communities Council Directive (2010/63/EU) for care of laboratory animals. The mice (aged 6–8 weeks) unilaterally received 2 μl of sonicated aSyn fibrils 1B (4 mg ml−1) by stereotactic delivery at a flow rate of 0.4 µl min−1, and the pipette was left in place for 5 min after injection to avoid leakage. Delivery was performed within the right striatum (coordinates from bregma: anteroposterior (AP), −0.1; lateral (L), +2.5; dorsoventral (DV), +3.8), or above the right SN (coordinates from bregma: AP, −2.9; L, −1.3; DV, −4.5). WT mice (n = 20) and transgenic M83 hemizygous mice (n = 28) injected with either PBS, 1B fibrils or brain homogenates were euthanized after 6 weeks, 17 months, 24 months or when reaching the humane end points defined as death regarding the transgenic M83 mice, and were transcardially perfused with either Tris-buffered saline (pH 7.4) followed by 4% paraformaldehyde in PBS pH 7.4 at 4 °C, or with cacodylate buffer 0.1 M with 1 mM CaCl2 (pH 7.4) followed by 4% paraformaldehyde plus 0.1% glutaraldehyde in 0.1 M cacodylate buffer (pH 7.4). Brains were subsequently postfixed in the same fixative. For standard neuropathology, the brains were paraffin embedded, and 10-µm-thick sections were cut using a rotative microtome (Leica). The sections of interest were deparaffinized and processed for epitope retrieval: the slides were immersed in citrate buffer pH 6 (Dako Agilent Technologies) and placed into a pressure cooker (Bio SB) at 114–121 °C for 10 min. After a cooling period of 20 min, the slides were washed twice for 5 min in PBS at room temperature. The slides were then processed for simple or double immunofluorescence using the following primary antibodies diluted at 1:500, or their combinations: EP1536Y (Abcam) or pSyn#64 (Wako) for detecting pS129aSyn-positive inclusions; LB509 for detecting human aSyn (Abcam, mouse monoclonal); anti-SOX10 (Abcam, rabbit monoclonal) for detecting OL nuclei; EP1532Y and anti-TH raised in chicken (anti-tyrosine hydroxylase, Abcam) for detecting the nigrostriatal tract; and anti-IBA1 (Abcam) for detecting microglia. DRAQ7 or DAPI were used to image the nuclei. The AlexaFluor-coupled secondary antibodies were from Thermo Fisher Scientific (Alexa 488, 568 and 674). The sections were acquired using a Pannoramic slide scanner (3D HISTECH, MM France) in epifluorescence mode, and multichannel fluorescence optical sections of the samples were performed (thickness, <0.8 µm) using either a Zeiss CD7 platform of a Leica SP5 laser-scanning confocal microscope equipped with a spectral detector, 488, 561 and 633 nm laser lines, a motorized xy stage and a mixed stepping motor/piezo z controller. The pinhole was set to 1 airy unit. For 3D reconstructions and volume rendering/animations (corresponding to 360° tilt series of composite maximum pixel projections images), raw three-channel z-stack images were processed offline using Icy44 (v.2.4.0.0). Timed inbred-pregnant C57BL/6JOlaHsd (aSyn-knockout) female mice were received from Envigo 2 days before initiation of the primary culture. The pregnant mouse was euthanized by cervical dislocation, and the aSyn-knockout embryos (embryonic day 18) were surgically extracted and cold-euthanized. Cortices were collected from eight of them, pooled and dissociated enzymatically and mechanically (using a neural tissue dissociation kit, C Tubes and an Octo Dissociator with heaters; Miltenyi Biotech) to yield a homogenous cell suspension pool of eight individuals. The cells were then plated at 20,000 per well in 96-well plates (Corning, BioCoat poly-d-lysine imaging plates) in neuronal medium (MACS Neuro Medium, Miltenyi Biotech) containing 0.5% penicillin–streptomycin, 0.5 mM alanyl-glutamine and 2% NeuroBrew supplement (Miltenyi Biotech). The cultures were maintained under 5% CO2 at 37 °C in a humidified atmosphere. In such cultures, and under control conditions, neurons represented approximately 85 to 95% of the cell population; thus, for simplicity, they are here referred to as neurons. After 7 DIV, vehicle and sonicated 1B aSyn fibrils were added at a final concentration of 10 nM equivalent monomeric concentration. When relevant, neurons were infected at DIV 10 with AAV (adeno-associated virus) particles (multiplicity of infection, 1,000) carrying the cDNA of aSyn and its variants under the control of the human synapsin promoter, as previously described14. High content analysis (HCA) was performed after fixation and double immunofluorescence staining with the conformation-dependent SynF1 antibody (mouse monoclonal, BioLegend) and the human aSyn-specific MJFR1 antibody (rabbit monoclonal, Abcam) as previously described on images acquired at ×20 using the generic analysis module of an Incucyte S3 (Sartorius) system with Top-Hat segmentation14. For quantifications, the signal integral of the segmented areas in each field of view was recorded. Primary neuronal cultures from C57BL/6 embryos physiologically expressing endogenous mouse aSyn and not transduced with any AAV vector (Extended Data Fig. The CLEM protocol was performed as previously described26,27. In brief, sections (thickness, 20–40 µm) were cut from perfused mouse brain on a vibratome (Leica VT1200). Immunolabelling against aSyn pathology allowed for the identification of immunopositive sections to be used for further processing with EM. For this, the sections were incubated overnight with a primary EP1536Y (Abcam) antibody at 4 °C followed by Alexa-conjugated secondary antibody staining together with DAPI for cell nuclei identification. Fluorescence images were acquired using the Leica Thunder Tissue Imager equipped with the K8 fluorescent camera. Chosen sections with strong aSyn pathology were prepared for EM by post-fixing in reduced 2% osmium, thiocarbohydrazide and 2% osmium tetroxide, followed by contrasting in 2% uranyl acetate and lead aspartate treatment at 60 °C. The sections were dehydrated with an increasing acetone gradient, before infiltration with EMBED 812 resin for final flat embedding. The hardened sections were imaged again using light microscopy and the sections were overlayed with the overview fluorescence map by using identifiable tissue features. Regions of interest were marked and the coordinates used for laser dissection. The resulting blocks were laser cut using the Leica LMD7 system and glued onto a resin block. Serial sectioning with an ultra-microtome (Leica UC7) produced 120–200-nm-thick tissue sections that were alternatingly collected on EM grids (slot and hexagonal grids) or glass slides. The glass slides were further processed for IHC and Toluidine Blue staining. Light-microscopy images were acquired at ×40 magnification and correlated to each other for the generation of an image overlay of pathology signal (IHC) and high-contrast tissue morphology (Toluidine Blue). The resulting overlays were then correlated to low magnification (×640 nominal magnification) EM overviews for cell identification by using identifiable tissue features both in the LM and EM images. The identified pathology was finally imaged and merged at high magnification in a tile acquisition series. EM grids with tissue sections were placed on a droplet of etching solution (1% periodic acid in water, Sigma-Aldrich), with the sections facing down, for 2.5 min. The grids were washed three times for 2 min in double-distilled H2O, then washed twice for 2 min in washing buffer (stock solution of 10% BSA, Aurion, which was 1:50 diluted in double-distilled H2O), and blocked in blocking solution (Aurion) for 5 min to reduce non-specific binding. The grids were then incubated with primary antibodies (EP1526Y, Abcam, 1:50 diluted in washing buffer) for 60 min, washed 6 times for 2 min in blocking buffer and then incubated in a solution of gold nanoparticles with a size of 10 nm (protein A gold, Aurion, 1:50 diluted in washing buffer) for 90 min. They were washed three times for 2 min in blocking solution, three times for 2 min in washing buffer and four times for 1 min in double-distilled H2O. Finally, the grids were post-stained in 1% uranyl acetate for 1 min and washed three times for 10 s in double-distilled H2O. The sections were imaged using a transmission electron microscope (Thermo Fisher Scientific, Talos L120C), operated at an operating voltage of 120 kV, and images were recorded using a Thermo Fisher Scientific Ceta camera. Gold particles were detected automatically in the recorded EM images, using FIJI45, by applying a difference-of-Gaussian filter (sigma1 = 10 nm, sigma2 = √2 × sigma1). The resulting image was binarized with a manually selected threshold. Particle detections with an area smaller than 25 nm2 and a circularity (4π × area/perimeter2) below 0.8 were excluded from analysis. The gold particle detection was implemented as a FIJI macro and the density calculation as a Python notebook. Electron tomography data were collected using the Tomography software (Thermo Fisher Scientific) with multisite batch acquisition on the Thermo Fisher Scientific Talos F200C system with tilt series covering −60° to +60° in 2° steps. Tomograms were reconstructed using IMOD46 and filtered with a non-local-means filter with Amira (Thermo Fisher Scientific, Amira v.2021.2). For segmentation, nuclear membranes, fibrils and other membranous parts were manually annotated and traced throughout the tilt-series using Amira. Every few slices, the respective feature of interest was selected by drawing directly onto the image and then interpolated to generate an accurate 3D volume. The final volume was visualized directly in AMIRA. Extraction of the sarkosyl-insoluble fraction of a transgenic M83+/− mouse brain 16 weeks after inoculation with 1B was done using the same procedure as was previously published for fibrils purified from a patient with MSA11. Homogenates were brought to 2% sarkosyl and incubated for 30 min at 37 °C. The pellets were resuspended in 500 μl g−1 extraction buffer and centrifuged at 3,000g for 5 min. The supernatants were diluted threefold in 50 mM Tris-HCl, pH 7.5, containing 0.15 M NaCl, 10% sucrose and 0.2% sarkosyl, and centrifuged at 166,000g for 30 min. Sarkosyl-insoluble pellets were resuspended in 100 μl g−1 of 30 mM Tris-HCl, pH 7.4. A total of 3 µl of non-diluted sample was applied to 100 mesh Ni grids with formvar support (Electron Microscopy Sciences) that were glow-discharged at 15 mA for 60 s. After a 10 min incubation time, the grid was blotted, washed three times for 5 min with incubation solution (0.2% BSA-c/PBS; Aurion) and incubated for 1 h at room temperature in a solution of pS129aSyn antibody (EP1536Y, Abcam, ab51253) diluted 1:100 in incubation solution. The grid was washed six times for 5 min and incubated for 2 h in a solution containing gold nanobeads (diameter 6 nm) coupled to an anti-rabbit antibody (Aurion), diluted 1:50 in incubation solution. After washing six times for 5 min in incubation solution and three times for 5 min in PBS, the grid was immersed in 1% uranyl acetate for 40 s, blotted and air-dried. The grid was imaged on the Talos transmission electron microscope at an operating voltage of 200 kV (Thermo Fisher Scientific). Cryo-EM grids were prepared using the Leica GP2 plunge freezer in a BSL-2 environment at room temperature. Quantifoil 300-mesh gold grids were glow-discharged in air for 90 s. Grids were prepared using 3 µl of 1:1 and 1:2 diluted fibril samples in preparation buffer (H2O, 100 mM NaCl and 20 mM Tris-HCl, pH 7.40) and vitrification by rapid freezing in liquid ethane. The grids were screened for the presence of non-clustered fibres and high-resolution images were acquired with an automated EPU (v3.10) setup on the Thermo Fisher Scientific Titan Krios G4 electron microscope equipped with a cold-FEG 300 kV electron source and a Falcon4i camera. In total, 11,879 dose-fractionated videos (EER) were collected with a total dose of 40 e− Å−2 at a pixel spacing of 0.66 Å at the sample level. The sarkosyl-insoluble fraction of a transgenic M83+/− mouse brain was diluted 1:3 in Tris-HCl, 50 mM, 150 mM NaCl, pH 7.4 and 2.5 µl was applied to a Quantifoil 300-mesh gold grid, coated with an additional 5-nm-thick carbon layer. Then, 30 s after applying the sample, the grid was blotted from the front side for 6 s and plunge-frozen at room temperature and 80% humidity in a Leica GP2 plunge freezer. A total of 42,812 exposures was collected on the Titan Krios (Thermo Fisher Scientific) electron microscope equipped with a cold-FEG 300 kV electron source and a Falcon4i camera. The total dose applied per video (EER) was 50 e− Å−2 and the pixel spacing was 0.732 Å. All the subsequent image processing and helical reconstruction was carried out in RELION (v.4.0)47 (Extended Data Fig. The recorded videos were gain-corrected, motion-corrected and dose-weighted using RELION's own implementation of MOTIONCOR. The raw EER files were dose-fractionated to a total of 32 frames (1.23 e− Å−2 per frame). After CTF estimation using Ctffind (v.4.1)48, the aligned averages that had an estimated CTF resolution of better than 5 Å were selected, resulting in a total of 11,879 micrographs. aSyn fibres were picked manually as start-to-end helical segments and extracted with an interbox distance of 19 Å. Several rounds of reference-free 2D classification were performed to exclude suboptimal 2D segments from further processing. The four best classes were selected and binned four times to re-extract the segments with a bigger box size encompassing the entire helical crossover. Clearly visible twisted class averages containing the entire crossover were used to generate initial de novo models using the relion_helix_inimodel2d program. The most likely twist was estimated from a crossover distance measured from 2D class averages and corroborated with the direct measurements from micrographs. The helical segments (105,496) from the entire dataset were then re-extracted with a 600 pixel box size (binned by 2) comprising about 40% of the crossover. Exclusion of the suboptimal classes after 2D classification resulted in 64,889 particles. Multiple rounds of 3D autorefinement were carried out to optimize the helical twist and rise, and to check for the correct symmetry operators. This was validated from the reconstructions showing clear separation of β-strands along the helical axis. Imposing a 21 cork-screw symmetry (pseudo two-fold symmetry) yielded the clear separation of β-strands when compared to C1 and C2 symmetries. A 3D classification without image alignment was further performed to exclude the segments that gave suboptimal 3D volumes. This resulted in a final 51,272 particles. These were finally re-extracted at a 512 pixel (unbinned) box size and subjected to 3D autorefinement, followed by CTF refinement. Further iterations of Bayesian polishing coupled with CTF refinement yielded a map at 1.94 Å. The maps were sharpened using the standard post-processing procedure in RELION with an ad hoc B factor of 21 Å2. The atomic model for the backbone of the core of the 1B fibrils encompassing residues 34 to 95 was built de novo using Coot49 (v.0.9.8.96) Initially, three β-rungs were modelled in coot and were refined in real-space in PHENIX45 (v.1.21.2) Finally, these chains were extended to 9 β-rungs per protofilament and refined in tandem with phenix.real_space_refine and in coot to improve the Ramachandran and Geometric statistics. NCS and secondary structural restraints were included during the iterative refinement process. As most of the 42,812 collected videos did not contain any fibrillar material, they were first imported into cryoSPARC50 (v.4.7.0), motion corrected and manually inspected using a manually_curate_exposures job. After manual inspection, 156 videos with intact, twisting fibrils were selected for further processing in RELION447. They were dose-fractionated in RELION's MOTIONCOR implementation with 24 frames per fraction, corresponding to a dose of 1.28 e− Å−2 per frame, and CTF-corrected using Ctffind (v.4.1)48. Fibrils were found to cluster into three morphological groups (Extended Data Fig. Manual picking was done in three rounds, using 209, 7 and 40 videos for the three respective groups. One of the three morphological groups (40 videos) yielded 2D classes for which a clear twist and monomer separation was visible, while the other two morphological groups did not yield any interpretable result. Using the 2D class averages of the one usable group, a de novo initial model was generated using relion_helix_inimodel2d. Particles were re-extracted with a box size of 800 and a binning of 2 and, after multiple rounds of 2D classification, 1,913 particles were retained for a 3D classification job with 1 class and the de-novo inimodel2d reconstruction as initial model. When a fibril backbone was visible, the regularisation parameter (T) was gradually increased to 12 over the course of 50 iterations, and the result was used for a 3D refinement with a tau2 fudge factor of 2 to search for the optimal helical rise and twist. After CTF refinement and postprocessing, a 3D map at a resolution of 3.8 Å was obtained. To build the 1BP atomic model, one β-rung of the 1B model was fitted into the 1BP density using a rigid-body fit in ChimeraX51 (v.1.10.1), followed by a jiggle-fit and all-atom refinement in Coot49 (v.0.9.8.96). Position 53 was mutated into a threonine and the model was extended to five β-rungs, which were refined together in phenix.real_space_refine (PHENIX v.1.21.2). After refinement, the outer two β-rungs were removed in Coot. NCS and secondary structural restraints were included during the iterative refinement process. h-FTAA was synthesized and purified as previously described36 and paraffin-embedded sections of 1B-injected WT mouse brains (129SV) and regions from human donor brain samples were stained with h-FTAA and processed for immunofluorescence against pS129aSyn (EP1536Y) for the inclusions or against human aSyn (LB509) for the 1B inoculate and counterstained with the nuclear marker DAPI as previously described37. To confirm previous observations regarding the ability of h-FTAA to discriminate between Lewy bodies and GCIs using FLIM37, human brain samples from one individual with MSA and one individual with DLB were obtained from the repository of the University Hospital of Bordeaux (CRB-BBS). Participants or a legal representative had given informed written consent for collecting and using clinical and post-mortem data, and mandatory regulatory approval for post-mortem brain tissue use was obtained from the French Ministry of Higher Education and Research (MESR AC-2024-6715, DC-2024-6714). FLIM was performed on the Leica SP8 WLL2 confocal microscope on an inverted stand DMI6000 (Leica Microsystems), using the HCX Plan Apo CS2 ×63 oil NA 1.40 objective. This microscope was equipped with a pulsed white-light laser 2 (WLL2) with freely tuneable excitation from 470 to 670 nm (1 nm steps) and a diode laser at 405 nm. A FALCON module enabled us to perform FLIM measurement based on the time correlated single photon counting technique. Similitude analysis was performed using the Jaccard index52. For Gallyas silver impregnation, mouse brain sections were processed as previously described38 using a commercial Gallyas staining kit (Morphisto). 1B aSyn fibrils were deposited onto a freshly cleaved mica disk and left to adhere for 15 min in a humid chamber at room temperature. The sample was rinsed and imaged in saline buffer solution (H2O, HEPES 20 mM, NaCl 100 mM, pH 7.4). Atomic force microscopy (AFM) imaging was performed using the Dimension FastScan setup (Bruker) operating in PeakForce Quantitative Nano-Mechanics (PF-QNM) mode in liquid at room temperature, using sharp SNL-C probes (Bruker, Silicon tips on silicon nitride cantilevers), with a nominal spring constant of 0.24 N m−1, a resonance frequency of 56 kHz and a nominal tip radius of 2 nm. Fibrils were imaged with a peak force frequency of 1 kHz, a scan rate of 0.5 Hz and a constant setpoint force of 500 pN. For all experiments/conditions without quantitative analyses, the images shown were representative of at least three independent observations. In all other cases, statistical analyses data were presented as means ± s.d. or as otherwise stated in the figure legends, and were performed using GraphPad Prism (v.9). The numbers of independent samples as well as the statistics tests used are indicated in the figure legends for each experiment. No statistical methods were used to predetermine sample size. Sample size was determined based on our previous publications. Data assignment, organization and collection were randomized. Data collection and analysis were not performed blindly. No animals or datapoints were excluded from the analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. For 1B and 1BP, the raw cryo-EM images are available at the EMPIAR database under accession numbers 12012 and 12890, respectively. Source data are provided with this paper. Dickson, D. W., Lin, W., Liu, W. K. & Yen, S. H. Multiple system atrophy: a sporadic synucleinopathy. Multiple system atrophy: an oligodendroglioneural synucleinopathy. Dickson, D. et al. Glial cytoplasmic inclusions (GCI) of multiple system atrophy (MSA) contain α-synuclein. Tu, P. et al. Glial cytoplasmic inclusions in white matter oligodendrocytes of multiple system atrophy brains contain insoluble α-synuclein. Wakabayashi, K. et al. Accumulation of α-synuclein/NACP is a cytopathological feature common to Lewy body disease and multiple system atrophy. Gai, W., Power, J., Blumbergs, P. & Blessing, W. Multiple-system atrophy: a new α-synuclein disease? Transmission of multiple system atrophy prions to transgenic mice. Evidence for α-synuclein prions causing multiple system atrophy in humans with parkinsonism. Structures of α-synuclein filaments from multiple system atrophy. Lövestam, S. et al. Seeded assembly in vitro does not replicate the structures of α-synuclein filaments from multiple system atrophy. De Giorgi, F. et al. Neurons with cat's eyes: a synthetic strain of α-synuclein fibrils seeding neuronal intranuclear inclusions. Novel self-replicating α-synuclein polymorphs that escape ThT monitoring can spontaneously emerge and acutely spread in neurons. De Nuccio, F. et al. Oligodendrocytes prune axons containing α-synuclein aggregates in vivo: Lewy neurites as precursors of glial cytoplasmic inclusions in multiple system atrophy? & Lantos, P. L. The distribution of oligodendroglial inclusions in multiple system atrophy and its relevance to clinical symptomatology. Kato, S. et al. Argyrophilic ubiquitinated cytoplasmic inclusions of Leu-7-positive glial cells in olivopontocerebellar atrophy (multiple system atrophy). & Lantos, P. L. Accumulation of tubular structures in oligodendroglial and neuronal cells as the basic alteration in multiple system atrophy. Spillantini, M. G., Crowther, R. A., Jakes, R., Hasegawa, M. & Goedert, M. α-Synuclein in filamentous inclusions of Lewy bodies from Parkinson's disease and dementia with Lewy bodies. Gai, W. P., Power, J. H. T., Blumbergs, P. C., Culvenor, J. G. & Jensen, P. H. α-Synuclein immunoisolation of glial inclusions from multiple system atrophy brain tissue reveals multiprotein components. Peelaerts, W. et al. α-Synuclein strains cause distinct synucleinopathies after local and systemic administration. Slow progressive accumulation of oligodendroglial alpha-synuclein (α-Syn) pathology in synthetic α-Syn fibril-induced mouse models of synucleinopathy. Lin, W.-L., DeLucia, M. W. & Dickson, D. W. α-Synuclein immunoreactivity in neuronal nuclear inclusions and neurites in multiple system atrophy. Woulfe, J. M. Abnormalities of the nucleus and nuclear inclusions in neurodegenerative disease: a work in progress. Nishie, M., Mori, F., Yoshimoto, M., Takahashi, H. & Wakabayashi, K. A quantitative investigation of neuronal cytoplasmic and intranuclear inclusions in the pontine and inferior olivary nuclei in multiple system atrophy. Shafiei, N. et al. Correlative light and electron microscopy for human brain and other biological models. Shahmoradian, S. H. et al. Lewy pathology in Parkinson's disease consists of crowded organelles and lipid membranes. St-Pierre, M.-K. et al. Ultrastructural characterization of dark microglia during aging in a mouse model of Alzheimer's disease pathology and in human post-mortem brain samples. Tremblay, M. -È Microglial functional alteration and increased diversity in the challenged brain: insights into novel targets for intervention. Distinct ultrastructural phenotypes of glial and neuronal alpha-synuclein inclusions in multiple system atrophy. Robust α-synuclein pathology in select brainstem neuronal populations is a potential instigator of multiple system atrophy. Guerrero-Ferreira, R. et al. Cryo-EM structure of alpha-synuclein fibrils. Structural mechanism for specific binding of chemical compounds to amyloid fibrils. Shahnawaz, M. et al. Discriminating α-synuclein strains in Parkinson's disease and multiple system atrophy. Cooperative amyloid fibre binding and disassembly by the Hsp70 disaggregase. Rasmussen, J. et al. Amyloid polymorphisms constitute distinct clouds of conformational variants in different etiological subtypes of Alzheimer's disease. Klingstedt, T. et al. Luminescent conjugated oligothiophenes distinguish between α-synuclein assemblies of Parkinson's disease and multiple system atrophy. Uchihara, T. et al. Silver stainings distinguish Lewy bodies and glial cytoplasmic inclusions: comparison between Gallyas-Braak and Campbell-Switzer methods. The E46K mutation modulates α-synuclein prion replication in transgenic mice. Woerman, A. L. et al. Familial Parkinson's point mutation abolishes multiple system atrophy prion replication. Luk, K. C. et al. Molecular and biological compatibility with host alpha-synuclein influences fibril pathogenicity. Balana, A. T. et al. O-GlcNAc forces an α-synuclein amyloid strain with notably diminished seeding and pathology. Oueslati, A., Paleologou, K. E., Schneider, B. L., Aebischer, P. & Lashuel, H. A. Mimicking phosphorylation at serine 87 inhibits the aggregation of human α-synuclein and protects against its toxicity in a rat model of Parkinson's disease. Icy: an open bioimage informatics platform for extended reproducible research. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Kremer, J. R., Mastronarde, D. N. & McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. Kimanius, D., Dong, L., Sharov, G., Nakane, T. & Scheres, S. H. W. New tools for automated cryo-EM single-particle analysis in RELION-4.0. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. Jaccard, P. Étude comparative de la distribution florale dans une portion des Alpes et du Jura. Cryo-EM data were in part collected at the Dubochet Center for Imaging (DCI) in Lausanne. We thank A. Myasnikov for expert technical assistance in cryo-EM data collection; Æ. Letourneur and F. Cavallaro for their help with primary culture and immunofluorescence; F. Serinelli for support with paraffin histology; S. Dovero and N. Biendon for support with Pannoramic slide scanning; B. Dehay for sharing the AAV-Syn mutant vectors; M. Picquemal for support with animal surgery; L. Lakhdar, A. de César and D. Gaillard for animal experiments in M83 mice; S. Lacomme and F. Cordelières from the Bordeaux Imaging Center for their help with sample fixation for CLEM and with the Zeiss CD7 platform respectively. Room-temperature EM was collected in part at the electron microscopy facility (EMF) of the University of Lausanne. This work was supported by the Swiss National Science Foundation (grants 310030_188548 and 10.002.679), the Swiss Parkinson Foundation, the Department of Excellence Initiative of the Italian Ministry of Research, the Innovative Medicines Initiative 2 Joint Undertaking under grant agreement no. 116060 (IMPRiND), supported by the European Union's Horizon 2020 research and innovation program and EFPIA, the French Agence Nationale de la Recherche (grants ANR-21-CE18-0045-03, ANR-22-CE16-0002 and ANR-24-CE93-0003), and the Fondation pour la Recherche Médicale, grant MND202411019711. These authors contributed equally: Domenic Burger, Marianna Kashyrina, Lukas van den Heuvel Domenic Burger, Lukas van den Heuvel, Amanda J. Lewis, Inayathulla Mohammed & Henning Stahlberg Domenic Burger, Lukas van den Heuvel, Amanda J. Lewis, Inayathulla Mohammed & Henning Stahlberg Institut des Maladies Neurodégénératives, UMR 5293, Univ. Marianna Kashyrina, Hortense de La Seiglière, Marie-Laure Arotcarena, Aude Retailleau, Erwan Bezard, Marie-Hélène Canron, Wassilios G. Meissner, Florent Laferrière, Francesca De Giorgi & François Ichas Cécile Feuillie, Mélanie Berbon & Antoine Loquet Laboratoire des Maladies Neurodégénératives, UMR 9199, Univ. Bordeaux Imaging Center, UAR 3420, US 4, Univ. Laboratory of Human Anatomy, Department of Biological and Environmental Sciences and Technologies, Univ. Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar determined the 1B structure, performed CLEM experiments and immunogold labelling and wrote the manuscript. performed the AFM analysis of 1B fibrils. performed post-mortem brain perfusion/fixation of C57BL/6 WT mice. contributed to setting up the fibril isolation procedures from mouse brains and participated in early characterizations of 1B. designed experiments, performed the HCA experiments, as well as the confocal and slide-scanning neuropathology analyses of WT and M83 mice, and wrote the manuscript. conceived and managed the project, produced aSyn and fingerprinted 1B, performed the FLIM experiments, performed the HCA experiments, as well as the confocal and slide scanning neuropathology analyses of WT and M83 mice, and wrote the manuscript. Correspondence to Henning Stahlberg or François Ichas. The presented work was conducted before his employment and had no influence on the design, analysis, interpretation or reporting of the results of this study. The other authors declare no competing interests. Nature thanks Veerle Baekelandt, Michel Goedert and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Topographical (Height Sensor) and nano-mechanical mapping (Derjaguin-Müller-Toporov (DMT) Modulus, log representation) of 1B fibrils using AFM in liquid. Right panels: fibril height values (see cartoon for definitions) for n = 103 measurements (plot population statistics descriptors as shown in plot inset), and left twist periodicity for n = 12 fibrils. Examples of early “seeding junctions” between 1B and the nascent seeded inclusions in neuronal processes two weeks post seeding (arrowheads = seeding junctions). 96-well pooled primary cultures of cortical neurons from 8 wild-type mouse embryos (C57BL/6) were challenged at day-in-vitro 7 (DIV7) with 10 nM 1B fibrils (equivalent aSyn monomer concentration). The neurons were fixed two weeks later at DIV24 and both the 1B input fibril remnants (red) and the seeded aSyn inclusions (green) were revealed. Indeed, we took advantage of the fact that synthetic 1B is made of human aSyn and thus that it can be specifically recognized by LB509 which is a human aSyn-specific antibody. The seeded inclusions were revealed contemporary using EP1536Y against pS129 aSyn. This allowed us to observe many seeding junctions in the neuronal processes (arrows). Note that in a majority of images, and in agreement with early reports, the seed (1B in our case) is not phosphorylated, and that the growing seeded inclusions are instead pS129Syn-positive. At later time points 1B seeds get overwhelmed and are no longer discernible in the mass of the seeded inclusions (not shown). The disappearance of the 1B seeds is even faster in vivo (within a few days, not shown). Depending on the neuroanatomical structure and the time point under consideration, experimental GCIs (blue) coexisted with a variable load of surrounding neuronal somatic aSyn inclusions (NCIs & NIIs, both in red), which represent the forefront of the somatic aSyn pathology at earlier time points to progressively make way to a growing population of GCIs. 1f and were stained with DAPI for nuclei, Sox10 to identify the oligodendrocytes, and EP1536Y for pS129Syn. All the inclusions were manually annotated according to these 3 markers. Anatomical quantification regions were as follows: Am: amygdala; ACa: anterior limb of the anterior commissure; ACp: posterior limb of the anterior commissure; BN: bed nucleus of the stria terminalis; CX: cortex; St: caudate putamen (striatum); SN: substantia nigra. Note that NIIs form especially early (at 6 weeks) in the SN and the AM, then in other regions at 12 months, to eventually disappear but in the AM at 24 months. The mean number of GCIs in all regions at 24 months is significantly increased compared to the 2 other time points (Mean ± SD of the regions shown together with the individual region inclusion number values, * = one-way ANOVA p < 0.05: p = 0.0462 for 6w/24 m, p = 0.0278 for 12 m/24 m). a. aSyn inclusions are revealed using the anti-phospho aSyn antibody #64 (green), dopaminergic neurons using an anti-tyrosine hydroxylase antibody (TH, red) and nuclei using DAPI (white). Left and right SNs are visible with a loss of immunoreactivity symmetry with depletion of TH-positive neurons and unilateral aSyn inclusions on the right side. The TH-depleted/aSyn inclusion-rich region is enlarged in b, the corresponding left region is enlarged in c. d. Semi-topographic view of the n + 1 horizontal section consecutive to (a). The right SN enlarged in f contains numerous GCIs and sparse microglial cells. a-c. High magnification EM images of different types of fibril arrangements, indicated by hand made drawings in the upper row panels, and without indication in the lower row panels. a: “dispersed”, b: “bundled”, and c: “clustered”. d. Tomogram segmentation of aSyn fibril bundles. Tomograms were manually segmented to visualize the fibril bundling (blue), the nuclear membrane (red), the cell membrane (yellow), and other membranous compartments (green). a. Immunohistochemistry (IHC) labelling (inset) indicates two regions with aSyn pathology wrapping around the nucleus of an affected cell. The correlated EM image with on-grid immunolabelling against aSyn (left) shows an increase of gold particles in comparison to the background as indicated by the heatmap (right). Higher magnifications are indicated by the labelled boxes. b. Zoomed view on the left side of the nucleus, highlighted are the gold particles in red, laying on top of dispersed fibrils. The zoomed area shows four gold particles laying on top of fibril pathology. d. Zoom of right-side pathology, with highest concentration of gold particles, laying on fibril bundles. Cells with a distinct morphology, previously described as “dark microglia”, can be found in close proximity or in direct contact with pathological cells (white arrowhead). In this specific case the fibrillar pathology of the oligodendrocyte appears in a dispersed arrangement with two smaller clusters (red arrowheads). Total area of immunopositive IHC is indicated by a red dashed line. Both oligodendrocytes contain aSyn pathology, however, for the upper cell volume, it is not visible in this EM slice (based on IHC positive signal from other EM grids). Secondary beta-sheets are indicated in consecutive order by β1-9. c. Surface colour representation by the electrostatic potential. d. 3D class averages from early processing stages indicate a homogeneous sample. e. Fourier Shell Correlation (FSC) curve of 1B with cutoff at 0.143 of the final map at 1.94 Å. (a) Negative staining combined with immunolabeling against aSyn-pS129 reveals that, amid aggregated precipitates, individual aSyn fibrils are present. (b-d) Cryo-EM micrographs (left) of intact fibrils observed in the sarkosyl-insoluble fraction, together with their power spectral density (right). A reflection at ~4.8Å (white arrowheads) reveals that these are amyloid fibrils. Presumably, the fibrils in (b), (c) and (d) represent 3 distinct conformations as can be seen from their different morphologies, although the particle counts are too low to verify this with a 3D reconstruction. The crossover distance (X) and largest diameter (ø) are indicated, as well as the number of micrographs (N) displaying at least one crossover of a fibril of the same morphological group as determined by manual inspection. The fibrils in group (d) did eventually yield a 3.8Å structure, while those in groups (b) and (c) did not yield any structure despite many attempts. (f) 3D helical reconstruction of 1'913 particles selected from group (d), before (top) and after (bottom) post-processing in Relion. It was previously reported that FLIM of h-FTAA can be used in histological brain sections to differentiate the aSyn fibrils populating Lewy bodies (LBs) from those populating GCIs in MSA37. These findings established h-FTAA as a conformation-sensitive amyloid probe capable of discriminating distinct aSyn fibril strains in human brain sections via FLIM37. In agreement with these previous observations, FLIM of h-FTAA allowed us to derive fibril strain-specific phasor plot signatures directly on brain tissue sections. Using this approach, we confirmed that h-FTAA FLIM could indeed discriminate the pS129Syn-positive inclusion pathology of MSA vs. DLB (a: h-FTAA intensity images - pS129Syn not shown) and used the disease-specific hotspots in the respective FLIM phasor plots to define 2D quantification gates (b,c) (α, β: MSA-specific; χ: DLB-specific). The fluorescence lifetime of the gated pixels is shown in d for respectively GCIs (26,6 million inclusion pixels gated) and LBs (3.3 million inclusion pixels gated). In agreement with previously published data37, the distribution histograms make it clear that the majority of pixels belonging to LBs are characterized by a fluorescence lifetime which is in the nanosecond range, while those populating GCIs show an approximately halved fluorescence lifetime with 2 distinguishable populations. The phasor plot gates discriminating GCIs and LBs described in Extended Data Fig. 9 were applied in the phasor plots of the intracerebral 1B fibril inoculate (positive to the anti-human aSyn antibody LB509, not shown), still observable in the CP of wild-type mice 24 h after injection (a, from left to right: h-FTAA intensity image, corresponding phasor plot with gates and quantifications, distribution of gated pixels in image) or in the plots characterizing the inclusions that formed in neurons and OLs during the 16 months following 1B injection (pS129Syn-positive, not shown) (b). The number of inclusion pixels quantified by FLIM and gated in α, β, and χ was >106 in each case, and the distribution of the pixel population in the phasor gates α+β, and χ was eventually used to score the similarity of the different fibril inclusion types (Supplementary Information Table 3). The MSA-specific silver impregnation technique of Gallyas highlights somatic (yellow arrowheads) and neuritic (white arrowheads) inclusions which developed during 16 months in the substantia nigra of WT mice injected with 1B (a); other section in the same region reveals pS129Syn-positive inclusion pathology (b). 96-well pooled primary cultures of cortical neurons from 8 aSyn-KO mouse embryos were challenged or not at day-in-vitro 7 (DIV7) with 10 nM 1B fibrils (equivalent aSyn monomer concentration), and infected or not with AAV-aSyn or AAV-E46K aSyn at DIV11. The neurons were fixed at DIV30 and the aSyn inclusions revealed using the conformation-dependent antibody SynF1 (red); three immunofluorescence fields of views (FOVs) imaged in a test well for each condition, illustrate: (i) the absence of SynF1and human aSyn signal at DIV30 in the absence of endogenous aSyn (aSyn-/- line, left column), (ii) aSyn inclusion seeding by 1B in conditions of aSyn expression (aSyn -/- + 1B line, middle and right column), and (ii) a minor inclusion seeding when E46K aSyn is expressed instead of wild-type aSyn (aSyn -/- + 1B line, right versus middle column). High Content Analysis of the experiment shown in (a) and extended to a series of aSyn mutants based on 18 FOVs from 6 wells for each condition. Mean ± SD as well as individual values are plotted for each condition The AAV-aSyn condition was doubled to document robustness. Kruskal-Wallis test followed by a post-hoc Dunn's test for multiple comparisons indicates that E46K aSyn is the only mutation reducing inclusion seeding by 1B (p = 0.0001). In the AAV-only conditions, human aSyn deriving from AAV infection was quantified using the human aSyn-specific antibody MJFR1 (no endogenous mouse aSyn is present in these neurons). The results were compared with the same test and post-hoc test combination as before, which indicated the absence of statistically significant differences in terms of levels of expression driven by the different AAVs. In particular, no statistically significant difference existed between AAV wt aSyn and AAV E46K aSyn in terms of expression levels, indicating that the difference seen with SynF1 after seeding with 1B was not due to a lesser expression of E46K aSyn. Detailed processing overview representing individual jobs within RELION (white boxes), with the black line indicating subsequent steps. Red boxes indicate estimated resolution at given Refinement steps. Blue dotted line represents the reference volumes used. The background colour represents data preprocessing (grey), binned processing (green), and unbinned processing (yellow). The remaining 19'636 micrographs were processed after the data acquisition had completed and were later introduced to the dataset at step 6 and 11 respectively. For initial analysis manually traced fibril segments were extracted with a box size of 600px, binned to 300px. As such two groups were analysed: Add values. Manual picking of particles was done in parallel to the processing of the extracted particles from 3. Initial 3D-Classification was done with a featureless cylinder as a low-pass filtered reference volume. The full particle batch was cleaned by multiple rounds of 2D classification and was subsequently refined to 2.66Å with a reference volume from the previously obtained 4.95Å structure from the initial screening. With a well-aligned structure, the data was transitioned to unbinned processing, and the respective particles were re-extracted. No further manual selection of the full dataset was made. All particles are joined together for the final processing. The structure was subjected to Refinement and PostProcessing with intermediate steps of Polish and 13. CTF refinement until the final high-resolution structure was obtained. Processing concluded when no further improvement in the iterative process could be made. Supplementary Table 1: structural comparison of different aSyn folds with 1B. The best fit subunits were then further compared on three regions on the Cα r.m.s.d. values in respect to the different pocket regions and intermediate sequence based on ChimeraX r.m.s.d. of less than 1 Å suggests high similarity in the structural conformation. Supplementary Table 2: cryo-EM data collection, refinement and validation statistics. Supplementary Table 3: in situ comparison of 1B inoculate, 1B-seeded inclusions, GCIs in MSA and LBs in DLB using h-FTAA FLIM analysis of human and WT mouse brain tissue sections. Inclusion pixels quantified by FLIM and gated in α, β and χ in Extended data Fig. 10 (>106 in each case) were used for comparison of inoculate and inclusions. For each condition, the percentage of the inclusion pixel population situated in the phasor gates α + β (MSA specific) and χ (DLB specific) (Extended data Fig. Similarity matrix analysis demonstrates that both the input 1B fibrils and the resulting inclusions seeded in WT mice exhibit a high degree of conformational similarity which is shared with the MSA fibrils, and distinct from the DLB ones. Colour code: green, similar; yellow, dissimilar. Section hovering overview corresponding to Fig. 3D animation of a Tyrosine Hydroxylase (red) positive neuron from the substantia nigra of the mouse sampled in Fig. Both NCIs and NIIs can be observed without need for a nuclear marker 5 panel b, lower left: The NCI of the cortical neuron (bigger nucleus) seems to protrude within the interior of the lining glial cell (smaller nucleus): DAPI in white, pS129Syn in green, Tyrosine Hydroxylase in red (not expressed in this region) Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Burger, D., Kashyrina, M., van den Heuvel, L. et al. Synthetic α-synuclein fibrils replicate in mice causing MSA-like pathology. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09677-6'>Structural snapshots capture nucleotide release at the μ-opioid receptor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 16:06:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. As a member of the G protein-coupled receptor superfamily, the μ-opioid receptor (MOR) activates heterotrimeric G proteins by opening the Gα α-helical domain (AHD) to enable GDP–GTP exchange, with GDP release representing the rate-limiting step1,2. Here, using pharmacological assays, we show that agonist efficacy correlates with decreased GDP affinity, promoting GTP exchange, whereas antagonists increase GDP affinity, dampening activation. Further investigating this phenomenon, we provide 8 unique structural models and 16 cryogenic electron microscopy maps of MOR with naloxone or loperamide, capturing several intermediate conformations along the activation pathway. These include four GDP-bound states with previously undescribed receptor–G protein interfaces, AHD arrangements and transitions in the nucleotide-binding pocket required for GDP release. Naloxone stalls MOR in a ‘latent' state, whereas loperamide promotes an ‘engaged' state, which is structurally poised for opening of the AHD domain and subsequent GDP release. These findings, supported by molecular dynamics simulations, identify GDP-bound intermediates and AHD conformations as key determinants of nucleotide exchange rates, providing structural and mechanistic insights into G protein activation and ligand efficacy with broad implications for G protein-coupled receptor pharmacology. Among the superfamily of G protein-coupled receptors (GPCRs), MOR stands out as one of the most clinically important, and is extensively targeted for pain management3,4. However, the high abuse liability and association with life-threatening side effects of MOR-targeting agonists has made addressing these concerns a priority in current drug development strategies5,6,7. Several modulators are available to treat opioid overdose, with naloxone as the therapeutically most widely deployed example8. It is commonly considered an antagonist, whereas some previous literature has demonstrated low efficacy partial agonist activity9,10,11. Partial agonists have generally emerged as promising candidates for achieving safer therapeutic profiles, over conventionally used MOR agonists12. However, the ability to design ligands with customized efficacy is limited by an incomplete mechanistic understanding of how different levels of agonism are transmitted intracellularly. Opioid agonists regulate signalling by binding to opioid receptors, which primarily signal through Gi/o-type G proteins. MOR-mediated signalling involves an interaction with the G protein that promotes the transition of the G protein from its inactive GDP-bound heterotrimeric form (Gαβγ) to an active state, in which GTP replaces GDP and the complex dissociates into GTP-bound Gα and Gβγ subunits. The rate-limiting step in this process is the release of GDP from the Gα subunit, which is tightly bound under resting conditions1,2. Following activation, the subunits then engage downstream effectors to propagate signals such as adenylyl cyclase inhibition and potassium channel stimulation. In the Gα subunit, nucleotide binding, exchange and GTP hydrolysis are governed by conserved structural motifs at the interface between the Ras-homology domain (RHD) and AHD. On the AHD, Switch I together with αE and αF provide extra coordinating contacts. Receptor-induced extension, rotation and translation of the α5 helix into the receptor are proposed to destabilize nucleotide binding, facilitating GDP release13. In GPCR pharmacology, ligands are classified on the basis of their mode and extent of modulation of downstream signalling, a property known as efficacy: antagonists block activation, full agonists produce maximal activation, partial agonists yield submaximal responses and superagonists yield supramaximal responses, relative to a reference agonist14. A central question in GPCR pharmacology therefore concerns how different levels of ligand efficacy are linked to conformational states of the receptor–G protein complex. Although a variety of models have been proposed in the literature, the varying capacity of agonists to activate heterotrimeric G proteins has been linked to decreasing GDP affinity, increasing GTP affinity or a combination thereof2,15,16,17,18. In the past, several groups have reported a direct correlation between the efficacy of GPCR agonists and a corresponding decrease in GDP affinity at the cognate G protein16,17,19,20,21. Further, single molecule Förster resonance energy transfer spectroscopy and double electron–electron resonance experiments at MOR described a correlation between ligand efficacy and GDP release rates, by modulating the equilibrium of nucleotide-bound receptor–G protein complex states18,22. In summary, GDP-bound states have a critical role in G protein activation. However, high-resolution structural snapshots of these intermediate states have thus far been lacking, limiting our mechanistic understanding of GPCR signalling. Here we report pharmacological assays, single particle cryogenic electron microscopy (cryoEM) experiments and molecular dynamics (MD) simulations, confirming that efficacy is indeed correlated with GDP release rates, which are modulated by the stabilization of specific conformational states. To investigate the contradictory description of naloxone efficacy, we initially performed in vitro signalling assays. We deployed bioluminescence resonance energy transfer (BRET)-based biosensors monitoring the status of the G protein heterotrimer23,24 (TRUPATH), receptor–G protein (RG-BRET)23,25, the ONE-GO toolkit26 and a cyclic AMP (cAMP) biosensor (GloSensor) as a downstream signalling assay. In these assays, we used the inverse agonist alvimopan, naloxone, partial agonist mitragynine pseudoindoxyl, the reference full agonist (d-Ala2, N-MePhe4, Gly-ol)-enkephalin (DAMGO) and the superagonist loperamide (Lop). Naloxone fell within the range of efficacies between alvimopan and mitragynine pseudoindoxyl, unequivocally confirming partial agonist activity in all assay systems, ranging from 10% to 30% (Extended Data Fig. To confirm partial agonism of naloxone at MOR, we conducted radioligand binding experiments using [³H]naltrexone, a close chemical analogue of naloxone, which showed a decrease in binding affinity with increasing concentrations of GDPβS (Ki 0.35 nM → 27.2 nM, Extended Data Fig. This is in agreement with pioneering work that showed that opioid agonist binding is negatively affected by the presence of nucleotides, although naloxone binding was shown to be unaffected by nucleotides in these studies27. Next, we tested the hypothesis linking ligand efficacy to nucleotide binding affinities by adopting the RG-BRET assay23. In this assay, which we coined ‘nuc-BRET', we detect nucleotide dose-dependent changes in BRET signal, as a proxy of nucleotide affinities. To minimize the competition with endogenous nucleotides, we permeabilized cells expressing MOR–Rluc8 and Gαi1β1γ2–GFP2 with digitonin, and hydrolysed nucleotides using apyrase. Next, we carried out a nucleotide dose–response experiment, using non-hydrolysable nucleotide derivatives (GDPβS and GTPγS, Extended Data Fig. MOR in the apo state showed a GDPβS half-maximal effective concentration (EC50) of 57 ± 14 nM (GTPγS 3.8 ± 0.18 nM). Confirming the effect of agonists on decreasing GDP affinities, we observed a roughly 30-fold increase in GDPβS EC50 of 1.7 ± 0.7 µM with loperamide (GTPγS 25 ± 4.4 nM). Alvimopan increased the affinity below the apo condition with GDPβS EC50 of 14.7 ± 10 nM (GTPγS 4.1 ± 0.73 nM), all remaining compounds showed a detectable decrease, including naloxone with GDPβS EC50 of 89 ± 8.5 nM (GTPγS 3.9 ± 0.36 nM), mitragynine pseudoindoxyl with GDPβS EC50 of 150 ± 39 nM (GTPγS 2.0 ± 0.64 nM) and DAMGO with GDPβS EC50 of 0.7 ± 0.16 µM (GTPγS 11.6 ± 4.1 nM). All agonists had a modest but consistent negative effect on GTP affinities. These findings provide a clear link between ligand efficacy and GDP affinities. To further validate our findings in the cellular context, we performed experiments with intact cells (Supplementary Fig. We observed the same overall trends, however, signal changes for alvimopan and naloxone were too small to interpret, probably due to the competition with endogenous nucleotides. Average cytoplasmic concentrations are estimated at 50 µM of GDP and 500 µM of GTP28, suggesting a constant competition pressure between the two nucleotides. Although all conditions showed some degree of competition, GTP binding was only minimally affected in the agonist bound state and was more affected in the presence of an inverse agonist (Extended Data Fig. 2e), suggesting that inverse agonists act by enhancing the competition between the two nucleotides. Finally, we show a clear inverse correlation between signalling efficacies across all signalling assays and the respective GDPβS EC50s (R2 = 0.86, Extended Data Fig. To gain mechanistic insights into GDP affinity modulation, we next turned to single particle cryoEM experiments (Extended Data Figs. On the basis of our nuc-BRET experiments, we speculated that we would be able to capture GDP-bound MOR–Gi complexes, using wild-type receptor and G protein sequences. To prepare GDP-bound specimens, we incubated nucleotide-free complexes bound to either naloxone or loperamide, before grid freezing with 100 µM GDPβS for 1 h on ice (‘re-bound GDP'). Our rationale for this protocol (‘re-bound GDP') is based on a rich history of experimental approaches in GPCR pharmacology. Given that GDP release and binding are thought to be reversible18,29,30, many experimental strategies begin with nucleotide-free complexes, followed by the supplementation with GDP, to investigate the role of GDP-bound GPCR–G protein complexes. These include radioligand binding27,31, [35S]GTPγS binding15,32, single molecule Förster resonance energy transfer spectroscopy21,22, hydrogen/deuterium exchange mass spectrometry33,34, BRET assays35,36 and structural biology experiments37. To validate our findings, we also performed control experiments with GDP present at all times during the purification (‘constant GDP'), as outlined below (Extended Data Fig. We confirmed that cellular concentrations of Mg2+ had no detectable effect on GDP affinities, while significantly improving GTP affinities (Extended Data Fig. 2f,g) and, hence, omitted Mg2+ from our biochemical preparations of GDP-bound MOR–Gαi complexes. Using a single particle cryoEM pipeline, we obtained six conformational states along the MOR activation pathway. Among these, we observed four distinct GDPβS-bound states, obtained from the respective specimen (Fig. 1a), providing structural snapshots of the initial steps of G protein engagement and activation. We named these states latent, engaged, unlatched and primed, between which we observed a series of conformational rearrangements showing a progressive decrease in the number of GDP-coordinating residues (Fig. In addition, to obtain data points before G protein coupling and after nucleotide release, we determined an inactive MOR (3.4 Å) and a nucleotide-free MOR–Gi structure (3.0 Å). Considering underlying mechanisms of ligand efficacy, throughout the article, we are describing a detailed comparison between: (1) naloxone ‘latent' (3.8 Å), (2) loperamide ‘engaged' (3.2 Å), (3) loperamide ‘unlatched' (3.2 Å) and (4) loperamide ‘primed' (2.9 Å). Starting with this series of structures, we also performed extensive MD simulations. a, Overview of cryoEM reconstructions of MOR and MOR–Gαi1 complexes in complex with naloxone or loperamide in six conformational states, with or without GDPβS present: (1) inactive, (2) latent, (3) engaged, (4) unlatched, (5) primed and (6) nucleotide-free. b,c, Corresponding densities of GDPβS in G protein structures. b, Naloxone: latent, unlatched, primed. c, Loperamide: engaged, unlatched, primed. d, Naloxone in the orthosteric binding pocket across latent and primed states, with interacting residues highlighted in stick representation. e, Overlay of ligands in the orthosteric binding pocket. Naloxone across latent and primed structures, highlighting ligand pose; ‘shallow' versus ‘deep'. Light green, latent; orange, primed. f, Loperamide in the orthosteric binding pocket across engaged and primed state, with interacting residues highlighted in stick representation. The initial engagement between inactive MOR and GDP–Gαi1β1γ2, occurs through a unique interface. Unlike all previously determined structures, the G protein is rotated by 60°, relative to the receptor (Fig. 2a–c), and is engaged through all three intracellular loops (ICLs): ICL1 with Gβ1 (Fig. 2e), compared with R32 in the nucleotide-free state (Fig. 2g), ICL3 interacts extensively with the α5 helix (Fig. a, Structural models (side view) highlighting MOR–Gαi1 interface between the latent and engaged state. Inset highlights ordered C-terminus of Gαi1, engaging with TM5, TM6 and ICL3. Red, interacting residues within 4 Å between MOR and Gαi1β1γ2, highlighting the rearrangement of interactions through ICLs. b, Structural models (top view) highlighting MOR–Gαi1 interface between the latent and engaged state, with an outline of the receptor in black. Coloured, dashed boxes highlight ICL interfaces (green, ICL1; orange, ICL2; purple, ICL3). c, Cartoon representation of the MOR–Gαi1 in both states, with arrows highlighting the drastic rearrangement of the Gαi protein heterotrimer, relative to MOR. d, Latent, ICL1 interaction with Gβ1 residues. e, Latent, ICL2 interaction with αN. g, Engaged, ICL2 interactions with αN. i, MD simulations of latent and engaged states. Plots represent the average number of contacts (more than 4.5 Å) across all five MD trajectories of the respective state above 1 µs. k, Mutagenesis RG-BRET experiments of unique residues for each state (latent versus engaged) showing normalized efficacy, relative to the wild-type construct. Bar plots depict mean Δefficacy values from n = 3 biologically independent experiments. An initial step in G protein activation is breaking of the ‘hydrophobic core' of the Gα subunit, formed between the α5, β2–β3 and α1. This occurs during the extension of α5 into the receptor core and has been linked to GDP release13 (Fig. ICL3 shows contacts with several motifs that are part of, or connected to, this hydrophobic core (Fig. Comparing the latent state G protein to the receptor-free GDP–Gαi1β1γ2 (PDB: 1GP2), we observe important differences. In contrast to GDP–Gαi1β1γ2, the C-terminal α5 helix in the latent state, is extended and fully resolved (Fig. The four terminal residues protrude into the detergent micelle, and engage with the cytoplasmic ends of transmembrane (TM) helices TM5 and TM6. The terminal F354 is surrounded by TM6 residues R2756.28, R2786.31, R2826.35 (superscript values indicate Ballesteros–Weinstein numbering for GPCRs38), as well as T2816.34, which all have a critical role in coordinating the conserved DRY motif39, indicative of a regulatory role of this interaction (Fig. a, Overlay of Gαi subunit structures across receptor-free GDP–Gαi1β1γ2 (PDB ID 1GP2), latent, engaged, unlatched and primed states. b, Contact map highlighting interactions of Gαi residues with GDPβS across the five observed states. Contacts were determined by LigPlot+ (ref. 53) and the number of contacts is colour coded for the respective state and residue (light red 0 to dark blue more than 7 interactions per residue with GDP). Boxes (solid) in the first row represent backbone interactions. Boxes (solid) in the second row represent side-chain interactions. Boxes (dashed) in the third row represent hydrogen bonds (H bonds): only the total numbers of hydrogen bonds are shown in the centre. c, Overview of the hydrophobic core of Gαi, including α5 helix, α1 and α3, highlighting conformational changes across transition from latent to engaged state. d, Overview of the hydrophobic core of Gαi, including α5 helix, TCAT loop, α1 and α3, highlighting conformational changes across three conformational states (engaged, unlatched, primed), and most importantly translation and rotation of α5, resulting in retraction of the TCAT loop, and rearrangement of β6. e, αF transition and α1 movement across latent, engaged and primed states, required for GDP release. f, Successive change in relative angle between Gα and Gβ subunits. g, Comparison of SwI, P loop and SwIII across states. h, Nucleotide-binding pocket, highlighting rearrangements of E43 and R178 from a closed ‘seatbelt' conformation in GDP–Gαi1β1γ2 (PDB ID 1GP2) to latent state. i, Nucleotide-binding pocket, highlighting R178 protruding towards Mg2+ binding pocket in the engaged state, compared with GTP-bound Gαi1 (PDB ID 1GIL). j, α1, TCAT and P loop transitions towards the nucleotide-free state. The most intricate aspect of the latent interface is between ICL3 and key motifs of the Gαi heterotrimer (Fig. It is lined by residues R265, M266, S268, K2716.24, interacting with residues on the α5 helix, αN helix and D261 (Fig. Our MD simulations confirm that the receptor–G protein interface observed in this latent state is stable, showing a similar total number of interacting residues as the nucleotide-free-state interface, but fewer average numbers of contacts (Fig. To validate this binding interface, we carried out mutagenesis experiments of residues that are unique to the latent versus engaged/‘active-like' state (Fig. Most of these tested mutants showed reduced ligand efficacy, confirming that this state has a physiological role in signalling. Compared with GDP–Gαi1β1γ2, many backbone and side-chain interactions between the G protein and GDP remain intact in the latent state, including E43-T48 (P loop), K270, E272, L273 (β5/αG), C325-T327 (TCAT loop) of the RHD and D150-S151 (αD-αE loop), R176 (αF) and R178 (SwI) of the AHD (Extended Data Fig. However, we see a significant overall decrease of backbone (29 to 19) and side-chain (58 to 45) interactions. These are, in part, explained by a partial or complete loss of side-chain contacts of E43 (P loop), D150 (αD-αE loop) and N269 (NKxD) (Extended Data Fig. Conversely, residues R176 (αF) and R178 (SwI) gain side-chain interactions (Fig. Binding of receptor-free Gβγ to Gαi1, promotes hydrogen bond formation between E43 and R178, stabilizing a ‘seatbelt'-like conformation that occludes the nucleotide-binding pocket and may hinder GDP release40 (Fig. In the latent state, E43 loses this interaction, whereas R178 coordinates the phosphate groups of GDP, analogous to the GDP-bound monomeric Gαi41 (Fig. This transition was also predicted to be required for GTP binding and hydrolysis43. Furthermore, D150 loses most of its non-polar contacts with the ribose and the guanine groups (Fig. Confirming this hypothesis, although four MD trajectories closely resemble receptor-free GDP–Gαi1β1γ2, we observed transient AHD opening (Extended Data Fig. On transitioning to the ‘engaged' state, the G protein undergoes an extension of the α5 helix by 14 Å, rotation by 60° around its longest axis, translation towards MOR by 6 Å and sinks into the RHD by 3 Å (Fig. 3d,e), resulting in an interface identical to the nucleotide-free state. At the same time, MOR shows all classical features of activation. These include the opening of TM6, accompanied by canonical conformational rearrangements in microswitch motifs. These rearrangements are accompanied by the disruption of critical G protein–nucleotide contacts and displacement of the conserved F336 from its hydrophobic pocket formed by β2, β3 and α1, altering the network that stabilizes GDP binding (Fig. The engaged state remains in a closed AHD conformation, although showing a rotation relative to the RHD by 13° towards αN, and straightening of αA (Fig. Within the nucleotide-binding pocket, this state shows a partial retraction of the TCAT loop, whereas the C325 hydrogen bond with GDP remains intact (Fig. On its translation towards the receptor, F334 on α5 points towards the TCAT loop, and H322 on β6 remains pointing towards the Gαi C terminus (Fig. The α1/P loop moves towards TCAT by 1.2 Å, and continues on this trajectory throughout the next conformational states. Coinciding with the AHD rotation, we observe interesting conformational changes. The R176 rotamer on αF tilts up by 20° towards TCAT. D150 on αE loses the remaining hydrophobic contacts with GDP, whereas the only remaining AHD residue interacting with GDP is S151 (Fig. Furthermore, it results in a reconfiguration of the SwI motif. SwI is partly open and splays towards α1, resulting in a rotation of R178 by almost 90°. 3i) resulting in an ionic interaction with D200 from β3 and the β phosphate. This R178 conformation remains stable across all five MD trajectories (Extended Data Fig. Overall, we observe a loss of backbone (19 to 18) and side-chain (45 to 34) interactions with GDP (Fig. Given that the SwI coordination of GDP is critical for maintaining a closed AHD conformation, the rearrangements observed in this transient state probably initiates AHD opening. In agreement with this hypothesis, we observed that three MD trajectories resulted in a transient opening of the AHD and one trajectory showed complete opening (Extended Data Fig. 8c,h,k and Supplementary Video 1). The unlatched state adopts a dynamic AHD, analogous to previous nucleotide-free-state MOR–G protein structures. Despite a series of conformational changes around the nucleotide-binding pocket, we observed an unambiguous density for GDP (Fig. Residues F334 and H322 each undergo a rotamer flip, where F334 now points towards the Gαi C terminus, whereas H322 points towards TCAT. The transition also results in a loss of all residues from the AHD and SwI, including S151, R176 and R178. Furthermore, we observe a release of P loop backbone interactions of S44 and G45 with GDP (Fig. As a consequence of these losses in coordination (18 to 10 backbone, 34 to 22 side chain, 8 to 4 hydrogen bonds), GDP moves out of its pocket by roughly 2 Å (Extended Data Fig. α1 further moves towards α5, replacing hydrophobic contacts with polar interactions from the displaced β6–α5 loop (Fig. 3d,e,j and Extended Data Fig. The terminal two residues of α1, I56 and H57, involved in the hydrophobic core, disorder as a result of the AHD opening (Fig. The helix moves towards α5, identical to nucleotide-free, and Q52 supports the TCAT loop (Fig. One interesting observation is that the loop T128-E130 on Gβ1, which is poorly resolved in previous states40, now shows a well resolved density, with a retracted conformation relative to the latent and engaged states, in response to AHD opening (Extended Data Fig. In our MD simulations, the unlatched state shows the most heterogeneous mixture of AHD conformations, ranging from fully open to fully occluded. In the open AHD trajectories, we observed the guanine group release, while the nucleotide remains bound through its phosphate groups (Extended Data Fig. This overall heterogeneity could explain the disordered AHD density in this cryoEM reconstruction, whereas transiently occluded states may limit GDP release. Two key residues in this interaction are R134 and R137 of Gβ1, which coordinate the AHD. The connected β strand (βD2) is translated and rotated, relative to the ‘inactive' G protein states with a closed AHD (latent and engaged) (Extended Data Fig. This movement is required for the reorientation of R134. This loop conformation is also stabilized by scFv16, which could explain its ability to stabilize nucleotide-free complexes42 (Extended Data Fig. Within the nucleotide-binding pocket, the AHD opening results in a retraction of αF towards Gβ (Fig. 3a,e), resulting in increased dynamics of α1, which is further confirmed in our MD simulations (Extended Data Fig. AHD dynamics are markedly reduced compared with the ‘unlatched' state, indicating stable coordination by Gβ1. Confirming our hypothesis that the primed state shows the lowest affinity for GDP, our MD trajectories show the most heterogeneity for GDP (Extended Data Fig. In four trajectories, the guanine group is ejected, leaving the nucleotide bound only through phosphate interactions, and in one case, the nucleotide fully dissociates (Supplementary Video 1). The last observed rearrangement towards the nucleotide-free state is a collapse of the P loop and its contraction by 4 Å, hampering GDP binding (Extended Data Fig. Within the G protein heterotrimers, we see a clear trend comparing the relative angle between the Gαi and β subunit, which progressively becomes flatter from the receptor-free to nucleotide-free state, with an overall change of roughly 15° (Fig. This finding is consistent with the ‘lever arm' hypothesis for receptor-dependent G protein activation45,46. Comparing these cryoEM structures, we observed unambiguous densities for naloxone and loperamide in eight of our ten maps. A distinguishing feature between antagonists and agonists at opioid receptors is the relative elevation at which ligands bind within the orthosteric binding pocket47,48,49. Antagonists are consistently observed with a ‘shallow' pose compared with agonists. In agreement with this, naloxone binds in a shallow pose within the receptor, in our two ‘inactive' MOR conformations (Fig. 1e) compared with all subsequent states (Fig. By contrast, loperamide shows no significant movement in the orthosteric binding pocket, probably because our high-resolution loperamide-bound intermediate states all include ‘active-like' receptor conformations (Fig. Overall, although the observed changes are subtle, they show a correlation of the ligand binding pose with the coordination of GDP. Finally, as outlined above, our main protocol for determining the structure of GDP-bound complexes involves initially depleting the complex of nucleotides, followed by supplementing with GDP (‘re-bound GDP'). To rule out any potential artefacts in the resulting reconstructions, we purified GDP-bound Gαi heterotrimers in complex with naloxone-bound or loperamide-bound MOR (‘constant GDP'). In contrast to our ‘re-bound GDP' protocol, we kept 200–500 μM GDP in all purification buffers (Extended Data Fig. These complexes readily withstood purification and yielded three-dimensional (3D) reconstructions consistent with each of the above-described states. In agreement with our hypotheses, the naloxone-bound ‘latent' and the loperamide-bound ‘open AHD' conformation, provided by far the highest resolution reconstructions. In summary, these findings strongly support the validity of our original approach and proposed mechanisms (Fig. Here, we provide conformational snapshots along the initial steps of the MOR–G protein activation pathway, shedding light onto fundamental mechanisms of MOR pharmacology. We see a direct correlation between the ligand binding pose and conformational states of the G protein, before GDP release. In the ‘inactive' receptor conformation, naloxone adopts a shallow pose within the orthosteric binding pocket, whereas in the ‘active' conformation, both naloxone and loperamide engage significantly deeper within the pocket (Fig. This, in turn, suggests that ligands of increasing efficacy preferentially stabilize receptor conformations that facilitate nucleotide exchange47,48,49. We determined the structure of a latent MOR–G protein complex, capturing an early interaction state. Unlike previous structures, MOR interacts with the G protein through all three ICLs. Previous nuclear magnetic resonance studies suggested ICL1 dynamics increase on agonist binding, whereas full TM6 rearrangement requires the presence of a G protein50. Next, we determined an ‘engaged' state structure of a GDP-bound ternary complex, with a fully extended α5 helix and a closed AHD. Our MD simulations show that the AHD is, however, transiently open in most trajectories (Extended Data Fig. The existence of this state indicates that α5 extension, AHD opening and GDP release occur sequentially, rather than simultaneously. We also captured two states with open AHDs (‘unlatched' and ‘primed'), representing steps immediately before GDP release. Our MD results for the unlatched state, with a highly dynamic AHD, suggests that random collisions with the RHD may occlude GDP release. Finally, the primed state suggests a ‘trapdoor-like' mechanism, in which the retraction of the AHD leads to a concomitant retraction of the αF helix, resulting in the most dynamic GDP coordination observed across all states (Extended Data Fig. In one ‘primed' MD trajectory, this culminated in the ejection of GDP. Additionally, one ‘unlatched' MD trajectory gradually adopts a ‘primed' conformation, which also results in GDP release (Supplementary Video 1). As an ‘open AHD' conformation was observed in all of our GDP-containing specimens, it suggests that this state might be more prominent than previously considered. The existence of the four GDP-bound intermediate states also suggests that GDP binding, under our experimental conditions, is reversible and does not necessarily lead to the dissociation of the ternary complex. Our data indicate a model in which efficacy correlates with overcoming the rate-limiting step of G protein activation, GDP release, by shifting the equilibrium of receptor conformations (Fig. Antagonists and partial agonists stall MOR–Gi complexes in the latent state, which probably shows relatively low GDP release rates, as most coordinating interactions remain intact (Fig. The agonist loperamide, however, readily progresses into the engaged state, which is structurally configured for AHD domain opening and subsequent GDP release, while reducing the likelihood of GDP rebinding (Fig. As a result, more complexes stabilized by loperamide show lower GDP affinity, than those stabilized by naloxone. Nevertheless, we observed ‘open AHD' states (engaged, unlatched and primed), even in our ‘constant GDP' protocol, which suggests that these can indeed rebind GDP and are, under our experimental conditions, stable. Together, these findings demonstrate that GDP release is governed by the ligand through a modulation of the conformational landscape spanning inactive, latent, engaged, unlatched and primed states. Extending our framework, we propose that inverse agonists act by predominantly stabilizing the latent state, which could explain why, unlike agonists, MOR antagonist binding is unaffected by the presence of nucleotides27,51, analogous to our recent findings at KOR25. At present, we foresee a linear model for conformational rearrangements required for G protein activation, but it is possible that the observed states are connected through a variety of additional pathways. Although we cannot rule out that ligand binding might have any effect past the nucleotide-free state, we see no improvement in GTP binding affinities (Extended Data Fig. In fact, we see a decrease in GTP affinity for the superagonist loperamide, which further suggests that the primary mechanism of action for agonists is nucleotide release. Our data support a model in which efficacy is primarily driven by the transition from a signalling-silent ‘latent' state to an ‘engaged' state. This transition results in a hallmark ‘active-like' MOR conformation and an extended α5 helix, despite the AHD remaining closed. The engaged state is structurally primed for subsequent AHD opening, which is essential for GDP release. Stabilization of the latent state probably reduces nucleotide exchange rates, approximating basal activity seen in receptor-free GDP–Gαi1β1γ2, whereas progression through downstream states accelerates GDP release, thereby enhancing G protein activation. Our findings are complementary to single molecule Förster resonance energy transfer spectroscopy studies18,22 that showed that ligand efficacy correlates with changes in the equilibrium between ‘preactivated' G protein (GDP-bound) MOR conformations, and fully ‘active' conformations. Furthermore, our observed structures closely resemble recently described conformational states of MOR–Gαi complexes, post-GTP binding52. The study observes highly similar conformational changes, including global rearrangements such as comparable MOR–G protein interfaces, accompanied by a gradual closure of the AHD. The only significant difference is that we do only observe subtle changes within SwII and no dissociation between subunits (Fig. However, it is not surprising that GDP release closely mimics conformational changes and states, required for GTP binding. In summary, the present study sheds light on MOR ligand efficacy by revealing that ligands can stabilize distinct proportions of conformations, resulting in varying degrees of nucleotide exchange. It underscores the critical value of high-resolution structures of nucleotide-bound GPCR–G protein ternary complexes, offering experimental insight into intermediate conformations that underlie G protein functional selectivity at a level of structural detail that is not achievable by any other method. Given the observation of functionally selective intermediate states giving rise to specific pharmacological outcomes, this model may prove more broadly applicable across the GPCR superfamily. No statistical methods were used to predetermine sample size. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment. Alvimopan, naloxone, mitragynine pseudoindoxyl, DAMGO and loperamide were purchased from Cayman Chemicals and MedChemExpress. Detergents were purchased from Anatrace. The radioligand [3H]naltrexone (specific activity 48.19 Ci per mmol) was generously provided by the National Institute on Drug Abuse Drug Supply Program. All other general laboratory reagents and chemicals were purchased from Millipore Sigma, unless otherwise stated. For recombinant protein expression, full-length human MOR was subcloned into a modified pFastBac1 vector with an N-terminal Flag, 10xHis-tag and thermostabilized b562RIL (ref. 54) along with a C-terminal TwinStrep tag55 and an extra 10xHis-tag. Both N-terminal and C-terminal tags are flanked by HRV-3C protease cleavage sites to allow tag removal. For the G-protein-free ‘inactive' MOR structure, two point mutations M264L and K2696.24R in ICL3 were incorporated, to allow Nb6 binding56. For all BRET assays, transient transfections of wild-type full-length human MOR were performed using the pcDNA 3.1(−) vector in human embryonic kidney 293F (HEK293F) mammalian cells. These receptor constructs were expressed under the cytomegalovirus promoter and included an N-terminal haemagglutinin signal peptide followed by a Flag tag. For TRUPATH24 and GloSensor assays, the MOR construct was untagged and contained no fusion elements. In TRUPATH assays24, a tricistronic vector encoding Gαi1, Gβ1 and Gγ2 was used, in which GFP2 was fused to the N terminus of Gγ2 and Rluc8 was inserted after residue G90 of the Gαi1 subunit23. For RG-BRET assays, a Gαi1β1γ2 construct was used with GFP2 fused to the N terminus of Gγ2. The pOZITX-S1 plasmid, used as a control in these assays, was obtained from J. Javitch (Addgene plasmid no. HEK293F cells were cultured in FreeStyle293 Expression Medium (Gibco) at 37 °C with 5% CO2 and shaking at 110 rpm. For transfections, cells were seeded 1 day before and transfected at a cell density of 1.0 × 106 cells per ml using polyethyleneimine (PEI) with a DNA:PEI ratio of 1:2. For RG-BRET assays, MOR–Rluc8 and Gαi/Gβ1/Gγ2–GFP2 constructs were transfected at a 1:2 ratio. TRUPATH BRET assays used MOR and Gαi1–Rluc8/Gβ1/Gγ2–GFP2 transfected at a 1:1 ratio. When pOZITX-S1 was used, Gαi/Gβ1/Gγ2–GFP2 and pOZITX-S1 plasmids were transfected at a 1:1 ratio. For MOR–Gαi-mediated cAMP inhibition assays, cells were cotransfected with wild-type human MOR, along with a split-luciferase-based cAMP biosensor (GloSensor, Promega) at a 1:1 ratio. For experiments with intact cells, cells were washed with assay buffer (20 mM HEPES pH 7.5, 1× HBSS) and transferred to white-bottom 96-well plates. Cells were permeabilized using 10 μg ml−1 high-purity digitonin (GoldBio) and treated with apyrase (2 U ml−1, Sigma-Aldrich), optionally with increasing GDPβS concentrations, then transferred to 96-well plates containing the respective ligand in ligand buffer (20 mM HEPES pH 7.5, 1× HBSS, 0.1% BSA) or KPS for permeabilized cells. Steady state BRET measurements were performed using a PHERAstar FSX multimode plate reader (BMG Biotech). Coelenterazine 400a (5 μM; Nanolight) dissolved in NanoFuel Solvent was added to all wells immediately before measuring with Rluc8. Raw netBRET signals were calculated as the emission intensity at 515 nm divided by the emission intensity at 410 nm, with no normalization or further data treatment applied to reported results. Each assay was conducted in at least three biologically independent (separate transfection) experiments, on different days, with all data points measured in triplicate. EC50 and half-maximal inhibitory concentration (IC50) values were calculated in GraphPad Prism 10 (v.10.6.0) by fitting the raw netBRET results from each experiment, followed by non-linear regression using the equations: y = Bottom + (Top − Bottom)/(\(1+{10}^{({\mbox{logEC}}_{50}-x)}\)) and y = Bottom + (Top − Bottom)/(\(1+{10}^{(x-{\mbox{logIC}}_{50})}\)). At 48 h post-transfection, cells were washed with assay buffer (20 mM HEPES, 1× HBSS, pH 7.4) and plated into 96-well white cell culture plates at a density of 200,000 cells in 90 μl per well. Ligands were prepared as 3× solutions in assay buffer (20 mM HEPES, 1× HBSS, pH 7.4). Media was aspirated and cells were incubated with 30 μl per well of drug buffer (20 mM HEPES, 1× HBSS, pH 7.4), followed by addition of 30 μl of 3× drug solutions for 15 min in the dark at room temperature. Cells then received 30 μl of luciferin (Goldbio, 4 mM final concentration) supplemented with isoproterenol (100 μM final concentration) to stimulate production of endogenous cAMP through β2 adrenergic Gs activation, and were incubated in the dark at room temperature. After 15 min, luminescence intensity was quantified using a PHERAstar FSX multimode plate reader (BMG Biotech). Data were plotted as a function of ligand or nucleotide concentration and analysed using log (ligand or nucleotide) versus response in GraphPad Prism (v.10.6.0). Radioligand binding assays were conducted using membrane fractions from HEK293F cells transiently expressing human wild-type MOR. For membrane preparation, cell pellets expressing human wild-type MOR and Gαi1-Gβ1-Gγ2 at a 1:1 ratio were gathered and resuspended in hypotonic buffer (10 mM HEPES pH 7.5, 10 mM MgCl2, 20 mM KCl supplemented with 2 mM AEBSF, 14 μM E-64, 1 μM leupeptin and 0.3 μM aprotinin). Cells were dounce homogenized and centrifuged at 175,000g in two rounds to obtain membrane fractions. Membrane protein concentration was determined to be roughly 4 mg ml−1 using a Bradford assay (Pierce). Aliquots were flash frozen in liquid nitrogen and stored at −80 °C until use. Competition binding assays were setup in 96-well plates containing membrane fractions diluted to 0.15 mg ml−1, along with [3H]naltrexone at 2 nM and a ‘cold' naltrexone dose (100 µM–10 pM), all prepared in binding buffer (10 mM HEPES, 10 mM MgCl2, 20 mM KCl, 0.1% BSA and 100 µM Bacitracin). Competition reactions were incubated for 1 h and terminated by vacuum filtration onto cold 0.3% PEI-soaked GF/A filters, followed by 3 rounds of washing with cold 50 mM HEPES (pH 7.50). Counts were read using a Microbeta2 plate reader (PerkinElmer) for 1 min per well. Results were analysed in GraphPad Prism v.10.6.0, and the inhibitor constant (Ki) was determined using the ‘One site - Fit Ki' model equation: logEC50 = log(\({10}^{\mbox{log}{K}_{i}\times (1\,+\,\mbox{RadioligandNM}/\mbox{HotKdNM})}\)), y = Bottom + (Top − Bottom)/(\(1+{10}^{(x-{\mbox{logEC}}_{50})}\)). Recombinant expression of the pFastBac1-MOR constructs was carried out in Spodoptera frugiperda (Sf9) insect cells using the Bac-to-Bac expression system (Gibco). Recombinant baculovirus was added at a multiplicity of infection of 5 to Sf9 cells cultured at a density of 3–3.5 × 106 cells per ml in ESF921 medium (Expression Systems) supplemented with 1% (v/v) production boost additive. To enhance receptor surface expression and improve protein yield, 10 µM naloxone was added during protein expression. Cells were incubated for 48 h at 27 °C with shaking at 100 rpm, then collected by centrifugation, washed with PBS and stored at −80 °C until further use. For membrane preparation, frozen cell pellets were thawed on ice and resuspended in hypotonic buffer (10 mM HEPES pH 7.5, 10 mM MgCl2, 20 mM KCl, with protease inhibitors: 2 mM AEBSF, 14 μM E-64, 1 μM leupeptin and 0.3 μM aprotinin). The suspension was homogenized using a glass dounce homogenizer and further treated with hypertonic buffer (hypotonic buffer + 1 M NaCl). This washing and centrifugation process was repeated 2 more times, with 10 μM naloxone included in the final 2 washes. Final membrane pellets were homogenized in purification buffer (40 mM HEPES pH 7.5, 150 mM NaCl and 10 μM naloxone) supplemented with 20% glycerol and stored at −80 °C. For protein solubilization, membranes were thawed in the presence of 2 mg ml−1 iodoacetamide and diluted in 2× solubilization buffer (40 mM HEPES pH 7.5, 150 mM NaCl, 1.0% w/v lauryl maltose neopentyl glycol (LMNG) and 0.1% w/v cholesteryl hemisuccinate (CHS)) and incubated for 8 h at 4 °C. The supernatant was incubated with M2 anti-Flag affinity resin (Sigma) for 2 h at 4 °C. The resin was washed with 20 column volumes of wash buffer (40 mM HEPES pH 7.5, 150 mM NaCl, 0.001% LMNG, 0.0001% CHS and 10 μM naloxone). Protein was eluted using the same buffer supplemented with 200 μg ml−1 DYKDDDDK peptide (GenScript). The eluate was concentrated and used for complex formation with Nb6M or heterotrimeric G proteins. Wk6 Escherichia coli (American Type Culture Collection) were transformed by heat shock with the Nb6 plasmid56,57 and grown overnight in starter cultures containing 100 μg ml−1 ampicillin. These cultures were used to inoculate 2 l of Terrific Broth supplemented with 100 μg ml−1 ampicillin and grown at 37 °C with shaking until log phase (optical density at 600 nm (OD600) between 0.5 and 1) was reached. Protein expression was induced with 1 mM isopropyl-β-d-thiogalactoside (IPTG), and cultures were incubated overnight at 28 °C. Cell pellets were gathered by centrifugation, washed with PBS and flash frozen in liquid nitrogen. For purification, Nb6-containing pellets were thawed and resuspended in TES buffer (50 mM Tris pH 8.0, 0.5 mM EDTA, 20% w/v sucrose) at 15 ml per litre of culture, supplemented with protease inhibitor cocktail and incubated on an orbital shaker at 4 °C for 1 h. An extra 30 ml per litre of 1:4 diluted TES was added, followed by another 45 min of incubation at 4 °C. Cell debris was removed by ultracentrifugation at 100,000g for 30 min and the clarified supernatant was passed through a 0.45-µm filter. The filtered solution was supplemented with 20 mM imidazole and loaded onto a Ni-NTA affinity column prechilled to 4 °C. The column was washed with 10 column volumes of wash buffer (250 mM NaCl, 50 mM Tris pH 7.5, 10 mM imidazole) and Nb6M was eluted with buffer containing 250 mM imidazole. Fractions were pooled, concentrated, supplemented with 10% glycerol and snap-frozen in liquid nitrogen. Chemically competent E. coli C43 cells were transformed by heat shock with the NabFab plasmid58. A starter culture in Terrific Broth medium was inoculated and grown overnight at 37 °C with shaking. These cultures were used to inoculate 2 l of Terrific Broth supplemented with 100 μg ml−1 ampicillin and grown at 37 °C with shaking until log phase (OD600 between 0.5 and 1) was reached. Protein expression was induced with 1 mM IPTG, and cultures were incubated overnight at 28 °C. Cell pellets were collected by centrifugation, washed with PBS and flash frozen in liquid nitrogen. For purification, thawed cell pellets were resuspended in lysis buffer (20 mM sodium phosphate pH 7.4, 150 mM NaCl, DNaseI, and protease inhibitor cocktail) and lysed by sonication. The lysate was heat-treated at 63–65 °C for 30 min and clarified by centrifugation at 20,000g for 30 min. The supernatant was loaded onto a Protein G column pre-equilibrated with 20 mM sodium phosphate, 500 mM NaCl, pH 7.4. NabFab was eluted with 0.1 M acetic acid and directly applied to a Resource S cation exchange column equilibrated in buffer A (50 mM sodium acetate, pH 5.0). The column was washed with five column volumes of buffer A, and NabFab was eluted using a 0–100% gradient of buffer B (50 mM sodium acetate, 2 M NaCl, pH 5.0). Eluted protein was dialysed overnight into 150 mM NaCl, 20 mM HEPES, pH 7.5 and concentrated to 3 mg ml−1 for complex formation. Wild-type Gαi1 was co-expressed with Gβ1 and γ2 subunits in Sf9 insect cells using the Bac-to-Bac system in ESF921 medium (Expression Systems). Cells were seeded at a density of 2 × 106 cells per ml and infected with P1 baculovirus at a multiplicity of infection ratio of 10:5 (Gαi1:Gβγ). After 48 h of incubation at 27 °C with shaking at 100 rpm, cells were gathered by centrifugation, washed with ice-cold PBS and stored at −80 °C until use. Gαi1βγ heterotrimers were purified using established protocols59,60. Frozen pellets were thawed and homogenized in hypotonic buffer containing 10 µM GDP and 5 mM β-mercaptoethanol, followed by centrifugation at 100,000g. The supernatant was loaded onto Ni-NTA resin and eluted with buffer containing 300 mM imidazole. The protein was concentrated using a 50-kDa cutoff centrifugal concentrator (Amicon) and further purified through anion exchange chromatography using a 1 ml HiTrap Q FF column (Cytiva). The scFv16 single-chain antibody fragment was expressed in Sf9 cells at a density of 2 × 106 cells per ml with baculovirus infection at a multiplicity of infection of 5. After 72 h, culture media was collected by centrifugation at 1,000g for 15 min at 4 °C. The pH of the media was adjusted to 7.5 using 1 M Tris, followed by the addition of 1 mM NiCl2 and 5 mM CaCl2 to precipitate chelators. The precipitate was removed by ultracentrifugation at 100,000g. The clarified supernatant was incubated with Ni-NTA resin for 3 h, washed with buffer containing 20 mM HEPES pH 7.5, 100 mM NaCl, 15 mM imidazole, 0.00075% LMNG and 0.000075% CHS, and eluted with the same buffer supplemented with 300 mM imidazole. The purified scFv16 was concentrated to roughly 10 mg ml−1 and stored at −80 °C. Purified MOR–naloxone was incubated with Gαiβγ heterotrimer at a 1:1.2 molar ratio for 1 h at room temperature. Apyrase was then added to catalyse hydrolysis of GDP (for ‘re-bound GDP' and nucleotide-free specimen). For the nucleotide-free MOR structure, scFv16 was added at 1.2 molar excess. For the ‘constant GDP' samples, the specimen was not treated with apyrase, and instead supplemented with 200 μM (naloxone) or 500 μM (loperamide) GDP in all subsequent purification buffers. The mixture was incubated at room temperature for 90 min. The complex was subsequently purified with an extra round of Flag resin, to separate from excess G protein by size-exclusion chromatography using a Superdex 200 10/300 column equilibrated with buffer containing 40 mM HEPES pH 7.5, 100 mM NaCl, 10 μM naloxone, 0.00075% LMNG and 0.000075% CHS. Peak fractions were pooled, concentrated and immediately used for cryoEM studies. For the inactive MOR structure, purified MOR was incubated with Nb6M and NabFab in molar excess of 2:1, incubated on ice overnight and finally purified by size-exclusion chromatography, with identical conditions described above. MOR–Nb6–naloxone, MOR–Gi–naloxone, complexes in LMNG or CHS micelles were used immediately after concentrating the monomeric size-exclusion chromatography peak to 1–2 mg ml−1 using a 50-kDa cutoff Amicon concentrator. For cryoEM grid preparation, 3 µl of purified protein complex was applied to freshly glow-discharged UltrAuFoil 1.2/1.3 300 mesh grids (Quantifoil), blotted for 2.5–4 s at 95% relative humidity and 4 °C, then vitrified in liquid ethane using a Vitrobot Mark IV (Thermo Fisher). For the ‘re-bound GDP' samples, the purified specimen was incubated with 100 µM GDPβS for 1 h, before grid freezing. For the ‘constant GDP' samples, the purified specimen was further supplemented with 500 μM GDP (naloxone) and 1 mM GDP (loperamide) and incubated before grid freezing. Micrographs were collected using an aberration-free image shift data collection scheme (four images per hole) with EPU data acquisition software (version 2.0) on a Titan Krios microscope (Thermo Fisher) operating at 300 keV. The microscope was equipped with a K3 direct-electron detector and post-BioQuantum GIF energy filter using a 20 eV slit size (Gatan). Images were collected with a total exposure time of 1.8 s, total dose of 55–60 e−/Å2 and defocus ranging from −1 µm to −3 µm. Motion correction of micrograph videos was carried out using MotionCor2 (ref. All subsequent data processing steps were performed using the software package cryoSPARC62 (v.4.7.1; Structura Biotechnology) (Fig. 3–6 and Supplementary Tables 1–3), using established protocols. For naloxone-bound complexes, a total of 30,264 (MOR–Nb6), 45,722 (‘re-bound' MOR–Gi–GDP), 17,742 (‘constant' MOR–Gi–GDP) and 32,514 (MOR–Gi–nucleotide-free) micrographs were preprocessed through patch contrast transfer function estimation (default parameters), and micrographs with average contrast transfer function resolution estimates above 4 Å were discarded. For loperamide-bound complexes, 22,868 (re-bound MOR–Gi–GDP) and 22,146 (constant MOR–Gi–GDP) micrographs were collected and preprocessed similarly. An initial round of processing was carried out on a subset of the data (roughly 5,000 micrographs), using the reference-free Blob picker (140 Å particle diameter) routine, followed by two-dimensional (2D) classification until 2D classes with clearly distinguishable transmembrane domain (TMD) densities were obtained. Ab initio reconstruction of these 2D classes was performed to obtain a 3D model used for template-based picking and 3D reconstructions of all three full datasets (Extended Data Figs. Particle picking on the entire dataset (20 Å reference low pass) was followed by extraction with a box size of 128 pixels (bin 4; 512 pixels uncropped box size). Three to four rounds of 2D classification were performed to obtain homogenous 2D classes with distinguishable TMD densities (Extended Data Figs. 3–5), after which particles were re-extracted (bin 2, box size 256). Further 2D classification was used to separate complexes with ‘open' or ‘closed' AHD conformations, followed by multiclass ab initio reconstructions of each particle subset. The resulting 3D volumes were used to perform several rounds of heterogeneous refinement, on the re-extracted particle stack resulting in segregation of particles into five classes. A combination of ab initio reconstruction, non-uniform refinement and 2D classification was performed on particle subsets corresponding to MOR–Gi complexes. Finally, local refinements with a manually created mask around the TMD, masking the micelle and the heterotrimeric G protein yielded the final reconstructions of maps ranging between 2.8 Å and 3.8 Å. The final ‘focused' maps were merged in Chimera for subsequent model building and refinement. Refinements were based on PDB ID 1GP2 (Gαi1 protein heterotrimer, GDP-bound)40, PDB 7UL4 (MOR–Nb6-alvimopan)56 and PDB 8EF6 (MOR–Gi-DN-morphine)47, for which residues in the sequence were reverted to the respective wild-type Gαi. The ordered AHD conformation for the ‘primed' state was modelled after rhodopsin PDB ID 6CMO (ref. The models were fitted into the density map in UCSF Chimera (version 1.17.3)64 and manually adjusted to fit the density map in COOT (version 0.9.2)65. Subsequently, the generated model was automatically refined in phenix.real_space_refine66,67 (version 1.21.2) and manually adjusted in COOT (version 0.9.2)65, for several iterations. The final geometry validation statistics including clashscore and Ramachandran analysis were calculated by MolProbity68. The final refinement statistics were generated using the ‘comprehensive validation (cryoEM)' function in phenix (version 1.21.2)67. For our MD simulation, we deployed GROMACS v.2024.5 (ref. 69) using CHARMM36 all-atom force-field parameters and topologies70,71. Ligand force-field parameters and topologies for naloxone and loperamide were generated using the Ligand Reader & Modeler tool available through the CHARMM-GUI webserver. The starting conformations of these complexes were obtained from the corresponding cryoEM structures determined in this study. MOR, Gi and MOR–Gαi heterotrimers (GDP) in the respective state were embedded in a lipid bilayer consisting of dipalmitoylphosphatidylcholine (DPPC), dioleoylphosphatidylcholine (DOPC) and cholesterol with ratio DPPC:DOPC:CHL 0.55:0.15:0.30, referencing simulations performed on GPCRs72. The missing Gαi1 AHD in the unlatched structure was manually modelled using the latent and primed structures as a template, placing the AHD roughly at its halfway point, with structure regularization and torsion angle optimization carried out using phenix. The N-terminal Gly of Gα was myristoylated, while Cys3 of the same chain was palmitoylated. In Gγ, the C terminus was capped using N-methylamide, as well as geranylgeranylation of Cys68. The palmitoyl group was also added to Cys172 of MOR. Initial membrane coordinates were assigned by the Positioning of Proteins in Membranes server through the CHARMM-GUI interface73. Systems were solvated in TIP3 water molecules, and Na+, Cl−, Mg2+ ions were placed in the systems to obtain 100 mM NaCl and 10 mM MgCl2. The box size was determined on the basis of the protein extent of each model (xy from 80 Å for the inactive MOR–naloxone to 120 Å for MOR–Gαi1 complex, z 160 Å). All systems underwent initial energy minimization for 50,000 steps using the steepest descent algorithm and a 100 kJ mol−1 nm−1 threshold, followed by equilibration simulations for a total time of 10 ns. The equilibration was performed in the NVT ensemble followed by the NPT ensemble for 6 steps (time step of 1–2 fs) with V-rescale thermostat at 303.15 K and Parrinello–Rahman barostat at 1 atm. Five separate replicates of production runs were subsequently performed for 1 µs each under NPT ensemble with Parrinello–Rahman barostat at 1 atm and V-rescale thermostat at 303.15 K, with random assignment of velocities. Simulations were executed on the graphical processing unit clusters at the Center for Advanced Computing of the University of Southern California. MD trajectory analysis was carried out using the GROMACS analysis toolkit, the MDTraj software package74 and MDCiao75. For BRET2 nucleotide competition results, log transformed EC50 values were analysed using one-way analysis of variance within each drug group. For comparisons between only two groups, unpaired two-tailed Student's t-tests were performed to compare each condition. A significance threshold of α = 0.05 was applied for both one-way analysis of variance and Student's t-test analyses. EC50 values are reported as the mean ± s.d. 77), ChemDraw Professional v.22.2.0 (PerkinElmer), GraphPad Prism v.10.6.0 and Adobe Illustrator 2021 v.29.7.1 were used to create all illustrations and figures. All reported root mean-square deviation values were calculated using the align command in PyMol, with either global alignment of the receptor or Gαi RHD. All netBRET data are plotted as ratios of the raw emission data, without baseline correction, normalization or any other data treatment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The final cryoEM maps for inactive MOR–Nb6-naloxone, latent MOR–Gi–GDP-naloxone-re-bound, unlatched MOR–Gi–GDP-naloxone-re-bound, primed MOR–Gi–GDP-naloxone-re-bound and nucleotide-free MOR–Gi-naloxone, engaged MOR–Gi–GDP-loperamide-re-bound, unlatched MOR–Gi–GDP-loperamide-re-bound and primed MOR–Gi–GDP-loperamide-re-bound have been deposited in the Electron Microscopy Data Bank under accession codes EMD-72001, EMD-72003, EMD-72004, EMD-72005, EMD-72002, EMD-72006, EMD-72007 and EMD-72008. Corresponding atomic coordinates have been deposited in the PDB under accession codes 9PXU, 9PXW, 9PXX, 9PXY, 9PXV, 9PY2, 9PY3 and 9PY4. Maps for engaged MOR–G–GDP-naloxone-re-bound, latent MOR–Gi–GDP-loperamide-re-bound, latent MOR–Gi–GDP-naloxone-constant, engaged MOR–Gi–GDP-naloxone-constant, open AHD MOR–Gi–GDP-naloxone-constant, latent MOR–Gi–GDP-loperamide-constant, engaged MOR–Gi–GDP-loperamide-constant, open AHD MOR–Gi–GDP-loperamide-constant under accession codes EMD-72013, EMD-72015, EMD-72017, EMD-72019, EMD-72021, EMD-72023, EMD-72025 and EMD-72026. Source data are provided with this paper. Ferguson, K. M., Higashijima, T., Smigel, M. D. & Gilman, A. G. The influence of bound GDP on the kinetics of guanine nucleotide binding to G proteins. & Remmers, A. E. Relationship between rate and extent of G protein activation: comparison between full and partial opioid agonists. Mu opioids and their receptors: evolution of a concept. & Walsh, T. D. Clinical pharmacokinetics of morphine. National Institute on Drug Abuse. Drug overdose death rates: facts and figures. National Institutes of Health https://nida.nih.gov/research-topics/trends-statistics/overdose-death-rates (2023). & DuPont, R. L. Polysubstance use in the U.S. opioid crisis. Sawynok, J., Pinsky, C. & LaBella, F. S. On the specificity of naloxone as an opiate antagonist. Distinct profiles of functional discrimination among G proteins determine the actions of G protein-coupled receptors. & Bidlack, J. M. Unique pharmacological properties of the kappa opioid receptor signaling through Gαz as shown with bioluminescence resonance energy transfer. Wang, Z., Bilsky, E. J., Porreca, F. & Sadée, W. Accelerated communication: constitutive μ opioid receptor activation as a regulatory mechanism underlying narcotic tolerance and dependence. Gillis, A. et al. Low intrinsic efficacy for G protein activation can explain the improved side effect profiles of new opioid agonists. Oldham, W. M. & Hamm, H. E. Heterotrimeric G protein activation by G-protein-coupled receptors. Kenakin, T. A scale of agonism and allosteric modulation for assessment of selectivity, bias, and receptor mutation. Breivogel, C. S., Selley, D. E. & Childers, S. R. Cannabinoid receptor agonist efficacy for stimulating [35S]GTPgammaS binding to rat cerebellar membranes correlates with agonist-induced decreases in GDP affinity. Selley, D. E., Sim, L. J., Xiao, R., Liu, Q. & Childers, S. R. mu-Opioid receptor-stimulated guanosine-5'-O-(gamma-thio)-triphosphate binding in rat thalamus and cultured cell lines: signal transduction mechanisms underlying agonist efficacy. Roberts, D. J., Lin, H. & Strange, P. G. Mechanisms of agonist action at D2 dopamine receptors. Zhao, J. et al. Ligand efficacy modulates conformational dynamics of the µ-opioid receptor. Zhang, Q., Okamura, M., Guo, Z.-D., Niwa, S. & Haga, T. Effects of partial agonists and Mg2+ ions on the interaction of M2 muscarinic acetylcholine receptor and G protein Galpha i1 subunit in the M2-Galpha i1 fusion protein. Seifert, R., Gether, U., Wenzel-Seifert, K. & Kobilka, B. K. Effects of guanine, inosine, and xanthine nucleotides on beta(2)-adrenergic receptor/G(s) interactions: evidence for multiple receptor conformations. Gregorio, G. G. et al. Single-molecule analysis of ligand efficacy in β2AR-G-protein activation. Deutsch, J. et al. Ligand-dependent G protein dynamics underlying opioid signaling efficacy. Galés, C. et al. Probing the activation-promoted structural rearrangements in preassembled receptor-G protein complexes. Olsen, R. H. J. et al. TRUPATH, an open-source biosensor platform for interrogating the GPCR transducerome. Tyson, A. S. et al. Molecular mechanisms of inverse agonism via κ-opioid receptor–G protein complexes. Direct interrogation of context-dependent GPCR activity with a universal biosensor platform. Childers, S. R. & Snyder, S. H. Guanine nucleotides differentiate agonist and antagonist interactions with opiate receptors. Traut, T. W. Physiological concentrations of purines and pyrimidines. Batebi, H. et al. Mechanistic insights into G-protein coupling with an agonist-bound G-protein-coupled receptor. The receptor-bound ‘empty pocket' state of the heterotrimeric G-protein alpha-subunit is conformationally dynamic. DeVree, B. T. et al. Allosteric coupling from G protein to the agonist-binding pocket in GPCRs. Traynor, J. R. & Nahorski, S. R. Modulation by mu-opioid agonists of guanosine-5′-O-(3-[35S]thio)triphosphate binding to membranes from human neuroblastoma SH-SY5Y cells. Chung, K. Y. et al. Conformational changes in the G protein Gs induced by the β2 adrenergic receptor. Structural mechanism underlying primary and secondary coupling between GPCRs and the Gi/o family. Jang, W. et al. An inactive receptor-G protein complex maintains the dynamic range of agonist-induced signaling. Okashah, N. et al. Agonist-induced formation of unproductive receptor-G12 complexes. Structural flexibility of the G alpha s alpha-helical domain in the beta2-adrenoceptor Gs complex. The local environment at the cytoplasmic end of TM6 of the mu opioid receptor differs from those of rhodopsin and monoamine receptors: introduction of an ionic lock between the cytoplasmic ends of helices 3 and 6 by a L6.30(275)E mutation inactivates the mu opioid receptor and reduces the constitutive activity of its T6.34(279)K mutant. Tertiary and quaternary structural changes in Gi alpha 1 induced by GTP hydrolysis. Maeda, S. et al. Development of an antibody fragment that stabilizes GPCR/G-protein complexes. Mechanism of the intrinsic arginine finger in heterotrimeric G proteins. & Bourne, H. R. G-protein diseases furnish a model for the turn-on switch. Rondard, P. et al. Mutant G protein alpha subunit activated by Gbeta gamma: a model for receptor activation?. Zhuang, Y. et al. Molecular recognition of morphine and fentanyl by the human μ-opioid receptor. Kolinski, M. & Filipek, S. Molecular dynamics of μ opioid receptor complexes with agonists and antagonists. Sutcliffe, K. J., Henderson, G., Kelly, E. & Sessions, R. B. Drug binding poses relate structure with efficacy in the μ opioid receptor. Sounier, R. et al. Propagation of conformational changes during μ-opioid receptor activation. Childers, S. R. & Snyder, S. H. Differential regulation by guanine nucleotides or opiate agonist and antagonist receptor interactions. Robertson, M. J. et al. Non-equilibrium snapshots of ligand efficacy at the μ-opioid receptor. & Thornton, J. M. LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions. Chun, E. et al. Fusion partner toolchest for the stabilization and crystallization of G protein-coupled receptors. Schmidt, T. G. M. et al. Development of the Twin-Strep-tag® and its application for purification of recombinant proteins from cell culture supernatants. Structure determination of inactive-state GPCRs with a universal nanobody. Che, T. et al. Nanobody-enabled monitoring of kappa opioid receptor states. Bloch, J. S. et al. Development of a universal nanobody-binding Fab module for fiducial-assisted cryo-EM studies of membrane proteins. Dominant negative G proteins enhance formation and purification of agonist-GPCR-G protein complexes for structure determination. Crystal structure of the β2 adrenergic receptor-Gs protein complex. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Kang, Y. et al. Cryo-EM structure of human rhodopsin bound to an inhibitory G protein. Goddard, T. D., Huang, C. C. & Ferrin, T. E. Visualizing density maps with UCSF Chimera. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Chen, V. B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. Jo, S., Kim, T., Iyer, V. G. & Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. Leonard, A. N. & Lyman, E. Activation of G-protein-coupled receptors is thermodynamically linked to lipid solvation. Lomize, M. A., Pogozheva, I. D., Joo, H., Mosberg, H. I. & Lomize, A. L. OPM database and PPM web server: resources for positioning of proteins in membranes. McGibbon, R. T. et al. MDTraj: a modern open library for the analysis of molecular dynamics trajectories. Pérez-Hernández, G. & Hildebrand, P. W. Mdciao: accessible analysis and visualization of molecular dynamics simulation data. The PyMOL Molecular Graphics System, V.3.0 (Schrödinger). Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. We thank H. Khant for assistance with cryo-electron microscopy data collection. This research was supported by US National Institutes of Health grant no. We acknowledge the Center of Excellence for Nano Imaging (CNI) at the University of Southern California for cryoEM microscope time. We acknowledge the Center for Advanced Research Computing for support with high-performance computing resources. We acknowledge the National Institute on Drug Abuse Drug Supply Program for generously providing radiolabelled [3H]naltrexone for our binding experiments. The Bridge Institute, Michelson Center for Convergent Biosciences, University of Southern California, Los Angeles, CA, USA Saif Khan, Aaliyah S. Tyson, Mohsen Ranjbar, Zixin Zhang, Jaskaran Singh, Gye Won Han & Cornelius Gati Molecular and Computational Biology, Department of Biological Sciences, University of Southern California, Los Angeles, CA, USA Department of Chemistry, University of Southern California, Los Angeles, CA, USA Aaliyah S. Tyson, Mohsen Ranjbar, Jaskaran Singh & Cornelius Gati Keck School of Medicine, University of Southern California, Los Angeles, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived, designed and supervised the study. performed recombinant protein expression in mammalian and Sf9 cell cultures. performed sample preparation for the cryoEM studies. performed cryoEM data analysis and image processing. performed model building, refinement and validation. performed, analysed and interpreted BRET, One-GO, GloSensor and radioligand binding assays. performed [3H]naltrexone radioligand binding experiments. wrote the paper, with input from co-authors. The authors declare no competing interests. Nature thanks Henrik Dohlman and Cheng Zhang for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, b, c ONE-GO, RG-BRET and TruPATH assay confirm relative signaling efficacy of alvimopan, naloxone, mitragynineP, DAMGO and loperamide. Data is a global fit of individual replicates. Data are plotted as raw netBRET ratios. d, cAMP inhibition assay (GloSensor) confirms relative signaling efficacy of alvimopan, naloxone, mitragynineP, DAMGO and loperamide. Data is a global fit of individual replicates. e, Homologous competition radioligand binding assay of [3H]naltrexone. Data points are plotted as individual replicate values of raw netBRET ratios (a-c), luminescence (d), or specific binding (total binding minus nonspecific binding) (e), overlaid with nonlinear regression global fits. All plots are representative of three biologically independent experiments, with each data point carried out with three technical replicates. Nucleotide dose–response experiments are performed apo, and in the presence of the respective ligand alvimopan, naloxone, mitragynineP, DAMGO, and loperamide. a, b, Nucleotide experiments comparing the effect of GDPβS and GTPγS on relative BRET signals, observing a concentration-dependent decrease in signal. Data presented are representative of n = 3 biologically independent experiments. Traces show individual raw data points, overlaid with nonlinear regression global fits. c, control experiment using TruPATH constructs (MOR, Gαi1-Rluc8, Gβ, Gγ–GFP2)-Rluc8, in the presence and absence of OZITX, as a control to observe nucleotide affinities of receptor-free G proteins. Data presented are representative of n = 3 biologically independent experiments. Traces show individual raw data points, overlaid with nonlinear regression global fits. d, Comparison of GDPβS EC50s showing an inverse correlation between EC50 and efficacy of the respective ligand. Bar plots depict mean EC50 values from n = 3 biologically independent experiments. e, Nucleotide competition experiments, in which GTP EC50s are plotted in the presence of 0, 0.01, 0.1 and 1 µM GDPβS. Bar plots depict mean EC50 values from n = 3 biologically independent experiments. f-g, Nucleotide dose–response curves (GDPβS, GTPγS) in the presence and absence of 1 mM MgCl2. Bar plots depict mean EC50 values from n = 3 biologically independent experiments. h, Normalized efficacy across all tested signaling assays (One-GO, RG-BRET, TruPATH, GloSensor) of alvimopan, naloxone, mitragynineP, DAMGO, and loperamide plotted against the respective GDP EC50, showing a near perfect correlation, R2 = 0.86. for both signaling efficacies and GDP EC50s. Data presented are representative of n = 3 biologically independent experiments. d-g, Unpaired two-tailed Student's t-tests were conducted to compare nucleotide and MgCl2 treatment of each ligand. A significance threshold of α = 0.05 was applied for both the one-way ANOVA and the Student's t-test analyses. Scale bars in the micrographs represent 50 Å. Scale bar in the micrograph represents 50 Å. Scale bar in the micrograph represents 50 Å. a, Schematic overview of biochemical preparation with ‘constant GDP' continuously being supplied with GDP throughout the purification process. ‘Re-bound GDP' was treated with apyrase for complexation, followed by size-exclusion chromatography and finally supplemented with 100 µM GDP for cryoEM studies. b-c, Direct comparison of ‘constant GDP' versus ‘re-bound GDP' experiments for naloxone and loperamide, showing virtually identical 2D classes and 3D reconstructions of the respective state. Nucleotide binding pocket of Gαi, highlighting conformational rearrangements across six different conformational states receptor-free GDP-Gαi1β1γ2, latent, engaged, unlatched, primed, and nucleotide-free. MD simulations of GDP-bound cryoEM structures were performed to study conformational dynamics of the respective state. a-e, Overlays of 10 randomly selected snapshots of Gαi from one representative trajectory (out of 5 total) each with 1 µs duration. The opaque model highlights the initial model used for the MD simulation, while transparent structures were obtained from subsequent frames. The opaque model highlights the initial model used for the MD simulation, while transparent structures were obtained from subsequent frames. Backbone traces of individual frames were omitted for clarity, the shown backbone represents the starting structure after minimization. k, Violin plot showing the average center-of-mass (COM) distances of residues within Gαi nucleotide binding pocket with GDP, across all 5 simulated trajectories, highlighting a series of conformational changes across the different states. ‘Primed' trajectory 1 and ‘unlatched' trajectory 2 were omitted from this figure, as GDP ejects completely. i, Traces of individual MD trajectories, focusing on TM3-TM6 distances. 1–4, BRET assay controls, construct information, biochemistry and MD simulation results. Tables 1–3, CryoEM data collection, refinement and validation statistics. Overview of all 25 trajectories greater than 1 µs: GDP–Gαi1β1γ2, latent, engaged, unlatched and primed. The respective videos were generated using every fifth frame of the original 10,000 frames. Overview of all 25 trajectories over 1 µs - GDP-Gαi1β1γ2, latent, engaged, unlatched, primed. The respective videos were generated using every 5th frame of the original 10,000 frames. Source data for Supplementary Fig. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Khan, S., Tyson, A.S., Ranjbar, M. et al. Structural snapshots capture nucleotide release at the μ-opioid receptor. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09662-z'>Lymphoid gene expression supports neuroprotective microglia function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 16:06:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The microglial response to amyloid plaques in AD can range from neuroprotective to neurotoxic2. Here we show that the protective function of microglia is governed by the transcription factor PU.1, which becomes downregulated following microglial contact with plaques. Lowering PU.1 expression in microglia reduces the severity of amyloid disease pathology in mice and is linked to the expression of immunoregulatory lymphoid receptor proteins, particularly CD28, a surface receptor that is critical for T cell activation3,4. Microglia-specific deficiency in CD28, which is expressed by a small subset of plaque-associated PU.1low microglia, promotes a broad inflammatory microglial state that is associated with increased amyloid plaque load. Our findings indicate that PU.1low CD28-expressing microglia may operate as suppressive microglia that mitigate the progression of AD by reducing the severity of neuroinflammation. This role of CD28 and potentially other lymphoid co-stimulatory and co-inhibitory receptor proteins in governing microglial responses in AD points to possible immunotherapy approaches for treating the disease by promoting protective microglial functions. Alzheimer's disease is associated with diverse phenotypic changes in microglia5,6, resulting in distinct microglial states, including protective (aimed at amyloid clearance and neuroprotection7,8,9) and harmful (associated with microglia-driven neuroinflammation and toxicity)10,11. These opposing states are accompanied by broad changes in gene expression12,13,14,15,16,17, suggesting that defined transcriptional programs in microglia may govern their neuroprotective or neurotoxic functions. Here we present evidence that the neuroprotective state of microglia is governed by reduced expression of PU.1, a non-canonical pioneer transcription factor that functions as a master regulator of myeloid and lymphoid lineage differentiation18,19,20. PU.1 regulates lineage-specific gene expression in a dose-dependent manner21,22,23. More recently, PU.1 abundance in microglia has been linked to AD risk. Human genetic data show that a common variant, located within the 3′ untranslated region of the PU.1-encoding gene SPI1 and associated with reduced PU.1 expression in myeloid cells, correlates with delayed disease onset and reduced severity24. We found that PU.1 expression in microglia is regulated not only genetically but also epigenetically by the microenvironment. These PU.1low microglia preferentially co-localized with amyloid plaques, in both 5xFAD mice (Fig. 1c and Supplementary Table 2), and increase in numbers with disease progression (Extended Data Fig. Unlike non-plaque-associated (distal microglia; Extended Data Fig. 1b) or PU.1high plaque-associated microglia, this subpopulation appears independent of the activity of the essential microglial survival receptor CSF1R26,27 (Fig. The CSF1R-independent survival of PU.1low microglia at plaques may reflect either their non-microglial origin or the engagement of alternative compensatory survival pathways. Lineage-tracing experiments using a microglia-specific genetic approach (translating ribosome affinity purification (TRAP) mice28; Fig. 2 and Supplementary Table 3) confirmed that the plaque-associated CSF1R-independent PU.1low cells, despite showing low Csf1r gene expression (Supplementary Figs. a, Uniform manifold approximation and projections (UMAPs) of ex vivo isolated forebrain microglia from 8-month-old wild-type and 5xFAD mice (two males per group) show distinct PU.1 expression states (PU.1low, PU.1med and PU.1high) among DAM12 (red). b,c, PU.1 protein is reduced in a subset of plaque-associated microglia in the cortex of 6-month-old 5xFAD mice (n = 5; one female and four males) (b) and in the frontal cortex of individuals with AD (79–91 years; n = 3; one female and two males) (c). Bar graphs show the proportion of PU.1low cells in plaque-associated microglia versus distal microglia, analysed using a paired two-tailed t-test. d,e, Plaque-associated PU.1low microglia are resistant to CSF1R inhibition (CSF1Ri). d, UMAP visualizations of lineage-traced cortical microglia (TRAP28) from 6-month-old control (one female) or CSF1Ri-fed 5xFAD mice (two males), analysed using single-nucleus sequencing (DAM outlined with dotted line). The pie chart shows PU.1low fraction within DAM. e, Reduced PU.1 protein expression in CSF1Ri-resistant cortical microglia in the cortices of 6-month-old CSF1Ri (n = 4; one female and three males) versus control diet-fed (n = 8 mice; six females and two males) 5xFAD mice. Bar graph shows the proportion of PU.1low microglia, analysed using an unpaired two-tailed t-test. f, Microglial SYK/PLCγ2 signalling promotes the plaque-associated PU.1low state. Bar graphs show plaque-normalized microglial numbers and PU.1low microglia fractions, as determined by ordinary one-way ANOVA with multiple comparisons. Microglial nuclei identified using Imaris (b,f) and CellProfiler (e) in dotted circles. Illustrations in a, d and f were created using BioRender (https://biorender.com). The PU.1low microglial population at amyloid plaques may be established either through a pre-existing PU.1low microglial subpopulation that is recruited specifically to plaques or by the downregulation of PU.1 expression in microglia in response to plaque-associated cues. Our data support the latter mechanism. Microglial accumulation at plaques and plaque-driven morphological and transcriptional responses depend on microglial surface receptor-mediated signalling8,12,13,29, which involves activation of the tyrosine kinase (SYK)9,30 and one of its key downstream targets (PLCγ2)31,32. Microglia-specific inactivation of either SYK (5xFAD;Cx3cr1CreERT2/+;Sykfl/fl; referred to as 5xFAD;SYK-KO) or PLCγ2 (5xFAD;Cx3cr1CreERT2/+;Plcg2fl/fl; referred to as 5xFAD;PLCγ2-KO), which impairs microglial engagement with the plaques9,30,32 (Fig. 1f), also significantly reduces the number of PU.1low microglia (Fig. 1e) and averts their CSF1R-independent survival at the plaques (Extended Data Fig. Activation of SYK–PLCγ2-dependent signalling pathways in plaque-associated microglia occurs following the engagement of surface receptor proteins, including TREM2, CLEC7A and FcγR33. To confirm that PU.1 downregulation is driven by surface receptor-mediated signalling, we exposed ex vivo isolated microglia to phosphatidylserine/phosphatidylcholine (PS/PC)-containing liposomes31 or β-1,6-glucan pustulan34, which engage TREM2 and CLEC7A, respectively. Both ligands induced PU.1 downregulation (Extended Data Fig. 2a–d), which was attenuated by pharmacologic inhibition of downstream phospholipase C (PLC) activity35 (Extended Data Fig. Conversely, activation of PLC36 was sufficient to downregulate PU.1 even in the absence of external ligands (Extended Data Fig. 2e), confirming that activation of the SYK–PLCγ2 signalling pathways triggered by plaque-associated cues can regulate PU.1 expression in microglia. The PU.1low plaque-associated microglia displayed gene expression patterns that, in addition to promoting lipid metabolism and lysosomal function37,38,39,40 (Extended Data Fig. 3a), showed a resemblance to lymphoid lineage cells (Fig. 3a,b and Supplementary Table 1), lineage-traced single-nucleus (Extended Data Fig. 3c–e and Supplementary Table 3) or spatial transcriptomics analyses (Fig. 3 and Supplementary Table 4) revealed that PU.1low plaque-associated microglia express genes encoding immunoregulatory lymphoid receptors and signalling proteins, such as CD28, PD-1 (Pdcd1), PD-L1 (Cd274), CD5, CTLA-2A, CD52, LAT2 and CD72 (Extended Data Fig. The transcripts encoding these proteins are associated with ribosomes in 5xFAD microglia (Fig. Microglial lymphoid gene expression in PU.1low disease-associated microglia (DAM) is not restricted to mice but is also observed in individuals with AD15 (Fig. a, The volcano plot shows genes upregulated in PU.1low (red) versus PU.1high (blue) DAM in 8-month-old wild-type and 5xFAD mice (two males per group), as determined by Wilcoxon rank-sum test. b,c, PU.1low lymphoid gene-expressing cortical microglia localize near amyloid plaques. b, Representative MERFISH images of plaque-associated and distal microglia from an 8-month-old male 5xFAD brain. c, UMAP of spatial microglia distribution to the nearest plaque using MERFISH. The volcano plot shows differentially expressed genes in plaque-associated microglia (red; less than 15 μm to plaques) versus distal microglia (blue; greater than 15 μm to plaques), as determined by Wilcoxon rank-sum test. d, Lymphoid protein-encoding RNAs associate with ribosomes in lineage-traced microglia. The heat map (z-score) shows selected lymphoid lineage genes identified using microglia-specific TRAP sequencing28 from the cortices of 6-month-old 5xFAD mice (Cx3cr1CreErt2/+;5xFAD;Eef1a1LSL.eGFPL10a/+) versus control (Cx3cr1CreErt2/+;Eef1a1LSL.eGFPL10a/+ mice; n = 6 per group; three females and three males), as determined using DESeq2. Each column represents an individual mouse. e, PU.1low lymphoid gene-expressing microglia are present in individuals with AD. The dot plots show scaled average expression of selected genes in human microglia15 organized into homeostatic, lymphoid+ DAM and lymphoid– DAM clusters. Illustrations in c and d were created using BioRender (https://biorender.com). To test the causal role of PU.1 downregulation in mediating the transcriptional shift towards a lymphoid gene expression pattern, we engineered mice with genetically altered PU.1 expression levels in microglia. Microglia-specific Cre-dependent deletion of one Spi1 allele41 (PU.1-low mice; Cx3cr1CreERT2/+;Spi1fl/+) resulted in an approximately 50% PU.1 reduction (Fig. Conversely, PU.1-high mice were generated by Cre-dependent activation of an extra FLAG-tagged Spi1 gene (Cx3cr1CreERT2/+;Rosa26LSL.FLAG-Spi1/+), showing an approximately 25% increase in PU.1 expression (Fig. The changes in PU.1 expression levels were induced postnatally between 4 and 6 weeks to avoid developmental confounds. a, Microglial PU.1 protein expression (green) correlates with Spi1 gene copy number in PU.1-low, PU.1-wt and PU.1-high mice. Bar graph shows the mean PU.1 intensity within the cortical microglia in 6-month-old control (Cx3cr1CreErt2/+; n = 3; one female and two males), PU.1-low (Cx3cr1CreErt2/+;Spi1fl/+; n = 3; one female and two males) and PU.1-high (Cx3cr1CreErt2/+;R26LSL.F-Spi1/+; n = 3 females) mice, as determined by ordinary one-way ANOVA with multiple comparisons. Microglial nuclei identified by CellProfiler are outlined (dotted circles). The heat map shows differentially expressed genes (DESeq2; z-scored variance-stabilizing transformation (VST)) from cortical microglia-specific TRAP sequencing from 6-month-old PU.1-low (n = 6; four females and two males), PU.1-wt (n = 6; three females and three males) and PU.1-high (n = 4; three females and one male) mice. c,d, PU.1low microglia display T-cell-like chromatin accessibility profiles in vivo. c, Profile plots (top) and heat maps (bottom) show normalized genome-wide chromatin accessibility by assay for transposase-accessible chromatin sequencing (ATAC-seq) (reads per kilobase per million mapped reads (RPKM)) from the microglia of PU.1-low, PU.1-wt and PU.1-high mice (9–10 months old; n = 3 males per genotype) and splenic T cells (n = 2 males) from wild-type mice. Data are centered on ±4 kb around differentially accessible (DESeq2) peaks in PU.1-low compared with wild-type microglia. Peaks are ranked by signal intensity using the same minimum and maximum ranges for plots and heat maps. d, Genome browser views (Integrative Genomics Viewer) of Cd28 and Cd22 gene loci in T cells versus PU.1-low, PU.1-wt and PU.1-high microglia. Bracketed numbers indicate the minimum and maximum y-axis values for each sample. Illustrations in a were created using BioRender (https://biorender.com). Reduction of PU.1 expression in microglia was sufficient to recapitulate the lymphoid gene signature observed in plaque-associated PU.1low microglia, even in the absence of amyloid pathology (Fig. Moreover, PU.1low microglia, although maintaining an overall microglial lineage-specific chromatin accessibility pattern (Extended Data Fig. 6 and Supplementary Table 7), showed more T cell-like accessibility changes (Fig. This transcriptional shift towards lymphoid gene (Fig. 5), which is associated with an increase in microglial SYK–PLCγ2 signalling capacity (Extended Data Fig. 6), can be triggered in vitro by acute PU.1 knockdown through RNA interference in microglia-like BV2 cells39 (Extended Data Fig. 5b,c and Supplementary Table 8), primary mouse microglia (Extended Data Fig. 5d) or human induced pluripotent stem cell (iPSC)-derived microglia (induced microglia-like cells (iMgls); Extended Data Fig. These data support a role for reduced PU.1 expression in a microglial shift towards lymphoid-associated gene expression in mice and humans. By contrast, increased PU.1 expression in microglia is associated with a pro-inflammatory phenotype38,39 both in the non-diseased brain (Fig. 7 and Supplementary Table 4; single-cell sequencing, Extended Data Fig. Lowering PU.1 in the context of amyloid pathology (5xFAD; PU.1-low mice) has a protective effect on disease progression; it enhances microglia–plaque interactions42 by nearly doubling the number of plaque-associated and quadrupling the number of lymphoid gene-expressing/CD28+ microglia (MERFISH (Fig. 7 and Supplementary Table 10), TRAP sequencing (Extended Data Fig. 8 and Supplementary Table 11), morphological analysis (Fig. These microglia display phenotypes linked to reduced neurotoxic activity10,43,44, marked by decreased expression of type I interferon-stimulated genes (such as Mx1) and complement (such as C1q), at both RNA (MERFISH (Supplementary Fig. 7 and Supplementary Table 4), single-nucleus RNA-seq (Extended Data Fig. 9c,d) levels, as well as a decline in neutral lipid droplet accumulation (Extended Data Fig. PU.1low microglia abundance was further associated with decreased amyloid burden and enhanced plaque compaction (Fig. In addition, PU.1low microglia are more effective in blocking the spread of injected human Tau aggregates in the brain of 5xFAD mice, leading to an overall reduction in pathogenic Tau accumulation (Fig. a,b, UMAP (a) and bar graph (b) show microglial subpopulations and their distribution on the basis of MERFISH data in wild-type (n = 2 males), 5xFAD (n = 3 males) and 5xFAD–PU.1-low (n = 2 males) mice (8–10 months old). c–f, Representative cortical immunofluorescence and quantification in 6-month-old mice, analysed using an unpaired two-tailed t-test. f, Plaque composition (6E10+ soluble amyloid-β relative to ThioS+ dense-core plaques) in the subiculum of 5xFAD (n = 7; two females and five males) and 5xFAD–PU.1-low (n = 5; two females and three males) mice. g, Seeding of AT8+ phosphorylated Tau (p-Tau) in 10-month-old mouse brains injected with human Tau aggregates. Bar graphs show percentage of AT8+ area in 5xFAD (n = 12; seven females and five males) and 5xFAD–PU.1-low (n = 9; four females and five males) mice, analysed using an unpaired two-tailed t-test. h–k, Synaptic analyses were performed in 6-month-old mice, and physiological and behavioural analyses were performed in 9–10-month-old mice, using ordinary one-way ANOVA with multiple comparisons. i, Late-phase LTP in CA1 hippocampal neurons of control (n = 12 slices from five mice; three females and two males), 5xFAD (n = 13 slices from six mice; four females and two males) and 5xFAD–PU.1-low (n = 11 slices from five mice; one female and four males) mice. j, Open-field thigmotaxis (centre/periphery) in control (n = 15; nine females and six males), 5xFAD (n = 12; six females and six males) and 5xFAD–PU.1-low (n = 13; seven females and six males) mice. k, Novel object recognition scores in control (n = 10; five females and five males), 5xFAD (n = 11; six females and five males) and 5xFAD–PU.1-low (n = 15; seven females and eight males) mice. Scale bars, 20 μm (c–f,h), 1 mm (g). Illustration in g was created using BioRender (https://biorender.com). PU.1low microglia preserved synaptic integrity and associated cognitive function. Although 5xFAD mice were characterized by a significant loss of presynaptic neuronal terminals (Bassoon+ and VGLUT2+) in layer V of the cortex (Fig. 4h), impaired synaptic plasticity (long-term potentiation (LTP)) in the hippocampus (Fig. 9i,j), lowering PU.1 level in microglia was sufficient to prevent these disease-associated alterations and preserve near-wild-type-like phenotypes (Fig. The PU.1low microglial expression of lymphoid receptor proteins with potent signalling capacity raised a question about the contributions of these receptors to PU.1low microglia activity and neuroprotection. We demonstrated that CD28, a well-known T cell co-stimulatory signalling protein3,4, which is induced by PU.1 downregulation and in response to plaque-associated cues in mouse (Figs. 2e), contributes to microglial activation during AD. We found that despite being expressed in only a small fraction of total microglia in 5xFAD brains (Fig. 10c,d and Supplementary Table 12), CD28 plays an important regulatory role in broader microglia activation and disease progression. Microglia-specific deletion of CD28 in 5xFAD mice (Fig. 11a,b) increased amyloid plaque burden (Fig. 11c,d) and microglial activation, characterized by a broad pro-inflammatory type I interferon response in approximately 30% of all microglia (Fig. The frequency of microglia that display an inflammatory response during early disease progression exceeds the number of cells that express Cd28 by more than 10-fold (Fig. One interpretation of the observed broad inflammatory microglia activation is a possible trans-regulatory role for CD28+ PU.1low microglia in restraining neuroinflammation48 and associated amyloid plaque propagation. a,UMAPs show CD28+ microglia in 5xFAD DAM. Single-cell RNA-seq of forebrain microglia from 3-month-old, 6-month-old and 8-month-old control and 5xFAD mice (3 months (n = 1 control and n = 2 5xFAD), 6 months (n = 1 per group) and 8 months (n = 2 per group); all males); DAM outlined (dotted line). Dot and feature plots show microglial Cd28 expression. b, Inducible microglia-specific deletion of CD28 in 5xFAD mice. d, Single-cell sequencing of forebrain microglia from 5xFAD and 5xFAD;CD28-KO mice. Significant gene ontologies (Fisher's exact test) of differentially regulated genes using pseudobulk analysis (Wilcoxon rank-sum test) in 3- and 6-month-old mice by genotype (n = two 3-month-old male mice and n = one 6-month-old male mouse per group). e, UMAP (two 3-month-old male mice per group) and dot plots (control and 5xFAD (as in a) and 5xFAD;CD28-KO (two 3-month-old male mice and one 6-month-old male mouse) show expression of interferon-response genes. f,g, Human PU.1-lowering allele associates with increased lymphoid gene-expressing microglia. f, The heat map shows scaled average expression of indicated genes in surveillance clusters (Mic. g, PU.1 protective allele carriers have lower SPI1 expression, whereas risk allele carriers show higher SPI1 expression in AD microglia. Normalized SPI1 expression and fraction of CD28+ Mic. 13 population per individual versus cognitive diagnosis and PU.1 variant (rs1057233 A/A = risk (n = 61 not impaired; n = 64 AD); G/A and G/G = protective (n = 83 not impaired; n = 98 AD)) were analysed using two-way ANOVA with Tukey post hoc test. Bar graphs with individual data points show the mean ± s.e.m. Illustrations in b were created using BioRender (https://biorender.com). 3,4) and probably other functionally related immunoregulatory receptors in governing microglial responses is underscored by recent findings identifying the T cell immunoglobulin and mucin domain 3 (TIM-3) receptor as a novel regulator of microglial activity in AD49. TIM3 deficiency promotes a protective microglia state in 5xFAD mice, characterized by increased phagocytic and reduced pro-inflammatory activity. One of the distinct microglial populations that is expanded upon TIM-3 deficiency expresses CD28 alongside other immune activity regulators, including Pdcd1 (PD-1), Cd274 (PDL-1), Cd5, Btla and Lat2 while exhibiting markedly reduced PU.1 expression (Extended Data Fig. These findings raise the possibility that the neuroprotective effects of TIM-3 deficiency may involve CD28+PU.1low microglia, and that the coordinated activity of lymphoid-associated receptor gene expression in plaque-associated microglia underlies neuroprotection in AD. Our observations support a model in which the switch towards a neuroprotective microglial state depends on the expression of immunoregulatory lymphoid receptor proteins that potentially govern microglial activity in a manner analogous to T cell regulation3,4,50. It is tempting to speculate that PU.1low microglia expressing lymphoid gene signatures contribute to neuroprotection not only in mice but also in humans. A gene expression analysis from large cohorts of human microglia17 revealed that individuals carrying the PU.1-lowering protective allele (GG/GA versus AA), which was previously associated with improved cognitive performance and reduced AD severity24, have an increased abundance of the lymphoid gene-expressing microglia subset (Mic. The surface expression of CD28 and other lymphoid co-stimulatory or co-inhibitory receptor proteins on neuroprotective PU.1low microglia highlights a new opportunity for antibody-based microglia-driven immunotherapy in AD. Post-mortem human brain tissue samples for immunohistochemistry were obtained from patients who had enrolled in and provided consent for a brain donation programme through the Neuropathology Brain Bank and Research CoRE at the Icahn School of Medicine at Mount Sinai. These samples were collected in accordance with ethical guidelines and institutional review board approval at the Icahn School of Medicine at Mount Sinai, ensuring the privacy and dignity of the donors while supporting ongoing neurological research. Tissue was obtained from donors who had provided written informed consent for research use either directly or through their next of kin. Three post-mortem brain samples (frontal cortex) were obtained from patients affected by Alzheimer's disease neuropathological changes. Neuropathological assessments were performed at the respective centres using standardized criteria, including the Consortium to Establish a Registry for Alzheimer's Disease neuritic plaque assessment and Braak neurofibrillary tangle staging51,52. In this study, we did not recruit participants; we only de-identified post-mortem brain samples from individuals. Research with de-identified autopsy material does not meet the federal regulatory definition of human subject research, as defined in 45 Code of Federal Regulations Part 46. However, Health Insurance Portability and Accountability Act requirements still apply. Human samples of induced pluripotent stem cell lines were generated by the Icahn School of Medicine at Mount Sinai, University of California, Irvine Alzheimer's Disease Research Center (UCI ADRC)and Washington University Alzheimer's Disease Research Center (WashU ADRC) Induced Pluripotent Stem Cell Core from subject fibroblasts or peripheral blood mononuclear cells (PBMCs) with approved Institutional Review Boards and Human Stem Cell Research Oversight committee protocols at the Icahn School of Medicine at Mount Sinai, UCI ADRC and WashU ADRC Induced Pluripotent Stem Cell Core. The consent for reprogramming human somatic cells to human samples of induced pluripotent stem cell was obtained through the Human Stem Cell Research Oversight protocols 19-04, 2013-9561 and 2017-1061. Informed consent was received from each participant who donated fibroblasts or PBMCs. Human demographic information, including sex, age, AD diagnosis, Braak state, post-mortem interval and/or clinical cognitive status, can be found in Supplementary Table 2. This study does not report on race, ethnicity or other socially relevant groupings to minimize the likelihood of unintentional identification of de-identified samples. Mice were housed in cages with two to five animals per cage, with a 12-h light/dark cycle (lights on from 0700 to 1900 hours), a constant temperature (23 °C) and ad libitum access to food and water. All animal protocols were approved by the Institutional Animal Care and Use Committee at Icahn School of Medicine at Mount Sinai and performed according to the National Institutes of Health (NIH) guidelines. The Plcg2fl/+ mouse56 was generously provided by T. Inoue and T. Kurosaki (RIKEN Institute). The Eef1a1LSL.eGFPL10a/+ mouse was generously provided by A. Domingos (Instituto Gulbenkian de Ciência) and J. Friedman (Rockefeller University). The Mx1::GFP mouse57 was generously provided by A. García-Sastre (Icahn School of Medicine at Mount Sinai). To achieve ectopic expression of the Spi1 gene, a floxed transcriptional STOP cassette followed by FLAG-tagged mouse Spi1 complementary DNA (cDNA) was targeted into the Rosa R26 locus of embryonic stem cells using the TV-targeting vector for homologous recombination at the ROSA26 locus, generously provided by K. Rajewsky58 and available at Addgene (plasmid no. A schematic is shown in Extended Data Fig. In brief, a transcriptional STOP cassette consisting of six SV40 polyadenylation sites was flanked by loxP recognition sequences to facilitate Cre-mediated deletion and subsequent activation of exogenous FLAG-tagged Spi1. The original plasmid with mouse Spi1 sequence was provided by C. Vakoc24,59 (Cold Spring Harbor Laboratory). Routine genotyping of Rosa26LSL.FLAG-Spi1/+ mice (Extended Data Fig. 4a) was performed using the following primers: To generate SYK-KO mice, Cx3cr1CreErt2/+(Litt) (ref. Microglia-specific ribosomal profiling and single-nucleus sequencing were performed on mice further crossed with Eef1a1LSL.eGFPL10a/+ mice60. For ISRE-reporter, Cx3cr1CreErt2/+(Litt);Spi1fl/+;5xFAD mice were additionally crossed with Mx1::GFP mice57. Unless otherwise specified, Cx3cr1CreErt2/+(Litt) mice were used as controls. For all experiments involving disease models, mice were crossed with 5xFAD mice25. To control for potential differences in expression levels arising from paternal versus maternal transmission of the 5xFAD transgene61, litter-matched animals were used for comparison when applicable. Given the reported sex differences in some disease pathology phenotypes46,47,62, mice from both sexes were used unless otherwise specified. When both male and female mice were used, an analysis for sex differences was performed; unless otherwise noted, no sex differences were observed. All mice used in the experiments were backcrossed into the C57Bl/6 background for six or more generations. To activate tamoxifen-inducible Cre (CreErt2), all mice were gavaged at 4–6 weeks of age with six doses of 100 mg kg−1 of tamoxifen (T5648; Sigma) in corn oil (C8267; Sigma), administered at intervals of at least 48 h. For microglia ablation, 5xFAD mice (4–5 months old) were treated with CSF1R inhibitor PLX5622 (1,200-ppm diet, generously provided by Plexxikon) or vehicle for 4–6 weeks. PLC inhibitor (PLCi) injections were performed intraperitoneally daily for 5 days at 8.7 mg kg−1. 35,63) was first dissolved in chloroform at 100 mM. This solution was aliquoted and dehydrated using nitrogen. The dehydrated film was dissolved in dimethyl sulfoxide (DMSO) at 5 mM, diluted 1:8 in saline and injected. AD-Tau was isolated, as previously described, from the brain of a human with AD64,65. Using a bicinchoninic acid assay (Thermo Fisher Scientific; 23225), the total protein concentration of the AD-Tau preparation was 21.1 µg µl−1. The Tau concentration in the AD-Tau preparation was determined, as previously described, using a Tau-specific sandwich enzyme-linked immunosorbent assay, which was found to be 5.4 µg µl−1 (refs. 64,65). The AD-Tau preparation was diluted to 0.4 µg µl−1 before the injections and sonicated in a water bath sonicator (Qsonica; Q700) for 30 s at 60% amplitude at 4 °C. The 7-month-old 5xFAD (seven females and five males) and 5xFAD;PU.1-low mice (four females and five males) were anaesthetized with isofluorane, immobilized in a stereotactic frame (Kopf Instruments) and unilaterally injected with a total of 2-µg AD-Tau (1 µg per injection site) in the dentate gyrus (Bregma, −2.5 mm; lateral, −2.0 mm; depth, −2.2 mm) and overlying cortex (Bregma, −2.5 mm; lateral, −2.0 mm; depth, −1.0 mm) using a Hamilton syringe (80265–1702RNR; needle, 7803–07). The mice were allowed to recover on a 37 °C heating pad and monitored for the first 72 h after surgery. Although male mice show overall slightly lower levels of amyloid and Tau pathology62,66, significant differences between genotypes were preserved as trends in both sexes when the sexes were separated. Human iPSCs (see Supplementary Table 2 for demographic information) generated by the Icahn School of Medicine at Mount Sinai, UCI ADRC and WashU ADRC Induced Pluripotent Stem Cell Core from subject fibroblasts or PBMCs were maintained on Matrigel-coated six-well plates (BD Biosciences) in complete StemFlex (Thermo Fisher Scientific; A3349401). iPSCs were passaged every 5–6 days using ReLeSR dissociation reagent (STEMCELL Technologies; 05872) and used for haematopoietic stem cell differentiation with STEMdiff Hematopoietic Kit (STEMCELL Technologies; 05310), followed by differentiation to iMgls using a previously published protocol67. The iMgls were maintained and fed with a microglial medium supplemented with three factors (3F medium containing 100 ng ml−1 of interleukin (IL)-34, 50 ng ml−1 of transforming growth factor (TGF)β and 25 ng ml−1 of macrophage colony-stimulating factor) for 25 days. On day 25, cells were collected for nucleofection using P3 Primary Cell 4D-Nucleofector X Kit (Lonza; V4XP-3024) and 4D-Nucleofector System (Lonza; AAf-1003X). The cells were nucleofected with 50 nM of SMARTpool ON-TARGETplus short interfering RNA (siRNA) targeting human SPI1 (Dharmacon; L-010537-00-0005) or non-targeting pool (Dharmacon; D-001810-10-05). Microglia were isolated from neonatal (postnatal day P7–P10) mice for ex vivo cell culture and RNA isolation, as previously described68. Tissue was mechanically dissociated in glass homogenizers, and live cells were enriched by centrifugation over 20% Percoll (17-0891-02; GE HealthCare). Microglia were selected using anti-CD11B-coated microbeads (130-093-636; Miltenyi Biotec) with the QuadroMACS separator, following the manufacturer's recommendations. Cells were manually counted with a haemocytometer using trypan exclusion staining. For immunoblotting, immunocytochemistry and RNA in situ hybridization, the microglia were plated at 250,000 cells per well using DMEM supplemented with 10% fetal bovine serum (FBS; Gibco; 16140) and 100 U ml−1 of penicillin–streptomycin (Pen/Strep; Gibco; 15140) and allowed to adhere for 3–4 days before any experimental treatments. For immunocytochemistry and in situ experiments, cells were plated on poly-d-lysine-coated coverslips (8774384; Thermo Fisher Scientific). Initial validation of the protocol for microglial purity was described previously68. In each experiment, microglial purity was visually determined by IBA1 or CD11b immunostaining. Cultures were consistently found to be purely microglia. These microglia were used to determine the regulation of PU.1 or CD28 levels following TREM2, CLEC7A or PLC stimulation in Extended Data Figs. Primary microglia were isolated from postnatal days 2–3 CD1 mouse pups, as described previously69. Cortical tissue was lysed at 37 °C for 25 min using 2.5% trypsin (Thermo Fisher Scientific; 15090046). Afterward, DNase I (Sigma; D4263-1VL) was added, and cells were pelleted at 270g for 5 min at 4 °C. The pellet was resuspended in DMEM (Thermo Fisher Scientific; 11965092) supplemented with 10% FBS (Gibco; 10437) and 1% Pen/Strep/l-glutamine (Gibco; 10378016) and passed through a 70-μm cell strainer. The cells were seeded into poly-d-lysine-coated T75 flasks and incubated at 37 °C with 5% CO2. The medium was changed on day in vitro (DIV)2 and supplemented with L929-conditioned medium on DIV6 and DIV10. These microglia were used for PU.1 or CD28 quantification following siRNA magnetofection, as shown in Extended Data Fig. Primary microglia were magnetofected 24 h after seeding using the Glial-Mag magnetofection kit (GL00250), following the manufacturer's instructions. In brief, siRNA (non-targeting control pool (scramble), Spi1 and Cd28; Horizon Discovery; D-001810-10-05, L-041420-00-0005 and L-058630-00-0005, respectively) was diluted in DMEM (Gibco; 11-965-092) and mixed with Glial-Mag transfection reagent. After a 20-min incubation at room temperature, the solution was added to each respective well in a dropwise manner. The final concentration of siRNA was 25 nM. Cells were then incubated on top of a magnetic plate (OZ Biosciences; MF10000) inside a humidified incubator at 37 °C and 5% CO2 for 30 min. The magnetic plate was removed, and cells were incubated for an extra 3 h. Afterward, the medium was removed and replaced with 50% collected conditioned medium + 50% fresh TGFβ–IL34–cholesterol medium. Knockdown efficiency was analysed 48 h after magnetofection using quantitative polymerase chain reaction (qPCR) and western blot. The BV2 mouse microglial cell line was provided by M. Diamond (UT Southwestern Medical Center). Mycoplasma contamination testing was performed on a regular basis every 3 months, and all cell lines tested negative. The BV2 cells were cultured in DMEM (Gibco; 11965) supplemented with 5% heat-inactivated FBS and 100 U ml−1 of Pen/Strep. We have previously generated BV2 cells stably overexpressing mouse PU.1 and respective empty frame control, as well as BV2 cells with knockdown of PU.1 with the respective scrambled control39. Cells were used for western blot measurement of PU.1 and CD28 (Extended Data Fig. 5b), mass spectrometry analysis (Extended Data Fig. 5c) and signalling phospho-protein analysis (Extended Data Fig. For TREM2 activation, PS/PC liposomes (70% DOPC and 30% POPS) resuspended in phosphate-buffered saline (PBS) were purchased from CD Bioparticles (CDECPS-1676). Microglial cultures were treated with liposomes resuspended in PBS at 0.1 mg ml−1 and 1 mg ml−1 for 12 h and 3 days, unless otherwise stated. For washout experiments, cells were treated with PS/PC liposome or vehicle for 3 days, the liposome-containing medium was replaced with fresh medium and cells were analysed 4 days later. For PLCi and PLC activator (PLCa) experiments, we used 0.1 μM U73122 (PLCi; Tocris) and 40 μM m-3M3FBS (PLCa; Tocris). U73122 was first dissolved in chloroform at 100 mM. This solution was aliquoted and dehydrated using nitrogen. The dehydrated film was dissolved in DMSO at 5 mM. PLCi was added 1 h before liposome treatment. For CLEC7A activation, pustulan (InvivoGen; tlrl-pst) was resuspended in PBS. Microglial cultures were treated with 50 μg ml−1 of pustulan for 3 days. The 5-μm-thick paraffin-embedded post-mortem brain sections from individuals with AD (see Supplementary Table 2 for demographic information) were pretreated using Cell Conditioning CC1 antigen retrieval buffer (Tris/borate/EDTA buffer; pH 8.0–8.5; 950-224; Roche Diagnostics) and incubated in rounds of staining with primary antibodies against PU.1 (clone 9G7; Cell Signaling Technology; 2258S; 1:50), IBA1 (Thermo Fisher Scientific; PA5-27436; 1:500) to label microglia, β-amyloid (clone 4G8; BioLegend; 800701; 1:4,000) and secondary antibody Multimer HRP OmniMap Anti-Mouse and Multimer HRP OmniMAP Anti-Rabbit (760-4310 and 760-4311; Roche Diagnostics; 1:1). Detection was performed using the DISCOVERY Purple HRP Kit, DISCOVERY Green HRP Kit and DISCOVERY Yellow HRP Kit (760-229, 760-271 and 760-250, respectively; Roche Diagnostics). Sodium citrate antigen retrieval (pH 6.0) was used between rounds to remove the antibody from the previous round and avoid cross-reactivity. Digital images of the stained sections were captured at ×40 using the Aperio GT 450 high-resolution scanner (Leica Biosystems). Images were also acquired using the Nikon Eclipse Ci microscope at ×20 and ×40 magnification. For each sample, three regions of interest (ROI) were annotated in plaque-dense areas, and three ROI were annotated in plaque-depleted areas in the cortical grey matter, with each ROI measuring 1 mm2. Microglia were manually counted in each ROI. Microglia were considered plaque-associated if there was at least one IBA1+ process co-localized with an amyloid-β plaque. If microglia did not meet these criteria, the cell was considered distal. PU.1 level was manually scored as either low or high on the basis of the staining intensity of PU.1 within microglia nuclei. For PU.1 and CD28 quantification, primary microglia isolated by anti-CD11B antibody were washed twice with PBS and fixed with 4% paraformaldehyde (PFA) at room temperature for 15 min, followed by three more PBS washes. Coverslips were washed and permeabilized with 0.2% PBS–Tween, followed by a 1-h blocking step with 4% normal goat serum in 0.2% PBS–Tween. The coverslips were then incubated with the following primary antibodies overnight at 4 °C: PU.1 (1:200 Cell Signaling Technology; 2258), CD11B (1:500; Thermo Fisher Scientific; 14-0112-85) and IBA1 (1:500; Synaptic Systems; 234 009) to label microglia and/or CD28 (1:50; Cell Signaling Technology; 38774).The coverslips were washed three times with PBS + 0.2% Triton X-100 (PBST) and incubated for 1 h at room temperature with secondary antibodies (goat anti-rabbit 1:500 and goat anti-rat 1:500), conjugated to Alexa Fluor 488, Alexa Fluor 546 or Alexa Fluor 568. To label nuclei, the coverslips were washed twice in PBST and once in PBS and then stained with DAPI (1:10,000 in PBS) for 20 min at room temperature.The coverslips were then embedded on a microscopy slide using ProLong Gold Antifade Mountant with DAPI (Invitrogen) and dried overnight. Imaging was performed using an LSM 780 confocal microscope (ZEISS). For z-stack images, 20-μm z-stack confocal images were acquired at 2-μm intervals, with ×20 objective at ×1 zoom. Image processing was performed using ZEN 2011 software (ZEISS). Image quantification of CD28 in vitro, z-section confocal microscopy images were applied to custom ImageJ macros. Using the maximum intensity projected z stack of the CD28 channel, the CD28 signal was detected using the default ImageJ threshold and measured using integrated density (as shown in figure quantifications unless otherwise specified) and mean grey value, both of which yielded comparable results. Cells were washed twice with PBS and fixed with 4% PFA at room temperature for 15 min, followed by three more PBS washes. Coverslips were then dehydrated in 50%, 70% and 100% ethanol and allowed to dry before in situ hybridization. In situ hybridization was performed using RNAscope custom-designed probes for Spi1 in combination with the RNAScope 2.0 RED Kit, according to the manufacturer's recommendation (Advanced Cell Diagnostics). After completing in situ hybridization, sections were rinsed with double-distilled water and a gradient concentration of PBS (0.1×–1×) before the colorimetric reaction, blocked in 2% normal goat serum in PBS for 1 h at room temperature. The coverslips were then incubated with primary antibodies overnight at 4 °C: PU.1 (1:200; Cell Signaling Technology; 2258) and CD11B (1:500; Thermo Fisher Scientific; 14-0112-85) to label microglia. The coverslips were then washed three times with PBST and incubated with a secondary antibody (goat anti-rabbit 1:500; goat anti-rat 1:500, conjugated to the fluorophore Alexa Fluor 488 or Alexa Fluor 546) for 1 h at room temperature. To label nuclei, the coverslips were washed twice in PBST and once in PBS and then stained with DAPI (1:10,000 in PBS) for 20 min at room temperature. Sections were rinsed with a gradient concentration of PBS (0.1×–1×) and ddH2O, subjected to colorimetric reaction, dried for 15 min at 60 °C, mounted using EcoMount (EM897L; Biocare Medical) and dried overnight. Z-stack images were taken on a ZEISS LSM 780 confocal microscope with 20-μm z-stack confocal images acquired at 2-μm intervals and ×20 objective at ×1 zoom. Image processing was done using ZEN 2011 software. Z-section confocal microscopy images were applied to custom ImageJ macros. Nuclei in the DAPI channel were segmented using the default ImageJ threshold, followed by the ‘AnalyzeParticles' function without size and circularity constraints to identify all microglia nuclei. These DAPI ROI were then projected onto PU.1 maximum intensity projected z stack of the corresponding PU.1 channel to measure the nuclear PU.1 signal through mean grey value (as shown in figure quantifications unless otherwise specified) and integrated density measurements, both of which yielded comparable results in PU.1 signal in treatment versus control conditions. For the quantification of microglia Spi1 signal from in situ hybridization and correlation analyses between PU.1 versus Spi1 signal within microglia, the ImageJ freehand selection tool was used to manually outline the borders of individual microglia (ROI for Spi1) along with their corresponding nuclei (ROI for PU.1). Within these microglia, the ROI mean grey value and integrated density measurements were calculated, yielding comparable results in Spi1 and PU.1 signal in treatment versus control. At least 30 microglia from each coverslip from each condition were quantified this way, and the average mean grey value of these cells is shown in the figure quantifications unless otherwise specified. For immunofluorescence and RNA in situ hybridization of mouse tissue, mice were anaesthetized with ketamine (120 mg kg−1) and xylazine (24 mg kg−1) and perfused transcardially with 10-ml PBS and 40 ml of 4% PFA (Electron Microscopy Sciences), as previously described70. Following dehydration, the brains were frozen in Neg-50 (Life Technologies) on dry ice and stored at −80 °C until further processing. The brains were cut using a cryostat, and 10-μm or 30-μm sections were mounted on Superfrost Plus Microscope Slides (Thermo Fisher Scientific), which were stored at −80 °C until staining. For immunohistochemistry of AT8 and amyloid-β, 10-month-old 5xFAD (seven females and five males) and 5xFAD;PU.1-low (four females and five males) mice were killed through intraperitoneal injection of pentobarbital (200 mg kg−1), 3 months after the AD-Tau injection. Whole brains were extracted and fixed in 4% PFA for 24 h before being transferred to 30% sucrose and stored at 4 °C until sectioned. The brains were cut coronally into 30-μm sections on a freezing sliding microtome (Microm; HM 400) and stored in a cryoprotectant solution (0.2 M PBS, 15% sucrose and 33% ethylene glycol) at −20 °C until use. For immunohistochemical staining of AD-Tau64,65 and amyloid-β, sections were washed three times in Tris-buffered saline (TBS) for 5 min and incubated in 0.3% hydrogen peroxide for 10 min. After washing, sections were blocked for 30 min in 3% milk in TBS containing 0.25% Triton X-100. The primary antibody was diluted in 1% milk in TBS containing 0.25% Triton X-100, and sections were incubated in the primary antibody overnight at 4 °C (AD-Tau (AT8; mouse monoclonal; 1:500; Thermo Fisher Scientific; MN1020B) and amyloid-β (HJ3.4 biotinylated; anti-amyloid-β N terminus 1–16; mouse monoclonal; 2 µg ml−1 generated in-house71). The next day, sections were washed three times and then incubated in ABC Elite solution (Vectastain; PK-6100) for 1 h, prepared following the manufacturer's instructions, followed by another washing step. Sections were developed in DAB solution (Sigma; D5905 for AT8; anti-amyloid-β N terminus 1–16 antibody; clone HJ3.4), washed and mounted on slides. After drying overnight, the slides were dehydrated in increasing ethanol concentrations followed by xylene and coverslipped with CytoSeal 60 (Thermo Fisher Scientific; 8310). For microglia size and number quantification, plaque-associated microglia quantification, 6E10, ThioS and boron dipyrromethene (BODIPY) imaging, 30-μm slides from 5xFAD;PU.1-low, 5xFAD–SYK-KO, 5xFAD;PLCγ2-KO or 5xFAD;CD28-KO mice with sex-matched and age-matched littermate control 5xFAD mice were washed with PBS, permeabilized with PBST and blocked with 2% normal goat serum in PBST. Slides were incubated with primary antibodies in 2% normal goat serum in PBST overnight at 4 °C. The primary antibodies included CD11B (1:1,000; MCA711GT; Bio-Rad) and IBA1 (1:750; Synaptic Systems; 234 009) to label microglia and 6E10 (1:500; 803004; BioLegend) to label amyloid-β. The slides were then washed and incubated with Alexa Fluor-conjugated secondary antibody (Alexa Fluor 488-labelled, 546-labelled or 568-labelled goat anti-rat IgGs heavy and light chains (H + L); 1:500; Life Technologies) in 2% normal goat serum in PBST for 1 h at room temperature. The slides were washed and incubated with Thioflavin S for 2 min (1% Thioflavin S diluted 1:1,000 in PBS; Sigma-Aldrich; T1892) to label dense-core plaques, followed by DRAQ5 for 20 min (1:2,500 in PBS) to label nuclei. For neutral lipid droplet imaging, BODIPY 505/515 (4,4-difluoro-1,3,5,7-tetramethyl-4-bora-3a,4a-diaza-s-indacene; Thermo Fisher Scientific; D3921) was used to label lipid droplets, according to the manufacturer's recommendations. Imaging was performed using a ZEISS LSM 780 confocal microscope. Confocal z-stack images (20 μm) were acquired at 2-μm intervals using a ×40/1.3 oil objective. Image processing was performed using ZEN 2011 software (ZEISS). For PU.1 quantification, 30-μm slides from 6-month-old sex-matched control, 5xFAD, CSF1Ri-treated 5xFAD, 5xFAD–SYK-KO, 5xFAD;PLCγ2-KO, SYK-KO, PLCγ2-KO, PU.1-low and/or PU.1-high and 20-month-old control mice were washed with PBS. Antigen retrieval was performed using 0.1% Triton X-100 and 0.1% sodium citrate for 10 min at room temperature, followed by DNAse treatment of the fixed tissue to increase nuclear accessibility to primary antibodies. The slides were blocked with 2% normal goat serum in PBST and then incubated overnight at 4 °C with primary antibodies in the same blocking solution. The primary antibodies included PU.1 (1:200; Cell Signaling Technology; 2258) and CD11B (1:500 Thermo Fisher Scientific; 14-0112-85) or IBA1 (1:750; Synaptic Systems; 234 009) to label microglia. The slides were then washed and incubated with Alexa Fluor-conjugated secondary antibodies (Alexa Fluor 488-labelled, 546-labelled and 568-labelled goat anti-mouse, goat anti-rat, goat anti-chicken, goat anti-rabbit or donkey anti-goat IgGs (H + L); 1:500; Life Technologies) in 2% normal goat serum in PBST for 1 h at room temperature. To label nuclei, the slides were incubated with DRAQ5 (1:2,000 in PBS; Life Technologies; 62254), washed, coverslipped using ProLong Gold Antifade (Invitrogen) and dried overnight. For images containing plaques, the plaques were stained with Thioflavin S for 2 min after secondary antibody incubation (1% Thioflavin S diluted 1:5,000 in PBS; Sigma-Aldrich; T1892) to label dense-core plaques. Nuclei were labelled using DRAQ5 (1:2,000 in PBS) for 20 min. Imaging was performed using a ZEISS LSM 780 confocal microscope. Confocal z-stack images (20 μm) were acquired at 2-μm intervals using a ×40/1.3 oil objective. Image processing was performed using ZEN 2011 software (ZEISS). For CD28 quantification, 30-μm slides from 5xFAD;CD28-KO and 5xFAD;PU.1-low mice with sex-matched and age-matched littermate control 5xFAD mice were washed with PBS, permeabilized with PBST and blocked with 2% normal goat serum in PBST. The slides were incubated with primary antibodies in 2% normal goat serum in PBST overnight at 4 °C. The primary antibodies included IBA1 (1:500; Synaptic Systems; 234 009) to label microglia and CD28 (1:500; Cell Signaling Technology; 38774). The slides were then washed with PBST and incubated with Alexa Fluor-conjugated secondary antibodies for 1 h (Alexa Fluor 488-labelled goat anti-chicken IgG (H + L); 1:500; Life Technologies). The slides were washed and incubated with DRAQ5 (1:2,000 in PBS) for 20 min to label nuclei, followed by Thioflavin S staining (1% Thioflavin S diluted 1:5,000 in PBS; Sigma-Aldrich; T1892) for 2 min to label dense-core plaques. Imaging was performed using a ZEISS LSM 780 confocal microscope. Confocal z-stack images (20 μm) were acquired at 2-μm intervals using a ×40/1.3 oil objective. Image processing was performed using ZEN 2011 software (ZEISS). For VGluT2, Bassoon and C1Q quantification, 10-µm coronal brain sections from control, 5xFAD and 5xFAD;PU.1-low mice were blocked and permeabilized for 1 h at room temperature in 10% normal goat serum/0.1 M phosphate buffer containing 0.3% Triton X-100 (all from Sigma). Sections were then incubated with primary antibodies at room temperature overnight. The following primary monoclonal and polyclonal antibodies have been used: mouse monoclonal anti-Bassoon (Enzo Life Sciences; ADI-VAM-PS003-F; 1:500) and guinea pig polyclonal anti-VGLUT2 (Synaptic Systems; 135404; 1:2,000) to label presynaptic axonal terminals and C1Q (Abcam; ab182451; 1:500). The following day, sections were incubated with appropriate Alexa-fluorophore-conjugated secondary antibodies (Thermo Fisher Scientific), and β-amyloid plaques were visualized using 0.0005% Thioflavin S in 1× PBS (Sigma-Aldrich; T1892) to label dense-core plaques. After thorough washing, sections were mounted using VECTASHIELD without DAPI (Vector Laboratories). For Mx1–GFP+ microglia quantification, 30-μm slides from 5xFAD;Mx1–GFP (two females and two males) and 5xFAD;Mx1–GFP;PU.1-low mice (two females and one male) were washed with PBS, permeabilized with PBST and blocked with 2% normal goat serum in PBST. The slides were incubated with primary antibodies in 2% normal goat serum in PBST overnight at 4 °C. The primary antibodies included CD11B (1:500; eBioscience; 14-0112-82) to label microglia and turboGFP (1:5,000; Evrogen; AB513) to label Mx1–GFP+ interferon-responsive cells. The slides were then washed, and the signal for turboGFP was amplified for 10 min using tyramide signal amplification with SuperBoost (Invitrogen; B40922). The slides were washed and incubated with Thioflavin S for 2 min (1% Thioflavin S diluted 1:10,000 in PBS; Sigma-Aldrich; T1892) to label dense-core plaques and then with DRAQ5 for 20 min (1:2,000 in PBS) to label nuclei. For BODIPY imaging, the manufacturer's recommendations were followed. Imaging was performed using a ZEISS LSM 780 confocal microscope. Confocal z-stack images (20 μm) were acquired at 2-μm intervals using a ×40/1.3 oil objective. Image processing was performed using ZEN 2011 software (ZEISS). For most image analyses (with the exception of the analyses indicated below), single eight-tile z-stack images with a ×20 objective (×1 zoom) were analysed using Imaris software (Bitplane). First, a surface for plaques was generated. Next, microglia surfaces (processes) within 5 μm of the plaque surface were generated to select for plaque-associated microglia. We used this cutoff to select for microglia virtually touching the plaques. Finally, surfaces were generated for plaque-associated microglial nuclei. The distal microglia were counted similarly for those more than 5 μm from the plaque surface. The total microglia numbers were calculated similarly. For BODIPY quantification, a microglia surface was generated, and then a BODIPY+ surface was generated within microglia. For Bassoon and VGLUT2 density quantification, three to four stained sections from each sample containing the somatosensory cortex were imaged using a ZEISS Observer Spinning Disk Confocal microscope equipped with diode lasers (405 nm, 488 nm, 594 nm and 647 nm) and ZEN Blue acquisition software (ZEISS). Two to three randomly chosen ×63 fields of view within the somatosensory cortex were acquired with five z-stack steps at 0.68 µm spacing for each hemisphere. Identical settings were used to obtain images from all samples within one experiment, and data analyses were performed using ImageJ (National Institutes of Health, v.1.53k), as described previously, with some modifications72. In brief, a consistent threshold range was first determined using sample images for each genotype and condition. All images were subjected to background subtraction. Then, single z-planes of the z-stacks (five z-planes per animal) were subjected to the same background subtraction and thresholding. Data from single z-planes were averaged for each z-stack of each field of view, and the mean of all fields of view from one animal was determined and normalized to control animals to assess changes in synaptic densities. For inhibitor experiments, manual thresholding, blinded to condition and genotype for each channel within one experiment, was performed (ISODATA segmentation method; 85–255). For AT8 and HJ3.4–amyloid-β immunohistochemistry quantification, images were obtained from two to three sections per mouse. Slides were scanned on the NanoZoomer 2.0-HT system (Hamamatsu Photonics). The images were further processed using NDP viewing software (Hamamatsu Photonics) and Fiji software v.1.51 (National Institutes of Health). For 6E10–amyloid-β/ThioS ratio quantification from the hippocampus and subiculum, z-stack images of approximately 20-μm thickness acquired with a ×40 objective at ×1 zoom were used to quantify the area of the respective signals in maximum intensity Z projections in ImageJ. For PU.1 immunofluorescence quantification by CellProfiler (Figs. 1e and 3a), approximately 20 μm of mouse brain tissue stained for PU.1, CD11B, DRAQ5 and ThioS was captured by z-section confocal microscopy. The z sections were projected as summed slices and converted to 16-bit. Images stained for CD11B (microglia marker) were normalized to 0.35% pixel saturation to standardize microglia segmentation (outlining) in Fiji/ImageJ73. The z-projected images were then loaded into CellProfiler v.4.2.1 (ref. 74), in which microglia nuclei were identified, the nuclear outline was applied to the PU.1 channel and measurements of individual nuclei were recorded. CellProfiler classified nuclei as belonging to microglia using FilterObjects if they exceeded an upper quartile intensity threshold in the CD11B stain and passed spatial form factor and eccentricity thresholds. To further constrain microglia nuclei, CellProfiler was used to generate a properties file containing DRAQ5 nuclei objects and the associated CD11B and DRAQ5 measurements. The classifier feature in CellProfiler Analyst v.3.0.4 (ref. 75) was then trained using the properties file by visually selecting representative microglia and non-microglia nuclei, applying the training and saving a .model file. A separate CellProfiler pipeline was then created to incorporate this classifier model under ClassifyObjects. Nuclear outlines were reduced in size for consistency. Microglial PU.1 values, merged images containing outlines and mean intensity measurements to allow visual inspection were recorded. For PU.1 immunofluorescence quantification by Imaris (Fig. 1a,e), approximately 20 μm of mouse brain tissue from control, 5xFAD, 5xFAD;SYK-KO and 5xFAD–PLCγ2-KO, as well as PU.1-low and PU.1-high mice (as reference mice), stained for PU.1, IBA1, DRAQ5 and Thioflavin S, was captured using z-section confocal microscopy. The z sections were analysed using Imaris software (Bitplane). First, a surface for IBA1+ microglia was generated, and the DRAQ5 nuclei channel was masked within this surface. Next, microglia nuclei were specifically detected by creating a surface for the masked DRAQ5 channel. Low and high PU.1 cutoffs were defined on the basis of the fold change in mean PU.1 signal between nuclei from PU.1-high and PU.1-low microglia. The fold change (calculated as the mean PU.1 signal in PU.1-high nuclei divided by the mean PU.1 signal in PU.1-low nuclei) was determined to be 1.25. For each experiment, the high cutoff was set by multiplying and the low cutoff was set by dividing the mean PU.1 signal across all microglia of 6-month-old 5xFAD mice. Nuclei between these cutoffs were considered to have medium PU.1 signal. For CD28 quantification, approximately 20 μm of mouse brain tissue stained for CD28, IBA1, DRAQ5 and ThioS was captured by z-section confocal microscopy. The z sections were analysed using Imaris software (Bitplane). Microglia nuclei in each image were detected and counted using the Spots function. IBA1+CD28+ microglia were counted and normalized to the total number of microglia per image. In situ hybridization was performed on 30-μm cryosectioned slides from 5xFAD;CD28-KO mice with sex-matched and age-matched littermate control 5xFAD mice using RNAscope probes for Cd28 (Advanced Cell Diagnostics; 1070671-C1) and Mx1 (Advanced Cell Diagnostics; 474931-C1) to label interferon-responsive cells, with the RNAscope Multiplex Fluorescent Reagent Kit v.2 according to the manufacturer's recommendation for fixed-frozen tissue samples. Sections were incubated overnight at 4 °C with the primary antibody IBA1 (1:1,000; Synaptic Systems; 234 009) to label microglia. Sections were washed three times with PBST for 10 min and incubated with a secondary antibody (goat anti-chicken; 1:500; conjugated to the fluorophore Alexa Fluor 488) and DAPI (1:10,000) to label nuclei for 1 h at room temperature. Sections were washed twice in PBST and once in PBS, dried, mounted using ProLong Gold Antifade Mountant (P36930; Invitrogen) and dried overnight. Z-stack images were taken using a ZEISS LSM 780 confocal microscope with 30-μm z-stack confocal images acquired at 0.8-μm intervals and ×20 objective at ×1 zoom, and image processing was done using Fiji software. Cd28+ or Mx1+ microglia were manually identified by at least five dots (Cd28) or three dots (Mx1) of in situ signal within the microglia nuclei. The samples were then processed following the Vizgen Fresh Frozen Tissue Sample Preparation protocol. Briefly, brains were sagittally sectioned into 10-µm-thick slices and placed on a functionalized coverslip covered with fluorescent beads (Vizgen; 20400001). Once the tissue adhered to the coverslip, we fixed the tissue (4% PFA in 1× PBS) for 15 min at room temperature, followed by three washes with PBS. After aspiration, we added 70% ethanol to permeabilize the tissue for 24 h at 4 °C. The samples were photobleached for 3 h in 70% ethanol (MERSCOPE Photobleacher; Vizgen; 10100003), followed by incubation with a blocking solution (Vizgen Blocking Buffer C premix; 20300100; and 10% RNase inhibitor; New England Biolabs; M0314L) for 1 h at room temperature. Then, the samples were incubated with a primary antibody solution (purified azide-free anti-β-amyloid 6E10; 803004; BioLegend; and 1:100 plus 10:100 of RNase inhibitor; New England Biolabs; M0314L). The samples were washed three times with 1× PBS and incubated with the secondary antibody solution (Vizgen; Anti-Mouse Aux 4 20300101; and 1:100 plus 10:100 of RNase inhibitor; New England Biolabs; M0314L) for 1 h at room temperature. The samples were washed three times with 1× PBS, fixed (4% PFA in 1× PBS) for 15 min at room temperature and washed two times with 1× PBS. Following staining, the samples were incubated in formamide wash buffer (30% formamide in 2× saline–sodium citrate (SSC) buffer), and the MERFISH library mix was added and left for hybridization for 48 h. We then washed the samples twice and incubated them at 47 °C with formamide wash buffer. Then, the tissue was embedded in a polyacrylamide gel, followed by incubation overnight at 37 °C using a tissue clearing solution (2× SSC, 2% SDS, 0.5% v/v Triton X-100 and 1:100 proteinase K). The tissue was washed (2× SSC) and incubated with DAPI and PolyT solution (Vizgen) for 15 min at room temperature. The MERSCOPE 500 gene imaging kit (Vizgen; 10400003) was activated by adding 250 μl of Imaging Buffer Activator (Vizgen; 203000015) and 100 μl of RNase Inhibitor (New England Biolabs; M0314L). Mineral oil (15 ml) was overlaid on top of the imaging buffer through the activation port. A ×10 low-resolution mosaic of the sample was acquired. Following image acquisition, the resulting data were decoded using Vizgen's analysis pipeline incorporated in MERSCOPE. Segmentation was performed on the middle z plane (fourth of seven), and cell borders were propagated to z planes above and below. The following gene sets have been used to identify cell types in the representative images: macrophage-identifying gene (Mrc1), endothelial cell-identifying gene (Cldn5), T cell-identifying genes (Cd3e and Cd8a), microglia homeostatic genes (Cx3cr1, Tmem119, P2ry12, Csf1r, Hexb, Aif1, Tnf, Trem2, Itgam, C1qa, C1qb, C1qc, Sall1 and Selplg), neuron-identifying gene (Meg3), oligodendrocyte-identifying genes (Mog, Sox10 and Mag), lymphoid genes (Pdcd1, Cd274, Cd28, Cd72, Tnfsf13b and Cd4) and DAM-identifying genes (Itgax, Clec7a, Igf1, Cst7 and Spp1) (Extended Data Fig. Single-cell gene expression matrices were obtained by counting messenger RNA (mRNA) molecules within segmented cell boundaries and were further analysed in RStudio using R 4.2.2, Seurat 5.0.3 and custom-made scripts (https://github.com/SchaferLabUMassChan/Ayata-et-al_2025). We excluded cells containing fewer than 40 transcripts or a volume of less than 100 µm3. To account for global differences in mRNA counts between samples, we normalized data to the total transcripts per cell of each sample. We then normalized the gene expression of each cell by its volume to compute counts per cubic micrometre. Cells with less than 11 unique genes were further excluded. For annotation of cell types and nearest-neighbour analysis relative to plaques, data from all samples were merged into a single Seurat object for clustering and cell type annotation following a modified Seurat pipeline. Data were normalized by dividing gene counts per cubic micrometre for each cell by the total count per cubic micrometre for that cell, multiplied by 10,000 and log-transformed. Data were then scaled, and the principal components were calculated on all 398 measured genes. Thirty-nine principal components were used to calculate the UMAP embedding and perform clustering analysis using the Louvain algorithm with a resolution of 2.4. Clusters composed of these ‘hybrid' cells and clusters consisting of imaging artefacts were removed from the analysis. UMAP embedding and clustering analysis were iteratively repeated until all hybrid clusters were removed. Microglia sub-cluster analysis was performed using the same modified Seurat pipeline. Before the removal of clusters containing hybrids, all clusters expressing microglia cell type markers (Tmem119, Hexb, P2ry12, Trem2, C1qa and Cx3cr1) were separated out and sub-clustered. Sub-clusters composed of hybrid cells and sub-clusters composed of border-associated macrophages (Mrc1, Lyve1 and Cd163) were removed from the analysis. Seventeen principal components were used to calculate the UMAP embedding and perform clustering analysis using the Louvain algorithm with a resolution of 0.7. The X and Y coordinates of all amyloid-β plaques in the cortex of two 5xFAD MERFISH samples were manually recorded using MERSCOPE visualizer software (Vizgen) to visualize anti-β-amyloid 6E10 immunofluorescent signal. For each plaque, because plaques varied in size, 4 points along the perimeter were recorded. The X and Y coordinates outlining the cortex were acquired by manual tracing in the MERSCOPE visualizer software (Vizgen) using DAPI signal and Meg3, Cux2 and Rorb transcripts as landmarks. For all microglia within the cortex, the distance between each microglia and the nearest plaque was calculated using a custom Python script (find_nearest_neighbors2.py). Microglial nuclei within 15 µm of a plaque were defined as ‘plaque-associated', whereas those further than 15 µm from the nearest plaque were defined as ‘distal'. We used a larger distance cutoff for nuclei to account for the soma and processes. The MERFISH analysis plots were generated using Seurat 5.0.3 and scCustomize 3.0.1 R packages. MERFISH-processed data have been deposited in Gene Expression Omnibus (GEO) and are publicly available as of the date of publication. Processed MERFISH.vzg files for browsing on MERSCOPE visualizer software (Vizgen) and the MERFISH raw output files are available upon request. Any further information required to reanalyse the data reported in this paper is available upon request. For microglia phospho-protein analysis (Extended Data Fig. 6a), PU.1-low, wild-type and PU.1-high mice were euthanized by cervical dislocation, and brain regions were immediately removed. Frozen tissue was mechanically dissociated in glass homogenizers in Hank's balanced salt solution (HBSS) supplemented with protease and RNase inhibitors. The homogenate was filtered through a 70-μm mesh filter. Myelin removal was performed using Percoll (pH 7.4) density gradient separation. The resulting homogenate in a 21% Percoll gradient was centrifuged at 500g for 15 min at 4 °C. The pellet was washed and resuspended in HBSS. Microglia were gated as live cells expressing YFP+ in Cx3cr1CreErt2/+(Litt) mice (Supplementary Fig. Roughly 120,000 cells were used per sample. For ATAC-seq, wild-type, PU.1-low and PU.1-high mice were anaesthetized with an intraperitoneal injection of ketamine/xylazine, transcardially perfused with 12 ml of cold HBSS. The brains were collected, and the cerebellum and olfactory bulb were removed. Myelin removal was performed using Percoll (pH 7.4; GE-17-0891-02) density gradient separation in 30% Percoll centrifuged at 500g for 15 min at 4 °C. The pellet was washed and resuspended in magnetic-activated cell sorting buffer (2 mM EDTA and 0.5% bovine serum albumin in 1× PBS) with DAPI and sorted on the FACS Aria II (BD Biosciences). Microglia were gated as live cells expressing YFP+ (Supplementary Fig. Roughly 50,000 cells were used per sample. Fresh tissue was mechanically dissociated in glass homogenizers in HBSS and then supplemented with fluorescently conjugated antibodies (rat anti-mouse APC-CD11B; eBioscience; 17-0112-82; 1:100; and rat anti-mouse PerCP-Cy5.5-CD45; Invitrogen; 45-0451-82; 1:100) and incubated for 30 min on ice. Cells were washed twice (300g centrifugation for 5 min at 4 °C) and resuspended in FACS buffer (PBS and 2% bovine serum albumin). Cells were sorted on the FACS Aria II (BD Biosciences). Microglia were gated as live cells expressing CD11B+ and Cd45low (Supplementary Fig. 9b) cells were used per well for CD28 western blotting. For ATAC-seq, wild-type mice were anaesthetized with an intraperitoneal injection of ketamine/xylazine and transcardially perfused with 12 ml of cold HBSS. Spleens were collected and homogenized on ice using a 70-µm strainer. Red blood cells were lysed with RBC lysis buffer (eBioscience; 00-4333-57) for 5 min at room temperature, followed by a PBS wash and centrifugation. Cells were resuspended in magnetic-activated cell sorting buffer, incubated with Fc blocker (Bio-Rad; BUF041B; 1:100) on ice for 10 min, stained with Thy1.2 (eBioscience; 17-0902-82; 1:100) and DAPI and sorted on the FACS Aria II (BD Biosciences). For western blot analysis, cells were snap-frozen and prepared, as described below. 5a) cells were used for CD28 western blotting. For protein preparation from tissue, 6-month-old PU.1-low (three females and two males), control (two females and three males) and PU.1-high (three females) mice, as well as 6-month-old control (six males), 5xFAD (four males) and 5xFAD–PU.1-low mice (three males) were killed, and brain tissue was dissected and frozen immediately. The frozen brain tissue (cortex or hippocampus) was resuspended in 1% SDS (Ambion) supplemented with protease inhibitor (Roche), sonicated at 4 °C using a Bioruptor on high-power 30-s on/30-s off cycles for a total of ten cycles and boiled for 10 min in 1× lithium dodecyl sulfate sample buffer (Invitrogen) supplemented with dithiothreitol (DTT) to a final concentration of 200 mM (Sigma). The supernatant was aspirated, and the cell pellet was snap frozen in liquid nitrogen. The BV2 cells were detached from the culture plate using PBS + 2 mM EDTA for 5 min at room temperature, whereas the primary microglia required further scraping. Protein was lysed in cold RIPA buffer (Thermo Fisher Scientific; 89900) with protease inhibitors (Roche; 11873580001), phosphatase inhibitors (Roche; 4906845001) and benzonase (MilliporeSigma; 71206) for 1 h on ice. The supernatant was moved to a fresh tube. The protein concentration was determined using a bicinchoninic acid protein assay kit (Thermo Fisher Scientific) according to the manufacturer's instructions. Protein samples were diluted in equal volumes of 2× lithium dodecyl sulfate sample buffer (Invitrogen) and supplemented with DTT to a final concentration of 200 mM (Sigma). Protein samples (20–40 μg) or concentrated culture supernatant (100 μg) was separated on 4–12% Bolt Bis-Tris precast denaturing gels or 10% NuPAGE Bis-Tris precast denaturing gels (Invitrogen) and transferred onto polyvinylidene difluoride membranes. For CD28 and PU.1 immunoblotting of primary microglia, NuPAGE Antioxidant (Invitrogen) was added to the running buffer. The membranes were probed with primary antibodies diluted in 5% bovine serum albumin (BSA)– or milk–TBST solution overnight at 4 °C. The membranes were then washed and probed with horseradish peroxidase-conjugated anti-mouse (GE; 1:10,000), anti-rabbit IgG secondary antibody (GE 1:10,000) or anti-sheep IgG antibody (Jackson ImmunoResearch; 1:10,000) for 1 h at room temperature. The membranes were developed using an enhanced chemiluminescence substrate (PerkinElmer) and exposed on film. Exposed films were scanned, and protein bands were quantified using ImageJ software (National Institutes of Health). Protein quantities were normalized using β-actin or histone H3. Full scans of western blots (uncropped) are provided in Supplementary Figs. Sample preparation and liquid chromatography–tandem mass spectrometry measurements were performed, as previously described76. In brief, BV2 cells were lysed in sodium deoxycholate buffer (1% sodium deoxycholate, 10 mM tris(2-carboxy (ethyl)phosphine), 40 mM 2-chloroacetamide and 100 mM Tris–HCl (pH 8.5)), heated at 95 °C for 10 min and sonicated to shear DNA. Digests were desalted using in-house-made SDB-RPS StageTips. Concentrations were determined using a NanoDrop spectrophotometer and normalized between samples for equal peptide injection. Peptide mixtures were analysed using an EASY-nLC 1000 chromatographic system coupled to an Orbitrap Exploris 480 (Thermo Fisher Scientific). Peptides were separated by 120-min chromatographic gradients using a binary buffer system comprising buffer A (0.1% formic acid in liquid chromatography–mass spectrometry-grade water) and buffer B (80% acetonitrile and 0.1% formic acid in liquid chromatography–mass spectrometry-grade water) on a 50-cm in-house-made column (75-µm inner diameter) packed with 1.9-µm ReproSil C18 beads (Dr. Maisch GmbH) at a flow rate of 300 nl min−1 and 60 °C maintained by an in-house-made column oven. Data-independent acquisition mass spectrometry raw files were processed using DIA-NN77 (v.1.8) with FASTA digest for library-free search and deep learning-based spectra, retention time and ion mobility prediction enabled. Precursor false discovery rate was set to 1%, and default parameters were used with the following changes: the precursor range was restricted to 300–1,650 m/z and the fragment ion range to 200–1,650 m/z. Match between runs was enabled, and the neural network classifier was set to ‘double-pass mode' and the quantification strategy to ‘robust liquid chromatography (high accuracy)'. Spectra were matched against the UniProt FASTA database for mouse (December 2022 release). Exported peptide data were filtered for a false discovery rate of less than 0.01 (‘Lib.Q.Value' and ‘Lib.PG.Q.Value') and proteotypic peptides before applying a sample-wise median normalization on log10 normalized precursor quantities and recovering the original value range. Protein intensities were normalized using the MaxLFQ78 algorithm using an in-house script, with a minimum of more than one peptide. Bioinformatics analyses were performed using Perseus79 (v.1.6.15.0) and R (v.4.1.2). Before performing statistical analysis, quantified proteins were filtered for at least three valid values in at least one group of replicates. Primary microglia (250,000) were lysed in TRIzol (Life Technologies) for 5 min, followed by phase separation with chloroform. After extraction, RNA was precipitated for 30 min at −20 °C in isopropanol with 0.15 M sodium acetate and GlycoBlue (Life Technologies). The pellet was then washed three times with 70% ethanol, air-dried and suspended in nuclease-free water. cDNA was prepared from total RNA using the High-Capacity RNA-to-cDNA Kit (Applied Biosystems). Relative gene expression of the cDNA was assayed by qPCR using recommended TaqMan gene expression assays (Life Technologies) and following the manufacturer's recommendations (Cd28, Ifnb, Mx1, Ifitm3, Gapdh and Actb). Cycle counts for mRNA quantification were normalized to Gapdh or Actb. Relative expression (ΔCT) and quantification (RQ = 2−ΔΔCT) for each gene were calculated. RNA was extracted using the RNeasy Plus Mini Kit (QIAGEN; 74136) following the manufacturer's instructions. Libraries were prepared using the NEBNext Ultra II RNA Library Prep Kit following the manufacturer's instructions. Multiplexed libraries were directly loaded on NovaSeq X Plus (Illumina) with paired-end sequencing. Raw sequencing data were processed using Illumina bcl2fastq2 Conversion Software v.2.17. Raw sequencing reads were first quality checked and trimmed using Trim Galore (https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/ v.0.6.4, a wrapper program implementing Cutadapt v.2.9 (https://journal.embnet.org/index.php/embnetjournal/article/view/200) and FastQC v.0.11.9 (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and mapped to the human genome (hg38) using the HISAT2 package (v.2.2.0) (ref. 80). Reads were counted using featureCounts (v.2.0.0) (ref. The raw counts were processed through a VST procedure using the DESeq2 package82 to obtain transformed values that were more suitable than the raw read counts for certain data mining tasks. A pairwise comparison was performed on the count data of entire gene transcripts using the DESeq2 package (v.1.36.0) (ref. 82). Heat maps were made using R (v.3.1.1; https://www.R-project.org), in which the expression of each gene in VST was normalized to the mean across all samples (z-scored). ATAC-seq libraries were prepared using the Omni-ATAC protocol83,84,85. In brief, nuclei were isolated from 50,000 isolated YFP+ ex vivo microglia or THY1.2+ T cells. Transposition was performed using Tn5 transposase (Illumina) at 37 °C for 30 min, followed by DNA purification using the ZYMO DNA Clean & Concentrator Kit. Libraries were pre-amplified for five cycles using NEBNext High-Fidelity 2X PCR Master Mix. Further amplification cycles were determined through qPCR. Double-sided size selection was performed using AMPure XP beads. Libraries were quantified through Qubit and analysed using Bioanalyzer or TapeStation. Final libraries were pooled equimolarly and sequenced on an Illumina NovaSeq (2 × 150 bp), targeting approximately 40 million reads per sample. Read quality was assessed using FastQC (v.0.11.8)86. Reads were trimmed for adaptor sequences using Trim Galore (v.0.6.6)87 and aligned to the mouse reference genome (GRCm38; GENCODE M25) using Bowtie 2 (v.2.2.8) with a maximum fragment length of 2,000 bp (−X 2000) with default sensitivity settings88. Reads aligned to mitochondrial DNA and non-canonical chromosomes were removed. Picard (v.2.2.4) was used to remove duplicated reads. Post-filtering Binary Alignment Map files for all samples were merged using the merge function from Samtools (v.1.11), followed by peak calling using MACS2 (v.2.1.0) with parameters --nomodel, --nolambda, --keep-dup all and --slocal 10000, optimized for paired data (−f BAMPE) using the mouse genome (−g mm)89. Blacklisted regions were removed before downstream analysis. Peak summits were extended ±200 bp, and quantification of reads in peaks was performed using featureCounts from Subread (v.2.0.1). Peaks that fell below the top 67% of peaks across all samples were filtered out as low-signal regions. Coverage tracks were generated from Binary Alignment Map files using deepTools (v.3.2.1) bamCoverage with parameters –normalizeUsingRPKM –binsize 10 (ref. A peak was considered differentially accessible if the Benjamini–Hochberg adjusted P value < 0.05, and the absolute log2 fold change was greater/lower than 1. Heat maps of the union of all differentially accessible regions across multiple comparisons were generated using heatmaply (v.1.2.1), displaying DESeq2 VST-normalized ATAC read counts. Read counts were z-scored by row, and hierarchical clustering was applied to rows only. Heat maps and profile plots of peak signals were generated using deepTools (v.3.2.1) in reference-point mode to visualize peak signals across differentially accessible regions using RPKM-normalized bigWig files as input90. For peak-to-gene associations, annotated peaks were obtained using ChIPseeker (v.1.30.3). Peaks were first converted into a GRanges object using the makeGRangesFromDataFrame() function from GenomicRanges (v.1.46.1), retaining all relevant metadata (keep.extra.columns=TRUE). Peak annotation was performed with the annotatePeak() function using a TxDb object derived from the mouse reference genome (GRCm38; GENCODE M25). Peaks were mapped to gene features at the ‘gene' annotation level, with the transcription start site region defined as ±1,000 bp. Principal component analysis (PCA) variance-stabilized counts were generated using DESeq2 (v.1.34.0) and used as input for PCA. For each analysis, the top 5,000 most variable peaks were selected on the basis of row variance using matrixStats (v.0.62.0.9003). PCA was performed using prcomp in R (v.4.1.0). For visualization of microglia-only samples, PCA plots were generated using Plotly (v.4.9.4.1). To visualize both microglia and T cells in a combined PCA space, the first two principal components (PC1 and PC2) were plotted using ggplot2 (v.3.3.5) with fixed axis scaling to preserve relative distances. Transcription factor motif enrichment was done using HOMER (v.4.10). HOMER de novo motifs are shown. Crossing these mice to microglia-specific Cre mice allows for the isolation of ribosome-associated mRNAs in a cell-type-specific fashion using eGFP-based immunoaffinity purification28,91,93. Ribosome-associated mRNA from microglia was isolated from each region, as previously described92, in which each sample corresponds to a single mouse. Briefly, the brain tissue was thawed in an ice-cold Wheaton 33 low extractable borosilicate glass homogenizer containing 1-ml cell lysis buffer (20 mM HEPES–KOH (pH 7.3), 150 mM KCl, 10 mM MgCl2, 1% NP-40, 0.5 mM DTT, 100 μg ml−1 of cycloheximide and 10 μl ml−1 of RNasin (Promega) and SUPERase·In (Invitrogen). The samples were first manually homogenized using a polytetrafluoroethylene homogenizer (grinding chamber clearance of 0.1–0.15 mm) with three to five gentle strokes, followed by homogenization in a motor-driven overhead stirrer at 900 rpm with 12 complete strokes. The samples were then transferred to chilled Eppendorf tubes. NP-40 (final concentration of 1%) and DHPC (final concentration of 30 mM) were added to the supernatant. Dynabeads MyOne Streptavidin T1 (200 μl; Invitrogen), conjugated to 1 μg μl−1 of biotinylated Protein L (Pierce) and 50 μg each of anti-eGFP antibodies Htz–GFP–19F7 and Htz–GFP–19C8 (bioreactor supernatant from the Memorial Sloan Kettering Cancer Center Monoclonal Antibody Facility) was added to each supernatant. The unbound fraction was collected using a magnetic stand after overnight incubation at 4 °C with gentle end-over-end rotation. RNA clean-up from isolated microglial cells (50,000), TRAP samples and 5% of the unbound fractions from TRAP samples was performed using RNeasy Mini Kit (QIAGEN) following the manufacturer's instructions. RNA integrity was assayed using an RNA Pico chip on a Bioanalyzer 2100 (Agilent), and only samples with RNA integrity number greater than 9 were considered for subsequent analysis. Double-stranded cDNA was generated from 1–5 ng of RNA using the Ovation V2 kit (NuGEN) following the manufacturer's instructions. Libraries for sequencing were prepared using the Nextera XT Kit (Illumina) following the manufacturer's instructions. Multiplexed libraries were directly loaded on NovaSeq (Illumina) with single-read sequencing for 75 cycles. Raw sequencing data were processed using Illumina bcl2fastq2 Conversion Software v.2.17. Raw sequencing reads were first quality checked and trimmed using Trim Galore (https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/ v.0.6.4, a wrapper program implementing Cutadapt v.2.9 https://journal.embnet.org/index.php/embnetjournal/article/view/200 and FastQC v.0.11.9 https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and then mapped to a mouse genome (mm10) using the HISAT2 package (v.2.2.0)80. Reads were counted using featureCounts (v.2.0.0)81 against the Ensembl 99 annotation. The raw counts were processed through a VST procedure using the DESeq2 package82 to obtain transformed values that were more suitable than the raw read counts for certain data mining tasks. PCA was performed on the top 500 most variable genes across all samples on the basis of the VST data to assess for visual outliers. All pairwise comparisons were performed on the count data of entire gene transcripts using the DESeq2 package (v.1.36.0)82. For comparisons of baseline samples with 5xFAD samples, a cutoff of the adjusted P value < 0.1 and mean expression greater than 10 (DESeq2) was applied. For comparisons involving drug treatments, genetic manipulations and ageing, a cutoff of uncorrected P value < 0.05 and mean expression greater than 10 was applied (DESeq2). Additionally, TRAP-enriched genes were calculated using a cutoff of P value < 0.05 and fold change greater than 2 over their respective unbound fraction. All heat maps and scatter plots for bulk sequencing were made using R (v.3.1.1; https://www.R-project.org). For all heat maps, the expression of each gene in VST was normalized to the mean across all samples (z-scored). Selected and significantly enriched (P value < 0.05 with Benjamini–Hochberg correction) Gene Ontology terms for biological processes are represented in balloon graphs and were made using R (v.3.1.1; https://www.R-project.org). Pie charts for functional characterization of genes were built manually on the basis of Gene Ontology term enrichment analyses and a literature search. Gene expression data were further analysed using Gene Set Enrichment Analysis (GSEA 4.2.3 software; number of permutations = 1,000) using the DAM gene set from Keren-Shaul et al.12 (Supplementary Table 3). Microglial nuclei were isolated from microglia-specific TRAP mice bred to Eef1a1LSL.eGFPL10a/+ mice60 on the basis of the eGFP–L10a fluorescence of newly formed ribosomes in the microglia nucleoli, as described28. In brief, 6-month-old Cx3cr1CreErt2/+(Litt);5xFAD;Eef1a1LSL.eGFPL10a/+ mice (one female on control diet or two males on PLX5622 diet) or Cx3cr1CreErt2/+(Litt);5xFAD;Eef1a1LSL.eGFPL10a/+ 5xFAD (female), Cx3cr1CreErt2/+(Litt);5xFAD;Spi1fl/+;Eef1a1LSL.eGFPL10a/+ 5xFAD–PU.1-low mice (male) and Cx3cr1CreErt2/+(Litt);5xFAD; R26FLAG-Spi1/+;Eef1a1LSL.eGFPL10a/+ 5xFAD–PU.1-high mice (male) were euthanized with CO2. Brain regions were quickly dissected and frozen immediately. The frozen cortex tissue was homogenized in 0.25 M sucrose, 150 mM KCl, 5 mM MgCl2 and 20 mM tricine (pH 7.8), supplemented with protease and RNase inhibitors with a glass Dounce homogenizer (1984-10002; Kimble Chase). The buffers were supplemented with 10 μl ml−1 of RNasin, SUPERase·In and EDTA-free protease inhibitor cocktail (11836170001; Roche). The homogenate was then spun through a 29% iodixanol cushion. The resulting nuclear pellet was resuspended in 0.25 M sucrose, 150 mM KCl, 5 mM MgCl2 and 20 mM tricine (pH 7.8), supplemented with 10 μM DyeCycle Ruby (V10304; Invitrogen) and 10% donkey serum (017-000-121; Jackson ImmunoResearch). Microglial nuclei were sorted in a BD FACSAria Cell Sorter by gating for the lowest DyeCycle Ruby, which indicates nuclei singlets and a high GFP signal. For single-nucleus RNA-seq, isolated nuclei were used immediately. More than 10,000 nuclei were used for 10× sequencing. We used the Chromium platform (10x Genomics) with the 3′ gene expression v.2 kit with a targeted input of 5,000 nuclei or cells per sample on average. Barcoded cDNA was extracted from GEMs using Post GEM-RT Cleanup and amplified for 12 cycles. The amplified cDNA was fragmented and subjected to end-repair, poly-A-tailing, adaptor ligation and 10x-specific sample indexing following the manufacturer's protocol. Libraries were quantified using Bioanalyzer (Agilent) and Qubit (Thermo Fisher Scientific) analyses and then sequenced in paired-end mode on a HiSeq 2500 instrument (Illumina) targeting a depth of 50,000–100,000 reads per nucleus per cell. The raw read data were demultiplexed, aligned and analysed using 10x Cell Ranger (v.2.1.0). To capture the unspliced pre-mRNA in the single-nucleus RNA expression assay, intronic regions in the 10x Cell Ranger mm10 v.1.2.0 reference were marked as exonic, as suggested by 10x for pre-mRNA reference generation. Data from two experiments were used to generate Seurat objects in R. Quality control cutoffs (nFeature of more than 1,500; pct.mt of less than 5) were used to generate final datasets. The following clustering pipeline steps were applied: NormalizeData(), FindVariableFeatures(), ScaleData(), RunPCA(), FindNeighbors() and FindClusters(). Clusters that do not express Hexb (highly expressed pan-microglia gene marker) were excluded. Cluster quality was confirmed by VlnPlot() for nFeature, nCount and pct.mt (percentage of mitochondrial DNA). Three datasets were generated from single-cell RNA-seq: control versus 5xFAD (8-month-old mice; Figs. 1), time course of control and 5xFAD (Fig. 1a, 2a and 5a,d,e, Extended Data Figs. Cluster marker genes were identified using FindAllMarkers() with logfc.threshold = 0.25 and min.pct = 0.0. Sub-clustering of DAM microglia in Fig. 1 was performed after unbiased cell clustering and manual annotation of cluster identification on the basis of genes (Cst7, Apoe, Lpl and Itgax) driving the clustering. For the sub-clustering of DAM microglia, the DAM cluster was isolated and reclustered into three sub-clusters following the clustering pipeline steps: ScaleData(), FindVariableFeatures() with selection.method = ‘vst', nFeature = 3,000, RunPCA() with npcs = 30, RunUMAP() with reduction = ‘pca' and dims = 1:20, FindNeighbors() with reduction = ‘pca' and dims = 1:20 and FindClusters() with resolution = 0.4. Further differential analysis was performed using the Seurat package in R. Differential expression between the Spi1low and Spi1high (PU.1low and PU.1high) clusters was performed using FindMarkers(), with a log2fc cutoff of 0.25, with min.pct = 0.01. Differential expression between the homeostatic and DAM clusters was performed using the ‘FindMarkers' function, with a log2fc cutoff of 0.25 and min.pct = 0.01. For pseudobulk analysis comparing genotypes in Fig. 11e, Seurat identities were set to ‘genotype' using the feature Idents(). 2), data were obtained from 6-month-old 5xFAD mice fed with control diet (7,286 nuclei) and 6-month-old 5xFAD mice fed with CSF1Ri diet (3,995 nuclei). Cluster marker genes were identified using FindAllMarkers() with logfc.threshold = 0.25 and min.pct = 0.0. 2) was performed in the same fashion as single-cell sequencing. In brief, the DAM cluster was isolated and reclustered into three sub-clusters by applying the following clustering pipeline steps: ScaleData(), FindVariableFeatures() with selection.method = ‘vst', nFeature = 3,000, RunPCA() with npcs = 30, RunUMAP() with reduction = ‘pca' and dims = 1:20, FindNeighbors() with reduction = ‘pca' and dims = 1:20 and FindClusters() with resolution = 0.3. Differential expression between all microglia from the control and CSF1Ri-treated mice (Extended Data Fig. Seurat identities were set to ‘genotype' using the feature Idents(). For single-nucleus sequencing (Extended Data Fig. Cluster marker genes were identified using FindAllMarkers() with logfc.threshold = 0.2 and min.pct = 0.05. To perform differential expression analysis of PU.1low and PU.1high microglia in independent experiments, the filtered UMI count matrices from two independent experiments, each containing one control and one 5xFAD sample, were loaded into R (v.4.3.2)98 using SingleCellExperiment99 package (v.1.24.0) and Seurat (v.5.0.1)100. Low-quality cells and presumed doublets were removed using the R package Scatter101 (v.1.30.1). Non-microglia cells were identified by Seurat clustering after SCTransform102,103 on the basis of the expression of known microglia signature genes Hexb and Fcer1g. For each of the 5xFAD samples, one PU.1high cluster and one PU.1low cluster of cells were identified by Seurat clustering on the basis of known marker genes (PU.1high: Spi1, Il7r, Csf1r, Cd33, Tlr4, Il1a and C3ar1; PU.1low: Cst7, Cd274, Igf1, Apoe, Spp1, Csf1, and Pdcd1). The differential expression analyses between PU.1high and PU.1low clusters were conducted with the function ‘findMarkers()' of the Seurat package with default parameters, except min.pct = 0.01, on ‘data' layer of ‘RNA' assay after ‘NormalizeData()' transformation. The log2 fold changes of overlapping differentially expressed genes (DEGs) were compared between the 5xFAD samples in experiments 1 and 2 in a scatter plot highlighting key genes related to lymphoid or myeloid lineage or inflammation. This increased expression is seen randomly across all microglial clusters and is likely to reflect mRNA expression downstream of the deleted exons 2 and 3 in the CD28-KO. The ablation of Cd28 RNA and CD28 protein in these mice was confirmed using in situ hybridization (Extended Data Fig. Instead, upon removing one allele of the Spi1 gene, the non-microglia-expressed, co-regulated and adjacent Mybpc3 gene24 was upregulated, which serves as a positive indicator of Spi1 downregulation. Changes in PU.1 protein levels in PU.1-low and PU.1-high mice have been validated by western blotting (Extended Data Fig. For all experiments, plots were generated using the DimPlot(), FeaturePlot(), VlnPlot() and DotPlot() functions. Volcano plots were made using R. For the analysis of human microglia scRNA-seq15 (shown in Fig. The gene count matrix and corresponding cell barcodes for human ex vivo microglia were processed using a similar approach, as described by the authors. Briefly, UMI counts and cell annotations were imported into Seurat v.5.0.1 for downstream analysis. Data normalization and scaling were performed using the Seurat default parameters and included regression of UMI counts to account for technical noise. Subsequently, variable features were identified, and clustering was performed using the default Seurat settings, with a final resolution of 0.6 to determine cluster identities. Gene set module scores were computed per cell using the AddModuleScore function in Seurat to assess the relative expression of predefined gene sets, DAM, homeostatic and lymphoid genes (Supplementary Fig. Clusters were categorized into two main groups (homeostatic and DAM) on the basis of the average ratio of DAM to homeostatic module scores. DAM clusters were categorized as either lymphoid-positive or lymphoid-negative, depending on the levels of lymphoid-related gene expression. Finally, gene expression patterns and functional module scores across clusters were visualized using dot plots and UMAP projections to identify and compare gene signatures associated with different microglial clusters. New clusters were annotated on the basis of marker gene expression profiles and compared to the original Olah et al.15 cluster annotations. A Sankey plot was generated to visualize the redistribution of cells from original clusters to new clusters using the scmap R package. A confusion matrix was constructed to quantify the percentage overlap between the two clustering schemes. For each original cluster, the percentage of its cells found in each new cluster was calculated and plotted using a heat map, with a blue colour gradient indicating increasing overlap. The analysis of human microglia scRNA-seq data17 (shown in Fig. 5f,g) is, on the whole, based on data obtained from the AD Knowledge Portal (https://adknowledgeportal.org)104. Single-nucleus RNA-seq counts and cell-level metadata were downloaded from Synapse syn31512863 (ref. 17). All downstream processing and statistical tests were performed using R statistical software v.4.3.1 (ref. 98). To generate pan-microglial pseudobulks, the authors' cell annotations were used to filter microglia. Because most of the samples from the same donor were sequenced in two replicate batches, we further subsetted the data to include only those donors and batches and generated pseudobulks by summing read counts for all cells for each donor. Samples with at least ten cells were retained for further analysis (n = 428). Similarly, to generate cluster-specific pseudobulks, the same subset of donors and batches was included. Counts were summed for all cells in a cluster for each donor, and we retained samples with at least ten cells per cluster (n = 1,963). For visualization, counts were CPM normalized with edgeR105 and batch corrected with the regressBatches function from the batchelor package106. Heat maps were generated with the pheatmap package107. SPI1–AD risk variant differential expression and DAM abundance analysis were done after donor genotypes were downloaded from the AD Knowledge Portal104. Pseudobulks were annotated by matching on the ‘projid' included in the cell metadata and searching for rs1057233 genotype information108. On inspection of the data, we noted that rs1057233 and SPI1 expression showed a dominant inheritance pattern, in which donors with two copies of the risk-associated A allele showed slightly increased SPI1 expression compared with heterozygous carriers and non-carriers. Differential expression and DAM abundance analyses were performed on samples with diagnoses (cogdx) of either not cognitively impaired or AD and for which SPI1 variant status was known (n = 302 for differential expression and n = 306 for DAM abundance). For differential expression on the basis of carrier status, limma-voom109 was run on donor-level pseudobulks with batch as a blocking factor and AD status as a grouping variable. For the DAM abundance analysis, the proportion of microglia in the Mic 13 cluster was calculated for each donor, and a two-way ANOVA was performed to analyse the effect of cognitive diagnosis for AD and SPI1 carrier status on the proportion of DAM microglia. This was followed by Tukey's post hoc test, with P < 0.05 considered significant. For the bioinformatics analysis of Kimura et al.49, mouse microglia single-nucleus sequencing data were obtained on GEO with accession number GSE267764 and converted to Seurat objects. Quality control cutoffs (nFeature greater than 1,500; pct.mt less than 5) were used to generate final datasets. The following clustering pipeline steps were then applied: NormalizeData(), FindVariableFeatures(), ScaleData(), RunPCA(), FindNeighbors() and FindClusters(). Clusters that do not express Hexb (highly expressed pan-microglia gene marker) were excluded. Cluster quality was confirmed by VlnPlot() for nFeature, nCount and pct.mt (percentage of mitochondrial DNA). Nineteen clusters were generated; cluster-identifying markers were generated using the FindMarkers() with logfc.threshold = 0.25 and min.pct = 0. Sex-matched control, 5xFAD and 5xFAD;PU.1-low mice (9–10 months old) were deeply anaesthetized with isoflurane and subsequently decapitated. The brains were embedded in 2% agarose, and coronal slices (400 µm thick) were made using a Compresstome (Precisionary Instruments). Slices were allowed to recover for another hour in the recording chamber before recording the baseline. Field excitatory postsynaptic potentials were evoked by square-wave 50-µs pulses delivered every 30 s by MultiClamp 7b (Molecular Devices), and current was set to 50% of the maximum field excitatory postsynaptic potential amplitude using ISO-Flex (ampi). A recording electrode, filled with ACSF (2–4 MΩ), was placed in the stratum radiatum a few millimetres away from the stimulating electrode. LTP was induced using a theta-burst protocol: four trains, separated by 10 s, each comprised five bursts of four stimuli delivered at 100 Hz, with 200 ms between the bursts. Data were acquired using a MultiClamp 7B amplifier (Molecular Devices), filtered at 3 kHz, digitized at 10 kHz and analysed using pCLAMP 11 software (Molecular Devices). LTP was quantified as a percentage change in excitatory postsynaptic potential slope from baseline (the last 10 min of a 30-min baseline period). Behavioural analyses used sex-matched control, 5xFAD and 5xFAD;PU.1-low mice (9–10 months old). All behavioural analyses were performed within the 0700–1900 hours light cycle. All experiments were conducted in a blinded manner, and genotypes were decoded only after data processing and analysis. All procedures followed the National Institutes of Health Guide for the Care and Use of Laboratory Animals and were approved by the Institutional Animal Care and Use Committee at the Icahn School of Medicine at Mount Sinai. Locomotion and exploratory behaviour were measured using the open-field assay, as previously described28. Open-field activity was quantified using Fusion software (Omnitech Electronics). The mice were recorded for 60 min, and chambers were thoroughly cleaned with ethanol between rounds. Mice were habituated to a three-room novel object box for 20 min without objects. They were then returned to their home cage for a 1-h inter-test interval. Between each round, the box was thoroughly cleaned with ethanol. Videos of each session were measured using EthoVision video-tracking software (Noldus Information Technology). Blinded manual scoring was performed for the time spent sniffing the familiar and novel objects. Recognition score was calculated as the percentage of time spent sniffing the novel object relative to the total time spent sniffing either object. Control, 5xFAD, 5xFAD;PU1-low and 5xFAD;PU.1-high mice were monitored for survival and evaluated for health status at least once per week starting at tamoxifen induction (4–6 weeks of age) and continuing until death. Mice that were used for the experiments or died from environmental factors (such as cage flooding) were censored and excluded from death rate calculations. Both male and female mice were included in the study. No sex-specific differences in survival outcomes were observed; therefore, data from both sexes were pooled for analysis. Survival curves were generated using the Kaplan–Meier method, and statistical differences between groups were assessed using log-rank (Mantel–Cox) tests. Statistics were analysed using GraphPad Prism v.5.01, with significance determined as P < 0.05. Normal distribution was assessed using the Shapiro–Wilk normality test. Grubbs's test was used to identify outliers. Variance was determined using the F-test. An unpaired t-test was used for normally distributed datasets with two independent samples. For normally distributed datasets with two dependent samples (for example, plaque-associated and distal microglia from the same mice), we used a paired t-test. For normally distributed datasets with more than two samples, we used one-way ANOVA with multiple comparisons. Normally distributed data with unequal variance were analysed using Welch's correction. Non-normally distributed data involving two samples were analysed using a two-tailed Mann–Whitney U-test. Non-normally distributed data involving more than two samples were analysed using a two-tailed Kruskal–Wallis test. No statistical methods were used to predetermine sample size, but our sample sizes were similar to those generally used in the field. We did not include a justification of the sample size for this study. We used the minimum number of animals needed to reliably detect the expected effect size with an alpha rate of 0.05 in a standardly powered experiment and on the basis of extensive laboratory experience and literature in the field. There were further practical constraints related to the availability of samples and appropriate controls in the in vitro experiments, such as primary cell yield from animals and limitations in treatment or read-out methods, such as siRNA treatment or mass spectrometry. For human studies, the limiting factor was high-quality sample availability. These considerations collectively informed our sample size, which we deemed sufficient to observe biologically meaningful trends within the context of the aims of the study. Sample sizes are sufficient to detect significant changes on the basis of previous studies that used similar methodologies and experimental designs in the field. Data were formally excluded only if identified as a statistically significant outlier by Grubbs's outlier test. Any outliers identified by Grubbs's test were removed from the dataset, and the test was iterated until no outliers were detected. Each experiment was reproduced with similar results. Reproducibility is either indicated in the figure legends or shown as a quantification. For all experiments involving treatment with CSF1R or PLCi, animals or tissue culture wells were randomly assigned to groups. In other experiments, the mice were allocated into groups on the basis of either genotype or age. We did not compare human populations (control versus AD) but rather only within individuals with AD. Gene expression and western blot analyses were not conducted under blinded conditions because they rely on quantitative read-outs, which lack subjectivity and are not influenced by knowledge of the experimental condition. Samples were processed either under blinded conditions or in batches. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data have been made accessible in the indicated public repository. Upon request, the lead contacts are happy to provide any further information related to the data reported in this paper. Source data are provided with this paper. The code for MERFISH analysis is available at https://github.com/SchaferLabUMassChan/Ayata-et-al_2025. The code to analyse the human single-cell sequencing data of Olah et al.15 is available at https://github.com/tulsi92/pu1_paper_supp_fig7. The code to analyse the human single-cell sequencing data of Green et al.17 and AD Knowledge Portal can be found at https://github.com/maggieeiggam/pu1_paper/tree/main. Hansen, D. V., Hanson, J. E. & Sheng, M. Microglia in Alzheimer's disease. Alzheimer, A., Stelzmann, R. A., Schnitzlein, H. N. & Murtagh, F. R. An English translation of Alzheimer's 1907 paper, “Uber eine eigenartige Erkankung der Hirnrinde”. Deczkowska, A. et al. Disease-associated microglia: a universal immune sensor of neurodegeneration. & Grutzendler, J. Microglia constitute a barrier that prevents neurotoxic protofibrillar Aβ42 hotspots around plaques. Wang, Y. et al. TREM2-mediated early microglial response limits diffusion and toxicity of amyloid plaques. Ennerfelt, H. et al. SYK coordinates neuroprotective microglial responses in neurodegenerative disease. Hong, S. et al. Complement and microglia mediate early synapse loss in Alzheimer mouse models. Heneka, M. T. et al. Neuroinflammation in Alzheimer disease. Keren-Shaul, H. et al. A unique microglia type associated with restricting development of Alzheimer's disease. Mathys, H. et al. Single-cell transcriptomic analysis of Alzheimer's disease. Single cell RNA sequencing of human microglia uncovers a subset associated with Alzheimer's disease. Grubman, A. et al. Transcriptional signature in microglia associated with Aβ plaque phagocytosis. Green, G. S. et al. Cellular communities reveal trajectories of brain ageing and Alzheimer's disease. Rosenbauer, F. & Tenen, D. G. Transcription factors in myeloid development: balancing differentiation with transformation. Minderjahn, J. et al. Mechanisms governing the pioneering and redistribution capabilities of the non-classical pioneer PU.1. Nerlov, C. & Graf, T. PU.1 induces myeloid lineage commitment in multipotent hematopoietic progenitors. Dakic, A., Wu, L. & Nutt, S. L. Is PU.1 a dosage-sensitive regulator of haemopoietic lineage commitment and leukaemogenesis?. Huang, K. L. et al. A common haplotype lowers PU.1 expression in myeloid cells and delays onset of Alzheimer's disease. Oakley, H. et al. Intraneuronal β-amyloid aggregates, neurodegeneration, and neuron loss in transgenic mice with five familial Alzheimer's disease mutations: potential factors in amyloid plaque formation. Elmore, M. R. P. et al. Colony-stimulating factor 1 receptor signaling is necessary for microglia viability, unmasking a microglia progenitor cell in the adult brain. Stanley, E. R. & Chitu, V. CSF-1 receptor signaling in myeloid cells. Ayata, P. et al. Epigenetic regulation of brain region-specific microglia clearance activity. Huang, Y. et al. Microglia use TAM receptors to detect and engulf amyloid β plaques. Wang, S. et al. TREM2 drives microglia response to amyloid-β via SYK-dependent and -independent pathways. Andreone, B. J. et al. Alzheimer's-associated PLCγ2 is a signaling node required for both TREM2 function and the inflammatory response in human microglia. Tsai, A. P. et al. Genetic variants of phospholipase C-γ2 alter the phenotype and function of microglia and confer differential risk for Alzheimer's disease. Emerging roles for ITAM and ITIM receptor signaling in microglial biology and Alzheimer's disease-related amyloidosis. Recognition of yeast β-glucan particles triggers immunometabolic signaling required for trained immunity. Expression and pro-survival function of phospholipase Cγ2 in diffuse large B-cell lymphoma. The novel phospholipase C activator, m-3M3FBS, induces monocytic leukemia cell apoptosis. Rustenhoven, J. et al. PU.1 regulates Alzheimer's disease-associated genes in primary human microglia. Jones, R. E., Andrews, R., Holmans, P., Hill, M. & Taylor, P. R. Modest changes in Spi1 dosage reveal the potential for altered microglial function as seen in Alzheimer's disease. A. et al. Alzheimer's-associated PU.1 expression levels regulate microglial inflammatory response. Ralvenius, W. T. et al. A novel molecular class that recruits HDAC/MECP2 complexes to PU.1 motifs reduces neuroinflammation. Distinctive and indispensable roles of PU.1 in maintenance of hematopoietic stem cells and their differentiation. Yin, Z. et al. APOE4 impairs the microglial response in Alzheimer's disease by inducing TGFβ-mediated checkpoints. Marschallinger, J. et al. Lipid-droplet-accumulating microglia represent a dysfunctional and proinflammatory state in the aging brain. Concerted type I interferon signaling in microglia and neural cells promotes memory impairment associated with amyloid β plaques. Buskila, Y., Crowe, S. E. & Ellis-Davies, G. C. R. Synaptic deficits in layer 5 neurons precede overt structural decay in 5xFAD mice. Oblak, A. L. et al. Comprehensive evaluation of the 5XFAD mouse model for preclinical testing applications: a MODEL-AD study. Estrada-Capetillo, L. et al. CD28 is expressed by macrophages with anti-inflammatory potential and limits their T-cell activating capacity. Immune checkpoint TIM-3 regulates microglia and Alzheimer's disease. Wei, S. C., Duffy, C. R. & Allison, J. P. Fundamental mechanisms of immune checkpoint blockade therapy. The Consortium to Establish a Registry for Alzheimer's Disease (CERAD). Parkhurst, C. N. et al. Microglia promote learning-dependent synapse formation through brain-derived neurotrophic factor. Essential role of Src-family protein tyrosine kinases in NF-κB activation during B cell development. Zhang, R. et al. An obligate cell-intrinsic function for CD28 in Tregs. Cutting edge: essential role of phospholipase C-gamma 2 in B cell development and function. & García-Sastre, A. ISRE-reporter mouse reveals high basal and induced type I IFN responses in inflammatory monocytes. Sasaki, Y. et al. Canonical NF-κB activity, dispensable for B cell development, replaces BAFF-receptor signals and promotes B cell proliferation upon activation. & Vakoc, C. R. BET bromodomain inhibition suppresses the function of hematopoietic transcription factors in acute myeloid leukemia. Stanley, S. et al. Profiling of glucose-sensing neurons reveals that GHRH neurons are activated by hypoglycemia. Sasmita, A. O. et al. Parental origin of transgene modulates amyloid-β plaque burden in the 5xFAD mouse model of Alzheimer's disease. Bundy, J. L., Vied, C., Badger, C. & Nowakowski, R. S. Sex-biased hippocampal pathology in the 5XFAD mouse model of Alzheimer's disease: a multi-omic analysis. Shi, T. J. et al. Phospholipase Cβ3 in mouse and human dorsal root ganglia and spinal cord is a possible target for treatment of neuropathic pain. Activated microglia mitigate Aβ-associated tau seeding and spreading. Gratuze, M. et al. APOE antibody inhibits Aβ-associated tau seeding and spreading in a mouse model. McQuade, A. et al. Development and validation of a simplified method to generate human microglia from pluripotent stem cells. Negative feedback control of neuronal activity by microglia. Novel protocol for the isolation of highly purified neonatal murine microglia and astrocytes. Schaefer, A. et al. Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex. Blocking the interaction between apolipoprotein E and Aβ reduces intraneuronal accumulation of Aβ and inhibits synaptic degeneration. Targeted complement inhibition at synapses prevents microglial synaptic engulfment and synapse loss in demyelinating disease. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. A. CellProfiler Analyst 3.0: accessible data exploration and machine learning for image analysis. Retention of ES cell-derived 129S genome drives NLRP1 hypersensitivity and transcriptional deregulation in Nlrp3(tm1Flv) mice. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. The Perseus computational platform for comprehensive analysis of (prote)omics data. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. & Greenleaf, W. J. ATAC-seq: a method for assaying chromatin accessibility genome-wide. Corces, M. R. et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. & Johnson, D. S. Model-based analysis of ChIP-Seq (MACS). & Manke, T. deepTools: a flexible platform for exploring deep-sequencing data. Heiman, M. et al. A translational profiling approach for the molecular characterization of CNS cell types. Heiman, M., Kulicke, R., Fenster, R. J., Greengard, P. & Heintz, N. Cell type-specific mRNA purification by translating ribosome affinity purification (TRAP). Application of a translational profiling approach for the comparative analysis of CNS cell types. Massive mining of publicly available RNA-seq data from human and mouse. Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Xie, Z. et al. Gene set knowledge discovery with Enrichr. Amezquita, R. A. et al. Orchestrating single-cell analysis with Bioconductor. Dictionary learning for integrative, multimodal and scalable single-cell analysis. McCarthy, D. J., Campbell, K. R., Lun, A. T. & Wills, Q. F. Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R. Bioinformatics 33, 1179–1186 (2017). Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Choudhary, S. & Satija, R. Comparison and evaluation of statistical error models for scRNA-seq. Chibnik, L. B. et al. Susceptibility to neurofibrillary tangles: role of the PTPRD locus and limited pleiotropy with other neuropathologies. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Haghverdi, L., Lun, A. T. L., Morgan, M. D. & Marioni, J. C. Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. Kolde, R. pheatmap (Comprehensive R Archive Network, 2018). Lambert, J. C. et al. Meta-analysis of 74,046 individuals identifies 11 new susceptibility loci for Alzheimer's disease. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Tan, C. L. et al. MicroRNA-128 governs neuronal excitability and motor behavior in mice. We thank donors, patients, NIH NeuroBioBank (NIH Brain and Tissue Repository–California, Human Brain and Spinal Fluid Resource Center, VA West Los Angeles Medical Center, supported in part by the National Institutes of Health, US Department of Veteran Affairs, NIH NeuroBioBank at the University of Maryland and Mount Sinai NeuroBioBank), Neuropathology Brain Bank & Research CoRE at Mount Sinai, UCI ADRC and WashU ADRC Induced Pluripotent Stem Cell Core for providing the human tissues used in this study. We also thank T. Kurosaki and T. Inoue for the Plcg2fl/+ mice, A. García-Sastre (Icahn School of Medicine at Mount Sinai) for the ISRE Mx1:GFP mice, Rockefeller University Transgenic and Reproductive Technology Center for help with the generation of PU.1-high mice, R. Blitzer and the Mount Sinai Electrophysiology Core for electrophysiology experiments, Y.-H. E. Loh for TRAP and iMgl sequencing bioinformatics analysis, Tisch Cancer Institute Bioinformatics for Next Generation Sequencing (BiNGS) core and computational resources and staff expertise at Scientific Computing at Icahn School of Medicine for ATAC-seq analysis, Flow Cytometry Core at Icahn School of Medicine for cytometry assistance, R. Sebra and the Mount Sinai Center for Advanced Genomic Technology for 10x sequencing assistance, Rockefeller University Genomics Core for library quality control and sequencing assistance, A. Simkin (Brown University) for the find_nearest_neighbors2 Python script to analyse microglia–plaque nearest-neighbour distance and Plexxikon for providing PLX5622 chows and powder. We thank H. Strasburger for help with single-cell sequencing analysis; A. T. Chan for helping with the immunostaining of the Mx1+ microglia; B. Bushra, S. Khandare and A. Ramayanam for help with microglial PU.1 quantification in human tissue samples; M. Ryszawiec for maintaining iMgls; and other members of the Schaefer laboratory, Mount Sinai Ronald M. Loeb Center for Alzheimer's Disease and MD Anderson Cancer Center Belfer Neurodegeneration Consortium, specifically L.-H. Tsai and J. Ray, for helpful discussions. Schematics were created using BioRender (https://biorender.com). ROSMAP data were provided by the Rush Alzheimer's Disease Center at Rush University Medical Center. Further phenotypic data can be requested at www.radc.rush.edu. BiNGS core, Scientific Computing and Office of Research Infrastructure at the Icahn School of Medicine at Mount Sinai are supported by the Clinical and Translational Science Awards (CTSA) UL1TR004419, P30CA196521 and S10OD026880 and S10OD030463, respectively. A.T. is a Max Planck Fellow at the MPI for Biology of Ageing. Open access funding provided by Max Planck Society. These authors contributed equally: Pinar Ayata, Jessica M. Crowley, Matthew F. Challman Nash Family Department of Neuroscience, Department of Psychiatry, Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Pinar Ayata, Jessica M. Crowley, Matthew F. Challman, Vinaya Sahasrabuddhe, Diogo Ribeiro, Emma C. Hays, Philip Hwang, Anna A. Pimenova, Carmen Romero-Molina, Francesca Garretti, Tulsi A. Patel, Claudia De Sanctis, Edoardo Marcora, John F. Crary, Alison M. Goate & Anne Schaefer Ronald M. Loeb Center for Alzheimer's Disease, Center for Glial Biology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Pinar Ayata, Anna A. Pimenova, Carmen Romero-Molina, Francesca Garretti, Tulsi A. Patel, Claudia De Sanctis, Edoardo Marcora, John F. Crary, Alison M. Goate & Anne Schaefer Neuroscience Initiative, Advanced Science Research Center, Graduate Program in Biology, Graduate Program in Biochemistry, Graduate Program in Neuroscience, The City University of New York Graduate Center, New York, NY, USA Department of Neurology, Hope Center for Neurological Disorders, Charles F. and Joanne Knight Alzheimer's Disease Research Center, Washington University School of Medicine, St. Louis, MO, USA Maud Gratuze, Jason D. Ulrich & David M. Holtzman Department of Neurobiology, Brudnick Neuropsychiatric Research Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA Sebastian Werneburg, Violeta Durán-Laforet, Travis E. Faust, Robert E. Murphy & Dorothy P. Schafer Department of Ophthalmology and Visual Sciences, Kellogg Eye Center, Michigan Neuroscience Institute, Department of Molecular & Integrative Physiology, University of Michigan, Michigan Medicine, Ann Arbor, MI, USA Max Planck Institute for Biology of Ageing, Cologne, Germany Francisco Mendes Lopes, Chrysa Nikopoulou, Sarah Buchholz & Anne Schaefer Francisco Mendes Lopes, Chrysa Nikopoulou, Sarah Buchholz, Taoyu Mei, Andreas Beyer & Anne Schaefer Department of Genetics & Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Carmen Romero-Molina, Francesca Garretti, Tulsi A. Patel, Edoardo Marcora & Alison M. Goate Department of Pathology, Department of Artificial Intelligence & Human Health, Neuropathology Brain Bank and Research CoRE, Icahn School of Medicine at Mount Sinai, New York, NY, USA Bioinformatics for Next-Generation Sequencing (BiNGS) Core, Tisch Cancer Institute, Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Department of Human Genetics, Genentech Inc., South San Francisco, CA, USA Institute of Innate Immunity, Department for Systems Immunology and Proteomics, Medical Faculty, University of Bonn, Bonn, Germany German Rheumatology Research Center (DRFZ), A Leibniz Institute, Berlin, Germany Columbia Center for Human Development, Center for Stem Cell Therapies, Department of Medicine, Columbia University Vagelos College of Physicians and Surgeons, New York, NY, USA Laboratory of Immune Cell Epigenetics and Signaling, The Rockefeller University, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed single-cell and single-nucleus sequencing experiments. performed western blotting experiments and analyses. performed cell culture and iPSC experiments. generated and maintained all mouse lines. designed the Tau seeding experiment, which was performed by M.G. designed the synapse quantification, which was performed by S.W. performed the in situ microscopy imaging and analysis. designed the MERFISH analysis, which was performed by V.D.-L., T.E.F. A.S. designed the vectors, which were cloned by A.A.P. performed the ATAC-seq, which was analysed by A.V.R.J. performed the bioinformatics analysis of published human single-cell sequencing data. supervised the PU.1 staining in human tissue, which was performed by C.D.S. supervised the BV2 cell proteomics analysis, which was performed by F.D.W. wrote the paper, and all other authors provided critical feedback. Correspondence to Pinar Ayata or Anne Schaefer. receives financial compensation as an advisory board member for Genentech and Muna Therapeutics. is an inventor on a patent licenced by Washington University to NextCure on the therapeutic use of anti-apoE antibodies. co-founded, has equity and is on the scientific advisory board of C2N Diagnostics. The remaining authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Increased PU.1low microglia abundance during AD disease progression. PU.1 protein expression in cortical microglia is shown. Microglial nuclei, as identified by Imaris, are indicated in dotted circles. The bar graphs show the size of PU.1low microglia subpopulation in the cortex of 3-month-old (n = 5 mice, 2♀, 3♂), 6-month-old (n = 5 mice, 3♀, 2♂), 9-month-old (n = 3 mice, 3♀), and 12-month-old (n = 3 mice, 1♀, 2♂) 5xFAD mice—ordinary one-way ANOVA with multiple comparisons. (c) Strategy for microglia depletion using CSF1R inhibitor (CSF1Ri, PLX5622, 1200 ppm diet) for ~6 weeks starting at 4.5 months of age in control and 5xFAD mice. (d) CSF1R-independent microglia survival at the plaque. Bar graphs show: Left: the total number of all microglia per mm3 in the cortex of 6-month-old control diet- and CSF1Ri diet-fed wild-type (left) or 5xFAD mice (right). Middle: the number of distal (left) and plaque-associated (right) microglia per mm3. Right: the average number of plaque-associated microglia normalized to plaque numbers in control or CSF1Ri-treated 5xFAD mice. (e) SYK/PLCγ2 are required for PU.1low microglia at the plaque. (f) CSF1R-independent survival of plaque-associated PU.1low microglia depends on PLCγ2 activity. Bar graph shows the number of plaque-associated microglia per plaque and number of distal microglia per mm3 in CSF1Ri-treated 5xFAD (n = 4 mice, 2♀, 2♂), CSF1Ri- and PLCi-treated 5xFAD (pharmacological PLC inhibition: PLC inhibitor, 8.7 mg/kg U73122, n = 6 mice, 2♀, 4♂) and CSF1Ri-treated 5xFAD; PLCγ2-KO mice (genetic microglia-specific PLCγ2 deletion: n = 4 mice, 3♀, 1♂)—ordinary one-way ANOVA with multiple comparisons. Bar graphs with individual data points show mean ± SEM. Illustrations in b, c, e and f were created using BioRender (https://biorender.com). (a) Experimental design (left), representative images (middle), and quantification (right) of Spi1 mRNA (measured by single-molecule in situ hybridization) and PU.1 protein expression (measured by immunofluorescence) in cultured primary microglia treated with vehicle or 1 mg/ml of phosphatidylserine/phosphatidylcholine (PS/PC) liposomes for either 12 h or 3 days. The bar graphs show Spi1 mRNA expression (left) and PU.1 protein expression (right) normalized to the mean of the control for each time point—an unpaired two-tailed t-test was used. (b) Experimental design (top left), representative immunofluorescence images (bottom left), and quantification (right) of PU.1 intensity in cultured primary microglia treated with vehicle or PS/PC liposomes at 0.1 or 1 mg/ml for 3 days. The bar graph shows quantification of PU.1 protein expression in microglia nuclei with treatment of vehicle control (n = 6 wells), 0.1 mg/ml PS/PC liposomes (n = 3 wells), and 1 mg/ml PS/PC (n = 3 wells), normalized to the mean of vehicle control—ordinary one-way ANOVA with multiple comparisons. Experimental design (top left), representative immunofluorescence images (bottom left), and quantification (right) of PU.1 levels in cultured primary microglia treated with vehicle, PS/PC liposomes at 1 mg/ml for 3 days, and received washout (4 days). The bar graph shows PU.1 levels in microglial nuclei in vehicle-treated, 3-day liposome-treated (1 mg/ml), or liposome-treated followed by washout (n = 3 wells/group)—ordinary one-way ANOVA with multiple comparisons. Experimental design (top), representative immunofluorescence images (bottom left), and quantification (bottom right) of PU.1 expression in cultured primary microglia treated with vehicle or CLEC7A agonist, 50 μg/mL pustulan for 3 days—unpaired two-tailed t-test (n = 3 wells/group). (e) Experimental design (top left), representative immunofluorescence images of PU.1 expression (right), and quantification (bottom left) of cultured primary microglia treated with vehicle or 0.1 mg/ml liposomes for 3 days following a 1-h pre-treatment of PLC inhibitor (PLCi, 0.1 μM U73122) or vehicle DMSO, or only PLC activator (PLCa, 40 μM m-3M3FBS). The bar graphs show quantification of PU.1 expression in microglia nuclei normalized to vehicle control. Left for PLCi: n = 4 wells/group—ordinary one-way ANOVA with multiple comparisons. The PLCi experiment was independently repeated. (f) PLC inhibitor treatment alone does not change baseline PU.1 expression. Experimental design (top), representative immunofluorescence images (bottom left), and quantification (bottom right) of PU.1 expression in cultured primary microglia treated with vehicle (n = 4 wells) or PLC inhibitor (PLCi, 0.1 μM U73122) for 3 days—an unpaired two-tailed t-test was used (n = 4 wells/group). Bar graphs with individual data points show mean ± SEM. Illustrations in a–f were created using BioRender (https://biorender.com). (a, b) PU.1low plaque-associated microglia show lymphoid gene expression. (a) Top: Isolation of microglia by fluorescence-activated cell sorting (FACS) followed by single-cell mRNA sequencing. Bottom: The balloon plot shows differentially regulated gene ontologies (GOs, Fisher's exact test) in the PU.1low and PU.1high cortical disease-associated microglia (DAM) subpopulations. (b) Scatter plot shows the correlation of upregulated genes in cortical PU.1high (red) vs. PU.1low microglia (blue) between two independent experiments (n = 1 mouse/group per experiment) using FindMarkers() analysis in Seurat—Wilcoxon rank sum test. 558 genes are significantly upregulated and 445 genes are significantly downregulated in PU.1low microglia in two independent experiments, while 0 genes are regulated in opposing directions. (c) Plaque-associated, CSF1R-independent PU.1low microglia show lymphoid gene expression. Single-nuclei sequencing of cortical microglia from 6-month-old control (♀) and two CSF1Ri-treated 5xFAD mice (2♂ pooled). Left: Isolation of lineage-traced microglial nuclei from the cortex by fluorescence-activated nuclear sorting (FANS) followed by single-nuclei sequencing. Right: The volcano plot shows the differentially up- (red) and down- (blue) regulated genes in cortical microglia control and CSF1Ri-treated mice from Fig. 1d by FindMarkers() function in Seurat—Wilcoxon rank sum test. (d) PU.1low microglia-expressed lymphoid-associated signaling molecules (ligands, surface receptors, and downstream signaling proteins) in T-cells are shown. (e) Heatmap shows log2 (fold change) of selected differentially expressed genes in the PU.1low DAM compared to the PU.1high DAM, two independent experiments: (1) single-cell sequencing of forebrain microglia from wild-type and 5xFAD mice (Figs. (f) Top: Schematic shows multiplexed error-robust fluorescent in situ hybridization (MERFISH) within the brain of the 5xFAD model (8-month-old, ♂). (g) Representative images show different cell types as well as a PU.1low plaque-associated, and PU.1high distal microglia. T cell identifying genes Cd3e and Cd8a: green. Microglia homeostatic genes Cx3cr1, Tmem119, P2ry12, Csf1r, Hexb, Aif1, Tnf, Trem2, Itgam, C1qa, C1qb, C1qc, Sall1, Selplg: red. Oligodendrocyte-identifying genes Mog, Sox10, and Mag: brown. Marker genes are shown in Supplementary Table 4. Illustrations in a, c, d and f were created using BioRender (https://biorender.com). Brackets indicate the two homology arms mediating recombination in embryonic stem cells. F-Spi1 (green) encodes the FLAG-tagged PU.1 protein sequence, and loxP (red) sites are indicated by arrowheads. Cre-mediated removal of the loxP-flanked STOP codon leads to F-Spi1 expression. Right: Representative genotyping gel images of R26+/+ wild-type, R26LSL.F-Spi1/+ heterozygous, and R26LSL.F-Spi1/LSL.F-Spi1 homozygous mice are shown. (b) PU.1 protein expression in PU.1-low, PU.1-wild-type (PU.1-wt), and PU.1-high mice. Representative Western blot (left) and quantification of protein levels (right). Representative fluorescence-activated cell sorting (FACS) plots for microglia isolation are shown in Supplementary Fig. The bar graphs show PU.1 level normalized to microglial marker IBA1, IBA1 level normalized to Histone H3 in the cortex of 6-month-old PU.1-low (Cx3cr1CreErt2/+;Spi1fl/+, n = 5 mice, 3♀, 2♂), PU.1-wt (Cx3cr1CreErt2/+, n = 5 mice, 2♀, 3♂), and PU.1-high mice (Cx3cr1CreErt2/+;R26LSL.F-Spi1/+, n = 3 mice, 3♀)—ordinary one-way ANOVA with multiple comparisons. Representative immunofluorescence images (left) and quantification of microglia morphologies (right). The bar graphs show the number of microglia (left) and microglial cell size (volume, right) in the cortex of 6-month-old PU.1-low (n = 4 mice, 1♀, 3♂), PU.1-wt (n = 4 mice, 3♀, 1♂), and PU.1-high mice (n = 4 mice, 3♀, 1♂), as calculated by Imaris—ordinary one-way ANOVA with multiple comparisons. (d-f) PU.1-low and PU.1-high microglia show distinct changes in gene expression in the healthy brain, with PU.1-low inducing lymphoid and PU.1-high inducing pro-inflammatory and stress response patterns. (d) Experimental design (left) and MA plots (right, representing log-ratio (M) on the y-axis and mean average (A) on the x-axis) show gene expression changes (red, upregulated; blue, downregulated by DESeq2) shown in Supplementary Table 6. (e) Representative genome browser views (in Integrative Genomics Viewer, IGV) of normalized read counts of Cd28 ribosome-associated mRNA in PU.1-low, PU.1-wt, and PU.1-high microglia. Numbers in brackets indicate the minimum and maximum values of the y-axis shown for each gene. (f) The balloon plot shows enrichment of gene ontologies (Fisher's exact test) in differentially regulated genes associated with PU.1 level-dependent and independent gene sets by DESeq2. p-value (color) and combined score (size) are shown. (g-i) PU.1-low microglia show overall microglia-specific chromatin accessibility, with PU.1 expression levels governing distinct accessibility patterns. (g) Bottom: Isolation of PU.1-low, PU.1-wild-type (PU.1-wt), and PU.1-high microglia from the brain and wild-type T cells from the spleen by FACS, followed by ATAC sequencing. Top: Principal component analysis (PCA) of the 5,000 most differential ATAC sequencing peaks in microglia and T cells (left) or between microglia of the different PU.1 genotypes (right). Colors in the main heatmap (left) show scaled read counts (variance stabilizing transformations, vst) per sample. Colors on the side heatmap (right) show significantly up or downregulated peaks between indicated samples shown in Supplementary Table 7. (i) HOMER DNA-motif enrichment analysis of peaks with reduced and increased accessibility in PU.1-low microglia identifies enrichment in peaks containing the PU.1 DNA binding motif (down), and ETS1, MEF2D, and IRF1 (up). Bar graphs with individual data points show mean ± SEM. Illustrations in d and g were created using BioRender (https://biorender.com). (a) Microglia in vivo: Schematic showing microglia and T cell isolation (left) and Western blot with quantification for CD28 levels (normalized CD28 quantification shown below the blot) in ex vivo acutely isolated microglia from the forebrain of 12-month-old PU.1-low, PU.1-wild-type (PU.1-wt), and PU.1-high mice (n = 3 pooled mice, 1♀, 2♂ each) as well as T cells isolated from the spleen of a wild-type control (♂). Representative FACS plots for microglia isolation are shown in Supplementary Fig. Representative FACS plots for T cell isolation are shown in Supplementary Fig. (b-c) BV2 cells: control cells treated with scrambled small interfering (si) RNAs (scramble) and PU.1-knockdown (PU.1-KD) cells treated with siRNAs against Spi1, as well as control cells transfected with empty vector (empty) and PU.1-overexpressor (PU.1-OE) cells transfected with a vector containing the Spi1 transgene from Pimenova et al.39. (b) Representative protein expression analysis using Western blot (left) with experimental design (top) and quantification (right) of CD28 and PU.1 levels in scramble and PU.1-KD cells (n = 4 wells/group), as well as in empty and PU.1-OE cells (n = 3 wells/group) are shown—unpaired two-tailed t-test was used. (c) Experimental design (left) and bar graphs (right) show protein levels (n = 4 wells/group) identified by proteomics analysis—unpaired two-tailed t-test. Differentially expressed proteins are shown in Supplementary Table 8. (d) Primary microglia: Experimental design (left), representative Western blot (middle), and quantification (right) of CD28 and PU.1 levels in primary microglia treated with siRNAs against Spi1 (PU.1) or Cd28 or control (scramble, Scr, n = 3 wells/group)—ordinary one-way ANOVA with multiple comparisons used. (e-f) Human iMgls: (e) Experimental design (top), RNA expression quantification by qPCR (bottom left), and representative protein expression analysis using Western blot (bottom right) of SPI1/PU.1 level in human hematopoietic progenitor cell-(HPC)-derived microglial-like cells (iMgls) treated with siRNAs against SPI1/PU.1 (siSPI1) or control scramble siRNA (siSCR). The bar graph shows qPCR-based quantification of SPI1 normalized to GAPDH in siSCR and siSPI1-targeted iMgls (n = 4 independent experiments)—paired two-tailed t-test. Please see Table 2 for demographic information. (f) Heatmap shows z-scored vsts of the differentially regulated genes in siSCR and siSPI1-targeted iMgls identified by RNA sequencing and DESeq2 (n = 6 independent experiments), shown in Supplementary Table 9. Bar graphs with individual data points show mean ± SEM. Illustrations in a–e were created using BioRender (https://biorender.com). (a, b) Experimental design (left), representative protein analysis by Western blot (middle), and quantification (right). The bar graph shows SYK and PLCγ2 phosphorylation levels normalized to total SYK and PLCγ2 protein levels—unpaired two-tailed t-test. Protein extracts equivalent to 120,000 microglia were loaded for Western blot per sample. Representative FACS plots for microglia isolation are shown in Supplementary Fig. Bar graphs with individual data points show mean ± SEM. Illustrations in a and b were created using BioRender (https://biorender.com). (a-f) Single-nuclei mRNA sequencing of cortical microglia from a 6-month-old 5xFAD (♀), 5xFAD;PU.1-low (♂), and 5xFAD;PU.1-high mouse (♂). (a) Left: Isolation of lineage-traced microglial nuclei by fluorescence-activated nuclear sorting (FANS) followed by single-nuclei sequencing of cortical microglia from 6-month-old control, 5xFAD, and 5xFAD;PU.1-low mice. Right: Uniform manifold approximation and projection (UMAP) visualizations show microglia assigned into clusters that are manually annotated based on the marker genes. (b) The dot plot shows the scaled average expression per cluster (color) and the percentage of cells (size) from each DAM subcluster. (c) UMAP visualizations show microglia assigned into clusters that are manually annotated based on the marker genes shown in Supplementary Table 10. (e, f) Feature plots show the expression of indicated lymphoid (e) and interferon genes (f) and the distribution of cells expressing the genes. Illustrations in a were created using BioRender (https://biorender.com). (a) Experimental design (left) and MA plots (right, representing log-ratio (M) on the y-axis and mean average (A) on the x-axis) show gene expression changes (red, upregulated; blue, downregulated) by DESeq2, shown in Supplementary Table 11. (b) Heatmap shows z-scored variance stabilizing transformations (vsts, color) of genes selected by overlapping and non-overlapping genes between pairwise comparisons (5xFAD;PU.1-low vs. 5xFAD and 5xFAD vs. control). (c) The balloon plot shows significantly enriched gene ontologies (Fisher's exact test) associated with significantly regulated PU.1 level-dependent and independent gene sets (DESeq2). p-value (color) and combined score (size) are shown. Illustrations in a were created using BioRender (https://biorender.com). (a) 5xFAD-PU.1-low mice show similar numbers of distal microglia. The bar graphs show the density of distal and total microglia in the cortex of 6-month-old 5xFAD (n = 6 mice, 3♀, 3♂) and 5xFAD-PU.1-low mice (n = 5 mice, 3♀, 2♂)—unpaired two-tailed t-test. Western blot shows CD28 levels in ex vivo microglia acutely isolated from the forebrain of 12-month-old control (n = 3 pooled mice, 1♀, 2♂) and 5xFAD; PU.1-low mice (n = 3 pooled mice,1♀, 2♂). Normalized CD28 quantification is shown below the blot. (c-e) Microglia in 5xFAD-PU.1-low mice show reduced neutral lipid droplet load and complement C1Q expression. (c) Western blot (left) with quantification (right). The bar graph shows the C1QA level normalized to TUBB3 per sample in the hippocampi of 6-month-old control (n = 6 mice, 6♂), 5xFAD (n = 4 mice, 4♂), and 5xFAD-PU.1-low mice (n = 3 mice, 3♂)—ordinary one-way ANOVA with multiple comparisons was used. (d-e) Representative immunofluorescence images (left) with quantification (right) from the cortex of 6-month-old mice—unpaired two-tailed t-test. (e) CD11B+ microglia: red, BODIPY 493/503: green, ThioS+ dense-core plaques: blue. (f) 5xFAD-PU.1-low mice show a reduced soluble amyloid load. The bar graphs show the percent area covered by 6E10+ soluble Aβ or ThioS+ dense-core plaques in the subiculum from 6-month-old 5xFAD (n = 7 mice, 2♀, 5♂) and 5xFAD; PU.1-low mice (n = 5 mice, 2♀, 3♂)—unpaired two-tailed t-test. Representative images are shown in Fig. (g) 5xFAD-PU.1-low mice show a reduced spread of p-Tau following intracerebral injection of human pathogenic Tau isolated from the brain (schematic, top). Bar graphs (bottom) show the percent AT8+ phosphorylated Tau (p-Tau) area and HJ3.4+ amyloid β in the indicated brain regions of 10-month-old 5xFAD (n = 12 mice, 7♀, 5♂) and 5xFAD; PU.1-low mice (n = 9 mice, 4♀, 5♂) (h) LTP: The line graph shows the time course of long-term synaptic plasticity (LTP) quantified as a percentage change in excitatory postsynaptic potential (EPSP) slope from baseline (10-min period of a pre-and 60-min period of a post-100 Hz stimulation) in CA1 neurons of the hippocampus in 9-10-month-old control (n = 12 slices from 5 mice, 3♀, 2♂), 5xFAD (n = 13 slices from 6 mice, 4♀, 2♂), and 5xFAD; PU.1-low mice (n = 11 slices from 5 mice, 1♀, 4♂)—two-way ANOVA with repeated measures, Ptime = 0.0183, Pgenotype = 0.1031. (i) Open field analysis: Experimental design (left) and quantification (right) of the open field experiment. The area identified as “center” is marked by a red square. The bar graph shows the total distance traveled (m), time spent in the center (seconds), and the periphery (seconds) in a 10-min-long open field assay from 9–10 month-old control (n = 15 mice, 9♀, 6♂), 5xFAD (n = 12 mice, 6♀, 6♂), and 5xFAD; PU.1-low (n = 13 mice, 7♀, 6♂)—ordinary one-way ANOVA with multiple comparisons. (j) Novel object recognition: The schematic (top) and quantification of the novel object recognition experiment in a 3-room box. The bar graphs show the distance traveled during 20-min habituation, 10-min training, and 10-min testing; the total time spent sniffing any object during training and testing; and the “recognition score” calculated as the percent time spent sniffing the constant object to the total time spent sniffing either object during training from 9-10-month-old control (n = 10 mice, 5♀, 5♂), 5xFAD (n = 11 mice, 6♀, 5♂), and 5xFAD; PU.1-low mice (n = 15 mice, 7♀, 8♂)—ordinary one-way ANOVA with multiple comparisons. No sex differences are observed in behavioral analyses, and trends between genotypes were preserved in sex-specific analyses. Bar graphs with individual data points show mean ± SEM. Illustrations in g, i and j were created using BioRender (https://biorender.com). (a-b) Liposomes induce microglial CD28 RNA and protein expression in primary microglia in vitro: (a) Experimental design (left), qPCR quantification (middle left), representative Western blot (middle right), and Western blot quantification (right) of CD28 levels in primary microglia treated with 0.1 mg/ml liposomes for 3 days. The first bar graph shows Cd28 normalized to Gapdh in primary microglia treated with vehicle (n = 4 wells) or liposomes (n = 5 wells)—unpaired two-tailed t-test. The second bar graph shows CD28 levels normalized to ACTB in primary microglia treated with vehicle or liposomes (n = 3 wells/group)—unpaired two-tailed t-test. (b) Representative immunofluorescence images (left) with quantification (right) of CD28 in primary microglia treated with 0.1 mg/ml liposomes for 3 days. The bar graphs show normalized integrated CD28 intensity in liposome-treated (control: n = 6 wells, liposome: n = 7 wells) primary microglia—unpaired t-test. (c-d) The number of plaque-associated CD28+ cells increases with amyloid disease progression. (c) Cd28 RNA: Feature plots (right) show Cd28 expression in uniform manifold approximation and projection (UMAP) of single-cell sequencing of forebrain microglia from 3- and 8-month-old wild-type and 5xFAD mice (n = 2 mice/group except control at 3 months: n = 1 mouse, all ♂). (d) CD28 protein: Representative immunofluorescence images (left) with quantification (right) of CD28+ microglia in the subiculum. IBA1+ microglia: green, CD28: red, ThioS+ dense-core plaques: blue, DRAQ5+ nuclei: gray. The bar graphs show the fraction of CD28 + IBA1+ microglia in the subiculum of 3-month-old (n = 4 mice, 2♀, 2♂), 6-month-old (n = 4 mice, 1♀, 3♂), and 12-month-old (n = 4 mice, 3♀, 1♂) 5xFAD mice—ordinary one-way ANOVA with multiple comparisons. Bar graphs with individual data points show mean ± SEM. Illustrations in a were created using BioRender (https://biorender.com). (a) Microglial Cd28 ablation in the healthy brain has no impact on microglia numbers or morphology. The experimental design (left), representative immunofluorescence images (middle), and quantification (right) of microglia in the cortex of CD28-KO mice. The bar graphs show the total microglia numbers per image (left) and microglial cell volume (right, μm3) in the subiculum of 3-month-old control (n = 5 mice, 1♀, 4♂) and CD28-KO mice (Cx3cr1CreErt2/+(Litt); Cd28fl/fl, n = 3 mice, 3♂)—unpaired two-tailed t-test. (b) Microglia-specific Cd28 ablation in 5xFAD mice prevents plaque-induced Cd28 expression. Experimental design (left), representative RNA in situ hybridization with immunofluorescence images (middle), and quantification (right) of Cd28+ microglia. CD11B+ microglia: green, Cd28 mRNA: red, DAPI+ nuclei: blue. The bar graph shows the percent of Cd28 + IBA1+ microglia in the subiculum of 6-month-old 5xFAD (n = 3 mice, 1♀, 2♂) and 5xFAD;CD28-KO mice (n = 3 mice, 2♀, 1♂)—unpaired two-tailed t-test. (c) Microglial Cd28 ablation in 5xFAD mice does not affect microglia numbers. Representative immunofluorescence images (left) with quantification (right) of microglia numbers. IBA1+ microglia: green, ThioS+ dense-core plaques: blue, DRAQ5+ nuclei: gray. The bar graph shows the microglia numbers in the dentate gyrus of 3- and 6-month-old 5xFAD (n = 6 mice at 3 months, 1♀, 5♂ and n = 5 mice at 6 months, 2♀, 3♂) and 5xFAD; CD28-KO mice (n = 5 mice at 3 months, 2♀, 3♂ and n = 7 mice at 6 months, 4♀, 3♂)—two-way ANOVA with multiple comparisons. Pgenotype = 0.3913, Page < 0.0001 (d) Microglial Cd28 ablation in 5xFAD mice alters plaque load and composition. Representative immunofluorescence images (left) with quantification (right) of plaque load in the subiculum. 6E10+ amyloid beta: red, ThioS+ dense-core plaques: blue. The bar graphs show 6E10/ThioS ratio, total 6E10 level, and total ThioS level, number of ThioS+ plaques, and volume of ThioS+ plaques in the dentate gyrus of 3- and 6-month-old 5xFAD (n = 6 mice at 3 months, 1♀, 5♂ and n = 5 mice at 6 months, 2♀, 3♂) and 5xFAD; CD28-KO mice (n = 5 mice at 3 months, 2♀, 3♂ and n = 7 mice at 6 months, 4♀, 3♂)—two-way ANOVA was used (ThioS: Pgenotype = 0.3077, Page = 0.0005; 6E10: Pgenotype = 0.6140, Page = 0.0024; ThioS/6E10: Pgenotype = 0.1121, Page = 0.0271; average ThioS plaque numbers per image per mouse: Pgenotype = 0.9603, Page < 0.0001; ThioS volume per image per animal: Pgenotype = 0.0273, Page < 0.0001). Left: Microglia isolation strategy for single-cell sequencing. Right: Volcano plot shows significantly up- (red) and downregulated (blue) genes between all microglia from 5xFAD and 5xFAD;CD28-KO mice identified by DESeq2 in pseudobulk analysis using FindMarkers() function using Wilcoxon Rank Sum test in Seurat, shown in Supplementary Table 12. (f) Representative RNA in situ hybridization with immunofluorescence images (left) with quantification (right) of Mx1 levels in microglia. IBA1+ microglia: green, Mx1 RNA: red, DAPI+ nuclei: blue. Dotted circles indicate microglial nuclei, some of which are shown in higher magnifications on the bottom. Bar graphs with individual data points show mean ± SEM. Illustrations in a, b and e were created using BioRender (https://biorender.com). (a-d) Single-cell sequencing of human microglia from Kimura et al.49. (a) Uniform manifold approximation and projection (UMAP) visualizations (left). (b, c) Dot plot (b) and feature plots (c) show the expression of Cd28, Spi1, and selected PU.1low microglia expressed lymphoid genes (Cd28, Pdcd1, Cd274, Sox5, Dkk2, Prkcq, Btla) in UMAP visualizations. Marker genes for subsets are shown in Supplementary Table 13. Single-cell sequencing from wild-type and 5xFAD mice.Supplementary Table 2. ATAC sequencing from PU.1-low–wt–high microglia and T cells. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Lymphoid gene expression supports neuroprotective microglia function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69239341/scotland-underground-chamber/'>Archaeologists Discovered a Mysterious Underground Chamber from an Ancient Civilization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The archaeologists that found the stone-paved chamber, located inside a hilltop fort in Scotland, still aren't sure exactly what its purpose was. Archaeologists digging through the remains of an Iron Age hilltop fort in Scotland located a semi-underground stone-built chamber, complete with stone flooring. Just what it was for, though, remains a mystery, even after careful analysis. Excavations at the site came thanks to a road project. They discovered aerial photographs taken in the 1960s that revealed the settlement, Guard Archaeology project officer Kenny Green said in a statement. “Without these aerial photographs, no one would have known there was a hillfort here as there was no trace of it on the ground,” he said. “Centuries of ploughing had removed any surface trace.” and the first two centuries C.E., part of a distinctive Iron Age Scottish culture. The team found cereal grain on the floor, but not enough to suggest it was used for grain storage. Chemical analysis of the floor deposits was also inconclusive. A third ditch and rampart were then newly dug to surround the underground passage, while another outer rampart was created. Instead of serving as military outposts, they were fortified hilltop settlements. “Our team discovered the charred remains of wattle panels and pieces of daub from the roundhouses that belonged to this time,” he said. “These had been thrown into the ditches when the settlement had been cleared of old buildings at some point.” The team found bog ore and slag, waste from smelting iron ore. A fragmented of vitrified clay showed it was part of a furnace used for metalworking. “Many drivers and passengers traveling the new road may not imagine that they cross the site of a prehistoric fort,” Jillian Ferguson, roads and infrastructure manager at Perth and Kincross Council, said in a statement. “The construction of the Cross Tay Link Road at this site has provided an inadvertent but invaluable opportunity to learn more about how people near Perth lived more than 2,000 years ago.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Volunteers Dug Up Artifacts From a Roman Fort The First Americans May Have Sailed From Japan A Fisherman Digging Up Worms Found Buried Treasure</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/a69234721/zond-space-object/'>Scientists Think This Space Object Could Be From a Dead Civilization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The object, initially thought to be a meteorite, could actually be Zond 1, a Soviet spacecraft that was destined for Venus 60 years ago. Every object that trespasses on the inner solar system brings with it questions about its origin. Now, Loeb thinks the newest target of interest could also be a technosignature—one from Earth. Object Arjuna 2025 PN7, named for the Arjuna asteroid belt, was detected by the Pan-STARRS survey as far back as 2014, but it was so faint that it went unnoticed by astronomers until now. The Arjuna belt is made up of over a hundred asteroids in Earth-like orbits, quasi-satellites that move along with our planet but are not gravitationally bound to it, as opposed to mini moons that are gravitationally captured and orbit Earth temporarily. That civilization would be the Soviet Union, which sent a failed mission to Venus in April 1964. There were operation errors and glitches in communication with the probe when its pressurized orbit compartment began to leak. Its closet approach to Venus occurred in July of 1964, the same time that Zond 1 was supposed to arrive at its destination. “For every such trajectory, the spacecraft positions and velocities were compared against those of 2025 PN7, in order to ascertain whether this object could be associated with the mission in question,” Hibberd and Loeb said. If measurements of its light spectrum could at least reveal its surface composition, that may give away whether it is a piece of outdated technology, as opposed to being another asteroid. What is problematic about declaring the identity of 2025 PN7 to be Zond 1 is that it flew closer to the Sun than was predicted based on its trajectory. So could the Arjuna object actually be what remains of Zond 1? Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. If Aliens Exist, Why Haven't We Heard From Them? The Best Star Projectors Turn Your Ceiling Into a Night Sky A Scientist Says This Object May Be an Alien Craft Aliens May Have Started Life on Earth, Expert Says</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/long-lived-gamma-ray-burst-is-unlike-any-seen-before/'>These Cosmic Outbursts Normally Last for Minutes. This One Went on for Hours—And Nobody Knows Why</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A cosmic explosion known as GRB 250702B is by far the longest gamma-ray burst astronomers have ever seen—if it's even one at all GRBs are extremely energetic explosions that rank among the most powerful astrophysical events in the universe, so luminous they can be seen from billions of light-years away. When one of these jets is pointed directly at our solar system, we see it as a GRB. They aren't that unusual to spot—the universe is so big and GRBs are so bright that astronomers see one or two a day on average. Even so, we soon realized something very unusual was happening with GRB 250702B. In the roughly half century of GRB studies, no one has ever seen one of these epic events last more than a few hours at most. Yet almost a full day elapsed before GRB 250702B faded away. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. If it came from a GRB, it's the longest one ever detected,” explains Adelle Goodwin, an astronomer at Curtin University in Australia, who is leading follow-up radio observations with the Very Large Array in New Mexico. Another oddity was that this GRB seemed to repeat, with astronomers seeing what appeared to be three distinct pulses across its nearly day-long duration; repetition would be strange because GRBs are normally so catastrophic that they effectively destroy their astrophysical sources, preventing any recurrence. Astronomers are not ones to shy away from a challenge, but GRB 250702B has provided us with several. Luckily, the James Webb Space Telescope was designed to peer through all that dust, allowing a team of researchers to reveal the truth in a recent preprint. GRB 250702B's still-mysterious origins trace back to a galaxy more than five billion light-years away from us. Astronomers can't yet say with certainty, but an unusual GRB coming from an unusual galaxy might be a clue in itself on what actually caused the explosion. Of course, this is the real question behind GRB 250702B: What caused it? This very odd GRB could have arisen from a particularly unusual collapse of a massive star that launched an abnormally energetic jet, or instead it could have come from a tidal disruption event (TDE) in which a white dwarf star was ripped apart and devoured by an intermediate-mass black hole weighing on the order of 100,000 solar masses. “The duration of GRB emission cannot be explained by classical collapsar scenarios, which means more exotic scenarios need to be invoked,” Goodwin explains. I recently attended this year's meeting of the American Astronomical Society's High-Energy Astrophysics Division (HEAD), presenting some of my own research on TDEs; suffice to say that speculation on GRB 250702B's origin dominated much of the informal sessions. “It's too long to be a GRB but not luminous enough to be a TDE at that distance,” one of my colleagues confidently argued. Others were more cautious, pointing out that extraordinary claims require extraordinary evidence, and said they felt that threshold was not yet met for GRB 250702B. One thing is for sure: nothing quite drives excitement in a crowd of astronomers like an unusual signal none of us has seen before. (Despite all that excitement, many astronomers unfortunately declined to comment on the record for this story, citing a forthcoming NASA press release on GRB 250702B that is stuck in limbo because of the ongoing U.S. federal government shutdown.) There are, of course, even wilder possibilities—it may be that this event represents some fantastic and heretofore entirely unwitnessed phenomenon. Over time, the gravitational interactions would cause the remaining star and its black hole companion to spiral together, culminating with a spectacular merger that would unleash unprecedented astrophysical fireworks. Astronomers are still busy unraveling the full picture of where this space oddity fits in our strange and exciting universe. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03617-0'>Chinese scientists increasingly lead joint projects with the UK, US and Europe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 12:20:04
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). The number of Chinese scientists taking on leadership roles in international science projects is growing rapidly. But China needs to strengthen its leadership capabilities in disruptive basic research, “as Nobel-level original breakthroughs remain rare”, he adds. When such statements weren't available, the team developed a model that could predict leadership roles based on author experience, citation histories and the ideas researchers brought from their previous work, says James Evans, a co-author and computational sociologist at the University of Chicago in Illinois. Researchers who conceived, designed and guided the project or offered mentorship were classified as leaders; first-year students, people providing tech support and those performing experiments under direction were considered followers. Lead share describes the number of people in leadership roles from a given country. Chinese scientists still occupy supporting roles in many of these projects, says Evans. According to the research, in 2019, China's lead share reached parity with the United Kingdom's; it is likely to be on par with Europe lead share by 2025–27 and the US lead share by 2027–28. In some crucial technology areas, such as artificial intelligence, semiconductors and energy, Chinese leadership is further behind and is expected to catch up to the United States by 2030. China's dependency on US tech can not be easily overcome, as it is rooted in decades of deep scientific research, says Xiang. “Addressing these structural imbalances is the key to our journey from being a big science country to a truly strong one, and continued international collaboration remains vital for this endeavour,” Xiang adds. Simulations show that “if the US were to stop collaborating with China on projects related to critical technologies — such as space, artificial intelligence or quantum computing — it would prove to be very costly for the US”, he says. Evans says that US–China collaborations, particularly in fields such as artificial intelligence, are more likely to be successful than work done separately. Make science more collegial: why the time for ‘adversarial collaboration' has come Google Scholar-based tool gives extra credit to first and last authors Low-quality papers are flooding the cancer literature — can this AI tool help to catch them? AI tools could reduce the appeal of predatory journals How scientists are pushing back against Trump's funding ‘deal' for universities China should undertake more risky research to close the Nobel gap Japan's first female prime minister doesn't call herself a feminist — but the country needs her to tackle sexism in science My funding applications are taking up too much time. Make science more collegial: why the time for ‘adversarial collaboration' has come An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251104094144.htm'>Scientists uncover the secret triggers of ‘impossible' earthquakes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-05 02:39:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Earthquakes in places like Utah (USA), Soultz-sous-Forêts (France), and Groningen (the Netherlands) seem puzzling to scientists because, according to geological theory, they shouldn't be possible. In these regions, the shallow layers of the Earth's crust are thought to behave in a way that strengthens faults when they begin to move. Textbooks suggest that this strengthening effect should prevent earthquakes from happening at all. Researchers from Utrecht University set out to understand why. Their findings, recently published in Nature Communications, reveal that faults which have remained inactive for millions of years can accumulate extra stress over time. Eventually, that built-up pressure may be released in a single event. Faults in the shallow subsurface are usually stable, so we do not expect shock movements to occur along them," explains Dr. Ylona van Dinther, who led the study. These shallow earthquakes are often linked to human activities such as drilling, extraction, or fluid injection. The question, then, is why faults that normally grow stronger when they move can suddenly weaken and slip, releasing energy as an earthquake. Many human-induced earthquakes occur along ancient, inactive faults that have not shifted for millions of years. When that resistance is finally overcome, it can cause an abrupt acceleration along the fault. Because areas like these have no long-term record of seismic activity, local communities are often unprepared. Buildings and infrastructure are not designed to handle the shaking. "Furthermore, these earthquakes take place at a depth where human activities occur, in other words, no more than several kilometres deep. That is considerably less deep than the majority of natural earthquakes." This shallowness means that such quakes can cause more noticeable and potentially damaging ground movement. Interestingly, the Utrecht team found that these earthquakes are one-off events. "As a result, there is no more earthquake activity at that spot," says Van Dinther. When a fault strengthens as it moves, its broken sections can slide more easily past one another afterward, acting as natural barriers that prevent larger earthquakes from forming. It shows that even in regions considered geologically stable, earthquakes can occur under certain conditions -- but only once per fault. Understanding how faults behave, how they "heal," and what causes them to accelerate or slow down is essential for minimizing seismic risks associated with geothermal energy, carbon storage, and similar technologies. Scientists Create an Artificial “Leaf” That Turns CO₂ Into Useful Products Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            