
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-05</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-04080-0'>A minimally invasive dried blood spot biomarker test for the detection of Alzheimer's disease pathology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 16:11:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Blood biomarkers have emerged as accurate tools for detecting Alzheimer's disease (AD) pathology, offering a minimally invasive alternative to traditional diagnostic methods such as imaging and cerebrospinal fluid (CSF) analysis. Yet, the logistics surrounding venipuncture for blood collection, although considerably simpler than the acquisition of imaging and CSF, require precise processing and storage specific to AD biomarkers that are still guided by medical personnel. The DROP-AD project investigates the potential of dried plasma spot (DPS) and dried blood spot (DBS) analysis, derived from capillary blood, for detecting AD biomarkers, including phosphorylated tau at amino acid 217 (p-tau217), glial fibrillary acidic protein and neurofilament light. Here, 337 participants from 7 centers were included, with 304 participants providing paired capillary DPS or DBS and venous plasma samples. We observed strong correlations between DPS p-tau217 and venous plasma p-tau217 (rS = 0.74, P < 0.001). DPS p-tau217 progressively increased with increasing disease severity, and showed good accuracy in predicting CSF biomarker positivity (area under the curve = 0.864). Similarly, we demonstrated the successful detection of glial fibrillary acidic protein and neurofilament light with strong correlations between DBS and DPS, respectively, using paired venous plasma samples. Notably, the method was also effective in individuals with Down syndrome, a population at high genetic risk for AD but in whom standard blood sampling by venipuncture may be more complicated, revealing elevated biomarkers in those with dementia compared with asymptomatic individuals. The study also explored unsupervised blood collection, finding high concordance between supervised and self-collected samples. These findings underscore the potential of dried blood collection and capillary blood as a minimally invasive, scalable approach for AD biomarker testing in research settings. Yet, further refinement of collection and analytical protocols is needed to fully translate this approach to be viable and useful as a clinical tool. In less than a decade, the development of blood biomarkers for the identification of AD pathology has transitioned from a promising research endeavor to a valued tool that is now included in research diagnostic criteria1 and is increasingly being adopted in clinical practice. Phosphorylated tau at amino acid 217 (p-tau217) has emerged as an early and accurate AD blood biomarker2,3, offering higher accuracy compared with other putative blood biomarkers for detecting cerebral amyloid-β (Aβ) pathology4, a required hallmark for an AD diagnosis5,6. Several blood p-tau217 assays, spanning different immunological detection methods and mass spectrometry techniques7 are now available with some—but not all—meeting the recommended criteria for clinical usefulness, approved for clinical use, or currently under regulatory evaluation8. Thus, a cost-effective and timely tool is now available to identify individuals who may benefit from approved and emerging treatments or to monitor disease progression9. Specifically, the most likely clinical applications of blood p-tau217 will be based on a two-cutoff approach10 aimed at identifying people at either very high or very low risk of brain amyloidosis and for whom additional biomarker investigations are unnecessary, thereby lowering the need for positron emission tomography (PET) or CSF testing11. Other supportive blood biomarkers also offer insights into disease pathophysiology. Specifically, glial fibrillary acidic protein (GFAP), a marker of astrogliosis, has been associated with the onset of Aβ deposition12,13; and neurofilament light (NfL), a marker for axonal degeneration across neurodegenerative diseases14, has also been developed and is widely deployed in research and some clinical and therapeutic settings. Although current guidelines recommend AD blood biomarker testing for symptomatic individuals15, there is also the potential to screen cognitively unimpaired (CU) older adults using a simplified test in a research setting and prevention strategies. Moreover, broader implementation of blood biomarkers will likely substantially advance the treatment, management and biological understanding of AD and related disorders in populations and communities currently underrepresented in research; for example, in individuals with Down syndrome (DS). Substantial efforts have been made to ensure that blood tests become widely accessible, rather than confined to specialized laboratories. A major advancement in this area is the development of high-performing commercial and fully automated immunoassays for AD blood biomarkers, in particular p-tau21717. These fully automated immunoassays demonstrate performance identical or almost identical to immunoprecipitation mass spectrometry, as shown in a series of papers17,18,19,20,21. Although immunoprecipitation mass spectrometry remains difficult for widespread research and clinical implementation because of its high costs and limited instrument availability, automated immunoassays offer reliable and scalable solutions that address these limitations. Yet, for their global adoption to be fully realized, logistical challenges surrounding blood collection—such as the need for timely and standardized sample handling and storage22, as well as limited access to phlebotomy services—must be overcome to avoid constraining the impact of blood-based biomarker testing. The DROP-AD project aims to streamline blood sample collection for larger-scale research, therapeutic trial enrollment and, potentially, clinical care, by introducing an alternative method that addresses the logistical challenges of traditional venipuncture blood collection and processing. By using capillary blood collected on DBS or DPS cards, the latter needed for p-tau217, reliance on guided venipuncture, immediate centrifugation and temperature-controlled shipment is eliminated. This approach enables a simplified, and potentially self-administered, protocol using fingertip blood collection. We have previously demonstrated the feasibility of measuring AD biomarkers from dried blood spots23, with the blood source being venous, followed by manual transfer onto a card for shipping and storage advantages. Here we extend this previous work by evaluating the feasibility of remote biomarker assessment using capillary blood, obtained by fingerstick collection, thus potentially self-collected and remote. A total of 337 participants were recruited across 7 European centers to assess the quantification of key AD pathology and neurodegeneration biomarkers—plasma p-tau217, NfL and GFAP—from capillary-derived blood collected from the finger. These values were directly compared with those obtained by standard venous plasma sampling, as well as to CSF biomarker concentrations routinely used in clinical diagnostics. age, 70.8 (11.7) years; 167 women (53.4%) (Table 1) from 7 centers. Depending on the research site, and the evolution of the DROP-AD project, participants followed different capillary blood collection and testing procedures, which are summarized in Fig. Cohort characteristics are shown in Supplementary Table 1. For DPS and DBS sample collection, a finger prick was carried out by trained study personnel and a few drops of capillary blood were spotted onto DPS and DBS collection devices. DPS and DBS were collected via semi-automated spot collectors and incubated with analyte-specific extraction buffer in a 96-well filter plate. After incubation and centrifugation, the eluate was immediately measured using ultrasensitive immunoassays on the single molecule array platform. b–f, Participant numbers and collection device numbers per cohort: capillary p-tau217 (b), capillary GFAP (c), capillary NfL (d), DS cohort (e) and self-sampling cohort (f). We found a high correlation between DPS p-tau217 and venous plasma p-tau217 across all merged cohorts (Spearman's rank correlation (rS) = 0.74, 95% confidence interval (CI) 0.678–0.791; P < 0.001) (Fig. The strength of this correlation varied among participating centers (Extended Data Fig. 1) and was highest in the Gothenburg cohort (rS = 0.904, 95% CI 0.731–0.968; P < 0.0001) and the Brescia cohort (rS = 0.838, 95% CI 0.694–0.917; P < 0.0001), followed by the Exeter (rS = 0.765, 95% CI 0.530–0.891; P < 0.0001) and Barcelona (rS = 0.735, 95% CI 0.646–0.805; P < 0.0001) cohorts, and the Malmö cohort (rS = 0.429, 95% CI 0.159–0.640; P < 0.001), the only cohort in which a different assay (Lilly2) for venous plasma was used. To further demonstrate the strength the relationship between capillary and venous blood, we stratified plasma p-tau217 concentrations into tertiles and computed Spearman correlations in each tertile, allowing us to assess agreement at low, medium and high concentrations (Extended Data Fig. Significant correlations were observed for p-tau217 concentrations in tertile two (rS = 0.51; P < 0.0001) and tertile three (rS = 0.62; P < 0.0001), but no relationship in tertile one, where all participants were CSF Aβ42/p-tau181 negative. The strength of the relationship between capillary and venous blood was not confounded by age or sex (Supplementary Table 2). a, Correlation between capillary p-tau217 and venous plasma p-tau217 (n = 252). Dots correspond to individual data points. b, Correlation between capillary p-tau217 and MMSE (n = 209). c, Correlation between capillary p-tau217 and age (n = 249). A mean regression line is presented in all panels, with ribbons representing 95% CI. For numerical representation of the correlation, we present Spearman coefficients alongside their P values. Next, we investigated the association of capillary biomarkers with cognitive testing. DPS p-tau217 showed significant correlations with both Mini Mental State Evaluation (MMSE) (n = 209; rS = −0.374, 95% CI −0.485 to −0.251; P < 0.0001) (Fig. 2b) and age (n = 249; rS = 0.334, 95% CI 0.219–0.440; P < 0.0001) (Fig. 2c), which were similar to the correlations of venous plasma p-tau217 with MMSE (n = 209; rS = −0.410, 95% CI −0.517 to −0.290; P < 0.0001) (Fig. 2b) and age (n = 249; rS = −0.317, 95% CI 0.200 to 0.424; P < 0.0001) (Fig. DPS p-tau217 was significantly increased in clinically defined mild cognitive impairment (MCI) and AD (no biomarker classification) compared with CU participants and clinically defined non-AD dementias (Fig. Next, we investigated the discriminative accuracy of DPS p-tau217 to detect abnormal CSF biomarkers. In participants with DPS p-tau217, venous plasma p-tau217 and CSF Aβ42/p-tau181 (n = 176; mean (s.d.) age, 74.6 (7.9) years; 102 women (58.0%)), capillary DPS p-tau217 was significantly increased (+198%; P < 0.001) in the AD CSF biomarker-positive group (Fig. DPS p-tau217 had an area under the curve (AUC) of 0.863 (95% CI 0.809–0.917); however, this was significantly lower than venous plasma p-tau217 which had an AUC of 0.982 (95% CI 0.968–0.996; P < 0.0001) (Extended Data Fig. We also show the distribution of capillary p-tau217 across clinico-biological diagnostic groups (Fig. 3c), which shows a similar pattern to venous derived p-tau217, with similar statistical significance across groups (Extended Data Fig. Results demonstrating DPS p-tau217 against CSF Aβ42/Aβ40 as the standard of truth are shown in Extended Data Fig. Next, we tested the accuracy of capillary DPS p-tau217 to determine abnormal venous plasma p-tau217 (n = 252), which had predetermined cutoff validated against Aβ-PET (ALZpath single molecule array (Simoa) > 0.42 pg ml−1)3. DPS p-tau217 was more concordant with venous plasma p-tau217 and was increased by 217% in individuals with venous plasma p-tau217 > 0.42 pg ml−1, compared with individuals with venous plasma p-tau217 ≤ 0.42 pg ml−1, and had a discriminative accuracy to detect abnormal venous plasma p-tau217 of 0.868 (95% CI 0.825–0.911) (Extended Data Fig. Horizontal solid-line bars represent group-wise comparisons alongside P values, obtained from post-hoc contrasting of a linear model adjusted for age and sex. In all panels, individual data points for each participant are shown and an overlaid boxplot represents group-wise distributions. Boxplots show the median (center line), interquartile range (IQR; box limits, 25th–75th percentiles), whiskers extending to the most extreme values within 1.5× IQR from the quartiles. A mean regression line is presented with ribbons representing 95% CI. Statistical tests were two-sided, and for group comparisons Tukey's adjustment was used. Exploratory diagnostic accuracy metrics were derived in a subset of individuals (n = 176) with paired capillary and CSF Aβ42/p-tau181 metrics, at a prevalence of 56.3% of CSF biomarker positivity. A capillary p-tau217 cutoff of 0.01 pg ml−1, with 90% sensitivity for abnormal CSF Aβ42/p-tau181, led to a positive predictive value (PPV) of 0.738 (95% CI 0.653–0.808) and a negative predictive value (NPV) of 0.833 (95% CI 0.713–0.910), at a specificity of 58.4%. age, 69.9 (10.8) years; 99 women (48.8%)) measured using at least one of the three candidate DBS methods, we found that B50 and Telimmune collection cards were most compatible for GFAP and were combined for this analysis (Extended Data Fig. When comparing GFAP levels from capillary samples with venous plasma, a strong correlation was found (r = 0.773, 95% CI 0.710–0.823; P < 0.0001) (Fig. 4b) and MMSE (capillary GFAP, r = −0.448, 95% CI −0.558 to −0.324; P < 0.0001; venous plasma, r = −0.436, 95% CI −0.547 to −0.310; P < 0.0001) (Fig. A mean regression line is presented in all panels, with ribbons representing 95% CI. For numerical representation of the correlation, we present Spearman coefficients alongside their P values. Statistical tests were two-sided, and for group comparisons Tukey's adjustment was used. Based on a set with 237 individuals measured with at least one of the three DBS method candidates, only Telimmune DPS cards were useful in examining capillary NfL using our protocol (Extended Data Fig. Therefore, we examined 72 participants for NfL using Telimmune DPS cards (mean (s.d.) When comparing NfL levels from capillary DPS to venous plasma, a strong correlation was observed (r = 0.83, 95% CI 0.743–0.892; P < 0.0001) (Fig. Similarly to GFAP, we observed similar correlations between DPS and venous plasma NfL in relation to age (capillary DPS, r = 0.429, 95% CI 0.219–0.601; P < 0.001; venous plasma, r = 0.524, 95% CI 0.333–0.674; P < 0.0001) (Fig. 4e) and MMSE (capillary DPS, r = −0.269, 95% CI −0.471 to −0.039; P = 0.02; venous plasma, r = −0.367, 95% CI −0.552 to −0.148; P < 0.001) (Fig. We examined 31 participants with DS and DBS biomarker data. As with the euploid participants, we found a significant relationship between biomarkers measured in capillary blood and venous blood (p-tau217, r = 0.875, 95% CI 0.503–0.973 (Fig. Capillary biomarker levels were, as also shown in venous plasma (Fig. 5e,f), increased in DS with dementia (dDS), compared with DS without AD-related cognitive impairment (aDS), for both p-tau217 (Fig. Participants positive for CSF p-tau181/Aβ42 more often had higher levels of capillary GFAP (Fig. 5f), although there were not sufficient participants with DS and DBS p-tau217 and CSF biomarker data (n = 5, CSF-negative only). a, Scatterplot representing the association between capillary and venous plasma p-tau217 in the DS Barcelona cohort, alongside their Spearman correlation coefficient and associated P value (n = 9). c, Scatterplot representing the association between capillary and venous plasma GFAP, with a Spearman correlation coefficient and its associated P value presented (n = 30). f,g, Boxplots of capillary GFAP (f) and venous plasma GFAP (g) based on CSF Aβ42/p-tau181 status (CSF-negative, n = 6; CSF-positive, n = 7). For scatterplots, a mean regression line is presented with 95% CI. When group comparisons are presented with boxplots, horizontal solid-line bars represent group-wise comparisons alongside P values, obtained from post-hoc contrasting of a linear model adjusted for age and sex. Statistical tests were two-sided, and for group comparisons Tukey adjustment was used. In the previous result sections, all capillary DPS or DBS collection was supervised and guided by trained personnel. Here we evaluated the within-person difference if collection was supervised compared with unsupervised. In 30 participants, capillary blood guided by study personnel and self-collected unsupervised samples showed a very high concordance with little difference between timepoints (DPS p-tau217, 0.014 pg ml−1 versus 0.013 pg ml−1, P = 0.57 (Extended Data Fig. Because only one Telimmune card was sampled per participant dedicated to p-tau217 quantification, no NfL data were obtained. The DROP-AD project, constituting an effort to assess biomarkers for AD-type pathology and neurodegeneration from capillary blood, showcases the capability of quantifying p-tau217, GFAP and NfL protein levels. The study evaluated straightforward capillary blood collection methods, a new extraction protocol and ultrasensitive immunoassay biomarker determination. Biomarker levels from capillary blood correlated well with conventional venipuncture-collected plasma measures, and in the case of p-tau217, predicted with good accuracy, abnormal AD CSF biomarkers, as demonstrated in individuals classified as asymptomatic, MCI, dementia, as well as in individuals with DS, who are at high-risk for AD. In blood, p-tau217 is the principal blood biomarker for determining AD pathology8 and is increasingly adopted as a reliable metric in research, clinical trials and clinical practice. It has the capabilities of high diagnostic accuracy to detect AD pathology, primarily amyloid3,24, but is also tightly associated with severity of tau pathology assessed by tangle counts at post-mortem examination2 and by tau PET during life25 not only in the symptomatic phase of the disease4, but also in the asymptomatic phase26. Therefore, p-tau217 holds promise not only for clinical use, but also population-level screening, identifying at-risk individuals in preclinical phases and enabling early intervention strategies27. Plasma p-tau217 has already been used to assess outcomes in secondary preventive trials16,28. A drawback in expanding blood biomarker testing outside specialized centers, is the strict protocol and guided venipuncture collection, sample handling and shipment. Dried blood sampling23 overcomes this limitation by enabling simplified, minimally invasive and potentially, remote self-collection, reducing the need for specialized personnel and facilitating broader population access to biomarker testing. The DROP-AD project, conducted across multiple centers, highlights the strong potential of using dried capillary blood samples to accurately quantify plasma p-tau217. We observed robust correlations between p-tau217 concentrations measured from DPS and matched venous plasma samples, although the strength of these correlations varied by site. Importantly, p-tau217 levels showed a stepwise increase across clinical stages—CU, MCI and AD—and demonstrated good accuracy in predicting CSF biomarker-confirmed AD pathology. In addition to p-tau217, we successfully quantified GFAP and NfL using DBS and DPS matrices, respectively. Although our primary focus was on biomarkers of AD neuropathology, the reliable detection of NfL from DPS samples has broader implications. Given its established role as a diagnostic, prognostic and monitoring biomarker, capillary-based NfL measurement could be transformative for other neurodegenerative and neurological conditions—including frontotemporal dementia, atypical parkinsonian syndromes, multiple sclerosis, amyotrophic lateral sclerosis and acute neurological injuries. Biomarker levels extracted from DPS or DBS cards, for all analytes of interest, were substantially lower than those quantified from venous plasma, which we believe is attributable to the elution of dried blood or plasma with buffer, resulting in dilution. Protein concentrations were not adjusted using a uniform dilution factor, because we cannot currently estimate the volume of plasma that is dried onto a card. Attempts to measure Aβ42 and Aβ40 using this technique yielded mixed results. Although Aβ40 was readily quantifiable, Aβ42 levels were mainly below the limit of detection and could not be included in the analysis, limiting the utility of this approach for this biomarker. Imaging, CSF and blood-based biomarkers for AD pathology have shown strong translational applicability in individuals with DS, which represents the most common genetically determined form of AD29. Given the near-universal risk of AD in this population, there is a critical need for scalable and accessible methods to enable longitudinal biomarker monitoring, particularly in the context of preventive and disease-modifying clinical trials. Collection of blood samples by standard venipuncture may be complicated in individuals with DS—for example, due to relatively high rates of institutionalization and a lack of professionals—and remote blood collection thus offers a promising solution by reducing reliance on in-clinic visits and facilitating broader participation across diverse spectrum of intellectual disability. To evaluate the feasibility of this approach, we conducted a pilot study in which capillary blood samples were successfully collected from individuals with DS across a spectrum of cognitive stages. Our results revealed significantly elevated levels of capillary-derived GFAP and p-tau217 in participants with symptomatic AD compared with those who were cognitively asymptomatic. Importantly, biomarker concentrations derived from capillary samples showed strong concordance with those obtained from matched venous plasma, supporting the reliability and translational potential of remote sampling for biomarker quantification and ultimately, AD diagnosis, in this high-risk population. First, we have indicated that capillary blood collection may be useful in an unsupervised fashion, remotely. This has not been fully examined in this proof-of-principle study, where all capillary sampling was performed in research centers guided by trained staff. To gain some initial insights, we conducted a pilot in 30 participants who provided two capillary samples: one sampled by research staff and one unsupervised—1 h later. Further, our venous plasma analyses were performed in single-batch analysis for all study sites, and this is particularly important to consider when comparing results directly to capillary testing, which was analyzed prospectively in multiple batches (less than 4 weeks from collection) throughout the 24-month study period. The observed lower accuracies to determine AD pathology by capillary p-tau217 could be partially attributed to this key difference in analytical design. This 24-month period also reflects a time of protocol optimization, in sample collection at multiple study sites and biomarker determination in the laboratory. This may, in part, reflect the inherent challenges of fingertip capillary blood sampling in clinical practice, where achieving consistent blood flow from a fingerstick collection is difficult and often complicated by hemolysis or admixture of interstitial fluid due to external compression of the fingertip30. We believe that diligent training of the study personnel and patients and/or caregivers and the provision of informational material is essential for successful collection of dried blood; however, alternative capillary blood collection methods—other than fingerstick—should be considered and examined given the encouraging finding from this study. Moreover, studies with larger cohorts are needed to investigate the impact of confounders on DPS or DBS biomarker levels. In conclusion, our findings demonstrate that dried blood analysis offers a feasible and scalable approach for detecting AD pathology, particularly in research, population-based and epidemiological contexts. This minimally invasive method has the potential to substantially broaden our understanding of the prevalence and distribution of AD pathology across the general population, while also facilitating the inclusion of historically underrepresented populations and geographically diverse regions in AD research. However, despite the promise shown, we do not currently recommend the use of dried blood analysis for clinical use, decision-making or patient management, because of observed differences in analytical performance and diagnostic accuracy between capillary-derived and venous blood samples. To evaluate the feasibility of capillary-derived blood as a simplified collection method compatible with AD biomarker analysis, paired venous plasma and capillary blood samples obtained by fingerstick were collected from CU and cognitively impaired individuals across seven European study centers. Capillary blood collection was conducted by trained study personnel at each site. Dried blood cards were shipped without temperature control to the Neurochemistry Laboratory at the University of Gothenburg, Sweden, within 1–40 days of collection. At each study site, all participants provided written informed consent before enrollment, and the studies were approved by local ethical review authorities. The inclusion criteria for each cohort are depicted below and summarized in Supplementary Table 1. Participants were not compensated for participation in this study. Biological sex was determined based on self-identification. The Ace Alzheimer Center Barcelona, Spain (the ‘Barcelona' cohort) included participants under investigation for cognitive complaints recruited between September 2022 and April 2024. At Fundació ACE, clinical diagnosis was carried out through a comprehensive neuropsychological evaluation using the NBACE battery31, assessment of functional status with the Clinical Dementia Rating (CDR) scale, and supported by biological diagnosis through CSF biomarkers following the AT(N) classification framework32. Individuals with MCI and dementia were offered a voluntary (and informed consented) lumbar puncture in accordance with established consensus recommendations. Venous plasma, CSF and capillary DPS or DBS samples were collected on the same day under fasting conditions. All biospecimens obtained were part of the ACE collection, which was registered in Instituto de Salud Carlos III (ISCIII, Ministry of Health of Spain) under the code C.0000299. Capillary DPS and DBS samples, venous plasma and CSF samples were collected at the same study visit. Cognitive testing (MMSE and CDR) was performed in each participant. Ethical approval for H70 Clinical Studies was provided by The Swedish Ethical Review Authority (Etikprövningsmyndigheten; EPM: 2023-06137-02). In the BioFINDER Primary Care (NCT06120361) and BioFINDER Preclinical AD (NCT06121544) studies (the ‘Malmö' cohort), cognitively asymptomatic volunteers (asymptomatic AD or healthy controls) and individuals with cognitive symptoms undergoing cognitive diagnostic evaluation in primary care were included between December 2023 and November 2024. The exclusion criteria were (1) not undergoing CSF or blood sampling as part of clinical practice and (2) not undergoing cognitive testing as part of clinical practice. Cognitive testing (MMSE) and CSF samples were available for each participant. Capillary DPS and DBS were collected at the same day as venous plasma samples, stored at room temperature and shipped to the Neurochemistry Laboratory between 1 and 7 days after the collection. The studies were approved by Swedish Ethical Review Authority (Dnr. Participants enrolled at the Center for Neurodegenerative Disorders at the University of Brescia, Italy (the ‘Brescia' cohort) met current clinical criteria for the diagnosis of fontotemperal dementia33,34 or AD35, or were healthy individuals recruited among spouses or family members. Consecutive recruitment took place between October 2023 and June 2024. Cognitive testing (MMSE and CDR) was available for each participant and CSF samples were collected in a subgroup. Participants enrolled at the University of Exeter Medical School (the ‘Exeter' cohort) were adults aged 50 years or above with a body mass index >25 kg m−2 and within 2 h travel of Exeter consecutively recruited from PROTECT-UK (Platform for Research Online to investigate Cognition and Genetics in Ageing) taking part in the DailyColors polyphenol supplement study in January 2024 (ref. Exclusion criteria were the diagnosis of dementia and participation in an interventional clinical trial. Capillary dried blood samples were stored at room temperature and shipped to the Neurochemistry Laboratory between 1 and 10 days after collection. Participants under investigation of a neurodegenerative disease from the memory clinic at Rigshospitalet, Copenhagen University Hospital (the ‘Copenhagen' cohort) were enrolled between May 2024 and July 2024. Individuals were excluded if they did not consent to the Danish Dementia Biobank, if the lumbar puncture was unsuccessful, or if they were clinically evaluated as incapable of participating in the project. Capillary dried blood samples were stored at room temperature and shipped within 7–40 days after collection. This study was approved by the Danish Research Ethics Committee (Ref. For the Sant Pau cohort, participants with DS with (dDS or prodromal AD (pDS)) and without AD-related cognitive impairment (aDS) were consecutively recruited at the Sant Pau Memory Unit, Barcelona, Spain from the Down Alzheimer Barcelona Neuroimaging Initiative (DABNI) study between May 2024 and November 2024. Capillary DPS and DBS samples were collected at the same day as venous EDTA plasma samples and stored at room temperature and shipped within 1–14 days after collection. This study was approved by the Sant Pau Ethics Committee. All participants or their legally authorized representative gave written informed consent before enrollment. Three different dried blood spot collection devices were used in this study. Capitainer SEP-10 and the Telimmune Plasma Separation Card were used interchangeably to measure p-tau217. Capitainer B50 and the Telimmune Plasma Separation Card were utilized for GFAP quantification. Telimmune Plasma Separation Cards alone were used for NfL measurements based on comparative studies (Extended Data Fig. In all cohorts, capillary blood was collected by study personnel from the middle or index finger using a single-use lancet with a 1.5-mm wide and 2.0-mm deep cut. In the Exeter cohort, a secondary unsupervised capillary blood collection was carried out by all participants on the same day. The Capitainer B50 and Capitainer SEP-10 cards collect 50 μl and 70 μl of capillary whole blood, respectively. In the SEP-10 cards, blood cells are separated from the whole blood generating 10 μl of a plasma-like sample. Whole blood and plasma-like spots are left to dry at room temperature for 30 min. The Telimmune Plasma Separation Card does not restrict blood volume and 50 μl of capillary whole blood, equivalent to 6 μl of plasma-like sample, was pipetted from the finger to the card to standardize collection volume. Telimmune Plasma Separation Cards were left to dry for 3 min, then the cell separation membrane layer was removed, and the samples were left to dry for additional 30 min. After drying, all cards were stored at room temperature and shipped without temperature control or cooling on a regular basis to the Neurochemistry Laboratory, Gothenburg, Sweden. Before analysis, one filter disk from Capitainer B50 and Capitainer SEP-10 cards and both filter disks from the Telimmune Plasma Separation Cards were removed from the card using a semi-automated spot collector (Capitainer AB), and transferred to a deep 96-well plate (Sirocco protein precipitation plate, Waters). Samples were then incubated shaking with 170–300 μl of protein extraction buffer depending on the analyte of interest and at 37 °C and 500 rpm for 30 min; for p-tau217 quantification, filter papers were eluted with 170 μl buffer for all card types (Quanterix, catalogue number 105909); for N2PE and N4PE assays, Capitainer B50 filter papers were eluted with 300 μL and Telimmune filter papers with 170 μl of analyte-specific buffer (Quanterix, catalogue numbers 103659 (N4PE) and 103516 (N2PB)). After incubation, the samples were centrifuged at 20 °C and 2,626g for 15 min and the eluate was collected in a conical 96-well plate (Quanterix). After spinning, the filter disks and protein precipitation plate were discarded, and the eluate was immediately used for biomarker analysis in singlecates by Simoa technology on the HD-X platform using a neat protocol (ALZpath Simoa pTau-217 v2 Assay3 (Quanterix, catalogue number 104371), Simoa Neurology 4-plex E (Quanterix, catalogue number 103607) or Simoa Neurology 2-plex B (Quanterix, catalogue number 103520). Venous plasma samples collected by venipuncture and CSF collected by lumbar puncture are summarized for each cohort37,38,39,40,41,42. Before immunoassay procedures, venous EDTA plasma samples were thawed for 45 min at room temperature, vortexed for 30 s at 2,000 rpm and centrifuged at 4,000g and 20 °C for 10 min. Venous plasma samples were analyzed in singlecates with the same immunoassays using standard operating procedures, except for the Malmö cohort where venous plasma p-tau217 was quantified using an immunoassay on the Meso Scale Discovery platform developed by Lilly2. For the Gothenburg cohort, paired capillary blood extracts and venous plasma samples were measured in the same experiment. In all other cohorts, for logistic reasons, capillary blood extracts were measured prospectively with the accompanying venous plasma samples measured in a single batch at the end of the study. High and low dried blood quality controls were developed for p-tau217, GFAP and NfL on Capitainer SEP-10 filter disks. Dried blood quality controls were extracted on the day of each experiment and measured in duplicates at the beginning and end of each plate. Moreover, additional high and low venous plasma controls were run in duplicates at the beginning and end of each plate. To test the potential of capillary dried blood collection as self-sampling method, participants in the Exeter cohort underwent a second independently performed sampling session using Capitainer SEP-10, Capitainer B50 and Telimmune cards. During the first collection carried out by the study personnel, participants observed the sampling process and were handed written instructions (short text boxes and pictograms). Participants were then left alone with the instructions and performed the capillary dried blood collection independently (n = 44 pairs for DPS p-tau217; n = 18 pairs for DBS GFAP). For visualizing associations between capillary dried blood spot biomarkers and variables of interest, we used scatterplots and presented a mean linear regression line to represent trends in associations. To numerically quantify and compare these associations, we used Spearman's rho. To visualize between-group differences in DBS biomarkers, we plotted individual data points overlaid with boxplots. When group comparisons were made, we used linear models adjusted for age, sex and center (when applicable), and obtained P values from post-hoc Tukey contrasts between the levels of categorical variables. P values and 95% CIs were presented or described when appropriate. When evaluating biomarker discriminative ability for binary outcomes such as CSF biomarker positivity, we computed the AUC of receiver operating characteristics and used the DeLong test when receiver operating characteristic curves were compared. When evaluating diagnostic properties of DPS p-tau217, we assessed cutoffs derived with 90% sensitivity, 90% specificity or maximum Youdens' Index for CSF or venous plasma biomarker positivity and computed their respective NPV and PPV based on the prevalence of biomarker positivity in the subset of participants with available data for each analysis. Statistical significance was set as a two-sided alpha = 0.05. We did not control for multiple comparisons, and statistical significance was interpreted taking this into consideration. All analyses were performed in R v.4.2.1 (2022-06-23) on macOS 15.6.1. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. This study includes no data deposited in external repositories. Blinded and anonymized data can be shared with academic investigators, for the sole purpose of replicating procedures and results presented in the article, as long as data transfer agrees with local legislation and with the local Ethical Review Board of each cohort, which must be regulated in a material or data transfer agreement. Data requests will be evaluated based on scientific merit and compliance with ethical and legal requirements; requests are typically processed and accepted within 2–3 months. Source data are provided with this paper. All analyses were performed in R v.4.2.1 (2022-06-23) on macOS 15.6.1. The R code that supports the main results of this study is publicly available on GitHub (https://github.com/wsbrum/dropad_natmed). All models were built using publicly available packages and functions in the R programming language. Jack, C. R. Jr et al. Revised criteria for the diagnosis and staging of Alzheimer's disease. Palmqvist, S. et al. Discriminative accuracy of plasma phospho-tau217 for Alzheimer disease vs other neurodegenerative disorders. Diagnostic accuracy of a plasma phosphorylated tau 217 immunoassay for Alzheimer disease pathology. Jack, C. R. Jr et al. Revised criteria for diagnosis and staging of Alzheimer's disease: Alzheimer's Association Workgroup. Dubois, B. et al. Alzheimer disease as a clinical–biological construct—an international working group recommendation. The Alzheimer's Association Global Biomarker Standardization Consortium (GBSC) plasma phospho-tau round robin study. Acceptable performance of blood biomarker tests of amyloid pathology – recommendations from the Global CEO Initiative on Alzheimer's Disease. Repeated plasma p-tau217 measurements to monitor clinical progression heterogeneity. Brum, W. S. et al. A two-step workflow based on plasma p-tau217 to screen for amyloid beta positivity with further confirmatory testing only in uncertain cases. Bellaver, B. et al. Astrocyte reactivity influences amyloid-beta effects on tau pathology in preclinical Alzheimer's disease. Pontecorvo, M. J. et al. Association of donanemab treatment with exploratory plasma biomarkers in early symptomatic Alzheimer disease: a secondary analysis of the TRAILBLAZER-ALZ randomized clinical trial. Ashton, N. J. et al. A multicentre validation study of the diagnostic value of plasma neurofilament light. The AHEAD 3-45 study: design of a prevention trial for Alzheimer's disease. Plasma phospho-tau217 for Alzheimer's disease diagnosis in primary and secondary care using a fully automated platform. Warmenhoven, N. et al. A comprehensive head-to-head comparison of key plasma phosphorylated tau 217 biomarker tests. Highly accurate blood test for Alzheimer's disease is similar or superior to clinical cerebrospinal fluid tests. Meyer, M. R. et al. Clinical validation of the PrecivityAD2 blood test: a mass spectrometry-based test with algorithm combining %p-tau217 and Aβ42/40 ratio to identify presence of brain amyloid. Montoliu-Gaya, L. et al. Optimal blood tau species for the detection of Alzheimer's disease neuropathology: an immunoprecipitation mass spectrometry and autopsy study. Characterization of pre-analytical sample handling effects on a panel of Alzheimer's disease-related blood-based biomarkers: results from the Standardization of Alzheimer's Blood Biomarkers (SABB) working group. Huber, H. et al. Biomarkers of Alzheimer's disease and neurodegeneration in dried blood spots—a new collection method for remote settings. Mattsson-Carlgren, N. et al. Longitudinal plasma p-tau217 is increased in early stages of Alzheimer's disease. Comparison of two plasma p-tau217 assays to detect and monitor Alzheimer's pathology. Prevalence of Alzheimer's disease pathology in the community. Sims, J. R. et al. Donanemab in early symptomatic Alzheimer disease: the TRAILBLAZER-ALZ 2 randomized clinical trial. Fortea, J. et al. Alzheimer's disease associated with Down syndrome: a genetic form of dementia. Alegret, M. et al. Cut-off scores of a brief neuropsychological battery (NBACE) for Spanish individual adults older than 44 years old. Orellana, A. et al. Establishing in-house cutoffs of CSF Alzheimer's disease biomarkers for the AT(N) stratification of the Alzheimer Center Barcelona Cohort. Classification of primary progressive aphasia and its variants. The diagnosis of dementia due to Alzheimer's disease: recommendations from the National Institute on Aging–Alzheimer's Association workgroups on diagnostic guidelines for Alzheimer's disease. Effects of the DailyColors polyphenol supplement on serum proteome, cognitive function, and health in older adults at risk of cognitive and functional decline. Cano, A. et al. Clinical value of plasma pTau181 to predict Alzheimer's disease pathology in a large real-world cohort of a memory clinic. Clemmensen, F. K. et al. Short-term variability of Alzheimer's disease plasma biomarkers in a mixed memory clinic cohort. Predicting survival rate by plasma biomarkers and clinical variables in syndromes associated with frontotemporal lobar degeneration. Evaluation of two plasma-based proteotyping assays against APOE epsilon4 genotyping in a memory clinic setting: The Gothenburg H70 Clinical Studies. Performance of fully automated plasma assays as screening tests for Alzheimer disease-related beta-amyloid status. Wangdi, J. T. et al. Tart cherry supplement enhances skeletal muscle glutathione peroxidase expression and functional recovery after muscle damage. has received grants from Demensförbundet, Adlerbertska forskningsstiftelsen, Anna-Lisa och Bror Björnssons Stiftelse and the German Research Foundation under Germany's Excellence Strategy (grant number EXC2151-390873048). reports grants from the Fondo de Investigaciones Sanitario (FIS), Instituto de Salud Carlos III (grant numbers PI18/00335, PI22/00758, ICI23/00032) and the CIBERNED program (Program 1, Alzheimer Disease and SIGNAL study, www.signalstudy.es), partly jointly funded by Fondo Europeo de Desarrollo Regional, Unión Europea, Una manera de hacer Europa; Alzheimer's Association (grant number AARG-22-973966), Global Brain Health Institute (grant number GBHI_ALZ-18-543740), Jérôme Lejeune Foundation (grant number 1913 cycle 2019B) and Societat Catalana de Neurologia (grant number SCN2020). is supported by the Swedish Research Council (grant numbers 2017-00915 and 2022-00732), the Swedish Alzheimer Foundation (grant numbers AF-930351, AF-939721, AF-968270 and AF-994551), Hjärnfonden, Sweden (grant numbers ALZ2022-0006, FO2024-0048-TK-130 and FO2024-0048-HK-24), the Swedish state under the agreement between the Swedish government and the county councils, the ALF-agreement (grant numbers ALFGBG-965240 and ALFGBG-1006418), the European Union Joint Program for Neurodegenerative Disorders (grant number JPND2019-466-236), the Alzheimer's Association 2021 Zenith Award (grant number ZEN-21-848495), the Alzheimer's Association 2022-2025 Grant (grant number SG-23-1038904 QC), La Fondation Recherche Alzheimer (FRA), Paris, France, the Kirsten and Freddy Johansen Foundation, Copenhagen, Denmark, Familjen Rönströms Stiftelse, Stockholm, Sweden and an anonymous filantropist and donor. is a Wallenberg Scholar and a Distinguished Professor at the Swedish Research Council supported by grants from the Swedish Research Council (grant numbers 2023-00356, 2022-01018 and 2019-02397), the European Union's Horizon Europe research and innovation programme under grant agreement number 101053962, Swedish State Support for Clinical Research (grant number ALFGBG-71320), the Alzheimer Drug Discovery Foundation (ADDF), USA (grant numbers 201809-2016862), the AD Strategic Fund and the Alzheimer's Association (grant numbers ADSF-21-831376-C, ADSF-21-831381-C, ADSF-21-831377-C and ADSF-24-1284328-C), the European Partnership on Metrology, co-financed from the European Union's Horizon Europe research and innovation programme and by the participating States (NEuroBioStand, grant number 22HLT07), the Bluefield Project, Cure Alzheimer's Fund, the Olav Thon Foundation, the Erling-Persson Family Foundation, Familjen Rönströms Stiftelse, Stiftelsen för Gamla Tjänarinnor, Hjärnfonden, Sweden (grant number FO2022-0270), the European Union's Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement number 860197 (MIRIADE), the European Union Joint Programme—Neurodegenerative Disease Research (grant number JPND2021-00694), the National Institute for Health and Care Research University College London Hospitals Biomedical Research Centre, the UK Dementia Research Institute at UCL (grant number UKDRI-1003) and an anonymous donor. The DABNI study is funded by the Instituto de Salud Carlos III (Ministerio de Asuntos Económicos y Transformación Digital, Gobierno de España) through the projects INT21/00073, PI20/01473 and PI23/01786 to J.F. ; PI18/00335, PI22/00758, ICI23/00032 to MCI; PI18/00435, PI22/00611, INT19/00016, INT23/00048 to D. Alcolea; and PI14/1561, PI20/01330 to A.L., the Centro de Investigación Biomédica en Red sobre Enfermedades Neurodegenerativas CIBERNED Program 1, partly jointly funded by Fondo Europeo de Desarrollo Regional, Unión Europea, Una Manera de Hacer Europa. This work was also supported by the National Institutes of Health (grant numbers R01 AG056850 R21 AG056974, R01 AG061566, R01 AG081394 and R61AG066543 and 1RF1AG080769-01 to J.F.). It was also supported by Alzheimer's Association (grant number AARG-22-973966 to M.C.I. ), Global Brain Health Institute (grant number GBHI_ALZ-18-543740 to M.C.I. ), Jérôme Lejeune Foundation (grant number 1913 cycle 2019B to M.C.I. ), Fundación Tatiana Pérez de Guzmán el Bueno (grant number IIBSP-DOW-2020-151 to J.F.) Open access funding provided by University of Gothenburg. These authors contributed equally: Hanna Huber, Laia Montoliu-Gaya, Wagner S. Brum. These authors jointly supervised this work: Kaj Blennow, Henrik Zetterberg, Nicholas J. Ashton. Hanna Huber, Laia Montoliu-Gaya, Wagner S. Brum, Jakub Vávra, Yara Yakoub, Haley Weninger, Luisa Sophie Braun-Wohlfahrt, Joel Simrén, Anna Dittrich, Ingmar Skoog, Silke Kern, Kaj Blennow, Henrik Zetterberg & Nicholas J. Ashton German Center for Neurodegenerative Diseases, Bonn, Germany Graduate Program in Biological Sciences: Biochemistry, Universidade Federal do Rio Grande do Sul (UFRGS), Porto Alegre, Brazil Integrated Program in Neuroscience, McGill University, Montreal, Quebec, Canada Research Center of the Douglas Mental Health University Institute, Montreal, Quebec, Canada Ace Alzheimer Center Barcelona-Universitat Internacional de Catalunya, Barcelona, Spain Mercé Boada, Agustín Ruiz, Amanda Cano, Adelina Orellana, Sergi Valero, Laia Cañada, Natalia Tantinya, Ana Belen Nogales, Pilar Sanz-Cartagena & Xavier Morató Networking Research Center on Neurodegenerative Diseases (CIBERNED), Instituto de Salud Carlos III, Madrid, Spain Mercé Boada, Agustín Ruiz, Amanda Cano, Adelina Orellana, Sergi Valero & Xavier Morató Region Västra Götaland, Sahlgrenska University Hospital, Department of Neuropsychiatry, Mölndal, Sweden Anna Dittrich, Ingmar Skoog & Silke Kern Millie Sander-Long, Clive Ballard, Megan Richards, Mary O'Leary & Anne Corbett Frederikke Kragh Clemmensen, Hannah H. D. Wandall & Anja Hviid Simonsen Daniele Altomare, Valentina Cantoni & Barbara Borroni Competence Centre on Ageing (CCA), Department of Business Economics, Health and Social Care (DEASS), University of Applied Sciences and Arts of Southern Switzerland (SUPSI), Manno, Switzerland Erik Stomrud, Sebastian Palmqvist & Oskar Hansson Memory Clinic, Skåne University Hospital, Malmö, Sweden Alberto Lleo, Daniel Alcolea, Maria Carmona Iragui, Aida Sanjuan Hernandez, Bessy Benejam, Laura Videla Toro & Juan Fortea Alberto Lleo, Daniel Alcolea, Maria Carmona Iragui, Aida Sanjuan Hernandez, Bessy Benejam, Laura Videla Toro & Juan Fortea Maria Carmona Iragui, Aida Sanjuan Hernandez, Bessy Benejam, Laura Videla Toro & Juan Fortea Alpana Singh, Marisa N. Denkinger & Nicholas J. Ashton Neuropsychiatric Epidemiology Unit, Department of Psychiatry and Neurochemistry, Institute of Neuroscience and Physiology, Sahlgrenska Academy, Centre for Ageing and Health (AGECAP) at the University of Gothenburg, Mölndal, Sweden Molecular Markers Laboratory, IRCCS Istituto Centro San Giovanni di Dio Fatebenefratelli, Brescia, Italy Clinical Neurochemistry Lab, Sahlgrenska University Hospital, Mölndal, Sweden Centre for Brain Research, Indian Institute of Science, Bangalore, India Banner Alzheimer's Institute and University of Arizona, Phoenix, AZ, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors acquired, analyzed and interpreted the data. All authors critically revised the paper for important intellectual content. obtained funding for the present study. All authors reviewed and approved the final paper. L.M.-G. has received speaker and/or consultancy fees from Esteve and Quanterix. has received speaker honoraria from Roche Diagnostics. D. Alcolea participated in advisory boards from Fujirebio-Europe, Roche Diagnostics, Grifols S.A. and Lilly, and received speaker honoraria from Fujirebio-Europe, Roche Diagnostics, Nutricia, Krka Farmacéutica S.L., Zambon S.A.U., Neuraxpharm, Alter Medica, Lilly and Esteve Pharmaceuticals S.A. D.A. declare a filed patent application (WO2019175379 A1 markers of synaptopathy in neurodegenerative disease). has served as a consultant or on advisory boards for Almirall, Beckman-Coulter, Fujirebio-Europe, Roche, Biogen, Grifols, Novartis, Eisai, Lilly, and Nutricia. reported receiving personal fees for service on the advisory boards, speaker honoraria or educational activities from Esteve, Lilly, Adium Pharma, Neuraxpharm and Roche. has acquired research support (for the institution) from C2N Diagnostics, Fujirebio, GE Healthcare and Roche Diagnostics. has acquired research support (for the institution) from Avid and ki elements through ADDF. In the past 2 years, he has received consultancy and/or speaker fees from BioArctic, Biogen, Eisai, Eli-Lilly, Novo Nordisk and Roche. has served at scientific advisory boards, speaker and/or as consultant for Roche, Eli-Lilly, Geras Solutions, Optoceutics, Biogen, Eisai, Merry Life, Triolab, Novo Nordisk and BioArctic, unrelated to present study content. A.C. has received consultancy funding from Novartis, Addex, Acadia, Suven and J&J pharmaceutical companies and grant funding from Novo Nordisk, ReMYND and TheriniBio pharmaceutical companies. reported serving on the advisory boards, adjudication committees, or speaker honoraria from AC Immune, Adamed, Alzheon, Biogen, Eisai, Esteve, Fujirebio, Ionis, Laboratorios Carnot, Life Molecular Imaging, Lilly, Lundbeck, Novo Nordisk, Perha, Roche, Zambón, Spanish Neurological Society, T21 Research Society, Lumind foundation, Jérôme Lejeune Foundation, Alzheimer's Association, National Institutes of Health USA, and Instituto de Salud Carlos III. reports holding a patent for markers of synaptopathy in neurodegenerative disease (licensed to ADx, EPI8382175.0). No other competing interests were reported. has received a one-time consulting fee, paid to the institution from Eisai–BioArctic (2025). has served as a consultant and at advisory boards for Abbvie, AC Immune, ALZpath, AriBio, Beckman-Coulter, BioArctic, Biogen, Eisai, Lilly, Moleac Pte. Ltd, Neurimmune, Novartis, Ono Pharma, Prothena, Quanterix, Roche Diagnostics, Sanofi and Siemens Healthineers; has served at data monitoring committees for Julius Clinical and Novartis; has given lectures, produced educational materials and participated in educational programs for AC Immune, Biogen, Celdara Medical, Eisai and Roche Diagnostics; and is a co-founder of Brain Biomarker Solutions in Gothenburg AB (BBS), which is a part of the GU Ventures Incubator Program, outside the work presented in this paper. has served at scientific advisory boards and/or as a consultant for Abbvie, Acumen, Alector, Alzinova, ALZpath, Amylyx, Annexon, Apellis, Artery Therapeutics, AZTherapies, Cognito Therapeutics, CogRx, Denali, Eisai, Enigma, LabCorp, Merry Life, Nervgen, Novo Nordisk, Optoceutics, Passage Bio, Pinteon Therapeutics, Prothena, Quanterix, Red Abbey Labs, reMYND, Roche, Samumed, Siemens Healthineers, Triplet Therapeutics and Wave; has given lectures sponsored by Alzecure, BioArctic, Biogen, Cellectricon, Fujirebio, Lilly, Novo Nordisk, Roche and WebMD; and is a co-founder of Brain Biomarker Solutions in Gothenburg AB (BBS), which is a part of the GU Ventures Incubator Program (outside submitted work). has received consultancy and/or speaker fees from Alamar Biosciences, BioArctic, Biogen, Eli-Lilly, Neurogen Biomarking, Roche, Spear Bio, Quanterix and Vigil Neurosciences. The other authors declare no competing interests. Nature Medicine thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Scatterplots display the relationship between venous plasma p-tau217 (y-axis) and capillary p-tau217 (x-axis), stratified in each panel according to the cohort in which biomarkers were measured. Spearman correlation coefficients are displayed alongside their p-values and a mean regression line with 95% confidence intervals. Scatterplots representing the association between venous plasma (y-axis) and capillary (x-axis) p-tau217 values, stratified by tertiles (n = 84 for each panel; A, in the first venous plasma tertile; B, in the second venous plasma tertile; and C, in the third venous plasma tertile) of plasma p-tau217 distribution so as to visualize the magnitude of venous plasma-capillary correlation across different concentration strata. Spearman correlation coefficients are shown alongside their p-values and a mean regression line with 95% confidence intervals. ROC curve representing the discriminative accuracies of capillary (red) and venous plasma (brown) p-tau217 for abnormal CSF Aβ42/p-tau181 values based on a dataset of individuals with paired venous plasma p-tau217, capillary p-tau217 and CSF data (n = 176). Aβ = Amyloid-β; AUC = Area under the curve; CI = Confidence interval; CSF = Cerebrospinal fluid; p-tau = Phosphorylated tau; ROC = Receiver operating characteristic. P-values for between-group comparisons were obtained with post-hoc contrasts from linear models adjusted for age and sex and Tukey's adjustment was used. Aβ = Amyloid-β; AD = Alzheimer's disease; CSF = cerebrospinal fluid; CU = Cognitively unimpaired; IQR = Interquartile range; MCI = Mild cognitive impairment; p-tau = Phosphorylated tau. (A) Boxplots indicating capillary p-tau217 concentrations (y-axis) according to CSF Aβ42/Aβ40 status (x-axis; blue for Aβ-negative, red for Aβ-positive; CSF-negative, n = 54; CSF-positive, n = 119). (B) ROC curve representing the discriminative accuracies of capillary (red) and venous plasma (brown) p-tau217 for abnormal CSF Aβ42/Aβ40 values (n = 173). P-values for between-group comparisons were obtained from group-contrasts from linear models adjusted for age and sex, with Tukey's adjustment. Aβ = Amyloid-β; AD = Alzheimer's disease; AUC = Area under the curve; CSF = Cerebrospinal fluid; CU = Cognitively unimpaired; IQR = Interquartile range; MCI = Mild cognitive impairment; p-tau217 = Phosphorylated tau 217; ROC = Receiver operating characteristic. P-values for between-group comparisons were obtained from group-contrasts from linear models adjusted for age and sex, with Tukey's adjustment. (B) ROC curve for the discriminative ability of capillary p-tau217 in detecting venous plasma p-tau217 positivity (n = 252). AUC = Area under the curve; IQR = Interquartile range; p-tau217 = Phosphorylated tau 217; ROC = Receiver operating characteristic. Scatterplots showing the relationship between biomarkers measured in capillary blood (x-axis) and in venous plasma (y-axis) according to different capillary blood sampling methods (B50, SEP10, Telimmune) displayed in each panel. Spearman correlation coefficients are shown alongside their p-values and a mean regression line with 95% confidence intervals. GFAP = Glial fibrillary acidic protein; NfL = Neurofilament light; p-tau 217 = Phosphorylated tau 217. Boxplots representing biomarker distributions for p-tau217 (A, n = 43) and GFAP (B, n = 18) in professionally collected dried blood spots versus self-collected, with p-value for between-group comparisons obtained from a non-parametric Wilcoxon signed rank test. GFAP = Glial fibrillary acidic protein; IQR = Interquartile range; p-tau217 = Phosphorylated tau 217. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. A minimally invasive dried blood spot biomarker test for the detection of Alzheimer's disease pathology. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43018-025-01084-0'>De novo H3.3K27M-altered diffuse midline glioma in human brainstem organoids to dissect GD2 CAR T cell function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 11:26:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Diffuse midline glioma (DMG) is a highly aggressive and untreatable pediatric cancer primarily arising in the pontine brainstem region, necessitating the development of representative models for treatment advance. Here we developed an FGF4-driven human brainstem organoid model, which we used to genetically engineer H3.3K27M-altered DMG. We demonstrated that brainstem pontine glial specification is critical for DMG tumorigenesis, yielding infiltrative tumors that recapitulate patient-representative intratumoral heterogeneity. Prolonged GD2 chimeric antigen receptor (CAR) T cell treatment mirrored clinical outcomes and revealed extensive transcriptional heterogeneity, from which both potent effector and dysfunctional CAR T cell populations could be identified. Furthermore, incorporation of myeloid cells generated DMG-specific microglia that reduced treatment efficacy and revealed CAR T cell functional states most vulnerable to microglia-mediated immunosuppression. Thus, we present a representative DMG model offering a months-long experimental window in vitro, which we leveraged to delineate CAR T cell functionality and microglial impact, aiding therapy development for this devastating disease. Diffuse midline gliomas (DMGs) are rare and aggressive pediatric brain tumors often caused by somatic mutations in histone 3 (H3) genes, commonly a K27M substitution1, occurring at a high prevalence in the pons region of the brainstem2. Primarily, affecting children under 10 years3, they present the highest mortality rate of any cancer, with a median overall survival of only 9–15 months4,5. This detrimental prognosis necessitates a better understanding of the disease's biology to develop effective treatments. Single-cell analyses of H3K27M-altered DMG revealed intratumoral heterogeneity, with a spectrum of tumor cell profiles ranging from stalled stem-like oligodendrocyte progenitor cell (OPC-like) to more differentiated astrocyte (AC-like) and oligodendrocyte (OC-like) phenotypes, which closely resemble normal developmental cell types, alongside a recently identified mesenchymal-like (MES-like) state6,7,8. In addition, insights from both animal9,10,11 and human pluripotent stem-cell-derived12,13 studies suggest an early neurodevelopmental window of tumor initiation. Thus, dysregulated mechanisms during hindbrain development10,14,15, particularly involving glial progenitors in the region responsible for brainstem pons formation16, likely have a central role in driving H3K27M-altered gliomagenesis. Capturing this region-specific embryonic patterning is, therefore, crucial for accurately modeling pontine DMG. Furthermore, previous work identified a tight relationship between DMG progression and its unique environment, including neuron and synaptic signaling17,18,19, which can promote glioma growth20. Human brain organoids have become valuable in vitro tools for investigating brain development and understanding the onset, progression and potential therapeutic targeting of nervous system disorders, including cancer21,22,23. Given the rarity and inoperable nature of DMG, which limits the availability of patients participant material2, organoids could offer a scalable model for generating DMG tumors de novo and enabling in vitro testing of emerging therapies. This includes the latest advances in immunotherapy for DMG, GD2 chimeric antigen receptor (CAR) T cells5, which showed promising, yet variable treatment outcomes between patients in a recent clinical trial24. Correlative data from this trial suggest that an expansion of the immunosuppressive myeloid compartment coincides with unfavorable treatment outcomes25. Uncovering the functional profiles of CAR T cells and their interplay with the immunosuppressive tumor microenvironment may offer key insights to improve their therapeutic outcomes in DMG. Here, we report a human cerebral guided organoid model for the brainstem region, enriched for pontine medulla glial lineages. Genetic modeling of H3.3K27M-altered DMG in these brainstem-regionalized organoids (BrOs) replicates the infiltrative nature and transcriptomic landscape of DMG. We demonstrate the utility of this accessible human DMG organoid model (DMGO) for modeling CAR T cell functional heterogeneity during prolonged treatment (up to 1 month) and within the context of brain-resident microglia. To generate human organoids with appropriate hindbrain brainstem identity for DMG modeling, we applied sequential morphogen guidance using a timely sequence of Wnt, dual SMAD inhibitors, retinoic acid (RA), fibroblast growth factors (FGFs) and sonic hedgehog (SHH) (Fig. While FGF2 and FGF8 can be used in combination with RA and Wnt to pattern the midbrain26, cerebellum27 or spinal cord28 in growing organoids, we evaluated FGF4 because of its role in specifying rostral hindbrain, particularly in the pontine area29,30, as well as its involvement in the development of hindbrain-specific serotonergic neurons31. A direct comparison of replacing common FGF2 supplementation with FGF4 after 7 days of patterning demonstrated that 10 ng ml−1 FGF4 specifically promotes developing the pontine, including the prepontine to retropontine area, according to bulk sequencing data (Fig. Furthermore, among HOX genes important for hindbrain formation, expression of HOXB1, a marker of pontine precursor cells32, emerged already at an early stage (day 14) (Extended Data Fig. 1c) and three-dimensional (3D) imaging revealed HOXB1-expressing cells within early neurodevelopmental SOX2+ neural rosette structures (Fig. Bulk sequencing analysis from day 7 to day 84 showed that the patterning remained consistent and reproducible across and within multiple batches, as well as between human embryonic stem cell (hES cell) and induced pluripotent stem cell (iPS cell) sources (Extended Data Fig. a, Schematic representation of timely morphogen stimulated patterning of hES cells and hiPS cells toward brainstem organoids and their subsequent application for DMG tumor, CAR T cell treatment and microglia-enriched tumor microenvironment modeling. b, Heat map of z score measuring relative brain region identity on the basis of VoxHunt similarity mapping for various supplemented concentrations of FGF2 or FGF4 (n = 3 experimental repeats, with n = 3 organoids pooled; total n = 9 organoids per condition). c, Immunofluorescence 3D images of a 200-µm-thick organoid slice on day 21 labeled for F-actin (white), SOX2 (yellow) and HOXB1 (red). Right, zoomed-in view of area in white inset. d, VoxHunt spatial correlation map of day 120 brainstem organoids with E18.5 mouse brain. The pons area is delineated in red (n = 9 organoids from three independent batches). e, Integrated UMAP representation of developing brainstem organoids from different time points, colored by cell annotation. f, Area plot following the relative distribution of cell types over time. Cell types are color-coded as in e. g, UMAP of the HNOCA34 colored for brainstem organoid presence score. A high score indicates a high likelihood that these HNOCA cells are present in the brainstem organoid dataset. Inset, UMAP colored by coarse regional annotation. Positive values correspond to an increased abundance of cells from the indicated regional identity or glial lineage. i, Cell clusters in the HDBCA35 with gained coverage in brainstem organoids relative to the HNOCA. The horizontal line indicates the threshold used to define a cluster as gained or not. j, UMAP of the HDBCA showing, in shades of red, the HDBCA clusters gained in brainstem organoids, mostly related to oligos and glioblasts. Gray represents clusters below the threshold used to define gained. Inset, UMAP colored by coarse regional annotation. For e–j, n = 84 organoids in total with 5–24 organoids pooled per time point (details in Supplementary Table 1). To investigate cellular composition and regional identities at higher resolution, we performed time-course single-cell RNA sequencing (scRNA-seq) across eight time points, spanning from day 5 to day 120 (Extended Data Fig. Following quality control (Extended Data Fig. Spatial similarity mapping using VoxHunt33, a tool based on Mus musculus in situ hibridization data from the Allen Brain Atlas, confirmed a hindbrain identity with a more pronounced pontine signature (Fig. We next generated an integrated uniform manifold approximation and projection (UMAP) representation of the different time points and performed cell-based annotation using reference datasets, including the recently published Human Neural Organoid Cell Atlas (HNOCA)34 and the Human Developing Brain Cell Atlas (HDBCA)35,36 (Fig. Temporal analysis revealed an initial phase of high proliferation that diminished over time (Extended Data Fig. 3a), as cells transitioned from pluripotent stem cells to the neuroepithelium and radial glial cells, as well as into distinct neuronal and glial populations that emerged by days 14 and 60, respectively (Fig. 1f), reflecting the natural occurring segregation of neurogenesis and gliogenesis phases35. Projection of the organoid dataset onto the HNOCA that was regionally annotated for neuronal lineages (Fig. 1g) revealed that most neuronal precursor cells, neuroblasts and neurons originated from a heterogeneous nontelencephalic cluster with enrichment for midbrain, pons and medulla regions (Fig. These neuroblasts and early neurons expressed STMN2 and RBFOX3 (NeuN) but lacked the telencephalic marker FOXG1 (ref. Furthermore, neurotransmitter transporter analysis revealed a predominance of excitatory (glutamatergic) and inhibitory (GABAergic) neurons, the latter known to form synapses with DMG and promote its growth17,18. Smaller proportions of cholinergic and dopaminergic neurons were also detected, consistent with their distribution in the HDBCA (Extended Data Fig. In addition, immunofluorescence analysis identified cells expressing tryptophan hydroxylase 2 (Extended Data Fig. 3e), a key enzyme involved in serotonergic synthesis, suggesting that, albeit undetectable at the scRNA-seq level similar to the HDBCA35, this population of neurons is present. Thus, consistent with HNOCA and adult brain data showing greater heterogeneity and intermixing among nontelencephalic neurons, including hypothalamic, brainstem and hindbrain neurons (referred to as splatter neurons) compared to cortical neurons34,38, our organoid model recapitulates this regional diversity, with an enrichment in brainstem identity compared to profiles described in most HNOCA protocols. DMG is rooted in the glial lineage8, prompting us to investigate the glial composition within our BrO model. First, we showed the presence of committed ACs (GFAP+AQP4+) and OCs (OLIG2+) at the protein level (Extended Data Fig. At the single-cell transcriptomic level, we identified glial populations spanning pre-OPCs, OPCs, committed OC precursors, glioblasts and ACs, offering a detailed representation of glial diversity and maturation states (Fig. By comparing age-matched cells of HNOCA-covered protocols, BrOs showed significant enrichment in the glial lineage, particularly glioblasts and OPCs (Fig. Additionally, we assessed glycolysis, an indicator of cell stress in brain organoids34. Consistent with models described in the HNOCA, we observed similar glycolysis levels (Extended Data Fig. However, in the glial lineage, glycolysis levels were lower (Extended Data Fig. 3h), suggesting reduced stress and a healthier metabolic state of glial cells in BrOs. Moreover, OPC (referred to as oligo in the HDBCA35) and glioblast populations demonstrated a reduced number of differentially expressed genes (DEGs) compared to HNOCA datasets (Extended Data Fig. 3i), reflecting higher transcriptional fidelity and closer alignment with primary counterparts in the HBDCA35. To date, no comprehensive region-wide analysis has been conducted on glial cells derived from organoids. However, the HBDCA revealed strong region specificity in the glial lineage during early brain development, which may be particularly relevant for H3.3K27M-altered DMG in the brainstem pontine region. 3j,k) and comparison to organoid protocols embedded in the HNCOA revealed significant coverage of glial clusters in BrOs. Notably, 13 of the 19 gained clusters exhibited pontine and medulla-specific identities (Fig. Thus, our newly generated BrO model offers an experimental framework for studying gliogenesis within the context of pontine medulla regionality, offering relevance for DMG modeling. We next investigated whether BrOs could be exploited to model DMG tumors. Plasmids11 expressing the most common H3.3K27M-defining DMG mutation39, alongside typical accompanying and pons-specific tumor suppressor TP53 and platelet-derived growth factor A (PDGFRA) alterations3,40,41, were introduced using in situ electroporation of developing BrOs (Fig. This mutation cocktail has been shown to be time sensitive in in utero electroporation mouse models9,11,42; hence, we tested different time points of electroporation between days 11 and 28. We identified day 11 as the time point most efficiently inducing tumorigenic growth (Fig. 4a), reinforcing the concept of a restricted early developmental time window for DMG transformation9,11. At this stage of development, we observed a dominance of radial glia and neuroepithelial stem-like cells in BrOs (Fig. 1f), aligning with earlier work identifying neural stem cells as a permissive cell state for H3.3K27M-driven neoplastic transformation9,11,12,13,14. Tracking tumor growth over 2 months showed that the resulting tumors display infiltrative growth (Fig. In contrast, the use of empty control plasmids resulted in only a few localized electroporated cells (Extended Data Fig. Whole-organoid 3D imaging at week 16 (4 months after electroporation) with tumors color-coded for invasion depth further confirmed a diffuse growth pattern characteristic of DMG (Fig. Quantification of H3.3K27M expression, combined with dominant-negative TP53 (DNp53) and PDGFRA-D842V at the protein level, showed incorporation of all three mutations into the majority of GFP-positive cells (Fig. These findings illustrate DMG invasive outgrowth in our guided brain organoids dependent on combined common driver mutations typically observed. NS, not significant; **P < 0.01, according to a two-tailed independent t-test. Data are shown as the mean ± s.e.m. b, Tumorigenic outgrowth of the same DMGO at weeks 4, 6 and 8. White arrowheads depict invasive and diffuse patterns. c, Representative immunofluorescence 3D image of intact DMGOs with GFP signal color-coded for z depth on a rainbow scale. The gray outline was created by masking of propidium iodide fluorescence (n = 2 DMGOs). d,e, Representative multispectral 3D images of tumor GFP (green), H3K27M (magenta) and DNp53 (yellow) (d) or tumor GFP (green) and PDGFRA (red) (e) in consecutive slices of a week 8 DMGO (n = 3 DMGOs). f, Percentage of GFP+ tumor cells expressing H3K27M, DNp53 or PDGFRA detected by multispectral 3D imaging as in d,e (n = 2 DMGOs with n = 3 ROIs imaged per DMGO). g, Methylation profile of DMGO compared to DMG or resembling tumor types. Each dot represents one patient sample. GBM, glioblastoma; EPN, ependymoma; tSNE, t-distributed stochastic neighbor embedding. For DMGO, the dot represents a pooled sample of n = 3 DMGOs. h, Integrated Force Atlas representation of DMGO tumors colored by tumor cell state (n = 14 DMGOs from four independent batches. The average similarity (color intensity) represents an averaged prediction score of all DMGO subsetted tumor cells mapped into a merged dataset consisting of transcriptomic model-derived and patient datasets6,7,8 (n = 14 DMGOs from four independent batches). To further assess the representability of our in vitro grown tumor model (DMGO), we conducted histological comparisons to patient samples sharing the same mutational profile. This showed that H3.3K27M cells (H3K27M+) display loss in H3K27 trimethylation (H3K27me3) in both patient samples and DMGOs (Extended Data Fig. Furthermore, our in vitro grown tumors exhibited a global methylation profile closely resembling DMG, distinguishing our tumors from glioblastoma and posterior fossa (PFA1 and PFA2) epyndomas, which present with a similar loss of H3K27M trimethylation caused by H3K27M substitution or EZHIP overexpression, respectively43 (Fig. We next conducted scRNA-seq profiling of sorted GFP+ tumor cells and, after quality control filtering, analyzed approximately 7,000 cells from 14 DMGOs (Fig. The malignant state of these cells was further supported by the analysis of inferred copy-number variation (iCNV) from scRNA-seq data, which showed large-scale amplifications and deletions in these cells compared to healthy cells, including losses of chromosomes 10 and 13 and a gain of chromosome 19q (Extended Data Fig. Using published DMG references6,8, we first annotated cancer cell states previously described for DMG, including OPC-like, AC-like and MES-like cell states and a population of cycling cells. In line with the early developmental window of our model, we identified only few cells with a more mature OC-like phenotype. Importantly, we identified a major proportion of OPC-like tumor cells that resembled recently defined OPC-like 2 and 3 states, both described as pediatric and pons-specific pre-OPC states8 (Fig. Furthermore, we found the highest similarity score between DMGOs and primary DMG patient material6, as opposed to cell lines, patient-derived xenografts (PDXs) and material from patients with glioblastoma6 or both PFA subtypes44 (Fig. Together, this highlights the ability of DMGOs to closely resemble primary DMG tumors. We investigated the mechanisms driving tumorigenesis to identify the attributes of BrOs that appear to be critical for supporting the growth of DMG tumors. We used TrackerSeq, a PiggyBac-based genetic lineage-tracing approach (Extended Data Fig. 6a), and analyzed cancer clone dynamics at 2 months after electroporation. We retrieved 167 unique barcodes from six DMGOs and two healthy BrOs (Extended Data Fig. 6b–h and Supplementary Table 5) and detected individual clones spanning up to approximately 800 cells per barcode, indicative of cancerous transformation (Extended Data Fig. By comparing large versus small, traced clones (Fig. 3a,b) through DEG and METASCAPE analysis, we identified glial specification as a critical feature driving cancer clone expansion in contrast to neuronal specification enriched in smaller clones (for example, synapse organization and modulation of chemical synaptic transmission) (Fig. Larger clones were characterized by higher gene expression of OLIG1, a canonical OPC marker, as well as IER2, JUNB, FOS and EGR1, previously described as key markers of the OPC-like 3 pre-OPC state8. Interestingly, we also identified a higher expression of AQP1, an aquaporin previously shown to be exclusive to ACs in the human brainstem45. Furthermore, analysis of patient data7 revealed AQP1 expression in tumors located in the pons but not in those arising from the cortex or thalamic regions (Extended Data Fig. These data hint toward a glial-specific tumorigenic process that is, furthermore, dependent on pontine location. This is further illustrated by genes upregulated in large DMG tumor clones mapping back to the glial lineage of BrOs (Fig. 3c), indicating that tumorigenesis is dependent on gliogenesis. To confirm this experimentally, we performed in situ electroporation of our mutation cocktail in unguided cerebral organoids, revealing a reduction in tumor induction (Fig. In addition, the outgrowth was nondiffuse, with almost no GFP-positive tumorigenic cells carrying the H3.3K27M substitution (Fig. These findings demonstrate that H3.3K27M-driven tumorigenesis depends on the correct anatomical cell identity, which our BrO model recapitulates. Consensus non-negative matrix factorization (cNMF) (Fig. 6l,m) and lineage relationship analysis identified malignant metagene programs 1 and 2 to be present in the highest number of clones (30 and 26 of 34 clones, respectively; Fig. 3h) and belonging to the OPC-like lineage, emphasizing the central role of this lineage in H3K27M DMG tumorigenesis8. More specifically, we show overlap with the pre-OPC state, OPC-like 2 (Extended Data Fig. In the context of human early gestation, regionally distinct gene signatures for the glial lineage have been suggested to underlie the strong region-specific occurrence pattern of glial-related diseases, such as DMG35. In line with this, both programs 1 and 2 specifically enrich for the hindbrain pons OC precursor lineage35, as opposed to midbrain and forebrain lineages (Fig. Collectively, these findings highlight the role of pons glial specification, captured in BrOs, in driving DMG tumorigenesis, emphasizing the need for spatial and developmental precision in modeling DMG and establishing a human-relevant experimental system for therapeutic testing. a, Volcano plot showing top DEGs in larger and smaller clones. b, METASCAPE results showing selected GO terms from the highest-scoring summary GO terms for small and large clones. c, Presence of large (red) and small (blue) clones in the integrated UMAP representation of developing BrOs, showing a preference for gliogenesis and neurogenesis, respectively. For a–c, n = 14 DMGOs from four independent batches. d, Bar plot quantifying electroporation efficacy (light-gray columns; guided versus unguided, P = 0.342) and tumor induction (dark-gray columns; guided versus unguided, P = 0.023) for guided brainstem organoids as compared to unguided neural organoids at day 11. *P < 0.05, according to a two-tailed independent t-test. Data are shown as the mean ± s.e.m. (n = 35 organoids from three independent experiments per condition; details in Supplementary Table 1). e, Representative images of tumorigenic outgrowth (GFP, green) at weeks 4 and 8 for unguided neural organoids (n = 35 organoids from three independent batches). f, Representative multispectral 3D images of tumor GFP (green), H3K27M (magenta) and DNp53 (yellow) in unguided neural organoids (n = 2 organoids). g, UMAP of traced DMGO cells, colored by their respective highest-scoring cNMF program. h, UpSet plot displaying clonal intersection events. Only clonal families found in more than one cNMF module are depicted and filtered with at least three cells present per unique barcode. Bar plots depict the frequency of each lineage combination (top) and the number of clones that contain each program (left). Coloring of dots matches the cNMF program annotation as in g. i, Heat map presenting the mean cellular enrichment scores of cNMF programs 1 and 2 for forebrain, midbrain and hindbrain or pons oligo lineage signatures in HDBCA35. For g–i, n = 8 DMGOs from three independent batches. Given the relevant tumor progression observed in DMGOs, we evaluated their potential as a human in vitro platform for preclinical testing of GD2 CAR T cells (Fig. 1a), motivated by promising yet variable treatment outcomes in a recent first clinical trial in patients with H3K27M-mutant DMG24,25. By exposing untransformed BrOs to GD2 CAR T cells, we first visually inspected with brightfield imaging that the presence of GD2 CAR T cells did not affect the general health of the model (Extended Data Fig. Next, we confirmed GD2 target expression in DMGO (Extended Data Fig. Having established these experimental preconditions, we treated DMGOs 4 months after tumor induction by administrating CD8+ GD2 CAR T cells on days 0 and 7 and monitored T cell activation, measured by interferon-γ (IFNγ) secretion (Extended Data Fig. 7c) and tumor control (Extended Data Fig. Similar to heterogeneous outcomes reported in individuals24,25, we observed an overall partial reduction in tumor burden (Fig. As GD2 CAR T cell activation was evident by a robust IFNγ response for all treated DMGOs (Extended Data Fig. 7c), limited response profiles (for example, DMGO179) are unlikely to result from a lack of antigen recognition. Therapy effects could be detected after >1 month of treatment (Fig. 4c), offering advantages for modeling CAR T cell functionality in vitro in a manner that is representative of T cell states at the tumor site in vivo, including potential exhaustion profiles associated with prolonged tumor exposure. To test our model for this purpose, we sequenced over 20,000 GD2 CAR T cells retrieved from DMGOs, as well as unexposed GD2 CAR T cells (Supplementary Table 6). This revealed a substantial level of heterogeneity induced upon DMGO exposure (Fig. In GD2 CAR T cells retrieved from DMGOs, we identified nine transcriptional states (Fig. 4e) that, on the basis of a combined interrogation of curated gene signatures (Extended Data Fig. 8a), DEGs, DEG-associated Gene Ontology (GO) terms (Extended Data Fig. 8b–f), expression of canonical immune effector (Fig. 4g) and comparison to a pan-cancer tumor-infiltrating lymphocyte (TIL) dataset including brain malignancies46 (Extended Data Fig. 8g), reflected different T cell activation, differentiation and effector states. For instance, we identified a GD2 CAR T cell population that, albeit activated (on the basis of HLA gene expression) (Extended Data Fig. 8a), does not fully differentiate toward effector function (undifferentiated; TUND) (Extended Data Fig. 8c,i), probably differentiating into effector T cells (Extended Data Fig. In addition, we observed an interferon-stimulated gene (ISG)-expressing population (TISG) (Extended Data Fig. 8a), corresponding to ISG-expressing TILs46 (Extended Data Fig. 8j) and considered as an interferon-induced activation state47,48. Other clusters included a CAR T cell population with migrating properties and interconnectivity that appeared to be influenced by its neuronal environment (Extended Data Fig. 8e) and metabolically stressed T cells (TMS) (Extended Data Fig. Importantly, we distinguished potential DMG-targeting effector T cell populations on the basis of their cytotoxic profile (Fig. 4f) and putative level of exhaustion (Fig. While one of these clusters predominantly expressed GZMK (TGZK), cytotoxic T cells (TCYT) expressed GZMB, PRF1 and IFNG (Fig. In contrast, exhausted T cells (TEX) displayed reduced IFNG and concomitant expression of immune checkpoint genes, LAG3, HAVCR2, TIGIT (ref. Weekly CAR T cell administration (days 0 and 7) did not improve TEX reduction or TCYT enrichment over a single dose (Extended Data Fig. a, Representative multispectral 3D imaging of GD2 CAR T cells (CD3; cyan), DMG tumor cells (GFP; green) and cleaved caspase 3 (cCasp3; red) in DMGOs (n = 2 DMGOs). b, GD2 CAR T cell treatment outcome measured as a relative change in tumor GFP intensity quantified by imaging compared to the start of treatment (100%). DMGOs were left untreated (gray line; n = 1 DMGO) or treated with mock-transduced T cells (black line; n = 2) or GD2 CAR T cells (orange line; n = 4 DMGOs); for each treatment condition, a smoothed trend line of the averaged values at different time points was plotted using the locally estimated scatter plot smoothing algorithm. Arrows indicate the time points of T cell administration. c, Representative images of the tumor GFP signal at the indicated time points for a DMGO subjected to prolonged GD2 CAR T cell treatment administrated on day 0, day 8 and day 15 (n = 1 DMGO). d, Sankey plot illustrating the shift in the relative proportions of unbiasedly identified GD2 CAR T cell clusters before and after DMGO exposure. e, UMAP visualization of annotated GD2 CAR T cell clusters. f, Cytotoxic effector molecule and cytokine gene expression across the GD2 CAR T cell clusters. g, Gene expression of selected exhaustion-associated receptors, ligands and transcription factors across the GD2 CAR T cell clusters. f,g, Dot plot representing the percentage of cells expressing selected genes. h,i, Heat map depicting the relative expression of exhaustion markers (h) and exhaustion-associated transcription factors and functional regulators (i) in nonexposed (left) and DMGO-exposed (right) GD2 CAR T cells within the TEX cluster. j, Super-engager signature score on a blue-to-red color scale, showing the enrichment of a previously identified T cell serial killer gene set60 atop UMAP cell embeddings of the GD2 CAR T cell dataset. For d–j, GD2 CAR T cells retrieved from n = 4 treated DMGOs and n = 2 independent batches of unexposed GD2 CAR T cells. k, Dot chart depicting the fold enrichment in tumor killing by NCAM1+ GD2 CAR T cells over NCAM1− GD2 CAR T cells quantified as the change in tumor area detected by GFP compared to the start of treatment (n = 2 DMGOs per treatment condition). l, Percentage of cells per TCYT, TEX and THS cluster for NCAM1− GD2 CAR T cells (left; retrieved from n = 2 DMGOs) and NCAM1+ GD2 CAR T cells (right; retrieved from n = 2 DMGOs). To confirm that exhaustion detected in our DMGO model reflects representative T cell exhaustion at the tumor site, we compared the TEX phenotype present upon DMGO exposure to preexposure GD2 CAR T cells that, although alleviated by the 4-1BB endodomain, can still display exhaustion features resulting from tonic signaling52. Indeed, a fraction of preexposure GD2 CAR T cells overlapped with our TEX cluster detected upon DMGO exposure (Extended Data Fig. However, separating the cells in this cluster according to experimental condition (Extended Data Fig. 9c) revealed that DMGO-exposed TEX upregulated a wide array of additional exhaustion markers (Fig. 4i) that include those described in cancer patients across TIL datasets (Supplementary Table 7). In addition, overlap with exhaustion markers found in the antigen-driven lymphocytic choriomeningitis virus mouse model of chronic infection53,54, as well as an in vitro model of CAR T cell dysfunction based on continuous antigen exposure55, demonstrates that the observed exhaustion profile is antigen driven (Supplementary Table 7). For in vitro model systems, this has not been achieved in the context of naturally expressed tumor antigen, only through persistent anti-CD3 and anti-CD28 antibody stimulation56 or by using repeated rounds of stimulation with antigen- pulsed57 or overexpressing58 tumor cell lines55. Thus, DMGOs model the functional heterogeneity of CAR T cells, including representative T cell functional exhaustion, an actionable axis for improving outcomes59 and, therefore, critical factor to evaluate preclinically. Consistent with their potent cytotoxicity and lack of exhaustion, the TCYT population overlaps with the ‘killer' gene signature of ‘super-engager' engineered T cells that we recently identified to have profound tumor-targeting capacity and serial killing behavior in a short-term coculture assay60 (Fig. As we previously identified NCAM1 as a selection marker to enrich for this population60, we exploited this strategy (Fig. 1a) to further investigate the relevance of this CAR T cell functional profile in a prolonged treatment setting. We sorted GD2 CAR T cells on the basis of NCAM1 expression before DMGO treatment (Extended Data Fig. This demonstrated the initial potent antitumor activity of NCAM1+ GD2 CAR T cells, with a 1.4-fold enrichment in tumor control over NCAM1− GD2 CAR T cells on day 2. However, this enhanced potency stabilized between days 5 and 7, with NCAM1− T cells displaying more gradual antitumor activity over time, slightly outperforming NCAM1+ cells by day 7 (Fig. 4k), in line with a higher recovery of NCAM1− cells at day 14 (Extended Data Fig. To gain insight into potential transcriptomic profiles explaining these differential outcomes, we performed scRNA-seq of NCAM1− and NCAM1+ GD2 CAR T cells and mapped them back to our previously identified GD2 CAR T cell signatures (Extended Data Fig. This revealed an additional stressed GD2 CAR T cell cluster specific to NCAM1− cells (THS) (Fig. 9h) that differed from the TMS cluster through expression of heat-shock proteins (Supplementary Table 8) and overlapped with the stress response state identified in TILs that associates with immunotherapy resistance46 (Extended Data Fig. Further aligning with the initially enhanced tumor control observed (Fig. However, in line with poor persistence of the cells (Extended Data Fig. 9f), NCAM1+ T cells were additionally enriched for TEX (Fig. 4l), explaining their reduced performance over time (Fig. Together, this identified NCAM1+ cells as a potent tumor-targeting, yet short-lived effector GD2 CAR T cell population and offers proof of concept for cell selection as a means to narrow CAR T cell functional heterogeneity before administration. The upregulation of features associated with tissue residency48, including the canonical marker CD103 (ITGAE) used to identify tissue-resident T cells61,62 (Fig. 5a and Supplementary Table 7), underscores the capacity of DMGOs to model T cell performance within tissue. While this may inform strategies to enhance CAR T cell trafficking and tissue residency63, DMGOs lack the myeloid compartment, a key regulator of T cell responses. Therefore, to enhance the complexity of DMGOs, we incorporated microglia, a main component of the DMG tumor microenvironment8. We generated primitive macrophage progenitors (PMPs) from hES cells, previously shown to differentiate into mature microglia in mouse brains64 and human midbrain organoids65, as well as in coculture with neurons66. PMPs similarly integrated into BrOs and, within 7 days, adopted the ramified morphology characteristic of homeostatic microglia67 (Fig. Confirming functional maturation, the cells displayed typical microglia behavior, migrating to sites of myelin injection (Fig. Furthermore, 3 weeks after incorporation in BrOs, above 80% of cells expressed the microglia-specific marker P2RY12 at the protein level (Fig. 5f,g) and scRNA-seq analysis (Supplementary Table 9) demonstrated increased expression of microglia-specific transcription factors69,70 (Fig. Additionally, they resembled an adult state when referenced against microglia developmental programs identified in mice71 (Fig. Comparison to a myeloid cell reference dataset from DMG tumors72 confirmed microglia as opposed to macrophage identity (Fig. a, Gene expression of tissue-resident markers in nonexposed (top; n = 2 independent batches of unexposed GD2 CAR T cells) and DMGO-exposed (bottom; retrieved from n = 4 DMGOs) GD2 CAR T cells within the TEX cluster. Dot plot representing the percentage of cells expressing selected genes. b, Schematic overview of PMP integration in BrOs and DMGOs and treatment with GD2 CAR T cells. c, Representative brightfield images of BrOs for mScarlet+ PMPs (white) 3 or 7 days after integration (n = 19 BrOs). Right, zoomed-in view of area in white insets. d,e, Live 3D imaging of microglia (orange) and CFSE-labeled myelin debris (green), showing homing of microglia to sites of myelin debris injection (d) and phagocytosis of myelin debris (e) (n = 2 BrOs). f, Immunofluorescence 2D images of BrO with microglia integrated for 3 weeks, labeled for DAPI (white), IBA1 (magenta) and P2RY12 (cyan) (n = 1 BrO). h, Heat map depicting the relative expression of microglia-associated transcription factors in PMPs (left) and BrO-derived microglia (right) 3 weeks after integration. i, Heat map depicting representation of microglia developmental stages71 in PMP or microglia derived from BrOs or DMGOs. j, Violin plot showing the expression level of microglia and macrophage gene signatures72 in PMP (left) and microglia derived from BrO (right). k, Violin plots showing expression levels of DMG-associated microglia states72 in microglia derived from BrO (blue) or DMGO (green). l, Dot plot showing the relative expression of selected chemokines and genes associated with immunosuppression72 in microglia derived from BrO (left) or DMGO (right). Microglia incorporation in tumor-bearing DMGOs led to the acquisition of recently described DMG-associated functional phenotypes72, including an IFN-activated, phago lipid and hypoxic state (Fig. Moreover, compared to BrOs, microglia from DMGOs showed enrichment of GO terms related to antigen presentation and immune responses, such as peptide processing mediated by major histocompatibility complex class II and type I IFN response, previously described as upregulated in DMG-associated microglia and macrophages73 (Extended Data Fig. These DMG-specific microglia states were accompanied by reduced chemokine expression and upregulation of genes associated with an immunosuppressive profile, in line with patient data72 (Fig. This was confirmed by protein expression of CD163, associated with an anti-inflammatory state, and SPP1, associated with immunosuppressive lipid-laden macrophages74 (Extended Data Fig. Together, these findings show that, within an appropriate neuronal environment, PMPs differentiate into mature microglia that, in the presence, of DMG tumor cells adopt a DMG-specific immunosuppressive phenotype. Correlative clinical data suggest that expansion of the immunosuppressive myeloid compartment may be associated with poor GD2 CAR T cell outcomes25 and myeloid cells, including microglia, are also implicated in CAR T cell-induced toxicity75. To address this experimentally, we performed GD2 CAR T cell treatment in DMGOs with integrated microglia. Confocal imaging showed interactions between GD2 CAR T cells and microglia within tumors (Fig. Moreover, we observed increased cytokine secretion in the presence of both GD2 CAR T cells and integrated microglia (Fig. This included upregulated chemokines related to myeloid cell chemotaxis, including MCP3 and CXCL1, as well as myeloid-cell-associated growth factors, such as macrophage colony-stimulating factor (M-CSF). We also observed elevated interleukin (IL)-1α and IL-6 levels, key proinflammatory cytokines linked to CAR T cell (neuro)toxicity and clinically targeted to manage adverse effect24, suggesting that microglia-integrated DMGOs may serve as a relevant model to study CAR T cell–microglia interactions underlying treatment-related toxicity. Furthermore, analysis of DMGO-induced CAR T cell transcriptional heterogeneity in the presence of microglia (Extended Data Fig. 6c), showed an increased proportion of the exhausted cluster and identified a microglia-affected GD2 CAR T cell population (TMA), aligning with undifferentiated TILs, as well as naive and tissue-resident memory cells, from the pan-cancer TIL atlas46 (Extended Data Fig. To validate the low effector profile of the TMA population, we compared curated gene signatures from the same resource46 across GD2 CAR T cell clusters (Fig. TMA cells showed reduced activation and effector signatures, including cytotoxicity, but higher senescence-related genes. Thus, the presence of microglia shifts the transcriptional profile of GD2 CAR T cells toward reduced effector function. In line with this, when we monitored CAR T cell-mediated tumor control, it was reduced in integrated-microglia DMGOs (Fig. Together, these findings demonstrated that microglia integrated in DMGOs yield a suitable representative phenotype for probing CAR T cell function and toxicity. This establishes a direct role for microglia in shaping CAR T cell functional states and impairing tumor control. a, Representative immunofluorescence 3D images of a 200-µm-thick slice containing microglia, 1 week after start of GD2 CAR T cell treatment. Bottom, zoomed-in view of area in white inset (n = 2 DMGOs). b, Heat map depicting the fold change in concentration of selected cytokines, chemokines and growth factors of DMGOs containing microglia normalized against no microglia on day 3, day 7 and day 14 after GD2 CAR T cell addition (n = 6 DMGOs). c, Percentage of cells within GD2 CAR T cell clusters, including a new microglia-affected cluster, for nonexposed GD2 CAR T cells (left), GD2 CAR T cells retrieved from DMGO without (middle) or with integrated microglia (right). d, Heat map highlighting the average scaled expression of curated TIL gene signatures46 in the TMA GD2 CAR T cell cluster. e, Reduction in tumor area (normalized z score per DMGO relative to time point 0) after the addition of GD2 CAR T cells in DMGO without (blue) or with (orange) integrated microglia. Data are shown as the mean ± s.e.m. Statistical analysis at each time point was performed using a linear mixed-effects model, accounting for experimental and organoid variation. For b–e, n = 6 DMGOs with microglia and n = 6 DMGOs without microglia from four independent batches. Here, we showed that morphogen-guided patterning with FGF4 and RA produces BrOs, characterized across region-specific neuronal and glial lineages through benchmarking against recent single-cell organoid and brain atlases34,35,36. BrOs enable spatial and temporal modeling of the developing brainstem, including pons-specific features, the regional origin and niche for H3.3K27M-altered DMG. We demonstrated an interplay between the H3.3K27M substitution and pontine glial fate in driving DMG tumorigenesis, resulting in an experimentally accessible organoid model that recapitulates features of DMG tumors. While DMG patient-derived organoids are emerging as platforms for drug testing60,76,77, our model offers a complementary approach to generate in vitro tumors for this rare and fatal disease, for which tissue samples are limited. Moreover, the use of iPS cells as a cell source establishes the potential for individualized modeling, supporting personalized drug evaluation and direct comparison to clinical outcomes. However, because the model is derived from hES cells and iPS cells, it is limited in recapitulating postnatal tissue, as reported for other neural organoids34,78. Furthermore, the current model does not capture complex intraregional interactions, which are particularly relevant in the pons—a key relay between the forebrain and motor or sensory pathways. Assembloid methodologies79,80, such as linking cortical organoids with BrOs and DMGOs, could improve neuronal health and lineage diversity while enabling modeling of DMG invasion and progression across brain regions in response to neural secretion18,81 and activity17,19,20,82. Such neuronal interplay should be further characterized in DMGOs in the future to validate the model for these critical processes, for instance, through live calcium imaging of synaptic signaling or retrograde labeling of GABAergic neurons using viral tracers17. Despite current limitations, the model provides a therapeutically relevant platform to interrogate CAR T cell function in DMG tissue-specific context. Prolonged treatment of DMGOs reflects the variable outcomes seen in individuals24,25,83, revealing CAR T cell heterogeneity and functional exhaustion. From this heterogeneity, we identified the most potent yet short-lived CAR T cell population and validated a means to enrich for these cells, offering a potential approach to optimize therapy composition84. This model could also be leveraged to uncover how CAR T cells modulate cancer cell states and reveal mechanisms of acquired resistance that may be cotargeted to improve clinical efficacy. Given the recognition that the tumor microenvironment can significantly impact treatment response, we integrated microglia, a key immune component in DMG70. In line with representative tumor cell states observed in DMGOs, microglia differentiated into DMG-specific and immunosuppressive profiles72. This enabled an experimental interrogation of microglia impact on CAR T cell functionality, revealing increased exhaustion and a TMA population, marked by stalled differentiation and low effector function. This shift toward dysfunction correlated with reduced tumor control, offering a framework to counteract microglia-induced resistance and enhance antitumor efficacy. While T cell exhaustion might be addressed through immune checkpoint inhibition, strategies targeting the newly identified TMA population require further investigation. Our imaging data revealing direct microglia–CAR T cell interactions, suggest that dissecting the underlying signaling involved could be a critical starting point. Furthermore, similar approaches could be used to assess microglial influence on tumor phenotypes, as prior studies linked MES tumor states to tumor-associated macrophages8. Altogether, we generated a human brainstem organoid model for DMG with applications toward understanding CAR T cell functionality in the context of the tumor microenvironment that could aid further therapy development for this detrimental disease. All murine experiments were conducted in compliance with the Animal Welfare Body of the Princess Máxima Center for Pediatric Oncology based on local and international regulations under CCD license AVD39900 202216507. For the use of DMG samples, patients and/or their parents or guardians provided written informed consent according to national laws and in agreement with the declaration of Helsinki (2013). This study was approved by the Institutional Review Board (IVB) and registered under national registry number 2020.142. Brain organoids were generated from three different cell lines encompassing H9 (WA09, WiCell) and H1 (WA01, WiCell) hES cells (both derived from human blastocysts85) and C7-a iPS cells (RUID 06C52463, derived from CD4+ T cells). The iPS cell line C7-a was obtained from Rutgers University Cell and DNA Repository and contains a Cre-inducible H3.3K27M reading frame in the endogenous H3F3A locus13. Cell lines were cultured in mTeSR Plus medium (StemCell Technologies, 100-0276) and incubated at 37 °C with 5% CO2. The cells were grown on Matrigel-coated (Corning, 354277) six-well plates and passaged when 70–80% confluent by nonenzymatic detachment of colonies using Gentle Cell Dissociation Reagent (GCDR) (StemCell Technologies, 100-0485). All cell cultures were routinely tested for the presence of Mycoplasma species. Cells were washed with 1× Dubecco's PBS (Gibco, 14190144), detached with GCDR and spun down at 300g for 5 min, before resuspending in base medium (1:1 advanced DMEM/F-12 medium (Gibco, 12634010) and Neurobasal medium (Gibco, 10888022), supplemented with 1× GlutaMax (Gibco, 35050061)) and counting. A total of 70,000 cells per ml were added to day 0 medium (base medium, 10 µM Y-27632 (ROCKi, AbMole BioScience, M1817) and 4 ng ml−1 FGF2 (PeproTech, 100-18 C)). For EB formation, 7,000 cells in 100 µl of medium were seeded per well of an ultralow-attachment treated U-bottom 96-well plate (Nexcelom, ULA96U020; PHC Europe, MS-9096UZ) and incubated at 37 °C with 5% CO2. From day 2 to day 21, patterning medium (base medium, 1× N2 (Gibco, 17502048) and 1 mg ml−1 heparin solution (StemCell Technologies, 07980)) was used. In week 1, week 1 medium (patterning medium, 50 ng ml−1 FGF2, 1 µM dorsomorphin (DM; StemCell Technologies, 72102), 10 µM SB431542 (SB43; StemCell Technologies, 72232) and 3 µM CHIR99021 (CHIR; StemCell Technologies, 72052)) was used. On day 2, 100 µl of week 1 medium was added per well. On day 11, EBs were embedded in 12 µl Matrigel droplets and five droplets were transferred to each well of a 12-well suspension plate (Greiner Bio-One,665102) with 1 ml of week 2 medium and incubated at 37 °C with 5% CO2. In week 3, week 3 medium (patterning medium, 10 ng ml−1 FGF4, 10 µM RA and 1 µM PMA) was used. On day 17, the plates were placed on an orbital shaker inside a 5% CO2 incubator at 37 °C. PCAGPbase, PBCAG_DNp53_IRES_luciferase, PBCAG_PDGFRA-D842V_IRES_eGFP and PBCAG_H3K27M_eGFP plasmids were used to induce tumor growth in hES cell-derived organoids. In iPS cell-derived organoids, the H3K27M-expressing plasmid was replaced with 1.00 µg µl−1 SSi-Cre. PCAGPbase and PB_Venus were used as a control. All plasmids were kindly provided by the T. N. Phoenix laboratory11. For genetic lineage tracing, 1.50 µg µl−1 TrackerSeq86 was added to the plasmid mix. On day 11, unless stated otherwise, organoids were injected with a mixture of plasmid DNA (1.50 µg µl−1 per plasmid) and 0.1% (w/v) FastaGreen (Merck, F7252-5G) using a FemtoJet 4i (Eppendorf, 5252000013) with the following parameters: injection pressure, 15 hPa; compensation pressure, 5 hPa. Subsequently, electroporation was performed using a NEPA21 Super Electroporator (Nepagene) and CUY650P1 (Nepagene) tweezers with the following parameters: voltage, 50 V; pulse length, 10 ms; pulse interval, 50 ms; number of pulses, four; decay rate, 10%. Using the impedance (kΩ) measurement of the NEPA21 Super Electroporator, voltage was automatically readjusted to optimize cell perforation and viability per individual organoid. Electroporation was performed by applying a shock twice in orthogonal direction and organoids were incubated at 37 °C with 5% CO2 for at least 2 h to recover. No statistical methods were used to predetermine sample sizes but they are close to those previously published87. NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ (NSG) mice (The Jackson Laboratory, 005557) were housed at 45–65% humidity and 20.5–23.5 °C with a 12-h light–dark cycle, in specific-opportunist-pathogen-free conditions using individually ventilated cages and sterile food and water ad libitum. Ten 3–4-week-old male and female NSG mice were anesthetized using isoflurane/O2 inhalation and transferred to a stereotaxic frame. After removing hair from the surgical site, a 1-cm incision was made in the skin to expose the skull and 3 mg kg−1 lidocaine was applied topically. Under a stereo microscope, a Dremel was used to drill a circular groove of 5 mm in the skull above the right cerebral cortex. Cortex buffer was applied before the dura mater and 2 mm3 of brain tissue was removed to accommodate the DMGO transplant. DMGOs were preselected on the basis of GFP signal 1–2 weeks after electroporation and, if too big in size, cut in half before transplantation. After surgery, 0.06 mg ml−1 carprofen was provided in the drinking water for 3–5 days and mice were monitored 2–3 times per week for signs of weight loss, lack of grooming and/or reduced mobility. If mice reached the study (21 days) or humane endpoint according to the monitoring of clinical symptoms, they were put under deep anesthesia by intraperitoneal injection of 75 mg kg−1 ketamine and 1 mg kg−1 medetomidine. Transcardiac perfusion was performed with PBS and 4% paraformaldehyde and, after resection, brains were cut into 300-μm sections using a vibratome. Staining, clearing and imaging were performed as described below. Antibodies used are listed in Supplementary Table 10. Organoids were fixed in 4% paraformaldehyde (Sigma-Aldrich, 441244) for 30 min at 4 °C, washed three times in PBST (1:1,000 Tween-20 in 1× PBS) for 15 min at 4 °C, embedded in 4% low-melting-point agarose (Invitrogen, 16520-050) and sliced into 100–250-µm sections using a Leica VT 1200 S vibratome. mLSR-3D was performed on the sliced organoids as described previously88. All combinations of primary and secondary antibodies used are listed in Supplementary Table 10. The slices were imaged using a Zeiss LSM 880 confocal microscope with a ×25 (numerical aperture (NA): 0.8) objective and Leica Stellaris with ×20 (NA: 0.75) and ×40 (NA: 1.3) objectives. Alternatively, intact organoids were fixed and cleared using the organic solvent-based vDISCO method89 and imaged using a Leica SP8 microscope with a ×16 (NA: 0.6) BABB-compatible objective. Routine histopathology procedures were followed to obtain FFPE tissue for World Health Organization standardized tumor classification. Immunohistochemical staining was performed on the Leica BOND RX fully automated research stainer using the bond polymer refine detection kit (Leica, DS9800). Stained tissue sections were analyzed by an experienced neuropathologist. The DNA methylation profile of a pooled DMGO sample consisting of three independent replicates was compared to cases of DMG, glioblastoma and PFA ependymoma obtained from published datasets90,91. Data were loaded in R (version 4.3.1), probe filtering was performed using package ChAMP92 and each array platform was processed separately (HumanMethylation450 or EPIC) using the ‘minfi' method93 and filtering out probes located on single-nucleotide polymorphisms or sex chromosomes or with detection P value > 0.01. Raw β values were merged using function combineArrays and normalized with method BMIQ94. In total, 10,000 probes with the highest s.d. were selected and the Pearson correlation between samples was calculated, weighted by the inverse of variance. This correlation matrix was used to compute a distance matrix, which served as the input for the Rtsne function from the Rtsne package. Organoid FFPE sections were deparaffinized in xylene (three times, 3 min each) followed by rehydration in a graded alcohol series for 1 min each (100% twice, 95% twice and 70% once). Sections were washed in deionized water (two times, 1 min each) and put in target retrieval solution, pH 9 (Agilent Dako). Antigen retrieval was performed for 40 min at 95 °C. Sections were allowed to cool to room temperature and washed for 5 min in deionized water followed by storage in PBS until further use. Cyclical immunofluorescence imaging was performed as previously described95. All combinations of primary and secondary antibodies used are listed in Supplementary Table 10. Imaging was performed on a Leica DMi8 Thunder imaging system with an HC PL APO ×20 (NA: 0.80) objective. Images of each cycle were aligned based on DAPI signal using a previously developed tool (https://github.com/Dream3DLab/CycFluoCoreg). The resulting composite images were imported into QuPath (version 0.4.4)96, where nuclei were segmented using a cell expansion of 2.5 μm. An object classifier using RandomTrees was trained for each marker on two separate images. These object classifiers were combined into a composite classifier that was applied to all images. The resulting dataset containing the count of classified cells in each image was exported to R for quantification and visualization. CD8 GD2 CAR T cells (14G2a GD2-4-1BBz CAR) and donor-matched mock-transduced CD8 T cells were produced as previously described98. CAR T cells and mock-transduced T cells were expanded using a rapid expansion protocol99 and cryopreserved after 14 days. T cells were thawed and rested in RPMI-1640 medium, supplemented with GlutaMax, 10% FBS (Thermo Fisher, 10500064), 1% Pen–Strep, 50 U per ml IL-2 (Miltenyi, 130-097-743), 2,000 U per ml IL-7 (Miltenyi, 130-095-367) and 50 U per ml IL-15 (Miltenyi, 130-095-760) for 3 days at 37 °C with 5% CO2. For selection of GD2 CAR T cells based on NCAM1 expression, a similar expansion protocol was used but without the addition of IL-15 and Daudi cells. After resting, cells were washed and stained for 30 min at 4 °C in flow cytometry (FC) buffer (1× PBS with 2% FBS) with live–dead fixable near-IR dead cell stain (1:1,000; Thermo Fisher), CD3–APC (1:80; BD BioLegend, clone SK7) and NCAM1–HiLyte-488 (1:200; QVQ, FSH-10B10). CD3+NCAM1− or CD3+NCAM1+ GD2 CAR T cells underwent fluorescence-activated cell sorting on a BD FACSAria II and were immediately used for experiments. Firstly, 4 months after tumor induction, DMGOs were untreated or treated with 500,000 CD8+ GD2 CAR T cells or mock-transduced CD8+ T cells per DMGO added on days 0 and 7. For prolonged treatment, GD2 CAR T cells were administrated on days 0, 8 and 15. Tumor size was monitored by imaging using a Leica Thunder DMi8 microscope with a ×10 objective. After THUNDER software-mediated computational clearing of the imaging data, tumor size for each time point was quantified using Fiji. Background signal, defined as GFP-negative areas within the organoid, was subtracted. Supernatant of the cocultures was collected and IFNγ concentration was measured with ELISA (R&D Systems, DY285B). Untransformed BrOs were similarly treated and organoid appearance was monitored by brightfield imaging. Firstly, 60 days after electroporation, a DMGO sample was dissected for the tumor region, mechanically dissociated and cultured for two additional weeks to expand tumor cells. Cells were retrieved from the culture plate using StemPro Accutase (Gibco, A1110501) and passed through a 70-µm Flowmi cell strainer (Merck, BAH136800070) to create a single-cell suspension. Dissociated cells were centrifuged at 500g for 5 min at 4 °C and resuspended and washed in FC buffer. Cells were either left unstained or stained with live–dead fixable near-IR dead cell stain (1:1,000; Thermo Fisher) and GD2–PE (1:200; clone 14.G2a, BD Biosciences, 562100) for 30 min at 4 °C. Cells were washed twice in FC buffer, acquired on a Sony SH800s (Sony Biotechnology) and analyzed using FlowJo software (version 10.9.0). Cell suspensions were filtered using a 70-µm Flowmi cell strainer and stained in FC buffer with CD3–APC (1:80; BD Biosciences, clone SK7) and live–dead fixable near-IR dead cell stain (1:1000; Thermo Fisher) for 30 min at 4 °C. CD3+ T cells were sorted on a CytoFLEX SRT benchtop cell sorter (Beckman Coulter) and immediately processed for scRNA-seq. Doublets were identified and removed using the scDblFinder package100, with default settings. The Seurat (version 4)101 workflow was used to normalize and scale reads and 3,000 highly variable genes were determined. Principal component analysis was performed using the ‘RunPCA' function. Clustering analysis was performed using the Seurat package ‘FindNeighbors' and ‘FindClusters' functions. To identify subpopulations, marker genes for each cluster were determined through the ‘FindAllMarkers' function. Markers (adjusted P value < 0.05) were examined to profile genes associated with known CD8 T cell subsets, as well as project previously published signatures. In addition, DEGs were used as input for GO enrichment analysis using the GO resource (https://geneontology.org). To integrate three batches of NCAM1-selected cells, we applied the Seurat-based canonical correlation analysis integration method. As integration features, we used 1,000 variable features from each dataset, along with DEGs between conditions to account for biological variability. 4e, we estimated the proportion of cells from these clusters for every cluster of the integrated dataset. Newly emerging populations were defined on the basis of differentially expressed markers and their origin, categorized by whether they originated from exposed, unexposed, NCAM1+ or NCAM1− populations. The GD2 CAR T cell identities were then transferred to this dataset with TransferData(), retaining only high-confidence predictions (score > 0.5). These transferred identities were used to calculate the proportion of TMA GD2 CAR T cells within each CD8+ TIL subset. To evaluate the expression of established T cell signatures, we used a gene signature specific to serial killer engineered T cells that we previously obtained60. Using the VISION R package103, we computed and visualized the overall enrichment of the identified gene set atop UMAP cell embeddings of our dataset. In addition, we projected our GD2 CAR T cell signatures onto a pan-cancer CD8 TIL atlas46 through T Cell Map (https://singlecell.mdanderson.org/TCM/) using the VISION package. For each GD2 CAR T cell subset, markers obtained through DEG analysis were filtered using an adjusted P value < 0.00001. PMPs were generated using an adjusted protocol104. Briefly, 70–80% confluent H1 stem cells were detached with GCDR. For EB formation, 7,000 cells were plated per well of an ultralow-attachment treated U-bottom 96-well plate (Greiner Bio-One, 650970) in mTeSR+ (StemCell Technologies, 100-0276) medium, containing 50 μM ROCK inhibitor (Y-27632; AbMole, M1817), 50 ng ml−1 bone morphogenetic protein 4 (StemCell Technologies, 78211), 50 ng ml−1 VEGF (PeproTech, 100-20-100ug) and 20 ng ml−1 SCF (Miltenyi Biotec, 130-093-991). On day 2, fresh medium was added and, on day 4, EBs were transferred to a six-well plate with X-VIVO 15 medium (Lonza, BE02-060F), containing 1× GlutaMax (Gibco), 1× Pen–Strep (Gibco), 100 ng ml−1 M-CSF (PeproTech, 300-25-50ug) and 25 ng ml−1 IL-3 (PeproTech, AF-200-03-10ug). The medium was refreshed once a week. PMPs were collected and 100,000–200,000 cells were added per brain organoid in maturation medium (1:1 advanced DMEM/F-12 (Gibco) and Neurobasal (Thermo Fisher) medium, 1× GlutaMax, 0.5× N2 (Gibco), 0.5× B27 without vitamin A (Gibco) and 1× Pen–Strep). Organoids were kept on a microtiter orbital shaker inside a 37 °C 5% CO2 incubator for 1–3 weeks for PMP integration and differentiation. For CAR T cell treatment experiments, organoids were sectioned into 200-µm-thick slices using a vibratome, transferred to a 24-well suspension plate in 750 µl of maturation medium and incubated at 5% CO2 and 37 °C for 3 days. A total of 50,000–200,000 PMPs were added per slice in 750 µl of maturation medium in a 24-well suspension plate for 7 days before adding 200,000 CD8+ GD2 CAR T cells in 750 µl of maturation medium. Tumor size during treatment was monitored by imaging using a Leica DMIL LED FLUO microscope with a ×10 objective. BrOs and DMGOs containing microglia and optionally treated with GD2 CAR T cells were dissociated 21 days after initial microglia incorporation as described above for the preparation of scRNA-seq and TrackerSeq libraries. Dissociated cells, control PMPs and unexposed GD2 CAR T cells were washed and stained in FC buffer with CD3–BV421 (1:100; BD Biosciences, clone SK7) and live–dead fixable near-IR dead cell stain (1:1,000; Thermo Fisher) for 30 min at 4 °C. CD3+ T cells and mScarlet+ microglia and PMPs were single-cell-sorted into 386-well plates containing well-specific barcoded primers (Single Cell Discoveries) on a Sony SH800s (Sony Biotechnology). Plates containing sorted cells were immediately snap-frozen on dry ice and processed for SORT-seq by Single Cell Discoveries. Cells were heat-lysed at 65 °C followed by complementary (cDNA) synthesis. All the barcoded material from one plate was pooled into one library and amplified using in vitro transcription. Library preparation was performed following the CEL-Seq2 protocol105 to prepare a cDNA library for sequencing using TruSeq small RNA primers (Illumina). The DNA library was paired-end sequenced on an Illumina Nextseq 500, with high output, using a 1× 75-bp Illumina kit (read 1: 26 cycles, index read: 6 cycles, read 2: 60 cycles). CSFE-labeled myelin debris, kindly provided by the L. Akkari lab74, was injected into PMP-integrated organoid slices using a glass needle and a FemtoJet 4i. The slices were immediately imaged on a Leica STELLARIS microscope at 37 °C and 5% CO2 overnight with a time interval of 5–10 min. Protein was detected in the culture supernatant by Luminex. Acquisition of data was performed using a FLEXMAP 3D system (Bio-Rad) with xPONENT 4.3u1 software (Luminex). Data analysis was performed using Bio-Plex Manager 6.2 (Bio-Rad). All assays were performed at the ISO9001:2008-certified MultiPlex Core Facility of the University Medical Center Utrecht. Statistics on bulk sequencing data were computed using built-in functions of R (‘stats', version 4.3.1) through a one-way analysis of variance with a post hoc Tukey honestly significant difference test. Statistics on electroporation efficiency and tumor induction were calculated using two-tailed independent t-tests (function: t.test). All statistical tests were performed with a confidence interval of at least 95% (α = 0.05). Data distribution was assumed to be normal but this was not formally tested, unless otherwise indicated, and no statistical method was used to predetermine sample size. To evaluate tumor response to GD2 CAR T cells in the presence or absence of microglia, the normalized tumor area at each time point was analyzed using a linear mixed-effects model, accounting for fixed and random effects related to batch and organoid variation. Multiple models were tested and the best-fitting model was selected. P values were adjusted for multiple comparisons using the false discovery rate (Benjamini–Hochberg). The investigators were not blinded to allocation during experiments and outcome assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw sequencing and methylation data that support the findings of this study were deposited to the European Genome–Phenome Archive under accession codes E-MTAB-15147 and E-MTAB-15559, respectively. Processed sequencing data were deposited to Zenodo (https://doi.org/10.5281/zenodo.16992353)106. Sequencing metadata are provided in Supplementary Tables 2–4, 6 and 9. Source data are provided with this paper. All used R and Python scripts for analysis are available from GitHub (https://github.com/Dream3DLab/DMGO_analysis). Pipelines for analyzing TrackerSeq data can also be found on GitHub (https://github.com/anna-alemany/TrackerSeq_BROs). Findlay, I. J. et al. Pharmaco-proteogenomic profiling of pediatric diffuse midline glioma to inform future treatment strategies. Schwartzentruber, J. et al. Driver mutations in histone H3.3 and chromatin remodelling genes in paediatric glioblastoma. Vuong, H. G., Ngo, T. N. M., Le, H. T. & Dunn, I. F. The prognostic significance of HIST1H3B/C and H3F3A K27M mutations in diffuse midline gliomas is influenced by patient age. CAR T cell therapies for diffuse midline glioma. Filbin, M. G. et al. Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq. The landscape of tumor cell states and spatial organization in H3-K27M mutant diffuse midline glioma across age and location. Pathania, M. et al. H3.3K27M cooperates with Trp53 loss and PDGFRA gain in mouse embryonic neural progenitor cells to induce invasive high-grade gliomas. Larson, J. D. et al. Histone H3.3 K27M accelerates spontaneous brainstem glioma and drives restricted changes in bivalent gene expression. Patel, S. K. et al. Generation of diffuse intrinsic pontine glioma mouse models by brainstem targeted in utero electroporation. Funato, K., Major, T., Lewis, P. W., Allis, C. D. & Tabar, V. Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation. Haag, D. et al. H3.3-K27M drives neural stem cell-specific gliomagenesis in a human iPSC-derived model. Regional identity of human neural stem cells determines oncogenic responses to histone H3.3 mutants. Pun, M. et al. Common molecular features of H3K27M DMGs and PFA ependymomas map to hindbrain developmental pathways. Stalled developmental programs at the root of pediatric brain tumors. Barron, T. et al. GABAergic neuron-to-glioma synapses in diffuse midline gliomas. Taylor, K. R. et al. Glioma synapses recruit mechanisms of adaptive plasticity. Venkatesh, H. S. et al. Neuronal activity promotes glioma growth through neuroligin-3 secretion. Venkataramani, V. et al. Glutamatergic synaptic input to glioma cells drives brain tumour progression. Lancaster, M. A. et al. Cerebral organoids model human brain development and microcephaly. Genetically engineered cerebral organoids model brain tumor formation. Hendriks, D. et al. Human fetal brain self-organizes into long-term expanding organoids. Monje, M. et al. Intravenous and intracranial GD2-CAR T cells for H3K27M+ diffuse midline gliomas. Majzner, R. G. et al. GD2-CAR T cell therapy for H3K27M-mutated diffuse midline gliomas. Qian, X. et al. Brain-region-specific organoids using mini-bioreactors for modeling ZIKV exposure. Muguruma, K., Nishiyama, A., Kawakami, H., Hashimoto, K. & Sasai, Y. Self-organization of polarized cerebellar tissue in 3D culture of human pluripotent stem cells. Andersen, J. et al. Generation of functional human 3D cortico-motor assembloids. & Rosenthal, A. FGF and Shh signals control dopaminergic and serotonergic cell fate in the anterior neural plate. Wurst, W. & Bally-Cuif, L. Neural plate patterning: Upstream and downstream of the isthmic organizer. Lu, J. et al. Generation of serotonin neurons from human pluripotent stem cells. Philippidou, P. & Dasen, J. S. Hox genes: choreographers in neural development, architects of circuit organization. Resolving organoid brain region identities by mapping single-cell genomic data to reference atlases. He, Z. et al. An integrated transcriptomic cell atlas of human neural organoids. Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Futile wound healing drives mesenchymal-like cell phenotypes in human glioblastoma. The Federative International Programme for Anatomical Terminology. Siletti, K. et al. Transcriptomic diversity of cell types across the adult human brain. Harutyunyan, A. S. et al. H3K27M induces defective chromatin spread of PRC2-mediated repressive H3K27me2/me3 and is essential for glioma tumorigenesis. Integrated molecular meta-analysis of 1,000 pediatric high-grade and diffuse intrinsic pontine glioma. et al. K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas. Siddaway, R. & Hawkins, C. Modeling DIPG in the mouse brainstem. Mariet, C. et al. Posterior fossa ependymoma H3 K27-mutant: an integrated radiological and histomolecular tumor analysis. Gillen, A. E. et al. Single-cell RNA sequencing of childhood ependymoma reveals neoplastic cell subpopulations that impact molecular classification and etiology. Gao, J. et al. Cellular localization of aquaporin-1 in the human and mouse trigeminal systems. Chu, Y. et al. Pan-cancer T cell atlas links a cellular stress response state to immunotherapy resistance. Zheng, L. et al. Pan-cancer single-cell landscape of tumor-infiltrating T cells. Szabo, P. A. et al. Single-cell transcriptomics of human T cells reveals tissue and activation signatures in health and disease. Anderson, A. C., Joller, N. & Kuchroo, V. K. Lag-3, Tim-3, and TIGIT: co-inhibitory receptors with specialized functions in immune regulation. Tinoco, R. et al. PSGL-1 is an immune checkpoint regulator that promotes T cell exhaustion. Shin, H. et al. A role for the transcriptional repressor Blimp-1 in CD8+ T cell exhaustion during chronic viral infection. Long, A. H. et al. 4-1BB costimulation ameliorates T cell exhaustion induced by tonic signaling of chimeric antigen receptors. Beltra, J.-C. et al. Developmental relationships of four exhausted CD8+ T cell subsets reveals underlying transcriptional and epigenetic landscape control mechanisms. Khan, O. et al. TOX transcriptionally and epigenetically programs CD8+ T cell exhaustion. Good, C. R. et al. An NK-like CAR T cell transition in CAR T cell dysfunction. & Sharma, S. The current state and future of T-cell exhaustion research. Trefny, M. P. et al. Deletion of SNX9 alleviates CD8 T cell exhaustion for effective cellular cancer immunotherapy. Impaired mitochondrial oxidative phosphorylation limits the self-renewal of T cells exposed to persistent antigen. & Wolchok, J. D. Clinical implications of T cell exhaustion for cancer immunotherapy. Dekkers, J. F. et al. Uncovering the mode of action of engineered T cells in patient cancer organoids. Ganesan, A.-P. et al. Tissue-resident memory features are linked to the magnitude of cytotoxic T cell responses in human lung cancer. Schenkel, J. M. & Masopust, D. Tissue-resident memory T cells. et al. Tissue-resident memory CAR T cells with stem-like characteristics display enhanced efficacy against solid and liquid tumors. Xu, R. et al. Human iPSC-derived mature microglia retain their identity and functionally integrate in the chimeric mouse brain. Sabate-Soler, S. et al. Microglia integration into human midbrain organoids leads to increased neuronal maturation and functionality. Haenseler, W. et al. A highly efficient human pluripotent stem cell microglia model displays a neuronal-co-culture-specific expression profile and inflammatory response. Brain organoid models for studying the function of iPSC-derived microglia in neurodegeneration and brain tumours. Hughes, A. N. & Appel, B. Microglia phagocytose myelin sheaths to modify developmental myelination. Gosselin, D. et al. An environment-dependent transcriptional network specifies human microglia identity. Park, D. S. et al. iPS-cell-derived microglia promote brain organoid maturation via cholesterol transfer. Matcovitch-Natan, O. et al. Microglia development follows a stepwise program to regulate brain homeostasis. Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function. Lin, G. L. et al. Non-inflammatory tumor microenvironment of diffuse intrinsic pontine glioma. Kloosterman, D. J. et al. Macrophage-mediated myelin recycling fuels brain cancer malignancy. Targeting TGFβ-activated kinase-1 activation in microglia reduces CAR T immune effector cell-associated neurotoxicity syndrome. Sarnow, K. et al. Neuroimmune-competent human brain organoid model of diffuse midline glioma. Deligne, C. et al. Establishing a living biobank of pediatric high-grade glioma and ependymoma suitable for cancer pharmacology. Long-term maturation of human cortical organoids matches key early postnatal transitions. Assembly of functionally integrated human forebrain spheroids. Kim, J. et al. Human assembloid model of the ascending neural sensory pathway. Qin, E. Y. et al. Neural precursor-derived pleiotrophin mediates subventricular zone invasion by glioma. Venkatesh, H. S. et al. Electrical and synaptic integration of glioma into neural circuits. Vitanza, N. A. et al. Intracerebroventricular B7-H3-targeting CAR T cells for diffuse intrinsic pontine glioma: a phase 1 trial. Albelda, S. M. CAR T cell therapy for patients with solid tumours: key lessons to learn and unlearn. A. et al. Embryonic stem cell lines derived from human blastocysts. Bandler, R. C. et al. Single-cell delineation of lineage and genetic identity in the mouse brain. Genetically engineered brain organoids recapitulate spatial and developmental states of glioblastoma progression. Ineveld, R. L. V. et al. Multispectral confocal 3D imaging of intact healthy and tumor tissue using mLSR-3D. Cai, R. et al. Whole-mouse clearing and imaging at the cellular level with vDISCO. Auffret, L. et al. A new subtype of diffuse midline glioma, H3 K27 and BRAF/FGFR1 co-altered: a clinico-radiological and histomolecular characterisation. Capper, D. et al. DNA methylation-based classification of central nervous system tumours. ChAMP: updated methylation analysis pipeline for Illumina BeadChips. Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. Teschendorff, A. E. et al. A beta-mixture quantile normalization method for correcting probe design bias in Illumina Infinium 450 k DNA methylation data. Watson, S. S. et al. Microenvironmental reorganization in brain tumors following radiotherapy and recurrence revealed by hyperplexed immunofluorescence imaging. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Ariese, H. C. R. et al. Comprehensive transcriptomic analysis of brainstem-regionalized organoids and associated diffuse midline glioma organoids. Andersch, L. et al. CD171- and GD2-specific CAR-T cells potently target retinoblastoma cells in preclinical in vitro testing. Germain, P.-L., Lun, A., Meixide, C. G., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Integrated analysis of multimodal single-cell data. & Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. DeTomaso, D. et al. Functional interpretation of single cell similarity maps. Large-scale production of human iPSC-derived macrophages for drug screening. Hashimshony, T. et al. CEL-Seq2: sensitive highly-multiplexed single-cell RNA-Seq. Ariese, H. C. R. & Alieva, M. Transcriptomic profiling of BrO, DMGO and GD2 CAR T cells: processed datasets. We thank the Princess Máxima Center Single-Cell Genomics Facility, the Leiden Genome Technology Center and Single Cell Discoveries for performing scRNA-sequencing, R. Moeniralam and E. de Boed from the Princess Máxima Center Pathology Diagnostic Laboratory for performing immunohistochemical staining and the FC facilities at the Princess Máxima Center and Laboratory of Translational Immunology and M. Nicolasen for cell sorting. We thank J. Lammers for providing gene expression data from patient material, Z. Odé for her collaboration on establishing cerebral organoids, H. R. Johnson and A. Zomer for assistance with in vivo experiments and J. Bunt for sharing his knowledge on neural development. This work was financially supported by the Princess Máxima Center for Pediatric Oncology, Oncode Institute, the Children Cancer Free (KiKa) Foundation (grant no. 473) and the Charlie Teo Foundation (Research Rebel-Alegra's Army grant). A.C.R was supported by an European Research Council starting grant (2018 project, no. was supported by a research grant from Stichting Proefdiervrij. These authors contributed equally: Nils Bessler, Amber K. L. Wezenaar, Hendrikus C. R. Ariese, Celina Honhoff. Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands Nils Bessler, Amber K. L. Wezenaar, Hendrikus C. R. Ariese, Celina Honhoff, Ellen J. Wehrens, Cristian Ruiz Moreno, Thijs J. M. van den Broek, Raphaël V. U. Collot, Daan J. Kloosterman, Farid Keramati, Mieke Roosen, Sam de Blank, Esmée van Vliet, Mario Barrera Román, Marcel Kool, Mariëtte E. G. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Hendrik G. Stunnenberg & Anne C. Rios Nils Bessler, Amber K. L. Wezenaar, Hendrikus C. R. Ariese, Celina Honhoff, Ellen J. Wehrens, Thijs J. M. van den Broek, Raphaël V. U. Collot, Daan J. Kloosterman, Sam de Blank, Esmée van Vliet, Mario Barrera Román & Anne C. Rios Farid Keramati, Lucrezia C. D. E. Gatti, Jürgen Kuball, Zsolt Sebestyén, Annelisa M. Cornel & Stefan Nierkens Hopp Children's Cancer Center (KiTZ), Heidelberg, Germany Division of Pediatric Neurooncology, German Cancer Research Center (DKFZ) and German Cancer Research Consortium (DKTK), Heidelberg, Germany Onco-Hematology, Cell Therapy, Gene Therapies and Hemopoietic Transplant, Bambino Gesù Children's Hospital, Istituto di Ricovero e Cura a Carattere Scientifico, Rome, Italy Max Planck Institute of Neurobiology, Martinsried, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and A.C.R conceptualized the work with critical input from H.C.R.A., A.K.L.W., N.D., C.R.M., H.G.S., C.H., M.A. grew pontine organoids and performed DMG tumor induction. analyzed bulk and scRNA-seq datasets with critical input from H.G.S. provided TrackerSeq and critical input for analysis of lineage-tracing experiments. performed multispectral 3D imaging using specific technology provided by A.E. integrated microglia and performed all GD2 CAR T cell experiments with help from A.M.C. provided protocols for T cell rapid expansion. analyzed DMGO CAR T cell treatment data, including scRNA-seq analysis of GD2 CAR T cells and microglia. provided healthy brain organoids as a refence dataset for iCNV analysis and knowledge related to brain organoid generation. offered critical computational support and infrastructure for data analysis. are listed as inventors on a pending patent related to the novel BrO model (P382144NL). are listed as inventors on a pending patent related to the development of marker-based T cell selection (P102253NL). The other authors declare no competing interests. Nature Cancer thanks Harry Hill and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Overview of patterning approach and corresponding representative brightfield images of brainstem organoids over time. n = 3 BrOs for early time points, n = 15 BrOs for timepoint 60 days, scale bar of single organoids and multiple organoids is 500 µm and 2 mm, respectively. b, Schematic representation of a human foetal brain in gestational week (GW) 5 with indicated morphogens influencing the differentiation of hindbrain rhombomeres (r) and their HOX gene code respectively. c, Heatmap showing relative bulk RNA expression of homeobox (HOX) genes at week 2 and 3 depicted as log2 fold change normalized to week 1. n = 3 pooled BrOs per batch. d, Boxplot representation of Spearman's rank coefficient between different organoid batches from week 2 to 12. e, PCA of organoids at different timepoints from week 1 to 12 (grey scale) and derived from hESCs (circles) or iPSCs (square). a, Unintegrated UMAP representation of developing brainstem organoids, colored by collection day. c, scIB integration benchmarking of different assessed integration methods, showing metrics for preservation of biological variation and batch correction. d, scPoli latent embedding of brainstem organoid cells, colored by timepoint (top) and snapseed annotation (bottom). e, scPoli integrated UMAP of brainstem organoids, colored by timepoint. g, Matrixplot of scPoli (HNOCA34) or scANVI (HDBCA35) mediated label transfer annotation compared to the final annotation. For panels a-g, n = 84 organoids in total with 5-24 organoids pooled per timepoint, see Supplementary Table S1 for details. b, Dotplot showing the expression of selected markers for each cell type annotation, colored by their respective cell class. FOXG1 is represented in bold, indicating absence of expression. c, Annotation of neurotransmitter transporters (NTT) for cells assigned as neuroblast or neuron. For panels a-d, n = 84 organoids in total with 5-24 organoids pooled per timepoint, see Supplementary Table S1 for details. e, Representative 3D confocal image of a 200 µm thick organoid slice at day 110 labelled for TUBB3 (orange) and TPH2 (blue). n = 2 BrOs. White insert indicates zoom area displayed on the right. f, Representative optical section of immunofluorescent 3D imaging of a 200 µm thick organoid slice at week 16 labelled for neurofilament (NF, white), GFAP (red-to-white gradient) and AQP4 (green) (left) or for DAPI (white) and OLIG2 (red) (right). g, Glycolysis scores over the HNOCA34 and BrO datasets. Dashed line represents the mean score over all HNOCA datasets. h, Glycolysis scores for the OPC (left) and Glioblast (right) lineages in the HNOCA and brainstem organoids (BrOs), statistically analyzed using a permutation test. i, DEG analysis comparing Oligo (left) and Glioblast (right) from the HNOCA and brainstem organoids (BrOs) to the HDBCA counterparts. k, Proportional brainstem organoid presence scores for the HDBCA, showing the ratio of cells being represented in brainstem organoids (BrOs) per annotated cell class. For panels g-k, n = 84 organoids in total with 5-24 organoids pooled per timepoint, see Supplementary Table S1 for details. a, b, Representative image of GFP expression (a; tumor-inducing mix) or control (b; PiggyBac backbone including CAG-mVenus) as a measure of tumor outgrowth at week 6. n = 3 DMGOs and 3 controls, scale bars = 1 mm. c, Representative 3D confocal image of a 300 µm brain section of a DMGO-transplanted NSG mouse developing tumor outgrow labelled for CD31 (red), tumor-GFP (green) and DAPI (grey). d, Representative routine histopathological characterization of a DMG patient tumor sample harbouring H3.3K27M, TP53 and PDGFRA mutations. Bottom panels; Haematoxylin and eosin (HE), glial fibrillary acidic protein (GFAP) and neurofilament (NF). e, Representative routine histopathological characterization of a DMGO at day 120. a, Violin plots showing the number of genes, number of counts, percentage mitochondrial genes and percentage of ribosomal genes after filtering on a sample-by-sample basis. b, Unintegrated UMAP representation of DMGO cells, colored by sequencing batch. c, iCNV profile of DMGO cells showing chromosomal aberrations compared to healthy cells from the BrO time course data shown in Fig. Orange depicts tumor and green represents healthy cells. d, Dotplot showing marker gene expression for different tumor states, color-coded for their respective annotation. e, Matrixplot with the obtained tumor state annotations compared to reference datasets7,8. f, Mapping scores of DMGO cells in Liu et al.8, showing high presence of the annotated cell states, except for cycling cells. For panels a-f, n = 14 DMGOs from 4 independent batches. a, Schematic representation of the applied approach for simultaneously recovering transcriptomic information, HTO hashtags and lineage barcodes from DMGOs on a single cell level. A nested PCR strategy was applied for the TrackerSeq barcode. b–f, Quality control assessment and filters used to select barcodes from experimental replicate 1 (top) and experimental replicate 2 (bottom). Histogram depicting the total number of UMI counts prior to any filtering (b). Histogram displaying read counts and the cut-off (red dashed line) set as a minimum total read count of log102 and log103 for experimental replicate 1 and 2, respectively (c). Scatter plot depicting read counts plotted against UMI counts and the applied threshold for minimum total read counts indicated (red dashed line) (d). Histogram depicting the mean oversequence per barcode and thresholding applied (dashed blue line) (e). Scatter plot depicting read counts plotted against max mean oversequence showing both thresholds applied (f). g, UMAP embedding of lineage-traced DMGOs and BrO controls. Cells are colored according to unique lineage barcodes. Cells without barcodes are presented in gray. h, Representation of each DMGO and BrO control in the final UMAP embedding. i, Bargraph depicting the total number of cells for each clonal barcode after applying the filtering of >3 cells per clonal family. j, Pie charts of the relative size of each recovered clonal family among all barcoded cells per sample used for the large versus small clone comparison. Percentage is depicted for clonal families that are equal to, or above 20%, which are defined as large clones. k, Dotplot showing normalized expression of AQP1 and AQP4 in DMG cells7. In contrast to AQP1, AQP4 - a canonical AC-like marker - was present in DMG tumors across all locations. l. Module scores of gene programs as derived from cNMF projected onto the UMAP of lineage-traced cells. m, Heatmap of Jaccard index scores, indicating the overlap of the cNMF derived programs to tumor state annotations. For panels b-j, l, m, n = 14 DMGOs and n = 2 BrOs, see Supplementary Table S1 for details. For panel l and m, n = 14 DMGOs and n = 2 BrOs, see Supplementary Table S1 for details. a, Representative brightfield images at day 14 of untransformed brainstem organoids (BrOs) left untreated (n = 8 BrOs) or exposed to GD2 CAR T cells (n = 8 BrOs) at day 0 and 7. b, DMGO tumor cell GD2 expression (orange) analyzed by flow cytometry compared to an unstained control (black). c, d, IFNγ levels measured in the culture supernatant (c) and GD2 CAR T cell treatment outcome measured as tumor GFP intensity relative to the start of treatment (day 0, 100%) with a smoothed line trend plotted between the values at different timepoints for each DMGO using the LOESS algorithm (d). DMGOs were either left untreated (gray line, n = 1 DMGO), treated with mock transduced T cells (black lines, n = 2 DMGOs), or GD2 CAR T cells (orange lines, n = 4 DMGOs). Arrows indicate the timepoints of T cell administration. e, Images of tumor GFP signal on day 0, 7, 10 and 14 for an untreated DMGO (n = 1 DMGO) and DMGOs treated with mock transduced T cells (n = 2 DMGOs), or GD2 GAR T cells (n = 4 DMGOs). GD2 CAR T cells and mock transduced T cells were administrated at day 0 and 7. a, Dot plot showing key marker gene expression (selected from the top 20 DEGs) across the GD2 CAR T cell clusters. Dot size is proportional to the percentage of cells expressing a gene and color intensity to the average scaled gene expression. b–f, Selected significant GO terms associated with the DEGs of the TUND (b), TIL-2 (c), TMI (d), TPR (e) and TMS (f) GD2 CAR T cell clusters. g, UMAP visualization of the CD8+ TIL clusters from the pan-cancer atlas46 used as a reference dataset. Annotated clusters are highlighted because of their overlap with, or use in defining the GD2 CAR T cell clusters. h–j, Marker gene signatures (DEG analysis adjusted p-value < 0.00001) of the TUND (h), TIL-2 (i) and TISG (j) GD2 CAR T cell clusters projected onto the CD8+ TIL dataset from g. For panels a-j, GD2 CAR T cells retrieved from n = 4 treated DMGOs. a, Percentage of cells within GD2 CAR T cell clusters, including those identified in Fig. Data is shown separately for GD2 CAR T cells retrieved from DMGO at day 14 after administration weekly (at day 0 and 7, left; n = 4 DMGOs) or once (at day 0, right; n = 2 DMGOs). b, UMAP representation of the integrated GD2 CAR T cell dataset, illustrating the distribution of non-exposed GD2 CAR T cells. c, UMAP embedding of DMGO-exposed (pink) and non-exposed (green) GD2 CAR T cells within the TEX cluster. For panel b and c, GD2 CAR T cells retrieved from n = 4 DMGOs and n = 2 independent batches of unexposed GD2 CAR T cells. d, Applied gating strategy (top panels) and obtained purity (bottom panels) of sorted NCAM1− and NCAM1+ GD2 CAR T cells. e, Representative images of tumor control measured by GFP imaging in DMGOs treated with sorted NCAM1− (top) or NCAM1+ (bottom) GD2 CAR T cells. f, Number of NCAM1− and NCAM1+ GD2 CAR T cells retrieved from each DMGO sample after two weeks of treatment. g, Proportion of cells within GD2 CAR T cell clusters, including those identified in Fig. Data is shown separately for NCAM1− GD2 CAR T cells (left) and NCAM1+ GD2 CAR T cells (right). h, Selected significant GO terms associated with the DEGs of the THS NCAM1− specific GD2 CAR T cell cluster. i, Upregulated marker gene signature (DEG analysis adjusted p-value < 0.05; avg_log2FC > 0) of the THS GD2 CAR T cell cluster projected onto the pan-cancer CD8+ TIL dataset46. a, Selected significant GO terms associated with DEGs in microglia derived from BrOs (left; n = 5 BrOs) or DMGOs (right; n = 4 DMGOs). b, Representative Immunofluorescent 2D images of DMGOs with microglia, 3 weeks after integration, labeled for DAPI (white), IBA1 (green), SPP1 (magenta) and CD163 (blue). c, Quantification of the percentage of CD163+ (left) or SPP1+ cells (right) in IBA1+ microglia in DMGOs. d, Gating strategy used to sort mScarlet+ PMP/microglia and CD3+ GD2 CAR T cells for scRNA-seq. e, Bargraph depicting the proportion of TMA GD2 CAR T cells matched to identified pan-cancer CD8 TIL subsets46. Microglia home to sites of myelin debris injection. Live 3D imaging of microglia (orange) and CFSE-labelled myelin debris (green). Areas of myelin debris injection are outlined by white dashed lines. Live 3D imaging of microglia (orange) and CFSE-labelled myelin debris (green). White arrows indicate myelin debris that is phagocytosed by microglia outlined by white dashed lines. Source data and mean gains for each HDBCA cluster in the BrO model. Source data and electroporation efficiency data, including statistical analysis, DEGs and selected GO terms for large versus small clones, gene lists derived from cNMF and oligo lineage signatures derived from Braun et al.39. Source data, tumor GFP intensity and fold enrichment in tumor control for NCAM1+ over NCAM1− GD2 CAR T cells. Source data and gene lists derived from cNMF. Source data and DEGs from the GD2 CAR T clusters. Source data and DEGs from the GD2 CAR T clusters. Source data and DEGs in microglia derived from DMGO compared to BrO. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. De novo H3.3K27M-altered diffuse midline glioma in human brainstem organoids to dissect GD2 CAR T cell function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/how-to-keep-your-new-years-resolutions-behavioral-science-tips-that-work/'>Why Most Resolutions Fail—And How Science Can Help You Stick to Yours</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Want to Make Your Resolution Stick This Year? Behavioral economist Katy Milkman explains why most New Year's resolutions fail and shares how science-backed strategies can build habits that last. I love the first few days of a new year. It evokes a feeling that change is possible. That feeling, in part, leads some of us to set New Year's resolutions. An estimated 40 percent of U.S. adults set resolutions any given year. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. But it doesn't have to be that way, according to Katy Milkman, a behavioral economist at the Wharton School of the University of Pennsylvania. Katy says there are science-supported tools we can use to spark positive changes in our lives. Doing so involves not only asking high-level questions about what we want to achieve and why but also finding ways to make the path toward achieving those goals, well, fun. Pierre-Louis: In your book you talk about how moments like a new year, even a move, you talk about them as fresh starts, and can you talk a little bit about what a fresh start is, and how they can be useful in changing our behaviors and even, also, their limitations? The “fresh start effect” is one of my favorite topics I've ever studied. It's work I did with Hengchen Dai of [the University of California, Los Angeles,] and Jason Riis, a former colleague here at Wharton. We got really interested in this after I visited Google in, I think, it was 2012, and learned that they were struggling to motivate their employees to take advantage of lots of wonderful benefits. And they had brought in a bunch of outside speakers to share insights about what could be done to sort of nudge people towards making positive change; I was one of them. And after I presented some research I got this great question from a member of the audience about whether there are moments when people are more open to making a change in their life, and that's what kick-started this research agenda. The immediate response was, “I don't know. The research really hasn't looked at whether or not our motivation varies over time.” But my collaborators and I all immediately had a very strong intuition—and that's what drove us to work on this question—that there are moments that bring us added motivation; the first one, of course, that comes to mind is New Year's. But what we did is we started digging into the literature on the way that people think about their lives and what's sort of driving this effect. We learned there's a whole literature on what's called “autobiographical memory” and that the way we think about time is not linear. Instead, we actually think about our lives like we're characters in a novel and there are chapter breaks in that storyline, if you will. And instead of thinking about every day being equally weighted, those chapter breaks are really momentous. We can say, “Oh, that was the old me, and the old me didn't do XYZ that I wanted to do, but the new me will be different.” It gives us optimism about what we're capable of, and also, with that sense of possibility, we often become more reflective at these chapter breaks and do big picture thinking. So what was really interesting in our work, though, is these chapter breaks don't just come at major life shifts. It's not a major chapter break in your life story ... We also found that there are other moments on the calendar like New Year's that haven't been as widely discussed that have the same effect to a smaller degree, so every Monday is a miniature fresh start ... Milkman: The start of a new month, celebrating birthdays and other holidays that we associate with new beginnings, so that might be Easter or Rosh Hashanah, Eid—so each religion has its own marker of new beginnings. And all those dates tend to spur positive behavior change. In our research we've looked at when people show up and attend the gym; when people search for the term “diet” on Google, which is the most popular New Year's resolution, for better or for worse; and also, we see it when we look at when people set goals on a popular goal-setting website online about everything from their health to their finances and the environment. Pierre-Louis: So one of the studies that you reference in your book about baseball players who get traded is that fresh starts aren't always positive, right? Can we talk a little bit about that? Milkman: Yeah, so Hengchen did a number of experiments and also analyzed data on Major League Baseball players. What she was interested in is the fact that trades in Major League Baseball have different implications for your performance statistics depending on whether you're traded across leagues or within leagues. But if you're traded within league, all of your season-to-date statistics are retained, and you just keep working on that baseline. She was interested in these two people who essentially are experiencing the same thing—they're both moving to a new city; they're both working with new teammates—but one of them has a much bigger fresh start than the other. Is there a difference in how that affects them? So two people, both get traded, both have been performing really well, but one of them has to deal with the fact that they're starting their season over in terms of their statistics, that person's harmed more than the person who gets to hold on to their record. On the flip side, though, for two players who were both underperforming season to date and they both get traded—one of them gets to hold on to their record, and the other doesn't—the clean slate is beneficial and improves the performance of the person who gets that clean slate when they've been having a tough season. So this is sort of the double-edged sword of a fresh start in a very nice field study, showing that when things are going well, these kinds of fresh starts and disruptions can be harmful. Pierre-Louis: And it felt like, to me, that kind of underpinning it is a question of, like, “What happens to our habits?” And when we're trying to make a big change, in many ways, what we're trying to do is change our habits, right? And so I think that's the thing most people struggle with, is: “How do we develop and maintain these consistent habits, and when we get disrupted how do we get back to it?” Can you talk about some tips and techniques that people can use? It's no longer going to be automatic to engage in the same sets of routines when you come back, and so you need to be deliberate about planning: “Okay, when I get back how am I going to start up this habit that I had built again? How am I gonna make sure it's worked into my schedule?” That can be through making explicit plans—this is boring but important—like, you know, when are you gonna do it? And it can also be by being deliberate about using some of the other tools we know help a lot with habit formation, like ensuring it's rewarding and that you have a fun way to get it done. So exercise, the example is, like, you know, “I only let myself binge-watch my favorite TV shows while I'm exercising.” Maybe you only get to listen to your favorite podcast or open your favorite bottle of wine when you're cooking a fresh meal for your family, and you set that as a rule when you wanna get back on that habit. Maybe it's somebody who you're going to exercise with, right? And in fact, research shows that when you have a workout buddy, that can boost your likelihood to follow through by both making it fun and ensuring you're accountable to someone. But basically, you need to use these tools that we know help us start healthy habits to a greater degree after a disruption. But something that kind of struck me when I was reading that portion of the book is: sometimes it feels like people are setting goals for themselves that they, like, want to want to set; they don't necessarily actually want it. But in their head, because of the way we talk about exercise in society, it becomes very prescriptive, so they're like, “I should run a marathon because that's what fit people do.” But it also seems like sometimes, especially with New Year's resolutions, we kind of lose a forest for the trees, where we're focusing on a specific form of exercise or a specific type of diet and not sort of a bigger picture of, like, what healthy eating looks like or what exercise can look like and finding the joy in things we naturally want to do. Milkman: I love that takeaway from the book and 100 percent agree with you that one of the things we can do is just step back and think big picture about whether we're setting the right goals and what the higher-level goal is and if there's another path to the higher-level goal that's more likely to work, so if your high-level goal is “be in shape this year,” and you've chosen to pursue it in an unpleasant way that you don't like, then step back and ask, “Can I do something that I will enjoy more to get to the same outcome?” Because one of the best predictors of success is whether you enjoy the process of pursuing your goals. So yes, ice-skate rather than running a marathon [Laughs] if that will bring you joy. Any way you move your body is good for you, whether it's going to dance class with a friend, taking a walk in the morning in the fresh air with a cup of coffee and someone you enjoy talking to—you know, making it social is another really important way to improve how much we enjoy goal pursuit, and the same is true for eating right and, and, frankly, achieving goals at work. Pierre-Louis: One of the things that I thought was really interesting is that, like, we talk about making things enjoyable, but you also talk about self-imposed constraints and how sometimes, to execute on a goal that we want, we can choose to opt in to constraints to reach that goal. Can you talk a little bit about that? Milkman: Yeah, I think this is some of the most counterintuitive but powerful research in behavioral science and goal pursuit. The idea is really, you know, we know how useful it is when we have a great boss or a great teacher or a great parent who is, you know, holding our feet to the fire and saying, like, “These are the deadlines. But what we often, I think, fail to appreciate is that we have the power to be our own boss [Laughs], our own teacher or our own parent and create constraints and deadlines with consequences in a way that will motivate us and help us achieve more. So let me give you a really concrete example of a study that I think illustrates just how powerful this way of thinking can be. This is a study that was done by Dean Karlan of Northwestern University and collaborators where they were looking at whether they could help people quit smoking ... Milkman: So, like, a really tough goal, right? The tool was: randomly assign people to either get a standard smoking-cessation program or that standard program plus what we call a commitment device. A commitment device, in this case, meant a savings account that you could put your own money into—it's optional—but you learn that if you fail a nicotine or cotinine test in your urine six months later, all the money will be taken away. And what the researchers found is: those who had access to this account, they quit at a 30 percent higher rate than the standard group. And there are many ways you can do this. And they can also be as simple as just creating friction in your life, so it doesn't have to involve money. So there's a lot of different tools we can use that fall into this category of creating constraints for ourselves, behaving like our own boss, in order to set ourselves up for success with our goals. Pierre-Louis: One of the things I really appreciated in your book is when you talked about how people will often harp on the fact that [more than] 80 percent of people who set New Year's resolutions fail, but that means 20 percent succeeded, right? And you can give yourself a better probability of success, though, if you do more than just saying, “I'm gonna try to be healthy this year,” “I'm gonna try to raise my performance at work,” or “I'm gonna try to improve my relationships with my family.” Be thinking about: What's a measurable goal that you wanna achieve? And then, P.S., if it doesn't work out this time, that doesn't mean New Year's resolutions are a bad idea next time. And P.S., you can make a new resolution next Monday, the beginning of the next week, on your birthday or on any arbitrary day because all of this is in our head about fresh starts anyhow. So give yourself some grace and try your best, and then if it doesn't work out, try again the next time. Pierre-Louis: Thank you so much for joining us today. Milkman: Yeah, thank you so much for having me. Don't forget to tune in on Wednesday, when we look at how the Trump administration's policies are impacting children's health. Science Quickly is produced by me, Kendra Pierre-Louis, along with Fonda Mwangi, Sushmita Pathak and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. She has worked for Gimlet, the Bloomberg and Popular Science. She previously worked at NPR and was a regular contributor to The World from PRX and The Christian Science Monitor. Fonda Mwangi is an award-winning multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, the Recount and WTOP News. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-02018-0'>)−H coupling of N -alkyl anilines via metallaphotoredox catalysis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 10:24:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Chemistry (2026)Cite this article Amine functionalization is crucial in pharmaceutical and agrochemical synthesis yet direct enantioselective α-C(sp3)–H functionalization of N-alkyl anilines remains challenging. Here we show a metallaphotoredox-catalysed radical approach for α-C(sp3)–H arylation of N-alkyl anilines, introducing a simple, sterically hindered aryl ketone photocatalyst. This key innovation slows undesired back-electron transfer, enabling efficient α-anilinoalkyl radical generation. Our strategy uses a sequential single-electron transfer and proton transfer process, thereby overcoming multiple limitations of existing methods. In conjunction with a chiral nickel catalyst, we have achieved site-selective, enantioselective arylation of diverse N-alkyl anilines with various (hetero)aryl halides. The method exhibits exceptional functional group tolerance, enabling modular functionalization of complex molecular structures. This approach provides an effective route to valuable α-aryl amines, offering significant possibilities for drug discovery and streamlining challenging synthetic sequences. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available within the article and Supplementary Information. Details about materials and methods, experimental procedures, characterization data, mechanistic studies, NMR and HPLC spectra are available in the Supplementary Information. Crystallographic data for the structures reported in this article have been deposited at the Cambridge Crystallographic Data Centre (CCDC), under deposition numbers 2383177 (39) and 2383178 (17). Copies of the data can be obtained free of charge via https://www.ccdc.cam.ac.uk/structures/. Bhutani, P. et al. US FDA approved drugs from 2015–June 2020: a perspective. Google Scholar Marshall, C. M., Federice, J. G., Bell, C. N., Cox, P. B. & Njardarson, J. T. An update on the nitrogen heterocycle compositions and properties of US FDA-approved pharmaceuticals (2013–2023). Google Scholar McGrath, N. A., Brichacek, M. & Njardarson, J. T. A graphical journey of innovative organic architectures that have improved our lives. Google Scholar Cernak, T., Dykstra, K. D., Tyagarajan, S., Vachal, P. & Krska, S. W. The medicinal chemist's toolbox for late stage functionalization of drug-like molecules. Google Scholar Lovering, F., Bikker, J. & Humblet, C. Escape from flatland: increasing saturation as an approach to improving clinical success. Google Scholar Chen, W., Ma, L., Paul, A. & Seidel, D. Direct α-C–H bond functionalization of unprotected cyclic amines. Google Scholar Dutta, S., Li, B., Rickertsen, D. R. L., Valles, D. A. & Seidel, D. C–H bond functionalization of amines: a graphical overview of diverse methods. Google Scholar Brown, D. G. & Boström, J. Analysis of past and present synthetic methodologies on medicinal chemistry: where have all the new reactions gone? Google Scholar Boström, J., Brown, D. G., Young, R. J. & Keserü, G. M. Expanding the medicinal chemistry synthetic toolbox. Drug Discov. Article PubMed Google Scholar Saint-Denis, T. G., Zhu, R.-Y., Chen, G., Wu, Q.-F. & Yu, J.-Q. Enantioselective C(sp3)‒H bond activation by chiral transition metal catalysts. Google Scholar Jain, P., Verma, P., Xia, G. & Yu, J.-Q. Enantioselective amine α-functionalization via palladium-catalysed C–H arylation of thioamides. Google Scholar Zhang, R. K. et al. Enzymatic assembly of carbon–carbon bonds via iron-catalysed sp3 C–H functionalization. Google Scholar Zhang, J., Huang, X., Zhang, R. K. & Arnold, F. H. Enantiodivergent α-amino C–H fluoroalkylation catalyzed by engineered cytochrome P450s. Google Scholar Ren, X. et al. Enantioselective single and dual α-C–H bond functionalization of cyclic amines via enzymatic carbene transfer. Google Scholar Shu, X., Huan, L., Huang, Q. & Huo, H. Direct enantioselective C(sp3)–H acylation for the synthesis of α-amino ketones. Google Scholar Shu, X., Zhong, D., Lin, Y., Qin, X. & Huo, H. Modular access to chiral α-(hetero)aryl amines via Ni/photoredox-catalyzed enantioselective cross-coupling. Google Scholar Li, J. et al. Enantioselective alkylation of α-amino C(sp3)−H bonds via photoredox and nickel catalysis. Google Scholar Golden, D. L., Suh, S. E. & Stahl, S. S. Radical C(sp3)-H functionalization and cross-coupling reactions. Google Scholar Holmberg-Douglas, N. & Nicewicz, D. A. Photoredox-catalyzed C−H functionalization reactions. Google Scholar Zhang, J. & Rueping, M. Metallaphotoredox catalysis for sp3 C–H functionalizations through hydrogen atom transfer (HAT). Article PubMed Google Scholar Bellotti, P., Huang, H.-M., Faber, T. & Glorius, F. Photocatalytic late-stage C–H functionalization. Google Scholar Lipp, A., Badir, S. O. & Molander, G. A. Stereoinduction in metallaphotoredox catalysis. Google Scholar Chan, A. Y. et al. Metallaphotoredox: the merger of photoredox and transition metal catalysis. Google Scholar & Huo, H. Photoinduced nickel-catalyzed enantioselective coupling reactions. Google Scholar Xu, W. & Xu, T. Dual nickel- and photoredox-catalyzed asymmetric reductive cross-couplings: just a change of the reduction system? Google Scholar Zuo, Z. et al. Enantioselective decarboxylative arylation of α-amino acids via the merger of photoredox and nickel catalysis. Google Scholar Cheng, X., Lu, H. & Lu, Z. Enantioselective benzylic C–H arylation via photoredox and nickel dual catalysis. Google Scholar Xu, S. et al. Enantioselective C(sp3)–H functionalization of oxacycles via photo-HAT/nickel dual catalysis. Google Scholar Hu, X., Cheng-Sánchez, I., Kong, W. Q., Molander, G. A. & Nevado, C. Nickel-catalysed enantioselective alkene dicarbofunctionalization enabled by photochemical aliphatic C–H bond activation. Google Scholar McNally, A., Prier, C. K. & MacMillan, D. W. Discovery of an α-amino C−H arylation reaction using the strategy of accelerated serendipity. Google Scholar Ahneman, D. T. & Doyle, A. G. C−H functionalization of amines with aryl halides by nickel-photoredox catalysis. Google Scholar Beatty, J. W. & Stephenson, C. R. J. Amine functionalization via oxidative photoredox catalysis: methodology development and complex molecule synthesis. Google Scholar Zhang, C., Li, Z.-L., Gu, Q.-S. & Liu, X.-Y. Catalytic enantioselective C(sp3)–H functionalization involving radical intermediates. Google Scholar Xu, G. Q., Wang, W. D. & Xu, P. F. Photocatalyzed enantioselective functionalization of C(sp3)–H bonds. Google Scholar Chen, X. & Kramer, S. Photoinduced transition-metal-catalyzed enantioselective functionalization of non-acidic C(sp3)−H bonds. Chem Catal. Google Scholar Dombrowski, G. W. et al. Efficient unimolecular deprotonation of aniline radical cations. Google Scholar Zhao, H. & Leonori, D. Minimization of back-electron transfer enables the elusive sp3 C−H functionalization of secondary anilines. Google Scholar Shen, Y., Gu, Y. & Martin, R. sp3 C–H arylation and alkylation enabled by the synergy of triplet excited ketones and nickel catalysts. Google Scholar Dantas, J. A., Correia, J. T. M., Paixão, M. W. & Corrêa, A. G. Photochemistry of carbonyl compounds: application in metal-free reactions. Google Scholar Mateos, J., Cuadros, S., Vega-Peñaloza, A. & Dell'Amico, L. Unlocking the synthetic potential of light-excited aryl ketones: applications in direct photochemistry and photoredox catalysis. Google Scholar Iziumchenko, V. & Gevorgyan, V. Recent advances in dual triplet ketone/transition-metal catalysis. Google Scholar Trowbridge, A., Reich, D. & Gaunt, M. J. Multicomponent synthesis of tertiary alkylamines by photocatalytic olefin-hydroaminoalkylation. Google Scholar Henry Blackwell, J., Harris, G. R., Smith, M. A. & Gaunt, M. J. Modular photocatalytic synthesis of α-trialkyl-α-tertiary amines. Google Scholar Waller, H. C. & Gaunt, M. J. Silyl radicals as single-electron reductants: α-aminoalkyl radical formation via a photocatalytic oxidatively initiated radical chain process. Ryder, A. S. H. et al. Photocatalytic α-tertiary amine synthesis via C−H alkylation of unmasked primary amines. Askey, H. E. et al. Photocatalytic hydroaminoalkylation of styrenes with unprotected primary alkylamines. Shen, Y. & Rovis, T. Late-stage N−Me selective arylation of trialkylamines enabled by Ni/photoredox dual catalysis. Yang, T. et al. Multicomponent construction of tertiary alkylamines by photoredox/nickel-catalyzed aminoalkylation of organohalides. Lin, Q., Spielvogel, E. H. & Diao, T. N. Carbon-centered radical capture at nickel(II) complexes: spectroscopic evidence, rates, and selectivity. Cabré, A., Verdaguer, X. & Riera, A. Recent advances in the enantioselective synthesis of chiral amines via transition metal-catalyzed asymmetric hydrogenation. Article PubMed Barrios-Rivera, J., Xu, Y., Wills, M. & Vyas, V. K. A diversity of recently reported methodology for asymmetric imine reduction. Tian, Y., Hu, L. A., Wang, Y.-Z., Zhang, X. & Yin, Q. Recent advances on transition-metal-catalysed asymmetric reductive amination. Trowbridge, A., Walton, S. M. & Gaunt, M. J. New strategies for the transition-metal catalyzed synthesis of aliphatic amines. Ni, S. et al. C–heteroatom coupling with electron-rich aryls enabled by nickel catalysis and light. Nayak, S. & Ackerman-Biegasiewicz, L. K. G. Harnessing electron-rich arenes in nickel photoredox catalysis. Diccianni, J., Lin, Q. & Diao, T. Mechanisms of nickel-catalyzed coupling reactions and applications in alkene functionalization. Grimm, M. L., Allen, W. J., Finn, M., Castagnoli, N. & Tanko, J. M. Reaction of benzophenone triplet with aliphatic amines. What a potent neurotoxin can tell us about the reaction mechanism. Simon, J. D. & Peters, K. S. Picosecond studies of organic photoreactions. Cohen, S. G., Parola, A. & Parsons, G. H. Jr. Photoreduction by amines. Capaldo, L., Ravelli, D. & Fagnoni, M. Direct photocatalyzed hydrogen atom transfer (HAT) for aliphatic C−H bonds elaboration. Guerin, B. & Johnston, L. J. Laser flash photolysis studies of 2,4,6-trialkylphenyl ketones. Chung, C.-H., Tseng, S.-L., Yang, Y.-N. & Chen, Y.-F. Use of glucagon receptor antagonists or inverse agonists for treatment of diabetes. International patent WO 2018/140338 A1 (2018). Chung, C.-H., Tseng, S.-L. & Hsu, H.-E. Processes for producing amide compounds, and their crystalline and salt form. International patent WO 2021/178486 A1 (2021). Sasaki, M. et al. Preparation of aromatic ring compounds as glucagon antagonists. International patent WO 2013/147026 A1 (2013). Conner, S. E., Li, J. & Zhu, G. N-(2-Carboxyethyl)benzamides as glucagon receptor antagonists or inverse agonists, their preparation, pharmaceutical compositions and use in therapy. International patent WO 2007/106181 A2 (2007). Ting, S. I. et al. 3d–d excited states of Ni(II) complexes relevant to photoredox catalysis: spectroscopic identification and mechanistic implications. Cagan, D. A. et al. Elucidating the mechanism of excited-state bond homolysis in nickel–bipyridine photoredox catalysts. Download references We thank H. Xu and Y. Yang for insightful discussions. We are grateful for financial support provided by the National Key R&D Program of China (2021YFA1502500 to H.H. ), the National Natural Science Foundation of China (22071203 and 22471228 to H.H. ), the Natural Science Foundation of Fujian Province (2022J01524 to C.Z.) and the Fundamental Research Funds for the Central Universities (20720240125 to H.H. State Key Laboratory of Physical Chemistry of Solid Surfaces, Key Laboratory of Chemical Biology of Fujian Province, College of Chemistry and Chemical Engineering, Xiamen University, Xiamen, China Weisai Zu, Xiang Wan, Haoran Wu, Jingwen Huo, Cankun Zhang, Chengyang Li, Yongliang Huang, Zhen Xu, Yumin Xu, Tao Li, Junliang Cheng, Jian-Liang Ye, Cheng Wang & Haohua Huo Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the reactions and performed the majority of synthetic experiments and mechanistic investigations. assisted with synthetic experiments and analysed the data. directed the project. wrote the paper with input from all authors. Correspondence to Haohua Huo. The authors declare no competing interests. Nature Chemistry thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. All yields and stereoselectivities represent the average of two experiments, and the percent yield represents purified products. See Supplementary Methods for experimental details. aAryl iodide was used as the electrophile. Supplementary Methods, Tables 1–9 and Figs. 1–320, experimental procedures and characterization data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zu, W., Wan, X., Wu, H. et al. Direct enantioselective C(sp3)−H coupling of N-alkyl anilines via metallaphotoredox catalysis. Download citation Received: 06 December 2024 Accepted: 29 October 2025 Published: 05 January 2026 Version of record: 05 January 2026 DOI: https://doi.org/10.1038/s41557-025-02018-0 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemistry ISSN 1755-4349 (online) ISSN 1755-4330 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41567-025-03122-1'>Geometry-driven asymmetric cell divisions pattern cell cycles and zygotic genome activation in the zebrafish embryo</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 10:08:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Early embryo geometry is one of the most invariant species-specific traits, yet its role in ensuring developmental reproducibility and robustness remains underexplored. Here we show that in zebrafish, the geometry of the fertilized egg—specifically its curvature and volume—serves as a critical initial condition triggering a cascade of events that influence development. The embryo geometry guides patterned asymmetric cell divisions in the blastoderm, generating radial gradients of cell volume and nucleocytoplasmic ratio. These gradients generate mitotic phase waves, with the nucleocytoplasmic ratio determining individual cell cycle periods independently of other cells. We demonstrate that reducing cell autonomy reshapes these waves, emphasizing the instructive role of geometry-derived volume patterns in setting the intrinsic period of the cell cycle oscillator. In addition to organizing cell cycles, early embryo geometry spatially patterns zygotic genome activation at the midblastula transition, a key step in establishing embryonic autonomy. Disrupting the embryo shape alters the zygotic genome activation pattern and causes ectopic germ layer specification, underscoring the developmental significance of geometry. Together, our findings reveal a symmetry-breaking function of early embryo geometry in coordinating cell cycle and transcriptional patterning. Developmental reproducibility and robustness are critical for the survival of a species. Understanding the foundations of this robustness is, therefore, a question of fundamental importance and has become a central focus of research. Although most studies have concentrated on the genetic and molecular mechanisms underlying development, the potential influence of geometry on developmental robustness has remained largely overlooked. Recent work using minimal active matter models has revealed that geometry and curvature play a pivotal role in the mechanical response of epithelial sheets1,2,3,4,5. These insights can, for example, account for the anisotropic distribution of myosin II observed in Drosophila embryos6. Such findings highlight geometry as a key organizer of mechanical force generation during morphogenesis. Yet, whether—and how—geometry governs more complex, large-scale developmental processes, such as embryo patterning and coordinated cell divisions, remains an open question. Metazoan development begins with a single cell, the zygote, which undergoes several rounds of rapid divisions (cleavages) to generate a large population of cells capable of adopting multiple fates and exhibiting diverse morphogenetic behaviours. In many species, these early divisions are initially highly synchronous. This cell cycle synchrony is best understood in Drosophila, where it is driven by the propagation of a chemical wave. In cleavage-stage Drosophila embryos, which are syncytial, nuclear cycles synchronize through ‘sweep waves'—coupling-independent phase waves emerging on top of gradients of cyclin-dependent kinase 1 (Cdk1) activity established through reaction–diffusion dynamics7,8. Similar mechanisms have been observed in Xenopus egg extracts, another syncytial system, where nuclear divisions are coordinated by trigger waves of Cdk1 activity9. By contrast, in intact Xenopus embryos, where cells are fully separated, early cleavage divisions synchronize independently of cell–cell coupling10. Importantly, across systems, the propagation of cell cycle waves slows progressively with successive division rounds, leading to ‘metasynchronization', where cell cycles remain spatially patterned before eventually becoming fully desynchronized7,10,11. Initial cleavages in the zebrafish embryo are meroblastic; that is, the cytoplasm but not the yolk is partitioned through cytokinesis11. Cell cycles at this stage occur highly synchronously, to begin with, and gradually desynchronize (metasynchrony) before completely desynchronizing at division round 10 (ref. Importantly, although the cleavage-stage cells cycle between only the S and M phases with no detectable gap phases, complete desynchronization around division round 10 is thought to occur mainly due to a heterogeneous introduction of gap phases to the cell cycles12. To determine the spatiotemporal pattern of cell divisions in cleavage-stage zebrafish embryos, we recorded time-lapse images of Tg(actb2:rfp-pcna) embryos. Due to the absence of gap phases in the cleavage stages, the disappearance of the nuclear red fluorescent protein (RFP)-proliferating cell nuclear antigen (PCNA) localization allowed us to reliably score the conclusion of the S phase and the onset of mitosis13. Consistent with previous reports, we observed three distinct phases with different extents of mitotic synchrony11,14. In the first three cell division rounds, cell cycles exhibited very high synchrony (synchronous phase) (Fig. Thereafter, cell divisions from division rounds 4–9 showed slight but clearly detectable delays (metasynchronous phase) (Fig. These delays increased throughout the metasynchronous phase, with cells in division round 4 dividing with a mean division timing variance of 0.10 min2 and in division round 9 with 3.43 min2 (Fig. In particular, as previously reported, these metasynchronous divisions occurred in a spatiotemporally patterned manner, forming a radial ‘mitotic wave' that originates near the animal pole (AP), where cells divide first and propagate towards the margin, where they divide last (Fig. Furthermore, consistent with the increasing variance in division timings during the metasynchronous phase, the speed at which this mitotic wave travels gradually decreased over consecutive division rounds (Fig. After division round 9, cells divided largely non-synchronously, marking the onset of the asynchronous phase. a, Schematic of a zebrafish embryo at the late cleavage stage. The last row of blastoderm cells (cyan) constitutes the margin. b, Line plot of cell division timing variance in Tg(actb2:rfp-pcna) embryos for cleavage division rounds 4–11. Grey lines, individual embryos; black line, mean across three embryos. The inset shows variance (mean ± s.e.m.) c, Contour plot of the radial mitotic wave across all surface cells at division round 8 in a representative wild-type embryo. Colours represent the division timing for each nucleus. d, Bar plot of mitotic wave velocities (mean ± s.d. ), calculated by generating distance–time plots, for cleavage division rounds 2–9, with the red dots showing wave velocities for individual embryos. In sequence from division rounds 2–9, n = 5, 10, 11, 12, 12, 12, 11 and 10 embryos, respectively. e,f, Representative contour plots of mitotic waves in division round 8 in embryos injected with either 0.3 ng of Histone 1 protein into a single blastomere at the 32-cell stage (e) or 12-pg chek1 mRNA at the 1-cell stage (f). g, Line plots showing the ratios of cell cycle periods at the margin and AP in division rounds 6–8 in individual embryos (grey lines) and across all embryos (green line, mean ± s.e.m.) The red dashed line indicates a ratio of 1, where both marginal and AP cells have the same cell cycle period. h, Bar plot of the M-phase and S-phase lengths (mean ± s.e.m.) Dots represent data from individual embryos. i, Box plot of S-phase duration in division round 8 as a function of cell position relative to the AP. To understand how cell cycles are seemingly coordinated into radial mitotic waves during the metasynchronous phase, we asked if their synchronization occurs via cell–cell coupling. To that end, we artificially desynchronized cell cycles at the 32/64-cell stage through mosaic Histone 1 protein injections, previously shown to effectively alter cell cycle progression17,18,19. We found that Histone 1-positive and thus desynchronized cells failed to resynchronize with their neighbouring control cells by division round 9, after which cell cycles normally desynchronize in control embryos (Fig. To address the possibility that these cell cycles could have resynchronized given sufficient time, we assessed the cell cycle progression in embryos ectopically expressing the Cdk1 inhibitor, Chek1, through chek1 mRNA injection in the one-cell stage embryo, which, by being heterogeneously distributed in the embryo, has previously been shown to interfere with cell cycle synchronization20. chek1-overexpressing embryos exhibited cell cycle desynchronization considerably earlier than control embryos, and failed to resynchronize despite completing all cleavage division rounds (Fig. Collectively, these observations suggest that cell cell coupling plays only a minor role, if any, in cell cycle synchronization in the zebrafish embryo. To determine by which coupling-independent mechanism the radial mitotic wave is formed, we analysed the evolution of the mitotic wave through consecutive division rounds. We found that the transition from synchrony to metasynchrony was gradual rather than abrupt, suggesting a progressive accumulation of delays rather than distinct switches leading to the observed loss of synchrony (Fig. Importantly, this gradual loss of synchrony was due to unequal cell cycle periods. For instance, between the metasynchronous division rounds 6–8, cells at the margin cycled with 2%–4% longer periods than those at the AP (Fig. This discrepancy arose exclusively due to a longer S phase (assuming absent gap phases), which increased in cells as a function of their distance from the AP (Fig. Although noisy, such period gradients may lead to the observed gradual slowing down of the radial mitotic wave by accumulating increasing delays in the marginal cells over consecutive division rounds. To test this hypothesis, we assumed that the period T is a linear function of the distance z from the AP: \(T={T}_{0}\,(1+\frac{kz}{L})\), where T0 is the typical period at the AP, L is the total AP–margin distance and k is a dimensionless constant of proportionality. This implies that the speed of the mitotic wave, v, in the nth division round is expected to be inversely proportional to n (Fig. To compare our experimental observations with these theoretical predictions, we first performed linear regression on the division timings to obtain the wave speed for each division round (Extended Data Fig. We then fitted the obtained v against \(\frac{1}{n}\) using the least squares fit. Using k = 1.9%, we obtained a close match between experiments and theory (Fig. 1d), suggesting that a 1.9% slower cell cycle at the margin would, in principle, be sufficient to explain the observed slowing down of the mitotic waves during consecutive division rounds. Considering that we experimentally observed a 2%–4% longer period at the margin, close to the theoretically predicted 1.9%, this suggests that a linear gradient of cell cycle periods, rather than cell–cell coupling, gives rise to the observed radial mitotic waves found during embryonic cleavages. We note that constant T and κ are simplifying assumptions, as cell cycle periods increase over rounds and exhibit embryo-to-embryo variations. Incorporating these variations would complicate our analysis due to the difficulty in accurately quantifying the appropriate parameter variations, without altering our qualitative conclusions. Hence, given the reasonable theory–experiment agreement (Fig. 1d) and in the spirit of minimal models, we maintain these assumptions throughout our model analysis, allowing us to focus on the consequences of cell–cell coupling mechanisms. Next, to challenge our conclusion that the mitotic waves are produced in a coupling-independent manner, we asked how they would appear if the cell cycles were instead coupled. To that end, we performed numerical simulations of the Kuramoto model of interacting oscillators on a hemisphere with free boundary conditions21 (Fig. The oscillators were placed on a Fibonacci lattice covering the hemisphere, with the nearest neighbours obtained with Delaunay triangulation. Due to cell–cell variations, we further assumed the presence of noise in the cell cycle lengths. Mathematically, we decomposed the frequencies into two parts \({\omega }_{i}=\varpi \left(z\right)+{{\wedge }}_{i}\), where the first term \(\varpi \left(z\right),\) is the inherent AP–margin oscillation gradient, and the second term is an independent spatial noise with covariance \(\left\langle {{\wedge }}_{i}{{\wedge }}_{j}\right\rangle ={\sigma \delta }_{{ij}}\). a, Schematic of our adapted Kuramoto model and the equation describing the evolution of the phase of a cell cycle. In the equation, θi represents the cell cycle phase of cell i, and Ni denotes its nearest neighbouring cells, determined through Delaunay triangulation of the hemisphere. The coupling strength ε represents how strongly each cell influences its neighbours' timing. The intrinsic frequency ωi represents how quickly cell i would progress through its cell cycle in isolation. In the model, each black dot represents a cell positioned on a hemispherical Fibonacci lattice. The dot size increases with height (z) to aid visualization in this top-down projection. Double-headed arrows between dots (shown for one representative cell) indicate coupled neighbours that can influence each other's cell cycle timing. b, Phase diagram showing the origin of waves in ε–σ space. The boundary between AP originated and margin originated is thresholded at z = 0.5 and the region of desynchronization corresponds to 〈eiθj〉j < 0.6, where 〈〉j denotes averaging over all sites. c, Contour plot of the radial mitotic wave across all surface cells at division round 8, as predicted using the Kuramoto model of interacting oscillators, assuming an AP–margin period gradient of 1.9% and exponentially decaying cell–cell interaction strength (ε = 0.08e−n, σ = 0.0016). d, Contour plot of the mitotic wave across all surface cells at division round 8, as predicted using the Kuramoto model of interacting oscillators, assuming an AP–margin period gradient of 1.9% and partial interaction between cells (ε = 0.08, σ = 0.028). e, Left: representative image of a 128-cell stage control Tg(actb2:lyn-tdtomato) embryo injected with 1 ng of Histone 1-Alexa Fluor 488 at the 1-cell stage to show the presence of a plasma membrane between individual nuclei. Right: contour plot of the radial mitotic wave across all surface cells at division round 8 observed in a control embryo, where nuclear divisions were visualized using either Histone 1-Alexa Fluor 488 injections or using Tg(actb2:rfp-pcna) embryos. f, Left: representative image of a 128-cell stage Tg(actb2:lyn-tdtomato) embryo injected with 1 ng of Histone 1-Alexa Fluor 488 at the 1-cell stage and treated with 50 μM of AZD1152 to show the absence of a plasma membrane between the individual nuclei. Right: contour plot of the mitotic wave across all surface cells at division round 8 observed in a representative syncytial embryo, where nuclear divisions were visualized using either Histone 1-Alexa Fluor 488 injections or using Tg(actb2:rfp-pcna) embryos. g, Phase diagram of the Kuramoto model in ε–σ space. The green solid line indicates the contour line of the observed variance at division round 8 in syncytial embryos. h, Variances of division timings for each round for a fixed noise value of σ = 0.028 as ε varies. The green dashed curve shows the variances measured in syncytial embryos. i, Variances of division timings for each round in wild-type embryos (orange) fitted with a curve described by exponentially decaying interaction strength, ε(n) = 0.08e−n, and low noise, σ = 0.0016 (blue). Specifically, simulations with small interaction strength (ε) and low noise (σ) qualitatively recapitulated the wild-type behaviour, where mitotic waves originated at the AP, and accumulated delays, causing a gradual slowdown over time (Fig. To reach a better quantitative agreement with the data, we considered that the initial cleavage events are incomplete, and consequently, the cytoplasm of early blastomeres remains interconnected through the yolk and via cytoplasmic bridges22. These connections are thought to progressively diminish over successive division rounds as cells become smaller—an assumption supported by our observation that single-blastomere injections of 40-kDa dextran, comparable in size to key cell cycle regulators such as Cyclin B1, Cdk1 and Chek1, resulted in its widespread distribution within a single division cycle when performed at the 16-cell stage but not at the 64-cell stage (Extended Data Fig. On the basis of this, we modelled wild-type embryos as exhibiting a time-dependent coupling that decays to near-zero values, which provided a substantially better fit to the experimental data than models assuming constant, minimal coupling throughout (Supplementary Section 4). By contrast, the introduction of greater interaction strength and noise led the system to self-organize into smooth mitotic waves originating at the margin, similar to the case in Xenopus egg extracts, even though the ‘natural' cell cycle lengths at the AP were preset to be shorter on average23 (Fig. For even larger values of noise, multiple waves emerged from random points of the simulated embryo (Fig. Overall, this theoretical phase diagram suggests that wild-type embryos are characterized by low-to-intermediate noise and progressively decaying coupling, and that increasing cell coupling should, in theory, shift the origin of the mitotic wave from the AP to the blastoderm margin. To experimentally test this prediction, we syncytialized cleavage-stage embryos by preventing cytokinesis—but allowing karyokinesis—through transient inactivation of Aurora kinase B24 (Fig. We reasoned that such a syncytialized embryo would resemble conditions observed in Drosophila embryos and Xenopus egg extracts—both syncytial systems in which cell cycles are partially or fully synchronized via coupling-dependent trigger waves—and would, therefore, function as a coupled system. Remarkably, consistent with our theoretical predictions, we found that in syncytial embryos, the mitotic wave originated at the margin rather than at the AP, as seen in wild-type embryos (Fig. Moreover, by analysing the wave characteristics at division round 8 in syncytial embryos, we were able to estimate the effective ratio of coupling and noise, two competing factors promoting synchronization and desynchronization, respectively (Fig. In particular, our model also predicted key differences in the evolution of cell cycle synchronization—measured by the variance in division timing across division rounds—between wild-type and syncytial embryos: in the absence of coupling (wild-type embryos), noise is expected to accumulate progressively, resulting in a monotonically increasing variance in division timing. By contrast, in the presence of coupling (syncytial embryos), this variance should plateau over time, with a characteristic timescale determined by the coupling strength (Fig. Strikingly, our experimental observations confirmed this prediction. In syncytial embryos—unlike in wild-types—‘cells' continued to cycle metasynchronously even beyond division round 10, indicating a strong coupling between cell cycles (Extended Data Fig. The temporal evolution of variance closely matched our theoretical prediction, particularly for ε = 0.08 and σ = 0.028 (Fig. By contrast, wild-type embryos exhibited continuously increasing variance, which could be quantitatively explained by the incomplete nature of cytokinesis during early cleavages, suggesting the presence of a transient coupling that rapidly diminishes as development progresses. Finally, to further challenge the notion of the increased coupling of cells in syncytial embryos, we injected chek1 mRNA into embryos before syncytialization and assessed whether they could resist premature desynchronization typically observed when chek1 was overexpressed in wild-type embryos (Fig. We theoretically reasoned that the coupling inferred for syncytial embryos is large enough that it can override a substantial increase in noise, such as the desynchronizing effects of chek1 expression (Fig. Indeed, we found that in chek1-overexpressing embryos, the mitotic waves originated at the margin and cells continued to cycle metasynchronously even beyond division round 10, suggesting that cells in the syncytial embryo are, in fact, tightly coupled (Extended Data Fig. By extension, this implies that such a coupling is absent/minimal in wild-type embryos and, consequently, that mitotic waves are produced largely independently of cell–cell interactions in wild-type embryos. Collectively, these findings suggest that cell cycles are only weakly coupled during cleavages and that metasynchrony arises predominantly by cell-autonomous processes. For the cell cycle to slowdown from the AP to the margin in a predominantly cell-autonomous manner, there must be a factor causing an unequal lengthening of the S phase along the AP–margin axis. A high N/C ratio causes S-phase lengthening, for instance, by retarding Cdk1 activation25. Consistent with such an effect of the N/C ratio on cell cycle length, we found that the length of the S phase across the metasynchronous division rounds negatively correlated with cell volume (Fig. Therefore, we hypothesized that an AP–margin gradient of cell volumes in the cleavage-stage embryo might be responsible for unequally lengthening cell cycles from the AP to the margin. To test this hypothesis, we measured cell volumes by semiautomatically segmenting surface cells at the 128-cell stage (division round 8). Indeed, we found that, on average, cells closer to the AP were significantly larger than those away from it (Figs. To determine at what stage such a volume gradient first becomes apparent, we also measured N/C in embryos at the 8-cell stage (division round 4) and the 16-cell stage (division round 5)—the first stages in which cells can be classified as being ‘central' or ‘peripheral' based on their position. Interestingly, we observed that cells, but not nuclei, closer to the AP were consistently larger than the marginal cells, leading to central cells typically exhibiting a smaller N/C ratio than peripheral cells (Fig. 3b,c and Extended Data Fig. Furthermore, the cell volume gradient not only persisted but was also further enhanced by subsequent cell divisions, as mother cells having unequal sizes themselves again divided asymmetrically to produce yet unequally sized daughters, thereby further diversifying cell volumes through cell lineage in addition to the already established position-driven cell volume diversity (Fig. For such unequal cell divisions to robustly give rise to a gradient in cell volumes along the AP–margin axis, upon division, the daughter cell closer to the AP must be, on average, larger than the daughter cell further from the AP. In line with this, we found that cell divisions in the early cleavage stages showed a tendency to produce larger daughter cells closer to AP, with the central daughter cell (closer to AP) being, on average, ~12.4% larger by volume than its peripheral sister cell (away from the AP) (Fig. a, Representative image of a 128-cell stage embryo (division round 8) with surface cells segmented (left) and line plots showing normalized cell volumes as a function of cell position with respect to the AP (right) in individual embryos. Cell volumes were internally normalized to the average cell volume at the AP in each embryo. n = 11 embryos; two-tailed Student's t-test with Benjamini–Hochberg correction for multiple comparisons. b, Representative image of an 8-cell stage embryo (division round 4) with surface cells segmented (top) and the box plot showing volumes of individual nuclei and cells based on their position relative to the AP (bottom). Central cells are positioned at the AP, whereas peripheral cells are positioned away from the AP. n = 20 central and 19 peripheral nuclei, 5 embryos; 12 central and 12 peripheral cells, 3 embryos. c, Representative image of a 16-cell stage embryo (division round 5) with surface cells segmented (top) and the box plot showing volumes of individual cells based on their position relative to the AP. n = 8 central and 24 peripheral cells, 2 embryos. d, Schematic showing the lineage relationship between cells at the 8-cell stage (top left) and 16-cell stage (bottom left). At the 8-cell stage, cells marked ‘C' are centrally positioned, and those marked ‘P' are peripherally positioned. At the 16-cell stage, cells are labelled to indicate both their positions (CD, centrally; PD, peripherally) and the identities of their mother cells (CM, centrally positioned mother cell; PM, peripherally positioned mother cell). Box plot showing the distribution of cell volumes at the 16-cell stage based on each cell's position and lineage identity (right). n = 8 cells for each category, 2 embryos. e, Schematic showing the pipeline for predicting daughter cell volumes at the 16-cell stage (left). Each segmented cell surface from the 8-cell stage was used to identify the longest cell axis, which was then bisected to obtain daughter cells at the 16-cell stage. Box plot showing a comparison between the ratios of volumes of the central and peripheral daughter cells obtained from each cell division, respectively, either predicted (Pred.) a, Line plots showing mean S-phase lengths in individual (thin, grey lines) and across (thick, black line, mean ± 95% confidence interval) embryos as a function of the inverse of the mean nearest-neighbour distance (mean of 3) in the metasynchronous cell cycle stages. Division rounds are indicated by the colour-bands overlaid on the dots (n = 6 embryos). b, Linear regression plots showing the relationship between the inverse of the nearest-neighbour distance (mean of 3) for each cell and the length of its S phase. Each colour represents data for a unique embryo. The thick, black line shows a summary best fit for the data of all cells across all embryos. For each embryo, cells only within the 5th–95th percentile of their nearest-neighbour distances and S-phase length were considered for the analysis, which minimizes erroneous measurements at the extremes but also underestimates the relationship (n = 6 embryos, 964 cells). c, Schematic of the experiment in which a fraction of the cytoplasm was aspirated from the blastodisc within approximately 10 min before the first cell division to increase the N/C ratio. d, Representative line plot showing the S-phase length in each division round in control embryos and embryos from which the cytoplasm was aspirated (mean ± s.d., n = 3 embryos). Only the nucleated, control half initially develops (that is, shows mitotic cycles) (middle) until nuclei from this half invade the enucleated half (bottom), which then begins to develop with a smaller N/C ratio. f, Representative line plot showing S-phase length in each division round in the control and enucleated halves of the embryo depicted in e (mean ± s.d., n = 3 embryos). g, Box plot showing normalized cell volumes as a function of cell position with respect to the AP. Cell volumes were internally normalized by dividing each cell volume by the volume of the largest cell in the embryo. Each dot represents the data for an individual cell. Data are the same as in Fig. 3a, but the cells here have been classified into only two bins. n = 432 cells, 11 embryos; two-tailed Student's t-test, ****P = 0.001. To determine what causes such patterned asymmetric cell divisions, we investigated how cell divisions are oriented and the cleavage plane positioned in the early embryo. Although several sophisticated models have been proposed to explain the orientation and positioning of the cleavage plane across systems, we started by first testing if following the simplest model, the ~140-year-old Hertwig's rule, can recapitulate the experimentally observed pattern of asymmetry27,28. More specifically, we assumed that the mitotic spindle in dividing blastomeres would orient along the longest cell axis, thereby dividing the cell along this axis. To test if such a geometry-driven mechanism would suffice to generate the observed pattern of asymmetric cell divisions and, therefore, the cell volume gradient, we predicted daughter cell volumes at the 16-cell stage by first identifying the longest axes of segmented cells at the 8-cell stage through principal component analysis and then bisecting them to obtain two daughter cells from each division29 (Fig. Indeed, we found that with this approach, the predicted asymmetry in the daughter cell size closely matched the observed asymmetry for most cell divisions, such that the central daughter cells (closer to the AP) at the 16-cell stage were, on average, ~18.5% larger than their peripheral sister cells (further away from the AP) by volume (Fig. Importantly, this tendency of central daughter cells being bigger than peripheral ones is determined by the specific geometry of the first cell of the fertilized egg, the blastodisc, which is delineated not only by a plasma membrane at its outer side but also by the interface to the yolk granules on its inner side11,30. As the interface to the yolk granules is curved, the first two cell divisions will give rise to equally sized daughter cells that are more pointed and narrow at their peripheral ends. Dividing these cells again along their long axis will then, as a result, generate daughter cells in which the central cell is larger than the peripheral one. This suggests that a gradient of cell volumes along the AP–margin axis in the cleaving embryo can be explained by the specific geometry of the first cell in the fertilized egg, leading to unequal cell division along the AP–margin axis during subsequent division rounds. Although the influence of cell volume on cell cycle length has been established in several systems, its role in zebrafish embryos remains unclear due to conflicting reports regarding the determinative effect of cell size in this context14,16. Consistent with the known role of elevated N/C ratios in prolonging cell cycles, we observed that, across different embryos, the S phase either lengthened prematurely or showed greater lengthening during division round 10, and the cell cycles desynchronized earlier than in control embryos (Fig. To test the converse, we decreased the N/C ratio by replicating a previous experiment14, aspirating one nucleus from embryos at the 2-cell or 4-cell stage (Fig. In these embryos, only the control half—containing the nucleus—underwent initial cleavages, whereas no divisions occurred in the enucleated half (Fig. However, in a subset of embryos, nuclei from the control half eventually invaded the enucleated half around division round 4 or 5, prompting the onset of cleavages in that region (Fig. In particular, although all nuclei had undergone the same number of division cycles by this stage, those in the previously enucleated half now resided in a substantially larger cytoplasmic volume—comparable to the blastomere volume of cells three to four divisions earlier—and, therefore, had a lower N/C ratio. Consistent with earlier findings, these nuclei did not exhibit S-phase lengthening during division round 10—the point at which cell cycles typically begin to desynchronize in the control half, but instead continued to cycle metasynchronously. This, in line with previous observations of delayed and premature cell cycle lengthening in haploid and tetraploid embryos, respectively, supports the notion that a higher N/C ratio promotes S-phase lengthening and contributes to the onset of cell cycle desynchronization beyond division round 10 (ref. However, it should be noted that although our analyses establish a deterministic role of N/C in S-phase length, they do not entirely exclude the possibility that additional geometry- or N/C-independent mechanisms may also influence the S-phase length. Having confirmed that the N/C ratio, in principle, can influence the S-phase length, we asked how perturbing the embryo-scale cell volume gradient within the cleaving embryo would affect the cell cycle (meta)synchrony and, consequently, the mitotic wave. To alter the cell volume gradient, we first generated bilobed embryos, displaying two ectopic (morphological) APs, by mounting 2-cell stage embryos in low-melting-point agarose (0.67% w/v in E3 buffer) when the first cytokinesis was in progress (Fig. We reasoned that spatially confining and, therefore, reshaping the embryo in this manner would produce larger cells away from the AP at the two ectopic APs. Interestingly, we found that not only the gradient of cell volumes was reshaped in the embryo but also that two mitotic waves emerged, one from each of the two ectopic APs, supporting the notion that the cell volume gradient is linked to the emergence of the mitotic wave (Fig. a, Representative images of 128-cell stage (division round 8) control and bilobed embryos with surface cells segmented (left) and box plot showing cell volumes as a function of the cell position with respect to the AP (right) for this bilobed embryo. Cyan asterisks mark the ectopic APs in the bilobed embryo. n = 37 cells, representative of five embryos. b, Montage of images (top) and a contour plot (bottom) showing the propagation of mitotic waves in a representative bilobed Tg(actb2:rfp-pcna) embryo at division round 8. Asterisks mark the ectopic APs in the bilobed embryo (n = 5 embryos). c, Schematic (top-left) and representative (bottom-left) images of a control embryo and an embryo in which the yolk has been severed at the 2-cell stage to increase the curvature of the blastoderm-to-yolk cell interface. Lines inside the cells indicate the longest cell axis along which the divisions (division round 3) are expected to occur. Cells marked in magenta and cyan represent the resulting central and marginal daughter cells, respectively. The contour plot (right) shows mitotic wave propagation in division round 8 of a yolk-severed embryo (AP view). d, Schematic (top) of the decreased angle of cell divisions in division round 4 in yolk-severed embryos due to an increased curvature of the blastoderm-to-yolk cell interface compared with the corresponding control embryos. Angles are represented as mean ± s.d. Line plot (bottom) showing the variance in cell division timings in control and yolk-severed embryos (mean ± s.e.m. e, Representative contour plot (top) showing mitotic wave propagation in a Chek1 inhibitor (PF477736)-treated embryo in division round 8 (AP view). Line plot (bottom) showing the variance in cell division timings in control and PF477736-treated embryos (mean ± s.e.m. Not only does cell size decrease along the AP–margin axis but also the proximity of cells to the yolk increases. This raises the possibility that yolk-derived factors, in addition to or alternatively to cell size, might influence the gradient in cell cycle length. However, direct cytoplasmic connectivity with the yolk is largely confined to the most marginal cells, making it unlikely to account for a cell cycle length gradient across the entire AP–margin axis11. To directly assess the effect of yolk proximity, we analysed correlations between division timing, cell size and yolk distance in bilobed embryos, where cell size and position relative to the margin are changed. Consistent with our model, smaller cells divided later than larger cells regardless of their AP–margin position (Extended Data Fig. 6), arguing against a primary role for the yolk in regulating division timing. On the basis of our analysis indicating that the cell volume gradient is a product of the curvature of the blastodisc–yolk interface, we reasoned that this increased curvature would reorient the longest axes of the two cells, such that they align more along the animal–vegetal axis, which would then amplify the volume asymmetry of cleavage divisions and, consequently, the phase difference between cell cycles (Fig. Strikingly, we found that in yolk-severed embryos, with cell divisions more aligned along the animal–vegetal axis, the mitotic wave indeed showed a greater cell division timing variance compared with the control embryos (Fig. Taken together, these observations strongly support the idea that a gradient of cell volume along the AP–margin axis, guided by embryo geometry, causes the generation of a gradient of S-phase lengths along the AP–margin axis of the blastoderm, thereby generating radial mitotic waves in the zebrafish embryo. Among the major responders to a high N/C ratio is Chek1, which, when active, delays the S–M transition7,25,31. Although cell cycles have been shown to lengthen independently of Chek1 in Xenopus egg extracts, recent in vivo studies in Drosophila embryos have established it as a central factor that not only controls the mitotic wave speed but also regulates the desynchronization of cell cycles in embryos with heterogeneous N/C7,26,31. Therefore, we enquired whether Chek1 mediates the effect of cell volume on S-phase length. To that end, we inhibited Chek1 activity by treating embryos with 10 µM of PF477736, a specific Chek1 activity inhibitor32. If Chek1 activity is required for the gradual metasynchronization of cell cycles through unequal S-phase lengthening, the cell cycle progression would be expected to show lesser variation in the treated embryos. Consistent with this, we found that, although Chek1-inhibited embryos produced mitotic waves like those in control embryos, they indeed exhibited a lower mitotic division timing variance (Fig. This suggests that Chek1 activity is critical for translating a cell volume gradient into a mitotic wave along the AP–margin axis of the blastoderm. Finally, we asked whether the geometry-derived cell volume gradient—and the resulting patterned mitotic metasynchrony—plays a developmental role that extends beyond the cleavage stages and impacts subsequent embryogenesis. In zebrafish embryos, mitotic metasynchrony diminishes in parallel with the onset of zygotic genome activation (ZGA), a pivotal process that lays the foundation for cell fate heterogeneity and is central to metazoan development14. Given previously reported roles of cell volume (N/C ratio promotes ZGA) and cell cycle length (shorter cell cycle prevents ZGA) in regulating ZGA, we investigated whether the observed gradient in cell volume and mitotic metasynchrony influences the onset of zygotic transcription33,34,35,36,37,38. To address this, we tracked the timing and spatial pattern of zygotic transcription initiation by imaging endogenous pri-miR430 transcripts using the morpholino visualization of expression39 (Fig. miR430 genes are among the earliest and most abundantly transcribed genes during zebrafish ZGA, making them ideal markers for assessing transcriptional onset40,41. a, Representative maximum intensity projection image showing the AP view of a control Tg(actb2:rfp-pcna) embryo injected with MO1-3-fluorescein in division round 9. The three insets, each 120 µm × 120 µm in size, demarcate the regions (1, AP; 2, intermediate region; 3, margin) whose higher-magnification images are shown in b and e. b, Zoomed-in views of the insets shown in a. Cyan dots, marked with white arrowheads, represent nascent mi430 transcripts detected by MO1-3-fluorescein. MO1-3 foci without an overlapping nuclear signal indicates that the cell is in mitosis (n = 4 embryos). c, Lateral view of a control embryo in division round 10 injected with MO1-3-fluorescein. d, Schematic (top) and a representative orthogonal maximum intensity projection image (bottom) of a bilobed Tg(actb2:rfp-pcna) embryo injected with the MO1-3-fluorescein embryo in division round 9. e, Zoomed-in views of the equivalent regions (1, AP; 2, intermediate region; 3, margin) demarcated in a for a bilobed embryo in division round 9. Cyan dots represent nascent mi430 transcripts. MO1-3 foci without an overlapping nuclear signal indicates that the cell is in mitosis (n = 3 embryos). f, AP view of control and bilobed Tg(sebox:egfp) embryos, with comparable regions from a indicated in the bilobed embryo. The arrowhead indicates the presence of ectopic sebox:egfp expression at the original AP, where a valley is formed between the two domes (ectopic APs) of the bilobed embryo. g, Schematic representing the geometry-driven asymmetric cell division patterning in the early embryo. Patterned asymmetric divisions generate a gradient of cell volume, producing mitotic phase waves and spatially patterning ZGA onset along the AP–margin axis. In most embryos, we observed the first signs of miR430 transcription during division round 9. In particular, during rounds 9 and 10, transcription was initiated in a graded manner: miR430 transcription foci first appeared—and/or persisted longer—in cells near the blastoderm margin. They then emerged in an intermediate zone between the margin and the AP, and finally in cells located at the AP (Fig. These findings suggest that zygotic transcription begins as a spatiotemporal gradient along the AP–margin axis, with marginal cells exhibiting higher transcriptional activity or competence than AP cells over the course of one to two division rounds. To determine whether this pattern of ZGA onset is driven by the underlying gradients in cell volume and/or cell cycle length, we examined bilobed embryos in which both of these gradients are perturbed (Figs. In contrast to control embryos, cells at the AP in bilobed embryos are smaller and cycle more slowly than those in the intermediate zone (Fig. Remarkably, in these embryos, miR430 transcription was initiated first at both the margin and the AP—where cells are the smallest—before appearing in the intermediate region (Fig. These results suggest that the gradient of cell volume and/or cell cycle length along the AP–margin axis influences the spatiotemporal onset of zygotic transcription. Thus, early embryo geometry can be linked directly to the initiation of zygotic gene expression, providing a mechanistic bridge between physical form and transcriptional timing in early development. To determine whether the geometry-derived transcriptional gradient within the embryo plays a role in embryonic development, we asked whether altering the spatiotemporal pattern of ZGA onset in bilobed embryos would affect the specification of the first cell fates within the blastoderm. To investigate this, we used Tg(sebox:egfp) embryos to visualize the earliest mesendoderm progenitors, specified at the onset of gastrulation42,43. In control embryos, mesendoderm progenitors were exclusively specified at the blastoderm margin—the region in which ZGA is first initiated. By contrast, approximately 30% of bilobed embryos exhibited mesendoderm progenitors not only at the margin but also ectopically within the ‘valley' between the two domes, reflecting the altered ZGA pattern in these embryos (Fig. These findings suggest that zygote geometry triggers a cascade of developmental events necessary for correct cell fate specification—an essential aspect of ensuring developmental robustness (Fig. Previous studies have shown that the shape of the zebrafish egg—particularly the blastodisc, the region from which all embryonic cells and tissues originate—is established by ooplasmic streaming, which segregates yolk granules from the cytoplasm within the fertilized egg30,44. Building on this, our findings suggest that the initial shape of the blastodisc drives asymmetric cell divisions during successive reductive cleavages by guiding mitotic spindle orientation, based on the simple premise that the spindle consistently aligns with the cell's longest axis, as described by Hertwig's rule27,28,45. These asymmetric divisions progressively generate a gradient of cell volumes along the AP–margin axis of the developing blastoderm, which, in turn, influences both cell cycle progression and ZGA along the same axis. Although we do not rule out the potential contribution of more complex mechanisms linking blastodisc shape to asymmetric cell divisions—such as differential interactions between spindle microtubules and yolk granules, the cell cortex or the plasma membrane—our results suggest that such factors are not required to explain the emergence of asymmetric divisions. Thus, the formation of a gradient in cell volume, as well as the consequent regulation of cell cycle dynamics and ZGA patterns, appears to be a direct outcome of the blastodisc's geometry. Interestingly, our data connect the observed gradients in cell volume and/or cell cycle length to the onset of ZGA. This aligns with recent findings in Xenopus embryos, where ZGA is first initiated in the smaller cells at the AP38. Multiple molecular mechanisms have been proposed to explain the link between cell cycle length and the onset of ZGA, including the titration of cytoplasmic transcriptional repressors as the N/C ratio increases, and/or progressively lengthening cell cycles that permit sustained zygotic transcription. Our observation that smaller, slower-cycling cells at the margin are transcriptionally more active, initiating transcription first and retaining the transcription foci longer (Extended Data Fig. 7), suggests that both mechanisms may contribute to the regulation of zygotic gene activation. We have previously shown that radially patterned cell behaviours and asymmetric tissue compaction along the AP–margin axis of the blastoderm emerge at the onset of gastrulation, providing essential conditions for proper morphogenetic movements during this stage of development20,46. It is plausible that these differences in tissue properties originate from distinct transcriptional profiles and cell fate specification, which are themselves driven by the differential patterns of ZGA, and ultimately embryo geometry, as described in this study. The dependence of cell fate marker expression on early embryo geometry raises a central question: how do noisy gradients in cell size, cell cycle and transcriptional activity yield robust fate specification? Although individual fate markers may be regulated by distinct mechanisms, their spatial patterns probably emerge from the integration of cellular dynamics with physical constraints. For example, although Nodal signals, inducing mesendodermal cell fates within the blastoderm margin, originate in the yolk, proper Nodal reception in blastoderm cells requires zygotically expressed factors such as the co-receptor oep47. Margin cells, which undergo earlier ZGA, may, therefore, respond more strongly to Nodal due to an earlier and/or higher co-receptor expression, sharpening mesendodermal marker expression near the margin. In bilobed embryos, altered cell size, cell cycle and transcription patterns could shift this sensitivity—elevated transcription in the valley between the two lobes might allow cells to respond to lower Nodal levels and, consequently, adopt a mesendodermal cell fate. Thus, biochemical and molecular prepatterning, achieved by cellular properties such as cell cycle, size and transcriptional activity, may provide the foundation for robust cell fate specification. Our findings, thus, establish a probable mechanistic link between two critical developmental milestones: the determination of blastodisc geometry immediately after fertilization and the onset of asymmetric tissue fluidization during gastrulation. This connection is further supported by our earlier observation that asymmetric tissue fluidization depends on proper mesendoderm specification20,46, as well as by our current finding that embryos with perturbed geometry exhibit mesendodermal fate misspecification that parallels the altered pattern of ZGA onset. This suggests that inherent geometric asymmetries in blastodisc shape can have long-lasting developmental consequences, manifesting only later as their effects accumulate during the early proliferative phase of embryogenesis. Consistent with this notion, several other organisms—including Caenorhabditis elegans, sea urchins and Xenopus—exhibit alignment between axes of cell cycle length and/or cell volume and the axes along which germ layer fates diverge, underscoring the developmental significance of geometry-derived patterns as robust and reproducible determinants of embryonic organization. Nonetheless, the specific outcomes of this fundamental relationship are likely to vary across species and must be examined within the unique developmental context of each organism. Tg(actb2:rfp-pcna) embryos were dechorionated and, at the desired stage, mounted in 0.5% agarose gels prepared in E3 buffer with the AP facing upwards in casts made with 2% agarose solution in E3 buffer13. RFP fluorescence was imaged at 23.5 °C using a 20× water-dipping objective on ZEISS LSM800 (numerical aperture of 0.8), LSM880 (numerical aperture of 1.0) or LSM900 (numerical aperture of 1.0) upright confocal microscopes. Then, 200–350-µm-thick Z stacks were acquired for different experiments with temporal resolutions ranging between 17 and 65 s per Z stack. The presence or absence of nuclear RFP-PCNA fluorescence was used to determine if a cell was in the S or M phase of the cell cycle, respectively48,49,50. The nuclear signal was segmented in Fiji (v. 2.16.0/1.54p; Java 1.8.0_172) using the Labkit plug-in (v. 0.4.0)51,52. The segmented objects were imported into Bitplane Imaris (v. 9.9) (https://imaris.oxinst.com/) to generate tracks and obtain tracking-related data, including track start time (S-phase entry), track end time (M-phase entry), track duration (S-phase length) and the nuclear coordinates at the last S-phase time point (position). These data were then analysed using the Pandas package (v. 1.5.3) in Python (v. 3.12.9) to calculate, for instance, the division timing variance in a given division round, followed by plotting using mainly the Matplotlib (v. 3.8.2) and seaborn (v. 0.12.2) packages53,54,55. The mitotic wave speed for a given division round was determined by first identifying the position of the wave origin and the timing of the wave origination, measuring all other nuclear distances and mitotic entry timings relative to this position and time, and then generating the best-fitting distance–time linear curve. In the cases where more than one nucleus could be considered the origin of the mitotic wave due to a simultaneous entry into mitosis, the mean of the 3D coordinates of such nuclei was considered as the position of wave origin. Furthermore, it should be noted that, especially during division rounds 2 and 3, the wave traverses the embryo nearly instantaneously, and in most of our experiments, we could not temporally resolve these early cell divisions, giving us infinite mitotic wave speeds. However, for analytical reasons, we only considered those embryos in which the divisions could be resolved, disregarding embryos exhibiting infinite speeds, which would provide an underestimate of the mitotic wave speed. To visualize cell cycle progression using Histone 1 as the marker, 0.5 ng of Histone 1-Alexa Fluor 488 (catalogue number H13188, Invitrogen) was injected at the 1-cell stage into wild-type AB or Tg(actb2:lyn-tdTomato) embryos56. Mosaic perturbation of the cycling period of a subset of the cells was carried out by mounting 32/64-cell stage embryos in 2% agarose solution in E3 buffer and injecting 0.2–0.3 ng of Histone 1-Alexa Fluor 488 into such cells. chek1 overexpression was achieved by injecting 12 pg chek1 mRNA into the one-cell embryo20,46. mRNA was prepared as described previously21. To test the cellular connectivity at the 16- and 64-cell stages, 0.0625 ng of fluorescein isothiocyanate–carboxymethyl–dextran (40 kDa) was injected into a single cell at the desired stage, and the embryos were mounted and imaged as described above. Aurkb function was inhibited by transiently incubating dechorionated <5 min post-fertilization (mpf) embryos in 50 µM of AZD1152 (catalogue number SML0268, Sigma-Aldrich) solution for 15 min, after which they were returned to the E3 buffer. Embryos showing successful nuclear cycling despite a failure of cytokinesis were then imaged as described above. Inhibition of the Aurkb function concomitantly with the chek1 overexpression or Histone 1-Alexa Fluor 488 visualization was performed by injecting 12 pg of chek1 mRNA or 0.5 ng of Histone 1-Alexa Fluor 488, respectively, at the 1-cell stage, as described above, followed by incubation in the Aurkb inhibitor. Chek1 activity inhibition was performed by dechorionating <10-mpf embryos and incubating them in 10 µM of PF477736 (catalogue number HY-10032, MedChem Express) solution in E3 buffer throughout the duration of the analysis. Nuclear RFP-PCNA fluorescence was segmented from Z stacks by thresholding using Fiji (v. 2.16.0/1.54p; Java 1.8.0_172), and their volumes were measured using the 3D Objects Counter plug-in (v. 2.0.1)57. Cell volumes at the 8- and 16-cell stages were measured by semiautomatically generating cell segmentation using the ‘Contour' function for surface creation in Bitplane Imaris (v. 9.9) from Z stacks of Tg(actb2:mCherry-CAAX) embryos58. Once segmented, the physical attributes of individual cells, such as volume and position, were exported from Bitplane Imaris (v. 9.9) as CSV files. Cell volumes at the 128-cell stage were measured by generating cell segmentations using the LimeSeg plug-in (v. 0.4.2) in Fiji (v. 2.16.0/1.54p; Java 1.8.0_172) and exporting the cell volume data for individual segmented objects59. All downstream data analysis was performed in Python (v. 3.12.9), mainly using the Pandas package (v. 1.5.3), and the plots were produced using the Matplotlib (v. 3.8.2) and seaborn (v. 0.12.2) packages. To directly measure the N/C for cells at the 8- and the 16-cell stages, Tg(actb2:mCherry-CAAX) embryos injected with 0.5 ng of Histone 1-Alexa Fluor 488 at the 1-cell stage were imaged as described above. Cell volumes were estimated by semiautomatically creating cell and nuclear surfaces in Imaris (v. 9.9), whose volumes were then used to determine the N/C ratio for each cell. Cell surfaces generated in Bitplane Imaris (v. 9.9) at the 8-cell stage were exported as WRL files, imported into MeshLab (https://www.meshlab.net/), from where individual cell surfaces were exported as PLY files. These files were subsequently processed in Python (v. 3.12.9) to identify the longest cell axis by performing principal component analysis, along which the cell was bisected into two daughter cells representing cells at the 16-cell stage29. Nuclei positions were obtained from Bitplane Imaris (v. 9.9), as detailed above. For each nucleus, the three nearest neighbours were identified, and their mean distance from the reference nucleus was calculated using the Pandas package (v. 1.5.3) in Python (v. 3.12.9). At approximately 30 mpf, Tg(actb2:rfp-pcna) embryos were mounted in E3 buffer on a substrate prepared with 2% agarose solution in E3 buffer. A 20-µm blunt-end needle attached to a 10-µl Hamilton syringe was inserted through the yolk into the blastodisc, and the cytoplasm was carefully aspirated. The nuclear RFP-PCNA signal was constantly observed using a Leica stereofluorescence microscope to ensure the nucleus was not extracted together with the cytoplasm. The setup and mounting procedure for nucleus aspiration were the same as for cytoplasm aspiration described above, except that, in this case, the nucleus was aspirated rather than the cytoplasm and from embryos at the 2-cell stage rather than the 1-cell stage. Care was exercised to ensure that the nucleus was extracted with a negligible volume of the cytoplasm being aspirated in the process. Bilobed embryos were created by spatially confining dechorionated embryos at the 2-cell stage, still undergoing cytokinesis from division round 1, in 0.5%–0.6% agarose in E3 buffer. Microscopy was performed as described above. To sever the yolk, dechorionated embryos at the 2-cell stage were placed on a substrate of 3% methylcellulose in E3 buffer in a glass dish. Using an eyelash attached to the end of a Pasteur pipette, the vegetal half of the yolk was first punctured to prevent the embryo from exploding due to a build-up of internal pressure during the severing procedure, and then carefully sliced. The embryo was allowed to recover for at least 15 min after the procedure, followed by mounting and microscopy, as detailed above. Four orthogonal sections, each showing four nuclei along the plane of division round 4 at the 16-cell stage, were generated using the reslice function in Fiji from Z stacks of individual Tg(actb2:rfp-pcna) embryos. The angles between the nuclei were then measured using Fiji (v. 2.16.0/1.54p; Java 1.8.0_172), as represented in Fig. 5d, to obtain the angle of cell divisions in division round 4. pri-miR430 transcripts were labelled as described previously39. Tg(actb2:rfp-pcna) embryos injected with MO1-3-fluorescein (Gene Tools) at the 1-cell stage were mounted in 0.5% agarose solution in E3 buffer in #3 fluorinated ethylene propylene tubes, and the Z stacks were generated with a temporal resolution of approximately 1 min per Z stack using a 20×/1.33-numerical-aperture objective on a ZEISS Lightsheet 7 microscope. Maximum intensity projections of these time-lapse recordings were then used to assess the transcription status of the individual nuclei. All animal breeding and procedures were performed in accordance with the European Union animal welfare guidelines and involve only low-severity lines, in accordance with the authorized animal breeding licences (66018/8-II/3b/2013 and 2023-0.288.351) in the Institute of Science and Technology Austria Aquatics Facility (approval number 024-0.730.856). All experiments were performed before 5 days post-fertilization, a period during which zebrafish do not feed independently and in line with the principle of 3Rs. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. All other data that are necessary to interpret, verify and extend the research within this article are available from the corresponding author upon reasonable request. The code relevant to this article is available via GitHub at https://github.com/Irene-Li/MitoticWaves. Cell monolayers sense curvature by exploiting active mechanics and nuclear mechanoadaptation. Cheikh, M. I. et al. A comprehensive model of Drosophila epithelium reveals the role of embryo geometry and cell topology in mechanical responses. Lou, Y., Rupprecht, J.-F., Theis, S., Hiraiwa, T. & Saunders, T. E. Curvature-induced cell rearrangements in biological tissues. & Gabriele, S. How multiscale curvature couples forces to cellular functions. Gehrels, E. W., Chakrabortty, B., Perrin, M.-E., Merkel, M. & Lecuit, T. Curvature gradient drives polarized tissue flow in the Drosophila embryo. Lefebvre, M. F., Claussen, N. H., Mitchell, N. P., Gustafson, H. J. & Streichan, S. J. Geometric control of myosin II orientation during axis elongation. Deneke, V. E., Melbinger, A., Vergassola, M. & Di Talia, S. Waves of Cdk1 activity in S phase synchronize the cell cycle in Drosophila embryos. Vergassola, M., Deneke, V. E. & Di Talia, S. Mitotic waves in the early embryogenesis of Drosophila: bistability traded for speed. & Ferrell, J. E. Mitotic trigger waves and the spatial coordination of the Xenopus cell cycle. Anderson, G. A., Gelens, L., Baker, J. C. & Ferrell, J. E. Desynchronizing embryonic cell division waves reveals the robustness of Xenopus laevis development. & Yarden, A. Transcription-dependent induction of G1 phase during the zebra fish midblastula transition. Boothe, T. et al. A tunable refractive index matching medium for live imaging cells, tissues and model organisms. & Stelzer, E. H. K. Reconstruction of zebrafish early embryonic development by scanned light sheet microscopy. Cell lineage reconstruction of early zebrafish embryos using label-free nonlinear microscopy. Berloco, M., Fanti, L., Breiling, A., Orlando, V. & Pimpinelli, S. The maternal effect gene, abnormal oocyte (abo), of Drosophila melanogaster encodes a specific negative regulator of histones. A Rad53 kinase-dependent surveillance mechanism that regulates histone protein levels in S. cerevisiae. Differential effect of H1 variant overexpression on cell cycle progression and gene expression. Petridou, N. I., Corominas-Murtra, B., Heisenberg, C.-P. & Hannezo, E. Rigidity percolation uncovers a structural basis for embryonic tissue phase transitions. Chemical Oscillations, Waves, and Turbulence Vol. & Law, R. D. Cell lineage of zebrafish blastomeres: I. Cleavage pattern and cytoplasmic bridges between cells. Nuclei determine the spatial origin of mitotic waves. Murata-Hori, M., Tatsuka, M. & Wang, Y.-L. Probing the dynamics and functions of aurora B kinase in living cells during mitosis and cytokinesis. Conn, C. W., Lewellyn, A. L. & Maller, J. L. The DNA damage checkpoint in embryonic cell cycles is dependent on the DNA-to-cytoplasmic ratio. The nuclear-cytoplasmic ratio controls the cell-cycle period in compartmentalized frog egg extract. Minc, N., Burgess, D. & Chang, F. Influence of cell geometry on division-plane positioning. Hertwig, O. Das Problem der Befruchtung und der Isotropie des Eies: eine Theorie der Vererbung (Verlag von Gustav Fischer, 1884). Wold, S., Esbensen, K. & Geladi, P. Principal component analysis. Bulk actin dynamics drive phase segregation in zebrafish oocytes. Deneke, V. E. et al. Self-organized nuclear positioning synchronizes the cell cycle in Drosophila embryos. The role of the DNA damage response in zebrafish and cellular models of Diamond Blackfan anemia. & Kirschner, M. A major developmental transition in early Xenopus embryos: I. Characterization and timing of cellular changes at the midblastula stage. & Kirschner, M. A major developmental transition in early Xenopus embryos: II. Langley, A. R., Smith, J. C., Stemple, D. L. & Harvey, S. A. New insights into the maternal to zygotic transition. & Schubiger, G. Parameters controlling transcriptional activation during early Drosophila development. Kimelman, D., Kirschner, M. & Scherson, T. The events of the midblastula transition in Xenopus are regulated by changes in the cell cycle. Chen, H., Einstein, L. C., Little, S. C. & Good, M. C. Spatiotemporal patterning of zygotic genome activation in a model vertebrate embryo. Hadzhiev, Y. et al. A cell cycle-coordinated polymerase II transcription compartment encompasses gene expression before global genome activation. Giraldez, A. J. et al. Zebrafish MiR-430 promotes deadenylation and clearance of maternal mRNAs. The earliest transcribed zygotic genes are short, newly evolved, and different across species. Poulain, M. & Lepage, T. Mezzo, a paired-like homeobox protein is an immediate target of nodal signalling and regulates endoderm specification in zebrafish. Ruprecht, V. et al. Cortical contractility triggers a stochastic switch to fast amoeboid cell motility. Roosen-Runge, E. C. On the early development—bipolar differentiation and cleavage—of the zebra fish, Brachydanio rerio. Middelkoop, T. C. et al. A cytokinetic ring-driven cell rotation achieves Hertwig's rule in early development. Petridou, N. I., Grigolon, S., Salbreux, G., Hannezo, E. & Heisenberg, C.-P. Fluidization-mediated tissue spreading by mitotic cell rounding and non-canonical Wnt signalling. The EGF-CFC protein one-eyed pinhead is essential for nodal signaling. Budke, H. et al. Assessment of cell proliferation in paraffin sections of normal bone marrow by the monoclonal antibodies Ki-67 and PCNA. Kelman, Z. PCNA: structure, functions and interactions. Proliferating cell nuclear antigen (PCNA) as a proliferative marker during embryonic and adult zebrafish hematopoiesis. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Arzt, M. et al. LABKIT: labeling and segmentation toolkit for big image data. McKinney, W. Data structures for statistical computing in Python. 9th Python in Science Conference 56–61 (SciPy, 2010). Hunter, J. D. Matplotlib: a 2D graphics environment. Waskom, M. L. seaborn: statistical data visualization. Stegmaier, J. et al. Real-time three-dimensional cell segmentation in large-scale microscopy data of developing embryos. Bolte, S. & Cordelières, F. P. A guided tour into subcellular colocalization analysis in light microscopy. Tavano, S. et al. BMP-dependent patterning of ectoderm tissue material properties modulates lateral mesendoderm cell migration during early zebrafish gastrulation. Machado, S., Mercier, V. & Chiaruttini, N. LimeSeg: a coarse-grained lipid membrane simulation for 3D image segmentation. We thank N. Petridou (EMBL) for sharing results before publication. N.M. was supported by funding from the European Union's Horizon 2020 programme under the Marie Skłodowska-Curie COFUND Actions ISTplus grant agreement number 754411. acknowledges funding from the European Union's Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement number 101034413. The research was supported by funding to C.-P.H. from the NOMIS Foundation, Project ID 1.844. We would like to thank past and present members of the Heisenberg and Hannezo groups for discussions, particularly S. Shamipour, V. Doddihal, M. Jovic, N. Hino, F. N. Arslan, R. Kobylinska and C. Camelo for feedback on the draft manuscript. This research was supported by the Scientific Service Units (SSU) of Institute of Science and Technology Austria through resources provided by the Aquatics Facility, Imaging & Optics Facility (IOF), Scientific Computing (SciComp) facility and Lab Support Facility (LSF). Open access funding provided by Institute of Science and Technology (IST Austria). Nikhil Mishra, Yuting Irene Li, Edouard Hannezo & Carl-Philipp Heisenberg Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar (theory) acquired and analysed the data. prepared the paper with input and feedback from all authors. The authors declare no competing interests. Nature Physics thanks Stefano Di Talia and Nicolas Minc for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Scatter plot of division timing for individual cells starting at cleavage division round 3 (x-axis) relative to their position along the animal-margin axis (y-axis) for one representative wild-type embryo. The green dotted line outlines cells located at a distance of <10 μm from the bottom of the image stack that could not be reliably tracked. (b) Mean RFP-PCNA intensity, as a marker for mitosis onset, as a function of time. Mean PCNA intensity increases during the S-phase, peaks just before the S-M transition, and declines in the M-phase. Thus, a narrow peak indicates greater synchrony and vice versa. (c) Representative scheme showing the position of the mitotic wave origin in division round 8 (n = 7 embryos). (d) Representative plot of cleavage division timing as a function of cell distance from the mitotic wave origin in cleavage division round 8 from a wild-type embryo. Cells undergoing the S-M transition were binned together. Representative scatter plots of the division timing for individual surface cells relative to their position along the animal-margin axis in control embryos (a), and embryos injected with either 12 pg chek1 mRNA at the 1-cell stage (b) or 0.3 ng Histone1 protein at the 32-cell stage (c). Blue and red dotted lines demarcate the onset of the 4th and 8th cleavage divisions, respectively. Green dotted line outlines cells located at a distance of <10 μm from the bottom of the image stack that could not be reliably tracked. (a) Representative scatter plot of the division timings for individual cells relative to their position along the animal-margin axis in a syncytial Tg(actb2:rfp-pcna) embryo (n = 7 embryos). (b) Contour plot of the mitotic wave across all surface cells at the 8th round of cleavage division round in syncytialized Tg(actb2:rfp-pcna) embryos injected with 12 pg chek1 mRNA at the one-cell stage, and (c) a scatter plot of division timings relative to position along the AP-margin axis for a representative syncytial embryo injected with 12 pg chek1 mRNA. A schematic each for single-cell injections of 40 kDa FITC-CM-Dextran at the 16- (a) and the 64-cell stage (b) in Tg(actb2:mCherry-CAAX) embryos and representative images one division round later (n = 3 and 6 embryos, respectively). (a) Representative images showing segmented cells and nuclei from time-lapse images of Tg(actb2:mCherry-CAAX) embryos injected with Histone 1-Alexa Fluor 488 at the 8- and 16-cell stages. (b) A dot-box plot showing the quantifications for the N/C in individual cells positioned centrally or peripherally. n = 23 cells for each cleavage stage. Data obtained from 4 and 2 embryos for the 8- and 16-cell stage, respectively. (a, d) Representative lateral-view maximum intensity projections, (b, e) cell division timings as a function of the nearest neighbor distance (n = 140 and 133 cells, respectively), and (c, f) as a function of cell position along the AP-margin axis (n = 138 and 133 cells, respectively) in representative control (left) and bilobed (right) Tg(actb2:rfp-pcna) embryos during division round 10. Points outside the range represent outliers. Data representative of 10 control and 5 bilobed embryos. (a) Zoomed-in views of the insets (1, animal pole; 2, intermediate region; 3, margin) shown in Fig. 6A for a control Tg(actb2:rfp-pcna) embryo injected with MO1-3-Fluorescein in the 10th division round. Cyan, MO1-3-Fluorescein (nascent mi430 transcripts); magenta, nuclei (PCNA). Each inset represents an area of 120 µm X 120 µm in size. (b) A scatter plot showing cell division timing as a function of the nearest neighbor distance with dot color representing the number of pri-mRNA transcription foci in a cell. n = 72 cells, data representative of 4 embryos. (c) A dot-box-plot showing the ratio of S-phase length and nearest neighbor distance for cells with one or two transcription foci for a representative embryo during division round 10. Dots outside the range represent outliers. n = 72 cells, data representative of 4 embryos. Animal pole (AP)-view of the cell cycle progression beginning from division round 3 in a Tg(actb2:rfp-pcna) embryo. AP-view of the cell cycle progression beginning from division round 7 in a Tg(actb2:rfp-pcna) embryo with a mosaic injection of 0.3 ng Histone 1-Alexa Flour 488 at the 32-cell stage. AP-view of the cell cycle progression beginning from division round 4 in a Tg(actb2:rfp-pcna) embryo injected with 12 pg chek1 mRNA at the one-cell stage. AP-view of the nuclear cycling beginning from division round 4 in a Tg(actb2:rfp-pcna) embryo syncytialized by inhibiting Aurkb activity through 50 µM AZD1152-treatment at the one-cell stage. AP-view of the nuclear cycling beginning from division round 3 in a syncytialized Tg(actb2:rfp-pcna) embryo injected with 12 pg chek1 mRNA at the one-cell stage. AP-view of the cell cycle progression beginning from division round 4 in a Tg(actb2:rfp-pcna) embryo from which one nucleus at the two-cell stage was aspirated. AP-view of the cell cycle progression beginning from division round 4 in a bilobed Tg(actb2:rfp-pcna) embryo. AP-view of the cell cycle progression beginning from division round 5 in a Tg(actb2:rfp-pcna) embryo with the yolk-blastoderm curvature increased through a partial severing of the yolk at the two-cell stage. AP-view of the cell cycle progression beginning from division round 3 in a Tg(actb2:rfp-pcna) embryo treated with 10 µM PF477736 to inhibit Chek1 activity. AP-view of the cell cycle progression and mirR430 pri-mRNA beginning from division round 8 in a control Tg(actb2:rfp-pcna) embryo injected with MO1-3-Fluorescein at the one-cell stage. Lateral view of the cell cycle progression and mirR430 pri-mRNA beginning from division round 3 in a control Tg(actb2:rfp-pcna) embryo injected with MO1-3-Fluorescein at the one-cell stage. AP-view of the cell cycle progression and mirR430 pri-mRNA beginning from division round 8 in a bilobed Tg(actb2:rfp-pcna) embryo injected with MO1-3-Fluorescein at the one-cell stage. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Mishra, N., Li, Y.I., Hannezo, E. et al. Geometry-driven asymmetric cell divisions pattern cell cycles and zygotic genome activation in the zebrafish embryo. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260104202738.htm'>A legendary fossil is forcing scientists to rethink human origins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 07:11:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An international research team led by scientists from La Trobe University in Australia and the University of Cambridge is questioning how one of the most complete early human fossils has been classified. Their findings suggest the specimen may not belong to any known human ancestor species and could represent an entirely new one. The fossil, uncovered in South Africa's Sterkfontein Caves in 1998 and nicknamed "Little Foot," has long been considered part of the Australopithecus genus. Other researchers argued it belonged to Australopithecus africanus, a species first described in 1925 by Australian anatomist Raymond Dart and already known from the same region. "This fossil remains one of the most important discoveries in the hominin record and its true identity is key to understanding our evolutionary past," Dr. Martin said. This is more likely a previously unidentified, human relative. Little Foot demonstrates in all likelihood he's right about that. Formally known as StW 573, Little Foot is still considered the most complete ancient hominin skeleton ever found. "Our findings challenge the current classification of Little Foot and highlight the need for further careful, evidence-based taxonomy in human evolution," Dr. Martin said. Dr. Martin, who holds an adjunct position at La Trobe University and is a postdoctoral research fellow at Cambridge, will continue this work alongside La Trobe students. Professor Herries emphasized the fossil's importance for understanding early human diversity and how ancient relatives adapted to the varied environments of southern Africa. "It is clearly different from the type specimen of Australopithecus prometheus, which was a name defined on the idea these early humans made fire, which we now know they didn't. Note: Content may be edited for style and length. The Brain's Strange Way of Computing Could Explain Consciousness A Bizarre Seven-Hour Gamma-Ray Explosion From Deep Space Has Left Astronomers Puzzled Scientists Track Human Fitness for Nearly 50 Years and Discover When Physical Aging Really Starts Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41573-025-01336-9'>Pharmacological targeting of the JAK–STAT pathway: new concepts and emerging indications</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-05 01:16:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Drug Discovery (2026)Cite this article Type I/II cytokine receptors mediate cytokine-specific biological responses by employing a defined combination of four Janus kinases (JAKs) and seven signal transducers and activators of transcription (STATs) for cellular signal transduction. Deregulation of the JAK–STAT pathway leads to various diseases, with JAK and STAT proteins representing attractive therapeutic targets. Fifteen JAK inhibitors are approved for several immunological and haematological diseases, offering significant benefits for patients. However, safety restrictions have limited their clinical use. Mechanistic and structural insights are driving current drug development approaches focused on improving their potency, selectivity and safety. Development of STAT inhibitors has been more challenging, and none has yet received clinical approval, although promising new compounds are now entering clinical trials. This Review discusses the recent advances in JAK and STAT inhibitor development and presents emerging therapeutic indications for JAK–STAT inhibition. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Philips, R. L. et al. JAK–STAT pathway at 30: much learned, much more to do. Google Scholar Caveney, N. A. et al. Structural basis of Janus kinase trans-activation. Google Scholar Abraham, B. G. et al. Molecular basis of JAK2 activation in erythropoietin receptor and pathogenic JAK2 signaling. Google Scholar Leonard, W. J. Strategies to therapeutically modulate cytokine action. Google Scholar Hammarén, H. M., Virtanen, A. T., Raivola, J. & Silvennoinen, O. The regulation of JAKs in cytokine signaling and its breakdown in disease. Google Scholar Jain, M. et al. Role of JAK/STAT in the neuroinflammation and its association with neurological disorders. Google Scholar Sabaawy, H. E., Ryan, B. M., Khiabanian, H. & Pine, S. R. JAK/STAT of all trades: linking inflammation with cancer development, tumor progression and therapy resistance. Google Scholar Russell, M. A., Richardson, S. J. & Morgan, N. G. The role of the interferon/JAK–STAT axis in driving islet HLA-I hyperexpression in type 1 diabetes. Google Scholar Eggel, A., Pennington, L. F. & Jardetzky, T. S. Therapeutic monoclonal antibodies in allergy: targeting IgE, cytokine, and alarmin pathways. Google Scholar Saxton, R. A., Glassman, C. R. & Garcia, K. C. Emerging principles of cytokine pharmacology and therapeutics. Google Scholar Tokareva, K. et al. JAK inhibitors and black box warnings: what is the future for JAK inhibitors? Google Scholar Ferrao, R. & Lupardus, P. J. The Janus kinase (JAK) FERM and SH2 domains: bringing specificity to JAK–receptor interactions. Ferrao, R. D., Wallweber, H. J. & Lupardus, P. J. Receptor-mediated dimerization of JAK2 FERM domains is required for JAK2 activation. Google Scholar Wallweber, H. J. A., Tam, C., Franke, Y., Starovasnik, M. A. & Lupardus, P. J. Structural basis of IFNα receptor recognition by TYK2. Google Scholar McNally, R., Toms, A. V. & Eck, M. J. Crystal structure of the FERM–SH2 module of human Jak2. PLoS One 11, e0156218 (2016). Google Scholar Zhang, D., Wlodawer, A. & Lubkowski, J. Crystal structure of a complex of the intracellular domain of interferon λ receptor 1 (IFNLR1) and the FERM/SH2 domains of human JAK1. Google Scholar Glassman, C. R. et al. Structure of a Janus kinase cytokine receptor complex reveals the basis for dimeric activation. Google Scholar Philips, R. L. et al. The JAK–STAT pathway at 30: much learned, much more to do. Google Scholar Rengachari, S. et al. Structural basis of STAT2 recognition by IRF9 reveals molecular insights into ISGF3 function. Natl Acad. Google Scholar Gingras, S., Simard, J., Groner, B. & Pfitzner, E. p300/CBP is required for transcriptional induction by interleukin-4 and interacts with Stat6. Nucleic Acids Res. Google Scholar Wojciak, J. M., Martinez-Yamout, M. A., Dyson, H. J. & Wright, P. E. Structural basis for recruitment of CBP/p300 coactivators by STAT1 and STAT2 transactivation domains. EMBO J. Google Scholar Bousoik, E. & Montazeri Aliabadi, H. “Do we know jack” about JAK? A closer look at JAK/STAT signaling pathway. Google Scholar Baxter, E. J. et al. Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders. Google Scholar Kralovics, R. et al. A gain-of-function mutation of JAK2 in myeloproliferative disorders. Google Scholar James, C. et al. A unique clonal JAK2 mutation leading to constitutive signalling causes polycythaemia vera. Google Scholar Horesh, M. E. et al. Individuals with JAK1 variants are affected by syndromic features encompassing autoimmunity, atopy, colitis, and dermatitis. Google Scholar Xiang, Z. et al. Identification of somatic JAK1 mutations in patients with acute myeloid leukemia. Google Scholar Macchi, P. et al. Mutations of Jak-3 gene in patients with autosomal severe combined immune deficiency (SCID). Google Scholar Walters, D. K. et al. Activating alleles of JAK3 in acute megakaryoblastic leukemia. Cancer Cell 10, 65–75 (2006). Google Scholar Pellenz, F. M. et al. Association of TYK2 polymorphisms with autoimmune diseases: a comprehensive and updated systematic review with meta-analysis. Google Scholar Boggon, T. J., Li, Y., Manley, P. W. & Eck, M. J. Crystal structure of the Jak3 kinase domain in complex with a staurosporine analog. Google Scholar Lucet, I. S. et al. The structural basis of Janus kinase 2 inhibition by a potent and specific pan-Janus kinase inhibitor. Google Scholar Miao, Y. et al. Functional and structural characterization of clinical-stage Janus kinase 2 inhibitors identifies determinants for drug selectivity. Google Scholar Virtanen, A. et al. Differences in JAK isoform selectivity among different types of JAK inhibitors evaluated for rheumatic diseases through in vitro profiling. Arthritis Rheumatol. Google Scholar Virtanen, A. et al. JAK inhibitor selectivity: new opportunities, better drugs? Google Scholar et al. Discovery of a JAK3-selective inhibitor: functional differentiation of Jak3-selective inhibition over pan-JAK or JAK1-selective inhibition. ACS Chem. Google Scholar Li, S. et al. Discovery of hexahydrofuro[3,2-b]furans as new kinase-selective and orally bioavailable JAK3 inhibitors for the treatment of leukemia harboring a JAK3 activating mutant. Google Scholar Hammarén, H. M. et al. Janus kinase 2 activation mechanisms revealed by analysis of suppressing mutations. Allergy Clin. Google Scholar Raivola, J., Haikarainen, T. & Silvennoinen, O. Characterization of JAK1 pseudokinase domain in cytokine signaling. Google Scholar Leroy, E. et al. Differential effect of inhibitory strategies of the V617 mutant of JAK2 on cytokine receptor signaling. Allergy Clin. Google Scholar Silvennoinen, O. & Ungureanu, D. Dual activity kinase domains and uses thereof. US Patent 8,841,078 (23 September 2014). Hammarén, H. M. et al. ATP binding to the pseudokinase domain of JAK2 is critical for pathogenic activation. Natl Acad. Google Scholar Toms, A. V. et al. Structure of a pseudokinase-domain switch that controls oncogenic activation of Jak kinases. Google Scholar Bandaranayake, R. M. et al. Crystal structures of the JAK2 pseudokinase domain and the pathogenic mutant V617F. Google Scholar Min, X. et al. Structural and functional characterization of the JH2 pseudokinase domain of JAK family tyrosine kinase 2 (TYK2). Google Scholar McNally, R. et al. Discovery and structural characterization of ATP-site ligands for the wild-type and V617F mutant JAK2 pseudokinase domain. ACS Chem. Google Scholar Virtanen, A. T. et al. Identification of novel small molecule ligands for JAK2 pseudokinase domain. Google Scholar Puleo, D. E. et al. Identification and characterization of JAK2 pseudokinase domain small molecule binders. ACS Med. Google Scholar Liosi, M.-E. et al. Insights on JAK2 modulation by potent, selective, and cell-permeable pseudokinase-domain ligands. Google Scholar Henry, S. P. et al. Covalent modification of the JH2 domain of Janus kinase 2. ACS Med. Google Scholar Henry, S. P. et al. Conversion of a false virtual screen hit into selective JAK2 JH2 domain binders using convergent design strategies. ACS Med. Google Scholar Liosi, M.-E. et al. Selective Janus Kinase 2 (JAK2) pseudokinase ligands with a diaminotriazole core. Google Scholar Coltoff, A. & Mascarenhas, J. Fedratinib in 2025 and beyond: indications and future applications. Blood Adv. Google Scholar Keam, S. J. Momelotinib: first approval. Google Scholar Lamb, Y. N. Pacritinib: first approval. Google Scholar Masarova, L. et al. Ten years of experience with ruxolitinib since approval for polycythemia vera: a review of clinical efficacy and safety. Google Scholar Traynor, K. FDA approves tofacitinib for rheumatoid arthritis. Health Syst. Google Scholar Markham, A. Baricitinib: first global approval. Google Scholar Koppikar, P. et al. Heterodimeric JAK–STAT activation as a mechanism of persistence to JAK2 inhibitor therapy. Google Scholar Meyer, S. C. et al. CHZ868, a type II JAK2 inhibitor, reverses type I JAK inhibitor persistence and demonstrates efficacy in myeloproliferative neoplasms. Cancer Cell 28, 15–28 (2015). Google Scholar Palandri, F. et al. Ruxolitinib discontinuation syndrome: incidence, risk factors, and management in 251 patients with myelofibrosis. Blood Cancer J. Google Scholar Song, Y. et al. Phase I dose escalation and expansion study of golidocitinib, a highly selective JAK1 inhibitor, in relapsed or refractory peripheral T-cell lymphomas. Google Scholar Song, Y. et al. Golidocitinib, a selective JAK1 tyrosine-kinase inhibitor, in patients with refractory or relapsed peripheral T-cell lymphoma (JACKPOT8 Part B): a single-arm, multinational, phase 2 study. Lancet Oncol. Google Scholar Horwitz, S. M. et al. Results from an open-label phase 2a study of cerdulatinib, a dual spleen tyrosine kinase/janus kinase inhibitor, in relapsed/refractory peripheral T-cell lymphoma. Google Scholar Moskowitz, A. J. et al. A phase 2 biomarker-driven study of ruxolitinib demonstrates effectiveness of JAK/STAT targeting in T-cell lymphomas. Google Scholar Wu, H. et al. JAK1–STAT3 blockade by JAK inhibitor SHR0302 attenuates inflammatory responses of adjuvant-induced arthritis rats and decreases TH17 and total B cells. Bone Spine 83, 525–532 (2016). Google Scholar Kirby, J. S. et al. Efficacy and safety of the oral Janus kinase 1 inhibitor povorcitinib (INCB054707) in patients with hidradenitis suppurativa in a phase 2, randomized, double-blind, dose-ranging, placebo-controlled study. Google Scholar Huang, K. et al. The effect of food on the pharmacokinetics of WXFL10203614, a potential selective JAK1 inhibitor, in healthy Chinese subjects. Google Scholar et al. Safety and efficacy of the selective TYK2/JAK1 inhibitor TLL-018 in moderate-to-severe plaque psoriasis: a phase 1b, randomized, double-blind, placebo-controlled study. Andraos, R. et al. Modulation of activation-loop phosphorylation by JAK inhibitors is binding mode dependent. Cancer Discov. Google Scholar Wu, S.-C. et al. Activity of the type II JAK2 inhibitor CHZ868 in B cell acute lymphoblastic leukemia. Cancer Cell 28, 29–41 (2015). Google Scholar Arwood, M. L. et al. New scaffolds for type II JAK2 inhibitors overcome the acquired G993A resistance mutation. Cell Chem. Google Scholar Mascarenhas, J. O. et al. A multicenter, open-label, phase 1 clinical trial of AJ1-11095 administered as oral monotherapy in patients with primary myelofibrosis (PMF), Post-polycythemia vera myelofibrosis (PPV-MF), or post-essential thrombocythemia myelofibrosis (PET-MF) who have been failed by a type I JAK2 Inhibitor (JAK2i). Google Scholar Tokarski, J. S. et al. Tyrosine kinase 2-mediated signal transduction in T lymphocytes is blocked by pharmacological stabilization of its pseudokinase domain. Google Scholar Wrobleski, S. T. et al. Highly selective inhibition of tyrosine kinase 2 (TYK2) for the treatment of autoimmune diseases: discovery of the allosteric inhibitor BMS-986165. Google Scholar Burke, J. R. et al. Autoimmune pathways in mice and humans are blocked by pharmacological stabilization of the TYK2 pseudokinase domain. Google Scholar Moslin, R. et al. Identification of imidazo[1,2-b]pyridazine TYK2 pseudokinase ligands as potent and selective allosteric inhibitors of TYK2 signalling. Google Scholar Moslin, R. et al. Identification of N-methyl nicotinamide and N-methyl pyridazine-3-carboxamide pseudokinase domain ligands as highly selective allosteric inhibitors of tyrosine kinase 2 (TYK2). Google Scholar Armstrong, A. W. et al. Safety and efficacy of deucravacitinib in moderate to severe plaque psoriasis for up to 3 years: an open-label extension of randomized clinical trials. JAMA Dermatol. Google Scholar Leit, S. et al. Discovery of a potent and selective tyrosine kinase 2 inhibitor: TAK-279. Google Scholar Breinlinger, E. et al. Targeting the tyrosine kinase 2 (TYK2) pseudokinase domain: discovery of the selective TYK2 inhibitor ABBV-712. Google Scholar Stubbs, M. C. et al. Preclinical evaluation of INCB160058—a novel and potentially disease-modifying therapy for JAK2V617F mutant myeloproliferative neoplasms. Google Scholar Backus, K. M. et al. Proteome-wide covalent ligand discovery in native biological systems. Google Scholar Kavanagh, M. E. et al. Selective inhibitors of JAK1 targeting an isoform-restricted allosteric cysteine. Google Scholar Wylie, A. A. et al. The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1. Google Scholar Shah, R. R. et al. Hi-JAK-ing the ubiquitin system: the design and physicochemical optimisation of JAK PROTACs. Google Scholar Chang, Y. et al. Degradation of Janus kinases in CRLF2-rearranged acute lymphoblastic leukemia. Google Scholar Wu, J. et al. JAK1/JAK2 degraders based on PROTAC for topical treatment of atopic dermatitis. Google Scholar Zhang, X. et al. Discovery of a potent and selective JAK1-targeting PROTAC degrader with anti-tumor activities. Google Scholar Kato, J.-Y., Korenaga, S. & Iwakura, M. Discovery of a potent and subtype-selective TYK2 degrader based on an allosteric TYK2 inhibitor. Google Scholar Dunbar, A. J. et al. Jak2V617F reversible activation shows its essential requirement in myeloproliferative neoplasms. Cancer Discov. Google Scholar Ekman, F., Selvaraj, S., Gotlib, J., Cromer, M. K. & Porteus, M. H. Targeting the JAK2-V617F mutation in polycythemia vera using CRISPR/AAV6 genome editing. Google Scholar Tang, Q. et al. Rational design of a JAK1-selective siRNA inhibitor for the modulation of autoimmunity in the skin. Google Scholar Bousoik, E., Mahdipoor, P., Alhazza, A. & Montazeri Aliabadi, H. Combinational silencing of components involved in JAK/STAT signaling pathway. Google Scholar Clément, F. et al. Therapeutic siRNAs targeting the JAK/STAT signalling pathway in inflammatory bowel diseases. J. Crohns Colitis 16, 286–300 (2022). Google Scholar Wang, W., Lopez McDonald, M. C., Hariprasad, R., Hamilton, T. & Frank, D. A. Oncogenic STAT transcription factors as targets for cancer therapy: innovative strategies and clinical translation. Google Scholar He, X., Liu, P., Luo, Y., Fu, X. & Yang, T. STATs, promising targets for the treatment of autoimmune and inflammatory diseases. Google Scholar Xia, Y., Xie, Y., Zhang, H. & Liu, L. STAT4 gene polymorphisms in human diseases. Google Scholar Sharma, M. et al. Human germline heterozygous gain-of-function STAT6 variants cause severe allergic disease. Google Scholar Baris, S. et al. Severe allergic dysregulation due to a gain of function mutation in the transcription factor STAT6. Allergy Clin. Google Scholar Frank, D. A. Targeting STATs for cancer therapy. Google Scholar Turkson, J. et al. Phosphotyrosyl peptides block Stat3-mediated DNA binding activity, gene regulation, and cell transformation. Google Scholar Turkson, J. et al. Novel peptidomimetic inhibitors of signal transducer and activator of transcription 3 dimerization and biological activity. Cancer Ther. Google Scholar Mandal, P. K., Heard, P. A., Ren, Z., Chen, X. & McMurray, J. S. Solid-phase synthesis of Stat3 inhibitors incorporating O-carbamoylserine and O-carbamoylthreonine as glutamine mimics. Google Scholar Mandal, P. K. et al. Conformationally constrained peptidomimetic inhibitors of signal transducer and activator of transcription. 3: evaluation and molecular modeling. Google Scholar Mandal, P. K., Liao, W. S.-L. & McMurray, J. S. Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3. Google Scholar Auzenne, E. J. et al. A phosphopeptide mimetic prodrug targeting the SH2 domain of Stat3 inhibits tumor growth and angiogenesis. Google Scholar Mandal, P. K. et al. Potent and selective phosphopeptide mimetic prodrugs targeted to the Src homology 2 (SH2) domain of signal transducer and activator of transcription 3. Google Scholar Morlacchi, P., Mandal, P. K. & McMurray, J. S. Synthesis and in vitro evaluation of a peptidomimetic inhibitor targeting the Src homology 2 (SH2) domain of STAT6. ACS Med. Google Scholar Mandal, P. K. et al. Targeting the Src homology 2 (SH2) domain of signal transducer and activator of transcription 6 (STAT6) with cell-permeable, phosphatase-stable phosphopeptide mimics potently inhibits Tyr641 phosphorylation and transcriptional activity. Google Scholar Knight, J. M. et al. Small molecule targeting of the STAT5/6 Src homology 2 (SH2) domains to inhibit allergic airway disease. Google Scholar Chen, H. et al. Discovery of the highly selective and potent STAT3 inhibitor for pancreatic cancer treatment. ACS Cent. Google Scholar Chen, H. et al. Targeting STAT3 by a small molecule suppresses pancreatic cancer progression. Google Scholar Brambilla, L. et al. Hitting the right spot: mechanism of action of OPB-31121, a novel and potent inhibitor of the signal transducer and activator of transcription 3 (STAT3). Google Scholar Haftchenary, S. et al. Potent targeting of the STAT3 protein in brain cancer stem cells: a promising route for treating glioblastoma. ACS Med. Google Scholar Page, B. D. G. et al. Inhibiting aberrant signal transducer and activator of transcription protein activation with tetrapodal, small molecule Src homology 2 domain binders: promising agents against multiple myeloma. Google Scholar Zhang, X. et al. A novel inhibitor of STAT3 homodimerization selectively suppresses STAT3 activity and malignant transformation. Cancer Res. Google Scholar Zhang, X. et al. Orally bioavailable small-molecule inhibitor of transcription factor Stat3 regresses human breast and lung cancer xenografts. Natl Acad. Google Scholar Ren, X. et al. Identification of niclosamide as a new small-molecule inhibitor of the STAT3 signaling pathway. ACS Med. Google Scholar Matsuno, K. et al. Identification of a new series of STAT3 inhibitors by virtual screening. ACS Med. Google Scholar Zhang, X. et al. A novel small-molecule disrupts Stat3 SH2 domain–phosphotyrosine interactions and Stat3-dependent tumor processes. Google Scholar Wingelhofer, B. et al. Pharmacologic inhibition of STAT5 in acute myeloid leukemia. Google Scholar Elumalai, N. et al. Rational development of Stafib-2: a selective, nanomolar inhibitor of the transcription factor STAT5b. Google Scholar Elumalai, N., Berg, A., Natarajan, K., Scharow, A. & Berg, T. Nanomolar inhibitors of the transcription factor STAT5b with high selectivity over STAT5a. Google Scholar Cumaraswamy, A. A. et al. Nanomolar-potency small molecule inhibitor of STAT5 protein. ACS Med. Google Scholar Page, B. D. G. et al. Small molecule STAT5–SH2 domain inhibitors exhibit potent antileukemia activity. Google Scholar Müller, J., Sperl, B., Reindl, W., Kiessling, A. & Berg, T. Discovery of chromone-based inhibitors of the transcription factor STAT5. Google Scholar Martínez-López, J. et al. Biomarker-driven phase Ib clinical trial of OPB-111077 in acute myeloid leukemia. Google Scholar Yoo, C. et al. Phase I dose-finding study of OPB-111077, a novel STAT3 inhibitor, in patients with advanced hepatocellular carcinoma. Cancer Res. Google Scholar Ogura, M. et al. Phase I study of OPB-51602, an oral inhibitor of signal transducer and activator of transcription 3, in patients with relapsed/refractory hematological malignancies. Cancer Sci. Google Scholar et al. Phase I study of OPB-31121, an oral STAT3 Inhibitor, in patients with advanced solid tumors. Cancer Res. Google Scholar Redell, M. S., Ruiz, M. J., Alonzo, T. A., Gerbing, R. B. & Tweardy, D. J. Stat3 signaling in acute myeloid leukemia: ligand-dependent and -independent activation and induction of apoptosis by a novel small-molecule Stat3 inhibitor. Google Scholar Xu, X., Kasembeli, M. M., Jiang, X., Tweardy, B. J. & Tweardy, D. J. Chemical probes that competitively and selectively inhibit Stat3 activation. PLoS One 4, e4783 (2009). Google Scholar Tsimberidou, A. M. et al. Phase I trial of TTI-101, a first-in-class oral inhibitor of STAT3, in patients with advanced solid tumors. Cancer Res. Google Scholar Brovchenko, N. et al. Biaryl phosphates and phosphonates as selective inhibitors of the transcription factor STAT4. Google Scholar Metz, P. et al. Highly selective and reversible STAT6 inhibition demonstrates potential for differentiated efficacy and safety profile in type 2 allergic inflammation. Care Med. Google Scholar Yang, J. & Stark, G. R. Roles of unphosphorylated STATs in signaling. Cell Res. Google Scholar Yang, J. et al. Novel roles of unphosphorylated STAT3 in oncogenesis and transcriptional regulation. Cancer Res. Google Scholar Yang, J. et al. Unphosphorylated STAT3 accumulates in response to IL-6 and activates transcription by binding to NFκB. Google Scholar Leong, P. L. et al. Targeted inhibition of Stat3 with a decoy oligonucleotide abrogates head and neck cancer cell growth. Natl Acad. Google Scholar Zhang, Q. et al. Serum-resistant CpG-STAT3 decoy for targeting survival and immune checkpoint signaling in acute myeloid leukemia. Google Scholar Zhang, X. et al. Inhibitory effects of STAT3 decoy oligodeoxynucleotides on human epithelial ovarian cancer cell growth in vivo. Google Scholar Shen, J., Li, R. & Li, G. Inhibitory effects of decoy-ODN targeting activated STAT3 on human glioma growth in vivo. In Vivo 23, 237–243 (2009). Google Scholar Gu, J. et al. Blockage of the STAT3 signaling pathway with a decoy oligonucleotide suppresses growth of human malignant glioma cells. Google Scholar Zhang, X., Zhang, J., Wang, L., Wei, H. & Tian, Z. Therapeutic effects of STAT3 decoy oligodeoxynucleotide on human lung cancer in xenograft mice. BMC Cancer 7, 149 (2007). Google Scholar Sun, Z., Yao, Z., Liu, S., Tang, H. & Yan, X. An oligonucleotide decoy for Stat3 activates the immune response of macrophages to breast cancer. Google Scholar Xi, S., Gooding, W. E. & Grandis, J. R. In vivo antitumor efficacy of STAT3 blockade using a transcription factor decoy approach: implications for cancer therapy. Google Scholar Chan, K. S. et al. Disruption of Stat3 reveals a critical role in both the initiation and the promotion stages of epithelial carcinogenesis. Google Scholar Sen, M. et al. First-in-human trial of a STAT3 decoy oligonucleotide in head and neck tumors: implications for cancer therapy. Cancer Discov. Google Scholar Grandis, J. R. et al. Safety and efficacy of a STAT3-targeted cyclic oligonucleotide: From murine models to a phase 1 clinical trial in pet cats with oral cancer. Cancer Cell https://doi.org/10.1016/j.ccell.2025.07.015 (2025). Google Scholar Huang, W. et al. A small molecule compound targeting STAT3 DNA-binding domain inhibits cancer cell proliferation, migration, and invasion. ACS Chem. Google Scholar Huang, W. et al. Small-molecule inhibitors targeting the DNA-binding domain of STAT3 suppress tumor growth, metastasis and STAT3 target gene expression in vivo. Google Scholar Huang, W. et al. Small-molecule compounds targeting the STAT3 DNA-binding domain suppress survival of cisplatin-resistant human ovarian cancer cells by inducing apoptosis. Google Scholar Son, D. J. et al. MMPP attenuates non-small cell lung cancer growth by inhibiting the STAT3 DNA-binding activity via direct binding to the STAT3 DNA-binding domain. Google Scholar Groot, J. de et al. A first-in-human phase I trial of the oral p-STAT3 inhibitor WP1066 in patients with recurrent malignant glioma. CNS Oncol. Google Scholar Iwamaru, A. et al. A novel inhibitor of the STAT3 pathway induces apoptosis in malignant glioma cells both in vitro and in vivo. Google Scholar Froeling, F. E. M. et al. Bioactivation of napabucasin triggers reactive oxygen species-mediated cancer cell death. Cancer Res. Google Scholar Zhang, C. et al. Design, synthesis, and biological evaluation of novel napabucasin–melatonin hybrids as potent STAT3 inhibitors. Google Scholar Sabanés Zariquiey, F., da Souza, J. V., Estrada-Tejedor, R. & Bronowska, A. K. If you cannot win them, join them: understanding new ways to target STAT3 by small molecules. ACS Omega 4, 13913–13921 (2019). Google Scholar Li, C., Yang, A. & Rogoff, H. Method of Targeting Stat3 and Other Non-Druggable Proteins. Patent application PCT/US2016/045043 (9 February 2017). Shah, M. A. et al. Randomized, double-blind, placebo-controlled phase III study of paclitaxel ± napabucasin in pretreated advanced gastric or gastroesophageal junction adenocarcinoma. Cancer Res. Google Scholar Bekaii-Saab, T. et al. Napabucasin plus nab-paclitaxel with gemcitabine versus nab-paclitaxel with gemcitabine in previously untreated metastatic pancreatic adenocarcinoma: an adaptive multicentre, randomised, open-label, phase 3, superiority trial. Google Scholar Shah, M. A. et al. Napabucasin plus FOLFIRI in patients with previously treated metastatic colorectal cancer: results from the open-label, randomized phase III CanStem303C study. Colorectal Cancer 22, 100–110 (2023). Google Scholar Fan, C. et al. Natural product-inspired discovery of naphthoquinone–furo-piperidine derivatives as novel STAT3 inhibitors for the treatment of triple-negative breast cancer. Google Scholar Feng, K.-R. et al. Design, synthesis and biological evaluation of novel potent STAT3 inhibitors based on BBI608 for cancer therapy. Google Scholar Li, C. et al. A novel series of napabucasin derivatives as orally active inhibitors of signal transducer and activator of transcription 3 (STAT3). Google Scholar Zhou, Q. et al. Design, synthesis and activity of BBI608 derivatives targeting on stem cells. Google Scholar Chen, J. et al. Structure-based design of conformationally constrained, cell-permeable STAT3 inhibitors. ACS Med. Google Scholar Zhou, H. et al. Structure-based discovery of SD-36 as a potent, selective, and efficacious PROTAC degrader of STAT3 protein. Google Scholar Bai, L. et al. A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression in vivo. Cancer Cell 36, 498–511.e17 (2019). Google Scholar Zhou, H. et al. SD-91 as a potent and selective STAT3 degrader capable of achieving complete and long-lasting tumor regression. ACS Med. Google Scholar Jin, J. et al. Small-molecule PROTAC mediates targeted protein degradation to treat STAT3-dependent epithelial cancer. JCI Insight 7, e160606 (2022). Google Scholar Kaneshige, A. et al. A selective small-molecule STAT5 PROTAC degrader capable of achieving tumor regression in vivo. Google Scholar Kaneshige, A. et al. Discovery of a potent and selective STAT5 PROTAC degrader with strong antitumor activity in vivo in acute myeloid leukemia. Google Scholar Kaneshige, A. et al. Discovery of AK-1690: a potent and highly selective STAT6 PROTAC degrader. Google Scholar Li, S. et al. Decoy-PROTAC for specific degradation of ‘Undruggable' STAT3 transcription factor. Cell Death Dis. Google Scholar Shih, P.-C., Naganuma, M., Tsuji, G., Demizu, Y. & Naito, M. Development of decoy oligonucleotide-warheaded chimeric molecules targeting STAT3. Google Scholar Lu, J., Liu, J., Li, L., Lan, Y. & Liang, Y. Cytokines in type 1 diabetes: mechanisms of action and immunotherapeutic targets. Google Scholar Herold, K. C. et al. Teplizumab: a disease-modifying therapy for type 1 diabetes that preserves β-cell function. Diabetes Care 46, 1848–1856 (2023). Google Scholar Colli, M. L. et al. An integrated multi-omics approach identifies the landscape of interferon-α-mediated responses of human pancreatic β cells. Google Scholar Coomans de Brachène, A. et al. IFN-α induces a preferential long-lasting expression of MHC class I in human pancreatic β cells. Google Scholar Ge, T. et al. The JAK1 selective inhibitor ABT 317 Blocks signaling through interferon-γ and common γ chain cytokine receptors to reverse autoimmune diabetes in NOD mice. Google Scholar Trivedi, P. M. et al. Repurposed JAK1/JAK2 inhibitor reverses established autoimmune insulitis in NOD mice. Google Scholar Marroqui, L. et al. TYK2, a candidate gene for type 1 diabetes, modulates apoptosis and the innate immune response in human pancreatic β-cells. Google Scholar Gorman, J. A. et al. The TYK2-P1104A autoimmune protective variant limits coordinate signals required to generate specialized T cell subsets. Google Scholar et al. Fine-mapping and functional studies highlight potential causal variants for rheumatoid arthritis and type 1 diabetes. Google Scholar Chandra, V. et al. The type 1 diabetes gene TYK2 regulates β-cell development and its responses to interferon-α. Google Scholar Dos Santos, R. S., Guzman-Llorens, D., Perez-Serna, A. A., Nadal, A. & Marroqui, L. Deucravacitinib, a tyrosine kinase 2 pseudokinase inhibitor, protects human EndoC-βH1 β-cells against proinflammatory insults. Google Scholar Coomans de Brachène, A. et al. Preclinical evaluation of tyrosine kinase 2 inhibitors for human beta-cell protection in type 1 diabetes. Diabetes Obes. Google Scholar Waibel, M. et al. Baricitinib and β-cell function in patients with new-onset type 1 diabetes. Google Scholar Abadie, V. et al. IL-15, gluten and HLA-DQ8 drive tissue destruction in coeliac disease. Google Scholar Schuppan, D. et al. A randomized trial of a transglutaminase 2 inhibitor for celiac disease. Google Scholar Choung, R. S. et al. Ritlecitinib, a JAK3 /TEC inhibitor, modulates the markers of celiac autoimmunity in alopecia areata and vitiligo patients. Google Scholar Verdelho Machado, M. Refractory celiac disease: what the gastroenterologist should know. Google Scholar Nandi, N. et al. Normalization of duodenal mucosa after treatment with Janus kinase (JAK) inhibitor in refractory celiac disease type 2. Google Scholar Grewal, J. K., Kassardjian, A. & Weiss, G. A. Successful novel use of tofacitinib for type II refractory coeliac disease. BMJ Case Rep. 15, e244692 (2022). Google Scholar Dieckman, T. et al. Enduring clinical remission in refractory celiac disease type II with tofacitinib: an open-label clinical study. Google Scholar Diaz, K. et al. Peripheral inflammatory cytokines and motor symptoms in persons with Parkinson's disease. Brain Behav. Google Scholar Hong, H. et al. Suppression of the JAK/STAT pathway inhibits neuroinflammation in the line 61-PFF mouse model of Parkinson's disease. Google Scholar Alshammari, A. et al. Protective effect of CP690550 in MPTP-induced Parkinson's like behavioural, biochemical and histological alterations in mice. Google Scholar Rusek, M. et al. The role of the JAK/STAT signaling pathway in the pathogenesis of Alzheimer's Disease: New Potential Treatment Target. Google Scholar Nevado-Holgado, A. J. et al. Genetic and real-world clinical data, combined with empirical validation, nominate Jak–Stat signaling as a target for Alzheimer's disease therapeutic development. Google Scholar Taylor, J. M. et al. Type-1 interferon signaling mediates neuro-inflammatory events in models of Alzheimer's disease. Google Scholar Desai, R. J. et al. Targeting abnormal metabolism in Alzheimer's disease: the Drug Repurposing for Effective Alzheimer's Medicines (DREAM) study. Alzheimers Dement. Google Scholar König, L. E. et al. TYK2 as a novel therapeutic target in Alzheimer's disease with TDP-43 inclusions. Preprint at bioRxiv https://doi.org/10.1101/2024.06.04.595773 (2024). Mehla, J. et al. STAT3 inhibitor mitigates cerebral amyloid angiopathy and parenchymal amyloid plaques while improving cognitive functions and brain networks. Acta Neuropathol. Google Scholar Fan, L. & Zhou, L. AG490 protects cerebral ischemia/reperfusion injury via inhibiting the JAK2/3 signaling pathway. Brain Behav. Google Scholar Matsushita, T. et al. Inhibitory effect of baricitinib on microglia and STAT3 in a region with a weak blood–brain barrier in a mouse model of rheumatoid arthritis. Faquetti, M. L. et al. Baricitinib and tofacitinib off-target profile, with a focus on Alzheimer's disease. Alzheimers Dement. Molitor, T. P. et al. Central TYK2 inhibition identifies TYK2 as a key neuroimmune modulator. Natl Acad. Okamoto, O. K. et al. Whole transcriptome analysis of the hippocampus: toward a molecular portrait of epileptogenesis. BMC Genomics 11, 230 (2010). Tipton, A. E. et al. Selective neuronal knockout of STAT3 function inhibits epilepsy progression, improves cognition, and restores dysregulated gene networks in a temporal lobe epilepsy model. Han, C.-L. et al. The lncRNA H19 binding to let-7b promotes hippocampal glial cell activation and epileptic seizures by targeting Stat3 in a rat model of temporal lobe epilepsy. Cell Prolif. Hoffman, O. R. et al. Disease modification upon 2 weeks of tofacitinib treatment in a mouse model of chronic epilepsy. Liu, Y. et al. The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia. Kołodrubiec, J. et al. Efficacy of ruxolitinib in acute lymphoblastic leukemia: a systematic review. Dao, K.-H. T. et al. Efficacy of ruxolitinib in patients with chronic neutrophilic leukemia and atypical chronic myeloid leukemia. Qureshy, Z. et al. STAT3 activation as a predictive biomarker for ruxolitinib response in head and neck cancer. Cancer Res. Xia, D. et al. CHAF1A promotes the proliferation and growth of epithelial ovarian cancer cells by affecting the phosphorylation of JAK2/STAT3 signaling pathway. Datta, J. et al. Combined MEK and STAT3 inhibition uncovers stromal plasticity by enriching for cancer-associated fibroblasts with mesenchymal stem cell-like features to overcome immunotherapy resistance in pancreatic cancer. Taverna, J. A. et al. Single-cell proteomic profiling identifies combined AXL and JAK1 inhibition as a novel therapeutic strategy for lung cancer. Cancer Res. Schneider, J., Jeon, Y. W., Suh, Y. J. & Lim, S. T. Effects of ruxolitinib and calcitriol combination treatment on various molecular subtypes of breast cancer. Doheny, D. et al. Combined inhibition of JAK2–STAT3 and SMO–GLI1/tGLI1 pathways suppresses breast cancer stem cells, tumor growth, and metastasis. Zak, J. et al. JAK inhibition enhances checkpoint blockade immunotherapy in patients with Hodgkin lymphoma. Mathew, D. et al. Combined JAK inhibition and PD-1 immunotherapy for non-small cell lung cancer patients. Johnson, D. E., O'Keefe, R. A. & Grandis, J. R. Targeting the IL-6/JAK/STAT3 signalling axis in cancer. Curtis, J. R. et al. Malignancy risk with tofacitinib versus TNF inhibitors in rheumatoid arthritis: results from the open-label, randomised controlled ORAL Surveillance trial. Simpson, E. L. et al. Efficacy and safety of abrocitinib in adults and adolescents with moderate-to-severe atopic dermatitis (JADE MONO-1): a multicentre, double-blind, randomised, placebo-controlled, phase 3 trial. Rubbert-Roth, A. et al. Malignancy in the upadacitinib clinical trials for rheumatoid arthritis, psoriatic arthritis, ankylosing spondylitis, and non-radiographic axial spondyloarthritis. Keam, S. J. Golidocitinib: first approval. Ou, A., Ott, M., Fang, D. & Heimberger, A. The role and therapeutic targeting of JAK/STAT signaling in glioblastoma. Smedley, W. & Patra, A. JAK3 inhibition regulates stemness and thereby controls glioblastoma pathogenesis. Mukthavaram, R. et al. Effect of the JAK2/STAT3 inhibitor SAR317461 on human glioblastoma tumorspheres. Fu, W., Hou, X., Dong, L. & Hou, W. Roles of STAT3 in the pathogenesis and treatment of glioblastoma. Cell Dev. Jensen, K. V., Cseh, O., Aman, A., Weiss, S. & Luchman, H. A. The JAK2/STAT3 inhibitor pacritinib effectively inhibits patient-derived GBM brain tumor initiating cells in vitro and when used in combination with temozolomide increases survival in an orthotopic xenograft model. PLoS One 12, e0189670 (2017). McFarland, B. C. et al. Therapeutic potential of AZD1480 for the treatment of human glioblastoma. Cancer Ther. Liu, T., Li, A., Xu, Y. & Xin, Y. Momelotinib sensitizes glioblastoma cells to temozolomide by enhancement of autophagy via JAK2/STAT3 inhibition. Binnewies, M. et al. Understanding the tumor immune microenvironment (TIME) for effective therapy. Legat, F. J. Itch in atopic dermatitis - What is new? Makabe, K. et al. Baricitinib ameliorates inflammatory and neuropathic pain in collagen antibody-induced arthritis mice by modulating the IL-6/JAK/STAT3 pathway and CSF-1 expression in dorsal root ganglion neurons. Arthritis Res. Butler, D. C. et al. Chronic pruritus: a review. Lee, J. et al. Treatment with upadacitinib in refractory prurigo nodularis: a prospective cohort study. Allergy Asthma Immunol. Pezzolo, E. et al. Effective response to upadacitinib in patients affected by prurigo nodularis and by atopic dermatitis with a predominant prurigo nodularis pattern: a multicenter case series study. Muntaner-Virgili, C., Moreno-Vilchez, C., Torrecilla-Vall-Llossera, C. & Figueras-Nart, I. Upadacitinib for prurigo nodularis. JAAD Case Rep. 48, 131–133 (2024). Sardana, K., Mathachan, S. R., Muddebihal, A., Agrawal, D. & Ahuja, A. Translating tissue expression of STAT 1, 3 and 6 in prurigo nodularis to clinical efficacy of oral tofacitinib—a prospective single-arm investigational study. Indian J Dermatol Venereol Leprol https://doi.org/10.25259/IJDVL_1017_2024 (2025). Dai, R., Chi, C.-C., Lou, Y., Hsieh, I.-J. Efficacy and safety of tofacitinib in patients with refractory prurigo nodularis: a 16-week, single-center, prospective, observational pilot study. Dermatologic Ther. Kwatra, S. G. et al. Efficacy and safety of abrocitinib in prurigo nodularis and chronic pruritus of unknown origin: a nonrandomized controlled trial. JAMA Dermatol. Liang, J. et al. Abrocitinib monotherapy for refractory prurigo nodularis: report of two successful cases. Sun, F. & Wu, Z. Successful treatment of refractory prurigo nodularis with abrocitinib. Yew, Y. W. & Yeo, P. M. Comparison between dupilumab and oral Janus kinase inhibitors in the treatment of prurigo nodularis with or without atopic dermatitis in a tertiary care center in Singapore. JAAD Int. Ju, T., Labib, A., Vander Does, A. & Yosipovitch, G. Topical Janus kinase-signal transducers and activators of transcription inhibitor tofacitinib is effective in reducing nonatopic dermatitis chronic itch: a case series. Le Voyer, T. et al. JAK inhibitors are effective in a subset of patients with juvenile dermatomyositis: a monocentric retrospective study. Ding, Y. et al. Janus kinase inhibitor significantly improved rash and muscle strength in juvenile dermatomyositis. Yu, Z., Wang, L., Quan, M., Zhang, T. & Song, H. Successful management with Janus kinase inhibitor tofacitinib in refractory juvenile dermatomyositis: a pilot study and literature review. Sabbagh, S. et al. Treatment of anti-MDA5 autoantibody-positive juvenile dermatomyositis using tofacitinib. Aeschlimann, F. A. et al. A child with severe juvenile dermatomyositis treated with ruxolitinib. Kim, H. et al. Janus kinase (JAK) inhibition with baricitinib in refractory juvenile dermatomyositis. Papadopoulou, C., Hong, Y., Omoyinmi, E., Brogan, P. A. & Eleftheriou, D. Janus kinase 1/2 inhibition with baricitinib in the treatment of juvenile dermatomyositis. Loricera, J. et al. Effectiveness of janus kinase inhibitors in relapsing giant cell arteritis in real-world clinical practice and review of the literature. Arthritis Res. Eriksson, P., Skoglund, O., Hemgren, C. & Sjöwall, C. Clinical experience and safety of Janus kinase inhibitors in giant cell arteritis: a retrospective case series from Sweden. Prigent, K., Aouba, A., Aide, N. & de Boysson, H. JAK inhibitor effectiveness in giant-cell arteritis with large-vessel involvement assessed by 18F-FDG PET-CT. Clin. Zhang, H. et al. Inhibition of JAK–STAT signaling suppresses pathogenic immune responses in medium and large vessel vasculitis. Blockmans, D. et al. A phase 3 trial of upadacitinib for giant-cell arteritis. Sanada, A., Abe, N., Bohgaki, M. & Kasahara, H. Therapeutic effectiveness of upadacitinib combined with glucocorticoid on remission induction and maintenance in giant cell arteritis. Schwartz, D. M. et al. JAK inhibition as a therapeutic strategy for immune and inflammatory diseases. Nezamololama, N., Fieldhouse, K., Metzger, K. & Gooderham, M. Emerging systemic JAK inhibitors in the treatment of atopic dermatitis: a review of abrocitinib, baricitinib, and upadacitinib. Drugs Context. Su, Q. et al. Discovery of (2R)-N-[3-[2-[(3-methoxy-1-methyl-pyrazol-4-yl)amino]pyrimidin-4-yl]-1H-indol-7-yl]-2-(4-methylpiperazin-1-yl)propenamide (AZD4205) as a potent and sSelective Janus kinase 1 inhibitor. Center for Drug Evaluation and Research. Multi-discipline review Application Number: 217900Orig1s000 (deuruxolitinib). Chrencik, J. E. et al. Structural and thermodynamic characterization of the TYK2 and JAK3 kinase domains in complex with CP-690550 and CMP-6. Williams, N. K. et al. Dissecting specificity in the Janus kinases: the structures of JAK-specific inhibitors complexed to the JAK1 and JAK2 protein tyrosine kinase domains. Hamaguchi, H. et al. Discovery and structural characterization of peficitinib (ASP015K) as a novel and potent JAK inhibitor. Medicinal Chem. Gupta, V. et al. Matching-adjusted indirect comparison of the pelabresib–ruxolitinib combination vs JAKi monotherapy in myelofibrosis. Blood Adv. Mascarenhas, J. et al. MANIFEST: pelabresib in combination with ruxolitinib for Janus kinase inhibitor treatment-naïve myelofibrosis. Rampal, R. K. et al. Pelabresib plus ruxolitinib for JAK inhibitor-naive myelofibrosis: a randomized phase 3 trial. Metzger, M., Avigan, Z. M., Vachhani, P., Waksal, J. & Mascarenhas, J. A novel application of XPO1 inhibition for the treatment of myelofibrosis. Blood Neoplasia 1, 100010 (2024). Mascarenhas, J. O. et al. Results from the randomized, multicenter, global phase 3 BOREAS study: navtemadlin versus best available therapy in JAK inhibitor relapsed/refractory myelofibrosis. Vachhani, P. et al. POIESIS: a randomized, double-blind, placebo-controlled, multicenter, global phase 3 study of navtemadlin as add-on to ruxolitinib in JAK inhibitor-naïve patients with myelofibrosis who have a suboptimal response to ruxolitinib. Download references This work was funded by the Academy of Finland (grant numbers 340572 and 335437 for T.H. and O.S., respectively), the National Cancer Institute (grant number R35 CA231991 for B.F.C. ), Tampere Tuberculosis Foundation, Sigrid Juselius Foundation, Finnish Cancer Foundation and competitive research funding from Tampere University Hospital. Faculty of Medicine and Health Technology, Tampere University, Tampere, Finland Teemu Haikarainen, Anniina T. Virtanen & Olli Silvennoinen Fimlab Laboratories, Tampere, Finland Teemu Haikarainen & Olli Silvennoinen Department of Chemistry, Scripps Research, La Jolla, CA, USA Benjamin F. Cravatt Institute of Biotechnology, HiLIFE Helsinki Institute of Life Science, University of Helsinki, Helsinki, Finland Olli Silvennoinen Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors researched data for the article and made a substantial contribution to discussion of content, writing, reviewing and editing of the manuscript. Correspondence to Teemu Haikarainen or Olli Silvennoinen. holds a patent on JAK kinases (US Patent 8,841,078) and is a co-founder and adviser for Ajax Therapeutics. is a founder and scientific adviser of Vividion Therapeutics and Magnet Therapeutics. The other authors declare no competing interests. Nature Reviews Drug Discovery thanks Jean-Baptiste Telliez and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Haikarainen, T., Virtanen, A.T., Cravatt, B.F. et al. Pharmacological targeting of the JAK–STAT pathway: new concepts and emerging indications. Nat Rev Drug Discov  (2026). Download citation Accepted: 04 November 2025 Published: 05 January 2026 Version of record: 05 January 2026 DOI: https://doi.org/10.1038/s41573-025-01336-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Drug Discovery (Nat Rev Drug Discov) ISSN 1474-1784 (online) ISSN 1474-1776 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            