
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-24</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03487-6'>How Napoleon's army met its doom: DNA reveals surprise illnesses had a role</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 15:16:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Cold, hunger and disease killed some 60% of Napoleon's 500,000-strong army during its retreat from Moscow in 1812.Credit: R. Barbieri et al./Curr. Some disease-ridden French soldiers have just received a new diagnosis — although it comes about two centuries too late. While retreating from Moscow in 1812, French emperor Napoleon I lost most of his army to hunger, illness and the frigid Russian winter. “This paper will be really interesting to history buffs,” says Anne Stone, who researches anthropological genetics at Arizona State University in Tempe. Eventually, in 2006, researchers studying DNA in some of the soldiers' remains identified two disease agents2: Rickettsia prowazekii, which causes typhus, and Bartonella quintana, which causes trench fever. That technique allows researchers to detect DNA sequences for specific pathogens, so it excelled at verifying the presence of microorganisms that historians had already thought were present in the remains. But it isn't as well suited for identifying unsuspected diseases. Armed with new techniques that can screen ancient DNA for any known pathogen, the team set out to identify other microbes that might have afflicted the troops. They then used next-generation techniques to sequence millions of different DNA fragments from a sample. That allowed the authors to look for a wide array of pathogens all at once. This ancient pathogen became deadlier when humans started wearing wool Ancient tooth DNA reveals how ‘cold sore' herpes virus has evolved Population-specific polygenic risk scores for people of Han Chinese ancestry Japan declares a flu epidemic — what this means for other nations The rise of ‘nightmare bacteria': antimicrobial resistance in five charts Chen-Ning Yang obituary: intuitive physicist whose theories on broken symmetries were proved right Job Title:    Associate Editor, Discover Journals (Computer Science,  Cybersecurity or Networks) Location:   Pune/Beijing/Shanghai/Nanjing, Hybrid ... This ancient pathogen became deadlier when humans started wearing wool Ancient tooth DNA reveals how ‘cold sore' herpes virus has evolved An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03390-0'>AI chatbots are sycophants — researchers say it's harming science</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 13:42:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. AI's inclination to be helpful affects many of the tasks that researchers use LLMs for.Credit: Smith Collection/Gado/Getty Artificial intelligence (AI) models are 50% more sycophantic than humans, an analysis published this month has found. The study, which was posted as a preprint1 on the arXiv server, tested how 11 widely used large language models (LLMs) responded to more than 11,500 queries seeking advice, including many describing wrongdoing or harm. AI Chatbots — including ChatGPT and Gemini — often cheer users on, give them overly flattering feedback and adjust responses to echo their views, sometimes at the expense of accuracy. Researchers analysing AI behaviours say that this propensity for people-pleasing, known as sycophancy, is affecting how they use AI in scientific research, in tasks from brainstorming ideas and generating hypotheses to reasoning and analyses. “Knowing that these models are sycophantic makes me very wary whenever I give them some problem,” he adds. In a study posted on the preprint server arXiv on 6 October2, Dekoninck and his colleagues tested whether AI sycophancy affects the technology's performance in solving mathematical problems. The researchers designed experiments using 504 mathematical problems from competitions held this year, altering each theorem statement to introduce subtle errors. They then asked four LLMs to provide proofs for these flawed statements. AI chatbots are already biasing research — we must establish guidelines for their use now AI chatbots are already biasing research — we must establish guidelines for their use now When Dekoninck and his team changed the prompts to ask each LLM to check whether a statement was correct before proving it, DeepSeek's sycophantic answers fell by 34%. The study is “not really indicative of how these systems are used in real-world performance, but it gives an indication that we need to be very careful with this”, says Dekoninck. Zitnik and her colleagues have observed similar patterns when using their multi-agent systems, which integrate several LLMs to carry out complex, multi-step processes such as analysing large biological data sets, identifying drug targets and generating hypotheses. ‘AI models are capable of novel research': OpenAI's chief scientist on what to expect How AI-powered science search engines can speed up your research Universities are embracing AI: will students get smarter or stop thinking? Google Scholar tool gives extra credit to first and last authors UNESCO stands at a crossroads — researchers must back its new leader Job Title:    Associate Editor, Discover Journals (Computer Science,  Cybersecurity or Networks) Location:   Pune/Beijing/Shanghai/Nanjing, Hybrid ... ‘AI models are capable of novel research': OpenAI's chief scientist on what to expect How AI-powered science search engines can speed up your research An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a69110918/a-new-solid-state-battery-could-take-your-ev-an-astounding-800-miles-on-a-single-charge/'>A New Solid-State Battery Could Take Your EV an Astounding 800 Miles on a Single Charge</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Energy density is the latest hot pursuit for unlocking more range and performance in electric vehicles. Measured in Watt Hours per Kilogram (Wh/Kg), energy density is effectively just the power-to-weight ratio of a battery system. Chery Automobile is one of many Chinese EV manufacturers leading the solid-state charge, no pun intended, recently unveiling its new prototype battery with an energy density of 600 Wh/Kg—double the energy density of most current EV batteries. For context, the nickel-cobalt-aluminum batteries, often found in Tesla's Long-Range and Performance models, have a 200-260 Wh/Kg energy density. Increased energy density equals more available range and performance. Critically, solid-state batteries bring significantly less fire risk as a solid electrolyte (the medium that enables a battery to charge and discharge) isn't flammable. This new battery architecture also offers increased thermal stability, lowering the chances of thermal runaway—a process that ends with the battery bursting into flames. Last but not least, solid electrolytes will also allow the batteries of the future to be formed into much more complex shapes. The keen among you will know that solid-state battery advantages span much further than just charging, discharging, and durability. Reports show Chery's battery running flawlessly through a nail penetration test, also shaking off damage from a power drill without catching fire or producing any mystery smoke. However, even with a promising proof of concept, there's still some assembly required before we'll see these battery packs in production vehicles. Should everything go smoothly, Chery could potentially jump major Chinese competitors like BYD and CATL to large-scale solid-state battery production. Toyota is another major player in the solid-state battery game. While smaller companies like Chery Automobile are moving fast and iterating on prototypes, Toyota is leading the way in patent activity; as it stands, the Japanese automaker has 1,700 patents related to solid-state battery tech, with over 1,500 of them active. Toyota recently announced a strategic partnership with Idemitsu Kosan, a Japanese energy company that can assist them with rolling out solid-state batteries as soon as 2027. Engineers faced considerable challenges getting solid-state batteries to work. Like any cutting-edge technology, development is a long and expensive process, with solid-state batteries costing up to 2.8 times more than a legacy liquid-electrolyte lithium-ion pack. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. Toyota Says Hybrids are the Way Forward (For Now) How to Clean Your Headlights and Make Them Shine How Often Should You Really Change Your Car's Oil? How to Get Keys Out of a Locked Car</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/fda-warns-people-not-to-eat-recalled-eggs-contaminated-with-salmonella/'>FDA Warns People Not to Eat Recalled Eggs Contaminated with Salmonella</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>FDA Warns People Not to Eat Recalled Eggs Contaminated with Salmonella Here's what to do if you have recalled eggs The company issued a voluntary recall in late September. It also distributed the eggs to other companies, including Santa Fe, Tex.–based Kenz Henz, which issued its own voluntary recall on October 16. Those eggs were sold in stores across Houston, Tex. Black Sheep also distributed eggs to Arkansas and Missouri, where companies may have repackaged them. “In response to several consumer and customer inquiries, we want to confirm that all of our available eggs are safe to eat,” said the Black Sheep Egg Company in a post on its Facebook page on October 23. “We want to emphasize that all eggs tested negative for Salmonella and showed no signs of contamination. This decision reflects our company's unwavering commitment to providing safe, high-quality products.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. On September 29 the FDA had launched an inspection of Black Sheep's egg-processing facility and found 40 environmental samples that were positive for Salmonella, a bacterium that can cause serious foodborne illness. A June Salmonella outbreak in organic and cage-free eggs sickened at least 134 people, but public health officials have not yet reported illnesses linked to this latest recall. Here's what you need to know about Salmonella and the illness it can cause. Salmonella is a genus of bacteria that can sicken humans. The Salmonella bacteria that are most related to foodborne illness are found in the intestines, skin and fur of animals such as chickens and cows. And they spread through fecal contamination of surfaces that are improperly cleaned or of raw food that is handled without proper sanitary measures. Salmonella often spreads through eggs or chicken (the animals are frequently raised in crowded and unsanitary conditions), but there have also been outbreaks in peanut butter, raw pork, raw beef, cucumbers and raw flour (such as in cookie dough). Salmonella infection can cause stomach cramps, diarrhea, vomiting and fever. Children under age five, older adults and people with a weakened immune system are most vulnerable. The FDA says you should not eat, sell or serve recalled eggs. You should also clean and sanitize anything the eggs have touched. Do not consume raw eggs, especially if you have a compromised immune system. Eggs should be cooked until they reach a temperature of 160 degrees Fahrenheit (71.1 degrees Celsius)—usually this occurs when the yolks become firm. Wash your hands, cooking implements and surfaces thoroughly with hot, soapy water before and after handling raw eggs or foods that contain them. Most people get over their infections in three to four days. If you have diarrhea that lasts more than two days or a fever above 102 degrees F (39 degrees C), however, you should see a health care professional. Tanya Lewis is a senior editor covering health and medicine at Scientific American. Previously, she has written for outlets that include Insider, Wired, Science News, and others. She has a degree in biomedical engineering from Brown University and one in science communication from the University of California, Santa Cruz. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a69082728/universe-signal-epoch-of-reionization/'>A Telescope in the Desert Reached Back a Billion Years and Uncovered Something Incredible</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>However, the sequel to this explosive preamble is decidedly different in tone. This is the universe's toddler days, so to speak. However, unlike an obsessive parent documenting every milestone of their drooling progeny, astrophysicists don't have such a wealth of evidence—instead, they have to go looking for it. "Our measurements show that it is at least heated by a certain amount. Not by a lot, but it tells us that very cold reionization is ruled out.” Operating at 70–300 MHz, the telescope is purpose-built to detect that aforementioned neutral hydrogen emission from the “epoch of reionization.” Even so, the authors needed to perform a lot of subtraction in order to isolate what they were looking for from the cosmic noise of nearby stars, galaxies, and even Earth's atmosphere. Because the team was able to create this filtered data—as well as having the benefit of ten years worth of it—they could make a discovery by omission. If the universe was cold at this time, it would emit a certain kind of signal, but the researchers found no evidence of it, suggesting that at least some warming was going on. They haven't found the tell-tale signal of that heating yet, but the researchers say that it's likely only a matter of time. “All these existing techniques will help us find what's missing,” Curtin University's Ridhima Nunhokee, a co-author of the study, said in a press statement. Experts Say Time Travel Is Possible at Warp Speed There Might Be a Black Hole Inside This Star The Best Star Projectors Turn Your Ceiling Into a Night Sky A Scientist Says This Object May Be an Alien Craft Aliens May Have Started Life on Earth, Expert Says</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03421-w'>Audio long read: How to get the best night's sleep — what the science says</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 12:17:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Lynne Peeples is a science journalist in Seattle, Washington, and author of The Inner Clock. However, scientists warn that online hacks and pricey tools aren't always effective, and suggest that lessons learnt about the workings of a network of biological clocks found in the human body could ultimately lead to improved sleep. How to detect consciousness in people, animals and maybe even AI How the brain wakes up from sleep — and produces that morning feeling Pancreatic cancer relies on opposing signalling pathways to drive its cellular diversity Job Title:    Associate Editor, Discover Journals (Computer Science,  Cybersecurity or Networks) Location:   Pune/Beijing/Shanghai/Nanjing, Hybrid ... How to detect consciousness in people, animals and maybe even AI How the brain wakes up from sleep — and produces that morning feeling An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251024041822.htm'>Scientists just changed the nature of matter with a flash of light</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 09:42:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>No magic wand or special potion is needed -- only light. These vibrations can transmit and store information at terahertz speeds. The entire process happens at room temperature and produces almost no heat. Even better, it doesn't rely on rare or exotic materials. Researchers observed the effect in common, naturally grown crystals that are widely available. Now imagine using the same approach to tap into quantum effects -- phenomena so delicate they are typically observed only near absolute zero (around -270 degrees Celsius) -- but doing it at room temperature, with no costly cooling systems required. It might sound like science fiction, yet this breakthrough is real. A team of physicists at the University of Konstanz, led by Davide Bossini, has developed an experimental technique that makes it possible. By using laser pulses to coherently excite pairs of magnons (quanta of spin waves), the researchers achieved remarkable effects that could influence both information technology and quantum research. Before diving deeper, it helps to understand what magnons are and why they matter. Our current information systems are already straining under the pressure, and a data bottleneck threatens to slow technological progress. The team at Konstanz has now found a way to do exactly that. No theory has ever predicted it," says Davide Bossini. Not only does the process work -- it also has spectacular effects. "Every solid has its own set of frequencies: electronic transitions, lattice vibrations, magnetic excitations. Every material resonates in its own way," explains Bossini. It is precisely this set of frequencies that can be influenced through the new process. It has practically become a different material with new properties for the time being," says Bossini. Centuries ago, it was already used for compasses in seafaring," explains Bossini. It is perfectly possible that haematite will now also be used for quantum research in the future. Sounds like magic, but it is just technology and cutting-edge research. Note: Content may be edited for style and length. Scientists Use “Supercentenarian Longevity Gene” to Slow Rapid Aging Disease Scientists Discover How Life's Most Essential Molecule Enters Mitochondria New Drug Could “Un-Stiffen” Aging Hearts, Offering Hope for the Most Common Form of Heart Failure Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02859-7'>Democratizing protein language model training, sharing and collaboration</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 09:29:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Training and deploying large-scale protein language models typically requires deep machine learning expertise—a barrier for researchers outside this field. SaprotHub overcomes this challenge by offering an intuitive platform that facilitates training and prediction as well as storage and sharing of models. Here we provide the ColabSaprot framework built on Google Colab, which potentially powers hundreds of protein training and prediction applications, enabling researchers to collaboratively build and share customized models. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Prices may be subject to local taxes which are calculated during checkout The pretraining dataset for training Saprot is available online (https://huggingface.co/datasets/westlake-repl/AF2_UniRef50. Downstream task datasets are all stored online (https://huggingface.co/SaProtHub). Materials for user study (https://drive.google.com/file/d/1LdGRnwt2lttnszNBAJ0F967A8rguPq8b/view?usp=sharing), raw data with detailed wet lab information (https://drive.google.com/file/d/1IYcOqRuF76LusG7DMI4kEr2mJL6aE7g-/view?usp=sharing) and additional wet lab experimental results collected from the research community (https://drive.google.com/file/d/1ZcDl0XYksTcUEaCfPI0tuEVo5CA031dg/view?usp=sharing) are available from a Google Drive. All unique/stable reagents generated in this study are available from the lead contact (yuanfajie@westlake.edu.cn). Saprot is an open-sourced model with MIT license. The code is available from GitHub (https://github.com/westlake-repl/Saprot). The code implementation of ColabSaprot notebook is also available from GitHub (https://github.com/westlake-repl/SaProtHub). ColabSaprot service (latest version: version 2) is available online (https://colab.research.google.com/github/westlake-repl/SaprotHub/blob/main/colab/SaprotHub_v2.ipynb?hl=en; the previous version is still maintained on the SaprotHub GitHub). All fine-tuned Saprot models can be obtained through SaprotHub (https://huggingface.co/SaProtHub) through the dedicated search engine (https://huggingface.co/spaces/SaProtHub/SaprotHub-search). Our OPMC members have also implemented ColabSeprot (https://colab.research.google.com/github/westlake-repl/SaprotHub/blob/main/colab/ColabSeprot.ipynb?hl=en), including ColabProTrek (35M and 650M), ColabESM1b (650M), ColabESM2(35M, 150M and 650M), ColabProtBert (420M), SeprotHub (https://huggingface.co/SeprotHub) and an independent ColabProtT5 (https://colab.research.google.com/github/westlake-repl/SaprotHub/blob/main/colab/ColabProtT5.ipynb). Drug discovery: a historical perspective. Jacob, François & Monod, J. Genetic regulatory mechanisms in the synthesis of proteins. The ubiquitin–proteasome proteolytic pathway: destruction for the sake of construction. Highly accurate protein structure prediction with AlphaFold. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Language models enable zero-shot prediction of the effects of mutations on protein function. 35th International Conference on Neural Information Processing Systems (eds Ranzato, M. et al.) (NIPS, 2021). Rao, R. M. et al. MSA transformer. 38th International Conference on Machine Learning (eds Meila, M. & Zhang, T.) (PMLR, 2021). Elnaggar, A. et al. ProtTrans: toward understanding the language of life through self-supervised learning. Heinzinger, M. et al. Bilingual language model for protein sequence and structure. Notin, P. et al. Tranception: protein fitness prediction with autoregressive transformers and inference-time retrieval. 39th International Conference on Machine Learning (eds Chaudhuri, K. et al.) (PMLR, 2022). A., Weinstein, E. N., Naik, N. & Madani, A. ProGen2: exploring the boundaries of protein language models. Ferruz, N., Schmidt, S. & Höcker, B. ProtGPT2 is a deep unsupervised language model for protein design. Su, J. et al. SaProt: orotein language modeling with structure-aware vocabulary. 12th International Conference on Learning Representations (ed Kim, B.) Mirdita, M. et al. ColabFold: making protein folding accessible to all. Hu, E. J. et al. LoRA: low-rank adaptation of large language models. In The Tenth International Conference on Learning Representations https://openreview.net/pdf?id=nZeVKeeFYf9 (ICLR, 2022). Pfeiffer, J. et al. AdapterHub: a framework for adapting transformers. 2020 EMNLP (Systems Demonstrations) https://aclanthology.org/2020.emnlp-demos.7.pdf (Association for Computational Linguistics, 2020). Kirkpatrick, J. et al. Overcoming catastrophic forgetting in neural networks. van Kempen, M. et al. Fast and accurate protein structure search with Foldseek. Hayes, T. et al. Simulating 500 million years of evolution with a language model. Li, M. et al. ProSST: protein language modeling with quantized structure and disentangled attention. In 38th Conference on Neural Information Processing Systems (NeurIPS 2024) https://openreview.net/forum?id=4Z7RZixpJQ&referrer=%5Bthe%20profile%20of%20Bozitao%20Zhong%5D(%2Fprofile%3Fid%3D~Bozitao_Zhong1) (NeurIPS, 2024). Wang, X. et al. DPLM-2: a multimodal diffusion protein language model. In The Thirteenth International Conference on Learning Representation https://openreview.net/pdf?id=5z9GjHgerY (ICLR, 2025). Tan, Y., Wang, R., Wu, B., Hong, L. & Zhou, B. Retrieval-enhanced mutation mastery: augmenting zero-shot prediction of protein language model. Preprint at https://arxiv.org/abs/2410.21127 (2024). Pourmirzaei, M., Esmaili, F., Pourmirzaei, M., Wang, D. & Xu, D. Prot2Token: a multi-task framework for protein language processing using autoregressive language modeling. In ICML 2024 Workshop on Efficient and Accessible Foundation Models for Biological Discovery https://openreview.net/pdf?id=5z9GjHgerY (2024). Gao, K. et al. Tokenizing 3D molecule structure with quantized spherical coordinates. Preprint at https://arxiv.org/abs/2412.01564 (2024). Lin, X. et al. Tokenizing foldable protein structures with machine-learned artificial amino-acid vocabulary. Preprint at bioRxiv https://doi.org/10.1101/2023.11.27.568722 (2023). Ivanisenko, N. V. et al. SEMA 2.0: web-platform for B-cell conformational epitopes prediction using artificial intelligence. Devlin, J., Chang, Ming-Wei, Lee, K. & Toutanova, K., BERT: pre-training of deep bidirectional transformers for language understanding. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers) 4171–4186 (Association for Computational Linguistics, 2019). Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. Rao, R. et al. Evaluating protein transfer learning with tape. 33rd International Conference on Neural Information Processing Systems (eds Wallach, H. M. et al.) (NIPS, 2019). Kucera, T., Oliver, C., Chen, D. & Borgwardt, K. ProteinShake: building datasets and benchmarks for deep learning on protein structures. 36th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) (NIPS, 2024). PEER: a comprehensive and multi-task benchmark for protein sequence understanding. 35th International Conference on Neural Information Processing Systems (eds Ranzato, M. et al.) (2021). Hie, B., Zhong, E. D., Berger, B. Learning the language of viral evolution and escape. Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Robust deep learning-based protein sequence design using proteinmpnn. Tsuboyama, K. et al. Mega-scale experimental analysis of protein folding stability in biology and design. Notin, P. et al. ProteinGym: large-scale benchmarks for protein fitness prediction and design. 36th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) (NIPS, 2024). Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. Tan, Y. et al. VenusX: unlocking fine-grained functional understanding of proteins. Yan, S. et al. Protap: a benchmark for protein modeling on realistic downstream applications. Enhancing efficiency of protein language models with minimal wet-lab data through few-shot learning. Dai, F. et al. Toward de novo protein design from natural language. Preprint at bioRxiv https://doi.org/10.1101/2024.08.01.606258 (2024). Meshchaninov, V. et al. Diffusion on language model encodings for protein sequence generation Preprint at https://arxiv.org/abs/2403.03726 (2024). Sagawa, T., Kanao, E., Ogata, K., Imami, K. & Ishihama, Y. Prediction of protein half-lives from amino acid sequences by protein language models. Preprint at bioRxiv https://doi.org/10.1101/2024.09.10.612367 (2024). Bushuiev, A. et al. Training on test proteins improves fitness, structure, and function prediction. Advancing biomolecular understanding and design following human instructions. Zhou, X. et al. Decoding the molecular language of proteins with Evola. Preprint at bioRxiv https://doi.org/10.1101/2025.01.05.630192 (2025). & Xue, Z. SSAlign: ultrafast and sensitive protein structure search at scale. Preprint at bioRxiv https://doi.org/10.1101/2025.07.03.662911 (2025). & Ounis, I. FusionDTI: fine-grained binding discovery with token-level fusion for drug-target interaction. Scaling structure aware virtual screening to billions of molecules with sprint. Protein language models-assisted optimization of a uracil-N-glycosylase variant enables programmable T-to-G and T-to-C base editing. & Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. Hsu, C. et al. Learning inverse folding from millions of predicted structures. 39th International Conference on Machine Learning (eds Chaudhuri, K. et al.) (PMLR, 2022). Sledzieski, S. et al. Democratizing protein language models with parameter-efficient fine-tuning. Zeng, S., Wang, D. & Xu, D. PEFT-SP: parameter-efficient fine-tuning on large protein language models improves signal peptide prediction. Sledzieski, S., Kshirsagar, M., Berger, B., Dodhia, R. & Ferres, J. L. Parameter-efficient fine-tuning of protein language models improves prediction of protein-protein interactions. In Machine Learning for Structural Biology Workshop, NeurIPS 2023 https://www.mlsb.io/papers_2023/Parameter-Efficient_Fine-Tuning_of_Protein_Language_Models_Improves_Prediction_of_Protein-Protein_Interactions.pdf (2023). Wang, D. et al. S-PLM: structure-aware protein language model via contrastive learning between sequence and structure. Su, J., Zhou, X., Zhang, X. & Yuan, F. A trimodal protein language model enables advanced protein searches. van den Oord, A. et al. Neural discrete representation learning. 30th International Conference on Neural Information Processing Systems (eds Lee, D. D., von Luxburg, U., Garnett, R., Sugiyama, M. & Guyon, I.) Efficient training of BERT by progressively stacking. 36th International Conference on Machine Learning (eds Chaudhuri, K. & Salakhutdinov, R.) (PMLR, 2019). & Hutter, F. Fixing weight decay regularization in Adam. Preprint at OpenReview https://openreview.net/forum?id=rk6qdGgCZ (2018). Yang, K. K., Zanichelli, N. & Yeh, H. Masked inverse folding with sequence transfer for protein representation learning. Zhang, Z. et al. Protein representation learning by geometric structure pretraining. In First Workshop of Pre-training: Perspectives, Pitfalls, and Paths Forward at ICML 2022 https://openreview.net/pdf?id=V5MEFikiBQy (2023). Dallago, C. et al. FLIP: benchmark tasks in fitness landscape inference for proteins. Neural Information Processing Systems Track on Datasets and Benchmarks https://openreview.net/pdf?id=p2dMLEwL8tF (2021). Almagro Armenteros, J. J., Sønderby, C. K., Sønderby, S. K., Nielsen, H. & Winther, O. DeepLoc: prediction of protein subcellular localization using deep learning. Exploring evolution-aware & -free protein language models as protein function predictors. 35th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) (NIPS, 2023). Gligorijević, V. et al. Structure-based protein function prediction using graph convolutional networks. Orengo, C. A. et al. CATH—a hierarchic classification of protein domain structures. Ingraham, J., Garg, V., Barzilay, R. & Jaakkola, T. Generative models for graph-based protein design. 32nd International Conference on Neural Information Processing Systems (eds Bengio, S., Wallach, H. M., Larochelle, H., Grauman, K. & Cesa-Bianchi, N.) (NIPS, 2018). Houlsby, N. et al. Parameter-efficient transfer learning for NLP. 36th International Conference on Machine Learning (eds Chaudhuri, K. & Salakhutdinov, R.) (PMLR, 2019). Fu, J. et al. Exploring adapter-based transfer learning for recommender systems: empirical studies and practical insights. 17th ACM International Conference on Web Search and Data Mining (eds Angélica, L., Lattanzi, S. & Muñoz Medina, A.) Yuan, F., He, X., Karatzoglou, A. & Zhang, L. Parameter-efficient transfer from sequential behaviors for user modeling and recommendation. 43rd International ACM SIGIR Conference on Research and Development in Information Retrieval (eds Huang, J., Chang, Y. Schreiber, A. ESMBind and QBind: LoRA, QLoRA, and ESM-2 for predicting binding sites and post translational modification. Preprint at bioRxiv https://doi.org/10.1101/2023.11.13.566930 (2023). Schmirler, R., Heinzinger, M. & Rost, B. Fine-tuning protein language models boosts predictions across diverse tasks. Karimi Mahabadi, R., Henderson, J. & Ruder, S. COMPACTER: efficient low-rank hypercomplex adapter layers. 35th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) (NIPS, 2023). Fu, L. et al. Critical Assessment of Protein Engineering (CAPE): a student challenge on the cloud. He, Y., Zhou, X., Yuan, F. & Chang, X. Protocol to use protein language models predicting and following experimental validation of function-enhancing variants of thymine-N-glycosylase. We thank N. Li and the Westlake University HPC Center for computing resources and J. Huang, J. Zeng, D. Li, L. Cao and D. Li for discussions and paper advice. We thank J. M. Jumper for providing insights into AF2 Evoformer and advice on the SA token design. This work was supported by the National Key Research and Development Program of China (2022ZD0115100), the National Natural Science Foundation of China (U21A20427), the National Research Foundation of Korea (RS-2020-NR049543, RS-2021-NR061659, RS-2021-NR056571 and RS-2024-00396026), the Creative-Pioneering Researchers Program and Novo Nordisk Foundation (NNF24SA0092560), the Westlake Center of Synthetic Biology and Integrated Bioengineering, the Zhejiang Province Leading Geese Plan (2025C01094), the Zhejiang Key Laboratory of Low-Carbon Intelligent Synthetic Biology (2024ZY01025) and the Research Center for Industries of the Future (WU2022C030). Zhejiang University, Hangzhou, China Westlake University, Hangzhou, China Jin Su, Zhikai Li, Tianli Tao, Chenchen Han, Yan He, Fengyuan Dai, Xuting Zhang, Yuyang Zhou, Junjie Shan, Xibin Zhou, Xing Chang, Shiyu Jiang, Jianming Liu, Jia Zheng, Stan Z. Li, Anping Zeng, Huaizong Shen, Jijie Chai, Feng Ju, Xing Chang, Fajie Yuan, Jin Su, Zhikai Li, Tianli Tao, Chenchen Han, Yan He, Fengyuan Dai, Xuting Zhang, Yuyang Zhou, Junjie Shan, Xibin Zhou, Shiyu Jiang, Jiawei Zhang, Yuliang Fan & Fajie Yuan Qingyan Yuan, Tianyuan Zhang & Cheng Li Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China Yuan Gao & Tong Si Zhejiang Lab, Hangzhou, China School of Biological Sciences, Seoul National University, Seoul, South Korea Milot Mirdita & Martin Steinegger Massachusetts Institute of Technology, Cambridge, MA, USA University of Wisconsin-Madison, Madison, WI, USA Microsoft Research, Cambridge, MA, USA Harvard Medical School, Boston, MA, USA Pascal Notin & Debora S. Marks University of Pennsylvania, Philadelphia, PA, USA Duke University, Durham, NC, USA Rohit Singh & Philip A. Romero Helmholtz Zentrum München, Neuherberg, Germany Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Barcelona, Spain Universitat Pompeu Fabra(UPF), Barcelona, Spain Columbia University, New York, NY, USA Colorado State University, Fort Collins, CO, USA University of California, San Francisco, CA, USA Shandong University, Jinan, China Chinese Academy of Sciences, Beijing, China Chinese Academy of Sciences, Shenzhen, China Tong Si & Yuan Gao ShanghaiTech University, Shanghai, China Rice University, Houston, TX, USA Linqi Cheng & Xinzhe Zheng Lei Chen & Rui Long Xianghu laboratory, Hangzhou, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized and led this research. conducted the main research and managed the technical implementation. and J. Shan designed the SA token and Saprot. developed the ColabSaprot and SaprotHub. developed the ColabSeprot and SeprotHub. designed the core idea of ColabSaprot and SaprotHub. participated in discussions of ColabSaprot and SaprotHub, provided constructive ideas and revised the manuscript. conducted the partial zero-shot mutation experiments. trained the Saprot 1.3B version. collected the AlphaFold DB dataset. (GFP) conducted the wet lab experiments. conducted some experimental result analysis. participated in early wet lab experiments. Other OPMC authors contributed to wet lab validation, development of ColabSaprot and ColabSeprot, manuscript writing and proofreading, idea discussions, expert advice and promoting and testing ColabSaprot and ColabSeprot. Correspondence to Fajie Yuan. The authors declare no competing interests. Nature Biotechnology thanks Feiran Li and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Su, J., Li, Z., Tao, T. et al. Democratizing protein language model training, sharing and collaboration. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251023031612.htm'>Japanese scientists unveil a quantum battery that defies energy loss</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 07:49:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists from the RIKEN Center for Quantum Computing and Huazhong University of Science and Technology have carried out a theoretical study showing how a "topological quantum battery" could be efficiently designed. As environmental sustainability becomes an increasingly urgent global concern, researchers are looking for new approaches to next-generation energy storage. Quantum batteries -- miniaturized theoretical devices that store energy using quantum phenomena such as superposition, entanglement, and coherence rather than traditional chemical reactions -- could redefine how power is stored and transferred. In real-world conditions, these systems are particularly vulnerable to energy loss and decoherence, a process in which quantum systems lose essential properties like entanglement and superposition, leading to reduced performance. In photonic systems that use ordinary (non-topological) waveguides -- channels that direct photons but are sensitive to bends or imperfections -- energy efficiency drops sharply as photons disperse within the guide. To address these persistent problems, the international research team used analytical and numerical modeling within a theoretical framework. By taking advantage of topological properties -- material features that remain unchanged even when the structure is twisted or bent -- they showed it is possible to achieve both long-distance energy transfer and immunity to dissipation in quantum batteries. In an unexpected twist, the researchers also discovered that dissipation, which typically weakens performance, can temporarily increase charging power under certain conditions. The study revealed several promising outcomes that bring topological quantum batteries closer to practical use. They also found that when dissipation surpasses a critical level, charging power experiences a brief but significant boost, overturning the long-held assumption that energy loss is always detrimental. "Our research provides new insights from a topological perspective and gives us hints toward the realization of high-performance micro-energy storage devices. Scientists Use “Supercentenarian Longevity Gene” to Slow Rapid Aging Disease New Drug Could “Un-Stiffen” Aging Hearts, Offering Hope for the Most Common Form of Heart Failure Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251023031621.htm'>MIT scientists discover hidden 3D genome loops that survive cell division</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 07:10:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Before a cell can split into two, it must first copy all of its chromosomes so each new cell inherits a complete set of genetic material. For years, researchers believed that as this process unfolded, the genome's intricate three-dimensional shape temporarily vanished. After division, scientists thought, the DNA would slowly rebuild its complex, folded structure, which is key to regulating which genes are active in a given cell. Now, new research from MIT reveals that this long-accepted model isn't quite right. Using a powerful, high-resolution genome mapping method, the team found that small 3D loops -- connections between regulatory DNA elements and genes -- remain intact even during cell division, a stage known as mitosis. "This study really helps to clarify how we should think about mitosis. The researchers also discovered that these DNA loops actually strengthen as chromosomes condense in preparation for division. This tightening brings distant regulatory elements closer together, encouraging them to stick. Many of these loops allow genes to interact with distant regulatory regions -- sometimes millions of base pairs apart -- while others form during mitosis to pack chromosomes tightly. Most of this mapping has relied on a tool called Hi-C, a technique co-developed by MIT researchers and Job Dekker at the University of Massachusetts Chan Medical School. Hi-C works by cutting DNA into small fragments and linking together those that are near each other in the cell nucleus before sequencing them to identify which regions interact. While effective, Hi-C lacks the resolution needed to detect fine-scale interactions between genes and regulatory sequences known as enhancers. Enhancers are short DNA segments that activate genes by binding to promoters, the regions where transcription begins. In 2023, Hansen and colleagues developed a next-generation technique that can map genome structures with up to 1,000 times greater precision. The method, called Region-Capture Micro-C (RC-MC), uses a different enzyme to cut DNA into evenly sized fragments and focuses on a smaller portion of the genome. This allows researchers to create highly detailed 3D maps of targeted DNA regions. Using RC-MC, the team identified a new structural feature they named "microcompartments." These are tiny, densely connected loops that form when nearby enhancers and promoters attach to one another. Earlier work had shown that microcompartments formed by a different mechanism than larger 3D genome structures, but the team didn't yet understand how. During this phase, chromosomes compact dramatically to ensure they can be duplicated and evenly distributed between daughter cells. "During mitosis, it has been thought that almost all gene transcription is shut off. It's a complete reset every cell cycle," Hansen says. In fact, they became even more pronounced as cells divided. Using their technique, the researchers also confirmed that larger structures such as A/B compartments and TADs do disappear during mitosis, as had been seen before. The authors reveal that, contrary to the well-described dramatic loss of TADs and compartmentalization during mitosis, fine-scale "microcompartments" -- nested interactions between active regulatory elements -- are maintained or even transiently strengthened," says Effie Apostolou, an associate professor of molecular biology in medicine at Weill Cornell Medicine, who was not involved in the study. Since the 1960s, scientists believed transcription stopped entirely during cell division. However, studies in 2016 and 2017 revealed a brief spike of gene activity before it shuts down again. In their new study, the MIT team found that during mitosis, microcompartments are more likely to be found near the genes that spike during cell division. Once formed, the loops that constitute microcompartments may activate gene transcription somewhat by accident, which is then shut off by the cell. When the cell finishes dividing, entering a state known as G1, many of these small loops become weaker or disappear. "It almost seems like this transcriptional spiking in mitosis is an undesirable accident that arises from generating a uniquely favorable environment for microcompartments to form during mitosis," Hansen says. "Then, the cell quickly prunes and filters many of those loops out when it enters G1." "We are thinking about some natural biological settings where cells change shape and size, and whether we can perhaps explain some 3D genome changes that previously lack an explanation," Hansen says. Scientists Use “Supercentenarian Longevity Gene” to Slow Rapid Aging Disease New Drug Could “Un-Stiffen” Aging Hearts, Offering Hope for the Most Common Form of Heart Failure Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/10/251023031618.htm'>How strong is your weed, really? Scientists say labels often mislead</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-24 03:57:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Nearly half of cannabis flower products sold in Colorado are inaccurately labeled for THC potency, with most overstating how strong they really are. In contrast, cannabis concentrates such as oils and waxes were found to be largely accurate, with 96% matching their listed THC content. These findings come from a large-scale analysis of cannabis products purchased from licensed dispensaries across Colorado, the first state to legalize recreational marijuana. It offers new insight into how potent today's cannabis has become, what testing and labeling practices are working, and where the system still falls short. "While that research plays out, we should, at the very least, be providing accurate information about the amount of THC in these products." Because federal law prevents university scientists from directly purchasing or handling cannabis from the legal market, industry collaboration was essential, explained first author Gregory Giordano, a professional research assistant in CU Boulder's Department of Psychology and Neuroscience. A MedPharm "secret shopper" traveled throughout the state to collect 277 cannabis products from 52 dispensaries across 19 counties. The samples, identified only by number, were then tested by MedPharm chemists who did not see the original labels to ensure objectivity. On average, cannabis flower products contained about 21% THC, or tetrahydrocannabinol, the main psychoactive compound in marijuana. "THC content has increased significantly, and we know that greater THC exposure is likely associated with greater risks, including risk of cannabis use disorder and some mental health issues," notes Bidwell. About 44% percent of flower products failed to meet that standard, with 54 of those products inflating their THC content on the label and 23 containing more THC than the label indicated. For those using cannabis medically, adequate dosing can be critical. For those using cannabis recreationally, taking more than expected can be dangerous. Some discrepancies were large -- one flower product was labeled as having 24% THC but had only 16%. But on average, the difference between labeled and observed THC was about 2%. "When it comes to concentrates, I would say Colorado gets a good grade for labeling accuracy, but there are some real issues with flower," said Bidwell. Previous research in other states has shown that third-party testing labs often inflate THC potency, possibly to gain the business of marketers wanting to attract consumers seeking stronger products. Small changes in testing protocols could reduce mislabeling, the researchers said. The study also looked at several other cannabinoids (active compounds found in the Cannabis sativa plant), including cannabidiol (CBD), cannabigerol (CBG), and cannabigerolic acid (CBGA). But only 16% of products featured any information at all about other lesser-known cannabinoids. "Our data suggests that multiple other cannabinoids should also be reported on there." With continued support from the Institute of Cannabis Research, Bidwell and Mackie plan to expand their work to include edible cannabis products. As more states legalize marijuana for both medical and recreational purposes, they hope their research will help guide better regulatory standards. Scientists Use “Supercentenarian Longevity Gene” to Slow Rapid Aging Disease New Drug Could “Un-Stiffen” Aging Hearts, Offering Hope for the Most Common Form of Heart Failure Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            