
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-16</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251015230952.htm'>From poison to power: How lead exposure helped shape human intelligence</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 14:32:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>What made the modern human brain so different from that of our extinct relatives, such as Neanderthals? Researchers at the University of California San Diego School of Medicine, along with an international team, have discovered that ancient hominids, including early humans and great apes, came into contact with lead far earlier than previously believed -- up to two million years before modern humans began mining it. This long-term exposure may have influenced how early brains evolved, possibly hindering language and social development in all but modern humans, who possess a unique protective genetic variant. The team examined fossilized teeth from 51 hominids found across Africa, Asia, and Europe. The samples included both modern and archaic humans such as Neanderthals, early human ancestors like Australopithecus africanus, and extinct great apes including Gigantopithecus blacki. It was previously thought that humans began facing significant lead exposure only in recorded history, especially during the Roman era, when lead pipes were used for water systems, and later during the Industrial Revolution. "We stopped using lead in our daily lives when we realized how toxic it is, but nobody had ever studied lead in prehistory," said corresponding author Alysson Muotri, Ph.D., professor of pediatrics and cellular & molecular medicine at UC San Diego School of Medicine, associate director of the Archealization Center, and director of the Sanford Integrated Space Stem Cell Orbital Research Center. The scientists suggest that ancient humans and their relatives might have encountered lead through their search for water, much like the Romans did later in history. "One possibility is that they were looking for caves with running water inside," Muotri said. Based on the tooth enamel studies, it started very early in infancy." Faced with this evidence, Muotri and his team began to question how modern humans managed to thrive despite such toxic conditions during their evolutionary past. Acting as a key regulator of neurodevelopment, NOVA1 helps determine how neural progenitor cells react to lead exposure, and disturbances in its activity are linked to neurological disorders. Nearly all modern humans carry a version of the NOVA1 gene that differs by a single DNA base pair from the version found in Neanderthals. Earlier work from Muotri's group showed that swapping the modern NOVA1 with the older variant in miniature brain models, called organoids, caused dramatic changes in brain structure and connectivity. The archaic variant accelerated brain maturation but resulted in less complexity over time. "If all humans have this newer mutation in all corners of the world, very strong genetic pressure must have selected for it in our species." They found that lead changed NOVA1 activity in both types of organoids, influencing genes linked to conditions such as autism and epilepsy. People with certain FOXP2 mutations struggle to form complex words and sentences. This could have given modern humans a significant evolutionary advantage over Neanderthals, even in the presence of lead contamination. Muotri believes these results have important implications for understanding how environmental stressors shaped brain development during human evolution. He speculates that lead exposure may have contributed to the extinction of Neanderthals around 40,000 years ago. Understanding how NOVA1 gene variants can affect FOXP2 expression helps elucidate the relationship between lead contamination and brain development and also sheds light on neurological conditions related to language, including speech apraxia -- a condition that makes it difficult to produce speech sounds correctly -- and autism. Disclosures: Muotri is the co-founder of and has an equity interest in TISMOO, a company specializing in genetic analysis and human brain organogenesis. The terms of this arrangement have been reviewed and approved by the University of California San Diego in accordance with its conflict-of-interest policies. Forget Fiber: The Surprising Foods That Actually Relieve Constipation, According to Science Memory Problems Have Nearly Doubled in Adults Under 40, Study Finds New Theory Suggests We've Been Looking for Dark Matter All Wrong Scientists Discover “Meal Memory” Neurons That Control When and How Much We Eat Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a69033254/amber-deposit-time-capsule/'>Scientists Found a 112-Million-Year-Old Time Capsule With Life From a Lost World</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Found in an Ecuadorian quarry, this amber preserves insects and plant material and provides an intimate look into life on the supercontinent Gondwana. We may earn commission if you buy from a link. Although Jurassic Park captured the scientific imagination of its audience, it didn't get everything right about dinosaurs. But when it comes to amber being an invaluable time capsule of Earth's dinosaur days, Jurassic Park scores a perfect 10. Now, a new study from the journal Communications Earth & Environment reinforces that fact. But it is the first to yield sufficient quantities ripe for paleontological study—it contained various arthropods, pollen, and plant material. While it's not exactly the dino DNA find portrayed in Jurassic Park, it still provides an unprecedented snapshot into life on the ancient supercontinent Gondwana, which included two-thirds of today's continental area. The rock surrounding these amber time capsules also contained a variety of plant fossils, including pollen and spores. The amber and fossils together paint a picture of a warm, humid Gondwana forest teeming with plant and animal life. The scientists hope that future fieldwork from this deposit will reveal connections among other Gondwana regions—including Antarctica, South Africa, Australia, and New Zealand, which also contain similar amber deposits—and more details about the Age of Dinosaurs. Scientists Go Deeper Into Mantle Than Ever Before Scientists Just Solved a Major Mystery About Hail The Amazon's Trees Are Getting Bigger Every Decade This Crystal Is a Map of Earth's Galactic Journey Scientists Found Evidence of Unknown Life in Rocks This “Battery In a Rock” Changes the Energy Game</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69036276/armenia-mystery-idol-face/'>A Mysterious Idol Just Emerged From the Earth. It May Show the Face of a Forgotten God.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>As archaeologists excavated the ruins of the fortified city of Argishtikhinili, built some 2,500 years ago on Surb Davti Blur Hill in Armenia, a lost civilization began to emerge. Then, as dust was cleared from ceramic vessels and other artifacts in a storage room, a strangely humanoid face carved from volcanic stone began to materialize. The identity of this unknown face is a mystery. It still stands just as it did thousands of years ago; at about 1.6 feet (half a meter) high, the face of this figure was sculpted with a long, slender nose between close-set eyes and a stoically narrow mouth. It seems to mirror similar sculptures that ancient Armenians built to honor ancestors or use for ritual purposes in fertility cults. Found next to the idol was a stone chest, one that may offer deeper insights into ritual practices. Subjecting the chest to chemical analysis could reveal organic residues of offerings, such as the remnants of wine and hallucinogens found in Egypt's Bes mugs, or other materials used for specific rituals. Figures discovered in Armenian sites from this time period have revealed that the spiritual roots of the Urartians ran deep, even in everyday life. What is known about their religion is they worshipped a pantheon of deities, some indigenous, some assimilated from recent conquests, many derived from earlier Mesopotamian and Anatolian beliefs. Along with the names of 79 gods literally set in stone were lists of the sacrifices that should be made to them. The most important of the Urartian gods were supreme deity Haldi, Teisheba, who ruled over storms and thunder, and Sun god Shivini, whose winged solar disk was likely borrowed from the Egyptian solar god Ra. While there are no surviving Urartian temples, Assyrian reliefs from the same time period offer a glimpse inside them, and fortresses were also associated with certain deities. Haldi was often engraved on the trappings of war, including weapons and shields, though no such artifacts were found at the Surb Davti Blur Hill site. Future investigations may finally show us the idol's real face. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. NASA Found a Secret U.S. Army Base Buried in Ice An Elusive Rat Finally Showed Its Face to Science</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03999-8'>MRI-based multi-organ clocks for healthy aging and disease assessment</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 09:27:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Biological aging clocks across organ systems and tissues have advanced understanding of human aging and disease. In this study, we expand this framework to develop seven magnetic resonance imaging-based multi-organ biological age gaps (MRIBAGs), including the brain, heart, liver, adipose tissue, spleen, kidney and pancreas. Using data from 313,645 individuals curated by the MULTI Consortium, we link the seven MRIBAGs to 2,923 plasma proteins, 327 metabolites and 6,477,810 common genetic variants. Genome-wide associations identify 53 MRIBAG−locus pairs (P < 5 × 10−8). Genetic correlation and Mendelian randomization analyses support organ-specific and cross-organ interconnection, including 24 non-MRI biological aging clocks and 525 disease endpoints. Through functional gene mapping and Bayesian co-localization multi-omics evidence, we prioritize nine druggable genes as targets for future anti-aging treatments. Furthermore, the seven MRIBAGs are linked to future risk of systemic disease endpoints (for example, diabetes mellitus) and all-cause mortality. Finally, participants with more youthful versus more aged brain profiles exhibited distinct cognitive decline trajectories over 240 weeks of treatment with the Alzheimer's disease drug solanezumab, although this heterogeneity cannot be fully attributed to the drug. In summary, we developed seven MRIBAGs that enhance the existing multi-organ biological aging framework, and we demonstrate their clinical potential to advance aging research. Magnetic resonance imaging (MRI1) provides a non-invasive window into structural and functional changes occurring with age, enabling the development of personalized disease biomarkers. Brain MRI-based aging clocks, often referred to as ‘brain age' and derived from artificial intelligence and machine learning (AI/ML) applied to MRI data, have been widely used as clinical biomarkers of neurological aging, cognitive decline and neurodegenerative disease risk2. However, although brain imaging has been extensively used for aging research, no studies have systematically extended this concept to other organ systems. With the advent of large-scale, multi-organ MRI datasets, such as those from the UK Biobank (UKBB), we now have an unprecedented opportunity to conduct a more holistic, system-wide investigation of biological aging across multiple organs. Beyond imaging-based aging clocks, researchers have also developed various omics-based aging clocks to capture different biological dimensions of aging. For example, plasma proteomics-based aging clocks3,4 have been introduced to enhance the granularity and coverage of multi-organ aging clocks. Plasma proteomics-based aging clocks capture the biological aging process by analyzing the abundance of proteins in circulation. Because proteins are the functional units of the cell, these clocks provide insights into the downstream effects of genetic regulation and environmental influences on aging. Metabolomics-based aging clocks5,6 capture biological aging by profiling small-molecule metabolites that represent the biochemical activity of cellular and systemic metabolism. Metabolomics examines the end products of metabolic pathways, making it highly dynamic and sensitive to external factors such as diet, microbiome composition, lifestyle and environmental exposures. Finally, epigenetics-based aging clocks7 focus on the analysis of DNA methylation patterns, particularly at specific CpG sites across the genome, to estimate biological age. Integrating these aging clocks into the multi-organ and multi-omics concept8,9,10 is essential for gaining a comprehensive understanding of aging biology, age-related diseases and longevity. In our previous efforts, we developed nine phenome-based biological age gaps (BAGs)11,12 (PhenoBAGs), 11 proteome-based BAGs (ProtBAGs13) and five metabolome-based BAGs (MetBAGs5). In the present study, we derived six additional MRIBAGs, on top of the brain MRIBAG14, for the heart, liver, spleen, adipose, kidney and pancreas using data consolidated from the MULTI Consortium (Method 1). We first benchmarked the age prediction performance of the six additional MRIBAGs (Method 2). Next, we associated the seven MRIBAGs with 2,923 plasma proteins for proteome-wide associations (ProWASs) and 327 metabolites for metabolome-wide associations (MetWASs) (Method 3). Through genome-wide association studies (GWASs) and post-GWAS analyses, we depicted their genetic architecture and identified potential druggable genes (Method 4). Finally, we assessed the clinical applicability and predictability of the MRIBAGs and their polygenic risk scores (PRSs) (Method 5). All results and pre-trained AI/ML models are publicly available at the MEDICINE portal: https://labs-laboratory.com/medicine/. The brain MRIBAG was initially developed in our previous study14 using 119 gray matter volumes, and we re-trained the brain MRIBAG here using a consistent nested cross-validation procedure along with a within-distribution, holdout test dataset (Extended Data Fig. Specifically, to rigorously evaluate the performance (that is, overfitting and generalizability) of biological age prediction models, we partitioned the healthy control (CN, without any pathologies) participants into the CN training/validation/test (3,573 < N < 6,327 as the sample sizes vary across organs) and the within-distribution, holdout test (N = 500) datasets (Method 2 and Supplementary Table 1). When fitting the organ-specific MRI features (Fig. 1a, Method 2 and Supplementary File 1), the two AI/ML models (least absolute shrinkage and selection operator (LASSO) regression and support vector regressor) exhibited slight variability in performance, with no single model consistently outperforming the others (Fig. The selected optimal models for the seven MRIBAGs demonstrated moderate Pearson's r coefficients (ranging from 0.23 to 0.77) and the mean absolute errors (MAEs) of approximately 5 years in the within-distribution, holdout test dataset (Fig. 1c) before applying the age bias correction15,16. Supplementary Table 2 presents detailed statistics for the age prediction tasks before the age bias correction (Supplementary Fig. 2 presents the incremental statistical power, beyond age and sex, of the seven MRIBAGs in predicting mortality and disease categories. Model improvements may be possible for abdominal features by employing more advanced modeling approaches, such as deep neural networks that directly process voxel-wise MRI data17. Additionally, we found that high collinearity among imaging features from abdominal MRI led to poor generalizability in within-distribution, holdout test datasets (Supplementary Fig. 4 shows potential domain shift while applying the pre-trained model of the brain MRIBAGs to external datasets, exemplified by the A4 study18,19. 5 presents a sensitivity analysis for the Alzheimerʼs disease drug differential responses. 6 discusses sex differences in the seven MRIBAGs. 7 and 8 discuss the feature importance of deriving these MRIBAGs and their biological interpretation. For all subsequent analyses, we used the age bias-corrected16 MRIBAGs. a, The seven MRIBAGs are derived from various imaging modalities available in the UKBB. The number (N) of IDPs used to derive the seven MRIBAGs is displayed. b, For the seven MRIBAGs, the MAE of age prediction models using LASSO regression and linear support vector regressor (SVR) is shown for both the training dataset (cross-validated training/validation/test; 3,582 < N < 6,325) and the within-distribution, holdout test dataset (N = 500). Cohen's d quantifies the effect size of the difference between these datasets, reflecting potential model generalizability, assuming similar age and sex distributions. The optimal model (#) for each organ and tissue was selected based on the lower Cohen's d value, and these models were used for all subsequent analyses. All MAEs are reported without age bias correction. c, Scatter plots display the optimal model for each organ/tissue in the within-distribution, holdout test dataset, with Pearson's r and two-sided P values indicating the association between chronological age and predicted age. All results are shown before the correction for age bias was applied16. To evaluate the generalizability of the brain MRIBAG model, we validated it using two independent studies. In b, individual data points are overlaid on box plots summarizing the distribution. In our ProWAS analyses (Method 3), we identified 603 protein−MRIBAG significant associations (P < 0.05/2,923/7). Among these, the kidney MRIBAG exhibited the highest number of significant protein associations (N = 301; for example, NPDC1, IGFBP6 and TAFA5), followed by the spleen MRIBAG for 136 associations (for example, VCAM1, PTPRH and C1QA), the liver MRIBAG for 62 associations (for example, NCAN, SEZ6L and LEP), the adipose MRIBAG for 57 signals (for example, GDF15, CHI3L1 and CA14), the pancreas MRIBAG for 21 associations (for example, PLA2G1B, CTRC and CELA2A), the brain MRIBAG for 16 associations (for example, BCAN, NCAN and GDF15) and the heart MRIBAG for only REN (Fig. a, ProWAS between the seven MRIBAGs and 2,923 plasma proteins via a linear regression model, accounting for a full set of covariates (P < 0.05/2,923/7). An online interactive webpage is available at https://labs-laboratory.com/medicine/mribag_prowas.html to ease visualization. Plots of individual organs are shown in Extended Data Fig. 2. b, MetWAS between the seven MRIBAGs and 327 plasma metabolites using a linear regression model while adjusting for a comprehensive set of covariates (P < 0.05/327/7). Plots of individual organs are shown in Extended Data Fig. An online interactive webpage is available at https://labs-laboratory.com/medicine/mribag_metwas.html to ease visualization. VCAM1 was associated with the spleen MRIBAG (β = −0.89 ± 0.04; P = 7.05 × 10−81; r = −0.38); PLA2G1B was associated with the pancreas MRIBAG (β = −1.14 ± 0.07; P = 8.26 × 10−56; r = −0.30) and the adipose and spleen MRIBAGs. The two proteins are organ enriched in respective organ and tissue, as identified by the Human Protein Atlas (HPA; https://www.proteinatlas.org/), emphasizing their relevance to organ-specific aging processes, which has been widely used to define organ-enriched proteins in previous proteome-based aging clock papers4,13. Supplementary File 2a presents detailed statistics. 9 and Supplementary File 2b explore the potential replication of UKBB Olink ProWAS signals in Baltimore Longitudinal Study of Aging (BLSA) SomaScan data. 2 and 3 present individual volcano plots and protein set enrichment results via the STRING platform (version 12.0 (ref. 20)) for each MRIBAG (Supplementary Figs. 15 presents the sex-specific ProWAS results. In our MetWAS (Method 3), we identified 758 metabolite−MRIBAG significant associations (P < 0.05/327/7) between the seven MRIBAGs and metabolites, showing relatively weak associations (Pearson's r < 0.3). The spleen MRIBAG showed the highest number of significant metabolite associations (N = 199; for example, acetate, acetoacetate and albumin), followed by the adipose MRIBAG (N = 196; for example, L_HDL_PL_pct, L_HDL_C_pct and PUFA_pct), the heart MRIBAG (N = 139; for example, PUFA_pct, M_HDL_TG and HDL_TG), the brain MRIBAG (N = 97; for example, LA_pct, Omega_6_pct and ApoA1), the pancreas MRIBAG (N = 57; for example, GlycA, Glucose_Lactate and HDL_PL_pct), the liver MRIBAG (N = 51; for example, Gly, Tyr and L_LDL_PL_pct) and the kidney MRIBAG (N = 19; for example, creatinine, citrate and DHA) (Fig. Descriptions of these metabolites are provided in Supplementary Table 3. Creatinine was linked to the kidney (β = 21.77 ± 0.65; P = 1.58 × 10−258; r = 0.28) and adipose (β = −4.37 ± 0.83; P = 1.48×10 −7; r = −0.18) MRIBAGs, which may suggest distinct roles of the kidney and adipose tissue in creatinine regulation. In addition to these small-molecule metabolites, other lipid complexes and subclasses also showed significant associations. Total concentration of phospholipids (Total_P) was negatively associated with liver (β = 51.57 ± 3.59; P= 2.52 × 10−46; r = 0.18). Phospholipids in small HDL (S_HDL_PL) were linked to the brain, heart, spleen and adipose MRIBAGs. Similarly, cholesteryl esters in very large HDL (XL_HDL_CE) associated with the adipose and heart MRIBAGs. Supplementary File 3 presents detailed statistics. 4 presents individual volcano plots for each MRIBAG. Metabolite set enrichment analysis is presented in Extended Data Fig. 6 presents example associations for the ProWAS and MetWAS. 17 presents the sex-specific MetWAS results. We conducted GWAS (Method 4) for the seven MRIBAGs (19,686 < N < 31,557 participants with European ancestries) and identified 53 (P < 5 × 10−8) genomic locus−BAG pairs. We denoted the genomic loci using their top lead single-nucleotide polymorphisms (SNPs) defined by FUMA, considering linkage disequilibrium; the genomic loci are presented in Supplementary Table 4. We visually present the shared genomic loci annotated by cytogenetic regions based on the GRCh37 cytoband (Fig. 7 detail the robustness of our GWASs and sex-specific GWAS analyses. a, Cytogenetic regions associated with the seven MRIBAGs, with significant genomic loci identified using the genome-wide significance threshold (two-sided P < 5 × 10−8). b, SNP-based heritability estimates of the seven MRIBAGs (sample size: 18,778 < N < 31,557). c, Partitioned heritability enrichment analysis using chromatin-specific multi-tissue chromatin data, multi-tissue gene expression profiles and cell-type-specific datasets. Only significant results (two-sided P < 0.05/697) are displayed (sample size: 18,778 < N < 31,557). d, Genetic correlation estimates via LDSC between the seven MRIBAGs and eight PhenoBAGs (excluding the multimodal brain PhenoBAG11), 11 ProtBAGs13 and five MetBAGs5 (two-sided P < 0.05/11) as well as 525 disease endpoints from FinnGen and PGC (P < 0.05/525). e, Potential causal relationships between the seven MRIBAGs (that is, number of instrumental variables > 7) and 525 disease endpoints were examined through two networks: BAG2DE, where the seven MRIBAGs serve as exposure variables and the 525 DEs as outcome variables, with two-sided P < 0.05/525, and DE2BAG, where the 214 disease endpoints serve as effective exposure variables (that is, number of instrumental variables > 7), with two-sided P < 0.05/214. Although multiple sensitivity checks were performed to evaluate potential violations of underlying assumptions, these findings should be interpreted with caution. Bar plots represent the mean (center of bar), with error bars indicating ±1 s.e. We estimated the SNP-based heritability (\({h}_{{SNP}}^{2}\)) for the seven MRIBAGs using GCTA21 software (Method 4), with values ranging from 0.29 to 0.47 (Fig. Supplementary Table 5 presents detailed statistics. We validated the GWAS signals through partitioned heritability analyses (Method 4) using LDSC software, revealing strong organ-specific enrichment. Notably, the heart MRIBAG exhibited chromatin state-specific enrichment across four distinct heart tissues (Fig. For example, significant heritability enrichment was shown in the right atrium in the H3K4me3 region (P = 1.00 × 10−6) and in the left ventricle in the H3K27ac region (P = 2.00 × 10−5) for the heart MRIBAG (Supplementary File 4). We then assessed the genetic correlations (Method 4) between the seven MRIBAGs and 24 previously developed multi-organ, multi-omics aging clocks, including eight PhenoBAGs11 (excluding the brain PhenoBAG based on multimodal imaging), 11 ProtBAGs13 and five MetBAGs5. After Bonferroni correction, we found seven within-organ and inter-organ significant genetic correlations (P < 0.05/11), exemplified between the kidney MRIBAG and the renal PhenoBAG (gc = 0.23 ± 0.05; P < 2.00 × 10−5) and between the spleen MRIBAG and the hepatic MetBAG (gc = 0.23 ± 0.08; P < 7.00 × 10−4). We also performed genetic correlation between the seven MRIBAGs and 525 disease endpoints and found six significant signals (P < 0.05/525). For example, within-organ connections were demonstrated between the heart MRIBAG and two forms of hypertension, which is a major risk factor for cardiovascular diseases (for example, FinnGen code: I9_HYPTENS: gc = 0.28 ± 0.06; P < 1.00 × 10−5); cross-organ interactions were also identified between the adipose MRIBAG and substance abuse (gc = 0.27 ± 0.06; P < 5.00 × 10−5) (Fig. We conducted bidirectional Mendelian randomization analyses (Method 4) between the seven MRIBAGs and 525 disease endpoints, ensuring at least seven valid linkage disequilibrium-considered SNPs as instrumental variables after quality control. Although the MRIBAGs were generally underpowered to meet this threshold, we identified six potential causal relationships. The within-organ interactions between the heart MRIBAG and hypertension were reaffirmed by a potential causal relationship from various forms of hypertension to the heart MRIBAG (for example, I9_HYPTENS: P = 2.12 × 10−7; odds ratio (95% confidence interval) = 1.13 (1.08−1.18); number of instrumental variables = 110). Additionally, we identified a potential causal relationship from Alzheimer's disease to the liver MRIBAG (G6_AD_WIDE: P = 3.02 × 10−5; odds ratio (95% confidence interval) = 0.91 (0.88−0.96); number of instrumental variables = 8), consistent with our previous findings linking Alzheimer's disease to the hepatic PhenoBAG11. Finally, we found that the kidney MRIBAG was negatively linked to two forms of type 2 diabetes mellitus (T2D) (for example, T2D: P = 8.40 × 10−5; odds ratio (95% confidence interval) = 0.78 (0.70−0.88); number of instrumental variables = 9) (Fig. Overall, these PRSs exhibit limited predictive power. We conducted multiple post-GWAS analyses (Method 4) to identify potential druggable genes associated with the seven MRIBAGs. Our overarching hypothesis posits that these imaging-derived aging clocks, as AI-driven endophenotypes22 (see Supplementary Fig. 26 for a conceptual visualization), reside along the causal pathway of human aging and disease, linking genetic predisposition to multi-omics data, such as gene expression, proteomics and metabolomics, leading to disease outcomes and cognitive decline. Initially, we mapped independent GWAS signals to genes using three approaches—(1) positional mapping, (2) expression quantitative trait locus (eQTL) mapping and (3) chromatin interaction mapping—identifying a total of 1,164 unique genes. To further refine these genes, we prioritized genes that demonstrated genetic evidence from both eQTL mapping and chromatin interactions, resulting in a final set of 246 unique genes linked to the seven MRIBAGs. For instance, MAPT was prioritized in the brain MRIBAG through both eQTL and chromatin interaction mapping and was previously identified as a genetically supported druggable gene23. Subsequently, to further extend genetic evidence to other omics data types, we conducted Bayesian co-localization analysis on genomic loci associated with the seven MRIBAGs and aging clocks derived from plasma proteomics13 (ProtBAGs) and metabolomics (MetBAGs5). The key hypothesis is that a causal variant positionally or functionally linked to a gene influencing both MRIBAGs and other omics-based aging clocks is more likely to play a causal role in human aging and disease. By integrating these findings with the 246 prioritized genes, we identified 62 MRIBAG–gene pairs (comprising 62 unique genes) that share at least one causal SNP with either MetBAGs or ProtBAGs (Fig. a, To prioritize potential druggable genes, leveraging aging clocks derived from plasma proteomics (ProtBAGs) and metabolomics (MetBAGs), we filtered and mapped genes to retain only those showing significant co-localization with any ProtBAG or MetBAG. As an example, we highlighted a specific genomic locus demonstrating co-localization among the spleen MRIBAG, the immune MetBAG and the pulmonary ProtBAG. The shared causal variant (rs233721 on chromosome 12) exhibited a strong co-localization signal, with a posterior probability (PP.H4.ABF) of 0.99 between the spleen MRIBAG and the immune MetBAG, implying a single shared causal variant influencing both traits within this locus. To determine the significance of the H4 hypothesis, we set a threshold of PP.H4.ABF > 0.8. This variant was mapped to the TRAFD1 and ALDH2 genes through both eQTL mapping (for example, ALDH2 in the aorta from GTEx) and chromatin interaction mapping (for example, TRAFD1 in the left ventricle and liver). b, By integrating this multi-level genetic evidence, we identified 62 unique genes associated with the liver, spleen and pancreas MRIBAGs. Among the 62 genes with multi-level genetic evidence supporting their potential causal role in human aging and disease, we further explored existing drug−gene interaction data to identify potential drug repurposing opportunities and evidence. To achieve this, we queried the Drug Gene Interaction database (DGIdb) (https://dgidb.org/; query date: 21 March 2025) and identified nine genes (linked to the liver, spleen and pancreas MRIBAGs) with known interactions involving 122 unique drugs (Fig. Among the 122 unique drugs, many indications are manifested, including immunosuppressant agents (for example, everolimus; DrugBank ID: DB01590), antipsychotic agents (for example, loxapine for the management of the manifestations of psychotic disorders such as schizophrenia; DrugBank ID: DB00408), antidiabetic agents (for example, bromocriptine for treating T2D; DrugBank ID: DB01200) and antiparkinson agents (lisuride for the management of Parkinson's disease; DrugBank ID: DB00589), as well as many antineoplastic agents. a, We queried the 62 prioritized MRIBAG genes in the DGIdb platform (https://dgidb.org/) to identify drug−gene interactions and potential drug repurposing opportunities. We show only the nine MRIBAG genes linked to existing drugs, whether approved or unapproved, using curated data via the DGIdb. For visualization, we annotated representative drug names; the complete results can be found at an online interactive webpage at https://labs-laboratory.com/medicine/mribag_gdi.html. b, Drug indications for existing drug−gene interactions related to the spleen MRIBAG-linked ALDH2 gene. We highlighted the spleen MRIBAG-linked gene ALDH2 as exemplified in our co-localization analyses (Fig. Multi-level genetic evidence links the spleen BAG, ALDH2 and 15 drugs (loxapine, clozapine, rotigotine, sulpiride, prochlorperazine, terguride, sertindole, fenoldopam, pergolide, trifluoperazine, naltrexone, cabergoline, apomorphine, lisuride and bromocriptine) with multiple clinical indications. These indications include antipsychotic (for example, clozapine for treating severe forms of psychiatric conditions such as schizophrenia and for preventing suicide in patients who have this condition); antiparkinson (for example, rotigotine); antidepressive (for example, sulpiride, used to treat acute and chronic schizophrenia and major depressive disorder but potentially toxic with a high dose, with side effects for depression); antimigraine (for example, prochlorperazine, used to treat psychotic symptoms such as schizophrenia and non-psychotic anxiety but has off-label uses, including emergency use to treat migraine in adults and children); antihypertensive (for example, fenoldopam, a medication used to temporarily treat high blood pressure); antiemetics (for example, trifluoperazine, a medication used as an antipsychotic and an antiemetic); appetite suppressant; treatment of erectile dysfunction; antidiabetic; treatment of pulmonary arterial hypertension; treatment of substance abuse; analgesic; and treatment of sexual dysfunction in women (Fig. The 15 drugs linked to ALDH2 encompass a broad range of therapeutic areas, many of which are relevant to aging-related diseases such as neurodegeneration, metabolic disorders and immune system aging. As a central immune organ, the spleen's aging may indicate immune dysfunction that contributes to neuroinflammation. ALDH2 could influence both immune cell activity in the spleen and neuroinflammation in the brain, especially concerning drugs for neurodegenerative and psychiatric disorders. Drugs targeting ALDH2 (or related pathways) for neurodegenerative diseases may also affect systemic aging, positioning them as potential candidates for modulating spleen or immune aging as well (Supplementary Fig. We highlight the clinical potential of the seven MRIBAGs and their PRSs in predicting (1) the incidence of 53 individual disease conditions (with at least 50 patients) using International Classification of Diseases, Tenth Revision (ICD-10) codes; (2) all-cause mortality risk through survival analyses; and (3) differentiation in cognitive decline trajectories in Alzheimer's disease drug solanezumab (Method 5). We found that the brain, adipose and pancreas MRIBAGs and the pancreas PRS could significantly (P< 0.05/7) predict non-insulin-dependent diabetes mellitus (E119; 1.44 < hazard ratio < 1.70). The brain MRIBAG also predicted personal history of psychoactive substance abuse (Z864; hazard ratio = 1.27 (1.11−1.46); P = 6.70 × 10−4), anxiety disorders (F419; hazard ratio = 1.60 (1.20−2.11); P = 1.31 × 10−3) and gastrointestinal hemorrhage (K922; hazard ratio = 1.46 (1.11−1.91); P = 6.63 × 10−3). The heart MRIBAG was associated with an increased risk of hypertension (I10; hazard ratio = 1.16 (1.05−1.28); P = 4.80 × 10−3), reinforcing findings from genetic correlation and Mendelian randomization analyses (Fig. a, The seven MRIBAGs and their PRSs show significant associations with the incidence of single disease entities (for example, I10 for hypertension). We only included disease entities with more than 50 cases (two-sided P < 0.05/7 = 0.0071 corrected for the number of organ systems), and non-cases are participants who do not show any disease diagnosis (disease free) after enrolling in the UKBB, so that all non-case populations remained the same reference population across all tasks. Age and sex were included as covariates in the Cox proportional hazard model. An asterisk (*) denotes results that remain significant after applying the Bonferroni correction for multiple comparisons (two-sided P < 0.05/53 = 0.00094, accounting for the number of disease endpoints), whereas a hash symbol (#) indicates significance under a more stringent threshold (two-sided P < 0.05/53/7 = 0.00013). b, The brain, adipose, spleen and liver MRIBAGs showed significant associations with all-cause mortality risk. To further validate the negative associations of the spleen and liver MRIBAGs with mortality, we conducted a disease-free analysis, including only participants without any prior disease diagnoses, to rule out potential confounders. c, At week 240, PACC scores differed between the accelerated and decelerated aging groups—stratified by brain MRIBAG expression—among participants originally assigned to the drug group in the solanezumab trial. d, At week 240, PACC scores differed between the accelerated and decelerated aging groups among participants originally assigned to the placebo group in the solanezumab trial. e, At week 240 of the solanezumab trial, PACC scores in the accelerated aging group showed no difference between participants receiving the drug and those receiving a placebo. f, At week 240 of the solanezumab trial, PACC scores in the decelerated aging group showed no difference between participants receiving the drug and those receiving a placebo. Supplementary File 9 provides all statistical details, including sample size, P values, hazard ratios (HRs) and confidence intervals (CIs). For a and b, each dot or shape represents the estimated HR for a given disease or mortality. For c–f, all P values are two-sided, and the corresponding t values are reported for signals with P < 0.05. We further found that several MRIBAGs significantly (P < 0.05/7) predicted all-cause mortality (Fig. Among these, the brain and adipose MRIBAGs were identified as risk biomarkers (hazard ratio > 1) for all-cause mortality, whereas the liver and spleen MRIBAGs (hazard ratio < 1) appeared to have a protective effect. We conducted additional sensitivity analyses by excluding participants with any disease diagnoses to assess disease-free survival. For instance, the spleen MRIBAG retained nominal significance under this approach (P = 0.04; hazard ratio = 0.62 (0.40−0.99)). 28 and 29 present a detailed discussion on additional sensitivity check results. Finally, we assessed the use of the brain MRIBAG for stratifying clinical trial participants into subgroups to (1) demonstrate differential cognitive decline trajectories across subgroups and (2) provide insights for future clinical trial design, using data from the A4 trial. 6c,d, participants exhibiting decelerated aging signatures (that is, youthful brain: MRIBAG below the median) showed a higher Preclinical Alzheimer Cognitive Composite (PACC) score than participants within the accelerated aging group (that is, aged brain), both in the drug group (P = 0.003 and t = 2.923) and the placebo group (P = 0.001 and t = 4.049). A permutation test showed that the cognitive differences between accelerated and decelerated agers, quantified by t values, were significantly more pronounced in the placebo group compared to the drug group (one-sided P = 0.03 based on 1,000 permutations). A one-sided P value was used, as our hypothesis was directional; we specifically aimed to test whether the placebo group exhibited stronger cognitive separation between aging subgroups, consistent with the observed pattern in cognitive trajectories (Fig. Furthermore, neither the decelerated nor the accelerated aging group exhibited significant drug effects (Fig. 6e,f) in preventing cognitive decline. This study introduces seven MRI-based aging clocks by leveraging large-scale multi-organ and multi-omics data from 313,645 participants curated by the MULTI Consortium. Collectively, this work expands existing biological aging clocks and highlights their clinical potential for future research. Overall, the age prediction models highlight limitations in relying on abdominal MRI features, including a limited number of informative features and substantial multicollinearity, which reduced model performance and hindered generalizability to unseen data. Compared to the brain imaging community24, the development of imaging protocols and standardized imaging-derived phenotypes (IDPs) for other organs may be relatively underdeveloped. This disparity is reflected in large-scale biobanks such as the UKBB, where the majority of well-characterized IDPs are derived from brain MRI (N > 4,000) and heart MRI, whereas imaging-derived features, returned and made available to the community, for other organs are limited in both number and diversity. Several factors contribute to this imbalance, including the historical focus of neuroimaging research, the availability of standardized brain imaging pipelines25 and the widespread adoption of advanced computational tools tailored for brain MRI analysis. Imaging data from other organs should also be incorporated into this framework. For instance, integrating retina imaging26 with brain MRI, genetic data and systemic biomarkers could reveal mechanisms underlying aging and disease. The ProWAS results highlight both organ-specific aging mechanisms and systemic inter-organ communication, providing a foundation for understanding multi-tissue aging dynamics. Such organ specificity of the brain MRIBAG was also reinforced by two previous studies that used plasma proteomics data to develop brain tissue-enriched aging clocks (that is, the brain ProtBAG) using both SomaScan and Olink platforms. Oh et al.4 identified the three brain tissue-enriched proteins based on Genotype-Tissue Expression (GTEx) data to construct the brain ProtBAG, whereas Wen et al.13 leveraged the HPA platform, which integrates more extensive curation at both RNA and protein levels, to define brain tissue-specific proteins. Other proteins include CHI3L1, a well-established neuroinflammatory marker elevated in Alzheimer's disease and multiple sclerosis27, and GDF15, a stress-induced cytokine linked to cognitive decline, Alzheimer's disease and aging28,29. The significant metabolites identified in the MetWAS analysis further reinforce both cross-organ and within-organ associations. Although certain proteins or metabolites, such as GDF15 and FABP4, serve as systemic aging markers, others, such as NCAN and IGFBP6, exhibit tissue-specific roles. Shared underlying biological pathways may include inflammation30, extracellular matrix dysregulation31 and metabolic dysfunction32. The results of our genetic analyses reveal that, although the MRIBAGs share certain genetic underpinnings, each organ also demonstrates a distinct genetic architecture shaped by its unique biological processes. Genetic correlations between the MRIBAGs and other multi-omics aging clocks, as well as the MRIBAGs' associations with various disease endpoints, strengthen the notion of both shared and organ-specific genetic influences. For example, the correlation between the kidney MRIBAG and the renal PhenoBAG highlights how kidney-specific genetic factors can interact with broader multi-organ aging mechanisms at multiple omics levels33. Notably, although our analyses show that these organs share common genetic influences that link them to broader disease pathways, they also exhibit unique, organ-specific contributions that reflect their individual physiological roles and functions. Together, these findings provide strong support for the hypothesis that the MRIBAGs share genetic influences across organs while also demonstrating distinct genetic signatures that contribute to the specificity of each organ's role in aging and disease. 30 and Supplementary Table 9 performed mediation analyses to link the MRIBAGs to ProtBAGs and late-life depression34. The broad spectrum of drug indications linked to the prioritized genes highlights the opportunity to repurpose existing therapies for targeting aging and age-related diseases in different organs, providing a promising direction for future therapeutic advancements. For example, everolimus, a rapamycin analog, has shown promise in extending lifespan and improving lifespan in humans35,36, particularly in delaying age-related diseases and improving immune function. It is known for its ability to suppress immune system activity, which can reduce chronic inflammation, a hallmark of aging. Studies have suggested that chronic low-grade inflammation, often referred to as ‘inflammaging'37, contributes to various age-related diseases, such as cardiovascular disease38 and Alzheimer's disease39. Immunosuppressants may mitigate this process by modulating immune responses and promoting tissue repair and regeneration. Notably, the mTOR pathway35, targeted by drugs such as everolimus, has been shown to regulate cellular processes such as growth, survival and metabolism, and its inhibition has been linked to lifespan extension in model organisms, including yeast, worms and mice40. The counterintuitive finding that higher values of the liver and spleen MRIBAGs are associated with a lower risk of mortality could reflect their role in preserving organ function and overall physiological balance as people age. Potentially, the higher MRIBAG values likely reflect healthier, more resilient organs that maintain their metabolic and immune functions more effectively over time. Specifically, a higher liver MRIBAG may signal better metabolic health, efficient detoxification and regulation of systemic inflammation, which are crucial for maintaining overall homeostasis. Similarly, a higher spleen MRIBAG could indicate better immune function, including more effective immune surveillance and response to infections and other stressors. In the context of aging, these organs might act as ‘reserve systems'41, buffering the body against environmental insults and chronic diseases, which would reduce the risk of mortality. This study has several limitations. First, the analysis is based on cross-sectional data. Longitudinal studies will be crucial to better investigate the dynamic trajectories of MRIBAGs over time and their temporal relationships with health outcomes. Second, although the analysis focused primarily on genetic and imaging data, environmental factors such as lifestyle, diet and pollution were not considered, yet these may play important roles in aging and disease42. Third, the imaging features for the five abdominal organs and tissues were relatively underdeveloped, underscoring the need to implement more advanced feature engineering techniques and include external independent studies. Furthermore, the genetic analysis was focused primarily on individuals of European ancestry, and future studies should extend these findings to underrepresented populations to enhance their generalizability. Future refinement through larger, multi-ethnic GWASs and the inclusion of rare variants may improve their utility and biological interpretability. In addition, the current results derived from the UKBB study would benefit from confirmation in other large-scale cohorts to ensure their generalizability across populations and study designs. Moreover, as a future avenue, studies incorporating more Alzheimer's disease-oriented and AI-derived imaging biomarkers (for example, subtypes informed by tau or amyloid positron emission tomography (PET) data) may enhance statistical power and clinical relevance to benefit future clinical trials. Finally, the observed protective associations for the liver and spleen MRIBAGs are biologically ambiguous and complex. As such, these findings should be interpreted cautiously. The MULTI Consortium is an ongoing initiative to integrate and consolidate multi-organ data and multi-omics data, including imaging, genetics and proteomics. Building upon existing consortia and studies, such as those listed below, MULTI aims to curate and harmonize the data to model human aging and disease across the lifespan. In the present study, in total, individual-level data for 313,645 participants were analyzed, including multi-organ MRI data across seven organs and tissues (Category ID: 100003), genetics, plasma proteomics data and metabolomics data. GWAS summary statistics from FinnGen and the Psychiatric Genomics Consortium (PGC) were downloaded and harmonized for our post-GWAS analyses. Refer to Supplementary Table 1 for comprehensive information, including the complete list of data analyzed and their respective sample characteristics. The MULTI Consortium received approval from the institutional review board at Columbia University (AAAV6751). Participants provided written informed consent for each study that included individual-level data, whereas FinnGen and PGC provided only summary-level statistics. Specifically, the UKBB obtained research ethics approval and received guidance from an Ethics Advisory Committee, with additional ethical input provided to the Access Committee, which oversees decisions regarding access to data and biological samples. The BLSA protocol was approved by the institutional review board of the National Institute of Environmental Health Science, National Institutes of Health (03AG0325). For the A4 trial, institutional review board approval was obtained at each trial site, and all participants provided written informed consent. The study was conducted in accordance with International Council for Harmonization Good Clinical Practice guidelines. The UKBB43 is a population-based research initiative comprising approximtely 500,000 individuals from the United Kingdom between 2006 and 2010. Ethical approval for the UKBB study was secured, and information about the ethics committee can be found here: https://www.ukbiobank.ac.uk/learn-more-about-uk-biobank/governance/ethics-advisory-committee. This study collectively analyzed multi-organ MRI data from seven organ systems and tissues, including the brain, heart, liver, pancreas, spleen, adipose and kidney. For the genetic data, we conducted a quality check on the imputed genotype data43 for the entire UKBB population (approximately 500,000 individuals). Subsequently, we merged the processed data with the organ-specific populations for all genetic analyses. Finally, we also included Olink plasma proteomics data released by the UK Biobank Pharma Proteomics Project (UKB-PPP) and metabolomics data from the UKBB, which are detailed in the following sections. Tracking physiological and cognitive changes over time aims to identify risk factors for age-related diseases, study patterns of decline and discover predictors of healthy aging. BLSA44,45 brain MRI and SomaScan proteomics data (https://www.blsa.nih.gov/) were used to compare and replicate the ProWAS results from the UKBB Olink data. After quality checks in this study, we included 1,114 brain MRI scans at baseline and measurements of 7,268 plasma proteins from 909 participants quantified with the SomaScan version 4.1 platform. Age (years), sex (male/female), race (white/non-white) and education level (years) were defined based on participant self-reports. The A4 study18,19 (https://atri.usc.edu/study/a4-study/) is a clinical trial study to test a specific way to prevent memory loss associated with Alzheimer's disease (clinical trial number: NCT02008357). The A4 study focused on symptom-free adults at higher risk for Alzheimer's disease to assess whether an investigational drug (that is, solanezumab) could slow memory decline linked to amyloid plaques in the brain. It also examined whether solanezumab could delay Alzheimer's disease progression, measuring related brain changes using imaging, blood biomarkers and baseline PET scans to assess amyloid levels. This study analyzed 1,055 participants at baseline with brain MRI scans to derive the brain MRIBAG. Longitudinal outcomes from the clinical trial, with the PACC score as the primary measure over 312 weeks, were included. The PACC scores between groups were evaluated at week 240. The FinnGen46 study is a large-scale genomics initiative that has analyzed more than 500,000 Finnish biobank samples and correlated genetic variation with health data to understand disease mechanisms and predispositions. The project is a collaboration between research organizations and biobanks within Finland and international industry partners. For the benefit of research, FinnGen generously made their GWAS findings accessible to the wider scientific community (https://www.finngen.fi/en/access_results). This research used the publicly released GWAS summary statistics (version R9), which became available on 11 May 2022, after harmonization by the consortium. No individual data were used in the current study. FinnGen published the R9 version of GWAS summary statistics via REGENIE software (version 2.2.4)47, covering 2,272 disease endpoints, including 2,269 binary traits and three quantitative traits. Genotype imputation was referenced on the population-specific SISu version 4.0 panel. We included GWAS summary statistics for 521 FinnGen disease endpoints in our analyses. PGC48 is an international collaboration of researchers studying the genetic basis of psychiatric disorders. PGC aims to identify and understand the genetic factors contributing to various psychiatric disorders, such as schizophrenia, bipolar disorder, major depressive disorder and others. The GWAS summary statistics were acquired from the PGC website (https://pgc.unc.edu/for-researchers/download-results/), underwent quality checks and were harmonized to ensure seamless integration into our analysis. No individual data were used from PGC. Each study detailed its specific GWAS models and methodologies, and the consortium consolidated the release of GWAS summary statistics derived from individual studies. In the present study, we included summary data for four brain diseases for which allele frequencies were present. In our previous analysis of raw brain MRI data from the UKBB and other studies included in this work, we extracted 119 gray matter regions of interest from T1-weighted MRI to generate brain MRIBAGs through the iSTAGING consortium49. For the heart MRIBAG, we used 80 heart MRI traits from Bai et al.50 and used these imaging features in a previous study9. To develop the remaining five MRIBAGs based on abdominal MRI (Category ID: 105 (refs. 51,52,53,54,55,56)), we incorporated 16 imaging features for the adipose MRIBAG, four for the liver MRIBAG, three for the kidney MRIBAG, three for the spleen MRIBAG and three for the pancreas MRIBAG. Notably, in the development of adipose and kidney MRIBAGs, we observed a high degree of collinearity among certain imaging features, which led to overfitting. To mitigate this issue, we removed highly correlated features to improve model robustness. The abdominal MRI data underwent initial quality control procedures prior to public release, including the removal of low-quality images and biologically implausible values. As noted on the UKBB website (https://biobank.ndph.ox.ac.uk/ukb/label.cgi?id=158), different pipelines were used to generate imaging features from abdominal MRI, and data should not be combined without careful consideration. To account for this, we ensured that each abdominal feature was derived from a consistent pipeline, selecting the one with the largest available sample size across different pipelines. For liver MRI metrics specifically, we included only images acquired using the IDEAL protocol (Data-Field 40063, acquisition protocol 2). Following these additional quality control steps, our analyses incorporated abdominal MRI biomarkers from all available UKBB participants for the liver, kidney, spleen, pancreas and adipose MRIBAGs. Our previous study systematically evaluated age prediction performance across various AI/ML models using multimodal brain MRI features14 for the brain MRIBAG. Applying the same framework, we assessed the performance of models in deriving the six additional MRIBAGs using several ML methods. Hyperparameter tuning was performed through nested, repeated holdout cross-validation57 with 50 repetitions (80% training/validation and 20% testing). Specifically, the hyperparameters of each model were tuned using grid search: LASSO regression's alpha (α), linear support vector regressor's C and both α and L1-ratio for the elastic net. Due to the flexibility and large hyperparameter space of neural networks (for example, number of layers and neurons), they were not included in the nested cross-validation pipeline. Instead, the neural network was manually tuned and evaluated on the cross-validation test dataset, and the final model was subsequently assessed on the within-distribution, holdout test set. The within-distribution, holdout test dataset was held out to unbiasedly evaluate model performance (Supplementary Fig. To rigorously train the AI/ML models, we first defined participants without any pathologies based on ICD code and clinical history as CN (N = 5,291 for the heart MRIBAG, varying across the seven organs/tissues). CN within-distribution, holdout test dataset: 500 participants were randomly drawn from the CN population. Within-distribution, holdout test datasets are ideal for objectively evaluating ML model performance, especially in studies with large sample sizes, such as the present one. CN training/validation dataset: 80% of the remaining 4,791 CN (taking the heart MRIBAG for illustration) were used for the inner loop 10-fold cross-validation for hyperparameter selection. CN cross-validated test dataset: 20% of the remaining 4,791 CN were used for the outer loop 50 repetitions. Patient dataset: 29,785 patients who have at least one ICD-10-based diagnosis. The CN training/validation/test datasets were used for model development and were employed with a nested cross-validation procedure for four AI/ML models (LASSO regression and support vector regressor, elastic net and neural network), whereas the within-distribution, holdout test set provided an unbiased assessment of model performance. Model evaluation metrics included MAE and Pearson's r. Notably, consistent with our previous studies, only CN participants were included in the training/validation dataset for modeling and clinical interpretation considerations13. Age bias correction was applied using the approach outlined by Beheshti et al.16 and was systematically discussed in our previous comment13. To summarize, we applied an age bias correction method commonly used in neuroimaging studies, where a linear regression of the BAG on chronological age is used to estimate and remove systematic age-related bias via the model's slope and intercept. Notably, to address potential domain shift in the patient data compared to the CN training set, we computed the regression parameters directly from the patient set to more accurately correct for this bias (refer to our comment13 for further details). 32 details the feature importance across UKBB, BLSA and A4. The initial quality control procedures were described in the original study59, and we implemented additional quality control steps as outlined below. Our analysis focused on the first instance of the proteomics data (‘instance' = 0). We then integrated Olink files containing coding information, batch numbers, assay details and limit of detection (LOD) data (Category ID: 1839) by matching them to the proteomics dataset ID. Finally, we excluded Normalized Protein eXpression (NPX) values that fell below the protein-specific LOD. We conducted ProWASs by linking the seven MRIBAGs to 2,923 unique plasma proteins measured in 53,016 participants (with sample sizes ranging from 10,018 to 39,489 per protein after quality control) using the Olink platform. Additionally, organ-specific covariates were incorporated, such as brain positioning in the scanner (lateral, transverse and longitudinal; Field ID: 25756−25758), head motion (Field ID: 25741) and intracranial volume for the brain MRIBAG as well as diastolic (Field ID: 4079) and systolic (Field ID: 4080) blood pressure for the heart MRIBAG, among others. Multiple testing corrections were applied using Bonferroni adjustment (P < 0.05/2,923/7). To identify and exclude extreme outliers, we defined an upper threshold as the mean plus six times the s.d. Finally, ProWAS signals identified in the UKBB Olink dataset were compared with SomaScan data from the BLSA. We used the original data (Category ID: 220), which were analyzed and made available to the community by Nightingale Health Plc. The original data (1) were calibrated absolute concentrations (or ratios) and not raw NMR spectra and (2), before release, had already been subjected to quality control procedures by Nightingale Health Plc60. Following the additional procedures described in Ritchie et al.61, we performed additional quality check steps to remove a range of unwanted technical variations, including shipping batch, 96-well plate, well position, aliquoting robo and aliquot tip. We focused our analysis on the first instance of the metabolomics data (‘instance' = 0). The analysis included 327 metabolites (comprising both small molecules and lipoprotein measures), of which 107 were non-derived metabolites and the remainder were composite metabolites, across 274,247 participants. Descriptions of these metabolites are provided in Supplementary Table 3. We conducted MetWASs by linking the seven MRIBAGs to 327 plasma metabolites. Additionally, organ-specific covariates were incorporated, such as brain positioning in the scanner (lateral, transverse and longitudinal; Field ID: 25756−25758), head motion (Field ID: 25741) and intracranial volume for the brain MRIBAG, as well as diastolic (Field ID: 4079) and systolic (Field ID: 4080) blood pressure for the heart MRIBAG, among others. Multiple testing corrections were applied using Bonferroni adjustment (P < 0.05/327/7). We used the imputed genotype data from the UKBB for all genetic analyses. Our quality check pipeline focused on European ancestry in the UKBB (6,477,810 SNPs passing quality checks), and the quality-checked genetic data were merged with respective organ-specific populations for GWAS. We summarize our genetic quality check steps. First, we skipped the step for family relationship inference62 because the linear mixed model via fastGWA63 inherently addresses population stratification, encompassing additional cryptic population stratification factors. We then removed duplicated variants from all 22 autosomal chromosomes. Individuals whose genetically identified sex did not match their self-acknowledged sex were removed. Other excluding criteria included (1) individuals with more than 3% of missing genotypes; (2) variants with minor allele frequency (MAF; dosage mode) of less than 1%; (3) variants with more than 3% missing genotyping rate; and (4) variants that failed the Hardy−Weinberg test at 1 × 10−10. To further adjust for population stratification,64 we derived the first 40 genetic principal components using FlashPCA software65. Details of the genetic quality check protocol are described elsewhere11,14,34,49,66,67. We applied a linear mixed model regression to the European ancestry populations using fastGWA63 implemented in GCTA21. We used fastGWA to perform the seven MRIBAG GWASs, adjusting common variates, including age, dataset status (training/validation/test or within-distribution, holdout test), age-squared, sex, interactions of age with sex, BMI, waist circumference, standing height, weight and the first 40 genetic principal components, as well as organ-specific covariates, including the brain scan positions for the brain MRIBAG and systolic/diastolic blood pressure for the heart MRIBAG. We applied a genome-wide significance threshold (5 × 10−8) to annotate the significant independent genomic loci. We previously conducted GWAS of 2,923 plasma proteins and 327 metabolites using fastGWA5,68. For all GWASs, genomic loci were annotated using FUMA69. For genomic loci annotation, FUMA initially identified lead SNPs (correlation r2 ≤ 0.1, distance < 250 kb) and assigned them to non-overlapping genomic loci. Further details on the definitions of top lead SNP, lead SNP, independent significant SNP and candidate SNP can be found in the FUMA documentation (https://fuma.ctglab.nl/). For visualization purposes in Fig. We estimated the SNP-based heritability (h2) using GCTA21 with the same covariates as in GWAS. GCTA estimates the SNP-based heritability using a method called restricted maximum likelihood (REML) to quantify the proportion of phenotypic variance in a trait that the additive effects of all common SNPs can explain. The main steps involved in GCTA include constructing the genetic relationship matrix, modeling phenotypic variance and using REML to estimate the h2. The partitioned heritability analysis via stratified linkage disequilibrium score regression calculates the extent to which heritability enrichment can be attributed to predefined and annotated genome regions and categories70. Three sets of functional categories and cell-specific and tissue-specific types were considered. First, the partitioned heritability was calculated for 53 general functional categories (one including the entire set of SNPs). Subsequently, cell-type-specific and tissue-type-specific partitioned heritability was estimated using gene sets from Cahoy et al.71 for three main cell types (that is, astrocyte, neuron and oligodendrocyte), multi-tissue chromatin state-specific data (Roadmap72 and ENTEx73) and multi-tissue gene expression data (GTEx version 8 (ref. Bonferroni correction was performed for all tested annotations and categories. The linkage disequilibrium scores and allele frequencies for the European ancestry were obtained from a predefined version based on data from the 1000 Genomes project. We estimated the genetic correlation (gc) using LDSC75 software. We employed precomputed linkage disequilibrium scores from the 1000 Genomes of European ancestry, maintaining default settings for other parameters in LDSC. It is worth noting that LDSC corrects for sample overlap, ensuring an unbiased genetic correlation estimate76. Statistical significance was determined using Bonferroni correction. We constructed two-sample bidirectional Mendelian randomization by linking the seven MRIBAGs and 525 disease endpoints from FinnGen46 and PGC48. In total, two networks were established: MRIBAG2DE and DE2MRIBAG. The systematic quality-checking procedures to ensure unbiased exposure/outcome variable and instrumental variable selection are detailed below. We used a two-sample Mendelian randomization approach implemented in the TwoSampleMR package77 to infer the causal relationships within these networks. We employed five distinct Mendelian randomization methods, including the inverse variance weighted (IVW) method, Egger, weighted median and simple mode and weighted mode estimators. The STROBE-MR Statement78 guided our analyses to increase transparency and reproducibility, encompassing the selection of exposure and outcome variables, reporting statistics and implementing sensitivity checks to identify potential violations of underlying assumptions. First, we performed an unbiased quality check on the GWAS summary statistics. Notably, the absence of population overlapping bias79 was confirmed, given that FinnGen and UKBB participants largely represent populations of European ancestry without explicit overlap with the UKBB. PGC GWAS summary data were ensured to exclude UKBB participants. Furthermore, the GWAS summary statistics from all consortia were based on or lifted to GRCh37. Subsequently, we selected the effective exposure variables by assessing the statistical power of the exposure GWAS summary statistics in terms of instrumental variables, ensuring that the number of instrumental variables exceeded seven before harmonizing the data. Crucially, the function ‘clump_data' was applied to the exposure GWAS data, considering linkage disequilibrium. Bonferroni correction was applied to all tested traits based on the number of effective disease endpoints. Finally, we conducted multiple sensitivity analyses. First, we conducted a heterogeneity test to scrutinize potential violations of the instrumental variables' assumptions. To assess horizontal pleiotropy, which indicates the instrumental variables' exclusivity assumption80, we used a funnel plot, single-SNP Mendelian randomization methods and the Egger estimator. Furthermore, we performed a leave-one-out analysis, systematically excluding one instrument (SNP/instrumental variable) at a time, to gauge the sensitivity of the results to individual SNPs. PRS was computed using split-sample GWASs (split1 and split2) for the seven MRIBAG GWASs. Both base and target data underwent rigorous quality control procedures involving several steps: (1) excluding duplicated and ambiguous SNPs in the base data and (2) excluding high heterozygosity samples in the target data and (3) eliminating duplicated, mismatching and ambiguous SNPs in the target data. PRS-CS applies a continuous shrinkage prior, which adjusts the SNP effect sizes based on their linkage disequilibrium structure. SNPs with weaker evidence are ‘shrunk' toward zero, whereas those with stronger evidence retain larger effect sizes. This avoids overfitting and improves prediction performance. The shrinkage parameter was not set, and the algorithm learned it via a fully Bayesian approach. We conducted multiple analyses to establish genetic evidence for prioritizing potential druggable genes in future drug repurposing. Our central hypothesis is that genes with functional implications and causal roles validated across multiple omics layers, such as genomics, transcriptomics and proteomics, are more likely to be actionable for drug repurposing, as they offer a stronger and more robust foundation for identifying therapeutic targets82,83. For eQTL and chromatin interaction mapping, we used data from various resources, including GTEx, PsychENCODE, EyeGEx, TIGER, DICE, eQTLGen, Blood eQTL, MuTHER, CommonMind Consortium, BRAINEAC, FANTOM5, the Brain Open Chromatin Atlas and the Roadmap 111 epigenomes, all consolidated through the FUMA platform. By requiring both eQTL and chromatin interaction support, we enhance confidence in selecting functionally relevant genes because eQTL mapping identifies genetic variants that regulate gene expression. Chromatin interaction mapping links distal regulatory elements (for example, enhancers) to target genes, offering spatial genomic context. Together, these criteria ensure that the prioritized genes are not only associated with GWAS loci but also have strong regulatory and functional evidence, making them more biologically plausible as druggable targets. Subsequently, we connected MRIBAG, as an imaging-derived endophenotype, to aging clocks derived from plasma proteomics and metabolomics to further strengthen the genetic evidence for prioritizing these druggable genes. To achieve this, we used a Bayesian co-localization approach to assess whether the genomic loci associated with the seven MRIBAGs share common causal variants with the 11 ProtBAGs or the five MetBAGs. We used the R package ‘coloc' to investigate the genetic co-localization signals between two traits (for example, the brain MRIBAG versus the brain ProtBAG) at each genomic locus associated with the MRIBAG. This method (‘coloc.abf') examines the posterior probability to evaluate hypothesis H4 (PP.H4.ABF), which suggests the presence of a single shared causal variant associated with both traits within a specific genomic locus. To determine the significance of the H4 hypothesis, we set a threshold of PP.H4.ABF > 0.8 (ref. All other parameters (for example, the prior probability of p12) were set as the default. For each pair of traits, the genomic locus was defined by default from FUMA for one trait, and then the ‘coloc' package extracted and harmonized the GWAS summary statistics within this locus for the other trait. Finally, we searched the DGIdb platform (https://dgidb.org/) for genes that demonstrated both functional mapping evidence and causal associations to investigate their drug−gene interactions, revealing existing drugs and their therapeutic indications. We investigated the clinical promise of the seven MRIBAGs and their PRSs (MRIBAG-PRS) in three sets of prediction analyses: (1) survival analysis for the incidence of single disease entities based on the ICD-10 code, (2) survival analysis for the risk of all-cause mortality and (3) differential response to the Alzheimer's disease drug (solanezumab). We employed a Cox proportional hazard model while adjusting for covariates (that is, age and sex) to test the associations of the seven MRIBAGs with the incidence of ICD-based single disease entities. Participants who were diagnosed for a specific disease of interest after enrolling in the study were classified as cases; non-cases were defined by participants without any disease diagnoses. We employed a Cox proportional hazard model while adjusting for covariates (that is, age and sex) to test the associations of the seven MRIBAGs and seven MRIBAG-PRSs with all-cause mortality. The hazard ratio, exp(βR), was calculated and reported as the effect size measure that indicates the influence of each biomarker on the risk of mortality. Participants who died after enrolling in the study were classified as cases. We also conducted a disease-free survival analysis, including only participants without any diagnosed conditions, to eliminate potential confounding effects from disease pathology. To evaluate our hypothesis that individuals with varying aging clock paces may progress differently to cognitive decline, we analyzed clinical trial data from the A4 (ref. Specifically, we investigated whether brain MRIBAG could be used to stratify participants into decelerated and accelerated aging groups and demonstrated different cognitive profiles at 240 weeks by taking solanezumab. The original trial did not demonstrate cognitive decline slowing compared to the placebo over 240 weeks, with the treatment group even showing a slight worsening in the PACC score. To evaluate this, we performed four comparisons using natural cubic spline modeling: (1) decelerated aging (for example, below the brain MRIBAG median) versus accelerated aging (for example, above the median) within the drug group and (2) within the placebo group; (3) between drug and placebo groups among participants with accelerated aging; and (4) between drug and placebo groups among those with decelerated aging. From a statistical perspective, this is to model repeated measures (that is, PACC as the primary trial outcome for global cognition) as a continuous outcome in a mixed-effect model. We used the same method proposed by the original work from Donohue et al.85, in which the authors proposed a constrained longitudinal data analysis with natural cubic splines that treated time as continuous and used test version effects to model the mean over time for PACC. Fixed effects included the following terms: (1) spline basis expansion terms (two terms); (2) interaction of the spline basis expansion terms with treatment (two terms); (3) the version of the PACC test implemented; (4) baseline age; (5) education; (6) APOE4 carrier status (yes/no); and (7) baseline florbetapir cortical standardized uptake value ratio (SUVR) value. At week 240, we compared the mean PACC values between each pair of the two groups. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The GWAS summary statistics and pre-trained AI models from this study are publicly accessible via the MEDICINE Knowledge Portal (https://labs-laboratory.com/medicine/) and Synapse (https://www.synapse.org/Synapse:syn64923248/wiki/630992 (ref. Our study used data generated by the HPA (https://www.proteinatlas.org). GWAS summary data for the disease endpoints were downloaded from the official websites of FinnGen (R9: https://www.finngen.fi/en/access_results) and PGC (https://pgc.unc.edu/for-researchers/download-results/). Individual data from the UKBB can be requested with proper registration at https://www.ukbiobank.ac.uk/. The A4 study data can be requested at https://www.a4studydata.org/. The NIA-supported BLSA data can be requested at https://www.nia.nih.gov/research/labs/blsa. Due to privacy considerations, allele frequency information is available from J.W. (junhao.wen89@gmail.com) upon request for research purposes only, with responses typically provided within 1 week. The software and resources used in this study are all publicly available: MLNI87 (version 0.1.2 and version 0.1.5): https://github.com/anbai106/mlni, MRIBAG generation; FUMA (version 1.5.0): https://fuma.ctglab.nl/, gene mapping and genomic locus annotation; GCTA (version 1.94.1): https://yanglab.westlake.edu.cn/software/gcta/#Overview, fastGWA; LDSC (git version aa33296): https://github.com/bulik/ldsc, genetic correlation; TwoSampleMR (version 0.5.6): https://mrcieu.github.io/TwoSampleMR/index.html, Mendelian randomization; PRScs (release date: 10 August 2023): https://github.com/getian107/PRScs, PRS calculation; Lifelines (version 0.27.8): https://lifelines.readthedocs.io/en/latest/, survival analysis; String (version 12.0): https://string-db.org/, protein analyses; and coloc (version 5): https://github.com/chr1swallace/coloc; Bayesian co-localization. All main figures, extended data figures, the code used to derive the seven MRIBAGs and the supplementary files are available on Zenodo: https://zenodo.org/records/15857817 (ref. Eisenstein, M. Pushing the limits of MRI brain imaging. Cole, J. H., Marioni, R. E., Harris, S. E. & Deary, I. J. Brain age and other bodily ‘ages': implications for neuropsychiatry. Argentieri, M. A. et al. Proteomic aging clock predicts mortality and risk of common age-related diseases in diverse populations. Organ aging signatures in the plasma proteome track health and disease. Anagnostakis, F. et al. Multi-organ metabolome biological age implicates cardiometabolic conditions and mortality risk. Mutz, J., Iniesta, R. & Lewis, C. M. Metabolomic age (MileAge) predicts health and life span: A comparison of multiple machine learning algorithms. Teschendorff, A. E. & Horvath, S. Epigenetic ageing clocks: statistical methods and emerging computational challenges. Boquet-Pujadas, A. et al. MUTATE: a human genetic atlas of multiorgan artificial intelligence endophenotypes using genome-wide association summary statistics. The MULTI consortium et al. Brain-heart-eye axis revealed by multi-organ imaging genetics and proteomics. Towards a multi-organ and multi-omics medical digital twin. The genetic architecture of biological age in nine human organ systems. Tian, Y. E. et al. Heterogeneous aging across multiple organ systems and prediction of chronic disease and mortality. Refining the generation, interpretation and application of multi-organ, multi-omics biological aging clocks. The genetic architecture of multimodal human brain age. de Lange, A.-M. G. & Cole, J. H. Commentary: Correction procedures in brain-age prediction. & Duchesne, S. Bias-adjustment in neuroimaging-based brain age frameworks: a robust scheme. Bashyam, V. M. et al. MRI signatures of brain age and disease over the lifespan based on a deep brain network and 14 468 individuals worldwide. Sperling, R. A. et al. Trial of solanezumab in preclinical Alzheimer's disease. Sperling, R. A. et al. Association of factors with elevated amyloid burden in clinically normal older individuals. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Kendler, K. & Neale, M. Endophenotype: a conceptual analysis. Genetically supported targets and drug repurposing for brain aging: a systematic study in the UK Biobank. The OpenNeuro resource for sharing of neuroscience data. Gorgolewski, K. et al. Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in Python. MacGillivray, T. J. et al. Suitability of UK Biobank retinal images for automatic analysis of morphometric properties of the vasculature. Potential role of chitinase-3-like protein 1 (CHI3L1/YKL-40) in neurodegeneration and Alzheimer's disease. The expression pattern of GDF15 in human brain changes during aging and in Alzheimer's disease. Beydoun, M. A. et al. GDF15 and its association with cognitive performance over time in a longitudinal study of middle-aged urban adults. Statzer, C., Park, J. Y. C. & Ewald, C. Y. Extracellular matrix dynamics as an emerging yet understudied hallmark of aging and longevity. Rowland, J. et al. Uncovering genetic mechanisms of kidney aging through transcriptomics, genomics, and epigenomics. Wen, J. et al. Characterizing heterogeneity in neuroimaging, cognition, clinical symptoms, and genetics among patients with late-life depression. B. et al. mTOR inhibition improves immune function in the elderly. Blagosklonny, M. V. Rejuvenating immunity: ‘anti-aging drug today' eight years later. Franceschi, C., Garagnani, P., Parini, P., Giuliani, C. & Santoro, A. Inflammaging: a new immune–metabolic viewpoint for age-related diseases. Barcena, M. L., Aslam, M., Pozdniakova, S., Norman, K. & Ladilov, Y. Cardiovascular inflammaging: mechanisms and translational aspects. Kosyreva, A. M., Sentyabreva, A. V., Tsvetkov, I. S. & Makarova, O. V. Alzheimer's disease and inflammaging. Harrison, D. E. et al. Rapamycin fed late in life extends lifespan in genetically heterogeneous mice. & Lui, F. & Dhahbi, J. M. Organ reserve, excess metabolic capacity, and aging. The UK Biobank resource with deep phenotyping and genomic data. Duggan, M. R. et al. Proteomic analyses reveal plasma EFEMP1 and CXCL12 as biomarkers and determinants of neurodegeneration. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. O'Donovan, M. C. What have we learned from the Psychiatric Genomics Consortium. Wen, J. et al. Genomic loci influence patterns of structural covariance in the human brain. Bai, W. et al. A population-based phenome-wide association study of cardiac and aortic structure and function. Mojtahed, A. et al. Reference range of liver corrected T1 values in a population at low risk for fatty liver disease—a UK Biobank sub-study, with an appendix of interesting cases. Precision MRI phenotyping enables detection of small changes in body composition for longitudinal cohorts. Parisinos, C. A. et al. Genome-wide and Mendelian randomisation studies of liver MRI yield insights into the pathogenesis of steatohepatitis. Karlsson, A. et al. Automatic and quantitative assessment of regional muscle volume by multi-atlas segmentation using whole-body water-fat MRI. Borga, M. et al. Validation of a fast method for quantification of intra-abdominal and subcutaneous adipose tissue for large-scale human studies. Liu, Y. et al. Genetic architecture of 11 organ traits derived from abdominal MRI using deep learning. Wen, J. et al. Convolutional neural networks for classification of Alzheimer's disease: overview and reproducible evaluation. Li, W. U. K. Biobank pharma proteomics resource. Plasma proteomic associations with genetics and health in the UK Biobank. Würtz, P. et al. Quantitative serum nuclear magnetic resonance metabolomics in large-scale epidemiology: a primer on -omic technologies. Ritchie, S. C. et al. Quality control and removal of technical variation of NMR metabolic biomarker data in ~120,000 UK Biobank participants. Robust relationship inference in genome-wide association studies. Jiang, L. et al. A resource-efficient tool for mixed model association analysis of large-scale data. Price, A. L., Zaitlen, N. A., Reich, D. & Patterson, N. New approaches to population stratification in genome-wide association studies. & Inouye, M. FlashPCA2: principal component analysis of Biobank-scale genotype datasets. Wen, J. et al. Genetic and clinical correlates of two neuroanatomical AI dimensions in the Alzheimer's disease continuum. Wen, J. et al. Neuroimaging endophenotypes reveal underlying mechanisms and genetic factors contributing to progression and development of four brain disorders. Multiorgan biological age shows that no organ system is an island. Watanabe, K., Taskesen, E., van Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. The NIH Roadmap Epigenomics Mapping Consortium. Dunham, I. et al. An integrated encyclopedia of DNA elements in the human genome. The Genotype-Tissue Expression (GTEx) project. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Strengthening the Reporting of Observational Studies in Epidemiology Using Mendelian Randomization: the STROBE-MR Statement. Sanderson, E. et al. Mendelian randomization. Bowden, J. et al. A framework for the investigation of pleiotropy in two-sample summary data Mendelian randomization. Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Qi, T., Song, L., Guo, Y., Chen, C. & Yang, J. From genetic associations to genes: methods, applications, and challenges. Discovery of drug–omics associations in type 2 diabetes with generative deep-learning models. Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. Donohue, M. C. et al. Natural cubic splines for the analysis of Alzheimer's clinical trials. MEDICINE: AI-derived GWAS summary statistics sharing portal. Machine Learning in NeuroImaging (MLNI), version 0.1.4. https://github.com/anbai106/mlni (2025). This work is supported by the National Institute on Aging (NIA), National Institutes of Health (NIH) (RF1AG092412). ; UKBB application number: 647044) aims to integrate multi-organ imaging and multi-omics data to advance understanding of human aging and disease mechanisms. This study used the UKBB resource under application numbers 647044 (J.W. We would like to express our sincere gratitude to the UKBB team for their invaluable contribution to advancing clinical research in our field (https://www.ukbiobank.ac.uk/). We would like to acknowledge the participants and investigators of the FinnGen study and the PGC, and we thank FinnGen (https://www.finngen.fi/en) and PGC (https://pgc.unc.edu/) for their generosity in sharing the GWAS summary statistics with the scientific community. We thank the BLSA participants and staff for their participation and continued dedication. The BLSA protocol was approved by the institutional review board of the National Institute of Environmental Health Science, NIH (03AG0325). The A4 study was a secondary prevention trial in preclinical Alzheimer's disease, aiming to slow cognitive decline associated with brain amyloid accumulation in clinically normal older individuals. The A4 study was funded by a public−private−philanthropic partnership, including funding from the NIA/NIH, Eli Lilly and Company, the Alzheimer's Association, the Accelerating Medicines Partnership, the GHR Foundation, an anonymous foundation and additional private donors, with in-kind support from Avid Radiopharmaceuticals, Cogstate, Albert Einstein College of Medicine and the Foundation for Neurologic Diseases. The companion observational Longitudinal Evaluation of Amyloid Risk and Neurodegeneration (LEARN) study was funded by the Alzheimer's Association and the GHR Foundation. The A4 and LEARN studies were led by R. Sperling at Brigham and Women's Hospital, Harvard Medical School, and by P. Aisen at the Alzheimer's Therapeutic Research Institute at the University of Southern California. The complete A4 study team list is available at https://www.actcinfo.org/a4-study-team-lists/. We would like to acknowledge the dedication of the study participants and their partners who made the A4 and LEARN studies possible. We acknowledge the leadership of the Brain Imaging Genetics workgroup, led by T. Evans, N. Vilor-Tejedor and J.W., within the International Society to Advance Alzheimer's Research and Treatment (ISTAART) community, for advocating brain imaging genetics in Alzheimer's disease and aging research. Laboratory of AI and Biomedical Science (LABS), Columbia University, New York, NY, USA Huizi Cao, Zhiyuan Song & Junhao Wen Laboratory of Behavioral Neuroscience, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA Artificial Intelligence in Biomedical Imaging Laboratory (AIBIL), Center for AI and Data Science for Integrated Diagnostics (AI2D), Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Guray Erus, Dhivya Srinivasan & Christos Davatzikos Ye Ella Tian & Andrew Zalesky Department of Brain Sciences and Department of Computing, Imperial College London, London, UK Alzheimer's Therapeutic Research Institute, Keck School of Medicine of the University of Southern California, San Diego, CA, USA Robert N Butler Columbia Aging Center and Department of Epidemiology, Columbia University Mailman School of Public Health, New York, NY, USA Center for Innovation in Imaging Biomarkers and Integrated Diagnostics (CIMBID), Department of Radiology, Columbia University, New York, NY, USA Department of Radiology, Columbia University, New York, NY, USA Department of Biomedical Engineering, Columbia University, New York, NY, USA Zuckerman Institute, Columbia University, New York, NY, USA Data Science Institute (DSI), Columbia University, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar has full access to all the study data and is responsible for integrity and accuracy. Critical revision of the manuscript for important intellectual content: all authors. derived the original heart IDPs. derived the nine original PhenoBAGs, and we only used the derived GWAS summary statistics in this study. is an inventor of the DunedinPACE epigenetic clock, which is licensed by Duke University and the University of Otago to TruDiagnostic, from which he receives royalties (DunedinPACE is freely available to researchers). All other authors declare no competing interests. Nature Medicine thanks David Furman and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Anna Maria Ranzoni in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) For each organ-specific MRIBAG, we split the UK Biobank populations into healthy controls (CN, without any pathologies) and patients (PT) based on ICD-10 codes and their clinical history. b) To develop the 7 MRIBAGs, we trained two AI/ML models exclusively on the CN training/validation/test population using a nested cross-validation procedure to identify the optimal model. Hold-out datasets included the CN within-distribution, hold-out test dataset (N = 500), and the PT population. c) The study's analytical workflow encompassed deriving the 7 MRIBAGs and their PRSs, performing ProWAS, MetWAS, GWAS, and post-GWAS analyses, and assessing the predictive performance of the MRIBAG and MRIBAG-PRSs for single disease entities, mortality, and drug outcomes. Data from the BLSA and A4 studies were used to generate the brain MRIBAG in independent cohorts. Proteome-wide association studies (ProWAS) between the 7 MRIBAGs and 2923 plasma proteins via a linear regression model, accounting for a full set of covariates. Representative significant (P-value < 0.05/2923/7, indicated by the dotted horizontal line) MRIBAG-protein associations are annotated by the respective gene symbol. We used the standardized β value as an effect size measurement. STRING conducts protein-protein interaction (PPI) analysis by combining various sources of interaction evidence, such as experimental data, computational predictions, and text mining. In addition to interactions, STRING can carry out functional enrichment analysis to identify Gene Ontology (GO) terms (for example, biological processes, molecular functions), KEGG pathways (for example, metabolic and signaling pathways), and protein domains (for example, shared structural motifs). a) PPI network of the brain MRIBAG-related proteins. b) Functional PSEA of the brain MRIBAG-related proteins. Results and figures are generated using STRING v12.0 (https://string-db.org/). Metabolome-wide association studies (MetWAS) between the 7 MRIBAGs and 327 plasma metabolites via a linear regression model, accounting for a full set of covariates. Representative significant (P-value < 0.05/327/7, indicated by the dotted horizontal line) MRIBAG-metabolite associations are annotated. We used the standardized β value as an effect size measurement. a) Top 25 MSEA results for the spleen MRIBAG-related small molecule metabolites. b) The MMI network results for these metabolite sets are displayed as follows: each node represents a metabolite set, with color indicating the enrichment P-value and size reflecting the fold enrichment ratio. Two metabolite sets are connected by an edge if the proportion of shared metabolites between them exceeds 25% of the total combined metabolites. Results and figures are generated using MetaboAnalyst v6.0 (https://www.metaboanalyst.ca/). We used 3694 metabolite and lipid pathways from RaMP-DB (which integrates KEGG via HMDB, Reactome, and WikiPathways, curated via MetaboAnalyst) as the background metabolite dataset. Only metabolite sets with at least two entries were included. f) MetWAS results for total concentration of phospholipids (Total_P) and the spleen MRIBAG. A linear regression model was fitted, and the resulting Pearson's r and P-value are reported. We present the independent, significant SNPs defined in FUMA. All supplementary figures, notes and tables are included. This is study-specific code to produce the seven MRIBAGs using MLNI software. The pre-trained AI model was also shared via our public MEDICINE portal. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. The MULTI Consortium., Cao, H., Song, Z. et al. MRI-based multi-organ clocks for healthy aging and disease assessment. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64619-0'>Global climate mode resonance due to rapidly intensifying El Niño-Southern Oscillation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 09:14:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The El Niño-Southern Oscillation (ENSO) influences climate variability globally, encompassing various other modes of variability, and thus represents a key predictable climate signal on seasonal timescales. Yet, its response to greenhouse warming remains uncertain, with models projecting a range of outcomes. Here, we demonstrate that in response to warming, a state-of-the-art high-resolution climate model simulates a rapid transition from a moderate-amplitude irregular regime, as observed in the current climate, to a highly regular oscillation with intensifying amplitude. This behaviour can be attributed to increasing air-sea feedbacks, which approach criticality in the second half of this century, and growing atmospheric noise. As ENSO intensifies in this model, it synchronizes with other prominent climate modes, such as the North Atlantic Oscillation and the Indian Ocean Dipole, thereby imprinting its regular, predictable variability on them. If realized, this global climate mode resonance would have wide-ranging whiplash impacts on regional hydroclimates. Despite the profound influence of the El Niño-Southern Oscillation (ENSO) on the global climate system1,2, its response to greenhouse warming remains uncertain. Climate models exhibit a wide range of possible future ENSO behaviors1,3,4,5,6,7, hindering confidence in regional climate projections. The dynamics of ENSO are governed by a delicate balance of positive and negative feedbacks8,9 that determine both ENSO's instability10 and periodicity11. The relative strengths of the individual feedbacks are, in turn, determined by both model parametrizations and the structure of the climate mean state12,13,14. Previous research, using both simple low-order models15 and intermediate complexity models16, has demonstrated how changes in the climate mean state can affect the strength of these feedbacks (such as the zonal advective and thermocline feedbacks) and thereby ENSO characteristics (such as its growth rate, periodicity, and spatial pattern). In addition, recent studies also indicated that the interactions of ENSO with the seasonal cycle17,18,19 as well as with other more damped empirical modes in the climate system, such as the Indian Ocean Dipole (IOD)20, the Tropical North Atlantic (TNA) mode21, or the North Atlantic Oscillation (NAO)22 can shape the dynamics of both ENSO and these other modes23,24,25. Over 20 years ago, an ENSO-resolving coupled general circulation model5,26 exhibited a very intriguing ENSO behavior. It showed a gradual increase in the linear ENSO growth rate in response to greenhouse warming and the crossing of a Hopf bifurcation in the mid-twenty-first century26. This ENSO regime shift towards supercriticality resulted in a rapid intensification in ENSO's amplitude. Such a drastic transition in qualitative ENSO behavior has not been reported in other complex climate models. Here we revisit the issue of anthropogenically forced rapid emergence of ENSO supercriticality and its potential repercussions on global climate using an ensemble of state-of-the-art high resolution climate model simulations (AWI-CM3, TCo319 horizonal resolution with ~31 km and 137 vertical layers in the atmosphere and ~4–25 km with 80 vertical layers in the ocean)27, subject to SSP5-8.5 greenhouse gas forcing (see “Methods”). We demonstrate that this model simulates a rapid intensification of ENSO by mid-twenty-first century, a transition to a regular, strongly seasonally-locked oscillation (with periodicities of 2, 3, 4, and 5 years), and an unprecedented resonance with other important modes of climate variability. Using a hierarchy of simplified dynamical ENSO models, with parameters estimated from the complex AWI-CM3 simulations, we study the underlying mechanisms for the qualitative change in ENSO behavior, focusing on coupled air-sea feedbacks and atmospheric noise. While models from the Coupled Model Intercomparison Project Phase 6 (CMIP6) show a wide range of possible future ENSO regularity and amplitude projections, a few models show qualitatively similar behavior to AWI-CM3. If this peculiar ENSO dynamical scenario were to materialize in the future, it could lead to both increased ENSO predictability, due to greater regularity, and, at the same time, to warming-amplified “whiplash impacts”28 on regional climates. These impacts would arise from the compounding effects of (1) more regular ENSO transitions, (2) increased ENSO sea surface temperature (SST) variance, (3) larger ENSO impacts on precipitation and the atmospheric circulation for the same SST anomaly29, and (4) synchronized fluctuations of the other climate modes. As the Earth warms in response to increasing greenhouse gas concentrations, the AWI-CM3 TCo319 model27 simulates a future increase in variability of both SST (Fig. 1e) and sea level pressure (SLP) (Fig. 1f) between the current (Period P1; 2015–2035) and end-of-the-century (Period P2; 2080–2100) climate in many regions of the globe. Further looking at the key regional aspects of these SST and SLP variability changes, we see a striking projected increase in ENSO SST variance as well as in ENSO regularity, i.e., a tendency from an intermittent to a more cyclic ENSO behavior, reminiscent of a Hopf bifurcation30 over this century (Figs. Interestingly, the model simulates qualitatively similar changes in the NAO index (Fig. 1c, d), the dominant empirical mode of atmospheric variability over the North Atlantic and Europe, especially in boreal winter (December–January–February: DJF). Both the dominant interannual ENSO timescale, as well as its near-annual combination tones17,18, are evident in the NAO wavelet power spectrum above a white noise background, and their variance increases with time and rising greenhouse gas concentrations (Fig. Increasing ENSO influences on the NAO are likely to have important implications for atmospheric impacts and climate predictability over Europe27. a, c Time evolution of the El Niño-Southern Oscillation (ENSO) sea surface temperature (SST) index [K] and the normalized North Atlantic Oscillation (NAO) index [n.u.] for ensemble member 0 (3-month running-mean applied). b, d Ensemble-mean (four members post year 2055) wavelets of the ENSO SST and NAO indices. Black contours enclose regions above 90% confidence level significance tested against a Lorentzian (b) and a white noise (d) spectrum, respectively. e, f Spatial maps of the ensemble-mean standard deviation changes between period 2 (P2, 2080–2100) and period 1 (P1, 2015–2035) of the monthly SST anomaly (SSTA) (e) and December–January–February (DJF) sea level pressure (SLP) anomaly (SLPA) (f). Dots indicate regions where all four ensemble members show disagreement in the sign of changes. Boxes in (e) indicate the cold tongue ENSO region as well as the key regions associated with SST variability of the North Pacific Meridional Mode (NPMM), Indian Ocean Dipole (IOD), Indian Ocean Basin (IOB) mode, and the Tropical North Atlantic (TNA) mode. Boxes in (f) indicate the SLP centers of the NAO. Moving 21-year changes of sample entropy (SampEn) (a) and standard deviation (SD) (b) of the Niño3.4 SST anomaly index in observations (black), the 150-year AWI-CM3 1950 control simulation (CTL1950, blue), and the AWI-CM3 shared socio-economic pathway (SSP)5-8.5 scenario simulation (red). Solid lines and shading indicate the multi-product/multi-member mean and the 1 SD spread of 10,000 inter-realizations using a bootstrap method, respectively. The error bars indicate the mean and 1 SD of values across all overlapping 21-year windows for the observations (1950–2024), CTL1950 (150-yr), and SSP5-8.5 (2050–2100). Violin plots showing the probability density distribution of ENSO regularity change (c) and amplitude change (d) across 49 CMIP6 models. Dots represent individual model results, with E3SM-1-1, EC-Earth3, and EC-Earth3-Veg highlighted in distinct colors; the red star marks the AWI-CM3 result (2050–2100 minus CTL1950). In a and c, the y-axis is inverted to emphasize decreasing SampEn, indicating increasing regularity. a Ensemble-mean moving 21-year seasonally varying (shading) and monthly (blue curve) standard deviation (SD) changes of the ENSO sea surface temperature anomaly (SSTA) index relative to the monthly SD during 2015–2035 (ratio). b Growth rate of the leading eigenmode (ENSO) obtained via Floquet analysis for the Recharge Oscillator (RO) (blue curve) and eXtended Recharge Oscillator (XRO) (magenta curve) models in a moving 21-year window. The seasonal RO growth rate is indicated in shading. c Noise amplitude defined as the seasonally varying SD (shading) and monthly SD (blue curve) of the XRO fit residual. Different ENSO feedbacks (as a function of calendar month) obtained via Bjerknes stability analysis for period 1 (P1) (d, 2015–2035; one ensemble member) and period 2 (P2) (e, 2080–2100; ensemble-mean of four members), as well as their difference (f). The horizontal axes in d–f show the Bjerknes (BJ) index, SSTA growth rate \((R)\), ocean damping rate (\(-\varepsilon\)), thermocline feedback (TH), zonal advective feedback (ZA), Ekman feedback (EK), thermal damping by the net surface heat flux (TD), dynamic damping by mean horizontal currents (DD), all residuals from nonlinear dynamic heating and unresolved processes (AllRes), as well as the dynamic damping components by mean zonal (DDu) and meridional (DDv) currents and the thermal damping components by shortwave radiation (QSW), longwave radiation (QLW), latent heat flux (QLH), and sensible heat flux (QSH). g Latitudinal profiles (averaged of 160°E–150°W) of regressed zonal wind stress anomalies \({\tau }_{x}^{{\prime} }\) onto the monthly ENSO SST index in a moving 21-year window. Dashed curves and shadings indicate the ensemble mean and one SD spread of meridional boundaries defined by the local minima. h, i Basin thermocline adjustment rate (\(-\varepsilon\)) and Recharge/discharge efficiency (\(-{F}_{2}\)) in a moving 21-year window. In h and i, red curves denote the meridional width of ENSO zonal wind stress anomalies \({\tau }_{x}^{{\prime} }\) (see “Methods”). Next, using sample entropy (SampEn; see “Methods”) of Niño3.4 SST anomalies as a metric for ENSO regularity together with the Niño3.4 SST anomaly standard deviation (SD), we show that a 1950 control simulation of AWI-CM3 TCo319 exhibits both ENSO regularity (Fig. 2a) and ENSO amplitude (Fig. 2b) similar to the observations and that the increase of both metrics in the AWI-CM3 TCo319 future projections cannot be explained by internal variability. Furthermore, analysis of the CMIP6 model archive demonstrates that while there is a wide range of possible changes in future ENSO regularity (Fig. 2d), 55% of CMIP6 models show an increase in ENSO regularity (Fig. 2c) and 82% an increase in ENSO SST anomaly amplitude (Fig. 2d) by the second half of this century under the SSP5-8.5 scenario. In addition, a few models (i.e., E3SM-1-1, EC-Earth3, and EC-Earth3-Veg) show similar increased ENSO regularity and amplitude as AWI-CM3 TCo319 (Fig. 2c, d and Supplementary Fig. We emphasize that we do not expect strong inter-model agreement of ENSO projections across the CMIP6 archive given that considerable biases of climate mean states and ENSO dynamics result in different ENSO regimes across models8,24. To understand the simulated regime shift in ENSO toward higher variance and increased regularity in AWI-CM3 TCo319, we derive two conceptual ENSO models directly from the simulation output. The first is a version of the recharge oscillator (RO) model13 that includes seasonal cycle modulations9 and encapsulates the fundamental ENSO dynamics in two coupled ordinary differential equations (ODEs) for SST anomalies in the Niño3.4 region and the zonal-mean equatorial upper-ocean warm water volume. The second is a modified version of the extended recharge oscillator (XRO)24, which also takes into account the seasonally-modulated interactions between ENSO (as described by the RO) and the North Pacific Meridional mode (NPMM), TNA, IOD, and the Indian Ocean Basin (IOB) mode (yielding six coupled ODEs; see “Methods”). We estimate the time-evolving parameters of both conceptual models using multi-linear regression in a moving 21-year window. This allows us to compute the time-dependent linear growth rates and frequencies of ENSO via Floquet eigen analysis31 and assess changes in the damping rates and coupling strengths of the interacting modes in response to greenhouse warming. Both models show that ENSO's linear growth rate increases considerably later in the twenty-first century, particularly during boreal summer (Fig. This is consistent with the increase in ENSO variance in boreal winter (Fig. We emphasize that while ENSO, in an annual mean sense, is still in the stable (subcritical) regime, the presence of atmospheric stochastic noise, can lead to a noise-induced Hopf bifurcation that occurs before criticality (i.e., the point at which its growth rate switches from negative to positive) is reached32,33. Moreover, the noise amplitude, calculated as the residual from the XRO model projection, increases substantially in boreal spring to summer (Fig. 3c), together with the growth rate, pushing ENSO into the high-variance and highly-regular regime. This is consistent with increased tropical intraseasonal atmospheric variability in the AWI-CM3 TCo319 simulation (Supplementary Fig. 27), which is well known to energize the ENSO mode2,9,34 and also seen in other climate model simulations35. 1b) with the linear ENSO frequency for both the RO and XRO (Supplementary Fig. 3), we see close agreement at the dominant interannual frequencies of ~2, 3, 4, and 5 yr−1. Next, we use the Bjerknes stability analysis (see “Methods”) to determine which feedback changes are responsible for ENSO's growing instability. Comparing the difference (Fig. 3d) of the individual ENSO feedbacks that make up ENSO's net growth rate, we find that the increased growth rate primarily results from a reduced damping from thermocline adjustment (\(\varepsilon\)) and a modest enhancement of the Bjerknes feedback (R). The changes in R can be explained by enhanced Ekman feedback (EK) and reduced dynamic damping (DD), which are slightly offset by a reduced thermocline feedback (TH) and increased thermodynamic damping (TD). The enhanced Ekman feedback is primarily driven by enhanced stratification and an amplified anomalous vertical current response to SST anomalies (Supplementary Fig. The reduced dynamic damping is linked to the weakening of the climatological trade winds and surface currents. For the zonal advective feedback changes, we see a cancellation effect between the mean state and feedback changes, with reduced \(d\overline{T}/{dx}\) but strongly enhanced anomalous zonal current response to SST anomalies (Supplementary Fig. The reduced thermocline feedback is explained by a weakening of the climatological trade winds and climatological upwelling (Supplementary Fig. We attribute the reduced damping from thermocline adjustment (\(\varepsilon\)) to the widening of ENSO wind stress anomalies (Fig. 3g, h and Supplementary Fig. 5a), which leads to increased wind stress curl off the equator, exciting longer Rossby waves. These are less effectively reflected at the western boundary and thus lead to reduced damping from thermocline adjustment (see “ENSO wind stress structure” in “Methods”). The changes of R and \({\varepsilon}\) together lead to the largest growth rate increase in the boreal spring season (Fig. The AWI-CM3 model explicitly resolves Tropical Instability Waves (TIWs), which also play a key role in ENSO thermodynamics4,37, mainly as a negative feedback38. With ocean background currents changing in response to the simulated greenhouse warming, the statistics of TIWs will also change, and their contribution to the ENSO heat budget4 will also change. This effect has not been explicitly accounted for in our calculations of the ENSO instability index, but it is captured, among other effects, in the residual term (Fig. For climate modes, which are characterized by the variations in SST, such as the TNA mode (Fig. 4e), we find a considerable intensification of their amplitude, ranging from 40–75%. All these modes are known to interact with ENSO18,23,24. To further elucidate the apparent emergent synchronization between ENSO and the other climate modes (i.e., NPMM, TNA, IOD, and IOB) in response to greenhouse warming (Fig. 4), we calculate the power spectrum for each mode for P1 and P2 respectively (Supplementary Fig. We see a clear indication of statistically significant and intensifying spectral peaks for the NPMM, TNA, IOD, and IOB indices at the dominant ENSO frequencies (fE) as well as at the near-annual ENSO combination tone frequencies (1±fE) above a Lorentz/Hasselmann background spectrum39, consistent with theoretical expectations17,18. The emergence of interannual and near-annual ENSO timescales in the spectra of other climate modes provides the first clear evidence that their coupling with ENSO strengthens in response to greenhouse warming in this model. In addition, we see a clear intensification of both the ENSO and ENSO combination mode17 wind stress variability (Supplementary Fig. a–e Ensemble-mean moving 21-year seasonally varying (shading) and monthly (blue curve) standard deviation (SD) changes of each climate mode relative to its monthly SD during 2015–2035 (ratio) for the North Atlantic Oscillation (NAO), Tropical North Atlantic (TNA) mode, Indian Ocean Basin (IOB) mode, Indian Ocean Dipole (IOD), and North Pacific Meridional Mode (NPMM) indices, respectively. f–j Ensemble-mean moving 21-year phase synchronization quantified by the histogram of phase differences (shading) between each climate mode and the Niño3.4 index for NAO, TNA, IOB, IOD, and NPMM, respectively. The blue curves in f–j indicate synchronization strength defined by the SD of the histogram density (PDF) of phase differences. To better quantify the dynamical linkage between different climate modes, we calculate the phase difference (see “Methods”) between the Niño3.4 index (representing ENSO) and the other indices (Fig. A bounded phase difference indicates stronger phase-coupling between the climate modes under consideration (Supplementary Fig. Here, we focus on 1:1 phase synchronization, meaning that the oscillators maintain a (near-)constant phase difference. Starting around 2060, a strong preference for bounded phase differences emerges between ENSO and each of the other modes (i.e., blue curves in Fig. 4f–j have high values), thus providing further evidence of an intensifying connection between ENSO and the other modes as time progresses. The time-evolving probability distribution of phase differences (red shading in Fig. 4f–j) also documents that the range of phase differences narrows considerably for all modes, supporting the evidence for strengthened phase synchronization with ENSO. Under present-day conditions, the influence of ENSO on the NAO is detectable but generally weak and indirect. Consequently, current seasonal forecasts for the European climate do not treat ENSO as a primary driver of NAO-related variability or impacts. According to our simulations, however, this could change in the future. 4a), but the overall seasonal amplitude modulation with a peak in February remains relatively stable. As greenhouse gas concentrations increase, the preferred phase difference between ENSO and the NAO becomes more robust (Fig. 4f), with dominant peaks between −π and −π/2, consistent with an increasingly negative correlation in boreal winter. Overall, we expect these dynamics to potentially influence the seasonal predictability of the NAO, as illustrated in Supplementary Fig. 9, which shows the projected extended persistence of both ENSO and NAO. It is also illustrated in Fig. 1, which shows that after 2060, 2-year-long La Niña events typically create two subsequent strong NAO events. Whether the intensification of the NAO linkage to ENSO emerges, simply because the ENSO SST amplitude increases during the simulation, or whether large-scale reorganizations of the extratropical atmospheric circulation facilitate the coupling between the Pacific-North American (PNA) pattern and the NAO in wintertime (DJF) is quantified by a regression analysis for both periods (Fig. For P1, the regression of precipitation and 500 hPa geopotential height (Z500) anomalies onto Niño3.4 SST anomalies and Niño4 precipitation shows no substantial impact in Europe. During P2, for the regressions with both the Niño3.4 SST index in degrees Kelvin (Fig. 5e) and with the normalized Niño3.4 SST index (Fig. 5d), we see increases in the regression coefficient amplitudes and a clear negative NAO pattern emerging. The strongest signal can be found for the normalized Niño3.4 index, which indicates that the amplification of the SST signal itself plays a vital role in strengthening the teleconnection. However, even a 1 K SST change in the Niño3.4 region, the NAO pattern emerges much more strongly in P2 than in P1, indicating a higher sensitivity of the atmospheric response to the same amplitude of warming in the eastern equatorial Pacific. As a result of the increased synchronization between ENSO and the NAO pattern in boreal winter, we also find an enhanced precipitation response over western Europe27 with considerably wetter conditions occurring over the Iberian Peninsula during El Niño events (Fig. Recent studies have suggested that the ENSO–NAO teleconnection may strengthen in response to greenhouse warming, either due to amplified ENSO forcing or increased extratropical sensitivity. Reference40 showed that CMIP5 models simulate both enhanced tropical Pacific precipitation variability and a more robust ENSO–NAO correlation in the future, with evidence for stronger stratospheric and tropospheric pathways. Reference41, using a pacemaker approach with fixed ENSO SST anomalies, attributed the strengthened NAO response to increased extratropical sensitivity linked to a more zonally extended Pacific jet. Our study confirms and extends these results by showing that in a high-resolution coupled model, both mechanisms occur simultaneously: ENSO amplitude increases due to stronger air–sea feedbacks and enhanced atmospheric noise, while normalized regressions reveal that the extratropical circulation becomes more sensitive to ENSO forcing. In addition, we identify a regime shift toward a highly regular, seasonally phase-locked ENSO that entrains other modes like the NAO, suggesting a possible global synchronization of interannual variability under climate change. a Regression of normalized Niño3.4 sea surface temperature anomalies (SSTA) with precipitation anomalies (PrA) (shading, [mm day−1]) and 500 hPa geopotential height anomalies (Z500A) (contours, ±5, ±15, … [m]) during December–January–February (DJF) 2015–2035 (P1); b Regression of Niño3.4 SST anomalies with precipitation anomalies (shading, [mm day−1 K−1]) and Z500 anomalies (contours, ±5, ±15, … [m K−1]) during DJF 2015–2035; c Regression of Niño4 precipitation anomalies with precipitation anomalies (shading, [mm day−1 (mm day−1)−1]) and Z500 anomalies (contours, ±5, ±15, … [m (mm day−1)−1]) during DJF 2015–2035. d–f Same as a–c but for ensemble-mean results during DJF 2080–2100 (P2), respectively. Stippling and dark contours indicate statistical significance at the 95% level using a one-sided Student's t test. The increase in ENSO variance and regularity directly leads to increased variance and regularity of the other SST climate modes. In addition, we find that both a reduction of their individual damping rates (Supplementary Fig. 10a–d) and increased ENSO teleconnection strength (Supplementary Fig. 10e–l) also contribute to the global climate mode resonance. The enhanced variance of the NPMM is primarily driven by reduced damping from February to July (Supplementary Fig. 10a), likely associated with SST warming in the subtropical northeastern Pacific and enhanced wind–evaporation–SST (WES) feedback42. The reduction in damping rate and the strengthening of ENSO teleconnection are essential for increasing IOD and TNA variance (Supplementary Fig. Interestingly, the IOB exhibits an enhanced early variance peak in February–March and a secondary peak in May–June by the end of the twenty-first century (Fig. This change is related to reduced damping in addition to enhanced ENSO teleconnection from November to April (Supplementary Fig. Previous work has shown that the atmosphere is “ringing” with a range of combination tones in response to ENSO forcing, causing an “ENSO frequency cascade”43. Here we show that ENSO not only causes these deterministic signals in the atmosphere, but its intensification can also energize both atmospheric and air-sea coupled climate modes across the globe in response to greenhouse warming. This global climate resonance towards the strengthening ENSO signal is apparent in the raw timeseries (Fig. 1), their amplitudes, spectra, and phase synchronization characteristics (Fig. ENSO variability increases in the AWI-CM3 simulation because of an intensification of atmospheric noise in boreal summer (Fig. 2c), reduced damping from thermocline adjustment, and the modest increase of the Bjerknes feedback (Fig. Its global synchronization towards the extratropics (e.g., with the NAO) is boosted by the growing ENSO amplitude, but also by an overall reorganization of atmospheric teleconnection patterns (Fig. 5), due to a different future atmospheric mean flow. The changes in the mean flow—particularly a stronger, more zonally extended Pacific jet and increased Atlantic baroclinicity—can enhance the propagation, amplification, and momentum deposition of ENSO-forced stationary waves into the North Atlantic sector, thereby strengthening the teleconnection to the NAO. Increasing ENSO amplitude and teleconnection patterns imply that remote extratropical precipitation responses, such as in Southern California and the Iberian Peninsula (Fig. 5), will also become stronger between alternating El Niño and La Niña events. Even though the recurrence of these events may become more predictable due to the increased regularity/periodicity and amplitude (Fig. 1)44, future ENSO teleconnections might generate enhanced whiplash effects on hydroclimate, which requires additional planning and management strategies to minimize the costs of climate damage. While similar increases in ENSO regularity and amplitude can be found in some CMIP6 model projections (Fig. 2), future work needs to assess the detailed dynamics across these models and assess the likelihood of the ENSO regime changes seen in AWI-CM3. This study employs the Alfred Wegener Institute Climate Model (AWI-CM3), which couples the OpenIFS (Open Integrated Forecasting System) atmosphere model with the FESOM2 (Finite-volume Sea Ice-Ocean Model) ocean model. The atmospheric component operates at a horizontal resolution of approximately 31 km (TCo319) with 137 vertical pressure levels. It contains the WAM (Wave Model) surface gravity wave model and the hydrology model H-TESSEL (Hydrology in the Tiled ECMWF Scheme for Surface Exchange over Land). The ocean model features a variable horizontal resolution ranging from 5 to 27 km, depending among others on latitude, and includes 80 vertical layers. The reader is referred to a more detailed description of AWI-CM3 and the simulations (except the additional ensemble members)27. A full transient simulation covering the period from 1950 to 2100, branched off from a 100-year-long spin-up simulation, was conducted using historical forcing from 1950 to 2014 CE, followed by the high-emission shared socio-economic pathway (SSP) 5-8.5 scenario thereafter. To assess the robustness of our results, three additional ensemble simulations spanning the period from 2055 to 2100 were conducted. In line with the micro-perturbation initialization strategy (e.g., ref. 35), a small perturbation was added to the wave model restart files. A 150-year TCo319 control simulation with fixed 1950 forcing (CTL1950) was also analyzed to characterize internal climate variability and to serve as a baseline comparison. The CTL1950 simulation reproduces the observed SST and precipitation mean state and SST variability reasonably well (Supplementary Fig. The CTL1950 simulation also reproduces the observed characteristics of ENSO, including its seasonal synchronization, spectral characteristics, and teleconnections (Supplementary Fig. We use four observational SST reconstructions/reanalysis: the Hadley Centre Sea Ice and Sea Surface Temperature dataset v.1.1 (HadISST45), the Extended Reconstructed Sea Surface Temperature v.5 (ERSSTv546), the Centennial in situ Observation-Based Estimates of Sea Surface Temperature v.2 (COBE247) for 1871–2024, and the European Centre for Medium-Range Weather Forecasts (ECMWF) Reanalysis 5 (ERA548) for 1940–2024. In addition, ERA5 monthly precipitation and sea level pressure (SLP) fields are used for 1940–2024. We also analyze simulations from 49 CMIP6 models, including historical runs and SSP5-8.5 projections, providing monthly SST and SLP data (Supplementary Table 1). The simulations are forced by historical anthropogenic and natural forcings up to 2014, followed by future greenhouse-gas forcing under the SSP5-8.5 scenario through 2100, covering the period 1850–2100. All model output was re-gridded to a common 1° × 1° horizontal resolution using bilinear interpolation. All monthly fields are first quadratically detrended, and then anomalies are computed by subtracting the 21-year running monthly climatology. We then calculate the climate mode indices with the detrended anomalies. The ENSO SST index is represented as SST anomalies averaged over the cold tongue region (180°–90°W, 6°S–6°N). The Niño3.4 SST index is defined as SST anomalies averaged over the Niño3.4 region (170°–120°W, 5°S–5°N). The Niño4 precipitation index is defined as precipitation anomalies averaged over the Niño4 region (160°E–150°W, 5°S–5°N). The North Pacific Meridional Mode (NPMM) index is defined as SST anomalies averaged over 160°–120°W, 10°–25°N49. The NPMM SST anomaly index reproduces the original maximum covariance analysis (MCA)-based NPMM index with a correlation of ~0.9549. The Tropical North Atlantic (TNA) index is defined as SST anomalies averaged over 55°–15°W, 5°–25°N50. The Indian Ocean Basin (IOB) mode index is defined as SST anomalies averaged over 40°–100°E, 20°S–20°N51. The Indian Ocean Dipole (IOD) mode index is defined as SST anomalies averaged over 50°–70°E, 10°S–10°N minus those averaged over 90°–110°E, 10°S–0°N20. The North Atlantic Oscillation (NAO) index is defined as the principal-component (PC) time series associated with the leading Empirical Orthogonal Function (EOF) of area-weighted SLP anomalies over the North Atlantic sector 90°W–40°E, 20°–80°N52. Wavelet analysis was used to explore the different climate modes' time-frequency characteristics and identify dominant periodicities53. A continuous wavelet transform was performed using the Morlet wavelet, which offers an optimal balance between frequency and time localization, making it ideal for capturing oscillatory behavior. The analysis was conducted on normalized Niño3.4 SST and NAO indices, with the resulting wavelet power spectra computed for each ensemble member, followed by calculating the ensemble-mean of the spectra (Fig. 1b, d) to highlight changes in dominant frequencies over time. Statistical significance was assessed using Monte Carlo simulations: Niño3.4 SST was tested against an AR(1) process with a lag(−1 month) autocorrelation coefficient of 0.9, while the NAO index was tested against a white-noise process. The power spectral density (PSD) of the various climate mode indices (the four ensemble members for P2 were concatenated, leading to 84 years of monthly data for P2 but only 21 years of monthly data for P1) was calculated using the Multi-Taper Method (MTM) with 3 tapers and nfft = 256 for P1 and nfft = 1024 for P254. A Lorentzian background spectrum null hypothesis was chosen for the SST modes. The significance level of spectral peaks was therefore determined from the respective PSD percentile at each frequency of PSDs calculated with the same MTM method for 10,000 discrete AR(1) processes with (i) the same lag(−1 month) autocorrelation, (ii) the same variance, and (iii) the same data length as the respective index that is tested against. A white noise null hypothesis was chosen for the NAO atmospheric mode. Here, the significance level of spectral peaks was determined from the respective PSD percentile at each frequency of PSDs calculated with the same MTM method, generated from 10,000 realizations of white noise time series with (i) the same variance and (ii) the same data length as the index that is tested against. We assessed ENSO regularity by calculating the sample entropy (SampEn) of the Niño3.4 SST anomaly index, a nonlinear metric that measures how predictable a time series is55. SampEn is an alternative to approximate entropy (ApEn56) for quantifying the randomness of a time series; unlike ApEn, it excludes self-matches, thereby reducing bias and improving consistency across different data segments57. Compared to an ENSO regularity metric that is defined by the sharpness of the ENSO spectral peak58, SampEn is less sensitive to record length, spectral windowing, and other frequency-domain constraints. Lower SampEn values indicate more regular (periodic) behavior, whereas higher values indicate more irregular behavior. Given a time series \(\{{x}_{1},{x}_{2},\ldots,{x}_{N}\}\), SampEn estimates the negative natural logarithm of the conditional probability that two sequences of length \(m\) that match within tolerance \(r\) will also match for \(m+1\) points, while excluding self-matches to reduce bias and improve consistency. We used an embedding dimension \(m=2\) (the dimensionality of the ENSO RO model) and a tolerance \(r=0.2\sigma\), where \(\sigma\) is the standard deviation of the time series. The SampEn metric is defined as: where \({A}^{m}\left(r\right)\) the probability that two sequences of length m + 1 are similar (matches), \({B}^{m}\left(r\right)\) is the probability that two sequences of length \(m\) are similar (candidates). Since the number of matches is always less than or equal to the number of possible vectors, the ratio \({A}^{m}\left(r\right)/{B}^{m}\left(r\right)\) is a conditional probability less than unity, ensuring SampEn is positive. This method provides a single diagnostic value characterizing the regularity of the ENSO time series, facilitating consistent comparison across observations and simulations under different climate scenarios. The observed SampEn effectively captures the decadal variations in ENSO forecast skill, including the modest decrease in predictability after the 2000s (black curve in Fig. The Recharge-Oscillator (RO) model is a widely recognized framework for diagnosing and understanding ENSO dynamics in observations and climate models9,13. The RO model captures the oscillatory behavior of El Niño and La Niña events through two coupled equations describing the evolution of ENSO SST anomalies (\({T}_{{{\rm{ENSO}}}}\)) and equatorial Pacific zonal-mean thermocline depth anomalies (\(h\)). In its linear form, the model is expressed as: (4), \({{{\bf{L}}}}_{{{\rm{ENSO}}}}=\left(\begin{array}{cc}R & {F}_{1}\\ -{F}_{2} & -\varepsilon \end{array}\right)\), where \(R\) represents the SST anomaly growth rate, collectively describing the Bjerknes feedback, \(\varepsilon\) denotes ocean damping rate related to the energy leakage at the western boundary and mixing, \({F}_{1}\) represents the effectiveness of the heat content discharge–recharge in controlling SST anomalies, and \({F}_{2}\) represents the discharge–recharge efficiency; \({{{\boldsymbol{\xi }}}}_{{{\rm{ENSO}}}}\) and \({{{\boldsymbol{\xi }}}}_{{{\rm{h}}}}\) are the residual terms encompassing non-resolved nonlinear processes and stochastic noise. The complex eigenvalue of the RO model operator \({{{\bf{L}}}}_{{{\rm{ENSO}}}}\) is the Bjerknes-Wyrtki-Jin (BWJ) index for the ENSO growth rate and periodicity and can be written as: where the real part of the index is referred to as the ENSO Bjerknes stability (BJ) index in ref. Due to the strong seasonal dependence of ENSO, we explicitly incorporate seasonality by estimating the parameters as: The parameters of the operators are estimated by multivariate linear regression. We applied the RO model fitting to each ensemble member of the AWI-CM3 simulation using a 21-year moving window. To understand the climate mode interactions, we use a modified version of a recently developed extended Recharge Oscillator (XRO), which allows for two-way interactions between ENSO and the other modes24, which consists of a recharge oscillator model for ENSO coupled to seasonally-modulated stochastic-deterministic models for the other climate modes25: The linear dynamical operator L contains four submatrices, organized as follows: where the linear operator submatrix \({{{\bf{L}}}}_{{{\rm{ENSO}}}}\) describes the ENSO internal recharge-discharge dynamics, \({{{\bf{L}}}}_{{{\rm{M}}}}\) represents the internal processes and interactions among the other climate modes; \({{\bf{C}}}\) are coupling submatrices, with \({{{\bf{C}}}}_{2}\) describing the impact of ENSO on other climate modes and \({{{\bf{C}}}}_{1}\) describing the feedback of other modes on ENSO. Similar to the RO model fit, we explicitly incorporate seasonality by estimating the XRO parameters with Eq. There are a total of 12 noise parameters, i.e., a noise amplitude and decorrelation time scale for each of the 6 state variables in the system. The decorrelation time scales are estimated as \({{{\boldsymbol{r}}}}_{{{\boldsymbol{\xi }}}}=-{\mathrm{ln}}({{{\boldsymbol{a}}}}_{1})/\delta t\), where \({{{\boldsymbol{a}}}}_{1}\) are the lag(−1 month) autocorrelations of the residual of the XRO fit. The range of observed noise time scales \({{{\boldsymbol{r}}}}_{{{\boldsymbol{\xi }}}}^{-1}\) are between 0.25 ~ 0.70 months. Floquet theory is used to determine the stability of the coupled system with an annual cycle basic state. We determine the Floquet ENSO growth rate and periodicity using the RO model operator (\({{{\bf{L}}}}_{{{\rm{ENSO}}}}\)) or XRO model operator \({{\bf{L}}}\) via Floquet exponent analysis31. The Floquet exponents are determined numerically by integrating \(d{{\boldsymbol{Q}}}/{dt}={{\boldsymbol{L}}}\cdot {{\boldsymbol{Q}}}\) over 1 year with a time step of 3.65 days from an initial condition \({{\boldsymbol{Q}}}\left(0\right)={{\boldsymbol{I}}}\) to form a monodromy matrix \({{\boldsymbol{M}}}={{\boldsymbol{Q}}}\left(T\right),T=1\,{{\rm{year}}}\). To quantify changes in atmospheric noise amplitude, we define atmospheric noise as surface zonal wind variability that is independent of SST-related variability, following ref. Specifically, an EOF analysis was applied to monthly tropical SST anomalies to derive the leading modes of variability and their associated PCs. Multivariate linear regression was then performed, regressing the monthly surface zonal wind stress anomalies onto the first 15 leading PCs, which together account for approximately 75% of the variance in tropical SST anomalies. The time series of atmospheric noise was then defined as the residual wind stress, obtained from removing the SST-forced signals. Finally, the amplitude of noise is defined as the standard deviation of the residual wind stress. To assess possible physical processes that control the changes of ENSO growth rate, we calculate the ocean mixed layer heat budget using a partial flux form63,64: The anomalous heat flux term \({Q}^{\prime}\) is calculated as: where \(H\) is the ocean mixed layer depth (50 m), \(\rho\) is the seawater density (1025 kg m−3), \({C}_{p}\) is the specific heat capacity of seawater (3994 J kg−1 K−1), and \({Q}_{{{\rm{net}}}}^{{\prime} }\) represents the net surface heat flux, consisting of four components: representing shortwave radiation, longwave radiation, latent heat flux, and sensible heat flux anomalies, respectively. The shortwave radiation transmitted through the bottom of the mixed layer (\({Q}_{{{\rm{bot}}}}^{{\prime} }\)) is parameterized following ref. Taking the volume-average by integrating both hand sides of Eq. (9) from the ocean mixed layer depth to the ocean surface and spatially over the eastern equatorial Pacific box where the SST variability of ENSO is dominant, the heat budget equation can be written as follows: where the angle brackets denote volume/area-average variables over the eastern Pacific box (180°–90°W, 6°S–6°N), \({T}_{{{\rm{bot}}}}^{{\prime} }\) and \({\overline{w}}_{{{\rm{bot}}}}\) the anomalous temperature and climatological vertical velocity at the bottom interface of the ocean mixed layer, respectively. We group the various feedbacks on the right-hand side of Eq. (9) as thermal damping by the net surface heat flux (TD), dynamic damping by mean horizontal currents (DD), thermocline feedback (TH), zonal advective feedback (ZA), meridional advective feedback and vertical advective feedback (combined as Ekman feedback, EK), nonlinear dynamic heating (NDH), and sub-grid scale contributions (SG), respectively. Our definitions of DD and TH represent the advection of anomalous temperature by mean currents via lateral and bottom interfaces, respectively. Next, we apply the following linear relationship between various oceanic processes with \(\left\langle {T}^{{\prime} }\right\rangle\) and h9 and replacing \(\langle {T}^{{\prime} }\rangle\) with \({T}_{{{\rm{e}}}}\) for simplicity: Substituting these into the ENSO mixed-layer temperature equation, the equation reduces to: From this, the total SST growth rate \(R\) and phase-transition rate \({F}_{1}\) are expressed as: To estimate these parameters, we account for seasonality explicitly outlined in Eq. (6) using multivariate linear regression. Ensemble mean parameters are calculated by performing regression on concatenated results from all ensemble members. These derived parameters capture the contributions of each process and provide a comprehensive understanding of ENSO feedbacks and their contribution to the ENSO growth rate. The \({R}_{{{\rm{NDH}}}}\) and \({R}_{{{\rm{Res}}}}\) terms are combined as \({R}_{{{\rm{AllRes}}}}\), which consists of tropical instability waves and other unresolved processes. We have also tested our analysis using different ocean mixed layer depths (30 m, 40 m, and 50 m) and alternative definitions of the eastern Pacific box (e.g., the Niño3.4 region). The results remain quantitatively similar and do not affect our conclusions. The ENSO wind stress structure affects both the growth rate and periodicity of ENSO9,36. Here, we determine the ENSO wind stress pattern using linear regression of monthly zonal wind stress anomalies \(({\tau }_{x}^{{\prime} })\) onto the monthly ENSO SST anomaly index. 36, the meridional width of \({\tau }_{x}^{{\prime} }\) is defined as the latitudinal distance between the meridional boundaries, the nearest local minima of \({\tau }_{x}^{{\prime} }\) on either side of the equator, based on the meridional profile averaged over 160°E–150°W (Fig. The eastern boundary is identified as the zero-crossing point in the zonal profile; the western boundary is defined as the local minimum within the region from 140°E to 190°E; and zonal centroid location as the midpoint between zonal boundaries (Supplementary Fig. Under the assumption that the zonal wind stress anomaly \({\tau }_{x}^{{\prime} }\) has a Gaussian meridional structure, the RO \(h\) equation can be reformulated to explicitly incorporate the wind stress structure36: where \({\varepsilon }_{m}\) is a linear mechanical damping coefficient (2.5 yr)−1, the parameter \(\alpha\) characterizes the meridional structure of \({\tau }_{x}^{{\prime} }\), such that the meridional width of \({\tau }_{x}^{{\prime} }\) is proportional to \(1/\sqrt{\alpha }\), and \({x}_{c}\) denotes the nondimensional location of the center of wind stress anomalies along the equator measured from the western boundary (with \({x}_{c}\approx 0.35\), Supplementary Fig. A meridional widening of \({\tau }_{x}^{{\prime} }\) leads to a reduction in \(\alpha\), thereby weakening the damping from thermocline adjustment (\(\varepsilon\), Fig. 2h) and recharge/discharge efficiency (\({F}_{2}\), Fig. These changes enhance the growth rate and increase the periodicity of ENSO following the RO eigenvalue Eq. Phase synchronization characterizes how interactions between two different processes can lead to synergistic dynamical phase behavior (e.g., the temporal alignment of respective local maxima or minima). A special case of phase-synchronization is frequency-locking31, where two systems with normally independent oscillatory behavior develop a joint frequency and locked dynamics. In general, for phase-synchronized systems, the difference between the phases of the interacting processes, or multiples thereof, is statistically distinguishable from a uniform white noise process. In particular, phase synchronization can emerge in nonlinear dynamical systems, which include product terms between two signals. Practically, phase relationships between two processes, which are represented by their timeseries \(x\left(t\right)\) and \(y\left(t\right)\) can be calculated using their complex analytical signals66,67 \({z}_{1}\left(t\right)={A}_{1}\left(t\right){e}^{i{{{\rm{\phi }}}}_{1}\left(t\right)}=x\left(t\right)+i{x}_{H}\left(t\right),\) and \({z}_{2}\left(t\right)={A}_{2}\left(t\right){e}^{i{{{\rm{\phi }}}}_{2}\left(t\right)}=y\left(t\right)+i{y}_{H}\left(t\right)\), where \({x}_{H}\left(t\right),\) \({y}_{H}\left(t\right),\) represent the Hilbert transforms of \(x\left(t\right),y(t)\) and \({{{\rm{\phi }}}}_{1}\left({{\rm{t}}}\right),{{{\rm{\phi }}}}_{2}({{\rm{t}}})\) are the respective phases of the signal. The only requirement for the two signals \(x\left(t\right),y(t)\) is that they satisfy Bedrosian's Product Theorem68. This can be achieved by appropriate band-pass filtering. For two un-synchronized (decoherent) processes, we would expect (1) the time evolution of the unwrapped phases \(\delta {{{\rm{\phi }}}}_{\left\{{{\rm{n}}},{{\rm{m}}}\right\}}\) to essentially increase, without any long periods of phase “friction”, which can be characterized as consistent plateaus of near constant \(\delta {{{\rm{\phi }}}}_{\left\{{{\rm{n}}},{{\rm{m}}}\right\}}\); (2) the probability distribution of \(\delta {{{\rm{\phi }}}}_{\left\{{{\rm{n}}},{{\rm{m}}}\right\}}\) (without unwrapping procedure) to be indistinguishable from a uniform distribution between 0–360 degrees66,69. Respectively, for m:n synchronized signals, (1) we would expect to see long periods of little growth in unwrapped phase differences; (2) the frequency histogram of \(\delta {{{\rm{\phi }}}}_{\left\{{{\rm{n}}},{{\rm{m}}}\right\}}\) (without unwrapping procedure) across all angles (0–360°) should deviate statistically (e.g., Kolmogorov-Smirnov test) from a uniform distribution. We use this approach to calculate the 1:1 difference of unwrapped phases between timeseries characterizing different climate modes simulated by the AWI-CM3 model. All time series were bandpass filtered between 2.5 months and 10 years using a Butterworth filter prior to the analysis to focus on annual to interannual variability and to satisfy Bedrosian's theorem. We use the standard deviation of the phase difference PDF, \(\sigma [{{\rm{PDF}}}(\delta {\Phi }_{1,1})]\), as a measure of the synchronization strength. If there is no synchronization, the phase differences are uniformly distributed, yielding a flat PDF and \(\left.\sigma [{{\rm{PDF}}}(\delta {\Phi }_{1,1})]\right)\approx 0\) (Supplementary Fig. In contrast, if there is strong synchronization, the phase differences concentrate around a preferred value (−π/2 for the example shown in Supplementary Fig. 8c, d), producing a sharply peaked PDF, resulting in a larger \(\sigma [{{\rm{PDF}}}(\delta {\Phi }_{1,1})]\) (Supplementary Fig. The Fisher z-transformation was used to test statistical significance of the correlation differences as follows: where \({r}_{1}\) and \({r}_{2}\) are the correlation coefficients, \({n}_{1}\) and \({n}_{2}\) are the sample sizes of the first and second group samples. The absolute value |Z| is then compared against a critical value from the t-distribution for a two-tailed test. We rejected the null hypothesis that the two correlations are not significantly different at a 90% confidence level if |Z| exceeds the critical value. The Student's t test was used to test statistical significance of the regression coefficient. Given small sample sizes, we apply a bootstrap method to assess whether changes in ENSO regularity are statistically significant. Specifically, the 21-year moving-window values of ENSO regularity are randomly resampled with replacement to generate 10,000 synthetic realizations. From these realizations, we compute the standard deviation of the resulting distribution of mean values. Selected variables of the AWI-CM3 simulations are available on the ICCP Climate Data Website at https://climatedata.ibs.re.kr/data/papers/stuecker-et-al-2025-nature-communications/). 45) data at https://www.metoffice.gov.uk/hadobs/hadisst/; the ERSSTv5 (ref. 46) data at https://psl.noaa.gov/data/gridded/data.noaa.ersst.v5.html; the COBE2 (ref. 47) data at https://ds.data.jma.go.jp/tcc/tcc/products/elnino/cobesst_doc.html; ERA5 monthly reanalysis (ref. The data to generate the main figures in this study have been deposited in the Zenodo database (ref. The sample entropy (SampEn) code (ref. 71) is publicly available at https://github.com/senclimate/xentropy. The XRO model code (ref. 72) is publicly available at https://github.com/senclimate/XRO. The phase synchronization code (ref. 73) is publicly available at https://github.com/senclimate/xphasesync. Other data processing and analysis code is available upon request from the author S.Z. & Cai, W. El Niño Southern Oscillation in a Changing Climate 506 (John Wiley and Sons, Inc., 2020). Timmermann, A. et al. El Niño-Southern Oscillation complexity. Cai, W. et al. Changing El Niño–Southern Oscillation in a warming climate. Wengel, C. et al. Future high-resolution El Niño/Southern Oscillation dynamics. Increased El Niño frequency in a climate model forced by future greenhouse warming. Kohyama, T., Hartmann, D. L. & Battisti, D. S. Weakening of nonlinear ENSO under global warming. The future of the El Niño–Southern Oscillation: using large ensembles to illuminate time-varying responses and inter-model differences. Evaluating climate models with the CLIVAR 2020 ENSO metrics package. In El Niño Southern Oscillation in a Changing Climate Geophysical Monograph Series (eds McPhaden, M. J., Santoso, A. Jin, F.-F., Kim, S. T. & Bejarano, L. A coupled-stability index for ENSO. Lu, B., Jin, F.-F. & Ren, H.-L. A coupled dynamic index for ENSO periodicity. Fedorov, A. V. & Philander, S. G. H. Is El Niño changing?. Jin, F.-F. An equatorial ocean recharge paradigm for ENSO. Tropical ocean-atmosphere interaction, the Pacific Cold Tongue, and the El Niño-Southern Oscillation. & Jin, F. F. An eigen analysis of the interdecadal changes in the structure and frequency of ENSO mode. & Wang, B. Interdecadal change of the structure of the ENSO mode and its impact on the ENSO frequency. Stuecker, M. F., Timmermann, A., Jin, F.-F., McGregor, S. & Ren, H.-L. A combination mode of the annual cycle and the El Niño/Southern Oscillation. Stuecker, M. F. The climate variability trio: stochastic fluctuations, El Niño, and the seasonal cycle. Stein, K., Timmermann, A., Schneider, N., Jin, F.-F. & Stuecker, M. F. ENSO seasonal synchronization theory. Saji, N. H., Goswami, B. N., Vinayachandran, P. N. & Yamagata, T. A dipole mode in the tropical Indian Ocean. Tropical Atlantic sea surface temperature variability and its relation to El Niño-Southern Oscillation. Fraedrich, K. An ENSO impact on Europe? Cai, W. et al. Pantropical climate interactions. Explainable El Niño predictability from climate mode interactions. Stuecker, M. F. et al. Revisiting ENSO/Indian Ocean Dipole phase relationships. Changes of ENSO stability due to greenhouse warming. et al. Earth's future climate and its variability simulated at 9 km global resolution. Swain, D. L., Langenbrunner, B., Neelin, J. D. & Hall, A. Increasing precipitation volatility in twenty-first-century California. Yun, K.-S. et al. Increasing ENSO–rainfall variability due to changes in future tropical temperature–rainfall relationship. Hopf, E. Abzweigung einer periodischen Lösung von einer stationären eines Differentialsystems. Jin, F.-F., Neelin, J. D. & Ghil, M. El Niño/Southern Oscillation and the annual cycle: subharmonic frequency-locking and aperiodicity. B., Tung, W. W. & Rao, N. Noise-induced Hopf-bifurcation-type sequence and transition to chaos in the lorenz equations. Juel, A., Darbyshire, A. G. & Mullin, T. The effect of noise on pitchfork and Hopf bifurcations. Capotondi, A., Sardeshmukh, P. D. & Ricciardulli, L. The nature of the stochastic wind forcing of ENSO. Rodgers, K. B. et al. Ubiquity of human-induced changes in climate variability. Stuivenvolt-Allen, J., Fedorov, A. V., Fu, M. & Heede, U. Widening of wind stress anomalies amplifies ENSO in a warming climate. Wang, S. et al. El Niño/Southern Oscillation inhibited by submesoscale ocean eddies. An, S. I. Interannual variations of the Tropical Ocean instability wave and ENSO. Hasselmann, K. Stochastic climate models Part I. Fereday, D. R., Chadwick, R., Knight, J. R. & Scaife, A. Tropical rainfall linked to stronger future ENSO-NAO teleconnection in CMIP5 models. Drouard, M. & Cassou, C. A modeling- and process-oriented study to investigate the projected change of ENSO-forced wintertime teleconnectivity in a warmer world. Increased occurrences of consecutive La Nina events under global warming. Stuecker, M. F., Jin, F.-F. & Timmermann, A. El Niño-Southern Oscillation frequency cascade. & Jin, F.-F. Predictability of coupled processes. In Predictability of Weather and Climate (eds Palmer, T. & Hagedorn, R.) Ch. Global analyses of sea surface temperature, sea ice, and night marine air temperature since the late nineteenth century. Huang, B. et al. Extended Reconstructed Sea Surface Temperature, Version 5 (ERSSTv5): upgrades, validations, and intercomparisons. Hirahara, S., Ishii, M. & Fukuda, Y. Centennial-scale sea surface temperature analysis and its uncertainty. Richter, I., Stuecker, M. F., Takahashi, N. & Schneider, N. Disentangling the North Pacific Meridional Mode from tropical Pacific variability. Enfield, D. B., Mestas-Nuñez, A. M., Mayer, D. A. & Cid-Serrano, L. How ubiquitous is the dipole relationship in tropical Atlantic sea surface temperatures?. Xie, S.-P. et al. Indian ocean capacitor effect on Indo–Western Pacific climate during the summer following El Niño. Hurrell, J. W. & Deser, C. North Atlantic climate variability: the role of the North Atlantic Oscillation. Torrence, C. & Compo, G. P. A practical guide to wavelet analysis. Spectrum estimation and harmonic analysis. Richman, J. S. & Moorman, J. R. Physiological time-series analysis using approximate entroy and sample entropy. Pincus, S. M. Approximate entropy as a measure of system complexity. Berner, J., Christensen, H. M. & Sardeshmukh, P. D. Does ENSO regularity increase in a warming climate?. Zhao, S., Jin, F. F. & Stuecker, M. F. Understanding lead times of warm-water-volumes to ENSO sea surface temperature anomalies. Lou, J., Newman, M. & Hoell, A. Multi-decadal variation of ENSO forecast skill since the late 1800s. Ehsan, M. A., L'Heureux, M. L., Tippett, M. K., Robertson, A. W. & Turmelle, J. Real-time ENSO forecast skill evaluated over the last two decades, with focus on the onset of ENSO events. Persistently active El Nino-Southern Oscillation since the Mesozoic. The role of zonal advection feedback in phase transition and growth of ENSO in the Cane-Zebiak model. Kim, S. T. & Jin, F.-F. An ENSO stability analysis. Part I: results from a hybrid coupled model. & Simpson, J. J. Irradiance measurements in the upper ocean. Rosenblum, M. G. et al. Synchronization in noisy systems and cardiorespiratory interaction. Gabor, D. Theory of communication. Honari, H., Choe, A. S. & Lindquist, M. A. Evaluating phase synchronization methods in fMRI: a comparison study and new approaches. Pikovsky, A., Rosenblum, M. & Kurths, J. Phase synchronization in regular and chaotic systems. Source data for “Global climate mode resonance due to rapidly intensifying El Niño-Southern Oscillation” [Data set]. Zhao, S. xentropy: entropy-based measures for time series analysis with NumPy and Xarray, version 0.1.0. Zhao, S. Extended nonlinear recharge oscillator (XRO) model for ‘Explainable El Niño predictability from climate mode interactions'. Zhao, S. xphasesync: phase synchronization analysis of two time series using Numpy and Xarray. This material is based upon work supported by the U.S. Department of Energy, Office of Science, Office of Biological and Environmental Research, under Award Number DE-SC0025595. This report was prepared as an account of work sponsored by an agency of the United States Government. Neither the United States Government nor any agency thereof, nor any of their employees, makes any warranty, express or implied, or assumes any legal liability or responsibility for the accuracy, completeness, or usefulness of any information, apparatus, product, or process disclosed, or represents that its use would not infringe privately owned rights. Reference herein to any specific commercial product, process, or service by trade name, trademark, manufacturer, or otherwise does not necessarily constitute or imply its endorsement, recommendation, or favouring by the United States Government or any agency thereof. The views and opinions of authors expressed herein do not necessarily state or reflect those of the United States Government or any agency thereof. This work was supported by the Institute for Basic Science (IBS) under IBS-R028-D1. The AWI-CM3 simulations were conducted on the IBS/ICCP supercomputer “Aleph,” 1.43 petaflops high-performance Cray XC50-LC Skylake computing system with 18,720 processor cores, 9.59 PB storage, and 43 PB tape archive space. We also acknowledge the support of KREONET. and T.J. received support from the EERIE project (Grant Agreement No. 101081383) funded by the European Union. Views and opinions expressed are, however, those of the author(s) only and do not necessarily reflect those of the European Union or the European Climate Infrastructure and Environment Executive Agency (CINEA). Neither the European Union nor the granting authority can be held responsible for them. and F.-F.J. received support from the NOAA Climate Program Office's Modeling, Analysis, Predictions, and Projections (MAPP) Program Grant NA23OAR4310602 and the U.S. National Science Foundation Grant AGS-2219257. This is IPRC publication 1645 and SOEST contribution 11996. These authors contributed equally: Malte F. Stuecker, Sen Zhao. Department of Oceanography, University of Hawaiʻi at Mānoa, Honolulu, USA International Pacific Research Center, University of Hawaiʻi at Mānoa, Honolulu, USA Malte F. Stuecker & Fei-Fei Jin Department of Atmospheric Sciences, University of Hawaiʻi at Mānoa, Honolulu, USA Sen Zhao (赵森) & Fei-Fei Jin Center for Climate Physics, Institute for Basic Science, Busan, Republic of Korea Axel Timmermann, Sun-Seon Lee & Ja-Yeon Moon Pusan National University, Busan, Republic of Korea Axel Timmermann, Sun-Seon Lee & Ja-Yeon Moon Alfred Wegener Institute, Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany Rohit Ghosh & Thomas Jung Department of Physics and Electrical Engineering, University of Bremen, Bremen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conducted most of the analysis in discussion with M.F.S. Additional analysis contributions were from A.T., M.F.S., and R.G. wrote the first manuscript draft. J.-Y.M., S.-S.L., and A.T. conducted the additional ensemble experiments. Correspondence to Malte F. Stuecker or Axel Timmermann. The authors declare no competing interests. Nature Communications thanks Muhammad Azhar Ehsan, Minmin Fu, and the other, anonymous, reviewer for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Stuecker, M.F., Zhao, S., Timmermann, A. et al. Global climate mode resonance due to rapidly intensifying El Niño-Southern Oscillation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251015230939.htm'>Tiny brain nanotubes found by Johns Hopkins may spread Alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 05:31:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at Johns Hopkins Medicine report that they have identified how mammalian brains build intricate networks of tiny tubes that move toxins in and out of brain cells, much like pneumatic tubes send items through systems in factories and stores. Their experiments, which used genetically modified mice and advanced imaging tools, were supported by the National Institutes of Health and published on Oct. 2 in Science. According to the team, the discovery could deepen scientific understanding of how Alzheimer's disease and other neurodegenerative disorders develop, offering potential pathways for new treatments. In the study, the scientists observed that these microscopic tubes, known as nanotubes, primarily formed to help neurons expel toxic small molecules such as amyloid-beta. "Cells have to get rid of toxic molecules, and by producing a nanotube, they can then transmit this toxic molecule to a neighbor cell," says corresponding author Hyungbae Kwon, associate professor of neuroscience at the Johns Hopkins University School of Medicine. "Unfortunately, this also results in spreading harmful proteins to other areas of the brain." Kwon notes that these insights could help scientists refine approaches to treating Alzheimer's and similar conditions. To explore the phenomenon, the researchers collected small brain tissue samples from healthy mice and examined them with high-resolution microscopy, allowing them to visualize the nanotubes in remarkable detail and track how they moved materials between neurons. They then compared these samples with brain tissue from mice genetically engineered to develop Alzheimer's-like amyloid buildup. By taking a closer look at human neurons (sampled with permission from a publicly available electron microscopy database), the scientists identified nanotubes with similar morphology forming between neurons in the same way that the laboratory mice developed them. "When designing a potential treatment based on this work, we can target how nanotubes are produced -- by either increasing or decreasing their formation -- according to the stage of the disease," Kwon says. Funding for this research was provided by the National Institutes of Health (DP1MH119428 and R01NS138176). Forget Fiber: The Surprising Foods That Actually Relieve Constipation, According to Science Memory Problems Have Nearly Doubled in Adults Under 40, Study Finds New Theory Suggests We've Been Looking for Dark Matter All Wrong Scientists Discover “Meal Memory” Neurons That Control When and How Much We Eat Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251015230949.htm'>Who or what dug Mars' mysterious gullies? The answer is explosive</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 04:17:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Yet some of the planet's strange surface features might seem to hint at it. Earth scientist Dr. Lonneke Roelofs of Utrecht University set out to study the origin of mysterious gullies carved into Martian dunes. In her laboratory experiments, she discovered that blocks of frozen carbon dioxide (CO2 ice) can actually dig these channels on their own. For years, scientists suspected CO2 ice might be responsible for the odd formations, but no one had been able to show it directly. Roelofs succeeded by creating gullies in the lab using CO2 ice blocks, replicating a natural process that doesn't occur on Earth and had never before been seen in action. Because Mars has a thin atmosphere and a sharp temperature difference between warm sand and the ice, the underside of these blocks instantly turns to gas in a process known as sublimation. Since gas takes up far more space than solid ice, the pressure builds rapidly, making the ice appear to "explode." "In our simulation, I saw how this high gas pressure blasts away the sand around the block in all directions," says Roelofs. As a result, the block digs itself into the slope and becomes trapped in a hollow surrounded by small ridges of settled sand. As the ice keeps vaporizing, it gradually slides downhill, carving a long, deep trench with ridges on either side. These artificial gullies match the ones seen on Mars almost exactly. Roelofs studies the geological forces that sculpt Mars' surface. "But the gullies from this research looked different," she explains. "Therefore, a different process was behind this, but which? That is what I set out to discover." Together with master student Simone Visschers, she traveled to the English city of Milton Keynes to solve the mystery behind these unusual sand gullies. "We tried out various things by simulating a dune slope at different angles of steepness. "After finding the right slope, we finally saw results. But how exactly do these blocks of ice form? In spring, this ice begins to warm up and sublimate. Why does this planet fascinate people so much? This zone lies exactly far enough from the Sun to make the presence of liquid water possible, which is a prerequisite for life. This allows you to pose slightly different questions, which in turn can deliver new insights for processes here on our planet." Note: Content may be edited for style and length. Forget Fiber: The Surprising Foods That Actually Relieve Constipation, According to Science Memory Problems Have Nearly Doubled in Adults Under 40, Study Finds New Theory Suggests We've Been Looking for Dark Matter All Wrong Scientists Discover “Meal Memory” Neurons That Control When and How Much We Eat Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251009033128.htm'>Harvard astrophysicist suggests mysterious interstellar object may be an alien probe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-16 03:49:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>On July 1, astronomers detected a strange, fast-moving object racing toward the Sun. Named 3I/ATLAS, this unusual traveler immediately stood out for one remarkable reason: its orbital path revealed that it had come from beyond our Solar System. This marked only the third time in history that scientists have identified an interstellar visitor entering our cosmic neighborhood. However, 3I/ATLAS exhibits such strange characteristics that some scientists have begun to wonder whether it might be something more unusual. Harvard astrophysicist Avi Loeb and his team recently posted a paper titled "Is the Interstellar Object 3I/ATLAS Alien Technology?" Loeb, a well-known and sometimes divisive figure in the scientific community, previously argued that the first observed interstellar object, 1I/ʻOumuamua, discovered in 2017, could have been an extraterrestrial spacecraft. Among several features he considers potentially artificial, Loeb points out that 3I/ATLAS follows an orbit that takes it unusually close to Venus, Mars, and Jupiter -- an alignment he finds intriguing. So it's not a stretch to think that alien civilizations -- if they exist -- would have launched their own galactic explorers. However, this brings us to a crucial question: short of little green men popping out to say hello, how would we actually know if 3I/ATLAS, or any other interstellar object, was an alien probe? Most things we see in our Solar System don't emit light of their own. Larger objects generally reflect more sunlight, so they are easier for us to see. So what we see tends to be larger comets and asteroid, especially farther from Earth. Our own Voyager probes are about ten meters in size (if we include their radio antennas). If an alien probe was similar, we probably wouldn't spot it until it was somewhere in the asteroid belt between Jupiter and Mars. If we did spot something suspicious, to figure out if it really were a probe or not we would look for a few telltales. First off, because a natural origin is most likely, we would look for evidence that no aliens were involved. However, we might also want to look for hints of alien origin. This is assuming the probe was still in working order, and not completely defunct. We might also look for signs of electrostatic discharge caused by sunlight hitting the probe. Another dead giveaway would be signs of maneuvering or propulsion. Not to brag, but Earth is genuinely the most interesting place in the Solar System -- we have water, a healthy atmosphere, a strong magnetic field and life. A probe with any decision-making capacity would likely want to investigate and collect data about our interesting little planet. Objects like 3I/ATLAS remind us that space is vast, strange, and full of surprises. For now, 3I/ATLAS is likely just an unusually fast, old and icy visitor from a distant system. But it also serves as a test case: a chance to refine the way we search, observe and ask questions about the universe. Forget Fiber: The Surprising Foods That Actually Relieve Constipation, According to Science Memory Problems Have Nearly Doubled in Adults Under 40, Study Finds New Theory Suggests We've Been Looking for Dark Matter All Wrong Scientists Discover “Meal Memory” Neurons That Control When and How Much We Eat Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            