
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-02</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08814-5'>Global impoverishment of natural vegetation revealed by dark diversity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 18:43:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement


Nature

                         (2025)Cite this article


Metrics details

Anthropogenic biodiversity decline threatens the functioning of ecosystems and the many benefits they provide to humanity1. As well as causing species losses in directly affected locations, human influence might also reduce biodiversity in relatively unmodified vegetation if far-reaching anthropogenic effects trigger local extinctions and hinder recolonization. Here we show that local plant diversity is globally negatively related to the level of anthropogenic activity in the surrounding region. Impoverishment of natural vegetation was evident only when we considered community completeness: the proportion of all suitable species in the region that are present at a site. To estimate community completeness, we compared the number of recorded species with the dark diversity—ecologically suitable species that are absent from a site but present in the surrounding region2. In the sampled regions with a minimal human footprint index, an average of 35% of suitable plant species were present locally, compared with less than 20% in highly affected regions. Besides having the potential to uncover overlooked threats to biodiversity, dark diversity also provides guidance for nature conservation. Species in the dark diversity remain regionally present, and their local populations might be restored through measures that improve connectivity between natural vegetation fragments and reduce threats to population persistence.

Direct detrimental effects of anthropogenic activity on the biodiversity of natural ecosystems have been extensively documented3,4. For example, conversion of natural forest into urban landcover5 or transformation of grassland into cropland6 causes conspicuous declines in biodiversity. Biodiversity may also decline in ecosystems that are not directly modified but occur in regions in which human activities have caused habitat fragmentation7 or exert diffuse effects on natural areas—through pollution, for example8. Although compelling case studies show the influence of human activities on surrounding natural vegetation, beyond a direct area of impact8,9,10, there is no empirical evidence demonstrating the generality of regional-scale anthropogenic effects on local biodiversity in natural vegetation. Comparisons of relatively undisturbed vegetation inside and outside protected areas have revealed no discernible differences in local biodiversity11, but this overlooks the possibility that biodiversity has declined systematically in both settings12,13. The lack of empirical evidence might stem from the masking effect of high variation in biodiversity across regions and along ecological gradients14,15,16. We hypothesize that anthropogenic impoverishment of natural ecosystems can be revealed by the dark diversity—species that are ecologically suitable and present in a region but currently absent from a given site2. Dark diversity allows estimation of community completeness, a biodiversity metric that represents the proportion of all suitable species in a region that are actually present at a site17. This metric is globally comparable because it accounts for natural variation in potential biodiversity. Estimating the ecological suitability of species that are absent from a site is challenging, but methodological advances offer a solution based on species co-occurrences18.

The notion of dark diversity aligns with Whittaker's classic alpha–beta–gamma diversity framework19—a cornerstone of modern biodiversity research (Fig. 1). In Whittaker's work, alpha diversity represented the number of species at a particular site, gamma diversity comprised all species found in the surrounding region and beta diversity described changes in community composition along environmental gradients. The dark diversity concept is taxon-oriented, because it considers the suitability of each absent species for a study site. When aggregated, alpha and dark diversity together constitute the site-specific species pool, which includes only those species from the region that are suitable for a given site on the basis of its ecological conditions. In this context, beta diversity, as first defined by Whittaker, can be articulated as the change in site-specific species pools within gamma diversity. This is sometimes referred to as ‘structured' beta diversity, whereas ‘unstructured' beta diversity represents the variation in species composition among sampled sites within an ecologically similar area20,21. The dark diversity concept enhances the alpha–beta–gamma framework by providing a site-specific toolbox that complements alpha diversity at a site with the set of suitable yet absent species (dark diversity), the biodiversity potential of the site (species pool size) and the degree to which this potential is realized (community completeness).

a, Data included a local study site where certain species were present, but many species sampled elsewhere in the region were absent. To estimate the probability that a species that is absent from the site but present in the region belongs to the dark diversity of the site, we used information about species co-occurrences at other sites in the region. b, We calculated an indicator matrix in which each present species indicated the ecological suitability of each absent species for the study site. We compared the observed number of co-occurrences with the number of co-occurrences expected at random (according to the hypergeometric distribution) and standardized the difference using the standard deviation from the hypergeometric distribution. c, By averaging across all observed species, each absent species was assigned a probability of belonging to the dark diversity for the study site. Consequently, the dark diversity was a fuzzy set to which species belonged to varying degrees. d, Several biodiversity metrics were characterized for each site in the region. Alpha diversity was the number of species recorded at the site, and gamma diversity was the total number of species recorded in a region. The size of dark diversity was estimated as the sum of the probabilities of absent species belonging to the dark diversity of the study site. Alpha and dark diversity together formed the site-specific species pool, and gamma diversity not falling into this category was considered the unsuitable part of gamma diversity; that is, belonging to the species pools of other sites. We investigated the percentage of the species pool that was present among the alpha diversity (community completeness) and the turnover of species pools in the region, expressed as the percentage of gamma diversity that was unsuitable for the study site (beta diversity).

Alpha diversity is the most commonly used biodiversity metric, but it depends on variation in natural biodiversity potential between regions (for example, boreal versus temperate regions; North America versus East Asia) and ecological conditions within regions (for example, wetlands versus forests; south-facing versus north-facing slopes). Speciation, large-scale dispersal, species sorting and stochastic variation have produced site-specific species pools of considerably different sizes22. Community completeness accounts for such variation by quantifying the extent to which the biodiversity potential (that is, the site-specific species pool) is realized locally17. Even in natural ecosystems, some suitable taxa might be absent owing to natural processes that cause local extinction or limit recolonization. Such limiting processes vary along environmental gradients, reflected in the global patterns of plant persistence strategies23 and interactions with other organisms; for example, seed predators24. Consequently, there is likely to be natural variation in community completeness across broad environmental gradients25. In addition, regions with high geodiversity or a mosaic of vegetation types (that is, high structured beta diversity) might have lower community completeness because the isolation of natural habitat fragments and the likelihood of local extinction increase26. Furthermore, climatic conditions leave some regions prone to extreme events, such as natural fire, that cause local species loss27,28. Nevertheless, in addition to natural variation, human activities might strongly influence community completeness by reducing the persistence of local populations; for example, by promoting highly competitive taxa (through eutrophication, for instance8) or by restricting mutualistic interactions (reducing pollinators, for instance29). Similarly, human activities might hinder the recolonization of suitable sites through habitat fragmentation7 and loss of seed-dispersing animals30.

To determine whether anthropogenic impoverishment of natural vegetation is a worldwide phenomenon, we established DarkDivNet, a global collaborative research network31. Using a standardized methodology, we assessed both the alpha and the dark diversity of vascular plants across 5,415 sites with relatively intact natural or semi-natural vegetation, in 119 regions, spanning a wide range of vegetation types and representative of most global climatic conditions on all vegetated continents (Extended Data Figs. 1 and 2).

In our study, ‘site' refers to a 100-m2 area in which vegetation was sampled, and ‘region' represents the surrounding area of approximately 300 km2. Each region encompasses at least 30 sites, representing the natural and semi-natural vegetation typical of the region. We first confirmed that the sampling area of 100 m2 provided highly similar estimates of dark diversity to those obtained from a considerably larger area of 2,500 m2 (Extended Data Fig. 3). We assessed alpha diversity as the number of all vascular plant species found at each site. To estimate dark diversity, we used a fuzzy set approach in which all species occurring in the region but absent from the site were assigned a probability of inclusion in the dark diversity on the basis of an established co-occurrence methodology18. The use of probabilities maximizes the amount of information used for estimating dark diversity. Specifically, co-occurrences were based on the species composition of 30 randomly selected sites in the region (Fig. 1a). Using a subset of regions in which 60 sites were available yielded highly similar outcomes, indicating that 30 sites were sufficient for estimating co-occurrence patterns among species (Extended Data Fig. 3). We estimated the degree to which each species present in a region but absent from a site co-occurred with species found at the site, and compared it with random expectation, mathematically described by the hypergeometric distribution (Fig. 1b). If an absent species co-occurred with a present species more than would be randomly expected, they probably shared ecological requirements, and the present species provided a positive indication of the site's suitability for the absent species. The overall suitability of the site for the absent species was estimated by averaging the suitability indications from all species present at the site (Fig. 1c). The magnitude of dark diversity at a site was then estimated as the sum of these suitability estimates (probabilities of absent species belonging to the dark diversity of the site, ranging between 0 and 1) across all absent species. The unsuitable fraction of gamma diversity reflects the species belonging to different site-specific species pools in the same region. Using alpha diversity, dark diversity and the unsuitable diversity found in the region, we calculated other biodiversity metrics for each site to have a full description of biodiversity (Fig. 1d): site-specific species pool size as the sum of alpha and dark diversity; gamma diversity as the total set of species found in a region (this value was the same for each site within a region); community completeness as the proportion of the site-specific species pool size represented by alpha diversity; and beta diversity as a quantification of the extent to which gamma diversity exceeds the site-specific species pool size (that is, the proportion of gamma diversity that is unsuitable for the study site and is more likely to be associated with different site-specific species pools in the region). In this way, we specifically quantified the ‘structured' beta diversity, or turnover in site-specific species pool composition due to environmental gradients. In the statistical analyses, community completeness and beta diversity were included as log-ratios (logit transformation of percentages) to improve the distribution of the data. We used two independent datasets (expert assessments and examination of species found in the close vicinity of the site) to ensure that the co-occurrence method provided consistent estimates of species suitability for dark diversity (Extended Data Fig. 2). We also determined that, for this particular dataset, the hypergeometric method outperformed an alternative approach—joint species distribution modelling32 (see Supplementary Methods).

The median community completeness of sites across all regions was 25% (95% confidence interval 15–46%), highlighting a frequent absence of suitable species despite their presence in surrounding regions (Fig. 2a). The existence of relatively high dark diversity is clearly a general phenomenon, but the large variation meant that sometimes much fewer species were present locally than might be expected from the specific site conditions. To understand how much variation in alpha diversity was explained by community completeness besides beta and gamma diversity, we used variation partitioning. We found that 33% (26–43%) of the variation in alpha diversity was explained by community completeness. Consequently, if human activities reduce the colonization and persistence of suitable species, resulting in lower community completeness, this could substantially affect alpha diversity. The largest proportion of variation in alpha diversity, 52% (40–61%), was explained by gamma diversity, reflecting the well-known match of local and regional diversity33, whereas 14% (9–21%) was explained by beta diversity, reflecting how gamma diversity is distributed across different site-specific species pools. The strong dependence of alpha diversity on regional richness is clearly sufficient to mask the negative effect of human activities on alpha diversity.

a, Relationship between community completeness in natural vegetation and the human footprint index in the surrounding area, defined by a radius of 300 km. The prediction line from a multiple linear regression model is shown with the 95% confidence intervals. Note that community completeness values on the y axis are back-transformed from the logit scale. The symbol tones indicate forest cover (0–100%). R² value of the model and two-tailed P value of the relationship are shown; n = 116 regions. The distribution of community completeness is shown in the histogram on the right (median, 25%). b, Left, model summaries linking community completeness to the human footprint index and its components across spatial scales. Human influence was averaged over various spatial scales around the study regions (radii 10 km, 50 km, 100 km, 200 km, 300 km and 400 km), and the respective models were compared using the Akaike information criterion (AIC). Filled symbols indicate significant relationships (P < 0.05), and the large symbol indicates the set of best significant models (ΔAIC < 2). Right, from the best model (the smallest scale at which ΔAIC < 2), the effect of the human footprint index or one of its components is shown as a standardized coefficient (dot) with a 95% confidence interval (CI; line); n = 116 regions. Filled symbols and bold confidence interval lines indicate significant effects. c, Map of sampling regions, with community completeness indicated by symbol size and the underlying map showing the global variation in the human footprint index34 (the highest value within each grid cell of around 0.25° × 0.25°). The inset shows part of Europe containing a large number of study regions. Triangles indicate regions in which only woody species were sampled. Symbol tones indicate the percentage of forests in regions.

We tested the hypothesis that impoverishment of natural vegetation is related to anthropogenic influence in the surrounding region by building a series of models with various biodiversity metrics (community completeness, alpha diversity, beta diversity, gamma diversity, dark diversity and species pool size) as response variables. To estimate the intensity of human activities in the surrounding regions, we used the human footprint index from 2018 (the year our sampling began)—a well-established cumulative metric of human influence34—along with all of its eight components, including human population density and various human infrastructure layers. We averaged human influence at various spatial scales around the study region (radii from 10 km to 400 km), because human influence can reach far from mapped features. For example, poaching and logging can occur tens of kilometres from human settlements35 and are facilitated by many unmapped ‘ghost roads' that start from documented roads and lead into natural areas36. Similarly, anthropogenic ignition of fires can occur hundreds of kilometres from main roads37. Aerial pollution is often deposited several hundreds of kilometres from its source35, and land use can change local climate over similar scales38. To account for the effects of natural processes on biodiversity (for example, geodiversity, habitat patchiness and likelihood of natural fires), we included in our statistical models variables describing climatic, soil and topographic conditions, which we derived from global GIS layers and summarized using four principal component axes. Using fivefold spatial block cross-validation, we determined that linear models produced lower prediction errors with test data, compared with nonlinear alternatives (around 20% versus 40%). We therefore used linear models in further analyses.

The human footprint index and community completeness exhibited a robust negative linear relationship (Fig. 2a), which was already significant when the average human footprint index within a 50-km radius around the site was used, but became even more pronounced when radii of 300 km or larger were considered (Fig. 2b and Extended Data Table 1). In the sampled regions with minimal human footprint index values (close to zero), an average of 35% of suitable species were found in the 100-m2 sites, but this proportion declined to less than 20% in regions with high human impact. However, there was still variation in community completeness at both the low and the high ends of the human footprint index, showing that sites do not respond uniformly. In contrast to community completeness, alpha diversity was not strongly related to the human footprint index, and nor were the other tested metrics, except beta diversity (Extended Data Fig. 4 and Extended Data Table 1). These results are consistent with our hypothesis that local biodiversity is lower in natural vegetation surrounded by regions with more human activity, but this effect was evident only when we considered community completeness. Raw estimates of alpha diversity were strongly influenced by the wide natural variation in diversity potential determined by the specific biogeographical history of each region. Our results were consistent for six of the eight individual components of the human footprint index: human population density, the extent of electric infrastructure, railways, roads, built environments and croplands all exhibited negative relationships with community completeness (Fig. 2b and Extended Data Table 2). The extent of pasture was an exception to this pattern, because it was not negatively related to community completeness. This could be due to the influence of semi-natural grasslands, in which long-term moderate human influence, including grazing of domestic animals, cultural burning and haymaking, has resulted in highly diverse and well-functioning ecosystems, exemplifying how certain human activities can actually promote native biodiversity39. We found that the effect of the human footprint index was strongest when averaged over a range of several hundred kilometres. Besides incorporating far-reaching human influence, larger scales might also more accurately capture cumulative human influence in a particular region over long time periods40. However, including in the model a variable representing change in the index between 2000 and 2013 did not reduce the Akaike information criterion (AIC) by more than two units, which suggests that anthropogenic effects have operated over longer timescales. To account for the effects of natural processes on community completeness, our models included environmental variables. We found that community completeness decreased along the first principal component (Extended Data Table 1). Thus, suitable species are more likely to fall into the dark diversity in regions characterized by acidic organic soils and higher precipitation (see correlations of principal component axes in Extended Data Fig. 5). Dark diversity, gamma diversity and species pool size increased along the first axis (representing higher soil carbon content, acidity and precipitation; Extended Data Table 1). Alpha and beta diversities showed no significant relationships with the environmental axes.

The negative effect of human activities on community completeness might be associated with several phenomena. Human activities might have led to the fragmentation or reduction of suitable habitats, resulting in smaller populations that are more susceptible to random extinction9. In addition, habitat loss is likely to have decreased connectivity between remaining patches of natural vegetation, making it difficult for species to move between areas41, and defaunation might have disrupted plant seed dispersal networks30. Beyond habitat loss, some anthropogenic disturbances, such as tree cutting, illegal harvesting of plants and human-induced wildfires, can cause local extinctions in natural vegetation10,42. Moreover, regional human impact can affect natural ecosystems through pollution from roads and other human infrastructure; eutrophication is the most serious threat to plant diversity, because it disproportionally favours a few competitively superior species at the expense of a greater number of other species8.

Using average human influence as an explanatory variable can mask differences between regions. For example, regions that comprise both highly modified areas (for example, cities) and nature reserves, as well as those experiencing moderate human influence throughout (for example, agricultural landscapes with smaller settlements), might both exhibit an intermediate level of average human influence. We therefore tested how the distribution of the human footprint index within regions affected community completeness. Notably, we found that community completeness had an even stronger negative relationship with anthropogenic influence when we used the 30% quantile of the human footprint index values found within regions (Extended Data Fig. 6). This result suggests that completeness is determined mainly by the extent to which the most natural areas in a region already experience human influence. The idea that 30% coverage of natural vegetation in a landscape supports the persistence of many specialist taxa was proposed previously43, and aligns with the global target of the Convention on Biological Diversity to protect 30% of land by the year 2030. Our results therefore underscore the importance of devising regional-scale conservation strategies that include maintaining well-preserved natural areas44.

The turnover of site-specific species pools within regions (structured beta diversity) was significantly positively associated with the human footprint index (Extended Data Fig. 4 and Supplementary Table 1). This might reflect a human preference for naturally diverse regions with a range of different resources45. Alternatively, human activities could have promoted plant diversity over millennia by expanding semi-natural habitats and modifying natural ecosystems39. Most components of the human footprint index generally exhibited similar relationships, except for the extent of navigable waterways and pastures, which were negatively related to beta diversity (Supplementary Table 1). It is likely that coastal and riverine regions, and those suitable for livestock grazing, naturally exhibit relatively low variation in vegetation types.

The finding that high human footprint index values in a region are associated with low community completeness persisted in several other robustness tests (Supplementary Methods). Statistical interactions between the human footprint index and environmental gradients did not improve the model. Because naturally high beta diversity might decrease community completeness owing to the spatial separation of ecologically similar sites, and because beta diversity was correlated with human influence, we used structural equation modelling to examine the direct and indirect effects of human influence on community completeness. The negative direct effect of the human footprint index on community completeness persisted even if there was an additional negative direct effect of beta diversity. In addition, the effect of the human footprint index on community completeness was consistent across sampling scales (2,500 m2 or twice as many sites for species co-occurrences), when we excluded alien or very rare species, when regions with only woody species records were included and when we considered the proportion of forest cover in regions. Community completeness was slightly lower in more forested regions. The most parsimonious explanation for this might be a scaling effect—fewer large plant individuals can fit into a fixed area46. We also examined the possible effect of geographically uneven sampling by selecting a single study region from each ecoregion (the anthropogenic effect was always negative), adding the European continent as a factor to the model (the negative relationship remained significant) and investigating model residuals (no significant spatial autocorrelation was apparent). Community completeness was slightly lower in Europe than in other regions, which could reflect a cumulative effect of long-term human influence40.

This global-scale study reveals general patterns, and linkage to specific drivers is based on ecological interpretation rather than experimentation. It is also clear that the human footprint index does not provide a proxy for all potentially important processes, such as the disruption of biotic interaction networks, increasingly frequent climate extremes or the habitat destruction and fragmentation caused by war. The plethora of processes affecting biodiversity certainly contributes to variation around the general trends revealed by our models. The significant relationships we identified apply to the sampled range of the human footprint index, whereas index values outside this range might produce different relationships. In addition, even if the uneven distribution of study regions did not produce an effect in statistical models, the under-representation of several parts of Africa, the Americas and Asia might mean that some human impacts on biodiversity were not well represented. Future work should examine the exact patterns and processes of natural vegetation impoverishment in these undersampled regions.

Our finding of a globally consistent negative relationship between human influence and local plant diversity in relatively natural vegetation is alarming, because plants form the foundation of all terrestrial ecosystems. Reduced community completeness indicates that many species present in the region do not inhabit suitable sites, and this can affect local ecosystem functioning47. Although vegetation functioning depends mainly on the traits of co-existing taxa, the presence of a larger proportion of suitable taxa increases the chance that essential functions are represented48. We also found that negative human influence was most evident when considered at a scale spanning several hundred kilometres; in other words, biodiversity in natural ecosystems is reduced far beyond human infrastructure. Therefore, conservation actions and land-use planning should consider not only the observed alpha diversity of a site, but also a broader regional context. Ecology has a rich history of conceptual frameworks for biodiversity across scales, such as species–area relationships49, alpha–beta–gamma diversity19, community saturation and assembly33 and the meta-community concept50. Building on this collective knowledge, the dark diversity concept offers a species-oriented toolkit for evaluating community patterns and explaining the underlying processes. By allowing the estimation of a site's biodiversity potential (site-specific species pool) and its realization (community completeness), it fosters the comparative study of biodiversity across regions, ecosystem types and taxonomic groups2. This improved understanding could help conservation biologists, land managers and policymakers to prevent further losses of biodiversity51. Moreover, while site-specific species pools are not depleted, dark diversity offers a narrow window of opportunity for restoration because it indicates which missing species are still regionally present52,53,54.

In 2018, we launched a global collaborative research consortium to sample both locally observed alpha diversity and dark diversity of terrestrial plant communities using a standardized methodology. A detailed sampling protocol was produced before fieldwork began31. Each study region covered an area of approximately 300 km2, defined by a circle of 20-km diameter with the available area influenced by geographical and practical limitations (coastline, private ownership and other access restrictions). This spatial scale was selected on the basis of the authors' expertise, in the expectation that it would incorporate areas with a relatively uniform biogeographical history while still exhibiting variation in natural vegetation. In addition, mechanisms of long-distance seed dispersal can operate at this scale55. In each region, we defined at least 30 sites, in which we sampled a 100-m2 (10 m × 10 m) area by recording all vascular plant species. Where feasible, we sampled more sites in the region to examine how sampling intensity might affect the results. The sites were selected to proportionally represent the typical natural vegetation types of the region without major human influence. These included semi-natural grasslands, representing habitats that have developed over thousands of years through grazing by domestic animals and mowing, and forests that had been managed with low intensity and had species composition and tree-layer structure similar to old-growth forests. Here we report the results from 5,415 sites in 119 regions for which sampling was completed by 1 February 2024 (Supplementary Table 2).

To assess whether dark diversity methods could predict species that were absent from the 100-m2 area but present in its immediate vicinity, and to estimate the effect of spatial scale on dark diversity, we selected one to three sites per region in which we sampled vascular plants in a 2,500-m2 (50 m × 50 m) area within which the 100-m2 area was nested. In four regions, sampling of the larger area was not possible or the large area had no new taxa, so these regions were omitted from the respective test. In addition, in 76 regions, we had sufficient expertise to assess which of the species found in the region were ecologically well-suited for a selected site (that is, belonging to the site-specific species pool). This information allowed us to test the applicability of dark diversity methods within our sampling framework (see below).

Biodiversity metrics were determined for each site in each region (Fig. 1). Alpha diversity A was defined as the number of vascular plant species found in the 100-m2 area describing a site (Fig. 1a). Dark diversity D was quantified for each site k by examining species co-occurrences within the surrounding region using the hypergeometric method, implemented in the R package DarkDiv (ref. 18). This technique uses information about how each species i that is absent from the study site but present in the surrounding region co-occurs with species j that is present at the study site. If an absent species co-occurs more frequently with observed species than it would do under random expectation, it is likely to belong to the dark diversity. The expected number of co-occurrences is mathematically defined by the hypergeometric distribution. For each pair of absent and present species, we compared the observed number of co-occurrences Mij with the expected value, which is defined as the mean of the hypergeometric distribution:

where ni and nj are the total number of occurrences of species i and j, respectively, and N is the total number of sites sampled in that region. The standardized effect size (SES) was used as an indicator of the suitability of absent species i for site k on the basis of co-occurrences with present species j (Fig. 1b), and was calculated as the difference between the observed and the expected numbers of co-occurrences divided by the standard deviation of the expected number of co-occurrences, as derived from the hypergeometric distribution:

We estimated the suitability of site k for all species i absent from the site but present in the region, by averaging suitability indicator values from all present species j using the number of species found in site k (nk):

The SESki values were subsequently transformed to a 0–1 scale by applying inverse probit transformation, which places the SESki value within the cumulative normal distribution function with mean = 0 and standard deviation = 1 (Fig. 1c):

This estimate expressed the probability that species i belonged to the dark diversity of site k. Our estimated dark diversity probabilities were supported by two independent tests, one investigating which absent species were found in the immediate vicinity of a site and another using expert assessment (Extended Data Fig. 2). We also considered how the suitability of absent species might be estimated using an alternative technique—joint species distribution modelling (JSDM)56 (Supplementary Methods).

Dark diversity size for a study site was the sum of the probabilities Pki of all locally absent species found elsewhere in the region (Fig. 1d). For co-occurrences, we always considered 30 sites (each described by a 100-m2 area) within the same region (Fig. 1a), which is the minimum number sampled and generally sufficient for the method18. For regions with more than 30 sampled sites, we used an iterative procedure, each time randomly selecting 30 sites for species co-occurrences. Dark diversity size in those regions was estimated as the median from 100 iterations. Similarly, estimates of gamma diversity G were obtained using iteration, taking the median cumulative species number from 30 sites in a region. To test whether 30 sites was sufficient to estimate the variation in regional richness, we estimated species richness with complete sample coverage using incidence-based extrapolation based on the Bernoulli product model57, implemented within the R iNEXT package58. Gamma diversity from 30 sites correlated strongly with the extrapolated value (Spearman r = 0.95; Extended Data Fig. 3a)

Using alpha, dark and gamma diversities for each site, we calculated: species pool size as the sum of alpha and dark diversity: P = A + D; community completeness as the percentage of alpha diversity among all suitable species for that site: C = A/(A + D) × 100%; and beta diversity as the percentage of gamma diversity belonging to other species pools in the region and unsuitable for the specific site: B = (G – A – D) / G × 100% (Fig. 1d). This metric is identical to Whittaker's effective turnover at the species pool level, expressed as a percentage rather than a ratio (G/P) – 1. In analyses, all biodiversity metrics were transformed to improve distributions: those based on counts or sums (alpha, dark and gamma diversity, species pool size) were log-transformed, and those based on percentages (community completeness and beta diversity) were logit transformed. To aid intuitive understanding, we show untransformed values on graph axes. Because several of the diversity metrics are either subsets or calculated from each other, it is expected that these are closely related. However, bivariate relationships between our study variables (Extended Data Fig. 7) showed that all metrics bear some independent information, and the variability among and within regions is large.

All of our biodiversity metrics depend on the sampling scheme, including characteristics such as sample area or number of sites. To investigate how much our biodiversity metrics change if using a larger sample area, we used 1–3 sites in each region where both 100-m2 and 2,500-m2 areas were sampled. Similarly, we examined the effect of using a larger number of sites to characterize co-occurrences; using 60 sites from 27 regions where they were available. Overall, global variation in our metrics was highly correlated regardless of sample area and the number of sites considered (Spearman correlation > 0.8; Extended Data Fig. 3b–k).

According to the DarkDivNet protocol, in very diverse tropical regions we only sampled woody vascular plant species. Although alpha diversity, dark diversity, species pool size and gamma diversity are evidently smaller when herbaceous species are omitted, community completeness and beta diversity should still be relatively comparable with other regions because these metrics are unitless. To ensure full comparability between biodiversity metrics, we used only the 116 regions in which all vascular plants were sampled in the main analyses, but repeated the main tests for community completeness with all 119 regions within robustness analyses (Supplementary Methods).

Alpha diversity can be seen as a subset of gamma diversity in which the species pool has been filtered according to beta diversity, and the realization of the species pool is defined by community completeness (Fig. 1). We examined how much of the variation in alpha diversity is determined by variation in gamma diversity, beta diversity (these two define the site-specific species pool size) and community completeness. We randomly selected one site from each region in order to have independent local and regional variables (gamma diversity is the same for all sites in a region). The contribution of each source of variation was calculated using hierarchical variation partitioning (function varpart in the vegan package59 in R). This procedure was repeated 100 times to obtain a median and confidence interval.

In further statistical analyses, we used the medians of biodiversity variables across sites per region. We related community completeness and other calculated biodiversity variables (alpha diversity, beta diversity, gamma diversity, dark diversity and species pool size) to the human footprint index from the year 201834. The index ranges from 0 to 50 and is calculated from eight components (human population density, electric infrastructure, railways, roads, navigable waterways, the extent of built-up land, pastures and croplands). The resolution of the human influence data layers was 100 m, and we calculated average values over various spatial extents around the centre of each region (radii 10 km, 50 km, 100 km, 200 km, 300 km and 400 km). The averaging did not include areas representing water bodies. Because all regions included at least some areas less affected by humans, the total range of the averaged human footprint index values used in our analyses was somewhat lower than the maximum value. To test how well our sampled regions captured global variation in the human footprint index, we generated 500 random points worldwide using the discrete global grid system (which maintains uniform point density across the globe). From random points, we omitted glaciated regions of Antarctica and Greenland. We averaged the human footprint index in the surroundings of these random points in the same manner as we did with our empirical data. This revealed a high degree of correspondence between the average human footprint index ranges around sampled and randomly generated points at different scales: at radii of 50 km (sampled range 1.1–25.4, random 0.0–24.5), 200 km (sampled range 0.3–20.7, random 0.0–20.7) and 400 km (sampled 0.2–17.7, random 0.1–16.6).

To account for natural processes affecting community completeness, we included environmental variables in the multiple linear regression models. We used mean annual temperature and annual precipitation from the CHELSA database (resolution 1 km)60,61, soil pH, organic carbon content, sand fraction proportion from SoilGrids (resolution 250 m)62 and the topographic ruggedness of the terrain (resolution 250 m)63. Environmental factors were averaged within a 100-km radius to describe the broader region and consolidated through principal component analysis (PCA). For PCA, variables with only positive values were log-transformed if this resulted in a distribution closer to normal, and all variables were standardized. We kept the four first principal components, which described more than 90% of the variation. The first component was positively correlated with soil organic carbon content, acidity and precipitation; the second with temperature; the third with soil sand content; and the fourth with topographic ruggedness (Extended Data Fig. 5).

We fitted both linear and nonlinear (generalized additive models, function gam in the R package; ref. 64) models, incorporating the 116 regions in which all vascular plants were sampled. The estimates of the human footprint index at the different spatial scales were inherently strongly related to each other. Therefore, we constructed models for each scale at which the human footprint index (or its components) was averaged. We examined which scales produced the best models (ΔAIC < 2) and selected the smallest scale, which is most directly related to the study region. We compared linear and nonlinear models using spatial block validation, implemented in the R package blockCV (ref. 65). We used fivefold cross-validation across hexagons (Extended Data Fig. 2). To estimate the variation in model predictive power we further implemented a bootstrap approach66 by selecting bootstrap samples within each fold and then performing cross-validation. We used the normalized root mean square error (normalized by minimum and maximum values) to compare the predictive error of linear and nonlinear models, and found that linear models had much lower error in test sets (around 20% of the range compared with around 40% of the range; see Extended Data Fig. 8). Linear models were therefore used as a more general option.

We report the results of the best linear model (the smallest spatial scale at which ΔAIC < 2) for each biodiversity metric and note significant relationships (P < 0.05). We used the variance inflation factor (VIF) to confirm that correlations between environmental gradients and human impact (Extended Data Fig. 5) were not confounding in the models (VIF < 2). We applied type III model testing. Consequently, the effect of human impact was tested only after the environmental effects were accounted for. We visualized the results of the fitted models in terms of how the predictor variable human footprint index affects the outcome of community completeness using the visreg function and package67 in R. Model summary tables can be found in Extended Data Tables 1 and 2.

Besides the human footprint index from 2018, we also examined whether including change in the human footprint index during recent years improved the model68. Specifically, we tested whether a model including human footprint index change yielded a lower AIC value (by more than two units) compared with the model without change. We derived the measure of human footprint index change from a source that used a consistent methodology69 during a temporal range 2000–2013. Change in human footprint index was quantified as log(human footprint index value from 2013/human footprint index value from 2000).

We tested whether community completeness is better described by certain quantiles of the human footprint index at different scales around study regions. Compared with the mean, considering quantiles allowed us to determine the extent to which it is important to maintain a certain proportion of area with lower human influence. We compared models incorporating as predictor variables the 10–90% quantiles of the human footprint index using AIC and recorded cases in which the quantiles yielded a better model than the mean (models with AIC lower by more than two units were considered superior).

We also tested the robustness of the relationship between community completeness and the human footprint index by looking at statistical interactions between human influence and the environment, indirect effects, the role of sampling scale, alien or rare species; by including areas in which only woody species were recorded and considering forest cover in regions; and by examining the effect of geographically uneven sampling (see Supplementary Methods).

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

All data supporting the findings of this study, along with the R scripts to handle them, can be found in Figshare: https://doi.org/10.6084/m9.figshare.25158059 (ref. 70). We also used published data for the human footprint index34,69; from the CHELSA database60,61 for annual mean temperature and annual precipitation; from SoilGrids62 for soil pH, organic carbon content and sand fraction proportion; and from the Geomorpho90m database63 for the topographic ruggedness of the terrain.

Ceballos, G. et al. Accelerated modern human-induced species losses: entering the sixth mass extinction. Sci. Adv. 1, e1400253 (2015).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Pärtel, M., Szava-Kovats, R. & Zobel, M. Dark diversity: shedding light on absent species. Trends Ecol. Evol. 26, 124–128 (2011).

PubMed 
    
                    Google Scholar 
                

Jaureguiberry, P. et al. The direct drivers of recent global anthropogenic biodiversity loss. Sci. Adv. 8, eabm9982 (2022).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Newbold, T. et al. Global effects of land use on local terrestrial biodiversity. Nature 520, 45–50 (2015).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Williams, N. S. G. et al. A conceptual framework for predicting the effects of urban environments on floras. J. Ecol. 97, 4–9 (2009).

MATH 
    
                    Google Scholar 
                

Le Provost, G. et al. Grassland-to-crop conversion in agricultural landscapes has lasting impact on the trait diversity of bees. Landsc. Ecol. 36, 281–295 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Haddad, N. M. et al. Habitat fragmentation and its lasting impact on Earth's ecosystems. Sci. Adv. 1, e1500052 (2015).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Stevens, C. J., Thompson, K., Grime, J. P., Long, C. J. & Gowing, D. J. G. Contribution of acidification and eutrophication to declines in species richness of calcifuge grasslands along a gradient of atmospheric nitrogen deposition. Funct. Ecol. 24, 478–484 (2010).


                    Google Scholar 
                

Chase, J. M., Blowes, S. A., Knight, T. M., Gerstner, K. & May, F. Ecosystem decay exacerbates biodiversity loss with habitat loss. Nature 584, 238–243 (2020).

ADS 
    PubMed 
    
                    Google Scholar 
                

Barlow, J. et al. Anthropogenic disturbance in tropical forests can double biodiversity loss from deforestation. Nature 535, 144–147 (2016).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Gray, C. L. et al. Local biodiversity is higher inside than outside terrestrial protected areas worldwide. Nat. Commun. 7, 12306 (2016).

ADS 
    PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Santangeli, A. et al. Mixed effects of a national protected area network on terrestrial and freshwater biodiversity. Nat. Commun. 14, 5426 (2023).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Jones, K. R. et al. One-third of global protected land is under intense human pressure. Science 360, 788–791 (2018).

PubMed 
    MATH 
    
                    Google Scholar 
                

Cai, L. et al. Global models and predictions of plant diversity based on advanced machine learning techniques. New Phytol. 237, 1432–1445 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Sabatini, F. M. et al. Global patterns of vascular plant alpha diversity. Nat. Commun. 13, 4683 (2022).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Fraser, L. H. et al. Coordinated distributed experiments: an emerging tool for testing global hypotheses in ecology and environmental science. Front. Ecol. Environ. 11, 147–155 (2013).

MATH 
    
                    Google Scholar 
                

Pärtel, M., Szava-Kovats, R. & Zobel, M. Community completeness: linking local and dark diversity within the species pool concept. Folia Geobot. 48, 307–317 (2013).


                    Google Scholar 
                

Carmona, C. P. & Pärtel, M. Estimating probabilistic site-specific species pools and dark diversity from co-occurrence data. Glob. Ecol. Biogeogr. 30, 316–326 (2021).

MATH 
    
                    Google Scholar 
                

Whittaker, R. H. Vegetation of the Siskiyou Mountains, Oregon and California. Ecol. Monogr. 30, 279–338 (1960).

MATH 
    
                    Google Scholar 
                

Harrison, S., Vellend, M. & Damschen, E. I. ‘Structured' beta diversity increases with climatic productivity in a classic dataset. Ecosphere 2, 1–13 (2011).


                    Google Scholar 
                

Anderson, M. J. et al. Navigating the multiple meanings of beta diversity: a roadmap for the practicing ecologist. Ecol. Lett. 14, 19–28 (2011).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Vellend, M. Conceptual synthesis in community ecology. Q. Rev. Biol. 85, 183–206 (2010).

PubMed 
    
                    Google Scholar 
                

Salguero-Gómez, R. et al. Fast–slow continuum and reproductive strategies structure plant life-history variation worldwide. Proc. Natl Acad. Sci. USA 113, 230–235 (2016).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Peco, B., Laffan, S. W. & Moles, A. T. Global patterns in post-dispersal seed removal by invertebrates and vertebrates. PLoS One 9, e91256 (2014).

ADS 
    PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Riibak, K. et al. Drivers of plant community completeness differ at regional and landscape scales. Agric. Ecosyst. Environ. 301, 107004 (2020).

MATH 
    
                    Google Scholar 
                

Ben-Hur, E. & Kadmon, R. Heterogeneity–diversity relationships in sessile organisms: a unified framework. Ecol. Lett. 23, 193–207 (2019).

PubMed 
    MATH 
    
                    Google Scholar 
                

González-Trujillo, J. D., Román-Cuesta, R. M., Muñiz-Castillo, A. I., Amaral, C. H. & Araújo, M. B. Multiple dimensions of extreme weather events and their impacts on biodiversity. Clim. Change 176, 155 (2023).

ADS 
    
                    Google Scholar 
                

Gallagher, R. V. et al. High fire frequency and the impact of the 2019–2020 megafires on Australian plant diversity. Divers. Distrib. 27, 1166–1179 (2021).

MATH 
    
                    Google Scholar 
                

Outhwaite, C. L., McCann, P. & Newbold, T. Agriculture and climate change are reshaping insect biodiversity worldwide. Nature 605, 97–102 (2022).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Dirzo, R. et al. Defaunation in the Anthropocene. Science 345, 401–406 (2014).

ADS 
    PubMed 
    
                    Google Scholar 
                

Pärtel, M. et al. DarkDivNet—a global research collaboration to explore the dark diversity of plant communities. J. Veg. Sci. 30, 1039–1043 (2019).

MATH 
    
                    Google Scholar 
                

Tikhonov, G. et al. Joint species distribution modelling with the R-package HMSC. Methods Ecol. Evol. 11, 442–447 (2020).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Ricklefs, R. E. Community diversity: relative roles of local and regional processes. Science 235, 167–171 (1987).

ADS 
    PubMed 
    
                    Google Scholar 
                

Gassert, F. et al. An operational approach to near real time global high resolution mapping of the terrestrial Human Footprint. Front. Remote Sens. 4, 1130896 (2023).

MATH 
    
                    Google Scholar 
                

McDonald, R. I. et al. Urban effects, distance, and protected areas in an urbanizing world. Landsc. Urban Plann. 93, 63–75 (2009).

MATH 
    
                    Google Scholar 
                

Engert, J. E. et al. Ghost roads and the destruction of Asia-Pacific tropical forests. Nature 629, 370–375 (2024).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Fusco, E. J., Abatzoglou, J. T., Balch, J. K., Finn, J. T. & Bradley, B. A. Quantifying the human influence on fire ignition across the western USA. Ecol. Appl. 26, 2390–2401 (2016).


                    Google Scholar 
                

Lawton, R. O., Nair, U. S., Pielke, R. A. & Welch, R. M. Climatic impact of tropical lowland deforestation on nearby montane cloud forests. Science 294, 584–587 (2001).

ADS 
    PubMed 
    
                    Google Scholar 
                

Bengtsson, J. et al. Grasslands—more important for ecosystem services than you might think. Ecosphere 10, e02582 (2019).

MATH 
    
                    Google Scholar 
                

Ellis, E. C. Land use and ecological change: a 12,000-year history. Annu. Rev. Environ. Resour. 46, 1–33 (2021).

MATH 
    
                    Google Scholar 
                

Taylor, P. D., Fahrig, L., Henein, K. & Merriam, G. Connectivity is a vital element of landscape structure. Oikos 68, 571–573 (1993).

ADS 
    MATH 
    
                    Google Scholar 
                

van Wees, D. et al. The role of fire in global forest loss dynamics. Glob. Chang. Biol. 27, 2377–2391 (2021).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Hanski, I. Habitat loss, the dynamics of biodiversity, and a perspective on conservation. Ambio 40, 248–255 (2011).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Aavik, T. & Helm, A. Restoration of plant species and genetic diversity depends on landscape-scale dispersal. Restor. Ecol. 26, S92–S102 (2018).

MATH 
    
                    Google Scholar 
                

Araujo, M. B. The coincidence of people and biodiversity in Europe. Glob. Ecol. Biogeogr. 12, 5–12 (2003).

MATH 
    
                    Google Scholar 
                

Oksanen, J. Is the humped relationship between species richness and biomass an artefact due to plot size? J. Ecol. 84, 293–295 (1996).

MATH 
    
                    Google Scholar 
                

Hooper, D. U. et al. Effects of biodiversity on ecosystem functioning: a consensus of current knowledge. Ecol. Monogr. 75, 3–35 (2005).

MATH 
    
                    Google Scholar 
                

Reich, P. B. et al. Impacts of biodiversity loss escalate through time as redundancy fades. Science 336, 589–592 (2012).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Storch, D. The theory of the nested species–area relationship: geometric foundations of biodiversity scaling. J. Veg. Sci. 27, 880–891 (2016).

MATH 
    
                    Google Scholar 
                

Leibold, M. A. et al. The metacommunity concept: a framework for multi-scale community ecology. Ecol. Lett. 7, 601–613 (2004).

MATH 
    
                    Google Scholar 
                

Leclère, D. et al. Bending the curve of terrestrial biodiversity needs an integrated strategy. Nature 585, 551–556 (2020).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Lewis, R. J. et al. Applying the dark diversity concept to nature conservation. Conserv. Biol. 31, 40–47 (2017).

PubMed 
    MATH 
    
                    Google Scholar 
                

Deschênes, É., Santala, K. R., Lavigne, J. & Aubin, I. Using a trait-based dark diversity approach to evaluate natural recovery potential in forests. Restor. Ecol. 32, e14251 (2024).


                    Google Scholar 
                

Moeslund, J. E. et al. Using dark diversity and plant characteristics to guide conservation and restoration. J. Appl. Ecol. 54, 1730–1741 (2017).

MATH 
    
                    Google Scholar 
                

Nathan, R. et al. Mechanisms of long-distance seed dispersal. Trends Ecol. Evol. 23, 638–647 (2008).

PubMed 
    MATH 
    
                    Google Scholar 
                

Pollock, L. J. et al. Understanding co-occurrence by modelling species simultaneously with a Joint Species Distribution Model (JSDM). Methods Ecol. Evol. 5, 397–406 (2014).

MATH 
    
                    Google Scholar 
                

Chao, A. et al. Rarefaction and extrapolation with Hill numbers: a framework for sampling and estimation in species diversity studies. Ecol. Monogr. 84, 45–67 (2014).

MATH 
    
                    Google Scholar 
                

Hsieh, T. C., Ma, K. H. & Chao, A. iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). Methods Ecol. Evol. 7, 1451–1456 (2016).

MATH 
    
                    Google Scholar 
                

Oksanen, J. et al. vegan: Community Ecology Package. R version 4.2.2 https://CRAN.R-project.org/package=vegan (2022).

Karger, D. N. et al. Climatologies at high resolution for the earth's land surface areas. Sci. Data 4, 170122 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Karger, D. N. et al. Data from: Climatologies at high resolution for the earth's land surface areas. Dryad Digital Repository https://doi.org/10.5061/dryad.kd1d4 (2017).

Hengl, T. et al. SoilGrids250m: global gridded soil information based on machine learning. PLoS One 12, e0169748 (2017).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Amatulli, G., McInerney, D., Sethi, T., Strobl, P. & Domisch, S. Geomorpho90m, empirical evaluation and accuracy assessment of global high-resolution geomorphometric layers. Sci. Data 7, 162 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Wood, S. N. Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models. J. R. Stat. Soc. B 73, 3–36 (2011).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Valavi, R., Elith, J., Lahoz-Monfort, J. J. & Guillera-Arroita, G. blockCV: an R package for generating spatially or environmentally separated folds for k-fold cross-validation of species distribution models. Methods Ecol. Evol. 10, 225–232 (2019).

MATH 
    
                    Google Scholar 
                

Fu, W. J., Carroll, R. J. & Wang, S. Estimating misclassification error with small samples via bootstrap cross-validation. Bioinformatics 21, 1979–1986 (2005).

PubMed 
    MATH 
    
                    Google Scholar 
                

Breheny, P. & Burchett, W. Visualization of regression models using visreg. R J. 9, 56–71 (2017).

MATH 
    
                    Google Scholar 
                

Di Marco, M., Venter, O., Possingham, H. P. & Watson, J. E. M. Changes in human footprint drive changes in species extinction risk. Nat. Commun. 9, 4621 (2018).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Williams, B. A. et al. Change in terrestrial human footprint drives continued loss of intact ecosystems. One Earth 3, 371–382 (2020).

ADS 
    MATH 
    
                    Google Scholar 
                

Pärtel, M. et al. Supporting data for ‘Global impoverishment of natural vegetation revealed by dark diversity'. Figshare https://doi.org/10.6084/m9.figshare.25158059 (2025).

Download references

We thank the Estonian Research Council (PRG609, PRG1065, PSG293 and PRG2142) and the Estonian Ministry of Education and Research (Centre of Excellence AgroCropFuture, TK200) for funding the DarkDivNet steering committee, and we thank the many students and volunteers who helped with the fieldwork. Acknowledgements related to specific authors can be found in the Supplementary Notes.

Institute of Ecology and Earth Sciences, University of Tartu, Tartu, Estonia

Meelis Pärtel, Riin Tamme, Carlos P. Carmona, Kersti Riibak, Mari Moora, Maarja Öpik, John Davison, Junichi Fujinuma, Nele Ingerpuu, Madli Jõks, Ene Kook, Tatjana Oja, Bruno Paganeli, Triin Reitalu, Enrico Tordoni, Diego P. F. Trindade, Oscar Zárate Martínez & Martin Zobel

Department of Plant Sciences, University of Saskatchewan, Saskatoon, Saskatchewan, Canada

Jonathan A. Bennett

Department of Biological, Geological and Environmental Sciences, Alma Mater Studiorum —University of Bologna, Bologna, Italy

Alessandro Chiarucci, Silvia Del Vecchio & Arianna Ferrara

Department of Botany and Zoology, Faculty of Science, Masaryk University, Brno, Czech Republic

Milan Chytrý

CIDE, CSIC-UV-GVA, Valencia, Spain

Francesco de Bello, Daniel A. Rodríguez Ginart & Diego P. F. Trindade

Department of Botany, Faculty of Science, University of South Bohemia, České Budějovice, Czech Republic

Francesco de Bello, Jiri Dolezal, Eva Janíková, Marie Konečná, Aleš Lisner & Mercedes Valerio

Department of Ecology, Environment and Plant Sciences, Stockholm University, Stockholm, Sweden

Ove Eriksson

Department of Environmental Science and Policy, University of California Davis, Davis, CA, USA

Susan Harrison

Norwegian Institute for Nature Research, Bergen, Norway

Robert John Lewis

Evolution and Ecology Research Centre, School of Biological, Earth and Environmental Sciences, UNSW Sydney, Sydney, New South Wales, Australia

Angela T. Moles

Gulbali Institute, Charles Sturt University, Albury, New South Wales, Australia

Jodi N. Price

Plant Ecology Group, Institute of Evolution and Ecology, University of Tübingen, Tübingen, Germany

Vistorina Amputu

Independent researcher, Tehran, Iran

Diana Askarizadeh

Department of Reclamation of Arid and Mountainous Regions, University of Tehran, Tehran, Iran

Diana Askarizadeh

Department of Biology, Faculty of Science, Ferdowsi University of Mashhad, Mashhad, Iran

Zohreh Atashgahi

Great Lakes Forestry Centre, Canadian Forest Service, Natural Resources Canada, Sault Ste Marie, Ontario, Canada

Isabelle Aubin & Laura Boisvert-Marsh

Terrestrial Ecology Group, Department of Ecology, Universidad Autónoma de Madrid, Madrid, Spain

Francisco M. Azcárate, Irene Guerrero & Begoña Peco

Centro de Investigación en Biodiversidad y Cambio Global (CIBC-UAM), Universidad Autónoma de Madrid, Madrid, Spain

Francisco M. Azcárate

Australian Tropical Herbarium, James Cook University, Smithfield, Queensland, Australia

Matthew D. Barrett

Department of Range and Watershed Management, Faculty of Natural Resources and Environment, Ferdowsi University of Mashhad, Mashhad, Iran

Maral Bashirzadeh

Department of Ecology, University of Szeged, Szeged, Hungary

Zoltán Bátori, András Kelemen & Csaba Tölgyesi

Centre for Environmental Sciences, Hasselt University, Hasselt, Belgium

Natalie Beenaerts

Plant Ecology and Nature Conservation, University of Potsdam, Potsdam, Germany

Kolja Bergholz, Florian Jeltsch, Michael Ristow & Lina Weiss

Department of Biological Sciences, University of Bergen, Bergen, Norway

Kristine Birkeli, Ruben S. Thormodsæter & Vigdis Vandvik

Bjerknes Centre for Climate Research, University of Bergen, Bergen, Norway

Kristine Birkeli & Vigdis Vandvik

Department of Plant Biology and Ecology, University of the Basque Country UPV/EHU, Bilbao, Spain

Idoia Biurrun & Juan A. Campos

Department of Evolutionary Biology, Ecology and Environmental Sciences (Botany and Mycology), Universitat de Barcelona, Barcelona, Spain

José M. Blanco-Moreno & Aaron Pérez-Haase

Biodiversity Research Institute (IRBio), Universitat de Barcelona, Barcelona, Spain

José M. Blanco-Moreno & Aaron Pérez-Haase

Department of Biology, University of North Carolina at Greensboro, Greensboro, NC, USA

Kathryn J. Bloodworth & Kimberly J. Komatsu

Department of Biology, National University of Mongolia, Ulaanbaatar, Mongolia

Bazartseren Boldgiv & Khaliun Sanchir

Department of Forest Sciences, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, Brazil

Pedro H. S. Brancalion & Joannès Guillemot

Re.green, Rio de Janeiro, Brazil

Pedro H. S. Brancalion

Department of Natural Sciences, Manchester Metropolitan University, Manchester, UK

Francis Q. Brearley

Département de biologie, Université de Sherbrooke, Sherbrooke, Quebec, Canada

Charlotte Brown

Department of Biological Sciences, University of Alberta, Edmonton, Alberta, Canada

Charlotte Brown, James F. Cahill, Raytha A. Murillo, Karina Salimbayeva & Viktoria Wagner

Instituto Pirenaico de Ecologia, CSIC, Jaca, Spain

C. Guillermo Bueno & Pablo Tejero

Department of Environmental Sciences, Informatics and Statistics, Ca' Foscari University of Venice, Venice, Italy

Gabriella Buffa, Edy Fantinato & Giulia Silan

Department of Life, Health and Environmental Science, University of L'Aquila, Coppito, L'Aquila, Italy

Giacomo Cangelmi

Department of Chemistry, Life Sciences and Environmental Sustainability, University of Parma, Parma, Italy

Michele Carbognani, T'ai G. W. Forte, Alessandro Petraglia & Marcello Tomaselli

École Pratique des Hautes Études, Paris Sciences Lettres University (EPHE-PSL), Paris, France

Christopher Carcaillet

University Claude Bernard Lyon 1, LEHNA UMR5023, CNRS, ENTPE, Villeurbanne, France

Christopher Carcaillet

Department of Biotechnology and Life Science, University of Insubria, Varese, Italy

Bruno E. L. Cerabolini & Michele Dalle Fratte

Conservatoire d'espaces naturels Centre–Val de Loire, Orléans, France

Richard Chevalier

Research Group Plants and Ecosystems (PLECO), University of Antwerp, Wilrijk, Belgium

Jan S. Clavel, Jonas J. Lembrechts & Dajana Radujković

Centre for Functional Ecology, Associate Laboratory TERRA, Department of Life Sciences, University of Coimbra, Coimbra, Portugal

José M. Costa, Ruben H. Heleno & Daniel Montesinos

Department of Physical Geography, Stockholm University, Stockholm, Sweden

Sara A. O. Cousins

Department of Invasion Ecology, Institute of Botany, Czech Academy of Sciences, Průhonice, Czech Republic

Jan Čuda, Martin Hejda, Jiří Sádlo, Hana Skálová, Kateřina Štajerová, Michaela Vítková & Martin Vojík

Instituto de Biociências, Lab of Vegetation Ecology, Universidade Estadual Paulista (UNESP), Rio Claro, Brazil

Mariana Dairel & Alessandra Fidelis

Independent researcher, Kirovsk, Russia

Alena Danilova & Natalia Koroleva

Lendület Seed Ecology Research Group, Institute of Ecology and Botany, HUN-REN Centre for Ecological Research, Vácrátót, Hungary

Balázs Deák, András Kelemen, Katalin Lukács & Orsolya Valkó

Institute of Environmental Biology, Faculty of Biology, University of Warsaw, Warsaw, Poland

Iwona Dembicz, Łukasz Kozub & Nadiia Skobel

Vegetation Ecology Research Group, Institute of Natural Resource Sciences (IUNR), Zurich University of Applied Sciences (ZHAW), Wädenswil, Switzerland

Jürgen Dengler

Institute of Botany, Czech Academy of Sciences, Průhonice, Czech Republic

Jiri Dolezal & Miroslav Dvorsky

CREAF (Centre for Ecological Research and Forestry Applications), Bellaterra, Spain

Xavier Domene

Universitat Autònoma de Barcelona, Bellaterra, Spain

Xavier Domene

Quantitative Plant Ecology and Biodiversity Research Lab, Department of Biology, Faculty of Science, Ferdowsi University of Mashhad, Mashdad, Iran

Hamid Ejtehadi & Sahar Karami

Instituto Multidisciplinario de Biología Vegetal (CONICET-UNC), Córdoba, Argentina

Lucas Enrico & Melisa A. Giorgis

FCEFyN, Universidad Nacional de Córdoba, Córdoba, Argentina

Lucas Enrico & Melisa A. Giorgis

Independent researcher, Moscow, Russia

Dmitrii Epikhin

Department of Ecology and Genetics, University of Oulu, Oulu, Finland

Anu Eskelinen & Risto Virtanen

German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany

Anu Eskelinen, Lotte Korell & Soroor Rahmanian

Division of BioInvasions, Global Change and Macroecology, University of Vienna, Vienna, Austria

Franz Essl

State Key Laboratory of Vegetation and Environmental Change, Institute of Botany, Chinese Academy of Sciences, Beijing, China

Gaohua Fan & Houjia Liu

Chair of Plant Ecology, University of Bayreuth, Bayreuth, Germany

Fatih Fazlioglu

Department of Molecular Biology and Genetics, Faculty of Arts and Sciences, Ordu University, Ordu, Turkey

Fatih Fazlioglu

Biodiversity Research Institute (IMIB), University of Oviedo–CSIC–Principality of Asturias, Mieres, Spain

Eduardo Fernández-Pascual & Borja Jiménez-Alfaro

Department of Organismal and Systems Biology, University of Ovidedo, Oviedo, Spain

Eduardo Fernández-Pascual & Borja Jiménez-Alfaro

Institute of Plant Sciences, University of Bern, Bern, Switzerland

Markus Fischer & Lena Neuenkamp

Department of Agricultural and Food Chemistry, Universidad Autónoma de Madrid, Madrid, Spain

Maren Flagmeier & Eduardo Moreno-Jiménez

Department of Natural Resource Sciences, Thompson Rivers University, Kamloops, British Columbia, Canada

Lauchlan H. Fraser

Graduate Program in Botany, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil

Fernando F. Furquim

Independent researcher, Mancelona,  MI, USA

Berle Garris

Au Sable Institute of Environmental Studies, Mancelona, MI, USA

Heath W. Garris

Department of Geological, Biological and Environmental Sciences, University of Catania, Catania, Italy

Gianpietro Giusso del Galdo

Departamento de Biología Animal, Biología Vegetal y Ecología, Universidad de Jaén, Jaén, Spain

Ana González-Robles & Rubén Tarifa

Instituto Interuniversitario del Sistema Tierra de Andalucía, Universidad de Jaén, Jaén, Spain

Ana González-Robles

School of Agriculture, Food and Ecosystem Sciences, University of Melbourne, Melbourne, Victoria, Australia

Megan K. Good

Unit of Botany, Department of Animal and Plant Biology and Ecology, Universitat Autònoma de Barcelona, Bellaterra, Spain

Moisès Guardiola

Department of Biological, Chemical and Pharmaceutical Sciences and Technologies, University of Palermo, Palermo, Italy

Riccardo Guarino

CIRAD, UMR Eco&Sols, Montpellier, France

Joannès Guillemot & Agnès A. Robin

Eco&Sols, University Montpellier, CIRAD, INRAE, Institut Agro, IRD, Montpellier, France

Joannès Guillemot & Agnès A. Robin

Biology Education, Dokuz Eylül University, Buca, Turkey

Behlül Güler

Key Laboratory of Soybean Molecular Design Breeding, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Changchun, China

Yinjie Guo

Department of Earth and Environmental Sciences, KU Leuven, Leuven, Belgium

Stef Haesen, Karlien Moeys & Koenraad Van Meerbeek

KU Leuven Plant Institute, KU Leuven, Leuven, Belgium

Stef Haesen & Koenraad Van Meerbeek

Department of Ecoscience, Aarhus University, Aarhus, Denmark

Toke T. Høye & Jesper Erenskjold Moeslund

Arctic Research Centre, Aarhus University, Aarhus, Denmark

Toke T. Høye

Institute of Botany, Plant Science and Biodiversity Center, Slovak Academy of Sciences, Bratislava, Slovakia

Richard Hrivnák & Ivana Svitková

State Key Laboratory of Black Soils Conservation and Utilization, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Changchun, China

Yingxin Huang & Xuhe Liu

Jilin Songnen Grassland Ecosystem National Observation and Research Station, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Changchun, China

Yingxin Huang & Xuhe Liu

Jilin Provincial Key Laboratory of Grassland Farming, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Changchun, China

Yingxin Huang & Xuhe Liu

School of Environmental and Rural Science, University of New England, Armidale, New South Wales, Australia

John T. Hunter

Institute of Biological Sciences, University of Zielona Góra, Zielona Góra, Poland

Dmytro Iakushenko

F. Falz-Fein Biosphere Reserve Askania Nova, Kyiv, Ukraine

Dmytro Iakushenko

Departamento de Biología Ambiental, Facultad de Ciencias, Universidad de Navarra, Pamplona, Spain

Ricardo Ibáñez & Mercedes Valerio

Biogeography and Biodiversity Lab, Institute of Physical Geography, Goethe-University Frankfurt, Frankfurt am Main, Germany

Severin D. H. Irl

Faculty of Agricultural and Environmental Sciences, University of Rostock, Rostock, Germany

Florian Jansen

Department of Disturbance Ecology, Bayreuth Center of Ecology and Environmental Research (BayCEER), University of Bayreuth, Bayreuth, Germany

Anke Jentsch & Andreas von Hessberg

Department of Range and Watershed Management, Faculty of Natural Resources, Islamic Azad University Nour Branch, Nour, Iran

Mohammad H. Jouri

Chair of Sensor-based Geoinformatics, Faculty of Environment and Natural Resources, University of Freiburg, Freiburg, Germany

Negin Katal

Independent researcher, Kazan, Russia

Bulat I. Khairullin, Maria V. Kozhevnikova & Vadim E. Prokhorov

Centre for Biodiversity and Taxonomy, Department of Botany, University of Kashmir, Srinagar, India

Anzar A. Khuroo & Sajad A. Wani

Department of Species Interaction Ecology, Helmholtz Centre for Environmental Research—UFZ, Leipzig, Germany

Lotte Korell

Department of Functional Ecology, Institute of Botany, Czech Academy of Sciences, Třeboň, Czech Republic

Kirill A. Korznikov & Vojtech Lanta

Chair of Biodiversity and Nature Tourism, Estonian University of Life Sciences, Tartu, Estonia

Lauri Laanisto & Petr Macek

Kalmar County Administrative Board, Färjestaden, Sweden

Helena Lager & Michael Tholin

Instituto Nacional de Tecnología Agropecuaria (INTA), Río Gallegos, Argentina

Romina G. Lasagno & Pablo L. Peri

Ecology and Biodiversity (E&B), Utrecht University, Utrecht, The Netherlands

Jonas J. Lembrechts

Aerospace Information Research Institute, Chinese Academy of Sciences, Beijing, China

Liping Li & Yichen Tian

State Key Laboratory of Grassland and Agro-Ecosystems, School of Life Sciences, Lanzhou University, Lanzhou, China

Kun Liu & Sa Xiao

School of Grassland Science, Beijing Forestry University, Beijing, China

Xuhe Liu

Higher Technical School of Agricultural and Forestry Engineering, Castilla-La Mancha University, Albacete, Spain

Manuel Esteban Lucas-Borja & Pedro Antonio Plaza-Álvarez

Applied Plant Ecology, Institute of Plant Science and Microbiology, University of Hamburg, Hamburg, Germany

Kristin Ludewig

Netzwerk für Angewandte Ökologie, Hamburg, Germany

Jona Luther-Mosebach

Institute of Hydrobiology, Biology Centre of the Czech Academy of Sciences, Ceske Budejovice, Czech Republic

Petr Macek

Department of Life and Environmental Sciences, University of Cagliari, Cagliari, Italy

Michela Marignani

Univ. Bordeaux, CNRS, Bordeaux INP, EPOC, UMR 5805, Pessac, France

Richard Michalet & Blaise Touzard

ÖMKi—Research Institute of Organic Agriculture, Budapest, Hungary

Tamás Miglécz

Australian Tropical Herbarium, James Cook University, Cairns, Queensland, Australia

Daniel Montesinos

College of Science and Engineering, James Cook University, Cairns, Queensland, Australia

Daniel Montesinos

Institute for Advanced Research in Chemical Sciences (IAdChem), Universidad Autónoma de Madrid, Madrid, Spain

Eduardo Moreno-Jiménez

Department of Botany, Kherson State University, Kherson, Ukraine

Ivan Moysiyenko & Nadiia Skobel

Harry Butler Institute, Murdoch University, Perth, Western Australia, Australia

Ladislav Mucina & James L. Tsakalos

Department of Geography and Environmental Studies, Stellenbosch University, Stellenbosch, South Africa

Ladislav Mucina

Laboratorio de Biodiversidad y Funcionamiento Ecosistémico Instituto de Recursos Naturales y Agrobiología de Sevilla (IRNAS), CSIC, Sevilla, Spain

Miriam Muñoz-Rojas

Centre for Ecosystem Science, UNSW Sydney, Sydney, New South Wales, Australia

Miriam Muñoz-Rojas

Department of Agriculture and Natural Resource Sciences, Namibia University of Science and Technology, Windhoek, Namibia

Sylvia M. Nambahu

Institute of Landscape Ecology, University of Münster, Münster, Germany

Lena Neuenkamp

Center for Sustainable Landscapes Under Global Change, Department of Biology, Aarhus University, Aarhus, Denmark

Signe Normand

Botanical Garden, Center for Biological Diversity Conservation, Polish Academy of Sciences, Warszawa, Poland

Arkadiusz Nowak & Sebastian Świerszcz

Greenpeace España, Madrid, Spain

Paloma Nuche

Shenzhen MSU-BIT University, Shenzhen, China

Vladimir G. Onipchenko

Department of Ecology and Environmental Protection, Faculty of Biology, Sofia University St Kliment Ohridski, Sofia, Bulgaria

Kalina L. Pachedjieva

Departemento Biología y Geología, Física y Química Inorgánica, Universidad Rey Juan Carlos, Móstoles, Spain

Ana M. L. Peralta

Universidad Nacional de la Patagonia Austral (UNPA), CONICET, Río Gallegos, Argentina

Pablo L. Peri

Department of Civil and Environmental Engineering, University of the Andes, Bogotá, Colombia

Gwendolyn Peyre

Swedish Biodiversity Centre, Department of Urban and Rural Development, Swedish University of Agricultural Sciences, Uppsala, Sweden

Jan Plue

Department of Biology, Lund University, Lund, Sweden

Honor C. Prentice

Remote Sensing Centre for Earth System Research (RSC4Earth), Leipzig University, Leipzig, Germany

Soroor Rahmanian

Institute of Geology, Tallinn University of Technology, Tallinn, Estonia

Triin Reitalu

Department of Soil Sciences, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, Brazil

Agnès A. Robin

Estación Experimental del Zaidín (CSIC), Granada, Spain

Ana Belén Robles

Department of Agronomy, University of Almería, Almería, Spain

Raúl Román

Norwegian Institute for Nature Research, Oslo, Norway

Ruben E. Roos

Faculty of Environmental Sciences and Natural Resource Management, Norwegian University of Life Sciences, Ås, Norway

Ruben E. Roos

Scuola di Scienze Agrarie, Forestali, Alimentari e Ambientali, Università della Basilicata, Potenza, Italy

Leonardo Rosati

Departamento de Biodiversidad, Ecología y Evolución, Facultad de Ciencias Biológicas, Universidad Complutense de Madrid, Madrid, Spain

Rut Sánchez de Dios & Enrique Valencia

School of Natural Sciences, Macquarie University, Sydney, New South Wales, Australia

Cornelia Sattler & Julian Schrader

Department of Ecosystem Science and Management, Laramie Research and Extension Center, University of Wyoming, Laramie, WY, USA

John D. Scasta

Institute of Plant Science and Microbiology, University of Hamburg, Hamburg, Germany

Ute Schmiedel

Future Regions Research Centre, Federation University Australia, Ballarat, Victoria, Australia

Nick L. Schultz

CIRAD-UMR EcoFoG, Kourou, French Guiana

Giacomo Sellan

Botanical Institute of Barcelona (CSIC-CMCNB), Barcelona, Spain

Josep M. Serra-Diaz

Department of Ecology, University of Debrecen, Debrecen, Hungary

Judit Sonkoly

Institute of Agroecology and Plant Production, Wrocław University of Environmental and Life Sciences, Wrocław, Poland

Sebastian Świerszcz

Ecosystems and Global Change Group, School of the Environment, Trent University, Peterborough, Ontario, Canada

Andrew J. Tanentzap

Department of Plant Sciences, University of Cambridge, Cambridge, UK

Fallon M. Tanentzap

Estación Experimental de Zonas Áridas (EEZA-CSIC), Almería, Spain

Rubén Tarifa

Independent reseacher, Teberda, Russia

Dzhamal K. Tekeev

Yuriy Fedkovych Chernivtsi National University, Chernivtsi, Ukraine

Alla Tokaryuk

HUN-REN-UD Functional and Restoration Ecology Research Group, Department of Ecology, University of Debrecen, Debrecen, Hungary

Péter Török

HUN-REN-UD Biodiversity and Ecosystem Services Research Group, Department of Ecology, University of Debrecen, Debrecen, Hungary

Béla Tóthmérész

Centre de Recherche sur la Biodiversité et l'Environnement (CRBE), UMR 5300 UPS-CNRS-IRD-INP, Université Paul Sabatier–Toulouse 3, Toulouse, France

Aurèle Toussaint

School of Biosciences and Veterinary Medicine, University of Camerino, Camerino, Italy

James L. Tsakalos

Department of Mathematics and Science Education, Faculty of Education, Ordu University, Ordu, Turkey

Sevda Türkiş

Departamento de Ciencias de la Vida, Universidad de Alcalá, Alcalá de Henares, Spain

Jesus Villellas

Department of Applied Ecology, Faculty of Environmental Sciences, Czech University of Life Sciences, Prague, Czech Republic

Martin Vojík

Nature Conservation Agency of the Czech Republic, Prague, Czech Republic

Martin Vojík

Department of Biology, Aarhus University, Aarhus, Denmark

Jonathan von Oppen

Department of Environmental Sciences, University of Basel, Basel, Switzerland

Jonathan von Oppen

Key Laboratory of Mountain Surface Processes and Ecological Regulation, Institute of Mountain Hazards and Environment, Chinese Academy of Sciences, Chengdu, China

Ji-Zhong Wan

Sichuan Academy of Forestry, Chengdu, China

Chun-Jing Wang

National Monitoring Centre for Biodiversity Germany, Leipzig, Germany

Lina Weiss

Deakin University, Burwood, Victoria, Australia

Tricia Wevill

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

The steering committee members (M.P., R. Tamme, C.P.C., K.R., M. Moora and M.Z.) initiated and coordinated DarkDivNet, analysed the data and wrote the manuscript. The advisory board members (J.A.B., A.C., M. Chytrý, F.B., O.E., S. Harrison, R.J.L., A.T.M., M.Ö. and J.N.P.) contributed to the study design and to the initial versions of the manuscript. V.A., D.A., Z.A., I.A., F.M.A., M.D.B., M.B., Z.B., N.B., K. Bergholz, K. Birkeli, I.B., J.M.B.-M., K.J.B., L.B.-M., B.B., P.H.S.B., F.Q.B., C.B., C.G.B., G.B., J.F.C., J.A.C., G.C., M. Carbognani, C.C., B.E.L.C., R.C., J.S.C., J.M.C., S.A.O.C., J.Č., M. Dairel, M.D.F., A.D., J. Davison, B.D., S.D.V., I.D., J. Dengler, J. Dolezal, X.D., M. Dvorsky, H.E., L.E., D.E., A.E., F.E., G.F., E.F., F.F., E.F.-P., A. Ferrara, A. Fidelis, M. Fischer, M. Flagmeier, T.G.W.F., L.H.F., J.F., F.F.F., B. Garris, H.W.G., M.A.G., G.G.G., A.G.-R., M.K.G., M.G., R.G., I.G., J.G., B. Güler, Y.G., S Haesen, M.H., R.H.H., T.T.H., R.H., Y.H., J.T.H., D.I., R.I., N.I., S.D.H.I., E.J., F. Jansen, F. Jeltsch, A.J., B.J.-A., M.J., M.H.J., S.K., N. Katal, A.K., B.I.K., A.A.K., K.J.K., M.K., E.K., L.K., N. Koroleva, K.A.K., M.V.K., Ł.K., L. Laanisto, H. Lager, V.L., R.G.L., J.J.L., L. Li, A.L., H. Liu, K. Liu, X.L., M.E.L.-B., K. Ludewig, K. Lukács, J.L.-M., P.M., M. Marignani, R.M., T.M., J.E.M., K.M., D.M., E.M.-J., I.M., L.M., M.M.-R., R.A.M., S.M.N., L.N., S.N., A.N., P.N., T.O., V.G.O., K.L.P., B. Paganeli, B. Peco, A.M.L.P., A.P.-H., P.L.P., A.P., G.P., P.A.P.-Á., J.P., H.C.P., V.E.P., D.R., S.R., T.R., M.R., A.A.R., A.B.R., D.A.R.G., R.R., R.E.R., L.R., J. Sádlo, K. Salimbayeva, R.S.D., K. Sanchir, C.S., J.D.S., U.S., J. Schrader, N.L.S., G. Sellan, J.M.S.-D., G. Silan, H.S., N.S., J. Sonkoly, K. Štajerova, I.S., S.Ś., A.J.T., F.M.T., R. Tarifa, P. Tejero, D.K.T., M. Tholin, R.S.T., Y.T., A. Tokaryuk, C.T., M. Tomaselli, E.T., P. Török, B. Tóthmérész, A. Toussaint, B. Touzard, D.P.F.T., J.L.T., S.T., E.V., M. Valerio, O.V., K.V.M., V.V., J.V., R.V., M. Vítkova, M. Vojík, A.H., J.O., V.W., J.-Z.W., C.-J.W., S.A.W., L.W., T.W., S.X. and O.Z.M. provided data and were involved in the interpretation of the results and manuscript preparation.

Correspondence to
                Meelis Pärtel.

The authors declare no competing interests.

Nature thanks Marta Jarzyna, Alejandro Ordonez and Moreno Di Marco for their contribution to the peer review of this work.

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Lines indicate ranges within a radius of 100 km. Approximate broad biomes are shown. Triangles indicate regions in which only woody species were sampled.

We used two tests. In the Vicinity test, we examined whether species absent from the site (100 m2) but present in the immediate vicinity (2500 m2) have higher estimated suitabilities than absent species found further away. The sample area and vicinity area are assumed to share relatively similar ecological conditions. In the Expert test, we compared whether species absent from the site but assessed by expert opinion to be ecologically suitable (i.e., belong to the site-specific species pool) have higher calculated suitabilities than those absent species that were evaluated as unsuitable. In both cases, we calculated the log response ratio of the mean suitability of species in the respective groups. Positive log response ratios indicate agreement between assessments of suitability calculated from co-occurrences and from the independent information considered in the tests. The length of the lines (vertical for the Vicinity test and horizontal for the Expert test) shown at study region locations indicates the magnitude of the log response ratio; negative values are in red and positive values are in blue. Both tests comprised data from a subset of study regions. The box plot on the left (centre line, median; box limits, upper and lower quartiles; whiskers, the range, excluding outlying points that exceed the quartiles by more than 1.5× the interquartile range) shows the results of single-sample two-sided t-tests (difference from zero), with log response ratios significantly larger than zero in both cases, n = 115 regions for the Vicinity test and n = 76 regions for the Expert test. Hexagons on the map (made with Natural Earth; free vector and raster map data; https://www.naturalearthdata.com/) delimit the spatial blocks used in cross-validation.

a, Gamma diversity from 30 sites compared with extrapolation up to complete sampling coverage. b–f, Sites described in a 2500 m2 area in addition to the DarkDivNet standard of 100 m2. g–k, Biodiversity metrics when 60 sites were used to estimate co-occurrences in addition to the DarkDivNet standard of 30. The scatter plots show mean values for regions where the respective sampling scheme was applied (n = 119 regions for a, n = 116 regions for b–f and n = 27 for g–k). The 1:1 lines are shown as diagonals. Estimates of Spearman correlation for each comparison are shown above the panels. Comparisons where the alternative sampling method did not influence the metric (i.e. gamma diversity when using a larger sample area or alpha diversity when using more sites) are not shown.

a, Alpha diversity. b, Beta diversity. c, Gamma diversity. d, Dark diversity. e, Species pool size. A similar graph for community completeness is shown in Fig. 2a. For each metric, the relationships from the spatial scale producing the best multiple linear regression model is shown (n = 116 regions, see Extended Data Table 1 for details of the models). The prediction lines are shown with 95% confidence intervals. The solid line indicates a significant relationship (two-tailed p < 0.05); the dashed lines indicate non-significant trends. Note that the range of the human footprint index varies when averaged at different spatial scales. Diversity values on y-axes are back-transformed from log or logit scales.

Correlations with raw environmental variables (used in the PCA; top) and the human footprint index or its components (not used in the PCA; bottom).

Filled symbols indicate significant relationships (n = 116 regions, two-tailed p < 0.05), and the large symbols indicate models where the Akaike information criterion (AIC) is lower than the minimal value among the models using the mean human footprint index value (Extended Data Table 1). The asterisk indicates a combination of quantile and spatial scale that yielded a considerably better model (AIC value lower by more than 2 units).

a-e, Alpha diversity. a, f-i, Dark diversity. b, f, j-l, Species pool size. c, g, j, m-o, Gamma diversity. d, h, k, m, o, Community completeness. e, i, l, n, o, Beta diversity. Lines indicate variation within regions (99% quantiles, i.e. omitting outliers), crossing at median values within regions. Several metrics are inherently related (see Fig. 1), and strong relationships are expected. However, variation within regions can be large, indicating the importance of site-specific metrics. The colours of lines reflect the different biodiversity metrics (see Fig. 1) to facilitate comparisons across panels.

a, Community completeness. b, Alpha diversity. c, Dark diversity. d, Species pool size. e, Gamma diversity. f, Beta diversity. We used fivefold spatial cross-validation (see Extended Data Fig. 2) with bootstrapping to estimate the variation. The box plots (centre line = median; box limits = upper and lower quartiles; whiskers = the range, excluding outlying points that exceed the quartiles by more than 1.5× the interquartile range) show that while the nonlinear models had lower errors for the training set, the test data were predicted with lower error by the linear models.

This file contains Supplementary Tables 1 and 2, Supplementary Methods and Supplementary Notes.

Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/.

Reprints and permissions

Pärtel, M., Tamme, R., Carmona, C.P. et al. Global impoverishment of natural vegetation revealed by dark diversity.
                    Nature  (2025). https://doi.org/10.1038/s41586-025-08814-5

Download citation

Received: 07 February 2024

Accepted: 19 February 2025

Published: 02 April 2025

DOI: https://doi.org/10.1038/s41586-025-08814-5

Anyone you share the following link with will be able to read this content:

Sorry, a shareable link is not currently available for this article.




                            Provided by the Springer Nature SharedIt content-sharing initiative
                        

Advertisement



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited

Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.livescience.com/space/space-exploration/nasas-spherex-telescope-opens-its-eyes-on-the-universe-taking-stunning-debut-image-of-100-000-galaxies-and-stars'>'Wow! Just wow!': Scientists rejoice as NASA's SPHEREx telescope reveals debut images</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.livescience.com', 'title': 'Live Science'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 16:51:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

SPHEREx's first images — containing roughly 100,000 points of light stars, galaxies and nebulae — have confirmed that the telescope is working according to its design.


When you purchase through links on our site, we may earn an affiliate commission. Here's how it works.


A new NASA space telescope has turned on its detectors for the first time, capturing its first light in images that contain tens of thousands of galaxies and stars.

The Spectro-Photometer for the History of the Universe, Epoch of Reionization, and Ices Explorer (SPHEREx) arrived in orbit atop a SpaceX Falcon 9 rocket on March 11.

The six released images, collected by the space telescope on March 27, were each snapped by three different detectors. The top three images span the telescope's complete field of view, and are captured again in the bottom three which are colored differently to represent varying ranges of infrared wavelengths.

Within each image's full field of view — an area roughly 20 times wider than the full moon — roughly 100,000 light sources from stars, galaxies, and nebulas can be glimpsed.

"Our spacecraft has opened its eyes on the universe," Olivier Doré, a SPHEREx project scientist at Caltech and NASA's Jet Propulsion Laboratory, said in a statement. "It's performing just as it was designed to."

Related: Euclid space telescope: ESA's groundbreaking mission to study dark matter and dark energy

Costing a total of $488 million to build and launch, the new telescope has been in development for roughly a decade, and is set to map the universe by observing both optical and infrared light. It will orbit Earth 14.5 times a day, completing 11,000 orbits during its lifetime to filter infrared light from distant gas and dust clouds using a technique called spectroscopy.

Get the world's most fascinating discoveries delivered straight to your inbox.

Once it is fully online in April, SPHEREX will scan the entire night sky a total of four times using 102 separate infrared color sensors, enabling it to collect data from more than 450 million galaxies during its planned two-year operation. This amounts to roughly 600 exposures a day, according to NASA.

This dataset will give scientists key insights into some of the biggest questions in cosmology, enabling astronomers to study galaxies at various stages in their evolution; trace the ice floating in empty space to see how life may have begun; and even understand the period of rapid inflation the universe underwent immediately after the Big Bang.

—Our entire galaxy is warping, and a gigantic blob of dark matter could be to blame

—Dark matter's secret identity could be hiding in distorted 'Einstein rings'

—James Webb telescope reveals 3 possible 'dark stars' — galaxy-sized objects powered by invisible dark matter

SPHEREx's wide panorama view makes it the perfect complement for the James Webb Space Telescope, flagging regions of interest for the latter to study with greater depth and resolution.

After lofting it to space, NASA scientists and engineers have performed a nail-biting series of checks on the new telescope. This includes ensuring that its sensitive infrared equipment is cooling down to its final temperature of around minus 350 degrees Fahrenheit (minus 210 degrees Celsius) and that the telescope is set to the right focus — something that cannot be adjusted in space.

Based on these stunning preliminary images, it appears that everything has worked out.

"This is the high point of spacecraft checkout; it's the thing we wait for," Beth Fabinsky, SPHEREx deputy project manager at JPL, said in the statement. "There's still work to do, but this is the big payoff. And wow! Just wow!"

Ben Turner is a U.K. based staff writer at Live Science. He covers physics and astronomy, among other topics like tech and climate change. He graduated from University College London with a degree in particle physics before training as a journalist. When he's not writing, Ben enjoys reading literature, playing the guitar and embarrassing himself with chess.

Please logout and then login again, you will then be prompted to enter your display name.

'A notch above a gimmick': Experts question scientific merit of billionaire's Fram2 'space adventure' around Earth's poles

China now has a 'kill mesh' in orbit, Space Force vice chief says

'A notch above a gimmick': Experts question scientific merit of billionaire's Fram2 'space adventure' around Earth's poles


Live Science is part of Future US Inc, an international media group and leading digital publisher. Visit our corporate site.


©
Future US, Inc. Full 7th Floor, 130 West 42nd Street,
New York,
NY 10036. </p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08813-6'>Psilocybin's lasting action requires pyramidal cell types and 5-HT2A receptors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 16:05:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement


Nature

                         (2025)Cite this article


Metrics details

Psilocybin is a serotonergic psychedelic with therapeutic potential for treating mental illnesses1,2,3,4. At the cellular level, psychedelics induce structural neural plasticity5,6, exemplified by the drug-evoked growth and remodelling of dendritic spines in cortical pyramidal cells7,8,9. A key question is how these cellular modifications map onto cell-type-specific circuits to produce the psychedelics' behavioural actions10. Here we use in vivo optical imaging, chemogenetic perturbation and cell-type-specific electrophysiology to investigate the impact of psilocybin on the two main types of pyramidal cells in the mouse medial frontal cortex. We find that a single dose of psilocybin increases the density of dendritic spines in both the subcortical-projecting, pyramidal tract (PT) and intratelencephalic (IT) cell types. Behaviourally, silencing the PT neurons eliminates psilocybin's ability to ameliorate stress-related phenotypes, whereas silencing IT neurons has no detectable effect. In PT neurons only, psilocybin boosts synaptic calcium transients and elevates firing rates acutely after administration. Targeted knockout of 5-HT2A receptors abolishes psilocybin's effects on stress-related behaviour and structural plasticity. Collectively, these results identify that a pyramidal cell type and the 5-HT2A receptor in the medial frontal cortex have essential roles in psilocybin's long-term drug action.

This is a preview of subscription content, access via your institution


        Access Nature and 54 other Nature Portfolio journals
      


        Get Nature+, our best-value online-access subscription
      


$29.99 / 30 days


cancel any time

Subscribe to this journal


        Receive 51 print issues and online access
      

$199.00 per year

only $3.90 per issue

Buy this article

Prices may be subject to local taxes which are calculated during checkout

Details for sample sizes and statistical tests for all experiments are provided in Supplementary Table 1. Data associated with the study are available at GitHub (https://github.com/Kwan-Lab/shaoliao2025). The RNA-seq dataset was obtained from publicly available sources at the Allen Institute (https://doi.org/10.1016/j.cell.2021.04.021). Source data are provided with this paper.

Code for data analysis associated with the study are available at GitHub (https://github.com/Kwan-Lab/shaoliao2025), and from the corresponding author on request.

Goodwin, G. M. et al. Single-dose psilocybin for a treatment-resistant episode of major depression. N. Engl. J. Med. 387, 1637–1648 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Davis, A. K. et al. Effects of psilocybin-assisted therapy on major depressive disorder: a randomized clinical trial. JAMA Psychiatry 78, 481–489 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Carhart-Harris, R. et al. Trial of psilocybin versus escitalopram for depression. N. Engl. J. Med. 384, 1402–1411 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Bogenschutz, M. P. et al. Percentage of heavy drinking days following psilocybin-assisted psychotherapy vs placebo in the treatment of adult patients with alcohol use disorder: a randomized clinical trial. JAMA Psychiatry 79, 953–962 (2022).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Ly, C. et al. Psychedelics promote structural and functional neural plasticity. Cell Rep. 23, 3170–3182 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Jones, K. A. et al. Rapid modulation of spine morphology by the 5-HT2A serotonin receptor through kalirin-7 signaling. Proc. Natl Acad. Sci. USA 106, 19575–19580 (2009).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Shao, L. X. et al. Psilocybin induces rapid and persistent growth of dendritic spines in frontal cortex in vivo. Neuron 109, 2535–2544 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

de la Fuente Revenga, M. et al. Prolonged epigenomic and synaptic plasticity alterations following single exposure to a psychedelic in mice. Cell Rep. 37, 109836 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Jefferson, S. J. et al. 5-MeO-DMT modifies innate behaviors and promotes structural neural plasticity in mice. Neuropsychopharmacology 48, 1257–1266 (2023).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Kwan, A. C., Olson, D. E., Preller, K. H. & Roth, B. L. The neural basis of psychedelic action. Nat. Neurosci. 25, 1407–1419 (2022).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Duman, R. S. & Aghajanian, G. K. Synaptic dysfunction in depression: potential therapeutic targets. Science 338, 68–72 (2012).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Cameron, L. P. et al. A non-hallucinogenic psychedelic analogue with therapeutic potential. Nature 589, 474–479 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Lu, J. et al. An analog of psychedelics restores functional neural circuits disrupted by unpredictable stress. Mol. Psychiatry 26, 6237–6252 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Anastasiades, P. G. & Carter, A. G. Circuit organization of the rodent medial prefrontal cortex. Trends Neurosci. 44, 550–563 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Baker, A. et al. Specialized subpopulations of deep-layer pyramidal neurons in the neocortex: bridging cellular properties to functional consequences. J. Neurosci. 38, 5441–5455 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Shepherd, G. M. Corticostriatal connectivity and its role in disease. Nat. Rev. Neurosci. 14, 278–291 (2013).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Li, N., Chen, T. W., Guo, Z. V., Gerfen, C. R. & Svoboda, K. A motor cortex circuit for motor planning and movement. Nature 519, 51–56 (2015).

PubMed 
    
                    Google Scholar 
                

Musall, S. et al. Pyramidal cell types drive functionally distinct cortical activity patterns during decision-making. Nat. Neurosci. 26, 495–505 (2023).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Tang, L. & Higley, M. J. Layer 5 circuits in V1 differentially control visuomotor behavior. Neuron 105, 346–354 (2020).

PubMed 
    MATH 
    
                    Google Scholar 
                

Garcia, A. F., Crummy, E. A., Webb, I. G., Nooney, M. N. & Ferguson, S. M. Distinct populations of cortical pyramidal neurons mediate drug reward and aversion. Nat. Commun. 12, 182 (2021).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Davies, M. F., Deisz, R. A., Prince, D. A. & Peroutka, S. J. Two distinct effects of 5-hydroxytryptamine on single cortical neurons. Brain Res. 423, 347–352 (1987).

PubMed 
    
                    Google Scholar 
                

Araneda, R. & Andrade, R. 5-Hydroxytryptamine2 and 5-hydroxytryptamine1A receptors mediate opposing responses on membrane excitability in rat association cortex. Neuroscience 40, 199–412 (1991).


                    Google Scholar 
                

Avesar, D. & Gulledge, A. T. Selective serotonergic excitation of callosal projection neurons. Front. Neural Circuits 6, 12 (2012).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Elliott, M. C., Tanaka, P. M., Schwark, R. W. & Andrade, R. Serotonin differentially regulates L5 pyramidal cell classes of the medial prefrontal cortex in rats and mice. eNeuro https://doi.org/10.1523/ENEURO.0305-17.2018 (2018).

Amargos-Bosch, M. et al. Co-expression and in vivo interaction of serotonin1A and serotonin2A receptors in pyramidal neurons of prefrontal cortex. Cereb. Cortex 14, 281–299 (2004).

PubMed 
    
                    Google Scholar 
                

Savalia, N. K., Shao, L. X. & Kwan, A. C. A dendrite-focused framework for understanding the actions of ketamine and psychedelics. Trends Neurosci. 44, 260–275 (2021).

PubMed 
    
                    Google Scholar 
                

Puig, M. V., Celada, P., Diaz-Mataix, L. & Artigas, F. In vivo modulation of the activity of pyramidal neurons in the rat medial prefrontal cortex by 5-HT2A receptors: relationship to thalamocortical afferents. Cereb. Cortex 13, 870–882 (2003).

PubMed 
    
                    Google Scholar 
                

Kim, Y. et al. Whole-brain mapping of neuronal activity in the learned helplessness model of depression. Front. Neural Circuits 10, 3 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Davoudian, P. A., Shao, L. X. & Kwan, A. C. Shared and distinct brain regions targeted for immediate early gene expression by ketamine and psilocybin. ACS Chem. Neurosci. 14, 468–480 (2023).

PubMed 
    
                    Google Scholar 
                

Armbruster, B. N., Li, X., Pausch, M. H., Herlitze, S. & Roth, B. L. Evolving the lock to fit the key to create a family of G protein-coupled receptors potently activated by an inert ligand. Proc. Natl Acad. Sci. USA 104, 5163–5168 (2007).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Matho, K. S. et al. Genetic dissection of the glutamatergic neuron system in cerebral cortex. Nature 598, 182–187 (2021).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Nagai, Y. et al. Deschloroclozapine, a potent and selective chemogenetic actuator enables rapid neuronal and behavioral modulations in mice and monkeys. Nat. Neurosci. 23, 1157–1167 (2020).

PubMed 
    
                    Google Scholar 
                

Halberstadt, A. L., Chatha, M., Klein, A. K., Wallach, J. & Brandt, S. D. Correlation between the potency of hallucinogens in the mouse head-twitch response assay and their behavioral and subjective effects in other species. Neuropharmacology 167, 107933 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Malenka, R. C., Lancaster, B. & Zucker, R. S. Temporal limits on the rise in postsynaptic calcium required for the induction of long-term potentiation. Neuron 9, 121–128 (1992).

PubMed 
    MATH 
    
                    Google Scholar 
                

Bittner, K. C., Milstein, A. D., Grienberger, C., Romani, S. & Magee, J. C. Behavioral time scale synaptic plasticity underlies CA1 place fields. Science 357, 1033–1036 (2017).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Lee, S. J., Escobedo-Lozoya, Y., Szatmari, E. M. & Yasuda, R. Activation of CaMKII in single dendritic spines during long-term potentiation. Nature 458, 299–304 (2009).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Friedrich, J., Zhou, P. & Paninski, L. Fast online deconvolution of calcium imaging data. PLoS Comput. Biol. 13, e1005423 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Ali, F. et al. Ketamine disinhibits dendrites and enhances calcium signals in prefrontal dendritic spines. Nat. Commun. 11, 72 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Chen, T. W. et al. Ultrasensitive fluorescent proteins for imaging neuronal activity. Nature 499, 295–300 (2013).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

DeFelipe, J., Arellano, J. I., Gomez, A., Azmitia, E. C. & Munoz, A. Pyramidal cell axons show a local specialization for GABA and 5-HT inputs in monkey and human cerebral cortex. J. Comp. Neurol. 433, 148–155 (2001).

PubMed 
    
                    Google Scholar 
                

Jun, J. J. et al. Fully integrated silicon probes for high-density recording of neural activity. Nature 551, 232–236 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Cameron, L. P. et al. 5-HT2ARs mediate therapeutic behavioral effects of psychedelic tryptamines. ACS Chem. Neurosci. 14, 351–358 (2023).

PubMed 
    MATH 
    
                    Google Scholar 
                

Hesselgrave, N., Troppoli, T. A., Wulff, A. B., Cole, A. B. & Thompson, S. M. Harnessing psilocybin: antidepressant-like behavioral and synaptic actions of psilocybin are independent of 5-HT2R activation in mice. Proc. Natl Acad. Sci. USA 118, e2022489118 (2021).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Sekssaoui, M., Bockaert, J., Marin, P. & Bécamel, C. Antidepressant-like effects of psychedelics in a chronic despair mouse model: is the 5-HT2A receptor the unique player? Neuropsychopharmacology 49, 747–756 (2024).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Choi, W. et al. Serotonin signals through a gut-liver axis to regulate hepatic steatosis. Nat. Commun. 9, 4824 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Yao, Z. et al. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. Cell 184, 3222–3241 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Aghajanian, G. K. & Marek, G. J. Serotonin induces excitatory postsynaptic potentials in apical dendrites of neocortical pyramidal cells. Neuropharmacology 36, 589–599 (1997).

PubMed 
    MATH 
    
                    Google Scholar 
                

Kaplan, A. L. et al. Bespoke library docking for 5-HT2A receptor agonists with antidepressant activity. Nature 610, 582–591 (2022).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Cao, D. et al. Structure-based discovery of nonhallucinogenic psychedelic analogs. Science 375, 403–411 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Dembrow, N. C., Zemelman, B. V. & Johnston, D. Temporal dynamics of L5 dendrites in medial prefrontal cortex regulate integration versus coincidence detection of afferent inputs. J. Neurosci. 35, 4501–4514 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Anastasiades, P. G., Collins, D. P. & Carter, A. G. Mediodorsal and ventromedial thalamus engage distinct L1 circuits in the prefrontal cortex. Neuron 109, 314–330 (2021).

PubMed 
    
                    Google Scholar 
                

Silberberg, G. & Markram, H. Disynaptic inhibition between neocortical pyramidal cells mediated by Martinotti cells. Neuron 53, 735–746 (2007).

PubMed 
    
                    Google Scholar 
                

Wu, S. J. et al. Cortical somatostatin interneuron subtypes form cell-type-specific circuits. Neuron 111, 2675–2692 (2023).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Suzuki, M. & Larkum, M. E. General anesthesia decouples cortical pyramidal neurons. Cell 180, 666–676 (2020).

PubMed 
    MATH 
    
                    Google Scholar 
                

Bharioke, A. et al. General anesthesia globally synchronizes activity selectively in layer 5 cortical pyramidal neurons. Neuron 110, 2024–2040 (2022).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Wilkinson, S. T., Holtzheimer, P. E., Gao, S., Kirwin, D. S. & Price, R. B. Leveraging neuroplasticity to enhance adaptive learning: the potential for synergistic somatic-behavioral treatment combinations to improve clinical outcomes in depression. Biol. Psychiatry 85, 454–465 (2019).

PubMed 
    
                    Google Scholar 
                

Feng, G. et al. Imaging neuronal subsets in transgenic mice expressing multiple spectral variants of GFP. Neuron 28, 41–51 (2000).

PubMed 
    MATH 
    
                    Google Scholar 
                

Tervo, D. G. et al. A designer AAV variant permits efficient retrograde access to projection. Neuron 92, 372–382 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Shamash, P., Carandini, M., Harris, K. D. & Steinmetz, N. A. A tool for analyzing electrode tracks from slice histology. Preprint at bioRxiv https://doi.org/10.1101/447995 (2018).

Wang, Q. et al. The Allen Mouse Brain Common Coordinate Framework: a 3D reference atlas. Cell 181, 936–953 (2020).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Claudi, F. et al. Visualizing anatomically registered data with brainrender. eLife 10, e65751 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Pologruto, T. A., Sabatini, B. L. & Svoboda, K. ScanImage: flexible software for operating laser scanning microscopes. Biomed. Eng. Online 2, 13 (2003).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Thevenaz, P., Ruttimann, U. E. & Unser, M. A pyramid approach to subpixel registration based on intensity. IEEE Trans. Image Process. 7, 27–41 (1998).

PubMed 
    MATH 
    
                    Google Scholar 
                

Holtmaat, A. et al. Long-term, high-resolution imaging in the mouse neocortex through a chronic cranial window. Nat. Protoc. 4, 1128–1144 (2009).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Mitrić, M. et al. Layer- and subregion-specific electrophysiological and morphological changes of the medial prefrontal cortex in a mouse model of neuropathic pain. Sci. Rep. 9, 9479 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Radnikow, G. & Feldmeyer, D. Layer- and cell type-specific modulation of excitatory neuronal activity in the neocortex. Front. Neuroanat. 12, 1 (2018).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Pnevmatikakis, E. A. & Giovannucci, A. NoRMCorre: an online algorithm for piecewise rigid motion correction of calcium imaging data. J. Neurosci. Methods 291, 83–94 (2017).

PubMed 
    
                    Google Scholar 
                

Ali, F. et al. Inhibitory regulation of calcium transients in prefrontal dendritic spines is compromised by a nonsense Shank3 mutation. Mol. Psychiatry 26, 1945–1966 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Lutcke, H., Gerhard, F., Zenke, F., Gerstner, W. & Helmchen, F. Inference of neuronal network spike dynamics and topology from calcium imaging data. Front. Neural Circuits 7, 201 (2013).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Friedman, A. et al. A corticostriatal path targeting striosomes controls decision-making under conflict. Cell 161, 1320–1333 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Boudreau, E. et al. Intraperitoneal catheter placement for pharmacological imaging studies in conscious mice. Lab. Anim. 39, 23–25 (2010).

MATH 
    
                    Google Scholar 
                

Siegle, J. H. et al. Open Ephys: an open-source, plugin-based platform for multichannel electrophysiology. J. Neural Eng. 14, 045003 (2017).

PubMed 
    MATH 
    
                    Google Scholar 
                

Buccino, A. P. et al. SpikeInterface, a unified framework for spike sorting. eLife 9, e61834 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Pachitariu, M. et al. Kilosort: realtime spike-sorting for extracellular electrophysiology with hundreds of channels. Preprint at bioRxiv https://doi.org/10.1101/061481 (2016).

Liu, R. J. & Aghajanian, G. K. Stress blunts serotonin- and hypocretin-evoked EPSCs in prefrontal cortex: role of corticosterone-mediated apical dendritic atrophy. Proc. Natl Acad. Sci. USA 105, 359–364 (2008).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Download references

We thank L. Sun for help with analysing the slice electrophysiology data. Psilocybin was provided by Usona Institute's Investigational Drug & Material Supply Program. The Usona Institute IDMSP is supported by A. Sherwood, R. Kargbo and K. Kaylo. This work was supported by NIH grants R01MH121848, R01MH128217, R01MH137047, U01NS128660, One Mind–COMPASS Rising Star Award (A.C.K.); NIH training grants T32GM007205 (P.A.D. and N.K.S.), T32NS041228 (C.L.); NIH fellowships F30DA059437 (P.A.D.) and F30MH129085 (N.K.S.); Source Research Foundation student grant (P.A.D.); NIH instrumentation grants S10RR025502 and S10OD032251 (Cornell Biotechnology Resource Center Imaging Facility); NIH grants R00NS114166, R01NS133434 and R01DA059378 (A.C.); State of Connecticut, Department of Mental Health and Addiction Services (A.C. and R.-J.L.).

These authors contributed equally: Ling-Xiao Shao, Clara Liao

Meinig School of Biomedical Engineering, Cornell University, Ithaca, NY, USA

Ling-Xiao Shao, Clara Liao, Pasha A. Davoudian, Neil K. Savalia, Quan Jiang, Cassandra Wojtasiewicz, Diran Tan, Jack D. Nothnagel, Samuel C. Woodburn, Olesia M. Bilash & Alex C. Kwan

Department of Psychiatry, Yale University School of Medicine, New Haven, CT, USA

Ling-Xiao Shao, Rong-Jian Liu, Alicia Che & Alex C. Kwan

Interdepartmental Neuroscience Program, Yale University School of Medicine, New Haven, CT, USA

Clara Liao, Pasha A. Davoudian & Neil K. Savalia

Medical Scientist Training Program, Yale University School of Medicine, New Haven, CT, USA

Pasha A. Davoudian & Neil K. Savalia

Graduate School of Medical Science and Engineering, KAIST, Daejeon, Republic of Korea

Hail Kim

Department of Psychiatry, Weill Cornell Medicine, New York, NY, USA

Alex C. Kwan

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

L.-X.S., C.L. and A.C.K. planned the study. L.-X.S. and C.L. conducted and analysed the imaging and behavioural experiments. P.A.D. and Q.J. conducted and analysed the electrophysiological experiments. N.K.S. and O.M.B. analysed the dendritic calcium imaging data. R.-J.L. and A.C. conducted slice electrophysiology experiments. Q.J. assisted in animal surgery. Q.J., D.T., C.W. and J.D.N. assisted in behavioural experiments and histology. S.C.W. and C.W. conducted pilot studies to validate the protocols for the behavioural assays. H.K. generated and provided the Htr2af/f mice. L.-X.S., C.L. and A.C.K. drafted the manuscript. All of the authors reviewed the manuscript before submission.

Correspondence to
                Alex C. Kwan.

A.C.K. has been a scientific advisor or consultant for Boehringer Ingelheim, Empyrean Neuroscience, Freedom Biosciences and Xylo Bio. A.C.K. has received research support from Intra-Cellular Therapies. The other authors declare no competing interests.

Nature thanks Adema Ribic, Bryan Roth and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available.

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

a, To express GFP in IT neurons, we injected AAV-CAG-FLEX-eGFP in the ACAd and medial MOs portion of medial frontal cortex and low titre of the retrogradely transported AAVretro-hSyn-Cre in the contralateral striatum of adult C57BL/6J mice. Post hoc histology and imaging of the GFP fluorescence in coronal sections shows ipsilateral and contralateral projections to various striatal and cortical regions. CP, caudoputamen. b, To express GFP in PT neurons, we injected AAV-CAG-FLEX-eGFP in the ACAd and medial MOs portion of medial frontal cortex and low titre of the retrogradely transported AAVretro-hSyn-Cre in the ipsilateral pons of adult C57BL/6J mice. Post hoc histology and imaging of the GFP fluorescence in coronal sections shows ipsilateral projections to striatum and subcortical regions including the pons (lower rightmost image). c, In vivo two-photon images of apical dendrites from PT and IT neurons targeted to express GFP using retrogradely transported viruses. d, Baseline spine density for all imaged dendrites on Day -3 prior to any psilocybin or saline administration. PT neurons have lower spine density than IT neurons (P < 0.001, two-sample t-test). Yellow, PT neurons. Purple, IT neurons. PT: n = 160 branches from 17 mice. IT: n = 142 branches from 16 mice. e, Similar to (d) for spine head width. PT: n = 1071 spines from 6 mice. IT: n = 615 spines from 5 mice. PT neurons have larger spine head width than IT neurons (P < 0.001, two-sample t-test). ***, p < 0.001. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, b, Potential differential effect of psilocybin on dendritic spine head width in frontal cortical PT and IT neurons. Spine head width in the apical tuft of PT neurons (a) after psilocybin (yellow; 1 mg/kg, i.p.) or saline (grey) across days, expressed as fold-change from baseline in first imaging session (day -3). b, Similar to (a) for IT neurons after psilocybin (purple) or saline (light purple). There was cell-type difference in psilocybin's effect on spine head width (interaction effect of treatment × time × cell type: P = 0.007, mixed effects model). These results show that PT neurons have a more pronounced increase in spine head width than IT neurons, indicative of a strengthening of excitatory connections in addition to gaining new inputs for PT neurons. This enlargement in spine head for PT neurons was less durable than the changes in spine density, returning to baseline after 35 days. We note the results should be interpreted with the consideration that the spine head width is near the spatial resolution limit for in vivo two-photon microscopy. c, Density of dendritic spines in the apical tuft of PT neurons after psilocybin (yellow; 1 mg/kg, i.p.) or saline (grey) across days. d-f, Similar to (c) for spine head width, formation rate, and elimination rate. g-j, Similar to (c-f) for IT neurons. These figure panels correspond to Fig. 1i–n and panel (a,b in this figure), except here across-dendrite values are shown, without taking advantage of the longitudinal data for within-dendrite baseline normalization. n = 8 mice (PT neurons, saline), n = 9 mice (PT neurons, psilocybin) in (a, c-f). n = 8 mice (IT neurons, saline), n = 8 mice (IT neurons, psilocybin) in (b, g-j). k-r, Psilocybin effects on structural plasticity in PT and IT neurons by sex of the animals. k, Density of dendritic spines in apical tuft of PT neurons in female (top row) and male mice (bottom row) after psilocybin (1 mg/kg, i.p.) or saline across days, expressed as fold-change from baseline in first imaging session (day -3). l, Similar to (k) for spine head width. m, Spine formation rate determined by number of new and existing spines in consecutive imaging sessions across two-day interval, expressed as difference from baseline in first interval (day -3 to day -1). n, Similar to (m) for elimination rate. n = 4 mice (PT neurons, saline, female), n = 4 (PT neurons, psilocybin, female), n = 4 (PT neurons, saline, male), n = 5 (PT neurons, psilocybin, male) in (k-n). o-r, Similar to (k-n) for IT neurons in female (top row) and male mice (bottom row). n = 5 mice (IT neurons, saline, female), n = 5 (IT neurons, psilocybin, female), n = 3 (IT neurons, saline, male), n = 3 (IT neurons, psilocybin, male). We did not detect effect of sex for any of the measures (interaction effect of treatment × sex × cell type, indicated in plots, mixed effects model). s-v, Psilocybin has no effect on spine protrusion length. s, Protrusion length of dendritic spines in apical tuft of PT neurons for all mice (left), or separately plotted for females (middle) and males (right), after psilocybin (1 mg/kg, i.p.) or saline across days, expressed as fold-change from baseline in first imaging session (day -3). t, Similar to (s) for IT neurons. Psilocybin had no detectable effect on spine protrusion length (main effect of treatment: P = 0.309, mixed effects model). u, Protrusion length of dendritic spines in apical tuft of PT neurons for all mice after psilocybin (1 mg/kg, i.p.) or saline across days, without taking advantage of the longitudinal data for within-dendrite baseline normalization. v, Similar to (u) for IT neurons. n = 8 mice (PT neurons, saline, 4 females, 4 males), n = 9 (PT neurons, psilocybin, 4 females, 5 males) in (s, u). n = 8 (IT neurons, saline, 5 females, 3 males), n = 8 (IT neurons, psilocybin, 5 females, 3 males) in (t, v). Data are mean and s.e.m. across dendrites. *, p < 0.05. ***, p < 0.001, post hoc with Bonferroni correction for multiple comparisons. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, Volumetric reconstruction from z-stack images of GFP-expressing IT neurons. IT neurons can be divided into two groups based on the laminar position of their cell body (200–400 μm or 450–600 μm). For the deep-lying IT neurons, sometimes the cell body could not be imaged due to the depth limitation of two-photon microscopy, but nonetheless the apical trunk was observed at >450 μm. b, Density of dendritic spines (left), spine head width (middle), and spine protrusion length (right) in apical tuft of IT neurons residing in layer 2/3 (top row) or layer 5 (bottom row) after psilocybin (1 mg/kg, i.p.) or saline across days, expressed as fold-change from baseline in first imaging session (day -3). The same mice were used for both depth ranges (200–400 μm and 450–600 μm). c, Left: spine formation rate determined by number of new and existing spines in consecutive imaging sessions across two-day interval, expressed as difference from baseline in first interval (day -3 to day -1). Right: similar to left for elimination rate. d-e, Similar to (b) for density of dendritic spines and spine head width but further divided the data based on the sex of the animal. n = 8 mice (IT neurons, saline, 5 females, 3 males), n = 8 (IT neurons, psilocybin, 5 females, 3 males) in (b-e). The analysis was motivated by the question: is psilocybin-evoked increase in spine size specific to cell type (IT versus PT), or specific to laminar position (layer 2/3 versus layer 5)? This is because IT neurons can be both superficial and deep, but PT is only found in deep layer. We detected no significant depth dependence for spine size for layer 2/3 and deep layer 5 IT neurons (interaction effect of treatment × soma depth, indicated in plots, mixed effects model). Data are mean and s.e.m. across dendrites. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, For head-twitch response assessed 10 min after drug administration, effect of chemogenetic inactivation in male (left) and female (right) Fezf2-creER mice during psilocybin (1 mg/kg, i.p.) or saline administration. Circle, individual animal. n = 14 (mCherry_psilocybin, 6 males and 8 females), n = 13 (mCherry_saline, 6 males and 7 females), n = 20 (hM4DGi_psilocybin, 12 males and 8 females), and n = 15 (hM4DGi_saline, 7 males and 8 females). b, Similar to (a) for Plxnd1-creER mice. n = 15 (mCherry_psilocybin, 8 males and 7 females), n = 12 (mCherry_saline, 6 males and 6 females), n = 11 (hM4DGi_psilocybin, 6 males and 5 females), and n = 13 (hM4DGi_saline, 7 males and 6 females). c-d, Similar to (a-b) for learned helplessness assessed 24 h after drug administration. For both male and female mice, there were significant effect of PT inactivation interfering with psilocybin's impact on learned helplessness (interaction effect of treatment and DREADD: P = 0.001 for males and P = 0.009 for females, two-factor ANOVA). Fezf2-creER mice: n = 13 (mCherry_psilocybin, 8 males and 5 females), n = 15 (mCherry_saline, 8 males and 7 females), n = 16 (hM4DGi_psilocybin, 9 males and 7 females), and n = 13 (hM4DGi_saline, 5 males and 8 females). Plxnd1-creER mice: n = 14 (mCherry_psilocybin, 7 males and 7 females), n = 11 (mCherry_saline, 5 males and 6 females), n = 11 (hM4DGi_psilocybin, 6 males and 5 females), and n = 11 (hM4DGi_saline, 5 males and 6 females). e-f, Similar to (a-b) for tail suspension test assessed 24 h after drug administration. For male mice, there was significant effect of PT inactivation interfering with psilocybin's impact on tail suspension (interaction effect of treatment and DREADD: P = 0.002 for males and P = 0.08 for females, two-factor ANOVA). Fezf2-creER mice: n = 14 (mCherry_psilocybin, 7 males and 7 females), n = 10 (mCherry_saline, 5 males and 5 females), n = 13 (hM4DGi_psilocybin, 6 males and 7 females), and n = 12 (hM4DGi_saline, 8 males and 4 females). Plxnd1-creER mice: n = 14 (mCherry_psilocybin, 7 males and 7 females), n = 9 (mCherry_saline, 5 males and 4 females), n = 9 (hM4DGi_psilocybin, 5 males and 4 females), and n = 9 (hM4DGi_saline, 5 males and 4 females). *, p < 0.05. ** p < 0.01. ***, p < 0.001, post hoc with Bonferroni correction for multiple comparisons. Data are mean and s.e.m. across mice. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, Chronic stress-induced resistance to fear extinction. b, Time spent freezing during conditioning. c, Time spent freezing during extinction session. Chemogenetic inactivation of frontal cortical PT neurons significantly diminished psilocybin's facilitating effect (interaction effect of treatment × DREADD: P = 0.003, two-factor ANOVA). d-e, Similar to (c) for extinction retention sessions. n = 13 (mCherry, saline), n = 17 (mCherry, psilocybin), n = 15 (hM4DGi, saline), n = 13 (hM4DGi, psilocybin) in (b-d). n = 13 (mCherry, saline), n = 16 (mCherry, psilocybin), n = 15 (hM4DGi, saline), n = 13 (hM4DGi, psilocybin) in (e). Data are mean and s.e.m. across mice. *, p < 0.05. ** p < 0.01. ***, p < 0.001, post hoc with Bonferroni correction for multiple comparisons. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, An example in vivo two-photon image of GCaMP6f-expressing apical dendrite and spines from a PT neuron. Dashed lines, the dendritic branch. Arrows, spines attached to the branch. b, Top, ΔF/F0 trace of a PT dendritic branch (ΔF/Fbranch). Middle, ΔF/F0 trace recorded from a dendritic spine attached to that branch (ΔF/Fspine). Bottom, the branch-subtracted spine calcium signal (ΔF/Fsynaptic), subtracting the scaled version of ΔF/Fbranch signal from the ΔF/Fspine signals. The synaptic calcium signals of dendritic spines shown and analysed in the paper are branch-subtracted ΔF/Fspine transients. Note that negative values can appear due to the subtraction procedure; however, negative transients are not detected by the peeling algorithm and therefore do not impact the subsequent analyses. Left, ΔF/F0 signals before psilocybin. Right, ΔF/F0 signals after psilocybin. c, Similar to (b) for ΔF/F0 traces in an IT neuron. d, Left, ΔF/F0 from two different dendritic branches of PT neurons before and after saline. Right, ΔF/F0 from two different dendritic branches of PT neurons, before and after psilocybin (1 mg/kg, i.p.). e, Similar to (d) for dendritic branches of IT neurons. f-g, Similar to (d-e) for dendritic spines.

a, Schematic illustrating how fluorescent transients are processed and analysed to derive event rate, amplitude, and frequency. b, Fractional change in frequency detected in dendritic branches of PT neurons (top row) and dendritic branches of IT neurons (bottom row) after psilocybin (PT, yellow; IT, purple) or saline (PT, grey; IT, light purple). c, Similar to (b) for amplitude. d, e, Similar to (b, c) for dendritic spines. PT: n = 140 branches from 4 mice (Saline), n = 149 branches from 4 mice (Psilocybin). IT: n = 90 branches from 3 mice (Saline), n = 95 branches from 3 mice (Psilocybin). ** p < 0.01. ***, p < 0.001, post hoc with Bonferroni correction for multiple comparisons. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, The three quality metrics, including the empirical distribution of recorded units and thresholds used for curation of single units to include for the analysis of the electrophysiology data. b, Mean spike waveform features for all opto-tagged neurons and other untagged cells in Fezf2-2A-creER and Plxnd1-2A-creER mice. There is no single feature of the spike waveform that can reliably classify the two types of opto-tagged neurons or from the untagged cells. c-g, More examples of opto-tagged neurons and untagged neurons. c, d, Spike raster of neurons classified as opto-tagged in Fezf2-2A-creER (left) and Plxnd1-2A-creER (right) mice. e-g, Spike raster of neurons classified as untagged in Fezf2-2A-creER (left) and Plxnd1-2A-creER (right) mice.

Source Data

a, Example sEPSC traces from 5 GFP+ layer 5 pyramidal neurons, with each set of 3 traces coming from recording of the same cell, including baseline (black), after bath application of 20 μM 5-HT (red), and after bath application of 20 μM 5-HT with 100 nM MDL100,907 (grey). Cells 1 and 2 are from control animals. Cells 3, 4, and 5 are animals with local 5-HT2A receptor knockout. b, Mean sEPSC amplitude from GFP+ layer 5 pyramidal neurons for baseline, 20 μM 5-HT, and 20 μM 5-HT + 100 nM MDL100,907 conditions, for control mice (grey) and local 5-HT2A receptor knockout mice (blue). Circle, individual cell. Mean and s.e.m. across cells. n = 22 cells (Baseline), n = 22 cells (+5-HT), n = 6 cells (+5-HT + MDL) from 4 mice in the Control condition, n = 23 cells (Baseline), n = 23 cells (+5-HT), n = 7 cells (+5-HT + MDL) from 4 mice in the 5-HT2AR local KO condition. c-e, Constitutive Camk2acre;Htr2af/f mice had reduced Htr2a transcripts and fewer psilocybin-induced head-twitch response. c, Breeding scheme to generate Camk2acre;Htr2af/f mice. d, Transcript expression via qPCR from whole-brain tissue. n = 2 (Control), n = 3 (Camk2acre;Htr2af/f). e, Head-twitch response induced by psilocybin (1 mg/kg, i.p.). Mean and s.e.m. across mice. n = 11 (Control), n = 12 (Camk2acre;Htr2af/f). ** p < 0.01. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

a, HTR2A antibody staining shows colocalization of 5-HT2A receptors and GFP-expressing cell bodies and neurites in the medial frontal cortex of a Thy1GFP line M mouse. b, To image dendrites from neurons without 5-HT2A receptors, we injected a low titre of AAV-hSyn-Cre-P2A-dTomato into the medial frontal cortex of Thy1GFP; Htr2af/f mouse. The subset of neurons with viral-mediated transgene expression would have Cre recombinase for knockout of 5-HT2A receptors and have dTomato for identification. Only dendrites that expressed both dTomato and GFP were scored. Control animals were Thy1GFP; Htr2a+/+. c, Density of dendritic spines in the apical tuft of dTomato- and GFP-expressing neurons after psilocybin (green) or saline (grey) in Thy1GFP; Htr2af/f mice or after psilocybin in Thy1GFP; Htr2a+/+ mice (orange), expressed as fold-change from baseline in first imaging session (day -3). n = 38 dendrites from 2 mice (Thy1GFP; Htr2a+/+, psilocybin), n = 49 dendrites from 5 mice (Thy1GFP; Htr2af/f, psilocybin), n = 98 dendrites from 3 mice (Thy1GFP; Htr2af/f, saline). d, HTR2A antibody staining shows colocalization of 5-HT2A receptors and PT neurons targeted to express GFP using retrogradely transported virus in a C57BL/6J mouse (left). For PT neuron-targeted 5-HT2A receptor knockout, 5-HT2A receptors were absent in PT neurons targeted using retrogradely transported virus in a Htr2af/f mouse (right). e, From two-photon microscopy, spine head width across days, expressed as fold-change from baseline in first imaging session (day -3), in wild type mice after saline (light grey) or psilocybin (yellow) and in mice with PT neuron-targeted 5-HT2A receptor knockout after saline (grey) or psilocybin (blue). Post hoc test compared WT:saline and WT:psilocybin groups. f, Similar to (e) for spine protrusion length. g, Density of dendritic spines, spine head width, and spine protrusion length in the apical tuft of PT neurons across days, in wild type mice after saline (light grey) or psilocybin (yellow) and in mice with PT neuron-targeted 5-HT2A receptor knockout after saline (grey) or psilocybin (blue), without taking advantage of the longitudinal data for within-dendrite baseline normalization. h, Similar to (g) for formation rate and elimination rate. n = 7 mice (WT, saline), n = 5 (WT, psilocybin), n = 5 (PT_5-HT2AR KO, saline), n = 6 (PT_5-HT2AR KO, psilocybin) in (e-h). i, Spine head width on day 1 plotted separately for pre-existing versus newly formed spines in different conditions. n = 7 mice (WT, saline), n = 5 (WT, psilocybin), n = 4 (PT_5-HT2AR KO, saline), n = 5 (PT_5-HT2AR KO, psilocybin). j, From confocal microscopy, spine head width in the apical tuft of PT neurons in wild type mice after saline (light grey) or psilocybin (yellow) and in mice with PT neuron-targeted 5-HT2A receptor knockout after saline (grey) or psilocybin (blue, interaction effect of treatment × genotype: P = 0.025, two-factor ANOVA). Circle, individual dendritic segment. n = 7 mice (WT, saline), n = 5 (WT, psilocybin), n = 3 (PT_5-HT2AR KO, saline), n = 5 (PT_5-HT2AR KO, psilocybin). k, Similar to (j) protrusion length of dendritic spines. n = same as in (j). Data are mean and s.e.m. across dendrites. * p < 0.05. **, p < 0.01. ***, p < 0.001, post hoc with Bonferroni correction for multiple comparisons. Detailed sample size n values are provided in Methods. Statistical analyses are provided in Supplementary Table 1.

Source Data

Detailed statistics for all data. This table provides comprehensive statistical details for all datasets reported in the Article, including statistical analyses, P values, error estimates and sample sizes.

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

Reprints and permissions

Shao, LX., Liao, C., Davoudian, P.A. et al. Psilocybin's lasting action requires pyramidal cell types and 5-HT2A receptors.
                    Nature  (2025). https://doi.org/10.1038/s41586-025-08813-6

Download citation

Received: 05 August 2023

Accepted: 19 February 2025

Published: 02 April 2025

DOI: https://doi.org/10.1038/s41586-025-08813-6

Anyone you share the following link with will be able to read this content:

Sorry, a shareable link is not currently available for this article.




                            Provided by the Springer Nature SharedIt content-sharing initiative
                        

Advertisement



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited

Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.livescience.com/technology/engineering/supersonic-vehicles-could-become-stronger-faster-and-more-durable-thanks-to-new-findings'>Supersonic vehicles could become stronger, faster and more durable thanks to new findings</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.livescience.com', 'title': 'Live Science'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 16:01:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Surprising results from hypersonic air flow simulations could help design stronger, faster and more durable supersonic vehicles. 


When you purchase through links on our site, we may earn an affiliate commission. Here's how it works.


A close look at air flow around high-speed shapes reveals surprising turbulence, according to a new study. The findings, published March 7 in the journal Physical Review Fluids, could inform the design of future high-speed vehicles.

In the study, researchers used three-dimensional simulations to reveal unexpected disturbances around fast-moving cones.

At hypersonic speeds — above Mach 5, or more than 5 times the speed of sound (3,836 mph or 6,174 kilometers per hour) — the flow of air around a vehicle's surface becomes complex and bumpy. Most simulations assume that the flow is symmetrical around the whole cone, but until recently, studies of the transition from streamlined to turbulent were only possible in two dimensions so we couldn't be sure that there weren't any asymmetries in flow around a three-dimensional structure.

The findings could help engineers design stronger, faster vehicles able to withstand the extreme temperatures, pressures and vibrations felt during hypersonic flight.

"Transitioning flows are 3D and unsteady in nature, regardless of the flow geometry," study co-author Irmak Taylan Karpuzcu, an aerospace engineer at the University of Illinois Urbana-Champaign, said in a statement. "Experiments were conducted in 3D in the early 2000s [but they] didn't provide enough data to determine any 3D effects or unsteadiness because there weren't enough sensors all around the cone-shaped model. It wasn't wrong. It was just all that was possible then."

Using the Frontera supercomputer at the Texas Advanced Computing Center, Karpuzcu and aerospace engineer Deborah Levin simulated how air flow around a cone-shaped object — often used as a simplified model for hypersonic vehicles — changes in three dimensions at high speed. They studied both a single cone and a double cone, which helps scientists study how multiple shock waves interact with each other.

"Normally, you would expect the flow around the cone to be concentric ribbons, but we noticed breaks in the flow within shock layers both in the single and double cone shapes," Karpuzcu said.

Get the world's most fascinating discoveries delivered straight to your inbox.

These breaks were particularly prevalent around the tip of the cone. At high speeds, the shock wave lies closer to the cone, squeezing air molecules into unstable layers and amplifying instabilities in the airflow. The team confirmed their findings by running a program that tracks each simulated air molecule and captures how collisions between the molecules affect air flow.

—Stealth destroyer 1st to carry hypersonic missiles that travel 5 times the speed of sound — with testing imminent

—Never-ending detonations could blast hypersonic craft into space

—Startup Hermeus wants to build a hypersonic jet that flies at 5 times the speed of sound

The disturbances also seem to develop at high speeds. "As you increase the Mach number, the shock gets closer to the surface and promotes these instabilities. It would be too expensive to run the simulation at every speed, but we did run it at Mach 6 and did not see a break in the flow," Karpuzcu said.

The breaks could affect design considerations for hypersonic vehicles, which could be used for shipping, weapons and transportation, Karpuzcu said, as engineers will need to account for the newly observed discontinuities.

Skyler Ware is a freelance science journalist covering chemistry, biology, paleontology and Earth science. She was a 2023 AAAS Mass Media Science and Engineering Fellow at Science News. Her work has also appeared in Science News Explores, ZME Science and Chembites, among others. Skyler has a Ph.D. in chemistry from Caltech.

Please logout and then login again, you will then be prompted to enter your display name.

Flat, razor-thin telescope lens could change the game in deep space imaging — and production could start soon

NASA captures stunning new image of shock waves from next-gen supersonic plane as it flies across the sun

'Be ready to move quickly to higher ground': Forecaster delivers ominous warning of 1-in-1,000-year flood coming for central US


Live Science is part of Future US Inc, an international media group and leading digital publisher. Visit our corporate site.


©
Future US, Inc. Full 7th Floor, 130 West 42nd Street,
New York,
NY 10036. </p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08800-x'>A natural experiment on the effect of herpes zoster vaccination on dementia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 15:42:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement


Nature

                         (2025)Cite this article


Metrics details

Neurotropic herpesviruses may be implicated in the development of dementia1,2,3,4,5. Moreover, vaccines may have important off-target immunological effects6,7,8,9. Here we aim to determine the effect of live-attenuated herpes zoster vaccination on the occurrence of dementia diagnoses. To provide causal as opposed to correlational evidence, we take advantage of the fact that, in Wales, eligibility for the zoster vaccine was determined on the basis of an individual's exact date of birth. Those born before 2 September 1933 were ineligible and remained ineligible for life, whereas those born on or after 2 September 1933 were eligible for at least 1 year to receive the vaccine. Using large-scale electronic health record data, we first show that the percentage of adults who received the vaccine increased from 0.01% among patients who were merely 1 week too old to be eligible, to 47.2% among those who were just 1 week younger. Apart from this large difference in the probability of ever receiving the zoster vaccine, individuals born just 1 week before 2 September 1933 are unlikely to differ systematically from those born 1 week later. Using these comparison groups in a regression discontinuity design, we show that receiving the zoster vaccine reduced the probability of a new dementia diagnosis over a follow-up period of 7 years by 3.5 percentage points (95% confidence interval (CI) = 0.6–7.1, P = 0.019), corresponding to a 20.0% (95% CI = 6.5–33.4) relative reduction. This protective effect was stronger among women than men. We successfully confirm our findings in a different population (England and Wales's combined population), with a different type of data (death certificates) and using an outcome (deaths with dementia as primary cause) that is closely related to dementia, but less reliant on a timely diagnosis of dementia by the healthcare system10. Through the use of a unique natural experiment, this study provides evidence of a dementia-preventing or dementia-delaying effect from zoster vaccination that is less vulnerable to confounding and bias than the existing associational evidence.

Recently, evidence has grown that neurotropic herpesviruses may have a role in the pathogenesis of dementia1,2,3,4,5. One approach to targeting herpesviruses is vaccination. However, vaccines are also increasingly being recognized as eliciting a broader immune response that can have important off-target effects, particularly in the case of live-attenuated vaccines6,7,8,9. Such effects have frequently been observed to differ strongly by sex7.

To date, studies in cohort and electronic health record data on the effect of vaccination receipt on dementia have simply compared the occurrence of dementia among those who received a given vaccination and those who did not11. These studies have to assume that all characteristics that are different between those who are vaccinated and those who are not (and that are also related to dementia) have been sufficiently well measured and modelled in the analysis, such that no factors confound the relationship between vaccination receipt and dementia12. This assumption of no confounding bias is often implausible because it has to be assumed that the study has detailed data on factors that are difficult to measure, such as personal motivation or health literacy13. It is also an assumption that cannot be empirically verified.

We used a fundamentally different approach that takes advantage of the fact that, in Wales, starting on 1 September 2013, those born on or after 2 September 1933 were eligible for herpes zoster vaccination for at least 1 year, while those born earlier never became eligible14. Using detailed large-scale electronic health record data, we were able to compare adults who were ineligible for the vaccine because they were born immediately before the eligibility cut-off date with those born immediately after who were eligible. Importantly, individuals who are only a few weeks apart in age are not expected to differ systematically from each other. That is, all potential confounding variables are in expectation balanced between our comparison groups. By taking advantage of this unique natural experiment, we were able to avoid confounding more credibly than all existing studies on the topic15,16,17,18,19,20,21,22,23,24, which have simply compared vaccine recipients to non-recipients while trying to control for the myriad of differences between these groups.

Adults born immediately after the 2 September 1933 date-of-birth eligibility cut-off had a 47.2 percentage point higher probability (from 0.01% to 47.2%) of ever receiving the herpes zoster vaccine than those born immediately before this cut-off date. As expected, other than this abrupt change in herpes zoster vaccination uptake, patients were balanced across the 2 September 1933 date-of-birth eligibility threshold in their uptake of other preventive health services, past common disease diagnoses and educational attainment. We then used this ‘quasi-randomization' in a regression discontinuity analysis to first replicate the known finding from clinical trials that the herpes zoster vaccine reduces new diagnoses of shingles. Second, we extended this approach to an outcome—dementia—that was never assessed in clinical trials of the herpes zoster vaccine, and find that the vaccine reduces the probability of a new dementia diagnosis over a seven-year follow-up period by approximately one-fifth. Third, we show that the herpes zoster vaccine did not affect the occurrence of any other common causes of mortality or morbidity other than shingles and dementia. Similarly, we show that receipt of the herpes zoster vaccine did not lead to increased uptake of other vaccinations or preventive health measures. Fourth, we provide evidence that no other intervention (such as health insurance eligibility) in Wales used the identical date of birth (2 September 1933) as eligibility cut-off as was used to define eligibility for the herpes zoster vaccine. Fifth, we show that all findings remain similar when using a different analysis approach. Sixth, we show that changes in healthcare pathways as a result of a shingles episode are unlikely to explain our findings. Seventh, we provide exploratory evidence from our electronic health record data on the mechanism through which herpes zoster vaccination could affect dementia. Our study focuses on the live-attenuated herpes zoster vaccine (Zostavax; hereafter, zoster vaccine), because the newer recombinant subunit zoster vaccine (Shingrix) became available in the UK only after our follow-up period ended25.

We used the Secure Anonymised Information Linkage (SAIL) Databank26, which contains detailed electronic health record data on primary care visits from approximately 80% of primary care providers in Wales, linked to secondary care records and the country's death register data. The study population for our primary analyses consisted of all adults born between 1 September 1925 and 1 September 1942 who were registered with a primary care provider (which is the case for over 98% of adults residing in Wales27), resided in Wales and did not have a diagnosis of dementia at the time of the start of the zoster vaccine program in Wales (on 1 September 2013). Basic sociodemographic and clinical characteristics of the sample of 282,541 adults in our primary analysis cohort are shown in Supplementary Table 1.

In Wales, individuals born between 2 September 1933 and 1 September 1934 (16,595 adults in our data) became eligible for the zoster vaccine for at least 1 year on 1 September 2013. Eligibility was then progressively extended to younger, but not older, age cohorts annually on the basis of their exact date of birth (Methods).

We find that being born just 1 week after 2 September 1933, and therefore being eligible for the zoster vaccine for at least 1 year, caused an abrupt increase in the probability of ever receiving the zoster vaccine from 0.01% to 47.2% (P < 0.001; Fig. 1). This provides a unique opportunity to avoid confounding concerns because it is unlikely that individuals born immediately around the date-of-birth eligibility threshold systematically differ from each other by anything but a one-week difference in age and a large difference in the probability of receiving the zoster vaccine. We substantiate this empirically by showing that, at the time of the start date of the zoster vaccination program, neither the prevalence of common disease diagnoses (including having been diagnosed with dementia before the vaccination program rollout), dementia risk as predicted from a series of clinical and sociodemographic variables, nor the prevalence of preventive behaviours (other than zoster vaccine uptake) display a discontinuity at the date-of-birth eligibility threshold for the zoster vaccine (Fig. 1 and Supplementary Figs. 1–4). Thus, after flexibly controlling for age, our two comparison groups (one with a low and one with a high probability of receiving the zoster vaccine) born immediately on either side of the 2 September 1933 date-of-birth eligibility threshold are probably exchangeable with each other on all observed and unobserved potential confounding variables.

a–f, The date-of-birth eligibility cut-off led to a large discontinuity in zoster vaccine receipt (a), but there is baseline exchangeability across the cut-off for uptake of other preventive interventions (flu vaccine (d), pneumococcal polysaccharide vaccine (PPV) (e) and statin medications (f)) as well as past shingles (b) and dementia (c) diagnoses. The data source for this analysis was the SAIL database for Wales. All analyses were run on the same sample as those for the effect of the zoster vaccine on dementia occurrence. The exception is c, for which we did not exclude individuals with a diagnosis of dementia before the start of the zoster vaccine program. The grey dots show the mean value for each 10-week increment in week of birth. The grey shading of the dots is proportionate to the weight that observations from this 10-week increment received in the analysis.

Our analysis approach primarily compares those who were ineligible for zoster vaccination because they had their 80th birthday immediately before the program's start date with those who were eligible because they had their 80th birthday immediately after the start date. As is standard practice in regression discontinuity analyses28,29, the effect of actually receiving the vaccine (as opposed to merely being eligible) was determined using a two-stage least-squares regression, which divides the magnitude of the abrupt change in the outcome at the date-of-birth eligibility threshold by the magnitude of the abrupt change in vaccine uptake at this threshold (Methods). Thus, the fact that not all those who were eligible received zoster vaccination does not bias our analysis.

We first demonstrate that our approach successfully reproduces the known causal effect from clinical trials that the vaccine reduces the occurrence of shingles30. Specifically, using a regression discontinuity design28,29, we compared the occurrence of shingles between adults born close to either side of the date-of-birth eligibility threshold for the zoster vaccine. Consistent with the approach used by clinical trials of the zoster vaccine30, our outcome was whether or not an individual had at least one shingles diagnosis during the follow-up period. During our follow-up period of 7 years, a total of 14,465 among 296,324 adults in our sample had at least one diagnosis of shingles. Over the same follow-up time, we find that being eligible for the vaccine reduced the probability of having at least one shingles diagnosis by 1.0 (95% CI = 0.2–1.7; P = 0.010) percentage point (Fig. 2a), corresponding to a relative reduction of 18.8% (95% CI = 8.8–28.9). When calculating the effect of actually receiving the zoster vaccine, we find a reduction in the probability of having at least one shingles diagnosis of 2.3 (95% CI = 0.5–3.9; P = 0.011) percentage points over the seven-year follow-up period (Fig. 2b); an effect (37.2% (95% CI = 19.7–54.7) in relative terms) that is similar in size to that observed in clinical trials of the live-attenuated zoster vaccine (Zostavax)30.

a–c, Effect estimates of being eligible for (a), and having received (across different follow-up periods (b) and across different grace periods (c)), the zoster vaccine on the probability of having at least one shingles diagnosis during the follow-up period. For a, the MSE-optimal bandwidth is 145.7 weeks (95,227 adults). The grey dots show the mean value for each 10-week increment in week of birth. The grey shading of the dots is proportionate to the weight that observations from this 10-week increment received in the analysis. For b and c, the MSE-optimal bandwidth for our primary specification is 116.9 weeks (76,316 adults). The triangles (rather than points) depict our primary specification. The red (as opposed to white) fillings denote statistical significance (P < 0.05). Grace periods refer to time periods since the index date after which the follow-up time is considered to begin. The grey vertical bars show the 95% CIs around the point estimate of the regression coefficient (two-sided t tests).

We show that our estimated effect is not sensitive to the chosen functional form of the regression used to model the relationship of shingles occurrence with week of birth (Supplementary Fig. 5), the width of the week-of-birth window (bandwidth) around the date-of-birth eligibility cut-off that defines our analysis sample (Supplementary Fig. 6a) or to different grace periods (Fig. 2c). With ‘grace periods', we refer to time periods since the index date after which the follow-up time is considered to begin (Methods). There was also a strong indication that the zoster vaccine reduced the probability of having at least one diagnosis of postherpetic neuralgia (a common complication of shingles), although this effect did not reach statistical significance in all specifications (Supplementary Fig. 7).

Given the neuropathological overlap between dementia types and the difficulty in distinguishing dementia types clinically31, as well as our reduced statistical power when studying less-common outcomes, we defined dementia as dementia of any type or cause as our outcome. We considered an individual to have developed dementia if there was a new diagnosis of dementia in our electronic health record data (which includes all diagnoses made in primary or secondary care) or dementia was listed as a primary or contributory cause of death on the death certificate. The Read and ICD-10-codes used to define dementia are listed in the Supplementary Codes. During our seven-year follow-up period, 35,307 among 282,541 adults in our sample were newly diagnosed with dementia.

Using our regression discontinuity approach, we estimate that the effect of being eligible for the zoster vaccine is a 1.3 (95% CI = 0.2–2.7; P = 0.022) percentage points absolute and 8.5% (95% CI = 1.9–15.1) relative reduction in the probability of a new dementia diagnosis over 7 years (Fig. 3a). Scaled to account for the fact that not all those who were eligible received the vaccine, we find that actually receiving the zoster vaccine reduced the probability of a new dementia diagnosis by 3.5 (95% CI = 0.6–7.1; P = 0.019) percentage points, corresponding to a relative reduction of 20.0% (95% CI = 6.5–33.4) (Fig. 3b). The effect estimates were generally not sensitive to different grace periods (Fig. 3c), the functional form of our regressions (Supplementary Fig. 8) nor the width of the week-of-birth window (bandwidth) drawn around the date-of-birth eligibility cut-off (Supplementary Fig. 6b). We also find significant effects of the zoster vaccine on reducing dementia diagnoses if a diagnosis is defined solely as a new prescription of a medication (donepezil hydrochloride, galantamine, rivastigmine or memantine hydrochloride) that is frequently prescribed to slow the progression of Alzheimer's disease (Supplementary Table 2 (column 2)). Similarly, the effects remain similar when adjusting for all input variables to the Dementia Risk Score32 (as recorded before 1 September 2013) (Supplementary Table 2 (column 7)).

a–c, Effect estimates of being eligible for (a), and having received (across different follow-up periods (b) and across different grace periods (c)), the zoster vaccine on new diagnoses of dementia. For a, the MSE-optimal bandwidth is 134.4 weeks (83,167 adults). The grey dots show the mean value for each 10-week increment in week of birth. The grey shading of the dots is proportionate to the weight that observations from this 10-week increment received in the analysis. For b and c, the MSE-optimal bandwidth for our primary specification is 90.6 weeks (56,098 adults). The triangles (rather than points) depict our primary specification. The red (as opposed to white) fillings denote statistical significance (P < 0.05). Grace periods refer to time periods since the index date after which the follow-up time is considered to begin. The grey vertical bars show the 95% CIs around the point estimate of the regression coefficient (two-sided t tests).

The key strength of our study is that a confounding variable can bias our analysis only if the variable changes abruptly at the 2 September 1933 date-of-birth threshold28,29. Thus, confounding bias could occur if another intervention also used the date of birth cut-off of 2 September 1933 as an eligibility criterion. Such an intervention is unlikely to affect only the risk of developing dementia without also influencing other health outcomes. We therefore implemented the same regression discontinuity approach as we did for shingles and dementia for the ten leading causes of disability-adjusted life years and mortality for the age group 70+ years in Wales in 201933, and all conditions that are part of the Charlson Comorbidity Index34. As shown in Supplementary Figs. 9 and 10, we generally do not detect effects of zoster vaccination on new diagnoses of these other common health outcomes.

We undertook four additional types of analysis, all of which provide evidence against another intervention having used the identical day-month-year combination (2 September 1933) as was used as the date-of-birth eligibility threshold for the zoster vaccine rollout. First, we show that the 2 September 1933 date-of-birth threshold does not affect the probability of taking up other preventive health interventions (Supplementary Fig. 11). Second, we examined whether the day–month (that is, 2 September) date-of-birth cut-off used for zoster vaccine eligibility was also used by other interventions that affect dementia risk. We did so by implementing the identical analysis as for 1 September 2013 (the actual date on which the zoster vaccine program started) for 1 September of each of the three years before and after 2013. Thus, for example, when shifting the start date of the program to 1 September 2012, we compared those around the 2 September 1932 date-of-birth threshold with the follow-up period starting on 1 September 2012. As an additional check that enabled us to maintain the length of the seven-year follow-up period used in our primary analyses, we shifted the program start date to 1 September of each of the 6 years preceding (but not after) 2013. As expected, for both of these checks, we find a significant effect on dementia occurrence only for the date-of-birth cutoff (2 September 1933) that was actually used by the zoster vaccination program (Supplementary Figs. 12 and 13). Third, we find that there is no difference in the seven-year incidence of dementia between age cohorts around the 2 September 1933 date-of-birth threshold for the seven-year period before the zoster vaccine rollout (Supplementary Fig. 14). Fourth, using data from the 2011 Census, we show in Supplementary Figs. 15–17 that there are no discontinuities across the 2 September 1933 threshold in the proportion of individuals in Wales who reached a particular level of education.

As an additional test of the robustness of our findings, we implemented all primary analyses using a difference-in-differences instrumental variable analysis (DID-IV) that takes advantage of the fact that the only 2 September date-of-birth threshold at which we would expect an abrupt change in the outcome is the 2 September threshold in 1933 (that is, the day–month–year combination that was used as eligibility cut-off by the zoster vaccination program). In doing so, our analysis relaxes the continuity assumption of regression discontinuity (that is, the assumption that potential confounding variables do not display a sudden change at the 2 September 1933 date-of-birth eligibility threshold), and instead assumes that (in the absence of the zoster vaccination program) a possible discontinuity in the outcome at the 2 September 1933 threshold is not different in size from a discontinuity at the 2 September threshold in previous birth years. Details of our approach are provided in the Methods. Encouragingly, the effect of zoster vaccine receipt on the probability of a new dementia diagnosis during our seven-year follow-up period is remarkably similar between the DID-IV and regression discontinuity approach (−3.1 (95% CI = −5.8 to −0.4, P = 0.024) versus −3.5 (95% CI = −7.1 to −0.6, P = 0.019) percentage points) (Fig. 4). This is also the case for the outcomes of shingles and postherpetic neuralgia (Fig. 4). We conducted the same checks for balance in health characteristics between our comparison groups for the DID-IV as we implemented for our regression discontinuity analyses (Supplementary Fig. 18). We also verified that our DID-IV approach yields significant effects only for the outcomes of dementia, shingles and postherpetic neuralgia, but not for any other common health outcomes (Supplementary Fig. 18).

Comparison of absolute effect estimates of having received the zoster vaccine on new diagnoses of dementia, shingles and postherpetic neuralgia between the DID-IV and the regression discontinuity analyses. The data source for this analysis was the SAIL database for Wales. The sample size for the dementia outcome is 96,767 adults and the sample for the shingles and postherpetic neuralgia outcomes is 105,258 adults. P values were calculated using two-sided t-tests. The P value for the DID-IV effect on shingles is 0.001. The error bars depict the 95% CIs around the point estimate of the regression coefficient (two-sided t-tests).

A protective effect of zoster vaccination on dementia diagnoses could arise from three (non-mutually exclusive) mechanisms: (1) changes in healthcare pathways as a result of a shingles episode; (2) a reduction in reactivations of the varicella zoster virus (VZV); and (3) a VZV-independent immunomodulatory effect (for example, one mediated through heterologous adaptive immunity or trained innate immunity). In this section, we present evidence to examine each of these mechanisms.

Reduced healthcare use resulting averted shingles episodes from zoster vaccination receipt could have translated to fewer opportunities for the health system to (1) diagnose dementia (ascertainment bias); or (2) implement care changes (for example, initiation of a new medication) that increase the risk of being diagnosed with dementia in the future. It is important to point out that this mechanism is unlikely to fully explain our findings, because the size of our effect estimates for reductions in shingles episodes from zoster vaccination were considerably too small to plausibly account for the observed reduction in dementia diagnoses.

We nonetheless conducted five types of analysis to examine this potential mechanism further. First, if shingles episodes presented an opportunity for the health system to diagnose dementia, then they would probably also present an opportunity to diagnose other chronic conditions. We therefore applied the same regression discontinuity approach as for shingles and dementia to all chronic conditions that are either among the ten leading causes of disability-adjusted life years and mortality for the age group 70+ years in Wales in 201933 or part of the Charlson Comorbidity Index34. We plotted our estimates across one-year increments in the follow-up period. With the exception of rheumatological diseases, we show that being eligible for the zoster vaccine did not have an effect on new chronic disease diagnoses (Supplementary Fig. 19). Second, we adjusted our regressions for the frequency of health service use (the number of primary care visits, outpatient visits, hospital admissions and influenza vaccinations received) during the follow-up period, which did not substantially change our effect estimates (Supplementary Table 2 (column 4)). Third, we implemented our analyses when restricting the analysis cohort to the 247,784 (87.6% of the analysis cohort for our primary analyses) patients who visited their primary care provider at least once a year during each of the 5 years before the start of the zoster vaccine rollout. The rationale for this analysis is that, among patients who already interact frequently with the health system, a reduction of one further contact with the health system due to an averted shingles episode is less likely to affect the probability of detecting undiagnosed dementia. The effect sizes among this cohort of frequent healthcare users remain similar to those in our primary analytical cohort (Supplementary Table 2 (column 3)). Fourth, we added whether individuals experienced a shingles episode during the follow-up period as a covariate in our primary regression discontinuity analysis. We found that adjusting our analysis for shingles episodes did not substantially change our point estimate (Supplementary Fig. 20). Fifth, we implemented an event study among those participants in the mean-squared-error (MSE)-optimal bandwidth of our primary regression discontinuity analysis for dementia who received a shingles diagnosis during the follow-up period. To investigate whether episodes of shingles led to changes in healthcare received by patients, we examined the effect of the shingles diagnosis on the following outcomes in each of the 36 months after the diagnosis: (1) the probability of receiving a new dementia diagnosis; (2) a set of indicators of health service use; (3) the probability of receiving a new medication prescription for antiviral drugs, opioid medications, gabapentin or pregabalin, and any of 216 medications that were associated with an increased risk of dementia in another analysis in the SAIL database23; and (4) the probability of being diagnosed with any of the chronic conditions that are part of the Charlson Comorbidity Index34. We found that shingles diagnoses did not increase the probability of receiving a new dementia diagnosis in the months after the shingles diagnosis, and led to only short-term increases in healthcare service use and new medication prescriptions (Supplementary Fig. 21). The increase in the probability of receiving a gabapentin or pregabalin prescription in the months after the shingles episode, while more sustained, was small in magnitude. Similarly, the increase in the probability of being diagnosed with any chronic condition in the month of a shingles episode compared with the month before the episode was less than one percentage point (Supplementary Fig. 21).

As the effect of zoster vaccination on shingles episodes is moderate (Fig. 2), and the five types of analysis in this section document only small and short-lived effects of shingles episodes on healthcare pathways, even the most conservative assumptions about the effect of these care paths on dementia imply that changes in healthcare as a result of a shingles episode cannot explain our findings.

As described in the previous section, adjusting our regression discontinuity analysis for whether a patient had a record of at least one shingles episode during the follow-up period did not change our point estimate substantially (Supplementary Fig. 20). However, conclusions from this analysis regarding reductions in VZV reactivations as the effect mechanism are limited by the fact that (1) zoster vaccination probably reduces both clinical as well as subclinical reactivations of VZV30,35; and (2) having a shingles episode may be an unreliable indicator of the degree of subclinical VZV reactivations experienced during the entire follow-up period, given that shingles episodes may boost immunity for VZV30,35. We therefore conducted the following analyses to further examine reductions in VZV reactivations as the effect mechanism.

First, we examined the time during the follow-up period at which the effect of zoster vaccination on dementia appears to begin. Specifically, among patients who were born in close proximity to the 2 September 1933 date-of-birth threshold, we plotted the Kaplan–Meier and cumulative incidence curves for dementia for those who were eligible versus ineligible for zoster vaccination (Methods). If the effect mechanism is through a reduction in VZV reactivations, then one would expect that the effects of the vaccine on reductions in clinical and subclinical reactivations of the virus would begin before observing an effect on dementia. The live-attenuated zoster vaccine is thought to begin being efficacious within weeks after vaccine administration30,36. Consistent with the principle that the effect on VZV reactivations should precede the dementia effect, we observe that the reduction in the incidence of dementia begins to emerge only after more than one year, both among the full population as well as among women only (Supplementary Fig. 22).

Second, while a shingles episode may boost VZV immunity and, therefore, reduce subsequent subclinical VZV reactivations30,35, individuals who experience multiple episodes as opposed to a single shingles episode during the follow-up period probably experience a greater degree of both clinical and subclinical VZV reactivations during the follow-up period30. Using propensity score matching (Methods), we therefore compared the association with dementia from experiencing multiple versus a single shingles episode. We find a higher incidence of dementia among those who experienced multiple shingles episodes (Supplementary Fig. 23).

Third, if VZV reactivations increase the risk of dementia, then limiting the degree of replication of the virus during a shingles episode through antiviral medication could be expected to decrease dementia incidence. Using a multivariable Cox proportional hazards model (Methods), we therefore compared the association with dementia between individuals whose shingles episode was treated with antiviral medication and those for whom the episode was untreated. We find that antiviral treatment of a shingles episode is associated with a reduced incidence of dementia (Supplementary Fig. 23).

To probe this mechanism, we take advantage of two observations on pathogen-independent immunomodulatory effects from vaccination in the literature: they tend to (1) vary strongly by sex, with beneficial effects from live-attenuated vaccination often seen only in female but not male individuals6,7,8; and (2) depend on the receipt of other vaccines before, or at the same time as, receipt of the vaccine in question6,7,8. Consistent with these observations, we find that the effect of zoster vaccination on new diagnoses of dementia was markedly greater among women than men (Fig. 5 and Supplementary Table 3 (column 1)). There was no significant difference between women and men in the effect of the zoster vaccine on diagnoses of shingles and postherpetic neuralgia (Supplementary Table 3 (columns 2 and 3)). Similarly, the magnitude of the abrupt increase in vaccine uptake at the 2 September 1933 date-of-birth eligibility threshold was comparable between women and men (Supplementary Fig. 24), with a slightly larger magnitude among men.

a–f, Effect estimates of being eligible for (a (women) and d (men)) and having received (b and c (women) and e and f (men); across different follow-up periods (b and e) and across different grace periods (c and f)) the zoster vaccine on new diagnoses of dementia, separately for women and men. The data source for this analysis was the SAIL database for Wales. The triangles (rather than points) depict our primary specification. Red (as opposed to white) fillings denote statistical significance (P < 0.05). Grace periods refer to time periods since the index date after which the follow-up time is considered to begin. The grey vertical bars depict the 95% CIs around the point estimate of the regression coefficient (two-sided t-test). The grey dots show the mean value for each 10-week increment in week of birth. For a, among women, the MSE-optimal bandwidth is 95.5 weeks (32,601 women). For b and c, among women, the MSE-optimal bandwidth for our primary specification is 149.1 weeks (50,816 women). For d, among men, the MSE-optimal bandwidth for our primary specification is 121.3 weeks (33,725 men). For e and f, among men, the MSE-optimal bandwidth for our primary specification is 91.8 weeks (25,563 men). The grey shading of the dots is proportionate to the weight that observations from this 10-week increment received in the analysis.

We also find strong effect heterogeneity by receipt of previous influenza vaccination. Specifically, the protective effect of zoster vaccination for dementia was larger among those who did not recently receive the influenza vaccine (Supplementary Fig. 25). Influenza vaccination is the only vaccine that was provided within the 5 years preceding zoster vaccination eligibility to a substantial proportion of individuals in our study population (pneumococcal vaccination is already provided at age 65 years in the United Kingdom37).

Finally, we examined the differences in the effect of the zoster vaccine on dementia incidence between those with versus without an autoimmune or allergic condition. Our reasoning for this analysis was based on the observation that the incidence of shingles is increased among individuals with an autoimmune or allergic condition38,39,40,41, while there do not appear to be major differences in vaccine immunogenicity and its relative effectiveness for shingles prevention between those with versus without such conditions30. Thus, if the protective effect of zoster vaccination for dementia is mainly driven through a reduction of clinical and subclinical virus reactivations, then those with an autoimmune condition will likely benefit equally or more than those without such a condition. However, because autoimmune and allergic conditions are generally characterized by a heightened activation of the (adaptive) immune system42,43, individuals with such a condition might benefit less from further activation of more generalized, VZV-independent, immune system pathways than those without such a condition. Consistent with this second hypothesis, we observe suggestive evidence for stronger effectiveness of the zoster vaccine for dementia among those without an autoimmune or allergic condition than those with such a condition (Supplementary Fig. 25). The patterns that we observe remain largely unaffected by whether or not patients were taking any immunosuppressive medications in the year preceding the start of the zoster vaccination program.

Thus, overall and with the caveat that these exploratory analyses are suggestive only, our analyses indicate that both a mechanism of action through a reduction in clinical and subclinical reactivations of VZV as well as through a VZV-independent immunomodulatory effect are plausible. Importantly, these two mechanisms are not mutually exclusive.

Here we found that the zoster vaccine reduced the probability of a new dementia diagnosis by approximately one-fifth over a seven-year follow-up period. By taking advantage of the fact that the unique way in which the zoster vaccine was rolled out in Wales constitutes a natural experiment, and examining each possible remaining source of bias, our study provides evidence that is more likely to be causal in nature than the existing, exclusively associational15,16,17,18,19,20,21,22,23,24, evidence on this topic. Our substantial effect sizes, combined with the relatively low cost of the zoster vaccine, imply that, if these findings are truly causal, the zoster vaccine will be both far more effective as well as cost-effective in preventing or delaying dementia than existing pharmaceutical interventions.

Our quasi-experimental approach reduces the probability of confounding compared with more standard associational analyses. Moreover, we have provided evidence from a series of analyses against any of the possible remaining sources of bias being a likely explanation of our findings. Nonetheless, it is possible (even if statistically unlikely) that our findings are due to chance. Confirmation of our findings in other populations, settings and data sources is therefore critical. Importantly, we have successfully confirmed our findings using country-wide death certificate data from England and Wales10. Specifically, because England rolled out the zoster vaccine in an almost identical way to Wales44, we were able to use the same quasi-experimental approach as in our electronic health record data from Wales to determine the effect of eligibility for zoster vaccination based on one's date of birth on deaths for which the underlying cause was recorded as being dementia. We found that, over a nine-year follow-up period, approximately 1 in 20 such deaths were averted from being eligible for zoster vaccination. This study constitutes an important confirmation of our results because it analysed a different population (England's population accounts for approximately 95% of England's and Wales's combined population45), type of data (death certificates as opposed to electronic health records) and outcome (deaths due to dementia). In addition to this confirmation of our results in mortality data, the probability of a chance finding is further reduced by the fact that we successfully replicate our main findings using a second analysis approach (DID-IV) and that our effect sizes remain stable across a multitude of analysis choices, including choice of grace periods, follow-up periods, study population definitions (for example, restriction to frequent healthcare users), functional form of our regressions, width of the week-of-birth window drawn around the date-of-birth eligibility cut-off and index date definitions.

We observed large differences in the effect of zoster vaccination on dementia between women and men, with women benefitting more than men. In our view, these large differences between women and men are plausible for several reasons. First, we cannot exclude the possibility of substantial reductions in new dementia diagnoses from zoster vaccination among men, especially given the lower incidence of dementia in older age among men than women in our data and, therefore, our wider confidence intervals for analyses among men. Second, off-target effects of vaccines have often been observed to be far stronger among female than male individuals, with female individuals benefiting more from live-attenuated vaccines in particular6,7. Third, there appear to be important sex differences in the immunological response to vaccines more generally46. Lastly, there is a growing body of evidence that there may be differences in the pathogenesis of dementia between women and men47.

Other than investing into randomized trials, investments into basic science research on the potential role of VZV and the immune response to the zoster vaccine in the pathogenesis of dementia could provide critical mechanistic insights. There are already several lines of evidence on plausible mechanistic pathways that link VZV reactivations to dementia. Specifically, VZV reactivations have been found to lead to long-lasting cognitive impairment through vasculopathy48,49, amyloid deposition and aggregation of tau proteins50, neuroinflammation51,52,53,54, as well as a similar spectrum of cerebrovascular disease as seen in Alzheimer's disease, including small to large vessel disease, ischaemia, infarction and haemorrhage51,52,53,54,55,56. As suggested by a recent study57, it may also be the case that reducing subclinical and clinical reactivations of VZV reduces reactivations of the herpes simplex virus-1 in the brain through neuroinflammatory pathways. This mechanism would link VZV to the body of literature on the role of herpes simplex virus-1 in the pathogenesis of dementia1,2,3,4,5. Nonetheless, our exploratory analyses on the effect mechanism that links zoster vaccination to dementia suggest that both a mechanism through reducing clinical and subclinical reactivations of VZV as well as a pathogen-independent immune mechanism are plausible. Some of these possible pathogen-independent immune mechanisms have recently been detailed elsewhere58.

Our study has several limitations. First, our outcome ascertainment probably suffers from some degree of under-detection, both in whether and in how timely a fashion dementia is diagnosed. Importantly, because the probability of under-detecting dementia, as well as the delay in doing so, is unlikely to change abruptly at the 2 September 1933 date-of-birth eligibility threshold for zoster vaccination, this outcome misclassification is most likely non-differential. Our effect estimates are therefore likely to be conservative (that is, our absolute effect sizes would be an underestimate of the true absolute effect magnitude). Similarly, changes in the accuracy and timeliness of dementia ascertainment over the years of our follow-period, such as due to changing clinical practice or health system incentives to detect and record dementia, affected those born immediately before versus immediately after 2 September 1933 equally. We would therefore not expect these changes to be a source of bias in our analyses. Second, we are unable to provide estimates for the effectiveness of the zoster vaccine for reducing dementia occurrence in age groups other than those who were weighted most heavily in our regression discontinuity analyses (primarily those aged 79 to 80 years). Third, the COVID-19 pandemic probably affected the timeliness with which dementia was diagnosed. However, the follow-up period used in our primary analyses ended before the start of the COVID-19 pandemic. Moreover, because the pandemic affected those born just before versus just after 2 September 1933 equally, pandemic-related under-detection of dementia is unlikely to bias our relative effect estimates. Fourth, we were limited to a maximum follow-up period of 8 years. Our study can therefore not inform on the effectiveness of the zoster vaccine for reducing dementia occurrence beyond this time period. Lastly, because the newer recombinant subunit zoster vaccine (Shingrix) replaced the live-attenuated zoster vaccine (Zostavax) in the United Kingdom only in September 202325, which is after our follow-up period ended, our effect estimates apply to the live-attenuated zoster vaccine only.

The live-attenuated zoster vaccine (Zostavax) was made available to eligible individuals in Wales through a staggered rollout system starting on 1 September 2013. Under this system, individuals aged 71 years or older were categorized into three groups on 1 September of each year: (1) an ineligible cohort of those aged 71 to 78 years (or 77 years, depending on the year of the program), who became eligible in the future; (2) a catch-up cohort, consisting of individuals aged 79 years (or 78 years, again depending on the year of the program); and (3) those who were ineligible as they were aged 80 years or older and who never became eligible.

Our analysis focused on adults born between 1 September 1925 (88 years old at program start) and 1 September 1942 (71 years old at program start). Those born between 1 September 1925 and 1 September 1933 never became eligible, whereas those born between 2 September 1933 and 1 September 1942 became progressively eligible in a catch-up cohort. Specifically, the vaccine was offered to those born between 2 September 1933 and 1 September 1934 in the first year of the program (1 September 2013 to 31 August 2014); those born between 2 September 1934 and 1 September 1936 in the second year (1 September 2014 to 31 August 2015); those born between 2 September 1936 and 1 September 1937 in the third year (1 September 2015 to 31 August 2016); and those born between 2 September 1937 and 1 September 1938 in the fourth year (1 September 2016 to 31 August 2017). As of 1 April 2017, individuals become eligible for the vaccine on their 78th birthday and remain eligible until their 80th birthday. Our analysis principally compared individuals born on or shortly after 2 September 1933, to individuals who never became eligible as they were born shortly before 2 September 1933. We show in Supplementary Figs. 26–28 that most eligible individuals, especially in the first two eligibility cohorts, which are the focus of our analysis, took up the vaccination during their first year of eligibility (as opposed to during later years) and that vaccination uptake in these first two eligibility cohorts was of a similar magnitude.

Healthcare in Wales is provided through the Welsh National Health Service (NHS), which is part of the United Kingdom's single-payer single-provider healthcare system59. NHS Wales and Swansea University created the SAIL Databank26,60,61,62,63,64, which includes full electronic health record data for primary care visits linked to information on hospital-based care as well as the country's death register data.

SAIL generates a list of all individuals who have ever been registered with a primary care provider in Wales (which is the case for over 98% of adults residing in Wales27) from the Welsh Demographic Service Dataset65. SAIL then links this universe of individuals to each of the following datasets. Electronic health record data from primary care providers is made available in SAIL through the Welsh Longitudinal General Practice dataset66, which contains data from approximately 80% of primary care practices in Wales and 83% of the Welsh population. These electronic health record data use Read codes, which provide detailed information on patients and their care encounters, including diagnoses, clinical signs and observations, symptoms, laboratory tests and results, procedures performed and administrative items67. Zoster vaccination was defined using both codes for the administration of the vaccine as well as product codes (Supplementary Table 1). Diagnoses made and procedures performed in the hospital setting (as part of inpatient admissions or day-case procedures) are provided in SAIL through linkage to the Patient Episode Database for Wales68, which begins in 1991 and contains data for all hospital-based care in Wales as well as hospital-based care provided in England to Welsh residents. Procedures are encoded using OPCS-4 codes69 and diagnoses using ICD-10 codes70. Attendance information at any NHS Wales hospital outpatient department is provided through linkage to the Outpatient Database for Wales71, which starts in 2004. ICD-10-encoded diagnoses of cancers are identified through linkage to the Welsh Cancer Intelligence and Surveillance Unit72, which is the national cancer registry for Wales that records all cancer diagnoses provided to Welsh residents wherever they were diagnosed or treated. This dataset begins in 1994. Finally, cause-of-death data are provided for all Welsh residents (regardless of where they died in the United Kingdom) through linkage to the Annual District Death Extract73, which begins in 1996 and includes primary and contributory causes of death from death certificates. Dates for deaths were those on which the death was registered, as opposed to when it occurred. Cause-of-death data use ICD-9 coding until 2001 and ICD-10 coding thereafter.

When testing for any discontinuities in educational attainment across the date-of-birth eligibility threshold, we used a dataset provided by the Office for National Statistics (ONS)74. This dataset was generated by the ONS from the 2011 UK Census for all usual residents aged 16 or over, born in Wales between January 1925 and December 1950, regardless of their employment status. The data were categorized by the ONS by sex, month and year of birth (January 1925 to December 1950), highest level of qualification and occupation.

Ethics approval was granted by the Information Governance Review Panel (IGRP, application number, 1306). Composed of government, regulatory and professional agencies, the IGRP oversees and approves applications to use the SAIL databank. All analyses were approved and considered minimal risk by the Stanford University Institutional Review Board on 9 June 2023 (protocol number, 70277).

Our study population consisted of 296,603 individuals born between 1 September 1925 and 1 September 1942 who were registered with a primary care provider in Wales on the start date of the zoster vaccine program rollout (1 September 2013). As we only had access to the date of the Monday of the week in which an individual was born, we were unable to determine whether the individuals born in the cut-off week starting on 28 August 1933 were eligible for the zoster vaccine in the first year of its rollout. We therefore excluded 279 individuals born in this particular week. Among the remaining individuals, 13,783 had a diagnosis of dementia before 1 September 2013 and were therefore excluded from the analyses with new diagnoses of dementia as outcome. The size of our final analysis cohort for all primary analyses for new dementia diagnoses was therefore 282,541. This analysis cohort was used for all analyses except those with incidence of dementia before zoster vaccination program start, shingles and postherpetic neuralgia as outcomes; analyses for which we did not exclude individuals with a dementia diagnosis before 1 September 2013.

We followed individuals from 1 September 2013 to 31 August 2021, which allowed for a maximum follow-up period of 8 years. In our primary specification, we selected a follow-up period of 7 years (that is, until 31 August 2020) because this enabled us to include grace periods of up to 12 months while still keeping the follow-up period constant for individuals on either side of the date-of-birth eligibility cut-off. However, we also show all results for follow-up periods from one to eight years in one-year increments. Owing to the unique anonymized NHS number assigned to each patient, we were able to follow individuals across time even if they changed primary care provider. Patients were therefore only lost to follow-up in our cohort if they emigrated out of Wales or changed to one of the approximately 20% of primary care practices in Wales that did not contribute data to SAIL. Over our seven-year follow-up period, this was the case for 23,049 (8.2%) of adults in our primary analysis cohort, with no significant difference in this proportion between those born just before versus just after the 2 September 1933 eligibility threshold. In total, 92,629 (37.8%) of adults in our primary analysis cohort died during the seven-year follow-up period. Our primary analysis approach does not adjust for any competing risk of death for three reasons. First and foremost, in the absence of a zoster vaccination program, there is no reason that the competing risk of death should differ across the 2 September 1933 date-of-birth eligibility threshold. Second, not adjusting for competing risk of death in our setting is a conservative choice because eligibility for zoster vaccination may reduce (but is very unlikely to increase) all-cause mortality10,75. Thus, those eligible for zoster vaccination will, on average, be exposed to a longer time period during which they could become newly diagnosed with dementia. Third, to date, no well-established approach exists for survival analysis in a regression discontinuity framework, including the ability to determine the CACE and optimal bandwidth76.

Owing to the neuropathological overlap between dementia types and difficulty in distinguishing dementia types clinically77,78,79, we chose to define dementia as dementia of any type or cause. Dementia was defined as a diagnosis of dementia made either in primary care (as recorded in the primary care electronic health record data), specialist care or hospital-based care, or dementia being named as a primary or contributory cause of death on the death certificate. The date of the first recording of dementia across any of these data sources was used to define the date on which the patient was diagnosed with dementia. Similarly, all other outcomes were defined using a diagnosis made in any care setting or mentioning as a primary or contributory cause of death. For chronic conditions, the date of the first recording across any of these data sources was used to define the date on which the chronic condition occurred. For non-chronic conditions or events (that is, shingles, postherpetic neuralgia, stroke, lower respiratory tract infections, falls, lower back pain, medication prescriptions, influenza vaccination and breast cancer screening), the date of first recording after the program date across any of these data sources was used for defining the occurrence of the outcome during the follow-up. The Read and ICD-10 codes used to define all outcomes are detailed in the Supplementary Codes.

The two authors who analysed the data (M.E. and M.X.) have coded all parts of the analysis independently. Occasional minor differences, resulting from different data coding choices, were resolved through discussion.

We used a regression discontinuity design to analyse our data, which is a well-established method for causal inference in the social sciences80. Regression discontinuity analysis estimates expected outcome probabilities just left and just right of the cut-off, to obtain an estimate of the treatment effect. We used local linear triangular kernel regressions (assigning a higher weight to observations lying closer to the date-of-birth eligibility threshold) in our primary analyses and quadratic polynomials in robustness checks. This is the recommended and most robust approach for regression discontinuity analyses even in situations in which the relationship between the assignment variable (here, date of birth) and the outcome is exponential81. An important choice in regression discontinuity analyses is the width of the data window (the bandwidth) that is drawn around the threshold. Following standard practice, we used an MSE-optimal bandwidth82, which minimizes the MSEs of the regression fit, in our primary analyses. We determined this optimal bandwidth separately for each combination of sample and outcome definition. In robustness checks, we examined the degree to which our point estimates vary across different bandwidth choices ranging from 0.25 times to two times the MSE-optimal bandwidth. We used robust bias-corrected standard errors for inference83.

In the first step, we determined the effect of being eligible for the zoster vaccine (regardless of whether the individual actually received the vaccine) on our outcomes. To do so, we estimated the following regression equation:

where Yi is a binary variable equal to one if an individual experienced the outcome (for example, shingles or dementia). The binary variable Di indicates eligibility for the zoster vaccine and is equal to one if an individual was born on or after the cut-off date of 2 September 1933. The term (WOBi − C0) indicates an individual's week of birth centred around the cut-off date. The interaction term Di × (WOBi − C0) allows for the slope of the regression line to differ on either side of the threshold. The parameter β1 identifies the absolute effect of being eligible for the vaccine on the outcome. Wherever we report relative effects, we calculated these by dividing the absolute effect estimate β1 by the mean outcome just left of the date-of-birth eligibility threshold, that is, the estimate of α.

In the second step, we estimated the effect of actually receiving the zoster vaccine on our outcomes. This effect is commonly referred to as the complier average causal effect (CACE) in the econometrics literature84. As is standard practice84, we used a fuzzy regression discontinuity design to estimate the CACE. Fuzzy regression discontinuity analysis takes into account the fact that the vaccine is not deterministically assigned at the week-of-birth cut-off. Instead, a proportion of ineligible individuals still received the vaccine and a proportion of eligible individuals did not receive the vaccine. To account for this fuzziness in the assignment, the fuzzy regression discontinuity design uses an instrumental variable approach, with the instrumental variable being the binary variable that indicates whether or not an individual was eligible to receive the vaccine, that is, is born on or after 2 September 1933. As we verify in our plot of vaccine receipt by week of birth (Fig. 1a), individuals who were born immediately after the date-of-birth eligibility threshold had a far higher probability of receiving the zoster vaccine compared with those born immediately before the threshold. Other than the abrupt change in the probability of receiving the zoster vaccine, there probably is no other difference in characteristics that affect the probability of our outcomes occurring between those born immediately after versus immediately before the date-of-birth eligibility threshold. Thus, the indicator variable for the date-of-birth eligibility threshold is a valid instrumental variable to identify the causal effect of receipt of the zoster vaccine on our outcomes. To compare the probability of experiencing the outcome between those who actually received the zoster vaccine versus those who did not, the instrumental variable estimation scales the effect size for being eligible for the zoster vaccine by the size of the abrupt change in the probability of receiving the vaccine at the date-of-birth eligibility threshold. The size of the jump is estimated through the following first-stage regression equation:

where Vi is a binary variable indicating whether the individual received the zoster vaccine and θ1 identifies the discontinuous increase in vaccine receipt at the date-of-birth eligibility threshold. All other parameters are the same as in regression (1).

The CACE estimated by rescaling the effect of eligibility with the first-stage effect from equation (2) can be represented as an IV estimate for μ1 from the following second-stage regression:

where \({\hat{V}}_{i}\) is the predicted probability of zoster vaccine receipt obtained from the first-stage estimation from equation (2). This CACE, μ1, represents the (absolute) average causal effect of receiving the vaccine among compliers, that is, patients who take up the vaccine if and only if they are eligible.

To compute relative effect sizes, we first introduce some notation. Let R0,c be the mean outcome among unvaccinated compliers and R1,c the mean outcome among vaccinated compliers just at the threshold. By definition, the absolute CACE is μ1 = R1,c − R0,c and the relative effect is \(\frac{{\mu }_{1}}{{R}_{0,c}}\). To estimate the relative effect, we need an estimate for R0,c. While it is impossible to identify compliers individually, we can estimate means of their observable characteristics, including R0,c (ref. 85). Let R1 denote the mean outcome among all vaccinated individuals (including compliers) at the cut-off. Assuming no defiers exist (patients who get vaccinated if and only if they are not eligible), all vaccinated people are either compliers or always-takers (patients who get vaccinated irrespective of their eligibility). Thus, R1 is equal to the population-weighted average of the mean outcomes among vaccinated compliers and always-takers: R1 = Pc × R1,c + Pa × R1,a, where Pc and Pa are the population share of the compliers and always-takers and R1,a is the mean outcome among always-takers at the cut-off. Solving for R1,c yields \({R}_{1,c}=\frac{{R}_{1}-{R}_{1,a}\times {P}_{a}}{{P}_{c}\,}\). All right-hand-side quantities in this equation can be estimated from data. First, R1 and R1,a can be estimated, respectively, as α + β1 and α from re-estimating regression (1) only among vaccinated individuals. Second, Pa and Pc can be estimated, respectively, as \(\frac{\gamma }{{\theta }_{1}+\gamma }\) and \(\frac{{\theta }_{1}}{{\theta }_{1}+\gamma }\) from regression (2). Finally, we estimate R1,c by the above formula and R0,c = R1,c − μ1. The relative effect is estimated as \(\frac{{\mu }_{1}}{{R}_{0,c}}\). All regressions involved in these steps can be stacked and jointly estimated, so that the relative effect is expressed as a differentiable function of known estimators a 95% confidence interval of the relative CACE can be estimated using the delta method86.

Our analysis can only be confounded if the confounding variable changes abruptly at the 2 September 1933 date-of-birth eligibility threshold such that individuals very close to either side of this threshold would no longer be exchangeable with each other. The most plausible scenario of such a confounding variable would be the existence of an intervention that used the exact same date-of-birth eligibility threshold as the zoster vaccine rollout and that also affected the probability of a dementia diagnosis during our follow-up period. We conducted five analyses to demonstrate that the existence of such an intervention is unlikely, by establishing that measures of outcomes and behaviours that would be affected by such an intervention are smooth across the date-of-birth eligibility cut-off.

First, across a range of birthdates around the 2 September 1933 eligibility threshold, we plotted the probability of having received the following diagnoses or interventions before the start of the zoster vaccine program (on 1 September 2013): diagnosis of shingles, influenza vaccine receipt in the preceding 12 months, receipt of the pneumococcal vaccine as an adult, current statin use (defined as a new or repeat prescription of a statin in the 3 months preceding program start), current use of an antihypertensive medication (defined as a new or repeat prescription of an antihypertensive drug in the 3 months preceding the program start), participation in breast cancer screening (defined as the proportion of women with a record of referral to, attendance at or a report from breast cancer screening or mammography), each of the ten leading causes of disability-adjusted life years and mortality for Wales in 2019 as estimated by the Global Burden of Disease Project33, and all comorbidities (except for AIDS, which falls under privacy-protected diagnoses not made available by the SAIL database) that are included in the widely-used Charlson Comorbidity Index34. Moreover, we used each of these conditions, gender, decile of Welsh Index of Multiple Deprivation56, as well as all input variables to the Dementia Risk Score (as recorded before 1 September 2013)32, to predict the probability (while imputing a fixed age) of a new dementia diagnosis for each patient in the MSE-optimal bandwidth in our primary regression discontinuity analysis for dementia. In addition to plotting these predicted probabilities across a range of birthdates around the 2 September 1933 eligibility threshold, we also plotted the distribution of these predicted probabilities for patients who were eligible versus patients who were ineligible for zoster vaccination. The Read codes for each of these variables are provided in Supplementary Tables 1 and 2. As is the case for balance tables in clinical trials, these plots provide reassurance that individuals close to either side of the 2 September 1933 eligibility threshold are likely to be exchangeable with each other.

Second, we conducted the same analysis as we did for individuals with birthdays on either side of the 2 September 1933 threshold also for people with birthdays around 2 September of each of the three years of birth preceding and succeeding 1933. For example, when moving the start date of the program to 1 September 2011, we started the follow-up period on 1 September 2011 and compared individuals around the 2 September 1931 eligibility threshold. To ensure the same length of follow-up in each of these comparisons, we had to reduce the follow-up period to 5 years for this set of analyses. Thus, as an additional check, we shifted the start date of the program to 1 September of each of the six years preceding (but not succeeding) 2013, which enabled us to maintain the same seven-year follow-up period as in our primary analysis. If another intervention that affects dementia risk also used the 2 September threshold to define eligibility, we may then expect to observe effects on dementia incidence for these comparisons of individuals just around the 2 September thresholds of other birth years.

Third, we conducted the identical comparison of individuals around the 2 September 1933 date-of-birth threshold as in our primary analysis, except for starting the follow-up period 7 years before the start of the zoster vaccine program rollout. If there was an intervention that used the 2 September 1933 date-of-birth eligibility threshold but was implemented before the rollout of the zoster vaccine program, then we may expect to see an effect of the September 1933 threshold on dementia incidence in this analysis.

Fourth, we verified that the effects that we observed in our analyses for dementia incidence appear to be specific to dementia. If an intervention that used the exact same date-of-birth eligibility threshold as the zoster vaccine program indeed existed, it would be unlikely to only affect dementia risk without also having an influence on other health outcomes. We therefore conducted the same analysis as for when using dementia incidence as the outcome but for each of the ten leading causes of disability-adjusted life years and mortality in Wales in 2019 for the age group 70+ years33, as well as all conditions that are part of the Charlson Comorbidity Index34.

Fifth, we tested for discontinuities in educational attainment at the 2 September 1933 date-of-birth threshold using data from the 2011 census in Wales74. If an educational policy had used a 2 September (or specifically 2 September 1933) date-of-birth threshold and the policy was effective in increasing educational attainment, we would then expect discontinuities at the 2 September 1933 threshold in the attained education level between eligible and ineligible individuals. We used the identical analysis approach for this balance test as for our primary analyses in the SAIL database, except that we computed ‘honest' confidence intervals based on the approach by Armstrong and Kolesár because the assignment variable (month of birth) in these data was monthly, and therefore coarser than the assignment variable (week of birth) in our analyses in the SAIL database87,88. This approach guards against potential vulnerability to model misspecification and resulting under-coverage of confidence intervals computed with more standard methods. These honest confidence intervals are conservative in the sense that they have good coverage properties irrespective of whether the functional form in the regression discontinuity analysis is misspecified, provided that the true functional form falls within a certain class of functions. For this class, we considered a function class defined by bounds on the second derivative of the conditional expectation function mapping date of birth to the probability attaining a certain educational level. We used conservative bounds of the respective curvatures by relying on global estimation of higher-order polynomials as proposed by Armstrong and Kolesár88.

We additionally used a difference-in-differences instrumental variable approach (DID-IV) to confirm the findings from our regression discontinuity design because, in contrast to the regression discontinuity analysis, this approach does not rely on the continuity assumption (that is, the assumption that potential confounding variables do not abruptly change at precisely the date-of-birth eligibility threshold for the zoster vaccine program). To do so, we restricted our sample to patients born between 1 March 1926 and 28 February 1934. This sample consists of 96,767 adults, of whom 7,752 (8.0%) were eligible for, and 3,949 (4.1%) actually received, zoster vaccination. We then divided our sample into yearly cohorts centred around 1 September (that is, a cohort is all patients born between 1 March of one year and 28 February of the following year). Finally, we divided each yearly cohort into a pre-September birth season and a post-September birth season. Using a difference-in-differences approach, we then compared the outcome (new diagnoses of dementia) between patients born in pre- and post-September birth seasons and across yearly cohorts. More precisely, we tested whether the difference in outcomes across birth seasons is different for the 1933/1934 cohort than for the other cohorts. In doing so, we exploit the fact that zoster vaccination eligibility only differs between the two birth seasons in the 1933/1934-cohort but not in other cohorts, while accounting for the possibility that pre-September and post-September birth seasons may be systematically different for other reasons.

Our difference-in-differences setup implies that the interaction between the post-September birth season indicator and the 1933/1934-cohort indicator constitutes an instrumental variable for receipt of the zoster vaccine, enabling us to estimate the CACE (that is, the effect of actually receiving the vaccine among the compliers). This DID-IV approach relies on two important assumptions. As per the standard exclusion restriction assumption of IV analyses, the IV component of our DID-IV approach assumes that vaccine eligibility affects the outcome solely through a change in actual vaccine receipt. The DID component of our DID-IV approach assumes that, in the absence of the vaccine eligibility rule, the between-birth-season difference in vaccine uptake and in dementia incidence would have been the same in the 1933/1934 cohort as in the other cohorts. To investigate the validity of this assumption, we plotted the mean vaccine uptake and dementia incidence with 95% CIs by birth cohorts and birth seasons (Supplementary Fig. 29). As expected, we find that the between-birth-season differences in vaccine uptake diverge only in the 1933/1934 birth cohort. The absence of a between-birth-season difference in other birth cohorts supports the validity of our DID assumption.

To estimate the CACE in this DID-IV framework, we used two-stage least-squares regression. In the first stage, we identify the vaccine uptake due to the exogeneous change in vaccination eligibility by the following regression equation:

where Vi is a binary variable indicating patient i actually received the zoster vaccine. Si and Ci are binary variables indicating that patient i is born in the post-September birth season and in the 1933/1934 birth cohort, respectively. γ identifies the vaccine uptake due to the change in eligibility. θ, ηm and ηc are the constant term, birth month (January, February, …, December) and birth cohort (1926/1927, 1927/1928, …, 1933/1934) fixed effect, respectively. ϵi is the error term.

In the second stage, we estimate the effect of vaccine receipt by the following regression:

where Yi is the outcome of patient i. \({\widehat{V}}_{i}\) is the probability of vaccine receipt predicted from the first-stage regression (4). The coefficient β identifies the CACE. α, ηm and ηc are the constant term, birth month and birth cohort (1926/1927, 1927/1928, …, 1933/1934) fixed effect, respectively. ϵi is the error term.

We conducted a series of additional robustness checks. First, instead of starting the follow-up period for all individuals on 1 September 2013, we adjusted the follow-up period to account for the staggered rollout of the program by beginning the follow-up period for each individual on the date on which they first became eligible for the zoster vaccine (as described in the ‘Description of the zoster vaccine rollout in Wales' section) (Supplementary Fig. 30). We controlled for cohort fixed effects in these analyses to account for the one- to two-year (depending on the year of the program) differences between cohorts in the calendar year in which this moving follow-up window started. That is, we defined one cohort fixed effect for ineligible individuals and the first catch-up cohort and then included additional cohort fixed effects for each group of patients who became eligible at the same time. Second, we varied our definition of a new diagnosis of dementia by implementing our analysis when defining dementia as a new prescription of donepezil hydrochloride, galantamine, rivastigmine or memantine hydrochloride. Third, we tested whether our results for the effect of being eligible for zoster vaccination on new diagnoses of dementia, shingles and postherpetic neuralgia hold across grace periods (that is, time periods since the index date after which follow-up time is considered to begin to allow for the time needed for a full immune response to develop after vaccine administration) of 0, 2, 4, 6, 8, 10 and 12 months (Supplementary Fig. 31). Fourth, we show our results with bandwidth choices of 0.25, 0.50, 0.75, 1.00, 1.25, 1.50, 1.75 and 2.00 times the MSE-optimal bandwidth (Supplementary Fig. 32). Fifth, we verified that our results are similar when using a local second-order polynomial specification instead of local linear regression.

We conducted four analyses to examine this potential effect mechanism, the first three of which are described in detail in the main text. The fourth analysis was an event study that focused on the date of a shingles diagnosis during the follow-up period. Our event study compared the mean outcome in each month relative to the month before the date of the shingles diagnosis. Our regression model controls for changes over time (such as due to ageing of the study population or seasonal patterns in healthcare provider visits) using month-level fixed effects.

To implement our event study, we restricted our study population to those 56,098 individuals born within the MSE-optimal bandwidth of our primary regression discontinuity analysis for dementia. We then aggregated our event-level data into monthly longitudinal data, spanning September 2013 to March 2022. For each outcome of interest (as described in the main text), we then estimated the following event-study regression:

where Yit is the outcome of interest for individual i in period t; shingles is a binary variable equal to one if the individual was diagnosed with shingles during the follow-up period; Dk are indicator variables for the k months before and after the shingles diagnosis (with k = −36, −35,…, 35, 36, and set to zero for individuals who were never diagnosed with shingles during the follow-up period); γk are the coefficients of interest, which capture the difference in the outcome in month k relative to the month before the shingles diagnosis; ηi is an individual-level fixed effect capturing time-invariant differences across individuals; and λt is a month-level fixed effect, capturing differences across periods. We used standard errors that allowed for clustering at the individual level, and therefore for autocorrelation.

We conducted four analyses to examine this potential effect mechanism. First, we implemented the identical regression discontinuity as in our primary analysis, except that we included a binary variable for being diagnosed with shingles during the follow-up period as a covariate. For the resulting estimate to be an unbiased measure of the degree to which the effect of zoster vaccination on dementia incidence is mediated by shingles diagnoses, there must be no variables that are related to both new dementia diagnoses and the probability of being diagnosed with shingles (that is, no confounding of the mediator-to-outcome relationship)89.

Second, to examine when during the follow-up period the dementia-delaying or dementia-preventing effect of zoster vaccination begins to emerge, we plotted Kaplan–Meier plots and (to account for competing risks) cumulative incidence curves among individuals born close to 2 September 1933. This analysis was based on the concept of local randomization28,29, which relies on exchangeability of individuals born immediately before versus immediately after 2 September 1933. To define the bandwidth for our analysis in which we could reasonably assume exchangeability across the threshold while maximizing statistical power, we used the widest bandwidth for which we achieved balance in baseline demographic and clinical characteristics of individuals eligible versus ineligible for zoster vaccination. We evaluated bandwidths ranging from 100% to 10% of the MSE-optimal bandwidth (90.6 weeks) in our primary regression discontinuity analysis in 10% decrements. The variables we used for our balance tests were the 14 variables listed in Supplementary Figs. 1 and 2 (except for the more sex-specific variables of past breast cancer screening, breast cancer and prostate cancer diagnoses) using a significance threshold of P < 0.05, while controlling for the false-discovery rate using the Benjamini–Hochberg procedure90. The largest bandwidth that achieved balance across all variables was 54.4 weeks.

Third, to investigate whether antiviral treatment during a shingles episode was associated with a reduction in the risk of dementia relative to not receiving treatment during a shingles episode, we restricted our study population to those individuals who received a diagnosis of shingles at any time after 1 January 2000 and had not received a diagnosis of dementia before 1 January 2000. Our exposure of interest in this analysis was whether or not an individual received a prescription of antiviral medication (acyclovir, famcyclovir, valacyclovir or inosine pranobex) within three months of the first shingles diagnosis. Individuals were followed up from the date of first shingles diagnosis until either the date of death, moving out of Wales, GP deregistration or end of data availability (1 March 2022). We then used a multivariable Cox proportional hazards model to regress diagnoses of dementia made after the date of the first recorded shingles episode onto whether or not the patient received an antiviral medication prescription for the first shingles episode. In a robustness check, we required that a new diagnosis of dementia must have been made at least 12 months after the date of the first shingles diagnosis. We adjusted our regressions for gender, restricted cubic splines (with three knots) of age at the first shingles infection, and the 12 variables in Supplementary Fig. 1 (excluding past breast and prostate cancer diagnoses).

Fourth, to explore whether experiencing recurrent shingles episodes was associated with a higher risk of dementia than having only a single episode, we used the same study population as in our analysis for treated versus untreated shingles. We matched individuals (via 1:1 propensity score matching) who had more than one shingles diagnosis (with the diagnosis dates having to be at least three months apart) after 1 January 2000 to individuals who only received a single shingles diagnosis after 1 January 2000. We matched individuals on proximity in the date of their first shingles diagnosis as well as the same list of baseline variables as for our analysis of treated versus untreated shingles, and forced an exact match on week of birth and gender. In each matched pair, we used the date of the second shingles diagnosis of the individual with more than one shingles diagnosis as the start date of the follow-up period. Using a Cox proportional hazards model, we then regressed new diagnoses of dementia made during the follow-up period onto whether or not the individual had received more than one shingles diagnosis. In a robustness check, we again required that a new diagnosis of dementia must have been made at least 12 months after the start date of the follow-up period.

To estimate the treatment effect heterogeneities described under this section in the main text, we fully interacted our fuzzy regression discontinuity model with a binary variable that indicates having the condition in question (for example, an autoimmune condition). Precisely, the fully interacted model was specified as:

where the subscript i indexes individuals. Yi is a binary variable equal to 1 if an individual was newly diagnosed with dementia during the follow-up period. The binary variable Vi indicates receipt of the zoster vaccine. The binary variable Di indicates eligibility for the zoster vaccine (that is, born on or after 2 September 1933). The term WOBi − c0 indicates an individual's week of birth centred around the date-of-birth eligibility threshold. The interaction term Di × (WOBi − c0) allows for the slope of the regression line to differ on either side of the date-of-birth eligibility threshold. The binary variable HETi is equal to one if an individual had the condition in question. Adding the terms (WOBi − c0) × HETi and Di × (WOBi − c0) × HETi allows the slopes to vary by this condition.

Vi and Vi × HETi are instrumented by Di and Di × HETi. Using the two-stage least-squares approach, the parameter β4 identifies the effect heterogeneity, that is, the difference in CACE on the outcome between patients with and without the condition. β1 and β1 + β4 identify the effect among compliers in the reference and comparison group, respectively. The estimates of the effects and heterogeneity are reported in absolute terms. To be consistent with our primary fuzzy regression discontinuity model (that is, without the HETi and interaction terms), we used local linear triangular kernel regressions and the MSE-optimal bandwidth from the primary model of the respective outcome.

For our analyses for autoimmune and allergic conditions, we used the 19 most common autoimmune conditions as defined previously91, and grouped the 11 least common conditions among them into a rare conditions category. We judged those conditions to be rare that had an incidence of less than 1% during the follow-up period in our cohort. These rare conditions included Addison's disease, ankylosing spondylitis, Coeliac disease, Hashimoto's thyroiditis, multiple sclerosis, myasthenia gravis, primary biliary cirrhosis, Sjögren's syndrome, systemic lupus erythematosus, systemic sclerosis and vitiligo. For common allergic conditions, we used those defined previously92.

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

The data supporting the findings of this study are available from the SAIL Databank26. Researchers must request access to the data directly from SAIL. The authors have no permission to share the data. This paragraph describes how access to the data in the SAIL Databank can be obtained. All proposals to use SAIL data are subject to review by an independent Information Governance Review Panel (IGRP). Before any data can be accessed, approval must be given by the IGRP. The IGRP carefully considers each project to ensure the proper and appropriate use of SAIL data. When access has been granted, it is gained through a privacy-protecting trusted research environment (TRE) and remote access system referred to as the SAIL Gateway. SAIL has established an application process, which includes the payment of a fee, to be followed by anyone who would like to access data through SAIL at https://saildatabank.com/ data/apply-to-work-with-the-data/. Once approved, researchers will have to sign a data access agreement and request a gateway account. After the account is approved, researchers will be able to log into the secure SAIL Gateway remotely. Once logged in, researchers can import our SQL/R code and run the analyses by downloading our replication package (https://osf.io/cfnr6/?view_only=d3774e4fda2649e2b2031431b1234874), uploading the package (SQL and R scripts) to the SAIL Gateway through the secure file upload process, and executing the scripts in the Gateway environment.

All Read and ICD-10 codes to define variables are available in the Supplementary Codes. All statistical packages including version numbers for version control, algorithms to define variables and R analysis code are provided in an OSF repository (https://osf.io/cfnr6/?view_only=d3774e4fda2649e2b2031431b1234874)93.

Wainberg, M. et al. The viral hypothesis: how herpesviruses may contribute to Alzheimer's disease. Mol. Psychiatry 26, 5476–5480 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Devanand, D. P. Viral hypothesis and antiviral treatment in Alzheimer's disease. Curr. Neurol. Neurosci. Rep. 18, 55 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Moir, R. D., Lathe, R. & Tanzi, R. E. The antimicrobial protection hypothesis of Alzheimer's disease. Alzheimers Dement. 14, 1602–1614 (2018).

PubMed 
    MATH 
    
                    Google Scholar 
                

Itzhaki, R. F. et al. Microbes and Alzheimer's disease. J. Alzheimers Dis. 51, 979–984 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Hyde, V. R. et al. Anti-herpetic tau preserves neurons via the cGAS-STING-TBK1 pathway in Alzheimer's disease. Cell Rep. 44, 115109 (2025).

Benn, C. S., Fisker, A. B., Rieckmann, A., Sørup, S. & Aaby, P. Vaccinology: time to change the paradigm? Lancet Infect. Dis. 20, e274–e283 (2020).

PubMed 
    
                    Google Scholar 
                

Benn, C. S. et al. Implications of non-specific effects for testing, approving, and regulating vaccines. Drug Saf. 46, 439–448 (2023).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Aaby, P. et al. The non-specific and sex-differential effects of vaccines. Nat. Rev. Immunol. 20, 464–470 (2020).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

McGovern, M. E. & Canning, D. Vaccination and all-cause child mortality from 1985 to 2011: global evidence from the Demographic and Health Surveys. Am. J. Epidemiol. 182, 791–798 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Michalik, F. et al. The effect of herpes zoster vaccination on the occurrence of deaths due to dementia in England and Wales. Preprint at medRxiv https://doi.org/10.1101/2023.09.08.23295225 (2023).

Wu, X. et al. Adult vaccination as a protective factor for dementia: a meta-analysis and systematic review of population-based observational studies. Front. Immunol. 13, 872542 (2022).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Hernan, M. A. & Robins, J. M. Causal Inference: What If (Chapman & Hall/CRC, 2020).

Simonsen, L., Taylor, R. J., Viboud, C., Miller, M. A. & Jackson, L. A. Mortality benefits of influenza vaccination in elderly people: an ongoing controversy. Lancet Infect. Dis. 7, 658–666 (2007).

PubMed 
    
                    Google Scholar 
                

Shingles: Guidance and Vaccination Programme www.gov.uk/government/collections/shingles-vaccination-programme (UK Health Security Agency, 2021).

Harris, K. et al. The impact of routine vaccinations on Alzheimer's disease risk in persons 65 years and older: a claims-based cohort study using propensity score matching. J. Alzheimers Dis. 95, 703–718 (2023).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Lophatananon, A. et al. Shingles, Zostavax vaccination and risk of developing dementia: a nested case-control study-results from the UK Biobank cohort. BMJ Open 11, e045871 (2021).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Scherrer, J. F. et al. Impact of herpes zoster vaccination on incident dementia: A retrospective study in two patient cohorts. PLoS ONE 16, e0257405 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Schnier, C., Janbek, J., Lathe, R. & Haas, J. Reduced dementia incidence after varicella zoster vaccination in Wales 2013–2020. Alzheimers Dement. 8, e12293 (2022).


                    Google Scholar 
                

Wiemken, T. L. et al. Comparison of rates of dementia among older adult recipients of two, one, or no vaccinations. J. Am. Geriatr. Soc. 70, 1157–1168 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Lophatananon, A. et al. The association of herpes zoster and influenza vaccinations with the risk of developing dementia: a population-based cohort study within the UK Clinical Practice Research Datalink. BMC Publ. Health 23, 1903 (2023).

MATH 
    
                    Google Scholar 
                

Ukraintseva, S. et al. Associations of infections and vaccines with Alzheimer's disease point to a role of compromised immunity rather than specific pathogen in AD. Exp. Gerontol. 190, 112411 (2024).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Lehrer, S. & Rheinstein, P. H. Herpes zoster vaccination reduces risk of dementia. In Vivo 35, 3271–3275 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Wilkinson, T. et al. Drug prescriptions and dementia incidence: a medication-wide association study of 17000 dementia cases among half a million participants. J. Epidemiol. Community Health 76, 223–229 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Tang, E., Ray, I., Arnold, B. F. & Acharya, N. R. Recombinant zoster vaccine and the risk of dementia. Vaccine 46, 126673 (2024).

PubMed 
    
                    Google Scholar 
                

Introduction of Shingrix Vaccine for the Whole Programme and Expansion of Eligible Cohorts Letter https://www.gov.uk/government/publications/shingles-vaccination-programme-changes-from-september-2023-letter/introduction-of-shingrix-vaccine-for-the-whole-programme-and-expansion-of-eligible-cohorts-letter (UK Health Security Agency, 2023).

SAIL Databank saildatabank.com/ (Swansea University, 2025).

Attribution Data Set GP-Registered Populations Scaled to ONS Population Estimates—2011 digital.nhs.uk/data-and-information/publications/statistical/attribution-dataset-gp-registered-populations/attribution-data-set-gp-registered-populations-scaled-to-ons-population-estimates-2011 (NHS Digital, 2012).

Cattaneo, M. D., Keele, L. & Titiunik, R. A guide to regression discontinuity designs in medical applications. Stat. Med. 42, 4484–4513 (2023).

MathSciNet 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Cattaneo, M. D., Idrobo, N. & Titiunik, R. A Practical Introduction to Regression Discontinuity Designs: Foundations (Cambridge Univ. Press, 2020).

Harbecke, R., Cohen, J. I. & Oxman, M. N. Herpes zoster vaccines. J. Infect. Dis. 224, S429–S442 (2021).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

2024 Alzheimer's disease facts and figures. Alzheimers Dement. 20, 3708–3821 (2024).

Walters, K. et al. Predicting dementia risk in primary care: development and validation of the Dementia Risk Score using routinely collected data. BMC Med. 14, 6 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Global Burden of Disease Project 2019. GBD Compare vizhub.healthdata.org/gbd-compare (Institute for Health Metrics and Evaluation, 2020).

Charlson, M. E., Pompei, P., Ales, K. L. & MacKenzie, C. R. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. J. Chronic Dis. 40, 373–383 (1987).

PubMed 
    
                    Google Scholar 
                

Oxman, M. N. Zoster vaccine: current status and future prospects. Clin. Infect. Dis. 51, 197–213 (2010).

PubMed 
    MATH 
    
                    Google Scholar 
                

Oxman, M. N. et al. A vaccine to prevent herpes zoster and postherpetic neuralgia in older adults. N. Engl. J. Med. 352, 2271–2284 (2005).

PubMed 
    MATH 
    
                    Google Scholar 
                

Campling, J. et al. A review of evidence for pneumococcal vaccination in adults at increased risk of pneumococcal disease: risk group definitions and optimization of vaccination coverage in the United Kingdom. Expert Rev. Vaccines 22, 785–800 (2023).

PubMed 
    MATH 
    
                    Google Scholar 
                

Yun, H. et al. Risk of herpes zoster in autoimmune and inflammatory diseases: implications for vaccination. Arthritis Rheumatol. 68, 2328–2337 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Marra, F., Parhar, K., Huang, B. & Vadlamudi, N. Risk factors for herpes zoster infection: a meta-analysis. Open Forum Infect. Dis. 7, ofaa005 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Joesoef, R. M., Harpaz, R., Leung, J. & Bialek, S. R. Chronic medical conditions as risk factors for herpes zoster. Mayo Clin. Proc. 87, 961–967 (2012).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Kawai, K. & Yawn, B. P. Risk factors for herpes zoster: a systematic review and meta-analysis. Mayo Clin. Proc. 92, 1806–1821 (2017).

PubMed 
    MATH 
    
                    Google Scholar 
                

McGonagle, D. & McDermott, M. F. A proposed classification of the immunological diseases. PLoS Med. 3, e297 (2006).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Gandhi, N. A. et al. Targeting key proximal drivers of type 2 inflammation in disease. Nat. Rev. Drug Discov. 15, 35–50 (2016).

PubMed 
    MATH 
    
                    Google Scholar 
                

Shingles Vaccine Coverage (England): Report for Quarter 1 of the Financial Year 2022 to 2023 www.gov.uk/government/publications/ herpes-zoster-shingles- immunisation-programme-2022-to-2023- evaluation-reports/shingles-vaccine- coverage-england-report-for-quarter-1- of-the-financial-year-2022-to-2023 (UK Health Security Agency, 2024).

Population and Household estimates, England and Wales: Census 2021, Unrounded Data www.ons.gov.uk/peoplepopulationandcommunity/ populationandmigration/populationestimates/ bulletins/ populationandhouseholdestimatesenglandandwales/ census2021unroundeddata (ONS, 2022).

Fischinger, S., Boudreau, C. M., Butler, A. L., Streeck, H. & Alter, G. Sex differences in vaccine-induced humoral immunity. Semin. Immunopathol. 41, 239–249 (2019).

PubMed 
    
                    Google Scholar 
                

Ferretti, M. T. et al. Sex and gender differences in Alzheimer's disease: current challenges and implications for clinical practice: position paper of the Dementia and Cognitive Disorders Panel of the European Academy of Neurology. Eur. J. Neurol. 27, 928–943 (2020).

PubMed 
    MATH 
    
                    Google Scholar 
                

Silver, B. et al. Varicella zoster virus vasculopathy: a treatable form of rapidly progressive multi-infarct dementia after 2 years' duration. J. Neurol. Sci. 323, 245–247 (2012).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Gilden, D. et al. Successful antiviral treatment after 6years of chronic progressive neurological disease attributed to VZV brain infection. J. Neurol. Sci. 368, 240–242 (2016).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Bubak, A. N. et al. Targeted RNA sequencing of VZV-infected brain vascular adventitial fibroblasts indicates that amyloid may be involved in VZV vasculopathy. Neurol. Neuroimmunol. Neuroinflamm. 9, e1103 (2022).

PubMed 
    
                    Google Scholar 
                

Nagel, M. A. & Bubak, A. N. Varicella zoster virus vasculopathy. J. Infect. Dis. 218, S107–S112 (2018).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Nagel, M. A. et al. The varicella zoster virus vasculopathies: clinical, CSF, imaging, and virologic features. Neurology 70, 853–860 (2008).

PubMed 
    MATH 
    
                    Google Scholar 
                

Nagel, M. A. et al. Varicella zoster virus vasculopathy: analysis of virus-infected arteries. Neurology 77, 364–370 (2011).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Nagel, M. A. et al. Varicella-zoster virus vasculopathy: immune characteristics of virus-infected arteries. Neurology 80, 62–68 (2013).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Attems, J. & Jellinger, K. A. The overlap between vascular disease and Alzheimer's disease-lessons from pathology. BMC Med. 12, 206 (2014).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Boyle, P. A. et al. Cerebral amyloid angiopathy and cognitive outcomes in community-based older persons. Neurology 85, 1930–1936 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Cairns, D. M., Itzhaki, R. F. & Kaplan, D. L. Potential involvement of varicella zoster virus in Alzheimer's disease via reactivation of quiescent herpes simplex virus type 1. J. Alzheimers Dis. 88, 1189–1200 (2022).

PubMed 
    
                    Google Scholar 
                

Bukhbinder, A. S., Ling, Y., Harris, K., Jiang, X. & Schulz, P. E. Do vaccinations influence the development of Alzheimer disease? Hum. Vaccin. Immunother. 19, 2216625 (2023).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Anderson, M. et al. United Kingdom: health system review. Health Syst. Transit. 24, 1–194 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Ford, D. V. et al. The SAIL Databank: building a national architecture for e-health research and evaluation. BMC Health Serv. Res. 9, 157 (2009).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Jones, K. H. et al. A case study of the Secure Anonymous Information Linkage (SAIL) Gateway: a privacy-protecting remote access system for health-related research and evaluation. J. Biomed. Inform. 50, 196–204 (2014).

Lyons, R. A. et al. The SAIL databank: linking multiple health and social care datasets. BMC Med. Inform. Decis. Mak. 9, 3 (2009).

Rodgers, S. E., Demmler J., Dsilva R. & Lyons R. Protecting health data privacy while using residence-based environment and demographic data. Health Place 18, 209–217 (2012).

Rodgers, S. E. et al. Residential Anonymous Linking Fields (RALFs): a novel information infrastructure to study the interaction between the environment and individuals' health. J. Public Health 31, 582–588 (2009).

Welsh Demographic Service Dataset (WDSD) web.www.healthdatagateway.org/dataset/cea328df-abe5-48fb-8bcb-c0a5b6377446# (Digital Health and Care Wales, 2022).

Welsh Longitudinal General Practice Dataset (WLGP)—Welsh Primary Care web.www.healthdatagateway.org/dataset/33fc3ffd-aa4c-4a16-a32f-0c900aaea3d2 (SAIL Databank, 2023).

Benson, T. The history of the Read Codes: the inaugural James Read Memorial Lecture 2011. Inform. Prim. Care 19, 173–182 (2011).

PubMed 
    MATH 
    
                    Google Scholar 
                

Patient Episode Database for Wales dhcw.nhs.wales/information-services/health-intelligence/pedw-data-online/ (Digital Health and Care Wales, 2020).

OPCS Classification of Interventions and Procedures www.datadictionary.nhs.uk/supporting_information/opcs_classification_of_interventions_and_procedures.html (NHS England, 2023).

ICD-10: International Statistical Classification of Diseases and Related Health Problems: Tenth Revision apps.who.int/iris/handle/10665/42980 (WHO, 2004).

Outpatient Database for Wales (OPDW) web.www.healthdatagateway.org/dataset/d331159b-b286-4ab9-8b36-db39123ec229 (Digital Health and Care Wales, 2022).

Welsh Cancer Intelligence and Surveillance Unit (WCISU) phw.nhs.wales/services-and-teams/welsh-cancer-intelligence-and-surveillance-unit-wcisu/ (Public Health Wales, 2023).

Annual District Death Extract (ADDE) web.www.healthdatagateway.org/dataset/15cf4241-abad-4dcc-95b0-8cd7c02be999 (Digital Health and Care Wales, 2023).

CT1303_2011 Census www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/adhocs/2089ct13032011census (ONS, 2024).

Bricout, H., Haugh, M., Olatunde, O. & Prieto, R. G. Herpes zoster-associated mortality in Europe: a systematic review. BMC Public Health 15, 466 (2015).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Stoltenberg, E. A. Regression discontinuity design with right-censored survival data. Preprint at arxiv.org/abs/2210.02548 (2022).

Alzheimer's Association. 2022 Alzheimer's disease facts and figures. Alzheimers Dement. 18, 700–789 (2022).

MATH 
    
                    Google Scholar 
                

Boyle, P. A. et al. Attributable risk of Alzheimer's dementia attributed to age-related neuropathologies. Ann. Neurol. 85, 114–124 (2019).

PubMed 
    MATH 
    
                    Google Scholar 
                

Boyle, P. A. et al. Person-specific contribution of neuropathologies to cognitive loss in old age. Ann. Neurol. 83, 74–83 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Villamizar-Villegas, M., Pinzon-Puerto, F. A. & Ruiz-Sanchez, M. A. A comprehensive history of regression discontinuity designs: an empirical survey of the last 60 years. J. Econ. Surv. 36, 1130–1178 (2022).

Gelman, A. & Imbens, G. Why high-order polynomials should not be used in regression discontinuity designs. J. Bus. Econ. Stat. https://doi.org/10.1080/07350015.2017.1366909 (2017).

Imbens, G. & Kalyanaraman, K. Optimal bandwidth choice for the regression discontinuity estimator. Rev. Econ. Stud. 79, 933–959 (2012).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Calonico, S., Cattaneo, M. D. & Titiunik, R. Robust nonparametric confidence intervals for regression-discontinuity designs. Econometrica 82, 2295–2326 (2014).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Imbens, G. W. & Lemieux, T. Regression discontinuity designs: a guide to practice. J. Econometr. 142, 615–635 (2008).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Bor, J., Moscoe, E., Mutevedzi, P., Newell, M. L. & Barnighausen, T. Regression discontinuity designs in epidemiology: causal inference without randomized trials. Epidemiology 25, 729–737 (2014).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Oehlert, G. W. A note on the delta method. Am. Stat. 46, 27–29 (1992).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Kolesár, M. & Rothe, C. Inference in regression discontinuity designs with a discrete running variable. Am. Econ. Rev. 108, 2277–2304 (2018).

MATH 
    
                    Google Scholar 
                

Armstrong, T. B. & Kolesár, M. Simple and honest confidence intervals in nonparametric regression. Quant. Econ. 11, 1–39 (2020).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

VanderWeele, T. J. Mediation analysis: a practitioner's guide. Ann. Rev. Publ. Health 37, 17–32 (2016).

MATH 
    
                    Google Scholar 
                

Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. B 57, 289–300 (1995).

MathSciNet 
    MATH 
    
                    Google Scholar 
                

Conrad, N. et al. Autoimmune diseases and cardiovascular risk: a population-based study on 19 autoimmune diseases and 12 cardiovascular diseases in 22 million individuals in the UK. Lancet 400, 733–743 (2022).

PubMed 
    
                    Google Scholar 
                

Mukherjee, M., Wyatt, J. C., Simpson, C. R. & Sheikh, A. Usage of allergy codes in primary care electronic health records: a national evaluation in Scotland. Allergy 71, 1594–1602 (2016).

PubMed 
    
                    Google Scholar 
                

Xie, M. et al. A natural experiment on the effect of herpes zoster vaccination on dementia. Preprint at https://osf.io/cfnr6/?view_only=d3774e4fda2649e2b2031431b1234874 (2025).

Download references

This study makes use of anonymized data held in the SAIL Databank. We thank the members of the SAIL Databank analytical services team for continuous advice and support throughout all stages of the project. We acknowledge all the data providers who made anonymized data available for research. The responsibility for the interpretation of the data supplied by SAIL is that of the authors alone. SAIL bears no responsibility for the further analysis or interpretation of their data, over and above that published by SAIL. This study was funded by grants to P.G. by The Phil & Penny Knight Initiative for Brain Resilience at the Wu Tsai Neurosciences Institute, Stanford University (KPI-003), the National Institute on Aging (R01AG084535), National Institute of Allergy and Infectious Diseases (DP2AI171011) and Chan Zuckerberg Biohub–San Francisco.

These authors contributed equally: Markus Eyting, Min Xie

Division of Primary Care and Population Health, Department of Medicine, Stanford University, Stanford, CA, USA

Markus Eyting, Min Xie, Felix Michalik, Seunghun Chung & Pascal Geldsetzer

Leibniz Institute for Financial Research SAFE, Frankfurt am Main, Germany

Markus Eyting

Faculty of Law and Economics, Johannes Gutenberg University Mainz, Mainz, Germany

Markus Eyting

Heidelberg Institute of Global Health (HIGH), Heidelberg University Hospital, Heidelberg, Germany

Min Xie & Felix Michalik

Department of Economics, Vienna University of Economics and Business, Vienna, Austria

Simon Heß

Department of Epidemiology and Population Health, Stanford University, Stanford, CA, USA

Pascal Geldsetzer

The Phil and Penny Knight Initiative for Brain Resilience at the Wu Tsai Neurosciences Institute, Stanford University, Stanford, CA, USA

Pascal Geldsetzer

Chan Zuckerberg Biohub – San Francisco, San Francisco, CA, USA

Pascal Geldsetzer

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

M.E. and M.X. contributed equally to this work. M.E. co-conceived the study, devised the methodology, analysed and processed the data, created data visualizations, interpreted the results, wrote the Methods section of the original draft, and reviewed and edited the original draft. M.X. co-conceived the study, devised the methodology, analysed and processed the data, created data visualizations, interpreted the results, wrote the Methods section of the original draft, and reviewed and edited the original draft. F.M. interpreted the results, and reviewed and edited the original draft. S.H. devised the methodology, consulted on the data analysis, wrote the Methods section of the original draft, interpreted the results, and reviewed and edited the original draft. S.C. interpreted the results, and reviewed and edited the original draft. P.G. conceived the overall project, acquired funding, co-conceived the study, devised the methodology, was responsible for administration and supervision, interpreted the results and wrote the original draft.

Correspondence to
                Pascal Geldsetzer.

The authors declare no competing interests.

Nature thanks Anupam Jena, Mike Nalls and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary Figs. 1–32 and Supplementary Tables 1–3.

All Read and ICD-10 codes used throughout the analysis.

Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/.

Reprints and permissions

Eyting, M., Xie, M., Michalik, F. et al. A natural experiment on the effect of herpes zoster vaccination on dementia.
                    Nature  (2025). https://doi.org/10.1038/s41586-025-08800-x

Download citation

Received: 04 November 2023

Accepted: 18 February 2025

Published: 02 April 2025

DOI: https://doi.org/10.1038/s41586-025-08800-x

Anyone you share the following link with will be able to read this content:

Sorry, a shareable link is not currently available for this article.




                            Provided by the Springer Nature SharedIt content-sharing initiative
                        

Advertisement



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited

Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01020-3'>Revealed: first DNA profiles of ancient people who roamed a lush Sahara</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 15:31:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement

You can also search for this author in PubMed
 Google Scholar



                You have full access to this article via your institution.

Green oasis: during a time known as the African Humid Period, it's thought that the Sahara was a lush savannah.Credit: Henrik Karlsson/Getty

The Sahara Desert has not always been the arid, inhospitable landscape we know today. Between 14,500 and 5,000 years ago, the area was unrecognizable, transformed into a lush savannah by an unusually wet interval called the African Humid Period. People roamed this green landscape for thousands of years before it was again lost to sand.

From Vikings to Beethoven: what your DNA says about your ancient relatives

From Vikings to Beethoven: what your DNA says about your ancient relatives

Ancient DNA extracted from two women who died in what is now Libya around 7,000 years ago is now helping researchers to reconstruct the origins of these early Saharans. The women's DNA profiles, described in a study published on 2 April in Nature1, represent the first full Saharan genomes from the African Humid Period — and reveal that the people were remarkably isolated from other African populations.

“The prehistory of North Africa is a big puzzle, and we only have a few pieces available,” says Rosa Fregel, a geneticist at the University of La Laguna in San Cristobal, Spain, who was not involved in the research. The work is “a significant contribution to the palaeogenomics of North Africa”, she says.

Ancient genomes from North Africa are hard to come by. Almost all palaeogenetic work is concentrated in Europe and Asia. Ancient DNA is especially rare in the Sahara, where high temperatures and strong ultraviolet light quickly degrade genetic material in remains.

The Takarkori rock shelter in Libya, where the remains were unearthed.Credit: University of Rome La Sapienza

That's why it's important to explore sites that are protected from the elements, says Nada Salem, an archaeologist at the Max Plank Institute for Evolutionary Anthropology in Leipzig, Germany. 

One such site is the Takarkori rock shelter in southwestern Libya. Between 2003 and 2007, archaeologists uncovered the remains of 15 people who were buried between 8,900 and 4,800 years ago at Takarkori. Two of the corpses — both belonging to women who lived between 7,000 and 6,000 years ago — had naturally mummified.

Archaeological evidence at the site suggested that the Takarkori women belonged to a group of herders who appeared in the region around 8,000 years ago. This marked a major transition in the way of life of early Saharans, who had previously all been hunter-gatherers. Some researchers have suggested that Saharans learnt herding by intermarrying with people who were migrating into North Africa from the Levant.

To test this, Salem and her colleagues sequenced the Takarkori genomes and compared the DNA to that of around 800 modern humans and 117 ancient genomes from around Africa, southern Europe and the Middle East. The team found that the Takarkori women had only small traces of Levant ancestry — suggesting that any intermingling had happened long before the advent of herding in the region.

What's more, the analysis struggled to connect these early Saharans to any other ancient group. “This was puzzling for us. How is it that this lineage has not spread either to the east or the west or to the south?” says Salem.

or

doi: https://doi.org/10.1038/d41586-025-01020-3

Read the related News & Views article: ‘Rare ancient DNA from Sahara opens a window on the region's verdant past'

Salem, N. et al. Nature https://doi.org/10.1038/s41586-025-08793-7 (2025).

Article 
    
                    Google Scholar 
                

Vai, S. et al. Sci. Rep. 9, 3530 (2019).

Article 
    PubMed 
    
                    Google Scholar 
                

Download references

Reprints and permissions




                        
                        Read the paper: Ancient DNA from the Green Sahara reveals ancestral North African lineage
                    





                        
                        From Vikings to Beethoven: what your DNA says about your ancient relatives
                    





                        
                        Ancient DNA from Maya ruins tells story of ritual human sacrifices
                    





                        
                        Ancient DNA traces origin of Black Death
                    





                        
                        Ancient DNA reveals origins of multiple sclerosis in Europe
                    





                        
                        Ancient voyage carried Native Americans' DNA to remote Pacific islands
                    




                        
                        Ancient dog DNA reveals 11,000 years of canine evolution
                    




                        
                        How one language family took over the world: ancient DNA traces its spread
                    


Ancient DNA from the Green Sahara reveals ancestral North African lineage

Article 02 APR 25

Why Africans should be telling the story of human origins

Career Q&A 24 MAR 25

Long-term studies provide unique insights into evolution

Review Article 19 MAR 25

Rare ancient DNA from Sahara opens a window on the region's verdant past

News & Views 02 APR 25

Ancient DNA from the Green Sahara reveals ancestral North African lineage

Article 02 APR 25

23andMe plans to sell its huge genetic database: could science benefit?

News 31 MAR 25

Job Title: Chief Editor, Nature Biomedical Engineering Locations: New York, Beijing or Shanghai (Hybrid Working Model) Application Deadline: April ...

New York City, New York (US)

Springer Nature Ltd

Leading Scholars、Excellent Young Scholars(Overseas)、Outstanding Young Talents、Professor 、Associate Professor

Xian, Shaanxi (CN)

Hospital of Stomatology Xi'an Jiaotong University

Faculty Positions in Advanced Materials Thrust, Function Hub, HKUST(GZ).

Guangzhou, Guangdong, China

The Hong Kong University of Science and Technology (Guangzhou)

The Department of Dermatology at the University of California, Irvine anticipates openings for a postdoctoral scholar. Applications are being sough...

University of California Irvine, Irvine

Ampi Montiel, AP&HR Manager

Montréal, Quebec (CA)

University of Montreal (UdeM)




                        
                        Read the paper: Ancient DNA from the Green Sahara reveals ancestral North African lineage
                    





                        
                        From Vikings to Beethoven: what your DNA says about your ancient relatives
                    





                        
                        Ancient DNA from Maya ruins tells story of ritual human sacrifices
                    





                        
                        Ancient DNA traces origin of Black Death
                    





                        
                        Ancient DNA reveals origins of multiple sclerosis in Europe
                    





                        
                        Ancient voyage carried Native Americans' DNA to remote Pacific islands
                    




                        
                        Ancient dog DNA reveals 11,000 years of canine evolution
                    




                        
                        How one language family took over the world: ancient DNA traces its spread
                    


An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday.

Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.livescience.com/planet-earth/geology/north-america-is-dripping-down-into-earths-mantle-scientists-discover'>A lost chunk of ancient continent is sucking bits of North America into Earth's mantle</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.livescience.com', 'title': 'Live Science'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 15:27:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Seismic mapping of North America has revealed that an ancient slab of crust buried beneath the Midwest is causing the crust above it to "drip" and suck down rocks from across the continent.


When you purchase through links on our site, we may earn an affiliate commission. Here's how it works.


An ancient slab of Earth's crust buried deep beneath the Midwest is sucking huge swatches of present-day's North American crust down into the mantle, researchers say.

The slab's pull has created giant "drips" that hang from the underside of the continent down to about 400 miles (640 kilometers) deep inside the mantle, according to a new study. These drips are located beneath an area spanning from Michigan to Nebraska and Alabama, but their presence appears to be impacting the entire continent.

The dripping area looks like a large funnel, with rocks from across North America being pulled toward it horizontally before getting sucked down. As a result, large parts of North America are losing material from the underside of their crust, the researchers said.

"A very broad range is experiencing some thinning," study lead author Junlin Hua, a geoscientist who conducted the research during a postdoctoral fellowship at The University of Texas (UT) at Austin, said in a statement. "Luckily, we also got the new idea about what drives this thinning," said Hua, now a professor at the University of Science and Technology of China.

Related: Earth's crust is peeling away under California

The researchers found that the drips result from the downward dragging force of a chunk of oceanic crust that broke off from an ancient tectonic plate called the Farallon plate.

The Farallon plate and the North American plate once formed a subduction zone along the continent's west coast, with the former sliding beneath the latter and recycling its material into the mantle. The Farallon plate splintered due to the advance of the Pacific plate roughly 20 million years ago, and remnant slabs subducted beneath the North American plate slowly drifted off.

Get the world's most fascinating discoveries delivered straight to your inbox.

One of these slabs currently straddles the boundary between the mantle transition zone and the lower mantle roughly 410 miles (660 km) beneath the Midwest. Dubbed the "Farallon slab" and first imaged in the 1990s, this piece of oceanic crust is responsible for a process known as "cratonic thinning," according to the new study, which was published March 28 in the journal Nature Geoscience.



Cratonic thinning refers to the wearing away of cratons, which are regions of Earth's continental crust and upper mantle that have mostly remained intact for billions of years. Despite their stability, cratons can undergo changes, but this has never been observed in action due to the huge geologic time scales involved, according to the study.

Now, for the first time, researchers have documented cratonic thinning as it occurs. The discovery was possible thanks to a wider project led by Hua to map what lies beneath North America using a high-resolution seismic imaging technique called "full-waveform inversion." This technique uses different types of seismic waves to extract all the available information about physical parameters underground.



"This sort of thing is important if we want to understand how a planet has evolved over a long time," study co-author Thorsten Becker, a distinguished chair in geophysics at UT Austin, said in the statement. "Because of the use of this full-waveform method, we have a better representation of that important zone between the deep mantle and the shallower lithosphere [crust and upper mantle]."

—Scientists discover 'sunken worlds' hidden deep within Earth's mantle that shouldn't be there

—Earth's crust may be building mountains by dripping into the mantle

—Gargantuan waves in Earth's mantle may make continents rise, new study finds

To test their results, the researchers simulated the impact of the Farallon slab on the craton above using a computer model. A dripping area formed when the slab was present, but it disappeared when the slab was absent, confirming that — theoretically, at least — a sunken slab can drag rocks across a large area down into Earth's interior.

Dripping beneath the Midwest won't lead to changes at the surface anytime soon, the researchers said, adding that it may even stop as the Farallon slab sinks deeper into the lower mantle and its influence over the craton wanes.

The findings could help researchers piece together the enormous puzzle of how Earth came to look the way it does today. "It helps us understand how do you make continents, how do you break them, and how do you recycle them," Becker said.

Sascha is a U.K.-based staff writer at Live Science. She holds a bachelor's degree in biology from the University of Southampton in England and a master's degree in science communication from Imperial College London. Her work has appeared in The Guardian and the health website Zoe. Besides writing, she enjoys playing tennis, bread-making and browsing second-hand shops for hidden gems.

Please logout and then login again, you will then be prompted to enter your display name.

Lake Salda: The only place on Earth similar to Jezero crater on Mars

30,000-year-old fossilized vulture feathers 'nothing like what we usually see' preserved in volcanic ash

Best smart telescopes 2025: The latest technology for exploring the universe


Live Science is part of Future US Inc, an international media group and leading digital publisher. Visit our corporate site.


©
Future US, Inc. Full 7th Floor, 130 West 42nd Street,
New York,
NY 10036. </p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08793-7'>Ancient DNA from the Green Sahara reveals ancestral North African lineage</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 15:11:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement


Nature

                         (2025)Cite this article


Metrics details

Although it is one of the most arid regions today, the Sahara Desert was a green savannah during the African Humid Period (AHP) between 14,500 and 5,000 years before present, with water bodies promoting human occupation and the spread of pastoralism in the middle Holocene epoch1. DNA rarely preserves well in this region, limiting knowledge of the Sahara's genetic history and demographic past. Here we report ancient genomic data from the Central Sahara, obtained from two approximately 7,000-year-old Pastoral Neolithic female individuals buried in the Takarkori rock shelter in southwestern Libya. The majority of Takarkori individuals' ancestry stems from a previously unknown North African genetic lineage that diverged from sub-Saharan African lineages around the same time as present-day humans outside Africa and remained isolated throughout most of its existence. Both Takarkori individuals are closely related to ancestry first documented in 15,000-year-old foragers from Taforalt Cave, Morocco2, associated with the Iberomaurusian lithic industry and predating the AHP. Takarkori and Iberomaurusian-associated individuals are equally distantly related to sub-Saharan lineages, suggesting limited gene flow from sub-Saharan to Northern Africa during the AHP. In contrast to Taforalt individuals, who have half the Neanderthal admixture of non-Africans, Takarkori shows ten times less Neanderthal ancestry than Levantine farmers, yet significantly more than contemporary sub-Saharan genomes. Our findings suggest that pastoralism spread through cultural diffusion into a deeply divergent, isolated North African lineage that had probably been widespread in Northern Africa during the late Pleistocene epoch.

Following the last glacial period, a climatic transformation in the Sahara desert led to the AHP, which peaked around 11,000 to 5,000 years ago3,4. During this period of increased humidity, the region transformed into a ‘Green Sahara' with savanna-like landscapes, varying tree cover, permanent lakes and extensive river systems5 (Fig. 1). Evidence from ancient lake deposits, pollen samples and archaeological artifacts confirm human presence, hunting, herding and resource gathering in the currently arid desert region1,6,7. However, despite this rich history, much about the genetic history of the human population of the Green Sahara remains unclear due to limited DNA preservation under the current climatic conditions. Ancient DNA data from northwestern Africa points to a stable and isolated genetic population from at least 15,000 to 7,500 years ago2,8. This stability was disrupted by the arrival of early farming groups from southwestern Europe between 7,500 and 5,700 years ago who marked the beginning of the Neolithic in the Maghreb by introducing farming practices to the local foragers9. The earliest herders with their livestock entered Africa probably along the Sinai and the Red Sea routes, after which they rapidly spread into northeastern Africa and reached the Central Sahara around 8,300 years ago10. By 6,400 years ago, further gene flow occurred with the appearance of ancestry associated with Neolithic groups from the Levant, whose archaeological signatures are visible in the Eastern Sahara9,11,12. A previous study13 analysed mitochondrial DNA (mtDNA) from individuals recovered from the Takarkori rock shelter in the Tadrart Acacus Mountains of southwestern Libya—the same individuals examined in this study—providing the first ancient DNA from pastoralists of the Green Sahara. However, non-recombining and therefore effectively single genetic loci like mtDNA have much less statistical power to reveal population dynamics than genome-wide autosomal data. Their origins and whether the arrival of pastoralism into the Green Sahara was linked to the movement of peoples from the Levant or rather cultural diffusion remain a matter of debate10,14.

a, Timeline of climate phases and subsistence strategies during the late Pleistocene and the Holocene in North-East Africa and Central Sahara. The radiocarbon dates for both Takarkori individuals are given by the black diamond and circle. b,c, The distribution of ecozones in Northern Africa in the Green Sahara period during the early Holocene 9,000 years ago (b) and in recent times (1901–1930) (c) using the dynamic vegetation model Carbon Assimilation in the Biosphere (CARAIB). The location of the Takarkori rock shelter site is marked with a black square. The maps are adapted from refs. 20 and 60 under a Creative Commons licence CC BY 4.0.

Here we present first genome-wide data obtained from the same two approximately 7,000-year-old Saharan herders, recovered from the Takarkori rock shelter in the Central Sahara (Supplementary Figs. 1.2–1.5), a site that has yielded an exceptional wealth of data and material remains15,16,17,18,19. Our findings show that these individuals predominantly carry a previously unknown ancestral North African lineage that lacks the Neanderthal admixture typically found outside Africa and appears to have remained largely isolated, with the notable exception of small traces of Levantine admixture. These results support that pastoralism in the Sahara was established through cultural diffusion10 rather than significant human gene flow. Furthermore, the Takarkori individuals exhibit a close genetic affinity to Northwestern African foragers but no substantial ties with sub-Saharan African lineages, implying no detectable genetic exchange across the Green Sahara during the AHP from sub-Saharan to northern Africa. For a non-peer-reviewed Arabic summary of the article, see Supplementary Note 3.

The Takarkori rock shelter—situated in southwestern Libya's Tadrart Acacus Mountains—offers a remarkable glimpse into the Sahara's greener past15,20. The excavations at this archaeological site revealed a timeline of human settlement from Late Acacus hunter-gatherer-fishers from around 10,200 calibrated years before the present (cal. bp) to a long Pastoral Neolithic occupation, dated from approximately 8,300 to 4,200  cal. bp10,21. Data from these latter periods traces the sociocultural trajectories of Neolithic herding societies in Central Sahara, from early livestock introduction to the development of a full pastoral economy characterized by transhumance and the use of secondary product22,23 (Supplementary Note 1). Within the deepest recess of the rock shelter, 15 human burials were unearthed, pertaining to a timeframe between approximately 8,900 and 4,800 cal. bp22, with the majority dating back to the Early (8,300–7,300 cal. bp) and Middle (7,100–5,600 cal. bp) Pastoral period22. Strontium isotope analysis on the remains, primarily of women of reproductive age, children and juveniles, indicated a local geographical origin24. Two naturally mummified adult female individuals, attributed to the Middle Pastoral Period, were selected for our DNA analysis. These individuals were directly radiocarbon dated to 7,158–6,796 and 6,555–6,281 cal. bp (95.4% probability), respectively13 (Supplementary Note 1).

We extracted DNA from the powdered tooth root from individual TKH001 and from two fibula fragments from TKH009. Given their extremely low endogenous human DNA content (0.085–1.363%; Supplementary Table 2.1), we opted for a DNA capture approach to cost-effectively retrieve informative single-nucleotide polymorphisms (SNPs) for autosomal analysis. Sampling of other skeletal elements in the future may provide DNA extracts with higher percentage of endogenous DNA that allow for whole-genome shotgun sequencing. Using dedicated ancient DNA protocols (Methods), we prepared DNA libraries and enriched them through a DNA hybridization approach with the Twist Ancient DNA panel25 targeting 1.4 million SNPs for TKH001 and the 1240K panel26 targeting 1.2 million SNPs for TKH009. Despite challenging preservation conditions, sequencing yielded 881,765 SNPs for TKH001 and 23,317 SNPs for TKH009 (Supplementary Table 2.4). For TKH001, we conducted additional enrichment for 1.7 million SNPs, targeting sites informative about Neanderthal and Denisovan admixture (Archaic Admixture SNP capture panel)27. DNA sequences from both Takarkori individuals had a post-mortem degradation pattern typical of ancient DNA and low contamination estimates (Supplementary Figs. 2.1, 2.2 and Supplementary Table 2.3).

To visualize the variation in the genetic ancestry of the Takarkori individuals, we performed a principal component analysis (PCA) using genome-wide data from 795 present-day individuals from the whole African continent, the Near East and Southern Europe, all genotyped on the Human Origins SNP panel28,29,30,31,32,33,34,35,36,37,38 (Supplementary Data 1 and 2). We then projected the Takarkori individuals and 117 relevant published ancient genomes2,8,9,35,36,39,40,41,42,43,44 (Supplementary Data 2) onto the first two principal components (PCs). The Takarkori individuals plot broadly intermediate between West African and Near Eastern groups on PC1, albeit closer to West Africans (Extended Data Fig. 1, Supplementary Fig. 2.6 and Supplementary Data 3). To obtain a finer-scale view, we restricted the African populations to West Africa, the Sahel and East Africa, while retaining the Near Eastern and Southern European populations (Supplementary Data 3). This PCA captured the geographical distributions of these populations, whereby PC1 separates African from non-African populations and PC2 differentiates within Africa, particularly separating Sahel/West African from East African populations (Extended Data Fig. 2). Both Takarkori individuals maintained a distinct position, intermediate between Sahel/West and East African populations (Fig. 2a), and formed a tight cluster with overlapping 95% confidence interval (CI) ellipses (Supplementary Fig. 2.4). The PCA projections broadly mirror geography, including the placement of Takarkori. For subsequent population genetic analyses, we merged the low-coverage data from the TKH009 individual with the high-coverage data from the TKH001 individual. We note that many of the signals in this group analysis are probably driven by TKH001 owing to its much higher coverage.

a, PCA with projecting key ancient groups from the region (Supplementary Data 3). b, The geographical locations of ancient genomes from Africa and the Near East included in our analysis. ChL, Chalcolithic; EN, Early Neolithic; EpiPalaeo, Epipalaeolithic; IA, Iron Age; IAM, Ifri n'Amr o'Moussa; KEB, Kehf el Baroud; KTG, Kaf Taht el-Ghar; LIA, Late Iron Age; LN, Late Neolithic; MN, Middle Neolithic; OUB, Ifri Ouberrid; Palaeo, Palaeolithic; SKH, Skhirat-Rouazi.

To probe for shared genetic drift of Takarkori with other ancient and modern genomes, we computed outgroup f3 statistics30 of the form f3(Takarkori, X; South Africa 2,000 cal. bp), with X representing worldwide ancient and present-day test populations and South Africa 2,000 cal. bp as the deepest modern human outgroup lineage44. We found that the Takarkori individuals share the most genetic drift with Moroccan Palaeolithic and Early Neolithic individuals. Specifically, we observed the highest level with an Epipalaeolithic individual from Ifri Ouberrid and similarly elevated shared drift with the 15,000-year-old foragers from Taforalt and the Early Neolithic individuals from Ifri n'Amr o'Moussa (Fig. 3a, Extended Data Fig. 3 and Supplementary Fig. 2.12). Both Epipalaeolithic and Early Neolithic groups have been previously shown to maintain high genetic continuity with the much older Taforalt group8,9, explaining the similarly elevated shared drift statistics between all three groups and the Takarkori individuals.

a, Outgroup-f3 statistics f3(Takarkori, X; South Africa 2,000 cal. bp), where X represents ancient groups, mapped at their geographical positions. The colour gradient from blue to green indicates the genetic proximity to Takarkori, with the bluer colours representing closer genetic relationships. The statistics and their associated s.e. values for the top 70 signals are presented in Supplementary Fig. 2.12. b, No group shares extra affinity with Takarkori genomes compared with Taforalt, as measured by f4 statistics of the form f4(chimpanzee, X; Takarkori, Taforalt). The error bars represent 3 s.e. Group colours follow the same scheme as in Fig. 2. A more extensive list is presented in Supplementary Fig. 2.19. LSA, Late Stone Age; N, neolithic.

When restricting the comparative groups to present-day African and Near Eastern populations, we found that a group of individuals defined as FulaniA in a previous study38, which includes individuals from relatively less admixed Fulani herders from eight Sahelian countries, shows an increased genetic affinity to Takarkori, although less so to Taforalt (Extended Data Fig. 4 and Supplementary Fig. 2.16). This finding aligns with ref. 38, which observed a non-sub-Saharan ancestry component in FulaniA similar to that found in the Taforalt and Late Neolithic Moroccans. To further probe for Takarkori-like ancestry in FulaniA, we conducted an f4 analysis f4(chimpanzee, X; Masai/Datog/Iraqw, Takarkori), using Masai/Datog/Iraqw as baseline references owing to their similar amount of out-of-Africa (OoA) ancestry to Takarkori (Supplementary Fig. 2.20.1–2.20.3). The results indicated that the FulaniA have an increased affinity to Takarkori-like ancestry, as do other Sahelian and West African groups. These findings are consistent with the archaeological evidence of the southward expansion of Pastoral Neolithic groups from Central Sahara. Rock art, ceramic production and funerary practices provide detailed indications of the spread of these herders at the end of the Middle Holocene, probably driven by the progressive aridification of the Central Saharan regions45,46.

Given the high amount of shared genetic drift between the Takarkori and the ancestry first appearing in the 15,000-year-old Taforalt individuals in the outgroup f3 statistics, we subsequently investigated whether the Takarkori genomes share more alleles with other human groups compared to the Taforalt genome. For this analysis, we computed the statistics f4(chimpanzee, X; Takarkori, Taforalt), where X represents ancient and present-day groups from Africa and Eurasia. We obtained significantly positive values for Eurasian and Eurasian-admixed African groups, suggesting that the Taforalt group shares more alleles with these groups than Takarkori does. Conversely, none of the ancient or modern groups tested showed a significantly negative signal, indicating no detected closer affinity with Takarkori than Taforalt. Notably, both ancient and modern sub-Saharan groups, who are mostly unadmixed with Eurasian groups, yielded no significant value (|Z| < 3), suggesting that these groups are equally distant from both the Takarkori and Taforalt groups (Fig. 3b and Supplementary Data 4). Moreover, when running statistics f4(Chimp, Takarkori; X, Taforalt), we found significantly positive values for every population X, indicating that Takarkori shares more alleles with Taforalt than any other group (Extended Data Fig. 5 and Supplementary Data 4).

At the mtDNA level, both Takarkori individuals belong to a basal branch of haplogroup N, representing one of the deepest mtDNA lineages outside sub-Saharan Africa and predating present-day N-derived mtDNAs13. Using BEAST analysis for mtDNA dating, which included additional sequences from Upper Palaeolithic individuals and the dataset described previously13, we corroborate the previous findings of ref. 13 that the Takarkori individuals carried a basal N haplogroup lineage13, and refined the molecular split date estimate to 61,343 years old (95% highest posterior density (HPD) = 54,408–69,046) (Extended Data Fig. 6). Notably, the mtDNA lineage of the Oase 1 individual falls more basal to haplogroup N, suggesting an earlier split from the OoA lineage before the divergence of the Takarkori lineage. However, owing to incomplete lineage sorting and mtDNA representing a single lineage, the exact timing of the underlying population splits remains uncertain.

Previous work modelled the Taforalt group's ancestry as a two-way admixture of approximately 63.5% Natufian (ancient Levantine foragers) and 36.5% sub-Saharan African ancestries2. However, this model using the software qpAdm2 could not pinpoint the origin of Taforalt's African ancestry, resulting in unknown ghost ancestry only broadly linked to South, East and Central African groups2. Here we included Takarkori as a possible source of the African ancestry in Taforalt in comparison to several potential sources (namely Yoruba, Dinka, Mota, Cameroon Shum Laka, Botswana Xaro Early Iron Age (EIA) and Tanzania Zanzibar 1,300 cal. bp) through rotation-based qpAdm. We found that Saharan Takarkori provides a much better fit as an African proxy for Taforalt than the sub-Saharan groups, attaining a P value of >0.05, indicative of a much better model fit compared to the other sources (P < 2.84 × 10−34) (Extended Data Figs. 7, 8 and Supplementary Tables 2.6, 2.7). With this revised model, we estimated that the Taforalt ancestry retains a comparable 60.8% (±1.8%) contribution from Natufians, with the remaining 39.2% (±1.8%) derived from Takarkori.

We next explored the direct genetic affinity between Takarkori and the OoA ancestry found in all non-African modern humans, in contrast to African groups. For this, we computed f4 statistics of the form f4(chimpanzee, Zlatý kůň; African, Takarkori). Zlatý kůň, a 45,000-year-old Upper Palaeolithic individual from Czechia, was used as a proxy for OoA ancestry as it is probably the oldest modern human sequenced to date and represents the deepest known human lineage after the OoA lineage splits from the African lineages47. Our results showed positive values for Takarkori, indicating that it is genetically closer to Zlatý kůň than to sub-Saharan Africans, including Mota, a 4,500-year-old genome from East Africa. Nevertheless, various African populations with substantial OoA admixture were still genetically closer to Zlatý kůň than to Takarkori (Extended Data Fig. 9 and Supplementary Data 5).

These results raise the question of whether Takarkori's ancestors are closely related to OoA groups but remained in Africa, or whether they received later gene flow from OoA groups. If Takarkori experienced such later gene flow, it would carry Neanderthal admixture that is found in all OoA groups. To explore this signal, we used the data generated using the Archaic Admixture SNP panel for Takarkori and the software admixfrog48 that detects Neanderthal segments and included other ancient African and Eurasian groups as well as modern sub-Saharan African populations for comparison (Supplementary Note 2). We detected a total of 12 Neanderthal fragments that surpass 0.05 cM in length (approximately 50 kb), with the longest fragment located on chromosome 1 (Extended Data Fig. 10 and Supplementary Fig. 2.32), translating to a low level of Neanderthal ancestry in the Takarkori genome of approximately 0.15% (Fig. 4a). This percentage is less than a quarter of the Neanderthal ancestry in segments longer than 0.05 cM found in Taforalt and Neolithic Morocco individuals (0.6–0.9%), and about tenfold lower than in most OoA groups (1.4–2.36%), yet significantly higher than that in other ancient and contemporary sub-Saharan African genomes, where Neanderthal ancestry was completely absent. This pattern suggests that the Takarkori individuals have received a small amount of ancestry from OoA groups. However, the estimation of admixture dating of this OoA ancestry with DATES49 based on linkage disequilibrium (Supplementary Figs. 2.26.1–2.26.3) indicated very ancient admixture events with substantial uncertainty.

a, Detectable Neanderthal ancestry in segments longer than 0.05 cM in ancient individuals from Africa and Eurasia, along with present-day sub-Saharan African groups. The error bars represent the minimum and maximum estimates from all iterations. b, The geographical locations of groups included in the analysis. c, Admixture graph modelling of Takarkori's ancestral relationship with relevant populations.

Finally, we used the find_graphs() function from the ADMIXTOOLS 250 package to model Takarkori's ancestral relationship with other populations. We used a function for automated graph exploration with a model that includes Mota, Iran Neolithic, Natufian, Taforalt, Takarkori and the outgroup chimpanzee. The model fits with small f-statistic residuals (max |f4,expected − f4,observed| = 0.27 s.e.). The fitted graph suggests that Takarkori traces most of its ancestry (93%) to a hitherto unknown North African population, in agreement with the isolation signature obtained from the f4 statistics mentioned above (Fig. 3b). This unknown North African population is closely related to OoA populations and branches off the lineage leading to OoA later than the ancient individual from Mota Cave, Ethiopia, who represents the sub-Saharan African lineage most closely related to OoA groups identified so far. In our model, the remainder of the Takarkori individuals' ancestry (7%) is derived from a deeply ancient Levantine source. The Levantine gene flow also accounts for the Neanderthal ancestry found in Takarkori as the Levantine Neolithic genomes carry around 1.86% Neanderthal ancestry, aligning with our estimates using admixfrog. The graph also models Taforalt as a mixture of a 40% contribution from a Takarkori-related branch and 60% from a Natufian-related branch, consistent with our qpAdm results (Fig. 4c and Supplementary Data 6).

Our study introduces ancient genome-wide data from humans that inhabited the Green Sahara, providing unique insights into the genomic ancestry of populations in this region. The individuals from the Takarkori rock shelter predominantly carry an ancestral African lineage, representing an ancestry profile that has not been previously described. They share the most genetic drift with Epipalaeolithic and Early Neolithic individuals from Ifri Ouberrid and Ifri n'Amr o'Moussa, as well as the 15,000-year-old foragers from the Taforalt Cave in Morocco, suggesting a long-standing and stable population in North Africa before the AHP (14,500–5,000 bp). Plausibly, this ancestry was present in large parts of Northern Africa after the OoA event, and the Takarkori individuals inherited it from the group that inhabited the area during the final period of the Late Acacus (10,200–8,000 cal. bp). In Southwestern Libya, this period preceded the arrival of domesticates and is characterized by cultural advancements within those hunter–gatherer groups10. This included an increase in sedentism51, and the use of sophisticated material cultures such as pottery, basketry, and bone and wooden tools52,53,54.

We found that individuals from Takarkori exhibit only a marginal amount of genetic admixture from Levantine groups, suggesting that the emergence of pastoralism in the Sahara was primarily driven by the dissemination of cultural practices rather than through large-scale human migration, as suggested on archaeological basis10. Material culture at the onset of the earliest pastoral period shows both continuity and change, reflecting possibly complex assimilation dynamics during socioeconomic transitions10,19. These patterns may further suggest gradual cultural transformations rather than abrupt population replacement. This interpretation is further supported by the comparatively lower levels of Neanderthal genetic admixture found in the Takarkori individuals. Our admixture dating analysis points to events far back in time, suggesting a more heterogeneous spread of pastoralism and food production in the Sahara compared to Morocco and East Africa, where there was a noticeable increase in recent Levantine genetic admixture9,35,39,43. In addition to these findings, a run of homozygosity (ROH) analysis for TKH001 revealed no ROH segments larger than 12 cM, indicating no close-kin inbreeding (Supplementary Fig. 2.28). Several shorter ROH segments over 4 cM suggest an effective population size (Ne) of around 1,000 individuals, reflecting a moderately sized population.

Our research offers insights into the ancestry of the previously published Taforalt hunter-gatherers. While a previous study2 could not precisely ascribe the ‘sub-Saharan' component in the Taforalt genome, we now identify this ancestry as a deep North African lineage, with higher proportions found in the Saharan Takarkori individuals. This refines the earlier model, which proposed a dual admixture of Natufian and broadly sub-Saharan African ancestries. Our updated model suggests that the Taforalt ancestry is composed of a 60% contribution from a Natufian-like Levantine population, with the remaining 40% derived from a Takarkori-like ancestral North African population (Extended Data Fig. 8). Notably, both the late Pleistocene Taforalt and the mid-Holocene Takarkori individuals demonstrate equally distant relationships with sub-Saharan African lineages. This pattern suggests that no substantial genetic exchanges across the Green Sahara occurred during the AHP or other humid periods preceding the Later Pleistocene. The Sahara, spanning around 9 million km2 and housing diverse biomes, such as grasslands, wetlands, woodlands, lakes, mountains and savannas55,56, probably saw fragmented habitats impacting human gene flow. These ecological barriers, combined with social and cultural barriers, spatial structuring of populations, and the selective adoption of specific practices, may have facilitated the widespread dissemination of similar archaeological features57,58, while limiting extensive genetic admixture. This genetic discontinuity is consistent with modern data, which show substantial genetic differentiation across the Sahara beyond just a geographical gap59. Our findings suggest that sporadic Green Sahara events, particularly before pastoralism, were insufficient to allow for considerable genetic exchange, mirroring the Sahara's persistent role in limiting human genetic flow, as reflected in both ancient and modern population structures.

Our findings represent an important initial step, and future genetic studies could reveal more refined insights into human migration and gene flow across the Sahara. As sequencing costs continue to fall, whole-genome sequencing could enable more unbiased estimates regarding OoA events and other key aspects of human evolution.

The present-day communities of the Tadrart Acacus region, the Kel Tadrart pastoral group, have been actively engaged in both the excavation activities at the site (field assistance, sieving and so on) and in the decision-making processes and actions related to its conservation efforts. Other local community members contributed to the logistical management of the mission's tented camp, performing various specialized labour roles.

Although these communities maintain centuries-long connections to the territory, they do not express a specific cultural or historical affinity with the prehistoric burials uncovered there. This sentiment extends to the broader archaeological contexts and local rock art, which are viewed as expressions of ancient, pre-Islamic times, for which no direct cultural link is recognized or claimed.

Despite the lack of a perceived direct connection, ethical considerations and the implications of analysing and handling human remains have been thoroughly discussed with the Department of Antiquities (DoA) in Libya, with substantial input from M.F.M.A.-F. and M.T., both affiliated with the same institution. After the excavation concluded, and in subsequent years, the results were shared in meetings held at the local governorate office in Ghat. Unfortunately, the onset of the Libyan Revolution in 2011, shortly after the excavation activities ended, disrupted further contact. However, over the past year, coinciding with the preparation of this manuscript, online meetings with these co-authors and other DoA representatives have resumed, fostering a renewed dialogue on these issues. These inclusive approaches have ensured that both scientific research and conservation practices are aligned with local values and ethical standards, promoting a collaborative framework for heritage management and scientific collaboration.

Takarkori rock shelter counted 15 burials, varying in phases and preservation states, predominantly from the Early and Middle Pastoral periods, spanning the eighth and the seventh millennium  cal. bp16,22. The burials, found immediately adjacent to the rock wall and with scarce grave goods, appear to only consist of women and children13,22,61 (Supplementary Note 1). Osteological information is reported and discussed elsewhere22, although, in brief, age estimation for subadult individuals was based on methodologies for skeletal development62,63 and dental maturation and eruption64 developed previously. For adult individuals, sex determination was based on the observation of morphological traits of the skull and the pelvis, as synthesized previously65, whereas estimation of age at death was based on the modifications of the pubic symphysis as proposed previously66, cranial sutures closure67 as well as patterns of occlusal dental wear68, which took into account the possible role of the sandy environment. Detailed layouts of the excavation areas, along with the stratigraphic and chronological insights, are depicted in Supplementary Fig 1.4. Standard stratigraphical techniques were used in the excavation of skeletal remains, with strict adherence to precautions for the sampling of biological materials. Sediments associated with the remains underwent sieving using a 2 mm mesh, and samples for laboratory-based studies were collected. The excavation and handling of human remains have been extensively discussed with the Department of Antiquities (DOA) in Tripoli, Libya (formerly the Socialist People's Libyan Arab Jamahiriya), particularly concerning ethical issues. Approval for the archaeological excavation was granted on 28 January 2004, under the reference number  (translated as H98-10000 TATM 49463). Currently, the human remains are curated at the Museum of Anthropology of the University of Rome, La Sapienza.

Tooth root from individual TKH001 and fibula fragments from individual TKH009, both previously labelled as TK H1 and TK H9, respectively13, were sourced for this study. The sampling and DNA extraction procedures were carried out in a specialized ancient DNA facility at the University of Florence's Molecular Anthropology department, as outlined previously13. DNA was extracted according to the protocol described previously69. From the extracted DNA, a double-stranded DNA library was prepared in the same Florence facility, without undergoing uracil-DNA-glycosylase (UDG) treatment. An additional single-stranded DNA library, better at capturing short fragments, was prepared at the Max Planck Institute of Geoanthropology (formerly Max Planck Institute for the Science of Human History, MPI-SHH) and/or the Max Planck Institute for Evolutionary Anthropology (MPI-EVA). The prepared DNA libraries underwent shotgun sequencing to a depth of 3–5 million reads, using 75 bp single-end and/or 50 bp paired-end configurations on the Illumina HiSeq 4000 system at MPI-EVA for initial quality assessment (Supplementary Table 2.1). Both TKH001 and TKH009 libraries underwent in-solution capture targeting over a million SNPs using the Twist Ancient DNA25 and 1240k26 panels, with an additional specialized ‘archaic ancestry' panel applied to the single-stranded library from TKH001. After enrichment, all of the libraries were sequenced to 20 million reads on the Illumina HiSeq 4000 system at MPI-EVA. Read adapters were removed using AdapterRemoval70 v.2.3.0 as part of the EAGER (v.1.92.56)71, and the genome-wide captures were aligned to the human reference genome (hg19) using a mapping quality filter of 25 with BWA aligner72 v.7.12. Duplicate reads were eliminated using DeDup (v.0.12.2), which can be found at GitHub (https://github.com/apeltzer/DeDup). The contamination of the single-stranded library from TKH001 was evaluated with AuthentiCT73 v.1.0 and hapCon_ROH (https://haproh.readthedocs.io/en/latest/hapROH_with_contamination.html), and Schmutzi74 was used for the mtDNA-captured double-stranded library from TKH009 (Supplementary Table 2.3). The damage pattern was assessed with DamageProfiler75 v.1.1.

Double-stranded 1240k-captured sequences from TKH009 and single-stranded Twist-captured sequences from TKH001 were genotyped using Samtools v.1.3 and pileupCaller from SequenceTools v.1.4.0.2 (https://github.com/stschiff/sequenceTools). TKH001 has a total of 881,765 SNPs on the 1240k panel, and TKH009 libraries showed lower coverage of 22,484 SNPs. We merged the genotype data from TKH009 and TKH001. We then ran all the analyses throughout the manuscript (including PCA, f3 and f4 statistics, qpAdm, admixture graph, DATES and ADMIXTURE), except for admixfrog and hapROH, which, as individual analyses, use only the higher-coverage data from TKH001. To prevent potential artifacts from affecting the results, additional versions of the genotyped data for the TKH001 sample were generated. These included data from only the single-stranded library, data restricted to transversions only and data filtered using PMDtools (v.0.6)76 (Supplementary Table 2.4).

Using smartpca77 v.16000 from EIGENSOFT package v.8.0 with lsqmode enabled, PCAs were conducted on present-day individuals from Africa, Middle East and Southern Europe, genotyped on the Human Origins SNP panel28,29,30,31,32,33,34,35,36,37,38, alongside pertinent ancient groups from these regions2,8,9,35,36,39,40,41,42,43,44. All f3 and f4 statistics were calculated using the ADMIXTOOLS30 package v.5.1, using qp3pop v.435 for f3 statistics and qpDstat v.755 for f4 statistics. For the f3 outgroup test, the topology f3(outgroup; X, Takarkori) was used. When the outgroup was set as South Africa 2000 cal. bp, the ‘inbreed' option was enabled for qp3pop. On the other hand, when the outgroup was chimpanzee, the ‘outgroupmode' option was activated in qp3pop. This particular configuration was used to set a constant denominator of 0.01, as the inbuilt heterozygosity normalization in qp3pop does not function correctly when the outgroup is haploid, as in the case of the Chimpanzee. Furthermore, we used the Affymetrix Human Origins 1 Array, which comprises 13 unique SNP sets, to investigate potential sub-Saharan ancestry in the Takarkori and Taforalt populations. We conducted separate f4 statistics for Mbuti and Yoruba ascertained SNPs (Supplementary Figs. 2.21 and 2.22).

The Taforalt group was modelled as a two-way admixture between the Natufian population (left source) and variable African groups (right source) using qpAdm78 v.810 from ADMIXTOOLS package v.5.1. For every run, a consistent outgroup set was arranged, comprising Onge, Han, Papuan, Ust'-Ishim, Kostenki14, MA-1 and Iran_N. Every African group (Yoruba, Dinka, Mota, Cameroon Shum Laka, Botswana Xaro EIA, and Tanzania Zanzibar 1,300  cal. bp) was examined as a potential right source population in individual runs, while being omitted from the outgroup set (Supplementary Table 2.6). This methodology enabled the exploration of various admixture scenarios for the Taforalt group.

Neanderthal ancestry was inferred using admixfrog48 v.0.7.1 on single-stranded Takarkori and Taforalt libraries captured with the Archaic Admixture SNP panel27. The analysis included high-quality ancient shotgun sequencing data and present-day sub-Saharan genomes from the Allen Ancient Genome Diversity Project/John Templeton Ancient DNA Atlas (https://reich.hms.harvard.edu/ancient-genome-diversity-project) and the Human Genome Diversity Project32. Genomes were subsetted to positions in the Archaic Admixture SNP panel, with filters for mapping quality and base pair length. Libraries with insufficient SNP coverage or contamination were excluded (Supplementary Table 2.8). High-coverage genomes were subsetted to the positions covered by the Takarkori genome for direct comparison. The reference panel included three high-coverage Neanderthals79,80,81, one high-coverage Denisovan33 and the chimpanzee genome (panTro4) as the ancestral allele, along with Sub-Saharan African 1000 Genomes sequences82 for the African state.

The BEAST analysis of mitochondrial genomes involved 216 mtDNA sequences, aligned using MAFFT83 v.7.508 and adjusted by removing specific poly-C regions. The Hohlenstein–Stadel Neanderthal84 was the outgroup, and mtDNA sequences not in GenBank were processed using a specialized pipeline (mitoBench-ancientMT)85. The sequence consensus was determined using snpAD86 v.0.3.9, with haplogroups assigned by HaploGrep287 v.2.1.19. BEAST88 v.2.6.7 used these alignments, incorporating radiocarbon dates from Supplementary Table 2.9 as priors. Model suitability was assessed with bModelTest89 v.1.2.1, determining a combination of TIM(21) + I + G model, strict clock rate and ‘coalescent Bayesian skyline' tree prior as optimal. Eight independent analyses were conducted, combining results using LogCombiner and TreeAnnotator.

The ancestral relationship of the Takarkori population with others was investigated using the ‘find_graphs()' function from ADMIXTOOLS 2, which finds admixture graphs aligning with observed f statistics. Chimpanzee was set as an outgroup, and we included representatives from key relevant genetic clusters: ‘Morocco_Iberomaurusian', ‘Ethiopia_4500BP.SG', ‘Israel_Natufian_published' and ‘Iran_Ganj_Dareh_Neolithic'. Given the potential variability of ‘find_graphs()' outputs, 20 iterations were performed, using unique random graphs for each run. Graphs with a Z score of <|3| were selected (Supplementary Figs. 2.23–2.25). Moreover, Takarkori and Taforalt groups were treated as admixed populations, guided by previous knowledge from admixfrog results.

The DATES method49 v.753 was applied to the Takarkori group to detect Levantine admixture. A bin size of 0.001 and a fit range of 0.0045 to 1 in Morgan units were used. Admixture dates were estimated based on the decay of linkage disequilibrium between SNP pairs, with Levantine admixture modelled using sub-Saharan populations (Yoruba, Mbuti, Dinka, ancient groups from Ethiopia and Tanzania) and Eurasian populations (Natufian, PPNB, Ganj Dareh) (Supplementary Figs. 2.26.1–2.26.3).

The effective population size (Ne) for TKH001 was estimated using hapROH90 v.3.0. ROH segments longer than 4 cM were inferred using hapROH's recommended default settings, using the 1000 Genomes reference panel for comparison (Supplementary Fig. 2.27). This method analyses runs of homozygosity to estimate the population size and genetic background of the individual (Supplementary Fig. 2.28).

ADMIXTURE91 v.1.3.0 was used for unsupervised genetic clustering of global populations. Modern and ancient groups were subsetted from the HO-based dataset, converted to PLINK format, and transposed to pseudohaploid format to reduce artificial drift. Linkage disequilibrium pruning was performed using PLINK92 v.1.9 with a window size of 200 SNPs, a step size of 25 SNPs and an r2 threshold of 0.4. Five replicates for each k (2–9) were run, selecting the replicate with the highest log-likelihood.

Data visualization was performed in RStudio v.2022.12.0 + 353. The following R packages were used cowplot (v.1.1.2), ggplot (v.3.4.2), ggh4x (v.0.2.3), ggnewscale (v.0.4.8), janno (v.1.0.0), magrittr (v.2.0.3), maps (v.3.4.1), patchwork (v.1.1.2), purrr (v.1.0.1), RColorBrewer (v.1.1.3), readxl (v.1.4.1), tidyr (v.1.3.0) and tidyverse (v.1.3.2).

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

All newly reported ancient nuclear DNA data are archived in the European Nucleotide Archive (PRJEB84057).

Kuper, R. & Kröpelin, S. Climate-controlled Holocene occupation in the Sahara: motor of Africa's evolution. Science 313, 803–807 (2006).

PubMed 
    
                    Google Scholar 
                

van de Loosdrecht, M. et al. Pleistocene North African genomes link Near Eastern and sub-Saharan African human populations. Science 360, 548–552 (2018).

PubMed 
    
                    Google Scholar 
                

COHMAP Members. Climatic changes of the last 18,000 years: observations and model simulations. Science 241, 1043–1052 (1988).

MATH 
    
                    Google Scholar 
                

Ritchie, J. C., Eyles, C. H. & Haynes, C. V. Sediment and pollen evidence for an early to mid-Holocene humid period in the eastern Sahara. Nature 314, 352–355 (1985).


                    Google Scholar 
                

Tierney, J. E., Pausata, F. S. R. & deMenocal, P. B. Rainfall regimes of the Green Sahara. Sci. Adv. 3, e1601503 (2017).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Gasse, F. Hydrological changes in the African tropics since the Last Glacial Maximum. Quat. Sci. Rev. 19, 189–211 (2000).


                    Google Scholar 
                

Hoelzmann, P., Keding, B., Berke, H., Kröpelin, S. & Kruse, H.-J. Environmental change and archaeology: lake evolution and human occupation in the Eastern Sahara during the Holocene. Palaeogeogr. Palaeoclimatol. Palaeoecol. 169, 193–217 (2001).


                    Google Scholar 
                

Fregel, R. et al. Ancient genomes from North Africa evidence prehistoric migrations to the Maghreb from both the Levant and Europe. Proc. Natl Acad. Sci. USA 115, 6774–6779 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Simões, L. G. et al. Northwest African Neolithic initiated by migrants from Iberia and Levant. Nature https://doi.org/10.1038/s41586-023-06166-6 (2023).

di Lernia, S. Earliest Herders of the Central Sahara (Tadrart Acacus Mountains, Libya): a punctuated model for the emergence of pastoralism in Africa. J. World Prehist. 34, 531–594 (2021).


                    Google Scholar 
                

Smith, A. Pastoralism in Africa. Oxford Research Encyclopedia of African History https://doi.org/10.1093/acrefore/9780190277734.013.1066 (2021).

McDonald, M. M. A. The pattern of Neolithization in Dakhleh Oasis in the Eastern Sahara. Quat. Int. 410, 181–197 (2016).

MATH 
    
                    Google Scholar 
                

Vai, S. et al. Ancestral mitochondrial N lineage from the Neolithic ‘green' Sahara. Sci. Rep. 9, 3530 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Manning, K. et al. Habitat fragmentation and the sporadic spread of pastoralism in the mid-Holocene Sahara. Quat. Sci. Rev. 309, 108070 (2023).

MATH 
    
                    Google Scholar 
                

Cremaschi, M. et al. Takarkori rock shelter (SW Libya): an archive of Holocene climate and environmental changes in the central Sahara. Quat. Sci. Rev. 101, 36–60 (2014).

MATH 
    
                    Google Scholar 
                

Biagetti, S. & di Lernia, S. Holocene deposits of saharan rock shelters: the case of Takarkori and other sites from the Tadrart Acacus Mountains (Southwest Libya). Afr. Archaeol. Rev. 30, 305–338 (2013).


                    Google Scholar 
                

di Lernia, S., Massamba N'siala, I. & Mercuri, A. M. Saharan prehistoric basketry. Archaeological and archaeobotanical analysis of the early-middle Holocene assemblage from Takarkori (Acacus Mts., SW Libya). J. Archaeol. Sci. 39, 1837–1853 (2012).


                    Google Scholar 
                

Dunne, J., Mercuri, A. M., Evershed, R. P., Bruni, S. & di Lernia, S. Earliest direct evidence of plant processing in prehistoric Saharan pottery. Nat. Plants 3, 16194 (2016).

PubMed 
    
                    Google Scholar 
                

Rotunno, R., Cavorsi, L. & di Lernia, S. A Holocene ceramic sequence in the Central Sahara: pottery traditions and social dynamics seen from the Takarkori Rockshelter (SW Libya). Afr. Archaeol. Rev. 40, 647–672 (2023).


                    Google Scholar 
                

Van Neer, W. et al. Aquatic fauna from the Takarkori rock shelter reveals the Holocene central Saharan climate and palaeohydrography. PLoS ONE 15, e0228588 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Cherkinsky, A. & di Lernia, S. Bayesian approach to 14C dates for estimation of long-term archaeological sequences in arid environments: the Holocene site of Takarkori rockshelter, southwest Libya. Radiocarbon 55, 771–782 (2013).


                    Google Scholar 
                

Lernia, S. di, di Lernia, S. & Tafuri, M. A. Persistent deathplaces and mobile landmarks: the Holocene mortuary and isotopic record from Wadi Takarkori (SW Libya). J. Anthropological Archaeol. https://doi.org/10.1016/j.jaa.2012.07.002 (2013).

Dunne, J. et al. First dairying in green Saharan Africa in the fifth millennium BC. Nature 486, 390–394 (2012).

PubMed 
    MATH 
    
                    Google Scholar 
                

Tafuri, M. A., Bentley, R. A., Manzi, G. & di Lernia, S. Mobility and kinship in the prehistoric Sahara: strontium isotope analysis of Holocene human skeletons from the Acacus Mts. (southwestern Libya). J. Anthropol. Archaeol. 25, 390–402 (2006).


                    Google Scholar 
                

Rohland, N. et al. Three assays for in-solution enrichment of ancient human DNA at more than a million SNPs. Genome Res. 32, 2068–2078 (2022).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Mathieson, I. et al. Genome-wide patterns of selection in 230 ancient Eurasians. Nature 528, 499–503 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. Nature 524, 216–219 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Lazaridis, I. et al. Ancient human genomes suggest three ancestral populations for present-day Europeans. Nature 513, 409–413 (2014).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Fan, S. et al. African evolutionary history inferred from whole genome sequence data of 44 indigenous African populations. Genome Biol. 20, 82 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Patterson, N. et al. Ancient admixture in human history. Genetics 192, 1065–1093 (2012).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Bergström, A. et al. Insights into human genetic variation and population history from 929 diverse genomes. Science 367, eaay5012 (2020).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Mallick, S. et al. The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature 538, 201–206 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Meyer, M. et al. A high-coverage genome sequence from an archaic Denisovan individual. Science 338, 222–226 (2012).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Pickrell, J. K. et al. The genetic prehistory of southern Africa. Nat. Commun. 3, 1143 (2012).

PubMed 
    MATH 
    
                    Google Scholar 
                

Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Lazaridis, I. et al. Genomic insights into the origin of farming in the ancient Near East. Nature 536, 419–424 (2016).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Pickrell, J. K. et al. Ancient west Eurasian ancestry in southern and eastern Africa. Proc. Natl Acad. Sci. USA 111, 2632–2637 (2014).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

D'Atanasio, E. et al. The genomic echoes of the last Green Sahara on the Fulani and Sahelian people. Curr. Biol. 33, 5495–5504 (2023).

PubMed 
    MATH 
    
                    Google Scholar 
                

Wang, K. et al. Ancient genomes reveal complex patterns of population movement, interaction, and replacement in sub-Saharan Africa. Sci. Adv. 6, eaaz0183 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Lipson, M. et al. Ancient DNA and deep population structure in sub-Saharan African foragers. Nature 603, 290–296 (2022).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Llorente, M. G. et al. Ancient Ethiopian genome reveals extensive Eurasian admixture in Eastern Africa. Science 350, 820–822 (2015).

MATH 
    
                    Google Scholar 
                

Harney, É. et al. Ancient DNA from Chalcolithic Israel reveals the role of population mixture in cultural transformation. Nat. Commun. 9, 3336 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Prendergast, M. E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science 365, eaaw6275 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Schlebusch, C. M. et al. Southern African ancient genomes estimate modern human divergence to 350,000 to 260,000 years ago. Science 358, 652–655 (2017).

PubMed 
    
                    Google Scholar 
                

Paris, F. in The Origins and Development of African Livestock: Archaeology, Genetics, Linguistics, and Ethnography (ed. MacDonald, K. B. R.) 111–126 (UCL Press, 2000).

Vernet, R. in Droughts, Food and Culture 47–63 (Kluwer Academic, 2006).

Prüfer, K. et al. A genome sequence from a modern human skull over 45,000 years old from Zlatý kůň in Czechia. Nat. Ecol. Evol. 5, 820–825 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Peter, B. M. 100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains. Preprint at bioRxiv https://doi.org/10.1101/2020.03.13.990523 (2020).

Narasimhan, V. M. et al. The formation of human populations in South and Central Asia. Science 365, eaat7487 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Maier, R. et al. On the limits of fitting complex models of population history to f-statistics. eLife 12, e85492 (2023).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Garcea, E. A. A. An alternative way towards food production: the perspective from the Libyan Sahara. J. World Prehist. 18, 107–154 (2004).

MATH 
    
                    Google Scholar 
                

Mercuri, A. M., Fornaciari, R., Gallinaro, M., Vanin, S. & di Lernia, S. Plant behaviour from human imprints and the cultivation of wild cereals in Holocene Sahara. Nat. Plants 4, 71–81 (2018).

PubMed 
    
                    Google Scholar 
                

di Lernia, S. Dismantling dung: delayed use of food resources among Early Holocene foragers of the Libyan Sahara. J. Anthropol. Archaeol. 20, 408–441 (2001).


                    Google Scholar 
                

Rotunno, R., Mercuri, A. M., Florenzano, A., Zerboni, A. & di Lernia, S. Coprolites from rock shelters: hunter-gatherers ‘herding' barbary sheep in the Early Holocene Sahara. J. Afr. Archaeol. 17, 76–94 (2019).


                    Google Scholar 
                

Kröpelin, S. et al. Climate-driven ecosystem succession in the Sahara: the past 6000 years. Science 320, 765–768 (2008).

PubMed 
    MATH 
    
                    Google Scholar 
                

Hoelzmann, P. et al. in Past Climate Variability through Europe and Africa (eds Battarbee, R. W. et al.) 219–256 (Springer, 2004).

Di Lernia, S. Saharan Hunter-Gatherers: Specialization and Diversification in Holocene Southwestern Libya (Routledge, 2022).

Drake, N. A., Blench, R. M., Armitage, S. J., Bristow, C. S. & White, K. H. Ancient watercourses and biogeography of the Sahara explain the peopling of the desert. Proc. Natl Acad. Sci. USA 108, 458–462 (2011).

PubMed 
    
                    Google Scholar 
                

Peter, B. M., Petkova, D. & Novembre, J. Genetic landscapes reveal how human genetic diversity aligns with geography. Mol. Biol. Evol. 37, 943–951 (2020).

PubMed 
    
                    Google Scholar 
                

Cheddadi, R. et al. Early Holocene greening of the Sahara requires Mediterranean winter rainfall. Proc. Natl Acad. Sci. USA 118, e2024898118 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Profico, A. et al. Medical imaging as a taphonomic tool: the naturally-mummified bodies from Takarkori rock shelter (Tadrart Acacus, SW Libya, 6100-5600 uncal BP). J. Cult. Herit. Manage. Sust. Dev. 10, 144–156 (2019).


                    Google Scholar 
                

Stloukal, M. & Hanáková, H. Die länge der längsknochen Altslawischer bevölkerungen unter besonderer berücksichtigung von wachstumsfragen. Homo 29, 53–69 (1978).

France, D. L. Lab Manual and Workbook for Physical Anthropology (West, 1988).

Ubelaker, D. H. Human Skeletal Remains. Excavation, Analysis, Interpretation (Taraxacum, 1989)

Buikstra, J. E. & Ubelaker, D. H. Standards for Data Collection from Human Skeletal Remains (Arkansas Archaeological Survey, 1994).

Todd, T. W. Age changes in the pubic bone. I. The male white pubis. Am. J. Phys. Anthropol. 3, 285–334 (1920).

Meindl, R. S. & Lovejoy, C. O. Ectocranial suture closure: a revised method for the determination of skeletal age at death based on the lateral-anterior sutures. Am. J. Phys. Anthropol. 68, 57–66 (1985).

Lovejoy, C. O. Dental wear in the Libben population: its functional pattern and role in the determination of adult skeletal age at death. Am. J. Phys. Anthropol. 68, 47–56 (1985).

Dabney, J. et al. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. Proc. Natl Acad. Sci. USA 110, 15758–15763 (2013).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Schubert, M., Lindgreen, S. & Orlando, L. AdapterRemoval v2: rapid adapter trimming, identification, and read merging. BMC Res. Notes 9, 88 (2016).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Peltzer, A. et al. EAGER: efficient ancient genome reconstruction. Genome Biol. 17, 60 (2016).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Peyrégne, S. & Peter, B. M. AuthentiCT: a model of ancient DNA damage to estimate the proportion of present-day DNA contamination. Genome Biol. 21, 246 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Renaud, G., Slon, V., Duggan, A. T. & Kelso, J. Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA. Genome Biol. 16, 224 (2015).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Neukamm, J., Peltzer, A. & Nieselt, K. DamageProfiler: fast damage pattern calculation for ancient DNA. Bioinformatics 37, 3652–3653 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Skoglund, P. et al. Separating endogenous ancient DNA from modern day contamination in a Siberian Neandertal. Proc. Natl Acad. Sci. USA 111, 2229–2234 (2014).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Patterson, N., Price, A. L. & Reich, D. Population structure and eigenanalysis. PLoS Genet. 2, e190 (2006).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Haak, W. et al. Massive migration from the steppe was a source for Indo-European languages in Europe. Nature 522, 207–211 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Prüfer, K. et al. The complete genome sequence of a Neanderthal from the Altai Mountains. Nature 505, 43–49 (2014).

PubMed 
    MATH 
    
                    Google Scholar 
                

Prüfer, K. et al. A high-coverage Neandertal genome from Vindija Cave in Croatia. Science 358, 655–658 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Mafessoni, F. et al. A high-coverage Neandertal genome from Chagyrskaya Cave. Proc. Natl Acad. Sci. USA 117, 15132–15136 (2020).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

1000 Genomes Project Consortium. A global reference for human genetic variation. Nature 526, 68–74 (2015).


                    Google Scholar 
                

Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. 30, 772–780 (2013).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Posth, C. et al. Deeply divergent archaic mitochondrial genome provides lower time boundary for African gene flow into Neanderthals. Nat. Commun. 8, 16046 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Pugach, I. et al. Ancient DNA from Guam and the peopling of the Pacific. Proc. Natl Acad. Sci. USA 118, e2022112118 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Prüfer, K. snpAD: an ancient DNA genotype caller. Bioinformatics 34, 4165–4171 (2018).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic Acids Res. 44, W58–W63 (2016).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Bouckaert, R. et al. BEAST 2.5: an advanced software platform for Bayesian evolutionary analysis. PLoS Comput. Biol. 15, e1006650 (2019).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Bouckaert, R. R. & Drummond, A. J. bModelTest: Bayesian phylogenetic site model averaging and model comparison. BMC Evol. Biol. 17, 42 (2017).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Ringbauer, H., Novembre, J. & Steinrücken, M. Parental relatedness through time revealed by runs of homozygosity in ancient DNA. Nat. Commun. 12, 5425 (2021).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Alexander, D. H., Novembre, J. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 19, 1655–1664 (2009).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Download references

We thank the members of the Department of Antiquities in Tripoli for granting S.d.L. the necessary permits for the excavation, exportation, dating and analysis of human remains; M. Burri, A. Wissgott, R. Radzeviciute, M. Meyer and J. Zorn (all currently or formerly affiliated with MPI-EVA) for their substantial roles in processing the samples in the laboratory. Our research has benefited from the insights and advice of C. Jeong, E. Scerri, S. Oliveira, V. Villalba, E. Skourtanioti, S. Goldstein and M. Hajdinjak (all currently or formerly affiliated with MPI-EVA and MPI-SHH), as well as all of the members of the Population Genetics working group. The data were produced by the Ancient DNA Core Unit of the Max Planck Institute for Evolutionary Anthropology, which is funded by the Max Planck Society. We acknowledge the staff at the Multimedia Department at MPI-EVA for their expert graphical assistance in enhancing the visual aspects of our publication. This study was funded by the Max Planck Society and the Max Planck-Harvard Research Center for the Archaeoscience of the Ancient Mediterranean (MHAAM).

Open access funding provided by Max Planck Society.

Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany

Nada Salem, Marieke S. van de Loosdrecht, Arev Pelin Sümer, Alexander Hübner, Benjamin Peter, Raffaela A. Bianco, Abdeljalil Bouzouggar, Kay Prüfer, Harald Ringbauer & Johannes Krause

Max Planck−Harvard Research Center for the Archaeoscience of the Ancient Mediterranean (MHAAM), Leipzig, Germany

Nada Salem & Johannes Krause

Biosystematics Group, Wageningen University, Wageningen, The Netherlands

Marieke S. van de Loosdrecht

Department of Biology, University of Florence, Florence, Italy

Stefania Vai, Martina Lari, Alessandra Modi & David Caramelli

The Department of Antiquities (DOA), Tripoli, Libya

Mohamed Faraj Mohamed Al-Faloos & Mustafa Turjman

Institut National des Sciences de l'Archéologie et du Patrimoine, Origin and Evolution of Homo sapiens Cultures, Rabat, Morocco

Abdeljalil Bouzouggar

Department of Environmental Biology, Sapienza University of Rome, Rome, Italy

Mary Anne Tafuri & Giorgio Manzi

Department of Ancient World Studies, Sapienza University of Rome, Rome, Italy

Rocco Rotunno & Savino di Lernia

School of Geography, Archaeology and Environmental Studies (GAES), University of Witwatersrand, Johannesburg, South Africa

Savino di Lernia

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

J.K. and S.d.L. designed the research (Conceptualization), with N.S. leading the formal analysis assisted by M.S.v.d.L., A.P.S., A.H., H.R., B.P. and K.P.; S.d.L., D.C., M.A.T. and G.M. provided the archaeological materials and conducted excavations (resources and investigation). Laboratory work (investigation) was headed by S.V. and R.A.B. with help from R.R., M.L. and A.M. Data curation and visualization were primarily managed by N.S., with M.S.v.d.L. contributing. M.F.M.A.-F. and M.T. managed project administration and handled permit acquisitions (project administration), and A.B. contributed to embedding the findings into the broader archaeological framework (integration into archaeological framework). N.S. drafted the manuscript, with all of the authors reviewing and editing. K.P., H.R., J.K. and S.d.L. provided supervision, and J.K. secured funding. Contributions are recognized using the CRediT Taxonomy labels.

Correspondence to
                Nada Salem, Harald Ringbauer, David Caramelli, Savino di Lernia or Johannes Krause.

The authors declare no competing interests.

Nature thanks Kendra Sirak and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

A) with projections of key ancient groups from these regions. B) Geographic locations of ancient genomes from Africa and Near East included in our analysis. EN: Early Neolithic, LN: Late Neolithic, ChL: Chalcolithic, LSA: Late Stone Age, IA: Iron Age.

A) with Takarkori individuals projected and highlighted in black squares. B) Geographic locations of Takarkori individuals and relevant present-day populations included in our analysis.

Within the shared drift of Takarkori group, the highest genetic drift is exhibited with Taforalt-related groups, OUB and IAM. The colour scheme for populations in the legend follows that of PCA Fig. 1. The error bars represent three standard errors.

The Taforalt and later Epipaleolithic and Neolithic groups from Morocco are included for comparison due to their highest genetic drift with Takarkori. The Takarkori group shows the second highest affinity with FulaniA after Taforalt-related groups. The colour and shape scheme for populations in the legend follows that of the PCA in Extended Data Fig. 1. The error bars represent three standard errors.

All positive f4 values underscore shared genetic drift between Takarkori and Taforalt that surpasses other potential affinities tested here. The error bars represent three standard errors.

The phylogenetic tree is constructed for the Takarkori samples TKH001 and TKH009, combined with 209 published complete genomes from ancient and modern samples. The major mitochondrial lineages and sub-lineages for the N macrohaplogroup are differentiated by distinct colours.

Takarkori exhibits a notably higher affinity to Taforalt, with an f4 value approximately 3.5 times higher than the next closest African group. The error bars represent three standard errors.

Results of modelling the Taforalt group as a two-way admixture between Natufian and various African populations. Takarkori yielded the best model fit with a P-value > 0.05, indicating the sufficiency of the two-way admixture model for Taforalt. In contrast, the P-values for the other models were all <2.84 × 10−34. The error bars represent the standard error of the ancestry proportion estimates, calculated using 5 cM block jackknifing.

The results suggest a stronger connection between Takarkori and the OoA ancestry compared to sub-Saharan Africans. The error bars represent three standard errors.

A) Detected archaic ancestry fragments in Takarkori and Taforalt inferred by admixfrog. Dark and light blue regions are homozygous and heterozygous Neanderthal fragments, respectively. Grey fragments indicate African ancestry and bar height is proportional to the posterior probability of ancestry. B) The longest archaic fragment in Takarkori is on chromosome 1 compared to the reference panel. The y-axis refers to the frequency of the alternative allele in the reference population (Africans or Neanderthals in this case) at the given position, and the x-axis stands for the SNP positions.

Supplementary Notes 1–3, including Supplementary Figs., Tables and References – see Table of Contents for details.

Supplementary Data 1–6.

Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/.

Reprints and permissions

Salem, N., van de Loosdrecht, M.S., Sümer, A.P. et al. Ancient DNA from the Green Sahara reveals ancestral North African lineage.
                    Nature  (2025). https://doi.org/10.1038/s41586-025-08793-7

Download citation

Received: 04 January 2024

Accepted: 14 February 2025

Published: 02 April 2025

DOI: https://doi.org/10.1038/s41586-025-08793-7

Anyone you share the following link with will be able to read this content:

Sorry, a shareable link is not currently available for this article.




                            Provided by the Springer Nature SharedIt content-sharing initiative
                        

Advertisement



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited

Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08726-4'>Millimetre-scale bioresorbable optoelectronic systems for electrotherapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 15:10:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.

Advertisement


Nature
volume 640, pages 77–86 (2025)Cite this article


Metrics details

Temporary pacemakers are essential for the care of patients with short-lived bradycardia in post-operative and other settings1,2,3,4. Conventional devices require invasive open-heart surgery or less invasive endovascular surgery, both of which are challenging for paediatric and adult patients5,6,7,8. Other complications9,10,11 include risks of infections, lacerations and perforations of the myocardium, and of displacements of external power supplies and control systems. Here we introduce a millimetre-scale bioresorbable optoelectronic system with an onboard power supply and a wireless, optical control mechanism with generalized capabilities in electrotherapy and specific application opportunities in temporary cardiac pacing. The extremely small sizes of these devices enable minimally invasive implantation, including percutaneous injection and endovascular delivery. Experimental studies demonstrate effective pacing in mouse, rat, porcine, canine and human cardiac models at both single-site and multi-site locations. Pairing with a skin-interfaced wireless device allows autonomous, closed-loop operation upon detection of arrhythmias. Further work illustrates opportunities in combining these miniaturized devices with other medical implants, with an example of arrays of pacemakers for individual or collective use on the frames of transcatheter aortic valve replacement systems, to provide unique solutions that address risks for atrioventricular block following surgeries. This base technology can be readily adapted for a broad range of additional applications in electrotherapy, such as nerve and bone regeneration, wound therapy and pain management.

This is a preview of subscription content, access via your institution


        Access Nature and 54 other Nature Portfolio journals
      


        Get Nature+, our best-value online-access subscription
      


$29.99 / 30 days


cancel any time

Subscribe to this journal


        Receive 51 print issues and online access
      

$199.00 per year

only $3.90 per issue

Buy this article

Prices may be subject to local taxes which are calculated during checkout

The data supporting the results of this study are present in the paper and Supplementary Information. Source data are provided with this paper.

The code for connecting to the device via BLE, recording and analysing ECG data in real time, and configuring the pacing parameters in a closed-loop system is available on Code Ocean at https://codeocean.com/capsule/9406347/tree/v1 (ref. 49).

Choi, Y. S. et al. Fully implantable and bioresorbable cardiac pacemakers without leads or batteries. Nat. Biotechnol. 39, 1228–1238 (2021).

CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Zhang, Y. et al. Advances in bioresorbable materials and electronics. Chem. Rev. 123, 11722–11773 (2023).

CAS 
    PubMed 
    
                    Google Scholar 
                

Choi, Y. S. et al. A transient, closed-loop network of wireless, body-integrated devices for autonomous electrotherapy. Science 376, 1006–1012 (2022).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Lumia, F. J. & Rios, J. C. Temporary transvenous pacemaker therapy: an analysis of complications. Chest 64, 604–608 (1973).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Wood, M. A. & Ellenbogen, K. A. Cardiac pacemakers from the patient's perspective. Circulation 105, 2136–2138 (2002).

PubMed 
    MATH 
    
                    Google Scholar 
                

Bar-Cohen, Y. et al. Minimally invasive implantation of a micropacemaker into the pericardial space. Circ. Arrhythm. Electrophysiol. 11, e006307 (2018).

PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Zhao, J. et al. Permanent epicardial pacing in neonates and infants less than 1 year old: 12-year experience at a single center. Transl. Pediatr. 11, 825–833 (2022).

PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Wildbolz, M., Dave, H., Weber, R., Gass, M. & Balmer, C. Pacemaker implantation in neonates and infants: favorable outcomes with epicardial pacing systems. Pediatr. Cardiol. 41, 910–917 (2020).

PubMed 
    
                    Google Scholar 
                

Wilhelm, M. J. et al. Cardiac pacemaker infection: surgical management with and without extracorporeal circulation. Ann. Thorac. Surg. 64, 1707–1712 (1997).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Donovan, K. D. & Lee, K. Y. Indications for and complications of temporary transvenous cardiac pacing. Anaesth. Intensive Care 13, 63–70 (1985).

CAS 
    PubMed 
    
                    Google Scholar 
                

BRAUN, M. U. et al. Percutaneous lead implantation connected to an external device in stimulation-dependent patients with systemic infection—a prospective and controlled study. Pacing Clin. Electrophysiol. 29, 875–879 (2006).

ADS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Ouyang, H. et al. Symbiotic cardiac pacemaker. Nat. Commun. 10, 1821 (2019).

ADS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Lyu, H. et al. Synchronized biventricular heart pacing in a closed-chest porcine model based on wirelessly powered leadless pacemakers. Sci. Rep. 10, 2067 (2020).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Ho, J. S. et al. Wireless power transfer to deep-tissue microimplants. Proc. Natl Acad. Sci. USA 111, 7974–7979 (2014).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Wang, S. et al. A self-assembled implantable microtubular pacemaker for wireless cardiac electrotherapy. Sci. Adv. 9, eadj0540 (2023).

CAS 
    PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Prominski, A. et al. Porosity-based heterojunctions enable leadless optoelectronic modulation of tissues. Nat. Mater. 21, 647–655 (2022).

ADS 
    CAS 
    PubMed 
    
                    Google Scholar 
                

Liu, Z. et al. Photoelectric cardiac pacing by flexible and degradable amorphous Si radial junction stimulators. Adv. Healthc. Mater. 9, 1901342 (2020).

CAS 
    
                    Google Scholar 
                

Wang, L. et al. A fully biodegradable and self-electrified device for neuroregenerative medicine. Sci. Adv. 6, eabc6686 (2020).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    
                    Google Scholar 
                

Zhang, Y. et al. Self-powered, light-controlled, bioresorbable platforms for programmed drug delivery. Proc. Natl. Acad. Sci. USA 120, e2217734120 (2023).

Huang, I. et al. High performance dual-electrolyte magnesium–iodine batteries that can harmlessly resorb in the environment or in the body. Energy Environ. Sci. 15, 4095–4108 (2022).

CAS 
    MATH 
    
                    Google Scholar 
                

Won, S. M. et al. Natural wax for transient electronics. Adv. Funct. Mater. 28, 1801819 (2018).

MATH 
    
                    Google Scholar 
                

Choi, Y. S. et al. Biodegradable polyanhydrides as encapsulation layers for transient electronics. Adv. Funct. Mater. 30, 2000941 (2020).

CAS 
    
                    Google Scholar 
                

Song, G. Control of biodegradation of biocompatable magnesium alloys. Corros. Sci. 49, 1696–1701 (2007).

CAS 
    MATH 
    
                    Google Scholar 
                

Schauer, A. et al. Biocompatibility and degradation behavior of molybdenum in an in vivo rat model. Materials 14, 7776 (2021).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Yin, L. et al. Dissolvable metals for transient electronics. Adv. Funct. Mater. 24, 645–658 (2014).

CAS 
    MATH 
    
                    Google Scholar 
                

Yu, K. J. et al. Bioresorbable silicon electronics for transient spatiotemporal mapping of electrical activity from the cerebral cortex. Nat. Mater. 15, 782–791 (2016).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Kang, S.-K. et al. Bioresorbable silicon electronic sensors for the brain. Nature 530, 71–76 (2016).

ADS 
    CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Li, G. et al. Flexible transient phototransistors by use of wafer‐compatible transferred silicon nanomembranes. Small 14, e1802985 (2018).

ADS 
    PubMed 
    
                    Google Scholar 
                

Li, G. et al. Silicon nanomembrane phototransistor flipped with multifunctional sensors toward smart digital dust. Sci. Adv. 6, eaaz6511 (2020).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    
                    Google Scholar 
                

López Ayerbe, J. et al. Temporary pacemakers: current use and complications. Rev. Esp. Cardiol. Engl. Ed. 57, 1045–1052 (2004).

MATH 
    
                    Google Scholar 
                

Yu, L., Nina-Paravecino, F., Kaeli, D. & Fang, Q. Scalable and massively parallel Monte Carlo photon transport simulations for heterogeneous computing platforms. J. Biomed. Opt. 23, 1 (2018).

PubMed 
    
                    Google Scholar 
                

Fang, Q. & Boas, D. A. Monte Carlo simulation of photon migration in 3D turbid media accelerated by graphics processing units. Opt. Express 17, 20178 (2009).

ADS 
    CAS 
    PubMed 
    
                    Google Scholar 
                

Taroni, P., Pifferi, A., Torricelli, A., Comelli, D. & Cubeddu, R. In vivo absorption and scattering spectroscopy of biological tissues. Photochem. Photobiol. Sci. 2, 124–129 (2003).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Khan, R., Gul, B., Khan, S., Nisar, H. & Ahmad, I. Refractive index of biological tissues: review, measurement techniques, and applications. Photodiagnosis Photodyn. Ther. 33, 102192 (2021).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Green, M. A. & Keevers, M. J. Optical properties of intrinsic silicon at 300 K. Prog. Photovoltaics Res. Appl. 3, 189–192 (1995).

CAS 
    
                    Google Scholar 
                

Firbank, M., Hiraoka, M., Essenpreis, M. & Delpy, D. T. Measurement of the optical properties of the skull in the wavelength range 650–950 nm. Phys. Med. Biol. 38, 503–510 (1993).

CAS 
    PubMed 
    
                    Google Scholar 
                

Rahko, P. S. Evaluation of the skin-to-heart distance in the standing adult by two-dimensional echocardiography. J. Am. Soc. Echocardiogr. 21, 761–764 (2008).

PubMed 
    MATH 
    
                    Google Scholar 
                

Chen, R. et al. Deep brain optogenetics without intracranial surgery. Nat. Biotechnol. 39, 161–164 (2021).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Yin, R. T. et al. Open thoracic surgical implantation of cardiac pacemakers in rats. Nat. Protoc. 18, 374–395 (2023).

CAS 
    PubMed 
    MATH 
    
                    Google Scholar 
                

Yang, Q. et al. Photocurable bioresorbable adhesives as functional interfaces between flexible bioelectronic devices and soft biological tissues. Nat. Mater. 20, 1559–1570 (2021).

ADS 
    CAS 
    PubMed 
    PubMed Central 
    MATH 
    
                    Google Scholar 
                

Shea, J. B. & Sweeney, M. O. Cardiac resynchronization therapy a patient's guide. Circulation 108, e64–e66 (2003).

Connolly, S. J., Kerr, C., Gent, M. & Yusuf, S. Dual-chamber versus ventricular pacing. Circulation 94, 578–583 (1996).

CAS 
    PubMed 
    
                    Google Scholar 
                

Rodés-Cabau, J., Muntané-Carol, G. & Philippon, F. Managing conduction disturbances after TAVR: toward a tailored strategy. JACC Cardiovasc. Interv. 14, 992–994 (2021).

PubMed 
    MATH 
    
                    Google Scholar 
                

Urena, M. & Rodés-Cabau, J. Conduction abnormalities: the true Achilles' heel of transcatheter aortic valve replacement? JACC Cardiovasc. Interv. 9, 2217–2219 (2016).

PubMed 
    
                    Google Scholar 
                

Pagnesi, M. et al. Incidence, predictors, and prognostic impact of new permanent pacemaker implantation after TAVR with self-expanding valves. JACC Cardiovasc. Interv. 16, 2004–2017 (2023).

PubMed 
    MATH 
    
                    Google Scholar 
                

Reiter, C. et al. Delayed total atrioventricular block after transcatheter aortic valve replacement assessed by implantable loop recorders. JACC Cardiovasc. Interv. 14, 2723–2732 (2021).

PubMed 
    
                    Google Scholar 
                

Muntané-Carol, G. et al. Ambulatory electrocardiographic monitoring following minimalist transcatheter aortic valve replacement. JACC Cardiovasc. Interv. 14, 2711–2722 (2021).

PubMed 
    
                    Google Scholar 
                

Krishnaswamy, A. et al. Feasibility and safety of same-day discharge following transfemoral transcatheter aortic valve replacement. JACC Cardiovasc. Interv. 15, 575–589 (2022).

PubMed 
    MATH 
    
                    Google Scholar 
                

Millimetre-scale, bioresorbable optoelectronic systems for minimally invasive electrotherapy. Code Ocean https://codeocean.com/capsule/9406347/tree/v1 (2025).

Download references

We acknowledge support from the Querrey Simpson Institute for Bioelectronics, the Leducq Foundation grant ‘Bioelectronics for Neurocardiology' and the NIH grant (NIH R01 HL141470). Y.Z. acknowledges support from the National University of Singapore start-up grant and the AHA's Second Century Early Faculty Independence Award (grant: https://doi.org/10.58275/AHA.23SCEFIA1154076.pc.gr.173925). J. Gong and Z.M. acknowledge the support from AFOSR (grant number FA9550-21-1-0081). We thank E. Dempsey, Q. Ma, N. Ghoreishi-Haack, I. Stepien and S. Han for the help in the biocompatibility study and animal experiment. This work made use of the NUFAB facility of Northwestern University's NUANCE Center, which has received support from the SHyNE Resource (NSF ECCS-2025633), the IIN and Northwestern's MRSEC programme (NSF DMR-1720139). This work was supported by the Developmental Therapeutics Core and the Center for Advanced Molecular Imaging (RRID:SCR_021192) at Northwestern University and the Robert H. Lurie Comprehensive Cancer Center support grant (NCI P30 CA060553).

These authors contributed equally: Yamin Zhang, Eric Rytkin, Liangsong Zeng, Jong Uk Kim, Lichao Tang, Haohui Zhang

Center for Bio-Integrated Electronics, Northwestern University, Evanston, IL, USA

Yamin Zhang, Jong Uk Kim, Seung Gi Seo, Jianyu Gu, Tianyu Yang, Naijia Liu, Wei Ouyang & John A. Rogers

Querrey Simpson Institute for Bioelectronics, Northwestern University, Evanston, IL, USA

Yamin Zhang, Liangsong Zeng, Jong Uk Kim, Kaiyu Zhao, Yue Wang, Li Ding, Xinyue Lu, Elena Aprea, Gengming Jiang, Seung Gi Seo, Jin Wang, Jianyu Gu, Fei Liu, Tianyu Yang, Naijia Liu, Yinsheng Lu, Claire Hoepfner, Alex Hou, Rachel Nolander, Wei Ouyang, Igor R. Efimov & John A. Rogers

Department of Chemical and Biomolecular Engineering, National University of Singapore, Singapore, Singapore

Yamin Zhang & Lichao Tang

Department of Biomedical Engineering, Northwestern University, Evanston, IL, USA

Eric Rytkin, Yue Wang, Anastasia Lantsova, Altynai Melisova, Alex Hou, Rachel Nolander, Igor R. Efimov & John A. Rogers

Department of Mechanical Engineering, Northwestern University, Evanston, IL, USA

Liangsong Zeng, Shupeng Li, Yonggang Huang & John A. Rogers

Department of Civil and Environmental Engineering, Northwestern University, Evanston, IL, USA

Haohui Zhang & Zengyao Lv

Feinberg Cardiovascular and Renal Research Institute, Northwestern University Feinberg School of Medicine, Chicago, IL, USA

Aleksei Mikhailov, Anna Pfenniger, Andrey Ardashev, Rishi K. Arora & Igor R. Efimov

Department of Materials Science and Engineering, Northwestern University, Evanston, IL, USA

Kaiyu Zhao, Jiayang Liu, Fei Liu, Yat Fung Larry Li, Nathan S. Purwanto, Yinsheng Lu, John M. Torkelson & John A. Rogers

Department of Cardiovascular Surgery, Union Hospital, Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China

Li Ding

The BioRobotics Institute and Department of Excellence in Robotics and AI, Scuola Superiore Sant'Anna, Pisa, Italy

Elena Aprea

Department of Chemical and Biological Engineering, Northwestern University, Evanston, IL, USA

Tong Wang & John M. Torkelson

Alnylam Pharmaceuticals Inc, Cambridge, MA, USA

Keith Bailey

Center for Comparative Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USA

Amy Burrell

Department of Electrical and Computer Engineering, Northwestern University, Evanston, IL, USA

Yue Ying

Department of Electrical and Computer Engineering, University of Wisconsin-Madison, Madison, WI, USA

Jiarui Gong & Zhenqiang Ma

Department of Electronic Engineering, Incheon National University, Incheon, Republic of Korea

Jinheon Jeong & Sung Hun Jin

Department of Chemical Engineering, Dankook University, Yongin, Republic of Korea

Junhwan Choi

Department of Applied Physical Sciences, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA

Wubin Bai

Thayer School of Engineering, Dartmouth College, Hanover, NH, USA

Wei Ouyang

The University of Chicago Medicine, University of Chicago, Chicago, IL, USA

Rishi K. Arora

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

You can also search for this author inPubMed Google Scholar

Y.Z. and J.A.R. initiated and conceived the self-powered, light-controlled pacing concept. Y.Z., E.R., I.R.E. and J.A.R. designed the studies and analysed the results. Y.Z., L.Z., K.Z., X.L., A.L., G.J., J.L., F.L., Y.F.L.L., Y.L., C.H., A.H. and R.N. fabricated and characterized the pacemakers. E.R., L.T., A. Mikhailov, L.D., A.B., A.P., A.A. and A. Melisova conducted animal surgeries. Y.Z., E.R., L.Z., L.T., A. Mikhailov, L.D., J.W., A.B., A.P. and W.O., performed in vivo and ex vivo cardiac pacing experiments. W.O., Y.W., J. Gu, T.Y., Y.Y. and Y.L. developed closed-loop and optical control systems. J.U.K., S.G.S., J. Gong, J.J., J.C., S.H.J. and Z.M. designed and fabricated phototransistors. H.Z., S.L., Z.L. and Y.H. performed computational simulations. E.A. and W.B. fabricated bioresorbable optical filters. T.W., N.S.P. and J.M.T. developed and synthesized the hydrogels. L.T., L.D. and K.B. evaluated the biocompatibility. Y.Z., E.R., L.Z., J.U.K., L.T., A. Mikhailov., K.Z., X.L., Y.W., H.Z., A.L., E.A., G.J., S.L., S.G.S., K.B., N.L., W.O., R.K.A., I.R.E. and J.A.R. discussed and interpreted the data. Y.Z. and J.A.R. prepared figures and wrote the paper, with input from E.R., W.O., A. Mikhailov, R.K.A. and I.R.E. In addition, J.U.K., L.Z., L.T., Y.W., H.Z., S.L. and J. Gu. assisted with the preparation of figures and text. Y.Z., L.Z., L.T., H.Z., L.D., W.O., I.R.E. and J.A.R. revised the paper. Y.Z., E.R., L.Z., J.U.K., L.T. and H.Z. contributed equally to this work.

Correspondence to
                Yamin Zhang, Yonggang Huang, Wei Ouyang, Rishi K. Arora, Igor R. Efimov or John A. Rogers.

The authors declare no competing interests.

Nature thanks Gábor Duray, Hossam Haick and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available.

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

a, Comparisons between conventional pacemakers with leads, leadless pacemakers, bioresorbable pacemakers, and the pacemaker reported here. b, Table showing details of previously reported leadless pacemakers and the pacemaker reported here. Illustration of the pacemaker with leads in a was created with BioRender.com (https://biorender.com). Bioresorbable pacemaker in a adapted from ref. 1, Springer Nature America, Inc.

a,b, Characteristic curves of the phototransistors under various light intensities emitted from a NIR LED (850 nm, a) and a red LED (650 nm, b).

a, EIS of an agarose gel and chicken tissue. b, The output currents of the pacemaker over days. c, Output currents of the pacemaker over days under pulsed illumination.

a, Photograph showing a pacemaker placed on the surface of a mouse heart. b, ECG results before and during mouse heart pacing. c, Strength-duration curve when pacing at 480 bpm. n =  3 biologically independent animals.

a, Emission spectra for LEDs 1 and 2. b, Transmission curves for filters 1 and 2. c, Transmitted light intensities as a function of incident intensities from LEDs 1 and (2) for filters 1 and 2.

Supplementary Methods, Notes 1–11, Tables 1–3, Figs. 1–26 and References.

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

Reprints and permissions

Zhang, Y., Rytkin, E., Zeng, L. et al. Millimetre-scale bioresorbable optoelectronic systems for electrotherapy.
                    Nature 640, 77–86 (2025). https://doi.org/10.1038/s41586-025-08726-4

Download citation

Received: 14 February 2024

Accepted: 31 January 2025

Published: 02 April 2025

Issue Date: 03 April 2025

DOI: https://doi.org/10.1038/s41586-025-08726-4

Anyone you share the following link with will be able to read this content:

Sorry, a shareable link is not currently available for this article.




                            Provided by the Springer Nature SharedIt content-sharing initiative
                        

Advertisement



                    Nature (Nature)
                

ISSN 1476-4687 (online)
    

ISSN 0028-0836 (print)
    


© 2025 Springer Nature Limited

Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.etnownews.com/exams-results/gujarat-board-hsc-result-2025-date-when-is-gseb-class-12-science-result-article-151331967'>Gujarat Board HSC Result 2025 Date: When is GSEB Class 12 Science result?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.etnownews.com', 'title': 'ET Now'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-02 03:17:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>

Live TV

Business News

exams results

Updated Apr 2, 2025 08:47 IST

GSEB HSC Science Result 2025 date, links. Image: PTI/ ET Now Digital. 

Get the latest Business News,  Stock Markets, IPO Companies News, Breaking News Events on ET Now. Watch all the big and small   Share Market News  on  ET Now Live TV.

NIOS Admit Card 2025: 10th and 12th admit cards OUT! Download from nios.ac.in | GUIDE

How Vidya Devi Jindal School prepare students for CBSE & JEE? Principal Naina Dhillon shares key insights | EXCLUSIVE

JEE Main 2025 Paper Analysis: How was JEE shift one day one paper? Check candidates, experts' reactions

Karnataka KSEAB PUC 1 Result 2025 announced; How to check PUC 1 result 2025 at karresults.nic.in

NIOS 10th, 12th admit card 2025 OUT at sdmis.nios.ac.in; Steps to download NIOS hall ticket PDF

Viewers' Mutual Fund Queries Answered | Investment Ideas With Pankaj Mathpal | The Money Show

Investment Advice For You | Top Financial Planners, Top Advice | The Money Show | Shweta Jain

Closing Trades: Investors Strategy Amid Volatile Week | Nifty Regains Losses But Is It Enough?

Samara-Backed Agro Tech Buys Del Monte| Consumption, Pharma Offer Growth Potential? |Startup Central

Mutual Fund Queries Answered For Viewers | Investment Ideas With Hemant Rustagi | The Money Show

PGIM India Smallcap Fund In Focus | Decoding The Volatility Of Smallcap Funds | The Money Show

Anil Agarwal's Vedanta looking for global partner! Reason - $20 billion expansion projects

'What's happening in India right now...' - Why ChatGPT-maker OpenAI CEO Sam Altman is excited about India

Rs 5000 monthly stipend: Deadline extended! Prime Minister-backed scheme - Know eligibility, last date to apply

Poonam Gupta: RBI Deputy Governor is double MA! A look at her journey from World Bank, IMF to Reserve Bank

NIOS Admit Card 2025: 10th and 12th admit cards OUT! Download from nios.ac.in | GUIDE

Follow Us :

© 2025 Bennett, Coleman & Company Limited</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            