
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09192-8'>The spatiotemporal distribution of human pathogens in ancient Eurasia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 15:52:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Infectious diseases have had devastating effects on human populations throughout history, but important questions about their origins and past dynamics remain1. To create an archaeogenetic-based spatiotemporal map of human pathogens, we screened shotgun-sequencing data from 1,313 ancient humans covering 37,000 years of Eurasian history. We demonstrate the widespread presence of ancient bacterial, viral and parasite DNA, identifying 5,486 individual hits against 492 species from 136 genera. Among those hits, 3,384 involve known human pathogens2, many of which had not previously been identified in ancient human remains. Grouping the ancient microbial species according to their likely reservoir and type of transmission, we find that most groups are identified throughout the entire sampling period. Zoonotic pathogens are only detected from around 6,500 years ago, peaking roughly 5,000 years ago, coinciding with the widespread domestication of livestock3. Our findings provide direct evidence that this lifestyle change resulted in an increased infectious disease burden. They also indicate that the spread of these pathogens increased substantially during subsequent millennia, coinciding with the pastoralist migrations from the Eurasian Steppe4,5. Pathogens have been a constant threat to human health throughout our evolutionary history. Until around 1850, at least a quarter of all children died before the age of one, and around another quarter before turning 15. Infectious diseases are estimated to have been responsible for more than half of these deaths6. Larger disease outbreaks have profoundly affected human societies, sometimes devastatingly affecting entire civilizations7. Infectious diseases have left lasting impressions on human genomes, as selective pressures from pathogens have continuously shaped human genetic variation8,9. Where and when different human pathogens first emerged, how and why they spread, and how they affected human populations are important but largely unresolved questions. During the Holocene (beginning roughly 12,000 years ago), the agricultural transition created larger and more sedentary communities, facilitating pathogen transmission and persistence within populations10. Simultaneously, the rise of animal husbandry and pastoralism are thought to have increased the risk of zoonoses3. Technological advances, such as horses and carts, increased both mobility and the risk of disease transmission between populations11. It has been proposed that these changes led to the so-called ‘first epidemiological transition' characterized by increased infectious disease mortality3. However, direct evidence remains scarce, and the idea is debated12. Palaeopathological examinations of ancient skeletons offer insights into past infectious disease burden13, but are limited to the few diseases identifiable from the available tissue. Recent advances in ancient DNA (aDNA) techniques allow for the retrieval of direct genomic evidence of past microbial infections, enabling the reconstruction of complete ancient pathogen genomes. These studies have typically focused on specific pathogens and have provided surprising insights into the evolutionary history of the causative agents of some of the most historically important infectious diseases affecting humans, including plague (Yersinia pestis)14,15, tuberculosis (Mycobacterium tuberculosis)16,17, smallpox (variola virus)18,19, hepatitis B (hepatitis B virus, HBV)20,21,22 and others23,24,25,26,27,28. However, there is an unmet need to investigate the combined landscape of ancient bacteria, viruses and parasites that affected our ancestors across various regions and time periods. Here we use a new high-throughput computational workflow to screen for ancient microbial DNA and use our data to investigate long-standing questions in paleoepidemiology: when and where did important human pathogens arise? And what factors influenced their spatiotemporal distribution? To understand the distribution of ancient pathogens, we developed an accurate and scalable workflow to identify ancient microbial DNA in shotgun-sequenced aDNA data (Extended Data Figs. The data (roughly 405 billion sequencing reads) are derived from 1,313 ancient individuals from western Eurasia (n = 1,015; 77%), central and north Asia (n = 265; 20%) and southeast Asia (n = 33; 3%), spanning a roughly 37,000-year period, from the Upper Palaeolithic to historical times (Fig. As burial practices varied across cultures and time, these samples represent a subset of groups within past societies. Nevertheless, the identified pathogens probably affected the broader population, as diseases spread easily in communities with poor sanitation and hygiene29. Initial metagenomic classification showed a large fraction of reads classified as soil-dwelling taxa including genera such as Streptomyces or Pseudomonas, reflecting a predominantly environmental source of microbial DNA. Further characterization using a topic model, however, suggested that microbial DNA in ancient tooth samples often derives from genera commonly associated with the human oral microbiome such as Actinomyces or Streptococcus (Extended Data Fig. Spatiotemporal distribution and site contexts of the study samples. White squares in the geographic maps indicate locations of the full set of n = 1,313 study samples, whereas coloured circles highlight location and age of samples from the time period and region indicated in the respective panel. Bar plots show the number of samples obtained from different site type contexts in each region. We selected a set of 136 bacterial and protozoan genera (11,553 species total) containing human pathogenic species2 as well as 1,356 viral genera (259,979 species total) for further authentication and detection of ancient taxa. We found that ancient microbial DNA was widely detected, with 5,486 authenticated individual hits identified across 1,005 samples (Z-score for aDNA damage rate from metaDMG of greater than or equal to 1.5; Fig. 2a, Supplementary Table 2 and Extended Data Fig. Of those, 3,384 hits were found among 214 known human pathogen species2, with the remaining 2,104 hits involving 278 other species. The highest numbers were observed in bacterial genera associated with the human oral microbiome, such as Actinomyces (380; 28.5% of samples) and Streptococcus (242; 18.1% of samples), or those commonly found in soil environments, such as Clostridium (252; 18.9% of samples) and Pseudomonas (111; 8.3% of samples). a, Bar plot showing the total number of putative ancient microbial hits (overall detection rate in brackets) for bacterial, eukaryotic and viral (n ≥ 4) genera. Bar colour and shading distinguishes counts in the different aDNA damage categories. b,c, Distributions of ANI (b) and log10 fold change of mapped reads over median of reads classified at taxonomic rank of genus per sample (c) for individual species hits in selected example genera. Symbol colour indicates aDNA damage category. We observed marked differences in the distributions of the genetic similarity of the ancient microbial sequences to their reference assemblies, both among genera and between species within a genus (Fig. High average nucleotide identity (ANI) indicates that ancient microbial sequences are closely related to a reference assembly in the modern database, and was observed in hits across all species from some genera (for example, Yersinia, Fig. In other genera, only a few hits had a closely related database reference assembly match. An example is the genus Mycobacterium, in which only hits of the leprosy-causing bacterium Mycobacterium leprae were highly similar to their reference assembly (ANI > 99%, Fig. Low ANI indicates that the ancient microbial DNA is only distantly related to the reference assembly, for example, because of aDNA damage, poor representation of the diversity of the genus in the database or false-positive classification of ancient microbial reads deriving from a related genus (Extended Data Fig. Alternatively, ANI can also be reduced when reads mapped to a particular reference assembly originate from many closely related strains or species in a sample. To test for such mixtures, we quantified the rate of observing different alleles at two randomly sampled reads at nucleotide positions across the genomes of hits with read depths greater than or equal to one. We found a high multi-allele rate in many species associated with the human oral microbiome, such as Streptococcus sanguinis or Treponema denticola. Hits for these species also showed lower ANI, consistent with the expectation for mixtures of ancient microbial DNA (Extended Data Fig. The rate of read mapping varied by orders of magnitude between species, from hits in species with high read recruitment, such as M. leprae (greater than 100-fold enrichment over the median number of classified reads across target genera) to hits at the lower limits of detection, for example, for the louse-borne pathogen Borrelia recurrentis (lowest read recruitment roughly 100-fold less than the median number of classified reads across target genera; Fig. Ancient microbial DNA from species commonly found in soil, such as Clostridium botulinum, was detected at similar rates in tooth and bone samples. Conversely, species associated with the human oral microbiome (for example, Fusobacterium nucleatum, Streptococcus mutans and Porphyromonas gingivalis) or pathogenic infections (for example, Y. pestis and HBV) were significantly more frequently identified in tooth samples (Extended Data Fig. To further verify hits with low read numbers, we performed a BLASTn search for all reads of each hit with n ≤ 100 final reads (n = 712 hits total; Supplementary Table 3). Most hits showed a high proportion (greater than or equal to 80%) of reads assigned to the same species using BLASTn, and the species with the most top-ranked BLASTn hits generally matched the inferred hit species (Extended Data Fig. Our results show that ancient microbial DNA isolated from human remains originates from complex mixtures of distinct endogenous and exogenous sources. The high detection rate, high read recruitment, lower ANI and evidence of mixtures in genera such as Clostridium or Pseudomonas (Fig. 5 and 6) suggest that a substantial fraction of this ancient microbial metagenome derives from environmental sources, possibly associated with the ‘necrobiome' involved in post-mortem putrefaction processes30,31 (Supplementary Information 3). By contrast, species from other frequently observed genera, including Actinomyces or Streptococcus, were predominantly identified from teeth and probably originated from the endogenous oral microbiome30. Species representing likely cases of pathogenic infections (for example, Y. pestis and M. leprae) were often characterized by higher ANI and/or low multi-allele rate, consistent with pathogen load predominantly originating from a single dominant strain. Our dataset provides a unique opportunity to investigate the origins and spatiotemporal distribution of human pathogens in Eurasia, expanding the known range of some ancient pathogenic species and identifying others not previously reported using paleogenomic data (Supplementary Tables 3 and 5). Considering bacterial pathogens, we found widespread distribution of the plague-causing bacterium Y. pestis, consistent with previous studies15. We identified 42 putative cases of Y. pestis (35 newly reported, Extended Data Fig. 6e), corresponding to a detection rate of roughly 3% in our samples. These newly identified cases expand the spatial and temporal extent of ancient plague over previous results (Fig. The earliest three cases were dated between around 5,700–5,300 calibrated years before present (cal. bp), across a broad geographic area ranging from western Russia (NEO168, 5,583–5,322 cal. This broad range of detection among individuals predating 5,000 cal. bp challenges previous interpretations that early plague strains represent only isolated zoonotic spillovers33. We replicated previously identified cases of plague in Late Neolithic and Bronze Age (LNBA) contexts across the Eurasian Steppe15 and identified many instances in which several individuals from the same burial context were infected (Afanasievo Gora, Russia; Kytmanovo, Russia; Kapan, Armenia; Arban 1, Russia) (Supplementary Table 2). These results indicate that the transmissibility and potential for local epidemic outbreaks for strains at those sites were probably higher than previously assumed33. Finally, 11 out of 42 cases were identified in late medieval and early modern period individuals (800–200 bp) from two cemeteries in Denmark (Aalborg, Randers), highlighting the high burden of plague during this time in Europe. All but one hit (NEO627, n = 84 reads total) showed expected coverage for the virulence plasmids pCD1 and pMT1, with hits before 2,500 years bp characterized by the previously reported absence of a 19 kilobase region on pMT1 containing the ymt gene15 (Extended Data Fig. Each panel shows geographic distribution (top) and timeline (bottom) for identified cases of the respective pathogen (indicated by coloured circles). Geographic locations and age distributions of all n = 1,313 study samples are shown in each panel using white squares. Another bacterial pathogen frequently detected was the spirochaete bacterium B. recurrentis, the causative agent of louse-borne relapsing fever (LBRF), a disease with a mortality of 10–40% (Supplementary Information 5). Whereas previous paleogenomic evidence for LBRF is limited to a few cases from Scandinavia and Britain23,34, we report 34 new putative cases (2.5% detection rate; Extended Data Fig. 6e), with wide geographic distribution across Europe, central Asia and Siberia (Fig. We detected the earliest case in a Neolithic farmer individual from Scandinavia (NEO29, Lohals, 5,647–5,471 cal. bp), suggesting that human body lice were already vectors for infectious disease during the Neolithic period, supported by phylogenetic analyses of B. recurrentis34. LBRF outbreaks were historically associated with crowded living conditions, poor personal hygiene and wet and cold seasons, but are rare today in most regions (Supplementary Information 5). Our results indicate that B. recurrentis infections exerted a substantial disease burden on past populations. We also report new cases of other bacterial pathogens previously detected in paleogenomic data. The leprosy-causing bacterium M. leprae was identified in seven individuals (0.5% detection rate) from Scandinavia and only appeared from the Late Iron Age onwards (earliest case RISE174, 1,523–1,339 cal. bp). Because M. leprae can infect both red squirrels and humans35, and archaeological evidence demonstrates that fur trade from Scandinavia, including squirrel fur, increased substantially during the late Iron and Viking Ages36, our results support the suggestion that squirrel fur trade could have facilitated transmission37. Our findings are also consistent with the widespread distribution of leprosy in medieval Europe38. We further detected three putative cases of Treponema pallidum—subspecies of which are the causative agents of treponematoses such as yaws, and endemic and venereal syphilis—in three individuals from recent time periods (earliest case 101809T, Denmark, 600–500 bp; Extended Data Fig. Two cases were identified in individuals from Borneo in southeast Asia (around 500–300 years bp), expanding the range of paleogenomic evidence for treponemal disease into this region. Among the species newly reported using paleogenomic data, we identified 12 putative cases of Yersinia enterocolitica, the causative agent of yersiniosis, commonly contracted through consuming contaminated raw or undercooked meat (Fig. The animal reservoirs for Y. enterocolitica include boars, deer, horses, cattle and sheep. As Y. enterocolitica rarely enters the bloodstream, our results probably underestimate the disease burden. This species includes some of the only identified putative zoonotic infections in individuals from Mesolithic hunter-gatherer contexts (NEO941, Denmark, 6,446–6,302 cal. bp). We also detected other members of the order Enterobacterales, transmitted through the faecal–oral route, including members of the genera Shigella, Salmonella and Escherichia (Supplementary Table 2). We report the earliest evidence for ancient leptospirosis (genus Leptospira) dating back to the Neolithic, 5,650–5,477 cal. Whereas earlier cases predominantly involved L. borgpetersenii (n = 5, 0.4% detection rate), most hits were Leptospira interrogans (n = 20, 1.5% detection rate), almost exclusively in Scandinavian contexts from the Viking Age onwards (Fig. L. borgpetersenii is today primarily found in cattle, whereas L. interrogans is detected more broadly in both domestic and wild animals. Although the clinical manifestations are similar, with an untreated fatality rate of 1% today, transmission routes vary39. Although host-to-host transmission predominates for L. borgpetersenii, transmission by means of urine-contaminated environments dominates for L. interrogans. We also report two putative cases of Corynebacterium diphtheriae, the causative agent of diphtheria; the oldest of which dates back to the Mesolithic (Sidelkino, 11,336–11,181 cal. Other diseases associated with animals and livestock, such as listeriosis (Listeria monocytogenes) and brucellosis (genus Brucella), could not be reliably identified. Another main human pathogen not identified in our dataset is M. tuberculosis, which causes tuberculosis. However, as the M. tuberculosis load in blood is typically low in immunocompetent patients without advanced disease40 and latent tuberculosis develops in 60% of cases and can persist for decades, it is, on the basis of current knowledge, unlikely to be readily identified using aDNA data from tooth and bone remains sampled for ancient human DNA. Identifying eukaryotic pathogens is challenging as sequence contamination from other organisms frequently occurs in their large and often fragmented reference genomes41. An illustrative example in our dataset is the protozoan parasite Toxoplasma gondii, which we readily identified in hits with high ANI and aDNA damage but low support from coverage evenness statistics owing to reads mapping to short contigs representing human contamination (Extended Data Fig. Despite these challenges, we identified nine putative malaria infections across three different human-infecting species (Plasmodium vivax n = 5; Plasmodium malariae n = 3; Plasmodium falciparum n = 1; Extended Data Fig. The most widely detected parasite species was P. vivax, with the earliest evidence in a Bronze Age individual from central Europe (RISE564, 4,750–3,750 bp based on an archaeological context). Other cases include a medieval individual from central Asia (DA204, Kazakhstan; 1,053–1,025 cal. The P. vivax malaria vector Anopheles atroparvus is at present widespread in Europe and nearby regions, including the Pontic Steppe, and our cases indicate this was also true in the past42. The single case of P. falciparum malaria was found in a sample from Armenia (NEO111; 463-0 cal. bp), where malaria was eliminated in the 1960s43. Among DNA viral species, we found widespread infections with HBV (28 cases, 2.1% detection rate), consistent with previous studies20,21,22 (Extended Data Fig. Our newly reported HBV cases include individuals from Mesolithic (Kolyma River, n = 1) and Neolithic (Lake Baikal, n = 3) contexts in Siberia dating back to 9,906–9,665 cal. bp, expanding the spatiotemporal range of ancient HBV into those regions (Extended Data Fig. We also report a putative ancient case (n = 1) of torque teno virus dating back roughly 7,000 years (NEO498, Ukraine; 7,161–6,950 cal. bp). Torque teno virus infects around 80% of the human population today and, although it is not associated with any particular disease, it replicates rapidly in immunocompromised individuals44. Other ancient virus hits included viruses not known to infect humans, such as ancient phage DNA (for example, Escherichia phage T4, Proteus virus Isfahan; Supplementary Table 2) and one putative case of an ancient insect virus (Invertebrate iridescent virus 31 (IIV-31)) in a tooth sample of a Viking Age individual from Sweden (VK30, Varnhem; 950–650 bp)45. The virus source is probably exogenous, potentially originating from aDNA of food sources in the tooth remains. Coinfections with many pathogens can worsen disease progression and outcomes46 and they were probably an important morbidity factor in ancient human populations. Searching for individuals showing co-occurrence of distinct ancient microbial species, we identified 15 cases of putative coinfections in our dataset (Supplementary Table 2). A striking case was a Viking Age individual from Norway (VK388), in which we replicated previous results of infection with a probably smallpox-causing variola virus19 and furthermore found evidence of infection with the leprosy-causing bacterium M. leprae. Another case of possible coinfection with M. leprae was found in VK366, a Viking Age individual from Denmark, who also showed evidence for leptospirosis (L. interrogans). This suggests that some of these cases involved chronic hepatitis, possibly reflecting HBV infection during infancy, when hepatitis becomes chronic in 90–95% of modern cases compared to only 2–6% in adult infections. An intriguing early case of a possible coinfection was found in a Mesolithic hunter-gatherer from Russia (Sidelkino, 11,336–11,181 cal. bp). This individual showed evidence of the respiratory pathogen C. diphtheriae and Helicobacter pylori, usually restricted to gastric infections; however, rare contemporary examples of bacteraemia have been reported for both47,48. Overall, our results show that coinfections can be detected using ancient metagenomic screening but are probably underestimated given methodological limitations such as differences in pathogen load, tissue availability and other factors affecting detectability of ancient microbial DNA. Understanding the factors affecting the dynamics of past epidemics is a main aim of paleoepidemiology. Our dataset allows us to address this question using direct molecular evidence for ancient pathogens across prehistory. To investigate changes in pathogen incidence over time, we performed Bayesian change-point detection and time series decomposition on two pathogens with high detection rates, Y. pestis (plague) and B. recurrentis (LBRF), using the detection rate of the respective pathogen as a proxy for its incidence (Methods). For plague, we inferred a gradual rise in detection rate starting from roughly 6,000 bp, about 1,000 years after the estimated time to the most recent common ancestor of now-known ancient strains (7,100 cal. bp)33. It reached a first peak around 5,000 bp across Europe and the Eurasian Steppe, coinciding with the emergence and early spread of the LNBA− strains, believed to have had limited flea-borne transmissibility15,49 (Fig. Detection remained high with extra peaks for a roughly 3,000-year period, until an abrupt change around 2,800 bp led to a roughly 800-year period in which plague was only detected in one sample (VK522, Oland, Sweden 2,343–2,154 cal. bp). Starting at roughly 2,000 bp, plague reappeared in three samples from central Asia (DA92, DA101, DA104, Kazakhstan and Kyrgyzstan; Extended Data Fig. 7 and Supplementary Table 2), just before the first historically documented plague pandemic (Fig. Another hiatus of roughly 600 years led to a rise and peak associated with the second plague pandemic roughly 600 bp (European late medieval cases, Denmark and previously published cases; Fig. This pattern of change coincides with the extinction of the LNBA− strains roughly 2,700 bp (ref. 49) and the second Y. pestis diversification event starting roughly 3,700 bp, which gave rise to an extinct Bronze Age lineage (RT5, LNBA+)50 and present-day lineages; these had increased flea-mediated transmission adaptations favouring bubonic plague and led to all known later plague pandemics51. The lack of detection during both periods is also seen in publicly available ancient Y. pestis genomes from other Eurasian sites, suggesting that sampling bias is unlikely to substantially influence the observed dynamics. Panels show estimated trendlines and 95% credible interval for detection rates (top), probability distributions and locations (dotted lines) for change points (middle) and probability of trend slope (bottom) being positive (red), negative (blue) or zero (white), inferred using Bayesian change-point detection and time series decomposition. Tops of panels show temporal distributions of newly reported pathogen hits (blue circles) as well as previously published ancient pathogens (grey circles) from the respective species. The inferred temporal dynamics of LBRF show a first peak in detection around 5,500 bp, slightly more recent than for plague, but with more sporadic occurrences and sharper peaks during the first roughly 2,000 years (Fig. bp) to the Altai mountains (RISE503, Russia, 3,677–3,461 cal. From roughly 2,800 bp, LBRF was detected more consistently, peaking around 2,000 years ago, predominantly in the Eurasian Steppe region (Fig. This change from epidemic outbreaks to endemicity overlaps in time with the estimated emergence of a distinct B. recurrentis Iron Age clade34 (Supplementary Information 5). The period of high LBRF detection coincided with a time without detectable plague activity (Fig. 4), reinforcing that the absence of plague is not due to sample size limitations or poor DNA preservation. This opposing pattern is unlikely to result from any cross-immunity between Y. pestis and B. recurrentis but could plausibly, in part, be caused by population size decreases and behavioural and societal adjustments during plague epidemics. LBRF remained detectable until the end of the time series, particularly in Europe; the continued presence might have facilitated the emergence of a medieval B. recurrentis clade roughly 600 years bp (ref. A striking feature shared in the temporal dynamics of plague and LBRF was the absence of detectable cases before roughly 6,000 bp, coinciding with a transition of individuals in predominantly hunter-gatherer contexts to those in farming or pastoralist cultural contexts. It has been proposed that this transition led to a higher risk of zoonotic disease transmission and facilitated the spread of both old and new pathogens3. Our dataset allows us to test this hypothesis using molecular evidence for infectious disease burden. To increase power to detect changes in the load of different pathogen types, we focused on grouped ancient microbial hit categories (Supplementary Table 4). We found that species associated with the ancient oral microbiome showed the highest relative detection rate, accounting for up to 50% of ancient hits across various periods (Fig. Species in the ‘environmental' classes of probably exogenous origins were also detected at consistent rates throughout time. Species in the ‘infection' classes occurred at low detection rates throughout (mostly less than 10%). We found that species in the ‘zoonotic' reservoir classes were not detected until around 6,500 bp (Fig. Using Bayesian time series decomposition, we inferred an overall increase in the detection rates of the zoonotic reservoir classes from roughly 6,000 bp, remaining at elevated levels until the medieval period (Fig. Whereas species in the ‘anthroponotic' reservoir classes also occur earlier (predominantly species with human-to-human transmission, Extended Data Fig. 10a), we observe increased detection rates from roughly 2,500 bp onwards (Fig. Our results thus provide direct evidence for an epidemiological transition of increased infectious disease burden after the onset of agriculture through to historical times. a, Timeline of relative detection rates in sliding windows of n = 21 temporally consecutive samples for different ancient microbial species classes. Coloured horizontal lines indicate the expected rates if species in all classes would be detected at equal rates based on the total number of distinct species in each class. b, Trendlines for detection rates inferred using Bayesian change-point detection and time series decomposition for ancient microbial species in the zoonotic (top) and anthroponotic (bottom) reservoir class. We used Bayesian spatiotemporal modelling to investigate possible drivers of the observed ancient microbial incidences. We modelled the presence or absence of either individual microbial species or combined species groups using sets of putative covariates, including spatiotemporal variables (longitude, latitude and sample age), paleoclimatic variables (mean annual temperature and precipitation), human mobility and ancestry, sample material (tooth or other) and a proxy for ‘detectability' (the number of human-classified reads). In the models for the zoonotic or anthroponotic infection species classes, sample age was an important predictor, consistently negatively associated with incidence, and high effect sizes in the individual species models for B. recurrentis and L. interrogans (Extended Data Fig. Longitude was another important factor in the infection classes; it was positively associated with incidence rates for the combined anthroponotic class, and in individual models for Y. pestis and B. recurrentis. The positive effect of longitude suggests a higher detection rate in the eastern part of our spatiotemporal range, where samples from the Eurasian Steppe predominate. The increased infection incidence in Steppe populations could reflect an increased genetic susceptibility or a higher risk of acquiring diseases associated with the pastoralist lifestyle. The latter suggestion seems more plausible as continued exposure to selective pressures from certain infectious diseases probably would reduce susceptibility in these populations. Human ancestry showed small but consistent positive effects in some models, particularly the infection classes, for the Caucasus hunter-gatherers. Across all models, the incidence of ancient microorganisms was positively associated with teeth as sample material; the highest effect sizes were found in the oral microbiome and infection classes (Extended Data Fig. Teeth preserved ancient oral microbiome and pathogen DNA better than petrous bones (the source of 86% of our samples), probably because of oral cavity exposure and better access to microbial DNA in the bloodstream53. These results support the notion that species detected in those classes are predominantly of endogenous origin. During the Holocene, human lifestyles changed considerably as agriculture, animal husbandry and pastoralism became key practices but the impact on infectious disease incidence is debated. Our study represents a large-scale characterization of ancient pathogens across Eurasia, providing clear evidence that identifiable zoonotic pathogens emerged around 6,500 years ago and were consistently detected after 6,000 years ago. Although zoonotic cases probably existed before 6,500 years ago, the risk and extent of zoonotic transmission probably increased with the widespread adoption of husbandry practices and pastoralism. Today, zoonoses account for more than 60% of newly emerging infectious diseases54. We observed some of the highest detection rates at roughly 5,000 bp, a time of substantial demographic changes in Europe due to the migration of Steppe pastoralists and the displacement of earlier populations4,5. Steppe pastoralists, through their long-term continuous exposure to animals, probably developed some immunity to certain zoonoses and their dispersals may have carried these diseases westwards and eastwards. Consequently, the genetic upheaval in Europe could have been facilitated by epidemic waves of zoonotic diseases causing population declines, with depopulated areas subsequently being repopulated by opportunistic settlers who intermixed with the remaining original population. This scenario would mirror the population decline of Indigenous people in the Americas following their exposure to diseases introduced by European colonists55,56. Our findings support the interpretation of increased pathogen pressure as a likely driver of positive selection on immune genes associated with the risk of multiple sclerosis in Steppe populations roughly 5,000 years ago57, and immune gene adaptations having occurred predominantly after the onset of the Bronze Age in Europe9. Expanding our analyses to the broader pathogen landscape allowed us to infer and contrast incidence patterns between different species and types of pathogens to a greater extent than previously possible. If ancient pathogen DNA of a single species is not detected in a particular region or period, asserting whether this is due to low disease incidence or confounding factors such as differential DNA preservation between different periods and environments is challenging. Our analyses counter these limitations; we demonstrate that pathogens with known epidemic potential and high detection rates, such as Y. pestis (plague) and B. recurrentis (LBRF), show notable differences in their detection rate over time, suggesting that low detection rates in these cases represent an actual reduction in incidence. During the early period (roughly 5,700–2,700 years ago), the continuous detection of Y. pestis is suggestive of endemic disease. The succeeding pattern of distinct waves and periods without detection indicate epidemic outbreaks; these detection peaks match the historically described plague pandemics. This shift from endemic to epidemic is concurrent with important changes in the Y. pestis genome, particularly increased flea-transmissibility and pathogenicity15,50. The pattern for B. recurrentis is almost entirely the opposite, with narrow peaks and long periods without detection, suggesting local epidemics before roughly 2,700 years ago and consistent detection afterwards. This later endemicity of LBRF could be driven by changes in the bacterial genome and by human and environmental factors known to increase the risk of louse infestation34,58,59. Experimental studies have demonstrated that Y. pestis, like B. recurrentis, can infect body lice in the midgut and, sometimes, also the Pawlowsky gland, a putative salivary gland59. Body lice infected in the Pawlowsky gland can transmit Y. pestis in concentrations sufficient to initiate disease in humans, possibly contributing to transmission during plague outbreaks. Infected body lice have higher mortality than uninfected lice, and it remains unknown whether coinfection of body lice with Y. pestis and B. recurrentis is possible. Our study has some important limitations. Whereas ancient shotgun metagenomic data offer direct evidence of past infections, their usefulness depends on having a high pathogen load and the right tissue samples. Our ancient tooth and bone samples are well suited to detect high-load bloodstream infections such as Y. pestis and B. recurrentis, but pathogens with lower loads or different tissue preferences are underrepresented. Moreover, differentiating ancient infections from those arising from environmental sources, such as the necrobiome, is challenging. Finally, our dataset lacks information on RNA viruses, therefore underestimating the zoonotic disease burden. However, the timing is probably accurate as the conditions favouring zoonotic transmission of RNA viruses are similar to those of other zoonotic pathogens60. Our findings demonstrate how the nascent field of genomic paleoepidemiology can create a map of the spatial and temporal distribution of diverse human pathogens over millennia. This map will develop as more ancient specimens are investigated, as will our abilities to match their distribution with genetic, archaeological and environmental data. Our current map shows clear evidence that lifestyle changes in the Holocene led to an epidemiological transition, resulting in a greater burden of zoonotic infectious diseases. This transition profoundly affected human health and history throughout the millennia and continues to do so today. We compiled a dataset of aDNA shotgun-sequencing data from 1,313 ancient individuals previously sequenced for studies of human population history (references for previous publications describing laboratory procedures and sample and/or site descriptions in Supplementary Table 1). To facilitate ancient microbial DNA authentication, we excluded sequencing libraries subjected to uracil-DNA glycosylase (UDG) treatment that removes characteristic aDNA damage patterns from further analyses. Samples sequenced across several libraries were combined into single analysis units to maximize sensitivity for detection of ancient microbial DNA present in low abundance. We carried out screening for ancient microbial DNA using a computational workflow combining k-mer-based taxonomic classification, read mapping and aDNA authentication. We first performed taxonomic classification of the sequencing reads (minimum read length 30 base pairs (bp)) using KrakenUniq61, against a comprehensive database of complete bacterial, archaeal, viral, protozoan genomes in the RefSeq database (built with default parameters of k-mer size 31 and low-complexity sequences masked). To increase sensitivity for ancient viral DNA, we reran the classification on a viral-specific database of complete viral genomes and neighbour assemblies from RefSeq (https://www.ncbi.nlm.nih.gov/genome/viruses/about/assemblies/), using all reads classified as non-human from the previous run. Following this initial metagenomic classification, a subset of genera was further processed in the genus-level read mapping and authentication stages. For bacterial pathogens, we selected genera with two or more established species of human pathogens from a recent survey of human bacterial pathogens2 (n = 125 genera). Genera with a single pathogenic species were not included to balance between including genera responsible for substantial human pathogenic burden and computational feasibility. For each genus of interest showing more than or equal to 50 unique k-mers assigned, all sequencing reads classified were collected and aligned in parallel against a representative reference assembly for each individual species within the genus. We selected the assembly with the most unique k-mers assigned as the representative reference genome for each species in a particular sample. If no reads were assigned to any assembly of the species in KrakenUniq, we selected the first assembly for mapping. Read mapping against the selected assembly was carried out using bowtie2 (ref. Mapped BAM files were subjected to duplicate marking using ‘samtools markdup'63, and filtered for mapping quality MAPQ ≥ 20. The aDNA damage rates were estimated using metaDMG64. To authenticate ancient microbial DNA, we calculated sets of summary statistics quantifying expected molecular characteristics of true positive ancient microbial DNA hits65: Summary statistics in this category measure how similar sequencing reads are to a particular reference assembly, with true positive hits expected to show higher similarity than false-positive hits. Summary statistics used include the following. This describes the average number of mismatches in sequencing reads mapped to a particular reference (lower means more similar to the reference). The number of unique k-mers that are assigned to a particular reference assembly from the KrakenUniq classification (higher means more similar to the reference). Summary statistics in the aDNA characteristics category measure the evidence for sequencing reads deriving from an aDNA source. Summary statistics used include the following. This describes the average length in base pairs of sequencing reads mapped to a particular reference (shorter means more likely to be ancient). The terminal aDNA substitution rate is the frequency of C>T (G>A) substitutions observed at the 5′ (3′) terminal base across all sequencing reads mapped to a particular reference (higher means more likely to be ancient). Bayesian Dmax is an estimator of the aDNA damage rate from metaDMG (higher means more likely to be ancient). Bayesian Z is an estimator of the significance of evidence for the aDNA damage rate from metaDMG (higher means more likely to be ancient). Summary statistics in this category measure how evenly mapped sequencing reads are distributed across a reference assembly. Summary statistics used include the following. The breadth of coverage expected for a particular average read depth is calculated66 as This is the ratio of the breadth of coverage observed in the mapping to the breadth of coverage expected given observed average read depth (higher means more even coverage). This relative entropy is a measure for the information content of the genomic positions of mapped reads. To obtain this statistic, we calculate the frequency of read alignments with their start positions falling within windows along the reference assembly, using two different window sizes (100 and 1,000 bp). The obtained frequency vector is converted into Shannon information entropy, and normalized using the maximum entropy attainable if the same total number of reads were evenly distributed across the windows (higher means more even coverage). From this initial screening, we then selected a subset of putative microbial ‘hits' (sample–species combinations) for further downstream analysis based on a set of aDNA authentication summary statistics: number of mapped reads greater than or equal to 20 5′ C>T deamination rate greater than or equal to 0.01 3′ G>A deamination rate greater than or equal to 0.01 ratio of observed to expected breadth of coverage greater than or equal to 0.8 relative entropy of read start positions greater than or equal to 0.9 For this initial filtered list of putative microbial hits, we ran metaDMG using the full Bayesian inference method to obtain Z scores measuring the strength of evidence for observing aDNA damage. The final list of putative individual ancient microbial hits was then obtained using the filtering cutoffs For authentication of viral species, we used the same filtering cutoffs described above, except for a lower ANI cutoff (greater than 0.95), as well as a lower cutoff for relative entropy of read start positions (greater than 0.7) for short viral genomes (fewer than 10 kilobases). The result of this filtering is a single best-matching species hit for each sample and genus of interest (Supplementary Table 2). We note that this approach will miss potential cases in which aDNA from many species of the same genus are present in the sample. However, because of the considerable challenges involved in distinguishing this scenario from false positives due to cross-mapping of ancient reads from a single source of DNA to reference assemblies of a closely related species (for example, Y. pestis/Yersinia pseudotuberculosis), we opted for the conservative option of retaining only the best hit for each genus. To further authenticate putative hits with low read counts (n ≤ 100 final reads), we carried out a BLASTn analysis. We extracted the reads for a species hit from the final filtered BAM files and queried them against the ‘nt' database (downloaded 18 August 2024) using ‘blastn -task blastn -max_hsps 1'. For the reads of each putative ancient microbial hit, we then tabulated the number of times and proportion of the highest scoring BLAST hits matched either the genus or species inferred from our workflow (Supplementary Table 3). We simulated aDNA fragments from microbial reference genomes in silico using gargammel67. We chose nine species representing pathogens of interest, and for each selected an assembly not present in the pathogen screening workflow database: For each reference genome, we simulated 5 million single-end sequencing reads (100 bp read length) with adapter sequences, with read length distribution and damage patterns from a mapDamage2 results of a previously published ancient pathogen genome (RISE509, Y. pestis15). The full-length simulated reads were then adapter-trimmed using AdapterRemoval68. To investigate the ability of the workflow to detect low abundance ancient microorganisms, we randomly down-sampled the full read set for each reference genome using seqtk (https://github.com/lh3/seqtk) (50, 100, 200, 500 reads; ten replicates each). We carried out topic model analysis on taxonomic classification profiles for each sample using the R package fastTopics69 (https://github.com/stephenslab/fastTopics). We used the number of unique k-mers assigned to non-human genera from KrakenUniq as the observed count data for each sample, excluding genera with fewer than 50 unique k-mers assigned. The analysis was carried out for L = 2 and L = 3 topics to capture broad structures in the classification profiles. Environmental, to capture all hits derived from environmental sources including the necrobiome (labelled environment_background, environment_pathogen, to distinguish potential pathogenic species from non-pathogenic ones) Oral microbiome, including both commensal and pathogenic species (microbiome_oral) Probably pathogenic infections, further distinguished into different modes of transmission (infection_anthroponotic; infection_vector_borne; infection_zoonotic). We define zoonotic pathogens here as those transmitted from animals to humans or which made such a host jump in our sampling time frame70. To infer temporal dynamics of ancient microbial species, we calculated detection rates in a sliding window of k = 21 temporally consecutive samples across the entire timeline of the 1,266 samples with dating information. For analyses of species combined in classes, we calculated the detection rate as the ratio of the total number of hits within a class in the window over the total number of possible hits across all species in a window (21 samples × 258 species across all classes). For individual species with n ≥ 20 hits or combined species classes, we further performed Bayesian change-point detection and time series decomposition (BEAST)71 implemented in the R package Rbeast (https://github.com/zhaokg/Rbeast), using the detection frequencies described above as response variables. Data for previously published Y. pestis and B. recurrentis genomes were obtained from AncientMetagenomeDir (release v.23.12, https://doi.org/10.5281/zenodo.10580647)72. To identify possible drivers of the observed spatiotemporal ancient microbial incidence, we combined the individual microbial species and the combined species groups with palaeoclimatic variables, human mobility estimates and kriged estimates of ancestry composition for Holocene West Eurasia. Palaeoclimatic reconstructions were accessed using the CHELSA-Trace21k data, which provide global monthly climatologies for temperature and precipitation at 30-arcsec spatial resolution in 100-year time steps for the past 21,000 years (ref. Human mobility values were accessed from ref. 74 and roughly represent the distance in kilometres between the burial location of the ancient human individual and its putative ancestral origin on the basis of patterns of genetic similarity derived from multidimensional scaling (MDS) analysis. Microbial species and/or groups were paired with the mobility estimate of the ancient human individual that occurs closest in space and time. Kriged ancestry estimates were extracted from ref. To determine the influence of the covariates on the microbial incidence, we used a hierarchical Bayesian model implemented in the inlabru R package77,78, in which ancient microbial presence or absence follows a binomial distribution and the spatiotemporal variables (latitude, longitude and sample age), number of human-classified reads, sample material, palaeoclimatic variables, human mobility and human ancestry constitute the linear predictors. The sample material is a categorical variable indicating whether the material used for sequencing was a tooth or not (bone), which inlabru treats as a random effect variable. We followed the default inlabru priors, in which distributions are distributed as a Gaussian variable with mean μ and precision τ. All covariates were normalized before the analyses. For each microbial species and group, we tested many models with different sets of covariates: (1) palaeoclimate + mobility + ancestry, (2) palaeoclimate + mobility, (3) palaeoclimate + ancestry, (4) only climate, (5) mobility + ancestry, (6) only mobility, (7) only ancestry and (8) no climate, mobility nor ancestry. Spatiotemporal variables, number of human-classified reads and sample material were included in all models. Because covariates were normalized, results indicate deviations from the mean. The effect size is interpreted in units of standard deviation. We used a criterion (DIC score) that prioritizes both fit and simplicity (low number of effective parameters) for evaluating the performance of hierarchical Bayesian models. Parameter distributions of the full model tended not to differ by much to those of the best-performing models under this score, when those parameters were included in the model. Results for both the full and best-performing models (that is, models with the lowest DIC score for each microbial species or combined species group) are shown in Extended Data Fig. DIC scores as well as Watanabe–Akaike information criterion for all models we tested can be found in the Supplementary Table 6. All maps were created using the R project for statistical computing79, using the sf package80. Shape files for coastlines, rivers and lakes were obtained from Natural Earth (https://www.naturalearthdata.com) using the rnaturealearth package81. 1 were obtained from AWS Open Data Terrain Tiles (https://registry.opendata.aws/terrain-tiles) using the elevatr package82. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data for the 907 individuals for whom sequencing data as trimmed read files (FASTQ) are released in this study have been deposited at the European Nucleotide Archive under accession no. Accession numbers for sequencing read data of the remaining individuals are provided in Supplementary Table 1. Processed analysis files including the KrakenUniq database file and metagenomic profiling results, microbial species read alignments (BAM format) as well as per-sample summary tables and plots from the screening pipeline are available at https://doi.org/10.17894/ucph.f0f75211-7bc3-445d-90c0-b72a22ba0930. A snakemake workflow implementing the computational screening pipeline is available at https://github.com/martinsikora/pathopipe. Lewis, C. M. Jr, Akinyi, M. Y., DeWitte, S. N. & Stone, A. C. Ancient pathogens provide a window into health and well-being. Bartlett, A., Padfield, D., Lear, L., Bendall, R. & Vos, M. A comprehensive list of bacterial pathogens infecting humans. Emerging and re-emerging infectious diseases: the third epidemiologic transition. Massive migration from the steppe was a source for Indo-European languages in Europe. Allentoft, M. E. et al. Population genomics of Bronze Age Eurasia. Infant and child death in the human environment of evolutionary adaptation. Harper, K. Plagues Upon the Earth: Disease and the Course of Human History (Princeton Univ. Karlsson, E. K., Kwiatkowski, D. P. & Sabeti, P. C. Natural selection and infectious disease in human populations. Kerner, G. et al. Genetic adaptation to pathogens and increased risk of inflammatory disorders in post-Neolithic Europe. Reproductive trade-offs in extant hunter-gatherers suggest adaptive mechanism for the Neolithic expansion. & Besse, M. More than the sum of their parts: reconstituting the paleopathological profile of the individual and commingled Neolithic populations of western Switzerland. Bos, K. I. et al. A draft genome of Yersinia pestis from victims of the Black Death. Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago. Bos, K. I. et al. Pre-Columbian mycobacterial genomes reveal seals as a source of New World human tuberculosis. Vågene, Å. J. et al. Geographically dispersed zoonotic tuberculosis in pre-contact South American human populations. Duggan, A. T. et al. 17th Century variola virus reveals the recent history of smallpox. Diverse variola virus (smallpox) strains were widespread in northern Europe in the Viking Age. Ten millennia of hepatitis B virus evolution. Krause-Kyora, B. et al. Neolithic and medieval virus genomes reveal complex evolution of hepatitis B. eLife 7, e36666 (2018). Guellil, M. et al. Genomic blueprint of a relapsing fever pathogen in 15th century Scandinavia. Schuenemann, V. J. et al. Genome-wide comparison of medieval and modern Mycobacterium leprae. Ancient herpes simplex 1 genomes reveal recent viral structure in Eurasia. van Dorp, L. et al. Plasmodium vivax malaria viewed through the lens of an eradicated European strain. Emergence of human-adapted Salmonella enterica is linked to the Neolithization process. Ancient human parvovirus B19 in Eurasia reveals its long-term association with humans. Bonczarowska, J. H. et al. Pathogen genomics study of an early medieval community in Germany reveals extensive co-infections. Abdoun, A., Amir, N. & Fatima, M. Thanatomicrobiome in forensic medicine. Burcham, Z. M. et al. A conserved interdomain microbial network underpins cadaver decomposition despite environmental variables. Lethal plague outbreaks in Lake Baikal Hunter-gatherers 5500 years ago. Susat, J. et al. A 5,000-year-old hunter-gatherer already plagued by Yersinia pestis. Ancient Borrelia genomes document the evolutionary history of louse-borne relapsing fever. Avanzi, C. et al. Red squirrels in the British Isles are infected with leprosy bacilli. : Resource Colonisation, Raw Material Exploitation and Networks in Middle Iron Age Sweden. Ancient Mycobacterium leprae genome reveals medieval English red squirrels as animal leprosy host. Pfrengle, S. et al. Mycobacterium leprae diversity and population dynamics in medieval Europe from novel ancient genomes. Bulach, D. M. et al. Genome reduction in Leptospira borgpetersenii reflects limited transmission potential. Rees, C. E., Swift, B. M. & Haldar, P. State-of-the-art detection of Mycobacterium tuberculosis in blood during tuberculosis infection using phage technology. Steinegger, M. & Salzberg, S. L. Terminating contamination: large-scale search identifies more than 2,000,000 contaminated entries in GenBank. Hay, S. I., Guerra, C. A., Tatem, A. J., Noor, A. M. & Snow, R. W. The global distribution and population at risk of malaria: past, present, and future. Roberto, P. et al. Torque teno virus (TTV): a gentle spy virus of immune status, predictive marker of seroconversion to COVID-19 vaccine in kidney and lung transplant recipients. Z., Scully, E. D. & Özgen, A. Invertebrate iridoviruses: a glance over the last decade. & Mendenhall, E. Syndemics and the biosocial conception of health. The first case of septicemia due to nontoxigenic Corynebacterium diphtheriae in Poland: case report. Han, X. Y., Tarrand, J. J., Dickey, B. F. & Esteva, F. J. Helicobacter pylori bacteremia with sepsis syndrome. Andrades Valtueña, A. et al. Stone Age genomes shed light on the early evolution, diversity, and ecology of plague. Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague. Demeure, C. et al. Yersinia pestis and plague: an updated view on evolution, virulence determinants, immune subversion, vaccination and diagnostics. Sun, Y.-C., Jarrett, C. O., Bosio, C. F. & Hinnebusch, B. J. Retracing the evolutionary path that led to flea-borne transmission of Yersinia pestis. Ancient pathogen DNA in human teeth and petrous bones. Global trends in emerging infectious diseases. First Peoples in a New World: Colonizing Ice Age America (Univ. Barrie, W. et al. Elevated genetic risk for multiple sclerosis emerged in steppe pastoralist populations. Bland, D. M., Long, D., Rosenke, R. & Hinnebusch, B. J. Yersinia pestis can infect the Pawlowsky glands of human body lice and be transmitted by louse bite. B. Zoonotic spillover: understanding basic aspects for better prevention. Breitwieser, F. P., Baker, D. N. & Salzberg, S. L. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Michelsen, C. et al. MetaDMG—a fast and accurate ancient DNA damage toolkit for metagenomic data. Warinner, C. et al. A robust framework for microbial archaeology. Lander, E. S. & Waterman, M. S. Genomic mapping by fingerprinting random clones: a mathematical analysis. Renaud, G., Hanghøj, K., Willerslev, E. & Orlando, L. gargammel: a sequence simulator for ancient DNA. Schubert, M., Lindgreen, S. & Orlando, L. AdapterRemoval v2: rapid adapter trimming, identification, and read merging. & Stephens, M. Non-negative matrix factorization algorithms greatly improve topic model fits. Zhao, K. et al. Detecting change-point, trend, and seasonality in satellite time series data to track abrupt changes and nonlinear dynamics: a Bayesian ensemble algorithm. A. et al. Community-curated and standardised metadata of published ancient metagenomic samples with AncientMetagenomeDir. Karger, D. N., Nobis, M. P., Normand, S., Graham, C. H. & Zimmermann, N. E. CHELSA-TraCE21k–high-resolution (1 km) downscaled transient temperature and precipitation data since the Last Glacial Maximum. Schmid, C. & Schiffels, S. Estimating human mobility in Holocene western Eurasia with large-scale ancient genomic data. Allentoft, M. E. et al. Population genomics of post-glacial western Eurasia. Lindgren, F. & Rue, H. Bayesian spatial modelling with R-INLA. B. inlabru: an R package for Bayesian spatial modelling from ecological survey data. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2014); https://www.R-project.org/ Massicotte, P. & South, A. rnaturalearth: World Map data from Natural Earth. Hollister, J. et al. elevatr: access elevation data from various APIs. The Lundbeck Foundation GeoGenetics Centre is supported by the Lundbeck Foundation (grant nos. R302-2018-2155, R155-2013-16338), the Novo Nordisk Foundation (grant no. CF18-0024), the Danish National Research Foundation (grant nos. DNRF94, DNRF174), the University of Copenhagen (KU2016 programme) and Ferring Pharmaceuticals A/S to E.W. Extra support was provided by Germany's Excellence Strategy (EXC-2077), project no. 390741603 ‘The Ocean Floor – Earth's Uncharted Interface'. We thankfully acknowledge Illumina Inc. for collaboration. thanks St. John's College, Cambridge, for providing a stimulating environment of discussion and learning. This work was further supported by the Swedish Foundation for Humanities and Social Sciences grant (Riksbankens Jubileumsfond grant no. was supported by Maritime encounters, Riksbankens Jubileumsfond, grant no. was supported by Marie Skłodowska-Curie Actions of the EU (grant no. 10120) and Independent Research Fund Denmark (grant no. is supported by the Marie Skłodowska-Curie Individual Fellowship ‘PALAEO-ENEO' (grant agreement number 751349). was supported by a Carlsberg Semper Ardens grant (no. is supported by a Villum Young Investigator Grant (project no. 00025300), a Novo Nordisk Fonden Data Science Ascending Investigator Award (grant no. NNF22OC0076816) and by the ERC under the European Union's Horizon Europe programme (grant agreement nos. was supported by the Lundbeck Foundation (grant no. are financially supported by Knut and Alice Wallenberg Foundation as part of the National Bioinformatics Infrastructure Sweden at SciLifeLab. Centre for Ancient Environmental Genomics and The Lundbeck Foundation GeoGenetics Centre, Globe Institute, University of Copenhagen, Copenhagen, Denmark Martin Sikora, Antonio Fernandez-Guerra, Gabriele Scorrano, Morten E. Allentoft, Frederik Valeur Seersholm, Charleen Gaunitz, Jesper Stenderup, Lasse Vinner, Kristian Kristiansen, Astrid K. N. Iversen & Eske Willerslev Section for Molecular Ecology and Evolution, Globe Institute, University of Copenhagen, Copenhagen, Denmark Elisabetta Canteri, Evan K. Irving-Pease, Hannes Schroeder & Fernando Racimo Department of Biology, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Lund University, Lund, Sweden Department of Biology and Biological Engineering, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Chalmers University of Technology, Gothenburg, Sweden German Centre for Infection Research (DZIF), Partner Site Charité, Berlin, Germany Department of Bacteria, Parasites and Fungi, Statens Serum Institut, Copenhagen, Denmark Trace and Environmental DNA (TrEnD) Laboratory, School of Molecular and Life Sciences, Curtin University, Perth, Western Australia, Australia Department of Cell and Molecular Biology, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala, Sweden MARUM Center for Marine Environmental Sciences and Faculty of Geosciences, University of Bremen, Bremen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar were involved in method development and implementation. were involved in reviewing drafts and editing. Correspondence to Martin Sikora, Astrid K. N. Iversen or Eske Willerslev. The authors declare no competing interests. Nature thanks Jonathan Dushoff and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. b, Distribution of total number of sequencing reads screened across the n = 1,313 study samples. c, Violin plots showing distributions of proportions of reads classified as human, non-human or not classified for the study samples. Median values for each genus are indicated by horizontal lines. d, Violin plots showing fraction of reads classified on the taxonomic level of genus, for the top 20 most abundant genera. e, Barplots showing inferred proportions for L = 3 topics (indicated by fill colour) from topic model analysis for n = 1,272 study samples with sample material information. g, Boxplots showing distributions of proportions for topic K3 (associated with oral microbiome taxa) in different sample materials. a, Illustration showing phylogenetic context and expected average nucleotide identity (ANI) for a hypothetical sampled microbial species X and four genomes (A1, A2; B1, B2) of two genera (A, B) present in the reference database. b, Number of unique k-mers classified at the level of genus using KrakenUniq for replicates of different read numbers across all simulated species. Dashed line indicates cutoff used in analysis of real data (150 unique k-mers). c, Number of unique k-mers classified at the level of species as a function of average nucleotide identity for mappings against all individual species reference genomes in the genus of reads simulated for a particular species. Blue diamonds indicate results for the mapping against a reference genome from the same species as the simulated read data, whereas grey circles indicate reference genomes of other species. Dashed line indicates ANI ≥ 0.97 cutoff value. d, Barplots showing number of replicates where the true positive species reference genome was highest ranking in numbers of unique k-mers classified at level of species. a, Observed breadth of genomic coverage as a function of average read depth for distinct species hits (i.e., mappings with highest number of unique k-mers at species level for a genus; n ≥ 20 reads mapped). Each panel shows results for reads simulated from species indicated. Results for mappings against the simulated species are indicated by diamond shape, whereas mappings against species from other genera are indicated with circles. Symbol fill colour indicates average nucleotide identity for mapped reads (grey symbols ANI < 0.97). Solid black line shows theoretical expected breadth of coverage for a given average read depth. Vertical dashed line indicates 1X average read depth. b, Relative entropy statistic (1000 bp window size) as a function of average nucleotide identity. Blue diamonds indicate results for the mapping against reference genome from the same species as the simulated read data, whereas grey circles indicate reference genomes for species hits in other genera. Dashed lines indicate cutoffs used in analyses of real data (ANI ≥ 0.97, entropy ≥ 0.9). False positive hits of reads mapped to a reference genome from a different genome passing cutoffs and their final number of mapped reads (out of 5 million total simulated reads) are labelled. c, Illustration showing potential sources of false positive hits and expected results for authentication summary statistics. d, Matrix plot showing all microbial hits with n ≥ 20 reads mapped and their authentication statistics, for all simulated species and read numbers. Symbol colour and size indicates the number of replicates passing the cutoff for each of three summary statistics shown (ANI ≥ 0.97, ratio of observed/expected coverage breadth ≥ 0.8, entropy ≥ 0.9). Hits passing cutoffs for all three statistics are indicated with coloured outline and background lines (black - true positives; grey - cross-genus false positive mappings). a, Observed breadth of genomic coverage as a function of average read depth. Coloured symbols indicate hits in species Toxoplasma gondii (left panel) and Yersinia pestis (right panel), with symbol colour indicating relative entropy of read start positions. Solid black line shows theoretical expected breadth of coverage for a given average read depth66. Symbol colour indicates the average number of reads mapped to a specific contig across samples. c, Bayesian estimator of aDNA damage (D max) and significance (Z-score) obtained from metaDMG, for hits in species Clostridium botulinum (left) and Yersinia pestis (right). Error bars indicate ± 1 standard deviation, and symbol fill colour indicates average read depth for mapped reads. Samples used as examples in aDNA damage curves (d) are labelled and indicated with black circles. d, aDNA damage patterns for four example hits in species Clostridium botulinum and Yersinia pestis. Plots show observed nucleotide misincorporation frequencies (red symbols and line) and metaDMG fit (black line) and 68% credible intervals (shaded region) for C > T transitions as a function of distance from the 5' read end. a, b, Distributions of ANI (a) and log10-fold change of mapped reads over median of reads classified at taxonomic rank of genus per sample (b) for individual species hits detected in n ≥ 5 samples. Symbol colour indicates species hit category. a, Odds ratios for association of ancient hits with sample material (tooth or bone) across n = 61 species with ≥ 20 ancient hits. Symbols indicate significance of association (p ≤ 0.01, Fisher's exact test; white triangles - more frequently identified in tooth; grey circles - no significant association). Error bars indicate 95% confidence interval of odds ratio b, c, Rates of observing multiple alleles in 2 randomly sampled sequencing reads at genomic sites in n = 190 ancient hits (average read depth ≥ 1X) across n = 120 samples. b, Multi-allele rate as a function of ANI. Symbol colour indicates average read depth. c, Distribution of multi-allele rate across species hits. d, e Barplots showing number of hits identified in each microbial species group (d) or each species within groups of likely infections (e). Novel and previously reported ancient pathogen hits are distinguished by bar colour, with total number in each category labelled. Each panel shows geographic distribution (top) and timeline (bottom) for identified cases of the respective pathogen (indicated by coloured circle). Geographic locations and age distributions of all n = 1,313 study samples are shown in each panel using white squares. a, Bar plots showing proportion of reads assigned to same species (dark blue) or genus (light blue) using BLASTn for all hits with n ≤ 100 final reads (n = 712), stratified by genus and microbial source groups b, Bar plots showing the proportion of ancient microbial hits with n ≤ 100 final reads matching the species with most reads assigned using BLASTn, stratified by microbial source group. c, Heatmap showing number of reads mapped to Yersinia pestis CO92 chromosome and plasmids, for n = 42 Yersinia pestis hits. Results for plasmid pMT1 are shown for full plasmid, as well as separately for the 19 kb region containing the ymt gene absent in the LNBA- strains. Samples are ordered by decreasing age from top to bottom. a-h, Panels show estimated trendlines and 95% credible interval for detection rates (top), probability distributions and locations (dotted lines) for change points (middle) and probability of trend slope (bottom) being positive (red), negative (blue) or zero (white), inferred using Bayesian change-point detection and time series decomposition. a, Panels showing geographic distributions (top) and timelines (bottom) for identified cases of ancient microbial hits in the oral microbiome and infection groups classes (indicated by coloured circle). Geographic locations and age distributions of all 1,313 study samples are shown in each panel using white squares. b, Principal component analyses showing ancient and modern human genetic population structure in non-African (left panels) and west Eurasian (right panels) individuals. Grey crosses indicate present-day individuals, whereas coloured symbols indicate ancient individuals (coloured by sample age). Diamonds with black outlines indicate position in PCA space for samples with hits in combined infection groups. Major clines of known ancient and modern human ancestry groups are indicated with labels. a, Matrix showing effect sizes and of n = 12 potential predictors (columns) for presence of selected combined ancient microbial species groups inferred from spatiotemporal modelling. Predictors with positive effect (defined here as 2.5% and 97.5% posterior quantiles both positive) are shown as red triangles, whereas predictors with negative effect (defined here as 2.5% and 97.5% posterior quantiles both negative) are shown as blue inverted triangles. Predictors where the 95% posterior quantile range spans zero are indicated using white circles. Posterior 95% quantiles are indicated by the error bars. b, Deviance information criterion values for each model and response variable. c, Matrix as in (a), showing effect sizes and of n = 12 potential predictors (columns) for presence of combined ancient microbial species groups and individual species from spatiotemporal modelling. For each class, estimates for the full model (gray background) and the model with lowest deviance information criterion (white background) is shown. Symbols and colors as in (a). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The spatiotemporal distribution of human pathogens in ancient Eurasia. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09246-x'>Coenzyme Q headgroup intermediates can ameliorate a mitochondrial encephalopathy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 15:52:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Decreased brain levels of coenzyme Q10 (CoQ10), an endogenously synthesized lipophilic antioxidant1,2, underpin encephalopathy in primary CoQ10 deficiencies3,4 and are associated with common neurodegenerative diseases and the ageing process5,6. CoQ10 supplementation does not increase CoQ10 pools in the brain or in other tissues. The recent discovery of the mammalian CoQ10 headgroup synthesis pathway, in which 4-hydroxyphenylpyruvate dioxygenase-like protein (HPDL) makes 4-hydroxymandelate (4-HMA) to synthesize the CoQ10 headgroup precursor 4-hydroxybenzoate (4-HB)7, offers an opportunity to pharmacologically restore CoQ10 synthesis and mechanistically treat CoQ10 deficiencies. To test whether 4-HMA or 4-HB supplementation promotes CoQ10 headgroup synthesis in vivo, here we administered 4-HMA and 4-HB to Hpdl−/− mice, which model an ultra-rare, lethal mitochondrial encephalopathy in humans. Oral treatment of Hpdl−/− pups with 4-HMA or 4-HB enabled 90–100% of Hpdl−/− mice to live to adulthood. Furthermore, 4-HB treatment stabilized and improved the neurological symptoms of a patient with progressive spasticity due to biallelic HPDL variants. Our work shows that 4-HMA and 4-HB can modify the course of mitochondrial encephalopathy driven by HPDL variants and demonstrates that CoQ10 headgroup intermediates can restore CoQ10 synthesis in vivo. Many common diseases are associated with decreased tissue CoQ10 levels8. Furthermore, some pharmaceuticals, such as statins, reduce CoQ10 levels, resulting in statin-associated muscle symptoms9. The perceived role of decreased tissue CoQ10 in diseases has fuelled a long-standing desire to replete tissue CoQ10 pools10. However, CoQ10 pools are refractory to pharmacological augmentation11,12,13, particularly in the central nervous system, owing to the poor bioavailability11,14 of CoQ10. New therapeutic approaches are needed to address the limitations of CoQ10 supplementation or to promote endogenous CoQ10 synthesis. A potential avenue of enabling CoQ10 synthesis in vivo is our recent discovery of the mammalian CoQ10 headgroup synthesis pathway7, in which HPDL catalyses the synthesis of 4-HMA, which generates 4-HB, the immediate precursor of the CoQ10 headgroup5,6,15 (Fig. Our work, and recent research showing that high doses of 4-HB rescue the survival of mice with hypomorphic Coq2 (Coq2A252V)16, indicates that 4-HMA and 4-HB might enable mammalian CoQ10 synthesis in patients with deficits in the CoQ10 headgroup synthesis pathway, providing a therapeutic strategy for patients with CoQ10 deficiencies. a, The mammalian CoQ headgroup synthesis pathway. Left, tyrosine is deaminated to 4-hydroxyphenylpyruvate (4-HPPA). HPDL converts 4-HPPA to 4-HMA, which generates 4-HB. The carbon atoms from 4-HMA or 4-HB that enter the CoQ headgroup are highlighted in red. Hpdl+/+ and Hpdl+/− pups survive for more than one year. d, Oral supplementation of 4-HMA and 4-HB (10 mg kg−1 per day) improves the overall survival of Hpdl−/− pups. CoQ9, CoQ10 and related compounds do not improve Hpdl−/− pup survival. Treatment with 4-HMA and 4-HB was carried out until P30. e, Schematic of 13C incorporation from 13C6-4-HMA and 13C6-4-HB into CoQ9 or CoQ10, increasing the mass of the product by 6 Da (m6). n = 9 or 10. f, Fractional labelling of brain CoQ9 and CoQ10 in mice fed 13C6-4-HMA and 13C6-4-HB (10 mg kg−1 once daily) from P3 to P20. CoQ9 and CoQ10 data from Hpdl−/− untreated mice were obtained in the 4-HMA experiment and are presented in a faded shade of colour in the 4-HB experiment for ease of interpretation. Significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post hoc test. P values < 0.05 are presented on each plot. Survival data were analysed with the log-rank test. To investigate this hypothesis, we tested whether 4-HMA and 4-HB could be used to treat Hpdl−/− mice, which robustly model neurodevelopmental disorder with progressive spasticity and brain white matter abnormalities (NEDSWMA)17,18,19,20,21,22,23, an ultra-rare lethal primary CoQ10 deficiency caused by HPDL variants that presents with severe neurodevelopmental symptoms. Hpdl−/− mice develop movements consistent with spastic paresis and epileptic seizures and die by postnatal day 15 (P15)21. Here we show that treatment with 4-HMA or 4-HB restores CoQ biosynthesis in the brains of Hpdl−/− mice and demonstrate rescue of Hpdl−/− mice by supplementation with CoQ10 headgroup intermediates. We also show that 4-HB treatment improves neurological symptoms and modifies the clinical course of a patient with HPDL variants. Restoration of brain CoQ10 synthesis with CoQ10 headgroup intermediates might alleviate the neurological symptoms of primary CoQ10 deficiencies and diseases associated with low tissue CoQ10 pools. As previously described21, Hpdl−/− pups do not gain weight as quickly as do Hpdl+/+ pups after birth, develop seizures (Extended Data Fig. 1c) and a higher plasma lactate:pyruvate ratio (Extended Data Fig. We hypothesized that treatment of Hpdl−/− pups with the CoQ10 headgroup intermediates 4-HMA or 4-HB would improve the neurological symptoms of Hpdl−/− mice. 4-HMA and 4-HB are orally bioavailable with half-lives of 2.8 h and 0.35 h, respectively (Extended Data Fig. 1d and Supplementary Table 1), and neither compound induces inflammation or necrosis in the brain, heart, lungs, liver, kidneys or spleen of mice at single doses of 500 mg kg−1 or at a dose of 500 mg kg−1 per day for 3 months (Extended Data Fig. 4-HMA was detected in the plasma of 4-HMA-treated Hpdl−/− pups on P10 (Extended Data Fig. Hpdl−/− pups treated orally with 4-HMA or 4-HB at a minimum of 10 mg kg−1 once daily starting on P3 or P7 gained weight and had improved mobility (Extended Data Fig. Hpdl−/− pups treated on P10 did not survive (Extended Data Fig. After weaning, we treated Hpdl−/− pups with 10 mg kg−1 4-HMA or 4-HB per day in their drinking water and discontinued 4-HMA or 4-HB treatment on P30. By contrast, Hpdl−/− pups treated orally with CoQ9, CoQ10, the CoQ analogue idebenone or the mitochondrially targeted antioxidant MitoTEMPO24,25,26 did not survive (Fig. We conclude that 4-HMA or 4-HB treatment extends the lifespan of Hpdl−/− pups to the average lifespan of a mouse. To test whether 4-HMA and 4-HB crossed the blood–brain barrier and were incorporated into CoQ9, the predominant coenzyme Q species in mice, and CoQ10, which is also found in mouse brain, we treated Hpdl+/+, Hpdl+/− and Hpdl−/− pups with 13C6-4-HMA or 13C6-4-HB, in which the ring carbons are labelled with 13C, at 10 mg kg−1 per day from P3 to P20. Treatment with 13C6-4-HMA and 13C6-4-HB also increased brain CoQ9 and CoQ10 pools in Hpdl−/− pups (Extended Data Fig. 2a), albeit to levels lower than those of Hpdl+/+ and Hpdl+/− mice27, and these lower CoQ levels persisted after 4-HMA treatment was stopped (Extended Data Fig. 4-HMA did not increase CoQ9 and CoQ10 levels in the brains of P90 wild-type mice (Extended Data Fig. These data show that Hpdl loss results in a primary CoQ deficiency in the brains of mice and that orally administered 13C6-4-HMA and 13C6-4-HB enable CoQ9 and CoQ10 synthesis in the brain. CoQ10 is essential for electron transport chain activity and maintenance of mitochondrial membrane potential28. Moreover, primary CoQ10 deficiencies are a cause of cerebellar ataxia in patients30. Electron microscopy revealed that mitochondria in the cerebella of P11 Hpdl−/− pups were smaller than those of Hpdl+/+ or Hpdl+/− control pups and that mitochondria in the cerebella of Hpdl+/− pups were smaller than those of Hpdl+/+ controls (Fig. 4-HMA increased cerebellar mitochondrial size in P11 Hpdl−/− pups, although not to the same size as those of Hpdl+/+ pups (Fig. P11 Hpdl−/− pups had small cerebella, with a thinner external granular layer, vacuolization and intracellular oedema in cerebellar Purkinje cells (PCs) and depletion of the internal granular layer (Fig. Cerebellum size in Hpdl−/− pups was greatly reduced on P6 (Extended Data Fig. 3a–c), when these animals responded to 4-HMA (Extended Data Fig. Cerebellar histology in P6 and P11 Hpdl−/− pups and in P56–P57 mice was restored by 4-HMA supplementation (Fig. Immunohistochemical staining for calbindin showed underdeveloped PC dendritic arbors in Hpdl−/− pups on P11 (Extended Data Fig. 4-HMA treatment largely restored PC dendritic arborization (Extended Data Fig. 3e,f and Extended Data Fig. Granule cell precursors in P11 Hpdl−/− pups had lower Ki-67 staining than did those in Hpdl+/+ pups (Extended Data Fig. Ki-67 staining was restored by 4-HMA treatment (Extended Data Fig. Similarly, the external granular layer thickness was reduced in Hpdl−/− pups on P11 compared with Hpdl+/+ and Hpdl+/− pups and was partially restored with 4-HMA treatment (Extended Data Fig. To determine whether these changes were initiated before P11, we analysed cerebellar morphology on P6. PAX6 staining revealed that granule cells were depleted in the external and internal granular layers of P6 Hpdl−/− pups relative to Hpdl+/+ pups and was improved with 4-HMA treatment (Extended Data Fig. Calbindin staining of the PCs in P6 Hpdl−/− pups revealed a multilayer without dendrites. 4-HMA treatment resulted in a single layer of PCs with stunted dendrites (Extended Data Fig. The cerebella of P6 Hpdl−/− pups had higher TUNEL staining indicating cell death, which decreased with 4-HMA treatment of Hpdl−/− pups (Extended Data Fig. 4-HMA increased cerebellar size in P6 Hpdl−/− pups (Extended Data Fig. a, Representative electron micrographs of the cerebella of P11 Hpdl+/+, Hpdl+/− and Hpdl−/− mouse pups reveal substantial changes in cerebellar mitochondrial morphology. The mitochondria (marked by arrowheads) in Hpdl+/+ and Hpdl+/− mouse pups are intact, whereas those in the cerebella of Hpdl−/− mouse pups are fragmented. Supplementing Hpdl−/− mice with 10 mg kg−1 4-HMA starting on P2–3 (Hpdl−/− + 4-HMA) restores cerebellar mitochondrial ultrastructure. Dashed boxes indicate area of interest highlighted in the second row. b, Quantification of mitochondrial perimeter in electron micrographs of the cerebella of P11 Hpdl+/+, Hpdl+/− and Hpdl−/− mouse pups as well as P11 Hpdl−/− mouse pups treated with 4-HMA. Mitochondria in the Hpdl−/− mouse pups are smaller than those of the Hpdl+/+ and Hpdl+/− pups. c, P11 Hpdl−/− pups have marked cerebellar atrophy with PCs (arrowheads) showing vacuolization and intracellular oedema, thinning of the inner granular layer (IGL) and molecular layer and compaction of the external granular layer (EGL). These abnormalities are improved to near-wild-type morphology by supplementation with 4-HMA starting on P2–3 (Hpdl−/− + 4-HMA). Dashed boxes indicate area of interest highlighted in the next row. Raw data and means are shown. P values are indicated on each plot. P values > 0.05 are not shown. For b, significance was tested by two-way ANOVA followed by Tukey's post hoc test. The cerebral cortex of P11 Hpdl−/− pups showed no major changes in mitochondrial morphology (Extended Data Fig. 4b) but subtle loss of healthy cortical architecture (Extended Data Fig. The hippocampus was largely unremarkable (Extended Data Fig. However, TUNEL staining was higher in the cortex, striatum and thalamus of P6 Hpdl−/− pups and treatment with 4-HMA significantly decreased TUNEL staining in the striatum (Extended Data Fig. These findings indicate that part of the phenotype of Hpdl−/− mice involves cerebellar cell death associated with a delay in cerebellar maturation and increased cell death in the cortex, striatum and thalamus. Metabolite replacement therapy with 4-HMA, the metabolic product of HPDL, largely reverted these abnormalities. Our data demonstrate that 4-HMA-dependent CoQ synthesis is important for brain development and indicate that treatment with 4-HMA or 4-HB during infancy or childhood could improve the outcomes of patients with HPDL variants. To test whether Hpdl loss affects mitochondrial electron transport chain activity, we measured electron flow through complexes I, II and IV (Extended Data Fig. 6a) in mitochondria isolated from the brains of P9 Hpdl+/+, Hpdl+/− and Hpdl−/− pups and P9 Hpdl−/− pups treated with 4-HMA. Consistent with loss of CoQ, electron flow through complexes I and II was reduced in mitochondria from Hpdl−/− pups and was improved by 4-HMA treatment (Fig. There was a non-statistically significant decrease in electron flow, rescued by 4-HMA, through complex IV (Fig. We observed similar decreases in electron flow through complexes I and II in mitochondria from the cortices of Hpdl−/− pups (Extended Data Fig. Mitochondria isolated from the cortices and cerebella of P90 Hpdl−/− mice treated with 4-HMA from P3 to P30 had electron flow through complexes I, II and IV comparable to that of P90 Hpdl+/+ and Hpdl+/− mice (Extended Data Fig. 4-HMA treatment did not increase electron flow through complexes I, II or IV in mitochondria isolated from the cerebella of P12 Hpdl+/+ pups (Extended Data Fig. These data demonstrate that loss of Hpdl attenuates electron flow through isolated cortex and cerebellar mitochondria in Hpdl−/− pups, presumably because of decreased CoQ levels. 4-HMA treatment from P3 to P30 restores electron flow through complexes I and II to near-wild-type levels even on P90. These data indicate that there is a CoQ threshold above which there is a durable rescue of mitochondrial electron flow, and that HPDL maintains CoQ pools above that threshold during development. a, Electron flow in cerebellar mitochondria from P9 Hpdl+/+, Hpdl+/− and Hpdl−/− pups as well as P9 4-HMA-treated Hpdl−/− pups. Electron flow through complexes I and II is impaired in Hpdl−/− mice and is restored by 4-HMA treatment. There was a non-statistically significant decrease in electron flow, rescued by 4-HMA, through complex IV. n = 5 animals for all groups. b, Patch-clamping of PCs in brain slices from P8–10 pups increases input resistance (Ri) and decreases membrane capacitance (Cm) in PCs from Hpdl−/− pups relative to PCs from Hpdl+/+ and Hpdl+/− pups. c, 4-HMA supplementation restores input resistance and membrane capacitance of PCs from Hpdl−/− pups to levels comparable to those in Hpdl+/+ and Hpdl+/− pups on P17–19. There were no differences in gait speed between the groups. e,f, 4-HMA-treated 8–10-week-old Hpdl−/− mice have similar performance to Hpdl+/+ and Hpdl+/− mice in the pole test (e) and the transverse beam test (f). g, Forelimb grip strength (left) is decreased in 8–10-week-old Hpdl+/− and 4-HMA-treated Hpdl−/− mice relative to Hpdl+/+ mice. All-limb grip strength (right) is similar in all groups. For behavioural tests, data from male mice (n: Hpdl+/+ = 4, Hpdl+/− = 5, 4-HMA-treated Hpdl−/− = 4) are plotted as triangles. Data from female mice (n: Hpdl+/+ = 4, Hpdl+/− = 2, 4-HMA-treated Hpdl−/− = 5) are plotted as circles. Behavioural data do not include untreated Hpdl−/− mice. Significance was tested by two-way ANOVA with Tukey's post hoc test. PCs are the sole output of the cerebellar cortex and depend on high metabolic activity to sustain rapid, chronic firing. To determine whether Hpdl loss affected the function of PCs, we manually patch-clamped PCs in brain slices from Hpdl+/+ and Hpdl+/− mice, as well as untreated and 4-HMA-treated Hpdl−/− mice, in three age brackets: P8–10 pups (during cerebellar development), P17–19 pups (after cerebellar maturation was mostly completed)31 and P56–57 pups (the age at which we did our behavioural studies). The P17–19 and P56–57 groups did not have untreated Hpdl−/− pups as these animals die by P15. In the P8–10 group, PCs from Hpdl−/− mice had increased input resistance (Ri) compared with PCs from Hpdl+/+ and Hpdl+/− mice (Fig. 3b and Table 1), indicative of fewer membrane ion channels. There was a significant reduction in membrane capacitance (Cm) in Hpdl−/− PCs relative to Hpdl+/− PCs32,33, indicative of smaller or less developed cells (Fig. These electrophysiological deficits were partially restored by 4-HMA treatment such that Ri was comparable to that of PCs of Hpdl+/− mice and lower than the Ri of PCs from Hpdl+/+ mice (Fig. Moreover, the Cm of PCs from 4-HMA-treated Hpdl−/− mice significantly increased relative to that of PCs from Hpdl−/− mice and was higher than that of PCs from Hpdl+/+ and Hpdl+/− mice (Fig. By P17–19 or P56–57, no differences were observed in the Ri and Cm of PCs from Hpdl+/+, Hpdl+/− and Hpdl−/− pups treated with 4-HMA (Fig. 3c, Table 1 and Extended Data Fig. These data are consistent with 4-HMA treatment correcting the aberrant passive electrophysiological characteristics of cerebellar PCs from Hpdl−/− pups. We next tested whether Hpdl loss affected the current–voltage (I–V) curves of PCs in mouse pups. Cerebellar PCs from P8–10 Hpdl−/− pups had a tighter grouping of currents at hyperpolarized membrane potentials (−100 to −60 mV) relative to that of Hpdl+/+ PCs (Extended Data Fig. 7a, insets), resulting in a flatter I–V curve for Hpdl−/− PCs at hyperpolarized membrane potentials (Extended Data Fig. Interestingly, 4-HMA treatment seemed to overcompensate for this, resulting in a steeper I–V curve of PCs in Hpdl−/− pups relative to those of the other groups (Extended Data Fig. 7a), closer in slope and magnitude to the I–V curves of P17–19 PCs (Extended Data Fig. The I–V curves of PCs from P17–19 and P56–57 Hpdl+/+, Hpdl+/− and 4-HMA-treated Hpdl−/− pups were comparable (Extended Data Fig. These data are consistent with PCs from Hpdl−/− pups having differential sensitivity to inputs at hyperpolarized membrane potentials than do Hpdl+/+ and Hpdl+/− PCs and partial reversion of these electrophysiological properties by 4-HMA. The peak frequency of PCs from P8–10 Hpdl−/− pups was significantly lower than those of Hpdl+/+ and Hpdl+/− PCs but was increased by 4-HMA treatment to a level higher than that of Hpdl+/+ pups (Extended Data Fig. These differences resolved by P17–19 (Extended Data Fig. These findings are consistent with immaturity of Hpdl−/− PCs relative to PCs of Hpdl+/+ and Hpdl+/− pups and accelerated maturation of 4-HMA-treated Hpdl−/− PCs. Hpdl loss and 4-HMA rescue did not affect the firing rate–current curves of PCs from P7–10 or 17–19 pups (Extended Data Fig. However, PCs from P56–57 Hpdl+/+ mice had a higher firing rate at higher input currents than PCs from Hpdl+/− and 4-HMA-treated Hpdl−/− mice (Extended Data Fig. Likewise, action-potential parameters of PCs from Hpdl+/+, Hpdl+/− and Hpdl−/− pups were mostly similar (Extended Data Fig. However, spike thresholds of PCs from Hpdl−/− pups treated with 4-HMA were more hyperpolarized than those of PCs from Hpdl+/+, Hpdl+/− and untreated Hpdl−/− pups on P8–10 (Extended Data Fig. The action half potential and spike half-width of PCs from Hpdl−/− pups treated with 4-HMA were slightly less than those of PCs from Hpdl+/+ pups (Extended Data Fig. The differences in action half potential between PCs from Hpdl+/+ or untreated Hpdl−/− pups and those from Hpdl−/− pups treated with 4-HMA persisted on P17–19 (Extended Data Fig. No significant differences in peak frequency or action-potential characteristics in PCs from P56–57 (8-week-old) Hpdl+/+, Hpdl+/− and 4-HMA-treated Hpdl−/− mice were observed (Extended Data Fig. These results demonstrate that homozygous Hpdl loss delays the functional maturation of cerebellar PCs. 4-HMA treatment accelerates and ultimately restores the functional maturity of cerebellar PCs as measured by electrophysiological parameters. Patients with HPDL variants typically present with ataxia, spastic tetraplegia and other deficits in motor function. To test whether 4-HMA improved the metabolic physiology and neurological symptoms associated with Hpdl variants, we carried out metabolic home-cage measurements and tests of motor function on P58 (approximately 8-week-old) Hpdl−/− mice treated with 4-HMA and compared them with Hpdl+/+ and Hpdl+/− mice. 4-HMA reduced seizures in Hpdl−/− mice (Extended Data Fig. At the start of testing, male, but not female, Hpdl−/− mice weighed less than their Hpdl+/+ and Hpdl+/− counterparts (Extended Data Fig. 9c) but had no significant differences in food or water consumption, activity, maximal oxygen consumption, carbon dioxide production or energy expenditure (Extended Data Fig. Hpdl−/− male, but not female, mice treated with 4-HMA had a significantly elevated respiratory exchange ratio (RER) compared with Hpdl+/− and Hpdl+/+ mice (sex × genotype interaction: F2,17 = 9.913, P < 0.01), suggestive of altered energy substrate use (Extended Data Fig. This effect seemed to be gene-dosage dependent, as Hpdl+/− male mice had statistically intermediate RER values (male Hpdl+/− versus Hpdl+/+, P < 0.05; male Hpdl+/− versus Hpdl−/−, P < 0.05; Extended Data Fig. Hpdl−/− mice treated with 4-HMA had a mild, significant reduction in the regularity of their step sequence (F2,17 = 3.826, P < 0.05; Fig. The first-order effect of the gait analysis was mouse survival that enabled ambulation. However, 4-HMA-treated Hpdl−/− mice showed no significant impairments in any other gait parameter or in the open-field, rotarod, pole and transverse-beam tests, which assess motor learning, coordination and balance (Fig. A gene-dosage effect was observed in forelimb grip strength, with significant reductions found in 4-HMA-treated Hpdl−/− mice relative to Hpdl+/− and Hpdl+/+ mice (F2,17 = 38.908, P < 0.001), and with Hpdl+/− mice having significantly lower forelimb grip strength than Hpdl+/+ mice (Fig. This pattern was similar but less pronounced in measures of grip strength for all four limbs. Only Hpdl−/− mice treated with 4-HMA had significant reductions in grip strength relative to Hpdl+/+ mice (F2,17 = 4.358, P < 0.05; Fig 3g). These results are potentially consistent with the firing rate–current curves of PCs from P56–57 Hpdl+/− mice and 4-HMA-treated Hpdl−/− mice, which have lower firing rates at higher input currents than do PCs from P56–57 Hpdl+/+ mice (Extended Data Fig. The gene-dosage effects on RER in male mice and on mouse grip strength suggest that there are neurological and metabolic features affected by Hpdl loss that are partially restored by our current 4-HMA treatment regimen, indicating that treatment may need optimization or that some neurological features are refractory to 4-HMA treatment. However, the improvement of motor phenotypes in 4-HMA-treated Hpdl−/− mice suggests that 4-HMA could improve the outcomes of humans with neurological symptoms due to HPDL variants. An eight-year-old child with biallelic HPDL variants (c.3G>C (p.Met1Ile), c.789delG (p.P264Lfs*51)) had a healthy birth and healthy development until eight years of age, when he presented with rapidly progressing spasticity over 2–3 months, although he probably had subtle symptoms before then. He was started on systemic CoQ10 supplementation in mid-October 2023. In early November 2023, medical records note that he could stand without support but had difficulty maintaining an upright posture with feet together. By late November 2023, he fell frequently and his ambulation was impaired to the degree that physical therapists advised the family to obtain a wheelchair. The patient's family history included two younger siblings who had died in infancy (Fig. The second sibling received genetic testing, revealing biallelic HPDL variants (c.3G>C (p.Met1Ile), c.789delG (p.P264Lfs*51)). Both variants are reported in ClinVar34 as pathogenic (VCV001344855.6 and VCV002221909.2) and the association of the c.3G>C (p.Met1Ile) variant with NEDSWMA has previously been reported19. HPDL variants are ultra-rare based on calculated allele frequencies (Table 2). b, Decrease in the Modified Ashworth Scale, a validated measure of spasticity, over the initial treatment course. Mitochondrial diseases generally show stability or increases in the Modified Ashworth Scale over time. c, Decrease in the Spastic Paraplegia Rating Scale (SPRS), a measure of spasticity and its effects on quality of life, over the treatment course to date. Mitochondrial diseases generally show stable or increased SPRS scores over time. Given the rapid and apparently unrelenting progression of the patient's symptoms (including 1.5 months while the patient was taking CoQ10), the history of a sibling with NEDSWMA and the availability of 4-HB in highly pure form, we proposed treatment with 4-HB. The administration of 4-HB was approved by the US Food and Drug Administration (single patient investigational new drug (IND) 170119) and the treatment protocol was approved by the NYU institutional review board (IRB; protocol 23-01512). The parents provided informed consent and the child agreed to treatment with 4-HB; data are shared with permission. Initial assessments included a brain MRI, with scans showing patchy T2 hyperintensity of the corticospinal tract of unclear relevance and no major cerebellar abnormalities (Extended Data Fig. The patient started treatment with 30 mg of 4-HB in a 5 mg ml−1 solution (6 ml) in water on day 1. On the basis of preclinical data, the dose was rapidly increased over the first 4  days to 100 mg kg−1 (600 ml of solution); that dose was maintained until day 30 of treatment. The dose was then decreased to 50 mg kg−1 (300 ml of solution) for an additional 95 days and then decreased to the current dose of 25 mg kg−1. During the first 30 days of the study, on several occasions, the child had emesis shortly after drinking the 600 ml solution, attributed to the large volume of solution ingested over a short period of time (15–20 min). No further emesis occurred during the 30 days with the lower dose volume of 300 ml. Safety monitoring, including electrocardiograms and bloodwork, was reassuring; there was no substantial change in these values for the duration of treatment. The data presented include 250 days of treatment. CoQ10 was discontinued after several months of treatment with 4-HB. Outcome measures (Table 3) demonstrate improvement of spasticity according to the Modified Ashworth Scale35 with the sum total of scores decreasing from 10 to 6 (Fig. The Spastic Paraplegia Rating Scale (SPRS)36 score, a measure of spasticity and its effects on quality of life, improved from 13 to 7 (Fig. Scores for the nine-hole peg test37, a measure of hand–finger dexterity, are consistent with stabilization or subtle improvement of the child's hand movement to times typical for children his age (Table 3). The patient's brain MRI did not change after 30 days of treatment (Extended Data Fig. Subjective reports, some of which are components of the SPRS, include improvement in endurance (the patient initially struggled to walk across a hospital lobby and by 30 days walked for 20 min), improvement in balance (the patient initially was unable to stand in tandem and was able to do so about 2 weeks after treatment initiation; after 8 months of treatment he was able to step laterally to catch a ball), decreased fall frequency from multiple times daily to rarely and restoration of running by approximately 1.5 months of treatment (Table 4). These data and observations demonstrate the safety and tolerance of 4-HB treatment in a child at a dose of 100 mg kg−1 for 27 treatments, followed by 50 mg kg−1 for 95 treatments and subsequent treatment with a maintenance dose of 25 mg kg−1. We show stabilization over the course of a rapidly progressing spasticity, followed by objective and subjective improvement in the child's motor function, providing evidence of improvement in neurological symptoms in a patient with HPDL encephalopathy17,18,19,20,21,22,23,39,40. The central role of CoQ10 in mitochondrial health has long been understood. Attempts to improve CoQ10 synthesis have been difficult in the periphery41 and impossible in the central nervous system. Here we demonstrate that the CoQ10 headgroup intermediates 4-HMA and 4-HB efficiently rescue the neurological phenotypes of Hpdl−/− mice, which model an ultra-rare primary CoQ10 deficiency. 4-HB modified the course of an HPDL-variant mitochondrial disease in a patient with no toxicity during eight months of treatment. Our results demonstrate that the mammalian CoQ10 synthesis pathway is amenable to pharmacological repletion with CoQ10 headgroup intermediates. We demonstrate that metabolite replacement therapy improves the neurological symptoms of a primary CoQ10 deficiency in a patient. Although 4-HMA treatment improves most neurological deficits and improves the survival of Hpdl−/− mice when administered by day 6 or 7 postnatally, initiation of treatment after P10 did not improve Hpdl−/− pup survival. These data indicate that there is a critical window in neural development during which the effects of HPDL encephalopathy will be partially reversible in patients. Treatment with CoQ10 headgroup intermediates during the critical window restores cellular function and development and improves neurological symptoms, even though our current 4-HMA and 4-HB treatment regimen only partially restores CoQ pools in the brains of Hpdl−/− mice. Patients, such as the person we treated, may have a longer critical window if they retain any residual HPDL activity. Once brain development is complete42, CoQ10 headgroup synthesis might be dispensable. In support of this hypothesis, 4-HMA-treated Hpdl−/− mice survive without supplementation of CoQ10 headgroup intermediates after P30. The durable rescue of mitochondrial function in Hpdl−/− mice suggests a CoQ threshold effect as well as differential requirements for CoQ synthesis during development and in adulthood. CoQ headgroup synthesis may not be required for the survival of adult mice living under controlled conditions, particularly if there are alternative sources of CoQ or CoQ headgroups. Conditional deletion of Hpdl will enable us to address the tissue-specific and temporal requirements of HPDL activity. Loss of CoQ headgroup synthesis is associated with increased TUNEL staining, consistent with apoptosis, in the developing cerebellum and in the cortex, striatum and thalamus (Extended Data Figs. This may be caused by loss of mitochondrial function or increased mitochondrial or cellular reactive oxygen species due to decreased CoQ. Increased apoptosis in these brain regions may contribute to the clinical symptoms of patients with HPDL variants. Determining the mechanisms underlying increased sensitivity of specific brain regions to loss of HPDL and CoQ headgroup synthesis will require further research. Treatment with 13C6-4-HMA or 13C6-4-HB restores CoQ9 and CoQ10 pools in Hpdl−/− mice to about half that of wild-type mice, and a substantial fraction of CoQ9 and CoQ10 remains unlabelled. These results are consistent with a parallel CoQ headgroup synthesis pathway or the uptake of CoQ headgroup from maternal milk, diet or another source. This alternative pathway is not sufficient for development but enables the survival of mice once development is complete. An alternative source of CoQ10 headgroup intermediates (or CoQ10 itself) may account for some of the variability in the presentation of HPDL encephalopathy and other primary CoQ10 deficiencies. Surprisingly, lower CoQ9 and CoQ10 levels restore mitochondrial electron flow and improve the survival of Hpdl−/− mice, although lower CoQ levels may be responsible for the reduced grip strength of 4-HMA-treated Hpdl−/− mice. Additional studies will be needed to test whether prenatal treatment, higher doses and improved formulations of 4-HMA and 4-HB increase brain CoQ9 and CoQ10 pools to wild-type levels. Our data are consistent with a role of HPDL in controlling the amount of CoQ9 and CoQ10 in the brain. Oral administration of 4-HB at high doses rescues perinatal lethality in mice with hypomorphic Coq2, which conjugates 4-HB to its isoprenoid tail during CoQ10 synthesis16. This previous work indicates that CoQ10 headgroup intermediates could treat other primary CoQ10 deficiencies. The authors of this previous work suggested that the treatment of HPDL pathogenic variants with 4-HB would require further investigation. Here we demonstrate that both 4-HMA and 4-HB restore the growth of mice lacking Hpdl and that 4-HB can improve the clinical outcomes of a child with a neurodevelopmental disorder driven by HPDL variants. We anticipate that early treatment with 4-HMA or 4-HB, enabled by newborn screening tests to detect inborn errors in CoQ10 headgroup synthesis or conjugation, will maximize the therapeutic benefits of treatment with CoQ10 headgroup intermediates. Larger trials of CoQ10 headgroup replacement therapy in patients with HPDL variants will be required to determine the outcomes of treatment with CoQ10 headgroup intermediates. Note added in proof: We note the recent publication of two genotype–phenotype correlation studies characterizing HPDL variants43,44. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Metabolomics data are available at Zenodo (https://doi.org/10.5281/zenodo.15361205)46. Additional patient data that cannot be shared due to privacy reasons are located in controlled access storage at NYU Langone Health. Source data are provided with this paper. Code used to analyse Patch Clamp data is available at GitHub (https://github.com/songs05/IonLab/tree/Ephys-Analysis). Heterogeneity of coenzyme Q10 deficiency: patient study and literature review. Awad, A. M. et al. Coenzyme Q10 deficiencies: pathways in yeast and humans. & Hargreaves, I. P. Primary coenzyme Q10 deficiency: an update. & Pagliarini, D. J. Biochemistry of mitochondrial coenzyme Q biosynthesis. Guerra, R. M. & Pagliarini, D. J. Coenzyme Q biochemistry and biosynthesis. Quinzii, C. M. & Hirano, M. Primary and secondary CoQ10 deficiencies in humans. Mas, E. & Mori, T. A. Coenzyme Q10 and statin myalgia: what is the evidence? Bentinger, M. et al. Stimulation of coenzyme Q synthesis. Wang, Y., Oxer, D. & Hekimi, S. Mitochondrial function and lifespan of mice with controlled ubiquinone biosynthesis. Rescue of primary ubiquinone deficiency due to a novel COQ7 defect using 2,4-dihydroxybensoic acid. Wang, Y. et al. Pathogenicity of two COQ7 mutations and responses to 2,4‐dihydroxybenzoate bypass treatment. & Hekimi, S. The efficacy of coenzyme Q10 treatment in alleviating the symptoms of primary coenzyme Q10 deficiency: a systematic review. Fernández-del-Río, L. & Clarke, C. F. Coenzyme Q biosynthesis: an update on the origins of the benzenoid ring and discovery of new ring precursors. Corral-Sarasa, J. et al. 4-Hydroxybenzoic acid rescues multisystemic disease and perinatal lethality in a mouse model of mitochondrial disease. Husain, R. A. et al. Bi-allelic HPDL variants cause a neurodegenerative disease ranging from neonatal encephalopathy to adolescent-onset spastic paraplegia. Sun, Y. et al. HPDL deficiency causes a neuromuscular disease by impairing the mitochondrial respiration. Wiessner, M. et al. Biallelic variants in HPDL cause pure and complicated hereditary spastic paraplegia. Evidence that autosomal recessive spastic cerebral palsy-1 (CPSQ1) is caused by a missense variant in HPDL. Ghosh, S. G. et al. Biallelic variants in HPDL, encoding 4-hydroxyphenylpyruvate dioxygenase-like protein, lead to an infantile neurodegenerative condition. Novel bi‐allelic HPDL variants cause hereditary spastic paraplegia in a Chinese patient. Micule, I. et al. Case report: two families with HPDL related neurodegeneration. Mancuso, M., Orsucci, D., Volpi, L., Calsolaro, V. & Siciliano, G. Coenzyme Q10 in neuromuscular and neurodegenerative disorders. Treatment of CoQ10 deficient fibroblasts with ubiquinone, CoQ analogs, and vitamin C: time- and compound-dependent effects. Dare, A. J. et al. Protection against renal ischemia–reperfusion injury in vivo by the mitochondria targeted antioxidant MitoQ. Albano, C. B., Muralikrishnan, D. & Ebadi, M. Distribution of coenzyme Q homologues in brain. Tran, U. C. & Clarke, C. F. Endogenous synthesis of coenzyme Q in eukaryotes. A. et al. Cerebellar ataxia and coenzyme Q deficiency through loss of unorthodox kinase activity. Naini, A., Lewis, V. J., Hirano, M. & Dimauro, S. Primary coenzyme Q10 deficiency and the brain. Millen, K. J., Wurst, W., Herrup, K. & Joyner, A. L. Abnormal embryonic cerebellar development and patterning of postnatal foliation in two mouse Engrailed-2 mutants. Development of rat cerebellar Purkinje cells: electrophysiological properties following acute isolation and in long-term culture. Arancillo, M., White, J. J., Lin, T., Stay, T. L. & Sillitoe, R. V. In vivo analysis of Purkinje cell firing properties during postnatal mouse development. Landrum, M. J. et al. ClinVar: public archive of interpretations of clinically relevant variants. & Carles-Hernández, R. Inter- and intra-rater reliability of the Modified Ashworth Scale: a systematic review and meta-analysis. The Spastic Paraplegia Rating Scale (SPRS): a reliable and valid measure of disease severity. Wang, Y.-C., Bohannon, R. W., Kapellusch, J., Garg, A. A. et al. Methods of 10‐meter walk test and repercussions for reliability obtained in typically developing children. Kojima, F. et al. A novel homozygous HPDL variant in Japanese siblings with autosomal recessive hereditary spastic paraplegia: case report and literature review. Ma, Y. et al. Two novel heterozygous HPDL variants in a Chinese family with a neurodevelopmental disorder with progressive spasticity and brain white matter abnormalities. Yogev, Y. et al. Limb girdle muscular disease caused by HMGCR mutation and statin myopathy treatable with mevalonolactone. Pujol, J., Vendrell, P., Junqué, C., Martí‐Vilalta, J. L. & Capdevila, A. When does human brain development end? Evidence of corpus callosum growth up to adulthood. Alecu, J. E. et al. Quantitative natural history modeling of HPDL-related disease based on cross-sectional data reveals genotype-phenotype correlations. Lee, E. H. et al. HPDL variant type correlates with clinical disease onset and severity. The mutational constraint spectrum quantified from variation in 141,456 humans. Shi, G. et al. Metabolomics data for CoQ headgroup intermediate plasma concentrations and incorporation into CoQ. We thank the staff of the DART Microscopy Laboratory, A. Liang, J. Sall and J. Liang, for their consultation and assistance with TEM work; I. Diaz for help and statistical discussions in presenting the single-patient IND data; D. N. Stephen for sectioning early postnatal mouse brains; D. M. Sabatini, D. E. Biancur, G. Francis Jr, J. Rodencal, S. Feske and W. Coetzee for critical reading and comments on the paper; K. Astudillo for the assistance in implementing the IRB protocol; the NYU Metabolomics Core Resource Laboratory (RRID: SCR_017935), Rodent Behavioral Laboratory (RRID: SCR_017942), IonLab (RRID: SCR_021754), Experimental Pathology Laboratory (RRID: SCR_017928) and DART Microscopy Laboratory (RRID: SCR_017934) for their assistance in obtaining the data presented here. These cores are partially supported by the Cancer Center Support Grant P30CA016087 at the Laura and Isaac Perlmutter Cancer Center. We are particularly grateful to the patient and his family and to the members of the Division of Neurology I, US Food and Drug Administration and the NYU Institutional Review Board. is a Damon Runyon-Rachleff Innovation awardee supported in part by the Damon Runyon Cancer Research Foundation (DRR 63-20) and is the recipient of a Pershing Square Sohn Prize for Young Investigators in Cancer Research and the MIND Prize from the Pershing Square Foundation, the Tara Miller Melanoma Foundation – MRA Young Investigator Award (668365), a Basic Research Grant from the Harry J. Lloyd Charitable Trust, an American Cancer Society Research Scholar Grant (RSG-21-115-01-MM), an NIGMS R35 MIRA from the NIH (1R35GM147119), an Irma T. Hirschl Career Scientist Award, a Concern Foundation Conquer Cancer Now Grant, a Harrington Scholar–Innovator Award from the Harrington Discovery Institute and an Oxford-Harrington Rare Disease Scholar Award, and research funding from NYU Langone Health Technology Opportunities and Ventures. is a Damon Runyon Dale F. Frey awardee supported by the Damon Runyon Cancer Research Foundation (DRG-50-22) and the NCI (R37CA289040). is supported by grants from the NIH (R37MH085726 and R01NS092096) and by an NCI Cancer Center Support Grant (CCSG, P30 CA08748) and Cycle for Survival. was supported by a François Wallace Monahan Fellowship. was supported by R01NS119301, R01NS127435 and P50HD103555 (Cell and Tissue Pathogenesis Core of the BCM IDDRC). is supported by a fellowship of the Takeda Science Foundation, a fellowship of the Astellas Foundation for Research on Metabolic Disorders and a fellowship of the Mochida Memorial Foundation for Medical and Pharmaceutical Research. The clinical portion of this research is supported in part by the NYU-H + H Clinical and Translational Science Institute (CTSI), which is supported by the National Center for Advancing Translational Sciences (NCATS), National Institutes of Health, through grant award number UL1TR001445. The NYU Metabolomics Core Resource Laboratory (RRID: SCR_017935) is partially supported by Cancer Center Support Grant P30CA016087 at the Laura and Isaac Perlmutter Cancer Center. Support for NYU Langone Health's Ion Laboratory (RRID: SCR_021754) is partially provided through the Office of Scientific Research, NIH R01 AI097302, NIH subaward D009105901, Sophion Industries and Nanion Technologies. Behavioural and continuous metabolic home-cage experiments were performed in the NYULMC Rodent Behavior Laboratory (RRID: SCR_017942) supported by the NIH BRAIN Initiative U19NS1076 (A.C.M. These authors contributed equally: Guangbin Shi, Claire Miller Department of Radiation Oncology, NYU Grossman School of Medicine, New York, NY, USA Guangbin Shi, Sota Kuno, Zixuan Wang, Quentin Spillier, Drew R. Jones & Michael E. Pacold Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY, USA Guangbin Shi, Sota Kuno, Megan Korn, Wyatt C. Tran, Zixuan Wang, Quentin Spillier, Robert S. Banh & Michael E. Pacold Department of Neurology, The Marlene and Paolo Fresco Institute for Parkinsons and Movement Disorders, NYU Grossman School of Medicine, New York, NY, USA Department of Pathology and Immunology, Baylor College of Medicine, Jan and Dan Duncan Neurological Research Institute, Texas Children's Hospital, Houston, TX, USA Alejandro G. Rey Hipolito, Tao Lin & Roy V. Sillitoe Developmental Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA Salsabiel El Nagar & Alexandra L. Joyner Department of Biochemistry, NYU Grossman School of Medicine, New York, NY, USA Megan Korn, Wyatt C. Tran, Drew R. Jones & Robert S. Banh Metabolomics Core Research Laboratory, NYU Grossman School of Medicine, New York, NY, USA Department of Neuroscience and Physiology Neuroscience Institute, NYU Grossman School of Medicine, New York, NY, USA IonLab, Department of Pathology, NYU Langone Health, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and planned the research. designed and carried out experiments. designed, conducted, analysed and interpreted the behavioural assays. treated the patient and collected all single-patient data without input from the other authors. All authors reviewed and approved the paper. are co-inventors on patents related to the use of 4-HMA, 4-HB and analogues in the diagnosis and treatment of neurodevelopmental and other diseases assigned to New York University. directly supervised the preclinical research. The treatment was conducted and supervised by C.M. under an institutional conflict-of-interest management plan designed and implemented by NYU Langone Health in accordance with its policies. consulted on the clinical protocol and C.M. Financial support for the clinical treatment was provided by the Pershing Square Foundation. The other authors declare no competing interests. Nature thanks Jared Rutter and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Treatment of Hpdl−/− pups with 4-HMA (n = 4), the product of HPDL, and 4-HB (n = 5), the immediate precursor of the CoQ headgroup, results in weight gain similar to Hpdl+/− pups after P9. Error bars represent median ± interquartile range. b, Seizure frequencies in pups. All Hpdl−/− pups (n = 6) have observed seizures. Error bars represent mean ± standard deviation for data from 3 mice. e, Long-term toxicity histology for 4-HMA and 4-HB administered orally for 3 months at 500 mg kg−1 per day. Sections are representative of n = 4 mice in each group. f, Plasma concentrations of 4-HMA in P10 Hpdl−/− pups treated with 10 mg kg−1 per day, orally administered 4-HMA starting at P3. Plasma 4-HMA concentrations are higher in treated Hpdl−/− pups than in Hpdl+/+, Hpdl+/−, and untreated Hpdl−/− pups. Hpdl+/+, Hpdl+/−, and Hpdl−/− pup data were obtained for Fig. 1c and are presented in a faded shade for ease of interpretation. 4-HMA-treated Hpdl−/− pup n = 7. g, Hpdl−/− pups can stand 3 days after supplementation with 4-HMA (10 mg/kg/day, oral administration). h, Survival of Hpdl−/− pups treated on P7 or P10. Hpdl−/− pups treated by postnatal day 7 (n = 32) survive into adulthood. Treatment of Hpdl−/− pups at 10 days after birth (n = 4) does not improve their survival. Significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, CoQ9 and CoQ10 concentrations from mouse brains following treatment with 13C6-4-HMA or 13C6-4-HB starting at postnatal day 3. CoQ9 and CoQ10 data from Hpdl−/− untreated mice were obtained in the 4-HMA experiment and are presented in a faded shade of color in the 4-HB experiment for ease of interpretation. Only a minor fraction of CoQ9 and CoQ10 is labeled with 13C. Hpdl+/+ n = 3, Hpdl+/− n = 3, and Hpdl−/− treated n = 3. c, CoQ9 and CoQ10 concentrations from the brains of P90 wild-type mice that were not treated with 4-HMA, or treated with 4-HMA from P2-P30. 4-HMA does not increase the brain concentrations of CoQ9 and CoQ10 in wild-type mice. Significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, Hematoxylin and eosin staining of cerebellar samples from P6 Hpdl+/+, Hpdl−/−, and Hpdl−/− mice treated with 4-HMA (Hpdl−/− + 4-HMA). Hpdl−/− mouse cerebella have pronounced atrophy that is partially rescued after 4-HMA treatment. b, Immunostaining for PAX6 (green) and Calbindin (red) on mid-sagittal sections of P6 cerebella from Hpdl+/+, Hpdl−/−, and Hpdl−/− mice treated with 4-HMA from P2-3 (Hpdl−/− + 4-HMA) (Panels A-C). Nuclei were counterstained with Hoechst (blue). Panels D-I are higher-magnification views of the boxed area in each image above with single-channel PAX6 (green) and Calbindin (red) staining. PAX6 stanning of granule cells shows that the EGL and IGL are depleted in the Hpdl−/− cerebella (Panel F compared to D, H). Calbindin staining of Purkinje cells shows the cells form a multilayer in Hpdl−/− cerebella (panel G). Purkinje cells form a single layer following 4-HMA treatment, but dendrites are stunted (panel I). Panels J-L contain lobule 3 from mid-sagittal cerebellar sections immunostained for Hoechst (blue) and TUNEL (yellow), showing an increase in cell death in Hpdl−/− cerebella (panel K) compared to control cerebella (panel J). 4-HMA treatment decreases cell death (Panel L). The white dashed line delineates the external granular layer of lobule 3. c, Quantitation of cerebellar area in midline sections from P6 pups in Extended Data Fig. Cerebellar size in Hpdl−/− pups is partially rescued by 4-HMA treatment. TUNEL staining in Hpdl−/− pups is partially decreased by 4-HMA treatment, consistent with decreased levels of apoptosis when CoQ headgroup synthesis is restored. e, P11 Hpdl−/− pups show reduced Purkinje cell dendrite height (panels G-J) and EGL thickness (panels T-V) as revealed by calbindin antibody and Ki-67 antibody staining, respectively. P11 Hpdl−/− pups also demonstrate reduced cerebellar size (panel G) and ectopic Purkinje cells (panel H), indicative of abnormal Purkinje cell development. These abnormalities appear restored by 4-HMA treatment from P2-3 (panels K-M, W-Y). f, Quantitation of dendrite height shown in the calbindin stains from Extended Data Fig. 2e, which confirms that P11 Hpdl−/− pups have significantly reduced dendrite height that was rescued in the 4-HMA treated group. g, Quantitation of External Granular Layer (EGL) thickness shown in the Ki-67 stains from Extended Data Fig. 2e, which confirms that P11 Hpdl−/− pups have significantly reduced EGL thickness that was rescued in the 4-HMA treated group. h, Hematoxylin and eosin staining of cerebellar samples from P56-57 Hpdl+/+, Hpdl+/−, and Hpdl−/− mice treated with 4-HMA. The cerebellar histology is grossly normal in all three groups of animals. For panels c, d, f, and g, significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, Representative calbindin staining of cerebellar sections from P56-57 Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice and quantitation of dendrite height. The dendrite height is similar in all three groups of animals. b, Representative electron micrographs of the cortex in P11 Hpdl+/+, Hpdl+/−, and Hpdl−/− pups, as well as Hpdl−/− pups treated with 4-HMA. We observed minimal changes in cortical mitochondrial ultrastructure in Hpdl−/− pups (panels C and G) compared to Hpdl+/+ pups (panels A and E) and Hpdl+/− pups (panels B and F). c, P11 Hpdl−/− pups exhibit subtle loss of normal cortical layers of the cerebral cortex (panels C, G, K), as well as edema, shrinking of the cortical neurons, and red neurons. These histological findings are partially restored by supplementation with 4-HMA (panels D, H, L). For panel a, significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, Staining for Hoechst (blue) and TUNEL (yellow) showing the forebrain of Hpdl+/+, Hpdl−/− and Hpdl−/− + 4-HMA mice at P6 (panels A-C). Magnified view of the red-boxed areas are on the right (panels D-O). n = 3 slides per mouse and 3 mice/genotype. The graphs represent the density of TUNEL+ particles per mm squared in the cortex (panel P), striatum (panel Q), thalamus (panel R) and hippocampus (panel S) of the Hpdl+/+, Hpdl−/− and Hpdl−/− + 4-HMA mice at P6. For panels P-S, significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, Electron transport chain substrates and inhibitors used for the electron flow assay. b, Electron flow measurements of mitochondria isolated from the cortex of P9 Hpdl+/+, Hpdl+/−, and Hpdl−/− pups, as well as Hpdl−/− pups treated with 4-HMA. Electron flow through complex I and complex II is reduced in Hpdl−/− pups, and is improved by treatment with 4-HMA. c, Electron flow measurements of mitochondria isolated from the cortex and cerebellum of P90 (3-month-old) Hpdl+/+ and Hpdl+/− pups, as well as Hpdl−/− pups treated with 4-HMA from P3-30. Electron flow through complexes I, II, and IV are similar in all groups. d, Electron flow measurements of mitochondria isolated from P12 Hpdl+/+ mice treated with 4-HMA starting on P3. 4-HMA treatment does not increase electron flow through complex I, II, or IV in wild-type pups. Mitochondria were isolated from n = 5 animals for all groups. Significance was tested by two-way analysis of variance (ANOVA) followed by Tukey's post-hoc test. a, The I/V curves of Purkinje cells from P8-10 Hpdl−/− pups and 4-HMA-treated Hpdl−/− pups are significantly different from each other and from the I/V curves of Hpdl+/+ and Hpdl+/− pups, which are similar. P-values indicating statistical significance are listed. The Hpdl−/− pups have a flatter I/V curve at hyperpolarized membrane potentials relative to the other curves due to a tighter grouping of currents at hyperpolarized membrane potentials (-100 to -60 mV) than the Hpdl+/+ and Hpdl+/− pups (see insert). The 4-HMA-treated Hpdl−/− pups have a steeper I/V curve similar to more mature Purkinje cells. b, The I/V curves of Purkinje cells from P17-19 Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− pups are similar. The inset depicts an example Hpdl+/+ Purkinje cell current response to the voltage step commands, which is similar to the Hpdl+/− and 4-HMA-treated Hpdl−/− pup Purkinje cells responses. c, The I/V curves of Purkinje cells from P56-57 Hpdl+/+, Hpdl+/−, and 4-HMA-supplemented Hpdl−/− mice are comparable. d, Peak frequency of Purkinje cells from P8-10 Hpdl+/+, Hpdl+/−, Hpdl−/− and 4-HMA-treated Hpdl−/− pups. The 4-HMA treated Hpdl−/− group has higher peak frequency than any of the other groups. Differences in frequency were identified by two-way ANOVA followed by Dunnett's multiple comparison test. e, Peak frequency of Purkinje cells from P17-19 Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− pups. The peak frequencies in all three groups are comparable. f, F/I curves of Purkinje cells from P8-10 Hpdl+/+, Hpdl+/−, Hpdl−/− and 4-HMA-treated Hpdl−/− pups. There were no significant differences in the linear portions of each curve between each group. g, F/I curves of Purkinje cells from P17-19 Hpdl+/+, Hpdl+/−, Hpdl−/− and 4-HMA-treated Hpdl−/− pups. There were no significant differences in the linear portions of each curve between each group. The insert shows the simple firing pattern of a more mature Purkinje cell. h, Input resistance and membrane capacitance in Purkinje cells from P56-57 (8-week-old; age comparable to behavioral assays) Hpdl+/+ and 4-HMA-supplemented Hpdl−/− mice are comparable, with a nonsignificant decrease in input resistance and a nonsignificant increase in membrane capacitance in Hpdl+/− mice. i, The F/I curve of Purkinje cells from P56-57 Hpdl+/+ mice exhibits a higher firing rate at higher input currents than Purkinje cells from Hpdl+/− and 4-HMA-treated Hpdl−/− mice. All electrophysiological data are presented as the averages of at least 10 Purkinje cells (Supplementary Table 4). a, Action potential characteristics of Purkinje cells from P8-10 Hpdl+/+, Hpdl+/−, Hpdl−/− and 4-HMA-treated Hpdl−/− pups. An average action potential is shown in panel A and the phase plane diagram is shown in panel B. Significant action potential shape differences can be seen between Purkinje cells from Hpdl+/+ and 4-HMA-treated Hpdl−/− pups. Purkinje cells from 4-HMA-treated Hpdl−/− pups had a significantly lower spike threshold than the Hpdl+/+ (p = 0.0076) and Hpdl−/− pups (p = 0.0207; panel C). The spike threshold for Purkinje cells from the Hpdl+/− and 4-HMA-treated Hpdl−/− pups trended to difference but was not significant (p = 0.0506), consistent with 4-HMA treatment rescuing Hpdl−/− pup grip strength to the level of Hpdl+/− mice (Fig. The spike half-width of Purkinje cells from the 4-HMA-treated Hpdl−/− pups was significantly narrower than all three treatments (Hpdl+/+: p < 0.0001; Hpdl+/−: p = 0.0002; Hpdl−/−: p < 0.0001; panel E). Finally, Purkinje cells from 4-HMA-treated Hpdl−/− pups had a shallower action half-potential (AHP) amplitude compared to Hpdl+/+and Hpdl−/− pups (WT: p = 0.0008; KO: p = 0.0461; panel F). The 4-HMA-treated Hpdl−/− pup statistics resemble those of Hpdl+/+ pups in the P17-19 group (Extended Data Fig. b, Action potential characteristics of P17-19 Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− pups. An average action potential is shown in panel A and the phase plane diagram is shown in panel B. c, Peak frequency and action potential characteristics of Purkinje cells from P56-57 Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice. All electrophysiological data are presented as the averages of at least 10 Purkinje cells (Supplementary Table 4). a, Metabolic phenotypes such as food and water intake, activity, O2 consumption, CO2 production, and energy expenditure are similar in 4-6-week-old Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice, as measured in metabolic cages. There were no significant changes in any of these parameters between Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice. b, Respiratory exchange ratio (vO2/vCO2) in Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice. c, 4-HMA treatment partially restores the weight of male Hpdl−/− mice and rescues the weight of Hpdl−/− female mice. The combined data demonstrate that 4-HMA partially restores the weight of Hpdl−/− mice relative to Hpdl+/− mice. d, Respiratory exchange ratio summary in Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice. Male Hpdl−/− mice treated with 4-HMA had a significantly increased respiratory exchange ratio (RER) compared to both Hpdl+/+ male mice and Hpdl+/− and 4-HMA-treated Hpdl−/− female mice. f, Open field data from 6–8-week-old Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice revealed no significant differences in distance moved or time that mice spent in the center of the open field. For behavioral tests, data from male mice (n: Hpdl+/+= 4, Hpdl+/−= 5, 4-HMA-treated Hpdl−/−=4) are plotted as triangles. Data from female mice (n: Hpdl+/+= 4, Hpdl+/−=2, 4-HMA-treated Hpdl−/−=5) are plotted as circles. Behavioral data do not contain comparisons to untreated Hpdl−/− mice, which do not survive to the age at which these behavioral assays are carried out. Data presented are mean ± standard error of the mean, as well as raw data. Behavioral data are presented as box and whisker plots, where the center is the median, the box bounds are the 25th to 75th percentiles, and whiskers indicate minimum to maximum. a, Additional catwalk data from 6–8-week-old Hpdl+/+, Hpdl+/−, and 4-HMA-treated Hpdl−/− mice reveal no significant differences in gait parameters between the groups of mice. G, symmetry index for stride length. b, T1- and T2-weighted brain MRI images from a patient with biallelic HPDL variants before treatment with 4-HB (day 0; panels A and B) and 30 days after treatment with 4-HB (panels C and D). For behavioral tests, data from male mice (n: Hpdl+/+= 4, Hpdl+/−= 5, 4-HMA-treated Hpdl−/−=4) are plotted as triangles. Data from female mice (n: Hpdl+/+= 4, Hpdl+/−=2, 4-HMA-treated Hpdl−/−=5) are plotted as circles. Behavioral data do not contain comparisons to untreated Hpdl−/− mice, which do not survive to the age at which these behavioral assays are carried out. Data presented are mean ± standard error of the mean, as well as raw data. Behavioral data are presented as box and whisker plots, where the center is the median, the box bounds are the 25th to 75th percentiles, and whiskers indicate minimum to maximum. 4-HMA doses of 10 mg kg−1 per day or 20 mg kg−1 per day improves the 30-day survival of 90% of pups at 30 days after birth. Doses of 0.1 mg kg−1 per day or 1 mg kg−1 per day did not improve the overall survival of Hpdl−/− pups. Number of Purkinje cells that were used for patch-clamp for electrophysiological studies. P12 Hpdl−/− pups (left) have spasticity, posturing and seizures, whereas Hpdl+/+ and Hpdl+/− littermates are healthy in appearance. Effects of 4-HMA treatment on an Hpdl−/− pup. Two Hpdl−/− pups are on the left and two Hpdl+/− littermates are on the right. The Hpdl−/− pup second from the left was treated with 4-HMA at 10 mg per kg per day orally and showed improvement in growth and motor function. The untreated Hpdl−/− pup died by P15. Gait of an Hpdl+/+ mouse on the catwalk system. Example video of an 8-week-old Hpdl+/+ mouse during gait evaluation. Gait of an Hpdl+/− mouse on the catwalk system. Example video of an 8-week-old Hpdl+/− mouse during gait evaluation. Gait of an Hpdl−/− mouse treated with 4-HMA on the catwalk system. Example video of an 8-week-old Hpdl−/− mouse previously treated with 4-HMA during gait evaluation. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Shi, G., Miller, C., Kuno, S. et al. Coenzyme Q headgroup intermediates can ameliorate a mitochondrial encephalopathy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09233-2'>Selective remodelling of the adipose niche in obesity and weight loss</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 15:05:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Weight loss significantly improves metabolic and cardiovascular health in people with obesity1,2,3. The remodelling of adipose tissue (AT) is central to these varied and important clinical effects4. However, surprisingly little is known about the underlying mechanisms, presenting a barrier to treatment advances. Here we report a spatially resolved single-nucleus atlas (comprising 171,247 cells from 70 people) investigating the cell types, molecular events and regulatory factors that reshape human AT, and thus metabolic health, in obesity and therapeutic weight loss. We discover selective vulnerability to senescence in metabolic, precursor and vascular cells and reveal that senescence is potently reversed by weight loss. We define gene regulatory mechanisms and tissue signals that may drive a degenerative cycle of senescence, tissue injury and metabolic dysfunction. We find that weight loss reduces adipocyte hypertrophy and biomechanical constraint pathways, activating global metabolic flux and bioenergetic substrate cycles that may mediate systemic improvements in metabolic health. In the immune compartment, we demonstrate that weight loss represses obesity-induced macrophage infiltration but does not completely reverse activation, leaving these cells primed to trigger potential weight regain and worsen metabolic dysfunction. Throughout, we map cells to tissue niches to understand the collective determinants of tissue injury and recovery. Obesity affects more than one billion people worldwide5. Increased AT mass, which is the defining feature of obesity, is one of the main risk factors for type 2 diabetes, cardiovascular disease, certain cancers and early death6. Reduction in AT mass through weight loss (WL) significantly improves obesity-induced comorbidities and can reduce mortality1,2,3. A synergistic and detailed understanding of the biology underpinning these contrasting clinical effects is central to improving treatment options and health outcomes. ATs have a unique capacity to adapt their structure and functions to maintain metabolic homeostasis as energy demands change4,7. In obesity, excess expansion limits this flexibility and induces pathological remodelling changes, notably adipocyte hypertrophy, immune cell infiltration, pro-inflammatory cytokine release, impaired angiogenesis and fibrosis, that contribute to multiorgan inflammation, insulin resistance, metabolic dysfunction and disease4,7. However, despite extensive investigation, the molecular triggers, cellular phenotypes and signalling pathways underlying obese AT dysfunction, particularly in humans, are only partly understood. Therapeutic WL leads to a reduction in AT mass, systemic inflammation and insulin resistance, as well as subsequent improvements in obesity-related comorbidities3,4,8. Although this strongly suggests that WL ameliorates AT dysfunction and its harmful physiological effects, surprisingly little is known about the underlying mechanisms. Defining the cell types, regulatory mechanisms and signalling pathways responsible for pathological and therapeutic AT remodelling is needed to guide therapy development for the harmful health consequences of obesity. To better understand obese AT dysfunction and its reversal after WL, we carried out single-nucleus RNA sequencing of approximately 100,000 cells from men and women with extreme obesity (n = 25) before and after WL surgery and from healthy lean controls (n = 24; Fig. WL significantly improved metabolic parameters, although not to the lean baseline (Fig. We focused on abdominal subcutaneous AT because of its contribution to central obesity and its adverse metabolic effects relative to other subcutaneous AT depots10. This cohort formed the basis for between-group exploratory analyses. Our results were integrated with a further 50,000 cells (nuclei) from the largest published human subcutaneous adipose atlas to improve cell annotation11 (n = 9 obese and n = 4 lean samples; Extended Data Fig. Spatial transcriptomics in equivalent cohorts (approximately 25,000 cells, n = 4 per group; Fig. a, Graphical representation of the primary study cohort (left; single-nucleus analyses in n = 25 obese (OB) people before and after WL and n = 24 lean (LN) people, with spatial analyses in n = 4 people per group) and AT anatomical location (right). b, Clinical characteristics of the primary cohort (n = 24 LN and 25 paired OB–WL donors). Boxplot, median interquartile range minimum and maximum. BMI, body mass index (kg m–2); F insulin, fasting insulin (mIU L–1); HbA1c, haemoglobin A1c (%); HDL, high-density lipoprotein cholesterol (mM); DBP, diastolic blood pressure (mm Hg). c, Uniform manifold approximation and projection (UMAP) of 145,452 human AT cells (n = 74 samples of the primary cohort and n = 13 samples of the Emont published cohort11, single nucleus). d, Cell-type proportions (for the cell types in c) in the combined cohort, mean per group, and for each sample (single nucleus). e, Correlations between cell types and clinical traits (Pearson, LN and OB samples only, single nucleus). Illustration in a created using BioRender (Scott, W., https://BioRender.com/rtmnzaj; 2025). This captured a rich representation of the cellular, structural and functional dynamics of the subcutaneous AT niche in human weight gain and WL. 1c) and compositional analyses demonstrated extensive immune cell (mainly macrophage but also lymphocyte) infiltration in obese AT (Fig. Obese AT also showed a deficit in mature adipocytes, suggesting increased cell death and/or a failure to replenish mature adipocytes. Immune cell infiltration is a pathognomonic feature of obese AT7 but the impact of WL on inflammatory remodelling is unclear, with studies indicating opposing anti- and pro-inflammatory effects9. We clustered myeloid cells (n = 34,280; Fig. a, UMAP embedding of myeloid (MYE) cell classes (top) and densities (bottom). b, LAM subtype (ST) marker genes relative to the main macrophage classes. FCG, fraction of cells in the group. c, LAM subtype proportions in LN, OB and WL (left) and OB split into low and high fasting insulin (FI, right), relative to total macrophages/sample. Boxplot, median IQR minimum and maximum; n, number of donors. Wilcoxon paired (OB–WL) and unpaired (OB–LN, FI) two-tailed, FDR adjusted P-values. d, CellTypist predicted LAM subtypes in spatial datasets at CLS (top). ph, pentose phosphate pathway; OxPhos, oxidative phosphorylation; Glycol/glucoN, glycolysis/glucogenesis; FA syn., fatty acid synthesis; FA ox., fatty acid oxidation. Coloured by log2-transformed fold change (log2FC): red, obese high; blue, obese low; sized by adjusted –log10P-value; negative binomial mixed-effects model. Circled dots represent comparisons with absolute log2FC > 0.3 and adjusted P < 0.05. g, Transcriptomic flux-based analyses (top) showing global metabolic activation in LAMs compared with TRMs. SCENITH (bottom) basal respiration (HPG incorporation) and glycolytic capacity (change in HPG incorporation) in LAMs and TRMs from OB donors (n = 7, mean ± s.e.m., paired Student's t-test). h, Differential gene regulatory networks in: left, macrophage subtypes (scaled log2FC > 0.3, subtype versus all other subtypes, Wilcoxon, FDR < 0.05); and right, all macrophages (Mɸ) in LN–OB and OB–WL comparisons (log2FC, Wilcoxon, red, OB high). The increase in AT macrophages (mean from 14% to 31%) primarily comprised lipid-associated macrophages (LAMs; mature MYE2 and immature MYE3) expressing lysosomal, lipid metabolism and metabolic activation markers (CD9, TREM2, LPL and LIPA; Fig. Classical monocytes (MYE5) expressing VCAN also increased, indicating constitutive trafficking from blood. Visualization and marker gene patterns supported a differentiation continuum from monocytes, to immature and then mature LAMs (Fig. Proportional analyses revealed lower fractions of tissue-resident macrophages (MYE1 and TRMs) expressing homeostatic markers (LYVE1, FOLR2 and MRC1; Extended Data Fig. Neighbourhood graphs confirmed that this represented a relative (not absolute) TRM reduction (Extended Data Fig. Proliferative macrophages expressed MCP-1 (CCL2), TRM and LAM markers, supporting low-level MCP-1-dependent expansion of both populations in human obesity13 (Extended Data Fig. Independent of adiposity, LAM abundance increased with metabolic dysfunction (Extended Data Fig. This led us to hypothesize that LAMs might have pleiotropic adaptive and maladaptive features. Spatial and protein analyses indicated context-dependent orientations and functions, with adaptive LAMs aggregating at crown-like structures (CLS; around transcriptionally devoid adipocytes) and inflammatory LAMs being more abundant in isolation or pairs (Fig. To provide an unbiased understanding of macrophage metabolic reprogramming, we used gene expression to model metabolic flux systematically. This revealed a global activation state exclusive to obese macrophages, encompassing known16 and previously unrecognized metabolic changes (1,495 of 1,895 reactions, binomial test, P = 3.1 × 10−148; Fig. Specifically, we found a shift to a high-glycolysis (pro-inflammatory), high-respiratory (anti-inflammatory) profile consistent with extracellular flux analyses in obese mice16; corresponding changes in the pentose phosphate pathway and TCA cycle; pervasive activation of cholesterol, lipid and fatty acid synthesis, and oxidation pathways; obligatory upregulation of cellular transport (Fig. Taking fatty acids as an example, flux modelling uncovered significant activation of fatty acid desaturation (FADS1 and SCD) and mitochondrial β-oxidation (Fig. 2e,f), consistent with buffering and utilization of potentially toxic microenvironmental fatty acids for energy. 2h), establishing that diverse myeloid classes undergo extensive metabolic priming in obese AT. Experimental energetic profiling confirmed the higher basal activity and glycolytic capacity of LAMs over TRMs, substantiating our transcriptome-based flux results (Fig. WL led to marked reductions in myeloid cell numbers (mean from 31% to 18%) across subclasses (Extended Data Fig. Proportional and density analyses showed that myeloid-cell fractions did not differ between obesity and WL (Fig. 2c), and we verified this in situ (Extended Data Fig. WL did, however, shift LAMs towards less inflammatory subtypes (Fig. Overall, this indicated that obesity-induced myeloid cell states persist despite extensive WL. Transcriptomic flux analyses confirmed that global metabolic activation did not fully reverse with WL (Fig. But WL did significantly reduce some aspects of fatty acid synthesis and oxidation (mainly desaturases and acyl-CoA synthetases; Fig. 2e,f), temporally linking these pathways to microenvironmental lipid availability. By contrast, glycolysis, respiratory capacity and pentose phosphate pathway flux increased (Fig. 2e,f), implying a need to requisition energy from other sources as fatty acid levels diminish. Differential expression analyses demonstrated widespread reductions in inflammasome, proinflammatory cytokine and chemotaxis genes (Fig. Network analyses implicated specific transcription factors (TFs) in TRM and LAM specification and revealed patterns reinforcing the finding that WL improves inflammatory and homeostatic networks, but not LAM transcriptional reprogramming (Fig. Together, these results demonstrate a complex activation response in obese AT dominated by monocyte recruitment and persistent metabolic reprogramming. Low overall numbers (6,222 cells (4%); Extended Data Fig. 2l–n) meant that we were unable to evaluate lymphoid subclass-level variations. 17,18); Supplementary Tables 7 and 8), further supporting decreased inflammation. Mature adipocytes undergo profound phenotypic changes in obesity and WL, expanding and shrinking to fit evolving energy needs7. How this affects their molecular characteristics and diverse metabolic functions is largely unclear. Subclustering revealed 8 mature adipocyte subpopulations (AD1–AD8, n = 44,583 cells; Fig. Two subtypes exhibited ‘stressed' (AD3, JUN/NFKBIZ-hi) and ‘fibrotic' (AD6, NOX4/LOX-hi) profiles. Stressed and fibrotic adipocytes increased with obesity, indicating that there is selective vulnerability and pathogenicity to the tissue microenvironment (Fig. Another subpopulation with a lipid biosynthetic profile (AD5, PNPLA3/MOGAT1-hi) unexpectedly decreased in obesity (Fig. WL led to a marked reduction in stressed adipocytes (mean from 55% to 14%), a shift towards lower fibrotic numbers, and relative increases in lipid biosynthetic cells (Fig. Compositional changes in stressed and lipid biosynthetic populations were verified in situ (Extended Data Fig. Beige adipocytes were rare (AD8 GATM-hi, 1%) and invariant between conditions. a, Marker-gene expression in mature adipocyte subpopulations. Reaction level, Cohen's D, coloured by FDR < 0.05 (Wilcoxon): red, OB high; blue, OB low; grey, non-significant; cat., catabolism; syn., synthesis; ox., oxidation. Pie charts show the proportion of significant reactions. d, Scores measuring overall activity in major metabolic pathways in individual adipocytes, averaged by participant (density, median IQR), then compared between conditions. Two-tailed Wilcoxon test unpaired (LN–OB and LN–WL) and paired (OB–WL) FDR-adjusted P-values are shown (n = 24 LN; n = 25 paired OB–WL donors). e, Differential expression of enzymatic genes in lipid and BCAA metabolism pathways in OB compared with LN and WL adipocytes, separated by datasets. Coloured by log2FC: red, OB high; blue, OB low; sized by adjusted −log10 P-value, negative binomial mixed-effects model. Circles represent comparisons with absolute log2FC > 0.3 and adjusted P < 0.05. f, Overall activity in metabolic pathways in adipocyte subpopulations (scaled mean scores). g, Mean expression of mechanosensitive, stress, fibrotic and homeostatic genes across conditions and adipocyte subpopulations, in single nucleus (left) and spatial (middle) datasets (limited to genes in both datasets, nucleus segmentation). Spearman correlation (right) of genes with adipocyte areas in each condition and across all conditions combined (spatial dataset, boundary segmentation). The # denotes rank (high-to-low) across 97 genes (P-value threshold less than 1 × 10−5 in more than one correlation). h, Representative spatial sections showing altered adipocyte sizes (WGA segmented) and JUN (stress marker) expression across conditions. Bottom bars, mean JUN expression and mean log10area in adipocytes across all spatial samples for each condition. Expression-based metabolic flux analyses detected significant defects in fatty acids and branched-chain amino acid (BCAA) breakdown in obese compared with lean adipocytes, mirroring previous results19,20 and together suggesting that metabolic flexibility was impaired (Fig. By contrast, WL led to a marked global increase in adipocyte metabolic flux (1,485 of 1,895 reactions; binomial test, P = 1.4 × 10−142; Fig. Unexpected anabolic activity led us to investigate whether triglyceride mobilization, which is a physiological response to caloric restriction, might initiate lipid cycling (repetitive degradation and resynthesis). To verify flux models, we compared enzymatic activity scores and pathway-limiting enzymes in important substrate pathways across groups21 (Fig. WL systematically increased opposing lipid biosynthesis and breakdown pathways (Fig. Consistent with this, we found significant changes in canonical enzymes in sequential cycle steps (Fig. 3e), including DGAT2, which encodes an acyltransferase that catalyses triglyceride synthesis and mediates lipid cycling in vitro22. Because enzymatic expression is a crucial determinant of catalytic competence, this indicates that WL may initiate triglyceride cycling, a highly bioenergetic process with important lipid-diversifying, toxic fatty acid-quenching metabolic benefits22. WL also reversed defects in BCAA catabolism (pathway flux and canonical enzymes; Fig. 4f), the predicted consequences of which are systemic BCAA clearance and improved insulin sensitivity23. Lipid cycling was a feature of PNPLA3-hi adipocytes (AD5), whereas stressed (AD3) adipocytes were characterized by lower metabolic turnover (Fig. These typical catabolic and previously unrecognized anabolic effects of WL suggest that substrate mobilization engages cell-autonomous cycling pathways that may underlie widespread improvements in metabolic homeostasis. To see which TFs were explicitly responsible for WL-induced metabolic activation, we carried out network analyses limited to metabolic pathway genes (Extended Data Fig. MLXILP and SREBF1 ranked highly in triglyceride synthesis, validating our approach and implicating them in control of WL-induced lipid cycling. Other notable findings were TFs linked to redox biology and BCAA catabolism. Many of the leading TFs (38 of 53, P < 0.05 Bonferroni adjusted, more than 50 metabolic target genes) overlapped human metabolic disease genome-wide association study (GWAS) loci24 (Extended Data Fig. Differential expression analyses identified altered biomechanics as a potential driver of adipocyte stress and metabolic dysfunction that was mitigated by WL. We therefore evaluated whether adipocyte hypertrophy and mechanical strain might initiate these maladaptive changes, and whether adipocyte shrinkage during WL might reverse them. As expected, adipocyte sizes increased in obesity and reduced with WL (Fig. Despite intrasample heterogeneity, adipocyte size correlated positively with mechanosensitive, stressed and fibrotic gene expression and negatively with homeostatic genes (Fig. 3g, exemplified by the stress marker JUN in Fig. The levels of correlation indicated that this may be one of several factors eliciting tissue stress and fibrosis, or perhaps it is a driver event in a degenerative cycle. Adipocyte progenitor cells (APCs) regenerate mature adipocytes and maintain tissue stroma, crucial homeostatic functions that may become impaired in obesity7. APC3 exhibited a stressed profile similar to that observed in mature adipocytes, as well as higher expression of NOCT (Fig. In support of this, APC2 selectively expressed late-stage adipocyte maturation genes within a localized subregion (Extended Data Fig. Both populations had higher expression of hypoxia-inducible factor 1A (HIF1A; Fig. 6a,c), which promotes fibrosis and suppresses adipogenesis in mice (through PPARG phosphorylation)28. Correspondingly, WL downregulated hypoxia, profibrotic (TGFβ) and anti-adipogenic (WNT) genes (Extended Data Fig. Thus, WL may attenuate hypoxia-induced impairment of differentiation competency and profibrotic signalling in certain human APC subpopulations. a, Marker-gene expression profiles in basal and stressed subpopulations of mature adipocytes (AD), precursors (APC), endothelial cells (EC) and mural pericytes (Per.). c, Tissue-wide stress scores (calculated from the 24 common upregulated stress genes present in the spatial dataset, logged score) in representative LN, OB and WL spatial tissue sections, and the mean stress score for each condition in all samples. Top, mean cell state stress score in 50-µm bins. Middle, percentage of cells in stress quantiles, across all conditions, per cell state (Q1 low, Q4 high stress). e, Spatial niches in representative tissue sections. f, Imputed CellChat communication between spatial niches for THBS1 (top) and ADGRE5 (bottom). Links represent the scaled mean probability (line thickness) and directions of connectivity. g, CellChat communication between cell states for THBS1 (left) and ADGRE5 (right) in the single-nucleus dataset, across all conditions. Links represent the scaled mean probability (line thickness) and directions of connectivity. h, Ligand–receptor pathways with significant differential interactions in OB–LN and OB–WL comparisons (tissue-wide, single-nucleus dataset). Separated into reciprocal (significant in both comparisons, top) and other (significant in one comparison, bottom). Infinity (Inf) represents pathways that were present in only one of the conditions. Coordinated growth of the vascular network is essential for healthy AT expansion. Vascular cell subclustering recapitulated the endothelial (arterial, capillary and venous) and mural (smooth muscle and pericyte) zonations observed in other tissue types (Extended Data Fig. 4a), which changed reciprocally with adiposity (Extended Data Fig. Stressed mural cells enriched for ADAMTS1 (Fig. 4a), an anti-angiogenic protein linked to pericyte detachment, fibrotic transition and capillary rarefaction31. In distinct single-nucleus and spatial datasets, WL markedly reduced stressed vascular cell content and markers (Fig. 6a), implying the reversal of this pathological transformation. All stressed cell states upregulated a common gene signature (188 genes; Extended Data Fig. Multicellular stress, although highest in obesity, was a feature of lean tissues, where it increased with age and metabolic dysfunction (Fig. Gene and pathway analysis revealed putative mediators of multicellular stress (hypoxia, mechanical and oxidative stress, Gp130-mediated cytokines, DNA damage and cell cycle arrest; Extended Data Fig. In vitro induction of DNA damage (using Etoposide) recapitulated the in vivo effects on stress marker proteins and impaired ASPC differentiation capacity (Extended Data Fig. WL led to a marked reduction in multicellular stress genes (Extended Data Fig. 6c), overall emphasizing the importance of multicellular stress pathways in tissue injury and repair. We used our spatial datasets to investigate the orientation and impact of stressed cells in tissues. To define the cells most associated with stress signals, we quantified the cellular composition of low- and high-stress regions (50-µm bins; Fig. Stressed cell states were generally enriched in high-stress zonations, apart from stressed capillaries (EC2), which were spread throughout the tissue. We also found a strong association between regions of stress and immune cells, except TRM and NK cells, and an unexpected connection to arterial ECs (EC4; Fig. Although this localized individual cell states to stressed zonations, it did not address the non-random grouping of cells in microenvironmental compartments. To evaluate this, we used spatially resolved proximity enrichment (within 300 µm, to capture adipocytes) to search for tissue niches based on cell state neighbourhoods. This identified five distinct cellular communities, termed arterial, venous, adipocyte, stem and stress niches (Fig. No cell type was niche exclusive, indicating that these patterns reflect tissue gradations. Stem niches were enriched for multipotent ASC/APC1 and homeostatic TRMs. Stress niches were enriched for AD3, APC3, LAMs, other innate (cMono and cDC2) and adaptive (T cells) immune cells, implicating these states in stress induction and/or response. Arterial endothelial cells formed their own niche, associating with stressed precursors (APC3) and stressed mural cells (Mu4). Direct cell–cell colocalization uncovered immune cell proximity to large venous vessels and LAMs (Extended Data Fig. 8d), potentially reflecting extravasation and transmigration to CLS. The identification of tissue zonations enabled us to investigate intra- and inter-niche signalling patterns. Ligand–receptor inference analyses in the spatial dataset revealed a complex network of communications. Adipokines and neurotrophic factors were enriched in the adipocyte niche (ADIPOQ, LEP and NRXN3; Extended Data Fig. Canonical WNT and ECM components (FN1, collagens and laminins) were prominent components of the stem niche (Extended Data Fig. Parallel intercellular communication analyses in the larger single-nucleus dataset linked diverse niche signals to source and target cells, as well as disease pathobiology (Fig. For example, THBS1 (stressed AD3), ADGRE5 (pan-immune) and NAMPT (multicellular), which are emergent triggers of insulin resistance32, immune glycolytic metabolism33 and inflammation34, were all amplified in obesity and reversed by WL (Fig. This showed that stressed niches have a high concentration of signals implicated in pathological and restorative tissue remodelling. Differential expression analyses to define AT remodelling pathways established that WL broadly reverses the effects of obesity on gene regulation (Extended Data Fig. Many of the strongest transcriptional changes associated with WL were conserved across cell types (Extended Data Fig. 9c,d), indicating that these genes and their underlying pathways might represent important WL mechanisms. Genes altered by WL in multiple cell types (three or more cell types; FC > 0.5, P < 0.05 Bonferroni corrected) showed systematic downregulation (213 of 333 genes; binomial test, P = 3.9 × 10−7). Downregulated genes were grouped into hallmark pathways of AT dysfunction: inflammation (TNFA and IFNG); hypoxia; fibrosis; immune cell recruitment and activation; and oxidative stress4,7,26 (Extended Data Fig. WL also led to downregulation of cell cycle arrest genes (Extended Data Fig. 9e), together indicating that the reversal of cellular senescence might underlie the beneficial effects of WL on inflammation and metabolism. To examine this, we tested and confirmed the repression of diverse senescent signatures (Fig. In multiple cell types, WL led to the downregulation of CDKN1A (p21), which is one of the main cell cycle inhibitors in senescence, and the upregulation of cell cycle progression genes repressed by p21 (ref. Correspondingly, WL markedly decreased the expression of principal senescence markers and unbiased senescence scores (Fig. 10e), were most prevalent among stressed adipocyte, precursor and vascular cell states (Extended Data Fig. 10f), indicating that the shared stress profile reflects vulnerability and transition to senescence. Lean AT also contained substantial (albeit significantly lower) numbers of p21-positive cells (Fig. By contrast, WL almost completely eliminated p21-positive cells from the tissue (Fig. 5b), a finding that we verified in situ using spatial transcriptomics (Extended Data Fig. The repression of senescence mirrored enhanced adipocyte bioenergetics, indicating that these effects may be mechanistically coupled (Extended Data Fig. We therefore established that human WL has previously undescribed potent senolytic effects. a, Differences in the expression of cell cycle and senescence marker genes in WL among vulnerable cell types. Representative images of a pair of OB and WL tissue sections stained for p21 (scale bar, 100 µm; arrows depict p21-positive nuclei). d, Differential gene regulatory networks (left) in each cell state (scaled log2FC > 0.5 in one or more state versus all other states in that cell type; Wilcoxon two-tailed, FDR < 0.05) and in LN–OB (middle, dark red OB high) and OB–WL (right, red OB high) comparisons in each cell type (log2FC, Wilcoxon two-tailed, red OB high). Non-significant networks at P > 0.05 Bonferroni corrected are coloured grey. e, A network of TFs conserved across stressed cell states (scaled log2FC > 0.4 in three or more stressed cell states; Wilcoxon, FDR < 0.05), coloured by TF family, sized by number of forward interactions with other TFs, encircled if interaction with self (41 of 41 TFs) and linked by the shared number of target genes (width and colour, Jaccard index). f, Tissue-wide (50-µm bins) expression of SASP components, AREG and CXCL2, in representative spatial tissue sections for each condition. The identified TFs grouped into several classes (Fig. 10i): the AP-1 superfamily, which primes the senescence genome; hallmark signal-dependent TFs that activate inflammation and the senescence-associated secretory phenotype (SASP); Krüppel-Like TFs, which are implicated in cell cycle arrest; TFs that control ciliogenesis (RFX2/RFX3), which is a putative senescence regulator; orphan nuclear receptor TFs that are induced by DNA damage and oxidative stress, key senescence triggers; and multiple candidate TFs not previously linked to senescence36,37,38,39,40. Individual TFs exhibited autoregulatory effects and shared multiple target genes (Fig. 10i), including CDKN1A, indicating that these TFs may cooperate to potentiate a degenerative cycle of cell stress, senescence, SASP release, inflammation and tissue injury. This transcriptional cascade is turned off by WL. Because of its importance in reinforcing senescence, we sought to further define the signatures of the AT SASP by systematically comparing the expression of secretory proteins41 across stressed (high senescence) and basal cell states. Presumptive SASP components systematically replicated in situ and localized to stressed and arterial niches (Fig. Senescent cells strongly upregulated NAMPT, an intracellular driver of the SASP (through enzymatic activity in the NAD salvage pathway) and an extracellular adipocytokine (visfatin) with pleiotropic, context-dependent, predominantly pro-inflammatory effects34,37 (Fig. NAMPT expression was similarly enriched in obese macrophages and inflammatory LAMs (Fig. Tissue-level protein analyses confirmed that NAMPT abundance increased in obesity and reduced markedly with WL (Extended Data Fig. 10b), together highlighting that NAMPT is a likely driver of AT SASP. In summary, these analyses reveal diverse intracellular and extracellular mediators of the degenerative AT senescence cycle and support reversal of AT senescence as a key determinant of the metabolic health benefits of WL. As the number of people living with obesity surpasses one billion, there has never been a greater need to understand the opposing effects of obesity and WL on metabolic health. Here, we report a high-resolution single-nucleus and spatial atlas of human AT in people with extreme obesity undergoing therapeutic WL and healthy lean counterparts. The simultaneous analysis of obesity and WL enables us to understand core tissue remodelling principles; capture more than 20 cell states that vary with body weight (including degenerative and adaptive populations that bridge cell ontologies); and define molecular pathways, regulatory factors and intercellular signals that may drive tissue injury and subsequent recovery. Foremost, we reveal selective susceptibility to cellular stress and senescence in subpopulations of metabolic, precursor and vascular cells, but not in their immune counterparts. This susceptibility is amplified in obesity but is evident in ageing and metabolically unhealthy leanness. We discover that WL has potent mitigating effects on senescence in vulnerable cell types. We predict from transcriptional patterns and previous mechanistic studies43 that this leads to increased metabolic flexibility in mature adipocytes, improved differentiation capacity in precursors and recovery of vascular abnormalities. Deep molecular phenotyping across cell types, cohorts and modalities enables us to define a tightly conserved regulatory network that may elicit and reinforce human AT senescence, putative upstream triggers, key components of the degenerative AT SASP and vulnerabilities such as MCL1 that might be exploited with therapy44. We conclude that reversal of AT senescence may be central to the multiorgan anti-inflammatory and metabolic benefits of human WL. By modelling enzymatic gene expression, we show that WL induces global metabolic activation in mature adipocytes, presumably to release stored fuel to meet energy demands. Two prominent activated pathways are BCAA catabolism and lipid cycling (repetitive triglyceride hydrolysis and resynthesis). Contrary to expectation, both pathways are bioenergetic22, challenging the assumption that WL reduces energy expenditure (at least) in AT45, suggesting that there is compensation elsewhere and warranting further investigation. Irrespective of this, we propose that pervasive activation of substrate turnover in adipocytes has effects on insulin sensitivity and ectopic lipid that may be crucial to the multiorgan metabolic benefits of human WL. Despite these effects, our lean cohort was the healthiest, indicating that other factors are involved. In the immune compartment, we confirm that human obesity leads to monocyte and macrophage infiltration and activation to a LAM phenotype. We extend this LAM phenotype to metabolic dysfunction independent of body weight and uncover gene regulatory mechanisms and metabolic pathways implicated in LAM specification and activation. Consistent with an adaptive–maladaptive spectrum, we find that LAMs exist on a continuum. At one extreme, we define a TLR2–TREM1 inflammatory LAM signature that associates with adiposity and metabolic dysfunction, akin to a proatherosclerotic LAM subtype46. We show that after WL there are marked reductions in monocyte and macrophage (and lymphocyte) infiltration and inflammation pathways, which we predict, even in the absence of classical activation, to be anti-inflammatory. Despite overall reductions, we observe persistence of obesity-induced macrophage activation states that are probably epigenetically programmed47,48. In mice, AT immune and metabolic cell memory is implicated in weight regain and enhanced inflammation48,49. Thus, persistent macrophage activation in human AT may impede complete metabolic recovery, trigger weight regain (a major drawback of all WL interventions) and worsen long-term clinical outcomes9,50. We studied WL in its early phase to define potential driver mechanisms. Because of this, we cannot unravel the respective contributions of negative energy balance, weight change and absolute fat mass to the observed tissue and systemic effects. Other limitations include a focus on people without diabetes, the abdominal subcutaneous depot and surgical WL, variable biopsy methods and incomplete capture of rarer immune cells. Previous studies indicate that AT in different locations has important phenotypic differences that may contribute to variability in WL outcomes8. Thus, we anticipate conserved but also context-dependent adaptations across AT compartments and WL methods. Collectively, our results reveal that WL has significant effects on cellular processes that are known to affect metabolic health and longevity. More broadly, our findings highlight the need for proactive obesity prevention and support the possibility that sustained lifestyle changes could have long-term health benefits mediated through dynamic remodelling of diverse AT cell types. This rich representation of human AT biology and pathophysiology offers a valuable resource for mechanistic and therapeutic exploration. Single-nucleus RNA sequencing (snRNA-seq) was done in subcutaneous abdominal AT from 25 people with obesity before and after marked WL (paired samples) and in 26 healthy lean controls. Two lean samples were removed as described below. Spatial transcriptomics was done in equivalent groups (n = 4 per condition), as were histological cross-validation studies (n = 4–5 per condition). All molecular phenotyping was done after overnight fasting. Primary snRNA-seq data were integrated with previously published snRNA-seq data in whole subcutaneous AT from nine obese and four lean people to increase obese and lean cell numbers and improve cell annotation11. Participant characteristics are provided in Extended Data Table 1. The AT samples were obtained intra-operatively from morbidly obese individuals (BMI > 35 kg m−2) undergoing laparoscopic bariatric surgery (gastric bypass or gastric sleeve) and healthy controls (BMI < 26 kg m−2) undergoing non-bariatric laparoscopic abdominal surgery51. Subcutaneous AT was collected from abdominal surgical incision sites. Follow-up subcutaneous AT samples were collected from the peri-umbilical region using needle biopsy more than 5 months after WL intervention. Participants were unrelated, between 20 and 70 years of age, from a multiethnic background and free from systemic illnesses not related to obesity. People with treated type 2 diabetes were excluded because of the potential effects of hypoglycaemic medications on AT metabolism. Metabolic characteristics were collected at baseline and follow-up. The study complies with all relevant ethical regulations and was approved by the London – City Road and Hampstead Research Ethics Committee, United Kingdom (reference 13/LO/0477). Human tissue validation also used samples from the Imperial College Healthcare Tissue Bank, approved by Wales REC3 to release human material for research (reference 17/WA/0161). The AT nuclei from individual participants were pooled for library preparation and sequencing to increase efficiency, cohort diversity and study power. Pooled samples were separated by condition to avoid cross-over (4–5 samples per pool; a total of 6 pools per group). Sample pools for each experimental group were processed through to sequencing in lean–obese–WL trios to minimize between-group batch effects. For each participant sample, nucleus extraction was done using a modified version of a previously described protocol52. In brief, frozen human AT (about 100 mg) was cut into pieces of less than 0.2 cm and homogenized with 1 ml ice-cold lysis buffer (Tris-HCl 10 mM (Invitrogen, 15567-027), NaCl 10 mM (Invitrogen, AM9760G), MgCl2 3 mM (Invitrogen, AM9530G), 0.1% NP40 (BioBasic, NDB0385), 0.2 U µl−1 RNase inhibitor (Roche, 03335402001)) in a glass dounce homogenizer (Merck, T2690/P0485/P1110, 15 strokes, loose then tight pestles) on ice. After homogenization, samples were transferred through a 100 µM cell strainer (Greiner Bio-One, 542000) into a prechilled tube using ART wide-bore tips (Thermo Scientific, 2079 G). The filtered homogenate was then transferred to 1.5 ml low DNA-bind tubes (Sarstedt, 72.706.700) and centrifuged at 500g and 4 °C for 5 min. After lipid/supernatant removal, the nuclei pellet was resuspended in 1 ml wash buffer (PBS with 0.5% BSA (Invitrogen, AM2616) and 0.2 U µl−1 RNase inhibitor), transferred to new 1.5 ml low DNA-bind tubes and recentrifuged at 500g and 4 °C for 5 min. After repeat lipid/supernatant removal, the nuclei pellet was resuspended in 300 µl wash buffer containing DAPI (Thermo Scientific, 62248) at 0.1 µg ml−1 to stain nuclei, and filtered through a 35 µM cell strainer into a fluorescence-activated cell sorting (FACS) tube (Falcon, 352235) on ice. At this point, the isolated nuclei from 4–5 samples from the same experimental group were pooled before sorting by flow cytometry. FACS was used to clean up residual debris and lipid from isolated nuclei and to remove doublets. Pooled nuclei were sorted on a BD FACS Aria SORP. The sheath tank was bleach cleaned before each run and nuclease-free PBS (1×) (Invitrogen, AM9625) was used as sheath fluid. A 405 nm laser was used to excite DAPI, and emission was collected using a 450/50 nm bandpass filter. Single nuclei were selected by gating on the first DAPI-positive band on the DAPI versus forward scatter (FSC) plot and then subsequently gating on side scatter (SSC) versus FSC and FSC A versus FSC H to ensure better debris and doublet removal. All sorts were performed using an 85 μm nozzle. The sorted nuclei were collected into a BSA- and RNase inhibitor-rich collection buffer (70 µl of PBS with 1.375% BSA and 2.15 U µl−1 RNase inhibitor) in low DNA-bind tubes kept at 4 °C. Supernatant was removed to leave about 40 µl, which was used to resuspend pellets with a wide-bore pipette tip. For each pool, 5,000–10,000 nuclei were co-encapsulated with 10x barcoded gel beads to generate gel beads in emulsion (GEMs) using a 10x Chromium Controller and a 10x Genomics Single Cell 3′ v.3.1 kit, according to the manufacturer's instructions. After GEM-RT and clean-up, the quantity and fragment size distribution of amplified cDNAs derived from barcoded single-cell RNAs were assessed using an Agilent 2100 Bioanalyzer High Sensitivity DNA assay. From this cDNA, snRNA-seq libraries were constructed and sequenced (Illumina NextSeq2000) in three batches, containing equal numbers of obese, lean and control library pools, to minimize between-group batch effects. Each unique library was sequenced to a minimum depth of more than 20,000 paired-end reads per nucleus (read 1, 28 base pairs (bp) and read 2, 90 bp, with unique dual 10-bp indexes). Libraries were demultiplexed using CellRanger mkfastq based on the sample indices (allowing one mismatch), and the CellRanger count pipeline was used to perform alignment against human genome GRCh38 (using STAR), filtering and counting unique molecular identifiers (UMIs) (including introns). For each pooled library, raw count matrices from CellRanger were processed using CellBender53 (--epochs 150-200, --learning-rate 0.0001-0.00005) to remove ambient RNA molecules and random barcode swapping, and filter inferred cells. The number of expected cells was based on CellRanger estimations. Filtered count matrices were processed separately using Seurat54 and SeuratObject. Low-quality cells with low read or gene counts (less than 1,000 UMIs or less than 400 genes), low complexity (log10(genes per UMI) < 0.85) and high mitochondrial or ribosomal fractions (greater than 5%) were removed from each pooled dataset. Clean libraries were normalized and transformed (sctransform v.2 regularization55) to stabilize count variances. Potential doublet nuclei were detected using three approaches: expression-based DoubletFinder56, using doublet estimates from genotyping to set the expectation; genotype-based, Vireo57 (details below); and iterative clustering and detection of clusters with high expression or genotype-based doublet fractions. Assigned doublets, ambiguous cells and doublet clusters were then removed and singlet-only datasets were retransformed. Participant-level annotation information from genotyping was then added to generate high-quality cell datasets. Genotype information present in the RNA sequencing reads was aligned to existing genome-wide genotyping to attribute specific cells to specific participants in each sample pool. Participant-level genotype data were generated from whole blood using Illumina Infinium OmniExpress-24 v.1.2 bead chips. Directly genotyped single-nucleotide polymorphisms (SNPs) with call rates of less than 90%, minor allele frequency of less than 0.01, Hardy–Weinberg equilibrium P < 1 × 10−6, SNPs on sex chromosomes and duplicated SNPs were removed. After quality control, 649,007 SNPs were taken forward for imputation. SHAPEIT58 (v.2.r900) was used to infer haplotypes, and imputation was done in IMPUTE2 (v.2.3.2)59 using a 1,000 genomes reference panel phase 3 (all ancestries). Each chromosome was divided into 5-megabase chunks for imputation and merged at the end. An effective population size (Ne) reflecting genetic diversity was 20,000, as recommended when using a multi-population reference panel. After imputation, genotype data were available for 81,656,368 SNPs. Cell-level SNP data were generated for each pooled sample using cellsnp-lite60 (using the combined imputed SNP list as the reference). Cell-level SNP data were then intersected with participant-specific genotype references in Vireo57 to identify variants that segregated the samples, and we used these variants to demultiplex participant specific cells, ambiguous cells and doublets. A range of cellsnp-lite MAF settings were tested and MAF > 0.05 was selected to maximize singlet recovery. High-quality, doublet-removed cell libraries containing participant-level annotations were then integrated to a unifying atlas. Two samples, one with very high lymphocyte counts and one with very few cells, were removed at this stage, leaving 24 samples in the lean group. A further 13 whole subcutaneous AT samples from obese and lean people in a previously published dataset11 were also incorporated in the integration phase to increase cohort diversity and improve cell annotation. Of note, only samples meeting the following criteria were selected: whole tissue; nucleus only; subcutaneous depot; and BMI < 26 or BMI > 30 kg m–2. Previously published samples were individually reprocessed from raw counts using thresholds equivalent to our own datasets. Both datasets were then normalized to 10,000 counts per nuclei before proceeding with downstream analysis. To minimize any sample-driven effect for cell-type identification, we took a three-step approach. Then we calculated the PCA space on the highly variable genes, detected by Scanpy61, followed by correction of the PCA space with Harmonypy62 using samples as batches. Finally, we used BBKNN63 with samples as a batch to identify neighbourhoods. Cell type and state annotation was done in the combined (our own and that from ref. Primary exploratory analyses were performed in our own dataset, which was processed in experimental group trios (lean–obese–WL) to minimize batch effects and comprised paired obese–WL samples and age-, sex- and ethnicity-matched lean controls. Differential neighbourhood abundance and expression analyses between groups (in which biological, technical and batch covariates could be adjusted for) were repeated using the combined dataset to verify reproducibility. We identified the main cell types with unbiased clustering, using a low-resolution (0.15) Leiden algorithm, and each cell type was annotated according to known markers. To identify cell states, we isolated the barcodes for each of the main cell type identities, except for mast and lymphatic endothelial cells, owing to low numbers. Each cell type was then reintegrated and reclustered twice, as described above. First, we used a high-resolution Leiden (1.2 or higher) to identify barcodes that contained a mixed signature, with markers of different lineages. Then, we removed these barcodes and proceeded with the second round of reintegration and clustering. Resolution varied across cell types (0.65 or higher), with myeloid cells requiring the highest Leiden to identify rare, known cell types (2.25). Clusters that were similar to each other and had no unique identifiable features between them were merged. Cell states were annotated based on a mix between unbiased and known markers. We performed comparisons of lean–obese and WL–obese groups, adjusting for biological covariates in the lean–obese analyses. Neighbours were recalculated with BBKNN using samples as a batch, restricted to the comparison groups (lean–obese and WL–obese). To analyse global shifts, we used Milo on all cell types together and within each cell type to analyse shifts in cell state composition. Only neighbourhoods containing at least 90% of a single cell type or state were considered neighbourhoods, and those with a spatial FDR < 0.1 were considered significant. Fasting insulin adjusted for BMI abundance analyses were carried out in steady state lean and obese samples, using lean–obese neighbourhoods, adjusting for biological covariates. The metabolic profiles of different cells were inferred using flux-based analysis modelling in COMPASS65. For this, we created an expression matrix for every main cell type, consisting of the mean expression of each gene per sample. These matrices were then used to run COMPASS. Statistical analysis to compare conditions was performed with a Wilcoxon test for every reaction, using their COMPASS score. COMPASS plots consisted of both positive and negative reactions grouped by their defined subsystem. Differential expression analyses were carried out between obese cases and controls, and between obese–WL pairs, in Nebula66 using negative binomial mixed-effect models to correct for subject- and cell-level correlation structure. In all comparisons, further thresholding was applied (mitochondrial fraction less than 1% and ribosomal fraction less than 1%) to minimize false discovery, and fractions of mitochondrial and ribosomal counts were incorporated as technical covariates; in obese–lean comparisons, age, sex and ethnicity were included as covariates; in obese–WL comparisons of paired samples, biological covariates were not included. Statistical significance was inferred at P < 0.05 Bonferroni corrected for obese–WL pairwise comparisons (where power was higher) and FDR < 0.01 for lean–obese comparisons. Cell type and state differences were examined using Scanpy's rank_gene_groups function to perform a Wilcoxon test, as were spatial differences in gene expression within cell types between conditions. Amphiregulin (AREG), which is known to be secreted67, was added to the curated secretory protein list from the Human Protein Atlas41 for comparisons in stressed and basal cells. The expression matrix used consisted of nuclei from all 3 conditions, downsampled to the same number of nuclei (20,000 each). Genes that were expressed in all nuclei, or in less than 5% of nuclei for any given cell state, as well as mitochondrial, ribosomal, haemoglobin, non-coding, antisense, contig and microRNA genes, were also removed from the analysis. For TF binding sites, we used the Encode 2019/06/21 ChIP-seq hg38 refseq-r80 10 kilobases up and down database. Only regulons with a minimum of five target genes were considered. Analyses in adipocytes were restricted to all TF genes and genes in dysregulated metabolic pathways from COMPASS. Differential networks between cell states and within cell types between conditions were identified by comparing cell-level network scores between groups (non-parametric Wilcoxon rank-sum test). Significance was inferred at P < 0.05 (Bonferroni corrected). Within a cell state, fold changes were scaled for visualization. We used CellChat69 to infer intercellular communication, based on known receptor–ligand interactions. For the purpose of this analysis, to compare the differences between each condition, cellular communication was inferred for each condition separately. Each condition was down-sampled to 20,000 barcodes to avoid any confounding effects arising from higher cell numbers in obese and lean groups, and cell types with very low numbers were removed because these cell types often have higher mean gene expression owing to low cluster background. To analyse the differential communication between two conditions, we used the rankNet function in CellChat to obtain overall signalling differences, as well as pairwise comparison with each cell type as a sender and as a receiver. To analyse communication at the cell state level, we performed a condition-agnostic analysis to maintain cell states with low numbers of nuclei. For intra- and inter-niche communication analyses, because of the lack of most ligand–receptor pairs in the Xenium gene panel, we imputed spatial data using ENVI70. This was done for each condition separately, training on the single-nucleus data for each condition. We did this step ten times and averaged the results in a final imputed expression matrix because of the stochastic nature of imputation. Imputed genes with low expression (below the mean across all genes, the gene-level quality control) and those with below the mean for that gene (cell-level quality control) were removed. Gene list scoring was done in Scanpy using the score_genes function, with the normalized ln expression and a control size of 50. Housekeeping genes were obtained from the 20 most stable human transcripts in the Housekeeping Transcription Atlas73, supplemented with commonly used housekeeping genes (RRN18S, ACTB, GAPDH, PGK1, PPIA, RPL13A, RPLP0, ARBP, B2M, YWHAZ, SDHA, TFRC, GUSB, HMBS, HPRT1 and TBP). Pathway analyses of differentially expressed genes were done in ClusterProfiler74 using the Over Representation Analysis and MSigDB71,72 datasets (H, C2 and C5) as inputs. All genes present in the comparison datasets were used as background. Significant pathway enrichment was inferred at FDR < 0.01. Frozen stored AT samples (stored at −80 °C) were directly thawed in a 4% paraformaldehyde solution and kept at 4 °C for 24 h. Samples were then transferred to a 70% ethanol solution and stored until paraffin embedding. Ethanol-dehydrated samples were cleared with xylene, infiltrated with molten wax using the Sakura Tissue Tek VIP6 vacuum infiltration processor and embedded in paraffin using the Sakura Tissue Tek TEC5 embedding system. Formalin-fixed paraffin-embedded (FFPE) blocks were stored at 4 °C. Xenium slides stored at −20 °C were equilibrated to room temperature for 30 min before sectioning. The FFPE blocks were rehydrated in an ice bath with distilled water for 10–30 min and sectioned at 5 µm thickness. Sections were floated in a 42 °C water bath and slides containing tissue sections were incubated at 42 °C for 3 h and then dried overnight at room temperature in a desiccator. Slides were kept at 4 °C in a desiccator until use. All histology was done in RNase-free conditions using sterilized equipment. A technical pilot was done on a single frozen stored AT sample separated into three sections for fixation at 24 h, 48 h and 72 h to evaluate the effects on tissue integrity (H&E) and transcript recovery using the 10x Xenium Human Multi-Tissue and Cancer Panel (P/N 1000626), with two slides and one tissue section for each fixation time/slide (Institute of Developmental and Regenerative Medicine (IDRM), Oxford). A 10x Xenium Human Multi-Tissue and Cancer Panel (P/N 1000626) supplemented by 100 custom genes was selected to annotate prominent cell types, states and effector pathways identified in single-nucleus datasets. The FFPE tissues were analysed on a 10x Xenium Analyser following 10x Genomics Xenium in situ gene expression protocols CG000580, CG000582 and CG000584. In brief, 5-µm FFPE tissue sections on Xenium slides were deparaffinized and permeabilized to make the mRNA accessible. Gene panel probes were hybridized for 20 h overnight followed by washing, ligation of probe ends to targeted RNAs, generating circular DNA probes with high specificity. Rolling circle amplification was used to generate hundreds of copies of gene-specific barcodes for each RNA-binding event, resulting in a strong signal-to-noise ratio. Background fluorescence was quenched chemically to mitigate tissue auto-fluorescence. Tissues sections were stained with DAPI nuclear stain and Xenium slides were loaded onto the Xenium instrument for imaging and then decoding of image data to transcripts. Secondary analysis to segment cells and assign transcripts was performed on-instrument (Xenium Analyser v.1.7.1.0). Xenium Explorer was used to evaluate the initial data quality and visualize morphology images, transcript localization at subcellular resolution, segmentation and data clustering. After Xenium in situ transcriptomics, slides were kept in PBS and stored at 4 °C for up to 24 h. For immunofluorescence staining, slides were washed three times in PBS for 5 min and then incubated in CF 594 wheat germ agglutinin (1:200; Biotium, 29023-1) for 20 min. Slices were then rewashed three times with PBS, and tissue stained with DAPI (1:5,000; Thermo Scientific, 62248) for 10 min at room temperature. Finally, sections were rewashed as before and then mounted using antifade medium Vectashield (Vector Laboratories, H-1000). Full slide scans for the immunofluorescence channels were performed at 20× magnification using a ZEISS Axio Scan.Z1 slide scanner. Xenium data were analysed by three different methods, depending on the purpose of the analysis. Regardless of the type of analysis, only transcripts with a quality value higher than 35 were considered. To plot transcript and score densities, regardless of cell type we took a segmentation-free approach creating 50-µm bins using the transcript coordinates provided by Xenium. Only bins that contained more than ten transcripts were kept for downstream analysis. For cell-type identification, we took the nucleus segmentation from Xenium and assigned only transcripts within 2 µm of each nucleus (selected to maximize recovery of transcripts but minimize the capture of known cross-contaminating marker transcripts from adjacent cells, designated nucleus segmentation). The resulting matrices were then imported into Scanpy for analysis. Here, only nuclei with more than 40 transcripts were kept for downstream analysis. Clustering was performed similarly to the single-nucleus data, with Harmonypy62 and BBKNN63 used to correct batch effects in the PCA and neighbourhoods, respectively. However, here gene expression was scaled using Scanpy's61 scale function to give more weight to low-expression genes. A low-resolution Leiden algorithm was then used to identify the main cell types, and cell states were identified by reintegrating and reclustering each of these cell types individually. Clusters were labelled to match the single-nucleus reference. To delineate rarer LAM subtypes in the spatial dataset we used CellTypist for label transfer75, creating a model trained on the single-nucleus LAM subtypes and applying a ‘best match' prediction on the MYE2 LAM spatial cluster. To correlate genes with adipocyte size, we performed a semi-manual segmentation using ImageJ, designated boundary segmentation. WGA staining, performed after the Xenium run, was aligned to the Xenium data using the DAPI channel as a guide and utilized for segmentation. To avoid any issues arising for multiple adipocytes being merged in the segmentation, we manually closed some gaps where the WGA staining was not strong enough to be detected by the binary threshold of ImageJ. We then used the Analyse particles function of ImageJ to detect each object and measure the area and centroid coordinates. Furthermore, we created a separate table with coordinates for each pixel contained in each object. We then created a distance tree between the remaining transcript coordinates and the pixel coordinates obtained for every ImageJ object. This was achieved using the KDTree function from Scipy's spatial module. Adipocyte transcripts that were found on the cell boundary were assigned to the closest adipocyte(s) (any adipocyte within 2 µm of the nearest segmented pixel). Only objects with an area greater than 1,000 µm2 and less than 25,000 µm2 were considered as adipocytes for this analysis. As larger objects were found to have higher probability of capturing more transcripts, gene expression was normalized to the total number of counts per cell. Clustering was done as described above, using a high resolution to identify and then remove fine clusters containing contaminating transcripts from other cell types. A Spearman correlation was done to investigate which genes correlated with adipocyte area. Finally, to cluster cells in spatial niches, we made use of Scipy's KDTree function to create a distance tree between every cell in each sample. We then created a neighbourhood matrix by counting, for each cell, the number of proximate cells (within 300 µm) at a cell state level. Because adipocyte sizes increased in obesity, cells in lean samples had roughly twice the number of neighbouring cells that cells in obese samples did. To prevent this from biasing the niche clustering, the neighbourhood matrix was normalized such that each cell was represented by the percentage of neighbouring cells in each cell state. To cluster cells into niches, we created an anndata object of the neighbourhood matrix for use in Scanpy and corrected for batch effects with Harmony and BBKNN before Leiden clustering. Very similar clusters, driven by small fluctuations, were merged into the AD niche. The FFPE blocks were sectioned at 5 µm thickness for immunohistochemistry and immunofluorescence. Sections were deparaffinized and hydrated, and then heat-mediated antigen retrieval was done in an EDTA-based pH 9.0 solution. Endogenous peroxidase was quenched with 3% hydrogen peroxide. Sections were incubated with rabbit monoclonal to p21 Waf1/Cip1 (1:50 dilution; Cell Signalling, 2947, clone 12D1), followed by anti-rabbit IgG conjugated with polymeric horseradish peroxidase linker (25 μg ml−1; Leica Bond Polymer Refine Detection, DS9800). DAB was used as the chromogen and the sections were then counterstained with haematoxylin and mounted with DPX. Immunohistochemistry was performed on a Leica BOND RX. To evaluate p21-positive cells, full virtual slide scans were loaded into QuPath 0.5.1 (ref. 76) and the positive cell detection module was used to count the total haematoxylin and DAB-positive nuclei in two slices per sample. The fraction of p21-positive cells relative to the total cell number was then calculated for each slice, and the mean was used for between-group analyses. Tissue sections of 5 µm were deparaffinized by submerging three separate times in Histoclear (National Diagnostics, HS-200) for 5 min and then rehydrated by submerging in a series of graded alcohol solutions of decreasing concentrations for 5 min each. After rehydration, antigen retrieval was done by heating the samples in 10 mM sodium citrate buffer, pH 6 (Abcam, ab64236) for 5 min in a decloaking chamber (Biocare Medical, DC2012-220V). After blocking, sections were incubated in primary antibody solutions diluted in 0.5× block ACE at 4 °C overnight: anti-NAMPT (1:200, Affinity Biosciences, DF6059); anti-TREM2 (clone D8I4C, 1:400, Cell Signalling, 91068); or anti-TLR2 (clone TL2.1, 1:400, Invitrogen, 14-9922-82). After primary antibody removal, the tissue was washed in PBS and then incubated with secondary antibody, goat anti-rabbit Alexa Fluor 488 (1:200, Invitrogen, A11034), donkey anti-rabbit Alexa Fluor Plus 488 (1:250, Invitrogen, A32790) or goat anti-mouse Alexa Fluor Plus 647 (1:250, Invitrogen, A32728) in 0.5× block ACE for 45 min at room temperature. For NAMPT, sections were incubated with DyLight 594 Lycopersicon Esculentum Lectin (1:250, Invitrogen, L32471) for 20 min (room temperature), rewashed with PBS and then stained with a DAPI solution (1:5,000, Thermo Scientific, 62248) for 10 min at room temperature. For TREM2/TRL2 at CLS, only DAPI was used. Finally, sections were washed and mounted using antifade medium Vectashield (Vector Laboratories, H-1000). For each sample, representative images were taken at 40× magnification (NAMPT) or 20× (CLS) using a Leica SP8 DLS confocal microscope. Image analysis was done in QuPath 0.5.1 (ref. Measurement precision was evaluated between two images per sample (to confirm low within-sample variability) and the mean sample intensity was used for between-group analysis. We used a modified SCENITH-based approach to evaluate human macrophage metabolic pathways ex vivo77,78. Fresh subcutaneous AT was cut into approximately 2-mm pieces with 30 ml HBSS (Gibco, 14175-053) in a 50 ml tube, washed and collected using a 100 µM cell strainer. Tissue was digested for 20 min at 37 °C with 3 mg ml−1 collagenase II (Sigma C6885) in methionine-free RPMI (Sigma, R7513), 65 mg l−1 l-cystine dihydrochloride (Sigma, C6727), 1× GlutaMAX (Gibco, 35050061), 10% dialysed fetal bovine serum (FBS, Gibco, A3382001). Digested tissue was filtered through a 100 μm strainer and digestion was terminated by adding methionine-free RPMI containing 10% FBS, followed by centrifugation (300g at 4 °C for 7 min). After resuspension in methionine-free RPMI (65 mg l−1 cystine, 10% FBS, 1× glutamax), cells were plated (160 µl) into wells on a 96-well V-bottomed plate. Cells were methionine starved for a further 15 min (total starvation of 45 min including digestion and isolation) before treatment with inhibitors or control media (40 µl) for 15 min. The four treatments were medium, 2-deoxy-d-glucose (2-DG; 100 mM final concentration; Sigma, D8375), oligomycin (2 µM final concentration; Sigma, 495455) and 2-DG plus oligomycin (100 mM and 2 µM final concentration, respectively). Homopropargylglycine (HPG; Cayman Chemical, 11785) was then added to wells at a final concentration of 500 µM and incubated for 30 min to initiate cell HPG uptake. An additional well received cells and media but no HPG and no treatment (click chemistry negative control). After HPG uptake, cells were stained with zombie aqua live/dead stain (1:500 in PBS; BioLegend, 423101) for 20 min at room temperature in the dark, washed with PBS and then fixed with 2% PFA for 15 min. Fixed cells were permeabilized (0.1% saponin and 1% BSA in PBS) for 15 min, washed with click buffer (100 mM Tris-HCl, pH 7.4; Invitrogen, 1556-027) and incubated with Fc receptor blocker (25 µg ml−1 in PBS; Fc1, BD Biosciences, 564765) for 10 min. Cells were rewashed and incubated in 100 µl of click reaction mix in the dark at room temperature for 30 min. After click chemistry exposure, cells were washed using FACS buffer (PBS, 1% BSA, 5 mM EDTA, 25 mM HEPES) and stained with antibody mix (FACS buffer, anti-CD45 FITC (1:20; H130; BioLegend, 304006), anti-FOLR2 APC (1:20; 94b/FOLR2; BioLegend, 391705), anti-CD9 APC-fire (1:20; H19α; BioLegend, 312114), Fc block reagent (25 µg ml−1)) at 4 °C in the dark for 30 min. After rewashing, cells were filtered (35 µM cap strainer) for FACS analysis. Spectral flow cytometry was done on a Sony ID7000 in standardization mode. The ID7000 software was used to calculate distinct spectral signatures for each fluorochrome based on single stained controls. Fluorochrome signatures, together with autofluorescence signatures identified in unstained aliquots of each sample using the AF finder software feature, were used to unmix the signals in fully stained samples with the built-in WLSM algorithm. Unmixed signals were used for gating (Extended Data Fig. 1) and analysis of median fluorescence intensity in FlowJo. Immortalized human adipose-derived stromal cells (Bmi-1/hTERT, iHASC) were acquired from Applied Biological Materials (T0540). For differentiation experiment cells, iHASC were seeded in six-well plates. Differentiation was induced at confluence using growth medium (DMEM/F-12 (Gibco, D8437), 10% FBS (Gibco, F7524), 2 ng ml−1 rhbFGF (Z101455), 1% gentamicin (G255)) supplemented with 10 µg ml−1 insulin (Actrapic, Novo Nordisk), 500 µM 3-isobutyl-1-methylxanthine (Sigma, I5879), 1 µM dexamethasone (Sigma, D4902) and 2 µM rosiglitazone (Sigma, R2408) for 15 days. Etoposide (Sigma-Aldrich, E1383) was used to induce the DNA damage stress response79. From day 1 to day 5 of differentiation, cells were treated with DMSO (Fisher-Scientific, BP231100) (control) or etoposide 5 µM or 10 µM. For stress-marker experiments, undifferentiated cells were seeded in 96-well plates and treated with DMSO control or etoposide (5 µM and 10 µM) at 80% confluence. O-Red-oil (ORO) staining was performed as previously described51. After washing, stained cells were imaged on an Evos m7000 (Thermo Scientific) capturing a minimum of 100 fields at 20× magnification per well. Marker quantification was done in Qupath; nuclear segmentation was done using the cell-detection module in the DAPI channel. Mean ORO intensity was quantified in a 15 µm radius to each nucleus. Positive cells were called empirically at a threshold greater than 32.2, 8-bit depth. For stress-marker quantification, after etoposide and media treatment, 96-well plates were fixed in 10% formalin for 10 min and then washed with PBS. The following primary antibodies were used for staining: anti-STAT3 (clone 124H6, 1:500; Cell Signalling, 9139S) and anti-JUN (clone 60A8, 1:500; Cell Signalling, 9165S). Otherwise, staining procedures used the same steps, reagents and concentrations as for tissue immunofluorescence. After staining, wells were kept in PBS and imaged using a high-throughput fluorescent microscope IN Cell Analyzer 2500HS (Cytiva, objectives 20× for JUN and 40× for STAT3). Positive cells were determined using IN Carta image analysis software (v.1.14), based on the nuclear fluorescence intensity for the target protein (empirical positive threshold for JUN, greater than 396.9, and STAT3, greater than 505.3, 16-bit depth). Unless otherwise stated, significance was inferred at P < 0.05 for single-variable tests and FDR < 0.05 for multiple-hypothesis tests. For spatial datasets, where representative images are provided, all analyses were repeated in n = 4 samples per group. For histological verification, where representative images are shown, all analyses were repeated in n = 4–5 samples per group. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw single-cell and spatial transcriptomic datasets have been deposited on the Gene Expression Omnibus (accessions GSE295708 and GSE295862, respectively). Integrated single-nucleus and Xenium objects, together with auxiliary files, are available at the Single Cell Portal (accessions SCP3116 and SCP3117, respectively). The following publicly available datasets were used in this study: human AT single-nucleus transcriptomic data (Single Cell Portal, SCP1376; and GEO accession, GSE176171); human reference genome (https://cf.10xgenomics.com/refdata-gex-GRCh38-2020-A.tar.gz); Molecular Signatures Database (https://www.gsea-msigdb.org/gsea/msigdb/); secreted proteins in the Human Protein Atlas (https://www.proteinatlas.org/humanproteome/tissue/secretome); motifs for SCENIC (https://resources.aertslab.org/cistarget/databases/homo_sapiens/hg38/refseq_r80/tc_v1/gene_based/); and human GWAS (https://www.ebi.ac.uk/gwas/). Source data are provided with this paper. Data analysis pipelines used in this work can be obtained from https://github.com/WRScottImperial/WAT_single_cell_analysis_Nature_2024. Lean, M. E. et al. Primary care-led weight management for remission of type 2 diabetes (DiRECT): an open-label, cluster-randomised trial. Look AHEAD Research Group et al.Association of the magnitude of weight loss and changes in physical fitness with long-term cardiovascular disease outcomes in overweight or obese people with type 2 diabetes: a post-hoc analysis of the Look AHEAD randomised clinical trial. Blüher, M. et al. New insights into the treatment of obesity. Klein, S., Gastaldelli, A., Yki-Järvinen, H. & Scherer, P. E. Why does obesity cause diabetes? Phelps, N. H. et al. Worldwide trends in underweight and obesity from 1990 to 2022: a pooled analysis of 3663 population-representative studies with 222 million children, adolescents, and adults. & Wadden, T. A. Mechanisms, pathophysiology, and management of obesity. Sakers, A., Siqueira, M. K. D., Seale, P. & Villanueva, C. J. Adipose-tissue plasticity in health and disease. Sandforth, A. et al. Mechanisms of weight loss-induced remission in people with prediabetes: a post-hoc analysis of the randomised, controlled, multicentre Prediabetes Lifestyle Intervention Study (PLIS). Swarbrick, M. M. A lifetime on the hips: programming lower-body fat to protect against metabolic disease. Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Hildreth, A. D. et al. Single-cell sequencing of human white adipose tissue identifies new cell states in health and obesity. Amano, S. U. et al. Local proliferation of macrophages contributes to obesity-associated adipose tissue inflammation. & Sancho, D. Metabolism of tissue macrophages in homeostasis and pathology. Muthusamy, N., Barton, K. & Leiden, J. M. Defective activation and survival of T cells lacking the Ets-1 transcription factor. Kurowska, M. et al. Terminal transport of lytic granules to the immune synapse is mediated by the kinesin-1/Slp3/Rab27a complex. Arner, P. et al. Dynamics of human adipose lipid turnover in health and metabolic disease. & Adams, S. H. Branched-chain amino acids in metabolic signalling and insulin resistance. Sharma, A. K. & Wolfrum, C. Lipid cycling isn't all futile. Sharma, A. K., Khandelwal, R. & Wolfrum, C. Futile cycles: emerging utility from apparent futility. Yoneshiro, T. et al. BCAA catabolism in brown fat controls energy homeostasis through SLC25A44. The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. Horii, Y. et al. VGLL3 is a mechanosensitive protein that promotes cardiac fibrosis through liquid–liquid phase separation. Importance of the microenvironment and mechanosensing in adipose tissue biology. Identification of an adipose tissue-resident pro-preadipocyte population. Shao, M. et al. Pathologic HIF1α signaling drives adipose progenitor dysfunction in obesity. The vascular gene Apold1 is dispensable for normal development but controls angiogenesis under pathological conditions. Mahmoud, M. M. et al. Shear stress induces endothelial-to-mesenchymal transition via the transcription factor Snail. Schrimpf, C. et al. Pericyte TIMP3 and ADAMTS1 modulate vascular stability after kidney injury. Varma, V. et al. Thrombospondin-1 Is an adipokine associated with obesity, adipose inflammation, and insulin resistance. The expression profile and tumorigenic mechanisms of CD97 (ADGRE5) in glioblastoma render it a targetable vulnerability. & Yadav, U. C. S. Adipokine visfatin's role in pathogenesis of diabesity and related metabolic derangements. Martínez-Zamudio, R. I. et al. AP-1 imprints a reversible transcriptional programme of senescent cells. Krones-Herzig, A., Adamson, E. & Mercola, D. Early growth response 1 protein, an upstream gatekeeper of the p53 tumor suppressor, controls replicative senescence. Ma, X. et al. A stress-induced cilium-to-PML-NB route drives senescence initiation. Hsieh, P. N., Sweet, D. R., Fan, L. & Jain, M. K. Aging and the Krüppel-like factors. Uhlén, M. et al. Tissue-based map of the human proteome. Saul, D. et al. A new gene set identifies senescent cells and predicts senescence-associated pathways across tissues. Smith, U., Li, Q., Rydén, M. & Spalding, K. L. Cellular senescence and its role in white adipose tissue. Troiani, M. et al. Single-cell transcriptomics identifies Mcl-1 as a target for senolytic therapy in cancer. Changes in energy expenditure resulting from altered body weight. Dib, L. et al. Lipid-associated macrophages transition to an inflammatory state in human atherosclerosis, increasing the risk of cerebrovascular complications. Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. Hinte, L. C. et al. Adipose tissue retains an epigenetic memory of obesity after weight loss. Cottam, M. A., Caslin, H. L., Winn, N. C. & Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. Bangalore, S. et al. Body-weight fluctuations and outcomes in coronary disease. McAllan, L. et al. Integrative genomic analyses in adipocytes implicate DNA methylation in human obesity and diabetes. & Kessenbrock, K. Single nuclei RNA sequencing of breast adipose tissue (10x nuclei-seq). Fleming, S. J. et al. Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. & Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. & Zagury, J.-F. A linear complexity phasing method for thousands of genomes. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. & Huang, Y. Cellsnp-lite: an efficient tool for genotyping single cells. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Polański, K. et al. BBKNN: fast batch alignment of single cell transcriptomes. Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. He, L. et al. NEBULA is a fast negative binomial mixed model for differential or co-expression analysis of large-scale multi-subject single-cell data. Aibar, S. et al. SCENIC: single-cell regulatory network inference and clustering. Inference and analysis of cell-cell communication using CellChat. The covariance environment defines cellular niches for spatial inference. Extending support for mouse data in the Molecular Signatures Database (MSigDB). Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Hounkpe, B. W., Chenou, F., de Lima, F. & Paula, E. V. D. HRT Atlas v1.0 database: redefining human and mouse housekeeping genes and candidate reference transcripts by mining massive RNA-seq datasets. clusterProfiler: an R package for comparing biological themes among gene clusters. Conde, C. D. et al. Cross-tissue immune cell analysis reveals tissue-specific features in humans. Bankhead, P. et al. QuPath: Open source software for digital pathology image analysis. Argüello, R. J. et al. SCENITH: a flow cytometry-based method to functionally profile energy metabolism with single-cell resolution. Heieis, G. A. et al. Metabolic heterogeneity of tissue-resident macrophages in homeostasis and during helminth infection. Singh, V. et al. Quantification of single-strand DNA lesions caused by the topoisomerase II poison etoposide using single DNA molecule imaging. This work was funded by the Medical Research Council UK (MR/K002414/1, MC_UP_1605/7; to W.R.S. We thank J. Gil for insights into senescence pathobiology and detection. Antonio M. A. Miranda, Liam McAllan, Guianfranco Mazzei, Ivan Andrew, Meryem Ertugrul, Sophie Newton, Alice Pollard, Mikyung Jang, Ka Lok Choi, Scarlett Brown, Yasmin Rasool, Laurence Game, Damir Baranasic & William R. Scott MRC Laboratory of Medical Sciences, London, UK Antonio M. A. Miranda, Liam McAllan, Guianfranco Mazzei, Ivan Andrew, Meryem Ertugrul, Bhavik Patel, Sophie Newton, Mikyung Jang, Ka Lok Choi, Scarlett Brown, Yasmin Rasool, Laurence Game, Damir Baranasic & William R. Scott Iona Davies, Julia Kenkre & Tricia Tan Imperial College Healthcare NHS Trust, London, UK Julia Kenkre, Robert Goldin, Patricia Ortega, Ahmed R. Ahmed, John C. Chambers, Jaspal S. Kooner, Tricia Tan & William R. Scott Department of Cardiology, Ealing Hospital, London North West University Healthcare NHS Trust, Middlesex, UK Weihua Zhang, John C. Chambers & Jaspal S. Kooner Translational Research and Immunology Group, Nuffield Department of Surgical Sciences, University of Oxford, Oxford, UK Amy Cross, Oliver McCallion, Joanna Hester & Fadi Issa UCLH Bariatric Centre for Weight Loss, Weight Management and Metabolic and Endocrine Surgery, University College London Hospitals, London, UK Marco Adamo, Mohamed Elkalaawy, Andrew Jenkinson, Borzoueh Mohammadi, Majid Hashemi & Rachel L. Batterham MRC Centre for Environment and Health, School of Public Health, Imperial College London, London, UK Division of Electronics, Ruđer Bošković Institute, Zagreb, Croatia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Participant recruitment, human sample collection and tissue processing: I.D., M.Ertugrul, J.K., A.P., M.A., A.J., M.Elkalaawy, B.M., M.H., P.O., A.R.A., R.L.B., J.C.C., J.S.K., T.T. Spatial transcriptomic library preparation and data generation: I.A., K.L.C., L.M., G.M., H.K., R.G., A.C., O.M., L.G., J.H. has undertaken consultancy work for Novo Nordisk, ViiV Healthcare and Epitomee Medical, and is employed by Eli Lilly. The remaining authors declare no competing interests. Nature thanks Matthew Cottam, Evan Rosen and the other, anonymous, reviewer(s) for their contributions to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, UMAP embedding of AT cell types across conditions and datasets demonstrating successful integration and cell type conservation. b, Cell type marker genes in the single nucleus (Nuc, left) and spatial datasets (right). c, Cell state marker genes in the Nuc (left) and spatial datasets (right). b,c, Scaled mean expression and proportion (%) of cells expressing marker. Orange obese-high, blue obese-low, grey non-significant (NS). e, Proportional changes in adipocytes and macrophages between conditions in single Nuc and spatial datasets. Restricted to these cell types due to limited spatial cohort numbers (N = 4/group) and intra-sample/group heterogeneity in vascular and precursor cell numbers. Wilcoxon Paired (OB-WL) and Unpaired (OB-LN), FDR adjusted P value. Cohen's D, coloured at FDR < 0.05, red obese-high, blue obese-low. b, Beeswarm plots showing significant changes (Spatial FDR < 0.1) in neighbourhood abundance in myeloid cell classes. Lean-obese, obese-WL comparisons, Log2FC between conditions, red obese-high, blue obese-low. Fasting insulin adjusted for body mass index (FI adj BMI), Log2FC per unit change, red FI-high, blue FI-low. Lean-Obese unpaired, WL-Obese paired Two-tailed Wilcoxon test. d, Volcano plot of differentially expressed genes in LAM subtypes (ST) 1 (adaptive) and 2 (maladaptive/inflammatory). e, Representative spatial images of a CLS. Top, individual transcripts detected by Xenium for Adipocyte markers (ADIPOQ Orange, PLIN4 Cyan), a LAM marker (PLA2G7 Magenta), and a nuclei counterstain (DAPI Gray), showing LAMs surround a transcriptionally devoid/dead adipocyte. Bottom, CellTypist “best match” prediction of LAM ST at the CLS. f, Shared LAM subtype marker genes, scaled mean expression, proportion (%) of cells expressing marker, in the single nucleus (sNuc, top) and spatial (bottom) datasets. sNuc was used as the training dataset to predict a “best match” in the spatial dataset (CellTypist). g, Proportion of LAM ST1 and ST2 in CLS (defined as ≥3 LAMs) or isolated (defined as ≤2 LAMs in Neighbourhood). Cohen's D, coloured at FDR < 0.05. i, SCENITH strategy (top) for LAM and TRM metabolic activity from Obese donors (N = 7). HPG-AZ555 Click chemistry was used to measure metabolic activity. Cells were treated with combinations of drugs (Control, 2DG, Oligo, 2DG+Oligo) to assess metabolic profiles, calculated using formulas (right panel). Bottom, Click intensity (MFI) for each drug treatment (left) and calculated metabolic profiles (right). Paired Student's two-tailed t-test P value. k, Differentially expressed inflammatory cyto/chemokine genes between conditions in single nucleus (Nuc) and spatial datasets. Size adjusted -log10 P value, negative binomial mixed effects model. Circled dots represent comparisons with absolute log2FC > 0.3 and adjusted P value < 0.05. l, UMAP embedding of lymphoid cell classes, all conditions in single nucleus dataset. Two-tailed Wilcoxon paired (OB-WL) and unpaired (OB-LN) test. Presented for adipocytes and macrophages in which global metabolic shifts were observed and endothelial cells as a representative other cell type to demonstrate absence of global activation. Cohen's D, coloured at FDR < 0.05, red obese-high, blue obese-low. a, UMAP embedding of mature adipocytes, all conditions grouped. b, Adipocyte cell state proportions (0 to 1) in the combined cohort, mean (Av.) d, Scores measuring overall activity in major metabolic pathways in each adipocyte, averaged for each participant (density, median IQR) then compared between conditions. Two-tailed Wilcoxon unpaired (LN-OB, LN-WL) and paired (OB-WL) FDR adjusted P values (N = 24 LN; 25 paired OB/WL donors). e, Schematic of the triglyceride (TG) to glycerol cycle, broken down into reaction steps, and annotated by reaction enzyme families. Adapted from Sharma et al.23 f, Extended differentially expressed genes between conditions in single nucleus (Nuc) and spatial datasets in adipocytes. Encompassing enzymes in metabolic substrate pathways, including the TG cycle, and upstream regulators. Size adjusted -log10 P value, negative binomial mixed effects model. Circled dots represent comparisons with absolute log2FC > 0.3 and adjusted P value < 0.05. g, Differential gene regulatory networks between obesity and WL in mature adipocytes, restricted to metabolic pathway genes. TF networks with >50 metabolic genes/network and network P < 0.05 Bonferroni adjusted are shown. Coloured by proportion of all pathway genes in the network. Barplots show sum of genes in pathway (top) and network (right). Left, heatmaps show network (two-tailed Wilcoxon test) log2FC and human GWAS intersection. h, Pathways underlying reciprocally differentially expressed genes in lean-obese (LN-OB, log2FC > 0.5, FDR < 0.01) and obese-WL (OB-WL, log2FC > 0.5, P < 0.05, Bonferroni adjusted) comparisons. ORA, hypergeometric distribution, coloured by FDR adjusted -log10 P values, sized by count, enrichment factor is gene ratio/background ratio. i, Variations in mature adipocyte sizes (top, log10 Area; bottom, Area) between groups in spatial analyses, and two-tailed Wilcoxon test P value (N = 4850 LN; 3315 OB; 3909 WL; number of segmented adipocytes across 4 donors in each group). a, Adipocyte precursor (APC) subpopulation marker genes presented as scaled mean expression and proportion (%) of cells expressing marker. UMAP embedding of APCs, all conditions grouped, according to b, subtypes and c, subtype marker gene expression. d, Vascular endothelial cell (EC) subpopulation marker genes presented as scaled mean expression and proportion (%) of cells expressing marker. e, UMAP embedding of vascular EC, all conditions grouped. f, Mural cell subpopulation marker genes presented as scaled mean expression and proportion (%) of cells expressing marker. g, UMAP embedding, all conditions grouped. a,b,d–g, Cell states highlighted in bold represent stressed populations. a, Proportions (%) of differentially abundant neighbourhoods (Spatial FDR < 0.1) in lean-obese and obese-WL comparisons among basal and stressed cell states. Orange obese-high, blue obese-low, grey non-significant (n.s.). Mature adipocytes (AD), precursors (APC), endothelial cells (EC) and mural pericytes (Per.). b, Pairwise changes in basal and stressed cell proportions in obesity and subsequent WL for each donor in single nucleus (grey) and spatial (orange) datasets (N = 25 single nucleus; 4 spatial). c, Differential expression between conditions of common stress genes in all vulnerable cell types (left) and homeostatic and maladaptive genes in metabolic and precursor (right, top) and vascular (right, bottom) cell types. Size adjusted -log10 P value, negative binomial mixed effects model. Circled dots represent comparisons with absolute log2FC > 0.3 and adjusted P value < 0.05. d, Overlap of differentially expressed genes in stressed states compared to the respective basal state, among vulnerable cell types (Wilcoxon test, FDR < 0.05). Red represents a common set of 188 differentially upregulated and 15 downregulated genes in all represented stressed cell states (Single Nuc. e, UMAP embedding of example stress genes across susceptible cell types. f, Stress score based on 188 conserved upregulated genes in stress cell states (AD3, EC2, APC3, Mu4), by cell type and condition, represented as a scaled z-score. For AD3 and APC3, two-tailed Binomial sign test P values comparing the observed directions of effect in each cell neighbourhood with the expected null of 0.5. a, Violin plots of stress enriched-genes for example pathways, averaged per sample in stressed (dark grey) and basal (light grey) cell states. Violins outlined in black have Log2FC > 0.1 and FDR < 0.05 (Wilcoxon, Supplementary Table 12). b, Selected examples of enriched pathways underlying conserved stress genes (differentially expressed in ≥3 stressed-basal state comparisons). ORA, hypergeometric distribution, coloured by FDR adjusted -log10 P values, sized by count, enrichment factor is gene ratio/background ratio. c, In vitro effects of stress induction on: i. human adipocyte differentiation (left, % Oil Red-O (ORO) positive mature adipocytes) in undifferentiated (Negative Control, N = 8), 14-day differentiated (Positive Control, N = 6), and 14-day differentiated 5-day Etoposide treated (5 µM and 10 µM, N = 6) cells; ii. expression of stress marker proteins (middle/right, % JUN and STAT3 positive nuclei, immunohistochemistry) in undifferentiated control and Etoposide treated cells (N = 8 per group). d, Representative images of ORO accumulation and e, JUN and STAT3 protein expression in each experimental group. a, Representative images of the spatial datasets showing tissue architecture (top, WGA staining), stress scores in 50-µm bins (middle) and tissue niches (bottom). c, Proportions (0 to 1) of cell states in each tissue niche. d, average distance within 300 µm between spatial cell states. e, Clustermap of imputed scaled average ligand communication probabilities (CellChat) per tissue niche, limited to significant communications. f, Imputed CellChat communication between spatial niches for selected ligands. Links represent the scaled mean probability (line thickness) and directions of connectivity. All conditions were combined to identify the main niches underlying pathway effects. g, CellChat communication between single nucleus cell states for NAMPT (Visfatin, top) and TGFB1 (bottom). Links represent the scaled mean probability (line thickness) and directions of connectivity. All conditions were combined to identify the main cell states underlying pathway effects. Lower probability interactions for NAMPT were removed to improve visualisation. h, Sankey plots showing differential signalling pathways between source and target cells in lean-obese (left) and obese-WL comparisons (right). Connection size represents number of cell type interactions for each pathway and colour relative flow (red obese-high, blue obese-low). a, Number of differentially expressed genes in major AT cell types in lean-obese (FDR < 0.01) and obese-WL (P < 0.05 Bonferroni adjusted) comparisons. b, Heatmaps showing proportion of significant genes (0–1, green) in the primary comparison that had i. concordant directions of effect (concordant), ii. concordant and robustly significant (at FDR < 0.01 lean-obese or P < 0.05 Bonferroni adjusted obese-WL, concordant + pval stringent) in the alternative comparison, as well as the associated binomial test -log10 P value (orange). Barplots depict total number of robustly significant reciprocal genes. c, Volcano plots of differentially expressed genes associated with WL across AT cell types. Horizontal line, Bonferroni adjusted significance threshold. d, Pathway analysis of genes downregulated by WL (FC > 0.5, P < 0.05 Bonferroni adjusted) in cell type intrinsic analyses. Sized by FDR adjusted -log10 P values (ORA, hypergeometric distribution) and coloured by enrichment factor (gene ratio/background ratio). Shown 44 representative of 660 total pathways at FDR < 0.01. e, Pathway analysis of conserved genes, downregulated by weight-loss in ≥3 distinct cell types (FC > 0.5, P < 0.05 Bonferroni adjusted), clustered by gene (N = 213) and pathway (N = 304, ORA, hypergeometric distribution, FDR < 0.01). All differential expression analyses applied two-tailed neg. a, Differences in expression of cell cycle and senescence marker genes in WL, separated into vulnerable and other cell types. b, Immuno-fluorescence of NAMPT protein expression (N = 4 samples/group), scaled to connective tissue marker Lectin, paired student's two-tailed t-test (left). Representative images of an obese and WL pair, scale bar 50 µm resolution, Grn NAMPT, Rd Lectin, blue DAPI nuclei. c, Left (All), tissue-wide senescence score (Oncogene induced), averaged across every cell for each participant (density, median IQR), then compared between conditions. Two-tailed Wilcoxon unpaired (LN-OB, LN-WL) and paired (OB-WL) P values. Right, density heatmaps of cell-level senescence scores (Oncogene induced) encompassing all cell types for each individual sample separated into Lean, Obese, WL groups, single nucleus datasets (N = 24 LN; 25 paired OB/WL donors). d, Other unbiased senescence score heatmaps across groups and vulnerable cell types. e, Proportion of p21 negative (−) and p21 positive (+) cells with high senescence scores (defined by score higher than median in ≥3 of 4 distinct senescence scores). Proportion presented for each sample (N = 87). g, Mean proportions of p21 (range 0–1) positive cells in each sample across conditions in spatial datasets (N = 4/group). Two-tailed Wilcoxon unpaired (LN-OB) and paired (OB-WL) FDR adjusted P values. h, Heatmap showing pairwise correlation (Pearson, R) between delta (Δ) changes in pathway scores before and after WL in paired samples. Pathway scores were calculated from the mean pathways score in mature adipocytes within each paired sample. i, Gene regulatory networks upregulated in stressed, senescent cells (scaled log2FC > 0.4 compared to all other cell states in cell type, and in ≥3 stressed cell states) and coloured by number of shared genes in the network (Jaccard index, top). Interactions between TFs within the network (bottom), sized by number of interactions with other TFs, connected by forward interactions, # annotates self-interaction, coloured by Walktrap community. j, Expression of secretory proteins from the Human Protein Atlas (HPA) in stressed compared to basal cell states among vulnerable cell types. Wilcoxon test, Log2FC (positive, stress-high) coloured by cell type, grey if non-secretory or non-significant (P > 0.05 Bonferroni adjusted). AREG which is not in the HPA was included as a well-established secreted protein67. k, Scatter plot of 11 predicted SASP proteins present in both single nucleus and spatial datasets according to dataset log2FC in lean-obese and obese-WL comparisons (obese-high). l, Senescence and SASP gene expression (imputed) in tissue niches, represented as a scaled z-score. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Selective remodelling of the adipose niche in obesity and weight loss. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/home/lawn-garden/g65288849/prime-day-patio-furniture-deals-2025/'>Get Over 30% Off Patio and Backyard Essentials With These Prime Day Discounts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 14:55:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>From cozy swings to chic dining sets, these Prime Day deals have your outdoor space covered. We may earn commission if you buy from a link. There's no better time to upgrade your outdoor space than when summer evenings are at their peak, especially when the savings from Amazon Prime Day make these upgrades much more affordable. Prime Day deals have you covered, no matter what you're looking for, from a full dining set for your deck, rocker chairs for morning coffee on the porch, or a bench with storage to hold pool toys. Whether you're looking for a sleek, modern sectional that withstands the elements or a classic bistro set that adds a touch of elegance, these choices aren't just great deals; they're engineered for durability and designed with thoughtful features that make them worth the investment. We're seeing massive markdowns on several of our favorite patio furniture pieces, with savings up to an impressive 51 percent off. We'll be keeping an eye on Amazon all week and updating this story with the best discounts. At the time of this writing, the outdoor sofa from Signature Design by Ashley is currently on sale for more than $680 off. Constructed from solid eucalyptus wood, this sofa features high-quality Nuvella fabric cushions that are UV-resistant (which helps them maintain their color under sun exposure), stain-resistant, and easy to clean. The light brown natural finish also adds a sleek, coastal vibe. This three-piece outdoor rocker chair set from Hillga features two high-back wicker chairs and a side table, ideal for enjoying morning coffee on the patio or providing seating for an outdoor party. Constructed with a powder-coated steel frame, the set is designed for long-term durability with a weight capacity of up to 300 pounds per chair. The thick cushions are filled with high-density foam and covered in UV-resistant fabric. The swivel and rocking features bring a comforting, relaxing experience to your outdoor patio, and this set is now 20 percent off for Prime Day. While the standard price tag is quite high, you can get it for more than $500 off during Prime Day. A swing can bring a relaxing, rustic vibe to your patio, and you can get this high-quality farmhouse swing for ten percent off this Prime Day. The wood is treated with a water-resistant finish to withstand adverse weather conditions. The swing supports up to 600 pounds, providing secure two-person seating with heavy-duty hanging chains that ensure stability. It arrives ready to assemble, complete with directions through the BILT app, and features a five-year warranty for added peace of mind. The Yitahome Patio Furniture Set brings a fun, bohemian aesthetic to durable, all-weather rattan wicker construction. Thick cushions, made from high-density foam, provide comfort while the removable, machine-washable covers ensure easy maintenance. The sectional design allows for customizable configurations, adapting to different space layouts. For families with kiddos playing in the pool, this is a must-have. It's currently nine percent off and is constructed with handwoven rattan over a reinforced steel frame. The internal storage compartment, accessible via a lift-up seat, provides 40 gallons of capacity for cushions, garden tools, pool accessories, and outdoor toys. The included light gray cushion features a zippered, removable cover for easy cleaning. This seven‑piece wicker patio sectional set features a modular design comprising two corner units, four armless straight sections, and a tempered‑glass coffee table, allowing reconfiguration to fit varied outdoor layouts. Crafted with weather‑resistant PE rattan woven over a rust‑proof steel frame, it's designed to withstand UV exposure, rain, and temperature fluctuations. Each seating element includes foam cushions wrapped in water‑ and fade‑resistant polyester with zippered covers for easy laundering. This sectional is on Prime Day special for 33 percent off, saving you $200. The Yaheetech aluminum bistro table is engineered for compact outdoor spaces and features a lightweight yet corrosion-resistant aluminum alloy structure with a bronze finish. Its 30.7-inch diameter tabletop includes a 1.9-inch opening in the middle, which is compatible with most standard outdoor umbrellas. The textured, hollow-lattice tabletop not only lends decorative appeal but also prevents water pooling. Each leg includes adjustable anti-slip foot pads to compensate for uneven surfaces. These stackable dining chairs feature a powder-coated aluminum frame that supports up to 352 pounds, with cross-bracing beneath the seat for enhanced structural stability. Compact dimensions allow for the stacking of up to 20-23 units, facilitating efficient storage and making this chair set perfect for significant events, such as backyard weddings. Get a set of two for 26 percent off this Prime Day. The table also features a four-legged powder-coated steel frame base with protective floor glides, and the chairs are made from breathable fabric. This set is available in four different color options, all of which are on sale for Prime Day. This stylish loveseat features a high backrest and wide armrests, and it offers spacious seating for two, perfect for relaxing on your balcony, deck, or poolside. Compact yet comfortable, this small rattan loveseat blends style with functionality, offering the perfect seating solution for your outdoor space. Easy to maintain with machine-washable cushion covers, it's great for both casual and elegant outdoor settings. Right now, it is 25 percent off for Prime Day. For poolside family dinners, this picnic table set from Serwall has everything you need. The dining table has an umbrella hole to protect your family from UV rays on hot, sunny days, is measured at 67 inches in diameter, and can hold up to 220 pounds. Sarah Connor is a Reviews Editor for Popular Mechanics. She also contributes to Runner's World, Bicycling, Best Products, and Biography.com. She holds a master's in Journalism from Syracuse University and a dual bachelor's in English and Communications from the University of Pittsburgh. When she isn't writing up a review of the newest gadgets, you can find her heading to the nearest rink for a figure skating session or enjoying the outdoors with her beloved Australian Shepherd, Rosie.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250708045651.htm'>Matching your workouts to your personality could make exercising more enjoyable and give you better results</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 14:38:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Finding motivation to exercise can be the greatest challenge in working out. But what if working out could be more enjoyable? To this end, researchers in the UK now have examined how personality affects what types of exercise we prefer, and our commitment and engagement to them. "We found that our personality can influence how we engage with exercise, and particularly which forms of exercise we enjoy the most," said first author Dr Flaminia Ronca from University College London's (UCL) Institute of Sport, Exercise and Health. "Understanding personality factors in designing and recommending physical activity programs is likely to be very important in determining how successful a program is, and whether people will stick with it and become fitter," added senior author Prof Paul Burgess from the UCL Institute of Cognitive Neuroscience. The researchers recruited participants that attended lab testing for baseline fitness. During lab testing, the first intervention week, and after the intervention, all participants completed a questionnaire on how much they'd enjoyed each training session. "So it's not surprising that personality would also influence how we respond to different intensities of exercise." Contrary, people scoring high on neuroticism preferred private workouts. Others, scoring high on consciousness and openness were found to engage in exercise regardless of whether they particularly enjoyed it or were driven by curiosity, respectively. What was particularly interesting was the relationship between personality, change in fitness, and stress, the researchers said. "It's fantastic news, as it highlights that those who benefit the most from a reduction in stress respond very well to exercise," Ronca said. The researchers pointed out that the most important part about exercising is finding something we enjoy and not to be discouraged if we don't immediately find it. "It's ok if we don't enjoy a particular session," Ronca said. "We hope that if people can find physical activities that they enjoy they will more readily choose to do them," Burgess concluded. But for some reason, many of us humans seem poor at picking up on these messages it is sending to our brain." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/home/tools/a65267670/dewalt-prime-day-deals-2025/'>DeWalt is My Go-To Prime Day Tool Brand: These are The Best Deals We've Found So Far</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 14:14:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Prime Day is the perfect time to build out your arsenal of black and yellow tools We may earn commission if you buy from a link. Amazon Prime Day is always a good time to get the best deals on DeWalt tools, and this year's summer event is no exception. You'll find deals on cordless drills, impact drivers, circular saws, reciprocating saws, and jigsaws across the brand's standard and pro-focused “XR” product lines. Editor's Note, July 9, 2025: We've combed through all the listed products to make sure they are still on sale and in stock. You can still score a tremendous deal on DeWalt's versatile oscillating multi-tool at more than 50 percent off. We will be regularly checking this page regularly to ensure the recommended picks remain in stock, but inventory and prices can change at a moment's notice. There are deals on everything during Prime Day, including most of my favorite tool brands, so if you're specifically looking to buy DeWalt, it helps to narrow the field. Start by going to the DeWalt Amazon store, then click on the “all deals” tab to see what DeWalt products are on sale. DeWalt has historically been a big Prime Day backer, offering good deals every year. That said, I do generally recommend Prime Day shoppers do an extra check to make sure you're getting the best possible sale price. The company routinely puts more tools on sale than any other premium brand. This year, you'll find sales on both full tool and battery sets, as well as many deals on bare tools that come with no battery. Bare tools are a smart option if you're deeply integrated into the DeWalt platform and already have a bunch of good condition batteries. However, if if you have DeWalt tools with no batteries or aging batteries, this is a good time to get more power as there are a couple (though not a ton) of solid discounts on batteries. This dynamic duo is a great option for novice DIYers looking to build up a toolkit, or new homeowners who just need the absolute essentials for the occasional odd job. That's ample power for pretty much any summer DIY project you're planning. If you're like me and hate all the extra effort that goes into manually sanding your carpentry projects, then this deal might help you take this edge off the task. This DeWalt XR cordless sander is nearly half-off, giving you a cheap upgrade that will spare your arm. There's even a dust bag to catch some of the sawdust and minimize mess. An oscillating multi-tool can handle a variety of tasks you may run into during your next home improvement project, like cutting through drywall, grinding grout, and sanding wood. DeWalt's XR model is one of the best: Designed for pros, it runs at three different speeds, reaching up to 20,000 oscillations per minute (OPM). Its quick-change accessory suite makes it easy to switch between blade types on complex, multi-phase jobs. At its current DIY friendly price, this tool's a steal. Fall may feel like it's a long way off, but it's never a bad idea to buy your leaf clean-up tools during the off-season, when they're most likely to be on sale. It also has a variable speed trigger you can lock when taking on bigger jobs, saving your hand. This bare 6-1/2-inch circular saw can reach max speeds of 5,150 RPM, which is more than enough power for cutting through plywood, 2x4s, and other dimensional lumber. And at just 7 pounds, it's one of the easier circular saws to use. DIY carpenters and woodworkers will love the discount on this DeWalt jigsaw. It boasts a high blade speed of up to 3,000 strokes per minute (SPM), so you can count on it to cut clean dovetails and circles through wood. If your list of summer projects involves tearing things down to create something new, then this reciprocating saw can help you power through the demo work. With its long, 1-1/8 inch stroke length and max cutting speed of 3,000 SPM, it chews through 2x4s, nails, drywall, metal and plastic with ease. You can load blades in four different positions, making it easier to cut from various angles. I've been using the same DeWalt miter saw for 20 years now and it's still going strong, so this feels like an obvious recommendation to me. This DeWalt portable job site table saw is typically priced for professionals, but this Prime Day discount makes it a viable option for veteran DIYers. It features a 8-1/4-inch blade and produces a whopping 5,800 max RPM, generating the speed and cutting capacity to make tough rip cuts through thicker materials. It also has a 24.5 inches of rip capacity, so you can cut through wider boards. Plus, at 46 pounds it is truly portable–it's light enough for one person to load into a truck, or move in and out of storage. This is a solid deal on one of DeWalt's versatile FlexVolt batteries. While a $40 savings on two batteries may not sound the sale alarms like some of our other listed deals do, this is nonetheless some solid savings on a two-pack of DeWalt's MAX XR 2o volt batteries, compatible with the brand's more professional oriented tools. A combo pack like this with two batteries gives you more flexibility if you have aging batteries or find yourself swapping one battery between multiple tools. This 20-volt cordless model comes with a HEPA-rated filter and holds 2 gallons of debris. Its 5-foot hose and compact size make it ideal for toting to wherever in the house your project happens to be or for cleaning out the car. It weighs just 8 pounds and has a convenient carrying handle on the lid. It features two hoses: A high-pressure hose for inflating tires, and a low-pressure hose for bikes, balls, and other everyday inflatables. With its LED display, it's easy to read air pressure as you're inflating. I also really like that you can plug this tool into the wall, charge it using a 12V DC car accessory cable, or rely on DeWalt's standard 20V battery. Prime Day Deals on Patio Furniture You Can't Miss</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65350258/teenager-student-archimedes-mythical-death-ray/'>A Teenager Built Archimedes' Mythical Death Ray—and It May Actually Work</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 13:55:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The young scientist's twist on a legendary ancient weapon might just re-write history. It might be surprising to learn that then-12-year-old Brenden Sener of London, Ontario was enthralled by Ancient Greek concepts posited by the mathematician Archimedes. But it probably make more sense when you find out what exactly those concepts were. Archimedes wrote about harnessing the Sun's energy to create a death ray. While no archaeological evidence proves that the death ray—also known as a heat ray—was ever used, that doesn't necessarily mean it didn't happen. Sener explained that the ancient death ray was made up of a series of large concave mirrors, positioned to focus the Sun's rays onto a single focal point and potentially hitting a target up to 1,000 feet away. Sener crafted a miniature version of the concept. He believes his experiments show that a larger-scale version—such as the one described in an account of the battle of Syracuse from 214 to 212 BC, which is alleged to have helped set approaching Roman ships on fire—were conceivable. Ancient efforts may have either used mirrors or highly polished shields. In an interview with CNN, Sener said he first grew fascinated with the inventions of Archimedes during a family vacation to Greece. “And it really did revolutionize technology at that time because Archimedes was thinking about stuff that no one actually had before.” According to CNN, Cliff Ho—a senior scientist at Sandia National Laboratories who presented on the death ray about a decade ago—commended Sener for his scientific work. He also believes that the death ray was possible, if unlikely to have been constructed or used. As he notes in his paper, the MythBusters series from Discovery Channel tested scenarios three different times and failed to catch a boat on fire on each attempt. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. This Plasma May Melt the Current Rules of Reality Experts Found a Sarcophagus in an Ancient Tomb</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65310055/cern-plasma-atlas/'>Scientists Are Creating Plasma So Hot, It May Melt the Current Rules of Reality</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This is what happens when you smash atoms in conditions even protons and neutrons can't stand up to. Even the most powerful telescopes have not yet been able to see far enough back in time to witness events from the Big Bang. But by smashing atoms, it is possible to create a plasma that existed right after the universe was born. As part of CERN's ATLAS experiment, the Large Hadron Collider (LHC) is now crashing oxygen ions into each other, and will soon be doing the same with neon ions. Heavy ions like this can create the quark-gluon plasma that is thought to have existed when the universe first exploded into being. During those very early moments of the universe, things were extraordinarily hot, and quark-gluon plasma behaves in strange ways when super-heated. This critical force is propagated by fundamental particles known as gluons, which are both massless units of energy that have no electric charge and bosons, which spin in full integer values (as opposed to fermions which have odd half-integer spins). CERN is colliding oxygen ions with each other—and repeating the process with neon ions—because they have far fewer protons and neutrons than the lead ions which are usually smashed to create this plasma. This means that oxygen and neon will produce smaller blobs of quark-gluon plasma that could possibly reveal what happens somewhere between collisions of lighter particles (such as protons) in “cold” conditions and heavier particles (such as lead ions) in immensely hot and dense conditions. CERN researchers are eagerly hoping to observe some specific potential phenomena during these tests. Jet quenching, for one, occurs when highly energetic particle jets begin to lose energy as they zoom through quark-gluon plasma. If what theories have predicted actually happens, the results of the ATLAS experiment might show how dense quark-gluon plasma needs to get for jet quenching to begin. There are still many unknowns present in these interactions, colliding protons with oxygen ions could recreating the situations in a way that scientists could analyze up close and personal. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/chatgpt-and-gemini-ai-have-uniquely-different-writing-styles/'>ChatGPT And Gemini AI Have Their Own, Distinctive Writing Styles—Just As Humans Do</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Each AI Chatbot Has Its Own, Distinctive Writing Style—Just As Humans Do ChatGPT and Gemini AI write in different idioms, linguists find A few weeks ago, while comparing language proficiency in essays written by ChatGPT with that in essays by human authors, I had an aha! I realized that I was comparing a single voice—that of the large language model, or LLM, that powers ChatGPT—to a diverse range of voices from multiple writers. Linguists like me know that every person has a distinct way of expressing themselves, depending on their native language, age, gender, education and other factors. My insight: one could analyze the language produced by ChatGPT to find out whether it expresses itself in an idiolect—a single, distinct way. This field examines language use in police interviews with suspects, attributes authorship of documents and text messages, traces the linguistic backgrounds of asylum seekers and detects plagiarism, among other activities. While we don't (yet) need to put LLMs on the stand, a growing group of people, including teachers, worry about such models being used by students to the detriment of their education—for instance, by outsourcing writing assignments to ChatGPT. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. To test whether a text has been generated by an LLM, we need to examine not only the content but also the form—the language used. Research shows that ChatGPT tends to favor standard grammar and academic expressions, shunning slang or colloquialisms. Compared with texts written by human authors, ChatGPT tends to overuse sophisticated verbs, such as “delve,” “align” and “underscore,” and adjectives, such as “noteworthy,” “versatile” and “commendable.” We might consider these words typical for the idiolect of ChatGPT. But does ChatGPT express ideas differently than other LLM-powered tools when discussing the same topic? Online repositories are full of amazing datasets that can be used for research. One is a dataset compiled by computer scientist Muhammad Naveed, which contains hundreds of short texts on diabetes written by ChatGPT and Gemini. The texts are of virtually the same size, and, according to their creator's description, they can be used “to compare and analyze the performance of both AI models in generating informative and coherent content on a medical topic.” The similarities in topic and size make them ideal for determining whether the outputs appear to come from two distinct “authors” or from a single “individual.” The formula compares frequencies of words commonly used in the texts: words that function to express relationships with other words—a category that includes “and,” “it,” “of,” “the,” “that” and “for”—and content words such as “glucose” or “sugar.” In this way, the Delta method captures features that vary according to their authors' idiolects. In particular, it outputs numbers that measure the linguistic “distances” between the text being investigated and reference texts by preselected authors. In both cases, the authorship turns out to be quite clear, indicating that the two tools' models have distinct writing styles. To better understand these styles, let's imagine that we are looking at the diabetes texts and selecting words in groups of three. Such combinations are called “trigrams.” By seeing which trigrams are used most often, we can get a sense of someone's unique way of putting the words together. ChatGPT's trigrams in these texts suggest a more formal, clinical and academic idiolect, with phrases such as “individuals with diabetes,” “blood glucose levels,” “the development of,” “characterized by elevated” and “an increased risk.” In contrast, Gemini's trigrams are more conversational and explanatory, with phrases such as “the way for,” “the cascade of,” “is not a,” “high blood sugar” and “blood sugar control.” Choosing words such as “sugar” instead of “glucose” indicates a preference for simple, accessible language. Once a word or phrase becomes part of their linguistic repertoire during training, the models might continue using it and combine it with similar expressions, much like people have favorite words or phrases they use with above-average frequency in their speech or writing. Or it might be a form of priming—something that happens to humans when we hear a word and then are more likely to use it ourselves. Perhaps each model is in some way priming itself with words it uses repeatedly. Idiolects in LLMs might also reflect what are known as emergent abilities—skills the models were not explicitly trained to perform but that they nonetheless demonstrate. The fact that LLM-based tools produce different idiolects—which might change and develop across updates or new versions—matters for the ongoing debate regarding how far AI is from achieving human-level intelligence. It makes a difference if chatbots' models don't just average or mirror their training data but develop distinctive lexical, grammatical or syntactic habits in the process, much like humans are shaped by our experiences. Karolina Rudnicka is a linguist at the University of Gdańsk in Poland.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03798-1'>Plasma proteomics links brain and immune system aging with healthspan and longevity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 10:23:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Plasma proteins derived from specific organs can estimate organ age and mortality, but their sensitivity to environmental factors and their robustness in forecasting onset of organ diseases and mortality remain unclear. To address this gap, we estimate the biological age of 11 organs using plasma proteomics data (2,916 proteins) from 44,498 individuals in the UK Biobank. Organ age estimates were sensitive to lifestyle factors and medications and were associated with future onset (within 17 yearsʼ follow-up) of a range of diseases, including heart failure, chronic obstructive pulmonary disease, type 2 diabetes and Alzheimer's disease. Notably, having an especially aged brain posed a risk of Alzheimer's disease (hazard ratio (HR) = 3.1) that was similar to carrying one copy of APOE4, the strongest genetic risk factor for sporadic Alzheimer's disease, whereas a youthful brain (HR = 0.26) provided protection that was similar to carrying two copies of APOE2, independent of APOE genotype. Altogether, these findings support the use of plasma proteins for monitoring of organ health and point to the brain and immune systems as key targets for longevity interventions. Aging causes organ dysfunction, chronic disease and death. Although interventions such as caloric restriction, young plasma and rapamycin extend healthspan in model organisms1, potentially slowing aging, their human efficacy is unclear due to limited molecular understanding and assessment tools. Molecular biomarkers of human biological age linked to health and disease are needed. Recent studies show that human organs age at different rates2,3,4,5,6, as seen in animals7,8,9, necessitating organ-specific biological age measures. Prior estimates of organ age used clinical metrics and blood biochemistry, magnetic resonance imaging (MRI), DNA methylation or plasma proteins2,3,4,5,6. Furthermore, it is unclear which organs are key to longevity in humans. Plasma proteomics, enabled by clinical accessibility of blood and advancing technologies, is ideal for gaining molecular insights into organ health and estimating organ-specific biological age. Building on our prior SomaScan-based study (5,678 individuals, 5,000 proteins)4, where we introduced machine learning models to estimate organ age, in the present study we tested our approach on 44,498 individuals in the UK Biobank (UKB) (age 40–70 years), using an orthogonal proteomics platform (Olink, ~3,000 proteins, 1,823 protein name overlap with SomaScan). These data were previously generated by the UKB Pharma Proteomics Project (UKB-PPP) and have been characterized in detail10. The larger UKB cohort and expanded longitudinal phenotyping enabled us to explore organ age associations with a wider range of diseases (for example, chronic kidney disease, chronic obstructive pulmonary disease (COPD), heart failure and dementias) and focus on future disease incidence, which is a better proxy for biological age than disease prevalence. Lastly, we performed a nuanced examination of mortality risk, comparing organ age estimates with established aging biomarkers, including PhenoAge and estimated glomerular filtration rate, and we tested associations between extreme organ youth versus age acceleration on longevity versus early mortality. To derive estimates of organ-specific biological age from the plasma proteome, we (1) identified plasma proteins likely derived from a specific organ (Supplementary Tables 1 and 2, Methods and Supplementary Fig. 1); (2) trained a machine learning model to predict chronological age based on the levels of identified organ-enriched proteins (Supplementary Tables 3 and 4 and Extended Data Fig. a, Study design to estimate organ-specific biological age from plasma proteomics data in the UKB. A protein was called ‘organ-enriched' if the gene encoding the protein was expressed at least four-fold higher in one organ compared to any other organ in the GTEx organ bulk RNA sequencing atlas. Organ-enriched protein sets were used to train LASSO chronological age predictors. Samples from 11 of 22 centers (n = 23,140) were used for training, and the remaining samples (n = 21,358) were used for testing. An ‘organismal' model, which was trained on the levels of non-organ-specific (organ-shared) proteins, and a ‘conventional' model, which was trained on all proteins from the Olink assay, were also developed and assessed. Model age gaps were calculated and then z-score normalized per organ to allow for direct comparisons across organs. Age gaps were characterized and tested for associations with disease risk, modifiable lifestyle choices and mortality risk. b, Pairwise correlation of organ age gaps from all samples. c, A LASSO regression model was used to predict conventional age based on organ ages and organismal age. Organismal, brain and artery ages were sufficient to predict conventional age with r2 = 0.97. Relative weights are shown as a pie chart. d, Extreme agers were defined by a 1.5-s.d. increase or decrease in at least one age gap. The mean organ age gaps of extremely youthful brain agers and accelerated multi-organ agers are shown. Graphics in a and d created with BioRender.com. We performed these three steps for each of 11 major organs, including adipose tissue, artery, brain, heart, immune tissue, intestine, kidney, liver, lung, muscle and pancreas. We focused our analyses on these organs due to the availability of relevant age-related phenotype data in the UKB. To compare organ age gaps to organ-agnostic measures of biological age, we also derived age gaps from an ‘organismal' aging model trained on non-organ-specific proteins (80% of all proteins) and a ‘conventional' aging model trained on all proteins on the Olink assay. Data from 11 of 22 plasma collection centers were used for model training and the remaining 11 for testing (Fig. Model performance was highly stable across train and test centers (Extended Data Fig. Age gaps were z-scored per aging model to allow for direct comparison between organs in downstream analyses (Fig. We observed some sex differences between organ age gaps, with males having older kidneys, immune systems and intestines and females having older adipose tissue, arteries and hearts (Extended Data Fig. After deriving organ age gaps, we first determined the uniqueness of each measure. Pairwise correlations between age gaps showed that organ age gaps were only weakly correlated (mean r = 0.21; Fig. Organismal, brain and artery ages explained 97% of conventional age variance, with organismal age contributing 74% (Fig. We previously identified extreme organ agers who displayed especially fast aging in a single organ or in many organs4. We searched for extreme agers in the UKB by identifying individuals with age gaps above or below 1.5 s.d. from the population average in any organ (top and bottom ~6–7% percentiles). Here we found not only fast (≥1.5 z-age gap) organ agers but also slow (≤–1.5 z-age gap) organ agers (1–2% prevalence each, 33% total; Fig. Differences between studies may stem from different sample sizes, proteomics platforms and thresholds. We also identified multi-organ agers who had two or more extreme organ age gaps (26% of samples; Extended Data Fig. Fourteen percent of samples were ambiguous, with both positive and negative extreme age gaps, and 27% of remaining samples were defined as normal agers (Extended Data Fig. Multi-organ agers were significantly older than normal and single-organ agers, suggesting that aged organs accumulate over time (Extended Data Fig. To assess longitudinal stability of organ age gaps, we analyzed 1,176 individuals from the COVID-19 repeat imaging study with plasma proteomics (earlier version of Olink, 1,459 proteins) from 2–3 visits over 1–15 years (visits: baseline, Instance 2 and Instance 3) (Methods). Using organ aging models trained on the 1,459 proteins shared across Olink assay versions (correlated r ≥ 0.8 with 3,000-protein models), we found moderate to strong correlations (mean r = 0.6) between baseline and Instance 2 visit age gaps, indicating relative stability over approximately 9 years, with potential dynamic changes due to lifestyle, disease or technical variability (Extended Data Fig. Examining longitudinal stability of extreme ager status, we found that baseline extreme agers were 3–22 times more likely to remain extreme agers in the same organ at Instance 2 (Extended Data Fig. bins, 76% of baseline extreme agers maintained directional stability at Instance 2 and 72% at Instance 3, supporting relative stability (Extended Data Fig. Additional studies with larger cohorts, denser sampling and higher-coverage proteomics are needed to clarify technical versus biological stability. We next sought to compare our UKB Olink organ aging models with our previously developed SomaScan organ aging models. We generated Olink plasma proteomics data from 1,636 samples pooled across the Stanford Alzheimer's Disease Research Center (ADRC) and the Stanford Aging and Memory Study (SAMS)12,13. We previously generated SomaScan plasma proteomics data from 601 of these samples for our prior study4. This allowed us to directly compare organ age gaps derived across different proteomics platforms. We tested UKB-trained organ aging models on Stanford Olink data. Of note, five organ aging model proteins—ANGPTL7, EBI3.IL27, GZMB, PGLYRP1 and ADIPOQ—had missing values in all Stanford samples (Extended Data Fig. EBI3.IL27 and ADIPOQ had relatively strong weights in the liver and adipose aging models, respectively (Extended Data Fig. 4a) and, therefore, reduced liver and adipose age prediction accuracy in the Stanford samples (Extended Data Fig. Overall, UKB-trained organ aging models showed similar age prediction accuracies across UKB-train, UKB-test and Stanford-test healthy control samples, demonstrating robustness of the models (Extended Data Fig. We next compared organ age gaps between platforms. We observed moderate-strong correlations between the two platforms (r = 0.3–0.8; Extended Data Fig. 4c) with heart and kidney aging models displaying the most concordant age gaps across platforms (r = 0.81 and r = 0.82, respectively). Even if the same protein by name is measured, the two platforms may quantify different forms of the same proteins (that is, isoform, posttranslational modification), which may not necessarily correlate with each other. This suggested that each platform may provide complementary information in estimating organ age. Gene Ontology enrichment analyses showed that Olink brain aging proteins were enriched for perineuronal nets and glial differentiation, whereas SomaScan emphasized postsynaptic assembly; conversely, immune aging pathways were similar across platforms (Extended Data Fig. Both brain models were associated with prevalent Alzheimer's disease (Extended Data Fig. These findings show that organ-specific biological age estimates, likely related to organ health, can be derived from plasma proteomic data in the UKB. For an estimate of biological age to be informative, it must robustly associate with the physiological state of the organ or individual and, consequently, with age-related health and disease outcomes. Hence, we sought to determine whether organ age gaps could predict future diseases in their respective organs. We tested the associations between all 13 z-scored age gaps and 15 incident age-related diseases (2–17-year follow-up) using Cox proportional hazard regression, while adjusting for age and sex. Following Benjamini–Hochberg correction for multiple hypothesis testing, we identified 176 positive and four negative significant associations out of 195 tests (Extended Data Fig. We discovered highly significant associations between heart aging and atrial fibrillation (s.d. increase in heart age gap, HR = 1.83, q = 8.35 × 10−231), pancreas aging and kidney aging with chronic kidney disease (s.d. increase in pancreas age gap, HR = 1.80, q = 3.36 × 10−247; s.d. increase in kidney age gap HR = 1.66, q = 2.85 × 10−228), brain aging with Alzheimer's disease (s.d. increase in brain age gap, HR = 1.80, q = 1.21×10−67) and lung aging with COPD (s.d. increase in lung age gap, HR = 1.39, q = 6.82 × 10−49). increase in liver age gap, HR = 1.20, q = 3.87 × 10−13), albeit the strength of the association was modest and similar compared to other organs. Notably, organ-specific age gaps consistently exhibited higher HRs than conventional age gaps across all diseases (Extended Data Fig. The widespread significant associations between organ aging (176/195) and disease underscore the systemic nature of aging. To disentangle these systemic effects and identify organ age associations with disease independent of aging in other organs, we examined multivariate Cox models that included all organ age gaps (excluding ‘conventional' due to collinearity with ‘organismal', r = 0.87) as covariates. After accounting for the aging of other organs, we found that organ age gap associations with disease became much more organ specific (Fig. For example, incident heart failure was associated with all 13 baseline age gaps individually but only seven age gaps independently. Counterintuitively, we observed that youthful arteries were associated with increased risk for many diseases, including type 2 diabetes (s.d. increase in artery age gap, HR = 0.68, q = 2.11 × 10−64), COPD (s.d. increase in artery age gap, HR = 0.85, q = 8.85 × 10−8), chronic liver disease (s.d. increase in artery age gap, HR = 0.87, q = 7.58 × 10−5) and others, highlighting the complexity of aging's link with disease. Notably, Alzheimer's disease was highly specific to brain aging, both individually (s.d. increase in brain age gap, HR = 1.80, q = 1.21 × 10−67) and independently of other organs (s.d. increase in brain age gap, HR = 1.91, q = 2.37 × 10−67). a, Cox proportional hazard regression was used to test the association between age gaps and future disease risk, adjusted for age-at-blood-draw, sex and other age gaps. The heatmap is color coded by age gap log(HR). Heatmap columns are ordered by the Gini coefficient of age gaps per disease. b, Body plots showing logHR values for type 2 diabetes, atrial fibrillation and Alzheimer's disease from the heatmap in a. c, Cox proportional hazard regression was used to test the association between extreme brain youth/aging and future Alzheimer's disease risk, adjusted for age-at-blood-draw, sex and APOE4 and APOE2 genotype (n = 37,766). d, Cumulative incidence curves with 95% confidence intervals showing onset of Alzheimer's disease over time when comparing individuals with extremely aged brains, individuals with extremely youthful brains and normal brain agers. Age gap HRs and 95% confidence intervals from c are shown. e, Box plot visualization of brain age gap versus APOE genotype (n = 43,326). f, Plasma-based brain age gap at baseline versus MRI-based brain age gap at Instance 2 (several years after baseline visit). h, Pie chart displaying the proportion of brain aging proteins assigned to each brain cell type based on single-cell RNA sequencing. Cell type was assigned based on cell type with the maximum expression of a given gene. We also investigated the associations between extreme organ ager status and disease risk, independent of age, sex and other extreme organs (Extended Data Fig. Multi-organ agers 5–7 and 8+ were not included due to limited sample size for most diseases. Individuals with 2–4 aged organs had significantly increased risk for every disease that we examined. Individuals with extremely aged brains, hearts, kidneys or lungs also showed broad increased risk of disease. Conversely, individuals with 2–4 extremely youthful organs were protected from many diseases, including chronic kidney disease, osteoarthritis and COPD. Among youthful organ profiles, the brain, immune system and intestine were nominally protective for at least four diseases (nominal P < 0.10). A youthful brain was especially protective of Alzheimer's disease (extreme brain youth only, HR = 0.28, P = 0.076, q = 0.24) and other dementias, and a youthful intestine was especially protective of diabetes (extreme intestine youth only, HR = 0.30, q = 3.63 × 10−3; Extended Data Fig. Given the brain's strong and unique associations with Alzheimer's disease, we further examined all individuals with extremely youthful or aged brains—regardless of single or multi-organ ager status—and their associations with Alzheimer's disease risk, controlling for age, sex as well as APOE genotype, the strongest genetic risk factor of Alzheimer's disease. Interestingly, we found that extreme brain aging conferred a 3.1-fold increase in Alzheimer's disease risk (HR = 3.11, P = 1.41 × 10−28), and extreme brain youth conferred a 74% reduced risk of Alzheimer's disease (HR = 0.26, P = 4.37 × 10−4), independent of age, sex, APOE4 and APOE2 (Fig. Consequently, individuals with aged brains exhibited a 13.5-fold increased risk of developing Alzheimer's disease compared to those with youthful brains. Remarkably, when comparing the effects of brain age to APOE genotype, having an aged brain posed a risk similar to carrying one copy of APOE4, whereas having a youthful brain provided protection similar to carrying two copies of APOE2 (Fig. Brain age was very weakly correlated with APOE genotype (Fig. Over the 17-year follow-up period, 120 of 2,628 individuals (4.56%) with aged brains developed Alzheimer's disease, whereas only seven of 1,998 individuals (0.35%) with youthful brains developed the disease. The specificity of the association between brain aging and dementia led us to investigate whether organ age gaps were associated with brain volume based on MRI data from follow-up visits (Extended Data Fig. We found that the brain age gap at baseline visit was uniquely associated with increased volume of the ventricles and decreased volume of cortical regions at Instance 2 (~9 years later on average). MRI brain age was correlated with chronological age (r = 0.81, mean absolute error (MAE) = 3.65; Extended Data Fig. increase in the MRI brain age gap conferred a 3.2-times increased risk of future Alzheimer's disease (HR = 3.21, P = 2.55 × 10−36), confirming strong relevance to functional brain aging. Interestingly, however, MRI-based and plasma-based brain age gaps were only weakly correlated with each other (r = 0.18, P = 2.50 × 10−30; Fig. 2f), suggesting that each captures unique components of brain aging. Although this weak correlation is likely due, in part, to the approximately 9-year duration between MRI and plasma collection, a recent study showed that plasma brevican (BCAN) and glial fibrillary acidic protein (GFAP), but not other brain aging proteins identified in our study, were associated with MRI brain age17, suggesting that biological differences also contribute to the weak correlation. We speculate that MRI brain age captures global cell loss, whereas plasma brain age captures some of these volumetric changes as well as molecular alterations related to cell states and interactions. We also assessed organ age gap associations with disease progression, by regressing age gaps against years since diagnosis, for individuals who were diagnosed with disease before blood draw. We found that many organ age gaps increased throughout chronic kidney disease progression (Extended Data Fig. Interestingly, the brain age gap was not associated with dementia progression (Extended Data Fig. 5f), suggesting that it captures age-related changes that may predispose individuals to dementia but does not reflect changes occurring after disease onset. We next sought to gain further insights into organ aging by examining aging model proteins and their weights (Extended Data Fig. Aging models trained on the whole baseline sample versus only the randomly selected (86%) baseline samples resulted in essentially equivalent aging models and weights (Supplementary Fig. The strongest weighted protein in the brain aging model was neurofilament light chain (NEFL; Fig. 2h), which increases with age and is a clinical biomarker of axon degeneration that is often measured in clinical trials for Alzheimer's disease18,19 and was recently approved as a surrogate endpoint for a clinical trial to treat superoxide dismutase 1 amyotrophic lateral sclerosis (SOD1 ALS)20,21. Our data suggest that it, combined with other proteins, may also be a viable surrogate endpoint for brain aging and risk for dementia in normal people. Other highly weighted brain aging proteins include myelin oligodendrocyte protein (MOG), a component of the outer surface of myelin sheaths, and GFAP, a marker of reactive astrocytes, which both increased with age, as well as BCAN, a brain extracellular matrix component produced by oligodendrocyte precursor cells, and protein tyrosine phosphatase receptor type R (PTPRR), which both decreased with age (Fig. Plasma NEFL, GFAP and BCAN were previously highlighted as predictors of future dementia risk22. Using the permutation feature importance for biological aging (FIBA) algorithm4, we found that all of the top seven most highly weighted brain aging proteins contributed to the prediction of Alzheimer's disease risk, showing the importance of leveraging the information from many brain-derived proteins to understand brain aging and disease risk (Extended Data Fig. We then sought to determine which cell types these proteins were likely derived from by analyzing public human brain single-cell RNA sequencing data23 (Extended Data Fig. Interestingly, we found that approximately half of brain aging proteins were specific to the oligodendrocyte lineage, with the rest expressed mostly in neurons and then astrocytes (Fig. 2i), pointing to white matter as a key aging region, as suggested by human brain MRI and mouse brain RNA sequencing studies24,25. Lung aging was explained primarily by lysosome-associated membrane glycoprotein 3 (LAMP3), a protein expressed specifically in type II alveolar stem cells; secretoglobin family 1A member 1 (SCGB1A1), also known as club cell secretory protein (CCSP), a marker of club cells; and C-C motif chemokine ligand 18 (CCL18), a cytokine expressed by alveolar macrophages, potentially reflective of stem cell dysfunction and inflammation in the lung with age. Heart aging was explained primarily by N-terminal pro-B-type natriuretic peptide (NT-proBNP), a vasodilating hormone that increases in response to heart damage, whereas kidney aging was explained by renin (REN), a protein involved in blood pressure regulation. All aging model protein weights are provided in Supplementary Table 3, and the top 20 proteins for each model are shown in Extended Data Fig. Together, these data show that plasma protein-derived organ age estimates are linked with age-related organ diseases and can reveal insights into the aging biology of their respective organs. We next explored whether biological age estimates grounded in physiological states of organ function are sensitive to changes in lifestyle. We tested the associations between all 13 z-scored age gaps and 18 lifestyle factors (that is, diet, alcohol, smoking, exercise and insomnia) and socioeconomic factors (that is, education and Townsend Deprivation Index), adjusted for each other as well as age and sex, using linear regression (Supplementary Table 9). We found 69 positive and 57 negative significant associations (q < 0.05) after correcting for multiple hypothesis testing. In line with their known health associations, smoking, alcohol, processed meat intake, the Townsend Deprivation Index and insomnia were associated with age acceleration across several organs, whereas vigorous exercise, oily fish consumption, poultry consumption and higher education were associated with youthful organs (Fig. a, Linear regression was used to determine the association between age gaps and modifiable lifestyle factors while accounting for all other lifestyle factors as well as age and sex. The heatmap is color coded by signed log10(q). Only significant (q < 0.05) values are indicated by color coding. b, Linear regression was used to determine the association between age gaps and drugs/supplements intake while accounting for age and sex. Only drugs with significant (q < 0.05) youthful associations in at least two organs are shown. Heatmap is color coded by signed log10(q value). Only significant (q < 0.05) values are indicated by color coding. c, Multivariate linear regression was used to determine the association between age gaps versus early menopause and estrogen treatment independent of each other. Bar plot showing signed log10(P value) for menopause and estrogen covariates is shown. d, Box plot visualization of immune age gaps in individuals stratified by menopause status and estrogen treatment (n = 47). We also tested the associations between all 13 z-scored age gaps and consumption of 137 drugs/supplements (n cases ≥ 100; Supplementary Table 10). After multiple hypothesis test correction, we found six products—Premarin, ibuprofen, glucosamine, cod liver oil, multivitamins and vitamin C—that were significantly (q < 0.01) associated with youth in at least two organs (Fig. Ibuprofen, glucosamine, cod liver oil, multivitamins and vitamin C products were associated with youth primarily in the kidneys, brain and pancreas (Fig. Premarin is a conjugated estrogen medication typically prescribed to women experiencing postmenopausal symptoms, and estrogen medication was recently linked to reduced mortality risk in the UKB26. Thus, we wondered whether estrogen medications may extend longevity by preventing menopause-induced accelerated aging of organs and disease risk27. 3c), whereas estrogen treatment correlated with youthful immune, liver and artery profiles (Fig. Although cross-sectional analyses should be interpreted with caution, these findings underscore the sensitivity of plasma protein-derived organ age estimates to environmental factors and their potential for evaluating the effects of interventions. We next sought to determine whether organ age estimates were associated with future mortality. We tested associations between organ age gaps and all-cause mortality risk, adjusting for age and sex, over a 2–17-year follow-up using Cox proportional hazard regression (Supplementary Table 11). All organs showed significant associations, with a 20–60% increased mortality risk per s.d. 4a), similar to findings from SomaScan-based organ aging models4. These associations were robust to adjustment with blood cystatin C, a marker of kidney filtration rate, and PhenoAge, an established blood biochemistry/cell-count-based biological age estimate (whose age gap has a mortality risk HR of 1.38 in the UKB), suggesting that organ age estimates provide independent information not captured by existing clinical biomarkers. Surprisingly, brain aging was most strongly predictive (s.d. increase in brain age gap, HR = 1.59, P = 2.16 × 10−293; Fig. 4a), suggesting that the brain may be a central regulator of lifespan in humans similar to findings in animal models (worms, flies and mice)28,29,30. Indeed, individuals with aged brains had increased risk for several diseases beyond dementia, including COPD and heart failure (Extended Data Fig. 5a,b), consistent with previous studies showing that the brain regulates systemic inflammation31,32,33,34,35. a, Bar plot showing results from Cox proportional hazard regression analyses, testing the associations between each age gap and future all-cause mortality risk, controlling for labeled covariates (n = 21,775). PhenoAge age gap HR adjusted for age and sex (HR = 1.38) is shown as a dashed line for reference. b, Concordance indices from various LASSO-regularized Cox proportional hazard models were trained to predict mortality risk. Performance across train and test centers is shown. Covariates for each model, in addition to age-at-blood-draw and sex, are labeled on the x axis. c, Model coefficients are shown for the combined model (OrganAge + PhenoAge + CysC) from b. d, Forest plot showing results from Cox proportional hazard regression, testing the associations between extreme ager status (‘+' refers to aged; ‘−' refers to youthful) and future all-cause mortality risk, controlling for age-at-blood-draw and sex. e, Kaplan–Meier curves with 95% confidence intervals showing survival over a 17-year follow-up for normal agers, multi-organ agers (with 2–4, 5–7 or 8+ aged organs) and individuals with a youthful brain or immune system (brain– or immune–). f,g, Gene Ontology pathway enrichment analyses from the top 10 brain (f) and immune (g) aging proteins, as determined from the mortality risk FIBA score (Methods). To test whether organ age estimates provided additional predictive power, beyond cystatin C and PhenoAge, we compared concordance indices of mortality risk Cox models that included cystatin C, PhenoAge, organ ages or combinations, with age and sex as covariates. Organ ages alone performed similarly to the combined model, outperforming cystatin C and PhenoAge, indicating that they capture additional predictive information (Fig. Brain age, conventional age, PhenoAge and sex were key predictors in the combined model (Fig. Application of FIBA to understand contributions of brain and conventional aging model proteins on mortality risk highlighted BCAN, NEFL and PTPRR from the brain as well as ectodysplasin A2 receptor (EDA2R, organismal protein), chemokine C-X-C motif ligand 17 (CXCL17, organismal protein) and elastin (ELN, artery protein) from the conventional aging model as important proteins (Extended Data Fig. Although each organ age gap was associated with risk of death, we wondered whether the accrual of aged organs would be increasingly detrimental. Hence, we tested whether single-organ extreme agers and multi-organ agers (bins of 2–4, 5–7 and 8+ organs) had an increased risk of death compared to normal agers, while adjusting for age and sex (Supplementary Table 12). Interestingly, we found that, whereas having a single aged organ (brain, lung, intestine, heart, immune, kidney, liver or pancreas) conferred a 1.5–3-fold increased risk of death, having 2–4, 5–7 and 8+ extremely aged organs conferred a 2.3-fold, 4.5-fold and 8.3-fold (P = 2.05 × 10−70, P = 3.86 × 10−106 and P = 8.30 × 10−127) increased risk of death, respectively (Fig. Notably, over 60% of individuals with 8+ extremely aged organs at blood draw died within 15 years (Fig. We then sought to determine whether youthful organ profiles were associated with longevity (Supplementary Table 12). We found that individuals with youthful-appearing arteries had increased mortality risk, and those with multi-organ youth had no difference in mortality risk compared to normal agers (Fig. Using FIBA, we found that artery protein thrombospondin 2 (THBS2), a protein that decreases with age but has a positive association with mortality risk, was responsible for the nonlinear association between the artery age gap and mortality risk (Extended Data Fig. Why individuals with broad multi-organ youth are not protected is unclear, although this may be due to limited sample size. Notably, though, individuals with youthful brains (HR = 0.60, P = 7.49 × 10−3) and immune systems (HR = 0.58, P = 7.34 × 10−3) had significantly reduced mortality risk (Fig. 4d), similar to their unique protective associations with disease (Extended Data Fig. We, therefore, assessed individuals with both youthful brains and immune systems and found that this group was most strongly protected from mortality by effect size (HR = 0.44, P = 0.042; Fig. 4d,e), underscoring the benefits of having a resilient brain and immune system. Over 17 years, 792 of 10,000 (7.92%) normal agers died, whereas only six of 160 individuals (3.8%) with youthful brains and immune systems died. To probe the underlying mechanisms implicated in brain-related and immune-system-related longevity, we performed Gene Ontology enrichment analyses of the top 10 brain and immune aging model proteins based on mortality risk FIBA scores (Fig. Selection of brain-specific or immune-system-specific plasma proteins as background for the enrichment test did not result in significant associations, so we used all genes as background. Perineuronal net was the most enriched brain pathway (BCAN, PTPRZ1 and NCAN), and secretory granule was the most enriched immune system pathway. Intriguingly, neuroinflammation was another enriched immune system pathway, composed of metalloproteinase 9 (MMP9), a regulator of perineuronal net composition, as well as tumor necrosis factor receptor superfamily member 1B (TNFSRF1B) and integrin alpha M (ITGAM), also known as CD11B. Youthful brain agers had higher levels of BCAN and NCAN, and youthful immune agers had lower levels of MMP9 (Extended Data Fig. 6h,i), suggesting that global preservation of brain extracellular matrix partly due to reduced degradation by peripheral inflammatory factors and reduced chronic inflammation partly regulated by the brain could be crucial for promoting longevity. Together, these data reveal that plasma protein-derived organ age estimates improve upon existing aging biomarkers for mortality risk prediction, the accrual of aged organs progressively increases mortality risk, and a youthful brain and immune system are key to longevity. Our findings based on plasma proteomics data (~3,000 proteins) from nearly 45,000 individuals in the UKB establish plasma protein-derived organ age estimates as robust indicators of organ age, health and disease/mortality risk beyond gold standard clinical aging biomarkers and reveal key proteins implicated in the aging process. Furthermore, we show that organ age estimates are stable across train/test centers and longitudinal visits and are cross-sectionally associated with modifiable lifestyle choice, socioeconomic status and medications and, thus, lay the foundation for human experiments testing the effects of novel longevity interventions on the biological age of organs at the individual level. We note key considerations and limitations to inform future work. Although our organ enrichment classification based on bulk RNA sequencing atlases yielded robust results, confirming the true protein sources remains challenging; high-resolution gene expression maps including information on alternative splicing and changes with age and disease could strengthen confidence. Like many studies, we relied on cross-sectional age gaps, which only approximate aging rate histories. Longitudinal proteomics data, integrated with deep environmental and genetic phenotyping, are essential to accurately measure aging rates, distinguish baseline differences and uncover their causes16,36,37,38. Such data could also clarify the sequence of organ aging at individual and population levels. Although we observed that multi-organ aging increases with age, suggesting cumulative organ decline, the specific order of organ aging remains undefined. Surprisingly, individuals with highly youthful organ profiles (except for brain and immune system) were not protected from mortality, despite assumptions of benefit. This aligns with studies showing U-shaped associations between clinical biomarkers (for example, body mass index and platelet count) and mortality, where extremely youthful phenotypes are associated with elevated risk39,40,41, warranting further exploration in aging biomarker research. Although organ age estimates independently predict mortality beyond clinical biochemistry biomarkers, their added clinical value over established disease-specific biomarkers requires disease-by-disease investigation42. Finally, as UKB participants are predominantly of European ancestry, organ age estimates may need recalibration for diverse genetic and environmental contexts, necessitating broader studies. In the present study, we found evidence that the brain and immune system may be central regulators of aging and longevity in humans, as aged brains are most strongly predictive of earlier mortality, and youthful brains and immune systems are uniquely predictive of longevity. After all, the brain regulates numerous critical age-related functions throughout the body, including circadian rhythm, blood pressure, energy homeostasis and stress response, via the neuroendocrine and autonomic nervous systems, and chronic inflammation has been heavily implicated in aging1,43. These observations suggest that accelerated aging or maintenance of youth in the brain and immune system likely has broad age-related effects across the body, although additional studies are needed to test this hypothesis more rigorously in humans. Regarding the molecular alterations that occur with brain aging, we found many unexpected age-associated and disease-associated changes in oligodendrocyte lineage and extracellular matrix proteins, implying extensive changes beyond neuroinflammation and neurodegeneration. Moreover, the observation that white matter regions exhibit the most pronounced shifts in aging microglial transcriptomes25, coupled with the enrichment of genetic risk variants for neurodegenerative diseases in microglial and oligodendrocyte genes49, underscores a potential link between oligodendrocyte aging and age-related neuroinflammation and their relevance to neurodegeneration. Future studies exploring these multicellular and extracellular matrix interactions in the aging brain and their interactions with the periphery may reveal key insights into human health and longevity. The UKB is a population-based prospective cohort with omics and phenotypic data collected on approximately 500,000 participants, aged 40–69 years at recruitment, between 2006 and 2010. A subset of participants had subsequent visits after the baseline assessment: 20,337 participants had a first repeat assessment (Instance 1; 2012–2013); approximately 85,000 had a first MRI imaging visit (Instance 2; from 2014 to ongoing); and approximately 9,000 had a second MRI imaging visit (Instance 3; from 2019 to ongoing). Details on available phenotypes can be found at https://biobank.ndph.ox.ac.uk/showcase/, and all participants provided informed consent. The UKB-PPP consortium generated Olink Explore 3,072 proteomics data from blood plasma samples collected from 54,219 UKB participants. These data consisted of 46,595 randomly selected samples from the baseline visit and 6,376 additional baseline visit samples selectively prioritized by the UK-PPP (samples presumably enriched for rare diseases and polymorphisms). Additional Olink proteomics data (using an earlier version of the platform, ~1,500 proteins) were generated from 1,268 participants who participated in the COVID-19 repeat imaging study. Additional details on the proteomics data are provided in Sun et al.10 and at https://biobank.ndph.ox.ac.uk/ukb/ukb/docs/PPP_Phase_1_QC_dataset_companion_doc.pdf. UKB data were analyzed under application number 45420. We defined participants' last known age as either age at death or the difference between the latest date available in International Classification of Diseases (ICD)-9, ICD-10, operating procedure, cancer registry or UKB assessment visit fields and birth date; this age corresponds to censoring in the following survival analyses. The following disease groups were defined, as in ref. For heart-related diseases, the group definitions were based on ref. Medications reported in the verbal interview (https://biobank.ctsu.ox.ac.uk/crystal/label.cgi?id=100075) were also analyzed with a minimum of 100 participants per medication. Brain MRI-derived phenotypes extracted with FreeSurfer version 6 (https://biobank.ctsu.ox.ac.uk/crystal/label.cgi?id=110 and https://biobank.ctsu.ox.ac.uk/crystal/refer.cgi?id=1977) were analyzed, and age at MRI as well as estimated total intracranial volume were regressed out. The post-UKB-PPP quality control baseline visit data consist of 2,923 protein measurements from 53,018 samples. We performed additional quality control steps (Supplementary Fig. 2a) in the following order: 8,182 samples with more than 1,000 protein missing values were removed; seven proteins with missing values in over 10% of samples were removed; and 338 samples with discordant reported sex and genetic sex were removed. This resulted in a post-quality control dataset consisting of 44,498 samples with 2,916 protein measurements. We then performed missing value imputation of the proteomics data (Supplementary Fig. First, we split the data into train and test, with each split comprising 11 randomly selected centers (train centers: 11002, 11005, 11006, 11007, 11008, 11009, 11013, 11014, 11017, 11018 and 11023; test centers: 10003, 11001, 11003, 11004, 11010, 11011, 11012, 11016, 11020, 11021 and 11022). Protein values were z-score normalized based on the means and standard deviations of protein values in the train split. We evaluated the imputer on a subset of 5,591 samples (2,967 train and 2,624 test) with zero original missing values. Specifically, we randomly inserted missing values into this ‘ground truth' subsample at a rate equivalent to the missing value rate in the whole post-quality control dataset (3%). We then performed imputation on this subsample to calculate the error between imputed values and original ground truth values. We confirmed robust imputation with a total MAE of 0.57 (Supplementary Fig. 2c,d), which was consistent across both train and test. This is a relatively small error, considering that the data distribution is approximately 5 (z-score ±2.5 covers 99% of the distribution). Samples were acquired through the National Institute on Aging–funded Stanford ADRC. The Stanford ADRC cohort is a longitudinal observational study of individuals with clinical dementia and age-matched and sex-matched individuals without dementia. All healthy control participants were deemed cognitively unimpaired during a clinical consensus conference that included board-certified neurologists and neuropsychologists. Cognitively impaired participants underwent Clinical Dementia Rating and standardized neurological and neuropsychological assessments to determine cognitive and diagnostic status, including procedures of the National Alzheimer's Coordinating Center (https://naccdata.org/). Cognitive status was determined in a clinical consensus conference that included neurologists and neuropsychologists. All participants were free from acute infectious diseases and in good physical condition. SAMS is an ongoing longitudinal study of healthy aging. Blood and cerebrospinal fluid (CSF) collection and processing were done by the same team and using the same protocol as in the Stanford ADRC. All SAMS participants had Clinical Dementia Rating = 0 and a neuropsychological test score within the normal range, and all SAMS participants were deemed cognitively unimpaired during a clinical consensus conference that included neurologists and neuropsychologists. Plasma and CSF collection, processing and storage for all Stanford cohorts were performed using a single standard operating procedure. All studies were approved by the institutional review board of Stanford University, and written informed consent or assent was obtained from all participants or their legally authorized representative. In brief, approximately 10 cc of whole blood was collected in four vacutainer EDTA tubes (Becton Dickinson) and spun at 1,800g for 10 minutes to separate out plasma, leaving 1 cm of plasma above the buffy coat and taking care not to disturb the buffy coat to circumvent cell contamination. Plasma was aliquoted into polypropylene tubes and stored at −80 °C. All blood draws were done in the morning to minimize the impact of circadian rhythm on protein concentrations. CSF was collected via lumbar puncture using a 20–22-gauge spinal needle that was inserted in the L4–L5 or L5–S1 interspace. CSF samples were immediately centrifuged at 500g for 10 minutes, aliquoted in polypropylene tubes and stored at −80 °C. Olink Explore 3,072 plasma proteomics data from 601 samples with matched SomaScan plasma proteomics data were analyzed in this study. We used the same methods that we developed in our previous study4 to identify putative plasma proteins. In brief, we identified organ-enriched genes: genes that were expressed at least four times higher in a single organ compared to any other organ based on human tissue bulk RNA sequencing data from the Gene Tissue Expression Atlas (Supplementary Fig. We refer to our previous study for details4. Our classification of organ-enriched genes is provided in Supplementary Table 1. We mapped these genes to proteins measured by Olink to identify organ-enriched plasma proteins in the UKB dataset (Supplementary Table 2). Forty-eight of the 2,916 plasma proteins in the UKB dataset were not detected in Jiang et al. tissue proteomics data. Of the detected proteins, 80% of those we called ‘organ-enriched' were also called organ-enriched or organ-specific in the same organ in Jiang et al. (Supplementary Fig. 1e), suggesting strong concordance between RNA and protein-based enrichment classification. Twelve percent of our organ-enriched plasma proteins were not enriched in Jiang et al., and 8% were enriched in a different organ (Supplementary Fig. Most of the discordant proteins (20%) were secreted proteins produced by the liver; we called these proteins ‘liver-enriched' based on RNA, but, based on Jiang et al. tissue proteomics, they were called either artery-enriched or non-enriched (Supplementary Fig. This highlights a key distinction between organ enrichment classification based on tissue transcriptomics versus proteomics: transcriptomics captures the source of the protein, whereas proteomics captures its destination. Although understanding inter-organ communication by studying the source versus destination of proteins is an interesting area for future study, for our current study, our goal was to determine the putative organ source of plasma proteins to infer organ age. We used LASSO regression models to build chronological age predictors (also known as aging models) to estimate biological age. The lambda value that achieved 95% of the performance of the highest-performing lambda value was applied to scikit-learn's LASSO function to derive sparse aging models. Each organ aging model was trained using a distinct set of organ-enriched plasma proteins. An organismal aging model was trained using non-organ-enriched proteins. A conventional aging model was trained using all proteins. Age gaps were z-scored per aging model to normalize for differences in age prediction accuracy. z-scored age gaps were used for all analyses. Extreme agers were defined as individuals with an age gap z-score greater than 1.5 or a z-score less than –1.5 in a given aging model. Conventional age gaps were not included in the extreme ager analyses due to their high similarity to organismal age gaps (Fig. All aging model weights are provided in Supplementary Table 3. In addition to aging models trained on only proteins, we also assessed aging models trained on proteins and sex as well as aging models trained separately per sex. Pairwise correlations between age gaps produced by these different modeling frameworks showed very similar age gaps between the different modeling frameworks (Supplementary Fig. Age gap associations with mortality were highly similar across frameworks (Supplementary Fig. Extreme ager associations with mortality were similar, but sex-specific models showed no significant associations between extreme youth and longevity, although brain and immune youth trended toward significance (Supplementary Fig. For simplicity and robustness, we focused on aging models trained on only proteins for all primary analyses in this paper. We also assessed aging models trained on only the randomly selected participants (86%) in the dataset. As mentioned previously, 14% of the samples were prioritized by the UKB-PPP to study specific diseases of interest. Aging models trained on the whole dataset and aging models trained on only the randomly selected subset produced nearly equivalent age gaps (r = 1.00; Supplementary Fig. Age gap associations with incident disease and mortality were also nearly equivalent when examining either all participants or only randomly selected participants (r = 0.99 and r = 0.99; Supplementary Fig. Given these consistencies, we analyze the entire dataset in this paper. Longitudinal age gap analyses (Extended Data Fig. 3) required use of plasma proteomics data collected across multiple visits (baseline, Instance 2 and Instance 3) from the same individual. However, proteomics data from post-baseline samples were obtained from an earlier version of the Olink assay with 1,459 proteins and were not compatible with models trained on the approximately 3,000-protein platform. Therefore, for longitudinal age gap analyses, we trained a distinct set of organ aging models using the subset of 1,459 proteins that were measured across all visits. Four proteins with missing values in over 10% of samples were removed, leaving 1,459 proteins for model development. For model training, missing values for baseline samples were k-nearest neighbor imputed as described above. 1,500-protein-based aging models were trained on 44,406 baseline samples from individuals who did not have follow-up proteomics data. Samples from individuals with longitudinal proteomics data were not included in model training to prevent model training/evaluation contamination. To maximize the number of testable samples with unimputed data, we aimed to maximize sparsity of the models. The lambda value that achieved 90% of the performance of the highest-performing lambda value was used, followed by recursive feature elimination using scikit-learn's RFECV function. Liver and muscle 1,500-based aging models were removed due to low correlation (r < 0.8) with 3,000-based aging models. Models were tested and evaluated on longitudinal data from 1,176 unique individuals who had non-missing values for all remaining aging model proteins (880 baseline, 843 Instance 2 and 786 Instance 3 samples). We trained an MRI-based brain aging model using 479 FreeSurfer brain volumetric phenotypes from 45,574 individuals from Instance 2 (Extended Data Fig. This model was trained using the same pipeline as used for the organ aging models (LassoCV, five-fold cross-validation, lambda value with 95% performance). MRI brain age gaps from Instance 2 were correlated with plasma brain age gaps from baseline. Cox proportional hazard regression (CoxPHFitter function from the lifelines54 Python package) was used to assess the associations between organ age gaps and future disease or mortality risk. Linear regression (OLS function from the statsmodels55 Python package) was used to assess the associations between organ age gaps and environmental factors recorded at the time of blood draw. Multiple hypothesis testing correction was applied, when appropriate, using the Benjamini–Hochberg method, and the significance threshold was a 5% false discovery rate. Gene Ontology pathway enrichment analyses were performed using gProfiler56. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. UK Biobank data are available upon request to qualified researchers through a standard protocol (https://www.ukbiobank.ac.uk/register-apply). Organ age estimates for all participants analyzed in this study will be made available to UK Biobank researchers through Showcase. Stanford Olink/SomaScan from plasma/CSF and associated patient metadata are available upon reasonable request to the Stanford ADRC data release committee (https://web.stanford.edu/group/adrc/cgi-bin/web-proj/datareq.php). Data from specific Stanford cohorts can be requested to the following cohort leaders: ADRC, T.W.-C. (twc@stanford.edu); SAMS, E.M. (bmormino@stanford.edu) or A.D.W. Raw tissue bulk RNA sequencing data from the Adult Genotype Tissue Expression (GTEx) Project are available at the GTEx website (https://www.gtexportal.org/home/aboutAdultGtex). Public brain single-cell RNA sequencing datasets used in this study23 are available in the Gene Expression Omnibus under accession code GSE254205. Predicted age is the linear combination of protein coefficients and z-scored protein levels, plus the intercept value. All aging model coefficients are provided in Supplementary Table 3. A Python tutorial on applying these coefficients to independent datasets is available in the organageUKB GitHub repository (https://github.com/hamiltonoh/organageUKB). López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. Personal aging markers and ageotypes revealed by deep longitudinal profiling. Tian, Y. E. et al. Heterogeneous aging across multiple organ systems and prediction of chronic disease and mortality. Organ aging signatures in the plasma proteome track health and disease. Sehgal, R. et al. Systems Age: a single blood methylation test to quantify aging heterogeneity across 11 physiological systems. Plasma protein-based organ-specific aging and mortality models unveil diseases as accelerated aging of organismal systems. Herndon, L. A. et al. Stochastic and genetic factors influence tissue-specific decline in ageing C. elegans. Schaum, N. et al. Ageing hallmarks exhibit organ-specific temporal signatures. Plasma proteomic associations with genetics and health in the UK Biobank. Kuo, C.-L. et al. Proteomic aging clock (PAC) predicts age-related outcomes in middle-aged and older adults. Trelle, A. N. et al. Hippocampal and cortical mechanisms at retrieval explain variability in episodic remembering in older adults. Trelle, A. N. et al. Association of CSF biomarkers with hippocampal-dependent memory in preclinical Alzheimer disease. Large-scale plasma proteomics comparisons through genetics and disease associations. Cole, J. H. Multimodality neuroimaging brain-age in UK biobank: relationship to biomedical, lifestyle, and cognitive factors. Leonardsen, E. H. et al. Genetic architecture of brain age and its causal relations with brain and mental disorders. Plasma proteomics identify biomarkers and undulating changes of brain aging. H. et al. Lecanemab in early Alzheimer's disease. Pontecorvo, M. J. et al. Association of donanemab treatment with exploratory plasma biomarkers in early symptomatic Alzheimer disease: a secondary analysis of the TRAILBLAZER-ALZ randomized clinical trial. Miller, T. M. et al. Trial of antisense oligonucleotide tofersen for SOD1 ALS. Irwin, K. E., Sheth, U., Wong, P. C. & Gendron, T. F. Fluid biomarkers for amyotrophic lateral sclerosis: a review. Plasma proteomic profiles predict future dementia in healthy adults. Haney, M. S. et al. APOE4/4 is linked to damaging lipid droplets in Alzheimer's disease microglia. Characterization of brain volume changes in aging individuals with normal cognition using serial magnetic resonance imaging. Hahn, O. et al. Atlas of the aging mouse brain reveals white matter as vulnerable foci. & Perez, K. Association between prescription drugs and all-cause mortality risk in the UK population. Long-term health consequences of premature or early menopause and considerations for management. Frakes, A. E. et al. Four glial cells regulate ER stress resistance and longevity via neuropeptide signaling in C. elegans. Sheng, L. et al. Ensheathing glia promote increased lifespan and healthy brain aging. Tokizane, K., Brace, C. S. & Imai, S. DMHPpp1r17 neurons regulate aging and lifespan in mice through hypothalamic-adipose inter-tissue communication. Koren, T. et al. Insular cortex neurons encode and retrieve specific immune responses. A. et al. A preoptic neuronal population controls fever and appetite during sickness. Brain motor and fear circuits regulate leukocytes during acute stress. Circulating myeloid-derived MMP8 in stress susceptibility and depression. Jin, H., Li, M., Jeong, E., Castro-Martinez, F. & Zuker, C. S. A body–brain circuit that regulates body inflammatory responses. The genetic architecture of biological age in nine human organ systems. Foy, B. H. et al. Haematological setpoints are a stable and patient-specific deep phenotype. Smith, S. M., Miller, K. L. & Nichols, T. E. Characterising ongoing brain aging and baseline effects from cross-sectional data. Bonaccio, M. et al. Age-sex–specific ranges of platelet count and all-cause mortality: prospective findings from the MOLI-SANI study. Aune, D. et al. BMI and all cause mortality: systematic review and non-linear dose-response meta-analysis of 230 cohort studies with 3.74 million deaths among 30.3 million participants. Karaphillis, E., Goldstein, R., Murphy, S. & Qayyum, R. Serum alanine aminotransferase levels and all-cause mortality. The use and misuse of ‘biological aging' in health research. Ulrich-Lai, Y. M. & Herman, J. P. Neural regulation of endocrine and autonomic stress responses. Wang, F. et al. Myelin degeneration and diminished myelin renewal contribute to age-related deficits in memory. Depp, C. et al. Myelin dysfunction drives amyloid-β deposition in models of Alzheimer's disease. Crapser, J. D. et al. Microglia facilitate loss of perineuronal nets in the Alzheimer's disease brain. Brain cell type–specific enhancer–promoter interactome maps and disease risk association. Argentieri, M. A. et al. Proteomic aging clock predicts mortality and risk of common age-related diseases in diverse populations. Schuermans, A. et al. Integrative proteomic analyses across common cardiac diseases yield mechanistic insights and enhanced prediction. Jiang, L. et al. A quantitative proteome map of the human body. Pedregosa, F. et al. Scikit-learn: machine learning in Python. Seabold, S. & Perktold, J. Statsmodels: econometric and statistical modeling with Python. 9th Python in Science Conference (eds van der Walt, S. & Millman, J.) Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). This work was supported by the Stanford Alzheimer's Disease Research Center (National Institute on Aging grants P50AG047366 and P30AG066515); the National Institute on Aging (AG072255, T.W.-C); the Milky Way Research Foundation (T.W.-C.); and the Knight Initiative for Brain Resilience (T.W.-C.). The Stanford Aging and Memory Study was supported by the National Institutes of Health (NIH) (R01AG048076, A.D.W. This work was further supported by the Stanford Graduate Fellowship (H.S.O. ), the National Science Foundation Graduate Research Fellowship (H.S.O.) Hamilton Se-Hwee Oh, Deniz Yagmur Urey, Amelia Farinas, Divya Channappa & Tony Wyss-Coray Hamilton Se-Hwee Oh, Deniz Yagmur Urey, Amelia Farinas, Divya Channappa, Anthony D. Wagner, Anne Brunet & Tony Wyss-Coray Yann Le Guen, Divya Channappa, Elizabeth Mormino, Michael D. Greicius & Tony Wyss-Coray Quantitative Sciences Unit, Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA Search author on:PubMed Google Scholar led data processing of UK Biobank clinical phenotypes and advised on analyses. aided in longitudinal organ age gap analyses and figure generation. aided in disease progression analyses and figure generation. aided in Stanford data generation and analyses. All authors critically revised the paper for intellectual content. Correspondence to Hamilton Se-Hwee Oh or Tony Wyss-Coray. The remaining authors declare no competing interests. Nature Medicine thanks Bradford Gibson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Michael Basson, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, For all aging models, the top 20 aging model proteins and their weights are shown. Pearson correlations (r) and mean absolute errors (MAE) shown. c, Difference in correlation between predicted and actual age by biological sex. d, Mean difference in organ age gaps between males and females. e, Extreme ager sample sizes and proportions. f, Age distributions per extreme ager group (n = 44,498). a, Longitudinal proteomics data from a subset of 1,176 individuals were analyzed (880 baseline, 843 Instance 2, and 786 Instance 3 samples). Longitudinal data were available only on the 1.5k-protein Olink assay, so new aging models trained on the 1.5k-assay were developed. New aging models were trained on 44,406 samples without longitudinal data and tested on non-imputed samples with longitudinal data. Only 1.5k-aging models with age estimates that were correlated r > 0.8 with 3k-based age estimates were included for downstream analyses. Correlation between baseline and Instance 2 age gaps are shown. Equivalent plot for youthful agers is shown on the right. c, Age gaps were grouped into bins of 0.5 standard deviation to determine changes in age gap bins across visits. Individual trajectories across visits for extreme immune agers are shown. Equivalent plot for youthful immune agers is shown at the bottom. d, Pie chart showing percent distribution of immune age gap bins in Instance 2 and Instance 3 for individuals who are extreme immune agers at baseline. Equivalent plot for youthful immune agers is shown at the bottom. e, Stacked bar plot showing percent distribution of age gap bins in Instance 2 and Instance 3 for individuals who are extreme agers at baseline. Equivalent plot for youthful agers is shown at the bottom. a, Stanford Olink data contained missing values for the 5 aging model proteins shown. Protein ranks by aging model coefficient compared to total number of proteins in aging model are shown. b, Correlation between predicted versus chronological age in UKB train, UKB test, and Stanford test (cognitively normal controls) data. c, Olink versus SomaScan (Oh and Rutledge et. al. 2023) organ age gaps in Stanford data. Linear regressions with 95% confidence intervals are shown. d, Distribution of correlations between Olink and SomaScan overlapping proteins by name (from Eldjarn et. e, g:Profiler biological pathway enrichment of brain and immune aging model proteins per proteomics platform. f, Brain age gaps versus Alzheimer's disease diagnosis, per proteomics platform (n = 598). a, Cox proportional hazards regression was used to test the association between age gaps and future disease risk, adjusted for age-at-blood-draw and sex. Heatmap colored by age gap log(hazard ratio) is shown. Heatmap columns are ordered by the Gini-coefficient of age gaps per disease. The most significant associations per disease are highlighted with black borders. b, Cox proportional hazards regression was used to determine the association between extreme agers and future disease risk, controlling for age and sex and other extreme agers. Heatmap colored by age gap log(hazard ratio) is shown. Non-significant hazard ratios (p < 0.05) were set to zero. c, Linear regression was used to determine the association between baseline organ age gaps and Instance 2 brain MRI volumes, controlling for age-at-blood-draw, age-at-MRI, sex, and estimated total intracranial volume. Non-significant effect sizes (q < 0.05) were set to zero. Pearson correlations and mean absolute errors are shown. e, Linear regression was used to determine the association between organ age gaps and years since disease diagnosis. Non-significant effects (q < 0.05) were set to zero. f, Visualization of results from e. Organ age gap versus years since diagnosis shown for chronic kidney disease x pancreas age gap and Alzheimer's disease x brain age gap. Lowess regressions with 95% confidence intervals are shown. a, Scatterplot showing results from feature importance for biological aging (FIBA) algorithm to identify proteins in the brain aging model contributing to the brain age gap's association with Alzheimer's disease risk. FIBA score (y-axis) indicates Alzheimer's disease risk effect size loss after permutation of protein values. X-axis indicates absolute protein weight in the brain aging model. Color indicates protein weight in the brain aging model. b, Mean gene expression of brain aging protein-encoding genes in Haney et al. 2024 human brain scRNA-seq data. c-f, Scatterplots showing results from feature importance for biological aging (FIBA) algorithm to identify proteins in the brain (c), conventional (d), artery (e), and immune (f) aging models that contribute to the model age gap's association with future mortality risk. FIBA score (y-axis) indicates mortality risk effect size loss after permutation of protein values. X-axis indicates absolute protein weight in the aging model. Color indicates protein weight in the aging model. g, Forest plot showing results from Cox proportional hazards regression, testing the associations between extreme ager status and future all-cause mortality risk, controlling for age, sex. Points show extreme ager hazard ratios, error bars show 95% confidence intervals, and number on the right show number of events out of the total sample size. The top ten (5 decrease with age, 5 increase with age) proteins based on mortality risk FIBA score are shown. Each protein was linearly adjusted for age, sex, and every other protein in the brain aging model before plotting. Proteins are ordered by the aging model coefficient. i, As in h, but for the immune aging model (n = 12,847). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250708045703.htm'>Ice in a million-degree Fermi bubble reveals the Milky Way's recent eruption</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 10:01:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have found clouds of cold gas embedded deep within larger, superheated gas clouds -- or Fermi bubbles -- at the Milky Way's center. "Fermi bubbles are a relatively recent discovery -- they were first identified by telescopes that 'see' gamma rays in 2010 -- there are different theories about how it happened, but we do know that it was an extremely sudden and violent event, like a volcanic eruption but on a massive scale." These measurements were twice as sensitive as previous radio telescope surveys of the Fermi bubbles and allowed them to observe finer detail within the bubbles. "They're around 10,000 degrees Kelvin, so cooler than their surroundings by at least a factor of 100. Their existence is surprising because the hot (over 1 million degrees Kelvin), high-velocity environment of the nuclear outflow should have rapidly destroyed any cooler gas. "Computer models of cool gas interacting with hot outflowing gas in extreme environments like the Fermi bubbles show that cool clouds should be rapidly destroyed, usually within a few million years, a timescale that aligns with independent estimates of the Fermi bubbles' age," Bordoloi says. "It wouldn't be possible for the clouds to be present at all if the Fermi bubbles were 10 million years old or older. "What makes this discovery even more remarkable is its synergy with ultraviolet observations from the Hubble Space Telescope (HST)," Bordoloi says. "The clouds lie along a sightline previously observed with HST, which detected highly ionized multiphase gas, ranging in temperatures from a million to 100,000 Kelvin -- which is what you'd expect to see if a cold gas is getting evaporated." "These gases are moving around a million miles per hour, which also marks the Fermi bubbles as a recent development," Bordoloi says. "These clouds weren't here when dinosaurs roamed Earth. In cosmic time scales, a million years is the blink of an eye." This discovery challenges current understanding of how cold clouds can survive the extreme energetic environment of the Galactic Center, placing strong empirical constraints on how outflows interact with their surroundings. The findings provide a crucial benchmark for simulations of galactic feedback and evolution, reshaping our view of how energy and matter cycle through galaxies. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250708045701.htm'>Hidden DNA-sized crystals in cosmic ice could rewrite water—and life itself</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 07:12:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>"Space ice" contains tiny crystals and is not, as previously assumed, a completely disordered material like liquid water, according to a new study by scientists at UCL (University College London) and the University of Cambridge. They found that computer simulations of this ice best matched measurements from previous experiments if the ice was not fully amorphous but contained tiny crystals (about three nanometers wide, slightly wider than a single strand of DNA) embedded within its disordered structures. If the ice had been fully amorphous (fully disordered), the researchers concluded, it would not retain any imprint of its earlier form. The findings also have implications for one speculative theory about how life on Earth began. According to this theory, known as Panspermia, the building blocks of life were carried here on an ice comet, with low-density amorphous ice the space shuttle material in which ingredients such as simple amino acids were transported. Dr Davies said: "Our findings suggest this ice would be a less good transport material for these origin of life molecules. That is because a partly crystalline structure has less space in which these ingredients could become embedded. "The theory could still hold true, though, as there are amorphous regions in the ice where life's building blocks could be trapped and stored." Co-author Professor Christoph Salzmann, of UCL Chemistry, said: "Ice on Earth is a cosmological curiosity due to our warm temperatures. You can see its ordered nature in the symmetry of a snowflake. "Ice in the rest of the Universe has long been considered a snapshot of liquid water -- that is, a disordered arrangement fixed in place. "Our results also raise questions about amorphous materials in general. These materials have important uses in much advanced technology. If they do contain tiny crystals and we can remove them, this will improve their performance." They froze these virtual "boxes" of water molecules by cooling to -120 degrees Centigrade at different rates. They found that ice that was up to 20% crystalline (and 80% amorphous) appeared to closely match the structure of low-density amorphous ice as found in X-ray diffraction studies (that is, where researchers fire X-rays at the ice and analyse how these rays are deflected). Using another approach, they created large "boxes" with many small ice crystals closely squeezed together. In additional experimental work, the research team created real samples of low-density amorphous ice in a range of ways, from depositing water vapor on to an extremely cold surface (how ice forms on dust grains in interstellar clouds) to warming up what is known as high-density amorphous ice (ice that has been crushed at extremely cold temperatures). This was indirect evidence, they said, that low-density amorphous ice contained crystals. If it was fully disordered, they concluded, the ice would not retain any memory of its earlier forms. Amorphous ice was first discovered in its low-density form in the 1930s when scientists condensed water vapor on a metal surface cooled to -110 degrees Centigrade. Co-author Professor Angelos Michaelides, from the University of Cambridge, said: "Water is the foundation of life but we still do not fully understand it. Amorphous ices may hold the key to explaining some of water's many anomalies." Dr Davies said: "Ice is potentially a high-performance material in space. It could shield spacecraft from radiation or provide fuel in the form of hydrogen and oxygen. So we need to know about its various forms and properties." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250708045700.htm'>North america's oldest pterosaur unearthed in Arizona's Triassic time capsule</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 06:36:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A Smithsonian-led team of researchers have discovered North America's oldest known pterosaur, the winged reptiles that lived alongside dinosaurs and were the first vertebrates to evolve powered flight. In a paper published on July 7 in Proceedings of the National Academy of Sciences, researchers led by paleontologist Ben Kligman, a Peter Buck Postdoctoral Fellow at the Smithsonian's National Museum of Natural History, present the fossilized jawbone of the new species and describe the sea gull-sized pterosaur alongside hundreds of other fossils -- including one of the world's oldest turtle fossils -- unearthed at a remote bonebed in Petrified Forest National Park in Arizona. These fossils, which date back to the late Triassic period around 209 million years ago, preserve a snapshot of a dynamic ecosystem where older groups of animals, including giant amphibians and armored crocodile relatives, lived alongside evolutionary upstarts like frogs, turtles and pterosaurs. This cleared the way for newer groups, like dinosaurs, to diversify and dominate ecosystems worldwide. Direct evidence of this transition on land is difficult to find due to a lack of terrestrial fossil outcrops from right before the ETE. However, there are few better places to look than Petrified Forest National Park, which is famed for its Triassic fossil beds and colorful deposits of petrified wood. These rocks are also among the park's least studied according to William Parker, a paleontologist at Petrified Forest National Park and co-author of the new study. In 2011, a team co-led by research geologist Kay Behrensmeyer, the National Museum of Natural History's curator of vertebrate paleontology, braved the area's rugged badlands, which are home to rattlesnakes and wild horses. They were searching for fossils of prehistoric precursors to mammals and ended up discovering a bonebed containing an entire Triassic ecosystem. "That's the fun thing about paleontology: you go looking for one thing, and then you find something else that's incredible that you weren't expecting," said Kligman, who began working on this site as part of his doctorate in 2018. This part of northeastern Arizona was positioned in the middle of Pangaea and sat just above the equator 209 million years ago. The area's semi-arid environment was crisscrossed by small river channels and likely prone to seasonal floods. These floods washed sediment and volcanic ash into the channels. The site is so rich in small fossils that excavating them all in the field was impossible. In total, the team has uncovered more than 1,200 individual fossils, including bones, teeth, fish scales and coprolites, or fossilized poop. This assemblage contains 16 different groups of vertebrate animals that once inhabited a diverse ecosystem. The region's braided rivers were filled with fish, like freshwater sharks and coelacanths, as well as ancient amphibians, some of which grew up to 6 feet long. The surrounding environment was home to fearsome reptiles that evolved earlier in the Triassic, including armored herbivores and toothy predators that resembled giant crocodiles. Living alongside these strange creatures were a variety of more familiar critters, including relatives of tuataras and early frogs. The researchers also described the fossils of an ancient turtle with spike-like armor and a shell that could fit inside a shoebox. This tortoise-like animal lived around the same time as the oldest known turtle, whose fossils were previously uncovered in Germany. "This suggests that turtles rapidly dispersed across Pangaea, which is surprising for an animal that is not very large and is likely walking at a slow pace," Kligman said. The winged reptile would have been small enough to comfortably perch on a person's shoulder. "What was exciting about uncovering this specimen was that the teeth were still in the bone, so I knew the animal would be much easier to identify," McIntire said. The species name references its discoverer, McIntire, who retired last year. In addition to Kligman and Behrensmeyer, the new paper included contributions from Adam Fitch, who is also affiliated with the National Museum of Natural History. The study also includes authors affiliated with Columbia College Chicago, Massachusetts Institute of Technology, Denver Museum of Nature & Science and Petrified Forest National Park. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250708045656.htm'>MIT scientists just supercharged the enzyme that powers all plant life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-09 03:59:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>During photosynthesis, an enzyme called rubisco catalyzes a key reaction -- the incorporation of carbon dioxide into organic compounds to create sugars. However, rubisco, which is believed to be the most abundant enzyme on Earth, is very inefficient compared to the other enzymes involved in photosynthesis. Using a process known as directed evolution, they identified mutations that could boost rubisco's catalytic efficiency by up to 25 percent. MIT graduate student Julie McDonald is the paper's lead author. When plants or photosynthetic bacteria absorb energy from the sun, they first convert it into energy-storing molecules such as ATP. In the next phase of photosynthesis, cells use that energy to transform a molecule known as ribulose bisphosphate into glucose, which requires several additional reactions. Rubisco catalyzes the first of those reactions, known as carboxylation. During that reaction, carbon from CO2 is added to ribulose bisphosphate. Additionally, rubisco can also interact with oxygen, leading to a competing reaction that incorporates oxygen instead of carbon -- a process that wastes some of the energy absorbed from sunlight. "For protein engineers, that's a really attractive set of problems because those traits seem like things that you could hopefully make better by making changes to the enzyme's amino acid sequence," McDonald says. Most of those studies used directed evolution -- a technique in which a naturally occurring protein is randomly mutated and then screened for the emergence of new, desirable features. This process is usually done using error-prone PCR, a technique that first generates mutations in vitro (outside of the cell), typically introducing only one or two mutations in the target gene. In past studies on rubisco, this library of mutations was then introduced into bacteria that grow at a rate relative to rubisco activity. Limitations in error-prone PCR and in the efficiency of introducing new genes restrict the total number of mutations that can be generated and screened using this approach. Manual mutagenesis and selection steps also add more time to the process over multiple rounds of evolution. Their technique also enables them to mutate the target gene at a higher rate. "Our continuous directed evolution technique allows you to look at a lot more mutations in the enzyme than has been done in the past," McDonald says. Each of these mutations are located near the enzyme's active site (where it performs carboxylation or oxygenation). The researchers believe that these mutations improve the enzyme's ability to preferentially interact with carbon dioxide over oxygen, which leads to an overall increase in carboxylation efficiency. "The underlying question here is: Can you alter and improve the kinetic properties of Rubisco to operate better in environments where you want it to operate better?" That allows this rubisco to function well in an oxygen-rich environment, where normally it would constantly get distracted and react with oxygen, which you don't want it to do." Plants are believed to lose about 30 percent of the energy from the sunlight they absorb through a process called photorespiration, which occurs when rubisco acts on oxygen instead of carbon dioxide. "There are definite benefits to agricultural productivity that could be leveraged through a better rubisco." The research was funded, in part, by the National Science Foundation, the National Institutes of Health, an Abdul Latif Jameel Water and Food Systems Lab Grand Challenge grant, and a Martin Family Society Fellowship for Sustainability. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            