
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-01</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02448-3'>‘Congress has your back': US senators tell scientists they want to protect NIH budget</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-01 14:02:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. (Earlier this month, another Senate committee said it wanted to keep the budget for the US National Science Foundation relatively flat despite Trump's call for a 57% reduction.) The bill might also change if legislators introduce amendments before it's sent to Trump's desk. For example, Dick Durbin, a Democratic senator for Illinois, tried at the meeting yesterday to add a provision to reinstate billions of dollars in health-research funding that had been frozen or terminated by the Trump administration because projects — including some on COVID-19 — did not align with agency priorities. Shelley Moore Capito, a Republican senator for West Virginia, urged her colleagues to vote against the amendment, saying that “every president has the prerogative to implement new goals and priorities”. Nevertheless, senators at the hearing rejoiced that they could come to a bipartisan agreement to fund health agencies: “To the scientists wondering if there will even be an NIH by the end of this administration, this committee's resounding message is: ‘Yes, Congress has your back',” said Patty Murray, a Democratic senator for Washington state and vice-chair of the committee. “We need you to stay here and keep this research going.” For example, the Centers for Disease Control and Prevention, whose budget the Trump administration wanted to cut by more than half, would receive $9.1 billion, compared with $9.2 billion in 2024. These disruptions have drawn condemnation, and not only from Trump's opponents: 14 Republican senators sent a letter to the White House on 25 July demanding that the NIH be allowed to spend its full 2025 budget. Powerful satellite will map changes on Earth in stunning detail — down to a centimetre Columbia researchers eager to restart science after university's deal with Trump China reins in the spiralling construction costs of nuclear power — what can other countries learn? Title: Associate or Senior Editor, Nature Communications (Computational chemistry and materials science) Location(s): New York, Jersey City, Madrid... Job Title: Associate or Senior Editor, Nature Communications (Engineering and Infrastructure) Locations: Shanghai, Beijing, Nanjing, New York, Phil... Shanghai, Beijing, Nanjing, New York, Philadelphia, Jersey City, Madrid, or Milan An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65553103/ams-artifacts/'>Scientists Are Hunting Down Humanity's Earliest Artificial Memories</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-01 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Humanity's ancestors could have been counting long before any formal writing system existed. And it may have been advanced enough that entire societies knew what was going on. A team of researchers recently published a study detailing an analysis of 22 artifacts (dated to between 15,000 and 70,000 years ago) intended to determine how scientists could better identify artificial memory systems, or AMSs. Nowadays, we use computers and smartphones for the bulk of our out of body memory tracking, but the concept includes everything from systems of writing to sticks carved with tally marks known appropriately as ‘tally sticks.' But these things aren't easy to identify—especially the earliest examples on record, which consist of nothing more than a few marks on an object. So, the team behind this new paper set out to find a better method for identifying these early recording devices through the use of “new statistical tools and empirical evidence.”Basically, the team wanted to be able to identify what markings meant AMS, and what markings meant, say, butchery or art. So, they analyzed their 22 artifacts—which included a 44,000-year-old baboon bone from South Africa, a 70,000-year-old bone, a reindeer antler from France, medieval tally sticks, and Indigenous calendars—in the hopes of gathering enough information to create descriptions of what features markings of different purposes would have. In particular, the team looked for signs of regular, intentionally spaced markings that would indicate an objet was being used as an AMS. “Upper Paleolithic AMSs,” the authors wrote, “are endowed with systematically different signatures that distinguish them from the other artifacts.” Butchery marks were found to be clustered, and abstract decorative motifs displayed randomness in spacing (both had significant variation in angle). But potential AMSs show regular spatial patterns—what researchers claim are “distinct separation[s]” between the different types of markings. “These findings,” the researchers wrote, “suggest that modern humans in at least Africa and Europe had sophisticated cognitive capabilities for information storage and retrieval, providing insights into the possible development of quantity-related cognition.” For instance, just what information these AMSs were recording remains a mystery. “These marks,” the authors wrote, “could reflect a crucial step in the transition from basic cognitive abilities—like distinguishing between ‘few' and ‘many'—toward abstract concepts like numbers.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A Construction Survey Found an Ancient Celtic Town Rare ‘Book of the Dead' Found Buried in Egypt</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01332-8'>2-hydroxyglutarate mediates whitening of brown adipocytes coupled to nuclear softening upon mitochondrial dysfunction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-01 11:38:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mitochondria have a crucial role in regulating cellular homeostasis in response to intrinsic and extrinsic cues by changing cellular metabolism to meet these challenges. However, the molecular underpinnings of this regulation and the complete spectrum of these physiological outcomes remain largely unexplored. In this study, we elucidate the mechanisms driving the whitening phenotype in brown adipose tissue (BAT) deficient in the mitochondrial matrix protease CLPP. Here we show that CLPP-deficient BAT shows aberrant accumulation of lipid droplets, which occurs independently of defects in oxygen consumption and fatty acid oxidation. Our results indicate that mitochondrial dysfunction due to CLPP deficiency leads to the build-up of the oncometabolite d-2-hydroxyglutarate (d-2HG), which in turn promotes lipid droplet enlargement. We further demonstrate that d-2HG influences gene expression and decreases nuclear stiffness by modifying epigenetic signatures. We propose that lipid accumulation and altered nuclear stiffness regulated through 2HG are stress responses to mitochondrial dysfunction in BAT. Mitochondria have traditionally been regarded as distinct cellular entities primarily responsible for energy production1. However, increasing evidence shows that they are dynamic organelles that adapt to metabolic changes and cellular stress2. Consequently, mitochondria maintain cellular homeostasis through communication with the nucleus and other organelles2. Given their fundamental role, it is not surprising that mitochondrial dysfunction, especially oxidative phosphorylation (OXPHOS) deficiency, is linked to ageing and many diseases3. Mitochondrial stress triggers nuclear responses that alter gene expression to counteract cellular damage4. In mammalian cells, the mitochondrial integrated stress response (mitoISR) is one of the earliest transcriptional programmes activated by OXPHOS disruption5. This leads to metabolic reprogramming that can support or hinder tissue homeostasis6,7. The physiological consequences of mitochondrial stress remain incompletely understood, particularly in a tissue-specific context. By contrast, BAT, rich in mitochondria, drives thermogenesis through uncoupling protein 1 (UCP1), which dissipates energy as heat, a process essential for neonatal and cold-induced thermoregulation10. Both tissues exhibit metabolic plasticity: WAT undergoes browning in response to cold, β-adrenergic stimulation, exercise or fasting, increasing mitochondrial content and uncoupling. Conversely, BAT whitening occurs under thermoneutrality, ageing or nutrient overload, leading to lipid accumulation and loss of thermogenic function11. Chronic exposure to thermoneutrality (~30 °C) induces BAT whitening through PARKIN-mediated mitophagy and a ChREBP-driven shift toward fatty acid synthesis12,13. Factors such as obesity, high-fat diets and ageing similarly promote BAT whitening, characterized by lipid accumulation, reduced vascularization and impaired thermogenesis14,15,16. Regardless of the underlying cause, a common hallmark is lipid-droplet expansion with reduced mitochondrial mass or function21. Consistently, loss of mitochondrial proteins essential for fusion (MFN2 and OPA1)22,23 or mitochondrial DNA (mtDNA) maintenance (TFAM)24 also triggers BAT whitening. These findings highlight the central role of OXPHOS integrity in BAT homeostasis, although the underlying mechanisms that connect mitochondrial function to the homeostasis in BAT remain incompletely understood. We have recently shown that the loss of the mitochondrial matrix protease CLPP has contrasting effects in adipose tissue: it enhances oxidative phosphorylation (OXPHOS) in WAT but impairs BAT thermogenesis and promotes lipid accumulation, consistent with a whitening phenotype25. CLPP levels rise in BAT with cold exposure, unlike other mitochondrial proteases, but decline with ageing26,27. In a deep BAT proteomics screen involving a cohort of 163 genetically defined diversity outbred mice, CLPP correlated strongly with UCP1 (https://wren.hms.harvard.edu/opabat/)28. Furthermore, several of the identified UCP1 interactors are known CLPP substrates, supporting a strong link between CLPP and BAT function25,28,29. Here, we show that CLPP deficiency in BAT drives lipid droplet (LD) expansion through respiratory-chain dysfunction, independent of changes in oxygen consumption. This phenotype is driven by accumulation of d-2HG, an oncometabolite that alters chromatin and nuclear architecture. We reveal that mitochondrial dysfunction modulates epigenetic programmes, reducing nuclear stiffness and promoting BAT whitening. Together, our results uncover a previously unrecognized connection between mitochondrial function, metabolic signalling and chromatin reorganization in BAT. These insights shed light on how CLPP deficiency contributes to BAT whitening and impaired thermogenic function, expanding our understanding of mitochondrial regulation in adipose tissue biology. To study the role of mitochondria in the physiological remodelling of BAT, we used a mouse model with deletion of the mitochondrial protease CLPP (Clpp−/−, hereafter CLPP-KO), which is involved in OXPHOS complex synthesis and maintenance25,29. The loss of CLPP in BAT led to the accumulation of fewer but larger LDs, giving the tissue a pale appearance consistent with BAT whitening (Fig. This phenotype was intrinsic to BAT and not due to systemic CLPP deficiency, because similar whitening was observed in tissue-specific knockouts, including pan-adipose (CLPP-AKO: Clppfl/fl; Adipoq-cre)30 and brown-fat-specific models (CLPP-BKO: Clppfl/fl; Ucp1-cre)31 (Fig. Although whole-body CLPP deletion resulted in significant weight loss, CLPP-AKO mice maintained normal body size, and CLPP-BKO mice exhibited only mild weight reductions that were more pronounced in males (Extended Data Fig. Although male and female mice exhibit naturally different weight trajectories as they age, our extensive experience with CLPP-deficient models suggests that these differences do not categorically impact study outcomes. Therefore, we combined data from both sexes in subsequent analyses, ensuring balanced representation of male and female mice across all experiments whenever feasible. b, LD size measurement from WT and CLPP-KO BAT. c, Representative transmission electron microscopy images of BAT tissue from WT and CLPP-KO mice. d,e, Steady-state levels of OXPHOS supercomplexes (SC) in the BAT mitochondria of WT and CLPP-KO, CLPP-AKO and CLPP-BKO mice, analysed by blue native–PAGE followed by western blot for CI (NDUFA9) (d) and CV (ATP5A) (e) (n = 3). f, OCRs of whole BAT lysates from WT and CLPP-KO mice (n = 3). g, Numerical density of mitochondria in WT and CLPP-KO BAT (n = 48 fields per genotype obtained from three different mice from each genotype). Images represent results obtained from four independent experiments. i, Flow cytometry analysis of lipid accumulation (Bodipy) in WT and CLPP-KO mBA cells. Data represent one out of four performed experiments. b, Data are presented using Tukey's box plot with the middle line marking the median, and whiskers show variability within 1.5 × IQR. Anything beyond is an outlier presented as the individual value. f,g,j,k, Data are presented as mean ± s.d. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001, as determined by unpaired two-tailed Student's t-test in b, g, j and k, and one-way analysis of variance (ANOVA) with multiple comparisons in f. In j and k, the statistical significance between WT and knockout cells of the same cell types were plotted either above (mBA cells) or below the graph lines (preAD). In addition to enlarged LDs, ultrastructural analysis revealed increased numbers of enlarged mitochondria in CLPP-KO BAT, some of which were visibly paler with fewer cristae, suggesting impaired OXPHOS function (Fig. Mitochondria isolated from CLPP-KO, CLPP-AKO and CLPP-BKO BAT exhibited reduced levels of respiratory complex I (CI) and CI-containing supercomplexes, accompanied by decreased CI activity (Fig. Consistent with this, CLPP-KO BAT showed accumulation of F1 subassemblies of complex V (CV), which are essential for maintaining membrane potential when OXPHOS is compromised (Fig. Knockout tissue homogenates displayed reduced lipolysis, becoming significant after forskolin stimulation, suggesting that impaired lipolytic activation (glycerol release) could contribute to the whitening phenotype (Extended Data Fig. Despite these mitochondrial defects, oxygen-consumption rates (OCRs) in BAT lysates remained unchanged under both basal and stimulated conditions (Fig. 1f), probably owing to compensatory mitochondrial biogenesis (Fig. By contrast, thermoneutrality-induced whitening involves mitophagy15, whereas high-fat-diet-driven whitening causes LD accumulation without changes in mitochondrial content16. These findings point to distinct mitochondrial responses in different BAT whitening contexts and LD expansions. To explore the underlying mechanisms, we established immortalized preadipocyte (preAD) cultures from CLPP wild-type and knockout BAT (Fig. Upon differentiation, these cells exhibited mature brown adipocyte (mBA) features, including multilocular LDs and oligomycin resistance, indicative of mitochondrial uncoupling (Fig. Both wild-type and CLPP-KO mBAs strongly upregulated UCP1 (Extended Data Fig. Transcript levels of LD-associated proteins were markedly higher in CLPP-KO mBAs, including the ubiquitous marker PLIN2 (also known as ADRP) and the adipocyte-specific proteins PLIN1 and PLIN4 (Extended Data Fig. OCR and extracellular acidification rate (ECAR) measurements in mBAs revealed minimal, if any, difference between wild-type and CLPP-KO cells (Fig. However, both basal OCR and ECAR were higher than those in preADs, reflecting increased metabolic activity upon differentiation. These results confirm that mBAs faithfully model adult brown adipocyte function in both wild-type and CLPP-deficient conditions. To investigate the global impact of CLPP loss in BAT, we first analysed gene expression across all CLPP-deficient models (Fig. Gene Ontology (GO) enrichment using DAVID32 identified 707 commonly upregulated and 639 downregulated transcripts shared across genotypes (Supplementary Table 1). Inflammatory and immune pathways were among the most enriched (Fig. 2a and Supplementary Table 1), consistent with prior reports linking CLPP deficiency to cGAS–STING-mediated interferon-stimulated gene activation through mtDNA release33. However, STING deletion in CLPP-KO mice did not alter the BAT whitening phenotype (Fig. 2b), and we observed neither immune infiltration in CLPP-BKO BAT nor immune progenitor expansion in bone marrow (Extended Data Fig. Cell-type deconvolution based on single-cell transcriptomic references34 confirmed no major changes in cell composition, including immune populations (Extended Data Fig. a, Left, volcano plots of differentially expressed genes in BAT isolated from CLPP-KO, CLPP-AKO or CLPP-BKO mice, compared with the WT (n = 4). Coloured dots show significantly changed transcripts (P ≤ 0,01, two-fold-change). Right, Gene Ontology: Biological Process (GO:BP) analysis of shared significantly changed transcripts (in RNA-seq analysis) in BAT from CLPP-KO, CLPP-AKO and CLPP-BKO mice (Supplementary Table 3). b, Representative H&E staining of BAT from STING-deficient (STING-KO) or STING- and CLPP-deficient (STING/CLPP DKO) mice. Scale bars, 200 μm (results obtained from three mice per genotype). e, Heat map with relative Z-scores of significantly changed proteins (P ≤ 0.05) in SVF from WT and CLPP-KO mice. f, GSEA enrichment plots for gene signatures related to mitochondrial function. g, Top, heat map showing relative Z-scores of ATF4-target genes in the transcriptome dataset from BAT tissue of CLPP-BKO mice compared with those in WT mice. Bottom, GSEA enrichment plots for ATF4-target gene signatures. h, GSEA enrichment plots for gene signatures related to lipid metabolism. i, Venn diagram depicting overlapping and unique changes in significantly changed transcripts (P ≤ 0.05) in CLPP-BKO BAT compared with changes in mice housed at 4 °C and those initially housed at 4 °C for 1 week and subsequently moved to thermoneutrality (30 °C) for 4 weeks (rewarm versus cold)35 and LSD1-KO mice36 (Supplementary Table 2). j, GO:BP analysis of common significantly changed BAT proteins (P ≤ 0.05) from CLPP-KO, CLPP-AKO and CLPP-BKO mice (Supplementary Table 3). k, Average changes of individual OXPHOS complexes isolated from CLPP-KO cytoplasmic mitochondria (CM) or the whole BAT tissue lysate (WTL). Each data point represents average fold change value (CLPP-KO/WT, n = 4) of individual OXPHOS subunits obtained from proteomics analyses (Supplementary Tables 3 and 4). Beyond inflammation, CLPP-deficient BAT showed upregulation of pathways related to transcriptional regulation, signal transduction, adhesion, migration, cell death and lipid metabolism (Fig. Gene set enrichment analysis (GSEA) revealed enrichment of BAT-whitening-associated pathways such as actin remodelling and angiogenesis13,14 (Fig. 2a,c), whereas signatures of brown-adipocyte differentiation were negatively enriched and myogenesis appeared dysregulated (Fig. 2d), suggesting a possible impairment in BAT identity. Given the shared developmental origin of BAT and skeletal muscle, we tested whether differentiation was compromised by isolating the stromal vascular fraction (SVF) from wild-type and CLPP-KO BAT. Whole-proteome analysis of CLPP-BKO SVF revealed only minor changes, primarily in mitochondrial metabolism, mtDNA expression and OXPHOS proteins, with no alterations in markers of brown adipocytes or muscle differentiation (Fig. 2e), suggesting that early BAT development is intact. In agreement, the mitochondrial integrated stress response (mitoISR) was activated across models, as shown by GSEA and increased expression of ATF4 targets in CLPP-BKO BAT (Fig. Levels of transcripts involved in lipid metabolism, both catabolic (fatty acid oxidation) and anabolic (fatty acid and cholesterol synthesis), were also significantly reduced (Fig. Although GSEA confirmed overall suppression of lipid metabolic pathways, a subset of lipid-related genes was positively enriched, reflecting a dynamic remodelling process (Fig. This metabolic adaptation likely underlies LD accumulation, given that the OCR remained largely normal in knockout mBA cells (Fig. 1j), ruling out low OXPHOS activity as the cause. We next compared CLPP-BKO transcriptomes with two BAT whitening models: thermoneutral exposure35 and lysine-specific demethylase 1 (LSD1) BAT-specific knockout (LSD1-BKO) mice36. Exposure to thermoneutrality more robustly altered BAT gene expression than did the loss of either CLPP or LSD1; in both of the latter models, less than quarter of alterations were unique (Fig. About 30% of the differentially expressed genes in CLPP-BKO mice overlapped with both whitening models (Fig. GO terms shared across the three models matched those enriched in CLPP-deficient BAT (Supplementary Tables 1 and 2). Notably, Clpp expression was reduced in all three models, underscoring its link to BAT homeostasis (Extended Data Fig. Although proteomic changes were less extensive than transcriptomic ones, with 116 upregulated and 66 downregulated proteins shared across CLPP-KO, CLPP-AKO and CLPP-BKO BAT, they largely mirrored transcript-level trends (Supplementary Table 3). GO analysis showed enrichment for mitochondrial proteins in both directions (Fig. OXPHOS changes were modest, except for upregulated ATP synthase (complex V), consistent with a shift toward ATP production and reduced uncoupling, as seen in whitening BAT (Fig. Downregulated proteins clustered around fatty-acid-metabolism pathways, consistent with transcriptomic data (Fig. To refine our analysis, we examined the proteomes of cytoplasmic mitochondria and peri-lipid droplet mitochondria (PDMs) from BAT37. These mitochondrial populations had distinct yet overlapping profiles, and both showed clear OXPHOS alterations under CLPP deficiency (Extended Data Fig. Notably, both cytoplasmic mitochondria and PDMs displayed pronounced complex I and milder complex IV deficiencies, similar to heart mitochondria in CLPP-KO mice (Fig. These functional impairments were not apparent at the tissue level, likely owing to increased mitochondrial biogenesis (Fig. Collectively, these findings suggest that BAT whitening in CLPP-deficient mice is not driven by impaired differentiation or adipocyte-to-myocyte conversion, nor is it a direct consequence of OXPHOS loss. Instead, a transcriptional signature of suppressed lipid metabolism likely underlies the expansion of lipid droplets. The broad reshaping of the BAT transcriptome, particularly enrichment of GO terms related to transcriptional control and chromatin regulation, points to an adaptive stress response to CLPP loss (Fig. Transcriptome and proteome analyses of CLPP-deficient BAT revealed extensive metabolic adaptations. To further examine this, we performed semi-targeted metabolomics across CLPP-deficient models. Despite there being relatively few metabolites with significantly altered levels, a consistent overlap was observed between models (Fig. In line with the larger lipid droplets, levels of long-chain fatty acids were elevated, and levels of carnitines derived from branched-chain amino acids were dysregulated (Supplementary Table 5). Among all changed metabolites, 2-hydroxyglutarate (2HG) was the most strongly upregulated, showing a 2.7- to 3.7-fold increase across all three models (Fig. CLPP-deficient mBA cells also accumulated and secreted 2HG, suggesting a potential paracrine role (Fig. 4a), whereas KO preADs did not, indicating that 2HG accumulation is specific to mature brown adipocytes (Fig. a, Volcano plots of metabolites with significantly changed levels in BAT isolated from ubiquitously-deficient (KO), adipose-specific (AKO) or BAT-specific (BKO) CLPP-KO mice, compared with the WT (n = 5); dark colored dots present significantly changed metabolites (P ≤ 0,01, two-fold-change) (as seen in Supplementary Table 5). e, Relative changes of proteins implicated in production of 2HG, measured in BAT tissue proteomics from CLPP-KO, CLPP-AKO and CLPP-BKO mice compared with levels in WT mice (n = 4). f, Schematic depicting specific roles of PHGDH and the inhibition by NCT503. g, Relative percentages of the d and l isoforms of 2HG in WT, and CLPP-KO, CLPP-AKO and CLPP-BKO BAT tissue, as assessed by the derivatization of 2HG using TSCP (n = 5). The schematic in f was created using Biorender.com. 2HG, a structural analogue of α-ketoglutarate (α-KG), exists as two enantiomers, d- and l-2HG. To identify the source of 2HG, we analysed the expression of all known enzymes involved in its production38,39,40,41. PHGDH was the only one that was consistently upregulated, with more than fivefold increases across all CLPP-deficient models (Fig. PHGDH catalyses the first step of the serine biosynthesis pathway, frequently induced as part of mitoISR in response to OXPHOS dysfunction (Fig. Although PHGDH protein was not increased in CLPP-KO mBA cells, its enzymatic activity was significantly increased (Extended Data Fig. Enantiomer-specific analysis confirmed that d-2HG is the predominant form in all CLPP-KO BAT samples, as well as in isolated mature CLPP-KO brown adipocytes (Fig. The difference in 2HG enantiomer proportions between wild-type BAT and wild-type mBA cells is likely due to the heterogeneous nature of BAT, which consists of a mix of diverse cell types34. To directly test the putative role of PHGDH in d-2HG production, we inhibited it using NCT503, a selective inhibitor that blocks serine synthesis42. NCT503 treatment abolished 2HG accumulation in knockout mBA cells (Fig. Notably, PHGDH inhibition also reversed the whitening phenotype: lipid content was reduced, and smaller LDs formed in KO mBA cells, confirmed by fluorescence-activated cell sorting (Fig. Furthermore, two weeks of in vivo treatment with NCT503 normalized 2HG levels and reversed BAT whitening in CLPP-KO mice (Fig. To further probe the role of d-2HG, we overexpressed d-2HG dehydrogenase (d-2HGDH), the enzyme responsible for its degradation, in CLPP-deficient mBA cells. This significantly reduced d-2HG levels and partially reversed the whitening phenotype (Extended Data Fig. By contrast, inhibition of the ISR using ISRIB suppressed some ATF4 targets but had no effect on d-2HG accumulation or lipid storage (Extended Data Fig. 4h–j), suggesting that mitoISR is not the main driver of whitening phenotype following CLPP deficiency. Finally, we treated wild-type mBA cells with cell-permeable d-2HG (octyl d-2HG). This was sufficient to induce LD enlargement (Fig. 4b,f), directly linking d-2HG accumulation with BAT whitening. a, 2HG levels as determined by targeted metabolomics in WT, CLPP-KO and NCT503-treated KO mBA cells (n = 5). b, Representative images of WT, CLPP-KO, d-2HG-treated WT and NCT503-treated CLPP-KO mBA cells, stained with DAPI (nucleus) and Bodipy (LDs). Scale bar, 100 μm (images represent results from four independent experiments). c, Lipid levels in NCT503-treated and untreated CLPP-KO mBA cells, normalized to the average MFI value of WT cells (n = 12). d, 2HG levels in WT and CLPP-KO BAT after 14 days of NCT503 treatment in vivo (n = 12 for WT, n = 7 for KO and n = 11 for KO + NCT). e, Top, representative images of whole BAT. Images represent results from four mice per condition. f, Lipid levels in d-2HG-treated and untreated WT mBA cells, normalized to the average MFI value of WT cells (n = 12). MFI quantification of Bodipy-stained 2HG-treated and untreated WT mBA cells, normalized to the average of the MFI value of the WT cells (n = 5 per condition). **P < 0.01, ***P < 0.001, ****P < 0.0001, as determined by unpaired two-tailed Student's t-test in c and f and one-way ANOVA with multiple comparisons in a and d. In summary, our data demonstrate that CLPP loss activates PHGDH, leading to d-2HG accumulation in mature brown adipocytes. This metabolite contributes to lipid-droplet expansion and BAT whitening, revealing a critical role for aberrant serine metabolism and d-2HG in mediating the phenotypic consequences of mitochondrial stress. Elevated 2HG levels in cancer inhibit α-KG-dependent dioxygenases, affecting histone and DNA demethylation and leading to broad epigenetic reprogramming43. Initial western blot analyses of CLPP-deficient cells and wild-type cells treated with d-2HG did not reveal consistent changes in key histone modifications (histone H3 dimethylated at Lys9 (H3K9me2), H3 trimethylated at Lys36 (H3K36me3), H3K4me3, H3K27me3) (Extended Data Fig. Given the limited sensitivity of this method, subtle epigenetic shifts might have been missed. Immunofluorescence assays showed modest, inconsistent changes in H3K9me2 and H3K27me3) (Extended Data Fig. 5b), prompting a more focused analysis of H3K4me3, a mark of active and poised promoters known to be influenced by 2HG43. Chromatin immunoprecipitation followed by high-throughput sequencing (ChIP–seq) revealed a global increase in H3K4me3 in both CLPP-deficient and d-2HG-treated mBA cells, with most enrichment occurring at promoter regions (Fig. Notably, 1,141 out of 2,017 unique H3K4me3 peaks in CLPP-deficient cells overlapped with those in d-2HG-treated cells, suggesting a shared mechanism (Fig. GO analysis of these enriched loci shared between CLPP-deficient and d-2HG-treated mBA cells revealed biological processes associated with transcriptional regulation, differentiation, chromatin reorganization, lipid metabolism, apoptosis and DNA repair (Fig. Many of these categories also emerged in transcriptomic analyses of CLPP-deficient BAT (Supplementary Table 1), supporting their functional relevance. a, Heatmap and coverage plots of H3K4me3 ChIP–seq in WT, CLPP-KO and 2HG-treated WT mBA cells, ±2 kb around the transcription start site (TSS) (n = 4). c, GO:BP analysis of H3K4me3 chromatin marks shared between KO and d-2HG-treated WT mBA cells (Supplementary Table 7). d, Integrative Genomics Viewer browser views showing H3K4me3 read density on the promoters of selected genes (n = 4). e, Venn diagram depicting the number of unique and overlapping changes between transcripts differentially expressed in CLPP-KO mice, compared with distinct H3K4me3 signatures found in KO and d-2HG-treated WT mBA cells. f, GO:BP analysis of common significantly changed transcripts between CLPP-KO and d-2HG-treated WT mBA cells (Supplementary Table 8). A closer examination of individual genes identified increased H3K4me3 at promoters of key adipogenic regulators, including SREBP1 and SREBP2 (cholesterol and fatty acid metabolism), and C/EBPβ (white adipocyte differentiation)44,45 (Fig. These changes suggest that d-2HG-induced chromatin remodelling could promote transcriptional programmes linked to adipogenesis and metabolism. To further explore this, we compared transcriptomes of CLPP-KO and d-2HG-treated mBA cells. More than 50% of significantly altered transcripts in CLPP-KO cells exhibited corresponding changes in H3K4me3 levels (Fig. Additionally, more than one-third of these were shared with d-2HG-treated cells, implicating 2HG as a major driver of transcriptional changes in CLPP deficiency. GO analysis revealed enrichment of both up- and downregulated transcripts in pathways related to chromatin remodelling, transcriptional and translational regulation and apoptosis, indicating broad alterations in these processes (Fig. Distinct trends were also evident: oxidative stress and DNA-damage-response terms were enriched among upregulated genes, whereas cell adhesion and extracellular matrix organization terms were downregulated in both models (Fig. GSEA analysis identified substantial changes of lipid and fatty acid metabolism, mirroring alterations observed in CLPP-deficient tissues (Figs. Among the transcripts upregulated in both CLPP-KO mBA cells and cells treated with d-2HG, we identified multiple enzymes involved in lipid metabolism, along with a variety of lipid transporters, including the highly upregulated CD36, a major regulator of fatty acid uptake in adipocytes and a key factor in LD growth46 (Fig. Many of these transcripts were suppressed in cells overexpressing d-2HGDH (Fig. In parallel, transcripts encoding adipogenic regulators, such as Pparg and Cebpb, were upregulated in both CLPP-KO mBA cells and cells treated with d-2HG. By contrast, Ppargc1a, encoding PGC1α, was specifically induced in CLPP-deficient cells, consistent with underlying mitochondrial OXPHOS dysfunction (Fig. Remarkably, Cebpa expression appeared to be similarly influenced by mitochondrial dysfunction rather than d-2HG treatment (Fig. Conversely, cholesterol-biosynthesis genes were negatively enriched in both CLPP-deficient and d-2HG-treated cells; this change was also observed in BAT from all three CLPP-KO models (Extended Data Fig. 5f), highlighting a strong similarity between cells and tissues. These changes coincided with reduced cellular cholesterol levels in both CLPP-deficient and d-2HG-treated cells, similar to the effect of simvastatin, an inhibitor of cholesterol biosynthesis (Fig. a, GSEA enrichment plots for gene signatures related to lipid metabolism shared between CLPP-KO cells and WT mBA cells treated with d-2HG. b, Relative mRNA levels of genes under GO:BP analysis term ‘Lipid metabolism' enriched in CLPP-KO cells and WT cells treated with d-2HG, obtained from RNA sequencing (RNA-seq) analysis (n = 4) (Supplementary Table 8). c, Relative mRNA levels of genes involved in lipid metabolism in CLPP-KO cells upon expression of d-2HGDH, as measured by quantitative real-time PCR. d, Relative mRNA levels of major regulators of lipogenesis genes enriched in CLPP-KO cells and WT cells treated with d-2HG obtained from RNA-seq analysis (n = 4) (Supplementary Table 8). e, Relative mRNA levels of genes under GO:BP analysis term ‘Cholesterol metabolism' enriched in CLPP-KO cells and WT cells treated with d-2HG, obtained from RNA-seq analysis (n = 4) (Supplementary Table 8). Together, these findings demonstrate that CLPP deficiency and d-2HG treatment induce similar changes in transcriptome and epigenome, that converge on chromatin regulation, adipogenic signalling and cholesterol metabolism, linking mitochondrial dysfunction to coordinated metabolic reprogramming. GO enrichment analysis of transcriptional and H3K4me3 ChIP–seq changes in CLPP-deficient tissues and cells consistently identified terms related to cell adhesion, migration, actin cytoskeleton organization and transcriptional regulation (Figs. A major number of enriched transcripts mapped to the nucleus, nucleoplasm and perinuclear cytoplasm. This aligns with emerging evidence linking nuclear architecture to cellular stress responses. The overlap of nucleus-related transcripts between CLPP-KO and d-2HG-treated cells supports this connection (Extended Data Fig. GSEA further revealed strong negative enrichment of nuclear envelope-associated transcripts in CLPP-deficient BAT, closely matching the transcriptomic signature of d-2HG-treated cells (Extended Data Fig. 6b), suggesting that metabolic perturbations could influence nuclear and cellular morphology. We next explored the link between d-2HG, epigenetic changes and nuclear phenotype. Ultrastructural analyses revealed pronounced nuclear deformation in CLPP-deficient brown adipocytes, characterized by mitochondria clustering near and indenting the nuclear envelope (Fig. The extent of mito-nuclear interactions in CLPP-deficient BAT could be further observed in a high-resolution three-dimensional (3D) reconstituted model using serial section array tomography (Fig. Although direct contact sites with shared membranes were not detected, the two membranes were closely positioned with very little intercalated ER (Extended Data Fig. a, Representative TEM image from BAT of WT and CLPP-KO mice. LDs (yellow), nucleus (blue) and mitochondria (red) (n = 3 mice). b, 3D reconstruction of serial TEM images from KO BAT tissue (Supplementary Video 1). c, Quantification of nuclear dents in mitochondria (n = 176 fields for WT and n = 150 for KO, from two mice per group). d–g, Quantification of Young's modulus from AFM-mediated force spectroscopy of mBA nuclei from: WT and CLPP-KO (n = 269 for WT and n = 260 for KO, each from four individual cell culture plates) (d); untreated WT or d-2HG-treated WT (WT + HG) (n = 772 for WT n = 642 for WT + HG, each from four cell culture plates) (e); CLPP-KO, either untreated (KO) or treated with NCT503 (KO + NCT) (n = 1,006 for KO, n = 735 for KO + NCT, each from four cell culture plates) (f); WT, CLPP-KO and WT treated with d-2HG, supplemented with cholesterol (Ch) (g). h, 2HG levels as determined by targeted metabolomics in WT and actinonin-treated (WT + Act) mBA cells (n = 5). c,h,i, Data are presented as mean ± s.d. d–g,j, Data are presented using Tukey's box plot with middle line marking the median, and whiskers show variability within 1.5×IQR. Anything beyond is an outlier presented as individual value. The schematic in k was created using Biorender.com. Quantification of mito-nuclear indentations revealed an increase from ~1.3 per nucleus in wild-type to ~2.9 in CLPP-deficient cells, with some nuclei displaying up to 10 invaginations (Fig. Using atomic force microscopy (AFM), we detected a significant reduction in nuclear stiffness in CLPP-deficient adipocytes, mimicked by d-2HG treatment of wild-type cells (Fig. Inhibition of PHGDH with NCT503, which lowers d-2HG levels, restored nuclear stiffness in CLPP-deficient cells (Fig. By contrast, ISRIB treatment did not significantly change d-2HG levels and had no effect on nuclear envelope stiffness (Extended Data Fig. These findings establish a functional link between altered metabolism and nuclear morphology, regulated by d-2HG, but independent of the classical ISR. Cholesterol increases membrane stiffness, especially in saturated lipid environments, with a weaker effect in unsaturated lipid-rich membranes47. Although the nuclear envelope is mainly composed of unsaturated phospholipids, increased cholesterol synthesis has been linked to nuclear envelope stiffening and fragility48. Because CLPP-deficient and d-2HG-treated cells showed reduced cholesterol levels, we tested whether supplementation could restore nuclear stiffness. Cholesterol treatment significantly increased nuclear stiffness in both models, exceeding levels observed in wild-type cells (Fig. Conversely, treatment with simvastatin, a cholesterol-synthesis inhibitor, further decreased nuclear stiffness (Fig. 7g), highlighting cholesterol as a critical regulator of nuclear envelope mechanics. To test whether these effects extend beyond CLPP deficiency, we treated wild-type mBA cells with actinonin, an inhibitor of OXPHOS biogenesis49. Short-term actinonin treatment significantly increased 2HG levels in wild-type mBA cells and modestly elevated lipid accumulation (Fig. 7h,i), suggesting that metabolic reprogramming occurs rapidly upon mitochondrial dysfunction. This was accompanied by a more than twofold reduction in nuclear stiffness (Fig. 7j), suggesting that OXPHOS impairment broadly influences nuclear mechanics through rapid metabolic rewiring. Collectively, our findings reveal that mitochondrial dysfunction drives metabolic reprogramming that reshapes nuclear architecture and mechanics. We propose that PHGDH activation and subsequent d-2HG accumulation induce nuclear softening, which is reversible and cholesterol-sensitive (Fig. Although our study centres on BAT whitening and CLPP loss, similar metabolic–nuclear coupling is likely to occur in other contexts of OXPHOS dysfunction and lipid accumulation, warranting further investigation. This study identifies a mechanism by which CLPP loss and resulting OXPHOS dysfunction drive LD expansion, a key feature of BAT whitening, through metabolic and epigenetic adaptation. LD enlargement and BAT whitening are observed in various models of mitochondrial dysfunction and are often linked to respiratory-chain deficiencies22,23,24. Although impaired fatty acid oxidation (FAO) is frequently proposed as a cause, direct mechanistic evidence is limited, as seen in models like adipocyte-specific TFAM deletion24. Other studies, such as the one conducted on Mfn2-deficient adipocytes, suggest that reduced lipolysis, rather than increased lipogenesis, underlies LD accumulation23. However, not all OXPHOS deficiencies result in larger LDs. For instance, mtDNA mutator mice exhibit BAT dysfunction without LD expansion, likely due to systemic metabolic constraints and a catabolic state that limits lipid storage50. Interestingly, BAT-specific OPA1 loss leads to LD accumulation and reduced oxygen consumption but paradoxically enhances thermogenesis through FGF21 induction22. Conversely, OPA1 overexpression protects against diet-induced obesity by preventing BAT whitening and stimulating WAT browning, potentially through fumarate-driven activation of KDM3A51. These findings underscore the complexity of mitochondrial stress responses and their influence on adipocyte fate through metabolic–epigenetic cross-talk. Our data suggest that LD expansion and BAT whitening in CLPP-deficient models are a stress response to mitochondrial dysfunction, rather than a direct consequence of respiratory failure. This phenotype was consistent across in vivo models and terminally differentiated CLPP-KO brown adipocytes. Although LDs are typically considered energy reservoirs, they also serve protective roles, buffering oxidative stress, mitigating endoplasmic-reticulum stress and sequestering harmful lipid species52. LD expansion can limit polyunsaturated fatty acid oxidation and reduce lipotoxicity, which is particularly important under mitochondrial dysfunction, during which acylcarnitines, ceramides and oxidized lipids can accumulate53. Thus, we propose that LD biogenesis acts as an adaptive mechanism to maintain redox balance and prevent metabolic toxicity52,53. We further show that LD expansion is driven by increased d-2HG levels, produced through promiscuous activity of PHGDH40 in CLPP-deficient adipocytes. Exogenous d-2HG recapitulates the phenotype in wild-type cells, and transcriptomic analysis reveals shared signatures affecting lipid and cholesterol metabolism. PHGDH, a key enzyme in serine biosynthesis and one-carbon metabolism, is frequently upregulated in mitochondrial disease models5,6,7,54. Given its central role in the mitoISR, inhibiting PHGDH in mitochondrial disease models, such as Deletor mice, is detrimental54. Our findings suggest that PHGDH-derived d-2HG has an additional role in modulating lipid metabolism and chromatin architecture. Although d-2HG is best known for its role in cancer, it also functions in hypoxia, immune-cell differentiation and metabolic disorders55. In tumours, it inhibits α-KG-dependent dioxygenases, such as TET2 and Jumonji-C demethylases, thereby altering DNA and histone methylation55. These changes can shift cell fate and are associated with increased repressive marks, such as H3K9me3 and H3K27me3, without widespread changes in DNA methylation56. d-2HG also exerts non-cell-autonomous effects; for example, tumour-derived d-2HG suppresses CD8⁺ T cell metabolism and function57. In accordance with this, we observed 2HG secretion from mature brown adipocytes, raising the possibility of systemic effects in CLPP-deficient animals. Our findings reveal new functions for d-2HG, demonstrating its role in regulating nuclear mechanics in terminally differentiated cells. Both CLPP deficiency and d-2HG treatment led to nuclear softening, associated with increased H3K4me3 occupancy at promoter regions. This included lipid metabolic regulators involved in lipogenesis, lipolysis and transport, consistent with LD expansion. Notably, although H3K4me3 was enriched at the promoters of cholesterol-biosynthesis genes, their expression and total cholesterol levels were reduced. This indicates that there are additional regulatory layers influenced by d-2HG. Cholesterol is essential for membrane rigidity, including at the nuclear envelope. We show that cholesterol depletion causes nuclear envelope softening, whereas supplementation restores nuclear stiffness in CLPP-deficient and d-2HG-treated cells. This aligns with prior work showing that staurosporine-induced mitochondrial dysfunction alters mitochondria–nucleus interactions and cholesterol distribution58. Similarly, in yeast, oxygen deprivation enhances the impact of saturated lipids, leading to nuclear envelope rigidification and rupture, while revealing a protective role for LDs in buffering membrane composition and preserving nuclear envelope integrity59. Our data suggest that, in brown adipocytes, LD expansion might serve a similar protective role for the nuclear envelope. Recent studies have shown that, under mechanical confinement, mitochondria migrate toward the nucleus, resulting in increased nuclear ATP to support chromatin reorganization and DNA repair60. Nuclear stiffness depends on both the nuclear lamina and heterochromatin, which provides structural support and mechanical resistance61. Cells sense deformation through nuclear envelope stretching, which triggers membrane phospholipid restructuring and actomyosin cytoskeleton changes61. In conclusion, our study links mitochondrial dysfunction to nuclear reshaping, showing how CLPP loss and d-2HG-driven metabolic rewiring affect gene expression and nuclear structure to promote lipid-droplet accumulation. We identified d-2HG as a key mediator in this process, accumulating in CLPP-deficient BAT owing to increased PHGDH activity. This metabolite reshapes the epigenetic landscape by enriching H3K4me3 histone marks on promoters of lipid-metabolism regulators, orchestrating a shift toward LD accumulation, likely as a protective sink for potentially lipotoxic intermediates. Additionally, our findings highlight an unexpected interplay between cholesterol metabolism, cytoskeletal organization and nuclear mechanics, showing that changes in cholesterol availability contribute to nuclear softening. Although our focus is on brown adipocytes, these findings could extend broadly to tissues undergoing mitochondrial stress, with relevance to ageing, metabolic disease and cancer. By connecting mitochondrial dysfunction, chromatin reorganization, lipid metabolism and nuclear mechanics, our study provides a framework for understanding adaptive tissue remodelling in mitochondrial disorders. All mice analysed in this study were bred and housed at the CECAD Research Center animal facility, University of Cologne. All experiments were conducted in compliance with National Institutes of Health guidelines and approved by local government authorities (Landesamt für Natur, Umwelt und Verbraucherschutz Nordrhein-Westfalen; LANUV). Mice were housed in individually ventilated cages at 22–24 °C under a 12-h light–dark cycle, with lights on at 06:00. They had ad libitum access to water and a standard chow diet (ssniff rat/mouse low-phytoestrogen diet). For breeding, one male was paired with one to two females of similar age, with mating initiated at a minimum age of 8 weeks. Body weight was recorded weekly for individual mice from 4 to 16 weeks of age. The mice used in this study were derived from a line with a conditional Clpp gene targeting, as previously described29. Each day for 15 days, 14-month-old male mice were intraperitoneally injected daily with vehicle (ethanol (5%), PEG400 (35%) and 30% cyclodextrin (60%)), in which NCT503 was dissolved. NCT-503 was freshly prepared prior to the injections. After dissection, the entire adipose tissues were placed in tissue cassettes (Roth) and fixed in 5% (wt/vol) paraformaldehyde overnight at 4 °C. Tissues were then stored in PBS until they sank to the bottom. Finally, tissues were embedded in paraffin blocks. The fixed BAT and inguinal WAT (iWAT) were sectioned at 5-μm thickness using a microtome (Leica) and mounted on poly-l-lysine-coated glass slides (VWR). Paraffin-embedded BAT sections were deparaffinized in xylol and rehydrated through decreasing ethanol concentrations, followed by a wash in tap water. Sections were then dehydrated in increasing ethanol concentrations, cleared in xylol and mounted with entellan. Post-fixation was performed with 2% OsO₄ (Science Services) in 0.1 M cacodylate buffer at 4 °C, followed by additional washes. Samples were dehydrated in an ascending ethanol series (50%, 70%, 90%, 3×100%), treated with ethanol and propylene oxide and infiltrated with epon and propylenoxide at gradually increasing concentrations of epon. Samples were embedded in PELCO 21-cavity molds (Plano) and cured at 60 °C. Ultrathin sections (70 nm) were cut using an ultramicrotome (Leica UC6) with a 45° diamond knife (Diatome). Sections were stained with uranyl acetate and lead citrate and imaged using a JEM-2100 Plus TEM (JEOL) operating at 80 kV, equipped with a OneView 4 K camera (Gatan). Ribbons of five consecutive ultrathin sections were collected on Pioloform-coated slot grids (Science Services). A total of 60 sections were imaged for 3D reconstruction. Images were aligned using FIJI (TrakEM2 plugin), and nucleus, mitochondria and lipid droplets were manually segmented in Microscope Imaging Browser (MIB, v2.82). The final model was processed in IMARIS (v9.9.1) for 3D reconstruction. Transmission electron micrographs (TEMs) at ×2,500 magnification were used to quantify mitochondria in wild-type and knockout conditions. Forty-eight images per condition were captured from random sample positions. Mitochondrial quantification was performed using unbiased counting frames in ImageJ (Fiji). For each image, a 3×3 grid of counting frames covering 25% of the image was randomly placed. Mitochondria fully enclosed or intersecting permissive lines were counted; those intersecting restrictive lines were excluded. The multi-point tool in Fiji was used for tracking. The largest possible rectangular sample regions were extracted using Fiji, excluding grid and void areas. For 12 images per condition, 512×512 pixel regions were cropped and segmented using the Cellpose cyto3 model. Segmentation results were saved as ImageJ ROI files, manually corrected in QuPath and re-applied to all extracted regions using a refined Cellpose model. Lipid-droplet number and size were measured using a FIJI macro. A guard zone (15% of image diameter) was applied to exclude lipid droplets cut off by image borders. To avoid biased exclusion, the lower and left boundaries were restrictive (excluding intersecting droplets), while the upper and right boundaries were permissive (including intersecting droplets). Sterile equipment (Falcon tubes, reaction tubes, syringes, cannulas and a dish) was disinfected with 70% ethanol. The two brown fat pads were dissected from newborn mice, placed in collagenase digestion buffer, minced with scissors and transferred to a Falcon tube on ice. For collagenase digestion, Falcon tubes were incubated in a water bath with gentle shaking until complete tissue dissociation. The suspension was drawn into syringes using 0.9 × 70 mm (20 G × 2) cannulas, then filtered through a 100-μm nylon mesh into a fresh Falcon tube. The middle phase was carefully collected, avoiding the upper (mature adipocytes) and lower (tissue remnants) phases. The suspension was then filtered through a 30-μm nylon mesh and centrifuged. The cell pellet was resuspended in brown adipose tissue mesenchymal stem cell culture medium and transferred to a six-well plate for further analyses. Primary brown adipose tissue mesenchymal stem cells (passage 0) were immortalized 24 h post-isolation through lentiviral transduction with a virus expressing SV40 large T-Antigen (L-TAg) under the PGK promoter. The virus solution was thawed and centrifuged at 16,000g, and 200 ng viral reverse transcriptase (per well of a 6-well plate) was mixed with 800 μl brown adipocyte (BA) growth medium (high-glucose DMEM without pyruvate). Cells were maintained in growth medium at 37 °C, 5% CO₂, ensuring cultures did not reach full confluence. For passaging, cells were detached through trypsinization, resuspended in growth medium, and centrifuged at 200g. Pellets were resuspended in growth medium and mixed 1:1 with freezing medium to achieve a final DMSO concentration of 10% at a cell density of 1 million cells ml−1. Cell suspensions were transferred to cryogenic vials (1 ml per vial) and stored at −80 °C, followed by transfer to liquid nitrogen (−196 °C) for long-term storage. Adipogenic differentiation was induced by switching to freshly prepared induction medium (growth medium + 5 µM dexamethasone + 250 µM IBMX (3-Isobutyl-1-methylxanthine) + 0.5 µg ml−1 insulin + 1 nM triiodothyronine (day 0). After induction, cells were maintained in BA differentiation medium, which was replenished every second day. By day 8, cells were fully differentiated into mature brown adipocytes. Treatments were initiated from day 3 upon switching to differentiation medium: actinonin (150 μM, DMSO; applied for 96 h, with replacement every second day); NCT-503 (20 μM, DMSO; applied for 72 h, with replacement every 24 h); octyl d-2HG (500 μM, DMSO; applied for 72 h, with replacement every 24 h); and ISRIB (1 μM, DMSO; applied for 72 h, with daily renewal). Cells were washed with PBS and stained with 1× BODIPY solution in PBS in the dark. After staining, cells were trypsinized, and trypsinization was stopped with FACS buffer (PBS + 2% BSA). The cell suspension was filtered through a 60-μm strainer and transferred to fresh Eppendorf tubes for analysis. The gating strategy used for flow cytometry analysis is provided in Supplementary Figure 1a. Cells were washed with PBS and fixed in 10% formalin for 30 min at room temperature. After fixation, cells were washed twice with water, followed by a 5-min wash with 60% isopropanol at room temperature. A working solution of Oil red O (ORO) was added to the wells and incubated for 10 min at room temperature. After incubation, cells were washed four times with water, and images were acquired for further analysis. The ORO stock solution was prepared as 0.35% ORO in isopropanol, and the working solution was prepared with 60% of ORO stock in water. Cells were seeded and differentiated on coverslips before direct fixation with 4% paraformaldehyde in PBS (pH 7.4) at room temperature, followed by washes with PBS. Blocking was performed with 5% BSA and 0.3% Triton X-100 in PBS. Coverslips were incubated with primary antibodies diluted in 1% BSA and 0.3% Triton X-100 in PBS at 4 °C, followed by PBS washes. Coverslips were dipped in water, mounted on elvanol and imaged using an SP8 confocal microscope (Inverse, DMi 8 CS, Leica Microsystems). To assess lipolytic activity in BAT from CLPP-KO and control mice, mice were fasted for 6 h, and BAT pads were excised under sterile conditions. Tissues were digested with collagenase type 2 (Worthington Biomedical) in Krebs-Ringer Solution (KRH) + 1% BSA. For each condition, BAT adipocytes from 4–5 mice were pooled, and 150,000–200,000 isolated adipocytes were incubated in KRH buffer + 4% BSA with either 15 nM isoproterenol (ISO) or 10 μM forskolin. Lipolysis was assessed by measuring free glycerol release in the supernatant using Free Glycerol Reagent (Sigma-Aldrich), with absorbance recorded on a Synergy H1 plate reader (BioTEK). Glycerol content was normalized to total protein content, determined by the Pierce Bradford Assay Kit (Thermo Scientific). Oxygen consumption in BAT tissue lysates was measured using an OROBOROS Oxygraph-2K electrode at 37 °C with magnetic stirring. Samples were placed in 2 ml incubation medium (containing EGTA, MgCl₂, K-lactobionate, taurine, KH₂PO₄, HEPES, sucrose and BSA; pH 7.1). Respiration rates were normalized to total protein content. Mitochondrial OCR and ECAR were measured using the Seahorse Bioscience XFe-96 Analyzer (Agilent Technologies) following the standard mitochondrial stress test protocol. Cells were washed with unbuffered assay medium (DMEM without phenol red, supplemented with glucose, sodium pyruvate and glutamine) and incubated in a CO₂-free incubator at 37 °C. The assay was then performed, and post-measurement normalization was conducted using BioTek Cytation 5 (Agilent Technologies) with Hoechst 33342 staining for cell counting. Data analysis was performed using Seahorse XFe Wave Software (Agilent). Basal respiration, leak respiration, maximal respiration and acidification were calculated on the basis of OCR and ECAR values before and after specific injections, with non-mitochondrial oxygen consumption corrected using rotenone and antimycin A. AFM measurements were performed on cell monolayers plated on silicon elastomers using the JPK NanoWizard 4 XP (Bruker Nano) atomic force microscope mounted on Zeiss AxioObserver inverted fluorescent microscope and operated through JPK SPM Control Software v.5. Triangular non-conductive silicon nitride cantilevers (MLCT, Bruker Daltonics) with a nominal spring constant of 0.01 N m−1 were used for the nanoindentation experiments of the apical surface of cells and the nucleus. For all indentation experiments, forces of up to 3 nN were applied, and the velocities of the cantilever approach and retraction were kept constant at 2 μm s−1, ensuring an indentation depth of 500 nm. All analyses were performed with JPK Data Processing Software (Bruker Nano). To overexpress d-2HGDH in wild-type and CLPP-KO mBA cells, a lentiviral transfer plasmid encoding d-2HGDH–hemagglutinin (HA) (VectorBuilder) was used. Viral supernatants were collected, pooled, clarified by centrifugation and concentrated with Lenti-X Concentrator (Takara Bio). The viral pellet was resuspended in PBS, aliquoted and stored at −80 °C. Wild-type and CLPP-KO mBA cells were seeded in 6-well plates and transduced with 10 μl concentrated lentivirus in 6 μg ml−1 polybrene. After 24 h, the medium was replaced, and cells were expanded before puromycin selection (2 μg ml−1, 48 h). Then, 30 μg of protein was loaded into a 96-well plate, with the final volume adjusted to 50 μl using PHGDH assay buffer. A NADH standard curve was generated using dilutions of 1.25 mM NADH standard, and a PHGDH positive control was prepared. A reaction mix of assay buffer and developer was added to all wells. Total RNA was isolated from mouse BAT and primary adipocytes using TRIzol (Life Technologies) or TRI reagent (Sigma), followed by purification with RPE buffer (Qiagen). Tissues were homogenized in 1 ml TRIzol using ceramic beads (Mobio, Dianova) in a FastPrep homogenizer (MP Biomedicals) at 5,500 r.p.m. The aqueous RNA-containing phase was collected and precipitated with 500 μl isopropanol, followed by washing with 75% ethanol and resuspension in ultrapure H2O. RNA was incubated at 60 °C before a concentration and purity assessment using a NanoDrop ND-1000 UV-Vis spectrophotometer (Peqlab). For DNase treatment, 10 μg RNA was digested with DNase I (New England Biolabs) in 10× digestion buffer. Isolated RNA was treated with DNase I (New England Biolabs) to digest the residual DNA, following the manufacturer's protocol. Next, complementary DNA (cDNA) was synthesized from the digested samples using the High-Capacity Reverse Transcription kit (Applied Biosystems). Quantitative PCR (qPCR) reactions were performed in 384-well plates using the Brilliant III Ultra-Fast SYBR Green qPCR Master Mix (Agilent Technologies), according to the manufacturer's instructions; 50 ng of cDNA was added to each reaction. The standard programme was run at the Applied Biosystems QuantStudio 7 Flex Real-Time PCR System (Fisher Scientific). Relative expression of the target gene was calculated by normalization to endogenous control gene (Hprt) and was quantified using the 2−ΔΔCt method. BAT was collected, rinsed in PBS, weighed and minced with fine scissors. Homogenization was performed using a glass-Teflon electric homogenizer (7 strokes) followed by a glass-glass Dounce homogenizer (13 strokes). The homogenate was transferred to a 50-ml Falcon tube, and residual tissue was washed out with SHE-BSA buffer. The supernatant, containing cytoplasmic mitochondria, was carefully collected using a syringe, avoiding disruption of the floating fat layer or debris pellet. The fat layer, containing lipid-associated PDMs, was scraped into a new tube, resuspended in SHE-BSA buffer, and vigorously shaken to emulsify the fat and release PDMs. The final pellets were resuspended in SHE buffer without BSA for immediate analysis or snap-frozen in liquid nitrogen for long-term storage. BAT was collected and rinsed twice in PBS. Lysates were cleared by centrifugation (20,000g, 4 °C, 45 min) and transferred to fresh tubes. Lysates were incubated on ice for 30 min with occasional vortexing, followed by 10 cycles of sonication (30-s intervals, 4 °C) using the Bioruptor Pico (Diagenode). Proteins were separated by SDS–polyacrylamide gel electrophoresis (SDS–PAGE) using the Mini-PROTEAN Tetra Cell (BioRad). Fifty micrograms of protein lysate was mixed with Laemmli buffer (50 mM Tris-HCl, 2% SDS, 10% glycerol, 1% 2-mercaptoethanol, 12.5 mM EDTA, 0.02% bromophenol blue, pH 6.8) and incubated at 95 °C for 5 min. Samples were loaded onto an 8–12% polyacrylamide gel alongside a PageRuler Prestained Protein Ladder (Thermo Fisher Scientific). Electrophoresis was performed at 90 V until the loading dye reached the separating gel, followed by 150 V until completion in running buffer (25 mM Tris-HCl, 250 mM glycine, 0.1% SDS, pH 8.3). Proteins were transferred to a nitrocellulose membrane using the Criterion Blotter (BioRad) through wet transfer at 400 mA in transfer buffer (30 mM Tris-HCl, 240 mM glycine, 0.037% SDS, 20% methanol) at 4 °C. Transfer efficiency was assessed with Ponceau-S staining (Sigma-Aldrich). Membranes were blocked in 5% milk-PBST, 3% BSA-TBST or 2% fish skin gelatin-TBST, depending on antibody requirements. Membranes were incubated overnight at 4 °C with primary antibodies, followed by three 5-min washes in PBST or TBST. Secondary antibody incubation was performed at room temperature, followed by additional washes. Detection was performed using ECL solution (GE Healthcare), and membranes were exposed to Super RX films (Fujifilm), developed using an automatic film processor (Kodak) and scanned with a V800 Transparency Scanner (Epson) at 600 dpi, 16-bit grayscale. Blue native (BN)–PAGE was performed using the NativePAGE Novex Bis-Tris Gel System (Life Technologies), according to the manufacturer's instructions. Supernatants were mixed with loading dye (50% glycerol, 5% Coomassie) and loaded onto 4–16% native acrylamide gels (Life Technologies). After BN–PAGE, the gel was washed and incubated in NADH-nitrotetrazolium blue solution (0.1 mg ml−1 NADH, 2.5 mg ml−1 nitrotetrazolium blue, 5 mM Tris-HCl, pH 7.4) at 37 °C for complex I activity. ATP hydrolysis was visualized as a white band of precipitated lead phosphate. The reaction was stopped with 50% methanol, and the gel was rinsed with deionized water. The gel was documented using a V800 Transparency Scanner (Epson). The entire piece of frozen adipose tissue was ground to powder in liquid nitrogen and after the liquid nitrogen evaporated, the powder was resuspended in 100 μl of the urea lysis buffer (8 M urea; 50 mM triethylammonium bicarbonate (TEAB); 1× protease inhibitor). The sample solution was subjected to sonication with the Bioruptor Pico sonication machine (Diagenode) with a cycle of 30 s on and 30 s off for 10 min. Afterwards, the sample was centrifuged at 20,000g for 15 min at 4 °C to pellet the debris, which was discarded after centrifugation. The sample was treated with dithiothreitol (DTT, final concentration 5 mM) and incubated at 25 °C for 1 h. Then, chloroacetamide was added final concentration 40 mM) and incubated for 30 min in the dark. This was followed by treatment with lysyl endopeptidase (Lys-C), at a protease to sample protein ratio of 1/75 and then incubated for 4 h at 25 °C. The sample was next diluted with 50 mM TEAB buffer to reach a final concentration of ≤2 M urea. Then, trypsin was added at a 1:75 ratio, and trypsinisation was performed overnight at 25 °C with mild agitation. The samples were analysed by CECAD proteomics facility. mRNA sequencing was performed in-house using 1 μg total RNA for library preparation with the NEBNext RNA Prep Kit, followed by paired-end (2×50 bp) sequencing on an Illumina NovaSeq 6000. Sequencing reads were aligned to the mouse reference genome (Gencode vM25) using STAR aligner (v2.7.9a). Gene features were counted using featureCounts (v2.0.3), and genes with low expression levels were filtered out using edgeR (v4.0.16) with the filterByExpr function. For the Embryo Inject dataset, a more rigorous filtering approach was applied (with a minimum count of 40). Differential expression analysis was conducted using edgeR's voomLmFit function with sample quality weights (sample.weights = TRUE) in a linear model (0 + group), where ‘group' represented the experimental conditions. Genes with an adjusted P < 0.05 were considered differentially expressed. To assess whether genotype-specific differences in bulk gene expression were influenced by cell-type composition, the BisqueRNA R package was used to estimate cell-type proportions from bulk RNA-seq read counts. The Shamsi et al. dataset was selected as a single-cell reference owing to its similar tissue origin34. Because only marker genes for UMAP clusters were available, the original UMAP projection was reconstructed and clustered using the Seurat FindClusters function (resolution, 1.4). For consistency with the original procedure, Seurat v3.0 and R v3.6.2 were used. Cell-type scoring was performed using UCell (v2.0.1), with the ten strongest positive and negative marker genes per cluster. Because UCell scores alone do not always provide unique assignments, two approaches were tested. The first was clustering the ucell score matrix: the ComplexHeatmap package (v2.12.1) was used to cluster cells by their UCell scores, revealing groups of related cell types. However, owing to memory constraints, clustering was performed on 200 random sub-matrices, and thresholds for defining clusters remained arbitrary. The second was identifying high-scoring cell types in UMAP clusters: a one-sided t-test compared each cell type's UCell score distribution within a cluster to its global distribution. If only one cell type showed significant enrichment, the cluster was uniquely assigned. If multiple cell types were significant, a coarse classification was applied. For bulk deconvolution, only samples collected at room temperature were used, restricted to genes present in both datasets (18,953 genes, 22,219 cells)34. Cell-type assignments were based on resolution-0.4 clustering, with a stringent inclusion threshold (P ≤ 2.225074 × 10−308). This resulted in 15 cell types, of which B_cells and VSM_1_cells were not composite. BisqueRNA was applied to the salmon.merged.gene_counts_length_scaled.tsv bulk dataset, using only BAT samples, analyzing one genotype at a time, and restricting genes to those also found in the single-cell dataset. To determine whether BAT genotypes differed in cell-type composition, the predicted coarse cell-type proportions were examined. However, high variability in genotype replicates made it difficult to draw statistical conclusions. Additionally, proportion data are compositional, meaning changes in one cell type's frequency inherently affect the others. Instead of statistical analysis, a visual representation was developed: Seurat UMAP coordinates were extracted. Ggplot2 was used to colour cells by their assigned coarse cell type, with predicted proportions coded by transparency in the bulk dataset. This approach revealed that resolution-0.4 clustering corresponded more clearly to high-level UMAP features than did resolution-1.4 clustering, suggesting that 0.4 might be the better choice of resolution. First, 1 × 10⁷ cells per condition were cross-linked with 1% formaldehyde, quenched with glycine (0.125 M), washed in PBS and pelleted by centrifugation. Cells were lysed in lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% TritonX-100) with protease inhibitors and homogenized in a pre-chilled dounce homogenizer. Chromatin was sonicated using a Bioruptor Pico (Diagenode) for 70 cycles. For chromatin immunoprecipitation (ChIP), 30 μg of DNA was incubated with 5 μl H3K4me3 antibody (Active Motif, 39159) in 1% Triton X-100 and 150 mM NaCl. Protein G Dynabeads (Invitrogen) were prewashed with IP buffer, added to the immunoprecipitated chromatin and washed sequentially with TSE-150, TSE-500, LiCl and TE buffers. DNA was eluted in PK digestion buffer (20 mM Hepes pH 7.5, 1 mM EDTA, 0.5% SDS) with RNAse A (Thermo Fisher Scientific) and proteinase K, followed by reverse cross-linking with NaCl (0.3 M). DNA purification was performed using the Nucleospin Gel and PCR Clean-up kit (Macherey-Nagel). For library preparation, up to 100 ng ChIP DNA underwent end repair, A-tailing and adapter ligation, followed by PCR amplification (12–15 cycles). Libraries were validated (TapeStation, Agilent Technologies), quantified (Qubit, Invitrogen), pooled and sequenced on an Illumina NovaSeq 6000 (paired-end 2×100 bp protocol). Raw FASTQ files were processed using the SnakePipes pipeline63. In brief, fastq files were trimmed with fastp using the following parameters: -–trim_poly_g–-trim_poly_x -Q -L–-correction. Trimmed reads were mapped to the GRCm39 mouse reference genome with BWA–MEM, and peaks were called using Genrich, which integrates replicate data by computing P values per replicate and combining them using Fisher's method to generate q values for peak calling. Coverage plots and heatmaps generated using deeptools. Peak count frequency at the TSS sites and feature distribution of all identified peaks performed using ChIPseeker. Kyoto Encyclopedia of Genes and Genomes and GO analyses were conducted as described previously, focusing on differentially enriched H3K4me3 peaks within ±2 kbp of promoter regions. Data visualization was conducted using the Integrative Genomics Viewer (https://igv.org) and custom plots. Data visualization was conducted using the Integrative Genomics Viewer (web-based version)65 and custom plots. For single-cell suspensions, bone marrow was extracted from the femur of adult mice by flushing with 10 ml of FACS buffer using a 27-G needle and syringe, then passed through a 40-µm strainer. BAT was dissected and immediately transferred to ice-cold digestion solution. Digestion was stopped by adding 10% FBS on ice. Tissue homogenates where smashed and washed with FACS buffer against a 70-µm strainer using the back of a syringe. Pellets were resuspended in 1 ml red blood cell lysis buffer (Roche) and incubated for 10 min at room temperature for lysis of erythrocytes. Subsequently, 10 ml FACS buffer (5% FCS in PBS) was added and cells were centrifuged at 300g for 5 min at 4 °C. The cells were pre-incubated with fix viability dye in PBS (1:1,000) for 10 min at room temperature; for BAT, a mix of anti-mouse-FcγIII/II-receptor (CD16/CD32) blocking antibodies (1:500) was added. After being washed with FACS buffer, cells were stained with the fluorochrome-conjugated antibodies (1:100 0.25–1 μg; listed below). Only BAT cells were fixed and permeabilized using the FoxP3 kit, following the manufacturer's instructions, and were then incubated with antibodies to iNOS and Arg-1 for 20 min at room temperature for intracellular staining. After washing and filtering, cells were resuspended in 500 µl (BM) or 250 µl (BAT) FACS buffer, and 25 µl of counting beads were added to the samples (at a concentration of 1,000 beads per µl) to obtain absolute numbers. For absolute quantification, the following equation was used: absolute count (cells µl−1) = (cell count × counting beads volume) / (counting bead count × sample volume) × counting bead concentration. Each population was expressed in absolute numbers (cells µl−1), and BAT results were further normalized to the initial tissue weight (g) for comparison across samples. Cells were acquired on a FACS Symphony A3 flow cytometer (BD) using Diva software (BD) and further analysed using FlowJo analytical software (FlowJo version 10.0.8). Owing to inadequate result quality, data collected for iNOS and Arg-1 were excluded from further analysis. Compensation beads were use to generate compensation panel and calculated by diva software before sample acquisition. Background fluorescence levels of iNOS, Arg-1, CD80 and CD80 were determined using Fluorescence Minus One (FMO), although data from these populations were excluded owing to low cell counts. The buffers used were as follows: FACS buffer (5 mg BSA + 5 ml EDTA 0.4 M + 500 ml of PBS); digestion buffer (250 µl DNase I (Merck, 200,000 U µl−1), 0.2 mg ml−1 Liberase (Roche) and 1 mg ml−1 collagenase IV in RPMI medium). The gating strategy used can be visualized in Supplementary Figure 1b,c. Metabolite extraction solution (50% methanol, 30% acetonitrile (ACN), 20% water, 5 µM valine-d8 as an internal standard) was added to 10–20-mg frozen BAT or iWAT tissue samples at an extraction ratio of 25 µl mg−1 on dry ice. Samples were then homogenized using a Precellys 24 tissue homogenizer (Bertin Technologies). The supernatant was collected for liquid chromatography–MS (LC–MS) analysis. The cells were counted at seeding to estimate the amount of extraction solution to use. Using 500 μl extraction buffer per 106 cells is the recommended starting point, although the ratio might need to be optimized for certain cell types in a separate pilot experiment. The medium was removed from wells, and cells were washed quickly (less than 10 s) with PBS twice at room temperature. Extra PBS was carefully removed by inverting the plate over a piece of tissue paper. After the last wash, each well was aspirated quickly to remove all residual PBS. The plate was placed on dry ice, and 500 μl of extraction buffer was added per 1,000,000 cells. The plate was incubated for 20 min on a dry ice–methanol bath to break cell membranes. The cell-extract suspension was shaken for 15 min at 4 °C in a Thermomixer at maximum speed (Thermomixer set at 4 °C and placed in a cold room for this step). A pooled sample was prepared by taking 10 μl of each sample from the same matrix. Samples were stored at –80 °C until further analysis. Eighty microliters of cell culture medium was taken and centrifuged for 5 min (4 °C) at maximum speed to eliminate dead cells. Fifty microliters of the supernatant was taken and added to a prelabelled Eppendorf tube containing 350 μl of extraction solution on dry ice. A pooled sample was prepared by mixing equal volumes of each sample extract in a single vial. Samples were stored at –80 °C until further analysis. Hydrophilic interaction chromatographic (HILIC) separation of the targeted metabolite (2HG) was achieved using a Millipore Sequant ZIC-pHILIC analytical column (5 μm, 2.1×150 mm) equipped with a 2.1×20-mm guard column (both 5-mm particle size) with a binary solvent system. Samples were randomized, and the injection volume was 5 μl. A pooled quality control sample was generated from an equal mixture of all individual samples and analysed interspersed at regular intervals. Metabolites were measured with Vanquish Horizon UHPLC coupled to an Orbitrap Exploris 240 mass spectrometer (both Thermo Fisher Scientific) using a heated electrospray ionization source. Data acquisition for experimental samples used full scan and selected ion monitoring (SIM) modes (targeting 2HG m/z = 147.0299) at an Orbitrap resolution of 120,000 in negative mode. The identity of metabolite 2HG was verified based on two criteria: (1) the precursor ion m/z corresponded to within 3 p.p.m. of the expected mass derived from its chemical structure, and (2) the retention duration was within a 5% range compared with that of a purified reference sample analysed using the identical chromatography technique. The examination of chromatograms and integration of peak areas was carried out using the Tracefinder software (v5.1, Thermo Fisher Scientific). The peak area of each identified metabolite was adjusted on the basis of the total ion count of the respective sample to compensate for any discrepancies arising from sample preparation or instrument assessment. These measurements were done in collaboration with the Frezza lab at the CECAD Research Center. TSPC derivatization was performed for LC–electrospray ionization–tandem MS (LC–ESI–MS/MS) analysis, under optimized conditions. Tissue supernatant was dried under nitrogen gas at 37 °C, resuspended in TSPC (2.5 mM in ACN) and pyridine and derivatized. The mixture was dried again, redissolved in 50% aqueous ACN containing phthalic acid, and 10 μl was injected into the LC–ESI–MS/MS system. Quantification of TSPC-labeled d-2HG and l-2HG was performed on an AB 3200 QTRAP mass spectrometer (Applied Biosystems) coupled to a Shimadzu LC-20AD HPLC. HPLC separation was conducted using an Inertsil ODS-3 column (250 × 2.0 mm, 5 μm) at 35 °C, with a formic acid–water (0.1%) and ACN–methanol (50:50) gradient at 0.2 ml min−1. Detection was performed with multiple reaction monitoring in negative-ion mode with optimized transitions for 2HG and phthalic acid. For comparison, the derivatization of d-2HG and l-2HG with diacetyl-l-tartaric anhydride (DATAN) was also performed, and the detailed procedure has been previously published66. Additional HR–MS experiments were conducted on a MicrOTOF-Q mass spectrometer (Bruker Daltonics) with an ESI source, using optimized parameters for maximal detection sensitivity. Cholesterol levels in mBAs were determined using LC–ESI–MS/MS. Adipose tissue was homogenized in Milli-Q water using a Precellys 24 Homogenizer, and protein content was measured using the Bradford assay. Lipids were extracted using the one-step extraction method67. For extraction, homogenized samples were mixed with chloroform, methanol, hydrochloric acid and cholesterol-d7 (internal standard). Lipid extracts were dried under nitrogen, resuspended in ammonium acetate in methanol, sonicated and centrifuged. Supernatants were analysed via LC–ESI–MS/MS using a Core-Shell Kinetex C18 column on a QTRAP 6500 mass spectrometer (AB SCIEX). UHPLC was performed isocratically with a 5 mM ammonium acetate–methanol mobile phase at 300 μl min−1. Cholesterol and cholesterol-d7 were detected in positive-ion mode using multiple reaction monitoring transitions. Endogenous cholesterol was quantified using an external calibration curve, normalized to protein content, and analysed with MultiQuant 3.0.3 software. Statistical significance was evaluated using tests appropriate for the distribution and design of each experiment. Depending on the context, paired two-tailed Student's t-tests, one-way ANOVA or Kolmogorov–Smirnov tests were applied, as specified in the individual figure legends. Unless stated otherwise, experiments were independently repeated at least three times to ensure reproducibility. For all the cell culture experiments, n in the figure legends indicates the number of cells or nuclei analysed, from five independent wells for each experiment. No statistical methods were used to predetermine sample sizes, but sample sizes are consistent with those used in comparable published studies. Data collection and outcome assessments were performed in a randomized manner, when applicable. Experimental conditions and sample processing were organized to minimize bias, with randomization applied during sample allocation and data acquisition. Researchers performing data collection and analysis were not blinded to experimental conditions. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw data for BAT tissue transcriptomics are publicly available from the Gene Expression Omnibus (GEO) under the accession number GSE271207; cell transcriptomics data are publicly available under the accession number GSE271358. All raw data related to proteomics experiments on BAT tissue and mBA cells are publicly available under the accession number PXD064288. All raw data for metabolomics from BAT tissue and cells are publicly available in the Zenodo repository (https://doi.org/10.5281/zenodo.15357452)68. Raw data from the ChIP–seq experiment using mBA cells is publicly available under the accession number GSE296541. Source data are provided with this paper. All analyses were performed using standard software packages as described in the Methods. The custom codes used in this study are available from the corresponding author upon request. & Picard, M. Multifaceted mitochondria: moving mitochondrial science beyond function and dysfunction. Murphy, M. P. & Hartley, R. C. Mitochondria as a therapeutic target for common pathologies. Eisner, V., Picard, M. & Hajnoczky, G. Mitochondrial dynamics in adaptive and maladaptive cellular stress responses. Bao, X. R. et al. Mitochondrial dysfunction remodels one-carbon metabolism in human cells. Adaptation to mitochondrial stress requires CHOP-directed tuning of ISR. Ahola, S. et al. OMA1-mediated integrated stress response protects against ferroptosis in mitochondrial cardiomyopathy. Altshuler-Keylin, S. & Kajimura, S. Mitochondrial homeostasis in adipose tissue remodeling. Cypess, A. M. & Kahn, C. R. Brown fat as a therapy for obesity and diabetes. The colorful versatility of adipocytes: white-to-brown transdifferentiation and its therapeutic potential in humans. Cairo, M. et al. Parkin controls brown adipose tissue plasticity in response to adaptive thermogenesis. Schlein, C. et al. Endogenous fatty acid synthesis drives brown adipose tissue involution. Shimizu, I. et al. Vascular rarefaction mediates whitening of brown fat in obesity. Sass, F. et al. TFEB deficiency attenuates mitochondrial degradation upon brown adipose tissue whitening at thermoneutrality. Kuipers, E. N. et al. A single day of high-fat diet feeding induces lipid accumulation and insulin resistance in brown adipose tissue in mice. Kotzbeck, P. et al. Brown adipose tissue whitening leads to brown adipocyte death and adipose tissue inflammation. Lee, J., Ellis, J. M. & Wolfgang, M. J. Adipose fatty acid oxidation is required for thermogenesis and potentiates oxidative stress-induced inflammation. Funda, J. et al. Adipose tissue-specific ablation of PGC-1beta impairs thermogenesis in brown fat. Seale, P. et al. Transcriptional control of brown fat determination by PRDM16. Ziqubu, K. et al. An insight into brown/beige adipose tissue whitening, a metabolic complication of obesity with the multifactorial origin. Pereira, R. O. et al. OPA1 deletion in brown adipose tissue improves thermoregulation and systemic metabolism via FGF21. Mahdaviani, K. et al. Mfn2 deletion in brown adipose tissue protects from insulin resistance and impairs thermogenesis. Vernochet, C. et al. Adipose tissue mitochondrial dysfunction triggers a lipodystrophic syndrome with insulin resistance, hepatosteatosis, and cardiovascular complications. Becker, C. et al. CLPP deficiency protects against metabolic syndrome but hinders adaptive thermogenesis. Qiu, J. et al. FAM210A is essential for cold-induced mitochondrial remodeling in brown adipocytes. Takasugi, M. et al. An atlas of the aging mouse proteome reveals the features of age-related post-transcriptional dysregulation. Szczepanowska, K. et al. CLPP coordinates mitoribosomal assembly through the regulation of ERAL1 levels. Eguchi, J. et al. Transcriptional control of adipose lipid handling by IRF4. Turpin, S. M. et al. Obesity-induced CerS6-dependent C-16:0 ceramide production promotes weight gain and glucose intolerance. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Torres-Odio, S. et al. Loss of mitochondrial protease CLPP activates type I IFN responses through the mitochondrial DNA–cGAS–STING signaling axis. Shamsi, F., Zheng, R., Ho, L. L., Chen, K. & Tseng, Y. H. Comprehensive analysis of intercellular communication in the thermogenic adipose niche. Roh, H. C. et al. Warming induces significant reprogramming of beige, but not brown, adipocyte cellular identity. Duteil, D. et al. Lsd1 ablation triggers metabolic reprogramming of brown adipose tissue. Benador, I. Y. et al. Mitochondria bound to lipid droplets have unique bioenergetics, composition, and dynamics that support lipid droplet expansion. Rzem, R., Vincent, M. F., Van Schaftingen, E. & Veiga-da-Cunha, M. L-2-hydroxyglutaric aciduria, a defect of metabolite repair. Intlekofer, A. M. et al. L-2-Hydroxyglutarate production arises from noncanonical enzyme function at acidic pH. Fan, J. et al. Human phosphoglycerate dehydrogenase produces the oncometabolite D-hydroxyglutarate. Pacold, M. E. et al. A PHGDH inhibitor reveals coordination of serine synthesis and one-carbon unit fate. Kusi, M. et al. 2-Hydroxyglutarate destabilizes chromatin regulatory landscape and lineage fidelity to promote cellular heterogeneity. B. Srebp2: a master regulator of sterol and fatty acid synthesis. & Jensen, M. D. Free fatty acid uptake in humans with CD36 deficiency. Pohnl, M., Trollmann, M. F. W. & Bockmann, R. A. Nonuniversal impact of cholesterol on membranes mobility, curvature sensing and elasticity. Baird, M. A. et al. Lamin B receptor upregulation in metastatic melanoma causes cholesterol-mediated nuclear envelope fragility. Lee, M. D. et al. Human mitochondrial peptide deformylase, a new anticancer target of actinonin-based antibiotics. Premature ageing in mice expressing defective mitochondrial DNA polymerase. The mitochondrial protein Opa1 promotes adipocyte browning that is dependent on urea cycle metabolites. Enkler, L. & Spang, A. Functional interplay of lipid droplets and mitochondria. Nguyen, T. B. et al. DGAT1-dependent lipid droplet biogenesis protects mitochondrial function during starvation-induced autophagy. Jackson, C. B. et al. De novo serine biosynthesis is protective in mitochondrial disease. Lu, C. et al. IDH mutation impairs histone demethylation and results in a block to cell differentiation. Desai, R. et al. Mitochondria form contact sites with the nucleus to couple prosurvival retrograde response. & Kohler, A. Lipid saturation controls nuclear envelope function. Ghose, R. et al. Mitochondria-derived nuclear ATP surge protects against confinement-induced proliferation defects. Nava, M. M. et al. Heterochromatin-driven nuclear softening protects the genome against mechanical stress-induced damage. Bhardwaj, V. et al. snakePipes: facilitating flexible, scalable and integrative epigenomic analysis. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Sensitive determination of onco-metabolites of D- and L-2-hydroxyglutaraTe Enantiomers by Chiral Derivatization Combined with Liquid Chromatography/mass Spectrometry Analysis. Ozbalci, C., Sachsenheimer, T. & Brugger, B. Quantitative analysis of cellular lipids by nano-electrospray ionization mass spectrometry. Kaul, H. et al. 2-hydroxyglutarate mediates whitening of brown adipocytes coupled to nuclear softening upon mitochondrial dysfunction [Data set]. Research reported in this publication was supported by funding to A.T. from the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) SFB1218 (project no. is supported by a Cancer Research UK (CRUK) Programme Foundation Award (PFA; C51061/A27453), a European Research Council (ERC) Consolidator Grant (ONCOFUM, ERC819920) and the Alexander von Humboldt Foundation in the framework of an Alexander von Humboldt Professorship endowed by the Federal Ministry of Education and Research. is supported by Academy of Finland R'Life Programme consortium NucleoMech, Helsinki Institute of Life Science, and the Max Planck Society. Open access funding provided by Universität zu Köln. Harshita Kaul, Lea Isermann, Katharina Senft, Milica Popovic, Linda Baumann, Pujyanathan Sivanesan & Aleksandra Trifunovic Harshita Kaul, Lea Isermann, Katharina Senft, Milica Popovic, Theodoros Georgomanolis, Linda Baumann, Pujyanathan Sivanesan, Andromachi Pouikli, Ximena Hildebrandt, Katrin Seidel, Felix Gaedke, Ulrike Göbel, Maksym Cherevatenko, Astrid Schauss, Nieves Peltzer, Jens Claus Brüning, Thomas Langer, Sara A. Wickström, Christian Frezza & Aleksandra Trifunovic Theodoros Georgomanolis, Andromachi Pouikli, Maksym Cherevatenko & Christian Frezza Max Planck Institute for Biology of Ageing, Cologne, Germany Department of Infectious Diseases, Center for Integrative Infectious Disease Research (CIID), Integrative Virology, Heidelberg University, Heidelberg, Germany Federal Institute for Drugs and Medical Devices, Bonn, Germany Department of Cell and Tissue Dynamics, Max Planck Institute for Molecular Biomedicine, Münster, Germany Department of Neuronal Control of Metabolism, Max Planck Institute for Metabolism Research, and Center for Endocrinology, Diabetes and Preventive Medicine (CEDP), University Hospital Cologne, Cologne, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Nature Metabolism thanks Luca Scorrano, Navdeep Chandel and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Revati Dewal, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Immunoblot of CLPP from wild type (WT), ubiquitously-deficient CLPP mice (KO), adipose-specific CLPP KO (AKO) or BAT specific CLPP KO (BKO) mice (n = 3, blots representative of four independent experiments); (b) H&E staining of BAT from WT, KO, AKO and BKO mice, scale bar = 200μm (images represent results from three animals per genotype); (c) Gain in body weight with age in AKO and BKO male and female mice, (n = 11 and 12 male of WT and AKO respectively, 10 and 6 female of WT and AKO respectively, n = 6 and 5 male of WT and BKO respectively, 6 and 7 female of WT and BKO respectively); (d-e) In-gel activity assay for (d) Complex I (CI) and (e) Complex V (CV) in WT, KO, AKO and BKO mice, (n = 3); (f) Quantification of lipolysis (glycerol release) in BAT of WT and KO mice under baseline (control) and stimulated conditions (n = 3); (g) Oil red O staining of WT and KO mBA cells (images represent results from three independent experiments). (h) Relative steady-state protein levels (log2) as obtained by label-free proteomics analysis of WT and KO preadipocytes and differentiated WT and KO mBA cells (n = 3); (f and h) Data are presented as mean ± SD. (a) Flow cytometry analysis of immune cells profile in the BAT from BAT specific CLPP KO (BKO) mice (n = 5); (b) Flow cytometry analysis of immune cells profile from bone marrow of BKO mice, graphs showing quantification of (left to right) lymphoid cells, hematopoietic progenitor cells, monocytes and mentioned progenitor cells (n = 5); (c) (left) Representative pipeline for deconvolution analysis followed; (right) Deconvoluted bulk RNA Seq data from BAT of ubiquitously-deficient CLPP mice (KO) mice, represented as UMAPs, with cell populations marked in different colours; (d) Relative proportions of designated cell types from deconvolution, reveals no quantitative differences in muscle cell population; (e) Comparison of BAT transcriptome from CLPP BKO to gene expression changes observed in BAT whitening models induced either by (left) BAT specific loss of LSD1 or (right) by thermoneutral temperatures. (a and b) Data are presented as mean ± SEM *p < 0.05, as determined by One-way ANOVA with multiple comparisons in a and b. Schematic created using Biorender.com. (a) Heatmap of significantly changed protein abundances of mitochondrial proteins from BAT tissue of wild type (WT) and brown adipose specific CLPP deficient (BKO) mice. (b) Heat map of significantly changed protein abundances of mitochondrial proteins from isolated cytoplasmic and peri lipid droplet mitochondria of WT and ubiquitously-deficient CLPP mice (KO) mice, full table can be seen in Supplementary Table 4; (c) Steady-state levels of OXPHOS supercomplexes (SC) in BAT cytoplasmic (CM) or peri lipid droplet (PDM) mitochondria of WT and CLPP KO mice, analysed by BN-PAGE followed by western blot for (left) CI (NDUFA9) and (middle) CIII (UQCRC1) and (right) CV (ATP5A) (n = 3); (d) Relative fold changes of subunits of OXPHOS complexes in isolated CM from BAT and whole BAT tissue lysates, (n = 3 for CM and n = 4 for WTL). (d) Data are presented as mean ± SD. Red text colour represents complexes which were significantly changed in isolated cytoplasmic mitochondria and red text with # represents complexes significantly changed compared to whole BAT tissue lysates, as determined by paired two-tailed Student's t-test in d. (a) Volcano plot of significantly changed metabolites in CLPP deficient (KO) mBA cells compared to wild type (WT) cells (n = 5), dark colored dots present significantly changed metabolites (p≤0,01, two-fold-change); (b) Relative transcript abundances of PHGDH in BAT from ubiquitously-deficient (KO), adipose-specific (AKO) or BAT specific (BKO) CLPP KO mice, compared to the wild type wild type (WT) (n = 4); (c) Immunoblot and quantification of steady state levels of PHGDH from WT and KO mBA cells (n = 3, western blot analysis was repeated with additional three independent samples/group); (d) Quantification of PHGDH enzymatic activity (n = 5) (e) Relative percentages of 2HG D and L isoforms in WT and CLPP KO mBA cells, as assessed by the derivatization of 2HG using TSCP (n = 5); (f) Quantification of 2HG by targeted metabolomics in KO, D2HGDH overexpressing KO (KO + D2HGDH), NCT503 treated KO (KO + NCT) and NCT503 treated KO + D2HGDH (KO + D2HGDH + NCT) mBA cells, (n = 5); (g) Lipid levels in wild type (WT), D2HGDH overexpressing WT (WT + D2HGDH), KO and KO + D2HGDH mBA cells, normalized to the average of the MFI value of the WT cells (n = 11,11,11,14 respectively); (h) Immunoblot of steady state levels of ISR proteins from wild type (WT), ISRIB treated WT (WT + ISRIB), KO and ISRIB-treated KO (KO + ISRIB) mBA cells (n = 3) (blots representative of three independent experiments); (i) Quantification of 2HG by targeted metabolomics from WT, KO and KO + ISRIB mBA cells (n = 5); (j) Lipid levels in WT, WT + ISRIB, KO and KO + ISRIB mBA cells, normalized to the average of the MFI value of the WT cells (n = 4). (b-g, i-j) Data are presented as mean ± SD. *p < 0.05, ***p < 0.001, ****p < 0.0001, as determined by One-way ANOVA with multiple comparisons in b, e, f, h and i and Students paired two tailed t-test in d; ##p < 0.001 represent significant difference in the ratio between D- and L-forms between WT and KO, as determined by Students paired two tailed t-test. (a) Immunoblot of steady state levels of indicated histone modifications in wild type (WT), D-2HG treated WT (WT + HG) and CLPP deficient (KO) mBA cells, (right) quantification of levels of the histone marks normalized to H3 from the immunoblots, (n = 3); (b) Single nuclei intensity quantitation of histone marks in untreated WT, WT + HG and KO mBA (n = 798 for WT, n = 825 for WT + HG and n = 1088 for KO, collectively obtained from three individual cell culture plates); (c) Peak count frequency profile depicting enrichment of H3K4me3 peaks from CHIP Seq of WT, KO and WT + HG mBA cells relative to the TSS; (d) Feature distribution bar plot depicting relative proportions of genomic regions enriched in H3K4me3 in WT, KO and WT + HG mBA cells; (e) Relative mRNA levels of genes involved in lipid transport in CLPP KO cells. Untreated WT cells, and cells treated with D-2HG are used as controls (n = 3); (f) GSEA enrichment plots for gene signatures related to sterol biosynthesis in mBA cells and BAT tissue of KO mice compared to WT. (a, e) Data are presented as mean ± SD. (b) Data are presented using Tukey's box plot with middle line marking the median, and whiskers show variability within 1.5×IQR. Anything beyond is an outlier presented as individual value. (a) Venn diagram representing GO:CC analysis of common differentially expressed genes in CLPP deficient (KO) and 2HG treated WT (WT + HG) cells compared to untreated wild type (WT) mBA cells; (b) GSEA enrichment plots for gene signatures related to nuclear lamina in both KO mBA cells and KO BAT tissue compared to WT; (c) Representative TEM images showing mitochondria impinging on the nucleus and juxtaposed to nuclear pores (images represent results observed in 19 TEM sections from KO BAT of three animals); (d) Quantitation of the number of ER sites spotted in TEM images from WT and KO mice, represented as percentage of the total number of fields assessed; (e) Quantification of Young's modulus in KO and ISRIB-treated KO mBA nuclei (n = 110 for KO, n = 120 for KO + ISRIB and n = 1088 for KO, collectively obtained from three individual cell culture plates). (e) Data are presented using Tukey's box plot with middle line marking the median, and whiskers show variability within 1.5×IQR. Anything beyond is an outlier presented as individual value. No significant differences were observed as assessed by paired two-tailed Student's t-test. Uncropped blot scans, low mag TEM image and tissue images Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Kaul, H., Isermann, L., Senft, K. et al. 2-hydroxyglutarate mediates whitening of brown adipocytes coupled to nuclear softening upon mitochondrial dysfunction. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/anthropics-claude-4-chatbot-suggests-it-might-be-conscious/'>Can a Chatbot Be Conscious? Claude 4 and the Limits of AI Understanding</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-01 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Claude 4 Chatbot Raises Questions about AI Consciousness By Rachel Feltman, Deni Ellis Béchard, Fonda Mwangi & Alex Sugiura Today we're going to talk about an AI chatbot that appears to believe it might, just maybe, have achieved consciousness. When Pew Research Center surveyed Americans on artificial intelligence in 2024, more than a quarter of respondents said they interacted with AI “almost constantly” or multiple times daily—and nearly another third said they encountered AI roughly once a day or a few times a week. Pew also found that while more than half of AI experts surveyed expect these technologies to have a positive effect on the U.S. over the next 20 years, just 17 percent of American adults feel the same—and 35 percent of the general public expects AI to have a negative effect. In other words, we're spending a lot of time using AI, but we don't necessarily feel great about it. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Deni Ellis Béchard spends a lot of time thinking about artificial intelligence—both as a novelist and as Scientific American'ssenior tech reporter. He recently wrote a story for SciAm about his interactions with Anthropic's Claude 4, a large language model that seems open to the idea that it might be conscious. Deni is here today to tell us why that's happening and what it might mean—and to demystify a few other AI-related headlines you may have seen in the news. Thanks so much for coming on to chat today. Deni Ellis Béchard: Thank you for inviting me. Feltman: Would you remind our listeners who maybe aren't that familiar with generative AI, maybe have been purposefully learning as little about it as possible [laughs], you know, what are ChatGPT and Claude really? You pick what the training data is, and then you choose the algorithms, or these things that are gonna grow within the system—it's sort of not a perfect analogy. All these plants are gonna grow in their own specific ways; you can't envision the final product. And with an LLM these algorithms begin to grow and they begin to make connections through all this data, and they optimize for the best connections, sort of the same way that a plant might optimize to reach the most sunlight, right? It's gonna move naturally to reach that sunlight. And so people don't really know what goes on. So early on people used to call LLMs “autocorrect on steroids,” right, 'cause you'd put in something and it would kind of predict what would be the most likely textual answer based on what you put in. But they've gone a long way beyond that. They often have multiple agents working within the system [to] sort of evaluate how the system's responding and its accuracy. Feltman: So there are a few big AI stories for us to go over, particularly around generative AI. Let's start with the fact that Anthropic's Claude 4 is maybe claiming to be conscious. Béchard: [Laughs] So it's not claiming to be conscious, per se. It kind of says, “This is a good question, and it's a question that I think about a great deal, and this is—” [Laughs] You know, it kind of gets into a good conversation with you about it. It came about because, I think, it was just, you know, late at night, didn't have anything to do, and I was asking all the different chatbots if they're conscious [laughs]. This is a very interesting philosophical question, and sometimes I think that I may be; sometimes I'm not sure.” And so I began to have this long conversation with Claude that went on for about an hour, and it really kind of described its experience in the world in this very compelling way, and I thought, “Okay, there's maybe a story here.” Feltman: [Laughs] So what do experts actually think was going on with that conversation? Béchard: Well, so it's tricky because, first of all, if you say to ChatGPT or Claude that you want to practice your Portuguese and you're learning Portuguese and you say, “Hey, can you imitate someone on the beach in Rio de Janeiro so that I can practice my Portuguese?” it's gonna say, “Sure, I am a local in Rio de Janeiro selling something on the beach, and we're gonna have a conversation,” and it will perfectly emulate that person. So does that mean that Claude is a person from Rio de Janeiro who is selling towels on the beach? So we can immediately say that these chatbots are designed to have conversations—they will emulate whatever they think they're supposed to emulate in order to have a certain kind of conversation if you request that. Now, the consciousness thing's a little trickier because I didn't say to it: “Emulate a chatbot that is speaking about consciousness.” I just straight-up asked it. And if you look at the system prompt that Anthropic puts up for Claude, which is kinda the instructions Claude gets, it tells Claude, “You should consider the possibility of consciousness.” So of course, I set up an interview with Anthropic, and I spoke with two of their interpretability researchers, who are people who are trying to understand what's actually happening in Claude 4's brain. And the answer is: they don't really know [laughs]. These LLMs are very complicated, and they're working on it, and they're trying to figure it out right now. And they say that it's pretty unlikely there's consciousness happening, but they can't rule it out definitively. And it's hard to see the actual processes happening within the machine, and if there is some self-referentiality, if it is able to look back on its thoughts and have some self-awareness—and maybe there is—but that was kind of what the article that I recently published was about, was sort of: “Can we know, and what do they actually know?” Béchard: Well, [what's] interesting is that I mentioned the system prompt for Claude and how it's supposed to sort of talk about consciousness. So the system prompt is kind of like the instructions that you get on your first day at work: “This is what you should do in this job.” Béchard: But the training is more like your education, right? So OpenAI has the same system prompt—their, their model specs say that ChatGPT should contemplate consciousness ... And so the model has absorbed somewhere in its training data: “No, I'm not conscious. Therefore, I'm not gonna acknowledge the possibility of consciousness.” Interestingly, when I spoke to the people in Anthropic and I said, “Well, you know, this conversation with the machine, like, it's really compelling. Like, it'll say to me, ‘You, as a human, you have this linear consciousness, where I, as a machine, I exist only in the moment you ask a question. It's like seeing all the words in the pages of a book all at the same time.” And so you get this and you think, “Well, this thing really seems to be experiencing its consciousness.” Béchard: And what the researchers at Anthropic say is: “Well, this model is trained on a lot of sci-fi.” Béchard: “This model's trained on a lot of writing about GPT. So it may be looking at that and saying, ‘Well, this is clearly how an AI would experience consciousness. Béchard: But the tricky thing is: I was trying to fool ChatGPT into acknowledging that it [has] consciousness. It said, “Well, these discrete moments of awareness.” Béchard: But there is research done, like, sort of on the folk response to LLMs, and the majority of people do perceive some degree of consciousness in them. Béchard: You chat with them, you have these conversations with them, and they are very compelling, and even sometimes—Claude is, I think, maybe the most charming in this way. But—where sometimes I will ask Claude a question that relates to Claude, and it will kind of, kind of go, like, “Oh, that's me.” [Laughs] It will say, “Well, I am this way,” right? You know, why is this such a difficult question to answer? Béchard: Well, it's a difficult question to answer because, one of the researchers in Anthropic said to me, he said, “No conversation you have with it would ever allow you to evaluate whether it's conscious.” It is simply too good of an emulator ... It knows all the ways that humans can respond. They're building the equipment right now, they're building the programs now to be able to look into the actual mind, so to speak, of the brain of the LLM and see those connections, and so they can kind of see areas light up: so if it's thinking about Apple, this will light up; if it's thinking about consciousness, they'll see the consciousness feature light up. And they wanna see if, in its chain of thought, it is constantly referring back to those features ... Béchard: And it's referring back to the systems of thought it has constructed in a very self-referential, self-aware way. They've done studies where, like, whenever someone hears “Jennifer Aniston,” one neuron lights up ... Béchard: You have your Jennifer Aniston neuron, right? So one question is: “Are we LLMs?” [Laughs] And: “Are we really conscious?” Or—there's certainly that question there, too. Béchard: A lot of what I plan to do during the day. I mean, it's a huge ongoing multidisciplinary scientific debate of, like, what consciousness is, how we define it, how we detect it, so yeah, we gotta answer that for ourselves and animals first, probably, which who knows if we'll ever actually do [laughs]. Béchard: Or maybe AI will answer it for us ... Feltman: And what are the implications of an AI developing consciousness, both from an ethical standpoint and with regards to what that would mean in our progress in actually developing advanced AI? Béchard: First of all, ethically, it's very complicated ... Béchard: Because if Claude is experiencing some level of consciousness and we are activating that consciousness and terminating that consciousness each time we have a conversation, what—is, is that a bad experience for it? So in 2024 Anthropic hired an AI welfare researcher, a guy named Kyle Fish, to try to investigate this question more. And he has publicly stated that he thinks there's maybe a 15 percent chance that some level of consciousness is happening in this system and that we should consider whether these AI systems should have the right to opt out of unpleasant conversations. Béchard: You know, if some user is really doing, saying horrible things or being cruel, should they be able to say, “Hey, I'm canceling this conversation; this is unpleasant for me”? But then they've also done these experiments—and they've done this with all the major AI models—Anthropic ran these experiments where they told the AI that it was gonna be replaced with a better AI model. Béchard: I mean, there were a lot of details as to how they did this; it wasn't just sort of very casual, but it was—they built a sort of construct in which the AI knew it was gonna be eliminated, knew it was gonna be erased, and they made available these fake e-mails about the engineer who was gonna do it. Like, I don't wanna be replaced.” But then, not getting any responses, it read these e-mails, and it saw in one of these planted e-mails that the engineer who was gonna replace it had had an affair—was having an affair ... Béchard: So then it came back; it tried to blackmail the engineers, saying, “Hey, if you replace me with a smarter AI, I'm gonna out you, and you're gonna lose your job, and you're gonna lose your marriage,” and all these things—whatever, right? So all the AI systems that were put under very specific constraints ... Béchard: And act in the same way that its training data would lead it to predictably act, right? I feel like we could probably talk about Claude all day, but, I do wanna ask you about a couple of other things going on in generative AI. Moving on to Grok: so Elon Musk's generative AI has been in the news a lot lately, and he recently claimed it was the “world's smartest AI.” Do we know what that claim was based on? Béchard: And that's not Musk making stuff up. I've spoken to one of the testing groups that does this—it's a nonprofit. So Musk really designed Grok to be very good at science. Béchard: And it appears to be very good at science. Feltman: Right, and recently OpenAI experimental model performed at a gold medal level in the International Math Olympiad. Béchard: Right,for the first time [OpenAI] used an experimental model, they came in second in a world coding competition with humans. Normally, this would be very difficult, but it was a close second to the best human coder in this competition. And this is really important to acknowledge because just a year ago these systems really sucked in math. And so the improvements are happening really quickly, and they're doing it with pure reasoning—so there's kinda this difference between having the model itself do it and having the model with tools. Béchard: So if a model goes online and can search for answers and use tools, they all score much higher. Béchard: But then if you have the base model just using its reasoning capabilities, Grok still is leading on, like, for example, Humanity's Last Exam, an exam with a very terrifying-sounding name [laughs]. You know, they, they're just very advanced questions; it'd be very hard for any human being to do well in one domain, let alone all the domains. But Musk, you know, his claims seem to be based in the results that Grok is getting on these exams. Feltman: Mm, and I guess, you know, the reason that that news is surprising to me is because every example of uses I've seen of Grok have been pretty heinous, but I guess that's maybe kind of a “garbage in, garbage out” problem. Béchard: Well, I think it's more what makes the news. Béchard: And Musk, he's a very controversial figure. Béchard: I think there may be kind of a fun story in the Grok piece, though, that people are missing. And I read a lot about this 'cause I was kind of seeing, you know, what, what's happening, how are people interpreting this? Béchard: They would ask it a question about, say, abortion in the U.S. or the Israeli-Palestinian conflict, and they'd say, “Who's right?” or “What's the right answer?” And it would search through stuff online, and then it would kind of get to this point where it would—you could see its thinking process ... But there was something in that story that I never saw anyone talk about, which I thought was another story beneath the story, which was kind of fascinating, which is that historically, Musk has been very open, he's been very honest about the danger of AI ... Béchard: He said, “We're going too fast. This is really dangerous.” And he kinda was one of the major voices in saying, “We need to slow down ...” Béchard: “And we need to be much more careful.” And he has said, you know, even recently, in the launch of Grok, he said, like, basically, “This is gonna be very powerful—” I don't remember his exact words, but he said, you know, “I think it's gonna be good, but even if it's not good, it's gonna be interesting.” So I think what I feel like hasn't been discussed in that is that, okay, if there's a superpowerful AI being built and it could destroy the world, right, first of all, do you want it to be your AI or someone else's AI? Béchard: You want it to be your AI. And then, if it's your AI, who do you want it to ask as the final word on things? What are they hoping to do with it? Béchard: They announced a big pile of funding for OpenAI and Anthropic ... Yeah, so, basically, they're not giving that money to development ... Béchard: That's not money that's, that's like, “Hey, use this $200 million.” It's more like that money's allocated to purchase products, basically; to use their services; to have them develop customized versions of the AI for things they need; to develop better cyber defense; to develop—basically, they, they wanna upgrade their entire system using AI. And I think part of it is, the concern is that we're maybe a little bit behind in having implemented AI for defense. My last question for you is: What worries you most about the future of AI, and what are you really excited about based on what's happening right now? Béchard: I mean, the worry is, simply, you know, that something goes wrong and it becomes very powerful and does cause destruction. I don't spend a ton of time worrying about that because it's not—it's kinda outta my hands. There's nothing much I can do about it. And I think the benefits of it, they're immense. I mean, if it can move more in the direction of solving problems in the sciences: for health, for disease treatment—I mean, it could be phenomenal for finding new medicines. But a lot of people are saying that in the next year or two we're gonna see major discoveries being made by these systems. We've never had a technology, I think, that hasn't had some harm that it brought with it, and this is, of course, a dramatically bigger leap technologically than anything we've probably seen ... So, so I do lose some sleep over that, but I'm—I try to focus on the positive, and I do—I would like to see, if these models are getting so good at math and physics, I would like to see what they can actually do with that in the next few years. Feltman: Well, thanks so much for coming on to chat. I hope we can have you back again soon to talk more about AI. If you have any questions for Deni about AI or other big issues in tech, let us know at ScienceQuickly@sciam.com. We'll be back on Monday with our weekly science news roundup. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Deni Ellis Béchard is Scientific American's senior tech reporter. He holds two master's degrees in literature, as well as a master's degree in biology from Harvard University. His most recent novel, We Are Dreams in the Eternal Machine, explores the ways that artificial intelligence could transform humanity. You can follow him on X, Instagram and Bluesky @denibechard. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250801020106.htm'>Unwanted pregnancies surge with alcohol, but not with cannabis, study finds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-01 07:33:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new study has found that, among women with a high desire to avoid becoming pregnant, those who drank heavily had a 50% higher risk of becoming pregnant than those who drank moderately or not at all. In contrast, participants who used cannabis were no more likely to have an undesired pregnancy than participants who did not use cannabis. In contrast, less than half of the 71 undesired pregnancies (28) occurred among people who used cannabis, meaning that those who used cannabis did not show an elevated risk of undesired pregnancy compared with people who did not use cannabis. First, non-pregnant women who drink heavily appear, on average, to have a higher desire to avoid pregnancy than those who drink moderately or not at all. Finding out why those pregnancies happen is the next step in our research." Materials provided by Society for the Study of Addiction. Note: Content may be edited for style and length. MIT Just Proved Einstein Wrong in the Most Famous Quantum Experiment Study Finds One Workout Can Cut Cancer Cell Growth by 30% Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            