
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03955-6'>Physical activity as a modifiable risk factor in preclinical Alzheimer's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 16:05:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Physical inactivity is a recognized modifiable risk factor for Alzheimer's disease (AD), yet its relationship with progression of AD pathology in humans remains unclear, limiting the effective translation into prevention trials. Using pedometer-measured step counts in cognitively unimpaired older adults, we demonstrated an association between higher physical activity and slower cognitive and functional decline in individuals with elevated baseline amyloid. Importantly, this beneficial association was not related to lower amyloid burden at baseline or longitudinally. Instead, higher physical activity was associated with slower amyloid-related inferior temporal tau accumulation, which significantly mediated the association with slower cognitive decline. Dose–response analyses further revealed a curvilinear relationship, where the associations with slower tau accumulation and cognitive decline reached a plateau at a moderate level of physical activity (5,001–7,500 steps per day), potentially offering a more approachable goal for older sedentary individuals. Collectively, our findings support targeting physical inactivity as an intervention to modify the trajectory of preclinical AD in future prevention trials, and further suggest that preferentially enrolling sedentary individuals with elevated amyloid may maximize the likelihood of demonstrating a protective effect of physical activity on tau accumulation and cognitive and functional decline in early AD. It is estimated that nearly half of AD cases worldwide are attributable to modifiable risk factors1,2,3. Physical inactivity, in particular, plays a prominent role in the USA and Europe1,2. Although animal studies support a benefit of physical activity on AD progression4,5, important knowledge gaps remain in human literature, limiting the effective translation into AD prevention trials. Most existing observational studies rely on self-reported physical activity6,7,8,9,10,11,12, which is prone to recall bias and misreporting, especially in populations with or at risk for cognitive impairment. In addition, most studies focused on the clinical syndrome of AD7,9,12,13,14, with few directly examining core AD biomarkers15,16,17,18,19,20,21, and even fewer have done so longitudinally22,23. A clearer understanding of how objectively measured physical activity is associated with the progression of AD neuropathology is essential, particularly during the preclinical period, when there is likely the greatest potential to modify disease trajectory. With the increasing popularity of digital wearables, daily step count has become an easily accessible and understood measure of physical activity. Emerging literature suggests that higher step counts are linked to lower all-cause mortality24,25. Our previous study demonstrated that higher step counts in cognitively unimpaired (CU) older adults with elevated baseline beta-amyloid (Aβ) burden were associated with slower prospective cognitive decline26, supporting a potential protective role of physical activity in the preclinical stage of AD. However, it remains unknown whether this protective association with cognition is mediated by differences in AD neuropathology burden, or what levels of physical activity are associated with slower cognitive decline. The current study addressed these questions by leveraging an expanded Harvard Aging Brain Study (HABS) cohort of CU older individuals with pedometer-measured physical activity, longitudinal Aβ and tau positron emission tomography (PET) data, and annual cognitive assessments up to 14 years. We examined whether physical activity is associated with slower cognitive and functional decline through different rates of Aβ and tau accumulation. We further examined the dose–response associations with physical activity levels to help inform future AD prevention trials and public health policies. The current study examined 296 participants from HABS who were CU at baseline and followed longitudinally27. Cognition and function were assessed annually using the Preclinical Alzheimer's Cognitive Composite-5 (PACC5) and Clinical Dementia Rating28 Sum of Boxes (CDR-SOB) scores, respectively. Global Aβ and inferior temporal cortex (ITC) tau burdens were measured longitudinally in a subset of participants (Aβ n = 241; tau n = 172), with participants undergoing their first tau scan at 2.2 ± 1.5 years after baseline because the technique was introduced mid-study. Table 1 summarizes the participant characteristics and longitudinal follow-up. The unadjusted longitudinal trajectories for global Aβ, ITC tau, PACC5 and CDR-SOB are shown in Extended Data Fig. For illustration purposes only, to more clearly visualize the individual trajectories, the data were plotted according to high versus low baseline Aβ burden in columns and physical activity (pedometer-measured mean steps per day) in rows, defined by above and below the median values. We first examined the associations between physical activity and baseline demographics, imaging and cognitive variables. There was a negative association between physical activity and age (Spearman's r = −0.27, P < 0.001). Males had greater physical activity than females (Sex [males compared to females]: β = 927 [95% confidence intervals 247 to 1,607], P = 0.008). There were no significant associations between physical activity and education (rpartial = 0.08, P = 0.16), or baseline Aβ burden (rpartial = −0.004, P = 0.95), adjusting for age and sex. There was a significant interaction between lower baseline physical activity and elevated Aβ on greater initial ITC tau burden, adjusting for age, sex, years of education and time interval between study baseline and first tau scan (Physical activity × Aβ: β = −0.19 [−0.30 to −0.08], P = 0.001) (Extended Data Fig. There was no additional independent association between baseline physical activity and initial ITC tau burden (Physical activity: β = −0.06 [−0.18 to 0.05], P = 0.28). Importantly, there were no interactive or independent associations of initial Aβ and tau burden with baseline physical activity, adjusting for the same covariates (Aβ × ITC tau: β = −0.04 [−0.15 to 0.07], P = 0.44; Aβ: β = 0.10 [−0.07 to 0.28], P = 0.25; ITC tau: β = 0.01 [−0.21 to 0.22], P = 0.96) (Extended Data Fig. There were also no significant interactive (Physical activity × Aβ) or independent effects of physical activity on baseline PACC5 (Physical activity × Aβ: β = 0.003 [−0.07 to 0.08], P = 0.93; Physical activity: β = 0.003 [−0.07 to 0.08], P = 0.92). Baseline associations with CDR-SOB were not examined because participants were clinically unimpaired at baseline. For primary analyses, we examined the effects of baseline physical activity on longitudinal imaging and cognitive measures. Using a linear mixed effects model, we found no association between baseline physical activity and longitudinal Aβ burden (Physical activity × Time: β = −0.0006 [−0.01 to 0.01], P = 0.92) (Fig. a, Linear mixed effects model revealed no association between baseline physical activity and longitudinal Aβ burden (β = −0.0006 [−0.01 to 0.01], P = 0.92; n = 241). b–d, By contrast, there were significant interactions between baseline physical activity and Aβ burden on longitudinal ITC tau burden (b), longitudinal cognition measured with PACC5 (c) and longitudinal functional decline measured with CDR-SOB scores (d). Individuals with high baseline physical activity and elevated Aβ (solid red line) showed slower ITC tau accumulation (β = −0.13 [−0.19 to −0.06], P < 0.001; n = 172) (b), slower PACC5 decline (β = 0.10 [0.05 to 0.16], P < 0.001; n = 296) (c) and slower CDR-SOB progression (β = −0.14 [−0.22 to −0.05], P = 0.001; n = 296) (d). Statistical significance was assessed using two-tailed t-tests, with P < 0.05 considered statistically significant without adjustment for multiple comparisons. Baseline physical activity (mean steps per day) and Aβ burden were modeled as continuous variables. To visualize the model results, the estimated trajectories based on representative levels of low versus high baseline physical activity and (for tau, PACC5 and CDR-SOB models) low versus high baseline Aβ burden are presented, with error bands representing 95% confidence intervals for the estimated trajectories. relative to the mean (low, 2,800 steps per day; high, 8,700 steps per day). Low and high Aβ are represented, for illustration purposes, by the mean Aβ burden of Aβ-negative (PiB PVC-DVR = 1.17) and Aβ-positive (PiB PVC-DVR = 1.85) participants, respectively. The numbers of participants contributing longitudinal data to each 2.5-year segment for the respective statistical models are summarized in Extended Data Table 5. We then examined the interactive associations between baseline physical activity and Aβ burden on longitudinal cognition and function. Baseline Aβ was modeled as a continuous variable to leverage the full dynamic range of global Aβ burden29. Consistent with our earlier work26, this larger HABS sample with extended follow-up showed a significant physical activity by Aβ interaction on longitudinal PACC5, whereby higher physical activity was associated with slower Aβ-related PACC5 decline (Physical activity × Aβ × Time: β = 0.10 [0.05 to 0.16], P < 0.001; Fig. In addition, there was an independent association between higher physical activity and slower PACC5 decline (Physical activity × Time: β = 0.06 [0.004 to 0.12], P = 0.04). We further extended our previous findings and identified a significant physical activity by Aβ interaction on longitudinal CDR-SOB, whereby higher physical activity was associated with slower Aβ-related functional decline (Physical activity × Aβ × Time: β = −0.14 [−0.22 to -0.05], P = 0.001) (Fig. There was no additional main effect of physical activity on CDR-SOB progression (Physical activity × Time: β = −0.05 [−0.13 to 0.04], P = 0.29). We then addressed the question of whether the associations between physical activity and cognitive and functional decline were mediated by differences in tau pathology. We found a concordant interaction, whereby higher baseline physical activity was associated with attenuated Aβ-related ITC tau accumulation (Physical activity × Aβ × Time: β = −0.13 [−0.19 to −0.06], P < 0.001) (Fig. There was additionally an independent association between higher physical activity and slower tau accumulation (Physical activity × Time: β = −0.07 [−0.13 to −0.003], P = 0.04). Moderated mediation analysis demonstrated that slower ITC tau accumulation fully mediated the association between higher physical activity and slower PACC5 decline in individuals with elevated baseline Aβ (mediated effect: β = 0.59 [0.32 to 0.91], P < 0.001; direct effect: β = 0.11 [−0.30 to 0.52], P = 0.61; 84% mediated) (Fig. For CDR-SOB, slower ITC tau accumulation partially mediated the association between higher physical activity and slower CDR-SOB progression in individuals with elevated baseline Aβ (β = −0.43 [−0.71 to −0.20], P < 0.001; 40% mediated) (Fig. There remained a significant direct effect of physical activity on functional decline (β = −0.64 [−1.16 to −0.13], P = 0.01). In individuals with low baseline Aβ, there were no significant total or mediated effects of physical activity on PACC5 decline or CDR-SOB progression (P > 0.75). a,b, Individual slopes for ITC tau, PACC5 (a) and CDR-SOB (b) were extracted from linear mixed effects models for moderated mediation analyses (n = 172; in participants who have both longitudinal tau and cognitive data). We modeled physical activity (mean steps per day) as predictor, ITC tau slope as mediator and PACC5 or CDR-SOB slopes as outcome. Both physical activity and Aβ burden were modeled as continuous variables in the mediation models. For the moderation analyses, low and high levels of Aβ burden were represented by the mean Aβ burden of Aβ-negative (PiB PVC-DVR = 1.17) and Aβ-positive (PiB PVC-DVR = 1.85) participants, respectively. Statistical testing was performed using a quasi-Bayesian Monte Carlo method based on 10,000 simulations to generate the estimates and 95% confidence intervals, with two-tailed P < 0.05 considered statistically significant without adjustment for multiple comparisons. Results demonstrated that at an elevated level of baseline Aβ burden, slower ITC tau accumulation fully mediated the association between higher physical activity and slower PACC5 decline (β = 0.59 [0.32 to 0.91], P < 0.001, 84% mediated) (a) and partially mediated the association between higher physical activity and slower CDR-SOB progression (β = −0.43 [−0.71 to −0.20], P < 0.001, 40% mediated) (b). In individuals with low baseline Aβ burden, there were no significant total or mediated effects of physical activity on PACC5 decline (total effect: β = 0.03 [−0.27 to 0.31], P = 0.85; mediated effect: β = −0.002 [−0.16 to 0.15], P = 0.98) or CDR-SOB progression (total effect: β = 0.05 [−0.28 to 0.37], P = 0.75; mediated effect: β = 0.001 [−0.11 to 0.12], P = 0.98). The interaction between baseline physical activity and Aβ on longitudinal tau, PACC5 and CDR-SOB remained significant in sensitivity analyses assessing for possible confounds (Table 2): (1) using a more stringent physical activity cutoff excluding days that registered fewer than 1,000 steps; (2) using Aβ and tau PET data without partial volume correction (PVC); (3) including only participants age <70 years to minimize potential confounding effects of advanced age (the negative association between physical activity and age was absent in this age <70 subgroup; Spearman's r = −0.01, P = 0.93); (4) adjusting for the season of physical activity measurement, self-reported physical activity, systemic vascular risk and depressive symptoms; (5) excluding participants who developed mild cognitive impairment or dropped out of the study within the first 2 years of follow-up; and (6) adjusting for baseline tau, PACC or CDR-SOB in their respective models to account for physical activity effects on baseline outcome measures. Lastly, we examined the dose–response associations between levels of physical activity and longitudinal tau accumulation, cognitive and functional decline. Ordinal levels of physical activity were defined using cutoffs adapted from ref. 30, which were modified to better suit the lower activity levels in older adults and to ensure an adequate number of participants in each subgroup (Extended Data Table 1): ‘inactive' (≤3,000 per day), ‘low activity' (3,001–5,000 steps per day), ‘moderate activity' (5,001–7,500 steps per day) and ‘active' (≥7,501 steps per day). Participant characteristics across physical activity levels are summarized in Extended Data Table 2. Higher physical activity levels were associated with younger age, but baseline demographics, imaging and cognitive variables were not significantly different across physical activity groups adjusting for age and sex. In linear mixed effects models using the inactive subgroup as reference, all higher levels of physical activity were associated with slower Aβ-related ITC tau accumulation, PACC5 decline and CDR-SOB progression, except that the slower CDR-SOB progression in the low activity group did not reach statistical significance (Table 3). To better visualize the dose–response relationships, we modeled the physical activity level by Aβ interaction on extracted ITC tau, PACC5 and CDR-SOB slopes (Fig. In individuals with elevated baseline Aβ, even low levels of physical activity (3,001–5,000 steps per day) were associated with substantially slower rates of tau accumulation, cognitive and functional decline. There were further attenuations of tau accumulation and cognitive and functional decline at moderate activity (5,001–7,500 steps per day), with similar rates in the active group (≥7,501 steps per day), suggesting a plateauing of effects. a–c, Using extracted slopes for ITC tau (n = 172) (a), PACC5 (b) and CDR-SOB (c) (PACC5 and CDR-SOB, n = 296), we examined the interactive effects of baseline physical activity level (ordinal) and Aβ burden (continuous) using linear regression models. The number of individuals in each physical activity subgroup included in the tau, PACC5 and CDR-SOB analyses are summarized in Extended Data Table 1. Aβ burden was modeled as a continuous variable. For illustration purposes, low and high Aβ are represented by the mean Aβ burden of Aβ-negative (PiB PVC-DVR = 1.17) and Aβ-positive (PiB PVC-DVR = 1.85) participants, respectively. The error bars represent the 95% confidence intervals for the estimated effects of physical activity levels on tau and cognitive slopes at representative levels of low and high Aβ burden. Results demonstrate that in individuals with elevated baseline Aβ, even low levels of physical activity (3,001–5,000 steps) were associated with substantially slower rates of tau accumulation and cognitive decline compared to inactive individuals. There were further attenuations of tau accumulation and cognitive and functional decline at moderate activity (5,001–7,500 steps per day), with similar rates in the active group (≥7,501 steps per day). To help contextualize the associations between physical activity levels and cognitive and functional decline, we used the linear mixed effects models from Table 3 to estimate the longitudinal PACC5 and CDR-SOB trajectories at representative levels of low versus high baseline Aβ burden (represented by the mean global Aβ burden of Aβ-negative and Aβ-positive participants, respectively) across physical activity levels (Extended Data Fig. For individuals with elevated Aβ, from baseline to the median cognitive follow-up of 9 years, estimated PACC5 z-scores declined by 2.5 (inactive), 1.5 (low activity), 1.1 (moderate activity) and 1.2 (active) points, respectively. Compared to inactive individuals, cognitive decline was lower by 40%, 54% and 51% across increasing physical activity levels. Over the same period, estimated CDR-SOB scores increased by 2.2 (inactive), 1.4 (low activity), 1.2 (moderate activity) and 1.1 (active) points in individuals with elevated Aβ. Compared to inactivity, there were 34%, 45% and 51% slower functional decline across increasing physical activity levels. Using −1.5 PACC5 z-score as a threshold for impaired cognition31, CU individuals with elevated Aβ were estimated to reach this threshold for cognitive worsening at 6.5 (inactive), 9.6 (low activity), 13.6 (moderate activity) and 12.7 (active) years from baseline, respectively. Similarly, using a CDR-SOB increase of 1.5 as a threshold for clinically meaningful functional decline in early AD32, individuals with elevated Aβ were estimated to reach this threshold at 7.1 (inactive), 10.2 (low activity), 11.9 (moderate activity) and 13.6 (active) years from baseline, respectively. In an expanded cohort of 296 CU older adults with pedometer-measured physical activity, baseline Aβ PET and up to 14 years of annual cognitive follow-up, we replicated our previous finding of a protective association between higher physical activity and slower Aβ-related cognitive decline, and further demonstrated a similar relationship with functional decline. Importantly, we demonstrated that these associations with cognition and function were not related to differences in cross-sectional or longitudinal Aβ pathology. Instead, in a subset of 172 individuals with longitudinal tau PET, we found a new and concordant association between higher physical activity and slower Aβ-related early neocortical tau accumulation, which significantly mediated the relationships with slower cognitive and functional decline. Our dose–response analyses further suggest that the largest incremental differences between increasing physical activity levels were observed in the most sedentary individuals, with plateauing associations with both tau and cognition at a moderate level of activity (5,001–7,500 steps per day), potentially offering a more approachable physical activity goal for older sedentary individuals. Collectively, our results support targeting physical inactivity, either alone or in combination with anti-Aβ therapy, as an intervention in future prevention trials to slow the progression of tau pathology and delay the onset of cognitive and functional decline in preclinical AD. Many observational studies have demonstrated associations between physical activity and reduced risks of cognitive impairment and AD. However, most existing literature relied on self-reported physical activity, which is susceptible to recall bias or misreporting, especially among populations with or at risk for cognitive impairment. A few longitudinal studies have examined the effects of objectively measured physical activity (by pedometer or accelerometer) in older adults without dementia and found associations with reduced risks of cognitive decline, AD or all-cause dementia13,33,34,35. Our previous study26 extended these findings to CU individuals and demonstrated an association between higher step counts and slower prospective cognitive decline in preclinical AD. The current study replicated this finding using a considerably larger cohort, including younger individuals in late midlife, and over a substantially longer duration of follow-up (median of 9.3 years instead of 6 years). The latter is notable given that some observational studies found diminished benefits of physical activity with longer follow-up time36,37,38. We further expanded our cognitive findings and demonstrated a similar association with slower Aβ-related functional decline measured by CDR-SOB, which has been frequently used as an outcome measure in AD clinical trials39,40. There is growing interest in understanding whether physical activity associations with reduced cognitive decline are mediated by differences in AD neuropathology, with most studies focusing on Aβ. Although animal models suggest that physical exercise may reduce Aβ burden4,41, evidence in human studies has been mixed. A few cross-sectional studies found associations between higher self-reported physical activity and lower Aβ pathology15,16,17,18,19,20. Other large CU cohort studies have found no associations between self-reported physical activity and cross-sectional or longitudinal Aβ PET burden21,22,23. Consistent with the latter, our study found no associations between pedometer-measured physical activity and baseline or longitudinal Aβ burden, suggesting that our findings were unlikely to be driven by different rates of Aβ accumulation. By contrast, we demonstrated a new relationship between physical activity and longitudinal tau burden, where higher step counts were associated with slower early neocortical tau accumulation in individuals with elevated baseline Aβ. We further demonstrated that reduced tau accumulation significantly mediated the associations between higher physical activity and slower cognitive decline in preclinical AD. These observational findings support a potential role for physical activity in modifying the AD pathological cascade, which remains to be confirmed in future randomized clinical trials. Interestingly, although tau change fully mediated the effects of PACC5 decline (84% mediated), it only partially mediated (40%) the effects on CDR-SOB progression, with a significant remaining direct effect. This suggests that a substantial portion of the physical activity association with slower functional decline was mediated by mechanisms besides tau-related injury, which should be further examined in future studies (for example, physical frailty and functional reserve). A small number of prior studies have investigated the cross-sectional associations between physical activity and tau. A few have found associations between higher physical activity and lower cerebrospinal fluid phosphorylated tau or phosphorylated tau to Aβ42 ratios19,20,42,43. Others have examined the cross-sectional relationships between self-reported physical activity and tau PET with mixed results. Two studies found that higher self-reported physical activity was associated with lower cross-sectional tau burden44,45. However, a larger study of CU individuals found no cross-sectional associations between self-reported physical activity and regional tau burden and no significant mediation of physical activity effects on cross-sectional cognition by tau21. It is likely that our use of objectively measured physical activity and, more importantly, our longitudinal design increased our sensitivity to detect a relationship between physical activity and early aggregated tau pathology in preclinical AD. Although our rich longitudinal PET and cognitive data are a substantial strength, this study remains observational, and we are unable to rule out potential reverse causality (that is, individuals with prodromal AD may have reduced physical activity from their underlying disease). However, several factors reduce this likelihood. First, there were no associations between physical activity (independently or interactively with Aβ) and baseline cognition. Although there was a physical activity and Aβ interaction on baseline tau burden, a reversed relationship was not observed—higher cross-sectional Aβ or tau burden (individually or interactively) was not associated with lower baseline physical activity. In addition, the association between higher physical activity and slower longitudinal ITC tau accumulation remained significant after adjusting for the effects of lower initial tau burden. Our findings were further unchanged in sensitivity analyses that excluded participants who developed mild cognitive impairment (or dropped out) within the first 2 years of follow-up, reducing the likelihood that our findings were driven by individuals with impending cognitive and/or clinical changes. Although these observations are reassuring, it is important to emphasize that randomized clinical trials are ultimately needed to establish causal relationships between physical activity and preclinical AD progression. There is substantial public interest in understanding how much physical activity is needed to improve health. Although this question cannot be directly answered by observational studies, we examined the dose–response associations between physical activity levels and progression of tau and cognitive decline, which may help inform the design of future AD prevention trials. Our results suggest that even a modest increase in physical activity may be associated with attenuated tau accumulation and cognitive decline in sedentary individuals on a preclinical AD trajectory: compared to inactive individuals, low levels of physical activity were associated with 34% to 40% slower cognitive and functional decline over 9 years (the median duration of cognitive follow-up). Notably, the associations with more favorable tau and cognitive trajectories reached a plateau by moderate levels of physical activity (5,001–7,500 steps per day), which may be a less daunting goal for sedentary older adults than the popular goal of 10,000 steps per day commonly referenced in lay media. Several studies have demonstrated a similar curvilinear relationship between step count and all-cause mortality or dementia24,25,34, with benefits plateauing around 6,000–8,000 steps per day for older adults and around 8,000–10,000 steps per day for younger populations, suggesting a ceiling effect. Specifically for the current study, our findings suggest that there may be a limit to how much increasing physical activity may dampen the detrimental effects of Aβ on tau and cognitive and functional decline, and support designing clinical trials that combine lifestyle interventions with anti-Aβ therapy to more effectively modify the trajectory of preclinical AD. Future studies are needed to understand the mechanisms underlying the protective effects of physical activity on Aβ-related tau pathology and cognitive decline. Although adjusting for composite vascular risk did not impact our findings, better vascular fitness remains a likely potential mechanism. Greater physical activity has been linked to better cardiopulmonary fitness46,47, which has, in turn, been shown to dampen the negative association between Aβ and cognition48. In addition, greater accelerometer physical activity in CU older adults has been associated with greater cerebral blood flow49, which has been shown to be impaired early in the AD cascade50. Lastly, greater physical activity and/or exercise have also been shown in animal models and humans to decrease inflammation and upregulate VEGF-A, BDNF and Irisin51,52,53,54,55, which are candidate pathways with the potential to modulate Aβ effects on tau and cognition56,57,58,59,60. The current study has several limitations. As discussed above in greater detail, this is an observational study and we are unable to fully rule out the potential influence of reverse causality. Future randomized clinical trials are required to establish causal relationships. Although our pedometer-measured physical activity is a strength, it was only assessed at baseline and did not classify the duration, intensity or history of physical activity, nor capture nonstep-related physical activity (for example, swimming, resistance training). Future studies with longitudinal physical activity measurements (for example, with actigraphy) and life course physical activity data are needed to better delineate temporal trends. The HABS cohort consists of highly educated, predominantly non-Hispanic white individuals and excluded those with symptomatic cerebrovascular disease and poorly controlled vascular risk factors at baseline, which may limit the generalizability of our findings to other populations. In summary, leveraging a deeply phenotyped CU cohort with objectively measured physical activity and longitudinal neuroimaging and cognitive assessments, we demonstrated an association between higher physical activity and attenuated accumulation of early neocortical tau pathology, which significantly mediated the relationship with slower cognitive and functional decline in preclinical AD. Our dose–response analyses further suggest that the incremental differences in associations between higher physical activity levels and slower tau and cognitive changes were greatest for the most sedentary individuals, and appeared to reach a plateau by moderate levels of physical activity (5,001–7,500 steps per day). Taken together, our findings support targeting physical inactivity as a strategy in future randomized clinical trials to modify the trajectory of tau and cognition in preclinical AD, and potentially provide an easily understood and more attainable physical activity goal for older sedentary individuals at high risk of cognitive decline. Our results further suggest that AD prevention trials using activity-based interventions may benefit from preferentially enrolling sedentary individuals with elevated Aβ levels, longer trial durations, and consideration of examining biomarker endpoint with tau PET, because these approaches may maximize the likelihood of demonstrating a protective effect of physical activity on early AD progression. The current study included 296 participants from HABS27, a community cohort of individuals aged 50 to 90 who were CU at baseline (global CDR28 of 0, education-adjusted Mini-Mental State Examination61 score of 27 or greater, and Logical Memory IIa Delayed Recall performance in the normal range62). We examined all eligible participants with baseline physical activity measurement, Aβ PET imaging and at least two longitudinal cognitive assessments. 5 illustrates that of 348 eligible HABS participants at baseline, 50 were excluded from the current study because of missing data. Importantly, the included group did not differ in demographics or baseline cognition (Extended Data Table 3). Baseline characteristics by self-reported sex are summarized in Extended Data Table 4. Additional subsets of participants were used in the longitudinal Aβ PET (n = 241) and tau PET (n = 172) analyses (Table 1). Data were collected from April 2010 to February 2025. The Mass General Brigham Institutional Review Board approved the HABS protocol and procedures, and all participants signed a written informed consent before the completion of any study procedures. Baseline physical activity was measured using a waistband-mounted pedometer (HJ-720ITC; Omron Healthcare), which has been shown to accurately measure step counts63. Participants were asked to wear the pedometer for seven consecutive days during waking hours. Using previously published cutoffs for pedometer data quality, days that registered <100 or >30,000 steps were excluded26,64. We included participants with at least 4 days of recorded activity within these cutoffs64. Mean steps per day were used as the primary measure of physical activity, which was square-root transformed to account for skewness. Detailed methods have been published previously65. Briefly, global Aβ burden was measured using PiB PET as a DVR across a composite of frontal, lateral temporal and parietal, and retrosplenial regional uptake defined using FreeSurfer (v.6.0). 18F-Flortaucipir PET was introduced mid-study to measure tau, with participants undergoing their first 18F-flortaucipir PET at 2.2 ± 1.5 years after baseline. Tau was measured as standardized uptake value ratio in the ITC, a prominent site of early neocortical tau accumulation in preclinical AD associated with emerging cognitive impairment65. Both Aβ and tau PET used cerebellar gray matter as reference and the geometric transfer matrix method for PVC66. Sensitivity analyses were performed using PET data without PVC. Cognition was assessed using the PACC5 (ref. 67), a composite z-score that includes the Mini-Mental State Examination61, Wechsler Adult Intelligence Scale-Revised Digit Symbol Coding68, Wechsler Memory Scale-Revised Logical Memory delayed recall62, Free and Cued Selective Reminding Test (free recall plus total recall)69 and Category Fluency Test70. Functional decline was assessed using the CDR-SOB score. Continuous variables were z-transformed before analysis to obtain standardized effect sizes, with the exception of PACC5 (already a z-score). For illustrative purposes, nonstandardized variables were used when plotting model results to enhance interpretability (‘ggplot2' and ‘plot_model' packages). A two-tailed P < 0.05 was considered statistically significant. We first examined the associations between baseline physical activity with age and sex. We also examined the associations between physical activity and baseline imaging and cognitive variables, adjusting for age and sex. For primary analyses, we used linear mixed effects model (‘nlme' package) to examine: (1) the effects of physical activity on longitudinal Aβ; and (2) the interactive effects of physical activity and baseline Aβ on longitudinal ITC tau, PACC5 and CDR-SOB, respectively. Baseline Aβ was modeled as a continuous variable to leverage the full dynamic range of Aβ burden, which has been shown to capture additional information on preclinical AD progression over a dichotomous approach and avoids variability introduced by positivity threshold selection, which remains an evolving area of research29. Time was operationalized as years from baseline assessment. We adjusted for age, sex, APOE ε4 carrier status, education, their interactions with time, and included random intercepts and slopes. There were no significant APOE ε4 effects on longitudinal tau, PACC5 or CDR-SOB; therefore, APOE was removed from these models to include three participants with missing genotype data. We conducted the following sensitivity analyses to examine the robustness of our primary findings: (1) using a more stringent physical activity cutoff excluding days that registered <1,000 steps; (2) using PET data without PVC; (3) including only participants age <70 years to minimize potential confounding effects of advanced age; (4) adjusting for additional potential confounders and their interactions with time—season of physical activity measurement, total self-reported physical activity (CHAMPS questionnaire71), systemic vascular risk (office-based Framingham Heart Study cardiovascular disease risk score72) and depressive symptoms (geriatric depression scale); (5) excluding participants who developed mild cognitive impairment (n = 5) or dropped out (n = 11) within the first 2 years of follow-up to reduce the potential for reverse causality; and (6) adjusting for baseline tau, PACC or CDR-SOB in their respective models to account for physical activity effects on baseline outcome measures. We further used moderated mediation analysis (‘mediation' package73) to examine whether tau accumulation mediated the interactive effects of physical activity and Aβ on PACC5 decline or CDR-SOB progression. Individual slopes of ITC tau, PACC5 and CDR-SOB change were extracted from unadjusted linear mixed models with time as the only predictor, including both random slopes and intercepts. We conducted moderated mediation analysis with PACC5 or CDR-SOB slope as outcome, ITC tau slope as mediator, baseline Aβ (continuous) as moderator, and adjusted for age, sex, education and time interval between baseline Aβ and first tau scan. To examine for moderation effects by baseline Aβ, mediation models were run at both low and high levels of baseline Aβ, which were represented by the mean Aβ burden of Aβ-negative (PiB PVC-DVR = 1.17) and Aβ-positive (PiB PVC-DVR = 1.85) participants respectively, defined using the conventional Aβ-positivity threshold (PiB PVC-DVR = 1.324). Statistical testing was performed using a quasi-Bayesian Monte Carlo method based on 10,000 simulations to generate the estimates and 95% confidence intervals. Lastly, we examined the dose–response relationships between the levels of physical activity and associations with Aβ-related tau accumulation and cognitive decline. Ordinal levels of physical activity were defined using cutoffs adapted from ref. 30: inactive (≤3,000 steps per day), low activity (3,001 to 5,000 steps per day), moderate activity (5,001 to 7,500 steps per day) and active (≥7,501 steps per day). Using linear mixed effects models, we examined the interactive effects of baseline physical activity levels (ordinal) and Aβ (continuous) on longitudinal ITC tau, PACC5 and CDR-SOB, adjusting for the same covariates. To contextualize the effects of physical activity on cognitive decline, we used the model results to estimate predicted PACC5 and CDR-SOB trajectories across physical activity levels at representative levels of low versus high baseline Aβ burden, which were represented by the mean global Aβ burden of Aβ-negative (PiB PVC-DVR = 1.17) and Aβ-positive participants (PiB PVC-DVR = 1.85) respectively as described above. We calculated the predicted changes in PACC5 and CDR-SOB scores from baseline to year 9 (median duration of cognitive follow-up) and calculated the percent difference in decline across higher activity levels compared to inactive individuals. Lastly, we used literature-derived thresholds for cognitive impairment (lower than −1.5 PACC5 z-score31) and clinically meaningful functional decline in early AD (increase of 1.5 points on CDR-SOB32) to estimate the time from baseline to these thresholds of cognitive and functional worsening for individuals with elevated Aβ across physical activity levels. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Deidentified data from HABS can be requested online (https://www.synapse.org/habs). Details of the request process have been published previously27 and include the completion of an online data request form and acceptance of the terms of a data use agreement. The approval process is primarily designed to ensure that the proposed purpose of the data request is consistent with the data use agreement and would not pose a risk to the HABS participants. The most important restriction of use is to not attempt to re-identify the participants in any way. Other requirements include agreement to abide by human subject research policies, only using the data for the project detailed in the data request (separate requests can be made for additional projects), no commercialization of the data, and no marketing or fundraising with the data. All statistical analyses and raw figures were generated using R (v.4.3.1), with the following open source packages: car (v.3.1-2), dplyr (v.1.1.2), ggplot2 (v.3.5.1), mediation (v.4.5.0), nlme (v.3.1-162), ppcor (v.1.1), sjPlot (v.2.8.15). No custom code or packages were used in the statistical analyses. Norton, S., Matthews, F. E., Barnes, D. E., Yaffe, K. & Brayne, C. Potential for primary prevention of Alzheimer's disease: an analysis of population-based data. Nianogo, R. A. et al. Risk factors associated with Alzheimer disease and related dementias by sex and race and ethnicity in the US. Livingston, G. et al. Dementia prevention, intervention, and care: 2024 report of the Lancet standing Commission. Moore, K. M. et al. A spectrum of exercise training reduces soluble Aβ in a dose-dependent manner in a mouse model of Alzheimer's disease. Long-term exercise pre-training attenuates Alzheimer's disease-related pathology in a transgenic rat model of Alzheimer's disease. Yaffe, K., Barnes, D., Nevitt, M., Lui, L.-Y. & Covinsky, K. A prospective study of physical activity and cognitive decline in elderly women: women who walk. Physical activity, diet, and risk of Alzheimer disease. The effect of midlife physical activity on cognitive function among older adults: AGES—Reykjavik Study. Leisure-time physical activity from mid- to late life, body mass index, and risk of dementia. Physical activity, brain volume, and dementia risk: the Framingham Study. Physical activity, cognitive decline, and risk of dementia: 28 year follow-up of Whitehall II cohort study. Ogino, E., Manly, J. J., Schupf, N., Mayeux, R. & Gu, Y. Current and past leisure time physical activity in relation to risk of Alzheimer's disease in older adults. Buchman, A. S. et al. Total daily physical activity and the risk of AD and cognitive decline in older adults. Najar, J. et al. Cognitive and physical activity and dementia: a 44-year longitudinal population study of women. Exercise and Alzheimer's disease biomarkers in cognitively normal older adults. Exercise engagement as a moderator of the effects of APOE genotype on amyloid deposition. Physical activity and amyloid-β plasma and brain levels: results from the Australian Imaging, Biomarkers and Lifestyle Study of Ageing. Relationship between physical activity, cognition, and Alzheimer pathology in autosomal dominant Alzheimer's disease. Associations of healthy lifestyles with cerebrospinal fluid biomarkers of Alzheimer's disease pathology in cognitively intact older adults: the CABLE study. Associations of physical activity with Alzheimer's disease pathologies and cognition: the CABLE study. Protective effects of sleep duration and physical activity on cognitive performance are influenced by β-amyloid and brain volume but not tau burden among cognitively unimpaired older adults. Pedrero-Chamizo, R. et al. Alzheimer's disease prevention: apolipoprotein e4 moderates the effect of physical activity on brain beta-amyloid deposition in healthy older adults. Physical activity and brain amyloid beta: a longitudinal analysis of cognitively unimpaired older adults. Lee, I.-M. et al. Association of step volume and intensity with all-cause mortality in older women. Paluch, A. E. et al. Daily steps and all-cause mortality: a meta-analysis of 15 international cohorts. Associations of physical activity and β-amyloid with longitudinal cognition and neurodegeneration in clinically normal older adults. Dagley, A. et al. Harvard Aging Brain Study: dataset and accessibility. Morris, J. C. The Clinical Dementia Rating (CDR): current version and scoring rules. Farrell, M. E. et al. Association of longitudinal cognitive decline with amyloid burden in middle-aged and older adults: evidence for a dose–response relationship. Tudor-Locke, C., Johnson, W. D. & Katzmarzyk, P. T. Accelerometer-determined steps per day in US adults. Papp, K. V. et al. Clinical meaningfulness of subtle cognitive decline on longitudinal testing in preclinical AD. Lansdall, C. J. et al. Care partner‐informed meaningful change thresholds for the Clinical Dementia Rating‐Sum of Boxes for trials of early Alzheimer's disease. Memel, M., Buchman, A. S., Bennett, D. A. & Casaletto, K. Relationship between objectively measured physical activity on neuropathology and cognitive outcomes in older adults: resistance versus resilience? del Pozo Cruz, B., Ahmadi, M., Naismith, S. L. & Stamatakis, E. Association of daily step count and intensity with incident dementia in 78430 adults living in the UK. Nguyen, S. et al. Accelerometer‐measured physical activity and sitting with incident mild cognitive impairment or probable dementia among older women. The association between physical activity and dementia in an elderly population: the Rotterdam Study. Body mass index, diet, physical inactivity, and the incidence of dementia in 1 million UK women. Physical inactivity, cardiometabolic disease, and risk of dementia: an individual-participant meta-analysis. et al. Lecanemab in early Alzheimer's disease. Sims, J. R. et al. Donanemab in early symptomatic Alzheimer disease. Adlard, P. A., Perreau, V. M., Pop, V. & Cotman, C. W. Voluntary exercise decreases amyloid load in a transgenic model of Alzheimer's disease. Moderate intensity physical activity associates with CSF biomarkers in a cohort at risk for Alzheimer's disease. Roccati, E. et al. Modifiable dementia risk factors and AT(N) biomarkers: findings from the EPAD cohort. Brown, B. M. et al. Self-reported physical activity is associated with tau burden measured by positron emission tomography. Genetically identical twins show comparable tau PET load and spatial distribution. Vidoni, E. D. et al. Effect of aerobic exercise on amyloid accumulation in preclinical Alzheimer's: a 1-year randomized controlled trial. The effects of a moderate physical activity intervention on physical fitness and cognition in healthy elderly with low levels of physical activity: a randomized controlled trial. Schultz, S. A. et al. Cardiorespiratory fitness alters the influence of a polygenic risk score on biomarkers of AD. Z. et al. Dose-dependent association of accelerometer-measured physical activity and sedentary time with brain perfusion in aging. Iturria-Medina, Y., Sotero, R. C., Toussaint, P. J., Mateos-Pérez, J. M. & Evans, A. C. Early role of vascular dysregulation on late-onset Alzheimer's disease based on multifactorial data-driven analysis. Geffken, D. F. et al. Association between physical activity and markers of inflammation in a healthy elderly population. Exercise training ameliorates cognitive dysfunction in amyloid beta-injected rat model: possible mechanisms of Angiostatin/VEGF signaling. Exercise training improves vascular function in patients with Alzheimer's disease. Exercise induces hippocampal BDNF through a PGC-1α/FNDC5 pathway. Exercise hormone irisin is a critical regulator of cognitive function. Gorlovoy, P., Larionov, S., Pham, T. T. H. & Neumann, H. Accumulation of tau induced in neurites by microglial proinflammatory mediators. Martin, L. et al. VEGF counteracts amyloid-β-induced synaptic dysfunction. Plasma VEGFA and PGF impact longitudinal tau and cognition in preclinical Alzheimer's disease. Long-term exercise training improves memory in middle-aged men and modulates peripheral levels of BDNF and Cathepsin B. Sci. Dicarlo, M. et al. Irisin levels in cerebrospinal fluid correlate with biomarkers and clinical dementia scores in Alzheimer disease. A practical method for grading the cognitive state of patients for the clinician. Holbrook, E. A., Barreira, T. V. & Kang, M. Validity and reliability of Omron pedometers for prescribed and self-paced walking. Tudor-Locke, C. et al. How many days of pedometer monitoring predict weekly physical activity in adults? Johnson, K. A. et al. Tau positron emission tomographic imaging in aging and early Alzheimer disease. & Evans, A. C. Correction for partial volume effects in PET: principle and validation. Papp, K. V., Rentz, D. M., Orlovsky, I., Sperling, R. A. & Mormino, E. C. Optimizing the preclinical Alzheimer's cognitive composite with semantic processing: the PACC5. Grober, E., Lipton, R. B., Hall, C. & Crystal, H. Memory impairment on free and cued selective reminding predicts dementia. CHAMPS physical activity questionnaire for older adults: outcomes for interventions. D'Agostino, R. B. et al. General cardiovascular risk profile for use in primary care: the Framingham Heart Study. Tingley, D., Yamamoto, T., Hirose, K., Keele, L. & Imai, K. mediation: R Package for Causal Mediation Analysis. We thank the study participants of the Harvard Aging Brain Study. This work was supported by the National Institutes of Health (NIH) (grant numbers K23 AG084868 (W.-Y.W.Y. )); the Doris Duke Charitable Foundation Clinical Scientist Development Award (J.P.C.). This research was carried out in part at the Athinoula A. Martinos Center for Biomedical Imaging at the Massachusetts General Hospital, using resources provided by the Center for Functional Neuroimaging Technologies, P41EB015896, a P41 Biotechnology Resource Grant supported by the National Institute of Biomedical Imaging and Bioengineering (NIBIB), NIH. This work also involved the use of instrumentation supported by the NIH Shared Instrumentation Grant Program and/or High-End Instrumentation Grant Program, specifically, grant numbers S10RR021110, S10RR023401 and S10RR023043. A substantial amount of computational resources required for this research were performed on hardware generously provided by the Massachusetts Life Sciences Center (https://www.masslifesciences.com/). Funding sources had no involvement in study design, in the collection, analysis and interpretation of data, in the writing of the report, and in the decision to submit the article for publication. Department of Neurology, Mass General Brigham, Boston, MA, USA Wai-Ying Wendy Yau, Dylan R. Kirn, Michael J. Properzi, Aaron P. Schultz, Zahra Shirzadi, Kailee Palmgren, Paola Matos, Courtney Maa, Stephanie A. Schultz, Rachel F. Buckley, Dorene M. Rentz, Keith A. Johnson, Reisa A. Sperling & Jasmeer P. Chhatwal Harvard Medical School, Boston, MA, USA Wai-Ying Wendy Yau, Aaron P. Schultz, Zahra Shirzadi, Stephanie A. Schultz, Rachel F. Buckley, Dorene M. Rentz, Keith A. Johnson, Reisa A. Sperling & Jasmeer P. Chhatwal Harquail Centre for Neuromodulation and Hurvitz Brain Sciences Program, Sunnybrook Research Institute, Toronto, Ontario, Canada Division of Neurology, Department of Medicine, Sunnybrook Health Sciences Centre, University of Toronto, Toronto, Ontario, Canada Department of Neurology, Banner Alzheimer's Institute, Phoenix, AZ, USA Department of Radiology, Mass General Brigham, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to the acquisition of data. contributed to interpretation of data. drafted the initial manuscript. All authors provided substantive contributions to the revisions and approve the final version of the submitted manuscript. Correspondence to Wai-Ying Wendy Yau or Jasmeer P. Chhatwal. The authors declare no competing interests relevant to the current study. Potential conflicts of interest outside the submitted work are included below. has served as a consultant for Eisai. has served as a consultant for Novartis, Merck and Janssen. has served as a consultant for AbbVie, AC Immune, Acumen, Alector, Apellis, Biohaven, Bristol Myers Squibb, Genentech, Ionis, Janssen, Oligomerix, Prothena, Roche and Vaxxinity over the past 3 years. have received research funding from Eisai and Eli Lilly for public–private partnership clinical trials but do not have any personal financial relationship with the companies. has served as a consultant for ExpertConnect. Nature Medicine thanks Henry Brodaty, Susan Landau and Terry Therneau for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Individual trajectories of (a) global Aβ, (b) inferior temporal cortex (ITC) tau, (c) Preclinical Alzheimer's Cognitive Composite-5 (PACC5) scores, and d) Clinical Dementia Rating Sum of Boxes (CDR-SOB) scores from all participants are shown without adjustment for any covariates. To allow clear visualization of the individual trajectories, participants were plotted according to high versus low baseline Aβ burden in columns and physical activity (mean steps per day) in rows, defined by above and below the median values. The trajectories are color coded by baseline Aβ burden according to the color bar, with each line representing one participant. The number of participants represented in each facet is provided for reference. The time of baseline PiB PET was used as the study baseline (time = 0). Timing of the first tau PET scan varied across participants (2.2 ± 1.5 years), as tau PET was introduced mid-study when it became available. Aβ = beta-amyloid; DVR = distribution volume ratio; PVC = partial volume correction; PiB = Pittsburgh compound-B; SUVR = standardized uptake value ratio. Linear regression model revealed a significant interaction between baseline physical activity and Aβ burden on initial inferior temporal cortex (ITC) tau burden (Physical activity*Aβ: β = -0.19 [-0.30 to -0.08], p = 0.001), adjusting for age, sex, years of education, and time interval between study baseline and first tau scan. There was no significant independent effect of physical activity on tau (Physical activity: β = -0.06 [-0.18 to 0.05], p = 0.28). Statistical significance was assessed using two-tailed t-tests, with p < 0.05 considered statistically significant without adjustment for multiple comparisons. Physical activity (mean steps per day) was square-root transformed prior to model entry to account for skewness with improvement in model fit (reduced BIC by 2.2). Non-transformed mean steps per day was used to plot the model result to enhance interpretability. Baseline Aβ burden was modeled as a continuous variable. To visualize the model results, the estimated ITC tau burden across the range of baseline physical activity at representative levels of low versus high baseline Aβ burden are presented. Error bands represent the 95% confidence intervals for the estimated tau burden. Low and high Aβ are represented, for illustration purposes, by the mean Aβ burden of Aβ-negative (PiB PVC-DVR 1.17) and Aβ-positive (PiB PVC-DVR 1.85) participants respectively. Aβ = beta-amyloid; DVR = distribution volume ratio; ITC = inferior temporal cortex; PVC = partial volume correction; SUVR = standardized uptake value ratio. Linear regression model revealed no significant interaction between cross-sectional Aβ and ITC tau burdens on baseline physical activity (Aβ*ITC tau: β = -0.04 [-0.15 to -0.07], p = 0.44), adjusting for age, sex, years of education, and time interval between study baseline and first tau scan. There were further no significant independent effects of Aβ or ITC tau on baseline physical activity (Aβ: β = 0.10 [-0.07 to 0.28], t = 1.16, p = 0.25; ITC tau: β = 0.01 [-0.21 to 0.22], p = 0.96). Statistical significance was assessed using two-tailed t-tests, with p < 0.05 considered statistically significant without adjustment for multiple comparisons. Physical activity (mean steps per day) was square-root transformed prior to model entry, but non-transformed mean steps per day was used to plot the model result to enhance interpretability. Baseline Aβ burden was modeled as a continuous variable. To visualize the model results, the estimated means steps per day across the range of initial ITC tau burden at representative levels of low versus high baseline Aβ burden are presented. Error bands represent the 95% confidence intervals for the estimated mean steps per day. Low and high Aβ are represented, for illustration purposes, by the mean Aβ burden of Aβ-negative (PiB PVC-DVR 1.17) and Aβ-positive (PiB PVC-DVR 1.85) participants respectively. Aβ = beta-amyloid; DVR = distribution volume ratio; ITC = inferior temporal cortex; PVC = partial volume correction; SUVR = standardized uptake value ratio. Linear mixed effects models revealed interaction between baseline physical activity levels (ordinal) and Aβ burden (continuous) on longitudinal PACC5 decline and CDR-SOB progression. Using the inactive subgroup as reference, all higher levels of physical activity were associated with slower Aβ-related PACC5 decline and CDR-SOB progression, except the slower CDR-SOB progression in the low activity group did not reach statistical significance (Table 3). To visualize the model results, the estimated trajectories based on representative levels of low versus high baseline Aβ burden across physical activity levels are presented, with error bands representing 95% confidence intervals for the estimated trajectories. Low and high Aβ are represented, for illustration purposes, by the mean Aβ burden of Aβ-negative (PiB PVC-DVR 1.17) and Aβ-positive (PiB PVC-DVR 1.85) participants respectively, defined using the conventional Aβ-positivity threshold (PiB PVC-DVR of 1.324). The horizontal dotted line represents thresholds for cognitive impairment (-1.5 PACC5 z-score) and functional decline (1.5 points on CDR-SOB). The vertical dot-dash line represents median duration of cognitive follow-up (9 years). Aβ = β-amyloid; DVR = distribution volume ratio; PiB = Pittsburgh compound-B; PVC = partial volume corrected; SUVR = standardized uptake value ratio. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Physical activity as a modifiable risk factor in preclinical Alzheimer's disease. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41561-025-01802-4'>Record grounded glacier retreat caused by an ice plain calving process</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 16:05:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Geoscience (2025)Cite this article Understanding and predicting marine-terminating glacier instability presents one of the greatest challenges to forecasting future sea level rise. An extreme case of such instability is the Hektoria Glacier on the Eastern Antarctic Peninsula, which retreated ~25 km between January 2022 and March 2023. Here we investigate the dynamics and drivers of this retreat event primarily from analysis of geophysical data and satellite imagery. We find that retreat commenced immediately after the loss of decade-old fast ice in the Larsen B embayment and was associated with an almost 6-fold increase in flow speed and 40-fold increase in glacier thinning, relative to the period immediately before the fast ice loss. We also find that in November–December 2022, the glacier retreated a total of 8.2 ± 0.2 km in two months—a retreat rate nearly an order of magnitude faster than published values—and began with a transition from tabular iceberg calving to buoyancy-driven calving on an ice plain, a flat area where the glacier was only lightly grounded. Hence we conclude that in this case, retreat primarily resulted from an ice plain calving process, rather than atmospheric or oceanic conditions as suggested previously. This implies that marine-terminating glaciers with ice plain bed geometry can be easily destabilized. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout IceBridge MCoRDS radar data are publicly available on the CReSIS webpage at https://data.cresis.ku.edu/ (ref. 70) or the National Snow and Ice Data Center at https://doi.org/10.5067/GDQ0CUCVTE2Q (ref. All Landsat 8 and 9 (https://doi.org/10.5066/P975CC9B)72, MODIS and ICESat-2 data (https://doi.org/10.5067/ATLAS/ATL06.006)73 are publicly available on https://search.earthdata.nasa.gov/. Sentinel-1 and 2 data were provided by the Copernicus Program of the European Space Agency and are publicly available at https://browser.dataspace.copernicus.eu/. Planet and Worldview data are available in the main text figures and extended data materials or by contacting the Polar Geospatial Center or Plantscope with a CSDA agreement. ASTER (https://doi.org/10.5067/ASTER/AST_L1A.003)74 and SPOT5 HRS imagery are freely available, respectively, at https://search.earthdata.nasa.gov/ and https://regards.cnes.fr/user/swh/; corresponding DEMs are available upon request. The Sentinel-1 velocities, DEM elevations of extracted profiles from ASTER, Worldview and SPOT5, retreat rates, iceberg dimension analysis and corresponding shapefiles are available in the data repository for this manuscript at https://doi.org/10.15784/601973 (ref. All seismic data were downloaded from the EarthScope Consortium Web Services including data from the POLENET Network (https://doi.org/10.7914/SN/YT_2007)76, Antarctic Seismographic Argentinean Italian Network (https://doi.org/10.7914/SN/AI)77 and the Global Seismograph Network (https://doi.org/10.7914/SN/IU)78. Source data are provided with this paper. Ochwat, N. E. et al. Triggers of the 2022 Larsen B multi-year landfast sea ice breakout and initial glacier response. Google Scholar Fürst, J. J. et al. The safety band of Antarctic ice shelves. Gilbert, E. & Kittel, C. Surface melt and runoff on Antarctic ice shelves at 1.5 °C, 2 °C, and 4 °C of future warming. Glasser, N. F. et al. From ice-shelf tributary to tidewater glacier: continued rapid recession, acceleration and thinning of Röhss Glacier following the 1995 collapse of the Prince Gustav Ice Shelf, Antarctic Peninsula. Rott, H., Skvarca, P. & Nagler, T. Rapid collapse of northern Larsen Ice Shelf, Antarctica. Royston, S. & Gudmundsson, G. H. Changes in ice-shelf buttressing following the collapse of Larsen A Ice Shelf, Antarctica, and the resulting impact on tributaries. Rignot, E. et al. Accelerated ice discharge from the Antarctic Peninsula following the collapse of Larsen B ice shelf. Scambos, T. et al. Detailed ice loss pattern in the northern Antarctic Peninsula: widespread decline driven by ice front retreats. Rankl, M., Fürst, J. J., Humbert, A. & Braun, M. H. Dynamic changes on the Wilkins Ice Shelf during the 2006–2009 retreat derived from satellite observations. Hulbe, C. L., Scambos, T. A., Youngberg, T. & Lamb, A. K. Patterns of glacier response to disintegration of the Larsen B ice shelf, Antarctic Peninsula. Scambos, T. A., Bohlander, J. & Skvarca, P. Glacier acceleration and thinning after ice shelf collapse in the Larsen B embayment, Antarctica. Needell, C. & Holschuh, N. Evaluating the retreat, arrest, and regrowth of crane glacier against marine ice cliff process models. Surawy-Stepney, T. et al. The effect of landfast sea ice buttressing on ice dynamic speedup in the Larsen B embayment, Antarctica. Fluegel, B. L. & Walker, C. The two-decade evolution of Antarctica's Hektoria Glacier and its 2022 rapid retreat from satellite observations. Bassis, J. N. & Walker, C. C. Upper and lower limits on the stability of calving glaciers from the yield strength envelope of ice. Pollard, D., DeConto, R. M. & Alley, R. B. Potential Antarctic Ice Sheet retreat driven by hydrofracturing and ice cliff failure. Parizek, B. R. et al. Ice-cliff failure via retrogressive slumping. Schoof, C. Marine ice sheet stability. J. Fluid Mech. Favier, L. et al. Retreat of Pine Island Glacier controlled by marine ice-sheet instability. Gudmundsson, G. H., Paolo, F. S., Adusumilli, S. & Fricker, H. A. Instantaneous Antarctic ice sheet mass loss driven by thinning ice shelves. Brunt, K. M., Fricker, H. A. & Padman, L. Analysis of ice plains of the Filchner–Ronne Ice Shelf, Antarctica, using ICESat laser altimetry. Friedl, P., Weiser, F., Fluhrer, A. & Braun, M. H. Remote sensing of glacier and ice sheet grounding lines: a review. Earth Sci. Crawford, A. J. et al. Marine ice-cliff instability modeling shows mixed-mode ice-cliff failure and yields calving rate parameterization. Bassis, J. N., Berg, B., Crawford, A. J. & Benn, D. I. Transition to marine ice cliff instability controlled by ice thickness gradients and velocity. Melton, S. M. et al. Meltwater drainage and iceberg calving observed in high-spatiotemporal resolution at Helheim Glacier, Greenland. Murray, T. et al. Dynamics of glacier calving at the ungrounded margin of Helheim Glacier, southeast Greenland: dynamics of glacier calving. Earth Surf. Batchelor, C. L. et al. Rapid, buoyancy-driven ice-sheet retreat of hundreds of metres per day. Tuckett, P. A. et al. Reply to: ‘impact of marine processes on flow dynamics of northern Antarctic Peninsula outlet glaciers' by Rott et al. Nat. Alley, R. B. et al. Iceberg calving: regimes and transitions. Sulak, D. J., Sutherland, D. A., Enderlin, E. M., Stearns, L. A. & Hamilton, G. S. Iceberg properties and distributions in three Greenlandic fjords using satellite imagery. Robel, A. A. Thinning sea ice weakens buttressing force of iceberg mélange and promotes calving. Rott, H. et al. Changing pattern of ice flow and mass balance for glaciers discharging into the Larsen A and B embayments, Antarctic Peninsula, 2011 to 2016. Wallis, B. J., Hogg, A. E., Zhu, Y. & Hooper, A. Change in grounding line location on the Antarctic Peninsula measured using a tidal motion offset correlation method. Anandakrishnan, S., Voigt, D., Alley, R. & King, M. Ice stream D flow speed is strongly modulated by the tide beneath the Ross Ice Shelf. Voytenko, D. et al. Tidally driven ice speed variation at Helheim Glacier, Greenland, observed with terrestrial radar interferometry. Freer, B. I., Marsh, O. J., Hogg, A. E., Fricker, H. A. & Padman, L. Modes of Antarctic tidal grounding line migration revealed by ICESat-2 laser altimetry. Sergeant, A. et al. Monitoring Greenland ice sheet buoyancy-driven calving discharge using glacial earthquakes. Olsen, K. G. et al. Improved estimation of glacial‐earthquake size through new modeling of the seismic source. Earth Surf. Nettles, M. & Ekström, G. Glacial earthquakes in Greenland and Antarctica. Winberry, J. P. et al. Glacial earthquakes and precursory seismicity associated with Thwaites Glacier calving. Parsons, R., Sun, S., Gudmundsson, G. H., Wuite, J. & Nagler, T. Quantifying the buttressing contribution of landfast sea ice and mélange to Crane Glacier, Antarctic Peninsula. Pfeffer, W. T. A simple mechanism for irreversible tidewater glacier retreat. O'Neel, S., Pfeffer, W. T., Krimmel, R. & Meier, M. Evolving force balance at Columbia Glacier, Alaska, during its rapid retreat. Catania, G. A. et al. Geometric controls on tidewater glacier retreat in central western Greenland. JGR Earth Surf. Chudley, T. R., Howat, I. M., King, M. D. & Negrete, A. Atlantic water intrusion triggers rapid retreat and regime change at previously stable Greenland glacier. Milillo, P. et al. Rapid glacier retreat rates observed in West Antarctica. Shuman, C. A., Berthier, E. & Scambos, T. A. 2001–2009 elevation and mass losses in the Larsen A and B embayments, Antarctic Peninsula. Dussaillant, I. et al. Two decades of glacier mass loss along the Andes. Fox-Kemper, B. et al. in Climate Change 2021: The Physical Science Basis (eds Masson-Delmotte, V. et al.) 1211–1362 (IPCC, Cambridge Univ. Graham, A. G. C. et al. Rapid retreat of thwaites glacier in the pre-satellite era. Bindschadler, R., Vornberger, P. & Gray, L. Changes in the ice plain of Whillans Ice Stream, West Antarctica. Corr, H. F., Doake, C., Jenkins, A. & Vaughan, D. G. Investigations of an ‘ice plain' in the mouth of Pine Island Glacier, Antarctica. Freer, B. I. D. et al. Coincident lake drainage and grounding line retreat at Engelhardt Subglacial Lake, West Antarctica. Earth Surf. Fricker, H. A. et al. Mapping the grounding zone of the Amery Ice Shelf, East Antarctica using InSAR, MODIS, and ICESat. Matsuoka, K., Skoglund, A. & Roth, G. Quantarctica [Simple Basemap] (Norwegian Polar Institute, 2018). Gardner, A., Fahnestock, M. & Scambos, T. MEaSUREs ITS_LIVE Landsat image-pair glacier and ice sheet surface velocities. Version 1 Boulder, Colorado USA. NASA National Snow and Ice Data Center https://doi.org/10.5067/IMR9D3PEI28U (2022). Bernat, M. et al. Geodetic mass balance of Mýrdalsjökull Ice Cap, 1999–2021. Berthier, E., Scambos, T. A. Mass loss of Larsen B tributary glaciers (Antarctic Peninsula) unabated since 2002. Howat, I. M., Porter, C., Smith, B. E., Noh, M.-J. & Morin, P. The reference elevation model of Antarctica. Morlighem, M. et al. Deep glacial troughs and stabilizing ridges unveiled beneath the margins of the Antarctic ice sheet. Huss, M. & Farinotti, D. A high-resolution bedrock map for the Antarctic Peninsula. Shahateet, K., Navarro, F., Seehaus, T., Fürst, J. J. & Braun, M. Estimating ice discharge of the Antarctic Peninsula using different ice-thickness datasets. Luckman, A., Quincey, D. & Bevan, S. The potential of satellite radar interferometry and feature tracking for monitoring flow rates of Himalayan glaciers. Smith, B. et al. Land ice height-retrieval algorithm for NASA's ICESat-2 photon-counting laser altimeter. Antropova, Y. K. et al. Grounding-line retreat of Milne Glacier, Ellesmere Island, Canada over 1966–2023 from satellite, airborne, and ground radar data. Gades, A. M., Raymond, C. F., Conway, H. & Jagobel, R. Bed properties of Siple Dome and adjacent ice streams, West Antarctica, inferred from radio-echo sounding measurements. Copland, L. & Sharp, M. Mapping thermal and hydrological conditions beneath a polythermal glacier with radio-echo sounding. Walter, F. et al. Analysis of low-frequency seismic signals generated during a multiple-iceberg calving event at Jakobshavn Isbræ, Greenland. Earth Surf. Krischer, L. et al. On-demand custom broadband synthetic seismograms. Radar Depth Sounder Data, Lawrence, Kansas, USA (CReSIS, 2024); http://data.cresis.ku.edu/ Paden, J., Li, J., Leuschen, C., Rodriguez-Morales, F. & Hale, R. IceBridge MCoRDS L2 ice thickness (IRMCR2, Version 1). NASA National Snow and Ice Data Center Distributed Active Archive Center https://doi.org/10.5067/GDQ0CUCVTE2Q (2010). Smith, B. et al. ATLAS/ICESat-2 L3A land ice height. NASA National Snow and Ice Data Center Distributed Active Archive Center https://doi.org/10.5067/ATLAS/ATL06.006 (2023). Earth Resources Observation and Science (EROS) Center. Landsat 8-9 operational land imager/thermal infrared sensor level-1, collection 2. NASA/METI/AIST/Japan Spacesystems and US/Japan ASTER Science Team. ASTER level 1A data set - reconstructed, unprocessed instrument data. NASA Land Processes Distributed Active Archive Center https://doi.org/10.5067/ASTER/AST_L1A.003 (2001). Ochwat, N. et al. ‘Hektoria and Green Glacier changes 2022-2023' U.S. Antarctic Program (USAP) Data Center https://doi.org/10.15784/601973 (2025). Wiens D., Nyblade A. & Aster R. IPY POLENET-Antarctica: investigating links between geodynamics and ice sheets. International Federation of Digital Seismograph Networks https://doi.org/10.7914/SN/YT_2007 (2007). Istituto Nazionale di Oceanografia e di Geofisica Sperimentale. Antarctic seismographic Argentinean Italian network - ASAIN. International Federation of Digital Seismograph Networks https://doi.org/10.7914/SN/AI (1992). Albuquerque Seismological Laboratory/USGS. Global seismograph network (GSN - IRIS/USGS). International Federation of Digital Seismograph Networks https://doi.org/10.7914/SN/IU (1988). Download references We would like to thank NASA for funding our research and our fieldwork in 2024 (grant number 80NSSC22K0386). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. We are grateful for the US Geological Survey for providing access to the optical imagery archive. We thank the Polar Geospatial Center for providing the Worldview DEMs and imagery under NSF-OPP awards 1043681, 1559691 and 2129685. We acknowledge the use of data and/or data products from CReSIS generated with support from the University of Kansas, NASA Operation IceBridge grant NNX16AH54G, NSF grants ACI-1443054, OPP-1739003 and IIS-1838230, Lilly Endowment Incorporated and Indiana METACyt Initiative. We acknowledge the use of imagery from the NASA Worldview application (https://worldview.earthdata.nasa.gov), part of the NASA Earth Science Data and Information System (ESDIS). We thank Planet for providing access to the PlanetScope archive, provided through the CSDA agreement. We are grateful for the continued support from CIRES, particularly CIRES travel and D. Williams. We thank the Instituto Antártico Argentino for supporting our fieldwork to the Antarctic Peninsula and the helicopter pilots who flew us over Hektoria Glacier. We also thank the Norwegian Polar Institute for providing an open source Antarctic basemap from Quantarctica. We thank the many conversations with D. Benn, A. Crawford, T. Snow, C. Walker, B. Wallis, S. Melton and countless others. acknowledge support from the French Space Agency (Centre National d'Etudes Spatiales–CNES). acknowledges a fellowship from the SDU2E doctoral school of the University of Toulouse. Earth Science Observation Center, Cooperative Institute for Research in Environmental Science, University of Colorado Boulder, Boulder, CO, USA Naomi Ochwat & Ted Scambos Department of Geological Sciences, University of Colorado Boulder, Boulder, CO, USA Naomi Ochwat & Robert S. Anderson Institute for Arctic and Alpine Research, University of Colorado Boulder, Boulder, CO, USA Department of Geological Sciences, Central Washington University, Ellensburg, WA, USA J. Paul Winberry Department of Geography, Faculty of Science and Engineering, Swansea University, Swansea, UK Adrian Luckman Université de Toulouse, LEGOS (CNES/CNRS/IRD/UT3), Toulouse, France Etienne Berthier & Maud Bernat Department of Geography and Environmental Studies, Carleton University, Ottawa, Ontario, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized and designed the study. obtained and processed all optical satellite data, ICESat-2 data and calculated ice thickness and ice volume. obtained and processed all Sentinel-1 velocity data. obtained and processed the SPOT and ASTER into DEMs. obtained and processed the seismic data. obtained and processed the airborne radar data. created visualizations from the all the results, except for the airborne radar data and seismic data, in which Y.K.A. created, respectfully. analysed the results with input and supervision from T.S. wrote the original manuscript. all contributed to scientific discussion, interpretation of results and revisions and editing of the manuscript. Correspondence to Naomi Ochwat. The authors declare no competing interests. Nature Geoscience thanks Bryony Freer, Pietro Milillo and Peter Tuckett for their contribution to the peer review of this work. Primary Handling Editors: Stefan Lachowycz and Tom Richardson, in collaboration with the Nature Geoscience team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Tracks where velocity was extracted, black dashed line is grounding line1. b) Along-flow line velocity profile (red line) c) The Icebridge ATM track velocity profile (blue line), shown to compare the speeds to those presented in Ochwat et al.1. Ochwat et al.1 show a speed increase in Green Glacier of 500 m yr−1 to 1150 m yr−1 by January 2023, and here we show the speed has continued to increase to 1700 m yr−1 by December 2023 (measured at 7500 m distance along track), representing ~3.5-fold increase in speed since the loss of the fast ice. At 11 km upstream of the Green Glacier terminus the glacier flow speed has yet to be affected by the loss of the fast ice and new calving regime. Green Glacier underwent similar phases of change as Hektoria, yet only retreated 4.5 km of grounded ice through austral winter 2023. Since then, Green Glacier has undergone several periods of minimal advancement and subsequent retreat of up to 2 km until April 2024 and is still actively calving and retreating. Source data a) Series of Worldview 2 orthoimage on 3 October 2022 at different zoomed levels. b) Elevation profile at glacier front from Worldview derived DEM on 3 October 2022. The terminus has a ~ 50 m high cliff front and displays slumping and rifting upstream of the terminus. The iceberg at the front is clearly rotated ‘bottom-out' indicating it was calved via buoyancy-driven processes. Credit: images in a, Copyright 2022 Maxar. Source data The black dashed line is the grounding zone from our earlier study1, and the purple line is the ice front on 14 November 2022. The three panels share the same scale and depict different areal extents. Here, ‘width' refers to the longest dimension of the iceberg; ‘length' refers to the maximum dimension perpendicular to the width axis. The largest length is the minimum ice thickness. a) Average of nine icebergs resulted in a minimum ice thickness of 407 ± 11 m. b) Average of ten icebergs resulted in a minimum ice thickness of 378 ± 18 m. c) Average of nine icebergs resulting in a minimum ice thickness of 492 ± 30 m (Image © 2022 Planet Labs PBC. All Rights Reserved). Source data a) IceBridge track overlying a true-color Landsat-8 optical image acquired on 3 December 2017. The red contour and vertical line is the Ochwat et al.1 grounding line. b) Ice surface and bottom heights; c) Normalized Bed Reflection Power (NBRP) and Normalized Internal Reflection Power (NIRP). The grounded part of the glacier associated with relatively low NBRP values is highlighted by light orange color, the floating ice associated with relatively high NBRP values is highlighted by blue color, and the region where both NBRP and NIRP values start to increase is highlighted by gray color. This region is likely associated with the transition of the glacier from grounded to floating, however, the bed signal at the beginning of this zone is obscured by high internal reflection (NIRP > NBRP) associated with the crevasses clearly seen in the Landsat-8 images (Panel a). a) Landsat image from December 8, 2022 with the force vector inferred from the modeling of the waveforms. Note: force vector has been placed manually near the calving front of Hektoria Glacier for visual clarity. b) the bandpass-filtered (20–50 s) observed (black) and synthetic (red) waveforms from a grid search for horizontal force magnitude and direction. Stations are sorted by distance to location; their locations are shown in Fig. 5 in the main text. c) variance reduction as a function force azimuth and magnitude. Timelapse of Sentinel-1 SAR imagery from November 2021 to July 2025. GoPro footage of Hektoria Glacier reconnaissance flight 26 February 2024 courtesy of Captain Franco Saravalli Fuerza Aérea Argentina. Table of retreat rates and area loss. Icesat-2 is available on EarthExplorer; we have provided the unprocessed extracted Sentinel-1 velocities. We have provided a table of the unprocessed extracted elevations from four along-flow profiles. We have provided a table of the unprocessed extracted Sentinel-1 velocities. Table of extracted elevation profile. Table of measured iceberg thicknesses. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Ochwat, N., Scambos, T., Anderson, R.S. et al. Record grounded glacier retreat caused by an ice plain calving process. Download citation Received: 21 December 2024 Version of record: 03 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Geoscience © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69236115/revolutionary-war-barracks-remnants-discovery/'>Archaeologists Accidentally Found the Incredible Lost Remnants of America's First Soldiers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 15:18:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>What he hoped to destroy forever was recently found by archaeologists, giving experts a fresh perspective on the life of Continental soldiers during the Revolutionary War. As crews made way for the construction of a new sports center on Colonial Williamsburg Foundation property, archaeologists first took to the area to see if 18th century maps and period documents referencing the construction of barracks in 1776 through 1777 produced any archaeological fruit. They have plenty to pick from, as the nearly four-acre site has artifacts showing both the barracks and everyday life of soldiers. Archaeologists have excavated only a small section of the area—the sports center was shifted on the site to ensure future exploration of the barracks—but the team has already found bricks and an intact chimney base, gun hardware, lead musket shot with toothmarks, high-end ceramics, and personal items likely owned by officers. Those barracks didn't last long, though, with Gary saying a report from a soldier describes the burning of the barracks by British troops while they moved toward Yorktown in 1781. “Later on, after the Cornwallis troops moved through, they could see the barracks on fire in the distance … For us as archaeologists, this is a burning event, which is a catastrophic event,” Gary tells Fox News. “But it can also lend itself to really good preservation.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Are There More Humans on Earth Than We Thought? We May Have Been Wrong About the Origin of Life AUV Reveals Stunning Images of Dotson Ice Shelf A Fisherman Digging Up Worms Found Buried Treasure</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a69235688/deep-ocean-atacama-trench-crustacean-predator/'>A New Predator Has Emerged From the Deepest, Darkest Part of the Ocean</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>On October 14, 2024, NASA's Europa Clipper launched from the Kennedy Space Center in Florida for Jupiter's fourth largest moon. Its mission is to assess the moon's potential to host life in the vast ocean beneath its icy crust. While the Europa Clipper is a milestone in humanity's ability to study the oceans of other worlds, Earth's oceans still hold many mysteries yet unsolved by science. Without leaving Earth, scientists at Woods Hole Oceanographic Institute (WHOI) and the Universidad de Concepción in Chile have discovered new life themselves—a previously unknown, active predator in one of the deepest trenches in the world. Although its white carapace gives it a ghostly, almost Facehugger-from-Alien feel, the name is fitting if you consider that this predator lives out its life 7,000 meters below the aphotic zone, where absolute darkness reigns. Because of its deep ocean environment, this mighty crustacean, which is only roughly four centimeters in length, can also withstand pressures up to 800 times stronger than pressures found on land. “This finding underlines the importance of continued deep-ocean exploration, particularly in Chile's front yard,” IMO's Carolina González, co-lead author of the study, said in a press statement. “More discoveries are expected as we continue to study the Atacama Trench.” If something so small can persevere in conditions as adverse as those in the Hadal zone, maybe Europa Clipper has a non-zero shot at finding conditions that could support life in the vast, salty ocean of a whole different world. Are There More Humans on Earth Than We Thought? Roman Military Camp Discovered High in Swiss Alps We May Have Been Wrong About the Origin of Life A Fisherman Digging Up Worms Found Buried Treasure Scientists Found an Ancient Figurine With No Face</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03573-9'>Sex, drugs and the conscious brain: Francis Crick beyond the double helix</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 14:39:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Georgina Ferry is a science writer based in Oxford, UK, and the author of Dorothy Crowfoot Hodgkin (2019). Francis Crick served in the British Admiralty and conducted research into naval mines.Credit: GL Archive/Alamy Crick: A Mind in Motion — from DNA to the Brain Matthew Cobb Profile Books (2025) The pair were immortalized in Watson's sensational 1968 book, The Double Helix, with Crick painted as garrulous and cerebral and Watson as gauche but driven. At the same time, he makes a more sustained attempt than either of Crick's previous biographers (Matt Ridley and Robert Olby) to answer several questions. Crick was notoriously reluctant to divulge personal information or even have his photograph taken. Combing through a remarkably comprehensive set of personal and professional archives with meticulous attention to detail, Cobb has reconstructed Crick's relationships with those who were essential crew mates on his intellectual odyssey. As Cobb explains, Crick's future was transformed by his experiences during the Second World War. His PhD studies were interrupted when he was called up to serve in the British Admiralty, conducting research into naval mines — real-time problem-solving that suited his restless nature. She became Crick's second wife in 1949; their partnership was one of mutual devotion that allowed for his numerous affairs. He began a new PhD project in structural biology at the Medical Research Council's Unit for Research on the Molecular Structure of Biological Systems — later known as the Laboratory of Molecular Biology — in Cambridge, UK, where he worked for almost 30 years. Under the genial chairmanship of molecular biologist Max Perutz, Crick never had to teach or grapple with university administration: he applied for a grant only once in his life. He had the grace to admit that luck had played a part in his success, but to some extent he made his own luck. “They should have requested permission to use the data,” Cobb writes. In Crick's PhD thesis, he had called this the ‘central problem' of biology. Between them, they conceived the existence of messenger RNA, showed that the sequence of bases in mRNA acts as a code that dictates each protein's amino-acid sequence and found that three bases encoded each amino acid (F. H. C. Crick et al. It's time to admit that genes are not the blueprint for life It's time to admit that genes are not the blueprint for life Never underestimate RNA: how a molecule went from bit player to star of the show It's time to admit that genes are not the blueprint for life Personalized gene editing helped one baby: can it be rolled out widely? Chen-Ning Yang obituary: intuitive physicist whose work was inspired by the beauty of symmetry How Napoleon's army met its doom: DNA reveals surprise illnesses had a role We are looking for candidates pursuing curiosity-driven basic and translational research. Taipei, TW-a vibrant city blending modern innovation, rich culture, and convenient transportation. The Westlake Fellowship provides funding for global talent to study PhD programs in 9 areas of science, engineering, and technology. The aim of this research is to develop neuromorphic devices for next-generation brain-inspired computers that consume less power and are more compact. Never underestimate RNA: how a molecule went from bit player to star of the show It's time to admit that genes are not the blueprint for life An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251102205014.htm'>Scientists discover the nutrient that supercharges cellular energy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 14:27:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Because energy needs change continuously, mitochondria must constantly fine-tune their activity to keep up. Yet until recently, scientists did not fully understand how nutrients influence this process of energy adjustment. A research group led by Professor Dr. Thorsten Hoppe at the University of Cologne's Institute for Genetics and the CECAD Cluster of Excellence on Aging Research has now identified a new biological pathway showing how the amino acid leucine strengthens mitochondrial performance. Their work reveals that leucine stabilizes important mitochondrial proteins, enhancing the organelle's ability to produce energy. Leucine is one of the essential amino acids, meaning it must be obtained through food. These proteins are vital because they transport other metabolic molecules into the mitochondria to generate energy. By preserving them, leucine allows mitochondria to work more efficiently, resulting in higher energy output within the cell. "We were thrilled to discover that a cell's nutrient status, especially its leucine levels, directly impacts energy production," said Dr. Qiaochu Li, the study's first author. The study also revealed that a protein called SEL1L plays an important role in this process. SEL1L is part of the cell's quality control system that identifies and removes faulty or misfolded proteins. "Modulating leucine and SEL1L levels could be a strategy to boost energy production," said Li. SEL1L also plays a crucial role in preventing the accumulation of damaged proteins, which is essential for long-term cellular health." They discovered that problems in leucine metabolism can impair mitochondrial function and even cause fertility issues. By uncovering how leucine affects mitochondrial metabolism, the study points to potential new therapeutic targets for conditions linked to reduced energy production, including cancer and metabolic disorders. 100x More Potent: Chemists Discover Powerful New Antibiotic “Hidden in Plain Sight” Ancient Radio Signals Could Reveal What Dark Matter Really Is Twin Black Hole Collisions Offer the Most Precise Test of Einstein's Theory Yet Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69203757/cuneiform-tablet-taxes/'>Archaeologists Found an Ancient Tablet Inscribed With an Ominous Royal Threat</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Death and taxes have always been inevitable, even in the ancient world. Kings Hezekiah and Manasseh were the heads of Judah during this period. Ancient texts do not make either sound thrilled to bend the knee. Assyrian accounts of the time revealed that Sennacherib demanded exorbitant tribute payments from Judah after Jerusalem had been crushed. The Iron Age fragment is only an inch long, but carries a message with the resonance of a booming voice. It was her colleague Moriah Cohen who identified the seal while sifting through a pile of rubble. While cuneiform writings on clay tablets and prisms have survived for millennia, there had been no direct evidence of official correspondence between Assyria and Judah until the discovery of this artifact. “We have the connection between archeology, history, and science,” Assyriologist Peter Zilberg of Bar-Ilan University told The Times of Israel. Both the Bible and Assyrian sources say that Hezekiah eventually relented. He later sent a message to the king of Assyria at Lachish, admitting he had done wrong and giving the Assyrians permission to extract payment from him. Even back then, this is what happened if you didn't pay your taxes. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Are There More Humans on Earth Than We Thought? Roman Military Camp Discovered High in Swiss Alps The Place Where Time Moves 9% Slower for Humans We May Have Been Wrong About the Origin of Life AUV Reveals Stunning Images of Dotson Ice Shelf Scientists Found an Ancient Figurine With No Face</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69195317/roman-mechanic-shop/'>Archaeologists Discovered the Remains of an Ancient Roman Mechanic Shop</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A smattering of iron pieces and an array of nails discovered in the remains of a first century Roman building provided some clues as to what the ancient site's use must have been. The excavation resulted in evidence pointing to the site having once been a metalworking shop—one specifically tied to an ancient Roman posting station serving the needs of travelers by repairing carriages and horseshoes. Historically called Endidae, it was known for acting as a way stop along the Roman road network. Trading was heavy along the Via Claudia Augusta. The team also uncovered Roman roof tiles stamped with the manufacturer's name (Auresis)—which could help track where the objects came from—as well as a mix of other items popular in Roman trade, such as fragments of ceramic and glass pots, pieces of amphorae, and five coins. Experts date all the finds to between the first and third centuries C.E., keeping them in line with previous discoveries nearby. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Are There More Humans on Earth Than We Thought? Roman Military Camp Discovered High in Swiss Alps The Place Where Time Moves 9% Slower for Humans We May Have Been Wrong About the Origin of Life A Fisherman Digging Up Worms Found Buried Treasure</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64722-2'>Complex de novo structural variants are an underestimated cause of rare disorders</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 10:29:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Complex de novo structural variants (dnSVs) are crucial genetic factors in rare disorders, yet their prevalence and characteristics in rare disorders remain poorly understood. Here, we conduct a comprehensive analysis of whole-genome sequencing data of 12,568 families, including 13,698 offspring with rare diseases, obtained as part of the UK 100,000 Genomes Project. We identify 1,870 dnSVs, constituting the largest dnSV dataset reported to date. Complex dnSVs (n = 158; 8.4%) emerge as the third most common type of SV, following simple deletions and duplications. We classify 65% of these complex dnSVs into 11 subtypes. Among probands with dnSVs (n = 1,696), 9% exhibit exon-disrupting pathogenic dnSVs associated with the probands' phenotype. Notably, 12% of exon-disrupting pathogenic dnSVs and 22% of de novo deletions or duplications previously identified by array-based or whole-exome sequencing methods are found to be complex dnSVs. We also find distinct genomic properties of de novo deletions depending on the parent of origin. This study highlights the importance of complex dnSVs in the cause of rare disorders and demonstrates the necessity of specific genomic analysis to avoid overlooking these variants. Structural variants (SVs), defined as genetic changes \(\ge\)50 bp that encompass copy number variants (CNVs)1, rearrangements, and mobile element insertions, play an important role in cancer when occurring in somatic cells2. They also arise in the germline, with de novo structural variants (dnSVs) contributing to rare disorders3,4,5,6,7,8,9,10. For instance, chromosomal microarray (CMA), which is capable of detecting submicroscopic CNVs, demonstrated an average diagnostic yield of 12.2% in patients with developmental and intellectual disorders11. Beyond CNVs, other types of SVs, such as complex SVs involving clustered breakpoints originating from a single event3,12, have provided insights into the genetic aetiology of rare disorders13,14,15, surpassing the explanatory power of CNVs alone. Nevertheless, in contrast to de novo single-nucleotide variants (SNVs), there is limited information on the prevalence and characteristics of dnSVs, particularly complex rearrangements, in rare disorders, primarily due to the substantial technical challenges associated with their detection16. One prominent difficulty arises from the inherent limitations of short-read technologies in accurately capturing and characterising large-scale genomic rearrangements. The restricted read lengths often result in fragmented or incomplete representations of complex structural variations, leading to difficulties in assembling the complete picture of genomic architecture. This issue is particularly pronounced in regions with high sequence similarity, where distinguishing between homologous sequences presents significant computational and analytical challenges. Long-read sequencing mitigates the challenges associated with short-read platforms by offering a more direct span across SVs, thereby enabling better resolution and a more complete representation of complex genomic variations. While long-read sequencing offers unique advantages in studying SVs, the lack of substantial long-read sequence datasets from rare disorder cohorts highlights the ongoing importance of precise short-read based SV analytical pipelines17,18,19. These pipelines, essential for detecting a broad spectrum of SVs and reducing false positives. This is particularly pertinent in the absence of large cohort population datasets, which hampers accurate filtering and necessitates robust short-read analytical approaches20. Consequently, leveraging large-scale short-read sequence datasets with rigorous analytical approaches remains key for a nuanced understanding of SVs in diseases, particularly rare disorders. To shed light on their significance in rare diseases, we analysed dnSVs identified in 13,702 whole-genome-sequenced parent–child trios from 12,568 families from the rare disease programme of the 100,000 Genomes Project (Supplementary Data 1)21 using rigorous analytical approaches. The rare disease cohort encompasses individuals with a broad spectrum of conditions, with neurology and neurodevelopmental (NN) disorders making up half of the cohort. Other represented disorders include ultra-rare conditions, ophthalmological, renal and urinary tract, cardiovascular, endocrine, and additional disease groups (Supplementary Fig. We developed a rigorous pipeline to analyse an average of 13,980 candidate variants (standard deviation = 2550) per proband, already called by Genomic England using the Manta caller22. We identified a total of 1870 high-confidence dnSVs (Fig. 1 and Supplementary Data 2), all of which were visually inspected (“Methods”). Some of these dnSVs were validated using previously identified dnSVs detected in independent sequencing data from overlapping family cohorts. The validation rate was 100% (n = 44): 37 candidate dnSVs were confirmed by array/whole-exome sequencing from the Deciphering Developmental Disorders (DDD)23 study and 7 candidate dnSVs were confirmed using long-read sequencing data from Genomic England (GEL)21, respectively (Supplementary Fig. In addition, we validated 11 pathogenic dnSVs (Fig. 3) using RNA-seq data by confirming abnormal RNA reads supporting dnSVs (Supplementary Fig. 3b), or aberrant splicing patterns (Supplementary Fig. a Comparison of the proportion of probands with dnSVs between those with diagnostic SNVs and/or indels and those without. The P-values were calculated using a two-sided Fisher exact test. b Comparison of maternal and paternal age at birth between probands with dnSVs and those without dnSVs. The P-values were calculated using a one-sided t test. c Frequency of dnSVs classes. The box plots display the median (centre line) and interquartile range (25th to 75th percentile; boundaries of the box), with whiskers indicating minima and maxima. d Timing of duplications from maternal origin. Fraction of duplications according to the timing. The P-value was calculated using a two-tailed exact binomial test. e Frequency of complex dnSVs classes. a–g Schematic of major dnSV classes with copy numbers and read patterns (top). The schematic segments in blue, red, and orange represent deletion, duplication, and inversion, respectively. Integrative Genomics Viewer (IGV) screenshots illustrate dnSVs in probands (bottom). The potential causal genes affected by dnSVs are marked below the screenshots. g Distribution of mRNA expression level of the SRRM2 gene from 5546 samples (right). The expression level from the proband (left) is indicated by the red arrow (right). The P-value was calculated from a one-tailed z-score test. Using 1870 high-confidence dnSVs from 1696 probands (91% of probands had a single SV; Supplementary Fig. 1b), we estimated an overall mutation rate of 0.13 events per genome per generation, in line with previous reports24,25,26 (Supplementary Fig. The rate of dnSVs varies across the rare disorder categories; such that probands with NN disorders and those with cardiovascular disorders exhibit the highest dnSV rate (0.15 event per genome), whereas probands with ophthalmological and hearing and ear disorders show the lowest (0.1 event per genome; Supplementary Fig. It is worth noting that the rate of dnSVs is marginally higher in the probands (0.13 event per genome) than in unaffected siblings (n = 207; 0.09 event per genome; P = 0.05). Approximately 12% (n = 1696) of the probands harboured at least one dnSVs. We identified 4 individuals with a considerably higher number of dnSVs (n \(\ge\) 4). These individuals, recruited under different rare disease categories, are not among the previously reported germline SNV hypermutators27 and have no known history of parental exposure to chemotherapy. Unlike the known multiple dnCNVs phenomenon that shows a predominance of copy number gain28,29, 88% of the identified dnSVs in these individuals were a deletion (median = 1.5 kb), suggesting that further investigation is needed to characterise the multiple dnSVs in these individuals. We found a statistically significant positive correlation between the number of dnSVs and de novo SNVs/indels (P = 3.87E-07; Supplementary Fig. 5a), which is partly explained by the parental age effect26,30. However, the mechanistic basis of this correlation remains unclear. Interestingly, we observed a greater enrichment of dnSVs in probands without diagnostic SNVs/indels compared to those with diagnostic SNVs/indels (P < 5.00E-02; Fig. 1a), suggesting that a significant proportion of unsolved cases is likely to be explained by dnSVs. We also found a parental-age effect on the occurrence of dnSVs (Fig. Overall, we identified a significant increase in parental age at birth in probands with dnSVs compared to those without (P < 5.00E-02). Among the rare disorder classes, a significant difference in parental age distribution is observed in dysmorphic and congenital abnormality syndromes and skeletal disorders (P < 5.00E-02), while only the association with skeletal disorders remained significant after multiple testing correction (Benjamini-Hochberg corrected P < 0.05; Supplementary Fig. Additionally, we observed 67.8% of the phased dnSVs originated from paternal germ cells (Supplementary Fig. 5c), as a proportion consistent with previous studies on structural variation (66–74.4%)25,26. This finding aligns with the well-documented paternal bias in de novo SNVs and indels, reinforcing the broader trend of increased germline mutagenesis in the male lineage31. Among the different classes of dnSVs (Fig. 1c), simple deletion (n = 1377; 73.6%) was the most common, followed by tandem duplication (n = 245; 13.1%). The median detected deletion and tandem duplication sizes are 3.7 kb (range 52 bp − 61 Mb) and 49 kb (range 135 bp − 154 Mb), respectively (Supplementary Fig. Furthermore, we identified other classes such as complex SVs (n = 158; 8.4%), reciprocal inversion (n = 49; 2.6%; Fig. 2a), reciprocal translocation (n = 30; 1.6%; Fig. 2b), and templated insertion (n = 6; 0.7%; Fig. The representative probands with simple SVs disrupting phenotype-relevant genes are shown in Fig. For example, we identified a templated insertion that disrupted MECP2, which has a well-established function in neurodevelopment32 in probands with NN disorders. This gene is known to be recurrently affected by dnSVs10 (Fig. This was independently validated by long-read sequencing (Supplementary Fig. We inferred the timing of maternally derived duplication formation into meiosis I and II based on the fact that heterologous allele duplications are known to occur only before the separation of homologous chromosomes during meiosis I. In contrast, homologous allele duplications are known to occur before the separation of sister chromatids during meiosis II33. We identified 41 cases of dnSVs with duplications, which comprise 30 tandem duplications and 11 complex SVs (median = 380 kb; range 38 kb – 40 Mb), for timing analysis. We classified the timing of duplication of maternal origin into meiosis I and II (Methods and Supplementary Fig. This classification revealed that 85% of duplications in this cohort originated from maternal meiosis II (P = 4.87E-06; Fig. Further investigations using larger cohorts are required to confirm which step of meiosis contributes more significantly to dnSVs33,34. Notably, the third most common type of dnSVs is complex SVs. We further classified complex SVs into nine major classes (Fig. The most common class, termed ‘Loss-Loss' (n = 18;11.4%), comprised two adjacent deletions (Fig. For instance, the two adjacent deletions (2 kb and 3 kb in length) in a proband with an NN disorder affected two exons in CNOT2 (Fig. 2d) for which haploinsufficiency is known to cause a neurodevelopmental disorder with characteristic facial features9. In addition, adjacent deletions (5 kb and 1.7 kb in length) in a proband disrupted exon 2 of AMER1 (Supplementary Fig. 2c), for which deficiency is associated with osteopathia striata with cranial sclerosis35. Other classes comprising inversion and deletion are ‘Inv-Loss' (i.e., inversion with flanking deletion; n = 14;8.9%; Fig. 2e) and ‘Loss-Inv-Loss' (i.e., paired deletion inversion; n = 12; 7.6%; Fig. The representative cases with these types of complex SVs disrupting renal and urinary tract disorder-36 and NN disorder-related genes37, such as KMT2A, AFF2, FMR1, and SRRM2, are shown in Fig. We observed significantly reduced mRNA expression of SRRM2 (P = 4.00E-04), disrupted by ‘Loss-Inv-Loss' in a proband with an NN disorder (Fig. Another commonly observed class, termed ‘Loss-invDup' (n = 14; 8.9%), is characterised by copy-number loss plus a nearby duplication linked by inverted rearrangements. For instance, a ‘Loss-invDup' in a proband with NN affected an exon in AUTS2 (Fig. 3a), which has been implicated in neurodevelopment and as a candidate pathogenic gene for numerous neurological disorders38. Another class, ‘Deletion bridge' (i.e., bridge of templated insertion; n = 7; Fig. 7a, b), led to large deletions (15 Mb in chromosome X in Fig. 7b) containing genes involved in neurodevelopment39,40, such as GALNT2, MECP2, and CTNNB1, in probands with NN disorders (Supplementary Fig. ‘Translocation-Loss' (n = 3) led to deletions on either one chromosome (Fig. 3c) or both chromosomes (Supplementary Fig. 7c), resulting in the disruption of phenotype-relevant genes such as ARID1B41 in a proband with an NN disorder. a–e Schematic of major dnSV classes with copy numbers and read patterns (top). The schematic segments in blue, red, and orange represent deletion, duplication, and inversion, respectively. IGV screenshots illustrate dnSVs in probands (bottom). The potential causal genes affected by dnSVs are marked below the screenshots. d, e Unclassified types of complex dnSVs. The schematic segments (d, e) and coverage plot (e) are shown above IGV screenshots displaying long-read (top) and short-read (bottom) in probands. The other remaining classes, comprising duplication and inversion, are ‘DUP-NML-DUP'3,42 (i.e., paired duplication inversion or Dup-invDup; n = 7; 4.4%; Supplementary Fig. 8) and ‘DUP-TRP/INV-DUP'15 (i.e., Dup-Trp-Dup; n = 14; 8.9%; Supplementary Figs. 7d and 8), exhibit structures involving two duplications linked by inverted rearrangements and duplication–inverted-triplication–duplication, respectively. Interestingly, beyond local-2 jumps (i.e., clusters of two rearrangements) as described above, we also found three instances of ‘DUP-NML-DUP-NML-DUP'43,44,45 (i.e., known as local-3-jumps in the cancer field2) involving three local rearrangements (Supplementary Fig. Furthermore, other complex duplications, such as dispersed (n = 8; 5.1%) and inverted duplications (n = 2; 1.3%), were also observed. Although most pathogenic effects of these complex SV types involving duplication arise from overexpression of triplosensitive genes15,44 (i.e., gain-of-function), these variant types have been reported to cause disease by loss-of-function mechanisms such as gene disruption46, gene fusion at breakpoints47, and segmental uniparental disomy48. The complex SVs that did not fit into the described classes were categorised as ‘Unclassified' (n = 56; 35.4%). Two of the complex SVs under the “Unclassified” category had long-read data that enabled us to resolve their genomic configuration (Fig. 3d) had a deletion-inversion-deletion-inversion-deletion structure, which affected several exons in EFTUD2, for which deficiency is likely to lead to craniofacial anomalies49. Another case has a structure of duplication followed by ‘Loss-Inv-Loss' which disrupted the phenotype-relevant gene, EDA50 (Fig. In addition, a case with a small deletion (3 kb in chr22q.13.33) within a large deletion (20 kb in chr22q.13.33), where one of the breakpoints was the same, in a proband with an NN disorder disrupted SHANK3 associated with a broad spectrum of neurodevelopmental disorders51 (Supplementary Fig. Collectively, these results highlight the complex nature of dnSVs in rare disorders. Overall, our analysis reveals that among probands with dnSVs, 9% (145/1696) exhibit exon-disrupting pathogenic dnSVs associated with the probands' phenotype. Notably, 66 of these 145 (46%) pathogenic SVs were balanced rearrangements (e.g., reciprocal inversion) or CNVs affecting <3 exons that cannot be reliably detected by array-based or whole-exome sequencing methods (Fig. 4a and Supplementary Data 2), highlighting the importance of WGS-based genetic testing in routine clinical care. a Percentage of probands with pathogenic exon-disrupting dnSVs according to detectability by array-based / whole-exome sequencing. b Percentage of probands with pathogenic exon-disrupting dnSVs by SV type. c Percentage of dnCNVs identified by array-based / whole-exome sequencing by SV type. d Over-representation of intronic dnSVs in known pathogenic genes in NN disorders. A window of 50 kb up- and downstream was added to each intergenic dnSV for this enrichment analysis. The P-value was estimated using a one-sided permutation test. In our study, we observed that 1.4% of probands with NN disorders harboured complex dnSVs, a prevalence about two times higher than in a previous autism spectrum disorder study26 (0.76%; P = 2.00E-02; two-sided Fisher exact test). Notably, approximately 12% of pathogenic dnSVs in our dataset were identified as complex events (Fig. 4b), highlighting their significant role in rare disorders despite their lower frequency. Moreover, among probands with array-based or whole-exome sequencing data available, 22% of de novo CNVs identified by these data (8/37) were complex SVs, which were previously misclassified as simple dnSVs (Fig. Furthermore, our investigation reveals distinctive patterns within intronic and intergenic dnSVs among probands with NN disorders. Intronic dnSVs showed a significant enrichment in known pathogenic genes associated with NN disorders in the G2P database52 (P = 1.00E-03). In contrast, intergenic dnSVs, when assessed for genes within a 50 kb range up- and downstream of the dnSVs (“Methods”), did not show such an association, suggesting the potential pathogenic role of intronic dnSVs in rare disorders (Fig. Additional studies using RNA-seq and/or CRISPR/Cas-9 genome editing are needed to elucidate the functional impact of these intronic dnSVs on mRNA splicing and expression. In exploring genomic properties of dnSVs, we observed a prevalent distribution of de novo deletions (dnDELs) and tandem duplications (dnTDs) in gene-dense areas (Supplementary Fig. However, smaller dnDELs (< 10 kb) are enriched in early-replicating regions (P = 1.00E-03; Supplementary Fig. 10a), which is inconsistent with previous reports2. We observed that the majority of dnSVs, primarily simple dnDELs, exhibit enrichment at the subtelomeric regions across autosomes (Supplementary Fig. We also identified a positive association between the number of subtelomeric dnDELs and early replication regions, especially when they were within 15 Mb of telomere ends (Spearman's rho = 0.56, P = 6.46E-03; Supplementary Fig. In total, the density of dnDELs within 15 Mb of telomere ends (i.e., 1.3/Mb) is 2.8 times greater than the autosome-wide average (i.e., 0.457/Mb). We observed a distinctive sex difference in patterns of dnSVs, specifically, maternal dnSVs were enriched for larger deletions, while paternal dnSVs were enriched for smaller deletions (P = 4.99E-05; Fig. We further confirmed a similar enrichment pattern using an independent dataset26 (P = 1.63E-03; Fig. This gender-specific difference is potentially in line with a higher incidence of aneuploidy in oocytes than in sperm53. The higher rate of aneuploidy is known to be associated with the distinct features of oocytes54, such as the architecture of the meiotic spindle, the level of cortical tension at the oocyte surface, weaknesses in surveillance mechanisms that monitor chromosome segregation, and environmental factors. Additionally, we found that maternal dnDELs are enriched at the subtelomeric regions (< 15 Mb) of chromosome 16 (Fig. We observed similar maternal enrichment of dnDELs in subtelomeric regions of chromosome 16 in an independent cohort26 (Fig. 5b), suggesting possible sex-specific mechanisms in the generation of dnSVs. a Comparison of the size of dnDELs according to parent of origin in the GEL cohort (left) and CEPH & SFARI cohort (right). The P-values were calculated using a two-sample Kolmogorov-Smirnov test. b Proportion of dnDELs within 15 Mb of telomere ends according to parent of origin in the GEL (left) and in the CEPH & SFARI cohort (right). Our investigation provides a substantial advancement in the understanding of de novo structural variants in rare disorders, encompassing an extensive cohort of 13,702 parent–child trios. In particular, we provide insights into the role of complex SVs in the aetiology of rare disorders. The prevalence of dnSVs, affecting 12% of probands, highlights the importance of integrating these variants into the broader spectrum of genetic factors contributing to rare disorders. Unlike conventional cytogenetic methods, such as array Comparative Genomic Hybridisation (CGH)-based technologies, WGS offers unparalleled precision in characterising the genomic configuration of complex dnSVs. This is particularly crucial as some simple deletions and insertions may be integral components of complex SVs often overlooked by array-based / whole-exome sequencing methods. For instance, in 37 cases where array and or whole-exome sequencing data were available, we found that 8 complex dnSVs (22%) were misclassified as simple dnSVs. In addition, 51 of 151 (34%) pathogenic SVs identified in our study were balanced rearrangements (e.g., balanced inversion) or CNVs affecting \(\le 2\) exons that can't be detected by array-based / whole-exome sequencing methods, highlighting the importance of WGS-based genetic testing in routine clinical care. Notably, dnSVs exhibited non-random distribution patterns, showing enrichment in specific genomic locations associated with distinct features depending on the parent of origin. Strikingly, we observed an enrichment of maternal dnDELs within 15 Mb of the telomeres of chromosome 16. This enrichment positively correlates with skewed early replication regions across chromosomes. While the genomic basis for this maternal bias in subtelomeric regions of chromosome 16 is unknown, previous reports have suggested potential explanations55. These include early subtelomeric replication in meiosis56, increased rates of meiotic double-strand breaks in the distal parts of chromosomes57, or biased maternal non-crossover gene conversion58. Overall, these findings indicate the need for further investigation into parental influence and region-specific impacts on disease manifestation. We note several limitations in our approach, which open potential avenues for future investigations. Complex structural variants are still underestimated in our cohort because of the inherent limitations of short-read-based SV discovery. For example, SVs in repeat-rich regions (e.g., segmental duplications or retrotransposons) remain challenging to identify. Although we identified some of these SVs using read-depth-based algorithms (e.g., CANVAS), resolving the genomic configuration of complex SVs using read-depth-specific calls that do not provide read orientation, along with read-pattern-based calls, is challenging. Furthermore, short-read sequencing is known to fail to capture large SVs, especially large insertions. Due to the lower detection sensitivity of de novo retrotransposition, with the current pipeline, these variants have been excluded from our analysis. Specifically, we observed a rate of 0.01 events per genome, which is far lower than expected (0.03-0.038 per genome26,59). Future research could explore techniques such as long-read sequencing60 to enhance our ability to detect dnSVs in repetitive regions61. Finally, the systematic identification of gene duplication leading to triplosensitivity62 and inherited pathogenic SVs will be needed to facilitate more comprehensive diagnostics. Overall, our findings expand the understanding of dnSVs in rare disorders and highlight the need for ongoing research to unravel the complexities of their contribution to the aetiology of rare disorders and the potential for clinical application. We used available data from the rare disease cohort of the 100,000 Genomes Project generated by Genomics England21. The 100,000 Genomes Project was approved by the East of England—Cambridge Central Research Ethics Committee (REF 20/EE/0035). Genomic blood DNA libraries were prepared using the Illumina TruSeq PCR-free protocol, and whole-genome sequenced on the Illumina HiSeq X platform (2 × 150-bp paired-end reads). Read alignment and SV calling using Isaac63 and Manta22, respectively, were performed by the Genomics England Bioinformatics team21. The details of sequencing and variant calling have been previously described21. Manta VCF files were converted to BEDPE format using SVtools (v0.5.1)64 and then BEDTools (v2.31.0)65 was utilised to extract proband-specific SVs with ≥ 50 bp in length for each family (Supplementary Fig. We first removed SVs on Y chromosome and further removed SVs with evidence of clipped reads (i.e., split reads) at breakpoints in either parent. Specifically, SVs supported by ≥ 4 clipped reads at either breakpoint or ≥ 2 clipped reads at both breakpoints in either parent were excluded. SVs found in 3 or more samples were removed because such SVs were likely alignment artefacts. We selected SVs flagged as “PASS” or “MGE10kb” (i.e., Manta calls with length < 10 kb) and further narrowed down SVs with the Manta score > 30 and supporting discordant reads > 10. We rescued SVs with imprecise breakpoints if they were supported by CANVAS66. In detail, SVs tagged as “IMPRECISE” (i.e., imprecise breakpoints) were rescued if they were also flagged as “ColocalizedCanvas”. We excluded SVs with VAF < 0.1 (n = 10) to remove mosaic SVs. Translocation through retrotransposon-mediated 3′ transduction was excluded to focus on dnSVs. All SVs were manually validated to identify high-confidence de novo SVs using the IGV browser67. Specifically, we visually validated the presence of abnormal reads (i.e., discordant/split reads) in probands and the absence of such reads in their parents. We additionally called CNVs (i.e., deletion and duplication) with > 10 kb using CANVAS to identify more diagnostic CNVs. Read-depth-specific calls were only used to assess their pathogenicity because clustering and classification (i.e., resolving genomic configuration of complex SVs) of read-depth calls that did not have read-orientation were challenging. Long-read sequencing data (23 resequenced samples with PacBio technology), RNA-seq (whole blood RNA sequencing from 5,546 samples) and diagnostic SNV/Indels were obtained from Genomic England21. Previous dnCNV calls from array-based or whole-exome sequencing were obtained from the DDD study cohort23. Insertion events called by Manta22 were further classified into retrotranspositions using RepeatMasker. This process resulted in a lower sensitivity because Manta with Issac-based alignment is not optimised to call retrotranspositions. Retrotransposition-specific identification tools, such as MELT68 or xTea69, are needed to increase sensitivity for retrotransposition detection. To estimate the proportion of probands with complex SVs in the previous study26, we obtained 19 probands with complex SVs where “sv_type” was “CPX” and “role” was “proband”. We used ClusterSV2 (https://github.com/cancerit/ClusterSV) to group rearrangements (i.e., breakpoints), into rearrangement clusters. The key advantage of ClusterSV is to identify clusters of dispersed breakpoints45 without requiring a predefined distance threshold, allowing for a data-driven detection of complex genomic rearrangements. We defined complex SVs as those with ≥ 2 clustered breakpoints except for simple SVs involving reciprocal inversion, balanced translocation, templated insertion, and dispersed duplication. In general, we classified the types of complex SVs according to the previous study that comprehensively characterised somatic complex SVs using thousands of cancer genomes2. In short, complex SVs involving two inversions were categorised into Loss-invDup, DUP-TRP/INV-DUP, Inv-LossDU (i.e., inversion with flanking deletion), Loss-invLoss (i.e., paired deletion inversion), and DUP-NML-DUP (i.e., paired duplication inversion) according to read patterns and copy numbers (Supplementary Fig. Complex SVs involving two deletions were classified as Loss-Loss. Bridge deletion (i.e., bridge of templated insertion) and Translocation-Loss (i.e., translocation with deletion) were classified using the previously described criteria2,70. DUP-NML-DUP-NML-DUP (i.e., Local-3 jumps) involving three local rearrangements were discovered according to the read patterns described in the previous cancer study2. Breakpoints filtered out near unresolved SV classes were rescued if they could resolve the configuration of unresolved SV classes according to the types of SV defined. For the remaining unresolved SV, CANVAS calls were used to resolve their genomic configuration manually. Complex SVs that did not fit into the described classes were categorised as ‘Unclassified'. All complex SVs were manually validated using IGV browser (2.18.2)67, Samplot (v1.3.0)71, or BamSnap (v0.2.19)72. We used unfazed (v1.0.2)73, which employs both extended read-backed- and SNV allele-balance- phasing, to identify the parent of origin for dnSVs. Haplotypecaller (v3.3)74 was utilised to make an input for unfazed. Phasable dnSVs (51%; 962/1870) were used for downstream analysis concerning the parent of origin. To classify the timing of maternally derived duplication into meiosis I and II, we first identified duplication (including those in complex SVs) from maternal origin (step 1) and further classified them into meiosis I and II (step 2) using a set of informative genotypes (Supplementary Fig. 6) For binary classification at each step, the ratio of the number of SNPs supporting one class to another class was calculated, and a class for which the ratio was greater than 0.9 was chosen, At least three SNPs were required for either class at each step. These filtering criteria could time large duplications with a handful of erroneously called SNPs and remove ambiguous duplications such as those originating from both parents during early development (e.g., potentially due to mitotic crossing-over). The timing of paternally derived duplications was not inferred because duplications can also occur in a premeiotic state during male gametogenesis throughout life. The identified SVs disrupting exons were reviewed for potential clinical relevance by NHS clinical scientists and/or Genomics England. We considered SVs as being potential (likely) pathogenic SVs if at least one of the following criteria were fulfilled: (i) the variant had been clinically assessed as likely pathogenic or pathogenic by an NHS genomic laboratory hub. In detail, the variant had been assessed by an NHS clinical laboratory according to the best practice guidelines recommended by the Association for Clinical Genomic Science (ACGS) as being likely pathogenic or pathogenic and related to the primary phenotype for which the participant was recruited to the 100,000 Genomes Project. (ii) the variant had been reviewed on a research basis and considered to be a strong candidate diagnostic variant. In the research review, variants resulting in loss of function for genes in which haploinsufficiency is a known mechanism of disease or variants resulting in loss of a critical domain impacting genes associated with a phenotype relevant to the primary clinical indication were considered as candidate diagnostic variants. We classified pathogenic SVs annotated with “reciprocal inversion” or “reciprocal translocation” as not likely to be detected by array-based or whole-exome sequencing methods. In addition, the SVs involving deletions spanning < 3 exons of GENCODE canonical transcript75 were classified into this category. We first extracted the non-coding dnSVs (i.e., intronic and intergenic dnSVs) for which genomic coordinates did not include any exons in NN disorders based on the Gencode basic V45 GTF file and then obtained the known pathogenic genes associated with NN disorders from the Gene2Phenotype developmental disorders panel52. Specifically, we kept all genes with organ specificity equal to “Brain/Cognition”, allelic requirement equal to “monoallelic_autosomal”, and a confidence category equal to “strong” or “definitive” (n = 190 genes). We then computed the observed-over-expected ratio for the overlap between the non-coding dnSVs and known pathogenic genes in NN disorders using the Genome Association Tester software76. Intronic and intergenic regions were obtained based on the canonical transcript of protein-coding genes in the Gencode basic V45 GTF file using BioMart and GencoDymo R packages, and bedtools65. These two regions were used as a workspace in GAT to test the over-representation of dnSVs in intronic and/or intergenic regions of known pathogenic genes. We added a window of 5–500 kb (5, 10, 25, 50, and 500 kb) up- and downstream to each intergenic dnSVs to perform the enrichment test. The number of random samples (“--num-samples”) for each GAT run was set to 1000. We equally partitioned the genome into 5 Mb bins based on their distance to the telomere ends. For comparison, we also partitioned the genome based on their distance to the centromeres. For the validation cohort, we downloaded the all_dnsv.csv file from Belyeu et al. 26. In total, there are n = 309 CEPH and SFAI dnSVs across autosomes after removing chrX and chrY. Finally, only n = 192 dnDELs were used in the validation analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The data used in this study can be accessed via the Genomics England Research Environment, a secure cloud workspace. The raw data, including patient profiles and corresponding genomic sequencing data, are only available under restricted access for patient privacy reasons. Access can be obtained by first applying to become a member of either the Genomics England Research Network (https://www.genomicsengland.co.uk/research/academic) or the Discovery Forum (industry partners https://www.genomicsengland.co.uk/research/research­ environment). The process for joining the network is described at https://www.genomicsengland.co.uk/research/academic/join-gecip. Custom Python and R scripts for data analysis can be found at https://github.com/hj6-sanger/GEL_SV (version 1.0.0, https://doi.org/10.5281/zenodo.17093113). Global variation in copy number in the human genome. Patterns of somatic structural variation in human cancer genomes. Schuy, J., Grochowski, C. M., Carvalho, C. M. B. Complex genomic rearrangements: an underestimated cause of rare diseases. Lupski, J. R. Structural variation mutagenesis of the human genome: Impact on disease and evolution. Lupski, J. R. et al. Gene dosage is a mechanism for Charcot-Marie-Tooth disease type 1A. Potocki, L. et al. Molecular mechanism for duplication 17p11.2- the homologous recombination reciprocal of the Smith-Magenis microdeletion. Characterization of Potocki-Lupski syndrome (dup(17)(p11.2p11.2)) and delineation of a dosage-sensitive critical interval that can convey an autism phenotype. Stefansson, H. et al. Large recurrent microdeletions associated with schizophrenia. Lupski, J. R. Schizophrenia: Incriminating genomic evidence. Gardner, E. J. et al. Detecting cryptic clinically relevant structural variation in exome-sequencing data increases diagnostic yield for developmental disorders. Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies. Liu, P., Carvalho, C. M. B., Hastings, P. J. & Lupski, J. R. Mechanisms for recurrent and complex human genomic rearrangements. Sanchis-Juan, A. et al. Complex structural variants in Mendelian disorders: identification and breakpoint resolution using short- and long-read genome sequencing. Most large structural variants in cancer genomes can be detected without long reads. Grochowski, C. M. et al. Inverted triplications formed by iterative template switches generate structural variant diversity at genomic disorder loci. Challenges and standards in integrating surveys of structural variation. Conrad, D. F. et al. Mutation spectrum revealed by breakpoint sequencing of human germline CNVs. Mills, R. E. et al. Mapping copy number variation by population-scale genome sequencing. Collins, R. L. et al. A structural variation reference for medical and population genetics. Kernohan, K. D. & Boycott, K. M. The expanding diagnostic toolbox for rare genetic diseases. Turro, E. et al. Whole-genome sequencing of patients with rare diseases in a national health system. Chen, X. et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. Werling, D. M. et al. An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder. Brandler, W. M. et al. Paternally inherited cis-regulatory structural variants are associated with autism. Belyeu, J. R. et al. De novo structural mutation rates and gamete-of-origin biases revealed through genome sequencing of 2396 families. Kaplanis, J. et al. Genetic and chemotherapeutic influences on germline hypermutation. Liu, P. et al. An organismal CNV mutator phenotype restricted to early human development. The multiple de novo copy number variant (MdnCNV) phenomenon presents with peri-zygotic DNA mutational signatures and multilocus pathogenic variation. Girard, S. L. et al. Paternal age explains a major portion of de novo germline mutation rate variability in healthy individuals. Kong, A. et al. Rate of de novo mutations and the importance of father's age to disease risk. Gonzales, M. L. & LaSalle, J. M. The role of MeCP2 in brain development and neurodevelopmental disorders. Ma, R. et al. A clear bias in parental origin of de novo pathogenic CNVs related to intellectual disability, developmental delay and multiple congenital anomalies. Ariad, D. et al. Aberrant landscapes of maternal meiotic crossovers contribute to aneuploidies in human embryos. Gear, R. & Savarirayan, R. Osteopathia Striata with Cranial Sclerosis. Adam, M. P. et al.) (University of Washington, Seattle, 1993). Spectrum of congenital anomalies of the kidney and urinary tract (CAKUT) including renal parenchymal malformations during fetal life and the implementation of prenatal exome sequencing (WES). & van den Berg, D. L. C. Transcription pause and escape in neurodevelopmental disorders. Hori, K., Shimaoka, K. & Hoshino, M. AUTS2 gene: keys to understanding the pathogenesis of neurodevelopmental disorders. Novel congenital disorder of O-linked glycosylation caused by GALNT2 loss of function. Zhuang, W., Ye, T., Wang, W., Song, W. & Tan, T. CTNNB1 in neurodevelopmental disorders. Moffat, J. J., Smith, A. L., Jung, E.-M., Ka, M. & Kim, W.-Y. Neurobiology of ARID1B haploinsufficiency related to neurodevelopmental and psychiatric disorders. Gu, S. et al. Alu-mediated diverse and complex pathogenic copy-number variants within human chromosome 17 at p13.3. Beck, C. R. et al. Megabase length hypermutation accompanies human structural variation at 17p11.2. Distinct patterns of complex rearrangements and a mutational signature of microhomeology are frequently observed in PLP1 copy number gain structural variants. Carvalho, C. M. B. et al. Replicative mechanisms for CNV formation are error prone. Ishmukhametova, A. et al. Dissecting the structure and mechanism of a complex duplication-triplication rearrangement in the DMD gene. Zuccherato, L. W., Alleva, B., Whiters, M. A., Carvalho, C. M. B. & Lupski, J. R. Chimeric transcripts resulting from complex duplications in chromosome Xq28. Carvalho, C. M. B. et al. Interchromosomal template-switching as a novel molecular mechanism for imprinting perturbations associated with Temple syndrome. Wu, J. et al. EFTUD2 gene deficiency disrupts osteoblast maturation and inhibits chondrocyte differentiation via activation of the p53 signaling pathway. Deshmukh, S. & Prashanth, S. Ectodermal dysplasia: a genetic review. Accogli, A. et al. SHANK3 mutation and mosaic turner syndrome in a female patient with intellectual disability and psychiatric features. Flexible and scalable diagnostic filtering of genomic variants using G2P with Ensembl VEP. Insights into variation in meiosis from 31,228 human sperm genomes. & Schuh, M. Aneuploidy in mammalian oocytes and the impact of maternal ageing. Audano, P. A. et al. Characterizing the major structural variant alleles of the human genome. Pratto, F. et al. Meiotic recombination mirrors patterns of germline replication in mice and humans. Pratto, F., et al. DNA recombination. Recombination initiation maps of individual human genomes. Jónsson, H. et al. Parental influence on human germline de novo mutations in 1,548 trios from Iceland. Borges-Monroy, R. et al. Whole-genome analysis reveals the contribution of non-coding de novo transposon insertions to autism spectrum disorder. Mitsuhashi, S. & Matsumoto, N. Long-read sequencing for rare human genetic diseases. Kim, J. et al. Patient-Customized Oligonucleotide Therapy for a Rare Genetic Disease. Collins, R. L. et al. A cross-disorder dosage sensitivity map of the human genome. Raczy, C. et al. Isaac: ultra-fast whole-genome secondary analysis on Illumina sequencing platforms. Larson, D. E. et al. svtools: population-scale analysis of structural variation. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Roller, E., Ivakhno, S., Lee, S., Royce, T. & Tanner, S. Canvas: versatile and scalable detection of copy number variants. Robinson, J. T. et al. Integrative genomics viewer. The Mobile Element Locator Tool (MELT): population-scale mobile element discovery and biology. Chu, C. et al. Comprehensive identification of transposable element insertions using multiple sequencing technologies. Baca, S. C. et al. Punctuated evolution of prostate cancer genomes. Belyeu, J. R. et al. Samplot: a platform for structural variant visual validation and automated filtering. Kwon, M., Lee, S., Berselli, M., Chu, C. & Park, P. J. BamSnap: a lightweight viewer for sequencing reads in BAM files. Belyeu, J. R., Sasani, T. A., Pedersen, B. S. & Quinlan, A. R. Unfazed: parent-of-origin detection for large and small de novo variants. Van der Auwera, G. A. et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Mudge, J. M. et al. GENCODE 2025: reference gene annotation for human and mouse. Heger, A., Webber, C., Goodson, M., Ponting, C. P. & Lunter, G. GAT: a simulation framework for testing the association of genomic intervals. We thank the families and their clinicians for their participation and engagement, and our colleagues who assisted in the generation and processing of data. We would like to thank Ana Lisa Taylor Tavares for helpful discussions and advice. This research was made possible through access to the data and findings generated by the 100,000 Genomes Project. The 100,000 Genomes Project is managed by Genomics England Limited (a wholly owned company of the Department of Health and Social Care). The 100,000 Genomes Project is funded by the National Institute for Health Research and NHS England. The Wellcome Trust, Cancer Research UK, and the Medical Research Council have also funded research infrastructure. The 100,000 Genomes Project uses data provided by patients and collected by the National Health Service as part of their care and support. This research was supported in part by a Wellcome Trust grant. was supported by RCUK | Medical Research Council (MRC) (MR/W025353/1) and Cancer Research UK (CRUK). Hyunchul Jung, Tsun-Po Yang, Petr Danecek, O. Isaac Garcia-Salinas, Matthew D. C. Neville, Joseph Christopher, Isidro Cortés-Ciriano, Helen Firth, Matthew Hurles, Peter Campbell & Raheleh Rahbari Genomics England, London, UK Department of Clinical Genetics, Cambridge University Hospitals, Cambridge, UK Joseph Christopher & Helen Firth Department of Genomic Medicine, University of Cambridge, Cambridge, UK European Molecular Biology Laboratory, EBI, Hinxton, Cambridge, UK Department of Genetics, University of Cambridge, Downing Street, Cambridge, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the manuscript; all authors reviewed and edited the manuscript. led the analysis of the data with help from J.C., P.D., I.G.S., M.D.C.N., and S.W. reviewed the pathogenic complex dnSV candidates. H.F., helped with clinical interpretation of the dnSVs. helped with data interpretation and statistical analysis. Correspondence to Hyunchul Jung or Raheleh Rahbari. is a co-founder, shareholder, and consultant for Quotient Therapeutics, M.E.H. is a co-founder of, consultant to and holds shares in Congenica, a genetics diagnostic company. All remaining authors declare no competing interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Jung, H., Yang, TP., Walker, S. et al. Complex de novo structural variants are an underestimated cause of rare disorders. Version of record: 03 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02867-7'>Growth-coupled microbial biosynthesis of the animal pigment xanthommatin</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 10:13:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Engineering heterologous natural product pathways in bacteria has achieved broad success but most approaches suffer from low initial production levels that require extensive, resource-heavy iterative strain optimization. Xanthommatin is a structurally complex, color-changing animal ommochrome with material and cosmetic applications, yet production in microbial cell factories has been difficult. Here, we introduce a growth-coupled biosynthetic strategy involving a feedback loop where an excised one-carbon (C1) moiety is used as a driver of bacterial growth, simultaneously boosting bioproduction of the target compound. This broadly applicable, plug-and-play strategy is illustrated by enabling xanthommatin biosynthesis in a 5,10-methylenetetrahydrofolate auxotroph of the platform soil bacterium Pseudomonas putida. In this design, formate released during xanthommatin production relieves the C1 deficiency, thereby effectively coupling bacterial growth to pigment synthesis. Adaptive laboratory evolution streamlined xanthommatin's gram-scale bioproduction from glucose, establishing C1 restoration as a general biosynthetic approach to accelerate the engineering of natural product biosynthesis in bacteria. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Genome sequencing raw files are available from the Sequence Read Archive under BioProject PRJNA1169116. The variant calling data are available in ALEdb version 1.1.0 under project name C1ALE (https://aledb.org/ale/project/257/). All code associated with this work is available on Zenodo (https://doi.org/10.5281/zenodo.17016978)80. Cho, J. S., Kim, G. B., Eun, H., Moon, C. W. & Lee, S. Y. Designing microbial cell factories for the production of chemicals. Google Scholar Cravens, A., Payne, J. & Smolke, C. D. Synthetic biology strategies for microbial biosynthesis of plant natural products. Google Scholar Medema, M. H., de Rond, T. & Moore, B. S. Mining genomes to illuminate the specialized chemistry of life. Google Scholar Zhang, J. J., Tang, X. & Moore, B. S. Genetic platforms for heterologous expression of microbial natural products. Google Scholar Opgenorth, P. et al. Lessons from two design–build–test–learn cycles of dodecanol production in Escherichia coli aided by machine learning. ACS Synth. Google Scholar Choi, K. R. et al. Systems metabolic engineering strategies: integrating systems and synthetic biology with metabolic engineering. Trends Biotechnol. Google Scholar Ko, Y.-S. et al. Tools and strategies of systems metabolic engineering for the development of microbial cell factories for chemical production. Google Scholar Burgard, A. P., Pharkya, P. & Maranas, C. D. Optknock: a bilevel programming framework for identifying gene knockout strategies for microbial strain optimization. Google Scholar Fong, S. S. et al. In silico design and adaptive evolution of Escherichia coli for production of lactic acid. Google Scholar Alper, H., Jin, Y.-S., Moxley, J. F. & Stephanopoulos, G. Identifying gene targets for the metabolic engineering of lycopene biosynthesis in Escherichia coli. Google Scholar Jantama, K. et al. Combining metabolic engineering and metabolic evolution to develop nonrecombinant strains of Escherichia coli C that produce succinate and malate. Google Scholar Otero, J. M. et al. Industrial systems biology of Saccharomyces cerevisiae enables novel succinic acid cell factory. PLoS ONE 8, e54144 (2013). Google Scholar Klamt, S. & Mahadevan, R. On the feasibility of growth-coupled product synthesis in microbial strains. Google Scholar von Kamp, A. & Klamt, S. Growth-coupled overproduction is feasible for almost all metabolites in five major production organisms. Google Scholar Banerjee, D. et al. Genome-scale metabolic rewiring improves titers rates and yields of the non-native product indigoidine at scale. Google Scholar Yim, H. et al. Metabolic engineering of Escherichia coli for direct production of 1,4-butanediol. Google Scholar Dinh, H. V., King, Z. A., Palsson, B. O. & Feist, A. M. Identification of growth-coupled production strains considering protein costs and kinetic variability. Google Scholar Cicchillo, R. M. et al. An unusual carbon–carbon bond cleavage reaction during phosphinothricin biosynthesis. Google Scholar Patteson, J. B. et al. Biosynthesis of fluopsin C, a copper-containing antibiotic from Pseudomonas aeruginosa. Google Scholar Hagel, J. & Facchini, P. Biochemistry and occurrence of O-demethylation in plant metabolism. Augustin, M. M., Augustin, J. M., Brock, J. R. & Kutchan, T. M. Enzyme morphinan N-demethylase for more sustainable opiate processing. Google Scholar Soohoo, A. M., Cogan, D. P., Brodsky, K. L. & Khosla, C. Structure and mechanisms of assembly-line polyketide synthases. Google Scholar Chen, F. Y.-H., Jung, H.-W., Tsuei, C.-Y. & Liao, J. C. Converting Escherichia coli to a synthetic methylotroph growing solely on methanol. Google Scholar Kim, S. et al. Growth of E. coli on formate and methanol via the reductive glycine pathway. Google Scholar Jiang, W. et al. Metabolic engineering strategies to enable microbial utilization of C1 feedstocks. Google Scholar Figon, F. & Casas, J. Ommochromes in invertebrates: biochemistry and cell biology. Article PubMed Google Scholar Kumar, A., Williams, T. L., Martin, C. A., Figueroa-Navedo, A. M. & Deravi, L. F. Xanthommatin-based electrochromic displays inspired by nature. ACS Appl. Google Scholar Martin, C. L. et al. Color-changing paints enabled by photoresponsive combinations of bio-inspired colorants and semiconductors. Google Scholar Sullivan, P. A., Wilson, D. J., Vallon, M., Bower, D. Q. & Deravi, L. F. Inkjet printing bio-inspired electrochromic pixels. Google Scholar Wilson, D. J., Martín-Martínez, F. J. & Deravi, L. F. Wearable light sensors based on unique features of a natural biochrome. Google Scholar Martin, C. A. et al. Biomimetic colorants and coatings designed with cephalopod-inspired nanocomposites. ACS Appl. Bio Mater. Google Scholar A. Martin, C. et al. A bioinspired, photostable UV-filter that protects mammalian cells against UV-induced cellular damage. Google Scholar Deravi, L. F., Cui, I. & Martin, C. A. Using cephalopod-inspired chemistry to extend long-wavelength ultraviolet and visible light protection of mineral sunscreens. Google Scholar Riou, M. & Christidès, J.-P. Cryptic color change in a crab spider (Misumena vatia): identification and quantification of precursors and ommochrome pigments by HPLC. Google Scholar Williams, T. L. et al. Dynamic pigmentary and structural coloration within cephalopod chromatophore organs. Google Scholar Figon, F. et al. Uncyclized xanthommatin is a key ommochrome intermediate in invertebrate coloration. & Thulin, C. D. Ommochrome wing pigments in the monarch butterfly Danaus plexippus (Lepidoptera: Nymphalidae). J. Insect Sci. Huang, G., Song, L., Du, X., Huang, X. & Wei, F. Evolutionary genomics of camouflage innovation in the orchid mantis. Butenandt, A., Schiedt, U. & Biekert, E. Über ommochrome, III. Synthese des xanthommatins. J. Liebigs Ann. & de Lorenzo, V. Pseudomonas putida as a functional chassis for industrial biocatalysis: from native biochemistry to trans-metabolism. Turlin, J., Dronsella, B., De Maria, A., Lindner, S. N. & Nikel, P. I. Integrated rational and evolutionary engineering of genome-reduced Pseudomonas putida strains promotes synthetic formate assimilation. Buchanan, J. L., Rauckhorst, A. J. & Taylor, E. B. 3-hydroxykynurenine is a ROS-inducing cytotoxic tryptophan metabolite that disrupts the TCA cycle. Preprint at bioRxiv https://doi.org/10.1101/2023.07.10.548411 (2023). Lewis-Luján, L. M. et al. Inhibition of pathogenic bacteria and fungi by natural phenoxazinone from octopus ommochrome pigments. Kurnasov, O. et al. Aerobic tryptophan degradation pathway in bacteria: novel kynurenine formamidase. Kurnasov, O. et al. NAD biosynthesis: identification of the tryptophan to quinolinate pathway in bacteria. Matthijs, S. et al. The Pseudomonas siderophore quinolobactin is synthesized from xanthurenic acid, an intermediate of the kynurenine pathway. Le Roes-Hill, M., Goodwin, C. & Burton, S. Phenoxazinone synthase: what's in a name? Trends Biotechnol. Article PubMed Hughes, M. A., Baggs, M. J., al-Dulayymi, J., Baird, M. S. & Williams, P. A. Accumulation of 2-aminophenoxazin-3-one-7-carboxylate during growth of Pseudomonas putida TW3 on 4-nitro-substituted substrates requires 4-hydroxylaminobenzoate lyase (PnbB). et al. Synthesis of cinnabarinic acid by metabolically engineered Pseudomonas chlororaphis GP72. Christen, S., Southwell-Keely, P. T. & Stocker, R. Oxidation of 3-hydroxyanthranilic acid to the phenoxazinone cinnabarinic acid by peroxyl radicals and by compound I of peroxidases or catalase. Martínez-García, E., Nikel, P. I., Aparicio, T. & de Lorenzo, V. Pseudomonas 2.0: genetic upgrading of P. putida KT2440 as an enhanced host for heterologous gene expression. Yishai, O., Goldbach, L., Tenenboim, H., Lindner, S. N. & Bar-Even, A. Engineered assimilation of exogenous and endogenous formate in Escherichia coli. ACS Synth. Marx, C. J., Laukel, M., Vorholt, J. & Lidstrom, M. E. Purification of the formate-tetrahydrofolate ligase from Methylobacterium extorquens AM1 and demonstration of its requirement for methylotrophic growth. Nogales, J. et al. High-quality genome-scale metabolic modelling of Pseudomonas putida highlights its broad metabolic capabilities. & Ebert, B. E. Determination of growth-coupling strategies and their underlying principles. BMC Bioinformatics 20, 447 (2019). Alter, T. B. et al. Metabolic growth-coupling strategies for in vivo enzyme selection systems. Claassens, N. J. et al. Replacing the Calvin cycle with the reductive glycine pathway in Cupriavidus necator. Wirth, N. T., Kozaeva, E. & Nikel, P. I. Accelerated genome engineering of Pseudomonas putida by I-SceI-mediated recombination and CRISPR–Cas9 counterselection. Volke, D. C., Friis, L., Wirth, N. T., Turlin, J. Synthetic control of plasmid replication enables target- and self-curing of vectors and expedites genome engineering of Pseudomonas putida. Volke, D. C., Wirth, N. T. & Nikel, P. I. Rapid genome engineering of Pseudomonas assisted by fluorescent markers and tractable curing of plasmids. Hartmans, S., Smits, J. P., van der Werf, M. J., Volkering, F. & de Bont, J. A. M. Metabolism of styrene oxide and 2-phenylethanol in the styrene-degrading Xanthobacter strain 124X. Matthijs, S. et al. Thioquinolobactin, a Pseudomonas siderophore with antifungal and anti-Pythium activity. Farrow, J. M. & Pesci, E. C. Two distinct pathways supply anthranilate as a precursor of the Pseudomonas quinolone signal. Sazinas, P., Hansen, M. L., Aune, M. I., Fischer, M. H. & Jelsbak, L. A rare thioquinolobactin siderophore present in a bioactive Pseudomonas sp. Genome Biol. Lane, M. C., Alteri, C. J., Smith, S. N. & Mobley, H. L. T. Expression of flagella is coincident with uropathogenic Escherichia coli ascension to the upper urinary tract. Natl Acad. Sandberg, T. E., Salazar, M. J., Weng, L. L., Palsson, B. O. & Feist, A. M. The emergence of adaptive laboratory evolution as an efficient tool for biological discovery and industrial biotechnology. Guzmán, G. I. et al. Enzyme promiscuity shapes adaptation to novel growth substrates. Phaneuf, P. V., Gosting, D., Palsson, B. O. & Feist, A. M. ALEdb 1.0: a database of mutations from adaptive laboratory evolution experimentation. Nucleic Acids Res 47, D1164–D1171 (2019). Article PubMed Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Orsi, E., Claassens, N. J., Nikel, P. I. & Lindner, S. N. Growth-coupled selection of synthetic modules to accelerate cell factory development. Femmer, C., Bechtold, M., Held, M. & Panke, S. In vivo directed enzyme evolution in nanoliter reactors with antimetabolite selection. Luo, H. et al. Directed metabolic pathway evolution enables functional pterin-dependent aromatic-amino-acid hydroxylation in Escherichia coli. ACS Synth. Luo, H. et al. Coupling S-adenosylmethionine–dependent methylation to growth: design and uses. PLoS Biol. Lin, B. et al. Reconstitution of TCA cycle with DAOCS to engineer Escherichia coli into an efficient whole cell catalyst of penicillin G. Proc. Natl Acad. Eggert, C., Temp, U., Dean, J. F. D. & Eriksson, K.-E. L. Laccase-mediated formation of the phenoxazinone derivative, cinnabarinic acid. Gerhart, J. C. & Pardee, A. The enzymology of control by feedback inhibition. Lloyd, C. J. et al. The genetic basis for adaptation of model-designed syntrophic co-cultures. PLoS Comput. Luhavaya, H., Sigrist, R., Chekan, J. R., McKinnie, S. M. K. & Moore, B. S. Biosynthesis of L-4-chlorokynurenine, an antidepressant prodrug and a non-proteinogenic amino acid found in lipopeptide antibiotics. Walsh, C. T., Haynes, S. W. & Ames, B. D. Aminobenzoates as building blocks for natural product assembly lines. Article PubMed Model-based simulation of growth-coupled production of xanthommatin. Zenodo https://doi.org/10.5281/zenodo.17016978 (2025). Ebrahim, A., Lerman, J. A., Palsson, B. O. & Hyduke, D. R. COBRApy: constraints-based reconstruction and analysis for Python. Sandberg, T. E. et al. Evolution of Escherichia coli to 42 °C and subsequent genetic engineering reveals adaptive mechanisms and novel mutations. Deatherage, D. E. & Barrick, J. E. Identification of mutations in laboratory evolved microbes from next-generation sequencing data using breseq. The Galaxy Community The Galaxy platform for accessible, reproducible, and collaborative data analyses: 2024 update. Nucleic Acids Res. Langmead, B., Trapnell, C., Pop, M. & Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Garrison, E., Kronenberg, Z. N., Dawson, E. T., Pedersen, B. S. & Prins, P. A spectrum of free software tools for processing the VCF variant call format: vcflib, bio-vcf, cyvcf2, hts-nim and slivar. PLoS Comput. Dongol, M., El-Nahass, M. M., El-Denglawey, A., Elhady, A. F. & Abuelwafa, A. A. Optical properties of nano 5,10,15,20-tetraphenyl-21H,23H-prophyrin nickel (II) thin films. Kutuzova, S. et al. SmartPeak automates targeted and quantitative metabolomics data processing. Funding was generously provided by the National Institutes of Health (NIH; R01-GM085770 to B.S.M. ), the Office of Naval Research (N000142412649 to B.S.M. ), the Swiss National Science Foundation (postdoctoral fellowship grant number 214366 to L.D. ), the European Union's Horizon 2020 Research and Innovation Programme (grant 101082049 to P.I.N.) and the Novo Nordisk Foundation (NNF10CC1016517, NNF20CC0035580 and NNF18CC0033664 to P.I.N., T.B.A. We thank L. Jelsbak (DTU) for the kind gift of Pseudomonas sp. DTU12.1 and J. Turlin (DTU) and C. de Venecia (UCSD) for their research assistance. This publication includes data generated at the IGM Genomics Center (UCSD) using an Illumina NovaSeq X Plus purchased with NIH funding (S10-OD026929). Present address: Institute of Applied Microbiology, Aachen Biology and Biotechnology, RWTH Aachen University, Aachen, Germany Center for Marine Biotechnology and Biomedicine, Scripps Institution of Oceanography, University of California San Diego, La Jolla, CA, USA Leah B. Bushin, Lara Dürr, Mariah J. Avila & Bradley S. Moore The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Lyngby, Denmark Alter, María V. G. Alván-Vargas, Daniel C. Volke, Òscar Puiggené, Adam M. Feist & Pablo I. Nikel Department of Bioengineering, University of California, San Diego, La Jolla, CA, USA Elina C. Olson & Adam M. Feist Department of Chemistry and Chemical Biology, Northeastern University, Boston, MA, USA Taehwan Kim & Leila F. Deravi Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, La Jolla, CA, USA Bradley S. Moore Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the growth-coupled strategy. conducted all the biochemical, metabolic and genomic experiments, engineered the mutant strains with M.J.A. and isolated Xa with L.D. performed the metabolic modeling simulations. performed the ALE experiments under the supervision of A.M.F. and M.V.G.A.-V. conducted the bioreactor experiments. prepared and tested the synthetic Xa versus biosynthetic Xa. performed isotope labeling experiments. wrote the first draft of the paper with subsequent input from all authors. supervised the project. Correspondence to Pablo I. Nikel or Bradley S. Moore. have filed a joint patent application related to this work. Nature Biotechnology thanks Steffen Lindner and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 2D flux spaces spanned by growth rate and formate uptake rate for a P. putida wild-type strain (iJN1463) and its derivative PUMA including the heterologous formate assimilation pathway and the knockout of the glycine cleavage system (gcvTHP) and serine hydroxymethyltransferase (glyA). b. Flux space of the PUMA strain with a glycine supplementation is additionally shown. c. Metabolic destinations of the derivatives from the formate assimilation in the PUMA metabolic model and a corresponding flux distribution are shown. Fluxes are given relative to an arbitrary formate uptake rate of 100. A glucose minimal medium was applied under aerobic conditions for all simulations. Strains EM42-Kyn (a), PUMA-Kyn (b), EM42-Xa (c) and PUMA-Xa (d) were cultivated in shake flasks filled with 20% (v/v) minimal salt medium (MSM) containing glucose (20 mM), glycine (5 mM) and L-tryptophan (5 mM). Data are presented as mean values ± standard deviation of two biological replicates (n = 2). Abbreviations include L-kynurenine (Kyn), kynurenic acid (KYNA), L-3-hydroxykynurenine (3HK), xanthurenic acid (XANA), xanthommatin (Xa), and decarboxylated Xa (DC-Xa). PUMA is not able to grow in the absence of glycine whereas ‘re-engineered' PUMA mutants show growth comparable to an ePUMA clonal isolate. Data are presented as mean values ± standard deviation of two biological replicates (n = 2). Whole-genome sequencing of clonal isolates from endpoint populations of ePUMA-Kyn and ePUMA-Xa, weaned from L-tryptophan supplementation, revealed mutations in the promoter region controlling expression of qbs biosynthetic genes. Mutant plasmids were isolated from single clones (ALE IDs indicated) of ePUMA-kyn(Win) and ePUMA-Xa(Win) and transformed into the PUMA and ePUMA strains. The PUMA strains harboring the mutant plasmids were able to grow in MSM containing glucose (20 mM) and glycine (5 mM) in the absence of L-tryptophan whereas the PUMA strain carrying the original pS4413-qbsFH or pS4413-qbsFGH plasmid did not. Similarly, ePUMA only grew in MSM containing glucose as the sole carbon source when carrying mutant pS4413-qbsFH and pS4413-qbsFGH plasmids, but not the originals. Data are presented as mean values ± standard deviation of two biological replicates (n = 2). Feeding with [2-13C]glucose yields L-tryptophan (indole-2-¹³C). Labeled L-tryptophan is oxidized by QbsF, generating N-formyl-L-kynurenine, which is hydrolyzed by QbsH, yielding 13C-formate, which is ligated to THF by formate-THF ligase (FtfL), yielding labeled 10-formyl-THF13.C-formate is then incorporated into purine nucleotides by PurH. The 5-phosphoribosyl group of 5-phosphoribosyl diphosphate (PRPP) is transferred to ATP, yielding phosphoribosyl-ATP and inorganic pyrophosphate (not shown). Additional transformations yield N-(5'-Phospho-D-1'-ribulosylformimino)-5-amino-1-(5”-phospho-D-ribosyl)-4-imidazolecarboxamide (PRFAR) and ultimately histidine, which contains a labeled carbon, derived from ATP. Data are presented as Labeling experiments are averages from two biological replicates (n = 2)samples harvested at the end of exponential growth. In all cases standard deviations were ≤0.96%. Strains EM42-Kyn (a), ePUMA-Kyn(Win) (b), EM42-Xa (c) and ePUMA-Xa(Win) (d) were cultivated in shake flasks filled with 20% (v/v) MSM containing 20 mM glucose. Data are presented as mean values ± standard deviation of two biological replicates (n = 2). Abbreviations include L-kynurenine (Kyn), kynurenic acid (KYNA), L-3-hydroxykynurenine (3HK), xanthurenic acid (XANA), xanthommatin (Xa), and decarboxylated Xa (DC-Xa). A growth plot showing glucose concentration over the course of the fermentation. Samples taken every 6 h from the fermentation broth show a visual increase in xanthommatin production as indicated by the yellow to brown color change. c. and d. Fermentation was performed in a 250 mL Ambr microbioreactor. The final culture broth was colored deep burgundy. A P. putida methylenetetrahydrofolate auxotroph was generated from parental strain P. putida EM42 via five genomic modifications, which included deletion of glyA-I, glyA-II, and gcv-II, insertion of a formate assimilation module (P4→mtdA-fch-ftfL) and deletion of gcv-I, in that order. The resulting strain named PUMA was subjected to adaptive laboratory evolution (ALE) to remove its dependence on exogenous glycine for growth in minimal medium. One of the evolved populations of the resulting glycine-independent strain, named ePUMA, was transformed with the pathway plasmids pS4413-qbsFH or pS4413-qbsFGH, to generate ePUMA-Kyn and ePUMA-Xa, respectively. Single transformants were subsequently subjected to ALE in parallel to remove the dependence on exogenous L-tryptophan for growth in minimal medium. The resulting strains ePUMA-Kyn(Win) and ePUMA-Xa(Win) grow in minimal medium containing glucose as the sole carbon source. 1–6, Tables 1–8 and Note 1. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Bushin, L.B., Alter, T.B., Alván-Vargas, M.V.G. et al. Growth-coupled microbial biosynthesis of the animal pigment xanthommatin. Version of record: 03 November 2025 DOI: https://doi.org/10.1038/s41587-025-02867-7 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biotechnology ISSN 1546-1696 (online) ISSN 1087-0156 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251102205021.htm'>Breakthrough blood test finally confirms Chronic Fatigue Syndrome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 07:25:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists from the University of East Anglia and Oxford BioDynamics have created a highly accurate blood test capable of diagnosing Chronic Fatigue Syndrome, also known as Myalgic Encephalomyelitis (ME/CFS). Despite its prevalence, ME/CFS has remained poorly understood and notoriously difficult to diagnose. This new test, which demonstrates 96 percent accuracy, offers renewed hope to patients who have struggled for recognition and reliable answers. Researchers believe the discovery could also lead to a similar test for identifying long Covid. Lead researcher Prof Dmitry Pshezhetskiy of UEA's Norwich Medical School explained, "ME/CFS is a serious and often disabling illness characterized by extreme fatigue that is not relieved by rest. With no definitive tests, many patients have gone undiagnosed or misdiagnosed for years. "Our discovery offers the potential for a simple, accurate blood test to help confirm a diagnosis, which could lead to earlier support and more effective management." "Post-Covid syndrome, commonly referred to as long Covid, is one example of ME/CFS, where a similar cluster of symptoms is triggered by the Covid-19 virus, rather than by other known causes such as glandular fever. We therefore hope that our research will also help pave the way for a similar test to accurately diagnose long Covid." To develop the test, researchers employed Oxford BioDynamics' advanced EpiSwitch® 3D Genomics technology, which examines how DNA is folded within cells. The study analyzed blood samples from 47 people with severe ME/CFS and compared them to 61 healthy volunteers. Each human cell contains around two metres of DNA, intricately folded in three dimensions. Alexandre Akoulitchev, Chief Scientific Officer at Oxford BioDynamics, said, "Chronic Fatigue Syndrome is not a genetic disease you're born with. That's why using EpiSwitch 'epigenetic' markers -- which can change during a person's life, unlike fixed genetic code -- was key to reaching this high level of accuracy. "The EpiSwitch platform behind this test, together with OBD's vast 3D Genomic knowledgebase, has already been proven to deliver practical, rapid blood diagnostics accessible at scale. EpiSwitch technology has previously helped identify blood-based biomarkers for other complex conditions, including fast-progressing ALS (amyotrophic lateral sclerosis), rheumatoid arthritis, and several cancers. This deeper insight could advance scientific understanding of the illness. Researchers also observed signs of immune system involvement and inflammation pathways, suggesting potential biological targets for future therapies. These findings may help determine which patients are most likely to benefit from specific treatments. This research was led by UEA and Oxford BioDynamics in collaboration with The London School of Hygiene & Tropical Medicine and Royal Cornwall Hospitals NHS Trust. 100x More Potent: Chemists Discover Powerful New Antibiotic “Hidden in Plain Sight” Twin Black Hole Collisions Offer the Most Precise Test of Einstein's Theory Yet Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/orcas-repeatedly-attack-young-great-white-sharks-drone-footage-reveals/'>First-Ever Footage Shows Killer Whales Attacking Great White Shark Nursery</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 05:02:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>First-Ever Footage Shows Killer Whales Attacking Great White Shark Nursery An orca pod has been spotted for the first time repeatedly targeting and flipping young great white sharks onto their backs to paralyze and dismember them Orcas, or “killer whales,” are not known for their amiable demeanor. The famous bullies attack boats and pester other animals for reasons that sometimes aren't entirely clear. Researchers watched two hunts via drone while monitoring orcas in the Gulf of California. Newly released footage from August 2020 shows the first clash, during which five orcas teamed up to go great white shark tipping. At that point, the battle was already won. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. An orca strikes a shark in the belly. In both cases, they went to all of that trouble mainly for each shark's liver because the organ is loaded with nutrients and energy. Two years later, in August 2022, the researchers saw a repeat: five orcas flipped over a young great white shark and ate its liver. “This behavior is a testament to orcas' advanced intelligence, strategic thinking, and sophisticated social learning, as the hunting techniques are passed down through generations within their pods,” said marine biologist Erick Higuera Rivas, who led the new paper, in a recent press release. While scientists had previously seen orcas attack great whites, the killer whales mainly targeted adults, both for a larger meal and to get rid of competition for the same prey. That generally works fine because the animals have so few predators. “Nursery areas are where young sharks spend time growing and learning to forage, so displacing them from those habitats can be disruptive,” says Alison Towner, a marine biologist at Rhodes University in South Africa, who specializes in orca predation of sharks but wasn't involved in the new study. But if it becomes repeated, it could force juveniles into less suitable or riskier areas” and ultimately disrupt the entire ecosystem. Ashley Balzer Vigil writes about astrophysics for NASA's Goddard Space Flight Center by day and moonlights as a freelance environmental writer. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251102011213.htm'>Scientists discover 14 strange new species hidden in the deep sea</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 02:14:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Earth's oceans hold extraordinary biodiversity, yet only a small portion of an estimated two million marine species have been formally identified and described. By providing a faster, more standardized system, it dramatically shortens the time needed to publish new species. This speed is essential, as many marine species face extinction due to human-driven biodiversity loss before scientists even learn they exist. SOSA's mission is to foster global cooperation, supply technical resources for species documentation, and promote efficient, transparent taxonomic publishing. In the second major Ocean Species Discoveries collection, more than 20 researchers collaborated to describe 14 new marine invertebrate species and two new genera from across the globe. These included a variety of worms, mollusks, and crustaceans. These tools enable researchers to collect detailed, high-quality data for precise species descriptions. Among them is Veleropilina gretchenae, a newly identified mollusk recovered from the Aleutian Trench at 6,465 meters. It is one of the first members of the class Monoplacophora to have a high-quality genome published directly from its holotype specimen. This marks only the second bivalve species ever documented entirely through non-invasive micro-CT scanning. It is also the first in-depth anatomical description for any Myonera species and sets a new depth record -- found between 5,170 and 5,280 meters, about 800 meters deeper than any previously known specimen. One of the newly described species, Apotectonia senckenbergae, was named in honor of Johanna Rebecca Senckenberg (1716-1743), a naturalist and benefactor whose support for science and medicine helped inspire the formation of the Senckenberg Society for Nature Research. Some of the newly identified animals display peculiar traits. The tusk shell Laevidentalium wiesei, collected from depths exceeding 5,000 meters, was discovered with a sea anemone attached to its anterior (concave) side -- a relationship never before documented within this genus. These discoveries showcase how global collaboration and advanced imaging are transforming marine taxonomy. 100x More Potent: Chemists Discover Powerful New Antibiotic “Hidden in Plain Sight” Scientists Discover “Scary” New Species of Spider in California Ancient Radio Signals Could Reveal What Dark Matter Really Is Twin Black Hole Collisions Offer the Most Precise Test of Einstein's Theory Yet Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251102011206.htm'>Scientists achieve forensics' “Holy Grail” by recovering fingerprints from fired bullets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-03 01:40:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Two Irish scientists have created a groundbreaking method for recovering fingerprints from fired bullet casings -- something long believed to be impossible. Dr. Eithne Dempsey and her former PhD student, Dr. Colm McKeever, from the Department of Chemistry at Maynooth University in Ireland, have designed a novel electrochemical process that reveals fingerprints on brass casings even after exposure to the intense heat generated when a gun is fired. For decades, forensic experts have struggled to retrieve fingerprints from firearms or ammunition. The extreme temperatures, gas, and friction produced during gunfire typically destroy any biological residue. Because of this, many criminals have relied on the assumption that fired weapons and casings could not link them to a crime scene. "The Holy Grail in forensic investigation has always been retrieving prints from fired ammunition casings," said Dr. Dempsey. However, our technique has been able to reveal fingerprint ridges that would otherwise remain imperceptible." The research team discovered that coating brass casings with a thin layer of specially chosen materials can expose hidden fingerprint ridges. Unlike many existing forensic methods, this approach does not rely on toxic chemicals or expensive, high-powered equipment. Instead, it uses environmentally friendly polymers and requires very little energy to produce clear fingerprint images in seconds. Tests showed that this technique also worked on samples aged up to 16 months, demonstrating remarkable durability. The team focused specifically on brass ammunition casings, a substance that has been traditionally resistant to fingerprint detection and is the most common type of material used globally. The researchers believe that the test for fingerprints on brass they have developed could be adapted for other metallic surfaces, expanding its range of potential forensic applications, from firearm-related crimes to arson. Although early results are promising, the new fingerprint recovery method will need further testing and validation before it can be used by law enforcement agencies around the world. “Really Bizarre” – Physicists Uncover a Quantum Material That Breaks All the Rules We Were Not Alone: Earliest Humans Lived Beside Australopithecus, Fossils Reveal Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            